<html lang="en" class="pb-page fonts-loaded fonts-loaded" data-request-id="7020a340-bcf4-41aa-b7d9-ab0e041e52d5" style="--article-sticky-top: 82.00019999999999px; --scroll-padding: 163.594px auto auto;"><head data-pb-dropzone="head"><meta name="pbContext" content=";product:product:elsevier\:product\:cell;website:website:cell-site;issue:issue:pii\:S0092867420X00237;ctype:string:Journal Content;journal:journal:cell;pageGroup:string:Article View;article:article:pii\:S0092867421010655;page:string:Show Full Text;requestedJournal:journal:cell;wgroup:string:Default Website Group;subPage:string:Full Text">




<!-- if there is any customization for Responsive Project widget  -->


























    
    







    
            
        



        
                <meta name="robots" content="noarchive,nocache">
            
    






    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=1, user-scalable=1">











        <meta property="og:title" content="Enhanced prime editing systems by manipulating cellular determinants of editing outcomes">
    


        <meta property="og:type" content="Article">
    


        <meta property="og:url" content="https://www.cell.com/cell/abstract/S0092-8674(21)01065-5">
    



        <meta property="og:site_name" content="Cell">
    


        <meta property="og:image" content="https://www.cell.com/cms/asset/00efaa71-b796-41dd-8aa5-fb8b617fbce4/fx1.jpg">
    


        <meta property="og:description" content="PE4 and PE5 are efficient and precise prime editing systems developed by leveraging
insights into the way DNA repair pathways impact genome editing outcomes">
    











        <meta name="twitter:title" content="Enhanced prime editing systems by manipulating cellular determinants of editing outcomes">
    




        <meta name="twitter:image" content="https://www.cell.com/cms/asset/00efaa71-b796-41dd-8aa5-fb8b617fbce4/fx1.jpg">
    


        <meta name="twitter:description" content="PE4 and PE5 are efficient and precise prime editing systems developed by leveraging
insights into the way DNA repair pathways impact genome editing outcomes">
    

        <meta name="twitter:card" content="summary_large_image">
    











    
    
            
            
        <link rel="canonical" href="https://www.cell.com/cell/fulltext/S0092-8674(21)01065-5">
    


<meta name="tdm-reservation" content="1">
<meta name="tdm-policy" content="https://www.elsevier.com/tdm/tdmrep-policy.json">

<title>Enhanced prime editing systems by manipulating cellular determinants of editing outcomes: Cell</title><meta name="citation_journal_title" content="Cell"> <meta name="citation_journal_abbrev" content="Cell"> <meta name="citation_publisher" content="Elsevier"> <meta name="citation_title" content="Enhanced prime editing systems by manipulating cellular determinants of editing outcomes"> <meta name="citation_date" content="2021/10/28"> <meta name="citation_online_date" content="2021/10/14"> <meta name="citation_volume" content="184"> <meta name="citation_issue" content="22"> <meta name="citation_firstpage" content="5635"> <meta name="citation_lastpage" content="5652.e29"> <meta name="citation_pmid" content="34653350"> <meta name="citation_abstract_html_url" content="https://www.cell.com/action/showAbstract?pii=S0092867421010655"> <meta name="citation_fulltext_html_url" content="https://www.cell.com/action/showFullText?pii=S0092867421010655"> <meta name="citation_pdf_url" content="https://www.cell.com/action/showPdf?pii=S0092867421010655"> <meta name="citation_issn" content="0092-8674"> <meta name="citation_issn" content="1097-4172"> <meta name="citation_language" content="English"> <meta name="citation_doi" content="10.1016/j.cell.2021.09.018"> <meta name="citation_pmid" content="34653350"> <meta name="citation_author" content="Peter J. Chen"><meta name="citation_author_orcid" content="0000-0002-1094-7180"><meta name="citation_author_institution" content="Merkin Institute of Transformative Technologies in Healthcare, Broad Institute of Harvard and MIT, Cambridge, MA 02141, USA"><meta name="citation_author_institution" content="Department of Chemistry and Chemical Biology, Harvard University, Cambridge, MA 02138, USA"><meta name="citation_author_institution" content="Howard Hughes Medical Institute, Harvard University, Cambridge, MA 02138, USA"><meta name="citation_author" content="Jeffrey A. Hussmann"><meta name="citation_author_orcid" content="0000-0002-7778-9893"><meta name="citation_author_institution" content="Department of Cellular and Molecular Pharmacology, University of California, San Francisco, San Francisco, CA 94158, USA"><meta name="citation_author_institution" content="Department of Microbiology and Immunology, University of California, San Francisco, San Francisco, CA 94158, USA"><meta name="citation_author_institution" content="Howard Hughes Medical Institute, University of California, San Francisco, San Francisco, CA 94158, USA"><meta name="citation_author_institution" content="Whitehead Institute for Biomedical Research, Massachusetts Institute of Technology, Cambridge, MA 02142, USA"><meta name="citation_author" content="Jun Yan"><meta name="citation_author_institution" content="Department of Molecular Biology, Princeton University, Princeton, NJ 08544, USA"><meta name="citation_author" content="Friederike Knipping"><meta name="citation_author_institution" content="Department of Pediatrics, University of Minnesota, Minneapolis, MN 55454, USA"><meta name="citation_author_institution" content="Center for Genome Engineering, University of Minnesota, Minneapolis, MN 55108, USA"><meta name="citation_author_institution" content="Stem Cell Institute, University of Minnesota, Minneapolis, MN 55455, USA"><meta name="citation_author" content="Purnima Ravisankar"><meta name="citation_author_institution" content="Lewis-Sigler Institute for Integrative Genomics, Princeton University, Princeton, NJ 08544, USA"><meta name="citation_author" content="Pin-Fang Chen"><meta name="citation_author_institution" content="Human Neuron Core, Rosamund Stone Zander Translational Neuroscience Center, Boston Children’s Hospital, Boston, MA 02115, USA"><meta name="citation_author_institution" content="Department of Neurology, Boston Children’s Hospital, Boston, MA 02115, USA"><meta name="citation_author" content="Cidi Chen"><meta name="citation_author_orcid" content="0000-0001-7112-2230"><meta name="citation_author_institution" content="Human Neuron Core, Rosamund Stone Zander Translational Neuroscience Center, Boston Children’s Hospital, Boston, MA 02115, USA"><meta name="citation_author_institution" content="Department of Neurology, Boston Children’s Hospital, Boston, MA 02115, USA"><meta name="citation_author" content="James W. Nelson"><meta name="citation_author_institution" content="Merkin Institute of Transformative Technologies in Healthcare, Broad Institute of Harvard and MIT, Cambridge, MA 02141, USA"><meta name="citation_author_institution" content="Department of Chemistry and Chemical Biology, Harvard University, Cambridge, MA 02138, USA"><meta name="citation_author_institution" content="Howard Hughes Medical Institute, Harvard University, Cambridge, MA 02138, USA"><meta name="citation_author" content="Gregory A. Newby"><meta name="citation_author_institution" content="Merkin Institute of Transformative Technologies in Healthcare, Broad Institute of Harvard and MIT, Cambridge, MA 02141, USA"><meta name="citation_author_institution" content="Department of Chemistry and Chemical Biology, Harvard University, Cambridge, MA 02138, USA"><meta name="citation_author_institution" content="Howard Hughes Medical Institute, Harvard University, Cambridge, MA 02138, USA"><meta name="citation_author" content="Mustafa Sahin"><meta name="citation_author_orcid" content="0000-0001-7044-2953"><meta name="citation_author_institution" content="Human Neuron Core, Rosamund Stone Zander Translational Neuroscience Center, Boston Children’s Hospital, Boston, MA 02115, USA"><meta name="citation_author_institution" content="Department of Neurology, Boston Children’s Hospital, Boston, MA 02115, USA"><meta name="citation_author_institution" content="Harvard Medical School, Boston, MA 02115, USA"><meta name="citation_author" content="Mark J. Osborn"><meta name="citation_author_institution" content="Department of Pediatrics, University of Minnesota, Minneapolis, MN 55454, USA"><meta name="citation_author_institution" content="Center for Genome Engineering, University of Minnesota, Minneapolis, MN 55108, USA"><meta name="citation_author_institution" content="Stem Cell Institute, University of Minnesota, Minneapolis, MN 55455, USA"><meta name="citation_author" content="Jonathan S. Weissman"><meta name="citation_author_orcid" content="0000-0003-2445-670X"><meta name="citation_author_institution" content="Department of Cellular and Molecular Pharmacology, University of California, San Francisco, San Francisco, CA 94158, USA"><meta name="citation_author_institution" content="Howard Hughes Medical Institute, University of California, San Francisco, San Francisco, CA 94158, USA"><meta name="citation_author_institution" content="Whitehead Institute for Biomedical Research, Massachusetts Institute of Technology, Cambridge, MA 02142, USA"><meta name="citation_author" content="Britt Adamson"><meta name="citation_author_email" content="badamson@princeton.edu"><meta name="citation_author_institution" content="Department of Molecular Biology, Princeton University, Princeton, NJ 08544, USA"><meta name="citation_author_institution" content="Lewis-Sigler Institute for Integrative Genomics, Princeton University, Princeton, NJ 08544, USA"><meta name="citation_author" content="David R. Liu"><meta name="citation_author_email" content="drliu@fas.harvard.edu"><meta name="citation_author_orcid" content="0000-0002-9943-7557"><meta name="citation_author_institution" content="Merkin Institute of Transformative Technologies in Healthcare, Broad Institute of Harvard and MIT, Cambridge, MA 02141, USA"><meta name="citation_author_institution" content="Department of Chemistry and Chemical Biology, Harvard University, Cambridge, MA 02138, USA"><meta name="citation_author_institution" content="Howard Hughes Medical Institute, Harvard University, Cambridge, MA 02138, USA"><meta name="citation_keywords" content="prime editing; genome editing; CRISPR-Cas9; Repair-seq; mismatch repair"><meta name="citation_reference" content="citation_title=hMSH2 forms specific mispair-binding complexes with hMSH3 and hMSH6; citation_author=S. Acharya; citation_author=T. Wilson; citation_author=S. Gradia; citation_author=M.F. Kane; citation_author=S. Guerrette; citation_author=G.T. Marsischky; citation_author=R. Kolodner; citation_author=R. Fishel; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=93; citation_firstpage=13629; citation_lastpage=13634; citation_publication_date=1996; "><meta name="citation_reference" content="citation_title=Interpreting elevated fetal hemoglobin in pathology and health at the basic laboratory level: new and known γ- gene mutations associated with hereditary persistence of fetal hemoglobin; citation_author=A. Amato; citation_author=M.P. Cappabianca; citation_author=M. Perri; citation_author=I. Zaghis; citation_author=P. Grisanti; citation_author=D. Ponzini; citation_author=P. Di Biagio; citation_journal_title=Int. J. Lab. Hematol.; citation_volume=36; citation_firstpage=13; citation_lastpage=19; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Search-and-replace genome editing without double-strand breaks or donor DNA; citation_author=A.V. Anzalone; citation_author=P.B. Randolph; citation_author=J.R. Davis; citation_author=A.A. Sousa; citation_author=L.W. Koblan; citation_author=J.M. Levy; citation_author=P.J. Chen; citation_author=C. Wilson; citation_author=G.A. Newby; citation_author=A. Raguram; citation_author=D.R. Liu; citation_journal_title=Nature; citation_volume=576; citation_firstpage=149; citation_lastpage=157; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Genome editing with CRISPR-Cas nucleases, base editors, transposases and prime; citation_author=A.V. Anzalone; citation_author=L.W. Koblan; citation_author=D.R. Liu; citation_journal_title=Nat. Biotechnol.; citation_volume=38; citation_firstpage=824; citation_lastpage=844; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=A naturally occurring variant of the human prion protein completely prevents prion disease; citation_author=E.A. Asante; citation_author=M. Smidak; citation_author=A. Grimshaw; citation_author=R. Houghton; citation_author=A. Tomlinson; citation_author=A. Jeelani; citation_author=T. Jakubcova; citation_author=S. Hamdan; citation_author=A. Richard-Londt; citation_author=J.M. Linehan; citation_author=et al.; citation_journal_title=Nature; citation_volume=522; citation_firstpage=478; citation_lastpage=481; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Development of a fluorescent multiplex assay for detection of MSI-High tumors; citation_author=J.W. Bacher; citation_author=L.A. Flanagan; citation_author=R.L. Smalley; citation_author=N.A. Nassif; citation_author=L.J. Burgart; citation_author=R.B. Halberg; citation_author=W.M.A. Megid; citation_author=S.N. Thibodeau; citation_journal_title=Dis. Markers; citation_volume=20; citation_firstpage=237; citation_lastpage=250; citation_publication_date=2004; "><meta name="citation_reference" content="citation_title=Insights into the kinetics of siRNA-mediated gene silencing from live-cell and live-animal bioluminescent imaging; citation_author=D.W. Bartlett; citation_author=M.E. Davis; citation_journal_title=Nucleic Acids Res.; citation_volume=34; citation_firstpage=322; citation_lastpage=333; citation_publication_date=2006; "><meta name="citation_reference" content="citation_title=Precise genome engineering in Drosophila using prime editing; citation_author=J.A. Bosch; citation_author=G. Birchak; citation_author=N. Perrimon; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=118; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=Accurate DNA Assembly and Genome Engineering with Optimized Uracil Excision Cloning; citation_author=A.M. Cavaleiro; citation_author=S.H. Kim; citation_author=S. Seppälä; citation_author=M.T. Nielsen; citation_author=M.H.H. Nørholm; citation_journal_title=ACS Synth. Biol.; citation_volume=4; citation_firstpage=1042; citation_lastpage=1046; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Generation and characterization of human induced pluripotent stem cells (iPSCs) from three male and three female patients with CDKL5 Deficiency Disorder (CDD); citation_author=P.-F. Chen; citation_author=T. Chen; citation_author=T.E. Forman; citation_author=A.C. Swanson; citation_author=B. O’Kelly; citation_author=S.A. Dwyer; citation_author=E.D. Buttermore; citation_author=R. Kleiman; citation_author=S. Js Carrington; citation_author=D.J. Lavery; citation_author=et al.; citation_journal_title=Stem Cell Res. (Amst.); citation_volume=53; citation_firstpage=102276; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=CRISPResso2 provides accurate and rapid genome editing sequence analysis; citation_author=K. Clement; citation_author=H. Rees; citation_author=M.C. Canver; citation_author=J.M. Gehrke; citation_author=R. Farouni; citation_author=J.Y. Hsu; citation_author=M.A. Cole; citation_author=D.R. Liu; citation_author=J.K. Joung; citation_author=D.E. Bauer; citation_author=L. Pinello; citation_journal_title=Nat. Biotechnol.; citation_volume=37; citation_firstpage=224; citation_lastpage=226; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Identification of the human c-myc protein nuclear translocation signal; citation_author=C.V. Dang; citation_author=W.M. Lee; citation_journal_title=Mol. Cell. Biol.; citation_volume=8; citation_firstpage=4048; citation_lastpage=4054; citation_publication_date=1988; "><meta name="citation_reference" content="citation_title=A one pot, one step, precision cloning method with high throughput capability; citation_author=C. Engler; citation_author=R. Kandzia; citation_author=S. Marillonnet; citation_journal_title=PLoS ONE; citation_volume=3; citation_firstpage=e3647; citation_publication_date=2008; "><meta name="citation_reference" content="citation_title=Human strand-specific mismatch repair occurs by a bidirectional mechanism similar to that of the bacterial reaction; citation_author=W.H. Fang; citation_author=P. Modrich; citation_journal_title=J.&nbsp;Biol. Chem.; citation_volume=268; citation_firstpage=11838; citation_lastpage=11844; citation_publication_date=1993; "><meta name="citation_reference" content="citation_title=In&nbsp;vitro studies of DNA mismatch repair proteins; citation_author=H. Geng; citation_author=C. Du; citation_author=S. Chen; citation_author=V. Salerno; citation_author=C. Manfredi; citation_author=P. Hsieh; citation_journal_title=Anal. Biochem.; citation_volume=413; citation_firstpage=179; citation_lastpage=184; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=Isolation of MutSbeta from human cells and comparison of the mismatch repair specificities of MutSbeta and MutSalpha; citation_author=J. Genschel; citation_author=S.J. Littman; citation_author=J.T. Drummond; citation_author=P. Modrich; citation_journal_title=J.&nbsp;Biol. Chem.; citation_volume=273; citation_firstpage=19895; citation_lastpage=19901; citation_publication_date=1998; "><meta name="citation_reference" content="citation_title=Human exonuclease I is required for 5′ and 3′ mismatch repair; citation_author=J. Genschel; citation_author=L.R. Bazemore; citation_author=P. Modrich; citation_journal_title=J.&nbsp;Biol. Chem.; citation_volume=277; citation_firstpage=13302; citation_lastpage=13311; citation_publication_date=2002; "><meta name="citation_reference" content="citation_title=CRISPR-mediated modular RNA-guided regulation of transcription in eukaryotes; citation_author=L.A. Gilbert; citation_author=M.H. Larson; citation_author=L. Morsut; citation_author=Z. Liu; citation_author=G.A. Brar; citation_author=S.E. Torres; citation_author=N. Stern-Ginossar; citation_author=O. Brandman; citation_author=E.H. Whitehead; citation_author=J.A. Doudna; citation_author=et al.; citation_journal_title=Cell; citation_volume=154; citation_firstpage=442; citation_lastpage=451; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=Genome-Scale CRISPR-Mediated Control of Gene Repression and Activation; citation_author=L.A. Gilbert; citation_author=M.A. Horlbeck; citation_author=B. Adamson; citation_author=J.E. Villalta; citation_author=Y. Chen; citation_author=E.H. Whitehead; citation_author=C. Guimaraes; citation_author=B. Panning; citation_author=H.L. Ploegh; citation_author=M.C. Bassik; citation_author=et al.; citation_journal_title=Cell; citation_volume=159; citation_firstpage=647; citation_lastpage=661; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Structure of the MutLα C-terminal domain reveals how Mlh1 contributes to Pms1 endonuclease site; citation_author=E. Gueneau; citation_author=C. Dherin; citation_author=P. Legrand; citation_author=C. Tellier-Lebegue; citation_author=B. Gilquin; citation_author=P. Bonnesoeur; citation_author=F. Londino; citation_author=C. Quemener; citation_author=M.-H. Le Du; citation_author=J.A. Márquez; citation_author=et al.; citation_journal_title=Nat. Struct. Mol. Biol.; citation_volume=20; citation_firstpage=461; citation_lastpage=468; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=The interaction of the human MutL homologues in hereditary nonpolyposis colon cancer; citation_author=S. Guerrette; citation_author=S. Acharya; citation_author=R. Fishel; citation_journal_title=J.&nbsp;Biol. Chem.; citation_volume=274; citation_firstpage=6336; citation_lastpage=6341; citation_publication_date=1999; "><meta name="citation_reference" content="citation_title=Mechanism of mismatch recognition revealed by human MutSβ bound to unpaired DNA loops; citation_author=S. Gupta; citation_author=M. Gellert; citation_author=W. Yang; citation_journal_title=Nat. Struct. Mol. Biol.; citation_volume=19; citation_firstpage=72; citation_lastpage=78; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=MSIplus for Integrated Colorectal Cancer Molecular Testing by Next-Generation Sequencing; citation_author=J.A. Hempelmann; citation_author=S.M. Scroggins; citation_author=C.C. Pritchard; citation_author=S.J. Salipante; citation_journal_title=J.&nbsp;Mol. Diagn.; citation_volume=17; citation_firstpage=705; citation_lastpage=714; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Strand-specific mismatch correction in nuclear extracts of human and Drosophila melanogaster cell lines; citation_author=J. Holmes; citation_author=S. Clark; citation_author=P. Modrich; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=87; citation_firstpage=5837; citation_lastpage=5841; citation_publication_date=1990; "><meta name="citation_reference" content="citation_title=Mapping the genetic landscape of DNA double-strand break repair; citation_author=J.A. Hussmann; citation_author=J. Ling; citation_author=P. Ravisankar; citation_author=J. Yan; citation_author=A. Cirincione; citation_author=A. Xu; citation_author=D. Simpson; citation_author=D. Yang; citation_author=A. Bothmer; citation_author=C. Cotta-Ramusino; citation_author=J.S. Weissman; citation_author=B. Adamson; citation_journal_title=Cell; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=hMSH2 and hMSH6 play distinct roles in mismatch binding and contribute differently to the ATPase activity of hMutSalpha; citation_author=I. Iaccarino; citation_author=G. Marra; citation_author=F. Palombo; citation_author=J. Jiricny; citation_journal_title=EMBO J.; citation_volume=17; citation_firstpage=2677; citation_lastpage=2686; citation_publication_date=1998; "><meta name="citation_reference" content="citation_title=A specific chemical difference between the globins of normal human and sickle-cell anaemia haemoglobin; citation_author=V.M. Ingram; citation_journal_title=Nature; citation_volume=178; citation_firstpage=792; citation_lastpage=794; citation_publication_date=1956; "><meta name="citation_reference" content="citation_title=DNA mismatch repair: functions and mechanisms; citation_author=R.R. Iyer; citation_author=A. Pluciennik; citation_author=V. Burdett; citation_author=P.L. Modrich; citation_journal_title=Chem. Rev.; citation_volume=106; citation_firstpage=302; citation_lastpage=323; citation_publication_date=2006; "><meta name="citation_reference" content="citation_title=Genome-wide specificity of prime editors in plants; citation_author=S. Jin; citation_author=Q. Lin; citation_author=Y. Luo; citation_author=Z. Zhu; citation_author=G. Liu; citation_author=Y. Li; citation_author=K. Chen; citation_author=J.-L. Qiu; citation_author=C. Gao; citation_journal_title=Nat. Biotechnol.; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=Endonucleolytic function of MutLalpha in human mismatch repair; citation_author=F.A. Kadyrov; citation_author=L. Dzantiev; citation_author=N. Constantin; citation_author=P. Modrich; citation_journal_title=Cell; citation_volume=126; citation_firstpage=297; citation_lastpage=308; citation_publication_date=2006; "><meta name="citation_reference" content="citation_title=Unbiased investigation of specificities of prime editing systems in human cells; citation_author=D.Y. Kim; citation_author=S.B. Moon; citation_author=J.-H. Ko; citation_author=Y.-S. Kim; citation_author=D. Kim; citation_journal_title=Nucleic Acids Res.; citation_volume=48; citation_firstpage=10576; citation_lastpage=10589; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=DNA mismatch repair; citation_author=T.A. Kunkel; citation_author=D.A. Erie; citation_journal_title=Annu. Rev. Biochem.; citation_volume=74; citation_firstpage=681; citation_lastpage=710; citation_publication_date=2005; "><meta name="citation_reference" content="citation_title=DNA mismatch correction in a defined system; citation_author=R.S. Lahue; citation_author=K.G. Au; citation_author=P. Modrich; citation_journal_title=Science; citation_volume=245; citation_firstpage=160; citation_lastpage=164; citation_publication_date=1989; "><meta name="citation_reference" content="citation_title=Mechanisms and functions of DNA mismatch repair; citation_author=G.-M. Li; citation_journal_title=Cell Res.; citation_volume=18; citation_firstpage=85; citation_lastpage=98; citation_publication_date=2008; "><meta name="citation_reference" content="citation_title=Prime genome editing in rice and wheat; citation_author=Q. Lin; citation_author=Y. Zong; citation_author=C. Xue; citation_author=S. Wang; citation_author=S. Jin; citation_author=Z. Zhu; citation_author=Y. Wang; citation_author=A.V. Anzalone; citation_author=A. Raguram; citation_author=J.L. Doman; citation_author=et al.; citation_journal_title=Nat. Biotechnol.; citation_volume=38; citation_firstpage=582; citation_lastpage=585; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=HIV-1 inhibition in cells with CXCR4 mutant genome created by CRISPR-Cas9 and piggyBac recombinant technologies; citation_author=S. Liu; citation_author=Q. Wang; citation_author=X. Yu; citation_author=Y. Li; citation_author=Y. Guo; citation_author=Z. Liu; citation_author=F. Sun; citation_author=W. Hou; citation_author=C. Li; citation_author=L. Wu; citation_author=et al.; citation_journal_title=Sci. Rep.; citation_volume=8; citation_firstpage=8573; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Flap endonuclease 1: a central component of DNA metabolism; citation_author=Y. Liu; citation_author=H.-I. Kao; citation_author=R.A. Bambara; citation_journal_title=Annu. Rev. Biochem.; citation_volume=73; citation_doi=10.1146/annurev.biochem.73.012803.092453; citation_firstpage=589; citation_lastpage=615; citation_publication_date=2004; "><meta name="citation_reference" content="citation_title=Efficient generation of mouse models with the prime editing system; citation_author=Y. Liu; citation_author=X. Li; citation_author=S. He; citation_author=S. Huang; citation_author=C. Li; citation_author=Y. Chen; citation_author=Z. Liu; citation_author=X. Huang; citation_author=X. Wang; citation_journal_title=Cell Discov.; citation_volume=6; citation_firstpage=27; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Improved prime editors enable pathogenic allele correction and cancer modelling in adult mice; citation_author=P. Liu; citation_author=S.-Q. Liang; citation_author=C. Zheng; citation_author=E. Mintzer; citation_author=Y.G. Zhao; citation_author=K. Ponnienselvan; citation_author=A. Mir; citation_author=E.J. Sontheimer; citation_author=G. Gao; citation_author=T.R. Flotte; citation_author=et al.; citation_journal_title=Nat. Commun.; citation_volume=12; citation_firstpage=2121; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=Heterogeneous polymerase fidelity and mismatch repair bias genome variation and composition; citation_author=S.A. Lujan; citation_author=A.R. Clausen; citation_author=A.B. Clark; citation_author=H.K. MacAlpine; citation_author=D.M. MacAlpine; citation_author=E.P. Malc; citation_author=P.A. Mieczkowski; citation_author=A.B. Burkholder; citation_author=D.C. Fargo; citation_author=D.A. Gordenin; citation_author=T.A. Kunkel; citation_journal_title=Genome Res.; citation_volume=24; citation_firstpage=1751; citation_lastpage=1764; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Assessment of mismatch repair function in leukaemic cell lines and blasts from children with acute lymphoblastic leukaemia; citation_author=E.C. Matheson; citation_author=A.G. Hall; citation_journal_title=Carcinogenesis; citation_volume=24; citation_firstpage=31; citation_lastpage=38; citation_publication_date=2003; "><meta name="citation_reference" content="citation_title=A novel protective prion protein variant that colocalizes with kuru exposure; citation_author=S. Mead; citation_author=J. Whitfield; citation_author=M. Poulter; citation_author=P. Shah; citation_author=J. Uphill; citation_author=T. Campbell; citation_author=H. Al-Dujaily; citation_author=H. Hummerich; citation_author=J. Beck; citation_author=C.A. Mein; citation_author=et al.; citation_journal_title=N.&nbsp;Engl. J. Med.; citation_volume=361; citation_firstpage=2056; citation_lastpage=2065; citation_publication_date=2009; "><meta name="citation_reference" content="citation_title=Engineered pegRNAs improve prime editing efficiency; citation_author=J.W. Nelson; citation_author=P.B. Randolph; citation_author=S.P. Shen; citation_author=K.A. Everette; citation_author=P.J. Chen; citation_author=A.V. Anzalone; citation_author=G.A. Newby; citation_author=M. An; citation_author=J.C. Chen; citation_author=A. Hsu; citation_author=D.R. Liu; citation_journal_title=Nat. Biotechnol.; citation_doi=10.1038/s41587-021-01039-7; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=Cyclin-Dependent Kinase-Like 5 Deficiency Disorder: Clinical Review; citation_author=H.E. Olson; citation_author=S.T. Demarest; citation_author=E.M. Pestana-Knight; citation_author=L.C. Swanson; citation_author=S. Iqbal; citation_author=D. Lal; citation_author=H. Leonard; citation_author=J.H. Cross; citation_author=O. Devinsky; citation_author=T.A. Benke; citation_journal_title=Pediatr. Neurol.; citation_volume=97; citation_firstpage=18; citation_lastpage=25; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Targeted mutagenesis in mouse cells and embryos using an enhanced prime; citation_author=S.-J. Park; citation_author=T.Y. Jeong; citation_author=S.K. Shin; citation_author=D.E. Yoon; citation_author=S.-Y. Lim; citation_author=S.P. Kim; citation_author=J. Choi; citation_author=H. Lee; citation_author=J.-I. Hong; citation_author=J. Ahn; citation_author=et al.; citation_journal_title=Genome Biol.; citation_volume=22; citation_firstpage=170; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=Hypermutability and mismatch repair deficiency in RER+ tumor cells; citation_author=R. Parsons; citation_author=G.-M. Li; citation_author=M.J. Longley; citation_author=W.-H. Fang; citation_author=N. Papadopoulos; citation_author=J. Jen; citation_author=A. de la Chapelle; citation_author=K.W. Kinzler; citation_author=B. Vogelstein; citation_author=P. Modrich; citation_journal_title=Cell; citation_volume=75; citation_firstpage=1227; citation_lastpage=1236; citation_publication_date=1993; "><meta name="citation_reference" content="citation_title=Human DNA ligase I completely encircles and partially unwinds nicked DNA; citation_author=J.M. Pascal; citation_author=P.J. O’Brien; citation_author=A.E. Tomkinson; citation_author=T. Ellenberger; citation_journal_title=Nature; citation_volume=432; citation_doi=10.1038/nature03082; citation_firstpage=473; citation_lastpage=478; citation_publication_date=2004; "><meta name="citation_reference" content="citation_title=Crosstalk between BRCA-Fanconi anemia and mismatch repair pathways prevents MSH2-dependent aberrant DNA damage responses; citation_author=M. Peng; citation_author=J. Xie; citation_author=A. Ucher; citation_author=J. Stavnezer; citation_author=S.B. Cantor; citation_journal_title=EMBO J.; citation_volume=33; citation_firstpage=1698; citation_lastpage=1712; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=CRISPR prime editing with ribonucleoprotein complexes in zebrafish and primary human cells; citation_author=K. Petri; citation_author=W. Zhang; citation_author=J. Ma; citation_author=A. Schmidts; citation_author=H. Lee; citation_author=J.E. Horng; citation_author=D.Y. Kim; citation_author=I.C. Kurt; citation_author=K. Clement; citation_author=J.Y. Hsu; citation_author=et al.; citation_journal_title=Nat. Biotechnol.; citation_doi=10.1038/s41587-021-00901-y; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=N-terminus of hMLH1 confers interaction of hMutLalpha and hMutLbeta with hMutSalpha; citation_author=G. Plotz; citation_author=J. Raedle; citation_author=A. Brieger; citation_author=J. Trojan; citation_author=S. Zeuzem; citation_journal_title=Nucleic Acids Res.; citation_volume=31; citation_firstpage=3217; citation_lastpage=3226; citation_publication_date=2003; "><meta name="citation_reference" content="citation_title=PCNA function in the activation and strand direction of MutLα endonuclease in mismatch repair; citation_author=A. Pluciennik; citation_author=L. Dzantiev; citation_author=R.R. Iyer; citation_author=N. Constantin; citation_author=F.A. Kadyrov; citation_author=P. Modrich; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=107; citation_firstpage=16066; citation_lastpage=16071; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=In&nbsp;vivo genome editing using Staphylococcus aureus Cas9; citation_author=F.A. Ran; citation_author=L. Cong; citation_author=W.X. Yan; citation_author=D.A. Scott; citation_author=J.S. Gootenberg; citation_author=A.J. Kriz; citation_author=B. Zetsche; citation_author=O. Shalem; citation_author=X. Wu; citation_author=K.S. Makarova; citation_author=et al.; citation_journal_title=Nature; citation_volume=520; citation_firstpage=186; citation_lastpage=191; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Mutations within the hMLH1 and hPMS2 subunits of the human MutLalpha mismatch repair factor affect its ATPase activity, but not its ability to interact with hMutSalpha; citation_author=M. Räschle; citation_author=P. Dufner; citation_author=G. Marra; citation_author=J. Jiricny; citation_journal_title=J.&nbsp;Biol. Chem.; citation_volume=277; citation_firstpage=21810; citation_lastpage=21820; citation_publication_date=2002; "><meta name="citation_reference" content="citation_title=Prime editing for functional repair in patient-derived disease models; citation_author=I.F. Schene; citation_author=I.P. Joore; citation_author=R. Oka; citation_author=M. Mokry; citation_author=A.H.M. van Vugt; citation_author=R. van Boxtel; citation_author=H.P.J. van der Doef; citation_author=L.J.W. van der Laan; citation_author=M.M.A. Verstegen; citation_author=P.M. van Hasselt; citation_author=et al.; citation_journal_title=Nat. Commun.; citation_volume=11; citation_firstpage=5352; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Selective targeting of engineered T&nbsp;cells using orthogonal IL-2 cytokine-receptor complexes; citation_author=J.T. Sockolosky; citation_author=E. Trotta; citation_author=G. Parisi; citation_author=L. Picton; citation_author=L.L. Su; citation_author=A.C. Le; citation_author=A. Chhabra; citation_author=S.L. Silveria; citation_author=B.M. George; citation_author=I.C. King; citation_author=et al.; citation_journal_title=Science; citation_volume=359; citation_firstpage=1037; citation_lastpage=1042; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Deep mutational scanning of S. pyogenes Cas9 reveals important functional domains; citation_author=J.M. Spencer; citation_author=X. Zhang; citation_journal_title=Sci. Rep.; citation_volume=7; citation_firstpage=16836; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Destabilization of tracts of simple repetitive DNA in yeast by mutations affecting DNA mismatch repair; citation_author=M. Strand; citation_author=T.A. Prolla; citation_author=R.M. Liskay; citation_author=T.D. Petes; citation_journal_title=Nature; citation_volume=365; citation_firstpage=274; citation_lastpage=276; citation_publication_date=1993; "><meta name="citation_reference" content="citation_title=Mispair specificity of methyl-directed DNA mismatch correction in&nbsp;vitro; citation_author=S.S. Su; citation_author=R.S. Lahue; citation_author=K.G. Au; citation_author=P. Modrich; citation_journal_title=J.&nbsp;Biol. Chem.; citation_volume=263; citation_firstpage=6829; citation_lastpage=6835; citation_publication_date=1988; "><meta name="citation_reference" content="citation_title=Heteroduplex rejection during single-strand annealing requires Sgs1 helicase and mismatch repair proteins Msh2 and Msh6 but not Pms1; citation_author=N. Sugawara; citation_author=T. Goldfarb; citation_author=B. Studamire; citation_author=E. Alani; citation_author=J.E. Haber; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=101; citation_firstpage=9315; citation_lastpage=9320; citation_publication_date=2004; "><meta name="citation_reference" content="citation_title=Differential DNA mismatch repair underlies mutation rate variation across the human genome; citation_author=F. Supek; citation_author=B. Lehner; citation_journal_title=Nature; citation_volume=521; citation_firstpage=81; citation_lastpage=84; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Efficient Generation and Correction of Mutations in Human iPS Cells Utilizing mRNAs of CRISPR Base Editors and Prime; citation_author=D. Sürün; citation_author=A. Schneider; citation_author=J. Mircetic; citation_author=K. Neumann; citation_author=F. Lansing; citation_author=M. Paszkowski-Rogacz; citation_author=V. Hänchen; citation_author=M.A. Lee-Kirsch; citation_author=F. Buchholz; citation_journal_title=Genes (Basel); citation_volume=11; citation_firstpage=511; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Heteroduplex repair in extracts of human HeLa cells; citation_author=D.C. Thomas; citation_author=J.D. Roberts; citation_author=T.A. Kunkel; citation_journal_title=J.&nbsp;Biol. Chem.; citation_volume=266; citation_firstpage=3744; citation_lastpage=3751; citation_publication_date=1991; "><meta name="citation_reference" content="citation_title=Contribution of human mlh1 and pms2 ATPase activities to DNA mismatch repair; citation_author=G. Tomer; citation_author=A.B. Buermeyer; citation_author=M.M. Nguyen; citation_author=R.M. Liskay; citation_journal_title=J.&nbsp;Biol. Chem.; citation_volume=277; citation_firstpage=21801; citation_lastpage=21809; citation_publication_date=2002; "><meta name="citation_reference" content="citation_title=Hypermutability of homonucleotide runs in mismatch repair and DNA polymerase proofreading yeast mutants; citation_author=H.T. Tran; citation_author=J.D. Keen; citation_author=M. Kricker; citation_author=M.A. Resnick; citation_author=D.A. Gordenin; citation_journal_title=Mol. Cell. Biol.; citation_volume=17; citation_firstpage=2859; citation_lastpage=2865; citation_publication_date=1997; "><meta name="citation_reference" content="citation_title=Functional analysis of hMLH1 variants and HNPCC-related mutations using a human expression system; citation_author=J. Trojan; citation_author=S. Zeuzem; citation_author=A. Randolph; citation_author=C. Hemmerle; citation_author=A. Brieger; citation_author=J. Raedle; citation_author=G. Plotz; citation_author=J. Jiricny; citation_author=G. Marra; citation_journal_title=Gastroenterology; citation_volume=122; citation_firstpage=211; citation_lastpage=219; citation_publication_date=2002; "><meta name="citation_reference" content="citation_title=CIRCLE-seq: a highly sensitive in&nbsp;vitro screen for genome-wide CRISPR-Cas9 nuclease off-targets; citation_author=S.Q. Tsai; citation_author=N.T. Nguyen; citation_author=J. Malagon-Lopez; citation_author=V.V. Topkar; citation_author=M.J. Aryee; citation_author=J.K. Joung; citation_journal_title=Nat. Methods; citation_volume=14; citation_firstpage=607; citation_lastpage=614; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=DNA loop repair by human cell extracts; citation_author=A. Umar; citation_author=J.C. Boyer; citation_author=T.A. Kunkel; citation_journal_title=Science; citation_volume=266; citation_firstpage=814; citation_lastpage=816; citation_publication_date=1994; "><meta name="citation_reference" content="citation_title=Revised Bethesda Guidelines for hereditary nonpolyposis colorectal cancer (Lynch syndrome) and microsatellite instability; citation_author=A. Umar; citation_author=C.R. Boland; citation_author=J.P. Terdiman; citation_author=S. Syngal; citation_author=A. de la Chapelle; citation_author=J. Rüschoff; citation_author=R. Fishel; citation_author=N.M. Lindor; citation_author=L.J. Burgart; citation_author=R. Hamelin; citation_author=et al.; citation_journal_title=J.&nbsp;Natl. Cancer Inst.; citation_volume=96; citation_firstpage=261; citation_lastpage=268; citation_publication_date=2004; "><meta name="citation_reference" content="citation_title=Structure of the human MutSalpha DNA lesion recognition complex; citation_author=J.J. Warren; citation_author=T.J. Pohlhaus; citation_author=A. Changela; citation_author=R.R. Iyer; citation_author=P.L. Modrich; citation_author=L.S. Beese; citation_journal_title=Mol. Cell; citation_volume=26; citation_firstpage=579; citation_lastpage=592; citation_publication_date=2007; "><meta name="citation_reference" content="citation_title=The intracellular mobility of nuclear import receptors and NLS cargoes; citation_author=J. Wu; citation_author=A.H. Corbett; citation_author=K.M. Berland; citation_journal_title=Biophys. J.; citation_volume=96; citation_firstpage=3840; citation_lastpage=3849; citation_publication_date=2009; "><meta name="citation_reference" content="citation_title=Reconstitution of 5′-directed human mismatch repair in a purified system; citation_author=Y. Zhang; citation_author=F. Yuan; citation_author=S.R. Presnell; citation_author=K. Tian; citation_author=Y. Gao; citation_author=A.E. Tomkinson; citation_author=L. Gu; citation_author=G.-M. Li; citation_journal_title=Cell; citation_volume=122; citation_firstpage=693; citation_lastpage=705; citation_publication_date=2005; "><meta name="citation_reference" content="citation_title=Cytoplasmic localization of p21Cip1/WAF1 by Akt-induced phosphorylation in HER-2/neu-overexpressing cells; citation_author=B.P. Zhou; citation_author=Y. Liao; citation_author=W. Xia; citation_author=B. Spohn; citation_author=M.-H. Lee; citation_author=M.-C. Hung; citation_journal_title=Nat. Cell Biol.; citation_volume=3; citation_firstpage=245; citation_lastpage=252; citation_publication_date=2001; "><meta name="citation_reference" content="citation_title=A systematic CRISPR screen defines mutational mechanisms underpinning signatures caused by replication errors and endogenous DNA damage; citation_author=X. Zou; citation_author=G.C.C. Koh; citation_author=A.S. Nanda; citation_author=A. Degasperi; citation_author=K. Urgo; citation_author=T.I. Roumeliotis; citation_author=C.A. Agu; citation_author=C. Badja; citation_author=S. Momen; citation_author=J. Young; citation_author=et al.; citation_journal_title=Nat. Cancer; citation_volume=2; citation_firstpage=643; citation_lastpage=657; citation_publication_date=2021; "><meta name="citation_abstract" content="<h2>Summary</h2><p>While prime editing enables precise sequence changes in DNA, cellular determinants of prime editing remain poorly understood. Using pooled CRISPRi screens, we discovered that DNA mismatch repair (MMR) impedes prime editing and promotes undesired indel byproducts. We developed PE4 and PE5 prime editing systems in which transient expression of an engineered MMR-inhibiting protein enhances the efficiency of substitution, small insertion, and small deletion prime edits by an average 7.7-fold and 2.0-fold compared to PE2 and PE3 systems, respectively, while improving edit/indel ratios by 3.4-fold in MMR-proficient cell types. Strategic installation of silent mutations near the intended edit can enhance prime editing outcomes by evading MMR. Prime editor protein optimization resulted in a PEmax architecture that enhances editing efficacy by 2.8-fold on average in HeLa cells. These findings enrich our understanding of prime editing and establish prime editing systems that show substantial improvement across 191 edits in seven mammalian cell types.</p>"><meta charset="UTF-8"><!--

    Pagebuild admin UI
-->



<script type="text/javascript" src="https://bam.nr-data.net/1/16e2192192?a=916957862&amp;sa=1&amp;v=1177.96a4d39&amp;t=Unnamed%20Transaction&amp;rst=8539&amp;ck=1&amp;ref=https://www.cell.com/cell/fulltext/S0092-8674(21)01065-5&amp;be=3135&amp;fe=8499&amp;dc=7116&amp;af=err,xhr,stn,ins,spa&amp;perf=%7B%22timing%22:%7B%22of%22:1749591884504,%22n%22:0,%22f%22:2552,%22dn%22:2553,%22dne%22:2553,%22c%22:2553,%22s%22:2553,%22ce%22:2553,%22rq%22:2553,%22rp%22:3119,%22rpe%22:3844,%22dl%22:3125,%22di%22:4077,%22ds%22:7116,%22de%22:7127,%22dc%22:8498,%22l%22:8498,%22le%22:8529%7D,%22navigation%22:%7B%7D%7D&amp;jsonp=NREUM.setToken"></script><script src="https://js-agent.newrelic.com/nr-spa-1177.min.js"></script><script type="text/javascript">
    ;window.NREUM||(NREUM={});NREUM.init={privacy:{cookies_enabled:true}};
    window.NREUM||(NREUM={}),__nr_require=function(t,e,n){function r(n){if(!e[n]){var o=e[n]={exports:{}};t[n][0].call(o.exports,function(e){var o=t[n][1][e];return r(o||e)},o,o.exports)}return e[n].exports}if("function"==typeof __nr_require)return __nr_require;for(var o=0;o<n.length;o++)r(n[o]);return r}({1:[function(t,e,n){function r(t){try{c.console&&console.log(t)}catch(e){}}var o,i=t("ee"),a=t(26),c={};try{o=localStorage.getItem("__nr_flags").split(","),console&&"function"==typeof console.log&&(c.console=!0,o.indexOf("dev")!==-1&&(c.dev=!0),o.indexOf("nr_dev")!==-1&&(c.nrDev=!0))}catch(s){}c.nrDev&&i.on("internal-error",function(t){r(t.stack)}),c.dev&&i.on("fn-err",function(t,e,n){r(n.stack)}),c.dev&&(r("NR AGENT IN DEVELOPMENT MODE"),r("flags: "+a(c,function(t,e){return t}).join(", ")))},{}],2:[function(t,e,n){function r(t,e,n,r,c){try{l?l-=1:o(c||new UncaughtException(t,e,n),!0)}catch(f){try{i("ierr",[f,s.now(),!0])}catch(d){}}return"function"==typeof u&&u.apply(this,a(arguments))}function UncaughtException(t,e,n){this.message=t||"Uncaught error with no additional information",this.sourceURL=e,this.line=n}function o(t,e){var n=e?null:s.now();i("err",[t,n])}var i=t("handle"),a=t(27),c=t("ee"),s=t("loader"),f=t("gos"),u=window.onerror,d=!1,p="nr@seenError",l=0;s.features.err=!0,t(1),window.onerror=r;try{throw new Error}catch(h){"stack"in h&&(t(13),t(12),"addEventListener"in window&&t(6),s.xhrWrappable&&t(14),d=!0)}c.on("fn-start",function(t,e,n){d&&(l+=1)}),c.on("fn-err",function(t,e,n){d&&!n[p]&&(f(n,p,function(){return!0}),this.thrown=!0,o(n))}),c.on("fn-end",function(){d&&!this.thrown&&l>0&&(l-=1)}),c.on("internal-error",function(t){i("ierr",[t,s.now(),!0])})},{}],3:[function(t,e,n){t("loader").features.ins=!0},{}],4:[function(t,e,n){function r(){_++,T=g.hash,this[u]=y.now()}function o(){_--,g.hash!==T&&i(0,!0);var t=y.now();this[h]=~~this[h]+t-this[u],this[d]=t}function i(t,e){E.emit("newURL",[""+g,e])}function a(t,e){t.on(e,function(){this[e]=y.now()})}var c="-start",s="-end",f="-body",u="fn"+c,d="fn"+s,p="cb"+c,l="cb"+s,h="jsTime",m="fetch",v="addEventListener",w=window,g=w.location,y=t("loader");if(w[v]&&y.xhrWrappable){var x=t(10),b=t(11),E=t(8),R=t(6),O=t(13),N=t(7),M=t(14),P=t(9),S=t("ee"),C=S.get("tracer");t(16),y.features.spa=!0;var T,_=0;S.on(u,r),S.on(p,r),S.on(d,o),S.on(l,o),S.buffer([u,d,"xhr-done","xhr-resolved"]),R.buffer([u]),O.buffer(["setTimeout"+s,"clearTimeout"+c,u]),M.buffer([u,"new-xhr","send-xhr"+c]),N.buffer([m+c,m+"-done",m+f+c,m+f+s]),E.buffer(["newURL"]),x.buffer([u]),b.buffer(["propagate",p,l,"executor-err","resolve"+c]),C.buffer([u,"no-"+u]),P.buffer(["new-jsonp","cb-start","jsonp-error","jsonp-end"]),a(M,"send-xhr"+c),a(S,"xhr-resolved"),a(S,"xhr-done"),a(N,m+c),a(N,m+"-done"),a(P,"new-jsonp"),a(P,"jsonp-end"),a(P,"cb-start"),E.on("pushState-end",i),E.on("replaceState-end",i),w[v]("hashchange",i,!0),w[v]("load",i,!0),w[v]("popstate",function(){i(0,_>1)},!0)}},{}],5:[function(t,e,n){function r(t){}if(window.performance&&window.performance.timing&&window.performance.getEntriesByType){var o=t("ee"),i=t("handle"),a=t(13),c=t(12),s="learResourceTimings",f="addEventListener",u="resourcetimingbufferfull",d="bstResource",p="resource",l="-start",h="-end",m="fn"+l,v="fn"+h,w="bstTimer",g="pushState",y=t("loader");y.features.stn=!0,t(8),"addEventListener"in window&&t(6);var x=NREUM.o.EV;o.on(m,function(t,e){var n=t[0];n instanceof x&&(this.bstStart=y.now())}),o.on(v,function(t,e){var n=t[0];n instanceof x&&i("bst",[n,e,this.bstStart,y.now()])}),a.on(m,function(t,e,n){this.bstStart=y.now(),this.bstType=n}),a.on(v,function(t,e){i(w,[e,this.bstStart,y.now(),this.bstType])}),c.on(m,function(){this.bstStart=y.now()}),c.on(v,function(t,e){i(w,[e,this.bstStart,y.now(),"requestAnimationFrame"])}),o.on(g+l,function(t){this.time=y.now(),this.startPath=location.pathname+location.hash}),o.on(g+h,function(t){i("bstHist",[location.pathname+location.hash,this.startPath,this.time])}),f in window.performance&&(window.performance["c"+s]?window.performance[f](u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["c"+s]()},!1):window.performance[f]("webkit"+u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["webkitC"+s]()},!1)),document[f]("scroll",r,{passive:!0}),document[f]("keypress",r,!1),document[f]("click",r,!1)}},{}],6:[function(t,e,n){function r(t){for(var e=t;e&&!e.hasOwnProperty(u);)e=Object.getPrototypeOf(e);e&&o(e)}function o(t){c.inPlace(t,[u,d],"-",i)}function i(t,e){return t[1]}var a=t("ee").get("events"),c=t("wrap-function")(a,!0),s=t("gos"),f=XMLHttpRequest,u="addEventListener",d="removeEventListener";e.exports=a,"getPrototypeOf"in Object?(r(document),r(window),r(f.prototype)):f.prototype.hasOwnProperty(u)&&(o(window),o(f.prototype)),a.on(u+"-start",function(t,e){var n=t[1],r=s(n,"nr@wrapped",function(){function t(){if("function"==typeof n.handleEvent)return n.handleEvent.apply(n,arguments)}var e={object:t,"function":n}[typeof n];return e?c(e,"fn-",null,e.name||"anonymous"):n});this.wrapped=t[1]=r}),a.on(d+"-start",function(t){t[1]=this.wrapped||t[1]})},{}],7:[function(t,e,n){function r(t,e,n){var r=t[e];"function"==typeof r&&(t[e]=function(){var t=i(arguments),e={};o.emit(n+"before-start",[t],e);var a;e[m]&&e[m].dt&&(a=e[m].dt);var c=r.apply(this,t);return o.emit(n+"start",[t,a],c),c.then(function(t){return o.emit(n+"end",[null,t],c),t},function(t){throw o.emit(n+"end",[t],c),t})})}var o=t("ee").get("fetch"),i=t(27),a=t(26);e.exports=o;var c=window,s="fetch-",f=s+"body-",u=["arrayBuffer","blob","json","text","formData"],d=c.Request,p=c.Response,l=c.fetch,h="prototype",m="nr@context";d&&p&&l&&(a(u,function(t,e){r(d[h],e,f),r(p[h],e,f)}),r(c,"fetch",s),o.on(s+"end",function(t,e){var n=this;if(e){var r=e.headers.get("content-length");null!==r&&(n.rxSize=r),o.emit(s+"done",[null,e],n)}else o.emit(s+"done",[t],n)}))},{}],8:[function(t,e,n){var r=t("ee").get("history"),o=t("wrap-function")(r);e.exports=r;var i=window.history&&window.history.constructor&&window.history.constructor.prototype,a=window.history;i&&i.pushState&&i.replaceState&&(a=i),o.inPlace(a,["pushState","replaceState"],"-")},{}],9:[function(t,e,n){function r(t){function e(){s.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}function n(){s.emit("jsonp-error",[],p),s.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}var r=t&&"string"==typeof t.nodeName&&"script"===t.nodeName.toLowerCase();if(r){var o="function"==typeof t.addEventListener;if(o){var a=i(t.src);if(a){var u=c(a),d="function"==typeof u.parent[u.key];if(d){var p={};f.inPlace(u.parent,[u.key],"cb-",p),t.addEventListener("load",e,!1),t.addEventListener("error",n,!1),s.emit("new-jsonp",[t.src],p)}}}}}function o(){return"addEventListener"in window}function i(t){var e=t.match(u);return e?e[1]:null}function a(t,e){var n=t.match(p),r=n[1],o=n[3];return o?a(o,e[r]):e[r]}function c(t){var e=t.match(d);return e&&e.length>=3?{key:e[2],parent:a(e[1],window)}:{key:t,parent:window}}var s=t("ee").get("jsonp"),f=t("wrap-function")(s);if(e.exports=s,o()){var u=/[?&](?:callback|cb)=([^&#]+)/,d=/(.*)\.([^.]+)/,p=/^(\w+)(\.|$)(.*)$/,l=["appendChild","insertBefore","replaceChild"];Node&&Node.prototype&&Node.prototype.appendChild?f.inPlace(Node.prototype,l,"dom-"):(f.inPlace(HTMLElement.prototype,l,"dom-"),f.inPlace(HTMLHeadElement.prototype,l,"dom-"),f.inPlace(HTMLBodyElement.prototype,l,"dom-")),s.on("dom-start",function(t){r(t[0])})}},{}],10:[function(t,e,n){var r=t("ee").get("mutation"),o=t("wrap-function")(r),i=NREUM.o.MO;e.exports=r,i&&(window.MutationObserver=function(t){return this instanceof i?new i(o(t,"fn-")):i.apply(this,arguments)},MutationObserver.prototype=i.prototype)},{}],11:[function(t,e,n){function r(t){var e=a.context(),n=c(t,"executor-",e),r=new f(n);return a.context(r).getCtx=function(){return e},a.emit("new-promise",[r,e],e),r}function o(t,e){return e}var i=t("wrap-function"),a=t("ee").get("promise"),c=i(a),s=t(26),f=NREUM.o.PR;e.exports=a,f&&(window.Promise=r,["all","race"].forEach(function(t){var e=f[t];f[t]=function(n){function r(t){return function(){a.emit("propagate",[null,!o],i),o=o||!t}}var o=!1;s(n,function(e,n){Promise.resolve(n).then(r("all"===t),r(!1))});var i=e.apply(f,arguments),c=f.resolve(i);return c}}),["resolve","reject"].forEach(function(t){var e=f[t];f[t]=function(t){var n=e.apply(f,arguments);return t!==n&&a.emit("propagate",[t,!0],n),n}}),f.prototype["catch"]=function(t){return this.then(null,t)},f.prototype=Object.create(f.prototype,{constructor:{value:r}}),s(Object.getOwnPropertyNames(f),function(t,e){try{r[e]=f[e]}catch(n){}}),a.on("executor-start",function(t){t[0]=c(t[0],"resolve-",this),t[1]=c(t[1],"resolve-",this)}),a.on("executor-err",function(t,e,n){t[1](n)}),c.inPlace(f.prototype,["then"],"then-",o),a.on("then-start",function(t,e){this.promise=e,t[0]=c(t[0],"cb-",this),t[1]=c(t[1],"cb-",this)}),a.on("then-end",function(t,e,n){this.nextPromise=n;var r=this.promise;a.emit("propagate",[r,!0],n)}),a.on("cb-end",function(t,e,n){a.emit("propagate",[n,!0],this.nextPromise)}),a.on("propagate",function(t,e,n){this.getCtx&&!e||(this.getCtx=function(){if(t instanceof Promise)var e=a.context(t);return e&&e.getCtx?e.getCtx():this})}),r.toString=function(){return""+f})},{}],12:[function(t,e,n){var r=t("ee").get("raf"),o=t("wrap-function")(r),i="equestAnimationFrame";e.exports=r,o.inPlace(window,["r"+i,"mozR"+i,"webkitR"+i,"msR"+i],"raf-"),r.on("raf-start",function(t){t[0]=o(t[0],"fn-")})},{}],13:[function(t,e,n){function r(t,e,n){t[0]=a(t[0],"fn-",null,n)}function o(t,e,n){this.method=n,this.timerDuration=isNaN(t[1])?0:+t[1],t[0]=a(t[0],"fn-",this,n)}var i=t("ee").get("timer"),a=t("wrap-function")(i),c="setTimeout",s="setInterval",f="clearTimeout",u="-start",d="-";e.exports=i,a.inPlace(window,[c,"setImmediate"],c+d),a.inPlace(window,[s],s+d),a.inPlace(window,[f,"clearImmediate"],f+d),i.on(s+u,r),i.on(c+u,o)},{}],14:[function(t,e,n){function r(t,e){d.inPlace(e,["onreadystatechange"],"fn-",c)}function o(){var t=this,e=u.context(t);t.readyState>3&&!e.resolved&&(e.resolved=!0,u.emit("xhr-resolved",[],t)),d.inPlace(t,g,"fn-",c)}function i(t){y.push(t),h&&(b?b.then(a):v?v(a):(E=-E,R.data=E))}function a(){for(var t=0;t<y.length;t++)r([],y[t]);y.length&&(y=[])}function c(t,e){return e}function s(t,e){for(var n in t)e[n]=t[n];return e}t(6);var f=t("ee"),u=f.get("xhr"),d=t("wrap-function")(u),p=NREUM.o,l=p.XHR,h=p.MO,m=p.PR,v=p.SI,w="readystatechange",g=["onload","onerror","onabort","onloadstart","onloadend","onprogress","ontimeout"],y=[];e.exports=u;var x=window.XMLHttpRequest=function(t){var e=new l(t);try{u.emit("new-xhr",[e],e),e.addEventListener(w,o,!1)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}return e};if(s(l,x),x.prototype=l.prototype,d.inPlace(x.prototype,["open","send"],"-xhr-",c),u.on("send-xhr-start",function(t,e){r(t,e),i(e)}),u.on("open-xhr-start",r),h){var b=m&&m.resolve();if(!v&&!m){var E=1,R=document.createTextNode(E);new h(a).observe(R,{characterData:!0})}}else f.on("fn-end",function(t){t[0]&&t[0].type===w||a()})},{}],15:[function(t,e,n){function r(t){if(!c(t))return null;var e=window.NREUM;if(!e.loader_config)return null;var n=(e.loader_config.accountID||"").toString()||null,r=(e.loader_config.agentID||"").toString()||null,f=(e.loader_config.trustKey||"").toString()||null;if(!n||!r)return null;var h=l.generateSpanId(),m=l.generateTraceId(),v=Date.now(),w={spanId:h,traceId:m,timestamp:v};return(t.sameOrigin||s(t)&&p())&&(w.traceContextParentHeader=o(h,m),w.traceContextStateHeader=i(h,v,n,r,f)),(t.sameOrigin&&!u()||!t.sameOrigin&&s(t)&&d())&&(w.newrelicHeader=a(h,m,v,n,r,f)),w}function o(t,e){return"00-"+e+"-"+t+"-01"}function i(t,e,n,r,o){var i=0,a="",c=1,s="",f="";return o+"@nr="+i+"-"+c+"-"+n+"-"+r+"-"+t+"-"+a+"-"+s+"-"+f+"-"+e}function a(t,e,n,r,o,i){var a="btoa"in window&&"function"==typeof window.btoa;if(!a)return null;var c={v:[0,1],d:{ty:"Browser",ac:r,ap:o,id:t,tr:e,ti:n}};return i&&r!==i&&(c.d.tk=i),btoa(JSON.stringify(c))}function c(t){return f()&&s(t)}function s(t){var e=!1,n={};if("init"in NREUM&&"distributed_tracing"in NREUM.init&&(n=NREUM.init.distributed_tracing),t.sameOrigin)e=!0;else if(n.allowed_origins instanceof Array)for(var r=0;r<n.allowed_origins.length;r++){var o=h(n.allowed_origins[r]);if(t.hostname===o.hostname&&t.protocol===o.protocol&&t.port===o.port){e=!0;break}}return e}function f(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.enabled}function u(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.exclude_newrelic_header}function d(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&NREUM.init.distributed_tracing.cors_use_newrelic_header!==!1}function p(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.cors_use_tracecontext_headers}var l=t(23),h=t(17);e.exports={generateTracePayload:r,shouldGenerateTrace:c}},{}],16:[function(t,e,n){function r(t){var e=this.params,n=this.metrics;if(!this.ended){this.ended=!0;for(var r=0;r<p;r++)t.removeEventListener(d[r],this.listener,!1);e.aborted||(n.duration=a.now()-this.startTime,this.loadCaptureCalled||4!==t.readyState?null==e.status&&(e.status=0):i(this,t),n.cbTime=this.cbTime,u.emit("xhr-done",[t],t),c("xhr",[e,n,this.startTime]))}}function o(t,e){var n=s(e),r=t.params;r.host=n.hostname+":"+n.port,r.pathname=n.pathname,t.parsedOrigin=s(e),t.sameOrigin=t.parsedOrigin.sameOrigin}function i(t,e){t.params.status=e.status;var n=v(e,t.lastSize);if(n&&(t.metrics.rxSize=n),t.sameOrigin){var r=e.getResponseHeader("X-NewRelic-App-Data");r&&(t.params.cat=r.split(", ").pop())}t.loadCaptureCalled=!0}var a=t("loader");if(a.xhrWrappable){var c=t("handle"),s=t(17),f=t(15).generateTracePayload,u=t("ee"),d=["load","error","abort","timeout"],p=d.length,l=t("id"),h=t(21),m=t(20),v=t(18),w=window.XMLHttpRequest;a.features.xhr=!0,t(14),t(7),u.on("new-xhr",function(t){var e=this;e.totalCbs=0,e.called=0,e.cbTime=0,e.end=r,e.ended=!1,e.xhrGuids={},e.lastSize=null,e.loadCaptureCalled=!1,t.addEventListener("load",function(n){i(e,t)},!1),h&&(h>34||h<10)||window.opera||t.addEventListener("progress",function(t){e.lastSize=t.loaded},!1)}),u.on("open-xhr-start",function(t){this.params={method:t[0]},o(this,t[1]),this.metrics={}}),u.on("open-xhr-end",function(t,e){"loader_config"in NREUM&&"xpid"in NREUM.loader_config&&this.sameOrigin&&e.setRequestHeader("X-NewRelic-ID",NREUM.loader_config.xpid);var n=f(this.parsedOrigin);if(n){var r=!1;n.newrelicHeader&&(e.setRequestHeader("newrelic",n.newrelicHeader),r=!0),n.traceContextParentHeader&&(e.setRequestHeader("traceparent",n.traceContextParentHeader),n.traceContextStateHeader&&e.setRequestHeader("tracestate",n.traceContextStateHeader),r=!0),r&&(this.dt=n)}}),u.on("send-xhr-start",function(t,e){var n=this.metrics,r=t[0],o=this;if(n&&r){var i=m(r);i&&(n.txSize=i)}this.startTime=a.now(),this.listener=function(t){try{"abort"!==t.type||o.loadCaptureCalled||(o.params.aborted=!0),("load"!==t.type||o.called===o.totalCbs&&(o.onloadCalled||"function"!=typeof e.onload))&&o.end(e)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}};for(var c=0;c<p;c++)e.addEventListener(d[c],this.listener,!1)}),u.on("xhr-cb-time",function(t,e,n){this.cbTime+=t,e?this.onloadCalled=!0:this.called+=1,this.called!==this.totalCbs||!this.onloadCalled&&"function"==typeof n.onload||this.end(n)}),u.on("xhr-load-added",function(t,e){var n=""+l(t)+!!e;this.xhrGuids&&!this.xhrGuids[n]&&(this.xhrGuids[n]=!0,this.totalCbs+=1)}),u.on("xhr-load-removed",function(t,e){var n=""+l(t)+!!e;this.xhrGuids&&this.xhrGuids[n]&&(delete this.xhrGuids[n],this.totalCbs-=1)}),u.on("addEventListener-end",function(t,e){e instanceof w&&"load"===t[0]&&u.emit("xhr-load-added",[t[1],t[2]],e)}),u.on("removeEventListener-end",function(t,e){e instanceof w&&"load"===t[0]&&u.emit("xhr-load-removed",[t[1],t[2]],e)}),u.on("fn-start",function(t,e,n){e instanceof w&&("onload"===n&&(this.onload=!0),("load"===(t[0]&&t[0].type)||this.onload)&&(this.xhrCbStart=a.now()))}),u.on("fn-end",function(t,e){this.xhrCbStart&&u.emit("xhr-cb-time",[a.now()-this.xhrCbStart,this.onload,e],e)}),u.on("fetch-before-start",function(t){function e(t,e){var n=!1;return e.newrelicHeader&&(t.set("newrelic",e.newrelicHeader),n=!0),e.traceContextParentHeader&&(t.set("traceparent",e.traceContextParentHeader),e.traceContextStateHeader&&t.set("tracestate",e.traceContextStateHeader),n=!0),n}var n,r=t[1]||{};"string"==typeof t[0]?n=t[0]:t[0]&&t[0].url&&(n=t[0].url),n&&(this.parsedOrigin=s(n),this.sameOrigin=this.parsedOrigin.sameOrigin);var o=f(this.parsedOrigin);if(o&&(o.newrelicHeader||o.traceContextParentHeader))if("string"==typeof t[0]){var i={};for(var a in r)i[a]=r[a];i.headers=new Headers(r.headers||{}),e(i.headers,o)&&(this.dt=o),t.length>1?t[1]=i:t.push(i)}else t[0]&&t[0].headers&&e(t[0].headers,o)&&(this.dt=o)})}},{}],17:[function(t,e,n){var r={};e.exports=function(t){if(t in r)return r[t];var e=document.createElement("a"),n=window.location,o={};e.href=t,o.port=e.port;var i=e.href.split("://");!o.port&&i[1]&&(o.port=i[1].split("/")[0].split("@").pop().split(":")[1]),o.port&&"0"!==o.port||(o.port="https"===i[0]?"443":"80"),o.hostname=e.hostname||n.hostname,o.pathname=e.pathname,o.protocol=i[0],"/"!==o.pathname.charAt(0)&&(o.pathname="/"+o.pathname);var a=!e.protocol||":"===e.protocol||e.protocol===n.protocol,c=e.hostname===document.domain&&e.port===n.port;return o.sameOrigin=a&&(!e.hostname||c),"/"===o.pathname&&(r[t]=o),o}},{}],18:[function(t,e,n){function r(t,e){var n=t.responseType;return"json"===n&&null!==e?e:"arraybuffer"===n||"blob"===n||"json"===n?o(t.response):"text"===n||""===n||void 0===n?o(t.responseText):void 0}var o=t(20);e.exports=r},{}],19:[function(t,e,n){function r(){}function o(t,e,n){return function(){return i(t,[f.now()].concat(c(arguments)),e?null:this,n),e?void 0:this}}var i=t("handle"),a=t(26),c=t(27),s=t("ee").get("tracer"),f=t("loader"),u=NREUM;"undefined"==typeof window.newrelic&&(newrelic=u);var d=["setPageViewName","setCustomAttribute","setErrorHandler","finished","addToTrace","inlineHit","addRelease"],p="api-",l=p+"ixn-";a(d,function(t,e){u[e]=o(p+e,!0,"api")}),u.addPageAction=o(p+"addPageAction",!0),u.setCurrentRouteName=o(p+"routeName",!0),e.exports=newrelic,u.interaction=function(){return(new r).get()};var h=r.prototype={createTracer:function(t,e){var n={},r=this,o="function"==typeof e;return i(l+"tracer",[f.now(),t,n],r),function(){if(s.emit((o?"":"no-")+"fn-start",[f.now(),r,o],n),o)try{return e.apply(this,arguments)}catch(t){throw s.emit("fn-err",[arguments,this,t],n),t}finally{s.emit("fn-end",[f.now()],n)}}}};a("actionText,setName,setAttribute,save,ignore,onEnd,getContext,end,get".split(","),function(t,e){h[e]=o(l+e)}),newrelic.noticeError=function(t,e){"string"==typeof t&&(t=new Error(t)),i("err",[t,f.now(),!1,e])}},{}],20:[function(t,e,n){e.exports=function(t){if("string"==typeof t&&t.length)return t.length;if("object"==typeof t){if("undefined"!=typeof ArrayBuffer&&t instanceof ArrayBuffer&&t.byteLength)return t.byteLength;if("undefined"!=typeof Blob&&t instanceof Blob&&t.size)return t.size;if(!("undefined"!=typeof FormData&&t instanceof FormData))try{return JSON.stringify(t).length}catch(e){return}}}},{}],21:[function(t,e,n){var r=0,o=navigator.userAgent.match(/Firefox[\/\s](\d+\.\d+)/);o&&(r=+o[1]),e.exports=r},{}],22:[function(t,e,n){function r(t,e){var n=t.getEntries();n.forEach(function(t){"first-paint"===t.name?d("timing",["fp",Math.floor(t.startTime)]):"first-contentful-paint"===t.name&&d("timing",["fcp",Math.floor(t.startTime)])})}function o(t,e){var n=t.getEntries();n.length>0&&d("lcp",[n[n.length-1]])}function i(t){t.getEntries().forEach(function(t){t.hadRecentInput||d("cls",[t])})}function a(t){if(t instanceof h&&!v){var e=Math.round(t.timeStamp),n={type:t.type};e<=p.now()?n.fid=p.now()-e:e>p.offset&&e<=Date.now()?(e-=p.offset,n.fid=p.now()-e):e=p.now(),v=!0,d("timing",["fi",e,n])}}function c(t){d("pageHide",[p.now(),t])}if(!("init"in NREUM&&"page_view_timing"in NREUM.init&&"enabled"in NREUM.init.page_view_timing&&NREUM.init.page_view_timing.enabled===!1)){var s,f,u,d=t("handle"),p=t("loader"),l=t(25),h=NREUM.o.EV;if("PerformanceObserver"in window&&"function"==typeof window.PerformanceObserver){s=new PerformanceObserver(r);try{s.observe({entryTypes:["paint"]})}catch(m){}f=new PerformanceObserver(o);try{f.observe({entryTypes:["largest-contentful-paint"]})}catch(m){}u=new PerformanceObserver(i);try{u.observe({type:"layout-shift",buffered:!0})}catch(m){}}if("addEventListener"in document){var v=!1,w=["click","keydown","mousedown","pointerdown","touchstart"];w.forEach(function(t){document.addEventListener(t,a,!1)})}l(c)}},{}],23:[function(t,e,n){function r(){function t(){return e?15&e[n++]:16*Math.random()|0}var e=null,n=0,r=window.crypto||window.msCrypto;r&&r.getRandomValues&&(e=r.getRandomValues(new Uint8Array(31)));for(var o,i="xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx",a="",c=0;c<i.length;c++)o=i[c],"x"===o?a+=t().toString(16):"y"===o?(o=3&t()|8,a+=o.toString(16)):a+=o;return a}function o(){return a(16)}function i(){return a(32)}function a(t){function e(){return n?15&n[r++]:16*Math.random()|0}var n=null,r=0,o=window.crypto||window.msCrypto;o&&o.getRandomValues&&Uint8Array&&(n=o.getRandomValues(new Uint8Array(31)));for(var i=[],a=0;a<t;a++)i.push(e().toString(16));return i.join("")}e.exports={generateUuid:r,generateSpanId:o,generateTraceId:i}},{}],24:[function(t,e,n){function r(t,e){if(!o)return!1;if(t!==o)return!1;if(!e)return!0;if(!i)return!1;for(var n=i.split("."),r=e.split("."),a=0;a<r.length;a++)if(r[a]!==n[a])return!1;return!0}var o=null,i=null,a=/Version\/(\S+)\s+Safari/;if(navigator.userAgent){var c=navigator.userAgent,s=c.match(a);s&&c.indexOf("Chrome")===-1&&c.indexOf("Chromium")===-1&&(o="Safari",i=s[1])}e.exports={agent:o,version:i,match:r}},{}],25:[function(t,e,n){function r(t){function e(){t(a&&document[a]?document[a]:document[o]?"hidden":"visible")}"addEventListener"in document&&i&&document.addEventListener(i,e,!1)}e.exports=r;var o,i,a;"undefined"!=typeof document.hidden?(o="hidden",i="visibilitychange",a="visibilityState"):"undefined"!=typeof document.msHidden?(o="msHidden",i="msvisibilitychange"):"undefined"!=typeof document.webkitHidden&&(o="webkitHidden",i="webkitvisibilitychange",a="webkitVisibilityState")},{}],26:[function(t,e,n){function r(t,e){var n=[],r="",i=0;for(r in t)o.call(t,r)&&(n[i]=e(r,t[r]),i+=1);return n}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],27:[function(t,e,n){function r(t,e,n){e||(e=0),"undefined"==typeof n&&(n=t?t.length:0);for(var r=-1,o=n-e||0,i=Array(o<0?0:o);++r<o;)i[r]=t[e+r];return i}e.exports=r},{}],28:[function(t,e,n){e.exports={exists:"undefined"!=typeof window.performance&&window.performance.timing&&"undefined"!=typeof window.performance.timing.navigationStart}},{}],ee:[function(t,e,n){function r(){}function o(t){function e(t){return t&&t instanceof r?t:t?s(t,c,i):i()}function n(n,r,o,i){if(!p.aborted||i){t&&t(n,r,o);for(var a=e(o),c=m(n),s=c.length,f=0;f<s;f++)c[f].apply(a,r);var d=u[y[n]];return d&&d.push([x,n,r,a]),a}}function l(t,e){g[t]=m(t).concat(e)}function h(t,e){var n=g[t];if(n)for(var r=0;r<n.length;r++)n[r]===e&&n.splice(r,1)}function m(t){return g[t]||[]}function v(t){return d[t]=d[t]||o(n)}function w(t,e){f(t,function(t,n){e=e||"feature",y[n]=e,e in u||(u[e]=[])})}var g={},y={},x={on:l,addEventListener:l,removeEventListener:h,emit:n,get:v,listeners:m,context:e,buffer:w,abort:a,aborted:!1};return x}function i(){return new r}function a(){(u.api||u.feature)&&(p.aborted=!0,u=p.backlog={})}var c="nr@context",s=t("gos"),f=t(26),u={},d={},p=e.exports=o();p.backlog=u},{}],gos:[function(t,e,n){function r(t,e,n){if(o.call(t,e))return t[e];var r=n();if(Object.defineProperty&&Object.keys)try{return Object.defineProperty(t,e,{value:r,writable:!0,enumerable:!1}),r}catch(i){}return t[e]=r,r}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],handle:[function(t,e,n){function r(t,e,n,r){o.buffer([t],r),o.emit(t,e,n)}var o=t("ee").get("handle");e.exports=r,r.ee=o},{}],id:[function(t,e,n){function r(t){var e=typeof t;return!t||"object"!==e&&"function"!==e?-1:t===window?0:a(t,i,function(){return o++})}var o=1,i="nr@id",a=t("gos");e.exports=r},{}],loader:[function(t,e,n){function r(){if(!E++){var t=b.info=NREUM.info,e=l.getElementsByTagName("script")[0];if(setTimeout(u.abort,3e4),!(t&&t.licenseKey&&t.applicationID&&e))return u.abort();f(y,function(e,n){t[e]||(t[e]=n)});var n=a();s("mark",["onload",n+b.offset],null,"api"),s("timing",["load",n]);var r=l.createElement("script");r.src="https://"+t.agent,e.parentNode.insertBefore(r,e)}}function o(){"complete"===l.readyState&&i()}function i(){s("mark",["domContent",a()+b.offset],null,"api")}function a(){return R.exists&&performance.now?Math.round(performance.now()):(c=Math.max((new Date).getTime(),c))-b.offset}var c=(new Date).getTime(),s=t("handle"),f=t(26),u=t("ee"),d=t(24),p=window,l=p.document,h="addEventListener",m="attachEvent",v=p.XMLHttpRequest,w=v&&v.prototype;NREUM.o={ST:setTimeout,SI:p.setImmediate,CT:clearTimeout,XHR:v,REQ:p.Request,EV:p.Event,PR:p.Promise,MO:p.MutationObserver};var g=""+location,y={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",agent:"js-agent.newrelic.com/nr-spa-1177.min.js"},x=v&&w&&w[h]&&!/CriOS/.test(navigator.userAgent),b=e.exports={offset:c,now:a,origin:g,features:{},xhrWrappable:x,userAgent:d};t(19),t(22),l[h]?(l[h]("DOMContentLoaded",i,!1),p[h]("load",r,!1)):(l[m]("onreadystatechange",o),p[m]("onload",r)),s("mark",["firstbyte",c],null,"api");var E=0,R=t(28)},{}],"wrap-function":[function(t,e,n){function r(t){return!(t&&t instanceof Function&&t.apply&&!t[a])}var o=t("ee"),i=t(27),a="nr@original",c=Object.prototype.hasOwnProperty,s=!1;e.exports=function(t,e){function n(t,e,n,o){function nrWrapper(){var r,a,c,s;try{a=this,r=i(arguments),c="function"==typeof n?n(r,a):n||{}}catch(f){p([f,"",[r,a,o],c])}u(e+"start",[r,a,o],c);try{return s=t.apply(a,r)}catch(d){throw u(e+"err",[r,a,d],c),d}finally{u(e+"end",[r,a,s],c)}}return r(t)?t:(e||(e=""),nrWrapper[a]=t,d(t,nrWrapper),nrWrapper)}function f(t,e,o,i){o||(o="");var a,c,s,f="-"===o.charAt(0);for(s=0;s<e.length;s++)c=e[s],a=t[c],r(a)||(t[c]=n(a,f?c+o:o,i,c))}function u(n,r,o){if(!s||e){var i=s;s=!0;try{t.emit(n,r,o,e)}catch(a){p([a,n,r,o])}s=i}}function d(t,e){if(Object.defineProperty&&Object.keys)try{var n=Object.keys(t);return n.forEach(function(n){Object.defineProperty(e,n,{get:function(){return t[n]},set:function(e){return t[n]=e,e}})}),e}catch(r){p([r])}for(var o in t)c.call(t,o)&&(e[o]=t[o]);return e}function p(e){try{t.emit("internal-error",e)}catch(n){}}return t||(t=o),n.inPlace=f,n.flag=a,n}},{}]},{},["loader",2,16,5,3,4]);
    ;NREUM.loader_config={accountID:"2095297",trustKey:"2038175",agentID:"916957862",licenseKey:"16e2192192",applicationID:"916957862"}
    ;NREUM.info={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",licenseKey:"16e2192192",applicationID:"916957862",sa:1}
</script>
    <script src="https://cdn.cookielaw.org/scripttemplates/otSDKStub.js" type="text/javascript" charset="UTF-8" data-domain-script="dada9834-04e2-4bcd-b5ba-cae44d498dfa"></script>
    <script type="text/javascript">
        function getCookie(name) {
            var value = "; " + document.cookie;
            var parts = value.split("; " + name + "=");
            if (parts.length == 2) {
                return true;
            }
        }
            function OptanonWrapper() {
                var optCookie = getCookie("OptanonAlertBoxClosed");
                var isActiveCategory = "false";
                var isFoundCookieBox = "false";
                if(isActiveCategory == 'false' && isFoundCookieBox == 'false' && optCookie) {
                    OneTrust.InsertScript('/pb-assets/adobe/prod/adobedtm-1616606406233.js', 'head', null, null, '2');
                }
            }
    </script>


<noscript>
    <style>.requiresJS {
        display: none
    }

    .dartAd {
        display: block !important
    }</style>
</noscript>
        <script type="text/javascript" src="/wro/o4ny~selected-legacy.js"></script>
        <link rel="stylesheet" type="text/css" href="/wro/o4ny~selected-legacy.css">
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Open+Sans">
        <script>window.MSInputMethodContext && document.documentMode && document.write('<script src="https://cdn.jsdelivr.net/gh/nuxodin/ie11CustomProperties@4.1.0/ie11CustomProperties.min.js"><\x2fscript>');</script><link rel="stylesheet" href="/products/marlin/cell/releasedAssets/css/build-b26f3342b722959c7931.css"><link rel="stylesheet" href="/products/marlin/cell/releasedAssets/css/print-b26f3342b722959c7931.css" media="print"><link rel="stylesheet" href="/products/marlin/cell/releasedAssets/css/themes/theme-cell.css">







        
        
        <script class="optanon-category-4" type="text/plain" async="" src="https://www.youtube.com/iframe_api"></script>
        
    





<!---->
<meta http-equiv="X-UA-Compatible" content="IE=edge">












<script src="https://cdn.cookielaw.org/scripttemplates/202411.2.0/otBannerSdk.js" async="" type="text/javascript"></script><style id="onetrust-style">#onetrust-banner-sdk .onetrust-vendors-list-handler{cursor:pointer;color:#1f96db;font-size:inherit;font-weight:bold;text-decoration:none;margin-left:5px}#onetrust-banner-sdk .onetrust-vendors-list-handler:hover{color:#1f96db}#onetrust-banner-sdk:focus{outline:2px solid #000;outline-offset:-2px}#onetrust-banner-sdk a:focus{outline:2px solid #000}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{outline-offset:1px}#onetrust-banner-sdk.ot-bnr-w-logo .ot-bnr-logo{height:64px;width:64px}#onetrust-banner-sdk .ot-tcf2-vendor-count.ot-text-bold{font-weight:bold}#onetrust-banner-sdk .ot-close-icon,#onetrust-pc-sdk .ot-close-icon,#ot-sync-ntfy .ot-close-icon{background-size:contain;background-repeat:no-repeat;background-position:center;height:12px;width:12px}#onetrust-banner-sdk .powered-by-logo,#onetrust-banner-sdk .ot-pc-footer-logo a,#onetrust-pc-sdk .powered-by-logo,#onetrust-pc-sdk .ot-pc-footer-logo a,#ot-sync-ntfy .powered-by-logo,#ot-sync-ntfy .ot-pc-footer-logo a{background-size:contain;background-repeat:no-repeat;background-position:center;height:25px;width:152px;display:block;text-decoration:none;font-size:.75em}#onetrust-banner-sdk .powered-by-logo:hover,#onetrust-banner-sdk .ot-pc-footer-logo a:hover,#onetrust-pc-sdk .powered-by-logo:hover,#onetrust-pc-sdk .ot-pc-footer-logo a:hover,#ot-sync-ntfy .powered-by-logo:hover,#ot-sync-ntfy .ot-pc-footer-logo a:hover{color:#565656}#onetrust-banner-sdk h3 *,#onetrust-banner-sdk h4 *,#onetrust-banner-sdk h6 *,#onetrust-banner-sdk button *,#onetrust-banner-sdk a[data-parent-id] *,#onetrust-pc-sdk h3 *,#onetrust-pc-sdk h4 *,#onetrust-pc-sdk h6 *,#onetrust-pc-sdk button *,#onetrust-pc-sdk a[data-parent-id] *,#ot-sync-ntfy h3 *,#ot-sync-ntfy h4 *,#ot-sync-ntfy h6 *,#ot-sync-ntfy button *,#ot-sync-ntfy a[data-parent-id] *{font-size:inherit;font-weight:inherit;color:inherit}#onetrust-banner-sdk .ot-hide,#onetrust-pc-sdk .ot-hide,#ot-sync-ntfy .ot-hide{display:none !important}#onetrust-banner-sdk button.ot-link-btn:hover,#onetrust-pc-sdk button.ot-link-btn:hover,#ot-sync-ntfy button.ot-link-btn:hover{text-decoration:underline;opacity:1}#onetrust-pc-sdk .ot-sdk-row .ot-sdk-column{padding:0}#onetrust-pc-sdk .ot-sdk-container{padding-right:0}#onetrust-pc-sdk .ot-sdk-row{flex-direction:initial;width:100%}#onetrust-pc-sdk [type=checkbox]:checked,#onetrust-pc-sdk [type=checkbox]:not(:checked){pointer-events:initial}#onetrust-pc-sdk [type=checkbox]:disabled+label::before,#onetrust-pc-sdk [type=checkbox]:disabled+label:after,#onetrust-pc-sdk [type=checkbox]:disabled+label{pointer-events:none;opacity:.8}#onetrust-pc-sdk #vendor-list-content{transform:translate3d(0, 0, 0)}#onetrust-pc-sdk li input[type=checkbox]{z-index:1}#onetrust-pc-sdk li .ot-checkbox label{z-index:2}#onetrust-pc-sdk li .ot-checkbox input[type=checkbox]{height:auto;width:auto}#onetrust-pc-sdk li .host-title a,#onetrust-pc-sdk li .ot-host-name a,#onetrust-pc-sdk li .accordion-text,#onetrust-pc-sdk li .ot-acc-txt{z-index:2;position:relative}#onetrust-pc-sdk input{margin:3px .1ex}#onetrust-pc-sdk .pc-logo,#onetrust-pc-sdk .ot-pc-logo{height:60px;width:180px;background-position:center;background-size:contain;background-repeat:no-repeat;display:inline-flex;justify-content:center;align-items:center}#onetrust-pc-sdk .pc-logo img,#onetrust-pc-sdk .ot-pc-logo img{max-height:100%;max-width:100%}#onetrust-pc-sdk .screen-reader-only,#onetrust-pc-sdk .ot-scrn-rdr,.ot-sdk-cookie-policy .screen-reader-only,.ot-sdk-cookie-policy .ot-scrn-rdr{border:0;clip:rect(0 0 0 0);height:1px;margin:-1px;overflow:hidden;padding:0;position:absolute;width:1px}#onetrust-pc-sdk.ot-fade-in,.onetrust-pc-dark-filter.ot-fade-in,#onetrust-banner-sdk.ot-fade-in{animation-name:onetrust-fade-in;animation-duration:400ms;animation-timing-function:ease-in-out}#onetrust-pc-sdk.ot-hide{display:none !important}.onetrust-pc-dark-filter.ot-hide{display:none !important}#ot-sdk-btn.ot-sdk-show-settings,#ot-sdk-btn.optanon-show-settings{color:#fff;background-color:#468254;height:auto;white-space:normal;word-wrap:break-word;padding:.8em 2em;font-size:.8em;line-height:1.2;cursor:pointer;-moz-transition:.1s ease;-o-transition:.1s ease;-webkit-transition:1s ease;transition:.1s ease}#ot-sdk-btn.ot-sdk-show-settings:hover,#ot-sdk-btn.optanon-show-settings:hover{color:#fff;background-color:#2c6415}#ot-sdk-btn.ot-sdk-show-settings:active,#ot-sdk-btn.optanon-show-settings:active{color:#fff;background-color:#2c6415;border:1px solid rgba(162,192,169,.5)}.onetrust-pc-dark-filter{background:rgba(0,0,0,.5);z-index:2147483646;width:100%;height:100%;overflow:hidden;position:fixed;top:0;bottom:0;left:0}@keyframes onetrust-fade-in{0%{opacity:0}100%{opacity:1}}.ot-cookie-label{text-decoration:underline}@media only screen and (min-width: 426px)and (max-width: 896px)and (orientation: landscape){#onetrust-pc-sdk p{font-size:.75em}}#onetrust-banner-sdk .banner-option-input:focus+label{outline:1px solid #000;outline-style:auto}.category-vendors-list-handler+a:focus,.category-vendors-list-handler+a:focus-visible{outline:2px solid #000}#onetrust-pc-sdk .ot-userid-title{margin-top:10px}#onetrust-pc-sdk .ot-userid-title>span,#onetrust-pc-sdk .ot-userid-timestamp>span{font-weight:700}#onetrust-pc-sdk .ot-userid-desc{font-style:italic}#onetrust-pc-sdk .ot-host-desc a{pointer-events:initial}#onetrust-pc-sdk .ot-ven-hdr>p a{position:relative;z-index:2;pointer-events:initial}#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info a,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info a{margin-right:auto}#onetrust-pc-sdk .ot-pc-footer-logo img{width:136px;height:16px}#onetrust-pc-sdk .ot-pur-vdr-count{font-weight:400;font-size:.7rem;padding-top:3px;display:block}#onetrust-banner-sdk .ot-optout-signal,#onetrust-pc-sdk .ot-optout-signal{border:1px solid #32ae88;border-radius:3px;padding:5px;margin-bottom:10px;background-color:#f9fffa;font-size:.85rem;line-height:2}#onetrust-banner-sdk .ot-optout-signal .ot-optout-icon,#onetrust-pc-sdk .ot-optout-signal .ot-optout-icon{display:inline;margin-right:5px}#onetrust-banner-sdk .ot-optout-signal svg,#onetrust-pc-sdk .ot-optout-signal svg{height:20px;width:30px;transform:scale(0.5)}#onetrust-banner-sdk .ot-optout-signal svg path,#onetrust-pc-sdk .ot-optout-signal svg path{fill:#32ae88}#onetrust-consent-sdk .ot-general-modal{overflow:hidden;position:fixed;margin:0 auto;top:50%;left:50%;width:40%;padding:1.5rem;max-width:575px;min-width:575px;z-index:2147483647;border-radius:2.5px;transform:translate(-50%, -50%)}#onetrust-consent-sdk .ot-signature-health-group{margin-top:1rem;padding-left:1.25rem;padding-right:1.25rem;margin-bottom:.625rem;width:calc(100% - 2.5rem)}#onetrust-consent-sdk .ot-signature-health-group .ot-signature-health-form{gap:.5rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-health-form{width:70%;gap:.35rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-input{height:38px;padding:6px 10px;background-color:#fff;border:1px solid #d1d1d1;border-radius:4px;box-shadow:none;box-sizing:border-box}#onetrust-consent-sdk .ot-signature-health .ot-signature-subtitle{font-size:1.125rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-group-title{font-size:1.25rem;font-weight:bold}#onetrust-consent-sdk .ot-signature-health,#onetrust-consent-sdk .ot-signature-health-group{display:flex;flex-direction:column;gap:1rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-cont,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-cont{display:flex;flex-direction:column;gap:.25rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-paragraph,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-paragraph{margin:0;line-height:20px;font-size:max(14px,.875rem)}#onetrust-consent-sdk .ot-signature-health .ot-health-signature-error,#onetrust-consent-sdk .ot-signature-health-group .ot-health-signature-error{color:#4d4d4d;font-size:min(12px,.75rem)}#onetrust-consent-sdk .ot-signature-health .ot-signature-buttons-cont,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-buttons-cont{margin-top:max(.75rem,2%);gap:1rem;display:flex;justify-content:flex-end}#onetrust-consent-sdk .ot-signature-health .ot-signature-button,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-button{flex:1;height:auto;color:#fff;cursor:pointer;line-height:1.2;min-width:125px;font-weight:600;font-size:.813em;border-radius:2px;padding:12px 10px;white-space:normal;word-wrap:break-word;word-break:break-word;background-color:#68b631;border:2px solid #68b631}#onetrust-consent-sdk .ot-signature-health .ot-signature-button.reject,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-button.reject{background-color:#fff}#onetrust-consent-sdk .ot-input-field-cont{display:flex;flex-direction:column;gap:.5rem}#onetrust-consent-sdk .ot-input-field-cont .ot-signature-input{width:65%}#onetrust-consent-sdk .ot-signature-health-form{display:flex;flex-direction:column}#onetrust-consent-sdk .ot-signature-health-form .ot-signature-label{margin-bottom:0;line-height:20px;font-size:max(14px,.875rem)}@media only screen and (max-width: 600px){#onetrust-consent-sdk .ot-general-modal{min-width:100%}#onetrust-consent-sdk .ot-signature-health .ot-signature-health-form{width:100%}#onetrust-consent-sdk .ot-input-field-cont .ot-signature-input{width:100%}}#onetrust-banner-sdk,#onetrust-pc-sdk,#ot-sdk-cookie-policy,#ot-sync-ntfy{font-size:16px}#onetrust-banner-sdk *,#onetrust-banner-sdk ::after,#onetrust-banner-sdk ::before,#onetrust-pc-sdk *,#onetrust-pc-sdk ::after,#onetrust-pc-sdk ::before,#ot-sdk-cookie-policy *,#ot-sdk-cookie-policy ::after,#ot-sdk-cookie-policy ::before,#ot-sync-ntfy *,#ot-sync-ntfy ::after,#ot-sync-ntfy ::before{-webkit-box-sizing:content-box;-moz-box-sizing:content-box;box-sizing:content-box}#onetrust-banner-sdk div,#onetrust-banner-sdk span,#onetrust-banner-sdk h1,#onetrust-banner-sdk h2,#onetrust-banner-sdk h3,#onetrust-banner-sdk h4,#onetrust-banner-sdk h5,#onetrust-banner-sdk h6,#onetrust-banner-sdk p,#onetrust-banner-sdk img,#onetrust-banner-sdk svg,#onetrust-banner-sdk button,#onetrust-banner-sdk section,#onetrust-banner-sdk a,#onetrust-banner-sdk label,#onetrust-banner-sdk input,#onetrust-banner-sdk ul,#onetrust-banner-sdk li,#onetrust-banner-sdk nav,#onetrust-banner-sdk table,#onetrust-banner-sdk thead,#onetrust-banner-sdk tr,#onetrust-banner-sdk td,#onetrust-banner-sdk tbody,#onetrust-banner-sdk .ot-main-content,#onetrust-banner-sdk .ot-toggle,#onetrust-banner-sdk #ot-content,#onetrust-banner-sdk #ot-pc-content,#onetrust-banner-sdk .checkbox,#onetrust-pc-sdk div,#onetrust-pc-sdk span,#onetrust-pc-sdk h1,#onetrust-pc-sdk h2,#onetrust-pc-sdk h3,#onetrust-pc-sdk h4,#onetrust-pc-sdk h5,#onetrust-pc-sdk h6,#onetrust-pc-sdk p,#onetrust-pc-sdk img,#onetrust-pc-sdk svg,#onetrust-pc-sdk button,#onetrust-pc-sdk section,#onetrust-pc-sdk a,#onetrust-pc-sdk label,#onetrust-pc-sdk input,#onetrust-pc-sdk ul,#onetrust-pc-sdk li,#onetrust-pc-sdk nav,#onetrust-pc-sdk table,#onetrust-pc-sdk thead,#onetrust-pc-sdk tr,#onetrust-pc-sdk td,#onetrust-pc-sdk tbody,#onetrust-pc-sdk .ot-main-content,#onetrust-pc-sdk .ot-toggle,#onetrust-pc-sdk #ot-content,#onetrust-pc-sdk #ot-pc-content,#onetrust-pc-sdk .checkbox,#ot-sdk-cookie-policy div,#ot-sdk-cookie-policy span,#ot-sdk-cookie-policy h1,#ot-sdk-cookie-policy h2,#ot-sdk-cookie-policy h3,#ot-sdk-cookie-policy h4,#ot-sdk-cookie-policy h5,#ot-sdk-cookie-policy h6,#ot-sdk-cookie-policy p,#ot-sdk-cookie-policy img,#ot-sdk-cookie-policy svg,#ot-sdk-cookie-policy button,#ot-sdk-cookie-policy section,#ot-sdk-cookie-policy a,#ot-sdk-cookie-policy label,#ot-sdk-cookie-policy input,#ot-sdk-cookie-policy ul,#ot-sdk-cookie-policy li,#ot-sdk-cookie-policy nav,#ot-sdk-cookie-policy table,#ot-sdk-cookie-policy thead,#ot-sdk-cookie-policy tr,#ot-sdk-cookie-policy td,#ot-sdk-cookie-policy tbody,#ot-sdk-cookie-policy .ot-main-content,#ot-sdk-cookie-policy .ot-toggle,#ot-sdk-cookie-policy #ot-content,#ot-sdk-cookie-policy #ot-pc-content,#ot-sdk-cookie-policy .checkbox,#ot-sync-ntfy div,#ot-sync-ntfy span,#ot-sync-ntfy h1,#ot-sync-ntfy h2,#ot-sync-ntfy h3,#ot-sync-ntfy h4,#ot-sync-ntfy h5,#ot-sync-ntfy h6,#ot-sync-ntfy p,#ot-sync-ntfy img,#ot-sync-ntfy svg,#ot-sync-ntfy button,#ot-sync-ntfy section,#ot-sync-ntfy a,#ot-sync-ntfy label,#ot-sync-ntfy input,#ot-sync-ntfy ul,#ot-sync-ntfy li,#ot-sync-ntfy nav,#ot-sync-ntfy table,#ot-sync-ntfy thead,#ot-sync-ntfy tr,#ot-sync-ntfy td,#ot-sync-ntfy tbody,#ot-sync-ntfy .ot-main-content,#ot-sync-ntfy .ot-toggle,#ot-sync-ntfy #ot-content,#ot-sync-ntfy #ot-pc-content,#ot-sync-ntfy .checkbox{font-family:inherit;font-weight:normal;-webkit-font-smoothing:auto;letter-spacing:normal;line-height:normal;padding:0;margin:0;height:auto;min-height:0;max-height:none;width:auto;min-width:0;max-width:none;border-radius:0;border:none;clear:none;float:none;position:static;bottom:auto;left:auto;right:auto;top:auto;text-align:left;text-decoration:none;text-indent:0;text-shadow:none;text-transform:none;white-space:normal;background:none;overflow:visible;vertical-align:baseline;visibility:visible;z-index:auto;box-shadow:none}#onetrust-banner-sdk label:before,#onetrust-banner-sdk label:after,#onetrust-banner-sdk .checkbox:after,#onetrust-banner-sdk .checkbox:before,#onetrust-pc-sdk label:before,#onetrust-pc-sdk label:after,#onetrust-pc-sdk .checkbox:after,#onetrust-pc-sdk .checkbox:before,#ot-sdk-cookie-policy label:before,#ot-sdk-cookie-policy label:after,#ot-sdk-cookie-policy .checkbox:after,#ot-sdk-cookie-policy .checkbox:before,#ot-sync-ntfy label:before,#ot-sync-ntfy label:after,#ot-sync-ntfy .checkbox:after,#ot-sync-ntfy .checkbox:before{content:"";content:none}#onetrust-banner-sdk .ot-sdk-container,#onetrust-pc-sdk .ot-sdk-container,#ot-sdk-cookie-policy .ot-sdk-container{position:relative;width:100%;max-width:100%;margin:0 auto;padding:0 20px;box-sizing:border-box}#onetrust-banner-sdk .ot-sdk-column,#onetrust-banner-sdk .ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-column,#onetrust-pc-sdk .ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-column,#ot-sdk-cookie-policy .ot-sdk-columns{width:100%;float:left;box-sizing:border-box;padding:0;display:initial}@media(min-width: 400px){#onetrust-banner-sdk .ot-sdk-container,#onetrust-pc-sdk .ot-sdk-container,#ot-sdk-cookie-policy .ot-sdk-container{width:90%;padding:0}}@media(min-width: 550px){#onetrust-banner-sdk .ot-sdk-container,#onetrust-pc-sdk .ot-sdk-container,#ot-sdk-cookie-policy .ot-sdk-container{width:100%}#onetrust-banner-sdk .ot-sdk-column,#onetrust-banner-sdk .ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-column,#onetrust-pc-sdk .ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-column,#ot-sdk-cookie-policy .ot-sdk-columns{margin-left:4%}#onetrust-banner-sdk .ot-sdk-column:first-child,#onetrust-banner-sdk .ot-sdk-columns:first-child,#onetrust-pc-sdk .ot-sdk-column:first-child,#onetrust-pc-sdk .ot-sdk-columns:first-child,#ot-sdk-cookie-policy .ot-sdk-column:first-child,#ot-sdk-cookie-policy .ot-sdk-columns:first-child{margin-left:0}#onetrust-banner-sdk .ot-sdk-two.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-two.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-two.ot-sdk-columns{width:13.3333333333%}#onetrust-banner-sdk .ot-sdk-three.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-three.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-three.ot-sdk-columns{width:22%}#onetrust-banner-sdk .ot-sdk-four.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-four.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-four.ot-sdk-columns{width:30.6666666667%}#onetrust-banner-sdk .ot-sdk-eight.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-eight.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-eight.ot-sdk-columns{width:65.3333333333%}#onetrust-banner-sdk .ot-sdk-nine.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-nine.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-nine.ot-sdk-columns{width:74%}#onetrust-banner-sdk .ot-sdk-ten.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-ten.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-ten.ot-sdk-columns{width:82.6666666667%}#onetrust-banner-sdk .ot-sdk-eleven.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-eleven.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-eleven.ot-sdk-columns{width:91.3333333333%}#onetrust-banner-sdk .ot-sdk-twelve.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-twelve.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-twelve.ot-sdk-columns{width:100%;margin-left:0}}#onetrust-banner-sdk h1,#onetrust-banner-sdk h2,#onetrust-banner-sdk h3,#onetrust-banner-sdk h4,#onetrust-banner-sdk h5,#onetrust-banner-sdk h6,#onetrust-pc-sdk h1,#onetrust-pc-sdk h2,#onetrust-pc-sdk h3,#onetrust-pc-sdk h4,#onetrust-pc-sdk h5,#onetrust-pc-sdk h6,#ot-sdk-cookie-policy h1,#ot-sdk-cookie-policy h2,#ot-sdk-cookie-policy h3,#ot-sdk-cookie-policy h4,#ot-sdk-cookie-policy h5,#ot-sdk-cookie-policy h6{margin-top:0;font-weight:600;font-family:inherit}#onetrust-banner-sdk h1,#onetrust-pc-sdk h1,#ot-sdk-cookie-policy h1{font-size:1.5rem;line-height:1.2}#onetrust-banner-sdk h2,#onetrust-pc-sdk h2,#ot-sdk-cookie-policy h2{font-size:1.5rem;line-height:1.25}#onetrust-banner-sdk h3,#onetrust-pc-sdk h3,#ot-sdk-cookie-policy h3{font-size:1.5rem;line-height:1.3}#onetrust-banner-sdk h4,#onetrust-pc-sdk h4,#ot-sdk-cookie-policy h4{font-size:1.5rem;line-height:1.35}#onetrust-banner-sdk h5,#onetrust-pc-sdk h5,#ot-sdk-cookie-policy h5{font-size:1.5rem;line-height:1.5}#onetrust-banner-sdk h6,#onetrust-pc-sdk h6,#ot-sdk-cookie-policy h6{font-size:1.5rem;line-height:1.6}@media(min-width: 550px){#onetrust-banner-sdk h1,#onetrust-pc-sdk h1,#ot-sdk-cookie-policy h1{font-size:1.5rem}#onetrust-banner-sdk h2,#onetrust-pc-sdk h2,#ot-sdk-cookie-policy h2{font-size:1.5rem}#onetrust-banner-sdk h3,#onetrust-pc-sdk h3,#ot-sdk-cookie-policy h3{font-size:1.5rem}#onetrust-banner-sdk h4,#onetrust-pc-sdk h4,#ot-sdk-cookie-policy h4{font-size:1.5rem}#onetrust-banner-sdk h5,#onetrust-pc-sdk h5,#ot-sdk-cookie-policy h5{font-size:1.5rem}#onetrust-banner-sdk h6,#onetrust-pc-sdk h6,#ot-sdk-cookie-policy h6{font-size:1.5rem}}#onetrust-banner-sdk p,#onetrust-pc-sdk p,#ot-sdk-cookie-policy p{margin:0 0 1em 0;font-family:inherit;line-height:normal}#onetrust-banner-sdk a,#onetrust-pc-sdk a,#ot-sdk-cookie-policy a{color:#565656;text-decoration:underline}#onetrust-banner-sdk a:hover,#onetrust-pc-sdk a:hover,#ot-sdk-cookie-policy a:hover{color:#565656;text-decoration:none}#onetrust-banner-sdk .ot-sdk-button,#onetrust-banner-sdk button,#onetrust-pc-sdk .ot-sdk-button,#onetrust-pc-sdk button,#ot-sdk-cookie-policy .ot-sdk-button,#ot-sdk-cookie-policy button{margin-bottom:1rem;font-family:inherit}#onetrust-banner-sdk .ot-sdk-button,#onetrust-banner-sdk button,#onetrust-pc-sdk .ot-sdk-button,#onetrust-pc-sdk button,#ot-sdk-cookie-policy .ot-sdk-button,#ot-sdk-cookie-policy button{display:inline-block;height:38px;padding:0 30px;color:#555;text-align:center;font-size:.9em;font-weight:400;line-height:38px;letter-spacing:.01em;text-decoration:none;white-space:nowrap;background-color:rgba(0,0,0,0);border-radius:2px;border:1px solid #bbb;cursor:pointer;box-sizing:border-box}#onetrust-banner-sdk .ot-sdk-button:hover,#onetrust-banner-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):hover,#onetrust-banner-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):focus,#onetrust-pc-sdk .ot-sdk-button:hover,#onetrust-pc-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):hover,#onetrust-pc-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):focus,#ot-sdk-cookie-policy .ot-sdk-button:hover,#ot-sdk-cookie-policy :not(.ot-leg-btn-container)>button:not(.ot-link-btn):hover,#ot-sdk-cookie-policy :not(.ot-leg-btn-container)>button:not(.ot-link-btn):focus{color:#333;border-color:#888;opacity:.9}#onetrust-banner-sdk .ot-sdk-button:focus,#onetrust-banner-sdk :not(.ot-leg-btn-container)>button:focus,#onetrust-pc-sdk .ot-sdk-button:focus,#onetrust-pc-sdk :not(.ot-leg-btn-container)>button:focus,#ot-sdk-cookie-policy .ot-sdk-button:focus,#ot-sdk-cookie-policy :not(.ot-leg-btn-container)>button:focus{outline:2px solid #000}#onetrust-banner-sdk .ot-sdk-button.ot-sdk-button-primary,#onetrust-banner-sdk button.ot-sdk-button-primary,#onetrust-banner-sdk input[type=submit].ot-sdk-button-primary,#onetrust-banner-sdk input[type=reset].ot-sdk-button-primary,#onetrust-banner-sdk input[type=button].ot-sdk-button-primary,#onetrust-pc-sdk .ot-sdk-button.ot-sdk-button-primary,#onetrust-pc-sdk button.ot-sdk-button-primary,#onetrust-pc-sdk input[type=submit].ot-sdk-button-primary,#onetrust-pc-sdk input[type=reset].ot-sdk-button-primary,#onetrust-pc-sdk input[type=button].ot-sdk-button-primary,#ot-sdk-cookie-policy .ot-sdk-button.ot-sdk-button-primary,#ot-sdk-cookie-policy button.ot-sdk-button-primary,#ot-sdk-cookie-policy input[type=submit].ot-sdk-button-primary,#ot-sdk-cookie-policy input[type=reset].ot-sdk-button-primary,#ot-sdk-cookie-policy input[type=button].ot-sdk-button-primary{color:#fff;background-color:#33c3f0;border-color:#33c3f0}#onetrust-banner-sdk .ot-sdk-button.ot-sdk-button-primary:hover,#onetrust-banner-sdk button.ot-sdk-button-primary:hover,#onetrust-banner-sdk input[type=submit].ot-sdk-button-primary:hover,#onetrust-banner-sdk input[type=reset].ot-sdk-button-primary:hover,#onetrust-banner-sdk input[type=button].ot-sdk-button-primary:hover,#onetrust-banner-sdk .ot-sdk-button.ot-sdk-button-primary:focus,#onetrust-banner-sdk button.ot-sdk-button-primary:focus,#onetrust-banner-sdk input[type=submit].ot-sdk-button-primary:focus,#onetrust-banner-sdk input[type=reset].ot-sdk-button-primary:focus,#onetrust-banner-sdk input[type=button].ot-sdk-button-primary:focus,#onetrust-pc-sdk .ot-sdk-button.ot-sdk-button-primary:hover,#onetrust-pc-sdk button.ot-sdk-button-primary:hover,#onetrust-pc-sdk input[type=submit].ot-sdk-button-primary:hover,#onetrust-pc-sdk input[type=reset].ot-sdk-button-primary:hover,#onetrust-pc-sdk input[type=button].ot-sdk-button-primary:hover,#onetrust-pc-sdk .ot-sdk-button.ot-sdk-button-primary:focus,#onetrust-pc-sdk button.ot-sdk-button-primary:focus,#onetrust-pc-sdk input[type=submit].ot-sdk-button-primary:focus,#onetrust-pc-sdk input[type=reset].ot-sdk-button-primary:focus,#onetrust-pc-sdk input[type=button].ot-sdk-button-primary:focus,#ot-sdk-cookie-policy .ot-sdk-button.ot-sdk-button-primary:hover,#ot-sdk-cookie-policy button.ot-sdk-button-primary:hover,#ot-sdk-cookie-policy input[type=submit].ot-sdk-button-primary:hover,#ot-sdk-cookie-policy input[type=reset].ot-sdk-button-primary:hover,#ot-sdk-cookie-policy input[type=button].ot-sdk-button-primary:hover,#ot-sdk-cookie-policy .ot-sdk-button.ot-sdk-button-primary:focus,#ot-sdk-cookie-policy button.ot-sdk-button-primary:focus,#ot-sdk-cookie-policy input[type=submit].ot-sdk-button-primary:focus,#ot-sdk-cookie-policy input[type=reset].ot-sdk-button-primary:focus,#ot-sdk-cookie-policy input[type=button].ot-sdk-button-primary:focus{color:#fff;background-color:#1eaedb;border-color:#1eaedb}#onetrust-banner-sdk input[type=text],#onetrust-pc-sdk input[type=text],#ot-sdk-cookie-policy input[type=text]{height:38px;padding:6px 10px;background-color:#fff;border:1px solid #d1d1d1;border-radius:4px;box-shadow:none;box-sizing:border-box}#onetrust-banner-sdk input[type=text],#onetrust-pc-sdk input[type=text],#ot-sdk-cookie-policy input[type=text]{-webkit-appearance:none;-moz-appearance:none;appearance:none}#onetrust-banner-sdk input[type=text]:focus,#onetrust-pc-sdk input[type=text]:focus,#ot-sdk-cookie-policy input[type=text]:focus{border:1px solid #000;outline:0}#onetrust-banner-sdk label,#onetrust-pc-sdk label,#ot-sdk-cookie-policy label{display:block;margin-bottom:.5rem;font-weight:600}#onetrust-banner-sdk input[type=checkbox],#onetrust-pc-sdk input[type=checkbox],#ot-sdk-cookie-policy input[type=checkbox]{display:inline}#onetrust-banner-sdk ul,#onetrust-pc-sdk ul,#ot-sdk-cookie-policy ul{list-style:circle inside}#onetrust-banner-sdk ul,#onetrust-pc-sdk ul,#ot-sdk-cookie-policy ul{padding-left:0;margin-top:0}#onetrust-banner-sdk ul ul,#onetrust-pc-sdk ul ul,#ot-sdk-cookie-policy ul ul{margin:1.5rem 0 1.5rem 3rem;font-size:90%}#onetrust-banner-sdk li,#onetrust-pc-sdk li,#ot-sdk-cookie-policy li{margin-bottom:1rem}#onetrust-banner-sdk th,#onetrust-banner-sdk td,#onetrust-pc-sdk th,#onetrust-pc-sdk td,#ot-sdk-cookie-policy th,#ot-sdk-cookie-policy td{padding:12px 15px;text-align:left;border-bottom:1px solid #e1e1e1}#onetrust-banner-sdk button,#onetrust-pc-sdk button,#ot-sdk-cookie-policy button{margin-bottom:1rem;font-family:inherit}#onetrust-banner-sdk .ot-sdk-container:after,#onetrust-banner-sdk .ot-sdk-row:after,#onetrust-pc-sdk .ot-sdk-container:after,#onetrust-pc-sdk .ot-sdk-row:after,#ot-sdk-cookie-policy .ot-sdk-container:after,#ot-sdk-cookie-policy .ot-sdk-row:after{content:"";display:table;clear:both}#onetrust-banner-sdk .ot-sdk-row,#onetrust-pc-sdk .ot-sdk-row,#ot-sdk-cookie-policy .ot-sdk-row{margin:0;max-width:none;display:block}#onetrust-banner-sdk{box-shadow:0 0 18px rgba(0,0,0,.2)}#onetrust-banner-sdk.otFlat{position:fixed;z-index:2147483645;bottom:0;right:0;left:0;background-color:#fff;max-height:90%;overflow-x:hidden;overflow-y:auto}#onetrust-banner-sdk.otFlat.top{top:0px;bottom:auto}#onetrust-banner-sdk.otRelFont{font-size:1rem}#onetrust-banner-sdk>.ot-sdk-container{overflow:hidden}#onetrust-banner-sdk::-webkit-scrollbar{width:11px}#onetrust-banner-sdk::-webkit-scrollbar-thumb{border-radius:10px;background:#c1c1c1}#onetrust-banner-sdk{scrollbar-arrow-color:#c1c1c1;scrollbar-darkshadow-color:#c1c1c1;scrollbar-face-color:#c1c1c1;scrollbar-shadow-color:#c1c1c1}#onetrust-banner-sdk #onetrust-policy{margin:1.25em 0 .625em 2em;overflow:hidden}#onetrust-banner-sdk #onetrust-policy .ot-gv-list-handler{float:left;font-size:.82em;padding:0;margin-bottom:0;border:0;line-height:normal;height:auto;width:auto}#onetrust-banner-sdk #onetrust-policy-title{font-size:1.2em;line-height:1.3;margin-bottom:10px}#onetrust-banner-sdk #onetrust-policy-text{clear:both;text-align:left;font-size:.88em;line-height:1.4}#onetrust-banner-sdk #onetrust-policy-text *{font-size:inherit;line-height:inherit}#onetrust-banner-sdk #onetrust-policy-text a{font-weight:bold;margin-left:5px}#onetrust-banner-sdk #onetrust-policy-title,#onetrust-banner-sdk #onetrust-policy-text{color:dimgray;float:left}#onetrust-banner-sdk #onetrust-button-group-parent{min-height:1px;text-align:center}#onetrust-banner-sdk #onetrust-button-group{display:inline-block}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{background-color:#68b631;color:#fff;border-color:#68b631;margin-right:1em;min-width:125px;height:auto;white-space:normal;word-break:break-word;word-wrap:break-word;padding:12px 10px;line-height:1.2;font-size:.813em;font-weight:600}#onetrust-banner-sdk #onetrust-pc-btn-handler.cookie-setting-link{background-color:#fff;border:none;color:#68b631;text-decoration:underline;padding-left:0;padding-right:0}#onetrust-banner-sdk .onetrust-close-btn-ui{width:44px;height:44px;background-size:12px;border:none;position:relative;margin:auto;padding:0}#onetrust-banner-sdk .banner_logo{display:none}#onetrust-banner-sdk.ot-bnr-w-logo .ot-bnr-logo{position:absolute;top:50%;transform:translateY(-50%);left:0px}#onetrust-banner-sdk.ot-bnr-w-logo #onetrust-policy{margin-left:65px}#onetrust-banner-sdk .ot-b-addl-desc{clear:both;float:left;display:block}#onetrust-banner-sdk #banner-options{float:left;display:table;margin-right:0;margin-left:1em;width:calc(100% - 1em)}#onetrust-banner-sdk .banner-option-input{cursor:pointer;width:auto;height:auto;border:none;padding:0;padding-right:3px;margin:0 0 10px;font-size:.82em;line-height:1.4}#onetrust-banner-sdk .banner-option-input *{pointer-events:none;font-size:inherit;line-height:inherit}#onetrust-banner-sdk .banner-option-input[aria-expanded=true]~.banner-option-details{display:block;height:auto}#onetrust-banner-sdk .banner-option-input[aria-expanded=true] .ot-arrow-container{transform:rotate(90deg)}#onetrust-banner-sdk .banner-option{margin-bottom:12px;margin-left:0;border:none;float:left;padding:0}#onetrust-banner-sdk .banner-option:first-child{padding-left:2px}#onetrust-banner-sdk .banner-option:not(:first-child){padding:0;border:none}#onetrust-banner-sdk .banner-option-header{cursor:pointer;display:inline-block}#onetrust-banner-sdk .banner-option-header :first-child{color:dimgray;font-weight:bold;float:left}#onetrust-banner-sdk .banner-option-header .ot-arrow-container{display:inline-block;border-top:6px solid rgba(0,0,0,0);border-bottom:6px solid rgba(0,0,0,0);border-left:6px solid dimgray;margin-left:10px;vertical-align:middle}#onetrust-banner-sdk .banner-option-details{display:none;font-size:.83em;line-height:1.5;padding:10px 0px 5px 10px;margin-right:10px;height:0px}#onetrust-banner-sdk .banner-option-details *{font-size:inherit;line-height:inherit;color:dimgray}#onetrust-banner-sdk .ot-arrow-container,#onetrust-banner-sdk .banner-option-details{transition:all 300ms ease-in 0s;-webkit-transition:all 300ms ease-in 0s;-moz-transition:all 300ms ease-in 0s;-o-transition:all 300ms ease-in 0s}#onetrust-banner-sdk .ot-dpd-container{float:left}#onetrust-banner-sdk .ot-dpd-title{margin-bottom:10px}#onetrust-banner-sdk .ot-dpd-title,#onetrust-banner-sdk .ot-dpd-desc{font-size:.88em;line-height:1.4;color:dimgray}#onetrust-banner-sdk .ot-dpd-title *,#onetrust-banner-sdk .ot-dpd-desc *{font-size:inherit;line-height:inherit}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-text *{margin-bottom:0}#onetrust-banner-sdk.ot-iab-2 .onetrust-vendors-list-handler{display:block;margin-left:0;margin-top:5px;clear:both;margin-bottom:0;padding:0;border:0;height:auto;width:auto}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group button{display:block}#onetrust-banner-sdk.ot-close-btn-link{padding-top:25px}#onetrust-banner-sdk.ot-close-btn-link #onetrust-close-btn-container{top:15px;transform:none;right:15px}#onetrust-banner-sdk.ot-close-btn-link #onetrust-close-btn-container button{padding:0;white-space:pre-wrap;border:none;height:auto;line-height:1.5;text-decoration:underline;font-size:.69em}#onetrust-banner-sdk #onetrust-policy-text,#onetrust-banner-sdk .ot-dpd-desc,#onetrust-banner-sdk .ot-b-addl-desc{font-size:.813em;line-height:1.5}#onetrust-banner-sdk .ot-dpd-desc{margin-bottom:10px}#onetrust-banner-sdk .ot-dpd-desc>.ot-b-addl-desc{margin-top:10px;margin-bottom:10px;font-size:1em}@media only screen and (max-width: 425px){#onetrust-banner-sdk #onetrust-close-btn-container{position:absolute;top:6px;right:2px}#onetrust-banner-sdk #onetrust-policy{margin-left:0;margin-top:3em}#onetrust-banner-sdk #onetrust-button-group{display:block}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{width:100%}#onetrust-banner-sdk .onetrust-close-btn-ui{top:auto;transform:none}#onetrust-banner-sdk #onetrust-policy-title{display:inline;float:none}#onetrust-banner-sdk #banner-options{margin:0;padding:0;width:100%}}@media only screen and (min-width: 426px)and (max-width: 896px){#onetrust-banner-sdk #onetrust-close-btn-container{position:absolute;top:0;right:0}#onetrust-banner-sdk #onetrust-policy{margin-left:1em;margin-right:1em}#onetrust-banner-sdk .onetrust-close-btn-ui{top:10px;right:10px}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{width:95%}#onetrust-banner-sdk.ot-iab-2 #onetrust-group-container{width:100%}#onetrust-banner-sdk.ot-bnr-w-logo #onetrust-button-group-parent{padding-left:50px}#onetrust-banner-sdk #onetrust-button-group-parent{width:100%;position:relative;margin-left:0}#onetrust-banner-sdk #onetrust-button-group button{display:inline-block}#onetrust-banner-sdk #onetrust-button-group{margin-right:0;text-align:center}#onetrust-banner-sdk .has-reject-all-button #onetrust-pc-btn-handler{float:left}#onetrust-banner-sdk .has-reject-all-button #onetrust-reject-all-handler,#onetrust-banner-sdk .has-reject-all-button #onetrust-accept-btn-handler{float:right}#onetrust-banner-sdk .has-reject-all-button #onetrust-button-group{width:calc(100% - 2em);margin-right:0}#onetrust-banner-sdk .has-reject-all-button #onetrust-pc-btn-handler.cookie-setting-link{padding-left:0px;text-align:left}#onetrust-banner-sdk.ot-buttons-fw .ot-sdk-three button{width:100%;text-align:center}#onetrust-banner-sdk.ot-buttons-fw #onetrust-button-group-parent button{float:none}#onetrust-banner-sdk.ot-buttons-fw #onetrust-pc-btn-handler.cookie-setting-link{text-align:center}}@media only screen and (min-width: 550px){#onetrust-banner-sdk .banner-option:not(:first-child){border-left:1px solid #d8d8d8;padding-left:25px}}@media only screen and (min-width: 425px)and (max-width: 550px){#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group,#onetrust-banner-sdk.ot-iab-2 #onetrust-policy,#onetrust-banner-sdk.ot-iab-2 .banner-option{width:100%}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-accept-btn-handler,#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-reject-all-handler,#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-pc-btn-handler{width:100%}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-accept-btn-handler,#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-reject-all-handler{float:left}}@media only screen and (min-width: 769px){#onetrust-banner-sdk #onetrust-button-group{margin-right:30%}#onetrust-banner-sdk #banner-options{margin-left:2em;margin-right:5em;margin-bottom:1.25em;width:calc(100% - 7em)}}@media only screen and (min-width: 897px)and (max-width: 1023px){#onetrust-banner-sdk.vertical-align-content #onetrust-button-group-parent{position:absolute;top:50%;left:75%;transform:translateY(-50%)}#onetrust-banner-sdk #onetrust-close-btn-container{top:50%;margin:auto;transform:translate(-50%, -50%);position:absolute;padding:0;right:0}#onetrust-banner-sdk #onetrust-close-btn-container button{position:relative;margin:0;right:-22px;top:2px}}@media only screen and (min-width: 1024px){#onetrust-banner-sdk #onetrust-close-btn-container{top:50%;margin:auto;transform:translate(-50%, -50%);position:absolute;right:0}#onetrust-banner-sdk #onetrust-close-btn-container button{right:-12px}#onetrust-banner-sdk #onetrust-policy{margin-left:2em}#onetrust-banner-sdk.vertical-align-content #onetrust-button-group-parent{position:absolute;top:50%;left:60%;transform:translateY(-50%)}#onetrust-banner-sdk .ot-optout-signal{width:50%}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-title{width:50%}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-text,#onetrust-banner-sdk.ot-iab-2 :not(.ot-dpd-desc)>.ot-b-addl-desc{margin-bottom:1em;width:50%;border-right:1px solid #d8d8d8;padding-right:1rem}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-text{margin-bottom:0;padding-bottom:1em}#onetrust-banner-sdk.ot-iab-2 :not(.ot-dpd-desc)>.ot-b-addl-desc{margin-bottom:0;padding-bottom:1em}#onetrust-banner-sdk.ot-iab-2 .ot-dpd-container{width:45%;padding-left:1rem;display:inline-block;float:none}#onetrust-banner-sdk.ot-iab-2 .ot-dpd-title{line-height:1.7}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group-parent{left:auto;right:4%;margin-left:0}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group button{display:block}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{margin:auto;width:30%}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{width:60%}#onetrust-banner-sdk #onetrust-button-group{margin-right:auto}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{margin-top:1em}}@media only screen and (min-width: 890px){#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group-parent{padding-left:3%;padding-right:4%;margin-left:0}#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group{margin-right:0;margin-top:1.25em;width:100%}#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button{width:100%;margin-bottom:5px;margin-top:5px}#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button:last-of-type{margin-bottom:20px}}@media only screen and (min-width: 1280px){#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{width:55%}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{width:44%;padding-left:2%;padding-right:2%}#onetrust-banner-sdk:not(.ot-iab-2).vertical-align-content #onetrust-button-group-parent{position:absolute;left:55%}}
        #onetrust-consent-sdk #onetrust-banner-sdk {background-color: #FFF;}
            #onetrust-consent-sdk #onetrust-policy-title,
                    #onetrust-consent-sdk #onetrust-policy-text,
                    #onetrust-consent-sdk .ot-b-addl-desc,
                    #onetrust-consent-sdk .ot-dpd-desc,
                    #onetrust-consent-sdk .ot-dpd-title,
                    #onetrust-consent-sdk #onetrust-policy-text *:not(.onetrust-vendors-list-handler),
                    #onetrust-consent-sdk .ot-dpd-desc *:not(.onetrust-vendors-list-handler),
                    #onetrust-consent-sdk #onetrust-banner-sdk #banner-options *,
                    #onetrust-banner-sdk .ot-cat-header,
                    #onetrust-banner-sdk .ot-optout-signal
                    {
                        color: #2E2E2E;
                    }
            #onetrust-consent-sdk #onetrust-banner-sdk .banner-option-details {
                    background-color: #E9E9E9;}
             #onetrust-consent-sdk #onetrust-banner-sdk a[href],
                    #onetrust-consent-sdk #onetrust-banner-sdk a[href] font,
                    #onetrust-consent-sdk #onetrust-banner-sdk .ot-link-btn
                        {
                            color: #007398;
                        }#onetrust-consent-sdk #onetrust-accept-btn-handler,
                         #onetrust-banner-sdk #onetrust-reject-all-handler {
                            background-color: #007398;border-color: #007398;
                color: #FFF;
            }
            #onetrust-consent-sdk #onetrust-banner-sdk *:focus,
            #onetrust-consent-sdk #onetrust-banner-sdk:focus {
               outline-color: #000000;
               outline-width: 1px;
            }
            #onetrust-consent-sdk #onetrust-pc-btn-handler,
            #onetrust-consent-sdk #onetrust-pc-btn-handler.cookie-setting-link {
                color: #6CC04A; border-color: #6CC04A;
                background-color:
                #FFF;
            }/*! v6.10.0 2021-01-18 */
div#onetrust-consent-sdk #onetrust-banner-sdk{border-top:2px solid #d0d3d4!important;box-shadow:none;padding:24px}div#onetrust-consent-sdk button{border-radius:0!important;box-shadow:none!important;box-sizing:border-box!important;font-size:20px!important;font-weight:400!important;letter-spacing:0!important;max-width:none!important;white-space:nowrap!important}div#onetrust-consent-sdk button:not(.ot-link-btn){background-color:#007dbc!important;border:2px solid #007dbc!important;color:#fff!important;height:48px!important;padding:0 1em!important;width:auto!important}div#onetrust-consent-sdk button:hover{background-color:#fff!important;border-color:#949494!important;color:#2e2e2e!important}div#onetrust-consent-sdk button.ot-link-btn{color:#007dbc!important;font-size:16px!important;text-decoration:underline}div#onetrust-consent-sdk button.ot-link-btn:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-consent-sdk a,div#onetrust-pc-sdk a{color:#007dbc!important;text-decoration:underline!important}div#onetrust-consent-sdk a,div#onetrust-consent-sdk button,div#onetrust-consent-sdk p:hover{opacity:1!important}div#onetrust-consent-sdk a:focus,div#onetrust-consent-sdk button:focus,div#onetrust-consent-sdk input:focus{outline:2px solid #949494!important;outline-offset:1px!important}div#onetrust-banner-sdk .ot-sdk-container{padding:0;width:auto}div#onetrust-banner-sdk .ot-sdk-row{align-items:flex-start;box-sizing:border-box;display:flex;flex-direction:column;justify-content:space-between;margin:auto;max-width:1152px}div#onetrust-banner-sdk .ot-sdk-row:after{display:none}div#onetrust-banner-sdk #onetrust-group-container,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-group-container,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{flex-grow:1;width:auto}div#onetrust-banner-sdk #onetrust-policy,div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy{margin:0;overflow:visible}div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy-text,div#onetrust-consent-sdk #onetrust-policy-text{font-size:16px;line-height:24px;max-width:44em;margin:0}div#onetrust-consent-sdk #onetrust-policy-text a[href]{font-weight:400;margin-left:8px}div#onetrust-banner-sdk #onetrust-button-group-parent{flex:0 0 auto;margin:32px 0 0;width:100%}div#onetrust-banner-sdk #onetrust-button-group{display:flex;flex-direction:row;flex-wrap:wrap;justify-content:flex-end;margin:-8px}div#onetrust-banner-sdk .banner-actions-container{display:flex;flex:1 0 auto}div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button:last-of-type,div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto;margin:8px;width:auto}div#onetrust-consent-sdk #onetrust-pc-btn-handler{background-color:#fff!important;color:inherit!important}div#onetrust-banner-sdk #onetrust-close-btn-container{display:none}@media only screen and (min-width:556px){div#onetrust-consent-sdk #onetrust-banner-sdk{padding:40px}div#onetrust-banner-sdk #onetrust-policy{margin:0 40px 0 0}div#onetrust-banner-sdk .ot-sdk-row{align-items:center;flex-direction:row}div#onetrust-banner-sdk #onetrust-button-group-parent,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-button-group-parent,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{margin:0;padding:0;width:auto}div#onetrust-banner-sdk #onetrust-button-group,div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group{align-items:stretch;flex-direction:column-reverse;margin:0}div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto}}@media only screen and (min-width:768px){div#onetrust-banner-sdk #onetrust-policy{margin:0 48px 0 0}div#onetrust-consent-sdk #onetrust-banner-sdk{padding:48px}}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-size:16px;line-height:24px}div#onetrust-consent-sdk #onetrust-pc-sdk p,div#onetrust-pc-sdk #ot-pc-desc,div#onetrust-pc-sdk .category-host-list-handler,div#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{font-size:16px;font-weight:400;line-height:24px}div#onetrust-consent-sdk a:hover,div#onetrust-pc-sdk a:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-pc-sdk{border-radius:0;bottom:0;height:auto;left:0;margin:auto;max-width:100%;overflow:hidden;right:0;top:0;width:512px;max-height:800px}div#onetrust-pc-sdk .ot-pc-header{display:none}div#onetrust-pc-sdk #ot-pc-content{overscroll-behavior:contain;padding:0 12px 0 24px;margin:16px 4px 0 0;top:0;right:16px;left:0;width:auto}div#onetrust-pc-sdk #ot-category-title,div#onetrust-pc-sdk #ot-pc-title{font-size:24px;font-weight:400;line-height:32px;margin:0 0 16px}div#onetrust-pc-sdk #ot-pc-desc{padding:0}div#onetrust-pc-sdk #ot-pc-desc a{display:inline}div#onetrust-pc-sdk #accept-recommended-btn-handler{display:none!important}div#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:2px solid #007dbc;outline-offset:-1px;transition:none}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item{border-width:0 0 2px}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item:first-of-type{border-width:2px 0}div#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{padding:8px 0;width:100%}div#onetrust-pc-sdk .ot-plus-minus{transform:translateY(2px)}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item>button{background:0 0!important;border:0!important;height:44px!important;max-width:none!important;width:calc(100% - 48px)!important}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-weight:700}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr{padding:0}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item .ot-acc-grpdesc{padding:0;width:100%}div#onetrust-pc-sdk .ot-acc-grpcntr .ot-subgrp-cntr{border:0;padding:0}div#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp{margin:0}div#onetrust-pc-sdk .ot-always-active-group .ot-cat-header{width:calc(100% - 160px)}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{width:calc(100% - 88px)}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active{color:inherit;font-size:12px;font-weight:400;line-height:1.5;padding-right:48px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:before{border-radius:12px;position:absolute;right:0;top:0;content:'';background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:after{border-radius:50%;position:absolute;right:5px;top:4px;content:'';background-color:#007dbc;height:12px;width:12px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active,div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl{right:2px}div#onetrust-pc-sdk .ot-switch{display:block;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob,div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob{background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob:before{background-color:#737373;height:8px;left:4px;top:4px;width:8px}div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob:before{background-color:#007dbc;height:12px;left:0;top:2px;width:12px}div#onetrust-pc-sdk .ot-tgl input:focus+.ot-switch .ot-switch-nob{box-shadow:0 0;outline:2px solid #007dbc!important;outline-offset:1px;transition:none}div#onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-grpcntr.ot-acc-txt{background-color:transparent;padding-left:3px}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr,div#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr{overflow:visible;width:100%}div#onetrust-pc-sdk .ot-pc-footer{border-top:0 solid}div#onetrust-pc-sdk .ot-btn-container{padding-top:24px;text-align:center}div#onetrust-pc-sdk .ot-pc-footer button{margin:8px 0;background-color:#fff}div#onetrust-pc-sdk .ot-pc-footer-logo{background-color:#fff}div#onetrust-pc-sdk #ot-lst-title span{font-size:24px;font-weight:400;line-height:32px}div#onetrust-pc-sdk #ot-host-lst .ot-host-desc,div#onetrust-pc-sdk #ot-host-lst .ot-host-expand,div#onetrust-pc-sdk #ot-host-lst .ot-host-name,div#onetrust-pc-sdk #ot-host-lst .ot-host-name a,div#onetrust-pc-sdk .back-btn-handler,div#onetrust-pc-sdk .ot-host-opt li>div div{font-size:16px;font-weight:400;line-height:24px}div#onetrust-pc-sdk #ot-pc-lst{top:0}div#onetrust-pc-sdk .back-btn-handler{text-decoration:none!important}div#onetrust-pc-sdk #filter-btn-handler:hover svg{filter:invert(1)}div#onetrust-pc-sdk .back-btn-handler svg{width:16px;height:16px}div#onetrust-pc-sdk .ot-host-item>button{background:0 0!important;border:0!important;height:66px!important;max-width:none!important;width:calc(100% - 5px)!important;transform:translate(2px,2px)}div#onetrust-pc-sdk .ot-host-item{border-bottom:2px solid #b9b9b9;padding:0}div#onetrust-pc-sdk .ot-host-item .ot-acc-hdr{margin:0 0 -6px;padding:8px 0}div#onetrust-pc-sdk ul li:first-child{border-top:2px solid #b9b9b9}div#onetrust-pc-sdk .ot-host-item .ot-plus-minus{margin:0 8px 0 0}div#onetrust-pc-sdk .ot-search-cntr{width:calc(100% - 48px)}div#onetrust-pc-sdk .ot-host-opt .ot-host-info{background-color:transparent}div#onetrust-pc-sdk .ot-host-opt li>div div{padding:0}div#onetrust-pc-sdk #vendor-search-handler{border-radius:0;border-color:#939393;border-style:solid;border-width:2px 0 2px 2px;font-size:20px;height:48px;margin:0}div#onetrust-pc-sdk .ot-lst-subhdr svg{right:0;top:8px}div#onetrust-pc-sdk .ot-fltr-cntr{box-sizing:border-box;right:0;width:48px}div#onetrust-pc-sdk #filter-btn-handler{width:48px!important;padding:8px!important}div#onetrust-pc-sdk #ot-fltr-modal{max-height:400px}
#onetrust-pc-sdk.otPcCenter{overflow:hidden;position:fixed;margin:0 auto;top:5%;right:0;left:0;width:40%;max-width:575px;min-width:575px;border-radius:2.5px;z-index:2147483647;background-color:#fff;-webkit-box-shadow:0px 2px 10px -3px #999;-moz-box-shadow:0px 2px 10px -3px #999;box-shadow:0px 2px 10px -3px #999}#onetrust-pc-sdk.otPcCenter[dir=rtl]{right:0;left:0}#onetrust-pc-sdk.otRelFont{font-size:1rem}#onetrust-pc-sdk .ot-optout-signal{margin-top:.625rem}#onetrust-pc-sdk #ot-addtl-venlst .ot-arw-cntr,#onetrust-pc-sdk #ot-addtl-venlst .ot-plus-minus,#onetrust-pc-sdk .ot-hide-tgl{visibility:hidden}#onetrust-pc-sdk #ot-addtl-venlst .ot-arw-cntr *,#onetrust-pc-sdk #ot-addtl-venlst .ot-plus-minus *,#onetrust-pc-sdk .ot-hide-tgl *{visibility:hidden}#onetrust-pc-sdk #ot-gn-venlst .ot-ven-item .ot-acc-hdr{min-height:40px}#onetrust-pc-sdk .ot-pc-header{height:39px;padding:10px 0 10px 30px;border-bottom:1px solid #e9e9e9}#onetrust-pc-sdk #ot-pc-title,#onetrust-pc-sdk #ot-category-title,#onetrust-pc-sdk .ot-cat-header,#onetrust-pc-sdk #ot-lst-title,#onetrust-pc-sdk .ot-ven-hdr .ot-ven-name,#onetrust-pc-sdk .ot-always-active{font-weight:bold;color:dimgray}#onetrust-pc-sdk .ot-always-active-group .ot-cat-header{width:55%;font-weight:700}#onetrust-pc-sdk .ot-cat-item p{clear:both;float:left;margin-top:10px;margin-bottom:5px;line-height:1.5;font-size:.812em;color:dimgray}#onetrust-pc-sdk .ot-close-icon{height:44px;width:44px;background-size:10px}#onetrust-pc-sdk #ot-pc-title{float:left;font-size:1em;line-height:1.5;margin-bottom:10px;margin-top:10px;width:100%}#onetrust-pc-sdk #accept-recommended-btn-handler{margin-right:10px;margin-bottom:25px;outline-offset:-1px}#onetrust-pc-sdk #ot-pc-desc{clear:both;width:100%;font-size:.812em;line-height:1.5;margin-bottom:25px}#onetrust-pc-sdk #ot-pc-desc *{font-size:inherit;line-height:inherit}#onetrust-pc-sdk #ot-pc-desc ul li{padding:10px 0px}#onetrust-pc-sdk a{color:#656565;cursor:pointer}#onetrust-pc-sdk a:hover{color:#3860be}#onetrust-pc-sdk label{margin-bottom:0}#onetrust-pc-sdk #vdr-lst-dsc{font-size:.812em;line-height:1.5;padding:10px 15px 5px 15px}#onetrust-pc-sdk button{max-width:394px;padding:12px 30px;line-height:1;word-break:break-word;word-wrap:break-word;white-space:normal;font-weight:bold;height:auto}#onetrust-pc-sdk .ot-link-btn{padding:0;margin-bottom:0;border:0;font-weight:normal;line-height:normal;width:auto;height:auto}#onetrust-pc-sdk #ot-pc-content{position:absolute;overflow-y:scroll;padding-left:2px;padding-right:30px;top:60px;bottom:110px;margin:1px 3px 0 30px;width:calc(100% - 63px)}#onetrust-pc-sdk .ot-vs-list .ot-always-active,#onetrust-pc-sdk .ot-cat-grp .ot-always-active{float:right;clear:none;color:#3860be;margin:0;font-size:.813em;line-height:1.3}#onetrust-pc-sdk .ot-pc-scrollbar::-webkit-scrollbar-track{margin-right:20px}#onetrust-pc-sdk .ot-pc-scrollbar::-webkit-scrollbar{width:11px}#onetrust-pc-sdk .ot-pc-scrollbar::-webkit-scrollbar-thumb{border-radius:10px;background:#d8d8d8}#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:#000 1px solid}#onetrust-pc-sdk .ot-pc-scrollbar{scrollbar-arrow-color:#d8d8d8;scrollbar-darkshadow-color:#d8d8d8;scrollbar-face-color:#d8d8d8;scrollbar-shadow-color:#d8d8d8}#onetrust-pc-sdk .save-preference-btn-handler{margin-right:20px}#onetrust-pc-sdk .ot-pc-refuse-all-handler{margin-right:10px}#onetrust-pc-sdk #ot-pc-desc .privacy-notice-link{margin-left:0;margin-right:8px}#onetrust-pc-sdk #ot-pc-desc .ot-imprint-handler{margin-left:0;margin-right:8px}#onetrust-pc-sdk .ot-subgrp-cntr{display:inline-block;clear:both;width:100%;padding-top:15px}#onetrust-pc-sdk .ot-switch+.ot-subgrp-cntr{padding-top:10px}#onetrust-pc-sdk ul.ot-subgrps{margin:0;font-size:initial}#onetrust-pc-sdk ul.ot-subgrps li p,#onetrust-pc-sdk ul.ot-subgrps li h5{font-size:.813em;line-height:1.4;color:dimgray}#onetrust-pc-sdk ul.ot-subgrps .ot-switch{min-height:auto}#onetrust-pc-sdk ul.ot-subgrps .ot-switch-nob{top:0}#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr{display:inline-block;width:100%}#onetrust-pc-sdk ul.ot-subgrps .ot-acc-txt{margin:0}#onetrust-pc-sdk ul.ot-subgrps li{padding:0;border:none}#onetrust-pc-sdk ul.ot-subgrps li h5{position:relative;top:5px;font-weight:bold;margin-bottom:0;float:left}#onetrust-pc-sdk li.ot-subgrp{margin-left:20px;overflow:auto}#onetrust-pc-sdk li.ot-subgrp>h5{width:calc(100% - 100px)}#onetrust-pc-sdk .ot-cat-item p>ul,#onetrust-pc-sdk li.ot-subgrp p>ul{margin:0px;list-style:disc;margin-left:15px;font-size:inherit}#onetrust-pc-sdk .ot-cat-item p>ul li,#onetrust-pc-sdk li.ot-subgrp p>ul li{font-size:inherit;padding-top:10px;padding-left:0px;padding-right:0px;border:none}#onetrust-pc-sdk .ot-cat-item p>ul li:last-child,#onetrust-pc-sdk li.ot-subgrp p>ul li:last-child{padding-bottom:10px}#onetrust-pc-sdk .ot-pc-logo{height:40px;width:120px}#onetrust-pc-sdk .ot-pc-footer{position:absolute;bottom:0px;width:100%;max-height:160px;border-top:1px solid #d8d8d8}#onetrust-pc-sdk.ot-ftr-stacked .ot-pc-refuse-all-handler{margin-bottom:0px}#onetrust-pc-sdk.ot-ftr-stacked #ot-pc-content{bottom:160px}#onetrust-pc-sdk.ot-ftr-stacked .ot-pc-footer button{width:100%;max-width:none}#onetrust-pc-sdk.ot-ftr-stacked .ot-btn-container{margin:0 30px;width:calc(100% - 60px);padding-right:0}#onetrust-pc-sdk .ot-pc-footer-logo{height:30px;width:100%;text-align:right;background:#f4f4f4}#onetrust-pc-sdk .ot-pc-footer-logo a{display:inline-block;margin-top:5px;margin-right:10px}#onetrust-pc-sdk[dir=rtl] .ot-pc-footer-logo{direction:rtl}#onetrust-pc-sdk[dir=rtl] .ot-pc-footer-logo a{margin-right:25px}#onetrust-pc-sdk .ot-tgl{float:right;position:relative;z-index:1}#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob{background-color:#468254;border:1px solid #fff}#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob:before{-webkit-transform:translateX(20px);-ms-transform:translateX(20px);transform:translateX(20px);background-color:#fff;border-color:#fff}#onetrust-pc-sdk .ot-tgl input:focus+.ot-switch{outline:#000 solid 1px}#onetrust-pc-sdk .ot-switch{position:relative;display:inline-block;width:45px;height:25px}#onetrust-pc-sdk .ot-switch-nob{position:absolute;cursor:pointer;top:0;left:0;right:0;bottom:0;background-color:#767676;border:1px solid #ddd;transition:all .2s ease-in 0s;-moz-transition:all .2s ease-in 0s;-o-transition:all .2s ease-in 0s;-webkit-transition:all .2s ease-in 0s;border-radius:20px}#onetrust-pc-sdk .ot-switch-nob:before{position:absolute;content:"";height:18px;width:18px;bottom:3px;left:3px;background-color:#fff;-webkit-transition:.4s;transition:.4s;border-radius:20px}#onetrust-pc-sdk .ot-chkbox input:checked~label::before{background-color:#3860be}#onetrust-pc-sdk .ot-chkbox input+label::after{content:none;color:#fff}#onetrust-pc-sdk .ot-chkbox input:checked+label::after{content:""}#onetrust-pc-sdk .ot-chkbox input:focus+label::before{outline-style:solid;outline-width:2px;outline-style:auto}#onetrust-pc-sdk .ot-chkbox input[aria-checked=mixed]~label::before{background-color:#3860be}#onetrust-pc-sdk .ot-chkbox input[aria-checked=mixed]+label::after{content:""}#onetrust-pc-sdk .ot-chkbox label{position:relative;display:inline-block;padding-left:30px;cursor:pointer;font-weight:500}#onetrust-pc-sdk .ot-chkbox label::before,#onetrust-pc-sdk .ot-chkbox label::after{position:absolute;content:"";display:inline-block;border-radius:3px}#onetrust-pc-sdk .ot-chkbox label::before{height:18px;width:18px;border:1px solid #3860be;left:0px;top:auto}#onetrust-pc-sdk .ot-chkbox label::after{height:5px;width:9px;border-left:3px solid;border-bottom:3px solid;transform:rotate(-45deg);-o-transform:rotate(-45deg);-ms-transform:rotate(-45deg);-webkit-transform:rotate(-45deg);left:4px;top:5px}#onetrust-pc-sdk .ot-label-txt{display:none}#onetrust-pc-sdk .ot-chkbox input,#onetrust-pc-sdk .ot-tgl input{position:absolute;opacity:0;width:0;height:0}#onetrust-pc-sdk .ot-arw-cntr{float:right;position:relative;pointer-events:none}#onetrust-pc-sdk .ot-arw-cntr .ot-arw{width:16px;height:16px;margin-left:5px;color:dimgray;display:inline-block;vertical-align:middle;-webkit-transition:all 150ms ease-in 0s;-moz-transition:all 150ms ease-in 0s;-o-transition:all 150ms ease-in 0s;transition:all 150ms ease-in 0s}#onetrust-pc-sdk input:checked~.ot-acc-hdr .ot-arw,#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-arw-cntr svg{transform:rotate(90deg);-o-transform:rotate(90deg);-ms-transform:rotate(90deg);-webkit-transform:rotate(90deg)}#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:#000 1px solid}#onetrust-pc-sdk .ot-tgl-cntr,#onetrust-pc-sdk .ot-arw-cntr{display:inline-block}#onetrust-pc-sdk .ot-tgl-cntr{width:45px;float:right;margin-top:2px}#onetrust-pc-sdk #ot-lst-cnt .ot-tgl-cntr{margin-top:10px}#onetrust-pc-sdk .ot-always-active-subgroup{width:auto;padding-left:0px !important;top:3px;position:relative}#onetrust-pc-sdk .ot-label-status{font-size:.75em;display:none;font-size:.75em;position:relative;top:2px;padding-right:5px;float:left}#onetrust-pc-sdk .ot-arw-cntr{margin-top:-1px}#onetrust-pc-sdk .ot-arw-cntr svg{-webkit-transition:all 300ms ease-in 0s;-moz-transition:all 300ms ease-in 0s;-o-transition:all 300ms ease-in 0s;transition:all 300ms ease-in 0s;height:10px;width:10px}#onetrust-pc-sdk input:checked~.ot-acc-hdr .ot-arw{transform:rotate(90deg);-o-transform:rotate(90deg);-ms-transform:rotate(90deg);-webkit-transform:rotate(90deg)}#onetrust-pc-sdk .ot-arw{width:10px;margin-left:15px;transition:all 300ms ease-in 0s;-webkit-transition:all 300ms ease-in 0s;-moz-transition:all 300ms ease-in 0s;-o-transition:all 300ms ease-in 0s}#onetrust-pc-sdk .ot-vlst-cntr{margin-bottom:0}#onetrust-pc-sdk .ot-hlst-cntr{margin-top:5px;display:inline-block;width:100%}#onetrust-pc-sdk .category-vendors-list-handler,#onetrust-pc-sdk .category-vendors-list-handler+a,#onetrust-pc-sdk .category-host-list-handler{clear:both;color:#3860be;margin-left:0;font-size:.813em;text-decoration:none;float:left;overflow:hidden}#onetrust-pc-sdk .category-vendors-list-handler:hover,#onetrust-pc-sdk .category-vendors-list-handler+a:hover,#onetrust-pc-sdk .category-host-list-handler:hover{text-decoration-line:underline}#onetrust-pc-sdk .category-vendors-list-handler+a{clear:none}#onetrust-pc-sdk .ot-vlst-cntr .ot-ext-lnk,#onetrust-pc-sdk .ot-ven-hdr .ot-ext-lnk{display:inline-block;height:13px;width:13px;background-repeat:no-repeat;margin-left:1px;margin-top:6px;cursor:pointer}#onetrust-pc-sdk .ot-ven-hdr .ot-ext-lnk{margin-bottom:-1px}#onetrust-pc-sdk .back-btn-handler{font-size:1em;text-decoration:none}#onetrust-pc-sdk .back-btn-handler:hover{opacity:.6}#onetrust-pc-sdk #ot-lst-title h3{display:inline-block;word-break:break-word;word-wrap:break-word;margin-bottom:0;color:#656565;font-size:1em;font-weight:bold;margin-left:15px}#onetrust-pc-sdk #ot-lst-title{margin:10px 0 10px 0px;font-size:1em;text-align:left}#onetrust-pc-sdk #ot-pc-hdr{margin:0 0 0 30px;height:auto;width:auto}#onetrust-pc-sdk #ot-pc-hdr input::placeholder{color:#707070;font-style:italic}#onetrust-pc-sdk #vendor-search-handler{height:31px;width:100%;border-radius:50px;font-size:.8em;padding-right:35px;padding-left:15px;float:left;margin-left:15px}#onetrust-pc-sdk .ot-ven-name{display:block;width:auto;padding-right:5px}#onetrust-pc-sdk #ot-lst-cnt{overflow-y:auto;margin-left:20px;margin-right:7px;width:calc(100% - 27px);max-height:calc(100% - 80px);height:100%;transform:translate3d(0, 0, 0)}#onetrust-pc-sdk #ot-pc-lst{width:100%;bottom:100px;position:absolute;top:60px}#onetrust-pc-sdk #ot-pc-lst:not(.ot-enbl-chr) .ot-tgl-cntr .ot-arw-cntr,#onetrust-pc-sdk #ot-pc-lst:not(.ot-enbl-chr) .ot-tgl-cntr .ot-arw-cntr *{visibility:hidden}#onetrust-pc-sdk #ot-pc-lst .ot-tgl-cntr{right:12px;position:absolute}#onetrust-pc-sdk #ot-pc-lst .ot-arw-cntr{float:right;position:relative}#onetrust-pc-sdk #ot-pc-lst .ot-arw{margin-left:10px}#onetrust-pc-sdk #ot-pc-lst .ot-acc-hdr{overflow:hidden;cursor:pointer}#onetrust-pc-sdk .ot-vlst-cntr{overflow:hidden}#onetrust-pc-sdk #ot-sel-blk{overflow:hidden;width:100%;position:sticky;position:-webkit-sticky;top:0;z-index:3}#onetrust-pc-sdk #ot-back-arw{height:12px;width:12px}#onetrust-pc-sdk .ot-lst-subhdr{width:100%;display:inline-block}#onetrust-pc-sdk .ot-search-cntr{float:left;width:78%;position:relative}#onetrust-pc-sdk .ot-search-cntr>svg{width:30px;height:30px;position:absolute;float:left;right:-15px}#onetrust-pc-sdk .ot-fltr-cntr{float:right;right:50px;position:relative}#onetrust-pc-sdk #filter-btn-handler{background-color:#3860be;border-radius:17px;display:inline-block;position:relative;width:32px;height:32px;-moz-transition:.1s ease;-o-transition:.1s ease;-webkit-transition:1s ease;transition:.1s ease;padding:0;margin:0}#onetrust-pc-sdk #filter-btn-handler:hover{background-color:#3860be}#onetrust-pc-sdk #filter-btn-handler svg{width:12px;height:12px;margin:3px 10px 0 10px;display:block;position:static;right:auto;top:auto}#onetrust-pc-sdk .ot-ven-link,#onetrust-pc-sdk .ot-ven-legclaim-link{color:#3860be;text-decoration:none;font-weight:100;display:inline-block;padding-top:10px;transform:translate(0, 1%);-o-transform:translate(0, 1%);-ms-transform:translate(0, 1%);-webkit-transform:translate(0, 1%);position:relative;z-index:2}#onetrust-pc-sdk .ot-ven-link *,#onetrust-pc-sdk .ot-ven-legclaim-link *{font-size:inherit}#onetrust-pc-sdk .ot-ven-link:hover,#onetrust-pc-sdk .ot-ven-legclaim-link:hover{text-decoration:underline}#onetrust-pc-sdk .ot-ven-hdr{width:calc(100% - 160px);height:auto;float:left;word-break:break-word;word-wrap:break-word;vertical-align:middle;padding-bottom:3px}#onetrust-pc-sdk .ot-ven-link,#onetrust-pc-sdk .ot-ven-legclaim-link{letter-spacing:.03em;font-size:.75em;font-weight:400}#onetrust-pc-sdk .ot-ven-dets{border-radius:2px;background-color:#f8f8f8}#onetrust-pc-sdk .ot-ven-dets li:first-child p:first-child{border-top:none}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:not(:first-child){border-top:1px solid #ddd !important}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:nth-child(n+3) p{display:inline-block}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:nth-child(n+3) p:nth-of-type(odd){width:30%}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:nth-child(n+3) p:nth-of-type(even){width:50%;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc p,#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc h5{padding-top:5px;padding-bottom:5px;display:block}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc h5{display:inline-block}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc p:nth-last-child(-n+1){padding-bottom:10px}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc p:nth-child(-n+2):not(.disc-pur){padding-top:10px}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc .disc-pur-cont{display:inline}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc .disc-pur{position:relative;width:50% !important;word-break:break-word;word-wrap:break-word;left:calc(30% + 17px)}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc .disc-pur:nth-child(-n+1){position:static}#onetrust-pc-sdk .ot-ven-dets p,#onetrust-pc-sdk .ot-ven-dets h5,#onetrust-pc-sdk .ot-ven-dets span{font-size:.69em;text-align:left;vertical-align:middle;word-break:break-word;word-wrap:break-word;margin:0;padding-bottom:10px;padding-left:15px;color:#2e3644}#onetrust-pc-sdk .ot-ven-dets h5{padding-top:5px}#onetrust-pc-sdk .ot-ven-dets span{color:dimgray;padding:0;vertical-align:baseline}#onetrust-pc-sdk .ot-ven-dets .ot-ven-pur h5{border-top:1px solid #e9e9e9;border-bottom:1px solid #e9e9e9;padding-bottom:5px;margin-bottom:5px;font-weight:bold}#onetrust-pc-sdk #ot-host-lst .ot-sel-all{float:right;position:relative;margin-right:42px;top:10px}#onetrust-pc-sdk #ot-host-lst .ot-sel-all input[type=checkbox]{width:auto;height:auto}#onetrust-pc-sdk #ot-host-lst .ot-sel-all label{height:20px;width:20px;padding-left:0px}#onetrust-pc-sdk #ot-host-lst .ot-acc-txt{overflow:hidden;width:95%}#onetrust-pc-sdk .ot-host-hdr{position:relative;z-index:1;pointer-events:none;width:calc(100% - 125px);float:left}#onetrust-pc-sdk .ot-host-name,#onetrust-pc-sdk .ot-host-desc{display:inline-block;width:90%}#onetrust-pc-sdk .ot-host-name{pointer-events:none}#onetrust-pc-sdk .ot-host-hdr>a{text-decoration:underline;font-size:.82em;position:relative;z-index:2;float:left;margin-bottom:5px;pointer-events:initial}#onetrust-pc-sdk .ot-host-name+a{margin-top:5px}#onetrust-pc-sdk .ot-host-name,#onetrust-pc-sdk .ot-host-name a,#onetrust-pc-sdk .ot-host-desc,#onetrust-pc-sdk .ot-host-info{color:dimgray;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-host-name,#onetrust-pc-sdk .ot-host-name a{font-weight:bold;font-size:.82em;line-height:1.3}#onetrust-pc-sdk .ot-host-name a{font-size:1em}#onetrust-pc-sdk .ot-host-expand{margin-top:3px;margin-bottom:3px;clear:both;display:block;color:#3860be;font-size:.72em;font-weight:normal}#onetrust-pc-sdk .ot-host-expand *{font-size:inherit}#onetrust-pc-sdk .ot-host-desc,#onetrust-pc-sdk .ot-host-info{font-size:.688em;line-height:1.4;font-weight:normal}#onetrust-pc-sdk .ot-host-desc{margin-top:10px}#onetrust-pc-sdk .ot-host-opt{margin:0;font-size:inherit;display:inline-block;width:100%}#onetrust-pc-sdk .ot-host-opt li>div div{font-size:.8em;padding:5px 0}#onetrust-pc-sdk .ot-host-opt li>div div:nth-child(1){width:30%;float:left}#onetrust-pc-sdk .ot-host-opt li>div div:nth-child(2){width:70%;float:left;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-host-info{border:none;display:inline-block;width:calc(100% - 10px);padding:10px;margin-bottom:10px;background-color:#f8f8f8}#onetrust-pc-sdk .ot-host-info>div{overflow:auto}#onetrust-pc-sdk #no-results{text-align:center;margin-top:30px}#onetrust-pc-sdk #no-results p{font-size:1em;color:#2e3644;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk #no-results p span{font-weight:bold}#onetrust-pc-sdk #ot-fltr-modal{width:100%;height:auto;display:none;-moz-transition:.2s ease;-o-transition:.2s ease;-webkit-transition:2s ease;transition:.2s ease;overflow:hidden;opacity:1;right:0}#onetrust-pc-sdk #ot-fltr-modal .ot-label-txt{display:inline-block;font-size:.85em;color:dimgray}#onetrust-pc-sdk #ot-fltr-cnt{z-index:2147483646;background-color:#fff;position:absolute;height:90%;max-height:300px;width:325px;left:210px;margin-top:10px;margin-bottom:20px;padding-right:10px;border-radius:3px;-webkit-box-shadow:0px 0px 12px 2px #c7c5c7;-moz-box-shadow:0px 0px 12px 2px #c7c5c7;box-shadow:0px 0px 12px 2px #c7c5c7}#onetrust-pc-sdk .ot-fltr-scrlcnt{overflow-y:auto;overflow-x:hidden;clear:both;max-height:calc(100% - 60px)}#onetrust-pc-sdk #ot-anchor{border:12px solid rgba(0,0,0,0);display:none;position:absolute;z-index:2147483647;right:55px;top:75px;transform:rotate(45deg);-o-transform:rotate(45deg);-ms-transform:rotate(45deg);-webkit-transform:rotate(45deg);background-color:#fff;-webkit-box-shadow:-3px -3px 5px -2px #c7c5c7;-moz-box-shadow:-3px -3px 5px -2px #c7c5c7;box-shadow:-3px -3px 5px -2px #c7c5c7}#onetrust-pc-sdk .ot-fltr-btns{margin-left:15px}#onetrust-pc-sdk #filter-apply-handler{margin-right:15px}#onetrust-pc-sdk .ot-fltr-opt{margin-bottom:5px;margin-left:15px;min-height:20px;width:75%;position:relative}#onetrust-pc-sdk .ot-fltr-opt p{display:inline-block;margin:0;font-size:.9em;color:#2e3644}#onetrust-pc-sdk .ot-chkbox label span{font-size:.85em;color:dimgray}#onetrust-pc-sdk .ot-chkbox input[type=checkbox]+label::after{content:none;color:#fff}#onetrust-pc-sdk .ot-chkbox input[type=checkbox]:checked+label::after{content:""}#onetrust-pc-sdk .ot-chkbox input[type=checkbox][aria-checked=mixed]+label::after{content:""}#onetrust-pc-sdk .ot-chkbox input[type=checkbox]:focus+label::before{outline-style:solid;outline-width:2px;outline-style:auto}#onetrust-pc-sdk #ot-selall-vencntr,#onetrust-pc-sdk #ot-selall-adtlvencntr,#onetrust-pc-sdk #ot-selall-hostcntr,#onetrust-pc-sdk #ot-selall-licntr,#onetrust-pc-sdk #ot-selall-gnvencntr{right:15px;position:relative;width:20px;height:20px;float:right}#onetrust-pc-sdk #ot-selall-vencntr label,#onetrust-pc-sdk #ot-selall-adtlvencntr label,#onetrust-pc-sdk #ot-selall-hostcntr label,#onetrust-pc-sdk #ot-selall-licntr label,#onetrust-pc-sdk #ot-selall-gnvencntr label{float:left;padding-left:0}#onetrust-pc-sdk #ot-ven-lst:first-child{border-top:1px solid #e2e2e2}#onetrust-pc-sdk ul{list-style:none;padding:0}#onetrust-pc-sdk ul li{position:relative;margin:0;padding:15px 15px 15px 10px;border-bottom:1px solid #e2e2e2}#onetrust-pc-sdk ul li h3,#onetrust-pc-sdk ul li h4{font-size:.75em;color:#656565;margin:0;display:inline-block;width:70%;height:auto;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk ul li p{margin:0;font-size:.7em}#onetrust-pc-sdk ul li input[type=checkbox]{position:absolute;cursor:pointer;width:100%;height:100%;opacity:0;margin:0;top:0;left:0}#onetrust-pc-sdk .ot-cat-item>button:focus,#onetrust-pc-sdk .ot-acc-cntr>button:focus,#onetrust-pc-sdk li>button:focus{outline:#000 solid 2px}#onetrust-pc-sdk .ot-cat-item>button,#onetrust-pc-sdk .ot-acc-cntr>button,#onetrust-pc-sdk li>button{position:absolute;cursor:pointer;width:100%;height:100%;margin:0;top:0;left:0;z-index:1;max-width:none;border:none}#onetrust-pc-sdk .ot-cat-item>button[aria-expanded=false]~.ot-acc-txt,#onetrust-pc-sdk .ot-acc-cntr>button[aria-expanded=false]~.ot-acc-txt,#onetrust-pc-sdk li>button[aria-expanded=false]~.ot-acc-txt{margin-top:0;max-height:0;opacity:0;overflow:hidden;width:100%;transition:.25s ease-out;display:none}#onetrust-pc-sdk .ot-cat-item>button[aria-expanded=true]~.ot-acc-txt,#onetrust-pc-sdk .ot-acc-cntr>button[aria-expanded=true]~.ot-acc-txt,#onetrust-pc-sdk li>button[aria-expanded=true]~.ot-acc-txt{transition:.1s ease-in;margin-top:10px;width:100%;overflow:auto;display:block}#onetrust-pc-sdk .ot-cat-item>button[aria-expanded=true]~.ot-acc-grpcntr,#onetrust-pc-sdk .ot-acc-cntr>button[aria-expanded=true]~.ot-acc-grpcntr,#onetrust-pc-sdk li>button[aria-expanded=true]~.ot-acc-grpcntr{width:auto;margin-top:0px;padding-bottom:10px}#onetrust-pc-sdk .ot-host-item>button:focus,#onetrust-pc-sdk .ot-ven-item>button:focus{outline:0;border:2px solid #000}#onetrust-pc-sdk .ot-hide-acc>button{pointer-events:none}#onetrust-pc-sdk .ot-hide-acc .ot-plus-minus>*,#onetrust-pc-sdk .ot-hide-acc .ot-arw-cntr>*{visibility:hidden}#onetrust-pc-sdk .ot-hide-acc .ot-acc-hdr{min-height:30px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt){padding-right:10px;width:calc(100% - 37px);margin-top:10px;max-height:calc(100% - 90px)}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk{background-color:#f9f9fc;border:1px solid #e2e2e2;width:calc(100% - 2px);padding-bottom:5px;padding-top:5px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt{border:unset;background-color:unset}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt .ot-sel-all-hdr{display:none}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt .ot-sel-all{padding-right:.5rem}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt .ot-sel-all .ot-chkbox{right:0}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) .ot-sel-all{padding-right:34px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) .ot-sel-all-chkbox{width:auto}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) ul li{border:1px solid #e2e2e2;margin-bottom:10px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) .ot-acc-cntr>.ot-acc-hdr{padding:10px 0 10px 15px}#onetrust-pc-sdk.ot-addtl-vendors .ot-sel-all-chkbox{float:right}#onetrust-pc-sdk.ot-addtl-vendors .ot-plus-minus~.ot-sel-all-chkbox{right:34px}#onetrust-pc-sdk.ot-addtl-vendors #ot-ven-lst:first-child{border-top:none}#onetrust-pc-sdk .ot-acc-cntr{position:relative;border-left:1px solid #e2e2e2;border-right:1px solid #e2e2e2;border-bottom:1px solid #e2e2e2}#onetrust-pc-sdk .ot-acc-cntr input{z-index:1}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr{background-color:#f9f9fc;padding:5px 0 5px 15px;width:auto}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr .ot-plus-minus{vertical-align:middle;top:auto}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr .ot-arw-cntr{right:10px}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr input{z-index:2}#onetrust-pc-sdk .ot-acc-cntr.ot-add-tech .ot-acc-hdr{padding:10px 0 10px 15px}#onetrust-pc-sdk .ot-acc-cntr>input[type=checkbox]:checked~.ot-acc-hdr{border-bottom:1px solid #e2e2e2}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-txt{padding-left:10px;padding-right:10px}#onetrust-pc-sdk .ot-acc-cntr button[aria-expanded=true]~.ot-acc-txt{width:auto}#onetrust-pc-sdk .ot-acc-cntr .ot-addtl-venbox{display:none}#onetrust-pc-sdk .ot-vlst-cntr{margin-bottom:0;width:100%}#onetrust-pc-sdk .ot-vensec-title{font-size:.813em;vertical-align:middle;display:inline-block}#onetrust-pc-sdk .category-vendors-list-handler,#onetrust-pc-sdk .category-vendors-list-handler+a{margin-left:0;margin-top:10px}#onetrust-pc-sdk #ot-selall-vencntr.line-through label::after,#onetrust-pc-sdk #ot-selall-adtlvencntr.line-through label::after,#onetrust-pc-sdk #ot-selall-licntr.line-through label::after,#onetrust-pc-sdk #ot-selall-hostcntr.line-through label::after,#onetrust-pc-sdk #ot-selall-gnvencntr.line-through label::after{height:auto;border-left:0;transform:none;-o-transform:none;-ms-transform:none;-webkit-transform:none;left:5px;top:9px}#onetrust-pc-sdk #ot-category-title{float:left;padding-bottom:10px;font-size:1em;width:100%}#onetrust-pc-sdk .ot-cat-grp{margin-top:10px}#onetrust-pc-sdk .ot-cat-item{line-height:1.1;margin-top:10px;display:inline-block;width:100%}#onetrust-pc-sdk .ot-btn-container{text-align:right}#onetrust-pc-sdk .ot-btn-container button{display:inline-block;font-size:.75em;letter-spacing:.08em;margin-top:19px}#onetrust-pc-sdk #close-pc-btn-handler.ot-close-icon{position:absolute;top:10px;right:0;z-index:1;padding:0;background-color:rgba(0,0,0,0);border:none}#onetrust-pc-sdk #close-pc-btn-handler.ot-close-icon svg{display:block;height:10px;width:10px}#onetrust-pc-sdk #clear-filters-handler{margin-top:20px;margin-bottom:10px;float:right;max-width:200px;text-decoration:none;color:#3860be;font-size:.9em;font-weight:bold;background-color:rgba(0,0,0,0);border-color:rgba(0,0,0,0);padding:1px}#onetrust-pc-sdk #clear-filters-handler:hover{color:#2285f7}#onetrust-pc-sdk #clear-filters-handler:focus{outline:#000 solid 1px}#onetrust-pc-sdk .ot-enbl-chr h4~.ot-tgl,#onetrust-pc-sdk .ot-enbl-chr h4~.ot-always-active{right:45px}#onetrust-pc-sdk .ot-enbl-chr h4~.ot-tgl+.ot-tgl{right:120px}#onetrust-pc-sdk .ot-enbl-chr .ot-pli-hdr.ot-leg-border-color span:first-child{width:90px}#onetrust-pc-sdk .ot-enbl-chr li.ot-subgrp>h5+.ot-tgl-cntr{padding-right:25px}#onetrust-pc-sdk .ot-plus-minus{width:20px;height:20px;font-size:1.5em;position:relative;display:inline-block;margin-right:5px;top:3px}#onetrust-pc-sdk .ot-plus-minus span{position:absolute;background:#27455c;border-radius:1px}#onetrust-pc-sdk .ot-plus-minus span:first-of-type{top:25%;bottom:25%;width:10%;left:45%}#onetrust-pc-sdk .ot-plus-minus span:last-of-type{left:25%;right:25%;height:10%;top:45%}#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-arw,#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-plus-minus span:first-of-type,#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-plus-minus span:last-of-type{transform:rotate(90deg)}#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-plus-minus span:last-of-type{left:50%;right:50%}#onetrust-pc-sdk #ot-selall-vencntr label,#onetrust-pc-sdk #ot-selall-adtlvencntr label,#onetrust-pc-sdk #ot-selall-hostcntr label,#onetrust-pc-sdk #ot-selall-licntr label{position:relative;display:inline-block;width:20px;height:20px}#onetrust-pc-sdk .ot-host-item .ot-plus-minus,#onetrust-pc-sdk .ot-ven-item .ot-plus-minus{float:left;margin-right:8px;top:10px}#onetrust-pc-sdk .ot-ven-item ul{list-style:none inside;font-size:100%;margin:0}#onetrust-pc-sdk .ot-ven-item ul li{margin:0 !important;padding:0;border:none !important}#onetrust-pc-sdk .ot-pli-hdr{color:#77808e;overflow:hidden;padding-top:7.5px;padding-bottom:7.5px;width:calc(100% - 2px);border-top-left-radius:3px;border-top-right-radius:3px}#onetrust-pc-sdk .ot-pli-hdr span:first-child{top:50%;transform:translateY(50%);max-width:90px}#onetrust-pc-sdk .ot-pli-hdr span:last-child{padding-right:10px;max-width:95px;text-align:center}#onetrust-pc-sdk .ot-li-title{float:right;font-size:.813em}#onetrust-pc-sdk .ot-pli-hdr.ot-leg-border-color{background-color:#f4f4f4;border:1px solid #d8d8d8}#onetrust-pc-sdk .ot-pli-hdr.ot-leg-border-color span:first-child{text-align:left;width:70px}#onetrust-pc-sdk li.ot-subgrp>h5,#onetrust-pc-sdk .ot-cat-header{width:calc(100% - 130px)}#onetrust-pc-sdk li.ot-subgrp>h5+.ot-tgl-cntr{padding-left:13px}#onetrust-pc-sdk .ot-acc-grpcntr .ot-acc-grpdesc{margin-bottom:5px}#onetrust-pc-sdk .ot-acc-grpcntr .ot-subgrp-cntr{border-top:1px solid #d8d8d8}#onetrust-pc-sdk .ot-acc-grpcntr .ot-vlst-cntr+.ot-subgrp-cntr{border-top:none}#onetrust-pc-sdk .ot-acc-hdr .ot-arw-cntr+.ot-tgl-cntr,#onetrust-pc-sdk .ot-acc-txt h4+.ot-tgl-cntr{padding-left:13px}#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item .ot-subgrp>h5,#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item .ot-cat-header{width:calc(100% - 145px)}#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item h5+.ot-tgl-cntr,#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item .ot-cat-header+.ot-tgl{padding-left:28px}#onetrust-pc-sdk .ot-sel-all-hdr,#onetrust-pc-sdk .ot-sel-all-chkbox{display:inline-block;width:100%;position:relative}#onetrust-pc-sdk .ot-sel-all-chkbox{z-index:1}#onetrust-pc-sdk .ot-sel-all{margin:0;position:relative;padding-right:23px;float:right}#onetrust-pc-sdk .ot-consent-hdr,#onetrust-pc-sdk .ot-li-hdr{float:right;font-size:.812em;line-height:normal;text-align:center;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-li-hdr{max-width:100px;padding-right:10px}#onetrust-pc-sdk .ot-consent-hdr{max-width:55px}#onetrust-pc-sdk #ot-selall-licntr{display:block;width:21px;height:auto;float:right;position:relative;right:80px}#onetrust-pc-sdk #ot-selall-licntr label{position:absolute}#onetrust-pc-sdk .ot-ven-ctgl{margin-left:66px}#onetrust-pc-sdk .ot-ven-litgl+.ot-arw-cntr{margin-left:81px}#onetrust-pc-sdk .ot-enbl-chr .ot-host-cnt .ot-tgl-cntr{width:auto}#onetrust-pc-sdk #ot-lst-cnt:not(.ot-host-cnt) .ot-tgl-cntr{width:auto;top:auto;height:20px}#onetrust-pc-sdk #ot-lst-cnt .ot-chkbox{position:relative;display:inline-block;width:20px;height:20px}#onetrust-pc-sdk #ot-lst-cnt .ot-chkbox label{position:absolute;padding:0;width:20px;height:20px}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info-cntr{border:1px solid #d8d8d8;padding:.75rem 2rem;padding-bottom:0;width:auto;margin-top:.5rem}#onetrust-pc-sdk .ot-acc-grpdesc+.ot-leg-btn-container{padding-left:20px;padding-right:20px;width:calc(100% - 40px);margin-bottom:5px}#onetrust-pc-sdk .ot-subgrp .ot-leg-btn-container{margin-bottom:5px}#onetrust-pc-sdk #ot-ven-lst .ot-leg-btn-container{margin-top:10px}#onetrust-pc-sdk .ot-leg-btn-container{display:inline-block;width:100%;margin-bottom:10px}#onetrust-pc-sdk .ot-leg-btn-container button{height:auto;padding:6.5px 8px;margin-bottom:0;letter-spacing:0;font-size:.75em;line-height:normal}#onetrust-pc-sdk .ot-leg-btn-container svg{display:none;height:14px;width:14px;padding-right:5px;vertical-align:sub}#onetrust-pc-sdk .ot-active-leg-btn{cursor:default;pointer-events:none}#onetrust-pc-sdk .ot-active-leg-btn svg{display:inline-block}#onetrust-pc-sdk .ot-remove-objection-handler{text-decoration:underline;padding:0;font-size:.75em;font-weight:600;line-height:1;padding-left:10px}#onetrust-pc-sdk .ot-obj-leg-btn-handler span{font-weight:bold;text-align:center;font-size:inherit;line-height:1.5}#onetrust-pc-sdk.ot-close-btn-link #close-pc-btn-handler{border:none;height:auto;line-height:1.5;text-decoration:underline;font-size:.69em;background:none;right:15px;top:15px;width:auto;font-weight:normal}#onetrust-pc-sdk .ot-pgph-link{font-size:.813em !important;margin-top:5px;position:relative}#onetrust-pc-sdk .ot-pgph-link.ot-pgph-link-subgroup{margin-bottom:1rem}#onetrust-pc-sdk .ot-pgph-contr{margin:0 2.5rem}#onetrust-pc-sdk .ot-pgph-title{font-size:1.18rem;margin-bottom:2rem}#onetrust-pc-sdk .ot-pgph-desc{font-size:1rem;font-weight:400;margin-bottom:2rem;line-height:1.5rem}#onetrust-pc-sdk .ot-pgph-desc:not(:last-child):after{content:"";width:96%;display:block;margin:0 auto;padding-bottom:2rem;border-bottom:1px solid #e9e9e9}#onetrust-pc-sdk .ot-cat-header{float:left;font-weight:600;font-size:.875em;line-height:1.5;max-width:90%;vertical-align:middle}#onetrust-pc-sdk .ot-vnd-item>button:focus{outline:#000 solid 2px}#onetrust-pc-sdk .ot-vnd-item>button{position:absolute;cursor:pointer;width:100%;height:100%;margin:0;top:0;left:0;z-index:1;max-width:none;border:none}#onetrust-pc-sdk .ot-vnd-item>button[aria-expanded=false]~.ot-acc-txt{margin-top:0;max-height:0;opacity:0;overflow:hidden;width:100%;transition:.25s ease-out;display:none}#onetrust-pc-sdk .ot-vnd-item>button[aria-expanded=true]~.ot-acc-txt{transition:.1s ease-in;margin-top:10px;width:100%;overflow:auto;display:block}#onetrust-pc-sdk .ot-vnd-item>button[aria-expanded=true]~.ot-acc-grpcntr{width:auto;margin-top:0px;padding-bottom:10px}#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item{position:relative;border-radius:2px;margin:0;padding:0;border:1px solid #d8d8d8;border-top:none;width:calc(100% - 2px);float:left}#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item:first-of-type{margin-top:10px;border-top:1px solid #d8d8d8}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpdesc{padding-left:20px;padding-right:20px;width:calc(100% - 40px);font-size:.812em;margin-bottom:10px;margin-top:15px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpdesc>ul{padding-top:10px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpdesc>ul li{padding-top:0;line-height:1.5;padding-bottom:10px}#onetrust-pc-sdk .ot-accordion-layout div+.ot-acc-grpdesc{margin-top:5px}#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr:first-child{margin-top:10px}#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr:last-child,#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr:last-child{margin-bottom:5px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{padding-top:11.5px;padding-bottom:11.5px;padding-left:20px;padding-right:20px;width:calc(100% - 40px);display:inline-block}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-txt{width:100%;padding:0}#onetrust-pc-sdk .ot-accordion-layout .ot-subgrp-cntr{padding-left:20px;padding-right:15px;padding-bottom:0;width:calc(100% - 35px)}#onetrust-pc-sdk .ot-accordion-layout .ot-subgrp{padding-right:5px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpcntr{z-index:1;position:relative}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header+.ot-arw-cntr{position:absolute;top:50%;transform:translateY(-50%);right:20px;margin-top:-2px}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header+.ot-arw-cntr .ot-arw{width:15px;height:20px;margin-left:5px;color:dimgray}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{float:none;color:#2e3644;margin:0;display:inline-block;height:auto;word-wrap:break-word;min-height:inherit}#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr,#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr{padding-left:20px;width:calc(100% - 20px);display:inline-block;margin-top:0;padding-bottom:2px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{position:relative;min-height:25px}#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl,#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active{position:absolute;top:50%;transform:translateY(-50%);right:20px}#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl+.ot-tgl{right:95px}#onetrust-pc-sdk .ot-accordion-layout .category-vendors-list-handler,#onetrust-pc-sdk .ot-accordion-layout .category-vendors-list-handler+a{margin-top:5px}#onetrust-pc-sdk #ot-lst-cnt{margin-top:1rem;max-height:calc(100% - 96px)}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info-cntr{border:1px solid #d8d8d8;padding:.75rem 2rem;padding-bottom:0;width:auto;margin-top:.5rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info{margin-bottom:1rem;padding-left:.75rem;padding-right:.75rem;display:flex;flex-direction:column}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info[data-vnd-info-key*=DPOEmail]{border-top:1px solid #d8d8d8;padding-top:1rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info[data-vnd-info-key*=DPOLink]{border-bottom:1px solid #d8d8d8;padding-bottom:1rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info .ot-vnd-lbl{font-weight:bold;font-size:.85em;margin-bottom:.5rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info .ot-vnd-cnt{margin-left:.5rem;font-weight:500;font-size:.85rem}#onetrust-pc-sdk .ot-vs-list,#onetrust-pc-sdk .ot-vnd-serv{width:auto;padding:1rem 1.25rem;padding-bottom:0}#onetrust-pc-sdk .ot-vs-list .ot-vnd-serv-hdr-cntr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-serv-hdr-cntr{padding-bottom:.75rem;border-bottom:1px solid #d8d8d8}#onetrust-pc-sdk .ot-vs-list .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr{font-weight:600;font-size:.95em;line-height:2;margin-left:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item{border:none;margin:0;padding:0}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item button,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item button{outline:none;border-bottom:1px solid #d8d8d8}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item button[aria-expanded=true],#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item button[aria-expanded=true]{border-bottom:none}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item:first-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item:first-child{margin-top:.25rem;border-top:unset}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item:last-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item:last-child{margin-bottom:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item:last-child button,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item:last-child button{border-bottom:none}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info-cntr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info-cntr{border:1px solid #d8d8d8;padding:.75rem 1.75rem;padding-bottom:0;width:auto;margin-top:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info{margin-bottom:1rem;padding-left:.75rem;padding-right:.75rem;display:flex;flex-direction:column}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOEmail],#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOEmail]{border-top:1px solid #d8d8d8;padding-top:1rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOLink],#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOLink]{border-bottom:1px solid #d8d8d8;padding-bottom:1rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info .ot-vnd-lbl,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info .ot-vnd-lbl{font-weight:bold;font-size:.85em;margin-bottom:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info .ot-vnd-cnt,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info .ot-vnd-cnt{margin-left:.5rem;font-weight:500;font-size:.85rem}#onetrust-pc-sdk .ot-vs-list.ot-vnd-subgrp-cnt,#onetrust-pc-sdk .ot-vnd-serv.ot-vnd-subgrp-cnt{padding-left:40px}#onetrust-pc-sdk .ot-vs-list.ot-vnd-subgrp-cnt .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr,#onetrust-pc-sdk .ot-vnd-serv.ot-vnd-subgrp-cnt .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr{font-size:.8em}#onetrust-pc-sdk .ot-vs-list.ot-vnd-subgrp-cnt .ot-cat-header,#onetrust-pc-sdk .ot-vnd-serv.ot-vnd-subgrp-cnt .ot-cat-header{font-size:.8em}#onetrust-pc-sdk .ot-subgrp-cntr .ot-vnd-serv{margin-bottom:1rem;padding:1rem .95rem}#onetrust-pc-sdk .ot-subgrp-cntr .ot-vnd-serv .ot-vnd-serv-hdr-cntr{padding-bottom:.75rem;border-bottom:1px solid #d8d8d8}#onetrust-pc-sdk .ot-subgrp-cntr .ot-vnd-serv .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr{font-weight:700;font-size:.8em;line-height:20px;margin-left:.82rem}#onetrust-pc-sdk .ot-subgrp-cntr .ot-cat-header{font-weight:700;font-size:.8em;line-height:20px}#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-vnd-serv .ot-vnd-lst-cont .ot-accordion-layout .ot-acc-hdr div.ot-chkbox{margin-left:.82rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr{padding:.7rem 0;margin:0;display:flex;width:100%;align-items:center;justify-content:space-between}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr div:first-child,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr div:first-child,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr div:first-child{margin-left:.5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr div:last-child,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr div:last-child,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr div:last-child{margin-right:.5rem;margin-left:.5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-always-active{position:relative;right:unset;top:unset;transform:unset}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-plus-minus{top:0}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-arw-cntr{float:none;top:unset;right:unset;transform:unset;margin-top:-2px;position:relative}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-cat-header{flex:1;margin:0 .5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-tgl{position:relative;transform:none;right:0;top:0;float:none}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox{position:relative;margin:0 .5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox label{padding:0}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox label::before{position:relative}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox input{position:absolute;cursor:pointer;width:100%;height:100%;opacity:0;margin:0;top:0;left:0;z-index:1}#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp .ot-acc-hdr h5.ot-cat-header,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp .ot-acc-hdr h4.ot-cat-header{margin:0}#onetrust-pc-sdk .ot-vs-config .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp h5{top:0;line-height:20px}#onetrust-pc-sdk .ot-vs-list{display:flex;flex-direction:column;padding:0;margin:.5rem 4px}#onetrust-pc-sdk .ot-vs-selc-all{display:flex;padding:0;float:unset;align-items:center;justify-content:flex-start}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf{justify-content:flex-end}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf.ot-caret-conf .ot-sel-all-chkbox{margin-right:48px}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf .ot-sel-all-chkbox{margin:0;padding:0;margin-right:14px;justify-content:flex-end}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf #ot-selall-vencntr.ot-chkbox,#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf #ot-selall-vencntr.ot-tgl{display:inline-block;right:unset;width:auto;height:auto;float:none}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf #ot-selall-vencntr label{width:45px;height:25px}#onetrust-pc-sdk .ot-vs-selc-all .ot-sel-all-chkbox{margin-right:11px;margin-left:.75rem;display:flex;align-items:center}#onetrust-pc-sdk .ot-vs-selc-all .sel-all-hdr{margin:0 1.25rem;font-size:.812em;line-height:normal;text-align:center;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-vnd-list-cnt #ot-selall-vencntr.ot-chkbox{float:unset;right:0}#onetrust-pc-sdk[dir=rtl] #ot-back-arw,#onetrust-pc-sdk[dir=rtl] input~.ot-acc-hdr .ot-arw{transform:rotate(180deg);-o-transform:rotate(180deg);-ms-transform:rotate(180deg);-webkit-transform:rotate(180deg)}#onetrust-pc-sdk[dir=rtl] input:checked~.ot-acc-hdr .ot-arw{transform:rotate(270deg);-o-transform:rotate(270deg);-ms-transform:rotate(270deg);-webkit-transform:rotate(270deg)}#onetrust-pc-sdk[dir=rtl] .ot-chkbox label::after{transform:rotate(45deg);-webkit-transform:rotate(45deg);-o-transform:rotate(45deg);-ms-transform:rotate(45deg);border-left:0;border-right:3px solid}#onetrust-pc-sdk[dir=rtl] .ot-search-cntr>svg{right:0}@media only screen and (max-width: 600px){#onetrust-pc-sdk.otPcCenter{left:0;min-width:100%;height:100%;top:0;border-radius:0}#onetrust-pc-sdk #ot-pc-content,#onetrust-pc-sdk.ot-ftr-stacked .ot-btn-container{margin:1px 3px 0 10px;padding-right:10px;width:calc(100% - 23px)}#onetrust-pc-sdk .ot-btn-container button{max-width:none;letter-spacing:.01em}#onetrust-pc-sdk #close-pc-btn-handler{top:10px;right:17px}#onetrust-pc-sdk p{font-size:.7em}#onetrust-pc-sdk #ot-pc-hdr{margin:10px 10px 0 5px;width:calc(100% - 15px)}#onetrust-pc-sdk .vendor-search-handler{font-size:1em}#onetrust-pc-sdk #ot-back-arw{margin-left:12px}#onetrust-pc-sdk #ot-lst-cnt{margin:0;padding:0 5px 0 10px;min-width:95%}#onetrust-pc-sdk .switch+p{max-width:80%}#onetrust-pc-sdk .ot-ftr-stacked button{width:100%}#onetrust-pc-sdk #ot-fltr-cnt{max-width:320px;width:90%;border-top-right-radius:0;border-bottom-right-radius:0;margin:0;margin-left:15px;left:auto;right:40px;top:85px}#onetrust-pc-sdk .ot-fltr-opt{margin-left:25px;margin-bottom:10px}#onetrust-pc-sdk .ot-pc-refuse-all-handler{margin-bottom:0}#onetrust-pc-sdk #ot-fltr-cnt{right:40px}}@media only screen and (max-width: 476px){#onetrust-pc-sdk .ot-fltr-cntr,#onetrust-pc-sdk #ot-fltr-cnt{right:10px}#onetrust-pc-sdk #ot-anchor{right:25px}#onetrust-pc-sdk button{width:100%}#onetrust-pc-sdk:not(.ot-addtl-vendors) #ot-pc-lst:not(.ot-enbl-chr) .ot-sel-all{padding-right:9px}#onetrust-pc-sdk:not(.ot-addtl-vendors) #ot-pc-lst:not(.ot-enbl-chr) .ot-tgl-cntr{right:0}}@media(min-width: 768px){#onetrust-pc-sdk.ot-tgl-with-label .ot-label-status{display:inline}#onetrust-pc-sdk.ot-tgl-with-label #ot-pc-lst .ot-label-status{display:none}}@media only screen and (max-width: 896px)and (max-height: 425px)and (orientation: landscape){#onetrust-pc-sdk.otPcCenter{left:0;top:0;min-width:100%;height:100%;border-radius:0}#onetrust-pc-sdk .ot-pc-header{height:auto;min-height:20px}#onetrust-pc-sdk .ot-pc-header .ot-pc-logo{max-height:30px}#onetrust-pc-sdk .ot-pc-footer{max-height:60px;overflow-y:auto}#onetrust-pc-sdk #ot-pc-content,#onetrust-pc-sdk #ot-pc-lst{bottom:70px}#onetrust-pc-sdk.ot-ftr-stacked #ot-pc-content{bottom:70px}#onetrust-pc-sdk #ot-anchor{left:initial;right:50px}#onetrust-pc-sdk #ot-lst-title{margin-top:12px}#onetrust-pc-sdk #ot-lst-title *{font-size:inherit}#onetrust-pc-sdk #ot-pc-hdr input{margin-right:0;padding-right:45px}#onetrust-pc-sdk .switch+p{max-width:85%}#onetrust-pc-sdk #ot-sel-blk{position:static}#onetrust-pc-sdk #ot-pc-lst{overflow:auto}#onetrust-pc-sdk #ot-lst-cnt{max-height:none;overflow:initial}#onetrust-pc-sdk #ot-lst-cnt.no-results{height:auto}#onetrust-pc-sdk input{font-size:1em !important}#onetrust-pc-sdk p{font-size:.6em}#onetrust-pc-sdk #ot-fltr-modal{width:100%;top:0}#onetrust-pc-sdk ul li p,#onetrust-pc-sdk .category-vendors-list-handler,#onetrust-pc-sdk .category-vendors-list-handler+a,#onetrust-pc-sdk .category-host-list-handler{font-size:.6em}#onetrust-pc-sdk.ot-shw-fltr #ot-anchor{display:none !important}#onetrust-pc-sdk.ot-shw-fltr #ot-pc-lst{height:100% !important;overflow:hidden;top:0px}#onetrust-pc-sdk.ot-shw-fltr #ot-fltr-cnt{margin:0;height:100%;max-height:none;padding:10px;top:0;width:calc(100% - 20px);position:absolute;right:0;left:0;max-width:none}#onetrust-pc-sdk.ot-shw-fltr .ot-fltr-scrlcnt{max-height:calc(100% - 65px)}}
            #onetrust-consent-sdk #onetrust-pc-sdk,
                #onetrust-consent-sdk #ot-search-cntr,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-switch.ot-toggle,
                #onetrust-consent-sdk #onetrust-pc-sdk ot-grp-hdr1 .checkbox,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-title:after
                ,#onetrust-consent-sdk #onetrust-pc-sdk #ot-sel-blk,
                        #onetrust-consent-sdk #onetrust-pc-sdk #ot-fltr-cnt,
                        #onetrust-consent-sdk #onetrust-pc-sdk #ot-anchor {
                    background-color: #FFF;
                }
               
            #onetrust-consent-sdk #onetrust-pc-sdk h3,
                #onetrust-consent-sdk #onetrust-pc-sdk h4,
                #onetrust-consent-sdk #onetrust-pc-sdk h5,
                #onetrust-consent-sdk #onetrust-pc-sdk h6,
                #onetrust-consent-sdk #onetrust-pc-sdk p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-ven-lst .ot-ven-opts p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-desc,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-title,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-li-title,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-sel-all-hdr span,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-info,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-fltr-modal #modal-header,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-checkbox label span,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst #ot-sel-blk p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst #ot-lst-title h3,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst .back-btn-handler p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst .ot-ven-name,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst #ot-ven-lst .consent-category,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-inactive-leg-btn,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-label-status,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-chkbox label span,
                #onetrust-consent-sdk #onetrust-pc-sdk #clear-filters-handler,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-optout-signal
                {
                    color: #2E2E2E;
                }
             #onetrust-consent-sdk #onetrust-pc-sdk .privacy-notice-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-pgph-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk .category-vendors-list-handler,
                    #onetrust-consent-sdk #onetrust-pc-sdk .category-vendors-list-handler + a,
                    #onetrust-consent-sdk #onetrust-pc-sdk .category-host-list-handler,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-ven-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-ven-legclaim-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-name a,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-acc-hdr .ot-host-expand,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-info a,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-content #ot-pc-desc .ot-link-btn,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info a,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info a
                    {
                        color: #007398;
                    }
            #onetrust-consent-sdk #onetrust-pc-sdk .category-vendors-list-handler:hover { text-decoration: underline;}
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-grpcntr.ot-acc-txt,
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-txt .ot-subgrp-tgl .ot-switch.ot-toggle
             {
                background-color: #F8F8F8;
            }
             #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-info,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-txt .ot-ven-dets
                            {
                                background-color: #F8F8F8;
                            }
        #onetrust-consent-sdk #onetrust-pc-sdk
            button:not(#clear-filters-handler):not(.ot-close-icon):not(#filter-btn-handler):not(.ot-remove-objection-handler):not(.ot-obj-leg-btn-handler):not([aria-expanded]):not(.ot-link-btn),
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-active-leg-btn {
                background-color: #007398;border-color: #007398;
                color: #FFF;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-active-menu {
                border-color: #007398;
            }
            
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-remove-objection-handler{
                background-color: transparent;
                border: 1px solid transparent;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-inactive-leg-btn {
                background-color: #FFFFFF;
                color: #4D4D4D; border-color: #4D4D4D;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-tgl input:focus + .ot-switch, .ot-switch .ot-switch-nob, .ot-switch .ot-switch-nob:before,
            #onetrust-pc-sdk .ot-checkbox input[type="checkbox"]:focus + label::before,
            #onetrust-pc-sdk .ot-chkbox input[type="checkbox"]:focus + label::before {
                outline-color: #000000;
                outline-width: 1px;
                outline-offset: 1px;
            }
            #onetrust-pc-sdk .ot-host-item > button:focus, #onetrust-pc-sdk .ot-ven-item > button:focus {
                border: 1px solid #000000;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk *:focus,
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-vlst-cntr > a:focus {
               outline: 1px solid #000000;
               outline-offset: 1px;
            }#onetrust-pc-sdk .ot-vlst-cntr .ot-ext-lnk,  #onetrust-pc-sdk .ot-ven-hdr .ot-ext-lnk{
                    background-image: url('https://cdn.cookielaw.org/logos/static/ot_external_link.svg');
                }
            /*! v6.10.0 2021-01-18 */
div#onetrust-consent-sdk #onetrust-banner-sdk{border-top:2px solid #d0d3d4!important;box-shadow:none;padding:24px}div#onetrust-consent-sdk button{border-radius:0!important;box-shadow:none!important;box-sizing:border-box!important;font-size:20px!important;font-weight:400!important;letter-spacing:0!important;max-width:none!important;white-space:nowrap!important}div#onetrust-consent-sdk button:not(.ot-link-btn){background-color:#007dbc!important;border:2px solid #007dbc!important;color:#fff!important;height:48px!important;padding:0 1em!important;width:auto!important}div#onetrust-consent-sdk button:hover{background-color:#fff!important;border-color:#949494!important;color:#2e2e2e!important}div#onetrust-consent-sdk button.ot-link-btn{color:#007dbc!important;font-size:16px!important;text-decoration:underline}div#onetrust-consent-sdk button.ot-link-btn:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-consent-sdk a,div#onetrust-pc-sdk a{color:#007dbc!important;text-decoration:underline!important}div#onetrust-consent-sdk a,div#onetrust-consent-sdk button,div#onetrust-consent-sdk p:hover{opacity:1!important}div#onetrust-consent-sdk a:focus,div#onetrust-consent-sdk button:focus,div#onetrust-consent-sdk input:focus{outline:2px solid #949494!important;outline-offset:1px!important}div#onetrust-banner-sdk .ot-sdk-container{padding:0;width:auto}div#onetrust-banner-sdk .ot-sdk-row{align-items:flex-start;box-sizing:border-box;display:flex;flex-direction:column;justify-content:space-between;margin:auto;max-width:1152px}div#onetrust-banner-sdk .ot-sdk-row:after{display:none}div#onetrust-banner-sdk #onetrust-group-container,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-group-container,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{flex-grow:1;width:auto}div#onetrust-banner-sdk #onetrust-policy,div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy{margin:0;overflow:visible}div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy-text,div#onetrust-consent-sdk #onetrust-policy-text{font-size:16px;line-height:24px;max-width:44em;margin:0}div#onetrust-consent-sdk #onetrust-policy-text a[href]{font-weight:400;margin-left:8px}div#onetrust-banner-sdk #onetrust-button-group-parent{flex:0 0 auto;margin:32px 0 0;width:100%}div#onetrust-banner-sdk #onetrust-button-group{display:flex;flex-direction:row;flex-wrap:wrap;justify-content:flex-end;margin:-8px}div#onetrust-banner-sdk .banner-actions-container{display:flex;flex:1 0 auto}div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button:last-of-type,div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto;margin:8px;width:auto}div#onetrust-consent-sdk #onetrust-pc-btn-handler{background-color:#fff!important;color:inherit!important}div#onetrust-banner-sdk #onetrust-close-btn-container{display:none}@media only screen and (min-width:556px){div#onetrust-consent-sdk #onetrust-banner-sdk{padding:40px}div#onetrust-banner-sdk #onetrust-policy{margin:0 40px 0 0}div#onetrust-banner-sdk .ot-sdk-row{align-items:center;flex-direction:row}div#onetrust-banner-sdk #onetrust-button-group-parent,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-button-group-parent,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{margin:0;padding:0;width:auto}div#onetrust-banner-sdk #onetrust-button-group,div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group{align-items:stretch;flex-direction:column-reverse;margin:0}div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto}}@media only screen and (min-width:768px){div#onetrust-banner-sdk #onetrust-policy{margin:0 48px 0 0}div#onetrust-consent-sdk #onetrust-banner-sdk{padding:48px}}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-size:16px;line-height:24px}div#onetrust-consent-sdk #onetrust-pc-sdk p,div#onetrust-pc-sdk #ot-pc-desc,div#onetrust-pc-sdk .category-host-list-handler,div#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{font-size:16px;font-weight:400;line-height:24px}div#onetrust-consent-sdk a:hover,div#onetrust-pc-sdk a:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-pc-sdk{border-radius:0;bottom:0;height:auto;left:0;margin:auto;max-width:100%;overflow:hidden;right:0;top:0;width:512px;max-height:800px}div#onetrust-pc-sdk .ot-pc-header{display:none}div#onetrust-pc-sdk #ot-pc-content{overscroll-behavior:contain;padding:0 12px 0 24px;margin:16px 4px 0 0;top:0;right:16px;left:0;width:auto}div#onetrust-pc-sdk #ot-category-title,div#onetrust-pc-sdk #ot-pc-title{font-size:24px;font-weight:400;line-height:32px;margin:0 0 16px}div#onetrust-pc-sdk #ot-pc-desc{padding:0}div#onetrust-pc-sdk #ot-pc-desc a{display:inline}div#onetrust-pc-sdk #accept-recommended-btn-handler{display:none!important}div#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:2px solid #007dbc;outline-offset:-1px;transition:none}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item{border-width:0 0 2px}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item:first-of-type{border-width:2px 0}div#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{padding:8px 0;width:100%}div#onetrust-pc-sdk .ot-plus-minus{transform:translateY(2px)}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item>button{background:0 0!important;border:0!important;height:44px!important;max-width:none!important;width:calc(100% - 48px)!important}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-weight:700}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr{padding:0}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item .ot-acc-grpdesc{padding:0;width:100%}div#onetrust-pc-sdk .ot-acc-grpcntr .ot-subgrp-cntr{border:0;padding:0}div#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp{margin:0}div#onetrust-pc-sdk .ot-always-active-group .ot-cat-header{width:calc(100% - 160px)}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{width:calc(100% - 88px)}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active{color:inherit;font-size:12px;font-weight:400;line-height:1.5;padding-right:48px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:before{border-radius:12px;position:absolute;right:0;top:0;content:'';background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:after{border-radius:50%;position:absolute;right:5px;top:4px;content:'';background-color:#007dbc;height:12px;width:12px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active,div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl{right:2px}div#onetrust-pc-sdk .ot-switch{display:block;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob,div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob{background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob:before{background-color:#737373;height:8px;left:4px;top:4px;width:8px}div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob:before{background-color:#007dbc;height:12px;left:0;top:2px;width:12px}div#onetrust-pc-sdk .ot-tgl input:focus+.ot-switch .ot-switch-nob{box-shadow:0 0;outline:2px solid #007dbc!important;outline-offset:1px;transition:none}div#onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-grpcntr.ot-acc-txt{background-color:transparent;padding-left:3px}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr,div#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr{overflow:visible;width:100%}div#onetrust-pc-sdk .ot-pc-footer{border-top:0 solid}div#onetrust-pc-sdk .ot-btn-container{padding-top:24px;text-align:center}div#onetrust-pc-sdk .ot-pc-footer button{margin:8px 0;background-color:#fff}div#onetrust-pc-sdk .ot-pc-footer-logo{background-color:#fff}div#onetrust-pc-sdk #ot-lst-title span{font-size:24px;font-weight:400;line-height:32px}div#onetrust-pc-sdk #ot-host-lst .ot-host-desc,div#onetrust-pc-sdk #ot-host-lst .ot-host-expand,div#onetrust-pc-sdk #ot-host-lst .ot-host-name,div#onetrust-pc-sdk #ot-host-lst .ot-host-name a,div#onetrust-pc-sdk .back-btn-handler,div#onetrust-pc-sdk .ot-host-opt li>div div{font-size:16px;font-weight:400;line-height:24px}div#onetrust-pc-sdk #ot-pc-lst{top:0}div#onetrust-pc-sdk .back-btn-handler{text-decoration:none!important}div#onetrust-pc-sdk #filter-btn-handler:hover svg{filter:invert(1)}div#onetrust-pc-sdk .back-btn-handler svg{width:16px;height:16px}div#onetrust-pc-sdk .ot-host-item>button{background:0 0!important;border:0!important;height:66px!important;max-width:none!important;width:calc(100% - 5px)!important;transform:translate(2px,2px)}div#onetrust-pc-sdk .ot-host-item{border-bottom:2px solid #b9b9b9;padding:0}div#onetrust-pc-sdk .ot-host-item .ot-acc-hdr{margin:0 0 -6px;padding:8px 0}div#onetrust-pc-sdk ul li:first-child{border-top:2px solid #b9b9b9}div#onetrust-pc-sdk .ot-host-item .ot-plus-minus{margin:0 8px 0 0}div#onetrust-pc-sdk .ot-search-cntr{width:calc(100% - 48px)}div#onetrust-pc-sdk .ot-host-opt .ot-host-info{background-color:transparent}div#onetrust-pc-sdk .ot-host-opt li>div div{padding:0}div#onetrust-pc-sdk #vendor-search-handler{border-radius:0;border-color:#939393;border-style:solid;border-width:2px 0 2px 2px;font-size:20px;height:48px;margin:0}div#onetrust-pc-sdk .ot-lst-subhdr svg{right:0;top:8px}div#onetrust-pc-sdk .ot-fltr-cntr{box-sizing:border-box;right:0;width:48px}div#onetrust-pc-sdk #filter-btn-handler{width:48px!important;padding:8px!important}div#onetrust-pc-sdk #ot-fltr-modal{max-height:400px}
.ot-sdk-cookie-policy{font-family:inherit;font-size:16px}.ot-sdk-cookie-policy.otRelFont{font-size:1rem}.ot-sdk-cookie-policy h3,.ot-sdk-cookie-policy h4,.ot-sdk-cookie-policy h6,.ot-sdk-cookie-policy p,.ot-sdk-cookie-policy li,.ot-sdk-cookie-policy a,.ot-sdk-cookie-policy th,.ot-sdk-cookie-policy #cookie-policy-description,.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group,.ot-sdk-cookie-policy #cookie-policy-title{color:dimgray}.ot-sdk-cookie-policy #cookie-policy-description{margin-bottom:1em}.ot-sdk-cookie-policy h4{font-size:1.2em}.ot-sdk-cookie-policy h6{font-size:1em;margin-top:2em}.ot-sdk-cookie-policy th{min-width:75px}.ot-sdk-cookie-policy a,.ot-sdk-cookie-policy a:hover{background:#fff}.ot-sdk-cookie-policy thead{background-color:#f6f6f4;font-weight:bold}.ot-sdk-cookie-policy .ot-mobile-border{display:none}.ot-sdk-cookie-policy section{margin-bottom:2em}.ot-sdk-cookie-policy table{border-collapse:inherit}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy{font-family:inherit;font-size:1rem}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h3,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h4,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h6,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy p,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy li,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-title{color:dimgray}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description{margin-bottom:1em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-subgroup{margin-left:1.5em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group-desc,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-table-header,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy span,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td{font-size:.9em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td span,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td a{font-size:inherit}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group{font-size:1em;margin-bottom:.6em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-title{margin-bottom:1.2em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy>section{margin-bottom:1em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th{min-width:75px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a:hover{background:#fff}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy thead{background-color:#f6f6f4;font-weight:bold}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-mobile-border{display:none}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy section{margin-bottom:2em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-subgroup ul li{list-style:disc;margin-left:1.5em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-subgroup ul li h4{display:inline-block}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table{border-collapse:inherit;margin:auto;border:1px solid #d7d7d7;border-radius:5px;border-spacing:initial;width:100%;overflow:hidden}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table th,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table td{border-bottom:1px solid #d7d7d7;border-right:1px solid #d7d7d7}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr:last-child td{border-bottom:0px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr th:last-child,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr td:last-child{border-right:0px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-host,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-cookies-type{width:25%}.ot-sdk-cookie-policy[dir=rtl]{text-align:left}#ot-sdk-cookie-policy h3{font-size:1.5em}@media only screen and (max-width: 530px){.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) table,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) thead,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tbody,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) th,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr{display:block}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) thead tr{position:absolute;top:-9999px;left:-9999px}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr{margin:0 0 1em 0}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr:nth-child(odd),.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr:nth-child(odd) a{background:#f6f6f4}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td{border:none;border-bottom:1px solid #eee;position:relative;padding-left:50%}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td:before{position:absolute;height:100%;left:6px;width:40%;padding-right:10px}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) .ot-mobile-border{display:inline-block;background-color:#e4e4e4;position:absolute;height:100%;top:0;left:45%;width:2px}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td:before{content:attr(data-label);font-weight:bold}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) li{word-break:break-word;word-wrap:break-word}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table{overflow:hidden}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table td{border:none;border-bottom:1px solid #d7d7d7}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy thead,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy tbody,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy tr{display:block}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-host,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-cookies-type{width:auto}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy tr{margin:0 0 1em 0}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td:before{height:100%;width:40%;padding-right:10px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td:before{content:attr(data-label);font-weight:bold}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy li{word-break:break-word;word-wrap:break-word}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy thead tr{position:absolute;top:-9999px;left:-9999px;z-index:-9999}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr:last-child td{border-bottom:1px solid #d7d7d7;border-right:0px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr:last-child td:last-child{border-bottom:0px}}
                
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h5,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h6,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy li,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy p,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy span,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description {
                        color: #696969;
                    }
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th {
                        color: #696969;
                    }
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group {
                        color: #696969;
                    }
                    
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-title {
                            color: #696969;
                        }
                    
            
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table th {
                            background-color: #F8F8F8;
                        }
                    
            .ot-floating-button__front{background-image:url('https://cdn.cookielaw.org/logos/static/ot_persistent_cookie_icon.png')}</style><script src="https://static.addtoany.com/menu/modules/core.pt5ow5lr.js" type="module"></script><style type="text/css">.a2a_hide{display:none}.a2a_logo_color{background-color:#0166ff}.a2a_menu,.a2a_menu *{-moz-box-sizing:content-box;-webkit-box-sizing:content-box;box-sizing:content-box;float:none;margin:0;padding:0;position:static;height:auto;width:auto}.a2a_menu{border-radius:6px;display:none;direction:ltr;background:#FFF;font:16px sans-serif-light,HelveticaNeue-Light,"Helvetica Neue Light","Helvetica Neue",Arial,Helvetica,"Liberation Sans",sans-serif;color:#000;line-height:12px;border:1px solid #CCC;vertical-align:baseline;overflow:hidden}.a2a_mini{min-width:200px;position:absolute;width:300px;z-index:9999997}.a2a_overlay{display:none;background:#616c7deb;backdrop-filter:blur(4px);-webkit-backdrop-filter:blur(4px);position:fixed;top:0;right:0;left:0;bottom:0;z-index:9999998;-webkit-tap-highlight-color:transparent;transition:opacity .14s,backdrop-filter .14s}.a2a_full{background:#FFF;border:1px solid #FFF;box-shadow:#2a2a2a1a 0 0 20px 10px;height:auto;height:calc(320px);top:15%;left:50%;margin-left:-320px;position:fixed;text-align:center;width:640px;z-index:9999999;transition:transform .14s,opacity .14s}.a2a_full_footer,.a2a_full_header,.a2a_full_services{border:0;margin:0;padding:12px;box-sizing:border-box}.a2a_full_header{padding-bottom:8px}.a2a_full_services{height:280px;overflow-y:scroll;padding:0 12px;-webkit-overflow-scrolling:touch}.a2a_full_services .a2a_i{display:inline-block;float:none;width:181px;width:calc(33.334% - 18px)}div.a2a_full_footer{font-size:12px;text-align:center;padding:8px 14px}div.a2a_full_footer a,div.a2a_full_footer a:visited{display:inline;font-size:12px;line-height:14px;padding:8px 14px}div.a2a_full_footer a:focus,div.a2a_full_footer a:hover{background:0 0;border:0;color:#0166FF}div.a2a_full_footer a span.a2a_s_a2a,div.a2a_full_footer a span.a2a_w_a2a{background-size:14px;border-radius:3px;display:inline-block;height:14px;line-height:14px;margin:0 3px 0 0;vertical-align:top;width:14px}.a2a_modal{height:0;left:50%;margin-left:-320px;position:fixed;text-align:center;top:15%;width:640px;z-index:9999999;transition:transform .14s,opacity .14s;-webkit-tap-highlight-color:transparent}.a2a_modal_body{background:0 0;border:0;font:24px sans-serif-light,HelveticaNeue-Light,"Helvetica Neue Light","Helvetica Neue",Arial,Helvetica,"Liberation Sans",sans-serif;position:relative;height:auto;width:auto}.a2a_thanks{color:#fff;height:auto;margin-top:20px;width:auto}.a2a_thanks>div:first-child{margin:0 0 40px 0}.a2a_thanks div *{height:inherit}#a2a_copy_link{background:#FFF;border:1px solid #FFF;cursor:pointer;margin-top:15%}label.a2a_s_link#a2a_copy_link_icon,label.a2a_w_link#a2a_copy_link_icon{background-size:48px;border-radius:0;display:inline-block;height:48px;left:0;line-height:48px;margin:0 3px 0 0;position:absolute;vertical-align:top;width:48px}#a2a_modal input#a2a_copy_link_text{background-color:transparent;border:0;color:#2A2A2A;cursor:pointer;font:inherit;height:48px;left:62px;max-width:initial;min-height:auto;padding:0;position:relative;width:564px;width:calc(100% - 76px)}#a2a_copy_link_copied{background-color:#0166ff;color:#fff;display:none;font:inherit;font-size:16px;margin-top:1px;padding:3px 8px}@media (forced-colors:active){.a2a_color_buttons a,.a2a_svg{forced-color-adjust:none}}@media (prefers-color-scheme:dark){.a2a_menu a,.a2a_menu a.a2a_i,.a2a_menu a.a2a_i:visited,.a2a_menu a.a2a_more,i.a2a_i{border-color:#2a2a2a!important;color:#fff!important}.a2a_menu a.a2a_i:active,.a2a_menu a.a2a_i:focus,.a2a_menu a.a2a_i:hover,.a2a_menu a.a2a_more:active,.a2a_menu a.a2a_more:focus,.a2a_menu a.a2a_more:hover,.a2a_menu_find_container{border-color:#444!important;background-color:#444!important}.a2a_menu:not(.a2a_thanks){background-color:#2a2a2a;border-color:#2a2a2a}.a2a_menu_find{color:#fff!important}.a2a_menu label.a2a_s_find svg{background-color:transparent!important}.a2a_menu label.a2a_s_find svg path{fill:#fff!important}.a2a_full{box-shadow:#00000066 0 0 20px 10px}.a2a_overlay{background-color:#373737eb}}@media print{.a2a_floating_style,.a2a_menu,.a2a_overlay{visibility:hidden}}@keyframes a2aFadeIn{from{opacity:0}to{opacity:1}}.a2a_starting{opacity:0}.a2a_starting.a2a_full,.a2a_starting.a2a_modal{transform:scale(.8)}@media (max-width:639px){.a2a_full{border-radius:0;top:15%;left:0;margin-left:auto;width:100%}.a2a_modal{left:0;margin-left:10px;width:calc(100% - 20px)}}@media (min-width:318px) and (max-width:437px){.a2a_full .a2a_full_services .a2a_i{width:calc(50% - 18px)}}@media (max-width:317px){.a2a_full .a2a_full_services .a2a_i{width:calc(100% - 18px)}}@media (max-height:436px){.a2a_full{bottom:40px;height:auto;top:40px}}@media (max-height:550px){.a2a_modal{top:30px}}@media (max-height:360px){.a2a_modal{top:20px}.a2a_thanks>div:first-child{margin-bottom:20px}}.a2a_menu a{color:#0166FF;text-decoration:none;font:16px sans-serif-light,HelveticaNeue-Light,"Helvetica Neue Light","Helvetica Neue",Arial,Helvetica,"Liberation Sans",sans-serif;line-height:14px;height:auto;width:auto;outline:0}.a2a_menu a.a2a_i:visited,.a2a_menu a.a2a_more{color:#0166FF}.a2a_menu a.a2a_i:active,.a2a_menu a.a2a_i:focus,.a2a_menu a.a2a_i:hover,.a2a_menu a.a2a_more:active,.a2a_menu a.a2a_more:focus,.a2a_menu a.a2a_more:hover{color:#2A2A2A;border-color:#EEE;border-style:solid;background-color:#EEE;text-decoration:none}.a2a_menu label.a2a_s_find{background-size:24px;height:24px;left:8px;pointer-events:auto;position:absolute;top:7px;width:24px}.a2a_menu label.a2a_s_find svg{background-color:#FFF}.a2a_menu label.a2a_s_find svg path{fill:#CCC}#a2a_menu_container{display:inline-block}.a2a_menu_find_container{border:1px solid #CCC;border-radius:6px;padding:2px 24px 2px 0;position:relative;text-align:left}.a2a_cols_container .a2a_col1{overflow-x:hidden;overflow-y:auto;-webkit-overflow-scrolling:touch}#a2a_modal input,#a2a_modal input[type=text],.a2a_menu input,.a2a_menu input[type=text]{display:block;background-image:none;box-shadow:none;line-height:100%;margin:0;outline:0;overflow:hidden;padding:0;-moz-box-shadow:none;-webkit-box-shadow:none;-webkit-appearance:none}#a2afeed_find_container input,#a2afeed_find_container input[type=text],#a2apage_find_container input,#a2apage_find_container input[type=text]{background-color:transparent;border:0;box-sizing:content-box;color:#2A2A2A;float:none;font:inherit;font-size:16px;height:28px;line-height:20px;left:38px;outline:0;margin:0;max-width:initial;min-height:initial;padding:2px 0;position:relative;width:99%}.a2a_clear{clear:both}.a2a_svg{background-repeat:no-repeat;display:block;overflow:hidden;height:32px;line-height:32px;padding:0;pointer-events:none;width:32px}.a2a_svg svg{background-repeat:no-repeat;background-position:50% 50%;border:none;display:block;left:0;margin:0 auto;overflow:hidden;padding:0;position:relative;top:0;width:auto;height:auto}a.a2a_i,i.a2a_i{display:block;float:left;border:1px solid #FFF;line-height:24px;padding:6px 8px;text-align:left;white-space:nowrap;overflow:hidden;text-overflow:ellipsis;width:132px}a.a2a_i span,a.a2a_more span{display:inline-block;overflow:hidden;vertical-align:top}a.a2a_i .a2a_svg{margin:0 6px 0 0}a.a2a_i .a2a_svg,a.a2a_more .a2a_svg{background-size:24px;height:24px;line-height:24px;width:24px}a.a2a_sss:hover{border-left:1px solid #CCC}a.a2a_more{border-bottom:1px solid #FFF;border-left:0;border-right:0;line-height:24px;margin:6px 0 0;padding:6px;-webkit-touch-callout:none}a.a2a_more span{height:24px;margin:0 6px 0 0}.a2a_kit .a2a_svg{background-repeat:repeat}.a2a_default_style a:empty,.a2a_flex_style a:empty,.a2a_floating_style a:empty,.a2a_overlay_style a:empty{display:none}.a2a_color_buttons a,.a2a_floating_style a{text-decoration:none}.a2a_default_style:not(.a2a_flex_style) a{float:left;line-height:16px;padding:0 2px}.a2a_default_style a:hover .a2a_svg,.a2a_floating_style a:hover .a2a_svg,.a2a_overlay_style a:hover .a2a_svg svg{opacity:.7}.a2a_overlay_style.a2a_default_style a:hover .a2a_svg{opacity:1}.a2a_default_style .a2a_count,.a2a_default_style .a2a_svg,.a2a_floating_style .a2a_svg,.a2a_menu .a2a_svg,.a2a_vertical_style .a2a_count,.a2a_vertical_style .a2a_svg{border-radius:4px}.a2a_default_style .a2a_counter img,.a2a_default_style .a2a_dd,.a2a_default_style .a2a_svg{float:left}.a2a_default_style .a2a_img_text{margin-right:4px}.a2a_default_style .a2a_divider{border-left:1px solid #000;display:inline;float:left;height:16px;line-height:16px;margin:0 5px}.a2a_kit a{cursor:pointer;transition:none}.a2a_floating_style{background-color:#fff;border-radius:6px;position:fixed;z-index:9999995}.a2a_overlay_style{z-index:2147483647}.a2a_floating_style,.a2a_overlay_style{animation:a2aFadeIn .2s ease-in;padding:4px}.a2a_vertical_style:not(.a2a_flex_style) a{clear:left;display:block;overflow:hidden;padding:4px}.a2a_floating_style.a2a_default_style{bottom:0}.a2a_floating_style.a2a_default_style a,.a2a_overlay_style.a2a_default_style a{padding:4px}.a2a_count{background-color:#fff;border:1px solid #ccc;box-sizing:border-box;color:#2a2a2a;display:block;float:left;font:12px Arial,Helvetica,sans-serif;height:16px;margin-left:4px;position:relative;text-align:center;width:50px}.a2a_count:after,.a2a_count:before{border:solid transparent;border-width:4px 4px 4px 0;content:"";height:0;left:0;line-height:0;margin:-4px 0 0 -4px;position:absolute;top:50%;width:0}.a2a_count:before{border-right-color:#ccc}.a2a_count:after{border-right-color:#fff;margin-left:-3px}.a2a_count span{animation:a2aFadeIn .14s ease-in}.a2a_vertical_style .a2a_counter img{display:block}.a2a_vertical_style .a2a_count{float:none;margin-left:0;margin-top:6px}.a2a_vertical_style .a2a_count:after,.a2a_vertical_style .a2a_count:before{border:solid transparent;border-width:0 4px 4px 4px;content:"";height:0;left:50%;line-height:0;margin:-4px 0 0 -4px;position:absolute;top:0;width:0}.a2a_vertical_style .a2a_count:before{border-bottom-color:#ccc}.a2a_vertical_style .a2a_count:after{border-bottom-color:#fff;margin-top:-3px}.a2a_color_buttons .a2a_count,.a2a_color_buttons .a2a_count:after,.a2a_color_buttons .a2a_count:before,.a2a_color_buttons.a2a_vertical_style .a2a_count:after,.a2a_color_buttons.a2a_vertical_style .a2a_count:before{background-color:transparent;border:none;color:#fff;float:none;width:auto}.a2a_color_buttons.a2a_vertical_style .a2a_count{margin-top:0}.a2a_flex_style{display:flex;align-items:flex-start;gap:0}.a2a_default_style.a2a_flex_style{left:0;right:0;width:100%}.a2a_vertical_style.a2a_flex_style{flex-direction:column;top:0;bottom:0}.a2a_flex_style a{display:flex;justify-content:center;flex:1;padding:4px}.a2a_flex_style.a2a_vertical_style a{flex-direction:column}.a2a_floating_style.a2a_color_buttons,.a2a_floating_style.a2a_flex_style{border-radius:0;padding:0}.a2a_floating_style.a2a_default_style.a2a_flex_style{bottom:0}.a2a_kit.a2a_flex_style .a2a_counter img,.a2a_kit.a2a_flex_style .a2a_dd,.a2a_kit.a2a_flex_style .a2a_svg{float:none}.a2a_nowrap{white-space:nowrap}.a2a_note{margin:0 auto;padding:9px;font-size:12px;text-align:center}.a2a_note .a2a_note_note{margin:0;color:#2A2A2A}.a2a_wide a{display:block;margin-top:3px;border-top:1px solid #EEE;text-align:center}.a2a_label{position:absolute!important;clip-path:polygon(0px 0px,0px 0px,0px 0px);-webkit-clip-path:polygon(0px 0px,0px 0px,0px 0px);overflow:hidden;height:1px;width:1px}.a2a_kit,.a2a_menu,.a2a_modal,.a2a_overlay{-ms-touch-action:manipulation;touch-action:manipulation;outline:0}.a2a_dd{-webkit-user-drag:none}.a2a_dd img{border:0}.a2a_button_facebook_like iframe{max-width:none}</style><style></style><link href="https://crossmark-cdn.crossref.org/widget/v2.0/style.css" type="text/css" rel="stylesheet"></head>
<body class="pb-ui">
    <div class="skip">
        <a href="#page-body-id" id="skipNavigationLink" title="Skip to Main Content" tabindex="0" class="skipToContent">Skip to Main Content</a>
        <a href="#mega-menu-nav" id="skipToMainMenu" title="Skip to Main Menu" tabindex="0" class="skipToContent">Skip to Main Menu</a>
    </div>



    <div id="pb-page-content" data-ng-non-bindable="">
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="ux3-layout-widget" data-widget-id="832ad3f3-97ac-45ab-9af1-90c6f7bff54a" class="ux3">
        
    

        


    

            

            
                <div class="cell fonts-loaded">
            

            <div data-widget-id="832ad3f3-97ac-45ab-9af1-90c6f7bff54a" data-widget-name="ux3-layout-widget"><div>
    

        


    

            

            

            <div id="a1f1ba25-79af-4194-aaa9-66206104a800" class="widget loginPopup none widget-none widget-compact-all"><div class="wrapped"><div class="widget-body body body-none body-compact-all"><div role="dialog" aria-modal="true" aria-label="Login" class="popup login-popup hidden"><div class="content"><a href="#" aria-label="Close login popup" title="Close login popup" role="button" class="close"><i class="icon-close_thin"></i></a><h2 class="login-title">Login to your account</h2><div class="login-form"><p tabindex="-1" class="message error"></p><form action="/action/doLogin?redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2901065-5" method="post"><input type="hidden" name="redirectUri" value="https://www.cell.com/cell/fulltext/S0092-8674(21)01065-5"><input type="hidden" name="referer" value="/cell/fulltext/S0092-8674(21)01065-5"><input type="hidden" name="ecommRedirect"><input type="hidden" name="code" value="cell-site"><input type="hidden" name="UX3LoginWidget" value="true"><input type="hidden" name="isPopup" value="true"><div class="input-group login-input--wrapper"><div class="label"><label for="login">Email/Username</label></div><input id="login" type="text" name="login" value="" size="15" class="login"><div role="alert" class="error-hint js--hidden"><div class="error-hint__text"><div class="error-text">Your email address is a required field.</div><div class="email-example">E.g., j.smith@mail.com</div></div></div></div><div class="input-group login-input--wrapper"><div class="label"><label for="password">Password</label></div><input id="password" type="password" name="password" value="" autocomplete="off" class="password"><a href="#" class="login__field__action js-show-password">Show</a><div role="alert" class="error-hint js--hidden"><span class="error-hint__text">Your password is a required field.</span></div><div class="actions"><a href="https://www.cell.com/action/requestResetPassword" class="link show-request-reset-password">Forgot password?</a></div></div><div class="remember"><input type="checkbox" name="savePassword" id="a1f1ba25-79af-4194-aaa9-66206104a800-remember" value="1" checked="checked"><label for="a1f1ba25-79af-4194-aaa9-66206104a800-remember">Remember me</label></div><div class="submit"><input type="submit" name="submit" value="Log in" class="button submit primary"></div></form></div><div class="create-account"><div>Don’t have an account?</div><a href="/action/registration?redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2901065-5" class="show-registration">Create a Free Account</a></div></div></div></div></div></div>

            

        

    

        


    

            

            

            <div role="dialog" aria-modal="true" aria-label="Reset password" class="popup top-drawer request-reset-password-drawer hidden"><div class="content"><a href="#" aria-label="Close reset password popup" title="Close reset password popup" role="button" class="close"><i class="icon-close_thin"></i></a><div class="request-reset"><div class="form"><p class="sub">If you don't remember your password, you can reset it by entering your email address and clicking the Reset Password button. You will then receive an email that contains a secure link for resetting your password</p><div tabindex="-1" class="message error"></div><form action="/action/requestResetPassword" method="post" class="request-reset-password-form"><input type="hidden" name="platform"><input type="hidden" name="requestResetPassword" value="true"><input type="hidden" name="codeSite" value="cell-site"><div class="input-group"><div class="label"><label for="reset-password-email">Email*</label></div><input id="reset-password-email" type="text" name="email" value="" size="15" class="email"></div><div class="submit" disabled="disabled"><input type="submit" name="submit" value="Submit" disabled="disabled" class="button primary submit"></div></form></div><div class="success-template hidden"><p>If the address matches a valid account an email will be sent to __email__ with instructions for resetting your password</p></div></div><div class="request-reset-cancel"><a href="#" aria-label="Close reset password popup" title="Close reset password popup" class="cancel">Cancel</a></div></div></div>

            

        

    

        


    

            

            
                <div class="ux3-ads">
            

            <div class="pb-ad">
            <!-- Do Not Modify without contacting ELS-HSOnlineCampaign@elsevier.com -->
<!-- To be kept in sync with Global Commercial Publishing Operations-->
<!-- Start: GPT Async -->
<div id="gpt-Variables" data-adfile="CELLJOURNAL">
	<input type="hidden" id="gptSite" value="cell">
	<input type="hidden" id="gptLoggedIn" value="no">
	<input type="hidden" id="gptAccessType" value="ae:ANON_GUEST|ae:ANON_GUEST">
	<input type="hidden" id="gptDefaultSite" value="cell">
</div>
<div id="gpt-Interscroller" class="gpt-InScrollBox" style="display:none;">
	<div class="gpt-InScrollLabel">ADVERTISEMENT</div>
	<div class="gpt-InScrollContainer">
		<div class="gpt-InScrollWrapper">
			<div class="gpt-InScrollInner">
				<div class="gpt-adinterscroller" data-adsize="interscroller"></div>
			</div>
		</div>
	</div>
	<div class="gpt-InScrollLabel">SCROLL TO CONTINUE WITH CONTENT</div>
</div>
<div id="gpt-DebugDiv" class="gpt-DebugDiv" style="display:none;">
	<div style="width: 100%; display: table;">
		<div style="display: table-row">
			<div style="width: 200px; display: table-cell;">
				<button class="gpt-DebugBtn" onclick="gpt_ShowConsole()">Open GPT Console</button><br>
				<button id="gpt_GrapeShotDebugBtn" class="gpt-DebugBtn" onclick="gpt_GrapeShotDebug();">Open Oracle Keywords</button><br>
				<br>
				<button class="gpt-DebugBtn" onclick="gpt_LoadDebugValues()">Refresh Values</button>
			</div>
			<div style="display: table-cell;">
				<table class="gpt-DebugTable">
					<thead>
						<tr>
							<th>Property</th>
							<th>Value</th>
						</tr>
					</thead>
					<tbody>
						<tr><td class="gpt-DebugTitle">Status</td><td><div class="gptStatus_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Version</td><td><div class="gptVersion_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Ad File</td><td><div class="gptAdFile_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Disable Ads Flag</td><td><div class="gptDisableAds_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Environment</td><td><div class="gptTestENV_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Moat Init</td><td><div class="gptMoatInit_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Moat Ready</td><td><div class="gptMoatReady_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual Ready</td><td><div class="gptContextualReady_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual URL</td><td><div class="gptContextualURL_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual Initial Segments</td><td><div class="gptContextualInit_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual Used Segments</td><td><div class="gptContextualResult_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">AdUnit</td><td><div class="gptSite_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">SubAdUnit</td><td><div class="gptPage_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Custom Targeting</td><td><div class="gptCustomTargeting_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Ad Events</td><td><div class="gptEvent_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Invalid Ad Sizes</td><td><div class="gptInvalidAdSize_Debug"></div></td></tr>
					</tbody>
				</table>
			</div>
		</div>
	</div>
</div>
<style type="text/css">
	.gpt-DebugBtn {
		display: inline-block;
		text-align: center;
		text-decoration: none;
		margin: 2px 0;
		border: solid 1px #122b40;
		border-radius: 0.4em;
		padding: 0.5em 1em;
		color: #ffffff;
		background-color: #204d74;
		font-weight: normal;
		font-size: 12px;
	}

	.gpt-DebugDiv {
		margin: 5px;
	}

	.gpt-DebugTitle {
		font-weight: bold;
	}

	.gpt-DebugTable {
		border: 1px solid #dddddd;
		border-collapse: collapse;
		margin: 5px;
		font-size: 0.9em;
		font-family: sans-serif;
		min-width: 400px;
		margin-bottom: 25px;
	}

		.gpt-DebugTable td:first-child {
			padding: 5px;
		}

		.gpt-DebugTable thead tr th {
			background-color: #d9edf7;
			color: #ffffff;
			text-align: left;
			padding: 5px;
			border-bottom: 2px solid #204d74;
		}

		.gpt-DebugTable th,
		.gpt-DebugTable td {
			padding: 5px;
		}

		.gpt-DebugTable tbody tr {
			border-bottom: 1px solid #dddddd;
		}

			.gpt-DebugTable tbody tr:nth-of-type(even) {
				background-color: #f3f3f3;
			}

			.gpt-DebugTable tbody tr:last-of-type {
				border-bottom: 2px solid #204d74;
			}

	.gpt-InScrollLabel {
		background-color: rgb(0, 0, 0);
		color: rgb(255, 255, 255);
		position: relative;
		max-width: 100vw;
		left: 50%;
		right: 50%;
		margin-left: -50vw;
		margin-right: -50vw;
		padding: 0;
		text-align: center;
		width: 100vw !important;
		font-size: 10px;
	}

	.gpt-InScrollBox {
		height: 100vh !important;
		width: 100vw;
		cursor: pointer;
		max-width: 100vw;
		position: relative;
		left: 50%;
		right: 50%;
		margin-left: -50vw;
		margin-right: -50vw;
		margin-bottom: 50px;
		padding-top: 5px;
		padding-bottom: 5px;
		z-index: 10001;
		background-color: white;
	}

		.gpt-InScrollBox .gpt-InScrollContainer {
			height: 95vh !important;
			position: relative;
		}

		.gpt-InScrollBox .gpt-InScrollWrapper {
			position: absolute !important;
			top: 0 !important;
			left: 0;
			width: 100%;
			height: 95vh !important;
			border: 0 !important;
			margin: 0 !important;
			padding: 0 !important;
			clip: rect(0, auto, auto, 0) !important;
			-webkit-clip-path: polygon(0px 0px, 100% 0px, 100% 100%, 0px 100%) !important;
			clip-path: polygon(0px 0px, 100% 0px, 100% 100%, 0px 100%) !important;
		}

		.gpt-InScrollBox .gpt-InScrollInner {
			position: fixed !important;
			top: 0 !important;
			left: 0 !important;
			bottom: 0;
			margin: auto;
			width: 100vw;
			height: 95vh;
			-webkit-transform: translateZ(0) !important;
			display: -webkit-box;
			display: -ms-flexbox;
			display: flex;
			-webkit-box-orient: vertical;
			-webkit-box-direction: normal;
			-ms-flex-direction: column;
			flex-direction: column;
			-webkit-box-align: center;
			-ms-flex-align: center;
			align-items: center;
			-webkit-box-pack: center;
			-ms-flex-pack: center;
			justify-content: center;
			text-align: center;
		}
</style>
<script class="optanon-category-4" type="text/plain">
	//--------------------------------------------------------------------------
	var gpt_Version = '04072023';
	var gpt_AdFile = $('#gpt-Variables').data('adfile');
	var gpt_MaxMOATInitAttempts = 11;
	var gpt_MaxMOATReadyAttempts = 11;
	var gpt_MaxContextualReadyAttempts = 21;
	var gpt_MOATInitAttempts = 0;
	var gpt_MOATReadyAttempts = 0;
	var gpt_ContextualReadyAttempts = 0;
	var gpt_IsProd = true;
	var gpt_StartTime;
	var gs_channels = 'DEFAULT';
	var gpt_ContextualReady = false;
	var gpt_ContextualURL = 'Not enabled outside of Production';
	var gpt_DisableAdsFlagFound = false;

	var gptEvent_Debug = '';
	var gptCustomTargeting_Debug = '';
	var gptSite_Debug = "";
	var gptPage_Debug = "";
	var gptContextualEnabled = false;
	var gptContextualResult_Debug = '';
	var gptContextualInit_Debug = '';
	var gptContextualURL_Debug = '';
	var gptContextualReady_Debug = '';
	var gptMoatReady_Debug = '';
	var gptMoatInit_Debug = '';
	var gptTestENV_Debug = '';
	var gptDisableAds_Debug = '';
	var gptStatus_Debug = 'GPT Still Loading - Refresh Values';
	var gptInvalidAdSize_Debug = '';
	//--------------------------------------------------------------------------
	$(document).ready(function () {
		//Check to see if we are on a development env
		gpt_CheckEnvironment();

		if (!gpt_DisableAdsFlagFound) {
			gpt_StartTime = new Date();
			if (gpt_AdFile !== "PU") {
				//Once the dom has loaded, begin the process to init the ads.
				gpt_InitScripts();
				gpt_CheckForMOATInit();
			}
		}
	});
</script>
<script type="text/javascript">
	function gpt_CheckEnvironment() {
		if ($('#gptDisableAds').length > 0) {
			gptDisableAds_Debug = 'Flag Detected - Ads Disabled';
			gpt_DisableAdsFlagFound = true;
		}
		else {
			gptDisableAds_Debug = 'Ads Enabled';
			gpt_DisableAdsFlagFound = false;
		}
		switch (gpt_AdFile) {
			case 'PU':
				gptContextualEnabled = true;
				if ($("#Environment").val() !== "Prod") {
					gpt_IsProd = false;
					gptTestENV_Debug = 'Test Environment Detected';
				}
				else {
					gptTestENV_Debug = 'Production Environment Detected';
				}
				break;
			case 'JBS':
			case 'LANCET':
				gptContextualEnabled = true;
				if ($(".gpt-ad").length > 0) {
					//Other ads exist, allow the Prestitial
					$(document.body).append("<div class='gpt-ad' data-adsize='prestitial'></div>");
				}
				if (window.location.hostname.toLowerCase().indexOf('.literatumonline.') >= 0) {
					gpt_IsProd = false;
					gptTestENV_Debug = 'Test Environment Detected';
				}
				else {
					gptTestENV_Debug = 'Production Environment Detected';
				}
				break;
			default:
				//Cell
				gptContextualEnabled = false;
				if (window.location.hostname.toLowerCase().indexOf('.literatumonline.') >= 0) {
					gpt_IsProd = false;
					gptTestENV_Debug = 'Test Environment Detected';
				}
				else {
					gptTestENV_Debug = 'Production Environment Detected';
				}
				break;
		}
	}
	//--------------------------------------------------------------------------
    function gpt_GetNonCMEArticle() {
        var url;
        $('.articleCitation').each(function (i, obj) {
            if ($(obj).find('.toc__item__detials').length > 0) {
                var itemDetails = $(obj).first().find('.toc__item__detials');
                var cmeFound = 0;
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='CME']").length; }
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='(CME)']").length; }
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='Education']").length; }
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='education']").length; }
                if (cmeFound == 0) {
                    var itemLink = $(itemDetails).first().find('.toc__item__title');
                    var linkCount = $(itemLink).first().find("a:contains('CME')").length;
                    if (linkCount == 0) {
                        url = $(itemLink).first().find('a').attr('href');
                        return false;   //Found a URL, break the loop
                    }
                }
            }
        });
        return url;
    }
	//--------------------------------------------------------------------------
	function gpt_GetElapsedTime() {
		var endTime = new Date();
		var timeDiff = endTime - gpt_StartTime; //in ms
		// strip the ms
		timeDiff /= 1000;

		// get seconds
		return seconds = Math.round(timeDiff) + " seconds";
	}
	//--------------------------------------------------------------------------
	function gpt_CheckSSC() {
		//Check for ScreenShot Client
		var gpt_SSC = gpt_GetQuerystring('gpt_ssc', '');
		if (gpt_SSC == '1') {
			//it's the ScreenShotClient, disable MOAT IVT Detection
			$('#gpt-Variables').append('<input type="hidden" id="gptm_data">');
			$('#gptm_data').val(0);
			gpt_Target("m_data");
		}
	}
	//--------------------------------------------------------------------------
	function gpt_LocalSS(adTypeId) {
		window.scrollTo(0, 0);
		gpt_Sleep(1000);
		gpt_ScrollToAd(adTypeId);
		gpt_PreTearsheet(adTypeId);
	}
	//--------------------------------------------------------------------------
	function gpt_InitScripts() {
		if (gpt_AdFile == "PU") {
			if (gpt_IsProd) {
				gpt_ContextualURL = gpt_CheckForNULL($('#canonicalurl').val());
				if (gpt_ContextualURL === null) {
					gpt_ContextualURL = window.location.href.split('?')[0].split('#')[0];
				}
				else {
					gpt_ContextualURL = window.location.origin + gpt_ContextualURL;
				}
				var useSSL = 'https:' == document.location.protocol;
				var gs = document.createElement('script');
				gs.async = true;
				gs.type = 'text/javascript';
				gs.src = (useSSL ? 'https:' : 'http:') + '//elsevier.gscontxt.net/multizone/channels.cgi?url=' + encodeURIComponent(gpt_ContextualURL);
				gs.onload = function () {
					gpt_ContextualReady = true;
				};
				document.head.appendChild(gs);
			}
		}
		else {
			//JBS-Cell-Lancet
			var useSSL = 'https:' == document.location.protocol;

			var gads = document.createElement('script');
			gads.async = true;
			gads.type = 'text/javascript';
			gads.src = (useSSL ? 'https:' : 'http:') + '//securepubads.g.doubleclick.net/tag/js/gpt.js';
			document.head.appendChild(gads);

			if (gpt_IsProd && gptContextualEnabled) {
				gpt_ContextualURL = window.location.href.split('?')[0].split('#')[0];
				var gs = document.createElement('script');
				gs.async = true;
				gs.type = 'text/javascript';
				gs.src = (useSSL ? 'https:' : 'http:') + '//elsevier.gscontxt.net/multizone/channels.cgi?url=' + encodeURIComponent(gpt_ContextualURL);
				gs.onload = function () {
					gpt_ContextualReady = true;
				};
				document.head.appendChild(gs);
			}

			var elsevierMoatHeader = document.createElement('link');
			elsevierMoatHeader.rel = 'preload';
			elsevierMoatHeader.as = 'script';
			elsevierMoatHeader.src = (useSSL ? 'https:' : 'http:') + '//z.moatads.com/elsevierheader150204004183/moatheader.js';
			document.head.appendChild(elsevierMoatHeader);

			var elsevierMoatAds = document.createElement('link');
			elsevierMoatAds.rel = 'preconnect';
			elsevierMoatAds.src = (useSSL ? 'https:' : 'http:') + '//mb.moatads.com';
			document.head.appendChild(elsevierMoatAds);

			var moatHeader = document.createElement('script');
			moatHeader.async = true; gads.type = 'text/javascript';
			moatHeader.src = (useSSL ? 'https:' : 'http:') + '//z.moatads.com/elsevierheader150204004183/moatheader.js';
			document.head.appendChild(moatHeader);
		}
	}
	//--------------------------------------------------------------------------
	async function gpt_CheckForMOATInit() {
		//Check every 1/10 of a second until MOAT is Loaded or 1 second timeout
		if (gpt_MOATInitAttempts < gpt_MaxMOATInitAttempts) {
			gpt_MOATInitAttempts += 1;
			if (typeof window.moatPrebidApi === "object") {
				//MOAT LOADED
				gptMoatInit_Debug = '0.0' + gpt_MOATInitAttempts + ' seconds';
				gpt_CheckForMOATReady();
			}
			else {
				setTimeout(() => { gpt_CheckForMOATInit(); }, 100);
			}
		}
		else {
			//TIMEOUT: MOAT Not LOADED
			gptMoatInit_Debug = 'Timeout Expired';
			var googletag = googletag || {};
			googletag.cmd = googletag.cmd || [];
			gpt_CheckForContextualReady();
		}
	}
	//--------------------------------------------------------------------------
	function gpt_CheckForMOATReady() {
		//Check every 1/10 of a second until MOAT is Ready or 1 second timeout
		if (gpt_MOATReadyAttempts < gpt_MaxMOATReadyAttempts) {
			gpt_MOATReadyAttempts += 1;
			if (window.moatPrebidApi.safetyDataAvailable()) {
				//MOAT Ready
				gptMoatReady_Debug = '0.0' + gpt_MOATReadyAttempts + " seconds";
				gpt_CheckForContextualReady();
			}
			else {
				setTimeout(() => { gpt_CheckForMOATReady(); }, 100);
			}
		}
		else {
			//TIMEOUT: MOAT Not Ready
			gptMoatReady_Debug = 'Timeout Expired';
			gpt_CheckForContextualReady();
		}
	}
	//--------------------------------------------------------------------------
	function gpt_CheckForContextualReady() {
		//Check every 1/10 of a second until Contextual is Ready or 2 second timeout
		if (gpt_IsProd && gptContextualEnabled) {
			//Enabled
			if (gpt_ContextualReadyAttempts < gpt_MaxContextualReadyAttempts) {
				gpt_ContextualReadyAttempts += 1;
				if (gpt_ContextualReady) {
					//Contextual Ready
					gptContextualURL_Debug = gpt_ContextualURL;
					gptContextualReady_Debug = '0.0' + gpt_ContextualReadyAttempts + " seconds";
					gptContextualInit_Debug = gs_channels;
					gpt_InitAds();
				}
				else {
					setTimeout(() => { gpt_CheckForContextualReady(); }, 100);
				}
			}
			else {
				//LOG TIMEOUT error
				//REMOVE FOR NOW (SHOULD LOG TO A DIFF LOCATION)
				//TIMEOUT: Contextual Not Ready
				$('#gpt_GrapeShotDebugBtn').remove();
				gptContextualURL_Debug = gpt_ContextualURL;
				gptContextualReady_Debug = 'Timeout Expired';
				gptContextualInit_Debug = "";
				gpt_InitAds();
			}
		}
		else {
			//Not Enabled
			$('#gpt_GrapeShotDebugBtn').remove();
			switch (gpt_AdFile) {
				case 'PU':
				case 'LANCET':
				case 'JBS':
					gptContextualURL_Debug = 'Not enabled outside of Production';
					gptContextualReady_Debug = 'Not enabled outside of Production';
					gptContextualInit_Debug = 'Not enabled outside of Production';
					break;
				case "CELLJOURNAL":
				case "CELLBUCKET":
					gptContextualReady_Debug = 'Not enabled on Cell';
					gptContextualInit_Debug = 'Not enabled on Cell';
					gptContextualURL_Debug = 'Not enabled on Cell';
					break;
			}
			gpt_InitAds();
		}
	}
	//--------------------------------------------------------------------------
	function GPT_GetPageByURL() {
		var host = window.location.host.toLocaleLowerCase()
		var localpath = window.location.pathname.toLocaleLowerCase();

		///Check for Cell.com
		if (host.indexOf('cell.com') >= 0) {
			return null;
		}

		///Check for Cell.com (marlin-stag)
		if (host.indexOf('www-cell-com') >= 0) {
			return null;
		}

		///Check for Lancet
		if (host.indexOf('thelancet') >= 0) {
			return null;
		}

		if (localpath.indexOf('/article/') >= 0) {
			return 'article.fulltext';
		}

		if (localpath.indexOf('/action/dosearch') >= 0) {
			return 'search.searchResults';
		}

		if (localpath.indexOf('/content/') >= 0) {
			return 'periodical.editorial';
		}

		return 'periodical.home';
	}
	//--------------------------------------------------------------------------
	function gpt_CheckForNULL(x, valueifnull) {
		if (typeof valueifnull == "undefined") {
			valueifnull = null;
		}

		if (typeof x == 'undefined') {
			return valueifnull;
		}
		if (x == null) {
			return valueifnull;
		}
		if (x == "null") {
			return valueifnull;
		}
		if ($.trim(x) == '') {
			return valueifnull;
		}

		return x;
	}
	//--------------------------------------------------------------------------
	function gpt_GetAdditionalTargeting() {
		//adds page level targeting of custom variables
		//the id (minus gpt) and the value will be used as a key-value pair
		//eg: <input type="hidden" id="gptissuepii" value="x" class="gpt-Target">
		//eg: target issuepii=x
		$(".gpt-Target").each(function (index) {
			var gptTargetId = $(this).attr('id');
			gptTargetId = gptTargetId.replace('gpt', '').toLowerCase();
			if (gptTargetId !== '') {
				gpt_Target(gptTargetId);
			}
		})
	}
	//--------------------------------------------------------------------------
	function gpt_GetQuerystring(key, default_) {
		//function to read querystring variables
		if (default_ === null) default_ = '';
		key = key.replace(/[\[]/, "\\\[").replace(/[\]]/, "\\\]");
		var regex = new RegExp("[\\?&]" + key + "=([^&#]*)");
		var qs = regex.exec(window.location.href);
		if (qs === null)
			return default_;
		else
			return qs[1];
	}
	//--------------------------------------------------------------------------
	function gpt_GetDFPTestId() {
		//Get any request for a test id from querystring and set custom targeting parameter
		var gptDFPTestId = gpt_GetQuerystring('dfptestid', '');
		if (gptDFPTestId !== '') {
			$('#gpt-Variables').append('<input type="hidden" id="gptDFPTestId">');
			$('#gptDFPTestId').val(gptDFPTestId);
			gpt_Target("DFPTestId");
		}
	}
	//--------------------------------------------------------------------------
	function gpt_Target(targetName) {
		//Set page level target if value is found
		if ($('#gpt' + targetName).length > 0) {
			var targetValue = $('#gpt' + targetName).val();

			if (targetValue !== '') {
				targetValue = targetValue.replace(/[^-a-z0-9]/ig, '');	//Only Allow a-z, 0-9
				googletag.pubads().setTargeting(targetName.toLowerCase(), targetValue.toLowerCase());
				gptCustomTargeting_Debug = gptCustomTargeting_Debug + targetName.toLowerCase() + ' = ' + targetValue.toLowerCase() + '<br>';
			}
		}
	}
	//--------------------------------------------------------------------------
	function gpt_TargetAccessType() {
		var currentAccessType = $('#gptAccessType').val();
		if (typeof currentAccessType == "undefined") {
			currentAccessType = "";
		}
		if (currentAccessType.indexOf('ANON_IP') >= 0) {
			googletag.pubads().setTargeting("loggedin", "yes");
		}
		else {
			gpt_Target("LoggedIn");
		}
	}
	//--------------------------------------------------------------------------
	async function gpt_PreTearsheet(adTypeId) {
		if (typeof adTypeId === "undefined") {
			adTypeId = 0;
		}
		//Remove Elsevier Cookie Policy Notice
		if ($("#onetrust-consent-sdk").length > 0) {
			$("#onetrust-consent-sdk").remove();
		}

        //Mobile Nav
        $('#skip').remove();
        $('skipNavigationLink').remove();

		//JBS Modals
		if ($('.leadinModal').length > 0) {
			$('.leadinModal').remove();
		}
		if ($('.leadinModal').length > 0) {
			$('.leadinModal').remove();
		}
		if ($('.usabilla__overlay').length > 0) {
			$('.usabilla__overlay').remove();
		}

		//Pendo
		if ($('._pendo-step-container').length > 0) {
			$('._pendo-step-container').remove();
		}
		if ($('._pendo-guide-tt_').length > 0) {
			$('._pendo-guide-tt_').remove();
		}
		if ($('._pendo-image').length > 0) {
			$('._pendo-image').remove();
		}
		if ($('._pendo-badge-image').length > 0) {
			$('._pendo-badge-image').remove();
		}

		switch (gpt_AdFile) {
			case "CELLJOURNAL":
			case "CELLBUCKET":
				//Mobile Bug Fix on Cell
				if (parseInt(adTypeId) == 5) {
					$('.widget-horizontalAd').css('position', 'static');
					$('.body-horizontalAd').css('display', 'block');
				}
				break;
			case "JBS":
			case "LANCET":
				objLeaderboard = ".container-leaderboard";
				offset = 53;
				switch (parseInt(adTypeId)) {
					case 3: //MPU
					case 4: //SkyScraper
						//BUGFIX: Show Sidebar ad for JBS
						//skyscraper & mpu
						if ($('.widget-verticalAd').length > 0) {
							$('.widget-verticalAd').css('display', 'block');
						}
						$('#backgroundLayer').css('display', 'none');   //Hide any Prestitial
						break;
					case 6:     //Prestitial
						$('#backgroundLayer').css('display', 'block');
						break;
					default:
						$('#backgroundLayer').css('display', 'none');   //Hide any Prestitial
						break;
				}
				break;
		}
	}
	//--------------------------------------------------------------------------
	function gpt_Sleep(ms) {
		return new Promise(resolve => setTimeout(resolve, ms));
	}
	//--------------------------------------------------------------------------
	async function gpt_ScrollToAd(adTypeId) {
		if (typeof adTypeId === "undefined") {
			adTypeId = 0;
		}
		var adId = "";
		var offset = 0;
		var objLeaderboard = "";
		var gpt_ScrollToAd = true;

		switch (gpt_AdFile) {
			case "PU": //PracticeUpdate
				var hideOtherAds = true;
				$('#modalPrestitial').modal('hide');
				$('.jumbotron-welcome').hide();
				objLeaderboard = ".container-leaderboard";
				offset = 53;
				switch (parseInt(adTypeId)) {
					case 2:
						//LeaderBoard, no Scroll Needed
						gpt_ScrollToAd = false;
						adId = "adSlot1";
						break;
					case 3:
						//MPU
						adId = "adSlot2";
						break;
					case 4:
						//SkyScraper
						adId = "adSlot3";
						$('.most-read').hide();
						break;
					case 5:
						//Mobile, no Scroll Needed
						gpt_ScrollToAd = false;
						adId = "adSlot1";
						break;
					case 6:
						//Interstitial
						gpt_ScrollToAd = false;
						adId = "adSlot15";
						await gpt_Sleep(1000);
						$(objLeaderboard).hide();
						$('#modalPrestitial').unbind();
						$('#modalPrestitial').modal();
						break;
					case 7:
						//Native Ad
						break;
					case 10:
						//Master Companion
						hideOtherAds = false;
						gpt_ScrollToAd = false;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						adId = "adSlot1";
						break;
				}
				if (hideOtherAds) {
					//Hide all other ads
					$('.ad-container').each(function (i, obj) {
						if (this.id !== adId) {
							var tempParent = this.id;
							tempParent = tempParent.replace("adSlot", "adParent");
							$("#" + tempParent).addClass('hide');
						}
					});
				}
				break;
			case "JBS":
				objLeaderboard = ".js__adToHide";
				offset = 80;
				switch (parseInt(adTypeId)) {
					case 2:
						//Leaderboard
						gpt_ScrollToAd = false;
						$(".widget-verticalAd").hide();
						break;
					case 5:
						//Mobile
						//Scroll to Top
						gpt_ScrollToAd = false;
						break;
					case 3: //MPU
					case 4: //SkyScraper
						$(objLeaderboard).hide();
						adId = $('.gpt-ad-defined[data-adsize="sidebar"').attr('id');
						break;
					case 10:
						//MasterCompanion
						gpt_ScrollToAd = false;
						break;
					case 13:	    //Phone - Portrait
					case 14:        //Phone - Landscape
						$(objLeaderboard).hide();
						offset = 60;
						adId = 'gpt-Interscroller';
						gpt_ScrollToAd = true;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						break;
				}
				break;
			case "CELLJOURNAL":
			case "CELLBUCKET":
				objLeaderboard = ".js__adToHide";
				offset = 260;
				switch (parseInt(adTypeId)) {
					case 2:
						//Leaderboard
						gpt_ScrollToAd = false;
						$('.body-verticalAd').hide();
						break;
					case 5:
						//Mobile
						//Scroll to Top
						gpt_ScrollToAd = false;
						break;
					case 3: //MPU
					case 4: //SkyScraper
						$(objLeaderboard).hide();
						adId = $('.gpt-ad-defined[data-adsize="boombox"').attr('id');
						break;
					case 10:
						//MasterCompanion
						gpt_ScrollToAd = false;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						break;
				}
				//Hide the second ad if found
				var tCount = 0;
				$(".widget.literatumAd.none.GPT-ad-placeholder.widget-horizontalAd.widget-compact-all").each(function (index) {
					tCount += 1;
					if (tCount !== 1) {
						$(this).hide();
					}
				});
				break;
			case "LANCET":
				objLeaderboard = ".js__adToHide";
				offset = 80;
				switch (parseInt(adTypeId)) {
					case 2:
						//Leaderboard
						gpt_ScrollToAd = false;
						$('.body-verticalAd').hide();
						break;
					case 5:
						//Mobile
						//Scroll to Top
						gpt_ScrollToAd = false;
						break;
					case 3: //MPU
					case 4: //SkyScraper
						$(objLeaderboard).hide();
						adId = $('.gpt-ad-defined[data-adsize="skyscraper"').attr('id');
						break;
					case 10:
						//MasterCompanion
						gpt_ScrollToAd = false;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						break;
				}
				break;
		}

		//Scroll to Top
		var gpt_container = $("html,body");
		gpt_container.animate({ scrollTop: 0, scrollLeft: 0 });

		if (gpt_ScrollToAd) {
			//If Adexists, scroll to it
			var gpt_scrollTo = $('#' + adId);
			if (gpt_scrollTo.length > 0) {
				//Scroll to the Ad
				var gpt_topPosition = gpt_scrollTo.offset().top - gpt_container.offset().top + gpt_container.scrollTop();
				if (gpt_topPosition > 0) {
					gpt_container.animate({ scrollTop: gpt_topPosition, scrollLeft: 0 });
					if (gpt_topPosition - offset > 0 && offset > 0) {
						gpt_container.animate({ scrollTop: gpt_topPosition - offset, scrollLeft: 0 });
					}
				}
			}
		}
	}
	//--------------------------------------------------------------------------
	async function gpt_CheckForCookiePolicy() {
		var returnVal = false;
		if ($('#onetrust-accept-btn-handler').length > 0) {
			$('#onetrust-accept-btn-handler').click();
			returnVal = true;
		}
		return returnVal;
	}
	//--------------------------------------------------------------------------
	function gpt_InitAds() {
		var gpt_ResponsiveHomepage = false;

		switch (gpt_AdFile) {
			case "JBS":
			case "CELLJOURNAL":
			case "CELLBUCKET":
				//Responsive Check
				if ($('#hdrHomeBrand').length > 0) {
					gpt_ResponsiveHomepage = true;
				}
				break;
		}

		//InterScroller Check
		if (gpt_AdFile === "JBS") {
			var gpt_InterScrollerEnabled = false;
			var gpt_InterScrollerInsertObj;

			if ($("#articleHeader").length > 0) {
				//It's an article Page
				if ($('.article__sections > section').length > 0) {
					gpt_InterScrollerEnabled = true;
					gpt_InterScrollerInsertObj = $(".article__sections > section");
				}
			}
			else {
				if ($('.toc__section').length > 0) {
					//It's a TOC Page
					gpt_InterScrollerEnabled = true;
					gpt_InterScrollerInsertObj = $(".toc__section").first();
				}
			}

			//Add the Interscroller
			if (gpt_InterScrollerEnabled) {
				gpt_InterScrollerInsertObj.first().append($("#gpt-Interscroller").clone());
				$("#gpt-Interscroller").remove();
				$(".gpt-adinterscroller").addClass("gpt-ad");
			}
		}

		googletag.cmd.push(function () {
			var gptBoomBoxSizeMapping;
			var gptFluidMapping;
			var gptHomePageSidebarMapping;
			var gptCellHomePageSidebarMapping;
			var gptInterscrollerMapping;
			var gptLeaderboardSizeMapping;
			var gptMobileSizeMapping;
			var gptPrestitialMapping;
			var gptRectangleShortSizeMapping;
			var gptRectangleSizeMapping;
			var gptSideBarSizeMapping;
			var gptTallSizeMapping;
			var gptWideSizeMapping;
			var gptWideRectangleSizeMapping;

			//***********************************************
			//Start Define Size mapping based on client Viewport
			//***********************************************
			if (gpt_AdFile === 'PU') {
				gptTallSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[300, 600], [160, 600], [120, 600]]).
					addSize([0, 0], []).
					build();

				gptRectangleSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[300, 250]]).
					addSize([0, 0], []).
					build();

				gptRectangleShortSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[300, 250], [320, 50], [300, 50]]).
					addSize([0, 0], [[300, 250], [320, 50], [300, 50]]).
					build();

				gptWideSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[728, 90]]).
					addSize([760, 0], [[728, 90], [320, 50], [300, 50]]).
					addSize([0, 0], [[320, 50], [300, 50]]).
					build();

				gptWideRectangleSizeMapping = googletag.sizeMapping().
					addSize([992, 0], []).
					addSize([760, 0], [[320, 50], [300, 50], [300, 250]]).
					addSize([0, 0], [[320, 50], [300, 50], [300, 250]]).
					build();

				gptMobileSizeMapping = googletag.sizeMapping().
					addSize([0, 0], [[320, 50], [300, 50]]).
					build();

				gptFluidMapping = googletag.sizeMapping().
					addSize([0, 0], ['fluid']).
					build();

				gptPrestitialMapping = googletag.sizeMapping().
					addSize([760, 0], [[1, 1]]).
					build();
			}
			else {
				//JBS-CELL-LANCET
				gptLeaderboardSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[728, 90]]).
					addSize([768, 0], [[728, 90]]).
					addSize([0, 0], [[320, 50], [300, 50]]).
					build();

				gptHomePageSidebarMapping = googletag.sizeMapping().
					addSize([2090, 795], [[300, 600], [300, 250], [160, 600], [120, 600]]).
					addSize([1910, 795], [[160, 600], [120, 600]]).
                    addSize([1740, 795], [[120, 600]]).
					addSize([0, 0], []).
					build();

				gptCellHomePageSidebarMapping = googletag.sizeMapping().
                    addSize([2180, 795], [[300, 600], [160, 600], [120, 600], [300, 250], [336, 280]]).
					addSize([2180, 445], [[336, 280]]).
                    addSize([2090, 795], [[300, 600], [160, 600], [120, 600], [300, 250]]).
                    addSize([2090, 445], [[300, 250]]).
					addSize([1830, 795], [[160, 600], [120, 600]]).
                    addSize([1740, 795], [[120, 600]]).
					addSize([0, 0], []).
					build();

				gptSideBarSizeMapping = googletag.sizeMapping().
					addSize([1140, 795], [[300, 600], [160, 600], [120, 600], [300, 250]]).
					addSize([1140, 445], [[300, 250]]).
					addSize([0, 0], []). 				
					build();

				gptBoomBoxSizeMapping = googletag.sizeMapping().
					addSize([768, 0], [[300, 250], [336, 280]]).
					addSize([0, 0], []). 						
					build();

				gptFluidMapping = googletag.sizeMapping().
					addSize([0, 0], ['fluid']).
					build();

				gptPrestitialMapping = googletag.sizeMapping().
					addSize([600, 700], [[1, 1]]).
					addSize([0, 0], []).
					build();

				gptInterscrollerMapping = googletag.sizeMapping().
					addSize([1140, 445], []).
					addSize([1024, 768], [[1024, 768]]).
					addSize([768, 1024], [[768, 1024]]).
					addSize([480, 320], [[480, 320]]).
					addSize([320, 480], [[320, 480]]).
					build();
			}
			//***********************************************
			//End Define Size mapping based on client Viewport
			//***********************************************

			//***********************************************
			//Start detection for ad slots on page
			//***********************************************
			var gptMasterPage = gpt_CheckForNULL($('#gptPage').val());
			var gptMasterSite;

			if (gpt_AdFile === 'PU') {
				gptMasterSite = $('#gptSite').val();
			}
			else {
				//JBS-CELL-LANCET
				gptMasterSite = "dev";

				if (gpt_IsProd) {
					gptMasterSite = gpt_CheckForNULL($('#gptSite').val());

					//If there is no master site, check for a defaultsite
					if (gptMasterSite == null) {
						gptMasterSite = gpt_CheckForNULL($('#gptDefaultSite').val());
					}
				}
				else {
					//Dev ENV
					//If the page has no gptSite Value, assume no value which will disable ads
					if (gpt_CheckForNULL($('#gptSite').val()) === null) {
						gptMasterSite = null;
					}
				}

				//If there is no master Page, check for a Page using the URL
				if (gptMasterPage === null) {
					gptMasterPage = GPT_GetPageByURL();
				}
			}

			$(".gpt-ad").each(function (index) {
				var gptPage = '';
				var adSizeMapping = '';
				var gptPOS;
				var gptTC;
				var gptNative;
				var adId;
				var gptSite;
				var tempSlot = null;


                if (gpt_IsProd) {
					if (gpt_AdFile !== "CELLJOURNAL") {
						//Check if the ad slot has it's own site defined
						if (typeof $(this).data('site') === 'undefined') {
							gptSite = gptMasterSite;
						}
						else {
							gptSite = gpt_CheckForNULL($(this).data('site'));
						}
					}
					else {
						gptSite = gptMasterSite;
					}
                }
                else {
					gptSite = gptMasterSite;
                }



				if (gpt_AdFile === 'PU') {
					adId = $(this).attr('id');
				}
				else {
					//JBS-CELL-LANCET
					adId = 'gpt-ad-' + index + 1;
				}

				if (gpt_CheckForNULL(gptSite) !== null) {
					gptSite_Debug = gptSite;

					gptPOS = parseInt($(this).data('pos')) || -1;
					if (gptPOS === -1) {
						gptPOS = index + 1;
					}

					//Check if the ad slot has it's own page defined
					gptPage = $(this).data('page');
					if (gpt_CheckForNULL(gptPage) == null) {
						//Use the Master Page
						gptPage = gptMasterPage;
					}

					//If the page is not null, preceed it with a slash
					if (gptPage !== null) {
						gptPage = '/' + gptPage;
					}
					else {
						gptPage = '';
					}
					gptPage_Debug = gptPage;


					gptTC = $(this).data('tc');
					if (typeof gptTC === "undefined") {
						gptTC = '';
					}

					gptNative = $(this).data('native');
					if (typeof gptNative === "undefined") {
						gptNative = '';
					}

					//Determine sizemapping based on data-adsize
					if (gpt_AdFile === 'PU') {
						switch ($(this).data('adsize')) {
							case 'tall':
								adSizeMapping = gptTallSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600]], adId);
								break;
							case 'rectangle':
								adSizeMapping = gptRectangleSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250]], adId);
								break;
							case 'rectangleshort':
								adSizeMapping = gptRectangleShortSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250], [320, 50], [300, 50]], adId);
								break;
							case 'wide':
								adSizeMapping = gptWideSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[728, 90], [320, 50], [300, 50]], adId);
								break;
							case 'widerectangle':
								adSizeMapping = gptWideRectangleSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[320, 50], [300, 50], [300, 250]], adId);
								break;
							case 'mobile':
								adSizeMapping = gptMobileSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[320, 50], [300, 50]], adId);
								break;
							case 'fluid':
								adSizeMapping = gptFluidMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, 'fluid', adId);
								break;
							case 'prestitial':
								adSizeMapping = gptPrestitialMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[1, 1]], adId);
								break;
						}
					}
					else {
						//JBS-CELL-LANCET
						switch ($(this).data('adsize')) {
							case 'cell-leaderboard':
							case 'leaderboard':
								adSizeMapping = gptLeaderboardSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[728, 90], [320, 50], [300, 50]], adId);
								break;
							case 'bottom':
								if (gpt_ResponsiveHomepage) {
									adSizeMapping = gptHomePageSidebarMapping;
									tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600], [300, 250]], adId);
								}
								else {
									adSizeMapping = gptBoomBoxSizeMapping;
									tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250]], adId);
								}
								break;
							case 'boombox':
								if (gpt_AdFile === "CELLJOURNAL" || gpt_AdFile == "CELLBUCKET") {
									if (gpt_ResponsiveHomepage) {
										adSizeMapping = gptCellHomePageSidebarMapping;
										tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600], [300, 250], [336, 280]], adId);
									}
									else {
										adSizeMapping = gptBoomBoxSizeMapping;
										tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250], [336, 280]], adId);
									}
								}
								else {
									adSizeMapping = gptBoomBoxSizeMapping;
									tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250]], adId);
								}
								break;
							case 'interscroller':
								adSizeMapping = gptInterscrollerMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[320, 480], [480, 320], [768, 1024], [1024, 768]], adId);
								break;
							case 'sidebar':
							case 'skyscraper':
								adSizeMapping = gptSideBarSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600], [300, 250]], adId);
								break;
							case 'native':
							case 'fluid':
								adSizeMapping = gptFluidMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, 'fluid', adId);
								break;
							case 'prestitial':
								adSizeMapping = gptPrestitialMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, 'fluid', adId);
								break;
						}
					}
				}

				//Ignore any invalid adsizes that could be on the page
				if (adSizeMapping !== '') {
					$(this).addClass('gpt-ad-defined');
					if (gpt_AdFile !== 'PU') {
						$(this).attr('id', adId);
					}
					tempSlot.defineSizeMapping(adSizeMapping).addService(googletag.pubads()).setTargeting("pos", gptPOS);

					if (gptTC !== '') {
						tempSlot.setTargeting("tc", gptTC);
					}
					if (gptNative !== '') {
						tempSlot.setTargeting("native", gptNative);
					}
					gptEvent_Debug = gptEvent_Debug + $(this).data('adsize') + ' Requested<br>';
				}
				else {
					gptInvalidAdSize_Debug = gptInvalidAdSize_Debug + "Invalid Size Found, Id: " + $(this).prop('id') + ', adSize: ' + $(this).data('adsize') + '<br>';
					$(this).remove();
				}
			});
			if (gpt_CheckForNULL(gptInvalidAdSize_Debug) === null) {
				gptInvalidAdSize_Debug = "None Detected";
			}
			//***********************************************
			//End detection for ad slots on page
			//***********************************************

			//***********************************************
			//Start page level targeting
			//***********************************************
			if (gpt_AdFile !== 'PU') {
				//Logged in or IP Access
				gpt_TargetAccessType();

				//Check for Screenshot Client
				gpt_CheckSSC();
			}

			//Check for DFP Test
			gpt_GetDFPTestId();

			//Check for Additional Targeting
			gpt_GetAdditionalTargeting();

			if (gpt_IsProd && gptContextualEnabled) {
				var gpt_ContextualRetry = false;
				//Check for Contextual Targeting
				if (gs_channels !== "DEFAULT") {
					var gpt_ContextualCategories = Array.from(gs_channels);
					for (var i = 0; i < gpt_ContextualCategories.length; i++) {
						if (gpt_ContextualCategories[i] == 'gx_retry') {
							gpt_ContextualRetry = true;
						}
						if (!gpt_ContextualCategories[i].startsWith("custom_")) {
							//Remove any tag that is not custom
							gpt_ContextualCategories.splice(i, 1);
							i--;
						}
						else {
							//Strip the 'custom_' from the tag
							gpt_ContextualCategories[i] = gpt_ContextualCategories[i].replace("custom_", "")
						}
					}
					if (gpt_ContextualCategories.length > 0) {
						googletag.pubads().setTargeting("gs_cat", gpt_ContextualCategories);
					}
					else {
						gpt_ContextualCategories = 'none';
					}
					if (gpt_ContextualRetry) {
						gpt_ContextualCategories = "*Indexing*";
					}
					gptContextualResult_Debug = gpt_ContextualCategories;
				}
				else {
					gptContextualResult_Debug = 'No segments found';
				}
			}
			else {
				//Not Prod || Contextual not enabled
				gptContextualResult_Debug = 'Not enabled';
				
				if (!gpt_IsProd) {
                    var gpt_DevSite = gpt_CheckForNULL($('#gptSite').val(), "dev");
                    $('#gpt-Variables').append('<input type="hidden" id="gptEnvironment" value="' + gpt_DevSite + '">');
                    gpt_Target("Environment");
                }

			}
			//***********************************************
			//End page level targeting
			//***********************************************

			//Enable google dfp services
			if (parseInt($('#gptLazyLoading').val()) === 1) {
				googletag.pubads().enableLazyLoad({
					fetchMarginPercent: 200,
					renderMarginPercent: 0,
					mobileScaling: 2.0
				});
			}
			else {
				googletag.pubads().enableSingleRequest();
			}

			//Add Listener for RenderEnded
			googletag.pubads().addEventListener('slotRenderEnded', function (event) {
				if (!event.isEmpty) {
					if (typeof LocalAdLoaded === "function") {
						LocalAdLoaded(event);
					}
					if (gpt_ResponsiveHomepage) {
						if (typeof positionAd === "function") {
							positionAd();
						}
					}
					if (event.size[0] == "480" || event.size[1] == "480") {
						$('.gpt-InScrollBox').css('display', 'block');
						gptEvent_Debug = gptEvent_Debug + 'Interscroller ' + event.size + ' rendered: ' + gpt_GetElapsedTime() + '<br>';
					}
					else {
						gptEvent_Debug = gptEvent_Debug + event.size + ' rendered: ' + gpt_GetElapsedTime() + '<br>';
					}

				}
			});

			if (gpt_AdFile !== 'PU') {
				googletag.pubads().collapseEmptyDivs();
			}
			googletag.enableServices();

			//Populate Defined Ad Slots
			$(".gpt-ad-defined").each(function (index) {
				var adId = $(this).attr('id');
				googletag.cmd.push(function () { googletag.display(adId); });
			});

			gptStatus_Debug = 'GPT Complete';
		});
	}
	//--------------------------------------------------------------------------
	function gpt_DisplayDebug() {
		//Clone the template and append to Body
		$(document.body).append($('#gpt-DebugDiv').clone());
		$('#gpt-DebugDiv').remove();

		//Populate the Values
		gpt_LoadDebugValues();

		//Display and scroll to debug section
		$('.gpt-DebugDiv').css('display', 'block');
		window.scrollTo(0, document.body.scrollHeight);
	}
	//--------------------------------------------------------------------------
	function gpt_LoadDebugValues() {
		$('.gpt-DebugBtn').blur();
		$('.gptVersion_Debug').text(gpt_Version);
		$('.gptAdFile_Debug').html(gpt_AdFile);
		$('.gptEvent_Debug').html(gptEvent_Debug);
		$('.gptCustomTargeting_Debug').html(gptCustomTargeting_Debug);
		$('.gptSite_Debug').text(gptSite_Debug);
		$('.gptContextualResult_Debug').text(gptContextualResult_Debug);
		$('.gptContextualInit_Debug').text(gptContextualInit_Debug);
		$('.gptContextualURL_Debug').text(gptContextualURL_Debug);
		$('.gptContextualReady_Debug').text(gptContextualReady_Debug);
		$('.gptMoatReady_Debug').text(gptMoatReady_Debug);
		$('.gptMoatInit_Debug').text(gptMoatInit_Debug);
		$('.gptTestENV_Debug').text(gptTestENV_Debug);
		$('.gptDisableAds_Debug').text(gptDisableAds_Debug);
		$('.gptStatus_Debug').text(gptStatus_Debug);
		$('.gptPage_Debug').text(gptPage_Debug);
		$('.gptInvalidAdSize_Debug').html(gptInvalidAdSize_Debug);
	}
	//--------------------------------------------------------------------------
	function gpt_GrapeShotDebug() {
		$('.gpt-DebugBtn').blur();
		window.open("http://admin-elsevier.grapeshot.co.uk/custom/bookmarklet.cgi?blocking=1&vn=2&url=" + encodeURIComponent(document.location), "GRAPESHOT_POPUP", "width=650,height=600,scrollbars=yes,screenX=40,screenY=50,menubar=no,toolbar=no,resizable=yes", "Custom").focus();
	}
	//--------------------------------------------------------------------------
	function gpt_ShowConsole() {
		$('.gpt-DebugBtn').blur();
		googletag.openConsole();
	}
	//--------------------------------------------------------------------------
</script>
        </div>

            
                </div>
            

        

    

        


    

            

            

            <div data-widget-id="912135d6-7135-4d45-b457-b503cacc3c8b" data-widget-name="ux3-layout-widget"><header class="header base fixed">
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none adplaceholder adplaceholder--header js__adToHide widget-horizontalAd" id="9d6062d7-fad6-4e64-81cc-7dd25d99b23d">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-horizontalAd "><div class="horizontalAdContainer">
    <div class="horizontalAdLabelContainer">
        <div class="horizontalAdLabel">
                <img src="/templates/jsp/_style2/_marlin/images/bg_adHoriz.gif" alt="Advertisement">
        </div>
    </div>
    <div class="horizontalAd">
        





        <div class="pb-ad">
            <!-- Beginning Async AdSlot 1 for Ad unit els.cellpress/cell  ### size: [[728,90]] -->
<div class="gpt-ad" data-adsize="cell-leaderboard" data-page="" data-site="cell" data-pos="1"></div>
<!-- End AdSlot 1 -->
        </div>
    

    </div>
</div></div>
                </div>
                </div>

        










        <div data-widget-def="ux3-layout-widget" data-widget-id="3a83e2a6-2e7e-4931-892c-4ef43b6e3312" class="header__wrapper clearfix" data-db-parent-of="quickSearch"><div id="brandingRow"><a href="/" data-aa-name="science that inspires" data-aa-region="cp global header" title="Visit Cell.com">Science that inspires</a><p id="brandingRowRight"><a href="/" data-aa-name="cell press endorsement" data-aa-region="cp global header" title="Visit Cell.com">A Cell Press journal</a></p></div>
        
    

        


    

            

            

            <div data-widget-id="3a83e2a6-2e7e-4931-892c-4ef43b6e3312" data-widget-name="ux3-layout-widget"><div class="col-xs-6 header__left-side">
    

        


    

            

            
                <a href="/home" id="cpIconLnk" data-aa-name="cell press logo" data-aa-region="cp global header" title="Cell Press" target="_blank"><svg id="cp_logo" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 100 98.1" style="enable-background:new 0 0 100 98.1;" xml:space="preserve" height="100" width="98.1"><path style="fill:#FFFFFF;" d="M69.09,0.09c-4.01-0.6-7.83,1.94-8.78,5.89l-3.14,13.11c-0.76,3.19-3.44,5.56-6.7,5.93l-18.6,2.13          C11.83,29.31-2.53,47.66,0.37,67.78c2.3,15.96,15.18,27.77,30.54,30.08c4.01,0.6,7.83-1.94,8.78-5.89l3.14-13.11 c0.76-3.19,3.44-5.56,6.69-5.93l18.6-2.13c20.04-2.17,34.41-20.52,31.5-40.65C97.32,14.21,84.45,2.39,69.09,0.09z M65.06,47.2  l-11.75,1.4c-1.98,0.21-3.62,1.65-4.08,3.59l-3.43,14.3c-1.01,4.22-4.5,7.39-8.8,8c-6.66,0.72-12.63-4.2-13.15-10.92                c-0.51-6.5,4.6-12.11,11.08-12.81l11.75-1.39c1.98-0.22,3.62-1.65,4.08-3.59l3.43-14.3c1.01-4.22,4.5-7.39,8.8-8          c6.66-0.72,12.63,4.19,13.15,10.92C76.66,40.89,71.55,46.5,65.06,47.2z"></path></svg></a><div class="journal-logos">
            

            <span data-widget-id="35d8d734-3aa0-4ec8-9e2d-14babe803594" data-widget-name="ux3-general-image"><a href="/cell/home" title="Cell"><img alt="Cell" src="/pb-assets/ux3/logos/cell/Cell_white-1532516611593.svg"></a></span>

            
                </div>
            

        

    

        


    

            

            
                <div class="journal-logos">
            

            <span data-widget-id="3ffcbf1f-79fd-4ba3-85b5-bef10a63475f" data-widget-name="ux3-general-image"><a href="https://www.cell.com/open-access?utm_source=oa-label&amp;__hstc=109691226.e4e812b53ff324eccbdbdb2e4fb9a6ab.1644503737405.1644524234622.1644587109507.4&amp;__hssc=109691226.7.1644587109507&amp;__hsfp=3253153019" title="This journal offers authors two options (open access or subscription) to publish research"><img alt="This journal offers authors two options (open access or subscription) to publish research" src="/pb-assets/ux3/logos/cell/Supports_open_access_border-1645554937897.svg"></a></span>

            
                </div>
            

        
</div><div class="col-xs-6">
    

        


    

            

            

            <div data-widget-id="2555f6db-b333-4edb-8b15-ef9213f1f0a5" data-widget-name="ux3-layout-widget"><div class="header__nav row">
    

        


    

            

            

            <ul data-widget-id="52432d24-0331-4c9d-85f3-2033edc7208b" data-widget-name="UX3ProfileLinksWidget" class="rlist rlist--inline profile-links"><li class="profile-links__submit"><a data-adobe-name="Publish" class="profile-links__submit__btn" title="Submit" href="https://www.editorialmanager.com/cell" target="_self" data-aa-conversiondriver="jbs:global-header:submitpage" data-profile-link="true">Submit</a></li><li class="profile-links__login-dropdown"><div class="profile-links__user-list__wrapper"><button data-db-target-for="loginmenu-948070" aria-expanded="false" aria-controls="login-links-948070" title="Log in links dropdown" class="profile-links__username"><span>Log in</span></button><div data-db-target-of="loginmenu-948070" id="login-links--948070" class="profile-links__user-list__holder dropBlock__holder"><ul class="rlist profile-links__list"><li class="profile-links__login"><a data-adobe-name="Log In" class="profile-links__login__btn" title="Log in to Cell.com" href="https://www.cell.com/action/idLogin?type=login&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2901065-5" target="_self" data-profile-link="true">Log in</a></li><li class="profile-links__institution"><a data-adobe-name="Get institutional access" class="profile-links__institution__btn" title="Institutional access" href="https://www.cell.com/action/idLogin?type=institution&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2901065-5" target="_self" data-profile-link="true">Get Institutional Access</a></li></ul></div></div></li><li class="profile-links__register"><a data-adobe-name="Register" class="profile-links__register__btn" title="Register a username and password for free" href="https://www.cell.com/action/idLogin?type=registration&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2901065-5" target="_self" data-profile-link="true">Register</a></li><li class="hidden-lg"> <div class="profile-links__controls"> <button aria-expanded="false" aria-controls="links-dialog-948070" data-opens="#links-dialog-948070" class="profile-links__btn--open"><span class="sr-only">open links dialog</span><i aria-hidden="true" class="icon-gizmo-person"></i></button></div><dialog id="links-dialog-948070" aria-modal="true"><button data-closes="#links-dialog-948070" class="profile-links__btn--close"><span class="sr-only">close links dialog</span><i aria-hidden="true" class="icon-close-solid"></i></button><ul>   <li class="profile-links__submit"><a data-adobe-name="Publish" class="profile-links__submit__btn" title="Submit" href="https://www.editorialmanager.com/cell" target="_self" data-aa-conversiondriver="jbs:global-header:submitpage" data-profile-link="true">Submit</a></li><li class="profile-links__login"><a data-adobe-name="Log In" class="profile-links__login__btn" title="Log in to Cell.com" href="https://www.cell.com/action/idLogin?type=login&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2901065-5" target="_self" data-profile-link="true">Log in</a></li><li class="profile-links__institution"><a data-adobe-name="Get institutional access" class="profile-links__institution__btn" title="Institutional access" href="https://www.cell.com/action/idLogin?type=institution&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2901065-5" target="_self" data-profile-link="true">Get Institutional Access</a></li><li class="profile-links__register"><a data-adobe-name="Register" class="profile-links__register__btn" title="Register a username and password for free" href="https://www.cell.com/action/idLogin?type=registration&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2901065-5" target="_self" data-profile-link="true">Register</a></li></ul></dialog></li></ul>

            

        
</div></div>

            

        
</div></div>

            

        

        </div>
    

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget csasInstitutionBanner none  widget-none" id="51b7fa33-2137-4f58-9e98-9746c4400513" style="display: none;">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none "><div class="ui-state-highlight ui-corner-all csas-institution-banner hidden" data-doi="10.1016/j.cell.2021.09.018">
    <span class="ui-icon ui-icon-close banner-close"></span>
    <p>
        Access provided by <span class="accountName"></span>
    </p>
</div>

<style>
    .csas-institution-banner {
        padding: 10px 0 0 10px;
        margin: 5px;
    }

    .csas-institution-banner.ui-state-highlight .ui-icon {
        background-image: url("/templates/jsp/css/jquery-ui-1.10.2/base/images/ui-icons_454545_256x240.png");
        margin-right: 5px;
        margin-top: -4px;
    }

    .csas-institution-banner .banner-close {
        float: right;
    }
</style>

<script>

    function resetBanner(doi) {
        $.get('/action/getAccount?doi=' + doi, function(data) {
            if (data) {
                $(".csas-institution-banner p .accountName").html(data);
                $(".csasInstitutionBanner").fadeIn();
                $(".csas-institution-banner").fadeIn();
            }
        });
    }

    var flag = 0;
    $(document).ready(function() {
        if(!$(".csas-institution-banner").is(":visible")){
            $(".csasInstitutionBanner").hide();
        }

        $(".banner-close").click(function() {
            $.get("/action/dismissBanner", function(data) {
                $(".csasInstitutionBanner").fadeOut(function () {
                    if(typeof UX !== 'undefined') {
                        UX.pageBody.init();
                    }
                });
            });
        });

        $(".articleContent .tabs li[role='tab'].showFullText").click(function() {
           flag = 1;
        });

        $(document).on("reloadTabsContent", function() {
            var isSelected = $(".articleContent .tabs li[role='tab'].showFullText").hasClass("active");
            if (isSelected) {
                var isHidden =  $(".csas-institution-banner").is(".hidden");
                if (!isHidden) {
                    resetBanner($(".csas-institution-banner").data("doi"));
                    console.log("fadeIn");
                }
            } else {
                $(".csas-institution-banner").fadeOut();
                console.log("fadeOut");
            }
            flag = 0;
        });

        $(document).on("click", ".articleContent .tabs li[role='tab']", function() {

        });
    });
</script></div>
                </div>
                </div>

        
</header></div>

            

        










        <div data-widget-def="graphQueryWidget" data-widget-id="2a125dae-6dbc-4e76-a75e-3e563f1d3a1d" class="ux3">
        
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget graphQueryWidget none top-padding-handler noMargin axel-mega-menu widget-none widget-compact-all fonts-loaded" id="2a125dae-6dbc-4e76-a75e-3e563f1d3a1d" style="padding-top: 115.594px;">
                
                
                    
                

                <div class="wrapped " id="CP TOP LEVEL NAV MENU">
                    
                    <div class="widget-body body body-none  body-compact-all"><nav id="mega-menu-nav" role="navigation" class="mega-menu-nav">
  <!--.skip//a(href='#searchText' aria-label='Skip menu') Skip menu
  -->
  <div class="mega-menu-nav__wrapper">
    <button class="mega-menu__toggle"><span class="sr-only">Main menu</span><i aria-hidden="true" class="icon-gizmo-hamburger"></i></button>
    <ul role="menubar" class="mega-menu rlist">
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-1" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-1" data-item-link="main-menu-link-1" data-index="0" class="mega-menu__item-link menu-link"><span>This journal</span></a>
        <div id="main-menu-panel-1" aria-labelledby="main-menu-link-1" data-item-wrapper="main-menu-link-1" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <div aria-label="Menu dropzone section" tabindex="-1" class="listDropZone">
    

        


    

            

            

            <div class="cover-image"><div class="cover-image__thumb"><a href="/issue/S0092-8674(24)X0012-4" class="menu-link"><img src="/cms/asset/atypon:cms:attachment:img:d5e6:rev:1748270185670-389:pii:S0092867424X00124/cover.tif.jpg" alt="Cover Image - Cell, Volume 188, Issue 11"></a></div><div class="cover-image__meta"></div></div>

            

        
</div>
            </li>
            <li style="grid-column: 2/3" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-1-2" class="sub-menu__heading">Read
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/newarticles" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Online now</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/current" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Current issue</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/archive" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Archive</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/action/showPreferences?menuTab=Alerts" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Publication alerts</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/sneakpeek" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Sneak Peek - preprints</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 3/5" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-1-3" class="sub-menu__heading">About
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-2">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/advisory-board" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Advisory board</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/aims" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Aims and scope</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/contact" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Contact</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/editors-and-staff" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Editors and staff</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/editorial-ethics" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Ethics</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/information-for-authors/permissions" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Permissions</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/reviewers" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>Reviewer instructions</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-2" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-2" data-item-link="main-menu-link-2" data-index="1" class="mega-menu__item-link menu-link"><span>Journals</span></a>
        <div id="main-menu-panel-2" aria-labelledby="main-menu-link-2" data-item-wrapper="main-menu-link-2" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/5" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-2-1" class="sub-menu__heading">Life &amp; medical sciences
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-4">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/home" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cancer-cell/home" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Cancer Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-chemical-biology" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Chemical Biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-genomics/home" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Genomics</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-host-microbe/home" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Host &amp; Microbe</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-metabolism/home" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Metabolism</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports/home" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-medicine/home" data-index="7" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Medicine</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-stem-cell/home" data-index="8" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Stem Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-systems/home" data-index="9" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Systems</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/current-biology/home" data-index="10" role="menuitem" class="sub-menu__item-link menu-link"><span>Current Biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/developmental-cell/home" data-index="11" role="menuitem" class="sub-menu__item-link menu-link"><span>Developmental Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/immunity/home" data-index="12" role="menuitem" class="sub-menu__item-link menu-link"><span>Immunity</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/med/home" data-index="13" role="menuitem" class="sub-menu__item-link menu-link"><span>Med</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-cell/home" data-index="14" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/neuron/home" data-index="15" role="menuitem" class="sub-menu__item-link menu-link"><span>Neuron</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/structure/home" data-index="16" role="menuitem" class="sub-menu__item-link menu-link"><span>Structure</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/ajhg/home" data-index="17" role="menuitem" class="sub-menu__item-link menu-link"><span>American Journal of Human Genetics (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/biophysj/home" data-index="18" role="menuitem" class="sub-menu__item-link menu-link"><span>Biophysical Journal (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/biophysreports/home" data-index="19" role="menuitem" class="sub-menu__item-link menu-link"><span>Biophysical Reports (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/hgg-advances/home" data-index="20" role="menuitem" class="sub-menu__item-link menu-link"><span>Human Genetics and Genomics Advances (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-plant/home" data-index="21" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Plant (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/molecular-therapy/home" data-index="22" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/methods/home" data-index="23" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy Methods &amp; Clinical Development (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/nucleic-acids/home" data-index="24" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy Nucleic Acids (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/oncology/home" data-index="25" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy Oncology (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/plant-communications/home" data-index="26" role="menuitem" class="sub-menu__item-link menu-link"><span>Plant Communications (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/stem-cell-reports/home" data-index="27" role="menuitem" class="sub-menu__item-link menu-link"><span>Stem Cell Reports (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/biochemical-sciences/home" data-index="28" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Biochemical Sciences</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/cancer/home" data-index="29" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Cancer</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/cell-biology/home" data-index="30" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Cell Biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/ecology-evolution/home" data-index="31" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Ecology &amp; Evolution</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/endocrinology-metabolism/home" data-index="32" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Endocrinology &amp; Metabolism</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/genetics/home" data-index="33" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Genetics</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/immunology/home" data-index="34" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Immunology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/microbiology/home" data-index="35" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Microbiology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/molecular-medicine/home" data-index="36" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Molecular Medicine</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/neurosciences/home" data-index="37" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Neurosciences</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/parasitology/home" data-index="38" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Parasitology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/pharmacological-sciences/home" data-index="39" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Pharmacological Sciences</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/plant-science/home" data-index="40" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Plant Science</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 5/6" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-2-2" class="sub-menu__heading">Physical sciences &amp; engineering
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-biomaterials/home" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Biomaterials</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-physical-science/home" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Physical Science</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/chem/home" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Chem</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/chem-catalysis/home" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Chem Catalysis</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/device/home" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Device</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/joule/home" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Joule</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/matter/home" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>Matter</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/newton/home" data-index="7" role="menuitem" class="sub-menu__item-link menu-link"><span>Newton</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/chemistry/home" data-index="8" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Chemistry</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 6/7" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-2-3" class="sub-menu__heading">Multidisciplinary
              </h4>
              <ul role="menubar" data-index="2" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-methods/home" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Methods</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-sustainability/home" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Sustainability</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/heliyon/home" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Heliyon</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/iscience/home" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>iScience</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/one-earth/home" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>One Earth</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/patterns/home" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Patterns</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/star-protocols/home" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>STAR Protocols</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/nexus/home" data-index="7" role="menuitem" class="sub-menu__item-link menu-link"><span>Nexus (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/the-innovation/home" data-index="8" role="menuitem" class="sub-menu__item-link menu-link"><span>The Innovation (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/biotechnology/home" data-index="9" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Biotechnology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/cognitive-sciences/home" data-index="10" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Cognitive Sciences</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-3" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-3" data-item-link="main-menu-link-3" data-index="2" class="mega-menu__item-link menu-link"><span>Publish</span></a>
        <div id="main-menu-panel-3" aria-labelledby="main-menu-link-3" data-item-wrapper="main-menu-link-3" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-3-1" class="sub-menu__heading">Authors
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="https://www.editorialmanager.com/cell?__hstc=25856994.7201670a4397fa03148684920d4a8d38.1672762013831.1694716659709.1694718909620.146&amp;__hssc=25856994.15.1694718909620&amp;__hsfp=1366844671" data-index="0" target="_blank" role="menuitem" class="sub-menu__item-link menu-link"><span>Submit article
                      <div class="external-icon"></div><span class="sr-only">External Link</span></span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/information-for-authors" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Information for authors</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/open-access" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Open access</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/multi-journal-submission" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Multi-Journal Submission</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/information-for-authors/star-authors-guide" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>STAR Methods</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 2/3" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-3-2" class="sub-menu__heading">Reviewers
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/reviewers" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Information for reviewers</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-4" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-4" data-item-link="main-menu-link-4" data-index="3" class="mega-menu__item-link menu-link"><span>News &amp; events</span></a>
        <div id="main-menu-panel-4" aria-labelledby="main-menu-link-4" data-item-wrapper="main-menu-link-4" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/newsroom" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Newsroom</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/events" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Events</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/symposia" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Symposia</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/webinars" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Webinars</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/consortia" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Consortia Hub</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-5" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-5" data-item-link="main-menu-link-5" data-index="4" class="mega-menu__item-link menu-link"><span>About Cell Press</span></a>
        <div id="main-menu-panel-5" aria-labelledby="main-menu-link-5" data-item-wrapper="main-menu-link-5" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-5-1" class="sub-menu__heading">Cell Press
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/about" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>About Cell Press</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/contact" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Contact us</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="https://service.elsevier.com/app/overview/cell-press/" data-index="2" target="_blank" role="menuitem" class="sub-menu__item-link menu-link"><span>Help &amp; support
                      <div class="external-icon"></div><span class="sr-only">External Link</span></span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/open-access" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Open access at Cell Press</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/diversity" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Inclusion and diversity</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-press-50" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>50th anniversary</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 2/3" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-5-2" class="sub-menu__heading">Hubs
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/biotechnology-bioengineering" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Biotechnology and bioengineering</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-press-chemistry" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Chemistry</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/clinical-translational" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Clinical and translational</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/covid-19" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>COVID-19</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/fundamental-biology" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Fundamental biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/sustainability" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Sustainability</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 3/4" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-5-3" class="sub-menu__heading">Careers
              </h4>
              <ul role="menubar" data-index="2" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="https://relx.wd3.myworkdayjobs.com/en-US/ElsevierJobs" data-index="0" target="_blank" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Press careers
                      <div class="external-icon"></div><span class="sr-only">External Link</span></span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
    </ul>
    <div class="mega-menu-nav__quick-search">
    

        


    

            

            

            <div data-widget-id="a1f05748-166f-41e0-b01a-ca8ac1cc95d4" data-widget-name="UX3QuickSearchWidget" class="quick-search"><div class="dropBlock"><form action="https://www.cell.com/action/doSearch?journalCode=cell" name="quickSearchBar" method="get" title="Quick Search"><input id="type" type="hidden" value="quicksearch" name="type"><div class="quick-search__toggle autoCompleteWrapper"><input id="searchText" type="text" name="text1" placeholder="Search for..." enterkeyhint="search" autocomplete="off" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" data-group-titles-conf="3" data-publication-items-conf="3" aria-label="Enter a search term" class="searchText searchText-autocomplete ui-autocomplete-input"><button type="submit" title="Search" aria-label="Submit the search form" class="btn quick-search__button icon-search"><span>Go search</span></button><a href="https://www.cell.com/search/advanced?SeriesKey=cell&amp;ISSN=0092-8674&amp;journalCode=cell" title="Advanced search" class="quick-search__advanced-link">Advanced search</a><ul id="ui-id-1" tabindex="0" class="ui-menu ui-widget ui-widget-content ui-autocomplete ui-front" unselectable="on" style="display: none;"></ul></div><div class="quick-search__box"><div class="quick-search__box-inner"><div class="quick-search__box-wrapper"><select name="field1" aria-label="Select a search area"><option value="AllField">All content</option><option value="Title">Article title</option><option value="Contrib">Authors</option><option value="Keyword">Keywords</option><option value="Abstract">Abstract</option><option value="AbstractTitleKeywordFilterField">Article title, abstract, keywords</option></select><input type="hidden" name="journalCode" value="cell"><div class="quick-search__scope-options"><span>Search within:</span><label title="This journal" class="radio--primary"><input type="radio" name="SeriesKey" id="searchScopeSeries-a1f05748-166f-41e0-b01a-ca8ac1cc95d4" title="This journal" value="cell" checked="checked" aria-label="Search within current journal"><span>This journal</span></label><label title="Full site" class="radio--primary"><input type="radio" name="SeriesKey" id="searchScopeFullSite-a1f05748-166f-41e0-b01a-ca8ac1cc95d4" value="" title="Full site" aria-label="Search within full site"><span>Full site</span></label></div><a href="https://www.cell.com/search/advanced?SeriesKey=cell&amp;ISSN=0092-8674&amp;journalCode=cell" title="Advanced search" class="quick-search__advanced-link">Advanced search</a></div><a href="javascript:void(0)" aria-label="Close search options" title="Close search options" class="quick-search__box-close"><i aria-hidden="true" class="icon-tools_close"></i></a></div></div><div class="quick-search__alert hidden"><p>Please enter a term before submitting your search.</p><a href="#" class="quick-search__alert--close"> Ok</a></div></form></div></div>

            

        
</div>
  </div>
</nav>
<div id="page-body-id" class="page-body-flag"></div></div>
                </div>
                </div>

        

        </div>
    

    

        


    

            

            

            <div data-widget-id="f8c6daf9-9df1-4fd9-ba3a-7f5d7b3d4466" data-widget-name="UX3HTMLWidget"><style>
.journal-logos:nth-of-type(2) img {
max-height: 30px !important;
}
.journal-logos:nth-of-type(3) img {
max-height: 30px !important;
}
.ux3 .journal-logos + .journal-logos {
margin-left: 30px;
}
.header__wrapper.clearfix .col-xs-6:first-of-type {
width: 55%;
}
.header__wrapper.clearfix .col-xs-6:nth-of-type(2) {
width: 45%;
}
@media screen and (max-width: 1000px) {
.ux3 .journal-logos + .journal-logos {
display: none;
}
.header__wrapper.clearfix .col-xs-6:first-of-type {
width: 50%;
}
.header__wrapper.clearfix .col-xs-6:nth-of-type(2) {
width: 50%;
}
}

.ux3 .journal-logos:first-of-type {
   max-width: 500px;
}

.profile-links {
   white-space: nowrap;
}

@media only screen and (min-width: 740px) and (max-width: 1600px) {
   .journal-logos:nth-of-type(2) img {
      max-width: 250px !important;
   }
}

@media only screen and (min-width: 1024px) and (max-width: 1600px) {
   .header__wrapper.clearfix .col-xs-6:first-of-type {
      width: 60% !important;
   }
   .header__wrapper.clearfix .col-xs-6:nth-of-type(2) {
      width: 40% !important;
   }
}

@media only screen and (max-width: 1240px) {
   .ux3 .journal-logos + .journal-logos {
      display: none !important;
   }
}

</style></div>

            

        
</div></div>

            
                </div>
            

        

        </div>
    

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget axelPublicationContent none  widget-none" id="cadc9b41-a542-448a-9de4-8d0803320174">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none "><link id="build-style-article" rel="stylesheet" type="text/css" href="/products/marlin/cell/releasedAssets/css/build-article-b26f3342b722959c7931.css"><article xmlns="http://www.w3.org/1999/xhtml" data-design="pill" data-has="right-rail" vocab="http://schema.org/" typeof="ScholarlyArticle" data-pii="S0092-8674(21)01065-5" lang="en" dir="ltr"><div data-core-wrapper="header"><header data-extent="frontmatter"><div class="core-container"><div data-article-access="full" data-article-access-type="open" class="meta-panel"><span class="meta-panel__left-content"><span class="meta-panel__type"><span>Article</span></span></span><span class="meta-panel__middle-content"><span class="meta-panel__volumeIssue"><a href="/cell/issue?pii=S0092-8674(20)X0023-7"><span>Volume 184</span>, <span>Issue 22</span></a></span><span class="meta-panel__pages"><span>p5635-5652.e29</span></span><span class="meta-panel__onlineDate">October 28, 2021</span></span><span class="meta-panel__right-content"><span class="meta-panel__access meta-panel__access--open"><span>Open access</span></span></span><div class="article-header__download-full-issue"><button title="Download full issue" type="submit" formaction="/action/downloadMultiplePdf?action=downloadIssuePdf&amp;journal=cell&amp;volume=184&amp;issue=22&amp;pii=S0092-8674%2820%29X0023-7" class="btn--download-issue btn--cancel download-full-issue"><i class="icon-download-new"></i><span>Download Full Issue</span></button><div class="table-of-content__alert table-of-content__alert--download"><p>Download started</p><a href="#" class="table-of-content__alert--close">Ok</a></div></div></div><h1 property="name">Enhanced prime editing systems by manipulating cellular determinants of editing outcomes</h1><div class="contributors"><span class="authors"><span role="list"><span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au1" aria-controls="axel_author_au1"><span property="givenName">Peter J.</span> <span property="familyName">Chen</span></a><div id="au1" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Peter J.</span> <span property="familyName">Chen</span> <a class="orcid-id" href="https://orcid.org/0000-0002-1094-7180" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0002-1094-7180</a></div><div class="content"><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">16</div><div id="fn1"><div id="ntpara0010" role="paragraph">These authors contributed equally</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Merkin Institute of Transformative Technologies in Healthcare, Broad Institute of Harvard and MIT, Cambridge, MA 02141, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Chemistry and Chemical Biology, Harvard University, Cambridge, MA 02138, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Howard Hughes Medical Institute, Harvard University, Cambridge, MA 02138, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Chen/Peter+J" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">2</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">3</a>,<a role="doc-noteref">16</a></sup></span> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au2" aria-controls="axel_author_au2"><span property="givenName">Jeffrey A.</span> <span property="familyName">Hussmann</span></a><div id="au2" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Jeffrey A.</span> <span property="familyName">Hussmann</span> <a class="orcid-id" href="https://orcid.org/0000-0002-7778-9893" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0002-7778-9893</a></div><div class="content"><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">16</div><div id="fn1"><div id="ntpara0010" role="paragraph">These authors contributed equally</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Cellular and Molecular Pharmacology, University of California, San Francisco, San Francisco, CA 94158, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Microbiology and Immunology, University of California, San Francisco, San Francisco, CA 94158, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Howard Hughes Medical Institute, University of California, San Francisco, San Francisco, CA 94158, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Whitehead Institute for Biomedical Research, Massachusetts Institute of Technology, Cambridge, MA 02142, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Hussmann/Jeffrey+A" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">4</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">5</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">6</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">7</a>,<a role="doc-noteref">16</a></sup></span> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au3" aria-controls="axel_author_au3"><span property="givenName">Jun</span> <span property="familyName">Yan</span></a><div id="au3" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Jun</span> <span property="familyName">Yan</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Molecular Biology, Princeton University, Princeton, NJ 08544, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Yan/Jun" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">8</a></sup></span><span data-displayed-on="all"> ∙ … </span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au4" aria-controls="axel_author_au4"><span property="givenName">Friederike</span> <span property="familyName">Knipping</span></a><div id="au4" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Friederike</span> <span property="familyName">Knipping</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Pediatrics, University of Minnesota, Minneapolis, MN 55454, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Center for Genome Engineering, University of Minnesota, Minneapolis, MN 55108, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Stem Cell Institute, University of Minnesota, Minneapolis, MN 55455, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Knipping/Friederike" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">9</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">10</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">11</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au5" aria-controls="axel_author_au5"><span property="givenName">Purnima</span> <span property="familyName">Ravisankar</span></a><div id="au5" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Purnima</span> <span property="familyName">Ravisankar</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Lewis-Sigler Institute for Integrative Genomics, Princeton University, Princeton, NJ 08544, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Ravisankar/Purnima" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">12</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au6" aria-controls="axel_author_au6"><span property="givenName">Pin-Fang</span> <span property="familyName">Chen</span></a><div id="au6" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Pin-Fang</span> <span property="familyName">Chen</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Human Neuron Core, Rosamund Stone Zander Translational Neuroscience Center, Boston Children’s Hospital, Boston, MA 02115, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Neurology, Boston Children’s Hospital, Boston, MA 02115, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Chen/Pin-Fang" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">13</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">14</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au7" aria-controls="axel_author_au7"><span property="givenName">Cidi</span> <span property="familyName">Chen</span></a><div id="au7" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au7"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Cidi</span> <span property="familyName">Chen</span> <a class="orcid-id" href="https://orcid.org/0000-0001-7112-2230" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0001-7112-2230</a></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Human Neuron Core, Rosamund Stone Zander Translational Neuroscience Center, Boston Children’s Hospital, Boston, MA 02115, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Neurology, Boston Children’s Hospital, Boston, MA 02115, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Chen/Cidi" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">13</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">14</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au8" aria-controls="axel_author_au8"><span property="givenName">James W.</span> <span property="familyName">Nelson</span></a><div id="au8" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au8"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">James W.</span> <span property="familyName">Nelson</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Merkin Institute of Transformative Technologies in Healthcare, Broad Institute of Harvard and MIT, Cambridge, MA 02141, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Chemistry and Chemical Biology, Harvard University, Cambridge, MA 02138, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Howard Hughes Medical Institute, Harvard University, Cambridge, MA 02138, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Nelson/James+W" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">2</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">3</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au9" aria-controls="axel_author_au9"><span property="givenName">Gregory A.</span> <span property="familyName">Newby</span></a><div id="au9" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au9"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Gregory A.</span> <span property="familyName">Newby</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Merkin Institute of Transformative Technologies in Healthcare, Broad Institute of Harvard and MIT, Cambridge, MA 02141, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Chemistry and Chemical Biology, Harvard University, Cambridge, MA 02138, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Howard Hughes Medical Institute, Harvard University, Cambridge, MA 02138, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Newby/Gregory+A" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">2</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">3</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au10" aria-controls="axel_author_au10"><span property="givenName">Mustafa</span> <span property="familyName">Sahin</span></a><div id="au10" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au10"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Mustafa</span> <span property="familyName">Sahin</span> <a class="orcid-id" href="https://orcid.org/0000-0001-7044-2953" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0001-7044-2953</a></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Human Neuron Core, Rosamund Stone Zander Translational Neuroscience Center, Boston Children’s Hospital, Boston, MA 02115, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Neurology, Boston Children’s Hospital, Boston, MA 02115, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Harvard Medical School, Boston, MA 02115, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Sahin/Mustafa" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">13</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">14</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">15</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au11" aria-controls="axel_author_au11"><span property="givenName">Mark J.</span> <span property="familyName">Osborn</span></a><div id="au11" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au11"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Mark J.</span> <span property="familyName">Osborn</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Pediatrics, University of Minnesota, Minneapolis, MN 55454, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Center for Genome Engineering, University of Minnesota, Minneapolis, MN 55108, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Stem Cell Institute, University of Minnesota, Minneapolis, MN 55455, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Osborn/Mark+J" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">9</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">10</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">11</a></sup></span></span> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au12" aria-controls="axel_author_au12"><span property="givenName">Jonathan S.</span> <span property="familyName">Weissman</span></a><div id="au12" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au12"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Jonathan S.</span> <span property="familyName">Weissman</span> <a class="orcid-id" href="https://orcid.org/0000-0003-2445-670X" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0003-2445-670X</a></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Cellular and Molecular Pharmacology, University of California, San Francisco, San Francisco, CA 94158, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Howard Hughes Medical Institute, University of California, San Francisco, San Francisco, CA 94158, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Whitehead Institute for Biomedical Research, Massachusetts Institute of Technology, Cambridge, MA 02142, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Weissman/Jonathan+S" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">4</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">6</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">7</a></sup></span> ∙ <span property="author" typeof="Person" role="listitem" class="corresponding-author"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au13" aria-controls="axel_author_au13"><span property="givenName">Britt</span> <span property="familyName">Adamson</span></a><div id="au13" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au13"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Britt</span> <span property="familyName">Adamson</span></div><div class="content"><div class="core-author-emails"><div class="label">Correspondence</div><div>Corresponding author</div><div><a id="eadd0010" href="mailto:badamson@princeton.edu" property="email" title="Send email to Britt Adamson">badamson@princeton.edu</a></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Molecular Biology, Princeton University, Princeton, NJ 08544, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Lewis-Sigler Institute for Integrative Genomics, Princeton University, Princeton, NJ 08544, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Adamson/Britt" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">8</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">12</a></sup> <a id="eadd0010" href="mailto:badamson@princeton.edu" property="email" title="Send email to Britt Adamson">badamson@princeton.edu</a></span> ∙ <span property="author" typeof="Person" role="listitem" class="corresponding-author"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au14" aria-controls="axel_author_au14"><span property="givenName">David R.</span> <span property="familyName">Liu</span></a><div id="au14" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au14"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">David R.</span> <span property="familyName">Liu</span> <a class="orcid-id" href="https://orcid.org/0000-0002-9943-7557" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0002-9943-7557</a></div><div class="content"><div class="core-author-emails"><div class="label">Correspondence</div><div>Corresponding author</div><div><a id="eadd0015" href="mailto:drliu@fas.harvard.edu" property="email" title="Send email to David R. Liu">drliu@fas.harvard.edu</a></div></div><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">17</div><div id="fn2"><div id="ntpara0015" role="paragraph">Lead contact</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Merkin Institute of Transformative Technologies in Healthcare, Broad Institute of Harvard and MIT, Cambridge, MA 02141, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Chemistry and Chemical Biology, Harvard University, Cambridge, MA 02138, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Howard Hughes Medical Institute, Harvard University, Cambridge, MA 02138, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Liu/David+R" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">2</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">3</a>,<a role="doc-noteref">17</a></sup> <a id="eadd0015" href="mailto:drliu@fas.harvard.edu" property="email" title="Send email to David R. Liu">drliu@fas.harvard.edu</a></span><span data-displayed-on="all"> … <span><span data-action="reveal" tabindex="0" role="listitem">Show more</span></span> </span><span data-displayed-on="none" data-on-display="all" aria-hidden="true"> <span data-action="hide" tabindex="0" role="listitem">Show less</span></span></span></span></div><div class="core-self-citation"><div class="expandable-triggers"><button aria-controls="core-affiliations-notes" aria-expanded="false" tabindex="0" role="button"><span>Affiliations &amp; Notes</span><i class="icon-arrow_d_n" aria-hidden="true"></i></button><button aria-controls="core-content-info" aria-expanded="false" tabindex="0" role="button"><span>Article Info</span><i class="icon-arrow_d_n" aria-hidden="true"></i></button><button aria-controls="core-linked-content" aria-expanded="false" tabindex="0" role="button"><span>Linked Articles (1)</span><i class="icon-arrow_d_n" aria-hidden="true"></i></button></div><div class="expandable-content"><div id="core-affiliations-notes" role="region" data-expandable="item" style="display: none;"><div class="affiliations"><div property="affiliation" typeof="Organization"><span class="label">1</span><span id="aff1" property="name">Merkin Institute of Transformative Technologies in Healthcare, Broad Institute of Harvard and MIT, Cambridge, MA 02141, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">2</span><span id="aff2" property="name">Department of Chemistry and Chemical Biology, Harvard University, Cambridge, MA 02138, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">3</span><span id="aff3" property="name">Howard Hughes Medical Institute, Harvard University, Cambridge, MA 02138, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">4</span><span id="aff4" property="name">Department of Cellular and Molecular Pharmacology, University of California, San Francisco, San Francisco, CA 94158, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">5</span><span id="aff5" property="name">Department of Microbiology and Immunology, University of California, San Francisco, San Francisco, CA 94158, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">6</span><span id="aff6" property="name">Howard Hughes Medical Institute, University of California, San Francisco, San Francisco, CA 94158, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">7</span><span id="aff7" property="name">Whitehead Institute for Biomedical Research, Massachusetts Institute of Technology, Cambridge, MA 02142, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">8</span><span id="aff8" property="name">Department of Molecular Biology, Princeton University, Princeton, NJ 08544, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">9</span><span id="aff9" property="name">Department of Pediatrics, University of Minnesota, Minneapolis, MN 55454, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">10</span><span id="aff10" property="name">Center for Genome Engineering, University of Minnesota, Minneapolis, MN 55108, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">11</span><span id="aff11" property="name">Stem Cell Institute, University of Minnesota, Minneapolis, MN 55455, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">12</span><span id="aff12" property="name">Lewis-Sigler Institute for Integrative Genomics, Princeton University, Princeton, NJ 08544, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">13</span><span id="aff13" property="name">Human Neuron Core, Rosamund Stone Zander Translational Neuroscience Center, Boston Children’s Hospital, Boston, MA 02115, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">14</span><span id="aff14" property="name">Department of Neurology, Boston Children’s Hospital, Boston, MA 02115, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">15</span><span id="aff15" property="name">Harvard Medical School, Boston, MA 02115, USA</span></div></div><div class="core-authors-notes"><div role="doc-footnote" data-has="label"><div class="label">16</div><div id="fn1"><div id="ntpara0010" role="paragraph">These authors contributed equally</div></div></div><div role="doc-footnote" data-has="label"><div class="label">17</div><div id="fn2"><div id="ntpara0015" role="paragraph">Lead contact</div></div></div></div></div><div id="core-content-info" role="region" data-expandable="item" style="display: none;"><div class="history"><div>Publication History:</div><div><span>Received June 6, 2021</span>; <span>Revised August 9, 2021</span>; <span>Accepted September 9, 2021</span>; <span>Published online October 14, 2021</span></div></div><div class="core-links"><span class="doi">DOI: <a href="https://doi.org/10.1016/j.cell.2021.09.018" property="sameAs" target="_blank">10.1016/j.cell.2021.09.018 <span class="sr-only">External Link</span></a></span><span class="science-direct">Also available on <a href="https://www.sciencedirect.com/science/article/pii/S0092867421010655" target="_blank">ScienceDirect <span class="sr-only">External Link</span></a></span></div><div role="paragraph">Copyright: © 2021 The Author(s). Published by Elsevier Inc.</div><div class="core-links"><span>User License: <a href="http://creativecommons.org/licenses/by/4.0/" target="_blank">Creative Commons Attribution (CC BY 4.0)</a> | <a href="https://www.elsevier.com/open-access" target="_blank">Elsevier's open access license policy</a></span></div><div class="core-crossmark-badge"><a data-target="crossmark" href="#" title="Check for updates on crossmark" data-doi="10.1016/j.cell.2021.09.018" data-id="article-info-crossmark" class="crossmark__link" style="cursor: pointer;"><svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 290 58" height="22px" role="presentation"><path fill="url(#horizontal_crossmark)" stroke="#948F8F" stroke-miterlimit="10" stroke-width=".75" d="M287 1H3a2 2 0 0 0-2 2v52a2 2 0 0 0 2 2h284a2 2 0 0 0 2-2V3a2 2 0 0 0-2-2Z"></path><path fill="#535353" d="M80.3 23.58a6.395 6.395 0 0 0-.55-1.43 3.92 3.92 0 0 0-2.19-1.86 5.099 5.099 0 0 0-1.71-.29 5 5 0 0 0-2.46.58 4.831 4.831 0 0 0-1.69 1.54 6.921 6.921 0 0 0-1 2.19 10.16 10.16 0 0 0 0 5 6.921 6.921 0 0 0 1 2.19 4.83 4.83 0 0 0 1.69 1.5 5 5 0 0 0 2.46.58 4.44 4.44 0 0 0 1.85-.36 4.12 4.12 0 0 0 1.38-1c.39-.436.695-.942.9-1.49A7.094 7.094 0 0 0 80.4 29h3a8.46 8.46 0 0 1-.69 2.92 7.179 7.179 0 0 1-1.59 2.29 7 7 0 0 1-2.35 1.49 8.14 8.14 0 0 1-3 .52 8.43 8.43 0 0 1-3.59-.74 7.88 7.88 0 0 1-2.69-2 9.001 9.001 0 0 1-1.69-3 11.41 11.41 0 0 1 0-7.23 9.001 9.001 0 0 1 1.69-3 8.001 8.001 0 0 1 2.69-2 8.3 8.3 0 0 1 3.59-.75 9.44 9.44 0 0 1 2.83.41c.866.242 1.68.642 2.4 1.18A6.229 6.229 0 0 1 82.65 21a6.66 6.66 0 0 1 .78 2.58H80.3ZM85.9 17.9h2.85v6.63h.05a4.15 4.15 0 0 1 1.56-1.46 4.61 4.61 0 0 1 2.31-.59 4.83 4.83 0 0 1 3.33 1.1 4.21 4.21 0 0 1 1.22 3.3v8.88H94.4v-8.13a3.51 3.51 0 0 0-.65-2.21 2.36 2.36 0 0 0-1.87-.69 3.1 3.1 0 0 0-1.3.26 2.821 2.821 0 0 0-1 .73A3.41 3.41 0 0 0 89 26.8a3.88 3.88 0 0 0-.23 1.33v7.63H85.9V17.9ZM102.47 30c-.002.491.069.98.21 1.45.131.448.348.866.64 1.23a3.08 3.08 0 0 0 1.08.84 3.543 3.543 0 0 0 1.55.31 3.431 3.431 0 0 0 2-.54c.541-.402.94-.966 1.14-1.61h2.7A5.349 5.349 0 0 1 109.7 35a5.715 5.715 0 0 1-1.74.85 7 7 0 0 1-2 .29 6.848 6.848 0 0 1-2.7-.5 5.577 5.577 0 0 1-2-1.4A5.953 5.953 0 0 1 100 32a8.748 8.748 0 0 1-.41-2.75 7.768 7.768 0 0 1 .44-2.61 6.67 6.67 0 0 1 1.25-2.17 5.863 5.863 0 0 1 4.56-2 5.79 5.79 0 0 1 2.74.64c.783.41 1.466.986 2 1.69a6.824 6.824 0 0 1 1.16 2.41c.264.905.352 1.852.26 2.79h-9.53Zm6.65-1.87a4.605 4.605 0 0 0-.29-1.3c-.148-.4-.368-.77-.65-1.09a3.22 3.22 0 0 0-1-.75 3 3 0 0 0-1.31-.29 3.431 3.431 0 0 0-1.36.26 3.108 3.108 0 0 0-1 .73 3.594 3.594 0 0 0-.7 1.09 3.798 3.798 0 0 0-.29 1.35h6.65-.05ZM122.88 27.15a2.666 2.666 0 0 0-1-1.81 3.14 3.14 0 0 0-2-.61c-.4.005-.798.07-1.18.19-.435.136-.83.376-1.15.7a3.92 3.92 0 0 0-.87 1.43 6.783 6.783 0 0 0-.35 2.39c.002.539.065 1.076.19 1.6.113.509.316.993.6 1.43.273.41.635.752 1.06 1a3 3 0 0 0 1.58.39 2.872 2.872 0 0 0 2.06-.77 3.596 3.596 0 0 0 1-2.18h2.85a6.26 6.26 0 0 1-1.91 3.86 5.83 5.83 0 0 1-4 1.34 6.61 6.61 0 0 1-2.69-.51 5.695 5.695 0 0 1-2-1.4 6.001 6.001 0 0 1-1.21-2.11 8.279 8.279 0 0 1-.41-2.65 8.987 8.987 0 0 1 .4-2.72 6.237 6.237 0 0 1 1.2-2.21 5.614 5.614 0 0 1 2-1.47 6.7 6.7 0 0 1 2.79-.54 7.801 7.801 0 0 1 2.14.29 5.634 5.634 0 0 1 1.8.86c.529.386.969.88 1.29 1.45a5 5 0 0 1 .6 2.07h-2.85l.06-.02ZM128.13 17.9H131V28l5.15-5.22h3.5l-5 4.75 5.42 8.17h-3.47l-4-6.27-1.6 1.7v4.63h-2.85V17.9h-.02ZM147.15 22.83h2.13v-1.08a5.505 5.505 0 0 1 .3-2c.161-.456.436-.862.8-1.18.33-.273.721-.462 1.14-.55.44-.094.89-.141 1.34-.14a8.158 8.158 0 0 1 2 .18v2.24a4.113 4.113 0 0 0-.59-.11 6.343 6.343 0 0 0-.74 0 1.74 1.74 0 0 0-1 .27 1.204 1.204 0 0 0-.41 1.08v1.33h2.42V25h-2.42v10.8h-2.85V25h-2.12v-2.17ZM162.23 36.1a6.999 6.999 0 0 1-2.76-.51 5.877 5.877 0 0 1-2-1.41 5.998 5.998 0 0 1-1.33-2.18 8.775 8.775 0 0 1 0-5.48 6.008 6.008 0 0 1 1.27-2.15 5.894 5.894 0 0 1 2-1.41A7.71 7.71 0 0 1 165 23a5.894 5.894 0 0 1 2 1.41 6.003 6.003 0 0 1 1.28 2.15c.585 1.78.585 3.7 0 5.48a5.99 5.99 0 0 1-3.28 3.54 7.002 7.002 0 0 1-2.77.52Zm0-2.25a3.268 3.268 0 0 0 1.65-.4c.447-.262.83-.621 1.12-1.05.304-.444.528-.938.66-1.46a6.581 6.581 0 0 0 0-3.31 4.416 4.416 0 0 0-.66-1.46 3.526 3.526 0 0 0-1.15-1 3.6 3.6 0 0 0-3.3 0 3.526 3.526 0 0 0-1.15 1 4.414 4.414 0 0 0-.66 1.46 6.558 6.558 0 0 0 0 3.31c.132.522.356 1.016.66 1.46a3.47 3.47 0 0 0 1.15 1.05 3.26 3.26 0 0 0 1.68.4ZM171.13 22.83h2.67v2.5c.097-.362.264-.701.49-1 .251-.341.547-.647.88-.91a4.404 4.404 0 0 1 1.14-.66 3.41 3.41 0 0 1 1.28-.25c.23-.013.46-.013.69 0h.39v2.79l-.61-.09a5.173 5.173 0 0 0-.61 0 3.278 3.278 0 0 0-1.36.29 3.131 3.131 0 0 0-1.11.85 4.171 4.171 0 0 0-.75 1.39 6 6 0 0 0-.28 1.9v6.15h-2.85V22.83h.03ZM198.22 35.75h-2.8V34a3.907 3.907 0 0 1-1.56 1.56 4.25 4.25 0 0 1-2.11.59 4.742 4.742 0 0 1-3.75-1.31 5.537 5.537 0 0 1-1.12-3.84v-8.2h2.85v7.92a3.503 3.503 0 0 0 .65 2.4 2.38 2.38 0 0 0 1.83.7 3.599 3.599 0 0 0 1.5-.27 2.66 2.66 0 0 0 1-.74c.259-.326.444-.705.54-1.11a5.79 5.79 0 0 0 .16-1.4v-7.5h2.85v12.95h-.04ZM201.3 22.83h2.7v1.75a3.523 3.523 0 0 1 1.72-1.58 5.568 5.568 0 0 1 2.33-.49 6.17 6.17 0 0 1 2.66.54 5.246 5.246 0 0 1 1.89 1.47c.515.65.899 1.394 1.13 2.19.257.87.385 1.773.38 2.68.001.851-.114 1.7-.34 2.52a6.568 6.568 0 0 1-1 2.16 5.002 5.002 0 0 1-4.21 2.06c-.419 0-.837-.036-1.25-.11a5.39 5.39 0 0 1-1.2-.36A4.737 4.737 0 0 1 205 35a3.567 3.567 0 0 1-.81-.91v6.45h-2.89V22.83Zm10 6.47c0-.574-.078-1.146-.23-1.7a4.606 4.606 0 0 0-.67-1.46 3.475 3.475 0 0 0-1.12-1 3.06 3.06 0 0 0-1.55-.39A3.141 3.141 0 0 0 205 26a5.518 5.518 0 0 0-1 3.3c-.004.611.077 1.22.24 1.81.138.523.38 1.012.71 1.44a3.55 3.55 0 0 0 1.14 1 3.242 3.242 0 0 0 1.54.35 3.18 3.18 0 0 0 1.65-.4c.438-.25.817-.59 1.11-1 .299-.44.513-.931.63-1.45a7.38 7.38 0 0 0 .23-1.75h.05ZM228.67 35.75H226V34a3.38 3.38 0 0 1-1.67 1.61 5.668 5.668 0 0 1-2.33.49 6.17 6.17 0 0 1-2.66-.54 5.307 5.307 0 0 1-1.89-1.46 6.254 6.254 0 0 1-1.12-2.19 9.48 9.48 0 0 1-.37-2.71 8.67 8.67 0 0 1 .47-3 6.058 6.058 0 0 1 1.26-2.1 4.89 4.89 0 0 1 1.8-1.21 5.738 5.738 0 0 1 2.06-.39c.412 0 .824.037 1.23.11.413.073.815.194 1.2.36.38.166.736.381 1.06.64.32.255.594.562.81.91V17.9h2.85v17.85h-.03Zm-10-6.33c-.001.55.069 1.098.21 1.63.13.511.35.996.65 1.43.288.414.668.757 1.11 1a3.241 3.241 0 0 0 1.6.38 3.199 3.199 0 0 0 1.64-.4 3.398 3.398 0 0 0 1.13-1.05c.297-.447.517-.94.65-1.46.14-.542.211-1.1.21-1.66a5.227 5.227 0 0 0-1-3.35 3.175 3.175 0 0 0-2.61-1.2 3.23 3.23 0 0 0-1.69.41 3.478 3.478 0 0 0-1.12 1.07 4.494 4.494 0 0 0-.62 1.5 7.871 7.871 0 0 0-.11 1.7h-.05ZM242.6 32.88c-.018.258.03.516.14.75a.577.577 0 0 0 .54.23h.3c.135-.002.269-.019.4-.05v2l-.39.11-.49.08-.5.08h-.42a2.75 2.75 0 0 1-1.45-.35 1.731 1.731 0 0 1-.75-1.23 4.998 4.998 0 0 1-2.09 1.2 8.224 8.224 0 0 1-2.39.38 5.861 5.861 0 0 1-1.67-.24 4.394 4.394 0 0 1-1.41-.7 3.35 3.35 0 0 1-1-1.18 3.624 3.624 0 0 1-.36-1.66 3.84 3.84 0 0 1 .44-2 3.246 3.246 0 0 1 1.15-1.17 5.195 5.195 0 0 1 1.6-.61c.593-.127 1.19-.223 1.79-.29a14.77 14.77 0 0 1 1.47-.21 6.896 6.896 0 0 0 1.24-.21c.318-.077.611-.236.85-.46a1.286 1.286 0 0 0 .31-.94 1.458 1.458 0 0 0-.26-.9A1.759 1.759 0 0 0 239 25a2.807 2.807 0 0 0-.86-.25 6.941 6.941 0 0 0-.9-.06 3.58 3.58 0 0 0-2 .5 2 2 0 0 0-.87 1.55h-2.85c.021-.73.227-1.442.6-2.07.338-.54.797-.996 1.34-1.33a5.475 5.475 0 0 1 1.84-.7c.692-.134 1.395-.2 2.1-.2.632 0 1.262.067 1.88.2a5.337 5.337 0 0 1 1.66.65 3.6 3.6 0 0 1 1.19 1.16c.314.525.47 1.129.45 1.74v6.65l.02.04Zm-2.85-3.6a3.45 3.45 0 0 1-1.6.51c-.667.06-1.3.147-1.9.26a6.137 6.137 0 0 0-.87.21 2.64 2.64 0 0 0-.75.38c-.22.164-.395.38-.51.63a2.129 2.129 0 0 0-.19.94c-.011.29.085.575.27.8.182.215.407.388.66.51.269.128.555.216.85.26.277.05.558.076.84.08a4.63 4.63 0 0 0 1-.12 3.418 3.418 0 0 0 1-.43 2.73 2.73 0 0 0 .81-.76 1.93 1.93 0 0 0 .32-1.14v-2.14l.07.01ZM244.37 22.83h2.15V19h2.85v3.88h2.58V25h-2.58v6.9a7.064 7.064 0 0 0 0 .77c.016.194.074.381.17.55a.84.84 0 0 0 .41.34 2 2 0 0 0 .75.11h.6c.203-.01.404-.04.6-.09v2.2l-.93.1a8.403 8.403 0 0 1-.92 0 6.33 6.33 0 0 1-1.81-.21 2.421 2.421 0 0 1-1.08-.62 2.15 2.15 0 0 1-.52-1 7.791 7.791 0 0 1-.16-1.42V25h-2.15v-2.17h.04ZM256.22 30c-.002.491.069.98.21 1.45.131.448.348.866.64 1.23a3.08 3.08 0 0 0 1.08.84 3.543 3.543 0 0 0 1.55.31 3.431 3.431 0 0 0 2-.54c.541-.402.94-.966 1.14-1.61h2.7a5.352 5.352 0 0 1-2.1 3.26 5.715 5.715 0 0 1-1.74.85 7 7 0 0 1-2 .29 6.848 6.848 0 0 1-2.7-.5 5.577 5.577 0 0 1-2-1.4 5.947 5.947 0 0 1-1.21-2.18 8.754 8.754 0 0 1-.41-2.75 7.758 7.758 0 0 1 .44-2.61 6.67 6.67 0 0 1 1.25-2.17 5.863 5.863 0 0 1 4.56-2 5.79 5.79 0 0 1 2.74.64c.783.41 1.466.986 2 1.69a6.824 6.824 0 0 1 1.16 2.41c.243.916.308 1.87.19 2.81h-9.5V30Zm6.65-1.87a4.605 4.605 0 0 0-.29-1.3c-.148-.4-.368-.77-.65-1.09a3.22 3.22 0 0 0-1-.75 3 3 0 0 0-1.31-.29 3.431 3.431 0 0 0-1.36.26 3.108 3.108 0 0 0-1 .73 3.575 3.575 0 0 0-.7 1.09 3.798 3.798 0 0 0-.29 1.35h6.65-.05ZM270 31.6a2.154 2.154 0 0 0 .95 1.75c.607.35 1.3.524 2 .5.304-.002.608-.022.91-.06a3.943 3.943 0 0 0 1-.24c.286-.105.54-.28.74-.51a1.23 1.23 0 0 0 .26-.89 1.261 1.261 0 0 0-.4-.9 2.779 2.779 0 0 0-1-.56 9.315 9.315 0 0 0-1.34-.36l-1.5-.33a14.219 14.219 0 0 1-1.54-.42 4.937 4.937 0 0 1-1.33-.68 3.12 3.12 0 0 1-.94-1.09 3.472 3.472 0 0 1-.35-1.64 2.94 2.94 0 0 1 .51-1.76 3.913 3.913 0 0 1 1.3-1.15 5.66 5.66 0 0 1 1.75-.61 10.316 10.316 0 0 1 1.84-.17 8.418 8.418 0 0 1 1.91.21 5.19 5.19 0 0 1 1.65.69c.496.318.916.741 1.23 1.24.341.559.55 1.188.61 1.84h-3a1.783 1.783 0 0 0-.94-1.37 3.892 3.892 0 0 0-1.69-.35 6.093 6.093 0 0 0-.71 0 3.327 3.327 0 0 0-.78.19 1.642 1.642 0 0 0-.61.4 1 1 0 0 0-.25.69 1.093 1.093 0 0 0 .36.85c.294.245.634.428 1 .54.437.153.885.274 1.34.36l1.55.33a20.9 20.9 0 0 1 1.53.42c.479.157.931.386 1.34.68.41.277.751.643 1 1.07.255.494.379 1.045.36 1.6a3.48 3.48 0 0 1-.52 2 4.15 4.15 0 0 1-1.36 1.3 6.19 6.19 0 0 1-1.86.73 9.435 9.435 0 0 1-2 .23 8.823 8.823 0 0 1-2.26-.27 5.265 5.265 0 0 1-1.8-.84 4 4 0 0 1-1.2-1.4 4.445 4.445 0 0 1-.46-2h2.7v-.02Z"></path><path fill="#C72914" d="m21.94 38.74 18.12-12.08v-14.5H21.94v26.58Z"></path><path fill="#EF3340" d="M40.06 38.74 21.94 26.66v-14.5h18.12v26.58Z"></path><path fill="#3EB1C8" d="M31 9a20 20 0 1 0 0 40 20 20 0 0 0 0-40Zm0 35.1a15.1 15.1 0 1 1 0-30.2 15.1 15.1 0 0 1 0 30.2Z"></path><path fill="#FFC72C" d="M42.09 39.23A15.09 15.09 0 0 1 17 23.38l-4.2-2.64a20 20 0 0 0 33.49 21.14l-4.2-2.65Z"></path><defs><linearGradient id="horizontal_crossmark" x1="145" x2="145" y1="7.33" y2="56.82" gradientUnits="userSpaceOnUse"><stop stop-color="#fff"></stop><stop offset="1" stop-color="#c4c4c4"></stop></linearGradient></defs></svg></a></div><div class="core-content-info"></div></div><div id="core-linked-content" role="region" data-expandable="item" style="display: none;">
    

        


    

            

            

            <section data-widget-id="e587233e-22aa-4221-8ae1-22df489d46f7" data-widget-name="UX3linkedContentList" class="axel-linked-articles"><div class="section-paragraph linked-article"><ul class="rlist"><li class="linked-article__item"><div class="linked-article__article-type">Resource</div><a class="linked-article__title link--underlined" href="/cell/fulltext/S0092-8674(21)01176-4" target="_blank" title="Mapping the genetic landscape of DNA double-strand break repair">Mapping the genetic landscape of DNA double-strand break repair</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Hussmann et al.</li></ul></div><div class="linked-article__details"><span><i>Cell</i></span><span id="item_date">October 20, 2021</span></div></li></ul></div></section>

            

        
</div></div></div></div></header><div class="core-header-aside">
    

        


    

            

            

            <div class="cover-image cover-image--axel"><div class="cover-image__thumb"><a href="https://www.cell.com/cell/issue?pii=S0092-8674(20)X0023-7"><img src="/cms/asset/8cc74f04-4511-42ff-8aba-8b2500f3672c/cov200h.gif" alt="Cover Image - Cell, Volume 184, Issue 22"></a></div></div>

            

        
</div></div><div data-core-wrapper="content"><div id="dummy-sticky-observer"></div><div data-core-nav="header" data-extent="frontmatter"><div class="core-nav-wrapper core-container" aria-label="Article navigation"><div class="navigation-menu-pdf">
    

        


    

            

            

            <div data-widget-id="ecff07e8-0bc4-4533-b1f1-49365e2ead20" data-widget-name="UX3ArticleToolsWidget" class="article-tools"><a href="/action/showPdf?pii=S0092-8674%2821%2901065-5" title="Standard PDF (opens in a new tab)" target="_blank" aria-hidden="false" role="menuitem" class="article-tools__item__pdf"><svg focusable="false" viewBox="0 0 32 32" width="20" height="20" title="PDF" class="pdf-icon"><path id="pdfwhite" d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path id="pdfda0000" d="M.167 2.592H22.39V9.72H.166z" fill="#da0000"></path><path id="pdffff9f9" fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path id="pdff91d0a" d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="basic">Download PDF</span><span class="alternative">Download PDF</span></a></div>

            

        
</div><div class="core-sections-menu"><button data-id="article-toolbar-showhide" aria-controls="article_sections_menu" aria-expanded="false" aria-haspopup="true" aria-label="Toggle section navigation menu" style="margin-top: 0px;"><i class="icon-list" aria-hidden="true"></i><span>Outline</span></button><nav id="article_sections_menu" aria-label="Contents" data-core-nav="article" style="margin-top: 0px;"><div class="heading">Outline<button aria-haspopup="true" aria-controls="article_sections_menu" title="Close outline menu"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div><ul><li><a href="#author-highlights-abstract">Highlights</a></li><li><a href="#author-abstract">Summary</a></li><li><a href="#graphical-abstract">Graphical abstract</a></li><li><a href="#keywords">Keywords</a></li><li><a href="#sec-1">Introduction</a></li><li><a href="#sec-2">Results</a></li><li><a href="#sec-3">Discussion</a></li><li><a href="#sec-4">Limitations of the study</a></li><li><a href="#sec-5">STAR★Methods</a></li><li><a href="#acknowledgments">Acknowledgments</a></li><li><a href="#supplementary-material">Supplemental information (6)</a></li><li><a href="#data-availability">Data and code availability</a></li><li><a href="#references">References</a></li><li role="none" class="mobile-only"><a role="menuitem" href="#core-collateral-metrics">Article metrics</a></li><li role="none" class="mobile-only"><a role="menuitem" href="#core-collateral-relatedArticles">Related Articles</a></li></ul></nav></div><div class="navigation-menu-share">
    

        


    

            

            

            <!-- Go to https://www.addtoany.com/buttons/customize/ to customize your tools --><script type="text/javascript" defer="defer" src="https://static.addtoany.com/menu/page.js"></script><script type="text/javascript">var a2a_config = a2a_config || {};
a2a_config.thanks = {
    postShare: false,
    ad: false,
};</script><script>var a2a_config = a2a_config || {};
let res = window.location.href.replace(/\(/g, "%28").replace(/\)/g, "%29");
a2a_config.templates = a2a_config.templates || {};

a2a_config.templates.email = {
    body: res
};
</script><div data-widget-id="5f978446-c80a-450c-a185-94d633df2db2" data-widget-name="UX3share" aria-hidden="false" role="group" class="article-tools__dropblock article-tools__share"><button aria-label="Toggle share menu" aria-expanded="false" aria-haspopup="true" role="menuitem" aria-controls="navigation_menu_share_block" aria-labelledby="share-label" class="article-tools__item__share"><i aria-hidden="true" class="icon-article-share"></i><span id="share-label">Share</span></button><div id="navigation_menu_share_block" role="menu" class="share__block axel-share article-tools__dropblock__holder"><h4 class="axel-share__title heading">Share<button title="Close share menu" aria-expanded="false" aria-haspopup="true" aria-controls="navigation_menu_share_block"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></h4><div class="axel-share__inner"><span class="sr-only">Share on</span><ul class="a2a__share-list rlist w-slide--list a2a a2a_kit a2a_kit_size_32 a2a_default_style" data-a2a-url="https://www.cell.com/cell/fulltext/S0092-8674%2821%2901065-5" style="line-height: 32px;"><li class="a2a_listitem_custom"><a role="link" title="Email" class="a2a_button_email" target="_blank" rel="nofollow noopener" href="/#email"><i aria-hidden="true" class="at-icon-wrapper icon-gizmo-email-plus"></i><span>Email</span></a></li><li class="a2a_listitem_custom"><a role="link" title="formerly known as Twitter" class="a2a_button_twitter" target="_blank" rel="nofollow noopener" href="/#twitter"><i aria-hidden="true" class="at-icon-wrapper icon-x-default"></i><span>X</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Facebook" class="a2a_button_facebook" target="_blank" rel="nofollow noopener" href="/#facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook-square-2"></i><span>Facebook</span></a></li><li class="a2a_listitem_custom"><a role="link" title="LinkedIn" class="a2a_button_linkedin" target="_blank" rel="nofollow noopener" href="/#linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin-square"></i><span>LinkedIn</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Sina Weibo" class="a2a_button_sina_weibo" target="_blank" rel="nofollow noopener" href="/#sina_weibo"><i aria-hidden="true" class="at-icon-wrapper icon-weibo"></i><span>Sina Weibo</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Mendeley" class="a2a_button_mendeley" target="_blank" rel="nofollow noopener" href="/#mendeley"><i aria-hidden="true" class="at-icon-wrapper icon-mendeley-2"></i><span>Mendeley</span></a></li><li class="a2a_listitem_custom"><a role="link" title="bluesky" class="a2a_button_bluesky" target="_blank" rel="nofollow noopener" href="/#bluesky"><i aria-hidden="true" class="at-icon-wrapper icon-bluesky-logo"></i><span>bluesky</span></a></li><div style="clear: both;"></div></ul><div class="extra-links-drop-zone">
    

        


    

            

            

            <div data-widget-id="f1a208d3-ac5e-4db2-9c3a-40ef97ddfa10" data-widget-name="UX3HTMLWidget"><a class="axel-share__extra" target="_blank" href="/personalize/addFavoritePublication?doi=10.1016/j.cell.2021.09.018">
	<i class="icon-reading-list" aria-hidden="true"></i>
	<span>Add to my reading list</span>
</a></div>

            

        
</div></div></div></div>

            

        
</div><div><button data-id="article-nav-menubar-more" aria-controls="article_more_menu" aria-expanded="false" aria-haspopup="true" aria-label="Toggle more menu"><i aria-hidden="true" class="icon-more-mobile"></i><span>More</span></button><div class="info-panel"><div class="info-panel__left-content"><div class="info-panel__article_tools info-panel__item"><div id="article_more_menu" aria-label="More menu" data-core-nav="more"><div class="heading">More<button title="Close more menu" aria-controls="article_more_menu" aria-haspopup="true"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div>
    

        


    

            

            

            <ul data-widget-id="183a17a2-87b5-4e6f-bb85-6b5cda068c89" data-widget-name="UX3ArticleToolsWidget" class="article-tools"><li class="article-tools__item article-tools__pdf"><a href="/action/showPdf?pii=S0092-8674%2821%2901065-5" title="Standard PDF (opens in a new tab)" target="_blank" aria-hidden="false" class="article-tools__ctrl article-tools__item__pdf"><svg focusable="false" viewBox="0 0 32 32" width="20" height="20" title="PDF" class="pdf-icon"><path id="pdfwhite" d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path id="pdfda0000" d="M.167 2.592H22.39V9.72H.166z" fill="#da0000"></path><path id="pdffff9f9" fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path id="pdff91d0a" d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="basic">Download PDF</span><span class="alternative">Download PDF</span></a></li><li><a href="https://www.cell.com/action/showCitFormats?doi=10.1016%2Fj.cell.2021.09.018&amp;pii=S0092-8674%2821%2901065-5" title="Cite" target="_blank" aria-label="Open Cite in new tab" class="article-tools__item__displayExportCitation"><i aria-hidden="true" class="icon-export-Citation"></i><span>Cite</span></a></li><li id="article_tools_share">
    

        


    

            

            

            <!-- Go to https://www.addtoany.com/buttons/customize/ to customize your tools --><script type="text/javascript" defer="defer" src="https://static.addtoany.com/menu/page.js"></script><script type="text/javascript">var a2a_config = a2a_config || {};
a2a_config.thanks = {
    postShare: false,
    ad: false,
};</script><script>var a2a_config = a2a_config || {};
let res = window.location.href.replace(/\(/g, "%28").replace(/\)/g, "%29");
a2a_config.templates = a2a_config.templates || {};

a2a_config.templates.email = {
    body: res
};
</script><div data-widget-id="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da" data-widget-name="UX3share" class="dropBlock article-tools__dropblock article-tools__share"><button data-db-target-for="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da" aria-expanded="false" title="open Share menu" class="article-tools__ctrl share__ctrl"><i aria-hidden="true" class="icon-article-share"></i><span>Share</span></button><div data-db-target-of="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da" id="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da_Pop" class="share__block axel-share dropBlock__holder article-tools__dropblock__holder"><h4 class="axel-share__title">Share</h4><div class="axel-share__inner"><span class="sr-only">Share on</span><ul class="a2a__share-list rlist w-slide--list a2a a2a_kit a2a_kit_size_32 a2a_default_style" data-a2a-url="https://www.cell.com/cell/fulltext/S0092-8674%2821%2901065-5" style="line-height: 32px;"><li class="a2a_listitem_custom"><a role="link" title="Email" class="a2a_button_email" target="_blank" rel="nofollow noopener" href="/#email"><i aria-hidden="true" class="at-icon-wrapper icon-gizmo-email-plus"></i><span>Email</span></a></li><li class="a2a_listitem_custom"><a role="link" title="formerly known as Twitter" class="a2a_button_twitter" target="_blank" rel="nofollow noopener" href="/#twitter"><i aria-hidden="true" class="at-icon-wrapper icon-x-default"></i><span>X</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Facebook" class="a2a_button_facebook" target="_blank" rel="nofollow noopener" href="/#facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook-square-2"></i><span>Facebook</span></a></li><li class="a2a_listitem_custom"><a role="link" title="LinkedIn" class="a2a_button_linkedin" target="_blank" rel="nofollow noopener" href="/#linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin-square"></i><span>LinkedIn</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Sina Weibo" class="a2a_button_sina_weibo" target="_blank" rel="nofollow noopener" href="/#sina_weibo"><i aria-hidden="true" class="at-icon-wrapper icon-weibo"></i><span>Sina Weibo</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Mendeley" class="a2a_button_mendeley" target="_blank" rel="nofollow noopener" href="/#mendeley"><i aria-hidden="true" class="at-icon-wrapper icon-mendeley-2"></i><span>Mendeley</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Bluesky" class="a2a_button_bluesky" target="_blank" rel="nofollow noopener" href="/#bluesky"><i aria-hidden="true" class="at-icon-wrapper icon-bluesky-logo"></i><span>Bluesky</span></a></li><div style="clear: both;"></div></ul><div class="extra-links-drop-zone">
    

        


    

            

            

            <div data-widget-id="ae9bf5c5-a2ee-4f5c-8e41-9b21638b1a9f" data-widget-name="UX3HTMLWidget"><a class="axel-share__extra" target="_blank" href="/personalize/addFavoritePublication?doi=10.1016/j.cell.2021.09.018">
	<i class="icon-reading-list" aria-hidden="true"></i>
	<span>Add to my reading list</span>
</a></div>

            

        
</div></div></div></div>

            

        
</li><li><a href="https://www.cell.com/action/addCitationAlert?doi=10.1016%2Fj.cell.2021.09.018" title="Set Alert" target="_blank" aria-label="Open Set Alert in new tab" class="article-tools__item__displayCreateCitationAlert"><i aria-hidden="true" class="icon-set-alert"></i><span>Set Alert</span></a></li><li><a href="https://s100.copyright.com/AppDispatchServlet?publisherName=ELS&amp;contentID=S0092867421010655&amp;orderBeanReset=true&amp;orderSource=Phoenix" title="Get Rights" target="_blank" aria-label="Open Get Rights in new tab" class="article-tools__item__request"><i aria-hidden="true" class="icon-get-rights"></i><span>Get Rights</span></a></li><li><a href="https://www.contentednet.com/salesforce-form/?pii=S0092867421010655&amp;issn=00928674&amp;source=jbs" title="Reprints" target="_blank" aria-label="Open Reprints in new tab" class="article-tools__item__reprints"><i aria-hidden="true" class="icon-article-reprints"></i><span>Reprints</span></a></li></ul>

            

        
</div></div><div class="info-panel__download-issue info-panel__item"><div class="article-header__download-full-issue"><button title="Download full issue" type="submit" formaction="/action/downloadMultiplePdf?action=downloadIssuePdf&amp;journal=cell&amp;volume=184&amp;issue=22&amp;pii=S0092-8674%2820%29X0023-7" class="btn--download-issue btn--cancel download-full-issue"><i class="icon-download-new"></i><span>Download Full Issue</span></button><div class="table-of-content__alert table-of-content__alert--download"><p>Download started</p><a href="#" class="table-of-content__alert--close">Ok</a></div></div></div></div><div class="info-panel__right-content"><div class="info-panel__navigation info-panel__item"><div class="content-navigation"><a href="/cell/fulltext/S0092-8674(21)01060-6" aria-label="Previous article" aria-disabled="false" class="content-navigation__prev"><i aria-hidden="true" class="icon-arrow_l"></i><span>Previous article</span></a><a href="/cell/fulltext/S0092-8674(21)01176-4" aria-label="Next article" aria-disabled="false" class="content-navigation__next"><span>Next article</span><i aria-hidden="true" class="icon-arrow_r"></i></a></div></div></div></div></div></div></div><div class="core-sections-menu-outer"><div class="core-sections-menu"><button aria-controls="article_sections_menu-desktop" aria-haspopup="true" aria-expanded="false" aria-label="Toggle sections menu" data-id="article-toolbar-showhide"><i class="icon-show-outline" aria-hidden="true"></i><span>Show Outline</span></button><nav id="article_sections_menu-desktop" aria-label="Contents" data-core-nav="article" style="margin-top: 0px;"><button aria-controls="article_sections_menu-desktop" aria-haspopup="true" aria-expanded="false" aria-label="Toggle sections menu" data-id="article-toolbar-showhide"><i class="icon-hide-outline" aria-hidden="true"></i><span>Hide Outline</span></button><ul><li><a href="#author-highlights-abstract">Highlights</a></li><li><a href="#author-abstract">Summary</a></li><li><a href="#graphical-abstract">Graphical abstract</a></li><li><a href="#keywords">Keywords</a></li><li><a href="#sec-1">Introduction</a></li><li><a href="#sec-2">Results</a></li><li><a href="#sec-3">Discussion</a></li><li><a href="#sec-4">Limitations of the study</a></li><li><a href="#sec-5">STAR★Methods</a></li><li><a href="#acknowledgments">Acknowledgments</a></li><li><a href="#supplementary-material">Supplemental information (6)</a></li><li><a href="#data-availability">Data and code availability</a></li><li><a href="#references">References</a></li><li role="none" class="mobile-only"><a role="menuitem" href="#core-collateral-metrics">Article metrics</a></li><li role="none" class="mobile-only"><a role="menuitem" href="#core-collateral-relatedArticles">Related Articles</a></li></ul></nav></div></div><div><div class="core-content-inner"><div class="core-linked-content"></div><div id="abstracts" data-extent="frontmatter"><div class="core-container"><section id="author-highlights-abstract" property="abstract" typeof="Text" role="doc-abstract"><h2 property="name">Highlights</h2><div id="abspara0020" role="paragraph"><div id="ulist0010" role="list"><div id="u0010" role="listitem"><div class="label">•</div><div class="content"><div id="p0010" role="paragraph">Pooled CRISPRi screens reveal that MMR inhibits prime editing efficiency and precision</div></div></div><div id="u0015" role="listitem"><div class="label">•</div><div class="content"><div id="p0015" role="paragraph">PE4 and PE5 enhance editing outcomes through co-expression of dominant negative MLH1</div></div></div><div id="u0020" role="listitem"><div class="label">•</div><div class="content"><div id="p0020" role="paragraph">Programming additional silent mutations can enhance prime editing by evading MMR</div></div></div><div id="u0025" role="listitem"><div class="label">•</div><div class="content"><div id="p0025" role="paragraph">PEmax editor improves prime editing efficacy in synergy with PE4, PE5, and epegRNAs</div></div></div></div></div></section><section id="author-abstract" property="abstract" typeof="Text" role="doc-abstract"><h2 property="name">Summary</h2><div id="abspara0010" role="paragraph">While prime editing enables precise sequence changes in DNA, cellular determinants of prime editing remain poorly understood. Using pooled CRISPRi screens, we discovered that DNA mismatch repair (MMR) impedes prime editing and promotes undesired indel byproducts. We developed PE4 and PE5 prime editing systems in which transient expression of an engineered MMR-inhibiting protein enhances the efficiency of substitution, small insertion, and small deletion prime edits by an average 7.7-fold and 2.0-fold compared to PE2 and PE3 systems, respectively, while improving edit/indel ratios by 3.4-fold in MMR-proficient cell types. Strategic installation of silent mutations near the intended edit can enhance prime editing outcomes by evading MMR. Prime editor protein optimization resulted in a PEmax architecture that enhances editing efficacy by 2.8-fold on average in HeLa cells. These findings enrich our understanding of prime editing and establish prime editing systems that show substantial improvement across 191 edits in seven mammalian cell types.</div></section><section id="graphical-abstract" property="abstract" typeof="Text" role="doc-abstract"><h2 property="name">Graphical abstract</h2><div id="abspara0015" role="paragraph"><div class="figure-wrap"><figure id="undfig1" class="graphic" data-figureviewer-ignore="true"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.09.018/asset/9329f1cc-cc05-4bf3-856a-76e9a7043847/main.assets/fx1_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="fx1" alt="Graphical abstract undfig1" src="/cms/10.1016/j.cell.2021.09.018/asset/00efaa71-b796-41dd-8aa5-fb8b617fbce4/main.assets/fx1.jpg" height="375" width="375" data-viewer-src="/cms/10.1016/j.cell.2021.09.018/asset/9329f1cc-cc05-4bf3-856a-76e9a7043847/main.assets/fx1_lrg.jpg" loading="lazy"></a></figure></div></div></section></div></div><div class="core-linked-content"></div><section id="keywords" property="keywords"><h2>Keywords</h2><ol><li><a href="/action/doSearch?AllField=&quot;prime+editing&quot;" alt="Search on this keyword" rel="nofollow">prime editing</a></li><li><a href="/action/doSearch?AllField=&quot;genome+editing&quot;" alt="Search on this keyword" rel="nofollow">genome editing</a></li><li><a href="/action/doSearch?AllField=&quot;CRISPR-Cas9&quot;" alt="Search on this keyword" rel="nofollow">CRISPR-Cas9</a></li><li><a href="/action/doSearch?AllField=&quot;Repair-seq&quot;" alt="Search on this keyword" rel="nofollow">Repair-seq</a></li><li><a href="/action/doSearch?AllField=&quot;mismatch+repair&quot;" alt="Search on this keyword" rel="nofollow">mismatch repair</a></li></ol></section><div class="core-linked-content"></div><div class="core-linked-content"></div><section id="bodymatter" data-extent="bodymatter" property="articleBody" typeof="Text"><div class="core-container"><section id="sec-1"><h2>Introduction</h2><div id="p0035" role="paragraph">The ability to manipulate the genome in a programmable manner has illuminated biology and shown promise in the clinical treatment of genetic diseases. Toward the goal of enabling a wide range of sequence changes, we developed prime editing, a versatile gene editing approach that can install all types of targeted DNA base pair substitutions, small insertions, small deletions, and combinations thereof without requiring double-strand DNA breaks (DSBs) or donor DNA templates (<span class="dropBlock reference-citations"><a id="body-ref-sref4" href="#" role="doc-biblioref" data-xml-rid="bib4" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref4" aria-controls="sref4">Anzalone et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">4.</div><div>Anzalone, A.V. ∙ Koblan, L.W. ∙ Liu, D.R.</div><div><strong>Genome editing with CRISPR-Cas nucleases, base editors, transposases and prime</strong></div><div><em>Nat. Biotechnol.</em> 2020; <strong>38</strong>:824-844</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41587-020-0561-9" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-85086735534" target="_blank" aria-label="Open Scopus in new tab">Scopus (1256)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32572269/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41587-020-0561-9&amp;pmid=32572269" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref3-1" href="#" role="doc-biblioref" data-xml-rid="bib3" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref3-1" aria-controls="sref3-1">Anzalone et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref3-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">3.</div><div>Anzalone, A.V. ∙ Randolph, P.B. ∙ Davis, J.R. ...</div><div><strong>Search-and-replace genome editing without double-strand breaks or donor DNA</strong></div><div><em>Nature.</em> 2019; <strong>576</strong>:149-157</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-1711-4" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-85074571375" target="_blank" aria-label="Open Scopus in new tab">Scopus (2574)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31634902/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-1711-4&amp;pmid=31634902" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Prime editing has been broadly applied to introduce genetic changes in flies (<span class="dropBlock reference-citations"><a id="body-ref-sref8" href="#" role="doc-biblioref" data-xml-rid="bib8" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref8" aria-controls="sref8">Bosch et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref8"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">8.</div><div>Bosch, J.A. ∙ Birchak, G. ∙ Perrimon, N.</div><div><strong>Precise genome engineering in <i>Drosophila</i> using prime editing</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2021; <strong>118</strong></div><div>e2021996118</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.2021996118" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-85098172457" target="_blank" aria-label="Open Scopus in new tab">Scopus (64)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.2021996118" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), rice and wheat (<span class="dropBlock reference-citations"><a id="body-ref-sref35" href="#" role="doc-biblioref" data-xml-rid="bib35" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref35" aria-controls="sref35">Lin et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref35"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">35.</div><div>Lin, Q. ∙ Zong, Y. ∙ Xue, C. ...</div><div><strong>Prime genome editing in rice and wheat</strong></div><div><em>Nat. Biotechnol.</em> 2020; <strong>38</strong>:582-585</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41587-020-0455-x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-85081736686" target="_blank" aria-label="Open Scopus in new tab">Scopus (528)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32393904/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41587-020-0455-x&amp;pmid=32393904" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), zebrafish (<span class="dropBlock reference-citations"><a id="body-ref-sref47" href="#" role="doc-biblioref" data-xml-rid="bib47" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref47" aria-controls="sref47">Petri et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref47"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">49.</div><div>Petri, K. ∙ Zhang, W. ∙ Ma, J. ...</div><div><strong>CRISPR prime editing with ribonucleoprotein complexes in zebrafish and primary human cells</strong></div><div><em>Nat. Biotechnol.</em> 2021;</div><div>Published online April 29, 2021</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41587-021-00901-y" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-85105157695" target="_blank" aria-label="Open Scopus in new tab">Scopus (119)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33927418/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41587-021-00901-y&amp;pmid=33927418" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), mouse embryos (<span class="dropBlock reference-citations"><a id="body-ref-sref37" href="#" role="doc-biblioref" data-xml-rid="bib37" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref37" aria-controls="sref37">Liu et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref37"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">38.</div><div>Liu, Y. ∙ Li, X. ∙ He, S. ...</div><div><strong>Efficient generation of mouse models with the prime editing system</strong></div><div><em>Cell Discov.</em> 2020; <strong>6</strong>:27</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41421-020-0165-z" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_38_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-85083988172" target="_blank" aria-label="Open Scopus in new tab">Scopus (143)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32351707/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41421-020-0165-z&amp;pmid=32351707" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), post-natal mice (<span class="dropBlock reference-citations"><a id="body-ref-sref38-1" href="#" role="doc-biblioref" data-xml-rid="bib38" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref38-1" aria-controls="sref38-1">Liu et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref38-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">39.</div><div>Liu, P. ∙ Liang, S.-Q. ∙ Zheng, C. ...</div><div><strong>Improved prime editors enable pathogenic allele correction and cancer modelling in adult mice</strong></div><div><em>Nat. Commun.</em> 2021; <strong>12</strong>:2121</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41467-021-22295-w" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-85104130567" target="_blank" aria-label="Open Scopus in new tab">Scopus (163)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33837189/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41467-021-22295-w&amp;pmid=33837189" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), human stem cells (<span class="dropBlock reference-citations"><a id="body-ref-sref59" href="#" role="doc-biblioref" data-xml-rid="bib59" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref59" aria-controls="sref59">Sürün et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref59"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">61.</div><div>Sürün, D. ∙ Schneider, A. ∙ Mircetic, J. ...</div><div><strong>Efficient Generation and Correction of Mutations in Human iPS Cells Utilizing mRNAs of CRISPR Base Editors and Prime</strong></div><div><em>Genes (Basel).</em> 2020; <strong>11</strong>:511</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.3390/genes11050511" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-85084579488" target="_blank" aria-label="Open Scopus in new tab">Scopus (82)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32384610/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.3390%2Fgenes11050511&amp;pmid=32384610" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), and patient-derived organoids (<span class="dropBlock reference-citations"><a id="body-ref-sref52" href="#" role="doc-biblioref" data-xml-rid="bib52" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref52" aria-controls="sref52">Schene et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref52"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">54.</div><div>Schene, I.F. ∙ Joore, I.P. ∙ Oka, R. ...</div><div><strong>Prime editing for functional repair in patient-derived disease models</strong></div><div><em>Nat. Commun.</em> 2020; <strong>11</strong>:5352</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41467-020-19136-7" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-85093955574" target="_blank" aria-label="Open Scopus in new tab">Scopus (134)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33097693/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41467-020-19136-7&amp;pmid=33097693" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Despite its versatility, the efficiency of prime editing can vary widely across edit classes, target loci, and cell types (<span class="dropBlock reference-citations"><a id="body-ref-sref3-2" href="#" role="doc-biblioref" data-xml-rid="bib3" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref3-2" aria-controls="sref3-2">Anzalone et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref3-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">3.</div><div>Anzalone, A.V. ∙ Randolph, P.B. ∙ Davis, J.R. ...</div><div><strong>Search-and-replace genome editing without double-strand breaks or donor DNA</strong></div><div><em>Nature.</em> 2019; <strong>576</strong>:149-157</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-1711-4" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-85074571375" target="_blank" aria-label="Open Scopus in new tab">Scopus (2574)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31634902/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-1711-4&amp;pmid=31634902" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). To maximize the utility of prime editing, we sought to identify cellular determinants of prime editing outcomes and use the resulting insights to develop improved prime editing systems.</div><div id="p0040" role="paragraph">Prime editing minimally requires two components: an engineered reverse transcriptase (RT) fused to Cas9 nickase (the PE2 protein) and a prime editing guide RNA (pegRNA) that contains both a spacer sequence complementary to target DNA and a 3′ extension encoding the desired edit (<span class="dropBlock reference-citations"><a id="body-ref-sref3-3" href="#" role="doc-biblioref" data-xml-rid="bib3" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref3-3" aria-controls="sref3-3">Anzalone et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref3-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">3.</div><div>Anzalone, A.V. ∙ Randolph, P.B. ∙ Davis, J.R. ...</div><div><strong>Search-and-replace genome editing without double-strand breaks or donor DNA</strong></div><div><em>Nature.</em> 2019; <strong>576</strong>:149-157</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-1711-4" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-85074571375" target="_blank" aria-label="Open Scopus in new tab">Scopus (2574)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31634902/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-1711-4&amp;pmid=31634902" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) (<a id="crosref0270" href="#fig1">Figure&nbsp;1</a>A). The PE2–pegRNA complex binds one strand of a target DNA locus and nicks the opposite strand, exposing a DNA 3′ end that can hybridize to the primer binding site (PBS) in the pegRNA extension. Reverse transcription of the RT template within the pegRNA extension then generates a 3′ DNA flap that contains the edited sequence and ultimately leads to incorporation of that sequence into the genome. The “PE3” system differs from PE2 by using an additional single guide RNA (sgRNA) to nick the non-edited strand at a location away from the pegRNA target, which enhances editing efficiency. However, nicking the non-edited strand also increases the frequency of undesired insertions and deletions (indels) at the target site.</div><div class="figure-wrap"><figure id="fig1" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.09.018/asset/b4877c18-909c-4973-8ea9-a51acde54482/main.assets/gr1_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr1" src="/cms/10.1016/j.cell.2021.09.018/asset/fcdc7b5a-d634-4ddd-ac1d-2c7e98c43fc8/main.assets/gr1.jpg" height="937" width="673" aria-labelledby="fig1-title fig1-content" data-viewer-src="/cms/10.1016/j.cell.2021.09.018/asset/b4877c18-909c-4973-8ea9-a51acde54482/main.assets/gr1_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig1-title" class="figure__title"><span class="label figure__label">Figure&nbsp;1</span> <span class="figure__title__text">Pooled CRISPRi screens reveal genetic determinants of substitution prime editing outcomes</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig1-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0015" role="paragraph">(A) Prime editing with the PE2 system is mediated by the PE2 enzyme (<i>Streptococcus pyogenes</i> Cas9 [SpCas9] H840A nickase fused to an engineered reverse transcriptase) and a prime editing guide RNA (pegRNA). The PE3 system uses an additional single guide RNA (sgRNA) to nick the non-edited strand and yield higher editing efficiency. PBS, primer binding site; RT template, reverse transcription template.</div><div id="fspara0020" role="paragraph">(B) Overview of prime editing Repair-seq screens. CRISPRi cells are transduced with a library of CRISPRi sgRNAs and a pre-validated prime edit site, then transfected with prime editors targeting the edit site. Paired-end sequencing of CRISPRi sgRNA identities and prime edited sites links each genetic perturbation with the associated editing outcome.</div><div id="fspara0025" role="paragraph">(C) Effect of each CRISPRi sgRNA on the intended G⋅C-to-C⋅G prime edit at the targeted edit site in Repair-seq CRISPRi screens using PE2 in K562 cells.</div><div id="fspara0030" role="paragraph">(D) Effect of CRISPRi sgRNAs on the intended edit in all screen conditions. Black dots represent individual non-targeting sgRNAs, black lines show the mean of all non-targeting sgRNAs, and gray shading represents kernel density estimates of the distributions of all sgRNAs.</div><div id="fspara0035" role="paragraph">(E–G) Comparisons of gene-level effects of CRISPRi targeting on the intended G⋅C-to-C⋅G prime edit across different screen conditions. (E) K562 PE2 versus HeLa PE2. (F) K562 PE3+50 versus HeLa PE3+50. (G) K562 PE2 versus K562 PE3+50. The effect of each gene is calculated as the average log<sub>2</sub> fold change in frequency from non-targeting sgRNAs for the two most extreme sgRNAs targeting the gene. Dots represent the mean of n&nbsp;= 2 independent replicates for each cell type, and bars show the range of values spanned by the replicates. Black dots represent 20 random sets of three non-targeting sgRNAs.</div></div></div></div></figcaption></figure></div><div id="p0045" role="paragraph"><i>In&nbsp;vitro</i> experiments provide support for the early steps of prime editing (<span class="dropBlock reference-citations"><a id="body-ref-sref3-4" href="#" role="doc-biblioref" data-xml-rid="bib3" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref3-4" aria-controls="sref3-4">Anzalone et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref3-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">3.</div><div>Anzalone, A.V. ∙ Randolph, P.B. ∙ Davis, J.R. ...</div><div><strong>Search-and-replace genome editing without double-strand breaks or donor DNA</strong></div><div><em>Nature.</em> 2019; <strong>576</strong>:149-157</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-1711-4" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-85074571375" target="_blank" aria-label="Open Scopus in new tab">Scopus (2574)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31634902/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-1711-4&amp;pmid=31634902" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), but mechanisms downstream of 3′ flap synthesis remain speculative. According to the current model, the newly synthesized 3′ flap displaces an adjacent strand of genomic DNA through flap interconversion (<a id="crosref0280" href="#figs1">Figure&nbsp;S1</a>A). Excision of the displaced 5′ flap then allows ligation of the edited sequence into the genome. Nicking the non-edited strand in the PE3 system is thought to induce cellular replacement of the non-edited strand and thus promote installation of the edited sequence in both strands.</div><div class="figure-wrap"><figure id="figs1" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.09.018/asset/2e559411-23d0-4497-9e9c-8853f668cf79/main.assets/figs1_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs1" src="/cms/10.1016/j.cell.2021.09.018/asset/f48235dc-1c84-4b38-93cf-3568b4f3a08e/main.assets/figs1.jpg" height="922" width="670" aria-labelledby="figs1-title figs1-content" data-viewer-src="/cms/10.1016/j.cell.2021.09.018/asset/2e559411-23d0-4497-9e9c-8853f668cf79/main.assets/figs1_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs1-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S1</span> <span class="figure__title__text">Design and results of Repair-seq screens for substitution prime editing outcomes, related to <a id="crosref1745" href="#fig1">Figure&nbsp;1</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs1-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0255" role="paragraph">(A) Presumed model by which the reverse-transcribed 3′ DNA flap is permanently incorporated into the genome during prime editing (<span class="dropBlock reference-citations"><a id="body-ref-sref3-5" href="#" role="doc-biblioref" data-xml-rid="bib3" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref3-5" aria-controls="sref3-5">Anzalone et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref3-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">3.</div><div>Anzalone, A.V. ∙ Randolph, P.B. ∙ Davis, J.R. ...</div><div><strong>Search-and-replace genome editing without double-strand breaks or donor DNA</strong></div><div><em>Nature.</em> 2019; <strong>576</strong>:149-157</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-1711-4" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-85074571375" target="_blank" aria-label="Open Scopus in new tab">Scopus (2574)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31634902/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-1711-4&amp;pmid=31634902" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="fspara0260" role="paragraph">(B) Installation of a G⋅C-to-C⋅G edit within a lentivirally integrated <i>HBB</i> sequence using SaPE2 and Sa-pegRNAs in HEK293T cells. PBS, primer binding site. Data represent the mean of n&nbsp;= 3 independent replicates.</div><div id="fspara0265" role="paragraph">(C) Design of the prime editing Repair-seq lentiviral vector (pPC1000, additional details and full sequence information in <a id="crosref1755" href="#sec-5">STAR Methods</a>). In Repair-seq screens, a 453-bp region containing CRISPRi sgRNA sequence and prime editing outcome is amplified from genomic DNA for paired-end Illumina sequencing. The CRISPRi sgRNA is sequenced with a 44-nt Illumina forward read (R1), and the prime edited site (including&nbsp;+50 and –50 nick sites) is sequenced with a 263-nt Illumina reverse read (R2). Black triangles indicate positions of SaPE2-induced nicks programmed by Sa-pegRNA and Sa-sgRNAs. Sizes of all vector components are to scale.</div><div id="fspara0270" role="paragraph">(D) Schematic of PE2, PE3+50, and PE3–50 prime editing configurations with SaPE2 protein (SaCas9 N580A fused to an engineered MMLV RT).</div><div id="fspara0275" role="paragraph">(E) Validation of intended G⋅C-to-C⋅G editing at the lentivirally integrated Repair-seq edit site in HeLa cells expressing dCas9–BFP–KRAB cells. Bars represent the mean of n&nbsp;= 2 independent replicates.</div><div id="fspara0280" role="paragraph">(F) Prime editing at the Repair-seq edit site with blasticidin selection in HeLa cells expressing dCas9–BFP–KRAB. SaPE2–P2A–BlastR prime editor was used for all conditions. Bars represent the mean of n&nbsp;= 2 independent replicates.</div><div id="fspara0285" role="paragraph">(G) Functional annotation classes of the genes targeted by the pooled CRISPRi sgRNA library used in Repair-seq screens.</div><div id="fspara0290" role="paragraph">(H–K) Knockdown of <i>MSH2</i>, <i>MSH6</i>, <i>MLH1</i>, and <i>PMS2</i> increases the frequency of the intended&nbsp;+6 G⋅C-to-C⋅G prime edit in all Repair-seq screens. Dots represent individual CRISPRi sgRNAs.</div></div></div></div></figcaption></figure></div><div id="p0050" role="paragraph">The presumed involvement of cellular factors in these steps motivated us to study the roles of DNA repair mechanisms in prime editing and to develop improved prime editing systems through manipulation of those processes. Here, we used pooled CRISPR interference (CRISPRi)-based screens to systematically probe the effect of 476 genes involved in DNA repair and associated processes on substitution prime editing outcomes. We discovered that specific DNA mismatch repair (MMR) genes strongly suppress prime editing efficiency and promote indel formation. Consistent with a model in which MMR reverts heteroduplex DNA formed during prime editing, we identified classes of prime edits that are less vulnerable to MMR activity and are therefore generated more efficiently. Integrating these findings, we developed improved prime editing systems through transient expression of a dominant negative MMR protein (MLH1dn). In six MMR-proficient cell types, including induced pluripotent stem cells (iPSCs) and primary T&nbsp;cells, these PE4 (PE2+MLH1dn) and PE5 (PE3+MLH1dn) systems enhanced editing efficiency over PE2 and PE3 by an average of 7.7-fold and 2.0-fold, respectively, and increased edit/indel ratios (outcome purity) by 3.4-fold. Transient co-expression of MLH1dn did not result in detected changes to microsatellite repeat length, a clinical biomarker of MMR proficiency (<span class="dropBlock reference-citations"><a id="body-ref-sref66-1" href="#" role="doc-biblioref" data-xml-rid="bib66" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref66-1" aria-controls="sref66-1">Umar et&nbsp;al., 2004</a><div class="dropBlock__holder" data-db-target-of="sref66-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">68.</div><div>Umar, A. ∙ Boland, C.R. ∙ Terdiman, J.P. ...</div><div><strong>Revised Bethesda Guidelines for hereditary nonpolyposis colorectal cancer (Lynch syndrome) and microsatellite instability</strong></div><div><em>J.&nbsp;Natl. Cancer Inst.</em> 2004; <strong>96</strong>:261-268</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/jnci/djh034" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-10744233937" target="_blank" aria-label="Open Scopus in new tab">Scopus (2668)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/14970275/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fjnci%2Fdjh034&amp;pmid=14970275" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Strategic installation of additional silent mutations nearby an intended edit can also improve prime editing efficiency by evading MMR recognition, even in the absence of MLH1dn. Finally, we engineered an optimized “PEmax” prime editor architecture that further increased editing efficiency in synergy with PE4, PE5, and engineered pegRNAs (epegRNAs) (<span class="dropBlock reference-citations"><a id="body-ref-sref42-1" href="#" role="doc-biblioref" data-xml-rid="bib42" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref42-1" aria-controls="sref42-1">Nelson et al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref42-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">43.</div><div>Nelson, J.W. ∙ Randolph, P.B. ∙ Shen, S.P. ...</div><div><strong>Engineered pegRNAs improve prime editing efficiency</strong></div><div><em>Nat. Biotechnol.</em> 2021;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41587-021-01039-7" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-85116367544" target="_blank" aria-label="Open Scopus in new tab">Scopus (297)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/34608327/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41587-021-01039-7&amp;pmid=34608327" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). These findings deepen our understanding of prime editing and establish prime editing systems with substantially improved efficiency and outcome purity across 191 different edits at 20 loci in seven mammalian cell types.</div></section><section id="sec-2"><h2>Results</h2><section id="sec-2-1"><h3>Design of a pooled CRISPRi screen for prime editing outcomes</h3><div id="p0055" role="paragraph">We reasoned that identifying genetic determinants of prime editing sequence outcomes, including the original sequence, the desired edit, and indels, could inform strategies to maximize efficiency and minimize unwanted byproducts. We therefore used a genetic screening approach to study prime editing. This method, called Repair-seq, measures the effects of many loss-of-function perturbations on the outcomes of genome editing experiments by linking the identity of CRISPRi sgRNAs to edited sites in pooled screens (<span class="dropBlock reference-citations"><a id="body-ref-sref25-1" href="#" role="doc-biblioref" data-xml-rid="bib25" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref25-1" aria-controls="sref25-1">Hussmann et al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref25-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">25.</div><div>Hussmann, J.A. ∙ Ling, J. ∙ Ravisankar, P. ...</div><div><strong>Mapping the genetic landscape of DNA double-strand break repair</strong></div><div><em>Cell.</em> 2021;</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=10.1016%2Fj.cell.2021.10.002&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=10.1016%2Fj.cell.2021.10.002&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-85117855979" target="_blank" aria-label="Open Scopus in new tab">Scopus (80)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/34653350/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2021.10.002&amp;pmid=34653350" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) (<a id="crosref0300" href="#fig1">Figure&nbsp;1</a>B). Briefly, a library of sgRNAs are transduced into cells expressing the CRISPRi effector (dCas9–KRAB) such that most cells receive only one sgRNA, causing the knockdown of one gene per cell. After genome editing occurs at an adjacent target site delivered on the same lentiviral cassette, paired-end sequencing enables the frequency of each editing outcome to be measured for each linked CRISPRi perturbation.</div><div id="p0060" role="paragraph">To enable Repair-seq screens for prime editing outcomes, we first made prime editing and CRISPRi orthogonal. Typically, these systems both rely on <i>Streptococcus pyogenes</i> Cas9 (SpCas9). We constructed an SaPE2 prime editor variant by replacing the SpCas9 nickase domain in PE2 with <i>Staphylococcus aureus</i> Cas9 (SaCas9) N580A nickase (<span class="dropBlock reference-citations"><a id="body-ref-sref50" href="#" role="doc-biblioref" data-xml-rid="bib50" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref50" aria-controls="sref50">Ran et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref50"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">52.</div><div>Ran, F.A. ∙ Cong, L. ∙ Yan, W.X. ...</div><div><strong>In&nbsp;vivo genome editing using Staphylococcus aureus Cas9</strong></div><div><em>Nature.</em> 2015; <strong>520</strong>:186-191</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature14299" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-84927514894" target="_blank" aria-label="Open Scopus in new tab">Scopus (2080)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25830891/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature14299&amp;pmid=25830891" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and verified SaPE2 editing activity with orthogonal <i>S.&nbsp;aureus</i> pegRNAs (Sa-pegRNAs) (<a id="crosref0310" href="#figs1">Figure&nbsp;S1</a>B). Next, we built a Repair-seq vector for screening with a composite SaPE2 edit site. This site comprised a target protospacer that is efficiently prime edited in HEK293T cells (<a id="crosref0315" href="#figs1">Figure&nbsp;S1</a>B) and two flanking protospacers that allow complementary-strand nicks 50&nbsp;bp downstream (+50 nick) or upstream (−50 nick) of the target (<a id="crosref0320" href="#fig1">Figures 1</a>B and <a id="crosref0325" href="#figs1">S1</a>C). This design supports SaPE2 prime editing in three configurations: PE2, PE3 with a&nbsp;+50 nick (PE3+50), or PE3 with a −50 nick (PE3−50) (<a id="crosref0330" href="#figs1">Figure&nbsp;S1</a>D). In a validated HeLa CRISPRi cell line (<span class="dropBlock reference-citations"><a id="body-ref-sref18-1" href="#" role="doc-biblioref" data-xml-rid="bib18" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref18-1" aria-controls="sref18-1">Gilbert et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref18-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">18.</div><div>Gilbert, L.A. ∙ Larson, M.H. ∙ Morsut, L. ...</div><div><strong>CRISPR-mediated modular RNA-guided regulation of transcription in eukaryotes</strong></div><div><em>Cell.</em> 2013; <strong>154</strong>:442-451</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=10.1016%2Fj.cell.2013.06.044&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=10.1016%2Fj.cell.2013.06.044&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-84880571335" target="_blank" aria-label="Open Scopus in new tab">Scopus (2710)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23849981/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2013.06.044&amp;pmid=23849981" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), we observed G⋅C-to-C⋅G prime editing efficiencies up to 9.4% with PE2, 15% with PE3+50, and 3.5% with PE3−50 (<a id="crosref0340" href="#figs1">Figures S1</a>E and S1F) at this composite edit site, establishing a screening assay well suited to detect increases or decreases in editing from CRISPRi perturbations.</div></section><section id="sec-2-2"><h3>Identification of DNA repair genes that affect prime editing outcomes</h3><div id="p0065" role="paragraph">We performed Repair-seq screens of prime editing outcomes with PE2 and PE3+50 in K562 and HeLa cells and with PE3−50 in HeLa cells. We transduced a Repair-seq library of 1,513 sgRNAs targeting 476 genes (enriched for roles in DNA repair and associated processes; <a id="crosref0345" href="#figs1">Figure&nbsp;S1</a>G) and 60 non-targeting control sgRNAs into human K562 and HeLa CRISPRi cell lines (<span class="dropBlock reference-citations"><a id="body-ref-sref19-1" href="#" role="doc-biblioref" data-xml-rid="bib19" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref19-1" aria-controls="sref19-1">Gilbert et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref19-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">19.</div><div>Gilbert, L.A. ∙ Horlbeck, M.A. ∙ Adamson, B. ...</div><div><strong>Genome-Scale CRISPR-Mediated Control of Gene Repression and Activation</strong></div><div><em>Cell.</em> 2014; <strong>159</strong>:647-661</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=10.1016%2Fj.cell.2014.09.029&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=10.1016%2Fj.cell.2014.09.029&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-84908352138" target="_blank" aria-label="Open Scopus in new tab">Scopus (1883)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25307932/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2014.09.029&amp;pmid=25307932" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref18-2" href="#" role="doc-biblioref" data-xml-rid="bib18" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref18-2" aria-controls="sref18-2">Gilbert et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref18-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">18.</div><div>Gilbert, L.A. ∙ Larson, M.H. ∙ Morsut, L. ...</div><div><strong>CRISPR-mediated modular RNA-guided regulation of transcription in eukaryotes</strong></div><div><em>Cell.</em> 2013; <strong>154</strong>:442-451</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=10.1016%2Fj.cell.2013.06.044&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=10.1016%2Fj.cell.2013.06.044&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-84880571335" target="_blank" aria-label="Open Scopus in new tab">Scopus (2710)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23849981/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2013.06.044&amp;pmid=23849981" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref25-2" href="#" role="doc-biblioref" data-xml-rid="bib25" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref25-2" aria-controls="sref25-2">Hussmann et al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref25-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">25.</div><div>Hussmann, J.A. ∙ Ling, J. ∙ Ravisankar, P. ...</div><div><strong>Mapping the genetic landscape of DNA double-strand break repair</strong></div><div><em>Cell.</em> 2021;</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=10.1016%2Fj.cell.2021.10.002&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=10.1016%2Fj.cell.2021.10.002&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-85117855979" target="_blank" aria-label="Open Scopus in new tab">Scopus (80)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/34653350/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2021.10.002&amp;pmid=34653350" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) (<a id="crosref0365" href="#fig1">Figure&nbsp;1</a>B; <a id="crosref0370" href="#mmc1">Table S1</a>). Next, we transfected these cells with SaPE2, Sa-pegRNA, and Sa-sgRNA plasmids that program a G⋅C-to-C⋅G transversion at the co-transduced edit site. Finally, we extracted genomic DNA, amplified the CRISPRi sgRNA and edit site by PCR, and performed paired-end sequencing to measure the distribution of editing outcomes for each genetic perturbation (<a id="crosref0375" href="#figs1">Figure&nbsp;S1</a>C). To interpret the resulting data, we compared the frequencies of editing outcomes from cells containing a gene-targeting CRISPRi sgRNA to the corresponding&nbsp;frequencies from cells containing non-targeting sgRNA controls. Reduction in an outcome’s frequency upon gene knockdown suggests that the gene’s activity promotes formation of the outcome, while an increase in frequency suggests that the gene’s activity suppresses the outcome.</div><div id="p0070" role="paragraph">We first examined the effect of gene knockdowns on the frequency of the intended G⋅C-to-C⋅G edit. In cells with non-targeting CRISPRi sgRNAs, 4.3%–4.9% (K562) and 8.5%–8.7% (HeLa) of sequencing reads contained exactly the intended edit following PE2 editing (<a id="crosref0380" href="#fig1">Figures 1</a>C and 1D). These levels increased to 14%–16% (K562) and 14%–16% (HeLa) for PE3+50 but decreased to 2.1%–2.2% (HeLa) for PE3−50. Across all screen conditions, CRISPRi targeting of <i>MSH2</i>, <i>MSH6</i>, <i>MLH1</i>, and <i>PMS2</i>, components of the MutSα–MutLα MMR complex (<span class="dropBlock reference-citations"><a id="body-ref-sref28-1" href="#" role="doc-biblioref" data-xml-rid="bib28" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref28-1" aria-controls="sref28-1">Iyer et&nbsp;al., 2006</a><div class="dropBlock__holder" data-db-target-of="sref28-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">28.</div><div>Iyer, R.R. ∙ Pluciennik, A. ∙ Burdett, V. ...</div><div><strong>DNA mismatch repair: functions and mechanisms</strong></div><div><em>Chem. Rev.</em> 2006; <strong>106</strong>:302-323</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/cr0404794" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-33644626371" target="_blank" aria-label="Open Scopus in new tab">Scopus (732)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16464007/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Fcr0404794&amp;pmid=16464007" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref32-1" href="#" role="doc-biblioref" data-xml-rid="bib32" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref32-1" aria-controls="sref32-1">Kunkel and Erie, 2005</a><div class="dropBlock__holder" data-db-target-of="sref32-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">32.</div><div>Kunkel, T.A. ∙ Erie, D.A.</div><div><strong>DNA mismatch repair</strong></div><div><em>Annu. Rev. Biochem.</em> 2005; <strong>74</strong>:681-710</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1146/annurev.biochem.74.082803.133243" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-22244492665" target="_blank" aria-label="Open Scopus in new tab">Scopus (1084)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15952900/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1146%2Fannurev.biochem.74.082803.133243&amp;pmid=15952900" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref34-1" href="#" role="doc-biblioref" data-xml-rid="bib34" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref34-1" aria-controls="sref34-1">Li, 2008</a><div class="dropBlock__holder" data-db-target-of="sref34-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">34.</div><div>Li, G.-M.</div><div><strong>Mechanisms and functions of DNA mismatch repair</strong></div><div><em>Cell Res.</em> 2008; <strong>18</strong>:85-98</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/cr.2007.115" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-38049125557" target="_blank" aria-label="Open Scopus in new tab">Scopus (1020)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/18157157/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fcr.2007.115&amp;pmid=18157157" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), substantially increased editing efficiency by up to 5.8-fold for PE2, 2.5-fold for PE3+50, and 2.0-fold for PE3−50 (<a id="crosref0400" href="#fig1">Figures 1</a>E–1G and <a id="crosref0405" href="#figs1">S1</a>H–S1K; <a id="crosref0410" href="#mmc2">Table S2</a>). Knockdown of <i>EXO1</i>, an exonuclease with a role in MMR (<span class="dropBlock reference-citations"><a id="body-ref-sref17-1" href="#" role="doc-biblioref" data-xml-rid="bib17" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref17-1" aria-controls="sref17-1">Genschel et&nbsp;al., 2002</a><div class="dropBlock__holder" data-db-target-of="sref17-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">17.</div><div>Genschel, J. ∙ Bazemore, L.R. ∙ Modrich, P.</div><div><strong>Human exonuclease I is required for 5′ and 3′ mismatch repair</strong></div><div><em>J.&nbsp;Biol. Chem.</em> 2002; <strong>277</strong>:13302-13311</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=10.1074%2Fjbc.M111854200&amp;cf=fulltext&amp;site=jbc-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=10.1074%2Fjbc.M111854200&amp;cf=pdf&amp;site=jbc-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-0037066720" target="_blank" aria-label="Open Scopus in new tab">Scopus (204)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11809771/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1074%2Fjbc.M111854200&amp;pmid=11809771" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), also increased intended PE2 editing efficiency by up to 2.3-fold. In contrast, knockdown of <i>LIG1</i>, a nick-sealing DNA ligase (<span class="dropBlock reference-citations"><a id="body-ref-optvIsUdM54jg" href="#" role="doc-biblioref" data-xml-rid="bib72" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="optvIsUdM54jg" aria-controls="optvIsUdM54jg">Pascal et al., 2004</a><div class="dropBlock__holder" data-db-target-of="optvIsUdM54jg"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">47.</div><div>Pascal, J.M. ∙ O’Brien, P.J. ∙ Tomkinson, A.E. ...</div><div><strong>Human DNA ligase I completely encircles and partially unwinds nicked DNA</strong></div><div><em>Nature.</em> 2004; <strong>432</strong>:473-478</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature03082" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-9644289536" target="_blank" aria-label="Open Scopus in new tab">Scopus (274)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15565146/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature03082&amp;pmid=15565146" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), and of <i>FEN1</i>, a 5′ flap endonuclease (<span class="dropBlock reference-citations"><a id="body-ref-optVS2MxfBOKg" href="#" role="doc-biblioref" data-xml-rid="bib73" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="optVS2MxfBOKg" aria-controls="optVS2MxfBOKg">Liu et al., 2004</a><div class="dropBlock__holder" data-db-target-of="optVS2MxfBOKg"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">37.</div><div>Liu, Y. ∙ Kao, H.-I. ∙ Bambara, R.A.</div><div><strong>Flap endonuclease 1: a central component of DNA metabolism</strong></div><div><em>Annu. Rev. Biochem.</em> 2004; <strong>73</strong>:589-615</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1146/annurev.biochem.73.012803.092453" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-3943086339" target="_blank" aria-label="Open Scopus in new tab">Scopus (320)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15189154/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1146%2Fannurev.biochem.73.012803.092453&amp;pmid=15189154" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), reduced the frequency of intended editing, consistent with their previously proposed roles in nick ligation and 5′ flap excision during prime editing (<span class="dropBlock reference-citations"><a id="body-ref-sref3-6" href="#" role="doc-biblioref" data-xml-rid="bib3" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref3-6" aria-controls="sref3-6">Anzalone et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref3-6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">3.</div><div>Anzalone, A.V. ∙ Randolph, P.B. ∙ Davis, J.R. ...</div><div><strong>Search-and-replace genome editing without double-strand breaks or donor DNA</strong></div><div><em>Nature.</em> 2019; <strong>576</strong>:149-157</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-1711-4" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-85074571375" target="_blank" aria-label="Open Scopus in new tab">Scopus (2574)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31634902/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-1711-4&amp;pmid=31634902" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Together, these data suggest that MMR activity antagonizes the installation of point mutations by prime editing.</div><div id="p0075" role="paragraph">In addition to the intended edit, Repair-seq screens identified four primary categories of editing byproducts: deletions (<a id="crosref0425" href="#fig2">Figure&nbsp;2</a>A), tandem duplications (<a id="crosref0430" href="#fig2">Figure&nbsp;2</a>B), and two classes of outcomes containing unintended sequence from the pegRNA (<a id="crosref0435" href="#fig2">Figures 2</a>C and 2D). We observed low baseline frequencies of total unintended edits from PE2 (0.31% in K562, 0.60% in HeLa; <a id="crosref0440" href="#fig2">Figure&nbsp;2</a>E) but more frequent and diverse unintended byproducts from PE3−50 (58% in HeLa; <a id="crosref0445" href="#figs2">Figure&nbsp;S2</a>A) and PE3+50 (8.2% in K562, 9.5% in HeLa; <a id="crosref0450" href="#fig2">Figure&nbsp;2</a>F). The baseline frequencies and genetic modulators of these categories varied across PE2, PE3+50, and PE3−50 screens, providing a rich set of observations of how different prime editing configurations are processed (<a id="crosref0455" href="#figs2">Figures S2</a>A–S2I).</div><div class="figure-wrap"><figure id="fig2" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.09.018/asset/18cdba41-5034-43d2-a986-d039a388efc9/main.assets/gr2_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr2" src="/cms/10.1016/j.cell.2021.09.018/asset/2086f07d-cded-4210-9f00-7debf551e233/main.assets/gr2.jpg" height="890" width="657" aria-labelledby="fig2-title fig2-content" data-viewer-src="/cms/10.1016/j.cell.2021.09.018/asset/18cdba41-5034-43d2-a986-d039a388efc9/main.assets/gr2_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig2-title" class="figure__title"><span class="label figure__label">Figure&nbsp;2</span> <span class="figure__title__text">Genetic modulators of unintended prime editing outcomes</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig2-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0045" role="paragraph">(A–D) Representative examples of four categories of unintended prime editing outcomes observed in CRISPRi screens. Blue and orange lines between the editing outcome and the genome or pegRNA depict local sequence alignments. X’s represent mismatches in alignments, gaps represent insertions, and gray boxes represent the location of the programmed edit. Red and cyan rectangles on the genome mark SaCas9 protospacers and PAMs, and black vertical lines mark the locations of SaCas9 nick sites.</div><div id="fspara0050" role="paragraph">(E and F) Summary of editing outcome categories observed in PE2 screens (E) and in PE3+50 screens (F) in K562 cells. Plotted quantities are the mean ± SD of all sgRNAs for each indicated gene (60 non-targeting sgRNAs and three sgRNAs per targeted gene), averaged across n&nbsp;= 2 independent replicates.</div><div id="fspara0055" role="paragraph">(G and H) Comparison of the effects of knockdown of all genes targeted in CRISPRi screens on the frequency of joining of reverse-transcribed sequence at unintended location (G) or of deletions (H) from PE3+50. The effect of each gene is calculated as the average log<sub>2</sub> fold change in frequency from non-targeting sgRNAs for the two most extreme sgRNAs targeting the gene. Dots represent the mean of n&nbsp;= 2 independent replicates for each cell type, and bars show the range of values spanned by the replicates. Black dots represent 20 random sets of three non-targeting sgRNAs.</div><div id="fspara0060" role="paragraph">(I) Top: frequency of deletion as a function of genomic position relative to programmed PE3+50 nicks (dashed vertical lines) in K562 screen replicate 1 across all reads for indicated sets of CRISPRi sgRNAs (black line, 60 non-targeting sgRNAs; orange and green lines, three sgRNAs targeting each of <i>MSH2</i>, <i>MSH6</i>, <i>MLH1</i>, and <i>PMS2</i>). Bottom: log<sub>2</sub> fold change in frequency of deletion as a function of genomic position for <i>MSH2</i>, <i>MSH6</i>, <i>MLH1</i>, and <i>PMS2</i> sgRNAs compared to non-targeting sgRNAs.</div><div id="fspara0065" role="paragraph">(J) Effect of gene knockdowns on the fraction of all observed deletions that remove sequence at least 25 nt outside of programmed PE3+50 nicks in K562 screens. Each dot represents all reads for sgRNAs targeting an individual gene. Black dots represent 20 sets of three random non-targeting sgRNAs.</div></div></div></div></figcaption></figure></div><div class="figure-wrap"><figure id="figs2" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.09.018/asset/fbe51ba8-7942-45ef-802e-ebcd90dc4105/main.assets/figs2_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs2" src="/cms/10.1016/j.cell.2021.09.018/asset/65551f90-539a-42a2-9f72-7120ba04f656/main.assets/figs2.jpg" height="940" width="656" aria-labelledby="figs2-title figs2-content" data-viewer-src="/cms/10.1016/j.cell.2021.09.018/asset/fbe51ba8-7942-45ef-802e-ebcd90dc4105/main.assets/figs2_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs2-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S2</span> <span class="figure__title__text">Genetic modulators of unintended prime editing outcomes, related to <a id="crosref1760" href="#fig2">Figure&nbsp;2</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs2-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0300" role="paragraph">(A) Overview of PE3–50 outcomes in HeLa CRISPRi screens.&nbsp;<i>TP53BP1</i>&nbsp;knockdown dramatically reduces formation of all unintended editing outcomes.</div><div id="fspara0305" role="paragraph">(B) Additional details of PE2 outcomes in K562 CRISPRi screens, supplementing&nbsp;<a id="optfKpXrPMZbz" href="#fig2">Figure 2</a>E.</div><div id="fspara0310" role="paragraph">(C) Additional details of PE3+50 outcomes in K562 CRISPRi screens, supplementing information in <a id="crosref1770" href="#fig2">Figure&nbsp;2</a>F.</div><div id="fspara0315" role="paragraph">(D–I) Comparisons of effects of gene knockdown on frequencies of indicated outcome categories in indicated screen conditions. Plotted quantities are the mean of the log<sub>2</sub> fold changes from non-targeting sgRNAs for the two most extreme sgRNAs per gene, averaged over n&nbsp;= 2 independent replicates per condition. Error bars mark the range of values spanned by the replicates. Black dots represent 20 random sets of three non-targeting sgRNAs. (D) <i>MSH2</i>, <i>MLH1</i>, and <i>PMS2</i> knockdown produce larger fold changes in installation of additional edits than in intended edits in K562 PE2 screens. (E) Unintended joining of reverse transcribed sequence in PE2 screens in K562 and HeLa cells are most increased by knockdown of Fanconi anemia genes (red) as well as a set of <i>RAD51</i> homologs and other genes involved in homologous recombination (blue). (F) Deletions in PE2 screens in K562 and HeLa cells are most increased by a set of <i>RAD51</i> homologs and other genes involved in homologous recombination (blue). (G) In addition to <i>MSH2</i>, <i>MLH1</i>, and <i>PMS2</i>, <i>HLTF</i> knockdown produces larger fold changes in installation of additional edits than in intended edits in K562 PE3+50 screens. (H) Tandem duplications in HeLa and K562 PE3+50 screens are most decreased by knockdown of POLD and RFC subunits. (I) Deletions in HeLa PE3+50 and PE3–50 screens have dramatically divergent genetic regulators, highlighting differences in the processing of the different overhang configurations.</div></div></div></div></figcaption></figure></div><div id="p0080" role="paragraph">Two of these observations informed models for the role of MMR activity during prime editing. First, one unintended outcome contained the intended G⋅C-to-C⋅G edit as well as an additional base substitution and a 1-nt insertion near the target site (<a id="crosref0460" href="#fig2">Figure&nbsp;2</a>C). The sequence around these additional mutations perfectly matched 9 nt at the 3′ end of the pegRNA scaffold sequence, consistent with reverse transcription into the pegRNA scaffold and incorporation of the resulting 3′ DNA flap into partially homologous genomic sequence. Recoding the pegRNA scaffold to avoid sequence homology with the genomic target reduced the frequency of this outcome category (<a id="crosref0465" href="#figs3">Figures S3</a>A and S3B), suggesting a general approach to eliminate this class of editing byproduct. Notably, we observed that knockdown of MMR genes increased the frequency of this editing byproduct from 0.08% to 2.0% in PE2 K562 screens (<a id="crosref0470" href="#fig2">Figures 2</a>E and <a id="crosref0475" href="#figs2">S2</a>D). MMR thus suppresses the formation of this outcome to a larger extent than the intended edit, indicating that distinct prime editing intermediates can differ in the extent to which they are processed by MMR.</div><div class="figure-wrap"><figure id="figs3" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.09.018/asset/be01a2e8-7628-41cb-9824-8cb01efbb069/main.assets/figs3_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs3" src="/cms/10.1016/j.cell.2021.09.018/asset/bd43f744-f2ab-4f8e-b309-a1dc5acd0c3a/main.assets/figs3.jpg" height="942" width="684" aria-labelledby="figs3-title figs3-content" data-viewer-src="/cms/10.1016/j.cell.2021.09.018/asset/be01a2e8-7628-41cb-9824-8cb01efbb069/main.assets/figs3_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs3-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S3</span> <span class="figure__title__text">Validation of prime editing Repair-seq screen results, related to <a id="crosref1775" href="#fig2">Figures 2</a> and <a id="crosref1780" href="#fig3">3</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs3-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0325" role="paragraph">(A–B) Top: alignment of Sa-pegRNAs, their templated 3′ DNA flaps following SaPE2 reverse transcription, and the genomic target sequence. Compared to the Sa-pegRNA used in Repair-seq screens (A), an Sa-pegRNA with recoded scaffold sequence (B) templates an extended 3′ DNA flap with reduced homology with genomic target sequence. The recoded Sa-pegRNA contains 2 base pair changes that preserve base pairing interactions within the scaffold. Reverse transcription of the Sa-pegRNA scaffold can generate a misextended 3′ flap that is incorporated into the genome. Vertical lines depict base pairing. Red X’s depict mismatches between the misextended reverse-trancribed 3′ flap and genomic sequence. Bottom: frequencies of editing outcome categories observed at the screen edit site from arrayed PE2 and PE3+50 experiments in HeLa CRISPRi cells. Prime editing with the Sa-pegRNA used in Repair-seq screens (A) or a recoded Sa-pegRNA (B) results in different frequencies of installation of unintended edits from nearly matched scaffold. Plotted quantities are the mean ± SD of n&nbsp;= 4 independent replicates for cells containing a <i>MSH2</i> or non-targeting CRISPRi sgRNA.</div><div id="fspara0330" role="paragraph">(C) Mechanism of DNA mismatch repair in humans.</div><div id="fspara0335" role="paragraph">(D) Mismatch repair of a prime editing heteroduplex intermediate induces indel byproducts, potentially through MutLα endonuclease activity at the target locus or excision from these non-programmed nicks and subsequent repair of the resulting intermediates.</div><div id="fspara0340" role="paragraph">(E) Knockdown efficiency of siRNA treatment relative to a non-targeting siRNA control in HEK293T cells. Cells were transfected with siRNAs, incubated for 3&nbsp;days, transfected with PE2, pegRNAs, and the same siRNAs, then incubated for another 3&nbsp;days before relative RNA abundances were assayed by RT-qPCR. NT, non-targeting. Data represent the mean of n&nbsp;= 3 independent replicates. Each dot represents the mean of n&nbsp;= 3 technical replicates. Data supplements information in <a id="optnnrdyHbscv" href="#fig3">Figure 3</a>C.</div><div id="fspara0345" role="paragraph">(F) Editing in HEK293T cells co-transfected with prime editor components and siRNAs. Cells were not pre-treated with siRNAs before transfection with prime editors. Bars represent the mean of n&nbsp;= 3 independent replicates.</div></div></div></div></figcaption></figure></div><div id="p0085" role="paragraph">Second, MMR knockdown reduced the frequency of most categories of unintended outcomes from PE3+50 (<a id="crosref0480" href="#fig2">Figures 2</a>F–2H, <a id="crosref0485" href="#figs2">S2</a>H, and S2I), suggesting that transiently inhibiting some MMR activities may increase both the efficiency and outcome purity of prime editing. The most abundant class of unintended PE3+50 outcomes contained sequence from the reverse-transcribed 3′ DNA flap that does not rejoin genomic sequence at the intended flap annealing location (5.1% of non-targeting reads in K562, 3.8% in HeLa; <a id="crosref0490" href="#fig2">Figure&nbsp;2</a>D). In both cell types, knockdown of MMR genes reduced the frequency of unintended flap rejoining outcomes, by up to 1.7-fold (<a id="crosref0495" href="#fig2">Figure&nbsp;2</a>G). Similarly, MMR knockdown substantially reduced deletions from PE3+50 by up to 3.7-fold (<a id="crosref0500" href="#fig2">Figure&nbsp;2</a>H). Intriguingly, genomic sequence between the two SaPE2-induced nicks was most frequently deleted for PE3+50, but MMR knockdown in K562 cells decreased the frequency of deletions outside of these programmed nicks to a greater extent than deletions between them (<a id="crosref0505" href="#fig2">Figures 2</a>I and 2J), suggesting that MMR activity may generate longer deletion byproducts during prime editing. Finally, tandem duplications of sequence between the nicks, which were common for PE3−50 (<a id="crosref0510" href="#figs2">Figure&nbsp;S2</a>A) but rarer for PE3+50 (0.37% of non-targeting reads in K562, 2.3% in HeLa), were reduced by up to 3.7-fold (K562) and 1.5-fold (HeLa) by MMR knockdown (<a id="crosref0515" href="#fig2">Figures 2</a>F and <a id="crosref0520" href="#figs2">S2</a>H).</div></section><section id="sec-2-3"><h3>Model for MMR of prime editing intermediates</h3><div id="p0090" role="paragraph">The effects of MMR knockdown in these Repair-seq screens led to a working model for the role of MMR during prime editing. In eukaryotes, MMR resolves DNA heteroduplexes containing a base mismatch or small insertion-deletion loop (IDL) by selectively replacing nicked DNA strands (<span class="dropBlock reference-citations"><a id="body-ref-sref28-2" href="#" role="doc-biblioref" data-xml-rid="bib28" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref28-2" aria-controls="sref28-2">Iyer et&nbsp;al., 2006</a><div class="dropBlock__holder" data-db-target-of="sref28-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">28.</div><div>Iyer, R.R. ∙ Pluciennik, A. ∙ Burdett, V. ...</div><div><strong>DNA mismatch repair: functions and mechanisms</strong></div><div><em>Chem. Rev.</em> 2006; <strong>106</strong>:302-323</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/cr0404794" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-33644626371" target="_blank" aria-label="Open Scopus in new tab">Scopus (732)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16464007/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Fcr0404794&amp;pmid=16464007" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref32-2" href="#" role="doc-biblioref" data-xml-rid="bib32" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref32-2" aria-controls="sref32-2">Kunkel and Erie, 2005</a><div class="dropBlock__holder" data-db-target-of="sref32-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">32.</div><div>Kunkel, T.A. ∙ Erie, D.A.</div><div><strong>DNA mismatch repair</strong></div><div><em>Annu. Rev. Biochem.</em> 2005; <strong>74</strong>:681-710</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1146/annurev.biochem.74.082803.133243" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-22244492665" target="_blank" aria-label="Open Scopus in new tab">Scopus (1084)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15952900/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1146%2Fannurev.biochem.74.082803.133243&amp;pmid=15952900" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref34-2" href="#" role="doc-biblioref" data-xml-rid="bib34" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref34-2" aria-controls="sref34-2">Li, 2008</a><div class="dropBlock__holder" data-db-target-of="sref34-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">34.</div><div>Li, G.-M.</div><div><strong>Mechanisms and functions of DNA mismatch repair</strong></div><div><em>Cell Res.</em> 2008; <strong>18</strong>:85-98</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/cr.2007.115" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-38049125557" target="_blank" aria-label="Open Scopus in new tab">Scopus (1020)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/18157157/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fcr.2007.115&amp;pmid=18157157" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) (<a id="crosref0540" href="#figs3">Figure&nbsp;S3</a>C). To initiate MMR, the heteroduplex is first bound by MutSα (MSH2–MSH6) or MutSβ (MSH2–MSH3), which recognize base mismatches and IDLs less than 13 nt in length (<span class="dropBlock reference-citations"><a id="body-ref-sref22-1" href="#" role="doc-biblioref" data-xml-rid="bib22" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref22-1" aria-controls="sref22-1">Gupta et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref22-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">22.</div><div>Gupta, S. ∙ Gellert, M. ∙ Yang, W.</div><div><strong>Mechanism of mismatch recognition revealed by human MutSβ bound to unpaired DNA loops</strong></div><div><em>Nat. Struct. Mol. Biol.</em> 2011; <strong>19</strong>:72-78</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nsmb.2175" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-84855463183" target="_blank" aria-label="Open Scopus in new tab">Scopus (125)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22179786/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnsmb.2175&amp;pmid=22179786" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref67-1" href="#" role="doc-biblioref" data-xml-rid="bib67" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref67-1" aria-controls="sref67-1">Warren et&nbsp;al., 2007</a><div class="dropBlock__holder" data-db-target-of="sref67-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">69.</div><div>Warren, J.J. ∙ Pohlhaus, T.J. ∙ Changela, A. ...</div><div><strong>Structure of the human MutSalpha DNA lesion recognition complex</strong></div><div><em>Mol. Cell.</em> 2007; <strong>26</strong>:579-592</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=10.1016%2Fj.molcel.2007.04.018&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=10.1016%2Fj.molcel.2007.04.018&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-34248572591" target="_blank" aria-label="Open Scopus in new tab">Scopus (286)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17531815/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.molcel.2007.04.018&amp;pmid=17531815" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Next, MSH2 recruits MutLα (PMS2–MLH1), which incises the nick-containing strand around the heteroduplex (<span class="dropBlock reference-citations"><a id="body-ref-sref14" href="#" role="doc-biblioref" data-xml-rid="bib14" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref14" aria-controls="sref14">Fang and Modrich, 1993</a><div class="dropBlock__holder" data-db-target-of="sref14"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">14.</div><div>Fang, W.H. ∙ Modrich, P.</div><div><strong>Human strand-specific mismatch repair occurs by a bidirectional mechanism similar to that of the bacterial reaction</strong></div><div><em>J.&nbsp;Biol. Chem.</em> 1993; <strong>268</strong>:11838-11844</div></div><div class="external-links"><div class="core-xlink-abstract"><a href="/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=10.1016%2FS0021-9258%2819%2950276-9&amp;cf=abstract&amp;site=jbc-site" target="_blank" aria-label="Open Abstract in new tab">Abstract</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=10.1016%2FS0021-9258%2819%2950276-9&amp;cf=pdf&amp;site=jbc-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/8505312/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0021-9258%2819%2950276-9&amp;pmid=8505312" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref30-1" href="#" role="doc-biblioref" data-xml-rid="bib30" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref30-1" aria-controls="sref30-1">Kadyrov et&nbsp;al., 2006</a><div class="dropBlock__holder" data-db-target-of="sref30-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">30.</div><div>Kadyrov, F.A. ∙ Dzantiev, L. ∙ Constantin, N. ...</div><div><strong>Endonucleolytic function of MutLalpha in human mismatch repair</strong></div><div><em>Cell.</em> 2006; <strong>126</strong>:297-308</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=10.1016%2Fj.cell.2006.05.039&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=10.1016%2Fj.cell.2006.05.039&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-33746189409" target="_blank" aria-label="Open Scopus in new tab">Scopus (499)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16873062/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2006.05.039&amp;pmid=16873062" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref49" href="#" role="doc-biblioref" data-xml-rid="bib49" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref49" aria-controls="sref49">Pluciennik et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref49"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">51.</div><div>Pluciennik, A. ∙ Dzantiev, L. ∙ Iyer, R.R. ...</div><div><strong>PCNA function in the activation and strand direction of MutLα endonuclease in mismatch repair</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2010; <strong>107</strong>:16066-16071</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1010662107" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_51_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-77957979862" target="_blank" aria-label="Open Scopus in new tab">Scopus (215)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20713735/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1010662107&amp;pmid=20713735" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref60-1" href="#" role="doc-biblioref" data-xml-rid="bib60" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref60-1" aria-controls="sref60-1">Thomas et&nbsp;al., 1991</a><div class="dropBlock__holder" data-db-target-of="sref60-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">62.</div><div>Thomas, D.C. ∙ Roberts, J.D. ∙ Kunkel, T.A.</div><div><strong>Heteroduplex repair in extracts of human HeLa cells</strong></div><div><em>J.&nbsp;Biol. Chem.</em> 1991; <strong>266</strong>:3744-3751</div></div><div class="external-links"><div class="core-xlink-abstract"><a href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=10.1016%2FS0021-9258%2819%2967858-0&amp;cf=abstract&amp;site=jbc-site" target="_blank" aria-label="Open Abstract in new tab">Abstract</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=10.1016%2FS0021-9258%2819%2967858-0&amp;cf=pdf&amp;site=jbc-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/1995629/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0021-9258%2819%2967858-0&amp;pmid=1995629" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Finally, EXO1 excises the heteroduplex from these incisions (<span class="dropBlock reference-citations"><a id="body-ref-sref17-2" href="#" role="doc-biblioref" data-xml-rid="bib17" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref17-2" aria-controls="sref17-2">Genschel et&nbsp;al., 2002</a><div class="dropBlock__holder" data-db-target-of="sref17-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">17.</div><div>Genschel, J. ∙ Bazemore, L.R. ∙ Modrich, P.</div><div><strong>Human exonuclease I is required for 5′ and 3′ mismatch repair</strong></div><div><em>J.&nbsp;Biol. Chem.</em> 2002; <strong>277</strong>:13302-13311</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=10.1074%2Fjbc.M111854200&amp;cf=fulltext&amp;site=jbc-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=10.1074%2Fjbc.M111854200&amp;cf=pdf&amp;site=jbc-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-0037066720" target="_blank" aria-label="Open Scopus in new tab">Scopus (204)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11809771/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1074%2Fjbc.M111854200&amp;pmid=11809771" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), polymerase δ resynthesizes the excised DNA strand, and ligase I (LIG1) seals the nascent strand (<span class="dropBlock reference-citations"><a id="body-ref-sref69" href="#" role="doc-biblioref" data-xml-rid="bib69" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref69" aria-controls="sref69">Zhang et&nbsp;al., 2005</a><div class="dropBlock__holder" data-db-target-of="sref69"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">71.</div><div>Zhang, Y. ∙ Yuan, F. ∙ Presnell, S.R. ...</div><div><strong>Reconstitution of 5′-directed human mismatch repair in a purified system</strong></div><div><em>Cell.</em> 2005; <strong>122</strong>:693-705</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=10.1016%2Fj.cell.2005.06.027&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=10.1016%2Fj.cell.2005.06.027&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-24144447320" target="_blank" aria-label="Open Scopus in new tab">Scopus (312)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16143102/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2005.06.027&amp;pmid=16143102" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="p0095" role="paragraph">We hypothesized that MMR engages a specific prime editing intermediate, a DNA heteroduplex formed by hybridization of the reverse-transcribed 3′ DNA flap to adjacent genomic DNA (<a id="crosref0585" href="#fig3">Figure&nbsp;3</a>A). MutSα or MutSβ may recognize the heteroduplex within this structure, and the 3′ nick present after flap equilibration, but before ligation, could stimulate selective excision of the edited strand and subsequent repair to regenerate the original, unedited sequence. Alternatively, MMR may prevent productive flap interconversion by rejecting annealing of the edited 3′ flap to the genomic target (<span class="dropBlock reference-citations"><a id="body-ref-sref57" href="#" role="doc-biblioref" data-xml-rid="bib57" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref57" aria-controls="sref57">Sugawara et&nbsp;al., 2004</a><div class="dropBlock__holder" data-db-target-of="sref57"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">59.</div><div>Sugawara, N. ∙ Goldfarb, T. ∙ Studamire, B. ...</div><div><strong>Heteroduplex rejection during single-strand annealing requires Sgs1 helicase and mismatch repair proteins Msh2 and Msh6 but not Pms1</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2004; <strong>101</strong>:9315-9320</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.0305749101" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-3042546122" target="_blank" aria-label="Open Scopus in new tab">Scopus (165)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15199178/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.0305749101&amp;pmid=15199178" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). In either case, inhibiting MMR during prime editing could delay heteroduplex repair or increase the likelihood of nick ligation, removing bias in the repair of the edited product. Consistent with this model, knockdown of MutSα–MutLα genes strongly enhanced PE2 editing by up to 5.8-fold (<a id="crosref0595" href="#fig1">Figures 1</a>C and 1D). Interfering with MMR reversion of these intermediates can thus enhance prime editing efficiency.</div><div class="figure-wrap"><figure id="fig3" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.09.018/asset/c844bb6a-1910-40f5-a74f-bbd5029df4c4/main.assets/gr3_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr3" src="/cms/10.1016/j.cell.2021.09.018/asset/81dee803-fbc3-4ff2-8c7d-a304954eca8c/main.assets/gr3.jpg" height="420" width="506" aria-labelledby="fig3-title fig3-content" data-viewer-src="/cms/10.1016/j.cell.2021.09.018/asset/c844bb6a-1910-40f5-a74f-bbd5029df4c4/main.assets/gr3_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig3-title" class="figure__title"><span class="label figure__label">Figure&nbsp;3</span> <span class="figure__title__text">Model for mismatch repair of prime editing intermediates</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig3-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0075" role="paragraph">(A) Model for DNA mismatch repair (MMR) of PE2 intermediates. MMR replaces the nicked strand during repair of the heteroduplex PE intermediate. Ligation of the nick before MMR recognition removes&nbsp;the strand discrimination signal for MMR,&nbsp;resulting in unbiased resolution of the heteroduplex.</div><div id="fspara0080" role="paragraph">(B) Model for MMR of PE3 intermediates. Nicks on both DNA strands can direct MMR to replace either strand. Ligation of the nick on the edited strand would guide MMR to replace the non-edited strand.</div><div id="fspara0085" role="paragraph">(C) Prime editing at endogenous sites in HEK293T cells pretreated with siRNAs (details in <a id="crosref1735" href="#sec-5">STAR Methods</a>). Bars represent the mean of n&nbsp;= 3 independent replicates.</div><div id="fspara0090" role="paragraph">(D) Prime editing in HAP1 <i>ΔMSH2</i> and HAP1 <i>ΔMLH1</i> cells. Δ, gene knockout. Bars represent the mean of n&nbsp;= 3 independent replicates.</div></div></div></div></figcaption></figure></div><div id="p0100" role="paragraph">In the context of PE3, nicking the non-edited strand of the heteroduplex intermediate may direct MMR to more frequently replace that strand, leading to higher prime editing efficiency and dampened benefits of MMR suppression, as we observed (<a id="crosref0600" href="#fig1">Figures 1</a>G and <a id="crosref0605" href="#fig3">3</a>B). Nevertheless, the detrimental overall effect of MMR activity on PE3 suggests that heteroduplex intermediates that favorably bias MMR toward the desired edit—those with a ligated edited strand and a nicked non-edited strand—are uncommon. In addition to increasing the intended prime edit, knockdown of MMR genes also reduced indel byproducts from PE3+50 (<a id="crosref0610" href="#fig2">Figure&nbsp;2</a>F). Given this observation, we hypothesize that during repair of a prime editing heteroduplex, MMR activity may induce DSBs, possibly via nicking or excision of the target locus (<a id="crosref0615" href="#figs3">Figure&nbsp;S3</a>D). In agreement with this hypothesis, knockdown of MutSα–MutLα genes disproportionately reduced PE3+50 deletion outcomes outside of the sequence between pegRNA and sgRNA nicks (<a id="crosref0620" href="#fig2">Figure&nbsp;2</a>I). Altogether, these findings support a model in which MMR activity strongly suppresses intended prime editing outcomes and instead promotes indel byproducts.</div></section><section id="sec-2-4"><h3>MMR inhibition improves prime editing at endogenous loci</h3><div id="p0105" role="paragraph">To validate the above model, we tested the effect of MMR on prime editing with canonical SpCas9-based prime editors at endogenous genomic loci and in additional cell types. We treated HEK293T cells with small interfering RNA (siRNAs) targeting MutSα and MutLα genes, cultured the cells for 3&nbsp;days to allow siRNA-mediated knockdown (<a id="crosref0625" href="#figs3">Figure&nbsp;S3</a>E), and then transfected plasmids encoding PE2 and pegRNAs that program point mutations. Across three sites, we observed that mRNA knockdown strongly increased average PE2 editing from 7.7% to 25% with a decrease in indel frequency from 0.39% to 0.28% (<a id="crosref0630" href="#fig3">Figure&nbsp;3</a>C) but improved average PE3 editing efficiency to a lesser extent (from 25% to 37%). Additionally, knockdown of MMR genes reduced the frequency of PE3 indels from 5.5% down to 3.2% on average, increasing PE3 outcome purity by 2.9-fold (<a id="crosref0635" href="#fig3">Figure&nbsp;3</a>C). Thus, consistent with our model, the impact of MMR on PE3 editing efficiency is tempered by its opposing effects on reverting the 3′ flap intermediate (which impedes prime editing) and mediating replacement of the unedited strand (which promotes prime editing).</div><div id="p0110" role="paragraph">We also measured prime editing in MMR-deficient Δ<i>MSH2</i> or Δ<i>MLH1</i> haploid HAP1 cells. PE2 editing was much more efficient in MMR-deficient cells (17% at <i>HEK3</i> and 5.0% at <i>EMX1</i>) than in wild-type control cells (0.44% at <i>HEK3,</i> 0.07% at <i>EMX1</i>; <a id="crosref0640" href="#fig3">Figure&nbsp;3</a>D). However, complementary-strand nicking did not affect prime editing efficiency in MMR-deficient cells (<a id="crosref0645" href="#fig3">Figure&nbsp;3</a>D), consistent with our model that complementary-strand nicking improves prime editing by influencing MMR strand selectivity. Taken together, these results further support a model in which MMR impedes prime editing by promoting excision of the edited DNA strand, even though this effect is partially counterbalanced in the PE3 system by the role of MMR in replacing the non-edited strand.</div></section><section id="sec-2-5"><h3>Engineered dominant negative MLH1 enhances prime editing efficiency and precision</h3><div id="p0115" role="paragraph">Encouraged that cellular pretreatment with MMR-targeting siRNAs could enhance prime editing efficiency, we next explored strategies for simultaneous co-delivery of prime editors and MMR-inhibiting agents. Co-transfection of PE2 and <i>MLH1</i> siRNAs without pretreatment did not substantially increase editing efficiency after 3&nbsp;days (<a id="crosref0650" href="#figs3">Figure&nbsp;S3</a>F), as expected given the kinetics of RNA silencing (<span class="dropBlock reference-citations"><a id="body-ref-sref7" href="#" role="doc-biblioref" data-xml-rid="bib7" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref7" aria-controls="sref7">Bartlett and Davis, 2006</a><div class="dropBlock__holder" data-db-target-of="sref7"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">7.</div><div>Bartlett, D.W. ∙ Davis, M.E.</div><div><strong>Insights into the kinetics of siRNA-mediated gene silencing from live-cell and live-animal bioluminescent imaging</strong></div><div><em>Nucleic Acids Res.</em> 2006; <strong>34</strong>:322-333</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gkj439" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-31544456413" target="_blank" aria-label="Open Scopus in new tab">Scopus (406)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16410612/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgkj439&amp;pmid=16410612" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). We hypothesized that dominant negative MMR protein variants could instead be transiently co-expressed with PE2 or as fusions with PE2 to enhance prime editing. We co-transfected HEK293T cells with plasmids encoding PE2, pegRNAs, and catalytically impaired mutants of human MSH2, MSH6, PMS2, and MLH1 (<span class="dropBlock reference-citations"><a id="body-ref-sref20-1" href="#" role="doc-biblioref" data-xml-rid="bib20" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref20-1" aria-controls="sref20-1">Gueneau et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref20-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">20.</div><div>Gueneau, E. ∙ Dherin, C. ∙ Legrand, P. ...</div><div><strong>Structure of the MutLα C-terminal domain reveals how Mlh1 contributes to Pms1 endonuclease site</strong></div><div><em>Nat. Struct. Mol. Biol.</em> 2013; <strong>20</strong>:461-468</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nsmb.2511" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-84876193944" target="_blank" aria-label="Open Scopus in new tab">Scopus (91)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23435383/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnsmb.2511&amp;pmid=23435383" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref26" href="#" role="doc-biblioref" data-xml-rid="bib26" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref26" aria-controls="sref26">Iaccarino et&nbsp;al., 1998</a><div class="dropBlock__holder" data-db-target-of="sref26"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">26.</div><div>Iaccarino, I. ∙ Marra, G. ∙ Palombo, F. ...</div><div><strong>hMSH2 and hMSH6 play distinct roles in mismatch binding and contribute differently to the ATPase activity of hMutSalpha</strong></div><div><em>EMBO J.</em> 1998; <strong>17</strong>:2677-2686</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/emboj/17.9.2677" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_26_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-0032079736" target="_blank" aria-label="Open Scopus in new tab">Scopus (145)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/9564049/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Femboj%2F17.9.2677&amp;pmid=9564049" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref30-2" href="#" role="doc-biblioref" data-xml-rid="bib30" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref30-2" aria-controls="sref30-2">Kadyrov et&nbsp;al., 2006</a><div class="dropBlock__holder" data-db-target-of="sref30-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">30.</div><div>Kadyrov, F.A. ∙ Dzantiev, L. ∙ Constantin, N. ...</div><div><strong>Endonucleolytic function of MutLalpha in human mismatch repair</strong></div><div><em>Cell.</em> 2006; <strong>126</strong>:297-308</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=10.1016%2Fj.cell.2006.05.039&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=10.1016%2Fj.cell.2006.05.039&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-33746189409" target="_blank" aria-label="Open Scopus in new tab">Scopus (499)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16873062/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2006.05.039&amp;pmid=16873062" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref51" href="#" role="doc-biblioref" data-xml-rid="bib51" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref51" aria-controls="sref51">Räschle et&nbsp;al., 2002</a><div class="dropBlock__holder" data-db-target-of="sref51"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">53.</div><div>Räschle, M. ∙ Dufner, P. ∙ Marra, G. ...</div><div><strong>Mutations within the hMLH1 and hPMS2 subunits of the human MutLalpha mismatch repair factor affect its ATPase activity, but not its ability to interact with hMutSalpha</strong></div><div><em>J.&nbsp;Biol. Chem.</em> 2002; <strong>277</strong>:21810-21820</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=10.1074%2Fjbc.M108787200&amp;cf=fulltext&amp;site=jbc-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=10.1074%2Fjbc.M108787200&amp;cf=pdf&amp;site=jbc-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-0037077309" target="_blank" aria-label="Open Scopus in new tab">Scopus (103)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11948175/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1074%2Fjbc.M108787200&amp;pmid=11948175" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref61" href="#" role="doc-biblioref" data-xml-rid="bib61" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref61" aria-controls="sref61">Tomer et&nbsp;al., 2002</a><div class="dropBlock__holder" data-db-target-of="sref61"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">63.</div><div>Tomer, G. ∙ Buermeyer, A.B. ∙ Nguyen, M.M. ...</div><div><strong>Contribution of human mlh1 and pms2 ATPase activities to DNA mismatch repair</strong></div><div><em>J.&nbsp;Biol. Chem.</em> 2002; <strong>277</strong>:21801-21809</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=10.1074%2Fjbc.M111342200&amp;cf=fulltext&amp;site=jbc-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=10.1074%2Fjbc.M111342200&amp;cf=pdf&amp;site=jbc-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-0037077289" target="_blank" aria-label="Open Scopus in new tab">Scopus (51)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11897781/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1074%2Fjbc.M111342200&amp;pmid=11897781" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) (<a id="crosref0685" href="#fig4">Figures 4</a>A). Of these mutants, ATPase-impaired MLH1 E34A and endonuclease-impaired MLH1 Δ756 increased PE2 editing efficiency by 1.6- to 3.1-fold for three substitution prime edits. Testing additional MLH1 variants, we observed that a larger endonuclease-impaired MLH1 deletion (MLH1 Δ754–756) enhanced average PE2 editing efficiency to the greatest extent across 10 edits (3.2-fold), but combining ATPase and endonuclease mutations (MLH1 E34A Δ754–756) did not yield additional improvement (<a id="crosref0690" href="#fig4">Figures 4</a>B–4D and <a id="crosref0695" href="#figs4">S4</a>A). We therefore designated MLH1 Δ754–756 as MLH1dn.</div><div class="figure-wrap"><figure id="fig4" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.09.018/asset/903a1a22-9ae6-4868-934d-609315242d1f/main.assets/gr4_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr4" src="/cms/10.1016/j.cell.2021.09.018/asset/41259a2d-72fb-4a2c-b56f-9873d66d71a0/main.assets/gr4.jpg" height="575" width="657" aria-labelledby="fig4-title fig4-content" data-viewer-src="/cms/10.1016/j.cell.2021.09.018/asset/903a1a22-9ae6-4868-934d-609315242d1f/main.assets/gr4_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig4-title" class="figure__title"><span class="label figure__label">Figure&nbsp;4</span> <span class="figure__title__text">Engineered dominant negative MLH1 enhances prime editing outcomes</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig4-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0100" role="paragraph">(A) Co-expression of PE2 with dominant negative variants of human MMR proteins improves prime editing efficiency. All values from n&nbsp;= 3 independent replicates are shown.</div><div id="fspara0105" role="paragraph">(B) Functional annotation of the 756-aa human MLH1 protein.</div><div id="fspara0110" role="paragraph">(C) Editing enhancement from MLH1 variants co-expressed with PE2. Red boxes indicate mutations that inactivate MLH1 ATPase or endonuclease function. All values from n&nbsp;= 3 independent replicates are shown.</div><div id="fspara0115" role="paragraph">(D) Comparison of the top three MLH1 variants across ten prime edits. All values from n&nbsp;= 3 independent replicates are shown.</div><div id="fspara0120" role="paragraph">(E) Prime editing with PE2 and MLH1dn in <i>trans</i>, PE2 and MLH1<sup>NTD</sup>–NLS in <i>trans</i>, and PE2–P2A–MLH1dn (human codon-optimized). Bars represent the mean of n&nbsp;= 3 independent replicates.</div><div id="fspara0125" role="paragraph">(F) The PE4 editing system consists of a prime editor enzyme (nickase Cas9–RT fusion), MLH1dn, and pegRNA. The PE5 editing system consists of a prime editor enzyme, MLH1dn, pegRNA, and nicking sgRNA.</div><div id="fspara0130" role="paragraph">(G) PE2, PE3, PE4, and PE5 editing in HEK293T cells. Bars represent the mean of n&nbsp;= 3 independent replicates.</div></div></div></div></figcaption></figure></div><div class="figure-wrap"><figure id="figs4" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.09.018/asset/f4704c9a-43fc-4ba0-8dab-aacf0bc36a8e/main.assets/figs4a_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs4-part-1" src="/cms/10.1016/j.cell.2021.09.018/asset/1105041c-7f09-4664-b000-13d8d69ef24d/main.assets/figs4a.jpg" height="941" width="727" aria-labelledby="figs4-title figs4-content" data-viewer-src="/cms/10.1016/j.cell.2021.09.018/asset/f4704c9a-43fc-4ba0-8dab-aacf0bc36a8e/main.assets/figs4a_lrg.jpg" loading="lazy"></a><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.09.018/asset/caf4fb72-6947-4ab1-9938-5a3d3671900a/main.assets/figs4b_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs4-part-2" src="/cms/10.1016/j.cell.2021.09.018/asset/def4be6f-52ae-455d-b3ca-873044f82453/main.assets/figs4b.jpg" height="409" width="727" aria-labelledby="figs4-title figs4-content" data-viewer-src="/cms/10.1016/j.cell.2021.09.018/asset/caf4fb72-6947-4ab1-9938-5a3d3671900a/main.assets/figs4b_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs4-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S4</span> <span class="figure__title__text">Development and characterization of dominant negative MMR proteins that enhance prime editing outcomes, related to <a id="crosref1785" href="#fig4">Figure&nbsp;4</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs4-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0355" role="paragraph">(A) Prime editing efficiencies from MMR proteins or dominant negative variants expressed <i>in trans</i> with or fused directly to PE2 in HEK293T cells. 32aa, (SGGS)×2–XTEN16–(SGGS)×2 linker. codon opt., human codon-optimized. Data within the same graph originate from experiments performed at the same time. Data represent the mean ± SD of n&nbsp;= 3 independent replicates.</div><div id="fspara0360" role="paragraph">(B) Titration of MLH1dn plasmid and PE2 plasmid transfection doses in HEK293T cells. Maximum plasmid amounts tested were 200&nbsp;ng PE2 and 100&nbsp;ng MLH1dn. Data represent the mean ± SD of n&nbsp;= 3 independent replicates.</div><div id="fspara0365" role="paragraph">(C) Prime editing with MLH1dn co-expression in MMR-deficient HCT116 cells that contain a biallelic deletion in <i>MLH1</i>. Bars represent the mean of n&nbsp;= 3 independent replicates.</div><div id="fspara0370" role="paragraph">(D) <i>MLH1</i> knockout in clonal HeLa cell lines enhances prime editing efficiency to a greater extent than MLH1dn co-expression in clonal wild-type HeLa cells. Δ, knockout. Bars represent the mean of n&nbsp;= 3 or 4 independent replicates.</div><div id="fspara0375" role="paragraph">(E) Editing at the <i>HEK4</i> locus with complementary-strand nicks in HEK293T cells. “None” indicates the lack of a nick, which denotes a PE2 or PE4 editing strategy. Bars represent the mean of n&nbsp;= 3 independent replicates.</div><div id="fspara0380" role="paragraph">(F) Editing at the <i>FANCF</i> locus with PE3b and PE5b (complementary-strand nick that is specific for the edited sequence) in HEK293T cells. PE5b, PE3b editing system with MLH1dn co-expression. Bars represent the mean of n&nbsp;= 3 independent replicates.</div><div id="fspara0385" role="paragraph">(G) Comparison of prime editing with human MLH1dn (human codon-optimized) or mouse MLH1dn (mouse codon-optimized) in human HEK293T cells. Bars represent the mean of n&nbsp;= 3 independent replicates.</div><div id="fspara0390" role="paragraph">(H) Comparison of prime editing with human MLH1dn (human codon-optimized) or mouse MLH1dn (mouse codon-optimized) in mouse N2A cells. Bars represent the mean of n&nbsp;= 3 independent replicates.</div></div></div></div></figcaption></figure></div><div id="p0120" role="paragraph">We also identified shorter MLH1 truncations that can inhibit MMR during prime editing. The MLH1 N-terminal domain (NTD; residues 1–335) mediates MutLα recruitment to MSH2 during MMR (<span class="dropBlock reference-citations"><a id="body-ref-sref48" href="#" role="doc-biblioref" data-xml-rid="bib48" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref48" aria-controls="sref48">Plotz et&nbsp;al., 2003</a><div class="dropBlock__holder" data-db-target-of="sref48"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">50.</div><div>Plotz, G. ∙ Raedle, J. ∙ Brieger, A. ...</div><div><strong>N-terminus of hMLH1 confers interaction of hMutLalpha and hMutLbeta with hMutSalpha</strong></div><div><em>Nucleic Acids Res.</em> 2003; <strong>31</strong>:3217-3226</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gkg420" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-0037563786" target="_blank" aria-label="Open Scopus in new tab">Scopus (52)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/12799449/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgkg420&amp;pmid=12799449" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), while the MLH1 C-terminal domain (CTD; residues 501–756) dimerizes with PMS2 and contributes to MutLα endonuclease activity critical for MMR (<span class="dropBlock reference-citations"><a id="body-ref-sref20-2" href="#" role="doc-biblioref" data-xml-rid="bib20" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref20-2" aria-controls="sref20-2">Gueneau et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref20-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">20.</div><div>Gueneau, E. ∙ Dherin, C. ∙ Legrand, P. ...</div><div><strong>Structure of the MutLα C-terminal domain reveals how Mlh1 contributes to Pms1 endonuclease site</strong></div><div><em>Nat. Struct. Mol. Biol.</em> 2013; <strong>20</strong>:461-468</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nsmb.2511" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-84876193944" target="_blank" aria-label="Open Scopus in new tab">Scopus (91)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23435383/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnsmb.2511&amp;pmid=23435383" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref21" href="#" role="doc-biblioref" data-xml-rid="bib21" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref21" aria-controls="sref21">Guerrette et&nbsp;al., 1999</a><div class="dropBlock__holder" data-db-target-of="sref21"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">21.</div><div>Guerrette, S. ∙ Acharya, S. ∙ Fishel, R.</div><div><strong>The interaction of the human MutL homologues in hereditary nonpolyposis colon cancer</strong></div><div><em>J.&nbsp;Biol. Chem.</em> 1999; <strong>274</strong>:6336-6341</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=10.1074%2Fjbc.274.10.6336&amp;cf=fulltext&amp;site=jbc-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=10.1074%2Fjbc.274.10.6336&amp;cf=pdf&amp;site=jbc-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-0033525518" target="_blank" aria-label="Open Scopus in new tab">Scopus (157)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/10037723/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1074%2Fjbc.274.10.6336&amp;pmid=10037723" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) (<a id="crosref0715" href="#fig4">Figure&nbsp;4</a>B). The MLH1 NTD with a nuclear localization signal (NLS) tag (hereafter referred to as MLH1<sup>NTD</sup>–NLS) improved PE2 editing by 1.9- to 2.5-fold (<a id="crosref0720" href="#fig4">Figures 4</a>C and <a id="crosref0725" href="#figs4">S4</a>A), to a similar degree as full-length MLH1dn. In contrast, the MLH1 CTD did not substantially enhance PE2 editing, suggesting that MLH1 variants can inhibit MMR and improve prime editing by forming catalytically impaired MutLα complexes with PMS2 or by saturating the binding of MSH2. Given their domain architecture, MLH1dn and MLH1<sup>NTD</sup>–NLS both inhibit MMR through MSH2 binding. As expected, MLH1dn improved prime editing in a dose-dependent manner (<a id="crosref0730" href="#figs4">Figure&nbsp;S4</a>B) and did not increase editing in MMR-deficient HCT116 cells (<span class="dropBlock reference-citations"><a id="body-ref-sref45" href="#" role="doc-biblioref" data-xml-rid="bib45" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref45" aria-controls="sref45">Parsons et&nbsp;al., 1993</a><div class="dropBlock__holder" data-db-target-of="sref45"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">46.</div><div>Parsons, R. ∙ Li, G.-M. ∙ Longley, M.J. ...</div><div><strong>Hypermutability and mismatch repair deficiency in RER+ tumor cells</strong></div><div><em>Cell.</em> 1993; <strong>75</strong>:1227-1236</div></div><div class="external-links"><div class="core-xlink-abstract"><a href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=10.1016%2F0092-8674%2893%2990331-J&amp;cf=abstract&amp;site=cell-site">Abstract</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=10.1016%2F0092-8674%2893%2990331-J&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-0027137935" target="_blank" aria-label="Open Scopus in new tab">Scopus (992)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/8261516/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2F0092-8674%2893%2990331-J&amp;pmid=8261516" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) (<a id="crosref0740" href="#figs4">Figure&nbsp;S4</a>C).</div><div id="p0125" role="paragraph">Among 55 total dominant negative MMR protein candidates, including additional MLH1 variants and truncations, MLH1dn expressed in <i>trans</i> with PE2 provided the greatest average enhancement in PE2 editing efficiency in HEK293T cells (3.2-fold; <a id="crosref0745" href="#fig4">Figure&nbsp;4</a>E). We also observed strong improvement of PE2 editing from MLH1<sup>NTD</sup>–NLS expressed in <i>trans</i> (2.7-fold on average) and a PE2–P2A–MLH1dn construct (2.4-fold on average). These three constructs also increased PE3 editing efficiency by 1.2-fold on average and reduced indel byproducts by 1.4- to 4.0-fold (<a id="crosref0750" href="#fig4">Figure&nbsp;4</a>E). We thus designated PE2 editing with MLH1dn co-expression as the PE4 system, and PE3 editing with MLH1dn co-expression as the PE5 system (<a id="crosref0755" href="#fig4">Figure&nbsp;4</a>F). We note that one advantage of MLH1<sup>NTD</sup>–NLS is that it can also enhance prime editing efficiencies with a smaller protein (355 aa) compared to MLH1dn (753 aa). Intriguingly, <i>MLH1</i> knockout enhanced PE2 and PE3 editing to a larger degree than MLH1dn co-expression in clonal HeLa cells (<a id="crosref0760" href="#figs4">Figure&nbsp;S4</a>D), suggesting opportunities for additional prime editing enhancement through further modulation of this pathway.</div><div id="p0130" role="paragraph">We further assessed the generality of PE4 and PE5 systems across eight additional single-base substitution edits at different genomic loci in HEK293T cells. On average, PE4 improved editing efficiency over PE2 by 2.0-fold with minimal indels (&lt;0.4% on average; <a id="crosref0765" href="#fig4">Figure&nbsp;4</a>G) and was particularly effective at a locus in which complementary-strand nicks yield unproductive editing outcomes (<a id="crosref0770" href="#figs4">Figure&nbsp;S4</a>E). PE5 improved editing over PE3 by 1.2-fold and enhanced edit/indel purity by 3.0-fold (<a id="crosref0775" href="#fig4">Figure&nbsp;4</a>G). MLH1dn also increased efficiency for PE3b (PE5b&nbsp;= PE3b&nbsp;+ MLH1dn; <a id="crosref0780" href="#figs4">Figure&nbsp;S4</a>F), a prime editing strategy that uses a complementary-strand nick specific for the edited sequence to minimize coincident nicks on both strands that promote indel formation (<span class="dropBlock reference-citations"><a id="body-ref-sref3-7" href="#" role="doc-biblioref" data-xml-rid="bib3" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref3-7" aria-controls="sref3-7">Anzalone et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref3-7"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">3.</div><div>Anzalone, A.V. ∙ Randolph, P.B. ∙ Davis, J.R. ...</div><div><strong>Search-and-replace genome editing without double-strand breaks or donor DNA</strong></div><div><em>Nature.</em> 2019; <strong>576</strong>:149-157</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-1711-4" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-85074571375" target="_blank" aria-label="Open Scopus in new tab">Scopus (2574)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31634902/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-1711-4&amp;pmid=31634902" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Finally, both human- and mouse-derived MLH1dn improved prime editing efficiency in human HEK293T cells and mouse N2A cells (<a id="crosref0790" href="#figs4">Figures S4</a>G and S4H). Collectively, these data establish PE4 and PE5 systems that substantially enhance prime editing efficiency and outcome purity at a variety of endogenous genomic loci in mammalian cells.</div></section><section id="sec-2-6"><h3>Characterization of the types of prime edits enhanced by PE4 and PE5</h3><div id="p0135" role="paragraph">Next, we studied the extent to which MLH1dn improves prime editing across a wide range of edit types. Since MMR repairs different DNA mismatches with varying efficiencies (<span class="dropBlock reference-citations"><a id="body-ref-sref39-1" href="#" role="doc-biblioref" data-xml-rid="bib39" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref39-1" aria-controls="sref39-1">Lujan et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref39-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">40.</div><div>Lujan, S.A. ∙ Clausen, A.R. ∙ Clark, A.B. ...</div><div><strong>Heterogeneous polymerase fidelity and mismatch repair bias genome variation and composition</strong></div><div><em>Genome Res.</em> 2014; <strong>24</strong>:1751-1764</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/gr.178335.114" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_40_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-84923948943" target="_blank" aria-label="Open Scopus in new tab">Scopus (123)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25217194/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2Fgr.178335.114&amp;pmid=25217194" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), we anticipated that MLH1dn would more strongly enhance prime edits that proceed through mismatches that are more efficient substrates for MMR. Across 84 pegRNAs that together introduce all 12 possible single-base substitutions at seven endogenous loci in HEK293T cells, PE4 improved editing efficiency by 2.0-fold and reduced indel frequencies from 0.40% to 0.31% compared to PE2 (<a id="crosref0800" href="#fig5">Figures 5</a>A, 5B, <a id="crosref0805" href="#figs5">S5</a>A, and S5B). In contrast, PE5 yielded an average 1.2-fold increase in editing and 2.8-fold increase in edit/indel purity relative to PE3 (<a id="crosref0810" href="#figs5">Figure&nbsp;S5</a>C).</div><div class="figure-wrap"><figure id="fig5" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.09.018/asset/c2750039-f993-49c5-8328-6f871ddf0efc/main.assets/gr5_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr5" src="/cms/10.1016/j.cell.2021.09.018/asset/9d19d8b2-950f-4124-9d02-da2a6423f405/main.assets/gr5.jpg" height="942" width="710" aria-labelledby="fig5-title fig5-content" data-viewer-src="/cms/10.1016/j.cell.2021.09.018/asset/c2750039-f993-49c5-8328-6f871ddf0efc/main.assets/gr5_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig5-title" class="figure__title"><span class="label figure__label">Figure&nbsp;5</span> <span class="figure__title__text">Characterization of PE4 and PE5 across diverse prime edit classes and cell types</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig5-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0140" role="paragraph">(A) Summary of prime editing enhancement by PE4 and PE5 compared to PE2 and PE3 for 84 single-base substitution edits (7 for each substitution type) across 7 endogenous sites in HEK293T cells. The grand mean ± SD of all individual values of n&nbsp;= 3 independent replicates are shown.</div><div id="fspara0145" role="paragraph">(B) Substitution edits with PE2, PE3, PE4, and PE5 at the <i>FANCF</i> locus in HEK293T cells. The black triangle marks the location of the pegRNA-programmed nick. Bars represent the mean of n&nbsp;= 3 independent replicates.</div><div id="fspara0150" role="paragraph">(C) PE4 improves 1- and 3-bp indel prime edits compared to PE2 in HEK293T cells (mean of n&nbsp;= 3 independent replicates).</div><div id="fspara0155" role="paragraph">(D) PE4 editing enhancement over PE2 across 33 indel prime edits. Lines represent the mean of all individual values of n&nbsp;= 3 independent replicates.</div><div id="fspara0160" role="paragraph">(E and F) Summary of PE2 and PE4 editing efficiencies for 35 different substitutions of 1–5 contiguous bases at five endogenous sites in HEK293T cells. Seven pegRNAs were tested for each number of contiguous bases altered. The mean ± SD of all individual values of n&nbsp;= 3 replicates are shown.</div><div id="fspara0165" role="paragraph">(G and H) Installation of additional silent or benign mutations near the intended edit can increase editing efficiency by generating a heteroduplex substrate that evades MMR. The PAM sequence (NGG) for each target is underlined. The amino acid sequence of the targeted gene is centered above each triplet DNA codon. Values represent the mean ± SD of n&nbsp;= 3 independent replicates.</div><div id="fspara0170" role="paragraph">(I) Summary of PE4 and PE5 editing enhancement in MMR-deficient (MMR−) and MMR-proficient (MMR+) cells. A common set of 30 pegRNAs encoding point mutations were tested in HEK293T and HeLa cells. K562 and U2OS cells were edited with 10 total pegRNAs that are a subset of these 30 pegRNAs. The mean ± SD of all individual values of sets of n&nbsp;= 3 independent replicates are shown. p values were calculated with the Mann-Whitney <i>U</i> test.</div><div id="fspara0175" role="paragraph">(J) Prime editing with PE2, PE3, PE4, and PE5 in HeLa, K562, and U2OS cells. Bars represent the mean of n&nbsp;= 3 independent replicates.</div></div></div></div></figcaption></figure></div><div class="figure-wrap"><figure id="figs5" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.09.018/asset/eee43b84-1b87-4b1e-99f1-e325892c10a7/main.assets/figs5a_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs5-part-1" src="/cms/10.1016/j.cell.2021.09.018/asset/738faa00-a2e7-48e3-820a-d2ab776f9f23/main.assets/figs5a.jpg" height="899" width="753" aria-labelledby="figs5-title figs5-content" data-viewer-src="/cms/10.1016/j.cell.2021.09.018/asset/eee43b84-1b87-4b1e-99f1-e325892c10a7/main.assets/figs5a_lrg.jpg" loading="lazy"></a><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.09.018/asset/97961f8f-82fd-4c7a-bcce-19b80c22e750/main.assets/figs5b_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs5-part-2" src="/cms/10.1016/j.cell.2021.09.018/asset/69edc1aa-0fe2-4917-aa70-6608b0b47ae2/main.assets/figs5b.jpg" height="688" width="753" aria-labelledby="figs5-title figs5-content" data-viewer-src="/cms/10.1016/j.cell.2021.09.018/asset/97961f8f-82fd-4c7a-bcce-19b80c22e750/main.assets/figs5b_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs5-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S5</span> <span class="figure__title__text">Characterization of PE4 and PE5 systems and improved prime editing efficiency with additional silent mutations, related to <a id="crosref1790" href="#fig5">Figure&nbsp;5</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs5-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0400" role="paragraph">(A) Comparison of PE2, PE3, PE4, and PE5 for 84 single-base substitution prime edits across seven endogenous sites in HEK293T cells, supplementing information in <a id="opt7R1KYw9bgP" href="#fig5">Figures 5</a>A, <a id="optaMXc5BISx3" href="#fig6">6</a>D–F, <a id="optGEt8lhsF0p" href="#figs6">S6</a>B, and <a id="optNnlW8Ocm0H" href="#figs6">S6</a>D. Bars represent the mean of n&nbsp;= 3 independent replicates.</div><div id="fspara0405" role="paragraph">(B) Summary of PE4 enhancement in editing efficiency over PE2 for 84 single-base substitution edits across seven endogenous sites in HEK293T cells. PE4/PE2 fold improvements may be lower for PAM edits due to the high basal editing efficiency for PAM edits or the high representation of G⋅C-to-C⋅G edits (five out of 15 in this category). Data represent the mean ± SD of n&nbsp;= 3 independent replicates.</div><div id="fspara0410" role="paragraph">(C) Summary of PE5 enhancement in editing efficiency over PE3 for 84 single-base substitution edits in HEK293T cells. The grand mean ± SD of all individual values of n&nbsp;= 3 independent replicates are shown.</div><div id="fspara0415" role="paragraph">(D) Effect of siRNA knockdown of MMR genes on G⋅C-to-C⋅G editing at the <i>RNF2</i> locus in HEK293T cells. Bars represent the mean of n&nbsp;= 3 independent replicates.</div><div id="fspara0420" role="paragraph">(E) Effect of MMR gene knockout on G⋅C-to-C⋅G editing at the <i>RNF2</i> locus in HAP1 cells. Δ, gene knockout. Bars represent the mean of n&nbsp;= 3 independent replicates.</div><div id="fspara0425" role="paragraph">(F) Efficiencies of single-base substitution prime edits that alter the PAM (+5&nbsp;G or&nbsp;+6&nbsp;G bases) of prime editing target protospacers in HEK293T cells. Four G⋅C-to-A⋅T, five G⋅C-to-C⋅G, and six G⋅C-to-T⋅A PAM edits across a combined seven endogenous sites are shown. The mean of all individual values of n&nbsp;= 3 independent replicates are shown.</div><div id="fspara0430" role="paragraph">(G) Prime editing at the pre-validated Repair-seq screen edit site with CRISPRi knockdown in HeLa CRISPRi cells. PE2 indicates editing with SaPE2 protein and Sa-pegRNA. PE3+50 indicates editing with SaPE2 protein, Sa-pegRNA, and Sa-sgRNA that programs a&nbsp;+50 complementary-strand nick. Bars represent the mean of n&nbsp;= 5 independent replicates.</div><div id="fspara0435" role="paragraph">(H) PE5 improves editing efficiency and reduces indel byproducts compared to PE3 across small insertion and deletion prime edits in HEK293T cells.</div><div id="fspara0440" role="paragraph">(I) PE2 and PE4 editing efficiencies at 33 different insertion and deletion prime edits across a combined three endogenous loci. Lines represent the mean of all individual values of n&nbsp;= 3 independent replicates. Data supplements information in <a id="optY1l2wQ6iIy" href="#fig5">Figure 5</a>D.</div><div id="fspara0445" role="paragraph">(J) Substitutions of contiguous bases with PE2 and PE4 in HEK293T cells. The top sequence indicates the original, unedited genomic sequence. Numbers denote the position of the edited nucleotide relative to the pegRNA-directed nick site. Nucleotides within the SpCas9 PAM sequence (NGG) are underlined. Sequences of the intended edited product are shown below, with edited nucleotides marked in red. Bars represent the mean of n&nbsp;= 3 independent replicates. Data supplements information in <a id="optuAfyo7IIYZ" href="#fig5">Figures 5</a>E and 5F.</div><div id="fspara0450" role="paragraph">(K) Installation of additional silent mutations can increase prime editing efficiency by evading MMR. PE4/PE2 fold-change in editing frequency reflects the extent to which MMR activity impedes the indicated prime edit. Edited nucleotides that make the indicated coding mutation are marked in red, and edited nucleotides that make silent mutations are marked in green. Data represent the mean ± SD of n&nbsp;= 3 independent replicates.</div><div id="fspara0455" role="paragraph">(L) Installation of 22 single-base substitution prime edits across seven endogenous sites in HeLa cells with PE2, PE3, PE4, and PE5. Bars represent the mean of n&nbsp;= 3 independent replicates. Data supplements information in <a id="optXxKeHQurVM" href="#fig5">Figure 5</a>I.</div></div></div></div></figcaption></figure></div><div id="p0140" role="paragraph">Among the 12 types of base substitutions, G⋅C-to-C⋅G edits, which form C⋅C mismatches after 3′ flap hybridization, were by far the least improved with MLH1dn (1.2-fold comparing PE4 with PE2; <a id="crosref0815" href="#fig5">Figure&nbsp;5</a>A), consistent with previous studies establishing that C⋅C mismatches are not efficiently repaired by MMR (<span class="dropBlock reference-citations"><a id="body-ref-sref33" href="#" role="doc-biblioref" data-xml-rid="bib33" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref33" aria-controls="sref33">Lahue et&nbsp;al., 1989</a><div class="dropBlock__holder" data-db-target-of="sref33"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">33.</div><div>Lahue, R.S. ∙ Au, K.G. ∙ Modrich, P.</div><div><strong>DNA mismatch correction in a defined system</strong></div><div><em>Science.</em> 1989; <strong>245</strong>:160-164</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.2665076" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-0024396565" target="_blank" aria-label="Open Scopus in new tab">Scopus (474)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/2665076/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.2665076&amp;pmid=2665076" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref56" href="#" role="doc-biblioref" data-xml-rid="bib56" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref56" aria-controls="sref56">Su et&nbsp;al., 1988</a><div class="dropBlock__holder" data-db-target-of="sref56"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">58.</div><div>Su, S.S. ∙ Lahue, R.S. ∙ Au, K.G. ...</div><div><strong>Mispair specificity of methyl-directed DNA mismatch correction in&nbsp;vitro</strong></div><div><em>J.&nbsp;Biol. Chem.</em> 1988; <strong>263</strong>:6829-6835</div></div><div class="external-links"><div class="core-xlink-abstract"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=10.1016%2FS0021-9258%2818%2968718-6&amp;cf=abstract&amp;site=jbc-site" target="_blank" aria-label="Open Abstract in new tab">Abstract</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=10.1016%2FS0021-9258%2818%2968718-6&amp;cf=pdf&amp;site=jbc-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/2834393/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0021-9258%2818%2968718-6&amp;pmid=2834393" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref60-2" href="#" role="doc-biblioref" data-xml-rid="bib60" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref60-2" aria-controls="sref60-2">Thomas et&nbsp;al., 1991</a><div class="dropBlock__holder" data-db-target-of="sref60-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">62.</div><div>Thomas, D.C. ∙ Roberts, J.D. ∙ Kunkel, T.A.</div><div><strong>Heteroduplex repair in extracts of human HeLa cells</strong></div><div><em>J.&nbsp;Biol. Chem.</em> 1991; <strong>266</strong>:3744-3751</div></div><div class="external-links"><div class="core-xlink-abstract"><a href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=10.1016%2FS0021-9258%2819%2967858-0&amp;cf=abstract&amp;site=jbc-site" target="_blank" aria-label="Open Abstract in new tab">Abstract</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=10.1016%2FS0021-9258%2819%2967858-0&amp;cf=pdf&amp;site=jbc-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/1995629/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0021-9258%2819%2967858-0&amp;pmid=1995629" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). In support of this observation, MMR knockdown in HEK293T cells (<a id="crosref0835" href="#figs5">Figure&nbsp;S5</a>D) and MMR knockout in HAP1 cells (<a id="crosref0840" href="#figs5">Figure&nbsp;S5</a>E) did not change the efficiency of a G⋅C-to-C⋅G prime edit at the <i>RNF2</i> locus. These findings suggest that G⋅C-to-C⋅G edits more effectively evade MMR and may therefore yield higher basal editing efficiency. Consistent with this possibility, across seven loci, G⋅C-to-C⋅G edits with PE2 were substantially more efficient (27%) than G⋅C-to-A⋅T (18%) or G⋅C-to-T⋅A (20%) edits among prime edits that alter the protospacer adjacent motif (PAM) (<a id="crosref0845" href="#figs5">Figure&nbsp;S5</a>F). We also compared G⋅C-to-A⋅T, G⋅C-to-C⋅G, and G⋅C-to-T⋅A edits with SaPE2 at the pre-validated screening site in HeLa CRISPRi cells (<a id="crosref0850" href="#figs1">Figure&nbsp;S1</a>C). PE2 and PE3+50 more efficiently installed the G⋅C-to-C⋅G edit than the G⋅C-to-A⋅T or G⋅C-to-T⋅A edits, consistent with weaker MMR activity at C⋅C mismatches (<a id="crosref0855" href="#figs5">Figure&nbsp;S5</a>G). Furthermore, CRISPRi knockdown of <i>MSH2</i> improved G⋅C-to-A⋅T and G⋅C-to-T⋅A editing efficiencies (16-fold for PE2 and 4.3-fold for PE3+50) to a greater extent than for G⋅C-to-C⋅G (4.0-fold for PE2 and 1.9-fold for PE3+50). Collectively, these data suggest that G⋅C-to-C⋅G prime edits are less susceptible to repair by MMR and are thus installed with higher efficiency.</div><div id="p0145" role="paragraph">To determine if MLH1dn could also improve indel prime edits, we installed 1- and 3-bp indels with PE4 and PE5 in HEK293T cells. Across 12 pegRNAs at three loci, PE4 increased average editing efficiency by 2.2-fold over PE2, with no increase in unintended indel frequency, while PE5 increased editing efficiency by 1.2-fold and edit/indel purity by 2.9-fold over PE3 (<a id="crosref0860" href="#fig5">Figures 5</a>C and <a id="crosref0865" href="#figs5">S5</a>H). We also tested PE2 and PE4 with 33 pegRNAs that together program 1-, 3-, 6-, 10-, 15-, and 20-bp indels at the <i>HEK3</i> and <i>FANCF</i> loci. MLH1dn enhancement of prime editing efficiency declined as the length of the indels increased (<a id="crosref0870" href="#fig5">Figures 5</a>D and <a id="crosref0875" href="#figs5">S5</a>I), consistent with previous reports that MMR repairs IDLs up to 13 nt in length (<span class="dropBlock reference-citations"><a id="body-ref-sref1" href="#" role="doc-biblioref" data-xml-rid="bib1" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref1" aria-controls="sref1">Acharya et&nbsp;al., 1996</a><div class="dropBlock__holder" data-db-target-of="sref1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">1.</div><div>Acharya, S. ∙ Wilson, T. ∙ Gradia, S. ...</div><div><strong>hMSH2 forms specific mispair-binding complexes with hMSH3 and hMSH6</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 1996; <strong>93</strong>:13629-13634</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.93.24.13629" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_1_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-0030465237" target="_blank" aria-label="Open Scopus in new tab">Scopus (481)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/8942985/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.93.24.13629&amp;pmid=8942985" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref16" href="#" role="doc-biblioref" data-xml-rid="bib16" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref16" aria-controls="sref16">Genschel et&nbsp;al., 1998</a><div class="dropBlock__holder" data-db-target-of="sref16"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">16.</div><div>Genschel, J. ∙ Littman, S.J. ∙ Drummond, J.T. ...</div><div><strong>Isolation of MutSbeta from human cells and comparison of the mismatch repair specificities of MutSbeta and MutSalpha</strong></div><div><em>J.&nbsp;Biol. Chem.</em> 1998; <strong>273</strong>:19895-19901</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=10.1074%2Fjbc.273.31.19895&amp;cf=fulltext&amp;site=jbc-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=10.1074%2Fjbc.273.31.19895&amp;cf=pdf&amp;site=jbc-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-0032584384" target="_blank" aria-label="Open Scopus in new tab">Scopus (348)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/9677427/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1074%2Fjbc.273.31.19895&amp;pmid=9677427" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref65" href="#" role="doc-biblioref" data-xml-rid="bib65" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref65" aria-controls="sref65">Umar et&nbsp;al., 1994</a><div class="dropBlock__holder" data-db-target-of="sref65"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">67.</div><div>Umar, A. ∙ Boyer, J.C. ∙ Kunkel, T.A.</div><div><strong>DNA loop repair by human cell extracts</strong></div><div><em>Science.</em> 1994; <strong>266</strong>:814-816</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.7973637" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_67_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-0027988576" target="_blank" aria-label="Open Scopus in new tab">Scopus (125)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/7973637/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.7973637&amp;pmid=7973637" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). These results together demonstrate that PE4 and PE5 strategies can enhance small (&lt;15&nbsp;bp) targeted indels and suggest that longer indel edits benefit less from MLH1dn because their intermediates natively evade MMR.</div></section><section id="sec-2-7"><h3>Installing additional silent mutations can increase prime editing efficiency by evading MMR</h3><div id="p0150" role="paragraph">Next, we explored whether other classes of prime edits could bypass MMR. MutSα and MutSβ each recognize specific DNA heteroduplex structures (<span class="dropBlock reference-citations"><a id="body-ref-sref22-2" href="#" role="doc-biblioref" data-xml-rid="bib22" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref22-2" aria-controls="sref22-2">Gupta et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref22-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">22.</div><div>Gupta, S. ∙ Gellert, M. ∙ Yang, W.</div><div><strong>Mechanism of mismatch recognition revealed by human MutSβ bound to unpaired DNA loops</strong></div><div><em>Nat. Struct. Mol. Biol.</em> 2011; <strong>19</strong>:72-78</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nsmb.2175" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-84855463183" target="_blank" aria-label="Open Scopus in new tab">Scopus (125)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22179786/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnsmb.2175&amp;pmid=22179786" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref67-2" href="#" role="doc-biblioref" data-xml-rid="bib67" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref67-2" aria-controls="sref67-2">Warren et&nbsp;al., 2007</a><div class="dropBlock__holder" data-db-target-of="sref67-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">69.</div><div>Warren, J.J. ∙ Pohlhaus, T.J. ∙ Changela, A. ...</div><div><strong>Structure of the human MutSalpha DNA lesion recognition complex</strong></div><div><em>Mol. Cell.</em> 2007; <strong>26</strong>:579-592</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=10.1016%2Fj.molcel.2007.04.018&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=10.1016%2Fj.molcel.2007.04.018&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-34248572591" target="_blank" aria-label="Open Scopus in new tab">Scopus (286)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17531815/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.molcel.2007.04.018&amp;pmid=17531815" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), suggesting that a DNA bubble of contiguous mismatches could weaken recognition by these MMR components. To assess this possibility, we tested PE2 and PE4 with 35 different edits that generate 1- to 5-base contiguous substitutions at five genomic loci in HEK293T cells. Across 2-base substitutions, PE4 yielded 2.3-fold higher editing efficiency than PE2, similar to the 2.4-fold enhancement for single-base substitutions at the same target nucleotides (<a id="crosref0905" href="#fig5">Figures 5</a>E and <a id="crosref0910" href="#figs5">S5</a>J). In contrast, PE4 improved the editing of longer 3- to 5-base contiguous substitutions by 1.2- to 1.5-fold relative to PE2. The reduced impact of MMR on these larger edits was also reflected in higher average PE2 editing efficiency for 3- to 5-base contiguous substitutions (13% across 21 edits) compared to 1- or 2-base contiguous substitutions (4.8% across 14 edits) (<a id="crosref0915" href="#fig5">Figures 5</a>F and <a id="crosref0920" href="#figs5">S5</a>J).</div><div id="p0155" role="paragraph">Next, we asked whether installing additional silent mutations nearby the intended edits could similarly increase prime editing efficiency by weakening repair of the resulting heteroduplex (<a id="crosref0925" href="#fig5">Figure&nbsp;5</a>G), even in the absence of MMR inhibition. To test this idea, we designed pegRNAs that program a coding mutation and, optionally, additional silent mutations close to the coding edit (most fewer than 5&nbsp;bp away). At four of six gene targets, adding these silent mutations increased PE2 efficiency of the desired coding change by an average 1.8-fold for the best pegRNAs at each site (<a id="crosref0930" href="#fig5">Figures 5</a>H and <a id="crosref0935" href="#figs5">S5</a>K). Inhibiting MMR with MLH1dn (PE4) improved editing efficiency with the best silent mutations to a lesser extent (1.2-fold on average) compared to only the coding edits (1.7-fold), suggesting that these additional silent mutations enhance editing by evading MMR. Consistent with this mechanism, at the two sites in which silent mutations do not improve editing, the tested silent mutations do not affect MLH1dn enhancement of editing (<a id="crosref0940" href="#figs5">Figure&nbsp;S5</a>K). Collectively, these findings support that MMR less efficiently repairs heteroduplexes containing three or more contiguous mismatched bases and reveal that the strategic installation of additional benign mutations nearby the desired edit can increase prime editing efficiency by evading MMR, even without manipulating MMR activity.</div></section><section id="sec-2-8"><h3>PE4 and PE5 strongly improve prime editing in MMR-proficient cell types</h3><div id="p0160" role="paragraph">HEK293T cells are partially MMR deficient due to hypermethylation of the <i>MLH1</i> promoter (<span class="dropBlock reference-citations"><a id="body-ref-sref63" href="#" role="doc-biblioref" data-xml-rid="bib63" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref63" aria-controls="sref63">Trojan et&nbsp;al., 2002</a><div class="dropBlock__holder" data-db-target-of="sref63"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">65.</div><div>Trojan, J. ∙ Zeuzem, S. ∙ Randolph, A. ...</div><div><strong>Functional analysis of hMLH1 variants and HNPCC-related mutations using a human expression system</strong></div><div><em>Gastroenterology.</em> 2002; <strong>122</strong>:211-219</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=10.1053%2Fgast.2002.30296&amp;cf=fulltext&amp;site=ygast-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=10.1053%2Fgast.2002.30296&amp;cf=pdf&amp;site=ygast-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-0036143918" target="_blank" aria-label="Open Scopus in new tab">Scopus (178)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11781295/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1053%2Fgast.2002.30296&amp;pmid=11781295" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), which may explain higher prime editing efficiency observed in HEK293T cells compared to other mammalian cell types (<span class="dropBlock reference-citations"><a id="body-ref-sref3-8" href="#" role="doc-biblioref" data-xml-rid="bib3" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref3-8" aria-controls="sref3-8">Anzalone et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref3-8"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">3.</div><div>Anzalone, A.V. ∙ Randolph, P.B. ∙ Davis, J.R. ...</div><div><strong>Search-and-replace genome editing without double-strand breaks or donor DNA</strong></div><div><em>Nature.</em> 2019; <strong>576</strong>:149-157</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-1711-4" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-85074571375" target="_blank" aria-label="Open Scopus in new tab">Scopus (2574)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31634902/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-1711-4&amp;pmid=31634902" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). To evaluate whether MLH1dn improves prime editing to a greater degree in cells without MMR deficiency, we compared prime editing in partially MMR-deficient (MMR−) HEK293T cells and in three MMR-proficient (MMR+) cell types: HeLa (<span class="dropBlock reference-citations"><a id="body-ref-sref24" href="#" role="doc-biblioref" data-xml-rid="bib24" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref24" aria-controls="sref24">Holmes et&nbsp;al., 1990</a><div class="dropBlock__holder" data-db-target-of="sref24"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">24.</div><div>Holmes, Jr., J. ∙ Clark, S. ∙ Modrich, P.</div><div><strong>Strand-specific mismatch correction in nuclear extracts of human and Drosophila melanogaster cell lines</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 1990; <strong>87</strong>:5837-5841</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.87.15.5837" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-0025340001" target="_blank" aria-label="Open Scopus in new tab">Scopus (362)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/2116007/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.87.15.5837&amp;pmid=2116007" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref60-3" href="#" role="doc-biblioref" data-xml-rid="bib60" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref60-3" aria-controls="sref60-3">Thomas et&nbsp;al., 1991</a><div class="dropBlock__holder" data-db-target-of="sref60-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">62.</div><div>Thomas, D.C. ∙ Roberts, J.D. ∙ Kunkel, T.A.</div><div><strong>Heteroduplex repair in extracts of human HeLa cells</strong></div><div><em>J.&nbsp;Biol. Chem.</em> 1991; <strong>266</strong>:3744-3751</div></div><div class="external-links"><div class="core-xlink-abstract"><a href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=10.1016%2FS0021-9258%2819%2967858-0&amp;cf=abstract&amp;site=jbc-site" target="_blank" aria-label="Open Abstract in new tab">Abstract</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=10.1016%2FS0021-9258%2819%2967858-0&amp;cf=pdf&amp;site=jbc-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/1995629/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0021-9258%2819%2967858-0&amp;pmid=1995629" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), K562 (<span class="dropBlock reference-citations"><a id="body-ref-sref40" href="#" role="doc-biblioref" data-xml-rid="bib40" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref40" aria-controls="sref40">Matheson and Hall, 2003</a><div class="dropBlock__holder" data-db-target-of="sref40"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">41.</div><div>Matheson, E.C. ∙ Hall, A.G.</div><div><strong>Assessment of mismatch repair function in leukaemic cell lines and blasts from children with acute lymphoblastic leukaemia</strong></div><div><em>Carcinogenesis.</em> 2003; <strong>24</strong>:31-38</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/carcin/24.1.31" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-0037275722" target="_blank" aria-label="Open Scopus in new tab">Scopus (43)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/12538346/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fcarcin%2F24.1.31&amp;pmid=12538346" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), and U2OS (<span class="dropBlock reference-citations"><a id="body-ref-sref46" href="#" role="doc-biblioref" data-xml-rid="bib46" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref46" aria-controls="sref46">Peng et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref46"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">48.</div><div>Peng, M. ∙ Xie, J. ∙ Ucher, A. ...</div><div><strong>Crosstalk between BRCA-Fanconi anemia and mismatch repair pathways prevents MSH2-dependent aberrant DNA damage responses</strong></div><div><em>EMBO J.</em> 2014; <strong>33</strong>:1698-1712</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.15252/embj.201387530" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_48_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-84905381475" target="_blank" aria-label="Open Scopus in new tab">Scopus (44)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24966277/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.15252%2Fembj.201387530&amp;pmid=24966277" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). PE4 enhanced average editing efficiency over PE2 to a much greater extent in MMR+ cells (6.5-fold across 40 edits) than in MMR− cells (2.0-fold across 30 edits) while maintaining minimal indel frequencies (0.61% on average in MMR+ cells; <a id="crosref0975" href="#fig5">Figures 5</a>I, 5J, and <a id="crosref0980" href="#figs5">S5</a>L). Similarly, PE5 improved average editing efficiency over PE3 by 1.9-fold in MMR+ cells but only by 1.1-fold in MMR− cells. MLH1dn also increased edit/indel ratios to a similar degree in MMR+ and MMR− cells (2.8-fold and 3.2-fold, respectively; <a id="crosref0985" href="#fig5">Figure&nbsp;5</a>I). Intriguingly, although PE4 only increased G⋅C-to-C⋅G editing at <i>DNMT1</i> by 1.4-fold over PE2 in HEK293T cells (<a id="crosref0990" href="#figs5">Figure&nbsp;S5</a>A), we observed a larger improvement in MMR+ cells (averaging 2.7-fold; <a id="crosref0995" href="#fig5">Figure&nbsp;5</a>J), suggesting that PE4 and PE5 can enhance prime editing in MMR+ cell types for even classes of edits that evade MMR activity more effectively. Together, this comparison between 70 edits across seven endogenous sites in HEK293T, HeLa, K562, and U2OS cells illustrates that MLH1dn substantially improves prime editing efficiency, especially in MMR-proficient cells, which we expect to include most cell targets of prime editing.</div></section><section id="sec-2-9"><h3>Effect of MLH1dn on prime editing outcome purity</h3><div id="p0165" role="paragraph">Next, we examined in depth how MLH1dn reduces unintended prime editing outcomes. To decouple steps that lead to prime editing from those that lead to indel byproducts, we designed non-editing pegRNAs that template a 3′ DNA flap with perfect complementarity to the target locus and would result in no sequence change at the target locus (<a id="crosref1000" href="#fig6">Figure&nbsp;6</a>A). Across four endogenous sites in HEK293T cells, prime editing with non-editing pegRNAs yielded similar indel frequencies for PE3 (4.4%) and PE5 (4.3%; <a id="crosref1005" href="#fig6">Figures 6</a>B, 6C, and <a id="crosref1010" href="#figs6">S6</a>A), indicating that MLH1dn does not affect prime editing indel byproducts in the absence of a heteroduplex. In contrast, pegRNAs that program point mutations at these sites induced higher average indels (8.5% with PE3), which were reduced in frequency with MLH1dn (4.8% with PE5). MLH1dn also did not affect indels from PE2 with an inactivated RT (PE2–dRT) or SpCas9 H840A nickase (nCas9; <a id="crosref1015" href="#figs6">Figure&nbsp;S6</a>A), suggesting that MMR does not repair a doubly nicked intermediate lacking a 3′ flap. These results demonstrate that MMR engagement of the prime editing heteroduplex intermediate stimulates indel products that can be mitigated with PE5.</div><div class="figure-wrap"><figure id="fig6" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.09.018/asset/52831411-0f4a-45f5-a69f-cbd01da763c8/main.assets/gr6_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr6" src="/cms/10.1016/j.cell.2021.09.018/asset/7d8cebb9-dbc1-4445-aeb1-c17537c6cfa0/main.assets/gr6.jpg" height="707" width="664" aria-labelledby="fig6-title fig6-content" data-viewer-src="/cms/10.1016/j.cell.2021.09.018/asset/52831411-0f4a-45f5-a69f-cbd01da763c8/main.assets/gr6_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig6-title" class="figure__title"><span class="label figure__label">Figure&nbsp;6</span> <span class="figure__title__text">Effect of MLH1dn on prime editing outcome purity and off-targeting</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig6-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0185" role="paragraph">(A) Edit-encoding pegRNAs generate a heteroduplex following flap interconversion. Non-editing pegRNAs template a 3′ DNA flap with perfect complementarity to the genomic target site.</div><div id="fspara0190" role="paragraph">(B and C) Frequency of indels (B) and ratio of indel frequency (C) from PE3 or PE5 with four edit-encoding pegRNAs that program single-base mutations or four non-editing pegRNAs. Lines indicate mean of all individual values of sets of n&nbsp;= 3 independent replicates.</div><div id="fspara0195" role="paragraph">(D) Distribution of deletions at genomic target DNA formed by PE3 and PE5 using 12 substitution-encoding pegRNAs for each locus in HEK293T cells. Dotted lines indicate position of pegRNA- and sgRNA-directed nicks. Data represent the mean ± SD of n&nbsp;= 3 independent replicates.</div><div id="fspara0200" role="paragraph">(E and F) PE5/PE3 ratio of frequency of deletions that remove sequence greater than 25 nt outside of pegRNA- and sgRNA-directed nicks (E), and PE5/PE3 ratio of frequency of editing outcomes with unintended pegRNA scaffold sequence incorporation or unintended flap rejoining (F) in HEK293T cells. Each dot represents one of 84 total pegRNAs that program substitution edits (mean of n&nbsp;= 3 independent replicates).</div><div id="fspara0205" role="paragraph">(G) Off-target prime editing from PE2 and PE4 in HEK293T cells. Bars represent the mean of n&nbsp;= 3 independent replicates.</div><div id="fspara0210" role="paragraph">(H) High-throughput sequencing analysis of microsatellite repeat loci used for clinical diagnosis of MMR deficiency. HAP1 and HeLa cells are MMR proficient, and HCT116 cells have impaired MMR. HAP1 <i>ΔMSH2</i> cells underwent 60 cell divisions following <i>MSH2</i> knockout. HeLa cells were transiently transfected with PE2 or PE4 components. All values from n&nbsp;= 2 independent replicates are shown.</div></div></div></div></figcaption></figure></div><div class="figure-wrap"><figure id="figs6" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.09.018/asset/b7404c92-728e-440b-8b0f-ee35d761ebc4/main.assets/figs6_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs6" src="/cms/10.1016/j.cell.2021.09.018/asset/7a61dbc8-341f-4c05-af08-7e4504aee3df/main.assets/figs6.jpg" height="884" width="658" aria-labelledby="figs6-title figs6-content" data-viewer-src="/cms/10.1016/j.cell.2021.09.018/asset/b7404c92-728e-440b-8b0f-ee35d761ebc4/main.assets/figs6_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs6-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S6</span> <span class="figure__title__text">Effect of dominant negative MLH1 on prime editing outcome purity and off-targeting, related to <a id="crosref1795" href="#fig6">Figure&nbsp;6</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs6-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0465" role="paragraph">(A) Frequency of indels in HEK293T cells treated with pegRNAs, nicking sgRNAs, and PE2 enzyme, RT-impaired PE2 (PE2–dRT), or nickase Cas9 (SpCas9 H840A), with and without MLH1dn. Non-editing pegRNAs encode a 3′ DNA flap with perfect homology to the genomic target. Bars represent the mean of n&nbsp;= 3 independent replicates. Data supplements information in <a id="optGfQdKXnCi4" href="#fig6">Figures 6</a>B and 6C.</div><div id="fspara0470" role="paragraph">(B) Frequency of deletion as a function of genomic position relative to programmed nicks from PE3 and PE5 in HEK293T cells. 12 different pegRNAs that program single-base substitutions were tested at each indicated endogenous locus. Dotted lines indicate position of pegRNA- and sgRNA-directed nicks. Data represent the mean ± SD of n&nbsp;= 3 independent replicates.</div><div id="fspara0475" role="paragraph">(C) Distribution of deletion outcomes from PE3 and PE5 with an edit-encoding and non-editing pegRNA in HEK293T cells. The non-editing pegRNA templates a 3′&nbsp;DNA flap with perfect complementary to the genomic target sequence. Data represent the mean ± SD of n&nbsp;= 3 independent replicates.</div><div id="fspara0480" role="paragraph">(D) Frequency of all prime editing outcomes with unintended pegRNA scaffold sequence incorporation or unintended flap rejoining in HEK293T cells. 12 pegRNAs each programming a different single-base substitution were tested at each of the seven indicated loci. Each dot represents an individual pegRNA at the indicated locus (mean of n&nbsp;= 3 independent replicates).</div><div id="fspara0485" role="paragraph">(E) Off-target prime editing by PE2 and PE4 in HEK293T cells. Bars represent the mean of n&nbsp;= 3 independent replicates.</div><div id="fspara0490" role="paragraph">(F) Distribution and cumulative distribution of microsatellite repeat lengths in the indicated cell types and treatments. HAP1 and HeLa cells are MMR-proficient, and HCT116 cells have impaired MMR. HAP1 <i>ΔMSH2</i> cells underwent 60 cell divisions following knockout of <i>MSH2</i>. HeLa cells were transiently transfected with PE2 or PE4 components and grown for 3&nbsp;days before sequencing. wt, wild-type. All values from n&nbsp;= 2 independent replicates are shown.</div><div id="fspara0495" role="paragraph">(G) Prime editing at the on-target locus in HeLa cells transfected with PE2 or PE4 components. Bars represent the mean of n&nbsp;= 2 independent replicates. Microsatellite lengths were assayed from genomic DNA taken from these PE2 and PE4-treated HeLa cells.</div></div></div></div></figcaption></figure></div><div id="p0170" role="paragraph">We also measured the effect of MLH1dn on unintended prime editing outcome classes from 84 pegRNAs encoding single-base substitutions at endogenous loci in HEK293T cells. Similar to results from MMR gene knockdown (<a id="crosref1020" href="#fig2">Figure&nbsp;2</a>I), MLH1dn in the PE5 system reduced deletions outside of the pegRNA- and sgRNA-programmed nicks to a greater extent than deletions between these nicks (<a id="crosref1025" href="#fig6">Figures 6</a>D, 6E, and <a id="crosref1030" href="#figs6">S6</a>B), but not for a non-editing pegRNA that does not create a mismatch (<a id="crosref1035" href="#figs6">Figure&nbsp;S6</a>C). In addition, PE5 reduced the average frequency of pegRNA scaffold sequence incorporation (<a id="crosref1040" href="#fig2">Figure&nbsp;2</a>C) and unintended flap rejoining outcome categories (<a id="crosref1045" href="#fig2">Figure&nbsp;2</a>D) by 1.6-fold compared to PE3 (from 1.8% to 1.0%; <a id="crosref1050" href="#fig6">Figures 6</a>F and <a id="crosref1055" href="#figs6">S6</a>D). These outcomes were much rarer in the absence of a complementary-strand nick (0.27% frequency for PE2 and 0.28% for PE4; <a id="crosref1060" href="#figs6">Figure&nbsp;S6</a>D). Altogether, these data show that PE5 broadly narrows the size of unintended deletions, consistent with our model of DSB intermediate formation during PE3 (<a id="crosref1065" href="#figs3">Figure&nbsp;S3</a>D), and reduces the frequency of pegRNA scaffold sequence incorporation and unintended flap rejoining compared to PE3.</div></section><section id="sec-2-10"><h3>Effect of MLH1dn on off-target genomic DNA changes</h3><div id="p0175" role="paragraph">We next assessed whether MMR component manipulation could influence off-target editing. We tested PE2 and PE4 in HEK293T cells with eight pegRNAs and measured the resulting genomic changes at the four most common Cas9 off-target sites for each targeted locus (<span class="dropBlock reference-citations"><a id="body-ref-sref64-1" href="#" role="doc-biblioref" data-xml-rid="bib64" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref64-1" aria-controls="sref64-1">Tsai et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref64-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">66.</div><div>Tsai, S.Q. ∙ Nguyen, N.T. ∙ Malagon-Lopez, J. ...</div><div><strong>CIRCLE-seq: a highly sensitive in&nbsp;vitro screen for genome-wide CRISPR-Cas9 nuclease off-targets</strong></div><div><em>Nat. Methods.</em> 2017; <strong>14</strong>:607-614</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.4278" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-85018272560" target="_blank" aria-label="Open Scopus in new tab">Scopus (522)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28459458/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.4278&amp;pmid=28459458" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The average frequency of off-target prime editing remained very low with or without MLH1dn (0.094% with PE2, 0.12% with PE4), while the average efficiency of on-target editing increased from 9.7% for PE2 to 20% for PE4 (<a id="crosref1075" href="#fig6">Figures 6</a>G and <a id="crosref1080" href="#figs6">S6</a>E). These data are consistent with previous reports noting the high DNA specificity of prime editing (<span class="dropBlock reference-citations"><a id="body-ref-sref3-9" href="#" role="doc-biblioref" data-xml-rid="bib3" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref3-9" aria-controls="sref3-9">Anzalone et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref3-9"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">3.</div><div>Anzalone, A.V. ∙ Randolph, P.B. ∙ Davis, J.R. ...</div><div><strong>Search-and-replace genome editing without double-strand breaks or donor DNA</strong></div><div><em>Nature.</em> 2019; <strong>576</strong>:149-157</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-1711-4" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-85074571375" target="_blank" aria-label="Open Scopus in new tab">Scopus (2574)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31634902/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-1711-4&amp;pmid=31634902" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref29" href="#" role="doc-biblioref" data-xml-rid="bib29" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref29" aria-controls="sref29">Jin et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref29"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">29.</div><div>Jin, S. ∙ Lin, Q. ∙ Luo, Y. ...</div><div><strong>Genome-wide specificity of prime editors in plants</strong></div><div><em>Nat. Biotechnol.</em> 2021;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41587-021-00891-x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-85104810992" target="_blank" aria-label="Open Scopus in new tab">Scopus (87)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41587-021-00891-x" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref31" href="#" role="doc-biblioref" data-xml-rid="bib31" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref31" aria-controls="sref31">Kim et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref31"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">31.</div><div>Kim, D.Y. ∙ Moon, S.B. ∙ Ko, J.-H. ...</div><div><strong>Unbiased investigation of specificities of prime editing systems in human cells</strong></div><div><em>Nucleic Acids Res.</em> 2020; <strong>48</strong>:10576-10589</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gkaa764" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-85092749873" target="_blank" aria-label="Open Scopus in new tab">Scopus (98)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32941652/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgkaa764&amp;pmid=32941652" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and suggest that MLH1dn does not substantially increase guide-dependent off-target prime editing.</div><div id="p0180" role="paragraph">Next, we explored the idea that transient inhibition of MMR with MLH1dn might induce genomic mutations independent of prime editor activity. Because mutations that alter the length of repetitive microsatellite sequences are repaired almost exclusively by MMR (<span class="dropBlock reference-citations"><a id="body-ref-sref55" href="#" role="doc-biblioref" data-xml-rid="bib55" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref55" aria-controls="sref55">Strand et&nbsp;al., 1993</a><div class="dropBlock__holder" data-db-target-of="sref55"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">57.</div><div>Strand, M. ∙ Prolla, T.A. ∙ Liskay, R.M. ...</div><div><strong>Destabilization of tracts of simple repetitive DNA in yeast by mutations affecting DNA mismatch repair</strong></div><div><em>Nature.</em> 1993; <strong>365</strong>:274-276</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/365274a0" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-0027306173" target="_blank" aria-label="Open Scopus in new tab">Scopus (982)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/8371783/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2F365274a0&amp;pmid=8371783" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref62" href="#" role="doc-biblioref" data-xml-rid="bib62" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref62" aria-controls="sref62">Tran et&nbsp;al., 1997</a><div class="dropBlock__holder" data-db-target-of="sref62"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">64.</div><div>Tran, H.T. ∙ Keen, J.D. ∙ Kricker, M. ...</div><div><strong>Hypermutability of homonucleotide runs in mismatch repair and DNA polymerase proofreading yeast mutants</strong></div><div><em>Mol. Cell. Biol.</em> 1997; <strong>17</strong>:2859-2865</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/MCB.17.5.2859" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_64_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-0030962035" target="_blank" aria-label="Open Scopus in new tab">Scopus (284)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/9111358/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FMCB.17.5.2859&amp;pmid=9111358" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), microsatellite instability is used clinically as a measure of MMR activity in colorectal cancers (<span class="dropBlock reference-citations"><a id="body-ref-sref6-1" href="#" role="doc-biblioref" data-xml-rid="bib6" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref6-1" aria-controls="sref6-1">Bacher et&nbsp;al., 2004</a><div class="dropBlock__holder" data-db-target-of="sref6-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">6.</div><div>Bacher, J.W. ∙ Flanagan, L.A. ∙ Smalley, R.L. ...</div><div><strong>Development of a fluorescent multiplex assay for detection of MSI-High tumors</strong></div><div><em>Dis. Markers.</em> 2004; <strong>20</strong>:237-250</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1155/2004/136734" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-12844262940" target="_blank" aria-label="Open Scopus in new tab">Scopus (224)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15528789/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1155%2F2004%2F136734&amp;pmid=15528789" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref66-2" href="#" role="doc-biblioref" data-xml-rid="bib66" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref66-2" aria-controls="sref66-2">Umar et&nbsp;al., 2004</a><div class="dropBlock__holder" data-db-target-of="sref66-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">68.</div><div>Umar, A. ∙ Boland, C.R. ∙ Terdiman, J.P. ...</div><div><strong>Revised Bethesda Guidelines for hereditary nonpolyposis colorectal cancer (Lynch syndrome) and microsatellite instability</strong></div><div><em>J.&nbsp;Natl. Cancer Inst.</em> 2004; <strong>96</strong>:261-268</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/jnci/djh034" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-10744233937" target="_blank" aria-label="Open Scopus in new tab">Scopus (2668)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/14970275/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fjnci%2Fdjh034&amp;pmid=14970275" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). We evaluated microsatellite instability in HAP1, HeLa, and MMR-deficient HCT116 cells by high-throughput sequencing of 17 microsatellites previously validated as biomarkers of MMR activity in tumor specimens (<span class="dropBlock reference-citations"><a id="body-ref-sref23-1" href="#" role="doc-biblioref" data-xml-rid="bib23" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref23-1" aria-controls="sref23-1">Hempelmann et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref23-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">23.</div><div>Hempelmann, J.A. ∙ Scroggins, S.M. ∙ Pritchard, C.C. ...</div><div><strong>MSIplus for Integrated Colorectal Cancer Molecular Testing by Next-Generation Sequencing</strong></div><div><em>J.&nbsp;Mol. Diagn.</em> 2015; <strong>17</strong>:705-714</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_23_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=10.1016%2Fj.jmoldx.2015.05.008&amp;cf=fulltext&amp;site=jmdi-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_23_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=10.1016%2Fj.jmoldx.2015.05.008&amp;cf=pdf&amp;site=jmdi-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_23_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-84943630815" target="_blank" aria-label="Open Scopus in new tab">Scopus (42)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26322950/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.jmoldx.2015.05.008&amp;pmid=26322950" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). As expected from their MMR deficiency, HCT116 cells exhibited substantially shorter microsatellite lengths on average (13.9 nt) than HAP1 or HeLa cells (18.4 nt; <a id="crosref1125" href="#fig6">Figures 6</a>H and <a id="crosref1130" href="#figs6">S6</a>F). To gauge the sensitivity of this assay, we compared microsatellite instability in wild-type HAP1 cells and monoclonal HAP1 cells grown for 2&nbsp;months (∼60 cell divisions) following MMR knockout. These MMR knockout cells exhibited a 0.24-nt average decrease in microsatellite length (<a id="crosref1135" href="#fig6">Figures 6</a>H and <a id="crosref1140" href="#figs6">S6</a>F), establishing that even recent MMR impairment can be detected through the accumulation of microsatellite length erosion. To assess the effect of transient MLH1dn expression as used in PE4 and PE5 systems, we next measured microsatellite instability in MMR-proficient HeLa cells 3&nbsp;days after transfection with plasmids encoding PE2 or PE4. Although MLH1dn improved prime editing efficiency from 1.3% (PE2) to 7.6% (PE4) at the on-target locus (<a id="crosref1145" href="#figs6">Figure&nbsp;S6</a>G), average microsatellite lengths were indistinguishable between PE2- and PE4-treated cells (&lt;0.01 nt of difference; <a id="crosref1150" href="#fig6">Figures 6</a>H and <a id="crosref1155" href="#figs6">S6</a>F). These data indicate that transient MLH1dn expression can enhance prime editing without causing detected instability at 17 biomarker microsatellites sensitive to MMR deficiency.</div></section><section id="sec-2-11"><h3>PEmax systems with optimized editor architecture and synergy with engineered pegRNAs</h3><div id="p0185" role="paragraph">To further improve prime editing, we optimized the PE2 protein by varying RT codon usage, SpCas9 mutations, NLS sequences, and the length and composition of peptide linkers between nCas9 and RT (<a id="crosref1160" href="#figs7">Figure&nbsp;S7</a>A). Among 21 such variants tested, we observed the greatest enhancement in editing efficiency from a prime editor architecture that uses a human codon-optimized RT, a 34-aa linker containing a bipartite SV40 NLS (<span class="dropBlock reference-citations"><a id="body-ref-sref68" href="#" role="doc-biblioref" data-xml-rid="bib68" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref68" aria-controls="sref68">Wu et&nbsp;al., 2009</a><div class="dropBlock__holder" data-db-target-of="sref68"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">70.</div><div>Wu, J. ∙ Corbett, A.H. ∙ Berland, K.M.</div><div><strong>The intracellular mobility of nuclear import receptors and NLS cargoes</strong></div><div><em>Biophys. J.</em> 2009; <strong>96</strong>:3840-3849</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=10.1016%2Fj.bpj.2009.01.050&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=10.1016%2Fj.bpj.2009.01.050&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-67650351106" target="_blank" aria-label="Open Scopus in new tab">Scopus (34)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19413990/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.bpj.2009.01.050&amp;pmid=19413990" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), an additional C-terminal c-Myc NLS (<span class="dropBlock reference-citations"><a id="body-ref-sref12" href="#" role="doc-biblioref" data-xml-rid="bib12" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref12" aria-controls="sref12">Dang and Lee, 1988</a><div class="dropBlock__holder" data-db-target-of="sref12"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">12.</div><div>Dang, C.V. ∙ Lee, W.M.</div><div><strong>Identification of the human c-myc protein nuclear translocation signal</strong></div><div><em>Mol. Cell. Biol.</em> 1988; <strong>8</strong>:4048-4054</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/MCB.8.10.4048" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-0023731952" target="_blank" aria-label="Open Scopus in new tab">Scopus (287)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/3054508/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FMCB.8.10.4048&amp;pmid=3054508" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), and R221K N394K mutations in SpCas9 previously shown to improve Cas9 nuclease activity (<span class="dropBlock reference-citations"><a id="body-ref-sref54-1" href="#" role="doc-biblioref" data-xml-rid="bib54" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref54-1" aria-controls="sref54-1">Spencer and Zhang, 2017</a><div class="dropBlock__holder" data-db-target-of="sref54-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">56.</div><div>Spencer, J.M. ∙ Zhang, X.</div><div><strong>Deep mutational scanning of S. pyogenes Cas9 reveals important functional domains</strong></div><div><em>Sci. Rep.</em> 2017; <strong>7</strong>:16836</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41598-017-17081-y" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_56_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-85037086222" target="_blank" aria-label="Open Scopus in new tab">Scopus (34)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29203891/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41598-017-17081-y&amp;pmid=29203891" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) (<a id="crosref1180" href="#fig7">Figures 7</a>A and <a id="crosref1185" href="#figs7">S7</a>A). At seven target sites tested in HeLa cells, this optimized prime editor architecture (hereafter referred to as PEmax) outperforms other improved prime editor variants, including PE2<sup>∗</sup>, which includes additional NLS sequences (<span class="dropBlock reference-citations"><a id="body-ref-sref38-2" href="#" role="doc-biblioref" data-xml-rid="bib38" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref38-2" aria-controls="sref38-2">Liu et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref38-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">39.</div><div>Liu, P. ∙ Liang, S.-Q. ∙ Zheng, C. ...</div><div><strong>Improved prime editors enable pathogenic allele correction and cancer modelling in adult mice</strong></div><div><em>Nat. Commun.</em> 2021; <strong>12</strong>:2121</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41467-021-22295-w" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-85104130567" target="_blank" aria-label="Open Scopus in new tab">Scopus (163)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33837189/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41467-021-22295-w&amp;pmid=33837189" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), and CMP–PE–V1, which contains high-mobility peptides (<span class="dropBlock reference-citations"><a id="body-ref-sref44-1" href="#" role="doc-biblioref" data-xml-rid="bib44" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref44-1" aria-controls="sref44-1">Park et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref44-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">45.</div><div>Park, S.-J. ∙ Jeong, T.Y. ∙ Shin, S.K. ...</div><div><strong>Targeted mutagenesis in mouse cells and embryos using an enhanced prime</strong></div><div><em>Genome Biol.</em> 2021; <strong>22</strong>:170</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13059-021-02389-w" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_45_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-85107198784" target="_blank" aria-label="Open Scopus in new tab">Scopus (70)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/34082781/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13059-021-02389-w&amp;pmid=34082781" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) (<a id="crosref1200" href="#figs7">Figures S7</a>B–S7D). Inserting high-mobility peptides into PEmax (CMP–PEmax) did not further improve prime editing (<a id="crosref1205" href="#figs7">Figures S7</a>C and S7D).</div><div class="figure-wrap"><figure id="fig7" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.09.018/asset/1d884b08-b74e-4fdc-9ff0-0753171843c1/main.assets/gr7_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr7" src="/cms/10.1016/j.cell.2021.09.018/asset/8b8f0c17-531f-49d7-a07f-f3838af4c0c3/main.assets/gr7.jpg" height="743" width="764" aria-labelledby="fig7-title fig7-content" data-viewer-src="/cms/10.1016/j.cell.2021.09.018/asset/1d884b08-b74e-4fdc-9ff0-0753171843c1/main.assets/gr7_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig7-title" class="figure__title"><span class="label figure__label">Figure&nbsp;7</span> <span class="figure__title__text">PE4 and PE5 systems and PEmax architecture enhances editing at disease-relevant gene targets and cell types</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig7-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0220" role="paragraph">(A) Schematic of PE2 and PEmax editor architectures. bpNLS<sup>SV40</sup>, bipartite SV40 NLS. MMLV RT, Moloney murine leukemia virus RT pentamutant; codon opt., human codon-optimized.</div><div id="fspara0225" role="paragraph">(B) Prime editing with PE4 and PE5, PEmax, and epegRNAs at seven endogenous sites in HeLa and HEK293T cells. Fold changes indicate the average of fold increases from each edit tested. The mean ± SD of all individual values of n&nbsp;= 3 independent replicates are shown.</div><div id="fspara0230" role="paragraph">(C) Engineered pegRNAs (epegRNAs) contain a 3′ RNA structural motif and improve prime editing performance.</div><div id="fspara0235" role="paragraph">(D) Prime editing at therapeutically relevant sites (additional details in <a id="crosref1740" href="#sec-5">STAR Methods</a>) in wild-type HeLa and HEK293T cells. Bars represent the mean of n&nbsp;= 3 independent replicates.</div><div id="fspara0240" role="paragraph">(E) Correction of <i>CDKL5</i> c.1412delA in iPSCs derived from a patient heterozygous for the allele. Editing efficiencies indicate the percentage of sequencing reads with c.1412delA correction out of editable alleles that carry the mutation. Indel frequencies reflect all sequencing reads that contain any indels. Bars represent the mean of n&nbsp;= 3 independent replicates.</div><div id="fspara0245" role="paragraph">(F) Prime editing in primary human T&nbsp;cells. Bars represent the mean of n&nbsp;= 3 different T&nbsp;cell donors.</div></div></div></div></figcaption></figure></div><div class="figure-wrap"><figure id="figs7" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.09.018/asset/32155e21-fc16-4d0f-8199-10680fb6ca49/main.assets/figs7a_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs7-part-1" src="/cms/10.1016/j.cell.2021.09.018/asset/67aeff51-5b03-434f-82ae-3a4e63d23fa7/main.assets/figs7a.jpg" height="941" width="697" aria-labelledby="figs7-title figs7-content" data-viewer-src="/cms/10.1016/j.cell.2021.09.018/asset/32155e21-fc16-4d0f-8199-10680fb6ca49/main.assets/figs7a_lrg.jpg" loading="lazy"></a><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.09.018/asset/ae5dd588-664e-47ad-8bf2-9671ab209637/main.assets/figs7b_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs7-part-2" src="/cms/10.1016/j.cell.2021.09.018/asset/3599711b-5357-4094-8b24-68122e677539/main.assets/figs7b.jpg" height="375" width="697" aria-labelledby="figs7-title figs7-content" data-viewer-src="/cms/10.1016/j.cell.2021.09.018/asset/ae5dd588-664e-47ad-8bf2-9671ab209637/main.assets/figs7b_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs7-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S7</span> <span class="figure__title__text">Development of PEmax and application of PE4 and PE5 to primary cell types, related to <a id="crosref1800" href="#fig7">Figure&nbsp;7</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs7-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0505" role="paragraph">(A) Screen of prime editor variants for improved editing efficiency with the PE3 system in HeLa cells. All prime editor architectures carry a SpCas9 H840A mutation to prevent nicking of the complementary DNA strand at the target protospacer. NLS<sup>SV40</sup> indicates the bipartite SV40 NLS. <sup>∗</sup>NLS<sup>SV40</sup> contains a 1-aa deletion outside the PKKKRKV NLS<sup>SV40</sup> consensus sequence. All individual values of n&nbsp;= 3 independent replicates are shown.</div><div id="fspara0510" role="paragraph">(B) Architectures of the original PE2 editor (<span class="dropBlock reference-citations"><a id="body-ref-sref3-10" href="#" role="doc-biblioref" data-xml-rid="bib3" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref3-10" aria-controls="sref3-10">Anzalone et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref3-10"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">3.</div><div>Anzalone, A.V. ∙ Randolph, P.B. ∙ Davis, J.R. ...</div><div><strong>Search-and-replace genome editing without double-strand breaks or donor DNA</strong></div><div><em>Nature.</em> 2019; <strong>576</strong>:149-157</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-1711-4" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-85074571375" target="_blank" aria-label="Open Scopus in new tab">Scopus (2574)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31634902/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-1711-4&amp;pmid=31634902" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), PE2<sup>∗</sup> (<span class="dropBlock reference-citations"><a id="body-ref-sref38-3" href="#" role="doc-biblioref" data-xml-rid="bib38" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref38-3" aria-controls="sref38-3">Liu et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref38-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">39.</div><div>Liu, P. ∙ Liang, S.-Q. ∙ Zheng, C. ...</div><div><strong>Improved prime editors enable pathogenic allele correction and cancer modelling in adult mice</strong></div><div><em>Nat. Commun.</em> 2021; <strong>12</strong>:2121</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41467-021-22295-w" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-85104130567" target="_blank" aria-label="Open Scopus in new tab">Scopus (163)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33837189/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41467-021-22295-w&amp;pmid=33837189" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), CMP–PE–V1 (<span class="dropBlock reference-citations"><a id="body-ref-sref44-2" href="#" role="doc-biblioref" data-xml-rid="bib44" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref44-2" aria-controls="sref44-2">Park et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref44-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">45.</div><div>Park, S.-J. ∙ Jeong, T.Y. ∙ Shin, S.K. ...</div><div><strong>Targeted mutagenesis in mouse cells and embryos using an enhanced prime</strong></div><div><em>Genome Biol.</em> 2021; <strong>22</strong>:170</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13059-021-02389-w" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_45_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-85107198784" target="_blank" aria-label="Open Scopus in new tab">Scopus (70)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/34082781/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13059-021-02389-w&amp;pmid=34082781" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), and prime editor variants developed in this work (PEmax, CMP–PEmax). HN1, HMGN1; H1G, histone H1 central globular domain; codon opt., human codon optimized.</div><div id="fspara0515" role="paragraph">(C) PEmax outperforms other prime editor architectures tested with the PE3 system in HeLa cells. Bars represent the mean of n&nbsp;= 3 independent replicates.</div><div id="fspara0520" role="paragraph">(D) Fold-change in editing efficiency of prime editor architectures compared to PE2 with the PE3 system in HeLa cells. The mean ± SD of all individual values of n&nbsp;=&nbsp;3 independent replicates are shown.</div><div id="fspara0525" role="paragraph">(E) Intended editing and indel frequencies from PE4, PE4max (PE4 editing system with PEmax architecture), PE5, and PE5max (PE5 editing system with PEmax architecture) in HeLa and HEK293T cells cells. Seven substitution prime edits targeting different endogenous loci were tested for each condition. The mean ± SD of all individual values of n&nbsp;= 3 independent replicates are shown.</div><div id="fspara0530" role="paragraph">(F) Correction of <i>CDKL5</i> c.1412delA via an A⋅T insertion and a G⋅C-to-A⋅T edit in iPSCs derived from a patient heterozygous for the disease allele. Editing efficiencies indicate the percentage of sequencing reads with c.1412delA correction out of editable alleles that carry the mutation. Indel frequencies reflect all sequencing reads that contain any indels that do not map to the c.1412delA allele or wild-type sequence. 1&nbsp;μg of PE2 mRNA was used in all conditions shown. Bars represent the mean of n&nbsp;= 3 independent replicates. Data supplements information in <a id="optizqbMT0jQb" href="#fig7">Figure 7</a>E.</div><div id="fspara0535" role="paragraph">(G) Prime editing in primary T&nbsp;cells, supplementing information in <a id="optEULOYOiJ64" href="#fig7">Figure 7</a>F. Bars represent the mean of n&nbsp;= 3 independent replicates from different T&nbsp;cell donors.</div></div></div></div></figcaption></figure></div><div id="p0190" role="paragraph">Across seven substitution edits targeting different loci, using the PEmax architecture with PE2, PE3, PE4, or PE5 systems (hereafter referred to as PE2max, PE3max, PE4max, and PE5max, respectively) increased the average frequency of intended editing by 2.5-fold in HeLa cells and 1.2-fold in HEK293T cells compared the original PE2 editor architecture (<span class="dropBlock reference-citations"><a id="body-ref-sref3-11" href="#" role="doc-biblioref" data-xml-rid="bib3" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref3-11" aria-controls="sref3-11">Anzalone et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref3-11"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">3.</div><div>Anzalone, A.V. ∙ Randolph, P.B. ∙ Davis, J.R. ...</div><div><strong>Search-and-replace genome editing without double-strand breaks or donor DNA</strong></div><div><em>Nature.</em> 2019; <strong>576</strong>:149-157</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-1711-4" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-85074571375" target="_blank" aria-label="Open Scopus in new tab">Scopus (2574)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31634902/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-1711-4&amp;pmid=31634902" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) (<a id="crosref1215" href="#fig7">Figures 7</a>B and <a id="crosref1220" href="#figs7">S7</a>E). PE3max and PE5max also slightly reduced average edit/indel purity by 1.2-fold compared to PE3 and PE5, respectively, which may reflect enhanced nickase activity from the SpCas9 R221K and N394K mutations within the PEmax architecture (<span class="dropBlock reference-citations"><a id="body-ref-sref54-2" href="#" role="doc-biblioref" data-xml-rid="bib54" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref54-2" aria-controls="sref54-2">Spencer and Zhang, 2017</a><div class="dropBlock__holder" data-db-target-of="sref54-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">56.</div><div>Spencer, J.M. ∙ Zhang, X.</div><div><strong>Deep mutational scanning of S. pyogenes Cas9 reveals important functional domains</strong></div><div><em>Sci. Rep.</em> 2017; <strong>7</strong>:16836</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41598-017-17081-y" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_56_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-85037086222" target="_blank" aria-label="Open Scopus in new tab">Scopus (34)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29203891/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41598-017-17081-y&amp;pmid=29203891" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="p0195" role="paragraph">We also assessed whether PE4max and PE5max systems can synergize with epegRNAs, which contain an additional 3′ RNA structural motif that increases prime editing efficacy (<span class="dropBlock reference-citations"><a id="body-ref-sref42-2" href="#" role="doc-biblioref" data-xml-rid="bib42" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref42-2" aria-controls="sref42-2">Nelson et al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref42-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">43.</div><div>Nelson, J.W. ∙ Randolph, P.B. ∙ Shen, S.P. ...</div><div><strong>Engineered pegRNAs improve prime editing efficiency</strong></div><div><em>Nat. Biotechnol.</em> 2021;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41587-021-01039-7" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-85116367544" target="_blank" aria-label="Open Scopus in new tab">Scopus (297)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/34608327/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41587-021-01039-7&amp;pmid=34608327" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) (<a id="crosref1235" href="#fig7">Figure&nbsp;7</a>C). Across seven substitution edits, epegRNAs improved PE4max editing efficiency over normal pegRNAs by an average 2.5-fold (HeLa) and 1.5-fold (HEK293T; <a id="crosref1240" href="#fig7">Figure&nbsp;7</a>B). Similarly, epegRNAs enhanced PE5max editing over normal pegRNAs by 1.4-fold (HeLa) and 1.1-fold (HEK293T), without affecting edit/indel purity.</div><div id="p0200" role="paragraph">Combining all enhancements to prime editing systems described above (MLH1dn, PEmax, and epegRNAs) dramatically improved prime editing performance. PE4max with epegRNAs enhanced editing efficiency by an average of 72-fold in MMR-proficient HeLa cells and 3.5-fold in MMR-deficient HEK293T cells relative to PE2 with normal pegRNAs (<a id="crosref1245" href="#fig7">Figure&nbsp;7</a>B). PE5max with epegRNAs also improved editing efficiency over PE3 with pegRNAs by 12-fold (HeLa) and 1.6-fold (HEK293T) on average and increased outcome purity by 4.6-fold (HeLa) and 3.3-fold (HEK293T). Collectively, these results demonstrate that combining PE4/PE5, PEmax, and epegRNA strategies can greatly enhance prime editing outcomes.</div></section><section id="sec-2-12"><h3>Prime editing of disease-relevant loci and cell types with PE4 and PE5</h3><div id="p0205" role="paragraph">To establish the applicability of these improved editing systems, we used PE4max and PE5max to edit six genomic sites associated with sickle cell anemia (<span class="dropBlock reference-citations"><a id="body-ref-sref27-1" href="#" role="doc-biblioref" data-xml-rid="bib27" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref27-1" aria-controls="sref27-1">Ingram, 1956</a><div class="dropBlock__holder" data-db-target-of="sref27-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">27.</div><div>Ingram, V.M.</div><div><strong>A specific chemical difference between the globins of normal human and sickle-cell anaemia haemoglobin</strong></div><div><em>Nature.</em> 1956; <strong>178</strong>:792-794</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/178792a0" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-0000420850" target="_blank" aria-label="Open Scopus in new tab">Scopus (506)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/13369537/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2F178792a0&amp;pmid=13369537" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), prion disease (<span class="dropBlock reference-citations"><a id="body-ref-sref5-1" href="#" role="doc-biblioref" data-xml-rid="bib5" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref5-1" aria-controls="sref5-1">Asante et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref5-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">5.</div><div>Asante, E.A. ∙ Smidak, M. ∙ Grimshaw, A. ...</div><div><strong>A naturally occurring variant of the human prion protein completely prevents prion disease</strong></div><div><em>Nature.</em> 2015; <strong>522</strong>:478-481</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature14510" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-84933041955" target="_blank" aria-label="Open Scopus in new tab">Scopus (128)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26061765/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature14510&amp;pmid=26061765" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), CDKL5 deficiency disorder (<span class="dropBlock reference-citations"><a id="body-ref-sref43-1" href="#" role="doc-biblioref" data-xml-rid="bib43" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref43-1" aria-controls="sref43-1">Olson et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref43-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">44.</div><div>Olson, H.E. ∙ Demarest, S.T. ∙ Pestana-Knight, E.M. ...</div><div><strong>Cyclin-Dependent Kinase-Like 5 Deficiency Disorder: Clinical Review</strong></div><div><em>Pediatr. Neurol.</em> 2019; <strong>97</strong>:18-25</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=10.1016%2Fj.pediatrneurol.2019.02.015&amp;cf=fulltext&amp;site=pnu-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=10.1016%2Fj.pediatrneurol.2019.02.015&amp;cf=pdf&amp;site=pnu-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-85063410039" target="_blank" aria-label="Open Scopus in new tab">Scopus (144)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30928302/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.pediatrneurol.2019.02.015&amp;pmid=30928302" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), HIV infection (<span class="dropBlock reference-citations"><a id="body-ref-sref36-1" href="#" role="doc-biblioref" data-xml-rid="bib36" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref36-1" aria-controls="sref36-1">Liu et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref36-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">36.</div><div>Liu, S. ∙ Wang, Q. ∙ Yu, X. ...</div><div><strong>HIV-1 inhibition in cells with CXCR4 mutant genome created by CRISPR-Cas9 and piggyBac recombinant technologies</strong></div><div><em>Sci. Rep.</em> 2018; <strong>8</strong>:8573</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41598-018-26894-4" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-85048116209" target="_blank" aria-label="Open Scopus in new tab">Scopus (28)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29872154/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41598-018-26894-4&amp;pmid=29872154" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), and adoptive T&nbsp;cell transfer therapy (<span class="dropBlock reference-citations"><a id="body-ref-sref53-1" href="#" role="doc-biblioref" data-xml-rid="bib53" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref53-1" aria-controls="sref53-1">Sockolosky et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref53-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">55.</div><div>Sockolosky, J.T. ∙ Trotta, E. ∙ Parisi, G. ...</div><div><strong>Selective targeting of engineered T&nbsp;cells using orthogonal IL-2 cytokine-receptor complexes</strong></div><div><em>Science.</em> 2018; <strong>359</strong>:1037-1042</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aar3246" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-85047997059" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29496879/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aar3246&amp;pmid=29496879" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Across these sites, PE4max increased average prime editing efficiency over PE2 by 29-fold in HeLa cells and 2.1-fold in HEK293T cells (<a id="crosref1275" href="#fig7">Figure&nbsp;7</a>D). Notably, PE4max editing efficiencies (8.6% editing and 0.19% indels in HeLa, 20% editing and 0.26% indels in HEK293T) were similar to or exceeded those of PE3 (4.5% editing and 1.5% indels in HeLa and 24% editing and 5.4% indels in HEK293T) but with far fewer indels. In addition, PE5max improved disease-relevant allele conversion over PE3 by an average of 6.1-fold (HeLa) and 1.5-fold (HEK293T) and enhanced edit/indel purity by 6.4-fold (HeLa) and 3.5-fold (HEK293T; <a id="crosref1280" href="#fig7">Figure&nbsp;7</a>D). Taken together, these results demonstrate that PE4max and PE5max support substantially higher prime editing performance compared to PE2 and PE3 at therapeutically relevant gene targets in cell culture.</div><div id="p0210" role="paragraph">Next, we used PE4 and PE5 to correct the pathogenic <i>CDKL5</i> c.1412delA mutation in human iPSCs derived from a heterozygous patient (<span class="dropBlock reference-citations"><a id="body-ref-sref10-1" href="#" role="doc-biblioref" data-xml-rid="bib10" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref10-1" aria-controls="sref10-1">Chen et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref10-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">10.</div><div>Chen, P.-F. ∙ Chen, T. ∙ Forman, T.E. ...</div><div><strong>Generation and characterization of human induced pluripotent stem cells (iPSCs) from three male and three female patients with CDKL5 Deficiency Disorder (CDD)</strong></div><div><em>Stem Cell Res. (Amst.).</em> 2021; <strong>53</strong>:102276</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.scr.2021.102276" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-85102606970" target="_blank" aria-label="Open Scopus in new tab">Scopus (6)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33714067/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.scr.2021.102276&amp;pmid=33714067" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Electroporation of iPSCs with <i>in vitro</i>-transcribed PE2 mRNA and synthetic pegRNAs and nicking sgRNAs (PE3) yielded 17% correction of editable pathogenic alleles and 20% total indel products (<a id="crosref1290" href="#fig7">Figures 7</a>E and <a id="crosref1295" href="#figs7">S7</a>F). Co-electroporation of these components with MLH1dn mRNA (PE5) elevated correction efficiency to 34% and lowered the frequency of indels to 6.1%. PE4 and PE5b systems also improved allele correction by 2.5-fold and 2.1-fold over PE2 and PE3b, respectively, with few indels (0.34% from PE4 and 3.8% from PE5b). Thus, across these prime editing systems tested, MLH1dn enhances <i>CDKL5</i> c.1412delA correction by 2.2-fold in&nbsp;efficiency and 3.6-fold in outcome purity in patient-derived iPSCs.</div><div id="p0215" role="paragraph">Lastly, we tested mRNA delivery of PE5 in primary human T&nbsp;cells to introduce the protective <i>PRNP</i> G127V mutation, a G⋅C-to-T⋅A transversion at <i>FANCF</i>, and a 1-bp insertion at <i>RNF2</i>. We also installed the protective <i>CXCR4</i> P191A allele that prevents HIV infection (<span class="dropBlock reference-citations"><a id="body-ref-sref36-2" href="#" role="doc-biblioref" data-xml-rid="bib36" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref36-2" aria-controls="sref36-2">Liu et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref36-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">36.</div><div>Liu, S. ∙ Wang, Q. ∙ Yu, X. ...</div><div><strong>HIV-1 inhibition in cells with CXCR4 mutant genome created by CRISPR-Cas9 and piggyBac recombinant technologies</strong></div><div><em>Sci. Rep.</em> 2018; <strong>8</strong>:8573</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41598-018-26894-4" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-85048116209" target="_blank" aria-label="Open Scopus in new tab">Scopus (28)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29872154/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41598-018-26894-4&amp;pmid=29872154" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), and the <i>IL2RB</i> H134D Y135F variant that enables orthogonal IL-2 T&nbsp;cell stimulation (<span class="dropBlock reference-citations"><a id="body-ref-sref53-2" href="#" role="doc-biblioref" data-xml-rid="bib53" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref53-2" aria-controls="sref53-2">Sockolosky et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref53-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">55.</div><div>Sockolosky, J.T. ∙ Trotta, E. ∙ Parisi, G. ...</div><div><strong>Selective targeting of engineered T&nbsp;cells using orthogonal IL-2 cytokine-receptor complexes</strong></div><div><em>Science.</em> 2018; <strong>359</strong>:1037-1042</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aar3246" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-85047997059" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29496879/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aar3246&amp;pmid=29496879" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) using PE4 and PE5. Across these five sites, we found that MLH1dn in the PE4 or PE5 systems enhanced editing efficiency by 2.2-fold and the edit/indel ratio by 2.7-fold, achieving an average of 46% editing with 11% indels from PE5 (<a id="crosref1310" href="#fig7">Figures 7</a>F and <a id="crosref1315" href="#figs7">S7</a>G). Collectively, these results across six loci in human iPSCs and primary T&nbsp;cells establish PE4 and PE5 as enhanced prime editing systems that enable substantially greater editing efficiency and outcome purity in cell types relevant to the study and potential treatment of genetic disease.</div></section></section><section id="sec-3"><h2>Discussion</h2><div id="p0220" role="paragraph">Using pooled CRISPRi screens, we discovered that MMR activity strongly suppresses the efficiency and outcome purity of substitution prime edits. These insights informed the development of PE4 and PE5 systems that co-express MLH1dn to transiently inhibit MMR, enhance prime editing efficacy, and reduce indels without inducing substantial off-target genomic changes. Optimization of the prime editor protein resulted in a PEmax architecture that can synergize with PE4 and PE5 systems and with epegRNAs (<span class="dropBlock reference-citations"><a id="body-ref-sref42-3" href="#" role="doc-biblioref" data-xml-rid="bib42" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref42-3" aria-controls="sref42-3">Nelson et al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref42-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">43.</div><div>Nelson, J.W. ∙ Randolph, P.B. ∙ Shen, S.P. ...</div><div><strong>Engineered pegRNAs improve prime editing efficiency</strong></div><div><em>Nat. Biotechnol.</em> 2021;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41587-021-01039-7" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-85116367544" target="_blank" aria-label="Open Scopus in new tab">Scopus (297)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/34608327/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41587-021-01039-7&amp;pmid=34608327" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) to further enhance prime editing performance. Together, the model for DNA repair of prime editing supported by these findings, the PE4 and PE5 strategies developed to circumvent a prime editing bottleneck, and the improved PEmax prime editor architecture described here substantially advance the utility of prime editing for precision manipulation of the genome.</div><div id="p0225" role="paragraph">Broad characterization of PE4 and PE5 across 191 diverse prime edits reveals that prime editors can install certain types of edits with higher efficiency due to the ability of the corresponding prime editing intermediates to evade MMR. In addition to edit type, other properties could also affect the sensitivity of prime editing to MMR, such as the sequence context of the target site. Moreover, MMR more efficiently repairs early replicating euchromatin (<span class="dropBlock reference-citations"><a id="body-ref-sref58" href="#" role="doc-biblioref" data-xml-rid="bib58" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref58" aria-controls="sref58">Supek and Lehner, 2015</a><div class="dropBlock__holder" data-db-target-of="sref58"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">60.</div><div>Supek, F. ∙ Lehner, B.</div><div><strong>Differential DNA mismatch repair underlies mutation rate variation across the human genome</strong></div><div><em>Nature.</em> 2015; <strong>521</strong>:81-84</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature14173" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-84925615834" target="_blank" aria-label="Open Scopus in new tab">Scopus (245)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25707793/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature14173&amp;pmid=25707793" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and lagging strand DNA during replication (<span class="dropBlock reference-citations"><a id="body-ref-sref39-2" href="#" role="doc-biblioref" data-xml-rid="bib39" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref39-2" aria-controls="sref39-2">Lujan et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref39-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">40.</div><div>Lujan, S.A. ∙ Clausen, A.R. ∙ Clark, A.B. ...</div><div><strong>Heterogeneous polymerase fidelity and mismatch repair bias genome variation and composition</strong></div><div><em>Genome Res.</em> 2014; <strong>24</strong>:1751-1764</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/gr.178335.114" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_40_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-84923948943" target="_blank" aria-label="Open Scopus in new tab">Scopus (123)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25217194/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2Fgr.178335.114&amp;pmid=25217194" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). A systematic study across a larger set of edits will be needed to comprehensively elucidate these edit type, sequence context, and locus state dependencies on MMR and prime editing. Repair-seq may also be applied in the future to illuminate other classes of prime edits, such as long insertions or deletions, and suggest additional improved prime editing systems.</div><div id="p0230" role="paragraph">Our study of the types of prime editing intermediates that are repaired by MMR allows researchers to design prime editing experiments to evade MMR, even without expression of MLH1dn. We show that strategically installing additional nearby silent mutations can enhance prime editing outcomes by avoiding MMR reversal of prime editing intermediates. Other modalities for MMR inhibition may also prove beneficial for prime editing. Although no small molecules that selectively target MMR have been reported, chemical inhibitors would be useful in applications limited by MLH1dn delivery. For uses such as viral delivery that maintain long-term prime editor expression, RNA interference may offer an alternate means to transiently knock down MMR activity.</div><div id="p0235" role="paragraph">PE4 and PE5 systems powerfully enhance prime editing performance and precision in seven mammalian cell types tested and synergize with improvements from PEmax and epegRNAs. PE4max with epegRNAs uniquely enables efficient prime editing with low indel byproducts, particularly in cells with active MMR, making it most suitable for gene editing applications that require high outcome purity or cannot use nicking sgRNAs. In comparison, PE5max with epegRNAs achieves the highest levels of prime editing with reduced indel outcomes compared to PE3 systems. We therefore recommend the use of PE5max and epegRNAs for most prime editing applications.</div></section><section id="sec-4"><h2>Limitations of the study</h2><div id="p0240" role="paragraph">While complete knockout of MMR activity enhances prime editing, it remains unknown how a heteroduplex prime editing intermediate is resolved in the absence of MMR. Future work in post-mitotic MMR-deficient cells could illuminate the requirement of cellular factors or DNA replication for resolving the heteroduplex intermediate. In addition, we showed that transient MLH1dn expression minimally perturbs microsatellites sensitive to MMR activity, though the potential impact of MMR deficiency on genomic mutation rates is well documented (<span class="dropBlock reference-citations"><a id="body-ref-sref39-3" href="#" role="doc-biblioref" data-xml-rid="bib39" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref39-3" aria-controls="sref39-3">Lujan et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref39-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">40.</div><div>Lujan, S.A. ∙ Clausen, A.R. ∙ Clark, A.B. ...</div><div><strong>Heterogeneous polymerase fidelity and mismatch repair bias genome variation and composition</strong></div><div><em>Genome Res.</em> 2014; <strong>24</strong>:1751-1764</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/gr.178335.114" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_40_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-84923948943" target="_blank" aria-label="Open Scopus in new tab">Scopus (123)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25217194/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2Fgr.178335.114&amp;pmid=25217194" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref71" href="#" role="doc-biblioref" data-xml-rid="bib71" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref71" aria-controls="sref71">Zou et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref71"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">73.</div><div>Zou, X. ∙ Koh, G.C.C. ∙ Nanda, A.S. ...</div><div><strong>A systematic CRISPR screen defines mutational mechanisms underpinning signatures caused by replication errors and endogenous DNA damage</strong></div><div><em>Nat. Cancer.</em> 2021; <strong>2</strong>:643-657</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s43018-021-00200-0" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-85105126717" target="_blank" aria-label="Open Scopus in new tab">Scopus (77)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/34164627/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs43018-021-00200-0&amp;pmid=34164627" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). An analysis of genome-wide mutations induced by transient PE4 and PE5 expression would more sensitively quantify their off-target editing consequences. Lastly, we demonstrated that installing additional mutations near the intended edit can increase prime editing efficiency by evading MMR. To generalize this strategy, a larger set of edits may need to be tested to establish design rules for silent or benign edits that optimally evade MMR.</div></section><section id="sec-5"><h2>STAR★Methods</h2><section id="sec-5-1"><h3>Key resources table</h3><div id="p0245" role="paragraph"><div class="figure-wrap"><figure id="undtbl1" class="table" data-figureviewer-ignore="true"><div class="table-wrap"><table data-figureviewer-ignore="true" tabindex="0"><thead><tr class="rowsep"><th>REAGENT or RESOURCE</th><th>SOURCE</th><th>IDENTIFIER</th></tr></thead><tbody><tr class="rowsep"><td colspan="3"><b>Bacterial and virus strains</b></td></tr><tr><td>One Shot Mach1 T1 Phage-Resistant Chemically Competent <i>E. coli</i></td><td>Thermo Fisher Scientific</td><td>Cat#C862003</td></tr><tr class="rowsep"><td>MegaX DH10B T1 R Electrocomp Cells</td><td>Thermo Fisher Scientific</td><td>Cat#C640003</td></tr><tr class="rowsep"><td colspan="3"><b>Chemicals, peptides, and recombinant proteins</b></td></tr><tr><td>USER enzyme</td><td>New England BioLabs</td><td>Cat#M5505S</td></tr><tr><td>DpnI</td><td>New England BioLabs</td><td>Cat#R0176S</td></tr><tr><td>BsaI-HFv2</td><td>New England BioLabs</td><td>Cat#R3733S</td></tr><tr><td>T4 DNA Ligase</td><td>New England BioLabs</td><td>Cat#M0202S</td></tr><tr><td>BstXI</td><td>Thermo Fisher Scientific</td><td>Cat#ER1021</td></tr><tr><td>Bpu1102I (BlpI)</td><td>Thermo Fisher Scientific</td><td>Cat#ER0091</td></tr><tr><td>Lipofectamine 2000</td><td>Thermo Fisher Scientific</td><td>Cat#11668019</td></tr><tr><td>Lipofectamine 3000</td><td>Thermo Fisher Scientific</td><td>Cat#L3000015</td></tr><tr><td>Lipofectamine RNAiMAX</td><td>Thermo Fisher Scientific</td><td>Cat#13778150</td></tr><tr><td>TransIT-HelaMONSTER</td><td>Mirus Bio</td><td>Cat#MIR 2904</td></tr><tr><td>TrypLE</td><td>Thermo Fisher Scientific</td><td>Cat#12605010</td></tr><tr><td>Polybrene (Hexadimethrine bromide)</td><td>Sigma-Aldrich</td><td>Cat#107689-10G</td></tr><tr><td>Puromycin Dihydrochloride</td><td>Thermo Fisher Scientific</td><td>Cat#A1113803</td></tr><tr><td>Blasticidin S HCl</td><td>Thermo Fisher Scientific</td><td>Cat#A1113903</td></tr><tr><td>Penicillin-Streptomycin</td><td>Thermo Fisher Scientific</td><td>Cat#15070063</td></tr><tr><td>L-Glutamine</td><td>Corning</td><td>Cat#25-005-Cl</td></tr><tr><td>Proteinase K</td><td>New England BioLabs</td><td>Cat#P8107S</td></tr><tr><td>ViralBoost Reagent</td><td>ALSTEM</td><td>Cat#VB100</td></tr><tr><td>SPRIselect</td><td>Beckman Coulter</td><td>Cat#A63881</td></tr><tr><td>AMPure XP</td><td>Beckman Coulter</td><td>Cat#B23318</td></tr><tr><td>CleanCap Reagent AG</td><td>TriLink BioTechnologies</td><td>Cat#N-7113</td></tr><tr><td>N<sup>1</sup>-Methylpseudouridine-5′-Triphosphate</td><td>TriLink BioTechnologies</td><td>Cat#N-1081</td></tr><tr><td>LiCl Precipitation Solution (7.5 M)</td><td>Thermo Fisher Scientific</td><td>Cat#AM9480</td></tr><tr><td>StemFlex medium</td><td>Thermo Fisher Scientific</td><td>Cat#A3349401</td></tr><tr><td>Geltrex Basement Membrane Matrix</td><td>Thermo Fisher Scientific</td><td>Cat#A1413301</td></tr><tr><td>DMEM/F12, GlutaMAX supplement</td><td>Thermo Fisher Scientific</td><td>Cat#10565018</td></tr><tr><td>Gentle Cell Dissociation Reagent</td><td>STEMCELL Technologies</td><td>Cat#07174</td></tr><tr><td>rhLaminin-521</td><td>Thermo Fisher Scientific</td><td>Cat#A29249</td></tr><tr><td>Y-27632</td><td>Cayman Chemical</td><td>Cat#10005583 (CAS#129830-38-2)</td></tr><tr><td>Accutase</td><td>Innovative Cell Technologies</td><td>Cat#AT104</td></tr><tr><td>Lymphoprep density gradient medium</td><td>STEMCELL Technologies</td><td>Cat#07801</td></tr><tr><td>Dynabeads Human T-Expander CD3/CD28</td><td>Thermo Fisher Scientific</td><td>Cat#11141D</td></tr><tr><td>Human AB Serum</td><td>Valley Biomedical</td><td>Cat#HP1022HI</td></tr><tr><td>GlutaMAX supplement</td><td>Thermo Fisher Scientific</td><td>Cat#35050061</td></tr><tr><td><i>N</i>-Acetyl-L-cysteine</td><td>Sigma-Aldrich</td><td>Cat#A7250-100G</td></tr><tr><td>Recombinant Human IL-2</td><td>Peprotech</td><td>Cat#200-02</td></tr><tr><td>Recombinant Human IL-7</td><td>Peprotech</td><td>Cat#200-07</td></tr><tr><td>Recombinant Human IL-15</td><td>Peprotech</td><td>Cat#200-15</td></tr><tr class="rowsep"><td>CleanCap EGFP mRNA</td><td>TriLink BioTechnologies</td><td>Cat#L-7601</td></tr><tr class="rowsep"><td colspan="3"><b>Critical commercial assays</b></td></tr><tr><td>Phusion U Multiplex PCR Master Mix</td><td>Thermo Fisher Scientific</td><td>Cat#F562L</td></tr><tr><td>Q5 High-Fidelity 2 × Master Mix</td><td>New England BioLabs</td><td>Cat#M0492L</td></tr><tr><td>Phusion High-Fidelity DNA Polymerase</td><td>Thermo Fisher Scientific</td><td>Cat#F530S</td></tr><tr><td>NEBNext Ultra II Q5 Master Mix</td><td>New England BioLabs</td><td>Cat#M0544L</td></tr><tr><td>QIAquick PCR Purification Kit</td><td>QIAGEN</td><td>Cat#28104</td></tr><tr><td>QIAquick Gel Extraction Kit</td><td>QIAGEN</td><td>Cat#28704</td></tr><tr><td>NucleoSpin Gel and PCR Clean-up</td><td>Macherey-Nagel</td><td>Cat#74609.50</td></tr><tr><td>QIAGEN Plasmid <i>Plus</i> Midi Kit</td><td>QIAGEN</td><td>Cat#12943</td></tr><tr><td>QIAGEN Plasmid <i>Plus</i> Maxi Kit</td><td>QIAGEN</td><td>Cat#12963</td></tr><tr><td>ZymoPURE II Plasmid Maxiprep Kit</td><td>Zymo Research</td><td>Cat#D4202</td></tr><tr><td>PureYield Plasmid Miniprep System</td><td>Promega Corporation</td><td>Cat#A1222</td></tr><tr><td>Macherey-Nagel NucleoSpin Blood XL, Maxi kit</td><td>Machery-Nagel</td><td>Cat#740950.50</td></tr><tr><td>SE Cell Line 4D-Nucleofector X Kit S</td><td>Lonza</td><td>Cat#V4XC-1032</td></tr><tr><td>SF Cell Line 4D-Nucleofector X Kit S</td><td>Lonza</td><td>Cat#V4XC-2032</td></tr><tr><td>Bioanalyzer High Sensitivity DNA Kit</td><td>Agilent Technologies</td><td>Cat#5067-4626</td></tr><tr><td>SYBR Green Fast Advanced Cells-to-CT Kit</td><td>Thermo Fisher Scientific</td><td>Cat#A35380</td></tr><tr><td>NEB T7 HiScribe Kit</td><td>New England BioLabs</td><td>Cat#E2040S</td></tr><tr><td>EasySep Human T Cell Isolation Kit</td><td>STEMCELL Technologies</td><td>Cat#17951</td></tr><tr><td>NEON Transfection System 10 μL Kit</td><td>Thermo Fisher Scientific</td><td>Cat#MPK1025</td></tr><tr><td>NEON Transfection System</td><td>Thermo Fisher Scientific</td><td>Cat#MPK5000</td></tr><tr><td>PureLink Genomic DNA Mini Kit</td><td>Thermo Fisher Scientific</td><td>Cat#K182002</td></tr><tr><td>MiSeq Reagent Kit v2 (300-cycles)</td><td>Illumina</td><td>Cat#MS-102-2002</td></tr><tr><td>MiSeq Reagent Micro Kit v2 (300-cycles)</td><td>Illumina</td><td>Cat#MS-103-1002</td></tr><tr class="rowsep"><td>NovaSeq 6000 S1 Reagent Kit v1.5 (300-cycles)</td><td>Illumina</td><td>Cat#20028317</td></tr><tr class="rowsep"><td colspan="3"><b>Deposited data</b></td></tr><tr><td>Amplicon sequencing data</td><td>This paper</td><td>PRJNA735408</td></tr><tr><td>Data from Repair-seq screens</td><td>This paper</td><td>PRJNA734952</td></tr><tr class="rowsep"><td>Processed data from Repair-seq screens</td><td>This paper</td><td>DOI: <a id="PC_linkuMxChF7ycz" href="https://doi.org/10.5281/zenodo.5551032" target="_blank">https://doi.org/10.5281/zenodo.5551032</a></td></tr><tr class="rowsep"><td colspan="3"><b>Experimental models: Cell lines</b></td></tr><tr><td>Human (female): HEK293T</td><td>ATCC</td><td>Cat#CRL-3216</td></tr><tr><td>Human (female): HeLa expressing dCas9–BFP–KRAB (Addgene #46911)</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref18-3" href="#" role="doc-biblioref" data-xml-rid="bib18" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref18-3" aria-controls="sref18-3">Gilbert et al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref18-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">18.</div><div>Gilbert, L.A. ∙ Larson, M.H. ∙ Morsut, L. ...</div><div><strong>CRISPR-mediated modular RNA-guided regulation of transcription in eukaryotes</strong></div><div><em>Cell.</em> 2013; <strong>154</strong>:442-451</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=10.1016%2Fj.cell.2013.06.044&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=10.1016%2Fj.cell.2013.06.044&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-84880571335" target="_blank" aria-label="Open Scopus in new tab">Scopus (2710)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23849981/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2013.06.044&amp;pmid=23849981" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>N/A</td></tr><tr><td>Human (female): K562 expressing dCas9–BFP–KRAB (Addgene #46911)</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref19-2" href="#" role="doc-biblioref" data-xml-rid="bib19" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref19-2" aria-controls="sref19-2">Gilbert et al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref19-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">19.</div><div>Gilbert, L.A. ∙ Horlbeck, M.A. ∙ Adamson, B. ...</div><div><strong>Genome-Scale CRISPR-Mediated Control of Gene Repression and Activation</strong></div><div><em>Cell.</em> 2014; <strong>159</strong>:647-661</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=10.1016%2Fj.cell.2014.09.029&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=10.1016%2Fj.cell.2014.09.029&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-84908352138" target="_blank" aria-label="Open Scopus in new tab">Scopus (1883)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25307932/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2014.09.029&amp;pmid=25307932" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>N/A</td></tr><tr><td>Human (female): HeLa</td><td>ATCC</td><td>Cat#CCL-2</td></tr><tr><td>Human: HAP1 parental control</td><td>Horizon Discovery</td><td>Cat#C631</td></tr><tr><td>Human: HAP1 <i>ΔMSH2</i></td><td>Horizon Discovery</td><td>Cat#HZGHC024799c006</td></tr><tr><td>Human: HAP1 <i>ΔMLH1</i></td><td>Horizon Discovery</td><td>Cat#HZGHC000343c021</td></tr><tr><td>Human (male): HCT116</td><td>ATCC</td><td>Cat#CCL-247</td></tr><tr><td>Mouse (male): N2A</td><td>ATCC</td><td>Cat#CCL-131</td></tr><tr><td>Human (female): K562</td><td>ATCC</td><td>Cat#CCL-243</td></tr><tr><td>Human (female): U2OS</td><td>ATCC</td><td>Cat#HTB-96</td></tr><tr class="rowsep"><td>Human (female): <i>CDKL5</i> deficiency disorder patient-derived induced pluripotent stem cell</td><td>Coriell Institute</td><td>Cat#OR00007</td></tr><tr class="rowsep"><td colspan="3"><b>Oligonucleotides</b></td></tr><tr><td>ON-TARGETplus Human MSH2 siRNA</td><td>Horizon Discovery</td><td>Cat#L-003909-00-0005</td></tr><tr><td>ON-TARGETplus Human MSH6 siRNA</td><td>Horizon Discovery</td><td>Cat#L-019287-00-0005</td></tr><tr><td>ON-TARGETplus Human MLH1 siRNA</td><td>Horizon Discovery</td><td>Cat#L-003906-00-0005</td></tr><tr><td>ON-TARGETplus Human PMS2 siRNA</td><td>Horizon Discovery</td><td>Cat#L-010032-00-0005</td></tr><tr><td>ON-TARGETplus Non-targeting Control Pool siRNA</td><td>Horizon Discovery</td><td>Cat#D-001810-10-05</td></tr><tr><td><i>CDKL5</i> c.1412delA correction with silent edit epegRNA: mG<sup>∗</sup>mA<sup>∗</sup>mG<sup>∗</sup>rGrGrArCr<br>UrCrCrUrArGrArGrGrArCrUrGrGrUrUrUr<br>UrArGrArGrCrUrArGrArArArUrArGrCrArAr<br>GrUrUrArArArArUrArArGrGrCrUrArGrUrCr<br>CrGrUrUrArUrCrArArCrUrUrGrArArArArAr<br>GrUrGrGrCrArCrCrGrArGrUrCrGrGrUrGrCr<br>ArUrArUrUrGrArCrArCrArArUrUrCrCrCrCr<br>ArArUrCrCrUrCrUrArGrGrArGrUrCrArArAr<br>ArArCrArCrGrUrCrArGrGrGrUrCrArGrGrAr<br>GrCrCrCrCrCrCrCrCrCrUrGrCrArCrCrCrAr<br>GrGrArArArArCrCrCrUrCrArArArGrUrCrGr<br>GrGrGrGrGrCrArA<sup>∗</sup>mC<sup>∗</sup>mC<sup>∗</sup>mC</td><td>Integrated DNA Technologies</td><td>N/A</td></tr><tr><td><i>CDKL5</i> +85 nick sgRNA: mG<sup>∗</sup>mC<sup>∗</sup>mA<sup>∗</sup>rGr<br>ArArCrCrGrCrCrArCrUrCrArUrUrCrArGr<br>UrUrUrUrArGrArGrCrUrArGrArArArUrAr<br>GrCrArArGrUrUrArArArArUrArArGrGrCr<br>UrArGrUrCrCrGrUrUrArUrCrArArCrUr<br>UrGrArArArArArGrUrGrGrCrArCrCrGr<br>ArGrUrCrGrGrUrGrCrUmU<sup>∗</sup>mU<sup>∗</sup>mU</td><td>Synthego Corporation</td><td>N/A</td></tr><tr><td><i>CDKL5</i> −3 nick sgRNA (for PE3b/PE5b): mA<sup>∗</sup>mC<sup>∗</sup>mA<sup>∗</sup>rCrArArUrUrCrCrCrCrArArUr<br>CrCrUrCrUrGrUrUrUrUrArGrArGrCrUrAr<br>GrArArArUrArGrCrArArGrUrUrArArArAr<br>UrArArGrGrCrUrArGrUrCrCrGrUrUrAr<br>UrCrArArCrUrUrGrArArArArArGrUrGrGr<br>CrArCrCrGrArGrUrCrGrGrUrGrCrUmU<sup>∗</sup><br>mU<sup>∗</sup>mU</td><td>Synthego Corporation</td><td>N/A</td></tr><tr><td><i>PRNP</i> G127V install pegRNA: mG<sup>∗</sup>mC<sup>∗</sup><br>mA<sup>∗</sup>rGrUrGrGrUrGrGrGrGrGrGrCrCrUr<br>UrGrGrGrUrUrUrUrArGrArGrCrUrArGr<br>ArArArUrArGrCrArArGrUrUrArArArArUr<br>ArArGrGrCrUrArGrUrCrCrGrUrUrArUrCr<br>ArArCrUrUrGrArArArArArGrUrGrGrCrArCr<br>CrGrArGrUrCrGrGrUrGrCrArUrGrUrArGr<br>ArCrGrCrCrArArGrGrCrCrCrCrCrC<sup>∗</sup>mA<sup>∗</sup><br>mC<sup>∗</sup>mC</td><td>Integrated DNA Technologies</td><td>N/A</td></tr><tr><td><i>PRNP</i> +72 nick sgRNA: mG<sup>∗</sup>mC<sup>∗</sup>mA<sup>∗</sup>rUrGr<br>UrUrUrUrCrArCrGrArUrArGrUrArArGrUrUr<br>UrUrArGrArGrCrUrArGrArArArUrArGrCrAr<br>ArGrUrUrArArArArUrArArGrGrCrUrArGrUr<br>CrCrGrUrUrArUrCrArArCrUrUrGrArArArAr<br>ArGrUrGrGrCrArCrCrGrArGrUrCrGrGmUr<br>GrCrUmU<sup>∗</sup>mU<sup>∗</sup>mU</td><td>Synthego Corporation</td><td>N/A</td></tr><tr><td><i>FANCF</i> +5 G to T pegRNA: mG<sup>∗</sup>mG<sup>∗</sup><br>mA<sup>∗</sup>rArUrCrCrCrUrUrCrUrGrCrArGrCr<br>ArCrCrGrUrUrUrUrArGrArGrCrUrArGrAr<br>ArArUrArGrCrArArGrUrUrArArArArUrArAr<br>GrGrCrUrArGrUrCrCrGrUrUrArUrCrArAr<br>CrUrUrGrArArArArArGrUrGrGrCrArCrCr<br>GrArGrUrCrGrGrUrGrCrGrGrArArArArGr<br>CrGrArUrCrArArGrGrUrGrCrUrGrCrArGr<br>ArArG<sup>∗</sup>mG<sup>∗</sup>mG<sup>∗</sup>mA</td><td>Agilent Technologies</td><td>N/A</td></tr><tr><td><i>FANCF</i> +48 nick sgRNA: mG<sup>∗</sup>mG<sup>∗</sup><br>mG<sup>∗</sup>rGrUrCrCrCrArGrGrUrGrCrUrGr<br>ArCrGrUrGrUrUrUrUrArGrArGrCrUr<br>ArGrArArArUrArGrCrArArGrUrUrArAr<br>ArArUrArArGrGrCrUrArGrUrCrCrGr<br>UrUrArUrCrArArCrUrUrGrArArArAr<br>ArGrUrGrGrCrArCrCrGrArGrUrCrGr<br>GrUrGrCrUmU<sup>∗</sup>mU<sup>∗</sup>mU</td><td>Synthego Corporation</td><td>N/A</td></tr><tr><td><i>RNF2</i> +1 T insertion pegRNA: mG<sup>∗</sup>mU<sup>∗</sup><br>mC<sup>∗</sup>rArUrCrUrUrArGrUrCrArUrUrArCr<br>CrUrGrGrUrUrUrUrArGrArGrCrUrArGr<br>ArArArUrArGrCrArArGrUrUrArArArArUr<br>ArArGrGrCrUrArGrUrCrCrGrUrUrArUr<br>CrArArCrUrUrGrArArArArArGrUrGrGr<br>CrArCrCrGrArGrUrCrGrGrUrGrCrArAr<br>CrGrArArCrArCrCrUrCrArGrArGrUrAr<br>ArUrGrArCrUrArArG<sup>∗</sup>mA<sup>∗</sup>mU<sup>∗</sup>mG</td><td>Integrated DNA Technologies</td><td>N/A</td></tr><tr><td><i>RNF2</i> +41 nick sgRNA: mU<sup>∗</sup>mC<sup>∗</sup>mA<sup>∗</sup><br>rArCrCrArUrUrArArGrCrArArArArCrAr<br>UrGrUrUrUrUrArGrArGrCrUrArGrArAr<br>ArUrArGrCrArArGrUrUrArArArArUrAr<br>ArGrGrCrUrArGrUrCrCrGrUrUrArUr<br>CrArArCrUrUrGrArArArArArGrUrGr<br>GrCrArCrCrGrArGrUrCrGrGrUrGrCrU<br>mU<sup>∗</sup>mU<sup>∗</sup>mU</td><td>Synthego Corporation</td><td>N/A</td></tr><tr><td><i>CXCR4</i> P191A install pegRNA: mC<sup>∗</sup><br>mA<sup>∗</sup>mA<sup>∗</sup>rCrCrArCrCrCrArCrArArGr<br>UrCrArUrUrGrGrUrUrUrUrArGrArGr<br>CrUrArGrArArArUrArGrCrArArGrUr<br>UrArArArArUrArArGrGrCrUrArGrUr<br>CrCrGrUrUrArUrCrArArCrUrUrGrAr<br>ArArArArGrUrGrGrCrArCrCrGrArGr<br>UrCrGrGrUrGrCrUrGrArCrCrGrCrUr<br>UrCrUrArCrGrCrCrArArUrGrArCrUr<br>UrGrUrGrGrGrU<sup>∗</sup>mG<sup>∗</sup>mG<sup>∗</sup>mU</td><td>Integrated DNA Technologies</td><td>N/A</td></tr><tr><td><i>CXCR4</i> +43 nick sgRNA: mC<sup>∗</sup>mA<sup>∗</sup>mU<sup>∗</sup>rCrUr<br>UrUrGrCrCrArArCrGrUrCrArGrUrGrGrUrUr<br>UrUrArGrArGrCrUrArGrArArArUrArGrCrAr<br>ArGrUrUrArArArArUrArArGrGrCrUrArGrUr<br>CrCrGrUrUrArUrCrArArCrUrUrGrArArArAr<br>ArGrUrGrGrCrArCrCrGrArGrUrCrGrGrUrGr<br>CrUmU<sup>∗</sup>mU<sup>∗</sup>mU</td><td>Synthego Corporation</td><td>N/A</td></tr><tr><td><i>IL2RB</i> H134D Y135F install pegRNA: mC<sup>∗</sup><br>mC<sup>∗</sup>mA<sup>∗</sup>rGrGrUrGrUrCrUrUrUrCrArArArGr<br>UrArGrGrUrUrUrUrArGrArGrCrUrArGrArAr<br>ArUrArGrCrArArGrUrUrArArArArUrArArGr<br>GrCrUrArGrUrCrCrGrUrUrArUrCrArArCrUr<br>UrGrArArArArArGrUrGrGrCrArCrCrGrArGr<br>UrCrGrGrUrGrCrUrCrCrCrArArGrCrCrUrCr<br>CrGrArCrUrUrCrUrUrUrGrArArArGrA<sup>∗</sup>mC<sup>∗</sup><br>mA<sup>∗</sup>mC</td><td>Integrated DNA Technologies</td><td>N/A</td></tr><tr class="rowsep"><td><i>IL2RB</i> +55 nick sgRNA: mC<sup>∗</sup>mU<sup>∗</sup>mC<sup>∗</sup>rCr<br>CrUrCrCrArArGrUrUrGrUrCrCrArCrGrGrUr<br>UrUrUrArGrArGrCrUrArGrArArArUrArGrCr<br>ArArGrUrUrArArArArUrArArGrGrCrUrArGr<br>UrCrCrGrUrUrArUrCrArArCrUrUrGrArArAr<br>ArArGrUrGrGrCrArCrCrGrArGrUrCrGrGrUr<br>GrCrUmU<sup>∗</sup>mU<sup>∗</sup>mU</td><td>Synthego Corporation</td><td>N/A</td></tr><tr class="rowsep"><td colspan="3"><b>Recombinant DNA</b></td></tr><tr><td>pCMV–PE2</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref3-12" href="#" role="doc-biblioref" data-xml-rid="bib3" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref3-12" aria-controls="sref3-12">Anzalone et al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref3-12"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">3.</div><div>Anzalone, A.V. ∙ Randolph, P.B. ∙ Davis, J.R. ...</div><div><strong>Search-and-replace genome editing without double-strand breaks or donor DNA</strong></div><div><em>Nature.</em> 2019; <strong>576</strong>:149-157</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-1711-4" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-85074571375" target="_blank" aria-label="Open Scopus in new tab">Scopus (2574)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31634902/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-1711-4&amp;pmid=31634902" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>132775</td></tr><tr><td>pMD2.G</td><td>Addgene</td><td>12259</td></tr><tr><td>psPAX</td><td>Addgene</td><td>12260</td></tr><tr><td>pLX_311-Cas9</td><td>Addgene</td><td>96924</td></tr><tr><td>pCMV–SaPE2</td><td>Addgene</td><td>174817</td></tr><tr><td>pCMV–SaPE2–P2A–BSD</td><td>Addgene</td><td>174818</td></tr><tr><td>pCMV–PE2–P2A–BSD</td><td>Addgene</td><td>174819</td></tr><tr><td>pCMV–PEmax</td><td>Addgene</td><td>174820</td></tr><tr><td>pCMV–PEmax–P2A–BSD</td><td>Addgene</td><td>174821</td></tr><tr><td>pEF1α–hMLH1dn (original codon)</td><td>Addgene</td><td>174823</td></tr><tr><td>pEF1α–hMLH1dn (codon opt.)</td><td>Addgene</td><td>174824</td></tr><tr><td>pEF1α–mMLH1dn (codon opt.)</td><td>Addgene</td><td>174825</td></tr><tr><td>pEF1α–hMLH1NTD–NLS (codon opt.)</td><td>Addgene</td><td>174826</td></tr><tr><td>pCMV–PE2–P2A–MLH1dn (codon opt.)</td><td>Addgene</td><td>174827</td></tr><tr><td>pCMV–PEmax–P2A–MLH1dn (codon opt.)</td><td>Addgene</td><td>174828</td></tr><tr class="rowsep"><td>pPC1000 (Repair-seq sgRNA–prime edit site)</td><td>Addgene</td><td>174829</td></tr><tr class="rowsep"><td colspan="3"><b>Software and algorithms</b></td></tr><tr><td>CRISPResso2</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref11-1" href="#" role="doc-biblioref" data-xml-rid="bib11" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref11-1" aria-controls="sref11-1">Clement et al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref11-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">11.</div><div>Clement, K. ∙ Rees, H. ∙ Canver, M.C. ...</div><div><strong>CRISPResso2 provides accurate and rapid genome editing sequence analysis</strong></div><div><em>Nat. Biotechnol.</em> 2019; <strong>37</strong>:224-226</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41587-019-0032-3" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-85062070478" target="_blank" aria-label="Open Scopus in new tab">Scopus (740)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30809026/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41587-019-0032-3&amp;pmid=30809026" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0020" href="https://github.com/pinellolab/CRISPResso2" target="_blank">https://github.com/pinellolab/CRISPResso2</a></td></tr><tr><td>Prism</td><td>GraphPad</td><td><a id="intref0025" href="https://www.graphpad.com" target="_blank">https://www.graphpad.com</a></td></tr><tr><td>Repair-seq processing software</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref25-3" href="#" role="doc-biblioref" data-xml-rid="bib25" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref25-3" aria-controls="sref25-3">Hussmann et al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref25-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">25.</div><div>Hussmann, J.A. ∙ Ling, J. ∙ Ravisankar, P. ...</div><div><strong>Mapping the genetic landscape of DNA double-strand break repair</strong></div><div><em>Cell.</em> 2021;</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=10.1016%2Fj.cell.2021.10.002&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=10.1016%2Fj.cell.2021.10.002&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-85117855979" target="_blank" aria-label="Open Scopus in new tab">Scopus (80)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/34653350/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2021.10.002&amp;pmid=34653350" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td>https://github.com/jeffhussmann/repair-seq (DOI: <a id="PC_linkfTpzgwDirG" href="https://doi.org/10.5281/zenodo.5534778" target="_blank">https://doi.org/10.5281/zenodo.5534778</a>)</td></tr></tbody></table></div><figcaption><div class="core-table-tools"><ul><li><a class="core-table-show-action" href="/action/showFullTableHTML?isHtml=true&amp;tableId=undtbl1&amp;pii=S0092-8674%2821%2901065-5" target="_blank">Open table in a new tab</a></li></ul></div></figcaption></figure></div></div></section><section id="sec-5-2"><h3>Resource availability</h3><section id="sec-5-2-1"><h4>Lead contact</h4><div id="p0250" role="paragraph">Please direct requests for resources and reagents to Lead Contact David R. Liu (<a id="intref0035" href="mailto:drliu@fas.harvard.edu">drliu@fas.harvard.edu</a>).</div></section><section id="sec-5-2-2"><h4>Materials availability</h4><div id="p0255" role="paragraph">Plasmids generated in this study are available from Addgene (additional details provided in the <a id="crosref1365" href="#sec-5-1">Key resources table</a>).</div></section></section><section id="sec-5-3"><h3>Experimental model and subject details</h3><section id="sec-5-3-1"><h4>Culture conditions for immortalized cell lines</h4><div id="p0260" role="paragraph">HEK293T, HeLa, HCT116, and N2A cells were cultured in Dulbecco’s Modified Eagle Medium (DMEM) plus GlutaMAX (Thermo Fisher Scientific) supplemented with 10% fetal bovine serum (FBS) (Thermo Fisher Scientific). HeLa dCas9–BFP–KRAB cells were cultured in DMEM plus GlutaMAX supplemented with 10% FBS, 100&nbsp;U mL<sup>-1</sup> penicillin, and 100&nbsp;μg mL<sup>-1</sup> streptomycin (Thermo Fisher Scientific). K562 dCas9–BFP–KRAB and K562 cells were cultured in Roswell Park Memorial Institute (RPMI) 1640 medium (Thermo Fisher Scientific) supplemented with 10% FBS, 100 U mL<sup>-1</sup> penicillin, 100 μg mL<sup>-1</sup> streptomycin (Thermo Fisher Scientific), and 292 μg mL<sup>-1</sup> L-Glutamine (Corning). All HAP1 cell types were cultured in Iscove’s Modified Dulbecco’s Medium (IMDM) plus GlutaMAX (Thermo Fisher Scientific) supplemented with 10% FBS. U2OS cells were cultured in McCoy’s 5A medium (GIBCO) supplemented with 10% FBS, 100&nbsp;U mL<sup>-1</sup> penicillin, and 100&nbsp;μg mL<sup>-1</sup> streptomycin (Thermo Fisher Scientific). HeLa dCas9–BFP–KRAB and K562 dCas9–BFP–KRAB cell lines were verified by short tandem repeat marker testing. All cell types were passaged every 2–3&nbsp;days, maintained below 80% confluency, cultured at 37°C with 5% CO<sub>2</sub>, and tested negative for mycoplasma.</div></section><section id="sec-5-3-2"><h4>Isolation of primary human T&nbsp;cells</h4><div id="p0265" role="paragraph">Peripheral blood mononuclear cells (PBMCs) were isolated from the buffy coat of healthy donors (Memorial Blood Centers in St. Paul, Minnesota) by density centrifugation using Lymphoprep density gradient medium (STEMCELL Technologies) and SepMate tubes (STEMCELL Technologies). T&nbsp;cells were isolated from PBMCs using the EasySep Human T Cell Isolation Kit (STEMCELL Technologies).</div></section><section id="sec-5-3-3"><h4>Culture conditions for human patient-derived induced pluripotent stem cells</h4><div id="p0270" role="paragraph">All iPSC culturing work was performed by staff at the Human Neuron Core at Boston Children’s Hospital following institutional guidelines and under institutional approvals (IRB#: P00016119). A clonal iPS cell line, MAN0855-01 #A (Coriell Institute #OR00007), was expanded from a female <i>CDKL5</i> deficiency disorder patient carrying a heterozygous <i>CDKL5</i> c.1412delA p.D471fs mutation on the X chromosome (<span class="dropBlock reference-citations"><a id="body-ref-sref10-2" href="#" role="doc-biblioref" data-xml-rid="bib10" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref10-2" aria-controls="sref10-2">Chen et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref10-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">10.</div><div>Chen, P.-F. ∙ Chen, T. ∙ Forman, T.E. ...</div><div><strong>Generation and characterization of human induced pluripotent stem cells (iPSCs) from three male and three female patients with CDKL5 Deficiency Disorder (CDD)</strong></div><div><em>Stem Cell Res. (Amst.).</em> 2021; <strong>53</strong>:102276</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.scr.2021.102276" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-85102606970" target="_blank" aria-label="Open Scopus in new tab">Scopus (6)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33714067/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.scr.2021.102276&amp;pmid=33714067" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). MAN0855-01 #A was previously verified to express the mutant <i>CDKL5</i> transcript by Sanger sequencing of cDNA. The MAN0855-01 #A iPS cell line was cultured in StemFlex medium (Thermo Fisher Scientific) on Geltrex (Thermo Fisher Scientific) diluted 1:50 in DMEM/F12 (Thermo Fisher Scientific) and coated according to the manufacturer’s protocol. For regular maintenance, iPS cell colonies were clump-passaged using Gentle Cell Dissociation Reagent (STEMCELL Technologies) at 80% confluency every 5–7&nbsp;days.</div></section></section><section id="sec-5-4"><h3>Method details</h3><section id="sec-5-4-1"><h4>General methods and molecular cloning</h4><div id="p0275" role="paragraph">Lentiviral transfer plasmids and plasmids for mammalian expression of prime editors and other proteins were cloned using uracil excision (USER) assembly (<span class="dropBlock reference-citations"><a id="body-ref-sref9" href="#" role="doc-biblioref" data-xml-rid="bib9" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref9" aria-controls="sref9">Cavaleiro et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref9"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">9.</div><div>Cavaleiro, A.M. ∙ Kim, S.H. ∙ Seppälä, S. ...</div><div><strong>Accurate DNA Assembly and Genome Engineering with Optimized Uracil Excision Cloning</strong></div><div><em>ACS Synth. Biol.</em> 2015; <strong>4</strong>:1042-1046</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/acssynbio.5b00113" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-84942045165" target="_blank" aria-label="Open Scopus in new tab">Scopus (67)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26263045/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Facssynbio.5b00113&amp;pmid=26263045" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Briefly, DNA fragments were amplified with deoxyuracil-containing primers (Integrated DNA Technologies) using the uracil tolerant Phusion U Green Multiplex PCR Master Mix (Thermo Fisher Scientific). Deoxyuracil-incorporated DNA fragments were assembled with USER enzyme (New England BioLabs) and DpnI (New England BioLabs) according to the manufacturer’s protocol using junctions with a melting temperature of 42–60°C, followed by transformation into cells. All prime editor constructs were cloned into the pCMV–PE2 vector backbone (<span class="dropBlock reference-citations"><a id="body-ref-sref3-13" href="#" role="doc-biblioref" data-xml-rid="bib3" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref3-13" aria-controls="sref3-13">Anzalone et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref3-13"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">3.</div><div>Anzalone, A.V. ∙ Randolph, P.B. ∙ Davis, J.R. ...</div><div><strong>Search-and-replace genome editing without double-strand breaks or donor DNA</strong></div><div><em>Nature.</em> 2019; <strong>576</strong>:149-157</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-1711-4" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-85074571375" target="_blank" aria-label="Open Scopus in new tab">Scopus (2574)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31634902/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-1711-4&amp;pmid=31634902" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) (Addgene #132775) under constitutive expression from a CMV promoter. All prime editor constructs also contained the following mutations within the MMLV RT: D200N, T306K, W313F, T330P, and L603W. All DNA repair protein and RFP expression constructs were cloned into vectors under constitutive expression from an EF1α promoter. Human MSH2, MSH6, PMS2, and MLH1 sequences were subcloned from the plasmids pFB1_hMSH2 (Addgene #129423), pFB1_hMSH6 (Addgene #129424), pFB1_PMS2 (Addgene #129425), and pFB1_MLH1 (Addgene #129426) (<span class="dropBlock reference-citations"><a id="body-ref-sref15" href="#" role="doc-biblioref" data-xml-rid="bib15" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref15" aria-controls="sref15">Geng et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref15"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">15.</div><div>Geng, H. ∙ Du, C. ∙ Chen, S. ...</div><div><strong>In&nbsp;vitro studies of DNA mismatch repair proteins</strong></div><div><em>Anal. Biochem.</em> 2011; <strong>413</strong>:179-184</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.ab.2011.02.017" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-79955466020" target="_blank" aria-label="Open Scopus in new tab">Scopus (28)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21329650/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.ab.2011.02.017&amp;pmid=21329650" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Human CDKN1A sequence was subcloned from the plasmid Flag p21 WT (Addgene #16240) (<span class="dropBlock reference-citations"><a id="body-ref-sref70" href="#" role="doc-biblioref" data-xml-rid="bib70" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref70" aria-controls="sref70">Zhou et&nbsp;al., 2001</a><div class="dropBlock__holder" data-db-target-of="sref70"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">72.</div><div>Zhou, B.P. ∙ Liao, Y. ∙ Xia, W. ...</div><div><strong>Cytoplasmic localization of p21Cip1/WAF1 by Akt-induced phosphorylation in HER-2/neu-overexpressing cells</strong></div><div><em>Nat. Cell Biol.</em> 2001; <strong>3</strong>:245-252</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/35060032" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-0034745353" target="_blank" aria-label="Open Scopus in new tab">Scopus (950)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11231573/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2F35060032&amp;pmid=11231573" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Codon-optimized MLH1 sequences for human cell and mouse cell expression were designed using GenSmart Codon Optimization (Genscript) and ordered as gBlock gene fragments (Integrated DNA Technologies).</div><div id="p0280" role="paragraph">Plasmids for mammalian expression of pegRNAs or sgRNAs were cloned using Golden Gate assembly (<span class="dropBlock reference-citations"><a id="body-ref-sref13" href="#" role="doc-biblioref" data-xml-rid="bib13" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref13" aria-controls="sref13">Engler et&nbsp;al., 2008</a><div class="dropBlock__holder" data-db-target-of="sref13"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">13.</div><div>Engler, C. ∙ Kandzia, R. ∙ Marillonnet, S.</div><div><strong>A one pot, one step, precision cloning method with high throughput capability</strong></div><div><em>PLoS ONE.</em> 2008; <strong>3</strong>:e3647</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pone.0003647" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_13_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-56649114274" target="_blank" aria-label="Open Scopus in new tab">Scopus (1561)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/18985154/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pone.0003647&amp;pmid=18985154" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) as previously described (<span class="dropBlock reference-citations"><a id="body-ref-sref3-14" href="#" role="doc-biblioref" data-xml-rid="bib3" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref3-14" aria-controls="sref3-14">Anzalone et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref3-14"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">3.</div><div>Anzalone, A.V. ∙ Randolph, P.B. ∙ Davis, J.R. ...</div><div><strong>Search-and-replace genome editing without double-strand breaks or donor DNA</strong></div><div><em>Nature.</em> 2019; <strong>576</strong>:149-157</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-1711-4" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-85074571375" target="_blank" aria-label="Open Scopus in new tab">Scopus (2574)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31634902/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-1711-4&amp;pmid=31634902" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Briefly, a guide RNA vector backbone for human U6 promoter expression was digested overnight with BsaI-HFv2 (New England BioLabs) according to the manufacturer’s protocol, and linearized product was purified by electrophoresis with a 1% agarose gel using the QIAquick Gel Extraction Kit (QIAGEN). Oligonucleotides (Integrated DNA Technologies) for the spacer sequence, guide RNA scaffold, and 3′ extension were annealed, assembled with linearized U6 backbone DNA using T4 DNA ligase (New England BioLabs) according to the manufacturer’s protocol, and transformed into cells. Only guide RNA scaffold oligonucleotides were purchased with 5′ phosphorylation modifications. Some plasmids encoding pegRNAs and epegRNAs were synthesized by Twist Bioscience. A list of pegRNAs and nicking sgRNAs used in this work is provided in <a id="crosref1405" href="#mmc3">Table S3</a>.</div><div id="p0285" role="paragraph">Unless otherwise noted, assembled plasmids were transformed into One Shot Mach1 cells (Thermo Fisher Scientific) and grown on Luria-Bertani (LB) or 2&nbsp;× YT agar with 50&nbsp;μg ml<sup>-1</sup> carbenicillin (Gold Biotechnology). Plasmid sequences were fully verified by Sanger sequencing (Quintara Biosciences), and bacteria containing verified plasmids were grown in 2&nbsp;× YT medium with 100&nbsp;μg ml<sup>-1</sup> carbenicillin (Gold Biotechnology). Plasmid DNA were isolated using the QIAGEN Plasmid Plus Midi Kit or QIAGEN Plasmid Plus Maxi Kit with endotoxin removal and 2&nbsp;× the recommended amount of RNase A in Buffer P1. Some pegRNA and sgRNA plasmid DNA were isolated with the PureYield Plasmid Miniprep System (Promega Corporation) with endotoxin removal. Plasmid DNA purified using the PureYield Plasmid Miniprep System were only used for HEK293T and HeLa cell transfections. All plasmids were eluted in nuclease-free water (QIAGEN) and quantified using a NanoDrop One UV-Vis spectrophotometer (Thermo Fisher Scientific).</div></section><section id="sec-5-4-2"><h4>Lentivirus production for generating cell lines</h4><div id="p0290" role="paragraph">To package lentivirus for generating stable cell lines, HEK293T cells were seeded on 6-well plates (Corning) at 7.5&nbsp;× 10<sup>5</sup> cells per well in DMEM supplemented with 10% FBS. At 60% confluency 16&nbsp;h after seeding, cells were transfected with 12&nbsp;μL Lipofectamine 2000 (Thermo Fisher Scientific) according to the manufacturer’s protocol and 1.33&nbsp;μg lentiviral transfer plasmid, 0.67&nbsp;μg pMD2.G (Addgene #12259), and 1&nbsp;μg psPAX2 (Addgene #12260). 6&nbsp;h after transfection, media was exchanged with DMEM supplemented with 10% FBS. 48&nbsp;h after transfection, viral supernatant was centrifuged at 3000 <i>g</i> for 15&nbsp;min to remove cellular debris, filtered through a 0.45&nbsp;μm PVDF filter (Corning), and stored at −80°C.</div></section><section id="sec-5-4-3"><h4>Construction of HEK293T cell line with integrated modified HBB sequence</h4><div id="p0295" role="paragraph">A lentiviral transfer plasmid (pAX198) was previously designed to contain the coding sequence of human <i>HBB</i> and a PuroR–T2A–BFP marker under expression from an EF1α promoter (pEF1α) (<span class="dropBlock reference-citations"><a id="body-ref-sref25-4" href="#" role="doc-biblioref" data-xml-rid="bib25" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref25-4" aria-controls="sref25-4">Hussmann et al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref25-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">25.</div><div>Hussmann, J.A. ∙ Ling, J. ∙ Ravisankar, P. ...</div><div><strong>Mapping the genetic landscape of DNA double-strand break repair</strong></div><div><em>Cell.</em> 2021;</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=10.1016%2Fj.cell.2021.10.002&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=10.1016%2Fj.cell.2021.10.002&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-85117855979" target="_blank" aria-label="Open Scopus in new tab">Scopus (80)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/34653350/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2021.10.002&amp;pmid=34653350" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Lentivirus carrying this cassette were produced from HEK293T cells as described above. To stably integrate the <i>HBB</i> sequence, 6&nbsp;× 10<sup>5</sup> HEK293T cells were infected with lentivirus in 6-well plates (Corning) with DMEM supplemented with 10% FBS and 10&nbsp;μg mL<sup>-1</sup> polybrene (Sigma-Aldrich). BFP fluorescence was monitored daily using a CytoFLEX S Flow Cytometer (Beckman Coulter) to ensure an MOI of 0.1 and low copy number integration. Following infection for 2&nbsp;days, HEK293T cells were selected in 2&nbsp;μg μL<sup>-1</sup> puromycin (Thermo Fisher Scientific) for 3&nbsp;days and stable transduction was confirmed by measuring BFP fluorescence. The resulting cell line was used to optimize pegRNAs for prime editing. To measure editing, a 214-bp amplicon of the integrated <i>HBB</i> region was PCR amplified. Amplification of the endogenous <i>HBB</i> locus with these primers yields a differently sized 1064-bp amplicon.</div></section><section id="sec-5-4-4"><h4>Design and construction of HeLa cell line with CRISPRi sgRNA and prime edit target</h4><div id="p0300" role="paragraph">The lentiviral transfer plasmid backbone for prime editing Repair-seq screens (pPC1000) was designed and cloned to contain a specific prime edit site and express a control <i>S.&nbsp;pyogenes</i> sgRNA for CRISPRi (<a id="optbiws4GLVQe" href="#figs1">Figure S1</a>C). The prime edit site consisted of an <i>HBB</i> target for Sa-pegRNA flanked by two complementary-strand Sa-sgRNA targets derived from the <i>Saccharomyces cerevisiae</i> genome. These target sites were situated such that SaPE2–sgRNA complexes nick 50-bp upstream and 50-bp downstream of the nick formed by SaPE2–pegRNA. This 234-bp edit site was positioned adjacent to an <i>S.&nbsp;pyogenes</i> sgRNA expression cassette driven by a modified mouse U6 promoter such that an sgRNA and edit site could be amplified by PCR in the same 453-bp amplicon. The sgRNA expression cassette in pPC1000 encoded an EGFP-targeting control sgRNA (spacer, 5′–GACCAGGATGGGCACCACCC–3′) and an pEF1α–PuroR–T2A–BFP selection marker.</div><div id="p0305" role="paragraph">Lentivirus encoding the pPC1000 cassette were produced from HEK293T cells as described above. For stable integration, 2.5&nbsp;× 10<sup>5</sup> HeLa dCas9–BFP–KRAB cells were infected with pPC1000 lentivirus in 6-well plates (Corning) with DMEM supplemented with 10% FBS and 10&nbsp;μg mL<sup>-1</sup> polybrene (Sigma-Aldrich). BFP fluorescence was monitored with a CytoFLEX S Flow Cytometer (Beckman Coulter) to ensure an MOI of 0.1 and low copy integration. Following 2&nbsp;days of infection, cells were selected in 2&nbsp;μg μL<sup>-1</sup> puromycin (Thermo Fisher Scientific) for 3&nbsp;days and stable transduction was confirmed by measuring BFP fluorescence. The resulting HeLa dCas9–BFP–KRAB cell line with integrated pPC1000 sequence was used to pilot prime editing conditions, Sa-pegRNAs, and Sa-sgRNAs for Repair-seq screens.</div></section><section id="sec-5-4-5"><h4>Transfection of HEK293T, HeLa, HCT116, and N2A cells</h4><div id="p0310" role="paragraph">Unless otherwise noted, HEK293T cells were seeded on 96-well plates (Corning) at 1.6–1.8&nbsp;× 10<sup>4</sup> cells per well in DMEM plus GlutaMAX supplemented with 10% FBS. Between 16 and 24&nbsp;h after seeding, cells were transfected at 60%–80% confluency with 0.5&nbsp;μL Lipofectamine 2000 (Thermo Fisher Scientific) according to the manufacturer’s protocol and 200&nbsp;ng prime editor plasmid, 66&nbsp;ng pegRNA plasmid, 22&nbsp;ng sgRNA plasmid (where indicated), and 100&nbsp;ng plasmid for RFP or MMR protein expression (where indicated).</div><div id="p0315" role="paragraph">For arrayed experiments, HeLa dCas9–BFP–KRAB and HeLa cells were seeded on 96-well plates (Corning) at 8&nbsp;× 10<sup>3</sup> cells per well in DMEM plus GlutaMAX supplemented with 10% FBS. Between 16 and 24&nbsp;h after seeding, cells were transfected at 60%–80% confluency with 0.3&nbsp;μL <i>Trans</i>IT-HeLa reagent (Mirus Bio) according to the manufacturer’s protocol and 56.25&nbsp;ng prime editor plasmid bearing a P2A–BlastR selection marker, 18.75&nbsp;ng pegRNA plasmid, 6.25 sgRNA plasmid (where indicated), and 28.1&nbsp;ng human codon-optimized MLH1dn plasmid (where indicated). 24&nbsp;h following transfection, 10&nbsp;ng μL<sup>-1</sup> blasticidin (Thermo Fisher Scientific) was added to each well to select for cells expressing prime editor.</div><div id="p0320" role="paragraph">HCT116 cells were seeded on 96-well plates (Corning) at 1.6&nbsp;× 10<sup>4</sup> cells per well in DMEM plus GlutaMAX supplemented with 10% FBS. Between 16 and 20&nbsp;h after seeding, cells were transfected at 60%–80% confluency with 0.5&nbsp;μL Lipofectamine 3000 plus 0.8&nbsp;μL P3000 reagent (Thermo Fisher Scientific) according to the manufacturer’s protocol and 200&nbsp;ng prime editor plasmid bearing a P2A–BlastR selection marker, 66&nbsp;ng pegRNA plasmid, 22&nbsp;ng sgRNA plasmid (where indicated), and 100&nbsp;ng MLH1dn plasmid (where indicated). The day after transfection, media was replaced with fresh DMEM plus GlutaMAX supplemented with 10% FBS and 10&nbsp;ng μL<sup>-1</sup> blasticidin (Thermo Fisher Scientific) to select for cells expressing prime editor.</div><div id="p0325" role="paragraph">N2A cells were seeded on 96-well plates (Corning) at 1.6&nbsp;× 10<sup>4</sup> cells per well in DMEM plus GlutaMAX supplemented with 10% FBS. Between 16 and 20&nbsp;h after seeding, cells were transfected at 60%–80% confluency with 0.5&nbsp;μL Lipofectamine 2000 (Thermo Fisher Scientific) according to the manufacturer’s protocol and 175&nbsp;ng prime editor plasmid, 50&nbsp;ng pegRNA plasmid, 20&nbsp;ng sgRNA plasmid (where indicated), and 87.5&nbsp;ng plasmid encoding human codon-optimized hMLH1dn or mouse codon-optimized mMLH1dn where indicated. Genomic DNA was extracted 72&nbsp;h following transfection.</div></section><section id="sec-5-4-6"><h4>Electroporation of HAP1, K562, and U2OS cells</h4><div id="p0330" role="paragraph">HAP1 cells were electroporated using the SE Cell Line 4D-Nucleofector X Kit S (Lonza) according to the manufacturer’s protocol with 4&nbsp;× 10<sup>5</sup> cells (program DZ-113), 300&nbsp;ng PE2–P2A–BSD, 100&nbsp;ng pegRNA plasmid, and 33&nbsp;ng sgRNA plasmid (where indicated). After electroporation, cells were cultured in 48-well plates (Corning) with IMDM plus GlutaMAX supplemented with 10% FBS. The day after electroporation, media was replaced with fresh IMDM plus GlutaMAX supplemented with 10% FBS and 10&nbsp;ng μL<sup>-1</sup> blasticidin (Thermo Fisher Scientific) to select for cells expressing prime editor.</div><div id="p0335" role="paragraph">K562 cells were electroporated using the SF Cell Line 4D-Nucleofector X Kit S (Lonza) according to the manufacturer’s protocol with 5&nbsp;× 10<sup>5</sup> cells (program FF-120), 800&nbsp;ng prime editor plasmid, 200&nbsp;ng pegRNA plasmid, 83&nbsp;ng sgRNA plasmid (where indicated), and 400&nbsp;ng MLH1dn plasmid (where indicated). After electroporation, cells were cultured in 6-well plates (Corning) with RPMI 1640 medium supplemented with 10% FBS and 292&nbsp;μg mL<sup>-1</sup> L-Glutamine (Corning).</div><div id="p0340" role="paragraph">U2OS cells were electroporated using the SE Cell Line 4D-Nucleofector X Kit S (Lonza) according to the manufacturer’s protocol with 2&nbsp;× 10<sup>5</sup> cells (program DN-100), 1600&nbsp;ng PE2 or PE2–P2A–MLH1dn plasmid, 400&nbsp;ng pegRNA plasmid, and 166&nbsp;ng sgRNA plasmid (where indicated). After electroporation, cells were cultured in 12-well or 24-well plates (Greiner Bio-One) with McCoy’s 5A medium supplemented with 10% FBS.</div></section><section id="sec-5-4-7"><h4>Genomic DNA extraction</h4><div id="p0345" role="paragraph">Unless otherwise noted, HEK293T, HeLa dCas9–BFP–KRAB, HeLa, HCT116, N2A, HAP1, K562, and U2OS cells were cultured for 72&nbsp;h after transfection or electroporation before genomic DNA was isolated. Cells were washed once with PBS (Thermo Fisher Scientific) and lysed with gDNA lysis buffer (10&nbsp;mM Tris-HCl, pH 8.0; 0.05% SDS; 800 units μL<sup>-1</sup> proteinase K (New England BioLabs)) at 37°C for 1.5–2 h, followed by enzyme inactivation at 80°C for 30&nbsp;min.</div></section><section id="sec-5-4-8"><h4>High-throughput amplicon sequencing of genomic DNA samples</h4><div id="p0350" role="paragraph">To assess gene editing, loci were amplified from genomic DNA samples via two rounds of PCR then deep sequenced. Briefly, an initial PCR step (PCR1) amplified the genomic sequence of interest using primers (Integrated DNA Technologies) containing Illumina forward and reverse adapters. Each 20&nbsp;μL PCR1 reaction was performed with 500&nbsp;nM of each primer, 0.8 to 1.0&nbsp;μL genomic DNA, 1&nbsp;× SYBR Green (Thermo Fisher Scientific), and 10&nbsp;μL Q5 High-Fidelity 2&nbsp;× Master Mix (New England BioLabs) on a CFX96 Touch Real-Time PCR Detection System (Bio-Rad Laboratories) with the following thermocycling conditions: 98°C for 2&nbsp;min, 29–31 cycles of [98°C for 10 s, 61°C for 20 s, and 72°C for 30 s], followed by 72°C for 2&nbsp;min. PCR1 reactions were monitored with SYBR Green fluorescence to avoid over-amplification. A list of primers used for PCR1 reactions is provided in <a id="crosref1415" href="#mmc4">Table S4</a>, and a list of PCR1 amplicon sequences is provided in <a id="crosref1420" href="#mmc5">Table S5</a>. The subsequent PCR step (PCR2) added unique i7 and i5 Illumina barcode combinations to both ends of the PCR1 DNA fragment to enable sample demultiplexing. Each 12.5&nbsp;μL PCR2 reaction was performed with 500&nbsp;nM of each barcoding primer, 0.5&nbsp;μL PCR1 product, and 6.25&nbsp;μL Phusion U Green Multiplex PCR Master Mix (Thermo Fisher Scientific) with the following thermocycling conditions: 98°C for 2&nbsp;min, 9 cycles of [98°C for 15 s, 61°C for 20 s, and 72°C for 30 s], followed by 72°C for 2&nbsp;min. PCR2 products were pooled by common amplicons, separated by electrophoresis on a 1% agarose gel, purified using the QIAquick Gel Extraction Kit (QIAGEN), and eluted in nuclease-free water. DNA amplicon libraries were quantified with a Qubit 3.0 Fluorometer (Thermo Fisher Scientific), then sequenced using the MiSeq Reagent Kit v2 or MiSeq Reagent Micro Kit v2 (Illumina), with 280–300 single-read cycles. A list of FASTQ sequencing files generated in this work is provided in <a id="crosref1425" href="#mmc3">Table S3</a>.</div></section><section id="sec-5-4-9"><h4>Quantification of amplicon sequencing data</h4><div id="p0355" role="paragraph">All arrayed prime editing experiments from <a id="crosref1430" href="#fig3">Figures 3</a>, <a id="crosref1435" href="#fig4">4</a>, <a id="crosref1440" href="#fig5">5</a>, <a id="crosref1445" href="#fig6">6</a>, <a id="crosref1450" href="#fig7">7</a>, <a id="crosref1455" href="#figs1">S1</a>, <a id="crosref1460" href="#figs3">S3</a>F, <a id="crosref1465" href="#figs4">S4</a>, <a id="crosref1470" href="#figs5">S5</a>, <a id="crosref1475" href="#figs6">S6</a>, and <a id="crosref1480" href="#figs7">S7</a> were analyzed as follows. Sequencing reads were demultiplexed using MiSeq Reporter (Illumina). Amplicon sequences (<a id="crosref1485" href="#mmc5">Table S5</a>) were aligned to a reference sequence with CRISPResso2 (<span class="dropBlock reference-citations"><a id="body-ref-sref11-2" href="#" role="doc-biblioref" data-xml-rid="bib11" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref11-2" aria-controls="sref11-2">Clement et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref11-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">11.</div><div>Clement, K. ∙ Rees, H. ∙ Canver, M.C. ...</div><div><strong>CRISPResso2 provides accurate and rapid genome editing sequence analysis</strong></div><div><em>Nat. Biotechnol.</em> 2019; <strong>37</strong>:224-226</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41587-019-0032-3" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-85062070478" target="_blank" aria-label="Open Scopus in new tab">Scopus (740)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30809026/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41587-019-0032-3&amp;pmid=30809026" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) in standard mode using the parameters “-q 30” and “-discard_indel_reads TRUE.” For each amplicon, the CRISPResso2 quantification window was positioned to include the entire sequence between pegRNA- and sgRNA-directed Cas9 cut sites, as well as an additional ≥ 10 nt beyond both cut sites. For each amplicon, the same quantification window was used for PE2, PE3, PE4, and PE5 conditions, regardless of whether a nicking sgRNA was transfected. All prime editing efficiencies describe percentage of (number of reads with the intended edit that do not contain indels)/(number of reads that align to the amplicon). Single-base substitution prime editing frequencies were quantified as: (frequency of intended base substitution in reference-aligned, non-discarded reads) × (number of reference-aligned, non-discarded reads)/(number of reference-aligned reads). For all other prime edits (insertion, deletion, contiguous substitutions, combinations of edits), CRISPResso2 was run in HDR mode with all the same parameters described above and using the intended editing outcome as the expected allele (-e). Frequencies for these edits were quantified as: (number of HDR-aligned reads)/(number of reference-aligned reads). All indel frequencies were quantified as: (number of indel-containing reads)/(number of reference-aligned reads).</div></section><section id="sec-5-4-10"><h4>CRISPRi library cloning and lentiviral library production</h4><div id="p0360" role="paragraph">An oligonucleotide library of CRISPRi sgRNAs (Q-15620&nbsp;= AX227) was designed to contain 60 non-targeting control sgRNAs and 1,513 sgRNAs that target 476 genes involved in DNA repair and associated processes (<span class="dropBlock reference-citations"><a id="body-ref-sref25-5" href="#" role="doc-biblioref" data-xml-rid="bib25" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref25-5" aria-controls="sref25-5">Hussmann et al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref25-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">25.</div><div>Hussmann, J.A. ∙ Ling, J. ∙ Ravisankar, P. ...</div><div><strong>Mapping the genetic landscape of DNA double-strand break repair</strong></div><div><em>Cell.</em> 2021;</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=10.1016%2Fj.cell.2021.10.002&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=10.1016%2Fj.cell.2021.10.002&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-85117855979" target="_blank" aria-label="Open Scopus in new tab">Scopus (80)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/34653350/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2021.10.002&amp;pmid=34653350" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). A list of targeted genes and sequences in the sgRNA library is provided in <a id="crosref1500" href="#mmc1">Table S1</a>. The oligonucleotide library was ordered from Twist Bioscience and sequences were amplified by PCR using Phusion High-Fidelity DNA Polymerase (Thermo Fisher Scientific) and purified with the NucleoSpin Gel and PCR Clean-up kit (Macherey-Nagel). The amplified sequences and the pPC1000 lentiviral screen vector containing the pre-validated prime edit site were digested with BstXI and BlpI restriction endonucleases (Thermo Fisher Scientific), ligated with T4 ligase (New England BioLabs), and transformed into MegaX DH10B electrocompetent cells (Thermo Fisher Scientific). The plasmid library was isolated from transformed cells using ZymoPURE II Plasmid Maxiprep Kit (Zymo Research), and the pooled library of plasmids was verified by PCR and sequencing on a MiSeq Reagent Kit v2 (Illumina).</div><div id="p0365" role="paragraph">To produce lentivirus with pPC1000 libraries, HEK293T cells were seeded in a 15&nbsp;cm dish with DMEM supplemented with 10% FBS. One day after seeding, cells were transfected using 60&nbsp;μL <i>Trans</i>IT-LT1 reagent (Mirus Bio) with 15&nbsp;μg pPC1000 plasmid library and 5&nbsp;μg packaging plasmids for expression of HIV-1 gag/pol, rev, tat, and VSV-G envelope protein. 24&nbsp;h after transfection, 40&nbsp;μL ViralBoost reagent (ALSTEM) was added to each 15&nbsp;cm dish. 48&nbsp;h after transfection, viral supernatant was collected, filtered through a 0.45&nbsp;μm PVDF filter (Corning), and stored at −80°C.</div></section><section id="sec-5-4-11"><h4>Repair-seq screens in HeLa cells</h4><div id="p0370" role="paragraph">PE2, PE3+50, PE3–50 Repair-seq screens were performed in duplicate in HeLa cells with integrated dCas9–BFP–KRAB (Addgene #46911) (<span class="dropBlock reference-citations"><a id="body-ref-sref18-4" href="#" role="doc-biblioref" data-xml-rid="bib18" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref18-4" aria-controls="sref18-4">Gilbert et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref18-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">18.</div><div>Gilbert, L.A. ∙ Larson, M.H. ∙ Morsut, L. ...</div><div><strong>CRISPR-mediated modular RNA-guided regulation of transcription in eukaryotes</strong></div><div><em>Cell.</em> 2013; <strong>154</strong>:442-451</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=10.1016%2Fj.cell.2013.06.044&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=10.1016%2Fj.cell.2013.06.044&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-84880571335" target="_blank" aria-label="Open Scopus in new tab">Scopus (2710)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23849981/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2013.06.044&amp;pmid=23849981" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). These HeLa CRISPRi cells were transduced with the lentiviral library at 0.1 MOI (10.8% BFP+) in DMEM supplemented with 10% FBS, 100&nbsp;U mL<sup>-1</sup> penicillin, 100&nbsp;μg mL<sup>-1</sup> streptomycin (Thermo Fisher Scientific), and 8&nbsp;μg mL<sup>-1</sup> polybrene (Sigma-Aldrich). 2&nbsp;days after infection, HeLa CRISPRi cells were treated with 1&nbsp;μg mL<sup>-1</sup> puromycin (Thermo Fisher Scientific) to select for HeLa CRISPRi cells with integrated library members. 3&nbsp;days after infection, an additional 2&nbsp;μg mL<sup>-1</sup> puromycin was added to cells. Throughout lentiviral transduction and selection steps, cells were analyzed for BFP fluorescence on a BD LSRII flow cytometer to ensure a MOI of 0.1 and completed selection. Following 3&nbsp;days of selection, media was changed to DMEM supplemented with 10% FBS, and HeLa CRISPRi cells were transfected at 50% confluency in 150&nbsp;mm culture dishes (Corning) with 30&nbsp;μg SaPE2–P2A–BSD plasmid, 10&nbsp;μg Sa-pegRNA plasmid for installing a&nbsp;+6 G⋅C to C⋅G edit at the pre-validated edit site, and 3.3&nbsp;μg Sa-sgRNA plasmid for&nbsp;+50 or –50 complementary-strand nicking (where indicated), using 140&nbsp;μL <i>Trans</i>IT-HeLa reagent (Mirus Bio) according to the manufacturer’s protocol. For an unedited HeLa control condition, cells were transfected with only 30&nbsp;μg SaPE2–P2A–BSD plasmid as described above. 24&nbsp;h following transfection, cells were treated with 10&nbsp;μg mL<sup>-1</sup> blasticidin (Thermo Fisher Scientific) to select for expression of SaPE2 protein. 72&nbsp;h after transfection, HeLa CRISPRi cells were washed with PBS (Thermo Fisher Scientific), resuspended using Trypsin and DMEM, and pelleted at 1000 <i>g</i> for 10&nbsp;min. Finally, cells were washed once more with PBS, pelleted at 1000 <i>g</i> for 10&nbsp;min, then stored at −80°C. The number of live cells collected from each Repair-seq condition is listed in <a id="crosref1510" href="#mmc6">Table S6</a>.</div></section><section id="sec-5-4-12"><h4>Repair-seq screens in K562 cells</h4><div id="p0375" role="paragraph">PE2 and PE3+50 Repair-seq screens were performed in duplicate in K562 cells with integrated dCas9–BFP–KRAB (Addgene #46911) (<span class="dropBlock reference-citations"><a id="body-ref-sref19-3" href="#" role="doc-biblioref" data-xml-rid="bib19" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref19-3" aria-controls="sref19-3">Gilbert et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref19-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">19.</div><div>Gilbert, L.A. ∙ Horlbeck, M.A. ∙ Adamson, B. ...</div><div><strong>Genome-Scale CRISPR-Mediated Control of Gene Repression and Activation</strong></div><div><em>Cell.</em> 2014; <strong>159</strong>:647-661</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=10.1016%2Fj.cell.2014.09.029&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=10.1016%2Fj.cell.2014.09.029&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-84908352138" target="_blank" aria-label="Open Scopus in new tab">Scopus (1883)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25307932/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2014.09.029&amp;pmid=25307932" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Cells were transduced with the lentiviral library at 0.2 MOI (18% BFP+) in RPMI supplemented with 10% FBS, 100&nbsp;U mL<sup>-1</sup> penicillin, 100&nbsp;μg mL<sup>-1</sup> streptomycin, 292&nbsp;μg mL<sup>-1</sup> L-Glutamine, and 8&nbsp;μg mL<sup>-1</sup> polybrene (Sigma-Aldrich) by centrifugation at 1000 <i>g</i> for 2&nbsp;h at room temperature. 2&nbsp;days post infection, cells were treated with 3&nbsp;μg mL<sup>-1</sup> puromycin (Gold Biotechnology) to select for cells with integrated library members. After infection, the density of the K562 CRISPRi cells was maintained at approximately 5&nbsp;× 10<sup>5</sup> mL<sup>-1</sup> and the media was replaced with fresh RPMI supplemented with 10% FBS, 100&nbsp;U mL<sup>-1</sup> penicillin, 100&nbsp;μg mL<sup>-1</sup> streptomycin, 292&nbsp;μg mL<sup>-1</sup> L-Glutamine, and 3&nbsp;μg mL<sup>-1</sup> puromycin 3&nbsp;days and 5&nbsp;days post infection. During media replacement, the cells were pelleted, washed with DPBS and resuspended in fresh media to remove dead cells. All centrifugations were performed at 150 <i>g</i> for 5&nbsp;min in 50&nbsp;mL canonical tubes. 6&nbsp;days post infection, the media was replaced twice by fresh RPMI supplemented with 10% FBS and 292&nbsp;μg mL<sup>-1</sup> L-Glutamine (Corning) to remove dead cells and antibiotics. Throughout lentiviral transduction and selection steps, cells were analyzed for BFP fluorescence on an Attune NxT flow cytometer to ensure completed selection.&nbsp;7&nbsp;days post infection, the cells were electroporated using the SE Cell Line 4D-Nucleofector X kit L (Lonza) with 1&nbsp;× 10<sup>7</sup> cells (program FF-120), 7.5&nbsp;μg SaPE2 plasmid, 2.5&nbsp;μg Sa-pegRNA plasmid for installing a&nbsp;+6 G⋅C to C⋅G edit at the pre-validated edit site, and 833&nbsp;ng Sa-sgRNA plasmid for&nbsp;+50 complementary-strand nicking (for PE3+50 conditions). For an unedited K562 control condition, cells were mock electroporated without any DNA plasmid as described above. After electroporation, the cells were seeded at a density of 5&nbsp;× 10<sup>5</sup> mL<sup>-1</sup> in RPMI supplemented with 10% FBS and 292&nbsp;μg mL<sup>-1</sup> L-Glutamine. 48&nbsp;h post electroporation, cultures were pipetted up and down 5 times to prevent cells from clumping. 84&nbsp;h post electroporation, the cells were pelleted at 1000 <i>g</i> for 10&nbsp;min, washed with DPBS (Thermo Fisher Scientific), pelleted at 1000 <i>g</i> for 10&nbsp;min, and then stored at −80°C. The number of live cells collected from each Repair-seq condition is listed in <a id="crosref1520" href="#mmc6">Table S6</a>.</div></section><section id="sec-5-4-13"><h4>High-throughput sequencing of Repair-seq libraries</h4><div id="p0380" role="paragraph">Genomic DNA was extracted from all Repair-seq screen cells using NucleoSpin Blood XL Maxi kit (Machery-Nagel). The entirety of the genomic DNA from each screen condition was used in the initial round of PCR (PCR1) to amplify the 453-bp region containing CRISPRi sgRNA and edit site. Each 100&nbsp;μL PCR1 reaction was performed with 10&nbsp;μg of genomic DNA as template, 1&nbsp;μM of each primer for amplifying pPC1000 sgRNA and edit site (<a id="crosref1525" href="#mmc4">Table S4</a>), and 50&nbsp;μL of NEBNext Ultra II Q5 Master Mix (New England BioLabs) on a BioRad C1000 thermal cycler with the following thermocycler conditions: 98°C for 30 s, 22 cycles of [98°C for 10 s, 65°C for 75 s], followed by 65°C for 5&nbsp;min. Amplification reactions were verified by TBE or agarose gel electrophoresis and ethidium bromide staining. For screens in HeLa cells, 1&nbsp;mL of PCR1 product from each test condition and 1.5&nbsp;mL of PCR1 product from each control condition were purified using SPRIselect (Beckman Coulter) with a double 0.5&nbsp;× right side selection and a 1&nbsp;× left side selection. The eluate was further purified with an additional 0.65&nbsp;× left side selection using SPRIselect. For screens in K562 cells, 250 μL of PCR1 product from each condition was purified using SPRIselect with a 0.8 × left side selection. For screens in both HeLa and K562 cells, purified PCR1 amplicons were quantified using a high sensitivity DNA chip (Agilent Technologies) on an Agilent 2100 Bioanalyzer. A following PCR step (PCR2) enabled indexing of the samples by the addition of i7 and i5 Illumina barcodes, and 4 50&nbsp;μL PCR2 reactions were performed for each screen condition. For each PCR2 reaction, 10&nbsp;ng of PCR1 product was used as a template along with 25&nbsp;μL of KAPA HiFi HotStart ReadyMix (Roche Molecular Systems) and 600&nbsp;nM of each barcoding primer on a ProFlex PCR System (Applied Biosystems) with the following thermocycler conditions: 95°C for 3&nbsp;min, 8 cycles of [98°C for 20 s, 65°C for 15 s, 72°C for 15 s], followed by 72°C for 1&nbsp;min. The reactions were verified by TBE or agarose gel electrophoresis and ethidium bromide staining. For screens in HeLa cells, PCR2 products were purified using SPRIselect with a 0.65 × left side selection and quantified on an Agilent 2100 Bioanalyzer prior to pooling. For screens in K562 cells, PCR2 products were purified using SPRIselect with a 0.8 × left side selection. Repair-seq libraries were sequenced with the NovaSeq 6000 S1 Reagent Kit v1.5 (Illumina) with two 8-nt index reads, 44 cycles for R1 read, 263 cycles for R2 read. The number of sequencing reads acquired for each screen condition and replicate are listed in <a id="crosref1530" href="#mmc6">Table S6</a>.</div></section><section id="sec-5-4-14"><h4>Processing of Repair-seq screen data</h4><div id="p0385" role="paragraph">Repair-seq screen data was processed using a modified version of the analysis approach described in (<span class="dropBlock reference-citations"><a id="body-ref-sref25-6" href="#" role="doc-biblioref" data-xml-rid="bib25" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref25-6" aria-controls="sref25-6">Hussmann et al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref25-6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">25.</div><div>Hussmann, J.A. ∙ Ling, J. ∙ Ravisankar, P. ...</div><div><strong>Mapping the genetic landscape of DNA double-strand break repair</strong></div><div><em>Cell.</em> 2021;</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=10.1016%2Fj.cell.2021.10.002&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=10.1016%2Fj.cell.2021.10.002&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-85117855979" target="_blank" aria-label="Open Scopus in new tab">Scopus (80)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/34653350/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2021.10.002&amp;pmid=34653350" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), with modifications made to accommodate the different library preparation strategy used in this study (direct amplification of genomic DNA without ligation of UMIs before amplification) and the qualitatively different categories of repair outcomes empirically observed in prime editing data.</div><div id="p0390" role="paragraph">Briefly, sequencing data for a batch of screens consists of 4 reads per cluster: 2 8-nt index reads, a 44-nt R1 read of the CRISPRi sgRNA, and a 263-nt R2 read of the repair outcome. Reads from a batch of screens are demultiplexed into individual screens based on index reads. Within each screen, reads are demultiplexed into sets representing outcomes from cells receiving each individual CRISPRi sgRNA by comparing R1 sequences to a table of expected CRISPRi sgRNAs, allowing up to one mismatch between observed and expected sequences. Because direct amplification without UMIs does not allow consensus error correction of multiple reads of each repair outcome, analysis must account for presence of errors in outcome sequences introduced by PCR or by sequencing to avoid interpreting such errors as genuine editing outcomes. As an initial triage, reads with less than 60% of base calls with a quality score greater than or equal to 30 were discarded.</div><div id="p0395" role="paragraph">To categorize a repair outcome sequencing read that passed this quality filter, the outcome was first locally aligned to the screen vector, the pegRNA sequence, the human genome (hg19) and the bos taurus genome (bosTau7) to identify a comprehensive set of alignments between portions of the outcome sequence and any of these reference sequences. The set of local alignments identified was then pruned to a parsimonious set of alignments that explains as much of the read as possible using the minimum number of alignments using a greedy approach. The parsimonious alignments are then parsed through a decision tree that examines their configuration to assign the outcome to a category.</div><div id="p0400" role="paragraph">Outcomes were classified as unedited if they consisted of a single alignment to the screen vector that did not contain the programmed SNV or any indels, with the exception of deletions of 1 nt that did not fall within 5 nt of a programmed nick, which were considered possible sequencing or PCR errors and were disregarded. Outcomes were classified as deletions if they consisted of two alignments to the screen vector that collectively covered the entire read but omitted a segment of the screen vector. Outcomes were classified as tandem duplications if they consisted of two or more alignments to the screen vector that collectively covered the entire read such that the portions of the screen vector covered by any two consecutive alignments on the read overlapped. Outcomes were classified as joining of pegRNA sequence at an unintended location if the set of parsimonious alignments included an alignment to the pegRNA such that the primer binding site (PBS) of the pegRNA was aligned to the same part of the read as the PBS in a screen vector alignment but the reverse transcription template (RTT) of the pegRNA was not aligned to the same part of the read as the RTT in&nbsp;a screen vector alignment. Note that in some such cases, the sequence produced is also consistent in theory with a multi-stage editing event consisting of an initial deletion or duplication that does not disrupt the PAM or protospacer followed by pegRNA-dependent editing of the resulting modified target sequence. Outcomes were classified as installation of additional edits from nearly matched scaffold sequence if the set of parsimonious alignments included an alignment to the pegRNA such that both the PBS and RTT were aligned to the same parts of the read as the PBS and RTT in a single alignment to the screen vector that covered the whole read and that the pegRNA alignment contained fewer edits relative to the outcome than the screen vector alignment.</div></section><section id="sec-5-4-15"><h4>Quantification of CRISPRi-induced changes in outcome frequencies</h4><div id="p0405" role="paragraph">Following categorization of all outcomes for all CRISPRi sgRNAs, counts of each category for each sgRNA are collected into a matrix for downstream analysis, and the total frequency of each category across all outcomes from cells receiving non-targeting sgRNAs is calculated to establish unperturbed baseline frequencies. Because not all CRISPRi sgRNAs achieve high levels of knockdown, calculation of gene-level effects of CRISPRi sgRNAs on outcome categories must strike a balance between assigning increased confidence to phenotypes that are supported by multiple sgRNAs per gene without penalizing genes if not all sgRNAs targeting the gene have high activity. To do this, gene-level changes in outcome category frequencies in a given screen replicate (used in <a id="crosref1540" href="#fig1">Figures 1</a>E–1G, <a id="crosref1545" href="#fig2">2</a>E–2H, and <a id="crosref1550" href="#figs2">S2</a>A–S2I) are calculated by first computing the log<sub>2</sub> fold change in frequency of the category for every targeting sgRNA relative to the combined frequency across all non-targeting sgRNAs. For each gene, the gene-level log<sub>2</sub> fold change is then taken to be the mean of these values for the two sgRNAs targeting the gene with the most extreme absolute values. To provide a qualitative estimate of the range of values produced in the absence of genuine signal by this process of selecting extreme values, the sixty non-targeting sgRNAs were randomly partitioned into 20 sets of 3 quasi-genes and the same process was applied to these quasi-genes.</div></section><section id="sec-5-4-16"><h4>Quantification of deletion boundaries</h4><div id="p0410" role="paragraph">To maximize signal to noise in calculation of position-specific profiles of deletion frequencies and in relative fraction of deletions that removed sequence far outside of programmed nicks, outcomes from all sgRNAs targeting each gene were grouped together. In each such group of outcomes, an array of counts for every position in the screen vector was initialized to 0. For each read classified as a deletion, an interval from the first position deleted to the last position deleted was incremented by 1 in the array of counts. The final array of counts was then divided by the total number of outcomes. Deletions flanked by microhomology result in sequence outcomes that are consistent with two or more degenerate pairs of deletion boundaries. For these deletions, the pair with the minimum values in the coordinate system of the screen vector was arbitrarily chosen. To prevent primer dimers or other non-specific amplification products from being incorrectly identified as long deletions, apparent deletions for which the deleted region overlapped a window of 10-nt around either amplicon primer were excluded from calculation of deletion boundary statistics.</div></section><section id="sec-5-4-17"><h4>Design of recoded Sa-pegRNA scaffold</h4><div id="p0415" role="paragraph">In Repair-seq screens performed, we observed an unintended editing outcome in which additional edits are installed from nearly matched scaffold sequence (<a id="crosref1555" href="#fig2">Figures 2</a>C, 2E, 2F, <a id="crosref1560" href="#figs2">S2</a>B–S2D, and S2G). This outcome contains a&nbsp;+17 T⋅A-to-C⋅G and&nbsp;+19 C⋅G insertion, in addition to the intended&nbsp;+6 G⋅C-to-C⋅G transversion (<a id="crosref1565" href="#fig2">Figures 2</a>C and <a id="crosref1570" href="#figs3">S3</a>A). These unintended edits are consistent with incorporation of an extended 3′ DNA flap generated from reverse transcription of the Sa-pegRNA scaffold sequence into the genome. Because this extended 3′ flap shares 5 nt of homology (5′–GCCAA–3′) with the genomic target sequence after the last edited nucleotide (<a id="crosref1575" href="#figs3">Figure&nbsp;S3</a>A), we hypothesized that disrupting this homology could reduce the frequency of incorporating these unintended edits from reverse transcription of the Sa-pegRNA scaffold. We therefore designed a recoded Sa-pegRNA that alters two base pairs within the Sa-pegRNA scaffold while preserving the same base pairing interactions (<a id="crosref1580" href="#figs3">Figure&nbsp;S3</a>B). The extended 3′ flap templated by this recoded Sa-pegRNA has reduced homology with the genomic target sequence. The spacer, PBS, and RT template sequences of the recoded Sa-pegRNA are identical to those for the Sa-pegRNA used in Repair-seq screens. We observed that prime editing with this recoded Sa-pegRNA mediates similar frequencies of intended editing but substantially reduced unintended scaffold sequence incorporation compared to the original Sa-pegRNA used in Repair-seq screens (<a id="crosref1585" href="#figs3">Figures S3</a>A and S3B).</div></section><section id="sec-5-4-18"><h4>HEK293T siRNA transfection</h4><div id="p0420" role="paragraph">For experiments in <a id="crosref1590" href="#fig3">Figure&nbsp;3</a>C, HEK293T cells were seeded on 6-well plates (Corning) at 7.5&nbsp;× 10<sup>5</sup> cells per well in DMEM plus GlutaMAX supplemented with 10% FBS. At 60% confluency 16&nbsp;h after seeding, cells were transfected with 9&nbsp;μL Lipofectamine RNAiMAX (Thermo Fisher Scientific) according to the manufacturer’s protocol and 90 pmol ON-TARGETplus SMARTpool siRNAs (Horizon Discovery). One day after transfection, media was replaced with fresh DMEM plus GlutaMAX supplemented with 10% FBS. 2&nbsp;days after transfection, cells were washed once with PBS and resuspended using TrypLE (Thermo Fisher Scientific) and DMEM plus GlutaMAX supplemented with 10% FBS. HEK293T cells were then seeded on 96-well plates (Corning) at 2.5&nbsp;× 10<sup>4</sup> cells per well. Between 16 and 24&nbsp;h after seeding, cells were transfected at 60%–80% confluency with 0.5&nbsp;μL Lipofectamine 2000 (Thermo Fisher Scientific) according to the manufacturer’s protocol and 200&nbsp;ng prime editor plasmid, 66&nbsp;ng pegRNA plasmid, 22&nbsp;ng sgRNA plasmid (where indicated), and 5 pmol of the same ON-TARGETplus SMARTpool siRNAs used in the first transfection. For control conditions, cells were treated with non-targeting siRNAs in both transfections. For experiments in <a id="crosref1595" href="#figs3">Figure&nbsp;S3</a>F, only the second transfection with PE components and siRNA was performed. Cells were cultured for 72&nbsp;h after the second transfection before genomic DNA extraction.</div></section><section id="sec-5-4-19"><h4>Real time quantitative PCR</h4><div id="p0425" role="paragraph">To measure RNAi knockdown (<a id="crosref1600" href="#figs3">Figure&nbsp;S3</a>E), RNA was isolated from HEK293T cells 72&nbsp;h after the second siRNA transfection and converted to cDNA using the SYBR Green Fast Advanced Cells-to-CT Kit (Thermo Fisher Scientific) with cell lysis for 10-15&nbsp;min using lysis solution containing 1:50 DNaseI to fully digest genomic DNA. All other steps were carried out according to the manufacturer’s protocol. Each 20&nbsp;μL qPCR reaction was performed in technical and biological triplicate with 500&nbsp;nM of each primer, 2&nbsp;μL cDNA, 1&nbsp;× SYBR Green (Thermo Fisher Scientific), and 10&nbsp;μL Q5 High-Fidelity 2&nbsp;× Master Mix (New England BioLabs) on a CFX96 Touch Real-Time PCR Detection System (Bio-Rad Laboratories) with the following thermocycling conditions: 98°C for 2&nbsp;min, and 40 cycles of [98°C for 15 s, 65°C for 20 s, and 72°C for 30 s]. β-actin (<i>ACTB</i>) served as a housekeeping gene to normalize the amount of cDNA in each qPCR reaction. Relative RNA abundances from gene knockdown were calculated in comparison to a non-targeting siRNA control by the 2<sup>-ΔΔCT</sup> method. A list of primers used for qPCR reactions is provided in <a id="crosref1605" href="#mmc4">Table S4</a>.</div></section><section id="sec-5-4-20"><h4>Plasmid transfection dose titration in HEK293T cells</h4><div id="p0430" role="paragraph">For experiments in Figure&nbsp;S4B, HEK293T cells were seeded on 96-well plates (Corning) at 1.6–1.8&nbsp;× 10<sup>4</sup> cells per well in DMEM plus GlutaMAX supplemented with 10% FBS. Between 16 and 24&nbsp;h after seeding, cells were transfected at 60%–80% confluency with 0.5&nbsp;μL Lipofectamine 2000 (Thermo Fisher Scientific) according to the manufacturer’s protocol and 66&nbsp;ng pegRNA plasmid, 0–200&nbsp;ng PE2 plasmid, and 0–100&nbsp;ng for MLH1dn or RFP plasmid. Empty pUC19 filler plasmid was combined with PE2, MLH1dn, and RFP plasmids in different amounts to maintain a constant amount of total plasmid transfected (366&nbsp;ng). In titrations varying the amount of total editor and <i>in trans</i> protein together, PE2 plasmid was used at a mass ratio of 2:1 with MLH1dn or RFP plasmid. Genomic DNA was isolated from cells 72&nbsp;h after transfection.</div></section><section id="sec-5-4-21"><h4>Generation of MLH1 knock-out HeLa cell clones</h4><div id="p0435" role="paragraph">One clonal wild-type HeLa cell line and two clonal Δ<i>MLH1</i> HeLa lines were used to compare prime editing enhancement from MLH1dn expression versus <i>MLH1</i> knockout (<a id="crosref1610" href="#figs4">Figure&nbsp;S4</a>F). To generate clonal lines, HeLa cells were seeded on 6-well plates (Corning) at 2.5&nbsp;× 10<sup>5</sup> cells per well in DMEM plus GlutaMAX supplemented with 10% FBS. At 60% confluency 18&nbsp;h after seeding, cells were transfected using 7.5&nbsp;μL <i>Trans</i>IT-HeLa reagent (Mirus Bio) according to the manufacturer’s protocol with 2&nbsp;μg pLX_331-Cas9 (SpCas9 with blasticidin marker, Addgene #96924) and 500&nbsp;ng sgRNA plasmid (spacer, 5′– GACAGTGGTGAACCGCATCG–3′). To make clonal wild-type HeLa cells as a control, 500&nbsp;ng pUC19 plasmid was transfected instead of sgRNA plasmid. 24&nbsp;h following transfection, 10&nbsp;ng μL<sup>-1</sup> blasticidin (Thermo Fisher Scientific) was added to each well to select for cells transfected with Cas9.</div><div id="p0440" role="paragraph">4&nbsp;days following transfection, cells were plated on 96-well plates at 1 cell per well with conditioned DMEM plus GlutaMAX supplemented with 10% FBS. Single clones were grown and expanded for 18&nbsp;days. To verify that Δ<i>MLH1</i> cells contain biallelic <i>MLH1</i> frameshift mutations and that control cells contain the wild-type genotype, the <i>MLH1</i> locus from clonal genomic DNA was amplified and sequenced on a MiSeq (Illumina) as described above. FASTQ sequencing files of <i>MLH1</i> in HeLa clones are listed in <a id="crosref1615" href="#mmc3">Table S3</a>. HeLa Δ<i>MLH1</i> clone 1 contains <i>MLH1</i> c.55_56insA and c.41_58delinsTAACTTCC alleles. HeLa Δ<i>MLH1</i> clone 2 contains <i>MLH1</i> c.55_56insA and c.20_66del alleles. All prime editing experiments with these clonal HeLa lines were performed as described above for HeLa cells.</div></section><section id="sec-5-4-22"><h4>Prime editing of contiguous substitutions and additional silent mutations</h4><div id="p0445" role="paragraph">Seven sets of prime edits that substitute 1–5 contiguous bases (35 edits total) were tested across five loci in HEK293T cells (<a id="crosref1620" href="#fig5">Figures 5</a>E, 5F, and <a id="crosref1625" href="#figs5">S5</a>J). Within each set of contiguous substitutions, all five edits altered at least one base within the seed region of the pegRNA protospacer (+1–3 nucleotides), at least one base within the PAM sequence of the pegRNA protospacer (+5&nbsp;G or&nbsp;+6 G), or no bases within the seed region or PAM sequence at all. Because prime edits that alter the seed region or PAM sequence are more efficiently made, the design of these contiguous substitution edits controls for these confounding effects on editing efficiency, thereby enabling comparison of editing efficiency within each set.</div><div id="p0450" role="paragraph">Six sets of prime edits that program a coding change with or without additional silent mutations (27 edits total) were tested across six gene targets in HEK293T cells (<a id="crosref1630" href="#fig5">Figures 5</a>H and <a id="crosref1635" href="#figs5">S5</a>K). Each of the six coding edits makes a transversion at one of the PAM nucleotides of the pegRNA protospacer (+5&nbsp;G or&nbsp;+6 G). This design controls for confounding effects on editing efficiency (as explained above), allowing comparison of editing efficiency within each set. Silent mutations were designed to be close to (typically within 5&nbsp;bp from) the intended coding edit to maximize interference of MMR recognition of the intended coding edit. The frequency of reads that contain the intended coding edit without indels and with or without any additional silent mutations was quantified using CRISPResso2 as described above.</div></section><section id="sec-5-4-23"><h4>Analysis of prime editor activity at Cas9 off-target sites</h4><div id="p0455" role="paragraph">Prime editor activity at known Cas9 off-target sites was determined by sequencing genomic DNA from HEK293T cells 3&nbsp;days after transfection with plasmids encoding PE2, pegRNAs, and MLH1dn (where indicated) as described above. The top 4 off-target sites for each of the <i>HEK3</i>, <i>EMX1</i>, <i>FANCF</i>, and <i>HEK4</i> spacers previously detected by circularization for <i>in vitro</i> reporting of cleavage effects by sequencing (CIRCLE-seq, <span class="dropBlock reference-citations"><a id="body-ref-sref64-2" href="#" role="doc-biblioref" data-xml-rid="bib64" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref64-2" aria-controls="sref64-2">Tsai et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref64-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">66.</div><div>Tsai, S.Q. ∙ Nguyen, N.T. ∙ Malagon-Lopez, J. ...</div><div><strong>CIRCLE-seq: a highly sensitive in&nbsp;vitro screen for genome-wide CRISPR-Cas9 nuclease off-targets</strong></div><div><em>Nat. Methods.</em> 2017; <strong>14</strong>:607-614</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.4278" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-85018272560" target="_blank" aria-label="Open Scopus in new tab">Scopus (522)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28459458/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.4278&amp;pmid=28459458" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) (16 sites total) were deep sequenced from genomic DNA samples as described above. To analyze off-target editing, reads were aligned to reference off-target amplicons using CRISPResso2 (<span class="dropBlock reference-citations"><a id="body-ref-sref11-3" href="#" role="doc-biblioref" data-xml-rid="bib11" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref11-3" aria-controls="sref11-3">Clement et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref11-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">11.</div><div>Clement, K. ∙ Rees, H. ∙ Canver, M.C. ...</div><div><strong>CRISPResso2 provides accurate and rapid genome editing sequence analysis</strong></div><div><em>Nat. Biotechnol.</em> 2019; <strong>37</strong>:224-226</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41587-019-0032-3" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-85062070478" target="_blank" aria-label="Open Scopus in new tab">Scopus (740)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30809026/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41587-019-0032-3&amp;pmid=30809026" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) in standard mode with the parameters “-q 30” and “-w 10.” Off-target reads were called as leniently as possible to capture all potential reverse transcription products. For each off-target reference amplicon, the nucleotide sequence 3′ of the Cas9 nick site (prime-editable target) was compared to the 3′ DNA flap sequence encoded by pegRNA reverse transcription. Counting from the 5′ ends, the minimum sequence of the 3′ DNA flap that deviates from prime-editable target sequence was designated as an off-target marker sequence. All reference-aligned reads that contain this off-target marker sequence directly 3′ of the Cas9 nick site (including indel-containing reads) were called as off-target reads. Off-target editing efficiencies were thus quantified as a percentage of (number of off-target reads)/(number of reference-aligned reads). We note that for some amplicons, mismatch rates at the relevant editing position were comparable to rates at other positions in the amplicon, suggesting that context-specific sequencing errors may contribute to apparent off-target prime editing and therefore this conservative approach may overestimate the true rate of pegRNA-mediated editing at off-target sites.</div></section><section id="sec-5-4-24"><h4>Sequencing of microsatellite instability in genomic DNA</h4><div id="p0460" role="paragraph">Microsatellite instability was assessed in genomic DNA from HCT116 cells, monoclonal wild-type HAP1 cells, monoclonal HAP1 cells grown for 2&nbsp;months (∼60 cell divisions) following <i>MSH2</i> knockout, and HeLa cells 3&nbsp;days after transfection with plasmids encoding PE2–P2A–BSD, pegRNA, and MLH1dn where indicated. HeLa cell transfections were performed as described above. 17 mononucleotide repeats that are highly sensitive to MMR activity and are widely used to diagnosis MMR deficiency in tumors (<span class="dropBlock reference-citations"><a id="body-ref-sref6-2" href="#" role="doc-biblioref" data-xml-rid="bib6" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref6-2" aria-controls="sref6-2">Bacher et&nbsp;al., 2004</a><div class="dropBlock__holder" data-db-target-of="sref6-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">6.</div><div>Bacher, J.W. ∙ Flanagan, L.A. ∙ Smalley, R.L. ...</div><div><strong>Development of a fluorescent multiplex assay for detection of MSI-High tumors</strong></div><div><em>Dis. Markers.</em> 2004; <strong>20</strong>:237-250</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1155/2004/136734" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-12844262940" target="_blank" aria-label="Open Scopus in new tab">Scopus (224)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15528789/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1155%2F2004%2F136734&amp;pmid=15528789" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref23-2" href="#" role="doc-biblioref" data-xml-rid="bib23" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref23-2" aria-controls="sref23-2">Hempelmann et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref23-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">23.</div><div>Hempelmann, J.A. ∙ Scroggins, S.M. ∙ Pritchard, C.C. ...</div><div><strong>MSIplus for Integrated Colorectal Cancer Molecular Testing by Next-Generation Sequencing</strong></div><div><em>J.&nbsp;Mol. Diagn.</em> 2015; <strong>17</strong>:705-714</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_23_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=10.1016%2Fj.jmoldx.2015.05.008&amp;cf=fulltext&amp;site=jmdi-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_23_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=10.1016%2Fj.jmoldx.2015.05.008&amp;cf=pdf&amp;site=jmdi-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_23_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-84943630815" target="_blank" aria-label="Open Scopus in new tab">Scopus (42)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26322950/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.jmoldx.2015.05.008&amp;pmid=26322950" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref66-3" href="#" role="doc-biblioref" data-xml-rid="bib66" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref66-3" aria-controls="sref66-3">Umar et&nbsp;al., 2004</a><div class="dropBlock__holder" data-db-target-of="sref66-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">68.</div><div>Umar, A. ∙ Boland, C.R. ∙ Terdiman, J.P. ...</div><div><strong>Revised Bethesda Guidelines for hereditary nonpolyposis colorectal cancer (Lynch syndrome) and microsatellite instability</strong></div><div><em>J.&nbsp;Natl. Cancer Inst.</em> 2004; <strong>96</strong>:261-268</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/jnci/djh034" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-10744233937" target="_blank" aria-label="Open Scopus in new tab">Scopus (2668)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/14970275/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fjnci%2Fdjh034&amp;pmid=14970275" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) were deep sequenced from genomic DNA samples. The first PCR reaction (PCR1) amplified the microsatellite sequence of interest using primers (Integrated DNA Technologies) containing Illumina forward and reverse adapters. Each 20&nbsp;μL PCR1 reaction was performed with 250&nbsp;nM of each primer, 0.8&nbsp;μL genomic DNA, 1&nbsp;× SYBR Green (Thermo Fisher Scientific), and 10&nbsp;μL Q5 High-Fidelity 2&nbsp;× Master Mix (New England BioLabs) on a CFX96 Touch Real-Time PCR Detection System (Bio-Rad Laboratories) with the following thermocycling conditions: 98°C for 3&nbsp;min, 30 cycles of [98°C for 15 s, 62°C for 30 s, and 72°C for 30 s], followed by 72°C for 3&nbsp;min. All 17 PCR1 products amplified from the same genomic DNA sample were pooled, purified with 0.8&nbsp;× AMPure XP beads (Beckman Coulter), and eluted in nuclease-free water. A list of primers used for PCR1 reactions is provided in <a id="crosref1665" href="#mmc4">Table S4</a>, and a list of PCR1 amplicon sequences if provided in <a id="crosref1670" href="#mmc5">Table S5</a>. The subsequent PCR step (PCR2) added unique i7 and i5 Illumina barcode combinations to both ends of the PCR1 DNA amplicons to enable sample demultiplexing. Each 20&nbsp;μL PCR2 reaction was performed with 500&nbsp;nM of each barcoding primer, 25&nbsp;ng of pooled PCR1 product, 1&nbsp;× SYBR Green, and 10&nbsp;μL Q5 High-Fidelity 2&nbsp;× Master Mix on a CFX96 Touch Real-time PCR Detection System with the following thermocycling conditions: 98°C for 2&nbsp;min, 8 cycles of [98°C for 15 s, 61°C for 20 s, and 72°C for 30 s], followed by 72°C for 2&nbsp;min. All PCR2 products were pooled, purified with 0.8&nbsp;× AMPure XP beads, and eluted in nuclease-free water. DNA amplicon libraries were quantified with a Qubit 3.0 Fluorometer (Thermo Fisher Scientific), then sequenced using the MiSeq Reagent Kit v2 (Illumina) with 300 single-read cycles. A list of FASTQ sequencing files generated in these experiments is provided in <a id="crosref1675" href="#mmc3">Table S3</a>.</div></section><section id="sec-5-4-25"><h4>Quantification of microsatellite instability</h4><div id="p0465" role="paragraph">The 17 microsatellites analyzed all consist of long homopolymers. To quantify the observed lengths of these microsatellites in a way that is robust against the high rate of sequencing errors observed in homopolymers, we searched each sequencing read for the sequences expected to flank the homopolyers and then considered the final length of the homopolymer to be the distance between these flanking sequences. Specifically, for each locus, the longest homopolymer within the amplicon was identified, and 20-nt of the expected reference sequence on either side was recorded. Sequencing reads were demultiplexed into their loci of origin based on the first 20-nt of each read. Within reads for each locus, for each sequencing read, the first occurrences of sequences within Hamming distance 2 of the two flanking sequences were recorded. If both flanking sequences were located in the expected relative orientation within 50-nt of each, the distance between was recorded.</div></section><section id="sec-5-4-26"><h4>Prime editing of therapeutically relevant loci</h4><div id="p0470" role="paragraph">To demonstrate the applicability of PEmax and PE4 and PE5 systems, we tested prime editing at six disease-relevant sites (<a id="crosref1680" href="#fig7">Figure&nbsp;7</a>D). First, we made a silent G⋅C-to-A⋅T transversion at the 6<sup>th</sup> codon of <i>HBB</i>, which is mutated in sickle cell disease patients (<span class="dropBlock reference-citations"><a id="body-ref-sref27-2" href="#" role="doc-biblioref" data-xml-rid="bib27" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref27-2" aria-controls="sref27-2">Ingram, 1956</a><div class="dropBlock__holder" data-db-target-of="sref27-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">27.</div><div>Ingram, V.M.</div><div><strong>A specific chemical difference between the globins of normal human and sickle-cell anaemia haemoglobin</strong></div><div><em>Nature.</em> 1956; <strong>178</strong>:792-794</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/178792a0" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-0000420850" target="_blank" aria-label="Open Scopus in new tab">Scopus (506)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/13369537/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2F178792a0&amp;pmid=13369537" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Second, we installed the G127V allele (a G⋅C-to-T⋅A transversion) in <i>PRNP</i> that confers resistance to prion disease (<span class="dropBlock reference-citations"><a id="body-ref-sref5-2" href="#" role="doc-biblioref" data-xml-rid="bib5" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref5-2" aria-controls="sref5-2">Asante et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref5-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">5.</div><div>Asante, E.A. ∙ Smidak, M. ∙ Grimshaw, A. ...</div><div><strong>A naturally occurring variant of the human prion protein completely prevents prion disease</strong></div><div><em>Nature.</em> 2015; <strong>522</strong>:478-481</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature14510" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-84933041955" target="_blank" aria-label="Open Scopus in new tab">Scopus (128)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26061765/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature14510&amp;pmid=26061765" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>;&nbsp;<span class="dropBlock reference-citations"><a id="body-ref-sref41" href="#" role="doc-biblioref" data-xml-rid="bib41" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref41" aria-controls="sref41">Mead et&nbsp;al., 2009</a><div class="dropBlock__holder" data-db-target-of="sref41"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">42.</div><div>Mead, S. ∙ Whitfield, J. ∙ Poulter, M. ...</div><div><strong>A novel protective prion protein variant that colocalizes with kuru exposure</strong></div><div><em>N.&nbsp;Engl. J. Med.</em> 2009; <strong>361</strong>:2056-2065</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1056/NEJMoa0809716" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-70949091205" target="_blank" aria-label="Open Scopus in new tab">Scopus (146)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19923577/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1056%2FNEJMoa0809716&amp;pmid=19923577" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Third, we introduced a silent C⋅G-to-T⋅A mutation at a <i>CDKL5</i> site known to contain a causative mutation for CDKL5 deficiency disorder, a severe neurodevelopmental condition (<span class="dropBlock reference-citations"><a id="body-ref-sref43-2" href="#" role="doc-biblioref" data-xml-rid="bib43" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref43-2" aria-controls="sref43-2">Olson et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref43-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">44.</div><div>Olson, H.E. ∙ Demarest, S.T. ∙ Pestana-Knight, E.M. ...</div><div><strong>Cyclin-Dependent Kinase-Like 5 Deficiency Disorder: Clinical Review</strong></div><div><em>Pediatr. Neurol.</em> 2019; <strong>97</strong>:18-25</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=10.1016%2Fj.pediatrneurol.2019.02.015&amp;cf=fulltext&amp;site=pnu-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=10.1016%2Fj.pediatrneurol.2019.02.015&amp;cf=pdf&amp;site=pnu-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-85063410039" target="_blank" aria-label="Open Scopus in new tab">Scopus (144)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30928302/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.pediatrneurol.2019.02.015&amp;pmid=30928302" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Fourth, we installed the <i>CXCR4</i> P191A allele (a G⋅C-to-C⋅G edit) that inhibits HIV infection in human cells (<span class="dropBlock reference-citations"><a id="body-ref-sref36-3" href="#" role="doc-biblioref" data-xml-rid="bib36" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref36-3" aria-controls="sref36-3">Liu et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref36-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">36.</div><div>Liu, S. ∙ Wang, Q. ∙ Yu, X. ...</div><div><strong>HIV-1 inhibition in cells with CXCR4 mutant genome created by CRISPR-Cas9 and piggyBac recombinant technologies</strong></div><div><em>Sci. Rep.</em> 2018; <strong>8</strong>:8573</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41598-018-26894-4" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-85048116209" target="_blank" aria-label="Open Scopus in new tab">Scopus (28)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29872154/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41598-018-26894-4&amp;pmid=29872154" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Fifth, we generated the <i>IL2RB</i> H134D Y135F (non-adjacent T⋅A-to-A⋅T and G⋅C-to-C⋅G edits) variant that enables orthogonal IL-2 receptor responsiveness for adoptive T&nbsp;cell transfer therapy (<span class="dropBlock reference-citations"><a id="body-ref-sref53-3" href="#" role="doc-biblioref" data-xml-rid="bib53" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref53-3" aria-controls="sref53-3">Sockolosky et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref53-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">55.</div><div>Sockolosky, J.T. ∙ Trotta, E. ∙ Parisi, G. ...</div><div><strong>Selective targeting of engineered T&nbsp;cells using orthogonal IL-2 cytokine-receptor complexes</strong></div><div><em>Science.</em> 2018; <strong>359</strong>:1037-1042</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aar3246" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-85047997059" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29496879/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aar3246&amp;pmid=29496879" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Lastly, we recoded the BCL11A repressor binding site within the <i>HBG1</i> and <i>HBG2</i> fetal hemoglobin gene promoters to a GATA1 transcriptional activator motif (non-adjacent G⋅C-to-A⋅T and C⋅G-to-A⋅T edits), which in principle could induce fetal hemoglobin expression for treatment of hemoglobinopathies (<span class="dropBlock reference-citations"><a id="body-ref-sref2" href="#" role="doc-biblioref" data-xml-rid="bib2" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref2" aria-controls="sref2">Amato et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">2.</div><div>Amato, A. ∙ Cappabianca, M.P. ∙ Perri, M. ...</div><div><strong>Interpreting elevated fetal hemoglobin in pathology and health at the basic laboratory level: new and known γ- gene mutations associated with hereditary persistence of fetal hemoglobin</strong></div><div><em>Int. J. Lab. Hematol.</em> 2014; <strong>36</strong>:13-19</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1111/ijlh.12094" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_2_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-84892481283" target="_blank" aria-label="Open Scopus in new tab">Scopus (33)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23621512/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1111%2Fijlh.12094&amp;pmid=23621512" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div></section><section id="sec-5-4-27"><h4><i>In&nbsp;vitro</i> transcription of prime editor and MLH1dn mRNA used in iPSC and T&nbsp;cell experiments</h4><div id="p0475" role="paragraph">As described previously (<span class="dropBlock reference-citations"><a id="body-ref-sref42-4" href="#" role="doc-biblioref" data-xml-rid="bib42" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref42-4" aria-controls="sref42-4">Nelson et al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref42-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">43.</div><div>Nelson, J.W. ∙ Randolph, P.B. ∙ Shen, S.P. ...</div><div><strong>Engineered pegRNAs improve prime editing efficiency</strong></div><div><em>Nat. Biotechnol.</em> 2021;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41587-021-01039-7" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-85116367544" target="_blank" aria-label="Open Scopus in new tab">Scopus (297)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/34608327/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41587-021-01039-7&amp;pmid=34608327" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), plasmids were cloned to encode an inactivated T7 promoter followed by a 5′ untranslated region (UTR), Kozak sequence, coding sequences of PE2 or MLH1dn, and a 3′ UTR. T7 promoter inactivation prevents potential transcription from circular plasmid template during mRNA generation. These components together were PCR amplified with Phusion U Green Multiplex Master Mix (Thermo Fisher Scientific) using primers that correct T7 promoter inactivation and append a 119-nt poly(A) tail to the 3′ UTR. The resulting PCR product was purified with the QIAquick PCR Purification Kit (Thermo Fisher Scientific) and served as a template for subsequent <i>in&nbsp;vitro</i> transcription. PE2 and MLH1dn mRNAs were transcribed from these templates using the HiScribe T7 High-Yield RNA Synthesis Kit (New England BioLabs) with co-transcriptional capping by CleanCap AG (TriLink Biotechnologies) and full replacement of UTP with N<sup>1</sup>-Methylpseudouridine-5′-triphosphate (TriLink Biotechnologies). Transcribed mRNAs were precipitated in 2.5&nbsp;M lithium chloride (Thermo Fisher Scientific), washed twice in 70% ethanol, then dissolved in nuclease-free water. The resulting PE2 and MLH1dn mRNA was quantified with a NanoDrop One UV-Vis spectrophotometer (Thermo Fisher Scientific) and was stored at −80°C.</div></section><section id="sec-5-4-28"><h4>Electroporation of human patient-derived induced pluripotent stem cells</h4><div id="p0480" role="paragraph">Prior to electroporation, 24-well culture plates (Thermo Fisher Scientific) were coated with 250&nbsp;μL rhLaminin-521 (Thermo Fisher Scientific) diluted 1:40 in DPBS (Thermo Fisher Scientific) per well, and incubated at 37°C in a 5% CO<sub>2</sub> incubator for 2 h. For electroporation, iPS cell colonies at 70%–80% confluency were washed once with DPBS and dissociated in pre-warmed Accutase (Innovative Cell Technologies) for 10&nbsp;min at 37°C in a 5% CO<sub>2</sub> incubator. Next, iPS cells were gently triturated, moved into a sterile 15&nbsp;mL conical tube, then combined with an equal volume of DMEM/F12 (Thermo Fisher Scientific) to quench dissociation enzyme activity. Cells were pelleted at 300 <i>g</i> for 3&nbsp;min and resuspended in StemFlex medium (Thermo Fisher Scientific) supplemented with 10&nbsp;μM Y-27632 (Cayman Chemical). Cell counts and viability were determined using the Countess II FL Automated Cell Counter (Thermo Fisher Scientific). For electroporation using the NEON Transfection System 10&nbsp;μL kit (Thermo Fisher Scientific), 2&nbsp;× 10<sup>5</sup> iPS cells were pelleted at 300 <i>g</i> for 3&nbsp;min and resuspended in 9&nbsp;μL NEON Buffer R. The cell solution was combined with a 3&nbsp;μL mixture of 1&nbsp;μg PE2 mRNA, 90 pmol synthetic pegRNA (Integrated DNA Technologies), 60 pmol synthetic sgRNA (Synthego) where indicated, and 0–2&nbsp;μg MLH1dn mRNA in NEON Buffer R. Synthetic pegRNAs and sgRNAs were dissolved in TE buffer (10&nbsp;mM Tris-HCl, pH 8.0; 0.1&nbsp;mM EDTA). Mock control electroporations were performed with 3&nbsp;μL NEON Buffer R without any RNA added. Directly prior to electroporation, rhLaminin-521 was aspirated and immediately replaced with 250&nbsp;μL pre-warmed StemFlex medium supplemented with 10&nbsp;μM Y-27632 per 24-well. Next, 10&nbsp;μL of the combined cell and RNA mixture was electroporated using the NEON Transfection System (Thermo Fisher Scientific) with the following parameters: 1400 V, 20&nbsp;ms, one pulse. Cells were seeded immediately into rhLaminin-521-coated 24-well plates with 250&nbsp;μL StemFlex medium supplemented with 10&nbsp;μM Y-27632 per well. Media was changed the following day with 500&nbsp;μL StemFlex medium supplemented with 5&nbsp;μM Y-27632. 72&nbsp;h following electroporation, media was changed to 500&nbsp;μL StemFlex medium per well. Genomic DNA was extracted 96&nbsp;h after electroporation by washing iPS cells once with DPBS, lysing with gDNA lysis buffer (10&nbsp;mM Tris-HCl, pH 8.0; 0.05% SDS; 800 units μL<sup>-1</sup> proteinase K (New England BioLabs)) at 37°C for 2 h, followed by enzyme inactivation at 80°C for 30&nbsp;min. All iPSC electroporations were performed in technical duplicate and biological triplicate.</div><div id="p0485" role="paragraph">Following amplicon sequencing of the edited <i>CDKL5</i> locus, frequencies of intended editing and indels were quantified with CRISPResso2 in HDR mode, as described above. Because patient-derived iPSCs were heterozygous for the c.1412delA allele, the frequency of editable alleles with the intended edit was quantified as: (editing frequency – editing frequency in mock controls)/(100 – editing frequency in mock controls). Frequency of editable alleles with indels was quantified as described above: (total number of indel-containing reads)/(number of amplicon-aligned reads). The resulting frequencies of editable alleles with the intended edit or indels were averaged between technical duplicates, and values from biological triplicates are shown.</div></section><section id="sec-5-4-29"><h4>Electroporation of primary human T&nbsp;cells</h4><div id="p0490" role="paragraph">Prior to electroporation, T&nbsp;cells were activated for 2&nbsp;days with Dynabeads Human T-Expander CD3/CD28 (Thermo Fisher Scientific) and cultured at 37°C and 5% CO<sub>2</sub> in T&nbsp;cell media (X-VIVO 15 Serum-free Hematopoietic Cell Medium (Lonza), supplemented with 5% AB human serum (Valley Biomedical), 1&nbsp;× GlutaMAX (Thermo Fisher Scientific), 12&nbsp;mM N-acetyl-cysteine (Sigma Aldrich), 50&nbsp;U mL<sup>-1</sup> penicillin and 50&nbsp;μg mL<sup>-1</sup> streptomycin (Thermo Fisher Scientific), 300 IU mL<sup>-1</sup> IL-2 (Peprotech), and 5&nbsp;ng mL<sup>-1</sup> recombinant human IL-7 (Peprotech) and IL-15 (Peprotech)). CD3/CD28 beads were removed from cells 5–7&nbsp;h before electroporation. For electroporation using the NEON Transfection System 10&nbsp;μL kit (Thermo Fisher Scientific), 3.0–3.5&nbsp;× 10<sup>5</sup> cells per sample were pelleted by centrifugation for 5&nbsp;min at 300 <i>g</i> and resuspended in 11&nbsp;μL NEON Buffer T. The cell solution was added to a mix of 1&nbsp;μg PE2 mRNA, 90 pmol synthetic pegRNA (Integrated DNA Technologies), 60 pmol synthetic sgRNA (Synthego), and 0–2&nbsp;μg MLH1dn mRNA. Synthetic pegRNAs and sgRNAs were dissolved in TE buffer (10&nbsp;mM Tris-HCl, pH 8.0; 0.1&nbsp;mM EDTA). Mock control electroporations were performed with 3&nbsp;μL NEON Buffer T without any RNA added. Electroporation on the NEON Transfection System (Thermo Fisher Scientific) was carried out using 10&nbsp;μL NEON tips with the following parameters: 1,400 V, 10&nbsp;ms, three pulses. Cells were plated in 600&nbsp;μL fresh T&nbsp;cell media in a 24-well plate. 2&nbsp;days after electroporation, cell counts and viability were determined using the Countess II Automated Cell Counter (Thermo Fisher Scientific), and 1&nbsp;mL fresh T&nbsp;cell media was added to cells. 4&nbsp;days after electroporation, cells were pelleted by centrifugation for 5&nbsp;min at 300 <i>g</i> and genomic DNA was isolated using the PureLink Genomic DNA Mini Kit (Thermo Fisher Scientific) following the “mammalian cells lysate” protocol with elution in nuclease-free water.</div></section></section><section id="sec-5-5"><h3>Quantification and statistical analysis</h3><div id="p0495" role="paragraph">The number of independent biological replicates and technical replicates for each experiment are described in the figure legends or <a id="crosref1725" href="#sec-5">STAR Methods</a> section. In <a id="crosref1730" href="#fig5">Figure&nbsp;5</a>I, a non-parametric Mann-Whitney <i>U</i> test was used to compare prime editing data in HEK293T cells with prime editing data in HeLa, K562, and U2OS cells.</div></section></section><section id="acknowledgments" role="doc-acknowledgments"><h2>Acknowledgments</h2><div id="p0500" role="paragraph">This work was supported by the Merkin Institute of Transformative Technologies in Healthcare; NIH grants U01AI142756, RM1HG009490, R01EB031172, and R35GM118062; the Howard Hughes Medical Institute; the Loulou Foundation; and the Bill &amp; Melinda Gates Foundation. P.J.C. was supported by the NSF Graduate Research Fellowship. J.A.H. was the Rebecca Ridley Kry Fellow of the Damon Runyon Cancer Research Foundation (DRG-2262-16). J.Y. was supported by a fellowship provided by the China Scholarship Council (CSC), based on the April 2015 Memorandum of Understanding between the CSC and Princeton University. J.W.N. was supported by a Jane Coffin Childs postdoctoral fellowship. G.A.N. was supported by a Helen Hay Whitney postdoctoral fellowship. M.J.O. was supported by a Saint Baldrick's Foundation Scholar Award, the Kidz1stFund, and the Chambers Family Innovation Fund. Work in the lab of B.A. was supported by NIH grant R35GM138167-01, Rutgers Cancer Institute of New Jersey via NCI-CCSG grant P30CA072720), and the Searle Scholars Program. We thank A. Anzalone, K. Everette, J. Davis, L. Koblan, A. Le, and D. Simpson for helpful discussions and technical advice; Dr. Anahita Vieira for assistance editing the manuscript; the Princeton University Flow Cytometry Resource Facility, which is supported by NCI-CCSG grant P30CA072720-5921; Wei Wang and the Genomics Core Facility in the Lewis-Sigler Institute for Integrative Genomics at Princeton; and the Human Neuron Core at Boston Children’s Hospital, which is supported by NIH grant U54HD090255.</div><section id="sec-6"><h3>Author contributions</h3><div id="p0505" role="paragraph">P.J.C., J.A.H., B.A., and D.R.L. designed the research and wrote the manuscript. P.J.C., J.Y., P.R., and B.A. performed Repair-seq screens. J.A.H. analyzed Repair-seq screen data. P.J.C. developed PE4, PE5, and PEmax. P.J.C. and J.A.H. analyzed sequencing data. J.Y. performed K562 and U2OS cell experiments. F.K. and M.J.O. performed T&nbsp;cell experiments with input from G.A.N. P.-F.C. and C.C. performed iPSC experiments with assistance from P.J.C. and J.W.N. M.S., M.J.O., J.S.W., B.A., and D.R.L. supervised the project.</div></section><section id="sec-7"><h3>Declaration of interests</h3><div id="p0510" role="paragraph">P.J.C., J.A.H., B.A., and D.R.L. have filed patent applications on aspects of this work through their respective institutions. J.A.H. is a consultant for Tessera Therapeutics. P.-F.C. is currently an employee of Tessera Therapeutics. B.A. was a member of a ThinkLab Advisory Board for, and holds equity in, Celsius Therapeutics. D.R.L. is a consultant and equity holder of Beam Therapeutics, Prime Medicine, Pairwise Plants, and Chroma Medicine.</div></section></section><section id="supplementary-material" data-type="supplementary-materials"><h2>Supplemental information (6)</h2><div id="p0520" role="paragraph"><div class="core-supplementary-material"><div class="core-link"><button class="downloadAllSupplementary"><span>Download all</span></button></div></div><div id="mmc1" class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="/cms/10.1016/j.cell.2021.09.018/attachment/879f206c-bfba-4958-acf9-6aca66f01b28/mmc1.xlsx" download="mmc1.xlsx" aria-labelledby="mmc1-heading mmc1-link"><span id="mmc1-link">Spreadsheet (173.62 KB)</span></a></div><div class="core-description"><div id="mmc1-heading" class="heading">Table S1. List of CRISPRi guide RNAs in prime editing Repair-seq libraries, related to Figures 1 and 2</div><div id="espara0010" role="paragraph">Sequences of each CRISPRi sgRNA spacer and the associated oligonucleotide used to clone lentiviral plasmid libraries for Repair-seq screens (additional details in STAR Methods).</div></div></div><div id="mmc2" class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="/cms/10.1016/j.cell.2021.09.018/attachment/17d80ebc-68d1-46a6-b9e0-e1210507bfb1/mmc2.xlsx" download="mmc2.xlsx" aria-labelledby="mmc2-heading mmc2-link"><span id="mmc2-link">Spreadsheet (250.52 KB)</span></a></div><div class="core-description"><div id="mmc2-heading" class="heading">Table S2. Frequencies of the intended prime edit in Repair-seq screens, related to Figure&nbsp;1</div><div id="espara0015" role="paragraph">Percentage of reads with only the programmed&nbsp;+6 G⋅C-to-C⋅G prime edit and no unintended byproducts in Repair-seq screens, grouped by CRISPRi sgRNA.</div></div></div><div id="mmc3" class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="/cms/10.1016/j.cell.2021.09.018/attachment/8c8bb321-ccf0-4b5c-a2b6-42505cfdafdd/mmc3.xlsx" download="mmc3.xlsx" aria-labelledby="mmc3-heading mmc3-link"><span id="mmc3-link">Spreadsheet (152.70 KB)</span></a></div><div class="core-description"><div id="mmc3-heading" class="heading">Table S3. List of sequencing data and guide RNAs from all experiments, related to Figures 1, 2, 3, 4, 5, 6, and 7</div><div id="espara0020" role="paragraph">List of every high-throughput sequencing experiment performed with the associated treatment conditions, sequences of pegRNA and sgRNA used, sequenced amplicons, and names of FASTQ files deposited at the NCBI Sequence Read Archive database under PRJNA735408.</div></div></div><div id="mmc4" class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="/cms/10.1016/j.cell.2021.09.018/attachment/7b04d3bb-e5cd-4d87-b54c-4c632cdddd70/mmc4.xlsx" download="mmc4.xlsx" aria-labelledby="mmc4-heading mmc4-link"><span id="mmc4-link">Spreadsheet (13.47 KB)</span></a></div><div class="core-description"><div id="mmc4-heading" class="heading">Table S4. Primers used for sequencing of genomic DNA and qRT-PCR, related to Figures 1, 2, 3, 4, 5, 6, and 7</div></div></div><div id="mmc5" class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="/cms/10.1016/j.cell.2021.09.018/attachment/217b0885-8ce2-42b6-a31a-3844b48d3bf0/mmc5.xlsx" download="mmc5.xlsx" aria-labelledby="mmc5-heading mmc5-link"><span id="mmc5-link">Spreadsheet (16.21 KB)</span></a></div><div class="core-description"><div id="mmc5-heading" class="heading">Table S5. Nucleotide sequences of amplicons from high-throughput sequencing, related to Figures 1, 2, 3, 4, 5, 6, and 7</div></div></div><div id="mmc6" class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="/cms/10.1016/j.cell.2021.09.018/attachment/59291224-27bc-4b94-987f-b5cf41a03035/mmc6.xlsx" download="mmc6.xlsx" aria-labelledby="mmc6-heading mmc6-link"><span id="mmc6-link">Spreadsheet (174.31 KB)</span></a></div><div class="core-description"><div id="mmc6-heading" class="heading">Table S6. Number of cells and sequencing reads from Repair-seq screens, related to Figures 1 and 2</div><div id="espara0025" role="paragraph">The first sheet contains the number of cells and sequencing reads recovered for each Repair-seq screen condition. The second sheet contains the number of sequencing reads recovered for each CRISPRi sgRNA in each Repair-seq screen. Unedited HeLa screen controls were transfected with SaPE2 without Sa-pegRNA. Unedited K562 screen controls were transfected without any plasmid DNA.</div></div></div></div></section></div></section><div class="core-linked-content">
    

        

</div><section id="backmatter" data-extent="backmatter"><div class="core-container"><section id="data-availability" class="core-data-availability"><h2>Data and code availability</h2><div id="p0030" role="paragraph">Amplicon sequencing data generated during this study are available at the NCBI Sequence Read Archive database under PRJNA735408. Data from Repair-seq screens are available under PRJNA734952. Processed Repair-seq screen data are available at DOI: <a id="PC_linkagpzCsHr6z" href="https://doi.org/10.5281/zenodo.5551032" target="_blank">https://doi.org/10.5281/zenodo.5551032</a>. The code used for data processing and analysis are available at <a id="intref0010" href="https://github.com/pinellolab/CRISPResso2" target="_blank">https://github.com/pinellolab/CRISPResso2</a> and <a id="PC_link4RQM4z1kA4" href="https://github.com/jeffhussmann/repair-seq" target="_blank">https://github.com/jeffhussmann/repair-seq</a> (DOI: <a id="PC_linkATrrOsudhx" href="https://doi.org/10.5281/zenodo.5534778" target="_blank">https://doi.org/10.5281/zenodo.5534778</a>).</div></section><section id="references"><h2>References</h2><div id="bibliography" role="doc-bibliography"><div role="list"><div role="listitem" data-has="label"><div id="bib1" class="citations"><div id="sref1" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref1" title="View in article">1.</a></div><div>Acharya, S. ∙ Wilson, T. ∙ Gradia, S. ...</div><div><strong>hMSH2 forms specific mispair-binding complexes with hMSH3 and hMSH6</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 1996; <strong>93</strong>:13629-13634</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.93.24.13629" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_1_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-0030465237" target="_blank" aria-label="Open Scopus in new tab">Scopus (481)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/8942985/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.93.24.13629&amp;pmid=8942985" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib2" class="citations"><div id="sref2" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref2" title="View in article">2.</a></div><div>Amato, A. ∙ Cappabianca, M.P. ∙ Perri, M. ...</div><div><strong>Interpreting elevated fetal hemoglobin in pathology and health at the basic laboratory level: new and known γ- gene mutations associated with hereditary persistence of fetal hemoglobin</strong></div><div><em>Int. J. Lab. Hematol.</em> 2014; <strong>36</strong>:13-19</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1111/ijlh.12094" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_2_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-84892481283" target="_blank" aria-label="Open Scopus in new tab">Scopus (33)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23621512/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1111%2Fijlh.12094&amp;pmid=23621512" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib3" class="citations"><div id="sref3" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref3-1" title="View in article">3.</a></div><div>Anzalone, A.V. ∙ Randolph, P.B. ∙ Davis, J.R. ...</div><div><strong>Search-and-replace genome editing without double-strand breaks or donor DNA</strong></div><div><em>Nature.</em> 2019; <strong>576</strong>:149-157</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-1711-4" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-85074571375" target="_blank" aria-label="Open Scopus in new tab">Scopus (2574)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31634902/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-1711-4&amp;pmid=31634902" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib4" class="citations"><div id="sref4" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref4" title="View in article">4.</a></div><div>Anzalone, A.V. ∙ Koblan, L.W. ∙ Liu, D.R.</div><div><strong>Genome editing with CRISPR-Cas nucleases, base editors, transposases and prime</strong></div><div><em>Nat. Biotechnol.</em> 2020; <strong>38</strong>:824-844</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41587-020-0561-9" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-85086735534" target="_blank" aria-label="Open Scopus in new tab">Scopus (1256)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32572269/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41587-020-0561-9&amp;pmid=32572269" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib5" class="citations"><div id="sref5" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref5-1" title="View in article">5.</a></div><div>Asante, E.A. ∙ Smidak, M. ∙ Grimshaw, A. ...</div><div><strong>A naturally occurring variant of the human prion protein completely prevents prion disease</strong></div><div><em>Nature.</em> 2015; <strong>522</strong>:478-481</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature14510" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-84933041955" target="_blank" aria-label="Open Scopus in new tab">Scopus (128)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26061765/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature14510&amp;pmid=26061765" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib6" class="citations"><div id="sref6" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref6-1" title="View in article">6.</a></div><div>Bacher, J.W. ∙ Flanagan, L.A. ∙ Smalley, R.L. ...</div><div><strong>Development of a fluorescent multiplex assay for detection of MSI-High tumors</strong></div><div><em>Dis. Markers.</em> 2004; <strong>20</strong>:237-250</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1155/2004/136734" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-12844262940" target="_blank" aria-label="Open Scopus in new tab">Scopus (224)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15528789/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1155%2F2004%2F136734&amp;pmid=15528789" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib7" class="citations"><div id="sref7" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref7" title="View in article">7.</a></div><div>Bartlett, D.W. ∙ Davis, M.E.</div><div><strong>Insights into the kinetics of siRNA-mediated gene silencing from live-cell and live-animal bioluminescent imaging</strong></div><div><em>Nucleic Acids Res.</em> 2006; <strong>34</strong>:322-333</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gkj439" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-31544456413" target="_blank" aria-label="Open Scopus in new tab">Scopus (406)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16410612/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgkj439&amp;pmid=16410612" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib8" class="citations"><div id="sref8" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref8" title="View in article">8.</a></div><div>Bosch, J.A. ∙ Birchak, G. ∙ Perrimon, N.</div><div><strong>Precise genome engineering in <i>Drosophila</i> using prime editing</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2021; <strong>118</strong></div><div>e2021996118</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.2021996118" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-85098172457" target="_blank" aria-label="Open Scopus in new tab">Scopus (64)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.2021996118" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib9" class="citations"><div id="sref9" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref9" title="View in article">9.</a></div><div>Cavaleiro, A.M. ∙ Kim, S.H. ∙ Seppälä, S. ...</div><div><strong>Accurate DNA Assembly and Genome Engineering with Optimized Uracil Excision Cloning</strong></div><div><em>ACS Synth. Biol.</em> 2015; <strong>4</strong>:1042-1046</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/acssynbio.5b00113" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-84942045165" target="_blank" aria-label="Open Scopus in new tab">Scopus (67)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26263045/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Facssynbio.5b00113&amp;pmid=26263045" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib10" class="citations"><div id="sref10" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref10-1" title="View in article">10.</a></div><div>Chen, P.-F. ∙ Chen, T. ∙ Forman, T.E. ...</div><div><strong>Generation and characterization of human induced pluripotent stem cells (iPSCs) from three male and three female patients with CDKL5 Deficiency Disorder (CDD)</strong></div><div><em>Stem Cell Res. (Amst.).</em> 2021; <strong>53</strong>:102276</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.scr.2021.102276" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-85102606970" target="_blank" aria-label="Open Scopus in new tab">Scopus (6)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33714067/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.scr.2021.102276&amp;pmid=33714067" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib11" class="citations"><div id="sref11" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref11-1" title="View in article">11.</a></div><div>Clement, K. ∙ Rees, H. ∙ Canver, M.C. ...</div><div><strong>CRISPResso2 provides accurate and rapid genome editing sequence analysis</strong></div><div><em>Nat. Biotechnol.</em> 2019; <strong>37</strong>:224-226</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41587-019-0032-3" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-85062070478" target="_blank" aria-label="Open Scopus in new tab">Scopus (740)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30809026/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41587-019-0032-3&amp;pmid=30809026" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib12" class="citations"><div id="sref12" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref12" title="View in article">12.</a></div><div>Dang, C.V. ∙ Lee, W.M.</div><div><strong>Identification of the human c-myc protein nuclear translocation signal</strong></div><div><em>Mol. Cell. Biol.</em> 1988; <strong>8</strong>:4048-4054</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/MCB.8.10.4048" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-0023731952" target="_blank" aria-label="Open Scopus in new tab">Scopus (287)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/3054508/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FMCB.8.10.4048&amp;pmid=3054508" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib13" class="citations"><div id="sref13" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref13" title="View in article">13.</a></div><div>Engler, C. ∙ Kandzia, R. ∙ Marillonnet, S.</div><div><strong>A one pot, one step, precision cloning method with high throughput capability</strong></div><div><em>PLoS ONE.</em> 2008; <strong>3</strong>:e3647</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pone.0003647" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_13_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-56649114274" target="_blank" aria-label="Open Scopus in new tab">Scopus (1561)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/18985154/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pone.0003647&amp;pmid=18985154" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib14" class="citations"><div id="sref14" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref14" title="View in article">14.</a></div><div>Fang, W.H. ∙ Modrich, P.</div><div><strong>Human strand-specific mismatch repair occurs by a bidirectional mechanism similar to that of the bacterial reaction</strong></div><div><em>J.&nbsp;Biol. Chem.</em> 1993; <strong>268</strong>:11838-11844</div></div><div class="external-links"><div class="core-xlink-abstract"><a href="/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=10.1016%2FS0021-9258%2819%2950276-9&amp;cf=abstract&amp;site=jbc-site" target="_blank" aria-label="Open Abstract in new tab">Abstract</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=10.1016%2FS0021-9258%2819%2950276-9&amp;cf=pdf&amp;site=jbc-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/8505312/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0021-9258%2819%2950276-9&amp;pmid=8505312" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib15" class="citations"><div id="sref15" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref15" title="View in article">15.</a></div><div>Geng, H. ∙ Du, C. ∙ Chen, S. ...</div><div><strong>In&nbsp;vitro studies of DNA mismatch repair proteins</strong></div><div><em>Anal. Biochem.</em> 2011; <strong>413</strong>:179-184</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.ab.2011.02.017" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-79955466020" target="_blank" aria-label="Open Scopus in new tab">Scopus (28)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21329650/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.ab.2011.02.017&amp;pmid=21329650" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib16" class="citations"><div id="sref16" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref16" title="View in article">16.</a></div><div>Genschel, J. ∙ Littman, S.J. ∙ Drummond, J.T. ...</div><div><strong>Isolation of MutSbeta from human cells and comparison of the mismatch repair specificities of MutSbeta and MutSalpha</strong></div><div><em>J.&nbsp;Biol. Chem.</em> 1998; <strong>273</strong>:19895-19901</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=10.1074%2Fjbc.273.31.19895&amp;cf=fulltext&amp;site=jbc-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=10.1074%2Fjbc.273.31.19895&amp;cf=pdf&amp;site=jbc-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-0032584384" target="_blank" aria-label="Open Scopus in new tab">Scopus (348)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/9677427/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1074%2Fjbc.273.31.19895&amp;pmid=9677427" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib17" class="citations"><div id="sref17" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref17-1" title="View in article">17.</a></div><div>Genschel, J. ∙ Bazemore, L.R. ∙ Modrich, P.</div><div><strong>Human exonuclease I is required for 5′ and 3′ mismatch repair</strong></div><div><em>J.&nbsp;Biol. Chem.</em> 2002; <strong>277</strong>:13302-13311</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=10.1074%2Fjbc.M111854200&amp;cf=fulltext&amp;site=jbc-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=10.1074%2Fjbc.M111854200&amp;cf=pdf&amp;site=jbc-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-0037066720" target="_blank" aria-label="Open Scopus in new tab">Scopus (204)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11809771/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1074%2Fjbc.M111854200&amp;pmid=11809771" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib18" class="citations"><div id="sref18" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref18-1" title="View in article">18.</a></div><div>Gilbert, L.A. ∙ Larson, M.H. ∙ Morsut, L. ...</div><div><strong>CRISPR-mediated modular RNA-guided regulation of transcription in eukaryotes</strong></div><div><em>Cell.</em> 2013; <strong>154</strong>:442-451</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=10.1016%2Fj.cell.2013.06.044&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=10.1016%2Fj.cell.2013.06.044&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-84880571335" target="_blank" aria-label="Open Scopus in new tab">Scopus (2710)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23849981/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2013.06.044&amp;pmid=23849981" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib19" class="citations"><div id="sref19" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref19-1" title="View in article">19.</a></div><div>Gilbert, L.A. ∙ Horlbeck, M.A. ∙ Adamson, B. ...</div><div><strong>Genome-Scale CRISPR-Mediated Control of Gene Repression and Activation</strong></div><div><em>Cell.</em> 2014; <strong>159</strong>:647-661</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=10.1016%2Fj.cell.2014.09.029&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=10.1016%2Fj.cell.2014.09.029&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-84908352138" target="_blank" aria-label="Open Scopus in new tab">Scopus (1883)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25307932/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2014.09.029&amp;pmid=25307932" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib20" class="citations"><div id="sref20" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref20-1" title="View in article">20.</a></div><div>Gueneau, E. ∙ Dherin, C. ∙ Legrand, P. ...</div><div><strong>Structure of the MutLα C-terminal domain reveals how Mlh1 contributes to Pms1 endonuclease site</strong></div><div><em>Nat. Struct. Mol. Biol.</em> 2013; <strong>20</strong>:461-468</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nsmb.2511" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-84876193944" target="_blank" aria-label="Open Scopus in new tab">Scopus (91)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23435383/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnsmb.2511&amp;pmid=23435383" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib21" class="citations"><div id="sref21" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref21" title="View in article">21.</a></div><div>Guerrette, S. ∙ Acharya, S. ∙ Fishel, R.</div><div><strong>The interaction of the human MutL homologues in hereditary nonpolyposis colon cancer</strong></div><div><em>J.&nbsp;Biol. Chem.</em> 1999; <strong>274</strong>:6336-6341</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=10.1074%2Fjbc.274.10.6336&amp;cf=fulltext&amp;site=jbc-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=10.1074%2Fjbc.274.10.6336&amp;cf=pdf&amp;site=jbc-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-0033525518" target="_blank" aria-label="Open Scopus in new tab">Scopus (157)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/10037723/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1074%2Fjbc.274.10.6336&amp;pmid=10037723" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib22" class="citations"><div id="sref22" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref22-1" title="View in article">22.</a></div><div>Gupta, S. ∙ Gellert, M. ∙ Yang, W.</div><div><strong>Mechanism of mismatch recognition revealed by human MutSβ bound to unpaired DNA loops</strong></div><div><em>Nat. Struct. Mol. Biol.</em> 2011; <strong>19</strong>:72-78</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nsmb.2175" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-84855463183" target="_blank" aria-label="Open Scopus in new tab">Scopus (125)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22179786/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnsmb.2175&amp;pmid=22179786" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib23" class="citations"><div id="sref23" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref23-1" title="View in article">23.</a></div><div>Hempelmann, J.A. ∙ Scroggins, S.M. ∙ Pritchard, C.C. ...</div><div><strong>MSIplus for Integrated Colorectal Cancer Molecular Testing by Next-Generation Sequencing</strong></div><div><em>J.&nbsp;Mol. Diagn.</em> 2015; <strong>17</strong>:705-714</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_23_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=10.1016%2Fj.jmoldx.2015.05.008&amp;cf=fulltext&amp;site=jmdi-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_23_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=10.1016%2Fj.jmoldx.2015.05.008&amp;cf=pdf&amp;site=jmdi-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_23_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-84943630815" target="_blank" aria-label="Open Scopus in new tab">Scopus (42)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26322950/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.jmoldx.2015.05.008&amp;pmid=26322950" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib24" class="citations"><div id="sref24" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref24" title="View in article">24.</a></div><div>Holmes, Jr., J. ∙ Clark, S. ∙ Modrich, P.</div><div><strong>Strand-specific mismatch correction in nuclear extracts of human and Drosophila melanogaster cell lines</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 1990; <strong>87</strong>:5837-5841</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.87.15.5837" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-0025340001" target="_blank" aria-label="Open Scopus in new tab">Scopus (362)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/2116007/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.87.15.5837&amp;pmid=2116007" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib25" class="citations"><div id="sref25" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref25-1" title="View in article">25.</a></div><div>Hussmann, J.A. ∙ Ling, J. ∙ Ravisankar, P. ...</div><div><strong>Mapping the genetic landscape of DNA double-strand break repair</strong></div><div><em>Cell.</em> 2021;</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=10.1016%2Fj.cell.2021.10.002&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=10.1016%2Fj.cell.2021.10.002&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-85117855979" target="_blank" aria-label="Open Scopus in new tab">Scopus (80)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/34653350/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2021.10.002&amp;pmid=34653350" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib26" class="citations"><div id="sref26" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref26" title="View in article">26.</a></div><div>Iaccarino, I. ∙ Marra, G. ∙ Palombo, F. ...</div><div><strong>hMSH2 and hMSH6 play distinct roles in mismatch binding and contribute differently to the ATPase activity of hMutSalpha</strong></div><div><em>EMBO J.</em> 1998; <strong>17</strong>:2677-2686</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/emboj/17.9.2677" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_26_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-0032079736" target="_blank" aria-label="Open Scopus in new tab">Scopus (145)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/9564049/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Femboj%2F17.9.2677&amp;pmid=9564049" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib27" class="citations"><div id="sref27" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref27-1" title="View in article">27.</a></div><div>Ingram, V.M.</div><div><strong>A specific chemical difference between the globins of normal human and sickle-cell anaemia haemoglobin</strong></div><div><em>Nature.</em> 1956; <strong>178</strong>:792-794</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/178792a0" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-0000420850" target="_blank" aria-label="Open Scopus in new tab">Scopus (506)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/13369537/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2F178792a0&amp;pmid=13369537" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib28" class="citations"><div id="sref28" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref28-1" title="View in article">28.</a></div><div>Iyer, R.R. ∙ Pluciennik, A. ∙ Burdett, V. ...</div><div><strong>DNA mismatch repair: functions and mechanisms</strong></div><div><em>Chem. Rev.</em> 2006; <strong>106</strong>:302-323</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/cr0404794" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-33644626371" target="_blank" aria-label="Open Scopus in new tab">Scopus (732)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16464007/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Fcr0404794&amp;pmid=16464007" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib29" class="citations"><div id="sref29" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref29" title="View in article">29.</a></div><div>Jin, S. ∙ Lin, Q. ∙ Luo, Y. ...</div><div><strong>Genome-wide specificity of prime editors in plants</strong></div><div><em>Nat. Biotechnol.</em> 2021;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41587-021-00891-x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-85104810992" target="_blank" aria-label="Open Scopus in new tab">Scopus (87)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41587-021-00891-x" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib30" class="citations"><div id="sref30" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref30-1" title="View in article">30.</a></div><div>Kadyrov, F.A. ∙ Dzantiev, L. ∙ Constantin, N. ...</div><div><strong>Endonucleolytic function of MutLalpha in human mismatch repair</strong></div><div><em>Cell.</em> 2006; <strong>126</strong>:297-308</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=10.1016%2Fj.cell.2006.05.039&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=10.1016%2Fj.cell.2006.05.039&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-33746189409" target="_blank" aria-label="Open Scopus in new tab">Scopus (499)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16873062/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2006.05.039&amp;pmid=16873062" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib31" class="citations"><div id="sref31" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref31" title="View in article">31.</a></div><div>Kim, D.Y. ∙ Moon, S.B. ∙ Ko, J.-H. ...</div><div><strong>Unbiased investigation of specificities of prime editing systems in human cells</strong></div><div><em>Nucleic Acids Res.</em> 2020; <strong>48</strong>:10576-10589</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gkaa764" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-85092749873" target="_blank" aria-label="Open Scopus in new tab">Scopus (98)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32941652/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgkaa764&amp;pmid=32941652" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib32" class="citations"><div id="sref32" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref32-1" title="View in article">32.</a></div><div>Kunkel, T.A. ∙ Erie, D.A.</div><div><strong>DNA mismatch repair</strong></div><div><em>Annu. Rev. Biochem.</em> 2005; <strong>74</strong>:681-710</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1146/annurev.biochem.74.082803.133243" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-22244492665" target="_blank" aria-label="Open Scopus in new tab">Scopus (1084)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15952900/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1146%2Fannurev.biochem.74.082803.133243&amp;pmid=15952900" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib33" class="citations"><div id="sref33" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref33" title="View in article">33.</a></div><div>Lahue, R.S. ∙ Au, K.G. ∙ Modrich, P.</div><div><strong>DNA mismatch correction in a defined system</strong></div><div><em>Science.</em> 1989; <strong>245</strong>:160-164</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.2665076" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-0024396565" target="_blank" aria-label="Open Scopus in new tab">Scopus (474)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/2665076/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.2665076&amp;pmid=2665076" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib34" class="citations"><div id="sref34" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref34-1" title="View in article">34.</a></div><div>Li, G.-M.</div><div><strong>Mechanisms and functions of DNA mismatch repair</strong></div><div><em>Cell Res.</em> 2008; <strong>18</strong>:85-98</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/cr.2007.115" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-38049125557" target="_blank" aria-label="Open Scopus in new tab">Scopus (1020)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/18157157/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fcr.2007.115&amp;pmid=18157157" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib35" class="citations"><div id="sref35" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref35" title="View in article">35.</a></div><div>Lin, Q. ∙ Zong, Y. ∙ Xue, C. ...</div><div><strong>Prime genome editing in rice and wheat</strong></div><div><em>Nat. Biotechnol.</em> 2020; <strong>38</strong>:582-585</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41587-020-0455-x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-85081736686" target="_blank" aria-label="Open Scopus in new tab">Scopus (528)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32393904/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41587-020-0455-x&amp;pmid=32393904" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib36" class="citations"><div id="sref36" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref36-1" title="View in article">36.</a></div><div>Liu, S. ∙ Wang, Q. ∙ Yu, X. ...</div><div><strong>HIV-1 inhibition in cells with CXCR4 mutant genome created by CRISPR-Cas9 and piggyBac recombinant technologies</strong></div><div><em>Sci. Rep.</em> 2018; <strong>8</strong>:8573</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41598-018-26894-4" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-85048116209" target="_blank" aria-label="Open Scopus in new tab">Scopus (28)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29872154/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41598-018-26894-4&amp;pmid=29872154" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib73" class="citations"><div id="optVS2MxfBOKg" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-optVS2MxfBOKg" title="View in article">37.</a></div><div>Liu, Y. ∙ Kao, H.-I. ∙ Bambara, R.A.</div><div><strong>Flap endonuclease 1: a central component of DNA metabolism</strong></div><div><em>Annu. Rev. Biochem.</em> 2004; <strong>73</strong>:589-615</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1146/annurev.biochem.73.012803.092453" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-3943086339" target="_blank" aria-label="Open Scopus in new tab">Scopus (320)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15189154/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1146%2Fannurev.biochem.73.012803.092453&amp;pmid=15189154" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib37" class="citations"><div id="sref37" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref37" title="View in article">38.</a></div><div>Liu, Y. ∙ Li, X. ∙ He, S. ...</div><div><strong>Efficient generation of mouse models with the prime editing system</strong></div><div><em>Cell Discov.</em> 2020; <strong>6</strong>:27</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41421-020-0165-z" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_38_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-85083988172" target="_blank" aria-label="Open Scopus in new tab">Scopus (143)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32351707/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41421-020-0165-z&amp;pmid=32351707" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib38" class="citations"><div id="sref38" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref38-1" title="View in article">39.</a></div><div>Liu, P. ∙ Liang, S.-Q. ∙ Zheng, C. ...</div><div><strong>Improved prime editors enable pathogenic allele correction and cancer modelling in adult mice</strong></div><div><em>Nat. Commun.</em> 2021; <strong>12</strong>:2121</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41467-021-22295-w" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-85104130567" target="_blank" aria-label="Open Scopus in new tab">Scopus (163)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33837189/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41467-021-22295-w&amp;pmid=33837189" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib39" class="citations"><div id="sref39" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref39-1" title="View in article">40.</a></div><div>Lujan, S.A. ∙ Clausen, A.R. ∙ Clark, A.B. ...</div><div><strong>Heterogeneous polymerase fidelity and mismatch repair bias genome variation and composition</strong></div><div><em>Genome Res.</em> 2014; <strong>24</strong>:1751-1764</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/gr.178335.114" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_40_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-84923948943" target="_blank" aria-label="Open Scopus in new tab">Scopus (123)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25217194/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2Fgr.178335.114&amp;pmid=25217194" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib40" class="citations"><div id="sref40" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref40" title="View in article">41.</a></div><div>Matheson, E.C. ∙ Hall, A.G.</div><div><strong>Assessment of mismatch repair function in leukaemic cell lines and blasts from children with acute lymphoblastic leukaemia</strong></div><div><em>Carcinogenesis.</em> 2003; <strong>24</strong>:31-38</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/carcin/24.1.31" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-0037275722" target="_blank" aria-label="Open Scopus in new tab">Scopus (43)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/12538346/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fcarcin%2F24.1.31&amp;pmid=12538346" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib41" class="citations"><div id="sref41" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref41" title="View in article">42.</a></div><div>Mead, S. ∙ Whitfield, J. ∙ Poulter, M. ...</div><div><strong>A novel protective prion protein variant that colocalizes with kuru exposure</strong></div><div><em>N.&nbsp;Engl. J. Med.</em> 2009; <strong>361</strong>:2056-2065</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1056/NEJMoa0809716" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-70949091205" target="_blank" aria-label="Open Scopus in new tab">Scopus (146)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19923577/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1056%2FNEJMoa0809716&amp;pmid=19923577" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib42" class="citations"><div id="sref42" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref42-1" title="View in article">43.</a></div><div>Nelson, J.W. ∙ Randolph, P.B. ∙ Shen, S.P. ...</div><div><strong>Engineered pegRNAs improve prime editing efficiency</strong></div><div><em>Nat. Biotechnol.</em> 2021;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41587-021-01039-7" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-85116367544" target="_blank" aria-label="Open Scopus in new tab">Scopus (297)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/34608327/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41587-021-01039-7&amp;pmid=34608327" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib43" class="citations"><div id="sref43" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref43-1" title="View in article">44.</a></div><div>Olson, H.E. ∙ Demarest, S.T. ∙ Pestana-Knight, E.M. ...</div><div><strong>Cyclin-Dependent Kinase-Like 5 Deficiency Disorder: Clinical Review</strong></div><div><em>Pediatr. Neurol.</em> 2019; <strong>97</strong>:18-25</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=10.1016%2Fj.pediatrneurol.2019.02.015&amp;cf=fulltext&amp;site=pnu-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=10.1016%2Fj.pediatrneurol.2019.02.015&amp;cf=pdf&amp;site=pnu-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-85063410039" target="_blank" aria-label="Open Scopus in new tab">Scopus (144)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30928302/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.pediatrneurol.2019.02.015&amp;pmid=30928302" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib44" class="citations"><div id="sref44" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref44-1" title="View in article">45.</a></div><div>Park, S.-J. ∙ Jeong, T.Y. ∙ Shin, S.K. ...</div><div><strong>Targeted mutagenesis in mouse cells and embryos using an enhanced prime</strong></div><div><em>Genome Biol.</em> 2021; <strong>22</strong>:170</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13059-021-02389-w" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_45_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-85107198784" target="_blank" aria-label="Open Scopus in new tab">Scopus (70)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/34082781/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13059-021-02389-w&amp;pmid=34082781" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib45" class="citations"><div id="sref45" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref45" title="View in article">46.</a></div><div>Parsons, R. ∙ Li, G.-M. ∙ Longley, M.J. ...</div><div><strong>Hypermutability and mismatch repair deficiency in RER+ tumor cells</strong></div><div><em>Cell.</em> 1993; <strong>75</strong>:1227-1236</div></div><div class="external-links"><div class="core-xlink-abstract"><a href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=10.1016%2F0092-8674%2893%2990331-J&amp;cf=abstract&amp;site=cell-site">Abstract</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=10.1016%2F0092-8674%2893%2990331-J&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-0027137935" target="_blank" aria-label="Open Scopus in new tab">Scopus (992)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/8261516/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2F0092-8674%2893%2990331-J&amp;pmid=8261516" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib72" class="citations"><div id="optvIsUdM54jg" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-optvIsUdM54jg" title="View in article">47.</a></div><div>Pascal, J.M. ∙ O’Brien, P.J. ∙ Tomkinson, A.E. ...</div><div><strong>Human DNA ligase I completely encircles and partially unwinds nicked DNA</strong></div><div><em>Nature.</em> 2004; <strong>432</strong>:473-478</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature03082" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-9644289536" target="_blank" aria-label="Open Scopus in new tab">Scopus (274)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15565146/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature03082&amp;pmid=15565146" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib46" class="citations"><div id="sref46" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref46" title="View in article">48.</a></div><div>Peng, M. ∙ Xie, J. ∙ Ucher, A. ...</div><div><strong>Crosstalk between BRCA-Fanconi anemia and mismatch repair pathways prevents MSH2-dependent aberrant DNA damage responses</strong></div><div><em>EMBO J.</em> 2014; <strong>33</strong>:1698-1712</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.15252/embj.201387530" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_48_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-84905381475" target="_blank" aria-label="Open Scopus in new tab">Scopus (44)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24966277/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.15252%2Fembj.201387530&amp;pmid=24966277" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib47" class="citations"><div id="sref47" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref47" title="View in article">49.</a></div><div>Petri, K. ∙ Zhang, W. ∙ Ma, J. ...</div><div><strong>CRISPR prime editing with ribonucleoprotein complexes in zebrafish and primary human cells</strong></div><div><em>Nat. Biotechnol.</em> 2021;</div><div>Published online April 29, 2021</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41587-021-00901-y" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-85105157695" target="_blank" aria-label="Open Scopus in new tab">Scopus (119)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33927418/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41587-021-00901-y&amp;pmid=33927418" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib48" class="citations"><div id="sref48" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref48" title="View in article">50.</a></div><div>Plotz, G. ∙ Raedle, J. ∙ Brieger, A. ...</div><div><strong>N-terminus of hMLH1 confers interaction of hMutLalpha and hMutLbeta with hMutSalpha</strong></div><div><em>Nucleic Acids Res.</em> 2003; <strong>31</strong>:3217-3226</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gkg420" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-0037563786" target="_blank" aria-label="Open Scopus in new tab">Scopus (52)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/12799449/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgkg420&amp;pmid=12799449" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib49" class="citations"><div id="sref49" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref49" title="View in article">51.</a></div><div>Pluciennik, A. ∙ Dzantiev, L. ∙ Iyer, R.R. ...</div><div><strong>PCNA function in the activation and strand direction of MutLα endonuclease in mismatch repair</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2010; <strong>107</strong>:16066-16071</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1010662107" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_51_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-77957979862" target="_blank" aria-label="Open Scopus in new tab">Scopus (215)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20713735/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1010662107&amp;pmid=20713735" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib50" class="citations"><div id="sref50" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref50" title="View in article">52.</a></div><div>Ran, F.A. ∙ Cong, L. ∙ Yan, W.X. ...</div><div><strong>In&nbsp;vivo genome editing using Staphylococcus aureus Cas9</strong></div><div><em>Nature.</em> 2015; <strong>520</strong>:186-191</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature14299" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-84927514894" target="_blank" aria-label="Open Scopus in new tab">Scopus (2080)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25830891/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature14299&amp;pmid=25830891" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib51" class="citations"><div id="sref51" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref51" title="View in article">53.</a></div><div>Räschle, M. ∙ Dufner, P. ∙ Marra, G. ...</div><div><strong>Mutations within the hMLH1 and hPMS2 subunits of the human MutLalpha mismatch repair factor affect its ATPase activity, but not its ability to interact with hMutSalpha</strong></div><div><em>J.&nbsp;Biol. Chem.</em> 2002; <strong>277</strong>:21810-21820</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=10.1074%2Fjbc.M108787200&amp;cf=fulltext&amp;site=jbc-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=10.1074%2Fjbc.M108787200&amp;cf=pdf&amp;site=jbc-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-0037077309" target="_blank" aria-label="Open Scopus in new tab">Scopus (103)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11948175/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1074%2Fjbc.M108787200&amp;pmid=11948175" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib52" class="citations"><div id="sref52" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref52" title="View in article">54.</a></div><div>Schene, I.F. ∙ Joore, I.P. ∙ Oka, R. ...</div><div><strong>Prime editing for functional repair in patient-derived disease models</strong></div><div><em>Nat. Commun.</em> 2020; <strong>11</strong>:5352</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41467-020-19136-7" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-85093955574" target="_blank" aria-label="Open Scopus in new tab">Scopus (134)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33097693/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41467-020-19136-7&amp;pmid=33097693" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib53" class="citations"><div id="sref53" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref53-1" title="View in article">55.</a></div><div>Sockolosky, J.T. ∙ Trotta, E. ∙ Parisi, G. ...</div><div><strong>Selective targeting of engineered T&nbsp;cells using orthogonal IL-2 cytokine-receptor complexes</strong></div><div><em>Science.</em> 2018; <strong>359</strong>:1037-1042</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aar3246" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-85047997059" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29496879/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aar3246&amp;pmid=29496879" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib54" class="citations"><div id="sref54" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref54-1" title="View in article">56.</a></div><div>Spencer, J.M. ∙ Zhang, X.</div><div><strong>Deep mutational scanning of S. pyogenes Cas9 reveals important functional domains</strong></div><div><em>Sci. Rep.</em> 2017; <strong>7</strong>:16836</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41598-017-17081-y" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_56_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-85037086222" target="_blank" aria-label="Open Scopus in new tab">Scopus (34)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29203891/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41598-017-17081-y&amp;pmid=29203891" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib55" class="citations"><div id="sref55" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref55" title="View in article">57.</a></div><div>Strand, M. ∙ Prolla, T.A. ∙ Liskay, R.M. ...</div><div><strong>Destabilization of tracts of simple repetitive DNA in yeast by mutations affecting DNA mismatch repair</strong></div><div><em>Nature.</em> 1993; <strong>365</strong>:274-276</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/365274a0" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-0027306173" target="_blank" aria-label="Open Scopus in new tab">Scopus (982)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/8371783/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2F365274a0&amp;pmid=8371783" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib56" class="citations"><div id="sref56" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref56" title="View in article">58.</a></div><div>Su, S.S. ∙ Lahue, R.S. ∙ Au, K.G. ...</div><div><strong>Mispair specificity of methyl-directed DNA mismatch correction in&nbsp;vitro</strong></div><div><em>J.&nbsp;Biol. Chem.</em> 1988; <strong>263</strong>:6829-6835</div></div><div class="external-links"><div class="core-xlink-abstract"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=10.1016%2FS0021-9258%2818%2968718-6&amp;cf=abstract&amp;site=jbc-site" target="_blank" aria-label="Open Abstract in new tab">Abstract</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=10.1016%2FS0021-9258%2818%2968718-6&amp;cf=pdf&amp;site=jbc-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/2834393/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0021-9258%2818%2968718-6&amp;pmid=2834393" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib57" class="citations"><div id="sref57" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref57" title="View in article">59.</a></div><div>Sugawara, N. ∙ Goldfarb, T. ∙ Studamire, B. ...</div><div><strong>Heteroduplex rejection during single-strand annealing requires Sgs1 helicase and mismatch repair proteins Msh2 and Msh6 but not Pms1</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2004; <strong>101</strong>:9315-9320</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.0305749101" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-3042546122" target="_blank" aria-label="Open Scopus in new tab">Scopus (165)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15199178/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.0305749101&amp;pmid=15199178" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib58" class="citations"><div id="sref58" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref58" title="View in article">60.</a></div><div>Supek, F. ∙ Lehner, B.</div><div><strong>Differential DNA mismatch repair underlies mutation rate variation across the human genome</strong></div><div><em>Nature.</em> 2015; <strong>521</strong>:81-84</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature14173" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-84925615834" target="_blank" aria-label="Open Scopus in new tab">Scopus (245)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25707793/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature14173&amp;pmid=25707793" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib59" class="citations"><div id="sref59" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref59" title="View in article">61.</a></div><div>Sürün, D. ∙ Schneider, A. ∙ Mircetic, J. ...</div><div><strong>Efficient Generation and Correction of Mutations in Human iPS Cells Utilizing mRNAs of CRISPR Base Editors and Prime</strong></div><div><em>Genes (Basel).</em> 2020; <strong>11</strong>:511</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.3390/genes11050511" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-85084579488" target="_blank" aria-label="Open Scopus in new tab">Scopus (82)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32384610/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.3390%2Fgenes11050511&amp;pmid=32384610" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib60" class="citations"><div id="sref60" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref60-1" title="View in article">62.</a></div><div>Thomas, D.C. ∙ Roberts, J.D. ∙ Kunkel, T.A.</div><div><strong>Heteroduplex repair in extracts of human HeLa cells</strong></div><div><em>J.&nbsp;Biol. Chem.</em> 1991; <strong>266</strong>:3744-3751</div></div><div class="external-links"><div class="core-xlink-abstract"><a href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=10.1016%2FS0021-9258%2819%2967858-0&amp;cf=abstract&amp;site=jbc-site" target="_blank" aria-label="Open Abstract in new tab">Abstract</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=10.1016%2FS0021-9258%2819%2967858-0&amp;cf=pdf&amp;site=jbc-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/1995629/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0021-9258%2819%2967858-0&amp;pmid=1995629" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib61" class="citations"><div id="sref61" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref61" title="View in article">63.</a></div><div>Tomer, G. ∙ Buermeyer, A.B. ∙ Nguyen, M.M. ...</div><div><strong>Contribution of human mlh1 and pms2 ATPase activities to DNA mismatch repair</strong></div><div><em>J.&nbsp;Biol. Chem.</em> 2002; <strong>277</strong>:21801-21809</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=10.1074%2Fjbc.M111342200&amp;cf=fulltext&amp;site=jbc-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=10.1074%2Fjbc.M111342200&amp;cf=pdf&amp;site=jbc-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-0037077289" target="_blank" aria-label="Open Scopus in new tab">Scopus (51)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11897781/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1074%2Fjbc.M111342200&amp;pmid=11897781" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib62" class="citations"><div id="sref62" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref62" title="View in article">64.</a></div><div>Tran, H.T. ∙ Keen, J.D. ∙ Kricker, M. ...</div><div><strong>Hypermutability of homonucleotide runs in mismatch repair and DNA polymerase proofreading yeast mutants</strong></div><div><em>Mol. Cell. Biol.</em> 1997; <strong>17</strong>:2859-2865</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/MCB.17.5.2859" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_64_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-0030962035" target="_blank" aria-label="Open Scopus in new tab">Scopus (284)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/9111358/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FMCB.17.5.2859&amp;pmid=9111358" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib63" class="citations"><div id="sref63" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref63" title="View in article">65.</a></div><div>Trojan, J. ∙ Zeuzem, S. ∙ Randolph, A. ...</div><div><strong>Functional analysis of hMLH1 variants and HNPCC-related mutations using a human expression system</strong></div><div><em>Gastroenterology.</em> 2002; <strong>122</strong>:211-219</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=10.1053%2Fgast.2002.30296&amp;cf=fulltext&amp;site=ygast-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=10.1053%2Fgast.2002.30296&amp;cf=pdf&amp;site=ygast-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-0036143918" target="_blank" aria-label="Open Scopus in new tab">Scopus (178)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11781295/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1053%2Fgast.2002.30296&amp;pmid=11781295" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib64" class="citations"><div id="sref64" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref64-1" title="View in article">66.</a></div><div>Tsai, S.Q. ∙ Nguyen, N.T. ∙ Malagon-Lopez, J. ...</div><div><strong>CIRCLE-seq: a highly sensitive in&nbsp;vitro screen for genome-wide CRISPR-Cas9 nuclease off-targets</strong></div><div><em>Nat. Methods.</em> 2017; <strong>14</strong>:607-614</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.4278" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-85018272560" target="_blank" aria-label="Open Scopus in new tab">Scopus (522)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28459458/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.4278&amp;pmid=28459458" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib65" class="citations"><div id="sref65" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref65" title="View in article">67.</a></div><div>Umar, A. ∙ Boyer, J.C. ∙ Kunkel, T.A.</div><div><strong>DNA loop repair by human cell extracts</strong></div><div><em>Science.</em> 1994; <strong>266</strong>:814-816</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.7973637" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_67_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-0027988576" target="_blank" aria-label="Open Scopus in new tab">Scopus (125)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/7973637/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.7973637&amp;pmid=7973637" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib66" class="citations"><div id="sref66" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref66-1" title="View in article">68.</a></div><div>Umar, A. ∙ Boland, C.R. ∙ Terdiman, J.P. ...</div><div><strong>Revised Bethesda Guidelines for hereditary nonpolyposis colorectal cancer (Lynch syndrome) and microsatellite instability</strong></div><div><em>J.&nbsp;Natl. Cancer Inst.</em> 2004; <strong>96</strong>:261-268</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/jnci/djh034" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-10744233937" target="_blank" aria-label="Open Scopus in new tab">Scopus (2668)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/14970275/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fjnci%2Fdjh034&amp;pmid=14970275" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib67" class="citations"><div id="sref67" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref67-1" title="View in article">69.</a></div><div>Warren, J.J. ∙ Pohlhaus, T.J. ∙ Changela, A. ...</div><div><strong>Structure of the human MutSalpha DNA lesion recognition complex</strong></div><div><em>Mol. Cell.</em> 2007; <strong>26</strong>:579-592</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=10.1016%2Fj.molcel.2007.04.018&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=10.1016%2Fj.molcel.2007.04.018&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-34248572591" target="_blank" aria-label="Open Scopus in new tab">Scopus (286)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17531815/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.molcel.2007.04.018&amp;pmid=17531815" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib68" class="citations"><div id="sref68" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref68" title="View in article">70.</a></div><div>Wu, J. ∙ Corbett, A.H. ∙ Berland, K.M.</div><div><strong>The intracellular mobility of nuclear import receptors and NLS cargoes</strong></div><div><em>Biophys. J.</em> 2009; <strong>96</strong>:3840-3849</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=10.1016%2Fj.bpj.2009.01.050&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=10.1016%2Fj.bpj.2009.01.050&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-67650351106" target="_blank" aria-label="Open Scopus in new tab">Scopus (34)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19413990/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.bpj.2009.01.050&amp;pmid=19413990" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib69" class="citations"><div id="sref69" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref69" title="View in article">71.</a></div><div>Zhang, Y. ∙ Yuan, F. ∙ Presnell, S.R. ...</div><div><strong>Reconstitution of 5′-directed human mismatch repair in a purified system</strong></div><div><em>Cell.</em> 2005; <strong>122</strong>:693-705</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=10.1016%2Fj.cell.2005.06.027&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=10.1016%2Fj.cell.2005.06.027&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-24144447320" target="_blank" aria-label="Open Scopus in new tab">Scopus (312)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16143102/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2005.06.027&amp;pmid=16143102" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib70" class="citations"><div id="sref70" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref70" title="View in article">72.</a></div><div>Zhou, B.P. ∙ Liao, Y. ∙ Xia, W. ...</div><div><strong>Cytoplasmic localization of p21Cip1/WAF1 by Akt-induced phosphorylation in HER-2/neu-overexpressing cells</strong></div><div><em>Nat. Cell Biol.</em> 2001; <strong>3</strong>:245-252</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/35060032" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-0034745353" target="_blank" aria-label="Open Scopus in new tab">Scopus (950)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11231573/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2F35060032&amp;pmid=11231573" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib71" class="citations"><div id="sref71" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref71" title="View in article">73.</a></div><div>Zou, X. ∙ Koh, G.C.C. ∙ Nanda, A.S. ...</div><div><strong>A systematic CRISPR screen defines mutational mechanisms underpinning signatures caused by replication errors and endogenous DNA damage</strong></div><div><em>Nat. Cancer.</em> 2021; <strong>2</strong>:643-657</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s43018-021-00200-0" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.018&amp;key=2-s2.0-85105126717" target="_blank" aria-label="Open Scopus in new tab">Scopus (77)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/34164627/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs43018-021-00200-0&amp;pmid=34164627" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div></div></div></section></div></section><div class="core-linked-content">
    

        


    

            

            

            <div data-widget-id="8633d695-b258-4908-b86f-e2c268ec3c6b" data-widget-name="UX3HTMLWidget"><script type="text/javascript">
if($('.content-navigation__body__title').length){
$('.content-navigation__body__title').each(function(){
var html=this.innerHTML;
var div = document.createElement("div");
div.innerHTML = html;
var text = div.textContent || div.innerText || "";
this.innerHTML=text;
});
}
</script></div>

            

        

    

        


    

            

            

            <div data-widget-id="393effff-5b44-4681-a9b4-4f029b3540b7" data-widget-name="UX3HTMLWidget"><style>
.linkback-ad .pb-ad {
    display: block;
   height: auto;
    margin-left: 0px;
    margin-right: auto;
}
</style></div>

            

        

    

        

</div></div></div><div class="core-collateral-aside"><section id="core-collateral-figures"><h2>Figures (14)<button class="js--openFigureViewer"><span>Figure Viewer</span></button></h2><div class="core-collateral-aside-wrapper"><div class="grid"><a class="figure__open__ctrl" href="#fig1" aria-label="Go to Figure&amp;nbsp;1"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.09.018/asset/fcdc7b5a-d634-4ddd-ac1d-2c7e98c43fc8/main.assets/gr1.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs1" aria-label="Go to Figure&amp;nbsp;S1"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.09.018/asset/f48235dc-1c84-4b38-93cf-3568b4f3a08e/main.assets/figs1.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig2" aria-label="Go to Figure&amp;nbsp;2"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.09.018/asset/2086f07d-cded-4210-9f00-7debf551e233/main.assets/gr2.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs2" aria-label="Go to Figure&amp;nbsp;S2"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.09.018/asset/65551f90-539a-42a2-9f72-7120ba04f656/main.assets/figs2.jpg" aria-hidden="true"></a></div><div id="collapsible-figures" data-expandable="item"><div class="content grid" style="display:none;" role="region" id="collapsible-figures_content"><a class="figure__open__ctrl" href="#figs3" aria-label="Go to Figure&amp;nbsp;S3"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.09.018/asset/bd43f744-f2ab-4f8e-b309-a1dc5acd0c3a/main.assets/figs3.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig3" aria-label="Go to Figure&amp;nbsp;3"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.09.018/asset/81dee803-fbc3-4ff2-8c7d-a304954eca8c/main.assets/gr3.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig4" aria-label="Go to Figure&amp;nbsp;4"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.09.018/asset/41259a2d-72fb-4a2c-b56f-9873d66d71a0/main.assets/gr4.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs4" aria-label="Go to Figure&amp;nbsp;S4"><img class="figure-part" src="https://www.cell.com/cms/10.1016/j.cell.2021.09.018/asset/1105041c-7f09-4664-b000-13d8d69ef24d/main.assets/figs4a.jpg" aria-hidden="true"><img class="figure-part" src="https://www.cell.com/cms/10.1016/j.cell.2021.09.018/asset/def4be6f-52ae-455d-b3ca-873044f82453/main.assets/figs4b.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig5" aria-label="Go to Figure&amp;nbsp;5"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.09.018/asset/9d19d8b2-950f-4124-9d02-da2a6423f405/main.assets/gr5.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs5" aria-label="Go to Figure&amp;nbsp;S5"><img class="figure-part" src="https://www.cell.com/cms/10.1016/j.cell.2021.09.018/asset/738faa00-a2e7-48e3-820a-d2ab776f9f23/main.assets/figs5a.jpg" aria-hidden="true"><img class="figure-part" src="https://www.cell.com/cms/10.1016/j.cell.2021.09.018/asset/69edc1aa-0fe2-4917-aa70-6608b0b47ae2/main.assets/figs5b.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig6" aria-label="Go to Figure&amp;nbsp;6"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.09.018/asset/7d8cebb9-dbc1-4445-aeb1-c17537c6cfa0/main.assets/gr6.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs6" aria-label="Go to Figure&amp;nbsp;S6"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.09.018/asset/7a61dbc8-341f-4c05-af08-7e4504aee3df/main.assets/figs6.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig7" aria-label="Go to Figure&amp;nbsp;7"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.09.018/asset/8b8f0c17-531f-49d7-a07f-f3838af4c0c3/main.assets/gr7.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs7" aria-label="Go to Figure&amp;nbsp;S7"><img class="figure-part" src="https://www.cell.com/cms/10.1016/j.cell.2021.09.018/asset/67aeff51-5b03-434f-82ae-3a4e63d23fa7/main.assets/figs7a.jpg" aria-hidden="true"><img class="figure-part" src="https://www.cell.com/cms/10.1016/j.cell.2021.09.018/asset/3599711b-5357-4094-8b24-68122e677539/main.assets/figs7b.jpg" aria-hidden="true"></a></div><button class="trigger" aria-controls="collapsible-figures_content" aria-expanded="false" tabindex="0" role="button"><span class="expand">Show all figures</span><span class="collapse">Hide figures</span><i class="icon-section_arrow_d" aria-hidden="true"></i></button></div></div></section>
    

        


    

            

            

            <section id="core-collateral-metrics" class="publication-list__axel-section"><h2>Article metrics</h2><div><ul class="publication-list rlist publication-list--list plum-x--lazy-load publication-list--axel" data-plum-x-url="/pb/widgets/ux3/publicationList/plumx?widgetId=66dd7e11-7768-4006-93d8-a72b60e2a264&amp;pbContext=%3Bproduct%3Aproduct%3Aelsevier%5C%3Aproduct%5C%3Acell%3Bwebsite%3Awebsite%3Acell-site%3Bissue%3Aissue%3Apii%5C%3AS0092867420X00237%3Bctype%3Astring%3AJournal+Content%3Bjournal%3Ajournal%3Acell%3BpageGroup%3Astring%3AArticle+View%3Barticle%3Aarticle%3Apii%5C%3AS0092867421010655%3Bpage%3Astring%3AShow+Full+Text%3BrequestedJournal%3Ajournal%3Acell%3Bwgroup%3Astring%3ADefault+Website+Group%3BsubPage%3Astring%3AFull+Text" data-pb-mode-enabled="false"><li aria-label="loading articles" role="alert" class="js--loading plum-x--spinner hidden"></li></ul></div></section>

            

        
<section id="core-collateral-supplementary"><h2>Supplemental information (6)</h2><div class="core-collateral-aside-wrapper"><div class="core-supplementary-material"><div class="core-link"><button class="downloadAllSupplementary"><span>Download all</span></button></div></div><div class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2021.09.018/attachment/879f206c-bfba-4958-acf9-6aca66f01b28/mmc1.xlsx" download="mmc1.xlsx"><span id="mmc1-link">Spreadsheet (173.62 KB)</span></a><div class="core-description">Table S1. List of CRISPRi guide RNAs in prime editing Repair-seq libraries, related to Figures 1 and 2</div></div></div><div class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2021.09.018/attachment/17d80ebc-68d1-46a6-b9e0-e1210507bfb1/mmc2.xlsx" download="mmc2.xlsx"><span id="mmc2-link">Spreadsheet (250.52 KB)</span></a><div class="core-description">Table S2. Frequencies of the intended prime edit in Repair-seq screens, related to Figure&nbsp;1</div></div></div><div class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2021.09.018/attachment/8c8bb321-ccf0-4b5c-a2b6-42505cfdafdd/mmc3.xlsx" download="mmc3.xlsx"><span id="mmc3-link">Spreadsheet (152.70 KB)</span></a><div class="core-description">Table S3. List of sequencing data and guide RNAs from all experiments, related to Figures 1, 2, 3, 4, 5, 6, and 7</div></div></div><div class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2021.09.018/attachment/7b04d3bb-e5cd-4d87-b54c-4c632cdddd70/mmc4.xlsx" download="mmc4.xlsx"><span id="mmc4-link">Spreadsheet (13.47 KB)</span></a><div class="core-description">Table S4. Primers used for sequencing of genomic DNA and qRT-PCR, related to Figures 1, 2, 3, 4, 5, 6, and 7</div></div></div><div class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2021.09.018/attachment/217b0885-8ce2-42b6-a31a-3844b48d3bf0/mmc5.xlsx" download="mmc5.xlsx"><span id="mmc5-link">Spreadsheet (16.21 KB)</span></a><div class="core-description">Table S5. Nucleotide sequences of amplicons from high-throughput sequencing, related to Figures 1, 2, 3, 4, 5, 6, and 7</div></div></div><div id="collapsible-supplementary" data-expandable="item"><div class="content" style="display:none;" role="region" id="collapsible-supplementary_content"><div class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2021.09.018/attachment/59291224-27bc-4b94-987f-b5cf41a03035/mmc6.xlsx" download="mmc6.xlsx"><span id="mmc6-link">Spreadsheet (174.31 KB)</span></a><div class="core-description">Table S6. Number of cells and sequencing reads from Repair-seq screens, related to Figures 1 and 2</div></div></div></div><button class="trigger" aria-controls="collapsible-supplementary_content" aria-expanded="false" tabindex="0" role="button"><span class="expand">Show more</span><span class="collapse">Show less</span><i class="icon-section_arrow_d" aria-hidden="true"></i></button></div></div></section>
    

        


    

            

            

            <section id="core-collateral-relatedArticles" class="axel-related-articles"><h2 id="relatedArticles" class="top axel-related-articles__title" data-title="Related articles">Related Articles</h2><div data-pb-widget-type="ux3-relatedArticles" data-widget-id="8d3844d1-450e-497c-ab75-a63fc41750ac" data-widget-name="AtyponTempUX3RelatedArticlesWidget" data-article-pii="S0092867421010655" class="ux3DeferredOnScrollWidget axel-related-articles__body" data-show-more="true"></div></section>

            

        
</div></div><aside data-core-aside="right-rail"><div class="inner-wrapper">

    

        


    

            

            

            <div data-widget-id="dca7470d-9085-4d21-a6f4-a66397718146" data-widget-name="UX3HTMLWidget"><input type="hidden" id="gptPage" value="cell-videos"></div>

            

        











        <div data-widget-def="literatumAd" data-widget-id="481d10c2-bbe0-470e-a5ba-207a81ecc8dc" class="adplaceholders" style="right: 0px; position: static;">
        
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd alignCenter  widget-verticalAd" id="481d10c2-bbe0-470e-a5ba-207a81ecc8dc">
                
                
                    
                

                <div class="wrapped " id="Marketplace-Recs">
                    
                    <div class="widget-body body body-verticalAd "><div class="verticalAdContainer">
    <div class="verticalAdLabelContainer">
        <div class="verticalAdLabel">
                <img src="/templates/jsp/_style2/_marlin/images/bg_adVert.gif" alt="Advertisement">
        </div>
    </div>
    <div class="verticalAd">
        





        <div class="pb-ad">
            <script src="//cdn.pbgrd.com/core-cell.js" async=""></script>
<div id="pbgrd-skyr-c"><div id="pbgrd-skyr" style="width: inherit; position: relative; background: transparent;"></div></div>
        </div>
    

    </div>
</div></div>
                </div>
                </div>

        

        </div>
    

</div></aside><div class="core-authors-details" style="display:none"><section class="core-authors"></section></div></article><div data-extent="article-wrapper"><div class="core-container"></div></div><script id="axel-publication-metadata" type="application/json">{"doi":"10.1016/j.cell.2021.09.018","data-is-retraction":false,"data-is-retired":false}
</script><script type="text/javascript" data-ot-ignore="data-ot-ignore" defer="defer" src="https://static.addtoany.com/menu/page.js" class="optanon-category-C0004 ot-vscat-C0004"></script><template id="figure_nav_template"><a href="#" class="open-in-viewer btn btn-dark figure-pop-btn"><i aria-hidden="true" class="icon-export_citation"></i><span>Open in viewer</span></a></template><div id="figure-viewer" role="dialog" aria-modal="true" aria-label="Figure Viewer" class="showCaption showThumbstrip figure-viewer"><nav class="nav-bar figure-viewer__top"><a href="#" title="Close Figure Viewer" role="button" class="figure-viewer__close"><i aria-hidden="true" class="icon-tools_close"></i></a><div class="figure-viewer__title">Enhanced prime editing systems by manipulating cellular determinants of editing outcomes</div></nav><div class="frames"><ul class="frames__list"><li><div class="thumbnail__image"><img id="gr1" src="/cms/10.1016/j.cell.2021.09.018/asset/fcdc7b5a-d634-4ddd-ac1d-2c7e98c43fc8/main.assets/gr1.jpg" height="937" width="673" aria-labelledby="fig1-title fig1-content" data-viewer-src="/cms/10.1016/j.cell.2021.09.018/asset/b4877c18-909c-4973-8ea9-a51acde54482/main.assets/gr1_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs1" src="/cms/10.1016/j.cell.2021.09.018/asset/f48235dc-1c84-4b38-93cf-3568b4f3a08e/main.assets/figs1.jpg" height="922" width="670" aria-labelledby="figs1-title figs1-content" data-viewer-src="/cms/10.1016/j.cell.2021.09.018/asset/2e559411-23d0-4497-9e9c-8853f668cf79/main.assets/figs1_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr2" src="/cms/10.1016/j.cell.2021.09.018/asset/2086f07d-cded-4210-9f00-7debf551e233/main.assets/gr2.jpg" height="890" width="657" aria-labelledby="fig2-title fig2-content" data-viewer-src="/cms/10.1016/j.cell.2021.09.018/asset/18cdba41-5034-43d2-a986-d039a388efc9/main.assets/gr2_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs2" src="/cms/10.1016/j.cell.2021.09.018/asset/65551f90-539a-42a2-9f72-7120ba04f656/main.assets/figs2.jpg" height="940" width="656" aria-labelledby="figs2-title figs2-content" data-viewer-src="/cms/10.1016/j.cell.2021.09.018/asset/fbe51ba8-7942-45ef-802e-ebcd90dc4105/main.assets/figs2_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs3" src="/cms/10.1016/j.cell.2021.09.018/asset/bd43f744-f2ab-4f8e-b309-a1dc5acd0c3a/main.assets/figs3.jpg" height="942" width="684" aria-labelledby="figs3-title figs3-content" data-viewer-src="/cms/10.1016/j.cell.2021.09.018/asset/be01a2e8-7628-41cb-9824-8cb01efbb069/main.assets/figs3_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr3" src="/cms/10.1016/j.cell.2021.09.018/asset/81dee803-fbc3-4ff2-8c7d-a304954eca8c/main.assets/gr3.jpg" height="420" width="506" aria-labelledby="fig3-title fig3-content" data-viewer-src="/cms/10.1016/j.cell.2021.09.018/asset/c844bb6a-1910-40f5-a74f-bbd5029df4c4/main.assets/gr3_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr4" src="/cms/10.1016/j.cell.2021.09.018/asset/41259a2d-72fb-4a2c-b56f-9873d66d71a0/main.assets/gr4.jpg" height="575" width="657" aria-labelledby="fig4-title fig4-content" data-viewer-src="/cms/10.1016/j.cell.2021.09.018/asset/903a1a22-9ae6-4868-934d-609315242d1f/main.assets/gr4_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs4-part-1" src="/cms/10.1016/j.cell.2021.09.018/asset/1105041c-7f09-4664-b000-13d8d69ef24d/main.assets/figs4a.jpg" height="941" width="727" aria-labelledby="figs4-title figs4-content" data-viewer-src="/cms/10.1016/j.cell.2021.09.018/asset/f4704c9a-43fc-4ba0-8dab-aacf0bc36a8e/main.assets/figs4a_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs4-part-2" src="/cms/10.1016/j.cell.2021.09.018/asset/def4be6f-52ae-455d-b3ca-873044f82453/main.assets/figs4b.jpg" height="409" width="727" aria-labelledby="figs4-title figs4-content" data-viewer-src="/cms/10.1016/j.cell.2021.09.018/asset/caf4fb72-6947-4ab1-9938-5a3d3671900a/main.assets/figs4b_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr5" src="/cms/10.1016/j.cell.2021.09.018/asset/9d19d8b2-950f-4124-9d02-da2a6423f405/main.assets/gr5.jpg" height="942" width="710" aria-labelledby="fig5-title fig5-content" data-viewer-src="/cms/10.1016/j.cell.2021.09.018/asset/c2750039-f993-49c5-8328-6f871ddf0efc/main.assets/gr5_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs5-part-1" src="/cms/10.1016/j.cell.2021.09.018/asset/738faa00-a2e7-48e3-820a-d2ab776f9f23/main.assets/figs5a.jpg" height="899" width="753" aria-labelledby="figs5-title figs5-content" data-viewer-src="/cms/10.1016/j.cell.2021.09.018/asset/eee43b84-1b87-4b1e-99f1-e325892c10a7/main.assets/figs5a_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs5-part-2" src="/cms/10.1016/j.cell.2021.09.018/asset/69edc1aa-0fe2-4917-aa70-6608b0b47ae2/main.assets/figs5b.jpg" height="688" width="753" aria-labelledby="figs5-title figs5-content" data-viewer-src="/cms/10.1016/j.cell.2021.09.018/asset/97961f8f-82fd-4c7a-bcce-19b80c22e750/main.assets/figs5b_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr6" src="/cms/10.1016/j.cell.2021.09.018/asset/7d8cebb9-dbc1-4445-aeb1-c17537c6cfa0/main.assets/gr6.jpg" height="707" width="664" aria-labelledby="fig6-title fig6-content" data-viewer-src="/cms/10.1016/j.cell.2021.09.018/asset/52831411-0f4a-45f5-a69f-cbd01da763c8/main.assets/gr6_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs6" src="/cms/10.1016/j.cell.2021.09.018/asset/7a61dbc8-341f-4c05-af08-7e4504aee3df/main.assets/figs6.jpg" height="884" width="658" aria-labelledby="figs6-title figs6-content" data-viewer-src="/cms/10.1016/j.cell.2021.09.018/asset/b7404c92-728e-440b-8b0f-ee35d761ebc4/main.assets/figs6_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr7" src="/cms/10.1016/j.cell.2021.09.018/asset/8b8f0c17-531f-49d7-a07f-f3838af4c0c3/main.assets/gr7.jpg" height="743" width="764" aria-labelledby="fig7-title fig7-content" data-viewer-src="/cms/10.1016/j.cell.2021.09.018/asset/1d884b08-b74e-4fdc-9ff0-0753171843c1/main.assets/gr7_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs7-part-1" src="/cms/10.1016/j.cell.2021.09.018/asset/67aeff51-5b03-434f-82ae-3a4e63d23fa7/main.assets/figs7a.jpg" height="941" width="697" aria-labelledby="figs7-title figs7-content" data-viewer-src="/cms/10.1016/j.cell.2021.09.018/asset/32155e21-fc16-4d0f-8199-10680fb6ca49/main.assets/figs7a_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs7-part-2" src="/cms/10.1016/j.cell.2021.09.018/asset/3599711b-5357-4094-8b24-68122e677539/main.assets/figs7b.jpg" height="375" width="697" aria-labelledby="figs7-title figs7-content" data-viewer-src="/cms/10.1016/j.cell.2021.09.018/asset/ae5dd588-664e-47ad-8bf2-9671ab209637/main.assets/figs7b_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li></ul><a href="#" title="Previous Frame" class="frameButton previousFrameButton frames__prev"><!--span Previous Frame--><i class="icon-arrow_l"></i></a><a href="#" title="Next Frame" class="frameButton nextFrameButton frames__next"><!--span Next Frame--><i class="icon-arrow_r"></i></a></div><div class="caption"><div class="figure-viewer__caption-body"></div></div><div class="toolbar figure-viewer__toolbar"><span class="figure-viewer__counter"></span><a href="#" class="button toggle-caption-button figure-viewer__caption__ctrl"><i aria-hidden="true" class="icon icon-info"></i><span class="label">Hide Caption</span></a><a href="#" class="button download-options-button"><i aria-hidden="true" class="icon icon-gizmo-folder-in download-size-icon"></i><span class="label">Download</span></a><a href="#figures" class="button see-figures figure-viewer__see-figure__ctrl"><i aria-hidden="true" class="icon icon-gizmo-image_figure viewer-size-icon"></i><span class="label">See figure in Article</span></a></div><div class="thumbstrip"><a href="#" class="toggleButton thumbstrip__ctrl"><span>Toggle Thumbstrip</span></a><a href="#" title="Previous Thumbnails" class="thumbstripButton previousThumbstripButton thumbstrip__prev"><i aria-hidden="true" class="icon-arrow_l"></i></a><a href="#" title="Next Thumbnails" class="thumbstripButton nextThumbstripButton thumbstrip__next"><i aria-hidden="true" class="icon-arrow_r"></i></a><div class="thumbstrip__wrapper"><ul class="rlist figure-viewer__thumbslist"><li><div class="thumbnail__label">Figure&nbsp;1</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr1" src="/cms/10.1016/j.cell.2021.09.018/asset/fcdc7b5a-d634-4ddd-ac1d-2c7e98c43fc8/main.assets/gr1.jpg" height="937" width="673" aria-labelledby="fig1-title fig1-content" data-viewer-src="/cms/10.1016/j.cell.2021.09.018/asset/b4877c18-909c-4973-8ea9-a51acde54482/main.assets/gr1_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S1</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs1" src="/cms/10.1016/j.cell.2021.09.018/asset/f48235dc-1c84-4b38-93cf-3568b4f3a08e/main.assets/figs1.jpg" height="922" width="670" aria-labelledby="figs1-title figs1-content" data-viewer-src="/cms/10.1016/j.cell.2021.09.018/asset/2e559411-23d0-4497-9e9c-8853f668cf79/main.assets/figs1_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;2</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr2" src="/cms/10.1016/j.cell.2021.09.018/asset/2086f07d-cded-4210-9f00-7debf551e233/main.assets/gr2.jpg" height="890" width="657" aria-labelledby="fig2-title fig2-content" data-viewer-src="/cms/10.1016/j.cell.2021.09.018/asset/18cdba41-5034-43d2-a986-d039a388efc9/main.assets/gr2_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S2</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs2" src="/cms/10.1016/j.cell.2021.09.018/asset/65551f90-539a-42a2-9f72-7120ba04f656/main.assets/figs2.jpg" height="940" width="656" aria-labelledby="figs2-title figs2-content" data-viewer-src="/cms/10.1016/j.cell.2021.09.018/asset/fbe51ba8-7942-45ef-802e-ebcd90dc4105/main.assets/figs2_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S3</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs3" src="/cms/10.1016/j.cell.2021.09.018/asset/bd43f744-f2ab-4f8e-b309-a1dc5acd0c3a/main.assets/figs3.jpg" height="942" width="684" aria-labelledby="figs3-title figs3-content" data-viewer-src="/cms/10.1016/j.cell.2021.09.018/asset/be01a2e8-7628-41cb-9824-8cb01efbb069/main.assets/figs3_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;3</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr3" src="/cms/10.1016/j.cell.2021.09.018/asset/81dee803-fbc3-4ff2-8c7d-a304954eca8c/main.assets/gr3.jpg" height="420" width="506" aria-labelledby="fig3-title fig3-content" data-viewer-src="/cms/10.1016/j.cell.2021.09.018/asset/c844bb6a-1910-40f5-a74f-bbd5029df4c4/main.assets/gr3_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;4</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr4" src="/cms/10.1016/j.cell.2021.09.018/asset/41259a2d-72fb-4a2c-b56f-9873d66d71a0/main.assets/gr4.jpg" height="575" width="657" aria-labelledby="fig4-title fig4-content" data-viewer-src="/cms/10.1016/j.cell.2021.09.018/asset/903a1a22-9ae6-4868-934d-609315242d1f/main.assets/gr4_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S4</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs4-part-1" src="/cms/10.1016/j.cell.2021.09.018/asset/1105041c-7f09-4664-b000-13d8d69ef24d/main.assets/figs4a.jpg" height="941" width="727" aria-labelledby="figs4-title figs4-content" data-viewer-src="/cms/10.1016/j.cell.2021.09.018/asset/f4704c9a-43fc-4ba0-8dab-aacf0bc36a8e/main.assets/figs4a_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S4</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs4-part-2" src="/cms/10.1016/j.cell.2021.09.018/asset/def4be6f-52ae-455d-b3ca-873044f82453/main.assets/figs4b.jpg" height="409" width="727" aria-labelledby="figs4-title figs4-content" data-viewer-src="/cms/10.1016/j.cell.2021.09.018/asset/caf4fb72-6947-4ab1-9938-5a3d3671900a/main.assets/figs4b_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;5</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr5" src="/cms/10.1016/j.cell.2021.09.018/asset/9d19d8b2-950f-4124-9d02-da2a6423f405/main.assets/gr5.jpg" height="942" width="710" aria-labelledby="fig5-title fig5-content" data-viewer-src="/cms/10.1016/j.cell.2021.09.018/asset/c2750039-f993-49c5-8328-6f871ddf0efc/main.assets/gr5_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S5</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs5-part-1" src="/cms/10.1016/j.cell.2021.09.018/asset/738faa00-a2e7-48e3-820a-d2ab776f9f23/main.assets/figs5a.jpg" height="899" width="753" aria-labelledby="figs5-title figs5-content" data-viewer-src="/cms/10.1016/j.cell.2021.09.018/asset/eee43b84-1b87-4b1e-99f1-e325892c10a7/main.assets/figs5a_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S5</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs5-part-2" src="/cms/10.1016/j.cell.2021.09.018/asset/69edc1aa-0fe2-4917-aa70-6608b0b47ae2/main.assets/figs5b.jpg" height="688" width="753" aria-labelledby="figs5-title figs5-content" data-viewer-src="/cms/10.1016/j.cell.2021.09.018/asset/97961f8f-82fd-4c7a-bcce-19b80c22e750/main.assets/figs5b_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;6</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr6" src="/cms/10.1016/j.cell.2021.09.018/asset/7d8cebb9-dbc1-4445-aeb1-c17537c6cfa0/main.assets/gr6.jpg" height="707" width="664" aria-labelledby="fig6-title fig6-content" data-viewer-src="/cms/10.1016/j.cell.2021.09.018/asset/52831411-0f4a-45f5-a69f-cbd01da763c8/main.assets/gr6_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S6</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs6" src="/cms/10.1016/j.cell.2021.09.018/asset/7a61dbc8-341f-4c05-af08-7e4504aee3df/main.assets/figs6.jpg" height="884" width="658" aria-labelledby="figs6-title figs6-content" data-viewer-src="/cms/10.1016/j.cell.2021.09.018/asset/b7404c92-728e-440b-8b0f-ee35d761ebc4/main.assets/figs6_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;7</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr7" src="/cms/10.1016/j.cell.2021.09.018/asset/8b8f0c17-531f-49d7-a07f-f3838af4c0c3/main.assets/gr7.jpg" height="743" width="764" aria-labelledby="fig7-title fig7-content" data-viewer-src="/cms/10.1016/j.cell.2021.09.018/asset/1d884b08-b74e-4fdc-9ff0-0753171843c1/main.assets/gr7_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S7</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs7-part-1" src="/cms/10.1016/j.cell.2021.09.018/asset/67aeff51-5b03-434f-82ae-3a4e63d23fa7/main.assets/figs7a.jpg" height="941" width="697" aria-labelledby="figs7-title figs7-content" data-viewer-src="/cms/10.1016/j.cell.2021.09.018/asset/32155e21-fc16-4d0f-8199-10680fb6ca49/main.assets/figs7a_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S7</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs7-part-2" src="/cms/10.1016/j.cell.2021.09.018/asset/3599711b-5357-4094-8b24-68122e677539/main.assets/figs7b.jpg" height="375" width="697" aria-labelledby="figs7-title figs7-content" data-viewer-src="/cms/10.1016/j.cell.2021.09.018/asset/ae5dd588-664e-47ad-8bf2-9671ab209637/main.assets/figs7b_lrg.jpg" loading="lazy" tabindex="0"></div></div></li></ul></div></div><div class="download figure-viewer__download"><div class="pointer"></div><ul class="rlist downlad__wrapper"><li class="hidden-xs hidden-sm"><a href="/action/downloadFigures?pii=e00489&amp;id=" class="downloadAllButton"><i class="icon icon-gizmo-ppt figure-viewer__download__viewer-size-icon figure-viewer__download__viewer-size-icon--ppt"></i><span class="figure-viewer__download__text"> Download .PPT</span></a></li></ul></div></div><template id="toCitationLink" data-citation="Citation" data-footnote="Footnote"><div class="to-citation__wrapper"><a class="to-citation"><i aria-hidden="true" class="icon-subdirectory_arrow_left"></i><span>Go to</span></a></div></template><template id="toCitationButton" data-citation="Citation" data-footnote="Footnote"><div class="to-citation__wrapper"><button class="to-citation"><i aria-hidden="true" class="icon-subdirectory_arrow_left"></i><span>Go to</span></button></div></template><template id="toCitationAccordion" data-collapse-citation="Hide Citations" data-expand-citation="Show Citations" data-collapse-footnote="Hide Footnotes" data-expand-footnote="Show Footnotes"><div class="to-citation__wrapper"><button aria-controls="aria-controls" aria-expanded="false" aria-label="Toggle citations menu" class="to-citation__toggle"><i aria-hidden="true" class="icon-plus"></i><span data-expand-title="data-expand-title" data-collapsed-title="data-collapsed-title" class="accordion__toggle__title"></span></button><div role="menu" aria-label="links" class="to-citation__accordion no-separator"><ul></ul></div></div></template><template id="toCitationAccordionItem"><li><a class="to-citation"><i aria-hidden="true" class="icon-subdirectory_arrow_left"></i><span></span></a></li></template><template id="citations_truncate_template"><div class="citations-truncation"><button data-label-expand="Show all references" data-label-collapse="Show fewer" data-label-remaining="references remaining" class="btn btn--inverse"><span>Show all references</span></button></div></template><template id="collateral_texts_template" data-do="Digital Object" data-figures="Figures" data-metrics="Article Metrics" data-related="Related articles" data-supplementary="Supplemental information"></template><template id="collapsible_authors_template"><button aria-expanded="false" data-expandable="all" data-label-expand="Expand All" data-label-collapse="Collapse All" class="collateral-contributors-control"><span>Expand All</span></button></template>
<template id="collapsible_tables_collapse_template"><div class="collapsible-figure-btn__wrapper expanded"><button aria-expanded="true" class="btn collapsible-figure-btn btn--inverse"><i aria-hidden="true" class="icon-arrow-up"></i><span class="text-uppercase">Collapse</span></button></div></template>
<template id="collapsible_tables_expand_template"><div class="collapsible-figure-btn__wrapper collapsed"><button aria-expanded="false" class="btn collapsible-figure-btn btn--inverse"><i aria-hidden="true" class="icon-arrow-down"></i><span class="text-uppercase">Expand Table</span></button></div></template><template id="authorsAffiliationsLink"><a href="#tab-contributors" class="to-authors-affiliations">Authors Info &amp; Affiliations</a></template></div>
                </div>
                </div>

        

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none  widget-  widget-compact-all" id="756cf18d-e910-4468-936c-0225ae7c669f">
                
                
                    
                

                <div class="wrapped " id="ad for nucleus-single-molecule-technologies">
                    
                    <div class="widget-body body body-  body-compact-all"><div class="pb-ad">
            <script type="text/javascript">
URL_var = 'https://www.cell.com/cell/fulltext/S0092-8674(21)01065-5';

if(URL_var.indexOf("S0165-6147(17)30205-5") > -1
|| URL_var.indexOf("S0167-7799(18)30204-X") > -1 
|| URL_var.indexOf("S0092-8674(18)31649-0") > -1 
|| URL_var.indexOf("S0092-8674(19)30404-0") > -1 
|| URL_var.indexOf("S1097-2765(18)30756-1") > -1 
|| URL_var.indexOf("S0092-8674(18)31452-1") > -1 
|| URL_var.indexOf("S1097-2765(19)30175-3") > -1 
|| URL_var.indexOf("S0092-8674(18)30852-3") > -1 
|| URL_var.indexOf("S0006-3495(18)30295-9") > -1 
|| URL_var.indexOf("S1097-2765(19)30063-2") > -1 
|| URL_var.indexOf("S0006-3495(17)33208-3") > -1 
|| URL_var.indexOf("S2211-1247(18)31639-5") > -1 
|| URL_var.indexOf("S1097-2765(18)30978-X") > -1 
)
    {
document.write('<input type="hidden" id="gpttc" value="nucleus-single-molecule-technologies" class="gpt-Target">')
}
</script>
        </div></div>
                </div>
                </div>

        

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none  widget-" id="754b1780-1117-4371-9285-9cd7020963d5">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body- "><div class="pb-ad">
            <script type="text/javascript">
URL_var = 'https://www.cell.com/cell/fulltext/S0092-8674(21)01065-5';

if(URL_var.indexOf("S1535-6108(19)30104-7") > -1
|| URL_var.indexOf("S0092-8674(18)31394-1") > -1
|| URL_var.indexOf("S1074-7613(19)30033-0") > -1
|| URL_var.indexOf("S1535-6108(18)30473-2") > -1
|| URL_var.indexOf("S0092-8674(19)30154-0") > -1
|| URL_var.indexOf("S1550-4131(18)30178-5") > -1
|| URL_var.indexOf("S1535-6108(19)30242-9") > -1
|| URL_var.indexOf("S1074-7613(18)30431-X") > -1
|| URL_var.indexOf("S0092-8674(19)30618-X") > -1
|| URL_var.indexOf("S1535-6108(18)30472-0") > -1
|| URL_var.indexOf("S1535-6108(18)30120-X") > -1
|| URL_var.indexOf("S0092-8674(18)31247-9") > -1
|| URL_var.indexOf("S1074-7613(18)30085-2") > -1
|| URL_var.indexOf("S1074-7613(18)30080-3") > -1
|| URL_var.indexOf("S1471-4906(18)30210-2") > -1
|| URL_var.indexOf("S1550-4131(19)30304-3") > -1
|| URL_var.indexOf("S1934-5909(18)30599-X") > -1
|| URL_var.indexOf("S2405-8033(19)30024-X") > -1
|| URL_var.indexOf("S1535-6108(18)30109-0") > -1
|| URL_var.indexOf("S0962-8924(18)30127-2") > -1
)
    {
document.write('<input type="hidden" id="gpttc" value="nucleus-tumor-immune-microenvironment" class="gpt-Target">')
}
</script>
        </div></div>
                </div>
                </div>

        

    

        


    

            

            

            <div data-widget-id="9c9ff7a1-a035-49f6-a336-600c9419c518" data-widget-name="UX3HTMLWidget"><!-- Start of HubSpot Embed Code --> 

<script type="text/plain" class="optanon-category-4" id="hs-script-loader" async="" defer="" src="//js.hs-scripts.com/519224.js"></script> 

<!-- End of HubSpot Embed Code --></div>

            

        

    

        


    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget sessionValidationWidget none  widget-none  widget-compact-all" id="0257aef7-6074-4898-a989-1c3f693225a2">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none  body-compact-all"><script>
    $.ajax({
        url: "https://www.cell.com/ssoSessionCheck",
        type: "GET",
        dataType: "jsonp",
        success: function (data) {
            if(data.RefreshSession) {
                location.reload();
            }
        },
        error : function (){
            console.warn("can't validate CSAS session.");
        }
    });
</script></div>
                </div>
                </div>

        

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget csasSsoUrls none  widget-none  widget-compact-all" id="948bc5c4-8b0d-47be-b5ad-6570559802d1">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none  body-compact-all"><!-- clear SSO URLs after all iFrames are rendered or after passing a time out-->
<script>
    setTimeout(clearSsoUrlsFun,10000);
    var iFramesList = document.getElementById("sso-urls-div").getElementsByTagName("iframe");
    var renderedIFramesCount = 0;
    var numberOfIFrames = iFramesList.length;
    var skipRefresh = false;
    for (var i = 0; i < iFramesList.length; i++) {
        var iFrame = iFramesList[i];
        iFrame.addEventListener("load", function() {
            renderedIFramesCount = renderedIFramesCount + 1;
            if (renderedIFramesCount >= numberOfIFrames)
            {
                clearSsoUrlsFun();
            }
        });
    }

    function clearSsoUrlsFun() {
        if (skipRefresh == false && numberOfIFrames > 0)
        {
            skipRefresh = true;
            $.ajax({
                url: "https://www.cell.com/action/clearSsoUrls",
                type: "GET",
                dataType: "jsonp",
                error : errorCallingAjx
            });
        }
    }

    function errorCallingAjx(){
        console.log("can't delete sso urls .")
    }
</script></div>
                </div>
                </div>

        










        <div data-widget-def="ux3-layout-widget" data-widget-id="d95b4a46-dc4d-4ca8-83f9-8b6724c649ec" class="ux3 ">
        
    

        


    

            

            

            <div data-widget-id="d95b4a46-dc4d-4ca8-83f9-8b6724c649ec" data-widget-name="ux3-layout-widget" class="fonts-loaded"><div class="footer clearfix">
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none adplaceholder adplaceholder--header widget-horizontalAd" id="a07ee92b-3942-49f8-9186-a2a08fc3fc56">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-horizontalAd "><div class="horizontalAdContainer">
    <div class="horizontalAdLabelContainer">
        <div class="horizontalAdLabel">
                <img src="/templates/jsp/_style2/_marlin/images/bg_adHoriz.gif" alt="Advertisement">
        </div>
    </div>
    <div class="horizontalAd">
        





        <div class="pb-ad">
            <!-- Beginning Async AdSlot 4 for Ad unit els.cellpress/cell  ### size: [[728,90]] -->
<div class="gpt-ad" data-adsize="cell-leaderboard" data-page="" data-site="cell" data-pos="4"></div>
<!-- End AdSlot 4 -->
        </div>
    

    </div>
</div></div>
                </div>
                </div>

        

    

        


    

            

            

            <div data-widget-id="23703bbc-af46-4f3f-a5bb-dd52891ff0ff" data-widget-name="UX3HTMLWidget"><div class="footer__top">
    <div class="footer__logo centered"><a href="/home" title="Cell Press"><!-- Generator: Adobe Illustrator 25.2.1, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 300 72" style="enable-background:new 0 0 300 72;" xml:space="preserve">
<style type="text/css">
	.st0{fill:#007DBC;}
</style>
<g>
	<path class="st0" d="M39.4,7.5c-2.3-0.3-4.4,1.1-4.9,3.3l-1.8,7.4c-0.4,1.8-1.9,3.1-3.8,3.3l-10.5,1.2C7.1,24-1,34.3,0.7,45.6
		c1.3,9,8.6,15.7,17.2,17c2.3,0.3,4.4-1.1,4.9-3.3l1.8-7.4c0.4-1.8,1.9-3.1,3.8-3.3l10.5-1.2c11.3-1.2,19.4-11.6,17.8-22.9
		C55.3,15.4,48.1,8.8,39.4,7.5z M37.1,34l-6.6,0.8c-1.1,0.1-2,0.9-2.3,2l-1.9,8.1c-0.6,2.4-2.5,4.2-5,4.5c-3.8,0.4-7.1-2.4-7.4-6.2
		c-0.3-3.7,2.6-6.8,6.3-7.2l6.6-0.8c1.1-0.1,2-0.9,2.3-2l1.9-8.1c0.6-2.4,2.5-4.2,5-4.5c3.8-0.4,7.1,2.4,7.4,6.2
		C43.7,30.5,40.8,33.6,37.1,34z"></path>
	<rect x="153.1" y="13.9" class="st0" width="8.2" height="42.3"></rect>
	<path class="st0" d="M90.2,20.4c7.3,0,8.6,4.8,9.4,7.3h8.8c-0.5-6.2-5.6-14.9-18.2-14.9c-10.9,0-19.4,7.5-19.4,22.3
		c0,14.6,8.1,22.2,19.3,22.2c10.6,0,16.6-6.4,18.4-15.6h-8.8c-1.1,5.2-4.5,7.9-9.4,7.9c-7.5,0-10.5-6.8-10.5-14.4
		C79.7,23.3,85.9,20.4,90.2,20.4z"></path>
	<path class="st0" d="M125.2,24c-13.7,0-15,12.1-15,15.8c0,12.9,6.8,17.1,15.6,17.1c5.4,0,11.6-2.5,13.8-10H131
		c-0.7,2.2-3,3.4-5.3,3.4c-6.6,0-7.1-5.3-7.3-7.7h21.5v-1.5C139.9,27,131.3,24,125.2,24z M118.6,37.4c0.5-4.1,2.5-6.6,6.4-6.6
		c2.6,0,5.9,1.5,6.4,6.6H118.6z"></path>
	<rect x="142" y="13.9" class="st0" width="8.2" height="42.3"></rect>
	<path class="st0" d="M174,13.8h-9.8V21h7.2c4.3,0,8.2,0.9,8.2,6.4c0,5.4-3.9,6.4-8.2,6.4h-7.2V41h9.8c10.6,0,14.7-6.7,14.7-13.6
		C188.7,20.5,184.6,13.8,174,13.8z"></path>
	<path class="st0" d="M210.5,24.9c-0.5-0.2-1.2-0.3-1.8-0.3c-4.1,0-8.3,2.7-9.8,6.5h-0.1v-5.7h-8v30.7h8.4V42.3
		c0-5.4,2.1-9.9,8.3-9.9c1,0,2.3,0.1,3,0.3V24.9z"></path>
	<path class="st0" d="M225.6,24.6c-9.3,0-15.7,7-15.7,16.2c0,9.5,6,16.1,15.7,16.1c6.9,0,12-3.1,14.4-10.3h-7.4
		c-0.5,1.9-3.3,4-6.6,4c-4.7,0-7.3-2.4-7.5-7.8h22.1C241.1,33.4,236,24.6,225.6,24.6z M218.4,37.5c0.1-2.4,1.7-6.5,7-6.5
		c4.1,0,5.9,2.3,6.7,6.5H218.4z"></path>
	<path class="st0" d="M260.3,37.7c-4.8-1.1-9.6-1.5-9.6-4.6c0-2.5,2.8-2.9,4.7-2.9c3,0,5.5,0.9,5.8,4.2h8c-0.7-7.5-6.8-9.8-13.5-9.8
		c-6.6,0-13.4,2-13.4,9.9c0,5.4,4.9,7.1,9.7,8.2c6.1,1.3,9.6,2.2,9.6,4.8c0,3-3.1,3.9-5.5,3.9c-3.3,0-6.4-1.5-6.5-5.2h-8
		C242,54.3,248.9,57,256,57c7,0,14-2.6,14-10.7C270.1,40.6,265.2,38.8,260.3,37.7z"></path>
	<path class="st0" d="M289.8,37.7c-4.8-1.1-9.6-1.5-9.6-4.6c0-2.5,2.8-2.9,4.7-2.9c3,0,5.5,0.9,5.8,4.2h8c-0.7-7.5-6.8-9.8-13.5-9.8
		c-6.6,0-13.4,2-13.4,9.9c0,5.4,4.9,7.1,9.7,8.2c6.1,1.3,9.6,2.2,9.6,4.8c0,3-3.1,3.9-5.5,3.9c-3.3,0-6.4-1.5-6.5-5.2h-8
		c0.4,8.1,7.3,10.8,14.4,10.8c7,0,14-2.6,14-10.7C299.6,40.6,294.7,38.8,289.8,37.7z"></path>
	<path class="st0" d="M34.1,57.6c0.3-0.2,0.4-0.6,0.4-1c0-0.5-0.2-0.9-0.5-1.1c-0.3-0.2-0.8-0.3-1.4-0.3h-1.9v5.1h0.8V58h0.8
		l1.3,2.2h0.9l-1.4-2.2C33.6,57.9,33.9,57.8,34.1,57.6z M32.5,57.3h-0.9v-1.5h1c0.1,0,0.3,0,0.4,0c0.1,0,0.3,0,0.4,0.1
		c0.1,0,0.2,0.1,0.3,0.2c0.1,0.1,0.1,0.2,0.1,0.4c0,0.2,0,0.3-0.1,0.4c-0.1,0.1-0.2,0.2-0.3,0.2c-0.1,0.1-0.2,0.1-0.4,0.1
		C32.8,57.3,32.6,57.3,32.5,57.3z"></path>
	<path class="st0" d="M35.6,54.2c-0.4-0.4-0.9-0.7-1.4-0.9c-0.5-0.2-1.1-0.3-1.7-0.3c-0.6,0-1.2,0.1-1.7,0.3c-0.6,0.2-1,0.5-1.4,0.9
		c-0.4,0.4-0.7,0.9-1,1.5c-0.2,0.6-0.4,1.2-0.4,1.9c0,0.7,0.1,1.3,0.3,1.8c0.2,0.6,0.5,1,0.9,1.5c0.4,0.4,0.9,0.7,1.4,1
		c0.6,0.2,1.2,0.4,1.8,0.4c0.6,0,1.2-0.1,1.7-0.3c0.5-0.2,1-0.5,1.4-0.9c0.4-0.4,0.7-0.9,1-1.5c0.2-0.6,0.4-1.2,0.4-1.9
		c0-0.7-0.1-1.3-0.4-1.9C36.4,55.1,36,54.6,35.6,54.2z M35.7,59.1c-0.2,0.5-0.4,0.9-0.8,1.2c-0.3,0.3-0.7,0.6-1.1,0.8
		c-0.4,0.2-0.9,0.3-1.4,0.3c-0.6,0-1-0.1-1.5-0.3c-0.4-0.2-0.8-0.5-1.1-0.8c-0.3-0.3-0.5-0.7-0.7-1.2c-0.2-0.5-0.2-1-0.2-1.5
		c0-0.6,0.1-1.1,0.3-1.5c0.2-0.5,0.4-0.9,0.8-1.2c0.3-0.3,0.7-0.6,1.1-0.8c0.4-0.2,0.9-0.3,1.4-0.3c0.5,0,1,0.1,1.4,0.3
		c0.4,0.2,0.8,0.4,1.1,0.8c0.3,0.3,0.6,0.7,0.8,1.2c0.2,0.5,0.3,1,0.3,1.5C36,58.1,35.9,58.6,35.7,59.1z"></path>
</g>
</svg></a></div>
    <div class="footer__social-links">
        <ul class="rlist rlist--inline centered">
            <li><a href="http://on.fb.me/1tgHakh" title="Facebook" target="_blank"><i aria-hidden="true" class="icon-Icon_Facebook"></i></a></li>

            <li><a href="https://www.instagram.com/cellpress/" title="Instagram" target="_blank"><svg width="18px" height="18px" viewBox="0 0 18 18" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-147.000000, -16.000000)" fill-rule="nonzero" fill="#969696"> <g id="Instagram-Logo" transform="translate(147.000000, 16.000000)"> <path d="M9,1.62580645 C11.4096774,1.62580645 11.7,1.62580645 12.6290323,1.68387097 C13.5,1.71290323 13.9935484,1.85806452 14.3129032,2.00322581 C14.7193548,2.17741935 15.0387097,2.3516129 15.3580645,2.67096774 C15.6774194,2.99032258 15.8806452,3.28064516 16.0258065,3.71612903 C16.1419355,4.03548387 16.2870968,4.5 16.3451613,5.4 C16.3741935,6.35806452 16.4032258,6.61935484 16.4032258,9.02903226 C16.4032258,11.4387097 16.4032258,11.7290323 16.3451613,12.6580645 C16.316129,13.5290323 16.1709677,14.0225806 16.0258065,14.3419355 C15.8516129,14.7483871 15.6774194,15.0677419 15.3580645,15.3870968 C15.0387097,15.7064516 14.7483871,15.9096774 14.3129032,16.0548387 C13.9935484,16.1709677 13.5290323,16.316129 12.6290323,16.3741935 C11.6709677,16.4032258 11.4096774,16.4322581 9,16.4322581 C6.59032258,16.4322581 6.3,16.4322581 5.37096774,16.3741935 C4.5,16.3451613 4.00645161,16.2 3.68709677,16.0548387 C3.28064516,15.8806452 2.96129032,15.7064516 2.64193548,15.3870968 C2.32258065,15.0677419 2.11935484,14.7774194 1.97419355,14.3419355 C1.85806452,14.0225806 1.71290323,13.5580645 1.65483871,12.6580645 C1.62580645,11.7 1.62580645,11.4096774 1.62580645,9 C1.62580645,6.59032258 1.62580645,6.3 1.68387097,5.37096774 C1.71290323,4.5 1.85806452,4.00645161 2.00322581,3.68709677 C2.17741935,3.28064516 2.3516129,2.96129032 2.67096774,2.64193548 C2.99032258,2.32258065 3.28064516,2.11935484 3.71612903,1.97419355 C4.03548387,1.85806452 4.5,1.71290323 5.4,1.65483871 C6.3,1.62580645 6.59032258,1.62580645 9,1.62580645 L9,1.62580645 Z M9,0 C6.56129032,0 6.24193548,0 5.28387097,0.0580645161 C4.32580645,0.116129032 3.65806452,0.261290323 3.10645161,0.464516129 C2.52580645,0.696774194 2.00322581,1.01612903 1.50967742,1.50967742 C1.01612903,2.00322581 0.696774194,2.52580645 0.464516129,3.10645161 C0.232258065,3.68709677 0.0870967742,4.32580645 0.0580645161,5.28387097 C0,6.24193548 0,6.56129032 0,9 C0,11.4387097 0,11.7580645 0.0580645161,12.716129 C0.116129032,13.6741935 0.261290323,14.3419355 0.464516129,14.8935484 C0.696774194,15.4741935 1.01612903,15.9967742 1.50967742,16.4903226 C2.00322581,16.983871 2.52580645,17.3032258 3.10645161,17.5354839 C3.68709677,17.7677419 4.32580645,17.9129032 5.28387097,17.9419355 C6.24193548,18 6.56129032,18 9,18 C11.4387097,18 11.7580645,18 12.716129,17.9419355 C13.6741935,17.883871 14.3419355,17.7387097 14.8935484,17.5354839 C15.4741935,17.3032258 15.9967742,16.983871 16.4903226,16.4903226 C16.983871,15.9967742 17.3032258,15.4741935 17.5354839,14.8935484 C17.7677419,14.3129032 17.9129032,13.6741935 17.9419355,12.716129 C18,11.7580645 18,11.4387097 18,9 C18,6.56129032 18,6.24193548 17.9419355,5.28387097 C17.883871,4.32580645 17.7387097,3.65806452 17.5354839,3.10645161 C17.3032258,2.52580645 16.983871,2.00322581 16.4903226,1.50967742 C15.9967742,1.01612903 15.4741935,0.696774194 14.8935484,0.464516129 C14.3129032,0.232258065 13.6741935,0.0870967742 12.716129,0.0580645161 C11.7580645,0 11.4387097,0 9,0 L9,0 Z"></path> <path d="M9,4.38387097 C6.44516129,4.38387097 4.38387097,6.44516129 4.38387097,9 C4.38387097,11.5548387 6.44516129,13.616129 9,13.616129 C11.5548387,13.616129 13.616129,11.5548387 13.616129,9 C13.616129,6.44516129 11.5548387,4.38387097 9,4.38387097 Z M9,11.9903226 C7.34516129,11.9903226 6.00967742,10.6548387 6.00967742,9 C6.00967742,7.34516129 7.34516129,6.00967742 9,6.00967742 C10.6548387,6.00967742 11.9903226,7.34516129 11.9903226,9 C11.9903226,10.6548387 10.6548387,11.9903226 9,11.9903226 Z"></path> <circle id="Oval" cx="13.7903226" cy="4.20967742" r="1.07419355"></circle> </g> </g> </g></svg></a></li>
            
            <li><a href="https://www.linkedin.com/company/cell-press/" title="LinkedIn" target="_blank"><i aria-hidden="true" class="icon-linkedin"></i></a></li>
            
                        <li><a href="http://bit.ly/1zq7Q5T" title="Twitter" target="_blank">
                <!--<i aria-hidden="true" class="icon-Icon_Twitter"></i></a>-->
                
                <svg id="xicon" viewBox="0 0 1200 1227" xmlns="http://www.w3.org/2000/svg" aria-hidden="true" role="none"><path fill="#969696" d="M714.163 519.284L1160.89 0H1055.03L667.137 450.887L357.328 0H0L468.492 681.821L0 1226.37H105.866L515.491 750.218L842.672 1226.37H1200L714.137 519.284H714.163ZM569.165 687.828L521.697 619.934L144.011 79.6944H306.615L611.412 515.685L658.88 583.579L1055.08 1150.3H892.476L569.165 687.854V687.828Z"></path></svg> 

                    <style>
                    .footer svg#xicon {
                      height: 16px;
                    }
                    </style>
            </a></li>

            
            <li><a href="/pb-assets/marketing/WeChat/WeChat-1557753222727.jpg" title="WeChat" target="_blank"><svg width="18px" height="15px" viewBox="0 0 18 15" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-122.000000, -17.000000)" fill-rule="nonzero" fill="#969696"> <g id="wechat" transform="translate(122.000000, 17.000000)"> <path d="M12.375,4.5 L12.375,4.5 L12.4875,4.5 L12.4875,4.5 C12.6898579,4.48604473 12.8520408,4.32704187 12.87,4.125 C12.8705905,4.04725572 12.8470025,3.9712499 12.8025,3.9075 C11.94,1.5675 9.4425,0 6.5625,0 C2.9475,0 0,2.52 0,5.625 C0.0313772094,7.34410122 0.895513317,8.94093526 2.3175,9.9075 L1.5675,11.46 C1.49009368,11.6087816 1.52088865,11.7908702 1.64291494,11.9059236 C1.76494122,12.0209769 1.94852501,12.0410168 2.0925,11.955 L4.08,10.8225 C4.61861992,11.0129999 5.17724298,11.1413322 5.745,11.205 C5.86564124,11.2178337 5.98503738,11.1715098 6.06545426,11.0806685 C6.14587113,10.9898272 6.17737153,10.865694 6.15,10.7475 C6.06148727,10.4219419 6.01114891,10.0871918 6,9.75 C6,6.855 8.8575,4.5 12.375,4.5 Z M8.25,3 C8.66421356,3 9,3.33578644 9,3.75 C9,4.16421356 8.66421356,4.5 8.25,4.5 C7.83578644,4.5 7.5,4.16421356 7.5,3.75 C7.5,3.33578644 7.83578644,3 8.25,3 Z M4.5,4.5 C4.08578644,4.5 3.75,4.16421356 3.75,3.75 C3.75,3.33578644 4.08578644,3 4.5,3 C4.91421356,3 5.25,3.33578644 5.25,3.75 C5.25,4.16421356 4.91421356,4.5 4.5,4.5 Z"></path> <path d="M18,9.75 C18,7.2675 15.48,5.25 12.375,5.25 C9.27,5.25 6.75,7.2675 6.75,9.75 C6.75,12.2325 9.27,14.25 12.375,14.25 C12.9568557,14.2502837 13.5364258,14.1772075 14.1,14.0325 L15.96,14.9625 C16.0114248,14.9874074 16.0678613,15.0002339 16.125,15 C16.2497193,14.9993787 16.3659682,14.9367912 16.4351508,14.8330172 C16.5043335,14.7292432 16.5173955,14.5978644 16.47,14.4825 L15.96,13.2075 C17.1784092,12.4716325 17.9450305,11.1723221 18,9.75 L18,9.75 Z M10.5,9 C10.0857864,9 9.75,8.66421356 9.75,8.25 C9.75,7.83578644 10.0857864,7.5 10.5,7.5 C10.9142136,7.5 11.25,7.83578644 11.25,8.25 C11.25,8.66421356 10.9142136,9 10.5,9 Z M14.25,9 C13.8357864,9 13.5,8.66421356 13.5,8.25 C13.5,7.83578644 13.8357864,7.5 14.25,7.5 C14.6642136,7.5 15,7.83578644 15,8.25 C15,8.66421356 14.6642136,9 14.25,9 Z" opacity="0.959999979"></path> </g> </g> </g></svg></a></li>
            
            <li><a href="https://www.weibo.com/CellPress" title="Weibo" target="_blank"><svg width="18px" height="15px" viewBox="0 0 18 15" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-99.000000, -17.000000)" fill-rule="nonzero" fill="#969696"> <g id="weibo" transform="translate(99.000000, 14.000000)"> <g transform="translate(0.000000, 3.000000)"> <path d="M13.2402211,7.47054172 C12.9918392,7.39866484 12.821608,7.34902326 12.9514975,7.03290287 C13.232804,6.34602189 13.2622915,5.75359781 12.9571055,5.33095486 C12.3847236,4.5380684 10.8191759,4.58064295 9.02514573,5.30975376 C9.02514573,5.30889193 8.46144724,5.54882627 8.60544724,5.11515185 C8.88150754,4.25435294 8.83953769,3.53386047 8.41043216,3.11759508 C7.43716583,2.17285363 4.84824121,3.15344733 2.62745729,5.3059617 C0.964944724,6.9186238 0,8.62798358 0,10.105855 C0,12.9324952 3.73748744,14.8235294 7.39429146,14.8235294 C12.1877186,14.8235294 15.3768844,11.9501778 15.3768844,9.80576471 C15.3768844,8.50974008 14.2514774,7.77442408 13.2402211,7.47054172 Z M7.40424121,13.5561176 C4.48625126,13.8315595 1.9678794,12.5686293 1.77811055,10.7327524 C1.58870352,8.89773735 3.80098492,7.18579207 6.71861307,6.90948837 C9.63678392,6.63266758 12.1556985,7.89663201 12.3447437,9.73164706 C12.533608,11.5675239 10.3227739,13.2796416 7.40424121,13.5561176 Z M18,5.16875787 C17.9985528,2.31402189 15.5695477,0 12.5728643,0 C12.222995,0 11.9396985,0.270098495 11.9396985,0.603283174 C11.9396985,0.936640219 12.222995,1.20656635 12.5728643,1.20656635 C14.8708945,1.20656635 16.7336683,2.98159781 16.7336683,5.17099863 C16.7336683,5.50435568 17.0173266,5.77428181 17.3668342,5.77428181 C17.7167035,5.77428181 18,5.50435568 18,5.17099863 C18,5.17099863 18,5.1696197 18,5.16875787 Z M15.4841608,5.08016142 C15.1948945,3.73415048 14.0812462,2.67323393 12.6683819,2.39744733 C12.3262915,2.33039672 11.9914372,2.54051163 11.9212462,2.86680164 C11.8506935,3.19309166 12.0712161,3.51179754 12.4133065,3.57919289 C13.3319397,3.75845417 14.0559196,4.44774829 14.2438794,5.32302599 C14.3146131,5.64931601 14.6491055,5.85925855 14.9915578,5.7923803 C15.3338291,5.72498495 15.5541709,5.40627907 15.4841608,5.08016142 Z M5.90761809,8.7069275 C4.7321005,8.9320383 3.93901508,9.87126402 4.1358392,10.8044569 C4.33248241,11.7378222 5.44504523,12.311803 6.62056281,12.0865198 C7.79571859,11.861409 8.58916583,10.922528 8.3921608,9.98899042 C8.19569849,9.0559699 7.08277387,8.48181669 5.90761809,8.7069275 Z"></path> </g> </g> </g> </g></svg></a></li>
            
            <li><a href="https://www.youtube.com/user/cellvideoabstracts" title="YouTube" target="_blank"><svg width="18px" height="13px" viewBox="0 0 18 13" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-76.000000, -19.000000)" fill-rule="nonzero" fill="#969696"> <g id="youtube" transform="translate(76.000000, 14.000000)"> <path d="M17.2984219,6.41974219 C16.6486641,5.64735937 15.4490625,5.33232422 13.1580703,5.33232422 L4.84175391,5.33232422 C2.49834375,5.33232422 1.27838672,5.66771484 0.631089844,6.49001953 C0,7.29172266 0,8.47304297 0,10.1080547 L0,13.2243047 C0,16.3917773 0.748792969,18 4.84175391,18 L13.1581055,18 C15.1448555,18 16.2457383,17.7219844 16.9579688,17.040375 C17.688375,16.3413984 18,15.2001211 18,13.2243047 L18,10.1080547 C18,8.38381641 17.951168,7.19553516 17.2984219,6.41974219 Z M11.5560703,12.0963867 L7.77965625,14.0700586 C7.69524609,14.1141797 7.60289062,14.136082 7.51071094,14.136082 C7.40629687,14.136082 7.30212891,14.1079922 7.20987891,14.0521289 C7.03613672,13.9468711 6.93007031,13.7585391 6.93007031,13.5554414 L6.93007031,9.62075391 C6.93007031,9.41800781 7.03585547,9.22985156 7.20924609,9.12452344 C7.38267187,9.01919531 7.59839062,9.01198828 7.77832031,9.10546875 L11.5547344,11.0664141 C11.7468633,11.1661523 11.8675195,11.3645039 11.8678008,11.5809258 C11.8680469,11.7975586 11.747918,11.9961914 11.5560703,12.0963867 Z"></path> </g> </g> </g></svg></a></li>
            
        </ul>
    </div>
</div>

<style>

.ux3 .footer__logo svg {
    max-width: 160px !important;
}

.ux3 .footer__social-links a svg {
    max-width: 26px;
    height: 1.15rem;
    width: auto;
}

.ux3 .footer__social-links a:hover svg path {
    fill: #353535;
}

</style></div>

            

        

    

        


    

            

            
                <div class="footer__bottom text-onDark">
            

            <div data-widget-id="06227714-c621-4651-9879-b0ce5176cccc" data-widget-name="ux3-layout-widget"><div class="col-sm-4 col-md-5 clearfix">
    

        


    

            

            

            <div data-widget-id="07e1d276-b4dd-4870-b858-ed74e70d64ef" data-widget-name="UX3HTMLWidget"><div class="groups">
	<div class="group">
		<h2 class="group__title">Life &amp; medical sciences journals</h2>
		<ul class="rlist clearfix sided">
		<li>
			<a alt="Cell" href="/cell/home">Cell</a>
		</li>
		<li>
			<a alt="Cancer Cell" href="/cancer-cell/home">Cancer Cell</a>
		</li>
		<li>
			<a alt="Cell Chemical Biology" href="/cell-chemical-biology">Cell Chemical Biology</a>
		</li>
		<li>
			<a alt="Cell Genomics" href="/cell-genomics/home">Cell Genomics</a>
		</li>
		<li>
			<a alt="Cell Host and Microbe" href="/cell-host-microbe/home">Cell Host &amp; Microbe</a>
		</li>
		<li>
			<a alt="Cell Metabolism" href="/cell-metabolism/home">Cell Metabolism</a>
		</li>
		<li>
			<a alt="Cell Reports" href="/cell-reports/home">Cell Reports</a>
		</li>
		<li>
			<a alt="Cell Reports Medicine" href="/cell-reports-medicine/home">Cell Reports Medicine</a>
		</li>
		<li>
			<a alt="Cell Stem Cell" href="/cell-stem-cell/home">Cell Stem Cell</a>
		</li>
		<li>
			<a alt="Cell Systems" href="/cell-systems/home">Cell Systems</a>
		</li>
		<li>
			<a alt="Current Biology" href="/current-biology/home">Current Biology</a>
		</li>
		<li>
			<a alt="Developmental Cell" href="/developmental-cell/home">Developmental Cell</a>
		</li>
		<li>
			<a alt="Immunity" href="/immunity/home">Immunity</a>
		</li>
		<li>
			<a alt="Med" href="/med/home">Med</a>
		</li>
		<li>
			<a alt="Molecular Cell" href="/molecular-cell/home">Molecular Cell</a>
		</li>
		<li>
			<a alt="Neuron" href="/neuron/home">Neuron</a>
		</li>
		<li>
			<a alt="Structure" href="/structure/home">Structure</a>
		</li>
		<li>
			<a alt="American Journal of Human Genetics" href="/ajhg/home">American Journal of Human Genetics (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Biophysical Journal" href="/biophysj/home">Biophysical Journal (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Biophysical Reports" href="/biophysreports/home">Biophysical Reports (<i>partner</i>)</a>
		</li>	
		<li>
			<a alt="HGG Advances" href="/hgg-advances/home">Human Genetics and Genomics Advances (<i>partner</i>)</a></li>
		<li>
			<a alt="Molecular Plant" href="/molecular-plant/home">Molecular Plant (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy" href="/molecular-therapy-family/molecular-therapy/home">Molecular Therapy (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy Methods and Clinical Development" href="/molecular-therapy-family/methods/home">Molecular Therapy Methods &amp; Clinical Development (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy Nucleic Acids" href="/molecular-therapy-family/nucleic-acids/home">Molecular Therapy Nucleic Acids (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy Oncology" href="/molecular-therapy-family/oncology/home">Molecular Therapy Oncology (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Plant Communications" href="/plant-communications/home">Plant Communications (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Stem Cell Reports" href="/stem-cell-reports/home">Stem Cell Reports (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Trends in Biochemical Sciences" href="/trends/biochemical-sciences/home">Trends in Biochemical Sciences</a>
		</li>
		<li>
		</li><li>
			<a alt="Trends in Cancer" href="/trends/cancer/home">Trends in Cancer</a>
		</li>
		<li>
			<a alt="Trends in Cell Biology" href="/trends/cell-biology/home">Trends in Cell Biology</a>
		</li>
		<li>
			<a alt="Trends in Ecology and Evolution" href="/trends/ecology-evolution/home">Trends in Ecology &amp; Evolution</a>
		</li>
		<li>
			<a alt="Trends in Endocrinology and Metabolism" href="/trends/endocrinology-metabolism/home">Trends in Endocrinology &amp; Metabolism</a>
		</li>
		<li>
			<a alt="Trends in Genetics" href="/trends/genetics/home">Trends in Genetics</a>
		</li>
		<li>
			<a alt="Trends in Immunology" href="/trends/immunology/home">Trends in Immunology</a>
		</li>
		<li>
			<a alt="Trends in Microbiology" href="/trends/microbiology/home">Trends in Microbiology</a>
		</li>
		<li>
			<a alt="Trends in Molecular Medicine" href="/trends/molecular-medicine/home">Trends in Molecular Medicine</a>
		</li>
		<li>
			<a alt="Trends in Neurosciences" href="/trends/neurosciences/home">Trends in Neurosciences</a>
		</li>
		<li>
			<a alt="Trends in Parasitology" href="/trends/parasitology/home">Trends in Parasitology</a>
		</li>
		<li>
			<a alt="Trends in Pharmacological Sciences" href="/trends/pharmacological-sciences/home">Trends in Pharmacological Sciences</a>
		</li>
		<li>
			<a alt="Trends in Plant Science" href="/trends/plant-science/home">Trends in Plant Science</a>
		</li>
		</ul>
	</div>

	<div class="group">
		<h2 class="group__title">Physical sciences &amp; engineering journals</h2>
		<ul class="rlist clearfix sided">
		  <li><a alt="Cell Biomaterials" href="/cell-biomaterials/home">Cell Biomaterials</a></li>
		  <li>
			<a alt="Cell Reports Physical Science" href="/cell-reports-physical-science/home">Cell Reports Physical Science</a>
		</li>
		<li>
			<a alt="Chem" href="/chem/home">Chem</a>
		</li>
		<li>
			<a alt="Chem Catalysis" href="/chem-catalysis/home">Chem Catalysis</a>
		</li>
		<li>
			<a alt="Device" href="/device/home">Device</a>
		</li>
		<li>
			<a alt="Joule" href="/joule/home">Joule</a>
		</li>
		<li>
			<a alt="Matter" href="/matter/home">Matter</a>
		</li>	
		<li><a alt="Matter" href="/newton/home">Newton</a></li>
		<li>
			<a alt="Trends in Chemistry" href="/trends/chemistry/home">Trends in Chemistry</a>
		</li>			
		</ul>
	</div>

<div class="group">
	<h2 class="group__title">Multidisciplinary journals</h2>
	<ul class="rlist clearfix sided">
	  		<li>
			<a alt="Cell Reports Methods" href="/cell-reports-methods/home">Cell Reports Methods</a>
		</li>
		<li>
			<a alt="Cell Reports Sustainability" href="/cell-reports-sustainability/home">Cell Reports Sustainability</a>
		</li>
		<li>
			<a alt="Heliyon" href="/heliyon/home">Heliyon</a>
		</li>	
		<li>
			<a alt="iScience" href="/iscience/home">iScience</a>
		</li>
		<li>
			<a alt="One Earth" href="/one-earth/home">One Earth</a>
		</li>
		<li>
			<a alt="Patterns" href="/patterns/home">Patterns</a>
		</li>
		<li>
			<a alt="STAR Protocols" href="/star-protocols/home">STAR Protocols</a>
		</li>
		<li>
			<a alt="Nexus" href="/nexus/home">Nexus (<i>partner</i>)</a>
		</li>			
        <li>
			<a alt="The Innovation" href="/the-innovation/home">The Innovation (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Trends in Biotechnology" href="/trends/biotechnology/home">Trends in Biotechnology</a>
		</li>
		<li>
			<a alt="Trends in Cognitive Sciences" href="/trends/cognitive-sciences/home">Trends in Cognitive Sciences</a>
		</li>
	</ul>
</div>

</div>
</div>

            

        
</div><div class="col-sm-4 col-md-5 clearfix">
    

        


    

            

            

            <div data-widget-id="850476de-00cb-4c0d-943c-bf1881da24f9" data-widget-name="UX3HTMLWidget"><div class="groups">
	<div class="clearfix">
		<div class="group sided">
			<h2 class="group__title">Authors</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Submit article" href="/submit-to-cell-press">Submit article</a>
				</li>
				<li>
					<a alt="Multi-Journal Submission" href="/multi-journal-submission">Multi-Journal Submission</a>
				</li>
				<li>
					<a alt="STAR Methods" href="/star-authors-guide">STAR Methods</a>
				</li>
				<li>
					<a alt="Sneak Peek – Preprints" href="/sneakpeek">Sneak Peek – Preprints</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
		<h2 class="group__title">Reviewers</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Reviewers" href="/reviewers">Information for reviewers</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
		<h2 class="group__title">News &amp; events</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Newsroom" href="/newsroom">Newsroom</a>
				</li>
				<li>
					<a alt="Events" href="/events">Events</a>
				</li>
				<li>
					<a alt="Cell Symposia" href="/symposia">Cell Symposia</a>
				</li>
				<li>
					<a alt="Consortia Hub" href="/consortia">Consortia Hub</a>
				</li>
				<li>
					<a alt="Webinars" href="/webinars">Webinars</a>
				</li>
								<li>
					&nbsp;
				</li>
			</ul>
		<h2 class="group__title">Multimedia</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Cell Press Podcast" href="/podcast">Cell Press Podcast</a>
				</li>
				<li>
					<a alt="Cell Press Videos" href="/video">Cell Press Videos</a>
				</li>
				<li>
					<a alt="Color and Comics" href="/comics">Coloring and Comics</a>
				</li>
				<li>
					<a alt="Figure360" href="/video-guidelines">Figure360</a>
				</li>
				<li>
					<a alt="Cell Picture Show" href="/pictureshow">Cell Picture Show</a>
				</li>
				<li>
					<a alt="Research Arc" href="/research-arc">Research Arc</a>
				</li>
			</ul>
		</div>

		<div class="group sided">
			<h2 class="group__title">About</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="About Cell Press" href="/about">About Cell Press</a>
				</li>

				<li>
					<a alt="Open access" href="/open-access">Open access</a>
				</li>
				<li>
					<a alt="COVID Hub" href="/covid-19">COVID Hub</a>
				</li>
				<li>
					<a alt="Sustainability" href="/sustainability">Sustainability hub</a>
				</li>
				<li>
					<a alt="Inclusion and diversity" href="/diversity">Inclusion and diversity</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
			<h2 class="group__title">Contact</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Contact us" href="/contact">Contact us</a>
				</li>
				<li>
					<a alt="Help and Support" href="https://service.elsevier.com/app/overview/cell-press/" target="_blank">Help &amp; Support</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
			<h2 class="group__title">Careers</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Cell Press Careers" href="https://relx.wd3.myworkdayjobs.com/en-US/ElsevierJobs" target="_blank">Cell Press Careers</a>
				</li>
			</ul>
		</div>
	</div>
</div></div>

            

        
</div><div class="col-sm-4 col-md-2 clearfix">
    

        


    

            

            

            <div data-widget-id="beafac41-1cfc-440b-9b55-74aba4a5612c" data-widget-name="UX3HTMLWidget"><div class="groups">
	<div class="group">
		<h2 class="group__title">Access</h2>
		<ul class="rlist clearfix">
			<li>
				<a alt="Subscribe" href="/subscribe">Subscribe</a>
			</li>
			<li>
				<a alt="Claim" href="/activate">Claim</a>
			</li>
			<li>
				<a alt="Read It Now" href="/read-it-now">Read-It-Now</a>
			</li>
			<li>
				<a alt="Recommend to Librarian" href="http://info.cell.com/request-access-to-cell-press-content" target="_blank">Recommend to Librarian</a>
			</li>
			<li>
				<a alt="Publication Alerts" href="/action/showPreferences?menuTab=Alerts">Publication Alerts</a>
			</li>
		</ul>
	</div>
	<div class="group">
		<h2 class="group__title">Collections</h2>
		<ul class="rlist clearfix">
			<li>
				<a alt="Best of Cell Press" href="/bestof">Best of Cell Press</a>
			</li>
			<li>
				<a alt="Cell Press Reviews" href="/reviews">Cell Press Reviews</a>
			</li>
			<li>
				<a alt="Cell Selections" href="/selections">Cell Press Selections</a>
			</li>
			<li>
				<a alt="Nucleus Collections" href="/nucleus">Nucleus Collections</a>
			</li>
			<li>
				<a alt="SnapShots" href="/snapshots">SnapShot Archive</a>
			</li>
		</ul>
	</div>
	<div class="group">
		<h2 class="group__title">Information</h2>
		<ul class="rlist clearfix">
			<li>
				<a alt="For Advertisters" href="/media-kit">For Advertisers</a>
			</li>
			<li>
				<a alt="For Recruiters" target="_blank" href="/pb-assets/Cell/common/CP-RecruitmentKit-US-2018-LowRes-1712070648.pdf">For Recruiters</a>
			</li>
			<li>
				<a alt="For Librarians" target="_blank" href="https://www.elsevier.com/products/sciencedirect/journals/direct-subscription-journals">For Librarians</a>
			</li>
			<li>
				<a alt="Privacy Policy" target="_blank" href="https://www.elsevier.com/legal/privacy-policy">Privacy Policy</a>
			</li>
			<li>
				<a alt="Terms and Conditions" target="_blank" href="https://www.elsevier.com/legal/elsevier-website-terms-and-conditions">Terms and Conditions</a>
			</li>
			<li>
				<a alt="Accessibility" href="/accessibility">Accessibility</a>
			</li>
		</ul>
	</div>
</div></div>

            

        
</div></div>

            
                </div>
            

        

    

        


    

            

            

            <div data-widget-id="779ad11f-7bcd-4a2a-a2cd-8c67b649cbb2" data-widget-name="UX3HTMLWidget"><style>
 
button#cookieBtn {
   text-decoration: none;
   background-color: transparent;
   color: #005789;
   font-weight: bold;
   padding: 0;
   margin: 0;
   font-size: 14px;
}
 
button#cookieBtn:hover {
   text-decoration: underline;
}

</style>
<script>
// HIDE ONETRUST BACKGROUND AFTER USER PRESSES ESCAPE KEY
setTimeout(function(){ 
   $("#onetrust-pc-sdk").on("keydown", function(e) { 
      if (e.keyCode == 27) {  // ESC key
         setTimeout(function(){ 
            $(".onetrust-pc-dark-filter").css("display","none");
         }, 100); 
      }
   });
}, 5000);
</script>

<span class="align-center copy-right">
<hr>
<p align="center"><b>The content on this site is intended for healthcare professionals and researchers across all fields of science.</b></p>
<hr>
<div>
<p>We use cookies to help provide and enhance our service and tailor content. To update your cookie settings, please visit the 
      <b><button id="cookieBtn" class="ot-sdk-show-settings" type="button" aria-label="Manage cookie consent preferences">Cookie Settings</button></b> for this site.<br>

All content on this site: Copyright © 2025 Elsevier Inc., its licensors, and contributors. <br>All rights are reserved, including those for text and data mining, AI training, and similar technologies. <br>For all open access content, the relevant licensing terms apply.
</p>
<ul class="rlist--inline">
			<li>
				<a alt="Privacy Policy" target="_blank" href="https://www.elsevier.com/legal/privacy-policy">Privacy Policy</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Terms and Conditions" target="_blank" href="https://www.elsevier.com/legal/elsevier-website-terms-and-conditions">Terms &amp; Conditions</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Accessibility" href="/accessibility">Accessibility</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Help and Support" target="_blank" href="https://service.elsevier.com/app/overview/cell-press/">Help &amp; Support</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Contact" href="/contact">Contact</a>
			</li>
			</ul>
<p>
<a href="https://www.relx.com"><img src="/pb/assets/raw/Health%20Advance/images/relx-1554393210983.png" alt="RELX" rel="nofollow" target="_new"></a></p>
</div>

</span></div>

            

        

    

        


    

            

            

            <div data-widget-id="c5e179c1-156e-41e3-af7f-5e307efa8acf" data-widget-name="UX3HTMLWidget"><style>#brandingRow{
    background-color:#fff;
    padding:7px 26px 7px 30px;
    color:#007DBC;
    font-size:14px;
    clear:both;
    width:100%
}
#brandingRow a{
    text-decoration:none;
    color:#007DBC
}
#brandingRow a:hover{
    text-decoration:underline
}
#brandingRowRight{
    display:inline;
    float:right;
    width:auto;
    padding:0;
    margin:0
}
svg#cp_logo {
  float: none;
  height: 36px;
  width: 36px;
  margin-left: 0px;
  margin-top: 0px;
}

footer hr {
  display: block;
}

@media screen and (max-width: 600px){
    #brandingRow{
        font-size:12px
    }
}
@media screen and (max-width: 750px){
    #brandingRow{
        padding:7px 18px 7px 16px
    }
    svg#cp_logo{
        margin-right:10px
    }
}
@media screen and (max-width: 800px) {
   a#cpIconLnk {
      display: none;
   }
}

.header__wrapper.clearfix .col-xs-6:first-of-type{
    width:55%
}
.header__wrapper.clearfix .col-xs-6:nth-of-type(2){
    width:45%
}
@media screen and (min-width: 800px) and (max-width: 1000px){
    .header__wrapper.clearfix .col-xs-6:first-of-type{
        width:75%
    }
    .header__wrapper.clearfix .col-xs-6:nth-of-type(2){
        width:25%
    }
}

</style>

<script>
var partnerx = false;
var overrideEndorsement = "";
var tmpj = $("input[name='journalCode']").attr("value");
	
if (window.location.pathname == "/molecular-therapy-family/home") {
	tmpj = "ymthe";
}
	
var ribbonx = "";
if (tmpj == "ajhg" || tmpj == "bpj" || tmpj == "molp" || tmpj == "xplc" || tmpj == "stemcr" || tmpj == "ymthe" || tmpj == "xhgg" || tmpj == "omtn" || tmpj == "omto" || tmpj == "omtm" || tmpj == "biophysreports" ||  tmpj == "bpr" || tmpj == "xinn" || tmpj == "omton") {
	partnerx = true;
	ribbonx = "<a href='https://www.cell.com' data-aa-name='cell press endorsement' data-aa-region='cp global header' title='Visit Cell.com'>A Cell Press partner journal</a>";
} else if (tmpj != "" && tmpj != "undefined" && tmpj != undefined) {
	ribbonx = "<a href='/' data-aa-name='cell press endorsement' data-aa-region='cp global header' title='Visit Cell.com'>A Cell Press journal</a>";
} else {
	ribbonx = "<a href='/' data-aa-name='cell press endorsement' data-aa-region='cp global header' title='Visit Cell.com'>A Cell Press resource</a>";
}
	
if (window.location.pathname == "/") {
	ribbonx = "";
}
if (overrideEndorsement != "") {
	ribbonx = overrideEndorsement;
}

$(".header__wrapper.clearfix").prepend("<div id='brandingRow'><a href='/' data-aa-name='science that inspires' data-aa-region='cp global header' title='Visit Cell.com'>Science that inspires</a><p id='brandingRowRight'>" + ribbonx + "</p></div>");
	
// handle scrolling effect on article pages
if ($("article").length > 0) {
	var scrollx;
	$(window).scroll(function() {
		scrollx = $(window).scrollTop();
		if (scrollx > 0) { 
			$("#brandingRow").css("display","none"); 
		} else {
			$("#brandingRow").css("display","block");
		}
	});
}

if (window.location.pathname == "/" || partnerx == false) {
               $("<a href='/home' id='cpIconLnk' data-aa-name='cell press logo' data-aa-region='cp global header' title='Cell Press' target='_blank'><svg id='cp_logo' version='1.1' xmlns='http://www.w3.org/2000/svg' xmlns:xlink='http://www.w3.org/1999/xlink' x='0px' y='0px' viewBox='0 0 100 98.1' style='enable-background:new 0 0 100 98.1;' xml:space='preserve' height='100' width='98.1'><path style='fill:#FFFFFF;' d='M69.09,0.09c-4.01-0.6-7.83,1.94-8.78,5.89l-3.14,13.11c-0.76,3.19-3.44,5.56-6.7,5.93l-18.6,2.13          C11.83,29.31-2.53,47.66,0.37,67.78c2.3,15.96,15.18,27.77,30.54,30.08c4.01,0.6,7.83-1.94,8.78-5.89l3.14-13.11 c0.76-3.19,3.44-5.56,6.69-5.93l18.6-2.13c20.04-2.17,34.41-20.52,31.5-40.65C97.32,14.21,84.45,2.39,69.09,0.09z M65.06,47.2  l-11.75,1.4c-1.98,0.21-3.62,1.65-4.08,3.59l-3.43,14.3c-1.01,4.22-4.5,7.39-8.8,8c-6.66,0.72-12.63-4.2-13.15-10.92                c-0.51-6.5,4.6-12.11,11.08-12.81l11.75-1.39c1.98-0.22,3.62-1.65,4.08-3.59l3.43-14.3c1.01-4.22,4.5-7.39,8.8-8          c6.66-0.72,12.63,4.19,13.15,10.92C76.66,40.89,71.55,46.5,65.06,47.2z'/></svg></a>").insertBefore(".header__wrapper.clearfix .journal-logos:eq(0)");
}


</script></div>

            

        

    

        


    

            

            

            <div data-widget-id="1af759b7-9bb6-4413-ada9-51a8333e8fb8" data-widget-name="UX3HTMLWidget"><style>
    /*This was added temporarily until the PB changelog tab issue got fixed. [LIT-444969 - c#4]*/
    [data-tab-id="changelogTab"] {
        display: none !important;
    }
    
    /*This was added temporarily until the PB changelog tab issue got fixed. [LIT-518839]*/
    @media (min-width: 769px) {
        .ux3 .quick-search .dropBlock [data-db-target-of],
        .quick-search .dropBlock [data-db-target-of] {
            width: 100vw !important;
        }
    }
</style></div>

            

        
</div></div>

            

        

        </div>
    

    

        


    

            

            

            <div data-widget-id="d7e0813b-a300-497a-ab37-9c3ddaf03e71" data-widget-name="UX3HTMLWidget"><!--
<div class="session-reminder hidden">
    <div class="session-reminder__overlay">
        <div class="session-reminder__dialog">
            <div class="session-reminder__dialog-inner">
                <div class="dialog-header" role="dialog" aria-labelledby="dialog-title" aria-describedby="dialog-desc">
                    <h2 class="dialog-header-title" id="dialog-title">
                        <span>Session Timeout</span>
                        <div class="dialog-timer">(2:00)</div>
                    </h2>
                    <button class="dialog-close" type="button" aria-label="close dialog">
                        <i class="icon-tools_close" aria-hidden="true"></i>
                    </button>
                </div>
                <div class="dialog-body">
                    <p class="dialog-desc" id="dialog-desc">Your session will expire shortly. If you are still working, click the ‘Keep Me Logged In’ button below. If you do not respond within the next minute, you will be automatically logged out.</p>
                    <p class="dialog-expired sr-only" aria-live="polite"></p>
                </div>
                <div class="dialog-footer">
                    <button class="session-reset" type="button">Keep Me Logged In</button>
                </div>
            </div>
        </div>
    </div>
</div>
<a class="sr-only" href="javascript:void(0)" aria-hidden="true"></a>
--></div>

            

        
        </div>
    </div>


































    



















        
        
            
                <script src="/products/marlin/cell/releasedAssets/js/main.bundle-90475670808d5d3eee41.js"></script>
            
        
        
        

    




<script type="text/javascript">
    var pageData =  {"journal":{"specialty":"Cell Press","issn":"0092-8674","name":"Cell"},"page":{"loadTimestamp":"1749591888343","server":"cell","environment":"prod","businessUnit":"ELS:RAP:JBS","noTracking":"","name":"fulltext:html","section":"Cell","language":"en","type":"CP-CA","productName":"JB"},"visitor":{"departmentName":"|ae:ScienceDirect Guests","accessType":"ae:ANON_GUEST|ae:ANON_GUEST","accountId":"ae:838949|ae:228598","accountName":"ae:Cell Guest Account|ae:ScienceDirect Guests","departmentId":"ae:977212|ae:291352","ipAddress":"189.120.74.179","loginStatus":"anonymous","userId":"ae:246228266|ae:12975512"},"content":[{"accessType":"article:free:openaccess","volumeNumber":"184","issueNumber":"22","viewState":"fulltext","format":"MIME-HTML","id":"pii:S0092867421010655","coverDate":"202110","type":"XOCS-JOURNAL:SCOPE-FULL","title":"Enhanced prime editing systems by manipulating cellular determinants of editing outcomes","status":"S300"}]};
</script>

<script type="text/javascript">
    if(typeof pageDataTracker !== "undefined") {
        pageDataTracker.trackPageLoad();
    }
</script>
<script>(function(){function c(){var b=a.contentDocument||a.contentWindow.document;if(b){var d=b.createElement('script');d.innerHTML="window.__CF$cv$params={r:'94dc0d4f3c614287',t:'MTc0OTU5MTg4Ny4wMDAwMDA='};var a=document.createElement('script');a.nonce='';a.src='/cdn-cgi/challenge-platform/scripts/jsd/main.js';document.getElementsByTagName('head')[0].appendChild(a);";b.getElementsByTagName('head')[0].appendChild(d)}}if(document.body){var a=document.createElement('iframe');a.height=1;a.width=1;a.style.position='absolute';a.style.top=0;a.style.left=0;a.style.border='none';a.style.visibility='hidden';document.body.appendChild(a);if('loading'!==document.readyState)c();else if(window.addEventListener)document.addEventListener('DOMContentLoaded',c);else{var e=document.onreadystatechange||function(){};document.onreadystatechange=function(b){e(b);'loading'!==document.readyState&&(document.onreadystatechange=e,c())}}}})();</script><iframe height="1" width="1" style="position: absolute; top: 0px; left: 0px; border: none; visibility: hidden;"></iframe><script defer="" src="https://static.cloudflareinsights.com/beacon.min.js/vcd15cbe7772f49c399c6a5babf22c1241717689176015" integrity="sha512-ZpsOmlRQV6y907TI0dKBHq9Md29nnaEIPlkf84rnaERnq6zvWvPUqr2ft8M1aS28oN72PdrCzSjY4U6VaAw1EQ==" data-cf-beacon="{&quot;rayId&quot;:&quot;94dc0d4f3c614287&quot;,&quot;serverTiming&quot;:{&quot;name&quot;:{&quot;cfExtPri&quot;:true,&quot;cfEdge&quot;:true,&quot;cfOrigin&quot;:true,&quot;cfL4&quot;:true,&quot;cfSpeedBrain&quot;:true,&quot;cfCacheStatus&quot;:true}},&quot;version&quot;:&quot;2025.5.0&quot;,&quot;token&quot;:&quot;57a02315956f4b2e82817b3dfa99b1a2&quot;}" crossorigin="anonymous"></script>


<div id="addtoany" style="position: static;"><div class="a2a_overlay" id="a2a_overlay"></div><div id="a2a_modal" class="a2a_modal a2a_hide" role="dialog" tabindex="-1" aria-label=""><div class="a2a_modal_body a2a_menu a2a_hide" id="a2a_copy_link"><label for="a2a_copy_link_text" id="a2a_copy_link_icon" class="a2a_svg a2a_s_link a2a_logo_color"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"><title>Copy link</title><path fill="#FFF" d="M7.591 21.177c0-.36.126-.665.377-.917l2.804-2.804a1.235 1.235 0 0 1 .913-.378c.377 0 .7.144.97.43-.026.028-.11.11-.255.25-.144.14-.24.236-.29.29a2.82 2.82 0 0 0-.2.256 1.056 1.056 0 0 0-.177.344 1.43 1.43 0 0 0-.046.37c0 .36.126.666.377.918a1.25 1.25 0 0 0 .918.377c.126.001.251-.015.373-.047.125-.037.242-.096.345-.175.09-.06.176-.127.256-.2.1-.094.196-.19.29-.29.14-.142.223-.23.25-.254.297.28.445.607.445.984 0 .36-.126.664-.377.916l-2.778 2.79a1.242 1.242 0 0 1-.917.364c-.36 0-.665-.118-.917-.35l-1.982-1.97a1.223 1.223 0 0 1-.378-.9l-.001-.004Zm9.477-9.504c0-.36.126-.665.377-.917l2.777-2.79a1.235 1.235 0 0 1 .913-.378c.35 0 .656.12.917.364l1.984 1.968c.254.252.38.553.38.903 0 .36-.126.665-.38.917l-2.802 2.804a1.238 1.238 0 0 1-.916.364c-.377 0-.7-.14-.97-.418.026-.027.11-.11.255-.25a7.5 7.5 0 0 0 .29-.29c.072-.08.139-.166.2-.255.08-.103.14-.22.176-.344.032-.12.048-.245.047-.37 0-.36-.126-.662-.377-.914a1.247 1.247 0 0 0-.917-.377c-.136 0-.26.015-.37.046-.114.03-.23.09-.346.175a3.868 3.868 0 0 0-.256.2c-.054.05-.15.148-.29.29-.14.146-.222.23-.25.258-.294-.278-.442-.606-.442-.983v-.003ZM5.003 21.177c0 1.078.382 1.99 1.146 2.736l1.982 1.968c.745.75 1.658 1.12 2.736 1.12 1.087 0 2.004-.38 2.75-1.143l2.777-2.79c.75-.747 1.12-1.66 1.12-2.737 0-1.106-.392-2.046-1.183-2.818l1.186-1.185c.774.79 1.708 1.186 2.805 1.186 1.078 0 1.995-.376 2.75-1.13l2.803-2.81c.751-.754 1.128-1.671 1.128-2.748 0-1.08-.382-1.993-1.146-2.738L23.875 6.12C23.13 5.372 22.218 5 21.139 5c-1.087 0-2.004.382-2.75 1.146l-2.777 2.79c-.75.747-1.12 1.66-1.12 2.737 0 1.105.392 2.045 1.183 2.817l-1.186 1.186c-.774-.79-1.708-1.186-2.805-1.186-1.078 0-1.995.377-2.75 1.132L6.13 18.426c-.754.755-1.13 1.672-1.13 2.75l.003.001Z"></path></svg></label><input id="a2a_copy_link_text" type="text" title="Copy link" readonly=""><div id="a2a_copy_link_copied">✓</div></div><div class="a2a_modal_body a2a_menu a2a_thanks a2a_hide" id="a2a_thanks"><div class="a2a_localize" data-a2a-localize="inner,ThanksForSharing">Thanks for sharing!</div></div></div><div class="a2a_menu a2a_full a2a_localize" id="a2apage_full" role="dialog" tabindex="-1" aria-label="Share" data-a2a-localize="title,Share"><div class="a2a_full_header"><div id="a2apage_find_container" class="a2a_menu_find_container"><label for="a2apage_find" id="a2apage_find_icon" class="a2a_svg a2a_s_find"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" aria-hidden="true"><title>Find any service</title><path fill="#CCC" d="M19.7 18.2l-4.5-4.5c.7-1.1 1.2-2.3 1.2-3.6 0-3.5-2.8-6.3-6.3-6.3s-6.3 2.8-6.3 6.3 2.8 6.3 6.3 6.3c1.4 0 2.6-.4 3.6-1.2l4.5 4.5c.6.6 1.3.7 1.7.2.5-.4.4-1.1-.2-1.7zm-9.6-3.6c-2.5 0-4.5-2.1-4.5-4.5 0-2.5 2.1-4.5 4.5-4.5 2.5 0 4.5 2.1 4.5 4.5s-2 4.5-4.5 4.5z"></path></svg></label><input id="a2apage_find" class="a2a_menu_find a2a_localize" type="text" autocomplete="off" title="Find any service" data-a2a-localize="title,FindAnyServiceToAddTo"></div></div><div class="a2a_full_services" id="a2apage_full_services" role="presentation"></div><div class="a2a_full_footer"><a href="https://www.addtoany.com" title="Share Buttons" rel="noopener" target="_blank"><span class="a2a_svg a2a_s__default a2a_s_a2a a2a_logo_color"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"><g fill="#FFF"><path d="M14 7h4v18h-4z"></path><path d="M7 14h18v4H7z"></path></g></svg></span>AddToAny</a></div></div><div id="a2apage_dropdown" class="a2a_menu a2a_mini a2a_localize a2a_hide" tabindex="-1" aria-label="Share" data-a2a-localize="label,Share"><div class="a2a_mini_services" id="a2apage_mini_services"></div><div id="a2apage_cols_container" class="a2a_cols_container"><div class="a2a_col1" id="a2apage_col1"></div><div id="a2apage_2_col1" class="a2a_hide"></div><div class="a2a_clear"></div></div><div class="a2apage_wide a2a_wide"><a href="#addtoany" id="a2apage_show_more_less" class="a2a_more a2a_localize" title="Show all" data-a2a-localize="title,ShowAll"><span class="a2a_svg a2a_s__default a2a_s_a2a a2a_logo_color"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"><g fill="#FFF"><path d="M14 7h4v18h-4z"></path><path d="M7 14h18v4H7z"></path></g></svg></span><span class="a2a_localize" data-a2a-localize="inner,More">More…</span></a></div></div><div style="height: 1px; width: 1px; position: absolute; z-index: 100000; top: 0px; visibility: hidden;"><iframe id="a2a_sm_ifr" title="AddToAny Utility Frame" aria-hidden="true" src="https://static.addtoany.com/menu/sm.25.html#type=core&amp;event=load" style="height: 1px; width: 1px; border: 0px; left: 0px; top: 0px; position: absolute; z-index: 100000; display: none;"></iframe></div></div><div role="status" aria-live="assertive" aria-relevant="additions" class="ui-helper-hidden-accessible"></div><div id="crossmark-widget" style="display: none;"><div class="crossmark-reset crossmark-overlay"></div><div class="crossmark-reset crossmark-popup"><div class="crossmark-reset crossmark-popup__offset"><div class="crossmark-reset crossmark-popup__inner"><div class="crossmark-reset crossmark-popup__header"><button class="crossmark-reset crossmark-popup__btn-close"><span class="sr-only">__("articleCrossmark.closePopup")</span></button></div><div class="crossmark-reset crossmark-popup__content-wrapper"><iframe class="crossmark-reset crossmark-popup__content" title="Crossmark service - Intentionally blank" aria-hidden="false" tabindex="0" src="https://crossmark.crossref.org/dialog?doi=10.1016%2Fj.cell.2021.09.018&amp;domain=www.cell.com&amp;uri_scheme=https%3A&amp;cm_version=v2.0"></iframe></div></div></div></div></div><div id="onetrust-consent-sdk" data-nosnippet="true"><div class="onetrust-pc-dark-filter ot-fade-in" style="z-index:2147483645;"></div><div id="onetrust-banner-sdk" class="otFlat bottom ot-wo-title ot-buttons-fw" tabindex="0" role="region" aria-label="Cookie banner"><div role="dialog" aria-modal="true" aria-label="Privacy"><div class="ot-sdk-container"><div class="ot-sdk-row"><div id="onetrust-group-container" class="ot-sdk-eight ot-sdk-columns"><div class="banner_logo"></div><div id="onetrust-policy"><div id="onetrust-policy-text">We use cookies that are necessary to make our site work. We may also use additional cookies to analyze, improve, and personalize our content and your digital experience. For more information, see our<a class="ot-cookie-policy-link" href="https://www.elsevier.com/legal/cookienotice" target="_blank" aria-label=", opens in a new tab" rel="noopener">Cookie Policy</a></div></div></div><div id="onetrust-button-group-parent" class="ot-sdk-three ot-sdk-columns"><div id="onetrust-button-group"><button id="onetrust-pc-btn-handler" class="cookie-setting-link">Cookie Settings</button>  <button id="onetrust-accept-btn-handler">Accept all cookies</button></div></div></div></div><!-- Close Button --><div id="onetrust-close-btn-container"></div><!-- Close Button END--></div></div><div id="onetrust-pc-sdk" class="otPcCenter ot-hide ot-fade-in" lang="en" aria-label="Preference center" role="region"><div role="dialog" aria-modal="true" style="height: 100%;" aria-label="Cookie Preference Center"><!-- Close Button --><div class="ot-pc-header"><!-- Logo Tag --><div class="ot-pc-logo" role="img" aria-label="Company Logo"><img alt="Company Logo" src="https://cdn.cookielaw.org/logos/static/ot_company_logo.png"></div></div><!-- Close Button --><div id="ot-pc-content" class="ot-pc-scrollbar"><div class="ot-optout-signal ot-hide"><div class="ot-optout-icon"><svg xmlns="http://www.w3.org/2000/svg"><path class="ot-floating-button__svg-fill" d="M14.588 0l.445.328c1.807 1.303 3.961 2.533 6.461 3.688 2.015.93 4.576 1.746 7.682 2.446 0 14.178-4.73 24.133-14.19 29.864l-.398.236C4.863 30.87 0 20.837 0 6.462c3.107-.7 5.668-1.516 7.682-2.446 2.709-1.251 5.01-2.59 6.906-4.016zm5.87 13.88a.75.75 0 00-.974.159l-5.475 6.625-3.005-2.997-.077-.067a.75.75 0 00-.983 1.13l4.172 4.16 6.525-7.895.06-.083a.75.75 0 00-.16-.973z" fill="#FFF" fill-rule="evenodd"></path></svg></div><span></span></div><h2 id="ot-pc-title">Cookie Preference Center</h2><div id="ot-pc-desc">We use cookies which are necessary to make our site work. We may also use additional cookies to analyse, improve and personalise our content and your digital experience. For more information, see our <a href="https://www.elsevier.com/legal/cookienotice/_nocache" target="_blank">Cookie Policy</a> and the list of <a href="https://support.google.com/admanager/answer/9012903" target="_blank">Google Ad-Tech Vendors</a>.
<br>
<br>
You may choose not to allow some types of cookies. However, blocking some types may impact your experience of our site and the services we are able to offer. See the different category headings below to find out more or change your settings.
<br>
</div><button id="accept-recommended-btn-handler">Allow all</button><section class="ot-sdk-row ot-cat-grp"><h3 id="ot-category-title"> Manage Consent Preferences</h3><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="1"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-1" aria-labelledby="ot-header-id-1 ot-status-id-1"></button><!-- Accordion header --><div class="ot-acc-hdr ot-always-active-group"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-1">Strictly Necessary Cookies</h4><div id="ot-status-id-1" class="ot-always-active">Always active</div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-1">These cookies are necessary for the website to function and cannot be switched off in our systems. They are usually only set in response to actions made by you which amount to a request for services, such as setting your privacy preferences, logging in or filling in forms. You can set your browser to block or alert you about these cookies, but some parts of the site will not then work. These cookies do not store any personally identifiable information.
<br><br></p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="1">Cookie Details List‎</button></div></div></div><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="3"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-3" aria-labelledby="ot-header-id-3"></button><!-- Accordion header --><div class="ot-acc-hdr"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-3">Functional Cookies</h4><div class="ot-tgl"><input type="checkbox" name="ot-group-id-3" id="ot-group-id-3" role="switch" class="category-switch-handler" data-optanongroupid="3" aria-labelledby="ot-header-id-3"> <label class="ot-switch" for="ot-group-id-3"><span class="ot-switch-nob"></span> <span class="ot-label-txt">Functional Cookies</span></label> </div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-3">These cookies enable the website to provide enhanced functionality and personalisation. They may be set by us or by third party providers whose services we have added to our pages. If you do not allow these cookies then some or all of these services may not function properly.</p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="3">Cookie Details List‎</button></div></div></div><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="2"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-2" aria-labelledby="ot-header-id-2"></button><!-- Accordion header --><div class="ot-acc-hdr"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-2">Performance Cookies</h4><div class="ot-tgl"><input type="checkbox" name="ot-group-id-2" id="ot-group-id-2" role="switch" class="category-switch-handler" data-optanongroupid="2" aria-labelledby="ot-header-id-2"> <label class="ot-switch" for="ot-group-id-2"><span class="ot-switch-nob"></span> <span class="ot-label-txt">Performance Cookies</span></label> </div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-2">These cookies allow us to count visits and traffic sources so we can measure and improve the performance of our site. They help us to know which pages are the most and least popular and see how visitors move around the site.</p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="2">Cookie Details List‎</button></div></div></div><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="4"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-4" aria-labelledby="ot-header-id-4"></button><!-- Accordion header --><div class="ot-acc-hdr"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-4">Targeting Cookies</h4><div class="ot-tgl"><input type="checkbox" name="ot-group-id-4" id="ot-group-id-4" role="switch" class="category-switch-handler" data-optanongroupid="4" aria-labelledby="ot-header-id-4"> <label class="ot-switch" for="ot-group-id-4"><span class="ot-switch-nob"></span> <span class="ot-label-txt">Targeting Cookies</span></label> </div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-4">These cookies may be set through our site by our advertising partners. They may be used by those companies to build a profile of your interests and show you relevant adverts on other sites. If you do not allow these cookies, you will experience less targeted advertising.</p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="4">Cookie Details List‎</button></div></div></div><!-- Groups sections starts --><!-- Group section ends --><!-- Accordion Group section starts --><!-- Accordion Group section ends --></section></div><section id="ot-pc-lst" class="ot-hide ot-hosts-ui ot-pc-scrollbar"><div id="ot-pc-hdr"><div id="ot-lst-title"><button class="ot-link-btn back-btn-handler" aria-label="Back"><svg id="ot-back-arw" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 444.531 444.531" xml:space="preserve"><title>Back Button</title><g><path fill="#656565" d="M213.13,222.409L351.88,83.653c7.05-7.043,10.567-15.657,10.567-25.841c0-10.183-3.518-18.793-10.567-25.835
                    l-21.409-21.416C323.432,3.521,314.817,0,304.637,0s-18.791,3.521-25.841,10.561L92.649,196.425
                    c-7.044,7.043-10.566,15.656-10.566,25.841s3.521,18.791,10.566,25.837l186.146,185.864c7.05,7.043,15.66,10.564,25.841,10.564
                    s18.795-3.521,25.834-10.564l21.409-21.412c7.05-7.039,10.567-15.604,10.567-25.697c0-10.085-3.518-18.746-10.567-25.978
                    L213.13,222.409z"></path></g></svg></button><h3>Cookie List</h3></div><div class="ot-lst-subhdr"><div class="ot-search-cntr"><p role="status" class="ot-scrn-rdr"></p><input id="vendor-search-handler" type="text" name="vendor-search-handler" placeholder="Search…" aria-label="Cookie list search"> <svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 -30 110 110" aria-hidden="true"><title>Search Icon</title><path fill="#2e3644" d="M55.146,51.887L41.588,37.786c3.486-4.144,5.396-9.358,5.396-14.786c0-12.682-10.318-23-23-23s-23,10.318-23,23
            s10.318,23,23,23c4.761,0,9.298-1.436,13.177-4.162l13.661,14.208c0.571,0.593,1.339,0.92,2.162,0.92
            c0.779,0,1.518-0.297,2.079-0.837C56.255,54.982,56.293,53.08,55.146,51.887z M23.984,6c9.374,0,17,7.626,17,17s-7.626,17-17,17
            s-17-7.626-17-17S14.61,6,23.984,6z"></path></svg></div><div class="ot-fltr-cntr"><button id="filter-btn-handler" aria-label="Filter" aria-haspopup="true"><svg role="presentation" aria-hidden="true" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 402.577 402.577" xml:space="preserve"><title>Filter Icon</title><g><path fill="#fff" d="M400.858,11.427c-3.241-7.421-8.85-11.132-16.854-11.136H18.564c-7.993,0-13.61,3.715-16.846,11.136
      c-3.234,7.801-1.903,14.467,3.999,19.985l140.757,140.753v138.755c0,4.955,1.809,9.232,5.424,12.854l73.085,73.083
      c3.429,3.614,7.71,5.428,12.851,5.428c2.282,0,4.66-0.479,7.135-1.43c7.426-3.238,11.14-8.851,11.14-16.845V172.166L396.861,31.413
      C402.765,25.895,404.093,19.231,400.858,11.427z"></path></g></svg></button></div><div id="ot-anchor"></div><section id="ot-fltr-modal"><div id="ot-fltr-cnt"><button id="clear-filters-handler">Clear</button><div class="ot-fltr-scrlcnt ot-pc-scrollbar"><div class="ot-fltr-opts" role="list"><div class="ot-fltr-opt" role="listitem"><div class="ot-chkbox"><input id="chkbox-id" type="checkbox" class="category-filter-handler"> <label for="chkbox-id"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div></div></div><div class="ot-fltr-btns"><button id="filter-apply-handler">Apply</button> <button id="filter-cancel-handler">Cancel</button></div></div></div></section></div></div><section id="ot-lst-cnt" class="ot-host-cnt ot-pc-scrollbar"><div id="ot-sel-blk"><div class="ot-sel-all"><div class="ot-sel-all-hdr"><span class="ot-consent-hdr">Consent</span> <span class="ot-li-hdr">Leg.Interest</span></div><div class="ot-sel-all-chkbox"><div class="ot-chkbox" id="ot-selall-hostcntr"><input id="select-all-hosts-groups-handler" type="checkbox"> <label for="select-all-hosts-groups-handler"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div><div class="ot-chkbox" id="ot-selall-vencntr"><input id="select-all-vendor-groups-handler" type="checkbox"> <label for="select-all-vendor-groups-handler"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div><div class="ot-chkbox" id="ot-selall-licntr"><input id="select-all-vendor-leg-handler" type="checkbox"> <label for="select-all-vendor-leg-handler"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div></div></div></div><div class="ot-sdk-row"><div class="ot-sdk-column"><ul id="ot-host-lst"></ul></div></div></section></section><div class="ot-pc-footer ot-pc-scrollbar"><div class="ot-btn-container"> <button class="save-preference-btn-handler onetrust-close-btn-handler">Confirm my choices</button></div><!-- Footer logo --><div class="ot-pc-footer-logo"><a href="https://www.onetrust.com/products/cookie-consent/" target="_blank" rel="noopener noreferrer" aria-label="Powered by OneTrust Opens in a new Tab"><img alt="Powered by Onetrust" src="https://cdn.cookielaw.org/logos/static/powered_by_logo.svg" title="Powered by OneTrust Opens in a new Tab"></a></div></div><!-- Cookie subgroup container --><!-- Vendor list link --><!-- Cookie lost link --><!-- Toggle HTML element --><!-- Checkbox HTML --><!-- plus minus--><!-- Arrow SVG element --><!-- Accordion basic element --><span class="ot-scrn-rdr" aria-atomic="true" aria-live="polite"></span><!-- Vendor Service container and item template --></div><iframe class="ot-text-resize" sandbox="allow-same-origin" title="onetrust-text-resize" style="position: absolute; top: -50000px; width: 100em;" aria-hidden="true"></iframe></div></div></body></html>