<html lang="en" class="pb-page fonts-loaded" data-request-id="2d582b68-6ade-4d9e-a3b5-732d85df9126" style="--article-sticky-top: 82.00019999999999px; --scroll-padding: 163.594px auto auto;"><head data-pb-dropzone="head"><meta name="pbContext" content=";product:product:elsevier\:product\:cell;website:website:cell-site;ctype:string:Journal Content;journal:journal:cell;pageGroup:string:Article View;page:string:Show Full Text;requestedJournal:journal:cell;article:article:pii\:S0092867420303391;wgroup:string:Default Website Group;subPage:string:Full Text;issue:issue:pii\:S0092867419X00094">




<!-- if there is any customization for Responsive Project widget  -->


























    
    







    
            
        



        
                <meta name="robots" content="noarchive,nocache">
            
    






    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=1, user-scalable=1">











        <meta property="og:title" content="G3BP1 Is a Tunable Switch that Triggers Phase Separation to Assemble Stress Granules">
    


        <meta property="og:type" content="Article">
    


        <meta property="og:url" content="https://www.cell.com/cell/abstract/S0092-8674(20)30339-1">
    



        <meta property="og:site_name" content="Cell">
    


        <meta property="og:image" content="https://www.cell.com/cms/asset/38d2d7c2-ec41-4c5e-aeb2-1dd31281e44d/fx1.jpg">
    


        <meta property="og:description" content="The molecular basis for stress granule assembly is a network of protein-RNA interactions
with G3BP1 as the central node and molecular switch that triggers phase separation
of granule components in response to free cytosolic RNA concentrations.">
    











        <meta name="twitter:title" content="G3BP1 Is a Tunable Switch that Triggers Phase Separation to Assemble Stress Granules">
    




        <meta name="twitter:image" content="https://www.cell.com/cms/asset/38d2d7c2-ec41-4c5e-aeb2-1dd31281e44d/fx1.jpg">
    


        <meta name="twitter:description" content="The molecular basis for stress granule assembly is a network of protein-RNA interactions
with G3BP1 as the central node and molecular switch that triggers phase separation
of granule components in response to free cytosolic RNA concentrations.">
    

        <meta name="twitter:card" content="summary_large_image">
    











    
    
            
            
        <link rel="canonical" href="https://www.cell.com/cell/fulltext/S0092-8674(20)30339-1">
    


<meta name="tdm-reservation" content="1">
<meta name="tdm-policy" content="https://www.elsevier.com/tdm/tdmrep-policy.json">

<title>G3BP1 Is a Tunable Switch that Triggers Phase Separation to Assemble Stress Granules: Cell</title><meta name="citation_journal_title" content="Cell"> <meta name="citation_journal_abbrev" content="Cell"> <meta name="citation_publisher" content="Elsevier"> <meta name="citation_title" content="G3BP1 Is a Tunable Switch that Triggers Phase Separation to Assemble Stress Granules"> <meta name="citation_date" content="2020/04/16"> <meta name="citation_volume" content="181"> <meta name="citation_issue" content="2"> <meta name="citation_firstpage" content="325"> <meta name="citation_lastpage" content="345.e28"> <meta name="citation_pmid" content="32302571"> <meta name="citation_abstract_html_url" content="https://www.cell.com/action/showAbstract?pii=S0092867420303391"> <meta name="citation_fulltext_html_url" content="https://www.cell.com/action/showFullText?pii=S0092867420303391"> <meta name="citation_pdf_url" content="https://www.cell.com/action/showPdf?pii=S0092867420303391"> <meta name="citation_issn" content="0092-8674"> <meta name="citation_issn" content="1097-4172"> <meta name="citation_language" content="English"> <meta name="citation_doi" content="10.1016/j.cell.2020.03.046"> <meta name="citation_pmid" content="32302571"> <meta name="citation_author" content="Peiguo Yang"><meta name="citation_author_institution" content="Department of Cell and Molecular Biology, St. Jude Children’s Research Hospital, Memphis, TN 38105, USA"><meta name="citation_author" content="Cécile Mathieu"><meta name="citation_author_institution" content="Department of Cell and Molecular Biology, St. Jude Children’s Research Hospital, Memphis, TN 38105, USA"><meta name="citation_author_institution" content="Howard Hughes Medical Institute, Chevy Chase, MD 20815, USA"><meta name="citation_author" content="Regina-Maria Kolaitis"><meta name="citation_author_institution" content="Department of Cell and Molecular Biology, St. Jude Children’s Research Hospital, Memphis, TN 38105, USA"><meta name="citation_author" content="Peipei Zhang"><meta name="citation_author_institution" content="Department of Cell and Molecular Biology, St. Jude Children’s Research Hospital, Memphis, TN 38105, USA"><meta name="citation_author" content="James Messing"><meta name="citation_author_institution" content="Department of Cell and Molecular Biology, St. Jude Children’s Research Hospital, Memphis, TN 38105, USA"><meta name="citation_author_institution" content="Howard Hughes Medical Institute, Chevy Chase, MD 20815, USA"><meta name="citation_author" content="Ugur Yurtsever"><meta name="citation_author_institution" content="Department of Cell and Molecular Biology, St. Jude Children’s Research Hospital, Memphis, TN 38105, USA"><meta name="citation_author_institution" content="Graduate School of Structure and Dynamics of Living Systems, Université Paris-Saclay, Gif-sur-Yvette, 91190, France"><meta name="citation_author" content="Zemin Yang"><meta name="citation_author_institution" content="Department of Cell and Molecular Biology, St. Jude Children’s Research Hospital, Memphis, TN 38105, USA"><meta name="citation_author" content="Jinjun Wu"><meta name="citation_author_institution" content="Department of Cell and Molecular Biology, St. Jude Children’s Research Hospital, Memphis, TN 38105, USA"><meta name="citation_author" content="Yuxin Li"><meta name="citation_author_institution" content="Center for Proteomics and Metabolomics, St. Jude Children’s Research Hospital, Memphis, TN 38105, USA"><meta name="citation_author" content="Qingfei Pan"><meta name="citation_author_institution" content="Department of Computational Biology, St. Jude Children’s Research Hospital, Memphis, TN 38105, USA"><meta name="citation_author" content="Jiyang Yu"><meta name="citation_author_institution" content="Department of Computational Biology, St. Jude Children’s Research Hospital, Memphis, TN 38105, USA"><meta name="citation_author" content="Erik W. Martin"><meta name="citation_author_institution" content="Department of Structural Biology, St. Jude Children’s Research Hospital, Memphis, TN 38105, USA"><meta name="citation_author" content="Tanja Mittag"><meta name="citation_author_institution" content="Department of Structural Biology, St. Jude Children’s Research Hospital, Memphis, TN 38105, USA"><meta name="citation_author" content="Hong Joo Kim"><meta name="citation_author_institution" content="Department of Cell and Molecular Biology, St. Jude Children’s Research Hospital, Memphis, TN 38105, USA"><meta name="citation_author" content="J. Paul Taylor"><meta name="citation_author_email" content="jpaul.taylor@stjude.org"><meta name="citation_author_institution" content="Department of Cell and Molecular Biology, St. Jude Children’s Research Hospital, Memphis, TN 38105, USA"><meta name="citation_author_institution" content="Howard Hughes Medical Institute, Chevy Chase, MD 20815, USA"><meta name="citation_keywords" content="G3BP1; core stress granule network; biomolecular condensate; stress granule; multivalency; liquid-liquid phase separation; Intrinsically disordered protein; molecular switch; membraneless organelle; cooperativity"><meta name="citation_reference" content="citation_title=Compositional control of phase-separated cellular bodies; citation_author=S.F. Banani; citation_author=A.M. Rice; citation_author=W.B. Peeples; citation_author=Y. Lin; citation_author=S. Jain; citation_author=R. Parker; citation_author=M.K. Rosen; citation_journal_title=Cell; citation_volume=166; citation_firstpage=651; citation_lastpage=663; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Biomolecular condensates: organizers of cellular biochemistry; citation_author=S.F. Banani; citation_author=H.O. Lee; citation_author=A.A. Hyman; citation_author=M.K. Rosen; citation_journal_title=Nat. Rev. Mol. Cell Biol.; citation_volume=18; citation_firstpage=285; citation_lastpage=298; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Emergence of scaling in random networks; citation_author=A.L. Barabasi; citation_author=R. Albert; citation_journal_title=Science; citation_volume=286; citation_firstpage=509; citation_lastpage=512; citation_publication_date=1999; "><meta name="citation_reference" content="citation_title=Structural characterization of flexible proteins using small-angle X-ray scattering; citation_author=P. Bernadó; citation_author=E. Mylonas; citation_author=M.V. Petoukhov; citation_author=M. Blackledge; citation_author=D.I. Svergun; citation_journal_title=J.&nbsp;Am. Chem. Soc.; citation_volume=129; citation_firstpage=5656; citation_lastpage=5664; citation_publication_date=2007; "><meta name="citation_reference" content="citation_title=Macromolecular size-and-shape distributions by sedimentation velocity analytical ultracentrifugation; citation_author=P.H. Brown; citation_author=P. Schuck; citation_journal_title=Biophys. J.; citation_volume=90; citation_firstpage=4651; citation_lastpage=4661; citation_publication_date=2006; "><meta name="citation_reference" content="citation_title=mRNP granules. Assembly, function, and connections with disease; citation_author=J.R. Buchan; citation_journal_title=RNA Biol.; citation_volume=11; citation_firstpage=1019; citation_lastpage=1030; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Eukaryotic stress granules are cleared by autophagy and Cdc48/VCP function; citation_author=J.R. Buchan; citation_author=R.M. Kolaitis; citation_author=J.P. Taylor; citation_author=R. Parker; citation_journal_title=Cell; citation_volume=153; citation_firstpage=1461; citation_lastpage=1474; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=Density and apparent specific volume of proteins; citation_inbook_title=Proteins, Amino Acids, and Peptides; citation_author=E.J. Cohn; citation_author=J.T. Edsall; citation_firstpage=370; citation_lastpage=381; citation_publication_date=1943; citation_publisher=Van Nostrand-Reinhold; "><meta name="citation_reference" content="citation_title=Unmasking the roles of N- and C-terminal flanking sequences from exon 1 of huntingtin as modulators of polyglutamine aggregation; citation_author=S.L. Crick; citation_author=K.M. Ruff; citation_author=K. Garai; citation_author=C. Frieden; citation_author=R.V. Pappu; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=110; citation_firstpage=20075; citation_lastpage=20080; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=Nuclear bodies: multifunctional companions of the genome; citation_author=M. Dundr; citation_journal_title=Curr. Opin. Cell Biol.; citation_volume=24; citation_firstpage=415; citation_lastpage=422; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=N6-methyladenosine (m6A) recruits and repels proteins to regulate mRNA homeostasis; citation_author=R.R. Edupuganti; citation_author=S. Geiger; citation_author=R.G.H. Lindeboom; citation_author=H. Shi; citation_author=P.J. Hsu; citation_author=Z. Lu; citation_author=S.Y. Wang; citation_author=M.P.A. Baltissen; citation_author=P.W.T.C. Jansen; citation_author=M. Rossa; citation_author=et al.; citation_journal_title=Nat. Struct. Mol. Biol.; citation_volume=24; citation_firstpage=870; citation_lastpage=878; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Monomer-dimer equilibrium in glutathione transferases: a critical re-examination; citation_author=R. Fabrini; citation_author=A. De Luca; citation_author=L. Stella; citation_author=G. Mei; citation_author=B. Orioni; citation_author=S. Ciccone; citation_author=G. Federici; citation_author=M. Lo Bello; citation_author=G. Ricci; citation_journal_title=Biochemistry; citation_volume=48; citation_firstpage=10473; citation_lastpage=10482; citation_publication_date=2009; "><meta name="citation_reference" content="citation_title=Profilin 1 associates with stress granules and ALS-linked mutations alter stress granule dynamics; citation_author=M.D. Figley; citation_author=G. Bieri; citation_author=R.M. Kolaitis; citation_author=J.P. Taylor; citation_author=A.D. Gitler; citation_journal_title=J.&nbsp;Neurosci.; citation_volume=34; citation_firstpage=8083; citation_lastpage=8097; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=A novel phosphorylation-dependent RNase activity of GAP-SH3 binding protein: a potential link between signal transduction and RNA stability; citation_author=I.E. Gallouzi; citation_author=F. Parker; citation_author=K. Chebli; citation_author=F. Maurier; citation_author=E. Labourier; citation_author=I. Barlat; citation_author=J.P. Capony; citation_author=B. Tocque; citation_author=J. Tazi; citation_journal_title=Mol. Cell. Biol.; citation_volume=18; citation_firstpage=3956; citation_lastpage=3965; citation_publication_date=1998; "><meta name="citation_reference" content="citation_title=Stress granule assembly is mediated by prion-like aggregation of TIA-1; citation_author=N. Gilks; citation_author=N. Kedersha; citation_author=M. Ayodele; citation_author=L. Shen; citation_author=G. Stoecklin; citation_author=L.M. Dember; citation_author=P. Anderson; citation_journal_title=Mol. Biol. Cell; citation_volume=15; citation_firstpage=5383; citation_lastpage=5398; citation_publication_date=2004; "><meta name="citation_reference" content="citation_title=GUItars: a GUI tool for analysis of high-throughput RNA interference screening data; citation_author=A.N. Goktug; citation_author=S.S. Ong; citation_author=T. Chen; citation_journal_title=PLoS ONE; citation_volume=7; citation_firstpage=e49386; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=The molecular language of membraneless organelles; citation_author=E. Gomes; citation_author=J. Shorter; citation_journal_title=J.&nbsp;Biol. Chem.; citation_volume=294; citation_firstpage=7115; citation_lastpage=7127; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=RNA-Induced Conformational Switching and Clustering of G3BP Drive Stress Granule Assembly by Condensation; citation_author=J. Guillén-Boixet; citation_author=A. Kopach; citation_author=A.S> Holehouse; citation_author=S. Wittmann; citation_author=M. Jahnel; citation_author=R. Schlüßler; citation_author=K. Kim; citation_author=I.R.E.A. Trussina; citation_author=J. Wang; citation_author=D. Mateju; citation_author=et al.; citation_journal_title=Cell; citation_volume=181; citation_firstpage=346; citation_lastpage=361; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=BioXTAS RAW: improvements to a free open-source program for small-angle X-ray scattering data reduction and analysis; citation_author=J.B. Hopkins; citation_author=R.E. Gillilan; citation_author=S. Skou; citation_journal_title=J.&nbsp;Appl. Cryst.; citation_volume=50; citation_firstpage=1545; citation_lastpage=1553; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Systematic and integrative analysis of large gene lists using DAVID bioinformatics resources; citation_author=W. Huang da; citation_author=B.T. Sherman; citation_author=R.A. Lempicki; citation_journal_title=Nat. Protoc.; citation_volume=4; citation_firstpage=44; citation_lastpage=57; citation_publication_date=2009; "><meta name="citation_reference" content="citation_title=VMD: visual molecular dynamics; citation_author=W. Humphrey; citation_author=A. Dalke; citation_author=K. Schulten; citation_journal_title=J.&nbsp;Mol. Graph.; citation_volume=14; citation_firstpage=33; citation_lastpage=38; citation_publication_date=1996; "><meta name="citation_reference" content="citation_title=The BioPlex network: a systematic exploration of the human interactome; citation_author=E.L. Huttlin; citation_author=L. Ting; citation_author=R.J. Bruckner; citation_author=F. Gebreab; citation_author=M.P. Gygi; citation_author=J. Szpyt; citation_author=S. Tam; citation_author=G. Zarraga; citation_author=G. Colby; citation_author=K. Baltier; citation_author=et al.; citation_journal_title=Cell; citation_volume=162; citation_firstpage=425; citation_lastpage=440; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Stress granules and processing bodies in translational control; citation_author=P. Ivanov; citation_author=N. Kedersha; citation_author=P. Anderson; citation_journal_title=Cold Spring Harb. Perspect. Biol.; citation_volume=11; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=ATPase-modulated stress granules contain a diverse proteome and substructure; citation_author=S. Jain; citation_author=J.R. Wheeler; citation_author=R.W. Walters; citation_author=A. Agrawal; citation_author=A. Barsic; citation_author=R. Parker; citation_journal_title=Cell; citation_volume=164; citation_firstpage=487; citation_lastpage=498; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Lethality and centrality in protein networks; citation_author=H. Jeong; citation_author=S.P. Mason; citation_author=A.L. Barabási; citation_author=Z.N. Oltvai; citation_journal_title=Nature; citation_volume=411; citation_firstpage=41; citation_lastpage=42; citation_publication_date=2001; "><meta name="citation_reference" content="citation_title=Tobacco etch virus protease: mechanism of autolysis and rational design of stable mutants with wild-type catalytic proficiency; citation_author=R.B. Kapust; citation_author=J. Tözsér; citation_author=J.D. Fox; citation_author=D.E. Anderson; citation_author=S. Cherry; citation_author=T.D. Copeland; citation_author=D.S. Waugh; citation_journal_title=Protein Eng.; citation_volume=14; citation_firstpage=993; citation_lastpage=1000; citation_publication_date=2001; "><meta name="citation_reference" content="citation_title=G3BP-Caprin1-USP10 complexes mediate stress granule condensation and associate with 40S subunits; citation_author=N. Kedersha; citation_author=M.D. Panas; citation_author=C.A. Achorn; citation_author=S. Lyons; citation_author=S. Tisdale; citation_author=T. Hickman; citation_author=M. Thomas; citation_author=J. Lieberman; citation_author=G.M. McInerney; citation_author=P. Ivanov; citation_author=P. Anderson; citation_journal_title=J.&nbsp;Cell Biol.; citation_volume=212; citation_firstpage=845; citation_lastpage=860; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=The stress granule transcriptome reveals principles of mRNA accumulation in stress granules; citation_author=A. Khong; citation_author=T. Matheny; citation_author=S. Jain; citation_author=S.F. Mitchell; citation_author=J.R. Wheeler; citation_author=R. Parker; citation_journal_title=Mol. Cell; citation_volume=68; citation_firstpage=808; citation_lastpage=820.e5; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Neuronal RNA granules: movers and makers; citation_author=M.A. Kiebler; citation_author=G.J. Bassell; citation_journal_title=Neuron; citation_volume=51; citation_firstpage=685; citation_lastpage=690; citation_publication_date=2006; "><meta name="citation_reference" content="citation_title=Improved radiation dose efficiency in solution SAXS using a sheath flow sample environment; citation_author=N. Kirby; citation_author=N. Cowieson; citation_author=A.M. Hawley; citation_author=S.T. Mudie; citation_author=D.J. McGillivray; citation_author=M. Kusel; citation_author=V. Samardzic-Boban; citation_author=T.M. Ryan; citation_journal_title=Acta Crystallogr. D Struct. Biol.; citation_volume=72; citation_firstpage=1254; citation_lastpage=1266; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=The deacetylase HDAC6 is a novel critical component of stress granules involved in the stress response; citation_author=S. Kwon; citation_author=Y. Zhang; citation_author=P. Matthias; citation_journal_title=Genes Dev.; citation_volume=21; citation_firstpage=3381; citation_lastpage=3394; citation_publication_date=2007; "><meta name="citation_reference" content="citation_title=Directed evolution of APEX2 for electron microscopy and proximity labeling.; citation_author=S.S. Lam; citation_author=J.D. Martell; citation_author=K.J. Kamer; citation_author=T.J. Deerinck; citation_author=M.H. Ellisman; citation_author=V.K. Mootha; citation_author=A.Y. Ting; citation_journal_title=Nat Methods; citation_volume=12; citation_firstpage=51; citation_lastpage=54; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Phase transitions in the assembly of multivalent signalling proteins; citation_author=P. Li; citation_author=S. Banjade; citation_author=H.C. Cheng; citation_author=S. Kim; citation_author=B. Chen; citation_author=L. Guo; citation_author=M. Llaguno; citation_author=J.V. Hollingsworth; citation_author=D.S. King; citation_author=S.F. Banani; citation_author=et al.; citation_journal_title=Nature; citation_volume=483; citation_firstpage=336; citation_lastpage=340; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=Formation and maturation of phase-separated liquid droplets by RNA-binding proteins; citation_author=Y. Lin; citation_author=D.S. Protter; citation_author=M.K. Rosen; citation_author=R. Parker; citation_journal_title=Mol. Cell; citation_volume=60; citation_firstpage=208; citation_lastpage=219; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=TIA1 mutations in amyotrophic lateral sclerosis and frontotemporal dementia promote phase separation and alter stress granule dynamics; citation_author=I.R. Mackenzie; citation_author=A.M. Nicholson; citation_author=M. Sarkar; citation_author=J. Messing; citation_author=M.D. Purice; citation_author=C. Pottier; citation_author=K. Annu; citation_author=M. Baker; citation_author=R.B. Perkerson; citation_author=A. Kurti; citation_author=et al.; citation_journal_title=Neuron; citation_volume=95; citation_firstpage=808; citation_lastpage=816.e9; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Context-dependent and disease-specific diversity in protein interactions within stress granules; citation_author=S. Markmiller; citation_author=S. Soltanieh; citation_author=K.L. Server; citation_author=R. Mak; citation_author=W. Jin; citation_author=M.Y. Fang; citation_author=E.C. Luo; citation_author=F. Krach; citation_author=D. Yang; citation_author=A. Sen; citation_author=et al.; citation_journal_title=Cell; citation_volume=172; citation_firstpage=590; citation_lastpage=604.e13; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Sequence determinants of the conformational properties of an intrinsically disordered protein prior to and upon multisite phosphorylation; citation_author=E.W. Martin; citation_author=A.S. Holehouse; citation_author=C.R. Grace; citation_author=A. Hughes; citation_author=R.V. Pappu; citation_author=T. Mittag; citation_journal_title=J.&nbsp;Am. Chem. Soc.; citation_volume=138; citation_firstpage=15323; citation_lastpage=15335; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Preferential binding of a stable G3BP ribonucleoprotein complex to intron-retaining transcripts in mouse brain and modulation of their expression in the cerebellum; citation_author=S. Martin; citation_author=N. Bellora; citation_author=J. González-Vallinas; citation_author=M. Irimia; citation_author=K. Chebli; citation_author=M. de Toledo; citation_author=M. Raabe; citation_author=E. Eyras; citation_author=H. Urlaub; citation_author=B.J. Blencowe; citation_author=J. Tazi; citation_journal_title=J.&nbsp;Neurochem.; citation_volume=139; citation_firstpage=349; citation_lastpage=368; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Higher-order oligomerization promotes localization of SPOP to liquid nuclear speckles; citation_author=M.R. Marzahn; citation_author=S. Marada; citation_author=J. Lee; citation_author=A. Nourse; citation_author=S. Kenrick; citation_author=H. Zhao; citation_author=G. Ben-Nissan; citation_author=R.M. Kolaitis; citation_author=J.L. Peters; citation_author=S. Pounds; citation_author=et al.; citation_journal_title=EMBO J.; citation_volume=35; citation_firstpage=1254; citation_lastpage=1275; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=The TIA1 RNA-binding protein family regulates EIF2AK2-mediated stress response and cell cycle progression; citation_author=C. Meyer; citation_author=A. Garzia; citation_author=M. Mazzola; citation_author=S. Gerstberger; citation_author=H. Molina; citation_author=T. Tuschl; citation_journal_title=Mol. Cell; citation_volume=69; citation_firstpage=622; citation_lastpage=635.e6; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Phase separation by low complexity domains promotes stress granule assembly and drives pathological fibrillization; citation_author=A. Molliex; citation_author=J. Temirov; citation_author=J. Lee; citation_author=M. Coughlin; citation_author=A.P. Kanagaraj; citation_author=H.J. Kim; citation_author=T. Mittag; citation_author=J.P. Taylor; citation_journal_title=Cell; citation_volume=163; citation_firstpage=123; citation_lastpage=133; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Bridging biophysics and neurology: aberrant phase transitions in neurodegenerative disease; citation_author=N.B. Nedelsky; citation_author=J.P. Taylor; citation_journal_title=Nat. Rev. Neurol.; citation_volume=15; citation_firstpage=272; citation_lastpage=286; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Phase transition of a disordered nuage protein generates environmentally responsive membraneless organelles; citation_author=T.J. Nott; citation_author=E. Petsalaki; citation_author=P. Farber; citation_author=D. Jervis; citation_author=E. Fussner; citation_author=A. Plochowietz; citation_author=T.D. Craggs; citation_author=D.P. Bazett-Jones; citation_author=T. Pawson; citation_author=J.D. Forman-Kay; citation_author=A.J. Baldwin; citation_journal_title=Mol. Cell; citation_volume=57; citation_firstpage=936; citation_lastpage=947; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=A functional RNAi screen links O-GlcNAc modification of ribosomal proteins to stress granule and processing body assembly; citation_author=T. Ohn; citation_author=N. Kedersha; citation_author=T. Hickman; citation_author=S. Tisdale; citation_author=P. Anderson; citation_journal_title=Nat. Cell Biol.; citation_volume=10; citation_firstpage=1224; citation_lastpage=1231; citation_publication_date=2008; "><meta name="citation_reference" content="citation_title=Stability of the human pregnane X receptor is regulated by E3 ligase UBR5 and serine/threonine kinase DYRK2; citation_author=S.S. Ong; citation_author=A.N. Goktug; citation_author=A. Elias; citation_author=J. Wu; citation_author=D. Saunders; citation_author=T. Chen; citation_journal_title=Biochem J; citation_volume=459; citation_firstpage=193; citation_lastpage=203; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Critical role of an antiviral stress granule containing RIG-I and PKR in viral detection and innate immunity; citation_author=K. Onomoto; citation_author=M. Jogi; citation_author=J.S. Yoo; citation_author=R. Narita; citation_author=S. Morimoto; citation_author=A. Takemura; citation_author=S. Sambhara; citation_author=A. Kawaguchi; citation_author=S. Osari; citation_author=K. Nagata; citation_author=et al.; citation_journal_title=PLoS ONE; citation_volume=7; citation_firstpage=e43031; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=Viral and cellular proteins containing FGDF motifs bind G3BP to block stress granule formation; citation_author=M.D. Panas; citation_author=T. Schulte; citation_author=B. Thaa; citation_author=T. Sandalova; citation_author=N. Kedersha; citation_author=A. Achour; citation_author=G.M. McInerney; citation_journal_title=PLoS Pathog.; citation_volume=11; citation_firstpage=e1004659; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Mechanistic insights into mammalian stress granule dynamics; citation_author=M.D. Panas; citation_author=P. Ivanov; citation_author=P. Anderson; citation_journal_title=J.&nbsp;Cell Biol.; citation_volume=215; citation_firstpage=313; citation_lastpage=323; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Phosphorylation of G3BP1-S149 does not influence stress granule assembly; citation_author=M.D. Panas; citation_author=N. Kedersha; citation_author=T. Schulte; citation_author=R.M. Branca; citation_author=P. Ivanov; citation_author=P. Anderson; citation_journal_title=J.&nbsp;Cell Biol.; citation_volume=218; citation_firstpage=2425; citation_lastpage=2432; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=A liquid-to-solid phase transition of the ALS protein FUS accelerated by disease mutation; citation_author=A. Patel; citation_author=H.O. Lee; citation_author=L. Jawerth; citation_author=S. Maharana; citation_author=M. Jahnel; citation_author=M.Y. Hein; citation_author=S. Stoynov; citation_author=J. Mahamid; citation_author=S. Saha; citation_author=T.M. Franzmann; citation_author=et al.; citation_journal_title=Cell; citation_volume=162; citation_firstpage=1066; citation_lastpage=1077; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Profilin reduces aggregation and phase separation of huntingtin N-terminal fragments by preferentially binding to soluble monomers and oligomers; citation_author=A.E. Posey; citation_author=K.M. Ruff; citation_author=T.S. Harmon; citation_author=S.L. Crick; citation_author=A. Li; citation_author=M.I. Diamond; citation_author=R.V. Pappu; citation_journal_title=J.&nbsp;Biol. Chem.; citation_volume=293; citation_firstpage=3734; citation_lastpage=3746; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Genome engineering using the CRISPR-Cas9 system; citation_author=F.A. Ran; citation_author=P.D. Hsu; citation_author=J. Wright; citation_author=V. Agarwala; citation_author=D.A. Scott; citation_author=F. Zhang; citation_journal_title=Nat. Protoc.; citation_volume=8; citation_firstpage=2281; citation_lastpage=2308; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=A ligand-reversible dimerization system for controlling protein-protein interactions; citation_author=C.T. Rollins; citation_author=V.M. Rivera; citation_author=D.N. Woolfson; citation_author=T. Keenan; citation_author=M. Hatada; citation_author=S.E. Adams; citation_author=L.J. Andrade; citation_author=D. Yaeger; citation_author=M.R. van Schravendijk; citation_author=D.A. Holt; citation_author=et al.; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=97; citation_firstpage=7096; citation_lastpage=7101; citation_publication_date=2000; "><meta name="citation_reference" content="citation_title=Axonal G3BP1 stress granule protein limits axonal mRNA translation and nerve regeneration; citation_author=P.K. Sahoo; citation_author=S.J. Lee; citation_author=P.B. Jaiswal; citation_author=S. Alber; citation_author=A.N. Kar; citation_author=S. Miller-Randolph; citation_author=E.E. Taylor; citation_author=T. Smith; citation_author=B. Singh; citation_author=T.S. Ho; citation_author=et al.; citation_journal_title=Nat. Commun.; citation_volume=9; citation_firstpage=3358; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=DDX3X acts as a live-or-die checkpoint in stressed cells by regulating NLRP3 inflammasome; citation_author=P. Samir; citation_author=S. Kesavardhana; citation_author=D.M. Patmore; citation_author=S. Gingras; citation_author=R.K.S. Malireddi; citation_author=R. Karki; citation_author=C.S. Guy; citation_author=B. Briard; citation_author=D.E. Place; citation_author=A. Bhattacharya; citation_author=et al.; citation_journal_title=Nature; citation_volume=573; citation_firstpage=590; citation_lastpage=594; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Competing Protein-RNA Interaction Networks Control Multiphase Intracellular Organization; citation_author=D.W. Sanders; citation_author=N. Kedersha; citation_author=D.S.W. Lee; citation_author=A.R. Strom; citation_author=V. Drake; citation_author=J.A. Riback; citation_author=D. Bracha; citation_author=J.M. Eeftens; citation_author=A. Iwanicki; citation_author=A. Wang; citation_author=M.-T. Wei; citation_author=et al.; citation_journal_title=Cell; citation_volume=181; citation_firstpage=306; citation_lastpage=324; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Size-distribution analysis of macromolecules by sedimentation velocity ultracentrifugation and lamm equation modeling; citation_author=P. Schuck; citation_journal_title=Biophys. J.; citation_volume=78; citation_firstpage=1606; citation_lastpage=1619; citation_publication_date=2000; "><meta name="citation_reference" content="citation_title=Cytoscape: a software environment for integrated models of biomolecular interaction networks; citation_author=P. Shannon; citation_author=A. Markiel; citation_author=O. Ozier; citation_author=N.S. Baliga; citation_author=J.T. Wang; citation_author=D. Ramage; citation_author=N. Amin; citation_author=B. Schwikowski; citation_author=T. Ideker; citation_journal_title=Genome Res.; citation_volume=13; citation_firstpage=2498; citation_lastpage=2504; citation_publication_date=2003; "><meta name="citation_reference" content="citation_title=STRING v10: protein-protein interaction networks, integrated over the tree of life; citation_author=D. Szklarczyk; citation_author=A. Franceschini; citation_author=S. Wyder; citation_author=K. Forslund; citation_author=D. Heller; citation_author=J. Huerta-Cepas; citation_author=M. Simonovic; citation_author=A. Roth; citation_author=A. Santos; citation_author=K.P. Tsafou; citation_author=et al.; citation_journal_title=Nucleic Acids Res.; citation_volume=43; citation_firstpage=D447; citation_lastpage=D452; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Inhibition of mTORC1 by astrin and stress granules prevents apoptosis in cancer cells; citation_author=K. Thedieck; citation_author=B. Holzwarth; citation_author=M.T. Prentzell; citation_author=C. Boehlke; citation_author=K. Kläsener; citation_author=S. Ruf; citation_author=A.G. Sonntag; citation_author=L. Maerz; citation_author=S.N. Grellscheid; citation_author=E. Kremmer; citation_author=et al.; citation_journal_title=Cell; citation_volume=154; citation_firstpage=859; citation_lastpage=874; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=The RasGAP-associated endoribonuclease G3BP assembles stress granules; citation_author=H. Tourrière; citation_author=K. Chebli; citation_author=L. Zekri; citation_author=B. Courselaud; citation_author=J.M. Blanchard; citation_author=E. Bertrand; citation_author=J. Tazi; citation_journal_title=J.&nbsp;Cell Biol.; citation_volume=160; citation_firstpage=823; citation_lastpage=831; citation_publication_date=2003; "><meta name="citation_reference" content="citation_title=Cytoplasmic RNA granules and viral infection; citation_author=W.C. Tsai; citation_author=R.E. Lloyd; citation_journal_title=Annu. Rev. Virol.; citation_volume=1; citation_firstpage=147; citation_lastpage=170; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Arginine demethylation of G3BP1 promotes stress granule assembly; citation_author=W.C. Tsai; citation_author=S. Gayatri; citation_author=L.C. Reineke; citation_author=G. Sbardella; citation_author=M.T. Bedford; citation_author=R.E. Lloyd; citation_journal_title=J.&nbsp;Biol. Chem.; citation_volume=291; citation_firstpage=22671; citation_lastpage=22685; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Cancer-associated DDX3X mutations drive stress granule assembly and impair global translation; citation_author=Y.A. Valentin-Vega; citation_author=Y.D. Wang; citation_author=M. Parker; citation_author=D.M. Patmore; citation_author=A. Kanagaraj; citation_author=J. Moore; citation_author=M. Rusch; citation_author=D. Finkelstein; citation_author=D.W. Ellison; citation_author=R.J. Gilbertson; citation_author=et al.; citation_journal_title=Sci. Rep.; citation_volume=6; citation_firstpage=25996; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Emerging roles for intermolecular RNA-RNA interactions in RNP assemblies; citation_author=B. Van Treeck; citation_author=R. Parker; citation_journal_title=Cell; citation_volume=174; citation_firstpage=791; citation_lastpage=802; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=RNA self-assembly contributes to stress granule formation and defining the stress granule transcriptome; citation_author=B. Van Treeck; citation_author=D.S.W. Protter; citation_author=T. Matheny; citation_author=A. Khong; citation_author=C.D. Link; citation_author=R. Parker; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=115; citation_firstpage=2734; citation_lastpage=2739; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Crystal structures of the human G3BP1 NTF2-like domain visualize FxFG Nup repeat specificity; citation_author=T. Vognsen; citation_author=I.R. Møller; citation_author=O. Kristensen; citation_journal_title=PLoS ONE; citation_volume=8; citation_firstpage=e80947; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=Social Network Analysis: Methods and Applications; citation_author=S. Wasserman; citation_author=K. Faust; citation_publication_date=1994; citation_publisher=Cambridge University; "><meta name="citation_reference" content="citation_title=SWISS-MODEL: homology modelling of protein structures and complexes; citation_author=A. Waterhouse; citation_author=M. Bertoni; citation_author=S. Bienert; citation_author=G. Studer; citation_author=G. Tauriello; citation_author=R. Gumienny; citation_author=F.T. Heer; citation_author=T.A.P. de Beer; citation_author=C. Rempfer; citation_author=L. Bordoli; citation_author=et al.; citation_journal_title=Nucleic Acids Res.; citation_volume=46; citation_number=W1; citation_firstpage=W296; citation_lastpage=W303; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Dual specificity kinase DYRK3 couples stress granule condensation/dissolution to mTORC1 signaling; citation_author=F. Wippich; citation_author=B. Bodenmiller; citation_author=M.G. Trajkovska; citation_author=S. Wanka; citation_author=R. Aebersold; citation_author=L. Pelkmans; citation_journal_title=Cell; citation_volume=152; citation_firstpage=791; citation_lastpage=805; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=Molecular basis for the specific and multivariant recognitions of RNA substrates by human hnRNP A2/B1; citation_author=B. Wu; citation_author=S. Su; citation_author=D.P. Patil; citation_author=H. Liu; citation_author=J. Gan; citation_author=S.R. Jaffrey; citation_author=J. Ma; citation_journal_title=Nat. Commun.; citation_volume=9; citation_firstpage=420; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=High-density proximity mapping reveals the subcellular organization of mRNA-associated granules and bodies; citation_author=J.Y. Youn; citation_author=W.H. Dunham; citation_author=S.J. Hong; citation_author=J.D.R. Knight; citation_author=M. Bashkurov; citation_author=G.I. Chen; citation_author=H. Bagci; citation_author=B. Rathod; citation_author=G. MacLeod; citation_author=S.W.M. Eng; citation_author=et al.; citation_journal_title=Mol. Cell; citation_volume=69; citation_firstpage=517; citation_lastpage=532.e11; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=A new method with flexible and balanced control of false negatives and false positives for hit selection in RNA interference high-throughput screening assays; citation_author=X.D. Zhang; citation_journal_title=J.&nbsp;Biomol. Screen.; citation_volume=12; citation_firstpage=645; citation_lastpage=655; citation_publication_date=2007; "><meta name="citation_reference" content="citation_title=Illustration of SSMD, z score, SSMD∗, z∗ score, and t statistic for hit selection in RNAi high-throughput screens; citation_author=X.D. Zhang; citation_journal_title=J.&nbsp;Biomol. Screen.; citation_volume=16; citation_firstpage=775; citation_lastpage=785; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=A general framework for weighted gene co-expression network analysis; citation_author=B. Zhang; citation_author=S. Horvath; citation_journal_title=Stat. Appl. Genet. Mol. Biol.; citation_volume=4; citation_firstpage=Article17; citation_publication_date=2005; "><meta name="citation_reference" content="citation_title=Robust statistical methods for hit selection in RNA interference high-throughput screening experiments; citation_author=X.D. Zhang; citation_author=X.C. Yang; citation_author=N. Chung; citation_author=A. Gates; citation_author=E. Stec; citation_author=P. Kunapuli; citation_author=D.J. Holder; citation_author=M. Ferrer; citation_author=A.S. Espeseth; citation_journal_title=Pharmacogenomics; citation_volume=7; citation_firstpage=299; citation_lastpage=309; citation_publication_date=2006; "><meta name="citation_reference" content="citation_title=Stress granule assembly disrupts nucleocytoplasmic transport; citation_author=K. Zhang; citation_author=J.G. Daigle; citation_author=K.M. Cunningham; citation_author=A.N. Coyne; citation_author=K. Ruan; citation_author=J.C. Grima; citation_author=K.E. Bowen; citation_author=H. Wadhwa; citation_author=P. Yang; citation_author=F. Rigo; citation_author=et al.; citation_journal_title=Cell; citation_volume=173; citation_firstpage=958; citation_lastpage=971.e17; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Overview of current methods in sedimentation velocity and sedimentation equilibrium analytical ultracentrifugation; citation_author=H. Zhao; citation_author=C.A. Brautigam; citation_author=R. Ghirlando; citation_author=P. Schuck; citation_journal_title=Curr. Protoc. Protein Sci.; citation_volume=Chapter 20; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=A multilaboratory comparison of calibration accuracy and the performance of external references in analytical ultracentrifugation; citation_author=H. Zhao; citation_author=R. Ghirlando; citation_author=C. Alfonso; citation_author=F. Arisaka; citation_author=I. Attali; citation_author=D.L. Bain; citation_author=M.M. Bakhtina; citation_author=D.F. Becker; citation_author=G.J. Bedwell; citation_author=A. Bekdemir; citation_author=et al.; citation_journal_title=PLoS ONE; citation_volume=10; citation_firstpage=e0126420; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Post-transcriptional gene regulation by mRNA modifications; citation_author=B.S. Zhao; citation_author=I.A. Roundtree; citation_author=C. He; citation_journal_title=Nat. Rev. Mol. Cell Biol.; citation_volume=18; citation_firstpage=31; citation_lastpage=42; citation_publication_date=2017; "><meta name="citation_abstract" content="<h2>Summary</h2><p>The mechanisms underlying ribonucleoprotein (RNP) granule assembly, including the basis for establishing and maintaining RNP granules with distinct composition, are unknown. One prominent type of RNP granule is the stress granule (SG), a dynamic and reversible cytoplasmic assembly formed in eukaryotic cells in response to stress. Here, we show that SGs assemble through liquid-liquid phase separation (LLPS) arising from interactions distributed unevenly across a core protein-RNA interaction network. The central node of this network is G3BP1, which functions as a molecular switch that triggers RNA-dependent LLPS in response to a rise in intracellular free RNA concentrations. Moreover, we show that interplay between three distinct intrinsically disordered regions (IDRs) in G3BP1 regulates its intrinsic propensity for LLPS, and this is fine-tuned by phosphorylation within the IDRs. Further regulation of SG assembly arises through positive or negative cooperativity by extrinsic G3BP1-binding factors that strengthen or weaken, respectively, the core SG network.</p>"><meta charset="UTF-8"><!--

    Pagebuild admin UI
-->



<script type="text/javascript" src="https://bam.nr-data.net/1/16e2192192?a=916957862&amp;sa=1&amp;v=1177.96a4d39&amp;t=Unnamed%20Transaction&amp;rst=8037&amp;ck=1&amp;ref=https://www.cell.com/cell/fulltext/S0092-8674(20)30339-1&amp;be=5054&amp;fe=7585&amp;dc=7306&amp;af=err,xhr,stn,ins,spa&amp;perf=%7B%22timing%22:%7B%22of%22:1749586003329,%22n%22:0,%22f%22:4131,%22dn%22:4132,%22dne%22:4132,%22c%22:4132,%22s%22:4132,%22ce%22:4132,%22rq%22:4133,%22rp%22:5023,%22rpe%22:6550,%22dl%22:5028,%22di%22:7121,%22ds%22:7306,%22de%22:7542,%22dc%22:7585,%22l%22:7585,%22le%22:7650%7D,%22navigation%22:%7B%7D%7D&amp;fp=6200&amp;fcp=6200&amp;jsonp=NREUM.setToken"></script><script src="https://js-agent.newrelic.com/nr-spa-1177.min.js"></script><script type="text/javascript">
    ;window.NREUM||(NREUM={});NREUM.init={privacy:{cookies_enabled:true}};
    window.NREUM||(NREUM={}),__nr_require=function(t,e,n){function r(n){if(!e[n]){var o=e[n]={exports:{}};t[n][0].call(o.exports,function(e){var o=t[n][1][e];return r(o||e)},o,o.exports)}return e[n].exports}if("function"==typeof __nr_require)return __nr_require;for(var o=0;o<n.length;o++)r(n[o]);return r}({1:[function(t,e,n){function r(t){try{c.console&&console.log(t)}catch(e){}}var o,i=t("ee"),a=t(26),c={};try{o=localStorage.getItem("__nr_flags").split(","),console&&"function"==typeof console.log&&(c.console=!0,o.indexOf("dev")!==-1&&(c.dev=!0),o.indexOf("nr_dev")!==-1&&(c.nrDev=!0))}catch(s){}c.nrDev&&i.on("internal-error",function(t){r(t.stack)}),c.dev&&i.on("fn-err",function(t,e,n){r(n.stack)}),c.dev&&(r("NR AGENT IN DEVELOPMENT MODE"),r("flags: "+a(c,function(t,e){return t}).join(", ")))},{}],2:[function(t,e,n){function r(t,e,n,r,c){try{l?l-=1:o(c||new UncaughtException(t,e,n),!0)}catch(f){try{i("ierr",[f,s.now(),!0])}catch(d){}}return"function"==typeof u&&u.apply(this,a(arguments))}function UncaughtException(t,e,n){this.message=t||"Uncaught error with no additional information",this.sourceURL=e,this.line=n}function o(t,e){var n=e?null:s.now();i("err",[t,n])}var i=t("handle"),a=t(27),c=t("ee"),s=t("loader"),f=t("gos"),u=window.onerror,d=!1,p="nr@seenError",l=0;s.features.err=!0,t(1),window.onerror=r;try{throw new Error}catch(h){"stack"in h&&(t(13),t(12),"addEventListener"in window&&t(6),s.xhrWrappable&&t(14),d=!0)}c.on("fn-start",function(t,e,n){d&&(l+=1)}),c.on("fn-err",function(t,e,n){d&&!n[p]&&(f(n,p,function(){return!0}),this.thrown=!0,o(n))}),c.on("fn-end",function(){d&&!this.thrown&&l>0&&(l-=1)}),c.on("internal-error",function(t){i("ierr",[t,s.now(),!0])})},{}],3:[function(t,e,n){t("loader").features.ins=!0},{}],4:[function(t,e,n){function r(){_++,T=g.hash,this[u]=y.now()}function o(){_--,g.hash!==T&&i(0,!0);var t=y.now();this[h]=~~this[h]+t-this[u],this[d]=t}function i(t,e){E.emit("newURL",[""+g,e])}function a(t,e){t.on(e,function(){this[e]=y.now()})}var c="-start",s="-end",f="-body",u="fn"+c,d="fn"+s,p="cb"+c,l="cb"+s,h="jsTime",m="fetch",v="addEventListener",w=window,g=w.location,y=t("loader");if(w[v]&&y.xhrWrappable){var x=t(10),b=t(11),E=t(8),R=t(6),O=t(13),N=t(7),M=t(14),P=t(9),S=t("ee"),C=S.get("tracer");t(16),y.features.spa=!0;var T,_=0;S.on(u,r),S.on(p,r),S.on(d,o),S.on(l,o),S.buffer([u,d,"xhr-done","xhr-resolved"]),R.buffer([u]),O.buffer(["setTimeout"+s,"clearTimeout"+c,u]),M.buffer([u,"new-xhr","send-xhr"+c]),N.buffer([m+c,m+"-done",m+f+c,m+f+s]),E.buffer(["newURL"]),x.buffer([u]),b.buffer(["propagate",p,l,"executor-err","resolve"+c]),C.buffer([u,"no-"+u]),P.buffer(["new-jsonp","cb-start","jsonp-error","jsonp-end"]),a(M,"send-xhr"+c),a(S,"xhr-resolved"),a(S,"xhr-done"),a(N,m+c),a(N,m+"-done"),a(P,"new-jsonp"),a(P,"jsonp-end"),a(P,"cb-start"),E.on("pushState-end",i),E.on("replaceState-end",i),w[v]("hashchange",i,!0),w[v]("load",i,!0),w[v]("popstate",function(){i(0,_>1)},!0)}},{}],5:[function(t,e,n){function r(t){}if(window.performance&&window.performance.timing&&window.performance.getEntriesByType){var o=t("ee"),i=t("handle"),a=t(13),c=t(12),s="learResourceTimings",f="addEventListener",u="resourcetimingbufferfull",d="bstResource",p="resource",l="-start",h="-end",m="fn"+l,v="fn"+h,w="bstTimer",g="pushState",y=t("loader");y.features.stn=!0,t(8),"addEventListener"in window&&t(6);var x=NREUM.o.EV;o.on(m,function(t,e){var n=t[0];n instanceof x&&(this.bstStart=y.now())}),o.on(v,function(t,e){var n=t[0];n instanceof x&&i("bst",[n,e,this.bstStart,y.now()])}),a.on(m,function(t,e,n){this.bstStart=y.now(),this.bstType=n}),a.on(v,function(t,e){i(w,[e,this.bstStart,y.now(),this.bstType])}),c.on(m,function(){this.bstStart=y.now()}),c.on(v,function(t,e){i(w,[e,this.bstStart,y.now(),"requestAnimationFrame"])}),o.on(g+l,function(t){this.time=y.now(),this.startPath=location.pathname+location.hash}),o.on(g+h,function(t){i("bstHist",[location.pathname+location.hash,this.startPath,this.time])}),f in window.performance&&(window.performance["c"+s]?window.performance[f](u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["c"+s]()},!1):window.performance[f]("webkit"+u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["webkitC"+s]()},!1)),document[f]("scroll",r,{passive:!0}),document[f]("keypress",r,!1),document[f]("click",r,!1)}},{}],6:[function(t,e,n){function r(t){for(var e=t;e&&!e.hasOwnProperty(u);)e=Object.getPrototypeOf(e);e&&o(e)}function o(t){c.inPlace(t,[u,d],"-",i)}function i(t,e){return t[1]}var a=t("ee").get("events"),c=t("wrap-function")(a,!0),s=t("gos"),f=XMLHttpRequest,u="addEventListener",d="removeEventListener";e.exports=a,"getPrototypeOf"in Object?(r(document),r(window),r(f.prototype)):f.prototype.hasOwnProperty(u)&&(o(window),o(f.prototype)),a.on(u+"-start",function(t,e){var n=t[1],r=s(n,"nr@wrapped",function(){function t(){if("function"==typeof n.handleEvent)return n.handleEvent.apply(n,arguments)}var e={object:t,"function":n}[typeof n];return e?c(e,"fn-",null,e.name||"anonymous"):n});this.wrapped=t[1]=r}),a.on(d+"-start",function(t){t[1]=this.wrapped||t[1]})},{}],7:[function(t,e,n){function r(t,e,n){var r=t[e];"function"==typeof r&&(t[e]=function(){var t=i(arguments),e={};o.emit(n+"before-start",[t],e);var a;e[m]&&e[m].dt&&(a=e[m].dt);var c=r.apply(this,t);return o.emit(n+"start",[t,a],c),c.then(function(t){return o.emit(n+"end",[null,t],c),t},function(t){throw o.emit(n+"end",[t],c),t})})}var o=t("ee").get("fetch"),i=t(27),a=t(26);e.exports=o;var c=window,s="fetch-",f=s+"body-",u=["arrayBuffer","blob","json","text","formData"],d=c.Request,p=c.Response,l=c.fetch,h="prototype",m="nr@context";d&&p&&l&&(a(u,function(t,e){r(d[h],e,f),r(p[h],e,f)}),r(c,"fetch",s),o.on(s+"end",function(t,e){var n=this;if(e){var r=e.headers.get("content-length");null!==r&&(n.rxSize=r),o.emit(s+"done",[null,e],n)}else o.emit(s+"done",[t],n)}))},{}],8:[function(t,e,n){var r=t("ee").get("history"),o=t("wrap-function")(r);e.exports=r;var i=window.history&&window.history.constructor&&window.history.constructor.prototype,a=window.history;i&&i.pushState&&i.replaceState&&(a=i),o.inPlace(a,["pushState","replaceState"],"-")},{}],9:[function(t,e,n){function r(t){function e(){s.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}function n(){s.emit("jsonp-error",[],p),s.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}var r=t&&"string"==typeof t.nodeName&&"script"===t.nodeName.toLowerCase();if(r){var o="function"==typeof t.addEventListener;if(o){var a=i(t.src);if(a){var u=c(a),d="function"==typeof u.parent[u.key];if(d){var p={};f.inPlace(u.parent,[u.key],"cb-",p),t.addEventListener("load",e,!1),t.addEventListener("error",n,!1),s.emit("new-jsonp",[t.src],p)}}}}}function o(){return"addEventListener"in window}function i(t){var e=t.match(u);return e?e[1]:null}function a(t,e){var n=t.match(p),r=n[1],o=n[3];return o?a(o,e[r]):e[r]}function c(t){var e=t.match(d);return e&&e.length>=3?{key:e[2],parent:a(e[1],window)}:{key:t,parent:window}}var s=t("ee").get("jsonp"),f=t("wrap-function")(s);if(e.exports=s,o()){var u=/[?&](?:callback|cb)=([^&#]+)/,d=/(.*)\.([^.]+)/,p=/^(\w+)(\.|$)(.*)$/,l=["appendChild","insertBefore","replaceChild"];Node&&Node.prototype&&Node.prototype.appendChild?f.inPlace(Node.prototype,l,"dom-"):(f.inPlace(HTMLElement.prototype,l,"dom-"),f.inPlace(HTMLHeadElement.prototype,l,"dom-"),f.inPlace(HTMLBodyElement.prototype,l,"dom-")),s.on("dom-start",function(t){r(t[0])})}},{}],10:[function(t,e,n){var r=t("ee").get("mutation"),o=t("wrap-function")(r),i=NREUM.o.MO;e.exports=r,i&&(window.MutationObserver=function(t){return this instanceof i?new i(o(t,"fn-")):i.apply(this,arguments)},MutationObserver.prototype=i.prototype)},{}],11:[function(t,e,n){function r(t){var e=a.context(),n=c(t,"executor-",e),r=new f(n);return a.context(r).getCtx=function(){return e},a.emit("new-promise",[r,e],e),r}function o(t,e){return e}var i=t("wrap-function"),a=t("ee").get("promise"),c=i(a),s=t(26),f=NREUM.o.PR;e.exports=a,f&&(window.Promise=r,["all","race"].forEach(function(t){var e=f[t];f[t]=function(n){function r(t){return function(){a.emit("propagate",[null,!o],i),o=o||!t}}var o=!1;s(n,function(e,n){Promise.resolve(n).then(r("all"===t),r(!1))});var i=e.apply(f,arguments),c=f.resolve(i);return c}}),["resolve","reject"].forEach(function(t){var e=f[t];f[t]=function(t){var n=e.apply(f,arguments);return t!==n&&a.emit("propagate",[t,!0],n),n}}),f.prototype["catch"]=function(t){return this.then(null,t)},f.prototype=Object.create(f.prototype,{constructor:{value:r}}),s(Object.getOwnPropertyNames(f),function(t,e){try{r[e]=f[e]}catch(n){}}),a.on("executor-start",function(t){t[0]=c(t[0],"resolve-",this),t[1]=c(t[1],"resolve-",this)}),a.on("executor-err",function(t,e,n){t[1](n)}),c.inPlace(f.prototype,["then"],"then-",o),a.on("then-start",function(t,e){this.promise=e,t[0]=c(t[0],"cb-",this),t[1]=c(t[1],"cb-",this)}),a.on("then-end",function(t,e,n){this.nextPromise=n;var r=this.promise;a.emit("propagate",[r,!0],n)}),a.on("cb-end",function(t,e,n){a.emit("propagate",[n,!0],this.nextPromise)}),a.on("propagate",function(t,e,n){this.getCtx&&!e||(this.getCtx=function(){if(t instanceof Promise)var e=a.context(t);return e&&e.getCtx?e.getCtx():this})}),r.toString=function(){return""+f})},{}],12:[function(t,e,n){var r=t("ee").get("raf"),o=t("wrap-function")(r),i="equestAnimationFrame";e.exports=r,o.inPlace(window,["r"+i,"mozR"+i,"webkitR"+i,"msR"+i],"raf-"),r.on("raf-start",function(t){t[0]=o(t[0],"fn-")})},{}],13:[function(t,e,n){function r(t,e,n){t[0]=a(t[0],"fn-",null,n)}function o(t,e,n){this.method=n,this.timerDuration=isNaN(t[1])?0:+t[1],t[0]=a(t[0],"fn-",this,n)}var i=t("ee").get("timer"),a=t("wrap-function")(i),c="setTimeout",s="setInterval",f="clearTimeout",u="-start",d="-";e.exports=i,a.inPlace(window,[c,"setImmediate"],c+d),a.inPlace(window,[s],s+d),a.inPlace(window,[f,"clearImmediate"],f+d),i.on(s+u,r),i.on(c+u,o)},{}],14:[function(t,e,n){function r(t,e){d.inPlace(e,["onreadystatechange"],"fn-",c)}function o(){var t=this,e=u.context(t);t.readyState>3&&!e.resolved&&(e.resolved=!0,u.emit("xhr-resolved",[],t)),d.inPlace(t,g,"fn-",c)}function i(t){y.push(t),h&&(b?b.then(a):v?v(a):(E=-E,R.data=E))}function a(){for(var t=0;t<y.length;t++)r([],y[t]);y.length&&(y=[])}function c(t,e){return e}function s(t,e){for(var n in t)e[n]=t[n];return e}t(6);var f=t("ee"),u=f.get("xhr"),d=t("wrap-function")(u),p=NREUM.o,l=p.XHR,h=p.MO,m=p.PR,v=p.SI,w="readystatechange",g=["onload","onerror","onabort","onloadstart","onloadend","onprogress","ontimeout"],y=[];e.exports=u;var x=window.XMLHttpRequest=function(t){var e=new l(t);try{u.emit("new-xhr",[e],e),e.addEventListener(w,o,!1)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}return e};if(s(l,x),x.prototype=l.prototype,d.inPlace(x.prototype,["open","send"],"-xhr-",c),u.on("send-xhr-start",function(t,e){r(t,e),i(e)}),u.on("open-xhr-start",r),h){var b=m&&m.resolve();if(!v&&!m){var E=1,R=document.createTextNode(E);new h(a).observe(R,{characterData:!0})}}else f.on("fn-end",function(t){t[0]&&t[0].type===w||a()})},{}],15:[function(t,e,n){function r(t){if(!c(t))return null;var e=window.NREUM;if(!e.loader_config)return null;var n=(e.loader_config.accountID||"").toString()||null,r=(e.loader_config.agentID||"").toString()||null,f=(e.loader_config.trustKey||"").toString()||null;if(!n||!r)return null;var h=l.generateSpanId(),m=l.generateTraceId(),v=Date.now(),w={spanId:h,traceId:m,timestamp:v};return(t.sameOrigin||s(t)&&p())&&(w.traceContextParentHeader=o(h,m),w.traceContextStateHeader=i(h,v,n,r,f)),(t.sameOrigin&&!u()||!t.sameOrigin&&s(t)&&d())&&(w.newrelicHeader=a(h,m,v,n,r,f)),w}function o(t,e){return"00-"+e+"-"+t+"-01"}function i(t,e,n,r,o){var i=0,a="",c=1,s="",f="";return o+"@nr="+i+"-"+c+"-"+n+"-"+r+"-"+t+"-"+a+"-"+s+"-"+f+"-"+e}function a(t,e,n,r,o,i){var a="btoa"in window&&"function"==typeof window.btoa;if(!a)return null;var c={v:[0,1],d:{ty:"Browser",ac:r,ap:o,id:t,tr:e,ti:n}};return i&&r!==i&&(c.d.tk=i),btoa(JSON.stringify(c))}function c(t){return f()&&s(t)}function s(t){var e=!1,n={};if("init"in NREUM&&"distributed_tracing"in NREUM.init&&(n=NREUM.init.distributed_tracing),t.sameOrigin)e=!0;else if(n.allowed_origins instanceof Array)for(var r=0;r<n.allowed_origins.length;r++){var o=h(n.allowed_origins[r]);if(t.hostname===o.hostname&&t.protocol===o.protocol&&t.port===o.port){e=!0;break}}return e}function f(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.enabled}function u(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.exclude_newrelic_header}function d(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&NREUM.init.distributed_tracing.cors_use_newrelic_header!==!1}function p(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.cors_use_tracecontext_headers}var l=t(23),h=t(17);e.exports={generateTracePayload:r,shouldGenerateTrace:c}},{}],16:[function(t,e,n){function r(t){var e=this.params,n=this.metrics;if(!this.ended){this.ended=!0;for(var r=0;r<p;r++)t.removeEventListener(d[r],this.listener,!1);e.aborted||(n.duration=a.now()-this.startTime,this.loadCaptureCalled||4!==t.readyState?null==e.status&&(e.status=0):i(this,t),n.cbTime=this.cbTime,u.emit("xhr-done",[t],t),c("xhr",[e,n,this.startTime]))}}function o(t,e){var n=s(e),r=t.params;r.host=n.hostname+":"+n.port,r.pathname=n.pathname,t.parsedOrigin=s(e),t.sameOrigin=t.parsedOrigin.sameOrigin}function i(t,e){t.params.status=e.status;var n=v(e,t.lastSize);if(n&&(t.metrics.rxSize=n),t.sameOrigin){var r=e.getResponseHeader("X-NewRelic-App-Data");r&&(t.params.cat=r.split(", ").pop())}t.loadCaptureCalled=!0}var a=t("loader");if(a.xhrWrappable){var c=t("handle"),s=t(17),f=t(15).generateTracePayload,u=t("ee"),d=["load","error","abort","timeout"],p=d.length,l=t("id"),h=t(21),m=t(20),v=t(18),w=window.XMLHttpRequest;a.features.xhr=!0,t(14),t(7),u.on("new-xhr",function(t){var e=this;e.totalCbs=0,e.called=0,e.cbTime=0,e.end=r,e.ended=!1,e.xhrGuids={},e.lastSize=null,e.loadCaptureCalled=!1,t.addEventListener("load",function(n){i(e,t)},!1),h&&(h>34||h<10)||window.opera||t.addEventListener("progress",function(t){e.lastSize=t.loaded},!1)}),u.on("open-xhr-start",function(t){this.params={method:t[0]},o(this,t[1]),this.metrics={}}),u.on("open-xhr-end",function(t,e){"loader_config"in NREUM&&"xpid"in NREUM.loader_config&&this.sameOrigin&&e.setRequestHeader("X-NewRelic-ID",NREUM.loader_config.xpid);var n=f(this.parsedOrigin);if(n){var r=!1;n.newrelicHeader&&(e.setRequestHeader("newrelic",n.newrelicHeader),r=!0),n.traceContextParentHeader&&(e.setRequestHeader("traceparent",n.traceContextParentHeader),n.traceContextStateHeader&&e.setRequestHeader("tracestate",n.traceContextStateHeader),r=!0),r&&(this.dt=n)}}),u.on("send-xhr-start",function(t,e){var n=this.metrics,r=t[0],o=this;if(n&&r){var i=m(r);i&&(n.txSize=i)}this.startTime=a.now(),this.listener=function(t){try{"abort"!==t.type||o.loadCaptureCalled||(o.params.aborted=!0),("load"!==t.type||o.called===o.totalCbs&&(o.onloadCalled||"function"!=typeof e.onload))&&o.end(e)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}};for(var c=0;c<p;c++)e.addEventListener(d[c],this.listener,!1)}),u.on("xhr-cb-time",function(t,e,n){this.cbTime+=t,e?this.onloadCalled=!0:this.called+=1,this.called!==this.totalCbs||!this.onloadCalled&&"function"==typeof n.onload||this.end(n)}),u.on("xhr-load-added",function(t,e){var n=""+l(t)+!!e;this.xhrGuids&&!this.xhrGuids[n]&&(this.xhrGuids[n]=!0,this.totalCbs+=1)}),u.on("xhr-load-removed",function(t,e){var n=""+l(t)+!!e;this.xhrGuids&&this.xhrGuids[n]&&(delete this.xhrGuids[n],this.totalCbs-=1)}),u.on("addEventListener-end",function(t,e){e instanceof w&&"load"===t[0]&&u.emit("xhr-load-added",[t[1],t[2]],e)}),u.on("removeEventListener-end",function(t,e){e instanceof w&&"load"===t[0]&&u.emit("xhr-load-removed",[t[1],t[2]],e)}),u.on("fn-start",function(t,e,n){e instanceof w&&("onload"===n&&(this.onload=!0),("load"===(t[0]&&t[0].type)||this.onload)&&(this.xhrCbStart=a.now()))}),u.on("fn-end",function(t,e){this.xhrCbStart&&u.emit("xhr-cb-time",[a.now()-this.xhrCbStart,this.onload,e],e)}),u.on("fetch-before-start",function(t){function e(t,e){var n=!1;return e.newrelicHeader&&(t.set("newrelic",e.newrelicHeader),n=!0),e.traceContextParentHeader&&(t.set("traceparent",e.traceContextParentHeader),e.traceContextStateHeader&&t.set("tracestate",e.traceContextStateHeader),n=!0),n}var n,r=t[1]||{};"string"==typeof t[0]?n=t[0]:t[0]&&t[0].url&&(n=t[0].url),n&&(this.parsedOrigin=s(n),this.sameOrigin=this.parsedOrigin.sameOrigin);var o=f(this.parsedOrigin);if(o&&(o.newrelicHeader||o.traceContextParentHeader))if("string"==typeof t[0]){var i={};for(var a in r)i[a]=r[a];i.headers=new Headers(r.headers||{}),e(i.headers,o)&&(this.dt=o),t.length>1?t[1]=i:t.push(i)}else t[0]&&t[0].headers&&e(t[0].headers,o)&&(this.dt=o)})}},{}],17:[function(t,e,n){var r={};e.exports=function(t){if(t in r)return r[t];var e=document.createElement("a"),n=window.location,o={};e.href=t,o.port=e.port;var i=e.href.split("://");!o.port&&i[1]&&(o.port=i[1].split("/")[0].split("@").pop().split(":")[1]),o.port&&"0"!==o.port||(o.port="https"===i[0]?"443":"80"),o.hostname=e.hostname||n.hostname,o.pathname=e.pathname,o.protocol=i[0],"/"!==o.pathname.charAt(0)&&(o.pathname="/"+o.pathname);var a=!e.protocol||":"===e.protocol||e.protocol===n.protocol,c=e.hostname===document.domain&&e.port===n.port;return o.sameOrigin=a&&(!e.hostname||c),"/"===o.pathname&&(r[t]=o),o}},{}],18:[function(t,e,n){function r(t,e){var n=t.responseType;return"json"===n&&null!==e?e:"arraybuffer"===n||"blob"===n||"json"===n?o(t.response):"text"===n||""===n||void 0===n?o(t.responseText):void 0}var o=t(20);e.exports=r},{}],19:[function(t,e,n){function r(){}function o(t,e,n){return function(){return i(t,[f.now()].concat(c(arguments)),e?null:this,n),e?void 0:this}}var i=t("handle"),a=t(26),c=t(27),s=t("ee").get("tracer"),f=t("loader"),u=NREUM;"undefined"==typeof window.newrelic&&(newrelic=u);var d=["setPageViewName","setCustomAttribute","setErrorHandler","finished","addToTrace","inlineHit","addRelease"],p="api-",l=p+"ixn-";a(d,function(t,e){u[e]=o(p+e,!0,"api")}),u.addPageAction=o(p+"addPageAction",!0),u.setCurrentRouteName=o(p+"routeName",!0),e.exports=newrelic,u.interaction=function(){return(new r).get()};var h=r.prototype={createTracer:function(t,e){var n={},r=this,o="function"==typeof e;return i(l+"tracer",[f.now(),t,n],r),function(){if(s.emit((o?"":"no-")+"fn-start",[f.now(),r,o],n),o)try{return e.apply(this,arguments)}catch(t){throw s.emit("fn-err",[arguments,this,t],n),t}finally{s.emit("fn-end",[f.now()],n)}}}};a("actionText,setName,setAttribute,save,ignore,onEnd,getContext,end,get".split(","),function(t,e){h[e]=o(l+e)}),newrelic.noticeError=function(t,e){"string"==typeof t&&(t=new Error(t)),i("err",[t,f.now(),!1,e])}},{}],20:[function(t,e,n){e.exports=function(t){if("string"==typeof t&&t.length)return t.length;if("object"==typeof t){if("undefined"!=typeof ArrayBuffer&&t instanceof ArrayBuffer&&t.byteLength)return t.byteLength;if("undefined"!=typeof Blob&&t instanceof Blob&&t.size)return t.size;if(!("undefined"!=typeof FormData&&t instanceof FormData))try{return JSON.stringify(t).length}catch(e){return}}}},{}],21:[function(t,e,n){var r=0,o=navigator.userAgent.match(/Firefox[\/\s](\d+\.\d+)/);o&&(r=+o[1]),e.exports=r},{}],22:[function(t,e,n){function r(t,e){var n=t.getEntries();n.forEach(function(t){"first-paint"===t.name?d("timing",["fp",Math.floor(t.startTime)]):"first-contentful-paint"===t.name&&d("timing",["fcp",Math.floor(t.startTime)])})}function o(t,e){var n=t.getEntries();n.length>0&&d("lcp",[n[n.length-1]])}function i(t){t.getEntries().forEach(function(t){t.hadRecentInput||d("cls",[t])})}function a(t){if(t instanceof h&&!v){var e=Math.round(t.timeStamp),n={type:t.type};e<=p.now()?n.fid=p.now()-e:e>p.offset&&e<=Date.now()?(e-=p.offset,n.fid=p.now()-e):e=p.now(),v=!0,d("timing",["fi",e,n])}}function c(t){d("pageHide",[p.now(),t])}if(!("init"in NREUM&&"page_view_timing"in NREUM.init&&"enabled"in NREUM.init.page_view_timing&&NREUM.init.page_view_timing.enabled===!1)){var s,f,u,d=t("handle"),p=t("loader"),l=t(25),h=NREUM.o.EV;if("PerformanceObserver"in window&&"function"==typeof window.PerformanceObserver){s=new PerformanceObserver(r);try{s.observe({entryTypes:["paint"]})}catch(m){}f=new PerformanceObserver(o);try{f.observe({entryTypes:["largest-contentful-paint"]})}catch(m){}u=new PerformanceObserver(i);try{u.observe({type:"layout-shift",buffered:!0})}catch(m){}}if("addEventListener"in document){var v=!1,w=["click","keydown","mousedown","pointerdown","touchstart"];w.forEach(function(t){document.addEventListener(t,a,!1)})}l(c)}},{}],23:[function(t,e,n){function r(){function t(){return e?15&e[n++]:16*Math.random()|0}var e=null,n=0,r=window.crypto||window.msCrypto;r&&r.getRandomValues&&(e=r.getRandomValues(new Uint8Array(31)));for(var o,i="xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx",a="",c=0;c<i.length;c++)o=i[c],"x"===o?a+=t().toString(16):"y"===o?(o=3&t()|8,a+=o.toString(16)):a+=o;return a}function o(){return a(16)}function i(){return a(32)}function a(t){function e(){return n?15&n[r++]:16*Math.random()|0}var n=null,r=0,o=window.crypto||window.msCrypto;o&&o.getRandomValues&&Uint8Array&&(n=o.getRandomValues(new Uint8Array(31)));for(var i=[],a=0;a<t;a++)i.push(e().toString(16));return i.join("")}e.exports={generateUuid:r,generateSpanId:o,generateTraceId:i}},{}],24:[function(t,e,n){function r(t,e){if(!o)return!1;if(t!==o)return!1;if(!e)return!0;if(!i)return!1;for(var n=i.split("."),r=e.split("."),a=0;a<r.length;a++)if(r[a]!==n[a])return!1;return!0}var o=null,i=null,a=/Version\/(\S+)\s+Safari/;if(navigator.userAgent){var c=navigator.userAgent,s=c.match(a);s&&c.indexOf("Chrome")===-1&&c.indexOf("Chromium")===-1&&(o="Safari",i=s[1])}e.exports={agent:o,version:i,match:r}},{}],25:[function(t,e,n){function r(t){function e(){t(a&&document[a]?document[a]:document[o]?"hidden":"visible")}"addEventListener"in document&&i&&document.addEventListener(i,e,!1)}e.exports=r;var o,i,a;"undefined"!=typeof document.hidden?(o="hidden",i="visibilitychange",a="visibilityState"):"undefined"!=typeof document.msHidden?(o="msHidden",i="msvisibilitychange"):"undefined"!=typeof document.webkitHidden&&(o="webkitHidden",i="webkitvisibilitychange",a="webkitVisibilityState")},{}],26:[function(t,e,n){function r(t,e){var n=[],r="",i=0;for(r in t)o.call(t,r)&&(n[i]=e(r,t[r]),i+=1);return n}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],27:[function(t,e,n){function r(t,e,n){e||(e=0),"undefined"==typeof n&&(n=t?t.length:0);for(var r=-1,o=n-e||0,i=Array(o<0?0:o);++r<o;)i[r]=t[e+r];return i}e.exports=r},{}],28:[function(t,e,n){e.exports={exists:"undefined"!=typeof window.performance&&window.performance.timing&&"undefined"!=typeof window.performance.timing.navigationStart}},{}],ee:[function(t,e,n){function r(){}function o(t){function e(t){return t&&t instanceof r?t:t?s(t,c,i):i()}function n(n,r,o,i){if(!p.aborted||i){t&&t(n,r,o);for(var a=e(o),c=m(n),s=c.length,f=0;f<s;f++)c[f].apply(a,r);var d=u[y[n]];return d&&d.push([x,n,r,a]),a}}function l(t,e){g[t]=m(t).concat(e)}function h(t,e){var n=g[t];if(n)for(var r=0;r<n.length;r++)n[r]===e&&n.splice(r,1)}function m(t){return g[t]||[]}function v(t){return d[t]=d[t]||o(n)}function w(t,e){f(t,function(t,n){e=e||"feature",y[n]=e,e in u||(u[e]=[])})}var g={},y={},x={on:l,addEventListener:l,removeEventListener:h,emit:n,get:v,listeners:m,context:e,buffer:w,abort:a,aborted:!1};return x}function i(){return new r}function a(){(u.api||u.feature)&&(p.aborted=!0,u=p.backlog={})}var c="nr@context",s=t("gos"),f=t(26),u={},d={},p=e.exports=o();p.backlog=u},{}],gos:[function(t,e,n){function r(t,e,n){if(o.call(t,e))return t[e];var r=n();if(Object.defineProperty&&Object.keys)try{return Object.defineProperty(t,e,{value:r,writable:!0,enumerable:!1}),r}catch(i){}return t[e]=r,r}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],handle:[function(t,e,n){function r(t,e,n,r){o.buffer([t],r),o.emit(t,e,n)}var o=t("ee").get("handle");e.exports=r,r.ee=o},{}],id:[function(t,e,n){function r(t){var e=typeof t;return!t||"object"!==e&&"function"!==e?-1:t===window?0:a(t,i,function(){return o++})}var o=1,i="nr@id",a=t("gos");e.exports=r},{}],loader:[function(t,e,n){function r(){if(!E++){var t=b.info=NREUM.info,e=l.getElementsByTagName("script")[0];if(setTimeout(u.abort,3e4),!(t&&t.licenseKey&&t.applicationID&&e))return u.abort();f(y,function(e,n){t[e]||(t[e]=n)});var n=a();s("mark",["onload",n+b.offset],null,"api"),s("timing",["load",n]);var r=l.createElement("script");r.src="https://"+t.agent,e.parentNode.insertBefore(r,e)}}function o(){"complete"===l.readyState&&i()}function i(){s("mark",["domContent",a()+b.offset],null,"api")}function a(){return R.exists&&performance.now?Math.round(performance.now()):(c=Math.max((new Date).getTime(),c))-b.offset}var c=(new Date).getTime(),s=t("handle"),f=t(26),u=t("ee"),d=t(24),p=window,l=p.document,h="addEventListener",m="attachEvent",v=p.XMLHttpRequest,w=v&&v.prototype;NREUM.o={ST:setTimeout,SI:p.setImmediate,CT:clearTimeout,XHR:v,REQ:p.Request,EV:p.Event,PR:p.Promise,MO:p.MutationObserver};var g=""+location,y={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",agent:"js-agent.newrelic.com/nr-spa-1177.min.js"},x=v&&w&&w[h]&&!/CriOS/.test(navigator.userAgent),b=e.exports={offset:c,now:a,origin:g,features:{},xhrWrappable:x,userAgent:d};t(19),t(22),l[h]?(l[h]("DOMContentLoaded",i,!1),p[h]("load",r,!1)):(l[m]("onreadystatechange",o),p[m]("onload",r)),s("mark",["firstbyte",c],null,"api");var E=0,R=t(28)},{}],"wrap-function":[function(t,e,n){function r(t){return!(t&&t instanceof Function&&t.apply&&!t[a])}var o=t("ee"),i=t(27),a="nr@original",c=Object.prototype.hasOwnProperty,s=!1;e.exports=function(t,e){function n(t,e,n,o){function nrWrapper(){var r,a,c,s;try{a=this,r=i(arguments),c="function"==typeof n?n(r,a):n||{}}catch(f){p([f,"",[r,a,o],c])}u(e+"start",[r,a,o],c);try{return s=t.apply(a,r)}catch(d){throw u(e+"err",[r,a,d],c),d}finally{u(e+"end",[r,a,s],c)}}return r(t)?t:(e||(e=""),nrWrapper[a]=t,d(t,nrWrapper),nrWrapper)}function f(t,e,o,i){o||(o="");var a,c,s,f="-"===o.charAt(0);for(s=0;s<e.length;s++)c=e[s],a=t[c],r(a)||(t[c]=n(a,f?c+o:o,i,c))}function u(n,r,o){if(!s||e){var i=s;s=!0;try{t.emit(n,r,o,e)}catch(a){p([a,n,r,o])}s=i}}function d(t,e){if(Object.defineProperty&&Object.keys)try{var n=Object.keys(t);return n.forEach(function(n){Object.defineProperty(e,n,{get:function(){return t[n]},set:function(e){return t[n]=e,e}})}),e}catch(r){p([r])}for(var o in t)c.call(t,o)&&(e[o]=t[o]);return e}function p(e){try{t.emit("internal-error",e)}catch(n){}}return t||(t=o),n.inPlace=f,n.flag=a,n}},{}]},{},["loader",2,16,5,3,4]);
    ;NREUM.loader_config={accountID:"2095297",trustKey:"2038175",agentID:"916957862",licenseKey:"16e2192192",applicationID:"916957862"}
    ;NREUM.info={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",licenseKey:"16e2192192",applicationID:"916957862",sa:1}
</script>
    <script src="https://cdn.cookielaw.org/scripttemplates/otSDKStub.js" type="text/javascript" charset="UTF-8" data-domain-script="dada9834-04e2-4bcd-b5ba-cae44d498dfa"></script>
    <script type="text/javascript">
        function getCookie(name) {
            var value = "; " + document.cookie;
            var parts = value.split("; " + name + "=");
            if (parts.length == 2) {
                return true;
            }
        }
            function OptanonWrapper() {
                var optCookie = getCookie("OptanonAlertBoxClosed");
                var isActiveCategory = "false";
                var isFoundCookieBox = "false";
                if(isActiveCategory == 'false' && isFoundCookieBox == 'false' && optCookie) {
                    OneTrust.InsertScript('/pb-assets/adobe/prod/adobedtm-1616606406233.js', 'head', null, null, '2');
                }
            }
    </script>


<noscript>
    <style>.requiresJS {
        display: none
    }

    .dartAd {
        display: block !important
    }</style>
</noscript>
        <script type="text/javascript" src="/wro/o4ny~selected-legacy.js"></script>
        <link rel="stylesheet" type="text/css" href="/wro/o4ny~selected-legacy.css">
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Open+Sans">
        <script>window.MSInputMethodContext && document.documentMode && document.write('<script src="https://cdn.jsdelivr.net/gh/nuxodin/ie11CustomProperties@4.1.0/ie11CustomProperties.min.js"><\x2fscript>');</script><link rel="stylesheet" href="/products/marlin/cell/releasedAssets/css/build-b26f3342b722959c7931.css"><link rel="stylesheet" href="/products/marlin/cell/releasedAssets/css/print-b26f3342b722959c7931.css" media="print"><link rel="stylesheet" href="/products/marlin/cell/releasedAssets/css/themes/theme-cell.css">







        
        
        <script class="optanon-category-4" type="text/plain" async="" src="https://www.youtube.com/iframe_api"></script>
        
    





<!---->
<meta http-equiv="X-UA-Compatible" content="IE=edge">












<script src="https://cdn.cookielaw.org/scripttemplates/202411.2.0/otBannerSdk.js" async="" type="text/javascript"></script><style id="onetrust-style">#onetrust-banner-sdk .onetrust-vendors-list-handler{cursor:pointer;color:#1f96db;font-size:inherit;font-weight:bold;text-decoration:none;margin-left:5px}#onetrust-banner-sdk .onetrust-vendors-list-handler:hover{color:#1f96db}#onetrust-banner-sdk:focus{outline:2px solid #000;outline-offset:-2px}#onetrust-banner-sdk a:focus{outline:2px solid #000}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{outline-offset:1px}#onetrust-banner-sdk.ot-bnr-w-logo .ot-bnr-logo{height:64px;width:64px}#onetrust-banner-sdk .ot-tcf2-vendor-count.ot-text-bold{font-weight:bold}#onetrust-banner-sdk .ot-close-icon,#onetrust-pc-sdk .ot-close-icon,#ot-sync-ntfy .ot-close-icon{background-size:contain;background-repeat:no-repeat;background-position:center;height:12px;width:12px}#onetrust-banner-sdk .powered-by-logo,#onetrust-banner-sdk .ot-pc-footer-logo a,#onetrust-pc-sdk .powered-by-logo,#onetrust-pc-sdk .ot-pc-footer-logo a,#ot-sync-ntfy .powered-by-logo,#ot-sync-ntfy .ot-pc-footer-logo a{background-size:contain;background-repeat:no-repeat;background-position:center;height:25px;width:152px;display:block;text-decoration:none;font-size:.75em}#onetrust-banner-sdk .powered-by-logo:hover,#onetrust-banner-sdk .ot-pc-footer-logo a:hover,#onetrust-pc-sdk .powered-by-logo:hover,#onetrust-pc-sdk .ot-pc-footer-logo a:hover,#ot-sync-ntfy .powered-by-logo:hover,#ot-sync-ntfy .ot-pc-footer-logo a:hover{color:#565656}#onetrust-banner-sdk h3 *,#onetrust-banner-sdk h4 *,#onetrust-banner-sdk h6 *,#onetrust-banner-sdk button *,#onetrust-banner-sdk a[data-parent-id] *,#onetrust-pc-sdk h3 *,#onetrust-pc-sdk h4 *,#onetrust-pc-sdk h6 *,#onetrust-pc-sdk button *,#onetrust-pc-sdk a[data-parent-id] *,#ot-sync-ntfy h3 *,#ot-sync-ntfy h4 *,#ot-sync-ntfy h6 *,#ot-sync-ntfy button *,#ot-sync-ntfy a[data-parent-id] *{font-size:inherit;font-weight:inherit;color:inherit}#onetrust-banner-sdk .ot-hide,#onetrust-pc-sdk .ot-hide,#ot-sync-ntfy .ot-hide{display:none !important}#onetrust-banner-sdk button.ot-link-btn:hover,#onetrust-pc-sdk button.ot-link-btn:hover,#ot-sync-ntfy button.ot-link-btn:hover{text-decoration:underline;opacity:1}#onetrust-pc-sdk .ot-sdk-row .ot-sdk-column{padding:0}#onetrust-pc-sdk .ot-sdk-container{padding-right:0}#onetrust-pc-sdk .ot-sdk-row{flex-direction:initial;width:100%}#onetrust-pc-sdk [type=checkbox]:checked,#onetrust-pc-sdk [type=checkbox]:not(:checked){pointer-events:initial}#onetrust-pc-sdk [type=checkbox]:disabled+label::before,#onetrust-pc-sdk [type=checkbox]:disabled+label:after,#onetrust-pc-sdk [type=checkbox]:disabled+label{pointer-events:none;opacity:.8}#onetrust-pc-sdk #vendor-list-content{transform:translate3d(0, 0, 0)}#onetrust-pc-sdk li input[type=checkbox]{z-index:1}#onetrust-pc-sdk li .ot-checkbox label{z-index:2}#onetrust-pc-sdk li .ot-checkbox input[type=checkbox]{height:auto;width:auto}#onetrust-pc-sdk li .host-title a,#onetrust-pc-sdk li .ot-host-name a,#onetrust-pc-sdk li .accordion-text,#onetrust-pc-sdk li .ot-acc-txt{z-index:2;position:relative}#onetrust-pc-sdk input{margin:3px .1ex}#onetrust-pc-sdk .pc-logo,#onetrust-pc-sdk .ot-pc-logo{height:60px;width:180px;background-position:center;background-size:contain;background-repeat:no-repeat;display:inline-flex;justify-content:center;align-items:center}#onetrust-pc-sdk .pc-logo img,#onetrust-pc-sdk .ot-pc-logo img{max-height:100%;max-width:100%}#onetrust-pc-sdk .screen-reader-only,#onetrust-pc-sdk .ot-scrn-rdr,.ot-sdk-cookie-policy .screen-reader-only,.ot-sdk-cookie-policy .ot-scrn-rdr{border:0;clip:rect(0 0 0 0);height:1px;margin:-1px;overflow:hidden;padding:0;position:absolute;width:1px}#onetrust-pc-sdk.ot-fade-in,.onetrust-pc-dark-filter.ot-fade-in,#onetrust-banner-sdk.ot-fade-in{animation-name:onetrust-fade-in;animation-duration:400ms;animation-timing-function:ease-in-out}#onetrust-pc-sdk.ot-hide{display:none !important}.onetrust-pc-dark-filter.ot-hide{display:none !important}#ot-sdk-btn.ot-sdk-show-settings,#ot-sdk-btn.optanon-show-settings{color:#fff;background-color:#468254;height:auto;white-space:normal;word-wrap:break-word;padding:.8em 2em;font-size:.8em;line-height:1.2;cursor:pointer;-moz-transition:.1s ease;-o-transition:.1s ease;-webkit-transition:1s ease;transition:.1s ease}#ot-sdk-btn.ot-sdk-show-settings:hover,#ot-sdk-btn.optanon-show-settings:hover{color:#fff;background-color:#2c6415}#ot-sdk-btn.ot-sdk-show-settings:active,#ot-sdk-btn.optanon-show-settings:active{color:#fff;background-color:#2c6415;border:1px solid rgba(162,192,169,.5)}.onetrust-pc-dark-filter{background:rgba(0,0,0,.5);z-index:2147483646;width:100%;height:100%;overflow:hidden;position:fixed;top:0;bottom:0;left:0}@keyframes onetrust-fade-in{0%{opacity:0}100%{opacity:1}}.ot-cookie-label{text-decoration:underline}@media only screen and (min-width: 426px)and (max-width: 896px)and (orientation: landscape){#onetrust-pc-sdk p{font-size:.75em}}#onetrust-banner-sdk .banner-option-input:focus+label{outline:1px solid #000;outline-style:auto}.category-vendors-list-handler+a:focus,.category-vendors-list-handler+a:focus-visible{outline:2px solid #000}#onetrust-pc-sdk .ot-userid-title{margin-top:10px}#onetrust-pc-sdk .ot-userid-title>span,#onetrust-pc-sdk .ot-userid-timestamp>span{font-weight:700}#onetrust-pc-sdk .ot-userid-desc{font-style:italic}#onetrust-pc-sdk .ot-host-desc a{pointer-events:initial}#onetrust-pc-sdk .ot-ven-hdr>p a{position:relative;z-index:2;pointer-events:initial}#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info a,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info a{margin-right:auto}#onetrust-pc-sdk .ot-pc-footer-logo img{width:136px;height:16px}#onetrust-pc-sdk .ot-pur-vdr-count{font-weight:400;font-size:.7rem;padding-top:3px;display:block}#onetrust-banner-sdk .ot-optout-signal,#onetrust-pc-sdk .ot-optout-signal{border:1px solid #32ae88;border-radius:3px;padding:5px;margin-bottom:10px;background-color:#f9fffa;font-size:.85rem;line-height:2}#onetrust-banner-sdk .ot-optout-signal .ot-optout-icon,#onetrust-pc-sdk .ot-optout-signal .ot-optout-icon{display:inline;margin-right:5px}#onetrust-banner-sdk .ot-optout-signal svg,#onetrust-pc-sdk .ot-optout-signal svg{height:20px;width:30px;transform:scale(0.5)}#onetrust-banner-sdk .ot-optout-signal svg path,#onetrust-pc-sdk .ot-optout-signal svg path{fill:#32ae88}#onetrust-consent-sdk .ot-general-modal{overflow:hidden;position:fixed;margin:0 auto;top:50%;left:50%;width:40%;padding:1.5rem;max-width:575px;min-width:575px;z-index:2147483647;border-radius:2.5px;transform:translate(-50%, -50%)}#onetrust-consent-sdk .ot-signature-health-group{margin-top:1rem;padding-left:1.25rem;padding-right:1.25rem;margin-bottom:.625rem;width:calc(100% - 2.5rem)}#onetrust-consent-sdk .ot-signature-health-group .ot-signature-health-form{gap:.5rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-health-form{width:70%;gap:.35rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-input{height:38px;padding:6px 10px;background-color:#fff;border:1px solid #d1d1d1;border-radius:4px;box-shadow:none;box-sizing:border-box}#onetrust-consent-sdk .ot-signature-health .ot-signature-subtitle{font-size:1.125rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-group-title{font-size:1.25rem;font-weight:bold}#onetrust-consent-sdk .ot-signature-health,#onetrust-consent-sdk .ot-signature-health-group{display:flex;flex-direction:column;gap:1rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-cont,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-cont{display:flex;flex-direction:column;gap:.25rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-paragraph,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-paragraph{margin:0;line-height:20px;font-size:max(14px,.875rem)}#onetrust-consent-sdk .ot-signature-health .ot-health-signature-error,#onetrust-consent-sdk .ot-signature-health-group .ot-health-signature-error{color:#4d4d4d;font-size:min(12px,.75rem)}#onetrust-consent-sdk .ot-signature-health .ot-signature-buttons-cont,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-buttons-cont{margin-top:max(.75rem,2%);gap:1rem;display:flex;justify-content:flex-end}#onetrust-consent-sdk .ot-signature-health .ot-signature-button,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-button{flex:1;height:auto;color:#fff;cursor:pointer;line-height:1.2;min-width:125px;font-weight:600;font-size:.813em;border-radius:2px;padding:12px 10px;white-space:normal;word-wrap:break-word;word-break:break-word;background-color:#68b631;border:2px solid #68b631}#onetrust-consent-sdk .ot-signature-health .ot-signature-button.reject,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-button.reject{background-color:#fff}#onetrust-consent-sdk .ot-input-field-cont{display:flex;flex-direction:column;gap:.5rem}#onetrust-consent-sdk .ot-input-field-cont .ot-signature-input{width:65%}#onetrust-consent-sdk .ot-signature-health-form{display:flex;flex-direction:column}#onetrust-consent-sdk .ot-signature-health-form .ot-signature-label{margin-bottom:0;line-height:20px;font-size:max(14px,.875rem)}@media only screen and (max-width: 600px){#onetrust-consent-sdk .ot-general-modal{min-width:100%}#onetrust-consent-sdk .ot-signature-health .ot-signature-health-form{width:100%}#onetrust-consent-sdk .ot-input-field-cont .ot-signature-input{width:100%}}#onetrust-banner-sdk,#onetrust-pc-sdk,#ot-sdk-cookie-policy,#ot-sync-ntfy{font-size:16px}#onetrust-banner-sdk *,#onetrust-banner-sdk ::after,#onetrust-banner-sdk ::before,#onetrust-pc-sdk *,#onetrust-pc-sdk ::after,#onetrust-pc-sdk ::before,#ot-sdk-cookie-policy *,#ot-sdk-cookie-policy ::after,#ot-sdk-cookie-policy ::before,#ot-sync-ntfy *,#ot-sync-ntfy ::after,#ot-sync-ntfy ::before{-webkit-box-sizing:content-box;-moz-box-sizing:content-box;box-sizing:content-box}#onetrust-banner-sdk div,#onetrust-banner-sdk span,#onetrust-banner-sdk h1,#onetrust-banner-sdk h2,#onetrust-banner-sdk h3,#onetrust-banner-sdk h4,#onetrust-banner-sdk h5,#onetrust-banner-sdk h6,#onetrust-banner-sdk p,#onetrust-banner-sdk img,#onetrust-banner-sdk svg,#onetrust-banner-sdk button,#onetrust-banner-sdk section,#onetrust-banner-sdk a,#onetrust-banner-sdk label,#onetrust-banner-sdk input,#onetrust-banner-sdk ul,#onetrust-banner-sdk li,#onetrust-banner-sdk nav,#onetrust-banner-sdk table,#onetrust-banner-sdk thead,#onetrust-banner-sdk tr,#onetrust-banner-sdk td,#onetrust-banner-sdk tbody,#onetrust-banner-sdk .ot-main-content,#onetrust-banner-sdk .ot-toggle,#onetrust-banner-sdk #ot-content,#onetrust-banner-sdk #ot-pc-content,#onetrust-banner-sdk .checkbox,#onetrust-pc-sdk div,#onetrust-pc-sdk span,#onetrust-pc-sdk h1,#onetrust-pc-sdk h2,#onetrust-pc-sdk h3,#onetrust-pc-sdk h4,#onetrust-pc-sdk h5,#onetrust-pc-sdk h6,#onetrust-pc-sdk p,#onetrust-pc-sdk img,#onetrust-pc-sdk svg,#onetrust-pc-sdk button,#onetrust-pc-sdk section,#onetrust-pc-sdk a,#onetrust-pc-sdk label,#onetrust-pc-sdk input,#onetrust-pc-sdk ul,#onetrust-pc-sdk li,#onetrust-pc-sdk nav,#onetrust-pc-sdk table,#onetrust-pc-sdk thead,#onetrust-pc-sdk tr,#onetrust-pc-sdk td,#onetrust-pc-sdk tbody,#onetrust-pc-sdk .ot-main-content,#onetrust-pc-sdk .ot-toggle,#onetrust-pc-sdk #ot-content,#onetrust-pc-sdk #ot-pc-content,#onetrust-pc-sdk .checkbox,#ot-sdk-cookie-policy div,#ot-sdk-cookie-policy span,#ot-sdk-cookie-policy h1,#ot-sdk-cookie-policy h2,#ot-sdk-cookie-policy h3,#ot-sdk-cookie-policy h4,#ot-sdk-cookie-policy h5,#ot-sdk-cookie-policy h6,#ot-sdk-cookie-policy p,#ot-sdk-cookie-policy img,#ot-sdk-cookie-policy svg,#ot-sdk-cookie-policy button,#ot-sdk-cookie-policy section,#ot-sdk-cookie-policy a,#ot-sdk-cookie-policy label,#ot-sdk-cookie-policy input,#ot-sdk-cookie-policy ul,#ot-sdk-cookie-policy li,#ot-sdk-cookie-policy nav,#ot-sdk-cookie-policy table,#ot-sdk-cookie-policy thead,#ot-sdk-cookie-policy tr,#ot-sdk-cookie-policy td,#ot-sdk-cookie-policy tbody,#ot-sdk-cookie-policy .ot-main-content,#ot-sdk-cookie-policy .ot-toggle,#ot-sdk-cookie-policy #ot-content,#ot-sdk-cookie-policy #ot-pc-content,#ot-sdk-cookie-policy .checkbox,#ot-sync-ntfy div,#ot-sync-ntfy span,#ot-sync-ntfy h1,#ot-sync-ntfy h2,#ot-sync-ntfy h3,#ot-sync-ntfy h4,#ot-sync-ntfy h5,#ot-sync-ntfy h6,#ot-sync-ntfy p,#ot-sync-ntfy img,#ot-sync-ntfy svg,#ot-sync-ntfy button,#ot-sync-ntfy section,#ot-sync-ntfy a,#ot-sync-ntfy label,#ot-sync-ntfy input,#ot-sync-ntfy ul,#ot-sync-ntfy li,#ot-sync-ntfy nav,#ot-sync-ntfy table,#ot-sync-ntfy thead,#ot-sync-ntfy tr,#ot-sync-ntfy td,#ot-sync-ntfy tbody,#ot-sync-ntfy .ot-main-content,#ot-sync-ntfy .ot-toggle,#ot-sync-ntfy #ot-content,#ot-sync-ntfy #ot-pc-content,#ot-sync-ntfy .checkbox{font-family:inherit;font-weight:normal;-webkit-font-smoothing:auto;letter-spacing:normal;line-height:normal;padding:0;margin:0;height:auto;min-height:0;max-height:none;width:auto;min-width:0;max-width:none;border-radius:0;border:none;clear:none;float:none;position:static;bottom:auto;left:auto;right:auto;top:auto;text-align:left;text-decoration:none;text-indent:0;text-shadow:none;text-transform:none;white-space:normal;background:none;overflow:visible;vertical-align:baseline;visibility:visible;z-index:auto;box-shadow:none}#onetrust-banner-sdk label:before,#onetrust-banner-sdk label:after,#onetrust-banner-sdk .checkbox:after,#onetrust-banner-sdk .checkbox:before,#onetrust-pc-sdk label:before,#onetrust-pc-sdk label:after,#onetrust-pc-sdk .checkbox:after,#onetrust-pc-sdk .checkbox:before,#ot-sdk-cookie-policy label:before,#ot-sdk-cookie-policy label:after,#ot-sdk-cookie-policy .checkbox:after,#ot-sdk-cookie-policy .checkbox:before,#ot-sync-ntfy label:before,#ot-sync-ntfy label:after,#ot-sync-ntfy .checkbox:after,#ot-sync-ntfy .checkbox:before{content:"";content:none}#onetrust-banner-sdk .ot-sdk-container,#onetrust-pc-sdk .ot-sdk-container,#ot-sdk-cookie-policy .ot-sdk-container{position:relative;width:100%;max-width:100%;margin:0 auto;padding:0 20px;box-sizing:border-box}#onetrust-banner-sdk .ot-sdk-column,#onetrust-banner-sdk .ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-column,#onetrust-pc-sdk .ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-column,#ot-sdk-cookie-policy .ot-sdk-columns{width:100%;float:left;box-sizing:border-box;padding:0;display:initial}@media(min-width: 400px){#onetrust-banner-sdk .ot-sdk-container,#onetrust-pc-sdk .ot-sdk-container,#ot-sdk-cookie-policy .ot-sdk-container{width:90%;padding:0}}@media(min-width: 550px){#onetrust-banner-sdk .ot-sdk-container,#onetrust-pc-sdk .ot-sdk-container,#ot-sdk-cookie-policy .ot-sdk-container{width:100%}#onetrust-banner-sdk .ot-sdk-column,#onetrust-banner-sdk .ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-column,#onetrust-pc-sdk .ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-column,#ot-sdk-cookie-policy .ot-sdk-columns{margin-left:4%}#onetrust-banner-sdk .ot-sdk-column:first-child,#onetrust-banner-sdk .ot-sdk-columns:first-child,#onetrust-pc-sdk .ot-sdk-column:first-child,#onetrust-pc-sdk .ot-sdk-columns:first-child,#ot-sdk-cookie-policy .ot-sdk-column:first-child,#ot-sdk-cookie-policy .ot-sdk-columns:first-child{margin-left:0}#onetrust-banner-sdk .ot-sdk-two.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-two.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-two.ot-sdk-columns{width:13.3333333333%}#onetrust-banner-sdk .ot-sdk-three.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-three.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-three.ot-sdk-columns{width:22%}#onetrust-banner-sdk .ot-sdk-four.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-four.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-four.ot-sdk-columns{width:30.6666666667%}#onetrust-banner-sdk .ot-sdk-eight.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-eight.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-eight.ot-sdk-columns{width:65.3333333333%}#onetrust-banner-sdk .ot-sdk-nine.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-nine.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-nine.ot-sdk-columns{width:74%}#onetrust-banner-sdk .ot-sdk-ten.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-ten.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-ten.ot-sdk-columns{width:82.6666666667%}#onetrust-banner-sdk .ot-sdk-eleven.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-eleven.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-eleven.ot-sdk-columns{width:91.3333333333%}#onetrust-banner-sdk .ot-sdk-twelve.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-twelve.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-twelve.ot-sdk-columns{width:100%;margin-left:0}}#onetrust-banner-sdk h1,#onetrust-banner-sdk h2,#onetrust-banner-sdk h3,#onetrust-banner-sdk h4,#onetrust-banner-sdk h5,#onetrust-banner-sdk h6,#onetrust-pc-sdk h1,#onetrust-pc-sdk h2,#onetrust-pc-sdk h3,#onetrust-pc-sdk h4,#onetrust-pc-sdk h5,#onetrust-pc-sdk h6,#ot-sdk-cookie-policy h1,#ot-sdk-cookie-policy h2,#ot-sdk-cookie-policy h3,#ot-sdk-cookie-policy h4,#ot-sdk-cookie-policy h5,#ot-sdk-cookie-policy h6{margin-top:0;font-weight:600;font-family:inherit}#onetrust-banner-sdk h1,#onetrust-pc-sdk h1,#ot-sdk-cookie-policy h1{font-size:1.5rem;line-height:1.2}#onetrust-banner-sdk h2,#onetrust-pc-sdk h2,#ot-sdk-cookie-policy h2{font-size:1.5rem;line-height:1.25}#onetrust-banner-sdk h3,#onetrust-pc-sdk h3,#ot-sdk-cookie-policy h3{font-size:1.5rem;line-height:1.3}#onetrust-banner-sdk h4,#onetrust-pc-sdk h4,#ot-sdk-cookie-policy h4{font-size:1.5rem;line-height:1.35}#onetrust-banner-sdk h5,#onetrust-pc-sdk h5,#ot-sdk-cookie-policy h5{font-size:1.5rem;line-height:1.5}#onetrust-banner-sdk h6,#onetrust-pc-sdk h6,#ot-sdk-cookie-policy h6{font-size:1.5rem;line-height:1.6}@media(min-width: 550px){#onetrust-banner-sdk h1,#onetrust-pc-sdk h1,#ot-sdk-cookie-policy h1{font-size:1.5rem}#onetrust-banner-sdk h2,#onetrust-pc-sdk h2,#ot-sdk-cookie-policy h2{font-size:1.5rem}#onetrust-banner-sdk h3,#onetrust-pc-sdk h3,#ot-sdk-cookie-policy h3{font-size:1.5rem}#onetrust-banner-sdk h4,#onetrust-pc-sdk h4,#ot-sdk-cookie-policy h4{font-size:1.5rem}#onetrust-banner-sdk h5,#onetrust-pc-sdk h5,#ot-sdk-cookie-policy h5{font-size:1.5rem}#onetrust-banner-sdk h6,#onetrust-pc-sdk h6,#ot-sdk-cookie-policy h6{font-size:1.5rem}}#onetrust-banner-sdk p,#onetrust-pc-sdk p,#ot-sdk-cookie-policy p{margin:0 0 1em 0;font-family:inherit;line-height:normal}#onetrust-banner-sdk a,#onetrust-pc-sdk a,#ot-sdk-cookie-policy a{color:#565656;text-decoration:underline}#onetrust-banner-sdk a:hover,#onetrust-pc-sdk a:hover,#ot-sdk-cookie-policy a:hover{color:#565656;text-decoration:none}#onetrust-banner-sdk .ot-sdk-button,#onetrust-banner-sdk button,#onetrust-pc-sdk .ot-sdk-button,#onetrust-pc-sdk button,#ot-sdk-cookie-policy .ot-sdk-button,#ot-sdk-cookie-policy button{margin-bottom:1rem;font-family:inherit}#onetrust-banner-sdk .ot-sdk-button,#onetrust-banner-sdk button,#onetrust-pc-sdk .ot-sdk-button,#onetrust-pc-sdk button,#ot-sdk-cookie-policy .ot-sdk-button,#ot-sdk-cookie-policy button{display:inline-block;height:38px;padding:0 30px;color:#555;text-align:center;font-size:.9em;font-weight:400;line-height:38px;letter-spacing:.01em;text-decoration:none;white-space:nowrap;background-color:rgba(0,0,0,0);border-radius:2px;border:1px solid #bbb;cursor:pointer;box-sizing:border-box}#onetrust-banner-sdk .ot-sdk-button:hover,#onetrust-banner-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):hover,#onetrust-banner-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):focus,#onetrust-pc-sdk .ot-sdk-button:hover,#onetrust-pc-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):hover,#onetrust-pc-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):focus,#ot-sdk-cookie-policy .ot-sdk-button:hover,#ot-sdk-cookie-policy :not(.ot-leg-btn-container)>button:not(.ot-link-btn):hover,#ot-sdk-cookie-policy :not(.ot-leg-btn-container)>button:not(.ot-link-btn):focus{color:#333;border-color:#888;opacity:.9}#onetrust-banner-sdk .ot-sdk-button:focus,#onetrust-banner-sdk :not(.ot-leg-btn-container)>button:focus,#onetrust-pc-sdk .ot-sdk-button:focus,#onetrust-pc-sdk :not(.ot-leg-btn-container)>button:focus,#ot-sdk-cookie-policy .ot-sdk-button:focus,#ot-sdk-cookie-policy :not(.ot-leg-btn-container)>button:focus{outline:2px solid #000}#onetrust-banner-sdk .ot-sdk-button.ot-sdk-button-primary,#onetrust-banner-sdk button.ot-sdk-button-primary,#onetrust-banner-sdk input[type=submit].ot-sdk-button-primary,#onetrust-banner-sdk input[type=reset].ot-sdk-button-primary,#onetrust-banner-sdk input[type=button].ot-sdk-button-primary,#onetrust-pc-sdk .ot-sdk-button.ot-sdk-button-primary,#onetrust-pc-sdk button.ot-sdk-button-primary,#onetrust-pc-sdk input[type=submit].ot-sdk-button-primary,#onetrust-pc-sdk input[type=reset].ot-sdk-button-primary,#onetrust-pc-sdk input[type=button].ot-sdk-button-primary,#ot-sdk-cookie-policy .ot-sdk-button.ot-sdk-button-primary,#ot-sdk-cookie-policy button.ot-sdk-button-primary,#ot-sdk-cookie-policy input[type=submit].ot-sdk-button-primary,#ot-sdk-cookie-policy input[type=reset].ot-sdk-button-primary,#ot-sdk-cookie-policy input[type=button].ot-sdk-button-primary{color:#fff;background-color:#33c3f0;border-color:#33c3f0}#onetrust-banner-sdk .ot-sdk-button.ot-sdk-button-primary:hover,#onetrust-banner-sdk button.ot-sdk-button-primary:hover,#onetrust-banner-sdk input[type=submit].ot-sdk-button-primary:hover,#onetrust-banner-sdk input[type=reset].ot-sdk-button-primary:hover,#onetrust-banner-sdk input[type=button].ot-sdk-button-primary:hover,#onetrust-banner-sdk .ot-sdk-button.ot-sdk-button-primary:focus,#onetrust-banner-sdk button.ot-sdk-button-primary:focus,#onetrust-banner-sdk input[type=submit].ot-sdk-button-primary:focus,#onetrust-banner-sdk input[type=reset].ot-sdk-button-primary:focus,#onetrust-banner-sdk input[type=button].ot-sdk-button-primary:focus,#onetrust-pc-sdk .ot-sdk-button.ot-sdk-button-primary:hover,#onetrust-pc-sdk button.ot-sdk-button-primary:hover,#onetrust-pc-sdk input[type=submit].ot-sdk-button-primary:hover,#onetrust-pc-sdk input[type=reset].ot-sdk-button-primary:hover,#onetrust-pc-sdk input[type=button].ot-sdk-button-primary:hover,#onetrust-pc-sdk .ot-sdk-button.ot-sdk-button-primary:focus,#onetrust-pc-sdk button.ot-sdk-button-primary:focus,#onetrust-pc-sdk input[type=submit].ot-sdk-button-primary:focus,#onetrust-pc-sdk input[type=reset].ot-sdk-button-primary:focus,#onetrust-pc-sdk input[type=button].ot-sdk-button-primary:focus,#ot-sdk-cookie-policy .ot-sdk-button.ot-sdk-button-primary:hover,#ot-sdk-cookie-policy button.ot-sdk-button-primary:hover,#ot-sdk-cookie-policy input[type=submit].ot-sdk-button-primary:hover,#ot-sdk-cookie-policy input[type=reset].ot-sdk-button-primary:hover,#ot-sdk-cookie-policy input[type=button].ot-sdk-button-primary:hover,#ot-sdk-cookie-policy .ot-sdk-button.ot-sdk-button-primary:focus,#ot-sdk-cookie-policy button.ot-sdk-button-primary:focus,#ot-sdk-cookie-policy input[type=submit].ot-sdk-button-primary:focus,#ot-sdk-cookie-policy input[type=reset].ot-sdk-button-primary:focus,#ot-sdk-cookie-policy input[type=button].ot-sdk-button-primary:focus{color:#fff;background-color:#1eaedb;border-color:#1eaedb}#onetrust-banner-sdk input[type=text],#onetrust-pc-sdk input[type=text],#ot-sdk-cookie-policy input[type=text]{height:38px;padding:6px 10px;background-color:#fff;border:1px solid #d1d1d1;border-radius:4px;box-shadow:none;box-sizing:border-box}#onetrust-banner-sdk input[type=text],#onetrust-pc-sdk input[type=text],#ot-sdk-cookie-policy input[type=text]{-webkit-appearance:none;-moz-appearance:none;appearance:none}#onetrust-banner-sdk input[type=text]:focus,#onetrust-pc-sdk input[type=text]:focus,#ot-sdk-cookie-policy input[type=text]:focus{border:1px solid #000;outline:0}#onetrust-banner-sdk label,#onetrust-pc-sdk label,#ot-sdk-cookie-policy label{display:block;margin-bottom:.5rem;font-weight:600}#onetrust-banner-sdk input[type=checkbox],#onetrust-pc-sdk input[type=checkbox],#ot-sdk-cookie-policy input[type=checkbox]{display:inline}#onetrust-banner-sdk ul,#onetrust-pc-sdk ul,#ot-sdk-cookie-policy ul{list-style:circle inside}#onetrust-banner-sdk ul,#onetrust-pc-sdk ul,#ot-sdk-cookie-policy ul{padding-left:0;margin-top:0}#onetrust-banner-sdk ul ul,#onetrust-pc-sdk ul ul,#ot-sdk-cookie-policy ul ul{margin:1.5rem 0 1.5rem 3rem;font-size:90%}#onetrust-banner-sdk li,#onetrust-pc-sdk li,#ot-sdk-cookie-policy li{margin-bottom:1rem}#onetrust-banner-sdk th,#onetrust-banner-sdk td,#onetrust-pc-sdk th,#onetrust-pc-sdk td,#ot-sdk-cookie-policy th,#ot-sdk-cookie-policy td{padding:12px 15px;text-align:left;border-bottom:1px solid #e1e1e1}#onetrust-banner-sdk button,#onetrust-pc-sdk button,#ot-sdk-cookie-policy button{margin-bottom:1rem;font-family:inherit}#onetrust-banner-sdk .ot-sdk-container:after,#onetrust-banner-sdk .ot-sdk-row:after,#onetrust-pc-sdk .ot-sdk-container:after,#onetrust-pc-sdk .ot-sdk-row:after,#ot-sdk-cookie-policy .ot-sdk-container:after,#ot-sdk-cookie-policy .ot-sdk-row:after{content:"";display:table;clear:both}#onetrust-banner-sdk .ot-sdk-row,#onetrust-pc-sdk .ot-sdk-row,#ot-sdk-cookie-policy .ot-sdk-row{margin:0;max-width:none;display:block}#onetrust-banner-sdk{box-shadow:0 0 18px rgba(0,0,0,.2)}#onetrust-banner-sdk.otFlat{position:fixed;z-index:2147483645;bottom:0;right:0;left:0;background-color:#fff;max-height:90%;overflow-x:hidden;overflow-y:auto}#onetrust-banner-sdk.otFlat.top{top:0px;bottom:auto}#onetrust-banner-sdk.otRelFont{font-size:1rem}#onetrust-banner-sdk>.ot-sdk-container{overflow:hidden}#onetrust-banner-sdk::-webkit-scrollbar{width:11px}#onetrust-banner-sdk::-webkit-scrollbar-thumb{border-radius:10px;background:#c1c1c1}#onetrust-banner-sdk{scrollbar-arrow-color:#c1c1c1;scrollbar-darkshadow-color:#c1c1c1;scrollbar-face-color:#c1c1c1;scrollbar-shadow-color:#c1c1c1}#onetrust-banner-sdk #onetrust-policy{margin:1.25em 0 .625em 2em;overflow:hidden}#onetrust-banner-sdk #onetrust-policy .ot-gv-list-handler{float:left;font-size:.82em;padding:0;margin-bottom:0;border:0;line-height:normal;height:auto;width:auto}#onetrust-banner-sdk #onetrust-policy-title{font-size:1.2em;line-height:1.3;margin-bottom:10px}#onetrust-banner-sdk #onetrust-policy-text{clear:both;text-align:left;font-size:.88em;line-height:1.4}#onetrust-banner-sdk #onetrust-policy-text *{font-size:inherit;line-height:inherit}#onetrust-banner-sdk #onetrust-policy-text a{font-weight:bold;margin-left:5px}#onetrust-banner-sdk #onetrust-policy-title,#onetrust-banner-sdk #onetrust-policy-text{color:dimgray;float:left}#onetrust-banner-sdk #onetrust-button-group-parent{min-height:1px;text-align:center}#onetrust-banner-sdk #onetrust-button-group{display:inline-block}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{background-color:#68b631;color:#fff;border-color:#68b631;margin-right:1em;min-width:125px;height:auto;white-space:normal;word-break:break-word;word-wrap:break-word;padding:12px 10px;line-height:1.2;font-size:.813em;font-weight:600}#onetrust-banner-sdk #onetrust-pc-btn-handler.cookie-setting-link{background-color:#fff;border:none;color:#68b631;text-decoration:underline;padding-left:0;padding-right:0}#onetrust-banner-sdk .onetrust-close-btn-ui{width:44px;height:44px;background-size:12px;border:none;position:relative;margin:auto;padding:0}#onetrust-banner-sdk .banner_logo{display:none}#onetrust-banner-sdk.ot-bnr-w-logo .ot-bnr-logo{position:absolute;top:50%;transform:translateY(-50%);left:0px}#onetrust-banner-sdk.ot-bnr-w-logo #onetrust-policy{margin-left:65px}#onetrust-banner-sdk .ot-b-addl-desc{clear:both;float:left;display:block}#onetrust-banner-sdk #banner-options{float:left;display:table;margin-right:0;margin-left:1em;width:calc(100% - 1em)}#onetrust-banner-sdk .banner-option-input{cursor:pointer;width:auto;height:auto;border:none;padding:0;padding-right:3px;margin:0 0 10px;font-size:.82em;line-height:1.4}#onetrust-banner-sdk .banner-option-input *{pointer-events:none;font-size:inherit;line-height:inherit}#onetrust-banner-sdk .banner-option-input[aria-expanded=true]~.banner-option-details{display:block;height:auto}#onetrust-banner-sdk .banner-option-input[aria-expanded=true] .ot-arrow-container{transform:rotate(90deg)}#onetrust-banner-sdk .banner-option{margin-bottom:12px;margin-left:0;border:none;float:left;padding:0}#onetrust-banner-sdk .banner-option:first-child{padding-left:2px}#onetrust-banner-sdk .banner-option:not(:first-child){padding:0;border:none}#onetrust-banner-sdk .banner-option-header{cursor:pointer;display:inline-block}#onetrust-banner-sdk .banner-option-header :first-child{color:dimgray;font-weight:bold;float:left}#onetrust-banner-sdk .banner-option-header .ot-arrow-container{display:inline-block;border-top:6px solid rgba(0,0,0,0);border-bottom:6px solid rgba(0,0,0,0);border-left:6px solid dimgray;margin-left:10px;vertical-align:middle}#onetrust-banner-sdk .banner-option-details{display:none;font-size:.83em;line-height:1.5;padding:10px 0px 5px 10px;margin-right:10px;height:0px}#onetrust-banner-sdk .banner-option-details *{font-size:inherit;line-height:inherit;color:dimgray}#onetrust-banner-sdk .ot-arrow-container,#onetrust-banner-sdk .banner-option-details{transition:all 300ms ease-in 0s;-webkit-transition:all 300ms ease-in 0s;-moz-transition:all 300ms ease-in 0s;-o-transition:all 300ms ease-in 0s}#onetrust-banner-sdk .ot-dpd-container{float:left}#onetrust-banner-sdk .ot-dpd-title{margin-bottom:10px}#onetrust-banner-sdk .ot-dpd-title,#onetrust-banner-sdk .ot-dpd-desc{font-size:.88em;line-height:1.4;color:dimgray}#onetrust-banner-sdk .ot-dpd-title *,#onetrust-banner-sdk .ot-dpd-desc *{font-size:inherit;line-height:inherit}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-text *{margin-bottom:0}#onetrust-banner-sdk.ot-iab-2 .onetrust-vendors-list-handler{display:block;margin-left:0;margin-top:5px;clear:both;margin-bottom:0;padding:0;border:0;height:auto;width:auto}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group button{display:block}#onetrust-banner-sdk.ot-close-btn-link{padding-top:25px}#onetrust-banner-sdk.ot-close-btn-link #onetrust-close-btn-container{top:15px;transform:none;right:15px}#onetrust-banner-sdk.ot-close-btn-link #onetrust-close-btn-container button{padding:0;white-space:pre-wrap;border:none;height:auto;line-height:1.5;text-decoration:underline;font-size:.69em}#onetrust-banner-sdk #onetrust-policy-text,#onetrust-banner-sdk .ot-dpd-desc,#onetrust-banner-sdk .ot-b-addl-desc{font-size:.813em;line-height:1.5}#onetrust-banner-sdk .ot-dpd-desc{margin-bottom:10px}#onetrust-banner-sdk .ot-dpd-desc>.ot-b-addl-desc{margin-top:10px;margin-bottom:10px;font-size:1em}@media only screen and (max-width: 425px){#onetrust-banner-sdk #onetrust-close-btn-container{position:absolute;top:6px;right:2px}#onetrust-banner-sdk #onetrust-policy{margin-left:0;margin-top:3em}#onetrust-banner-sdk #onetrust-button-group{display:block}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{width:100%}#onetrust-banner-sdk .onetrust-close-btn-ui{top:auto;transform:none}#onetrust-banner-sdk #onetrust-policy-title{display:inline;float:none}#onetrust-banner-sdk #banner-options{margin:0;padding:0;width:100%}}@media only screen and (min-width: 426px)and (max-width: 896px){#onetrust-banner-sdk #onetrust-close-btn-container{position:absolute;top:0;right:0}#onetrust-banner-sdk #onetrust-policy{margin-left:1em;margin-right:1em}#onetrust-banner-sdk .onetrust-close-btn-ui{top:10px;right:10px}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{width:95%}#onetrust-banner-sdk.ot-iab-2 #onetrust-group-container{width:100%}#onetrust-banner-sdk.ot-bnr-w-logo #onetrust-button-group-parent{padding-left:50px}#onetrust-banner-sdk #onetrust-button-group-parent{width:100%;position:relative;margin-left:0}#onetrust-banner-sdk #onetrust-button-group button{display:inline-block}#onetrust-banner-sdk #onetrust-button-group{margin-right:0;text-align:center}#onetrust-banner-sdk .has-reject-all-button #onetrust-pc-btn-handler{float:left}#onetrust-banner-sdk .has-reject-all-button #onetrust-reject-all-handler,#onetrust-banner-sdk .has-reject-all-button #onetrust-accept-btn-handler{float:right}#onetrust-banner-sdk .has-reject-all-button #onetrust-button-group{width:calc(100% - 2em);margin-right:0}#onetrust-banner-sdk .has-reject-all-button #onetrust-pc-btn-handler.cookie-setting-link{padding-left:0px;text-align:left}#onetrust-banner-sdk.ot-buttons-fw .ot-sdk-three button{width:100%;text-align:center}#onetrust-banner-sdk.ot-buttons-fw #onetrust-button-group-parent button{float:none}#onetrust-banner-sdk.ot-buttons-fw #onetrust-pc-btn-handler.cookie-setting-link{text-align:center}}@media only screen and (min-width: 550px){#onetrust-banner-sdk .banner-option:not(:first-child){border-left:1px solid #d8d8d8;padding-left:25px}}@media only screen and (min-width: 425px)and (max-width: 550px){#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group,#onetrust-banner-sdk.ot-iab-2 #onetrust-policy,#onetrust-banner-sdk.ot-iab-2 .banner-option{width:100%}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-accept-btn-handler,#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-reject-all-handler,#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-pc-btn-handler{width:100%}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-accept-btn-handler,#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-reject-all-handler{float:left}}@media only screen and (min-width: 769px){#onetrust-banner-sdk #onetrust-button-group{margin-right:30%}#onetrust-banner-sdk #banner-options{margin-left:2em;margin-right:5em;margin-bottom:1.25em;width:calc(100% - 7em)}}@media only screen and (min-width: 897px)and (max-width: 1023px){#onetrust-banner-sdk.vertical-align-content #onetrust-button-group-parent{position:absolute;top:50%;left:75%;transform:translateY(-50%)}#onetrust-banner-sdk #onetrust-close-btn-container{top:50%;margin:auto;transform:translate(-50%, -50%);position:absolute;padding:0;right:0}#onetrust-banner-sdk #onetrust-close-btn-container button{position:relative;margin:0;right:-22px;top:2px}}@media only screen and (min-width: 1024px){#onetrust-banner-sdk #onetrust-close-btn-container{top:50%;margin:auto;transform:translate(-50%, -50%);position:absolute;right:0}#onetrust-banner-sdk #onetrust-close-btn-container button{right:-12px}#onetrust-banner-sdk #onetrust-policy{margin-left:2em}#onetrust-banner-sdk.vertical-align-content #onetrust-button-group-parent{position:absolute;top:50%;left:60%;transform:translateY(-50%)}#onetrust-banner-sdk .ot-optout-signal{width:50%}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-title{width:50%}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-text,#onetrust-banner-sdk.ot-iab-2 :not(.ot-dpd-desc)>.ot-b-addl-desc{margin-bottom:1em;width:50%;border-right:1px solid #d8d8d8;padding-right:1rem}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-text{margin-bottom:0;padding-bottom:1em}#onetrust-banner-sdk.ot-iab-2 :not(.ot-dpd-desc)>.ot-b-addl-desc{margin-bottom:0;padding-bottom:1em}#onetrust-banner-sdk.ot-iab-2 .ot-dpd-container{width:45%;padding-left:1rem;display:inline-block;float:none}#onetrust-banner-sdk.ot-iab-2 .ot-dpd-title{line-height:1.7}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group-parent{left:auto;right:4%;margin-left:0}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group button{display:block}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{margin:auto;width:30%}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{width:60%}#onetrust-banner-sdk #onetrust-button-group{margin-right:auto}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{margin-top:1em}}@media only screen and (min-width: 890px){#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group-parent{padding-left:3%;padding-right:4%;margin-left:0}#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group{margin-right:0;margin-top:1.25em;width:100%}#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button{width:100%;margin-bottom:5px;margin-top:5px}#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button:last-of-type{margin-bottom:20px}}@media only screen and (min-width: 1280px){#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{width:55%}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{width:44%;padding-left:2%;padding-right:2%}#onetrust-banner-sdk:not(.ot-iab-2).vertical-align-content #onetrust-button-group-parent{position:absolute;left:55%}}
        #onetrust-consent-sdk #onetrust-banner-sdk {background-color: #FFF;}
            #onetrust-consent-sdk #onetrust-policy-title,
                    #onetrust-consent-sdk #onetrust-policy-text,
                    #onetrust-consent-sdk .ot-b-addl-desc,
                    #onetrust-consent-sdk .ot-dpd-desc,
                    #onetrust-consent-sdk .ot-dpd-title,
                    #onetrust-consent-sdk #onetrust-policy-text *:not(.onetrust-vendors-list-handler),
                    #onetrust-consent-sdk .ot-dpd-desc *:not(.onetrust-vendors-list-handler),
                    #onetrust-consent-sdk #onetrust-banner-sdk #banner-options *,
                    #onetrust-banner-sdk .ot-cat-header,
                    #onetrust-banner-sdk .ot-optout-signal
                    {
                        color: #2E2E2E;
                    }
            #onetrust-consent-sdk #onetrust-banner-sdk .banner-option-details {
                    background-color: #E9E9E9;}
             #onetrust-consent-sdk #onetrust-banner-sdk a[href],
                    #onetrust-consent-sdk #onetrust-banner-sdk a[href] font,
                    #onetrust-consent-sdk #onetrust-banner-sdk .ot-link-btn
                        {
                            color: #007398;
                        }#onetrust-consent-sdk #onetrust-accept-btn-handler,
                         #onetrust-banner-sdk #onetrust-reject-all-handler {
                            background-color: #007398;border-color: #007398;
                color: #FFF;
            }
            #onetrust-consent-sdk #onetrust-banner-sdk *:focus,
            #onetrust-consent-sdk #onetrust-banner-sdk:focus {
               outline-color: #000000;
               outline-width: 1px;
            }
            #onetrust-consent-sdk #onetrust-pc-btn-handler,
            #onetrust-consent-sdk #onetrust-pc-btn-handler.cookie-setting-link {
                color: #6CC04A; border-color: #6CC04A;
                background-color:
                #FFF;
            }/*! v6.10.0 2021-01-18 */
div#onetrust-consent-sdk #onetrust-banner-sdk{border-top:2px solid #d0d3d4!important;box-shadow:none;padding:24px}div#onetrust-consent-sdk button{border-radius:0!important;box-shadow:none!important;box-sizing:border-box!important;font-size:20px!important;font-weight:400!important;letter-spacing:0!important;max-width:none!important;white-space:nowrap!important}div#onetrust-consent-sdk button:not(.ot-link-btn){background-color:#007dbc!important;border:2px solid #007dbc!important;color:#fff!important;height:48px!important;padding:0 1em!important;width:auto!important}div#onetrust-consent-sdk button:hover{background-color:#fff!important;border-color:#949494!important;color:#2e2e2e!important}div#onetrust-consent-sdk button.ot-link-btn{color:#007dbc!important;font-size:16px!important;text-decoration:underline}div#onetrust-consent-sdk button.ot-link-btn:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-consent-sdk a,div#onetrust-pc-sdk a{color:#007dbc!important;text-decoration:underline!important}div#onetrust-consent-sdk a,div#onetrust-consent-sdk button,div#onetrust-consent-sdk p:hover{opacity:1!important}div#onetrust-consent-sdk a:focus,div#onetrust-consent-sdk button:focus,div#onetrust-consent-sdk input:focus{outline:2px solid #949494!important;outline-offset:1px!important}div#onetrust-banner-sdk .ot-sdk-container{padding:0;width:auto}div#onetrust-banner-sdk .ot-sdk-row{align-items:flex-start;box-sizing:border-box;display:flex;flex-direction:column;justify-content:space-between;margin:auto;max-width:1152px}div#onetrust-banner-sdk .ot-sdk-row:after{display:none}div#onetrust-banner-sdk #onetrust-group-container,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-group-container,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{flex-grow:1;width:auto}div#onetrust-banner-sdk #onetrust-policy,div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy{margin:0;overflow:visible}div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy-text,div#onetrust-consent-sdk #onetrust-policy-text{font-size:16px;line-height:24px;max-width:44em;margin:0}div#onetrust-consent-sdk #onetrust-policy-text a[href]{font-weight:400;margin-left:8px}div#onetrust-banner-sdk #onetrust-button-group-parent{flex:0 0 auto;margin:32px 0 0;width:100%}div#onetrust-banner-sdk #onetrust-button-group{display:flex;flex-direction:row;flex-wrap:wrap;justify-content:flex-end;margin:-8px}div#onetrust-banner-sdk .banner-actions-container{display:flex;flex:1 0 auto}div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button:last-of-type,div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto;margin:8px;width:auto}div#onetrust-consent-sdk #onetrust-pc-btn-handler{background-color:#fff!important;color:inherit!important}div#onetrust-banner-sdk #onetrust-close-btn-container{display:none}@media only screen and (min-width:556px){div#onetrust-consent-sdk #onetrust-banner-sdk{padding:40px}div#onetrust-banner-sdk #onetrust-policy{margin:0 40px 0 0}div#onetrust-banner-sdk .ot-sdk-row{align-items:center;flex-direction:row}div#onetrust-banner-sdk #onetrust-button-group-parent,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-button-group-parent,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{margin:0;padding:0;width:auto}div#onetrust-banner-sdk #onetrust-button-group,div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group{align-items:stretch;flex-direction:column-reverse;margin:0}div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto}}@media only screen and (min-width:768px){div#onetrust-banner-sdk #onetrust-policy{margin:0 48px 0 0}div#onetrust-consent-sdk #onetrust-banner-sdk{padding:48px}}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-size:16px;line-height:24px}div#onetrust-consent-sdk #onetrust-pc-sdk p,div#onetrust-pc-sdk #ot-pc-desc,div#onetrust-pc-sdk .category-host-list-handler,div#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{font-size:16px;font-weight:400;line-height:24px}div#onetrust-consent-sdk a:hover,div#onetrust-pc-sdk a:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-pc-sdk{border-radius:0;bottom:0;height:auto;left:0;margin:auto;max-width:100%;overflow:hidden;right:0;top:0;width:512px;max-height:800px}div#onetrust-pc-sdk .ot-pc-header{display:none}div#onetrust-pc-sdk #ot-pc-content{overscroll-behavior:contain;padding:0 12px 0 24px;margin:16px 4px 0 0;top:0;right:16px;left:0;width:auto}div#onetrust-pc-sdk #ot-category-title,div#onetrust-pc-sdk #ot-pc-title{font-size:24px;font-weight:400;line-height:32px;margin:0 0 16px}div#onetrust-pc-sdk #ot-pc-desc{padding:0}div#onetrust-pc-sdk #ot-pc-desc a{display:inline}div#onetrust-pc-sdk #accept-recommended-btn-handler{display:none!important}div#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:2px solid #007dbc;outline-offset:-1px;transition:none}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item{border-width:0 0 2px}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item:first-of-type{border-width:2px 0}div#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{padding:8px 0;width:100%}div#onetrust-pc-sdk .ot-plus-minus{transform:translateY(2px)}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item>button{background:0 0!important;border:0!important;height:44px!important;max-width:none!important;width:calc(100% - 48px)!important}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-weight:700}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr{padding:0}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item .ot-acc-grpdesc{padding:0;width:100%}div#onetrust-pc-sdk .ot-acc-grpcntr .ot-subgrp-cntr{border:0;padding:0}div#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp{margin:0}div#onetrust-pc-sdk .ot-always-active-group .ot-cat-header{width:calc(100% - 160px)}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{width:calc(100% - 88px)}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active{color:inherit;font-size:12px;font-weight:400;line-height:1.5;padding-right:48px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:before{border-radius:12px;position:absolute;right:0;top:0;content:'';background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:after{border-radius:50%;position:absolute;right:5px;top:4px;content:'';background-color:#007dbc;height:12px;width:12px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active,div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl{right:2px}div#onetrust-pc-sdk .ot-switch{display:block;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob,div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob{background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob:before{background-color:#737373;height:8px;left:4px;top:4px;width:8px}div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob:before{background-color:#007dbc;height:12px;left:0;top:2px;width:12px}div#onetrust-pc-sdk .ot-tgl input:focus+.ot-switch .ot-switch-nob{box-shadow:0 0;outline:2px solid #007dbc!important;outline-offset:1px;transition:none}div#onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-grpcntr.ot-acc-txt{background-color:transparent;padding-left:3px}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr,div#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr{overflow:visible;width:100%}div#onetrust-pc-sdk .ot-pc-footer{border-top:0 solid}div#onetrust-pc-sdk .ot-btn-container{padding-top:24px;text-align:center}div#onetrust-pc-sdk .ot-pc-footer button{margin:8px 0;background-color:#fff}div#onetrust-pc-sdk .ot-pc-footer-logo{background-color:#fff}div#onetrust-pc-sdk #ot-lst-title span{font-size:24px;font-weight:400;line-height:32px}div#onetrust-pc-sdk #ot-host-lst .ot-host-desc,div#onetrust-pc-sdk #ot-host-lst .ot-host-expand,div#onetrust-pc-sdk #ot-host-lst .ot-host-name,div#onetrust-pc-sdk #ot-host-lst .ot-host-name a,div#onetrust-pc-sdk .back-btn-handler,div#onetrust-pc-sdk .ot-host-opt li>div div{font-size:16px;font-weight:400;line-height:24px}div#onetrust-pc-sdk #ot-pc-lst{top:0}div#onetrust-pc-sdk .back-btn-handler{text-decoration:none!important}div#onetrust-pc-sdk #filter-btn-handler:hover svg{filter:invert(1)}div#onetrust-pc-sdk .back-btn-handler svg{width:16px;height:16px}div#onetrust-pc-sdk .ot-host-item>button{background:0 0!important;border:0!important;height:66px!important;max-width:none!important;width:calc(100% - 5px)!important;transform:translate(2px,2px)}div#onetrust-pc-sdk .ot-host-item{border-bottom:2px solid #b9b9b9;padding:0}div#onetrust-pc-sdk .ot-host-item .ot-acc-hdr{margin:0 0 -6px;padding:8px 0}div#onetrust-pc-sdk ul li:first-child{border-top:2px solid #b9b9b9}div#onetrust-pc-sdk .ot-host-item .ot-plus-minus{margin:0 8px 0 0}div#onetrust-pc-sdk .ot-search-cntr{width:calc(100% - 48px)}div#onetrust-pc-sdk .ot-host-opt .ot-host-info{background-color:transparent}div#onetrust-pc-sdk .ot-host-opt li>div div{padding:0}div#onetrust-pc-sdk #vendor-search-handler{border-radius:0;border-color:#939393;border-style:solid;border-width:2px 0 2px 2px;font-size:20px;height:48px;margin:0}div#onetrust-pc-sdk .ot-lst-subhdr svg{right:0;top:8px}div#onetrust-pc-sdk .ot-fltr-cntr{box-sizing:border-box;right:0;width:48px}div#onetrust-pc-sdk #filter-btn-handler{width:48px!important;padding:8px!important}div#onetrust-pc-sdk #ot-fltr-modal{max-height:400px}
#onetrust-pc-sdk.otPcCenter{overflow:hidden;position:fixed;margin:0 auto;top:5%;right:0;left:0;width:40%;max-width:575px;min-width:575px;border-radius:2.5px;z-index:2147483647;background-color:#fff;-webkit-box-shadow:0px 2px 10px -3px #999;-moz-box-shadow:0px 2px 10px -3px #999;box-shadow:0px 2px 10px -3px #999}#onetrust-pc-sdk.otPcCenter[dir=rtl]{right:0;left:0}#onetrust-pc-sdk.otRelFont{font-size:1rem}#onetrust-pc-sdk .ot-optout-signal{margin-top:.625rem}#onetrust-pc-sdk #ot-addtl-venlst .ot-arw-cntr,#onetrust-pc-sdk #ot-addtl-venlst .ot-plus-minus,#onetrust-pc-sdk .ot-hide-tgl{visibility:hidden}#onetrust-pc-sdk #ot-addtl-venlst .ot-arw-cntr *,#onetrust-pc-sdk #ot-addtl-venlst .ot-plus-minus *,#onetrust-pc-sdk .ot-hide-tgl *{visibility:hidden}#onetrust-pc-sdk #ot-gn-venlst .ot-ven-item .ot-acc-hdr{min-height:40px}#onetrust-pc-sdk .ot-pc-header{height:39px;padding:10px 0 10px 30px;border-bottom:1px solid #e9e9e9}#onetrust-pc-sdk #ot-pc-title,#onetrust-pc-sdk #ot-category-title,#onetrust-pc-sdk .ot-cat-header,#onetrust-pc-sdk #ot-lst-title,#onetrust-pc-sdk .ot-ven-hdr .ot-ven-name,#onetrust-pc-sdk .ot-always-active{font-weight:bold;color:dimgray}#onetrust-pc-sdk .ot-always-active-group .ot-cat-header{width:55%;font-weight:700}#onetrust-pc-sdk .ot-cat-item p{clear:both;float:left;margin-top:10px;margin-bottom:5px;line-height:1.5;font-size:.812em;color:dimgray}#onetrust-pc-sdk .ot-close-icon{height:44px;width:44px;background-size:10px}#onetrust-pc-sdk #ot-pc-title{float:left;font-size:1em;line-height:1.5;margin-bottom:10px;margin-top:10px;width:100%}#onetrust-pc-sdk #accept-recommended-btn-handler{margin-right:10px;margin-bottom:25px;outline-offset:-1px}#onetrust-pc-sdk #ot-pc-desc{clear:both;width:100%;font-size:.812em;line-height:1.5;margin-bottom:25px}#onetrust-pc-sdk #ot-pc-desc *{font-size:inherit;line-height:inherit}#onetrust-pc-sdk #ot-pc-desc ul li{padding:10px 0px}#onetrust-pc-sdk a{color:#656565;cursor:pointer}#onetrust-pc-sdk a:hover{color:#3860be}#onetrust-pc-sdk label{margin-bottom:0}#onetrust-pc-sdk #vdr-lst-dsc{font-size:.812em;line-height:1.5;padding:10px 15px 5px 15px}#onetrust-pc-sdk button{max-width:394px;padding:12px 30px;line-height:1;word-break:break-word;word-wrap:break-word;white-space:normal;font-weight:bold;height:auto}#onetrust-pc-sdk .ot-link-btn{padding:0;margin-bottom:0;border:0;font-weight:normal;line-height:normal;width:auto;height:auto}#onetrust-pc-sdk #ot-pc-content{position:absolute;overflow-y:scroll;padding-left:2px;padding-right:30px;top:60px;bottom:110px;margin:1px 3px 0 30px;width:calc(100% - 63px)}#onetrust-pc-sdk .ot-vs-list .ot-always-active,#onetrust-pc-sdk .ot-cat-grp .ot-always-active{float:right;clear:none;color:#3860be;margin:0;font-size:.813em;line-height:1.3}#onetrust-pc-sdk .ot-pc-scrollbar::-webkit-scrollbar-track{margin-right:20px}#onetrust-pc-sdk .ot-pc-scrollbar::-webkit-scrollbar{width:11px}#onetrust-pc-sdk .ot-pc-scrollbar::-webkit-scrollbar-thumb{border-radius:10px;background:#d8d8d8}#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:#000 1px solid}#onetrust-pc-sdk .ot-pc-scrollbar{scrollbar-arrow-color:#d8d8d8;scrollbar-darkshadow-color:#d8d8d8;scrollbar-face-color:#d8d8d8;scrollbar-shadow-color:#d8d8d8}#onetrust-pc-sdk .save-preference-btn-handler{margin-right:20px}#onetrust-pc-sdk .ot-pc-refuse-all-handler{margin-right:10px}#onetrust-pc-sdk #ot-pc-desc .privacy-notice-link{margin-left:0;margin-right:8px}#onetrust-pc-sdk #ot-pc-desc .ot-imprint-handler{margin-left:0;margin-right:8px}#onetrust-pc-sdk .ot-subgrp-cntr{display:inline-block;clear:both;width:100%;padding-top:15px}#onetrust-pc-sdk .ot-switch+.ot-subgrp-cntr{padding-top:10px}#onetrust-pc-sdk ul.ot-subgrps{margin:0;font-size:initial}#onetrust-pc-sdk ul.ot-subgrps li p,#onetrust-pc-sdk ul.ot-subgrps li h5{font-size:.813em;line-height:1.4;color:dimgray}#onetrust-pc-sdk ul.ot-subgrps .ot-switch{min-height:auto}#onetrust-pc-sdk ul.ot-subgrps .ot-switch-nob{top:0}#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr{display:inline-block;width:100%}#onetrust-pc-sdk ul.ot-subgrps .ot-acc-txt{margin:0}#onetrust-pc-sdk ul.ot-subgrps li{padding:0;border:none}#onetrust-pc-sdk ul.ot-subgrps li h5{position:relative;top:5px;font-weight:bold;margin-bottom:0;float:left}#onetrust-pc-sdk li.ot-subgrp{margin-left:20px;overflow:auto}#onetrust-pc-sdk li.ot-subgrp>h5{width:calc(100% - 100px)}#onetrust-pc-sdk .ot-cat-item p>ul,#onetrust-pc-sdk li.ot-subgrp p>ul{margin:0px;list-style:disc;margin-left:15px;font-size:inherit}#onetrust-pc-sdk .ot-cat-item p>ul li,#onetrust-pc-sdk li.ot-subgrp p>ul li{font-size:inherit;padding-top:10px;padding-left:0px;padding-right:0px;border:none}#onetrust-pc-sdk .ot-cat-item p>ul li:last-child,#onetrust-pc-sdk li.ot-subgrp p>ul li:last-child{padding-bottom:10px}#onetrust-pc-sdk .ot-pc-logo{height:40px;width:120px}#onetrust-pc-sdk .ot-pc-footer{position:absolute;bottom:0px;width:100%;max-height:160px;border-top:1px solid #d8d8d8}#onetrust-pc-sdk.ot-ftr-stacked .ot-pc-refuse-all-handler{margin-bottom:0px}#onetrust-pc-sdk.ot-ftr-stacked #ot-pc-content{bottom:160px}#onetrust-pc-sdk.ot-ftr-stacked .ot-pc-footer button{width:100%;max-width:none}#onetrust-pc-sdk.ot-ftr-stacked .ot-btn-container{margin:0 30px;width:calc(100% - 60px);padding-right:0}#onetrust-pc-sdk .ot-pc-footer-logo{height:30px;width:100%;text-align:right;background:#f4f4f4}#onetrust-pc-sdk .ot-pc-footer-logo a{display:inline-block;margin-top:5px;margin-right:10px}#onetrust-pc-sdk[dir=rtl] .ot-pc-footer-logo{direction:rtl}#onetrust-pc-sdk[dir=rtl] .ot-pc-footer-logo a{margin-right:25px}#onetrust-pc-sdk .ot-tgl{float:right;position:relative;z-index:1}#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob{background-color:#468254;border:1px solid #fff}#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob:before{-webkit-transform:translateX(20px);-ms-transform:translateX(20px);transform:translateX(20px);background-color:#fff;border-color:#fff}#onetrust-pc-sdk .ot-tgl input:focus+.ot-switch{outline:#000 solid 1px}#onetrust-pc-sdk .ot-switch{position:relative;display:inline-block;width:45px;height:25px}#onetrust-pc-sdk .ot-switch-nob{position:absolute;cursor:pointer;top:0;left:0;right:0;bottom:0;background-color:#767676;border:1px solid #ddd;transition:all .2s ease-in 0s;-moz-transition:all .2s ease-in 0s;-o-transition:all .2s ease-in 0s;-webkit-transition:all .2s ease-in 0s;border-radius:20px}#onetrust-pc-sdk .ot-switch-nob:before{position:absolute;content:"";height:18px;width:18px;bottom:3px;left:3px;background-color:#fff;-webkit-transition:.4s;transition:.4s;border-radius:20px}#onetrust-pc-sdk .ot-chkbox input:checked~label::before{background-color:#3860be}#onetrust-pc-sdk .ot-chkbox input+label::after{content:none;color:#fff}#onetrust-pc-sdk .ot-chkbox input:checked+label::after{content:""}#onetrust-pc-sdk .ot-chkbox input:focus+label::before{outline-style:solid;outline-width:2px;outline-style:auto}#onetrust-pc-sdk .ot-chkbox input[aria-checked=mixed]~label::before{background-color:#3860be}#onetrust-pc-sdk .ot-chkbox input[aria-checked=mixed]+label::after{content:""}#onetrust-pc-sdk .ot-chkbox label{position:relative;display:inline-block;padding-left:30px;cursor:pointer;font-weight:500}#onetrust-pc-sdk .ot-chkbox label::before,#onetrust-pc-sdk .ot-chkbox label::after{position:absolute;content:"";display:inline-block;border-radius:3px}#onetrust-pc-sdk .ot-chkbox label::before{height:18px;width:18px;border:1px solid #3860be;left:0px;top:auto}#onetrust-pc-sdk .ot-chkbox label::after{height:5px;width:9px;border-left:3px solid;border-bottom:3px solid;transform:rotate(-45deg);-o-transform:rotate(-45deg);-ms-transform:rotate(-45deg);-webkit-transform:rotate(-45deg);left:4px;top:5px}#onetrust-pc-sdk .ot-label-txt{display:none}#onetrust-pc-sdk .ot-chkbox input,#onetrust-pc-sdk .ot-tgl input{position:absolute;opacity:0;width:0;height:0}#onetrust-pc-sdk .ot-arw-cntr{float:right;position:relative;pointer-events:none}#onetrust-pc-sdk .ot-arw-cntr .ot-arw{width:16px;height:16px;margin-left:5px;color:dimgray;display:inline-block;vertical-align:middle;-webkit-transition:all 150ms ease-in 0s;-moz-transition:all 150ms ease-in 0s;-o-transition:all 150ms ease-in 0s;transition:all 150ms ease-in 0s}#onetrust-pc-sdk input:checked~.ot-acc-hdr .ot-arw,#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-arw-cntr svg{transform:rotate(90deg);-o-transform:rotate(90deg);-ms-transform:rotate(90deg);-webkit-transform:rotate(90deg)}#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:#000 1px solid}#onetrust-pc-sdk .ot-tgl-cntr,#onetrust-pc-sdk .ot-arw-cntr{display:inline-block}#onetrust-pc-sdk .ot-tgl-cntr{width:45px;float:right;margin-top:2px}#onetrust-pc-sdk #ot-lst-cnt .ot-tgl-cntr{margin-top:10px}#onetrust-pc-sdk .ot-always-active-subgroup{width:auto;padding-left:0px !important;top:3px;position:relative}#onetrust-pc-sdk .ot-label-status{font-size:.75em;display:none;font-size:.75em;position:relative;top:2px;padding-right:5px;float:left}#onetrust-pc-sdk .ot-arw-cntr{margin-top:-1px}#onetrust-pc-sdk .ot-arw-cntr svg{-webkit-transition:all 300ms ease-in 0s;-moz-transition:all 300ms ease-in 0s;-o-transition:all 300ms ease-in 0s;transition:all 300ms ease-in 0s;height:10px;width:10px}#onetrust-pc-sdk input:checked~.ot-acc-hdr .ot-arw{transform:rotate(90deg);-o-transform:rotate(90deg);-ms-transform:rotate(90deg);-webkit-transform:rotate(90deg)}#onetrust-pc-sdk .ot-arw{width:10px;margin-left:15px;transition:all 300ms ease-in 0s;-webkit-transition:all 300ms ease-in 0s;-moz-transition:all 300ms ease-in 0s;-o-transition:all 300ms ease-in 0s}#onetrust-pc-sdk .ot-vlst-cntr{margin-bottom:0}#onetrust-pc-sdk .ot-hlst-cntr{margin-top:5px;display:inline-block;width:100%}#onetrust-pc-sdk .category-vendors-list-handler,#onetrust-pc-sdk .category-vendors-list-handler+a,#onetrust-pc-sdk .category-host-list-handler{clear:both;color:#3860be;margin-left:0;font-size:.813em;text-decoration:none;float:left;overflow:hidden}#onetrust-pc-sdk .category-vendors-list-handler:hover,#onetrust-pc-sdk .category-vendors-list-handler+a:hover,#onetrust-pc-sdk .category-host-list-handler:hover{text-decoration-line:underline}#onetrust-pc-sdk .category-vendors-list-handler+a{clear:none}#onetrust-pc-sdk .ot-vlst-cntr .ot-ext-lnk,#onetrust-pc-sdk .ot-ven-hdr .ot-ext-lnk{display:inline-block;height:13px;width:13px;background-repeat:no-repeat;margin-left:1px;margin-top:6px;cursor:pointer}#onetrust-pc-sdk .ot-ven-hdr .ot-ext-lnk{margin-bottom:-1px}#onetrust-pc-sdk .back-btn-handler{font-size:1em;text-decoration:none}#onetrust-pc-sdk .back-btn-handler:hover{opacity:.6}#onetrust-pc-sdk #ot-lst-title h3{display:inline-block;word-break:break-word;word-wrap:break-word;margin-bottom:0;color:#656565;font-size:1em;font-weight:bold;margin-left:15px}#onetrust-pc-sdk #ot-lst-title{margin:10px 0 10px 0px;font-size:1em;text-align:left}#onetrust-pc-sdk #ot-pc-hdr{margin:0 0 0 30px;height:auto;width:auto}#onetrust-pc-sdk #ot-pc-hdr input::placeholder{color:#707070;font-style:italic}#onetrust-pc-sdk #vendor-search-handler{height:31px;width:100%;border-radius:50px;font-size:.8em;padding-right:35px;padding-left:15px;float:left;margin-left:15px}#onetrust-pc-sdk .ot-ven-name{display:block;width:auto;padding-right:5px}#onetrust-pc-sdk #ot-lst-cnt{overflow-y:auto;margin-left:20px;margin-right:7px;width:calc(100% - 27px);max-height:calc(100% - 80px);height:100%;transform:translate3d(0, 0, 0)}#onetrust-pc-sdk #ot-pc-lst{width:100%;bottom:100px;position:absolute;top:60px}#onetrust-pc-sdk #ot-pc-lst:not(.ot-enbl-chr) .ot-tgl-cntr .ot-arw-cntr,#onetrust-pc-sdk #ot-pc-lst:not(.ot-enbl-chr) .ot-tgl-cntr .ot-arw-cntr *{visibility:hidden}#onetrust-pc-sdk #ot-pc-lst .ot-tgl-cntr{right:12px;position:absolute}#onetrust-pc-sdk #ot-pc-lst .ot-arw-cntr{float:right;position:relative}#onetrust-pc-sdk #ot-pc-lst .ot-arw{margin-left:10px}#onetrust-pc-sdk #ot-pc-lst .ot-acc-hdr{overflow:hidden;cursor:pointer}#onetrust-pc-sdk .ot-vlst-cntr{overflow:hidden}#onetrust-pc-sdk #ot-sel-blk{overflow:hidden;width:100%;position:sticky;position:-webkit-sticky;top:0;z-index:3}#onetrust-pc-sdk #ot-back-arw{height:12px;width:12px}#onetrust-pc-sdk .ot-lst-subhdr{width:100%;display:inline-block}#onetrust-pc-sdk .ot-search-cntr{float:left;width:78%;position:relative}#onetrust-pc-sdk .ot-search-cntr>svg{width:30px;height:30px;position:absolute;float:left;right:-15px}#onetrust-pc-sdk .ot-fltr-cntr{float:right;right:50px;position:relative}#onetrust-pc-sdk #filter-btn-handler{background-color:#3860be;border-radius:17px;display:inline-block;position:relative;width:32px;height:32px;-moz-transition:.1s ease;-o-transition:.1s ease;-webkit-transition:1s ease;transition:.1s ease;padding:0;margin:0}#onetrust-pc-sdk #filter-btn-handler:hover{background-color:#3860be}#onetrust-pc-sdk #filter-btn-handler svg{width:12px;height:12px;margin:3px 10px 0 10px;display:block;position:static;right:auto;top:auto}#onetrust-pc-sdk .ot-ven-link,#onetrust-pc-sdk .ot-ven-legclaim-link{color:#3860be;text-decoration:none;font-weight:100;display:inline-block;padding-top:10px;transform:translate(0, 1%);-o-transform:translate(0, 1%);-ms-transform:translate(0, 1%);-webkit-transform:translate(0, 1%);position:relative;z-index:2}#onetrust-pc-sdk .ot-ven-link *,#onetrust-pc-sdk .ot-ven-legclaim-link *{font-size:inherit}#onetrust-pc-sdk .ot-ven-link:hover,#onetrust-pc-sdk .ot-ven-legclaim-link:hover{text-decoration:underline}#onetrust-pc-sdk .ot-ven-hdr{width:calc(100% - 160px);height:auto;float:left;word-break:break-word;word-wrap:break-word;vertical-align:middle;padding-bottom:3px}#onetrust-pc-sdk .ot-ven-link,#onetrust-pc-sdk .ot-ven-legclaim-link{letter-spacing:.03em;font-size:.75em;font-weight:400}#onetrust-pc-sdk .ot-ven-dets{border-radius:2px;background-color:#f8f8f8}#onetrust-pc-sdk .ot-ven-dets li:first-child p:first-child{border-top:none}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:not(:first-child){border-top:1px solid #ddd !important}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:nth-child(n+3) p{display:inline-block}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:nth-child(n+3) p:nth-of-type(odd){width:30%}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:nth-child(n+3) p:nth-of-type(even){width:50%;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc p,#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc h5{padding-top:5px;padding-bottom:5px;display:block}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc h5{display:inline-block}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc p:nth-last-child(-n+1){padding-bottom:10px}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc p:nth-child(-n+2):not(.disc-pur){padding-top:10px}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc .disc-pur-cont{display:inline}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc .disc-pur{position:relative;width:50% !important;word-break:break-word;word-wrap:break-word;left:calc(30% + 17px)}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc .disc-pur:nth-child(-n+1){position:static}#onetrust-pc-sdk .ot-ven-dets p,#onetrust-pc-sdk .ot-ven-dets h5,#onetrust-pc-sdk .ot-ven-dets span{font-size:.69em;text-align:left;vertical-align:middle;word-break:break-word;word-wrap:break-word;margin:0;padding-bottom:10px;padding-left:15px;color:#2e3644}#onetrust-pc-sdk .ot-ven-dets h5{padding-top:5px}#onetrust-pc-sdk .ot-ven-dets span{color:dimgray;padding:0;vertical-align:baseline}#onetrust-pc-sdk .ot-ven-dets .ot-ven-pur h5{border-top:1px solid #e9e9e9;border-bottom:1px solid #e9e9e9;padding-bottom:5px;margin-bottom:5px;font-weight:bold}#onetrust-pc-sdk #ot-host-lst .ot-sel-all{float:right;position:relative;margin-right:42px;top:10px}#onetrust-pc-sdk #ot-host-lst .ot-sel-all input[type=checkbox]{width:auto;height:auto}#onetrust-pc-sdk #ot-host-lst .ot-sel-all label{height:20px;width:20px;padding-left:0px}#onetrust-pc-sdk #ot-host-lst .ot-acc-txt{overflow:hidden;width:95%}#onetrust-pc-sdk .ot-host-hdr{position:relative;z-index:1;pointer-events:none;width:calc(100% - 125px);float:left}#onetrust-pc-sdk .ot-host-name,#onetrust-pc-sdk .ot-host-desc{display:inline-block;width:90%}#onetrust-pc-sdk .ot-host-name{pointer-events:none}#onetrust-pc-sdk .ot-host-hdr>a{text-decoration:underline;font-size:.82em;position:relative;z-index:2;float:left;margin-bottom:5px;pointer-events:initial}#onetrust-pc-sdk .ot-host-name+a{margin-top:5px}#onetrust-pc-sdk .ot-host-name,#onetrust-pc-sdk .ot-host-name a,#onetrust-pc-sdk .ot-host-desc,#onetrust-pc-sdk .ot-host-info{color:dimgray;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-host-name,#onetrust-pc-sdk .ot-host-name a{font-weight:bold;font-size:.82em;line-height:1.3}#onetrust-pc-sdk .ot-host-name a{font-size:1em}#onetrust-pc-sdk .ot-host-expand{margin-top:3px;margin-bottom:3px;clear:both;display:block;color:#3860be;font-size:.72em;font-weight:normal}#onetrust-pc-sdk .ot-host-expand *{font-size:inherit}#onetrust-pc-sdk .ot-host-desc,#onetrust-pc-sdk .ot-host-info{font-size:.688em;line-height:1.4;font-weight:normal}#onetrust-pc-sdk .ot-host-desc{margin-top:10px}#onetrust-pc-sdk .ot-host-opt{margin:0;font-size:inherit;display:inline-block;width:100%}#onetrust-pc-sdk .ot-host-opt li>div div{font-size:.8em;padding:5px 0}#onetrust-pc-sdk .ot-host-opt li>div div:nth-child(1){width:30%;float:left}#onetrust-pc-sdk .ot-host-opt li>div div:nth-child(2){width:70%;float:left;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-host-info{border:none;display:inline-block;width:calc(100% - 10px);padding:10px;margin-bottom:10px;background-color:#f8f8f8}#onetrust-pc-sdk .ot-host-info>div{overflow:auto}#onetrust-pc-sdk #no-results{text-align:center;margin-top:30px}#onetrust-pc-sdk #no-results p{font-size:1em;color:#2e3644;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk #no-results p span{font-weight:bold}#onetrust-pc-sdk #ot-fltr-modal{width:100%;height:auto;display:none;-moz-transition:.2s ease;-o-transition:.2s ease;-webkit-transition:2s ease;transition:.2s ease;overflow:hidden;opacity:1;right:0}#onetrust-pc-sdk #ot-fltr-modal .ot-label-txt{display:inline-block;font-size:.85em;color:dimgray}#onetrust-pc-sdk #ot-fltr-cnt{z-index:2147483646;background-color:#fff;position:absolute;height:90%;max-height:300px;width:325px;left:210px;margin-top:10px;margin-bottom:20px;padding-right:10px;border-radius:3px;-webkit-box-shadow:0px 0px 12px 2px #c7c5c7;-moz-box-shadow:0px 0px 12px 2px #c7c5c7;box-shadow:0px 0px 12px 2px #c7c5c7}#onetrust-pc-sdk .ot-fltr-scrlcnt{overflow-y:auto;overflow-x:hidden;clear:both;max-height:calc(100% - 60px)}#onetrust-pc-sdk #ot-anchor{border:12px solid rgba(0,0,0,0);display:none;position:absolute;z-index:2147483647;right:55px;top:75px;transform:rotate(45deg);-o-transform:rotate(45deg);-ms-transform:rotate(45deg);-webkit-transform:rotate(45deg);background-color:#fff;-webkit-box-shadow:-3px -3px 5px -2px #c7c5c7;-moz-box-shadow:-3px -3px 5px -2px #c7c5c7;box-shadow:-3px -3px 5px -2px #c7c5c7}#onetrust-pc-sdk .ot-fltr-btns{margin-left:15px}#onetrust-pc-sdk #filter-apply-handler{margin-right:15px}#onetrust-pc-sdk .ot-fltr-opt{margin-bottom:5px;margin-left:15px;min-height:20px;width:75%;position:relative}#onetrust-pc-sdk .ot-fltr-opt p{display:inline-block;margin:0;font-size:.9em;color:#2e3644}#onetrust-pc-sdk .ot-chkbox label span{font-size:.85em;color:dimgray}#onetrust-pc-sdk .ot-chkbox input[type=checkbox]+label::after{content:none;color:#fff}#onetrust-pc-sdk .ot-chkbox input[type=checkbox]:checked+label::after{content:""}#onetrust-pc-sdk .ot-chkbox input[type=checkbox][aria-checked=mixed]+label::after{content:""}#onetrust-pc-sdk .ot-chkbox input[type=checkbox]:focus+label::before{outline-style:solid;outline-width:2px;outline-style:auto}#onetrust-pc-sdk #ot-selall-vencntr,#onetrust-pc-sdk #ot-selall-adtlvencntr,#onetrust-pc-sdk #ot-selall-hostcntr,#onetrust-pc-sdk #ot-selall-licntr,#onetrust-pc-sdk #ot-selall-gnvencntr{right:15px;position:relative;width:20px;height:20px;float:right}#onetrust-pc-sdk #ot-selall-vencntr label,#onetrust-pc-sdk #ot-selall-adtlvencntr label,#onetrust-pc-sdk #ot-selall-hostcntr label,#onetrust-pc-sdk #ot-selall-licntr label,#onetrust-pc-sdk #ot-selall-gnvencntr label{float:left;padding-left:0}#onetrust-pc-sdk #ot-ven-lst:first-child{border-top:1px solid #e2e2e2}#onetrust-pc-sdk ul{list-style:none;padding:0}#onetrust-pc-sdk ul li{position:relative;margin:0;padding:15px 15px 15px 10px;border-bottom:1px solid #e2e2e2}#onetrust-pc-sdk ul li h3,#onetrust-pc-sdk ul li h4{font-size:.75em;color:#656565;margin:0;display:inline-block;width:70%;height:auto;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk ul li p{margin:0;font-size:.7em}#onetrust-pc-sdk ul li input[type=checkbox]{position:absolute;cursor:pointer;width:100%;height:100%;opacity:0;margin:0;top:0;left:0}#onetrust-pc-sdk .ot-cat-item>button:focus,#onetrust-pc-sdk .ot-acc-cntr>button:focus,#onetrust-pc-sdk li>button:focus{outline:#000 solid 2px}#onetrust-pc-sdk .ot-cat-item>button,#onetrust-pc-sdk .ot-acc-cntr>button,#onetrust-pc-sdk li>button{position:absolute;cursor:pointer;width:100%;height:100%;margin:0;top:0;left:0;z-index:1;max-width:none;border:none}#onetrust-pc-sdk .ot-cat-item>button[aria-expanded=false]~.ot-acc-txt,#onetrust-pc-sdk .ot-acc-cntr>button[aria-expanded=false]~.ot-acc-txt,#onetrust-pc-sdk li>button[aria-expanded=false]~.ot-acc-txt{margin-top:0;max-height:0;opacity:0;overflow:hidden;width:100%;transition:.25s ease-out;display:none}#onetrust-pc-sdk .ot-cat-item>button[aria-expanded=true]~.ot-acc-txt,#onetrust-pc-sdk .ot-acc-cntr>button[aria-expanded=true]~.ot-acc-txt,#onetrust-pc-sdk li>button[aria-expanded=true]~.ot-acc-txt{transition:.1s ease-in;margin-top:10px;width:100%;overflow:auto;display:block}#onetrust-pc-sdk .ot-cat-item>button[aria-expanded=true]~.ot-acc-grpcntr,#onetrust-pc-sdk .ot-acc-cntr>button[aria-expanded=true]~.ot-acc-grpcntr,#onetrust-pc-sdk li>button[aria-expanded=true]~.ot-acc-grpcntr{width:auto;margin-top:0px;padding-bottom:10px}#onetrust-pc-sdk .ot-host-item>button:focus,#onetrust-pc-sdk .ot-ven-item>button:focus{outline:0;border:2px solid #000}#onetrust-pc-sdk .ot-hide-acc>button{pointer-events:none}#onetrust-pc-sdk .ot-hide-acc .ot-plus-minus>*,#onetrust-pc-sdk .ot-hide-acc .ot-arw-cntr>*{visibility:hidden}#onetrust-pc-sdk .ot-hide-acc .ot-acc-hdr{min-height:30px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt){padding-right:10px;width:calc(100% - 37px);margin-top:10px;max-height:calc(100% - 90px)}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk{background-color:#f9f9fc;border:1px solid #e2e2e2;width:calc(100% - 2px);padding-bottom:5px;padding-top:5px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt{border:unset;background-color:unset}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt .ot-sel-all-hdr{display:none}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt .ot-sel-all{padding-right:.5rem}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt .ot-sel-all .ot-chkbox{right:0}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) .ot-sel-all{padding-right:34px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) .ot-sel-all-chkbox{width:auto}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) ul li{border:1px solid #e2e2e2;margin-bottom:10px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) .ot-acc-cntr>.ot-acc-hdr{padding:10px 0 10px 15px}#onetrust-pc-sdk.ot-addtl-vendors .ot-sel-all-chkbox{float:right}#onetrust-pc-sdk.ot-addtl-vendors .ot-plus-minus~.ot-sel-all-chkbox{right:34px}#onetrust-pc-sdk.ot-addtl-vendors #ot-ven-lst:first-child{border-top:none}#onetrust-pc-sdk .ot-acc-cntr{position:relative;border-left:1px solid #e2e2e2;border-right:1px solid #e2e2e2;border-bottom:1px solid #e2e2e2}#onetrust-pc-sdk .ot-acc-cntr input{z-index:1}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr{background-color:#f9f9fc;padding:5px 0 5px 15px;width:auto}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr .ot-plus-minus{vertical-align:middle;top:auto}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr .ot-arw-cntr{right:10px}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr input{z-index:2}#onetrust-pc-sdk .ot-acc-cntr.ot-add-tech .ot-acc-hdr{padding:10px 0 10px 15px}#onetrust-pc-sdk .ot-acc-cntr>input[type=checkbox]:checked~.ot-acc-hdr{border-bottom:1px solid #e2e2e2}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-txt{padding-left:10px;padding-right:10px}#onetrust-pc-sdk .ot-acc-cntr button[aria-expanded=true]~.ot-acc-txt{width:auto}#onetrust-pc-sdk .ot-acc-cntr .ot-addtl-venbox{display:none}#onetrust-pc-sdk .ot-vlst-cntr{margin-bottom:0;width:100%}#onetrust-pc-sdk .ot-vensec-title{font-size:.813em;vertical-align:middle;display:inline-block}#onetrust-pc-sdk .category-vendors-list-handler,#onetrust-pc-sdk .category-vendors-list-handler+a{margin-left:0;margin-top:10px}#onetrust-pc-sdk #ot-selall-vencntr.line-through label::after,#onetrust-pc-sdk #ot-selall-adtlvencntr.line-through label::after,#onetrust-pc-sdk #ot-selall-licntr.line-through label::after,#onetrust-pc-sdk #ot-selall-hostcntr.line-through label::after,#onetrust-pc-sdk #ot-selall-gnvencntr.line-through label::after{height:auto;border-left:0;transform:none;-o-transform:none;-ms-transform:none;-webkit-transform:none;left:5px;top:9px}#onetrust-pc-sdk #ot-category-title{float:left;padding-bottom:10px;font-size:1em;width:100%}#onetrust-pc-sdk .ot-cat-grp{margin-top:10px}#onetrust-pc-sdk .ot-cat-item{line-height:1.1;margin-top:10px;display:inline-block;width:100%}#onetrust-pc-sdk .ot-btn-container{text-align:right}#onetrust-pc-sdk .ot-btn-container button{display:inline-block;font-size:.75em;letter-spacing:.08em;margin-top:19px}#onetrust-pc-sdk #close-pc-btn-handler.ot-close-icon{position:absolute;top:10px;right:0;z-index:1;padding:0;background-color:rgba(0,0,0,0);border:none}#onetrust-pc-sdk #close-pc-btn-handler.ot-close-icon svg{display:block;height:10px;width:10px}#onetrust-pc-sdk #clear-filters-handler{margin-top:20px;margin-bottom:10px;float:right;max-width:200px;text-decoration:none;color:#3860be;font-size:.9em;font-weight:bold;background-color:rgba(0,0,0,0);border-color:rgba(0,0,0,0);padding:1px}#onetrust-pc-sdk #clear-filters-handler:hover{color:#2285f7}#onetrust-pc-sdk #clear-filters-handler:focus{outline:#000 solid 1px}#onetrust-pc-sdk .ot-enbl-chr h4~.ot-tgl,#onetrust-pc-sdk .ot-enbl-chr h4~.ot-always-active{right:45px}#onetrust-pc-sdk .ot-enbl-chr h4~.ot-tgl+.ot-tgl{right:120px}#onetrust-pc-sdk .ot-enbl-chr .ot-pli-hdr.ot-leg-border-color span:first-child{width:90px}#onetrust-pc-sdk .ot-enbl-chr li.ot-subgrp>h5+.ot-tgl-cntr{padding-right:25px}#onetrust-pc-sdk .ot-plus-minus{width:20px;height:20px;font-size:1.5em;position:relative;display:inline-block;margin-right:5px;top:3px}#onetrust-pc-sdk .ot-plus-minus span{position:absolute;background:#27455c;border-radius:1px}#onetrust-pc-sdk .ot-plus-minus span:first-of-type{top:25%;bottom:25%;width:10%;left:45%}#onetrust-pc-sdk .ot-plus-minus span:last-of-type{left:25%;right:25%;height:10%;top:45%}#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-arw,#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-plus-minus span:first-of-type,#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-plus-minus span:last-of-type{transform:rotate(90deg)}#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-plus-minus span:last-of-type{left:50%;right:50%}#onetrust-pc-sdk #ot-selall-vencntr label,#onetrust-pc-sdk #ot-selall-adtlvencntr label,#onetrust-pc-sdk #ot-selall-hostcntr label,#onetrust-pc-sdk #ot-selall-licntr label{position:relative;display:inline-block;width:20px;height:20px}#onetrust-pc-sdk .ot-host-item .ot-plus-minus,#onetrust-pc-sdk .ot-ven-item .ot-plus-minus{float:left;margin-right:8px;top:10px}#onetrust-pc-sdk .ot-ven-item ul{list-style:none inside;font-size:100%;margin:0}#onetrust-pc-sdk .ot-ven-item ul li{margin:0 !important;padding:0;border:none !important}#onetrust-pc-sdk .ot-pli-hdr{color:#77808e;overflow:hidden;padding-top:7.5px;padding-bottom:7.5px;width:calc(100% - 2px);border-top-left-radius:3px;border-top-right-radius:3px}#onetrust-pc-sdk .ot-pli-hdr span:first-child{top:50%;transform:translateY(50%);max-width:90px}#onetrust-pc-sdk .ot-pli-hdr span:last-child{padding-right:10px;max-width:95px;text-align:center}#onetrust-pc-sdk .ot-li-title{float:right;font-size:.813em}#onetrust-pc-sdk .ot-pli-hdr.ot-leg-border-color{background-color:#f4f4f4;border:1px solid #d8d8d8}#onetrust-pc-sdk .ot-pli-hdr.ot-leg-border-color span:first-child{text-align:left;width:70px}#onetrust-pc-sdk li.ot-subgrp>h5,#onetrust-pc-sdk .ot-cat-header{width:calc(100% - 130px)}#onetrust-pc-sdk li.ot-subgrp>h5+.ot-tgl-cntr{padding-left:13px}#onetrust-pc-sdk .ot-acc-grpcntr .ot-acc-grpdesc{margin-bottom:5px}#onetrust-pc-sdk .ot-acc-grpcntr .ot-subgrp-cntr{border-top:1px solid #d8d8d8}#onetrust-pc-sdk .ot-acc-grpcntr .ot-vlst-cntr+.ot-subgrp-cntr{border-top:none}#onetrust-pc-sdk .ot-acc-hdr .ot-arw-cntr+.ot-tgl-cntr,#onetrust-pc-sdk .ot-acc-txt h4+.ot-tgl-cntr{padding-left:13px}#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item .ot-subgrp>h5,#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item .ot-cat-header{width:calc(100% - 145px)}#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item h5+.ot-tgl-cntr,#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item .ot-cat-header+.ot-tgl{padding-left:28px}#onetrust-pc-sdk .ot-sel-all-hdr,#onetrust-pc-sdk .ot-sel-all-chkbox{display:inline-block;width:100%;position:relative}#onetrust-pc-sdk .ot-sel-all-chkbox{z-index:1}#onetrust-pc-sdk .ot-sel-all{margin:0;position:relative;padding-right:23px;float:right}#onetrust-pc-sdk .ot-consent-hdr,#onetrust-pc-sdk .ot-li-hdr{float:right;font-size:.812em;line-height:normal;text-align:center;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-li-hdr{max-width:100px;padding-right:10px}#onetrust-pc-sdk .ot-consent-hdr{max-width:55px}#onetrust-pc-sdk #ot-selall-licntr{display:block;width:21px;height:auto;float:right;position:relative;right:80px}#onetrust-pc-sdk #ot-selall-licntr label{position:absolute}#onetrust-pc-sdk .ot-ven-ctgl{margin-left:66px}#onetrust-pc-sdk .ot-ven-litgl+.ot-arw-cntr{margin-left:81px}#onetrust-pc-sdk .ot-enbl-chr .ot-host-cnt .ot-tgl-cntr{width:auto}#onetrust-pc-sdk #ot-lst-cnt:not(.ot-host-cnt) .ot-tgl-cntr{width:auto;top:auto;height:20px}#onetrust-pc-sdk #ot-lst-cnt .ot-chkbox{position:relative;display:inline-block;width:20px;height:20px}#onetrust-pc-sdk #ot-lst-cnt .ot-chkbox label{position:absolute;padding:0;width:20px;height:20px}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info-cntr{border:1px solid #d8d8d8;padding:.75rem 2rem;padding-bottom:0;width:auto;margin-top:.5rem}#onetrust-pc-sdk .ot-acc-grpdesc+.ot-leg-btn-container{padding-left:20px;padding-right:20px;width:calc(100% - 40px);margin-bottom:5px}#onetrust-pc-sdk .ot-subgrp .ot-leg-btn-container{margin-bottom:5px}#onetrust-pc-sdk #ot-ven-lst .ot-leg-btn-container{margin-top:10px}#onetrust-pc-sdk .ot-leg-btn-container{display:inline-block;width:100%;margin-bottom:10px}#onetrust-pc-sdk .ot-leg-btn-container button{height:auto;padding:6.5px 8px;margin-bottom:0;letter-spacing:0;font-size:.75em;line-height:normal}#onetrust-pc-sdk .ot-leg-btn-container svg{display:none;height:14px;width:14px;padding-right:5px;vertical-align:sub}#onetrust-pc-sdk .ot-active-leg-btn{cursor:default;pointer-events:none}#onetrust-pc-sdk .ot-active-leg-btn svg{display:inline-block}#onetrust-pc-sdk .ot-remove-objection-handler{text-decoration:underline;padding:0;font-size:.75em;font-weight:600;line-height:1;padding-left:10px}#onetrust-pc-sdk .ot-obj-leg-btn-handler span{font-weight:bold;text-align:center;font-size:inherit;line-height:1.5}#onetrust-pc-sdk.ot-close-btn-link #close-pc-btn-handler{border:none;height:auto;line-height:1.5;text-decoration:underline;font-size:.69em;background:none;right:15px;top:15px;width:auto;font-weight:normal}#onetrust-pc-sdk .ot-pgph-link{font-size:.813em !important;margin-top:5px;position:relative}#onetrust-pc-sdk .ot-pgph-link.ot-pgph-link-subgroup{margin-bottom:1rem}#onetrust-pc-sdk .ot-pgph-contr{margin:0 2.5rem}#onetrust-pc-sdk .ot-pgph-title{font-size:1.18rem;margin-bottom:2rem}#onetrust-pc-sdk .ot-pgph-desc{font-size:1rem;font-weight:400;margin-bottom:2rem;line-height:1.5rem}#onetrust-pc-sdk .ot-pgph-desc:not(:last-child):after{content:"";width:96%;display:block;margin:0 auto;padding-bottom:2rem;border-bottom:1px solid #e9e9e9}#onetrust-pc-sdk .ot-cat-header{float:left;font-weight:600;font-size:.875em;line-height:1.5;max-width:90%;vertical-align:middle}#onetrust-pc-sdk .ot-vnd-item>button:focus{outline:#000 solid 2px}#onetrust-pc-sdk .ot-vnd-item>button{position:absolute;cursor:pointer;width:100%;height:100%;margin:0;top:0;left:0;z-index:1;max-width:none;border:none}#onetrust-pc-sdk .ot-vnd-item>button[aria-expanded=false]~.ot-acc-txt{margin-top:0;max-height:0;opacity:0;overflow:hidden;width:100%;transition:.25s ease-out;display:none}#onetrust-pc-sdk .ot-vnd-item>button[aria-expanded=true]~.ot-acc-txt{transition:.1s ease-in;margin-top:10px;width:100%;overflow:auto;display:block}#onetrust-pc-sdk .ot-vnd-item>button[aria-expanded=true]~.ot-acc-grpcntr{width:auto;margin-top:0px;padding-bottom:10px}#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item{position:relative;border-radius:2px;margin:0;padding:0;border:1px solid #d8d8d8;border-top:none;width:calc(100% - 2px);float:left}#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item:first-of-type{margin-top:10px;border-top:1px solid #d8d8d8}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpdesc{padding-left:20px;padding-right:20px;width:calc(100% - 40px);font-size:.812em;margin-bottom:10px;margin-top:15px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpdesc>ul{padding-top:10px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpdesc>ul li{padding-top:0;line-height:1.5;padding-bottom:10px}#onetrust-pc-sdk .ot-accordion-layout div+.ot-acc-grpdesc{margin-top:5px}#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr:first-child{margin-top:10px}#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr:last-child,#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr:last-child{margin-bottom:5px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{padding-top:11.5px;padding-bottom:11.5px;padding-left:20px;padding-right:20px;width:calc(100% - 40px);display:inline-block}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-txt{width:100%;padding:0}#onetrust-pc-sdk .ot-accordion-layout .ot-subgrp-cntr{padding-left:20px;padding-right:15px;padding-bottom:0;width:calc(100% - 35px)}#onetrust-pc-sdk .ot-accordion-layout .ot-subgrp{padding-right:5px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpcntr{z-index:1;position:relative}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header+.ot-arw-cntr{position:absolute;top:50%;transform:translateY(-50%);right:20px;margin-top:-2px}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header+.ot-arw-cntr .ot-arw{width:15px;height:20px;margin-left:5px;color:dimgray}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{float:none;color:#2e3644;margin:0;display:inline-block;height:auto;word-wrap:break-word;min-height:inherit}#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr,#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr{padding-left:20px;width:calc(100% - 20px);display:inline-block;margin-top:0;padding-bottom:2px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{position:relative;min-height:25px}#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl,#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active{position:absolute;top:50%;transform:translateY(-50%);right:20px}#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl+.ot-tgl{right:95px}#onetrust-pc-sdk .ot-accordion-layout .category-vendors-list-handler,#onetrust-pc-sdk .ot-accordion-layout .category-vendors-list-handler+a{margin-top:5px}#onetrust-pc-sdk #ot-lst-cnt{margin-top:1rem;max-height:calc(100% - 96px)}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info-cntr{border:1px solid #d8d8d8;padding:.75rem 2rem;padding-bottom:0;width:auto;margin-top:.5rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info{margin-bottom:1rem;padding-left:.75rem;padding-right:.75rem;display:flex;flex-direction:column}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info[data-vnd-info-key*=DPOEmail]{border-top:1px solid #d8d8d8;padding-top:1rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info[data-vnd-info-key*=DPOLink]{border-bottom:1px solid #d8d8d8;padding-bottom:1rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info .ot-vnd-lbl{font-weight:bold;font-size:.85em;margin-bottom:.5rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info .ot-vnd-cnt{margin-left:.5rem;font-weight:500;font-size:.85rem}#onetrust-pc-sdk .ot-vs-list,#onetrust-pc-sdk .ot-vnd-serv{width:auto;padding:1rem 1.25rem;padding-bottom:0}#onetrust-pc-sdk .ot-vs-list .ot-vnd-serv-hdr-cntr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-serv-hdr-cntr{padding-bottom:.75rem;border-bottom:1px solid #d8d8d8}#onetrust-pc-sdk .ot-vs-list .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr{font-weight:600;font-size:.95em;line-height:2;margin-left:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item{border:none;margin:0;padding:0}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item button,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item button{outline:none;border-bottom:1px solid #d8d8d8}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item button[aria-expanded=true],#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item button[aria-expanded=true]{border-bottom:none}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item:first-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item:first-child{margin-top:.25rem;border-top:unset}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item:last-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item:last-child{margin-bottom:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item:last-child button,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item:last-child button{border-bottom:none}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info-cntr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info-cntr{border:1px solid #d8d8d8;padding:.75rem 1.75rem;padding-bottom:0;width:auto;margin-top:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info{margin-bottom:1rem;padding-left:.75rem;padding-right:.75rem;display:flex;flex-direction:column}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOEmail],#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOEmail]{border-top:1px solid #d8d8d8;padding-top:1rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOLink],#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOLink]{border-bottom:1px solid #d8d8d8;padding-bottom:1rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info .ot-vnd-lbl,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info .ot-vnd-lbl{font-weight:bold;font-size:.85em;margin-bottom:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info .ot-vnd-cnt,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info .ot-vnd-cnt{margin-left:.5rem;font-weight:500;font-size:.85rem}#onetrust-pc-sdk .ot-vs-list.ot-vnd-subgrp-cnt,#onetrust-pc-sdk .ot-vnd-serv.ot-vnd-subgrp-cnt{padding-left:40px}#onetrust-pc-sdk .ot-vs-list.ot-vnd-subgrp-cnt .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr,#onetrust-pc-sdk .ot-vnd-serv.ot-vnd-subgrp-cnt .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr{font-size:.8em}#onetrust-pc-sdk .ot-vs-list.ot-vnd-subgrp-cnt .ot-cat-header,#onetrust-pc-sdk .ot-vnd-serv.ot-vnd-subgrp-cnt .ot-cat-header{font-size:.8em}#onetrust-pc-sdk .ot-subgrp-cntr .ot-vnd-serv{margin-bottom:1rem;padding:1rem .95rem}#onetrust-pc-sdk .ot-subgrp-cntr .ot-vnd-serv .ot-vnd-serv-hdr-cntr{padding-bottom:.75rem;border-bottom:1px solid #d8d8d8}#onetrust-pc-sdk .ot-subgrp-cntr .ot-vnd-serv .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr{font-weight:700;font-size:.8em;line-height:20px;margin-left:.82rem}#onetrust-pc-sdk .ot-subgrp-cntr .ot-cat-header{font-weight:700;font-size:.8em;line-height:20px}#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-vnd-serv .ot-vnd-lst-cont .ot-accordion-layout .ot-acc-hdr div.ot-chkbox{margin-left:.82rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr{padding:.7rem 0;margin:0;display:flex;width:100%;align-items:center;justify-content:space-between}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr div:first-child,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr div:first-child,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr div:first-child{margin-left:.5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr div:last-child,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr div:last-child,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr div:last-child{margin-right:.5rem;margin-left:.5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-always-active{position:relative;right:unset;top:unset;transform:unset}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-plus-minus{top:0}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-arw-cntr{float:none;top:unset;right:unset;transform:unset;margin-top:-2px;position:relative}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-cat-header{flex:1;margin:0 .5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-tgl{position:relative;transform:none;right:0;top:0;float:none}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox{position:relative;margin:0 .5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox label{padding:0}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox label::before{position:relative}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox input{position:absolute;cursor:pointer;width:100%;height:100%;opacity:0;margin:0;top:0;left:0;z-index:1}#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp .ot-acc-hdr h5.ot-cat-header,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp .ot-acc-hdr h4.ot-cat-header{margin:0}#onetrust-pc-sdk .ot-vs-config .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp h5{top:0;line-height:20px}#onetrust-pc-sdk .ot-vs-list{display:flex;flex-direction:column;padding:0;margin:.5rem 4px}#onetrust-pc-sdk .ot-vs-selc-all{display:flex;padding:0;float:unset;align-items:center;justify-content:flex-start}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf{justify-content:flex-end}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf.ot-caret-conf .ot-sel-all-chkbox{margin-right:48px}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf .ot-sel-all-chkbox{margin:0;padding:0;margin-right:14px;justify-content:flex-end}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf #ot-selall-vencntr.ot-chkbox,#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf #ot-selall-vencntr.ot-tgl{display:inline-block;right:unset;width:auto;height:auto;float:none}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf #ot-selall-vencntr label{width:45px;height:25px}#onetrust-pc-sdk .ot-vs-selc-all .ot-sel-all-chkbox{margin-right:11px;margin-left:.75rem;display:flex;align-items:center}#onetrust-pc-sdk .ot-vs-selc-all .sel-all-hdr{margin:0 1.25rem;font-size:.812em;line-height:normal;text-align:center;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-vnd-list-cnt #ot-selall-vencntr.ot-chkbox{float:unset;right:0}#onetrust-pc-sdk[dir=rtl] #ot-back-arw,#onetrust-pc-sdk[dir=rtl] input~.ot-acc-hdr .ot-arw{transform:rotate(180deg);-o-transform:rotate(180deg);-ms-transform:rotate(180deg);-webkit-transform:rotate(180deg)}#onetrust-pc-sdk[dir=rtl] input:checked~.ot-acc-hdr .ot-arw{transform:rotate(270deg);-o-transform:rotate(270deg);-ms-transform:rotate(270deg);-webkit-transform:rotate(270deg)}#onetrust-pc-sdk[dir=rtl] .ot-chkbox label::after{transform:rotate(45deg);-webkit-transform:rotate(45deg);-o-transform:rotate(45deg);-ms-transform:rotate(45deg);border-left:0;border-right:3px solid}#onetrust-pc-sdk[dir=rtl] .ot-search-cntr>svg{right:0}@media only screen and (max-width: 600px){#onetrust-pc-sdk.otPcCenter{left:0;min-width:100%;height:100%;top:0;border-radius:0}#onetrust-pc-sdk #ot-pc-content,#onetrust-pc-sdk.ot-ftr-stacked .ot-btn-container{margin:1px 3px 0 10px;padding-right:10px;width:calc(100% - 23px)}#onetrust-pc-sdk .ot-btn-container button{max-width:none;letter-spacing:.01em}#onetrust-pc-sdk #close-pc-btn-handler{top:10px;right:17px}#onetrust-pc-sdk p{font-size:.7em}#onetrust-pc-sdk #ot-pc-hdr{margin:10px 10px 0 5px;width:calc(100% - 15px)}#onetrust-pc-sdk .vendor-search-handler{font-size:1em}#onetrust-pc-sdk #ot-back-arw{margin-left:12px}#onetrust-pc-sdk #ot-lst-cnt{margin:0;padding:0 5px 0 10px;min-width:95%}#onetrust-pc-sdk .switch+p{max-width:80%}#onetrust-pc-sdk .ot-ftr-stacked button{width:100%}#onetrust-pc-sdk #ot-fltr-cnt{max-width:320px;width:90%;border-top-right-radius:0;border-bottom-right-radius:0;margin:0;margin-left:15px;left:auto;right:40px;top:85px}#onetrust-pc-sdk .ot-fltr-opt{margin-left:25px;margin-bottom:10px}#onetrust-pc-sdk .ot-pc-refuse-all-handler{margin-bottom:0}#onetrust-pc-sdk #ot-fltr-cnt{right:40px}}@media only screen and (max-width: 476px){#onetrust-pc-sdk .ot-fltr-cntr,#onetrust-pc-sdk #ot-fltr-cnt{right:10px}#onetrust-pc-sdk #ot-anchor{right:25px}#onetrust-pc-sdk button{width:100%}#onetrust-pc-sdk:not(.ot-addtl-vendors) #ot-pc-lst:not(.ot-enbl-chr) .ot-sel-all{padding-right:9px}#onetrust-pc-sdk:not(.ot-addtl-vendors) #ot-pc-lst:not(.ot-enbl-chr) .ot-tgl-cntr{right:0}}@media(min-width: 768px){#onetrust-pc-sdk.ot-tgl-with-label .ot-label-status{display:inline}#onetrust-pc-sdk.ot-tgl-with-label #ot-pc-lst .ot-label-status{display:none}}@media only screen and (max-width: 896px)and (max-height: 425px)and (orientation: landscape){#onetrust-pc-sdk.otPcCenter{left:0;top:0;min-width:100%;height:100%;border-radius:0}#onetrust-pc-sdk .ot-pc-header{height:auto;min-height:20px}#onetrust-pc-sdk .ot-pc-header .ot-pc-logo{max-height:30px}#onetrust-pc-sdk .ot-pc-footer{max-height:60px;overflow-y:auto}#onetrust-pc-sdk #ot-pc-content,#onetrust-pc-sdk #ot-pc-lst{bottom:70px}#onetrust-pc-sdk.ot-ftr-stacked #ot-pc-content{bottom:70px}#onetrust-pc-sdk #ot-anchor{left:initial;right:50px}#onetrust-pc-sdk #ot-lst-title{margin-top:12px}#onetrust-pc-sdk #ot-lst-title *{font-size:inherit}#onetrust-pc-sdk #ot-pc-hdr input{margin-right:0;padding-right:45px}#onetrust-pc-sdk .switch+p{max-width:85%}#onetrust-pc-sdk #ot-sel-blk{position:static}#onetrust-pc-sdk #ot-pc-lst{overflow:auto}#onetrust-pc-sdk #ot-lst-cnt{max-height:none;overflow:initial}#onetrust-pc-sdk #ot-lst-cnt.no-results{height:auto}#onetrust-pc-sdk input{font-size:1em !important}#onetrust-pc-sdk p{font-size:.6em}#onetrust-pc-sdk #ot-fltr-modal{width:100%;top:0}#onetrust-pc-sdk ul li p,#onetrust-pc-sdk .category-vendors-list-handler,#onetrust-pc-sdk .category-vendors-list-handler+a,#onetrust-pc-sdk .category-host-list-handler{font-size:.6em}#onetrust-pc-sdk.ot-shw-fltr #ot-anchor{display:none !important}#onetrust-pc-sdk.ot-shw-fltr #ot-pc-lst{height:100% !important;overflow:hidden;top:0px}#onetrust-pc-sdk.ot-shw-fltr #ot-fltr-cnt{margin:0;height:100%;max-height:none;padding:10px;top:0;width:calc(100% - 20px);position:absolute;right:0;left:0;max-width:none}#onetrust-pc-sdk.ot-shw-fltr .ot-fltr-scrlcnt{max-height:calc(100% - 65px)}}
            #onetrust-consent-sdk #onetrust-pc-sdk,
                #onetrust-consent-sdk #ot-search-cntr,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-switch.ot-toggle,
                #onetrust-consent-sdk #onetrust-pc-sdk ot-grp-hdr1 .checkbox,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-title:after
                ,#onetrust-consent-sdk #onetrust-pc-sdk #ot-sel-blk,
                        #onetrust-consent-sdk #onetrust-pc-sdk #ot-fltr-cnt,
                        #onetrust-consent-sdk #onetrust-pc-sdk #ot-anchor {
                    background-color: #FFF;
                }
               
            #onetrust-consent-sdk #onetrust-pc-sdk h3,
                #onetrust-consent-sdk #onetrust-pc-sdk h4,
                #onetrust-consent-sdk #onetrust-pc-sdk h5,
                #onetrust-consent-sdk #onetrust-pc-sdk h6,
                #onetrust-consent-sdk #onetrust-pc-sdk p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-ven-lst .ot-ven-opts p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-desc,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-title,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-li-title,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-sel-all-hdr span,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-info,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-fltr-modal #modal-header,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-checkbox label span,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst #ot-sel-blk p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst #ot-lst-title h3,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst .back-btn-handler p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst .ot-ven-name,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst #ot-ven-lst .consent-category,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-inactive-leg-btn,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-label-status,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-chkbox label span,
                #onetrust-consent-sdk #onetrust-pc-sdk #clear-filters-handler,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-optout-signal
                {
                    color: #2E2E2E;
                }
             #onetrust-consent-sdk #onetrust-pc-sdk .privacy-notice-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-pgph-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk .category-vendors-list-handler,
                    #onetrust-consent-sdk #onetrust-pc-sdk .category-vendors-list-handler + a,
                    #onetrust-consent-sdk #onetrust-pc-sdk .category-host-list-handler,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-ven-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-ven-legclaim-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-name a,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-acc-hdr .ot-host-expand,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-info a,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-content #ot-pc-desc .ot-link-btn,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info a,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info a
                    {
                        color: #007398;
                    }
            #onetrust-consent-sdk #onetrust-pc-sdk .category-vendors-list-handler:hover { text-decoration: underline;}
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-grpcntr.ot-acc-txt,
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-txt .ot-subgrp-tgl .ot-switch.ot-toggle
             {
                background-color: #F8F8F8;
            }
             #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-info,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-txt .ot-ven-dets
                            {
                                background-color: #F8F8F8;
                            }
        #onetrust-consent-sdk #onetrust-pc-sdk
            button:not(#clear-filters-handler):not(.ot-close-icon):not(#filter-btn-handler):not(.ot-remove-objection-handler):not(.ot-obj-leg-btn-handler):not([aria-expanded]):not(.ot-link-btn),
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-active-leg-btn {
                background-color: #007398;border-color: #007398;
                color: #FFF;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-active-menu {
                border-color: #007398;
            }
            
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-remove-objection-handler{
                background-color: transparent;
                border: 1px solid transparent;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-inactive-leg-btn {
                background-color: #FFFFFF;
                color: #4D4D4D; border-color: #4D4D4D;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-tgl input:focus + .ot-switch, .ot-switch .ot-switch-nob, .ot-switch .ot-switch-nob:before,
            #onetrust-pc-sdk .ot-checkbox input[type="checkbox"]:focus + label::before,
            #onetrust-pc-sdk .ot-chkbox input[type="checkbox"]:focus + label::before {
                outline-color: #000000;
                outline-width: 1px;
                outline-offset: 1px;
            }
            #onetrust-pc-sdk .ot-host-item > button:focus, #onetrust-pc-sdk .ot-ven-item > button:focus {
                border: 1px solid #000000;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk *:focus,
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-vlst-cntr > a:focus {
               outline: 1px solid #000000;
               outline-offset: 1px;
            }#onetrust-pc-sdk .ot-vlst-cntr .ot-ext-lnk,  #onetrust-pc-sdk .ot-ven-hdr .ot-ext-lnk{
                    background-image: url('https://cdn.cookielaw.org/logos/static/ot_external_link.svg');
                }
            /*! v6.10.0 2021-01-18 */
div#onetrust-consent-sdk #onetrust-banner-sdk{border-top:2px solid #d0d3d4!important;box-shadow:none;padding:24px}div#onetrust-consent-sdk button{border-radius:0!important;box-shadow:none!important;box-sizing:border-box!important;font-size:20px!important;font-weight:400!important;letter-spacing:0!important;max-width:none!important;white-space:nowrap!important}div#onetrust-consent-sdk button:not(.ot-link-btn){background-color:#007dbc!important;border:2px solid #007dbc!important;color:#fff!important;height:48px!important;padding:0 1em!important;width:auto!important}div#onetrust-consent-sdk button:hover{background-color:#fff!important;border-color:#949494!important;color:#2e2e2e!important}div#onetrust-consent-sdk button.ot-link-btn{color:#007dbc!important;font-size:16px!important;text-decoration:underline}div#onetrust-consent-sdk button.ot-link-btn:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-consent-sdk a,div#onetrust-pc-sdk a{color:#007dbc!important;text-decoration:underline!important}div#onetrust-consent-sdk a,div#onetrust-consent-sdk button,div#onetrust-consent-sdk p:hover{opacity:1!important}div#onetrust-consent-sdk a:focus,div#onetrust-consent-sdk button:focus,div#onetrust-consent-sdk input:focus{outline:2px solid #949494!important;outline-offset:1px!important}div#onetrust-banner-sdk .ot-sdk-container{padding:0;width:auto}div#onetrust-banner-sdk .ot-sdk-row{align-items:flex-start;box-sizing:border-box;display:flex;flex-direction:column;justify-content:space-between;margin:auto;max-width:1152px}div#onetrust-banner-sdk .ot-sdk-row:after{display:none}div#onetrust-banner-sdk #onetrust-group-container,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-group-container,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{flex-grow:1;width:auto}div#onetrust-banner-sdk #onetrust-policy,div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy{margin:0;overflow:visible}div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy-text,div#onetrust-consent-sdk #onetrust-policy-text{font-size:16px;line-height:24px;max-width:44em;margin:0}div#onetrust-consent-sdk #onetrust-policy-text a[href]{font-weight:400;margin-left:8px}div#onetrust-banner-sdk #onetrust-button-group-parent{flex:0 0 auto;margin:32px 0 0;width:100%}div#onetrust-banner-sdk #onetrust-button-group{display:flex;flex-direction:row;flex-wrap:wrap;justify-content:flex-end;margin:-8px}div#onetrust-banner-sdk .banner-actions-container{display:flex;flex:1 0 auto}div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button:last-of-type,div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto;margin:8px;width:auto}div#onetrust-consent-sdk #onetrust-pc-btn-handler{background-color:#fff!important;color:inherit!important}div#onetrust-banner-sdk #onetrust-close-btn-container{display:none}@media only screen and (min-width:556px){div#onetrust-consent-sdk #onetrust-banner-sdk{padding:40px}div#onetrust-banner-sdk #onetrust-policy{margin:0 40px 0 0}div#onetrust-banner-sdk .ot-sdk-row{align-items:center;flex-direction:row}div#onetrust-banner-sdk #onetrust-button-group-parent,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-button-group-parent,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{margin:0;padding:0;width:auto}div#onetrust-banner-sdk #onetrust-button-group,div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group{align-items:stretch;flex-direction:column-reverse;margin:0}div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto}}@media only screen and (min-width:768px){div#onetrust-banner-sdk #onetrust-policy{margin:0 48px 0 0}div#onetrust-consent-sdk #onetrust-banner-sdk{padding:48px}}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-size:16px;line-height:24px}div#onetrust-consent-sdk #onetrust-pc-sdk p,div#onetrust-pc-sdk #ot-pc-desc,div#onetrust-pc-sdk .category-host-list-handler,div#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{font-size:16px;font-weight:400;line-height:24px}div#onetrust-consent-sdk a:hover,div#onetrust-pc-sdk a:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-pc-sdk{border-radius:0;bottom:0;height:auto;left:0;margin:auto;max-width:100%;overflow:hidden;right:0;top:0;width:512px;max-height:800px}div#onetrust-pc-sdk .ot-pc-header{display:none}div#onetrust-pc-sdk #ot-pc-content{overscroll-behavior:contain;padding:0 12px 0 24px;margin:16px 4px 0 0;top:0;right:16px;left:0;width:auto}div#onetrust-pc-sdk #ot-category-title,div#onetrust-pc-sdk #ot-pc-title{font-size:24px;font-weight:400;line-height:32px;margin:0 0 16px}div#onetrust-pc-sdk #ot-pc-desc{padding:0}div#onetrust-pc-sdk #ot-pc-desc a{display:inline}div#onetrust-pc-sdk #accept-recommended-btn-handler{display:none!important}div#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:2px solid #007dbc;outline-offset:-1px;transition:none}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item{border-width:0 0 2px}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item:first-of-type{border-width:2px 0}div#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{padding:8px 0;width:100%}div#onetrust-pc-sdk .ot-plus-minus{transform:translateY(2px)}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item>button{background:0 0!important;border:0!important;height:44px!important;max-width:none!important;width:calc(100% - 48px)!important}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-weight:700}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr{padding:0}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item .ot-acc-grpdesc{padding:0;width:100%}div#onetrust-pc-sdk .ot-acc-grpcntr .ot-subgrp-cntr{border:0;padding:0}div#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp{margin:0}div#onetrust-pc-sdk .ot-always-active-group .ot-cat-header{width:calc(100% - 160px)}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{width:calc(100% - 88px)}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active{color:inherit;font-size:12px;font-weight:400;line-height:1.5;padding-right:48px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:before{border-radius:12px;position:absolute;right:0;top:0;content:'';background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:after{border-radius:50%;position:absolute;right:5px;top:4px;content:'';background-color:#007dbc;height:12px;width:12px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active,div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl{right:2px}div#onetrust-pc-sdk .ot-switch{display:block;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob,div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob{background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob:before{background-color:#737373;height:8px;left:4px;top:4px;width:8px}div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob:before{background-color:#007dbc;height:12px;left:0;top:2px;width:12px}div#onetrust-pc-sdk .ot-tgl input:focus+.ot-switch .ot-switch-nob{box-shadow:0 0;outline:2px solid #007dbc!important;outline-offset:1px;transition:none}div#onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-grpcntr.ot-acc-txt{background-color:transparent;padding-left:3px}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr,div#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr{overflow:visible;width:100%}div#onetrust-pc-sdk .ot-pc-footer{border-top:0 solid}div#onetrust-pc-sdk .ot-btn-container{padding-top:24px;text-align:center}div#onetrust-pc-sdk .ot-pc-footer button{margin:8px 0;background-color:#fff}div#onetrust-pc-sdk .ot-pc-footer-logo{background-color:#fff}div#onetrust-pc-sdk #ot-lst-title span{font-size:24px;font-weight:400;line-height:32px}div#onetrust-pc-sdk #ot-host-lst .ot-host-desc,div#onetrust-pc-sdk #ot-host-lst .ot-host-expand,div#onetrust-pc-sdk #ot-host-lst .ot-host-name,div#onetrust-pc-sdk #ot-host-lst .ot-host-name a,div#onetrust-pc-sdk .back-btn-handler,div#onetrust-pc-sdk .ot-host-opt li>div div{font-size:16px;font-weight:400;line-height:24px}div#onetrust-pc-sdk #ot-pc-lst{top:0}div#onetrust-pc-sdk .back-btn-handler{text-decoration:none!important}div#onetrust-pc-sdk #filter-btn-handler:hover svg{filter:invert(1)}div#onetrust-pc-sdk .back-btn-handler svg{width:16px;height:16px}div#onetrust-pc-sdk .ot-host-item>button{background:0 0!important;border:0!important;height:66px!important;max-width:none!important;width:calc(100% - 5px)!important;transform:translate(2px,2px)}div#onetrust-pc-sdk .ot-host-item{border-bottom:2px solid #b9b9b9;padding:0}div#onetrust-pc-sdk .ot-host-item .ot-acc-hdr{margin:0 0 -6px;padding:8px 0}div#onetrust-pc-sdk ul li:first-child{border-top:2px solid #b9b9b9}div#onetrust-pc-sdk .ot-host-item .ot-plus-minus{margin:0 8px 0 0}div#onetrust-pc-sdk .ot-search-cntr{width:calc(100% - 48px)}div#onetrust-pc-sdk .ot-host-opt .ot-host-info{background-color:transparent}div#onetrust-pc-sdk .ot-host-opt li>div div{padding:0}div#onetrust-pc-sdk #vendor-search-handler{border-radius:0;border-color:#939393;border-style:solid;border-width:2px 0 2px 2px;font-size:20px;height:48px;margin:0}div#onetrust-pc-sdk .ot-lst-subhdr svg{right:0;top:8px}div#onetrust-pc-sdk .ot-fltr-cntr{box-sizing:border-box;right:0;width:48px}div#onetrust-pc-sdk #filter-btn-handler{width:48px!important;padding:8px!important}div#onetrust-pc-sdk #ot-fltr-modal{max-height:400px}
.ot-sdk-cookie-policy{font-family:inherit;font-size:16px}.ot-sdk-cookie-policy.otRelFont{font-size:1rem}.ot-sdk-cookie-policy h3,.ot-sdk-cookie-policy h4,.ot-sdk-cookie-policy h6,.ot-sdk-cookie-policy p,.ot-sdk-cookie-policy li,.ot-sdk-cookie-policy a,.ot-sdk-cookie-policy th,.ot-sdk-cookie-policy #cookie-policy-description,.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group,.ot-sdk-cookie-policy #cookie-policy-title{color:dimgray}.ot-sdk-cookie-policy #cookie-policy-description{margin-bottom:1em}.ot-sdk-cookie-policy h4{font-size:1.2em}.ot-sdk-cookie-policy h6{font-size:1em;margin-top:2em}.ot-sdk-cookie-policy th{min-width:75px}.ot-sdk-cookie-policy a,.ot-sdk-cookie-policy a:hover{background:#fff}.ot-sdk-cookie-policy thead{background-color:#f6f6f4;font-weight:bold}.ot-sdk-cookie-policy .ot-mobile-border{display:none}.ot-sdk-cookie-policy section{margin-bottom:2em}.ot-sdk-cookie-policy table{border-collapse:inherit}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy{font-family:inherit;font-size:1rem}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h3,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h4,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h6,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy p,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy li,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-title{color:dimgray}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description{margin-bottom:1em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-subgroup{margin-left:1.5em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group-desc,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-table-header,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy span,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td{font-size:.9em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td span,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td a{font-size:inherit}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group{font-size:1em;margin-bottom:.6em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-title{margin-bottom:1.2em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy>section{margin-bottom:1em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th{min-width:75px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a:hover{background:#fff}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy thead{background-color:#f6f6f4;font-weight:bold}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-mobile-border{display:none}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy section{margin-bottom:2em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-subgroup ul li{list-style:disc;margin-left:1.5em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-subgroup ul li h4{display:inline-block}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table{border-collapse:inherit;margin:auto;border:1px solid #d7d7d7;border-radius:5px;border-spacing:initial;width:100%;overflow:hidden}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table th,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table td{border-bottom:1px solid #d7d7d7;border-right:1px solid #d7d7d7}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr:last-child td{border-bottom:0px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr th:last-child,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr td:last-child{border-right:0px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-host,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-cookies-type{width:25%}.ot-sdk-cookie-policy[dir=rtl]{text-align:left}#ot-sdk-cookie-policy h3{font-size:1.5em}@media only screen and (max-width: 530px){.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) table,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) thead,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tbody,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) th,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr{display:block}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) thead tr{position:absolute;top:-9999px;left:-9999px}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr{margin:0 0 1em 0}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr:nth-child(odd),.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr:nth-child(odd) a{background:#f6f6f4}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td{border:none;border-bottom:1px solid #eee;position:relative;padding-left:50%}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td:before{position:absolute;height:100%;left:6px;width:40%;padding-right:10px}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) .ot-mobile-border{display:inline-block;background-color:#e4e4e4;position:absolute;height:100%;top:0;left:45%;width:2px}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td:before{content:attr(data-label);font-weight:bold}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) li{word-break:break-word;word-wrap:break-word}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table{overflow:hidden}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table td{border:none;border-bottom:1px solid #d7d7d7}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy thead,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy tbody,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy tr{display:block}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-host,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-cookies-type{width:auto}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy tr{margin:0 0 1em 0}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td:before{height:100%;width:40%;padding-right:10px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td:before{content:attr(data-label);font-weight:bold}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy li{word-break:break-word;word-wrap:break-word}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy thead tr{position:absolute;top:-9999px;left:-9999px;z-index:-9999}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr:last-child td{border-bottom:1px solid #d7d7d7;border-right:0px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr:last-child td:last-child{border-bottom:0px}}
                
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h5,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h6,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy li,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy p,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy span,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description {
                        color: #696969;
                    }
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th {
                        color: #696969;
                    }
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group {
                        color: #696969;
                    }
                    
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-title {
                            color: #696969;
                        }
                    
            
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table th {
                            background-color: #F8F8F8;
                        }
                    
            .ot-floating-button__front{background-image:url('https://cdn.cookielaw.org/logos/static/ot_persistent_cookie_icon.png')}</style><script src="https://static.addtoany.com/menu/modules/core.pt5ow5lr.js" type="module"></script><style type="text/css">.a2a_hide{display:none}.a2a_logo_color{background-color:#0166ff}.a2a_menu,.a2a_menu *{-moz-box-sizing:content-box;-webkit-box-sizing:content-box;box-sizing:content-box;float:none;margin:0;padding:0;position:static;height:auto;width:auto}.a2a_menu{border-radius:6px;display:none;direction:ltr;background:#FFF;font:16px sans-serif-light,HelveticaNeue-Light,"Helvetica Neue Light","Helvetica Neue",Arial,Helvetica,"Liberation Sans",sans-serif;color:#000;line-height:12px;border:1px solid #CCC;vertical-align:baseline;overflow:hidden}.a2a_mini{min-width:200px;position:absolute;width:300px;z-index:9999997}.a2a_overlay{display:none;background:#616c7deb;backdrop-filter:blur(4px);-webkit-backdrop-filter:blur(4px);position:fixed;top:0;right:0;left:0;bottom:0;z-index:9999998;-webkit-tap-highlight-color:transparent;transition:opacity .14s,backdrop-filter .14s}.a2a_full{background:#FFF;border:1px solid #FFF;box-shadow:#2a2a2a1a 0 0 20px 10px;height:auto;height:calc(320px);top:15%;left:50%;margin-left:-320px;position:fixed;text-align:center;width:640px;z-index:9999999;transition:transform .14s,opacity .14s}.a2a_full_footer,.a2a_full_header,.a2a_full_services{border:0;margin:0;padding:12px;box-sizing:border-box}.a2a_full_header{padding-bottom:8px}.a2a_full_services{height:280px;overflow-y:scroll;padding:0 12px;-webkit-overflow-scrolling:touch}.a2a_full_services .a2a_i{display:inline-block;float:none;width:181px;width:calc(33.334% - 18px)}div.a2a_full_footer{font-size:12px;text-align:center;padding:8px 14px}div.a2a_full_footer a,div.a2a_full_footer a:visited{display:inline;font-size:12px;line-height:14px;padding:8px 14px}div.a2a_full_footer a:focus,div.a2a_full_footer a:hover{background:0 0;border:0;color:#0166FF}div.a2a_full_footer a span.a2a_s_a2a,div.a2a_full_footer a span.a2a_w_a2a{background-size:14px;border-radius:3px;display:inline-block;height:14px;line-height:14px;margin:0 3px 0 0;vertical-align:top;width:14px}.a2a_modal{height:0;left:50%;margin-left:-320px;position:fixed;text-align:center;top:15%;width:640px;z-index:9999999;transition:transform .14s,opacity .14s;-webkit-tap-highlight-color:transparent}.a2a_modal_body{background:0 0;border:0;font:24px sans-serif-light,HelveticaNeue-Light,"Helvetica Neue Light","Helvetica Neue",Arial,Helvetica,"Liberation Sans",sans-serif;position:relative;height:auto;width:auto}.a2a_thanks{color:#fff;height:auto;margin-top:20px;width:auto}.a2a_thanks>div:first-child{margin:0 0 40px 0}.a2a_thanks div *{height:inherit}#a2a_copy_link{background:#FFF;border:1px solid #FFF;cursor:pointer;margin-top:15%}label.a2a_s_link#a2a_copy_link_icon,label.a2a_w_link#a2a_copy_link_icon{background-size:48px;border-radius:0;display:inline-block;height:48px;left:0;line-height:48px;margin:0 3px 0 0;position:absolute;vertical-align:top;width:48px}#a2a_modal input#a2a_copy_link_text{background-color:transparent;border:0;color:#2A2A2A;cursor:pointer;font:inherit;height:48px;left:62px;max-width:initial;min-height:auto;padding:0;position:relative;width:564px;width:calc(100% - 76px)}#a2a_copy_link_copied{background-color:#0166ff;color:#fff;display:none;font:inherit;font-size:16px;margin-top:1px;padding:3px 8px}@media (forced-colors:active){.a2a_color_buttons a,.a2a_svg{forced-color-adjust:none}}@media (prefers-color-scheme:dark){.a2a_menu a,.a2a_menu a.a2a_i,.a2a_menu a.a2a_i:visited,.a2a_menu a.a2a_more,i.a2a_i{border-color:#2a2a2a!important;color:#fff!important}.a2a_menu a.a2a_i:active,.a2a_menu a.a2a_i:focus,.a2a_menu a.a2a_i:hover,.a2a_menu a.a2a_more:active,.a2a_menu a.a2a_more:focus,.a2a_menu a.a2a_more:hover,.a2a_menu_find_container{border-color:#444!important;background-color:#444!important}.a2a_menu:not(.a2a_thanks){background-color:#2a2a2a;border-color:#2a2a2a}.a2a_menu_find{color:#fff!important}.a2a_menu label.a2a_s_find svg{background-color:transparent!important}.a2a_menu label.a2a_s_find svg path{fill:#fff!important}.a2a_full{box-shadow:#00000066 0 0 20px 10px}.a2a_overlay{background-color:#373737eb}}@media print{.a2a_floating_style,.a2a_menu,.a2a_overlay{visibility:hidden}}@keyframes a2aFadeIn{from{opacity:0}to{opacity:1}}.a2a_starting{opacity:0}.a2a_starting.a2a_full,.a2a_starting.a2a_modal{transform:scale(.8)}@media (max-width:639px){.a2a_full{border-radius:0;top:15%;left:0;margin-left:auto;width:100%}.a2a_modal{left:0;margin-left:10px;width:calc(100% - 20px)}}@media (min-width:318px) and (max-width:437px){.a2a_full .a2a_full_services .a2a_i{width:calc(50% - 18px)}}@media (max-width:317px){.a2a_full .a2a_full_services .a2a_i{width:calc(100% - 18px)}}@media (max-height:436px){.a2a_full{bottom:40px;height:auto;top:40px}}@media (max-height:550px){.a2a_modal{top:30px}}@media (max-height:360px){.a2a_modal{top:20px}.a2a_thanks>div:first-child{margin-bottom:20px}}.a2a_menu a{color:#0166FF;text-decoration:none;font:16px sans-serif-light,HelveticaNeue-Light,"Helvetica Neue Light","Helvetica Neue",Arial,Helvetica,"Liberation Sans",sans-serif;line-height:14px;height:auto;width:auto;outline:0}.a2a_menu a.a2a_i:visited,.a2a_menu a.a2a_more{color:#0166FF}.a2a_menu a.a2a_i:active,.a2a_menu a.a2a_i:focus,.a2a_menu a.a2a_i:hover,.a2a_menu a.a2a_more:active,.a2a_menu a.a2a_more:focus,.a2a_menu a.a2a_more:hover{color:#2A2A2A;border-color:#EEE;border-style:solid;background-color:#EEE;text-decoration:none}.a2a_menu label.a2a_s_find{background-size:24px;height:24px;left:8px;pointer-events:auto;position:absolute;top:7px;width:24px}.a2a_menu label.a2a_s_find svg{background-color:#FFF}.a2a_menu label.a2a_s_find svg path{fill:#CCC}#a2a_menu_container{display:inline-block}.a2a_menu_find_container{border:1px solid #CCC;border-radius:6px;padding:2px 24px 2px 0;position:relative;text-align:left}.a2a_cols_container .a2a_col1{overflow-x:hidden;overflow-y:auto;-webkit-overflow-scrolling:touch}#a2a_modal input,#a2a_modal input[type=text],.a2a_menu input,.a2a_menu input[type=text]{display:block;background-image:none;box-shadow:none;line-height:100%;margin:0;outline:0;overflow:hidden;padding:0;-moz-box-shadow:none;-webkit-box-shadow:none;-webkit-appearance:none}#a2afeed_find_container input,#a2afeed_find_container input[type=text],#a2apage_find_container input,#a2apage_find_container input[type=text]{background-color:transparent;border:0;box-sizing:content-box;color:#2A2A2A;float:none;font:inherit;font-size:16px;height:28px;line-height:20px;left:38px;outline:0;margin:0;max-width:initial;min-height:initial;padding:2px 0;position:relative;width:99%}.a2a_clear{clear:both}.a2a_svg{background-repeat:no-repeat;display:block;overflow:hidden;height:32px;line-height:32px;padding:0;pointer-events:none;width:32px}.a2a_svg svg{background-repeat:no-repeat;background-position:50% 50%;border:none;display:block;left:0;margin:0 auto;overflow:hidden;padding:0;position:relative;top:0;width:auto;height:auto}a.a2a_i,i.a2a_i{display:block;float:left;border:1px solid #FFF;line-height:24px;padding:6px 8px;text-align:left;white-space:nowrap;overflow:hidden;text-overflow:ellipsis;width:132px}a.a2a_i span,a.a2a_more span{display:inline-block;overflow:hidden;vertical-align:top}a.a2a_i .a2a_svg{margin:0 6px 0 0}a.a2a_i .a2a_svg,a.a2a_more .a2a_svg{background-size:24px;height:24px;line-height:24px;width:24px}a.a2a_sss:hover{border-left:1px solid #CCC}a.a2a_more{border-bottom:1px solid #FFF;border-left:0;border-right:0;line-height:24px;margin:6px 0 0;padding:6px;-webkit-touch-callout:none}a.a2a_more span{height:24px;margin:0 6px 0 0}.a2a_kit .a2a_svg{background-repeat:repeat}.a2a_default_style a:empty,.a2a_flex_style a:empty,.a2a_floating_style a:empty,.a2a_overlay_style a:empty{display:none}.a2a_color_buttons a,.a2a_floating_style a{text-decoration:none}.a2a_default_style:not(.a2a_flex_style) a{float:left;line-height:16px;padding:0 2px}.a2a_default_style a:hover .a2a_svg,.a2a_floating_style a:hover .a2a_svg,.a2a_overlay_style a:hover .a2a_svg svg{opacity:.7}.a2a_overlay_style.a2a_default_style a:hover .a2a_svg{opacity:1}.a2a_default_style .a2a_count,.a2a_default_style .a2a_svg,.a2a_floating_style .a2a_svg,.a2a_menu .a2a_svg,.a2a_vertical_style .a2a_count,.a2a_vertical_style .a2a_svg{border-radius:4px}.a2a_default_style .a2a_counter img,.a2a_default_style .a2a_dd,.a2a_default_style .a2a_svg{float:left}.a2a_default_style .a2a_img_text{margin-right:4px}.a2a_default_style .a2a_divider{border-left:1px solid #000;display:inline;float:left;height:16px;line-height:16px;margin:0 5px}.a2a_kit a{cursor:pointer;transition:none}.a2a_floating_style{background-color:#fff;border-radius:6px;position:fixed;z-index:9999995}.a2a_overlay_style{z-index:2147483647}.a2a_floating_style,.a2a_overlay_style{animation:a2aFadeIn .2s ease-in;padding:4px}.a2a_vertical_style:not(.a2a_flex_style) a{clear:left;display:block;overflow:hidden;padding:4px}.a2a_floating_style.a2a_default_style{bottom:0}.a2a_floating_style.a2a_default_style a,.a2a_overlay_style.a2a_default_style a{padding:4px}.a2a_count{background-color:#fff;border:1px solid #ccc;box-sizing:border-box;color:#2a2a2a;display:block;float:left;font:12px Arial,Helvetica,sans-serif;height:16px;margin-left:4px;position:relative;text-align:center;width:50px}.a2a_count:after,.a2a_count:before{border:solid transparent;border-width:4px 4px 4px 0;content:"";height:0;left:0;line-height:0;margin:-4px 0 0 -4px;position:absolute;top:50%;width:0}.a2a_count:before{border-right-color:#ccc}.a2a_count:after{border-right-color:#fff;margin-left:-3px}.a2a_count span{animation:a2aFadeIn .14s ease-in}.a2a_vertical_style .a2a_counter img{display:block}.a2a_vertical_style .a2a_count{float:none;margin-left:0;margin-top:6px}.a2a_vertical_style .a2a_count:after,.a2a_vertical_style .a2a_count:before{border:solid transparent;border-width:0 4px 4px 4px;content:"";height:0;left:50%;line-height:0;margin:-4px 0 0 -4px;position:absolute;top:0;width:0}.a2a_vertical_style .a2a_count:before{border-bottom-color:#ccc}.a2a_vertical_style .a2a_count:after{border-bottom-color:#fff;margin-top:-3px}.a2a_color_buttons .a2a_count,.a2a_color_buttons .a2a_count:after,.a2a_color_buttons .a2a_count:before,.a2a_color_buttons.a2a_vertical_style .a2a_count:after,.a2a_color_buttons.a2a_vertical_style .a2a_count:before{background-color:transparent;border:none;color:#fff;float:none;width:auto}.a2a_color_buttons.a2a_vertical_style .a2a_count{margin-top:0}.a2a_flex_style{display:flex;align-items:flex-start;gap:0}.a2a_default_style.a2a_flex_style{left:0;right:0;width:100%}.a2a_vertical_style.a2a_flex_style{flex-direction:column;top:0;bottom:0}.a2a_flex_style a{display:flex;justify-content:center;flex:1;padding:4px}.a2a_flex_style.a2a_vertical_style a{flex-direction:column}.a2a_floating_style.a2a_color_buttons,.a2a_floating_style.a2a_flex_style{border-radius:0;padding:0}.a2a_floating_style.a2a_default_style.a2a_flex_style{bottom:0}.a2a_kit.a2a_flex_style .a2a_counter img,.a2a_kit.a2a_flex_style .a2a_dd,.a2a_kit.a2a_flex_style .a2a_svg{float:none}.a2a_nowrap{white-space:nowrap}.a2a_note{margin:0 auto;padding:9px;font-size:12px;text-align:center}.a2a_note .a2a_note_note{margin:0;color:#2A2A2A}.a2a_wide a{display:block;margin-top:3px;border-top:1px solid #EEE;text-align:center}.a2a_label{position:absolute!important;clip-path:polygon(0px 0px,0px 0px,0px 0px);-webkit-clip-path:polygon(0px 0px,0px 0px,0px 0px);overflow:hidden;height:1px;width:1px}.a2a_kit,.a2a_menu,.a2a_modal,.a2a_overlay{-ms-touch-action:manipulation;touch-action:manipulation;outline:0}.a2a_dd{-webkit-user-drag:none}.a2a_dd img{border:0}.a2a_button_facebook_like iframe{max-width:none}</style><style></style><style>stream {
  position: relative;
  display: block;
}
</style></head>
<body class="pb-ui">
    <div class="skip">
        <a href="#page-body-id" id="skipNavigationLink" title="Skip to Main Content" tabindex="0" class="skipToContent">Skip to Main Content</a>
        <a href="#mega-menu-nav" id="skipToMainMenu" title="Skip to Main Menu" tabindex="0" class="skipToContent">Skip to Main Menu</a>
    </div>



    <div id="pb-page-content" data-ng-non-bindable="">
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="ux3-layout-widget" data-widget-id="832ad3f3-97ac-45ab-9af1-90c6f7bff54a" class="ux3">
        
    

        


    

            

            
                <div class="cell fonts-loaded">
            

            <div data-widget-id="832ad3f3-97ac-45ab-9af1-90c6f7bff54a" data-widget-name="ux3-layout-widget"><div>
    

        


    

            

            

            <div id="a1f1ba25-79af-4194-aaa9-66206104a800" class="widget loginPopup none widget-none widget-compact-all"><div class="wrapped"><div class="widget-body body body-none body-compact-all"><div role="dialog" aria-modal="true" aria-label="Login" class="popup login-popup hidden"><div class="content"><a href="#" aria-label="Close login popup" title="Close login popup" role="button" class="close"><i class="icon-close_thin"></i></a><h2 class="login-title">Login to your account</h2><div class="login-form"><p tabindex="-1" class="message error"></p><form action="/action/doLogin?redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2820%2930339-1" method="post"><input type="hidden" name="redirectUri" value="https://www.cell.com/cell/fulltext/S0092-8674(20)30339-1"><input type="hidden" name="referer" value="/cell/fulltext/S0092-8674(20)30339-1"><input type="hidden" name="ecommRedirect"><input type="hidden" name="code" value="cell-site"><input type="hidden" name="UX3LoginWidget" value="true"><input type="hidden" name="isPopup" value="true"><div class="input-group login-input--wrapper"><div class="label"><label for="login">Email/Username</label></div><input id="login" type="text" name="login" value="" size="15" class="login"><div role="alert" class="error-hint js--hidden"><div class="error-hint__text"><div class="error-text">Your email address is a required field.</div><div class="email-example">E.g., j.smith@mail.com</div></div></div></div><div class="input-group login-input--wrapper"><div class="label"><label for="password">Password</label></div><input id="password" type="password" name="password" value="" autocomplete="off" class="password"><a href="#" class="login__field__action js-show-password">Show</a><div role="alert" class="error-hint js--hidden"><span class="error-hint__text">Your password is a required field.</span></div><div class="actions"><a href="https://www.cell.com/action/requestResetPassword" class="link show-request-reset-password">Forgot password?</a></div></div><div class="remember"><input type="checkbox" name="savePassword" id="a1f1ba25-79af-4194-aaa9-66206104a800-remember" value="1" checked="checked"><label for="a1f1ba25-79af-4194-aaa9-66206104a800-remember">Remember me</label></div><div class="submit"><input type="submit" name="submit" value="Log in" class="button submit primary"></div></form></div><div class="create-account"><div>Don’t have an account?</div><a href="/action/registration?redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2820%2930339-1" class="show-registration">Create a Free Account</a></div></div></div></div></div></div>

            

        

    

        


    

            

            

            <div role="dialog" aria-modal="true" aria-label="Reset password" class="popup top-drawer request-reset-password-drawer hidden"><div class="content"><a href="#" aria-label="Close reset password popup" title="Close reset password popup" role="button" class="close"><i class="icon-close_thin"></i></a><div class="request-reset"><div class="form"><p class="sub">If you don't remember your password, you can reset it by entering your email address and clicking the Reset Password button. You will then receive an email that contains a secure link for resetting your password</p><div tabindex="-1" class="message error"></div><form action="/action/requestResetPassword" method="post" class="request-reset-password-form"><input type="hidden" name="platform"><input type="hidden" name="requestResetPassword" value="true"><input type="hidden" name="codeSite" value="cell-site"><div class="input-group"><div class="label"><label for="reset-password-email">Email*</label></div><input id="reset-password-email" type="text" name="email" value="" size="15" class="email"></div><div class="submit" disabled="disabled"><input type="submit" name="submit" value="Submit" disabled="disabled" class="button primary submit"></div></form></div><div class="success-template hidden"><p>If the address matches a valid account an email will be sent to __email__ with instructions for resetting your password</p></div></div><div class="request-reset-cancel"><a href="#" aria-label="Close reset password popup" title="Close reset password popup" class="cancel">Cancel</a></div></div></div>

            

        

    

        


    

            

            
                <div class="ux3-ads">
            

            <div class="pb-ad">
            <!-- Do Not Modify without contacting ELS-HSOnlineCampaign@elsevier.com -->
<!-- To be kept in sync with Global Commercial Publishing Operations-->
<!-- Start: GPT Async -->
<div id="gpt-Variables" data-adfile="CELLJOURNAL">
	<input type="hidden" id="gptSite" value="cell">
	<input type="hidden" id="gptLoggedIn" value="no">
	<input type="hidden" id="gptAccessType" value="ae:ANON_GUEST|ae:ANON_GUEST">
	<input type="hidden" id="gptDefaultSite" value="cell">
</div>
<div id="gpt-Interscroller" class="gpt-InScrollBox" style="display:none;">
	<div class="gpt-InScrollLabel">ADVERTISEMENT</div>
	<div class="gpt-InScrollContainer">
		<div class="gpt-InScrollWrapper">
			<div class="gpt-InScrollInner">
				<div class="gpt-adinterscroller" data-adsize="interscroller"></div>
			</div>
		</div>
	</div>
	<div class="gpt-InScrollLabel">SCROLL TO CONTINUE WITH CONTENT</div>
</div>
<div id="gpt-DebugDiv" class="gpt-DebugDiv" style="display:none;">
	<div style="width: 100%; display: table;">
		<div style="display: table-row">
			<div style="width: 200px; display: table-cell;">
				<button class="gpt-DebugBtn" onclick="gpt_ShowConsole()">Open GPT Console</button><br>
				<button id="gpt_GrapeShotDebugBtn" class="gpt-DebugBtn" onclick="gpt_GrapeShotDebug();">Open Oracle Keywords</button><br>
				<br>
				<button class="gpt-DebugBtn" onclick="gpt_LoadDebugValues()">Refresh Values</button>
			</div>
			<div style="display: table-cell;">
				<table class="gpt-DebugTable">
					<thead>
						<tr>
							<th>Property</th>
							<th>Value</th>
						</tr>
					</thead>
					<tbody>
						<tr><td class="gpt-DebugTitle">Status</td><td><div class="gptStatus_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Version</td><td><div class="gptVersion_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Ad File</td><td><div class="gptAdFile_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Disable Ads Flag</td><td><div class="gptDisableAds_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Environment</td><td><div class="gptTestENV_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Moat Init</td><td><div class="gptMoatInit_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Moat Ready</td><td><div class="gptMoatReady_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual Ready</td><td><div class="gptContextualReady_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual URL</td><td><div class="gptContextualURL_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual Initial Segments</td><td><div class="gptContextualInit_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual Used Segments</td><td><div class="gptContextualResult_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">AdUnit</td><td><div class="gptSite_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">SubAdUnit</td><td><div class="gptPage_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Custom Targeting</td><td><div class="gptCustomTargeting_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Ad Events</td><td><div class="gptEvent_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Invalid Ad Sizes</td><td><div class="gptInvalidAdSize_Debug"></div></td></tr>
					</tbody>
				</table>
			</div>
		</div>
	</div>
</div>
<style type="text/css">
	.gpt-DebugBtn {
		display: inline-block;
		text-align: center;
		text-decoration: none;
		margin: 2px 0;
		border: solid 1px #122b40;
		border-radius: 0.4em;
		padding: 0.5em 1em;
		color: #ffffff;
		background-color: #204d74;
		font-weight: normal;
		font-size: 12px;
	}

	.gpt-DebugDiv {
		margin: 5px;
	}

	.gpt-DebugTitle {
		font-weight: bold;
	}

	.gpt-DebugTable {
		border: 1px solid #dddddd;
		border-collapse: collapse;
		margin: 5px;
		font-size: 0.9em;
		font-family: sans-serif;
		min-width: 400px;
		margin-bottom: 25px;
	}

		.gpt-DebugTable td:first-child {
			padding: 5px;
		}

		.gpt-DebugTable thead tr th {
			background-color: #d9edf7;
			color: #ffffff;
			text-align: left;
			padding: 5px;
			border-bottom: 2px solid #204d74;
		}

		.gpt-DebugTable th,
		.gpt-DebugTable td {
			padding: 5px;
		}

		.gpt-DebugTable tbody tr {
			border-bottom: 1px solid #dddddd;
		}

			.gpt-DebugTable tbody tr:nth-of-type(even) {
				background-color: #f3f3f3;
			}

			.gpt-DebugTable tbody tr:last-of-type {
				border-bottom: 2px solid #204d74;
			}

	.gpt-InScrollLabel {
		background-color: rgb(0, 0, 0);
		color: rgb(255, 255, 255);
		position: relative;
		max-width: 100vw;
		left: 50%;
		right: 50%;
		margin-left: -50vw;
		margin-right: -50vw;
		padding: 0;
		text-align: center;
		width: 100vw !important;
		font-size: 10px;
	}

	.gpt-InScrollBox {
		height: 100vh !important;
		width: 100vw;
		cursor: pointer;
		max-width: 100vw;
		position: relative;
		left: 50%;
		right: 50%;
		margin-left: -50vw;
		margin-right: -50vw;
		margin-bottom: 50px;
		padding-top: 5px;
		padding-bottom: 5px;
		z-index: 10001;
		background-color: white;
	}

		.gpt-InScrollBox .gpt-InScrollContainer {
			height: 95vh !important;
			position: relative;
		}

		.gpt-InScrollBox .gpt-InScrollWrapper {
			position: absolute !important;
			top: 0 !important;
			left: 0;
			width: 100%;
			height: 95vh !important;
			border: 0 !important;
			margin: 0 !important;
			padding: 0 !important;
			clip: rect(0, auto, auto, 0) !important;
			-webkit-clip-path: polygon(0px 0px, 100% 0px, 100% 100%, 0px 100%) !important;
			clip-path: polygon(0px 0px, 100% 0px, 100% 100%, 0px 100%) !important;
		}

		.gpt-InScrollBox .gpt-InScrollInner {
			position: fixed !important;
			top: 0 !important;
			left: 0 !important;
			bottom: 0;
			margin: auto;
			width: 100vw;
			height: 95vh;
			-webkit-transform: translateZ(0) !important;
			display: -webkit-box;
			display: -ms-flexbox;
			display: flex;
			-webkit-box-orient: vertical;
			-webkit-box-direction: normal;
			-ms-flex-direction: column;
			flex-direction: column;
			-webkit-box-align: center;
			-ms-flex-align: center;
			align-items: center;
			-webkit-box-pack: center;
			-ms-flex-pack: center;
			justify-content: center;
			text-align: center;
		}
</style>
<script class="optanon-category-4" type="text/plain">
	//--------------------------------------------------------------------------
	var gpt_Version = '04072023';
	var gpt_AdFile = $('#gpt-Variables').data('adfile');
	var gpt_MaxMOATInitAttempts = 11;
	var gpt_MaxMOATReadyAttempts = 11;
	var gpt_MaxContextualReadyAttempts = 21;
	var gpt_MOATInitAttempts = 0;
	var gpt_MOATReadyAttempts = 0;
	var gpt_ContextualReadyAttempts = 0;
	var gpt_IsProd = true;
	var gpt_StartTime;
	var gs_channels = 'DEFAULT';
	var gpt_ContextualReady = false;
	var gpt_ContextualURL = 'Not enabled outside of Production';
	var gpt_DisableAdsFlagFound = false;

	var gptEvent_Debug = '';
	var gptCustomTargeting_Debug = '';
	var gptSite_Debug = "";
	var gptPage_Debug = "";
	var gptContextualEnabled = false;
	var gptContextualResult_Debug = '';
	var gptContextualInit_Debug = '';
	var gptContextualURL_Debug = '';
	var gptContextualReady_Debug = '';
	var gptMoatReady_Debug = '';
	var gptMoatInit_Debug = '';
	var gptTestENV_Debug = '';
	var gptDisableAds_Debug = '';
	var gptStatus_Debug = 'GPT Still Loading - Refresh Values';
	var gptInvalidAdSize_Debug = '';
	//--------------------------------------------------------------------------
	$(document).ready(function () {
		//Check to see if we are on a development env
		gpt_CheckEnvironment();

		if (!gpt_DisableAdsFlagFound) {
			gpt_StartTime = new Date();
			if (gpt_AdFile !== "PU") {
				//Once the dom has loaded, begin the process to init the ads.
				gpt_InitScripts();
				gpt_CheckForMOATInit();
			}
		}
	});
</script>
<script type="text/javascript">
	function gpt_CheckEnvironment() {
		if ($('#gptDisableAds').length > 0) {
			gptDisableAds_Debug = 'Flag Detected - Ads Disabled';
			gpt_DisableAdsFlagFound = true;
		}
		else {
			gptDisableAds_Debug = 'Ads Enabled';
			gpt_DisableAdsFlagFound = false;
		}
		switch (gpt_AdFile) {
			case 'PU':
				gptContextualEnabled = true;
				if ($("#Environment").val() !== "Prod") {
					gpt_IsProd = false;
					gptTestENV_Debug = 'Test Environment Detected';
				}
				else {
					gptTestENV_Debug = 'Production Environment Detected';
				}
				break;
			case 'JBS':
			case 'LANCET':
				gptContextualEnabled = true;
				if ($(".gpt-ad").length > 0) {
					//Other ads exist, allow the Prestitial
					$(document.body).append("<div class='gpt-ad' data-adsize='prestitial'></div>");
				}
				if (window.location.hostname.toLowerCase().indexOf('.literatumonline.') >= 0) {
					gpt_IsProd = false;
					gptTestENV_Debug = 'Test Environment Detected';
				}
				else {
					gptTestENV_Debug = 'Production Environment Detected';
				}
				break;
			default:
				//Cell
				gptContextualEnabled = false;
				if (window.location.hostname.toLowerCase().indexOf('.literatumonline.') >= 0) {
					gpt_IsProd = false;
					gptTestENV_Debug = 'Test Environment Detected';
				}
				else {
					gptTestENV_Debug = 'Production Environment Detected';
				}
				break;
		}
	}
	//--------------------------------------------------------------------------
    function gpt_GetNonCMEArticle() {
        var url;
        $('.articleCitation').each(function (i, obj) {
            if ($(obj).find('.toc__item__detials').length > 0) {
                var itemDetails = $(obj).first().find('.toc__item__detials');
                var cmeFound = 0;
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='CME']").length; }
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='(CME)']").length; }
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='Education']").length; }
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='education']").length; }
                if (cmeFound == 0) {
                    var itemLink = $(itemDetails).first().find('.toc__item__title');
                    var linkCount = $(itemLink).first().find("a:contains('CME')").length;
                    if (linkCount == 0) {
                        url = $(itemLink).first().find('a').attr('href');
                        return false;   //Found a URL, break the loop
                    }
                }
            }
        });
        return url;
    }
	//--------------------------------------------------------------------------
	function gpt_GetElapsedTime() {
		var endTime = new Date();
		var timeDiff = endTime - gpt_StartTime; //in ms
		// strip the ms
		timeDiff /= 1000;

		// get seconds
		return seconds = Math.round(timeDiff) + " seconds";
	}
	//--------------------------------------------------------------------------
	function gpt_CheckSSC() {
		//Check for ScreenShot Client
		var gpt_SSC = gpt_GetQuerystring('gpt_ssc', '');
		if (gpt_SSC == '1') {
			//it's the ScreenShotClient, disable MOAT IVT Detection
			$('#gpt-Variables').append('<input type="hidden" id="gptm_data">');
			$('#gptm_data').val(0);
			gpt_Target("m_data");
		}
	}
	//--------------------------------------------------------------------------
	function gpt_LocalSS(adTypeId) {
		window.scrollTo(0, 0);
		gpt_Sleep(1000);
		gpt_ScrollToAd(adTypeId);
		gpt_PreTearsheet(adTypeId);
	}
	//--------------------------------------------------------------------------
	function gpt_InitScripts() {
		if (gpt_AdFile == "PU") {
			if (gpt_IsProd) {
				gpt_ContextualURL = gpt_CheckForNULL($('#canonicalurl').val());
				if (gpt_ContextualURL === null) {
					gpt_ContextualURL = window.location.href.split('?')[0].split('#')[0];
				}
				else {
					gpt_ContextualURL = window.location.origin + gpt_ContextualURL;
				}
				var useSSL = 'https:' == document.location.protocol;
				var gs = document.createElement('script');
				gs.async = true;
				gs.type = 'text/javascript';
				gs.src = (useSSL ? 'https:' : 'http:') + '//elsevier.gscontxt.net/multizone/channels.cgi?url=' + encodeURIComponent(gpt_ContextualURL);
				gs.onload = function () {
					gpt_ContextualReady = true;
				};
				document.head.appendChild(gs);
			}
		}
		else {
			//JBS-Cell-Lancet
			var useSSL = 'https:' == document.location.protocol;

			var gads = document.createElement('script');
			gads.async = true;
			gads.type = 'text/javascript';
			gads.src = (useSSL ? 'https:' : 'http:') + '//securepubads.g.doubleclick.net/tag/js/gpt.js';
			document.head.appendChild(gads);

			if (gpt_IsProd && gptContextualEnabled) {
				gpt_ContextualURL = window.location.href.split('?')[0].split('#')[0];
				var gs = document.createElement('script');
				gs.async = true;
				gs.type = 'text/javascript';
				gs.src = (useSSL ? 'https:' : 'http:') + '//elsevier.gscontxt.net/multizone/channels.cgi?url=' + encodeURIComponent(gpt_ContextualURL);
				gs.onload = function () {
					gpt_ContextualReady = true;
				};
				document.head.appendChild(gs);
			}

			var elsevierMoatHeader = document.createElement('link');
			elsevierMoatHeader.rel = 'preload';
			elsevierMoatHeader.as = 'script';
			elsevierMoatHeader.src = (useSSL ? 'https:' : 'http:') + '//z.moatads.com/elsevierheader150204004183/moatheader.js';
			document.head.appendChild(elsevierMoatHeader);

			var elsevierMoatAds = document.createElement('link');
			elsevierMoatAds.rel = 'preconnect';
			elsevierMoatAds.src = (useSSL ? 'https:' : 'http:') + '//mb.moatads.com';
			document.head.appendChild(elsevierMoatAds);

			var moatHeader = document.createElement('script');
			moatHeader.async = true; gads.type = 'text/javascript';
			moatHeader.src = (useSSL ? 'https:' : 'http:') + '//z.moatads.com/elsevierheader150204004183/moatheader.js';
			document.head.appendChild(moatHeader);
		}
	}
	//--------------------------------------------------------------------------
	async function gpt_CheckForMOATInit() {
		//Check every 1/10 of a second until MOAT is Loaded or 1 second timeout
		if (gpt_MOATInitAttempts < gpt_MaxMOATInitAttempts) {
			gpt_MOATInitAttempts += 1;
			if (typeof window.moatPrebidApi === "object") {
				//MOAT LOADED
				gptMoatInit_Debug = '0.0' + gpt_MOATInitAttempts + ' seconds';
				gpt_CheckForMOATReady();
			}
			else {
				setTimeout(() => { gpt_CheckForMOATInit(); }, 100);
			}
		}
		else {
			//TIMEOUT: MOAT Not LOADED
			gptMoatInit_Debug = 'Timeout Expired';
			var googletag = googletag || {};
			googletag.cmd = googletag.cmd || [];
			gpt_CheckForContextualReady();
		}
	}
	//--------------------------------------------------------------------------
	function gpt_CheckForMOATReady() {
		//Check every 1/10 of a second until MOAT is Ready or 1 second timeout
		if (gpt_MOATReadyAttempts < gpt_MaxMOATReadyAttempts) {
			gpt_MOATReadyAttempts += 1;
			if (window.moatPrebidApi.safetyDataAvailable()) {
				//MOAT Ready
				gptMoatReady_Debug = '0.0' + gpt_MOATReadyAttempts + " seconds";
				gpt_CheckForContextualReady();
			}
			else {
				setTimeout(() => { gpt_CheckForMOATReady(); }, 100);
			}
		}
		else {
			//TIMEOUT: MOAT Not Ready
			gptMoatReady_Debug = 'Timeout Expired';
			gpt_CheckForContextualReady();
		}
	}
	//--------------------------------------------------------------------------
	function gpt_CheckForContextualReady() {
		//Check every 1/10 of a second until Contextual is Ready or 2 second timeout
		if (gpt_IsProd && gptContextualEnabled) {
			//Enabled
			if (gpt_ContextualReadyAttempts < gpt_MaxContextualReadyAttempts) {
				gpt_ContextualReadyAttempts += 1;
				if (gpt_ContextualReady) {
					//Contextual Ready
					gptContextualURL_Debug = gpt_ContextualURL;
					gptContextualReady_Debug = '0.0' + gpt_ContextualReadyAttempts + " seconds";
					gptContextualInit_Debug = gs_channels;
					gpt_InitAds();
				}
				else {
					setTimeout(() => { gpt_CheckForContextualReady(); }, 100);
				}
			}
			else {
				//LOG TIMEOUT error
				//REMOVE FOR NOW (SHOULD LOG TO A DIFF LOCATION)
				//TIMEOUT: Contextual Not Ready
				$('#gpt_GrapeShotDebugBtn').remove();
				gptContextualURL_Debug = gpt_ContextualURL;
				gptContextualReady_Debug = 'Timeout Expired';
				gptContextualInit_Debug = "";
				gpt_InitAds();
			}
		}
		else {
			//Not Enabled
			$('#gpt_GrapeShotDebugBtn').remove();
			switch (gpt_AdFile) {
				case 'PU':
				case 'LANCET':
				case 'JBS':
					gptContextualURL_Debug = 'Not enabled outside of Production';
					gptContextualReady_Debug = 'Not enabled outside of Production';
					gptContextualInit_Debug = 'Not enabled outside of Production';
					break;
				case "CELLJOURNAL":
				case "CELLBUCKET":
					gptContextualReady_Debug = 'Not enabled on Cell';
					gptContextualInit_Debug = 'Not enabled on Cell';
					gptContextualURL_Debug = 'Not enabled on Cell';
					break;
			}
			gpt_InitAds();
		}
	}
	//--------------------------------------------------------------------------
	function GPT_GetPageByURL() {
		var host = window.location.host.toLocaleLowerCase()
		var localpath = window.location.pathname.toLocaleLowerCase();

		///Check for Cell.com
		if (host.indexOf('cell.com') >= 0) {
			return null;
		}

		///Check for Cell.com (marlin-stag)
		if (host.indexOf('www-cell-com') >= 0) {
			return null;
		}

		///Check for Lancet
		if (host.indexOf('thelancet') >= 0) {
			return null;
		}

		if (localpath.indexOf('/article/') >= 0) {
			return 'article.fulltext';
		}

		if (localpath.indexOf('/action/dosearch') >= 0) {
			return 'search.searchResults';
		}

		if (localpath.indexOf('/content/') >= 0) {
			return 'periodical.editorial';
		}

		return 'periodical.home';
	}
	//--------------------------------------------------------------------------
	function gpt_CheckForNULL(x, valueifnull) {
		if (typeof valueifnull == "undefined") {
			valueifnull = null;
		}

		if (typeof x == 'undefined') {
			return valueifnull;
		}
		if (x == null) {
			return valueifnull;
		}
		if (x == "null") {
			return valueifnull;
		}
		if ($.trim(x) == '') {
			return valueifnull;
		}

		return x;
	}
	//--------------------------------------------------------------------------
	function gpt_GetAdditionalTargeting() {
		//adds page level targeting of custom variables
		//the id (minus gpt) and the value will be used as a key-value pair
		//eg: <input type="hidden" id="gptissuepii" value="x" class="gpt-Target">
		//eg: target issuepii=x
		$(".gpt-Target").each(function (index) {
			var gptTargetId = $(this).attr('id');
			gptTargetId = gptTargetId.replace('gpt', '').toLowerCase();
			if (gptTargetId !== '') {
				gpt_Target(gptTargetId);
			}
		})
	}
	//--------------------------------------------------------------------------
	function gpt_GetQuerystring(key, default_) {
		//function to read querystring variables
		if (default_ === null) default_ = '';
		key = key.replace(/[\[]/, "\\\[").replace(/[\]]/, "\\\]");
		var regex = new RegExp("[\\?&]" + key + "=([^&#]*)");
		var qs = regex.exec(window.location.href);
		if (qs === null)
			return default_;
		else
			return qs[1];
	}
	//--------------------------------------------------------------------------
	function gpt_GetDFPTestId() {
		//Get any request for a test id from querystring and set custom targeting parameter
		var gptDFPTestId = gpt_GetQuerystring('dfptestid', '');
		if (gptDFPTestId !== '') {
			$('#gpt-Variables').append('<input type="hidden" id="gptDFPTestId">');
			$('#gptDFPTestId').val(gptDFPTestId);
			gpt_Target("DFPTestId");
		}
	}
	//--------------------------------------------------------------------------
	function gpt_Target(targetName) {
		//Set page level target if value is found
		if ($('#gpt' + targetName).length > 0) {
			var targetValue = $('#gpt' + targetName).val();

			if (targetValue !== '') {
				targetValue = targetValue.replace(/[^-a-z0-9]/ig, '');	//Only Allow a-z, 0-9
				googletag.pubads().setTargeting(targetName.toLowerCase(), targetValue.toLowerCase());
				gptCustomTargeting_Debug = gptCustomTargeting_Debug + targetName.toLowerCase() + ' = ' + targetValue.toLowerCase() + '<br>';
			}
		}
	}
	//--------------------------------------------------------------------------
	function gpt_TargetAccessType() {
		var currentAccessType = $('#gptAccessType').val();
		if (typeof currentAccessType == "undefined") {
			currentAccessType = "";
		}
		if (currentAccessType.indexOf('ANON_IP') >= 0) {
			googletag.pubads().setTargeting("loggedin", "yes");
		}
		else {
			gpt_Target("LoggedIn");
		}
	}
	//--------------------------------------------------------------------------
	async function gpt_PreTearsheet(adTypeId) {
		if (typeof adTypeId === "undefined") {
			adTypeId = 0;
		}
		//Remove Elsevier Cookie Policy Notice
		if ($("#onetrust-consent-sdk").length > 0) {
			$("#onetrust-consent-sdk").remove();
		}

        //Mobile Nav
        $('#skip').remove();
        $('skipNavigationLink').remove();

		//JBS Modals
		if ($('.leadinModal').length > 0) {
			$('.leadinModal').remove();
		}
		if ($('.leadinModal').length > 0) {
			$('.leadinModal').remove();
		}
		if ($('.usabilla__overlay').length > 0) {
			$('.usabilla__overlay').remove();
		}

		//Pendo
		if ($('._pendo-step-container').length > 0) {
			$('._pendo-step-container').remove();
		}
		if ($('._pendo-guide-tt_').length > 0) {
			$('._pendo-guide-tt_').remove();
		}
		if ($('._pendo-image').length > 0) {
			$('._pendo-image').remove();
		}
		if ($('._pendo-badge-image').length > 0) {
			$('._pendo-badge-image').remove();
		}

		switch (gpt_AdFile) {
			case "CELLJOURNAL":
			case "CELLBUCKET":
				//Mobile Bug Fix on Cell
				if (parseInt(adTypeId) == 5) {
					$('.widget-horizontalAd').css('position', 'static');
					$('.body-horizontalAd').css('display', 'block');
				}
				break;
			case "JBS":
			case "LANCET":
				objLeaderboard = ".container-leaderboard";
				offset = 53;
				switch (parseInt(adTypeId)) {
					case 3: //MPU
					case 4: //SkyScraper
						//BUGFIX: Show Sidebar ad for JBS
						//skyscraper & mpu
						if ($('.widget-verticalAd').length > 0) {
							$('.widget-verticalAd').css('display', 'block');
						}
						$('#backgroundLayer').css('display', 'none');   //Hide any Prestitial
						break;
					case 6:     //Prestitial
						$('#backgroundLayer').css('display', 'block');
						break;
					default:
						$('#backgroundLayer').css('display', 'none');   //Hide any Prestitial
						break;
				}
				break;
		}
	}
	//--------------------------------------------------------------------------
	function gpt_Sleep(ms) {
		return new Promise(resolve => setTimeout(resolve, ms));
	}
	//--------------------------------------------------------------------------
	async function gpt_ScrollToAd(adTypeId) {
		if (typeof adTypeId === "undefined") {
			adTypeId = 0;
		}
		var adId = "";
		var offset = 0;
		var objLeaderboard = "";
		var gpt_ScrollToAd = true;

		switch (gpt_AdFile) {
			case "PU": //PracticeUpdate
				var hideOtherAds = true;
				$('#modalPrestitial').modal('hide');
				$('.jumbotron-welcome').hide();
				objLeaderboard = ".container-leaderboard";
				offset = 53;
				switch (parseInt(adTypeId)) {
					case 2:
						//LeaderBoard, no Scroll Needed
						gpt_ScrollToAd = false;
						adId = "adSlot1";
						break;
					case 3:
						//MPU
						adId = "adSlot2";
						break;
					case 4:
						//SkyScraper
						adId = "adSlot3";
						$('.most-read').hide();
						break;
					case 5:
						//Mobile, no Scroll Needed
						gpt_ScrollToAd = false;
						adId = "adSlot1";
						break;
					case 6:
						//Interstitial
						gpt_ScrollToAd = false;
						adId = "adSlot15";
						await gpt_Sleep(1000);
						$(objLeaderboard).hide();
						$('#modalPrestitial').unbind();
						$('#modalPrestitial').modal();
						break;
					case 7:
						//Native Ad
						break;
					case 10:
						//Master Companion
						hideOtherAds = false;
						gpt_ScrollToAd = false;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						adId = "adSlot1";
						break;
				}
				if (hideOtherAds) {
					//Hide all other ads
					$('.ad-container').each(function (i, obj) {
						if (this.id !== adId) {
							var tempParent = this.id;
							tempParent = tempParent.replace("adSlot", "adParent");
							$("#" + tempParent).addClass('hide');
						}
					});
				}
				break;
			case "JBS":
				objLeaderboard = ".js__adToHide";
				offset = 80;
				switch (parseInt(adTypeId)) {
					case 2:
						//Leaderboard
						gpt_ScrollToAd = false;
						$(".widget-verticalAd").hide();
						break;
					case 5:
						//Mobile
						//Scroll to Top
						gpt_ScrollToAd = false;
						break;
					case 3: //MPU
					case 4: //SkyScraper
						$(objLeaderboard).hide();
						adId = $('.gpt-ad-defined[data-adsize="sidebar"').attr('id');
						break;
					case 10:
						//MasterCompanion
						gpt_ScrollToAd = false;
						break;
					case 13:	    //Phone - Portrait
					case 14:        //Phone - Landscape
						$(objLeaderboard).hide();
						offset = 60;
						adId = 'gpt-Interscroller';
						gpt_ScrollToAd = true;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						break;
				}
				break;
			case "CELLJOURNAL":
			case "CELLBUCKET":
				objLeaderboard = ".js__adToHide";
				offset = 260;
				switch (parseInt(adTypeId)) {
					case 2:
						//Leaderboard
						gpt_ScrollToAd = false;
						$('.body-verticalAd').hide();
						break;
					case 5:
						//Mobile
						//Scroll to Top
						gpt_ScrollToAd = false;
						break;
					case 3: //MPU
					case 4: //SkyScraper
						$(objLeaderboard).hide();
						adId = $('.gpt-ad-defined[data-adsize="boombox"').attr('id');
						break;
					case 10:
						//MasterCompanion
						gpt_ScrollToAd = false;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						break;
				}
				//Hide the second ad if found
				var tCount = 0;
				$(".widget.literatumAd.none.GPT-ad-placeholder.widget-horizontalAd.widget-compact-all").each(function (index) {
					tCount += 1;
					if (tCount !== 1) {
						$(this).hide();
					}
				});
				break;
			case "LANCET":
				objLeaderboard = ".js__adToHide";
				offset = 80;
				switch (parseInt(adTypeId)) {
					case 2:
						//Leaderboard
						gpt_ScrollToAd = false;
						$('.body-verticalAd').hide();
						break;
					case 5:
						//Mobile
						//Scroll to Top
						gpt_ScrollToAd = false;
						break;
					case 3: //MPU
					case 4: //SkyScraper
						$(objLeaderboard).hide();
						adId = $('.gpt-ad-defined[data-adsize="skyscraper"').attr('id');
						break;
					case 10:
						//MasterCompanion
						gpt_ScrollToAd = false;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						break;
				}
				break;
		}

		//Scroll to Top
		var gpt_container = $("html,body");
		gpt_container.animate({ scrollTop: 0, scrollLeft: 0 });

		if (gpt_ScrollToAd) {
			//If Adexists, scroll to it
			var gpt_scrollTo = $('#' + adId);
			if (gpt_scrollTo.length > 0) {
				//Scroll to the Ad
				var gpt_topPosition = gpt_scrollTo.offset().top - gpt_container.offset().top + gpt_container.scrollTop();
				if (gpt_topPosition > 0) {
					gpt_container.animate({ scrollTop: gpt_topPosition, scrollLeft: 0 });
					if (gpt_topPosition - offset > 0 && offset > 0) {
						gpt_container.animate({ scrollTop: gpt_topPosition - offset, scrollLeft: 0 });
					}
				}
			}
		}
	}
	//--------------------------------------------------------------------------
	async function gpt_CheckForCookiePolicy() {
		var returnVal = false;
		if ($('#onetrust-accept-btn-handler').length > 0) {
			$('#onetrust-accept-btn-handler').click();
			returnVal = true;
		}
		return returnVal;
	}
	//--------------------------------------------------------------------------
	function gpt_InitAds() {
		var gpt_ResponsiveHomepage = false;

		switch (gpt_AdFile) {
			case "JBS":
			case "CELLJOURNAL":
			case "CELLBUCKET":
				//Responsive Check
				if ($('#hdrHomeBrand').length > 0) {
					gpt_ResponsiveHomepage = true;
				}
				break;
		}

		//InterScroller Check
		if (gpt_AdFile === "JBS") {
			var gpt_InterScrollerEnabled = false;
			var gpt_InterScrollerInsertObj;

			if ($("#articleHeader").length > 0) {
				//It's an article Page
				if ($('.article__sections > section').length > 0) {
					gpt_InterScrollerEnabled = true;
					gpt_InterScrollerInsertObj = $(".article__sections > section");
				}
			}
			else {
				if ($('.toc__section').length > 0) {
					//It's a TOC Page
					gpt_InterScrollerEnabled = true;
					gpt_InterScrollerInsertObj = $(".toc__section").first();
				}
			}

			//Add the Interscroller
			if (gpt_InterScrollerEnabled) {
				gpt_InterScrollerInsertObj.first().append($("#gpt-Interscroller").clone());
				$("#gpt-Interscroller").remove();
				$(".gpt-adinterscroller").addClass("gpt-ad");
			}
		}

		googletag.cmd.push(function () {
			var gptBoomBoxSizeMapping;
			var gptFluidMapping;
			var gptHomePageSidebarMapping;
			var gptCellHomePageSidebarMapping;
			var gptInterscrollerMapping;
			var gptLeaderboardSizeMapping;
			var gptMobileSizeMapping;
			var gptPrestitialMapping;
			var gptRectangleShortSizeMapping;
			var gptRectangleSizeMapping;
			var gptSideBarSizeMapping;
			var gptTallSizeMapping;
			var gptWideSizeMapping;
			var gptWideRectangleSizeMapping;

			//***********************************************
			//Start Define Size mapping based on client Viewport
			//***********************************************
			if (gpt_AdFile === 'PU') {
				gptTallSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[300, 600], [160, 600], [120, 600]]).
					addSize([0, 0], []).
					build();

				gptRectangleSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[300, 250]]).
					addSize([0, 0], []).
					build();

				gptRectangleShortSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[300, 250], [320, 50], [300, 50]]).
					addSize([0, 0], [[300, 250], [320, 50], [300, 50]]).
					build();

				gptWideSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[728, 90]]).
					addSize([760, 0], [[728, 90], [320, 50], [300, 50]]).
					addSize([0, 0], [[320, 50], [300, 50]]).
					build();

				gptWideRectangleSizeMapping = googletag.sizeMapping().
					addSize([992, 0], []).
					addSize([760, 0], [[320, 50], [300, 50], [300, 250]]).
					addSize([0, 0], [[320, 50], [300, 50], [300, 250]]).
					build();

				gptMobileSizeMapping = googletag.sizeMapping().
					addSize([0, 0], [[320, 50], [300, 50]]).
					build();

				gptFluidMapping = googletag.sizeMapping().
					addSize([0, 0], ['fluid']).
					build();

				gptPrestitialMapping = googletag.sizeMapping().
					addSize([760, 0], [[1, 1]]).
					build();
			}
			else {
				//JBS-CELL-LANCET
				gptLeaderboardSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[728, 90]]).
					addSize([768, 0], [[728, 90]]).
					addSize([0, 0], [[320, 50], [300, 50]]).
					build();

				gptHomePageSidebarMapping = googletag.sizeMapping().
					addSize([2090, 795], [[300, 600], [300, 250], [160, 600], [120, 600]]).
					addSize([1910, 795], [[160, 600], [120, 600]]).
                    addSize([1740, 795], [[120, 600]]).
					addSize([0, 0], []).
					build();

				gptCellHomePageSidebarMapping = googletag.sizeMapping().
                    addSize([2180, 795], [[300, 600], [160, 600], [120, 600], [300, 250], [336, 280]]).
					addSize([2180, 445], [[336, 280]]).
                    addSize([2090, 795], [[300, 600], [160, 600], [120, 600], [300, 250]]).
                    addSize([2090, 445], [[300, 250]]).
					addSize([1830, 795], [[160, 600], [120, 600]]).
                    addSize([1740, 795], [[120, 600]]).
					addSize([0, 0], []).
					build();

				gptSideBarSizeMapping = googletag.sizeMapping().
					addSize([1140, 795], [[300, 600], [160, 600], [120, 600], [300, 250]]).
					addSize([1140, 445], [[300, 250]]).
					addSize([0, 0], []). 				
					build();

				gptBoomBoxSizeMapping = googletag.sizeMapping().
					addSize([768, 0], [[300, 250], [336, 280]]).
					addSize([0, 0], []). 						
					build();

				gptFluidMapping = googletag.sizeMapping().
					addSize([0, 0], ['fluid']).
					build();

				gptPrestitialMapping = googletag.sizeMapping().
					addSize([600, 700], [[1, 1]]).
					addSize([0, 0], []).
					build();

				gptInterscrollerMapping = googletag.sizeMapping().
					addSize([1140, 445], []).
					addSize([1024, 768], [[1024, 768]]).
					addSize([768, 1024], [[768, 1024]]).
					addSize([480, 320], [[480, 320]]).
					addSize([320, 480], [[320, 480]]).
					build();
			}
			//***********************************************
			//End Define Size mapping based on client Viewport
			//***********************************************

			//***********************************************
			//Start detection for ad slots on page
			//***********************************************
			var gptMasterPage = gpt_CheckForNULL($('#gptPage').val());
			var gptMasterSite;

			if (gpt_AdFile === 'PU') {
				gptMasterSite = $('#gptSite').val();
			}
			else {
				//JBS-CELL-LANCET
				gptMasterSite = "dev";

				if (gpt_IsProd) {
					gptMasterSite = gpt_CheckForNULL($('#gptSite').val());

					//If there is no master site, check for a defaultsite
					if (gptMasterSite == null) {
						gptMasterSite = gpt_CheckForNULL($('#gptDefaultSite').val());
					}
				}
				else {
					//Dev ENV
					//If the page has no gptSite Value, assume no value which will disable ads
					if (gpt_CheckForNULL($('#gptSite').val()) === null) {
						gptMasterSite = null;
					}
				}

				//If there is no master Page, check for a Page using the URL
				if (gptMasterPage === null) {
					gptMasterPage = GPT_GetPageByURL();
				}
			}

			$(".gpt-ad").each(function (index) {
				var gptPage = '';
				var adSizeMapping = '';
				var gptPOS;
				var gptTC;
				var gptNative;
				var adId;
				var gptSite;
				var tempSlot = null;


                if (gpt_IsProd) {
					if (gpt_AdFile !== "CELLJOURNAL") {
						//Check if the ad slot has it's own site defined
						if (typeof $(this).data('site') === 'undefined') {
							gptSite = gptMasterSite;
						}
						else {
							gptSite = gpt_CheckForNULL($(this).data('site'));
						}
					}
					else {
						gptSite = gptMasterSite;
					}
                }
                else {
					gptSite = gptMasterSite;
                }



				if (gpt_AdFile === 'PU') {
					adId = $(this).attr('id');
				}
				else {
					//JBS-CELL-LANCET
					adId = 'gpt-ad-' + index + 1;
				}

				if (gpt_CheckForNULL(gptSite) !== null) {
					gptSite_Debug = gptSite;

					gptPOS = parseInt($(this).data('pos')) || -1;
					if (gptPOS === -1) {
						gptPOS = index + 1;
					}

					//Check if the ad slot has it's own page defined
					gptPage = $(this).data('page');
					if (gpt_CheckForNULL(gptPage) == null) {
						//Use the Master Page
						gptPage = gptMasterPage;
					}

					//If the page is not null, preceed it with a slash
					if (gptPage !== null) {
						gptPage = '/' + gptPage;
					}
					else {
						gptPage = '';
					}
					gptPage_Debug = gptPage;


					gptTC = $(this).data('tc');
					if (typeof gptTC === "undefined") {
						gptTC = '';
					}

					gptNative = $(this).data('native');
					if (typeof gptNative === "undefined") {
						gptNative = '';
					}

					//Determine sizemapping based on data-adsize
					if (gpt_AdFile === 'PU') {
						switch ($(this).data('adsize')) {
							case 'tall':
								adSizeMapping = gptTallSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600]], adId);
								break;
							case 'rectangle':
								adSizeMapping = gptRectangleSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250]], adId);
								break;
							case 'rectangleshort':
								adSizeMapping = gptRectangleShortSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250], [320, 50], [300, 50]], adId);
								break;
							case 'wide':
								adSizeMapping = gptWideSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[728, 90], [320, 50], [300, 50]], adId);
								break;
							case 'widerectangle':
								adSizeMapping = gptWideRectangleSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[320, 50], [300, 50], [300, 250]], adId);
								break;
							case 'mobile':
								adSizeMapping = gptMobileSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[320, 50], [300, 50]], adId);
								break;
							case 'fluid':
								adSizeMapping = gptFluidMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, 'fluid', adId);
								break;
							case 'prestitial':
								adSizeMapping = gptPrestitialMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[1, 1]], adId);
								break;
						}
					}
					else {
						//JBS-CELL-LANCET
						switch ($(this).data('adsize')) {
							case 'cell-leaderboard':
							case 'leaderboard':
								adSizeMapping = gptLeaderboardSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[728, 90], [320, 50], [300, 50]], adId);
								break;
							case 'bottom':
								if (gpt_ResponsiveHomepage) {
									adSizeMapping = gptHomePageSidebarMapping;
									tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600], [300, 250]], adId);
								}
								else {
									adSizeMapping = gptBoomBoxSizeMapping;
									tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250]], adId);
								}
								break;
							case 'boombox':
								if (gpt_AdFile === "CELLJOURNAL" || gpt_AdFile == "CELLBUCKET") {
									if (gpt_ResponsiveHomepage) {
										adSizeMapping = gptCellHomePageSidebarMapping;
										tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600], [300, 250], [336, 280]], adId);
									}
									else {
										adSizeMapping = gptBoomBoxSizeMapping;
										tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250], [336, 280]], adId);
									}
								}
								else {
									adSizeMapping = gptBoomBoxSizeMapping;
									tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250]], adId);
								}
								break;
							case 'interscroller':
								adSizeMapping = gptInterscrollerMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[320, 480], [480, 320], [768, 1024], [1024, 768]], adId);
								break;
							case 'sidebar':
							case 'skyscraper':
								adSizeMapping = gptSideBarSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600], [300, 250]], adId);
								break;
							case 'native':
							case 'fluid':
								adSizeMapping = gptFluidMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, 'fluid', adId);
								break;
							case 'prestitial':
								adSizeMapping = gptPrestitialMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, 'fluid', adId);
								break;
						}
					}
				}

				//Ignore any invalid adsizes that could be on the page
				if (adSizeMapping !== '') {
					$(this).addClass('gpt-ad-defined');
					if (gpt_AdFile !== 'PU') {
						$(this).attr('id', adId);
					}
					tempSlot.defineSizeMapping(adSizeMapping).addService(googletag.pubads()).setTargeting("pos", gptPOS);

					if (gptTC !== '') {
						tempSlot.setTargeting("tc", gptTC);
					}
					if (gptNative !== '') {
						tempSlot.setTargeting("native", gptNative);
					}
					gptEvent_Debug = gptEvent_Debug + $(this).data('adsize') + ' Requested<br>';
				}
				else {
					gptInvalidAdSize_Debug = gptInvalidAdSize_Debug + "Invalid Size Found, Id: " + $(this).prop('id') + ', adSize: ' + $(this).data('adsize') + '<br>';
					$(this).remove();
				}
			});
			if (gpt_CheckForNULL(gptInvalidAdSize_Debug) === null) {
				gptInvalidAdSize_Debug = "None Detected";
			}
			//***********************************************
			//End detection for ad slots on page
			//***********************************************

			//***********************************************
			//Start page level targeting
			//***********************************************
			if (gpt_AdFile !== 'PU') {
				//Logged in or IP Access
				gpt_TargetAccessType();

				//Check for Screenshot Client
				gpt_CheckSSC();
			}

			//Check for DFP Test
			gpt_GetDFPTestId();

			//Check for Additional Targeting
			gpt_GetAdditionalTargeting();

			if (gpt_IsProd && gptContextualEnabled) {
				var gpt_ContextualRetry = false;
				//Check for Contextual Targeting
				if (gs_channels !== "DEFAULT") {
					var gpt_ContextualCategories = Array.from(gs_channels);
					for (var i = 0; i < gpt_ContextualCategories.length; i++) {
						if (gpt_ContextualCategories[i] == 'gx_retry') {
							gpt_ContextualRetry = true;
						}
						if (!gpt_ContextualCategories[i].startsWith("custom_")) {
							//Remove any tag that is not custom
							gpt_ContextualCategories.splice(i, 1);
							i--;
						}
						else {
							//Strip the 'custom_' from the tag
							gpt_ContextualCategories[i] = gpt_ContextualCategories[i].replace("custom_", "")
						}
					}
					if (gpt_ContextualCategories.length > 0) {
						googletag.pubads().setTargeting("gs_cat", gpt_ContextualCategories);
					}
					else {
						gpt_ContextualCategories = 'none';
					}
					if (gpt_ContextualRetry) {
						gpt_ContextualCategories = "*Indexing*";
					}
					gptContextualResult_Debug = gpt_ContextualCategories;
				}
				else {
					gptContextualResult_Debug = 'No segments found';
				}
			}
			else {
				//Not Prod || Contextual not enabled
				gptContextualResult_Debug = 'Not enabled';
				
				if (!gpt_IsProd) {
                    var gpt_DevSite = gpt_CheckForNULL($('#gptSite').val(), "dev");
                    $('#gpt-Variables').append('<input type="hidden" id="gptEnvironment" value="' + gpt_DevSite + '">');
                    gpt_Target("Environment");
                }

			}
			//***********************************************
			//End page level targeting
			//***********************************************

			//Enable google dfp services
			if (parseInt($('#gptLazyLoading').val()) === 1) {
				googletag.pubads().enableLazyLoad({
					fetchMarginPercent: 200,
					renderMarginPercent: 0,
					mobileScaling: 2.0
				});
			}
			else {
				googletag.pubads().enableSingleRequest();
			}

			//Add Listener for RenderEnded
			googletag.pubads().addEventListener('slotRenderEnded', function (event) {
				if (!event.isEmpty) {
					if (typeof LocalAdLoaded === "function") {
						LocalAdLoaded(event);
					}
					if (gpt_ResponsiveHomepage) {
						if (typeof positionAd === "function") {
							positionAd();
						}
					}
					if (event.size[0] == "480" || event.size[1] == "480") {
						$('.gpt-InScrollBox').css('display', 'block');
						gptEvent_Debug = gptEvent_Debug + 'Interscroller ' + event.size + ' rendered: ' + gpt_GetElapsedTime() + '<br>';
					}
					else {
						gptEvent_Debug = gptEvent_Debug + event.size + ' rendered: ' + gpt_GetElapsedTime() + '<br>';
					}

				}
			});

			if (gpt_AdFile !== 'PU') {
				googletag.pubads().collapseEmptyDivs();
			}
			googletag.enableServices();

			//Populate Defined Ad Slots
			$(".gpt-ad-defined").each(function (index) {
				var adId = $(this).attr('id');
				googletag.cmd.push(function () { googletag.display(adId); });
			});

			gptStatus_Debug = 'GPT Complete';
		});
	}
	//--------------------------------------------------------------------------
	function gpt_DisplayDebug() {
		//Clone the template and append to Body
		$(document.body).append($('#gpt-DebugDiv').clone());
		$('#gpt-DebugDiv').remove();

		//Populate the Values
		gpt_LoadDebugValues();

		//Display and scroll to debug section
		$('.gpt-DebugDiv').css('display', 'block');
		window.scrollTo(0, document.body.scrollHeight);
	}
	//--------------------------------------------------------------------------
	function gpt_LoadDebugValues() {
		$('.gpt-DebugBtn').blur();
		$('.gptVersion_Debug').text(gpt_Version);
		$('.gptAdFile_Debug').html(gpt_AdFile);
		$('.gptEvent_Debug').html(gptEvent_Debug);
		$('.gptCustomTargeting_Debug').html(gptCustomTargeting_Debug);
		$('.gptSite_Debug').text(gptSite_Debug);
		$('.gptContextualResult_Debug').text(gptContextualResult_Debug);
		$('.gptContextualInit_Debug').text(gptContextualInit_Debug);
		$('.gptContextualURL_Debug').text(gptContextualURL_Debug);
		$('.gptContextualReady_Debug').text(gptContextualReady_Debug);
		$('.gptMoatReady_Debug').text(gptMoatReady_Debug);
		$('.gptMoatInit_Debug').text(gptMoatInit_Debug);
		$('.gptTestENV_Debug').text(gptTestENV_Debug);
		$('.gptDisableAds_Debug').text(gptDisableAds_Debug);
		$('.gptStatus_Debug').text(gptStatus_Debug);
		$('.gptPage_Debug').text(gptPage_Debug);
		$('.gptInvalidAdSize_Debug').html(gptInvalidAdSize_Debug);
	}
	//--------------------------------------------------------------------------
	function gpt_GrapeShotDebug() {
		$('.gpt-DebugBtn').blur();
		window.open("http://admin-elsevier.grapeshot.co.uk/custom/bookmarklet.cgi?blocking=1&vn=2&url=" + encodeURIComponent(document.location), "GRAPESHOT_POPUP", "width=650,height=600,scrollbars=yes,screenX=40,screenY=50,menubar=no,toolbar=no,resizable=yes", "Custom").focus();
	}
	//--------------------------------------------------------------------------
	function gpt_ShowConsole() {
		$('.gpt-DebugBtn').blur();
		googletag.openConsole();
	}
	//--------------------------------------------------------------------------
</script>
        </div>

            
                </div>
            

        

    

        


    

            

            

            <div data-widget-id="912135d6-7135-4d45-b457-b503cacc3c8b" data-widget-name="ux3-layout-widget"><header class="header base fixed">
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none adplaceholder adplaceholder--header js__adToHide widget-horizontalAd" id="9d6062d7-fad6-4e64-81cc-7dd25d99b23d">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-horizontalAd "><div class="horizontalAdContainer">
    <div class="horizontalAdLabelContainer">
        <div class="horizontalAdLabel">
                <img src="/templates/jsp/_style2/_marlin/images/bg_adHoriz.gif" alt="Advertisement">
        </div>
    </div>
    <div class="horizontalAd">
        





        <div class="pb-ad">
            <!-- Beginning Async AdSlot 1 for Ad unit els.cellpress/cell  ### size: [[728,90]] -->
<div class="gpt-ad" data-adsize="cell-leaderboard" data-page="" data-site="cell" data-pos="1"></div>
<!-- End AdSlot 1 -->
        </div>
    

    </div>
</div></div>
                </div>
                </div>

        










        <div data-widget-def="ux3-layout-widget" data-widget-id="3a83e2a6-2e7e-4931-892c-4ef43b6e3312" class="header__wrapper clearfix" data-db-parent-of="quickSearch"><div id="brandingRow"><a href="/" data-aa-name="science that inspires" data-aa-region="cp global header" title="Visit Cell.com">Science that inspires</a><p id="brandingRowRight"><a href="/" data-aa-name="cell press endorsement" data-aa-region="cp global header" title="Visit Cell.com">A Cell Press journal</a></p></div>
        
    

        


    

            

            

            <div data-widget-id="3a83e2a6-2e7e-4931-892c-4ef43b6e3312" data-widget-name="ux3-layout-widget"><div class="col-xs-6 header__left-side">
    

        


    

            

            
                <a href="/home" id="cpIconLnk" data-aa-name="cell press logo" data-aa-region="cp global header" title="Cell Press" target="_blank"><svg id="cp_logo" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 100 98.1" style="enable-background:new 0 0 100 98.1;" xml:space="preserve" height="100" width="98.1"><path style="fill:#FFFFFF;" d="M69.09,0.09c-4.01-0.6-7.83,1.94-8.78,5.89l-3.14,13.11c-0.76,3.19-3.44,5.56-6.7,5.93l-18.6,2.13          C11.83,29.31-2.53,47.66,0.37,67.78c2.3,15.96,15.18,27.77,30.54,30.08c4.01,0.6,7.83-1.94,8.78-5.89l3.14-13.11 c0.76-3.19,3.44-5.56,6.69-5.93l18.6-2.13c20.04-2.17,34.41-20.52,31.5-40.65C97.32,14.21,84.45,2.39,69.09,0.09z M65.06,47.2  l-11.75,1.4c-1.98,0.21-3.62,1.65-4.08,3.59l-3.43,14.3c-1.01,4.22-4.5,7.39-8.8,8c-6.66,0.72-12.63-4.2-13.15-10.92                c-0.51-6.5,4.6-12.11,11.08-12.81l11.75-1.39c1.98-0.22,3.62-1.65,4.08-3.59l3.43-14.3c1.01-4.22,4.5-7.39,8.8-8          c6.66-0.72,12.63,4.19,13.15,10.92C76.66,40.89,71.55,46.5,65.06,47.2z"></path></svg></a><div class="journal-logos">
            

            <span data-widget-id="35d8d734-3aa0-4ec8-9e2d-14babe803594" data-widget-name="ux3-general-image"><a href="/cell/home" title="Cell"><img alt="Cell" src="/pb-assets/ux3/logos/cell/Cell_white-1532516611593.svg"></a></span>

            
                </div>
            

        

    

        


    

            

            
                <div class="journal-logos">
            

            <span data-widget-id="3ffcbf1f-79fd-4ba3-85b5-bef10a63475f" data-widget-name="ux3-general-image"><a href="https://www.cell.com/open-access?utm_source=oa-label&amp;__hstc=109691226.e4e812b53ff324eccbdbdb2e4fb9a6ab.1644503737405.1644524234622.1644587109507.4&amp;__hssc=109691226.7.1644587109507&amp;__hsfp=3253153019" title="This journal offers authors two options (open access or subscription) to publish research"><img alt="This journal offers authors two options (open access or subscription) to publish research" src="/pb-assets/ux3/logos/cell/Supports_open_access_border-1645554937897.svg"></a></span>

            
                </div>
            

        
</div><div class="col-xs-6">
    

        


    

            

            

            <div data-widget-id="2555f6db-b333-4edb-8b15-ef9213f1f0a5" data-widget-name="ux3-layout-widget"><div class="header__nav row">
    

        


    

            

            

            <ul data-widget-id="52432d24-0331-4c9d-85f3-2033edc7208b" data-widget-name="UX3ProfileLinksWidget" class="rlist rlist--inline profile-links"><li class="profile-links__submit"><a data-adobe-name="Publish" class="profile-links__submit__btn" title="Submit" href="https://www.editorialmanager.com/cell" target="_self" data-aa-conversiondriver="jbs:global-header:submitpage" data-profile-link="true">Submit</a></li><li class="profile-links__login-dropdown"><div class="profile-links__user-list__wrapper"><button data-db-target-for="loginmenu-31843" aria-expanded="false" aria-controls="login-links-31843" title="Log in links dropdown" class="profile-links__username"><span>Log in</span></button><div data-db-target-of="loginmenu-31843" id="login-links--31843" class="profile-links__user-list__holder dropBlock__holder"><ul class="rlist profile-links__list"><li class="profile-links__login"><a data-adobe-name="Log In" class="profile-links__login__btn" title="Log in to Cell.com" href="https://www.cell.com/action/idLogin?type=login&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2820%2930339-1" target="_self" data-profile-link="true">Log in</a></li><li class="profile-links__institution"><a data-adobe-name="Get institutional access" class="profile-links__institution__btn" title="Institutional access" href="https://www.cell.com/action/idLogin?type=institution&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2820%2930339-1" target="_self" data-profile-link="true">Get Institutional Access</a></li></ul></div></div></li><li class="profile-links__register"><a data-adobe-name="Register" class="profile-links__register__btn" title="Register a username and password for free" href="https://www.cell.com/action/idLogin?type=registration&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2820%2930339-1" target="_self" data-profile-link="true">Register</a></li><li class="hidden-lg"> <div class="profile-links__controls"> <button aria-expanded="false" aria-controls="links-dialog-31843" data-opens="#links-dialog-31843" class="profile-links__btn--open"><span class="sr-only">open links dialog</span><i aria-hidden="true" class="icon-gizmo-person"></i></button></div><dialog id="links-dialog-31843" aria-modal="true"><button data-closes="#links-dialog-31843" class="profile-links__btn--close"><span class="sr-only">close links dialog</span><i aria-hidden="true" class="icon-close-solid"></i></button><ul>   <li class="profile-links__submit"><a data-adobe-name="Publish" class="profile-links__submit__btn" title="Submit" href="https://www.editorialmanager.com/cell" target="_self" data-aa-conversiondriver="jbs:global-header:submitpage" data-profile-link="true">Submit</a></li><li class="profile-links__login"><a data-adobe-name="Log In" class="profile-links__login__btn" title="Log in to Cell.com" href="https://www.cell.com/action/idLogin?type=login&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2820%2930339-1" target="_self" data-profile-link="true">Log in</a></li><li class="profile-links__institution"><a data-adobe-name="Get institutional access" class="profile-links__institution__btn" title="Institutional access" href="https://www.cell.com/action/idLogin?type=institution&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2820%2930339-1" target="_self" data-profile-link="true">Get Institutional Access</a></li><li class="profile-links__register"><a data-adobe-name="Register" class="profile-links__register__btn" title="Register a username and password for free" href="https://www.cell.com/action/idLogin?type=registration&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2820%2930339-1" target="_self" data-profile-link="true">Register</a></li></ul></dialog></li></ul>

            

        
</div></div>

            

        
</div></div>

            

        

        </div>
    

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget csasInstitutionBanner none  widget-none" id="51b7fa33-2137-4f58-9e98-9746c4400513" style="display: none;">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none "><div class="ui-state-highlight ui-corner-all csas-institution-banner hidden" data-doi="10.1016/j.cell.2020.03.046">
    <span class="ui-icon ui-icon-close banner-close"></span>
    <p>
        Access provided by <span class="accountName"></span>
    </p>
</div>

<style>
    .csas-institution-banner {
        padding: 10px 0 0 10px;
        margin: 5px;
    }

    .csas-institution-banner.ui-state-highlight .ui-icon {
        background-image: url("/templates/jsp/css/jquery-ui-1.10.2/base/images/ui-icons_454545_256x240.png");
        margin-right: 5px;
        margin-top: -4px;
    }

    .csas-institution-banner .banner-close {
        float: right;
    }
</style>

<script>

    function resetBanner(doi) {
        $.get('/action/getAccount?doi=' + doi, function(data) {
            if (data) {
                $(".csas-institution-banner p .accountName").html(data);
                $(".csasInstitutionBanner").fadeIn();
                $(".csas-institution-banner").fadeIn();
            }
        });
    }

    var flag = 0;
    $(document).ready(function() {
        if(!$(".csas-institution-banner").is(":visible")){
            $(".csasInstitutionBanner").hide();
        }

        $(".banner-close").click(function() {
            $.get("/action/dismissBanner", function(data) {
                $(".csasInstitutionBanner").fadeOut(function () {
                    if(typeof UX !== 'undefined') {
                        UX.pageBody.init();
                    }
                });
            });
        });

        $(".articleContent .tabs li[role='tab'].showFullText").click(function() {
           flag = 1;
        });

        $(document).on("reloadTabsContent", function() {
            var isSelected = $(".articleContent .tabs li[role='tab'].showFullText").hasClass("active");
            if (isSelected) {
                var isHidden =  $(".csas-institution-banner").is(".hidden");
                if (!isHidden) {
                    resetBanner($(".csas-institution-banner").data("doi"));
                    console.log("fadeIn");
                }
            } else {
                $(".csas-institution-banner").fadeOut();
                console.log("fadeOut");
            }
            flag = 0;
        });

        $(document).on("click", ".articleContent .tabs li[role='tab']", function() {

        });
    });
</script></div>
                </div>
                </div>

        
</header></div>

            

        










        <div data-widget-def="graphQueryWidget" data-widget-id="2a125dae-6dbc-4e76-a75e-3e563f1d3a1d" class="ux3">
        
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget graphQueryWidget none top-padding-handler noMargin axel-mega-menu widget-none widget-compact-all fonts-loaded" id="2a125dae-6dbc-4e76-a75e-3e563f1d3a1d" style="padding-top: 115.594px;">
                
                
                    
                

                <div class="wrapped " id="CP TOP LEVEL NAV MENU">
                    
                    <div class="widget-body body body-none  body-compact-all"><nav id="mega-menu-nav" role="navigation" class="mega-menu-nav">
  <!--.skip//a(href='#searchText' aria-label='Skip menu') Skip menu
  -->
  <div class="mega-menu-nav__wrapper">
    <button class="mega-menu__toggle"><span class="sr-only">Main menu</span><i aria-hidden="true" class="icon-gizmo-hamburger"></i></button>
    <ul role="menubar" class="mega-menu rlist">
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-1" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-1" data-item-link="main-menu-link-1" data-index="0" class="mega-menu__item-link menu-link"><span>This journal</span></a>
        <div id="main-menu-panel-1" aria-labelledby="main-menu-link-1" data-item-wrapper="main-menu-link-1" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <div aria-label="Menu dropzone section" tabindex="-1" class="listDropZone">
    

        


    

            

            

            <div class="cover-image"><div class="cover-image__thumb"><a href="/issue/S0092-8674(24)X0012-4" class="menu-link"><img src="/cms/asset/atypon:cms:attachment:img:d5e6:rev:1748270185670-389:pii:S0092867424X00124/cover.tif.jpg" alt="Cover Image - Cell, Volume 188, Issue 11"></a></div><div class="cover-image__meta"></div></div>

            

        
</div>
            </li>
            <li style="grid-column: 2/3" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-1-2" class="sub-menu__heading">Read
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/newarticles" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Online now</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/current" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Current issue</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/archive" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Archive</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/action/showPreferences?menuTab=Alerts" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Publication alerts</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/sneakpeek" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Sneak Peek - preprints</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 3/5" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-1-3" class="sub-menu__heading">About
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-2">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/advisory-board" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Advisory board</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/aims" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Aims and scope</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/contact" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Contact</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/editors-and-staff" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Editors and staff</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/editorial-ethics" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Ethics</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/information-for-authors/permissions" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Permissions</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/reviewers" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>Reviewer instructions</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-2" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-2" data-item-link="main-menu-link-2" data-index="1" class="mega-menu__item-link menu-link"><span>Journals</span></a>
        <div id="main-menu-panel-2" aria-labelledby="main-menu-link-2" data-item-wrapper="main-menu-link-2" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/5" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-2-1" class="sub-menu__heading">Life &amp; medical sciences
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-4">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/home" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cancer-cell/home" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Cancer Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-chemical-biology" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Chemical Biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-genomics/home" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Genomics</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-host-microbe/home" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Host &amp; Microbe</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-metabolism/home" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Metabolism</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports/home" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-medicine/home" data-index="7" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Medicine</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-stem-cell/home" data-index="8" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Stem Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-systems/home" data-index="9" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Systems</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/current-biology/home" data-index="10" role="menuitem" class="sub-menu__item-link menu-link"><span>Current Biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/developmental-cell/home" data-index="11" role="menuitem" class="sub-menu__item-link menu-link"><span>Developmental Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/immunity/home" data-index="12" role="menuitem" class="sub-menu__item-link menu-link"><span>Immunity</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/med/home" data-index="13" role="menuitem" class="sub-menu__item-link menu-link"><span>Med</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-cell/home" data-index="14" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/neuron/home" data-index="15" role="menuitem" class="sub-menu__item-link menu-link"><span>Neuron</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/structure/home" data-index="16" role="menuitem" class="sub-menu__item-link menu-link"><span>Structure</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/ajhg/home" data-index="17" role="menuitem" class="sub-menu__item-link menu-link"><span>American Journal of Human Genetics (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/biophysj/home" data-index="18" role="menuitem" class="sub-menu__item-link menu-link"><span>Biophysical Journal (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/biophysreports/home" data-index="19" role="menuitem" class="sub-menu__item-link menu-link"><span>Biophysical Reports (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/hgg-advances/home" data-index="20" role="menuitem" class="sub-menu__item-link menu-link"><span>Human Genetics and Genomics Advances (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-plant/home" data-index="21" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Plant (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/molecular-therapy/home" data-index="22" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/methods/home" data-index="23" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy Methods &amp; Clinical Development (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/nucleic-acids/home" data-index="24" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy Nucleic Acids (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/oncology/home" data-index="25" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy Oncology (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/plant-communications/home" data-index="26" role="menuitem" class="sub-menu__item-link menu-link"><span>Plant Communications (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/stem-cell-reports/home" data-index="27" role="menuitem" class="sub-menu__item-link menu-link"><span>Stem Cell Reports (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/biochemical-sciences/home" data-index="28" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Biochemical Sciences</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/cancer/home" data-index="29" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Cancer</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/cell-biology/home" data-index="30" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Cell Biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/ecology-evolution/home" data-index="31" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Ecology &amp; Evolution</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/endocrinology-metabolism/home" data-index="32" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Endocrinology &amp; Metabolism</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/genetics/home" data-index="33" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Genetics</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/immunology/home" data-index="34" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Immunology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/microbiology/home" data-index="35" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Microbiology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/molecular-medicine/home" data-index="36" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Molecular Medicine</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/neurosciences/home" data-index="37" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Neurosciences</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/parasitology/home" data-index="38" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Parasitology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/pharmacological-sciences/home" data-index="39" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Pharmacological Sciences</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/plant-science/home" data-index="40" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Plant Science</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 5/6" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-2-2" class="sub-menu__heading">Physical sciences &amp; engineering
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-biomaterials/home" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Biomaterials</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-physical-science/home" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Physical Science</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/chem/home" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Chem</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/chem-catalysis/home" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Chem Catalysis</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/device/home" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Device</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/joule/home" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Joule</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/matter/home" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>Matter</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/newton/home" data-index="7" role="menuitem" class="sub-menu__item-link menu-link"><span>Newton</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/chemistry/home" data-index="8" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Chemistry</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 6/7" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-2-3" class="sub-menu__heading">Multidisciplinary
              </h4>
              <ul role="menubar" data-index="2" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-methods/home" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Methods</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-sustainability/home" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Sustainability</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/heliyon/home" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Heliyon</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/iscience/home" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>iScience</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/one-earth/home" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>One Earth</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/patterns/home" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Patterns</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/star-protocols/home" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>STAR Protocols</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/nexus/home" data-index="7" role="menuitem" class="sub-menu__item-link menu-link"><span>Nexus (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/the-innovation/home" data-index="8" role="menuitem" class="sub-menu__item-link menu-link"><span>The Innovation (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/biotechnology/home" data-index="9" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Biotechnology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/cognitive-sciences/home" data-index="10" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Cognitive Sciences</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-3" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-3" data-item-link="main-menu-link-3" data-index="2" class="mega-menu__item-link menu-link"><span>Publish</span></a>
        <div id="main-menu-panel-3" aria-labelledby="main-menu-link-3" data-item-wrapper="main-menu-link-3" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-3-1" class="sub-menu__heading">Authors
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="https://www.editorialmanager.com/cell?__hstc=25856994.7201670a4397fa03148684920d4a8d38.1672762013831.1694716659709.1694718909620.146&amp;__hssc=25856994.15.1694718909620&amp;__hsfp=1366844671" data-index="0" target="_blank" role="menuitem" class="sub-menu__item-link menu-link"><span>Submit article
                      <div class="external-icon"></div><span class="sr-only">External Link</span></span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/information-for-authors" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Information for authors</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/open-access" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Open access</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/multi-journal-submission" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Multi-Journal Submission</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/information-for-authors/star-authors-guide" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>STAR Methods</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 2/3" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-3-2" class="sub-menu__heading">Reviewers
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/reviewers" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Information for reviewers</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-4" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-4" data-item-link="main-menu-link-4" data-index="3" class="mega-menu__item-link menu-link"><span>News &amp; events</span></a>
        <div id="main-menu-panel-4" aria-labelledby="main-menu-link-4" data-item-wrapper="main-menu-link-4" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/newsroom" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Newsroom</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/events" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Events</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/symposia" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Symposia</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/webinars" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Webinars</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/consortia" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Consortia Hub</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-5" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-5" data-item-link="main-menu-link-5" data-index="4" class="mega-menu__item-link menu-link"><span>About Cell Press</span></a>
        <div id="main-menu-panel-5" aria-labelledby="main-menu-link-5" data-item-wrapper="main-menu-link-5" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-5-1" class="sub-menu__heading">Cell Press
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/about" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>About Cell Press</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/contact" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Contact us</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="https://service.elsevier.com/app/overview/cell-press/" data-index="2" target="_blank" role="menuitem" class="sub-menu__item-link menu-link"><span>Help &amp; support
                      <div class="external-icon"></div><span class="sr-only">External Link</span></span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/open-access" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Open access at Cell Press</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/diversity" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Inclusion and diversity</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-press-50" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>50th anniversary</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 2/3" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-5-2" class="sub-menu__heading">Hubs
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/biotechnology-bioengineering" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Biotechnology and bioengineering</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-press-chemistry" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Chemistry</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/clinical-translational" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Clinical and translational</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/covid-19" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>COVID-19</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/fundamental-biology" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Fundamental biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/sustainability" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Sustainability</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 3/4" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-5-3" class="sub-menu__heading">Careers
              </h4>
              <ul role="menubar" data-index="2" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="https://relx.wd3.myworkdayjobs.com/en-US/ElsevierJobs" data-index="0" target="_blank" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Press careers
                      <div class="external-icon"></div><span class="sr-only">External Link</span></span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
    </ul>
    <div class="mega-menu-nav__quick-search">
    

        


    

            

            

            <div data-widget-id="a1f05748-166f-41e0-b01a-ca8ac1cc95d4" data-widget-name="UX3QuickSearchWidget" class="quick-search"><div class="dropBlock"><form action="https://www.cell.com/action/doSearch?journalCode=cell" name="quickSearchBar" method="get" title="Quick Search"><input id="type" type="hidden" value="quicksearch" name="type"><div class="quick-search__toggle autoCompleteWrapper"><input id="searchText" type="text" name="text1" placeholder="Search for..." enterkeyhint="search" autocomplete="off" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" data-group-titles-conf="3" data-publication-items-conf="3" aria-label="Enter a search term" class="searchText searchText-autocomplete ui-autocomplete-input"><button type="submit" title="Search" aria-label="Submit the search form" class="btn quick-search__button icon-search"><span>Go search</span></button><a href="https://www.cell.com/search/advanced?SeriesKey=cell&amp;ISSN=0092-8674&amp;journalCode=cell" title="Advanced search" class="quick-search__advanced-link">Advanced search</a><ul id="ui-id-1" tabindex="0" class="ui-menu ui-widget ui-widget-content ui-autocomplete ui-front" unselectable="on" style="display: none;"></ul></div><div class="quick-search__box"><div class="quick-search__box-inner"><div class="quick-search__box-wrapper"><select name="field1" aria-label="Select a search area"><option value="AllField">All content</option><option value="Title">Article title</option><option value="Contrib">Authors</option><option value="Keyword">Keywords</option><option value="Abstract">Abstract</option><option value="AbstractTitleKeywordFilterField">Article title, abstract, keywords</option></select><input type="hidden" name="journalCode" value="cell"><div class="quick-search__scope-options"><span>Search within:</span><label title="This journal" class="radio--primary"><input type="radio" name="SeriesKey" id="searchScopeSeries-a1f05748-166f-41e0-b01a-ca8ac1cc95d4" title="This journal" value="cell" checked="checked" aria-label="Search within current journal"><span>This journal</span></label><label title="Full site" class="radio--primary"><input type="radio" name="SeriesKey" id="searchScopeFullSite-a1f05748-166f-41e0-b01a-ca8ac1cc95d4" value="" title="Full site" aria-label="Search within full site"><span>Full site</span></label></div><a href="https://www.cell.com/search/advanced?SeriesKey=cell&amp;ISSN=0092-8674&amp;journalCode=cell" title="Advanced search" class="quick-search__advanced-link">Advanced search</a></div><a href="javascript:void(0)" aria-label="Close search options" title="Close search options" class="quick-search__box-close"><i aria-hidden="true" class="icon-tools_close"></i></a></div></div><div class="quick-search__alert hidden"><p>Please enter a term before submitting your search.</p><a href="#" class="quick-search__alert--close"> Ok</a></div></form></div></div>

            

        
</div>
  </div>
</nav>
<div id="page-body-id" class="page-body-flag"></div></div>
                </div>
                </div>

        

        </div>
    

    

        


    

            

            

            <div data-widget-id="f8c6daf9-9df1-4fd9-ba3a-7f5d7b3d4466" data-widget-name="UX3HTMLWidget"><style>
.journal-logos:nth-of-type(2) img {
max-height: 30px !important;
}
.journal-logos:nth-of-type(3) img {
max-height: 30px !important;
}
.ux3 .journal-logos + .journal-logos {
margin-left: 30px;
}
.header__wrapper.clearfix .col-xs-6:first-of-type {
width: 55%;
}
.header__wrapper.clearfix .col-xs-6:nth-of-type(2) {
width: 45%;
}
@media screen and (max-width: 1000px) {
.ux3 .journal-logos + .journal-logos {
display: none;
}
.header__wrapper.clearfix .col-xs-6:first-of-type {
width: 50%;
}
.header__wrapper.clearfix .col-xs-6:nth-of-type(2) {
width: 50%;
}
}

.ux3 .journal-logos:first-of-type {
   max-width: 500px;
}

.profile-links {
   white-space: nowrap;
}

@media only screen and (min-width: 740px) and (max-width: 1600px) {
   .journal-logos:nth-of-type(2) img {
      max-width: 250px !important;
   }
}

@media only screen and (min-width: 1024px) and (max-width: 1600px) {
   .header__wrapper.clearfix .col-xs-6:first-of-type {
      width: 60% !important;
   }
   .header__wrapper.clearfix .col-xs-6:nth-of-type(2) {
      width: 40% !important;
   }
}

@media only screen and (max-width: 1240px) {
   .ux3 .journal-logos + .journal-logos {
      display: none !important;
   }
}

</style></div>

            

        
</div></div>

            
                </div>
            

        

        </div>
    

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget axelPublicationContent none  widget-none" id="cadc9b41-a542-448a-9de4-8d0803320174">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none "><link id="build-style-article" rel="stylesheet" type="text/css" href="/products/marlin/cell/releasedAssets/css/build-article-b26f3342b722959c7931.css"><article xmlns="http://www.w3.org/1999/xhtml" data-design="pill" data-has="right-rail" vocab="http://schema.org/" typeof="ScholarlyArticle" data-pii="S0092-8674(20)30339-1" lang="en" dir="ltr"><div data-core-wrapper="header"><header data-extent="frontmatter"><div class="core-container"><div data-article-access="free" data-article-access-type="open" class="meta-panel"><span class="meta-panel__left-content"><span class="meta-panel__type"><span>Article</span></span></span><span class="meta-panel__middle-content"><span class="meta-panel__volumeIssue"><a href="/cell/issue?pii=S0092-8674(19)X0009-4"><span>Volume 181</span>, <span>Issue 2</span></a></span><span class="meta-panel__pages"><span>p325-345.e28</span></span><span class="meta-panel__onlineDate">April 16, 2020</span></span><span class="meta-panel__right-content"><span class="meta-panel__access meta-panel__access--archive"><span>Open Archive</span></span></span><div class="article-header__download-full-issue"><button title="Download full issue" type="submit" formaction="/action/downloadMultiplePdf?action=downloadIssuePdf&amp;journal=cell&amp;volume=181&amp;issue=2&amp;pii=S0092-8674%2819%29X0009-4" class="btn--download-issue btn--cancel download-full-issue"><i class="icon-download-new"></i><span>Download Full Issue</span></button><div class="table-of-content__alert table-of-content__alert--download"><p>Download started</p><a href="#" class="table-of-content__alert--close">Ok</a></div></div></div><h1 property="name">G3BP1 Is a Tunable Switch that Triggers Phase Separation to Assemble Stress Granules</h1><div class="contributors"><span class="authors"><span role="list"><span property="author" typeof="Person" role="listitem"><a href="#au1"><span property="givenName">Peiguo</span> <span property="familyName">Yang</span></a><sup class="xref"><a href="#aff1" property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a href="#fn1" role="doc-noteref">7</a></sup></span> ∙ <span property="author" typeof="Person" role="listitem"><a href="#au2"><span property="givenName">Cécile</span> <span property="familyName">Mathieu</span></a><sup class="xref"><a href="#aff1" property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a href="#aff2" property="affiliation" typeof="Organization" role="doc-noteref">2</a>,<a href="#fn1" role="doc-noteref">7</a></sup></span> ∙ <span property="author" typeof="Person" role="listitem"><a href="#au3"><span property="givenName">Regina-Maria</span> <span property="familyName">Kolaitis</span></a><sup class="xref"><a href="#aff1" property="affiliation" typeof="Organization" role="doc-noteref">1</a></sup></span><span data-displayed-on="all"> ∙ … </span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><a href="#au4"><span property="givenName">Peipei</span> <span property="familyName">Zhang</span></a><sup class="xref"><a href="#aff1" property="affiliation" typeof="Organization" role="doc-noteref">1</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><a href="#au5"><span property="givenName">James</span> <span property="familyName">Messing</span></a><sup class="xref"><a href="#aff1" property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a href="#aff2" property="affiliation" typeof="Organization" role="doc-noteref">2</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><a href="#au6"><span property="givenName">Ugur</span> <span property="familyName">Yurtsever</span></a><sup class="xref"><a href="#aff1" property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a href="#aff6" property="affiliation" typeof="Organization" role="doc-noteref">6</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><a href="#au7"><span property="givenName">Zemin</span> <span property="familyName">Yang</span></a><sup class="xref"><a href="#aff1" property="affiliation" typeof="Organization" role="doc-noteref">1</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><a href="#au8"><span property="givenName">Jinjun</span> <span property="familyName">Wu</span></a><sup class="xref"><a href="#aff1" property="affiliation" typeof="Organization" role="doc-noteref">1</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><a href="#au9"><span property="givenName">Yuxin</span> <span property="familyName">Li</span></a><sup class="xref"><a href="#aff5" property="affiliation" typeof="Organization" role="doc-noteref">5</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><a href="#au10"><span property="givenName">Qingfei</span> <span property="familyName">Pan</span></a><sup class="xref"><a href="#aff3" property="affiliation" typeof="Organization" role="doc-noteref">3</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><a href="#au11"><span property="givenName">Jiyang</span> <span property="familyName">Yu</span></a><sup class="xref"><a href="#aff3" property="affiliation" typeof="Organization" role="doc-noteref">3</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><a href="#au12"><span property="givenName">Erik W.</span> <span property="familyName">Martin</span></a><sup class="xref"><a href="#aff4" property="affiliation" typeof="Organization" role="doc-noteref">4</a></sup></span></span> ∙ <span property="author" typeof="Person" role="listitem"><a href="#au13"><span property="givenName">Tanja</span> <span property="familyName">Mittag</span></a><sup class="xref"><a href="#aff4" property="affiliation" typeof="Organization" role="doc-noteref">4</a></sup></span> ∙ <span property="author" typeof="Person" role="listitem"><a href="#au14"><span property="givenName">Hong Joo</span> <span property="familyName">Kim</span></a><sup class="xref"><a href="#aff1" property="affiliation" typeof="Organization" role="doc-noteref">1</a></sup></span> ∙ <span property="author" typeof="Person" role="listitem" class="corresponding-author"><a href="#au15"><span property="givenName">J. Paul</span> <span property="familyName">Taylor</span></a><sup class="xref"><a href="#aff1" property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a href="#aff2" property="affiliation" typeof="Organization" role="doc-noteref">2</a>,<a href="#fn2" role="doc-noteref">8</a></sup> <a id="eadd0010" href="mailto:jpaul.taylor@stjude.org" property="email" title="Send email to J. Paul Taylor">jpaul.taylor@stjude.org</a></span><span data-displayed-on="all"> … <span><span data-action="reveal">Show more</span></span> </span><span data-displayed-on="none" data-on-display="all" aria-hidden="true"> <span data-action="hide">Show less</span></span></span></span></div><div class="core-self-citation"><div id="core-affiliations-notes"><div class="affiliations"><div property="affiliation" typeof="Organization"><span class="label">1</span><span id="aff1" property="name">Department of Cell and Molecular Biology, St. Jude Children’s Research Hospital, Memphis, TN 38105, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">2</span><span id="aff2" property="name">Howard Hughes Medical Institute, Chevy Chase, MD 20815, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">3</span><span id="aff3" property="name">Department of Computational Biology, St. Jude Children’s Research Hospital, Memphis, TN 38105, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">4</span><span id="aff4" property="name">Department of Structural Biology, St. Jude Children’s Research Hospital, Memphis, TN 38105, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">5</span><span id="aff5" property="name">Center for Proteomics and Metabolomics, St. Jude Children’s Research Hospital, Memphis, TN 38105, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">6</span><span id="aff6" property="name">Graduate School of Structure and Dynamics of Living Systems, Université Paris-Saclay, Gif-sur-Yvette, 91190, France</span></div></div><div class="core-authors-notes"><div role="doc-footnote" data-has="label"><div class="label">7</div><div id="fn1"><div id="ntpara0010" role="paragraph">These authors contributed equally</div></div></div><div role="doc-footnote" data-has="label"><div class="label">8</div><div id="fn2"><div id="ntpara0015" role="paragraph">Lead Contact</div></div></div></div></div><div id="core-content-info"><div class="history"><div>Publication History:</div><div><span>Received January 29, 2019</span>; <span>Revised October 23, 2019</span>; <span>Accepted March 20, 2020</span></div></div><div class="core-links"><span class="doi">DOI: <a href="https://doi.org/10.1016/j.cell.2020.03.046" property="sameAs" target="_blank">10.1016/j.cell.2020.03.046 <span class="sr-only">External Link</span></a></span><span class="science-direct">Also available on <a href="https://www.sciencedirect.com/science/article/pii/S0092867420303391" target="_blank">ScienceDirect <span class="sr-only">External Link</span></a></span></div><div role="paragraph">Copyright: © 2020 Elsevier Inc.</div><div class="core-links"><span>User License: <a href="http://www.elsevier.com/open-access/userlicense/1.0/" target="_blank">Elsevier user license</a> | <a href="https://www.elsevier.com/open-access" target="_blank">Elsevier's open access license policy</a></span></div><div class="core-crossmark-badge"><a data-target="crossmark" href="#" title="Check for updates on crossmark" data-doi="10.1016/j.cell.2020.03.046" data-id="article-info-crossmark" class="crossmark__link"><svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 290 58" height="22px" role="presentation"><path fill="url(#horizontal_crossmark)" stroke="#948F8F" stroke-miterlimit="10" stroke-width=".75" d="M287 1H3a2 2 0 0 0-2 2v52a2 2 0 0 0 2 2h284a2 2 0 0 0 2-2V3a2 2 0 0 0-2-2Z"></path><path fill="#535353" d="M80.3 23.58a6.395 6.395 0 0 0-.55-1.43 3.92 3.92 0 0 0-2.19-1.86 5.099 5.099 0 0 0-1.71-.29 5 5 0 0 0-2.46.58 4.831 4.831 0 0 0-1.69 1.54 6.921 6.921 0 0 0-1 2.19 10.16 10.16 0 0 0 0 5 6.921 6.921 0 0 0 1 2.19 4.83 4.83 0 0 0 1.69 1.5 5 5 0 0 0 2.46.58 4.44 4.44 0 0 0 1.85-.36 4.12 4.12 0 0 0 1.38-1c.39-.436.695-.942.9-1.49A7.094 7.094 0 0 0 80.4 29h3a8.46 8.46 0 0 1-.69 2.92 7.179 7.179 0 0 1-1.59 2.29 7 7 0 0 1-2.35 1.49 8.14 8.14 0 0 1-3 .52 8.43 8.43 0 0 1-3.59-.74 7.88 7.88 0 0 1-2.69-2 9.001 9.001 0 0 1-1.69-3 11.41 11.41 0 0 1 0-7.23 9.001 9.001 0 0 1 1.69-3 8.001 8.001 0 0 1 2.69-2 8.3 8.3 0 0 1 3.59-.75 9.44 9.44 0 0 1 2.83.41c.866.242 1.68.642 2.4 1.18A6.229 6.229 0 0 1 82.65 21a6.66 6.66 0 0 1 .78 2.58H80.3ZM85.9 17.9h2.85v6.63h.05a4.15 4.15 0 0 1 1.56-1.46 4.61 4.61 0 0 1 2.31-.59 4.83 4.83 0 0 1 3.33 1.1 4.21 4.21 0 0 1 1.22 3.3v8.88H94.4v-8.13a3.51 3.51 0 0 0-.65-2.21 2.36 2.36 0 0 0-1.87-.69 3.1 3.1 0 0 0-1.3.26 2.821 2.821 0 0 0-1 .73A3.41 3.41 0 0 0 89 26.8a3.88 3.88 0 0 0-.23 1.33v7.63H85.9V17.9ZM102.47 30c-.002.491.069.98.21 1.45.131.448.348.866.64 1.23a3.08 3.08 0 0 0 1.08.84 3.543 3.543 0 0 0 1.55.31 3.431 3.431 0 0 0 2-.54c.541-.402.94-.966 1.14-1.61h2.7A5.349 5.349 0 0 1 109.7 35a5.715 5.715 0 0 1-1.74.85 7 7 0 0 1-2 .29 6.848 6.848 0 0 1-2.7-.5 5.577 5.577 0 0 1-2-1.4A5.953 5.953 0 0 1 100 32a8.748 8.748 0 0 1-.41-2.75 7.768 7.768 0 0 1 .44-2.61 6.67 6.67 0 0 1 1.25-2.17 5.863 5.863 0 0 1 4.56-2 5.79 5.79 0 0 1 2.74.64c.783.41 1.466.986 2 1.69a6.824 6.824 0 0 1 1.16 2.41c.264.905.352 1.852.26 2.79h-9.53Zm6.65-1.87a4.605 4.605 0 0 0-.29-1.3c-.148-.4-.368-.77-.65-1.09a3.22 3.22 0 0 0-1-.75 3 3 0 0 0-1.31-.29 3.431 3.431 0 0 0-1.36.26 3.108 3.108 0 0 0-1 .73 3.594 3.594 0 0 0-.7 1.09 3.798 3.798 0 0 0-.29 1.35h6.65-.05ZM122.88 27.15a2.666 2.666 0 0 0-1-1.81 3.14 3.14 0 0 0-2-.61c-.4.005-.798.07-1.18.19-.435.136-.83.376-1.15.7a3.92 3.92 0 0 0-.87 1.43 6.783 6.783 0 0 0-.35 2.39c.002.539.065 1.076.19 1.6.113.509.316.993.6 1.43.273.41.635.752 1.06 1a3 3 0 0 0 1.58.39 2.872 2.872 0 0 0 2.06-.77 3.596 3.596 0 0 0 1-2.18h2.85a6.26 6.26 0 0 1-1.91 3.86 5.83 5.83 0 0 1-4 1.34 6.61 6.61 0 0 1-2.69-.51 5.695 5.695 0 0 1-2-1.4 6.001 6.001 0 0 1-1.21-2.11 8.279 8.279 0 0 1-.41-2.65 8.987 8.987 0 0 1 .4-2.72 6.237 6.237 0 0 1 1.2-2.21 5.614 5.614 0 0 1 2-1.47 6.7 6.7 0 0 1 2.79-.54 7.801 7.801 0 0 1 2.14.29 5.634 5.634 0 0 1 1.8.86c.529.386.969.88 1.29 1.45a5 5 0 0 1 .6 2.07h-2.85l.06-.02ZM128.13 17.9H131V28l5.15-5.22h3.5l-5 4.75 5.42 8.17h-3.47l-4-6.27-1.6 1.7v4.63h-2.85V17.9h-.02ZM147.15 22.83h2.13v-1.08a5.505 5.505 0 0 1 .3-2c.161-.456.436-.862.8-1.18.33-.273.721-.462 1.14-.55.44-.094.89-.141 1.34-.14a8.158 8.158 0 0 1 2 .18v2.24a4.113 4.113 0 0 0-.59-.11 6.343 6.343 0 0 0-.74 0 1.74 1.74 0 0 0-1 .27 1.204 1.204 0 0 0-.41 1.08v1.33h2.42V25h-2.42v10.8h-2.85V25h-2.12v-2.17ZM162.23 36.1a6.999 6.999 0 0 1-2.76-.51 5.877 5.877 0 0 1-2-1.41 5.998 5.998 0 0 1-1.33-2.18 8.775 8.775 0 0 1 0-5.48 6.008 6.008 0 0 1 1.27-2.15 5.894 5.894 0 0 1 2-1.41A7.71 7.71 0 0 1 165 23a5.894 5.894 0 0 1 2 1.41 6.003 6.003 0 0 1 1.28 2.15c.585 1.78.585 3.7 0 5.48a5.99 5.99 0 0 1-3.28 3.54 7.002 7.002 0 0 1-2.77.52Zm0-2.25a3.268 3.268 0 0 0 1.65-.4c.447-.262.83-.621 1.12-1.05.304-.444.528-.938.66-1.46a6.581 6.581 0 0 0 0-3.31 4.416 4.416 0 0 0-.66-1.46 3.526 3.526 0 0 0-1.15-1 3.6 3.6 0 0 0-3.3 0 3.526 3.526 0 0 0-1.15 1 4.414 4.414 0 0 0-.66 1.46 6.558 6.558 0 0 0 0 3.31c.132.522.356 1.016.66 1.46a3.47 3.47 0 0 0 1.15 1.05 3.26 3.26 0 0 0 1.68.4ZM171.13 22.83h2.67v2.5c.097-.362.264-.701.49-1 .251-.341.547-.647.88-.91a4.404 4.404 0 0 1 1.14-.66 3.41 3.41 0 0 1 1.28-.25c.23-.013.46-.013.69 0h.39v2.79l-.61-.09a5.173 5.173 0 0 0-.61 0 3.278 3.278 0 0 0-1.36.29 3.131 3.131 0 0 0-1.11.85 4.171 4.171 0 0 0-.75 1.39 6 6 0 0 0-.28 1.9v6.15h-2.85V22.83h.03ZM198.22 35.75h-2.8V34a3.907 3.907 0 0 1-1.56 1.56 4.25 4.25 0 0 1-2.11.59 4.742 4.742 0 0 1-3.75-1.31 5.537 5.537 0 0 1-1.12-3.84v-8.2h2.85v7.92a3.503 3.503 0 0 0 .65 2.4 2.38 2.38 0 0 0 1.83.7 3.599 3.599 0 0 0 1.5-.27 2.66 2.66 0 0 0 1-.74c.259-.326.444-.705.54-1.11a5.79 5.79 0 0 0 .16-1.4v-7.5h2.85v12.95h-.04ZM201.3 22.83h2.7v1.75a3.523 3.523 0 0 1 1.72-1.58 5.568 5.568 0 0 1 2.33-.49 6.17 6.17 0 0 1 2.66.54 5.246 5.246 0 0 1 1.89 1.47c.515.65.899 1.394 1.13 2.19.257.87.385 1.773.38 2.68.001.851-.114 1.7-.34 2.52a6.568 6.568 0 0 1-1 2.16 5.002 5.002 0 0 1-4.21 2.06c-.419 0-.837-.036-1.25-.11a5.39 5.39 0 0 1-1.2-.36A4.737 4.737 0 0 1 205 35a3.567 3.567 0 0 1-.81-.91v6.45h-2.89V22.83Zm10 6.47c0-.574-.078-1.146-.23-1.7a4.606 4.606 0 0 0-.67-1.46 3.475 3.475 0 0 0-1.12-1 3.06 3.06 0 0 0-1.55-.39A3.141 3.141 0 0 0 205 26a5.518 5.518 0 0 0-1 3.3c-.004.611.077 1.22.24 1.81.138.523.38 1.012.71 1.44a3.55 3.55 0 0 0 1.14 1 3.242 3.242 0 0 0 1.54.35 3.18 3.18 0 0 0 1.65-.4c.438-.25.817-.59 1.11-1 .299-.44.513-.931.63-1.45a7.38 7.38 0 0 0 .23-1.75h.05ZM228.67 35.75H226V34a3.38 3.38 0 0 1-1.67 1.61 5.668 5.668 0 0 1-2.33.49 6.17 6.17 0 0 1-2.66-.54 5.307 5.307 0 0 1-1.89-1.46 6.254 6.254 0 0 1-1.12-2.19 9.48 9.48 0 0 1-.37-2.71 8.67 8.67 0 0 1 .47-3 6.058 6.058 0 0 1 1.26-2.1 4.89 4.89 0 0 1 1.8-1.21 5.738 5.738 0 0 1 2.06-.39c.412 0 .824.037 1.23.11.413.073.815.194 1.2.36.38.166.736.381 1.06.64.32.255.594.562.81.91V17.9h2.85v17.85h-.03Zm-10-6.33c-.001.55.069 1.098.21 1.63.13.511.35.996.65 1.43.288.414.668.757 1.11 1a3.241 3.241 0 0 0 1.6.38 3.199 3.199 0 0 0 1.64-.4 3.398 3.398 0 0 0 1.13-1.05c.297-.447.517-.94.65-1.46.14-.542.211-1.1.21-1.66a5.227 5.227 0 0 0-1-3.35 3.175 3.175 0 0 0-2.61-1.2 3.23 3.23 0 0 0-1.69.41 3.478 3.478 0 0 0-1.12 1.07 4.494 4.494 0 0 0-.62 1.5 7.871 7.871 0 0 0-.11 1.7h-.05ZM242.6 32.88c-.018.258.03.516.14.75a.577.577 0 0 0 .54.23h.3c.135-.002.269-.019.4-.05v2l-.39.11-.49.08-.5.08h-.42a2.75 2.75 0 0 1-1.45-.35 1.731 1.731 0 0 1-.75-1.23 4.998 4.998 0 0 1-2.09 1.2 8.224 8.224 0 0 1-2.39.38 5.861 5.861 0 0 1-1.67-.24 4.394 4.394 0 0 1-1.41-.7 3.35 3.35 0 0 1-1-1.18 3.624 3.624 0 0 1-.36-1.66 3.84 3.84 0 0 1 .44-2 3.246 3.246 0 0 1 1.15-1.17 5.195 5.195 0 0 1 1.6-.61c.593-.127 1.19-.223 1.79-.29a14.77 14.77 0 0 1 1.47-.21 6.896 6.896 0 0 0 1.24-.21c.318-.077.611-.236.85-.46a1.286 1.286 0 0 0 .31-.94 1.458 1.458 0 0 0-.26-.9A1.759 1.759 0 0 0 239 25a2.807 2.807 0 0 0-.86-.25 6.941 6.941 0 0 0-.9-.06 3.58 3.58 0 0 0-2 .5 2 2 0 0 0-.87 1.55h-2.85c.021-.73.227-1.442.6-2.07.338-.54.797-.996 1.34-1.33a5.475 5.475 0 0 1 1.84-.7c.692-.134 1.395-.2 2.1-.2.632 0 1.262.067 1.88.2a5.337 5.337 0 0 1 1.66.65 3.6 3.6 0 0 1 1.19 1.16c.314.525.47 1.129.45 1.74v6.65l.02.04Zm-2.85-3.6a3.45 3.45 0 0 1-1.6.51c-.667.06-1.3.147-1.9.26a6.137 6.137 0 0 0-.87.21 2.64 2.64 0 0 0-.75.38c-.22.164-.395.38-.51.63a2.129 2.129 0 0 0-.19.94c-.011.29.085.575.27.8.182.215.407.388.66.51.269.128.555.216.85.26.277.05.558.076.84.08a4.63 4.63 0 0 0 1-.12 3.418 3.418 0 0 0 1-.43 2.73 2.73 0 0 0 .81-.76 1.93 1.93 0 0 0 .32-1.14v-2.14l.07.01ZM244.37 22.83h2.15V19h2.85v3.88h2.58V25h-2.58v6.9a7.064 7.064 0 0 0 0 .77c.016.194.074.381.17.55a.84.84 0 0 0 .41.34 2 2 0 0 0 .75.11h.6c.203-.01.404-.04.6-.09v2.2l-.93.1a8.403 8.403 0 0 1-.92 0 6.33 6.33 0 0 1-1.81-.21 2.421 2.421 0 0 1-1.08-.62 2.15 2.15 0 0 1-.52-1 7.791 7.791 0 0 1-.16-1.42V25h-2.15v-2.17h.04ZM256.22 30c-.002.491.069.98.21 1.45.131.448.348.866.64 1.23a3.08 3.08 0 0 0 1.08.84 3.543 3.543 0 0 0 1.55.31 3.431 3.431 0 0 0 2-.54c.541-.402.94-.966 1.14-1.61h2.7a5.352 5.352 0 0 1-2.1 3.26 5.715 5.715 0 0 1-1.74.85 7 7 0 0 1-2 .29 6.848 6.848 0 0 1-2.7-.5 5.577 5.577 0 0 1-2-1.4 5.947 5.947 0 0 1-1.21-2.18 8.754 8.754 0 0 1-.41-2.75 7.758 7.758 0 0 1 .44-2.61 6.67 6.67 0 0 1 1.25-2.17 5.863 5.863 0 0 1 4.56-2 5.79 5.79 0 0 1 2.74.64c.783.41 1.466.986 2 1.69a6.824 6.824 0 0 1 1.16 2.41c.243.916.308 1.87.19 2.81h-9.5V30Zm6.65-1.87a4.605 4.605 0 0 0-.29-1.3c-.148-.4-.368-.77-.65-1.09a3.22 3.22 0 0 0-1-.75 3 3 0 0 0-1.31-.29 3.431 3.431 0 0 0-1.36.26 3.108 3.108 0 0 0-1 .73 3.575 3.575 0 0 0-.7 1.09 3.798 3.798 0 0 0-.29 1.35h6.65-.05ZM270 31.6a2.154 2.154 0 0 0 .95 1.75c.607.35 1.3.524 2 .5.304-.002.608-.022.91-.06a3.943 3.943 0 0 0 1-.24c.286-.105.54-.28.74-.51a1.23 1.23 0 0 0 .26-.89 1.261 1.261 0 0 0-.4-.9 2.779 2.779 0 0 0-1-.56 9.315 9.315 0 0 0-1.34-.36l-1.5-.33a14.219 14.219 0 0 1-1.54-.42 4.937 4.937 0 0 1-1.33-.68 3.12 3.12 0 0 1-.94-1.09 3.472 3.472 0 0 1-.35-1.64 2.94 2.94 0 0 1 .51-1.76 3.913 3.913 0 0 1 1.3-1.15 5.66 5.66 0 0 1 1.75-.61 10.316 10.316 0 0 1 1.84-.17 8.418 8.418 0 0 1 1.91.21 5.19 5.19 0 0 1 1.65.69c.496.318.916.741 1.23 1.24.341.559.55 1.188.61 1.84h-3a1.783 1.783 0 0 0-.94-1.37 3.892 3.892 0 0 0-1.69-.35 6.093 6.093 0 0 0-.71 0 3.327 3.327 0 0 0-.78.19 1.642 1.642 0 0 0-.61.4 1 1 0 0 0-.25.69 1.093 1.093 0 0 0 .36.85c.294.245.634.428 1 .54.437.153.885.274 1.34.36l1.55.33a20.9 20.9 0 0 1 1.53.42c.479.157.931.386 1.34.68.41.277.751.643 1 1.07.255.494.379 1.045.36 1.6a3.48 3.48 0 0 1-.52 2 4.15 4.15 0 0 1-1.36 1.3 6.19 6.19 0 0 1-1.86.73 9.435 9.435 0 0 1-2 .23 8.823 8.823 0 0 1-2.26-.27 5.265 5.265 0 0 1-1.8-.84 4 4 0 0 1-1.2-1.4 4.445 4.445 0 0 1-.46-2h2.7v-.02Z"></path><path fill="#C72914" d="m21.94 38.74 18.12-12.08v-14.5H21.94v26.58Z"></path><path fill="#EF3340" d="M40.06 38.74 21.94 26.66v-14.5h18.12v26.58Z"></path><path fill="#3EB1C8" d="M31 9a20 20 0 1 0 0 40 20 20 0 0 0 0-40Zm0 35.1a15.1 15.1 0 1 1 0-30.2 15.1 15.1 0 0 1 0 30.2Z"></path><path fill="#FFC72C" d="M42.09 39.23A15.09 15.09 0 0 1 17 23.38l-4.2-2.64a20 20 0 0 0 33.49 21.14l-4.2-2.65Z"></path><defs><linearGradient id="horizontal_crossmark" x1="145" x2="145" y1="7.33" y2="56.82" gradientUnits="userSpaceOnUse"><stop stop-color="#fff"></stop><stop offset="1" stop-color="#c4c4c4"></stop></linearGradient></defs></svg></a></div><div class="core-content-info"></div></div><div id="core-linked-content">
    

        


    

            

            

            <section data-widget-id="e587233e-22aa-4221-8ae1-22df489d46f7" data-widget-name="UX3linkedContentList" class="axel-linked-articles"><div class="section-paragraph linked-article"><ul class="rlist"><li class="linked-article__item"><div class="linked-article__article-type">Article</div><a class="linked-article__title link--underlined" href="/cell/fulltext/S0092-8674(20)30343-3" target="_blank" title="Competing Protein-RNA Interaction Networks Control Multiphase Intracellular Organization">Competing Protein-RNA Interaction Networks Control Multiphase Intracellular Organization</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Sanders et al.</li></ul></div><div class="linked-article__details"><span><i>Cell</i></span><span id="item_date">April 16, 2020</span></div></li><li class="linked-article__item"><div class="linked-article__article-type">Article</div><a class="linked-article__title link--underlined" href="/cell/fulltext/S0092-8674(20)30342-1" target="_blank" title="RNA-Induced Conformational Switching and Clustering of G3BP Drive Stress Granule Assembly by Condensation">RNA-Induced Conformational Switching and Clustering of G3BP Drive Stress Granule Assembly by Condensation</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Guillén-Boixet et al.</li></ul></div><div class="linked-article__details"><span><i>Cell</i></span><span id="item_date">April 16, 2020</span></div></li></ul></div></section>

            

        
</div></div></div></header><div class="core-header-aside">
    

        


    

            

            

            <div class="cover-image cover-image--axel"><div class="cover-image__thumb"><a href="https://www.cell.com/cell/issue?pii=S0092-8674(19)X0009-4"><img src="/cms/asset/8e0cf1f8-74dd-4ff1-8cf9-225c94c2740a/cov200h.gif" alt="Cover Image - Cell, Volume 181, Issue 2"></a></div></div>

            

        
</div></div><div data-core-wrapper="content"><div data-core-nav="header" data-extent="frontmatter"><div class="core-nav-wrapper core-container" aria-label="Article navigation"><div class="navigation-menu-pdf">
    

        


    

            

            

            <div data-widget-id="ecff07e8-0bc4-4533-b1f1-49365e2ead20" data-widget-name="UX3ArticleToolsWidget" class="article-tools"><a href="/action/showPdf?pii=S0092-8674%2820%2930339-1" title="Standard PDF (opens in a new tab)" target="_blank" aria-hidden="false" role="menuitem" class="article-tools__item__pdf"><svg focusable="false" viewBox="0 0 32 32" width="20" height="20" title="PDF" class="pdf-icon"><path id="pdfwhite" d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path id="pdfda0000" d="M.167 2.592H22.39V9.72H.166z" fill="#da0000"></path><path id="pdffff9f9" fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path id="pdff91d0a" d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="basic">Download PDF</span><span class="alternative">Download PDF</span></a></div>

            

        
</div><div><button data-id="article-toolbar-showhide" aria-controls="article_sections_menu" aria-expanded="false" aria-haspopup="true" aria-label="Toggle section navigation menu"><i class="icon-list" aria-hidden="true"></i><span>Outline</span></button><nav id="article_sections_menu" aria-label="Contents" data-core-nav="article"><ul><li><a href="#author-highlights-abstract">Highlights</a></li><li><a href="#author-abstract">Summary</a></li><li><a href="#graphical-abstract">Graphical Abstract</a></li><li><a href="#keywords">Keywords</a></li><li><a href="#sec-1">Introduction</a></li><li><a href="#sec-2">Results</a></li><li><a href="#sec-3">Discussion</a></li><li><a href="#sec-4">STAR★Methods</a></li><li><a href="#acknowledgments">Acknowledgments</a></li><li><a href="#supplementary-material">Supplemental Information (3)</a></li><li><a href="#references">References</a></li></ul></nav></div><div class="navigation-menu-share">
    

        


    

            

            

            <!-- Go to https://www.addtoany.com/buttons/customize/ to customize your tools --><script type="text/javascript" defer="defer" src="https://static.addtoany.com/menu/page.js"></script><script type="text/javascript">var a2a_config = a2a_config || {};
a2a_config.thanks = {
    postShare: false,
    ad: false,
};</script><script>var a2a_config = a2a_config || {};
let res = window.location.href.replace(/\(/g, "%28").replace(/\)/g, "%29");
a2a_config.templates = a2a_config.templates || {};

a2a_config.templates.email = {
    body: res
};
</script><div data-widget-id="5f978446-c80a-450c-a185-94d633df2db2" data-widget-name="UX3share" aria-hidden="false" role="group" class="article-tools__dropblock article-tools__share"><button aria-label="Toggle share menu" aria-expanded="false" aria-haspopup="true" role="menuitem" aria-controls="navigation_menu_share_block" aria-labelledby="share-label" class="article-tools__item__share"><i aria-hidden="true" class="icon-article-share"></i><span id="share-label">Share</span></button><div id="navigation_menu_share_block" role="menu" class="share__block axel-share article-tools__dropblock__holder"><h4 class="axel-share__title heading">Share<button title="Close share menu" aria-expanded="false" aria-haspopup="true" aria-controls="navigation_menu_share_block"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></h4><div class="axel-share__inner"><span class="sr-only">Share on</span><ul class="a2a__share-list rlist w-slide--list a2a a2a_kit a2a_kit_size_32 a2a_default_style" data-a2a-url="https://www.cell.com/cell/fulltext/S0092-8674%2820%2930339-1" style="line-height: 32px;"><li class="a2a_listitem_custom"><a role="link" title="Email" class="a2a_button_email" target="_blank" rel="nofollow noopener" href="/#email"><i aria-hidden="true" class="at-icon-wrapper icon-gizmo-email-plus"></i><span>Email</span></a></li><li class="a2a_listitem_custom"><a role="link" title="formerly known as Twitter" class="a2a_button_twitter" target="_blank" rel="nofollow noopener" href="/#twitter"><i aria-hidden="true" class="at-icon-wrapper icon-x-default"></i><span>X</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Facebook" class="a2a_button_facebook" target="_blank" rel="nofollow noopener" href="/#facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook-square-2"></i><span>Facebook</span></a></li><li class="a2a_listitem_custom"><a role="link" title="LinkedIn" class="a2a_button_linkedin" target="_blank" rel="nofollow noopener" href="/#linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin-square"></i><span>LinkedIn</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Sina Weibo" class="a2a_button_sina_weibo" target="_blank" rel="nofollow noopener" href="/#sina_weibo"><i aria-hidden="true" class="at-icon-wrapper icon-weibo"></i><span>Sina Weibo</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Mendeley" class="a2a_button_mendeley" target="_blank" rel="nofollow noopener" href="/#mendeley"><i aria-hidden="true" class="at-icon-wrapper icon-mendeley-2"></i><span>Mendeley</span></a></li><li class="a2a_listitem_custom"><a role="link" title="bluesky" class="a2a_button_bluesky" target="_blank" rel="nofollow noopener" href="/#bluesky"><i aria-hidden="true" class="at-icon-wrapper icon-bluesky-logo"></i><span>bluesky</span></a></li><div style="clear: both;"></div></ul><div class="extra-links-drop-zone">
    

        


    

            

            

            <div data-widget-id="f1a208d3-ac5e-4db2-9c3a-40ef97ddfa10" data-widget-name="UX3HTMLWidget"><a class="axel-share__extra" target="_blank" href="/personalize/addFavoritePublication?doi=10.1016/j.cell.2020.03.046">
	<i class="icon-reading-list" aria-hidden="true"></i>
	<span>Add to my reading list</span>
</a></div>

            

        
</div></div></div></div>

            

        
</div><div><button data-id="article-nav-menubar-more" aria-controls="article_more_menu" aria-expanded="false" aria-haspopup="true" aria-label="Toggle more menu"><i aria-hidden="true" class="icon-more-mobile"></i><span>More</span></button><div class="info-panel"><div class="info-panel__left-content"><div class="info-panel__article_tools info-panel__item"><div id="article_more_menu" aria-label="More menu" data-core-nav="more"><div class="heading">More<button title="Close more menu" aria-controls="article_more_menu" aria-haspopup="true"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div>
    

        


    

            

            

            <ul data-widget-id="183a17a2-87b5-4e6f-bb85-6b5cda068c89" data-widget-name="UX3ArticleToolsWidget" class="article-tools"><li class="article-tools__item article-tools__pdf"><a href="/action/showPdf?pii=S0092-8674%2820%2930339-1" title="Standard PDF (opens in a new tab)" target="_blank" aria-hidden="false" class="article-tools__ctrl article-tools__item__pdf"><svg focusable="false" viewBox="0 0 32 32" width="20" height="20" title="PDF" class="pdf-icon"><path id="pdfwhite" d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path id="pdfda0000" d="M.167 2.592H22.39V9.72H.166z" fill="#da0000"></path><path id="pdffff9f9" fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path id="pdff91d0a" d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="basic">Download PDF</span><span class="alternative">Download PDF</span></a></li><li><a href="https://www.cell.com/action/showCitFormats?doi=10.1016%2Fj.cell.2020.03.046&amp;pii=S0092-8674%2820%2930339-1" title="Cite" target="_blank" aria-label="Open Cite in new tab" class="article-tools__item__displayExportCitation"><i aria-hidden="true" class="icon-export-Citation"></i><span>Cite</span></a></li><li id="article_tools_share">
    

        


    

            

            

            <!-- Go to https://www.addtoany.com/buttons/customize/ to customize your tools --><script type="text/javascript" defer="defer" src="https://static.addtoany.com/menu/page.js"></script><script type="text/javascript">var a2a_config = a2a_config || {};
a2a_config.thanks = {
    postShare: false,
    ad: false,
};</script><script>var a2a_config = a2a_config || {};
let res = window.location.href.replace(/\(/g, "%28").replace(/\)/g, "%29");
a2a_config.templates = a2a_config.templates || {};

a2a_config.templates.email = {
    body: res
};
</script><div data-widget-id="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da" data-widget-name="UX3share" class="dropBlock article-tools__dropblock article-tools__share"><button data-db-target-for="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da" aria-expanded="false" title="open Share menu" class="article-tools__ctrl share__ctrl"><i aria-hidden="true" class="icon-article-share"></i><span>Share</span></button><div data-db-target-of="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da" id="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da_Pop" class="share__block axel-share dropBlock__holder article-tools__dropblock__holder"><h4 class="axel-share__title">Share</h4><div class="axel-share__inner"><span class="sr-only">Share on</span><ul class="a2a__share-list rlist w-slide--list a2a a2a_kit a2a_kit_size_32 a2a_default_style" data-a2a-url="https://www.cell.com/cell/fulltext/S0092-8674%2820%2930339-1" style="line-height: 32px;"><li class="a2a_listitem_custom"><a role="link" title="Email" class="a2a_button_email" target="_blank" rel="nofollow noopener" href="/#email"><i aria-hidden="true" class="at-icon-wrapper icon-gizmo-email-plus"></i><span>Email</span></a></li><li class="a2a_listitem_custom"><a role="link" title="formerly known as Twitter" class="a2a_button_twitter" target="_blank" rel="nofollow noopener" href="/#twitter"><i aria-hidden="true" class="at-icon-wrapper icon-x-default"></i><span>X</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Facebook" class="a2a_button_facebook" target="_blank" rel="nofollow noopener" href="/#facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook-square-2"></i><span>Facebook</span></a></li><li class="a2a_listitem_custom"><a role="link" title="LinkedIn" class="a2a_button_linkedin" target="_blank" rel="nofollow noopener" href="/#linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin-square"></i><span>LinkedIn</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Sina Weibo" class="a2a_button_sina_weibo" target="_blank" rel="nofollow noopener" href="/#sina_weibo"><i aria-hidden="true" class="at-icon-wrapper icon-weibo"></i><span>Sina Weibo</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Mendeley" class="a2a_button_mendeley" target="_blank" rel="nofollow noopener" href="/#mendeley"><i aria-hidden="true" class="at-icon-wrapper icon-mendeley-2"></i><span>Mendeley</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Bluesky" class="a2a_button_bluesky" target="_blank" rel="nofollow noopener" href="/#bluesky"><i aria-hidden="true" class="at-icon-wrapper icon-bluesky-logo"></i><span>Bluesky</span></a></li><div style="clear: both;"></div></ul><div class="extra-links-drop-zone">
    

        


    

            

            

            <div data-widget-id="ae9bf5c5-a2ee-4f5c-8e41-9b21638b1a9f" data-widget-name="UX3HTMLWidget"><a class="axel-share__extra" target="_blank" href="/personalize/addFavoritePublication?doi=10.1016/j.cell.2020.03.046">
	<i class="icon-reading-list" aria-hidden="true"></i>
	<span>Add to my reading list</span>
</a></div>

            

        
</div></div></div></div>

            

        
</li><li><a href="https://www.cell.com/action/addCitationAlert?doi=10.1016%2Fj.cell.2020.03.046" title="Set Alert" target="_blank" aria-label="Open Set Alert in new tab" class="article-tools__item__displayCreateCitationAlert"><i aria-hidden="true" class="icon-set-alert"></i><span>Set Alert</span></a></li><li><a href="https://s100.copyright.com/AppDispatchServlet?publisherName=ELS&amp;contentID=S0092867420303391&amp;orderBeanReset=true&amp;orderSource=Phoenix" title="Get Rights" target="_blank" aria-label="Open Get Rights in new tab" class="article-tools__item__request"><i aria-hidden="true" class="icon-get-rights"></i><span>Get Rights</span></a></li><li><a href="https://www.contentednet.com/salesforce-form/?pii=S0092867420303391&amp;issn=00928674&amp;source=jbs" title="Reprints" target="_blank" aria-label="Open Reprints in new tab" class="article-tools__item__reprints"><i aria-hidden="true" class="icon-article-reprints"></i><span>Reprints</span></a></li></ul>

            

        
</div></div><div class="info-panel__download-issue info-panel__item"><div class="article-header__download-full-issue"><button title="Download full issue" type="submit" formaction="/action/downloadMultiplePdf?action=downloadIssuePdf&amp;journal=cell&amp;volume=181&amp;issue=2&amp;pii=S0092-8674%2819%29X0009-4" class="btn--download-issue btn--cancel download-full-issue"><i class="icon-download-new"></i><span>Download Full Issue</span></button><div class="table-of-content__alert table-of-content__alert--download"><p>Download started</p><a href="#" class="table-of-content__alert--close">Ok</a></div></div></div></div><div class="info-panel__right-content"><div class="info-panel__navigation info-panel__item"><div class="content-navigation"><a href="/cell/fulltext/S0092-8674(20)30343-3" aria-label="Previous article" aria-disabled="false" class="content-navigation__prev"><i aria-hidden="true" class="icon-arrow_l"></i><span>Previous article</span></a><a href="/cell/fulltext/S0092-8674(20)30342-1" aria-label="Next article" aria-disabled="false" class="content-navigation__next"><span>Next article</span><i aria-hidden="true" class="icon-arrow_r"></i></a></div></div></div></div></div></div></div><div><div class="core-content-inner"><div class="core-linked-content"></div><div id="abstracts" data-extent="frontmatter"><div class="core-container"><section id="author-highlights-abstract" property="abstract" typeof="Text" role="doc-abstract"><h2 property="name">Highlights</h2><div id="abspara0020" role="paragraph"><div id="ulist0010" role="list"><div id="u0010" role="listitem"><div class="label">•</div><div class="content"><div id="p0010" role="paragraph">SG assembly is driven by the collective interactions of a core protein-RNA network</div></div></div><div id="u0015" role="listitem"><div class="label">•</div><div class="content"><div id="p0015" role="paragraph">The central node G3BP encodes a molecular switch that regulates RNA-dependent LLPS</div></div></div><div id="u0020" role="listitem"><div class="label">•</div><div class="content"><div id="p0020" role="paragraph">Interplay between 3 distinct IDRs in G3BP tunes the intrinsic propensity for LLPS</div></div></div><div id="u0025" role="listitem"><div class="label">•</div><div class="content"><div id="p0025" role="paragraph">Extrinsic factors regulate SG assembly through positive or negative cooperativity</div></div></div></div></div></section><section id="author-abstract" property="abstract" typeof="Text" role="doc-abstract"><h2 property="name">Summary</h2><div id="abspara0010" role="paragraph">The mechanisms underlying ribonucleoprotein (RNP) granule assembly, including the basis for establishing and maintaining RNP granules with distinct composition, are unknown. One prominent type of RNP granule is the stress granule (SG), a dynamic and reversible cytoplasmic assembly formed in eukaryotic cells in response to stress. Here, we show that SGs assemble through liquid-liquid phase separation (LLPS) arising from interactions distributed unevenly across a core protein-RNA interaction network. The central node of this network is G3BP1, which functions as a molecular switch that triggers RNA-dependent LLPS in response to a rise in intracellular free RNA concentrations. Moreover, we show that interplay between three distinct intrinsically disordered regions (IDRs) in G3BP1 regulates its intrinsic propensity for LLPS, and this is fine-tuned by phosphorylation within the IDRs. Further regulation of SG assembly arises through positive or negative cooperativity by extrinsic G3BP1-binding factors that strengthen or weaken, respectively, the core SG network.</div></section><section id="graphical-abstract" property="abstract" typeof="Text" role="doc-abstract"><h2 property="name">Graphical Abstract</h2><div id="abspara0015" role="paragraph"><div class="figure-wrap"><figure id="undfig1" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2020.03.046/asset/cacdc32c-0c18-4509-b97f-128eddf0b5d1/main.assets/fx1_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="fx1" alt="Graphical abstract undfig1" src="/cms/10.1016/j.cell.2020.03.046/asset/38d2d7c2-ec41-4c5e-aeb2-1dd31281e44d/main.assets/fx1.jpg" height="375" width="375" data-viewer-src="/cms/10.1016/j.cell.2020.03.046/asset/cacdc32c-0c18-4509-b97f-128eddf0b5d1/main.assets/fx1_lrg.jpg" loading="lazy"></a></figure></div></div></section></div></div><div class="core-linked-content"></div><section id="keywords" property="keywords"><h2>Keywords</h2><ol><li><a href="/action/doSearch?AllField=&quot;G3BP1&quot;" alt="Search on this keyword" rel="nofollow">G3BP1</a></li><li><a href="/action/doSearch?AllField=&quot;core+stress+granule+network&quot;" alt="Search on this keyword" rel="nofollow">core stress granule network</a></li><li><a href="/action/doSearch?AllField=&quot;biomolecular+condensate&quot;" alt="Search on this keyword" rel="nofollow">biomolecular condensate</a></li><li><a href="/action/doSearch?AllField=&quot;stress+granule&quot;" alt="Search on this keyword" rel="nofollow">stress granule</a></li><li><a href="/action/doSearch?AllField=&quot;multivalency&quot;" alt="Search on this keyword" rel="nofollow">multivalency</a></li><li><a href="/action/doSearch?AllField=&quot;liquid-liquid+phase+separation&quot;" alt="Search on this keyword" rel="nofollow">liquid-liquid phase separation</a></li><li><a href="/action/doSearch?AllField=&quot;Intrinsically+disordered+protein&quot;" alt="Search on this keyword" rel="nofollow">Intrinsically disordered protein</a></li><li><a href="/action/doSearch?AllField=&quot;molecular+switch&quot;" alt="Search on this keyword" rel="nofollow">molecular switch</a></li><li><a href="/action/doSearch?AllField=&quot;membraneless+organelle&quot;" alt="Search on this keyword" rel="nofollow">membraneless organelle</a></li><li><a href="/action/doSearch?AllField=&quot;cooperativity&quot;" alt="Search on this keyword" rel="nofollow">cooperativity</a></li></ol></section><div class="core-linked-content"></div><div class="core-linked-content"></div><section id="bodymatter" data-extent="bodymatter" property="articleBody" typeof="Text"><div class="core-container"><section id="sec-1"><h2>Introduction</h2><div id="p0030" role="paragraph">Biomolecular condensation is increasingly recognized as a vital strategy of cellular organization governing a variety of biological functions (<a id="crosref0120" href="#bib2" role="doc-biblioref" data-xml-rid="bib2">Banani et&nbsp;al., 2017</a>). One such function is the regulation of RNA metabolism, which occurs in condensates known as ribonucleoprotein (RNP) granules (<a id="crosrefs0010" href="#bib6" role="doc-biblioref" data-xml-rid="bib6 bib17">Buchan, 2014; Gomes and Shorter, 2019</a>). RNP granules are dynamic assemblies of RNA and protein that are abundant in both the nucleus (e.g., nucleoli, Cajal bodies, speckles, paraspeckles, and promyelocytic leukemia bodies; <a id="crosref0125" href="#bib10" role="doc-biblioref" data-xml-rid="bib10">Dundr, 2012</a>) and cytoplasm (e.g., P bodies, stress granules [SGs], and RNA transport granules; <a id="crosref0130" href="#bib28" role="doc-biblioref" data-xml-rid="bib28">Kiebler and Bassell, 2006</a>). Each type of RNP granule has a distinct identity defined by its constituents, material properties (e.g., surface tension, elasticity, and viscosity), morphology, subcellular localization, and functions.</div><div id="p0035" role="paragraph">RNP granules assemble by liquid-liquid phase separation (LLPS), which occurs when protein-laden RNAs that are dispersed in the cytoplasm or nucleoplasm (soluble phase) coalesce into a concentrated state (condensed phase). In this condensed phase, the highly concentrated RNAs and RNA-binding proteins (RBPs) behave as a single organelle with liquid-like properties. The constituents of membraneless organelles remain in dynamic equilibrium with the surrounding nucleoplasm or cytoplasm and may form transiently or persist indefinitely. Some RBPs, particularly those harboring low complexity domains (LCDs), undergo concentration-dependent LLPS <i>in&nbsp;vitro</i> (<a id="crosrefs0015" href="#bib32" role="doc-biblioref" data-xml-rid="bib32 bib39 bib47">Lin et&nbsp;al., 2015; Molliex et&nbsp;al., 2015; Patel et&nbsp;al., 2015</a>). However, how the LLPS of an individual RBP relates to assembly of an RNP granule consisting of hundreds of similar RBPs has remained unclear. Indeed, depletion or knockout of many individual RBPs that are capable of LLPS <i>in&nbsp;vitro</i> has no apparent impact on assembly of the RNP granule to which they belong (<a id="crosref0135" href="#bib7" role="doc-biblioref" data-xml-rid="bib7">Buchan et&nbsp;al., 2013</a>).</div><div id="p0040" role="paragraph">One prominent type of RNP granule is the SG, a cytoplasmic assembly formed in eukaryotic cells in response to a variety of stressors (<a id="crosref0140" href="#bib22" role="doc-biblioref" data-xml-rid="bib22">Ivanov et&nbsp;al., 2019</a>). SG assembly correlates with arrest of translation initiation, which is accompanied by polysome disassembly and a rise in the cytoplasmic concentration of uncoated mRNAs (<a id="crosref0145" href="#bib45" role="doc-biblioref" data-xml-rid="bib45">Panas et&nbsp;al., 2016</a>). The mechanism underlying SG assembly has remained ill defined, although insights have emerged from proteomic analysis of SG cores (<a id="crosref7150" href="#bib23" role="doc-biblioref" data-xml-rid="bib23">Jain et&nbsp;al., 2016</a>) and spatial proteomics (<a id="crosrefs0020" href="#bib23" role="doc-biblioref" data-xml-rid="bib23 bib34 bib68">Jain et&nbsp;al., 2016; Markmiller et&nbsp;al., 2018; Youn et&nbsp;al., 2018</a>), which have illuminated the protein content of SGs, along with similar studies that have characterized RNA content (<a id="crosref0155" href="#bib27" role="doc-biblioref" data-xml-rid="bib27">Khong et&nbsp;al., 2017</a>). These studies implicate hundreds of different proteins and thousands of different RNAs as constituents of SGs but do not reveal the mechanism of SG assembly or the relative contribution of individual constituents. Whereas RNA self-assembly contributes to SG assembly (<a id="crosref0160" href="#bib62" role="doc-biblioref" data-xml-rid="bib62">Van Treeck et&nbsp;al., 2018</a>), evidence indicates an important role for RBPs as well. Several studies have implicated various RBPs as “essential” to SG assembly, including TIA1 (<a id="crosref0165" href="#bib15" role="doc-biblioref" data-xml-rid="bib15">Gilks et&nbsp;al., 2004</a>), HDAC6 (<a id="crosref0170" href="#bib30" role="doc-biblioref" data-xml-rid="bib30">Kwon et&nbsp;al., 2007</a>), G3BP1 and G3BP2 (<a id="crosref0175" href="#bib26" role="doc-biblioref" data-xml-rid="bib26">Kedersha et&nbsp;al., 2016</a>), PRRC2C (<a id="crosref0180" href="#bib68" role="doc-biblioref" data-xml-rid="bib68">Youn et&nbsp;al., 2018</a>), CSDE1 (<a id="crosref0185" href="#bib68" role="doc-biblioref" data-xml-rid="bib68">Youn et&nbsp;al., 2018</a>), and UBAP2L (<a id="crosref0190" href="#bib34" role="doc-biblioref" data-xml-rid="bib34">Markmiller et&nbsp;al., 2018</a>). Yet insight into the relative importance of these proteins and how they relate to one another to govern assembly has been lacking. Indeed, a key gap in knowledge permitting us to understand the molecular basis of SG assembly and dynamics is lack of a conceptual framework built from fundamental biophysical principles.</div><div id="p0045" role="paragraph">Here, we took an unbiased approach to obtain a holistic view of the determinants of SG formation and how these factors work together to assemble this RNP granule. This approach revealed that SG formation is encoded by a core protein-RNA interaction network. When the collective interactions of this network breach a threshold saturation concentration, LLPS ensues and seeds SG assembly. The driving forces underlying LLPS are distributed unevenly across this network, with some nodes more important than others. G3BP1 and 2 are the nodes of highest centrality within this network and, by virtue of this position, are essential for SG assembly under certain conditions. Biophysical studies revealed details of how G3BP1 functions as a molecular switch that triggers RNA-dependent LLPS. Importantly, interplay between three distinct intrinsically disordered regions (IDRs) in G3BP1 regulates its intrinsic propensity for LLPS, and this is fine-tuned by phosphorylation within the IDRs. Further regulation of SG assembly arises through positive or negative cooperativity by extrinsic G3BP1-binding factors that strengthen or weaken, respectively, the core SG network.</div></section><section id="sec-2"><h2>Results</h2><section id="sec-2-1"><h3>G3BP Proteins Have the Highest Centrality within the Core SG Network</h3><div id="p0050" role="paragraph">To identify genes essential for SG assembly, we conducted a genome-wide RNAi screen in which we targeted 21,121 genes with pools of four different small interfering RNAs (siRNAs) per gene (84,484 total siRNAs) in a U2OS cell line expressing a G3BP1-GFP reporter (<a id="crosref0195" href="#fig1">Figures 1</a>A and <a id="crosref0200" href="#figs1">S1</a>A–S1E). After confirmation in quadruplicate, we identified 210 genes whose depletion limited SG assembly in response to an exogenous stressor (<a id="crosref0205" href="#mmc1">Table S1</a>). Gene ontological analysis revealed that this group was enriched in genes that regulate RNA metabolism and nucleocytoplasmic transport (<a id="crosref0210" href="#figs1">Figure&nbsp;S1</a>F). Of the 210 genes identified, 81 had been implicated in SG assembly in response to arsenite by a smaller scale RNAi screen targeting 7,317 genes (<a id="crosref0215" href="#bib42" role="doc-biblioref" data-xml-rid="bib42">Ohn et&nbsp;al., 2008</a>). This prior screen also identified 51 genes that regulate arsenite-dependent SG assembly that were not identified in our screen (<a id="crosref0220" href="#figs1">Figure&nbsp;S1</a>G; <a id="crosref0225" href="#mmc1">Table S1</a>). Thirteen genes implicated in SG assembly in previous studies were not identified in either screen; thus, at least 274 genes are implicated in regulating the assembly of SGs in response to a variety of stressors (<a id="crosref0230" href="#fig1">Figure&nbsp;1</a>A). We used these 274 genes to construct an interaction network of genes that regulate SG assembly. This “SG regulator network” was more densely connected (i.e., with higher edge density) than random networks (<a id="crosref0235" href="#fig1">Figure&nbsp;1</a>B), suggesting that this network of genes operates within a common functional module (<a id="crosref0240" href="#bib24" role="doc-biblioref" data-xml-rid="bib24">Jeong et&nbsp;al., 2001</a>).</div><div class="figure-wrap"><figure id="fig1" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2020.03.046/asset/4be50dc5-5261-475f-bae7-155ac4f581ba/main.assets/gr1_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr1" src="/cms/10.1016/j.cell.2020.03.046/asset/c3a67700-7317-4d7a-94b8-5a58cccb5aef/main.assets/gr1.jpg" height="864" width="770" aria-labelledby="fig1-title fig1-content" data-viewer-src="/cms/10.1016/j.cell.2020.03.046/asset/4be50dc5-5261-475f-bae7-155ac4f581ba/main.assets/gr1_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig1-title" class="figure__title"><span class="label figure__label">Figure&nbsp;1</span> <span class="figure__title__text">G3BP Is the Node of Highest Centrality within the Core Stress Granule (SG) Network</span></div><div class="accordion"><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><div id="fig1-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0015" role="paragraph">(A) Approach used to identify the core SG network. A genome-wide siRNA screen was performed in heat-shocked U2OS cells expressing G3BP1-GFP. Results from this screen were combined with published results and integrated with proteomic datasets. Genes that overlapped between the genetic screen and the proteomic datasets were assessed individually by CRISPR-Cas9-based knockout (KO).</div><div id="fspara0020" role="paragraph">(B and C) Network analyses showing connectivity (network edge density) of the SG regulator network (B) and SG constituent network (C). p values were estimated by empirically calculating the probability of observing a denser network by randomly sampling 1,000 subnetworks of similar size from the entire interactome.</div><div id="fspara0025" role="paragraph">(D) The core SG network of 36 core genes identified using the approach illustrated in (A). The color density and size of each node is proportional to its betweenness centrality in the network. Genes tested by CRISPR-Cas9 in (F) are in bold.</div><div id="fspara0030" role="paragraph">(E) Network analysis of 36 core SG genes comparing connectivity with the SG constituent network. The p value was estimated similarly as in (B) and (C) but using the network of SG constituents as background.</div><div id="fspara0035" role="paragraph">(F) U2OS cell lines with CRISPR-Cas9-based single or double KOs. Cells were exposed to sodium arsenite (500&nbsp;μM; 1 h) and SG assembly was assessed by staining with a panel of 17 SG markers.</div><div id="fspara0040" role="paragraph">(G) U2OS cells with KO of <i>G3BP1</i>, <i>G3BP2</i>, or both <i>G3BP1</i> and <i>G3BP2</i> were exposed to sodium arsenite (500&nbsp;μM; 1 h) and stained for the SG marker eIF3η. Scale bar, 20&nbsp;μm.</div><div id="fspara0045" role="paragraph">(H) Comparison of human G3BP1 and G3BP2 proteins. IDR, intrinsically disordered region; NTF2L, NTF2-like; RG-rich, arginine-glycine rich; RRM, RNA-recognition motif.</div><div id="fspara0050" role="paragraph">(I and J) <i>G3BP1/2</i> dKO cells were transfected with GFP-G3BP1 or GFP-G3BP2, exposed to sodium arsenite (500&nbsp;μM; 1 h), and stained for eIF3η. Cells were imaged (I), and the percentage of cells positive for SGs was quantified (J). Error bars indicate SD. <sup>∗∗∗∗</sup>p&nbsp;&lt; 0.0001 by one-way ANOVA; Tukey’s multiple comparisons test. Scale bar, 20&nbsp;μm.</div><div id="fspara0055" role="paragraph">See also <a id="crosref1340" href="#figs1">Figure&nbsp;S1</a> and <a id="crosref1345" href="#mmc1">Tables S1</a> and <a id="crosref1350" href="#mmc2">S2</a>.</div></div></div></div></figcaption></figure></div><div class="figure-wrap"><figure id="figs1" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2020.03.046/asset/ee0bf4e0-e263-4997-b7e4-f5ca1f30294c/main.assets/figs1_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs1" src="/cms/10.1016/j.cell.2020.03.046/asset/1660cc06-f478-4e9f-af12-140cfa4a4625/main.assets/figs1.jpg" height="877" width="770" aria-labelledby="figs1-title figs1-content" data-viewer-src="/cms/10.1016/j.cell.2020.03.046/asset/ee0bf4e0-e263-4997-b7e4-f5ca1f30294c/main.assets/figs1_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs1-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S1</span> <span class="figure__title__text">Genome-wide Screen Followed by CRISPR-Cas9 Knockout Identifies G3BP1/2 as Factors Uniquely Required for SG Assembly, Related to <a id="crosref1430" href="#fig1">Figure&nbsp;1</a></span></div><div class="accordion"><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><div id="figs1-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0425" role="paragraph">(A) Experimental workflow. U2OS cell lines were transfected with siRNA. Forty-eight hours post-transfection, cells were incubated at 43°C for 2 h, then fixed in 4% paraformaldehyde and stained for DAPI.</div><div id="fspara0430" role="paragraph">(B) Workflow for assay development of the high-throughput screen.</div><div id="fspara0435" role="paragraph">(C) Primary screen hits plotted in a waterfall diagram. Hits with SSMD ≤ -1.5 were selected for a further confirmation screen.</div><div id="fspara0440" role="paragraph">(D) Confirmatory screen hits plotted in a waterfall diagram. Hits with SSMD ≤ -1.5 were selected for further deconvolution.</div><div id="fspara0445" role="paragraph">(E) Results from the deconvolution screen showing the number of individual siRNAs (of a total of four siRNAs per gene) that yielded a significant effect on SG assembly.</div><div id="fspara0450" role="paragraph">(F) Gene ontology analysis of the hits of the deconvolution screen. For each functional category, the number of genes in that category that were identified in our screen is shown in blue; these numbers are graphed in proportion to the total number of genes implicated in that functional category in the dataset used for our analysis.</div><div id="fspara0455" role="paragraph">(G) Comparison of genes found in our screen to regulate assembly of SGs during heat shock compared to those previously identified as regulators of SG assembly during arsenite stress (<a id="crosref1435" href="#bib42" role="doc-biblioref" data-xml-rid="bib42">Ohn et&nbsp;al., 2008</a>).</div><div id="fspara0460" role="paragraph">(H) U2OS cell lines were generated using CRISPR-Cas9 to knock out genes identified as potential regulators of SG assembly. Cells were exposed to sodium arsenite (500&nbsp;μM, 1 h), stained for G3BP1 to identify SGs, and imaged. All knockout lines shown retained the ability to form SGs in response to arsenite. 70% of <i>UBAP2</i>/<i>UBAP2L</i> dKO cells showed normal SG assembly, but 30% failed to assemble SGs. Scale bar, 20&nbsp;μm.</div><div id="fspara0465" role="paragraph">(I) WT and <i>G3BP1/2</i> dKO cells were incubated at 43°C for 1&nbsp;h or exposed to 0.4&nbsp;M sorbitol for 1 h, stained for G3BP1 and eIF3η, and imaged. <i>G3BP1/2</i> dKO cells still formed eIF3η-positive foci under both heat shock and osmotic stresses. Scale bar, 10&nbsp;μm.</div><div id="fspara0470" role="paragraph">See also <a id="crosref1440" href="#mmc1">Tables S1</a> and <a id="crosref1445" href="#mmc2">S2</a>.</div></div></div></div></figcaption></figure></div><div id="p0055" role="paragraph">We next integrated this regulatory network with the 411 proteins previously defined as the SG proteome (<a id="crosref0245" href="#bib23" role="doc-biblioref" data-xml-rid="bib23">Jain et&nbsp;al., 2016</a>). This latter group formed a “SG constituent network” with higher edge density than random networks (<a id="crosref0250" href="#fig1">Figure&nbsp;1</a>C), suggesting that this network of proteins also operates within a common functional module. The SG regulator network and SG constituent network had 36 proteins in common, which themselves constitute a network of very high edge density and topological centrality that we have termed the “core SG network” (<a id="crosref0255" href="#fig1">Figures 1</a>D and 1E). Interestingly, this network included all seven proteins that have been described as “essential” to SG assembly: TIA1 (<a id="crosref0260" href="#bib15" role="doc-biblioref" data-xml-rid="bib15">Gilks et&nbsp;al., 2004</a>); HDAC6 (<a id="crosref0265" href="#bib30" role="doc-biblioref" data-xml-rid="bib30">Kwon et&nbsp;al., 2007</a>); G3BP1 and G3BP2 (<a id="crosref0270" href="#bib26" role="doc-biblioref" data-xml-rid="bib26">Kedersha et&nbsp;al., 2016</a>); PRRC2C (<a id="crosref0275" href="#bib68" role="doc-biblioref" data-xml-rid="bib68">Youn et&nbsp;al., 2018</a>); CSDE1 (<a id="crosref0280" href="#bib68" role="doc-biblioref" data-xml-rid="bib68">Youn et&nbsp;al., 2018</a>); and UBAP2L (<a id="crosref0285" href="#bib34" role="doc-biblioref" data-xml-rid="bib34">Markmiller et&nbsp;al., 2018</a>). Also worth noting, this integration of genetic screening and SG proteomics revealed that individual depletion of most SG proteins had no appreciable impact on SG assembly, indicating that, although they are constituents of the SG, they do not contribute significantly to its assembly.</div></section><section id="sec-2-2"><h3>Proteins within the Core SG Network Contribute to SG Assembly to Varying Degrees</h3><div id="p0060" role="paragraph">To examine the importance of the 36 proteins in the core SG network, we used CRISPR/Cas9 to create 23&nbsp;U2OS cell lines harboring knockout of one or both isoforms of these genes and assessed SG assembly in response to arsenite using 17 SG markers (16 protein markers and polyadenylated [polyA] RNA; <a id="crosref0290" href="#mmc2">Table S2</a>; <a id="crosref0295" href="#fig1">Figure&nbsp;1</a>F). To our surprise, nearly all of these genes were <i>not</i> essential for SG assembly (<a id="crosref0300" href="#fig1">Figure&nbsp;1</a>F). Indeed, even cells with double knockout of paralogous isoforms of supposed “essential” genes (e.g., <i>UBAP2</i> and <i>UBAP2L</i>, <i>TIA1</i> and <i>TIAR</i>) still formed SGs (<a id="crosref0305" href="#fig1">Figures 1</a>F and <a id="crosref0310" href="#figs1">S1</a>H). Nevertheless, knockout of genes in the core SG network were consistently associated with reduced and/or delayed SG assembly. In contrast, cells with double knockout of <i>G3BP1</i> and <i>G3BP2</i> (hereafter referred to as <i>G3BP1/2</i> dKO cells) were devoid of SGs (<a id="crosref0315" href="#fig1">Figure&nbsp;1</a>G; <a id="crosref0320" href="#mmc2">Table S2</a>), as previously reported (<a id="crosref0325" href="#bib26" role="doc-biblioref" data-xml-rid="bib26">Kedersha et&nbsp;al., 2016</a>). Notably, G3BP1 and G3BP2 have the highest betweenness centrality within the core SG network (<a id="crosref0330" href="#fig1">Figure&nbsp;1</a>D). Indeed, the impact of each gene deletion on SG assembly correlated with betweenness centrality within the core SG network (data not shown). Nevertheless, whereas <i>G3BP1/2</i> dKO cells formed no SGs in response to arsenite, these cells did assemble SGs in response to heat shock or osmotic stress (<a id="crosref0335" href="#figs1">Figure&nbsp;S1</a>I). Taken together, these observations indicate that the factors driving SG assembly differ depending on the initiating stimulus and suggest that the relative importance of these factors in the assembly process correlates with the degree of their centrality within a core SG network. These results also highlight G3BP as an important node within the network of interactions that underlie SG assembly and as an essential factor in the context of arsenite stress. However, the mechanism whereby G3BP1 and G3BP2 contribute to SG assembly is unknown, and elucidating this became the focus of our study.</div><div id="p0065" role="paragraph">G3BP1 and G3BP2 are paralogous proteins that share a common domain architecture, with 65% identity and 80% similarity in primary amino acid sequence (<a id="crosref0340" href="#fig1">Figure&nbsp;1</a>H). Expression of GFP-G3BP1 or GFP-G3BP2 were each sufficient to restore SG assembly in response to arsenite in <i>G3BP1/2</i> dKO cells based on 17 markers, indicating that these two proteins are redundant with respect to supporting SG assembly (<a id="crosref0345" href="#fig1">Figures 1</a>I and 1J; <a id="crosref0350" href="#mmc2">Table S2</a>). These data illustrate the utility of <i>G3BP1/2</i> dKO cells in assaying the ability of exogenous G3BP to restore SG assembly. Given the homology and functional redundancy between G3BP1 and G3BP2, we focused most of our subsequent studies on G3BP1.</div></section><section id="sec-2-3"><h3>G3BP1&nbsp;Undergoes RNA-Dependent Phase Separation</h3><div id="p0070" role="paragraph">Many protein constituents of SGs harbor IDRs that facilitate LLPS <i>in&nbsp;vitro</i>, and these findings have led to the proposal that IDRs are important drivers of SG assembly in cells (<a id="crosrefs0025" href="#bib32" role="doc-biblioref" data-xml-rid="bib32 bib39 bib41 bib47">Lin et&nbsp;al., 2015; Molliex et&nbsp;al., 2015; Nott et&nbsp;al., 2015; Patel et&nbsp;al., 2015</a>). Given that G3BP1 has multiple IDRs, we examined whether this protein undergoes LLPS <i>in&nbsp;vitro</i>. To our surprise, at physiological salt concentration (150&nbsp;mM NaCl), we found no evidence of LLPS across a range of G3BP1 concentrations (<a id="crosref0355" href="#fig2">Figures 2</a>A and <a id="crosref0360" href="#figs2">S2</a>A). G3BP1 did exhibit concentration-dependent LLPS upon addition of the molecular crowding agent Ficoll (<a id="crosref0365" href="#fig2">Figures 2</a>A and <a id="crosref0370" href="#figs2">S2</a>A), although the physiological relevance of this mono-component, crowder-dependent LLPS is questionable based on subsequent observations below.</div><div class="figure-wrap"><figure id="fig2" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2020.03.046/asset/8b295377-34f8-4ee8-a4cf-b2574c2a1420/main.assets/gr2_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr2" src="/cms/10.1016/j.cell.2020.03.046/asset/d55b2b7c-22df-4bdf-b04f-9b5f3144a273/main.assets/gr2.jpg" height="964" width="768" aria-labelledby="fig2-title fig2-content" data-viewer-src="/cms/10.1016/j.cell.2020.03.046/asset/8b295377-34f8-4ee8-a4cf-b2574c2a1420/main.assets/gr2_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig2-title" class="figure__title"><span class="label figure__label">Figure&nbsp;2</span> <span class="figure__title__text">G3BP1&nbsp;Undergoes LLPS with RNAs that Have Specific Features</span></div><div class="accordion"><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><div id="fig2-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0065" role="paragraph">(A) Summary of phase separation behaviors of G3BP1 FL with increasing concentration of Ficoll (crowding agent). Corresponding images are shown in <a id="crosref1355" href="#figs2">Figure&nbsp;S2</a>A.</div><div id="fspara0070" role="paragraph">(B and C) LLPS of purified recombinant G3BP1 in the absence of a crowding agent and increasing concentrations of total RNA purified from human cells. Scale bar, 20&nbsp;μm.</div><div id="fspara0075" role="paragraph">(D) LLPS of purified recombinant G3BP1 with 100&nbsp;ng/μL total RNA, with or without the addition of 1&nbsp;μg/mL RNase A. Scale bar, 50&nbsp;μm.</div><div id="fspara0080" role="paragraph">(E) LLPS of purified recombinant G3BP1 with the addition of RNA species as indicated. Scale bar, 20&nbsp;μm.</div><div id="fspara0085" role="paragraph">(F) LLPS of purified recombinant G3BP1 with increasing concentrations of total (left panels) or polyA (right panels) RNA. Scale bars, 50&nbsp;μm.</div><div id="fspara0090" role="paragraph">(G) Summary of phase separation behaviors of G3BP1 in (F).</div><div id="fspara0095" role="paragraph">(H) PolyA pull-down from U2OS cells exposed to sodium arsenite (500&nbsp;μM; 1 h), revealing RNA (SYBR Gold staining) and G3BP1 pull-down (immunoblot). Representative image from 4 experiments is shown. The level of G3BP1 pulled down by polyA RNA was normalized to the total RNA level. Quantification was performed using 4 replicates. Error bar indicates SD. <sup>∗∗</sup>p&nbsp;= 0.0038 by unpaired t test.</div><div id="fspara0100" role="paragraph">(I) LLPS of purified recombinant G3BP1 in the absence of a crowding agent and the addition of mRNAs as indicated. Scale bar, 20&nbsp;μm.</div><div id="fspara0105" role="paragraph">(J) LLPS of purified recombinant G3BP1 in the presence of long double strands of poly(I:C) or poly(A:U). Scale bar, 20&nbsp;μm.</div><div id="fspara0110" role="paragraph">(K) <i>In&nbsp;vitro</i> transcribed <i>HSPA8</i> sense-strand RNA of different lengths was assessed by agarose gel and ethidium bromide staining.</div><div id="fspara0115" role="paragraph">(L) <i>In&nbsp;vitro</i> transcribed <i>HSPA8</i> sense-strand RNA was mixed with 200&nbsp;μM recombinant G3BP1 to assess its ability to trigger LLPS in the absence of a crowding agent. Scale bar, 20&nbsp;μm.</div><div id="fspara0120" role="paragraph">(M) Differential interference contrast (DIC) images and RNA gels showing effect of helicase pre-treatment of RNA on G3BP1-RNA LLPS. Scale bar, 10&nbsp;μm.</div><div id="fspara0125" role="paragraph">All <i>in&nbsp;vitro</i> LLPS experiments were performed in 150&nbsp;mM NaCl.</div><div id="fspara0130" role="paragraph">See also <a id="crosref1360" href="#figs2">Figure&nbsp;S2</a> and <a id="crosref1365" href="#mmc4">Video S1</a>.</div></div></div></div></figcaption></figure></div><div class="figure-wrap"><figure id="figs2" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2020.03.046/asset/cd196854-6d3f-4ffa-a46c-79e73038b262/main.assets/figs2_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs2" src="/cms/10.1016/j.cell.2020.03.046/asset/add33679-8ccf-4d2a-87e9-71d61b1baa3f/main.assets/figs2.jpg" height="849" width="657" aria-labelledby="figs2-title figs2-content" data-viewer-src="/cms/10.1016/j.cell.2020.03.046/asset/cd196854-6d3f-4ffa-a46c-79e73038b262/main.assets/figs2_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs2-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S2</span> <span class="figure__title__text">G3BP1&nbsp;Undergoes RNA-Dependent Phase Separation, Related to <a id="crosref1450" href="#fig2">Figure&nbsp;2</a></span></div><div class="accordion"><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><div id="figs2-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0480" role="paragraph">(A) LLPS of purified recombinant G3BP1 full-length (FL) protein was assessed in the context of increasing concentrations of the crowding reagent Ficoll and increasing concentrations of protein. LLPS of G3BP1 was dependent on G3BP1 concentration and the concentration of Ficoll, Scale bar, 20&nbsp;μm.</div><div id="fspara0485" role="paragraph">(B) U2OS cells stably expressing G3BP1-GFP were exposed to sodium arsenite (500&nbsp;μM) and further treated with 0.2% Tween 20 in PBS for 10&nbsp;min. Cells were imaged in PBS alone or with RNase I (1&nbsp;U/μl)&nbsp;+ RNase A (100&nbsp;μg/ml) in PBS and imaged for indicated time. Scale bar, 50&nbsp;μm.</div><div id="fspara0490" role="paragraph">(C) <i>G3BP1/2</i> dKO cells were transiently transfected with FLAG-G3BP1 and exposed to sodium arsenite (500&nbsp;μM, 1 h). FLAG-G3BP1 was isolated and the RNA-binding affinity of FLAG-G3BP1 was assessed by EMSA. Free and bound RNA fractions were quantified and plotted as a function of relative concentrations of FLAG-G3BP1. Error bars indicate SD.</div><div id="fspara0495" role="paragraph">(D) 5&nbsp;μM GST-G3BP1 WT was incubated with 50&nbsp;ng/μl of indicated RNA at room temperature and exposed to UV. Complexes were treated with 4&nbsp;U/ml RNase I for 3&nbsp;min and isolated by GST pulldown. RNA was then biotinylated for further detection. Complexes were analyzed by anti-GST immunoblot to detect G3BP1and streptavidin to detect RNA.</div><div id="fspara0500" role="paragraph">(E) LLPS of purified recombinant G3BP1 was assessed in the absence of a crowding reagent, 150&nbsp;mM NaCl, and the addition of long single-stranded PolyA, PolyG, PolyC, or PolyU.</div><div id="fspara0505" role="paragraph">See also <a id="crosref1455" href="#mmc4">Video S1</a>.</div></div></div></div></figcaption></figure></div><div id="p0075" role="paragraph">Because SG assembly correlates with a rise in the cytoplasmic concentration of uncoated mRNAs upon arrest of translation (<a id="crosref0375" href="#bib45" role="doc-biblioref" data-xml-rid="bib45">Panas et&nbsp;al., 2016</a>), we next characterized the influence of RNA on LLPS of G3BP1. We purified total RNA from U2OS cells and added increasing concentrations of this RNA to G3BP1 protein <i>in&nbsp;vitro</i> in the absence of Ficoll and found that this RNA strongly promoted LLPS of G3BP1 (<a id="crosref0380" href="#fig2">Figures 2</a>B and 2C). The addition of RNase A rapidly reversed RNA-triggered LLPS <i>in&nbsp;vitro</i> (<a id="crosref0385" href="#fig2">Figure&nbsp;2</a>D) and disrupted SGs in cells (<a id="crosref0390" href="#figs2">Figure&nbsp;S2</a>B; <a id="crosref0395" href="#mmc4">Video S1</a>), confirming the role of RNA in enhancing LLPS of G3BP1.</div><div id="p0080" role="paragraph"><div id="mmc4" class="core-supplementary-material" data-category="video"><div class="core-link"><div class="cloudflare-stream-container" id="mmc4" data-type="mp4"><span id="video_stream_mmc4_src" class="video-stream-source" style="display:none">eyJraWQiOiI4ZjUxYWNhY2IzYjhiNjNlNzFlYmIzYWFmYTU5NmZmYyIsImFsZyI6IlJTMjU2In0.eyJzdWIiOiJkYzFkMTRjNzE4NDQ5Yjk1NmQ1ZTc0ZmNjNmNhNTRhMCIsImtpZCI6IjhmNTFhY2FjYjNiOGI2M2U3MWViYjNhYWZhNTk2ZmZjIiwiZXhwIjoxNzQ5NjA3NjA5fQ.BcyQv3PGsBDOUxqiPkYrWNwIaDT0DDjYM-jIF3fX1tS6I1KHTXN09kQVon3acIMuabEniep4mfGN-cimw_1LjzOZph52cqpmBvhbgxbg-RCtvG7WPtcSxv6Qcczk-ZDZxAvwEXJiyR4GZAXb99uFPRfRzkP_ewcjH8ga_TcFRo9SWX0fsPfyFauEwMXHUjnmfDZIP8KAxgvL_CQbuvYEa4nyW0r8mguegw9c71Tbscme4nklYeVGhoJc0_AhXbZgj1CtZxgL1mQGjy2afp1EIRDj8Q7Ve-hO4zmHhO-lQNl4EnwDpvWOfCblc36Olpf0Zgkqa_G00Cc0M7JsO0AAUA</span><stream src="eyJraWQiOiI4ZjUxYWNhY2IzYjhiNjNlNzFlYmIzYWFmYTU5NmZmYyIsImFsZyI6IlJTMjU2In0.eyJzdWIiOiJkYzFkMTRjNzE4NDQ5Yjk1NmQ1ZTc0ZmNjNmNhNTRhMCIsImtpZCI6IjhmNTFhY2FjYjNiOGI2M2U3MWViYjNhYWZhNTk2ZmZjIiwiZXhwIjoxNzQ5NjA3NjA5fQ.BcyQv3PGsBDOUxqiPkYrWNwIaDT0DDjYM-jIF3fX1tS6I1KHTXN09kQVon3acIMuabEniep4mfGN-cimw_1LjzOZph52cqpmBvhbgxbg-RCtvG7WPtcSxv6Qcczk-ZDZxAvwEXJiyR4GZAXb99uFPRfRzkP_ewcjH8ga_TcFRo9SWX0fsPfyFauEwMXHUjnmfDZIP8KAxgvL_CQbuvYEa4nyW0r8mguegw9c71Tbscme4nklYeVGhoJc0_AhXbZgj1CtZxgL1mQGjy2afp1EIRDj8Q7Ve-hO4zmHhO-lQNl4EnwDpvWOfCblc36Olpf0Zgkqa_G00Cc0M7JsO0AAUA" class="cloudflare-stream-player ready tracked" id="video_stream_mmc4" controls="true"><div class="player-share-button" title="Share Video"><div class="share-player-icon"></div></div><div class="player-share-container hidden"><span class="player-close-share"></span><div class="social-links-container"><ul class="social-links-list"><li class="html5-share-facebook-icon" data-share-url="https://www.facebook.com/sharer/sharer.php?u=https://www.cell.com/cell/fulltext/S0092-8674(20)30339-1" target="_blank"></li><li class="html5-share-twitter-icon" data-share-url="https://twitter.com/share?url=https://www.cell.com/cell/fulltext/S0092-8674(20)30339-1" target="_blank"></li><li class="html5-share-mail-icon" data-share-url="mailto:?body=https://www.cell.com/cell/fulltext/S0092-8674(20)30339-1" target="_blank"></li></ul><input type="text" value="https://www.cell.com/cell/fulltext/S0092-8674(20)30339-1" readonly="readonly" class="share-input"></div></div><div style="position: relative; height: 0px;"><iframe src="https://embed.cloudflarestream.com/embed/iframe.fla9.823df22.html?videoId=eyJraWQiOiI4ZjUxYWNhY2IzYjhiNjNlNzFlYmIzYWFmYTU5NmZmYyIsImFsZyI6IlJTMjU2In0.eyJzdWIiOiJkYzFkMTRjNzE4NDQ5Yjk1NmQ1ZTc0ZmNjNmNhNTRhMCIsImtpZCI6IjhmNTFhY2FjYjNiOGI2M2U3MWViYjNhYWZhNTk2ZmZjIiwiZXhwIjoxNzQ5NjA3NjA5fQ.BcyQv3PGsBDOUxqiPkYrWNwIaDT0DDjYM-jIF3fX1tS6I1KHTXN09kQVon3acIMuabEniep4mfGN-cimw_1LjzOZph52cqpmBvhbgxbg-RCtvG7WPtcSxv6Qcczk-ZDZxAvwEXJiyR4GZAXb99uFPRfRzkP_ewcjH8ga_TcFRo9SWX0fsPfyFauEwMXHUjnmfDZIP8KAxgvL_CQbuvYEa4nyW0r8mguegw9c71Tbscme4nklYeVGhoJc0_AhXbZgj1CtZxgL1mQGjy2afp1EIRDj8Q7Ve-hO4zmHhO-lQNl4EnwDpvWOfCblc36Olpf0Zgkqa_G00Cc0M7JsO0AAUA" allowfullscreen="true" allow="autoplay; picture-in-picture" style="height: 100%; width: 100%; position: absolute; top: 0px; left: 0px; border: none;"></iframe></div></stream></div><a href="/cms/10.1016/j.cell.2020.03.046/attachment/7f844485-84d3-4fc3-ad7f-e7d3700176ed/mmc4.mp4" download="mmc4.mp4" aria-labelledby="mmc4-heading mmc4-link"><span id="mmc4-link">Video (1.12 MB)</span></a></div><div class="core-description"><div id="mmc4-heading" class="heading">Video S1. RNase Treatment Leads to Dissolution of Stress Granules, Related to Figures 2 and S2</div><div id="espara0010" role="paragraph">U2OS cells stably expressing G3BP1-GFP were exposed to sodium arsenite (500&nbsp;μM) and further treated with 0.2% Tween 20 in PBS for 10&nbsp;min. Cells were imaged in PBS alone or with RNase I (1&nbsp;U/μl)&nbsp;+ RNase A (100&nbsp;μg/ml) in PBS as indicated.</div></div></div></div><div id="p0085" role="paragraph">Notably, whereas neither polyA-depleted cellular RNA nor tRNA triggered LLPS of G3BP1 in the absence of Ficoll (<a id="crosref0400" href="#fig2">Figure&nbsp;2</a>E), the polyA fraction of total cellular RNA triggered LLPS of G3BP1 nearly as potently as total RNA (<a id="crosref0405" href="#fig2">Figures 2</a>F and 2G). To pursue this finding in cells, we performed polyA RNA pull-down from U2OS cells. This pull-down recovered low levels of endogenous G3BP1, which increased upon arsenite treatment, indicating increased G3BP1-mRNA interaction in response to stress (<a id="crosref0410" href="#fig2">Figure&nbsp;2</a>H). Consistent results were obtained with G3BP2 (data not shown). The binding affinity of G3BP1 for mRNA was not significantly altered after arsenite treatment (<a id="crosref0415" href="#figs2">Figure&nbsp;S2</a>C), suggesting that the observed increase in G3BP1-mRNA interaction reflects increased availability of mRNA under stressed conditions. To further investigate the G3BP1-mRNA interaction, we used <i>in&nbsp;vitro</i> transcription to generate transcripts of four mRNAs found in SGs (<i>ACTB</i>, <i>HSPA8</i>, <i>PABPC1</i>, and <i>EEF2</i>; <a id="crosref0420" href="#bib27" role="doc-biblioref" data-xml-rid="bib27">Khong et&nbsp;al., 2017</a>). Both sense and antisense transcripts of these four mRNAs strongly triggered LLPS of G3BP1 (<a id="crosref0425" href="#fig2">Figure&nbsp;2</a>I). However, annealing the sense and antisense transcripts to generate double-stranded RNA abolished the effect on LLPS with G3BP1 (<a id="crosref0430" href="#fig2">Figure&nbsp;2</a>I). We also found that long (∼1–10 kB) homopolymeric RNAs (polyA, polyC, polyU, and polyG) promoted LLPS by G3BP1 (<a id="crosref0435" href="#figs2">Figures S2</a>D and S2E). However, long double strands of poly(I:C) and poly(A:U) failed to undergo LLPS with G3BP1 (<a id="crosref0440" href="#fig2">Figure&nbsp;2</a>J). From these results, we conclude that regions of single strandedness in RNA, but not specific sequence motifs, are required to trigger LLPS with G3BP1, consistent with previous reports that G3BP1 binds RNA in cells without clear preference for a consensus sequence (<a id="crosrefs0030" href="#bib11" role="doc-biblioref" data-xml-rid="bib11 bib36">Edupuganti et&nbsp;al., 2017; Martin et&nbsp;al., 2016b</a>).</div><div id="p0090" role="paragraph">To determine whether mRNA length influences LLPS with G3BP1, we <i>in&nbsp;vitro</i> transcribed full-length <i>HSPA8</i> (∼2 kb) or smaller fragments thereof, covering the length of this transcript. We found a strong correlation between mRNA length and the ability to drive LLPS of G3BP1, with a lower limit of ∼250 nt (<a id="crosref0445" href="#fig2">Figures 2</a>K and 2L). The lower limit mRNA length is about 10 times longer than the minimum RNA length necessary for binding G3BP1 (data not shown). These results may explain why RNA enrichment in SGs appears to be correlated with RNA length rather than sequence motifs (<a id="crosrefs0035" href="#bib27" role="doc-biblioref" data-xml-rid="bib27 bib62">Khong et&nbsp;al., 2017; Van Treeck et&nbsp;al., 2018</a>). Furthermore, pre-treatment of total RNA with the RNA helicase DDX19A (a constituent of the core SG network) and ATP strongly mitigated the ability of RNA to promote LLPS of G3BP1, presumably by limiting intermolecular RNA-RNA interactions (<a id="crosref0450" href="#fig2">Figure&nbsp;2</a>M). In summary, these results indicate that, beyond binding to G3BP1, the features of RNA that favor LLPS with G3BP1 include long length, single-strandedness, and the availability of RNA-RNA interactions. This latter observation is consistent with a recent report implicating RNA-RNA interactions in contributing to SG assembly in cells (<a id="crosref0455" href="#bib61" role="doc-biblioref" data-xml-rid="bib61">Van Treeck and Parker, 2018</a>).</div></section><section id="sec-2-4"><h3><i>In&nbsp;Vitro</i> Phase Separation of G3BP1 with RNA Predicts SG Reconstitution in Cells</h3><div id="p0095" role="paragraph">G3BP1 and G3BP2 have three major domains: an N-terminal NTF2-like (NTF2L) domain; a central segment predicted to be largely disordered; and a C-terminal RNA-binding domain (RBD) (<a id="crosref0460" href="#fig1">Figures 1</a>H, <a id="crosref0465" href="#fig3">3</a>A, <a id="crosref0470" href="#figs3">S3</a>A, and S3B). The long, central IDR of G3BP1 has a non-uniform charge distribution separated by a region of predicted order and may be considered as two segments (<a id="crosref0475" href="#fig3">Figures 3</a>A and 3B): IDR1 (amino acids [aas] 143–225), which is highly negatively charged and highly acidic, and IDR2 (aas 226–334), which is slightly positively charged. The RBD is composed of a folded RNA-recognition motif (RRM) in tandem with an arginine-glycine (RG)-rich IDR that is net positively charged and predicted to be disordered, which we designate IDR3 (<a id="crosref0480" href="#fig3">Figures 3</a>A and 3B).</div><div class="figure-wrap"><figure id="fig3" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2020.03.046/asset/38f266d6-1cc9-43c4-9367-255abbdfa842/main.assets/gr3_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr3" src="/cms/10.1016/j.cell.2020.03.046/asset/91f3d6a8-37a8-4fda-86bd-a2ef23cca6a3/main.assets/gr3.jpg" height="958" width="721" aria-labelledby="fig3-title fig3-content" data-viewer-src="/cms/10.1016/j.cell.2020.03.046/asset/38f266d6-1cc9-43c4-9367-255abbdfa842/main.assets/gr3_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig3-title" class="figure__title"><span class="label figure__label">Figure&nbsp;3</span> <span class="figure__title__text">G3BP1 Phase Separation Correlates with SG Reconstitution</span></div><div class="accordion"><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><div id="fig3-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0140" role="paragraph">(A) G3BP1 domains aligned with results of PONDR (predictor of natural disordered regions) and NCPR (net charge per residue; 5-amino acid window) analyses.</div><div id="fspara0145" role="paragraph">(B) Human G3BP1 protein is shown with major domains marked by highlighted colors. Charged residues are shown in blue (basic; Arg and Lys) and red (acidic; Asp and Glu).</div><div id="fspara0150" role="paragraph">(C) Constructs used to investigate the function of individual domains of G3BP1.</div><div id="fspara0155" role="paragraph">(D) LLPS of purified recombinant G3BP1 in 150&nbsp;mM NaCl and 50&nbsp;ng/μL total RNA and the absence of a crowding agent. Scale bar, 10&nbsp;μm.</div><div id="fspara0160" role="paragraph">(E) Summary of phase separation behaviors of G3BP1 shown in (D).</div><div id="fspara0165" role="paragraph">(F) <i>G3BP1/2</i> dKO cells were transfected with indicated G3BP1 constructs, exposed to sodium arsenite (500&nbsp;μM; 1 h), and stained for eIF3η. Scale bar, 20&nbsp;μm.</div><div id="fspara0170" role="paragraph">(G) LLPS of recombinant G3BP1 proteins in 150&nbsp;mM NaCl, with or without crowding agent or total RNA from human cells. Scale bar, 10&nbsp;μm.</div><div id="fspara0175" role="paragraph">(H) Electrophoretic mobility shift assay of purified G3BP1 proteins showing biotinylated RNA as detected by horseradish peroxidase (HRP)-conjugated streptavidin. Asterisk indicates RNA-G3BP1 complex.</div><div id="fspara0180" role="paragraph">(I) <i>G3BP1/2</i> dKO cells were transfected with indicated G3BP1 constructs, exposed to sodium arsenite (500&nbsp;μM; 1 h), and stained for eIF3η. Scale bar, 20&nbsp;μm.</div><div id="fspara0185" role="paragraph">See also <a id="crosref1370" href="#figs3">Figure&nbsp;S3</a>.</div></div></div></div></figcaption></figure></div><div class="figure-wrap"><figure id="figs3" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2020.03.046/asset/e963abc1-87e8-460c-bce4-736efb8f659c/main.assets/figs3_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs3" src="/cms/10.1016/j.cell.2020.03.046/asset/cc79055d-abce-42f9-a489-98bba94da1f2/main.assets/figs3.jpg" height="728" width="657" aria-labelledby="figs3-title figs3-content" data-viewer-src="/cms/10.1016/j.cell.2020.03.046/asset/e963abc1-87e8-460c-bce4-736efb8f659c/main.assets/figs3_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs3-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S3</span> <span class="figure__title__text"><i>In&nbsp;Vitro</i> Phase Separation of G3BP1 with RNA Strongly Predicts SG Reconstitution in Cells, Related to <a id="crosref1460" href="#fig3">Figure&nbsp;3</a></span></div><div class="accordion"><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><div id="figs3-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0515" role="paragraph">(A) G3BP2 domains are shown aligned with results of PONDR (Predictor of Natural Disordered Regions) and NCPR (net charge per residue; 5-amino acid window) analyses.</div><div id="fspara0520" role="paragraph">(B) The amino acid sequence of human G3BP2 is shown with major domains marked by highlighted colors. Charged residues are shown in blue (basic; Arg and Lys) and red (acidic; Asp and Glu).</div><div id="fspara0525" role="paragraph">(C) LLPS of G3BP1 deletion constructs in which the NTF2L domain, IDR1/2, or RBD have been removed. LLPS of purified recombinant protein was assessed in the presence of a crowding agent. G3BP1 lacking the NTF2L or RBD domains failed to phase separate, whereas G3BP1 lacking IDR1/2 did phase separate. Scale bars, 50&nbsp;μm.</div><div id="fspara0530" role="paragraph">(D) Representation of the phase separation results obtained in (C).</div><div id="fspara0535" role="paragraph">(E) G3BP1 ΔIDR3 fails to assemble SGs. We tested two IDR3 deletion mutants, G3BP1 ΔIDR3 (Δ410-466) and G3BP1 ΔIDR3 (Δ428-466). Although G3BP1 ΔIDR3 (Δ410-466) accumulated in cytoplasmic puncta, these puncta formed in the absence of stress, did not contain SG markers, and did not resolve after the removal of stress, suggesting that these were insoluble aggregates, not SGs. Modifying these boundaries to residue 428 to 466 (G3BP1 ΔIDR3 (Δ428-466)) alleviated this aggregation, but <i>G3BP1/2</i> dKO cells expressing this construct failed to assemble SGs. Scale bar, 20&nbsp;μm.</div><div id="fspara0540" role="paragraph">(F) Puncta formed by G3BP1 ΔIDR3 (410-466) were analyzed by FRAP and showed poor fluorescence recovery. Error bars indicate SD <i><sup>∗∗∗</sup>p &lt;</i> 0.001, <i><sup>∗∗∗∗</sup>p &lt;</i> 0.0001, ns; not significant by two-way ANOVA, Sidak’s multiple comparisons test.</div><div id="fspara0545" role="paragraph">(G) <i>G3BP1/2</i> dKO cells were transfected with GFP-tagged G3BP1 Δ340-415, exposed to sodium arsenite (500&nbsp;μM, 1 h), and stained for eIF3η. The majority of cells transfected with G3BP1 Δ340-415 did not form SGs, whereas few cells showed weak rescue of SG formation. Scale bar, 20&nbsp;μm.</div><div id="fspara0550" role="paragraph">(H) Quantification of result shown in (G). Error bars indicate SD. <i><sup>∗∗</sup>p&nbsp;=</i> 0.0021 by unpaired t test.</div></div></div></div></figcaption></figure></div><div id="p0100" role="paragraph">To understand the contribution of individual domains of G3BP1 to LLPS, we generated constructs in which NTF2L, the long, central IDR (IDR1/2), or the RBD were deleted (<a id="crosref0485" href="#fig3">Figure&nbsp;3</a>C). Deletion of NTF2L or the RBD abolished LLPS of G3BP1 in the presence of RNA (<a id="crosref0490" href="#fig3">Figures 3</a>C–3E). Interestingly, IDR1/2 was dispensable for LLPS. Indeed, deletion of IDR1/2 promoted LLPS at lower threshold concentrations, whereas at higher concentrations, G3BP1 ΔIDR1/2 had poor solubility and was prone to aggregation (<a id="crosref0495" href="#fig3">Figures 3</a>C–3E). Similar results were obtained when RNA was excluded and LLPS was driven by the addition of Ficoll (<a id="crosref0500" href="#figs3">Figures S3</a>C and S3D). We next assessed the ability of G3BP1 deletion mutants to reconstitute SG assembly in <i>G3BP1/2</i> dKO cells in response to arsenite. Consistent with their <i>in&nbsp;vitro</i> behavior, neither GFP-G3BP1 ΔNTF2L nor GFP-G3BP1 ΔRBD restored SG formation, whereas expression of G3BP1 ΔIDR1/2 did restore SG assembly (<a id="crosref0505" href="#fig3">Figure&nbsp;3</a>F). Indeed, despite the frequent observation that IDRs in many RBPs mediate LLPS <i>in&nbsp;vitro</i>, we found that the long central IDR of G3BP1 is dispensable for both LLPS and SG assembly in cells.</div></section><section id="sec-2-5"><h3>Two RNA-Binding Entities in G3BP1 Contribute Independently to SG Assembly</h3><div id="p0105" role="paragraph">The RBD of G3BP1 comprises two distinct RNA-binding entities: an RRM, a stable, folded structure that NMR spectroscopy shows to be composed of amino acids 335–410 (C.M. et&nbsp;al., unpublished data), and IDR3 (amino acids 411–466). To investigate the contributions of these domains to LLPS of G3BP1, we generated G3BP1 constructs in which these domains were individually deleted (<a id="crosref0510" href="#fig3">Figure&nbsp;3</a>C). RNA-dependent LLPS was severely impaired by either of these deletions, whereas Ficoll-dependent LLPS was not appreciably affected (<a id="crosref0515" href="#fig3">Figure&nbsp;3</a>G). Importantly, this observation indicates mechanistic differences in LLPS in the presence of RNA versus Ficoll.</div><div id="p0110" role="paragraph">We next assessed the RNA-binding capacity of these mutants using electrophoretic mobility shift assay (EMSA). Removal of both the RRM and IDR3 domains (ΔRBD) abolished RNA binding (<a id="crosref0520" href="#fig3">Figure&nbsp;3</a>H). Removal of the RRM significantly reduced RNA binding, whereas removal of IDR3 abolished RNA binding (<a id="crosref0525" href="#fig3">Figure&nbsp;3</a>H), consistent with impaired RNA-dependent LLPS of these mutants (<a id="crosref0530" href="#fig3">Figure&nbsp;3</a>G). Interestingly, when introduced into <i>G3BP1/2</i> dKO cells, G3BP1 ΔRRM and ΔIDR3 mutants exhibited severely impaired ability to support SG formation (<a id="crosref0535" href="#fig3">Figures 3</a>I, <a id="crosref0540" href="#figs3">S3</a>E, and S3F). We next further examined the RRM domain by introducing two point mutations (F380/382D) that impair RNA binding by this domain. This mutant also failed to support SG assembly, confirming the importance of RNA binding by the RRM domain (<a id="crosref0545" href="#fig3">Figures 3</a>C and 3I). A prior study reported that the RRM domain is dispensable for SG assembly (<a id="crosref0550" href="#bib26" role="doc-biblioref" data-xml-rid="bib26">Kedersha et&nbsp;al., 2016</a>), although that observation was based on a different deletion (residues 340–415). To address this apparent discrepancy, we recreated the G3BP1 Δ340–415 mutant and quantified rescue of SG assembly in <i>G3BP1/2</i> dKO cells. Despite good expression, we found no assembly of SGs in response to arsenite in ∼70% of cells and few, small SGs in ∼30% of cells (<a id="crosref0555" href="#figs3">Figures S3</a>G and S3H). Thus, using three different RRM mutants, we observed strong impairment of SG formation. Based on these observations, we conclude that both the RRM domain and IDR3 contribute to RNA-dependent LLPS and SG assembly.</div></section><section id="sec-2-6"><h3>Multivalent RNA Interaction Promotes SG Assembly</h3><div id="p0115" role="paragraph">We next conducted domain-swapping experiments to investigate the mechanism whereby RNA binding by G3BP1 contributes to SG assembly. We exchanged the RBD of G3BP1 with five different classes of RBDs: the tandem RRMs from hnRNPA1 and hnRNPA2B1; the zinc finger motifs from RBM22 and ZC3H11A; the KH domains from NOVA and QKI; the YTH domains from YTHDF1, YTHDF2, and YTHDF3 (three proteins that specifically bind m6A-RNA; <a id="crosref0560" href="#bib76" role="doc-biblioref" data-xml-rid="bib76">Zhao et&nbsp;al., 2017</a>); and the double-stranded RBDs (dsRBDs) from ADAR1 and STAU1 (<a id="crosref0565" href="#fig4">Figure&nbsp;4</a>A). Despite similar levels of expression (<a id="crosref0570" href="#figs4">Figure&nbsp;S4</a>A), only the tandem RRMs from hnRNPA1 and hnRNPA2B1 reconstituted SG assembly in <i>G3BP1/2</i> dKO cells (<a id="crosref0575" href="#fig4">Figure&nbsp;4</a>A). Importantly, the ability of these chimeric proteins to reconstitute SG assembly did not correlate with RNA-binding capacity (<a id="crosref0580" href="#figs4">Figure&nbsp;S4</a>B).</div><div class="figure-wrap"><figure id="fig4" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2020.03.046/asset/b90016e2-f02a-461f-a1fc-5dfe87bc4f9a/main.assets/gr4_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr4" src="/cms/10.1016/j.cell.2020.03.046/asset/369551fd-09dd-489d-bd32-4ea48a0ef326/main.assets/gr4.jpg" height="846" width="770" aria-labelledby="fig4-title fig4-content" data-viewer-src="/cms/10.1016/j.cell.2020.03.046/asset/b90016e2-f02a-461f-a1fc-5dfe87bc4f9a/main.assets/gr4_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig4-title" class="figure__title"><span class="label figure__label">Figure&nbsp;4</span> <span class="figure__title__text">Multivalency in RNA Binding Mediates SG Formation</span></div><div class="accordion"><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><div id="fig4-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0195" role="paragraph">(A–C) <i>G3BP1/2</i> dKO cells were transfected with G3BP1 constructs in which the RBD was substituted with different types of RBDs as indicated. Cells were exposed to sodium arsenite (500&nbsp;μM; 1 h) and stained for eIF3η. Scale bars, 20&nbsp;μm.</div><div id="fspara0200" role="paragraph">(D) Cells from (C) were imaged, and the percentage of cells positive for SGs was quantified. Error bars indicate SD. <sup>∗∗∗∗</sup>p&nbsp;&lt; 0.0001 versus no KH by one-way ANOVA; Dunnett’s multiple comparisons test.</div><div id="fspara0205" role="paragraph">(E) <i>G3BP1/2</i> dKO cells were transfected with indicated G3BP1 constructs. Immunoprecipitated G3BP1 cross-linked to RNA was assessed by immunoblotting for biotin (RNA) and GFP (G3BP1). Binding of RNA was quantified and normalized to protein signal. Results show quantification of triplicate experiments. Error bars indicate SD. <sup>∗∗∗∗</sup>p&nbsp;&lt; 0.0001 versus FL by one-way ANOVA; Dunnett’s multiple comparisons test.</div><div id="fspara0210" role="paragraph">(F) RNA-binding proteins with tandem RRM motifs are shown together with their top binding motif and domain structure.</div><div id="fspara0215" role="paragraph">(G) <i>G3BP1/2</i> dKO cells were transfected with G3BP1 constructs in which the RBD was substituted with indicated tandem RRM domains. Cells were exposed to sodium arsenite and stained as in (A). Scale bar, 20&nbsp;μm.</div><div id="fspara0220" role="paragraph">(H) <i>G3BP1/2</i> dKO cells were transfected with indicated G3BP1 constructs and analyzed as in (E). Green bars correspond to functional swap mutants for SG formation; orange bars correspond to non-functional swap mutants. Results show quantification from triplicate experiments. Error bars indicate SD.</div><div id="fspara0225" role="paragraph">See also <a id="crosref1375" href="#figs4">Figure&nbsp;S4</a>.</div></div></div></div></figcaption></figure></div><div class="figure-wrap"><figure id="figs4" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2020.03.046/asset/c2a80fef-622f-48d2-ad3c-71e1daa792b0/main.assets/figs4_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs4" src="/cms/10.1016/j.cell.2020.03.046/asset/79e670d6-99ff-4a2f-b187-8db5f4ea3e4b/main.assets/figs4.jpg" height="958" width="701" aria-labelledby="figs4-title figs4-content" data-viewer-src="/cms/10.1016/j.cell.2020.03.046/asset/c2a80fef-622f-48d2-ad3c-71e1daa792b0/main.assets/figs4_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs4-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S4</span> <span class="figure__title__text">Multivalency in RNA Binding Mediates SG Formation, Related to <a id="crosref1465" href="#fig4">Figure&nbsp;4</a></span></div><div class="accordion"><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><div id="figs4-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0560" role="paragraph">(A) <i>G3BP1/2</i> dKO cells were transfected with GFP-G3BP1 FL or indicated GFP-G3BP1 mutants. Twenty-four hours post-transfection, cells were lysed in 1x NuPAGE sample buffer and subjected to SDS-PAGE and immunoblotting for GFP (detection of G3BP1) and GAPDH (loading control).</div><div id="fspara0565" role="paragraph">(B) <i>G3BP1/2</i> dKO cells were transfected with GFP-G3BP1 FL or indicated GFP-G3BP1 mutants. Cellular protein-RNA complexes were cross-linked with UV and cells were lysed. Immunoprecipitated RNA cross-linked to G3BP1 was then treated with alkaline phosphatase and biotinylated. Complexes were subjected to SDS-PAGE and immunoblotting for biotin (detection of RNA) and GFP (detection of G3BP1). RNA binding was quantified and normalized to protein signal. Results show quantification from duplicate experiments. Error bars indicate SD. <sup>∗</sup>p&nbsp;= 0.0136 versus FL by one-way ANOVA, Dunnett’s multiple comparisons test.</div><div id="fspara0570" role="paragraph">(C) <i>G3BP1/2</i> dKO cells were transfected with GFP-G3BP1 FL or indicated GFP-G3BP1 mutants and assessed by immunoblot as in (A).</div><div id="fspara0575" role="paragraph">(D) <i>G3BP1/2</i> dKO cells were transfected with G3BP1 constructs in which the RBD was substituted with indicated combinations of ZnF domains. Cells were exposed to sodium arsenite (500&nbsp;μM, 1 h) and stained for eIF3η. G3BP1 with a single ZnF domain did not support SG formation, but G3BP1 with two or more ZnF domains did support SG formation. Scale bar, 20&nbsp;μm.</div><div id="fspara0580" role="paragraph">(E) SGs formed by indicated constructs were analyzed by FRAP. SGs assembled with 3 KH or ZnF domains were significantly less dynamic than those assembled with 2 KH or ZnF domains. Error bars indicate SD <i><sup>∗∗∗∗</sup>p &lt;</i> 0.0001 by two-way ANOVA, Sidak’s multiple comparisons test.</div><div id="fspara0585" role="paragraph">(F) <i>G3BP1/2</i> dKO cells were transfected with GFP-G3BP1 FL or G3BP1 2×RBD and exposed to sodium arsenite (500&nbsp;μM). SG formation was monitored using live cell imaging beginning when arsenite was added to the medium. Duplication of the G3BP1 RBD dramatically enhanced SG assembly. Scale bar, 20&nbsp;μm.</div><div id="fspara0590" role="paragraph">(G) <i>G3BP1/2</i> dKO cells were transfected with GFP-G3BP1 FL or G3BP1 2×RBD for 24 h. SGs were counted as spontaneous (no arsenite) or arsenite-induced (500&nbsp;μM, 1 h). Cells were immediately fixed with 4% PFA and representative images were shown for each condition. Scale bar, 50&nbsp;μm.</div><div id="fspara0595" role="paragraph">(H) Quantification of (G). The percentage of cells with spontaneous or arsenite-induced SGs are shown. Cells with at least 3 granules were counted as a SG-positive cell. <sup>∗</sup>p&nbsp;= 0.0431 by two-way ANOVA, Sidak’s multiple comparisons test. Error bars indicate SD.</div><div id="fspara0600" role="paragraph">(I) The ratio of GFP signal in SGs to the entire cell (GFP enrichment in SGs) from results obtained in (G). The SG enrichment percentage of G3BP1 FL protein and G3BP1 2×RBD protein were quantified by CellProfiler as previously described (<a id="crosref1470" href="#bib33" role="doc-biblioref" data-xml-rid="bib33">Mackenzie et&nbsp;al., 2017</a>). Briefly, the cell boundary and the boundary of SGs were identified by CellProfiler. The ratios of GFP fluorescent signal intensity in the SGs to that in the entire cell were used as indicators for the SG enrichment percentages of these two proteins. At least 100 cells were included in each condition. <sup>∗∗∗∗</sup>p&nbsp;&lt; 0.0001 by unpaired t test. Error bars indicate SD.</div><div id="fspara0605" role="paragraph">(J) SGs formed by G3BP1 2×RBD were analyzed by FRAP and showed poor fluorescence recovery. Error bars indicate SD <i><sup>∗∗∗∗</sup>p &lt;</i> 0.0001 by two-way ANOVA, Sidak’s multiple comparisons test.</div><div id="fspara0610" role="paragraph">(K) <i>G3BP1/2</i> dKO cells were transfected with GFP-G3BP1 FL or indicated GFP-G3BP1 mutants. 16 to 24 hours post-transfection, cells were lysed in 1x NuPAGE sample buffer and subjected to SDS-PAGE and immunoblotting for GFP (detection of G3BP1) and GAPDH (loading control).</div></div></div></div></figcaption></figure></div><div id="p0120" role="paragraph">Because two distinct RNA-binding entities (i.e., RRM and IDR3) are important for SG assembly (<a id="crosref0585" href="#fig3">Figure&nbsp;3</a>I), and the only domain swaps that successfully replaced the native RBD were also multivalent (i.e., tandem RRM1-RRM2 of hnRNPA1 and hnRNPA2B1; <a id="crosref0590" href="#fig4">Figure&nbsp;4</a>A), we hypothesized that the RNA-binding valency of G3BP1 may be an important feature in SG assembly. To test this idea, we reduced the RNA-binding valency of the G3BP1 chimera with tandem RRM domains. Deletion of either RRM1 or RRM2, or introduction of missense mutations that interrupt RNA binding by either or both RRM1 and RRM2, all abolished the ability of the G3BP1 chimera to support SG formation (<a id="crosref0595" href="#fig4">Figures 4</a>B and <a id="crosref0600" href="#figs4">S4</a>C). As a further test, we increased RNA-binding valency by swapping the native RBD of G3BP1 with one, two, or three KH domains from QKI (<a id="crosref0605" href="#fig4">Figures 4</a>C and 4D). Whereas a single KH domain substitution restored the RNA-binding capacity to that of G3BP1 wild-type (WT) (<a id="crosref0610" href="#fig4">Figure&nbsp;4</a>E), this was not sufficient to support SG assembly. In contrast, substitution with two or three KH domains reconstituted SG formation (<a id="crosref0615" href="#fig4">Figures 4</a>C and 4D), despite lower level expression of these proteins (<a id="crosref0620" href="#figs4">Figure&nbsp;S4</a>C). We found identical results upon increasing the valency of ZnF domains (<a id="crosref0625" href="#figs4">Figure&nbsp;S4</a>D). Based on these observations, we conclude that multivalency for RNA binding contributes to RNA-dependent LLPS and SG assembly.</div><div id="p0125" role="paragraph">Interestingly, whereas the efficiency of SG reconstitution was equivalent with the introduction of two versus three KH or ZnF domains, and the SGs were morphologically indistinguishable, the material properties of these SGs were different. Specifically, SGs assembled with three KH or ZnF domains were significantly less dynamic than those assembled with two KH or ZnF domains (<a id="crosref0630" href="#figs4">Figure&nbsp;S4</a>E), suggesting that the material properties of SGs are influenced by the degree of association of G3BP1 with RNA. We also examined the consequence of duplicating the G3BP1 RBD domain (2×RBD). G3BP1 2×RBD, which has double valency and RNA-binding capacity, efficiently reconstituted SGs in <i>G3BP1/2</i> dKO cells, but these were also less dynamic (<a id="crosref0635" href="#figs4">Figures S4</a>F–S4J). This inverse correlation between fluidity of a biomolecular condensate and interaction valency of its constituents is consistent with observations <i>in&nbsp;vitro</i> (<a id="crosref0640" href="#bib31" role="doc-biblioref" data-xml-rid="bib31">Li et&nbsp;al., 2012</a>) and may reflect a general principle controlling material properties of biomolecular condensates.</div></section><section id="sec-2-7"><h3>SG Assembly Does Not Require G3BP1 Interaction with Specific RNA Sequence Motifs</h3><div id="p0130" role="paragraph">The observation that different RNA-binding entities can substitute for the native RBD of G3BP1 suggests that no specific RNA motif interactions or RNA targets are essential for SG assembly. To investigate this further, we generated G3BP1 chimeras using six additional multivalent RBDs (<a id="crosref0645" href="#fig4">Figure&nbsp;4</a>F). When expressed at equivalent levels (<a id="crosref0650" href="#figs4">Figure&nbsp;S4</a>K), the ability of these RBDs to support SG assembly did not correlate with any motif interaction preferences but rather with relative RNA binding capacity (<a id="crosref0655" href="#fig4">Figures 4</a>G and 4H). Thus, the ability of G3BP1 to support SG assembly requires not only multivalent interactions with RNA but also sufficient RNA-binding capacity above a threshold concentration. We note that IDR3 has been reported to interact with two ribosomal proteins (RPS6 and RPS23), leading to the suggestion that G3BP1 interaction with the 40S ribosome may be important for SG assembly (<a id="crosref9660" href="#bib26" role="doc-biblioref" data-xml-rid="bib26">Kedersha et&nbsp;al., 2016</a>). The ability of multiple heterologous RNA-binding entities to replace IDR3 and reconstitute SG assembly suggests that IDR3 interaction with ribosomal proteins may not be essential for supporting SG assembly.</div></section><section id="sec-2-8"><h3>The NTF2L Domain Mediates Dimerization without Significant Higher Order Oligomerization</h3><div id="p0135" role="paragraph">Previous studies have reported conflicting results regarding higher order oligomerization (i.e., trimers and tetramers) of G3BP1 via the NTF2L domain (<a id="crosrefs0040" href="#bib57" role="doc-biblioref" data-xml-rid="bib57 bib63">Tourrière et&nbsp;al., 2003; Vognsen et&nbsp;al., 2013</a>). Given the frequency of oligomerization as a strategy to promote LLPS, including precedent for higher order oligomerization of dimers to promote LLPS (<a id="crosref0665" href="#bib37" role="doc-biblioref" data-xml-rid="bib37">Marzahn et&nbsp;al., 2016</a>), we sought to clarify the ability of the NTF2L domain to mediate the assembly of multimers greater than dimers. Size exclusion chromatography coupled to multi-angle light scattering revealed exclusively dimer formation by G3BP1 across a range of concentrations (<a id="crosref0670" href="#figs5">Figure&nbsp;S5</a>A). Analytical ultracentrifugation analysis of purified G3BP1 at 50&nbsp;μM confirmed the dimeric state of the protein, with less than 2% found in higher order species (<a id="crosref0675" href="#figs5">Figures S5</a>B and S5C). Finally, using <i>in&nbsp;vitro</i> cross-linking of purified protein with bissulfosuccinimidyl suberate (BS<sup>3</sup>), we confirmed the role of NTF2L in mediating this dimerization and saw no significant higher order oligomerization (<a id="crosref0680" href="#figs5">Figure&nbsp;S5</a>D). Consistent with these results, BS<sup>3</sup>-cross-linked G3BP1 from cells with or without arsenite treatment confirmed the NTF2L domain-dependent dimeric state of the purified protein and no significant higher order oligomerization (<a id="crosref0685" href="#figs5">Figures S5</a>E and S5F).</div><div class="figure-wrap"><figure id="figs5" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2020.03.046/asset/9189d11b-399f-40cc-8e19-e551e10b7308/main.assets/figs5_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs5" src="/cms/10.1016/j.cell.2020.03.046/asset/b088afbc-004d-4f7a-bf01-4bd4e6af5b08/main.assets/figs5.jpg" height="931" width="770" aria-labelledby="figs5-title figs5-content" data-viewer-src="/cms/10.1016/j.cell.2020.03.046/asset/9189d11b-399f-40cc-8e19-e551e10b7308/main.assets/figs5_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs5-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S5</span> <span class="figure__title__text">NTF2L Domain-Mediated Dimerization and Interaction with Core SG Components Regulate Assembly of SGs, Related to <a id="crosref1475" href="#fig5">Figure&nbsp;5</a></span></div><div class="accordion"><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><div id="figs5-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0620" role="paragraph">(A) Size exclusion chromatography (SEC) multi-angle light scattering (MALS) on purified G3BP1 at concentrations of 20, 50, or 100&nbsp;μM protein. Inset shows a distribution of average molecular weights superimposed on the chromatogram of UV absorbance at 280&nbsp;nm.</div><div id="fspara0625" role="paragraph">(B) Sedimentation velocity analytical ultracentrifugation (AUC) of G3BP1. The sedimentation velocity profiles (fringe displacement) were fitted to a continuous sedimentation coefficient distribution model. Data were acquired at 1-20&nbsp;min for high molecular mass species and 30-130&nbsp;min for low molecular mass species. Table at right shows (1) total concentration of sample (mg/ml); (2) weight-average sedimentation coefficient <i>s</i><sub><i>20</i></sub> calculated from the 2D <i>c</i>(<i>s</i>, <i>f/f</i><sub><i>0</i></sub>) model with percentage protein amount of total protein in parentheses; (3) standard sedimentation coefficient (<i>s</i><sub><i>20,w</i></sub> -value) in water at 20°C; (4) estimated molar mass calculated from (<i>s</i><sub><i>w</i></sub>, <i>f/f</i><sub><i>0,w</i></sub>) from the 2D <i>c</i>(<i>s, f/f</i><sub><i>0</i></sub>) model, with the theoretical dimer molar mass shown below table; and (5) weight-average frictional ratio (<i>f/f</i><sub><i>0</i></sub>) calculated from the 2D <i>c</i>(<i>s, f/f</i><sub><i>0</i></sub>) model.</div><div id="fspara0630" role="paragraph">(C) Two-dimensional size-and-shape distribution analyses of the sedimentation velocity data of G3BP1. The transformed velocity data are shown as contour plots (heatmaps) of <i>c</i>(<i>M, f/f</i><sub><i>0</i></sub>), <i>c</i>(<i>s, f/f</i><sub><i>0</i></sub>), and <i>c</i>(<i>s,M</i>) with 0 fringes/S (white) to maximum value fringes/S (red), with increasing color temperature indicating higher values. Table below shows (1) total concentration of sample (mg/ml); (2) weight-average sedimentation coefficient <i>s</i><sub><i>w</i></sub> calculated from the 2D <i>c(s,f/f</i><sub><i>0</i></sub><i>)</i> model with percentage protein amount of total protein in parentheses; (3) standard sedimentation coefficient (<i>s</i><sub><i>20,w</i></sub> -value) in water at 20°C; (4) estimated molar mass calculated from (<i>s</i><sub><i>w</i></sub><i>,f/f</i><sub><i>0,w</i></sub>) from the 2D <i>c(s,f/f</i><sub><i>0</i></sub><i>)</i> model, with the theoretical dimer molar mass shown below table; (5) weight-average frictional ratio (<i>f/f</i><sub><i>0w</i></sub><i>)</i> calculated from the 2D <i>c(s,f/f</i><sub><i>0</i></sub><i>)</i> model; and (6) Stokes radius (nm).</div><div id="fspara0635" role="paragraph">(D) Indicated recombinant proteins were incubated with the cross-linker bissulfosuccinimidyl suberate (BS<sup>3</sup>) and then analyzed by SDS-PAGE and Coomassie Blue staining. M, monomer; D, dimer.</div><div id="fspara0640" role="paragraph">(E) Extracts from <i>G3BP1/2</i> dKO cells transfected with GFP-tagged G3BP1 (G3BP1 FL, G3BP1 ΔNTF2L, or G3BP1 with the NTF2L domain replaced with GST) were cross-linked using BS<sup>3</sup> and analyzed by immunoblotting with GFP antibody. M, monomer; D, dimer. Asterisk indicates expected location of dimer formed by G3BP1 ΔNTF2L.</div><div id="fspara0645" role="paragraph">(F) U2OS cells were exposed to sodium arsenite (500&nbsp;μM, 1 h), cross-linked using BS<sup>3</sup>, and endogenous G3BP1 was analyzed by immunoblot with G3BP1 antibody. M, monomer; D, dimer.</div><div id="fspara0650" role="paragraph">(G) U2OS cells were constructed using CRISPR-Cas9 to express tdTomato-tagged endogenous G3BP1. Cells were then transfected with GFP-tagged G3BP1 constructs as indicated, exposed to sodium arsenite (500&nbsp;μM, 1 h), fixed, and imaged. Scale bar, 50&nbsp;μm.</div><div id="fspara0655" role="paragraph">(H) LLPS of purified recombinant G3BP1 was assessed in the presence of a crowding agent, 150&nbsp;mM NaCl, and indicated concentrations of D/D solubilizer. Increasing concentrations of the D/D solubilizer disrupted LLPS of FKBP<sup>F36M</sup>-G3BP1, but not of G3BP1 WT. Scale bar, 50&nbsp;μm.</div><div id="fspara0660" role="paragraph">(I-J) <i>G3BP1/2</i> dKO cells were transfected with indicated G3BP1 constructs, pre-incubated with 250&nbsp;nM D/D solubilizer for 30&nbsp;min, exposed to sodium arsenite (500&nbsp;μM, 1 h), and stained for eIF3η. Cells were imaged and the percentage of cells positive for SGs was quantified. Addition of D/D solubilizer disrupted SG formation in cells expressing FKBP<sup>F36M</sup>-G3BP1, but not in cells expressing G3BP1 WT. Scale bars, 50&nbsp;μm. Error bars indicate SD; <sup>∗∗∗∗</sup>p&nbsp;&lt; 0.0001 versus G3BP1 by two-way ANOVA, Sidak’s multiple comparisons test.</div><div id="fspara0665" role="paragraph">(K-L) <i>G3BP1/2</i> dKO cells were transfected with G3BP1-WT or FKBP<sup>F36M</sup>-G3BP1 and exposed to sodium arsenite (500&nbsp;μM). After 30&nbsp;min of exposure, 250&nbsp;nM of D/D solubilizer was added and cells were further incubated for 30&nbsp;min. Images show live-cell imaging prior to and within 2&nbsp;min after addition of D/D solubilizer. SG dissolution following treatment with D/D solubilizer was assessed by measuring the intensity of the SGs on the frame immediately before and after D/D solubilizer addition. Error bars indicate SD; <sup>∗∗∗</sup>p&nbsp;&lt; 0.001 by one-way ANOVA, Tukey’s multiple comparisons test. Scale bar, 50&nbsp;μm.</div><div id="fspara0670" role="paragraph">(M) Knockdown of <i>caprin-1</i> and <i>TIA1</i> in <i>G3BP1/2</i> dKO cells. <i>G3BP1/2</i> dKO cells were co-transfected with GFP-G3BP1 and a pool of siRNA targeting <i>caprin-1</i> or <i>TIA1</i>. After 48 h, cells were harvested, lysed in RIPA buffer, and expression levels of caprin-1 and TIA1 were analyzed by western blot. Error bars indicate SD; <sup>∗∗</sup>p&nbsp;= 0.0023 for caprin-1 and <sup>∗∗</sup>p&nbsp;= 0.0039 for TIA1 by unpaired t test.</div><div id="fspara0675" role="paragraph">See also <a id="crosref1480" href="#mmc5">Video S2</a> and <a id="crosref1485" href="#mmc2">Table S2</a>.</div></div></div></div></figcaption></figure></div></section><section id="sec-2-9"><h3>G3BP1 Dimerization Is Essential for LLPS <i>In&nbsp;Vitro</i> and SG Assembly in Cells</h3><div id="p0140" role="paragraph">The NTF2L domain mediates interaction with several other SG proteins, and these interactions influence the extent of SG assembly. For example, interaction of NTF2L with caprin-1 and UBAP2L promotes SG assembly, whereas interaction with USP10 limits assembly (<a id="crosrefs0045" href="#bib26" role="doc-biblioref" data-xml-rid="bib26 bib34 bib44 bib68">Kedersha et&nbsp;al., 2016; Markmiller et&nbsp;al., 2018; Panas et&nbsp;al., 2015; Youn et&nbsp;al., 2018</a>). Here, we showed that G3BP1 lacking the NTF2L domain failed to reconstitute SG assembly in <i>G3BP1/2</i> dKO cells, illustrating that this domain is essential for SG assembly (<a id="crosref0690" href="#fig3">Figure&nbsp;3</a>F). Theoretically, this requirement in cells could be due to essential interactions of the NTF2L domain with binding partners; yet we found that NTF2L is also essential for LLPS of G3BP1 with RNA <i>in&nbsp;vitro</i> where these partners are not present (<a id="crosref0695" href="#fig3">Figure&nbsp;3</a>D). Therefore, an alternative possibility is that the critical role of NTF2L is dimerization, which increases the valency of RBDs to promote LLPS of G3BP with RNA. We examined each of these possibilities in turn.</div><div id="p0145" role="paragraph">First, we fully replaced the NTF2L domain of G3BP1 with alternative dimerization domains that form constitutive dimers. Substituting NTF2L with glutathione S-transferase (GST) (211 amino acids; <a id="crosref0700" href="#bib12" role="doc-biblioref" data-xml-rid="bib12">Fabrini et&nbsp;al., 2009</a>) or a FK506-binding protein mutant (FKBP<sup>F36M</sup>; 108 amino acids; <a id="crosref0705" href="#bib50" role="doc-biblioref" data-xml-rid="bib50">Rollins et&nbsp;al., 2000</a>) each restored <i>in&nbsp;vitro</i> LLPS behavior to G3BP1 (<a id="crosref0710" href="#fig5">Figures 5</a>A–5C). When expressed in <i>G3BP1/2</i> dKO cells, both chimeric proteins successfully reconstituted SGs, although the efficiency of the FKBP<sup>F36M</sup> chimera was reduced (<a id="crosref0715" href="#fig5">Figures 5</a>D and 5E; <a id="crosref0720" href="#mmc2">Table S2</a>). Thus, dimerization via the N-terminal domain per se, independent of the nature of the dimer domain, is necessary and sufficient to support both LLPS <i>in&nbsp;vitro</i> and SG assembly in cells. The observed difference in reconstitution efficiency of the two chimeras tested may reflect their respective dimer dissociation constants (K<sub>D</sub> for GST&nbsp;= 0.34&nbsp;μM; K<sub>D</sub> for FKBP<sup>F36M</sup>&nbsp;= 40&nbsp;μM). Notably, using U2OS cells expressing tdTomato-tagged endogenous G3BP1, we confirmed that these chimeric versions of G3BP1 co-mingle in SGs with endogenous G3BP1 (<a id="crosref0725" href="#figs5">Figure&nbsp;S5</a>G).</div><div class="figure-wrap"><figure id="fig5" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2020.03.046/asset/69152595-80f6-4b04-abbf-9a65af5fb4cc/main.assets/gr5_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr5" src="/cms/10.1016/j.cell.2020.03.046/asset/f8017e5d-63c9-42de-8ff6-58c163069c62/main.assets/gr5.jpg" height="848" width="770" aria-labelledby="fig5-title fig5-content" data-viewer-src="/cms/10.1016/j.cell.2020.03.046/asset/69152595-80f6-4b04-abbf-9a65af5fb4cc/main.assets/gr5_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig5-title" class="figure__title"><span class="label figure__label">Figure&nbsp;5</span> <span class="figure__title__text">NTF2L-Domain-Mediated Dimerization and Interaction with Core SG Components Regulate SG Assembly</span></div><div class="accordion"><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><div id="fig5-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0235" role="paragraph">(A) Structures of G3BP1 NTF2L (PDB: <a id="intref0260" href="http://firstglance.jmol.org/fg.htm?mol4FCJ" target="_blank">4FCJ</a>), GST (PDB: <a id="intref0265" href="http://firstglance.jmol.org/fg.htm?mol1UA5" target="_blank">1UA5</a>), and FKBP<sup>F36M</sup> (PDB: <a id="intref0270" href="http://firstglance.jmol.org/fg.htm?mol1YEM" target="_blank">1YEM</a>) dimers.</div><div id="fspara0240" role="paragraph">(B) Constructs used to investigate the function of the NTF2L domain.</div><div id="fspara0245" role="paragraph">(C) LLPS of purified recombinant G3BP1 in the presence of a crowding agent and 150&nbsp;mM NaCl. Scale bars, 50&nbsp;μm.</div><div id="fspara0250" role="paragraph">(D and E) <i>G3BP1/2</i> dKO cells were transfected with indicated G3BP1 constructs, exposed to sodium arsenite (500&nbsp;μM; 1 h), and stained for eIF3η. Cells were imaged (D), and the percentage of cells positive for SGs was quantified (E). Scale bars, 50&nbsp;μm. Error bars indicate SD. <sup>∗∗</sup>p&nbsp;= 0.0098 versus G3BP1 by two-way ANOVA; Sidak’s multiple comparisons test.</div><div id="fspara0255" role="paragraph">(F) Intracellular phase diagrams of indicated G3BP1 constructs transfected into <i>G3BP1/2</i> dKO cells. Cells were exposed to sodium arsenite (30&nbsp;min), fixed, and stained for PABP. SG formation and GFP intensities were assessed cell by cell. Cells with SGs are plotted as filled circles; cells without SGs are plotted as empty circles. Boxes highlight the 25% highest levels of expression among SG-negative cells.</div><div id="fspara0260" role="paragraph">(G) Co-phase separation of purified recombinant G3BP1 with BSA or caprin-1 in the absence of a crowding reagent, in 150&nbsp;mM NaCl, and with increasing concentrations of total RNA from human cells.</div><div id="fspara0265" role="paragraph">(H) Co-phase separation of purified recombinant G3BP1 and caprin-1 variants as in (G).</div><div id="fspara0270" role="paragraph">(I) Co-phase separation of purified recombinant G3BP1 and TIA1 as in (G).</div><div id="fspara0275" role="paragraph">(J) Phase diagram of G3BP1 with caprin-1 and TIA1 as shown in (G)–(I).</div><div id="fspara0280" role="paragraph">(K) Intracellular phase diagrams of G3BP1 with addition of caprin-1. <i>G3BP1/2</i> dKO cells were co-transfected with mCherry-G3BP1 and GFP-caprin-1, and intracellular phase diagrams were measured as in (F). Expression of caprin-1 reduces the G3BP1 threshold for SG formation (blue arcs). Cells with high caprin-1 levels assembled SGs at very low levels of G3BP1 (red oval).</div><div id="fspara0285" role="paragraph">(L) Intracellular phase diagrams of G3BP1 after knockdown of <i>caprin-1</i> or <i>TIA1</i>. <i>G3BP1/2</i> dKO cells were co-transfected with GFP-G3BP1 WT and a pool of siRNA targeting expression of <i>caprin-1</i> or <i>TIA1</i>, and intracellular phase diagrams were measured as in (F).</div><div id="fspara0290" role="paragraph">See also <a id="crosref1380" href="#figs5">Figure&nbsp;S5</a>, <a id="crosref1385" href="#mmc2">Table S2</a>, and <a id="crosref1390" href="#mmc5">Video S2</a>.</div></div></div></div></figcaption></figure></div><div id="p0150" role="paragraph">To fully assess the ability of these chimeric proteins with different dimer domains to reconstitute SG assembly, we mapped their intracellular phase separation thresholds by taking advantage of the natural variation in expression of GFP-G3BP1 when transiently transfected into <i>G3BP1/2</i> dKO cells. Simultaneous cell-by-cell analysis of GFP-G3BP1 expression level and SG formation permitted a determination of the G3BP1 concentration threshold that initiates SG assembly. The concentration threshold for SG assembly was similar for G3BP1 WT and GST-G3BP1, whereas this threshold concentration was markedly higher for FKBP<sup>F36M</sup>-G3BP1 (<a id="crosref0730" href="#fig5">Figure&nbsp;5</a>F). These data illustrate that an essential function of the NTF2L domain is to support LLPS through dimerization, independent of any NTF2L-specific interactions.</div></section><section id="sec-2-10"><h3>G3BP1 Dimerization Is Essential to Maintain SG Assembly in Cells</h3><div id="p0155" role="paragraph">FKBP<sup>F36M</sup> homodimers can be dissociated by a small-molecule ligand, such as the rapamycin derivative AP21998 or “D/D solubilizer” (<a id="crosref0735" href="#bib50" role="doc-biblioref" data-xml-rid="bib50">Rollins et&nbsp;al., 2000</a>). Addition of this ligand in sub-stoichiometric amounts abolished LLPS by FKBP<sup>F36M</sup>-G3BP1 but had no effect on LLPS of G3BP1 WT, further illustrating that dimerization per se is required for LLPS (<a id="crosref0740" href="#figs5">Figure&nbsp;S5</a>H). To examine the effect of disrupting G3BP1 dimer formation on SG assembly in cells, we pre-incubated <i>G3BP1/2</i> dKO cells expressing FKBP<sup>F36M</sup>-G3BP1 with sub-stoichiometric amounts of this ligand and found that arsenite-induced SG assembly was abolished, confirming the importance of G3BP1 dimerization for this assembly (<a id="crosref0745" href="#figs5">Figures S5</a>I and S5J). Remarkably, fully assembled SGs rapidly disassembled within 30&nbsp;s by the addition of ligand to the culture medium (<a id="crosref0750" href="#figs5">Figures S5</a>K and S5L; <a id="crosref0755" href="#mmc5">Video S2</a>). These results indicate that dimerization of G3BP1 is not only essential for SG assembly but also for maintenance of SGs, an observation that may have implications for the mechanism of SG disassembly upon removal of stress.</div><div id="p0160" role="paragraph"><div id="mmc5" class="core-supplementary-material" data-category="video"><div class="core-link"><div class="cloudflare-stream-container" id="mmc5" data-type="mp4"><span id="video_stream_mmc5_src" class="video-stream-source" style="display:none">eyJraWQiOiI4ZjUxYWNhY2IzYjhiNjNlNzFlYmIzYWFmYTU5NmZmYyIsImFsZyI6IlJTMjU2In0.eyJzdWIiOiJmOWZkMmU5MzYzNGMwZmQyYWI0N2NkOGRjY2Q5M2E2YSIsImtpZCI6IjhmNTFhY2FjYjNiOGI2M2U3MWViYjNhYWZhNTk2ZmZjIiwiZXhwIjoxNzQ5NjA3NjA5fQ.NHnthFouq_a9BenoaZE4KN8g8GbE-BAxqzweHj56E3IZdfvUhNqPtZvYYyXIOZznHVs6fLLq4InwXEfZXfCQQZcm6s53LJFpLUcyIo7pFimqsghcCcNTDoQ19qOMS7g5VZpGF8HyMKwFqGbVGpttLTx6ar4ZMZ0cHoK_5Krh_M1BKoPtM5XRKj7wLDwHHxrjrHejHdBfTHbDqJg5-R3_jq2VWtNcXk5tAb4-3Er36lH9r9V7VnJEm4TW3W1xCPVs5hZR1_p8bft32LI97BixyqXBXVvifScZAYwuGI37ZoRle0yPrXJZucvqg3EqVwZlaQvDNRSAYZ2A4wgToRzjfQ</span><stream src="eyJraWQiOiI4ZjUxYWNhY2IzYjhiNjNlNzFlYmIzYWFmYTU5NmZmYyIsImFsZyI6IlJTMjU2In0.eyJzdWIiOiJmOWZkMmU5MzYzNGMwZmQyYWI0N2NkOGRjY2Q5M2E2YSIsImtpZCI6IjhmNTFhY2FjYjNiOGI2M2U3MWViYjNhYWZhNTk2ZmZjIiwiZXhwIjoxNzQ5NjA3NjA5fQ.NHnthFouq_a9BenoaZE4KN8g8GbE-BAxqzweHj56E3IZdfvUhNqPtZvYYyXIOZznHVs6fLLq4InwXEfZXfCQQZcm6s53LJFpLUcyIo7pFimqsghcCcNTDoQ19qOMS7g5VZpGF8HyMKwFqGbVGpttLTx6ar4ZMZ0cHoK_5Krh_M1BKoPtM5XRKj7wLDwHHxrjrHejHdBfTHbDqJg5-R3_jq2VWtNcXk5tAb4-3Er36lH9r9V7VnJEm4TW3W1xCPVs5hZR1_p8bft32LI97BixyqXBXVvifScZAYwuGI37ZoRle0yPrXJZucvqg3EqVwZlaQvDNRSAYZ2A4wgToRzjfQ" class="cloudflare-stream-player ready tracked" id="video_stream_mmc5" controls="true"><div class="player-share-button" title="Share Video"><div class="share-player-icon"></div></div><div class="player-share-container hidden"><span class="player-close-share"></span><div class="social-links-container"><ul class="social-links-list"><li class="html5-share-facebook-icon" data-share-url="https://www.facebook.com/sharer/sharer.php?u=https://www.cell.com/cell/fulltext/S0092-8674(20)30339-1" target="_blank"></li><li class="html5-share-twitter-icon" data-share-url="https://twitter.com/share?url=https://www.cell.com/cell/fulltext/S0092-8674(20)30339-1" target="_blank"></li><li class="html5-share-mail-icon" data-share-url="mailto:?body=https://www.cell.com/cell/fulltext/S0092-8674(20)30339-1" target="_blank"></li></ul><input type="text" value="https://www.cell.com/cell/fulltext/S0092-8674(20)30339-1" readonly="readonly" class="share-input"></div></div><div style="position: relative; height: 0px;"><iframe src="https://embed.cloudflarestream.com/embed/iframe.fla9.823df22.html?videoId=eyJraWQiOiI4ZjUxYWNhY2IzYjhiNjNlNzFlYmIzYWFmYTU5NmZmYyIsImFsZyI6IlJTMjU2In0.eyJzdWIiOiJmOWZkMmU5MzYzNGMwZmQyYWI0N2NkOGRjY2Q5M2E2YSIsImtpZCI6IjhmNTFhY2FjYjNiOGI2M2U3MWViYjNhYWZhNTk2ZmZjIiwiZXhwIjoxNzQ5NjA3NjA5fQ.NHnthFouq_a9BenoaZE4KN8g8GbE-BAxqzweHj56E3IZdfvUhNqPtZvYYyXIOZznHVs6fLLq4InwXEfZXfCQQZcm6s53LJFpLUcyIo7pFimqsghcCcNTDoQ19qOMS7g5VZpGF8HyMKwFqGbVGpttLTx6ar4ZMZ0cHoK_5Krh_M1BKoPtM5XRKj7wLDwHHxrjrHejHdBfTHbDqJg5-R3_jq2VWtNcXk5tAb4-3Er36lH9r9V7VnJEm4TW3W1xCPVs5hZR1_p8bft32LI97BixyqXBXVvifScZAYwuGI37ZoRle0yPrXJZucvqg3EqVwZlaQvDNRSAYZ2A4wgToRzjfQ" allowfullscreen="true" allow="autoplay; picture-in-picture" style="height: 100%; width: 100%; position: absolute; top: 0px; left: 0px; border: none;"></iframe></div></stream></div><a href="/cms/10.1016/j.cell.2020.03.046/attachment/aafd9233-2acf-4d3c-abf8-da339021e4c8/mmc5.mp4" download="mmc5.mp4" aria-labelledby="mmc5-heading mmc5-link"><span id="mmc5-link">Video (2.47 MB)</span></a></div><div class="core-description"><div id="mmc5-heading" class="heading">Video S2. Disrupting Dimerization of FKBP<sup>F36M</sup>-G3BP1 by D/D Solubilizer Causes Disassembly of Stress Granules in Cells, Related to Figures 5 and S5</div><div id="espara0015" role="paragraph"><i>G3BP1/2</i> dKO cells were transfected with G3BP1-WT or FKBP<sup>F36M</sup>-G3BP1 and exposed to sodium arsenite (500&nbsp;μM). After 30&nbsp;min of exposure, 250&nbsp;nM of D/D solubilizer was added and cells were further incubated for 30&nbsp;min. Images show live-cell imaging prior to and within 2&nbsp;min after addition of D/D solubilizer.</div></div></div></div></section><section id="sec-2-11"><h3>Cooperative Interactions between Members of the Core SG Network Regulate Assembly</h3><div id="p0165" role="paragraph">Given that NTF2L-mediated interactions are not essential for SG formation, the mechanism whereby interactors of G3BP1 exert influence over assembly was unclear. We hypothesized that interactors of G3BP1 contribute to and regulate SG assembly by contributing to the overall sum of interactions within the SG network (<a id="crosref0760" href="#fig1">Figure&nbsp;1</a>D). In support of this hypothesis, we found that the presence of an additional SG constituent (i.e., caprin-1) that binds the NTF2L domain of G3BP1 significantly reduced the threshold concentration of G3BP1 and RNA necessary for LLPS, thereby exhibiting positive cooperativity (<a id="crosref0765" href="#fig5">Figure&nbsp;5</a>G). This cooperativity reflects a specific interaction between caprin-1 and G3BP1, as evidenced by point mutations that disrupt binding between caprin-1 and G3BP1 (G3BP1 F33W and caprin-1 F372A), each of which abolished the ability of caprin-1 to promote LLPS of G3BP1 with RNA (<a id="crosref0770" href="#fig5">Figure&nbsp;5</a>H). Similar to caprin-1, we found that TIA1 also exhibited positive cooperativity with G3BP1 and RNA, significantly reducing the threshold concentration necessary for LLPS (<a id="crosref0775" href="#fig5">Figures 5</a>I and 5J).</div><div id="p0170" role="paragraph">To examine the significance of this biophysical behavior in cells, we measured the intracellular phase separation threshold for SG assembly as a function of intracellular concentrations of these binding partners. Consistent with <i>in&nbsp;vitro</i> results, overexpression of caprin-1 with G3BP1 in <i>G3BP1/2</i> dKO cells significantly reduced the concentration threshold of G3BP1 necessary to initiate SG assembly, even in the absence of arsenite (<a id="crosref0780" href="#fig5">Figure&nbsp;5</a>K). Interestingly, in cells with very high caprin-1 levels, even cells with very low levels of G3BP1 were able to assemble SGs. Consistent with these results, we found that depletion of either <i>caprin-1</i> or <i>TIA1</i> by siRNA-mediated knockdown significantly raised the critical concentration threshold of G3BP1 necessary to initiate SG assembly (<a id="crosref0785" href="#fig5">Figures 5</a>L and <a id="crosref0790" href="#figs5">S5</a>M). Thus, both <i>in&nbsp;vitro</i> and cell-based results suggest that components of the core SG network influence SG assembly through positive (e.g., caprin-1, UBAP2L, and TIA1) or negative (e.g., USP10) cooperativity, collectively setting the critical threshold concentration of constituents necessary for LLPS of G3BP1 with RNA (<a id="crosref0795" href="#fig5">Figure&nbsp;5</a>J). However, no specific interaction appears to be essential, because SGs still assemble in cells with knockout of individual binding partners (e.g., <i>caprin-1</i>, <i>UBAP2L</i>, or <i>TIA1</i>) or paralogous pairs (e.g., <i>TIA1/TIAR</i> or <i>UBAP2/UBAP2L</i>; <a id="crosref0800" href="#fig1">Figure&nbsp;1</a>F).</div></section><section id="sec-2-12"><h3>G3BP1 IDR1/2 Regulates SG Dynamics and Composition</h3><div id="p0175" role="paragraph">The precise role of IDRs in regulating the assembly or material properties of SGs remains poorly understood. To gain insight into this role, we pursued our surprising finding that the long central IDR (IDR1/2) of G3BP1 is not essential for LLPS of G3BP1 <i>in&nbsp;vitro</i> or for SG assembly in cells (<a id="crosref0805" href="#fig3">Figures 3</a>C–3F). The IDR1/2 segment did not exhibit LLPS <i>in&nbsp;vitro</i> even at high concentration (400&nbsp;μM) and the presence of a crowding agent (<a id="crosref0810" href="#figs6">Figure&nbsp;S6</a>A) and did not reconstitute SG assembly when expressed in <i>G3BP1/2</i> dKO cells (<a id="crosref0815" href="#figs6">Figure&nbsp;S6</a>B). Indeed, even when expressed in WT cells, the IDR1/2 segment was not recruited into arsenite-induced SGs arising from endogenous G3BP (<a id="crosref0820" href="#figs6">Figure&nbsp;S6</a>C). These data indicate that IDR1/2 does not encode interactions that are essential for the assembly and/or maintenance of SGs.</div><div class="figure-wrap"><figure id="figs6" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2020.03.046/asset/c9c433a9-34cb-467a-b0d7-97abd5c74b10/main.assets/figs6_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs6" src="/cms/10.1016/j.cell.2020.03.046/asset/80d75547-a3ea-42ab-b888-02606335305a/main.assets/figs6.jpg" height="959" width="705" aria-labelledby="figs6-title figs6-content" data-viewer-src="/cms/10.1016/j.cell.2020.03.046/asset/c9c433a9-34cb-467a-b0d7-97abd5c74b10/main.assets/figs6_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs6-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S6</span> <span class="figure__title__text">The Long Central IDR of G3BP1 Regulates SG Assembly, Dynamics, and Composition; G3BP1 IDR1 Is an Autoinhibitory Element, Related to <a id="crosref1490" href="#fig6">Figure&nbsp;6</a></span></div><div class="accordion"><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><div id="figs6-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0685" role="paragraph">(A) LLPS of purified recombinant full-length G3BP1 (FL) and IDR1/2 (a 192-amino acid fragment of G3BP1 representing the IDR1/2 segment) was assessed in the presence of a crowding agent and 150&nbsp;mM NaCl. Purified IDR1/2 failed to phase separate. Scale bar, 50&nbsp;μm.</div><div id="fspara0690" role="paragraph">(B-C) <i>G3BP1/2</i> dKO cells (B) or WT U2OS cells (C) were transfected with GFP-tagged IDR1/2, exposed to sodium arsenite (500&nbsp;μM, 1 h), and stained for eIF3η. Expression of IDR1/2 protein segment did not reconstitute SG assembly in <i>G3BP1/2</i> dKO cells (B), nor was the IDR1/2 protein segment recruited to SGs in WT cells (C). Scale bar, 20&nbsp;μm.</div><div id="fspara0695" role="paragraph">(D) <i>G3BP1/2</i> dKO cells were transfected with GFP-G3BP1 FL or ΔIDR1/2. Lysates were extracted with RIPA buffer, separated by SDS-PAGE, and immunoblotted for GFP and GAPDH. G3BP1 FL was largely found in the soluble fraction (S), whereas G3BP1 ΔIDR1/2 was largely recovered in the RIPA-insoluble pellet (I). Representative images from 3 replicates are shown. Band intensities were quantified based on 3 replicates. Error bars indicate SEM. <sup>∗∗∗</sup>p&nbsp;= 0.0022 by unpaired t test.</div><div id="fspara0705" role="paragraph">(E) <i>G3BP1/2</i> dKO cells were transfected with GFP-G3BP1 or GFP-G3BP1 ΔIDR1/2. Cells were exposed to sodium arsenite (500&nbsp;μM, 1 h), and SG assembly was assessed by formation of G3BP1-positive structures. Error bars indicate SD. <sup>∗∗∗∗</sup>p&nbsp;&lt; 0.0001 by two-way ANOVA, Sidak’s multiple comparison test.</div><div id="fspara0710" role="paragraph">(F) <i>G3BP1/2</i> dKO cells were co-transfected with GFP-G3BP1 FL or GFP-G3BP1 ΔIDR1/2 together with HA-G3BP1 FL. Lysates were extracted with RIPA buffer and incubated with GFP magnetic beads for IP. Washed beads were subjected to SDS-PAGE and blotted with GFP as well as HA antibodies. Both G3BP1 FL and ΔIDR1/2 could form dimers with HA-tagged G3BP1 FL.</div><div id="fspara0715" role="paragraph">(G) <i>G3BP1/2</i> dKO cells were transfected with GFP-G3BP1 FL or indicated GFP-G3BP1 mutants. 16 to 24 hours post-transfection, cells were lysed in 1x NuPAGE sample buffer and subjected to SDS-PAGE and immunoblotting for GFP (detection of G3BP1) and GAPDH (loading control).</div><div id="fspara0720" role="paragraph">(H) <i>G3BP1/2</i> dKO cells were reconstituted with GFP-tagged G3BP1 (WT, ΔIDR1, or ΔIDR2), exposed to sodium arsenite (500&nbsp;μM, 1 h), and stained for eIF3η. Scale bar, 20&nbsp;μm.</div><div id="fspara0725" role="paragraph">(I) Amino acid sequence of constructs shown in <a id="crosref1495" href="#fig6">Figure&nbsp;6</a>G.</div><div id="fspara0730" role="paragraph">(J-L) <i>G3BP1/2</i> dKO cells were transfected with indicated G3BP1 mutants, exposed to sodium arsenite (500&nbsp;μM, 1 h), and stained for eIF3η. Scale bars, 20&nbsp;μm.</div><div id="fspara0735" role="paragraph">(M-N) <i>G3BP1/2</i> dKO cells were transfected with either WT or a mutant G3BP1 construct in which the IDR2 was duplicated. Cells were exposed to sodium arsenite (500&nbsp;μM, 1 h) and stained for eIF3η. Cells were imaged and the percentage of cells positive for SGs was quantified. G3BP1-2xIDR2 supported SG formation more efficiently than G3BP1 WT. Scale bar, 20&nbsp;μm. Error bars indicate SD. <sup>∗∗∗</sup>p&nbsp;= 0.0004 by unpaired t test.</div></div></div></div></figcaption></figure></div><div id="p0180" role="paragraph">However, deletion of IDR1/2 did alter the biophysical properties of G3BP1. Specifically, G3BP1 ΔIDR1/2 showed reduced solubility and underwent LLPS at lower concentrations <i>in&nbsp;vitro</i> (<a id="crosref0825" href="#fig3">Figures 3</a>D and 3E) and had relatively poor solubility in cells (<a id="crosref0830" href="#figs6">Figure&nbsp;S6</a>D). Additionally, although G3BP1 ΔIDR1/2 reconstituted SG assembly in <i>G3BP1/2</i> dKO cells, efficiency was reduced (<a id="crosref0835" href="#figs6">Figure&nbsp;S6</a>E).</div><div id="p0185" role="paragraph">To explore the role of IDR1/2 in SG dynamics, we used U2OS cells with endogenous G3BP1 tagged with tdTomato. We introduced exogenous GFP-tagged G3BP1 (WT or ΔIDR1/2) to permit simultaneous monitoring of their mobility within SGs by fluorescence recovery after photobleaching (FRAP). As expected, when exogenous G3BP1 WT was introduced into these cells, endogenous and exogenous WT proteins showed identical rates of FRAP (<a id="crosref0840" href="#fig6">Figures 6</a>A and 6B; <a id="crosref0845" href="#mmc6">Video S3</a>). However, exogenous G3BP1 ΔIDR1/2 showed significantly slower FRAP and a greater immobile fraction compared with endogenous WT protein, revealing a role for IDR1/2 in defining G3BP1 dynamics (<a id="crosref7850" href="#fig6">Figures 6</a>C and 6D; <a id="crosref0855" href="#mmc7">Video S4</a>). By co-immunoprecipitation assay, we confirmed that GFP-G3BP1 (both WT and ΔIDR1/2) formed homodimers as well as heterodimers with hemagglutinin (HA)-G3BP1 WT (<a id="crosref0860" href="#figs6">Figure&nbsp;S6</a>F), suggesting that these FRAP experiments likely underestimate the extent to which the exchange rate of G3BP1 is impaired by deletion of IDR1/2.</div><div class="figure-wrap"><figure id="fig6" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2020.03.046/asset/6ffe6fd2-eecb-40f2-9f52-c7b93491f1bd/main.assets/gr6_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr6" src="/cms/10.1016/j.cell.2020.03.046/asset/719e43ae-f4f8-4330-8c50-18155d8f5c34/main.assets/gr6.jpg" height="958" width="728" aria-labelledby="fig6-title fig6-content" data-viewer-src="/cms/10.1016/j.cell.2020.03.046/asset/6ffe6fd2-eecb-40f2-9f52-c7b93491f1bd/main.assets/gr6_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig6-title" class="figure__title"><span class="label figure__label">Figure&nbsp;6</span> <span class="figure__title__text">The Long Central IDR of G3BP1 Regulates SG Assembly, Dynamics, and Composition; G3BP1 IDR1 Is an Autoinhibitory Element</span></div><div class="accordion"><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><div id="fig6-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0300" role="paragraph">(A–D) U2OS cells expressing tdTomato-tagged endogenous G3BP1 were transfected with indicated GFP-tagged G3BP1 constructs and exposed to sodium arsenite (500&nbsp;μM; 1 h), and the relative mobility of endogenous G3BP1-tdTomato was compared to exogenous GFP-G3BP1 by FRAP. Error bars indicate SEM. n.s., not significant; <i><sup>∗∗∗∗</sup></i>p <i>&lt;</i> 0.0001 by two-way ANOVA; Sidak’s multiple comparison test. Mobile fractions were 76% (G3BP1-tdTomato ), 73% (GFP-G3BP1 FL) in (A) and (B), and 79% (G3BP1-tdTomato) and 54% (GFP-G3BP1 ΔIDR1/2) in (C) and (D). Scale bar, 10&nbsp;μm.</div><div id="fspara0305" role="paragraph">(E and F) <i>G3BP1/2</i> dKO cells were transfected with indicated G3BP1 constructs and exposed to sodium arsenite (500&nbsp;μM). Cells were imaged, and the percentage of cells positive for SGs was quantified prior to arsenite exposure (0&nbsp;min) or 10&nbsp;min or 60&nbsp;min after arsenite exposure. Error bars indicate SEM. <sup>∗</sup>p&nbsp;= 0.0224; <sup>∗∗∗∗</sup>p&nbsp;&lt; 0.0001 by two-way ANOVA; Dunnett’s multiple comparisons test.</div><div id="fspara0310" role="paragraph">(G and H) <i>G3BP1/2</i> dKO cells were transfected, stressed, imaged, and quantified as in (E) and (F). G3BP1 mutant constructs had deletion of IDR2 and either a scrambled IDR1 sequence (Scr) or mutation of all 28 glutamates in IDR1 to glutamine (EQ28). Error bars indicate SEM. <sup>∗∗∗∗</sup>p&nbsp;&lt; 0.0001 by two-way ANOVA; Dunnett’s multiple comparisons test.</div><div id="fspara0315" role="paragraph">(I and J) <i>G3BP1/2</i> dKO cells were transfected, stressed, imaged, and quantified as in (E) and (F). G3BP1 mutant constructs had substitution of IDR2 with Ash1 IDR, substitution of all 33 proline residues in IDR1/2 to serine (PS33), or substitution of 17 positively charged residues to alanines (RKH/A17). Error bars indicate SEM. <sup>∗∗</sup>p&nbsp;= 0.0077 (0&nbsp;min) and 0.0014 (10&nbsp;min); <sup>∗∗∗∗</sup>p&nbsp;&lt; 0.0001 by two-way ANOVA; Dunnett’s multiple comparisons test.</div><div id="fspara0320" role="paragraph">(K) <i>G3BP1/2</i> dKO cells were transfected with indicated G3BP1 constructs, exposed to sodium arsenite (500&nbsp;μM; 1 h), fixed, and stained with SG markers as indicated. Confocal images were taken for partition coefficient analysis. Error bars indicate SD. <sup>∗∗</sup>p&nbsp;= 0.0011; <sup>∗∗∗</sup>p&nbsp;= 0.0001; <sup>∗∗∗∗</sup>p&nbsp;&lt; 0.0001 by two-way ANOVA; Sidak’s multiple comparisons test.</div><div id="fspara0325" role="paragraph">(L and M) <i>G3BP1/2</i> dKO cells were transfected with indicated G3BP1 constructs (L) and exposed to sodium arsenite (500&nbsp;μM; 1 h). Cells were imaged, and the percentage of cells positive for SGs was quantified (M). Error bars indicate SD.</div><div id="fspara0330" role="paragraph">(N) Fold change of spectral counts of proteins identified by APEX2 proximity labeling and p values are plotted. Colored circles indicate proteins more enriched in SGs formed with G3BP1-2×Ash1 IDR (red; 10 representative proteins are labeled) or with G3BP1 FL (blue).</div><div id="fspara0335" role="paragraph">See also <a id="crosref1395" href="#figs6">Figure&nbsp;S6</a>, <a id="crosref1400" href="#mmc3">Table S3</a>, and <a id="crosref1405" href="#mmc6">Videos S3</a> and <a id="crosref1410" href="#mmc7">S4</a>.</div></div></div></div></figcaption></figure></div><div id="p0190" role="paragraph"><div id="mmc6" class="core-supplementary-material" data-category="video"><div class="core-link"><div class="cloudflare-stream-container" id="mmc6" data-type="mp4"><span id="video_stream_mmc6_src" class="video-stream-source" style="display:none">eyJraWQiOiI4ZjUxYWNhY2IzYjhiNjNlNzFlYmIzYWFmYTU5NmZmYyIsImFsZyI6IlJTMjU2In0.eyJzdWIiOiI2NjYyMWE5YjZhZThmMWIyZmE3NmNiMjE3MzI0MWQwMiIsImtpZCI6IjhmNTFhY2FjYjNiOGI2M2U3MWViYjNhYWZhNTk2ZmZjIiwiZXhwIjoxNzQ5NjA3NjA5fQ.f-AS8hUcHvxgvIyXb0_QI7hPQgvWBOGxr3cF8d0HqhvPbRmast5JZo744tZw-w6nfeP-ySfJGupV1J9v4knN5Xs8fZPMaWdnregoDmL1YIdaNH-5MBG_n_DC7GVggEeCNq_HV44RARR1cYcSsnGkQ8YlkfsrxiN8iJSIbC7rG6gd4hLHMoEtltyet_abgY4mK7Y79XJirF35KdE0hox-FQY0Tz1M0qA_3ClRmdIDxe_eS2NBuxu0xuh3G8_KiYJDJ1s4e3QiD14xIjLrrupFzbE4Hv3o4iEL9jQZmj3KMOvz8wyMFuTAd2sP6U1IBEytVhGQG0S5hmaSXU6uLUdBrQ</span><stream src="eyJraWQiOiI4ZjUxYWNhY2IzYjhiNjNlNzFlYmIzYWFmYTU5NmZmYyIsImFsZyI6IlJTMjU2In0.eyJzdWIiOiI2NjYyMWE5YjZhZThmMWIyZmE3NmNiMjE3MzI0MWQwMiIsImtpZCI6IjhmNTFhY2FjYjNiOGI2M2U3MWViYjNhYWZhNTk2ZmZjIiwiZXhwIjoxNzQ5NjA3NjA5fQ.f-AS8hUcHvxgvIyXb0_QI7hPQgvWBOGxr3cF8d0HqhvPbRmast5JZo744tZw-w6nfeP-ySfJGupV1J9v4knN5Xs8fZPMaWdnregoDmL1YIdaNH-5MBG_n_DC7GVggEeCNq_HV44RARR1cYcSsnGkQ8YlkfsrxiN8iJSIbC7rG6gd4hLHMoEtltyet_abgY4mK7Y79XJirF35KdE0hox-FQY0Tz1M0qA_3ClRmdIDxe_eS2NBuxu0xuh3G8_KiYJDJ1s4e3QiD14xIjLrrupFzbE4Hv3o4iEL9jQZmj3KMOvz8wyMFuTAd2sP6U1IBEytVhGQG0S5hmaSXU6uLUdBrQ" class="cloudflare-stream-player ready tracked" id="video_stream_mmc6" controls="true"><div class="player-share-button" title="Share Video"><div class="share-player-icon"></div></div><div class="player-share-container hidden"><span class="player-close-share"></span><div class="social-links-container"><ul class="social-links-list"><li class="html5-share-facebook-icon" data-share-url="https://www.facebook.com/sharer/sharer.php?u=https://www.cell.com/cell/fulltext/S0092-8674(20)30339-1" target="_blank"></li><li class="html5-share-twitter-icon" data-share-url="https://twitter.com/share?url=https://www.cell.com/cell/fulltext/S0092-8674(20)30339-1" target="_blank"></li><li class="html5-share-mail-icon" data-share-url="mailto:?body=https://www.cell.com/cell/fulltext/S0092-8674(20)30339-1" target="_blank"></li></ul><input type="text" value="https://www.cell.com/cell/fulltext/S0092-8674(20)30339-1" readonly="readonly" class="share-input"></div></div><div style="position: relative; height: 0px;"><iframe src="https://embed.cloudflarestream.com/embed/iframe.fla9.823df22.html?videoId=eyJraWQiOiI4ZjUxYWNhY2IzYjhiNjNlNzFlYmIzYWFmYTU5NmZmYyIsImFsZyI6IlJTMjU2In0.eyJzdWIiOiI2NjYyMWE5YjZhZThmMWIyZmE3NmNiMjE3MzI0MWQwMiIsImtpZCI6IjhmNTFhY2FjYjNiOGI2M2U3MWViYjNhYWZhNTk2ZmZjIiwiZXhwIjoxNzQ5NjA3NjA5fQ.f-AS8hUcHvxgvIyXb0_QI7hPQgvWBOGxr3cF8d0HqhvPbRmast5JZo744tZw-w6nfeP-ySfJGupV1J9v4knN5Xs8fZPMaWdnregoDmL1YIdaNH-5MBG_n_DC7GVggEeCNq_HV44RARR1cYcSsnGkQ8YlkfsrxiN8iJSIbC7rG6gd4hLHMoEtltyet_abgY4mK7Y79XJirF35KdE0hox-FQY0Tz1M0qA_3ClRmdIDxe_eS2NBuxu0xuh3G8_KiYJDJ1s4e3QiD14xIjLrrupFzbE4Hv3o4iEL9jQZmj3KMOvz8wyMFuTAd2sP6U1IBEytVhGQG0S5hmaSXU6uLUdBrQ" allowfullscreen="true" allow="autoplay; picture-in-picture" style="height: 100%; width: 100%; position: absolute; top: 0px; left: 0px; border: none;"></iframe></div></stream></div><a href="/cms/10.1016/j.cell.2020.03.046/attachment/33631842-5f26-42d6-a181-27e4a44ef39d/mmc6.mp4" download="mmc6.mp4" aria-labelledby="mmc6-heading mmc6-link"><span id="mmc6-link">Video (1.11 MB)</span></a></div><div class="core-description"><div id="mmc6-heading" class="heading">Video S3. FRAP Analysis of GFP-G3BP1 WT, Related to Figure&nbsp;6</div><div id="espara0020" role="paragraph">U2OS cells were constructed using CRISPR-Cas9 to express tdTomato-tagged endogenous G3BP1. Cells were then transfected with exogenous full-length GFP-G3BP1. The relative mobility of endogenous (tdTomato-tagged) G3BP1 was compared to exogenous (GFP-tagged) G3BP1 by FRAP.</div></div></div></div><div id="p0195" role="paragraph"><div id="mmc7" class="core-supplementary-material" data-category="video"><div class="core-link"><div class="cloudflare-stream-container" id="mmc7" data-type="mp4"><span id="video_stream_mmc7_src" class="video-stream-source" style="display:none">eyJraWQiOiI4ZjUxYWNhY2IzYjhiNjNlNzFlYmIzYWFmYTU5NmZmYyIsImFsZyI6IlJTMjU2In0.eyJzdWIiOiJjMjY5Yzg3NDA3YWU0MzBkMWIxZThjNzNhN2IwZTZjYSIsImtpZCI6IjhmNTFhY2FjYjNiOGI2M2U3MWViYjNhYWZhNTk2ZmZjIiwiZXhwIjoxNzQ5NjA3NjA5fQ.jBYcsPZW0MCBFsXcOUdu_cHXARlhYwdpDPb9nPhUz22-N3ElWsypJ75fRFLFD-6y0u97WCvTBL0Q5vZFKc2LAi8sBr_z-SanW9ogdS5vkRQmVpFylOqcc6L3HtSXmSxY32i7Tcx4I95WGyH0QtPzwlNaCOw2E3RljfrrLPS-Fvixe8IrlpzqRfVpNEFE02F8iCf1MIUw-KlIpjv_Ay03D5RvyXUaO1uTPqSMWphJUt2ba_voUVw4sd0ZudnbEzAB1R2wIRpWp8LVkZ5N5Hf050pLB37QllFCGT7U0dS_sjZGPJ3oBILiEKj4Eq-9liFA9gxpFvrjoJ_YNn3uN3VNPw</span><stream src="eyJraWQiOiI4ZjUxYWNhY2IzYjhiNjNlNzFlYmIzYWFmYTU5NmZmYyIsImFsZyI6IlJTMjU2In0.eyJzdWIiOiJjMjY5Yzg3NDA3YWU0MzBkMWIxZThjNzNhN2IwZTZjYSIsImtpZCI6IjhmNTFhY2FjYjNiOGI2M2U3MWViYjNhYWZhNTk2ZmZjIiwiZXhwIjoxNzQ5NjA3NjA5fQ.jBYcsPZW0MCBFsXcOUdu_cHXARlhYwdpDPb9nPhUz22-N3ElWsypJ75fRFLFD-6y0u97WCvTBL0Q5vZFKc2LAi8sBr_z-SanW9ogdS5vkRQmVpFylOqcc6L3HtSXmSxY32i7Tcx4I95WGyH0QtPzwlNaCOw2E3RljfrrLPS-Fvixe8IrlpzqRfVpNEFE02F8iCf1MIUw-KlIpjv_Ay03D5RvyXUaO1uTPqSMWphJUt2ba_voUVw4sd0ZudnbEzAB1R2wIRpWp8LVkZ5N5Hf050pLB37QllFCGT7U0dS_sjZGPJ3oBILiEKj4Eq-9liFA9gxpFvrjoJ_YNn3uN3VNPw" class="cloudflare-stream-player ready tracked" id="video_stream_mmc7" controls="true"><div class="player-share-button" title="Share Video"><div class="share-player-icon"></div></div><div class="player-share-container hidden"><span class="player-close-share"></span><div class="social-links-container"><ul class="social-links-list"><li class="html5-share-facebook-icon" data-share-url="https://www.facebook.com/sharer/sharer.php?u=https://www.cell.com/cell/fulltext/S0092-8674(20)30339-1" target="_blank"></li><li class="html5-share-twitter-icon" data-share-url="https://twitter.com/share?url=https://www.cell.com/cell/fulltext/S0092-8674(20)30339-1" target="_blank"></li><li class="html5-share-mail-icon" data-share-url="mailto:?body=https://www.cell.com/cell/fulltext/S0092-8674(20)30339-1" target="_blank"></li></ul><input type="text" value="https://www.cell.com/cell/fulltext/S0092-8674(20)30339-1" readonly="readonly" class="share-input"></div></div><div style="position: relative; height: 0px;"><iframe src="https://embed.cloudflarestream.com/embed/iframe.fla9.823df22.html?videoId=eyJraWQiOiI4ZjUxYWNhY2IzYjhiNjNlNzFlYmIzYWFmYTU5NmZmYyIsImFsZyI6IlJTMjU2In0.eyJzdWIiOiJjMjY5Yzg3NDA3YWU0MzBkMWIxZThjNzNhN2IwZTZjYSIsImtpZCI6IjhmNTFhY2FjYjNiOGI2M2U3MWViYjNhYWZhNTk2ZmZjIiwiZXhwIjoxNzQ5NjA3NjA5fQ.jBYcsPZW0MCBFsXcOUdu_cHXARlhYwdpDPb9nPhUz22-N3ElWsypJ75fRFLFD-6y0u97WCvTBL0Q5vZFKc2LAi8sBr_z-SanW9ogdS5vkRQmVpFylOqcc6L3HtSXmSxY32i7Tcx4I95WGyH0QtPzwlNaCOw2E3RljfrrLPS-Fvixe8IrlpzqRfVpNEFE02F8iCf1MIUw-KlIpjv_Ay03D5RvyXUaO1uTPqSMWphJUt2ba_voUVw4sd0ZudnbEzAB1R2wIRpWp8LVkZ5N5Hf050pLB37QllFCGT7U0dS_sjZGPJ3oBILiEKj4Eq-9liFA9gxpFvrjoJ_YNn3uN3VNPw" allowfullscreen="true" allow="autoplay; picture-in-picture" style="height: 100%; width: 100%; position: absolute; top: 0px; left: 0px; border: none;"></iframe></div></stream></div><a href="/cms/10.1016/j.cell.2020.03.046/attachment/aa5dd9c8-21d0-4e09-91ae-4f7b4af77bfa/mmc7.mp4" download="mmc7.mp4" aria-labelledby="mmc7-heading mmc7-link"><span id="mmc7-link">Video (1.57 MB)</span></a></div><div class="core-description"><div id="mmc7-heading" class="heading">Video S4. FRAP Analysis of GFP-G3BP1 ΔIDR1/2, Related to Figure&nbsp;6</div><div id="espara0025" role="paragraph">U2OS cells were constructed using CRISPR-Cas9 to express tdTomato-tagged endogenous G3BP1. Cells were then transfected with exogenous GFP-G3BP1 ΔIDR1/2. The relative mobility of endogenous (tdTomato-tagged) G3BP1 was compared to exogenous (GFP-tagged) G3BP1 by FRAP.</div></div></div></div></section><section id="sec-2-13"><h3>The Acidic IDR1 Domain of G3BP1 Is an Autoinhibitory Element</h3><div id="p0200" role="paragraph">To further address the role of IDR1 and IDR2 in SG assembly, we directly compared SG assembly in <i>G3BP1/2</i> dKO cells reconstituted with G3BP1 mutants with alterations in IDR1 or IDR2. We examined assembly prior to treatment with arsenite (“spontaneous SGs”), as well as time points after arsenite treatment. Expression of G3BP1 ΔIDR1 not only fully reconstituted SG assembly following stress but resulted in strongly increased formation of spontaneous SGs (<a id="crosref0865" href="#fig6">Figures 6</a>E, 6F, <a id="crosref0870" href="#figs6">S6</a>G, and S6H). In contrast, expression of G3BP1 ΔIDR2 failed to support SG assembly (<a id="crosref0875" href="#fig6">Figures 6</a>E, 6F, <a id="crosref0880" href="#figs6">S6</a>G, and S6H), which contrasts sharply with the ability of G3BP1 ΔIDR1/2 to reconstitute SG assembly. Taken together, these results reveal that the acidic region comprising IDR1 is an autoinhibitory element and further that this inhibition is mitigated by the presence of IDR2.</div><div id="p0205" role="paragraph">We reasoned that the negative charge density present within IDR1, rather than motifs encoded in the primary amino acid sequence, might mediate its inhibitory effect on SG assembly. To test this notion, we generated three different G3BP1 mutants in which the primary IDR1 sequence was scrambled and the adjacent IDR2 was deleted (<a id="crosref0885" href="#fig6">Figures 6</a>G and <a id="crosref0890" href="#figs6">S6</a>I). All three of these mutants failed to reconstitute SG assembly, indicating that the inhibitory activity of IDR1 was preserved (<a id="crosref0895" href="#fig6">Figures 6</a>H and <a id="crosref0900" href="#figs6">S6</a>J). We also generated a mutant in which all 28 glutamates in IDR1 were substituted with charge-neutral glutamines (EQ28 ΔIDR2; <a id="crosref0905" href="#figs6">Figure&nbsp;S6</a>I). By contrast with the scrambled mutants, this mutant supported robust SG assembly in <i>G3BP1/2</i> dKO cells (<a id="crosref0910" href="#fig6">Figures 6</a>H and <a id="crosref0915" href="#figs6">S6</a>K). Together, these results indicate that the inhibitory activity of IDR1 is encoded by the negative charge density in this region rather than the primary sequence.</div><div id="p0210" role="paragraph">We next turned to IDR2, which has a net positive charge density and is enriched in proline residues that may confer functionally relevant rigidity to this segment. To investigate the importance of spacing, composition, and primary sequence of IDR2, we engineered a series of substitutions or mutations. First, we substituted IDR2 with an IDR of similar length and composition derived from the <i>S.&nbsp;cerevisiae</i> transcription factor Ash1 (G3BP1-Ash1 IDR). This 79-residue proline-rich IDR adopts expanded coil-like conformations (<a id="crosref0920" href="#bib35" role="doc-biblioref" data-xml-rid="bib35">Martin et&nbsp;al., 2016a</a>). When expressed in <i>G3BP1/2</i> dKO cells, G3BP1-Ash1 IDR supported WT levels of SG assembly (<a id="crosref0925" href="#fig6">Figures 6</a>I, 6J, and <a id="crosref0930" href="#figs6">S6</a>L), suggesting that the spacing and/or composition within this segment, rather than its primary sequence, is important for suppressing the inhibitory activity of IDR1.</div><div id="p0215" role="paragraph">To assess the importance of spacing, we introduced a second copy of IDR2 (G3BP1-2×IDR2) to double the putative spacing afforded by IDR2. This second copy of IDR2 significantly enhanced SG assembly relative to G3BP1 WT, perhaps by separating the negatively charged acidic IDR1 from the positively charged IDR3 within the RBD or by acting as a decoy interrupting this interaction (<a id="crosref0935" href="#figs6">Figures S6</a>M and S6N). We also assessed the importance of the proline residues in IDR2 by substituting all 33 proline residues in IDR1/2 with serine (PS33). When expressed in <i>G3BP1/2</i> dKO cells, this mutant supported WT levels of SG assembly, indicating that these proline residues are not critically important for the mitigating effect of IDR2 on IDR1 inhibition (<a id="crosref0940" href="#fig6">Figures 6</a>I, 6J, and <a id="crosref0945" href="#figs6">S6</a>L). On the other hand, a mutant with all 17 positively charged residues in IDR2 substituted with alanine (RKH/A17) was unable to support SG formation when expressed in <i>G3BP1/2</i> dKO cells, suggesting that, beyond the spacing afforded by IDR2, this domain might engage in charge-dependent interaction with IDR1, mitigating the interaction between IDR1 and IDR3 (<a id="crosref0950" href="#fig6">Figures 6</a>I, 6J, and <a id="crosref0955" href="#figs6">S6</a>L).</div><div id="p0220" role="paragraph">Notably, when we reconstituted SGs with G3BP1 ΔIDR1/2, 8 proteins from a panel of 17 SG markers showed reduced recruitment to SGs (<a id="crosref0960" href="#fig6">Figure&nbsp;6</a>K). This observation suggested that, in addition to regulating SG assembly and dynamics, IDR1/2 might play a role in determining the composition of SGs. To comprehensively assess the role of IDR1/2 in SG composition, we performed proximity proteomics using APEX2-labeled G3BP1. For this analysis, we compared G3BP1 WT to G3BP1-2×Ash1 IDR, in which both IDR1 and IDR2 were replaced by Ash1 IDR sequences. We selected this mutant for head-to-head comparison because G3BP1-2×Ash1 affords highly efficient SG assembly despite harboring a completely different IDR primary sequence (<a id="crosref0965" href="#fig6">Figures 6</a>L and 6M). Despite indistinguishable kinetics and frequency of SG assembly in response to arsenite, altering this IDR sequence resulted in significant differences in SG composition. Specifically, nine proteins that are consistently reported in SGs (<a id="crosrefs0050" href="#bib23" role="doc-biblioref" data-xml-rid="bib23 bib34">Jain et&nbsp;al., 2016; Markmiller et&nbsp;al., 2018</a>) were absent from SGs seeded by G3BP1-2×Ash1 IDR, and recruitment of an additional four SG proteins was significantly reduced (<a id="crosref0970" href="#fig6">Figure&nbsp;6</a>N; <a id="crosref0975" href="#mmc3">Table S3</a>). Moreover, SGs seeded with G3BP1-2×Ash1 IDR recruited 43 proteins not previously reported in SGs (<a id="crosrefs0055" href="#bib23" role="doc-biblioref" data-xml-rid="bib23 bib34">Jain et&nbsp;al., 2016; Markmiller et&nbsp;al., 2018</a>), and 43 other proteins showed significantly increased recruitment to these mutant SGs (<a id="crosref0980" href="#fig6">Figure&nbsp;6</a>N; <a id="crosref0985" href="#mmc3">Table S3</a>). Thus, the long central IDR plays a role in defining SG composition.</div></section><section id="sec-2-14"><h3>IDR1 Phosphorylation Tunes Interplay between 3 IDRs and Regulates LLPS</h3><div id="p0225" role="paragraph">Several studies have suggested that post-translational modifications regulate G3BP1 (<a id="crosrefs0060" href="#bib51" role="doc-biblioref" data-xml-rid="bib51 bib57 bib59">Sahoo et&nbsp;al., 2018; Tourrière et&nbsp;al., 2003; Tsai et&nbsp;al., 2016</a>). Among these, phosphorylation of S149 in IDR1 has been implicated as a regulator of SG assembly. Whereas the fact that S149 can be phosphorylated is clear, the precise role of this post-translational modification in SG assembly has been uncertain (<a id="crosrefs0065" href="#bib46" role="doc-biblioref" data-xml-rid="bib46 bib51 bib57">Panas et&nbsp;al., 2019; Sahoo et&nbsp;al., 2018; Tourrière et&nbsp;al., 2003</a>). Nevertheless, recent compelling evidence has been reported that G3BP1-S149 phosphorylation is functionally relevant during physiological stress in neurons (<a id="crosref0990" href="#bib51" role="doc-biblioref" data-xml-rid="bib51">Sahoo et&nbsp;al., 2018</a>).</div><div id="p0230" role="paragraph">To illuminate the biophysical mechanism of G3BP1 regulation by S149 phosphorylation, we generated G3BP1 variants in which the charge content of IDR1 was altered: a phosphomimetic form (S149E), in which the negative charge density is modestly increased; a non-phosphorylatable form (S149A) that does not alter charge; and EQ28, in which the negative charge density of IDR1 is substantially reduced. Consistent with a recent report (<a id="crosref0995" href="#bib46" role="doc-biblioref" data-xml-rid="bib46">Panas et&nbsp;al., 2019</a>), the phosphorylation state of S149 did not change the ability of G3BP1 to dimerize as assessed by <i>in&nbsp;vitro</i> cross-linking analysis (<a id="crosref1000" href="#figs7">Figures S7</a>A–S7C). We then examined the impact of altered IDR1 charge content on LLPS <i>in&nbsp;vitro</i>. Compared to G3BP1 WT, the S149E, S149A, and EQ28 variants had no significant influence on Ficoll-dependent phase transition (<a id="crosref1005" href="#figs7">Figure&nbsp;S7</a>D). In contrast, RNA-dependent LLPS of G3BP1 was strongly influenced by the charge content of IDR1: whereas G3BP1 S149A behaved similarly to WT, the S149E variant strongly impaired LLPS of G3BP1 with RNA, shifting the phase separation threshold to significantly higher concentrations of G3BP1 and RNA (<a id="crosref1010" href="#fig7">Figures 7</a>A and <a id="crosref1015" href="#figs7">S7</a>E). The charge-neutralizing EQ28 variant had the opposite effect and strongly promoted LLPS of G3BP1 with RNA (<a id="crosref1020" href="#fig7">Figures 7</a>A and <a id="crosref1025" href="#figs7">S7</a>E). Because only RNA-dependent LLPS of G3BP1 was influenced by the charge content of IDR1, we next tested the influence of IDR1 charge content on RNA binding by G3BP1. As predicted by the LLPS results, EMSA using purified protein and RNA revealed similar RNA binding capacity by G3BP1 WT and S149A but decreased RNA binding by G3BP1 S149E and increased RNA binding by G3BP1 EQ28 (<a id="crosref1030" href="#fig7">Figure&nbsp;7</a>B). Consistent results were obtained in cells, where S149E showed reduced RNA binding and EQ28 showed increased RNA binding (<a id="crosref1035" href="#fig7">Figure&nbsp;7</a>C).</div><div class="figure-wrap"><figure id="fig7" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2020.03.046/asset/d2292605-dee9-4ea2-b852-a830b7da089b/main.assets/gr7_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr7" src="/cms/10.1016/j.cell.2020.03.046/asset/2f45af03-d1dc-41b6-ad8a-63106be809cd/main.assets/gr7.jpg" height="904" width="770" aria-labelledby="fig7-title fig7-content" data-viewer-src="/cms/10.1016/j.cell.2020.03.046/asset/d2292605-dee9-4ea2-b852-a830b7da089b/main.assets/gr7_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig7-title" class="figure__title"><span class="label figure__label">Figure&nbsp;7</span> <span class="figure__title__text">IDR1 Phosphorylation Tunes Interplay between 3 IDRs and Regulates LLPS</span></div><div class="accordion"><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><div id="fig7-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0345" role="paragraph">(A) Summary of RNA-triggered LLPS of G3BP1 under indicated conditions, in the presence of 150&nbsp;mM NaCl. Corresponding images are shown in <a id="crosref1415" href="#figs7">Figure&nbsp;S7</a>E.</div><div id="fspara0350" role="paragraph">(B) Electrophoretic mobility shift assay of G3BP1 WT and IDR1 mutants.</div><div id="fspara0355" role="paragraph">(C) CLIP analysis of G3BP1 IDR1 mutants. Indicated G3BP1 constructs were transiently expressed in <i>G3BP1/2</i> dKO cells and analyzed as in <a id="crosref1420" href="#fig4">Figure&nbsp;4</a>E. Error bars indicate SD. <sup>∗∗</sup>p&nbsp;= 0.005 versus S149A by one-way ANOVA; Dunnett’s multiple comparisons test.</div><div id="fspara0360" role="paragraph">(D and E) GST pull-down of purified GST-RBD with HA-G3BP1 N-terminal mutants indicates an intramolecular interaction between G3BP1 RBD and IDR1, and IDR2 mitigates this interaction. Results are quantified in (E). Error bars indicate SD. <sup>∗∗∗</sup>p&nbsp;= 0.0002 and <sup>∗∗∗∗</sup>p&nbsp;&lt; 0.0001 versus NTF2L-IDR1-HA by one-way ANOVA; Dunnett’s multiple comparisons test.</div><div id="fspara0365" role="paragraph">(F and G) GST pull-down of GST-RBD with NTF2L-IDR1-HA. RBD interactions with WT or mutant IDR1 (S149E; EQ28) were assessed as in (D) and quantified in (G). Error bars indicate SD. <sup>∗∗∗</sup>p&nbsp;= 0.0006 and <sup>∗∗∗∗</sup>p&nbsp;&lt; 0.0001 by one-way ANOVA; Tukey’s multiple comparisons test.</div><div id="fspara0370" role="paragraph">(H and I) GST pull-down of GST-RBD mutants with NTF2L-IDR1-HA. IDR1 interactions with WT or RBD<sup>RA5</sup> were assessed as in (D) and quantified in (I). Error bar indicates SD. <sup>∗∗∗</sup>p&nbsp;= 0.0002 by unpaired t test.</div><div id="fspara0375" role="paragraph">(J) Small-angle X-ray scattering (SAXS) data for G3BP1 WT at high and low NaCl concentration. Data are presented as normalized Kratky plots, where the intensity is normalized by the zero-angle scattering and the momentum transfer (q) is normalized by the radius of gyration (R<sub>g</sub>). The intersection of dashed lines indicates the theoretical maximum of a solid sphere. Experimental data with a higher maximum coupled to a shift to the right are caused by conformational flexibility. Comparing the experimental data to synthetic data for a sphere and a self-avoiding random walk (SARW) indicates that G3BP1 is flexible in both conditions and expands toward maximal random dimensions at high NaCl concentration.</div><div id="fspara0380" role="paragraph">(K) R<sub>g</sub>s were extracted from SAXS data by Guinier analysis at 50–1,000&nbsp;mM NaCl. Data were fit to a logistic function, suggesting a minimum dimension of ∼55&nbsp;Å at low NaCl concentration to a maximum extension of ∼72&nbsp;Å at high NaCl concentration with a transition with a midpoint at ∼215&nbsp;mM NaCl, indicating an interaction between oppositely charged regions of the protein. Error bars indicate SD.</div><div id="fspara0385" role="paragraph">(L) Distribution of G3BP1 radii in randomly generated conformations.</div><div id="fspara0390" role="paragraph">(M) In random synthetic conformations, the distance between IDR1 and IDR3 is positively correlated with the total R<sub>G</sub>.</div><div id="fspara0395" role="paragraph">(N) Representative conformations of G3BP1. Conformations with representative radii were sampled from the randomly generated pool shown in (M) to show compact conformations enriched in buffers with ionic strength below 200&nbsp;mM (top) and extended conformations that would be enriched at high ionic strength (bottom).</div><div id="fspara0400" role="paragraph">(O) <i>G3BP1/2</i> dKO cells transiently expressing indicated G3BP1 constructs were exposed to sodium arsenite (500&nbsp;μM; 1 h), fixed, and stained for eIF3η. Scale bar, 50&nbsp;μm.</div><div id="fspara0405" role="paragraph">(P) Intracellular phase diagram of indicated G3BP1 constructs transfected into <i>G3BP1/2</i> dKO cells. Cells were exposed to 100&nbsp;μM sodium arsenite (30&nbsp;min), fixed, and stained for PABP. SG formation and GFP intensities were assessed cell by cell. Cells with SGs are plotted as filled circles; cells without SGs are plotted as empty circles. Boxes highlight the 25% highest levels of expression among SG-negative cells.</div><div id="fspara0410" role="paragraph">Vertical lines in blots in (D) and (F) indicate noncontiguous lanes from the same gel.</div><div id="fspara0415" role="paragraph">See also <a id="crosref1425" href="#figs7">Figure&nbsp;S7</a>.</div></div></div></div></figcaption></figure></div><div class="figure-wrap"><figure id="figs7" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2020.03.046/asset/36a19283-478b-4ac4-b2d3-9d0f1e9016b8/main.assets/figs7_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs7" src="/cms/10.1016/j.cell.2020.03.046/asset/13a0e72e-a823-4d76-bd61-39e1773e7476/main.assets/figs7.jpg" height="812" width="770" aria-labelledby="figs7-title figs7-content" data-viewer-src="/cms/10.1016/j.cell.2020.03.046/asset/36a19283-478b-4ac4-b2d3-9d0f1e9016b8/main.assets/figs7_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs7-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S7</span> <span class="figure__title__text">IDR1 Phosphorylation Tunes Interplay between 3 IDRs and Regulates LLPS, Related to <a id="crosref1500" href="#fig7">Figure&nbsp;7</a></span></div><div class="accordion"><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><div id="figs7-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0745" role="paragraph">(A) Phosphorylation-dependent charge distribution of G3BP1. Phosphorylation on S149 increases negative charges in IDR1.</div><div id="fspara0750" role="paragraph">(B) Indicated recombinant proteins were incubated with the cross-linker bissulfosuccinimidyl suberate (BS<sup>3</sup>) and then analyzed by SDS-PAGE and Coomassie Blue staining. M, monomer; D, dimer.</div><div id="fspara0755" role="paragraph">(C) Extracts obtained from <i>G3BP1/2</i> dKO cells transfected with indicated GFP-tagged G3BP1 constructs were cross-linked using BS<sup>3</sup> and analyzed by immunoblotting with GFP antibody. M, monomer; D, dimer.</div><div id="fspara0760" role="paragraph">(D) Ficoll-dependent LLPS of G3BP1 WT and IDR1 mutants (S149E, S149A, or EQ28). Indicated concentrations of purified G3BP1 were mixed with 10% Ficoll in HEPES 50&nbsp;mM, pH 7.5, 150&nbsp;mM NaCl, 1&nbsp;mM DTT, and imaged by DIC. Scale bar, 50&nbsp;μm.</div><div id="fspara0765" role="paragraph">(E) RNA-dependent LLPS of G3BP1 WT and IDR1 mutants. Indicated concentrations of purified G3BP1 were mixed with increasing concentration of total RNA purified from cells in HEPES 50&nbsp;mM, pH 7.5, 150&nbsp;mM NaCl, 1&nbsp;mM DTT, and imaged by DIC. Scale bars, 50&nbsp;μm.</div><div id="fspara0770" role="paragraph">(F) RNA-dependent (left) and Ficoll-dependent (right) LLPS of G3BP1<sup>RA5</sup> and G3BP1<sup>RK5</sup>. 200&nbsp;μM of purified G3BP1 was mixed with either 200&nbsp;ng/μL total RNA or 10% Ficoll and imaged by DIC. Substitution of five arginines in IDR3 with lysines (RBD<sup>RK5</sup>) permitted RNA-dependent LLPS <i>in&nbsp;vitro</i>, whereas substitution of the same arginines with alanines did not, illustrating that charge rather than other properties specific to arginine residues is important for LLPS. Scale bars, 10&nbsp;μm.</div><div id="fspara0775" role="paragraph">(G) <i>G3BP1/2</i> dKO cells were reconstituted with GFP-tagged G3BP1 (RA5 or RK5), exposed to sodium arsenite (500&nbsp;μM, 1 h), and stained for eIF3η. Scale bar, 20&nbsp;μm.</div><div id="fspara0780" role="paragraph">(H) <i>G3BP1/2</i> dKO cells were reconstituted with GFP-tagged G3BP1 or GFP-tagged synthetic G3BP1, exposed to sodium arsenite (500&nbsp;μM, 1 h), and stained for eIF3η. Scale bar, 20&nbsp;μm.</div><div id="fspara0785" role="paragraph">(I) The molecular switch in G3BP1 arises from competition between intra- and inter-molecular interactions of three IDRs. Phosphorylation of IDR1 favors the closed conformation by enhancing intramolecular interaction between IDR1 (which is highly acidic) and IDR3 (the basic component of the RBD). Upon increased RNA concentration, RNA displaces IDR1 to bind IDR3, which permits the G3BP1 homodimer to adopt an expanded, open conformation, initiating LLPS.</div></div></div></div></figcaption></figure></div><div id="p0235" role="paragraph">We hypothesized that the autoinhibitory activity of IDR1 reflects electrostatic intramolecular interaction with the RBD, thereby competing with RNA for binding. Further, we hypothesized that increased negative charge in IDR1 caused by phosphorylation enhances its autoinhibitory activity by strengthening the intramolecular IDR1-RBD interaction. To test these ideas, we performed GST pull-down experiments to investigate a putative IDR1-RBD interaction. We tested the ability of G3BP1 protein fragments (NTF2L domain alone, NTF2L-IDR1, and NTF2L-IDR1/2) to interact with a fragment consisting of the RBD (which contains RRM-IDR3). No significant interaction was detected between the RBD and the NTF2L domain alone, as expected, but strong interaction was detected when IDR1 was included (NTF2L-IDR1; <a id="crosref1040" href="#fig7">Figures 7</a>D and 7E). The presence of IDR2 (NTF2L-IDR1/2) mitigated this interaction (<a id="crosref1045" href="#fig7">Figures 7</a>D and 7E), consistent with the ability of IDR2 to mitigate the autoinhibitory activity of IDR1 in SG assembly (<a id="crosref1050" href="#fig6">Figures 6</a>I and 6J). This interaction was enhanced by the S149E mutation and diminished by the charge-neutralizing EQ28 mutations (<a id="crosref1055" href="#fig7">Figures 7</a>F and 7G). We reasoned that electrostatic interaction between the negatively charged IDR1 and RBD was mediated by the positive charge density in IDR3. To test this notion, we reduced the positive charge in IDR3 by introducing five arginine-to-alanine substitutions (RBD<sup>RA5</sup>). RBD<sup>RA5</sup> had greatly reduced interaction with NTF2L-IDR1, confirming electrostatic interaction between IDR1 and the RBD, mediated by IDR3 (<a id="crosref1060" href="#fig7">Figures 7</a>H and 7I). The positive charges in IDR3 are also essential for RNA binding, as illustrated by lack of RNA-dependent LLPS of G3BP1-RBD<sup>RA5</sup> and the inability of this mutant to reconstitute SG assembly in <i>G3BP1/2</i> dKO cells (<a id="crosref1065" href="#figs7">Figures S7</a>F and S7G). Of note, G3BP1-RBD<sup>RA5</sup> provides a further example of differences in RNA-dependent and Ficoll-dependent LLPS (<a id="crosref1070" href="#figs7">Figure&nbsp;S7</a>F). Substitution of these same arginines in IDR3 with lysines (RBD<sup>RK5</sup>) did permit RNA-dependent LLPS <i>in&nbsp;vitro</i> and reconstituted SG assembly in <i>G3BP1/2</i> dKO cells, illustrating that charge rather than other properties specific to arginine residues underlie these activities (<a id="crosref1075" href="#figs7">Figures S7</a>F and S7G).</div><div id="p0240" role="paragraph">We further explored the influence of IDR1-IDR3 interaction on G3BP1 conformation by using small-angle X-ray scattering (SAXS) to analyze purified G3BP1 across a range of salt concentrations. Increasing NaCl concentration shifted the protein from an ensemble of compact conformations with a mean radius of gyration (⟨R<sub>G</sub>⟩) of 55&nbsp;Å to extended conformations with ⟨R<sub>G</sub>⟩ of 72&nbsp;Å. The sharp transition from compact to extended conformations occurred within the range of physiological ionic strength, indicating that G3BP1 size and shape is determined by electrostatic interactions. Moreover, at physiological ionic strength, this conformational change in G3BP1 is likely sensitive to the small changes in charge that occur with phosphorylation (<a id="crosref1080" href="#fig7">Figures 7</a>J and 7K). To further characterize the conformational landscape of G3BP1, we compared experimental SAXS data with an ensemble of <i>in silico</i> generated conformations where the IDRs were modeled as self-avoiding polymers. The <i>in silico</i> ensemble had a mean radius of 69&nbsp;Å, consistent with the ⟨R<sub>G</sub>⟩ measured at high NaCl concentration by SAXS (<a id="crosref1085" href="#fig7">Figures 7</a>K and 7L). Thus, at lower, physiological ionic strength, the IDRs are compacted by attractive electrostatic interactions relative to an unrestrained self-avoiding polymer. We then measured the distances within the G3BP1 dimer between the two negatively charged IDR1s and the positively charged IDR3s and observed a linear correlation between the overall radius of the protein and the distance separating IDR1 and IDR3, indicating that overall compaction likely reports on close association between these domains (<a id="crosref1090" href="#fig7">Figures 7</a>M and 7N). These observations strongly implicate intramolecular interaction between IDR1 and IDR3 that controls RNA interaction and RNA-dependent LLPS.</div><div id="p0245" role="paragraph">Finally, we investigated the role of the G3BP1 conformational switch on the assembly of SGs. G3BP1 WT, S149E, S149A, and EQ28 all rescued SG formation in <i>G3BP1/2</i> dKO cells (<a id="crosref1095" href="#fig7">Figure&nbsp;7</a>O). By measuring the intracellular G3BP1 concentration threshold for phase separation by these mutants, we found that G3BP1 S149A required significantly lower concentrations to initiate SG assembly relative to G3BP1 WT (<a id="crosref1100" href="#fig7">Figure&nbsp;7</a>P), as expected because G3BP1 WT is found as a mixture of phosphorylated and unphosphorylated forms in cultured cells (<a id="crosref1105" href="#bib14" role="doc-biblioref" data-xml-rid="bib14">Gallouzi et&nbsp;al., 1998</a>). Consistent with this observation, G3BP1 EQ28 initiated SG assembly at very low concentrations (<a id="crosref1110" href="#fig7">Figure&nbsp;7</a>P). In contrast, G3BP1 S149E required higher concentrations to initiate SG assembly relative to WT (<a id="crosref1115" href="#fig7">Figure&nbsp;7</a>P). Together, these data indicate that IDR1 regulates G3BP1-RNA LLPS through a conformational switch that controls RNA binding and the subsequent assembly of SGs in cells and, further, that this switch can be tuned by phosphorylation status.</div><div id="p0250" role="paragraph">We were intrigued by the observation that each of the three major domains of G3BP1 was either non-essential or could be substituted by a heterologous domain with similar properties. This raised the question of whether any precise sequence or structure encoded by G3BP1 was essential to its function with respect to SG assembly. To address this question, we created a “synthetic G3BP1” molecule in which GST served as an N-terminal dimerization domain, tandem Ash-1 IDR sequences served as a disordered spacer segment, and tandem RRMs from hnRNPA1 served as a multivalent RNA-binding domain at the C terminus (<a id="crosref1120" href="#figs7">Figure&nbsp;S7</a>H). Remarkably, synthetic G3BP1 restored SG formation in <i>G3BP1/2</i> dKO cells (<a id="crosref1125" href="#figs7">Figure&nbsp;S7</a>H), although the fine-tuning ability through IDR1-IDR3 interaction and phosphorylation of IDR1 was lost.</div></section></section><section id="sec-3"><h2>Discussion</h2><div id="p0255" role="paragraph">Cells partition a large fraction of RNAs and associated proteins into RNP granules, an organizational strategy that not only governs RNA metabolism but also generates cellular compartments that do double duty by regulating other biological pathways. SGs, for example, compartmentalize a fraction of mRNAs stalled in translation, which may limit the accumulation of irreversible RNA aggregates, but also serve as a compartment that promotes other stress-related functions (<a id="crosrefs0070" href="#bib43" role="doc-biblioref" data-xml-rid="bib43 bib52 bib56 bib58 bib66">Onomoto et&nbsp;al., 2012; Samir et&nbsp;al., 2019; Thedieck et&nbsp;al., 2013; Tsai and Lloyd, 2014; Wippich et&nbsp;al., 2013</a>). The dynamic nature of RNP granules, including SGs, is essential to their function; indeed, impairment in the dynamics and material properties of RNP granules is strongly implicated as a factor in neurodegenerative diseases (<a id="crosref1130" href="#bib40" role="doc-biblioref" data-xml-rid="bib40">Nedelsky and Taylor, 2019</a>).</div><div id="p0260" role="paragraph">Here we introduce and functionally validate a conceptual framework for SG assembly in which assembly is driven by LLPS that occurs when the collective interactions of a core protein-RNA interaction network breach a threshold saturation concentration. This core network consists of ∼36 proteins and associated mRNAs. Importantly, the interactions driving this LLPS are distributed unevenly across the core network, with some nodes more important than others. The central node of this network is G3BP1, which functions as a molecular switch that triggers RNA-dependent LLPS in response to a rise in intracellular free RNA concentrations. The threshold for phase separation—and corresponding SG assembly—is set predominantly by intrinsic properties of G3BP1 and RNA and further regulated through positive or negative cooperativity by extrinsic factors that act upon core SG network.</div><div id="p0265" role="paragraph">With regard to the intrinsic properties of RNA that influence LLPS with G3BP, we found that length (&gt;250 nt) and single strandedness form the basis for this interaction. Furthermore, although intermolecular RNA-RNA interactions are not essential, they can promote assembly and lower the threshold of constituents required for LLPS. These conclusions are consistent with the observation that G3BP1 association with polyA RNA significantly increases after arsenite stress, as well as the previous report that the SG transcriptome is enriched predominantly in mRNAs and to a lesser extent non-coding RNAs (<a id="crosref1135" href="#bib27" role="doc-biblioref" data-xml-rid="bib27">Khong et&nbsp;al., 2017</a>). We found that no specific RNA sequence is required for LLPS with G3BP1, which is consistent with cross-linking experiments conducted in cells showing that G3BP1 binds a variety of coding and non-coding RNAs without a specific consensus sequence (<a id="crosrefs0075" href="#bib11" role="doc-biblioref" data-xml-rid="bib11 bib36">Edupuganti et&nbsp;al., 2017; Martin et&nbsp;al., 2016b</a>).</div><div id="p0270" role="paragraph">The intrinsic properties of G3BP1 that underlie LLPS with RNA are dimerization via the NTF2L domain and multivalency for RNA binding. Importantly, RNA binding by the RBDs (and therefore, LLPS) is controlled by a molecular switch. This switch in G3BP1 arises from competition between intra- and inter-molecular interactions (<a id="crosref1140" href="#figs7">Figure&nbsp;S7</a>I). When RNA concentrations are low, intramolecular interaction between the acidic IDR1 and the basic IDR3 is favored, creating a compact, “closed” conformation. Above a threshold concentration, RNA displaces IDR1 to bind IDR3, which permits the G3BP1 homodimer to adopt an expanded, “open” conformation, initiating LLPS. Thus, IDR1 functions as an autoinhibitory element that sets the threshold concentration of RNA necessary for LLPS. The autoinhibitory activity of IDR1 is regulated by IDR2, which is weakly basic and can impede IDR1 interaction with IDR3, and also by phosphorylation of IDR1, which strengthens the interaction with IDR3. Consistent findings regarding the molecular switch encoded by G3BP1 were found in the companion manuscript by Guillen-Boixet et&nbsp;al. (<a id="crosref0150" href="#bib78" role="doc-biblioref" data-xml-rid="bib78">Guillén-Boixet et al., 2020</a>), in this issue of <i>Cell</i>. The domain architecture of dimeric G3BP1 is another intrinsic property that influences LLPS. Dimerization via NTF2L not only doubles the valency for RNA binding but also determines spacing between the two RBDs, with NTF2L dimerization establishing the center of the volume occupied by dimeric G3BP1 and the RBDs able to explore the periphery of this volume, which likely influence the ability of a G3BP1 homodimer to cross-link different RNA molecules and promote LLPS. In sum, our findings regarding the IDRs of G3BP demonstrate that, in contrast to the conventional view that the role of IDRs is to “drive” LLPS, IDRs in G3BP1 may have evolved to fine-tune the saturation concentration for LLPS. Beyond this important role, the IDRs in G3BP1 influence both the material properties and the composition of SGs.</div><div id="p0275" role="paragraph">Whereas intrinsic properties of G3BP1 and RNA are the most important determinants of SG formation, other proteins influence this assembly. For example, overexpression of caprin-1 and TIA1 can lead to the formation of “spontaneous” SGs in the absence of an exogenous stress (<a id="crosref1145" href="#bib15" role="doc-biblioref" data-xml-rid="bib15">Gilks et&nbsp;al., 2004</a>). This phenomenon can now be fully explained within the conceptual framework for SG assembly that we advance here. The existence of a saturation concentration for LLPS is indicative of a cooperative process (<a id="crosrefs0080" href="#bib9" role="doc-biblioref" data-xml-rid="bib9 bib48">Crick et&nbsp;al., 2013; Posey et&nbsp;al., 2018</a>). A change in the network of interactions that lowers the saturation concentration for LLPS, thus impacting the cooperative transition positively, is termed “positive cooperativity.” In contrast, a change in the network of interactions that increases the saturation concentration represents “negative cooperativity.” The impact of a variety of extrinsic factors that regulate SG assembly can be understood to act through positive or negative cooperativity. For example, our data illustrate that introduction of either caprin-1 or TIA1 to a mixture of G3BP1 and RNA strengthens the collective network of interactions and lowers the saturation concentration for LLPS, exhibiting positive cooperativity. We document this same phenomenon in cells, where the concentration of endogenous caprin-1 or TIA1 influences the G3BP1 saturation concentration necessary to trigger SG assembly. We speculate that many components of the core SG network function in this manner, lowering the saturation concentration of G3BP1 and RNA necessary for SG assembly. On the other hand, it has also been observed that overexpression of USP10, another component of the core SG network, inhibits SG assembly (<a id="crosref1150" href="#bib26" role="doc-biblioref" data-xml-rid="bib26">Kedersha et&nbsp;al., 2016</a>). How might negative cooperativity be introduced to this system? USP10 and caprin-1 compete for overlapping binding sites in the NTF2L domain and exhibit mutually exclusive interaction with G3BP1 (<a id="crosref1155" href="#bib26" role="doc-biblioref" data-xml-rid="bib26">Kedersha et&nbsp;al., 2016</a>). It is highly likely that USP10, upon being substituted for caprin-1, weakens the collective network of interactions. Notably, affinity capture proteomics using the NTF2L domain as bait reveals that a large number of G3BP1 interactors in the core SG network can be traced to NTF2L-mediated interactions (P.Y., unpublished data), highlighting this domain as a central hub controlling network connectivity. Consistent findings regarding a central hub of interaction encoded by the NTF2L domain of G3BP1 were found in the companion manuscript by Sanders et&nbsp;al. (<a id="crosref0660" href="#bib79" role="doc-biblioref" data-xml-rid="bib79">Sanders et al., 2020</a>), in this issue of <i>Cell</i>.</div><div id="p0280" role="paragraph">In conclusion, these studies illuminate the molecular basis of SG assembly and provide a conceptual framework for understanding regulation of this process that builds upon fundamental biophysical principles. Importantly, this conceptual framework accommodates all known biophysical principles and regulators of SG assembly, including the role of other proteins, post-translational modifications of proteins, duplexing or unwinding of RNA, or chemical modification of RNAs, all of which may be viewed as impinging upon the collective interactions of a core protein-RNA interaction network. Furthermore, the principles established here are likely generalizable to many biomolecular condensates. A “scaffold/client” model was recently advanced based on observations in simple, reconstituted systems where the assembly of condensates was regulated by the valency, concentration, and molar ratio of a small number of scaffold molecules (<a id="crosref1160" href="#bib1" role="doc-biblioref" data-xml-rid="bib1">Banani et&nbsp;al., 2016</a>). In light of the work presented here, we can add network centrality as an important feature driving biomolecular condensation and providing a conceptual framework for how this condensation is regulated.</div></section><section id="sec-4"><h2>STAR★Methods</h2><section id="sec-4-1"><h3>Key Resources Table</h3><div id="p0285" role="paragraph"><div class="figure-wrap"><figure id="undtbl1" class="table" data-figureviewer-ignore="true"><div class="table-wrap"><table data-figureviewer-ignore="true"><thead><tr class="rowsep"><th>REAGENT or RESOURCE</th><th>SOURCE</th><th>IDENTIFIER</th></tr></thead><tbody><tr class="rowsep"><td colspan="3"><b>Antibodies</b></td></tr><tr><td>Mouse monoclonal anti-ATXN2 (22/Ataxin-2) antibody</td><td>BD Biosciences</td><td>Cat# 611378; RRID: <a id="intref0010" href="https://antibodyregistry.org/AB_398900" target="_blank">AB_398900</a></td></tr><tr><td>Mouse monoclonal anti-G3BP (23/G3BP) antibody</td><td>BD Biosciences</td><td>Cat# 611126; RRID: <a id="intref0015" href="https://antibodyregistry.org/AB_398437" target="_blank">AB_398437</a></td></tr><tr><td>Rabbit polyclonal anti-ATXN2L antibody</td><td>Bethyl Laboratories</td><td>Cat# A301-370A; RRID: <a id="intref0020" href="https://antibodyregistry.org/AB_937724" target="_blank">AB_937724</a></td></tr><tr><td>Rabbit polyclonal anti-CAPRIN1 antibody</td><td>Proteintech</td><td>Cat# 15112-1-AP; RRID: <a id="intref0025" href="https://antibodyregistry.org/AB_2070016" target="_blank">AB_2070016</a></td></tr><tr><td>Rabbit polyclonal anti-CSDE1 antibody</td><td>Bethyl Laboratories</td><td>Cat# A303-158A; RRID: <a id="intref0030" href="https://antibodyregistry.org/AB_10895727" target="_blank">AB_10895727</a></td></tr><tr><td>Mouse monoclonal anti-DDX3 antibody</td><td>Santa Cruz</td><td>Cat# sc-81247; RRID: <a id="intref0035" href="https://antibodyregistry.org/AB_2092867" target="_blank">AB_2092867</a></td></tr><tr><td>Rabbit polyclonal anti-FMR1 antibody</td><td>Bethyl Laboratories</td><td>Cat# A305-200A; RRID: <a id="intref0040" href="https://antibodyregistry.org/AB_2631593" target="_blank">AB_2631593</a></td></tr><tr><td>Rabbit polyclonal anti-G3BP2 antibody</td><td>Bethyl Laboratories</td><td>Cat# A302-041A; RRID: <a id="intref0045" href="https://antibodyregistry.org/AB_1576543" target="_blank">AB_1576543</a></td></tr><tr><td>Rabbit polyclonal anti-G3BP2 antibody</td><td>Proteintech</td><td>Cat# 16276-1-AP</td></tr><tr><td>Mouse monoclonal anti-GAPDH</td><td>Santa Cruz</td><td>Cat# sc47724; RRID: <a id="intref0050" href="https://antibodyregistry.org/AB_627678" target="_blank">AB_627678</a></td></tr><tr><td>Mouse monoclonal anti-GFP antibody</td><td>Santa Cruz</td><td>Cat# sc-9996; RRID: <a id="intref0055" href="https://antibodyregistry.org/AB_627695" target="_blank">AB_627695</a></td></tr><tr><td>Rabbit polyclonal anti-GST antibody</td><td>Abcam</td><td>Cat# ab19256, RRID: <a id="intref0060" href="https://antibodyregistry.org/AB_444809" target="_blank">AB_444809</a></td></tr><tr><td>Rabbit polyclonal anti-HA-tag antibody</td><td>Abcam</td><td>Cat# ab9110, RRID: <a id="intref0065" href="https://antibodyregistry.org/AB_307019" target="_blank">AB_307019</a></td></tr><tr><td>Rabbit monoclonal anti-HDAC6 antibody</td><td>Abcam</td><td>Cat# ab133493</td></tr><tr><td>Rabbit polyclonal anti-NUFIP2 antibody</td><td>Bethyl Laboratories</td><td>Cat# A301-600A; RRID: <a id="intref0070" href="https://antibodyregistry.org/AB_1078870" target="_blank">AB_1078870</a></td></tr><tr><td>Rabbit polyclonal anti-PABP antibody</td><td>Abcam</td><td>Cat# ab21060; RRID: <a id="intref0075" href="https://antibodyregistry.org/AB_777008" target="_blank">AB_777008</a></td></tr><tr><td>Rabbit polyclonal anti-PRRC2C antibody</td><td>Abcam</td><td>Cat# ab117790; RRID: <a id="intref0080" href="https://antibodyregistry.org/AB_10903523" target="_blank">AB_10903523</a></td></tr><tr><td>Rabbit monoclonal anti-TAF15 antibody</td><td>Abcam</td><td>Cat# ab134916; RRID: <a id="intref0085" href="https://antibodyregistry.org/AB_2614922" target="_blank">AB_2614922</a></td></tr><tr><td>Goat polyclonal anti-TIA1 antibody</td><td>Santa Cruz</td><td>Cat# sc-1751; RRID: <a id="intref0090" href="https://antibodyregistry.org/AB_2201433" target="_blank">AB_2201433</a></td></tr><tr><td>Mouse monoclonal anti-TIAR (6/TIAR) antibody</td><td>BD Biosciences</td><td>Cat# 610352; RRID: <a id="intref0095" href="https://antibodyregistry.org/AB_397742" target="_blank">AB_397742</a></td></tr><tr><td>Rabbit monoclonal anti-TRIM25 antibody</td><td>Abcam</td><td>Cat# ab167154; RRID: <a id="intref0100" href="https://antibodyregistry.org/AB_2721902" target="_blank">AB_2721902</a></td></tr><tr><td>Rabbit monoclonal anti-TRIM56 antibody</td><td>Abcam</td><td>Cat# ab154862;</td></tr><tr><td>Rabbit polyclonal anti-UBAP2 antibody</td><td>Bethyl Laboratories</td><td>Cat# A304-627A; RRID: <a id="intref0105" href="https://antibodyregistry.org/AB_2620822" target="_blank">AB_2620822</a></td></tr><tr><td>Rabbit polyclonal anti-UBAP2L antibody</td><td>Abcam</td><td>Cat# ab138309;</td></tr><tr><td>Rabbit polyclonal anti-USP10 antibody</td><td>Proteintech</td><td>Cat# 19374-1-AP; RRID: <a id="intref0110" href="https://antibodyregistry.org/AB_10858617" target="_blank">AB_10858617</a></td></tr><tr><td>Rabbit polyclonal anti-YB1 antibody</td><td>Cell Signaling Technology</td><td>Cat# 4202S; RRID: <a id="interref275" href="https://antibodyregistry.org/AB_1950384" target="_blank">AB_1950384</a></td></tr><tr><td>Rabbit polyclonal anti-YTHDF1 antibody</td><td>Proteintech</td><td>Cat# 17479-1-AP; RRID: <a id="intref0115" href="https://antibodyregistry.org/AB_2217473" target="_blank">AB_2217473</a></td></tr><tr><td>Rabbit polyclonal anti-YTHDF2 antibody</td><td>Proteintech</td><td>Cat# 24744-1-AP; RRID: <a id="interref280" href="https://antibodyregistry.org/AB_2687435" target="_blank">AB_2687435</a></td></tr><tr><td>Mouse monoclonal anti-YTHDF3 antibody</td><td>Santa Cruz</td><td>Cat# sc-377119; RRID: <a id="intref0120" href="https://antibodyregistry.org/AB_2687436" target="_blank">AB_2687436</a></td></tr><tr><td>Goat polyclonal anti-eIF3η (N-20)</td><td>Santa Cruz</td><td>Cat# sc-16377; RRID: <a id="intref0125" href="https://antibodyregistry.org/AB_671941" target="_blank">AB_671941</a></td></tr><tr><td>IRDye 680LT Streptavidin</td><td>LI-COR</td><td>P/N 926-68031</td></tr><tr><td>IRDye 800CW Goat anti-Rabbit IgG (H&nbsp;+ L)</td><td>LI-COR</td><td>Cat# 926-32211; RRID: <a id="interref285" href="https://antibodyregistry.org/AB_621843" target="_blank">AB_621843</a></td></tr><tr><td>IRDye 680RD Goat anti-Rabbit IgG (H&nbsp;+ L)</td><td>LI-COR</td><td>Cat# 926-68071; RRID: <a id="interref290" href="https://antibodyregistry.org/AB_10956166" target="_blank">AB_10956166</a></td></tr><tr><td>IRDye 800CW Goat anti-Mouse IgG (H&nbsp;+ L)</td><td>LI-COR</td><td>Cat# 926-32210; RRID: <a id="interref295" href="https://antibodyregistry.org/AB_621842" target="_blank">AB_621842</a></td></tr><tr><td>IRDye 680RD Goat anti-Mouse IgG (H&nbsp;+ L)</td><td>LI-COR</td><td>Cat# 926-68070; RRID: <a id="interref300" href="https://antibodyregistry.org/AB_10956588" target="_blank">AB_10956588</a></td></tr><tr><td>Donkey anti-Rabbit IgG (H+L) Secondary Antibody, Alexa Fluor 555</td><td>Thermo Fisher Scientific</td><td>Cat# A-31572; RRID: <a id="intref0130" href="https://antibodyregistry.org/AB_162543" target="_blank">AB_162543</a></td></tr><tr><td>Donkey anti-Goat IgG (H+L) Secondary Antibody, Alexa Fluor 555</td><td>Thermo Fisher Scientific</td><td>Cat# A-21432; RRID: <a id="intref0135" href="https://antibodyregistry.org/AB_2535853" target="_blank">AB_2535853</a></td></tr><tr><td>Donkey anti-Goat IgG (H+L) Secondary Antibody, Alexa Fluor 647</td><td>Thermo Fisher Scientific</td><td>Cat# A-21447; RRID: <a id="intref0140" href="https://antibodyregistry.org/AB_141844" target="_blank">AB_141844</a></td></tr><tr><td>Donkey anti-Mouse IgG (H+L) Secondary Antibody, Alexa Fluor 488</td><td>Thermo Fisher Scientific</td><td>Cat# A-21202; RRID: <a id="intref0145" href="https://antibodyregistry.org/AB_141607" target="_blank">AB_141607</a></td></tr><tr><td>Goat anti-Rabbit IgG (H+L) Secondary Antibody, Alexa Fluor 488</td><td>Thermo Fisher Scientific</td><td>Cat# A-11008; RRID: <a id="intref0150" href="https://antibodyregistry.org/AB_143165" target="_blank">AB_143165</a></td></tr><tr><td>Goat anti-Mouse IgG (H+L) Secondary Antibody, Alexa Fluor 488</td><td>Thermo Fisher Scientific</td><td>Cat# A-11029; RRID: <a id="intref0155" href="https://antibodyregistry.org/AB_138404" target="_blank">AB_138404</a></td></tr><tr><td>Goat anti-Rabbit IgG (H+L) Secondary Antibody, Alexa Fluor 555</td><td>Thermo Fisher Scientific</td><td>Cat# A-21428; RRID: <a id="intref0160" href="https://antibodyregistry.org/AB_141784" target="_blank">AB_141784</a></td></tr><tr class="rowsep"><td>Goat anti-Mouse IgG (H+L) Secondary Antibody, Alexa Fluor 555</td><td>Thermo Fisher Scientific</td><td>Cat# A-21422; RRID: <a id="intref0165" href="https://antibodyregistry.org/AB_141822" target="_blank">AB_141822</a></td></tr><tr class="rowsep"><td colspan="3"><b>Bacterial and Virus Strains</b></td></tr><tr><td>Rosetta 2(DE3) Competent Cells</td><td>Millipore</td><td>Cat# 71400-4</td></tr><tr><td>HI-Control BL21(DE3) Chemically Competent Cells (SOLOs)</td><td>Lucigen</td><td>Cat# 60435-1</td></tr><tr><td>HI-Control 10G Chemically Competent Cells (SOLOs)</td><td>Lucigen</td><td>Cat# 60110-1</td></tr><tr class="rowsep"><td>One Shot TOP10 Chemically Competent <i>E.&nbsp;coli</i></td><td>Thermo Fisher Scientific</td><td>Cat# C404003</td></tr><tr class="rowsep"><td colspan="3"><b>Chemicals, Peptides, and Recombinant Proteins</b></td></tr><tr><td>Lipofectamine 2000</td><td>Invitrogen</td><td>Cat# 1168019</td></tr><tr><td>Fugene</td><td>Promega</td><td>Cat# E2691</td></tr><tr><td>Dharmafect Duo</td><td>Dharmacon</td><td>Cat# T-2010</td></tr><tr><td>Protease inhibitor</td><td>Roche</td><td>Cat# 11697498001</td></tr><tr><td>OligodT25 dynabeads</td><td>Thermo Fisher Scientific</td><td>Cat# 61005</td></tr><tr><td>GFP-Trap_MA beads</td><td>ChromoTek</td><td>Cat# gtma-20</td></tr><tr><td>NuPAGE LDS sample buffer (4X)</td><td>Thermo Fisher Scientific</td><td>Cat# NP0007</td></tr><tr><td>20% paraformaldehyde</td><td>Electron Microscopy Science</td><td>Cat# 15713-S</td></tr><tr><td>ProLong Gold Antifade Mountant with DAPI</td><td>Thermo Fisher Scientific</td><td>Cat# P36931</td></tr><tr><td>RIPA Lysis and Extraction Buffer</td><td>Thermo Fisher Scientific</td><td>Cat# 89900</td></tr><tr><td>SimplyBlue SafeStain</td><td>Thermo Fisher Scientific</td><td>Cat# LC6065</td></tr><tr><td>SYBR Gold Nucleic Acid Gel Stain</td><td>Thermo Fisher Scientific</td><td>Cat# S11494</td></tr><tr><td>TRIzol Reagent</td><td>Thermo Fisher Scientific</td><td>Cat# 15596018</td></tr><tr><td>RNaseA</td><td>Thermo Fisher Scientific</td><td>Cat# EN0531</td></tr><tr><td>RNase I</td><td>Life Technologies</td><td>Cat# AM2294</td></tr><tr><td>RNasin</td><td>Promega</td><td>Cat# N2111</td></tr><tr><td>Turbo DNase</td><td>Thermo Fisher Scientific</td><td>Cat# AM2238</td></tr><tr><td>Alkaline Phosphatase, Calf Intestinal</td><td>Promega</td><td>Cat# M1821</td></tr><tr><td>Sodium Arsenite Solution</td><td>Sigma</td><td>Cat# 35000-1L-R</td></tr><tr><td>Ficoll400</td><td>Sigma</td><td>Cat# F2637</td></tr><tr><td>Ni-NTA agarose</td><td>GE</td><td>Cat# 17-5318-02</td></tr><tr><td>Glutathione Sepharose 4B</td><td>GE</td><td>Cat# 17075601</td></tr><tr><td>EZ view Red Protein A affinity gel</td><td>Millipore-Sigma</td><td>Cat# P6486</td></tr><tr><td>EZ view Red Protein G affinity gel</td><td>Millipore Sigma</td><td>Cat# E3403</td></tr><tr><td>EZ view Red Anti-Flag M2 affinity gel</td><td>Millipore Sigma</td><td>Cat# F2426</td></tr><tr><td>FLAG peptide</td><td>Sigma</td><td>Cat# F3290</td></tr><tr><td>Glutathione reduced</td><td>Sigma</td><td>Cat# G4251</td></tr><tr><td>Imidazole</td><td>Sigma</td><td>Cat# I2399</td></tr><tr><td>(Isopropyl β-D-1-thiogalactopyranoside) IPTG</td><td>Goldbio</td><td>Cat# 12481C100</td></tr><tr><td>Dithiothreitol (DTT)</td><td>Sigma</td><td>Cat# 43815</td></tr><tr><td>D/D compound</td><td>Takara</td><td>Cat# 635054</td></tr><tr><td>SMARTpool: ON-TARGETplus CAPRIN1 siRNA</td><td>Dharmacon</td><td>Cat# L-016057-00-0005</td></tr><tr><td>SMARTpool: ON-TARGETplus TIA1 siRNA</td><td>Dharmacon</td><td>Cat# L-013042-02-0005</td></tr><tr><td>tRNA</td><td>Sigma</td><td>Cat# 10109495001</td></tr><tr><td>polyA</td><td>Sigma</td><td>Cat# 10108626001</td></tr><tr><td>polyG</td><td>Sigma</td><td>Cat# P4404-5MG</td></tr><tr><td>polyC</td><td>Sigma</td><td>Cat# P4903-10MG</td></tr><tr><td>polyU</td><td>Sigma</td><td>Cat# P9528-10MG</td></tr><tr><td>Poly(I:C)HMW</td><td>Invivogen</td><td>Cat# tlrl-pic</td></tr><tr><td>Poly(I:C)LMW</td><td>Invivogen</td><td>Cat# tlrl-picw</td></tr><tr><td>Poly(A:U)</td><td>Invivogen</td><td>Cat# tlrl-pau</td></tr><tr><td>bis(sulfosuccinimidyl)suberate (BS3)</td><td>Thermo Fisher Scientific</td><td>Cat# 21580</td></tr><tr><td>Recombinant DDX19A</td><td>MyBioSource</td><td>Cat# MBS1371734</td></tr><tr><td>Biotin tyramide</td><td>Sigma</td><td>Cat# SML2135</td></tr><tr><td>H<sub>2</sub>O<sub>2</sub></td><td>Sigma</td><td>Cat# H1009</td></tr><tr><td>Sodium Ascorbate</td><td>Sigma</td><td>Cat# A7631</td></tr><tr><td>Trolox</td><td>Sigma</td><td>Cat# 238813</td></tr><tr><td>Dynabeads MyOne Streptavidin C1</td><td>Thermo Fisher Scientific</td><td>Cat# 65001</td></tr><tr><td>Biotin</td><td>Sigma</td><td>Cat# B4501</td></tr><tr><td>BSA</td><td>Sigma</td><td>Cat# A3294S</td></tr><tr class="rowsep"><td>ATP</td><td>Sigma</td><td>Cat# A2383</td></tr><tr class="rowsep"><td colspan="3"><b>Critical Commercial Assays</b></td></tr><tr><td>HiScribe high yield RNA synthesis kit</td><td>New England Biolab</td><td>Cat# E2040S</td></tr><tr><td>RNA 3′ end biotinylation kit</td><td>Thermo Fisher Scientific</td><td>Cat# 20160</td></tr><tr><td>Chemiluminescent Nucleic Acid Detection Module</td><td>Thermo Fisher Scientific</td><td>Cat# 89880</td></tr><tr><td>polyA Spin mRNA Isolation Kit</td><td>New England Biolabs</td><td>Cat# S1560S</td></tr><tr><td>NEBuilder HiFi DNA Assembly Master Mix</td><td>New England Biolabs</td><td>Cat# E2621S</td></tr><tr><td>Q5 Site-Directed Mutagenesis Kit</td><td>New England Biolabs</td><td>Cat# E0554S</td></tr><tr><td>Monarch RNA Cleanup Kit</td><td>New England Biolabs</td><td>Cat# T2040S</td></tr><tr><td>NEBNext Magnesium RNA Fragmentation Module</td><td>New England Biolabs</td><td>Cat# E6150S</td></tr><tr><td>SuperScript III First-Strand Synthesis System</td><td>Thermo Fisher Scientific</td><td>Cat# 18080051</td></tr><tr class="rowsep"><td>Magnetic mRNA Isolation Kit</td><td>New England Biolabs</td><td>Cat# S1550S</td></tr><tr class="rowsep"><td colspan="3"><b>Deposited Data</b></td></tr><tr class="rowsep"><td>G3BP1 RRM structural homology model</td><td><a id="crosref1165" href="#bib67" role="doc-biblioref" data-xml-rid="bib67">Wu et&nbsp;al., 2018</a></td><td>PDB: <a id="intref0170" href="http://firstglance.jmol.org/fg.htm?mol5HO4" target="_blank">5HO4</a></td></tr><tr class="rowsep"><td colspan="3"><b>Experimental Models: Cell Lines</b></td></tr><tr class="rowsep"><td>Human: U-2 OS</td><td>ATCC</td><td>Cat# HTB-96</td></tr><tr class="rowsep"><td colspan="3"><b>Oligonucleotides</b></td></tr><tr class="rowsep"><td>See <a id="crosref1170" href="#mmc2">Table S2</a> for gRNA sequences for SG gene KO</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr class="rowsep"><td colspan="3"><b>Recombinant DNA</b></td></tr><tr><td>Plasmid: EGFP-C3-G3BP1</td><td>This paper</td><td>N/A</td></tr><tr><td>Plasmid: EGFP-C3-G3BP2b</td><td>This paper</td><td>N/A</td></tr><tr><td>Plasmid: EGFP-C3-G3BP1 ΔNTF2L[1-142]</td><td>This paper</td><td>N/A</td></tr><tr><td>Plasmid: EGFP-C3-G3BP1ΔIDR1/2[143-334]</td><td>This paper</td><td>N/A</td></tr><tr><td>Plasmid: EGFP-C3-G3BP1ΔRBD[335-466]</td><td>This paper</td><td>N/A</td></tr><tr><td>Plasmid: EGFP-C3-GST-G3BP1[143-466]</td><td>This paper</td><td>N/A</td></tr><tr><td>Plasmid: EGFP-C3-FKBP<sup>F36M</sup>-G3BP1[143-466]</td><td>This paper</td><td>N/A</td></tr><tr><td>Plasmid: EGFP-C3-G3BP1 F380/382D</td><td>This paper</td><td>N/A</td></tr><tr><td>Plasmid: EGFP-C3-G3BP1-2xIDR2</td><td>This paper</td><td>N/A</td></tr><tr><td>Plasmid: EGFP-C3-G3BP1 S149A</td><td>This paper</td><td>N/A</td></tr><tr><td>Plasmid: EGFP-C3-G3BP1 S149D</td><td>This paper</td><td>N/A</td></tr><tr><td>Plasmid: CMV-HA-G3BP1</td><td>This paper</td><td>N/A</td></tr><tr><td>Plasmid: EGFP-C3-G3BP1[1-334]-HnRNPA1-RRM[12-181]</td><td>This paper</td><td>N/A</td></tr><tr><td>Plasmid: EGFP-C3-G3BP1[1-334]-HnRNPA2B1-RRM[7-179]</td><td>This paper</td><td>N/A</td></tr><tr><td>Plasmid: EGFP-C3-G3BP1[1-334]-HnRNPD-RRM[99-257]</td><td>This paper</td><td>N/A</td></tr><tr><td>Plasmid: EGFP-C3-G3BP1[1-334]-DAZAP1-RRM[10-190]</td><td>This paper</td><td>N/A</td></tr><tr><td>Plasmid: EGFP-C3-G3BP1[1-334]-SRSF4-RRM[2-177]</td><td>This paper</td><td>N/A</td></tr><tr><td>Plasmid: EGFP-C3-G3BP1[1-334]-TDP43-RRM[105-261]</td><td>This paper</td><td>N/A</td></tr><tr><td>Plasmid: EGFP-C3-G3BP1[1-334]-RBMS2-RRM[56-220]</td><td>This paper</td><td>N/A</td></tr><tr><td>Plasmid: EGFP-C3-G3BP1[1-334]-HnRNPH2-RRM[11-188]</td><td>This paper</td><td>N/A</td></tr><tr><td>Plasmid: EGFP-C3-G3BP1[1-334]-RBM22-ZF[159-185]</td><td>This paper</td><td>N/A</td></tr><tr><td>Plasmid: EGFP-C3-G3BP1[1-334]-ZC3H11A-ZF[2-110]</td><td>This paper</td><td>N/A</td></tr><tr><td>Plasmid: EGFP-C3-G3BP1[1-334]-NOVA-KH[51-120]</td><td>This paper</td><td>N/A</td></tr><tr><td>Plasmid: EGFP-C3-G3BP1[1-334]-QKI-KH[83-205]</td><td>This paper</td><td>N/A</td></tr><tr><td>Plasmid: EGFP-C3-G3BP1[1-334]-QKI-2KH[83-205]</td><td>This paper</td><td>N/A</td></tr><tr><td>Plasmid: EGFP-C3-G3BP1[1-334]-QKI-3KH[83-205]</td><td>This paper</td><td>N/A</td></tr><tr><td>Plasmid: EGFP-C3-G3BP1[1-334]-RBM22-2ZF[159-185]</td><td>This paper</td><td>N/A</td></tr><tr><td>Plasmid: EGFP-C3-G3BP1[1-334]-RBM22-3ZF[159-185]</td><td>This paper</td><td>N/A</td></tr><tr><td>Plasmid: EGFP-C3-G3BP1[1-334]-YTHDF1-YTH[390-523]</td><td>This paper</td><td>N/A</td></tr><tr><td>Plasmid: EGFP-C3-G3BP1[1-334]-YTHDF2-YTH[411-544]</td><td>This paper</td><td>N/A</td></tr><tr><td>Plasmid: EGFP-C3-G3BP1[1-334]-YTHDF3-YTH[417-550]</td><td>This paper</td><td>N/A</td></tr><tr><td>Plasmid: EGFP-C3-G3BP1[1-334]-HnRNPA1-RRM[12-181] F57/59D</td><td>This paper</td><td>N/A</td></tr><tr><td>Plasmid: EGFP-C3-G3BP1[1-334]-HnRNPA1-RRM[12-181] F148/150D</td><td>This paper</td><td>N/A</td></tr><tr><td>Plasmid: EGFP-C3-G3BP1[1-334]-HnRNPA1-RRM[12-181] F57/59/148/150D</td><td>This paper</td><td>N/A</td></tr><tr><td>Plasmid: EGFP-C3-GST-2xAsh1-IDR-HnRNPA1-RRM[12-181]</td><td>This paper</td><td>N/A</td></tr><tr><td>Plasmid: EGFP-C3-G3BP1 EQ28</td><td>This paper</td><td>N/A</td></tr><tr><td>Plasmid: EGFP-C3-G3BP1-2xRBD</td><td>This paper</td><td>N/A</td></tr><tr><td>Plasmid: EGFP-C3-G3BP1ΔIDR1[143-226]</td><td>This paper</td><td>N/A</td></tr><tr><td>Plasmid: EGFP-C3-G3BP1ΔIDR2[227-334]</td><td>This paper</td><td>N/A</td></tr><tr><td>Plasmid: EGFP-C3-G3BP1ΔIDR2[227-334]-Acidic-scrambled-1</td><td>This paper</td><td>N/A</td></tr><tr><td>Plasmid: EGFP-C3-G3BP1ΔIDR2[227-334]-Acidic-scrambled-2</td><td>This paper</td><td>N/A</td></tr><tr><td>Plasmid: EGFP-C3-G3BP1ΔIDR2[227-334]-Acidic-scrambled-3</td><td>This paper</td><td>N/A</td></tr><tr><td>Plasmid: EGFP-C3-G3BP1ΔIDR2[227-334]-EQ28</td><td>This paper</td><td>N/A</td></tr><tr><td>Plasmid: EGFP-C3-G3BP1ΔIDR1/2[143-334]+Ash1-IDR</td><td>This paper</td><td>N/A</td></tr><tr><td>Plasmid: EGFP-C3-G3BP1ΔIDR1[143-226]+Ash1-IDR</td><td>This paper</td><td>N/A</td></tr><tr><td>Plasmid: EGFP-C3-G3BP1ΔIDR2[227-334]+Ash1-IDR</td><td>This paper</td><td>N/A</td></tr><tr><td>Plasmid: EGFP-C3-G3BP1ΔIDR[143-334]+2xAsh1-IDR</td><td>This paper</td><td>N/A</td></tr><tr><td>Plasmid: EGFP-C3-G3BP1-IDR1/2(P-S)</td><td>This paper</td><td>N/A</td></tr><tr><td>Plasmid: EGFP-C3-G3BP1-IDR2(RKH-A17)</td><td>This paper</td><td>N/A</td></tr><tr><td>Plasmid: mCherry-G3BP1</td><td>This paper</td><td>N/A</td></tr><tr><td>Plasmid: EGFP-Caprin-1</td><td>This paper</td><td>N/A</td></tr><tr><td>Plasmid: CMV-Flag-G3BP1</td><td>This paper</td><td>N/A</td></tr><tr><td>Plasmid: CMV-HA-G3BP1</td><td>This paper</td><td>N/A</td></tr><tr><td>Plasmid: APEX2-NES-G3BP1</td><td>This paper</td><td>N/A</td></tr><tr><td>Plasmid: APEX2-NES-G3BP1ΔIDR[143-334]+2xAsh1-IDR</td><td>This paper</td><td>N/A</td></tr><tr><td>Plasmid: pGEX-2T-TEV-G3BP1</td><td>This paper</td><td>N/A</td></tr><tr><td>Plasmid: pGEX-2T-TEV-G3BP1 F33W</td><td>This paper</td><td>N/A</td></tr><tr><td>Plasmid: pGEX-2T-TEV-G3BP1ΔNTF2L[1-142]</td><td>This paper</td><td>N/A</td></tr><tr><td>Plasmid: pGEX-2T-TEV-G3BP1-NTF2L[1-142]</td><td>This paper</td><td>N/A</td></tr><tr><td>Plasmid: pGEX-2T-TEV-G3BP1-NTF2L[1-142]</td><td>This paper</td><td>N/A</td></tr><tr><td>Plasmid: pGEX-2T-TEV-G3BP1ΔIDR1/2[143-334]</td><td>This paper</td><td>N/A</td></tr><tr><td>Plasmid: pGEX-2T-TEV-G3BP1-IDR1/2[143-334]</td><td>This paper</td><td>N/A</td></tr><tr><td>Plasmid: pGEX-2T-TEV-G3BP1ΔRBD[335-466]</td><td>This paper</td><td>N/A</td></tr><tr><td>Plasmid: pGEX-2T-thrombin-G3BP1-ΔRRM[335-410]</td><td>This paper</td><td>N/A</td></tr><tr><td>Plasmid: pGEX-2T-thrombin-G3BP1-ΔRGG[411-466]</td><td>This paper</td><td>N/A</td></tr><tr><td>Plasmid: pGEX-2T-thrombin-G3BP1-RBD[335-466]</td><td>This paper</td><td>N/A</td></tr><tr><td>Plasmid: pGEX-2T-thrombin-FKBP<sup>F36M</sup>-G3BP1[143-466]</td><td>This paper</td><td>N/A</td></tr><tr><td>Plasmid: pGEX-2T-TEV-G3BP1-TEV-HIS S149A</td><td>This paper</td><td>N/A</td></tr><tr><td>Plasmid: pGEX-2T-TEV-G3BP1-TEV-HIS S149E</td><td>This paper</td><td>N/A</td></tr><tr><td>Plasmid: pGEX-2T-TEV-G3BP1 EQ28</td><td>This paper</td><td>N/A</td></tr><tr><td>Plasmid: pGEX-2T-thrombin-G3BP1 NTF2L-HA</td><td>This paper</td><td>N/A</td></tr><tr><td>Plasmid: pGEX-2T-thrombin-G3BP1 NTF2L-IDR1-HA</td><td>This paper</td><td>N/A</td></tr><tr><td>Plasmid: pGEX-2T-thrombin-G3BP1 NTF2L-IDR1/2-HA</td><td>This paper</td><td>N/A</td></tr><tr><td>Plasmid: pGEX-2T-TEV-G3BP1 RBD</td><td>This paper</td><td>N/A</td></tr><tr><td>Plasmid: pGEX-2T-TEV-G3BP1 RBD-RA5</td><td>This paper</td><td>N/A</td></tr><tr><td>Plasmid: pGEX-2T-thrombin-G3BP1 IDR1-S149E-HA</td><td>This paper</td><td>N/A</td></tr><tr><td>Plasmid: pGEX-2T-TEV-G3BP1 IDR1-EQ28-HA</td><td>This paper</td><td>N/A</td></tr><tr><td>Plasmid: HIS-SUMO-Caprin1</td><td>This paper</td><td>N/A</td></tr><tr><td>Plasmid: HIS-SUMO-Caprin1 F372A</td><td>This paper</td><td>N/A</td></tr><tr><td>Plasmid: HIS-SUMO-TIA1</td><td><a id="crosref1175" href="#bib33" role="doc-biblioref" data-xml-rid="bib33">Mackenzie et&nbsp;al., 2017</a></td><td>N/A</td></tr><tr><td>Plasmid: pSpCas9(BB)-2A-Puro (PX459) V2.0</td><td><a id="crosref1180" href="#bib49" role="doc-biblioref" data-xml-rid="bib49">Ran et&nbsp;al., 2013</a></td><td>Addgene Cat# 62988</td></tr><tr><td>Plasmid: pRK793</td><td><a id="crosref1185" href="#bib25" role="doc-biblioref" data-xml-rid="bib25">Kapust et&nbsp;al., 2001</a></td><td>Addgene Cat# <a id="intref0175" href="https://www.ebi.ac.uk/ontology-lookup/?termId=MI:8827" target="_blank">8827</a></td></tr><tr><td>pLVX-rHom-1</td><td>Takara</td><td>Cat# 635062</td></tr><tr class="rowsep"><td>APEX2-NES</td><td><a id="crosref0850" href="#bib77" role="doc-biblioref" data-xml-rid="bib77">Lam et&nbsp;al., 2015</a></td><td>Addgene Cat# 49386</td></tr><tr class="rowsep"><td colspan="3"><b>Software and Algorithms</b></td></tr><tr><td>ImageJ</td><td>NIH</td><td><a id="intref0180" href="https://imagej.nih.gov/ij/" target="_blank">https://imagej.nih.gov/ij/</a></td></tr><tr><td>Image Studio</td><td>LI-COR</td><td><a id="interref0305" href="https://www.licor.com/bio/products/software/image_studio_lite/" target="_blank">https://www.licor.com/bio/products/software/image_studio_lite/</a></td></tr><tr><td>GraphPad Prism Software</td><td>GraphPad</td><td><a id="interref0310" href="https://www.graphpad.com/scientificsoftware/prism/" target="_blank">https://www.graphpad.com/scientificsoftware/prism/</a></td></tr><tr><td>Igor Pro 6</td><td>Wavemetrics</td><td><a id="intref0195" href="https://www.wavemetrics.com/" target="_blank">https://www.wavemetrics.com/</a></td></tr><tr><td>SlideBook 6 software</td><td>Intelligent Imaging Innovations</td><td><a id="intref0200" href="https://www.intelligent-imaging.com/slidebook" target="_blank">https://www.intelligent-imaging.com/slidebook</a></td></tr><tr><td>CellProfiler</td><td>Broad Institute</td><td><a id="intref0205" href="https://cellprofiler.org/" target="_blank">https://cellprofiler.org/</a></td></tr><tr><td>R–igraph version 1.2.4.1</td><td>igraph</td><td><a id="intref0210" href="https://rdrr.io/cran/igraph/" target="_blank">https://rdrr.io/cran/igraph/</a></td></tr><tr><td>Cytoscape version 3.7.1</td><td>Cytoscape Consortium</td><td><a id="interref0315" href="https://cytoscape.org/release_notes_3_7_1.html" target="_blank">https://cytoscape.org/release_notes_3_7_1.html</a></td></tr></tbody></table></div><figcaption><div class="core-table-tools"><ul><li><a class="core-table-show-action" href="/action/showFullTableHTML?isHtml=true&amp;tableId=undtbl1&amp;pii=S0092-8674%2820%2930339-1" target="_blank">Open table in a new tab</a></li></ul></div></figcaption></figure></div></div></section><section id="sec-4-2"><h3>Lead Contact and Materials Availability</h3><div id="p0290" role="paragraph">Further information and requests for resources and reagents should be directed to and will be fulfilled by the Lead Contact, J. Paul Taylor (<a id="intref0220" href="mailto:jpaul.taylor@stjude.org">jpaul.taylor@stjude.org</a>). All reagents generated in this study will be made available on request but may require a payment and/or a completed Materials Transfer Agreement if there is potential for commercial application.</div></section><section id="sec-4-3"><h3>Experimental Model and Subject Details</h3><section id="sec-4-3-1"><h4>Cell Culture</h4><div id="p0295" role="paragraph">HEK293T and U2OS cells (both of female origin) were cultured in Dulbecco’s modified Eagle’s medium (HyClone) supplemented with 10% fetal bovine serum and 1% antibiotics, and maintained at 37°C in a humified incubator with 95% air and 5% CO<sub>2</sub>. Cells were authenticated by short tandem repeat (STR) profiling. For bacterial cell culture, TOP10 and Rosetta 2(DE3) chemically competent bacteria strains were cultured in LB media containing appropriate antibiotic as described below, at 37°C in a shaker incubator (200&nbsp;rpm).</div></section></section><section id="sec-4-4"><h3>Method Details</h3><section id="sec-4-4-1"><h4>Cell Culture and Transfection</h4><div id="p0300" role="paragraph">U2OS cells were purchased from ATCC (HTB-96). U2OS cells were cultured in Dulbecco’s modified Eagle’s medium (HyClone) supplemented with 10% fetal bovine serum (HyClone SH30071.03 and SH30396.03) and maintained at 37°C in a humidified incubator with 5% CO<sub>2</sub>. FuGENE 6 (Promega, E2691) or Lipofectamine 2000 (Thermo Fisher Scientific, 11668019) were used for transient transfections per the manufacturer’s instructions. <i>G3BP1/2</i> dKO cells have been previously described (<a id="crosref1190" href="#bib73" role="doc-biblioref" data-xml-rid="bib73">Zhang et&nbsp;al., 2018</a>). U2OS cells stably expressing G3BP1-GFP have been previously described (<a id="crosref1195" href="#bib13" role="doc-biblioref" data-xml-rid="bib13">Figley et&nbsp;al., 2014</a>).</div></section><section id="sec-4-4-2"><h4>Constructs</h4><div id="p0305" role="paragraph">DNA fragments encoding human G3BP1 and G3BP2 were PCR-amplified from a U2OS cDNA library. The cDNA library was created by SuperScript III First-Strand Synthesis System (Thermo, 18080051). The FKBP<sup>F36M</sup> fragment was PCR-amplified from pLVX-rHom-1 (Takara, 635062). DNA fragments encoding G3BP1 or G3BP2 were inserted into the pEGFP-C3 backbone using NEBuilder HiFi DNA Assembly Master Mix kit (NEB, E2621). Deletion mutants of EGFP-G3BP1 and EGFP-G3BP1 S149E or S149A were created by the Q5 Site-Directed Mutagenesis kit (NEB, E0554S). To create EGFP-G3BP1 swap mutants with different RNA-binding domains, RNA-binding domain fragments encoding hnRNPA1, hnRNPA2B1, hnRNPD, DAZAP1, SRSF4, TDP-43, RBMS2, hnRNPH2, RBM22, ZC3H11A, NOVA, QKI, YTHDF1, YTHDF2, YTHDF3, ADAR1, or STAU1 were amplified from a cDNA library and assembled into pEGFP-G3BP using NEBuilder HiFi DNA Assembly Master Mix. EGFP-G3BP1 swap mutants with different dimerization domains (GST or FKBP<sup>F36M</sup>) were obtained by overlapping extension-PCR methodology. Briefly, overlapping fragments encoding GST or FKBP<sup>F36M</sup> and G3BP1 ΔNTF2L were amplified by PCR, mixed in an equimolar ratio, and used as a template to generate the fusion proteins. The resulting PCR product was then cloned into pGEX-2T or pEGFP-C3 plasmids using BamHI/EcoRI and BglII/ApaI restriction enzymes, respectively. The EQ28 coding sequence, IDR1 scrambled sequences, and RKH-A17 IDR2 sequence were synthesized as dsDNA gene blocks through IDT and were further assembled into a pEGFP-G3BP1 construct. The Ash1 coding sequence was amplified from a His-Ash1 plasmid reported previously (<a id="crosref1200" href="#bib35" role="doc-biblioref" data-xml-rid="bib35">Martin et&nbsp;al., 2016a</a>) and further assembled into a pEGFP-G3BP1 plasmid. For protein expression, G3BP1-coding DNA was cut by BamHI and EcoRI and ligated into the pGEX-2T plasmid in which the thrombin site had been replaced with a TEV site using the Q5 Site-Directed Mutagenesis kit (NEB, E0554S). G3BP1 deletion mutants and G3BP1 S149E or S149A were further created through the Q5 Site-Directed Mutagenesis kit (NEB, E0554S). To create tandem KH, ZnF, and tandem G3BP1-RBD constructs, XbaI-SpeI mixed sites and BioBrick Assembly method was used. All constructs were confirmed by sequencing.</div></section><section id="sec-4-4-3"><h4>Genome-Wide RNAi Screen</h4><div id="p0310" role="paragraph">The human kinome collection (Dharmacon) was reverse-transfected into U2OS cells stably expressing G3BP1-GFP using Lipofectamine RNAiMAX (Life Technologies) at a final concentration of 25&nbsp;nM. We used reverse transfection, an approach where cells are plated onto a mixture of lipid/siRNAs, because this approach permits high throughput (<a id="crosref1510" href="#bib80" role="doc-biblioref" data-xml-rid="bib80">Ong et&nbsp;al., 2014</a>). In brief,&nbsp;cells were incubated at 43°C for 2&nbsp;h at 48&nbsp;h post-transfection. The cells were then fixed in 4% paraformaldehyde (EMS), stained for DAPI, and imaged with an IN Cell 6000 imaging platform (GE Healthcare). Intensity, area, count of nuclei, and GFP-positive SGs were measured using the IN Cell Developer program. To identify&nbsp;hits we used&nbsp;SSMD (strictly standardized mean difference)-based algorithm on GUItars (a GUI tool for analysis of high-throughput RNA interference screening) as described by (<a id="crosref1205" href="#bib16" role="doc-biblioref" data-xml-rid="bib16">Goktug et&nbsp;al., 2012</a>). SSMD is defined as the ratio of the mean to the SD of the random variable representing the difference between two independent populations.&nbsp;Greater magnitude in the SSMD between two populations indicates greater separation in features of these populations. siRNAs&nbsp;exhibiting&nbsp;an SSMD&nbsp;≤&nbsp;−1.5&nbsp;may be considered a hit. However, since high-throughput assays must be robust to outliers, a more robust version of SSMD (robust SSMD or SSMD<sup>∗</sup>) was obtained by replacing the mean with the median and SD with median absolute deviation (<a id="crosref1210" href="#bib70" role="doc-biblioref" data-xml-rid="bib70">Zhang, 2011</a>). The higher the SSMD score, the higher probability that the siRNA is a true positive (<a id="crosrefs0085" href="#bib69" role="doc-biblioref" data-xml-rid="bib69 bib72">Zhang, 2007; Zhang et&nbsp;al., 2006</a>). Putative positive regulators of SG assembly from the arrayed siRNA screen were identified as having a robust SSMD value of ≤ −1.5. The primary screen was performed in a single replicate, with each well containing a pool of 4 siRNAs that target the same gene. The hits with SSMD ≤ −1.5 were assayed in a confirmatory screen where the same putative hits were screened in quadruplicate. The hits from the confirmatory screen (SSMD ≤ −1.5) were further tested in a deconvolution screen where we tested the individual siRNAs for the targeted gene in quadruplicate.</div></section><section id="sec-4-4-4"><h4>Gene Ontology</h4><div id="p0315" role="paragraph">Gene-annotation enrichment and functional annotation clustering of the hits acquired from the genome-wide RNAi screen were analyzed using the Database for Annotation, Visualization and Integrated Discovery (DAVID) Bioinformatics Resources (<a id="crosref1215" href="#bib19" role="doc-biblioref" data-xml-rid="bib19">Huang da et&nbsp;al., 2009</a>).</div></section><section id="sec-4-4-5"><h4>Protein-Protein Interaction Network</h4><div id="p0320" role="paragraph">A composite protein-protein interaction (PPI) network was built by combining STRING (v10) (<a id="crosref1220" href="#bib55" role="doc-biblioref" data-xml-rid="bib55">Szklarczyk et&nbsp;al., 2015</a>), BioPlex (<a id="crosref1225" href="#bib21" role="doc-biblioref" data-xml-rid="bib21">Huttlin et&nbsp;al., 2015</a>), and InWeb (<a id="crosref1230" href="#bib18" role="doc-biblioref" data-xml-rid="bib18">Hopkins et&nbsp;al., 2017</a>), as well as three unbiased SG interactomes (<a id="crosrefs0090" href="#bib23" role="doc-biblioref" data-xml-rid="bib23 bib34 bib68">Jain et&nbsp;al., 2016; Markmiller et&nbsp;al., 2018; Youn et&nbsp;al., 2018</a>). Only physical protein-protein interactions were considered, and the interactome was filtered by edge confidence scores, with thresholds determined by best fitting the scale-free network property (<a id="crosrefs0095" href="#bib3" role="doc-biblioref" data-xml-rid="bib3 bib71">Barabasi and Albert, 1999; Zhang and Horvath, 2005</a>).</div></section><section id="sec-4-4-6"><h4>Network Enrichment Analysis</h4><div id="p0325" role="paragraph">The significance of the generated subnetworks (e.g., RNAi and SG subnetworks) was evaluated by empirically estimating the probability of observing a denser network by random sampling 1,000 subnetworks from the entire interactome (or any background network to be tested). Network density (<a id="crosref1235" href="#bib64" role="doc-biblioref" data-xml-rid="bib64">Wasserman and Faust, 1994</a>) was calculated using R–igraph version 1.2.4.1.</div></section><section id="sec-4-4-7"><h4>Network Centrality Analysis</h4><div id="p0330" role="paragraph">The network of 36 genes that was co-identified by the genetic screen and proteomics data was extracted from our curated PPI network. We used betweenness as the metric of node centrality and calculated the betweenness for each node in the 36-gene network using R–igraph version 1.2.4.1. Cytoscape version 3.7.1 (<a id="crosref1240" href="#bib54" role="doc-biblioref" data-xml-rid="bib54">Shannon et&nbsp;al., 2003</a>) was used for the visualization of this network in <a id="crosref1245" href="#fig1">Figure&nbsp;1</a>D.</div></section><section id="sec-4-4-8"><h4>CRISPR-Cas9-Mediated Knockout Cells</h4><div id="p0335" role="paragraph">gRNA for each gene was designed through <a id="intref0225" href="http://zlab.bio/guide-design-resources" target="_blank">http://zlab.bio/guide-design-resources</a> and synthesized DNA oligos were ligated into BbsI- (NEB) digested px459 vectors (Addgene, 62988) (<a id="crosref1250" href="#bib49" role="doc-biblioref" data-xml-rid="bib49">Ran et&nbsp;al., 2013</a>). Targeting vectors were verified by Sanger sequencing. WT U2OS cells were transiently transfected with gRNA vector for 1-3&nbsp;days, followed by the addition of 2&nbsp;μg/ml puromycin for 3&nbsp;days. Knockout clones were obtained by single-cell dilution in 96-well plates and successful knockout was verified by antibody staining. For certain gene knockouts, a mixture of puromycin-selected cells were stained directly without single-cell isolation. We note that <i>TIA1/TIAR</i> KO cells still contain one WT allele of <i>TIAR</i>. <i>TIA1</i>/<i>TIAR</i> dKO cells were lethal, similar to the situation in HEK293 cells (<a id="crosref1255" href="#bib38" role="doc-biblioref" data-xml-rid="bib38">Meyer et&nbsp;al., 2018</a>).</div></section><section id="sec-4-4-9"><h4>CRISPR-Cas9-Mediated Knockin Cells</h4><div id="p0340" role="paragraph">Genetically modified U2OS cells were generated using CRISPR-Cas9. Briefly, 400,000&nbsp;U2OS cells were transiently co-transfected with 200&nbsp;ng gRNA expression plasmid (cloned into Addgene, 43860), 500&nbsp;ng Cas9 expression plasmid (Addgene, 43945), and, if required, 500&nbsp;ng donor plasmid via nucleofection (Lonza, 4D-Nucleofector X-unit) using solution P3, program CM-104 in small cuvettes according to the manufacturer’s protocol. For the G3BP1-tdTomato tagged cell line, cells were single-cell sorted by FACS for tdTomato-positive cells into 96-well plates. After sorting, cells were clonally expanded and screened for the desired modification using targeted next generation sequencing and/or PCR-based assays.</div></section><section id="sec-4-4-10"><h4>Immunofluorescence and Microscopy</h4><div id="p0345" role="paragraph">Cells were grown in 4-well or 8-well chamber slides (Millipore). Transfected cells were fixed with 4% paraformaldehyde (Electron Microscopy Science) in PBS for 10&nbsp;min, permeabilized with 0.2% Triton X-100 in PBS for 10&nbsp;min, and then blocked with 1% BSA for 1 h, with all steps performed at room temperature. Samples were further incubated with primary antibodies in blocking buffer overnight at 4°C, then washed 3 times with PBST (0.1% Tween) and incubated with secondary antibody for 1&nbsp;h at room temperature. Primary antibodies used in this study are listed in the <a id="crosref1260" href="#sec-4-1">Key Resources Table</a>. Host-specific Alexa Fluor 488/555/647 (Thermo Fisher) secondary antibodies were used for visualization. For microscopic imaging, slides were mounted with ProLong Gold Antifade reagent with DAPI (Invitrogen). Images were captured using a Leica TCS SP8 STED 3X confocal microscope with a 63x oil objective.</div></section><section id="sec-4-4-11"><h4>G3BP1 Intracellular Phase Diagrams</h4><div id="p0350" role="paragraph">U2OS <i>G3BP1/2</i> dKO cells were transfected with GFP-G3BP1 WT along with an siRNA pool targeting <i>caprin-1</i> or <i>TIA1</i>, or were transfected with either GFP-G3BP1 WT or mutants (GST-G3BP1, FKBP<sup>F36M</sup>-G3BP1, S149A, S149E, or EQ28) or were co-transfected with GFP-Caprin-1 and mCherry-G3BP1. After 48&nbsp;h of expression, cells were placed in fresh medium for 1&nbsp;h and stressed with 100&nbsp;μM sodium arsenite for 30&nbsp;min at 37°C. After treatment, cells were fixed with 4% PFA and stained for PABP. Images were captured using Leica DMi8 microscope with a 20x objective. Phase diagrams were constructed by measuring GFP fluorescence intensity in each cell and assessing the presence of SGs using PABP as a marker, using Fiji software. Boxes highlight the 25% highest levels of expression of GFP-G3BP1 that did not support SG formation.</div></section><section id="sec-4-4-12"><h4>Bacterial Cell Culture</h4><div id="p0355" role="paragraph">cDNA clones were transformed into One Shot TOP10 chemically competent <i>E.&nbsp;coli</i> (Thermo Fisher Scientific). Single colonies were grown overnight at 37°C in LB media containing selection antibiotic (50&nbsp;μg/ml kanamycin for pEGFP vectors, 100&nbsp;μg/ml ampicillin for pGEX-2T vectors). Cells were pelleted by centrifugation and lysed for harvesting DNA. All competent bacterial cells were stored at −80°C until transformation for cloning and recombinant protein expression.</div></section><section id="sec-4-4-13"><h4>Protein Purification</h4><div id="p0360" role="paragraph">G3BP1 full-length and mutants were expressed and purified from <i>E.&nbsp;coli</i> Rosetta 2(DE3) cells (Millipore) and purified under native conditions unless otherwise noted. G3BP1-expressing constructs contain a TEV cleavage site between the N-terminal GST and fusion protein. The ΔNTF2L mutant as well as IDR fragments contain 2x GGS linkers between the TEV site and fusion proteins for efficient TEV cleavage. <i>E.&nbsp;coli</i> were grown to OD<sub>600</sub> of 0.8 and induced with 0.6&nbsp;mM IPTG at 16°C overnight. Pelleted cells were resuspended in lysis buffer (250&nbsp;mM NaCl, 50&nbsp;mM HEPES 7.5, 1&nbsp;mM DTT, protease inhibitor). After sonication, lysates were pelleted at 30,000 x g at 4°C for 30&nbsp;min. Supernatants were applied to packed GST columns with 10&nbsp;mL GST beads (GE) prewashed with lysis buffer at room temperature. Proteins were eluted with 10&nbsp;mM glutathione (Sigma) in lysis buffer. The eluted proteins were incubated with TEV protease at 4°C overnight (<a id="crosref1265" href="#bib25" role="doc-biblioref" data-xml-rid="bib25">Kapust et&nbsp;al., 2001</a>). The proteins were further treated with 0.1&nbsp;mg/ml RNaseA (Thermo Fisher) to remove RNA (<a id="crosref1270" href="#bib39" role="doc-biblioref" data-xml-rid="bib39">Molliex et&nbsp;al., 2015</a>). Cleaved proteins were further purified by ion exchange with a HiTrap SP or Q column. The fractions were analyzed by SDS-PAGE, pooled, and concentrated. The proteins were then purified by Superdex 200 16/200 column (GE) equilibrated in SEC buffer (400&nbsp;mM NaCl, 50&nbsp;mM HEPES 7.5, 1&nbsp;mM DTT). The fractions were analyzed by SDS-PAGE, pooled, concentrated, filtered, flash frozen in liquid nitrogen, and stored at −80°C.</div></section><section id="sec-4-4-14"><h4>Liquid-Liquid Phase Separation</h4><div id="p0365" role="paragraph"><i>In&nbsp;vitro</i> LLPS experiments were performed at room temperature unless otherwise indicated. LLPS of G3BP1 was induced by addition of indicated concentrations of Ficoll 400 or RNA. The samples were mixed in low binding tubes (COSTAR 3206) and transferred to a sandwiched chamber created by cover glass and a glass slide with a double-sided spacer (Sigma GBL654002). Samples were observed under a DIC microscope using a Leica DMi8 microscope with a 20x objective. All imaged were captured within 5&nbsp;min after LLPS induction. For LLPS experiments in which RNA was added, total RNA was isolated from U2OS cells using TRIzol (Thermo Fisher Scientific) and the concentration of RNA was measured by Nanodrop (Thermo Fisher Scientific). For LLPS experiments in which we tested mRNA length-dependent G3BP1 LLPS, we maintained a constant mass of mRNA; thus shorter mRNAs have an increased molar concentration.</div></section><section id="sec-4-4-15"><h4>D/D Solubilizer Treatment</h4><div id="p0370" role="paragraph">GFP-tagged FKBP<sup>F36M</sup>-G3BP1 was transfected into U2OS <i>G3BP1/2</i> dKO cells for 48 h, allowing the cells to reach 80% confluence. GFP-G3BP1 WT was used as a control. U2OS <i>G3BP1/2</i> dKO cells were pre-incubated with 250&nbsp;nM D/D solubilizer (Takara) for 30&nbsp;min before addition of 500&nbsp;μM sodium arsenite and further treated for 1&nbsp;h at 37°C, 5% CO2. After treatment, the cells were fixed and stained for SG markers and DAPI.</div></section><section id="sec-4-4-16"><h4>FKBP Live-Cell Dissolution</h4><div id="p0375" role="paragraph">Live cell experiments were performed with the Opterra II Swept Field Confocal Microscope (Bruker) using Prairie View 5.5 Software. <i>G3BP1/2</i> dKO cells were seeded into 35-mm poly-D-lysine-coated glass bottom dishes No. 1.5 (MatTek) 48&nbsp;h prior to imaging. Cells were transfected with GFP-G3BP1 WT or GFP-FKBP<sup>F36M</sup>-G3BP1 24&nbsp;h prior to imaging using FuGENE 6 (Promega). Immediately before imaging, the medium was changed to 1&nbsp;mL complete phenol red-free DMEM medium (HyClone). During imaging, cells were maintained at 37°C and supplied with 5% CO<sub>2</sub> using a Bold Line Cage Incubator (Okolabs) and an objective heater (Bioptechs). Imaging was performed using a 60x Plan Apo 1.40NA oil objective and Perfect Focus (Nikon) was engaged for the duration of the capture. A multipoint time lapse was taken, imaging each field every 30&nbsp;s with the 488-nm laser set at 80 power with 75-ms exposure using a 35-μm slit. Two min into imaging, 500&nbsp;μM sodium arsenite (Sigma) was added to the sample. Thirty-two min into imaging, 250&nbsp;nM D/D solubilizer was added while maintaining the 500&nbsp;μM sodium arsenite concentration. Images were imported using the Bio-Formats plugin and analyzed in ImageJ 1.51J (NIH). The intensity of the SGs was quantified on the frame immediately before and after D/D solubilizer addition, and set relative to the cytoplasmic intensity of the cell for the frame. The data was normalized to the maximum intensity of GFP-G3BP1 WT or GFP-FKBP<sup>F36M</sup>-G3BP1 on the frame prior to addition of D/D solubilizer.</div></section><section id="sec-4-4-17"><h4>Protein Solubility Assay</h4><div id="p0380" role="paragraph"><i>G3BP1/2</i> dKO U2OS cells were transfected with GFP-tagged G3BP1 FL or IDR1/2 deletion constructs. ∼24&nbsp;h after transfection, cells were washed with ice-cold 1x PBS and lysed in ice-cold RIPA buffer (25&nbsp;mM Tris-HCl, pH 7.6, 150&nbsp;mM NaCl, 1% NP-40, 1% sodium deoxycholate, 0.1% SDS; Thermo Fisher, 89900). Cell lysates were then passed through a 21-gauge needle 10 times before being cleared by centrifugation at 21,000 x g for 30&nbsp;min at 4°C to generate the RIPA-soluble fraction. The pellets were washed twice with ice-cold RIPA buffer and re-centrifuged for 10&nbsp;min. RIPA-insoluble pellets and soluble fractions were mixed with 1x LDS sample buffer containing 0.1&nbsp;M DTT and further boiled at 75°C for 5&nbsp;min. Proteins were resolved by NuPAGE Novex 4%–12% Bis-Tris Gel (Thermo Fisher) and followed by western blotting for GFP.</div></section><section id="sec-4-4-18"><h4>Western Blotting</h4><div id="p0385" role="paragraph">Cells were washed with PBS and lysed with RIPA buffer (25&nbsp;mM Tris-HCl, pH 7.6, 150&nbsp;mM NaCl, 1% NP-40, 1% sodium deoxycholate, 0.1% SDS; Pierce, 89901) supplemented with proteinase inhibitor cocktail (Roche, 1186153001). Lysates were centrifuged for 10&nbsp;min at 4°C at 21,000 x g. 4X NuPAGE LDS sample buffer (Thermo Fisher Scientific, NP0008) with 25&nbsp;mM DTT was added to the supernatant and samples were boiled at 70°C for 5&nbsp;min. Samples were separated in 4%–12% NuPAGE Bis-Tris gels (Invitrogen) and transferred to nitrocellulose membranes using an iBlot 2 transfer device (Thermo Fisher Scientific). Membranes were blocked with Odyssey blocking buffer (LI-COR) and then incubated with primary antibodies at 4°C overnight. Membranes were washed 3 times with PBST (0.1% Tween) and further incubated with dye-labeled secondary antibodies. Membranes were visualized with an Odyssey Fc imaging system (LI-COR).</div></section><section id="sec-4-4-19"><h4>Fluorescence Recovery After Photobleaching</h4><div id="p0390" role="paragraph">FRAP experiments were performed with the Opterra II Swept Field Confocal Microscope (Bruker) using Prairie View 5.5 Software. <i>G3BP1/2</i> dKO cells or G3BP1-tdTomato knock-in cells were seeded into a 4-well lab-Tek chambered coverglass (Nunc) 48&nbsp;h prior to imaging. Cells were transfected with GFP-G3BP1 WT or mutants 24&nbsp;h prior to imaging using FuGENE 6 (Promega). Immediately before imaging, the medium was changed to 1&nbsp;mL complete phenol red-free DMEM medium (HyClone). During imaging, cells were maintained at 37°C and supplied with 5% CO<sub>2</sub> using a Bold Line Cage Incubator (Okolabs) and an objective heater (Bioptechs). Imaging was performed using a 60x Plan Apo 1.40NA oil objective and Perfect Focus (Nikon) was engaged for the duration of the capture.</div><div id="p0395" role="paragraph">For imaging, cells without spontaneous SGs and of similar intensity were selected and the fields were stored as stage locations. The cells were then incubated with 500&nbsp;μM sodium arsenite (Sigma) for 30&nbsp;min. Following incubation, time lapses were taken using both the 488-nm and 561-nm imaging lasers set at 100 power and 100-ms exposure each set at max speed (∼0.27&nbsp;ms period) for 100 frames. Photobleaching of SGs occurred 2&nbsp;s into capture, using the 488-nm FRAP laser to bleach both the green and red channels simultaneously. A 3&nbsp;μm-diameter circle was used for all FRAP experiments. Data was repeated in triplicate for each condition, with each replicate having at least n&nbsp;= 15 cells.</div><div id="p0400" role="paragraph">Data was opened in ImageJ 1.51J (NIH) using the Prairie Reader plugin. ROIs were generated in the photobleach region, a non-photobleached cell, and the background for each time lapse, and the mean intensity of each was extracted. These values were exported into Igor Pro 7.0 (WaveMetrics), where photobleach and background correction were performed, and fit FRAP curves were generated.</div></section><section id="sec-4-4-20"><h4>PolyA RNA Co-Immunoprecipitation</h4><div id="p0405" role="paragraph">U2OS cells were exposed to 500&nbsp;μM sodium arsenite for 1&nbsp;h at 37°C. After treatment, cells were washed 3 times with PBS and harvested in 200&nbsp;μL lysis buffer containing 20&nbsp;mM Tris HCl, pH 7.5, 2.5&nbsp;mM MgCl<sub>2</sub>, 100&nbsp;mM NaCl, 0.5% Triton, protease inhibitors, 0.2&nbsp;U/mL RNase inhibitors (Promega). Cells were incubated 15&nbsp;min on ice and centrifuged for 5&nbsp;min at 1,000 x g at 4°C. 100&nbsp;μL oligo-dT Dynabeads (Invitrogen) pre-equilibrated with lysis buffer were added and binding was performed for 30&nbsp;min at 4°C on a rotating wheel. Dynabeads were washed five times with 20&nbsp;mM Tris HCl, pH 8, 150&nbsp;mM NaCl, 0.02% NP-40, 0.1&nbsp;U/mL RNase inhibitors, for 5&nbsp;min on ice. Following this step, 50&nbsp;μL of 20&nbsp;mM Tris HCl, pH 7.5, was added to the beads and heated for 45&nbsp;s at 80°C. Supernatant was then quickly separated from beads. Half of the sample was analyzed by western blot using anti-G3BP1 antibody and the other half was run on TBE-urea 6% gels and stained with SYBR Gold (Invitrogen) to detect RNA. The same experiment was performed in the absence of RNasin and in the presence of 0.1&nbsp;U/mL RNase A (Invitrogen) as a negative control.</div></section><section id="sec-4-4-21"><h4>Depletion of PolyA RNA</h4><div id="p0410" role="paragraph">Total cellular RNA was isolated from U2OS cells using TRIzol (Thermo Fisher, 15596018) and dissolved in H<sub>2</sub>O. The concentration of RNA was measured by Nanodrop (Thermo Fisher Scientific). For polyA depletion, 100&nbsp;μg total RNA was mixed with 500&nbsp;μL lysis/binding buffer (NEB) and 100&nbsp;μL washed magnetic oligo d(T)25 beads (NEB, S1550S) were added to the RNA sample. After incubating the sample at room temperature for 5&nbsp;min, another 100&nbsp;μL fresh beads were added. After 2 rounds of polyA RNA extraction, the leftover RNA was precipitated by isopropanol and washed by 75% ethanol and further resuspended in H<sub>2</sub>O.</div></section><section id="sec-4-4-22"><h4><i>In&nbsp;Vitro</i> Transcription</h4><div id="p0415" role="paragraph">DNA templates encoding full length <i>ACTB</i>, <i>HSPA8</i>, <i>PABPC1</i>, <i>EEF2</i> and fragments of <i>HSPA8</i> were amplified from a U2OS cDNA library. T7 promoter sequence: TAATACGACTCACTATAGGGAG was added in the 5′ end of the DNA template. Transcription was carried out overnight at 37°C in a final volume of 20&nbsp;μL and using 0.5&nbsp;μg DNA template using HiScribe RNA synthesis kit (NEB, E2040S). The reaction was stopped by the addition of DNase I, transcription and quality of RNA was checked by agarose gel, and RNA was purified using Monarch RNA purification kit (NEB).</div></section><section id="sec-4-4-23"><h4>RNA Treatment with DDX19A Helicase</h4><div id="p0420" role="paragraph">1&nbsp;μg/μl total cellular RNA was incubated with 0, 0.1, or 0.2&nbsp;μM purified DDX19A (MyBioSource) overnight at 37°C in 30&nbsp;mM HEPES, pH 7.5, 75&nbsp;mM NaCl, 5&nbsp;mM MgCl<sub>2</sub>, 0.2&nbsp;mM DTT, in presence or absence of 2&nbsp;mM ATP, to allow the disruption of RNA-RNA interactions. Pre-treated RNA was then diluted 10 times in 50&nbsp;mM HEPES, pH 7.5, 150&nbsp;mM NaCl, 1&nbsp;mM DTT. 50&nbsp;ng/μL helicase-treated RNA was then mixed with 50&nbsp;μM purified G3BP1 and imaged by DIC using a Leica DMi8 microscope with a 20x objective. In parallel, integrity and unwinding of RNA was analyzed on denaturing TBE-urea and native TBE RNA gels respectively. RNA was detected using SYBR Gold.</div></section><section id="sec-4-4-24"><h4>RNA Biotinylation</h4><div id="p0425" role="paragraph">Purified RNA was biotinylated using a Thermo Fisher RNA 3′ end biotinylation kit as recommended by the manufacturer. Briefly, a final concentration of 500&nbsp;μM RNA was incubated with 2&nbsp;mM biotinylated cytosine, T4 RNA ligase, PEG, and RNase inhibitors overnight at 16°C. RNA was then purified using a Monarch Total RNA Miniprep Kit and quantified by absorbance at 260&nbsp;nm.</div></section><section id="sec-4-4-25"><h4>Electrophoretic Mobility Shift Assay</h4><div id="p0430" role="paragraph">RNA binding of purified G3BP1 WT and mutants was assessed by incubating increasing concentrations of G3BP1 with 2&nbsp;nM biotinylated RNA (20 bases –AGAUUCCACCACAAAGACCC - for purified G3BP1 and 100 bases – AGCCUGAAUCUCAGAUUCCACCACAAAGACCUCAGCGGGAUCAAAGAGUGCGAGAACAACGAAUAAAUAUUCCUCCCCAAAGGGGACCCAGACCAAUCCG - for FLAG-tagged G3BP1 and purified G3BP1 RBD mutants) for 30&nbsp;min at RT, in HEPES 50&nbsp;mM, pH 7.5, 150&nbsp;mM NaCl, 1&nbsp;mM DTT, 10&nbsp;mM MgCl<sub>2</sub>. Complexes were then separated on TBE 4%–12% gels at 100&nbsp;V in TBE buffer 0.5X and transferred on a nylon membrane. RNA was revealed using Thermo Fisher Chemiluminescent Nucleic Acid Detection Module Kit according to manufacturer instructions.</div></section><section id="sec-4-4-26"><h4>GST Pulldown</h4><div id="p0435" role="paragraph">100&nbsp;nM GST, GST-RBD, or GST-RBD<sup>RA5</sup> were incubated with 100&nbsp;nM NTF2L-HA, NTF2-IDR1-HA WT or mutants, or NTF2L-IDR1/2-HA for 1&nbsp;h at room temperature, in 50&nbsp;mM HEPES, pH 7.5, 150&nbsp;mM NaCl, 0.1% Triton X-100, 1&nbsp;mM DTT. Complexes were pulled down using Glutathione-Sepharose 4B resin for 2&nbsp;h at 4°C, washed 3 times with 50&nbsp;mM HEPES, pH 7.5, 150&nbsp;mM NaCl, 0.1% Triton X-100, 1&nbsp;mM DTT, and analyzed by western blot using anti-GST and anti-HA antibodies.</div></section><section id="sec-4-4-27"><h4>Size Exclusion Chromatography Multi-Angle Light Scattering</h4><div id="p0440" role="paragraph">Size exclusion chromatography multi-angle light scattering (SEC-MALS) experiments were carried out using a WTC-0150S5 (MW range 5000-1,250,000 Da) size-exclusion column (Wyatt Technologies) with 3 detectors connected in series: an Agilent 1200 ultraviolet (UV) detector (Agilent Technologies), a Wyatt DAWN-HELEOS-II multi-angle light-scattering (MALS) detector, and a Wyatt Optilab T-rEX differential refractive index (RI) detector (Wyatt Technologies). The column was equilibrated with PBS. All data were collected at 25°C. A 100&nbsp;μL sample was injected into the column using an auto-sample injection method with a flow rate of 0.5&nbsp;ml/min. Protein in the eluent was detected via UV absorbance at 280&nbsp;nm, light scattering, and refractive index detectors. The data were recorded and analyzed with Wyatt Astra software (version 7.0). The refractive index increment, dn/dc, was assumed to be 0.185&nbsp;ml/g for measuring the concentration of the protein samples. EASI Graphs (Astra software) were exported and plotted as a molar mass distribution superimposed on light scattering and UV traces versus elution volume.</div></section><section id="sec-4-4-28"><h4>Analytical Ultracentrifugation</h4><div id="p0445" role="paragraph">Sedimentation velocity experiments were conducted in a ProteomeLab XL-I analytical ultracentrifuge (Beckman Coulter) following standard protocols unless mentioned otherwise (<a id="crosref1275" href="#bib74" role="doc-biblioref" data-xml-rid="bib74">Zhao et&nbsp;al., 2013</a>). Samples (50&nbsp;μM G3BP1) in buffer containing 20&nbsp;μM PBS, 50&nbsp;mM HEPES, pH 7, 400&nbsp;mM NaCl, 5&nbsp;mM DTT were loaded into cell assemblies composed of double sector charcoal-filled centerpieces with a 12-mm path length and sapphire windows. Buffer density and viscosity were determined in a DMA 5000&nbsp;M density meter (Anton Paar) and an AMVn automated micro-viscometer (Anton Paar), respectively. The partial specific volumes and the molecular masses of the proteins were calculated based on their amino acid compositions in SEDFIT (<a id="intref0230" href="https://sedfitsedphat.nibib.nih.gov/software/default.aspx" target="_blank">https://sedfitsedphat.nibib.nih.gov/software/default.aspx</a>) (<a id="crosref1280" href="#bib8" role="doc-biblioref" data-xml-rid="bib8">Cohn and Edsall, 1943</a>). The cell assemblies, containing identical sample and reference buffer volumes of 390&nbsp;μL, were placed in a rotor and temperature equilibrated at rest at 20°C for 2&nbsp;h before it was accelerated from 0 to 50,000&nbsp;rpm. Rayleigh interference optical data were collected at 1-min intervals for 12 h. The velocity data were modeled with diffusion-deconvoluted sedimentation coefficient distributions c(s) in SEDFIT (<a id="intref0235" href="https://sedfitsedphat.nibib.nih.gov/software/default.aspx" target="_blank">https://sedfitsedphat.nibib.nih.gov/software/default.aspx</a>) using algebraic noise decomposition and with signal-average frictional ratio and meniscus position refined with non-linear regression (<a id="crosref1285" href="#bib53" role="doc-biblioref" data-xml-rid="bib53">Schuck, 2000</a>). The s-value was corrected for time and finite acceleration of the rotor was accounted for in the evaluation of Lamm equation solutions (<a id="crosref1290" href="#bib75" role="doc-biblioref" data-xml-rid="bib75">Zhao et&nbsp;al., 2015</a>). Maximum entropy regularization was applied at a confidence level of P −0.68. The partial specific volumes of the proteins, based on their amino acid composition, were calculated in SEDFIT. All plots were created in GUSSI (<a id="intref0240" href="http://biophysics.swmed.edu/MBR/software.html" target="_blank">http://biophysics.swmed.edu/MBR/software.html</a>) (provided by Dr. Chad Brautigam). A two-dimensional size-shape distribution, c(<i>s, f/f</i><sub><i>0</i></sub>) (with the one dimension the <i>s-</i>distribution and the other the <i>f/f</i><sub><i>0</i></sub>-distribution) was calculated with an equidistant <i>f/f</i><sub><i>0</i></sub>-grid of 0.25 steps that varied from 1 to 3.5, a linear <i>s</i>-grid from 0.2 to 8&nbsp;S with 100 s-values. Tikhonov-Phillips regularization was at one standard deviation. The velocity data were transformed to <i>c(M,f/f</i><sub><i>0</i></sub><i>), c(s,f/f</i><sub><i>0</i></sub><i>)</i>, and <i>c(s,M)</i> distributions, with <i>M</i> the molecular weight, <i>f/f</i><sub><i>0</i></sub> the frictional ratio, s the sedimentation coefficient, and plotted as contour plots. This analysis was with regularization. The color temperature of the contour lines indicates the population of species. As in one-dimensional c(s), the peak width in <i>c(s,M)</i> contains contributions both from regularization (reflecting limited resolution given the signal-to-noise ratio of the data) and from true heterogeneity. The dotted lines of <i>c(M, f/f</i><sub><i>0</i></sub><i>)</i> indicate constant s and for the <i>c(s,M)</i> plot constant <i>f/f</i><sub><i>0.</i></sub> The distributions were not normalized (<a id="crosrefs0100" href="#bib5" role="doc-biblioref" data-xml-rid="bib5 bib74">Brown and Schuck, 2006; Zhao et&nbsp;al., 2013</a>).</div></section><section id="sec-4-4-29"><h4>Cross-Linking of Recombinant Proteins</h4><div id="p0450" role="paragraph">10&nbsp;μM recombinant proteins were incubated with 100&nbsp;μM amine-specific cross-linker BS<sup>3</sup> in a final volume of 20&nbsp;μl for 30&nbsp;min at room temperature. The cross-linking reaction was quenched by the addition of 50&nbsp;mM Tris HCl, pH 7.5, and incubated for at least 15&nbsp;min at room temperature. Samples were then analyzed by SDS-PAGE.</div></section><section id="sec-4-4-30"><h4>Cross-Linking of Cell Extracts</h4><div id="p0455" role="paragraph">U2OS cells or U2OS <i>G3BP1/2</i> dKO cells expressing indicating constructs were harvested in PBS, pelleted at 800 x g for 5&nbsp;min and resuspended in 200&nbsp;μl PBS buffer, pH 7, supplemented with 0.1% Triton X-100 and protease inhibitors. Cells were then incubated on ice for 15&nbsp;min and centrifuged for 5&nbsp;min at 1000 x g at 4°C. Supernatants were collected and protein concentration was assessed using Bradford reagent and BSA. 1&nbsp;mg/ml protein lysate was then incubated with 2.5&nbsp;mM BS<sup>3</sup> for 30&nbsp;min at room temperature in a final volume of 200&nbsp;μl. The reaction was quenched by the addition of 50&nbsp;mM Tris HCl, pH 7.5, and incubated for 15&nbsp;min at room temperature. Samples were then separated by Tris-acetate 3%–8% SDS-PAGE and transferred to nitrocellulose membrane. Membranes were blocked with 5% BSA prepared in TBS&nbsp;+ 0.1% Tween-20 and analyzed using mouse anti-G3BP1 antibody.</div></section><section id="sec-4-4-31"><h4>RNA Binding Assay of Recombinant G3BP1 with Isolated RNA</h4><div id="p0460" role="paragraph">5&nbsp;μM recombinant GST-tagged-G3BP1 was mixed with 50&nbsp;ng/μl indicated RNA in 50&nbsp;mM HEPES buffer, 150&nbsp;mM NaCl, 1&nbsp;mM DTT. Samples were incubated for 10&nbsp;min at room temperature to allow RNA binding, followed by exposure to UV (254&nbsp;nm, 400 mJ/cm<sup>2</sup> followed by 200 mJ/cm<sup>2</sup>). Four volumes of binding buffer containing 20&nbsp;mM Tris HCl, pH 7.5, 137&nbsp;mM NaCl, 1% Triton X-100, 2&nbsp;mM EDTA, and 1x protease inhibitor (Roche) were added to one volume of G3BP1/RNA mix. Samples were treated with 4&nbsp;U/mL of RNase I for 3&nbsp;min at 37°C and the reaction was stopped by the addition of RNasin (0.4&nbsp;U/μL). GST-G3BP1/RNA complexes were then isolated with Glutathione-Sepharose 4B beads for 2&nbsp;h at 4°C on a rotating wheel. For RNA detection, RNA-G3BP1 complexes were treated with alkaline phosphatase (Promega, M1821) and RNA was biotinylated using an RNA 3′ End Biotinylation Kit (Pierce) as described below.</div></section><section id="sec-4-4-32"><h4>RNA Binding Assay of Cellular G3BP1 WT and Mutants</h4><div id="p0465" role="paragraph">The RNA binding assay was adapted from a previous report (<a id="crosref1295" href="#bib60" role="doc-biblioref" data-xml-rid="bib60">Valentin-Vega et&nbsp;al., 2016</a>). U2OS <i>G3BP1/2</i> dKO cells expressing GFP-G3BP1 WT or indicated mutants were washed with PBS and exposed to UV (254&nbsp;nm, 400 mJ/cm<sup>2</sup> followed by 200 mJ/cm<sup>2</sup>). Cells were then harvested in lysis buffer containing 20&nbsp;mM Tris HCl, pH 7.5, 137&nbsp;mM NaCl, 1% Triton X-100, 2&nbsp;mM EDTA, 1x protease inhibitor (Roche) and incubated for 10&nbsp;min on ice. Lysates were then successively treated with 8&nbsp;U/mL DNase I for 5&nbsp;min at 37°C and 4&nbsp;U/mL RNase I for 3&nbsp;min at 37°C. The reaction was stopped by the addition of RNasin (0.4&nbsp;U/μL). Lysates were then centrifuged at 21,000 x g for 10&nbsp;min at 4°C and 10&nbsp;μL GFP-Trap beads (Chromotek, gtma-20) were incubated with the supernatant fraction at 4°C overnight. The beads were washed twice with lysis buffer, twice with 1&nbsp;M NaCl, and twice with lysis buffer again. Beads were further suspended in 100&nbsp;μL 10&nbsp;mM Tris-HCl, pH 7.5, and treated with 2 units of calf intestinal alkaline phosphatase (Promega, M182A) at 37°C for 10&nbsp;min at 1000&nbsp;rpm in a Thermomixer. Beads were washed with lysis buffer twice and RNA labeling was performed with an RNA 3′ End Biotinylation Kit (Pierce, 20160). After washing with lysis buffer twice, beads were boiled in 1x LDS sample buffer (Life Technologies, NP007) with 100&nbsp;mM DTT. Protein-RNA complexes were separated by 4%–12% NuPAGE Bis-Tris gels, transferred to nitrocellulose membranes, and blotted with IRDye 680LT streptavidin (LI-COR, 926-68031).</div></section><section id="sec-4-4-33"><h4>Measurement of G3BP1 RNA-Binding Affinity Before and After Stress</h4><div id="p0470" role="paragraph">FLAG-G3BP1 was transiently expressed in U2OS <i>G3BP1/2</i> dKO cells for 24 h. Cells were then exposed to 500&nbsp;μM sodium arsenite for 1 h, quickly washed with PBS, and harvested in lysis buffer containing PBS, pH 7, 5&nbsp;mM EDTA, 0.2% NP-40, 10% glycerol, 10&nbsp;μg/mL RNase A, 0.01 u/mL DNase I and protease inhibitors. Cells were further lysed by passing through a 21-gauge needle attached to a 1&nbsp;mL syringe 5 times. Samples were centrifuged for 5&nbsp;min at 1000 x g at 4°C, and the supernatant was collected. The lysate was pre-cleared for 1&nbsp;h at 4°C using protein A/G beads and then incubated with anti-FLAG M2 beads for 90&nbsp;min at 4°C on a wheel. Beads were successively washed with lysis buffer (twice), lysis buffer&nbsp;+ 250&nbsp;mM NaCl (400&nbsp;mM NaCl final concentration; twice) and lysis buffer (twice). Finally, FLAG-G3BP1 was eluted by incubating the beads in lysis buffer&nbsp;+ 200&nbsp;μg/mL FLAG peptide for 30&nbsp;min at room temperature while shaking. Supernatant containing FLAG-G3BP1 was collected and dialyzed against 50&nbsp;mM HEPES, pH 7.5, 150&nbsp;mM NaCl, 10&nbsp;mM MgCl<sub>2</sub>, 1&nbsp;mM DTT overnight at 4°C. Relative FLAG-G3BP1 concentrations were assessed by western blot and equal concentrations were used for EMSA. Free and bound RNA fractions were quantified using FIJI software and data were plotted against the Hill-Langmuir equation. Midpoint values for G3BP1 before and after stress were 0.211 ± 0.028 and 0.173 ± 0.095 respectively.</div></section><section id="sec-4-4-34"><h4>APEX2-Mediated Proximity Labeling of Stress Granules</h4><div id="p0475" role="paragraph">APEX2-G3BP1 and APEX2-G3BP1-2xAsh1 plasmids were constructed using pcDNA3 APEX2-NES, a gift from Alice Ting (Addgene plasmid 49386; <a id="intref0245" href="http://addgene.org/49386" target="_blank">http://addgene.org/49386</a>; RRID:Addgene_49386). 2&nbsp;μg APEX2-G3BP1 and APEX2-G3BP1-2xAsh1 plasmids were transfected into 10-cm dishes of <i>G3BP1/2</i> dKO U2OS cells with Lipofectamine 2000 (Thermo Fisher Scientific, 11668019) overnight. Cells were then exposed to 0.5&nbsp;mM sodium arsenite for 1 h. After 30&nbsp;min arsenite treatment, 0.5&nbsp;mM biotin phenol (Sigma, SML2135) was added into the medium for a total of 30&nbsp;min of incubation. Cells were further treated with 1&nbsp;mM H<sub>2</sub>O<sub>2</sub> for 1&nbsp;min followed by quenching buffer (10&nbsp;mM sodium ascorbate and 5&nbsp;mM Trolox in 1x PBS). Cells were lysed with 1&nbsp;mL RIPA buffer (25&nbsp;mM Tris HCl, pH 7.5, 150&nbsp;mM NaCl, 1% NP-40, 1% sodium deoxycholate, 0.1% SDS, protease inhibitor cocktail, 10&nbsp;mM sodium ascorbate, 5&nbsp;mM Trolox). The lysates were cleared by centrifugation at 21,000 x g for 10&nbsp;min at 4°C and then dialyzed in 500&nbsp;mL RIPA buffer (25&nbsp;mM Tris HCl, pH 7.5, 150&nbsp;mM NaCl, 1% NP-40, 1% sodium deoxycholate, 0.1% SDS) using Slide-A-Lyzer Dialysis Cassettes (10K MWCO) (Thermo Fisher Scientific, 66380) at 4°C overnight to remove free biotin phenol. Dialyzed lysates were incubated with 100&nbsp;μl streptavidin magnetic beads (Thermo Fisher Scientific, 65001) at 4°C overnight with gentle rotation. Beads were then washed with 2x 1&nbsp;mL RIPA lysis buffer, 1x 1&nbsp;mL of 1&nbsp;M NaCl, 1x 1&nbsp;mL of 2&nbsp;M urea in 10&nbsp;mM Tris HCl, pH 8.0, and again with 2x 1&nbsp;mL RIPA lysis buffer. To elute the biotinylated proteins, the beads were boiled for 10&nbsp;min in 50&nbsp;μL 1x NuPAGE LDS sample buffer (Thermo Fisher Scientific, NP0007) supplemented with 20&nbsp;mM dithiothreitol DTT and 2&nbsp;mM biotin. The streptavidin eluate was collected and run on an 10% SDS-PAGE gel for mass spectrometry. Proteins identified were plotted based on fold change of G3BP1_FL/2xAsh1 Swap as well as P value.</div></section><section id="sec-4-4-35"><h4>Small Angle X-ray Scattering</h4><div id="p0480" role="paragraph">Small angle X-ray scattering (SAXS) experiments were performed at BioCAT (beamline 18ID at the Advanced Photon Source, Chicago) with in-line size exclusion chromatography (SEC-SAXS) to separate dimeric protein from aggregates and ensure the best possible buffer subtraction. Samples were injected onto a Superdex 200 Increase 5/150 GL column (GE Lifesciences) preequilibrated in the desired buffer and ionic strength using an FPLC running at 0.45&nbsp;mL/min. The output of the column passed through UV and conductance monitors prior to injection into a coflow capillary system (<a id="crosref1300" href="#bib29" role="doc-biblioref" data-xml-rid="bib29">Kirby et&nbsp;al., 2016</a>) in which the sample is sheathed in a matched buffer allowing for slower flow without risk of radiation damage. Scattering intensity was recorded using a Pilatus3 1M (Dectris) detector placed 3.5&nbsp;m from the sample providing access to a q-range from 0.004-0.4&nbsp;Å<sup>-1</sup>. 0.5&nbsp;s exposures were acquired every second during the elution. Data were reduced using BioXTAS RAW 1.4.0 (<a id="crosref1305" href="#bib18" role="doc-biblioref" data-xml-rid="bib18">Hopkins et&nbsp;al., 2017</a>). The contribution of the buffer to the X-ray scattering curve was determined by averaging frames from the SEC eluent, which gave rise to baseline levels of integrated X-ray scattering, UV absorbance and conductance. Frames were selected as close to the protein elution as possible and, ideally, frames pre- and post-elution were averaged. Final q versus I(q) datasets were generated by subtracting the average buffer trace from all elution frames and averaging curves from elution volumes close to the maximum integrated scattering intensity; these frames were statistically similar in both small and large angles. Buffer subtraction, subsequent Guinier fits, and Kratky transformations were done using custom MATLAB (Mathworks) scripts.</div></section><section id="sec-4-4-36"><h4><i>In silico</i> Modeling</h4><div id="p0485" role="paragraph">The size distribution of G3BP1 was determined by generating a large pool of conformations where the structures of the folded domains were held fixed and the IDRs were modeled as random self-avoiding chains. The NTF2L dimer structure was fixed to those in the crystal structure 4FCM (<a id="crosref1310" href="#bib63" role="doc-biblioref" data-xml-rid="bib63">Vognsen et&nbsp;al., 2013</a>). Presently there exists no set of coordinates for the G3BP1 RRM; however, substantial sequence conservation with known RRMs allowed for reliable homology modeling implemented in Swiss-Prot (<a id="crosref1315" href="#bib65" role="doc-biblioref" data-xml-rid="bib65">Waterhouse et&nbsp;al., 2018</a>) using structural coordinates of hnRNPA2B1 (PDB: <a id="intref0250" href="http://firstglance.jmol.org/fg.htm?mol5HO4" target="_blank">5HO4</a>) (<a id="crosref1320" href="#bib67" role="doc-biblioref" data-xml-rid="bib67">Wu et&nbsp;al., 2018</a>). The IDRs linking the folded regions and C-terminal IDR3 were modeled using the “Random” algorithm in Ranch (<a id="crosref1325" href="#bib4" role="doc-biblioref" data-xml-rid="bib4">Bernadó et&nbsp;al., 2007</a>). This algorithm builds random chains via addition of residues using a pool of accepted peptide backbone angles. The only restriction imposed by the algorithm on the conformations is that there are no steric clashes. Sidechains are ignored, and the resulting IDRs are simply an alpha carbon trace of the peptide backbone. The Ranch algorithm implemented in this way generates a pool of conformations in which all IDRs are close to the self-avoiding random walk limit, i.e., there are no attractive interactions between IDRs and/or folded domains. For this study, 20,000 independent conformations were generated to ensure proper sampling of the conformational space.</div><div id="p0490" role="paragraph">Radii of gyration for the random pool of conformations were calculated by using the ‘measure radius’ function in VMD (<a id="crosref1330" href="#bib20" role="doc-biblioref" data-xml-rid="bib20">Humphrey et&nbsp;al., 1996</a>) on a selection containing all alpha carbons. This method does not account for the contribution of hydration shell water to the radius; thus the numerical values are a slight underestimate with respect to values measured by SAXS. Analysis of the distribution of radii was performed using MATLAB (Mathworks). The distances between different regions of G3BP1 were calculated by first measuring the location of the centers of mass of the relevant atoms, then calculating the vector distance between the two centers of mass. In order to produce a relevant metric to correlate the IDR distances to the global radius, we wanted to create a value that informed on whether IDR3 of each monomer was close in space to IDR1 of either monomer. This value was computed using the equation:<div id="ufd1" class="display-formula"><div class="equation" role="math"><div class="inner"><math xmlns="http://www.w3.org/1998/Math/MathML" alttext="No alternative text available"><mrow><mi>m</mi><mi>e</mi><mi>a</mi><mi>n</mi><mspace width="0.25em"></mspace><mi>d</mi><mi>i</mi><mi>s</mi><mi>t</mi><mi>a</mi><mi>n</mi><mi>c</mi><mi>e</mi><mo linebreak="badbreak">=</mo><mfrac><mrow><msup><mrow><mrow><mo>[</mo><mrow><mfrac><mn>1</mn><mrow><msub><mi>r</mi><mrow><mi>a</mi><mi>b</mi></mrow></msub></mrow></mfrac><mo linebreak="badbreak">+</mo><mfrac><mn>1</mn><mrow><msub><mi>r</mi><mrow><mi>a</mi><mi>a</mi></mrow></msub></mrow></mfrac></mrow><mo>]</mo></mrow></mrow><mrow><mo>−</mo><mn>1</mn></mrow></msup><mo linebreak="badbreak">+</mo><msup><mrow><mrow><mo>[</mo><mrow><mfrac><mn>1</mn><mrow><msub><mi>r</mi><mrow><mi>b</mi><mi>b</mi></mrow></msub></mrow></mfrac><mo linebreak="badbreak">+</mo><mfrac><mn>1</mn><mrow><msub><mi>r</mi><mrow><mi>b</mi><mi>b</mi></mrow></msub></mrow></mfrac></mrow><mo>]</mo></mrow></mrow><mrow><mo>−</mo><mn>1</mn></mrow></msup></mrow><mn>2</mn></mfrac></mrow></math></div></div></div>where r<sub>xx</sub> represent the relevant distances between IDR3 on monomer “a” and “b.”</div></section></section><section id="sec-4-5"><h3>Quantification and Statistical Analysis</h3><section id="sec-4-5-1"><h4>Quantification of Stress Granule Rescue in U2OS Cells by G3BP1 Mutants</h4><div id="p0495" role="paragraph">U2OS <i>G3BP1/2</i> dKO cells transiently expressing indicated G3BP1 mutants were exposed to 500&nbsp;μM sodium arsenite for indicated times (0, 10&nbsp;min, or 60&nbsp;min) to induce SG formation, fixed, and stained for eIF3η. All images were taken with the same settings to allow the comparison of expression level between different conditions. For each cell, average GFP intensity was measured using Fiji software to control for expression level of the GFP-fused proteins, and cells were scored for the presence of SGs using eIF3η as a SG marker. When possible, only equivalent levels of expression were used to quantify SG rescue. Otherwise, all cells were considered for the quantification.</div></section><section id="sec-4-5-2"><h4>Enrichment of GFP-G3BP1 WT and Mutants in Stress Granules</h4><div id="p0500" role="paragraph">The SG enrichment percentage of G3BP1 FL protein and G3BP1 2xRBD protein were quantified by CellProfiler as previously described (<a id="crosref1335" href="#bib33" role="doc-biblioref" data-xml-rid="bib33">Mackenzie et&nbsp;al., 2017</a>). Briefly, the cell boundary and the boundary of SGs were identified using CellProfiler. The ratios of GFP fluorescent signal intensity in the SGs to that in the entire cell were used as indicators for the SG enrichment percentages of these two proteins. At least 100 cells were included in each condition. <sup>∗∗∗∗</sup>p&nbsp;&lt; 0.0001 by unpaired t test.</div></section><section id="sec-4-5-3"><h4>Statistical Analysis</h4><div id="p0505" role="paragraph">p &gt; 0.05 was considered not significant. <sup>∗</sup><i>P</i> <math xmlns="http://www.w3.org/1998/Math/MathML" alttext="No alternative text available"><mo>≤</mo></math> 0.05, <sup>∗∗</sup><i>P</i> <math xmlns="http://www.w3.org/1998/Math/MathML" alttext="No alternative text available"><mo>≤</mo></math> 0.01, <sup>∗∗∗</sup><i>P</i> <math xmlns="http://www.w3.org/1998/Math/MathML" alttext="No alternative text available"><mo>≤</mo></math> 0.001, <sup>∗∗∗∗</sup><i>P</i> <math xmlns="http://www.w3.org/1998/Math/MathML" alttext="No alternative text available"><mo>≤</mo></math> 0.0001 by two-tailed Student’s t test, one-way ANOVA, or Log-rank (Mantel-Cox) test. Statistical analysis was performed in GraphPad Prism or Excel.</div></section></section><section id="sec-4-6"><h3>Data and Code Availability</h3><section id="sec-4-6-1"><h4>Data Resources</h4><div id="p0510" role="paragraph">The published article includes all datasets generated or analyzed during this study.</div></section></section></section><section id="acknowledgments" role="doc-acknowledgments"><h2>Acknowledgments</h2><div id="p0515" role="paragraph">We thank Natalia Nedelsky for editorial assistance, the CAGE at St. Jude Children’s Research Hospital for assistance with Cas9/CRISPR-modified cell lines, Taosheng Chen for assistance with high-throughput RNAi screening, Amanda Nourse and Sivaraja Vaithiyalingam for assistance with AUC and MALS, and Alex Holehouse and Rohit Pappu for valuable discussions. This research used resources of the Advanced Photon Source, a US Department of Energy (DOE) Office of Science User Facility operated for the DOE Office of Science by Argonne National Laboratory under contract no. DE-AC02-06CH11357. Beamline 18ID (BioCAT) is supported by grant 9 P41 GM103622 from the National Institute of General Medical Sciences of the National Institutes of Health (NIH). This work was supported by grants from HHMI and NIH (R35NS097974) to J.P.T. and from the St. Jude Research Collaborative on Membraneless Organelles to T.M. and J.P.T.</div><section id="sec-5"><h3>Author Contributions</h3><div id="p0520" role="paragraph">J.P.T. conceived and supervised the project. P.Y., C.M., R.M.K., P.Z., J.M., U.Y., Z.Y., J.W., E.W.M., T.M., H.J.K., and J.P.T. designed and/or performed the experiments and analyzed data. Y.L., Q.P., and J.Y. performed network analyses. P.Y., C.M., H.J.K., and J.P.T. wrote the primary draft of manuscript, and all authors contributed to the final version.</div></section><section id="sec-6"><h3>Declaration of Interests</h3><div id="p0525" role="paragraph">J.P.T. is a consultant for 5AM and Third Rock Ventures.</div></section></section><section id="supplementary-material" data-type="supplementary-materials"><h2>Supplemental Information (3)</h2><div id="p0535" role="paragraph"><div id="mmc1" class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="/cms/10.1016/j.cell.2020.03.046/attachment/4a4af942-2044-4791-9220-e9de5fd0a82e/mmc1.xlsx" download="mmc1.xlsx" aria-labelledby="mmc1-heading mmc1-link"><span id="mmc1-link">Spreadsheet (35.26 KB)</span></a></div><div class="core-description"><div id="mmc1-heading" class="heading">Table S1. List of 210 Genes Whose Depletion Limited SG Assembly in Response to Heat Shock, Related to Figures 1 and S1</div></div></div><div id="mmc2" class="core-supplementary-material" data-category="pdf"><div class="core-link"><a href="/cms/10.1016/j.cell.2020.03.046/attachment/1e8acf08-eab5-4dd7-b69a-9dc86d76f634/mmc2.pdf" download="mmc2.pdf" aria-labelledby="mmc2-heading mmc2-link"><span id="mmc2-link">PDF (38.12 KB)</span></a></div><div class="core-description"><div id="mmc2-heading" class="heading">Table S2. Validation of SG Markers in Different Cell Lines Used, Related to Figures 1, 5, S1, and S5</div></div></div><div id="mmc3" class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="/cms/10.1016/j.cell.2020.03.046/attachment/5460b4d3-4528-42d3-8e08-512d6ceb7885/mmc3.xlsx" download="mmc3.xlsx" aria-labelledby="mmc3-heading mmc3-link"><span id="mmc3-link">Spreadsheet (47.09 KB)</span></a></div><div class="core-description"><div id="mmc3-heading" class="heading">Table S3. Summary of Protein-Protein Interaction Data for APEX-G3BP1 FL and APEX-G3BP1 2×Ash1 IDR, Related to Figure&nbsp;6</div></div></div></div></section></div></section><div class="core-linked-content">
    

        

</div><section id="backmatter" data-extent="backmatter"><div class="core-container"><section id="references"><h2>References</h2><div id="bibliography" role="doc-bibliography"><div role="list"><div role="listitem" data-has="label"><div id="bib1" class="citations"><div id="sref1" class="citation"><div class="citation-content"><div class="label">1.</div><div>Banani, S.F. ∙ Rice, A.M. ∙ Peeples, W.B. ...</div><div><strong>Compositional control of phase-separated cellular bodies</strong></div><div><em>Cell.</em> 2016; <strong>166</strong>:651-663</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_1_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.03.046&amp;key=10.1016%2Fj.cell.2016.06.010&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_1_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.03.046&amp;key=10.1016%2Fj.cell.2016.06.010&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_1_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.046&amp;key=2-s2.0-84978472995" target="_blank" aria-label="Open Scopus in new tab">Scopus (783)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27374333/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2016.06.010&amp;pmid=27374333" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib2" class="citations"><div id="sref2" class="citation"><div class="citation-content"><div class="label">2.</div><div>Banani, S.F. ∙ Lee, H.O. ∙ Hyman, A.A. ...</div><div><strong>Biomolecular condensates: organizers of cellular biochemistry</strong></div><div><em>Nat. Rev. Mol. Cell Biol.</em> 2017; <strong>18</strong>:285-298</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nrm.2017.7" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_2_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.046&amp;key=2-s2.0-85013491971" target="_blank" aria-label="Open Scopus in new tab">Scopus (3258)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28225081/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnrm.2017.7&amp;pmid=28225081" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib3" class="citations"><div id="sref3" class="citation"><div class="citation-content"><div class="label">3.</div><div>Barabasi, A.L. ∙ Albert, R.</div><div><strong>Emergence of scaling in random networks</strong></div><div><em>Science.</em> 1999; <strong>286</strong>:509-512</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.286.5439.509" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.046&amp;key=2-s2.0-0038483826" target="_blank" aria-label="Open Scopus in new tab">Scopus (28265)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/10521342/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.286.5439.509&amp;pmid=10521342" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib4" class="citations"><div id="sref4" class="citation"><div class="citation-content"><div class="label">4.</div><div>Bernadó, P. ∙ Mylonas, E. ∙ Petoukhov, M.V. ...</div><div><strong>Structural characterization of flexible proteins using small-angle X-ray scattering</strong></div><div><em>J.&nbsp;Am. Chem. Soc.</em> 2007; <strong>129</strong>:5656-5664</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/ja069124n" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.046&amp;key=2-s2.0-34247891557" target="_blank" aria-label="Open Scopus in new tab">Scopus (986)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17411046/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Fja069124n&amp;pmid=17411046" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib5" class="citations"><div id="sref5" class="citation"><div class="citation-content"><div class="label">5.</div><div>Brown, P.H. ∙ Schuck, P.</div><div><strong>Macromolecular size-and-shape distributions by sedimentation velocity analytical ultracentrifugation</strong></div><div><em>Biophys. J.</em> 2006; <strong>90</strong>:4651-4661</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.03.046&amp;key=10.1529%2Fbiophysj.106.081372&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.03.046&amp;key=10.1529%2Fbiophysj.106.081372&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.046&amp;key=2-s2.0-33744809773" target="_blank" aria-label="Open Scopus in new tab">Scopus (464)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16565040/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1529%2Fbiophysj.106.081372&amp;pmid=16565040" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib6" class="citations"><div id="sref6" class="citation"><div class="citation-content"><div class="label">6.</div><div>Buchan, J.R.</div><div><strong>mRNP granules. Assembly, function, and connections with disease</strong></div><div><em>RNA Biol.</em> 2014; <strong>11</strong>:1019-1030</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.4161/15476286.2014.972208" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.046&amp;key=2-s2.0-84920847059" target="_blank" aria-label="Open Scopus in new tab">Scopus (334)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25531407/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.4161%2F15476286.2014.972208&amp;pmid=25531407" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib7" class="citations"><div id="sref7" class="citation"><div class="citation-content"><div class="label">7.</div><div>Buchan, J.R. ∙ Kolaitis, R.M. ∙ Taylor, J.P. ...</div><div><strong>Eukaryotic stress granules are cleared by autophagy and Cdc48/VCP function</strong></div><div><em>Cell.</em> 2013; <strong>153</strong>:1461-1474</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.03.046&amp;key=10.1016%2Fj.cell.2013.05.037&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.03.046&amp;key=10.1016%2Fj.cell.2013.05.037&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.046&amp;key=2-s2.0-84879349589" target="_blank" aria-label="Open Scopus in new tab">Scopus (546)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23791177/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2013.05.037&amp;pmid=23791177" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib8" class="citations"><div id="sref8" class="citation"><div class="citation-content"><div class="label">8.</div><div>Cohn, E.J. ∙ Edsall, J.T.</div><div><strong>Density and apparent specific volume of proteins</strong></div><div><div>Cohn, E.J. ∙ Edsall, J.T. (Editors)</div></div><div><strong>Proteins, Amino Acids, and Peptides</strong></div><div>Van Nostrand-Reinhold, 1943; 370-381</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=E.J.CohnJ.T.EdsallDensity+and+apparent+specific+volume+of+proteinsE.J.CohnJ.T.EdsallProteins%2C+Amino+Acids%2C+and+Peptides1943Van+Nostrand-Reinhold370381" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib9" class="citations"><div id="sref9" class="citation"><div class="citation-content"><div class="label">9.</div><div>Crick, S.L. ∙ Ruff, K.M. ∙ Garai, K. ...</div><div><strong>Unmasking the roles of N- and C-terminal flanking sequences from exon 1 of huntingtin as modulators of polyglutamine aggregation</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2013; <strong>110</strong>:20075-20080</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1320626110" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.046&amp;key=2-s2.0-84890288534" target="_blank" aria-label="Open Scopus in new tab">Scopus (166)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24282292/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1320626110&amp;pmid=24282292" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib10" class="citations"><div id="sref10" class="citation"><div class="citation-content"><div class="label">10.</div><div>Dundr, M.</div><div><strong>Nuclear bodies: multifunctional companions of the genome</strong></div><div><em>Curr. Opin. Cell Biol.</em> 2012; <strong>24</strong>:415-422</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.ceb.2012.03.010" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.046&amp;key=2-s2.0-84861975139" target="_blank" aria-label="Open Scopus in new tab">Scopus (141)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22541757/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.ceb.2012.03.010&amp;pmid=22541757" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib11" class="citations"><div id="sref11" class="citation"><div class="citation-content"><div class="label">11.</div><div>Edupuganti, R.R. ∙ Geiger, S. ∙ Lindeboom, R.G.H. ...</div><div><strong>N<sup>6</sup>-methyladenosine (m<sup>6</sup>A) recruits and repels proteins to regulate mRNA homeostasis</strong></div><div><em>Nat. Struct. Mol. Biol.</em> 2017; <strong>24</strong>:870-878</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nsmb.3462" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.046&amp;key=2-s2.0-85030784919" target="_blank" aria-label="Open Scopus in new tab">Scopus (399)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28869609/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnsmb.3462&amp;pmid=28869609" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib12" class="citations"><div id="sref12" class="citation"><div class="citation-content"><div class="label">12.</div><div>Fabrini, R. ∙ De Luca, A. ∙ Stella, L. ...</div><div><strong>Monomer-dimer equilibrium in glutathione transferases: a critical re-examination</strong></div><div><em>Biochemistry.</em> 2009; <strong>48</strong>:10473-10482</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/bi901238t" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.046&amp;key=2-s2.0-70350475681" target="_blank" aria-label="Open Scopus in new tab">Scopus (76)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19795889/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Fbi901238t&amp;pmid=19795889" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib13" class="citations"><div id="sref13" class="citation"><div class="citation-content"><div class="label">13.</div><div>Figley, M.D. ∙ Bieri, G. ∙ Kolaitis, R.M. ...</div><div><strong>Profilin 1 associates with stress granules and ALS-linked mutations alter stress granule dynamics</strong></div><div><em>J.&nbsp;Neurosci.</em> 2014; <strong>34</strong>:8083-8097</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1523/JNEUROSCI.0543-14.2014" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_13_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.046&amp;key=2-s2.0-84902157113" target="_blank" aria-label="Open Scopus in new tab">Scopus (113)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24920614/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1523%2FJNEUROSCI.0543-14.2014&amp;pmid=24920614" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib14" class="citations"><div id="sref14" class="citation"><div class="citation-content"><div class="label">14.</div><div>Gallouzi, I.E. ∙ Parker, F. ∙ Chebli, K. ...</div><div><strong>A novel phosphorylation-dependent RNase activity of GAP-SH3 binding protein: a potential link between signal transduction and RNA stability</strong></div><div><em>Mol. Cell. Biol.</em> 1998; <strong>18</strong>:3956-3965</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/MCB.18.7.3956" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.046&amp;key=2-s2.0-0031832234" target="_blank" aria-label="Open Scopus in new tab">Scopus (179)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/9632780/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FMCB.18.7.3956&amp;pmid=9632780" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib15" class="citations"><div id="sref15" class="citation"><div class="citation-content"><div class="label">15.</div><div>Gilks, N. ∙ Kedersha, N. ∙ Ayodele, M. ...</div><div><strong>Stress granule assembly is mediated by prion-like aggregation of TIA-1</strong></div><div><em>Mol. Biol. Cell.</em> 2004; <strong>15</strong>:5383-5398</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1091/mbc.E04-08-0715" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.046&amp;key=2-s2.0-9444279617" target="_blank" aria-label="Open Scopus in new tab">Scopus (797)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15371533/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1091%2Fmbc.E04-08-0715&amp;pmid=15371533" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib16" class="citations"><div id="sref16" class="citation"><div class="citation-content"><div class="label">16.</div><div>Goktug, A.N. ∙ Ong, S.S. ∙ Chen, T.</div><div><strong>GUItars: a GUI tool for analysis of high-throughput RNA interference screening data</strong></div><div><em>PLoS ONE.</em> 2012; <strong>7</strong>:e49386</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pone.0049386" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.046&amp;key=2-s2.0-84869860890" target="_blank" aria-label="Open Scopus in new tab">Scopus (14)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23185323/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pone.0049386&amp;pmid=23185323" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib17" class="citations"><div id="sref17" class="citation"><div class="citation-content"><div class="label">17.</div><div>Gomes, E. ∙ Shorter, J.</div><div><strong>The molecular language of membraneless organelles</strong></div><div><em>J.&nbsp;Biol. Chem.</em> 2019; <strong>294</strong>:7115-7127</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.03.046&amp;key=10.1074%2Fjbc.TM118.001192&amp;cf=fulltext&amp;site=jbc-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.03.046&amp;key=10.1074%2Fjbc.TM118.001192&amp;cf=pdf&amp;site=jbc-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.046&amp;key=2-s2.0-85065664809" target="_blank" aria-label="Open Scopus in new tab">Scopus (471)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30045872/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1074%2Fjbc.TM118.001192&amp;pmid=30045872" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib78" class="citations"><div id="sref78" class="citation"><div class="citation-content"><div class="label">18.</div><div>Guillén-Boixet, J. ∙ Kopach, A. ∙ Holehouse, A.S&gt; ...</div><div><strong>RNA-Induced Conformational Switching and Clustering of G3BP Drive Stress Granule Assembly by Condensation</strong></div><div><em>Cell.</em> 2020; <strong>181</strong>:346-361</div><div>this issue</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.03.046&amp;key=10.1016%2Fj.cell.2020.03.049&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.03.046&amp;key=10.1016%2Fj.cell.2020.03.049&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.046&amp;key=2-s2.0-85083031389" target="_blank" aria-label="Open Scopus in new tab">Scopus (467)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32302572/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2020.03.049&amp;pmid=32302572" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib18" class="citations"><div id="sref18" class="citation"><div class="citation-content"><div class="label">19.</div><div>Hopkins, J.B. ∙ Gillilan, R.E. ∙ Skou, S.</div><div><strong><i>BioXTAS RAW</i>: improvements to a free open-source program for small-angle X-ray scattering data reduction and analysis</strong></div><div><em>J.&nbsp;Appl. Cryst.</em> 2017; <strong>50</strong>:1545-1553</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1107/S1600576717011438" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.046&amp;key=2-s2.0-85030550441" target="_blank" aria-label="Open Scopus in new tab">Scopus (400)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29021737/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1107%2FS1600576717011438&amp;pmid=29021737" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib19" class="citations"><div id="sref19" class="citation"><div class="citation-content"><div class="label">20.</div><div>Huang da, W. ∙ Sherman, B.T. ∙ Lempicki, R.A.</div><div><strong>Systematic and integrative analysis of large gene lists using DAVID bioinformatics resources</strong></div><div><em>Nat. Protoc.</em> 2009; <strong>4</strong>:44-57</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nprot.2008.211" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.046&amp;key=2-s2.0-61449172037" target="_blank" aria-label="Open Scopus in new tab">Scopus (27977)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19131956/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnprot.2008.211&amp;pmid=19131956" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib20" class="citations"><div id="sref20" class="citation"><div class="citation-content"><div class="label">21.</div><div>Humphrey, W. ∙ Dalke, A. ∙ Schulten, K.</div><div><strong>VMD: visual molecular dynamics</strong></div><div><em>J.&nbsp;Mol. Graph.</em> 1996; <strong>14</strong>:33-38</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/0263-7855(96)00018-5" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.046&amp;key=2-s2.0-0029878720" target="_blank" aria-label="Open Scopus in new tab">Scopus (47200)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/8744570/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2F0263-7855%2896%2900018-5&amp;pmid=8744570" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib21" class="citations"><div id="sref21" class="citation"><div class="citation-content"><div class="label">22.</div><div>Huttlin, E.L. ∙ Ting, L. ∙ Bruckner, R.J. ...</div><div><strong>The BioPlex network: a systematic exploration of the human interactome</strong></div><div><em>Cell.</em> 2015; <strong>162</strong>:425-440</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.03.046&amp;key=10.1016%2Fj.cell.2015.06.043&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.03.046&amp;key=10.1016%2Fj.cell.2015.06.043&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.046&amp;key=2-s2.0-84937213168" target="_blank" aria-label="Open Scopus in new tab">Scopus (1002)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26186194/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2015.06.043&amp;pmid=26186194" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib22" class="citations"><div id="sref22" class="citation"><div class="citation-content"><div class="label">23.</div><div>Ivanov, P. ∙ Kedersha, N. ∙ Anderson, P.</div><div><strong>Stress granules and processing bodies in translational control</strong></div><div><em>Cold Spring Harb. Perspect. Biol.</em> 2019; <strong>11</strong></div><div>a032813</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/cshperspect.a032813" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_23_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.046&amp;key=2-s2.0-85055683937" target="_blank" aria-label="Open Scopus in new tab">Scopus (283)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30082464/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2Fcshperspect.a032813&amp;pmid=30082464" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib23" class="citations"><div id="sref23" class="citation"><div class="citation-content"><div class="label">24.</div><div>Jain, S. ∙ Wheeler, J.R. ∙ Walters, R.W. ...</div><div><strong>ATPase-modulated stress granules contain a diverse proteome and substructure</strong></div><div><em>Cell.</em> 2016; <strong>164</strong>:487-498</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.03.046&amp;key=10.1016%2Fj.cell.2015.12.038&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.03.046&amp;key=10.1016%2Fj.cell.2015.12.038&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.046&amp;key=2-s2.0-84955625345" target="_blank" aria-label="Open Scopus in new tab">Scopus (1024)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26777405/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2015.12.038&amp;pmid=26777405" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib24" class="citations"><div id="sref24" class="citation"><div class="citation-content"><div class="label">25.</div><div>Jeong, H. ∙ Mason, S.P. ∙ Barabási, A.L. ...</div><div><strong>Lethality and centrality in protein networks</strong></div><div><em>Nature.</em> 2001; <strong>411</strong>:41-42</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/35075138" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.046&amp;key=2-s2.0-0035799707" target="_blank" aria-label="Open Scopus in new tab">Scopus (4161)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11333967/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2F35075138&amp;pmid=11333967" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib25" class="citations"><div id="sref25" class="citation"><div class="citation-content"><div class="label">26.</div><div>Kapust, R.B. ∙ Tözsér, J. ∙ Fox, J.D. ...</div><div><strong>Tobacco etch virus protease: mechanism of autolysis and rational design of stable mutants with wild-type catalytic proficiency</strong></div><div><em>Protein Eng.</em> 2001; <strong>14</strong>:993-1000</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/protein/14.12.993" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_26_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.046&amp;key=2-s2.0-0035711194" target="_blank" aria-label="Open Scopus in new tab">Scopus (686)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11809930/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fprotein%2F14.12.993&amp;pmid=11809930" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib26" class="citations"><div id="sref26" class="citation"><div class="citation-content"><div class="label">27.</div><div>Kedersha, N. ∙ Panas, M.D. ∙ Achorn, C.A. ...</div><div><strong>G3BP-Caprin1-USP10 complexes mediate stress granule condensation and associate with 40S subunits</strong></div><div><em>J.&nbsp;Cell Biol.</em> 2016; <strong>212</strong>:845-860</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1083/jcb.201508028" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.046&amp;key=2-s2.0-84971547291" target="_blank" aria-label="Open Scopus in new tab">Scopus (395)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27022092/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1083%2Fjcb.201508028&amp;pmid=27022092" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib27" class="citations"><div id="sref27" class="citation"><div class="citation-content"><div class="label">28.</div><div>Khong, A. ∙ Matheny, T. ∙ Jain, S. ...</div><div><strong>The stress granule transcriptome reveals principles of mRNA accumulation in stress granules</strong></div><div><em>Mol. Cell.</em> 2017; <strong>68</strong>:808-820.e5</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.03.046&amp;key=10.1016%2Fj.molcel.2017.10.015&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.03.046&amp;key=10.1016%2Fj.molcel.2017.10.015&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.046&amp;key=2-s2.0-85033388302" target="_blank" aria-label="Open Scopus in new tab">Scopus (460)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29129640/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.molcel.2017.10.015&amp;pmid=29129640" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib28" class="citations"><div id="sref28" class="citation"><div class="citation-content"><div class="label">29.</div><div>Kiebler, M.A. ∙ Bassell, G.J.</div><div><strong>Neuronal RNA granules: movers and makers</strong></div><div><em>Neuron.</em> 2006; <strong>51</strong>:685-690</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.03.046&amp;key=10.1016%2Fj.neuron.2006.08.021&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.03.046&amp;key=10.1016%2Fj.neuron.2006.08.021&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.046&amp;key=2-s2.0-33748526877" target="_blank" aria-label="Open Scopus in new tab">Scopus (461)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16982415/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.neuron.2006.08.021&amp;pmid=16982415" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib29" class="citations"><div id="sref29" class="citation"><div class="citation-content"><div class="label">30.</div><div>Kirby, N. ∙ Cowieson, N. ∙ Hawley, A.M. ...</div><div><strong>Improved radiation dose efficiency in solution SAXS using a sheath flow sample environment</strong></div><div><em>Acta Crystallogr. D Struct. Biol.</em> 2016; <strong>72</strong>:1254-1266</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1107/S2059798316017174" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.046&amp;key=2-s2.0-85006073392" target="_blank" aria-label="Open Scopus in new tab">Scopus (116)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27917826/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1107%2FS2059798316017174&amp;pmid=27917826" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib30" class="citations"><div id="sref30" class="citation"><div class="citation-content"><div class="label">31.</div><div>Kwon, S. ∙ Zhang, Y. ∙ Matthias, P.</div><div><strong>The deacetylase HDAC6 is a novel critical component of stress granules involved in the stress response</strong></div><div><em>Genes Dev.</em> 2007; <strong>21</strong>:3381-3394</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/gad.461107" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.046&amp;key=2-s2.0-37449030154" target="_blank" aria-label="Open Scopus in new tab">Scopus (322)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/18079183/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2Fgad.461107&amp;pmid=18079183" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib77" class="citations"><div id="sref77" class="citation"><div class="citation-content"><div class="label">32.</div><div>Lam, S.S. ∙ Martell, J.D. ∙ Kamer, K.J. ...</div><div><strong>Directed evolution of APEX2 for electron microscopy and proximity labeling.</strong></div><div><em>Nat Methods.</em> 2015; <strong>12</strong>:51-54</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.3179" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.046&amp;key=2-s2.0-84926166583" target="_blank" aria-label="Open Scopus in new tab">Scopus (853)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25419960/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.3179&amp;pmid=25419960" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib31" class="citations"><div id="sref31" class="citation"><div class="citation-content"><div class="label">33.</div><div>Li, P. ∙ Banjade, S. ∙ Cheng, H.C. ...</div><div><strong>Phase transitions in the assembly of multivalent signalling proteins</strong></div><div><em>Nature.</em> 2012; <strong>483</strong>:336-340</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature10879" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.046&amp;key=2-s2.0-84862776582" target="_blank" aria-label="Open Scopus in new tab">Scopus (1681)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22398450/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature10879&amp;pmid=22398450" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib32" class="citations"><div id="sref32" class="citation"><div class="citation-content"><div class="label">34.</div><div>Lin, Y. ∙ Protter, D.S. ∙ Rosen, M.K. ...</div><div><strong>Formation and maturation of phase-separated liquid droplets by RNA-binding proteins</strong></div><div><em>Mol. Cell.</em> 2015; <strong>60</strong>:208-219</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.03.046&amp;key=10.1016%2Fj.molcel.2015.08.018&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.03.046&amp;key=10.1016%2Fj.molcel.2015.08.018&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.046&amp;key=2-s2.0-84944884978" target="_blank" aria-label="Open Scopus in new tab">Scopus (1097)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26412307/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.molcel.2015.08.018&amp;pmid=26412307" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib33" class="citations"><div id="sref33" class="citation"><div class="citation-content"><div class="label">35.</div><div>Mackenzie, I.R. ∙ Nicholson, A.M. ∙ Sarkar, M. ...</div><div><strong>TIA1 mutations in amyotrophic lateral sclerosis and frontotemporal dementia promote phase separation and alter stress granule dynamics</strong></div><div><em>Neuron.</em> 2017; <strong>95</strong>:808-816.e9</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.03.046&amp;key=10.1016%2Fj.neuron.2017.07.025&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.03.046&amp;key=10.1016%2Fj.neuron.2017.07.025&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.046&amp;key=2-s2.0-85027355567" target="_blank" aria-label="Open Scopus in new tab">Scopus (440)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28817800/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.neuron.2017.07.025&amp;pmid=28817800" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib34" class="citations"><div id="sref34" class="citation"><div class="citation-content"><div class="label">36.</div><div>Markmiller, S. ∙ Soltanieh, S. ∙ Server, K.L. ...</div><div><strong>Context-dependent and disease-specific diversity in protein interactions within stress granules</strong></div><div><em>Cell.</em> 2018; <strong>172</strong>:590-604.e13</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.03.046&amp;key=10.1016%2Fj.cell.2017.12.032&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.03.046&amp;key=10.1016%2Fj.cell.2017.12.032&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.046&amp;key=2-s2.0-85041692639" target="_blank" aria-label="Open Scopus in new tab">Scopus (555)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29373831/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2017.12.032&amp;pmid=29373831" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib35" class="citations"><div id="sref35" class="citation"><div class="citation-content"><div class="label">37.</div><div>Martin, E.W. ∙ Holehouse, A.S. ∙ Grace, C.R. ...</div><div><strong>Sequence determinants of the conformational properties of an intrinsically disordered protein prior to and upon multisite phosphorylation</strong></div><div><em>J.&nbsp;Am. Chem. Soc.</em> 2016; <strong>138</strong>:15323-15335</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/jacs.6b10272" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.046&amp;key=2-s2.0-84999751284" target="_blank" aria-label="Open Scopus in new tab">Scopus (174)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27807972/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Fjacs.6b10272&amp;pmid=27807972" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib36" class="citations"><div id="sref36" class="citation"><div class="citation-content"><div class="label">38.</div><div>Martin, S. ∙ Bellora, N. ∙ González-Vallinas, J. ...</div><div><strong>Preferential binding of a stable G3BP ribonucleoprotein complex to intron-retaining transcripts in mouse brain and modulation of their expression in the cerebellum</strong></div><div><em>J.&nbsp;Neurochem.</em> 2016; <strong>139</strong>:349-368</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1111/jnc.13768" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_38_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.046&amp;key=2-s2.0-84991716621" target="_blank" aria-label="Open Scopus in new tab">Scopus (17)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27513819/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1111%2Fjnc.13768&amp;pmid=27513819" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib37" class="citations"><div id="sref37" class="citation"><div class="citation-content"><div class="label">39.</div><div>Marzahn, M.R. ∙ Marada, S. ∙ Lee, J. ...</div><div><strong>Higher-order oligomerization promotes localization of SPOP to liquid nuclear speckles</strong></div><div><em>EMBO J.</em> 2016; <strong>35</strong>:1254-1275</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.15252/embj.201593169" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.046&amp;key=2-s2.0-84971219304" target="_blank" aria-label="Open Scopus in new tab">Scopus (153)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27220849/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.15252%2Fembj.201593169&amp;pmid=27220849" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib38" class="citations"><div id="sref38" class="citation"><div class="citation-content"><div class="label">40.</div><div>Meyer, C. ∙ Garzia, A. ∙ Mazzola, M. ...</div><div><strong>The TIA1 RNA-binding protein family regulates EIF2AK2-mediated stress response and cell cycle progression</strong></div><div><em>Mol. Cell.</em> 2018; <strong>69</strong>:622-635.e6</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_40_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.03.046&amp;key=10.1016%2Fj.molcel.2018.01.011&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_40_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.03.046&amp;key=10.1016%2Fj.molcel.2018.01.011&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_40_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.046&amp;key=2-s2.0-85041696902" target="_blank" aria-label="Open Scopus in new tab">Scopus (78)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29429924/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.molcel.2018.01.011&amp;pmid=29429924" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib39" class="citations"><div id="sref39" class="citation"><div class="citation-content"><div class="label">41.</div><div>Molliex, A. ∙ Temirov, J. ∙ Lee, J. ...</div><div><strong>Phase separation by low complexity domains promotes stress granule assembly and drives pathological fibrillization</strong></div><div><em>Cell.</em> 2015; <strong>163</strong>:123-133</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.03.046&amp;key=10.1016%2Fj.cell.2015.09.015&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.03.046&amp;key=10.1016%2Fj.cell.2015.09.015&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.046&amp;key=2-s2.0-84944907005" target="_blank" aria-label="Open Scopus in new tab">Scopus (1752)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26406374/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2015.09.015&amp;pmid=26406374" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib40" class="citations"><div id="sref40" class="citation"><div class="citation-content"><div class="label">42.</div><div>Nedelsky, N.B. ∙ Taylor, J.P.</div><div><strong>Bridging biophysics and neurology: aberrant phase transitions in neurodegenerative disease</strong></div><div><em>Nat. Rev. Neurol.</em> 2019; <strong>15</strong>:272-286</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41582-019-0157-5" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.046&amp;key=2-s2.0-85063287030" target="_blank" aria-label="Open Scopus in new tab">Scopus (125)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30890779/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41582-019-0157-5&amp;pmid=30890779" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib41" class="citations"><div id="sref41" class="citation"><div class="citation-content"><div class="label">43.</div><div>Nott, T.J. ∙ Petsalaki, E. ∙ Farber, P. ...</div><div><strong>Phase transition of a disordered nuage protein generates environmentally responsive membraneless organelles</strong></div><div><em>Mol. Cell.</em> 2015; <strong>57</strong>:936-947</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.03.046&amp;key=10.1016%2Fj.molcel.2015.01.013&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.03.046&amp;key=10.1016%2Fj.molcel.2015.01.013&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.046&amp;key=2-s2.0-84923878820" target="_blank" aria-label="Open Scopus in new tab">Scopus (1217)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25747659/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.molcel.2015.01.013&amp;pmid=25747659" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib42" class="citations"><div id="sref42" class="citation"><div class="citation-content"><div class="label">44.</div><div>Ohn, T. ∙ Kedersha, N. ∙ Hickman, T. ...</div><div><strong>A functional RNAi screen links O-GlcNAc modification of ribosomal proteins to stress granule and processing body assembly</strong></div><div><em>Nat. Cell Biol.</em> 2008; <strong>10</strong>:1224-1231</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ncb1783" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.046&amp;key=2-s2.0-53349165578" target="_blank" aria-label="Open Scopus in new tab">Scopus (328)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/18794846/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fncb1783&amp;pmid=18794846" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib80" class="citations"><div id="sref80" class="citation"><div class="citation-content"><div class="label">45.</div><div>Ong, S.S. ∙ Goktug, A.N. ∙ Elias, A. ...</div><div><strong>Stability of the human pregnane X receptor is regulated by E3 ligase UBR5 and serine/threonine kinase DYRK2</strong></div><div><em>Biochem J.</em> 2014; <strong>459</strong>:193-203</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1042/BJ20130558" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_45_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.046&amp;key=2-s2.0-84896135872" target="_blank" aria-label="Open Scopus in new tab">Scopus (36)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24438055/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1042%2FBJ20130558&amp;pmid=24438055" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib43" class="citations"><div id="sref43" class="citation"><div class="citation-content"><div class="label">46.</div><div>Onomoto, K. ∙ Jogi, M. ∙ Yoo, J.S. ...</div><div><strong>Critical role of an antiviral stress granule containing RIG-I and PKR in viral detection and innate immunity</strong></div><div><em>PLoS ONE.</em> 2012; <strong>7</strong>:e43031</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pone.0043031" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.046&amp;key=2-s2.0-84865060036" target="_blank" aria-label="Open Scopus in new tab">Scopus (296)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22912779/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pone.0043031&amp;pmid=22912779" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib44" class="citations"><div id="sref44" class="citation"><div class="citation-content"><div class="label">47.</div><div>Panas, M.D. ∙ Schulte, T. ∙ Thaa, B. ...</div><div><strong>Viral and cellular proteins containing FGDF motifs bind G3BP to block stress granule formation</strong></div><div><em>PLoS Pathog.</em> 2015; <strong>11</strong>:e1004659</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.ppat.1004659" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.046&amp;key=2-s2.0-84924405914" target="_blank" aria-label="Open Scopus in new tab">Scopus (111)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25658430/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.ppat.1004659&amp;pmid=25658430" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib45" class="citations"><div id="sref45" class="citation"><div class="citation-content"><div class="label">48.</div><div>Panas, M.D. ∙ Ivanov, P. ∙ Anderson, P.</div><div><strong>Mechanistic insights into mammalian stress granule dynamics</strong></div><div><em>J.&nbsp;Cell Biol.</em> 2016; <strong>215</strong>:313-323</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1083/jcb.201609081" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_48_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.046&amp;key=2-s2.0-84995977436" target="_blank" aria-label="Open Scopus in new tab">Scopus (266)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27821493/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1083%2Fjcb.201609081&amp;pmid=27821493" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib46" class="citations"><div id="sref46" class="citation"><div class="citation-content"><div class="label">49.</div><div>Panas, M.D. ∙ Kedersha, N. ∙ Schulte, T. ...</div><div><strong>Phosphorylation of G3BP1-S149 does not influence stress granule assembly</strong></div><div><em>J.&nbsp;Cell Biol.</em> 2019; <strong>218</strong>:2425-2432</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1083/jcb.201801214" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.046&amp;key=2-s2.0-85069266978" target="_blank" aria-label="Open Scopus in new tab">Scopus (34)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31171631/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1083%2Fjcb.201801214&amp;pmid=31171631" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib47" class="citations"><div id="sref47" class="citation"><div class="citation-content"><div class="label">50.</div><div>Patel, A. ∙ Lee, H.O. ∙ Jawerth, L. ...</div><div><strong>A liquid-to-solid phase transition of the ALS protein FUS accelerated by disease mutation</strong></div><div><em>Cell.</em> 2015; <strong>162</strong>:1066-1077</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.03.046&amp;key=10.1016%2Fj.cell.2015.07.047&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.03.046&amp;key=10.1016%2Fj.cell.2015.07.047&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.046&amp;key=2-s2.0-84940403835" target="_blank" aria-label="Open Scopus in new tab">Scopus (1868)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26317470/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2015.07.047&amp;pmid=26317470" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib48" class="citations"><div id="sref48" class="citation"><div class="citation-content"><div class="label">51.</div><div>Posey, A.E. ∙ Ruff, K.M. ∙ Harmon, T.S. ...</div><div><strong>Profilin reduces aggregation and phase separation of huntingtin N-terminal fragments by preferentially binding to soluble monomers and oligomers</strong></div><div><em>J.&nbsp;Biol. Chem.</em> 2018; <strong>293</strong>:3734-3746</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_51_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.03.046&amp;key=10.1074%2Fjbc.RA117.000357&amp;cf=fulltext&amp;site=jbc-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_51_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.03.046&amp;key=10.1074%2Fjbc.RA117.000357&amp;cf=pdf&amp;site=jbc-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_51_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.046&amp;key=2-s2.0-85043722258" target="_blank" aria-label="Open Scopus in new tab">Scopus (84)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29358329/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1074%2Fjbc.RA117.000357&amp;pmid=29358329" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib49" class="citations"><div id="sref49" class="citation"><div class="citation-content"><div class="label">52.</div><div>Ran, F.A. ∙ Hsu, P.D. ∙ Wright, J. ...</div><div><strong>Genome engineering using the CRISPR-Cas9 system</strong></div><div><em>Nat. Protoc.</em> 2013; <strong>8</strong>:2281-2308</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nprot.2013.143" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.046&amp;key=2-s2.0-84887010498" target="_blank" aria-label="Open Scopus in new tab">Scopus (7792)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24157548/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnprot.2013.143&amp;pmid=24157548" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib50" class="citations"><div id="sref50" class="citation"><div class="citation-content"><div class="label">53.</div><div>Rollins, C.T. ∙ Rivera, V.M. ∙ Woolfson, D.N. ...</div><div><strong>A ligand-reversible dimerization system for controlling protein-protein interactions</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2000; <strong>97</strong>:7096-7101</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.100101997" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.046&amp;key=2-s2.0-12944311043" target="_blank" aria-label="Open Scopus in new tab">Scopus (156)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/10852943/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.100101997&amp;pmid=10852943" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib51" class="citations"><div id="sref51" class="citation"><div class="citation-content"><div class="label">54.</div><div>Sahoo, P.K. ∙ Lee, S.J. ∙ Jaiswal, P.B. ...</div><div><strong>Axonal G3BP1 stress granule protein limits axonal mRNA translation and nerve regeneration</strong></div><div><em>Nat. Commun.</em> 2018; <strong>9</strong>:3358</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41467-018-05647-x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.046&amp;key=2-s2.0-85052088342" target="_blank" aria-label="Open Scopus in new tab">Scopus (104)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30135423/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41467-018-05647-x&amp;pmid=30135423" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib52" class="citations"><div id="sref52" class="citation"><div class="citation-content"><div class="label">55.</div><div>Samir, P. ∙ Kesavardhana, S. ∙ Patmore, D.M. ...</div><div><strong>DDX3X acts as a live-or-die checkpoint in stressed cells by regulating NLRP3 inflammasome</strong></div><div><em>Nature.</em> 2019; <strong>573</strong>:590-594</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-1551-2" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.046&amp;key=2-s2.0-85072687312" target="_blank" aria-label="Open Scopus in new tab">Scopus (256)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31511697/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-1551-2&amp;pmid=31511697" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib79" class="citations"><div id="sref79" class="citation"><div class="citation-content"><div class="label">56.</div><div>Sanders, D.W. ∙ Kedersha, N. ∙ Lee, D.S.W. ...</div><div><strong>Competing Protein-RNA Interaction Networks Control Multiphase Intracellular Organization</strong></div><div><em>Cell.</em> 2020; <strong>181</strong>:306-324</div><div>this issue</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_56_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.03.046&amp;key=10.1016%2Fj.cell.2020.03.050&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_56_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.03.046&amp;key=10.1016%2Fj.cell.2020.03.050&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_56_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.046&amp;key=2-s2.0-85083031388" target="_blank" aria-label="Open Scopus in new tab">Scopus (446)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32302570/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2020.03.050&amp;pmid=32302570" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib53" class="citations"><div id="sref53" class="citation"><div class="citation-content"><div class="label">57.</div><div>Schuck, P.</div><div><strong>Size-distribution analysis of macromolecules by sedimentation velocity ultracentrifugation and lamm equation modeling</strong></div><div><em>Biophys. J.</em> 2000; <strong>78</strong>:1606-1619</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.03.046&amp;key=10.1016%2FS0006-3495%2800%2976713-0&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.03.046&amp;key=10.1016%2FS0006-3495%2800%2976713-0&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.046&amp;key=2-s2.0-0034009520" target="_blank" aria-label="Open Scopus in new tab">Scopus (3271)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/10692345/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0006-3495%2800%2976713-0&amp;pmid=10692345" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib54" class="citations"><div id="sref54" class="citation"><div class="citation-content"><div class="label">58.</div><div>Shannon, P. ∙ Markiel, A. ∙ Ozier, O. ...</div><div><strong>Cytoscape: a software environment for integrated models of biomolecular interaction networks</strong></div><div><em>Genome Res.</em> 2003; <strong>13</strong>:2498-2504</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/gr.1239303" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.046&amp;key=2-s2.0-0242490780" target="_blank" aria-label="Open Scopus in new tab">Scopus (32549)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/14597658/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2Fgr.1239303&amp;pmid=14597658" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib55" class="citations"><div id="sref55" class="citation"><div class="citation-content"><div class="label">59.</div><div>Szklarczyk, D. ∙ Franceschini, A. ∙ Wyder, S. ...</div><div><strong>STRING v10: protein-protein interaction networks, integrated over the tree of life</strong></div><div><em>Nucleic Acids Res.</em> 2015; <strong>43</strong>:D447-D452</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gku1003" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.046&amp;key=2-s2.0-84941051170" target="_blank" aria-label="Open Scopus in new tab">Scopus (7758)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25352553/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgku1003&amp;pmid=25352553" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib56" class="citations"><div id="sref56" class="citation"><div class="citation-content"><div class="label">60.</div><div>Thedieck, K. ∙ Holzwarth, B. ∙ Prentzell, M.T. ...</div><div><strong>Inhibition of mTORC1 by astrin and stress granules prevents apoptosis in cancer cells</strong></div><div><em>Cell.</em> 2013; <strong>154</strong>:859-874</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.03.046&amp;key=10.1016%2Fj.cell.2013.07.031&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.03.046&amp;key=10.1016%2Fj.cell.2013.07.031&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.046&amp;key=2-s2.0-84882800242" target="_blank" aria-label="Open Scopus in new tab">Scopus (224)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23953116/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2013.07.031&amp;pmid=23953116" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib57" class="citations"><div id="sref57" class="citation"><div class="citation-content"><div class="label">61.</div><div>Tourrière, H. ∙ Chebli, K. ∙ Zekri, L. ...</div><div><strong>The RasGAP-associated endoribonuclease G3BP assembles stress granules</strong></div><div><em>J.&nbsp;Cell Biol.</em> 2003; <strong>160</strong>:823-831</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1083/jcb.200212128" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.046&amp;key=2-s2.0-0037451173" target="_blank" aria-label="Open Scopus in new tab">Scopus (719)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/12642610/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1083%2Fjcb.200212128&amp;pmid=12642610" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib58" class="citations"><div id="sref58" class="citation"><div class="citation-content"><div class="label">62.</div><div>Tsai, W.C. ∙ Lloyd, R.E.</div><div><strong>Cytoplasmic RNA granules and viral infection</strong></div><div><em>Annu. Rev. Virol.</em> 2014; <strong>1</strong>:147-170</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1146/annurev-virology-031413-085505" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.046&amp;key=2-s2.0-85016938715" target="_blank" aria-label="Open Scopus in new tab">Scopus (82)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26958719/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1146%2Fannurev-virology-031413-085505&amp;pmid=26958719" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib59" class="citations"><div id="sref59" class="citation"><div class="citation-content"><div class="label">63.</div><div>Tsai, W.C. ∙ Gayatri, S. ∙ Reineke, L.C. ...</div><div><strong>Arginine demethylation of G3BP1 promotes stress granule assembly</strong></div><div><em>J.&nbsp;Biol. Chem.</em> 2016; <strong>291</strong>:22671-22685</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1074/jbc.M116.739573" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.046&amp;key=2-s2.0-84992345066" target="_blank" aria-label="Open Scopus in new tab">Scopus (140)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27601476/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1074%2Fjbc.M116.739573&amp;pmid=27601476" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib60" class="citations"><div id="sref60" class="citation"><div class="citation-content"><div class="label">64.</div><div>Valentin-Vega, Y.A. ∙ Wang, Y.D. ∙ Parker, M. ...</div><div><strong>Cancer-associated DDX3X mutations drive stress granule assembly and impair global translation</strong></div><div><em>Sci. Rep.</em> 2016; <strong>6</strong>:25996</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/srep25996" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_64_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.046&amp;key=2-s2.0-84968754575" target="_blank" aria-label="Open Scopus in new tab">Scopus (111)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27180681/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fsrep25996&amp;pmid=27180681" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib61" class="citations"><div id="sref61" class="citation"><div class="citation-content"><div class="label">65.</div><div>Van Treeck, B. ∙ Parker, R.</div><div><strong>Emerging roles for intermolecular RNA-RNA interactions in RNP assemblies</strong></div><div><em>Cell.</em> 2018; <strong>174</strong>:791-802</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.03.046&amp;key=10.1016%2Fj.cell.2018.07.023&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.03.046&amp;key=10.1016%2Fj.cell.2018.07.023&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.046&amp;key=2-s2.0-85050892341" target="_blank" aria-label="Open Scopus in new tab">Scopus (259)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30096311/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2018.07.023&amp;pmid=30096311" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib62" class="citations"><div id="sref62" class="citation"><div class="citation-content"><div class="label">66.</div><div>Van Treeck, B. ∙ Protter, D.S.W. ∙ Matheny, T. ...</div><div><strong>RNA self-assembly contributes to stress granule formation and defining the stress granule transcriptome</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2018; <strong>115</strong>:2734-2739</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1800038115" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.046&amp;key=2-s2.0-85043765662" target="_blank" aria-label="Open Scopus in new tab">Scopus (326)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29483269/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1800038115&amp;pmid=29483269" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib63" class="citations"><div id="sref63" class="citation"><div class="citation-content"><div class="label">67.</div><div>Vognsen, T. ∙ Møller, I.R. ∙ Kristensen, O.</div><div><strong>Crystal structures of the human G3BP1 NTF2-like domain visualize FxFG Nup repeat specificity</strong></div><div><em>PLoS ONE.</em> 2013; <strong>8</strong>:e80947</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pone.0080947" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_67_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.046&amp;key=2-s2.0-84891898406" target="_blank" aria-label="Open Scopus in new tab">Scopus (38)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24324649/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pone.0080947&amp;pmid=24324649" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib64" class="citations"><div id="sref64" class="citation"><div class="citation-content"><div class="label">68.</div><div>Wasserman, S. ∙ Faust, K.</div><div><strong>Social Network Analysis: Methods and Applications</strong></div><div>Cambridge University, 1994</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1017/CBO9780511815478" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1017%2FCBO9780511815478" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib65" class="citations"><div id="sref65" class="citation"><div class="citation-content"><div class="label">69.</div><div>Waterhouse, A. ∙ Bertoni, M. ∙ Bienert, S. ...</div><div><strong>SWISS-MODEL: homology modelling of protein structures and complexes</strong></div><div><em>Nucleic Acids Res.</em> 2018; <strong>46</strong>:W296-W303</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gky427" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.046&amp;key=2-s2.0-85050235281" target="_blank" aria-label="Open Scopus in new tab">Scopus (7778)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29788355/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgky427&amp;pmid=29788355" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib66" class="citations"><div id="sref66" class="citation"><div class="citation-content"><div class="label">70.</div><div>Wippich, F. ∙ Bodenmiller, B. ∙ Trajkovska, M.G. ...</div><div><strong>Dual specificity kinase DYRK3 couples stress granule condensation/dissolution to mTORC1 signaling</strong></div><div><em>Cell.</em> 2013; <strong>152</strong>:791-805</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.03.046&amp;key=10.1016%2Fj.cell.2013.01.033&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.03.046&amp;key=10.1016%2Fj.cell.2013.01.033&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.046&amp;key=2-s2.0-84874040052" target="_blank" aria-label="Open Scopus in new tab">Scopus (448)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23415227/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2013.01.033&amp;pmid=23415227" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib67" class="citations"><div id="sref67" class="citation"><div class="citation-content"><div class="label">71.</div><div>Wu, B. ∙ Su, S. ∙ Patil, D.P. ...</div><div><strong>Molecular basis for the specific and multivariant recognitions of RNA substrates by human hnRNP A2/B1</strong></div><div><em>Nat. Commun.</em> 2018; <strong>9</strong>:420</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41467-017-02770-z" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.046&amp;key=2-s2.0-85041369833" target="_blank" aria-label="Open Scopus in new tab">Scopus (282)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29379020/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41467-017-02770-z&amp;pmid=29379020" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib68" class="citations"><div id="sref68" class="citation"><div class="citation-content"><div class="label">72.</div><div>Youn, J.Y. ∙ Dunham, W.H. ∙ Hong, S.J. ...</div><div><strong>High-density proximity mapping reveals the subcellular organization of mRNA-associated granules and bodies</strong></div><div><em>Mol. Cell.</em> 2018; <strong>69</strong>:517-532.e11</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.03.046&amp;key=10.1016%2Fj.molcel.2017.12.020&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.03.046&amp;key=10.1016%2Fj.molcel.2017.12.020&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.046&amp;key=2-s2.0-85041119474" target="_blank" aria-label="Open Scopus in new tab">Scopus (466)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29395067/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.molcel.2017.12.020&amp;pmid=29395067" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib69" class="citations"><div id="sref69" class="citation"><div class="citation-content"><div class="label">73.</div><div>Zhang, X.D.</div><div><strong>A new method with flexible and balanced control of false negatives and false positives for hit selection in RNA interference high-throughput screening assays</strong></div><div><em>J.&nbsp;Biomol. Screen.</em> 2007; <strong>12</strong>:645-655</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1177/1087057107300645" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.046&amp;key=2-s2.0-34547809341" target="_blank" aria-label="Open Scopus in new tab">Scopus (70)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17517904/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1177%2F1087057107300645&amp;pmid=17517904" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib70" class="citations"><div id="sref70" class="citation"><div class="citation-content"><div class="label">74.</div><div>Zhang, X.D.</div><div><strong>Illustration of SSMD, z score, SSMD<sup>∗</sup>, z<sup>∗</sup> score, and t statistic for hit selection in RNAi high-throughput screens</strong></div><div><em>J.&nbsp;Biomol. Screen.</em> 2011; <strong>16</strong>:775-785</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1177/1087057111405851" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.046&amp;key=2-s2.0-79960924226" target="_blank" aria-label="Open Scopus in new tab">Scopus (144)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21515799/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1177%2F1087057111405851&amp;pmid=21515799" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib71" class="citations"><div id="sref71" class="citation"><div class="citation-content"><div class="label">75.</div><div>Zhang, B. ∙ Horvath, S.</div><div><strong>A general framework for weighted gene co-expression network analysis</strong></div><div><em>Stat. Appl. Genet. Mol. Biol.</em> 2005; <strong>4</strong>:Article17</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.2202/1544-6115.1128" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_75_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.046&amp;key=2-s2.0-23944458138" target="_blank" aria-label="Open Scopus in new tab">Scopus (3978)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16646834/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.2202%2F1544-6115.1128&amp;pmid=16646834" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib72" class="citations"><div id="sref72" class="citation"><div class="citation-content"><div class="label">76.</div><div>Zhang, X.D. ∙ Yang, X.C. ∙ Chung, N. ...</div><div><strong>Robust statistical methods for hit selection in RNA interference high-throughput screening experiments</strong></div><div><em>Pharmacogenomics.</em> 2006; <strong>7</strong>:299-309</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.2217/14622416.7.3.299" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_76_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.046&amp;key=2-s2.0-33646034336" target="_blank" aria-label="Open Scopus in new tab">Scopus (81)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16610941/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.2217%2F14622416.7.3.299&amp;pmid=16610941" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib73" class="citations"><div id="sref73" class="citation"><div class="citation-content"><div class="label">77.</div><div>Zhang, K. ∙ Daigle, J.G. ∙ Cunningham, K.M. ...</div><div><strong>Stress granule assembly disrupts nucleocytoplasmic transport</strong></div><div><em>Cell.</em> 2018; <strong>173</strong>:958-971.e17</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.03.046&amp;key=10.1016%2Fj.cell.2018.03.025&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.03.046&amp;key=10.1016%2Fj.cell.2018.03.025&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.046&amp;key=2-s2.0-85044766434" target="_blank" aria-label="Open Scopus in new tab">Scopus (255)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29628143/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2018.03.025&amp;pmid=29628143" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib74" class="citations"><div id="sref74" class="citation"><div class="citation-content"><div class="label">78.</div><div>Zhao, H. ∙ Brautigam, C.A. ∙ Ghirlando, R. ...</div><div><strong>Overview of current methods in sedimentation velocity and sedimentation equilibrium analytical ultracentrifugation</strong></div><div><em>Curr. Protoc. Protein Sci.</em> 2013; <strong>Chapter 20</strong></div><div>Unit20.12</div></div><div class="external-links"><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23377850/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?pmid=23377850" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib75" class="citations"><div id="sref75" class="citation"><div class="citation-content"><div class="label">79.</div><div>Zhao, H. ∙ Ghirlando, R. ∙ Alfonso, C. ...</div><div><strong>A multilaboratory comparison of calibration accuracy and the performance of external references in analytical ultracentrifugation</strong></div><div><em>PLoS ONE.</em> 2015; <strong>10</strong>:e0126420</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pone.0126420" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_79_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.046&amp;key=2-s2.0-84930615100" target="_blank" aria-label="Open Scopus in new tab">Scopus (79)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25997164/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pone.0126420&amp;pmid=25997164" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib76" class="citations"><div id="sref76" class="citation"><div class="citation-content"><div class="label">80.</div><div>Zhao, B.S. ∙ Roundtree, I.A. ∙ He, C.</div><div><strong>Post-transcriptional gene regulation by mRNA modifications</strong></div><div><em>Nat. Rev. Mol. Cell Biol.</em> 2017; <strong>18</strong>:31-42</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nrm.2016.132" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_80_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.03.046&amp;key=2-s2.0-84994108096" target="_blank" aria-label="Open Scopus in new tab">Scopus (1528)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27808276/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnrm.2016.132&amp;pmid=27808276" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div></div></div></section></div></section><div class="core-linked-content">
    

        


    

            

            

            <div data-widget-id="8633d695-b258-4908-b86f-e2c268ec3c6b" data-widget-name="UX3HTMLWidget"><script type="text/javascript">
if($('.content-navigation__body__title').length){
$('.content-navigation__body__title').each(function(){
var html=this.innerHTML;
var div = document.createElement("div");
div.innerHTML = html;
var text = div.textContent || div.innerText || "";
this.innerHTML=text;
});
}
</script></div>

            

        

    

        


    

            

            

            <div data-widget-id="393effff-5b44-4681-a9b4-4f029b3540b7" data-widget-name="UX3HTMLWidget"><style>
.linkback-ad .pb-ad {
    display: block;
   height: auto;
    margin-left: 0px;
    margin-right: auto;
}
</style></div>

            

        

    

        

</div></div></div><div class="core-collateral-aside"><section id="core-collateral-figures"></section>
    

        


    

            

            

            <section id="core-collateral-metrics" class="publication-list__axel-section"><h2>Article metrics</h2><div><ul class="publication-list rlist publication-list--list plum-x--lazy-load publication-list--axel js--loading" data-plum-x-url="/pb/widgets/ux3/publicationList/plumx?widgetId=66dd7e11-7768-4006-93d8-a72b60e2a264&amp;pbContext=%3Bproduct%3Aproduct%3Aelsevier%5C%3Aproduct%5C%3Acell%3Bwebsite%3Awebsite%3Acell-site%3Bctype%3Astring%3AJournal+Content%3Bjournal%3Ajournal%3Acell%3BpageGroup%3Astring%3AArticle+View%3Bpage%3Astring%3AShow+Full+Text%3BrequestedJournal%3Ajournal%3Acell%3Barticle%3Aarticle%3Apii%5C%3AS0092867420303391%3Bwgroup%3Astring%3ADefault+Website+Group%3BsubPage%3Astring%3AFull+Text%3Bissue%3Aissue%3Apii%5C%3AS0092867419X00094" data-pb-mode-enabled="false"><li aria-label="loading articles" role="alert" class="js--loading plum-x--spinner"></li></ul></div></section>

            

        
<section id="core-collateral-supplementary"></section>
    

        


    

            

            

            <section id="core-collateral-relatedArticles" class="axel-related-articles"><h2 id="relatedArticles" class="top axel-related-articles__title" data-title="Related articles">Related Articles</h2><div data-pb-widget-type="ux3-relatedArticles" data-widget-id="8d3844d1-450e-497c-ab75-a63fc41750ac" data-widget-name="AtyponTempUX3RelatedArticlesWidget" data-article-pii="S0092867420303391" class="ux3DeferredOnScrollWidget axel-related-articles__body" data-show-more="true"></div></section>

            

        
</div></div><aside data-core-aside="right-rail"><div class="inner-wrapper">

    

        


    

            

            

            <div data-widget-id="dca7470d-9085-4d21-a6f4-a66397718146" data-widget-name="UX3HTMLWidget"><input type="hidden" id="gptPage" value="cell-videos"></div>

            

        











        <div data-widget-def="literatumAd" data-widget-id="481d10c2-bbe0-470e-a5ba-207a81ecc8dc" class="adplaceholders" style="right: 0px; position: static;">
        
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd alignCenter  widget-verticalAd" id="481d10c2-bbe0-470e-a5ba-207a81ecc8dc">
                
                
                    
                

                <div class="wrapped " id="Marketplace-Recs">
                    
                    <div class="widget-body body body-verticalAd "><div class="verticalAdContainer">
    <div class="verticalAdLabelContainer">
        <div class="verticalAdLabel">
                <img src="/templates/jsp/_style2/_marlin/images/bg_adVert.gif" alt="Advertisement">
        </div>
    </div>
    <div class="verticalAd">
        





        <div class="pb-ad">
            <script src="//cdn.pbgrd.com/core-cell.js" async=""></script>
<div id="pbgrd-skyr-c"><div id="pbgrd-skyr" style="width: inherit; position: relative; background: transparent;"></div></div>
        </div>
    

    </div>
</div></div>
                </div>
                </div>

        

        </div>
    

</div></aside><div class="core-authors-details" style="display:none"><section class="core-authors"><div id="au1" property="author" typeof="Person"><div class="heading"><span property="givenName">Peiguo</span> <span property="familyName">Yang</span></div><div class="content"><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">7</div><div id="fn1"><div id="ntpara0010" role="paragraph">These authors contributed equally</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Cell and Molecular Biology, St. Jude Children’s Research Hospital, Memphis, TN 38105, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Yang/Peiguo" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><div id="au2" property="author" typeof="Person"><div class="heading"><span property="givenName">Cécile</span> <span property="familyName">Mathieu</span></div><div class="content"><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">7</div><div id="fn1"><div id="ntpara0010" role="paragraph">These authors contributed equally</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Cell and Molecular Biology, St. Jude Children’s Research Hospital, Memphis, TN 38105, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Howard Hughes Medical Institute, Chevy Chase, MD 20815, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Mathieu/C%C3%A9cile" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><div id="au3" property="author" typeof="Person"><div class="heading"><span property="givenName">Regina-Maria</span> <span property="familyName">Kolaitis</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Cell and Molecular Biology, St. Jude Children’s Research Hospital, Memphis, TN 38105, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Kolaitis/Regina-Maria" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><div id="au4" property="author" typeof="Person"><div class="heading"><span property="givenName">Peipei</span> <span property="familyName">Zhang</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Cell and Molecular Biology, St. Jude Children’s Research Hospital, Memphis, TN 38105, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Zhang/Peipei" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><div id="au5" property="author" typeof="Person"><div class="heading"><span property="givenName">James</span> <span property="familyName">Messing</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Cell and Molecular Biology, St. Jude Children’s Research Hospital, Memphis, TN 38105, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Howard Hughes Medical Institute, Chevy Chase, MD 20815, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Messing/James" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><div id="au6" property="author" typeof="Person"><div class="heading"><span property="givenName">Ugur</span> <span property="familyName">Yurtsever</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Cell and Molecular Biology, St. Jude Children’s Research Hospital, Memphis, TN 38105, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Graduate School of Structure and Dynamics of Living Systems, Université Paris-Saclay, Gif-sur-Yvette, 91190, France</span></div></div><div class="core-author-link"><a href="/authored-by/Yurtsever/Ugur" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><div id="au7" property="author" typeof="Person"><div class="heading"><span property="givenName">Zemin</span> <span property="familyName">Yang</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Cell and Molecular Biology, St. Jude Children’s Research Hospital, Memphis, TN 38105, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Yang/Zemin" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><div id="au8" property="author" typeof="Person"><div class="heading"><span property="givenName">Jinjun</span> <span property="familyName">Wu</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Cell and Molecular Biology, St. Jude Children’s Research Hospital, Memphis, TN 38105, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Wu/Jinjun" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><div id="au9" property="author" typeof="Person"><div class="heading"><span property="givenName">Yuxin</span> <span property="familyName">Li</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Center for Proteomics and Metabolomics, St. Jude Children’s Research Hospital, Memphis, TN 38105, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Li/Yuxin" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><div id="au10" property="author" typeof="Person"><div class="heading"><span property="givenName">Qingfei</span> <span property="familyName">Pan</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Computational Biology, St. Jude Children’s Research Hospital, Memphis, TN 38105, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Pan/Qingfei" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><div id="au11" property="author" typeof="Person"><div class="heading"><span property="givenName">Jiyang</span> <span property="familyName">Yu</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Computational Biology, St. Jude Children’s Research Hospital, Memphis, TN 38105, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Yu/Jiyang" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><div id="au12" property="author" typeof="Person"><div class="heading"><span property="givenName">Erik W.</span> <span property="familyName">Martin</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Structural Biology, St. Jude Children’s Research Hospital, Memphis, TN 38105, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Martin/Erik+W" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><div id="au13" property="author" typeof="Person"><div class="heading"><span property="givenName">Tanja</span> <span property="familyName">Mittag</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Structural Biology, St. Jude Children’s Research Hospital, Memphis, TN 38105, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Mittag/Tanja" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><div id="au14" property="author" typeof="Person"><div class="heading"><span property="givenName">Hong Joo</span> <span property="familyName">Kim</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Cell and Molecular Biology, St. Jude Children’s Research Hospital, Memphis, TN 38105, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Kim/Hong+Joo" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><div id="au15" property="author" typeof="Person"><div class="heading"><span property="givenName">J. Paul</span> <span property="familyName">Taylor</span></div><div class="content"><div class="core-author-emails"><div class="label">Correspondence</div><div>Corresponding author</div><div><a id="eadd0010" href="mailto:jpaul.taylor@stjude.org" property="email" title="Send email to J. Paul Taylor">jpaul.taylor@stjude.org</a></div></div><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">8</div><div id="fn2"><div id="ntpara0015" role="paragraph">Lead Contact</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Cell and Molecular Biology, St. Jude Children’s Research Hospital, Memphis, TN 38105, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Howard Hughes Medical Institute, Chevy Chase, MD 20815, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Taylor/J+Paul" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div></section></div></article><div data-extent="article-wrapper"><div class="core-container"></div></div><script id="axel-publication-metadata" type="application/json">{"doi":"10.1016/j.cell.2020.03.046","data-is-retraction":false,"data-is-retired":false}
</script><script type="text/javascript" data-ot-ignore="data-ot-ignore" defer="defer" src="https://static.addtoany.com/menu/page.js" class="optanon-category-C0004 ot-vscat-C0004"></script><template id="figure_nav_template"><a href="#" class="open-in-viewer btn btn-dark figure-pop-btn"><i aria-hidden="true" class="icon-export_citation"></i><span>Open in viewer</span></a></template><div id="figure-viewer" role="dialog" aria-modal="true" aria-label="Figure Viewer" class="showCaption showThumbstrip figure-viewer"><nav class="nav-bar figure-viewer__top"><a href="#" title="Close Figure Viewer" role="button" class="figure-viewer__close"><i aria-hidden="true" class="icon-tools_close"></i></a><div class="figure-viewer__title"></div></nav><div class="frames"><ul class="frames__list"></ul><a href="#" title="Previous Frame" class="frameButton previousFrameButton frames__prev"><!--span Previous Frame--><i class="icon-arrow_l"></i></a><a href="#" title="Next Frame" class="frameButton nextFrameButton frames__next"><!--span Next Frame--><i class="icon-arrow_r"></i></a></div><div class="caption"><div class="figure-viewer__caption-body"></div></div><div class="toolbar figure-viewer__toolbar"><span class="figure-viewer__counter"></span><a href="#" class="button toggle-caption-button figure-viewer__caption__ctrl"><i aria-hidden="true" class="icon icon-info"></i><span class="label">Hide Caption</span></a><a href="#" class="button download-options-button"><i aria-hidden="true" class="icon icon-gizmo-folder-in download-size-icon"></i><span class="label">Download</span></a><a href="#figures" class="button see-figures figure-viewer__see-figure__ctrl"><i aria-hidden="true" class="icon icon-gizmo-image_figure viewer-size-icon"></i><span class="label">See figure in Article</span></a></div><div class="thumbstrip"><a href="#" class="toggleButton thumbstrip__ctrl"><span>Toggle Thumbstrip</span></a><a href="#" title="Previous Thumbnails" class="thumbstripButton previousThumbstripButton thumbstrip__prev"><i aria-hidden="true" class="icon-arrow_l"></i></a><a href="#" title="Next Thumbnails" class="thumbstripButton nextThumbstripButton thumbstrip__next"><i aria-hidden="true" class="icon-arrow_r"></i></a><div class="thumbstrip__wrapper"><ul class="rlist figure-viewer__thumbslist"></ul></div></div><div class="download figure-viewer__download"><div class="pointer"></div><ul class="rlist downlad__wrapper"><li><a target="_blank" href="/specs/products/marlin/components/figure-viewer/images/facets-2015-0010-f10-standard.jpg" title="View high resolution image in a new tab" class="downloadThisButton"><i class="icon icon-gizmo-image_figure figure-viewer__download__viewer-size-icon"></i><span class="figure-viewer__download__text">Download Hi-res image</span></a></li><li class="hidden-xs hidden-sm"><a href="/action/downloadFigures?pii=e00489&amp;id=" class="downloadAllButton"><i class="icon icon-gizmo-ppt figure-viewer__download__viewer-size-icon figure-viewer__download__viewer-size-icon--ppt"></i><span class="figure-viewer__download__text"> Download .PPT</span></a></li></ul></div></div><template id="toCitationLink" data-citation="Citation" data-footnote="Footnote"><div class="to-citation__wrapper"><a class="to-citation"><i aria-hidden="true" class="icon-subdirectory_arrow_left"></i><span>Go to</span></a></div></template><template id="toCitationButton" data-citation="Citation" data-footnote="Footnote"><div class="to-citation__wrapper"><button class="to-citation"><i aria-hidden="true" class="icon-subdirectory_arrow_left"></i><span>Go to</span></button></div></template><template id="toCitationAccordion" data-collapse-citation="Hide Citations" data-expand-citation="Show Citations" data-collapse-footnote="Hide Footnotes" data-expand-footnote="Show Footnotes"><div class="to-citation__wrapper"><button aria-controls="aria-controls" aria-expanded="false" aria-label="Toggle citations menu" class="to-citation__toggle"><i aria-hidden="true" class="icon-plus"></i><span data-expand-title="data-expand-title" data-collapsed-title="data-collapsed-title" class="accordion__toggle__title"></span></button><div role="menu" aria-label="links" class="to-citation__accordion no-separator"><ul></ul></div></div></template><template id="toCitationAccordionItem"><li><a class="to-citation"><i aria-hidden="true" class="icon-subdirectory_arrow_left"></i><span></span></a></li></template><template id="citations_truncate_template"><div class="citations-truncation"><button data-label-expand="Show all references" data-label-collapse="Show fewer" data-label-remaining="references remaining" class="btn btn--inverse"><span>Show all references</span></button></div></template><template id="collateral_texts_template" data-do="Digital Object" data-figures="Figures" data-metrics="Article Metrics" data-related="Related articles" data-supplementary="Supplemental information"></template><template id="collapsible_authors_template"><button aria-expanded="false" data-expandable="all" data-label-expand="Expand All" data-label-collapse="Collapse All" class="collateral-contributors-control"><span>Expand All</span></button></template>
<template id="collapsible_tables_collapse_template"><div class="collapsible-figure-btn__wrapper expanded"><button aria-expanded="true" class="btn collapsible-figure-btn btn--inverse"><i aria-hidden="true" class="icon-arrow-up"></i><span class="text-uppercase">Collapse</span></button></div></template>
<template id="collapsible_tables_expand_template"><div class="collapsible-figure-btn__wrapper collapsed"><button aria-expanded="false" class="btn collapsible-figure-btn btn--inverse"><i aria-hidden="true" class="icon-arrow-down"></i><span class="text-uppercase">Expand Table</span></button></div></template><template id="authorsAffiliationsLink"><a href="#tab-contributors" class="to-authors-affiliations">Authors Info &amp; Affiliations</a></template></div>
                </div>
                </div>

        

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none  widget-  widget-compact-all" id="756cf18d-e910-4468-936c-0225ae7c669f">
                
                
                    
                

                <div class="wrapped " id="ad for nucleus-single-molecule-technologies">
                    
                    <div class="widget-body body body-  body-compact-all"><div class="pb-ad">
            <script type="text/javascript">
URL_var = 'https://www.cell.com/cell/fulltext/S0092-8674(20)30339-1';

if(URL_var.indexOf("S0165-6147(17)30205-5") > -1
|| URL_var.indexOf("S0167-7799(18)30204-X") > -1 
|| URL_var.indexOf("S0092-8674(18)31649-0") > -1 
|| URL_var.indexOf("S0092-8674(19)30404-0") > -1 
|| URL_var.indexOf("S1097-2765(18)30756-1") > -1 
|| URL_var.indexOf("S0092-8674(18)31452-1") > -1 
|| URL_var.indexOf("S1097-2765(19)30175-3") > -1 
|| URL_var.indexOf("S0092-8674(18)30852-3") > -1 
|| URL_var.indexOf("S0006-3495(18)30295-9") > -1 
|| URL_var.indexOf("S1097-2765(19)30063-2") > -1 
|| URL_var.indexOf("S0006-3495(17)33208-3") > -1 
|| URL_var.indexOf("S2211-1247(18)31639-5") > -1 
|| URL_var.indexOf("S1097-2765(18)30978-X") > -1 
)
    {
document.write('<input type="hidden" id="gpttc" value="nucleus-single-molecule-technologies" class="gpt-Target">')
}
</script>
        </div></div>
                </div>
                </div>

        

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none  widget-" id="754b1780-1117-4371-9285-9cd7020963d5">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body- "><div class="pb-ad">
            <script type="text/javascript">
URL_var = 'https://www.cell.com/cell/fulltext/S0092-8674(20)30339-1';

if(URL_var.indexOf("S1535-6108(19)30104-7") > -1
|| URL_var.indexOf("S0092-8674(18)31394-1") > -1
|| URL_var.indexOf("S1074-7613(19)30033-0") > -1
|| URL_var.indexOf("S1535-6108(18)30473-2") > -1
|| URL_var.indexOf("S0092-8674(19)30154-0") > -1
|| URL_var.indexOf("S1550-4131(18)30178-5") > -1
|| URL_var.indexOf("S1535-6108(19)30242-9") > -1
|| URL_var.indexOf("S1074-7613(18)30431-X") > -1
|| URL_var.indexOf("S0092-8674(19)30618-X") > -1
|| URL_var.indexOf("S1535-6108(18)30472-0") > -1
|| URL_var.indexOf("S1535-6108(18)30120-X") > -1
|| URL_var.indexOf("S0092-8674(18)31247-9") > -1
|| URL_var.indexOf("S1074-7613(18)30085-2") > -1
|| URL_var.indexOf("S1074-7613(18)30080-3") > -1
|| URL_var.indexOf("S1471-4906(18)30210-2") > -1
|| URL_var.indexOf("S1550-4131(19)30304-3") > -1
|| URL_var.indexOf("S1934-5909(18)30599-X") > -1
|| URL_var.indexOf("S2405-8033(19)30024-X") > -1
|| URL_var.indexOf("S1535-6108(18)30109-0") > -1
|| URL_var.indexOf("S0962-8924(18)30127-2") > -1
)
    {
document.write('<input type="hidden" id="gpttc" value="nucleus-tumor-immune-microenvironment" class="gpt-Target">')
}
</script>
        </div></div>
                </div>
                </div>

        

    

        


    

            

            

            <div data-widget-id="9c9ff7a1-a035-49f6-a336-600c9419c518" data-widget-name="UX3HTMLWidget"><!-- Start of HubSpot Embed Code --> 

<script type="text/plain" class="optanon-category-4" id="hs-script-loader" async="" defer="" src="//js.hs-scripts.com/519224.js"></script> 

<!-- End of HubSpot Embed Code --></div>

            

        

    

        


    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget sessionValidationWidget none  widget-none  widget-compact-all" id="0257aef7-6074-4898-a989-1c3f693225a2">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none  body-compact-all"><script>
    $.ajax({
        url: "https://www.cell.com/ssoSessionCheck",
        type: "GET",
        dataType: "jsonp",
        success: function (data) {
            if(data.RefreshSession) {
                location.reload();
            }
        },
        error : function (){
            console.warn("can't validate CSAS session.");
        }
    });
</script></div>
                </div>
                </div>

        

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget csasSsoUrls none  widget-none  widget-compact-all" id="948bc5c4-8b0d-47be-b5ad-6570559802d1">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none  body-compact-all"><!-- clear SSO URLs after all iFrames are rendered or after passing a time out-->
<script>
    setTimeout(clearSsoUrlsFun,10000);
    var iFramesList = document.getElementById("sso-urls-div").getElementsByTagName("iframe");
    var renderedIFramesCount = 0;
    var numberOfIFrames = iFramesList.length;
    var skipRefresh = false;
    for (var i = 0; i < iFramesList.length; i++) {
        var iFrame = iFramesList[i];
        iFrame.addEventListener("load", function() {
            renderedIFramesCount = renderedIFramesCount + 1;
            if (renderedIFramesCount >= numberOfIFrames)
            {
                clearSsoUrlsFun();
            }
        });
    }

    function clearSsoUrlsFun() {
        if (skipRefresh == false && numberOfIFrames > 0)
        {
            skipRefresh = true;
            $.ajax({
                url: "https://www.cell.com/action/clearSsoUrls",
                type: "GET",
                dataType: "jsonp",
                error : errorCallingAjx
            });
        }
    }

    function errorCallingAjx(){
        console.log("can't delete sso urls .")
    }
</script></div>
                </div>
                </div>

        










        <div data-widget-def="ux3-layout-widget" data-widget-id="d95b4a46-dc4d-4ca8-83f9-8b6724c649ec" class="ux3 ">
        
    

        


    

            

            

            <div data-widget-id="d95b4a46-dc4d-4ca8-83f9-8b6724c649ec" data-widget-name="ux3-layout-widget" class="fonts-loaded"><div class="footer clearfix">
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none adplaceholder adplaceholder--header widget-horizontalAd" id="a07ee92b-3942-49f8-9186-a2a08fc3fc56">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-horizontalAd "><div class="horizontalAdContainer">
    <div class="horizontalAdLabelContainer">
        <div class="horizontalAdLabel">
                <img src="/templates/jsp/_style2/_marlin/images/bg_adHoriz.gif" alt="Advertisement">
        </div>
    </div>
    <div class="horizontalAd">
        





        <div class="pb-ad">
            <!-- Beginning Async AdSlot 4 for Ad unit els.cellpress/cell  ### size: [[728,90]] -->
<div class="gpt-ad" data-adsize="cell-leaderboard" data-page="" data-site="cell" data-pos="4"></div>
<!-- End AdSlot 4 -->
        </div>
    

    </div>
</div></div>
                </div>
                </div>

        

    

        


    

            

            

            <div data-widget-id="23703bbc-af46-4f3f-a5bb-dd52891ff0ff" data-widget-name="UX3HTMLWidget"><div class="footer__top">
    <div class="footer__logo centered"><a href="/home" title="Cell Press"><!-- Generator: Adobe Illustrator 25.2.1, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 300 72" style="enable-background:new 0 0 300 72;" xml:space="preserve">
<style type="text/css">
	.st0{fill:#007DBC;}
</style>
<g>
	<path class="st0" d="M39.4,7.5c-2.3-0.3-4.4,1.1-4.9,3.3l-1.8,7.4c-0.4,1.8-1.9,3.1-3.8,3.3l-10.5,1.2C7.1,24-1,34.3,0.7,45.6
		c1.3,9,8.6,15.7,17.2,17c2.3,0.3,4.4-1.1,4.9-3.3l1.8-7.4c0.4-1.8,1.9-3.1,3.8-3.3l10.5-1.2c11.3-1.2,19.4-11.6,17.8-22.9
		C55.3,15.4,48.1,8.8,39.4,7.5z M37.1,34l-6.6,0.8c-1.1,0.1-2,0.9-2.3,2l-1.9,8.1c-0.6,2.4-2.5,4.2-5,4.5c-3.8,0.4-7.1-2.4-7.4-6.2
		c-0.3-3.7,2.6-6.8,6.3-7.2l6.6-0.8c1.1-0.1,2-0.9,2.3-2l1.9-8.1c0.6-2.4,2.5-4.2,5-4.5c3.8-0.4,7.1,2.4,7.4,6.2
		C43.7,30.5,40.8,33.6,37.1,34z"></path>
	<rect x="153.1" y="13.9" class="st0" width="8.2" height="42.3"></rect>
	<path class="st0" d="M90.2,20.4c7.3,0,8.6,4.8,9.4,7.3h8.8c-0.5-6.2-5.6-14.9-18.2-14.9c-10.9,0-19.4,7.5-19.4,22.3
		c0,14.6,8.1,22.2,19.3,22.2c10.6,0,16.6-6.4,18.4-15.6h-8.8c-1.1,5.2-4.5,7.9-9.4,7.9c-7.5,0-10.5-6.8-10.5-14.4
		C79.7,23.3,85.9,20.4,90.2,20.4z"></path>
	<path class="st0" d="M125.2,24c-13.7,0-15,12.1-15,15.8c0,12.9,6.8,17.1,15.6,17.1c5.4,0,11.6-2.5,13.8-10H131
		c-0.7,2.2-3,3.4-5.3,3.4c-6.6,0-7.1-5.3-7.3-7.7h21.5v-1.5C139.9,27,131.3,24,125.2,24z M118.6,37.4c0.5-4.1,2.5-6.6,6.4-6.6
		c2.6,0,5.9,1.5,6.4,6.6H118.6z"></path>
	<rect x="142" y="13.9" class="st0" width="8.2" height="42.3"></rect>
	<path class="st0" d="M174,13.8h-9.8V21h7.2c4.3,0,8.2,0.9,8.2,6.4c0,5.4-3.9,6.4-8.2,6.4h-7.2V41h9.8c10.6,0,14.7-6.7,14.7-13.6
		C188.7,20.5,184.6,13.8,174,13.8z"></path>
	<path class="st0" d="M210.5,24.9c-0.5-0.2-1.2-0.3-1.8-0.3c-4.1,0-8.3,2.7-9.8,6.5h-0.1v-5.7h-8v30.7h8.4V42.3
		c0-5.4,2.1-9.9,8.3-9.9c1,0,2.3,0.1,3,0.3V24.9z"></path>
	<path class="st0" d="M225.6,24.6c-9.3,0-15.7,7-15.7,16.2c0,9.5,6,16.1,15.7,16.1c6.9,0,12-3.1,14.4-10.3h-7.4
		c-0.5,1.9-3.3,4-6.6,4c-4.7,0-7.3-2.4-7.5-7.8h22.1C241.1,33.4,236,24.6,225.6,24.6z M218.4,37.5c0.1-2.4,1.7-6.5,7-6.5
		c4.1,0,5.9,2.3,6.7,6.5H218.4z"></path>
	<path class="st0" d="M260.3,37.7c-4.8-1.1-9.6-1.5-9.6-4.6c0-2.5,2.8-2.9,4.7-2.9c3,0,5.5,0.9,5.8,4.2h8c-0.7-7.5-6.8-9.8-13.5-9.8
		c-6.6,0-13.4,2-13.4,9.9c0,5.4,4.9,7.1,9.7,8.2c6.1,1.3,9.6,2.2,9.6,4.8c0,3-3.1,3.9-5.5,3.9c-3.3,0-6.4-1.5-6.5-5.2h-8
		C242,54.3,248.9,57,256,57c7,0,14-2.6,14-10.7C270.1,40.6,265.2,38.8,260.3,37.7z"></path>
	<path class="st0" d="M289.8,37.7c-4.8-1.1-9.6-1.5-9.6-4.6c0-2.5,2.8-2.9,4.7-2.9c3,0,5.5,0.9,5.8,4.2h8c-0.7-7.5-6.8-9.8-13.5-9.8
		c-6.6,0-13.4,2-13.4,9.9c0,5.4,4.9,7.1,9.7,8.2c6.1,1.3,9.6,2.2,9.6,4.8c0,3-3.1,3.9-5.5,3.9c-3.3,0-6.4-1.5-6.5-5.2h-8
		c0.4,8.1,7.3,10.8,14.4,10.8c7,0,14-2.6,14-10.7C299.6,40.6,294.7,38.8,289.8,37.7z"></path>
	<path class="st0" d="M34.1,57.6c0.3-0.2,0.4-0.6,0.4-1c0-0.5-0.2-0.9-0.5-1.1c-0.3-0.2-0.8-0.3-1.4-0.3h-1.9v5.1h0.8V58h0.8
		l1.3,2.2h0.9l-1.4-2.2C33.6,57.9,33.9,57.8,34.1,57.6z M32.5,57.3h-0.9v-1.5h1c0.1,0,0.3,0,0.4,0c0.1,0,0.3,0,0.4,0.1
		c0.1,0,0.2,0.1,0.3,0.2c0.1,0.1,0.1,0.2,0.1,0.4c0,0.2,0,0.3-0.1,0.4c-0.1,0.1-0.2,0.2-0.3,0.2c-0.1,0.1-0.2,0.1-0.4,0.1
		C32.8,57.3,32.6,57.3,32.5,57.3z"></path>
	<path class="st0" d="M35.6,54.2c-0.4-0.4-0.9-0.7-1.4-0.9c-0.5-0.2-1.1-0.3-1.7-0.3c-0.6,0-1.2,0.1-1.7,0.3c-0.6,0.2-1,0.5-1.4,0.9
		c-0.4,0.4-0.7,0.9-1,1.5c-0.2,0.6-0.4,1.2-0.4,1.9c0,0.7,0.1,1.3,0.3,1.8c0.2,0.6,0.5,1,0.9,1.5c0.4,0.4,0.9,0.7,1.4,1
		c0.6,0.2,1.2,0.4,1.8,0.4c0.6,0,1.2-0.1,1.7-0.3c0.5-0.2,1-0.5,1.4-0.9c0.4-0.4,0.7-0.9,1-1.5c0.2-0.6,0.4-1.2,0.4-1.9
		c0-0.7-0.1-1.3-0.4-1.9C36.4,55.1,36,54.6,35.6,54.2z M35.7,59.1c-0.2,0.5-0.4,0.9-0.8,1.2c-0.3,0.3-0.7,0.6-1.1,0.8
		c-0.4,0.2-0.9,0.3-1.4,0.3c-0.6,0-1-0.1-1.5-0.3c-0.4-0.2-0.8-0.5-1.1-0.8c-0.3-0.3-0.5-0.7-0.7-1.2c-0.2-0.5-0.2-1-0.2-1.5
		c0-0.6,0.1-1.1,0.3-1.5c0.2-0.5,0.4-0.9,0.8-1.2c0.3-0.3,0.7-0.6,1.1-0.8c0.4-0.2,0.9-0.3,1.4-0.3c0.5,0,1,0.1,1.4,0.3
		c0.4,0.2,0.8,0.4,1.1,0.8c0.3,0.3,0.6,0.7,0.8,1.2c0.2,0.5,0.3,1,0.3,1.5C36,58.1,35.9,58.6,35.7,59.1z"></path>
</g>
</svg></a></div>
    <div class="footer__social-links">
        <ul class="rlist rlist--inline centered">
            <li><a href="http://on.fb.me/1tgHakh" title="Facebook" target="_blank"><i aria-hidden="true" class="icon-Icon_Facebook"></i></a></li>

            <li><a href="https://www.instagram.com/cellpress/" title="Instagram" target="_blank"><svg width="18px" height="18px" viewBox="0 0 18 18" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-147.000000, -16.000000)" fill-rule="nonzero" fill="#969696"> <g id="Instagram-Logo" transform="translate(147.000000, 16.000000)"> <path d="M9,1.62580645 C11.4096774,1.62580645 11.7,1.62580645 12.6290323,1.68387097 C13.5,1.71290323 13.9935484,1.85806452 14.3129032,2.00322581 C14.7193548,2.17741935 15.0387097,2.3516129 15.3580645,2.67096774 C15.6774194,2.99032258 15.8806452,3.28064516 16.0258065,3.71612903 C16.1419355,4.03548387 16.2870968,4.5 16.3451613,5.4 C16.3741935,6.35806452 16.4032258,6.61935484 16.4032258,9.02903226 C16.4032258,11.4387097 16.4032258,11.7290323 16.3451613,12.6580645 C16.316129,13.5290323 16.1709677,14.0225806 16.0258065,14.3419355 C15.8516129,14.7483871 15.6774194,15.0677419 15.3580645,15.3870968 C15.0387097,15.7064516 14.7483871,15.9096774 14.3129032,16.0548387 C13.9935484,16.1709677 13.5290323,16.316129 12.6290323,16.3741935 C11.6709677,16.4032258 11.4096774,16.4322581 9,16.4322581 C6.59032258,16.4322581 6.3,16.4322581 5.37096774,16.3741935 C4.5,16.3451613 4.00645161,16.2 3.68709677,16.0548387 C3.28064516,15.8806452 2.96129032,15.7064516 2.64193548,15.3870968 C2.32258065,15.0677419 2.11935484,14.7774194 1.97419355,14.3419355 C1.85806452,14.0225806 1.71290323,13.5580645 1.65483871,12.6580645 C1.62580645,11.7 1.62580645,11.4096774 1.62580645,9 C1.62580645,6.59032258 1.62580645,6.3 1.68387097,5.37096774 C1.71290323,4.5 1.85806452,4.00645161 2.00322581,3.68709677 C2.17741935,3.28064516 2.3516129,2.96129032 2.67096774,2.64193548 C2.99032258,2.32258065 3.28064516,2.11935484 3.71612903,1.97419355 C4.03548387,1.85806452 4.5,1.71290323 5.4,1.65483871 C6.3,1.62580645 6.59032258,1.62580645 9,1.62580645 L9,1.62580645 Z M9,0 C6.56129032,0 6.24193548,0 5.28387097,0.0580645161 C4.32580645,0.116129032 3.65806452,0.261290323 3.10645161,0.464516129 C2.52580645,0.696774194 2.00322581,1.01612903 1.50967742,1.50967742 C1.01612903,2.00322581 0.696774194,2.52580645 0.464516129,3.10645161 C0.232258065,3.68709677 0.0870967742,4.32580645 0.0580645161,5.28387097 C0,6.24193548 0,6.56129032 0,9 C0,11.4387097 0,11.7580645 0.0580645161,12.716129 C0.116129032,13.6741935 0.261290323,14.3419355 0.464516129,14.8935484 C0.696774194,15.4741935 1.01612903,15.9967742 1.50967742,16.4903226 C2.00322581,16.983871 2.52580645,17.3032258 3.10645161,17.5354839 C3.68709677,17.7677419 4.32580645,17.9129032 5.28387097,17.9419355 C6.24193548,18 6.56129032,18 9,18 C11.4387097,18 11.7580645,18 12.716129,17.9419355 C13.6741935,17.883871 14.3419355,17.7387097 14.8935484,17.5354839 C15.4741935,17.3032258 15.9967742,16.983871 16.4903226,16.4903226 C16.983871,15.9967742 17.3032258,15.4741935 17.5354839,14.8935484 C17.7677419,14.3129032 17.9129032,13.6741935 17.9419355,12.716129 C18,11.7580645 18,11.4387097 18,9 C18,6.56129032 18,6.24193548 17.9419355,5.28387097 C17.883871,4.32580645 17.7387097,3.65806452 17.5354839,3.10645161 C17.3032258,2.52580645 16.983871,2.00322581 16.4903226,1.50967742 C15.9967742,1.01612903 15.4741935,0.696774194 14.8935484,0.464516129 C14.3129032,0.232258065 13.6741935,0.0870967742 12.716129,0.0580645161 C11.7580645,0 11.4387097,0 9,0 L9,0 Z"></path> <path d="M9,4.38387097 C6.44516129,4.38387097 4.38387097,6.44516129 4.38387097,9 C4.38387097,11.5548387 6.44516129,13.616129 9,13.616129 C11.5548387,13.616129 13.616129,11.5548387 13.616129,9 C13.616129,6.44516129 11.5548387,4.38387097 9,4.38387097 Z M9,11.9903226 C7.34516129,11.9903226 6.00967742,10.6548387 6.00967742,9 C6.00967742,7.34516129 7.34516129,6.00967742 9,6.00967742 C10.6548387,6.00967742 11.9903226,7.34516129 11.9903226,9 C11.9903226,10.6548387 10.6548387,11.9903226 9,11.9903226 Z"></path> <circle id="Oval" cx="13.7903226" cy="4.20967742" r="1.07419355"></circle> </g> </g> </g></svg></a></li>
            
            <li><a href="https://www.linkedin.com/company/cell-press/" title="LinkedIn" target="_blank"><i aria-hidden="true" class="icon-linkedin"></i></a></li>
            
                        <li><a href="http://bit.ly/1zq7Q5T" title="Twitter" target="_blank">
                <!--<i aria-hidden="true" class="icon-Icon_Twitter"></i></a>-->
                
                <svg id="xicon" viewBox="0 0 1200 1227" xmlns="http://www.w3.org/2000/svg" aria-hidden="true" role="none"><path fill="#969696" d="M714.163 519.284L1160.89 0H1055.03L667.137 450.887L357.328 0H0L468.492 681.821L0 1226.37H105.866L515.491 750.218L842.672 1226.37H1200L714.137 519.284H714.163ZM569.165 687.828L521.697 619.934L144.011 79.6944H306.615L611.412 515.685L658.88 583.579L1055.08 1150.3H892.476L569.165 687.854V687.828Z"></path></svg> 

                    <style>
                    .footer svg#xicon {
                      height: 16px;
                    }
                    </style>
            </a></li>

            
            <li><a href="/pb-assets/marketing/WeChat/WeChat-1557753222727.jpg" title="WeChat" target="_blank"><svg width="18px" height="15px" viewBox="0 0 18 15" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-122.000000, -17.000000)" fill-rule="nonzero" fill="#969696"> <g id="wechat" transform="translate(122.000000, 17.000000)"> <path d="M12.375,4.5 L12.375,4.5 L12.4875,4.5 L12.4875,4.5 C12.6898579,4.48604473 12.8520408,4.32704187 12.87,4.125 C12.8705905,4.04725572 12.8470025,3.9712499 12.8025,3.9075 C11.94,1.5675 9.4425,0 6.5625,0 C2.9475,0 0,2.52 0,5.625 C0.0313772094,7.34410122 0.895513317,8.94093526 2.3175,9.9075 L1.5675,11.46 C1.49009368,11.6087816 1.52088865,11.7908702 1.64291494,11.9059236 C1.76494122,12.0209769 1.94852501,12.0410168 2.0925,11.955 L4.08,10.8225 C4.61861992,11.0129999 5.17724298,11.1413322 5.745,11.205 C5.86564124,11.2178337 5.98503738,11.1715098 6.06545426,11.0806685 C6.14587113,10.9898272 6.17737153,10.865694 6.15,10.7475 C6.06148727,10.4219419 6.01114891,10.0871918 6,9.75 C6,6.855 8.8575,4.5 12.375,4.5 Z M8.25,3 C8.66421356,3 9,3.33578644 9,3.75 C9,4.16421356 8.66421356,4.5 8.25,4.5 C7.83578644,4.5 7.5,4.16421356 7.5,3.75 C7.5,3.33578644 7.83578644,3 8.25,3 Z M4.5,4.5 C4.08578644,4.5 3.75,4.16421356 3.75,3.75 C3.75,3.33578644 4.08578644,3 4.5,3 C4.91421356,3 5.25,3.33578644 5.25,3.75 C5.25,4.16421356 4.91421356,4.5 4.5,4.5 Z"></path> <path d="M18,9.75 C18,7.2675 15.48,5.25 12.375,5.25 C9.27,5.25 6.75,7.2675 6.75,9.75 C6.75,12.2325 9.27,14.25 12.375,14.25 C12.9568557,14.2502837 13.5364258,14.1772075 14.1,14.0325 L15.96,14.9625 C16.0114248,14.9874074 16.0678613,15.0002339 16.125,15 C16.2497193,14.9993787 16.3659682,14.9367912 16.4351508,14.8330172 C16.5043335,14.7292432 16.5173955,14.5978644 16.47,14.4825 L15.96,13.2075 C17.1784092,12.4716325 17.9450305,11.1723221 18,9.75 L18,9.75 Z M10.5,9 C10.0857864,9 9.75,8.66421356 9.75,8.25 C9.75,7.83578644 10.0857864,7.5 10.5,7.5 C10.9142136,7.5 11.25,7.83578644 11.25,8.25 C11.25,8.66421356 10.9142136,9 10.5,9 Z M14.25,9 C13.8357864,9 13.5,8.66421356 13.5,8.25 C13.5,7.83578644 13.8357864,7.5 14.25,7.5 C14.6642136,7.5 15,7.83578644 15,8.25 C15,8.66421356 14.6642136,9 14.25,9 Z" opacity="0.959999979"></path> </g> </g> </g></svg></a></li>
            
            <li><a href="https://www.weibo.com/CellPress" title="Weibo" target="_blank"><svg width="18px" height="15px" viewBox="0 0 18 15" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-99.000000, -17.000000)" fill-rule="nonzero" fill="#969696"> <g id="weibo" transform="translate(99.000000, 14.000000)"> <g transform="translate(0.000000, 3.000000)"> <path d="M13.2402211,7.47054172 C12.9918392,7.39866484 12.821608,7.34902326 12.9514975,7.03290287 C13.232804,6.34602189 13.2622915,5.75359781 12.9571055,5.33095486 C12.3847236,4.5380684 10.8191759,4.58064295 9.02514573,5.30975376 C9.02514573,5.30889193 8.46144724,5.54882627 8.60544724,5.11515185 C8.88150754,4.25435294 8.83953769,3.53386047 8.41043216,3.11759508 C7.43716583,2.17285363 4.84824121,3.15344733 2.62745729,5.3059617 C0.964944724,6.9186238 0,8.62798358 0,10.105855 C0,12.9324952 3.73748744,14.8235294 7.39429146,14.8235294 C12.1877186,14.8235294 15.3768844,11.9501778 15.3768844,9.80576471 C15.3768844,8.50974008 14.2514774,7.77442408 13.2402211,7.47054172 Z M7.40424121,13.5561176 C4.48625126,13.8315595 1.9678794,12.5686293 1.77811055,10.7327524 C1.58870352,8.89773735 3.80098492,7.18579207 6.71861307,6.90948837 C9.63678392,6.63266758 12.1556985,7.89663201 12.3447437,9.73164706 C12.533608,11.5675239 10.3227739,13.2796416 7.40424121,13.5561176 Z M18,5.16875787 C17.9985528,2.31402189 15.5695477,0 12.5728643,0 C12.222995,0 11.9396985,0.270098495 11.9396985,0.603283174 C11.9396985,0.936640219 12.222995,1.20656635 12.5728643,1.20656635 C14.8708945,1.20656635 16.7336683,2.98159781 16.7336683,5.17099863 C16.7336683,5.50435568 17.0173266,5.77428181 17.3668342,5.77428181 C17.7167035,5.77428181 18,5.50435568 18,5.17099863 C18,5.17099863 18,5.1696197 18,5.16875787 Z M15.4841608,5.08016142 C15.1948945,3.73415048 14.0812462,2.67323393 12.6683819,2.39744733 C12.3262915,2.33039672 11.9914372,2.54051163 11.9212462,2.86680164 C11.8506935,3.19309166 12.0712161,3.51179754 12.4133065,3.57919289 C13.3319397,3.75845417 14.0559196,4.44774829 14.2438794,5.32302599 C14.3146131,5.64931601 14.6491055,5.85925855 14.9915578,5.7923803 C15.3338291,5.72498495 15.5541709,5.40627907 15.4841608,5.08016142 Z M5.90761809,8.7069275 C4.7321005,8.9320383 3.93901508,9.87126402 4.1358392,10.8044569 C4.33248241,11.7378222 5.44504523,12.311803 6.62056281,12.0865198 C7.79571859,11.861409 8.58916583,10.922528 8.3921608,9.98899042 C8.19569849,9.0559699 7.08277387,8.48181669 5.90761809,8.7069275 Z"></path> </g> </g> </g> </g></svg></a></li>
            
            <li><a href="https://www.youtube.com/user/cellvideoabstracts" title="YouTube" target="_blank"><svg width="18px" height="13px" viewBox="0 0 18 13" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-76.000000, -19.000000)" fill-rule="nonzero" fill="#969696"> <g id="youtube" transform="translate(76.000000, 14.000000)"> <path d="M17.2984219,6.41974219 C16.6486641,5.64735937 15.4490625,5.33232422 13.1580703,5.33232422 L4.84175391,5.33232422 C2.49834375,5.33232422 1.27838672,5.66771484 0.631089844,6.49001953 C0,7.29172266 0,8.47304297 0,10.1080547 L0,13.2243047 C0,16.3917773 0.748792969,18 4.84175391,18 L13.1581055,18 C15.1448555,18 16.2457383,17.7219844 16.9579688,17.040375 C17.688375,16.3413984 18,15.2001211 18,13.2243047 L18,10.1080547 C18,8.38381641 17.951168,7.19553516 17.2984219,6.41974219 Z M11.5560703,12.0963867 L7.77965625,14.0700586 C7.69524609,14.1141797 7.60289062,14.136082 7.51071094,14.136082 C7.40629687,14.136082 7.30212891,14.1079922 7.20987891,14.0521289 C7.03613672,13.9468711 6.93007031,13.7585391 6.93007031,13.5554414 L6.93007031,9.62075391 C6.93007031,9.41800781 7.03585547,9.22985156 7.20924609,9.12452344 C7.38267187,9.01919531 7.59839062,9.01198828 7.77832031,9.10546875 L11.5547344,11.0664141 C11.7468633,11.1661523 11.8675195,11.3645039 11.8678008,11.5809258 C11.8680469,11.7975586 11.747918,11.9961914 11.5560703,12.0963867 Z"></path> </g> </g> </g></svg></a></li>
            
        </ul>
    </div>
</div>

<style>

.ux3 .footer__logo svg {
    max-width: 160px !important;
}

.ux3 .footer__social-links a svg {
    max-width: 26px;
    height: 1.15rem;
    width: auto;
}

.ux3 .footer__social-links a:hover svg path {
    fill: #353535;
}

</style></div>

            

        

    

        


    

            

            
                <div class="footer__bottom text-onDark">
            

            <div data-widget-id="06227714-c621-4651-9879-b0ce5176cccc" data-widget-name="ux3-layout-widget"><div class="col-sm-4 col-md-5 clearfix">
    

        


    

            

            

            <div data-widget-id="07e1d276-b4dd-4870-b858-ed74e70d64ef" data-widget-name="UX3HTMLWidget"><div class="groups">
	<div class="group">
		<h2 class="group__title">Life &amp; medical sciences journals</h2>
		<ul class="rlist clearfix sided">
		<li>
			<a alt="Cell" href="/cell/home">Cell</a>
		</li>
		<li>
			<a alt="Cancer Cell" href="/cancer-cell/home">Cancer Cell</a>
		</li>
		<li>
			<a alt="Cell Chemical Biology" href="/cell-chemical-biology">Cell Chemical Biology</a>
		</li>
		<li>
			<a alt="Cell Genomics" href="/cell-genomics/home">Cell Genomics</a>
		</li>
		<li>
			<a alt="Cell Host and Microbe" href="/cell-host-microbe/home">Cell Host &amp; Microbe</a>
		</li>
		<li>
			<a alt="Cell Metabolism" href="/cell-metabolism/home">Cell Metabolism</a>
		</li>
		<li>
			<a alt="Cell Reports" href="/cell-reports/home">Cell Reports</a>
		</li>
		<li>
			<a alt="Cell Reports Medicine" href="/cell-reports-medicine/home">Cell Reports Medicine</a>
		</li>
		<li>
			<a alt="Cell Stem Cell" href="/cell-stem-cell/home">Cell Stem Cell</a>
		</li>
		<li>
			<a alt="Cell Systems" href="/cell-systems/home">Cell Systems</a>
		</li>
		<li>
			<a alt="Current Biology" href="/current-biology/home">Current Biology</a>
		</li>
		<li>
			<a alt="Developmental Cell" href="/developmental-cell/home">Developmental Cell</a>
		</li>
		<li>
			<a alt="Immunity" href="/immunity/home">Immunity</a>
		</li>
		<li>
			<a alt="Med" href="/med/home">Med</a>
		</li>
		<li>
			<a alt="Molecular Cell" href="/molecular-cell/home">Molecular Cell</a>
		</li>
		<li>
			<a alt="Neuron" href="/neuron/home">Neuron</a>
		</li>
		<li>
			<a alt="Structure" href="/structure/home">Structure</a>
		</li>
		<li>
			<a alt="American Journal of Human Genetics" href="/ajhg/home">American Journal of Human Genetics (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Biophysical Journal" href="/biophysj/home">Biophysical Journal (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Biophysical Reports" href="/biophysreports/home">Biophysical Reports (<i>partner</i>)</a>
		</li>	
		<li>
			<a alt="HGG Advances" href="/hgg-advances/home">Human Genetics and Genomics Advances (<i>partner</i>)</a></li>
		<li>
			<a alt="Molecular Plant" href="/molecular-plant/home">Molecular Plant (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy" href="/molecular-therapy-family/molecular-therapy/home">Molecular Therapy (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy Methods and Clinical Development" href="/molecular-therapy-family/methods/home">Molecular Therapy Methods &amp; Clinical Development (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy Nucleic Acids" href="/molecular-therapy-family/nucleic-acids/home">Molecular Therapy Nucleic Acids (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy Oncology" href="/molecular-therapy-family/oncology/home">Molecular Therapy Oncology (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Plant Communications" href="/plant-communications/home">Plant Communications (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Stem Cell Reports" href="/stem-cell-reports/home">Stem Cell Reports (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Trends in Biochemical Sciences" href="/trends/biochemical-sciences/home">Trends in Biochemical Sciences</a>
		</li>
		<li>
		</li><li>
			<a alt="Trends in Cancer" href="/trends/cancer/home">Trends in Cancer</a>
		</li>
		<li>
			<a alt="Trends in Cell Biology" href="/trends/cell-biology/home">Trends in Cell Biology</a>
		</li>
		<li>
			<a alt="Trends in Ecology and Evolution" href="/trends/ecology-evolution/home">Trends in Ecology &amp; Evolution</a>
		</li>
		<li>
			<a alt="Trends in Endocrinology and Metabolism" href="/trends/endocrinology-metabolism/home">Trends in Endocrinology &amp; Metabolism</a>
		</li>
		<li>
			<a alt="Trends in Genetics" href="/trends/genetics/home">Trends in Genetics</a>
		</li>
		<li>
			<a alt="Trends in Immunology" href="/trends/immunology/home">Trends in Immunology</a>
		</li>
		<li>
			<a alt="Trends in Microbiology" href="/trends/microbiology/home">Trends in Microbiology</a>
		</li>
		<li>
			<a alt="Trends in Molecular Medicine" href="/trends/molecular-medicine/home">Trends in Molecular Medicine</a>
		</li>
		<li>
			<a alt="Trends in Neurosciences" href="/trends/neurosciences/home">Trends in Neurosciences</a>
		</li>
		<li>
			<a alt="Trends in Parasitology" href="/trends/parasitology/home">Trends in Parasitology</a>
		</li>
		<li>
			<a alt="Trends in Pharmacological Sciences" href="/trends/pharmacological-sciences/home">Trends in Pharmacological Sciences</a>
		</li>
		<li>
			<a alt="Trends in Plant Science" href="/trends/plant-science/home">Trends in Plant Science</a>
		</li>
		</ul>
	</div>

	<div class="group">
		<h2 class="group__title">Physical sciences &amp; engineering journals</h2>
		<ul class="rlist clearfix sided">
		  <li><a alt="Cell Biomaterials" href="/cell-biomaterials/home">Cell Biomaterials</a></li>
		  <li>
			<a alt="Cell Reports Physical Science" href="/cell-reports-physical-science/home">Cell Reports Physical Science</a>
		</li>
		<li>
			<a alt="Chem" href="/chem/home">Chem</a>
		</li>
		<li>
			<a alt="Chem Catalysis" href="/chem-catalysis/home">Chem Catalysis</a>
		</li>
		<li>
			<a alt="Device" href="/device/home">Device</a>
		</li>
		<li>
			<a alt="Joule" href="/joule/home">Joule</a>
		</li>
		<li>
			<a alt="Matter" href="/matter/home">Matter</a>
		</li>	
		<li><a alt="Matter" href="/newton/home">Newton</a></li>
		<li>
			<a alt="Trends in Chemistry" href="/trends/chemistry/home">Trends in Chemistry</a>
		</li>			
		</ul>
	</div>

<div class="group">
	<h2 class="group__title">Multidisciplinary journals</h2>
	<ul class="rlist clearfix sided">
	  		<li>
			<a alt="Cell Reports Methods" href="/cell-reports-methods/home">Cell Reports Methods</a>
		</li>
		<li>
			<a alt="Cell Reports Sustainability" href="/cell-reports-sustainability/home">Cell Reports Sustainability</a>
		</li>
		<li>
			<a alt="Heliyon" href="/heliyon/home">Heliyon</a>
		</li>	
		<li>
			<a alt="iScience" href="/iscience/home">iScience</a>
		</li>
		<li>
			<a alt="One Earth" href="/one-earth/home">One Earth</a>
		</li>
		<li>
			<a alt="Patterns" href="/patterns/home">Patterns</a>
		</li>
		<li>
			<a alt="STAR Protocols" href="/star-protocols/home">STAR Protocols</a>
		</li>
		<li>
			<a alt="Nexus" href="/nexus/home">Nexus (<i>partner</i>)</a>
		</li>			
        <li>
			<a alt="The Innovation" href="/the-innovation/home">The Innovation (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Trends in Biotechnology" href="/trends/biotechnology/home">Trends in Biotechnology</a>
		</li>
		<li>
			<a alt="Trends in Cognitive Sciences" href="/trends/cognitive-sciences/home">Trends in Cognitive Sciences</a>
		</li>
	</ul>
</div>

</div>
</div>

            

        
</div><div class="col-sm-4 col-md-5 clearfix">
    

        


    

            

            

            <div data-widget-id="850476de-00cb-4c0d-943c-bf1881da24f9" data-widget-name="UX3HTMLWidget"><div class="groups">
	<div class="clearfix">
		<div class="group sided">
			<h2 class="group__title">Authors</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Submit article" href="/submit-to-cell-press">Submit article</a>
				</li>
				<li>
					<a alt="Multi-Journal Submission" href="/multi-journal-submission">Multi-Journal Submission</a>
				</li>
				<li>
					<a alt="STAR Methods" href="/star-authors-guide">STAR Methods</a>
				</li>
				<li>
					<a alt="Sneak Peek – Preprints" href="/sneakpeek">Sneak Peek – Preprints</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
		<h2 class="group__title">Reviewers</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Reviewers" href="/reviewers">Information for reviewers</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
		<h2 class="group__title">News &amp; events</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Newsroom" href="/newsroom">Newsroom</a>
				</li>
				<li>
					<a alt="Events" href="/events">Events</a>
				</li>
				<li>
					<a alt="Cell Symposia" href="/symposia">Cell Symposia</a>
				</li>
				<li>
					<a alt="Consortia Hub" href="/consortia">Consortia Hub</a>
				</li>
				<li>
					<a alt="Webinars" href="/webinars">Webinars</a>
				</li>
								<li>
					&nbsp;
				</li>
			</ul>
		<h2 class="group__title">Multimedia</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Cell Press Podcast" href="/podcast">Cell Press Podcast</a>
				</li>
				<li>
					<a alt="Cell Press Videos" href="/video">Cell Press Videos</a>
				</li>
				<li>
					<a alt="Color and Comics" href="/comics">Coloring and Comics</a>
				</li>
				<li>
					<a alt="Figure360" href="/video-guidelines">Figure360</a>
				</li>
				<li>
					<a alt="Cell Picture Show" href="/pictureshow">Cell Picture Show</a>
				</li>
				<li>
					<a alt="Research Arc" href="/research-arc">Research Arc</a>
				</li>
			</ul>
		</div>

		<div class="group sided">
			<h2 class="group__title">About</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="About Cell Press" href="/about">About Cell Press</a>
				</li>

				<li>
					<a alt="Open access" href="/open-access">Open access</a>
				</li>
				<li>
					<a alt="COVID Hub" href="/covid-19">COVID Hub</a>
				</li>
				<li>
					<a alt="Sustainability" href="/sustainability">Sustainability hub</a>
				</li>
				<li>
					<a alt="Inclusion and diversity" href="/diversity">Inclusion and diversity</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
			<h2 class="group__title">Contact</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Contact us" href="/contact">Contact us</a>
				</li>
				<li>
					<a alt="Help and Support" href="https://service.elsevier.com/app/overview/cell-press/" target="_blank">Help &amp; Support</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
			<h2 class="group__title">Careers</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Cell Press Careers" href="https://relx.wd3.myworkdayjobs.com/en-US/ElsevierJobs" target="_blank">Cell Press Careers</a>
				</li>
			</ul>
		</div>
	</div>
</div></div>

            

        
</div><div class="col-sm-4 col-md-2 clearfix">
    

        


    

            

            

            <div data-widget-id="beafac41-1cfc-440b-9b55-74aba4a5612c" data-widget-name="UX3HTMLWidget"><div class="groups">
	<div class="group">
		<h2 class="group__title">Access</h2>
		<ul class="rlist clearfix">
			<li>
				<a alt="Subscribe" href="/subscribe">Subscribe</a>
			</li>
			<li>
				<a alt="Claim" href="/activate">Claim</a>
			</li>
			<li>
				<a alt="Read It Now" href="/read-it-now">Read-It-Now</a>
			</li>
			<li>
				<a alt="Recommend to Librarian" href="http://info.cell.com/request-access-to-cell-press-content" target="_blank">Recommend to Librarian</a>
			</li>
			<li>
				<a alt="Publication Alerts" href="/action/showPreferences?menuTab=Alerts">Publication Alerts</a>
			</li>
		</ul>
	</div>
	<div class="group">
		<h2 class="group__title">Collections</h2>
		<ul class="rlist clearfix">
			<li>
				<a alt="Best of Cell Press" href="/bestof">Best of Cell Press</a>
			</li>
			<li>
				<a alt="Cell Press Reviews" href="/reviews">Cell Press Reviews</a>
			</li>
			<li>
				<a alt="Cell Selections" href="/selections">Cell Press Selections</a>
			</li>
			<li>
				<a alt="Nucleus Collections" href="/nucleus">Nucleus Collections</a>
			</li>
			<li>
				<a alt="SnapShots" href="/snapshots">SnapShot Archive</a>
			</li>
		</ul>
	</div>
	<div class="group">
		<h2 class="group__title">Information</h2>
		<ul class="rlist clearfix">
			<li>
				<a alt="For Advertisters" href="/media-kit">For Advertisers</a>
			</li>
			<li>
				<a alt="For Recruiters" target="_blank" href="/pb-assets/Cell/common/CP-RecruitmentKit-US-2018-LowRes-1712070648.pdf">For Recruiters</a>
			</li>
			<li>
				<a alt="For Librarians" target="_blank" href="https://www.elsevier.com/products/sciencedirect/journals/direct-subscription-journals">For Librarians</a>
			</li>
			<li>
				<a alt="Privacy Policy" target="_blank" href="https://www.elsevier.com/legal/privacy-policy">Privacy Policy</a>
			</li>
			<li>
				<a alt="Terms and Conditions" target="_blank" href="https://www.elsevier.com/legal/elsevier-website-terms-and-conditions">Terms and Conditions</a>
			</li>
			<li>
				<a alt="Accessibility" href="/accessibility">Accessibility</a>
			</li>
		</ul>
	</div>
</div></div>

            

        
</div></div>

            
                </div>
            

        

    

        


    

            

            

            <div data-widget-id="779ad11f-7bcd-4a2a-a2cd-8c67b649cbb2" data-widget-name="UX3HTMLWidget"><style>
 
button#cookieBtn {
   text-decoration: none;
   background-color: transparent;
   color: #005789;
   font-weight: bold;
   padding: 0;
   margin: 0;
   font-size: 14px;
}
 
button#cookieBtn:hover {
   text-decoration: underline;
}

</style>
<script>
// HIDE ONETRUST BACKGROUND AFTER USER PRESSES ESCAPE KEY
setTimeout(function(){ 
   $("#onetrust-pc-sdk").on("keydown", function(e) { 
      if (e.keyCode == 27) {  // ESC key
         setTimeout(function(){ 
            $(".onetrust-pc-dark-filter").css("display","none");
         }, 100); 
      }
   });
}, 5000);
</script>

<span class="align-center copy-right">
<hr>
<p align="center"><b>The content on this site is intended for healthcare professionals and researchers across all fields of science.</b></p>
<hr>
<div>
<p>We use cookies to help provide and enhance our service and tailor content. To update your cookie settings, please visit the 
      <b><button id="cookieBtn" class="ot-sdk-show-settings" type="button" aria-label="Manage cookie consent preferences">Cookie Settings</button></b> for this site.<br>

All content on this site: Copyright © 2025 Elsevier Inc., its licensors, and contributors. <br>All rights are reserved, including those for text and data mining, AI training, and similar technologies. <br>For all open access content, the relevant licensing terms apply.
</p>
<ul class="rlist--inline">
			<li>
				<a alt="Privacy Policy" target="_blank" href="https://www.elsevier.com/legal/privacy-policy">Privacy Policy</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Terms and Conditions" target="_blank" href="https://www.elsevier.com/legal/elsevier-website-terms-and-conditions">Terms &amp; Conditions</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Accessibility" href="/accessibility">Accessibility</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Help and Support" target="_blank" href="https://service.elsevier.com/app/overview/cell-press/">Help &amp; Support</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Contact" href="/contact">Contact</a>
			</li>
			</ul>
<p>
<a href="https://www.relx.com"><img src="/pb/assets/raw/Health%20Advance/images/relx-1554393210983.png" alt="RELX" rel="nofollow" target="_new"></a></p>
</div>

</span></div>

            

        

    

        


    

            

            

            <div data-widget-id="c5e179c1-156e-41e3-af7f-5e307efa8acf" data-widget-name="UX3HTMLWidget"><style>#brandingRow{
    background-color:#fff;
    padding:7px 26px 7px 30px;
    color:#007DBC;
    font-size:14px;
    clear:both;
    width:100%
}
#brandingRow a{
    text-decoration:none;
    color:#007DBC
}
#brandingRow a:hover{
    text-decoration:underline
}
#brandingRowRight{
    display:inline;
    float:right;
    width:auto;
    padding:0;
    margin:0
}
svg#cp_logo {
  float: none;
  height: 36px;
  width: 36px;
  margin-left: 0px;
  margin-top: 0px;
}

footer hr {
  display: block;
}

@media screen and (max-width: 600px){
    #brandingRow{
        font-size:12px
    }
}
@media screen and (max-width: 750px){
    #brandingRow{
        padding:7px 18px 7px 16px
    }
    svg#cp_logo{
        margin-right:10px
    }
}
@media screen and (max-width: 800px) {
   a#cpIconLnk {
      display: none;
   }
}

.header__wrapper.clearfix .col-xs-6:first-of-type{
    width:55%
}
.header__wrapper.clearfix .col-xs-6:nth-of-type(2){
    width:45%
}
@media screen and (min-width: 800px) and (max-width: 1000px){
    .header__wrapper.clearfix .col-xs-6:first-of-type{
        width:75%
    }
    .header__wrapper.clearfix .col-xs-6:nth-of-type(2){
        width:25%
    }
}

</style>

<script>
var partnerx = false;
var overrideEndorsement = "";
var tmpj = $("input[name='journalCode']").attr("value");
	
if (window.location.pathname == "/molecular-therapy-family/home") {
	tmpj = "ymthe";
}
	
var ribbonx = "";
if (tmpj == "ajhg" || tmpj == "bpj" || tmpj == "molp" || tmpj == "xplc" || tmpj == "stemcr" || tmpj == "ymthe" || tmpj == "xhgg" || tmpj == "omtn" || tmpj == "omto" || tmpj == "omtm" || tmpj == "biophysreports" ||  tmpj == "bpr" || tmpj == "xinn" || tmpj == "omton") {
	partnerx = true;
	ribbonx = "<a href='https://www.cell.com' data-aa-name='cell press endorsement' data-aa-region='cp global header' title='Visit Cell.com'>A Cell Press partner journal</a>";
} else if (tmpj != "" && tmpj != "undefined" && tmpj != undefined) {
	ribbonx = "<a href='/' data-aa-name='cell press endorsement' data-aa-region='cp global header' title='Visit Cell.com'>A Cell Press journal</a>";
} else {
	ribbonx = "<a href='/' data-aa-name='cell press endorsement' data-aa-region='cp global header' title='Visit Cell.com'>A Cell Press resource</a>";
}
	
if (window.location.pathname == "/") {
	ribbonx = "";
}
if (overrideEndorsement != "") {
	ribbonx = overrideEndorsement;
}

$(".header__wrapper.clearfix").prepend("<div id='brandingRow'><a href='/' data-aa-name='science that inspires' data-aa-region='cp global header' title='Visit Cell.com'>Science that inspires</a><p id='brandingRowRight'>" + ribbonx + "</p></div>");
	
// handle scrolling effect on article pages
if ($("article").length > 0) {
	var scrollx;
	$(window).scroll(function() {
		scrollx = $(window).scrollTop();
		if (scrollx > 0) { 
			$("#brandingRow").css("display","none"); 
		} else {
			$("#brandingRow").css("display","block");
		}
	});
}

if (window.location.pathname == "/" || partnerx == false) {
               $("<a href='/home' id='cpIconLnk' data-aa-name='cell press logo' data-aa-region='cp global header' title='Cell Press' target='_blank'><svg id='cp_logo' version='1.1' xmlns='http://www.w3.org/2000/svg' xmlns:xlink='http://www.w3.org/1999/xlink' x='0px' y='0px' viewBox='0 0 100 98.1' style='enable-background:new 0 0 100 98.1;' xml:space='preserve' height='100' width='98.1'><path style='fill:#FFFFFF;' d='M69.09,0.09c-4.01-0.6-7.83,1.94-8.78,5.89l-3.14,13.11c-0.76,3.19-3.44,5.56-6.7,5.93l-18.6,2.13          C11.83,29.31-2.53,47.66,0.37,67.78c2.3,15.96,15.18,27.77,30.54,30.08c4.01,0.6,7.83-1.94,8.78-5.89l3.14-13.11 c0.76-3.19,3.44-5.56,6.69-5.93l18.6-2.13c20.04-2.17,34.41-20.52,31.5-40.65C97.32,14.21,84.45,2.39,69.09,0.09z M65.06,47.2  l-11.75,1.4c-1.98,0.21-3.62,1.65-4.08,3.59l-3.43,14.3c-1.01,4.22-4.5,7.39-8.8,8c-6.66,0.72-12.63-4.2-13.15-10.92                c-0.51-6.5,4.6-12.11,11.08-12.81l11.75-1.39c1.98-0.22,3.62-1.65,4.08-3.59l3.43-14.3c1.01-4.22,4.5-7.39,8.8-8          c6.66-0.72,12.63,4.19,13.15,10.92C76.66,40.89,71.55,46.5,65.06,47.2z'/></svg></a>").insertBefore(".header__wrapper.clearfix .journal-logos:eq(0)");
}


</script></div>

            

        

    

        


    

            

            

            <div data-widget-id="1af759b7-9bb6-4413-ada9-51a8333e8fb8" data-widget-name="UX3HTMLWidget"><style>
    /*This was added temporarily until the PB changelog tab issue got fixed. [LIT-444969 - c#4]*/
    [data-tab-id="changelogTab"] {
        display: none !important;
    }
    
    /*This was added temporarily until the PB changelog tab issue got fixed. [LIT-518839]*/
    @media (min-width: 769px) {
        .ux3 .quick-search .dropBlock [data-db-target-of],
        .quick-search .dropBlock [data-db-target-of] {
            width: 100vw !important;
        }
    }
</style></div>

            

        
</div></div>

            

        

        </div>
    

    

        


    

            

            

            <div data-widget-id="d7e0813b-a300-497a-ab37-9c3ddaf03e71" data-widget-name="UX3HTMLWidget"><!--
<div class="session-reminder hidden">
    <div class="session-reminder__overlay">
        <div class="session-reminder__dialog">
            <div class="session-reminder__dialog-inner">
                <div class="dialog-header" role="dialog" aria-labelledby="dialog-title" aria-describedby="dialog-desc">
                    <h2 class="dialog-header-title" id="dialog-title">
                        <span>Session Timeout</span>
                        <div class="dialog-timer">(2:00)</div>
                    </h2>
                    <button class="dialog-close" type="button" aria-label="close dialog">
                        <i class="icon-tools_close" aria-hidden="true"></i>
                    </button>
                </div>
                <div class="dialog-body">
                    <p class="dialog-desc" id="dialog-desc">Your session will expire shortly. If you are still working, click the ‘Keep Me Logged In’ button below. If you do not respond within the next minute, you will be automatically logged out.</p>
                    <p class="dialog-expired sr-only" aria-live="polite"></p>
                </div>
                <div class="dialog-footer">
                    <button class="session-reset" type="button">Keep Me Logged In</button>
                </div>
            </div>
        </div>
    </div>
</div>
<a class="sr-only" href="javascript:void(0)" aria-hidden="true"></a>
--></div>

            

        
        </div>
    </div>


































    



















        
        
            
                <script src="/products/marlin/cell/releasedAssets/js/main.bundle-90475670808d5d3eee41.js"></script>
            
        
        
        

    




<script type="text/javascript">
    var pageData =  {"journal":{"specialty":"Cell Press","issn":"0092-8674","name":"Cell"},"page":{"loadTimestamp":"1749586009705","server":"cell","environment":"prod","businessUnit":"ELS:RAP:JBS","noTracking":"","name":"fulltext:html","section":"Cell","language":"en","type":"CP-CA","productName":"JB"},"visitor":{"departmentName":"|ae:ScienceDirect Guests","accessType":"ae:ANON_GUEST|ae:ANON_GUEST","accountId":"ae:838949|ae:228598","accountName":"ae:Cell Guest Account|ae:ScienceDirect Guests","departmentId":"ae:977212|ae:291352","ipAddress":"189.120.74.179","loginStatus":"anonymous","userId":"ae:246228266|ae:12975512"},"content":[{"accessType":"article:free:openarchive","volumeNumber":"181","issueNumber":"2","viewState":"fulltext","format":"MIME-HTML","id":"pii:S0092867420303391","coverDate":"202004","type":"XOCS-JOURNAL:SCOPE-FULL","title":"G3BP1 Is a Tunable Switch that Triggers Phase Separation to Assemble Stress Granules","status":"S300"}]};
</script>

<script type="text/javascript">
    if(typeof pageDataTracker !== "undefined") {
        pageDataTracker.trackPageLoad();
    }
</script>
<script>(function(){function c(){var b=a.contentDocument||a.contentWindow.document;if(b){var d=b.createElement('script');d.innerHTML="window.__CF$cv$params={r:'94db7dc2ce6327fa',t:'MTc0OTU4NjAwOC4wMDAwMDA='};var a=document.createElement('script');a.nonce='';a.src='/cdn-cgi/challenge-platform/scripts/jsd/main.js';document.getElementsByTagName('head')[0].appendChild(a);";b.getElementsByTagName('head')[0].appendChild(d)}}if(document.body){var a=document.createElement('iframe');a.height=1;a.width=1;a.style.position='absolute';a.style.top=0;a.style.left=0;a.style.border='none';a.style.visibility='hidden';document.body.appendChild(a);if('loading'!==document.readyState)c();else if(window.addEventListener)document.addEventListener('DOMContentLoaded',c);else{var e=document.onreadystatechange||function(){};document.onreadystatechange=function(b){e(b);'loading'!==document.readyState&&(document.onreadystatechange=e,c())}}}})();</script><iframe height="1" width="1" style="position: absolute; top: 0px; left: 0px; border: none; visibility: hidden;"></iframe><script defer="" src="https://static.cloudflareinsights.com/beacon.min.js/vcd15cbe7772f49c399c6a5babf22c1241717689176015" integrity="sha512-ZpsOmlRQV6y907TI0dKBHq9Md29nnaEIPlkf84rnaERnq6zvWvPUqr2ft8M1aS28oN72PdrCzSjY4U6VaAw1EQ==" data-cf-beacon="{&quot;rayId&quot;:&quot;94db7dc2ce6327fa&quot;,&quot;serverTiming&quot;:{&quot;name&quot;:{&quot;cfExtPri&quot;:true,&quot;cfEdge&quot;:true,&quot;cfOrigin&quot;:true,&quot;cfL4&quot;:true,&quot;cfSpeedBrain&quot;:true,&quot;cfCacheStatus&quot;:true}},&quot;version&quot;:&quot;2025.5.0&quot;,&quot;token&quot;:&quot;57a02315956f4b2e82817b3dfa99b1a2&quot;}" crossorigin="anonymous"></script>


<div id="addtoany" style="position: static;"><div class="a2a_overlay" id="a2a_overlay"></div><div id="a2a_modal" class="a2a_modal a2a_hide" role="dialog" tabindex="-1" aria-label=""><div class="a2a_modal_body a2a_menu a2a_hide" id="a2a_copy_link"><label for="a2a_copy_link_text" id="a2a_copy_link_icon" class="a2a_svg a2a_s_link a2a_logo_color"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"><title>Copy link</title><path fill="#FFF" d="M7.591 21.177c0-.36.126-.665.377-.917l2.804-2.804a1.235 1.235 0 0 1 .913-.378c.377 0 .7.144.97.43-.026.028-.11.11-.255.25-.144.14-.24.236-.29.29a2.82 2.82 0 0 0-.2.256 1.056 1.056 0 0 0-.177.344 1.43 1.43 0 0 0-.046.37c0 .36.126.666.377.918a1.25 1.25 0 0 0 .918.377c.126.001.251-.015.373-.047.125-.037.242-.096.345-.175.09-.06.176-.127.256-.2.1-.094.196-.19.29-.29.14-.142.223-.23.25-.254.297.28.445.607.445.984 0 .36-.126.664-.377.916l-2.778 2.79a1.242 1.242 0 0 1-.917.364c-.36 0-.665-.118-.917-.35l-1.982-1.97a1.223 1.223 0 0 1-.378-.9l-.001-.004Zm9.477-9.504c0-.36.126-.665.377-.917l2.777-2.79a1.235 1.235 0 0 1 .913-.378c.35 0 .656.12.917.364l1.984 1.968c.254.252.38.553.38.903 0 .36-.126.665-.38.917l-2.802 2.804a1.238 1.238 0 0 1-.916.364c-.377 0-.7-.14-.97-.418.026-.027.11-.11.255-.25a7.5 7.5 0 0 0 .29-.29c.072-.08.139-.166.2-.255.08-.103.14-.22.176-.344.032-.12.048-.245.047-.37 0-.36-.126-.662-.377-.914a1.247 1.247 0 0 0-.917-.377c-.136 0-.26.015-.37.046-.114.03-.23.09-.346.175a3.868 3.868 0 0 0-.256.2c-.054.05-.15.148-.29.29-.14.146-.222.23-.25.258-.294-.278-.442-.606-.442-.983v-.003ZM5.003 21.177c0 1.078.382 1.99 1.146 2.736l1.982 1.968c.745.75 1.658 1.12 2.736 1.12 1.087 0 2.004-.38 2.75-1.143l2.777-2.79c.75-.747 1.12-1.66 1.12-2.737 0-1.106-.392-2.046-1.183-2.818l1.186-1.185c.774.79 1.708 1.186 2.805 1.186 1.078 0 1.995-.376 2.75-1.13l2.803-2.81c.751-.754 1.128-1.671 1.128-2.748 0-1.08-.382-1.993-1.146-2.738L23.875 6.12C23.13 5.372 22.218 5 21.139 5c-1.087 0-2.004.382-2.75 1.146l-2.777 2.79c-.75.747-1.12 1.66-1.12 2.737 0 1.105.392 2.045 1.183 2.817l-1.186 1.186c-.774-.79-1.708-1.186-2.805-1.186-1.078 0-1.995.377-2.75 1.132L6.13 18.426c-.754.755-1.13 1.672-1.13 2.75l.003.001Z"></path></svg></label><input id="a2a_copy_link_text" type="text" title="Copy link" readonly=""><div id="a2a_copy_link_copied">✓</div></div><div class="a2a_modal_body a2a_menu a2a_thanks a2a_hide" id="a2a_thanks"><div class="a2a_localize" data-a2a-localize="inner,ThanksForSharing">Thanks for sharing!</div></div></div><div class="a2a_menu a2a_full a2a_localize" id="a2apage_full" role="dialog" tabindex="-1" aria-label="Share" data-a2a-localize="title,Share"><div class="a2a_full_header"><div id="a2apage_find_container" class="a2a_menu_find_container"><label for="a2apage_find" id="a2apage_find_icon" class="a2a_svg a2a_s_find"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" aria-hidden="true"><title>Find any service</title><path fill="#CCC" d="M19.7 18.2l-4.5-4.5c.7-1.1 1.2-2.3 1.2-3.6 0-3.5-2.8-6.3-6.3-6.3s-6.3 2.8-6.3 6.3 2.8 6.3 6.3 6.3c1.4 0 2.6-.4 3.6-1.2l4.5 4.5c.6.6 1.3.7 1.7.2.5-.4.4-1.1-.2-1.7zm-9.6-3.6c-2.5 0-4.5-2.1-4.5-4.5 0-2.5 2.1-4.5 4.5-4.5 2.5 0 4.5 2.1 4.5 4.5s-2 4.5-4.5 4.5z"></path></svg></label><input id="a2apage_find" class="a2a_menu_find a2a_localize" type="text" autocomplete="off" title="Find any service" data-a2a-localize="title,FindAnyServiceToAddTo"></div></div><div class="a2a_full_services" id="a2apage_full_services" role="presentation"></div><div class="a2a_full_footer"><a href="https://www.addtoany.com" title="Share Buttons" rel="noopener" target="_blank"><span class="a2a_svg a2a_s__default a2a_s_a2a a2a_logo_color"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"><g fill="#FFF"><path d="M14 7h4v18h-4z"></path><path d="M7 14h18v4H7z"></path></g></svg></span>AddToAny</a></div></div><div id="a2apage_dropdown" class="a2a_menu a2a_mini a2a_localize a2a_hide" tabindex="-1" aria-label="Share" data-a2a-localize="label,Share"><div class="a2a_mini_services" id="a2apage_mini_services"></div><div id="a2apage_cols_container" class="a2a_cols_container"><div class="a2a_col1" id="a2apage_col1"></div><div id="a2apage_2_col1" class="a2a_hide"></div><div class="a2a_clear"></div></div><div class="a2apage_wide a2a_wide"><a href="#addtoany" id="a2apage_show_more_less" class="a2a_more a2a_localize" title="Show all" data-a2a-localize="title,ShowAll"><span class="a2a_svg a2a_s__default a2a_s_a2a a2a_logo_color"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"><g fill="#FFF"><path d="M14 7h4v18h-4z"></path><path d="M7 14h18v4H7z"></path></g></svg></span><span class="a2a_localize" data-a2a-localize="inner,More">More…</span></a></div></div><div style="height: 1px; width: 1px; position: absolute; z-index: 100000; top: 0px; visibility: hidden;"><iframe id="a2a_sm_ifr" title="AddToAny Utility Frame" aria-hidden="true" src="https://static.addtoany.com/menu/sm.25.html#type=core&amp;event=load" style="height: 1px; width: 1px; border: 0px; left: 0px; top: 0px; position: absolute; z-index: 100000; display: none;"></iframe></div></div><div id="onetrust-consent-sdk" data-nosnippet="true"><div class="onetrust-pc-dark-filter ot-fade-in" style="z-index:2147483645;"></div><div id="onetrust-banner-sdk" class="otFlat bottom ot-wo-title ot-buttons-fw" tabindex="0" role="region" aria-label="Cookie banner"><div role="dialog" aria-modal="true" aria-label="Privacy"><div class="ot-sdk-container"><div class="ot-sdk-row"><div id="onetrust-group-container" class="ot-sdk-eight ot-sdk-columns"><div class="banner_logo"></div><div id="onetrust-policy"><div id="onetrust-policy-text">We use cookies that are necessary to make our site work. We may also use additional cookies to analyze, improve, and personalize our content and your digital experience. For more information, see our<a class="ot-cookie-policy-link" href="https://www.elsevier.com/legal/cookienotice" target="_blank" aria-label=", opens in a new tab" rel="noopener">Cookie Policy</a></div></div></div><div id="onetrust-button-group-parent" class="ot-sdk-three ot-sdk-columns"><div id="onetrust-button-group"><button id="onetrust-pc-btn-handler" class="cookie-setting-link">Cookie Settings</button>  <button id="onetrust-accept-btn-handler">Accept all cookies</button></div></div></div></div><!-- Close Button --><div id="onetrust-close-btn-container"></div><!-- Close Button END--></div></div><div id="onetrust-pc-sdk" class="otPcCenter ot-hide ot-fade-in" lang="en" aria-label="Preference center" role="region"><div role="dialog" aria-modal="true" style="height: 100%;" aria-label="Cookie Preference Center"><!-- Close Button --><div class="ot-pc-header"><!-- Logo Tag --><div class="ot-pc-logo" role="img" aria-label="Company Logo"><img alt="Company Logo" src="https://cdn.cookielaw.org/logos/static/ot_company_logo.png"></div></div><!-- Close Button --><div id="ot-pc-content" class="ot-pc-scrollbar"><div class="ot-optout-signal ot-hide"><div class="ot-optout-icon"><svg xmlns="http://www.w3.org/2000/svg"><path class="ot-floating-button__svg-fill" d="M14.588 0l.445.328c1.807 1.303 3.961 2.533 6.461 3.688 2.015.93 4.576 1.746 7.682 2.446 0 14.178-4.73 24.133-14.19 29.864l-.398.236C4.863 30.87 0 20.837 0 6.462c3.107-.7 5.668-1.516 7.682-2.446 2.709-1.251 5.01-2.59 6.906-4.016zm5.87 13.88a.75.75 0 00-.974.159l-5.475 6.625-3.005-2.997-.077-.067a.75.75 0 00-.983 1.13l4.172 4.16 6.525-7.895.06-.083a.75.75 0 00-.16-.973z" fill="#FFF" fill-rule="evenodd"></path></svg></div><span></span></div><h2 id="ot-pc-title">Cookie Preference Center</h2><div id="ot-pc-desc">We use cookies which are necessary to make our site work. We may also use additional cookies to analyse, improve and personalise our content and your digital experience. For more information, see our <a href="https://www.elsevier.com/legal/cookienotice/_nocache" target="_blank">Cookie Policy</a> and the list of <a href="https://support.google.com/admanager/answer/9012903" target="_blank">Google Ad-Tech Vendors</a>.
<br>
<br>
You may choose not to allow some types of cookies. However, blocking some types may impact your experience of our site and the services we are able to offer. See the different category headings below to find out more or change your settings.
<br>
</div><button id="accept-recommended-btn-handler">Allow all</button><section class="ot-sdk-row ot-cat-grp"><h3 id="ot-category-title"> Manage Consent Preferences</h3><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="1"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-1" aria-labelledby="ot-header-id-1 ot-status-id-1"></button><!-- Accordion header --><div class="ot-acc-hdr ot-always-active-group"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-1">Strictly Necessary Cookies</h4><div id="ot-status-id-1" class="ot-always-active">Always active</div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-1">These cookies are necessary for the website to function and cannot be switched off in our systems. They are usually only set in response to actions made by you which amount to a request for services, such as setting your privacy preferences, logging in or filling in forms. You can set your browser to block or alert you about these cookies, but some parts of the site will not then work. These cookies do not store any personally identifiable information.
<br><br></p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="1">Cookie Details List‎</button></div></div></div><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="3"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-3" aria-labelledby="ot-header-id-3"></button><!-- Accordion header --><div class="ot-acc-hdr"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-3">Functional Cookies</h4><div class="ot-tgl"><input type="checkbox" name="ot-group-id-3" id="ot-group-id-3" role="switch" class="category-switch-handler" data-optanongroupid="3" aria-labelledby="ot-header-id-3"> <label class="ot-switch" for="ot-group-id-3"><span class="ot-switch-nob"></span> <span class="ot-label-txt">Functional Cookies</span></label> </div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-3">These cookies enable the website to provide enhanced functionality and personalisation. They may be set by us or by third party providers whose services we have added to our pages. If you do not allow these cookies then some or all of these services may not function properly.</p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="3">Cookie Details List‎</button></div></div></div><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="2"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-2" aria-labelledby="ot-header-id-2"></button><!-- Accordion header --><div class="ot-acc-hdr"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-2">Performance Cookies</h4><div class="ot-tgl"><input type="checkbox" name="ot-group-id-2" id="ot-group-id-2" role="switch" class="category-switch-handler" data-optanongroupid="2" aria-labelledby="ot-header-id-2"> <label class="ot-switch" for="ot-group-id-2"><span class="ot-switch-nob"></span> <span class="ot-label-txt">Performance Cookies</span></label> </div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-2">These cookies allow us to count visits and traffic sources so we can measure and improve the performance of our site. They help us to know which pages are the most and least popular and see how visitors move around the site.</p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="2">Cookie Details List‎</button></div></div></div><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="4"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-4" aria-labelledby="ot-header-id-4"></button><!-- Accordion header --><div class="ot-acc-hdr"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-4">Targeting Cookies</h4><div class="ot-tgl"><input type="checkbox" name="ot-group-id-4" id="ot-group-id-4" role="switch" class="category-switch-handler" data-optanongroupid="4" aria-labelledby="ot-header-id-4"> <label class="ot-switch" for="ot-group-id-4"><span class="ot-switch-nob"></span> <span class="ot-label-txt">Targeting Cookies</span></label> </div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-4">These cookies may be set through our site by our advertising partners. They may be used by those companies to build a profile of your interests and show you relevant adverts on other sites. If you do not allow these cookies, you will experience less targeted advertising.</p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="4">Cookie Details List‎</button></div></div></div><!-- Groups sections starts --><!-- Group section ends --><!-- Accordion Group section starts --><!-- Accordion Group section ends --></section></div><section id="ot-pc-lst" class="ot-hide ot-hosts-ui ot-pc-scrollbar"><div id="ot-pc-hdr"><div id="ot-lst-title"><button class="ot-link-btn back-btn-handler" aria-label="Back"><svg id="ot-back-arw" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 444.531 444.531" xml:space="preserve"><title>Back Button</title><g><path fill="#656565" d="M213.13,222.409L351.88,83.653c7.05-7.043,10.567-15.657,10.567-25.841c0-10.183-3.518-18.793-10.567-25.835
                    l-21.409-21.416C323.432,3.521,314.817,0,304.637,0s-18.791,3.521-25.841,10.561L92.649,196.425
                    c-7.044,7.043-10.566,15.656-10.566,25.841s3.521,18.791,10.566,25.837l186.146,185.864c7.05,7.043,15.66,10.564,25.841,10.564
                    s18.795-3.521,25.834-10.564l21.409-21.412c7.05-7.039,10.567-15.604,10.567-25.697c0-10.085-3.518-18.746-10.567-25.978
                    L213.13,222.409z"></path></g></svg></button><h3>Cookie List</h3></div><div class="ot-lst-subhdr"><div class="ot-search-cntr"><p role="status" class="ot-scrn-rdr"></p><input id="vendor-search-handler" type="text" name="vendor-search-handler" placeholder="Search…" aria-label="Cookie list search"> <svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 -30 110 110" aria-hidden="true"><title>Search Icon</title><path fill="#2e3644" d="M55.146,51.887L41.588,37.786c3.486-4.144,5.396-9.358,5.396-14.786c0-12.682-10.318-23-23-23s-23,10.318-23,23
            s10.318,23,23,23c4.761,0,9.298-1.436,13.177-4.162l13.661,14.208c0.571,0.593,1.339,0.92,2.162,0.92
            c0.779,0,1.518-0.297,2.079-0.837C56.255,54.982,56.293,53.08,55.146,51.887z M23.984,6c9.374,0,17,7.626,17,17s-7.626,17-17,17
            s-17-7.626-17-17S14.61,6,23.984,6z"></path></svg></div><div class="ot-fltr-cntr"><button id="filter-btn-handler" aria-label="Filter" aria-haspopup="true"><svg role="presentation" aria-hidden="true" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 402.577 402.577" xml:space="preserve"><title>Filter Icon</title><g><path fill="#fff" d="M400.858,11.427c-3.241-7.421-8.85-11.132-16.854-11.136H18.564c-7.993,0-13.61,3.715-16.846,11.136
      c-3.234,7.801-1.903,14.467,3.999,19.985l140.757,140.753v138.755c0,4.955,1.809,9.232,5.424,12.854l73.085,73.083
      c3.429,3.614,7.71,5.428,12.851,5.428c2.282,0,4.66-0.479,7.135-1.43c7.426-3.238,11.14-8.851,11.14-16.845V172.166L396.861,31.413
      C402.765,25.895,404.093,19.231,400.858,11.427z"></path></g></svg></button></div><div id="ot-anchor"></div><section id="ot-fltr-modal"><div id="ot-fltr-cnt"><button id="clear-filters-handler">Clear</button><div class="ot-fltr-scrlcnt ot-pc-scrollbar"><div class="ot-fltr-opts" role="list"><div class="ot-fltr-opt" role="listitem"><div class="ot-chkbox"><input id="chkbox-id" type="checkbox" class="category-filter-handler"> <label for="chkbox-id"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div></div></div><div class="ot-fltr-btns"><button id="filter-apply-handler">Apply</button> <button id="filter-cancel-handler">Cancel</button></div></div></div></section></div></div><section id="ot-lst-cnt" class="ot-host-cnt ot-pc-scrollbar"><div id="ot-sel-blk"><div class="ot-sel-all"><div class="ot-sel-all-hdr"><span class="ot-consent-hdr">Consent</span> <span class="ot-li-hdr">Leg.Interest</span></div><div class="ot-sel-all-chkbox"><div class="ot-chkbox" id="ot-selall-hostcntr"><input id="select-all-hosts-groups-handler" type="checkbox"> <label for="select-all-hosts-groups-handler"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div><div class="ot-chkbox" id="ot-selall-vencntr"><input id="select-all-vendor-groups-handler" type="checkbox"> <label for="select-all-vendor-groups-handler"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div><div class="ot-chkbox" id="ot-selall-licntr"><input id="select-all-vendor-leg-handler" type="checkbox"> <label for="select-all-vendor-leg-handler"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div></div></div></div><div class="ot-sdk-row"><div class="ot-sdk-column"><ul id="ot-host-lst"></ul></div></div></section></section><div class="ot-pc-footer ot-pc-scrollbar"><div class="ot-btn-container"> <button class="save-preference-btn-handler onetrust-close-btn-handler">Confirm my choices</button></div><!-- Footer logo --><div class="ot-pc-footer-logo"><a href="https://www.onetrust.com/products/cookie-consent/" target="_blank" rel="noopener noreferrer" aria-label="Powered by OneTrust Opens in a new Tab"><img alt="Powered by Onetrust" src="https://cdn.cookielaw.org/logos/static/powered_by_logo.svg" title="Powered by OneTrust Opens in a new Tab"></a></div></div><!-- Cookie subgroup container --><!-- Vendor list link --><!-- Cookie lost link --><!-- Toggle HTML element --><!-- Checkbox HTML --><!-- plus minus--><!-- Arrow SVG element --><!-- Accordion basic element --><span class="ot-scrn-rdr" aria-atomic="true" aria-live="polite"></span><!-- Vendor Service container and item template --></div><iframe class="ot-text-resize" sandbox="allow-same-origin" title="onetrust-text-resize" style="position: absolute; top: -50000px; width: 100em;" aria-hidden="true"></iframe></div></div><div role="status" aria-live="assertive" aria-relevant="additions" class="ui-helper-hidden-accessible"></div></body></html>