<html lang="en" class="pb-page fonts-loaded fonts-loaded" data-request-id="6a94ab10-2a2e-4248-8318-2d27c1971ce7" style="--article-sticky-top: 82.00019999999999px; --scroll-padding: 163.594px auto auto;"><head data-pb-dropzone="head"><meta name="pbContext" content=";product:product:elsevier\:product\:cell;website:website:cell-site;ctype:string:Journal Content;journal:journal:cell;pageGroup:string:Article View;page:string:Show Full Text;requestedJournal:journal:cell;issue:issue:pii\:S0092867421X00025;wgroup:string:Default Website Group;article:article:pii\:S009286742101446X;subPage:string:Full Text">




<!-- if there is any customization for Responsive Project widget  -->


























    
    







    
            
        



        
                <meta name="robots" content="noarchive,nocache">
            
    






    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=1, user-scalable=1">











        <meta property="og:title" content="Limb development genes underlie variation in human fingerprint patterns">
    


        <meta property="og:type" content="Article">
    


        <meta property="og:url" content="https://www.cell.com/cell/abstract/S0092-8674(21)01446-X">
    



        <meta property="og:site_name" content="Cell">
    


        <meta property="og:image" content="https://www.cell.com/cms/asset/7f316104-f2f6-49ae-93c1-e2881d411698/fx1.jpg">
    


        <meta property="og:description" content="Genome-wide association scans in Han Chinese individuals and a trans-ethnic meta-analysis
reveal genetic regions that are associated with specific fingerprint patterning; functional
studies in mouse models confirm a role of EVI1 in early limb development and ridge
patterning.">
    











        <meta name="twitter:title" content="Limb development genes underlie variation in human fingerprint patterns">
    




        <meta name="twitter:image" content="https://www.cell.com/cms/asset/7f316104-f2f6-49ae-93c1-e2881d411698/fx1.jpg">
    


        <meta name="twitter:description" content="Genome-wide association scans in Han Chinese individuals and a trans-ethnic meta-analysis
reveal genetic regions that are associated with specific fingerprint patterning; functional
studies in mouse models confirm a role of EVI1 in early limb development and ridge
patterning.">
    

        <meta name="twitter:card" content="summary_large_image">
    











    
    
            
            
        <link rel="canonical" href="https://www.cell.com/cell/fulltext/S0092-8674(21)01446-X">
    


<meta name="tdm-reservation" content="1">
<meta name="tdm-policy" content="https://www.elsevier.com/tdm/tdmrep-policy.json">

<title>Limb development genes underlie variation in human fingerprint patterns: Cell</title><meta name="citation_journal_title" content="Cell"> <meta name="citation_journal_abbrev" content="Cell"> <meta name="citation_publisher" content="Elsevier"> <meta name="citation_title" content="Limb development genes underlie variation in human fingerprint patterns"> <meta name="citation_date" content="2022/01/06"> <meta name="citation_volume" content="185"> <meta name="citation_issue" content="1"> <meta name="citation_firstpage" content="95"> <meta name="citation_lastpage" content="112.e18"> <meta name="citation_pmid" content="34995520"> <meta name="citation_abstract_html_url" content="https://www.cell.com/action/showAbstract?pii=S009286742101446X"> <meta name="citation_fulltext_html_url" content="https://www.cell.com/action/showFullText?pii=S009286742101446X"> <meta name="citation_pdf_url" content="https://www.cell.com/action/showPdf?pii=S009286742101446X"> <meta name="citation_issn" content="0092-8674"> <meta name="citation_issn" content="1097-4172"> <meta name="citation_language" content="English"> <meta name="citation_doi" content="10.1016/j.cell.2021.12.008"> <meta name="citation_pmid" content="34995520"> <meta name="citation_author" content="Jinxi Li"><meta name="citation_author_orcid" content="0000-0002-1366-9593"><meta name="citation_author_institution" content="State Key Laboratory of Genetic Engineering, Collaborative Innovation Center for Genetics and Development, School of Life Sciences, and Human Phenome Institute, Fudan University, Shanghai 200438, PRC"><meta name="citation_author_institution" content="CAS Key Laboratory of Computational Biology, Shanghai Institute of Nutrition and Health, University of Chinese Academy of Sciences, Chinese Academy of Sciences, Shanghai 200031, PRC"><meta name="citation_author" content="James D. Glover"><meta name="citation_author_orcid" content="0000-0003-4678-5643"><meta name="citation_author_institution" content="The Roslin Institute and Royal (Dick) School of Veterinary Studies, University of Edinburgh, Edinburgh, UK"><meta name="citation_author" content="Haiguo Zhang"><meta name="citation_author_institution" content="Ministry of Education Key Laboratory of Contemporary Anthropology, Department of Anthropology and Human Genetics, School of Life Sciences, Fudan University, Shanghai 200438, PRC"><meta name="citation_author" content="Meifang Peng"><meta name="citation_author_institution" content="CAS Key Laboratory of Computational Biology, Shanghai Institute of Nutrition and Health, University of Chinese Academy of Sciences, Chinese Academy of Sciences, Shanghai 200031, PRC"><meta name="citation_author_institution" content="Ministry of Education Key Laboratory of Contemporary Anthropology, Department of Anthropology and Human Genetics, School of Life Sciences, Fudan University, Shanghai 200438, PRC"><meta name="citation_author" content="Jingze Tan"><meta name="citation_author_institution" content="Ministry of Education Key Laboratory of Contemporary Anthropology, Department of Anthropology and Human Genetics, School of Life Sciences, Fudan University, Shanghai 200438, PRC"><meta name="citation_author" content="Chandana Basu Mallick"><meta name="citation_author_orcid" content="0000-0001-9459-2710"><meta name="citation_author_institution" content="The Roslin Institute and Royal (Dick) School of Veterinary Studies, University of Edinburgh, Edinburgh, UK"><meta name="citation_author_institution" content="Centre for Genetic Disorders, Institute of Science, Banaras Hindu University, Varanasi, Uttar Pradesh, India"><meta name="citation_author" content="Dan Hou"><meta name="citation_author_institution" content="Chinese Academy of Sciences Key Laboratory of Tissue Microenvironment and Tumor, Shanghai Institute of Nutrition and Health, Shanghai Institutes for Biological Sciences, University of Chinese Academy of Sciences, Chinese Academy of Sciences, Shanghai 200031, PRC"><meta name="citation_author" content="Yajun Yang"><meta name="citation_author_institution" content="Ministry of Education Key Laboratory of Contemporary Anthropology, Department of Anthropology and Human Genetics, School of Life Sciences, Fudan University, Shanghai 200438, PRC"><meta name="citation_author" content="Sijie Wu"><meta name="citation_author_institution" content="State Key Laboratory of Genetic Engineering, Collaborative Innovation Center for Genetics and Development, School of Life Sciences, and Human Phenome Institute, Fudan University, Shanghai 200438, PRC"><meta name="citation_author_institution" content="CAS Key Laboratory of Computational Biology, Shanghai Institute of Nutrition and Health, University of Chinese Academy of Sciences, Chinese Academy of Sciences, Shanghai 200031, PRC"><meta name="citation_author" content="Yu Liu"><meta name="citation_author_institution" content="CAS Key Laboratory of Computational Biology, Shanghai Institute of Nutrition and Health, University of Chinese Academy of Sciences, Chinese Academy of Sciences, Shanghai 200031, PRC"><meta name="citation_author" content="Qianqian Peng"><meta name="citation_author_institution" content="CAS Key Laboratory of Computational Biology, Shanghai Institute of Nutrition and Health, University of Chinese Academy of Sciences, Chinese Academy of Sciences, Shanghai 200031, PRC"><meta name="citation_author" content="Shijie C. Zheng"><meta name="citation_author_orcid" content="0000-0002-2902-3164"><meta name="citation_author_institution" content="CAS Key Laboratory of Computational Biology, Shanghai Institute of Nutrition and Health, University of Chinese Academy of Sciences, Chinese Academy of Sciences, Shanghai 200031, PRC"><meta name="citation_author" content="Edie I. Crosse"><meta name="citation_author_institution" content="Centre for Regenerative Medicine, University of Edinburgh, Edinburgh, UK"><meta name="citation_author" content="Alexander Medvinsky"><meta name="citation_author_institution" content="Centre for Regenerative Medicine, University of Edinburgh, Edinburgh, UK"><meta name="citation_author" content="Richard A. Anderson"><meta name="citation_author_institution" content="MRC Centre for Reproductive Health, Queens Medical Research Institute, University of Edinburgh, Edinburgh, UK"><meta name="citation_author" content="Helen Brown"><meta name="citation_author_institution" content="The Roslin Institute and Royal (Dick) School of Veterinary Studies, University of Edinburgh, Edinburgh, UK"><meta name="citation_author" content="Ziyu Yuan"><meta name="citation_author_institution" content="Fudan-Taizhou Institute of Health Sciences, Taizhou, Jiangsu 225326, PRC"><meta name="citation_author" content="Shen Zhou"><meta name="citation_author_institution" content="Shanghai Foreign Language School, Shanghai 200083, PRC"><meta name="citation_author" content="Yanqing Xu"><meta name="citation_author_institution" content="Forest Ridge School of the Sacred Heart, Bellevue, WA 98006, USA"><meta name="citation_author" content="John P. Kemp"><meta name="citation_author_orcid" content="0000-0002-9105-2249"><meta name="citation_author_institution" content="University of Queensland Diamantina Institute, University of Queensland, Brisbane, QLD, Australia"><meta name="citation_author" content="Yvonne Y.W. Ho"><meta name="citation_author_institution" content="QIMR Berghofer Medical Rese Institute, Brisbane, QLD, Australia"><meta name="citation_author" content="Danuta Z. Loesch"><meta name="citation_author_institution" content="Psychology Department, La Trobe University, Melbourne, VIC, Australia"><meta name="citation_author" content="Lizhong Wang"><meta name="citation_author_orcid" content="0000-0002-3571-8771"><meta name="citation_author_institution" content="WeGene, Shenzhen, Guangdong 518040, PRC"><meta name="citation_author" content="Yingxiang Li"><meta name="citation_author_orcid" content="0000-0003-0835-9280"><meta name="citation_author_institution" content="WeGene, Shenzhen, Guangdong 518040, PRC"><meta name="citation_author" content="Senwei Tang"><meta name="citation_author_institution" content="WeGene, Shenzhen, Guangdong 518040, PRC"><meta name="citation_author" content="Xiaoli Wu"><meta name="citation_author_institution" content="WeGene, Shenzhen, Guangdong 518040, PRC"><meta name="citation_author" content="Robin G. Walters"><meta name="citation_author_orcid" content="0000-0002-9179-0321"><meta name="citation_author_institution" content="Clinical Trial Service Unit and Epidemiological Studies Unit, Nuffield Department of Clinical Medicine, University of Oxford, Oxford, UK"><meta name="citation_author_institution" content="Medical Research Council Population Health Research Unit, Nuffield Department of Clinical Medicine, University of Oxford, Oxford, UK"><meta name="citation_author" content="Kuang Lin"><meta name="citation_author_institution" content="Clinical Trial Service Unit and Epidemiological Studies Unit, Nuffield Department of Clinical Medicine, University of Oxford, Oxford, UK"><meta name="citation_author" content="Ruogu Meng"><meta name="citation_author_institution" content="Center for Data Science in Health and Medicine, Peking University, Beijing 100191, PRC"><meta name="citation_author" content="Jun Lv"><meta name="citation_author_institution" content="Department of Epidemiology and Biostatistics, School of Public Health, Peking University Health Science Center, Beijing 100191, PRC"><meta name="citation_author" content="Jonathan M. Chernus"><meta name="citation_author_institution" content="Department of Human Genetics, University of Pittsburgh, Pittsburgh, PA 15261, USA"><meta name="citation_author" content="Katherine Neiswanger"><meta name="citation_author_institution" content="Center for Craniofacial and Dental Genetics, Department of Oral and Craniofacial Sciences, University of Pittsburgh, Pittsburgh, PA 15219, USA"><meta name="citation_author" content="Eleanor Feingold"><meta name="citation_author_institution" content="Department of Human Genetics, University of Pittsburgh, Pittsburgh, PA 15261, USA"><meta name="citation_author" content="David M. Evans"><meta name="citation_author_institution" content="University of Queensland Diamantina Institute, University of Queensland, Brisbane, QLD, Australia"><meta name="citation_author_institution" content="Institute for Molecular Bioscience, University of Queensland, Brisbane, QLD, Australia"><meta name="citation_author_institution" content="MRC Integrative Epidemiology Unit, University of Bristol, Bristol, UK"><meta name="citation_author" content="Sarah E. Medland"><meta name="citation_author_institution" content="QIMR Berghofer Medical Rese Institute, Brisbane, QLD, Australia"><meta name="citation_author" content="Nicholas G. Martin"><meta name="citation_author_institution" content="QIMR Berghofer Medical Rese Institute, Brisbane, QLD, Australia"><meta name="citation_author" content="Seth M. Weinberg"><meta name="citation_author_institution" content="Department of Human Genetics, University of Pittsburgh, Pittsburgh, PA 15261, USA"><meta name="citation_author_institution" content="Center for Craniofacial and Dental Genetics, Department of Oral and Craniofacial Sciences, University of Pittsburgh, Pittsburgh, PA 15219, USA"><meta name="citation_author_institution" content="Department of Anthropology, University of Pittsburgh, Pittsburgh, PA 15260, USA"><meta name="citation_author" content="Mary L. Marazita"><meta name="citation_author_institution" content="Department of Human Genetics, University of Pittsburgh, Pittsburgh, PA 15261, USA"><meta name="citation_author_institution" content="Center for Craniofacial and Dental Genetics, Department of Oral and Craniofacial Sciences, University of Pittsburgh, Pittsburgh, PA 15219, USA"><meta name="citation_author_institution" content="Clinical and Translational Science, University of Pittsburgh, Pittsburgh, PA 15261, USA"><meta name="citation_author_institution" content="Department of Psychiatry, University of Pittsburgh, Pittsburgh, PA 15261, USA"><meta name="citation_author" content="Gang Chen"><meta name="citation_author_institution" content="WeGene, Shenzhen, Guangdong 518040, PRC"><meta name="citation_author" content="Zhengming Chen"><meta name="citation_author_institution" content="Clinical Trial Service Unit and Epidemiological Studies Unit, Nuffield Department of Clinical Medicine, University of Oxford, Oxford, UK"><meta name="citation_author_institution" content="Medical Research Council Population Health Research Unit, Nuffield Department of Clinical Medicine, University of Oxford, Oxford, UK"><meta name="citation_author" content="Yong Zhou"><meta name="citation_author_institution" content="Clinical Research Institute, Shanghai General Hospital, Shanghai Jiao Tong University School of Medicine, Shanghai, PRC"><meta name="citation_author" content="Michael Cheeseman"><meta name="citation_author_orcid" content="0000-0001-5968-0176"><meta name="citation_author_institution" content="The Roslin Institute and Royal (Dick) School of Veterinary Studies, University of Edinburgh, Edinburgh, UK"><meta name="citation_author" content="Lan Wang"><meta name="citation_author_institution" content="Chinese Academy of Sciences Key Laboratory of Tissue Microenvironment and Tumor, Shanghai Institute of Nutrition and Health, Shanghai Institutes for Biological Sciences, University of Chinese Academy of Sciences, Chinese Academy of Sciences, Shanghai 200031, PRC"><meta name="citation_author" content="Li Jin"><meta name="citation_author_email" content="lijin@fudan.edu.cn"><meta name="citation_author_institution" content="State Key Laboratory of Genetic Engineering, Collaborative Innovation Center for Genetics and Development, School of Life Sciences, and Human Phenome Institute, Fudan University, Shanghai 200438, PRC"><meta name="citation_author_institution" content="CAS Key Laboratory of Computational Biology, Shanghai Institute of Nutrition and Health, University of Chinese Academy of Sciences, Chinese Academy of Sciences, Shanghai 200031, PRC"><meta name="citation_author_institution" content="Research Unit of Dissecting the Population Genetics and Developing New Technologies for Treatment and Prevention of Skin Phenotypes and Dermatological Diseases (2019RU058), Chinese Academy of Medical Sciences, Shanghai 200438, PRC"><meta name="citation_author" content="Denis J. Headon"><meta name="citation_author_email" content="denis.headon@roslin.ed.ac.uk"><meta name="citation_author_orcid" content="0000-0002-0452-4480"><meta name="citation_author_institution" content="The Roslin Institute and Royal (Dick) School of Veterinary Studies, University of Edinburgh, Edinburgh, UK"><meta name="citation_author" content="Sijia Wang"><meta name="citation_author_email" content="wangsijia@picb.ac.cn"><meta name="citation_author_orcid" content="0000-0001-6961-7867"><meta name="citation_author_institution" content="CAS Key Laboratory of Computational Biology, Shanghai Institute of Nutrition and Health, University of Chinese Academy of Sciences, Chinese Academy of Sciences, Shanghai 200031, PRC"><meta name="citation_author_institution" content="Center for Excellence in Animal Evolution and Genetics, Chinese Academy of Sciences, Kunming 650223, PRC"><meta name="citation_keywords" content="fingerprint pattern; genetics; genome-wide association study; trans-ethnic meta-analysis; limb development; EVI1"><meta name="citation_reference" content="citation_title=Merlin--rapid analysis of dense genetic maps using sparse gene flow trees; citation_author=G.R. Abecasis; citation_author=S.S. Cherny; citation_author=W.O. Cookson; citation_author=L.R. Cardon; citation_journal_title=Nat. Genet.; citation_volume=30; citation_firstpage=97; citation_lastpage=101; citation_publication_date=2002; "><meta name="citation_reference" content="citation_title=The Ensembl gene annotation system; citation_author=B.L. Aken; citation_author=S. Ayling; citation_author=D. Barrell; citation_author=L. Clarke; citation_author=V. Curwen; citation_author=S. Fairley; citation_author=J. Fernandez Banet; citation_author=K. Billis; citation_author=C. García Girón; citation_author=T. Hourlier; citation_author=et al.; citation_journal_title=Database (Oxford); citation_volume=2016; citation_firstpage=baw093; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Genomic features of human limb specific enhancers; citation_author=S. Ali; citation_author=B. Amina; citation_author=S. Anwar; citation_author=R. Minhas; citation_author=N. Parveen; citation_author=U. Nawaz; citation_author=S.S. Azam; citation_author=A.A. Abbasi; citation_journal_title=Genomics; citation_volume=108; citation_firstpage=143; citation_lastpage=150; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Integrating common and rare genetic variation in diverse human populations; citation_author=D.M. Altshuler; citation_author=R.A. Gibbs; citation_author=L. Peltonen; citation_author=E. Dermitzakis; citation_author=S.F. Schaffner; citation_author=F. Yu; citation_author=L. Peltonen; citation_author=International HapMap 3 Consortium; citation_author=et al.; citation_journal_title=Nature; citation_volume=467; citation_firstpage=52; citation_lastpage=58; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=Fingertip moisture is optimally modulated during object manipulation; citation_author=T. André; citation_author=P. Lefèvre; citation_author=J.L. Thonnard; citation_journal_title=J.&nbsp;Neurophysiol.; citation_volume=103; citation_firstpage=402; citation_lastpage=408; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=Twin study of digital dermatoglyphic traits: Investigation of heritability; citation_author=M.I. Arrieta; citation_author=L. Salazar; citation_author=B. Criado; citation_author=B. Martinez; citation_author=C.M. Lostao; citation_journal_title=Am. J. Hum. Biol.; citation_volume=3; citation_firstpage=11; citation_lastpage=15; citation_publication_date=1991; "><meta name="citation_reference" content="citation_title=Mutations in the gene encoding the basal body protein RPGRIP1L, a nephrocystin-4 interactor, cause Joubert syndrome; citation_author=H.H. Arts; citation_author=D. Doherty; citation_author=S.E. van Beersum; citation_author=M.A. Parisi; citation_author=S.J. Letteboer; citation_author=N.T. Gorden; citation_author=T.A. Peters; citation_author=T. Märker; citation_author=K. Voesenek; citation_author=A. Kartono; citation_author=et al.; citation_journal_title=Nat. Genet.; citation_volume=39; citation_firstpage=882; citation_lastpage=888; citation_publication_date=2007; "><meta name="citation_reference" content="citation_title=Gene ontology: tool for the unification of biology; citation_author=M. Ashburner; citation_author=C.A. Ball; citation_author=J.A. Blake; citation_author=D. Botstein; citation_author=H. Butler; citation_author=J.M. Cherry; citation_author=A.P. Davis; citation_author=K. Dolinski; citation_author=S.S. Dwight; citation_author=J.T. Eppig; citation_author=The Gene Ontology Consortium; citation_author=et al.; citation_journal_title=Nat. Genet.; citation_volume=25; citation_firstpage=25; citation_lastpage=29; citation_publication_date=2000; "><meta name="citation_reference" content="citation_title=Prenatal development of dermatoglyphic digital patterns: Associations with epidermal ridge, volar pad and bone morphology; citation_author=W. Babler; citation_journal_title=Coll. Antropol.; citation_volume=11; citation_firstpage=297; citation_lastpage=303; citation_publication_date=1987; "><meta name="citation_reference" content="citation_title=Embryologic development of epidermal ridges and their configurations; citation_author=W.J. Babler; citation_journal_title=Birth Defects Orig. Artic. Ser.; citation_volume=27; citation_firstpage=95; citation_lastpage=112; citation_publication_date=1991; "><meta name="citation_reference" content="citation_title=Dbx2 regulation in limbs suggests interTAD sharing of enhancers; citation_author=L. Beccari; citation_author=G. Jaquier; citation_author=L. Lopez-Delisle; citation_author=E. Rodriguez-Carballo; citation_author=B. Mascrez; citation_author=S. Gitto; citation_author=J. Woltering; citation_author=D. Duboule; citation_journal_title=Dev. Dyn.; citation_volume=250; citation_firstpage=1280; citation_lastpage=1299; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=The NIH Roadmap Epigenomics Mapping Consortium; citation_author=B.E. Bernstein; citation_author=J.A. Stamatoyannopoulos; citation_author=J.F. Costello; citation_author=B. Ren; citation_author=A. Milosavljevic; citation_author=A. Meissner; citation_author=M. Kellis; citation_author=M.A. Marra; citation_author=A.L. Beaudet; citation_author=J.R. Ecker; citation_author=et al.; citation_journal_title=Nat. Biotechnol.; citation_volume=28; citation_firstpage=1045; citation_lastpage=1048; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=Studies on papillary patterns of human fingers; citation_author=K. Bonnevie; citation_journal_title=J.&nbsp;Genet.; citation_volume=15; citation_firstpage=1; citation_lastpage=111; citation_publication_date=1924; "><meta name="citation_reference" content="citation_title=Cohort Profile: the ‘children of the 90s’--the index offspring of the Avon Longitudinal Study of Parents and Children; citation_author=A. Boyd; citation_author=J. Golding; citation_author=J. Macleod; citation_author=D.A. Lawlor; citation_author=A. Fraser; citation_author=J. Henderson; citation_author=L. Molloy; citation_author=A. Ness; citation_author=S. Ring; citation_author=G. Davey Smith; citation_journal_title=Int. J. Epidemiol.; citation_volume=42; citation_firstpage=111; citation_lastpage=127; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=Haploinsufficiency of ANO6, NELL2 and DBX2 in a boy with intellectual disability and growth delay; citation_author=E.Ø. Carlsen; citation_author=E. Frengen; citation_author=M. Fannemel; citation_author=D. Misceo; citation_journal_title=Am. J. Med. Genet. A.; citation_volume=167A; citation_firstpage=1890; citation_lastpage=1896; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Expression patterns of the four nuclear factor I genes during mouse embryogenesis indicate a potential role in development; citation_author=A.Z. Chaudhry; citation_author=G.E. Lyons; citation_author=R.M. Gronostajski; citation_journal_title=Dev. Dyn.; citation_volume=208; citation_firstpage=313; citation_lastpage=325; citation_publication_date=1997; "><meta name="citation_reference" content="citation_title=Investigation of Prx1 protein expression provides evidence for conservation of cardiac-specific posttranscriptional regulation in vertebrates; citation_author=E.S. Chesterman; citation_author=G.D. Gainey; citation_author=A.C. Varn; citation_author=R.E. Peterson; citation_author=M.J. Kern; citation_journal_title=Dev. Dyn.; citation_volume=222; citation_firstpage=459; citation_lastpage=470; citation_publication_date=2001; "><meta name="citation_reference" content="citation_title=TCF-4 binds beta-catenin and is expressed in distinct regions of the embryonic brain and limbs; citation_author=E.A. Cho; citation_author=G.R. Dressler; citation_journal_title=Mech. Dev.; citation_volume=77; citation_firstpage=9; citation_lastpage=18; citation_publication_date=1998; "><meta name="citation_reference" content="citation_title=PRSice-2: Polygenic Risk Score software for biobank-scale data; citation_author=S.W. Choi; citation_author=P.F. O’Reilly; citation_journal_title=Gigascience; citation_volume=8; citation_firstpage=giz082; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Genome-wide mapping of global-to-local genetic effects on human facial shape; citation_author=P. Claes; citation_author=J. Roosenboom; citation_author=J.D. White; citation_author=T. Swigut; citation_author=D. Sero; citation_author=J. Li; citation_author=M.K. Lee; citation_author=A. Zaidi; citation_author=B.C. Mattern; citation_author=C. Liebowitz; citation_author=et al.; citation_journal_title=Nat. Genet.; citation_volume=50; citation_firstpage=414; citation_lastpage=423; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Control of skeletal patterning by ephrinB1-EphB interactions; citation_author=A. Compagni; citation_author=M. Logan; citation_author=R. Klein; citation_author=R.H. Adams; citation_journal_title=Dev. Cell; citation_volume=5; citation_firstpage=217; citation_lastpage=230; citation_publication_date=2003; "><meta name="citation_reference" content="citation_title=The genetics of dermal ridges; citation_author=H. Cummins; citation_journal_title=Am. J. Hum. Genet.; citation_volume=21; citation_firstpage=516; citation_publication_date=1969; "><meta name="citation_reference" content="citation_title=Palmar and plantar epidermal ridge configurations (dermatoglyphics) in European-Americans; citation_author=H. Cummins; citation_author=C. Midlo; citation_journal_title=Am. J. Phys. Anthropol.; citation_volume=9; citation_firstpage=471; citation_lastpage=502; citation_publication_date=1926; "><meta name="citation_reference" content="citation_title=The Encyclopedia of DNA elements (ENCODE): data portal update; citation_author=C.A. Davis; citation_author=B.C. Hitz; citation_author=C.A. Sloan; citation_author=E.T. Chan; citation_author=J.M. Davidson; citation_author=I. Gabdank; citation_author=J.A. Hilton; citation_author=K. Jain; citation_author=U.K. BBaymuradov; citation_author=A.K. Narayanan; citation_author=et al.; citation_journal_title=Nucleic Acids Res; citation_volume=46; citation_firstpage=D794; citation_lastpage=D801; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=A linear complexity phasing method for thousands of genomes; citation_author=O. Delaneau; citation_author=J. Marchini; citation_author=J.F. Zagury; citation_journal_title=Nat. Methods; citation_volume=9; citation_firstpage=179; citation_lastpage=181; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=The ciliary gene RPGRIP1L is mutated in cerebello-oculo-renal syndrome (Joubert syndrome type B) and Meckel syndrome; citation_author=M. Delous; citation_author=L. Baala; citation_author=R. Salomon; citation_author=C. Laclef; citation_author=J. Vierkotten; citation_author=K. Tory; citation_author=C. Golzio; citation_author=T. Lacoste; citation_author=L. Besse; citation_author=C. Ozilou; citation_author=et al.; citation_journal_title=Nat. Genet.; citation_volume=39; citation_firstpage=875; citation_lastpage=881; citation_publication_date=2007; "><meta name="citation_reference" content="citation_title=High-throughput discovery of novel developmental phenotypes; citation_author=M.E. Dickinson; citation_author=A.M. Flenniken; citation_author=X. Ji; citation_author=L. Teboul; citation_author=M.D. Wong; citation_author=J.K. White; citation_author=T.F. Meehan; citation_author=W.J. Weninger; citation_author=H. Westerberg; citation_author=H. Adissu; citation_author=International Mouse Phenotyping Consortium; citation_author=Jackson Laboratory; citation_author=Infrastructure Nationale PHENOMIN, Institut Clinique de la Souris (ICS); citation_author=Charles River Laboratories; citation_author=MRC Harwell; citation_author=Toronto Centre for Phenogenomics; citation_author=Wellcome Trust Sanger Institute; citation_author=RIKEN BioResource Center; citation_author=et al.; citation_journal_title=Nature; citation_volume=537; citation_firstpage=508; citation_lastpage=514; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=RBPjkappa-dependent Notch signaling regulates mesenchymal progenitor cell proliferation and differentiation during skeletal development; citation_author=Y. Dong; citation_author=A.M. Jesse; citation_author=A. Kohn; citation_author=L.M. Gunnell; citation_author=T. Honjo; citation_author=M.J. Zuscik; citation_author=R.J. O’Keefe; citation_author=M.J. Hilton; citation_journal_title=Development; citation_volume=137; citation_firstpage=1461; citation_lastpage=1471; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=A new Adamts9 conditional mouse allele identifies its non-redundant role in interdigital web regression; citation_author=J. Dubail; citation_author=N. Aramaki-Hattori; citation_author=H.L. Bader; citation_author=C.M. Nelson; citation_author=N. Katebi; citation_author=B. Matuska; citation_author=B.R. Olsen; citation_author=S.S. Apte; citation_journal_title=Genesis; citation_volume=52; citation_firstpage=702; citation_lastpage=712; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Mouse Genome Informatics (MGI): Resources for Mining Mouse Genetic, Genomic, and Biological Data in Support of Primary and Translational Research; citation_author=J.T. Eppig; citation_author=C.L. Smith; citation_author=J.A. Blake; citation_author=M. Ringwald; citation_author=C.J. Bult; citation_publication_date=2017; citation_publisher=Springer New York; "><meta name="citation_reference" content="citation_title=Genome-wide association study of primary tooth eruption identifies pleiotropic loci associated with height and craniofacial distances; citation_author=G. Fatemifar; citation_author=C.J. Hoggart; citation_author=L. Paternoster; citation_author=J.P. Kemp; citation_author=I. Prokopenko; citation_author=M. Horikoshi; citation_author=V.J. Wright; citation_author=J.H. Tobias; citation_author=S. Richmond; citation_author=A.I. Zhurov; citation_author=et al.; citation_journal_title=Hum. Mol. Genet.; citation_volume=22; citation_firstpage=3807; citation_lastpage=3817; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=Identification and analysis of a conserved Tcfap2a intronic enhancer element required for expression in facial and limb bud mesenchyme; citation_author=W. Feng; citation_author=J. Huang; citation_author=J. Zhang; citation_author=T. Williams; citation_journal_title=Mol. Cell. Biol.; citation_volume=28; citation_firstpage=315; citation_lastpage=325; citation_publication_date=2008; "><meta name="citation_reference" content="citation_title=A sensitive and bright single-cell resolution live imaging reporter of Wnt/ß-catenin signaling in the mouse; citation_author=A. Ferrer-Vaquer; citation_author=A. Piliszek; citation_author=G. Tian; citation_author=R.J. Aho; citation_author=D. Dufort; citation_author=A.K. Hadjantonakis; citation_journal_title=BMC Dev. Biol.; citation_volume=10; citation_firstpage=121; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=Cohort Profile: the Avon Longitudinal Study of Parents and Children: ALSPAC mothers cohort; citation_author=A. Fraser; citation_author=C. Macdonald-Wallis; citation_author=K. Tilling; citation_author=A. Boyd; citation_author=J. Golding; citation_author=G. Davey Smith; citation_author=J. Henderson; citation_author=J. Macleod; citation_author=L. Molloy; citation_author=A. Ness; citation_author=et al.; citation_journal_title=Int. J. Epidemiol.; citation_volume=42; citation_firstpage=97; citation_lastpage=110; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=The UCSC Genome Browser database: update 2011; citation_author=P.A. Fujita; citation_author=B. Rhead; citation_author=A.S. Zweig; citation_author=A.S. Hinrichs; citation_author=D. Karolchik; citation_author=M.S. Cline; citation_author=M. Goldman; citation_author=G.P. Barber; citation_author=H. Clawson; citation_author=A. Coelho; citation_author=et al.; citation_journal_title=Nucleic Acids Res.; citation_volume=39; citation_firstpage=D876; citation_lastpage=D882; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=Finger Prints; citation_author=F. Galton; citation_publication_date=1892; citation_publisher=Macmillan and Company; "><meta name="citation_reference" content="citation_title=Planar cell polarity in vertebrate limb morphogenesis; citation_author=B. Gao; citation_author=Y. Yang; citation_journal_title=Curr. Opin. Genet. Dev.; citation_volume=23; citation_firstpage=438; citation_lastpage=444; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=A biochemical hypothesis on the formation of fingerprints using a turing patterns approach; citation_author=D.A. Garzón-Alvarado; citation_author=A.M. Ramírez Martinez; citation_journal_title=Theor. Biol. Med. Model.; citation_volume=8; citation_firstpage=24; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=The transition zone protein Rpgrip1l regulates proteasomal activity at the primary cilium; citation_author=C. Gerhardt; citation_author=J.M. Lier; citation_author=S. Burmühl; citation_author=A. Struchtrup; citation_author=K. Deutschmann; citation_author=M. Vetter; citation_author=T. Leu; citation_author=S. Reeg; citation_author=T. Grune; citation_author=U. Rüther; citation_journal_title=J.&nbsp;Cell Biol.; citation_volume=210; citation_firstpage=115; citation_lastpage=133; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Evidence of a role for T-box genes in the evolution of limb morphogenesis and the specification of forelimb/hindlimb identity; citation_author=J.J. Gibson-Brown; citation_author=S.I. Agulnik; citation_author=D.L. Chapman; citation_author=M. Alexiou; citation_author=N. Garvey; citation_author=L.M. Silver; citation_author=V.E. Papaioannou; citation_journal_title=Mech. Dev.; citation_volume=56; citation_firstpage=93; citation_lastpage=101; citation_publication_date=1996; "><meta name="citation_reference" content="citation_title=The role of EVI1 in myeloid malignancies; citation_author=C. Glass; citation_author=M. Wilson; citation_author=R. Gonzalez; citation_author=Y. Zhang; citation_author=A.S. Perkins; citation_journal_title=Blood Cells Mol. Dis.; citation_volume=53; citation_firstpage=67; citation_lastpage=76; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Genetic effects on gene expression across human tissues; citation_author=GTEx Consortium; citation_journal_title=Nature; citation_volume=550; citation_firstpage=204; citation_lastpage=213; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Primate origins: evolutionary change in digital ray patterning and segmentation; citation_author=M.W. Hamrick; citation_journal_title=J.&nbsp;Hum. Evol.; citation_volume=40; citation_firstpage=339; citation_lastpage=351; citation_publication_date=2001; "><meta name="citation_reference" content="citation_title=Random-effects model aimed at discovering associations in meta-analysis of genome-wide association studies; citation_author=B. Han; citation_author=E. Eskin; citation_journal_title=Am. J. Hum. Genet.; citation_volume=88; citation_firstpage=586; citation_lastpage=598; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=GENCODE: the reference human genome annotation for The ENCODE Project; citation_author=J. Harrow; citation_author=A. Frankish; citation_author=J.M. Gonzalez; citation_author=E. Tapanari; citation_author=M. Diekhans; citation_author=F. Kokocinski; citation_author=B.L. Aken; citation_author=D. Barrell; citation_author=A. Zadissa; citation_author=S. Searle; citation_author=et al.; citation_journal_title=Genome research; citation_volume=22; citation_firstpage=1760; citation_lastpage=1774; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=Involvement of a novel Tnf receptor homologue in hair follicle induction; citation_author=D.J. Headon; citation_author=P.A. Overbeek; citation_journal_title=Nat. Genet.; citation_volume=22; citation_firstpage=370; citation_lastpage=374; citation_publication_date=1999; "><meta name="citation_reference" content="citation_title=Multiple roles of mesenchymal beta-catenin during murine limb patterning; citation_author=T.P. Hill; citation_author=M.M. Taketo; citation_author=W. Birchmeier; citation_author=C. Hartmann; citation_journal_title=Development; citation_volume=133; citation_firstpage=1219; citation_lastpage=1229; citation_publication_date=2006; "><meta name="citation_reference" content="citation_title=Morphological evidence concerning the problem of skin ridge formation; citation_author=W. Hirsch; citation_author=J.U. Schweichel; citation_journal_title=J.&nbsp;Ment. Defic. Res.; citation_volume=17; citation_firstpage=58; citation_lastpage=72; citation_publication_date=1973; "><meta name="citation_reference" content="citation_title=Common Genetic Variants Influence Whorls in Fingerprint Patterns; citation_author=Y.Y.W. Ho; citation_author=D.M. Evans; citation_author=G.W. Montgomery; citation_author=A.K. Henders; citation_author=J.P. Kemp; citation_author=N.J. Timpson; citation_author=B. St Pourcain; citation_author=A.C. Heath; citation_author=P.A.F. Madden; citation_author=D.Z. Loesch; citation_author=et al.; citation_journal_title=J.&nbsp;Invest. Dermatol.; citation_volume=136; citation_firstpage=859; citation_lastpage=862; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Distinct but overlapping expression patterns of two vertebrate slit homologs implies functional roles in CNS development and organogenesis; citation_author=G.P. Holmes; citation_author=K. Negus; citation_author=L. Burridge; citation_author=S. Raman; citation_author=E. Algar; citation_author=T. Yamada; citation_author=M.H. Little; citation_journal_title=Mech. Dev.; citation_volume=79; citation_firstpage=57; citation_lastpage=72; citation_publication_date=1998; "><meta name="citation_reference" content="citation_title=The Genetics of Dermal Ridges; citation_author=S.B. Holt; citation_author=L.S. Penrose; citation_publication_date=1968; citation_publisher=Charles C. Thomas; "><meta name="citation_reference" content="citation_title=A flexible and accurate genotype imputation method for the next generation of genome-wide association studies; citation_author=B.N. Howie; citation_author=P. Donnelly; citation_author=J. Marchini; citation_journal_title=PLoS Genet.; citation_volume=5; citation_firstpage=e1000529; citation_publication_date=2009; "><meta name="citation_reference" content="citation_title=The Evi1 proto-oncogene is required at midgestation for neural, heart, and paraxial mesenchyme development; citation_author=P.R. Hoyt; citation_author=C. Bartholomew; citation_author=A.J. Davis; citation_author=K. Yutzey; citation_author=L.W. Gamer; citation_author=S.S. Potter; citation_author=J.N. Ihle; citation_author=M.L. Mucenski; citation_journal_title=Mech. Dev.; citation_volume=65; citation_firstpage=55; citation_lastpage=70; citation_publication_date=1997; "><meta name="citation_reference" content="citation_title=The International HapMap Project; citation_author=The International HapMap Consortium; citation_journal_title=Nature; citation_volume=426; citation_firstpage=789; citation_lastpage=796; citation_publication_date=2003; "><meta name="citation_reference" content="citation_title=Integrating common and rare genetic variation in diverse human populations; citation_author=International HapMap; citation_author=C. Altshuler; citation_author=D.M. Gibbs; citation_author=R.A. Peltonen; citation_author=L. Altshuler; citation_author=D.M. Gibbs; citation_author=R.A. Peltonen; citation_author=L. Dermitzakis; citation_author=E. Schaffner; citation_author=et al.; citation_journal_title=Nature; citation_volume=467; citation_firstpage=52; citation_lastpage=58; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=Variance component model to account for sample structure in genome-wide association studies; citation_author=H.M. Kang; citation_author=J.H. Sul; citation_author=S.K. Service; citation_author=N.A. Zaitlen; citation_author=S.Y. Kong; citation_author=N.B. Freimer; citation_author=C. Sabatti; citation_author=E. Eskin; citation_journal_title=Nat. Genet.; citation_volume=42; citation_firstpage=348; citation_lastpage=354; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=Hypohidrotic ectodermal dysplasia: dental, clinical, genetic and dermatoglyphic findings of three cases; citation_author=B. Kargül; citation_author=T. Alcan; citation_author=U. Kabalay; citation_author=M. Atasu; citation_journal_title=J.&nbsp;Clin. Pediatr. Dent.; citation_volume=26; citation_firstpage=5; citation_lastpage=12; citation_publication_date=2001; "><meta name="citation_reference" content="citation_title=Inheritance of 18 quantitative dermatoglyphic traits based on factors in MZ and DZ twins; citation_author=B. Karmakar; citation_author=I. Malkin; citation_author=E. Kobyliansky; citation_journal_title=Anthropol. Anz.; citation_volume=68; citation_firstpage=185; citation_lastpage=193; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=Sp8 and Sp9, two closely related buttonhead-like transcription factors, regulate Fgf8 expression and limb outgrowth in vertebrate embryos; citation_author=Y. Kawakami; citation_author=C.R. Esteban; citation_author=T. Matsui; citation_author=J. Rodríguez-León; citation_author=S. Kato; citation_author=J.C. Izpisúa Belmonte; citation_journal_title=Development; citation_volume=131; citation_firstpage=4763; citation_lastpage=4774; citation_publication_date=2004; "><meta name="citation_reference" content="citation_title=Sall genes regulate region-specific morphogenesis in the mouse limb by modulating Hox activities; citation_author=Y. Kawakami; citation_author=Y. Uchiyama; citation_author=C. Rodriguez Esteban; citation_author=T. Inenaga; citation_author=N. Koyano-Nakagawa; citation_author=H. Kawakami; citation_author=M. Marti; citation_author=M. Kmita; citation_author=P. Monaghan-Nichols; citation_author=R. Nishinakamura; citation_author=J.C. Izpisua Belmonte; citation_journal_title=Development; citation_volume=136; citation_firstpage=585; citation_lastpage=594; citation_publication_date=2009; "><meta name="citation_reference" content="citation_title=Integrating functional data to prioritize causal variants in statistical fine-mapping studies; citation_author=G. Kichaev; citation_author=W.Y. Yang; citation_author=S. Lindstrom; citation_author=F. Hormozdiari; citation_author=E. Eskin; citation_author=A.L. Price; citation_author=P. Kraft; citation_author=B. Pasaniuc; citation_journal_title=PLoS Genet.; citation_volume=10; citation_firstpage=e1004722; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Cytoscape: software for visualization and analysis of biological networks; citation_inbook_title=Data Mining in Proteomics; citation_author=M. Kohl; citation_author=S. Wiese; citation_author=B. Warscheid; citation_firstpage=291; citation_lastpage=303; citation_publication_date=2011; citation_publisher=Springer; "><meta name="citation_reference" content="citation_title=Ectodysplasin A (EDA) - EDA receptor signalling and its pharmacological modulation; citation_author=C. Kowalczyk-Quintas; citation_author=P. Schneider; citation_journal_title=Cytokine Growth Factor Rev.; citation_volume=25; citation_firstpage=195; citation_lastpage=203; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Models for fingerprint pattern formation; citation_author=M. Kücken; citation_journal_title=Forensic Sci. Int.; citation_volume=171; citation_firstpage=85; citation_lastpage=96; citation_publication_date=2007; "><meta name="citation_reference" content="citation_title=Analysis of the regulation of lin-41 during chick and mouse limb development; citation_author=J.J. Lancman; citation_author=N.C. Caruccio; citation_author=B.D. Harfe; citation_author=A.E. Pasquinelli; citation_author=J.J. Schageman; citation_author=A. Pertsemlidis; citation_author=J.F. Fallon; citation_journal_title=Dev. Dyn.; citation_volume=234; citation_firstpage=948; citation_lastpage=960; citation_publication_date=2005; "><meta name="citation_reference" content="citation_title=Estimation of pleiotropy between complex diseases using single-nucleotide polymorphism-derived genomic relationships and restricted maximum likelihood; citation_author=S.H. Lee; citation_author=J. Yang; citation_author=M.E. Goddard; citation_author=P.M. Visscher; citation_author=N.R. Wray; citation_journal_title=Bioinformatics; citation_volume=28; citation_firstpage=2540; citation_lastpage=2542; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=Conserved expression domains for genes upstream and within the HoxA and HoxD clusters suggests a long-range enhancer existed before cluster duplication; citation_author=J.A. Lehoczky; citation_author=M.E. Williams; citation_author=J.W. Innis; citation_journal_title=Evol. Dev.; citation_volume=6; citation_firstpage=423; citation_lastpage=430; citation_publication_date=2004; "><meta name="citation_reference" content="citation_title=Spatiotemporal regulation of GLI target genes in the mammalian limb bud; citation_author=J.P. Lewandowski; citation_author=F. Du; citation_author=S. Zhang; citation_author=M.B. Powell; citation_author=K.N. Falkenstein; citation_author=H. Ji; citation_author=S.A. Vokes; citation_journal_title=Dev. Biol.; citation_volume=406; citation_firstpage=92; citation_lastpage=103; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Adjusting multiple testing in multilocus analyses using the eigenvalues of a correlation matrix; citation_author=J. Li; citation_author=L. Ji; citation_journal_title=Heredity; citation_volume=95; citation_firstpage=221; citation_lastpage=227; citation_publication_date=2005; "><meta name="citation_reference" content="citation_title=Formation of proximal and anterior limb skeleton requires early function of Irx3 and Irx5 and is negatively regulated by Shh signaling; citation_author=D. Li; citation_author=R. Sakuma; citation_author=N.A. Vakili; citation_author=R. Mo; citation_author=V. Puviindran; citation_author=S. Deimling; citation_author=X. Zhang; citation_author=S. Hopyan; citation_author=C.C. Hui; citation_journal_title=Dev. Cell; citation_volume=29; citation_firstpage=233; citation_lastpage=240; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=MaCH: using sequence and genotype data to estimate haplotypes and unobserved genotypes; citation_author=Y. Li; citation_author=C.J. Willer; citation_author=J. Ding; citation_author=P. Scheet; citation_author=G.R. Abecasis; citation_journal_title=Genet Epidemiol; citation_volume=34; citation_firstpage=816; citation_lastpage=834; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=Sonic hedgehog signaling directly targets Hyaluronic Acid Synthase 2, an essential regulator of phalangeal joint patterning; citation_author=J. Liu; citation_author=Q. Li; citation_author=M.R. Kuehn; citation_author=Y. Litingtung; citation_author=S.A. Vokes; citation_author=C. Chiang; citation_journal_title=Dev. Biol.; citation_volume=375; citation_firstpage=160; citation_lastpage=171; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=Quantitative Dermatoglyphics. Classification, Genetics, and Pathology; citation_author=D.Z. Loesch; citation_publication_date=1983; citation_publisher=Oxford University Press; "><meta name="citation_reference" content="citation_title=Finger ridge patterns and tactile sensitivity; citation_author=D.Z. Loesch; citation_author=N.G. Martin; citation_journal_title=Ann. Hum. Biol.; citation_volume=11; citation_firstpage=113; citation_lastpage=124; citation_publication_date=1984; "><meta name="citation_reference" content="citation_title=Efficient Bayesian mixed-model analysis increases association power in large cohorts; citation_author=P.R. Loh; citation_author=G. Tucker; citation_author=B.K. Bulik-Sullivan; citation_author=B.J. Vilhjálmsson; citation_author=H.K. Finucane; citation_author=R.M. Salem; citation_author=D.I. Chasman; citation_author=P.M. Ridker; citation_author=B.M. Neale; citation_author=B. Berger; citation_author=et al.; citation_journal_title=. Nat Genet; citation_volume=47; citation_firstpage=284; citation_lastpage=290; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Fine-Tuning of Shh/Gli Signaling Gradient by Non-proteolytic Ubiquitination during Neural Patterning; citation_author=P. Ma; citation_author=N.N. Song; citation_author=Y. Li; citation_author=Q. Zhang; citation_author=L. Zhang; citation_author=L. Zhang; citation_author=Q. Kong; citation_author=L. Ma; citation_author=X. Yang; citation_author=B. Ren; citation_author=et al.; citation_journal_title=Cell Rep.; citation_volume=28; citation_firstpage=541; citation_lastpage=553.e4; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Digital dermatoglyphic heritability differences as evidenced by a female twin study; citation_author=J.F. Machado; citation_author=P.R. Fernandes; citation_author=R.W. Roquetti; citation_author=J.F. Filho; citation_journal_title=Twin Res. Hum. Genet.; citation_volume=13; citation_firstpage=482; citation_lastpage=489; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=Mox2 is a component of the genetic hierarchy controlling limb muscle development; citation_author=B.S. Mankoo; citation_author=N.S. Collins; citation_author=P. Ashby; citation_author=E. Grigorieva; citation_author=L.H. Pevny; citation_author=A. Candia; citation_author=C.V. Wright; citation_author=P.W. Rigby; citation_author=V. Pachnis; citation_journal_title=Nature; citation_volume=400; citation_firstpage=69; citation_lastpage=73; citation_publication_date=1999; "><meta name="citation_reference" content="citation_title=Genetic evidence that FGFs have an instructive role in limb proximal-distal patterning; citation_author=F.V. Mariani; citation_author=C.P. Ahn; citation_author=G.R. Martin; citation_journal_title=Nature; citation_volume=453; citation_firstpage=401; citation_lastpage=405; citation_publication_date=2008; "><meta name="citation_reference" content="citation_title=A genetical analysis of covariation between finger ridge counts; citation_author=N.G. Martin; citation_author=L.J. Eaves; citation_author=D.Z. Loesch; citation_journal_title=Ann. Hum. Biol.; citation_volume=9; citation_firstpage=539; citation_lastpage=552; citation_publication_date=1982; "><meta name="citation_reference" content="citation_title=Expression of the receptor tyrosine kinase genes, Ror1 and Ror2, during mouse development; citation_author=T. Matsuda; citation_author=M. Nomi; citation_author=M. Ikeya; citation_author=S. Kani; citation_author=I. Oishi; citation_author=T. Terashima; citation_author=S. Takada; citation_author=Y. Minami; citation_journal_title=Mech. Dev.; citation_volume=105; citation_firstpage=153; citation_lastpage=156; citation_publication_date=2001; "><meta name="citation_reference" content="citation_title=ADAMTS metalloproteases generate active versican fragments that regulate interdigital web regression; citation_author=D.R. McCulloch; citation_author=C.M. Nelson; citation_author=L.J. Dixon; citation_author=D.L. Silver; citation_author=J.D. Wylie; citation_author=V. Lindner; citation_author=T. Sasaki; citation_author=M.A. Cooley; citation_author=W.S. Argraves; citation_author=S.S. Apte; citation_journal_title=Dev. Cell; citation_volume=17; citation_firstpage=687; citation_lastpage=698; citation_publication_date=2009; "><meta name="citation_reference" content="citation_title=GREAT improves functional interpretation of cis-regulatory regions; citation_author=C.Y. McLean; citation_author=D. Bristor; citation_author=M. Hiller; citation_author=S.L. Clarke; citation_author=B.T. Schaar; citation_author=C.B. Lowe; citation_author=A.M. Wenger; citation_author=G. Bejerano; citation_journal_title=Nat. Biotechnol.; citation_volume=28; citation_firstpage=495; citation_lastpage=501; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=Linkage analysis of a model quantitative trait in humans: finger ridge count shows significant multivariate linkage to 5q14.1; citation_author=S.E. Medland; citation_author=D.Z. Loesch; citation_author=B. Mdzewski; citation_author=G. Zhu; citation_author=G.W. Montgomery; citation_author=N.G. Martin; citation_journal_title=PLoS Genet.; citation_volume=3; citation_firstpage=1736; citation_lastpage=1744; citation_publication_date=2007; "><meta name="citation_reference" content="citation_title=A variant in LIN28B is associated with 2D:4D finger-length ratio, a putative retrospective biomarker of prenatal testosterone exposure; citation_author=S.E. Medland; citation_author=T. Zayats; citation_author=B. Glaser; citation_author=D.R. Nyholt; citation_author=S.D. Gordon; citation_author=M.J. Wright; citation_author=G.W. Montgomery; citation_author=M.J. Campbell; citation_author=A.K. Henders; citation_author=N.J. Timpson; citation_author=et al.; citation_journal_title=Am. J. Hum. Genet.; citation_volume=86; citation_firstpage=519; citation_lastpage=525; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=Dermatoglyphic defects in children with leukaemia; citation_author=M.A. Menser; citation_author=S.G. Purvis-Smith; citation_journal_title=Lancet; citation_volume=1; citation_firstpage=1076; citation_lastpage=1078; citation_publication_date=1969; "><meta name="citation_reference" content="citation_title=Purified Wnt5a protein activates or inhibits beta-catenin-TCF signaling depending on receptor context; citation_author=A.J. Mikels; citation_author=R. Nusse; citation_journal_title=PLoS Biol.; citation_volume=4; citation_firstpage=e115; citation_publication_date=2006; "><meta name="citation_reference" content="citation_title=Mutations in the human homologue of mouse dl cause autosomal recessive and dominant hypohidrotic ectodermal dysplasia; citation_author=A.W. Monreal; citation_author=B.M. Ferguson; citation_author=D.J. Headon; citation_author=S.L. Street; citation_author=P.A. Overbeek; citation_author=J. Zonana; citation_journal_title=Nat. Genet.; citation_volume=22; citation_firstpage=366; citation_lastpage=369; citation_publication_date=1999; "><meta name="citation_reference" content="citation_title=Generation of the primary hair follicle pattern; citation_author=C. Mou; citation_author=B. Jackson; citation_author=P. Schneider; citation_author=P.A. Overbeek; citation_author=D.J. Headon; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=103; citation_firstpage=9075; citation_lastpage=9080; citation_publication_date=2006; "><meta name="citation_reference" content="citation_title=The genesis of dermatoglyphics; citation_author=J.J. Mulvihill; citation_author=D.W. Smith; citation_journal_title=J.&nbsp;Pediatr.; citation_volume=75; citation_firstpage=579; citation_lastpage=589; citation_publication_date=1969; "><meta name="citation_reference" content="citation_title=Pattern influence on the fingers; citation_author=A.S. Nagy; citation_author=M. Pap; citation_journal_title=Homo; citation_volume=56; citation_firstpage=51; citation_lastpage=67; citation_publication_date=2005; "><meta name="citation_reference" content="citation_title=Secreted metalloproteases ADAMTS9 and ADAMTS20 have a non-canonical role in ciliary vesicle growth during ciliogenesis; citation_author=S. Nandadasa; citation_author=C.M. Kraft; citation_author=L.W. Wang; citation_author=A. O’Donnell; citation_author=R. Patel; citation_author=H.Y. Gee; citation_journal_title=Nat. Commun.; citation_volume=10; citation_firstpage=953; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=A chicken homeobox gene related to Drosophila paired is predominantly expressed in the developing limb; citation_author=T. Nohno; citation_author=E. Koyama; citation_author=F. Myokai; citation_author=S. Taniguchi; citation_author=H. Ohuchi; citation_author=T. Saito; citation_author=S. Noji; citation_journal_title=Dev. Biol.; citation_volume=158; citation_firstpage=254; citation_lastpage=264; citation_publication_date=1993; "><meta name="citation_reference" content="citation_title=Development of dermal ridges in the fetus; citation_author=M. Okajima; citation_journal_title=J.&nbsp;Med. Genet.; citation_volume=12; citation_firstpage=243; citation_lastpage=250; citation_publication_date=1975; "><meta name="citation_reference" content="citation_title=Activities of N-Myc in the developing limb link control of skeletal size with digit separation; citation_author=S. Ota; citation_author=Z.Q. Zhou; citation_author=D.R. Keene; citation_author=P. Knoepfler; citation_author=P.J. Hurlin; citation_journal_title=Development; citation_volume=134; citation_firstpage=1583; citation_lastpage=1592; citation_publication_date=2007; "><meta name="citation_reference" content="citation_title=Towards an integrated experimental-theoretical approach for assessing the mechanistic basis of hair and feather morphogenesis; citation_author=K.J. Painter; citation_author=G.S. Hunt; citation_author=K.L. Wells; citation_author=J.A. Johansson; citation_author=D.J. Headon; citation_journal_title=Interface Focus; citation_volume=2; citation_firstpage=433; citation_lastpage=450; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=Mutation at the Evi1 locus in Junbo mice causes susceptibility to otitis media; citation_author=N. Parkinson; citation_author=R.E. Hardisty-Hughes; citation_author=H. Tateossian; citation_author=H.-T. Tsai; citation_author=D. Brooker; citation_author=S. Morse; citation_author=Z. Lalane; citation_author=F. MacKenzie; citation_author=M. Fray; citation_author=P. Glenister; citation_author=et al.; citation_journal_title=PLoS Genet.; citation_volume=2; citation_firstpage=e149; citation_publication_date=2006; "><meta name="citation_reference" content="citation_title=Dermatoglyphic Topology; citation_author=L.S. Penrose; citation_journal_title=Nature; citation_volume=205; citation_firstpage=544; citation_lastpage=546; citation_publication_date=1965; "><meta name="citation_reference" content="citation_title=On the geometry of loops and deltas; citation_author=L.S. Penrose; citation_journal_title=Ann. Hum. Genet.; citation_volume=29; citation_firstpage=207; citation_lastpage=211; citation_publication_date=1965; "><meta name="citation_reference" content="citation_title=The development of the epidermal ridges; citation_author=L.S. Penrose; citation_author=P.T. Ohara; citation_journal_title=J.&nbsp;Med. Genet.; citation_volume=10; citation_firstpage=201; citation_lastpage=208; citation_publication_date=1973; "><meta name="citation_reference" content="citation_title=A sonic hedgehog-independent, retinoid-activated pathway of neurogenesis in the ventral spinal cord; citation_author=A. Pierani; citation_author=S. Brenner-Morton; citation_author=C. Chiang; citation_author=T.M. Jessell; citation_journal_title=Cell; citation_volume=97; citation_firstpage=903; citation_lastpage=915; citation_publication_date=1999; "><meta name="citation_reference" content="citation_title=Principal components analysis corrects for stratification in genome-wide association studies; citation_author=A.L. Price; citation_author=N.J. Patterson; citation_author=R.M. Plenge; citation_author=M.E. Weinblatt; citation_author=N.A. Shadick; citation_author=D. Reich; citation_journal_title=Nat. Genet.; citation_volume=38; citation_firstpage=904; citation_lastpage=909; citation_publication_date=2006; "><meta name="citation_reference" content="citation_title=LocusZoom: regional visualization of genome-wide association scan results; citation_author=R.J. Pruim; citation_author=R.P. Welch; citation_author=S. Sanna; citation_author=T.M. Teslovich; citation_author=P.S. Chines; citation_author=T.P. Gliedt; citation_author=M. Boehnke; citation_author=G.R. Abecasis; citation_author=C.J. Willer; citation_journal_title=Bioinformatics; citation_volume=26; citation_firstpage=2336; citation_lastpage=2337; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=RefSeq: an update on mammalian reference sequences; citation_author=K.D. Pruitt; citation_author=G.R. Brown; citation_author=S.M. Hiatt; citation_author=F. Thibaud-Nissen; citation_author=A. Astashyn; citation_author=O. Ermolaeva; citation_author=C.M. Farrell; citation_author=et al.; citation_journal_title=Nucleic Acids Res; citation_volume=42; citation_firstpage=D756; citation_lastpage=D763; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=PLINK: a tool set for whole-genome association and population-based linkage analyses; citation_author=S. Purcell; citation_author=B. Neale; citation_author=K. Todd-Brown; citation_author=L. Thomas; citation_author=M.A. Ferreira; citation_author=D. Bender; citation_author=J. Maller; citation_author=P. Sklar; citation_author=P.I. de Bakker; citation_author=M.J. Daly; citation_author=P.C. Sham; citation_journal_title=Am. J. Hum. Genet.; citation_volume=81; citation_firstpage=559; citation_lastpage=575; citation_publication_date=2007; "><meta name="citation_reference" content="citation_title=A new biometric: In utero growth curves for metacarpal and phalangeal lengths reveal an embryonic patterning ratio; citation_author=R. Rao; citation_author=J. Gornbein; citation_author=Y. Afshar; citation_author=L.D. Platt; citation_author=G.R. DeVore; citation_author=D. Krakow; citation_journal_title=Prenat. Diagn.; citation_volume=39; citation_firstpage=200; citation_lastpage=208; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Dermatoglyphic Patterns of Acute Leukemia Patients; citation_author=R. Rathee; citation_author=N. Kamal; citation_author=A. Kumar; citation_author=M. Vashist; citation_author=R. Yadav; citation_journal_title=Int. Res. J. Biol. Sci.; citation_volume=3; citation_firstpage=90; citation_lastpage=93; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Genetic analysis of dermatoglyphic patterns in twins; citation_author=T. Reed; citation_author=F.R. Sprague; citation_author=K.W. Kang; citation_author=W.E. Nance; citation_author=J.C. Christian; citation_journal_title=Hum. Hered.; citation_volume=25; citation_firstpage=263; citation_lastpage=275; citation_publication_date=1975; "><meta name="citation_reference" content="citation_title=Blimp1 regulates development of the posterior forelimb, caudal pharyngeal arches, heart and sensory vibrissae in mice; citation_author=E.J. Robertson; citation_author=I. Charatsi; citation_author=C.J. Joyner; citation_author=C.H. Koonce; citation_author=M. Morgan; citation_author=A. Islam; citation_author=C. Paterson; citation_author=E. Lejsek; citation_author=S.J. Arnold; citation_author=A. Kallies; citation_author=et al.; citation_journal_title=Development; citation_volume=134; citation_firstpage=4335; citation_lastpage=4345; citation_publication_date=2007; "><meta name="citation_reference" content="citation_title=Dermatoglyphics of leukaemic children; citation_author=F. Rosner; citation_journal_title=Lancet; citation_volume=2; citation_firstpage=272; citation_lastpage=273; citation_publication_date=1969; "><meta name="citation_reference" content="citation_title=Detailed expression profile of the six Glypicans and their modifying enzyme, Notum during chick limb and feather development; citation_author=K. Saad; citation_author=S. Theis; citation_author=A. Otto; citation_author=G. Luke; citation_author=K. Patel; citation_journal_title=Gene; citation_volume=610; citation_firstpage=71; citation_lastpage=79; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=The role of fingerprints in the coding of tactile information probed with a biomimetic sensor; citation_author=J. Scheibert; citation_author=S. Leurent; citation_author=A. Prevost; citation_author=G. Debrégeas; citation_journal_title=Science; citation_volume=323; citation_firstpage=1503; citation_lastpage=1506; citation_publication_date=2009; "><meta name="citation_reference" content="citation_title=Mode of inheritance of finger dermatoglyphic traits among Vaidyas of West Bengal, India; citation_author=M. Sengupta; citation_author=B. Karmakar; citation_journal_title=Ann. Hum. Biol.; citation_volume=31; citation_firstpage=526; citation_lastpage=540; citation_publication_date=2004; "><meta name="citation_reference" content="citation_title=A cross-species analysis of Satb2 expression suggests deep conservation across vertebrate lineages; citation_author=K. Sheehan-Rooney; citation_author=B. Pálinkášová; citation_author=J.K. Eberhart; citation_author=M.J. Dixon; citation_journal_title=Dev. Dyn.; citation_volume=239; citation_firstpage=3481; citation_lastpage=3491; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=dbSNP: the NCBI database of genetic variation; citation_author=S.T. Sherry; citation_author=M.-H. Ward; citation_author=M. Kholodov; citation_author=J. Baker; citation_author=L. Phan; citation_author=E.M. Smigielski; citation_author=K. Sirotkin; citation_journal_title=Nucleic Acids Res.; citation_volume=29; citation_firstpage=308; citation_lastpage=311; citation_publication_date=2001; "><meta name="citation_reference" content="citation_title=Differential expression of the seven-pass transmembrane cadherin genes Celsr1-3 and distribution of the Celsr2 protein during mouse development; citation_author=Y. Shima; citation_author=N.G. Copeland; citation_author=D.J. Gilbert; citation_author=N.A. Jenkins; citation_author=O. Chisaka; citation_author=M. Takeichi; citation_author=T. Uemura; citation_journal_title=Dev. Dyn.; citation_volume=223; citation_firstpage=321; citation_lastpage=332; citation_publication_date=2002; "><meta name="citation_reference" content="citation_title=Exome sequencing and CRISPR/Cas genome editing identify mutations of ZAK as a cause of limb defects in humans and mice; citation_author=M. Spielmann; citation_author=N. Kakar; citation_author=N. Tayebi; citation_author=C. Leettola; citation_author=G. Nürnberg; citation_author=N. Sowada; citation_author=D.G. Lupiáñez; citation_author=I. Harabula; citation_author=R. Flöttmann; citation_author=D. Horn; citation_author=et al.; citation_journal_title=Genome Res.; citation_volume=26; citation_firstpage=183; citation_lastpage=191; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=The Tabby phenotype is caused by mutation in a mouse homologue of the EDA gene that reveals novel mouse and human exons and encodes a protein (ectodysplasin-A) with collagenous domains; citation_author=A.K. Srivastava; citation_author=J. Pispa; citation_author=A.J. Hartung; citation_author=Y. Du; citation_author=S. Ezer; citation_author=T. Jenks; citation_author=T. Shimada; citation_author=M. Pekkanen; citation_author=M.L. Mikkola; citation_author=M.S. Ko; citation_author=et al.; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=94; citation_firstpage=13069; citation_lastpage=13074; citation_publication_date=1997; "><meta name="citation_reference" content="citation_title=RSPO2 inhibition of RNF43 and ZNRF3 governs limb development independently of LGR4/5/6; citation_author=E. Szenker-Ravi; citation_author=U. Altunoglu; citation_author=M. Leushacke; citation_author=C. Bosso-Lefèvre; citation_author=M. Khatoo; citation_author=H. Thi Tran; citation_author=T. Naert; citation_author=R. Noelanders; citation_author=A. Hajamohideen; citation_author=C. Beneteau; citation_author=et al.; citation_journal_title=Nature; citation_volume=557; citation_firstpage=564; citation_lastpage=569; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=STRING v11: protein-protein association networks with increased coverage, supporting functional discovery in genome-wide experimental datasets; citation_author=D. Szklarczyk; citation_author=A.L. Gable; citation_author=D. Lyon; citation_author=A. Junge; citation_author=S. Wyder; citation_author=J. Huerta-Cepas; citation_author=M. Simonovic; citation_author=N.T. Doncheva; citation_author=J.H. Morris; citation_author=P. Bork; citation_author=et al.; citation_journal_title=Nucleic Acids Res.; citation_volume=47; citation_number=D1; citation_firstpage=D607; citation_lastpage=D613; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=PLS path modeling; citation_author=M. Tenenhaus; citation_author=V.E. Vinzi; citation_author=Y.M. Chatelin; citation_author=C. Lauro; citation_journal_title=Comput Stat Data Anal; citation_volume=48; citation_firstpage=159; citation_lastpage=205; citation_publication_date=2005; "><meta name="citation_reference" content="citation_title=Growing models of vertebrate limb development; citation_author=M. Towers; citation_author=C. Tickle; citation_journal_title=Development; citation_volume=136; citation_firstpage=179; citation_lastpage=190; citation_publication_date=2009; "><meta name="citation_reference" content="citation_title=Dermatoglyphics on volar skin of mice: the normal pattern; citation_author=M. Tsugane; citation_author=M. Yasuda; citation_journal_title=Anat. Rec.; citation_volume=242; citation_firstpage=225; citation_lastpage=232; citation_publication_date=1995; "><meta name="citation_reference" content="citation_title=qqman: an R package for visualizing GWAS results using QQ and manhattan plots; citation_author=S.D. Turner; citation_journal_title=bioRxiv; citation_doi=10.1101/005165; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=mRNA expression patterns of the IGF system during mouse limb bud development, determined by whole mount in&nbsp;situ hybridization; citation_author=M. van Kleffens; citation_author=C. Groffen; citation_author=R.R. Rosato; citation_author=S.M. van den Eijnde; citation_author=J.W. van Neck; citation_author=D.J. Lindenbergh-Kortleve; citation_author=E.C. Zwarthoff; citation_author=S.L. Drop; citation_journal_title=Mol. Cell. Endocrinol.; citation_volume=138; citation_firstpage=151; citation_lastpage=161; citation_publication_date=1998; "><meta name="citation_reference" content="citation_title=Hypohidrotic (or anhidrotic) ectodermal dysplasia--an appraisal of diagnostic methods; citation_author=J. Verbov; citation_journal_title=Br. J. Dermatol.; citation_volume=83; citation_firstpage=341; citation_lastpage=348; citation_publication_date=1970; "><meta name="citation_reference" content="citation_title=Rationales, design and recruitment of the Taizhou Longitudinal Study; citation_author=X. Wang; citation_author=M. Lu; citation_author=J. Qian; citation_author=Y. Yang; citation_author=S. Li; citation_author=D. Lu; citation_author=S. Yu; citation_author=W. Meng; citation_author=W. Ye; citation_author=L. Jin; citation_journal_title=BMC Public Health; citation_volume=9; citation_firstpage=223; citation_publication_date=2009; "><meta name="citation_reference" content="citation_title=The 3D Genome Browser: a web-based browser for visualizing 3D genome organization and long-range chromatin interactions; citation_author=Y. Wang; citation_author=F. Song; citation_author=B. Zhang; citation_author=L. Zhang; citation_author=J. Xu; citation_author=D. Kuang; citation_author=D. Li; citation_author=M.N.K. Choudhary; citation_author=Y. Li; citation_author=M. Hu; citation_author=et al.; citation_journal_title=Genome Biol.; citation_volume=19; citation_firstpage=151; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=HaploReg: a resource for exploring chromatin states, conservation, and regulatory motif alterations within sets of genetically linked variants; citation_author=L.D. Ward; citation_author=M. Kellis; citation_journal_title=Nucleic Acids Res.; citation_volume=40; citation_firstpage=D930; citation_lastpage=D934; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=The Pittsburgh Oral-Facial Cleft study: expanding the cleft phenotype. Background and justification; citation_author=S.M. Weinberg; citation_author=K. Neiswanger; citation_author=R.A. Martin; citation_author=M.P. Mooney; citation_author=A.A. Kane; citation_author=S.L. Wenger; citation_author=J. Losee; citation_author=F. Deleyiannis; citation_author=L. Ma; citation_author=J.E. De Salamanca; citation_author=et al.; citation_journal_title=Cleft Palate Craniofac. J.; citation_volume=43; citation_firstpage=7; citation_lastpage=20; citation_publication_date=2006; "><meta name="citation_reference" content="citation_title=The critical stage of friction ridge and pattern formation; citation_author=K. Wertheim; citation_author=A. Maceo; citation_journal_title=J.&nbsp;Forensic Identif.; citation_volume=52; citation_firstpage=35; citation_publication_date=2002; "><meta name="citation_reference" content="citation_title=METAL: fast and efficient meta-analysis of genomewide association scans; citation_author=C.J. Willer; citation_author=Y. Li; citation_author=G.R. Abecasis; citation_journal_title=Bioinformatics; citation_volume=26; citation_firstpage=2190; citation_lastpage=2191; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=A Wnt5a pathway underlies outgrowth of multiple structures in the vertebrate embryo; citation_author=T.P. Yamaguchi; citation_author=A. Bradley; citation_author=A.P. McMahon; citation_author=S. Jones; citation_journal_title=Development; citation_volume=126; citation_firstpage=1211; citation_lastpage=1223; citation_publication_date=1999; "><meta name="citation_reference" content="citation_title=Hoxa13 regulates expression of common Hox target genes involved in cartilage development to coordinate the expansion of the autopodal anlage; citation_author=S. Yamamoto; citation_author=Y. Uchida; citation_author=T. Ohtani; citation_author=E. Nozaki; citation_author=C. Yin; citation_author=Y. Gotoh; citation_author=N. Yakushiji-Kaminatsui; citation_author=T. Higashiyama; citation_author=T. Suzuki; citation_author=T. Takemoto; citation_author=et al.; citation_journal_title=Dev. Growth Differ.; citation_volume=61; citation_firstpage=228; citation_lastpage=251; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=GCTA: a tool for genome-wide complex trait analysis; citation_author=J. Yang; citation_author=S.H. Lee; citation_author=M.E. Goddard; citation_author=P.M. Visscher; citation_journal_title=Am. J. Hum. Genet.; citation_volume=88; citation_firstpage=76; citation_lastpage=82; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=Fingerprint ridges allow primates to regulate grip; citation_author=S.M. Yum; citation_author=I.K. Baek; citation_author=D. Hong; citation_author=J. Kim; citation_author=K. Jung; citation_author=S. Kim; citation_author=K. Eom; citation_author=J. Jang; citation_author=S. Kim; citation_author=M. Sattorov; citation_author=et al.; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=117; citation_firstpage=31665; citation_lastpage=31673; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=PCP effector gene Inturned is an important regulator of cilia formation and embryonic development in mammals; citation_author=H. Zeng; citation_author=A.N. Hoover; citation_author=A. Liu; citation_journal_title=Dev. Biol.; citation_volume=339; citation_firstpage=418; citation_lastpage=428; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=Dermatoglyphics from all Chinese ethnic groups reveal geographic patterning; citation_author=H.-G. Zhang; citation_author=Y.-F. Chen; citation_author=M. Ding; citation_author=L. Jin; citation_author=D.T. Case; citation_author=Y.-P. Jiao; citation_author=X.-P. Wang; citation_author=C.-X. Bai; citation_author=G. Jin; citation_author=J.-M. Yang; citation_author=et al.; citation_journal_title=PLoS ONE; citation_volume=5; citation_firstpage=e8783; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=MDM2 negatively regulates the human telomerase RNA gene promoter; citation_author=J. Zhao; citation_author=A. Bilsland; citation_author=K. Jackson; citation_author=W.N. Keith; citation_journal_title=BMC Cancer; citation_volume=5; citation_firstpage=6; citation_publication_date=2005; "><meta name="citation_abstract" content="<h2>Summary</h2><p>Fingerprints are of long-standing practical and cultural interest, but little is known about the mechanisms that underlie their variation. Using genome-wide scans in Han Chinese cohorts, we identified 18 loci associated with fingerprint type across the digits, including a genetic basis for the long-recognized &quot;pattern-block&quot; correlations among the middle three digits. In particular, we identified a variant near <i>EVI1</i> that alters regulatory activity and established a role for EVI1 in dermatoglyph patterning in mice. Dynamic EVI1 expression during human development supports its role in shaping the limbs and digits, rather than influencing skin patterning directly. Trans-ethnic meta-analysis identified 43 fingerprint-associated loci, with nearby genes being strongly enriched for general limb development pathways. We also found that fingerprint patterns were genetically correlated with hand proportions. Taken together, these findings support the key role of limb development genes in influencing the outcome of fingerprint patterning.</p>"><meta charset="UTF-8"><!--

    Pagebuild admin UI
-->



<script type="text/javascript" src="https://bam.nr-data.net/1/16e2192192?a=916957862&amp;sa=1&amp;v=1177.96a4d39&amp;t=Unnamed%20Transaction&amp;rst=9472&amp;ck=1&amp;ref=https://www.cell.com/cell/fulltext/S0092-8674(21)01446-X&amp;be=3568&amp;fe=9289&amp;dc=6885&amp;af=err,xhr,stn,ins,spa&amp;perf=%7B%22timing%22:%7B%22of%22:1749592280528,%22n%22:0,%22f%22:2938,%22dn%22:2939,%22dne%22:2939,%22c%22:2939,%22s%22:2939,%22ce%22:2939,%22rq%22:2939,%22rp%22:3552,%22rpe%22:4346,%22dl%22:3557,%22di%22:4882,%22ds%22:6885,%22de%22:7175,%22dc%22:9288,%22l%22:9289,%22le%22:9450%7D,%22navigation%22:%7B%7D%7D&amp;jsonp=NREUM.setToken"></script><script src="https://js-agent.newrelic.com/nr-spa-1177.min.js"></script><script type="text/javascript">
    ;window.NREUM||(NREUM={});NREUM.init={privacy:{cookies_enabled:true}};
    window.NREUM||(NREUM={}),__nr_require=function(t,e,n){function r(n){if(!e[n]){var o=e[n]={exports:{}};t[n][0].call(o.exports,function(e){var o=t[n][1][e];return r(o||e)},o,o.exports)}return e[n].exports}if("function"==typeof __nr_require)return __nr_require;for(var o=0;o<n.length;o++)r(n[o]);return r}({1:[function(t,e,n){function r(t){try{c.console&&console.log(t)}catch(e){}}var o,i=t("ee"),a=t(26),c={};try{o=localStorage.getItem("__nr_flags").split(","),console&&"function"==typeof console.log&&(c.console=!0,o.indexOf("dev")!==-1&&(c.dev=!0),o.indexOf("nr_dev")!==-1&&(c.nrDev=!0))}catch(s){}c.nrDev&&i.on("internal-error",function(t){r(t.stack)}),c.dev&&i.on("fn-err",function(t,e,n){r(n.stack)}),c.dev&&(r("NR AGENT IN DEVELOPMENT MODE"),r("flags: "+a(c,function(t,e){return t}).join(", ")))},{}],2:[function(t,e,n){function r(t,e,n,r,c){try{l?l-=1:o(c||new UncaughtException(t,e,n),!0)}catch(f){try{i("ierr",[f,s.now(),!0])}catch(d){}}return"function"==typeof u&&u.apply(this,a(arguments))}function UncaughtException(t,e,n){this.message=t||"Uncaught error with no additional information",this.sourceURL=e,this.line=n}function o(t,e){var n=e?null:s.now();i("err",[t,n])}var i=t("handle"),a=t(27),c=t("ee"),s=t("loader"),f=t("gos"),u=window.onerror,d=!1,p="nr@seenError",l=0;s.features.err=!0,t(1),window.onerror=r;try{throw new Error}catch(h){"stack"in h&&(t(13),t(12),"addEventListener"in window&&t(6),s.xhrWrappable&&t(14),d=!0)}c.on("fn-start",function(t,e,n){d&&(l+=1)}),c.on("fn-err",function(t,e,n){d&&!n[p]&&(f(n,p,function(){return!0}),this.thrown=!0,o(n))}),c.on("fn-end",function(){d&&!this.thrown&&l>0&&(l-=1)}),c.on("internal-error",function(t){i("ierr",[t,s.now(),!0])})},{}],3:[function(t,e,n){t("loader").features.ins=!0},{}],4:[function(t,e,n){function r(){_++,T=g.hash,this[u]=y.now()}function o(){_--,g.hash!==T&&i(0,!0);var t=y.now();this[h]=~~this[h]+t-this[u],this[d]=t}function i(t,e){E.emit("newURL",[""+g,e])}function a(t,e){t.on(e,function(){this[e]=y.now()})}var c="-start",s="-end",f="-body",u="fn"+c,d="fn"+s,p="cb"+c,l="cb"+s,h="jsTime",m="fetch",v="addEventListener",w=window,g=w.location,y=t("loader");if(w[v]&&y.xhrWrappable){var x=t(10),b=t(11),E=t(8),R=t(6),O=t(13),N=t(7),M=t(14),P=t(9),S=t("ee"),C=S.get("tracer");t(16),y.features.spa=!0;var T,_=0;S.on(u,r),S.on(p,r),S.on(d,o),S.on(l,o),S.buffer([u,d,"xhr-done","xhr-resolved"]),R.buffer([u]),O.buffer(["setTimeout"+s,"clearTimeout"+c,u]),M.buffer([u,"new-xhr","send-xhr"+c]),N.buffer([m+c,m+"-done",m+f+c,m+f+s]),E.buffer(["newURL"]),x.buffer([u]),b.buffer(["propagate",p,l,"executor-err","resolve"+c]),C.buffer([u,"no-"+u]),P.buffer(["new-jsonp","cb-start","jsonp-error","jsonp-end"]),a(M,"send-xhr"+c),a(S,"xhr-resolved"),a(S,"xhr-done"),a(N,m+c),a(N,m+"-done"),a(P,"new-jsonp"),a(P,"jsonp-end"),a(P,"cb-start"),E.on("pushState-end",i),E.on("replaceState-end",i),w[v]("hashchange",i,!0),w[v]("load",i,!0),w[v]("popstate",function(){i(0,_>1)},!0)}},{}],5:[function(t,e,n){function r(t){}if(window.performance&&window.performance.timing&&window.performance.getEntriesByType){var o=t("ee"),i=t("handle"),a=t(13),c=t(12),s="learResourceTimings",f="addEventListener",u="resourcetimingbufferfull",d="bstResource",p="resource",l="-start",h="-end",m="fn"+l,v="fn"+h,w="bstTimer",g="pushState",y=t("loader");y.features.stn=!0,t(8),"addEventListener"in window&&t(6);var x=NREUM.o.EV;o.on(m,function(t,e){var n=t[0];n instanceof x&&(this.bstStart=y.now())}),o.on(v,function(t,e){var n=t[0];n instanceof x&&i("bst",[n,e,this.bstStart,y.now()])}),a.on(m,function(t,e,n){this.bstStart=y.now(),this.bstType=n}),a.on(v,function(t,e){i(w,[e,this.bstStart,y.now(),this.bstType])}),c.on(m,function(){this.bstStart=y.now()}),c.on(v,function(t,e){i(w,[e,this.bstStart,y.now(),"requestAnimationFrame"])}),o.on(g+l,function(t){this.time=y.now(),this.startPath=location.pathname+location.hash}),o.on(g+h,function(t){i("bstHist",[location.pathname+location.hash,this.startPath,this.time])}),f in window.performance&&(window.performance["c"+s]?window.performance[f](u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["c"+s]()},!1):window.performance[f]("webkit"+u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["webkitC"+s]()},!1)),document[f]("scroll",r,{passive:!0}),document[f]("keypress",r,!1),document[f]("click",r,!1)}},{}],6:[function(t,e,n){function r(t){for(var e=t;e&&!e.hasOwnProperty(u);)e=Object.getPrototypeOf(e);e&&o(e)}function o(t){c.inPlace(t,[u,d],"-",i)}function i(t,e){return t[1]}var a=t("ee").get("events"),c=t("wrap-function")(a,!0),s=t("gos"),f=XMLHttpRequest,u="addEventListener",d="removeEventListener";e.exports=a,"getPrototypeOf"in Object?(r(document),r(window),r(f.prototype)):f.prototype.hasOwnProperty(u)&&(o(window),o(f.prototype)),a.on(u+"-start",function(t,e){var n=t[1],r=s(n,"nr@wrapped",function(){function t(){if("function"==typeof n.handleEvent)return n.handleEvent.apply(n,arguments)}var e={object:t,"function":n}[typeof n];return e?c(e,"fn-",null,e.name||"anonymous"):n});this.wrapped=t[1]=r}),a.on(d+"-start",function(t){t[1]=this.wrapped||t[1]})},{}],7:[function(t,e,n){function r(t,e,n){var r=t[e];"function"==typeof r&&(t[e]=function(){var t=i(arguments),e={};o.emit(n+"before-start",[t],e);var a;e[m]&&e[m].dt&&(a=e[m].dt);var c=r.apply(this,t);return o.emit(n+"start",[t,a],c),c.then(function(t){return o.emit(n+"end",[null,t],c),t},function(t){throw o.emit(n+"end",[t],c),t})})}var o=t("ee").get("fetch"),i=t(27),a=t(26);e.exports=o;var c=window,s="fetch-",f=s+"body-",u=["arrayBuffer","blob","json","text","formData"],d=c.Request,p=c.Response,l=c.fetch,h="prototype",m="nr@context";d&&p&&l&&(a(u,function(t,e){r(d[h],e,f),r(p[h],e,f)}),r(c,"fetch",s),o.on(s+"end",function(t,e){var n=this;if(e){var r=e.headers.get("content-length");null!==r&&(n.rxSize=r),o.emit(s+"done",[null,e],n)}else o.emit(s+"done",[t],n)}))},{}],8:[function(t,e,n){var r=t("ee").get("history"),o=t("wrap-function")(r);e.exports=r;var i=window.history&&window.history.constructor&&window.history.constructor.prototype,a=window.history;i&&i.pushState&&i.replaceState&&(a=i),o.inPlace(a,["pushState","replaceState"],"-")},{}],9:[function(t,e,n){function r(t){function e(){s.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}function n(){s.emit("jsonp-error",[],p),s.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}var r=t&&"string"==typeof t.nodeName&&"script"===t.nodeName.toLowerCase();if(r){var o="function"==typeof t.addEventListener;if(o){var a=i(t.src);if(a){var u=c(a),d="function"==typeof u.parent[u.key];if(d){var p={};f.inPlace(u.parent,[u.key],"cb-",p),t.addEventListener("load",e,!1),t.addEventListener("error",n,!1),s.emit("new-jsonp",[t.src],p)}}}}}function o(){return"addEventListener"in window}function i(t){var e=t.match(u);return e?e[1]:null}function a(t,e){var n=t.match(p),r=n[1],o=n[3];return o?a(o,e[r]):e[r]}function c(t){var e=t.match(d);return e&&e.length>=3?{key:e[2],parent:a(e[1],window)}:{key:t,parent:window}}var s=t("ee").get("jsonp"),f=t("wrap-function")(s);if(e.exports=s,o()){var u=/[?&](?:callback|cb)=([^&#]+)/,d=/(.*)\.([^.]+)/,p=/^(\w+)(\.|$)(.*)$/,l=["appendChild","insertBefore","replaceChild"];Node&&Node.prototype&&Node.prototype.appendChild?f.inPlace(Node.prototype,l,"dom-"):(f.inPlace(HTMLElement.prototype,l,"dom-"),f.inPlace(HTMLHeadElement.prototype,l,"dom-"),f.inPlace(HTMLBodyElement.prototype,l,"dom-")),s.on("dom-start",function(t){r(t[0])})}},{}],10:[function(t,e,n){var r=t("ee").get("mutation"),o=t("wrap-function")(r),i=NREUM.o.MO;e.exports=r,i&&(window.MutationObserver=function(t){return this instanceof i?new i(o(t,"fn-")):i.apply(this,arguments)},MutationObserver.prototype=i.prototype)},{}],11:[function(t,e,n){function r(t){var e=a.context(),n=c(t,"executor-",e),r=new f(n);return a.context(r).getCtx=function(){return e},a.emit("new-promise",[r,e],e),r}function o(t,e){return e}var i=t("wrap-function"),a=t("ee").get("promise"),c=i(a),s=t(26),f=NREUM.o.PR;e.exports=a,f&&(window.Promise=r,["all","race"].forEach(function(t){var e=f[t];f[t]=function(n){function r(t){return function(){a.emit("propagate",[null,!o],i),o=o||!t}}var o=!1;s(n,function(e,n){Promise.resolve(n).then(r("all"===t),r(!1))});var i=e.apply(f,arguments),c=f.resolve(i);return c}}),["resolve","reject"].forEach(function(t){var e=f[t];f[t]=function(t){var n=e.apply(f,arguments);return t!==n&&a.emit("propagate",[t,!0],n),n}}),f.prototype["catch"]=function(t){return this.then(null,t)},f.prototype=Object.create(f.prototype,{constructor:{value:r}}),s(Object.getOwnPropertyNames(f),function(t,e){try{r[e]=f[e]}catch(n){}}),a.on("executor-start",function(t){t[0]=c(t[0],"resolve-",this),t[1]=c(t[1],"resolve-",this)}),a.on("executor-err",function(t,e,n){t[1](n)}),c.inPlace(f.prototype,["then"],"then-",o),a.on("then-start",function(t,e){this.promise=e,t[0]=c(t[0],"cb-",this),t[1]=c(t[1],"cb-",this)}),a.on("then-end",function(t,e,n){this.nextPromise=n;var r=this.promise;a.emit("propagate",[r,!0],n)}),a.on("cb-end",function(t,e,n){a.emit("propagate",[n,!0],this.nextPromise)}),a.on("propagate",function(t,e,n){this.getCtx&&!e||(this.getCtx=function(){if(t instanceof Promise)var e=a.context(t);return e&&e.getCtx?e.getCtx():this})}),r.toString=function(){return""+f})},{}],12:[function(t,e,n){var r=t("ee").get("raf"),o=t("wrap-function")(r),i="equestAnimationFrame";e.exports=r,o.inPlace(window,["r"+i,"mozR"+i,"webkitR"+i,"msR"+i],"raf-"),r.on("raf-start",function(t){t[0]=o(t[0],"fn-")})},{}],13:[function(t,e,n){function r(t,e,n){t[0]=a(t[0],"fn-",null,n)}function o(t,e,n){this.method=n,this.timerDuration=isNaN(t[1])?0:+t[1],t[0]=a(t[0],"fn-",this,n)}var i=t("ee").get("timer"),a=t("wrap-function")(i),c="setTimeout",s="setInterval",f="clearTimeout",u="-start",d="-";e.exports=i,a.inPlace(window,[c,"setImmediate"],c+d),a.inPlace(window,[s],s+d),a.inPlace(window,[f,"clearImmediate"],f+d),i.on(s+u,r),i.on(c+u,o)},{}],14:[function(t,e,n){function r(t,e){d.inPlace(e,["onreadystatechange"],"fn-",c)}function o(){var t=this,e=u.context(t);t.readyState>3&&!e.resolved&&(e.resolved=!0,u.emit("xhr-resolved",[],t)),d.inPlace(t,g,"fn-",c)}function i(t){y.push(t),h&&(b?b.then(a):v?v(a):(E=-E,R.data=E))}function a(){for(var t=0;t<y.length;t++)r([],y[t]);y.length&&(y=[])}function c(t,e){return e}function s(t,e){for(var n in t)e[n]=t[n];return e}t(6);var f=t("ee"),u=f.get("xhr"),d=t("wrap-function")(u),p=NREUM.o,l=p.XHR,h=p.MO,m=p.PR,v=p.SI,w="readystatechange",g=["onload","onerror","onabort","onloadstart","onloadend","onprogress","ontimeout"],y=[];e.exports=u;var x=window.XMLHttpRequest=function(t){var e=new l(t);try{u.emit("new-xhr",[e],e),e.addEventListener(w,o,!1)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}return e};if(s(l,x),x.prototype=l.prototype,d.inPlace(x.prototype,["open","send"],"-xhr-",c),u.on("send-xhr-start",function(t,e){r(t,e),i(e)}),u.on("open-xhr-start",r),h){var b=m&&m.resolve();if(!v&&!m){var E=1,R=document.createTextNode(E);new h(a).observe(R,{characterData:!0})}}else f.on("fn-end",function(t){t[0]&&t[0].type===w||a()})},{}],15:[function(t,e,n){function r(t){if(!c(t))return null;var e=window.NREUM;if(!e.loader_config)return null;var n=(e.loader_config.accountID||"").toString()||null,r=(e.loader_config.agentID||"").toString()||null,f=(e.loader_config.trustKey||"").toString()||null;if(!n||!r)return null;var h=l.generateSpanId(),m=l.generateTraceId(),v=Date.now(),w={spanId:h,traceId:m,timestamp:v};return(t.sameOrigin||s(t)&&p())&&(w.traceContextParentHeader=o(h,m),w.traceContextStateHeader=i(h,v,n,r,f)),(t.sameOrigin&&!u()||!t.sameOrigin&&s(t)&&d())&&(w.newrelicHeader=a(h,m,v,n,r,f)),w}function o(t,e){return"00-"+e+"-"+t+"-01"}function i(t,e,n,r,o){var i=0,a="",c=1,s="",f="";return o+"@nr="+i+"-"+c+"-"+n+"-"+r+"-"+t+"-"+a+"-"+s+"-"+f+"-"+e}function a(t,e,n,r,o,i){var a="btoa"in window&&"function"==typeof window.btoa;if(!a)return null;var c={v:[0,1],d:{ty:"Browser",ac:r,ap:o,id:t,tr:e,ti:n}};return i&&r!==i&&(c.d.tk=i),btoa(JSON.stringify(c))}function c(t){return f()&&s(t)}function s(t){var e=!1,n={};if("init"in NREUM&&"distributed_tracing"in NREUM.init&&(n=NREUM.init.distributed_tracing),t.sameOrigin)e=!0;else if(n.allowed_origins instanceof Array)for(var r=0;r<n.allowed_origins.length;r++){var o=h(n.allowed_origins[r]);if(t.hostname===o.hostname&&t.protocol===o.protocol&&t.port===o.port){e=!0;break}}return e}function f(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.enabled}function u(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.exclude_newrelic_header}function d(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&NREUM.init.distributed_tracing.cors_use_newrelic_header!==!1}function p(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.cors_use_tracecontext_headers}var l=t(23),h=t(17);e.exports={generateTracePayload:r,shouldGenerateTrace:c}},{}],16:[function(t,e,n){function r(t){var e=this.params,n=this.metrics;if(!this.ended){this.ended=!0;for(var r=0;r<p;r++)t.removeEventListener(d[r],this.listener,!1);e.aborted||(n.duration=a.now()-this.startTime,this.loadCaptureCalled||4!==t.readyState?null==e.status&&(e.status=0):i(this,t),n.cbTime=this.cbTime,u.emit("xhr-done",[t],t),c("xhr",[e,n,this.startTime]))}}function o(t,e){var n=s(e),r=t.params;r.host=n.hostname+":"+n.port,r.pathname=n.pathname,t.parsedOrigin=s(e),t.sameOrigin=t.parsedOrigin.sameOrigin}function i(t,e){t.params.status=e.status;var n=v(e,t.lastSize);if(n&&(t.metrics.rxSize=n),t.sameOrigin){var r=e.getResponseHeader("X-NewRelic-App-Data");r&&(t.params.cat=r.split(", ").pop())}t.loadCaptureCalled=!0}var a=t("loader");if(a.xhrWrappable){var c=t("handle"),s=t(17),f=t(15).generateTracePayload,u=t("ee"),d=["load","error","abort","timeout"],p=d.length,l=t("id"),h=t(21),m=t(20),v=t(18),w=window.XMLHttpRequest;a.features.xhr=!0,t(14),t(7),u.on("new-xhr",function(t){var e=this;e.totalCbs=0,e.called=0,e.cbTime=0,e.end=r,e.ended=!1,e.xhrGuids={},e.lastSize=null,e.loadCaptureCalled=!1,t.addEventListener("load",function(n){i(e,t)},!1),h&&(h>34||h<10)||window.opera||t.addEventListener("progress",function(t){e.lastSize=t.loaded},!1)}),u.on("open-xhr-start",function(t){this.params={method:t[0]},o(this,t[1]),this.metrics={}}),u.on("open-xhr-end",function(t,e){"loader_config"in NREUM&&"xpid"in NREUM.loader_config&&this.sameOrigin&&e.setRequestHeader("X-NewRelic-ID",NREUM.loader_config.xpid);var n=f(this.parsedOrigin);if(n){var r=!1;n.newrelicHeader&&(e.setRequestHeader("newrelic",n.newrelicHeader),r=!0),n.traceContextParentHeader&&(e.setRequestHeader("traceparent",n.traceContextParentHeader),n.traceContextStateHeader&&e.setRequestHeader("tracestate",n.traceContextStateHeader),r=!0),r&&(this.dt=n)}}),u.on("send-xhr-start",function(t,e){var n=this.metrics,r=t[0],o=this;if(n&&r){var i=m(r);i&&(n.txSize=i)}this.startTime=a.now(),this.listener=function(t){try{"abort"!==t.type||o.loadCaptureCalled||(o.params.aborted=!0),("load"!==t.type||o.called===o.totalCbs&&(o.onloadCalled||"function"!=typeof e.onload))&&o.end(e)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}};for(var c=0;c<p;c++)e.addEventListener(d[c],this.listener,!1)}),u.on("xhr-cb-time",function(t,e,n){this.cbTime+=t,e?this.onloadCalled=!0:this.called+=1,this.called!==this.totalCbs||!this.onloadCalled&&"function"==typeof n.onload||this.end(n)}),u.on("xhr-load-added",function(t,e){var n=""+l(t)+!!e;this.xhrGuids&&!this.xhrGuids[n]&&(this.xhrGuids[n]=!0,this.totalCbs+=1)}),u.on("xhr-load-removed",function(t,e){var n=""+l(t)+!!e;this.xhrGuids&&this.xhrGuids[n]&&(delete this.xhrGuids[n],this.totalCbs-=1)}),u.on("addEventListener-end",function(t,e){e instanceof w&&"load"===t[0]&&u.emit("xhr-load-added",[t[1],t[2]],e)}),u.on("removeEventListener-end",function(t,e){e instanceof w&&"load"===t[0]&&u.emit("xhr-load-removed",[t[1],t[2]],e)}),u.on("fn-start",function(t,e,n){e instanceof w&&("onload"===n&&(this.onload=!0),("load"===(t[0]&&t[0].type)||this.onload)&&(this.xhrCbStart=a.now()))}),u.on("fn-end",function(t,e){this.xhrCbStart&&u.emit("xhr-cb-time",[a.now()-this.xhrCbStart,this.onload,e],e)}),u.on("fetch-before-start",function(t){function e(t,e){var n=!1;return e.newrelicHeader&&(t.set("newrelic",e.newrelicHeader),n=!0),e.traceContextParentHeader&&(t.set("traceparent",e.traceContextParentHeader),e.traceContextStateHeader&&t.set("tracestate",e.traceContextStateHeader),n=!0),n}var n,r=t[1]||{};"string"==typeof t[0]?n=t[0]:t[0]&&t[0].url&&(n=t[0].url),n&&(this.parsedOrigin=s(n),this.sameOrigin=this.parsedOrigin.sameOrigin);var o=f(this.parsedOrigin);if(o&&(o.newrelicHeader||o.traceContextParentHeader))if("string"==typeof t[0]){var i={};for(var a in r)i[a]=r[a];i.headers=new Headers(r.headers||{}),e(i.headers,o)&&(this.dt=o),t.length>1?t[1]=i:t.push(i)}else t[0]&&t[0].headers&&e(t[0].headers,o)&&(this.dt=o)})}},{}],17:[function(t,e,n){var r={};e.exports=function(t){if(t in r)return r[t];var e=document.createElement("a"),n=window.location,o={};e.href=t,o.port=e.port;var i=e.href.split("://");!o.port&&i[1]&&(o.port=i[1].split("/")[0].split("@").pop().split(":")[1]),o.port&&"0"!==o.port||(o.port="https"===i[0]?"443":"80"),o.hostname=e.hostname||n.hostname,o.pathname=e.pathname,o.protocol=i[0],"/"!==o.pathname.charAt(0)&&(o.pathname="/"+o.pathname);var a=!e.protocol||":"===e.protocol||e.protocol===n.protocol,c=e.hostname===document.domain&&e.port===n.port;return o.sameOrigin=a&&(!e.hostname||c),"/"===o.pathname&&(r[t]=o),o}},{}],18:[function(t,e,n){function r(t,e){var n=t.responseType;return"json"===n&&null!==e?e:"arraybuffer"===n||"blob"===n||"json"===n?o(t.response):"text"===n||""===n||void 0===n?o(t.responseText):void 0}var o=t(20);e.exports=r},{}],19:[function(t,e,n){function r(){}function o(t,e,n){return function(){return i(t,[f.now()].concat(c(arguments)),e?null:this,n),e?void 0:this}}var i=t("handle"),a=t(26),c=t(27),s=t("ee").get("tracer"),f=t("loader"),u=NREUM;"undefined"==typeof window.newrelic&&(newrelic=u);var d=["setPageViewName","setCustomAttribute","setErrorHandler","finished","addToTrace","inlineHit","addRelease"],p="api-",l=p+"ixn-";a(d,function(t,e){u[e]=o(p+e,!0,"api")}),u.addPageAction=o(p+"addPageAction",!0),u.setCurrentRouteName=o(p+"routeName",!0),e.exports=newrelic,u.interaction=function(){return(new r).get()};var h=r.prototype={createTracer:function(t,e){var n={},r=this,o="function"==typeof e;return i(l+"tracer",[f.now(),t,n],r),function(){if(s.emit((o?"":"no-")+"fn-start",[f.now(),r,o],n),o)try{return e.apply(this,arguments)}catch(t){throw s.emit("fn-err",[arguments,this,t],n),t}finally{s.emit("fn-end",[f.now()],n)}}}};a("actionText,setName,setAttribute,save,ignore,onEnd,getContext,end,get".split(","),function(t,e){h[e]=o(l+e)}),newrelic.noticeError=function(t,e){"string"==typeof t&&(t=new Error(t)),i("err",[t,f.now(),!1,e])}},{}],20:[function(t,e,n){e.exports=function(t){if("string"==typeof t&&t.length)return t.length;if("object"==typeof t){if("undefined"!=typeof ArrayBuffer&&t instanceof ArrayBuffer&&t.byteLength)return t.byteLength;if("undefined"!=typeof Blob&&t instanceof Blob&&t.size)return t.size;if(!("undefined"!=typeof FormData&&t instanceof FormData))try{return JSON.stringify(t).length}catch(e){return}}}},{}],21:[function(t,e,n){var r=0,o=navigator.userAgent.match(/Firefox[\/\s](\d+\.\d+)/);o&&(r=+o[1]),e.exports=r},{}],22:[function(t,e,n){function r(t,e){var n=t.getEntries();n.forEach(function(t){"first-paint"===t.name?d("timing",["fp",Math.floor(t.startTime)]):"first-contentful-paint"===t.name&&d("timing",["fcp",Math.floor(t.startTime)])})}function o(t,e){var n=t.getEntries();n.length>0&&d("lcp",[n[n.length-1]])}function i(t){t.getEntries().forEach(function(t){t.hadRecentInput||d("cls",[t])})}function a(t){if(t instanceof h&&!v){var e=Math.round(t.timeStamp),n={type:t.type};e<=p.now()?n.fid=p.now()-e:e>p.offset&&e<=Date.now()?(e-=p.offset,n.fid=p.now()-e):e=p.now(),v=!0,d("timing",["fi",e,n])}}function c(t){d("pageHide",[p.now(),t])}if(!("init"in NREUM&&"page_view_timing"in NREUM.init&&"enabled"in NREUM.init.page_view_timing&&NREUM.init.page_view_timing.enabled===!1)){var s,f,u,d=t("handle"),p=t("loader"),l=t(25),h=NREUM.o.EV;if("PerformanceObserver"in window&&"function"==typeof window.PerformanceObserver){s=new PerformanceObserver(r);try{s.observe({entryTypes:["paint"]})}catch(m){}f=new PerformanceObserver(o);try{f.observe({entryTypes:["largest-contentful-paint"]})}catch(m){}u=new PerformanceObserver(i);try{u.observe({type:"layout-shift",buffered:!0})}catch(m){}}if("addEventListener"in document){var v=!1,w=["click","keydown","mousedown","pointerdown","touchstart"];w.forEach(function(t){document.addEventListener(t,a,!1)})}l(c)}},{}],23:[function(t,e,n){function r(){function t(){return e?15&e[n++]:16*Math.random()|0}var e=null,n=0,r=window.crypto||window.msCrypto;r&&r.getRandomValues&&(e=r.getRandomValues(new Uint8Array(31)));for(var o,i="xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx",a="",c=0;c<i.length;c++)o=i[c],"x"===o?a+=t().toString(16):"y"===o?(o=3&t()|8,a+=o.toString(16)):a+=o;return a}function o(){return a(16)}function i(){return a(32)}function a(t){function e(){return n?15&n[r++]:16*Math.random()|0}var n=null,r=0,o=window.crypto||window.msCrypto;o&&o.getRandomValues&&Uint8Array&&(n=o.getRandomValues(new Uint8Array(31)));for(var i=[],a=0;a<t;a++)i.push(e().toString(16));return i.join("")}e.exports={generateUuid:r,generateSpanId:o,generateTraceId:i}},{}],24:[function(t,e,n){function r(t,e){if(!o)return!1;if(t!==o)return!1;if(!e)return!0;if(!i)return!1;for(var n=i.split("."),r=e.split("."),a=0;a<r.length;a++)if(r[a]!==n[a])return!1;return!0}var o=null,i=null,a=/Version\/(\S+)\s+Safari/;if(navigator.userAgent){var c=navigator.userAgent,s=c.match(a);s&&c.indexOf("Chrome")===-1&&c.indexOf("Chromium")===-1&&(o="Safari",i=s[1])}e.exports={agent:o,version:i,match:r}},{}],25:[function(t,e,n){function r(t){function e(){t(a&&document[a]?document[a]:document[o]?"hidden":"visible")}"addEventListener"in document&&i&&document.addEventListener(i,e,!1)}e.exports=r;var o,i,a;"undefined"!=typeof document.hidden?(o="hidden",i="visibilitychange",a="visibilityState"):"undefined"!=typeof document.msHidden?(o="msHidden",i="msvisibilitychange"):"undefined"!=typeof document.webkitHidden&&(o="webkitHidden",i="webkitvisibilitychange",a="webkitVisibilityState")},{}],26:[function(t,e,n){function r(t,e){var n=[],r="",i=0;for(r in t)o.call(t,r)&&(n[i]=e(r,t[r]),i+=1);return n}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],27:[function(t,e,n){function r(t,e,n){e||(e=0),"undefined"==typeof n&&(n=t?t.length:0);for(var r=-1,o=n-e||0,i=Array(o<0?0:o);++r<o;)i[r]=t[e+r];return i}e.exports=r},{}],28:[function(t,e,n){e.exports={exists:"undefined"!=typeof window.performance&&window.performance.timing&&"undefined"!=typeof window.performance.timing.navigationStart}},{}],ee:[function(t,e,n){function r(){}function o(t){function e(t){return t&&t instanceof r?t:t?s(t,c,i):i()}function n(n,r,o,i){if(!p.aborted||i){t&&t(n,r,o);for(var a=e(o),c=m(n),s=c.length,f=0;f<s;f++)c[f].apply(a,r);var d=u[y[n]];return d&&d.push([x,n,r,a]),a}}function l(t,e){g[t]=m(t).concat(e)}function h(t,e){var n=g[t];if(n)for(var r=0;r<n.length;r++)n[r]===e&&n.splice(r,1)}function m(t){return g[t]||[]}function v(t){return d[t]=d[t]||o(n)}function w(t,e){f(t,function(t,n){e=e||"feature",y[n]=e,e in u||(u[e]=[])})}var g={},y={},x={on:l,addEventListener:l,removeEventListener:h,emit:n,get:v,listeners:m,context:e,buffer:w,abort:a,aborted:!1};return x}function i(){return new r}function a(){(u.api||u.feature)&&(p.aborted=!0,u=p.backlog={})}var c="nr@context",s=t("gos"),f=t(26),u={},d={},p=e.exports=o();p.backlog=u},{}],gos:[function(t,e,n){function r(t,e,n){if(o.call(t,e))return t[e];var r=n();if(Object.defineProperty&&Object.keys)try{return Object.defineProperty(t,e,{value:r,writable:!0,enumerable:!1}),r}catch(i){}return t[e]=r,r}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],handle:[function(t,e,n){function r(t,e,n,r){o.buffer([t],r),o.emit(t,e,n)}var o=t("ee").get("handle");e.exports=r,r.ee=o},{}],id:[function(t,e,n){function r(t){var e=typeof t;return!t||"object"!==e&&"function"!==e?-1:t===window?0:a(t,i,function(){return o++})}var o=1,i="nr@id",a=t("gos");e.exports=r},{}],loader:[function(t,e,n){function r(){if(!E++){var t=b.info=NREUM.info,e=l.getElementsByTagName("script")[0];if(setTimeout(u.abort,3e4),!(t&&t.licenseKey&&t.applicationID&&e))return u.abort();f(y,function(e,n){t[e]||(t[e]=n)});var n=a();s("mark",["onload",n+b.offset],null,"api"),s("timing",["load",n]);var r=l.createElement("script");r.src="https://"+t.agent,e.parentNode.insertBefore(r,e)}}function o(){"complete"===l.readyState&&i()}function i(){s("mark",["domContent",a()+b.offset],null,"api")}function a(){return R.exists&&performance.now?Math.round(performance.now()):(c=Math.max((new Date).getTime(),c))-b.offset}var c=(new Date).getTime(),s=t("handle"),f=t(26),u=t("ee"),d=t(24),p=window,l=p.document,h="addEventListener",m="attachEvent",v=p.XMLHttpRequest,w=v&&v.prototype;NREUM.o={ST:setTimeout,SI:p.setImmediate,CT:clearTimeout,XHR:v,REQ:p.Request,EV:p.Event,PR:p.Promise,MO:p.MutationObserver};var g=""+location,y={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",agent:"js-agent.newrelic.com/nr-spa-1177.min.js"},x=v&&w&&w[h]&&!/CriOS/.test(navigator.userAgent),b=e.exports={offset:c,now:a,origin:g,features:{},xhrWrappable:x,userAgent:d};t(19),t(22),l[h]?(l[h]("DOMContentLoaded",i,!1),p[h]("load",r,!1)):(l[m]("onreadystatechange",o),p[m]("onload",r)),s("mark",["firstbyte",c],null,"api");var E=0,R=t(28)},{}],"wrap-function":[function(t,e,n){function r(t){return!(t&&t instanceof Function&&t.apply&&!t[a])}var o=t("ee"),i=t(27),a="nr@original",c=Object.prototype.hasOwnProperty,s=!1;e.exports=function(t,e){function n(t,e,n,o){function nrWrapper(){var r,a,c,s;try{a=this,r=i(arguments),c="function"==typeof n?n(r,a):n||{}}catch(f){p([f,"",[r,a,o],c])}u(e+"start",[r,a,o],c);try{return s=t.apply(a,r)}catch(d){throw u(e+"err",[r,a,d],c),d}finally{u(e+"end",[r,a,s],c)}}return r(t)?t:(e||(e=""),nrWrapper[a]=t,d(t,nrWrapper),nrWrapper)}function f(t,e,o,i){o||(o="");var a,c,s,f="-"===o.charAt(0);for(s=0;s<e.length;s++)c=e[s],a=t[c],r(a)||(t[c]=n(a,f?c+o:o,i,c))}function u(n,r,o){if(!s||e){var i=s;s=!0;try{t.emit(n,r,o,e)}catch(a){p([a,n,r,o])}s=i}}function d(t,e){if(Object.defineProperty&&Object.keys)try{var n=Object.keys(t);return n.forEach(function(n){Object.defineProperty(e,n,{get:function(){return t[n]},set:function(e){return t[n]=e,e}})}),e}catch(r){p([r])}for(var o in t)c.call(t,o)&&(e[o]=t[o]);return e}function p(e){try{t.emit("internal-error",e)}catch(n){}}return t||(t=o),n.inPlace=f,n.flag=a,n}},{}]},{},["loader",2,16,5,3,4]);
    ;NREUM.loader_config={accountID:"2095297",trustKey:"2038175",agentID:"916957862",licenseKey:"16e2192192",applicationID:"916957862"}
    ;NREUM.info={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",licenseKey:"16e2192192",applicationID:"916957862",sa:1}
</script>
    <script src="https://cdn.cookielaw.org/scripttemplates/otSDKStub.js" type="text/javascript" charset="UTF-8" data-domain-script="dada9834-04e2-4bcd-b5ba-cae44d498dfa"></script>
    <script type="text/javascript">
        function getCookie(name) {
            var value = "; " + document.cookie;
            var parts = value.split("; " + name + "=");
            if (parts.length == 2) {
                return true;
            }
        }
            function OptanonWrapper() {
                var optCookie = getCookie("OptanonAlertBoxClosed");
                var isActiveCategory = "false";
                var isFoundCookieBox = "false";
                if(isActiveCategory == 'false' && isFoundCookieBox == 'false' && optCookie) {
                    OneTrust.InsertScript('/pb-assets/adobe/prod/adobedtm-1616606406233.js', 'head', null, null, '2');
                }
            }
    </script>


<noscript>
    <style>.requiresJS {
        display: none
    }

    .dartAd {
        display: block !important
    }</style>
</noscript>
        <script type="text/javascript" src="/wro/o4ny~selected-legacy.js"></script>
        <link rel="stylesheet" type="text/css" href="/wro/o4ny~selected-legacy.css">
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Open+Sans">
        <script>window.MSInputMethodContext && document.documentMode && document.write('<script src="https://cdn.jsdelivr.net/gh/nuxodin/ie11CustomProperties@4.1.0/ie11CustomProperties.min.js"><\x2fscript>');</script><link rel="stylesheet" href="/products/marlin/cell/releasedAssets/css/build-b26f3342b722959c7931.css"><link rel="stylesheet" href="/products/marlin/cell/releasedAssets/css/print-b26f3342b722959c7931.css" media="print"><link rel="stylesheet" href="/products/marlin/cell/releasedAssets/css/themes/theme-cell.css">







        
        
        <script class="optanon-category-4" type="text/plain" async="" src="https://www.youtube.com/iframe_api"></script>
        
    





<!---->
<meta http-equiv="X-UA-Compatible" content="IE=edge">












<script src="https://cdn.cookielaw.org/scripttemplates/202411.2.0/otBannerSdk.js" async="" type="text/javascript"></script><style id="onetrust-style">#onetrust-banner-sdk .onetrust-vendors-list-handler{cursor:pointer;color:#1f96db;font-size:inherit;font-weight:bold;text-decoration:none;margin-left:5px}#onetrust-banner-sdk .onetrust-vendors-list-handler:hover{color:#1f96db}#onetrust-banner-sdk:focus{outline:2px solid #000;outline-offset:-2px}#onetrust-banner-sdk a:focus{outline:2px solid #000}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{outline-offset:1px}#onetrust-banner-sdk.ot-bnr-w-logo .ot-bnr-logo{height:64px;width:64px}#onetrust-banner-sdk .ot-tcf2-vendor-count.ot-text-bold{font-weight:bold}#onetrust-banner-sdk .ot-close-icon,#onetrust-pc-sdk .ot-close-icon,#ot-sync-ntfy .ot-close-icon{background-size:contain;background-repeat:no-repeat;background-position:center;height:12px;width:12px}#onetrust-banner-sdk .powered-by-logo,#onetrust-banner-sdk .ot-pc-footer-logo a,#onetrust-pc-sdk .powered-by-logo,#onetrust-pc-sdk .ot-pc-footer-logo a,#ot-sync-ntfy .powered-by-logo,#ot-sync-ntfy .ot-pc-footer-logo a{background-size:contain;background-repeat:no-repeat;background-position:center;height:25px;width:152px;display:block;text-decoration:none;font-size:.75em}#onetrust-banner-sdk .powered-by-logo:hover,#onetrust-banner-sdk .ot-pc-footer-logo a:hover,#onetrust-pc-sdk .powered-by-logo:hover,#onetrust-pc-sdk .ot-pc-footer-logo a:hover,#ot-sync-ntfy .powered-by-logo:hover,#ot-sync-ntfy .ot-pc-footer-logo a:hover{color:#565656}#onetrust-banner-sdk h3 *,#onetrust-banner-sdk h4 *,#onetrust-banner-sdk h6 *,#onetrust-banner-sdk button *,#onetrust-banner-sdk a[data-parent-id] *,#onetrust-pc-sdk h3 *,#onetrust-pc-sdk h4 *,#onetrust-pc-sdk h6 *,#onetrust-pc-sdk button *,#onetrust-pc-sdk a[data-parent-id] *,#ot-sync-ntfy h3 *,#ot-sync-ntfy h4 *,#ot-sync-ntfy h6 *,#ot-sync-ntfy button *,#ot-sync-ntfy a[data-parent-id] *{font-size:inherit;font-weight:inherit;color:inherit}#onetrust-banner-sdk .ot-hide,#onetrust-pc-sdk .ot-hide,#ot-sync-ntfy .ot-hide{display:none !important}#onetrust-banner-sdk button.ot-link-btn:hover,#onetrust-pc-sdk button.ot-link-btn:hover,#ot-sync-ntfy button.ot-link-btn:hover{text-decoration:underline;opacity:1}#onetrust-pc-sdk .ot-sdk-row .ot-sdk-column{padding:0}#onetrust-pc-sdk .ot-sdk-container{padding-right:0}#onetrust-pc-sdk .ot-sdk-row{flex-direction:initial;width:100%}#onetrust-pc-sdk [type=checkbox]:checked,#onetrust-pc-sdk [type=checkbox]:not(:checked){pointer-events:initial}#onetrust-pc-sdk [type=checkbox]:disabled+label::before,#onetrust-pc-sdk [type=checkbox]:disabled+label:after,#onetrust-pc-sdk [type=checkbox]:disabled+label{pointer-events:none;opacity:.8}#onetrust-pc-sdk #vendor-list-content{transform:translate3d(0, 0, 0)}#onetrust-pc-sdk li input[type=checkbox]{z-index:1}#onetrust-pc-sdk li .ot-checkbox label{z-index:2}#onetrust-pc-sdk li .ot-checkbox input[type=checkbox]{height:auto;width:auto}#onetrust-pc-sdk li .host-title a,#onetrust-pc-sdk li .ot-host-name a,#onetrust-pc-sdk li .accordion-text,#onetrust-pc-sdk li .ot-acc-txt{z-index:2;position:relative}#onetrust-pc-sdk input{margin:3px .1ex}#onetrust-pc-sdk .pc-logo,#onetrust-pc-sdk .ot-pc-logo{height:60px;width:180px;background-position:center;background-size:contain;background-repeat:no-repeat;display:inline-flex;justify-content:center;align-items:center}#onetrust-pc-sdk .pc-logo img,#onetrust-pc-sdk .ot-pc-logo img{max-height:100%;max-width:100%}#onetrust-pc-sdk .screen-reader-only,#onetrust-pc-sdk .ot-scrn-rdr,.ot-sdk-cookie-policy .screen-reader-only,.ot-sdk-cookie-policy .ot-scrn-rdr{border:0;clip:rect(0 0 0 0);height:1px;margin:-1px;overflow:hidden;padding:0;position:absolute;width:1px}#onetrust-pc-sdk.ot-fade-in,.onetrust-pc-dark-filter.ot-fade-in,#onetrust-banner-sdk.ot-fade-in{animation-name:onetrust-fade-in;animation-duration:400ms;animation-timing-function:ease-in-out}#onetrust-pc-sdk.ot-hide{display:none !important}.onetrust-pc-dark-filter.ot-hide{display:none !important}#ot-sdk-btn.ot-sdk-show-settings,#ot-sdk-btn.optanon-show-settings{color:#fff;background-color:#468254;height:auto;white-space:normal;word-wrap:break-word;padding:.8em 2em;font-size:.8em;line-height:1.2;cursor:pointer;-moz-transition:.1s ease;-o-transition:.1s ease;-webkit-transition:1s ease;transition:.1s ease}#ot-sdk-btn.ot-sdk-show-settings:hover,#ot-sdk-btn.optanon-show-settings:hover{color:#fff;background-color:#2c6415}#ot-sdk-btn.ot-sdk-show-settings:active,#ot-sdk-btn.optanon-show-settings:active{color:#fff;background-color:#2c6415;border:1px solid rgba(162,192,169,.5)}.onetrust-pc-dark-filter{background:rgba(0,0,0,.5);z-index:2147483646;width:100%;height:100%;overflow:hidden;position:fixed;top:0;bottom:0;left:0}@keyframes onetrust-fade-in{0%{opacity:0}100%{opacity:1}}.ot-cookie-label{text-decoration:underline}@media only screen and (min-width: 426px)and (max-width: 896px)and (orientation: landscape){#onetrust-pc-sdk p{font-size:.75em}}#onetrust-banner-sdk .banner-option-input:focus+label{outline:1px solid #000;outline-style:auto}.category-vendors-list-handler+a:focus,.category-vendors-list-handler+a:focus-visible{outline:2px solid #000}#onetrust-pc-sdk .ot-userid-title{margin-top:10px}#onetrust-pc-sdk .ot-userid-title>span,#onetrust-pc-sdk .ot-userid-timestamp>span{font-weight:700}#onetrust-pc-sdk .ot-userid-desc{font-style:italic}#onetrust-pc-sdk .ot-host-desc a{pointer-events:initial}#onetrust-pc-sdk .ot-ven-hdr>p a{position:relative;z-index:2;pointer-events:initial}#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info a,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info a{margin-right:auto}#onetrust-pc-sdk .ot-pc-footer-logo img{width:136px;height:16px}#onetrust-pc-sdk .ot-pur-vdr-count{font-weight:400;font-size:.7rem;padding-top:3px;display:block}#onetrust-banner-sdk .ot-optout-signal,#onetrust-pc-sdk .ot-optout-signal{border:1px solid #32ae88;border-radius:3px;padding:5px;margin-bottom:10px;background-color:#f9fffa;font-size:.85rem;line-height:2}#onetrust-banner-sdk .ot-optout-signal .ot-optout-icon,#onetrust-pc-sdk .ot-optout-signal .ot-optout-icon{display:inline;margin-right:5px}#onetrust-banner-sdk .ot-optout-signal svg,#onetrust-pc-sdk .ot-optout-signal svg{height:20px;width:30px;transform:scale(0.5)}#onetrust-banner-sdk .ot-optout-signal svg path,#onetrust-pc-sdk .ot-optout-signal svg path{fill:#32ae88}#onetrust-consent-sdk .ot-general-modal{overflow:hidden;position:fixed;margin:0 auto;top:50%;left:50%;width:40%;padding:1.5rem;max-width:575px;min-width:575px;z-index:2147483647;border-radius:2.5px;transform:translate(-50%, -50%)}#onetrust-consent-sdk .ot-signature-health-group{margin-top:1rem;padding-left:1.25rem;padding-right:1.25rem;margin-bottom:.625rem;width:calc(100% - 2.5rem)}#onetrust-consent-sdk .ot-signature-health-group .ot-signature-health-form{gap:.5rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-health-form{width:70%;gap:.35rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-input{height:38px;padding:6px 10px;background-color:#fff;border:1px solid #d1d1d1;border-radius:4px;box-shadow:none;box-sizing:border-box}#onetrust-consent-sdk .ot-signature-health .ot-signature-subtitle{font-size:1.125rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-group-title{font-size:1.25rem;font-weight:bold}#onetrust-consent-sdk .ot-signature-health,#onetrust-consent-sdk .ot-signature-health-group{display:flex;flex-direction:column;gap:1rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-cont,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-cont{display:flex;flex-direction:column;gap:.25rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-paragraph,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-paragraph{margin:0;line-height:20px;font-size:max(14px,.875rem)}#onetrust-consent-sdk .ot-signature-health .ot-health-signature-error,#onetrust-consent-sdk .ot-signature-health-group .ot-health-signature-error{color:#4d4d4d;font-size:min(12px,.75rem)}#onetrust-consent-sdk .ot-signature-health .ot-signature-buttons-cont,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-buttons-cont{margin-top:max(.75rem,2%);gap:1rem;display:flex;justify-content:flex-end}#onetrust-consent-sdk .ot-signature-health .ot-signature-button,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-button{flex:1;height:auto;color:#fff;cursor:pointer;line-height:1.2;min-width:125px;font-weight:600;font-size:.813em;border-radius:2px;padding:12px 10px;white-space:normal;word-wrap:break-word;word-break:break-word;background-color:#68b631;border:2px solid #68b631}#onetrust-consent-sdk .ot-signature-health .ot-signature-button.reject,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-button.reject{background-color:#fff}#onetrust-consent-sdk .ot-input-field-cont{display:flex;flex-direction:column;gap:.5rem}#onetrust-consent-sdk .ot-input-field-cont .ot-signature-input{width:65%}#onetrust-consent-sdk .ot-signature-health-form{display:flex;flex-direction:column}#onetrust-consent-sdk .ot-signature-health-form .ot-signature-label{margin-bottom:0;line-height:20px;font-size:max(14px,.875rem)}@media only screen and (max-width: 600px){#onetrust-consent-sdk .ot-general-modal{min-width:100%}#onetrust-consent-sdk .ot-signature-health .ot-signature-health-form{width:100%}#onetrust-consent-sdk .ot-input-field-cont .ot-signature-input{width:100%}}#onetrust-banner-sdk,#onetrust-pc-sdk,#ot-sdk-cookie-policy,#ot-sync-ntfy{font-size:16px}#onetrust-banner-sdk *,#onetrust-banner-sdk ::after,#onetrust-banner-sdk ::before,#onetrust-pc-sdk *,#onetrust-pc-sdk ::after,#onetrust-pc-sdk ::before,#ot-sdk-cookie-policy *,#ot-sdk-cookie-policy ::after,#ot-sdk-cookie-policy ::before,#ot-sync-ntfy *,#ot-sync-ntfy ::after,#ot-sync-ntfy ::before{-webkit-box-sizing:content-box;-moz-box-sizing:content-box;box-sizing:content-box}#onetrust-banner-sdk div,#onetrust-banner-sdk span,#onetrust-banner-sdk h1,#onetrust-banner-sdk h2,#onetrust-banner-sdk h3,#onetrust-banner-sdk h4,#onetrust-banner-sdk h5,#onetrust-banner-sdk h6,#onetrust-banner-sdk p,#onetrust-banner-sdk img,#onetrust-banner-sdk svg,#onetrust-banner-sdk button,#onetrust-banner-sdk section,#onetrust-banner-sdk a,#onetrust-banner-sdk label,#onetrust-banner-sdk input,#onetrust-banner-sdk ul,#onetrust-banner-sdk li,#onetrust-banner-sdk nav,#onetrust-banner-sdk table,#onetrust-banner-sdk thead,#onetrust-banner-sdk tr,#onetrust-banner-sdk td,#onetrust-banner-sdk tbody,#onetrust-banner-sdk .ot-main-content,#onetrust-banner-sdk .ot-toggle,#onetrust-banner-sdk #ot-content,#onetrust-banner-sdk #ot-pc-content,#onetrust-banner-sdk .checkbox,#onetrust-pc-sdk div,#onetrust-pc-sdk span,#onetrust-pc-sdk h1,#onetrust-pc-sdk h2,#onetrust-pc-sdk h3,#onetrust-pc-sdk h4,#onetrust-pc-sdk h5,#onetrust-pc-sdk h6,#onetrust-pc-sdk p,#onetrust-pc-sdk img,#onetrust-pc-sdk svg,#onetrust-pc-sdk button,#onetrust-pc-sdk section,#onetrust-pc-sdk a,#onetrust-pc-sdk label,#onetrust-pc-sdk input,#onetrust-pc-sdk ul,#onetrust-pc-sdk li,#onetrust-pc-sdk nav,#onetrust-pc-sdk table,#onetrust-pc-sdk thead,#onetrust-pc-sdk tr,#onetrust-pc-sdk td,#onetrust-pc-sdk tbody,#onetrust-pc-sdk .ot-main-content,#onetrust-pc-sdk .ot-toggle,#onetrust-pc-sdk #ot-content,#onetrust-pc-sdk #ot-pc-content,#onetrust-pc-sdk .checkbox,#ot-sdk-cookie-policy div,#ot-sdk-cookie-policy span,#ot-sdk-cookie-policy h1,#ot-sdk-cookie-policy h2,#ot-sdk-cookie-policy h3,#ot-sdk-cookie-policy h4,#ot-sdk-cookie-policy h5,#ot-sdk-cookie-policy h6,#ot-sdk-cookie-policy p,#ot-sdk-cookie-policy img,#ot-sdk-cookie-policy svg,#ot-sdk-cookie-policy button,#ot-sdk-cookie-policy section,#ot-sdk-cookie-policy a,#ot-sdk-cookie-policy label,#ot-sdk-cookie-policy input,#ot-sdk-cookie-policy ul,#ot-sdk-cookie-policy li,#ot-sdk-cookie-policy nav,#ot-sdk-cookie-policy table,#ot-sdk-cookie-policy thead,#ot-sdk-cookie-policy tr,#ot-sdk-cookie-policy td,#ot-sdk-cookie-policy tbody,#ot-sdk-cookie-policy .ot-main-content,#ot-sdk-cookie-policy .ot-toggle,#ot-sdk-cookie-policy #ot-content,#ot-sdk-cookie-policy #ot-pc-content,#ot-sdk-cookie-policy .checkbox,#ot-sync-ntfy div,#ot-sync-ntfy span,#ot-sync-ntfy h1,#ot-sync-ntfy h2,#ot-sync-ntfy h3,#ot-sync-ntfy h4,#ot-sync-ntfy h5,#ot-sync-ntfy h6,#ot-sync-ntfy p,#ot-sync-ntfy img,#ot-sync-ntfy svg,#ot-sync-ntfy button,#ot-sync-ntfy section,#ot-sync-ntfy a,#ot-sync-ntfy label,#ot-sync-ntfy input,#ot-sync-ntfy ul,#ot-sync-ntfy li,#ot-sync-ntfy nav,#ot-sync-ntfy table,#ot-sync-ntfy thead,#ot-sync-ntfy tr,#ot-sync-ntfy td,#ot-sync-ntfy tbody,#ot-sync-ntfy .ot-main-content,#ot-sync-ntfy .ot-toggle,#ot-sync-ntfy #ot-content,#ot-sync-ntfy #ot-pc-content,#ot-sync-ntfy .checkbox{font-family:inherit;font-weight:normal;-webkit-font-smoothing:auto;letter-spacing:normal;line-height:normal;padding:0;margin:0;height:auto;min-height:0;max-height:none;width:auto;min-width:0;max-width:none;border-radius:0;border:none;clear:none;float:none;position:static;bottom:auto;left:auto;right:auto;top:auto;text-align:left;text-decoration:none;text-indent:0;text-shadow:none;text-transform:none;white-space:normal;background:none;overflow:visible;vertical-align:baseline;visibility:visible;z-index:auto;box-shadow:none}#onetrust-banner-sdk label:before,#onetrust-banner-sdk label:after,#onetrust-banner-sdk .checkbox:after,#onetrust-banner-sdk .checkbox:before,#onetrust-pc-sdk label:before,#onetrust-pc-sdk label:after,#onetrust-pc-sdk .checkbox:after,#onetrust-pc-sdk .checkbox:before,#ot-sdk-cookie-policy label:before,#ot-sdk-cookie-policy label:after,#ot-sdk-cookie-policy .checkbox:after,#ot-sdk-cookie-policy .checkbox:before,#ot-sync-ntfy label:before,#ot-sync-ntfy label:after,#ot-sync-ntfy .checkbox:after,#ot-sync-ntfy .checkbox:before{content:"";content:none}#onetrust-banner-sdk .ot-sdk-container,#onetrust-pc-sdk .ot-sdk-container,#ot-sdk-cookie-policy .ot-sdk-container{position:relative;width:100%;max-width:100%;margin:0 auto;padding:0 20px;box-sizing:border-box}#onetrust-banner-sdk .ot-sdk-column,#onetrust-banner-sdk .ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-column,#onetrust-pc-sdk .ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-column,#ot-sdk-cookie-policy .ot-sdk-columns{width:100%;float:left;box-sizing:border-box;padding:0;display:initial}@media(min-width: 400px){#onetrust-banner-sdk .ot-sdk-container,#onetrust-pc-sdk .ot-sdk-container,#ot-sdk-cookie-policy .ot-sdk-container{width:90%;padding:0}}@media(min-width: 550px){#onetrust-banner-sdk .ot-sdk-container,#onetrust-pc-sdk .ot-sdk-container,#ot-sdk-cookie-policy .ot-sdk-container{width:100%}#onetrust-banner-sdk .ot-sdk-column,#onetrust-banner-sdk .ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-column,#onetrust-pc-sdk .ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-column,#ot-sdk-cookie-policy .ot-sdk-columns{margin-left:4%}#onetrust-banner-sdk .ot-sdk-column:first-child,#onetrust-banner-sdk .ot-sdk-columns:first-child,#onetrust-pc-sdk .ot-sdk-column:first-child,#onetrust-pc-sdk .ot-sdk-columns:first-child,#ot-sdk-cookie-policy .ot-sdk-column:first-child,#ot-sdk-cookie-policy .ot-sdk-columns:first-child{margin-left:0}#onetrust-banner-sdk .ot-sdk-two.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-two.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-two.ot-sdk-columns{width:13.3333333333%}#onetrust-banner-sdk .ot-sdk-three.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-three.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-three.ot-sdk-columns{width:22%}#onetrust-banner-sdk .ot-sdk-four.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-four.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-four.ot-sdk-columns{width:30.6666666667%}#onetrust-banner-sdk .ot-sdk-eight.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-eight.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-eight.ot-sdk-columns{width:65.3333333333%}#onetrust-banner-sdk .ot-sdk-nine.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-nine.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-nine.ot-sdk-columns{width:74%}#onetrust-banner-sdk .ot-sdk-ten.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-ten.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-ten.ot-sdk-columns{width:82.6666666667%}#onetrust-banner-sdk .ot-sdk-eleven.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-eleven.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-eleven.ot-sdk-columns{width:91.3333333333%}#onetrust-banner-sdk .ot-sdk-twelve.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-twelve.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-twelve.ot-sdk-columns{width:100%;margin-left:0}}#onetrust-banner-sdk h1,#onetrust-banner-sdk h2,#onetrust-banner-sdk h3,#onetrust-banner-sdk h4,#onetrust-banner-sdk h5,#onetrust-banner-sdk h6,#onetrust-pc-sdk h1,#onetrust-pc-sdk h2,#onetrust-pc-sdk h3,#onetrust-pc-sdk h4,#onetrust-pc-sdk h5,#onetrust-pc-sdk h6,#ot-sdk-cookie-policy h1,#ot-sdk-cookie-policy h2,#ot-sdk-cookie-policy h3,#ot-sdk-cookie-policy h4,#ot-sdk-cookie-policy h5,#ot-sdk-cookie-policy h6{margin-top:0;font-weight:600;font-family:inherit}#onetrust-banner-sdk h1,#onetrust-pc-sdk h1,#ot-sdk-cookie-policy h1{font-size:1.5rem;line-height:1.2}#onetrust-banner-sdk h2,#onetrust-pc-sdk h2,#ot-sdk-cookie-policy h2{font-size:1.5rem;line-height:1.25}#onetrust-banner-sdk h3,#onetrust-pc-sdk h3,#ot-sdk-cookie-policy h3{font-size:1.5rem;line-height:1.3}#onetrust-banner-sdk h4,#onetrust-pc-sdk h4,#ot-sdk-cookie-policy h4{font-size:1.5rem;line-height:1.35}#onetrust-banner-sdk h5,#onetrust-pc-sdk h5,#ot-sdk-cookie-policy h5{font-size:1.5rem;line-height:1.5}#onetrust-banner-sdk h6,#onetrust-pc-sdk h6,#ot-sdk-cookie-policy h6{font-size:1.5rem;line-height:1.6}@media(min-width: 550px){#onetrust-banner-sdk h1,#onetrust-pc-sdk h1,#ot-sdk-cookie-policy h1{font-size:1.5rem}#onetrust-banner-sdk h2,#onetrust-pc-sdk h2,#ot-sdk-cookie-policy h2{font-size:1.5rem}#onetrust-banner-sdk h3,#onetrust-pc-sdk h3,#ot-sdk-cookie-policy h3{font-size:1.5rem}#onetrust-banner-sdk h4,#onetrust-pc-sdk h4,#ot-sdk-cookie-policy h4{font-size:1.5rem}#onetrust-banner-sdk h5,#onetrust-pc-sdk h5,#ot-sdk-cookie-policy h5{font-size:1.5rem}#onetrust-banner-sdk h6,#onetrust-pc-sdk h6,#ot-sdk-cookie-policy h6{font-size:1.5rem}}#onetrust-banner-sdk p,#onetrust-pc-sdk p,#ot-sdk-cookie-policy p{margin:0 0 1em 0;font-family:inherit;line-height:normal}#onetrust-banner-sdk a,#onetrust-pc-sdk a,#ot-sdk-cookie-policy a{color:#565656;text-decoration:underline}#onetrust-banner-sdk a:hover,#onetrust-pc-sdk a:hover,#ot-sdk-cookie-policy a:hover{color:#565656;text-decoration:none}#onetrust-banner-sdk .ot-sdk-button,#onetrust-banner-sdk button,#onetrust-pc-sdk .ot-sdk-button,#onetrust-pc-sdk button,#ot-sdk-cookie-policy .ot-sdk-button,#ot-sdk-cookie-policy button{margin-bottom:1rem;font-family:inherit}#onetrust-banner-sdk .ot-sdk-button,#onetrust-banner-sdk button,#onetrust-pc-sdk .ot-sdk-button,#onetrust-pc-sdk button,#ot-sdk-cookie-policy .ot-sdk-button,#ot-sdk-cookie-policy button{display:inline-block;height:38px;padding:0 30px;color:#555;text-align:center;font-size:.9em;font-weight:400;line-height:38px;letter-spacing:.01em;text-decoration:none;white-space:nowrap;background-color:rgba(0,0,0,0);border-radius:2px;border:1px solid #bbb;cursor:pointer;box-sizing:border-box}#onetrust-banner-sdk .ot-sdk-button:hover,#onetrust-banner-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):hover,#onetrust-banner-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):focus,#onetrust-pc-sdk .ot-sdk-button:hover,#onetrust-pc-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):hover,#onetrust-pc-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):focus,#ot-sdk-cookie-policy .ot-sdk-button:hover,#ot-sdk-cookie-policy :not(.ot-leg-btn-container)>button:not(.ot-link-btn):hover,#ot-sdk-cookie-policy :not(.ot-leg-btn-container)>button:not(.ot-link-btn):focus{color:#333;border-color:#888;opacity:.9}#onetrust-banner-sdk .ot-sdk-button:focus,#onetrust-banner-sdk :not(.ot-leg-btn-container)>button:focus,#onetrust-pc-sdk .ot-sdk-button:focus,#onetrust-pc-sdk :not(.ot-leg-btn-container)>button:focus,#ot-sdk-cookie-policy .ot-sdk-button:focus,#ot-sdk-cookie-policy :not(.ot-leg-btn-container)>button:focus{outline:2px solid #000}#onetrust-banner-sdk .ot-sdk-button.ot-sdk-button-primary,#onetrust-banner-sdk button.ot-sdk-button-primary,#onetrust-banner-sdk input[type=submit].ot-sdk-button-primary,#onetrust-banner-sdk input[type=reset].ot-sdk-button-primary,#onetrust-banner-sdk input[type=button].ot-sdk-button-primary,#onetrust-pc-sdk .ot-sdk-button.ot-sdk-button-primary,#onetrust-pc-sdk button.ot-sdk-button-primary,#onetrust-pc-sdk input[type=submit].ot-sdk-button-primary,#onetrust-pc-sdk input[type=reset].ot-sdk-button-primary,#onetrust-pc-sdk input[type=button].ot-sdk-button-primary,#ot-sdk-cookie-policy .ot-sdk-button.ot-sdk-button-primary,#ot-sdk-cookie-policy button.ot-sdk-button-primary,#ot-sdk-cookie-policy input[type=submit].ot-sdk-button-primary,#ot-sdk-cookie-policy input[type=reset].ot-sdk-button-primary,#ot-sdk-cookie-policy input[type=button].ot-sdk-button-primary{color:#fff;background-color:#33c3f0;border-color:#33c3f0}#onetrust-banner-sdk .ot-sdk-button.ot-sdk-button-primary:hover,#onetrust-banner-sdk button.ot-sdk-button-primary:hover,#onetrust-banner-sdk input[type=submit].ot-sdk-button-primary:hover,#onetrust-banner-sdk input[type=reset].ot-sdk-button-primary:hover,#onetrust-banner-sdk input[type=button].ot-sdk-button-primary:hover,#onetrust-banner-sdk .ot-sdk-button.ot-sdk-button-primary:focus,#onetrust-banner-sdk button.ot-sdk-button-primary:focus,#onetrust-banner-sdk input[type=submit].ot-sdk-button-primary:focus,#onetrust-banner-sdk input[type=reset].ot-sdk-button-primary:focus,#onetrust-banner-sdk input[type=button].ot-sdk-button-primary:focus,#onetrust-pc-sdk .ot-sdk-button.ot-sdk-button-primary:hover,#onetrust-pc-sdk button.ot-sdk-button-primary:hover,#onetrust-pc-sdk input[type=submit].ot-sdk-button-primary:hover,#onetrust-pc-sdk input[type=reset].ot-sdk-button-primary:hover,#onetrust-pc-sdk input[type=button].ot-sdk-button-primary:hover,#onetrust-pc-sdk .ot-sdk-button.ot-sdk-button-primary:focus,#onetrust-pc-sdk button.ot-sdk-button-primary:focus,#onetrust-pc-sdk input[type=submit].ot-sdk-button-primary:focus,#onetrust-pc-sdk input[type=reset].ot-sdk-button-primary:focus,#onetrust-pc-sdk input[type=button].ot-sdk-button-primary:focus,#ot-sdk-cookie-policy .ot-sdk-button.ot-sdk-button-primary:hover,#ot-sdk-cookie-policy button.ot-sdk-button-primary:hover,#ot-sdk-cookie-policy input[type=submit].ot-sdk-button-primary:hover,#ot-sdk-cookie-policy input[type=reset].ot-sdk-button-primary:hover,#ot-sdk-cookie-policy input[type=button].ot-sdk-button-primary:hover,#ot-sdk-cookie-policy .ot-sdk-button.ot-sdk-button-primary:focus,#ot-sdk-cookie-policy button.ot-sdk-button-primary:focus,#ot-sdk-cookie-policy input[type=submit].ot-sdk-button-primary:focus,#ot-sdk-cookie-policy input[type=reset].ot-sdk-button-primary:focus,#ot-sdk-cookie-policy input[type=button].ot-sdk-button-primary:focus{color:#fff;background-color:#1eaedb;border-color:#1eaedb}#onetrust-banner-sdk input[type=text],#onetrust-pc-sdk input[type=text],#ot-sdk-cookie-policy input[type=text]{height:38px;padding:6px 10px;background-color:#fff;border:1px solid #d1d1d1;border-radius:4px;box-shadow:none;box-sizing:border-box}#onetrust-banner-sdk input[type=text],#onetrust-pc-sdk input[type=text],#ot-sdk-cookie-policy input[type=text]{-webkit-appearance:none;-moz-appearance:none;appearance:none}#onetrust-banner-sdk input[type=text]:focus,#onetrust-pc-sdk input[type=text]:focus,#ot-sdk-cookie-policy input[type=text]:focus{border:1px solid #000;outline:0}#onetrust-banner-sdk label,#onetrust-pc-sdk label,#ot-sdk-cookie-policy label{display:block;margin-bottom:.5rem;font-weight:600}#onetrust-banner-sdk input[type=checkbox],#onetrust-pc-sdk input[type=checkbox],#ot-sdk-cookie-policy input[type=checkbox]{display:inline}#onetrust-banner-sdk ul,#onetrust-pc-sdk ul,#ot-sdk-cookie-policy ul{list-style:circle inside}#onetrust-banner-sdk ul,#onetrust-pc-sdk ul,#ot-sdk-cookie-policy ul{padding-left:0;margin-top:0}#onetrust-banner-sdk ul ul,#onetrust-pc-sdk ul ul,#ot-sdk-cookie-policy ul ul{margin:1.5rem 0 1.5rem 3rem;font-size:90%}#onetrust-banner-sdk li,#onetrust-pc-sdk li,#ot-sdk-cookie-policy li{margin-bottom:1rem}#onetrust-banner-sdk th,#onetrust-banner-sdk td,#onetrust-pc-sdk th,#onetrust-pc-sdk td,#ot-sdk-cookie-policy th,#ot-sdk-cookie-policy td{padding:12px 15px;text-align:left;border-bottom:1px solid #e1e1e1}#onetrust-banner-sdk button,#onetrust-pc-sdk button,#ot-sdk-cookie-policy button{margin-bottom:1rem;font-family:inherit}#onetrust-banner-sdk .ot-sdk-container:after,#onetrust-banner-sdk .ot-sdk-row:after,#onetrust-pc-sdk .ot-sdk-container:after,#onetrust-pc-sdk .ot-sdk-row:after,#ot-sdk-cookie-policy .ot-sdk-container:after,#ot-sdk-cookie-policy .ot-sdk-row:after{content:"";display:table;clear:both}#onetrust-banner-sdk .ot-sdk-row,#onetrust-pc-sdk .ot-sdk-row,#ot-sdk-cookie-policy .ot-sdk-row{margin:0;max-width:none;display:block}#onetrust-banner-sdk{box-shadow:0 0 18px rgba(0,0,0,.2)}#onetrust-banner-sdk.otFlat{position:fixed;z-index:2147483645;bottom:0;right:0;left:0;background-color:#fff;max-height:90%;overflow-x:hidden;overflow-y:auto}#onetrust-banner-sdk.otFlat.top{top:0px;bottom:auto}#onetrust-banner-sdk.otRelFont{font-size:1rem}#onetrust-banner-sdk>.ot-sdk-container{overflow:hidden}#onetrust-banner-sdk::-webkit-scrollbar{width:11px}#onetrust-banner-sdk::-webkit-scrollbar-thumb{border-radius:10px;background:#c1c1c1}#onetrust-banner-sdk{scrollbar-arrow-color:#c1c1c1;scrollbar-darkshadow-color:#c1c1c1;scrollbar-face-color:#c1c1c1;scrollbar-shadow-color:#c1c1c1}#onetrust-banner-sdk #onetrust-policy{margin:1.25em 0 .625em 2em;overflow:hidden}#onetrust-banner-sdk #onetrust-policy .ot-gv-list-handler{float:left;font-size:.82em;padding:0;margin-bottom:0;border:0;line-height:normal;height:auto;width:auto}#onetrust-banner-sdk #onetrust-policy-title{font-size:1.2em;line-height:1.3;margin-bottom:10px}#onetrust-banner-sdk #onetrust-policy-text{clear:both;text-align:left;font-size:.88em;line-height:1.4}#onetrust-banner-sdk #onetrust-policy-text *{font-size:inherit;line-height:inherit}#onetrust-banner-sdk #onetrust-policy-text a{font-weight:bold;margin-left:5px}#onetrust-banner-sdk #onetrust-policy-title,#onetrust-banner-sdk #onetrust-policy-text{color:dimgray;float:left}#onetrust-banner-sdk #onetrust-button-group-parent{min-height:1px;text-align:center}#onetrust-banner-sdk #onetrust-button-group{display:inline-block}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{background-color:#68b631;color:#fff;border-color:#68b631;margin-right:1em;min-width:125px;height:auto;white-space:normal;word-break:break-word;word-wrap:break-word;padding:12px 10px;line-height:1.2;font-size:.813em;font-weight:600}#onetrust-banner-sdk #onetrust-pc-btn-handler.cookie-setting-link{background-color:#fff;border:none;color:#68b631;text-decoration:underline;padding-left:0;padding-right:0}#onetrust-banner-sdk .onetrust-close-btn-ui{width:44px;height:44px;background-size:12px;border:none;position:relative;margin:auto;padding:0}#onetrust-banner-sdk .banner_logo{display:none}#onetrust-banner-sdk.ot-bnr-w-logo .ot-bnr-logo{position:absolute;top:50%;transform:translateY(-50%);left:0px}#onetrust-banner-sdk.ot-bnr-w-logo #onetrust-policy{margin-left:65px}#onetrust-banner-sdk .ot-b-addl-desc{clear:both;float:left;display:block}#onetrust-banner-sdk #banner-options{float:left;display:table;margin-right:0;margin-left:1em;width:calc(100% - 1em)}#onetrust-banner-sdk .banner-option-input{cursor:pointer;width:auto;height:auto;border:none;padding:0;padding-right:3px;margin:0 0 10px;font-size:.82em;line-height:1.4}#onetrust-banner-sdk .banner-option-input *{pointer-events:none;font-size:inherit;line-height:inherit}#onetrust-banner-sdk .banner-option-input[aria-expanded=true]~.banner-option-details{display:block;height:auto}#onetrust-banner-sdk .banner-option-input[aria-expanded=true] .ot-arrow-container{transform:rotate(90deg)}#onetrust-banner-sdk .banner-option{margin-bottom:12px;margin-left:0;border:none;float:left;padding:0}#onetrust-banner-sdk .banner-option:first-child{padding-left:2px}#onetrust-banner-sdk .banner-option:not(:first-child){padding:0;border:none}#onetrust-banner-sdk .banner-option-header{cursor:pointer;display:inline-block}#onetrust-banner-sdk .banner-option-header :first-child{color:dimgray;font-weight:bold;float:left}#onetrust-banner-sdk .banner-option-header .ot-arrow-container{display:inline-block;border-top:6px solid rgba(0,0,0,0);border-bottom:6px solid rgba(0,0,0,0);border-left:6px solid dimgray;margin-left:10px;vertical-align:middle}#onetrust-banner-sdk .banner-option-details{display:none;font-size:.83em;line-height:1.5;padding:10px 0px 5px 10px;margin-right:10px;height:0px}#onetrust-banner-sdk .banner-option-details *{font-size:inherit;line-height:inherit;color:dimgray}#onetrust-banner-sdk .ot-arrow-container,#onetrust-banner-sdk .banner-option-details{transition:all 300ms ease-in 0s;-webkit-transition:all 300ms ease-in 0s;-moz-transition:all 300ms ease-in 0s;-o-transition:all 300ms ease-in 0s}#onetrust-banner-sdk .ot-dpd-container{float:left}#onetrust-banner-sdk .ot-dpd-title{margin-bottom:10px}#onetrust-banner-sdk .ot-dpd-title,#onetrust-banner-sdk .ot-dpd-desc{font-size:.88em;line-height:1.4;color:dimgray}#onetrust-banner-sdk .ot-dpd-title *,#onetrust-banner-sdk .ot-dpd-desc *{font-size:inherit;line-height:inherit}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-text *{margin-bottom:0}#onetrust-banner-sdk.ot-iab-2 .onetrust-vendors-list-handler{display:block;margin-left:0;margin-top:5px;clear:both;margin-bottom:0;padding:0;border:0;height:auto;width:auto}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group button{display:block}#onetrust-banner-sdk.ot-close-btn-link{padding-top:25px}#onetrust-banner-sdk.ot-close-btn-link #onetrust-close-btn-container{top:15px;transform:none;right:15px}#onetrust-banner-sdk.ot-close-btn-link #onetrust-close-btn-container button{padding:0;white-space:pre-wrap;border:none;height:auto;line-height:1.5;text-decoration:underline;font-size:.69em}#onetrust-banner-sdk #onetrust-policy-text,#onetrust-banner-sdk .ot-dpd-desc,#onetrust-banner-sdk .ot-b-addl-desc{font-size:.813em;line-height:1.5}#onetrust-banner-sdk .ot-dpd-desc{margin-bottom:10px}#onetrust-banner-sdk .ot-dpd-desc>.ot-b-addl-desc{margin-top:10px;margin-bottom:10px;font-size:1em}@media only screen and (max-width: 425px){#onetrust-banner-sdk #onetrust-close-btn-container{position:absolute;top:6px;right:2px}#onetrust-banner-sdk #onetrust-policy{margin-left:0;margin-top:3em}#onetrust-banner-sdk #onetrust-button-group{display:block}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{width:100%}#onetrust-banner-sdk .onetrust-close-btn-ui{top:auto;transform:none}#onetrust-banner-sdk #onetrust-policy-title{display:inline;float:none}#onetrust-banner-sdk #banner-options{margin:0;padding:0;width:100%}}@media only screen and (min-width: 426px)and (max-width: 896px){#onetrust-banner-sdk #onetrust-close-btn-container{position:absolute;top:0;right:0}#onetrust-banner-sdk #onetrust-policy{margin-left:1em;margin-right:1em}#onetrust-banner-sdk .onetrust-close-btn-ui{top:10px;right:10px}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{width:95%}#onetrust-banner-sdk.ot-iab-2 #onetrust-group-container{width:100%}#onetrust-banner-sdk.ot-bnr-w-logo #onetrust-button-group-parent{padding-left:50px}#onetrust-banner-sdk #onetrust-button-group-parent{width:100%;position:relative;margin-left:0}#onetrust-banner-sdk #onetrust-button-group button{display:inline-block}#onetrust-banner-sdk #onetrust-button-group{margin-right:0;text-align:center}#onetrust-banner-sdk .has-reject-all-button #onetrust-pc-btn-handler{float:left}#onetrust-banner-sdk .has-reject-all-button #onetrust-reject-all-handler,#onetrust-banner-sdk .has-reject-all-button #onetrust-accept-btn-handler{float:right}#onetrust-banner-sdk .has-reject-all-button #onetrust-button-group{width:calc(100% - 2em);margin-right:0}#onetrust-banner-sdk .has-reject-all-button #onetrust-pc-btn-handler.cookie-setting-link{padding-left:0px;text-align:left}#onetrust-banner-sdk.ot-buttons-fw .ot-sdk-three button{width:100%;text-align:center}#onetrust-banner-sdk.ot-buttons-fw #onetrust-button-group-parent button{float:none}#onetrust-banner-sdk.ot-buttons-fw #onetrust-pc-btn-handler.cookie-setting-link{text-align:center}}@media only screen and (min-width: 550px){#onetrust-banner-sdk .banner-option:not(:first-child){border-left:1px solid #d8d8d8;padding-left:25px}}@media only screen and (min-width: 425px)and (max-width: 550px){#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group,#onetrust-banner-sdk.ot-iab-2 #onetrust-policy,#onetrust-banner-sdk.ot-iab-2 .banner-option{width:100%}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-accept-btn-handler,#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-reject-all-handler,#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-pc-btn-handler{width:100%}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-accept-btn-handler,#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-reject-all-handler{float:left}}@media only screen and (min-width: 769px){#onetrust-banner-sdk #onetrust-button-group{margin-right:30%}#onetrust-banner-sdk #banner-options{margin-left:2em;margin-right:5em;margin-bottom:1.25em;width:calc(100% - 7em)}}@media only screen and (min-width: 897px)and (max-width: 1023px){#onetrust-banner-sdk.vertical-align-content #onetrust-button-group-parent{position:absolute;top:50%;left:75%;transform:translateY(-50%)}#onetrust-banner-sdk #onetrust-close-btn-container{top:50%;margin:auto;transform:translate(-50%, -50%);position:absolute;padding:0;right:0}#onetrust-banner-sdk #onetrust-close-btn-container button{position:relative;margin:0;right:-22px;top:2px}}@media only screen and (min-width: 1024px){#onetrust-banner-sdk #onetrust-close-btn-container{top:50%;margin:auto;transform:translate(-50%, -50%);position:absolute;right:0}#onetrust-banner-sdk #onetrust-close-btn-container button{right:-12px}#onetrust-banner-sdk #onetrust-policy{margin-left:2em}#onetrust-banner-sdk.vertical-align-content #onetrust-button-group-parent{position:absolute;top:50%;left:60%;transform:translateY(-50%)}#onetrust-banner-sdk .ot-optout-signal{width:50%}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-title{width:50%}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-text,#onetrust-banner-sdk.ot-iab-2 :not(.ot-dpd-desc)>.ot-b-addl-desc{margin-bottom:1em;width:50%;border-right:1px solid #d8d8d8;padding-right:1rem}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-text{margin-bottom:0;padding-bottom:1em}#onetrust-banner-sdk.ot-iab-2 :not(.ot-dpd-desc)>.ot-b-addl-desc{margin-bottom:0;padding-bottom:1em}#onetrust-banner-sdk.ot-iab-2 .ot-dpd-container{width:45%;padding-left:1rem;display:inline-block;float:none}#onetrust-banner-sdk.ot-iab-2 .ot-dpd-title{line-height:1.7}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group-parent{left:auto;right:4%;margin-left:0}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group button{display:block}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{margin:auto;width:30%}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{width:60%}#onetrust-banner-sdk #onetrust-button-group{margin-right:auto}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{margin-top:1em}}@media only screen and (min-width: 890px){#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group-parent{padding-left:3%;padding-right:4%;margin-left:0}#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group{margin-right:0;margin-top:1.25em;width:100%}#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button{width:100%;margin-bottom:5px;margin-top:5px}#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button:last-of-type{margin-bottom:20px}}@media only screen and (min-width: 1280px){#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{width:55%}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{width:44%;padding-left:2%;padding-right:2%}#onetrust-banner-sdk:not(.ot-iab-2).vertical-align-content #onetrust-button-group-parent{position:absolute;left:55%}}
        #onetrust-consent-sdk #onetrust-banner-sdk {background-color: #FFF;}
            #onetrust-consent-sdk #onetrust-policy-title,
                    #onetrust-consent-sdk #onetrust-policy-text,
                    #onetrust-consent-sdk .ot-b-addl-desc,
                    #onetrust-consent-sdk .ot-dpd-desc,
                    #onetrust-consent-sdk .ot-dpd-title,
                    #onetrust-consent-sdk #onetrust-policy-text *:not(.onetrust-vendors-list-handler),
                    #onetrust-consent-sdk .ot-dpd-desc *:not(.onetrust-vendors-list-handler),
                    #onetrust-consent-sdk #onetrust-banner-sdk #banner-options *,
                    #onetrust-banner-sdk .ot-cat-header,
                    #onetrust-banner-sdk .ot-optout-signal
                    {
                        color: #2E2E2E;
                    }
            #onetrust-consent-sdk #onetrust-banner-sdk .banner-option-details {
                    background-color: #E9E9E9;}
             #onetrust-consent-sdk #onetrust-banner-sdk a[href],
                    #onetrust-consent-sdk #onetrust-banner-sdk a[href] font,
                    #onetrust-consent-sdk #onetrust-banner-sdk .ot-link-btn
                        {
                            color: #007398;
                        }#onetrust-consent-sdk #onetrust-accept-btn-handler,
                         #onetrust-banner-sdk #onetrust-reject-all-handler {
                            background-color: #007398;border-color: #007398;
                color: #FFF;
            }
            #onetrust-consent-sdk #onetrust-banner-sdk *:focus,
            #onetrust-consent-sdk #onetrust-banner-sdk:focus {
               outline-color: #000000;
               outline-width: 1px;
            }
            #onetrust-consent-sdk #onetrust-pc-btn-handler,
            #onetrust-consent-sdk #onetrust-pc-btn-handler.cookie-setting-link {
                color: #6CC04A; border-color: #6CC04A;
                background-color:
                #FFF;
            }/*! v6.10.0 2021-01-18 */
div#onetrust-consent-sdk #onetrust-banner-sdk{border-top:2px solid #d0d3d4!important;box-shadow:none;padding:24px}div#onetrust-consent-sdk button{border-radius:0!important;box-shadow:none!important;box-sizing:border-box!important;font-size:20px!important;font-weight:400!important;letter-spacing:0!important;max-width:none!important;white-space:nowrap!important}div#onetrust-consent-sdk button:not(.ot-link-btn){background-color:#007dbc!important;border:2px solid #007dbc!important;color:#fff!important;height:48px!important;padding:0 1em!important;width:auto!important}div#onetrust-consent-sdk button:hover{background-color:#fff!important;border-color:#949494!important;color:#2e2e2e!important}div#onetrust-consent-sdk button.ot-link-btn{color:#007dbc!important;font-size:16px!important;text-decoration:underline}div#onetrust-consent-sdk button.ot-link-btn:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-consent-sdk a,div#onetrust-pc-sdk a{color:#007dbc!important;text-decoration:underline!important}div#onetrust-consent-sdk a,div#onetrust-consent-sdk button,div#onetrust-consent-sdk p:hover{opacity:1!important}div#onetrust-consent-sdk a:focus,div#onetrust-consent-sdk button:focus,div#onetrust-consent-sdk input:focus{outline:2px solid #949494!important;outline-offset:1px!important}div#onetrust-banner-sdk .ot-sdk-container{padding:0;width:auto}div#onetrust-banner-sdk .ot-sdk-row{align-items:flex-start;box-sizing:border-box;display:flex;flex-direction:column;justify-content:space-between;margin:auto;max-width:1152px}div#onetrust-banner-sdk .ot-sdk-row:after{display:none}div#onetrust-banner-sdk #onetrust-group-container,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-group-container,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{flex-grow:1;width:auto}div#onetrust-banner-sdk #onetrust-policy,div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy{margin:0;overflow:visible}div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy-text,div#onetrust-consent-sdk #onetrust-policy-text{font-size:16px;line-height:24px;max-width:44em;margin:0}div#onetrust-consent-sdk #onetrust-policy-text a[href]{font-weight:400;margin-left:8px}div#onetrust-banner-sdk #onetrust-button-group-parent{flex:0 0 auto;margin:32px 0 0;width:100%}div#onetrust-banner-sdk #onetrust-button-group{display:flex;flex-direction:row;flex-wrap:wrap;justify-content:flex-end;margin:-8px}div#onetrust-banner-sdk .banner-actions-container{display:flex;flex:1 0 auto}div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button:last-of-type,div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto;margin:8px;width:auto}div#onetrust-consent-sdk #onetrust-pc-btn-handler{background-color:#fff!important;color:inherit!important}div#onetrust-banner-sdk #onetrust-close-btn-container{display:none}@media only screen and (min-width:556px){div#onetrust-consent-sdk #onetrust-banner-sdk{padding:40px}div#onetrust-banner-sdk #onetrust-policy{margin:0 40px 0 0}div#onetrust-banner-sdk .ot-sdk-row{align-items:center;flex-direction:row}div#onetrust-banner-sdk #onetrust-button-group-parent,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-button-group-parent,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{margin:0;padding:0;width:auto}div#onetrust-banner-sdk #onetrust-button-group,div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group{align-items:stretch;flex-direction:column-reverse;margin:0}div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto}}@media only screen and (min-width:768px){div#onetrust-banner-sdk #onetrust-policy{margin:0 48px 0 0}div#onetrust-consent-sdk #onetrust-banner-sdk{padding:48px}}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-size:16px;line-height:24px}div#onetrust-consent-sdk #onetrust-pc-sdk p,div#onetrust-pc-sdk #ot-pc-desc,div#onetrust-pc-sdk .category-host-list-handler,div#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{font-size:16px;font-weight:400;line-height:24px}div#onetrust-consent-sdk a:hover,div#onetrust-pc-sdk a:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-pc-sdk{border-radius:0;bottom:0;height:auto;left:0;margin:auto;max-width:100%;overflow:hidden;right:0;top:0;width:512px;max-height:800px}div#onetrust-pc-sdk .ot-pc-header{display:none}div#onetrust-pc-sdk #ot-pc-content{overscroll-behavior:contain;padding:0 12px 0 24px;margin:16px 4px 0 0;top:0;right:16px;left:0;width:auto}div#onetrust-pc-sdk #ot-category-title,div#onetrust-pc-sdk #ot-pc-title{font-size:24px;font-weight:400;line-height:32px;margin:0 0 16px}div#onetrust-pc-sdk #ot-pc-desc{padding:0}div#onetrust-pc-sdk #ot-pc-desc a{display:inline}div#onetrust-pc-sdk #accept-recommended-btn-handler{display:none!important}div#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:2px solid #007dbc;outline-offset:-1px;transition:none}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item{border-width:0 0 2px}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item:first-of-type{border-width:2px 0}div#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{padding:8px 0;width:100%}div#onetrust-pc-sdk .ot-plus-minus{transform:translateY(2px)}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item>button{background:0 0!important;border:0!important;height:44px!important;max-width:none!important;width:calc(100% - 48px)!important}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-weight:700}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr{padding:0}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item .ot-acc-grpdesc{padding:0;width:100%}div#onetrust-pc-sdk .ot-acc-grpcntr .ot-subgrp-cntr{border:0;padding:0}div#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp{margin:0}div#onetrust-pc-sdk .ot-always-active-group .ot-cat-header{width:calc(100% - 160px)}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{width:calc(100% - 88px)}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active{color:inherit;font-size:12px;font-weight:400;line-height:1.5;padding-right:48px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:before{border-radius:12px;position:absolute;right:0;top:0;content:'';background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:after{border-radius:50%;position:absolute;right:5px;top:4px;content:'';background-color:#007dbc;height:12px;width:12px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active,div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl{right:2px}div#onetrust-pc-sdk .ot-switch{display:block;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob,div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob{background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob:before{background-color:#737373;height:8px;left:4px;top:4px;width:8px}div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob:before{background-color:#007dbc;height:12px;left:0;top:2px;width:12px}div#onetrust-pc-sdk .ot-tgl input:focus+.ot-switch .ot-switch-nob{box-shadow:0 0;outline:2px solid #007dbc!important;outline-offset:1px;transition:none}div#onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-grpcntr.ot-acc-txt{background-color:transparent;padding-left:3px}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr,div#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr{overflow:visible;width:100%}div#onetrust-pc-sdk .ot-pc-footer{border-top:0 solid}div#onetrust-pc-sdk .ot-btn-container{padding-top:24px;text-align:center}div#onetrust-pc-sdk .ot-pc-footer button{margin:8px 0;background-color:#fff}div#onetrust-pc-sdk .ot-pc-footer-logo{background-color:#fff}div#onetrust-pc-sdk #ot-lst-title span{font-size:24px;font-weight:400;line-height:32px}div#onetrust-pc-sdk #ot-host-lst .ot-host-desc,div#onetrust-pc-sdk #ot-host-lst .ot-host-expand,div#onetrust-pc-sdk #ot-host-lst .ot-host-name,div#onetrust-pc-sdk #ot-host-lst .ot-host-name a,div#onetrust-pc-sdk .back-btn-handler,div#onetrust-pc-sdk .ot-host-opt li>div div{font-size:16px;font-weight:400;line-height:24px}div#onetrust-pc-sdk #ot-pc-lst{top:0}div#onetrust-pc-sdk .back-btn-handler{text-decoration:none!important}div#onetrust-pc-sdk #filter-btn-handler:hover svg{filter:invert(1)}div#onetrust-pc-sdk .back-btn-handler svg{width:16px;height:16px}div#onetrust-pc-sdk .ot-host-item>button{background:0 0!important;border:0!important;height:66px!important;max-width:none!important;width:calc(100% - 5px)!important;transform:translate(2px,2px)}div#onetrust-pc-sdk .ot-host-item{border-bottom:2px solid #b9b9b9;padding:0}div#onetrust-pc-sdk .ot-host-item .ot-acc-hdr{margin:0 0 -6px;padding:8px 0}div#onetrust-pc-sdk ul li:first-child{border-top:2px solid #b9b9b9}div#onetrust-pc-sdk .ot-host-item .ot-plus-minus{margin:0 8px 0 0}div#onetrust-pc-sdk .ot-search-cntr{width:calc(100% - 48px)}div#onetrust-pc-sdk .ot-host-opt .ot-host-info{background-color:transparent}div#onetrust-pc-sdk .ot-host-opt li>div div{padding:0}div#onetrust-pc-sdk #vendor-search-handler{border-radius:0;border-color:#939393;border-style:solid;border-width:2px 0 2px 2px;font-size:20px;height:48px;margin:0}div#onetrust-pc-sdk .ot-lst-subhdr svg{right:0;top:8px}div#onetrust-pc-sdk .ot-fltr-cntr{box-sizing:border-box;right:0;width:48px}div#onetrust-pc-sdk #filter-btn-handler{width:48px!important;padding:8px!important}div#onetrust-pc-sdk #ot-fltr-modal{max-height:400px}
#onetrust-pc-sdk.otPcCenter{overflow:hidden;position:fixed;margin:0 auto;top:5%;right:0;left:0;width:40%;max-width:575px;min-width:575px;border-radius:2.5px;z-index:2147483647;background-color:#fff;-webkit-box-shadow:0px 2px 10px -3px #999;-moz-box-shadow:0px 2px 10px -3px #999;box-shadow:0px 2px 10px -3px #999}#onetrust-pc-sdk.otPcCenter[dir=rtl]{right:0;left:0}#onetrust-pc-sdk.otRelFont{font-size:1rem}#onetrust-pc-sdk .ot-optout-signal{margin-top:.625rem}#onetrust-pc-sdk #ot-addtl-venlst .ot-arw-cntr,#onetrust-pc-sdk #ot-addtl-venlst .ot-plus-minus,#onetrust-pc-sdk .ot-hide-tgl{visibility:hidden}#onetrust-pc-sdk #ot-addtl-venlst .ot-arw-cntr *,#onetrust-pc-sdk #ot-addtl-venlst .ot-plus-minus *,#onetrust-pc-sdk .ot-hide-tgl *{visibility:hidden}#onetrust-pc-sdk #ot-gn-venlst .ot-ven-item .ot-acc-hdr{min-height:40px}#onetrust-pc-sdk .ot-pc-header{height:39px;padding:10px 0 10px 30px;border-bottom:1px solid #e9e9e9}#onetrust-pc-sdk #ot-pc-title,#onetrust-pc-sdk #ot-category-title,#onetrust-pc-sdk .ot-cat-header,#onetrust-pc-sdk #ot-lst-title,#onetrust-pc-sdk .ot-ven-hdr .ot-ven-name,#onetrust-pc-sdk .ot-always-active{font-weight:bold;color:dimgray}#onetrust-pc-sdk .ot-always-active-group .ot-cat-header{width:55%;font-weight:700}#onetrust-pc-sdk .ot-cat-item p{clear:both;float:left;margin-top:10px;margin-bottom:5px;line-height:1.5;font-size:.812em;color:dimgray}#onetrust-pc-sdk .ot-close-icon{height:44px;width:44px;background-size:10px}#onetrust-pc-sdk #ot-pc-title{float:left;font-size:1em;line-height:1.5;margin-bottom:10px;margin-top:10px;width:100%}#onetrust-pc-sdk #accept-recommended-btn-handler{margin-right:10px;margin-bottom:25px;outline-offset:-1px}#onetrust-pc-sdk #ot-pc-desc{clear:both;width:100%;font-size:.812em;line-height:1.5;margin-bottom:25px}#onetrust-pc-sdk #ot-pc-desc *{font-size:inherit;line-height:inherit}#onetrust-pc-sdk #ot-pc-desc ul li{padding:10px 0px}#onetrust-pc-sdk a{color:#656565;cursor:pointer}#onetrust-pc-sdk a:hover{color:#3860be}#onetrust-pc-sdk label{margin-bottom:0}#onetrust-pc-sdk #vdr-lst-dsc{font-size:.812em;line-height:1.5;padding:10px 15px 5px 15px}#onetrust-pc-sdk button{max-width:394px;padding:12px 30px;line-height:1;word-break:break-word;word-wrap:break-word;white-space:normal;font-weight:bold;height:auto}#onetrust-pc-sdk .ot-link-btn{padding:0;margin-bottom:0;border:0;font-weight:normal;line-height:normal;width:auto;height:auto}#onetrust-pc-sdk #ot-pc-content{position:absolute;overflow-y:scroll;padding-left:2px;padding-right:30px;top:60px;bottom:110px;margin:1px 3px 0 30px;width:calc(100% - 63px)}#onetrust-pc-sdk .ot-vs-list .ot-always-active,#onetrust-pc-sdk .ot-cat-grp .ot-always-active{float:right;clear:none;color:#3860be;margin:0;font-size:.813em;line-height:1.3}#onetrust-pc-sdk .ot-pc-scrollbar::-webkit-scrollbar-track{margin-right:20px}#onetrust-pc-sdk .ot-pc-scrollbar::-webkit-scrollbar{width:11px}#onetrust-pc-sdk .ot-pc-scrollbar::-webkit-scrollbar-thumb{border-radius:10px;background:#d8d8d8}#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:#000 1px solid}#onetrust-pc-sdk .ot-pc-scrollbar{scrollbar-arrow-color:#d8d8d8;scrollbar-darkshadow-color:#d8d8d8;scrollbar-face-color:#d8d8d8;scrollbar-shadow-color:#d8d8d8}#onetrust-pc-sdk .save-preference-btn-handler{margin-right:20px}#onetrust-pc-sdk .ot-pc-refuse-all-handler{margin-right:10px}#onetrust-pc-sdk #ot-pc-desc .privacy-notice-link{margin-left:0;margin-right:8px}#onetrust-pc-sdk #ot-pc-desc .ot-imprint-handler{margin-left:0;margin-right:8px}#onetrust-pc-sdk .ot-subgrp-cntr{display:inline-block;clear:both;width:100%;padding-top:15px}#onetrust-pc-sdk .ot-switch+.ot-subgrp-cntr{padding-top:10px}#onetrust-pc-sdk ul.ot-subgrps{margin:0;font-size:initial}#onetrust-pc-sdk ul.ot-subgrps li p,#onetrust-pc-sdk ul.ot-subgrps li h5{font-size:.813em;line-height:1.4;color:dimgray}#onetrust-pc-sdk ul.ot-subgrps .ot-switch{min-height:auto}#onetrust-pc-sdk ul.ot-subgrps .ot-switch-nob{top:0}#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr{display:inline-block;width:100%}#onetrust-pc-sdk ul.ot-subgrps .ot-acc-txt{margin:0}#onetrust-pc-sdk ul.ot-subgrps li{padding:0;border:none}#onetrust-pc-sdk ul.ot-subgrps li h5{position:relative;top:5px;font-weight:bold;margin-bottom:0;float:left}#onetrust-pc-sdk li.ot-subgrp{margin-left:20px;overflow:auto}#onetrust-pc-sdk li.ot-subgrp>h5{width:calc(100% - 100px)}#onetrust-pc-sdk .ot-cat-item p>ul,#onetrust-pc-sdk li.ot-subgrp p>ul{margin:0px;list-style:disc;margin-left:15px;font-size:inherit}#onetrust-pc-sdk .ot-cat-item p>ul li,#onetrust-pc-sdk li.ot-subgrp p>ul li{font-size:inherit;padding-top:10px;padding-left:0px;padding-right:0px;border:none}#onetrust-pc-sdk .ot-cat-item p>ul li:last-child,#onetrust-pc-sdk li.ot-subgrp p>ul li:last-child{padding-bottom:10px}#onetrust-pc-sdk .ot-pc-logo{height:40px;width:120px}#onetrust-pc-sdk .ot-pc-footer{position:absolute;bottom:0px;width:100%;max-height:160px;border-top:1px solid #d8d8d8}#onetrust-pc-sdk.ot-ftr-stacked .ot-pc-refuse-all-handler{margin-bottom:0px}#onetrust-pc-sdk.ot-ftr-stacked #ot-pc-content{bottom:160px}#onetrust-pc-sdk.ot-ftr-stacked .ot-pc-footer button{width:100%;max-width:none}#onetrust-pc-sdk.ot-ftr-stacked .ot-btn-container{margin:0 30px;width:calc(100% - 60px);padding-right:0}#onetrust-pc-sdk .ot-pc-footer-logo{height:30px;width:100%;text-align:right;background:#f4f4f4}#onetrust-pc-sdk .ot-pc-footer-logo a{display:inline-block;margin-top:5px;margin-right:10px}#onetrust-pc-sdk[dir=rtl] .ot-pc-footer-logo{direction:rtl}#onetrust-pc-sdk[dir=rtl] .ot-pc-footer-logo a{margin-right:25px}#onetrust-pc-sdk .ot-tgl{float:right;position:relative;z-index:1}#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob{background-color:#468254;border:1px solid #fff}#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob:before{-webkit-transform:translateX(20px);-ms-transform:translateX(20px);transform:translateX(20px);background-color:#fff;border-color:#fff}#onetrust-pc-sdk .ot-tgl input:focus+.ot-switch{outline:#000 solid 1px}#onetrust-pc-sdk .ot-switch{position:relative;display:inline-block;width:45px;height:25px}#onetrust-pc-sdk .ot-switch-nob{position:absolute;cursor:pointer;top:0;left:0;right:0;bottom:0;background-color:#767676;border:1px solid #ddd;transition:all .2s ease-in 0s;-moz-transition:all .2s ease-in 0s;-o-transition:all .2s ease-in 0s;-webkit-transition:all .2s ease-in 0s;border-radius:20px}#onetrust-pc-sdk .ot-switch-nob:before{position:absolute;content:"";height:18px;width:18px;bottom:3px;left:3px;background-color:#fff;-webkit-transition:.4s;transition:.4s;border-radius:20px}#onetrust-pc-sdk .ot-chkbox input:checked~label::before{background-color:#3860be}#onetrust-pc-sdk .ot-chkbox input+label::after{content:none;color:#fff}#onetrust-pc-sdk .ot-chkbox input:checked+label::after{content:""}#onetrust-pc-sdk .ot-chkbox input:focus+label::before{outline-style:solid;outline-width:2px;outline-style:auto}#onetrust-pc-sdk .ot-chkbox input[aria-checked=mixed]~label::before{background-color:#3860be}#onetrust-pc-sdk .ot-chkbox input[aria-checked=mixed]+label::after{content:""}#onetrust-pc-sdk .ot-chkbox label{position:relative;display:inline-block;padding-left:30px;cursor:pointer;font-weight:500}#onetrust-pc-sdk .ot-chkbox label::before,#onetrust-pc-sdk .ot-chkbox label::after{position:absolute;content:"";display:inline-block;border-radius:3px}#onetrust-pc-sdk .ot-chkbox label::before{height:18px;width:18px;border:1px solid #3860be;left:0px;top:auto}#onetrust-pc-sdk .ot-chkbox label::after{height:5px;width:9px;border-left:3px solid;border-bottom:3px solid;transform:rotate(-45deg);-o-transform:rotate(-45deg);-ms-transform:rotate(-45deg);-webkit-transform:rotate(-45deg);left:4px;top:5px}#onetrust-pc-sdk .ot-label-txt{display:none}#onetrust-pc-sdk .ot-chkbox input,#onetrust-pc-sdk .ot-tgl input{position:absolute;opacity:0;width:0;height:0}#onetrust-pc-sdk .ot-arw-cntr{float:right;position:relative;pointer-events:none}#onetrust-pc-sdk .ot-arw-cntr .ot-arw{width:16px;height:16px;margin-left:5px;color:dimgray;display:inline-block;vertical-align:middle;-webkit-transition:all 150ms ease-in 0s;-moz-transition:all 150ms ease-in 0s;-o-transition:all 150ms ease-in 0s;transition:all 150ms ease-in 0s}#onetrust-pc-sdk input:checked~.ot-acc-hdr .ot-arw,#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-arw-cntr svg{transform:rotate(90deg);-o-transform:rotate(90deg);-ms-transform:rotate(90deg);-webkit-transform:rotate(90deg)}#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:#000 1px solid}#onetrust-pc-sdk .ot-tgl-cntr,#onetrust-pc-sdk .ot-arw-cntr{display:inline-block}#onetrust-pc-sdk .ot-tgl-cntr{width:45px;float:right;margin-top:2px}#onetrust-pc-sdk #ot-lst-cnt .ot-tgl-cntr{margin-top:10px}#onetrust-pc-sdk .ot-always-active-subgroup{width:auto;padding-left:0px !important;top:3px;position:relative}#onetrust-pc-sdk .ot-label-status{font-size:.75em;display:none;font-size:.75em;position:relative;top:2px;padding-right:5px;float:left}#onetrust-pc-sdk .ot-arw-cntr{margin-top:-1px}#onetrust-pc-sdk .ot-arw-cntr svg{-webkit-transition:all 300ms ease-in 0s;-moz-transition:all 300ms ease-in 0s;-o-transition:all 300ms ease-in 0s;transition:all 300ms ease-in 0s;height:10px;width:10px}#onetrust-pc-sdk input:checked~.ot-acc-hdr .ot-arw{transform:rotate(90deg);-o-transform:rotate(90deg);-ms-transform:rotate(90deg);-webkit-transform:rotate(90deg)}#onetrust-pc-sdk .ot-arw{width:10px;margin-left:15px;transition:all 300ms ease-in 0s;-webkit-transition:all 300ms ease-in 0s;-moz-transition:all 300ms ease-in 0s;-o-transition:all 300ms ease-in 0s}#onetrust-pc-sdk .ot-vlst-cntr{margin-bottom:0}#onetrust-pc-sdk .ot-hlst-cntr{margin-top:5px;display:inline-block;width:100%}#onetrust-pc-sdk .category-vendors-list-handler,#onetrust-pc-sdk .category-vendors-list-handler+a,#onetrust-pc-sdk .category-host-list-handler{clear:both;color:#3860be;margin-left:0;font-size:.813em;text-decoration:none;float:left;overflow:hidden}#onetrust-pc-sdk .category-vendors-list-handler:hover,#onetrust-pc-sdk .category-vendors-list-handler+a:hover,#onetrust-pc-sdk .category-host-list-handler:hover{text-decoration-line:underline}#onetrust-pc-sdk .category-vendors-list-handler+a{clear:none}#onetrust-pc-sdk .ot-vlst-cntr .ot-ext-lnk,#onetrust-pc-sdk .ot-ven-hdr .ot-ext-lnk{display:inline-block;height:13px;width:13px;background-repeat:no-repeat;margin-left:1px;margin-top:6px;cursor:pointer}#onetrust-pc-sdk .ot-ven-hdr .ot-ext-lnk{margin-bottom:-1px}#onetrust-pc-sdk .back-btn-handler{font-size:1em;text-decoration:none}#onetrust-pc-sdk .back-btn-handler:hover{opacity:.6}#onetrust-pc-sdk #ot-lst-title h3{display:inline-block;word-break:break-word;word-wrap:break-word;margin-bottom:0;color:#656565;font-size:1em;font-weight:bold;margin-left:15px}#onetrust-pc-sdk #ot-lst-title{margin:10px 0 10px 0px;font-size:1em;text-align:left}#onetrust-pc-sdk #ot-pc-hdr{margin:0 0 0 30px;height:auto;width:auto}#onetrust-pc-sdk #ot-pc-hdr input::placeholder{color:#707070;font-style:italic}#onetrust-pc-sdk #vendor-search-handler{height:31px;width:100%;border-radius:50px;font-size:.8em;padding-right:35px;padding-left:15px;float:left;margin-left:15px}#onetrust-pc-sdk .ot-ven-name{display:block;width:auto;padding-right:5px}#onetrust-pc-sdk #ot-lst-cnt{overflow-y:auto;margin-left:20px;margin-right:7px;width:calc(100% - 27px);max-height:calc(100% - 80px);height:100%;transform:translate3d(0, 0, 0)}#onetrust-pc-sdk #ot-pc-lst{width:100%;bottom:100px;position:absolute;top:60px}#onetrust-pc-sdk #ot-pc-lst:not(.ot-enbl-chr) .ot-tgl-cntr .ot-arw-cntr,#onetrust-pc-sdk #ot-pc-lst:not(.ot-enbl-chr) .ot-tgl-cntr .ot-arw-cntr *{visibility:hidden}#onetrust-pc-sdk #ot-pc-lst .ot-tgl-cntr{right:12px;position:absolute}#onetrust-pc-sdk #ot-pc-lst .ot-arw-cntr{float:right;position:relative}#onetrust-pc-sdk #ot-pc-lst .ot-arw{margin-left:10px}#onetrust-pc-sdk #ot-pc-lst .ot-acc-hdr{overflow:hidden;cursor:pointer}#onetrust-pc-sdk .ot-vlst-cntr{overflow:hidden}#onetrust-pc-sdk #ot-sel-blk{overflow:hidden;width:100%;position:sticky;position:-webkit-sticky;top:0;z-index:3}#onetrust-pc-sdk #ot-back-arw{height:12px;width:12px}#onetrust-pc-sdk .ot-lst-subhdr{width:100%;display:inline-block}#onetrust-pc-sdk .ot-search-cntr{float:left;width:78%;position:relative}#onetrust-pc-sdk .ot-search-cntr>svg{width:30px;height:30px;position:absolute;float:left;right:-15px}#onetrust-pc-sdk .ot-fltr-cntr{float:right;right:50px;position:relative}#onetrust-pc-sdk #filter-btn-handler{background-color:#3860be;border-radius:17px;display:inline-block;position:relative;width:32px;height:32px;-moz-transition:.1s ease;-o-transition:.1s ease;-webkit-transition:1s ease;transition:.1s ease;padding:0;margin:0}#onetrust-pc-sdk #filter-btn-handler:hover{background-color:#3860be}#onetrust-pc-sdk #filter-btn-handler svg{width:12px;height:12px;margin:3px 10px 0 10px;display:block;position:static;right:auto;top:auto}#onetrust-pc-sdk .ot-ven-link,#onetrust-pc-sdk .ot-ven-legclaim-link{color:#3860be;text-decoration:none;font-weight:100;display:inline-block;padding-top:10px;transform:translate(0, 1%);-o-transform:translate(0, 1%);-ms-transform:translate(0, 1%);-webkit-transform:translate(0, 1%);position:relative;z-index:2}#onetrust-pc-sdk .ot-ven-link *,#onetrust-pc-sdk .ot-ven-legclaim-link *{font-size:inherit}#onetrust-pc-sdk .ot-ven-link:hover,#onetrust-pc-sdk .ot-ven-legclaim-link:hover{text-decoration:underline}#onetrust-pc-sdk .ot-ven-hdr{width:calc(100% - 160px);height:auto;float:left;word-break:break-word;word-wrap:break-word;vertical-align:middle;padding-bottom:3px}#onetrust-pc-sdk .ot-ven-link,#onetrust-pc-sdk .ot-ven-legclaim-link{letter-spacing:.03em;font-size:.75em;font-weight:400}#onetrust-pc-sdk .ot-ven-dets{border-radius:2px;background-color:#f8f8f8}#onetrust-pc-sdk .ot-ven-dets li:first-child p:first-child{border-top:none}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:not(:first-child){border-top:1px solid #ddd !important}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:nth-child(n+3) p{display:inline-block}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:nth-child(n+3) p:nth-of-type(odd){width:30%}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:nth-child(n+3) p:nth-of-type(even){width:50%;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc p,#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc h5{padding-top:5px;padding-bottom:5px;display:block}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc h5{display:inline-block}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc p:nth-last-child(-n+1){padding-bottom:10px}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc p:nth-child(-n+2):not(.disc-pur){padding-top:10px}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc .disc-pur-cont{display:inline}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc .disc-pur{position:relative;width:50% !important;word-break:break-word;word-wrap:break-word;left:calc(30% + 17px)}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc .disc-pur:nth-child(-n+1){position:static}#onetrust-pc-sdk .ot-ven-dets p,#onetrust-pc-sdk .ot-ven-dets h5,#onetrust-pc-sdk .ot-ven-dets span{font-size:.69em;text-align:left;vertical-align:middle;word-break:break-word;word-wrap:break-word;margin:0;padding-bottom:10px;padding-left:15px;color:#2e3644}#onetrust-pc-sdk .ot-ven-dets h5{padding-top:5px}#onetrust-pc-sdk .ot-ven-dets span{color:dimgray;padding:0;vertical-align:baseline}#onetrust-pc-sdk .ot-ven-dets .ot-ven-pur h5{border-top:1px solid #e9e9e9;border-bottom:1px solid #e9e9e9;padding-bottom:5px;margin-bottom:5px;font-weight:bold}#onetrust-pc-sdk #ot-host-lst .ot-sel-all{float:right;position:relative;margin-right:42px;top:10px}#onetrust-pc-sdk #ot-host-lst .ot-sel-all input[type=checkbox]{width:auto;height:auto}#onetrust-pc-sdk #ot-host-lst .ot-sel-all label{height:20px;width:20px;padding-left:0px}#onetrust-pc-sdk #ot-host-lst .ot-acc-txt{overflow:hidden;width:95%}#onetrust-pc-sdk .ot-host-hdr{position:relative;z-index:1;pointer-events:none;width:calc(100% - 125px);float:left}#onetrust-pc-sdk .ot-host-name,#onetrust-pc-sdk .ot-host-desc{display:inline-block;width:90%}#onetrust-pc-sdk .ot-host-name{pointer-events:none}#onetrust-pc-sdk .ot-host-hdr>a{text-decoration:underline;font-size:.82em;position:relative;z-index:2;float:left;margin-bottom:5px;pointer-events:initial}#onetrust-pc-sdk .ot-host-name+a{margin-top:5px}#onetrust-pc-sdk .ot-host-name,#onetrust-pc-sdk .ot-host-name a,#onetrust-pc-sdk .ot-host-desc,#onetrust-pc-sdk .ot-host-info{color:dimgray;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-host-name,#onetrust-pc-sdk .ot-host-name a{font-weight:bold;font-size:.82em;line-height:1.3}#onetrust-pc-sdk .ot-host-name a{font-size:1em}#onetrust-pc-sdk .ot-host-expand{margin-top:3px;margin-bottom:3px;clear:both;display:block;color:#3860be;font-size:.72em;font-weight:normal}#onetrust-pc-sdk .ot-host-expand *{font-size:inherit}#onetrust-pc-sdk .ot-host-desc,#onetrust-pc-sdk .ot-host-info{font-size:.688em;line-height:1.4;font-weight:normal}#onetrust-pc-sdk .ot-host-desc{margin-top:10px}#onetrust-pc-sdk .ot-host-opt{margin:0;font-size:inherit;display:inline-block;width:100%}#onetrust-pc-sdk .ot-host-opt li>div div{font-size:.8em;padding:5px 0}#onetrust-pc-sdk .ot-host-opt li>div div:nth-child(1){width:30%;float:left}#onetrust-pc-sdk .ot-host-opt li>div div:nth-child(2){width:70%;float:left;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-host-info{border:none;display:inline-block;width:calc(100% - 10px);padding:10px;margin-bottom:10px;background-color:#f8f8f8}#onetrust-pc-sdk .ot-host-info>div{overflow:auto}#onetrust-pc-sdk #no-results{text-align:center;margin-top:30px}#onetrust-pc-sdk #no-results p{font-size:1em;color:#2e3644;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk #no-results p span{font-weight:bold}#onetrust-pc-sdk #ot-fltr-modal{width:100%;height:auto;display:none;-moz-transition:.2s ease;-o-transition:.2s ease;-webkit-transition:2s ease;transition:.2s ease;overflow:hidden;opacity:1;right:0}#onetrust-pc-sdk #ot-fltr-modal .ot-label-txt{display:inline-block;font-size:.85em;color:dimgray}#onetrust-pc-sdk #ot-fltr-cnt{z-index:2147483646;background-color:#fff;position:absolute;height:90%;max-height:300px;width:325px;left:210px;margin-top:10px;margin-bottom:20px;padding-right:10px;border-radius:3px;-webkit-box-shadow:0px 0px 12px 2px #c7c5c7;-moz-box-shadow:0px 0px 12px 2px #c7c5c7;box-shadow:0px 0px 12px 2px #c7c5c7}#onetrust-pc-sdk .ot-fltr-scrlcnt{overflow-y:auto;overflow-x:hidden;clear:both;max-height:calc(100% - 60px)}#onetrust-pc-sdk #ot-anchor{border:12px solid rgba(0,0,0,0);display:none;position:absolute;z-index:2147483647;right:55px;top:75px;transform:rotate(45deg);-o-transform:rotate(45deg);-ms-transform:rotate(45deg);-webkit-transform:rotate(45deg);background-color:#fff;-webkit-box-shadow:-3px -3px 5px -2px #c7c5c7;-moz-box-shadow:-3px -3px 5px -2px #c7c5c7;box-shadow:-3px -3px 5px -2px #c7c5c7}#onetrust-pc-sdk .ot-fltr-btns{margin-left:15px}#onetrust-pc-sdk #filter-apply-handler{margin-right:15px}#onetrust-pc-sdk .ot-fltr-opt{margin-bottom:5px;margin-left:15px;min-height:20px;width:75%;position:relative}#onetrust-pc-sdk .ot-fltr-opt p{display:inline-block;margin:0;font-size:.9em;color:#2e3644}#onetrust-pc-sdk .ot-chkbox label span{font-size:.85em;color:dimgray}#onetrust-pc-sdk .ot-chkbox input[type=checkbox]+label::after{content:none;color:#fff}#onetrust-pc-sdk .ot-chkbox input[type=checkbox]:checked+label::after{content:""}#onetrust-pc-sdk .ot-chkbox input[type=checkbox][aria-checked=mixed]+label::after{content:""}#onetrust-pc-sdk .ot-chkbox input[type=checkbox]:focus+label::before{outline-style:solid;outline-width:2px;outline-style:auto}#onetrust-pc-sdk #ot-selall-vencntr,#onetrust-pc-sdk #ot-selall-adtlvencntr,#onetrust-pc-sdk #ot-selall-hostcntr,#onetrust-pc-sdk #ot-selall-licntr,#onetrust-pc-sdk #ot-selall-gnvencntr{right:15px;position:relative;width:20px;height:20px;float:right}#onetrust-pc-sdk #ot-selall-vencntr label,#onetrust-pc-sdk #ot-selall-adtlvencntr label,#onetrust-pc-sdk #ot-selall-hostcntr label,#onetrust-pc-sdk #ot-selall-licntr label,#onetrust-pc-sdk #ot-selall-gnvencntr label{float:left;padding-left:0}#onetrust-pc-sdk #ot-ven-lst:first-child{border-top:1px solid #e2e2e2}#onetrust-pc-sdk ul{list-style:none;padding:0}#onetrust-pc-sdk ul li{position:relative;margin:0;padding:15px 15px 15px 10px;border-bottom:1px solid #e2e2e2}#onetrust-pc-sdk ul li h3,#onetrust-pc-sdk ul li h4{font-size:.75em;color:#656565;margin:0;display:inline-block;width:70%;height:auto;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk ul li p{margin:0;font-size:.7em}#onetrust-pc-sdk ul li input[type=checkbox]{position:absolute;cursor:pointer;width:100%;height:100%;opacity:0;margin:0;top:0;left:0}#onetrust-pc-sdk .ot-cat-item>button:focus,#onetrust-pc-sdk .ot-acc-cntr>button:focus,#onetrust-pc-sdk li>button:focus{outline:#000 solid 2px}#onetrust-pc-sdk .ot-cat-item>button,#onetrust-pc-sdk .ot-acc-cntr>button,#onetrust-pc-sdk li>button{position:absolute;cursor:pointer;width:100%;height:100%;margin:0;top:0;left:0;z-index:1;max-width:none;border:none}#onetrust-pc-sdk .ot-cat-item>button[aria-expanded=false]~.ot-acc-txt,#onetrust-pc-sdk .ot-acc-cntr>button[aria-expanded=false]~.ot-acc-txt,#onetrust-pc-sdk li>button[aria-expanded=false]~.ot-acc-txt{margin-top:0;max-height:0;opacity:0;overflow:hidden;width:100%;transition:.25s ease-out;display:none}#onetrust-pc-sdk .ot-cat-item>button[aria-expanded=true]~.ot-acc-txt,#onetrust-pc-sdk .ot-acc-cntr>button[aria-expanded=true]~.ot-acc-txt,#onetrust-pc-sdk li>button[aria-expanded=true]~.ot-acc-txt{transition:.1s ease-in;margin-top:10px;width:100%;overflow:auto;display:block}#onetrust-pc-sdk .ot-cat-item>button[aria-expanded=true]~.ot-acc-grpcntr,#onetrust-pc-sdk .ot-acc-cntr>button[aria-expanded=true]~.ot-acc-grpcntr,#onetrust-pc-sdk li>button[aria-expanded=true]~.ot-acc-grpcntr{width:auto;margin-top:0px;padding-bottom:10px}#onetrust-pc-sdk .ot-host-item>button:focus,#onetrust-pc-sdk .ot-ven-item>button:focus{outline:0;border:2px solid #000}#onetrust-pc-sdk .ot-hide-acc>button{pointer-events:none}#onetrust-pc-sdk .ot-hide-acc .ot-plus-minus>*,#onetrust-pc-sdk .ot-hide-acc .ot-arw-cntr>*{visibility:hidden}#onetrust-pc-sdk .ot-hide-acc .ot-acc-hdr{min-height:30px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt){padding-right:10px;width:calc(100% - 37px);margin-top:10px;max-height:calc(100% - 90px)}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk{background-color:#f9f9fc;border:1px solid #e2e2e2;width:calc(100% - 2px);padding-bottom:5px;padding-top:5px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt{border:unset;background-color:unset}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt .ot-sel-all-hdr{display:none}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt .ot-sel-all{padding-right:.5rem}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt .ot-sel-all .ot-chkbox{right:0}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) .ot-sel-all{padding-right:34px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) .ot-sel-all-chkbox{width:auto}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) ul li{border:1px solid #e2e2e2;margin-bottom:10px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) .ot-acc-cntr>.ot-acc-hdr{padding:10px 0 10px 15px}#onetrust-pc-sdk.ot-addtl-vendors .ot-sel-all-chkbox{float:right}#onetrust-pc-sdk.ot-addtl-vendors .ot-plus-minus~.ot-sel-all-chkbox{right:34px}#onetrust-pc-sdk.ot-addtl-vendors #ot-ven-lst:first-child{border-top:none}#onetrust-pc-sdk .ot-acc-cntr{position:relative;border-left:1px solid #e2e2e2;border-right:1px solid #e2e2e2;border-bottom:1px solid #e2e2e2}#onetrust-pc-sdk .ot-acc-cntr input{z-index:1}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr{background-color:#f9f9fc;padding:5px 0 5px 15px;width:auto}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr .ot-plus-minus{vertical-align:middle;top:auto}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr .ot-arw-cntr{right:10px}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr input{z-index:2}#onetrust-pc-sdk .ot-acc-cntr.ot-add-tech .ot-acc-hdr{padding:10px 0 10px 15px}#onetrust-pc-sdk .ot-acc-cntr>input[type=checkbox]:checked~.ot-acc-hdr{border-bottom:1px solid #e2e2e2}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-txt{padding-left:10px;padding-right:10px}#onetrust-pc-sdk .ot-acc-cntr button[aria-expanded=true]~.ot-acc-txt{width:auto}#onetrust-pc-sdk .ot-acc-cntr .ot-addtl-venbox{display:none}#onetrust-pc-sdk .ot-vlst-cntr{margin-bottom:0;width:100%}#onetrust-pc-sdk .ot-vensec-title{font-size:.813em;vertical-align:middle;display:inline-block}#onetrust-pc-sdk .category-vendors-list-handler,#onetrust-pc-sdk .category-vendors-list-handler+a{margin-left:0;margin-top:10px}#onetrust-pc-sdk #ot-selall-vencntr.line-through label::after,#onetrust-pc-sdk #ot-selall-adtlvencntr.line-through label::after,#onetrust-pc-sdk #ot-selall-licntr.line-through label::after,#onetrust-pc-sdk #ot-selall-hostcntr.line-through label::after,#onetrust-pc-sdk #ot-selall-gnvencntr.line-through label::after{height:auto;border-left:0;transform:none;-o-transform:none;-ms-transform:none;-webkit-transform:none;left:5px;top:9px}#onetrust-pc-sdk #ot-category-title{float:left;padding-bottom:10px;font-size:1em;width:100%}#onetrust-pc-sdk .ot-cat-grp{margin-top:10px}#onetrust-pc-sdk .ot-cat-item{line-height:1.1;margin-top:10px;display:inline-block;width:100%}#onetrust-pc-sdk .ot-btn-container{text-align:right}#onetrust-pc-sdk .ot-btn-container button{display:inline-block;font-size:.75em;letter-spacing:.08em;margin-top:19px}#onetrust-pc-sdk #close-pc-btn-handler.ot-close-icon{position:absolute;top:10px;right:0;z-index:1;padding:0;background-color:rgba(0,0,0,0);border:none}#onetrust-pc-sdk #close-pc-btn-handler.ot-close-icon svg{display:block;height:10px;width:10px}#onetrust-pc-sdk #clear-filters-handler{margin-top:20px;margin-bottom:10px;float:right;max-width:200px;text-decoration:none;color:#3860be;font-size:.9em;font-weight:bold;background-color:rgba(0,0,0,0);border-color:rgba(0,0,0,0);padding:1px}#onetrust-pc-sdk #clear-filters-handler:hover{color:#2285f7}#onetrust-pc-sdk #clear-filters-handler:focus{outline:#000 solid 1px}#onetrust-pc-sdk .ot-enbl-chr h4~.ot-tgl,#onetrust-pc-sdk .ot-enbl-chr h4~.ot-always-active{right:45px}#onetrust-pc-sdk .ot-enbl-chr h4~.ot-tgl+.ot-tgl{right:120px}#onetrust-pc-sdk .ot-enbl-chr .ot-pli-hdr.ot-leg-border-color span:first-child{width:90px}#onetrust-pc-sdk .ot-enbl-chr li.ot-subgrp>h5+.ot-tgl-cntr{padding-right:25px}#onetrust-pc-sdk .ot-plus-minus{width:20px;height:20px;font-size:1.5em;position:relative;display:inline-block;margin-right:5px;top:3px}#onetrust-pc-sdk .ot-plus-minus span{position:absolute;background:#27455c;border-radius:1px}#onetrust-pc-sdk .ot-plus-minus span:first-of-type{top:25%;bottom:25%;width:10%;left:45%}#onetrust-pc-sdk .ot-plus-minus span:last-of-type{left:25%;right:25%;height:10%;top:45%}#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-arw,#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-plus-minus span:first-of-type,#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-plus-minus span:last-of-type{transform:rotate(90deg)}#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-plus-minus span:last-of-type{left:50%;right:50%}#onetrust-pc-sdk #ot-selall-vencntr label,#onetrust-pc-sdk #ot-selall-adtlvencntr label,#onetrust-pc-sdk #ot-selall-hostcntr label,#onetrust-pc-sdk #ot-selall-licntr label{position:relative;display:inline-block;width:20px;height:20px}#onetrust-pc-sdk .ot-host-item .ot-plus-minus,#onetrust-pc-sdk .ot-ven-item .ot-plus-minus{float:left;margin-right:8px;top:10px}#onetrust-pc-sdk .ot-ven-item ul{list-style:none inside;font-size:100%;margin:0}#onetrust-pc-sdk .ot-ven-item ul li{margin:0 !important;padding:0;border:none !important}#onetrust-pc-sdk .ot-pli-hdr{color:#77808e;overflow:hidden;padding-top:7.5px;padding-bottom:7.5px;width:calc(100% - 2px);border-top-left-radius:3px;border-top-right-radius:3px}#onetrust-pc-sdk .ot-pli-hdr span:first-child{top:50%;transform:translateY(50%);max-width:90px}#onetrust-pc-sdk .ot-pli-hdr span:last-child{padding-right:10px;max-width:95px;text-align:center}#onetrust-pc-sdk .ot-li-title{float:right;font-size:.813em}#onetrust-pc-sdk .ot-pli-hdr.ot-leg-border-color{background-color:#f4f4f4;border:1px solid #d8d8d8}#onetrust-pc-sdk .ot-pli-hdr.ot-leg-border-color span:first-child{text-align:left;width:70px}#onetrust-pc-sdk li.ot-subgrp>h5,#onetrust-pc-sdk .ot-cat-header{width:calc(100% - 130px)}#onetrust-pc-sdk li.ot-subgrp>h5+.ot-tgl-cntr{padding-left:13px}#onetrust-pc-sdk .ot-acc-grpcntr .ot-acc-grpdesc{margin-bottom:5px}#onetrust-pc-sdk .ot-acc-grpcntr .ot-subgrp-cntr{border-top:1px solid #d8d8d8}#onetrust-pc-sdk .ot-acc-grpcntr .ot-vlst-cntr+.ot-subgrp-cntr{border-top:none}#onetrust-pc-sdk .ot-acc-hdr .ot-arw-cntr+.ot-tgl-cntr,#onetrust-pc-sdk .ot-acc-txt h4+.ot-tgl-cntr{padding-left:13px}#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item .ot-subgrp>h5,#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item .ot-cat-header{width:calc(100% - 145px)}#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item h5+.ot-tgl-cntr,#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item .ot-cat-header+.ot-tgl{padding-left:28px}#onetrust-pc-sdk .ot-sel-all-hdr,#onetrust-pc-sdk .ot-sel-all-chkbox{display:inline-block;width:100%;position:relative}#onetrust-pc-sdk .ot-sel-all-chkbox{z-index:1}#onetrust-pc-sdk .ot-sel-all{margin:0;position:relative;padding-right:23px;float:right}#onetrust-pc-sdk .ot-consent-hdr,#onetrust-pc-sdk .ot-li-hdr{float:right;font-size:.812em;line-height:normal;text-align:center;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-li-hdr{max-width:100px;padding-right:10px}#onetrust-pc-sdk .ot-consent-hdr{max-width:55px}#onetrust-pc-sdk #ot-selall-licntr{display:block;width:21px;height:auto;float:right;position:relative;right:80px}#onetrust-pc-sdk #ot-selall-licntr label{position:absolute}#onetrust-pc-sdk .ot-ven-ctgl{margin-left:66px}#onetrust-pc-sdk .ot-ven-litgl+.ot-arw-cntr{margin-left:81px}#onetrust-pc-sdk .ot-enbl-chr .ot-host-cnt .ot-tgl-cntr{width:auto}#onetrust-pc-sdk #ot-lst-cnt:not(.ot-host-cnt) .ot-tgl-cntr{width:auto;top:auto;height:20px}#onetrust-pc-sdk #ot-lst-cnt .ot-chkbox{position:relative;display:inline-block;width:20px;height:20px}#onetrust-pc-sdk #ot-lst-cnt .ot-chkbox label{position:absolute;padding:0;width:20px;height:20px}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info-cntr{border:1px solid #d8d8d8;padding:.75rem 2rem;padding-bottom:0;width:auto;margin-top:.5rem}#onetrust-pc-sdk .ot-acc-grpdesc+.ot-leg-btn-container{padding-left:20px;padding-right:20px;width:calc(100% - 40px);margin-bottom:5px}#onetrust-pc-sdk .ot-subgrp .ot-leg-btn-container{margin-bottom:5px}#onetrust-pc-sdk #ot-ven-lst .ot-leg-btn-container{margin-top:10px}#onetrust-pc-sdk .ot-leg-btn-container{display:inline-block;width:100%;margin-bottom:10px}#onetrust-pc-sdk .ot-leg-btn-container button{height:auto;padding:6.5px 8px;margin-bottom:0;letter-spacing:0;font-size:.75em;line-height:normal}#onetrust-pc-sdk .ot-leg-btn-container svg{display:none;height:14px;width:14px;padding-right:5px;vertical-align:sub}#onetrust-pc-sdk .ot-active-leg-btn{cursor:default;pointer-events:none}#onetrust-pc-sdk .ot-active-leg-btn svg{display:inline-block}#onetrust-pc-sdk .ot-remove-objection-handler{text-decoration:underline;padding:0;font-size:.75em;font-weight:600;line-height:1;padding-left:10px}#onetrust-pc-sdk .ot-obj-leg-btn-handler span{font-weight:bold;text-align:center;font-size:inherit;line-height:1.5}#onetrust-pc-sdk.ot-close-btn-link #close-pc-btn-handler{border:none;height:auto;line-height:1.5;text-decoration:underline;font-size:.69em;background:none;right:15px;top:15px;width:auto;font-weight:normal}#onetrust-pc-sdk .ot-pgph-link{font-size:.813em !important;margin-top:5px;position:relative}#onetrust-pc-sdk .ot-pgph-link.ot-pgph-link-subgroup{margin-bottom:1rem}#onetrust-pc-sdk .ot-pgph-contr{margin:0 2.5rem}#onetrust-pc-sdk .ot-pgph-title{font-size:1.18rem;margin-bottom:2rem}#onetrust-pc-sdk .ot-pgph-desc{font-size:1rem;font-weight:400;margin-bottom:2rem;line-height:1.5rem}#onetrust-pc-sdk .ot-pgph-desc:not(:last-child):after{content:"";width:96%;display:block;margin:0 auto;padding-bottom:2rem;border-bottom:1px solid #e9e9e9}#onetrust-pc-sdk .ot-cat-header{float:left;font-weight:600;font-size:.875em;line-height:1.5;max-width:90%;vertical-align:middle}#onetrust-pc-sdk .ot-vnd-item>button:focus{outline:#000 solid 2px}#onetrust-pc-sdk .ot-vnd-item>button{position:absolute;cursor:pointer;width:100%;height:100%;margin:0;top:0;left:0;z-index:1;max-width:none;border:none}#onetrust-pc-sdk .ot-vnd-item>button[aria-expanded=false]~.ot-acc-txt{margin-top:0;max-height:0;opacity:0;overflow:hidden;width:100%;transition:.25s ease-out;display:none}#onetrust-pc-sdk .ot-vnd-item>button[aria-expanded=true]~.ot-acc-txt{transition:.1s ease-in;margin-top:10px;width:100%;overflow:auto;display:block}#onetrust-pc-sdk .ot-vnd-item>button[aria-expanded=true]~.ot-acc-grpcntr{width:auto;margin-top:0px;padding-bottom:10px}#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item{position:relative;border-radius:2px;margin:0;padding:0;border:1px solid #d8d8d8;border-top:none;width:calc(100% - 2px);float:left}#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item:first-of-type{margin-top:10px;border-top:1px solid #d8d8d8}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpdesc{padding-left:20px;padding-right:20px;width:calc(100% - 40px);font-size:.812em;margin-bottom:10px;margin-top:15px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpdesc>ul{padding-top:10px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpdesc>ul li{padding-top:0;line-height:1.5;padding-bottom:10px}#onetrust-pc-sdk .ot-accordion-layout div+.ot-acc-grpdesc{margin-top:5px}#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr:first-child{margin-top:10px}#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr:last-child,#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr:last-child{margin-bottom:5px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{padding-top:11.5px;padding-bottom:11.5px;padding-left:20px;padding-right:20px;width:calc(100% - 40px);display:inline-block}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-txt{width:100%;padding:0}#onetrust-pc-sdk .ot-accordion-layout .ot-subgrp-cntr{padding-left:20px;padding-right:15px;padding-bottom:0;width:calc(100% - 35px)}#onetrust-pc-sdk .ot-accordion-layout .ot-subgrp{padding-right:5px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpcntr{z-index:1;position:relative}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header+.ot-arw-cntr{position:absolute;top:50%;transform:translateY(-50%);right:20px;margin-top:-2px}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header+.ot-arw-cntr .ot-arw{width:15px;height:20px;margin-left:5px;color:dimgray}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{float:none;color:#2e3644;margin:0;display:inline-block;height:auto;word-wrap:break-word;min-height:inherit}#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr,#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr{padding-left:20px;width:calc(100% - 20px);display:inline-block;margin-top:0;padding-bottom:2px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{position:relative;min-height:25px}#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl,#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active{position:absolute;top:50%;transform:translateY(-50%);right:20px}#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl+.ot-tgl{right:95px}#onetrust-pc-sdk .ot-accordion-layout .category-vendors-list-handler,#onetrust-pc-sdk .ot-accordion-layout .category-vendors-list-handler+a{margin-top:5px}#onetrust-pc-sdk #ot-lst-cnt{margin-top:1rem;max-height:calc(100% - 96px)}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info-cntr{border:1px solid #d8d8d8;padding:.75rem 2rem;padding-bottom:0;width:auto;margin-top:.5rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info{margin-bottom:1rem;padding-left:.75rem;padding-right:.75rem;display:flex;flex-direction:column}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info[data-vnd-info-key*=DPOEmail]{border-top:1px solid #d8d8d8;padding-top:1rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info[data-vnd-info-key*=DPOLink]{border-bottom:1px solid #d8d8d8;padding-bottom:1rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info .ot-vnd-lbl{font-weight:bold;font-size:.85em;margin-bottom:.5rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info .ot-vnd-cnt{margin-left:.5rem;font-weight:500;font-size:.85rem}#onetrust-pc-sdk .ot-vs-list,#onetrust-pc-sdk .ot-vnd-serv{width:auto;padding:1rem 1.25rem;padding-bottom:0}#onetrust-pc-sdk .ot-vs-list .ot-vnd-serv-hdr-cntr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-serv-hdr-cntr{padding-bottom:.75rem;border-bottom:1px solid #d8d8d8}#onetrust-pc-sdk .ot-vs-list .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr{font-weight:600;font-size:.95em;line-height:2;margin-left:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item{border:none;margin:0;padding:0}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item button,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item button{outline:none;border-bottom:1px solid #d8d8d8}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item button[aria-expanded=true],#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item button[aria-expanded=true]{border-bottom:none}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item:first-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item:first-child{margin-top:.25rem;border-top:unset}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item:last-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item:last-child{margin-bottom:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item:last-child button,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item:last-child button{border-bottom:none}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info-cntr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info-cntr{border:1px solid #d8d8d8;padding:.75rem 1.75rem;padding-bottom:0;width:auto;margin-top:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info{margin-bottom:1rem;padding-left:.75rem;padding-right:.75rem;display:flex;flex-direction:column}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOEmail],#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOEmail]{border-top:1px solid #d8d8d8;padding-top:1rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOLink],#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOLink]{border-bottom:1px solid #d8d8d8;padding-bottom:1rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info .ot-vnd-lbl,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info .ot-vnd-lbl{font-weight:bold;font-size:.85em;margin-bottom:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info .ot-vnd-cnt,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info .ot-vnd-cnt{margin-left:.5rem;font-weight:500;font-size:.85rem}#onetrust-pc-sdk .ot-vs-list.ot-vnd-subgrp-cnt,#onetrust-pc-sdk .ot-vnd-serv.ot-vnd-subgrp-cnt{padding-left:40px}#onetrust-pc-sdk .ot-vs-list.ot-vnd-subgrp-cnt .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr,#onetrust-pc-sdk .ot-vnd-serv.ot-vnd-subgrp-cnt .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr{font-size:.8em}#onetrust-pc-sdk .ot-vs-list.ot-vnd-subgrp-cnt .ot-cat-header,#onetrust-pc-sdk .ot-vnd-serv.ot-vnd-subgrp-cnt .ot-cat-header{font-size:.8em}#onetrust-pc-sdk .ot-subgrp-cntr .ot-vnd-serv{margin-bottom:1rem;padding:1rem .95rem}#onetrust-pc-sdk .ot-subgrp-cntr .ot-vnd-serv .ot-vnd-serv-hdr-cntr{padding-bottom:.75rem;border-bottom:1px solid #d8d8d8}#onetrust-pc-sdk .ot-subgrp-cntr .ot-vnd-serv .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr{font-weight:700;font-size:.8em;line-height:20px;margin-left:.82rem}#onetrust-pc-sdk .ot-subgrp-cntr .ot-cat-header{font-weight:700;font-size:.8em;line-height:20px}#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-vnd-serv .ot-vnd-lst-cont .ot-accordion-layout .ot-acc-hdr div.ot-chkbox{margin-left:.82rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr{padding:.7rem 0;margin:0;display:flex;width:100%;align-items:center;justify-content:space-between}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr div:first-child,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr div:first-child,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr div:first-child{margin-left:.5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr div:last-child,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr div:last-child,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr div:last-child{margin-right:.5rem;margin-left:.5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-always-active{position:relative;right:unset;top:unset;transform:unset}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-plus-minus{top:0}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-arw-cntr{float:none;top:unset;right:unset;transform:unset;margin-top:-2px;position:relative}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-cat-header{flex:1;margin:0 .5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-tgl{position:relative;transform:none;right:0;top:0;float:none}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox{position:relative;margin:0 .5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox label{padding:0}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox label::before{position:relative}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox input{position:absolute;cursor:pointer;width:100%;height:100%;opacity:0;margin:0;top:0;left:0;z-index:1}#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp .ot-acc-hdr h5.ot-cat-header,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp .ot-acc-hdr h4.ot-cat-header{margin:0}#onetrust-pc-sdk .ot-vs-config .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp h5{top:0;line-height:20px}#onetrust-pc-sdk .ot-vs-list{display:flex;flex-direction:column;padding:0;margin:.5rem 4px}#onetrust-pc-sdk .ot-vs-selc-all{display:flex;padding:0;float:unset;align-items:center;justify-content:flex-start}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf{justify-content:flex-end}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf.ot-caret-conf .ot-sel-all-chkbox{margin-right:48px}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf .ot-sel-all-chkbox{margin:0;padding:0;margin-right:14px;justify-content:flex-end}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf #ot-selall-vencntr.ot-chkbox,#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf #ot-selall-vencntr.ot-tgl{display:inline-block;right:unset;width:auto;height:auto;float:none}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf #ot-selall-vencntr label{width:45px;height:25px}#onetrust-pc-sdk .ot-vs-selc-all .ot-sel-all-chkbox{margin-right:11px;margin-left:.75rem;display:flex;align-items:center}#onetrust-pc-sdk .ot-vs-selc-all .sel-all-hdr{margin:0 1.25rem;font-size:.812em;line-height:normal;text-align:center;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-vnd-list-cnt #ot-selall-vencntr.ot-chkbox{float:unset;right:0}#onetrust-pc-sdk[dir=rtl] #ot-back-arw,#onetrust-pc-sdk[dir=rtl] input~.ot-acc-hdr .ot-arw{transform:rotate(180deg);-o-transform:rotate(180deg);-ms-transform:rotate(180deg);-webkit-transform:rotate(180deg)}#onetrust-pc-sdk[dir=rtl] input:checked~.ot-acc-hdr .ot-arw{transform:rotate(270deg);-o-transform:rotate(270deg);-ms-transform:rotate(270deg);-webkit-transform:rotate(270deg)}#onetrust-pc-sdk[dir=rtl] .ot-chkbox label::after{transform:rotate(45deg);-webkit-transform:rotate(45deg);-o-transform:rotate(45deg);-ms-transform:rotate(45deg);border-left:0;border-right:3px solid}#onetrust-pc-sdk[dir=rtl] .ot-search-cntr>svg{right:0}@media only screen and (max-width: 600px){#onetrust-pc-sdk.otPcCenter{left:0;min-width:100%;height:100%;top:0;border-radius:0}#onetrust-pc-sdk #ot-pc-content,#onetrust-pc-sdk.ot-ftr-stacked .ot-btn-container{margin:1px 3px 0 10px;padding-right:10px;width:calc(100% - 23px)}#onetrust-pc-sdk .ot-btn-container button{max-width:none;letter-spacing:.01em}#onetrust-pc-sdk #close-pc-btn-handler{top:10px;right:17px}#onetrust-pc-sdk p{font-size:.7em}#onetrust-pc-sdk #ot-pc-hdr{margin:10px 10px 0 5px;width:calc(100% - 15px)}#onetrust-pc-sdk .vendor-search-handler{font-size:1em}#onetrust-pc-sdk #ot-back-arw{margin-left:12px}#onetrust-pc-sdk #ot-lst-cnt{margin:0;padding:0 5px 0 10px;min-width:95%}#onetrust-pc-sdk .switch+p{max-width:80%}#onetrust-pc-sdk .ot-ftr-stacked button{width:100%}#onetrust-pc-sdk #ot-fltr-cnt{max-width:320px;width:90%;border-top-right-radius:0;border-bottom-right-radius:0;margin:0;margin-left:15px;left:auto;right:40px;top:85px}#onetrust-pc-sdk .ot-fltr-opt{margin-left:25px;margin-bottom:10px}#onetrust-pc-sdk .ot-pc-refuse-all-handler{margin-bottom:0}#onetrust-pc-sdk #ot-fltr-cnt{right:40px}}@media only screen and (max-width: 476px){#onetrust-pc-sdk .ot-fltr-cntr,#onetrust-pc-sdk #ot-fltr-cnt{right:10px}#onetrust-pc-sdk #ot-anchor{right:25px}#onetrust-pc-sdk button{width:100%}#onetrust-pc-sdk:not(.ot-addtl-vendors) #ot-pc-lst:not(.ot-enbl-chr) .ot-sel-all{padding-right:9px}#onetrust-pc-sdk:not(.ot-addtl-vendors) #ot-pc-lst:not(.ot-enbl-chr) .ot-tgl-cntr{right:0}}@media(min-width: 768px){#onetrust-pc-sdk.ot-tgl-with-label .ot-label-status{display:inline}#onetrust-pc-sdk.ot-tgl-with-label #ot-pc-lst .ot-label-status{display:none}}@media only screen and (max-width: 896px)and (max-height: 425px)and (orientation: landscape){#onetrust-pc-sdk.otPcCenter{left:0;top:0;min-width:100%;height:100%;border-radius:0}#onetrust-pc-sdk .ot-pc-header{height:auto;min-height:20px}#onetrust-pc-sdk .ot-pc-header .ot-pc-logo{max-height:30px}#onetrust-pc-sdk .ot-pc-footer{max-height:60px;overflow-y:auto}#onetrust-pc-sdk #ot-pc-content,#onetrust-pc-sdk #ot-pc-lst{bottom:70px}#onetrust-pc-sdk.ot-ftr-stacked #ot-pc-content{bottom:70px}#onetrust-pc-sdk #ot-anchor{left:initial;right:50px}#onetrust-pc-sdk #ot-lst-title{margin-top:12px}#onetrust-pc-sdk #ot-lst-title *{font-size:inherit}#onetrust-pc-sdk #ot-pc-hdr input{margin-right:0;padding-right:45px}#onetrust-pc-sdk .switch+p{max-width:85%}#onetrust-pc-sdk #ot-sel-blk{position:static}#onetrust-pc-sdk #ot-pc-lst{overflow:auto}#onetrust-pc-sdk #ot-lst-cnt{max-height:none;overflow:initial}#onetrust-pc-sdk #ot-lst-cnt.no-results{height:auto}#onetrust-pc-sdk input{font-size:1em !important}#onetrust-pc-sdk p{font-size:.6em}#onetrust-pc-sdk #ot-fltr-modal{width:100%;top:0}#onetrust-pc-sdk ul li p,#onetrust-pc-sdk .category-vendors-list-handler,#onetrust-pc-sdk .category-vendors-list-handler+a,#onetrust-pc-sdk .category-host-list-handler{font-size:.6em}#onetrust-pc-sdk.ot-shw-fltr #ot-anchor{display:none !important}#onetrust-pc-sdk.ot-shw-fltr #ot-pc-lst{height:100% !important;overflow:hidden;top:0px}#onetrust-pc-sdk.ot-shw-fltr #ot-fltr-cnt{margin:0;height:100%;max-height:none;padding:10px;top:0;width:calc(100% - 20px);position:absolute;right:0;left:0;max-width:none}#onetrust-pc-sdk.ot-shw-fltr .ot-fltr-scrlcnt{max-height:calc(100% - 65px)}}
            #onetrust-consent-sdk #onetrust-pc-sdk,
                #onetrust-consent-sdk #ot-search-cntr,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-switch.ot-toggle,
                #onetrust-consent-sdk #onetrust-pc-sdk ot-grp-hdr1 .checkbox,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-title:after
                ,#onetrust-consent-sdk #onetrust-pc-sdk #ot-sel-blk,
                        #onetrust-consent-sdk #onetrust-pc-sdk #ot-fltr-cnt,
                        #onetrust-consent-sdk #onetrust-pc-sdk #ot-anchor {
                    background-color: #FFF;
                }
               
            #onetrust-consent-sdk #onetrust-pc-sdk h3,
                #onetrust-consent-sdk #onetrust-pc-sdk h4,
                #onetrust-consent-sdk #onetrust-pc-sdk h5,
                #onetrust-consent-sdk #onetrust-pc-sdk h6,
                #onetrust-consent-sdk #onetrust-pc-sdk p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-ven-lst .ot-ven-opts p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-desc,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-title,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-li-title,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-sel-all-hdr span,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-info,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-fltr-modal #modal-header,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-checkbox label span,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst #ot-sel-blk p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst #ot-lst-title h3,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst .back-btn-handler p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst .ot-ven-name,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst #ot-ven-lst .consent-category,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-inactive-leg-btn,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-label-status,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-chkbox label span,
                #onetrust-consent-sdk #onetrust-pc-sdk #clear-filters-handler,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-optout-signal
                {
                    color: #2E2E2E;
                }
             #onetrust-consent-sdk #onetrust-pc-sdk .privacy-notice-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-pgph-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk .category-vendors-list-handler,
                    #onetrust-consent-sdk #onetrust-pc-sdk .category-vendors-list-handler + a,
                    #onetrust-consent-sdk #onetrust-pc-sdk .category-host-list-handler,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-ven-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-ven-legclaim-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-name a,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-acc-hdr .ot-host-expand,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-info a,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-content #ot-pc-desc .ot-link-btn,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info a,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info a
                    {
                        color: #007398;
                    }
            #onetrust-consent-sdk #onetrust-pc-sdk .category-vendors-list-handler:hover { text-decoration: underline;}
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-grpcntr.ot-acc-txt,
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-txt .ot-subgrp-tgl .ot-switch.ot-toggle
             {
                background-color: #F8F8F8;
            }
             #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-info,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-txt .ot-ven-dets
                            {
                                background-color: #F8F8F8;
                            }
        #onetrust-consent-sdk #onetrust-pc-sdk
            button:not(#clear-filters-handler):not(.ot-close-icon):not(#filter-btn-handler):not(.ot-remove-objection-handler):not(.ot-obj-leg-btn-handler):not([aria-expanded]):not(.ot-link-btn),
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-active-leg-btn {
                background-color: #007398;border-color: #007398;
                color: #FFF;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-active-menu {
                border-color: #007398;
            }
            
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-remove-objection-handler{
                background-color: transparent;
                border: 1px solid transparent;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-inactive-leg-btn {
                background-color: #FFFFFF;
                color: #4D4D4D; border-color: #4D4D4D;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-tgl input:focus + .ot-switch, .ot-switch .ot-switch-nob, .ot-switch .ot-switch-nob:before,
            #onetrust-pc-sdk .ot-checkbox input[type="checkbox"]:focus + label::before,
            #onetrust-pc-sdk .ot-chkbox input[type="checkbox"]:focus + label::before {
                outline-color: #000000;
                outline-width: 1px;
                outline-offset: 1px;
            }
            #onetrust-pc-sdk .ot-host-item > button:focus, #onetrust-pc-sdk .ot-ven-item > button:focus {
                border: 1px solid #000000;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk *:focus,
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-vlst-cntr > a:focus {
               outline: 1px solid #000000;
               outline-offset: 1px;
            }#onetrust-pc-sdk .ot-vlst-cntr .ot-ext-lnk,  #onetrust-pc-sdk .ot-ven-hdr .ot-ext-lnk{
                    background-image: url('https://cdn.cookielaw.org/logos/static/ot_external_link.svg');
                }
            /*! v6.10.0 2021-01-18 */
div#onetrust-consent-sdk #onetrust-banner-sdk{border-top:2px solid #d0d3d4!important;box-shadow:none;padding:24px}div#onetrust-consent-sdk button{border-radius:0!important;box-shadow:none!important;box-sizing:border-box!important;font-size:20px!important;font-weight:400!important;letter-spacing:0!important;max-width:none!important;white-space:nowrap!important}div#onetrust-consent-sdk button:not(.ot-link-btn){background-color:#007dbc!important;border:2px solid #007dbc!important;color:#fff!important;height:48px!important;padding:0 1em!important;width:auto!important}div#onetrust-consent-sdk button:hover{background-color:#fff!important;border-color:#949494!important;color:#2e2e2e!important}div#onetrust-consent-sdk button.ot-link-btn{color:#007dbc!important;font-size:16px!important;text-decoration:underline}div#onetrust-consent-sdk button.ot-link-btn:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-consent-sdk a,div#onetrust-pc-sdk a{color:#007dbc!important;text-decoration:underline!important}div#onetrust-consent-sdk a,div#onetrust-consent-sdk button,div#onetrust-consent-sdk p:hover{opacity:1!important}div#onetrust-consent-sdk a:focus,div#onetrust-consent-sdk button:focus,div#onetrust-consent-sdk input:focus{outline:2px solid #949494!important;outline-offset:1px!important}div#onetrust-banner-sdk .ot-sdk-container{padding:0;width:auto}div#onetrust-banner-sdk .ot-sdk-row{align-items:flex-start;box-sizing:border-box;display:flex;flex-direction:column;justify-content:space-between;margin:auto;max-width:1152px}div#onetrust-banner-sdk .ot-sdk-row:after{display:none}div#onetrust-banner-sdk #onetrust-group-container,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-group-container,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{flex-grow:1;width:auto}div#onetrust-banner-sdk #onetrust-policy,div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy{margin:0;overflow:visible}div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy-text,div#onetrust-consent-sdk #onetrust-policy-text{font-size:16px;line-height:24px;max-width:44em;margin:0}div#onetrust-consent-sdk #onetrust-policy-text a[href]{font-weight:400;margin-left:8px}div#onetrust-banner-sdk #onetrust-button-group-parent{flex:0 0 auto;margin:32px 0 0;width:100%}div#onetrust-banner-sdk #onetrust-button-group{display:flex;flex-direction:row;flex-wrap:wrap;justify-content:flex-end;margin:-8px}div#onetrust-banner-sdk .banner-actions-container{display:flex;flex:1 0 auto}div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button:last-of-type,div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto;margin:8px;width:auto}div#onetrust-consent-sdk #onetrust-pc-btn-handler{background-color:#fff!important;color:inherit!important}div#onetrust-banner-sdk #onetrust-close-btn-container{display:none}@media only screen and (min-width:556px){div#onetrust-consent-sdk #onetrust-banner-sdk{padding:40px}div#onetrust-banner-sdk #onetrust-policy{margin:0 40px 0 0}div#onetrust-banner-sdk .ot-sdk-row{align-items:center;flex-direction:row}div#onetrust-banner-sdk #onetrust-button-group-parent,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-button-group-parent,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{margin:0;padding:0;width:auto}div#onetrust-banner-sdk #onetrust-button-group,div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group{align-items:stretch;flex-direction:column-reverse;margin:0}div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto}}@media only screen and (min-width:768px){div#onetrust-banner-sdk #onetrust-policy{margin:0 48px 0 0}div#onetrust-consent-sdk #onetrust-banner-sdk{padding:48px}}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-size:16px;line-height:24px}div#onetrust-consent-sdk #onetrust-pc-sdk p,div#onetrust-pc-sdk #ot-pc-desc,div#onetrust-pc-sdk .category-host-list-handler,div#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{font-size:16px;font-weight:400;line-height:24px}div#onetrust-consent-sdk a:hover,div#onetrust-pc-sdk a:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-pc-sdk{border-radius:0;bottom:0;height:auto;left:0;margin:auto;max-width:100%;overflow:hidden;right:0;top:0;width:512px;max-height:800px}div#onetrust-pc-sdk .ot-pc-header{display:none}div#onetrust-pc-sdk #ot-pc-content{overscroll-behavior:contain;padding:0 12px 0 24px;margin:16px 4px 0 0;top:0;right:16px;left:0;width:auto}div#onetrust-pc-sdk #ot-category-title,div#onetrust-pc-sdk #ot-pc-title{font-size:24px;font-weight:400;line-height:32px;margin:0 0 16px}div#onetrust-pc-sdk #ot-pc-desc{padding:0}div#onetrust-pc-sdk #ot-pc-desc a{display:inline}div#onetrust-pc-sdk #accept-recommended-btn-handler{display:none!important}div#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:2px solid #007dbc;outline-offset:-1px;transition:none}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item{border-width:0 0 2px}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item:first-of-type{border-width:2px 0}div#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{padding:8px 0;width:100%}div#onetrust-pc-sdk .ot-plus-minus{transform:translateY(2px)}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item>button{background:0 0!important;border:0!important;height:44px!important;max-width:none!important;width:calc(100% - 48px)!important}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-weight:700}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr{padding:0}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item .ot-acc-grpdesc{padding:0;width:100%}div#onetrust-pc-sdk .ot-acc-grpcntr .ot-subgrp-cntr{border:0;padding:0}div#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp{margin:0}div#onetrust-pc-sdk .ot-always-active-group .ot-cat-header{width:calc(100% - 160px)}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{width:calc(100% - 88px)}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active{color:inherit;font-size:12px;font-weight:400;line-height:1.5;padding-right:48px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:before{border-radius:12px;position:absolute;right:0;top:0;content:'';background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:after{border-radius:50%;position:absolute;right:5px;top:4px;content:'';background-color:#007dbc;height:12px;width:12px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active,div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl{right:2px}div#onetrust-pc-sdk .ot-switch{display:block;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob,div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob{background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob:before{background-color:#737373;height:8px;left:4px;top:4px;width:8px}div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob:before{background-color:#007dbc;height:12px;left:0;top:2px;width:12px}div#onetrust-pc-sdk .ot-tgl input:focus+.ot-switch .ot-switch-nob{box-shadow:0 0;outline:2px solid #007dbc!important;outline-offset:1px;transition:none}div#onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-grpcntr.ot-acc-txt{background-color:transparent;padding-left:3px}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr,div#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr{overflow:visible;width:100%}div#onetrust-pc-sdk .ot-pc-footer{border-top:0 solid}div#onetrust-pc-sdk .ot-btn-container{padding-top:24px;text-align:center}div#onetrust-pc-sdk .ot-pc-footer button{margin:8px 0;background-color:#fff}div#onetrust-pc-sdk .ot-pc-footer-logo{background-color:#fff}div#onetrust-pc-sdk #ot-lst-title span{font-size:24px;font-weight:400;line-height:32px}div#onetrust-pc-sdk #ot-host-lst .ot-host-desc,div#onetrust-pc-sdk #ot-host-lst .ot-host-expand,div#onetrust-pc-sdk #ot-host-lst .ot-host-name,div#onetrust-pc-sdk #ot-host-lst .ot-host-name a,div#onetrust-pc-sdk .back-btn-handler,div#onetrust-pc-sdk .ot-host-opt li>div div{font-size:16px;font-weight:400;line-height:24px}div#onetrust-pc-sdk #ot-pc-lst{top:0}div#onetrust-pc-sdk .back-btn-handler{text-decoration:none!important}div#onetrust-pc-sdk #filter-btn-handler:hover svg{filter:invert(1)}div#onetrust-pc-sdk .back-btn-handler svg{width:16px;height:16px}div#onetrust-pc-sdk .ot-host-item>button{background:0 0!important;border:0!important;height:66px!important;max-width:none!important;width:calc(100% - 5px)!important;transform:translate(2px,2px)}div#onetrust-pc-sdk .ot-host-item{border-bottom:2px solid #b9b9b9;padding:0}div#onetrust-pc-sdk .ot-host-item .ot-acc-hdr{margin:0 0 -6px;padding:8px 0}div#onetrust-pc-sdk ul li:first-child{border-top:2px solid #b9b9b9}div#onetrust-pc-sdk .ot-host-item .ot-plus-minus{margin:0 8px 0 0}div#onetrust-pc-sdk .ot-search-cntr{width:calc(100% - 48px)}div#onetrust-pc-sdk .ot-host-opt .ot-host-info{background-color:transparent}div#onetrust-pc-sdk .ot-host-opt li>div div{padding:0}div#onetrust-pc-sdk #vendor-search-handler{border-radius:0;border-color:#939393;border-style:solid;border-width:2px 0 2px 2px;font-size:20px;height:48px;margin:0}div#onetrust-pc-sdk .ot-lst-subhdr svg{right:0;top:8px}div#onetrust-pc-sdk .ot-fltr-cntr{box-sizing:border-box;right:0;width:48px}div#onetrust-pc-sdk #filter-btn-handler{width:48px!important;padding:8px!important}div#onetrust-pc-sdk #ot-fltr-modal{max-height:400px}
.ot-sdk-cookie-policy{font-family:inherit;font-size:16px}.ot-sdk-cookie-policy.otRelFont{font-size:1rem}.ot-sdk-cookie-policy h3,.ot-sdk-cookie-policy h4,.ot-sdk-cookie-policy h6,.ot-sdk-cookie-policy p,.ot-sdk-cookie-policy li,.ot-sdk-cookie-policy a,.ot-sdk-cookie-policy th,.ot-sdk-cookie-policy #cookie-policy-description,.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group,.ot-sdk-cookie-policy #cookie-policy-title{color:dimgray}.ot-sdk-cookie-policy #cookie-policy-description{margin-bottom:1em}.ot-sdk-cookie-policy h4{font-size:1.2em}.ot-sdk-cookie-policy h6{font-size:1em;margin-top:2em}.ot-sdk-cookie-policy th{min-width:75px}.ot-sdk-cookie-policy a,.ot-sdk-cookie-policy a:hover{background:#fff}.ot-sdk-cookie-policy thead{background-color:#f6f6f4;font-weight:bold}.ot-sdk-cookie-policy .ot-mobile-border{display:none}.ot-sdk-cookie-policy section{margin-bottom:2em}.ot-sdk-cookie-policy table{border-collapse:inherit}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy{font-family:inherit;font-size:1rem}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h3,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h4,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h6,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy p,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy li,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-title{color:dimgray}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description{margin-bottom:1em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-subgroup{margin-left:1.5em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group-desc,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-table-header,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy span,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td{font-size:.9em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td span,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td a{font-size:inherit}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group{font-size:1em;margin-bottom:.6em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-title{margin-bottom:1.2em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy>section{margin-bottom:1em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th{min-width:75px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a:hover{background:#fff}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy thead{background-color:#f6f6f4;font-weight:bold}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-mobile-border{display:none}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy section{margin-bottom:2em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-subgroup ul li{list-style:disc;margin-left:1.5em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-subgroup ul li h4{display:inline-block}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table{border-collapse:inherit;margin:auto;border:1px solid #d7d7d7;border-radius:5px;border-spacing:initial;width:100%;overflow:hidden}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table th,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table td{border-bottom:1px solid #d7d7d7;border-right:1px solid #d7d7d7}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr:last-child td{border-bottom:0px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr th:last-child,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr td:last-child{border-right:0px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-host,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-cookies-type{width:25%}.ot-sdk-cookie-policy[dir=rtl]{text-align:left}#ot-sdk-cookie-policy h3{font-size:1.5em}@media only screen and (max-width: 530px){.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) table,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) thead,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tbody,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) th,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr{display:block}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) thead tr{position:absolute;top:-9999px;left:-9999px}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr{margin:0 0 1em 0}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr:nth-child(odd),.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr:nth-child(odd) a{background:#f6f6f4}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td{border:none;border-bottom:1px solid #eee;position:relative;padding-left:50%}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td:before{position:absolute;height:100%;left:6px;width:40%;padding-right:10px}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) .ot-mobile-border{display:inline-block;background-color:#e4e4e4;position:absolute;height:100%;top:0;left:45%;width:2px}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td:before{content:attr(data-label);font-weight:bold}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) li{word-break:break-word;word-wrap:break-word}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table{overflow:hidden}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table td{border:none;border-bottom:1px solid #d7d7d7}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy thead,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy tbody,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy tr{display:block}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-host,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-cookies-type{width:auto}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy tr{margin:0 0 1em 0}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td:before{height:100%;width:40%;padding-right:10px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td:before{content:attr(data-label);font-weight:bold}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy li{word-break:break-word;word-wrap:break-word}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy thead tr{position:absolute;top:-9999px;left:-9999px;z-index:-9999}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr:last-child td{border-bottom:1px solid #d7d7d7;border-right:0px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr:last-child td:last-child{border-bottom:0px}}
                
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h5,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h6,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy li,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy p,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy span,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description {
                        color: #696969;
                    }
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th {
                        color: #696969;
                    }
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group {
                        color: #696969;
                    }
                    
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-title {
                            color: #696969;
                        }
                    
            
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table th {
                            background-color: #F8F8F8;
                        }
                    
            .ot-floating-button__front{background-image:url('https://cdn.cookielaw.org/logos/static/ot_persistent_cookie_icon.png')}</style><script src="https://static.addtoany.com/menu/modules/core.pt5ow5lr.js" type="module"></script><style type="text/css">.a2a_hide{display:none}.a2a_logo_color{background-color:#0166ff}.a2a_menu,.a2a_menu *{-moz-box-sizing:content-box;-webkit-box-sizing:content-box;box-sizing:content-box;float:none;margin:0;padding:0;position:static;height:auto;width:auto}.a2a_menu{border-radius:6px;display:none;direction:ltr;background:#FFF;font:16px sans-serif-light,HelveticaNeue-Light,"Helvetica Neue Light","Helvetica Neue",Arial,Helvetica,"Liberation Sans",sans-serif;color:#000;line-height:12px;border:1px solid #CCC;vertical-align:baseline;overflow:hidden}.a2a_mini{min-width:200px;position:absolute;width:300px;z-index:9999997}.a2a_overlay{display:none;background:#616c7deb;backdrop-filter:blur(4px);-webkit-backdrop-filter:blur(4px);position:fixed;top:0;right:0;left:0;bottom:0;z-index:9999998;-webkit-tap-highlight-color:transparent;transition:opacity .14s,backdrop-filter .14s}.a2a_full{background:#FFF;border:1px solid #FFF;box-shadow:#2a2a2a1a 0 0 20px 10px;height:auto;height:calc(320px);top:15%;left:50%;margin-left:-320px;position:fixed;text-align:center;width:640px;z-index:9999999;transition:transform .14s,opacity .14s}.a2a_full_footer,.a2a_full_header,.a2a_full_services{border:0;margin:0;padding:12px;box-sizing:border-box}.a2a_full_header{padding-bottom:8px}.a2a_full_services{height:280px;overflow-y:scroll;padding:0 12px;-webkit-overflow-scrolling:touch}.a2a_full_services .a2a_i{display:inline-block;float:none;width:181px;width:calc(33.334% - 18px)}div.a2a_full_footer{font-size:12px;text-align:center;padding:8px 14px}div.a2a_full_footer a,div.a2a_full_footer a:visited{display:inline;font-size:12px;line-height:14px;padding:8px 14px}div.a2a_full_footer a:focus,div.a2a_full_footer a:hover{background:0 0;border:0;color:#0166FF}div.a2a_full_footer a span.a2a_s_a2a,div.a2a_full_footer a span.a2a_w_a2a{background-size:14px;border-radius:3px;display:inline-block;height:14px;line-height:14px;margin:0 3px 0 0;vertical-align:top;width:14px}.a2a_modal{height:0;left:50%;margin-left:-320px;position:fixed;text-align:center;top:15%;width:640px;z-index:9999999;transition:transform .14s,opacity .14s;-webkit-tap-highlight-color:transparent}.a2a_modal_body{background:0 0;border:0;font:24px sans-serif-light,HelveticaNeue-Light,"Helvetica Neue Light","Helvetica Neue",Arial,Helvetica,"Liberation Sans",sans-serif;position:relative;height:auto;width:auto}.a2a_thanks{color:#fff;height:auto;margin-top:20px;width:auto}.a2a_thanks>div:first-child{margin:0 0 40px 0}.a2a_thanks div *{height:inherit}#a2a_copy_link{background:#FFF;border:1px solid #FFF;cursor:pointer;margin-top:15%}label.a2a_s_link#a2a_copy_link_icon,label.a2a_w_link#a2a_copy_link_icon{background-size:48px;border-radius:0;display:inline-block;height:48px;left:0;line-height:48px;margin:0 3px 0 0;position:absolute;vertical-align:top;width:48px}#a2a_modal input#a2a_copy_link_text{background-color:transparent;border:0;color:#2A2A2A;cursor:pointer;font:inherit;height:48px;left:62px;max-width:initial;min-height:auto;padding:0;position:relative;width:564px;width:calc(100% - 76px)}#a2a_copy_link_copied{background-color:#0166ff;color:#fff;display:none;font:inherit;font-size:16px;margin-top:1px;padding:3px 8px}@media (forced-colors:active){.a2a_color_buttons a,.a2a_svg{forced-color-adjust:none}}@media (prefers-color-scheme:dark){.a2a_menu a,.a2a_menu a.a2a_i,.a2a_menu a.a2a_i:visited,.a2a_menu a.a2a_more,i.a2a_i{border-color:#2a2a2a!important;color:#fff!important}.a2a_menu a.a2a_i:active,.a2a_menu a.a2a_i:focus,.a2a_menu a.a2a_i:hover,.a2a_menu a.a2a_more:active,.a2a_menu a.a2a_more:focus,.a2a_menu a.a2a_more:hover,.a2a_menu_find_container{border-color:#444!important;background-color:#444!important}.a2a_menu:not(.a2a_thanks){background-color:#2a2a2a;border-color:#2a2a2a}.a2a_menu_find{color:#fff!important}.a2a_menu label.a2a_s_find svg{background-color:transparent!important}.a2a_menu label.a2a_s_find svg path{fill:#fff!important}.a2a_full{box-shadow:#00000066 0 0 20px 10px}.a2a_overlay{background-color:#373737eb}}@media print{.a2a_floating_style,.a2a_menu,.a2a_overlay{visibility:hidden}}@keyframes a2aFadeIn{from{opacity:0}to{opacity:1}}.a2a_starting{opacity:0}.a2a_starting.a2a_full,.a2a_starting.a2a_modal{transform:scale(.8)}@media (max-width:639px){.a2a_full{border-radius:0;top:15%;left:0;margin-left:auto;width:100%}.a2a_modal{left:0;margin-left:10px;width:calc(100% - 20px)}}@media (min-width:318px) and (max-width:437px){.a2a_full .a2a_full_services .a2a_i{width:calc(50% - 18px)}}@media (max-width:317px){.a2a_full .a2a_full_services .a2a_i{width:calc(100% - 18px)}}@media (max-height:436px){.a2a_full{bottom:40px;height:auto;top:40px}}@media (max-height:550px){.a2a_modal{top:30px}}@media (max-height:360px){.a2a_modal{top:20px}.a2a_thanks>div:first-child{margin-bottom:20px}}.a2a_menu a{color:#0166FF;text-decoration:none;font:16px sans-serif-light,HelveticaNeue-Light,"Helvetica Neue Light","Helvetica Neue",Arial,Helvetica,"Liberation Sans",sans-serif;line-height:14px;height:auto;width:auto;outline:0}.a2a_menu a.a2a_i:visited,.a2a_menu a.a2a_more{color:#0166FF}.a2a_menu a.a2a_i:active,.a2a_menu a.a2a_i:focus,.a2a_menu a.a2a_i:hover,.a2a_menu a.a2a_more:active,.a2a_menu a.a2a_more:focus,.a2a_menu a.a2a_more:hover{color:#2A2A2A;border-color:#EEE;border-style:solid;background-color:#EEE;text-decoration:none}.a2a_menu label.a2a_s_find{background-size:24px;height:24px;left:8px;pointer-events:auto;position:absolute;top:7px;width:24px}.a2a_menu label.a2a_s_find svg{background-color:#FFF}.a2a_menu label.a2a_s_find svg path{fill:#CCC}#a2a_menu_container{display:inline-block}.a2a_menu_find_container{border:1px solid #CCC;border-radius:6px;padding:2px 24px 2px 0;position:relative;text-align:left}.a2a_cols_container .a2a_col1{overflow-x:hidden;overflow-y:auto;-webkit-overflow-scrolling:touch}#a2a_modal input,#a2a_modal input[type=text],.a2a_menu input,.a2a_menu input[type=text]{display:block;background-image:none;box-shadow:none;line-height:100%;margin:0;outline:0;overflow:hidden;padding:0;-moz-box-shadow:none;-webkit-box-shadow:none;-webkit-appearance:none}#a2afeed_find_container input,#a2afeed_find_container input[type=text],#a2apage_find_container input,#a2apage_find_container input[type=text]{background-color:transparent;border:0;box-sizing:content-box;color:#2A2A2A;float:none;font:inherit;font-size:16px;height:28px;line-height:20px;left:38px;outline:0;margin:0;max-width:initial;min-height:initial;padding:2px 0;position:relative;width:99%}.a2a_clear{clear:both}.a2a_svg{background-repeat:no-repeat;display:block;overflow:hidden;height:32px;line-height:32px;padding:0;pointer-events:none;width:32px}.a2a_svg svg{background-repeat:no-repeat;background-position:50% 50%;border:none;display:block;left:0;margin:0 auto;overflow:hidden;padding:0;position:relative;top:0;width:auto;height:auto}a.a2a_i,i.a2a_i{display:block;float:left;border:1px solid #FFF;line-height:24px;padding:6px 8px;text-align:left;white-space:nowrap;overflow:hidden;text-overflow:ellipsis;width:132px}a.a2a_i span,a.a2a_more span{display:inline-block;overflow:hidden;vertical-align:top}a.a2a_i .a2a_svg{margin:0 6px 0 0}a.a2a_i .a2a_svg,a.a2a_more .a2a_svg{background-size:24px;height:24px;line-height:24px;width:24px}a.a2a_sss:hover{border-left:1px solid #CCC}a.a2a_more{border-bottom:1px solid #FFF;border-left:0;border-right:0;line-height:24px;margin:6px 0 0;padding:6px;-webkit-touch-callout:none}a.a2a_more span{height:24px;margin:0 6px 0 0}.a2a_kit .a2a_svg{background-repeat:repeat}.a2a_default_style a:empty,.a2a_flex_style a:empty,.a2a_floating_style a:empty,.a2a_overlay_style a:empty{display:none}.a2a_color_buttons a,.a2a_floating_style a{text-decoration:none}.a2a_default_style:not(.a2a_flex_style) a{float:left;line-height:16px;padding:0 2px}.a2a_default_style a:hover .a2a_svg,.a2a_floating_style a:hover .a2a_svg,.a2a_overlay_style a:hover .a2a_svg svg{opacity:.7}.a2a_overlay_style.a2a_default_style a:hover .a2a_svg{opacity:1}.a2a_default_style .a2a_count,.a2a_default_style .a2a_svg,.a2a_floating_style .a2a_svg,.a2a_menu .a2a_svg,.a2a_vertical_style .a2a_count,.a2a_vertical_style .a2a_svg{border-radius:4px}.a2a_default_style .a2a_counter img,.a2a_default_style .a2a_dd,.a2a_default_style .a2a_svg{float:left}.a2a_default_style .a2a_img_text{margin-right:4px}.a2a_default_style .a2a_divider{border-left:1px solid #000;display:inline;float:left;height:16px;line-height:16px;margin:0 5px}.a2a_kit a{cursor:pointer;transition:none}.a2a_floating_style{background-color:#fff;border-radius:6px;position:fixed;z-index:9999995}.a2a_overlay_style{z-index:2147483647}.a2a_floating_style,.a2a_overlay_style{animation:a2aFadeIn .2s ease-in;padding:4px}.a2a_vertical_style:not(.a2a_flex_style) a{clear:left;display:block;overflow:hidden;padding:4px}.a2a_floating_style.a2a_default_style{bottom:0}.a2a_floating_style.a2a_default_style a,.a2a_overlay_style.a2a_default_style a{padding:4px}.a2a_count{background-color:#fff;border:1px solid #ccc;box-sizing:border-box;color:#2a2a2a;display:block;float:left;font:12px Arial,Helvetica,sans-serif;height:16px;margin-left:4px;position:relative;text-align:center;width:50px}.a2a_count:after,.a2a_count:before{border:solid transparent;border-width:4px 4px 4px 0;content:"";height:0;left:0;line-height:0;margin:-4px 0 0 -4px;position:absolute;top:50%;width:0}.a2a_count:before{border-right-color:#ccc}.a2a_count:after{border-right-color:#fff;margin-left:-3px}.a2a_count span{animation:a2aFadeIn .14s ease-in}.a2a_vertical_style .a2a_counter img{display:block}.a2a_vertical_style .a2a_count{float:none;margin-left:0;margin-top:6px}.a2a_vertical_style .a2a_count:after,.a2a_vertical_style .a2a_count:before{border:solid transparent;border-width:0 4px 4px 4px;content:"";height:0;left:50%;line-height:0;margin:-4px 0 0 -4px;position:absolute;top:0;width:0}.a2a_vertical_style .a2a_count:before{border-bottom-color:#ccc}.a2a_vertical_style .a2a_count:after{border-bottom-color:#fff;margin-top:-3px}.a2a_color_buttons .a2a_count,.a2a_color_buttons .a2a_count:after,.a2a_color_buttons .a2a_count:before,.a2a_color_buttons.a2a_vertical_style .a2a_count:after,.a2a_color_buttons.a2a_vertical_style .a2a_count:before{background-color:transparent;border:none;color:#fff;float:none;width:auto}.a2a_color_buttons.a2a_vertical_style .a2a_count{margin-top:0}.a2a_flex_style{display:flex;align-items:flex-start;gap:0}.a2a_default_style.a2a_flex_style{left:0;right:0;width:100%}.a2a_vertical_style.a2a_flex_style{flex-direction:column;top:0;bottom:0}.a2a_flex_style a{display:flex;justify-content:center;flex:1;padding:4px}.a2a_flex_style.a2a_vertical_style a{flex-direction:column}.a2a_floating_style.a2a_color_buttons,.a2a_floating_style.a2a_flex_style{border-radius:0;padding:0}.a2a_floating_style.a2a_default_style.a2a_flex_style{bottom:0}.a2a_kit.a2a_flex_style .a2a_counter img,.a2a_kit.a2a_flex_style .a2a_dd,.a2a_kit.a2a_flex_style .a2a_svg{float:none}.a2a_nowrap{white-space:nowrap}.a2a_note{margin:0 auto;padding:9px;font-size:12px;text-align:center}.a2a_note .a2a_note_note{margin:0;color:#2A2A2A}.a2a_wide a{display:block;margin-top:3px;border-top:1px solid #EEE;text-align:center}.a2a_label{position:absolute!important;clip-path:polygon(0px 0px,0px 0px,0px 0px);-webkit-clip-path:polygon(0px 0px,0px 0px,0px 0px);overflow:hidden;height:1px;width:1px}.a2a_kit,.a2a_menu,.a2a_modal,.a2a_overlay{-ms-touch-action:manipulation;touch-action:manipulation;outline:0}.a2a_dd{-webkit-user-drag:none}.a2a_dd img{border:0}.a2a_button_facebook_like iframe{max-width:none}</style><style></style><script type="text/javascript" src="https://www.cell.com/products/marlin/cell/releasedAssets/public/mathjax-511ffcc670c4d93d695bf67802ba7aad/tex-mml-chtml.js" async=""></script><link href="https://crossmark-cdn.crossref.org/widget/v2.0/style.css" type="text/css" rel="stylesheet"><style type="text/css">.CtxtMenu_InfoClose {  top:.2em; right:.2em;}
.CtxtMenu_InfoContent {  overflow:auto; text-align:left; font-size:80%;  padding:.4em .6em; border:1px inset; margin:1em 0px;  max-height:20em; max-width:30em; background-color:#EEEEEE;  white-space:normal;}
.CtxtMenu_Info.CtxtMenu_MousePost {outline:none;}
.CtxtMenu_Info {  position:fixed; left:50%; width:auto; text-align:center;  border:3px outset; padding:1em 2em; background-color:#DDDDDD;  color:black;  cursor:default; font-family:message-box; font-size:120%;  font-style:normal; text-indent:0; text-transform:none;  line-height:normal; letter-spacing:normal; word-spacing:normal;  word-wrap:normal; white-space:nowrap; float:none; z-index:201;  border-radius: 15px;                     /* Opera 10.5 and IE9 */  -webkit-border-radius:15px;               /* Safari and Chrome */  -moz-border-radius:15px;                  /* Firefox */  -khtml-border-radius:15px;                /* Konqueror */  box-shadow:0px 10px 20px #808080;         /* Opera 10.5 and IE9 */  -webkit-box-shadow:0px 10px 20px #808080; /* Safari 3 & Chrome */  -moz-box-shadow:0px 10px 20px #808080;    /* Forefox 3.5 */  -khtml-box-shadow:0px 10px 20px #808080;  /* Konqueror */  filter:progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color="gray", Positive="true"); /* IE */}
</style><style type="text/css">.CtxtMenu_MenuClose {  position:absolute;  cursor:pointer;  display:inline-block;  border:2px solid #AAA;  border-radius:18px;  -webkit-border-radius: 18px;             /* Safari and Chrome */  -moz-border-radius: 18px;                /* Firefox */  -khtml-border-radius: 18px;              /* Konqueror */  font-family: "Courier New", Courier;  font-size:24px;  color:#F0F0F0}
.CtxtMenu_MenuClose span {  display:block; background-color:#AAA; border:1.5px solid;  border-radius:18px;  -webkit-border-radius: 18px;             /* Safari and Chrome */  -moz-border-radius: 18px;                /* Firefox */  -khtml-border-radius: 18px;              /* Konqueror */  line-height:0;  padding:8px 0 6px     /* may need to be browser-specific */}
.CtxtMenu_MenuClose:hover {  color:white!important;  border:2px solid #CCC!important}
.CtxtMenu_MenuClose:hover span {  background-color:#CCC!important}
.CtxtMenu_MenuClose:hover:focus {  outline:none}
</style><style type="text/css">.CtxtMenu_Menu {  position:absolute;  background-color:white;  color:black;  width:auto; padding:5px 0px;  border:1px solid #CCCCCC; margin:0; cursor:default;  font: menu; text-align:left; text-indent:0; text-transform:none;  line-height:normal; letter-spacing:normal; word-spacing:normal;  word-wrap:normal; white-space:nowrap; float:none; z-index:201;  border-radius: 5px;                     /* Opera 10.5 and IE9 */  -webkit-border-radius: 5px;             /* Safari and Chrome */  -moz-border-radius: 5px;                /* Firefox */  -khtml-border-radius: 5px;              /* Konqueror */  box-shadow:0px 10px 20px #808080;         /* Opera 10.5 and IE9 */  -webkit-box-shadow:0px 10px 20px #808080; /* Safari 3 & Chrome */  -moz-box-shadow:0px 10px 20px #808080;    /* Forefox 3.5 */  -khtml-box-shadow:0px 10px 20px #808080;  /* Konqueror */}
.CtxtMenu_MenuItem {  padding: 1px 2em;  background:transparent;}
.CtxtMenu_MenuArrow {  position:absolute; right:.5em; padding-top:.25em; color:#666666;  font-family: null; font-size: .75em}
.CtxtMenu_MenuActive .CtxtMenu_MenuArrow {color:white}
.CtxtMenu_MenuArrow.CtxtMenu_RTL {left:.5em; right:auto}
.CtxtMenu_MenuCheck {  position:absolute; left:.7em;  font-family: null}
.CtxtMenu_MenuCheck.CtxtMenu_RTL { right:.7em; left:auto }
.CtxtMenu_MenuRadioCheck {  position:absolute; left: .7em;}
.CtxtMenu_MenuRadioCheck.CtxtMenu_RTL {  right: .7em; left:auto}
.CtxtMenu_MenuInputBox {  padding-left: 1em; right:.5em; color:#666666;  font-family: null;}
.CtxtMenu_MenuInputBox.CtxtMenu_RTL {  left: .1em;}
.CtxtMenu_MenuComboBox {  left:.1em; padding-bottom:.5em;}
.CtxtMenu_MenuSlider {  left: .1em;}
.CtxtMenu_SliderValue {  position:absolute; right:.1em; padding-top:.25em; color:#333333;  font-size: .75em}
.CtxtMenu_SliderBar {  outline: none; background: #d3d3d3}
.CtxtMenu_MenuLabel {  padding: 1px 2em 3px 1.33em;  font-style:italic}
.CtxtMenu_MenuRule {  border-top: 1px solid #DDDDDD;  margin: 4px 3px;}
.CtxtMenu_MenuDisabled {  color:GrayText}
.CtxtMenu_MenuActive {  background-color: #606872;  color: white;}
.CtxtMenu_MenuDisabled:focus {  background-color: #E8E8E8}
.CtxtMenu_MenuLabel:focus {  background-color: #E8E8E8}
.CtxtMenu_ContextMenu:focus {  outline:none}
.CtxtMenu_ContextMenu .CtxtMenu_MenuItem:focus {  outline:none}
.CtxtMenu_SelectionMenu {  position:relative; float:left;  border-bottom: none; -webkit-box-shadow:none; -webkit-border-radius:0px; }
.CtxtMenu_SelectionItem {  padding-right: 1em;}
.CtxtMenu_Selection {  right: 40%; width:50%; }
.CtxtMenu_SelectionBox {  padding: 0em; max-height:20em; max-width: none;  background-color:#FFFFFF;}
.CtxtMenu_SelectionDivider {  clear: both; border-top: 2px solid #000000;}
.CtxtMenu_Menu .CtxtMenu_MenuClose {  top:-10px; left:-10px}
</style><style id="MJX-CHTML-styles">
mjx-container[overflow="scroll"][display] {
  overflow-x: auto;
  min-width: initial !important;
}

mjx-container[overflow="truncate"][display] {
  overflow-x: hidden;
  min-width: initial !important;
}

mjx-container[jax="CHTML"] {
  white-space: nowrap;
}

mjx-container [space="1"] {
  margin-left: .111em;
}

mjx-container [space="2"] {
  margin-left: .167em;
}

mjx-container [space="3"] {
  margin-left: .222em;
}

mjx-container [space="4"] {
  margin-left: .278em;
}

mjx-container [space="5"] {
  margin-left: .333em;
}

mjx-container [rspace="1"] {
  margin-right: .111em;
}

mjx-container [rspace="2"] {
  margin-right: .167em;
}

mjx-container [rspace="3"] {
  margin-right: .222em;
}

mjx-container [rspace="4"] {
  margin-right: .278em;
}

mjx-container [rspace="5"] {
  margin-right: .333em;
}

mjx-container [size="s"] {
  font-size: 70.7%;
}

mjx-container [size="ss"] {
  font-size: 50%;
}

mjx-container [size="Tn"] {
  font-size: 60%;
}

mjx-container [size="sm"] {
  font-size: 85%;
}

mjx-container [size="lg"] {
  font-size: 120%;
}

mjx-container [size="Lg"] {
  font-size: 144%;
}

mjx-container [size="LG"] {
  font-size: 173%;
}

mjx-container [size="hg"] {
  font-size: 207%;
}

mjx-container [size="HG"] {
  font-size: 249%;
}

mjx-container [width="full"] {
  width: 100%;
}

mjx-box {
  display: inline-block;
}

mjx-block {
  display: block;
}

mjx-itable {
  display: inline-table;
}

mjx-row {
  display: table-row;
}

mjx-row > * {
  display: table-cell;
}

mjx-container [inline-breaks] {
  display: inline;
}

mjx-mtext {
  display: inline-block;
  text-align: left;
}

mjx-mstyle {
  display: inline-block;
}

mjx-merror {
  display: inline-block;
  color: red;
  background-color: yellow;
}

mjx-mphantom {
  visibility: hidden;
}

mjx-math {
  display: inline-block;
  text-align: left;
  line-height: 0;
  text-indent: 0;
  font-style: normal;
  font-weight: normal;
  font-size: 100%;
  font-size-adjust: none;
  letter-spacing: normal;
  word-wrap: normal;
  word-spacing: normal;
  direction: ltr;
  padding: 1px 0;
}

mjx-container[jax="CHTML"][display="true"] {
  display: block;
  text-align: center;
  justify-content: center;
  margin: 1em 0;
}

mjx-container[jax="CHTML"][display="true"][width="full"] {
  display: flex;
}

mjx-container[jax="CHTML"][display="true"] mjx-math {
  padding: 0;
}

mjx-container[jax="CHTML"][justify="left"] {
  text-align: left;
  justify-content: left;
}

mjx-container[jax="CHTML"][justify="right"] {
  text-align: right;
  justify-content: right;
}

mjx-container[jax="CHTML"] mjx-break::after {
  content: " ";
  white-space: normal;
  font-family: MJX-BRK;
}

mjx-break[size="0"] {
  letter-spacing: -0.999em;
}

mjx-break[size="1"] {
  letter-spacing: -0.889em;
}

mjx-break[size="2"] {
  letter-spacing: -0.833em;
}

mjx-break[size="3"] {
  letter-spacing: -0.778em;
}

mjx-break[size="4"] {
  letter-spacing: -0.722em;
}

mjx-break[size="5"] {
  letter-spacing: -0.667em;
}

mjx-math[breakable] {
  display: inline;
}

mjx-mrow {
  display: inline-block;
  text-align: left;
}

mjx-linestack, mjx-mrow[break-top] {
  display: inline-table;
  width: 100%;
}

mjx-linestack[break-align="bottom"], mjx-mrow[break-top][break-align="bottom"] {
  display: inline-block;
}

mjx-linestack[break-align="middle"], mjx-mrow[break-top][break-align="middle"] {
  vertical-align: middle;
}

mjx-linestack[break-align="center"], mjx-mrow[break-top][break-align="center"] {
  vertical-align: middle;
}

mjx-linestack[breakable] {
  display: inline;
}

mjx-linestack[breakable] > mjx-linebox {
  display: inline;
}

mjx-linestack[breakable] > mjx-linebox::before {
  white-space: pre;
  content: "\A";
}

mjx-linestack[breakable] > mjx-linebox::after {
  white-space: normal;
  content: " ";
  letter-spacing: -.999em;
  font-family: MJX-BRK;
}

mjx-linestack[breakable] > mjx-linebox:first-of-type::before {
  display: none;
}

mjx-linestack[breakable] > mjx-linebox:last-of-type::after {
  display: none;
}

mjx-linebox {
  display: block;
}

mjx-linebox[align="left"] {
  text-align: left;
}

mjx-linebox[align="center"] {
  text-align: center;
}

mjx-linebox[align="right"] {
  text-align: right;
}

mjx-linestrut {
  display: inline-block;
  height: 1em;
  vertical-align: -.25em;
}

mjx-msubsup {
  display: inline-block;
  text-align: left;
}

mjx-script {
  display: inline-block;
  padding-right: .05em;
  padding-left: .033em;
}

mjx-script > mjx-spacer {
  display: block;
}

mjx-mi {
  display: inline-block;
  text-align: left;
}

mjx-c {
  display: inline-block;
  width: 0;
  text-align: right;
}

mjx-utext {
  display: inline-block;
  padding: .75em 0 .2em 0;
}

mjx-mn {
  display: inline-block;
  text-align: left;
}

mjx-mo {
  display: inline-block;
  text-align: left;
}

mjx-stretchy-h {
  display: inline-block;
}

mjx-stretchy-h > * {
  display: inline-block;
  width: 0;
}

mjx-stretchy-h > mjx-ext {
  /* IE */ overflow: hidden;
  /* others */ overflow: clip visible;
  width: 100%;
  border: 0px solid transparent;
  box-sizing: border-box;
  text-align: center;
}

mjx-stretchy-h > mjx-ext > mjx-c {
  transform: scalex(500);
  width: 0;
}

mjx-stretchy-v {
  display: inline-block;
  text-align: center;
}

mjx-stretchy-v > * {
  display: block;
  height: 0;
}

mjx-stretchy-v > mjx-ext {
  /* IE */ overflow: hidden;
  /* others */ overflow: visible clip;
  height: 100%;
  border: 0px solid transparent;
  box-sizing: border-box;
}

mjx-mark {
  display: inline-block;
  height: 0;
}

.MM-N {
  font-family: MJX-MM-ZERO, MJX-MM-N;
}

.MM-B {
  font-family: MJX-MM-ZERO, MJX-MM-B;
}

.MM-I {
  font-family: MJX-MM-ZERO, MJX-MM-I;
}

.MM-BI {
  font-family: MJX-MM-ZERO, MJX-MM-BI;
}

.MM-DS {
  font-family: MJX-MM-ZERO, MJX-MM-DS;
}

.MM-F {
  font-family: MJX-MM-ZERO, MJX-MM-F;
}

.MM-FB {
  font-family: MJX-MM-ZERO, MJX-MM-FB;
}

.MM-S {
  font-family: MJX-MM-ZERO, MJX-MM-S;
}

.MM-SB {
  font-family: MJX-MM-ZERO, MJX-MM-SB;
}

.MM-SS {
  font-family: MJX-MM-ZERO, MJX-MM-SS;
}

.MM-SSB {
  font-family: MJX-MM-ZERO, MJX-MM-SSB;
}

.MM-SSI {
  font-family: MJX-MM-ZERO, MJX-MM-SSI;
}

.MM-SSBI {
  font-family: MJX-MM-ZERO, MJX-MM-SSBI;
}

.MM-M {
  font-family: MJX-MM-ZERO, MJX-MM-M;
}

.MM-SO {
  font-family: MJX-MM-ZERO, MJX-MM-SO;
}

.MM-LO {
  font-family: MJX-MM-ZERO, MJX-MM-LO;
}

.MM-S3 {
  font-family: MJX-MM-ZERO, MJX-MM-S3;
}

.MM-S4 {
  font-family: MJX-MM-ZERO, MJX-MM-S4;
}

.MM-S5 {
  font-family: MJX-MM-ZERO, MJX-MM-S5;
}

.MM-S6 {
  font-family: MJX-MM-ZERO, MJX-MM-S6;
}

.MM-S7 {
  font-family: MJX-MM-ZERO, MJX-MM-S7;
}

.MM-MI {
  font-family: MJX-MM-ZERO, MJX-MM-MI;
}

.MM-C {
  font-family: MJX-MM-ZERO, MJX-MM-C;
}

.MM-CB {
  font-family: MJX-MM-ZERO, MJX-MM-CB;
}

.MM-OS {
  font-family: MJX-MM-ZERO, MJX-MM-OS;
}

.MM-OB {
  font-family: MJX-MM-ZERO, MJX-MM-OB;
}

.MM-V {
  font-family: MJX-MM-ZERO, MJX-MM-V;
}

.MM-LT {
  font-family: MJX-MM-ZERO, MJX-MM-LT;
}

.MM-RB {
  font-family: MJX-MM-ZERO, MJX-MM-RB;
}

.MM-EM {
  font-family: MJX-MM-ZERO, MJX-MM-EM;
}

@font-face /* MJX-MM-ZERO */ {
  font-family: MJX-MM-ZERO;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-zero.woff") format("woff");
}

@font-face /* MJX-BRK */ {
  font-family: MJX-BRK;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-brk.woff") format("woff");
}

@font-face /* MJX-MM-N */ {
  font-family: MJX-MM-N;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-n.woff") format("woff");
}

@font-face /* MJX-MM-B */ {
  font-family: MJX-MM-B;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-b.woff") format("woff");
}

@font-face /* MJX-MM-I */ {
  font-family: MJX-MM-I;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-i.woff") format("woff");
}

@font-face /* MJX-MM-BI */ {
  font-family: MJX-MM-BI;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-bi.woff") format("woff");
}

@font-face /* MJX-MM-DS */ {
  font-family: MJX-MM-DS;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-ds.woff") format("woff");
}

@font-face /* MJX-MM-F */ {
  font-family: MJX-MM-F;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-f.woff") format("woff");
}

@font-face /* MJX-MM-FB */ {
  font-family: MJX-MM-FB;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-fb.woff") format("woff");
}

@font-face /* MJX-MM-S */ {
  font-family: MJX-MM-S;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-s.woff") format("woff");
}

@font-face /* MJX-MM-SB */ {
  font-family: MJX-MM-SB;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-sb.woff") format("woff");
}

@font-face /* MJX-MM-SS */ {
  font-family: MJX-MM-SS;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-ss.woff") format("woff");
}

@font-face /* MJX-MM-SSB */ {
  font-family: MJX-MM-SSB;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-ssb.woff") format("woff");
}

@font-face /* MJX-MM-SSI */ {
  font-family: MJX-MM-SSI;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-ssi.woff") format("woff");
}

@font-face /* MJX-MM-SSBI */ {
  font-family: MJX-MM-SSBI;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-ssbi.woff") format("woff");
}

@font-face /* MJX-MM-M */ {
  font-family: MJX-MM-M;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-m.woff") format("woff");
}

@font-face /* MJX-MM-SO */ {
  font-family: MJX-MM-SO;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-so.woff") format("woff");
}

@font-face /* MJX-MM-LO */ {
  font-family: MJX-MM-LO;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-lo.woff") format("woff");
}

@font-face /* MJX-MM-S3 */ {
  font-family: MJX-MM-S3;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-s3.woff") format("woff");
}

@font-face /* MJX-MM-S4 */ {
  font-family: MJX-MM-S4;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-s4.woff") format("woff");
}

@font-face /* MJX-MM-S5 */ {
  font-family: MJX-MM-S5;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-s5.woff") format("woff");
}

@font-face /* MJX-MM-S6 */ {
  font-family: MJX-MM-S6;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-s6.woff") format("woff");
}

@font-face /* MJX-MM-S7 */ {
  font-family: MJX-MM-S7;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-s7.woff") format("woff");
}

@font-face /* MJX-MM-MI */ {
  font-family: MJX-MM-MI;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-mi.woff") format("woff");
}

@font-face /* MJX-MM-C */ {
  font-family: MJX-MM-C;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-c.woff") format("woff");
}

@font-face /* MJX-MM-CB */ {
  font-family: MJX-MM-CB;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-cb.woff") format("woff");
}

@font-face /* MJX-MM-OS */ {
  font-family: MJX-MM-OS;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-os.woff") format("woff");
}

@font-face /* MJX-MM-OB */ {
  font-family: MJX-MM-OB;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-ob.woff") format("woff");
}

@font-face /* MJX-MM-V */ {
  font-family: MJX-MM-V;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-v.woff") format("woff");
}

@font-face /* MJX-MM-LT */ {
  font-family: MJX-MM-LT;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-lt.woff") format("woff");
}

@font-face /* MJX-MM-RB */ {
  font-family: MJX-MM-RB;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-rb.woff") format("woff");
}

@font-face /* MJX-MM-EM */ {
  font-family: MJX-MM-EM;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-em.woff") format("woff");
}

mjx-c.mjx-c210E {
  padding: 0.694em 0.576em 0.011em 0;
}

mjx-c.mjx-c32 {
  padding: 0.666em 0.5em 0 0;
}

mjx-c.mjx-c1D436 {
  padding: 0.705em 0.76em 0.022em 0;
}

mjx-c.mjx-c2062 {
  padding: 0 0 0 0;
}

mjx-c.mjx-c1D443 {
  padding: 0.683em 0.754em 0 0;
}

mjx-c.mjx-c3BB {
  padding: 0.694em 0.563em 0.013em 0;
}

mjx-c.mjx-c31 {
  padding: 0.666em 0.5em 0 0;
}

mjx-c.mjx-c30 {
  padding: 0.666em 0.5em 0.022em 0;
}

mjx-c.mjx-c36 {
  padding: 0.666em 0.5em 0.022em 0;
}
</style></head>
<body class="pb-ui">
    <div class="skip">
        <a href="#page-body-id" id="skipNavigationLink" title="Skip to Main Content" tabindex="0" class="skipToContent">Skip to Main Content</a>
        <a href="#mega-menu-nav" id="skipToMainMenu" title="Skip to Main Menu" tabindex="0" class="skipToContent">Skip to Main Menu</a>
    </div>



    <div id="pb-page-content" data-ng-non-bindable="">
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="ux3-layout-widget" data-widget-id="832ad3f3-97ac-45ab-9af1-90c6f7bff54a" class="ux3">
        
    

        


    

            

            
                <div class="cell fonts-loaded">
            

            <div data-widget-id="832ad3f3-97ac-45ab-9af1-90c6f7bff54a" data-widget-name="ux3-layout-widget"><div>
    

        


    

            

            

            <div id="a1f1ba25-79af-4194-aaa9-66206104a800" class="widget loginPopup none widget-none widget-compact-all"><div class="wrapped"><div class="widget-body body body-none body-compact-all"><div role="dialog" aria-modal="true" aria-label="Login" class="popup login-popup hidden"><div class="content"><a href="#" aria-label="Close login popup" title="Close login popup" role="button" class="close"><i class="icon-close_thin"></i></a><h2 class="login-title">Login to your account</h2><div class="login-form"><p tabindex="-1" class="message error"></p><form action="/action/doLogin?redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2901446-X" method="post"><input type="hidden" name="redirectUri" value="https://www.cell.com/cell/fulltext/S0092-8674(21)01446-X"><input type="hidden" name="referer" value="/cell/fulltext/S0092-8674(21)01446-X"><input type="hidden" name="ecommRedirect"><input type="hidden" name="code" value="cell-site"><input type="hidden" name="UX3LoginWidget" value="true"><input type="hidden" name="isPopup" value="true"><div class="input-group login-input--wrapper"><div class="label"><label for="login">Email/Username</label></div><input id="login" type="text" name="login" value="" size="15" class="login"><div role="alert" class="error-hint js--hidden"><div class="error-hint__text"><div class="error-text">Your email address is a required field.</div><div class="email-example">E.g., j.smith@mail.com</div></div></div></div><div class="input-group login-input--wrapper"><div class="label"><label for="password">Password</label></div><input id="password" type="password" name="password" value="" autocomplete="off" class="password"><a href="#" class="login__field__action js-show-password">Show</a><div role="alert" class="error-hint js--hidden"><span class="error-hint__text">Your password is a required field.</span></div><div class="actions"><a href="https://www.cell.com/action/requestResetPassword" class="link show-request-reset-password">Forgot password?</a></div></div><div class="remember"><input type="checkbox" name="savePassword" id="a1f1ba25-79af-4194-aaa9-66206104a800-remember" value="1" checked="checked"><label for="a1f1ba25-79af-4194-aaa9-66206104a800-remember">Remember me</label></div><div class="submit"><input type="submit" name="submit" value="Log in" class="button submit primary"></div></form></div><div class="create-account"><div>Don’t have an account?</div><a href="/action/registration?redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2901446-X" class="show-registration">Create a Free Account</a></div></div></div></div></div></div>

            

        

    

        


    

            

            

            <div role="dialog" aria-modal="true" aria-label="Reset password" class="popup top-drawer request-reset-password-drawer hidden"><div class="content"><a href="#" aria-label="Close reset password popup" title="Close reset password popup" role="button" class="close"><i class="icon-close_thin"></i></a><div class="request-reset"><div class="form"><p class="sub">If you don't remember your password, you can reset it by entering your email address and clicking the Reset Password button. You will then receive an email that contains a secure link for resetting your password</p><div tabindex="-1" class="message error"></div><form action="/action/requestResetPassword" method="post" class="request-reset-password-form"><input type="hidden" name="platform"><input type="hidden" name="requestResetPassword" value="true"><input type="hidden" name="codeSite" value="cell-site"><div class="input-group"><div class="label"><label for="reset-password-email">Email*</label></div><input id="reset-password-email" type="text" name="email" value="" size="15" class="email"></div><div class="submit" disabled="disabled"><input type="submit" name="submit" value="Submit" disabled="disabled" class="button primary submit"></div></form></div><div class="success-template hidden"><p>If the address matches a valid account an email will be sent to __email__ with instructions for resetting your password</p></div></div><div class="request-reset-cancel"><a href="#" aria-label="Close reset password popup" title="Close reset password popup" class="cancel">Cancel</a></div></div></div>

            

        

    

        


    

            

            
                <div class="ux3-ads">
            

            <div class="pb-ad">
            <!-- Do Not Modify without contacting ELS-HSOnlineCampaign@elsevier.com -->
<!-- To be kept in sync with Global Commercial Publishing Operations-->
<!-- Start: GPT Async -->
<div id="gpt-Variables" data-adfile="CELLJOURNAL">
	<input type="hidden" id="gptSite" value="cell">
	<input type="hidden" id="gptLoggedIn" value="no">
	<input type="hidden" id="gptAccessType" value="ae:ANON_GUEST|ae:ANON_GUEST">
	<input type="hidden" id="gptDefaultSite" value="cell">
</div>
<div id="gpt-Interscroller" class="gpt-InScrollBox" style="display:none;">
	<div class="gpt-InScrollLabel">ADVERTISEMENT</div>
	<div class="gpt-InScrollContainer">
		<div class="gpt-InScrollWrapper">
			<div class="gpt-InScrollInner">
				<div class="gpt-adinterscroller" data-adsize="interscroller"></div>
			</div>
		</div>
	</div>
	<div class="gpt-InScrollLabel">SCROLL TO CONTINUE WITH CONTENT</div>
</div>
<div id="gpt-DebugDiv" class="gpt-DebugDiv" style="display:none;">
	<div style="width: 100%; display: table;">
		<div style="display: table-row">
			<div style="width: 200px; display: table-cell;">
				<button class="gpt-DebugBtn" onclick="gpt_ShowConsole()">Open GPT Console</button><br>
				<button id="gpt_GrapeShotDebugBtn" class="gpt-DebugBtn" onclick="gpt_GrapeShotDebug();">Open Oracle Keywords</button><br>
				<br>
				<button class="gpt-DebugBtn" onclick="gpt_LoadDebugValues()">Refresh Values</button>
			</div>
			<div style="display: table-cell;">
				<table class="gpt-DebugTable">
					<thead>
						<tr>
							<th>Property</th>
							<th>Value</th>
						</tr>
					</thead>
					<tbody>
						<tr><td class="gpt-DebugTitle">Status</td><td><div class="gptStatus_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Version</td><td><div class="gptVersion_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Ad File</td><td><div class="gptAdFile_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Disable Ads Flag</td><td><div class="gptDisableAds_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Environment</td><td><div class="gptTestENV_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Moat Init</td><td><div class="gptMoatInit_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Moat Ready</td><td><div class="gptMoatReady_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual Ready</td><td><div class="gptContextualReady_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual URL</td><td><div class="gptContextualURL_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual Initial Segments</td><td><div class="gptContextualInit_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual Used Segments</td><td><div class="gptContextualResult_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">AdUnit</td><td><div class="gptSite_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">SubAdUnit</td><td><div class="gptPage_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Custom Targeting</td><td><div class="gptCustomTargeting_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Ad Events</td><td><div class="gptEvent_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Invalid Ad Sizes</td><td><div class="gptInvalidAdSize_Debug"></div></td></tr>
					</tbody>
				</table>
			</div>
		</div>
	</div>
</div>
<style type="text/css">
	.gpt-DebugBtn {
		display: inline-block;
		text-align: center;
		text-decoration: none;
		margin: 2px 0;
		border: solid 1px #122b40;
		border-radius: 0.4em;
		padding: 0.5em 1em;
		color: #ffffff;
		background-color: #204d74;
		font-weight: normal;
		font-size: 12px;
	}

	.gpt-DebugDiv {
		margin: 5px;
	}

	.gpt-DebugTitle {
		font-weight: bold;
	}

	.gpt-DebugTable {
		border: 1px solid #dddddd;
		border-collapse: collapse;
		margin: 5px;
		font-size: 0.9em;
		font-family: sans-serif;
		min-width: 400px;
		margin-bottom: 25px;
	}

		.gpt-DebugTable td:first-child {
			padding: 5px;
		}

		.gpt-DebugTable thead tr th {
			background-color: #d9edf7;
			color: #ffffff;
			text-align: left;
			padding: 5px;
			border-bottom: 2px solid #204d74;
		}

		.gpt-DebugTable th,
		.gpt-DebugTable td {
			padding: 5px;
		}

		.gpt-DebugTable tbody tr {
			border-bottom: 1px solid #dddddd;
		}

			.gpt-DebugTable tbody tr:nth-of-type(even) {
				background-color: #f3f3f3;
			}

			.gpt-DebugTable tbody tr:last-of-type {
				border-bottom: 2px solid #204d74;
			}

	.gpt-InScrollLabel {
		background-color: rgb(0, 0, 0);
		color: rgb(255, 255, 255);
		position: relative;
		max-width: 100vw;
		left: 50%;
		right: 50%;
		margin-left: -50vw;
		margin-right: -50vw;
		padding: 0;
		text-align: center;
		width: 100vw !important;
		font-size: 10px;
	}

	.gpt-InScrollBox {
		height: 100vh !important;
		width: 100vw;
		cursor: pointer;
		max-width: 100vw;
		position: relative;
		left: 50%;
		right: 50%;
		margin-left: -50vw;
		margin-right: -50vw;
		margin-bottom: 50px;
		padding-top: 5px;
		padding-bottom: 5px;
		z-index: 10001;
		background-color: white;
	}

		.gpt-InScrollBox .gpt-InScrollContainer {
			height: 95vh !important;
			position: relative;
		}

		.gpt-InScrollBox .gpt-InScrollWrapper {
			position: absolute !important;
			top: 0 !important;
			left: 0;
			width: 100%;
			height: 95vh !important;
			border: 0 !important;
			margin: 0 !important;
			padding: 0 !important;
			clip: rect(0, auto, auto, 0) !important;
			-webkit-clip-path: polygon(0px 0px, 100% 0px, 100% 100%, 0px 100%) !important;
			clip-path: polygon(0px 0px, 100% 0px, 100% 100%, 0px 100%) !important;
		}

		.gpt-InScrollBox .gpt-InScrollInner {
			position: fixed !important;
			top: 0 !important;
			left: 0 !important;
			bottom: 0;
			margin: auto;
			width: 100vw;
			height: 95vh;
			-webkit-transform: translateZ(0) !important;
			display: -webkit-box;
			display: -ms-flexbox;
			display: flex;
			-webkit-box-orient: vertical;
			-webkit-box-direction: normal;
			-ms-flex-direction: column;
			flex-direction: column;
			-webkit-box-align: center;
			-ms-flex-align: center;
			align-items: center;
			-webkit-box-pack: center;
			-ms-flex-pack: center;
			justify-content: center;
			text-align: center;
		}
</style>
<script class="optanon-category-4" type="text/plain">
	//--------------------------------------------------------------------------
	var gpt_Version = '04072023';
	var gpt_AdFile = $('#gpt-Variables').data('adfile');
	var gpt_MaxMOATInitAttempts = 11;
	var gpt_MaxMOATReadyAttempts = 11;
	var gpt_MaxContextualReadyAttempts = 21;
	var gpt_MOATInitAttempts = 0;
	var gpt_MOATReadyAttempts = 0;
	var gpt_ContextualReadyAttempts = 0;
	var gpt_IsProd = true;
	var gpt_StartTime;
	var gs_channels = 'DEFAULT';
	var gpt_ContextualReady = false;
	var gpt_ContextualURL = 'Not enabled outside of Production';
	var gpt_DisableAdsFlagFound = false;

	var gptEvent_Debug = '';
	var gptCustomTargeting_Debug = '';
	var gptSite_Debug = "";
	var gptPage_Debug = "";
	var gptContextualEnabled = false;
	var gptContextualResult_Debug = '';
	var gptContextualInit_Debug = '';
	var gptContextualURL_Debug = '';
	var gptContextualReady_Debug = '';
	var gptMoatReady_Debug = '';
	var gptMoatInit_Debug = '';
	var gptTestENV_Debug = '';
	var gptDisableAds_Debug = '';
	var gptStatus_Debug = 'GPT Still Loading - Refresh Values';
	var gptInvalidAdSize_Debug = '';
	//--------------------------------------------------------------------------
	$(document).ready(function () {
		//Check to see if we are on a development env
		gpt_CheckEnvironment();

		if (!gpt_DisableAdsFlagFound) {
			gpt_StartTime = new Date();
			if (gpt_AdFile !== "PU") {
				//Once the dom has loaded, begin the process to init the ads.
				gpt_InitScripts();
				gpt_CheckForMOATInit();
			}
		}
	});
</script>
<script type="text/javascript">
	function gpt_CheckEnvironment() {
		if ($('#gptDisableAds').length > 0) {
			gptDisableAds_Debug = 'Flag Detected - Ads Disabled';
			gpt_DisableAdsFlagFound = true;
		}
		else {
			gptDisableAds_Debug = 'Ads Enabled';
			gpt_DisableAdsFlagFound = false;
		}
		switch (gpt_AdFile) {
			case 'PU':
				gptContextualEnabled = true;
				if ($("#Environment").val() !== "Prod") {
					gpt_IsProd = false;
					gptTestENV_Debug = 'Test Environment Detected';
				}
				else {
					gptTestENV_Debug = 'Production Environment Detected';
				}
				break;
			case 'JBS':
			case 'LANCET':
				gptContextualEnabled = true;
				if ($(".gpt-ad").length > 0) {
					//Other ads exist, allow the Prestitial
					$(document.body).append("<div class='gpt-ad' data-adsize='prestitial'></div>");
				}
				if (window.location.hostname.toLowerCase().indexOf('.literatumonline.') >= 0) {
					gpt_IsProd = false;
					gptTestENV_Debug = 'Test Environment Detected';
				}
				else {
					gptTestENV_Debug = 'Production Environment Detected';
				}
				break;
			default:
				//Cell
				gptContextualEnabled = false;
				if (window.location.hostname.toLowerCase().indexOf('.literatumonline.') >= 0) {
					gpt_IsProd = false;
					gptTestENV_Debug = 'Test Environment Detected';
				}
				else {
					gptTestENV_Debug = 'Production Environment Detected';
				}
				break;
		}
	}
	//--------------------------------------------------------------------------
    function gpt_GetNonCMEArticle() {
        var url;
        $('.articleCitation').each(function (i, obj) {
            if ($(obj).find('.toc__item__detials').length > 0) {
                var itemDetails = $(obj).first().find('.toc__item__detials');
                var cmeFound = 0;
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='CME']").length; }
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='(CME)']").length; }
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='Education']").length; }
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='education']").length; }
                if (cmeFound == 0) {
                    var itemLink = $(itemDetails).first().find('.toc__item__title');
                    var linkCount = $(itemLink).first().find("a:contains('CME')").length;
                    if (linkCount == 0) {
                        url = $(itemLink).first().find('a').attr('href');
                        return false;   //Found a URL, break the loop
                    }
                }
            }
        });
        return url;
    }
	//--------------------------------------------------------------------------
	function gpt_GetElapsedTime() {
		var endTime = new Date();
		var timeDiff = endTime - gpt_StartTime; //in ms
		// strip the ms
		timeDiff /= 1000;

		// get seconds
		return seconds = Math.round(timeDiff) + " seconds";
	}
	//--------------------------------------------------------------------------
	function gpt_CheckSSC() {
		//Check for ScreenShot Client
		var gpt_SSC = gpt_GetQuerystring('gpt_ssc', '');
		if (gpt_SSC == '1') {
			//it's the ScreenShotClient, disable MOAT IVT Detection
			$('#gpt-Variables').append('<input type="hidden" id="gptm_data">');
			$('#gptm_data').val(0);
			gpt_Target("m_data");
		}
	}
	//--------------------------------------------------------------------------
	function gpt_LocalSS(adTypeId) {
		window.scrollTo(0, 0);
		gpt_Sleep(1000);
		gpt_ScrollToAd(adTypeId);
		gpt_PreTearsheet(adTypeId);
	}
	//--------------------------------------------------------------------------
	function gpt_InitScripts() {
		if (gpt_AdFile == "PU") {
			if (gpt_IsProd) {
				gpt_ContextualURL = gpt_CheckForNULL($('#canonicalurl').val());
				if (gpt_ContextualURL === null) {
					gpt_ContextualURL = window.location.href.split('?')[0].split('#')[0];
				}
				else {
					gpt_ContextualURL = window.location.origin + gpt_ContextualURL;
				}
				var useSSL = 'https:' == document.location.protocol;
				var gs = document.createElement('script');
				gs.async = true;
				gs.type = 'text/javascript';
				gs.src = (useSSL ? 'https:' : 'http:') + '//elsevier.gscontxt.net/multizone/channels.cgi?url=' + encodeURIComponent(gpt_ContextualURL);
				gs.onload = function () {
					gpt_ContextualReady = true;
				};
				document.head.appendChild(gs);
			}
		}
		else {
			//JBS-Cell-Lancet
			var useSSL = 'https:' == document.location.protocol;

			var gads = document.createElement('script');
			gads.async = true;
			gads.type = 'text/javascript';
			gads.src = (useSSL ? 'https:' : 'http:') + '//securepubads.g.doubleclick.net/tag/js/gpt.js';
			document.head.appendChild(gads);

			if (gpt_IsProd && gptContextualEnabled) {
				gpt_ContextualURL = window.location.href.split('?')[0].split('#')[0];
				var gs = document.createElement('script');
				gs.async = true;
				gs.type = 'text/javascript';
				gs.src = (useSSL ? 'https:' : 'http:') + '//elsevier.gscontxt.net/multizone/channels.cgi?url=' + encodeURIComponent(gpt_ContextualURL);
				gs.onload = function () {
					gpt_ContextualReady = true;
				};
				document.head.appendChild(gs);
			}

			var elsevierMoatHeader = document.createElement('link');
			elsevierMoatHeader.rel = 'preload';
			elsevierMoatHeader.as = 'script';
			elsevierMoatHeader.src = (useSSL ? 'https:' : 'http:') + '//z.moatads.com/elsevierheader150204004183/moatheader.js';
			document.head.appendChild(elsevierMoatHeader);

			var elsevierMoatAds = document.createElement('link');
			elsevierMoatAds.rel = 'preconnect';
			elsevierMoatAds.src = (useSSL ? 'https:' : 'http:') + '//mb.moatads.com';
			document.head.appendChild(elsevierMoatAds);

			var moatHeader = document.createElement('script');
			moatHeader.async = true; gads.type = 'text/javascript';
			moatHeader.src = (useSSL ? 'https:' : 'http:') + '//z.moatads.com/elsevierheader150204004183/moatheader.js';
			document.head.appendChild(moatHeader);
		}
	}
	//--------------------------------------------------------------------------
	async function gpt_CheckForMOATInit() {
		//Check every 1/10 of a second until MOAT is Loaded or 1 second timeout
		if (gpt_MOATInitAttempts < gpt_MaxMOATInitAttempts) {
			gpt_MOATInitAttempts += 1;
			if (typeof window.moatPrebidApi === "object") {
				//MOAT LOADED
				gptMoatInit_Debug = '0.0' + gpt_MOATInitAttempts + ' seconds';
				gpt_CheckForMOATReady();
			}
			else {
				setTimeout(() => { gpt_CheckForMOATInit(); }, 100);
			}
		}
		else {
			//TIMEOUT: MOAT Not LOADED
			gptMoatInit_Debug = 'Timeout Expired';
			var googletag = googletag || {};
			googletag.cmd = googletag.cmd || [];
			gpt_CheckForContextualReady();
		}
	}
	//--------------------------------------------------------------------------
	function gpt_CheckForMOATReady() {
		//Check every 1/10 of a second until MOAT is Ready or 1 second timeout
		if (gpt_MOATReadyAttempts < gpt_MaxMOATReadyAttempts) {
			gpt_MOATReadyAttempts += 1;
			if (window.moatPrebidApi.safetyDataAvailable()) {
				//MOAT Ready
				gptMoatReady_Debug = '0.0' + gpt_MOATReadyAttempts + " seconds";
				gpt_CheckForContextualReady();
			}
			else {
				setTimeout(() => { gpt_CheckForMOATReady(); }, 100);
			}
		}
		else {
			//TIMEOUT: MOAT Not Ready
			gptMoatReady_Debug = 'Timeout Expired';
			gpt_CheckForContextualReady();
		}
	}
	//--------------------------------------------------------------------------
	function gpt_CheckForContextualReady() {
		//Check every 1/10 of a second until Contextual is Ready or 2 second timeout
		if (gpt_IsProd && gptContextualEnabled) {
			//Enabled
			if (gpt_ContextualReadyAttempts < gpt_MaxContextualReadyAttempts) {
				gpt_ContextualReadyAttempts += 1;
				if (gpt_ContextualReady) {
					//Contextual Ready
					gptContextualURL_Debug = gpt_ContextualURL;
					gptContextualReady_Debug = '0.0' + gpt_ContextualReadyAttempts + " seconds";
					gptContextualInit_Debug = gs_channels;
					gpt_InitAds();
				}
				else {
					setTimeout(() => { gpt_CheckForContextualReady(); }, 100);
				}
			}
			else {
				//LOG TIMEOUT error
				//REMOVE FOR NOW (SHOULD LOG TO A DIFF LOCATION)
				//TIMEOUT: Contextual Not Ready
				$('#gpt_GrapeShotDebugBtn').remove();
				gptContextualURL_Debug = gpt_ContextualURL;
				gptContextualReady_Debug = 'Timeout Expired';
				gptContextualInit_Debug = "";
				gpt_InitAds();
			}
		}
		else {
			//Not Enabled
			$('#gpt_GrapeShotDebugBtn').remove();
			switch (gpt_AdFile) {
				case 'PU':
				case 'LANCET':
				case 'JBS':
					gptContextualURL_Debug = 'Not enabled outside of Production';
					gptContextualReady_Debug = 'Not enabled outside of Production';
					gptContextualInit_Debug = 'Not enabled outside of Production';
					break;
				case "CELLJOURNAL":
				case "CELLBUCKET":
					gptContextualReady_Debug = 'Not enabled on Cell';
					gptContextualInit_Debug = 'Not enabled on Cell';
					gptContextualURL_Debug = 'Not enabled on Cell';
					break;
			}
			gpt_InitAds();
		}
	}
	//--------------------------------------------------------------------------
	function GPT_GetPageByURL() {
		var host = window.location.host.toLocaleLowerCase()
		var localpath = window.location.pathname.toLocaleLowerCase();

		///Check for Cell.com
		if (host.indexOf('cell.com') >= 0) {
			return null;
		}

		///Check for Cell.com (marlin-stag)
		if (host.indexOf('www-cell-com') >= 0) {
			return null;
		}

		///Check for Lancet
		if (host.indexOf('thelancet') >= 0) {
			return null;
		}

		if (localpath.indexOf('/article/') >= 0) {
			return 'article.fulltext';
		}

		if (localpath.indexOf('/action/dosearch') >= 0) {
			return 'search.searchResults';
		}

		if (localpath.indexOf('/content/') >= 0) {
			return 'periodical.editorial';
		}

		return 'periodical.home';
	}
	//--------------------------------------------------------------------------
	function gpt_CheckForNULL(x, valueifnull) {
		if (typeof valueifnull == "undefined") {
			valueifnull = null;
		}

		if (typeof x == 'undefined') {
			return valueifnull;
		}
		if (x == null) {
			return valueifnull;
		}
		if (x == "null") {
			return valueifnull;
		}
		if ($.trim(x) == '') {
			return valueifnull;
		}

		return x;
	}
	//--------------------------------------------------------------------------
	function gpt_GetAdditionalTargeting() {
		//adds page level targeting of custom variables
		//the id (minus gpt) and the value will be used as a key-value pair
		//eg: <input type="hidden" id="gptissuepii" value="x" class="gpt-Target">
		//eg: target issuepii=x
		$(".gpt-Target").each(function (index) {
			var gptTargetId = $(this).attr('id');
			gptTargetId = gptTargetId.replace('gpt', '').toLowerCase();
			if (gptTargetId !== '') {
				gpt_Target(gptTargetId);
			}
		})
	}
	//--------------------------------------------------------------------------
	function gpt_GetQuerystring(key, default_) {
		//function to read querystring variables
		if (default_ === null) default_ = '';
		key = key.replace(/[\[]/, "\\\[").replace(/[\]]/, "\\\]");
		var regex = new RegExp("[\\?&]" + key + "=([^&#]*)");
		var qs = regex.exec(window.location.href);
		if (qs === null)
			return default_;
		else
			return qs[1];
	}
	//--------------------------------------------------------------------------
	function gpt_GetDFPTestId() {
		//Get any request for a test id from querystring and set custom targeting parameter
		var gptDFPTestId = gpt_GetQuerystring('dfptestid', '');
		if (gptDFPTestId !== '') {
			$('#gpt-Variables').append('<input type="hidden" id="gptDFPTestId">');
			$('#gptDFPTestId').val(gptDFPTestId);
			gpt_Target("DFPTestId");
		}
	}
	//--------------------------------------------------------------------------
	function gpt_Target(targetName) {
		//Set page level target if value is found
		if ($('#gpt' + targetName).length > 0) {
			var targetValue = $('#gpt' + targetName).val();

			if (targetValue !== '') {
				targetValue = targetValue.replace(/[^-a-z0-9]/ig, '');	//Only Allow a-z, 0-9
				googletag.pubads().setTargeting(targetName.toLowerCase(), targetValue.toLowerCase());
				gptCustomTargeting_Debug = gptCustomTargeting_Debug + targetName.toLowerCase() + ' = ' + targetValue.toLowerCase() + '<br>';
			}
		}
	}
	//--------------------------------------------------------------------------
	function gpt_TargetAccessType() {
		var currentAccessType = $('#gptAccessType').val();
		if (typeof currentAccessType == "undefined") {
			currentAccessType = "";
		}
		if (currentAccessType.indexOf('ANON_IP') >= 0) {
			googletag.pubads().setTargeting("loggedin", "yes");
		}
		else {
			gpt_Target("LoggedIn");
		}
	}
	//--------------------------------------------------------------------------
	async function gpt_PreTearsheet(adTypeId) {
		if (typeof adTypeId === "undefined") {
			adTypeId = 0;
		}
		//Remove Elsevier Cookie Policy Notice
		if ($("#onetrust-consent-sdk").length > 0) {
			$("#onetrust-consent-sdk").remove();
		}

        //Mobile Nav
        $('#skip').remove();
        $('skipNavigationLink').remove();

		//JBS Modals
		if ($('.leadinModal').length > 0) {
			$('.leadinModal').remove();
		}
		if ($('.leadinModal').length > 0) {
			$('.leadinModal').remove();
		}
		if ($('.usabilla__overlay').length > 0) {
			$('.usabilla__overlay').remove();
		}

		//Pendo
		if ($('._pendo-step-container').length > 0) {
			$('._pendo-step-container').remove();
		}
		if ($('._pendo-guide-tt_').length > 0) {
			$('._pendo-guide-tt_').remove();
		}
		if ($('._pendo-image').length > 0) {
			$('._pendo-image').remove();
		}
		if ($('._pendo-badge-image').length > 0) {
			$('._pendo-badge-image').remove();
		}

		switch (gpt_AdFile) {
			case "CELLJOURNAL":
			case "CELLBUCKET":
				//Mobile Bug Fix on Cell
				if (parseInt(adTypeId) == 5) {
					$('.widget-horizontalAd').css('position', 'static');
					$('.body-horizontalAd').css('display', 'block');
				}
				break;
			case "JBS":
			case "LANCET":
				objLeaderboard = ".container-leaderboard";
				offset = 53;
				switch (parseInt(adTypeId)) {
					case 3: //MPU
					case 4: //SkyScraper
						//BUGFIX: Show Sidebar ad for JBS
						//skyscraper & mpu
						if ($('.widget-verticalAd').length > 0) {
							$('.widget-verticalAd').css('display', 'block');
						}
						$('#backgroundLayer').css('display', 'none');   //Hide any Prestitial
						break;
					case 6:     //Prestitial
						$('#backgroundLayer').css('display', 'block');
						break;
					default:
						$('#backgroundLayer').css('display', 'none');   //Hide any Prestitial
						break;
				}
				break;
		}
	}
	//--------------------------------------------------------------------------
	function gpt_Sleep(ms) {
		return new Promise(resolve => setTimeout(resolve, ms));
	}
	//--------------------------------------------------------------------------
	async function gpt_ScrollToAd(adTypeId) {
		if (typeof adTypeId === "undefined") {
			adTypeId = 0;
		}
		var adId = "";
		var offset = 0;
		var objLeaderboard = "";
		var gpt_ScrollToAd = true;

		switch (gpt_AdFile) {
			case "PU": //PracticeUpdate
				var hideOtherAds = true;
				$('#modalPrestitial').modal('hide');
				$('.jumbotron-welcome').hide();
				objLeaderboard = ".container-leaderboard";
				offset = 53;
				switch (parseInt(adTypeId)) {
					case 2:
						//LeaderBoard, no Scroll Needed
						gpt_ScrollToAd = false;
						adId = "adSlot1";
						break;
					case 3:
						//MPU
						adId = "adSlot2";
						break;
					case 4:
						//SkyScraper
						adId = "adSlot3";
						$('.most-read').hide();
						break;
					case 5:
						//Mobile, no Scroll Needed
						gpt_ScrollToAd = false;
						adId = "adSlot1";
						break;
					case 6:
						//Interstitial
						gpt_ScrollToAd = false;
						adId = "adSlot15";
						await gpt_Sleep(1000);
						$(objLeaderboard).hide();
						$('#modalPrestitial').unbind();
						$('#modalPrestitial').modal();
						break;
					case 7:
						//Native Ad
						break;
					case 10:
						//Master Companion
						hideOtherAds = false;
						gpt_ScrollToAd = false;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						adId = "adSlot1";
						break;
				}
				if (hideOtherAds) {
					//Hide all other ads
					$('.ad-container').each(function (i, obj) {
						if (this.id !== adId) {
							var tempParent = this.id;
							tempParent = tempParent.replace("adSlot", "adParent");
							$("#" + tempParent).addClass('hide');
						}
					});
				}
				break;
			case "JBS":
				objLeaderboard = ".js__adToHide";
				offset = 80;
				switch (parseInt(adTypeId)) {
					case 2:
						//Leaderboard
						gpt_ScrollToAd = false;
						$(".widget-verticalAd").hide();
						break;
					case 5:
						//Mobile
						//Scroll to Top
						gpt_ScrollToAd = false;
						break;
					case 3: //MPU
					case 4: //SkyScraper
						$(objLeaderboard).hide();
						adId = $('.gpt-ad-defined[data-adsize="sidebar"').attr('id');
						break;
					case 10:
						//MasterCompanion
						gpt_ScrollToAd = false;
						break;
					case 13:	    //Phone - Portrait
					case 14:        //Phone - Landscape
						$(objLeaderboard).hide();
						offset = 60;
						adId = 'gpt-Interscroller';
						gpt_ScrollToAd = true;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						break;
				}
				break;
			case "CELLJOURNAL":
			case "CELLBUCKET":
				objLeaderboard = ".js__adToHide";
				offset = 260;
				switch (parseInt(adTypeId)) {
					case 2:
						//Leaderboard
						gpt_ScrollToAd = false;
						$('.body-verticalAd').hide();
						break;
					case 5:
						//Mobile
						//Scroll to Top
						gpt_ScrollToAd = false;
						break;
					case 3: //MPU
					case 4: //SkyScraper
						$(objLeaderboard).hide();
						adId = $('.gpt-ad-defined[data-adsize="boombox"').attr('id');
						break;
					case 10:
						//MasterCompanion
						gpt_ScrollToAd = false;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						break;
				}
				//Hide the second ad if found
				var tCount = 0;
				$(".widget.literatumAd.none.GPT-ad-placeholder.widget-horizontalAd.widget-compact-all").each(function (index) {
					tCount += 1;
					if (tCount !== 1) {
						$(this).hide();
					}
				});
				break;
			case "LANCET":
				objLeaderboard = ".js__adToHide";
				offset = 80;
				switch (parseInt(adTypeId)) {
					case 2:
						//Leaderboard
						gpt_ScrollToAd = false;
						$('.body-verticalAd').hide();
						break;
					case 5:
						//Mobile
						//Scroll to Top
						gpt_ScrollToAd = false;
						break;
					case 3: //MPU
					case 4: //SkyScraper
						$(objLeaderboard).hide();
						adId = $('.gpt-ad-defined[data-adsize="skyscraper"').attr('id');
						break;
					case 10:
						//MasterCompanion
						gpt_ScrollToAd = false;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						break;
				}
				break;
		}

		//Scroll to Top
		var gpt_container = $("html,body");
		gpt_container.animate({ scrollTop: 0, scrollLeft: 0 });

		if (gpt_ScrollToAd) {
			//If Adexists, scroll to it
			var gpt_scrollTo = $('#' + adId);
			if (gpt_scrollTo.length > 0) {
				//Scroll to the Ad
				var gpt_topPosition = gpt_scrollTo.offset().top - gpt_container.offset().top + gpt_container.scrollTop();
				if (gpt_topPosition > 0) {
					gpt_container.animate({ scrollTop: gpt_topPosition, scrollLeft: 0 });
					if (gpt_topPosition - offset > 0 && offset > 0) {
						gpt_container.animate({ scrollTop: gpt_topPosition - offset, scrollLeft: 0 });
					}
				}
			}
		}
	}
	//--------------------------------------------------------------------------
	async function gpt_CheckForCookiePolicy() {
		var returnVal = false;
		if ($('#onetrust-accept-btn-handler').length > 0) {
			$('#onetrust-accept-btn-handler').click();
			returnVal = true;
		}
		return returnVal;
	}
	//--------------------------------------------------------------------------
	function gpt_InitAds() {
		var gpt_ResponsiveHomepage = false;

		switch (gpt_AdFile) {
			case "JBS":
			case "CELLJOURNAL":
			case "CELLBUCKET":
				//Responsive Check
				if ($('#hdrHomeBrand').length > 0) {
					gpt_ResponsiveHomepage = true;
				}
				break;
		}

		//InterScroller Check
		if (gpt_AdFile === "JBS") {
			var gpt_InterScrollerEnabled = false;
			var gpt_InterScrollerInsertObj;

			if ($("#articleHeader").length > 0) {
				//It's an article Page
				if ($('.article__sections > section').length > 0) {
					gpt_InterScrollerEnabled = true;
					gpt_InterScrollerInsertObj = $(".article__sections > section");
				}
			}
			else {
				if ($('.toc__section').length > 0) {
					//It's a TOC Page
					gpt_InterScrollerEnabled = true;
					gpt_InterScrollerInsertObj = $(".toc__section").first();
				}
			}

			//Add the Interscroller
			if (gpt_InterScrollerEnabled) {
				gpt_InterScrollerInsertObj.first().append($("#gpt-Interscroller").clone());
				$("#gpt-Interscroller").remove();
				$(".gpt-adinterscroller").addClass("gpt-ad");
			}
		}

		googletag.cmd.push(function () {
			var gptBoomBoxSizeMapping;
			var gptFluidMapping;
			var gptHomePageSidebarMapping;
			var gptCellHomePageSidebarMapping;
			var gptInterscrollerMapping;
			var gptLeaderboardSizeMapping;
			var gptMobileSizeMapping;
			var gptPrestitialMapping;
			var gptRectangleShortSizeMapping;
			var gptRectangleSizeMapping;
			var gptSideBarSizeMapping;
			var gptTallSizeMapping;
			var gptWideSizeMapping;
			var gptWideRectangleSizeMapping;

			//***********************************************
			//Start Define Size mapping based on client Viewport
			//***********************************************
			if (gpt_AdFile === 'PU') {
				gptTallSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[300, 600], [160, 600], [120, 600]]).
					addSize([0, 0], []).
					build();

				gptRectangleSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[300, 250]]).
					addSize([0, 0], []).
					build();

				gptRectangleShortSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[300, 250], [320, 50], [300, 50]]).
					addSize([0, 0], [[300, 250], [320, 50], [300, 50]]).
					build();

				gptWideSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[728, 90]]).
					addSize([760, 0], [[728, 90], [320, 50], [300, 50]]).
					addSize([0, 0], [[320, 50], [300, 50]]).
					build();

				gptWideRectangleSizeMapping = googletag.sizeMapping().
					addSize([992, 0], []).
					addSize([760, 0], [[320, 50], [300, 50], [300, 250]]).
					addSize([0, 0], [[320, 50], [300, 50], [300, 250]]).
					build();

				gptMobileSizeMapping = googletag.sizeMapping().
					addSize([0, 0], [[320, 50], [300, 50]]).
					build();

				gptFluidMapping = googletag.sizeMapping().
					addSize([0, 0], ['fluid']).
					build();

				gptPrestitialMapping = googletag.sizeMapping().
					addSize([760, 0], [[1, 1]]).
					build();
			}
			else {
				//JBS-CELL-LANCET
				gptLeaderboardSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[728, 90]]).
					addSize([768, 0], [[728, 90]]).
					addSize([0, 0], [[320, 50], [300, 50]]).
					build();

				gptHomePageSidebarMapping = googletag.sizeMapping().
					addSize([2090, 795], [[300, 600], [300, 250], [160, 600], [120, 600]]).
					addSize([1910, 795], [[160, 600], [120, 600]]).
                    addSize([1740, 795], [[120, 600]]).
					addSize([0, 0], []).
					build();

				gptCellHomePageSidebarMapping = googletag.sizeMapping().
                    addSize([2180, 795], [[300, 600], [160, 600], [120, 600], [300, 250], [336, 280]]).
					addSize([2180, 445], [[336, 280]]).
                    addSize([2090, 795], [[300, 600], [160, 600], [120, 600], [300, 250]]).
                    addSize([2090, 445], [[300, 250]]).
					addSize([1830, 795], [[160, 600], [120, 600]]).
                    addSize([1740, 795], [[120, 600]]).
					addSize([0, 0], []).
					build();

				gptSideBarSizeMapping = googletag.sizeMapping().
					addSize([1140, 795], [[300, 600], [160, 600], [120, 600], [300, 250]]).
					addSize([1140, 445], [[300, 250]]).
					addSize([0, 0], []). 				
					build();

				gptBoomBoxSizeMapping = googletag.sizeMapping().
					addSize([768, 0], [[300, 250], [336, 280]]).
					addSize([0, 0], []). 						
					build();

				gptFluidMapping = googletag.sizeMapping().
					addSize([0, 0], ['fluid']).
					build();

				gptPrestitialMapping = googletag.sizeMapping().
					addSize([600, 700], [[1, 1]]).
					addSize([0, 0], []).
					build();

				gptInterscrollerMapping = googletag.sizeMapping().
					addSize([1140, 445], []).
					addSize([1024, 768], [[1024, 768]]).
					addSize([768, 1024], [[768, 1024]]).
					addSize([480, 320], [[480, 320]]).
					addSize([320, 480], [[320, 480]]).
					build();
			}
			//***********************************************
			//End Define Size mapping based on client Viewport
			//***********************************************

			//***********************************************
			//Start detection for ad slots on page
			//***********************************************
			var gptMasterPage = gpt_CheckForNULL($('#gptPage').val());
			var gptMasterSite;

			if (gpt_AdFile === 'PU') {
				gptMasterSite = $('#gptSite').val();
			}
			else {
				//JBS-CELL-LANCET
				gptMasterSite = "dev";

				if (gpt_IsProd) {
					gptMasterSite = gpt_CheckForNULL($('#gptSite').val());

					//If there is no master site, check for a defaultsite
					if (gptMasterSite == null) {
						gptMasterSite = gpt_CheckForNULL($('#gptDefaultSite').val());
					}
				}
				else {
					//Dev ENV
					//If the page has no gptSite Value, assume no value which will disable ads
					if (gpt_CheckForNULL($('#gptSite').val()) === null) {
						gptMasterSite = null;
					}
				}

				//If there is no master Page, check for a Page using the URL
				if (gptMasterPage === null) {
					gptMasterPage = GPT_GetPageByURL();
				}
			}

			$(".gpt-ad").each(function (index) {
				var gptPage = '';
				var adSizeMapping = '';
				var gptPOS;
				var gptTC;
				var gptNative;
				var adId;
				var gptSite;
				var tempSlot = null;


                if (gpt_IsProd) {
					if (gpt_AdFile !== "CELLJOURNAL") {
						//Check if the ad slot has it's own site defined
						if (typeof $(this).data('site') === 'undefined') {
							gptSite = gptMasterSite;
						}
						else {
							gptSite = gpt_CheckForNULL($(this).data('site'));
						}
					}
					else {
						gptSite = gptMasterSite;
					}
                }
                else {
					gptSite = gptMasterSite;
                }



				if (gpt_AdFile === 'PU') {
					adId = $(this).attr('id');
				}
				else {
					//JBS-CELL-LANCET
					adId = 'gpt-ad-' + index + 1;
				}

				if (gpt_CheckForNULL(gptSite) !== null) {
					gptSite_Debug = gptSite;

					gptPOS = parseInt($(this).data('pos')) || -1;
					if (gptPOS === -1) {
						gptPOS = index + 1;
					}

					//Check if the ad slot has it's own page defined
					gptPage = $(this).data('page');
					if (gpt_CheckForNULL(gptPage) == null) {
						//Use the Master Page
						gptPage = gptMasterPage;
					}

					//If the page is not null, preceed it with a slash
					if (gptPage !== null) {
						gptPage = '/' + gptPage;
					}
					else {
						gptPage = '';
					}
					gptPage_Debug = gptPage;


					gptTC = $(this).data('tc');
					if (typeof gptTC === "undefined") {
						gptTC = '';
					}

					gptNative = $(this).data('native');
					if (typeof gptNative === "undefined") {
						gptNative = '';
					}

					//Determine sizemapping based on data-adsize
					if (gpt_AdFile === 'PU') {
						switch ($(this).data('adsize')) {
							case 'tall':
								adSizeMapping = gptTallSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600]], adId);
								break;
							case 'rectangle':
								adSizeMapping = gptRectangleSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250]], adId);
								break;
							case 'rectangleshort':
								adSizeMapping = gptRectangleShortSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250], [320, 50], [300, 50]], adId);
								break;
							case 'wide':
								adSizeMapping = gptWideSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[728, 90], [320, 50], [300, 50]], adId);
								break;
							case 'widerectangle':
								adSizeMapping = gptWideRectangleSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[320, 50], [300, 50], [300, 250]], adId);
								break;
							case 'mobile':
								adSizeMapping = gptMobileSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[320, 50], [300, 50]], adId);
								break;
							case 'fluid':
								adSizeMapping = gptFluidMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, 'fluid', adId);
								break;
							case 'prestitial':
								adSizeMapping = gptPrestitialMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[1, 1]], adId);
								break;
						}
					}
					else {
						//JBS-CELL-LANCET
						switch ($(this).data('adsize')) {
							case 'cell-leaderboard':
							case 'leaderboard':
								adSizeMapping = gptLeaderboardSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[728, 90], [320, 50], [300, 50]], adId);
								break;
							case 'bottom':
								if (gpt_ResponsiveHomepage) {
									adSizeMapping = gptHomePageSidebarMapping;
									tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600], [300, 250]], adId);
								}
								else {
									adSizeMapping = gptBoomBoxSizeMapping;
									tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250]], adId);
								}
								break;
							case 'boombox':
								if (gpt_AdFile === "CELLJOURNAL" || gpt_AdFile == "CELLBUCKET") {
									if (gpt_ResponsiveHomepage) {
										adSizeMapping = gptCellHomePageSidebarMapping;
										tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600], [300, 250], [336, 280]], adId);
									}
									else {
										adSizeMapping = gptBoomBoxSizeMapping;
										tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250], [336, 280]], adId);
									}
								}
								else {
									adSizeMapping = gptBoomBoxSizeMapping;
									tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250]], adId);
								}
								break;
							case 'interscroller':
								adSizeMapping = gptInterscrollerMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[320, 480], [480, 320], [768, 1024], [1024, 768]], adId);
								break;
							case 'sidebar':
							case 'skyscraper':
								adSizeMapping = gptSideBarSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600], [300, 250]], adId);
								break;
							case 'native':
							case 'fluid':
								adSizeMapping = gptFluidMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, 'fluid', adId);
								break;
							case 'prestitial':
								adSizeMapping = gptPrestitialMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, 'fluid', adId);
								break;
						}
					}
				}

				//Ignore any invalid adsizes that could be on the page
				if (adSizeMapping !== '') {
					$(this).addClass('gpt-ad-defined');
					if (gpt_AdFile !== 'PU') {
						$(this).attr('id', adId);
					}
					tempSlot.defineSizeMapping(adSizeMapping).addService(googletag.pubads()).setTargeting("pos", gptPOS);

					if (gptTC !== '') {
						tempSlot.setTargeting("tc", gptTC);
					}
					if (gptNative !== '') {
						tempSlot.setTargeting("native", gptNative);
					}
					gptEvent_Debug = gptEvent_Debug + $(this).data('adsize') + ' Requested<br>';
				}
				else {
					gptInvalidAdSize_Debug = gptInvalidAdSize_Debug + "Invalid Size Found, Id: " + $(this).prop('id') + ', adSize: ' + $(this).data('adsize') + '<br>';
					$(this).remove();
				}
			});
			if (gpt_CheckForNULL(gptInvalidAdSize_Debug) === null) {
				gptInvalidAdSize_Debug = "None Detected";
			}
			//***********************************************
			//End detection for ad slots on page
			//***********************************************

			//***********************************************
			//Start page level targeting
			//***********************************************
			if (gpt_AdFile !== 'PU') {
				//Logged in or IP Access
				gpt_TargetAccessType();

				//Check for Screenshot Client
				gpt_CheckSSC();
			}

			//Check for DFP Test
			gpt_GetDFPTestId();

			//Check for Additional Targeting
			gpt_GetAdditionalTargeting();

			if (gpt_IsProd && gptContextualEnabled) {
				var gpt_ContextualRetry = false;
				//Check for Contextual Targeting
				if (gs_channels !== "DEFAULT") {
					var gpt_ContextualCategories = Array.from(gs_channels);
					for (var i = 0; i < gpt_ContextualCategories.length; i++) {
						if (gpt_ContextualCategories[i] == 'gx_retry') {
							gpt_ContextualRetry = true;
						}
						if (!gpt_ContextualCategories[i].startsWith("custom_")) {
							//Remove any tag that is not custom
							gpt_ContextualCategories.splice(i, 1);
							i--;
						}
						else {
							//Strip the 'custom_' from the tag
							gpt_ContextualCategories[i] = gpt_ContextualCategories[i].replace("custom_", "")
						}
					}
					if (gpt_ContextualCategories.length > 0) {
						googletag.pubads().setTargeting("gs_cat", gpt_ContextualCategories);
					}
					else {
						gpt_ContextualCategories = 'none';
					}
					if (gpt_ContextualRetry) {
						gpt_ContextualCategories = "*Indexing*";
					}
					gptContextualResult_Debug = gpt_ContextualCategories;
				}
				else {
					gptContextualResult_Debug = 'No segments found';
				}
			}
			else {
				//Not Prod || Contextual not enabled
				gptContextualResult_Debug = 'Not enabled';
				
				if (!gpt_IsProd) {
                    var gpt_DevSite = gpt_CheckForNULL($('#gptSite').val(), "dev");
                    $('#gpt-Variables').append('<input type="hidden" id="gptEnvironment" value="' + gpt_DevSite + '">');
                    gpt_Target("Environment");
                }

			}
			//***********************************************
			//End page level targeting
			//***********************************************

			//Enable google dfp services
			if (parseInt($('#gptLazyLoading').val()) === 1) {
				googletag.pubads().enableLazyLoad({
					fetchMarginPercent: 200,
					renderMarginPercent: 0,
					mobileScaling: 2.0
				});
			}
			else {
				googletag.pubads().enableSingleRequest();
			}

			//Add Listener for RenderEnded
			googletag.pubads().addEventListener('slotRenderEnded', function (event) {
				if (!event.isEmpty) {
					if (typeof LocalAdLoaded === "function") {
						LocalAdLoaded(event);
					}
					if (gpt_ResponsiveHomepage) {
						if (typeof positionAd === "function") {
							positionAd();
						}
					}
					if (event.size[0] == "480" || event.size[1] == "480") {
						$('.gpt-InScrollBox').css('display', 'block');
						gptEvent_Debug = gptEvent_Debug + 'Interscroller ' + event.size + ' rendered: ' + gpt_GetElapsedTime() + '<br>';
					}
					else {
						gptEvent_Debug = gptEvent_Debug + event.size + ' rendered: ' + gpt_GetElapsedTime() + '<br>';
					}

				}
			});

			if (gpt_AdFile !== 'PU') {
				googletag.pubads().collapseEmptyDivs();
			}
			googletag.enableServices();

			//Populate Defined Ad Slots
			$(".gpt-ad-defined").each(function (index) {
				var adId = $(this).attr('id');
				googletag.cmd.push(function () { googletag.display(adId); });
			});

			gptStatus_Debug = 'GPT Complete';
		});
	}
	//--------------------------------------------------------------------------
	function gpt_DisplayDebug() {
		//Clone the template and append to Body
		$(document.body).append($('#gpt-DebugDiv').clone());
		$('#gpt-DebugDiv').remove();

		//Populate the Values
		gpt_LoadDebugValues();

		//Display and scroll to debug section
		$('.gpt-DebugDiv').css('display', 'block');
		window.scrollTo(0, document.body.scrollHeight);
	}
	//--------------------------------------------------------------------------
	function gpt_LoadDebugValues() {
		$('.gpt-DebugBtn').blur();
		$('.gptVersion_Debug').text(gpt_Version);
		$('.gptAdFile_Debug').html(gpt_AdFile);
		$('.gptEvent_Debug').html(gptEvent_Debug);
		$('.gptCustomTargeting_Debug').html(gptCustomTargeting_Debug);
		$('.gptSite_Debug').text(gptSite_Debug);
		$('.gptContextualResult_Debug').text(gptContextualResult_Debug);
		$('.gptContextualInit_Debug').text(gptContextualInit_Debug);
		$('.gptContextualURL_Debug').text(gptContextualURL_Debug);
		$('.gptContextualReady_Debug').text(gptContextualReady_Debug);
		$('.gptMoatReady_Debug').text(gptMoatReady_Debug);
		$('.gptMoatInit_Debug').text(gptMoatInit_Debug);
		$('.gptTestENV_Debug').text(gptTestENV_Debug);
		$('.gptDisableAds_Debug').text(gptDisableAds_Debug);
		$('.gptStatus_Debug').text(gptStatus_Debug);
		$('.gptPage_Debug').text(gptPage_Debug);
		$('.gptInvalidAdSize_Debug').html(gptInvalidAdSize_Debug);
	}
	//--------------------------------------------------------------------------
	function gpt_GrapeShotDebug() {
		$('.gpt-DebugBtn').blur();
		window.open("http://admin-elsevier.grapeshot.co.uk/custom/bookmarklet.cgi?blocking=1&vn=2&url=" + encodeURIComponent(document.location), "GRAPESHOT_POPUP", "width=650,height=600,scrollbars=yes,screenX=40,screenY=50,menubar=no,toolbar=no,resizable=yes", "Custom").focus();
	}
	//--------------------------------------------------------------------------
	function gpt_ShowConsole() {
		$('.gpt-DebugBtn').blur();
		googletag.openConsole();
	}
	//--------------------------------------------------------------------------
</script>
        </div>

            
                </div>
            

        

    

        


    

            

            

            <div data-widget-id="912135d6-7135-4d45-b457-b503cacc3c8b" data-widget-name="ux3-layout-widget"><header class="header base fixed">
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none adplaceholder adplaceholder--header js__adToHide widget-horizontalAd" id="9d6062d7-fad6-4e64-81cc-7dd25d99b23d">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-horizontalAd "><div class="horizontalAdContainer">
    <div class="horizontalAdLabelContainer">
        <div class="horizontalAdLabel">
                <img src="/templates/jsp/_style2/_marlin/images/bg_adHoriz.gif" alt="Advertisement">
        </div>
    </div>
    <div class="horizontalAd">
        





        <div class="pb-ad">
            <!-- Beginning Async AdSlot 1 for Ad unit els.cellpress/cell  ### size: [[728,90]] -->
<div class="gpt-ad" data-adsize="cell-leaderboard" data-page="" data-site="cell" data-pos="1"></div>
<!-- End AdSlot 1 -->
        </div>
    

    </div>
</div></div>
                </div>
                </div>

        










        <div data-widget-def="ux3-layout-widget" data-widget-id="3a83e2a6-2e7e-4931-892c-4ef43b6e3312" class="header__wrapper clearfix" data-db-parent-of="quickSearch"><div id="brandingRow"><a href="/" data-aa-name="science that inspires" data-aa-region="cp global header" title="Visit Cell.com">Science that inspires</a><p id="brandingRowRight"><a href="/" data-aa-name="cell press endorsement" data-aa-region="cp global header" title="Visit Cell.com">A Cell Press journal</a></p></div>
        
    

        


    

            

            

            <div data-widget-id="3a83e2a6-2e7e-4931-892c-4ef43b6e3312" data-widget-name="ux3-layout-widget"><div class="col-xs-6 header__left-side">
    

        


    

            

            
                <a href="/home" id="cpIconLnk" data-aa-name="cell press logo" data-aa-region="cp global header" title="Cell Press" target="_blank"><svg id="cp_logo" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 100 98.1" style="enable-background:new 0 0 100 98.1;" xml:space="preserve" height="100" width="98.1"><path style="fill:#FFFFFF;" d="M69.09,0.09c-4.01-0.6-7.83,1.94-8.78,5.89l-3.14,13.11c-0.76,3.19-3.44,5.56-6.7,5.93l-18.6,2.13          C11.83,29.31-2.53,47.66,0.37,67.78c2.3,15.96,15.18,27.77,30.54,30.08c4.01,0.6,7.83-1.94,8.78-5.89l3.14-13.11 c0.76-3.19,3.44-5.56,6.69-5.93l18.6-2.13c20.04-2.17,34.41-20.52,31.5-40.65C97.32,14.21,84.45,2.39,69.09,0.09z M65.06,47.2  l-11.75,1.4c-1.98,0.21-3.62,1.65-4.08,3.59l-3.43,14.3c-1.01,4.22-4.5,7.39-8.8,8c-6.66,0.72-12.63-4.2-13.15-10.92                c-0.51-6.5,4.6-12.11,11.08-12.81l11.75-1.39c1.98-0.22,3.62-1.65,4.08-3.59l3.43-14.3c1.01-4.22,4.5-7.39,8.8-8          c6.66-0.72,12.63,4.19,13.15,10.92C76.66,40.89,71.55,46.5,65.06,47.2z"></path></svg></a><div class="journal-logos">
            

            <span data-widget-id="35d8d734-3aa0-4ec8-9e2d-14babe803594" data-widget-name="ux3-general-image"><a href="/cell/home" title="Cell"><img alt="Cell" src="/pb-assets/ux3/logos/cell/Cell_white-1532516611593.svg"></a></span>

            
                </div>
            

        

    

        


    

            

            
                <div class="journal-logos">
            

            <span data-widget-id="3ffcbf1f-79fd-4ba3-85b5-bef10a63475f" data-widget-name="ux3-general-image"><a href="https://www.cell.com/open-access?utm_source=oa-label&amp;__hstc=109691226.e4e812b53ff324eccbdbdb2e4fb9a6ab.1644503737405.1644524234622.1644587109507.4&amp;__hssc=109691226.7.1644587109507&amp;__hsfp=3253153019" title="This journal offers authors two options (open access or subscription) to publish research"><img alt="This journal offers authors two options (open access or subscription) to publish research" src="/pb-assets/ux3/logos/cell/Supports_open_access_border-1645554937897.svg"></a></span>

            
                </div>
            

        
</div><div class="col-xs-6">
    

        


    

            

            

            <div data-widget-id="2555f6db-b333-4edb-8b15-ef9213f1f0a5" data-widget-name="ux3-layout-widget"><div class="header__nav row">
    

        


    

            

            

            <ul data-widget-id="52432d24-0331-4c9d-85f3-2033edc7208b" data-widget-name="UX3ProfileLinksWidget" class="rlist rlist--inline profile-links"><li class="profile-links__submit"><a data-adobe-name="Publish" class="profile-links__submit__btn" title="Submit" href="https://www.editorialmanager.com/cell" target="_self" data-aa-conversiondriver="jbs:global-header:submitpage" data-profile-link="true">Submit</a></li><li class="profile-links__login-dropdown"><div class="profile-links__user-list__wrapper"><button data-db-target-for="loginmenu-913988" aria-expanded="false" aria-controls="login-links-913988" title="Log in links dropdown" class="profile-links__username"><span>Log in</span></button><div data-db-target-of="loginmenu-913988" id="login-links--913988" class="profile-links__user-list__holder dropBlock__holder"><ul class="rlist profile-links__list"><li class="profile-links__login"><a data-adobe-name="Log In" class="profile-links__login__btn" title="Log in to Cell.com" href="https://www.cell.com/action/idLogin?type=login&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2901446-X" target="_self" data-profile-link="true">Log in</a></li><li class="profile-links__institution"><a data-adobe-name="Get institutional access" class="profile-links__institution__btn" title="Institutional access" href="https://www.cell.com/action/idLogin?type=institution&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2901446-X" target="_self" data-profile-link="true">Get Institutional Access</a></li></ul></div></div></li><li class="profile-links__register"><a data-adobe-name="Register" class="profile-links__register__btn" title="Register a username and password for free" href="https://www.cell.com/action/idLogin?type=registration&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2901446-X" target="_self" data-profile-link="true">Register</a></li><li class="hidden-lg"> <div class="profile-links__controls"> <button aria-expanded="false" aria-controls="links-dialog-913988" data-opens="#links-dialog-913988" class="profile-links__btn--open"><span class="sr-only">open links dialog</span><i aria-hidden="true" class="icon-gizmo-person"></i></button></div><dialog id="links-dialog-913988" aria-modal="true"><button data-closes="#links-dialog-913988" class="profile-links__btn--close"><span class="sr-only">close links dialog</span><i aria-hidden="true" class="icon-close-solid"></i></button><ul>   <li class="profile-links__submit"><a data-adobe-name="Publish" class="profile-links__submit__btn" title="Submit" href="https://www.editorialmanager.com/cell" target="_self" data-aa-conversiondriver="jbs:global-header:submitpage" data-profile-link="true">Submit</a></li><li class="profile-links__login"><a data-adobe-name="Log In" class="profile-links__login__btn" title="Log in to Cell.com" href="https://www.cell.com/action/idLogin?type=login&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2901446-X" target="_self" data-profile-link="true">Log in</a></li><li class="profile-links__institution"><a data-adobe-name="Get institutional access" class="profile-links__institution__btn" title="Institutional access" href="https://www.cell.com/action/idLogin?type=institution&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2901446-X" target="_self" data-profile-link="true">Get Institutional Access</a></li><li class="profile-links__register"><a data-adobe-name="Register" class="profile-links__register__btn" title="Register a username and password for free" href="https://www.cell.com/action/idLogin?type=registration&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2901446-X" target="_self" data-profile-link="true">Register</a></li></ul></dialog></li></ul>

            

        
</div></div>

            

        
</div></div>

            

        

        </div>
    

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget csasInstitutionBanner none  widget-none" id="51b7fa33-2137-4f58-9e98-9746c4400513" style="display: none;">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none "><div class="ui-state-highlight ui-corner-all csas-institution-banner hidden" data-doi="10.1016/j.cell.2021.12.008">
    <span class="ui-icon ui-icon-close banner-close"></span>
    <p>
        Access provided by <span class="accountName"></span>
    </p>
</div>

<style>
    .csas-institution-banner {
        padding: 10px 0 0 10px;
        margin: 5px;
    }

    .csas-institution-banner.ui-state-highlight .ui-icon {
        background-image: url("/templates/jsp/css/jquery-ui-1.10.2/base/images/ui-icons_454545_256x240.png");
        margin-right: 5px;
        margin-top: -4px;
    }

    .csas-institution-banner .banner-close {
        float: right;
    }
</style>

<script>

    function resetBanner(doi) {
        $.get('/action/getAccount?doi=' + doi, function(data) {
            if (data) {
                $(".csas-institution-banner p .accountName").html(data);
                $(".csasInstitutionBanner").fadeIn();
                $(".csas-institution-banner").fadeIn();
            }
        });
    }

    var flag = 0;
    $(document).ready(function() {
        if(!$(".csas-institution-banner").is(":visible")){
            $(".csasInstitutionBanner").hide();
        }

        $(".banner-close").click(function() {
            $.get("/action/dismissBanner", function(data) {
                $(".csasInstitutionBanner").fadeOut(function () {
                    if(typeof UX !== 'undefined') {
                        UX.pageBody.init();
                    }
                });
            });
        });

        $(".articleContent .tabs li[role='tab'].showFullText").click(function() {
           flag = 1;
        });

        $(document).on("reloadTabsContent", function() {
            var isSelected = $(".articleContent .tabs li[role='tab'].showFullText").hasClass("active");
            if (isSelected) {
                var isHidden =  $(".csas-institution-banner").is(".hidden");
                if (!isHidden) {
                    resetBanner($(".csas-institution-banner").data("doi"));
                    console.log("fadeIn");
                }
            } else {
                $(".csas-institution-banner").fadeOut();
                console.log("fadeOut");
            }
            flag = 0;
        });

        $(document).on("click", ".articleContent .tabs li[role='tab']", function() {

        });
    });
</script></div>
                </div>
                </div>

        
</header></div>

            

        










        <div data-widget-def="graphQueryWidget" data-widget-id="2a125dae-6dbc-4e76-a75e-3e563f1d3a1d" class="ux3">
        
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget graphQueryWidget none top-padding-handler noMargin axel-mega-menu widget-none widget-compact-all fonts-loaded" id="2a125dae-6dbc-4e76-a75e-3e563f1d3a1d" style="padding-top: 115.594px;">
                
                
                    
                

                <div class="wrapped " id="CP TOP LEVEL NAV MENU">
                    
                    <div class="widget-body body body-none  body-compact-all"><nav id="mega-menu-nav" role="navigation" class="mega-menu-nav">
  <!--.skip//a(href='#searchText' aria-label='Skip menu') Skip menu
  -->
  <div class="mega-menu-nav__wrapper">
    <button class="mega-menu__toggle"><span class="sr-only">Main menu</span><i aria-hidden="true" class="icon-gizmo-hamburger"></i></button>
    <ul role="menubar" class="mega-menu rlist">
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-1" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-1" data-item-link="main-menu-link-1" data-index="0" class="mega-menu__item-link menu-link"><span>This journal</span></a>
        <div id="main-menu-panel-1" aria-labelledby="main-menu-link-1" data-item-wrapper="main-menu-link-1" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <div aria-label="Menu dropzone section" tabindex="-1" class="listDropZone">
    

        


    

            

            

            <div class="cover-image"><div class="cover-image__thumb"><a href="/issue/S0092-8674(24)X0012-4" class="menu-link"><img src="/cms/asset/atypon:cms:attachment:img:d5e6:rev:1748270185670-389:pii:S0092867424X00124/cover.tif.jpg" alt="Cover Image - Cell, Volume 188, Issue 11"></a></div><div class="cover-image__meta"></div></div>

            

        
</div>
            </li>
            <li style="grid-column: 2/3" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-1-2" class="sub-menu__heading">Read
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/newarticles" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Online now</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/current" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Current issue</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/archive" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Archive</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/action/showPreferences?menuTab=Alerts" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Publication alerts</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/sneakpeek" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Sneak Peek - preprints</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 3/5" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-1-3" class="sub-menu__heading">About
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-2">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/advisory-board" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Advisory board</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/aims" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Aims and scope</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/contact" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Contact</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/editors-and-staff" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Editors and staff</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/editorial-ethics" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Ethics</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/information-for-authors/permissions" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Permissions</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/reviewers" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>Reviewer instructions</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-2" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-2" data-item-link="main-menu-link-2" data-index="1" class="mega-menu__item-link menu-link"><span>Journals</span></a>
        <div id="main-menu-panel-2" aria-labelledby="main-menu-link-2" data-item-wrapper="main-menu-link-2" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/5" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-2-1" class="sub-menu__heading">Life &amp; medical sciences
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-4">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/home" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cancer-cell/home" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Cancer Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-chemical-biology" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Chemical Biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-genomics/home" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Genomics</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-host-microbe/home" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Host &amp; Microbe</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-metabolism/home" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Metabolism</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports/home" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-medicine/home" data-index="7" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Medicine</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-stem-cell/home" data-index="8" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Stem Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-systems/home" data-index="9" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Systems</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/current-biology/home" data-index="10" role="menuitem" class="sub-menu__item-link menu-link"><span>Current Biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/developmental-cell/home" data-index="11" role="menuitem" class="sub-menu__item-link menu-link"><span>Developmental Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/immunity/home" data-index="12" role="menuitem" class="sub-menu__item-link menu-link"><span>Immunity</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/med/home" data-index="13" role="menuitem" class="sub-menu__item-link menu-link"><span>Med</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-cell/home" data-index="14" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/neuron/home" data-index="15" role="menuitem" class="sub-menu__item-link menu-link"><span>Neuron</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/structure/home" data-index="16" role="menuitem" class="sub-menu__item-link menu-link"><span>Structure</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/ajhg/home" data-index="17" role="menuitem" class="sub-menu__item-link menu-link"><span>American Journal of Human Genetics (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/biophysj/home" data-index="18" role="menuitem" class="sub-menu__item-link menu-link"><span>Biophysical Journal (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/biophysreports/home" data-index="19" role="menuitem" class="sub-menu__item-link menu-link"><span>Biophysical Reports (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/hgg-advances/home" data-index="20" role="menuitem" class="sub-menu__item-link menu-link"><span>Human Genetics and Genomics Advances (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-plant/home" data-index="21" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Plant (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/molecular-therapy/home" data-index="22" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/methods/home" data-index="23" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy Methods &amp; Clinical Development (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/nucleic-acids/home" data-index="24" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy Nucleic Acids (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/oncology/home" data-index="25" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy Oncology (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/plant-communications/home" data-index="26" role="menuitem" class="sub-menu__item-link menu-link"><span>Plant Communications (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/stem-cell-reports/home" data-index="27" role="menuitem" class="sub-menu__item-link menu-link"><span>Stem Cell Reports (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/biochemical-sciences/home" data-index="28" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Biochemical Sciences</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/cancer/home" data-index="29" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Cancer</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/cell-biology/home" data-index="30" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Cell Biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/ecology-evolution/home" data-index="31" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Ecology &amp; Evolution</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/endocrinology-metabolism/home" data-index="32" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Endocrinology &amp; Metabolism</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/genetics/home" data-index="33" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Genetics</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/immunology/home" data-index="34" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Immunology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/microbiology/home" data-index="35" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Microbiology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/molecular-medicine/home" data-index="36" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Molecular Medicine</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/neurosciences/home" data-index="37" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Neurosciences</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/parasitology/home" data-index="38" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Parasitology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/pharmacological-sciences/home" data-index="39" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Pharmacological Sciences</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/plant-science/home" data-index="40" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Plant Science</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 5/6" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-2-2" class="sub-menu__heading">Physical sciences &amp; engineering
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-biomaterials/home" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Biomaterials</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-physical-science/home" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Physical Science</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/chem/home" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Chem</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/chem-catalysis/home" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Chem Catalysis</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/device/home" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Device</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/joule/home" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Joule</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/matter/home" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>Matter</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/newton/home" data-index="7" role="menuitem" class="sub-menu__item-link menu-link"><span>Newton</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/chemistry/home" data-index="8" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Chemistry</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 6/7" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-2-3" class="sub-menu__heading">Multidisciplinary
              </h4>
              <ul role="menubar" data-index="2" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-methods/home" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Methods</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-sustainability/home" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Sustainability</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/heliyon/home" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Heliyon</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/iscience/home" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>iScience</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/one-earth/home" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>One Earth</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/patterns/home" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Patterns</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/star-protocols/home" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>STAR Protocols</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/nexus/home" data-index="7" role="menuitem" class="sub-menu__item-link menu-link"><span>Nexus (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/the-innovation/home" data-index="8" role="menuitem" class="sub-menu__item-link menu-link"><span>The Innovation (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/biotechnology/home" data-index="9" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Biotechnology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/cognitive-sciences/home" data-index="10" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Cognitive Sciences</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-3" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-3" data-item-link="main-menu-link-3" data-index="2" class="mega-menu__item-link menu-link"><span>Publish</span></a>
        <div id="main-menu-panel-3" aria-labelledby="main-menu-link-3" data-item-wrapper="main-menu-link-3" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-3-1" class="sub-menu__heading">Authors
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="https://www.editorialmanager.com/cell?__hstc=25856994.7201670a4397fa03148684920d4a8d38.1672762013831.1694716659709.1694718909620.146&amp;__hssc=25856994.15.1694718909620&amp;__hsfp=1366844671" data-index="0" target="_blank" role="menuitem" class="sub-menu__item-link menu-link"><span>Submit article
                      <div class="external-icon"></div><span class="sr-only">External Link</span></span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/information-for-authors" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Information for authors</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/open-access" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Open access</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/multi-journal-submission" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Multi-Journal Submission</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/information-for-authors/star-authors-guide" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>STAR Methods</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 2/3" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-3-2" class="sub-menu__heading">Reviewers
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/reviewers" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Information for reviewers</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-4" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-4" data-item-link="main-menu-link-4" data-index="3" class="mega-menu__item-link menu-link"><span>News &amp; events</span></a>
        <div id="main-menu-panel-4" aria-labelledby="main-menu-link-4" data-item-wrapper="main-menu-link-4" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/newsroom" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Newsroom</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/events" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Events</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/symposia" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Symposia</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/webinars" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Webinars</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/consortia" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Consortia Hub</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-5" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-5" data-item-link="main-menu-link-5" data-index="4" class="mega-menu__item-link menu-link"><span>About Cell Press</span></a>
        <div id="main-menu-panel-5" aria-labelledby="main-menu-link-5" data-item-wrapper="main-menu-link-5" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-5-1" class="sub-menu__heading">Cell Press
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/about" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>About Cell Press</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/contact" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Contact us</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="https://service.elsevier.com/app/overview/cell-press/" data-index="2" target="_blank" role="menuitem" class="sub-menu__item-link menu-link"><span>Help &amp; support
                      <div class="external-icon"></div><span class="sr-only">External Link</span></span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/open-access" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Open access at Cell Press</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/diversity" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Inclusion and diversity</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-press-50" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>50th anniversary</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 2/3" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-5-2" class="sub-menu__heading">Hubs
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/biotechnology-bioengineering" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Biotechnology and bioengineering</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-press-chemistry" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Chemistry</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/clinical-translational" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Clinical and translational</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/covid-19" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>COVID-19</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/fundamental-biology" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Fundamental biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/sustainability" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Sustainability</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 3/4" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-5-3" class="sub-menu__heading">Careers
              </h4>
              <ul role="menubar" data-index="2" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="https://relx.wd3.myworkdayjobs.com/en-US/ElsevierJobs" data-index="0" target="_blank" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Press careers
                      <div class="external-icon"></div><span class="sr-only">External Link</span></span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
    </ul>
    <div class="mega-menu-nav__quick-search">
    

        


    

            

            

            <div data-widget-id="a1f05748-166f-41e0-b01a-ca8ac1cc95d4" data-widget-name="UX3QuickSearchWidget" class="quick-search"><div class="dropBlock"><form action="https://www.cell.com/action/doSearch?journalCode=cell" name="quickSearchBar" method="get" title="Quick Search"><input id="type" type="hidden" value="quicksearch" name="type"><div class="quick-search__toggle autoCompleteWrapper"><input id="searchText" type="text" name="text1" placeholder="Search for..." enterkeyhint="search" autocomplete="off" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" data-group-titles-conf="3" data-publication-items-conf="3" aria-label="Enter a search term" class="searchText searchText-autocomplete ui-autocomplete-input"><button type="submit" title="Search" aria-label="Submit the search form" class="btn quick-search__button icon-search"><span>Go search</span></button><a href="https://www.cell.com/search/advanced?SeriesKey=cell&amp;ISSN=0092-8674&amp;journalCode=cell" title="Advanced search" class="quick-search__advanced-link">Advanced search</a><ul id="ui-id-1" tabindex="0" class="ui-menu ui-widget ui-widget-content ui-autocomplete ui-front" unselectable="on" style="display: none;"></ul></div><div class="quick-search__box"><div class="quick-search__box-inner"><div class="quick-search__box-wrapper"><select name="field1" aria-label="Select a search area"><option value="AllField">All content</option><option value="Title">Article title</option><option value="Contrib">Authors</option><option value="Keyword">Keywords</option><option value="Abstract">Abstract</option><option value="AbstractTitleKeywordFilterField">Article title, abstract, keywords</option></select><input type="hidden" name="journalCode" value="cell"><div class="quick-search__scope-options"><span>Search within:</span><label title="This journal" class="radio--primary"><input type="radio" name="SeriesKey" id="searchScopeSeries-a1f05748-166f-41e0-b01a-ca8ac1cc95d4" title="This journal" value="cell" checked="checked" aria-label="Search within current journal"><span>This journal</span></label><label title="Full site" class="radio--primary"><input type="radio" name="SeriesKey" id="searchScopeFullSite-a1f05748-166f-41e0-b01a-ca8ac1cc95d4" value="" title="Full site" aria-label="Search within full site"><span>Full site</span></label></div><a href="https://www.cell.com/search/advanced?SeriesKey=cell&amp;ISSN=0092-8674&amp;journalCode=cell" title="Advanced search" class="quick-search__advanced-link">Advanced search</a></div><a href="javascript:void(0)" aria-label="Close search options" title="Close search options" class="quick-search__box-close"><i aria-hidden="true" class="icon-tools_close"></i></a></div></div><div class="quick-search__alert hidden"><p>Please enter a term before submitting your search.</p><a href="#" class="quick-search__alert--close"> Ok</a></div></form></div></div>

            

        
</div>
  </div>
</nav>
<div id="page-body-id" class="page-body-flag"></div></div>
                </div>
                </div>

        

        </div>
    

    

        


    

            

            

            <div data-widget-id="f8c6daf9-9df1-4fd9-ba3a-7f5d7b3d4466" data-widget-name="UX3HTMLWidget"><style>
.journal-logos:nth-of-type(2) img {
max-height: 30px !important;
}
.journal-logos:nth-of-type(3) img {
max-height: 30px !important;
}
.ux3 .journal-logos + .journal-logos {
margin-left: 30px;
}
.header__wrapper.clearfix .col-xs-6:first-of-type {
width: 55%;
}
.header__wrapper.clearfix .col-xs-6:nth-of-type(2) {
width: 45%;
}
@media screen and (max-width: 1000px) {
.ux3 .journal-logos + .journal-logos {
display: none;
}
.header__wrapper.clearfix .col-xs-6:first-of-type {
width: 50%;
}
.header__wrapper.clearfix .col-xs-6:nth-of-type(2) {
width: 50%;
}
}

.ux3 .journal-logos:first-of-type {
   max-width: 500px;
}

.profile-links {
   white-space: nowrap;
}

@media only screen and (min-width: 740px) and (max-width: 1600px) {
   .journal-logos:nth-of-type(2) img {
      max-width: 250px !important;
   }
}

@media only screen and (min-width: 1024px) and (max-width: 1600px) {
   .header__wrapper.clearfix .col-xs-6:first-of-type {
      width: 60% !important;
   }
   .header__wrapper.clearfix .col-xs-6:nth-of-type(2) {
      width: 40% !important;
   }
}

@media only screen and (max-width: 1240px) {
   .ux3 .journal-logos + .journal-logos {
      display: none !important;
   }
}

</style></div>

            

        
</div></div>

            
                </div>
            

        

        </div>
    

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget axelPublicationContent none  widget-none" id="cadc9b41-a542-448a-9de4-8d0803320174">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none "><link id="build-style-article" rel="stylesheet" type="text/css" href="/products/marlin/cell/releasedAssets/css/build-article-b26f3342b722959c7931.css"><article xmlns="http://www.w3.org/1999/xhtml" data-design="pill" data-has="right-rail" vocab="http://schema.org/" typeof="ScholarlyArticle" data-pii="S0092-8674(21)01446-X" lang="en" dir="ltr"><div data-core-wrapper="header"><header data-extent="frontmatter"><div class="core-container"><div data-article-access="free" data-article-access-type="open" class="meta-panel"><span class="meta-panel__left-content"><span class="meta-panel__type"><span>Article</span></span></span><span class="meta-panel__middle-content"><span class="meta-panel__volumeIssue"><a href="/cell/issue?pii=S0092-8674(21)X0002-5"><span>Volume 185</span>, <span>Issue 1</span></a></span><span class="meta-panel__pages"><span>p95-112.e18</span></span><span class="meta-panel__onlineDate">January 06, 2022</span></span><span class="meta-panel__right-content"><span class="meta-panel__access meta-panel__access--archive"><span>Open Archive</span></span></span><div class="article-header__download-full-issue"><button title="Download full issue" type="submit" formaction="/action/downloadMultiplePdf?action=downloadIssuePdf&amp;journal=cell&amp;volume=185&amp;issue=1&amp;pii=S0092-8674%2821%29X0002-5" class="btn--download-issue btn--cancel download-full-issue"><i class="icon-download-new"></i><span>Download Full Issue</span></button><div class="table-of-content__alert table-of-content__alert--download"><p>Download started</p><a href="#" class="table-of-content__alert--close">Ok</a></div></div></div><h1 property="name">Limb development genes underlie variation in human fingerprint patterns</h1><div class="contributors"><span class="authors"><span role="list"><span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au1" aria-controls="axel_author_au1"><span property="givenName">Jinxi</span> <span property="familyName">Li</span></a><div id="au1" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Jinxi</span> <span property="familyName">Li</span> <a class="orcid-id" href="https://orcid.org/0000-0002-1366-9593" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0002-1366-9593</a></div><div class="content"><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">30</div><div id="fn1"><div id="ntpara0010" role="paragraph">These authors contributed equally</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">State Key Laboratory of Genetic Engineering, Collaborative Innovation Center for Genetics and Development, School of Life Sciences, and Human Phenome Institute, Fudan University, Shanghai 200438, PRC</span></div><div property="affiliation" typeof="Organization"><span property="name">CAS Key Laboratory of Computational Biology, Shanghai Institute of Nutrition and Health, University of Chinese Academy of Sciences, Chinese Academy of Sciences, Shanghai 200031, PRC</span></div></div><div class="core-author-link"><a href="/authored-by/Li/Jinxi" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">2</a>,<a role="doc-noteref">30</a></sup></span> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au2" aria-controls="axel_author_au2"><span property="givenName">James D.</span> <span property="familyName">Glover</span></a><div id="au2" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">James D.</span> <span property="familyName">Glover</span> <a class="orcid-id" href="https://orcid.org/0000-0003-4678-5643" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0003-4678-5643</a></div><div class="content"><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">30</div><div id="fn1"><div id="ntpara0010" role="paragraph">These authors contributed equally</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">The Roslin Institute and Royal (Dick) School of Veterinary Studies, University of Edinburgh, Edinburgh, UK</span></div></div><div class="core-author-link"><a href="/authored-by/Glover/James+D" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">3</a>,<a role="doc-noteref">30</a></sup></span> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au3" aria-controls="axel_author_au3"><span property="givenName">Haiguo</span> <span property="familyName">Zhang</span></a><div id="au3" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Haiguo</span> <span property="familyName">Zhang</span></div><div class="content"><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">30</div><div id="fn1"><div id="ntpara0010" role="paragraph">These authors contributed equally</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Ministry of Education Key Laboratory of Contemporary Anthropology, Department of Anthropology and Human Genetics, School of Life Sciences, Fudan University, Shanghai 200438, PRC</span></div></div><div class="core-author-link"><a href="/authored-by/Zhang/Haiguo" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">4</a>,<a role="doc-noteref">30</a></sup></span><span data-displayed-on="all"> ∙ … </span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au4" aria-controls="axel_author_au4"><span property="givenName">Meifang</span> <span property="familyName">Peng</span></a><div id="au4" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Meifang</span> <span property="familyName">Peng</span></div><div class="content"><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">30</div><div id="fn1"><div id="ntpara0010" role="paragraph">These authors contributed equally</div></div></div><div role="doc-footnote" data-has="label"><div class="label">32</div><div id="fn3"><div id="ntpara0020" role="paragraph">Present address: The Core Laboratory in Medical Center of Clinical Research, Department of Molecular Diagnostics and Endocrinology, Shanghai Ninth People’s Hospital, State Key Laboratory of Medical Genomics, Shanghai Jiao Tong University School of Medicine, Shanghai 200011, PRC</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">CAS Key Laboratory of Computational Biology, Shanghai Institute of Nutrition and Health, University of Chinese Academy of Sciences, Chinese Academy of Sciences, Shanghai 200031, PRC</span></div><div property="affiliation" typeof="Organization"><span property="name">Ministry of Education Key Laboratory of Contemporary Anthropology, Department of Anthropology and Human Genetics, School of Life Sciences, Fudan University, Shanghai 200438, PRC</span></div></div><div class="core-author-link"><a href="/authored-by/Peng/Meifang" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">2</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">4</a>,<a role="doc-noteref">30</a>,<a role="doc-noteref">32</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au5" aria-controls="axel_author_au5"><span property="givenName">Jingze</span> <span property="familyName">Tan</span></a><div id="au5" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Jingze</span> <span property="familyName">Tan</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Ministry of Education Key Laboratory of Contemporary Anthropology, Department of Anthropology and Human Genetics, School of Life Sciences, Fudan University, Shanghai 200438, PRC</span></div></div><div class="core-author-link"><a href="/authored-by/Tan/Jingze" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">4</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au6" aria-controls="axel_author_au6"><span property="givenName">Chandana Basu</span> <span property="familyName">Mallick</span></a><div id="au6" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Chandana Basu</span> <span property="familyName">Mallick</span> <a class="orcid-id" href="https://orcid.org/0000-0001-9459-2710" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0001-9459-2710</a></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">The Roslin Institute and Royal (Dick) School of Veterinary Studies, University of Edinburgh, Edinburgh, UK</span></div><div property="affiliation" typeof="Organization"><span property="name">Centre for Genetic Disorders, Institute of Science, Banaras Hindu University, Varanasi, Uttar Pradesh, India</span></div></div><div class="core-author-link"><a href="/authored-by/Mallick/Chandana+Basu" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">3</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">5</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au7" aria-controls="axel_author_au7"><span property="givenName">Dan</span> <span property="familyName">Hou</span></a><div id="au7" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au7"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Dan</span> <span property="familyName">Hou</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Chinese Academy of Sciences Key Laboratory of Tissue Microenvironment and Tumor, Shanghai Institute of Nutrition and Health, Shanghai Institutes for Biological Sciences, University of Chinese Academy of Sciences, Chinese Academy of Sciences, Shanghai 200031, PRC</span></div></div><div class="core-author-link"><a href="/authored-by/Hou/Dan" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">6</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au8" aria-controls="axel_author_au8"><span property="givenName">Yajun</span> <span property="familyName">Yang</span></a><div id="au8" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au8"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Yajun</span> <span property="familyName">Yang</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Ministry of Education Key Laboratory of Contemporary Anthropology, Department of Anthropology and Human Genetics, School of Life Sciences, Fudan University, Shanghai 200438, PRC</span></div></div><div class="core-author-link"><a href="/authored-by/Yang/Yajun" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">4</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au9" aria-controls="axel_author_au9"><span property="givenName">Sijie</span> <span property="familyName">Wu</span></a><div id="au9" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au9"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Sijie</span> <span property="familyName">Wu</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">State Key Laboratory of Genetic Engineering, Collaborative Innovation Center for Genetics and Development, School of Life Sciences, and Human Phenome Institute, Fudan University, Shanghai 200438, PRC</span></div><div property="affiliation" typeof="Organization"><span property="name">CAS Key Laboratory of Computational Biology, Shanghai Institute of Nutrition and Health, University of Chinese Academy of Sciences, Chinese Academy of Sciences, Shanghai 200031, PRC</span></div></div><div class="core-author-link"><a href="/authored-by/Wu/Sijie" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">2</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au10" aria-controls="axel_author_au10"><span property="givenName">Yu</span> <span property="familyName">Liu</span></a><div id="au10" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au10"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Yu</span> <span property="familyName">Liu</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">CAS Key Laboratory of Computational Biology, Shanghai Institute of Nutrition and Health, University of Chinese Academy of Sciences, Chinese Academy of Sciences, Shanghai 200031, PRC</span></div></div><div class="core-author-link"><a href="/authored-by/Liu/Yu" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">2</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au11" aria-controls="axel_author_au11"><span property="givenName">Qianqian</span> <span property="familyName">Peng</span></a><div id="au11" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au11"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Qianqian</span> <span property="familyName">Peng</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">CAS Key Laboratory of Computational Biology, Shanghai Institute of Nutrition and Health, University of Chinese Academy of Sciences, Chinese Academy of Sciences, Shanghai 200031, PRC</span></div></div><div class="core-author-link"><a href="/authored-by/Peng/Qianqian" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">2</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au12" aria-controls="axel_author_au12"><span property="givenName">Shijie C.</span> <span property="familyName">Zheng</span></a><div id="au12" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au12"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Shijie C.</span> <span property="familyName">Zheng</span> <a class="orcid-id" href="https://orcid.org/0000-0002-2902-3164" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0002-2902-3164</a></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">CAS Key Laboratory of Computational Biology, Shanghai Institute of Nutrition and Health, University of Chinese Academy of Sciences, Chinese Academy of Sciences, Shanghai 200031, PRC</span></div></div><div class="core-author-link"><a href="/authored-by/Zheng/Shijie+C" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">2</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au13" aria-controls="axel_author_au13"><span property="givenName">Edie I.</span> <span property="familyName">Crosse</span></a><div id="au13" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au13"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Edie I.</span> <span property="familyName">Crosse</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Centre for Regenerative Medicine, University of Edinburgh, Edinburgh, UK</span></div></div><div class="core-author-link"><a href="/authored-by/Crosse/Edie+I" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">7</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au14" aria-controls="axel_author_au14"><span property="givenName">Alexander</span> <span property="familyName">Medvinsky</span></a><div id="au14" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au14"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Alexander</span> <span property="familyName">Medvinsky</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Centre for Regenerative Medicine, University of Edinburgh, Edinburgh, UK</span></div></div><div class="core-author-link"><a href="/authored-by/Medvinsky/Alexander" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">7</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au15" aria-controls="axel_author_au15"><span property="givenName">Richard A.</span> <span property="familyName">Anderson</span></a><div id="au15" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au15"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Richard A.</span> <span property="familyName">Anderson</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">MRC Centre for Reproductive Health, Queens Medical Research Institute, University of Edinburgh, Edinburgh, UK</span></div></div><div class="core-author-link"><a href="/authored-by/Anderson/Richard+A" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">8</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au16" aria-controls="axel_author_au16"><span property="givenName">Helen</span> <span property="familyName">Brown</span></a><div id="au16" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au16"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Helen</span> <span property="familyName">Brown</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">The Roslin Institute and Royal (Dick) School of Veterinary Studies, University of Edinburgh, Edinburgh, UK</span></div></div><div class="core-author-link"><a href="/authored-by/Brown/Helen" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">3</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au17" aria-controls="axel_author_au17"><span property="givenName">Ziyu</span> <span property="familyName">Yuan</span></a><div id="au17" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au17"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Ziyu</span> <span property="familyName">Yuan</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Fudan-Taizhou Institute of Health Sciences, Taizhou, Jiangsu 225326, PRC</span></div></div><div class="core-author-link"><a href="/authored-by/Yuan/Ziyu" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">9</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au18" aria-controls="axel_author_au18"><span property="givenName">Shen</span> <span property="familyName">Zhou</span></a><div id="au18" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au18"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Shen</span> <span property="familyName">Zhou</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Shanghai Foreign Language School, Shanghai 200083, PRC</span></div></div><div class="core-author-link"><a href="/authored-by/Zhou/Shen" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">10</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au19" aria-controls="axel_author_au19"><span property="givenName">Yanqing</span> <span property="familyName">Xu</span></a><div id="au19" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au19"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Yanqing</span> <span property="familyName">Xu</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Forest Ridge School of the Sacred Heart, Bellevue, WA 98006, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Xu/Yanqing" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">11</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au20" aria-controls="axel_author_au20"><span property="givenName">John P.</span> <span property="familyName">Kemp</span></a><div id="au20" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au20"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">John P.</span> <span property="familyName">Kemp</span> <a class="orcid-id" href="https://orcid.org/0000-0002-9105-2249" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0002-9105-2249</a></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">University of Queensland Diamantina Institute, University of Queensland, Brisbane, QLD, Australia</span></div></div><div class="core-author-link"><a href="/authored-by/Kemp/John+P" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">12</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au21" aria-controls="axel_author_au21"><span property="givenName">Yvonne Y.W.</span> <span property="familyName">Ho</span></a><div id="au21" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au21"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Yvonne Y.W.</span> <span property="familyName">Ho</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">QIMR Berghofer Medical Rese Institute, Brisbane, QLD, Australia</span></div></div><div class="core-author-link"><a href="/authored-by/Ho/Yvonne+Y+W" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">13</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au22" aria-controls="axel_author_au22"><span property="givenName">Danuta Z.</span> <span property="familyName">Loesch</span></a><div id="au22" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au22"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Danuta Z.</span> <span property="familyName">Loesch</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Psychology Department, La Trobe University, Melbourne, VIC, Australia</span></div></div><div class="core-author-link"><a href="/authored-by/Loesch/Danuta+Z" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">14</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au23" aria-controls="axel_author_au23"><span property="givenName">Lizhong</span> <span property="familyName">Wang</span></a><div id="au23" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au23"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Lizhong</span> <span property="familyName">Wang</span> <a class="orcid-id" href="https://orcid.org/0000-0002-3571-8771" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0002-3571-8771</a></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">WeGene, Shenzhen, Guangdong 518040, PRC</span></div></div><div class="core-author-link"><a href="/authored-by/Wang/Lizhong" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">15</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au24" aria-controls="axel_author_au24"><span property="givenName">Yingxiang</span> <span property="familyName">Li</span></a><div id="au24" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au24"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Yingxiang</span> <span property="familyName">Li</span> <a class="orcid-id" href="https://orcid.org/0000-0003-0835-9280" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0003-0835-9280</a></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">WeGene, Shenzhen, Guangdong 518040, PRC</span></div></div><div class="core-author-link"><a href="/authored-by/Li/Yingxiang" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">15</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au25" aria-controls="axel_author_au25"><span property="givenName">Senwei</span> <span property="familyName">Tang</span></a><div id="au25" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au25"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Senwei</span> <span property="familyName">Tang</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">WeGene, Shenzhen, Guangdong 518040, PRC</span></div></div><div class="core-author-link"><a href="/authored-by/Tang/Senwei" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">15</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au26" aria-controls="axel_author_au26"><span property="givenName">Xiaoli</span> <span property="familyName">Wu</span></a><div id="au26" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au26"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Xiaoli</span> <span property="familyName">Wu</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">WeGene, Shenzhen, Guangdong 518040, PRC</span></div></div><div class="core-author-link"><a href="/authored-by/Wu/Xiaoli" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">15</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au27" aria-controls="axel_author_au27"><span property="givenName">Robin G.</span> <span property="familyName">Walters</span></a><div id="au27" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au27"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Robin G.</span> <span property="familyName">Walters</span> <a class="orcid-id" href="https://orcid.org/0000-0002-9179-0321" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0002-9179-0321</a></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Clinical Trial Service Unit and Epidemiological Studies Unit, Nuffield Department of Clinical Medicine, University of Oxford, Oxford, UK</span></div><div property="affiliation" typeof="Organization"><span property="name">Medical Research Council Population Health Research Unit, Nuffield Department of Clinical Medicine, University of Oxford, Oxford, UK</span></div></div><div class="core-author-link"><a href="/authored-by/Walters/Robin+G" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">16</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">17</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au28" aria-controls="axel_author_au28"><span property="givenName">Kuang</span> <span property="familyName">Lin</span></a><div id="au28" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au28"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Kuang</span> <span property="familyName">Lin</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Clinical Trial Service Unit and Epidemiological Studies Unit, Nuffield Department of Clinical Medicine, University of Oxford, Oxford, UK</span></div></div><div class="core-author-link"><a href="/authored-by/Lin/Kuang" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">16</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au29" aria-controls="axel_author_au29"><span property="givenName">Ruogu</span> <span property="familyName">Meng</span></a><div id="au29" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au29"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Ruogu</span> <span property="familyName">Meng</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Center for Data Science in Health and Medicine, Peking University, Beijing 100191, PRC</span></div></div><div class="core-author-link"><a href="/authored-by/Meng/Ruogu" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">18</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au30" aria-controls="axel_author_au30"><span property="givenName">Jun</span> <span property="familyName">Lv</span></a><div id="au30" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au30"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Jun</span> <span property="familyName">Lv</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Epidemiology and Biostatistics, School of Public Health, Peking University Health Science Center, Beijing 100191, PRC</span></div></div><div class="core-author-link"><a href="/authored-by/Lv/Jun" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">19</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au31" aria-controls="axel_author_au31"><span property="givenName">Jonathan M.</span> <span property="familyName">Chernus</span></a><div id="au31" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au31"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Jonathan M.</span> <span property="familyName">Chernus</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Human Genetics, University of Pittsburgh, Pittsburgh, PA 15261, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Chernus/Jonathan+M" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">20</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au32" aria-controls="axel_author_au32"><span property="givenName">Katherine</span> <span property="familyName">Neiswanger</span></a><div id="au32" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au32"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Katherine</span> <span property="familyName">Neiswanger</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Center for Craniofacial and Dental Genetics, Department of Oral and Craniofacial Sciences, University of Pittsburgh, Pittsburgh, PA 15219, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Neiswanger/Katherine" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">21</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au33" aria-controls="axel_author_au33"><span property="givenName">Eleanor</span> <span property="familyName">Feingold</span></a><div id="au33" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au33"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Eleanor</span> <span property="familyName">Feingold</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Human Genetics, University of Pittsburgh, Pittsburgh, PA 15261, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Feingold/Eleanor" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">20</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au34" aria-controls="axel_author_au34"><span property="givenName">David M.</span> <span property="familyName">Evans</span></a><div id="au34" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au34"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">David M.</span> <span property="familyName">Evans</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">University of Queensland Diamantina Institute, University of Queensland, Brisbane, QLD, Australia</span></div><div property="affiliation" typeof="Organization"><span property="name">Institute for Molecular Bioscience, University of Queensland, Brisbane, QLD, Australia</span></div><div property="affiliation" typeof="Organization"><span property="name">MRC Integrative Epidemiology Unit, University of Bristol, Bristol, UK</span></div></div><div class="core-author-link"><a href="/authored-by/Evans/David+M" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">12</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">22</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">23</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au35" aria-controls="axel_author_au35"><span property="givenName">Sarah E.</span> <span property="familyName">Medland</span></a><div id="au35" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au35"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Sarah E.</span> <span property="familyName">Medland</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">QIMR Berghofer Medical Rese Institute, Brisbane, QLD, Australia</span></div></div><div class="core-author-link"><a href="/authored-by/Medland/Sarah+E" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">13</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au36" aria-controls="axel_author_au36"><span property="givenName">Nicholas G.</span> <span property="familyName">Martin</span></a><div id="au36" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au36"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Nicholas G.</span> <span property="familyName">Martin</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">QIMR Berghofer Medical Rese Institute, Brisbane, QLD, Australia</span></div></div><div class="core-author-link"><a href="/authored-by/Martin/Nicholas+G" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">13</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au37" aria-controls="axel_author_au37"><span property="givenName">Seth M.</span> <span property="familyName">Weinberg</span></a><div id="au37" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au37"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Seth M.</span> <span property="familyName">Weinberg</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Human Genetics, University of Pittsburgh, Pittsburgh, PA 15261, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Center for Craniofacial and Dental Genetics, Department of Oral and Craniofacial Sciences, University of Pittsburgh, Pittsburgh, PA 15219, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Anthropology, University of Pittsburgh, Pittsburgh, PA 15260, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Weinberg/Seth+M" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">20</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">21</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">24</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au38" aria-controls="axel_author_au38"><span property="givenName">Mary L.</span> <span property="familyName">Marazita</span></a><div id="au38" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au38"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Mary L.</span> <span property="familyName">Marazita</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Human Genetics, University of Pittsburgh, Pittsburgh, PA 15261, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Center for Craniofacial and Dental Genetics, Department of Oral and Craniofacial Sciences, University of Pittsburgh, Pittsburgh, PA 15219, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Clinical and Translational Science, University of Pittsburgh, Pittsburgh, PA 15261, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Psychiatry, University of Pittsburgh, Pittsburgh, PA 15261, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Marazita/Mary+L" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">20</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">21</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">25</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">26</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au39" aria-controls="axel_author_au39"><span property="givenName">Gang</span> <span property="familyName">Chen</span></a><div id="au39" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au39"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Gang</span> <span property="familyName">Chen</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">WeGene, Shenzhen, Guangdong 518040, PRC</span></div></div><div class="core-author-link"><a href="/authored-by/Chen/Gang" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">15</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au40" aria-controls="axel_author_au40"><span property="givenName">Zhengming</span> <span property="familyName">Chen</span></a><div id="au40" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au40"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Zhengming</span> <span property="familyName">Chen</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Clinical Trial Service Unit and Epidemiological Studies Unit, Nuffield Department of Clinical Medicine, University of Oxford, Oxford, UK</span></div><div property="affiliation" typeof="Organization"><span property="name">Medical Research Council Population Health Research Unit, Nuffield Department of Clinical Medicine, University of Oxford, Oxford, UK</span></div></div><div class="core-author-link"><a href="/authored-by/Chen/Zhengming" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">16</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">17</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au41" aria-controls="axel_author_au41"><span property="givenName">Yong</span> <span property="familyName">Zhou</span></a><div id="au41" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au41"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Yong</span> <span property="familyName">Zhou</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Clinical Research Institute, Shanghai General Hospital, Shanghai Jiao Tong University School of Medicine, Shanghai, PRC</span></div></div><div class="core-author-link"><a href="/authored-by/Zhou/Yong" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">27</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au42" aria-controls="axel_author_au42"><span property="givenName">Michael</span> <span property="familyName">Cheeseman</span></a><div id="au42" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au42"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Michael</span> <span property="familyName">Cheeseman</span> <a class="orcid-id" href="https://orcid.org/0000-0001-5968-0176" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0001-5968-0176</a></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">The Roslin Institute and Royal (Dick) School of Veterinary Studies, University of Edinburgh, Edinburgh, UK</span></div></div><div class="core-author-link"><a href="/authored-by/Cheeseman/Michael" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">3</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au43" aria-controls="axel_author_au43"><span property="givenName">Lan</span> <span property="familyName">Wang</span></a><div id="au43" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au43"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Lan</span> <span property="familyName">Wang</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Chinese Academy of Sciences Key Laboratory of Tissue Microenvironment and Tumor, Shanghai Institute of Nutrition and Health, Shanghai Institutes for Biological Sciences, University of Chinese Academy of Sciences, Chinese Academy of Sciences, Shanghai 200031, PRC</span></div></div><div class="core-author-link"><a href="/authored-by/Wang/Lan" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">6</a></sup></span></span> ∙ <span property="author" typeof="Person" role="listitem" class="corresponding-author"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au44" aria-controls="axel_author_au44"><span property="givenName">Li</span> <span property="familyName">Jin</span></a><div id="au44" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au44"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Li</span> <span property="familyName">Jin</span></div><div class="content"><div class="core-author-emails"><div class="label">Correspondence</div><div>Corresponding author</div><div><a id="eadd0010" href="mailto:lijin@fudan.edu.cn" property="email" title="Send email to Li Jin">lijin@fudan.edu.cn</a></div></div><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">31</div><div id="fn2"><div id="ntpara0015" role="paragraph">Senior author</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">State Key Laboratory of Genetic Engineering, Collaborative Innovation Center for Genetics and Development, School of Life Sciences, and Human Phenome Institute, Fudan University, Shanghai 200438, PRC</span></div><div property="affiliation" typeof="Organization"><span property="name">CAS Key Laboratory of Computational Biology, Shanghai Institute of Nutrition and Health, University of Chinese Academy of Sciences, Chinese Academy of Sciences, Shanghai 200031, PRC</span></div><div property="affiliation" typeof="Organization"><span property="name">Research Unit of Dissecting the Population Genetics and Developing New Technologies for Treatment and Prevention of Skin Phenotypes and Dermatological Diseases (2019RU058), Chinese Academy of Medical Sciences, Shanghai 200438, PRC</span></div></div><div class="core-author-link"><a href="/authored-by/Jin/Li" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">2</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">28</a>,<a role="doc-noteref">31</a></sup> <a id="eadd0010" href="mailto:lijin@fudan.edu.cn" property="email" title="Send email to Li Jin">lijin@fudan.edu.cn</a></span> ∙ <span property="author" typeof="Person" role="listitem" class="corresponding-author"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au45" aria-controls="axel_author_au45"><span property="givenName">Denis J.</span> <span property="familyName">Headon</span></a><div id="au45" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au45"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Denis J.</span> <span property="familyName">Headon</span> <a class="orcid-id" href="https://orcid.org/0000-0002-0452-4480" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0002-0452-4480</a></div><div class="content"><div class="core-author-emails"><div class="label">Correspondence</div><div>Corresponding author</div><div><a id="eadd0015" href="mailto:denis.headon@roslin.ed.ac.uk" property="email" title="Send email to Denis J. Headon">denis.headon@roslin.ed.ac.uk</a></div></div><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">31</div><div id="fn2"><div id="ntpara0015" role="paragraph">Senior author</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">The Roslin Institute and Royal (Dick) School of Veterinary Studies, University of Edinburgh, Edinburgh, UK</span></div></div><div class="core-author-link"><a href="/authored-by/Headon/Denis+J" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">3</a>,<a role="doc-noteref">31</a></sup> <a id="eadd0015" href="mailto:denis.headon@roslin.ed.ac.uk" property="email" title="Send email to Denis J. Headon">denis.headon@roslin.ed.ac.uk</a></span> ∙ <span property="author" typeof="Person" role="listitem" class="corresponding-author"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au46" aria-controls="axel_author_au46"><span property="givenName">Sijia</span> <span property="familyName">Wang</span></a><div id="au46" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au46"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Sijia</span> <span property="familyName">Wang</span> <a class="orcid-id" href="https://orcid.org/0000-0001-6961-7867" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0001-6961-7867</a></div><div class="content"><div class="core-author-emails"><div class="label">Correspondence</div><div>Corresponding author</div><div><a id="eadd0020" href="mailto:wangsijia@picb.ac.cn" property="email" title="Send email to Sijia Wang">wangsijia@picb.ac.cn</a></div></div><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">31</div><div id="fn2"><div id="ntpara0015" role="paragraph">Senior author</div></div></div><div role="doc-footnote" data-has="label"><div class="label">33</div><div id="fn4"><div id="ntpara0025" role="paragraph">Lead contact</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">CAS Key Laboratory of Computational Biology, Shanghai Institute of Nutrition and Health, University of Chinese Academy of Sciences, Chinese Academy of Sciences, Shanghai 200031, PRC</span></div><div property="affiliation" typeof="Organization"><span property="name">Center for Excellence in Animal Evolution and Genetics, Chinese Academy of Sciences, Kunming 650223, PRC</span></div></div><div class="core-author-link"><a href="/authored-by/Wang/Sijia" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">2</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">29</a>,<a role="doc-noteref">31</a>,<a role="doc-noteref">33</a></sup> <a id="eadd0020" href="mailto:wangsijia@picb.ac.cn" property="email" title="Send email to Sijia Wang">wangsijia@picb.ac.cn</a></span><span data-displayed-on="all"> … <span><span data-action="reveal" tabindex="0" role="listitem">Show more</span></span> </span><span data-displayed-on="none" data-on-display="all" aria-hidden="true"> <span data-action="hide" tabindex="0" role="listitem">Show less</span></span></span></span></div><div class="core-self-citation"><div class="expandable-triggers"><button aria-controls="core-affiliations-notes" aria-expanded="false" tabindex="0" role="button"><span>Affiliations &amp; Notes</span><i class="icon-arrow_d_n" aria-hidden="true"></i></button><button aria-controls="core-content-info" aria-expanded="false" tabindex="0" role="button"><span>Article Info</span><i class="icon-arrow_d_n" aria-hidden="true"></i></button></div><div id="core-linked-content">
    

        

</div><div class="expandable-content"><div id="core-affiliations-notes" role="region" data-expandable="item" style="display: none;"><div class="affiliations"><div property="affiliation" typeof="Organization"><span class="label">1</span><span id="aff1" property="name">State Key Laboratory of Genetic Engineering, Collaborative Innovation Center for Genetics and Development, School of Life Sciences, and Human Phenome Institute, Fudan University, Shanghai 200438, PRC</span></div><div property="affiliation" typeof="Organization"><span class="label">2</span><span id="aff2" property="name">CAS Key Laboratory of Computational Biology, Shanghai Institute of Nutrition and Health, University of Chinese Academy of Sciences, Chinese Academy of Sciences, Shanghai 200031, PRC</span></div><div property="affiliation" typeof="Organization"><span class="label">3</span><span id="aff3" property="name">The Roslin Institute and Royal (Dick) School of Veterinary Studies, University of Edinburgh, Edinburgh, UK</span></div><div property="affiliation" typeof="Organization"><span class="label">4</span><span id="aff4" property="name">Ministry of Education Key Laboratory of Contemporary Anthropology, Department of Anthropology and Human Genetics, School of Life Sciences, Fudan University, Shanghai 200438, PRC</span></div><div property="affiliation" typeof="Organization"><span class="label">5</span><span id="aff5" property="name">Centre for Genetic Disorders, Institute of Science, Banaras Hindu University, Varanasi, Uttar Pradesh, India</span></div><div property="affiliation" typeof="Organization"><span class="label">6</span><span id="aff6" property="name">Chinese Academy of Sciences Key Laboratory of Tissue Microenvironment and Tumor, Shanghai Institute of Nutrition and Health, Shanghai Institutes for Biological Sciences, University of Chinese Academy of Sciences, Chinese Academy of Sciences, Shanghai 200031, PRC</span></div><div property="affiliation" typeof="Organization"><span class="label">7</span><span id="aff7" property="name">Centre for Regenerative Medicine, University of Edinburgh, Edinburgh, UK</span></div><div property="affiliation" typeof="Organization"><span class="label">8</span><span id="aff8" property="name">MRC Centre for Reproductive Health, Queens Medical Research Institute, University of Edinburgh, Edinburgh, UK</span></div><div property="affiliation" typeof="Organization"><span class="label">9</span><span id="aff9" property="name">Fudan-Taizhou Institute of Health Sciences, Taizhou, Jiangsu 225326, PRC</span></div><div property="affiliation" typeof="Organization"><span class="label">10</span><span id="aff10" property="name">Shanghai Foreign Language School, Shanghai 200083, PRC</span></div><div property="affiliation" typeof="Organization"><span class="label">11</span><span id="aff11" property="name">Forest Ridge School of the Sacred Heart, Bellevue, WA 98006, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">12</span><span id="aff12" property="name">University of Queensland Diamantina Institute, University of Queensland, Brisbane, QLD, Australia</span></div><div property="affiliation" typeof="Organization"><span class="label">13</span><span id="aff13" property="name">QIMR Berghofer Medical Rese Institute, Brisbane, QLD, Australia</span></div><div property="affiliation" typeof="Organization"><span class="label">14</span><span id="aff14" property="name">Psychology Department, La Trobe University, Melbourne, VIC, Australia</span></div><div property="affiliation" typeof="Organization"><span class="label">15</span><span id="aff15" property="name">WeGene, Shenzhen, Guangdong 518040, PRC</span></div><div property="affiliation" typeof="Organization"><span class="label">16</span><span id="aff16" property="name">Clinical Trial Service Unit and Epidemiological Studies Unit, Nuffield Department of Clinical Medicine, University of Oxford, Oxford, UK</span></div><div property="affiliation" typeof="Organization"><span class="label">17</span><span id="aff17" property="name">Medical Research Council Population Health Research Unit, Nuffield Department of Clinical Medicine, University of Oxford, Oxford, UK</span></div><div property="affiliation" typeof="Organization"><span class="label">18</span><span id="aff18" property="name">Center for Data Science in Health and Medicine, Peking University, Beijing 100191, PRC</span></div><div property="affiliation" typeof="Organization"><span class="label">19</span><span id="aff19" property="name">Department of Epidemiology and Biostatistics, School of Public Health, Peking University Health Science Center, Beijing 100191, PRC</span></div><div property="affiliation" typeof="Organization"><span class="label">20</span><span id="aff20" property="name">Department of Human Genetics, University of Pittsburgh, Pittsburgh, PA 15261, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">21</span><span id="aff21" property="name">Center for Craniofacial and Dental Genetics, Department of Oral and Craniofacial Sciences, University of Pittsburgh, Pittsburgh, PA 15219, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">22</span><span id="aff22" property="name">Institute for Molecular Bioscience, University of Queensland, Brisbane, QLD, Australia</span></div><div property="affiliation" typeof="Organization"><span class="label">23</span><span id="aff23" property="name">MRC Integrative Epidemiology Unit, University of Bristol, Bristol, UK</span></div><div property="affiliation" typeof="Organization"><span class="label">24</span><span id="aff24" property="name">Department of Anthropology, University of Pittsburgh, Pittsburgh, PA 15260, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">25</span><span id="aff25" property="name">Clinical and Translational Science, University of Pittsburgh, Pittsburgh, PA 15261, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">26</span><span id="aff26" property="name">Department of Psychiatry, University of Pittsburgh, Pittsburgh, PA 15261, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">27</span><span id="aff27" property="name">Clinical Research Institute, Shanghai General Hospital, Shanghai Jiao Tong University School of Medicine, Shanghai, PRC</span></div><div property="affiliation" typeof="Organization"><span class="label">28</span><span id="aff28" property="name">Research Unit of Dissecting the Population Genetics and Developing New Technologies for Treatment and Prevention of Skin Phenotypes and Dermatological Diseases (2019RU058), Chinese Academy of Medical Sciences, Shanghai 200438, PRC</span></div><div property="affiliation" typeof="Organization"><span class="label">29</span><span id="aff29" property="name">Center for Excellence in Animal Evolution and Genetics, Chinese Academy of Sciences, Kunming 650223, PRC</span></div></div><div class="core-authors-notes"><div role="doc-footnote" data-has="label"><div class="label">30</div><div id="fn1"><div id="ntpara0010" role="paragraph">These authors contributed equally</div></div></div><div role="doc-footnote" data-has="label"><div class="label">31</div><div id="fn2"><div id="ntpara0015" role="paragraph">Senior author</div></div></div><div role="doc-footnote" data-has="label"><div class="label">32</div><div id="fn3"><div id="ntpara0020" role="paragraph">Present address: The Core Laboratory in Medical Center of Clinical Research, Department of Molecular Diagnostics and Endocrinology, Shanghai Ninth People’s Hospital, State Key Laboratory of Medical Genomics, Shanghai Jiao Tong University School of Medicine, Shanghai 200011, PRC</div></div></div><div role="doc-footnote" data-has="label"><div class="label">33</div><div id="fn4"><div id="ntpara0025" role="paragraph">Lead contact</div></div></div></div></div><div id="core-content-info" role="region" data-expandable="item" style="display: none;"><div class="history"><div>Publication History:</div><div><span>Received April 16, 2021</span>; <span>Revised October 20, 2021</span>; <span>Accepted December 8, 2021</span></div></div><div class="core-links"><span class="doi">DOI: <a href="https://doi.org/10.1016/j.cell.2021.12.008" property="sameAs" target="_blank">10.1016/j.cell.2021.12.008 <span class="sr-only">External Link</span></a></span><span class="science-direct">Also available on <a href="https://www.sciencedirect.com/science/article/pii/S009286742101446X" target="_blank">ScienceDirect <span class="sr-only">External Link</span></a></span></div><div role="paragraph">Copyright: Crown Copyright © 2022 Published by Elsevier Inc.</div><div class="core-links"><span>User License: <a href="http://www.elsevier.com/open-access/userlicense/1.0/" target="_blank">Elsevier user license</a> | <a href="https://www.elsevier.com/open-access" target="_blank">Elsevier's open access license policy</a></span></div><div class="core-crossmark-badge"><a data-target="crossmark" href="#" title="Check for updates on crossmark" data-doi="10.1016/j.cell.2021.12.008" data-id="article-info-crossmark" class="crossmark__link" style="cursor: pointer;"><svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 290 58" height="22px" role="presentation"><path fill="url(#horizontal_crossmark)" stroke="#948F8F" stroke-miterlimit="10" stroke-width=".75" d="M287 1H3a2 2 0 0 0-2 2v52a2 2 0 0 0 2 2h284a2 2 0 0 0 2-2V3a2 2 0 0 0-2-2Z"></path><path fill="#535353" d="M80.3 23.58a6.395 6.395 0 0 0-.55-1.43 3.92 3.92 0 0 0-2.19-1.86 5.099 5.099 0 0 0-1.71-.29 5 5 0 0 0-2.46.58 4.831 4.831 0 0 0-1.69 1.54 6.921 6.921 0 0 0-1 2.19 10.16 10.16 0 0 0 0 5 6.921 6.921 0 0 0 1 2.19 4.83 4.83 0 0 0 1.69 1.5 5 5 0 0 0 2.46.58 4.44 4.44 0 0 0 1.85-.36 4.12 4.12 0 0 0 1.38-1c.39-.436.695-.942.9-1.49A7.094 7.094 0 0 0 80.4 29h3a8.46 8.46 0 0 1-.69 2.92 7.179 7.179 0 0 1-1.59 2.29 7 7 0 0 1-2.35 1.49 8.14 8.14 0 0 1-3 .52 8.43 8.43 0 0 1-3.59-.74 7.88 7.88 0 0 1-2.69-2 9.001 9.001 0 0 1-1.69-3 11.41 11.41 0 0 1 0-7.23 9.001 9.001 0 0 1 1.69-3 8.001 8.001 0 0 1 2.69-2 8.3 8.3 0 0 1 3.59-.75 9.44 9.44 0 0 1 2.83.41c.866.242 1.68.642 2.4 1.18A6.229 6.229 0 0 1 82.65 21a6.66 6.66 0 0 1 .78 2.58H80.3ZM85.9 17.9h2.85v6.63h.05a4.15 4.15 0 0 1 1.56-1.46 4.61 4.61 0 0 1 2.31-.59 4.83 4.83 0 0 1 3.33 1.1 4.21 4.21 0 0 1 1.22 3.3v8.88H94.4v-8.13a3.51 3.51 0 0 0-.65-2.21 2.36 2.36 0 0 0-1.87-.69 3.1 3.1 0 0 0-1.3.26 2.821 2.821 0 0 0-1 .73A3.41 3.41 0 0 0 89 26.8a3.88 3.88 0 0 0-.23 1.33v7.63H85.9V17.9ZM102.47 30c-.002.491.069.98.21 1.45.131.448.348.866.64 1.23a3.08 3.08 0 0 0 1.08.84 3.543 3.543 0 0 0 1.55.31 3.431 3.431 0 0 0 2-.54c.541-.402.94-.966 1.14-1.61h2.7A5.349 5.349 0 0 1 109.7 35a5.715 5.715 0 0 1-1.74.85 7 7 0 0 1-2 .29 6.848 6.848 0 0 1-2.7-.5 5.577 5.577 0 0 1-2-1.4A5.953 5.953 0 0 1 100 32a8.748 8.748 0 0 1-.41-2.75 7.768 7.768 0 0 1 .44-2.61 6.67 6.67 0 0 1 1.25-2.17 5.863 5.863 0 0 1 4.56-2 5.79 5.79 0 0 1 2.74.64c.783.41 1.466.986 2 1.69a6.824 6.824 0 0 1 1.16 2.41c.264.905.352 1.852.26 2.79h-9.53Zm6.65-1.87a4.605 4.605 0 0 0-.29-1.3c-.148-.4-.368-.77-.65-1.09a3.22 3.22 0 0 0-1-.75 3 3 0 0 0-1.31-.29 3.431 3.431 0 0 0-1.36.26 3.108 3.108 0 0 0-1 .73 3.594 3.594 0 0 0-.7 1.09 3.798 3.798 0 0 0-.29 1.35h6.65-.05ZM122.88 27.15a2.666 2.666 0 0 0-1-1.81 3.14 3.14 0 0 0-2-.61c-.4.005-.798.07-1.18.19-.435.136-.83.376-1.15.7a3.92 3.92 0 0 0-.87 1.43 6.783 6.783 0 0 0-.35 2.39c.002.539.065 1.076.19 1.6.113.509.316.993.6 1.43.273.41.635.752 1.06 1a3 3 0 0 0 1.58.39 2.872 2.872 0 0 0 2.06-.77 3.596 3.596 0 0 0 1-2.18h2.85a6.26 6.26 0 0 1-1.91 3.86 5.83 5.83 0 0 1-4 1.34 6.61 6.61 0 0 1-2.69-.51 5.695 5.695 0 0 1-2-1.4 6.001 6.001 0 0 1-1.21-2.11 8.279 8.279 0 0 1-.41-2.65 8.987 8.987 0 0 1 .4-2.72 6.237 6.237 0 0 1 1.2-2.21 5.614 5.614 0 0 1 2-1.47 6.7 6.7 0 0 1 2.79-.54 7.801 7.801 0 0 1 2.14.29 5.634 5.634 0 0 1 1.8.86c.529.386.969.88 1.29 1.45a5 5 0 0 1 .6 2.07h-2.85l.06-.02ZM128.13 17.9H131V28l5.15-5.22h3.5l-5 4.75 5.42 8.17h-3.47l-4-6.27-1.6 1.7v4.63h-2.85V17.9h-.02ZM147.15 22.83h2.13v-1.08a5.505 5.505 0 0 1 .3-2c.161-.456.436-.862.8-1.18.33-.273.721-.462 1.14-.55.44-.094.89-.141 1.34-.14a8.158 8.158 0 0 1 2 .18v2.24a4.113 4.113 0 0 0-.59-.11 6.343 6.343 0 0 0-.74 0 1.74 1.74 0 0 0-1 .27 1.204 1.204 0 0 0-.41 1.08v1.33h2.42V25h-2.42v10.8h-2.85V25h-2.12v-2.17ZM162.23 36.1a6.999 6.999 0 0 1-2.76-.51 5.877 5.877 0 0 1-2-1.41 5.998 5.998 0 0 1-1.33-2.18 8.775 8.775 0 0 1 0-5.48 6.008 6.008 0 0 1 1.27-2.15 5.894 5.894 0 0 1 2-1.41A7.71 7.71 0 0 1 165 23a5.894 5.894 0 0 1 2 1.41 6.003 6.003 0 0 1 1.28 2.15c.585 1.78.585 3.7 0 5.48a5.99 5.99 0 0 1-3.28 3.54 7.002 7.002 0 0 1-2.77.52Zm0-2.25a3.268 3.268 0 0 0 1.65-.4c.447-.262.83-.621 1.12-1.05.304-.444.528-.938.66-1.46a6.581 6.581 0 0 0 0-3.31 4.416 4.416 0 0 0-.66-1.46 3.526 3.526 0 0 0-1.15-1 3.6 3.6 0 0 0-3.3 0 3.526 3.526 0 0 0-1.15 1 4.414 4.414 0 0 0-.66 1.46 6.558 6.558 0 0 0 0 3.31c.132.522.356 1.016.66 1.46a3.47 3.47 0 0 0 1.15 1.05 3.26 3.26 0 0 0 1.68.4ZM171.13 22.83h2.67v2.5c.097-.362.264-.701.49-1 .251-.341.547-.647.88-.91a4.404 4.404 0 0 1 1.14-.66 3.41 3.41 0 0 1 1.28-.25c.23-.013.46-.013.69 0h.39v2.79l-.61-.09a5.173 5.173 0 0 0-.61 0 3.278 3.278 0 0 0-1.36.29 3.131 3.131 0 0 0-1.11.85 4.171 4.171 0 0 0-.75 1.39 6 6 0 0 0-.28 1.9v6.15h-2.85V22.83h.03ZM198.22 35.75h-2.8V34a3.907 3.907 0 0 1-1.56 1.56 4.25 4.25 0 0 1-2.11.59 4.742 4.742 0 0 1-3.75-1.31 5.537 5.537 0 0 1-1.12-3.84v-8.2h2.85v7.92a3.503 3.503 0 0 0 .65 2.4 2.38 2.38 0 0 0 1.83.7 3.599 3.599 0 0 0 1.5-.27 2.66 2.66 0 0 0 1-.74c.259-.326.444-.705.54-1.11a5.79 5.79 0 0 0 .16-1.4v-7.5h2.85v12.95h-.04ZM201.3 22.83h2.7v1.75a3.523 3.523 0 0 1 1.72-1.58 5.568 5.568 0 0 1 2.33-.49 6.17 6.17 0 0 1 2.66.54 5.246 5.246 0 0 1 1.89 1.47c.515.65.899 1.394 1.13 2.19.257.87.385 1.773.38 2.68.001.851-.114 1.7-.34 2.52a6.568 6.568 0 0 1-1 2.16 5.002 5.002 0 0 1-4.21 2.06c-.419 0-.837-.036-1.25-.11a5.39 5.39 0 0 1-1.2-.36A4.737 4.737 0 0 1 205 35a3.567 3.567 0 0 1-.81-.91v6.45h-2.89V22.83Zm10 6.47c0-.574-.078-1.146-.23-1.7a4.606 4.606 0 0 0-.67-1.46 3.475 3.475 0 0 0-1.12-1 3.06 3.06 0 0 0-1.55-.39A3.141 3.141 0 0 0 205 26a5.518 5.518 0 0 0-1 3.3c-.004.611.077 1.22.24 1.81.138.523.38 1.012.71 1.44a3.55 3.55 0 0 0 1.14 1 3.242 3.242 0 0 0 1.54.35 3.18 3.18 0 0 0 1.65-.4c.438-.25.817-.59 1.11-1 .299-.44.513-.931.63-1.45a7.38 7.38 0 0 0 .23-1.75h.05ZM228.67 35.75H226V34a3.38 3.38 0 0 1-1.67 1.61 5.668 5.668 0 0 1-2.33.49 6.17 6.17 0 0 1-2.66-.54 5.307 5.307 0 0 1-1.89-1.46 6.254 6.254 0 0 1-1.12-2.19 9.48 9.48 0 0 1-.37-2.71 8.67 8.67 0 0 1 .47-3 6.058 6.058 0 0 1 1.26-2.1 4.89 4.89 0 0 1 1.8-1.21 5.738 5.738 0 0 1 2.06-.39c.412 0 .824.037 1.23.11.413.073.815.194 1.2.36.38.166.736.381 1.06.64.32.255.594.562.81.91V17.9h2.85v17.85h-.03Zm-10-6.33c-.001.55.069 1.098.21 1.63.13.511.35.996.65 1.43.288.414.668.757 1.11 1a3.241 3.241 0 0 0 1.6.38 3.199 3.199 0 0 0 1.64-.4 3.398 3.398 0 0 0 1.13-1.05c.297-.447.517-.94.65-1.46.14-.542.211-1.1.21-1.66a5.227 5.227 0 0 0-1-3.35 3.175 3.175 0 0 0-2.61-1.2 3.23 3.23 0 0 0-1.69.41 3.478 3.478 0 0 0-1.12 1.07 4.494 4.494 0 0 0-.62 1.5 7.871 7.871 0 0 0-.11 1.7h-.05ZM242.6 32.88c-.018.258.03.516.14.75a.577.577 0 0 0 .54.23h.3c.135-.002.269-.019.4-.05v2l-.39.11-.49.08-.5.08h-.42a2.75 2.75 0 0 1-1.45-.35 1.731 1.731 0 0 1-.75-1.23 4.998 4.998 0 0 1-2.09 1.2 8.224 8.224 0 0 1-2.39.38 5.861 5.861 0 0 1-1.67-.24 4.394 4.394 0 0 1-1.41-.7 3.35 3.35 0 0 1-1-1.18 3.624 3.624 0 0 1-.36-1.66 3.84 3.84 0 0 1 .44-2 3.246 3.246 0 0 1 1.15-1.17 5.195 5.195 0 0 1 1.6-.61c.593-.127 1.19-.223 1.79-.29a14.77 14.77 0 0 1 1.47-.21 6.896 6.896 0 0 0 1.24-.21c.318-.077.611-.236.85-.46a1.286 1.286 0 0 0 .31-.94 1.458 1.458 0 0 0-.26-.9A1.759 1.759 0 0 0 239 25a2.807 2.807 0 0 0-.86-.25 6.941 6.941 0 0 0-.9-.06 3.58 3.58 0 0 0-2 .5 2 2 0 0 0-.87 1.55h-2.85c.021-.73.227-1.442.6-2.07.338-.54.797-.996 1.34-1.33a5.475 5.475 0 0 1 1.84-.7c.692-.134 1.395-.2 2.1-.2.632 0 1.262.067 1.88.2a5.337 5.337 0 0 1 1.66.65 3.6 3.6 0 0 1 1.19 1.16c.314.525.47 1.129.45 1.74v6.65l.02.04Zm-2.85-3.6a3.45 3.45 0 0 1-1.6.51c-.667.06-1.3.147-1.9.26a6.137 6.137 0 0 0-.87.21 2.64 2.64 0 0 0-.75.38c-.22.164-.395.38-.51.63a2.129 2.129 0 0 0-.19.94c-.011.29.085.575.27.8.182.215.407.388.66.51.269.128.555.216.85.26.277.05.558.076.84.08a4.63 4.63 0 0 0 1-.12 3.418 3.418 0 0 0 1-.43 2.73 2.73 0 0 0 .81-.76 1.93 1.93 0 0 0 .32-1.14v-2.14l.07.01ZM244.37 22.83h2.15V19h2.85v3.88h2.58V25h-2.58v6.9a7.064 7.064 0 0 0 0 .77c.016.194.074.381.17.55a.84.84 0 0 0 .41.34 2 2 0 0 0 .75.11h.6c.203-.01.404-.04.6-.09v2.2l-.93.1a8.403 8.403 0 0 1-.92 0 6.33 6.33 0 0 1-1.81-.21 2.421 2.421 0 0 1-1.08-.62 2.15 2.15 0 0 1-.52-1 7.791 7.791 0 0 1-.16-1.42V25h-2.15v-2.17h.04ZM256.22 30c-.002.491.069.98.21 1.45.131.448.348.866.64 1.23a3.08 3.08 0 0 0 1.08.84 3.543 3.543 0 0 0 1.55.31 3.431 3.431 0 0 0 2-.54c.541-.402.94-.966 1.14-1.61h2.7a5.352 5.352 0 0 1-2.1 3.26 5.715 5.715 0 0 1-1.74.85 7 7 0 0 1-2 .29 6.848 6.848 0 0 1-2.7-.5 5.577 5.577 0 0 1-2-1.4 5.947 5.947 0 0 1-1.21-2.18 8.754 8.754 0 0 1-.41-2.75 7.758 7.758 0 0 1 .44-2.61 6.67 6.67 0 0 1 1.25-2.17 5.863 5.863 0 0 1 4.56-2 5.79 5.79 0 0 1 2.74.64c.783.41 1.466.986 2 1.69a6.824 6.824 0 0 1 1.16 2.41c.243.916.308 1.87.19 2.81h-9.5V30Zm6.65-1.87a4.605 4.605 0 0 0-.29-1.3c-.148-.4-.368-.77-.65-1.09a3.22 3.22 0 0 0-1-.75 3 3 0 0 0-1.31-.29 3.431 3.431 0 0 0-1.36.26 3.108 3.108 0 0 0-1 .73 3.575 3.575 0 0 0-.7 1.09 3.798 3.798 0 0 0-.29 1.35h6.65-.05ZM270 31.6a2.154 2.154 0 0 0 .95 1.75c.607.35 1.3.524 2 .5.304-.002.608-.022.91-.06a3.943 3.943 0 0 0 1-.24c.286-.105.54-.28.74-.51a1.23 1.23 0 0 0 .26-.89 1.261 1.261 0 0 0-.4-.9 2.779 2.779 0 0 0-1-.56 9.315 9.315 0 0 0-1.34-.36l-1.5-.33a14.219 14.219 0 0 1-1.54-.42 4.937 4.937 0 0 1-1.33-.68 3.12 3.12 0 0 1-.94-1.09 3.472 3.472 0 0 1-.35-1.64 2.94 2.94 0 0 1 .51-1.76 3.913 3.913 0 0 1 1.3-1.15 5.66 5.66 0 0 1 1.75-.61 10.316 10.316 0 0 1 1.84-.17 8.418 8.418 0 0 1 1.91.21 5.19 5.19 0 0 1 1.65.69c.496.318.916.741 1.23 1.24.341.559.55 1.188.61 1.84h-3a1.783 1.783 0 0 0-.94-1.37 3.892 3.892 0 0 0-1.69-.35 6.093 6.093 0 0 0-.71 0 3.327 3.327 0 0 0-.78.19 1.642 1.642 0 0 0-.61.4 1 1 0 0 0-.25.69 1.093 1.093 0 0 0 .36.85c.294.245.634.428 1 .54.437.153.885.274 1.34.36l1.55.33a20.9 20.9 0 0 1 1.53.42c.479.157.931.386 1.34.68.41.277.751.643 1 1.07.255.494.379 1.045.36 1.6a3.48 3.48 0 0 1-.52 2 4.15 4.15 0 0 1-1.36 1.3 6.19 6.19 0 0 1-1.86.73 9.435 9.435 0 0 1-2 .23 8.823 8.823 0 0 1-2.26-.27 5.265 5.265 0 0 1-1.8-.84 4 4 0 0 1-1.2-1.4 4.445 4.445 0 0 1-.46-2h2.7v-.02Z"></path><path fill="#C72914" d="m21.94 38.74 18.12-12.08v-14.5H21.94v26.58Z"></path><path fill="#EF3340" d="M40.06 38.74 21.94 26.66v-14.5h18.12v26.58Z"></path><path fill="#3EB1C8" d="M31 9a20 20 0 1 0 0 40 20 20 0 0 0 0-40Zm0 35.1a15.1 15.1 0 1 1 0-30.2 15.1 15.1 0 0 1 0 30.2Z"></path><path fill="#FFC72C" d="M42.09 39.23A15.09 15.09 0 0 1 17 23.38l-4.2-2.64a20 20 0 0 0 33.49 21.14l-4.2-2.65Z"></path><defs><linearGradient id="horizontal_crossmark" x1="145" x2="145" y1="7.33" y2="56.82" gradientUnits="userSpaceOnUse"><stop stop-color="#fff"></stop><stop offset="1" stop-color="#c4c4c4"></stop></linearGradient></defs></svg></a></div><div class="core-content-info"></div></div></div></div></div></header><div class="core-header-aside">
    

        


    

            

            

            <div class="cover-image cover-image--axel"><div class="cover-image__thumb"><a href="https://www.cell.com/cell/issue?pii=S0092-8674(21)X0002-5"><img src="/cms/asset/1a139b97-bdc3-41c1-88a0-3cfc5396a48f/cov200h.gif" alt="Cover Image - Cell, Volume 185, Issue 1"></a></div></div>

            

        
</div></div><div data-core-wrapper="content"><div id="dummy-sticky-observer"></div><div data-core-nav="header" data-extent="frontmatter"><div class="core-nav-wrapper core-container" aria-label="Article navigation"><div class="navigation-menu-pdf">
    

        


    

            

            

            <div data-widget-id="ecff07e8-0bc4-4533-b1f1-49365e2ead20" data-widget-name="UX3ArticleToolsWidget" class="article-tools"><a href="/action/showPdf?pii=S0092-8674%2821%2901446-X" title="Standard PDF (opens in a new tab)" target="_blank" aria-hidden="false" role="menuitem" class="article-tools__item__pdf"><svg focusable="false" viewBox="0 0 32 32" width="20" height="20" title="PDF" class="pdf-icon"><path id="pdfwhite" d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path id="pdfda0000" d="M.167 2.592H22.39V9.72H.166z" fill="#da0000"></path><path id="pdffff9f9" fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path id="pdff91d0a" d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="basic">Download PDF</span><span class="alternative">Download PDF</span></a></div>

            

        
</div><div class="core-sections-menu"><button data-id="article-toolbar-showhide" aria-controls="article_sections_menu" aria-expanded="false" aria-haspopup="true" aria-label="Toggle section navigation menu" style="margin-top: 0px;"><i class="icon-list" aria-hidden="true"></i><span>Outline</span></button><nav id="article_sections_menu" aria-label="Contents" data-core-nav="article" style="margin-top: 0px;"><div class="heading">Outline<button aria-haspopup="true" aria-controls="article_sections_menu" title="Close outline menu"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div><ul><li><a href="#author-highlights-abstract">Highlights</a></li><li><a href="#author-abstract">Summary</a></li><li><a href="#graphical-abstract">Graphical abstract</a></li><li><a href="#keywords">Keywords</a></li><li><a href="#sec-1">Introduction</a></li><li><a href="#sec-2">Results</a></li><li><a href="#sec-3">Discussion</a></li><li><a href="#sec-4">STAR★Methods</a></li><li><a href="#acknowledgments">Acknowledgments</a></li><li><a href="#supplementary-material">Supplemental information (8)</a></li><li><a href="#app-2">Supporting citations</a></li><li><a href="#data-availability">Data and code availability</a></li><li><a href="#references">References</a></li><li role="none" class="mobile-only"><a role="menuitem" href="#core-collateral-metrics">Article metrics</a></li><li role="none" class="mobile-only"><a role="menuitem" href="#core-collateral-relatedArticles">Related Articles</a></li></ul></nav></div><div class="navigation-menu-share">
    

        


    

            

            

            <!-- Go to https://www.addtoany.com/buttons/customize/ to customize your tools --><script type="text/javascript" defer="defer" src="https://static.addtoany.com/menu/page.js"></script><script type="text/javascript">var a2a_config = a2a_config || {};
a2a_config.thanks = {
    postShare: false,
    ad: false,
};</script><script>var a2a_config = a2a_config || {};
let res = window.location.href.replace(/\(/g, "%28").replace(/\)/g, "%29");
a2a_config.templates = a2a_config.templates || {};

a2a_config.templates.email = {
    body: res
};
</script><div data-widget-id="5f978446-c80a-450c-a185-94d633df2db2" data-widget-name="UX3share" aria-hidden="false" role="group" class="article-tools__dropblock article-tools__share"><button aria-label="Toggle share menu" aria-expanded="false" aria-haspopup="true" role="menuitem" aria-controls="navigation_menu_share_block" aria-labelledby="share-label" class="article-tools__item__share"><i aria-hidden="true" class="icon-article-share"></i><span id="share-label">Share</span></button><div id="navigation_menu_share_block" role="menu" class="share__block axel-share article-tools__dropblock__holder"><h4 class="axel-share__title heading">Share<button title="Close share menu" aria-expanded="false" aria-haspopup="true" aria-controls="navigation_menu_share_block"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></h4><div class="axel-share__inner"><span class="sr-only">Share on</span><ul class="a2a__share-list rlist w-slide--list a2a a2a_kit a2a_kit_size_32 a2a_default_style" data-a2a-url="https://www.cell.com/cell/fulltext/S0092-8674%2821%2901446-X" style="line-height: 32px;"><li class="a2a_listitem_custom"><a role="link" title="Email" class="a2a_button_email" target="_blank" rel="nofollow noopener" href="/#email"><i aria-hidden="true" class="at-icon-wrapper icon-gizmo-email-plus"></i><span>Email</span></a></li><li class="a2a_listitem_custom"><a role="link" title="formerly known as Twitter" class="a2a_button_twitter" target="_blank" rel="nofollow noopener" href="/#twitter"><i aria-hidden="true" class="at-icon-wrapper icon-x-default"></i><span>X</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Facebook" class="a2a_button_facebook" target="_blank" rel="nofollow noopener" href="/#facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook-square-2"></i><span>Facebook</span></a></li><li class="a2a_listitem_custom"><a role="link" title="LinkedIn" class="a2a_button_linkedin" target="_blank" rel="nofollow noopener" href="/#linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin-square"></i><span>LinkedIn</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Sina Weibo" class="a2a_button_sina_weibo" target="_blank" rel="nofollow noopener" href="/#sina_weibo"><i aria-hidden="true" class="at-icon-wrapper icon-weibo"></i><span>Sina Weibo</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Mendeley" class="a2a_button_mendeley" target="_blank" rel="nofollow noopener" href="/#mendeley"><i aria-hidden="true" class="at-icon-wrapper icon-mendeley-2"></i><span>Mendeley</span></a></li><li class="a2a_listitem_custom"><a role="link" title="bluesky" class="a2a_button_bluesky" target="_blank" rel="nofollow noopener" href="/#bluesky"><i aria-hidden="true" class="at-icon-wrapper icon-bluesky-logo"></i><span>bluesky</span></a></li><div style="clear: both;"></div></ul><div class="extra-links-drop-zone">
    

        


    

            

            

            <div data-widget-id="f1a208d3-ac5e-4db2-9c3a-40ef97ddfa10" data-widget-name="UX3HTMLWidget"><a class="axel-share__extra" target="_blank" href="/personalize/addFavoritePublication?doi=10.1016/j.cell.2021.12.008">
	<i class="icon-reading-list" aria-hidden="true"></i>
	<span>Add to my reading list</span>
</a></div>

            

        
</div></div></div></div>

            

        
</div><div><button data-id="article-nav-menubar-more" aria-controls="article_more_menu" aria-expanded="false" aria-haspopup="true" aria-label="Toggle more menu"><i aria-hidden="true" class="icon-more-mobile"></i><span>More</span></button><div class="info-panel"><div class="info-panel__left-content"><div class="info-panel__article_tools info-panel__item"><div id="article_more_menu" aria-label="More menu" data-core-nav="more"><div class="heading">More<button title="Close more menu" aria-controls="article_more_menu" aria-haspopup="true"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div>
    

        


    

            

            

            <ul data-widget-id="183a17a2-87b5-4e6f-bb85-6b5cda068c89" data-widget-name="UX3ArticleToolsWidget" class="article-tools"><li class="article-tools__item article-tools__pdf"><a href="/action/showPdf?pii=S0092-8674%2821%2901446-X" title="Standard PDF (opens in a new tab)" target="_blank" aria-hidden="false" class="article-tools__ctrl article-tools__item__pdf"><svg focusable="false" viewBox="0 0 32 32" width="20" height="20" title="PDF" class="pdf-icon"><path id="pdfwhite" d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path id="pdfda0000" d="M.167 2.592H22.39V9.72H.166z" fill="#da0000"></path><path id="pdffff9f9" fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path id="pdff91d0a" d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="basic">Download PDF</span><span class="alternative">Download PDF</span></a></li><li><a href="https://www.cell.com/action/showCitFormats?doi=10.1016%2Fj.cell.2021.12.008&amp;pii=S0092-8674%2821%2901446-X" title="Cite" target="_blank" aria-label="Open Cite in new tab" class="article-tools__item__displayExportCitation"><i aria-hidden="true" class="icon-export-Citation"></i><span>Cite</span></a></li><li id="article_tools_share">
    

        


    

            

            

            <!-- Go to https://www.addtoany.com/buttons/customize/ to customize your tools --><script type="text/javascript" defer="defer" src="https://static.addtoany.com/menu/page.js"></script><script type="text/javascript">var a2a_config = a2a_config || {};
a2a_config.thanks = {
    postShare: false,
    ad: false,
};</script><script>var a2a_config = a2a_config || {};
let res = window.location.href.replace(/\(/g, "%28").replace(/\)/g, "%29");
a2a_config.templates = a2a_config.templates || {};

a2a_config.templates.email = {
    body: res
};
</script><div data-widget-id="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da" data-widget-name="UX3share" class="dropBlock article-tools__dropblock article-tools__share"><button data-db-target-for="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da" aria-expanded="false" title="open Share menu" class="article-tools__ctrl share__ctrl"><i aria-hidden="true" class="icon-article-share"></i><span>Share</span></button><div data-db-target-of="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da" id="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da_Pop" class="share__block axel-share dropBlock__holder article-tools__dropblock__holder"><h4 class="axel-share__title">Share</h4><div class="axel-share__inner"><span class="sr-only">Share on</span><ul class="a2a__share-list rlist w-slide--list a2a a2a_kit a2a_kit_size_32 a2a_default_style" data-a2a-url="https://www.cell.com/cell/fulltext/S0092-8674%2821%2901446-X" style="line-height: 32px;"><li class="a2a_listitem_custom"><a role="link" title="Email" class="a2a_button_email" target="_blank" rel="nofollow noopener" href="/#email"><i aria-hidden="true" class="at-icon-wrapper icon-gizmo-email-plus"></i><span>Email</span></a></li><li class="a2a_listitem_custom"><a role="link" title="formerly known as Twitter" class="a2a_button_twitter" target="_blank" rel="nofollow noopener" href="/#twitter"><i aria-hidden="true" class="at-icon-wrapper icon-x-default"></i><span>X</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Facebook" class="a2a_button_facebook" target="_blank" rel="nofollow noopener" href="/#facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook-square-2"></i><span>Facebook</span></a></li><li class="a2a_listitem_custom"><a role="link" title="LinkedIn" class="a2a_button_linkedin" target="_blank" rel="nofollow noopener" href="/#linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin-square"></i><span>LinkedIn</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Sina Weibo" class="a2a_button_sina_weibo" target="_blank" rel="nofollow noopener" href="/#sina_weibo"><i aria-hidden="true" class="at-icon-wrapper icon-weibo"></i><span>Sina Weibo</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Mendeley" class="a2a_button_mendeley" target="_blank" rel="nofollow noopener" href="/#mendeley"><i aria-hidden="true" class="at-icon-wrapper icon-mendeley-2"></i><span>Mendeley</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Bluesky" class="a2a_button_bluesky" target="_blank" rel="nofollow noopener" href="/#bluesky"><i aria-hidden="true" class="at-icon-wrapper icon-bluesky-logo"></i><span>Bluesky</span></a></li><div style="clear: both;"></div></ul><div class="extra-links-drop-zone">
    

        


    

            

            

            <div data-widget-id="ae9bf5c5-a2ee-4f5c-8e41-9b21638b1a9f" data-widget-name="UX3HTMLWidget"><a class="axel-share__extra" target="_blank" href="/personalize/addFavoritePublication?doi=10.1016/j.cell.2021.12.008">
	<i class="icon-reading-list" aria-hidden="true"></i>
	<span>Add to my reading list</span>
</a></div>

            

        
</div></div></div></div>

            

        
</li><li><a href="https://www.cell.com/action/addCitationAlert?doi=10.1016%2Fj.cell.2021.12.008" title="Set Alert" target="_blank" aria-label="Open Set Alert in new tab" class="article-tools__item__displayCreateCitationAlert"><i aria-hidden="true" class="icon-set-alert"></i><span>Set Alert</span></a></li><li><a href="https://s100.copyright.com/AppDispatchServlet?publisherName=ELS&amp;contentID=S009286742101446X&amp;orderBeanReset=true&amp;orderSource=Phoenix" title="Get Rights" target="_blank" aria-label="Open Get Rights in new tab" class="article-tools__item__request"><i aria-hidden="true" class="icon-get-rights"></i><span>Get Rights</span></a></li><li><a href="https://www.contentednet.com/salesforce-form/?pii=S009286742101446X&amp;issn=00928674&amp;source=jbs" title="Reprints" target="_blank" aria-label="Open Reprints in new tab" class="article-tools__item__reprints"><i aria-hidden="true" class="icon-article-reprints"></i><span>Reprints</span></a></li></ul>

            

        
</div></div><div class="info-panel__download-issue info-panel__item"><div class="article-header__download-full-issue"><button title="Download full issue" type="submit" formaction="/action/downloadMultiplePdf?action=downloadIssuePdf&amp;journal=cell&amp;volume=185&amp;issue=1&amp;pii=S0092-8674%2821%29X0002-5" class="btn--download-issue btn--cancel download-full-issue"><i class="icon-download-new"></i><span>Download Full Issue</span></button><div class="table-of-content__alert table-of-content__alert--download"><p>Download started</p><a href="#" class="table-of-content__alert--close">Ok</a></div></div></div></div><div class="info-panel__right-content"><div class="info-panel__navigation info-panel__item"><div class="content-navigation"><a href="/cell/fulltext/S0092-8674(21)01284-8" aria-label="Previous article" aria-disabled="false" class="content-navigation__prev"><i aria-hidden="true" class="icon-arrow_l"></i><span>Previous article</span></a><a href="/cell/fulltext/S0092-8674(21)01405-7" aria-label="Next article" aria-disabled="false" class="content-navigation__next"><span>Next article</span><i aria-hidden="true" class="icon-arrow_r"></i></a></div></div></div></div></div></div></div><div class="core-sections-menu-outer"><div class="core-sections-menu"><button aria-controls="article_sections_menu-desktop" aria-haspopup="true" aria-expanded="false" aria-label="Toggle sections menu" data-id="article-toolbar-showhide"><i class="icon-show-outline" aria-hidden="true"></i><span>Show Outline</span></button><nav id="article_sections_menu-desktop" aria-label="Contents" data-core-nav="article" style="margin-top: 0px;"><button aria-controls="article_sections_menu-desktop" aria-haspopup="true" aria-expanded="false" aria-label="Toggle sections menu" data-id="article-toolbar-showhide"><i class="icon-hide-outline" aria-hidden="true"></i><span>Hide Outline</span></button><ul><li><a href="#author-highlights-abstract">Highlights</a></li><li><a href="#author-abstract">Summary</a></li><li><a href="#graphical-abstract">Graphical abstract</a></li><li><a href="#keywords">Keywords</a></li><li><a href="#sec-1">Introduction</a></li><li><a href="#sec-2">Results</a></li><li><a href="#sec-3">Discussion</a></li><li><a href="#sec-4">STAR★Methods</a></li><li><a href="#acknowledgments">Acknowledgments</a></li><li><a href="#supplementary-material">Supplemental information (8)</a></li><li><a href="#app-2">Supporting citations</a></li><li><a href="#data-availability">Data and code availability</a></li><li><a href="#references">References</a></li><li role="none" class="mobile-only"><a role="menuitem" href="#core-collateral-metrics">Article metrics</a></li><li role="none" class="mobile-only"><a role="menuitem" href="#core-collateral-relatedArticles">Related Articles</a></li></ul></nav></div></div><div><div class="core-content-inner"><div class="core-linked-content"></div><div id="abstracts" data-extent="frontmatter"><div class="core-container"><section id="author-highlights-abstract" property="abstract" typeof="Text" role="doc-abstract"><h2 property="name">Highlights</h2><div id="abspara0020" role="paragraph"><div id="ulist0010" role="list"><div id="u0010" role="listitem"><div class="label">•</div><div class="content"><div id="p0010" role="paragraph">GWAS identifies variants associated with fingerprint type across all digits</div></div></div><div id="u0015" role="listitem"><div class="label">•</div><div class="content"><div id="p0015" role="paragraph">Fingerprint-associated genes are strongly enriched for limb development functions</div></div></div><div id="u0020" role="listitem"><div class="label">•</div><div class="content"><div id="p0020" role="paragraph"><i>Evi1</i> alters dermatoglyphs in mice by modulating limb rather than skin development</div></div></div><div id="u0025" role="listitem"><div class="label">•</div><div class="content"><div id="p0025" role="paragraph">Fingerprint patterns are genetically correlated with hand and finger proportions</div></div></div></div></div></section><section id="author-abstract" property="abstract" typeof="Text" role="doc-abstract"><h2 property="name">Summary</h2><div id="abspara0010" role="paragraph">Fingerprints are of long-standing practical and cultural interest, but little is known about the mechanisms that underlie their variation. Using genome-wide scans in Han Chinese cohorts, we identified 18 loci associated with fingerprint type across the digits, including a genetic basis for the long-recognized “pattern-block” correlations among the middle three digits. In particular, we identified a variant near <i>EVI1</i> that alters regulatory activity and established a role for EVI1 in dermatoglyph patterning in mice. Dynamic EVI1 expression during human development supports its role in shaping the limbs and digits, rather than influencing skin patterning directly. Trans-ethnic meta-analysis identified 43 fingerprint-associated loci, with nearby genes being strongly enriched for general limb development pathways. We also found that fingerprint patterns were genetically correlated with hand proportions. Taken together, these findings support the key role of limb development genes in influencing the outcome of fingerprint patterning.</div></section><section id="graphical-abstract" property="abstract" typeof="Text" role="doc-abstract"><h2 property="name">Graphical abstract</h2><div id="abspara0015" role="paragraph"><div class="figure-wrap"><figure id="undfig1" class="graphic" data-figureviewer-ignore="true"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.12.008/asset/408004f4-e061-4c0b-86ea-5682294acd9f/main.assets/fx1_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="fx1" alt="Graphical abstract undfig1" src="/cms/10.1016/j.cell.2021.12.008/asset/7f316104-f2f6-49ae-93c1-e2881d411698/main.assets/fx1.jpg" height="375" width="375" data-viewer-src="/cms/10.1016/j.cell.2021.12.008/asset/408004f4-e061-4c0b-86ea-5682294acd9f/main.assets/fx1_lrg.jpg" loading="lazy"></a></figure></div></div></section></div></div><div class="core-linked-content"></div><section id="keywords" property="keywords"><h2>Keywords</h2><ol><li><a href="/action/doSearch?AllField=&quot;fingerprint+pattern&quot;" alt="Search on this keyword" rel="nofollow">fingerprint pattern</a></li><li><a href="/action/doSearch?AllField=&quot;genetics&quot;" alt="Search on this keyword" rel="nofollow">genetics</a></li><li><a href="/action/doSearch?AllField=&quot;genome-wide+association+study&quot;" alt="Search on this keyword" rel="nofollow">genome-wide association study</a></li><li><a href="/action/doSearch?AllField=&quot;trans-ethnic+meta-analysis&quot;" alt="Search on this keyword" rel="nofollow">trans-ethnic meta-analysis</a></li><li><a href="/action/doSearch?AllField=&quot;limb+development&quot;" alt="Search on this keyword" rel="nofollow">limb development</a></li><li><a href="/action/doSearch?AllField=&quot;EVI1&quot;" alt="Search on this keyword" rel="nofollow"><i>EVI1</i></a></li></ol></section><div class="core-linked-content"></div><div class="core-linked-content"></div><section id="bodymatter" data-extent="bodymatter" property="articleBody" typeof="Text"><div class="core-container"><section id="sec-1"><h2>Introduction</h2><div id="p0050" role="paragraph">Dermatoglyphs are parallel ridge formations present on the skin of the palms and fingers of the hands and on the soles and toes of the feet (<span class="dropBlock reference-citations"><a id="body-ref-sref23" href="#" role="doc-biblioref" data-xml-rid="bib23" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref23" aria-controls="sref23">Cummins and Midlo, 1926</a><div class="dropBlock__holder" data-db-target-of="sref23"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">23.</div><div>Cummins, H. ∙ Midlo, C.</div><div><strong>Palmar and plantar epidermal ridge configurations (dermatoglyphics) in European-Americans</strong></div><div><em>Am. J. Phys. Anthropol.</em> 1926; <strong>9</strong>:471-502</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/ajpa.1330090422" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_23_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-0347914832" target="_blank" aria-label="Open Scopus in new tab">Scopus (64)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fajpa.1330090422" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). On the fingertips, these regular patterns of ridges and furrows form fingerprints of three principal pattern types: arch, loop, and whorl (<a id="crosref0390" href="#fig1">Figure&nbsp;1</a>A). Although fingerprints probably evolved to aid grasping (<span class="dropBlock reference-citations"><a id="body-ref-sref5" href="#" role="doc-biblioref" data-xml-rid="bib5" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref5" aria-controls="sref5">André et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">5.</div><div>André, T. ∙ Lefèvre, P. ∙ Thonnard, J.L.</div><div><strong>Fingertip moisture is optimally modulated during object manipulation</strong></div><div><em>J.&nbsp;Neurophysiol.</em> 2010; <strong>103</strong>:402-408</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1152/jn.00901.2009" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-74049123806" target="_blank" aria-label="Open Scopus in new tab">Scopus (49)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19906876/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1152%2Fjn.00901.2009&amp;pmid=19906876" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref126" href="#" role="doc-biblioref" data-xml-rid="bib126" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref126" aria-controls="sref126">Yum et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref126"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">136.</div><div>Yum, S.M. ∙ Baek, I.K. ∙ Hong, D. ...</div><div><strong>Fingerprint ridges allow primates to regulate grip</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2020; <strong>117</strong>:31665-31673</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.2001055117" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_136_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-85097560390" target="_blank" aria-label="Open Scopus in new tab">Scopus (5)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33257543/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.2001055117&amp;pmid=33257543" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and for sensing of surface textures (<span class="dropBlock reference-citations"><a id="body-ref-sref68" href="#" role="doc-biblioref" data-xml-rid="bib68" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref68" aria-controls="sref68">Loesch and Martin, 1984</a><div class="dropBlock__holder" data-db-target-of="sref68"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">74.</div><div>Loesch, D.Z. ∙ Martin, N.G.</div><div><strong>Finger ridge patterns and tactile sensitivity</strong></div><div><em>Ann. Hum. Biol.</em> 1984; <strong>11</strong>:113-124</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1080/03014468400006961" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/6732178/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1080%2F03014468400006961&amp;pmid=6732178" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref77-1" href="#" role="doc-biblioref" data-xml-rid="bib77" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref77-1" aria-controls="sref77-1">Medland et&nbsp;al., 2007</a><div class="dropBlock__holder" data-db-target-of="sref77-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">84.</div><div>Medland, S.E. ∙ Loesch, D.Z. ∙ Mdzewski, B. ...</div><div><strong>Linkage analysis of a model quantitative trait in humans: finger ridge count shows significant multivariate linkage to 5q14.1</strong></div><div><em>PLoS Genet.</em> 2007; <strong>3</strong>:1736-1744</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pgen.0030165" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_84_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-34848862129" target="_blank" aria-label="Open Scopus in new tab">Scopus (39)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17907812/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pgen.0030165&amp;pmid=17907812" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref103" href="#" role="doc-biblioref" data-xml-rid="bib103" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref103" aria-controls="sref103">Scheibert et&nbsp;al., 2009</a><div class="dropBlock__holder" data-db-target-of="sref103"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">112.</div><div>Scheibert, J. ∙ Leurent, S. ∙ Prevost, A. ...</div><div><strong>The role of fingerprints in the coding of tactile information probed with a biomimetic sensor</strong></div><div><em>Science.</em> 2009; <strong>323</strong>:1503-1506</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.1166467" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_112_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-62449256629" target="_blank" aria-label="Open Scopus in new tab">Scopus (251)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19179493/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.1166467&amp;pmid=19179493" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), since the 19<sup>th</sup> Century, the fingerprint has been widely used for personal identification because the patterns are unique to every individual, present from birth, and do not change over the lifespan (<span class="dropBlock reference-citations"><a id="body-ref-sref35" href="#" role="doc-biblioref" data-xml-rid="bib35" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref35" aria-controls="sref35">Galton, 1892</a><div class="dropBlock__holder" data-db-target-of="sref35"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">36.</div><div>Galton, F.</div><div><strong>Finger Prints</strong></div><div>Macmillan and Company, 1892</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=F.GaltonFinger+Prints1892Macmillan+and+Company" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div class="figure-wrap"><figure id="fig1" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.12.008/asset/48cc46d0-af80-4e9e-8324-179d32c4b9c4/main.assets/gr1_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr1" src="/cms/10.1016/j.cell.2021.12.008/asset/936ee3ba-2c16-49ec-92cc-c2411453bed5/main.assets/gr1.jpg" height="444" width="657" aria-labelledby="fig1-title fig1-content" data-viewer-src="/cms/10.1016/j.cell.2021.12.008/asset/48cc46d0-af80-4e9e-8324-179d32c4b9c4/main.assets/gr1_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig1-title" class="figure__title"><span class="label figure__label">Figure&nbsp;1</span> <span class="figure__title__text">Genetic variants associated with ordinal fingerprint patterns (arch, loop, and whorl) in Han Chinese (n&nbsp;= 9,909)</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig1-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0015" role="paragraph">(A) Pattern-types of fingerprints according to the number of triradii/deltas (triangles) and cores (circles) (<a id="crosref1620" href="#sec-4">STAR Methods</a>). There are three main types: arch, loop, and whorl. Each main group contains two sub-types according to the steepness, direction of ridges and the variable core.</div><div id="fspara0020" role="paragraph">(B) Genome-wide scans of the ordinal arch-loop-whorl phenotype identify 18 genomic regions associated with fingerprint patterns. The red line indicates the threshold for genome-wide significance after adjusting for the effective number of independent phenotypes (p<sub>adj</sub>&nbsp;&lt; 1.67&nbsp;× 10<sup>−8</sup>; <a id="crosref1625" href="#sec-4">STAR Methods</a>). Detailed patterns of adjusted association significance across different fingers are indicated by black squares for corresponding digits for significant associations (p<sub>adj</sub>&nbsp;&lt; 1.67&nbsp;× 10<sup>−8</sup>), and gray squares for marginal associations (p<sub>adj</sub>&nbsp;&lt; 3.33&nbsp;× 10<sup>−6</sup>). Notable genes are indicated for each locus (see <a id="crosref1630" href="#tbl1">Table 1</a> for selection criteria). D1–5L/R, digit 1 to 5 of left or right hand.</div><div id="fspara0025" role="paragraph">See also <a id="crosref1635" href="#mmc2">Tables S2</a> and <a id="crosref1640" href="#mmc3">S3</a>.</div></div></div></div></figcaption></figure></div><div id="p0055" role="paragraph">Dermatoglyphic patterns on fingers begin to develop on the digit tips after the 10<sup>th</sup> week of gestation, forming on the skin overlying the swollen and regressing volar pads. By the 14<sup>th</sup> week, the configuration of the future fingerprint pattern (arch, loop, or whorl) is defined at the epidermal-dermal junction by the primary ridges (<span class="dropBlock reference-citations"><a id="body-ref-sref10-1" href="#" role="doc-biblioref" data-xml-rid="bib10" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref10-1" aria-controls="sref10-1">Babler, 1991</a><div class="dropBlock__holder" data-db-target-of="sref10-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">10.</div><div>Babler, W.J.</div><div><strong>Embryologic development of epidermal ridges and their configurations</strong></div><div><em>Birth Defects Orig. Artic. Ser.</em> 1991; <strong>27</strong>:95-112</div></div><div class="external-links"><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/1786361/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?pmid=1786361" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref87" href="#" role="doc-biblioref" data-xml-rid="bib87" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref87" aria-controls="sref87">Okajima, 1975</a><div class="dropBlock__holder" data-db-target-of="sref87"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">94.</div><div>Okajima, M.</div><div><strong>Development of dermal ridges in the fetus</strong></div><div><em>J.&nbsp;Med. Genet.</em> 1975; <strong>12</strong>:243-250</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1136/jmg.12.3.243" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/51929/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1136%2Fjmg.12.3.243&amp;pmid=51929" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Several mechanisms have been proposed to explain the generation of these repeated patterns of epidermal ridges, including theories based on the resolution of mechanical strain on the epidermis through buckling (<span class="dropBlock reference-citations"><a id="body-ref-sref59-1" href="#" role="doc-biblioref" data-xml-rid="bib59" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref59-1" aria-controls="sref59-1">Kücken, 2007</a><div class="dropBlock__holder" data-db-target-of="sref59-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">64.</div><div>Kücken, M.</div><div><strong>Models for fingerprint pattern formation</strong></div><div><em>Forensic Sci. Int.</em> 2007; <strong>171</strong>:85-96</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.forsciint.2007.02.025" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_64_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-34547691328" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17459625/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.forsciint.2007.02.025&amp;pmid=17459625" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref91" href="#" role="doc-biblioref" data-xml-rid="bib91" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref91" aria-controls="sref91">Penrose, 1965a</a><div class="dropBlock__holder" data-db-target-of="sref91"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">98.</div><div>Penrose, L.S.</div><div><strong>Dermatoglyphic Topology</strong></div><div><em>Nature.</em> 1965; <strong>205</strong>:544-546</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/205544a0" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_98_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-0010505869" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2F205544a0" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), the arrangement of ridge configurations according to a template set by blood vessels or nerves (<span class="dropBlock reference-citations"><a id="body-ref-sref45" href="#" role="doc-biblioref" data-xml-rid="bib45" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref45" aria-controls="sref45">Hirsch and Schweichel, 1973</a><div class="dropBlock__holder" data-db-target-of="sref45"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">48.</div><div>Hirsch, W. ∙ Schweichel, J.U.</div><div><strong>Morphological evidence concerning the problem of skin ridge formation</strong></div><div><em>J.&nbsp;Ment. Defic. Res.</em> 1973; <strong>17</strong>:58-72</div></div><div class="external-links"><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/4794986/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?pmid=4794986" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), and the operation of reaction-diffusion signaling processes (<span class="dropBlock reference-citations"><a id="body-ref-sref37" href="#" role="doc-biblioref" data-xml-rid="bib37" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref37" aria-controls="sref37">Garzón-Alvarado and Ramírez Martinez, 2011</a><div class="dropBlock__holder" data-db-target-of="sref37"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">38.</div><div>Garzón-Alvarado, D.A. ∙ Ramírez Martinez, A.M.</div><div><strong>A biochemical hypothesis on the formation of fingerprints using a turing patterns approach</strong></div><div><em>Theor. Biol. Med. Model.</em> 2011; <strong>8</strong>:24</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/1742-4682-8-24" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21711537/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2F1742-4682-8-24&amp;pmid=21711537" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). However, the biological mechanisms underlying the generation of dermatoglyph patterns and the overall fingerprint configuration remain largely unknown.</div><div id="p0060" role="paragraph">Previous studies have reported population differences (<span class="dropBlock reference-citations"><a id="body-ref-sref128" href="#" role="doc-biblioref" data-xml-rid="bib128" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref128" aria-controls="sref128">Zhang et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref128"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">138.</div><div>Zhang, H.-G. ∙ Chen, Y.-F. ∙ Ding, M. ...</div><div><strong>Dermatoglyphics from all Chinese ethnic groups reveal geographic patterning</strong></div><div><em>PLoS ONE.</em> 2010; <strong>5</strong>:e8783</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pone.0008783" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_138_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-77649306234" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20098698/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pone.0008783&amp;pmid=20098698" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and considerable heritability of fingerprint pattern types (<i>h</i><sup><i>2</i></sup>&nbsp;= 0.3–0.8) (<span class="dropBlock reference-citations"><a id="body-ref-sref53" href="#" role="doc-biblioref" data-xml-rid="bib53" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref53" aria-controls="sref53">Karmakar et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref53"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">58.</div><div>Karmakar, B. ∙ Malkin, I. ∙ Kobyliansky, E.</div><div><strong>Inheritance of 18 quantitative dermatoglyphic traits based on factors in MZ and DZ twins</strong></div><div><em>Anthropol. Anz.</em> 2010; <strong>68</strong>:185-193</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1127/0003-5548/2011/0083" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-79952960577" target="_blank" aria-label="Open Scopus in new tab">Scopus (5)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21452683/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1127%2F0003-5548%2F2011%2F0083&amp;pmid=21452683" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref70" href="#" role="doc-biblioref" data-xml-rid="bib70" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref70" aria-controls="sref70">Machado et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref70"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">77.</div><div>Machado, J.F. ∙ Fernandes, P.R. ∙ Roquetti, R.W. ...</div><div><strong>Digital dermatoglyphic heritability differences as evidenced by a female twin study</strong></div><div><em>Twin Res. Hum. Genet.</em> 2010; <strong>13</strong>:482-489</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1375/twin.13.5.482" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20874471/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1375%2Ftwin.13.5.482&amp;pmid=20874471" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref104" href="#" role="doc-biblioref" data-xml-rid="bib104" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref104" aria-controls="sref104">Sengupta and Karmakar, 2004</a><div class="dropBlock__holder" data-db-target-of="sref104"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">113.</div><div>Sengupta, M. ∙ Karmakar, B.</div><div><strong>Mode of inheritance of finger dermatoglyphic traits among Vaidyas of West Bengal, India</strong></div><div><em>Ann. Hum. Biol.</em> 2004; <strong>31</strong>:526-540</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1080/03014460412331287164" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_113_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-10344236018" target="_blank" aria-label="Open Scopus in new tab">Scopus (25)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15739382/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1080%2F03014460412331287164&amp;pmid=15739382" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). A recent genome-wide association scan (GWAS) with moderate sample size discovered several loci associated with fingerprint patterns in a European-ancestry cohorts, but none of the loci had previously ascribed functions in either limb or skin development (<span class="dropBlock reference-citations"><a id="body-ref-sref46-1" href="#" role="doc-biblioref" data-xml-rid="bib46" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref46-1" aria-controls="sref46-1">Ho et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref46-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">49.</div><div>Ho, Y.Y.W. ∙ Evans, D.M. ∙ Montgomery, G.W. ...</div><div><strong>Common Genetic Variants Influence Whorls in Fingerprint Patterns</strong></div><div><em>J.&nbsp;Invest. Dermatol.</em> 2016; <strong>136</strong>:859-862</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=10.1016%2Fj.jid.2015.10.062&amp;cf=fulltext&amp;site=jid-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=10.1016%2Fj.jid.2015.10.062&amp;cf=pdf&amp;site=jid-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-84978086948" target="_blank" aria-label="Open Scopus in new tab">Scopus (11)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27045867/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.jid.2015.10.062&amp;pmid=27045867" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), yielding little insight into the underlying biological mechanisms. In the current study, by performing large-scale GWAS in Han Chinese populations, as well as trans-ethnic meta-analysis of more than 23,000 individuals, we identified numerous previously unreported loci underlying the systematic variation in human fingerprint patterns, implicating genes with important roles in embryonic limb development as the principal determinants of heritable fingerprint variation. We functionally validated the top signal near <i>EVI1</i> as altering enhancer regulatory function, established the importance of the EVI1 protein in dermatoglyph patterning in mouse models, and assessed EVI1 expression across limb development to dermatoglyph formation in human fetal tissues. We further found evidence of a shared genetic basis between fingerprint type and hand proportions. Our findings highlight the importance of limb development genes and processes in defining human fingerprint patterns.</div></section><section id="sec-2"><h2>Results</h2><section id="sec-2-1"><h3>Genome-wide scans identify 18 genomic regions associated with fingerprint patterns across all digits in Han Chinese cohorts</h3><div id="p0065" role="paragraph">In the discovery phase, we conducted GWAS on fingerprint patterns on all ten digits (D1–5L, D1–5R) in 9,909 individuals from three Han Chinese cohorts: the Taizhou Longitudinal Study (TZL, n&nbsp;= 2,961), the National Survey of Physical Traits (NSPT, n&nbsp;= 2,679), and the Jidong cohort study (JD, n&nbsp;= 4,269) (see also <a id="crosref0480" href="#mmc1">Table S1</a> for the details of cohorts). The GWAS on the ordinal phenotypes (coded as 0, 1, and 2 for arch, loop, and whorl, respectively) (<a id="crosref0485" href="#fig1">Figure&nbsp;1</a>A; <a id="crosref0490" href="#sec-4">STAR Methods</a>) identified 18 genome-wide significant signals after adjusting for multiple testing (p<sub>adj</sub>&nbsp;&lt; 1.67&nbsp;× 10<sup>−8</sup>) (<a id="crosref0495" href="#fig1">Figure&nbsp;1</a>B), whereas the GWAS based on the binary phenotypes (non-whorl or whorl) showed similar results (<a id="crosref0500" href="#mmc2">Table S2</a>). The majority of signals (17 of 18) were replicated in at least one of the two validation cohorts: the China Kadoorie Biobank (CKB, n&nbsp;= 1,785) and the WeGene cohort (WeGene, n&nbsp;= 2,152), with consistent allelic effects across all the cohorts (Table 1). The narrow-sense heritability (estimated using GCTA software; see <a id="crosref0505" href="#sec-4">STAR Methods</a>) of fingerprint patterns on each digit was between 0.295–0.432 in discovery cohorts. The 18 most significant SNPs explained 3.06%–5.56% of the phenotypic variance (R<sup>2</sup>; <a id="crosref0510" href="#mmc3">Table S3</a>). We further conducted GWAS on 66 different derived phenotypes (p<sub>adj</sub>&nbsp;&lt; 3.57&nbsp;× 10<sup>−9</sup>, e.g., binary, nominal, ordinal, and quantitative phenotypes), and found most of the significant associations overlapped with the 18 aforementioned genomic signals, suggesting the simple ordinal phenotypes cover most of the phenotypic information related to variation of fingerprint pattern (<a id="crosref0515" href="#mmc3">Table S3</a>).</div><div class="figure-wrap"><figure id="tbl1" class="table" data-figureviewer-ignore="true"><div class="table-wrap"><table data-figureviewer-ignore="true" tabindex="0"><thead><tr class="rowsep"><th rowspan="2">Locus</th><th rowspan="2">Top SNPs</th><th rowspan="2">Alt/Ref Allele</th><th rowspan="2">EAF<a id="crosref1755" href="#tbl1fna"><sup>a</sup></a></th><th rowspan="2">Notable Genes<a id="crosref1760" href="#tbl1fnb"><sup>b</sup></a></th><th rowspan="2">Associated Digits<a id="crosref1765" href="#tbl1fnc"><sup>c</sup></a></th><th colspan="2"><span>Discovery: TZL+NSPT+JD (n&nbsp;= 9,909)<a id="crosref1770" href="#tbl1fnd"><sup>d</sup></a></span></th><th colspan="4"><span>Replication 1: CKB (n&nbsp;= 1785)<a id="crosref1775" href="#tbl1fnd"><sup>d</sup></a></span></th><th colspan="4"><span>Replication 2: WeGene (n&nbsp;= 2152)<a id="crosref1780" href="#tbl1fnd"><sup>d</sup></a></span></th></tr><tr class="rowsep"><th>β</th><th>p<a id="crosref1785" href="#tbl1fne"><sup>e</sup></a> (Top Digit)</th><th>β<a id="crosref1790" href="#tbl1fnf"><sup>f</sup></a></th><th>p<a id="crosref1795" href="#tbl1fnf"><sup>f</sup></a></th><th>β<a id="crosref1800" href="#tbl1fng"><sup>g</sup></a></th><th>p<a id="crosref1805" href="#tbl1fng"><sup>g</sup></a></th><th>β<a id="crosref1810" href="#tbl1fnf"><sup>f</sup></a></th><th>p<a id="crosref1815" href="#tbl1fnf"><sup>f</sup></a></th><th>β<a id="crosref1820" href="#tbl1fng"><sup>g</sup></a></th><th>p<a id="crosref1825" href="#tbl1fng"><sup>g</sup></a></th></tr></thead><tbody><tr><td>2p24.3</td><td>rs16862838</td><td>T/C</td><td>0.197</td><td><i>MYCN(G/L)</i>, <i>DDX1(G/GE/R/L)</i>, <i>NBAS(L)</i></td><td>D2L,D3L<sup>∗</sup>,D4L,D5L</td><td>−0.106</td><td>2.92&nbsp;× 10<sup>−9</sup> (D3L)</td><td>−0.047</td><td>3.2&nbsp;× 10<sup>−2</sup> (L3)</td><td>−0.042</td><td>0.083</td><td>−0.086</td><td>3.91&nbsp;× 10<sup>−5</sup> (L1)</td><td>/</td><td>/</td></tr><tr><td>2q13</td><td>rs371242548</td><td>C/A</td><td>0.054</td><td><i>EDAR(G)</i>, <i>RANBP2(G)</i>, <i>LIMS1(L)</i></td><td>D3R<sup>∗</sup></td><td>−0.184</td><td>6.11&nbsp;× 10<sup>−9</sup> (D3R)</td><td>−0.104</td><td>7.8&nbsp;× 10<sup>−3</sup> (L3)</td><td>−0.018</td><td>0.63</td><td>−0.11</td><td>1.46&nbsp;× 10<sup>−3</sup> (L1)</td><td>/</td><td>/</td></tr><tr><td>2q31.1</td><td>rs11460049</td><td>AT/A</td><td>0.201</td><td><i>SP9(L)</i>, <i>SP3(G/L)</i>, <i><span data-style="underline">OLA1</span>(G/GE/R/L)</i></td><td>D1L<sup>∗</sup>/R,D2L<sup>∗</sup>/R<sup>∗</sup></td><td>−0.127</td><td>1.32&nbsp;× 10<sup>−12</sup> (D2R)</td><td>−0.104</td><td>4.6&nbsp;× 10<sup>−5</sup> (L2)</td><td>−0.08</td><td>3.6&nbsp;× 10<sup>−3</sup></td><td>–</td><td>–</td><td>–</td><td>–</td></tr><tr><td>2q33.1</td><td>rs4673509</td><td>C/A</td><td>0.184</td><td><i>SATB2(G/L)</i>, <i><span data-style="underline">FTCDNL1/FONG</span>(G/R/L)</i>, <i>TYW5(L)</i></td><td>D2L<sup>∗</sup>/R<sup>∗</sup>,D3L/R<sup>∗</sup></td><td>0.118</td><td>1.74&nbsp;× 10<sup>−10</sup> (D2L)</td><td>0.073</td><td>8.50&nbsp;× 10<sup>−3</sup> (L1)</td><td>/</td><td>/</td><td>0.05</td><td>1.64&nbsp;× 10<sup>−2</sup> (L3)</td><td>0.023</td><td>0.331</td></tr><tr><td>3p14.3</td><td>rs358075</td><td>T/C</td><td>0.048</td><td><i>WNT5A(G/L)</i>, <i>LRTM1(G/L)</i>, <i><span data-style="underline">CACNA2D3</span>(GE/R/L)</i></td><td>D3L<sup>∗</sup>/R<sup>∗</sup>,D4L/R</td><td>−0.222</td><td>2.99&nbsp;× 10<sup>−11</sup> (D3L)</td><td>−0.126</td><td>8.10&nbsp;× 10<sup>−3</sup> (L1)</td><td>/</td><td>/</td><td>−0.141</td><td>4.51&nbsp;× 10<sup>−4</sup> (L1)</td><td>/</td><td>/</td></tr><tr><td>3p14.1</td><td>rs17072351</td><td>G/A</td><td>0.225</td><td><i>ADAMTS9(G/L)</i>, <i>MAGI1(G/L)</i></td><td>D4L<sup>∗</sup>/R,D5L/R<sup>∗</sup></td><td>−0.133</td><td>7.42&nbsp;× 10<sup>−15</sup> (D4L)</td><td>−0.057</td><td>9.40&nbsp;× 10<sup>−3</sup> (L1)</td><td>/</td><td>/</td><td>−0.053</td><td>3.30&nbsp;× 10<sup>−3</sup> (L2)</td><td>−0.048</td><td>1.53&nbsp;× 10<sup>−2</sup></td></tr><tr><td>3q26.2</td><td>rs6444832</td><td>A/G</td><td>0.459</td><td><i>EVI1(G/R/L)</i>, <i>GOLIM4(G)</i>, <i>EGFEM1P(L)</i></td><td>D2L<sup>∗</sup>/R<sup>∗</sup>,D3L<sup>∗</sup>/R<sup>∗</sup>,D4L<sup>∗</sup>/R<sup>∗</sup></td><td>−0.133</td><td>1.31&nbsp;× 10<sup>−20</sup> (D3L)</td><td>−0.074</td><td>2.10&nbsp;× 10<sup>−4</sup> (L1)</td><td>/</td><td>/</td><td>−0.048</td><td>6.80&nbsp;× 10<sup>−3</sup> (L2)</td><td>−0.032</td><td>0.055</td></tr><tr><td>3q29</td><td>rs80252354</td><td>T/C</td><td>0.031</td><td><i>HES1(G/L)</i>, <i>OPA1(G/L)</i>, <i>DPPA2P3(L)</i></td><td>D4L<sup>∗</sup></td><td>−0.238</td><td>7.36&nbsp;× 10<sup>−9</sup> (D4L)</td><td>0.094</td><td>0.068</td><td>−0.065</td><td>0.22</td><td>0.115</td><td>0.125</td><td>0.008</td><td>0.917</td></tr><tr><td>6q21</td><td>rs28700026</td><td>C/T</td><td>0.254</td><td><i>PRDM1(G/L)</i>, <i>PREP(G/R/L)</i></td><td>D1R,D2L<sup>∗</sup>/R<sup>∗</sup>,D3L<sup>∗</sup>/R<sup>∗</sup>,D4L/R<sup>∗</sup>,D5L/R<sup>∗</sup></td><td>−0.12</td><td>2.57&nbsp;× 10<sup>−13</sup> (D2R)</td><td>−0.053</td><td>5.2&nbsp;× 10<sup>−3</sup> (L3)</td><td>−0.033</td><td>0.18</td><td>−0.043</td><td>4.09&nbsp;× 10<sup>−2</sup> (L2)</td><td>−0.019</td><td>0.355</td></tr><tr><td>7p14.3</td><td>rs2075127</td><td>C/T</td><td>0.284</td><td><i>CREB5(G/L)</i>, <i>CHN2(G/L)</i>, <i><span data-style="underline">CPVL</span>(GE/R/L)</i></td><td>D2L<sup>∗</sup>/R<sup>∗</sup>,D3L<sup>∗</sup>/R<sup>∗</sup>,D4L<sup>∗</sup>/R<sup>∗</sup>,D5R<sup>∗</sup></td><td>0.118</td><td>9.18&nbsp;× 10<sup>−14</sup> (D3R)</td><td>0.038</td><td>4.3&nbsp;× 10<sup>−2</sup> (L2)</td><td>0.036</td><td>0.063</td><td>0.058</td><td>2.32&nbsp;× 10<sup>−3</sup> (L2)</td><td>0.035</td><td>4.89&nbsp;× 10<sup>−2</sup></td></tr><tr><td>8q23.1</td><td>rs1494910</td><td>C/T</td><td>0.161</td><td><i>RSPO2(G/R/L)</i>, <i>ANGPT1(G/L)</i>, <i>EIF3E(L)</i></td><td>D1L/R<sup>∗</sup>,D4L,D5R<sup>∗</sup></td><td>0.215</td><td>1.30&nbsp;× 10<sup>−16</sup> (D1R)</td><td>0.142</td><td>1.70&nbsp;× 10<sup>−7</sup> (L2)</td><td>0.113</td><td>8.60&nbsp;× 10<sup>−6</sup></td><td>0.085</td><td>1.37&nbsp;× 10<sup>−4</sup> (L3)</td><td>0.033</td><td>0.155</td></tr><tr><td>12q12</td><td>rs2731043</td><td>C/T</td><td>0.182</td><td><i><span data-style="underline">DBX2</span>(G/GE/R/L)</i>, <i>NELL2(G/L)</i>, <i>ANO6(L)</i></td><td>D3R<sup>∗</sup></td><td>0.107</td><td>8.06&nbsp;× 10<sup>−9</sup> (D3R)</td><td>0.051</td><td>2.30&nbsp;× 10<sup>−2</sup> (L1)</td><td>/</td><td>/</td><td>0.028</td><td>0.184</td><td>/</td><td>/</td></tr><tr><td>12q24.21</td><td>rs7957733</td><td>G/C</td><td>0.28</td><td><i>TBX3(G/R/L)</i></td><td>D1L/R,D2L<sup>∗</sup>/R<sup>∗</sup>,D3L/R,D4L<sup>∗</sup>/R<sup>∗</sup>, D5L<sup>∗</sup>/R<sup>∗</sup></td><td>0.111</td><td>3.78&nbsp;× 10<sup>−12</sup> (D5L)</td><td>0.054</td><td>7.8&nbsp;× 10<sup>−3</sup> (L3)</td><td>0.032</td><td>0.082</td><td>0.075</td><td>4.40&nbsp;× 10<sup>−5</sup> (L2)</td><td>0.068</td><td>4.88&nbsp;× 10<sup>−5</sup></td></tr><tr><td>13q21.33</td><td>rs11618603</td><td>G/C</td><td>0.139</td><td><i>KLHL1(G/L)</i>, <i>DACH1(R/L)</i></td><td>D1L/R,D2L<sup>∗</sup>/R<sup>∗</sup>,D3L<sup>∗</sup>/R<sup>∗</sup></td><td>0.14</td><td>1.24&nbsp;× 10<sup>−11</sup> (D3L)</td><td>0.102</td><td>2.5&nbsp;× 10<sup>−4</sup> (L3)</td><td>0.02</td><td>0.47</td><td>0.096</td><td>3.57&nbsp;× 10<sup>−4</sup> (L2)</td><td>0.058</td><td>2.71&nbsp;× 10<sup>−2</sup></td></tr><tr><td>16q12.2</td><td>rs5005161</td><td>C/T</td><td>0.425</td><td><i><span data-style="underline">RPGRIP1L</span>(G/GE/R/L)</i>, <i>AKTIP(G/L)</i>, <i>FTO(L)</i>, <i>IRX3(G)</i></td><td>D3L<sup>∗</sup>/R</td><td>0.092</td><td>1.93&nbsp;× 10<sup>−10</sup> (D3L)</td><td>0.05</td><td>1.5&nbsp;× 10<sup>−3</sup> (L3)</td><td>0.01</td><td>0.59</td><td>0.063</td><td>1.95&nbsp;× 10<sup>−4</sup> (L1)</td><td>/</td><td>/</td></tr><tr><td>17p12</td><td>rs7208722</td><td>G/A</td><td>0.299</td><td><i>PMP22(G)</i>, <i>HS3ST3B1(G)</i>, <i>CDRT7(R/L)</i></td><td>D3R<sup>∗</sup>,D4R</td><td>0.108</td><td>4.63&nbsp;× 10<sup>−12</sup> (D3R)</td><td>0.073</td><td>1.5&nbsp;× 10<sup>−3</sup> (L3)</td><td>0.031</td><td>0.096</td><td>0.045</td><td>9.00&nbsp;× 10<sup>−3</sup> (L1)</td><td>/</td><td>/</td></tr><tr><td>18q21.2</td><td>rs17089876</td><td>C/T</td><td>0.441</td><td><i><span data-style="underline">TCF4</span>(G/GE/R/L)</i>, <i>CCDC68(G)</i>, <i>TXNL1(G)</i></td><td>D5L<sup>∗</sup>/R<sup>∗</sup></td><td>0.087</td><td>1.46&nbsp;× 10<sup>−9</sup> (D5L)</td><td>0.033</td><td>0.057</td><td>0.029</td><td>0.066</td><td>0.039</td><td>1.55&nbsp;× 10<sup>−2</sup> (L3)</td><td>0.027</td><td>0.068</td></tr><tr><td>18q23</td><td>rs2004773</td><td>A/G</td><td>0.382</td><td><i>SALL3(G/R/L)</i></td><td>D2L<sup>∗</sup>/R<sup>∗</sup>,D4R,D5L<sup>∗</sup>/R<sup>∗</sup></td><td>0.164</td><td>6.14&nbsp;× 10<sup>−29</sup> (D5R)</td><td>0.059</td><td>7.1&nbsp;× 10<sup>−4</sup> (L1)</td><td>/</td><td>/</td><td>0.101</td><td>1.89&nbsp;× 10<sup>−10</sup> (L1)</td><td>/</td><td>/</td></tr></tbody></table></div><figcaption><div class="caption"><span class="heading">Table 1</span><div id="tspara0010" role="paragraph">GWAS signals for fingerprint pattern type in Han Chinese discovery and replication populations</div></div><div class="notes"><div id="tspara0015" role="paragraph">Abbreviations: Alt, alternative; Ref, reference; EAF, effect or alt allele frequency; TZL, cohort from Taizhou Longitudinal Study; NSPT, cohort from National Survey of Physical Traits Project; JD, cohort from Jidong of Hebei Province; CKB, cohort from China Kadorie Biobank; WeGene, cohort from WeGene company; D1–5L/R, digit 1 to 5 of left or right hand.</div><div role="doc-footnote" data-has="label"><div class="label">a</div><div id="tbl1fna"><div id="ntpara0030" role="paragraph">The effect or alternative (Alt) allele frequency of the discovery populations.</div></div></div><div role="doc-footnote" data-has="label"><div class="label">b</div><div id="tbl1fnb"><div id="ntpara0035" role="paragraph">Notable genes are indicated as follows: (1) the two nearest genes within 1,000 kb of the most significantly associated SNP annotated by GREAT (G), which uses the subset of the UCSC Known Genes; (2) the nearest gene mapped by GENCODE (GE) or RefSeq (R); and (3) protein-coding genes within 1,000 kb of the most significantly associated SNP in regional LocusZoom plot (L). Underlining indicates that the best-associated SNP is located within the gene.</div></div></div><div role="doc-footnote" data-has="label"><div class="label">c</div><div id="tbl1fnc"><div id="ntpara0040" role="paragraph">Genome-wide significant level (<sup>∗</sup>p<sub>adj</sub>&nbsp;&lt; 1.67&nbsp;× 10<sup>−8</sup>) or suggestive level (p<sub>adj</sub>&nbsp;&lt; 3.33&nbsp;× 10<sup>−6</sup>) after multiple-testing adjustment.</div></div></div><div role="doc-footnote" data-has="label"><div class="label">d</div><div id="tbl1fnd"><div id="ntpara0045" role="paragraph">The sample sizes vary in GWAS on different phenotypes of digit: n = 5,415–9,909 for discovery cohort (fingerprint patterns on digit 1 are not available in JD cohort), n = 1,634–1,785 for replication cohort 1 (CKB), and n = 2,138–2,152 for replication cohort 2 (WeGene).</div></div></div><div role="doc-footnote" data-has="label"><div class="label">e</div><div id="tbl1fne"><div id="ntpara0050" role="paragraph">The associations between the top SNPs and the fingerprint pattern of the most significant digit (i.e., top digit, as indicated in parentheses).</div></div></div><div role="doc-footnote" data-has="label"><div class="label">f</div><div id="tbl1fnf"><div id="ntpara0055" role="paragraph">The signal was replicated at different levels of association as follows: the most significant replication is exactly the association between the top SNP and the top digit (L1); the most significant replication is the association between the top SNP and one of the other associated digits, whereas the association between the top SNP and the top digit is also significant (L2) or not significant (L3). “–”, not available (INDEL polymorphisms are not available in the WeGene cohort).</div></div></div><div role="doc-footnote" data-has="label"><div class="label">g</div><div id="tbl1fng"><div id="ntpara0060" role="paragraph">The associations between the top SNPs and top digits in replication cohorts. The “/” indicate that the associations have the same effect size and p value as the results of the two columns in front. See also <a id="crosref1830" href="#mmc1">Table S1</a>.</div></div></div></div><div class="core-table-tools"><ul><li><a class="core-table-show-action" href="/action/showFullTableHTML?isHtml=true&amp;tableId=tbl1&amp;pii=S0092-8674%2821%2901446-X" target="_blank">Open table in a new tab</a></li></ul></div></figcaption></figure></div></section><section id="sec-2-2"><h3>The <i>EVI1</i> locus contributes to the “pattern-block” correlation between the middle three digits on both hands</h3><div id="p0070" role="paragraph">Fingerprint pattern types on contralateral (left and right) digit pairs are highly correlated, also with strong genetic correlations among them (0.64–1) (<a id="crosref0520" href="#fig2">Figure&nbsp;2</a>A). In particular, the patterns on the middle three digits are more correlated with one another than are the patterns on the other digits, reflecting the long-recognized “pattern-block” (also known as “pattern influence”) (<span class="dropBlock reference-citations"><a id="body-ref-sref73" href="#" role="doc-biblioref" data-xml-rid="bib73" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref73" aria-controls="sref73">Martin et&nbsp;al., 1982</a><div class="dropBlock__holder" data-db-target-of="sref73"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">80.</div><div>Martin, N.G. ∙ Eaves, L.J. ∙ Loesch, D.Z.</div><div><strong>A genetical analysis of covariation between finger ridge counts</strong></div><div><em>Ann. Hum. Biol.</em> 1982; <strong>9</strong>:539-552</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1080/03014468200006061" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/7181445/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1080%2F03014468200006061&amp;pmid=7181445" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref84" href="#" role="doc-biblioref" data-xml-rid="bib84" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref84" aria-controls="sref84">Nagy and Pap, 2005</a><div class="dropBlock__holder" data-db-target-of="sref84"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">91.</div><div>Nagy, A.S. ∙ Pap, M.</div><div><strong>Pattern influence on the fingers</strong></div><div><em>Homo.</em> 2005; <strong>56</strong>:51-67</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.jchb.2004.10.003" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-16844367875" target="_blank" aria-label="Open Scopus in new tab">Scopus (4)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15901118/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.jchb.2004.10.003&amp;pmid=15901118" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) phenomenon. We found that using all SNPs across the genome, genetic correlations among the middle three digits on both hands (0.93–1) are significantly higher than those among all ten digits (<a id="crosref0535" href="#fig2">Figure&nbsp;2</a>B). To model the common elements underlying such correlations among the “pattern-block,” we adopted a partial least-square path model (PLSPM) (<a id="crosref0540" href="#sec-4">STAR Methods</a>) to extract the composite phenotype from the middle three digits on both hands. GWAS on this composite phenotype found 12 signals, 11 of which had been detected in the GWAS on the ordinal phenotypes; a previously unreported signal at 5p12 was also discovered (<a id="crosref0545" href="#fig2">Figure&nbsp;2</a>C; <a id="crosref0550" href="#mmc3">Table S3</a>). This composite phenotype showed strengthened narrow-sense heritability (<mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="h Subscript upper C upper P Superscript 2" tabindex="0" ctxtmenu_counter="0" style="font-size: 130%;"><mjx-math data-semantic-structure="(7 (6 0 (4 1 3 2)) 5)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="7" data-semantic-children="0,4,5" data-semantic-collapsed="(7 (6 0 4) 5)" data-semantic-owns="0 4 5" data-semantic-speech="h Subscript upper C upper P Superscript 2"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="7"><mjx-c class="mjx-c210E">ℎ</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.316em; margin-left: 0px;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="5" data-semantic-parent="7"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="4" data-semantic-children="1,2" data-semantic-content="3" data-semantic-parent="7" data-semantic-owns="1 3 2"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="4"><mjx-c class="mjx-c1D436">𝐶</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="3" data-semantic-parent="4" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="4"><mjx-c class="mjx-c1D443">𝑃</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msubsup></mjx-mrow></mjx-math></mjx-container>&nbsp;= 0.524 ± 0.039). Its phenotypic variance could be genetically explained by the top SNPs of the 12 signals with higher proportion (6.24%) than single phenotype (3.16%–4.88%) (R<sup>2</sup>; <a id="crosref0555" href="#mmc3">Table S3</a>), suggesting a shared genetic basis with the individual digit phenotypes from which the trait was extracted. The top signal at 3q26.2 was also effectively strengthened (p&nbsp;= 6.41&nbsp;× 10<sup>−22</sup>, whereas p values were between 1.31&nbsp;× 10<sup>−20</sup>–2.93&nbsp;× 10<sup>−11</sup> for single phenotypes).</div><div class="figure-wrap"><figure id="fig2" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.12.008/asset/95e35782-f618-45a9-912c-80006ed62b6e/main.assets/gr2_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr2" src="/cms/10.1016/j.cell.2021.12.008/asset/96e0cbec-0b1c-4711-a03e-f182465db123/main.assets/gr2.jpg" height="726" width="503" aria-labelledby="fig2-title fig2-content" data-viewer-src="/cms/10.1016/j.cell.2021.12.008/asset/95e35782-f618-45a9-912c-80006ed62b6e/main.assets/gr2_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig2-title" class="figure__title"><span class="label figure__label">Figure&nbsp;2</span> <span class="figure__title__text">Genetic basis of the middle three digit “pattern-block” phenomenon, with top signal near <i>EVI1</i></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig2-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0035" role="paragraph">(A) “Pattern block” of the middle three digits on both hands revealed by pairwise phenotypic correlation (blue) and genetic correlation (red) among the ten digits (n&nbsp;= 9,909). The dashed box indicates high correlations between the same digits of both hands and neighboring digits. The correlations from high to low were represented by both color and correlation coefficients (r) in the figure.</div><div id="fspara0040" role="paragraph">(B) The correlations of fingerprint patterns between the middle three digits on both hands (pattern-block pairs) are higher than the correlations of all random pairs of the ten digits (all pairs).</div><div id="fspara0045" role="paragraph">(C) Genome-wide scan on the composite phenotype extracted from the fingerprint pattern of the middle three digits on both hands. The loading coefficients of the composite phenotype on the six correlated variables are between 0.719 and 0.792.</div><div id="fspara0050" role="paragraph">(D) Fine mapping of signals at 3q26.2: LocusZoom plot (<span class="dropBlock reference-citations"><a id="body-ref-sref139-1" href="#" role="doc-biblioref" data-xml-rid="bib139" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref139-1" aria-controls="sref139-1">Pruim et al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref139-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">103.</div><div>Pruim, R.J. ∙ Welch, R.P. ∙ Sanna, S. ...</div><div><strong>LocusZoom: regional visualization of genome-wide association scan results</strong></div><div><em>Bioinformatics.</em> 2010; <strong>26</strong>:2336-2337</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btq419" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_103_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-77956586071" target="_blank" aria-label="Open Scopus in new tab">Scopus (1476)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20634204/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtq419&amp;pmid=20634204" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) of SNPs at the 3q26.2 region (top) and mapping of epigenetic marks H3K4me1, H3K27ac, DNase hypersensitivity, and conservation analysis at the same region, based on ENCODE and RMEC project data. SNP rs7646897 and rs7623083, indicated by red lines, area in a region that exhibits distinct active enhancer signatures defined by epigenetic marks, such as H3K4me1 (green), H3K27ac (blue) histone modifications, and DNase hypersensitivity (purple), and with enhancer function by chromatin state assay (yellow box) in fibroblast primary cells and in chondrocytes. The phastCon score indicates the evolutionarily conservation in primates.</div><div id="fspara0055" role="paragraph">(E and F) (E) Luciferase reporter assays on candidate regulatory elements carrying alternate alleles at SNPs rs7646897 and (F) rs7623083 in HEK293T cells. pGL3-basic is a negative control plasmid lacking enhancer activity, and pGL3-318 is a positive control derived from the <i>EVI1</i> promoter region. Symbols indicate significance in t test (<sup>∗</sup>p&nbsp;&lt; 0.05, <sup>∗∗</sup>p&nbsp;&lt; 0.01, <sup>∗∗∗</sup>p&nbsp;&lt; 0.001).</div><div id="fspara0060" role="paragraph">See also <a id="crosref1645" href="#figs1">Figures S1</a> and <a id="crosref1650" href="#figs2">S2</a>, <a id="crosref1655" href="#mmc3">Table S3</a>, <a id="crosref2100" href="#mmc1">Methods S1</a>, and <a id="crosref1660" href="#mmc7">Data S1</a>.</div></div></div></div></figcaption></figure></div><div id="p0075" role="paragraph">The top signal at 3q26.2 is located ∼100 kb downstream of the <i>EVI1</i> (Ecotropic Viral Integration Site 1) gene, also termed <i>MECOM</i> (<i>MDS1</i> and <i>EVI1</i> complex locus) (<a id="crosref0560" href="#fig2">Figure&nbsp;2</a>D). Fine-mapping of the 3q26.2 locus, which showed enhanced association with composite phenotype, using PAINTOR with functional annotation (<a id="crosref0565" href="#sec-4">STAR Methods</a>) found that the 99% credible set contained two SNPs: rs7646897 (posterior probability&nbsp;= 0.383) and rs7623083 (posterior probability&nbsp;= 0.617) (<a id="crosref0570" href="#fig2">Figure&nbsp;2</a>D, top). Interrogation of ENCODE and REMC databases revealed that the 3q26.2 signal region showed distinct active enhancer signatures in a range of human cell types (<a id="crosref0575" href="#fig2">Figure&nbsp;2</a>D, bottom). Further Hi-C data showed that the variant containing region and the <i>EVI1</i> gene are located in the same topologically associating domain (TAD), whereas other nearby genes are located outside the boundary of this domain (<a id="crosref0580" href="#figs1">Figure&nbsp;S1</a>A). There is also chromatin interaction between the promoter region of the <i>EVI1</i> gene and anchoring SNP rs7646897 (<a id="crosref0585" href="#figs1">Figure&nbsp;S1</a>B). To verify this potential modulating activity on the <i>EVI1</i> gene, we performed luciferase reporter assays (<a id="crosref0590" href="#sec-4">STAR Methods</a>; see also <a id="crosref0595" href="#mmc7">Data S1</a> and <a id="crosref0600" href="#mmc1">Methods S1</a> for the details of BLAST alignment for each fragment) on rs7646897 and rs7623083 in HEK293T cells and detected allele specific differences in modulating activity for rs7646897 (p&nbsp;= 7.00&nbsp;× 10<sup>−4</sup>) (<a id="crosref0605" href="#fig2">Figures 2</a>E, <a id="crosref0610" href="#figs2">S2</a>A–S2D, S2I, and S2J). SNP rs7623083 showed no effect for three experiments and allele specific differences for two (<a id="crosref0615" href="#fig2">Figures 2</a>F and <a id="crosref0620" href="#figs2">S2</a>E–S2H). Further assays showed that the alternate SNPs at rs7646897 modulated the expression of <i>EVI1</i> (p&nbsp;= 7.50&nbsp;× 10<sup>−3</sup>), but not the promoters of the closet up- and downstream genes <i>GOLIM4</i> (p&nbsp;= 0.364) and <i>TERC</i> (p&nbsp;= 0.778) (<a id="crosref0625" href="#figs2">Figure&nbsp;S2</a>I). Independent experimental repeats showed similar results (<a id="crosref0630" href="#figs2">Figure&nbsp;S2</a>J).</div><div class="figure-wrap"><figure id="figs1" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.12.008/asset/40210a55-1a22-4289-aea3-0fbac9f44ef6/main.assets/figs1_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs1" src="/cms/10.1016/j.cell.2021.12.008/asset/caf8acfa-021b-498d-9498-fdd3b39626d2/main.assets/figs1.jpg" height="293" width="654" aria-labelledby="figs1-title figs1-content" data-viewer-src="/cms/10.1016/j.cell.2021.12.008/asset/40210a55-1a22-4289-aea3-0fbac9f44ef6/main.assets/figs1_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs1-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S1</span> <span class="figure__title__text">Visualizing chromatin interaction between the <i>EVI1</i> gene, enhancer, and the rs7646897 region, related to <a id="crosref1725" href="#fig2">Figure&nbsp;2</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs1-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0185" role="paragraph">(A) Visualization of genome structure by Hi-C data as a heatmap indicated that the rs7646897 region and <i>EVI1</i> are located within the same TAD surveyed in 3D genome browser.</div><div id="fspara0190" role="paragraph">(B) Combined with three methods implemented in 3D Genome browser: Virtual 4C (Circular chromosomal conformation capture, top) that surveys for one-versus-many interactions in the genome, DNase I Hypersensitivity Site linkage (middle) that detects distal-proximal DHSs pairs, and ChIA-PET (bottom) that detects long-range interactions between genomic regions, chromatin interactions were identified between promoter region of the <i>EVI1</i> gene and enhancer harboring SNP rs7646897 (primarily supported by the DHS-linkage data).</div></div></div></div></figcaption></figure></div><div class="figure-wrap"><figure id="figs2" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.12.008/asset/64bde2b2-b6ff-4c5f-b61f-4a1a22b83702/main.assets/figs2_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs2" src="/cms/10.1016/j.cell.2021.12.008/asset/fed7c50d-cf6c-4713-be22-a60beca981ef/main.assets/figs2.jpg" height="657" width="657" aria-labelledby="figs2-title figs2-content" data-viewer-src="/cms/10.1016/j.cell.2021.12.008/asset/64bde2b2-b6ff-4c5f-b61f-4a1a22b83702/main.assets/figs2_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs2-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S2</span> <span class="figure__title__text">Independent luciferase reporter assays on candidate regulatory elements carrying alternate alleles at SNPs rs7646897 in HEK293T cells, related to <a id="crosref1730" href="#fig2">Figure&nbsp;2</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs2-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0200" role="paragraph">(A-D) Four independent luciferase reporter assays to test the effect of substitutions at rs7646897 in modulating the expression of <i>EVI1</i>. Three out of four experiments confirmed regulatory activity. Two of them showed a consistent direction of change.</div><div id="fspara0205" role="paragraph">(E-H) Four independent luciferase reporter assays to test the changes of rs7623083 modulating the expression of <i>EVI1</i>. The effect of rs7623083 on modulating expression of <i>EVI1</i> is inconsistent.</div><div id="fspara0210" role="paragraph">(I-J) Two independent luciferase reporter assays to test whether SNP rs7646897 modulates the expression of other closest up- and downstream genes <i>GOLIM4</i> and <i>TERC</i>.</div><div id="fspara0215" role="paragraph">pGL3-basic is a negative control plasmid lacking enhancer activity, pGL3-EVI1, pGL3-GOLIM4 and pGL3-TERC are the positive controls derived from EVI1, GOLIM4 and TERC promoter region, respectively. Symbols indicate significance in t test (<sup>∗</sup>p&nbsp;&lt; 0.05, <sup>∗∗</sup>&nbsp;&lt; 0.01, <sup>∗∗∗</sup>&nbsp;&lt; 0.001).</div></div></div></div></figcaption></figure></div></section><section id="sec-2-3"><h3><i>Evi1</i> mutation alters mouse dermatoglyph patterns and is expressed through early limb development stages</h3><div id="p0080" role="paragraph">The regulatory SNP rs7646897 is intergenic, lying 100 kb from the <i>EVI1</i> gene. As regulatory elements can exert effects at large physical distances, we assessed the importance of the <i>EVI1</i>-encoded protein itself on dermatoglyph pattern formation using mouse models. Based on their location on the ventral side of digits, their formation prior to birth (<a id="crosref0635" href="#figs3">Figures S3</a>A and S3B), and their parallel arrangement of ridges carrying the pores of sweat glands, we analyzed the transverse digital ridges in the mouse as the closest model phenotype of human dermatoglyphs (see <a id="crosref0640" href="#mmc1">Methods S1</a> for further justifications).</div><div class="figure-wrap"><figure id="figs3" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.12.008/asset/3b8f137b-117e-46af-82f0-acaf7c2c4070/main.assets/figs3_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs3" src="/cms/10.1016/j.cell.2021.12.008/asset/65f595d1-bedd-479a-9a0c-9132cd645731/main.assets/figs3.jpg" height="430" width="506" aria-labelledby="figs3-title figs3-content" data-viewer-src="/cms/10.1016/j.cell.2021.12.008/asset/3b8f137b-117e-46af-82f0-acaf7c2c4070/main.assets/figs3_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs3-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S3</span> <span class="figure__title__text">Development of digit ridge pattern in mouse and <i>Evi1</i><sup><i>Jbo</i></sup> effect on digit proportions and coexpression with <i>Prrx1</i>, related to <a id="crosref1735" href="#fig3">Figure&nbsp;3</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs3-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0225" role="paragraph">(A) Embryonic origin of digit ridge pattern in mouse. Ventral views of forelimbs of mice carrying a TCF/Lef::H2B-GFP WNT pathway reporter gene at embryonic day 17.5 (E17.5) and newborn (P1). Transverse digital ridges (examples indicated by arrows) are apparent from E17.5. Scale bar&nbsp;= 500&nbsp;μm.</div><div id="fspara0230" role="paragraph">(B) Development of mouse digital ridge patterning through different time points starting from P0 (at birth) followed by post natal days P4, P8 and P12. Palmar dermal surface of toluidine blue stained right forepaws from wild-type mice. Scale bar,&nbsp;500&nbsp;μm.</div><div id="fspara0235" role="paragraph">(C) Altered digit ridge patterns in <i>EDA</i> pathway mutant mice. Palmar dermal surface of toluidine blue stained right forepaws from wild-type, <i>Eda</i><sup><i>Ta</i></sup>, <i>Edar</i><sup><i>dlJ/dlJ</i></sup> and heterozygous <i>Edar</i><sup><i>dlJ/+</i></sup> mice at postnatal day 21, showing footpad and digit pad types and the distorted ridge pattern in the mutants. Scale bar, 500&nbsp;μm.</div><div id="fspara0240" role="paragraph">(D) Digit length, normalized to palm length, in wild-type (WT) and <i>Evi1</i><sup><i>Jbo</i>/+</sup> heterozygous adult mice. Digits are shorter in mutant animals.</div><div id="fspara0245" role="paragraph">(E) RNAscope <i>in situ</i> hybridization detecting <i>Evi1</i> and the limb mesenchyme marker <i>Prrx1</i> transcripts in mouse E13.5 embryonic limb. Right panels are higher magnification views of area indicated by the arrow on the left panel. Dotted line demarcates epithelium, as defined by immunofluorescent detection of KERATIN14 (K14). <i>Evi1</i> and <i>Prrx1</i> are coexpressed in mesenchymal cells, with <i>Evi1</i> also expressed prominently in mesenchymal cells condensing to form cartilage of the digits, as in human limb development. No expression is detected in epithelium. Nuclei are counterstained with DAPI. Scale bar&nbsp;= 100&nbsp;μm.</div></div></div></div></figcaption></figure></div><div id="p0085" role="paragraph">We analyzed dermatoglyph patterns in 21&nbsp;day old <i>Evi1</i><sup><i>Jbo</i>/+</sup> heterozygous mutant (encoding EVI1p.Asn763Ile) and wild-type littermate digits 2, 3, and 4 (<a id="crosref0645" href="#fig3">Figures 3</a>A–3C; <a id="crosref0650" href="#mmc4">Table S4</a>). Homozygous <i>Evi1</i><sup><i>Jbo/Jbo</i></sup> embryos die between midgestation and birth, whereas heterozygotes display a small spur on digit 5 (<a id="crosref0655" href="#fig3">Figure&nbsp;3</a>A) (<span class="dropBlock reference-citations"><a id="body-ref-sref90" href="#" role="doc-biblioref" data-xml-rid="bib90" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref90" aria-controls="sref90">Parkinson et&nbsp;al., 2006</a><div class="dropBlock__holder" data-db-target-of="sref90"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">97.</div><div>Parkinson, N. ∙ Hardisty-Hughes, R.E. ∙ Tateossian, H. ...</div><div><strong>Mutation at the Evi1 locus in Junbo mice causes susceptibility to otitis media</strong></div><div><em>PLoS Genet.</em> 2006; <strong>2</strong>:e149</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pgen.0020149" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_97_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-39049182374" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17029558/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pgen.0020149&amp;pmid=17029558" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), and slightly decreased digit length (<a id="crosref0665" href="#figs3">Figure&nbsp;S3</a>D). Using a mixed ordinal logistic regression model, we found reduced frequency of continuous ridges in all mutant digits (p<sub>digit2</sub> <i>=</i> 0.02; p<sub>digit3</sub>&nbsp;= 0.0005; p<sub>digit4</sub> <i>&lt;</i> 0.0001), with digits 3 and 4 also carrying more discontinuous ridges (p<sub>digit3</sub>&nbsp;= 0.03; p<sub>digit4</sub> <i>&lt;</i> 0.003) (<a id="crosref0670" href="#fig3">Figure&nbsp;3</a>C). These results demonstrate directly that EVI1 itself is a modulator of dermatoglyph patterns.</div><div class="figure-wrap"><figure id="fig3" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.12.008/asset/f3827bdd-47dc-468d-ad22-5930a8971e67/main.assets/gr3_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr3" src="/cms/10.1016/j.cell.2021.12.008/asset/0b0db4e4-60d7-4b2a-82e5-a67ec235e310/main.assets/gr3.jpg" height="611" width="663" aria-labelledby="fig3-title fig3-content" data-viewer-src="/cms/10.1016/j.cell.2021.12.008/asset/f3827bdd-47dc-468d-ad22-5930a8971e67/main.assets/gr3_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig3-title" class="figure__title"><span class="label figure__label">Figure&nbsp;3</span> <span class="figure__title__text">EVI1 in dermatoglyph patterning and limb development</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig3-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0070" role="paragraph">(A) Palmar dermal surface of toluidine blue stained paws from wild-type and <i>Evi1</i><sup><i>Jbo</i>/+</sup> adult mice showing dermatoglyph arrangement. Arrow indicates spur on the mutant digit 5 (D5).</div><div id="fspara0075" role="paragraph">(B) Schematic depicting transverse ridge categories on mouse digits and ventral surface of D4 of wild-type and <i>Evi1</i><sup><i>Jbo</i>/+</sup>. C, regions carrying continuous ridges are indicated; D, discontinuous ridges are indicated; I, incomplete ridges are indicated.</div><div id="fspara0080" role="paragraph">(C) Quantification of digit ridge pattern in wild-type and <i>Evi1</i><sup><i>Jbo/+</i></sup> mutants. Continuous ridges are reduced on all mutant digits, whereas D3 and D4 carry more discontinuous ridges.</div><div id="fspara0085" role="paragraph">(D) Whole mount <i>in situ</i> hybridization detecting <i>Evi1</i> expression in mouse embryonic forelimbs. Ventral view.</div><div id="fspara0090" role="paragraph">(E) RNAscope <i>in situ</i> hybridization detecting <i>Evi1</i> and the limb mesenchyme marker <i>Prrx1</i> transcripts in mouse embryonic limb and digits between E11.5 and E17.5.</div><div id="fspara0095" role="paragraph">(F) qRT-PCR determination of <i>Evi1</i> expression in mouse forelimb at E11.5 (whole limb bud), E13.5, E15.5, and E17.5 (autopod only).</div><div id="fspara0100" role="paragraph">(G–J) Immunofluorescence detecting EVI1 expression in human embryonic tissue. (G) Transverse section of CS17 embryo (∼6-week EGA) shows nuclear expression in mesenchymal cells of the limb bud (LB, magnified in lower panel). The neural tube (NT) indicates the dorsal midline. (H) Longitudinal section of 10-week EGA digit, arrow indicates the raised volar pad across which fingerprints form. (I) 13-week EGA digit and (J) 16-week EGA digit detecting EVI1 (arrow in week 13) and epithelial marker K14. SG, eccrine sweat gland.</div><div id="fspara0105" role="paragraph">(K and L) RNAscope <i>in situ</i> hybridization detecting <i>EVI1</i> and <i>PRRX1</i> transcripts in sectioned (K) 10-week EGA and (L) 16-week EGA human digit, with α-catenin immunofluorescence. Individual cells co-express <i>EVI1</i> and <i>PRRX1</i> at 10 weeks, indicated by arrow and magnified in right panel. Asterisks indicate autofluorescent blood cells.</div><div id="fspara0110" role="paragraph">(M) Detection of proliferative cell marker Ki67 in 10-week EGA digit. Dorsal (D) and ventral (V) axes are annotated: nuclei are stained with DAPI. Scale bars, 1&nbsp;mm (A); 500&nbsp;μm (D and G, upper); 100&nbsp;μm (E, G, lower, H–J, K, left, and M); 20&nbsp;μm (K, right, and L). Error bars indicate SEM.</div><div id="fspara0115" role="paragraph">See also <a id="crosref1665" href="#figs3">Figure&nbsp;S3</a>, <a id="crosref1670" href="#mmc4">Table S4</a>, and <a id="crosref2105" href="#mmc1">Methods S1</a>.</div></div></div></div></figcaption></figure></div><div id="p0090" role="paragraph">We assessed <i>Evi1</i> expression in intact embryonic mouse limbs by whole mount <i>in situ</i> hybridization, identifying high expression broadly in the autopod at embryonic day (E) 12.5, then becoming restricted to the distal regions of all emerging digits (<a id="crosref0675" href="#fig3">Figure&nbsp;3</a>D). RNAscope <i>in situ</i> hybridization permits transcript detection at later stages on sectioned tissue. This approach identified broad <i>Evi1</i> expression throughout limb mesenchyme at E11.5, becoming restricted to the distal end of the limb at E13.5, and excluded from the cartilage elements of the digits by E15.5, reducing further by E17.5 (<a id="crosref0680" href="#fig3">Figures 3</a>E and <a id="crosref0685" href="#figs4">S3</a>E). Quantification of <i>Evi1</i> expression in embryonic mouse limbs (<a id="crosref0690" href="#fig3">Figure&nbsp;3</a>F) agreed with the findings by <i>in situ</i> hybridization, finding a steep decline across the stages of limb bud outgrowth at E11.5, digit emergence (E13.5), digit outgrowth, and definition of dermatoglyph patterns (E15.5 and E17.5). In the distal digit <i>Evi1</i> expression was detected in the same cells as <i>Prrx1</i>, also named <i>Prx1</i>, a marker of limb bud mesenchyme (<span class="dropBlock reference-citations"><a id="body-ref-sref17" href="#" role="doc-biblioref" data-xml-rid="bib17" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref17" aria-controls="sref17">Chesterman et&nbsp;al., 2001</a><div class="dropBlock__holder" data-db-target-of="sref17"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">17.</div><div>Chesterman, E.S. ∙ Gainey, G.D. ∙ Varn, A.C. ...</div><div><strong>Investigation of Prx1 protein expression provides evidence for conservation of cardiac-specific posttranscriptional regulation in vertebrates</strong></div><div><em>Dev. Dyn.</em> 2001; <strong>222</strong>:459-470</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/dvdy.1198" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-0034759688" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11747080/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fdvdy.1198&amp;pmid=11747080" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref86" href="#" role="doc-biblioref" data-xml-rid="bib86" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref86" aria-controls="sref86">Nohno et&nbsp;al., 1993</a><div class="dropBlock__holder" data-db-target-of="sref86"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">93.</div><div>Nohno, T. ∙ Koyama, E. ∙ Myokai, F. ...</div><div><strong>A chicken homeobox gene related to Drosophila paired is predominantly expressed in the developing limb</strong></div><div><em>Dev. Biol.</em> 1993; <strong>158</strong>:254-264</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1006/dbio.1993.1184" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_93_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-0027249822" target="_blank" aria-label="Open Scopus in new tab">Scopus (65)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/8101172/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1006%2Fdbio.1993.1184&amp;pmid=8101172" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) (<a id="crosref0705" href="#figs3">Figure&nbsp;S3</a>E).</div><div class="figure-wrap"><figure id="figs4" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.12.008/asset/c2e0655f-a09b-44c1-8b83-649f24ee9b1f/main.assets/figs4_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs4" src="/cms/10.1016/j.cell.2021.12.008/asset/d8495a7e-34fe-49b0-95e9-cdaab04c548b/main.assets/figs4.jpg" height="926" width="657" aria-labelledby="figs4-title figs4-content" data-viewer-src="/cms/10.1016/j.cell.2021.12.008/asset/c2e0655f-a09b-44c1-8b83-649f24ee9b1f/main.assets/figs4_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs4-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S4</span> <span class="figure__title__text">Effect sizes (regression coefficients) for the derived allele at index SNPs in the genome regions associated with fingerprint patterns, related to <a id="crosref1740" href="#fig4">Figure&nbsp;4</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs4-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0255" role="paragraph">Estimates obtained in each cohort are shown as blue boxes. Box size is proportional to allele frequency of Allele1. Horizontal bars indicate confidence intervals representing 2&nbsp;× standard errors. Intervals that include zero (that is, non-significant effects) intersect the dashed vertical line. The estimates obtained in the Asian cohorts are shown in red boxes and those obtained in the European cohorts in blue boxes. Diamonds represent the beta and the error bars indicate the 95% confidence interval.</div></div></div></div></figcaption></figure></div><div id="p0095" role="paragraph">We determined EVI1 expression during human embryonic development by immunofluorescence, finding broad expression in the mesenchyme of the early outgrowing limb bud at Carnegie Stage 17 (CS17) (∼6&nbsp;weeks estimated gestation age [EGA]), in contrast to low or absent expression in the trunk (<a id="crosref0710" href="#fig3">Figure&nbsp;3</a>G). By 10&nbsp;weeks EGA, EVI1 was prominently expressed in the mesenchyme of the distal ends of the digits, notably under the volar pads, the sites of later fingerprint formation (<a id="crosref0715" href="#fig3">Figure&nbsp;3</a>H). By 13&nbsp;weeks EGA expression is largely lost, being present only at the periphery of the distal phalanx (<a id="crosref0720" href="#fig3">Figure&nbsp;3</a>I), and at 16&nbsp;weeks EGA, as dermatoglyphs are emerging as a periodically corrugated epidermis with extending sweat gland primordia, EVI1 expression is greatly reduced and it is not detected in these structures (<a id="crosref0725" href="#fig3">Figure&nbsp;3</a>J).</div><div id="p0100" role="paragraph">To further refine the cell population expressing <i>EVI1</i>, we assessed its coexpression with the general limb bud mesenchyme marker <i>PRRX1</i>. At week 10, expression is detected in the distal digit and the deeper parts of the distal volar pad, in the same cell population as expresses <i>PRRX1</i> (<a id="crosref0730" href="#fig3">Figure&nbsp;3</a>K). Expression of <i>EVI1</i> in human development, as in mouse (<a id="crosref0735" href="#fig3">Figure&nbsp;3</a>E), is similar on dorsal and ventral sides of the digit, though broader on the ventral side as it extends into the lower volar pad. As observed for the protein, <i>EVI1</i> mRNA is not detected in emerging fingerprint ridges at week 16 EGA (<a id="crosref0740" href="#fig3">Figure&nbsp;3</a>L). Thus, in human development EVI1 is expressed broadly in mesenchyme during limb growth and then under the digit pad upon which the dermatoglyph pattern will form, but expression is not associated with the epithelial folding of the fingerprint itself.</div><div id="p0105" role="paragraph">Early mouse and chicken limbs grow through proliferation from their distal end (<span class="dropBlock reference-citations"><a id="body-ref-sref112-1" href="#" role="doc-biblioref" data-xml-rid="bib112" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref112-1" aria-controls="sref112-1">Towers and Tickle, 2009</a><div class="dropBlock__holder" data-db-target-of="sref112-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">122.</div><div>Towers, M. ∙ Tickle, C.</div><div><strong>Growing models of vertebrate limb development</strong></div><div><em>Development.</em> 2009; <strong>136</strong>:179-190</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1242/dev.024158" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_122_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-62349111082" target="_blank" aria-label="Open Scopus in new tab">Scopus (133)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19103802/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1242%2Fdev.024158&amp;pmid=19103802" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), and we find high levels of cell proliferation, marked by Ki67 immunofluorescence, in 10-week EGA digits in the distal tip mesenchyme (<a id="crosref0750" href="#fig3">Figure&nbsp;3</a>M). This region is a major site of EVI1 expression and because the principal function of this transcription factor is to promote proliferation (<span class="dropBlock reference-citations"><a id="body-ref-sref50-1" href="#" role="doc-biblioref" data-xml-rid="bib50" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref50-1" aria-controls="sref50-1">Hoyt et&nbsp;al., 1997</a><div class="dropBlock__holder" data-db-target-of="sref50-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">53.</div><div>Hoyt, P.R. ∙ Bartholomew, C. ∙ Davis, A.J. ...</div><div><strong>The Evi1 proto-oncogene is required at midgestation for neural, heart, and paraxial mesenchyme development</strong></div><div><em>Mech. Dev.</em> 1997; <strong>65</strong>:55-70</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/S0925-4773(97)00057-9" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-0031173051" target="_blank" aria-label="Open Scopus in new tab">Scopus (129)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/9256345/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0925-4773%2897%2900057-9&amp;pmid=9256345" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), it is likely that altered <i>EVI1</i> regulation changes patterns of cell growth and thus the length and shape of the distal limb and digits.</div></section><section id="sec-2-4"><h3>Trans-ethnic meta-analysis reveals a fingerprint pattern-associated gene set enriched for limb development functions</h3><div id="p0110" role="paragraph">To more fully understand the genetic architecture of fingerprint patterns, we performed a meta-analysis of both East Asian (EAS)-ancestry cohorts (TZL, NSPT, JD, CKB and WeGene), and European (EUR)-ancestry cohorts, including the Avon Longitudinal Study of Parents and Children (ALSPAC) birth cohort study, the Queensland Institute of Medical Research (QIMR) twin studies, and the Pittsburgh cohort (Pittsburgh) using the binary phenotypes (non-whorl or whorl) on all ten digits (<a id="crosref0760" href="#mmc1">Table S1</a>). This large-scale meta-analysis of more than 23,000 individuals identified 43 independent signals mapping to 105 notable genes (p<sub>adj</sub>&nbsp;&lt; 1.67&nbsp;× 10<sup>−8</sup>) (<a id="crosref0765" href="#fig4">Figure&nbsp;4</a>A; <a id="crosref0770" href="#mmc5">Table S5</a>; see also <a id="crosref0775" href="#mmc7">Data S1</a> for detailed LocusZoom plots), which explained 4.6%–7.9% of the phenotypic variance by polygenic risk score (<a id="crosref0780" href="#sec-4">STAR Methods</a>). Of these 43 signals, 4 signals were independently identified in both EAS-ancestry and EUR-ancestry cohorts; 27 were only identified in EAS-ancestry cohorts, 9 of which were also nominally significant in EUR-ancestry cohorts; 2 were only identified in EUR-ancestry cohorts, both of which were also nominally significant in EAS-ancestry cohorts; 10 were identified only after trans-ethnic meta-analysis (<a id="crosref0785" href="#fig4">Figure&nbsp;4</a>B; <a id="crosref0790" href="#mmc5">Table S5</a>). Among the 18 signals that were only identified in EAS-ancestry and did not reach nominal significance in EUR-ancestry cohorts, 9 were either not available or with low minor allele frequency (MAF&nbsp;&lt; 0.04, whereas the other 9 all showed suggestive level of association in EUR-ancestry cohorts (6.61&nbsp;× 10<sup>−6</sup>&nbsp;&lt; p&nbsp;&lt; 5.69&nbsp;× 10<sup>−3</sup>), with exactly the same effect direction as in EAS-ancestry cohorts (<a id="crosref0795" href="#mmc5">Table S5</a>). Three signals were genome-wide significant in either the EAS or EUR-ancestry cohorts, but were only nominally significant in the trans-ethnic meta-analysis (<a id="crosref0800" href="#mmc5">Table S5</a>). These results largely indicated that the fingerprint related genes are generally the same in European and East Asian ancestry populations, with some differences that are likely explained by differing allelic effect sizes or frequencies among populations (<a id="crosref0805" href="#figs4">Figure&nbsp;S4</a>).</div><div class="figure-wrap"><figure id="fig4" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.12.008/asset/fb1d51d0-1ccf-48a3-a4e1-817e292dbfaf/main.assets/gr4_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr4" src="/cms/10.1016/j.cell.2021.12.008/asset/29542a0e-957f-46c2-a880-aa6406278697/main.assets/gr4.jpg" height="840" width="657" aria-labelledby="fig4-title fig4-content" data-viewer-src="/cms/10.1016/j.cell.2021.12.008/asset/fb1d51d0-1ccf-48a3-a4e1-817e292dbfaf/main.assets/gr4_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig4-title" class="figure__title"><span class="label figure__label">Figure&nbsp;4</span> <span class="figure__title__text">A meta-analysis of fingerprint patterns showing signals enriched in limb development</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig4-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0125" role="paragraph">(A) A Manhattan plot showing the results of the meta-analyses combining GWAS of East Asian (EAS)-ancestry (TZL, NSPT, JD, CKB, and WeGene) and European (EUR)-ancestry cohorts (ALSPAC, QIMR, and Pittsburgh) across all ten digits (D1L/R were unavailable in JD and ALSPAC). There were 43 signals associated with fingerprint patterns of at least one digit (p<sub>adj</sub>&nbsp;&lt; 1.67&nbsp;× 10<sup>−8</sup>; <a id="crosref1675" href="#mmc5">Table S5</a>), with gene names in different colors: purple indicating significant in both EAS and EUR; red and blue indicating only significant in EAS and EUR, respectively; and green indicating not significant in either EAS or EUR but only significant after the meta-analysis combining both. Bold genes showed associations with limb phenotypes abnormalities (<a id="crosref1680" href="#tbl2">Table 2</a>). The block map on the right represented the digits corresponding to the signals on the left. Red and blue triangles indicate significance in EAS and EUR, respectively, whereas dark and light colors represented signals that reached the adjusted genome-wide significant (p<sub>adj</sub>&nbsp;&lt; 1.67&nbsp;× 10<sup>−8</sup>) and suggestive levels (p<sub>adj</sub>&nbsp;&lt; 3.33&nbsp;× 10<sup>−6</sup>), respectively. Bold frame indicated genome-wide significant (black) or suggestive (gray) significant after combined meta-analyses.</div><div id="fspara0130" role="paragraph">(B) Venn diagram summarizing fingerprint-associated signals corresponding to (A).</div><div id="fspara0135" role="paragraph">(C) Enrichment of annotations across ontologies for the 43 fingerprint-associated signals. The red asterisk indicates limb-relevant terms that genes are significantly enriched in after Bonferroni correction (the red dotted lines). Only the top 10 terms ranked after enrichment analysis and top 5 epithelial/skin-related terms are shown.</div><div id="fspara0140" role="paragraph">(D) Fingerprint pattern-associated proteins and their interactions. The nodes represent proteins and the links represent the existence of protein-protein or regulatory interactions. Edge thickness was proportional to the weight of the edge (assigned with respect to STRING score). Filled nodes indicate proteins involved in the interaction network, whereas empty nodes indicate proteins that are independent of the network. The two nodes indicated by red arrows (WNT5A and IRX3) represent the centered, highly connected proteins. Red nodes denote proteins reported to be involved in limb development, while blue nodes have not. Gray nodes are extended additional nodes to restrict the number of direct interactions with input nodes to 10 in the current network. Green triangles indicate notable genes associated with composite phenotypes (<a id="crosref1685" href="#fig2">Figure&nbsp;2</a>C; <a id="crosref1690" href="#mmc3">Table S3</a>).</div><div id="fspara0145" role="paragraph">See also <a id="crosref1695" href="#figs4">Figure&nbsp;S4</a>, <a id="crosref1700" href="#mmc5">Tables S5</a> and <a id="crosref1705" href="#mmc6">S6</a>, and <a id="crosref1710" href="#mmc7">Data S1</a>.</div></div></div></div></figcaption></figure></div><div id="p0115" role="paragraph">Genomic enrichment analysis using GREAT V4.0.4 (<span class="dropBlock reference-citations"><a id="body-ref-sref76-1" href="#" role="doc-biblioref" data-xml-rid="bib76" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref76-1" aria-controls="sref76-1">McLean et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref76-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">83.</div><div>McLean, C.Y. ∙ Bristor, D. ∙ Hiller, M. ...</div><div><strong>GREAT improves functional interpretation of cis-regulatory regions</strong></div><div><em>Nat. Biotechnol.</em> 2010; <strong>28</strong>:495-501</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nbt.1630" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_83_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-77952214662" target="_blank" aria-label="Open Scopus in new tab">Scopus (2170)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20436461/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnbt.1630&amp;pmid=20436461" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) (<a id="crosref0815" href="#mmc5">Table S5</a>; <a id="crosref0820" href="#sec-4">STAR Methods</a>) found that the fingerprint-associated signals were significantly enriched for embryonic-development and morphogenesis-related Gene Ontology biological processes including “limb development,” “Embryonic limb morphogenesis,” and “limb morphogenesis” (-log10(p) &gt; 8; <a id="crosref0825" href="#fig4">Figure&nbsp;4</a>C); whereas epithelial and skin-related pathways including “morphogenesis of an epithelium” and “epithelial tube morphogenesis” did not reach the significance threshold after correction for multiple-testing (-log10(p<sub>thr</sub>)&nbsp;= 5.42, <a id="crosref0830" href="#fig4">Figure&nbsp;4</a>C). Similar patterns of enrichment for limb rather than skin-related terms were observed when using human phenotype annotations and mouse morphology terms for the analysis (<a id="crosref0835" href="#fig4">Figure&nbsp;4</a>C). Enrichment was also observed for expression in mouse limb tissues at Theiler Stage 17 (E10.5) and 19 (E11.5) when the forelimbs are divided into two regions indicating proximal limb and autopod but not in the developing autopod epithelium (<a id="crosref0840" href="#fig4">Figure&nbsp;4</a>C).</div><div id="p0120" role="paragraph">Network analysis using STRING v11 (<span class="dropBlock reference-citations"><a id="body-ref-sref111-1" href="#" role="doc-biblioref" data-xml-rid="bib111" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref111-1" aria-controls="sref111-1">Szklarczyk et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref111-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">120.</div><div>Szklarczyk, D. ∙ Gable, A.L. ∙ Lyon, D. ...</div><div><strong>STRING v11: protein-protein association networks with increased coverage, supporting functional discovery in genome-wide experimental datasets</strong></div><div><em>Nucleic Acids Res.</em> 2019; <strong>47</strong>:D607-D613</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gky1131" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_120_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-85059797047" target="_blank" aria-label="Open Scopus in new tab">Scopus (4748)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30476243/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgky1131&amp;pmid=30476243" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) linked fingerprint-associated GWAS genes and proteins by known functions and regulatory interactions (see <a id="crosref0850" href="#sec-4">STAR Methods</a>), showing a network centered around WNT5A and IRX3 (<a id="crosref0855" href="#fig4">Figure&nbsp;4</a>D), suggesting relevant developmental pathways. Interestingly, both WNT5A and IRX3 were among the GWAS signals of the composite phenotype of middle three fingers (<a id="crosref0860" href="#fig2">Figure&nbsp;2</a>C). Eight proteins from 6 of the 12 composite phenotype-associated signals appeared within two steps of the centered proteins with higher degrees in the network (WNT5A and IRX3), suggesting that these developmental pathways are relevant to the biological mechanism underlying the “pattern-block” phenomenon. Another network was centered on NOTCH signaling (NOTCH1, although not itself a GWAS hit), responsible for maintaining mesenchymal progenitors of the limb in an undifferentiated and proliferative state during its development (<span class="dropBlock reference-citations"><a id="body-ref-sref27" href="#" role="doc-biblioref" data-xml-rid="bib27" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref27" aria-controls="sref27">Dong et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref27"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">28.</div><div>Dong, Y. ∙ Jesse, A.M. ∙ Kohn, A. ...</div><div><strong>RBPjkappa-dependent Notch signaling regulates mesenchymal progenitor cell proliferation and differentiation during skeletal development</strong></div><div><em>Development.</em> 2010; <strong>137</strong>:1461-1471</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1242/dev.042911" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-77951229591" target="_blank" aria-label="Open Scopus in new tab">Scopus (134)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20335360/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1242%2Fdev.042911&amp;pmid=20335360" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div></section><section id="sec-2-5"><h3>Fingerprint patterns are significantly associated with hand proportions, showing strong genetic correlations with fingerprint pattern type</h3><div id="p0125" role="paragraph">To further test the hypothesis that embryonic and fetal limb development may influence fingerprint pattern types, we examined the correlation between fingerprint patterns and limb-related phenotypes (i.e., hand phenotypes). We measured hand phenotypes (see <a id="crosref0870" href="#fig5">Figure&nbsp;5</a>A for hand and digit length and <a id="crosref0875" href="#figs5">Figure&nbsp;S5</a> for distal phalanx length and other hand phenotypes) in NSPT and JD cohorts. We found broad associations between hand phenotypes and fingerprint patterns (e.g., frequency of whorls) (<a id="crosref0880" href="#mmc6">Table S6</a>). Higher frequency of whorl patterns was associated with longer little finger (D5) relative to hand length (ratio of D5 length to hand length [DHR5]; β&nbsp;= 0.15, p&nbsp;= 2.42&nbsp;× 10<sup>−5</sup>) (<a id="crosref0885" href="#fig5">Figure&nbsp;5</a>B; <a id="crosref0890" href="#mmc6">Table S6</a>). This was clearly seen in the direct comparison of DHR5 in individuals with whorl and non-whorl fingerprint patterns on different digits, particularly on D5 (p<sub>D5L</sub>&nbsp;= 1.74&nbsp;× 10<sup>−13</sup>, p<sub>D5R</sub>&nbsp;= 5.85&nbsp;× 10<sup>−13</sup>) (<a id="crosref0895" href="#fig5">Figure&nbsp;5</a>C). Individuals with whorl pattern on D5 of both hands (56.83 ± 4.34&nbsp;mm) had on average a 1.32&nbsp;mm longer D5 than individuals with no whorl pattern on D5 (55.51 ± 4.52&nbsp;mm). These phenotypic correlations also showed a strong genetic basis because significant genetic correlations were found between DHR5 and fingerprint patterns on different digits, especially on D5 (r<sub>g-D5L</sub>&nbsp;= 0.31, p&nbsp;= 3.89&nbsp;× 10<sup>−16</sup>, r<sub>g-D5R</sub>&nbsp;= 0.22, p&nbsp;= 1.26&nbsp;× 10<sup>−11</sup>) (<a id="crosref0900" href="#fig5">Figure&nbsp;5</a>D; <a id="crosref0905" href="#mmc6">Table S6</a>). Indeed, we found that fingerprint-associated signals (2p24.3-<i>MYCN</i>, 3p14.3-<i>WNT5A</i>, 12q24.21-<i>TBX3</i>, and 18q23-<i>SALL3</i>) were also associated with the length of D5 or DHR5 (<a id="crosref0910" href="#mmc5">Table S5</a>). Apart from the digit length, we found that the distal phalanx of D2 and D3 (ratio of D2 or D3 distal phalanx length to hand length [DPHR2 or DPHR3]) were also associated with whorl patterns (β<sub>DPHR2</sub>&nbsp;= −0.31, p&nbsp;= 9.13&nbsp;× 10<sup>−18</sup>, β<sub>DPHR3</sub>&nbsp;= −0.24, p&nbsp;= 2.02&nbsp;× 10<sup>−11</sup>) (<a id="crosref0915" href="#mmc6">Table S6</a>) and with fingerprint-associated signals (2q31.1-<i>SP9</i>, 8q23.1-<i>RSPO2</i>, etc.) (<a id="crosref0920" href="#mmc5">Table S5</a>). Together, these results provided further evidence that fingerprint patterns are strongly influenced by the process of limb development.</div><div class="figure-wrap"><figure id="fig5" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.12.008/asset/4a580541-3d9e-4238-9200-24eb52782f2f/main.assets/gr5_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr5" src="/cms/10.1016/j.cell.2021.12.008/asset/abd3aead-8aa8-4cd4-9bdb-f22760165c9f/main.assets/gr5.jpg" height="392" width="506" aria-labelledby="fig5-title fig5-content" data-viewer-src="/cms/10.1016/j.cell.2021.12.008/asset/4a580541-3d9e-4238-9200-24eb52782f2f/main.assets/gr5_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig5-title" class="figure__title"><span class="label figure__label">Figure&nbsp;5</span> <span class="figure__title__text">Association between fingerprint patterns and hand phenotypes (n&nbsp;= 6,318)</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig5-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0155" role="paragraph">(A) Diagrammed human hand with measured phenotypes, including hand and digit length. The digit–hand ratio (DHR) is the ratio of digit length and hand length.</div><div id="fspara0160" role="paragraph">(B) The association between the whorl frequency of eight digits (D2–D5) and the DHR of each digit. We used <i>Z</i> score to standardize the mean DHR of left and right hands. Red dots indicate the average values and short black lines the standard deviation for each group. The arrow indicates the linear regression passes the significance test.</div><div id="fspara0165" role="paragraph">(C) Bar plot of fingerprint patterns of each digit (D2-D5) and the mean DHR of D5. Error bars indicate SEM. <sup>∗</sup>p&nbsp;&lt; 0.05, <sup>∗∗</sup>p&nbsp;&lt; 0.01, <sup>∗∗∗</sup>p&nbsp;&lt; 0.001.</div><div id="fspara0170" role="paragraph">(D) Genetic correlations between fingerprint patterns and the mean DHR of D5. Estimates and tests were performed using the bivariate GREML of GCTA software. Error bars indicate SEM.</div><div id="fspara0175" role="paragraph">See also <a id="crosref1715" href="#figs5">Figure&nbsp;S5</a> and <a id="crosref1720" href="#mmc6">Table S6</a>.</div></div></div></div></figcaption></figure></div><div class="figure-wrap"><figure id="figs5" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.12.008/asset/2a1c0e81-ac83-49a4-99f9-4494ce3de105/main.assets/figs5_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs5" src="/cms/10.1016/j.cell.2021.12.008/asset/c5187c16-fff0-486f-a85c-3e655c722e32/main.assets/figs5.jpg" height="398" width="657" aria-labelledby="figs5-title figs5-content" data-viewer-src="/cms/10.1016/j.cell.2021.12.008/asset/2a1c0e81-ac83-49a4-99f9-4494ce3de105/main.assets/figs5_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs5-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S5</span> <span class="figure__title__text">Association between fingerprint patterns and hand proportions, related to <a id="crosref1745" href="#fig5">Figure&nbsp;5</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs5-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0265" role="paragraph">(A) Diagrammed human hand with measured phenotypes, including hand and distal phalanx length. The distal phalanx-hand ratio (DPHR) is the ratio of distal phalanx length to hand length.</div><div id="fspara0270" role="paragraph">(B) The association between the whorl frequency on eight digits and the DPHR of each digit. We used Z-score to standardize the mean DPHR of left and right hands. Solid dots indicated the average values and short black lines were standard deviation for each group. The arrow indicates the linear regression passes the significance test.</div><div id="fspara0275" role="paragraph">(C-D) Bar plot of fingerprint pattern type (non-whorl versus whorl) of each digit (D2-D5) and the mean DPHR of D2 or D3. Error bars indicate SEM. <sup>∗</sup>p&nbsp;&lt; 0.05, <sup>∗∗</sup>&nbsp;&lt; 0.01, <sup>∗∗∗</sup>&nbsp;&lt; 0.001.</div><div id="fspara0280" role="paragraph">(E-F) Genetic correlations between fingerprint patterns and the mean DPHR of D2 or D3. Estimates and tests were performed using the bivariate GREML of GCTA software. Error bars indicate SEM.</div></div></div></div></figcaption></figure></div></section></section><section id="sec-3"><h2>Discussion</h2><div id="p0130" role="paragraph">By leveraging a large-scale GWAS, we identified many fingerprint pattern-associated genetic variants and inferred their associated biological processes. As with other complex traits, the phenotypic variance explained by the 43 top genetic signals from meta-analysis using polygenic risk scores was small, and together with the inherent stochasticity of skin patterning processes (<span class="dropBlock reference-citations"><a id="body-ref-sref89" href="#" role="doc-biblioref" data-xml-rid="bib89" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref89" aria-controls="sref89">Painter et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref89"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">96.</div><div>Painter, K.J. ∙ Hunt, G.S. ∙ Wells, K.L. ...</div><div><strong>Towards an integrated experimental-theoretical approach for assessing the mechanistic basis of hair and feather morphogenesis</strong></div><div><em>Interface Focus.</em> 2012; <strong>2</strong>:433-450</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1098/rsfs.2011.0122" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_96_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-84862663863" target="_blank" aria-label="Open Scopus in new tab">Scopus (23)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23919127/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1098%2Frsfs.2011.0122&amp;pmid=23919127" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), it is clear that this work will not permit prediction of fingerprint pattern from an individual’s genotype. This work shows that the enormous diversity of fingerprint patterns is at a basic level influenced by the dynamics and shaping of the underlying limb structure.</div><div id="p0135" role="paragraph">There are various ways to quantify fingerprint phenotypes (see <a id="crosref0930" href="#mmc3">Table S3</a>). Interestingly, we found that GWAS of the ordinal phenotype (0, 1, and 2 for arch, loop, and whorl, respectively) provided the most significant and the greatest number of signals, suggesting that this ordinal phenotype might better reflect the underlying genetic mechanisms of pattern formation. It has been proposed that the morphology (i.e., height, shape, and size) of volar pads (<span class="dropBlock reference-citations"><a id="body-ref-sref13" href="#" role="doc-biblioref" data-xml-rid="bib13" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref13" aria-controls="sref13">Bonnevie, 1924</a><div class="dropBlock__holder" data-db-target-of="sref13"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">13.</div><div>Bonnevie, K.</div><div><strong>Studies on papillary patterns of human fingers</strong></div><div><em>J.&nbsp;Genet.</em> 1924; <strong>15</strong>:1-111</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/BF02983100" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2FBF02983100" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref92" href="#" role="doc-biblioref" data-xml-rid="bib92" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref92" aria-controls="sref92">Penrose, 1965b</a><div class="dropBlock__holder" data-db-target-of="sref92"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">99.</div><div>Penrose, L.S.</div><div><strong>On the geometry of loops and deltas</strong></div><div><em>Ann. Hum. Genet.</em> 1965; <strong>29</strong>:207-211</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1111/j.1469-1809.1965.tb00515.x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_99_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-0013815364" target="_blank" aria-label="Open Scopus in new tab">Scopus (5)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/5863842/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1111%2Fj.1469-1809.1965.tb00515.x&amp;pmid=5863842" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref121" href="#" role="doc-biblioref" data-xml-rid="bib121" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref121" aria-controls="sref121">Wertheim and Maceo, 2002</a><div class="dropBlock__holder" data-db-target-of="sref121"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">131.</div><div>Wertheim, K. ∙ Maceo, A.</div><div><strong>The critical stage of friction ridge and pattern formation</strong></div><div><em>J.&nbsp;Forensic Identif.</em> 2002; <strong>52</strong>:35</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=K.WertheimA.MaceoThe+critical+stage+of+friction+ridge+and+pattern+formationJ.%C2%A0Forensic+Identif.52200235" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and the growth stresses on the pad surface (<span class="dropBlock reference-citations"><a id="body-ref-sref59-2" href="#" role="doc-biblioref" data-xml-rid="bib59" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref59-2" aria-controls="sref59-2">Kücken, 2007</a><div class="dropBlock__holder" data-db-target-of="sref59-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">64.</div><div>Kücken, M.</div><div><strong>Models for fingerprint pattern formation</strong></div><div><em>Forensic Sci. Int.</em> 2007; <strong>171</strong>:85-96</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.forsciint.2007.02.025" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_64_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-34547691328" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17459625/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.forsciint.2007.02.025&amp;pmid=17459625" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) play important roles to influence pattern types. In particular, although ridges forming on high volar pads typically conform to the whorl-type pattern, low volar pads produce arch-type patterns, and asymmetric and intermediate height volar pads often form loop-type patterns (<span class="dropBlock reference-citations"><a id="body-ref-sref9" href="#" role="doc-biblioref" data-xml-rid="bib9" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref9" aria-controls="sref9">Babler, 1987</a><div class="dropBlock__holder" data-db-target-of="sref9"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">9.</div><div>Babler, W.</div><div><strong>Prenatal development of dermatoglyphic digital patterns: Associations with epidermal ridge, volar pad and bone morphology</strong></div><div><em>Coll. Antropol.</em> 1987; <strong>11</strong>:297-303</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=W.BablerPrenatal+development+of+dermatoglyphic+digital+patterns%3A+Associations+with+epidermal+ridge%2C+volar+pad+and+bone+morphologyColl.+Antropol.111987297303" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref83" href="#" role="doc-biblioref" data-xml-rid="bib83" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref83" aria-controls="sref83">Mulvihill and Smith, 1969</a><div class="dropBlock__holder" data-db-target-of="sref83"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">90.</div><div>Mulvihill, J.J. ∙ Smith, D.W.</div><div><strong>The genesis of dermatoglyphics</strong></div><div><em>J.&nbsp;Pediatr.</em> 1969; <strong>75</strong>:579-589</div></div><div class="external-links"><div class="core-xlink-abstract"><a href="/servlet/linkout?suffix=e_1_5_1_2_90_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=10.1016%2FS0022-3476%2869%2980453-1&amp;cf=abstract&amp;site=ympd-site" target="_blank" aria-label="Open Abstract in new tab">Abstract</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_90_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=10.1016%2FS0022-3476%2869%2980453-1&amp;cf=pdf&amp;site=ympd-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/4309281/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0022-3476%2869%2980453-1&amp;pmid=4309281" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref93" href="#" role="doc-biblioref" data-xml-rid="bib93" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref93" aria-controls="sref93">Penrose and Ohara, 1973</a><div class="dropBlock__holder" data-db-target-of="sref93"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">100.</div><div>Penrose, L.S. ∙ Ohara, P.T.</div><div><strong>The development of the epidermal ridges</strong></div><div><em>J.&nbsp;Med. Genet.</em> 1973; <strong>10</strong>:201-208</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1136/jmg.10.3.201" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/4774533/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1136%2Fjmg.10.3.201&amp;pmid=4774533" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). This proposed link between height of volar pads and type of fingerprint pattern is consistent with the empirical justification of the ordinal phenotype (regarding loop as an intermediate phenotype between arch and whorl) in our study.</div><div id="p0140" role="paragraph">Of the genes identified in the 43 meta-GWAS signals, only <i>EDAR</i> had previously been implicated in dermatoglyph formation, with loss of EDAR activity producing highly abnormal fingerprints as part of the rare condition hypohidrotic ectodermal dysplasia (<span class="dropBlock reference-citations"><a id="body-ref-sref52" href="#" role="doc-biblioref" data-xml-rid="bib52" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref52" aria-controls="sref52">Kargül et&nbsp;al., 2001</a><div class="dropBlock__holder" data-db-target-of="sref52"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">57.</div><div>Kargül, B. ∙ Alcan, T. ∙ Kabalay, U. ...</div><div><strong>Hypohidrotic ectodermal dysplasia: dental, clinical, genetic and dermatoglyphic findings of three cases</strong></div><div><em>J.&nbsp;Clin. Pediatr. Dent.</em> 2001; <strong>26</strong>:5-12</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.17796/jcpd.26.1.g156r02u66w30729" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11688814/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.17796%2Fjcpd.26.1.g156r02u66w30729&amp;pmid=11688814" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref116" href="#" role="doc-biblioref" data-xml-rid="bib116" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref116" aria-controls="sref116">Verbov, 1970</a><div class="dropBlock__holder" data-db-target-of="sref116"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">126.</div><div>Verbov, J.</div><div><strong>Hypohidrotic (or anhidrotic) ectodermal dysplasia--an appraisal of diagnostic methods</strong></div><div><em>Br. J. Dermatol.</em> 1970; <strong>83</strong>:341-348</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1111/j.1365-2133.1970.tb15713.x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_126_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-0014845184" target="_blank" aria-label="Open Scopus in new tab">Scopus (27)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/5479305/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1111%2Fj.1365-2133.1970.tb15713.x&amp;pmid=5479305" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). We found that in <i>Edar</i> mutant mice, the transverse digital ridges are profoundly aberrant, though, as in humans, limb and digit structure appears normal (<a id="crosref0980" href="#figs3">Figure&nbsp;S3</a>C). EDAR signaling in the surface ectoderm is thus likely to influence dermatoglyph patterns directly, in a manner similar to this pathway’s role in defining hair follicle spatial arrangement (<span class="dropBlock reference-citations"><a id="body-ref-sref82" href="#" role="doc-biblioref" data-xml-rid="bib82" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref82" aria-controls="sref82">Mou et&nbsp;al., 2006</a><div class="dropBlock__holder" data-db-target-of="sref82"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">89.</div><div>Mou, C. ∙ Jackson, B. ∙ Schneider, P. ...</div><div><strong>Generation of the primary hair follicle pattern</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2006; <strong>103</strong>:9075-9080</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.0600825103" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_89_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-33745121169" target="_blank" aria-label="Open Scopus in new tab">Scopus (149)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16769906/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.0600825103&amp;pmid=16769906" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Collectively, however, the set of genes implicated from our GWAS represents a strong signal of limb growth regulation, particularly of the distal limb, evidenced by their developmental expression patterns and the anatomical phenotypes induced by their mutation in humans and mice.</div><div id="p0145" role="paragraph"><i>Evi1</i> mutation in mice alters the transverse digital ridge pattern, establishing the EVI1 protein as a determinant of dermatoglyph pattern. During human development, we did not detect EVI1 expression in the epidermal ridges that form the dermatoglyph; rather, expression was prominent at an earlier stage in the distal ends of the digits and under the transiently raised pads at the fingertips on which the fingerprints later form. These volar pads, built on a mesenchymal core, have been postulated to be key determinants of dermatoglyph characteristics because their variable shapes in the fetus are reported to be correlated with different dermatoglyph types (<span class="dropBlock reference-citations"><a id="body-ref-sref10-2" href="#" role="doc-biblioref" data-xml-rid="bib10" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref10-2" aria-controls="sref10-2">Babler, 1991</a><div class="dropBlock__holder" data-db-target-of="sref10-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">10.</div><div>Babler, W.J.</div><div><strong>Embryologic development of epidermal ridges and their configurations</strong></div><div><em>Birth Defects Orig. Artic. Ser.</em> 1991; <strong>27</strong>:95-112</div></div><div class="external-links"><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/1786361/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?pmid=1786361" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref67" href="#" role="doc-biblioref" data-xml-rid="bib67" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref67" aria-controls="sref67">Loesch, 1983</a><div class="dropBlock__holder" data-db-target-of="sref67"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">73.</div><div>Loesch, D.Z.</div><div><strong>Quantitative Dermatoglyphics. Classification, Genetics, and Pathology</strong></div><div>Oxford University Press, 1983</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=D.Z.LoeschQuantitative+Dermatoglyphics.+Classification%2C+Genetics%2C+and+Pathology1983Oxford+University+Press" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). EVI1 promotes cell proliferation during development (<span class="dropBlock reference-citations"><a id="body-ref-sref50-2" href="#" role="doc-biblioref" data-xml-rid="bib50" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref50-2" aria-controls="sref50-2">Hoyt et&nbsp;al., 1997</a><div class="dropBlock__holder" data-db-target-of="sref50-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">53.</div><div>Hoyt, P.R. ∙ Bartholomew, C. ∙ Davis, A.J. ...</div><div><strong>The Evi1 proto-oncogene is required at midgestation for neural, heart, and paraxial mesenchyme development</strong></div><div><em>Mech. Dev.</em> 1997; <strong>65</strong>:55-70</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/S0925-4773(97)00057-9" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-0031173051" target="_blank" aria-label="Open Scopus in new tab">Scopus (129)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/9256345/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0925-4773%2897%2900057-9&amp;pmid=9256345" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), and its expression in the proliferative mesenchyme of the distal limb suggests that it may modulate the shape and size of these pads and distal digit elongation by altering cell production. Volar pads are present but undergoing regression across the period of fingerprint establishment, though mutation at fingerprint pattern-associated genes <i>TCF4</i> or the <i>DBX2</i> locus are associated with retention of fetal pads into adulthood, indicating a role for these genes in pad growth and regression (<a id="crosref1005" href="#tbl2">Table 2</a>). <i>EVI1</i> is also a proto-oncogene, with increased expression driving acute myeloid leukemia and pediatric mixed lineage leukemia through suppression of cellular differentiation and apoptosis in hematopoietic lineages (<span class="dropBlock reference-citations"><a id="body-ref-sref40" href="#" role="doc-biblioref" data-xml-rid="bib40" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref40" aria-controls="sref40">Glass et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref40"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">41.</div><div>Glass, C. ∙ Wilson, M. ∙ Gonzalez, R. ...</div><div><strong>The role of EVI1 in myeloid malignancies</strong></div><div><em>Blood Cells Mol. Dis.</em> 2014; <strong>53</strong>:67-76</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.bcmd.2014.01.002" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-84900033263" target="_blank" aria-label="Open Scopus in new tab">Scopus (46)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24495476/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.bcmd.2014.01.002&amp;pmid=24495476" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Fingerprint pattern type has been reported to be associated with incidence of leukemia (<span class="dropBlock reference-citations"><a id="body-ref-sref79" href="#" role="doc-biblioref" data-xml-rid="bib79" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref79" aria-controls="sref79">Menser and Purvis-Smith, 1969</a><div class="dropBlock__holder" data-db-target-of="sref79"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">86.</div><div>Menser, M.A. ∙ Purvis-Smith, S.G.</div><div><strong>Dermatoglyphic defects in children with leukaemia</strong></div><div><em>Lancet.</em> 1969; <strong>1</strong>:1076-1078</div></div><div class="external-links"><div class="core-xlink-abstract"><a href="/servlet/linkout?suffix=e_1_5_1_2_86_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=10.1016%2FS0140-6736%2869%2991710-3&amp;cf=abstract&amp;site=lancet-site" target="_blank" aria-label="Open Abstract in new tab">Abstract</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/4181736/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0140-6736%2869%2991710-3&amp;pmid=4181736" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref98" href="#" role="doc-biblioref" data-xml-rid="bib98" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref98" aria-controls="sref98">Rathee et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref98"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">107.</div><div>Rathee, R. ∙ Kamal, N. ∙ Kumar, A. ...</div><div><strong>Dermatoglyphic Patterns of Acute Leukemia Patients</strong></div><div><em>Int. Res. J. Biol. Sci.</em> 2014; <strong>3</strong>:90-93</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=R.RatheeN.KamalA.KumarM.VashistR.YadavDermatoglyphic+Patterns+of+Acute+Leukemia+PatientsInt.+Res.+J.+Biol.+Sci.320149093" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref101" href="#" role="doc-biblioref" data-xml-rid="bib101" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref101" aria-controls="sref101">Rosner, 1969</a><div class="dropBlock__holder" data-db-target-of="sref101"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">110.</div><div>Rosner, F.</div><div><strong>Dermatoglyphics of leukaemic children</strong></div><div><em>Lancet.</em> 1969; <strong>2</strong>:272-273</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="/servlet/linkout?suffix=e_1_5_1_2_110_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=10.1016%2FS0140-6736%2869%2990043-9&amp;site=lancet-site" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/4184135/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0140-6736%2869%2990043-9&amp;pmid=4184135" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), with the functional <i>EVI1</i> variants we identify potentially explaining some of this phenotypic association. Further clinical studies are needed to formally test this potential pleiotropic effect of <i>EVI1</i>.</div><div class="figure-wrap"><figure id="tbl2" class="table" data-figureviewer-ignore="true"><div class="table-wrap"><table data-figureviewer-ignore="true" tabindex="0"><thead><tr class="rowsep"><th>Genes</th><th>➀ Syndrome and Phenotype Description When Mutated, in Human (OMIM Number) or ➁ Mouse ➂ Expression Site in Embryonic Limb (Mouse, Unless Stated)</th></tr></thead><tbody><tr><td>1p34.1-<i>RNF220</i></td><td>➂ anterior limb bud (<span class="dropBlock reference-citations"><a id="body-ref-sref69" href="#" role="doc-biblioref" data-xml-rid="bib69" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref69" aria-controls="sref69">Ma et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref69"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">76.</div><div>Ma, P. ∙ Song, N.N. ∙ Li, Y. ...</div><div><strong>Fine-Tuning of Shh/Gli Signaling Gradient by Non-proteolytic Ubiquitination during Neural Patterning</strong></div><div><em>Cell Rep.</em> 2019; <strong>28</strong>:541-553.e4</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_76_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=10.1016%2Fj.celrep.2019.06.017&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_76_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=10.1016%2Fj.celrep.2019.06.017&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_76_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-85068110805" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31291587/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.celrep.2019.06.017&amp;pmid=31291587" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td></tr><tr><td>1p31.3-<i>NFIA</i></td><td>➀ brain malformations with or without urinary tract defects (<a id="intref0265" href="https://omim.org/entry/613735" target="_blank">613735</a>); ➂ distal limb bud (<span class="dropBlock reference-citations"><a id="body-ref-sref16" href="#" role="doc-biblioref" data-xml-rid="bib16" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref16" aria-controls="sref16">Chaudhry et&nbsp;al., 1997</a><div class="dropBlock__holder" data-db-target-of="sref16"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">16.</div><div>Chaudhry, A.Z. ∙ Lyons, G.E. ∙ Gronostajski, R.M.</div><div><strong>Expression patterns of the four nuclear factor I genes during mouse embryogenesis indicate a potential role in development</strong></div><div><em>Dev. Dyn.</em> 1997; <strong>208</strong>:313-325</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/(SICI)1097-0177(199703)208:3%3C313::AID-AJA3%3E3.0.CO;2-L" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-0031050274" target="_blank" aria-label="Open Scopus in new tab">Scopus (171)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/9056636/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2F%28SICI%291097-0177%28199703%29208%3A3%253C313%3A%3AAID-AJA3%253E3.0.CO%3B2-L&amp;pmid=9056636" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td></tr><tr><td>2p24.3-<i>MYCN</i><a id="crosref1845" href="#tbl2fna"><sup>a</sup></a></td><td>➀ Feingold syndrome; syndactyly and reduced middle phalanges (<a id="intref0270" href="https://omim.org/entry/164280" target="_blank">164280</a>); ➂ proliferating cells of distal limb bud mesenchyme (<span class="dropBlock reference-citations"><a id="body-ref-sref88" href="#" role="doc-biblioref" data-xml-rid="bib88" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref88" aria-controls="sref88">Ota et&nbsp;al., 2007</a><div class="dropBlock__holder" data-db-target-of="sref88"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">95.</div><div>Ota, S. ∙ Zhou, Z.Q. ∙ Keene, D.R. ...</div><div><strong>Activities of N-Myc in the developing limb link control of skeletal size with digit separation</strong></div><div><em>Development.</em> 2007; <strong>134</strong>:1583-1592</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1242/dev.000703" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_95_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-34248589385" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17360777/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1242%2Fdev.000703&amp;pmid=17360777" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td></tr><tr><td>2p16.1-<i>BCL11A</i></td><td>➀ Dias-Logan syndrome; Intellectual development disorder with persistent fetal hemoglobin (61711); ➂ mesenchyme of early limb bud then autopod, anterior and posterior margin of proximal limb, subsequently interdigital (<span class="dropBlock reference-citations"><a id="body-ref-sref124" href="#" role="doc-biblioref" data-xml-rid="bib124" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref124" aria-controls="sref124">Yamamoto et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref124"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">134.</div><div>Yamamoto, S. ∙ Uchida, Y. ∙ Ohtani, T. ...</div><div><strong>Hoxa13 regulates expression of common Hox target genes involved in cartilage development to coordinate the expansion of the autopodal anlage</strong></div><div><em>Dev. Growth Differ.</em> 2019; <strong>61</strong>:228-251</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1111/dgd.12601" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_134_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-85063132528" target="_blank" aria-label="Open Scopus in new tab">Scopus (4)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30895612/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1111%2Fdgd.12601&amp;pmid=30895612" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td></tr><tr><td>2q13-<i>EDAR</i><a id="crosref1860" href="#tbl2fnb"><sup>b</sup></a></td><td>➀ hypohidrotic ectodermal dysplasia; skin appendages absent, aberrant dermatoglyphs (<a id="intref0275" href="https://omim.org/entry/224900" target="_blank">224900</a>)</td></tr><tr><td>2q31.1-<i>SP9</i></td><td>➂ apical ectodermal ridge, regulated by FGF10 (<span class="dropBlock reference-citations"><a id="body-ref-sref54" href="#" role="doc-biblioref" data-xml-rid="bib54" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref54" aria-controls="sref54">Kawakami et&nbsp;al., 2004</a><div class="dropBlock__holder" data-db-target-of="sref54"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">59.</div><div>Kawakami, Y. ∙ Esteban, C.R. ∙ Matsui, T. ...</div><div><strong>Sp8 and Sp9, two closely related buttonhead-like transcription factors, regulate Fgf8 expression and limb outgrowth in vertebrate embryos</strong></div><div><em>Development.</em> 2004; <strong>131</strong>:4763-4774</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1242/dev.01331" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-7244248753" target="_blank" aria-label="Open Scopus in new tab">Scopus (123)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15358670/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1242%2Fdev.01331&amp;pmid=15358670" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td></tr><tr><td>2q31.1-<i>MAP3K20/ZAK</i><a id="crosref1870" href="#tbl2fna"><sup>a</sup></a></td><td>➀ split-foot malformation; mesoaxial polydactyly, nail duplications (<a id="intref0280" href="https://omim.org/entry/616890" target="_blank">616890</a>); ➂ entire early limb bud (<span class="dropBlock reference-citations"><a id="body-ref-sref108" href="#" role="doc-biblioref" data-xml-rid="bib108" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref108" aria-controls="sref108">Spielmann et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref108"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">117.</div><div>Spielmann, M. ∙ Kakar, N. ∙ Tayebi, N. ...</div><div><strong>Exome sequencing and CRISPR/Cas genome editing identify mutations of ZAK as a cause of limb defects in humans and mice</strong></div><div><em>Genome Res.</em> 2016; <strong>26</strong>:183-191</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/gr.199430.115" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_117_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-84956905665" target="_blank" aria-label="Open Scopus in new tab">Scopus (31)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26755636/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2Fgr.199430.115&amp;pmid=26755636" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td></tr><tr><td>2q33.1-<i>SATB2</i><a id="crosref1880" href="#tbl2fna"><sup>a</sup></a></td><td>➀ Glass syndrome; digit anomalies, sparse hair (<a id="intref0285" href="https://omim.org/entry/612313" target="_blank">612313</a>); ➂ apical ectodermal ridge (<span class="dropBlock reference-citations"><a id="body-ref-sref105" href="#" role="doc-biblioref" data-xml-rid="bib105" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref105" aria-controls="sref105">Sheehan-Rooney et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref105"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">114.</div><div>Sheehan-Rooney, K. ∙ Pálinkášová, B. ∙ Eberhart, J.K. ...</div><div><strong>A cross-species analysis of Satb2 expression suggests deep conservation across vertebrate lineages</strong></div><div><em>Dev. Dyn.</em> 2010; <strong>239</strong>:3481-3491</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/dvdy.22483" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_114_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-78649712329" target="_blank" aria-label="Open Scopus in new tab">Scopus (50)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21089028/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fdvdy.22483&amp;pmid=21089028" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td></tr><tr><td>2q37.3-<i>GPC1</i></td><td>➂ limb bud mesenchyme (chicken) (<span class="dropBlock reference-citations"><a id="body-ref-sref102" href="#" role="doc-biblioref" data-xml-rid="bib102" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref102" aria-controls="sref102">Saad et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref102"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">111.</div><div>Saad, K. ∙ Theis, S. ∙ Otto, A. ...</div><div><strong>Detailed expression profile of the six Glypicans and their modifying enzyme, Notum during chick limb and feather development</strong></div><div><em>Gene.</em> 2017; <strong>610</strong>:71-79</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.gene.2017.02.012" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_111_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-85014296035" target="_blank" aria-label="Open Scopus in new tab">Scopus (3)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28192166/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.gene.2017.02.012&amp;pmid=28192166" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td></tr><tr><td>3p22.3-<i>TRIM71 (LIN41)</i></td><td>➂ distal limb and digit mesenchyme (mouse and chicken) (<span class="dropBlock reference-citations"><a id="body-ref-sref60-1" href="#" role="doc-biblioref" data-xml-rid="bib60" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref60-1" aria-controls="sref60-1">Lancman et&nbsp;al., 2005</a><div class="dropBlock__holder" data-db-target-of="sref60-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">65.</div><div>Lancman, J.J. ∙ Caruccio, N.C. ∙ Harfe, B.D. ...</div><div><strong>Analysis of the regulation of lin-41 during chick and mouse limb development</strong></div><div><em>Dev. Dyn.</em> 2005; <strong>234</strong>:948-960</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/dvdy.20591" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-28444491759" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16245339/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fdvdy.20591&amp;pmid=16245339" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td></tr><tr><td>3p14.3-<i>WNT5A</i><a id="crosref1900" href="#tbl2fna"><sup>a</sup></a></td><td>➀ Robinow syndrome; limb shortening, brachydactyly (<a id="intref0290" href="https://omim.org/entry/180700" target="_blank">180700</a>); ➂ apical ectodermal ridge and progress zone (<span class="dropBlock reference-citations"><a id="body-ref-sref123-1" href="#" role="doc-biblioref" data-xml-rid="bib123" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref123-1" aria-controls="sref123-1">Yamaguchi et&nbsp;al., 1999</a><div class="dropBlock__holder" data-db-target-of="sref123-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">133.</div><div>Yamaguchi, T.P. ∙ Bradley, A. ∙ McMahon, A.P. ...</div><div><strong>A Wnt5a pathway underlies outgrowth of multiple structures in the vertebrate embryo</strong></div><div><em>Development.</em> 1999; <strong>126</strong>:1211-1223</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1242/dev.126.6.1211" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/10021340/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1242%2Fdev.126.6.1211&amp;pmid=10021340" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td></tr><tr><td>3p14.1-<i>ADAMTS9</i></td><td>➁ limb-specific deletion causes syndactyl; ➂ broadly in early limb mesenchyme, subsequently digit perichondrium (<span class="dropBlock reference-citations"><a id="body-ref-sref75" href="#" role="doc-biblioref" data-xml-rid="bib75" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref75" aria-controls="sref75">McCulloch et&nbsp;al., 2009</a><div class="dropBlock__holder" data-db-target-of="sref75"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">82.</div><div>McCulloch, D.R. ∙ Nelson, C.M. ∙ Dixon, L.J. ...</div><div><strong>ADAMTS metalloproteases generate active versican fragments that regulate interdigital web regression</strong></div><div><em>Dev. Cell.</em> 2009; <strong>17</strong>:687-698</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_82_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=10.1016%2Fj.devcel.2009.09.008&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_82_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=10.1016%2Fj.devcel.2009.09.008&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_82_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-71549163138" target="_blank" aria-label="Open Scopus in new tab">Scopus (178)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19922873/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.devcel.2009.09.008&amp;pmid=19922873" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td></tr><tr><td>3q26.2-<i>EVI1</i><a id="crosref1915" href="#tbl2fna"><sup>a</sup></a></td><td>➀ RUSAT2; radioulnar synostosis, digit defects (<a id="intref0295" href="https://omim.org/entry/616738" target="_blank">616738</a>); ➂ limb bud and digit pad mesenchyme (human, <a id="crosref1920" href="#fig3">Figures 3</a>G–3I)</td></tr><tr><td>3q27.1-<i>EPHB3</i></td><td>➂ limb bud epithelium and nonchondrogenic mesenchyme (<span class="dropBlock reference-citations"><a id="body-ref-sref21" href="#" role="doc-biblioref" data-xml-rid="bib21" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref21" aria-controls="sref21">Compagni et&nbsp;al., 2003</a><div class="dropBlock__holder" data-db-target-of="sref21"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">21.</div><div>Compagni, A. ∙ Logan, M. ∙ Klein, R. ...</div><div><strong>Control of skeletal patterning by ephrinB1-EphB interactions</strong></div><div><em>Dev. Cell.</em> 2003; <strong>5</strong>:217-230</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=10.1016%2FS1534-5807%2803%2900198-9&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=10.1016%2FS1534-5807%2803%2900198-9&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-0042442460" target="_blank" aria-label="Open Scopus in new tab">Scopus (188)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/12919674/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS1534-5807%2803%2900198-9&amp;pmid=12919674" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td></tr><tr><td>4p15.31-<i>SLIT2</i></td><td>➂ interdigital mesenchyme and digit lateral margins (<span class="dropBlock reference-citations"><a id="body-ref-sref47" href="#" role="doc-biblioref" data-xml-rid="bib47" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref47" aria-controls="sref47">Holmes et&nbsp;al., 1998</a><div class="dropBlock__holder" data-db-target-of="sref47"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">50.</div><div>Holmes, G.P. ∙ Negus, K. ∙ Burridge, L. ...</div><div><strong>Distinct but overlapping expression patterns of two vertebrate slit homologs implies functional roles in CNS development and organogenesis</strong></div><div><em>Mech. Dev.</em> 1998; <strong>79</strong>:57-72</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/S0925-4773(98)00174-9" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-0032416836" target="_blank" aria-label="Open Scopus in new tab">Scopus (128)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/10349621/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0925-4773%2898%2900174-9&amp;pmid=10349621" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td></tr><tr><td>4p15.2-<i>RBPJ</i><a id="crosref1935" href="#tbl2fna"><sup>a</sup></a></td><td>➀ Adams-Oliver syndrome; scalp and distal limb defects (short distal phalanges) (<a id="intref0300" href="https://omim.org/entry/614814" target="_blank">614814</a>)</td></tr><tr><td>4q28.1-<i>INTU</i><a id="crosref1940" href="#tbl2fna"><sup>a</sup></a></td><td>➀ short-rib thoracic dysplasia 20 with polydactyly (<a id="intref0305" href="https://omim.org/entry/617925" target="_blank">617925</a>)</td></tr><tr><td>5q23.3-<i>ADAMTS19</i></td><td>➂ posterior-proximal limb bud, regulated by SHH signaling (<span class="dropBlock reference-citations"><a id="body-ref-sref63" href="#" role="doc-biblioref" data-xml-rid="bib63" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref63" aria-controls="sref63">Lewandowski et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref63"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">68.</div><div>Lewandowski, J.P. ∙ Du, F. ∙ Zhang, S. ...</div><div><strong>Spatiotemporal regulation of GLI target genes in the mammalian limb bud</strong></div><div><em>Dev. Biol.</em> 2015; <strong>406</strong>:92-103</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.ydbio.2015.07.022" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-84942195571" target="_blank" aria-label="Open Scopus in new tab">Scopus (32)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26238476/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.ydbio.2015.07.022&amp;pmid=26238476" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td></tr><tr><td>6p24.3-<i>TFAP2A</i><a id="crosref1950" href="#tbl2fna"><sup>a</sup></a></td><td>➀ branchiooculofacial syndrome, incompletely penetrant polydactyly (<a id="intref0310" href="https://omim.org/entry/113620" target="_blank">113620</a>); ➂ limb bud ectoderm and distal mesenchyme (<span class="dropBlock reference-citations"><a id="body-ref-sref31" href="#" role="doc-biblioref" data-xml-rid="bib31" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref31" aria-controls="sref31">Feng et&nbsp;al., 2008</a><div class="dropBlock__holder" data-db-target-of="sref31"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">32.</div><div>Feng, W. ∙ Huang, J. ∙ Zhang, J. ...</div><div><strong>Identification and analysis of a conserved Tcfap2a intronic enhancer element required for expression in facial and limb bud mesenchyme</strong></div><div><em>Mol. Cell. Biol.</em> 2008; <strong>28</strong>:315-325</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/MCB.01168-07" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-37549020745" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17984226/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FMCB.01168-07&amp;pmid=17984226" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td></tr><tr><td>6q21-<i>PRDM1</i></td><td>➁ mutants lack posterior digits, whisker development; ➂ limb bud posterior mesenchyme, including zone of polarizing activity (<span class="dropBlock reference-citations"><a id="body-ref-sref100" href="#" role="doc-biblioref" data-xml-rid="bib100" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref100" aria-controls="sref100">Robertson et&nbsp;al., 2007</a><div class="dropBlock__holder" data-db-target-of="sref100"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">109.</div><div>Robertson, E.J. ∙ Charatsi, I. ∙ Joyner, C.J. ...</div><div><strong>Blimp1 regulates development of the posterior forelimb, caudal pharyngeal arches, heart and sensory vibrissae in mice</strong></div><div><em>Development.</em> 2007; <strong>134</strong>:4335-4345</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1242/dev.012047" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_109_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-38349080867" target="_blank" aria-label="Open Scopus in new tab">Scopus (98)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/18039967/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1242%2Fdev.012047&amp;pmid=18039967" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td></tr><tr><td>7p21.2-<i>MEOX2</i></td><td>➁ mutants have selectively reduced limb musculature; ➂ limb bud myoblasts (<span class="dropBlock reference-citations"><a id="body-ref-sref71" href="#" role="doc-biblioref" data-xml-rid="bib71" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref71" aria-controls="sref71">Mankoo et&nbsp;al., 1999</a><div class="dropBlock__holder" data-db-target-of="sref71"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">78.</div><div>Mankoo, B.S. ∙ Collins, N.S. ∙ Ashby, P. ...</div><div><strong>Mox2 is a component of the genetic hierarchy controlling limb muscle development</strong></div><div><em>Nature.</em> 1999; <strong>400</strong>:69-73</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/21892" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_78_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-0033167972" target="_blank" aria-label="Open Scopus in new tab">Scopus (145)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/10403250/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2F21892&amp;pmid=10403250" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td></tr><tr><td>7p14.3-<i>CREB5</i></td><td>➂ interdigital condensing mesenchyme (<span class="dropBlock reference-citations"><a id="body-ref-sref62" href="#" role="doc-biblioref" data-xml-rid="bib62" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref62" aria-controls="sref62">Lehoczky et&nbsp;al., 2004</a><div class="dropBlock__holder" data-db-target-of="sref62"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">67.</div><div>Lehoczky, J.A. ∙ Williams, M.E. ∙ Innis, J.W.</div><div><strong>Conserved expression domains for genes upstream and within the HoxA and HoxD clusters suggests a long-range enhancer existed before cluster duplication</strong></div><div><em>Evol. Dev.</em> 2004; <strong>6</strong>:423-430</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1111/j.1525-142X.2004.04050.x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_67_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-9344246836" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15509224/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1111%2Fj.1525-142X.2004.04050.x&amp;pmid=15509224" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td></tr><tr><td>7p12.3-<i>IGFBP3</i></td><td>➂ interdigital region (<span class="dropBlock reference-citations"><a id="body-ref-sref115" href="#" role="doc-biblioref" data-xml-rid="bib115" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref115" aria-controls="sref115">van Kleffens et&nbsp;al., 1998</a><div class="dropBlock__holder" data-db-target-of="sref115"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">125.</div><div>van Kleffens, M. ∙ Groffen, C. ∙ Rosato, R.R. ...</div><div><strong>mRNA expression patterns of the IGF system during mouse limb bud development, determined by whole mount in&nbsp;situ hybridization</strong></div><div><em>Mol. Cell. Endocrinol.</em> 1998; <strong>138</strong>:151-161</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/S0303-7207(98)00007-0" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_125_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-0032536742" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/9685224/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0303-7207%2898%2900007-0&amp;pmid=9685224" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td></tr><tr><td>8q21.13-<i>ZFHX4</i></td><td>➂ conserved human enhancer drives reporter expression in mouse limb buds (<span class="dropBlock reference-citations"><a id="body-ref-sref3" href="#" role="doc-biblioref" data-xml-rid="bib3" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref3" aria-controls="sref3">Ali et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">3.</div><div>Ali, S. ∙ Amina, B. ∙ Anwar, S. ...</div><div><strong>Genomic features of human limb specific enhancers</strong></div><div><em>Genomics.</em> 2016; <strong>108</strong>:143-150</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.ygeno.2016.08.003" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-84994065473" target="_blank" aria-label="Open Scopus in new tab">Scopus (2)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27580967/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.ygeno.2016.08.003&amp;pmid=27580967" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td></tr><tr><td>8q23.1-<i>RSPO2</i><a id="crosref1985" href="#tbl2fna"><sup>a</sup></a></td><td>➀ tetraamelia syndrome 2; absence of limbs (<a id="intref0315" href="https://omim.org/entry/618021" target="_blank">618021</a>); ➂ apical ectodermal ridge (<span class="dropBlock reference-citations"><a id="body-ref-sref110" href="#" role="doc-biblioref" data-xml-rid="bib110" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref110" aria-controls="sref110">Szenker-Ravi et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref110"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">119.</div><div>Szenker-Ravi, E. ∙ Altunoglu, U. ∙ Leushacke, M. ...</div><div><strong>RSPO2 inhibition of RNF43 and ZNRF3 governs limb development independently of LGR4/5/6</strong></div><div><em>Nature.</em> 2018; <strong>557</strong>:564-569</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-018-0118-y" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_119_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-85047253329" target="_blank" aria-label="Open Scopus in new tab">Scopus (76)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29769720/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-018-0118-y&amp;pmid=29769720" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td></tr><tr><td>8q24.12-<i>HAS2</i></td><td>➁ short limbs with phalanx duplication and misplaced interphalangeal joints; ➂ distal limb bud mesenchyme, regulated by SHH (<span class="dropBlock reference-citations"><a id="body-ref-sref66-1" href="#" role="doc-biblioref" data-xml-rid="bib66" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref66-1" aria-controls="sref66-1">Liu et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref66-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">72.</div><div>Liu, J. ∙ Li, Q. ∙ Kuehn, M.R. ...</div><div><strong>Sonic hedgehog signaling directly targets Hyaluronic Acid Synthase 2, an essential regulator of phalangeal joint patterning</strong></div><div><em>Dev. Biol.</em> 2013; <strong>375</strong>:160-171</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.ydbio.2012.12.018" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-84874251615" target="_blank" aria-label="Open Scopus in new tab">Scopus (17)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23313125/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.ydbio.2012.12.018&amp;pmid=23313125" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td></tr><tr><td>9q22.31-<i>ROR2</i><a id="crosref2000" href="#tbl2fna"><sup>a</sup></a></td><td>➀ Robinow syndrome; limb shortening, including brachydactyly (<a id="intref0320" href="https://omim.org/entry/268310" target="_blank">268310</a>); ➂ distal limb mesenchyme (<span class="dropBlock reference-citations"><a id="body-ref-sref74" href="#" role="doc-biblioref" data-xml-rid="bib74" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref74" aria-controls="sref74">Matsuda et&nbsp;al., 2001</a><div class="dropBlock__holder" data-db-target-of="sref74"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">81.</div><div>Matsuda, T. ∙ Nomi, M. ∙ Ikeya, M. ...</div><div><strong>Expression of the receptor tyrosine kinase genes, Ror1 and Ror2, during mouse development</strong></div><div><em>Mech. Dev.</em> 2001; <strong>105</strong>:153-156</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/S0925-4773(01)00383-5" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-0034948866" target="_blank" aria-label="Open Scopus in new tab">Scopus (111)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11429290/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0925-4773%2801%2900383-5&amp;pmid=11429290" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td></tr><tr><td>12q12-<i>DBX2</i><a id="crosref2010" href="#tbl2fna"><sup>a</sup></a></td><td>➀ deletion at this locus alters hand size, digit morphology, and causes retention of fetal digital pads (<span class="dropBlock reference-citations"><a id="body-ref-sref15" href="#" role="doc-biblioref" data-xml-rid="bib15" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref15" aria-controls="sref15">Carlsen et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref15"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">15.</div><div>Carlsen, E.Ø. ∙ Frengen, E. ∙ Fannemel, M. ...</div><div><strong>Haploinsufficiency of ANO6, NELL2 and DBX2 in a boy with intellectual disability and growth delay</strong></div><div><em>Am. J. Med. Genet. A.</em> 2015; <strong>167A</strong>:1890-1896</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/ajmg.a.37079" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-84937975018" target="_blank" aria-label="Open Scopus in new tab">Scopus (11)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25846056/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fajmg.a.37079&amp;pmid=25846056" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>); ➂ genes at this locus are coordinately expressed in distal limb mesenchyme and embryonic digits (<span class="dropBlock reference-citations"><a id="body-ref-sref11" href="#" role="doc-biblioref" data-xml-rid="bib11" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref11" aria-controls="sref11">Beccari et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref11"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">11.</div><div>Beccari, L. ∙ Jaquier, G. ∙ Lopez-Delisle, L. ...</div><div><strong>Dbx2 regulation in limbs suggests interTAD sharing of enhancers</strong></div><div><em>Dev. Dyn.</em> 2021; <strong>250</strong>:1280-1299</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/dvdy.303" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-85101792011" target="_blank" aria-label="Open Scopus in new tab">Scopus (4)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33497014/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fdvdy.303&amp;pmid=33497014" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td></tr><tr><td>12q12-<i>NELL2</i><a id="crosref2025" href="#tbl2fna"><sup>a</sup></a></td><td>➂ distal limb and digit mesenchyme (mouse and chicken) (<span class="dropBlock reference-citations"><a id="body-ref-sref60-2" href="#" role="doc-biblioref" data-xml-rid="bib60" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref60-2" aria-controls="sref60-2">Lancman et&nbsp;al., 2005</a><div class="dropBlock__holder" data-db-target-of="sref60-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">65.</div><div>Lancman, J.J. ∙ Caruccio, N.C. ∙ Harfe, B.D. ...</div><div><strong>Analysis of the regulation of lin-41 during chick and mouse limb development</strong></div><div><em>Dev. Dyn.</em> 2005; <strong>234</strong>:948-960</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/dvdy.20591" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-28444491759" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16245339/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fdvdy.20591&amp;pmid=16245339" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td></tr><tr><td>12q24.21-<i>TBX3</i><a id="crosref2035" href="#tbl2fna"><sup>a</sup></a></td><td>➀ ulnar-mammary syndrome; posterior digits reduced or absent (<a id="intref0325" href="https://omim.org/entry/181450" target="_blank">181450</a>); ➂ anterior and posterior mesenchyme of embryonic limb bud and apical ectodermal ridge (<span class="dropBlock reference-citations"><a id="body-ref-sref39" href="#" role="doc-biblioref" data-xml-rid="bib39" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref39" aria-controls="sref39">Gibson-Brown et&nbsp;al., 1996</a><div class="dropBlock__holder" data-db-target-of="sref39"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">40.</div><div>Gibson-Brown, J.J. ∙ Agulnik, S.I. ∙ Chapman, D.L. ...</div><div><strong>Evidence of a role for T-box genes in the evolution of limb morphogenesis and the specification of forelimb/hindlimb identity</strong></div><div><em>Mech. Dev.</em> 1996; <strong>56</strong>:93-101</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/0925-4773(96)00514-X" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_40_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-0029977495" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/8798150/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2F0925-4773%2896%2900514-X&amp;pmid=8798150" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td></tr><tr><td>13q12.11-<i>FGF9</i><a id="crosref2045" href="#tbl2fna"><sup>a</sup></a></td><td>➀ multiple synostoses syndrome 3; impaired interphalanx joint formation, broadened thumbs (<a id="intref0330" href="https://omim.org/entry/612961" target="_blank">612961</a>); ➂ apical ectodermal ridge (<span class="dropBlock reference-citations"><a id="body-ref-sref72" href="#" role="doc-biblioref" data-xml-rid="bib72" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref72" aria-controls="sref72">Mariani et&nbsp;al., 2008</a><div class="dropBlock__holder" data-db-target-of="sref72"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">79.</div><div>Mariani, F.V. ∙ Ahn, C.P. ∙ Martin, G.R.</div><div><strong>Genetic evidence that FGFs have an instructive role in limb proximal-distal patterning</strong></div><div><em>Nature.</em> 2008; <strong>453</strong>:401-405</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature06876" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_79_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-43749110077" target="_blank" aria-label="Open Scopus in new tab">Scopus (253)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/18449196/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature06876&amp;pmid=18449196" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td></tr><tr><td>16q12.2-<i>RPGRIP1L</i><a id="crosref2055" href="#tbl2fna"><sup>a</sup></a></td><td>➀ Meckel syndrome; polydactyly (<a id="intref0335" href="https://omim.org/entry/611561" target="_blank">611561</a>); ➂ protein located at primary cilium of embryonic limb mesenchymal cells; regulates SHH signaling (<span class="dropBlock reference-citations"><a id="body-ref-sref38" href="#" role="doc-biblioref" data-xml-rid="bib38" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref38" aria-controls="sref38">Gerhardt et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref38"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">39.</div><div>Gerhardt, C. ∙ Lier, J.M. ∙ Burmühl, S. ...</div><div><strong>The transition zone protein Rpgrip1l regulates proteasomal activity at the primary cilium</strong></div><div><em>J.&nbsp;Cell Biol.</em> 2015; <strong>210</strong>:115-133</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1083/jcb.201408060" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-84957560272" target="_blank" aria-label="Open Scopus in new tab">Scopus (49)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26150391/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1083%2Fjcb.201408060&amp;pmid=26150391" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td></tr><tr><td>16q12.2-<i>FTO</i><a id="crosref2065" href="#tbl2fnc"><sup>c</sup></a></td><td>➀ growth retardation, developmental delay, and facial dysmorphism; brachydactyly and cutis marmorata (<a id="intref0340" href="https://omim.org/entry/612938" target="_blank">612938</a>)</td></tr><tr><td>16q12.2-<i>IRX3</i></td><td>➁ smaller limb when mutation combined with Irx5 mutation; ➂ proximal-anterior limb bud, interacts with SHH (<span class="dropBlock reference-citations"><a id="body-ref-sref65-1" href="#" role="doc-biblioref" data-xml-rid="bib65" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref65-1" aria-controls="sref65-1">Li et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref65-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">70.</div><div>Li, D. ∙ Sakuma, R. ∙ Vakili, N.A. ...</div><div><strong>Formation of proximal and anterior limb skeleton requires early function of Irx3 and Irx5 and is negatively regulated by Shh signaling</strong></div><div><em>Dev. Cell.</em> 2014; <strong>29</strong>:233-240</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=10.1016%2Fj.devcel.2014.03.001&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=10.1016%2Fj.devcel.2014.03.001&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-84899634426" target="_blank" aria-label="Open Scopus in new tab">Scopus (67)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24726282/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.devcel.2014.03.001&amp;pmid=24726282" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td></tr><tr><td>18q12.1-<i>MAPRE2</i><a id="crosref2075" href="#tbl2fnb"><sup>b</sup></a></td><td>➀ congenital symmetric circumferential skin creases; excess skin leading to ringed creases, principally on limbs (<a id="intref0345" href="https://omim.org/entry/616734" target="_blank">616734</a>)</td></tr><tr><td>18q21.2-<i>TCF4</i><a id="crosref2095" href="#tbl2fna"><sup>a</sup></a></td><td>➀ Pitt-Hopkins syndrome; persistent fetal digital pads (<a id="intref0350" href="https://omim.org/entry/610954" target="_blank">610954</a>); ➂ peridigital mesenchyme at distal digit tips (<span class="dropBlock reference-citations"><a id="body-ref-sref18" href="#" role="doc-biblioref" data-xml-rid="bib18" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref18" aria-controls="sref18">Cho and Dressler, 1998</a><div class="dropBlock__holder" data-db-target-of="sref18"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">18.</div><div>Cho, E.A. ∙ Dressler, G.R.</div><div><strong>TCF-4 binds beta-catenin and is expressed in distinct regions of the embryonic brain and limbs</strong></div><div><em>Mech. Dev.</em> 1998; <strong>77</strong>:9-18</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/S0925-4773(98)00131-2" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-0032170317" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/9784592/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0925-4773%2898%2900131-2&amp;pmid=9784592" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td></tr><tr><td>18q23-<i>SALL3</i></td><td>➁ lack of digit development when mutated with SALL1; ➂ distal posterior mesenchyme, regulated by SHH (<span class="dropBlock reference-citations"><a id="body-ref-sref55-1" href="#" role="doc-biblioref" data-xml-rid="bib55" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref55-1" aria-controls="sref55-1">Kawakami et&nbsp;al., 2009</a><div class="dropBlock__holder" data-db-target-of="sref55-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">60.</div><div>Kawakami, Y. ∙ Uchiyama, Y. ∙ Rodriguez Esteban, C. ...</div><div><strong>Sall genes regulate region-specific morphogenesis in the mouse limb by modulating Hox activities</strong></div><div><em>Development.</em> 2009; <strong>136</strong>:585-594</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1242/dev.027748" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-64549145826" target="_blank" aria-label="Open Scopus in new tab">Scopus (58)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19168674/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1242%2Fdev.027748&amp;pmid=19168674" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td></tr><tr><td>22q13.31-<i>CELSR1</i></td><td>➂ early limb bud with distal bias (<span class="dropBlock reference-citations"><a id="body-ref-sref107" href="#" role="doc-biblioref" data-xml-rid="bib107" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref107" aria-controls="sref107">Shima et&nbsp;al., 2002</a><div class="dropBlock__holder" data-db-target-of="sref107"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">116.</div><div>Shima, Y. ∙ Copeland, N.G. ∙ Gilbert, D.J. ...</div><div><strong>Differential expression of the seven-pass transmembrane cadherin genes Celsr1-3 and distribution of the Celsr2 protein during mouse development</strong></div><div><em>Dev. Dyn.</em> 2002; <strong>223</strong>:321-332</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/dvdy.10054" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_116_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-0036198940" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11891983/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fdvdy.10054&amp;pmid=11891983" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td></tr></tbody></table></div><figcaption><div class="caption"><span class="heading">Table 2</span><div id="tspara0020" role="paragraph">Functional annotation for notable genes (subset)</div></div><div class="notes"><div role="doc-footnote" data-has="label"><div class="label">a</div><div id="tbl2fna"><div id="ntpara0065" role="paragraph">Abnormalities on limb phenotype when gene mutated in human.</div></div></div><div role="doc-footnote" data-has="label"><div class="label">b</div><div id="tbl2fnb"><div id="ntpara0070" role="paragraph">Abnormalities on skin or skin appendage phenotypes when gene mutated in human.</div></div></div><div role="doc-footnote" data-has="label"><div class="label">c</div><div id="tbl2fnc"><div id="ntpara0075" role="paragraph">Abnormalities on both skin and limb phenotypes when gene mutated in human.</div></div></div></div><div class="core-table-tools"><ul><li><a class="core-table-show-action" href="/action/showFullTableHTML?isHtml=true&amp;tableId=tbl2&amp;pii=S0092-8674%2821%2901446-X" target="_blank">Open table in a new tab</a></li></ul></div></figcaption></figure></div><div id="p0150" role="paragraph">Proximal-distal limb growth is driven by proliferation of distal mesenchyme in response to signals from its overlying apical ectodermal ridge (AER) (<span class="dropBlock reference-citations"><a id="body-ref-sref112-2" href="#" role="doc-biblioref" data-xml-rid="bib112" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref112-2" aria-controls="sref112-2">Towers and Tickle, 2009</a><div class="dropBlock__holder" data-db-target-of="sref112-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">122.</div><div>Towers, M. ∙ Tickle, C.</div><div><strong>Growing models of vertebrate limb development</strong></div><div><em>Development.</em> 2009; <strong>136</strong>:179-190</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1242/dev.024158" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_122_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-62349111082" target="_blank" aria-label="Open Scopus in new tab">Scopus (133)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19103802/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1242%2Fdev.024158&amp;pmid=19103802" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Apically produced WNT5A, and its receptor ROR2 (<span class="dropBlock reference-citations"><a id="body-ref-sref80" href="#" role="doc-biblioref" data-xml-rid="bib80" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref80" aria-controls="sref80">Mikels and Nusse, 2006</a><div class="dropBlock__holder" data-db-target-of="sref80"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">87.</div><div>Mikels, A.J. ∙ Nusse, R.</div><div><strong>Purified Wnt5a protein activates or inhibits beta-catenin-TCF signaling depending on receptor context</strong></div><div><em>PLoS Biol.</em> 2006; <strong>4</strong>:e115</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pbio.0040115" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_87_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-33646241997" target="_blank" aria-label="Open Scopus in new tab">Scopus (961)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16602827/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pbio.0040115&amp;pmid=16602827" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), RSPO2, and the downstream regulator TCF4, play roles in β-catenin transcriptional function, which is finely balanced to achieve appropriate AER maintenance and limb outgrowth (<span class="dropBlock reference-citations"><a id="body-ref-sref44" href="#" role="doc-biblioref" data-xml-rid="bib44" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref44" aria-controls="sref44">Hill et&nbsp;al., 2006</a><div class="dropBlock__holder" data-db-target-of="sref44"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">47.</div><div>Hill, T.P. ∙ Taketo, M.M. ∙ Birchmeier, W. ...</div><div><strong>Multiple roles of mesenchymal beta-catenin during murine limb patterning</strong></div><div><em>Development.</em> 2006; <strong>133</strong>:1219-1229</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1242/dev.02298" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-33646130168" target="_blank" aria-label="Open Scopus in new tab">Scopus (104)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16495310/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1242%2Fdev.02298&amp;pmid=16495310" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Our network analysis of fingerprint pattern-associated genes extends this canonical WNT/β-catenin cluster to the planar cell polarity pathway, arising from the dual roles of WNT5A and ROR2, and connecting to INTU and CELSR1 as cytoplasmic effector and plasma membrane cell polarity factors. Planar cell polarity is crucial for coordinating directed limb outgrowth (<span class="dropBlock reference-citations"><a id="body-ref-sref36" href="#" role="doc-biblioref" data-xml-rid="bib36" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref36" aria-controls="sref36">Gao and Yang, 2013</a><div class="dropBlock__holder" data-db-target-of="sref36"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">37.</div><div>Gao, B. ∙ Yang, Y.</div><div><strong>Planar cell polarity in vertebrate limb morphogenesis</strong></div><div><em>Curr. Opin. Genet. Dev.</em> 2013; <strong>23</strong>:438-444</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.gde.2013.05.003" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-84883455409" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23747034/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.gde.2013.05.003&amp;pmid=23747034" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and, in particular, regulating the formation of distal skeletal elements, consistent with the brachydactyly caused by <i>ROR2</i> or <i>WNT5A</i> mutation in Robinow syndrome and polydactyly caused by <i>INTU</i> mutation (<a id="crosref1050" href="#tbl2">Table 2</a>).</div><div id="p0155" role="paragraph">A second node of network connectivity centers on IRX3, expressed in the proximal limb bud and antagonistically regulated by SHH, a key morphogen imparting anterior-posterior polarity to the limb (<span class="dropBlock reference-citations"><a id="body-ref-sref65-2" href="#" role="doc-biblioref" data-xml-rid="bib65" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref65-2" aria-controls="sref65-2">Li et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref65-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">70.</div><div>Li, D. ∙ Sakuma, R. ∙ Vakili, N.A. ...</div><div><strong>Formation of proximal and anterior limb skeleton requires early function of Irx3 and Irx5 and is negatively regulated by Shh signaling</strong></div><div><em>Dev. Cell.</em> 2014; <strong>29</strong>:233-240</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=10.1016%2Fj.devcel.2014.03.001&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=10.1016%2Fj.devcel.2014.03.001&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-84899634426" target="_blank" aria-label="Open Scopus in new tab">Scopus (67)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24726282/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.devcel.2014.03.001&amp;pmid=24726282" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). SHH, emanating from the zone of polarizing activity (ZPA) located in the posterior autopod mesenchyme, coordinates growth and digit identity across the handplate, resulting in altered digit number when SHH signaling is modulated (<span class="dropBlock reference-citations"><a id="body-ref-sref112-3" href="#" role="doc-biblioref" data-xml-rid="bib112" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref112-3" aria-controls="sref112-3">Towers and Tickle, 2009</a><div class="dropBlock__holder" data-db-target-of="sref112-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">122.</div><div>Towers, M. ∙ Tickle, C.</div><div><strong>Growing models of vertebrate limb development</strong></div><div><em>Development.</em> 2009; <strong>136</strong>:179-190</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1242/dev.024158" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_122_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-62349111082" target="_blank" aria-label="Open Scopus in new tab">Scopus (133)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19103802/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1242%2Fdev.024158&amp;pmid=19103802" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). RPGRIP1L and ADAMTS9 regulate SHH signal reception by contributing to the formation and functioning of cilia, which serve as a signal-receiving projection from the cell surface, with mutations in <i>RPGRIP1L</i> causing formation of extra digits (polydactyly) (<span class="dropBlock reference-citations"><a id="body-ref-sref7" href="#" role="doc-biblioref" data-xml-rid="bib7" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref7" aria-controls="sref7">Arts et&nbsp;al., 2007</a><div class="dropBlock__holder" data-db-target-of="sref7"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">7.</div><div>Arts, H.H. ∙ Doherty, D. ∙ van Beersum, S.E. ...</div><div><strong>Mutations in the gene encoding the basal body protein RPGRIP1L, a nephrocystin-4 interactor, cause Joubert syndrome</strong></div><div><em>Nat. Genet.</em> 2007; <strong>39</strong>:882-888</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng2069" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-34347356500" target="_blank" aria-label="Open Scopus in new tab">Scopus (245)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17558407/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng2069&amp;pmid=17558407" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref25" href="#" role="doc-biblioref" data-xml-rid="bib25" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref25" aria-controls="sref25">Delous et&nbsp;al., 2007</a><div class="dropBlock__holder" data-db-target-of="sref25"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">26.</div><div>Delous, M. ∙ Baala, L. ∙ Salomon, R. ...</div><div><strong>The ciliary gene RPGRIP1L is mutated in cerebello-oculo-renal syndrome (Joubert syndrome type B) and Meckel syndrome</strong></div><div><em>Nat. Genet.</em> 2007; <strong>39</strong>:875-881</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng2039" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_26_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-34347324031" target="_blank" aria-label="Open Scopus in new tab">Scopus (341)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17558409/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng2039&amp;pmid=17558409" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref127" href="#" role="doc-biblioref" data-xml-rid="bib127" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref127" aria-controls="sref127">Zeng et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref127"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">137.</div><div>Zeng, H. ∙ Hoover, A.N. ∙ Liu, A.</div><div><strong>PCP effector gene Inturned is an important regulator of cilia formation and embryonic development in mammals</strong></div><div><em>Dev. Biol.</em> 2010; <strong>339</strong>:418-428</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.ydbio.2010.01.003" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_137_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-77149146791" target="_blank" aria-label="Open Scopus in new tab">Scopus (73)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20067783/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.ydbio.2010.01.003&amp;pmid=20067783" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), whereas <i>ADAMTS9</i> mutation causes fusion of adjacent digits (syndactyly) (<span class="dropBlock reference-citations"><a id="body-ref-sref28" href="#" role="doc-biblioref" data-xml-rid="bib28" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref28" aria-controls="sref28">Dubail et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref28"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">29.</div><div>Dubail, J. ∙ Aramaki-Hattori, N. ∙ Bader, H.L. ...</div><div><strong>A new Adamts9 conditional mouse allele identifies its non-redundant role in interdigital web regression</strong></div><div><em>Genesis.</em> 2014; <strong>52</strong>:702-712</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/dvg.22784" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24753090/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fdvg.22784&amp;pmid=24753090" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref85" href="#" role="doc-biblioref" data-xml-rid="bib85" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref85" aria-controls="sref85">Nandadasa et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref85"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">92.</div><div>Nandadasa, S. ∙ Kraft, C.M. ∙ Wang, L.W. ...</div><div><strong>Secreted metalloproteases ADAMTS9 and ADAMTS20 have a non-canonical role in ciliary vesicle growth during ciliogenesis</strong></div><div><em>Nat. Commun.</em> 2019; <strong>10</strong>:953</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41467-019-08520-7" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_92_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-85062261861" target="_blank" aria-label="Open Scopus in new tab">Scopus (27)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30814516/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41467-019-08520-7&amp;pmid=30814516" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). SHH signaling regulates the expression of other fingerprint pattern-associated genes, including the <i>HAS2</i> (<span class="dropBlock reference-citations"><a id="body-ref-sref66-2" href="#" role="doc-biblioref" data-xml-rid="bib66" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref66-2" aria-controls="sref66-2">Liu et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref66-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">72.</div><div>Liu, J. ∙ Li, Q. ∙ Kuehn, M.R. ...</div><div><strong>Sonic hedgehog signaling directly targets Hyaluronic Acid Synthase 2, an essential regulator of phalangeal joint patterning</strong></div><div><em>Dev. Biol.</em> 2013; <strong>375</strong>:160-171</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.ydbio.2012.12.018" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-84874251615" target="_blank" aria-label="Open Scopus in new tab">Scopus (17)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23313125/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.ydbio.2012.12.018&amp;pmid=23313125" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), <i>SALL3</i> (<span class="dropBlock reference-citations"><a id="body-ref-sref55-2" href="#" role="doc-biblioref" data-xml-rid="bib55" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref55-2" aria-controls="sref55-2">Kawakami et&nbsp;al., 2009</a><div class="dropBlock__holder" data-db-target-of="sref55-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">60.</div><div>Kawakami, Y. ∙ Uchiyama, Y. ∙ Rodriguez Esteban, C. ...</div><div><strong>Sall genes regulate region-specific morphogenesis in the mouse limb by modulating Hox activities</strong></div><div><em>Development.</em> 2009; <strong>136</strong>:585-594</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1242/dev.027748" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-64549145826" target="_blank" aria-label="Open Scopus in new tab">Scopus (58)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19168674/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1242%2Fdev.027748&amp;pmid=19168674" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), and <i>DBX2</i> (<span class="dropBlock reference-citations"><a id="body-ref-sref94" href="#" role="doc-biblioref" data-xml-rid="bib94" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref94" aria-controls="sref94">Pierani et&nbsp;al., 1999</a><div class="dropBlock__holder" data-db-target-of="sref94"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">101.</div><div>Pierani, A. ∙ Brenner-Morton, S. ∙ Chiang, C. ...</div><div><strong>A sonic hedgehog-independent, retinoid-activated pathway of neurogenesis in the ventral spinal cord</strong></div><div><em>Cell.</em> 1999; <strong>97</strong>:903-915</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_101_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=10.1016%2FS0092-8674%2800%2980802-8&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_101_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=10.1016%2FS0092-8674%2800%2980802-8&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_101_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-0033603209" target="_blank" aria-label="Open Scopus in new tab">Scopus (279)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/10399918/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0092-8674%2800%2980802-8&amp;pmid=10399918" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) genes, further implicating the embryonic limb’s anterior-posterior specification system in fingerprint type determination.</div><div id="p0160" role="paragraph">The influence of both the anterior-posterior and the proximal-distal limb development systems on dermatoglyph type is also supported by our assessment of their correlation with hand and digit lengths. The proximal-distal length proportions of hand, digit, and phalanx are established during the fetal period, at the stage of fingerprint determination (<span class="dropBlock reference-citations"><a id="body-ref-sref41" href="#" role="doc-biblioref" data-xml-rid="bib41" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref41" aria-controls="sref41">Hamrick, 2001</a><div class="dropBlock__holder" data-db-target-of="sref41"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">43.</div><div>Hamrick, M.W.</div><div><strong>Primate origins: evolutionary change in digital ray patterning and segmentation</strong></div><div><em>J.&nbsp;Hum. Evol.</em> 2001; <strong>40</strong>:339-351</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1006/jhev.2001.0467" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-0035074837" target="_blank" aria-label="Open Scopus in new tab">Scopus (72)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11312586/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1006%2Fjhev.2001.0467&amp;pmid=11312586" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref97-1" href="#" role="doc-biblioref" data-xml-rid="bib97" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref97-1" aria-controls="sref97-1">Rao et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref97-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">106.</div><div>Rao, R. ∙ Gornbein, J. ∙ Afshar, Y. ...</div><div><strong>A new biometric: In utero growth curves for metacarpal and phalangeal lengths reveal an embryonic patterning ratio</strong></div><div><em>Prenat. Diagn.</em> 2019; <strong>39</strong>:200-208</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/pd.5397" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_106_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-85061354603" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30488473/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fpd.5397&amp;pmid=30488473" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). We found the strongest phenotypic associations between fingerprint type and the proportions of the anterior-most and posterior-most fingers, substantiating a role for the anterior-posterior system in their selection. Moreover, 3p21.1-<i>WNT5A</i> (p&nbsp;= 6.93&nbsp;× 10<sup>−5</sup>) and 18q23-<i>SALL3</i> (p&nbsp;= 5.07&nbsp;× 10<sup>−4</sup>) loci, as well as 3q26.2<i>-EVI1</i> (p&nbsp;= 7.62&nbsp;× 10<sup>−3</sup>) which was similar to the mouse finding (<a id="crosref1115" href="#figs3">Figure&nbsp;S3</a>D), were associated with both fingerprint type and length of digit 5 (<a id="crosref1120" href="#mmc5">Table S5</a>). This is consistent with demonstrated roles for SALL3 as a key regulator of autopod development (<span class="dropBlock reference-citations"><a id="body-ref-sref55-3" href="#" role="doc-biblioref" data-xml-rid="bib55" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref55-3" aria-controls="sref55-3">Kawakami et&nbsp;al., 2009</a><div class="dropBlock__holder" data-db-target-of="sref55-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">60.</div><div>Kawakami, Y. ∙ Uchiyama, Y. ∙ Rodriguez Esteban, C. ...</div><div><strong>Sall genes regulate region-specific morphogenesis in the mouse limb by modulating Hox activities</strong></div><div><em>Development.</em> 2009; <strong>136</strong>:585-594</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1242/dev.027748" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-64549145826" target="_blank" aria-label="Open Scopus in new tab">Scopus (58)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19168674/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1242%2Fdev.027748&amp;pmid=19168674" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and WNT5A promoting distal limb and digit outgrowth during embryogenesis (<span class="dropBlock reference-citations"><a id="body-ref-sref123-2" href="#" role="doc-biblioref" data-xml-rid="bib123" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref123-2" aria-controls="sref123-2">Yamaguchi et&nbsp;al., 1999</a><div class="dropBlock__holder" data-db-target-of="sref123-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">133.</div><div>Yamaguchi, T.P. ∙ Bradley, A. ∙ McMahon, A.P. ...</div><div><strong>A Wnt5a pathway underlies outgrowth of multiple structures in the vertebrate embryo</strong></div><div><em>Development.</em> 1999; <strong>126</strong>:1211-1223</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1242/dev.126.6.1211" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/10021340/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1242%2Fdev.126.6.1211&amp;pmid=10021340" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Unlike the length proportions of the mature hand, which reflect their configuration from the stage at which fingerprint patterns were being established (<span class="dropBlock reference-citations"><a id="body-ref-sref97-2" href="#" role="doc-biblioref" data-xml-rid="bib97" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref97-2" aria-controls="sref97-2">Rao et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref97-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">106.</div><div>Rao, R. ∙ Gornbein, J. ∙ Afshar, Y. ...</div><div><strong>A new biometric: In utero growth curves for metacarpal and phalangeal lengths reveal an embryonic patterning ratio</strong></div><div><em>Prenat. Diagn.</em> 2019; <strong>39</strong>:200-208</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/pd.5397" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_106_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-85061354603" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30488473/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fpd.5397&amp;pmid=30488473" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), other embryonic structures, such as the volar pads, are transient and their variation is unlikely to be detectable in adult limb proportions. In conclusion, the limb development genes uncovered in this study and the correlation of hand proportions with dermatoglyph types demonstrate the key role of embryonic limb growth processes in defining the intricate surface patterning of the human fingerprint.</div><section id="sec-3-1"><h3>Limitations of the study</h3><div id="p0165" role="paragraph">We performed functional analysis on the most prominent signal of all for the composite phenotype, adjacent to <i>EVI1</i>, finding that the best supported SNP at this locus lies within an enhancer, with its alternate alleles displaying different regulatory activities. We mapped enhancer activities from functional datasets using 12 epithelial or mesenchymal cell types similar to those of the developing limb (see <a id="crosref1140" href="#sec-4">STAR Methods</a>). However, this work could be extended and supported by generating functional datasets for limb cells at various developmental stages. Because the entire region of the 3q26.2 signal covering all the associated SNPs appeared in the credible sets using various methods, this region (chromosome 3 [chr3]:168448858-168948263) is in the same TAD as the <i>EVI1</i> gene (chr3:168600000-169520000) (<a id="crosref1145" href="#figs1">Figure&nbsp;S1</a>A), and <i>EVI1</i> is strongly supported as the most likely causal gene. For our luciferase reporter assays, we used the HEK293T cell line, widely used to assay enhancer activity and for other applications, because they are easy to transfect and handle in the laboratory. In addition, the HEK293T cell line is used here because mesenchymal cell lines derived from early limb formation stages do not exist. From the results of luciferase reporter assays, we verified the regulatory effect of SNP rs7646897 on the <i>EVI1</i> gene. Although we have ruled out the effect of this SNP on the expression of the two adjacent genes (<i>GOLIM4</i> and <i>TERC</i>), it is possible that SNP rs7646897 could potentially modulate the expression of other genes more distally. In addition, the present study lacks sufficient evidence to determine whether rs7623083 has a modulating effect on <i>EVI1</i> expression. Two experiments showed an effect and three did not. Furthermore, the plasmids containing the variants (rs7646897 or rs7623083) had repressing, activating or neutral effects on enhancer activity in different experimental trials, indicating significant variation in the reporter assay. Understanding the relative importance of these two closely linked SNPs on <i>EVI1</i> expression and dermatoglyph phenotype, and the detailed regulatory mechanisms at play, will thus require further experimental studies.</div></section></section><section id="sec-4"><h2>STAR★Methods</h2><section id="sec-4-1"><h3>Key resources table</h3><div id="p0170" role="paragraph"><div class="figure-wrap"><figure id="undtbl1" class="table" data-figureviewer-ignore="true"><div class="table-wrap"><table data-figureviewer-ignore="true" tabindex="0"><thead><tr class="rowsep"><th>REAGENT or RESOURCE</th><th>SOURCE</th><th>IDENTIFIER</th></tr></thead><tbody><tr class="rowsep"><td colspan="3"><b>Antibodies</b></td></tr><tr><td>Evi-1 (C50E12), Rabbit mAb (1:200)</td><td>Cell Signaling Technologies</td><td>Cat#2593, RRID:<a id="interref355" href="https://antibodyregistry.org/AB_2184098" target="_blank">AB_2184098</a></td></tr><tr><td>Anti-Cytokeratin 14 antibody [LL002], Mouse mAb (1:200)</td><td>Abcam</td><td>Cat# ab7800, RRID:<a id="interref360" href="https://antibodyregistry.org/AB_306091" target="_blank">AB_306091</a></td></tr><tr><td>Anti-α E-catenin antibody (G-11), Mouse mAb (1:100)</td><td>Santa Cruz Biotechnology</td><td>Cat# sc9988, RRID:<a id="interref365" href="https://antibodyregistry.org/AB_626805" target="_blank">AB_626805</a></td></tr><tr><td>Recombinant Anti-Ki67 antibody [SP6], Rabbit mAb (1:200)</td><td>Abcam</td><td>Cat# ab16667, RRID:<a id="interref370" href="https://antibodyregistry.org/AB_302459" target="_blank">AB_302459</a></td></tr><tr><td>Goat anti-Mouse IgG (H+L) Highly Cross-Adsorbed Secondary Antibody, Alexa Fluor 488 (1:500)</td><td>Thermo Fisher Scientific</td><td>Cat# A-11029, RRID:<a id="interref375" href="https://antibodyregistry.org/AB_2534088" target="_blank">AB_2534088</a></td></tr><tr><td>Goat anti-Rabbit IgG (H+L) Highly Cross-Adsorbed Secondary Antibody, Alexa Fluor 546 (1:500)</td><td>Thermo Fisher Scientific</td><td>Cat#A-11035, RRID:<a id="interref380" href="https://antibodyregistry.org/AB_2534093" target="_blank">AB_2534093</a></td></tr><tr class="rowsep"><td>Anti-Digoxigenin-AP, Fab fragments from sheep (1:2000)</td><td>Sigma Aldrich</td><td>Cat# 11093274910, RRID:<a id="interref385" href="https://antibodyregistry.org/AB_2734716" target="_blank">AB_2734716</a></td></tr><tr class="rowsep"><td colspan="3"><b>Biological samples</b></td></tr><tr class="rowsep"><td>Human fetal tissue – digit and embryo samples</td><td>Royal Infirmary of Edinburgh (see <a id="crosref1150" href="#sec-4">STAR Methods</a> for details)</td><td>N/A</td></tr><tr class="rowsep"><td colspan="3"><b>Chemicals, peptides, and recombinant proteins</b></td></tr><tr><td>4’,6-Diamidino-2-phenylindole dihydrochloride (DAPI)</td><td>Sigma Aldrich</td><td>Cat# D9542</td></tr><tr><td>FastStart Universal Sybr Green Master (Rox)</td><td>Sigma Aldrich</td><td>Cat# 4913850001</td></tr><tr><td>SuperScript III Reverse Transcriptase</td><td>Thermo Fisher Scientific</td><td>Cat# 18080093</td></tr><tr><td>RNaseOUT</td><td>Thermo Fisher Scientific</td><td>Cat# 10777019</td></tr><tr><td>Nitrotetrazolium Blue Chloride</td><td>Sigma Aldrich</td><td>Cat# N6876</td></tr><tr><td>5-Bromo-4-chloro-3-indolyl phosphate p-toludine salt</td><td>Sigma Aldrich</td><td>Cat# B8503</td></tr><tr><td>TrueBlack</td><td>Biotium</td><td>Cat# 23007</td></tr><tr><td>RNAscope multiplex fluorescent reagent kit V2</td><td>ACD bio</td><td>Cat# 323100</td></tr><tr><td>Opal 570 fluorophore (1:1500)</td><td>Akoya biosciences</td><td>Cat# FP1488001KT</td></tr><tr class="rowsep"><td>Opal 690 fluorophore (1:1500)</td><td>Akoya biosciences</td><td>Cat# FP1497001KT</td></tr><tr class="rowsep"><td colspan="3"><b>Deposited data</b></td></tr><tr><td>Raw and analyzed data: GWAS data from Chinese cohorts</td><td>This paper</td><td>NODE: OEP000198 <a id="interref0390" href="http://www.biosino.org/node/project/detail/OEP000198" target="_blank">http://www.biosino.org/node/project/detail/OEP000198</a></td></tr><tr><td>Raw and analyzed data: GWAS data from European cohorts</td><td>This paper</td><td>NODE: OEP000198 <a id="interref0395" href="http://www.biosino.org/node/project/detail/OEP000198" target="_blank">http://www.biosino.org/node/project/detail/OEP000198</a></td></tr><tr><td>Pittsburgh Orofacial Cleft Study</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref120-1" href="#" role="doc-biblioref" data-xml-rid="bib120" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref120-1" aria-controls="sref120-1">Weinberg et&nbsp;al., 2006</a><div class="dropBlock__holder" data-db-target-of="sref120-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">130.</div><div>Weinberg, S.M. ∙ Neiswanger, K. ∙ Martin, R.A. ...</div><div><strong>The Pittsburgh Oral-Facial Cleft study: expanding the cleft phenotype. Background and justification</strong></div><div><em>Cleft Palate Craniofac. J.</em> 2006; <strong>43</strong>:7-20</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1597/04-122R1.1" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_130_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-32044438969" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16405378/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1597%2F04-122R1.1&amp;pmid=16405378" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0015" href="https://www.dental.pitt.edu/pittsburgh-orofacial-cleft-studies" target="_blank">https://www.dental.pitt.edu/pittsburgh-orofacial-cleft-studies</a></td></tr><tr><td>HapMap Phase 1-3</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref4-1" href="#" role="doc-biblioref" data-xml-rid="bib4" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref4-1" aria-controls="sref4-1">Altshuler et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref4-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">4.</div><div>Altshuler, D.M. ∙ Gibbs, R.A. ∙ Peltonen, L. ..., International HapMap 3 Consortium</div><div><strong>Integrating common and rare genetic variation in diverse human populations</strong></div><div><em>Nature.</em> 2010; <strong>467</strong>:52-58</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature09298" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-77956331627" target="_blank" aria-label="Open Scopus in new tab">Scopus (1889)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20811451/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature09298&amp;pmid=20811451" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0020" href="https://www.genome.gov/10001688/international-hapmap-project" target="_blank">https://www.genome.gov/10001688/international-hapmap-project</a></td></tr><tr><td>1000 Genomes Project Phase 3</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref4-2" href="#" role="doc-biblioref" data-xml-rid="bib4" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref4-2" aria-controls="sref4-2">Altshuler et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref4-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">4.</div><div>Altshuler, D.M. ∙ Gibbs, R.A. ∙ Peltonen, L. ..., International HapMap 3 Consortium</div><div><strong>Integrating common and rare genetic variation in diverse human populations</strong></div><div><em>Nature.</em> 2010; <strong>467</strong>:52-58</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature09298" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-77956331627" target="_blank" aria-label="Open Scopus in new tab">Scopus (1889)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20811451/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature09298&amp;pmid=20811451" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0025" href="https://www.internationalgenome.org/category/phase-3/" target="_blank">https://www.internationalgenome.org/category/phase-3/</a></td></tr><tr><td>HAPMAP2 r22.36 CEU</td><td>Open source</td><td>ftp://share.sph.umich.edu/1000genomes/resources/CEU.r22.orig.tgz</td></tr><tr><td>NCBI dbSNP database</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref106-1" href="#" role="doc-biblioref" data-xml-rid="bib106" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref106-1" aria-controls="sref106-1">Sherry et&nbsp;al., 2001</a><div class="dropBlock__holder" data-db-target-of="sref106-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">115.</div><div>Sherry, S.T. ∙ Ward, M.-H. ∙ Kholodov, M. ...</div><div><strong>dbSNP: the NCBI database of genetic variation</strong></div><div><em>Nucleic Acids Res.</em> 2001; <strong>29</strong>:308-311</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/29.1.308" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11125122/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2F29.1.308&amp;pmid=11125122" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="interref0400" href="http://www.ncbi.nlm.nih.gov/SNP" target="_blank">http://www.ncbi.nlm.nih.gov/SNP</a></td></tr><tr><td>RefSeq database</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref132-1" href="#" role="doc-biblioref" data-xml-rid="bib132" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref132-1" aria-controls="sref132-1">Pruitt et al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref132-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">104.</div><div>Pruitt, K.D. ∙ Brown, G.R. ∙ Hiatt, S.M. ...</div><div><strong>RefSeq: an update on mammalian reference sequences</strong></div><div><em>Nucleic Acids Res.</em> 2014; <strong>42</strong>:D756-D763</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gkt1114" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_104_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-84891767394" target="_blank" aria-label="Open Scopus in new tab">Scopus (651)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24259432/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgkt1114&amp;pmid=24259432" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0035" href="https://www.ncbi.nlm.nih.gov/refseq/" target="_blank">https://www.ncbi.nlm.nih.gov/refseq/</a></td></tr><tr><td>UCSC genome browser</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref34-1" href="#" role="doc-biblioref" data-xml-rid="bib34" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref34-1" aria-controls="sref34-1">Fujita et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref34-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">35.</div><div>Fujita, P.A. ∙ Rhead, B. ∙ Zweig, A.S. ...</div><div><strong>The UCSC Genome Browser database: update 2011</strong></div><div><em>Nucleic Acids Res.</em> 2011; <strong>39</strong>:D876-D882</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gkq963" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-78651320424" target="_blank" aria-label="Open Scopus in new tab">Scopus (815)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20959295/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgkq963&amp;pmid=20959295" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0040" href="http://genome.ucsc.edu" target="_blank">http://genome.ucsc.edu</a></td></tr><tr><td>Ensemble genome browser 89</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref2-1" href="#" role="doc-biblioref" data-xml-rid="bib2" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref2-1" aria-controls="sref2-1">Aken et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref2-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">2.</div><div>Aken, B.L. ∙ Ayling, S. ∙ Barrell, D. ...</div><div><strong>The Ensembl gene annotation system</strong></div><div><em>Database (Oxford).</em> 2016; <strong>2016</strong>:baw093</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/database/baw093" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27337980/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fdatabase%2Fbaw093&amp;pmid=27337980" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0045" href="http://asia.ensembl.org/Homo_sapiens/Info/Index" target="_blank">http://asia.ensembl.org/Homo_sapiens/Info/Index</a></td></tr><tr><td>Roadmap Epigenomics Project</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref12-1" href="#" role="doc-biblioref" data-xml-rid="bib12" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref12-1" aria-controls="sref12-1">Bernstein et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref12-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">12.</div><div>Bernstein, B.E. ∙ Stamatoyannopoulos, J.A. ∙ Costello, J.F. ...</div><div><strong>The NIH Roadmap Epigenomics Mapping Consortium</strong></div><div><em>Nat. Biotechnol.</em> 2010; <strong>28</strong>:1045-1048</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nbt1010-1045" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-77957940722" target="_blank" aria-label="Open Scopus in new tab">Scopus (1125)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20944595/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnbt1010-1045&amp;pmid=20944595" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0050" href="http://www.roadmapepigenomics.org" target="_blank">http://www.roadmapepigenomics.org</a></td></tr><tr><td>GENCODE v19</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref133-1" href="#" role="doc-biblioref" data-xml-rid="bib133" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref133-1" aria-controls="sref133-1">Harrow et al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref133-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">45.</div><div>Harrow, J. ∙ Frankish, A. ∙ Gonzalez, J.M. ...</div><div><strong>GENCODE: the reference human genome annotation for The ENCODE Project</strong></div><div><em>Genome research.</em> 2012; <strong>22</strong>:1760-1774</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/gr.135350.111" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_45_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-84865760395" target="_blank" aria-label="Open Scopus in new tab">Scopus (2415)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22955987/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2Fgr.135350.111&amp;pmid=22955987" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0055" href="https://www.gencodegenes.org/human/release_19.html" target="_blank">https://www.gencodegenes.org/human/release_19.html</a></td></tr><tr><td>GTEx v8</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref134-1" href="#" role="doc-biblioref" data-xml-rid="bib134" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref134-1" aria-controls="sref134-1">GTEx Consortium, 2017</a><div class="dropBlock__holder" data-db-target-of="sref134-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">42.</div><div>GTEx Consortium</div><div><strong>Genetic effects on gene expression across human tissues</strong></div><div><em>Nature.</em> 2017; <strong>550</strong>:204-213</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature24277" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-85031313737" target="_blank" aria-label="Open Scopus in new tab">Scopus (1546)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29022597/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature24277&amp;pmid=29022597" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0060" href="https://gtexportal.org/home/" target="_blank">https://gtexportal.org/home/</a></td></tr><tr><td>ENCODE</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref135-1" href="#" role="doc-biblioref" data-xml-rid="bib135" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref135-1" aria-controls="sref135-1">Davis et al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref135-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">24.</div><div>Davis, C.A. ∙ Hitz, B.C. ∙ Sloan, C.A. ...</div><div><strong>The Encyclopedia of DNA elements (ENCODE): data portal update</strong></div><div><em>Nucleic Acids Res.</em> 2018; <strong>46</strong>:D794-D801</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gkx1081" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-85040942752" target="_blank" aria-label="Open Scopus in new tab">Scopus (573)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29126249/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgkx1081&amp;pmid=29126249" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="interref0405" href="https://www.encodeproject.org/" target="_blank">https://www.encodeproject.org/</a></td></tr><tr><td>HaploReg v4.1</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref119-1" href="#" role="doc-biblioref" data-xml-rid="bib119" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref119-1" aria-controls="sref119-1">Ward and Kellis, 2012</a><div class="dropBlock__holder" data-db-target-of="sref119-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">129.</div><div>Ward, L.D. ∙ Kellis, M.</div><div><strong>HaploReg: a resource for exploring chromatin states, conservation, and regulatory motif alterations within sets of genetically linked variants</strong></div><div><em>Nucleic Acids Res.</em> 2012; <strong>40</strong>:D930-D934</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gkr917" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_129_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-84858779229" target="_blank" aria-label="Open Scopus in new tab">Scopus (1526)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22064851/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgkr917&amp;pmid=22064851" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0070" href="https://pubs.broadinstitute.org/mammals/haploreg/haploreg.php" target="_blank">https://pubs.broadinstitute.org/mammals/haploreg/haploreg.php</a></td></tr><tr><td>STRING v11.0</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref111-2" href="#" role="doc-biblioref" data-xml-rid="bib111" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref111-2" aria-controls="sref111-2">Szklarczyk et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref111-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">120.</div><div>Szklarczyk, D. ∙ Gable, A.L. ∙ Lyon, D. ...</div><div><strong>STRING v11: protein-protein association networks with increased coverage, supporting functional discovery in genome-wide experimental datasets</strong></div><div><em>Nucleic Acids Res.</em> 2019; <strong>47</strong>:D607-D613</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gky1131" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_120_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-85059797047" target="_blank" aria-label="Open Scopus in new tab">Scopus (4748)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30476243/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgky1131&amp;pmid=30476243" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0075" href="https://string-db.org/" target="_blank">https://string-db.org/</a></td></tr><tr class="rowsep"><td>OMIM</td><td>Johns Hopkins School of Medicine</td><td><a id="intref0080" href="https://www.omim.org" target="_blank">https://www.omim.org</a></td></tr><tr class="rowsep"><td colspan="3"><b>Experimental models: organisms/strains</b></td></tr><tr><td>Mouse: FVB/N</td><td>Charles River</td><td>Strain:207</td></tr><tr><td>Mouse: TCF/Lef::H2B-GFP (<span class="dropBlock reference-citations"><a id="body-ref-sref32" href="#" role="doc-biblioref" data-xml-rid="bib32" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref32" aria-controls="sref32">Ferrer-Vaquer et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref32"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">33.</div><div>Ferrer-Vaquer, A. ∙ Piliszek, A. ∙ Tian, G. ...</div><div><strong>A sensitive and bright single-cell resolution live imaging reporter of Wnt/ß-catenin signaling in the mouse</strong></div><div><em>BMC Dev. Biol.</em> 2010; <strong>10</strong>:121</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/1471-213X-10-121" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-78650255796" target="_blank" aria-label="Open Scopus in new tab">Scopus (185)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21176145/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2F1471-213X-10-121&amp;pmid=21176145" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td>Roslin Institute</td><td>N/A</td></tr><tr><td>Mouse: Tabby (<i>Eda</i><sup>Ta</sup>)</td><td>Roslin Institute</td><td>N/A</td></tr><tr><td>Mouse: downless mutants (<i>Edar</i><sup><i>dlJ/dlJ</i></sup> and <i>Edar</i><sup><i>dlJ/+</i></sup><i>)</i></td><td>Roslin Institute</td><td>N/A</td></tr><tr class="rowsep"><td>Mouse: Junbo (C3H/HeH background)</td><td>European Mouse Mutant archive – Bred at Roslin Institute &amp; Harwell</td><td>EM: 00091</td></tr><tr class="rowsep"><td colspan="3"><b>Oligonucleotides</b></td></tr><tr><td><i>Evi1</i> F: 5′-GCTATGATCAGCACAACCTTGTTG-3′</td><td>Thermo Fisher Scientific</td><td>N/A</td></tr><tr><td><i>Evi1</i> R: 5′-TGTCTGCGACTACTCGGTAGAATATC-3′</td><td>Thermo Fisher Scientific</td><td>N/A</td></tr><tr><td><i>Gapdh</i> F: 5′-CGTATTGGGCGCCTGGTCAC-3′</td><td>Thermo Fisher Scientific</td><td>N/A</td></tr><tr><td><i>Gapdh</i> R: 5′-ATGATGACCCTTTTGGCTCC-3′</td><td>Thermo Fisher Scientific</td><td>N/A</td></tr><tr><td>RNAscope Probe - Mm-Mecom - <i>Mus musculus</i> MDS1 and EVI1 complex locus (Mecom) transcript variant 1 mRNA</td><td>ACD bio</td><td>Cat# 432231</td></tr><tr><td>RNAscope Probe - Hs-MECOM - Homo sapiens MDS1 and EVI1 complex locus (MECOM) transcript variant 1 mRNA</td><td>ACD bio</td><td>Cat# 518021</td></tr><tr><td>RNAscope Probe - Mm-Prrx1-C2 - <i>Mus musculus</i> paired related homeobox 1 (Prrx1) transcript variant 1 mRNA</td><td>ACD bio</td><td>Cat# 485231-C2</td></tr><tr><td>RNAscope Probe - Hs-PRRX1-No-XMm-C2 - Homo sapiens paired related homeobox 1 (PRRX1) transcript variant pmx-1a mRNA</td><td>ACD bio</td><td>Cat# 511001-C2</td></tr><tr><td>RNAscope 3-plex Negative Control Probe</td><td>ACD bio</td><td>Cat# 320871</td></tr><tr><td>RNAscope 3-plex Positive Control Probe_Mm</td><td>ACD bio</td><td>Cat# 320881</td></tr><tr class="rowsep"><td>RNAscope 3-plex Positive Control Probe_Hs</td><td>ACD bio</td><td>Cat# 320861</td></tr><tr class="rowsep"><td colspan="3"><b>Recombinant DNA</b></td></tr><tr class="rowsep"><td><i>Mus musculus</i> Mecom (Evi1) EST clone</td><td>Source Bioscience</td><td>IMAGp998F2411055Q</td></tr><tr class="rowsep"><td colspan="3"><b>Software and algorithms</b></td></tr><tr><td>MATLAB_R2019a</td><td>MathWorks</td><td><a id="intref0085" href="https://www.mathworks.com/products/matlab.html" target="_blank">https://www.mathworks.com/products/matlab.html</a>; RRID: <a id="intref0090" href="https://scicrunch.org/resources/Any/search?q=undefined&amp;l=SCR_013499" target="_blank">SCR_013499</a></td></tr><tr><td>R v3.6.1</td><td>The R Foundation</td><td><a id="intref0095" href="https://www.r-project.org" target="_blank">https://www.r-project.org</a>; RRID: N/A</td></tr><tr><td>PLS-PM</td><td>The R CRAN</td><td><a id="interref0410" href="https://github.com/gastonstat/plspm" target="_blank">https://github.com/gastonstat/plspm</a>; RRID: N/A</td></tr><tr><td>PLINK v1.9</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref96-1" href="#" role="doc-biblioref" data-xml-rid="bib96" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref96-1" aria-controls="sref96-1">Purcell et&nbsp;al., 2007</a><div class="dropBlock__holder" data-db-target-of="sref96-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">105.</div><div>Purcell, S. ∙ Neale, B. ∙ Todd-Brown, K. ...</div><div><strong>PLINK: a tool set for whole-genome association and population-based linkage analyses</strong></div><div><em>Am. J. Hum. Genet.</em> 2007; <strong>81</strong>:559-575</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_105_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=10.1086%2F519795&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_105_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=10.1086%2F519795&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_105_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-34548292504" target="_blank" aria-label="Open Scopus in new tab">Scopus (17562)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17701901/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1086%2F519795&amp;pmid=17701901" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0105" href="https://www.cog-genomics.org/plink2" target="_blank">https://www.cog-genomics.org/plink2</a>; RRID: N/A</td></tr><tr><td>SHAPEIT</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref24-1" href="#" role="doc-biblioref" data-xml-rid="bib24" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref24-1" aria-controls="sref24-1">Delaneau et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref24-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">25.</div><div>Delaneau, O. ∙ Marchini, J. ∙ Zagury, J.F.</div><div><strong>A linear complexity phasing method for thousands of genomes</strong></div><div><em>Nat. Methods.</em> 2011; <strong>9</strong>:179-181</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.1785" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-84856478855" target="_blank" aria-label="Open Scopus in new tab">Scopus (1024)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22138821/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.1785&amp;pmid=22138821" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0110" href="http://www.shapeitforum.com" target="_blank">http://www.shapeitforum.com</a>; RRID: N/A</td></tr><tr><td>IMPUTE2</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref49-1" href="#" role="doc-biblioref" data-xml-rid="bib49" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref49-1" aria-controls="sref49-1">Howie et&nbsp;al., 2009</a><div class="dropBlock__holder" data-db-target-of="sref49-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">52.</div><div>Howie, B.N. ∙ Donnelly, P. ∙ Marchini, J.</div><div><strong>A flexible and accurate genotype imputation method for the next generation of genome-wide association studies</strong></div><div><em>PLoS Genet.</em> 2009; <strong>5</strong>:e1000529</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pgen.1000529" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-67651222400" target="_blank" aria-label="Open Scopus in new tab">Scopus (2615)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19543373/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pgen.1000529&amp;pmid=19543373" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0115" href="https://mathgen.stats.ox.ac.uk/impute/impute_v2.html" target="_blank">https://mathgen.stats.ox.ac.uk/impute/impute_v2.html</a>; RRID: <a id="intref0120" href="https://scicrunch.org/resources/Any/search?q=undefined&amp;l=SCR_013055" target="_blank">SCR_013055</a></td></tr><tr><td>EIGENSOFT</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref95-1" href="#" role="doc-biblioref" data-xml-rid="bib95" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref95-1" aria-controls="sref95-1">Price et&nbsp;al., 2006</a><div class="dropBlock__holder" data-db-target-of="sref95-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">102.</div><div>Price, A.L. ∙ Patterson, N.J. ∙ Plenge, R.M. ...</div><div><strong>Principal components analysis corrects for stratification in genome-wide association studies</strong></div><div><em>Nat. Genet.</em> 2006; <strong>38</strong>:904-909</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng1847" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_102_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-33746512512" target="_blank" aria-label="Open Scopus in new tab">Scopus (6332)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16862161/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng1847&amp;pmid=16862161" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0125" href="https://data.broadinstitute.org/alkesgroup/EIGENSOFT/" target="_blank">https://data.broadinstitute.org/alkesgroup/EIGENSOFT/</a>; RRID: <a id="intref0130" href="https://scicrunch.org/resources/Any/search?q=undefined&amp;l=SCR_004965" target="_blank">SCR_004965</a></td></tr><tr><td>qqman v0.1.4</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref114-1" href="#" role="doc-biblioref" data-xml-rid="bib114" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref114-1" aria-controls="sref114-1">Turner, 2014</a><div class="dropBlock__holder" data-db-target-of="sref114-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">124.</div><div>Turner, S.D.</div><div><strong>qqman: an R package for visualizing GWAS results using QQ and manhattan plots</strong></div><div><em>bioRxiv.</em> 2014;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/005165" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2F005165" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0135" href="https://cran.r-project.org/web/packages/qqman/index.html" target="_blank">https://cran.r-project.org/web/packages/qqman/index.html</a>; RRID: N/A</td></tr><tr><td>BOLT-LMM v2.3.2</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref137-1" href="#" role="doc-biblioref" data-xml-rid="bib137" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref137-1" aria-controls="sref137-1">Loh et al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref137-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">75.</div><div>Loh, P.R. ∙ Tucker, G. ∙ Bulik-Sullivan, B.K. ...</div><div><strong>Efficient Bayesian mixed-model analysis increases association power in large cohorts</strong></div><div><em>. Nat Genet.</em> 2015; <strong>47</strong>:284-290</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng.3190" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_75_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-84924060689" target="_blank" aria-label="Open Scopus in new tab">Scopus (693)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25642633/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng.3190&amp;pmid=25642633" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0140" href="https://alkesgroup.broadinstitute.org/BOLT-LMM/BOLT-LMM_manual.html" target="_blank">https://alkesgroup.broadinstitute.org/BOLT-LMM/BOLT-LMM_manual.html</a>; RRID: N/A</td></tr><tr><td>MACH2DAT</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref138-1" href="#" role="doc-biblioref" data-xml-rid="bib138" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref138-1" aria-controls="sref138-1">Li et al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref138-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">71.</div><div>Li, Y. ∙ Willer, C.J. ∙ Ding, J. ...</div><div><strong>MaCH: using sequence and genotype data to estimate haplotypes and unobserved genotypes</strong></div><div><em>Genet Epidemiol.</em> 2010; <strong>34</strong>:816-834</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/gepi.20533" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-78649508578" target="_blank" aria-label="Open Scopus in new tab">Scopus (1447)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21058334/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fgepi.20533&amp;pmid=21058334" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0145" href="https://genome.sph.umich.edu/wiki/Mach2dat:_Association_with_MACH_output" target="_blank">https://genome.sph.umich.edu/wiki/Mach2dat:_Association_with_MACH_output</a>; RRID: <a id="intref0150" href="https://scicrunch.org/resources/Any/search?q=undefined&amp;l=SCR_009599" target="_blank">SCR_009599</a></td></tr><tr><td>MERLIN-OFFLINE</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref1-1" href="#" role="doc-biblioref" data-xml-rid="bib1" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref1-1" aria-controls="sref1-1">Abecasis et&nbsp;al., 2002</a><div class="dropBlock__holder" data-db-target-of="sref1-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">1.</div><div>Abecasis, G.R. ∙ Cherny, S.S. ∙ Cookson, W.O. ...</div><div><strong>Merlin--rapid analysis of dense genetic maps using sparse gene flow trees</strong></div><div><em>Nat. Genet.</em> 2002; <strong>30</strong>:97-101</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng786" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_1_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-0036338150" target="_blank" aria-label="Open Scopus in new tab">Scopus (2712)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11731797/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng786&amp;pmid=11731797" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0155" href="https://sph.umich.edu/csg/abecasis/Merlin/" target="_blank">https://sph.umich.edu/csg/abecasis/Merlin/</a>; RRID: N/A</td></tr><tr><td>EMMAX</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref51-1" href="#" role="doc-biblioref" data-xml-rid="bib51" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref51-1" aria-controls="sref51-1">Kang et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref51-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">56.</div><div>Kang, H.M. ∙ Sul, J.H. ∙ Service, S.K. ...</div><div><strong>Variance component model to account for sample structure in genome-wide association studies</strong></div><div><em>Nat. Genet.</em> 2010; <strong>42</strong>:348-354</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng.548" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_56_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-77950301214" target="_blank" aria-label="Open Scopus in new tab">Scopus (1414)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20208533/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng.548&amp;pmid=20208533" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0160" href="http://genetics.cs.ucla.edu/emmax_jemdoc/" target="_blank">http://genetics.cs.ucla.edu/emmax_jemdoc/</a>; RRID: N/A</td></tr><tr><td>METAL</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref122-1" href="#" role="doc-biblioref" data-xml-rid="bib122" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref122-1" aria-controls="sref122-1">Willer et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref122-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">132.</div><div>Willer, C.J. ∙ Li, Y. ∙ Abecasis, G.R.</div><div><strong>METAL: fast and efficient meta-analysis of genomewide association scans</strong></div><div><em>Bioinformatics.</em> 2010; <strong>26</strong>:2190-2191</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btq340" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_132_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-77955894071" target="_blank" aria-label="Open Scopus in new tab">Scopus (2264)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20616382/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtq340&amp;pmid=20616382" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0165" href="http://csg.sph.umich.edu/abecasis/Metal/" target="_blank">http://csg.sph.umich.edu/abecasis/Metal/</a>; RRID: <a id="intref0170" href="https://scicrunch.org/resources/Any/search?q=undefined&amp;l=SCR_002013" target="_blank">SCR_002013</a></td></tr><tr><td>METASOFT</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref42-1" href="#" role="doc-biblioref" data-xml-rid="bib42" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref42-1" aria-controls="sref42-1">Han and Eskin, 2011</a><div class="dropBlock__holder" data-db-target-of="sref42-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">44.</div><div>Han, B. ∙ Eskin, E.</div><div><strong>Random-effects model aimed at discovering associations in meta-analysis of genome-wide association studies</strong></div><div><em>Am. J. Hum. Genet.</em> 2011; <strong>88</strong>:586-598</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=10.1016%2Fj.ajhg.2011.04.014&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=10.1016%2Fj.ajhg.2011.04.014&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-79955860273" target="_blank" aria-label="Open Scopus in new tab">Scopus (297)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21565292/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.ajhg.2011.04.014&amp;pmid=21565292" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0175" href="http://genetics.cs.ucla.edu/meta/" target="_blank">http://genetics.cs.ucla.edu/meta/</a>; RRID: N/A</td></tr><tr><td>PAINTOR v2.1</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref56-1" href="#" role="doc-biblioref" data-xml-rid="bib56" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref56-1" aria-controls="sref56-1">Kichaev et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref56-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">61.</div><div>Kichaev, G. ∙ Yang, W.Y. ∙ Lindstrom, S. ...</div><div><strong>Integrating functional data to prioritize causal variants in statistical fine-mapping studies</strong></div><div><em>PLoS Genet.</em> 2014; <strong>10</strong>:e1004722</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pgen.1004722" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-84908324508" target="_blank" aria-label="Open Scopus in new tab">Scopus (227)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25357204/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pgen.1004722&amp;pmid=25357204" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0180" href="https://github.com/gkichaev/PAINTOR_V3.0" target="_blank">https://github.com/gkichaev/PAINTOR_V3.0</a>; RRID: N/A</td></tr><tr><td>GCTA-GREML</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref61-1" href="#" role="doc-biblioref" data-xml-rid="bib61" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref61-1" aria-controls="sref61-1">Lee et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref61-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">66.</div><div>Lee, S.H. ∙ Yang, J. ∙ Goddard, M.E. ...</div><div><strong>Estimation of pleiotropy between complex diseases using single-nucleotide polymorphism-derived genomic relationships and restricted maximum likelihood</strong></div><div><em>Bioinformatics.</em> 2012; <strong>28</strong>:2540-2542</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/bts474" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-84867284056" target="_blank" aria-label="Open Scopus in new tab">Scopus (344)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22843982/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbts474&amp;pmid=22843982" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref125-1" href="#" role="doc-biblioref" data-xml-rid="bib125" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref125-1" aria-controls="sref125-1">Yang et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref125-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">135.</div><div>Yang, J. ∙ Lee, S.H. ∙ Goddard, M.E. ...</div><div><strong>GCTA: a tool for genome-wide complex trait analysis</strong></div><div><em>Am. J. Hum. Genet.</em> 2011; <strong>88</strong>:76-82</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_135_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=10.1016%2Fj.ajhg.2010.11.011&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_135_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=10.1016%2Fj.ajhg.2010.11.011&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_135_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-78650856517" target="_blank" aria-label="Open Scopus in new tab">Scopus (3187)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21167468/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.ajhg.2010.11.011&amp;pmid=21167468" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0185" href="https://yanglab.westlake.edu.cn/software/gcta/" target="_blank">https://yanglab.westlake.edu.cn/software/gcta/</a>; RRID: N/A</td></tr><tr><td>LocusZoom</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref139-2" href="#" role="doc-biblioref" data-xml-rid="bib139" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref139-2" aria-controls="sref139-2">Pruim et al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref139-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">103.</div><div>Pruim, R.J. ∙ Welch, R.P. ∙ Sanna, S. ...</div><div><strong>LocusZoom: regional visualization of genome-wide association scan results</strong></div><div><em>Bioinformatics.</em> 2010; <strong>26</strong>:2336-2337</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btq419" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_103_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-77956586071" target="_blank" aria-label="Open Scopus in new tab">Scopus (1476)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20634204/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtq419&amp;pmid=20634204" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0190" href="https://genome.sph.umich.edu/wiki/LocusZoom" target="_blank">https://genome.sph.umich.edu/wiki/LocusZoom</a>; RRID: <a id="intref0195" href="https://scicrunch.org/resources/Any/search?q=undefined&amp;l=SCR_009257" target="_blank">SCR_009257</a></td></tr><tr><td>GREAT v4.0.4</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref76-2" href="#" role="doc-biblioref" data-xml-rid="bib76" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref76-2" aria-controls="sref76-2">McLean et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref76-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">83.</div><div>McLean, C.Y. ∙ Bristor, D. ∙ Hiller, M. ...</div><div><strong>GREAT improves functional interpretation of cis-regulatory regions</strong></div><div><em>Nat. Biotechnol.</em> 2010; <strong>28</strong>:495-501</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nbt.1630" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_83_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-77952214662" target="_blank" aria-label="Open Scopus in new tab">Scopus (2170)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20436461/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnbt.1630&amp;pmid=20436461" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0200" href="http://great.stanford.edu/public/html/index.php" target="_blank">http://great.stanford.edu/public/html/index.php</a>; RRID: <a id="intref0205" href="https://scicrunch.org/resources/Any/search?q=undefined&amp;l=SCR_005807" target="_blank">SCR_005807</a></td></tr><tr><td>Adobe Photoshop</td><td>Adobe</td><td>N/A</td></tr><tr><td>MxPro</td><td>Stratagene</td><td>N/A</td></tr><tr class="rowsep"><td>Zen Imaging Software</td><td>Carl Zeiss</td><td>N/A</td></tr><tr class="rowsep"><td colspan="3"><b>Other</b></td></tr><tr><td>Greenbit DactyScan26</td><td>Neurotechnology</td><td><a id="intref0210" href="https://www.neurotechnology.com/fingerprint-scanner-green-bit-dactyscan-26.html" target="_blank">https://www.neurotechnology.com/fingerprint-scanner-green-bit-dactyscan-26.html</a></td></tr><tr><td>Epson Perfection V370 Photo (EPSONScanV370)</td><td>EPSON</td><td><a id="intref0215" href="https://www.epson.com.cn/products/scanners/206/V370/" target="_blank">https://www.epson.com.cn/products/scanners/206/V370/</a></td></tr><tr><td>Hanlintongxin PU-JY500U</td><td>Beijing Hanlin ICT Information Technology</td><td>N/A</td></tr><tr><td>Smiths Heimann Biometrics ACCO1394</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref77-2" href="#" role="doc-biblioref" data-xml-rid="bib77" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref77-2" aria-controls="sref77-2">Medland et&nbsp;al., 2007</a><div class="dropBlock__holder" data-db-target-of="sref77-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">84.</div><div>Medland, S.E. ∙ Loesch, D.Z. ∙ Mdzewski, B. ...</div><div><strong>Linkage analysis of a model quantitative trait in humans: finger ridge count shows significant multivariate linkage to 5q14.1</strong></div><div><em>PLoS Genet.</em> 2007; <strong>3</strong>:1736-1744</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pgen.0030165" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_84_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-34848862129" target="_blank" aria-label="Open Scopus in new tab">Scopus (39)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17907812/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pgen.0030165&amp;pmid=17907812" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0220" href="https://www.smiths.com/news-and-media/2005/07/smiths-heimann-biometrics-and-cross-match-technologies-to-join-forces" target="_blank">https://www.smiths.com/news-and-media/2005/07/smiths-heimann-biometrics-and-cross-match-technologies-to-join-forces</a></td></tr><tr><td>Cytoscape</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref57-1" href="#" role="doc-biblioref" data-xml-rid="bib57" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref57-1" aria-controls="sref57-1">Kohl et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref57-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">62.</div><div>Kohl, M. ∙ Wiese, S. ∙ Warscheid, B.</div><div><strong>Cytoscape: software for visualization and analysis of biological networks</strong></div><div><strong>Data Mining in Proteomics</strong></div><div>Springer, 2011; 291-303</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/978-1-60761-987-1_18" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-79952202448" target="_blank" aria-label="Open Scopus in new tab">Scopus (668)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2F978-1-60761-987-1_18" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0225" href="https://apps.cytoscape.org/" target="_blank">https://apps.cytoscape.org/</a></td></tr></tbody></table></div><figcaption><div class="core-table-tools"><ul><li><a class="core-table-show-action" href="/action/showFullTableHTML?isHtml=true&amp;tableId=undtbl1&amp;pii=S0092-8674%2821%2901446-X" target="_blank">Open table in a new tab</a></li></ul></div></figcaption></figure></div></div></section><section id="sec-4-2"><h3>Resource availability</h3><section id="sec-4-2-1"><h4>Lead contact</h4><div id="p0175" role="paragraph">Further information and requests for resources should be directed to and will be fulfilled by the Lead Contact, Sijia Wang (<a id="intref0230" href="mailto:wangsijia@picb.ac.cn">wangsijia@picb.ac.cn</a>).</div></section><section id="sec-4-2-2"><h4>Materials availability</h4><div id="p0180" role="paragraph">This study did not generate new unique reagents.</div></section></section><section id="sec-4-3"><h3>Experimental model and subject details</h3><section id="sec-4-3-1"><h4>Human subjects</h4><div id="p0185" role="paragraph">There are in total 23,966 samples from 8 independent cohorts (<a id="crosref1260" href="#mmc1">Table S1</a>). In discovery stage, the Taizhou longitudinal (TZL) cohort includes 2961 adults (1059 males and 1902 females, aged 31-81 years); the National Survey of Physical Traits (NSPT) cohort comprises 2679 individuals from three different regions of China (1045 males and 1634 females, aged 18-83 years); the Jidong (JD) cohort includes 4269 adults (2104 males and 2165 females, aged 20-82 years). Two cohorts for replication: the Chinese Kadoorie biobank (CKB) includes 1785 adults (596 males and 1189 females, aged 36-55 years), and the WeGene cohort comprises 2152 unrelated individuals (954 males and 1198 females, aged 6-68 years). In <i>trans</i>-ethnic meta-analyses, the Pittsburgh cohort includes 1480 volunteers (690 males and 790 females, aged 0-86 years), the Avon Longitudinal Study of Parents and Children (ALSPAC) cohort includes 5339 individuals, and the Queensland Institute of Medical Research (QIMR) cohort includes 3301 individuals. All participants provided written informed consent, and all study protocols were approved by the institutional review boards of the pertinent research institutions. For detailed information about study populations, donor enrollment, blood extraction, specification of fingerprint patterns and hand traits review the <a id="crosref1265" href="#sec-4-4">Methods details</a>.</div></section><section id="sec-4-3-2"><h4>Cell lines and cultures</h4><div id="p0190" role="paragraph">Luciferase reporter plasmids were transfected into HEK293T cells. HEK293T cells were cultured in DMEM (GIBCO) supplemented with 10% fetal bovine serum (FBS, GIBCO) and 1% Penicillin-Streptomycin (GIBCO) and incubated at 37°C in 5% CO2. Plasmids construction and cell culture are indicated in the <a id="crosref1270" href="#sec-4-4">Methods details</a>.</div></section><section id="sec-4-3-3"><h4>Mice</h4><div id="p0195" role="paragraph">FVB, <i>Tabby</i> (<i>Eda</i><sup><i>Ta</i></sup>) and <i>downless</i> mutants (<i>Edar</i><sup><i>dlJ/dlJ</i></sup> and <i>Edar</i><sup><i>dlJ/+</i></sup><i>)</i> mice were bred at the Roslin Institute. <i>Junbo</i> mice were bred in both the Roslin Institute and MRC, Harwell centers. <i>Junbo</i> mice are congenic on a C3H/HeH genetic background (European Mouse Mutant Archive; EM: 00091) and maintained by crossing <i>Junbo</i> heterozygote males with C3H/HeH females. They bear a mutation in the transcription factor <i>Evi1/Mecom</i> and are therefore referred as <i>Evi1</i><sup><i>Jbo</i></sup>. This mouse model with altered EVI1 function through amino acid substitution ensures that <i>Evi1</i> itself is the modified gene. <i>downless</i> mice (<i>Edar</i> p.E379K) were maintained by intercrossing <i>Edar</i><sup><i>dlJ/+</i></sup> mice. Homozygous <i>Edar</i><sup><i>dlJ/dlJ</i></sup> represents a loss of function mutation, with mice having a sparse hair coat and hairless tail, whereas <i>Edar</i><sup><i>dlJ/+</i></sup> mice have an appearance identical to wild-type. Both <i>Eda</i> and <i>Edar</i> mutant strains were maintained on the FVB genetic background. The effect of <i>Evi1</i> mutation was studied at the age of P21 (postnatal 21&nbsp;days) and for other mouse samples the age is specified. 16 <i>Evi1</i><sup><i>Jbo</i></sup> and 10 wild-type mice, of both sexes, were used (<a id="crosref1275" href="#mmc4">Table S4</a>). Mice were killed by cervical dislocation and DNA from tail tips was used for determining the genotypes. Forelimbs were dissected at the distal end of the zeugopod. <i>Evi1</i><sup><i>Jbo/+</i></sup> mutants are characterized by the presence of an extra spur on digit 5, present in either left, right or both the forelimbs. The study was performed under UK Home Office license and approved by the Roslin Institute Animal Welfare and Ethical Review Body. Mouse embryonic tissue for <i>in situ</i> hybridization and qRT-PCR was obtained by timed mating of FVB/N male and female mice. Noon on the day of copulation plug detection denoted embryonic day 0.5.</div></section><section id="sec-4-3-4"><h4>Fetal tissue collection</h4><div id="p0200" role="paragraph">Fetal tissue samples, no gender requirement, used in immunofluorescence were obtained after elective medical termination of pregnancy from the Royal Infirmary of Edinburgh, UK with informed consent (approved by the Lothian Research Ethics Committee, Ref: 08/S1101/1). All were morphologically normal and gestational age was determined according to Carnegie Stages (CS) of human development for embryos&nbsp;&lt; 10&nbsp;weeks or by ultrasound &gt; 10&nbsp;weeks gestation.</div></section></section><section id="sec-4-4"><h3>Method details</h3><section id="sec-4-4-1"><h4>Ethics statement</h4><div id="p0205" role="paragraph">All participants provided written informed consent, and all study protocols were approved by the institutional review boards of the&nbsp;pertinent research institutions. The Taizhou Longitudinal Study (TZL) was approved by the Ethics Committee of Human Genetic Resources at the Shanghai institute of life Sciences, Chinese Academy of Sciences (ER-SIBS-261410). The Jidong cohort (JD) was approved by the Ethics Committee of Human Genetic Resources at the Shanghai Institute of Life Sciences, Chinese Academy of Sciences (ER-SIBS-261410-A1801). The National Survey of Physical Traits (NSPT) is the sub project of The National Science &amp; Technology Basic Research Project which was approved by the Ethics Committee of Human Genetic Resources of School of Life Sciences, Fudan University, Shanghai (14117). The CKB ethics approval was obtained from the Institutional Review Board (IRB) at the Peking University (IRB00001052-13055). Participants of WeGene cohort provided informed consent and participated in the research online, under a protocol approved by the Ethical Committee of WeGene. The ethical approval of ALSPAC birth cohort study was obtained from the ALSPAC Ethics and Law committee and the Local Research Ethics Committees. The QIMR study has been approved by QIMR Berghofer Human Research Ethics Committee (P193 &amp; P455). The Pittsburgh cohort ethics approval was obtained locally at each recruitment site, including the University of Pittsburgh, which served as the coordinating center for this project (IRB0405013). Written informed consent was granted for each participant before enrollment in the study. We confirm that our study is compliant with the Guidance of the Ministry of Science and Technology (MOST) for the Review and Approval of Human Genetic Resources. Ethical approval for analysis of human fetal tissue was obtained from Lothian Research Ethics Committee (study code LREC 08/S1101/1), with informed written consent.</div></section><section id="sec-4-4-2"><h4>Study population and design</h4><div id="p0210" role="paragraph">The overall design in current study is shown in <a id="crosref1280" href="#figs6">Figure&nbsp;S6</a>. This study is based on data from eight independent cohorts (<a id="crosref1285" href="#mmc1">Table S1</a>). Study participants in discovery stage were from three cohorts: 1) Taizhou, Jiangsu Province, as part of the Taizhou Longitudinal Study (TZL) (<span class="dropBlock reference-citations"><a id="body-ref-sref117" href="#" role="doc-biblioref" data-xml-rid="bib117" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref117" aria-controls="sref117">Wang et&nbsp;al., 2009</a><div class="dropBlock__holder" data-db-target-of="sref117"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">127.</div><div>Wang, X. ∙ Lu, M. ∙ Qian, J. ...</div><div><strong>Rationales, design and recruitment of the Taizhou Longitudinal Study</strong></div><div><em>BMC Public Health.</em> 2009; <strong>9</strong>:223</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/1471-2458-9-223" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_127_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-68049137338" target="_blank" aria-label="Open Scopus in new tab">Scopus (89)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19589173/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2F1471-2458-9-223&amp;pmid=19589173" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). In total, 2961 Han Chinese individuals (including 1059 males and 1902 females) who were aged 31-81 years were enrolled in 2014; 2) four sub-cohorts collected from three different regions of China in different years: 15HanTZ, 17HanZZ, 18HanNN, and 19HanTZ. These four sub-cohorts from The National Survey of Physical Traits (NSPT) were part of the National Science &amp; Technology Basic Research Project. Totally the NSPT cohort consisted of 1045 males and 1634 females, aged 18-83 years; 3) 2104 males and 2165 females, aged 20-82 years from Jidong of Hebei Province (JD). The summary statistics of fingerprint patterns for replication analysis were from two other independent Chinese Han cohorts consisting of: 1) 596 males and 1189 females, aged 36-55 years, as part of the China Kadoorie Biobank (CKB); 2) WeGene cohort of 2152 unrelated participants (954 males and 1198 females) who received the Personal Genome Service of WeGene, aged 6-68 recruited primarily online including organic posts on WeGene’s Wechat social media channels and website. The GWAS summary statistics of three European-ancestry populations (ALSPAC, QIMR and Pittsburgh cohorts) also were included in the large-scale meta-analysis stage. Data from the ALSPAC and QIMR studies have been described in detail in a previous study (<span class="dropBlock reference-citations"><a id="body-ref-sref46-2" href="#" role="doc-biblioref" data-xml-rid="bib46" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref46-2" aria-controls="sref46-2">Ho et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref46-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">49.</div><div>Ho, Y.Y.W. ∙ Evans, D.M. ∙ Montgomery, G.W. ...</div><div><strong>Common Genetic Variants Influence Whorls in Fingerprint Patterns</strong></div><div><em>J.&nbsp;Invest. Dermatol.</em> 2016; <strong>136</strong>:859-862</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=10.1016%2Fj.jid.2015.10.062&amp;cf=fulltext&amp;site=jid-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=10.1016%2Fj.jid.2015.10.062&amp;cf=pdf&amp;site=jid-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-84978086948" target="_blank" aria-label="Open Scopus in new tab">Scopus (11)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27045867/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.jid.2015.10.062&amp;pmid=27045867" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The Pittsburgh cohort comprised 1480 participants (690 males and 790 females) who were age 0-86 years. These individuals were recruited from a number of international sites as part of the larger Pittsburgh Orofacial Cleft Study (<span class="dropBlock reference-citations"><a id="body-ref-sref120-2" href="#" role="doc-biblioref" data-xml-rid="bib120" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref120-2" aria-controls="sref120-2">Weinberg et&nbsp;al., 2006</a><div class="dropBlock__holder" data-db-target-of="sref120-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">130.</div><div>Weinberg, S.M. ∙ Neiswanger, K. ∙ Martin, R.A. ...</div><div><strong>The Pittsburgh Oral-Facial Cleft study: expanding the cleft phenotype. Background and justification</strong></div><div><em>Cleft Palate Craniofac. J.</em> 2006; <strong>43</strong>:7-20</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1597/04-122R1.1" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_130_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-32044438969" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16405378/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1597%2F04-122R1.1&amp;pmid=16405378" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), a collaborative effort to investigate the genetics of orofacial clefts. Participants include individuals with orofacial clefts, their unaffected relatives, and individuals from control families without a history of clefting. Recruitment of the participants in this study took place in the United States (n&nbsp;= 630), Hungary (n&nbsp;= 678), Spain (n&nbsp;= 117), and Argentina (n&nbsp;= 55).</div><div class="figure-wrap"><figure id="figs6" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.12.008/asset/47983249-8bd7-4ac8-a3c8-5ad3ca90c5ad/main.assets/figs6_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs6" src="/cms/10.1016/j.cell.2021.12.008/asset/720f1ed8-7dec-4194-8a28-a0f0a7184c73/main.assets/figs6.jpg" height="725" width="506" aria-labelledby="figs6-title figs6-content" data-viewer-src="/cms/10.1016/j.cell.2021.12.008/asset/47983249-8bd7-4ac8-a3c8-5ad3ca90c5ad/main.assets/figs6_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs6-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S6</span> <span class="figure__title__text">Flow chart of analyses conducted in the study of fingerprint patterns, related to <a id="crosref1750" href="#sec-4">STAR Methods</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs6-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0290" role="paragraph">Flowchart depicting strategy for the association analysis and functional validation, as well as numbers of samples and SNPs by stage. The yellow diamonds represent bioinformatics tools and software, the green box indicates the main results of the analysis, and the purple box (asterisk) summarizes the conclusions of this study. Abbreviation: TZL&nbsp;= cohort from Taizhou Longitudinal Study, NSPT&nbsp;= cohort from National Survey of Physical Traits Project, JD&nbsp;= cohort from Jidong of Hebei Province, CKB&nbsp;= cohort from China Kadorie Biobank, WeGene&nbsp;= cohort from WeGene company, ALSPAC&nbsp;= The Avon Longitudinal Study of Parents and Children cohort, QIMR&nbsp;= The Queensland Institute of Medical Research cohort.</div></div></div></div></figcaption></figure></div></section><section id="sec-4-4-3"><h4>Specification of fingerprint patterns and hand traits</h4><div id="p0215" role="paragraph">Fingerprints were collected using rolled ink prints on paper, or using an electronic fingerprint scanner in TZL (Greenbit DactyScan26) and NSPT (DactyScan40i) cohorts. The images of the palmar surface of the hands were collected using an electronic scanner (EPSONScanV370) in NSPT and JD cohorts. As the full patterns of the thumbs were not clearly visible for JD samples, we excluded D1L and D1R digits from analyses in JD study. Fingerprint patterns were visually categorized by two investigators according to the number of triradii/delta (triangles) and core (circles) (<span class="dropBlock reference-citations"><a id="body-ref-sref22" href="#" role="doc-biblioref" data-xml-rid="bib22" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref22" aria-controls="sref22">Cummins, 1969</a><div class="dropBlock__holder" data-db-target-of="sref22"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">22.</div><div>Cummins, H.</div><div><strong>The genetics of dermal ridges</strong></div><div><em>Am. J. Hum. Genet.</em> 1969; <strong>21</strong>:516</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=H.CumminsThe+genetics+of+dermal+ridgesAm.+J.+Hum.+Genet.211969516" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref48" href="#" role="doc-biblioref" data-xml-rid="bib48" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref48" aria-controls="sref48">Holt and Penrose, 1968</a><div class="dropBlock__holder" data-db-target-of="sref48"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">51.</div><div>Holt, S.B. ∙ Penrose, L.S.</div><div><strong>The Genetics of Dermal Ridges</strong></div><div>Charles C. Thomas, 1968</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=S.B.HoltL.S.PenroseThe+Genetics+of+Dermal+Ridges1968Charles+C.+Thomas" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>): a) Arch pattern which has 0 triradii and 0 core with gentle (Simple Arch, As) or steep (Tented Arch, At) lines; b) Loop pattern which has 1 triradius and 1 core. Its ridge opens away from the triradius toward the radial or ulnar side and are sub classified as either Radial Loop (Lr) or Ulnar Loop (Lu), respectively; c) Whorl pattern which has 2 triradii and 1 (Simple Whorl, Ws) or 2 (Double Whorl, Wd) cores because of the highly variable inside. For highly correlated phenotypes, we extracted their composite phenotype by a partial least square path model using the “plspm” package in R (<a id="interref0415" href="https://github.com/gastonstat/plspm" target="_blank">https://github.com/gastonstat/plspm</a>) <span class="dropBlock reference-citations"><a id="body-ref-sref136" href="#" role="doc-biblioref" data-xml-rid="bib136" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref136" aria-controls="sref136">(Tenenhaus et al., 2005)</a><div class="dropBlock__holder" data-db-target-of="sref136"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">121.</div><div>Tenenhaus, M. ∙ Vinzi, V.E. ∙ Chatelin, Y.M. ...</div><div><strong>PLS path modeling</strong></div><div><em>Comput Stat Data Anal.</em> 2005; <strong>48</strong>:159-205</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.csda.2004.03.005" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_121_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-10144243373" target="_blank" aria-label="Open Scopus in new tab">Scopus (3076)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.csda.2004.03.005" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>. The bootstrap confidence interval test (bootstrap resampling times&nbsp;= 1000, significance level&nbsp;= 0.05) was applied to test the significance of each path coefficient. Further 66 derived phenotypes (e.g., binary, nominal, ordinal, and quantitative phenotypes) review <a id="crosref1315" href="#mmc3">Table S3</a>. Hand traits were measured by manually calibrating feature points on palmar images which were collected using electronic scanner (EPSONScanV370) in NSPT and JD cohorts. The digit length, distal phalanx length, palm length and palm width were calculated by coordinates of landmarks using MATLAB_R2019a. Further derived ratio phenotypes (e.g., the ratio of digit length to hand length (DHR), the ratio of distal phalanx to hand length (DPHR)) are detailed in <a id="crosref1320" href="#mmc6">Table S6</a>.</div><div id="p0220" role="paragraph">In CKB cohort, fingerprints were collected using an electronic fingerprint scanner (Hanlintongxin PU-JY500U) and categorized fingerprint patterns as both ordinal and binary phenotypes. Fingerprint patterns were obtained twice by manual work (two trained investigators). When two results were inconsistent, a third independent investigator would make final judgments. In WeGene cohort, the same judgments criteria were given to the participants who replied with their self-reported fingerprint patterns both ordinal and binary phenotypes. Within the ALSPAC sample (<span class="dropBlock reference-citations"><a id="body-ref-sref14" href="#" role="doc-biblioref" data-xml-rid="bib14" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref14" aria-controls="sref14">Boyd et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref14"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">14.</div><div>Boyd, A. ∙ Golding, J. ∙ Macleod, J. ...</div><div><strong>Cohort Profile: the ‘children of the 90s’--the index offspring of the Avon Longitudinal Study of Parents and Children</strong></div><div><em>Int. J. Epidemiol.</em> 2013; <strong>42</strong>:111-127</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/ije/dys064" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-84875577737" target="_blank" aria-label="Open Scopus in new tab">Scopus (1470)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22507743/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fije%2Fdys064&amp;pmid=22507743" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref33" href="#" role="doc-biblioref" data-xml-rid="bib33" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref33" aria-controls="sref33">Fraser et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref33"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">34.</div><div>Fraser, A. ∙ Macdonald-Wallis, C. ∙ Tilling, K. ...</div><div><strong>Cohort Profile: the Avon Longitudinal Study of Parents and Children: ALSPAC mothers cohort</strong></div><div><em>Int. J. Epidemiol.</em> 2013; <strong>42</strong>:97-110</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/ije/dys066" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-84875619270" target="_blank" aria-label="Open Scopus in new tab">Scopus (1066)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22507742/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fije%2Fdys066&amp;pmid=22507742" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), pattern type for each digit was scored from photocopies of the palmar surface of the hands, which were collected for the purpose of measuring digit ratio (<span class="dropBlock reference-citations"><a id="body-ref-sref78" href="#" role="doc-biblioref" data-xml-rid="bib78" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref78" aria-controls="sref78">Medland et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref78"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">85.</div><div>Medland, S.E. ∙ Zayats, T. ∙ Glaser, B. ...</div><div><strong>A variant in LIN28B is associated with 2D:4D finger-length ratio, a putative retrospective biomarker of prenatal testosterone exposure</strong></div><div><em>Am. J. Hum. Genet.</em> 2010; <strong>86</strong>:519-525</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=10.1016%2Fj.ajhg.2010.02.017&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=10.1016%2Fj.ajhg.2010.02.017&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-77950304213" target="_blank" aria-label="Open Scopus in new tab">Scopus (78)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20303062/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.ajhg.2010.02.017&amp;pmid=20303062" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Pattern type was manually coded into arch, loop, and whorl. Arches were not analyzed in ALSPAC study. Thumbs were excluded from analyses because the full patterns were not clearly visible. After initial quality control analyses, 8 variables were included in the study: the presence of whorls across all digits, except D1L and D1R digits. Please note the study website contains details of data available through a fully searchable data dictionary, <a id="intref0235" href="http://www.bristol.ac.uk/alspac/researchers/access/" target="_blank">http://www.bristol.ac.uk/alspac/researchers/access/</a>). In QIMR cohort, fingerprints were collected using rolled ink prints on paper, and or using an electronic rolled fingerprint scanner (Smiths Heimann Biometrics ACCO1394) (<span class="dropBlock reference-citations"><a id="body-ref-sref77-3" href="#" role="doc-biblioref" data-xml-rid="bib77" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref77-3" aria-controls="sref77-3">Medland et&nbsp;al., 2007</a><div class="dropBlock__holder" data-db-target-of="sref77-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">84.</div><div>Medland, S.E. ∙ Loesch, D.Z. ∙ Mdzewski, B. ...</div><div><strong>Linkage analysis of a model quantitative trait in humans: finger ridge count shows significant multivariate linkage to 5q14.1</strong></div><div><em>PLoS Genet.</em> 2007; <strong>3</strong>:1736-1744</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pgen.0030165" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_84_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-34848862129" target="_blank" aria-label="Open Scopus in new tab">Scopus (39)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17907812/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pgen.0030165&amp;pmid=17907812" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The fingerprint patterns were then manually coded. Arches were also not included in analysis. Presence of whorls across D4 and D5 were unavailable for QIMR adult sample. So, these four digits have smaller sample size than others. For the Pittsburgh cohort, fingerprints for all 10 digits were collected on paper using standard ink-based methods. Three raters independently classified each pattern as an arch, loop, or whorl. A few patterns (0.14%, from 157 people) could not be easily classified and were treated as missing data. In the discovery (TZL, NSPT, JD) and replication GWAS (CKB, WeGene), we considered fingerprint pattern as an ordinal phenotype according to the ordinal number of triradii. The same ordinal phenotype was previously used to estimate heritability (<span class="dropBlock reference-citations"><a id="body-ref-sref6" href="#" role="doc-biblioref" data-xml-rid="bib6" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref6" aria-controls="sref6">Arrieta et&nbsp;al., 1991</a><div class="dropBlock__holder" data-db-target-of="sref6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">6.</div><div>Arrieta, M.I. ∙ Salazar, L. ∙ Criado, B. ...</div><div><strong>Twin study of digital dermatoglyphic traits: Investigation of heritability</strong></div><div><em>Am. J. Hum. Biol.</em> 1991; <strong>3</strong>:11-15</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/ajhb.1310030104" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-84995256188" target="_blank" aria-label="Open Scopus in new tab">Scopus (7)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28520314/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fajhb.1310030104&amp;pmid=28520314" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref99" href="#" role="doc-biblioref" data-xml-rid="bib99" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref99" aria-controls="sref99">Reed et&nbsp;al., 1975</a><div class="dropBlock__holder" data-db-target-of="sref99"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">108.</div><div>Reed, T. ∙ Sprague, F.R. ∙ Kang, K.W. ...</div><div><strong>Genetic analysis of dermatoglyphic patterns in twins</strong></div><div><em>Hum. Hered.</em> 1975; <strong>25</strong>:263-275</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1159/000152734" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/1237463/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1159%2F000152734&amp;pmid=1237463" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). In the trans-ethnic meta-analyses, the binary phenotype according to the absence or presence of whorl type was used for comparability reason across the cohorts.</div></section><section id="sec-4-4-4"><h4>Genotyping, quality control, and imputation</h4><div id="p0225" role="paragraph">Genotyping was performed in separately in the eight cohorts (<a id="crosref1355" href="#mmc1">Table S1</a>). For TZL cohort, DNA was extracted from peripheral blood samples using GENEray™ DNA extraction kit and genotyped along with HapMap Phase 1-3 <span class="dropBlock reference-citations"><a id="body-ref-sref130" href="#" role="doc-biblioref" data-xml-rid="bib130" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref130" aria-controls="sref130">(The International HapMap Consortium, 2003)</a><div class="dropBlock__holder" data-db-target-of="sref130"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">54.</div><div>The International HapMap Consortium</div><div><strong>The International HapMap Project</strong></div><div><em>Nature.</em> 2003; <strong>426</strong>:789-796</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature02168" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-79959503826" target="_blank" aria-label="Open Scopus in new tab">Scopus (4812)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/14685227/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature02168&amp;pmid=14685227" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span> and 1000 Genomes Project (<span class="dropBlock reference-citations"><a id="body-ref-sref131" href="#" role="doc-biblioref" data-xml-rid="bib131" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref131" aria-controls="sref131">International HapMap et al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref131"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">55.</div><div>International HapMap ∙ Altshuler, C. ∙ Gibbs, D.M. ...</div><div><strong>Integrating common and rare genetic variation in diverse human populations</strong></div><div><em>Nature.</em> 2010; <strong>467</strong>:52-58</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature09298" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-77956331627" target="_blank" aria-label="Open Scopus in new tab">Scopus (1883)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20811451/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature09298&amp;pmid=20811451" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) samples for 776,213 SNPs on the Illumina HumanOmniZhongHua-8 chip. Genetic data cleaning and quality control was done using PLINK 1.9 (<span class="dropBlock reference-citations"><a id="body-ref-sref96-2" href="#" role="doc-biblioref" data-xml-rid="bib96" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref96-2" aria-controls="sref96-2">Purcell et&nbsp;al., 2007</a><div class="dropBlock__holder" data-db-target-of="sref96-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">105.</div><div>Purcell, S. ∙ Neale, B. ∙ Todd-Brown, K. ...</div><div><strong>PLINK: a tool set for whole-genome association and population-based linkage analyses</strong></div><div><em>Am. J. Hum. Genet.</em> 2007; <strong>81</strong>:559-575</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_105_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=10.1086%2F519795&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_105_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=10.1086%2F519795&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_105_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-34548292504" target="_blank" aria-label="Open Scopus in new tab">Scopus (17562)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17701901/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1086%2F519795&amp;pmid=17701901" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). In brief, samples were interrogated for sex, chromosomal aberrations, relatedness, and genotype call rate (&gt; 5% excluded). SNPs were interrogated for call rate (&gt; 5% excluded), discordance, Mendelian errors, deviations from Hardy-Weinberg equilibrium (HWE; p&nbsp;&lt; 1&nbsp;× 10<sup>−5</sup>) and differences in minor allele frequencies (MAF&nbsp;&lt; 1% excluded) and heterozygosity. The chip genotype data were first phased using SHAPEIT (<span class="dropBlock reference-citations"><a id="body-ref-sref24-2" href="#" role="doc-biblioref" data-xml-rid="bib24" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref24-2" aria-controls="sref24-2">Delaneau et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref24-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">25.</div><div>Delaneau, O. ∙ Marchini, J. ∙ Zagury, J.F.</div><div><strong>A linear complexity phasing method for thousands of genomes</strong></div><div><em>Nat. Methods.</em> 2011; <strong>9</strong>:179-181</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.1785" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-84856478855" target="_blank" aria-label="Open Scopus in new tab">Scopus (1024)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22138821/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.1785&amp;pmid=22138821" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). IMPUTE2 (<span class="dropBlock reference-citations"><a id="body-ref-sref49-2" href="#" role="doc-biblioref" data-xml-rid="bib49" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref49-2" aria-controls="sref49-2">Howie et&nbsp;al., 2009</a><div class="dropBlock__holder" data-db-target-of="sref49-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">52.</div><div>Howie, B.N. ∙ Donnelly, P. ∙ Marchini, J.</div><div><strong>A flexible and accurate genotype imputation method for the next generation of genome-wide association studies</strong></div><div><em>PLoS Genet.</em> 2009; <strong>5</strong>:e1000529</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pgen.1000529" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-67651222400" target="_blank" aria-label="Open Scopus in new tab">Scopus (2615)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19543373/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pgen.1000529&amp;pmid=19543373" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) was then used to impute unobserved variants using the 1000 Genomes Project (1KGP) Phase 3 (<span class="dropBlock reference-citations"><a id="body-ref-sref4-3" href="#" role="doc-biblioref" data-xml-rid="bib4" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref4-3" aria-controls="sref4-3">Altshuler et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref4-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">4.</div><div>Altshuler, D.M. ∙ Gibbs, R.A. ∙ Peltonen, L. ..., International HapMap 3 Consortium</div><div><strong>Integrating common and rare genetic variation in diverse human populations</strong></div><div><em>Nature.</em> 2010; <strong>467</strong>:52-58</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature09298" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-77956331627" target="_blank" aria-label="Open Scopus in new tab">Scopus (1889)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20811451/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature09298&amp;pmid=20811451" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) as the reference. SNPs with an imputation quality scores (INFO) less than 0.8, MAF less than 1% or a missing rate of more than 2% of genotypes were eliminated from further analyses. Finally, a total of 7,057,720 SNPs from 2961 individuals were passed quality control and were used for further analyses.</div><div id="p0230" role="paragraph">For NSPT and JD cohorts, genomic DNA was extracted from blood samples using the MagPure Blood DNA KF Kit. All samples were genotyped using the Illumina Infinium Global Screening Array that analyzes over 710,000 SNPs. It is a fully custom array designed by WeGene (<a id="intref0240" href="https://www.wegene.com/" target="_blank">https://www.wegene.com/</a>). Genetic data cleaning and quality control was done using PLINK v1.9 (<span class="dropBlock reference-citations"><a id="body-ref-sref96-3" href="#" role="doc-biblioref" data-xml-rid="bib96" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref96-3" aria-controls="sref96-3">Purcell et&nbsp;al., 2007</a><div class="dropBlock__holder" data-db-target-of="sref96-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">105.</div><div>Purcell, S. ∙ Neale, B. ∙ Todd-Brown, K. ...</div><div><strong>PLINK: a tool set for whole-genome association and population-based linkage analyses</strong></div><div><em>Am. J. Hum. Genet.</em> 2007; <strong>81</strong>:559-575</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_105_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=10.1086%2F519795&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_105_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=10.1086%2F519795&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_105_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-34548292504" target="_blank" aria-label="Open Scopus in new tab">Scopus (17562)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17701901/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1086%2F519795&amp;pmid=17701901" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). We excluded subjects with more than 5% missing data, duplicated subjects, and subject samples that failed the X chromosome gender concordance check. We excluded SNPs that had &gt; 2% missing data, MAF&nbsp;&lt; 1%, or a deviation from Hardy-Weinberg equilibrium (p&nbsp;&lt; 1&nbsp;× 10<sup>−5</sup>), leaving 707,146 SNPs from 4269 individuals in JD cohort and 2679 individuals in NSPT cohort for further analyses. Imputation of unobserved variants was performed using haplotypes from the 1000 Genomes Project Phase 3 as the reference. The chip genotype data was first phased using SHAPEIT. IMPUTE2 was then used to impute genotypes. SNPs with an imputation quality scores (INFO) less than 0.6, MAF&nbsp;&lt; 1% or a missing rate &gt; 1% of genotypes were eliminated from further analyses. Finally, a total of 8,039,700 SNPs were passed quality control and were used for further analyses.</div><div id="p0235" role="paragraph">The genotyping of the CKB cohort was performed using custom-designed 800K-SNP Affymetrix Axiom arrays (Axiom_CKB_1 and Axiom_CKB_2). Genetic data cleaning and QC procedures utilized PLINK v1.9. Subjects of non-East Asian ancestry, mismatch with reported gender, with genotype heterozygosity &gt; 3&nbsp;× SDs from the mean and X/Y aneuploidy were excluded. The selected SNPs are of &gt; 98% call rate and with less than 20% allele frequency difference from the 1KGP East Asian population. SNPs with HWE p&nbsp;&lt; 1&nbsp;× 10<sup>−6</sup> were manually examined. The hard-genotyped data were prephased using SHAPEIT3 and imputed using IMPUTE4. SNPs with 0 MAF in the 1KGP East Asian population were ignored.</div><div id="p0240" role="paragraph">For WeGene cohort, DNA extraction and genotyping were performed on saliva samples. Genotyping was performed on the Affymetrix WeGene V1 Arrays covering 596,744 SNPs at the WeGene genotyping center, Shenzhen. Quality control (QC) was performed in PLINK V1.07. Ancestry was assigned using self-reported ancestry obtained from customer surveys and further checked with principal components analysis. Unrelated filtering was done by checking pair wisely for all the samples and where identity by descent (IBD) scores of &gt; 0.125 (3<sup>rd</sup>-degree relative) were identified with one from each such pair removed. Individuals with discordant sex information were removed. The individuals with genotype call rate of&nbsp;&lt; 95% and outlying heterozygous rate were excluded. To minimize the influence of bias, the following SNP were discarded: (1) sites with unbalanced call rate in case and controls, (2) sites that failed the Hardy-Weinberg Equilibrium test (p&nbsp;&lt; 1&nbsp;× 10<sup>−5</sup>), and (3) MAF&nbsp;&lt; 1%. The 1000 Genomes Project Phase 3 data was used as imputation reference panel. Phasing and imputation on the autosomes were carried out using SHAPEIT2 and IMPUTE2. Post-imputation filtering was done by removing SNPs with imputation quality scores (INFO) less than 0.5, MAF less than 1% or missing rate more than 2%. The combined application of these filters left us with a dataset of 7,124,171 SNPs and these dataset was used for further analyses.</div><div id="p0245" role="paragraph">ALSPAC participants were genotyped using the Illumina HumanHap550 quad genome-wide SNP genotyping platform by the Wellcome Trust Sanger Institute, Cambridge, UK and the Laboratory Corporation of America, Burlington, NC, US. Genotype data were cleaned using standard thresholds (SNPs excluded if MAF&nbsp;&lt; 1%, call rate&nbsp;&lt; 95% and P value from an exact test of Hardy-Weinberg equilibrium&nbsp;&lt; 5&nbsp;× 10<sup>−7</sup>). Individual samples were excluded on the basis of incorrect sex assignment, minimal or excessive heterozygosity, and high levels of missingness or cryptic relatedness. We combined child’s genotypes with cleaned genome-wide SNP data from 9,048 ALSPAC mothers (<span class="dropBlock reference-citations"><a id="body-ref-sref30" href="#" role="doc-biblioref" data-xml-rid="bib30" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref30" aria-controls="sref30">Fatemifar et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref30"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">31.</div><div>Fatemifar, G. ∙ Hoggart, C.J. ∙ Paternoster, L. ...</div><div><strong>Genome-wide association study of primary tooth eruption identifies pleiotropic loci associated with height and craniofacial distances</strong></div><div><em>Hum. Mol. Genet.</em> 2013; <strong>22</strong>:3807-3817</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/hmg/ddt231" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-84883035112" target="_blank" aria-label="Open Scopus in new tab">Scopus (52)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23704328/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fhmg%2Fddt231&amp;pmid=23704328" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and removed subjects due to potential sample mismatches. ALSPAC samples were imputed using the Hapmap2 r22.36 CEU reference. Single nucleotide polymorphisms (SNPs) that had a MAF &gt; 0.01 and could be imputed with confidence (R<sup>2</sup> &gt; 0.3) were used in these analyses.</div><div id="p0250" role="paragraph">Participants in the QIMR cohort were genotyped on the Illumina Human610-Quad SNP chip. These samples were genotyped in the context of a larger genome-wide association project that resulted in the genotyping of 28,028 individuals59 using the Illumina 317, 370, 610, 660, Core+Exome, PsychChip, Omni2.5 and OmniExpress SNP chips which included data from twins, their siblings and their parents. Genotype data were screened for genotyping quality (&lt; 0.7), SNP and individual call rates (&lt; 0.95), HWE failure (p&nbsp;&lt; 1&nbsp;× 10<sup>−6</sup>) and MAF (&lt; 0.01). As these samples were genotyped in the context of a larger project, the data were integrated with the larger QIMR genotype project and the data were checked for pedigree, sex and Mendelian errors and for non-European ancestry. QIMR samples were imputed using the Hapmap2 r22.36 CEU reference. Single nucleotide polymorphisms (SNPs) that had a minor allele frequency &gt; 0.01 and could be imputed with confidence (R<sup>2</sup> &gt; 0.3) were used in these analyses.</div><div id="p0255" role="paragraph">For the Pittsburgh cohort, DNA samples primarily from blood or saliva were assayed for 557,577 SNP genotypes (including 15,890 SNPs of custom content) with the Illumina HumanCore+Exome platform at the Center for Inherited Disease Research. The University of Washington Genetics Coordinating Center performed cleaning and quality control analysis, which included investigation of genetic sex, chromosomal anomalies, relatedness, call rate, and batch effects. 455,449 SNPs initially passed filters that included call rate, deviation from Hardy-Weinberg equilibrium, difference by sex in allele frequency or heterozygosity, discordance in duplicate samples, and Mendelian errors in controls. Genotypes for 34.4 million additional SNPs were imputed to the 1000 Genomes Project Phase 3 worldwide reference panel using IMPUTE2 after phasing with SHAPEIT. Imputed SNPs were filtered out for low info score (&lt; 0.5), evidence of extreme deviation from HWE, and low MAF (&lt; 5%). Individual genotypes with low probability (&lt; 0.9) were treated as missing. After filtering, about 6.7 million SNPs were available for GWAS.</div></section><section id="sec-4-4-5"><h4>Population stratification analysis</h4><div id="p0260" role="paragraph">We corrected the effects of possible population stratification within cohort using EIGENSTRAT (<span class="dropBlock reference-citations"><a id="body-ref-sref95-2" href="#" role="doc-biblioref" data-xml-rid="bib95" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref95-2" aria-controls="sref95-2">Price et&nbsp;al., 2006</a><div class="dropBlock__holder" data-db-target-of="sref95-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">102.</div><div>Price, A.L. ∙ Patterson, N.J. ∙ Plenge, R.M. ...</div><div><strong>Principal components analysis corrects for stratification in genome-wide association studies</strong></div><div><em>Nat. Genet.</em> 2006; <strong>38</strong>:904-909</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng1847" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_102_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-33746512512" target="_blank" aria-label="Open Scopus in new tab">Scopus (6332)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16862161/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng1847&amp;pmid=16862161" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) utility from the EIGENSOFT package. 102,284 SNPs in low linkage equilibrium (r<sup>2</sup>&nbsp;&lt; 0.2) were selected for analysis. TZL cohort was compared with YRI, CHB, and CEU from 1000 Genomes Phase 3 (<span class="dropBlock reference-citations"><a id="body-ref-sref4-4" href="#" role="doc-biblioref" data-xml-rid="bib4" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref4-4" aria-controls="sref4-4">Altshuler et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref4-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">4.</div><div>Altshuler, D.M. ∙ Gibbs, R.A. ∙ Peltonen, L. ..., International HapMap 3 Consortium</div><div><strong>Integrating common and rare genetic variation in diverse human populations</strong></div><div><em>Nature.</em> 2010; <strong>467</strong>:52-58</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature09298" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-77956331627" target="_blank" aria-label="Open Scopus in new tab">Scopus (1889)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20811451/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature09298&amp;pmid=20811451" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and principal component (PC) analysis did not find any outliers. Here, we also adjusted top 4 PCs to further avoid inflation from different genetic background. The similar criteria were used in NSPT and JD cohorts and adjusted top 5 PCs for further analyses and without any inflation.</div><div id="p0265" role="paragraph">PCs of CKB (and 1KGP) subjects were obtained using the PLINK implementation of the GCTA algorithm (<span class="dropBlock reference-citations"><a id="body-ref-sref125-2" href="#" role="doc-biblioref" data-xml-rid="bib125" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref125-2" aria-controls="sref125-2">Yang et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref125-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">135.</div><div>Yang, J. ∙ Lee, S.H. ∙ Goddard, M.E. ...</div><div><strong>GCTA: a tool for genome-wide complex trait analysis</strong></div><div><em>Am. J. Hum. Genet.</em> 2011; <strong>88</strong>:76-82</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_135_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=10.1016%2Fj.ajhg.2010.11.011&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_135_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=10.1016%2Fj.ajhg.2010.11.011&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_135_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-78650856517" target="_blank" aria-label="Open Scopus in new tab">Scopus (3187)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21167468/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.ajhg.2010.11.011&amp;pmid=21167468" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). After the removal of high-LD regions, 142,165 SNPs with low linkage equilibrium (r<sup>2</sup>&nbsp;&lt; 0.2) were selected for the PCA of CKB. We used the combination of CKB and 1KGP for the detection of CKB subjects of non-East Asian ancestry. Eight PCs were employed for the further association studies.</div><div id="p0270" role="paragraph">For WeGene cohort, we selected 308 unrelated samples from the YRI, CEU and CHB populations (1KGP Phase 3), and then chose SNPs using following criteria: (1) MAF ≥ 0.05 and HWE p &gt; 10<sup>−6</sup>, in each of the populations YRI, CEU and CHB, (2) pairwise r<sup>2</sup> ≤ 0.1 to exclude SNPs in high LD (calculated using PLINK indep-pairwise function with a step window of size 1000&nbsp;bp), (3) remove C/G and A/T SNPs to avoid unresolvable strand mismatches. With the remaining 38,144 SNPs, we computed PCA for the combined samples. Ten PCs were employed in further association studies.</div><div id="p0275" role="paragraph">For Pittsburgh cohort, participants comprised a subset from a large, multiethnic study. PCs of ancestry were calculated for all participants in the larger study and then projected for the subset included here. Because there was no outlier, association studies were performed without adjusted genetic PCs.</div><div id="p0280" role="paragraph">As there was no evidence of systematic inflation in the ALSPAC dataset (λ&nbsp;= 1.007–1.034), results were not corrected for ancestry informative PCs.</div><div id="p0285" role="paragraph">For QIMR cohorts, EigenSoft (version 6.0.1) was used to perform principal component analysis. The QIMR data were combined with Genome-EUTWIN and HapMap Phase 3 populations. The EUTWIN and HapMap populations were used to produce the internal axes, and PCA coordinates were calculated on those axes for those populations and the QIMR participants. Individuals more than 6 standard deviations from the PC1 or PC2 means of European populations were excluded and we included the first 4 PCs as covariates to further avoid inflation from and residual population related effects in QIMR cohort.</div></section><section id="sec-4-4-6"><h4>Association analyses</h4><div id="p0290" role="paragraph">Genome-wise association analyses were conducted separately in TZL, NSPT, JD, CKB, ALSPAC, QIMR, Pittsburgh, and WeGene cohorts and sex was adjusted for in all analyses (<a id="crosref1405" href="#mmc1">Table S1</a>). Initial genome-wide association analyses on ordinal phenotype (coded as 0, 1, and 2 for arch, loop, and whorl, respectively) were performed in PLINK 1.9 (<span class="dropBlock reference-citations"><a id="body-ref-sref96-4" href="#" role="doc-biblioref" data-xml-rid="bib96" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref96-4" aria-controls="sref96-4">Purcell et&nbsp;al., 2007</a><div class="dropBlock__holder" data-db-target-of="sref96-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">105.</div><div>Purcell, S. ∙ Neale, B. ∙ Todd-Brown, K. ...</div><div><strong>PLINK: a tool set for whole-genome association and population-based linkage analyses</strong></div><div><em>Am. J. Hum. Genet.</em> 2007; <strong>81</strong>:559-575</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_105_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=10.1086%2F519795&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_105_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=10.1086%2F519795&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_105_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-34548292504" target="_blank" aria-label="Open Scopus in new tab">Scopus (17562)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17701901/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1086%2F519795&amp;pmid=17701901" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), using multiple linear regression model of additive allelic effects with additional 4 genetic PCs, 5 genetic PCs, and 5 genetic PCs as covariates in TZL, NSPT, and JD cohorts, respectively. Then we conducted genome-wide association analyses on different variety and derived phenotypes (e.g., binary, nominal, ordinal, and quantitative phenotypes), using a linear or logistic regression model also incorporating sex and 4 genetic PCs as covariates. All Manhattan plots and quantile-quantile plots were created using <i>qqman</i> package in R software (<span class="dropBlock reference-citations"><a id="body-ref-sref114-2" href="#" role="doc-biblioref" data-xml-rid="bib114" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref114-2" aria-controls="sref114-2">Turner, 2014</a><div class="dropBlock__holder" data-db-target-of="sref114-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">124.</div><div>Turner, S.D.</div><div><strong>qqman: an R package for visualizing GWAS results using QQ and manhattan plots</strong></div><div><em>bioRxiv.</em> 2014;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/005165" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2F005165" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Quantile-quantile plots were used for all association tests to assess systematic inflation from population stratification or other systematic causes of bias. The genomic control factor <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="lamda" tabindex="0" ctxtmenu_counter="1" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mtext data-semantic-type="text" data-semantic-role="unknown" data-semantic-font="normal" data-semantic-annotation="clearspeak:unit" data-semantic-id="0" data-semantic-speech="lamda"><mjx-c class="mjx-c3BB">λ</mjx-c></mjx-mtext></mjx-mrow></mjx-math></mjx-container> in all tests did not show any sign of inflation (&lt; 1.03, <a id="crosref1420" href="#mmc1">Table S1</a>). The narrow-sense heritability of fingerprint patterns was estimated using GCTA and then estimated the contribution of SNPs to phenotypic variance of ordinal fingerprint patterns using linear regression model (R<sup>2</sup>). GWAS within in CKB were performed using BOLT-LMM v.2.3.2 with the linear mixture model (<span class="dropBlock reference-citations"><a id="body-ref-sref137-2" href="#" role="doc-biblioref" data-xml-rid="bib137" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref137-2" aria-controls="sref137-2">Loh et al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref137-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">75.</div><div>Loh, P.R. ∙ Tucker, G. ∙ Bulik-Sullivan, B.K. ...</div><div><strong>Efficient Bayesian mixed-model analysis increases association power in large cohorts</strong></div><div><em>. Nat Genet.</em> 2015; <strong>47</strong>:284-290</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng.3190" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_75_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-84924060689" target="_blank" aria-label="Open Scopus in new tab">Scopus (693)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25642633/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng.3190&amp;pmid=25642633" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The covariates used are gender, regions, genotyping array versions and the 8 genetic PCs. For WeGene cohort, genome-wide association analyses were performed with PLINK 1.07 combined with top 10 principal components (PC1-10) generated from GCTA. A genomic inflation factor was generated on the basis of the χ2-values obtained from PLINK results using R programming (&lt; 1.021). Genome-wide analyses were conducted for each digit in each cohort using merlin-offline (QIMR) or Mach2dat (ALSPAC) (<span class="dropBlock reference-citations"><a id="body-ref-sref1-2" href="#" role="doc-biblioref" data-xml-rid="bib1" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref1-2" aria-controls="sref1-2">Abecasis et&nbsp;al., 2002</a><div class="dropBlock__holder" data-db-target-of="sref1-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">1.</div><div>Abecasis, G.R. ∙ Cherny, S.S. ∙ Cookson, W.O. ...</div><div><strong>Merlin--rapid analysis of dense genetic maps using sparse gene flow trees</strong></div><div><em>Nat. Genet.</em> 2002; <strong>30</strong>:97-101</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng786" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_1_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-0036338150" target="_blank" aria-label="Open Scopus in new tab">Scopus (2712)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11731797/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng786&amp;pmid=11731797" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref138-2" href="#" role="doc-biblioref" data-xml-rid="bib138" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref138-2" aria-controls="sref138-2">Li et al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref138-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">71.</div><div>Li, Y. ∙ Willer, C.J. ∙ Ding, J. ...</div><div><strong>MaCH: using sequence and genotype data to estimate haplotypes and unobserved genotypes</strong></div><div><em>Genet Epidemiol.</em> 2010; <strong>34</strong>:816-834</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/gepi.20533" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-78649508578" target="_blank" aria-label="Open Scopus in new tab">Scopus (1447)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21058334/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fgepi.20533&amp;pmid=21058334" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Both studies adjusted for sex, with the former also adjusting for 4 genetic PCs. Information in details have been described in a previous study (<span class="dropBlock reference-citations"><a id="body-ref-sref46-3" href="#" role="doc-biblioref" data-xml-rid="bib46" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref46-3" aria-controls="sref46-3">Ho et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref46-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">49.</div><div>Ho, Y.Y.W. ∙ Evans, D.M. ∙ Montgomery, G.W. ...</div><div><strong>Common Genetic Variants Influence Whorls in Fingerprint Patterns</strong></div><div><em>J.&nbsp;Invest. Dermatol.</em> 2016; <strong>136</strong>:859-862</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=10.1016%2Fj.jid.2015.10.062&amp;cf=fulltext&amp;site=jid-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=10.1016%2Fj.jid.2015.10.062&amp;cf=pdf&amp;site=jid-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-84978086948" target="_blank" aria-label="Open Scopus in new tab">Scopus (11)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27045867/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.jid.2015.10.062&amp;pmid=27045867" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). For Pittsburgh cohort, all association analyses were performed with the linear mixed model software EMMAX (<span class="dropBlock reference-citations"><a id="body-ref-sref51-2" href="#" role="doc-biblioref" data-xml-rid="bib51" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref51-2" aria-controls="sref51-2">Kang et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref51-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">56.</div><div>Kang, H.M. ∙ Sul, J.H. ∙ Service, S.K. ...</div><div><strong>Variance component model to account for sample structure in genome-wide association studies</strong></div><div><em>Nat. Genet.</em> 2010; <strong>42</strong>:348-354</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng.548" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_56_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-77950301214" target="_blank" aria-label="Open Scopus in new tab">Scopus (1414)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20208533/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng.548&amp;pmid=20208533" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), which tests the additive effect of each allele while using the subjects’ kinship matrix to account for relatedness and ancestry. Sex was included as a covariate. For each of the ten fingers, both an ordinal phenotype (coded as 0, 1, and 2 for arch, loop, and whorl, respectively) and a binary phenotype (presence versus absence of a whorl) were analyzed. Strong evidence of inflation was not observed (&lt; 1.018).</div></section><section id="sec-4-4-7"><h4>Meta-analyses</h4><div id="p0295" role="paragraph">To avoid batch effects from different SNP chip products, meta-analyses were performed in the discovery stage. Then to maximize the statistical power to detect associated genetic variants of small effect, we conducted trans-ethnic meta-analyses for TZL, NSPT, JD, WeGene, CKB, ALSPAC, QIMR, and Pittsburgh cohorts based on the summary results of binary phenotypes (presence versus absence of a whorl). GWAS results based on binary phenotype for each study were combined via sample-size-weighted fixed-effects analysis using METAL (<span class="dropBlock reference-citations"><a id="body-ref-sref122-2" href="#" role="doc-biblioref" data-xml-rid="bib122" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref122-2" aria-controls="sref122-2">Willer et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref122-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">132.</div><div>Willer, C.J. ∙ Li, Y. ∙ Abecasis, G.R.</div><div><strong>METAL: fast and efficient meta-analysis of genomewide association scans</strong></div><div><em>Bioinformatics.</em> 2010; <strong>26</strong>:2190-2191</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btq340" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_132_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-77955894071" target="_blank" aria-label="Open Scopus in new tab">Scopus (2264)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20616382/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtq340&amp;pmid=20616382" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). In addition, the heterogeneity of the associations across the different cohorts was assessed by the I<sup>2</sup> and Cochran’s Q statistics as reported by METAL. For SNPs with significant heterogeneity, a random effects model was applied for meta-analysis using METASOFT (<span class="dropBlock reference-citations"><a id="body-ref-sref42-2" href="#" role="doc-biblioref" data-xml-rid="bib42" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref42-2" aria-controls="sref42-2">Han and Eskin, 2011</a><div class="dropBlock__holder" data-db-target-of="sref42-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">44.</div><div>Han, B. ∙ Eskin, E.</div><div><strong>Random-effects model aimed at discovering associations in meta-analysis of genome-wide association studies</strong></div><div><em>Am. J. Hum. Genet.</em> 2011; <strong>88</strong>:586-598</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=10.1016%2Fj.ajhg.2011.04.014&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=10.1016%2Fj.ajhg.2011.04.014&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-79955860273" target="_blank" aria-label="Open Scopus in new tab">Scopus (297)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21565292/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.ajhg.2011.04.014&amp;pmid=21565292" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div></section><section id="sec-4-4-8"><h4>Multiple-testing corrections</h4><div id="p0300" role="paragraph">Given the burden of multiple comparisons, a strict significance threshold of p&nbsp;&lt; 5&nbsp;× 10<sup>−8</sup> was used to declare ‘genome-wide significance’, which corresponds to a Bonferroni correction for 1 million independent tests. Given that we tested fingerprint patterns variation as one of many derived phenotypes separately, the multiple-comparisons burden was magnified. Therefore, we also determined a more stringent threshold for declaring ‘study-wide significance’ (<span class="dropBlock reference-citations"><a id="body-ref-sref20" href="#" role="doc-biblioref" data-xml-rid="bib20" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref20" aria-controls="sref20">Claes et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref20"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">20.</div><div>Claes, P. ∙ Roosenboom, J. ∙ White, J.D. ...</div><div><strong>Genome-wide mapping of global-to-local genetic effects on human facial shape</strong></div><div><em>Nat. Genet.</em> 2018; <strong>50</strong>:414-423</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41588-018-0057-4" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-85042196407" target="_blank" aria-label="Open Scopus in new tab">Scopus (89)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29459680/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41588-018-0057-4&amp;pmid=29459680" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) corresponding to an additional adjustment for the effective number of independent test (<span class="dropBlock reference-citations"><a id="body-ref-sref64" href="#" role="doc-biblioref" data-xml-rid="bib64" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref64" aria-controls="sref64">Li and Ji, 2005</a><div class="dropBlock__holder" data-db-target-of="sref64"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">69.</div><div>Li, J. ∙ Ji, L.</div><div><strong>Adjusting multiple testing in multilocus analyses using the eigenvalues of a correlation matrix</strong></div><div><em>Heredity.</em> 2005; <strong>95</strong>:221-227</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/sj.hdy.6800717" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-30744434862" target="_blank" aria-label="Open Scopus in new tab">Scopus (913)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16077740/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fsj.hdy.6800717&amp;pmid=16077740" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The eigenvalues of pairwise multivariate corrections of 10 ordinal/binary phenotypes on ten fingers determined a total of 3 effective independent tests (the number of eigenvalues greater than 1 is the number of independent tests) in the discovery cohorts with available phenotype data. Therefore, the study-wide significance threshold was determined to be 1.67&nbsp;× 10<sup>−8</sup> (i.e., 5&nbsp;× 10<sup>−8</sup>/3). The same threshold applied to meta-analyses when binary phenotype was used. In addition, when 66 different derived phenotypes were further tested, they were also not strictly independent to each other and the eigenvalues of pairwise multivariate corrections of these phenotypes determined 14 effective independent tests. Therefore, the “study-wide significance” threshold for the association tests of the 66 different derived phenotypes was determined to be 3.57&nbsp;× 10<sup>−9</sup> (i.e., 5&nbsp;× 10<sup>−8</sup>/14).</div></section><section id="sec-4-4-9"><h4>Fine-mapping credible set analysis</h4><div id="p0305" role="paragraph">First, we created a 3q26.2 signal (GWA SNPs) space including SNPs with p&nbsp;&lt; 5&nbsp;× 10<sup>−8</sup> associated with composite phenotype. The online tool HaploReg (V4.1) was used to explore the function, chromatin states and the nearest genes for each SNP in the signal space. These SNPs were located in non-coding region and some of them showed potential regulatory function annotated in the reference epigenomes of 127 human tissues and cell types obtained from the NIH Roadmap Epigenomics Mapping Consortium (<span class="dropBlock reference-citations"><a id="body-ref-sref12-2" href="#" role="doc-biblioref" data-xml-rid="bib12" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref12-2" aria-controls="sref12-2">Bernstein et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref12-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">12.</div><div>Bernstein, B.E. ∙ Stamatoyannopoulos, J.A. ∙ Costello, J.F. ...</div><div><strong>The NIH Roadmap Epigenomics Mapping Consortium</strong></div><div><em>Nat. Biotechnol.</em> 2010; <strong>28</strong>:1045-1048</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nbt1010-1045" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-77957940722" target="_blank" aria-label="Open Scopus in new tab">Scopus (1125)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20944595/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnbt1010-1045&amp;pmid=20944595" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Then we performed fine mapping analysis to detect the potential causal variants for a 500 kb genomic interval flanking the top SNP (250 kb upstream and 250 kb downstream) of 3q26.2 locus using PAINTOR (<span class="dropBlock reference-citations"><a id="body-ref-sref56-2" href="#" role="doc-biblioref" data-xml-rid="bib56" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref56-2" aria-controls="sref56-2">Kichaev et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref56-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">61.</div><div>Kichaev, G. ∙ Yang, W.Y. ∙ Lindstrom, S. ...</div><div><strong>Integrating functional data to prioritize causal variants in statistical fine-mapping studies</strong></div><div><em>PLoS Genet.</em> 2014; <strong>10</strong>:e1004722</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pgen.1004722" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-84908324508" target="_blank" aria-label="Open Scopus in new tab">Scopus (227)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25357204/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pgen.1004722&amp;pmid=25357204" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). For each SNP within the 500-kb window, we calculated the posterior probability of driving the association, and then constructed 99% credibility set. We created credibility sets by using combined TZL, NSPT, and JD datasets. We also integrated the linkage disequilibrium information and functional annotation data including seven highlighted epigenomic marks (H3K4me3, H3K4me1, H3K36me3, H3K27me3, H3K9me3, H3K27ac and H3K9ac) for 12 epithelial or mesenchymal cell types (from ectoderm and mesoderm, possibly fingerprint-relevant types) of 127 human tissues and cell types above. Each annotation data was entering the model independently according to the suggested pipeline. The top one (with the highest sum log bayes factors) was selected for further analyses to compute trait-specific posterior probabilities for causality. A 99% credible set was then constructed by variants (going down the sorted list by posterior probability) whose cumulative posterior probability of representing the causal variant at each locus exceeded 0.99.</div></section><section id="sec-4-4-10"><h4>Gene mapping, functional annotation and genomic enrichment analyses</h4><div id="p0310" role="paragraph">The top SNPs found at association loci were used to query the evidence of the candidate genes based on physical distance, biological pathways, the tissue location of expression and whether other traits affected by the mutations in these genes in NCBI (<span class="dropBlock reference-citations"><a id="body-ref-sref106-2" href="#" role="doc-biblioref" data-xml-rid="bib106" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref106-2" aria-controls="sref106-2">Sherry et&nbsp;al., 2001</a><div class="dropBlock__holder" data-db-target-of="sref106-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">115.</div><div>Sherry, S.T. ∙ Ward, M.-H. ∙ Kholodov, M. ...</div><div><strong>dbSNP: the NCBI database of genetic variation</strong></div><div><em>Nucleic Acids Res.</em> 2001; <strong>29</strong>:308-311</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/29.1.308" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11125122/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2F29.1.308&amp;pmid=11125122" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), UCSC genome browser (<span class="dropBlock reference-citations"><a id="body-ref-sref34-2" href="#" role="doc-biblioref" data-xml-rid="bib34" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref34-2" aria-controls="sref34-2">Fujita et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref34-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">35.</div><div>Fujita, P.A. ∙ Rhead, B. ∙ Zweig, A.S. ...</div><div><strong>The UCSC Genome Browser database: update 2011</strong></div><div><em>Nucleic Acids Res.</em> 2011; <strong>39</strong>:D876-D882</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gkq963" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-78651320424" target="_blank" aria-label="Open Scopus in new tab">Scopus (815)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20959295/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgkq963&amp;pmid=20959295" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and Ensemble genome browser 89 (<span class="dropBlock reference-citations"><a id="body-ref-sref2-2" href="#" role="doc-biblioref" data-xml-rid="bib2" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref2-2" aria-controls="sref2-2">Aken et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref2-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">2.</div><div>Aken, B.L. ∙ Ayling, S. ∙ Barrell, D. ...</div><div><strong>The Ensembl gene annotation system</strong></div><div><em>Database (Oxford).</em> 2016; <strong>2016</strong>:baw093</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/database/baw093" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27337980/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fdatabase%2Fbaw093&amp;pmid=27337980" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). We used HaploReg V4.1(<span class="dropBlock reference-citations"><a id="body-ref-sref119-2" href="#" role="doc-biblioref" data-xml-rid="bib119" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref119-2" aria-controls="sref119-2">Ward and Kellis, 2012</a><div class="dropBlock__holder" data-db-target-of="sref119-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">129.</div><div>Ward, L.D. ∙ Kellis, M.</div><div><strong>HaploReg: a resource for exploring chromatin states, conservation, and regulatory motif alterations within sets of genetically linked variants</strong></div><div><em>Nucleic Acids Res.</em> 2012; <strong>40</strong>:D930-D934</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gkr917" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_129_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-84858779229" target="_blank" aria-label="Open Scopus in new tab">Scopus (1526)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22064851/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgkr917&amp;pmid=22064851" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), an integrative database browser combined of histone modification (ChIP-seq tracks) and ChIA-PET (Chromatin Interaction) from ENCODE (<span class="dropBlock reference-citations"><a id="body-ref-sref135-2" href="#" role="doc-biblioref" data-xml-rid="bib135" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref135-2" aria-controls="sref135-2">Davis et al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref135-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">24.</div><div>Davis, C.A. ∙ Hitz, B.C. ∙ Sloan, C.A. ...</div><div><strong>The Encyclopedia of DNA elements (ENCODE): data portal update</strong></div><div><em>Nucleic Acids Res.</em> 2018; <strong>46</strong>:D794-D801</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gkx1081" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-85040942752" target="_blank" aria-label="Open Scopus in new tab">Scopus (573)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29126249/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgkx1081&amp;pmid=29126249" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and Roadmap Epigenomics Project, eQTLs from GTEx (<span class="dropBlock reference-citations"><a id="body-ref-sref134-2" href="#" role="doc-biblioref" data-xml-rid="bib134" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref134-2" aria-controls="sref134-2">GTEx Consortium, 2017</a><div class="dropBlock__holder" data-db-target-of="sref134-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">42.</div><div>GTEx Consortium</div><div><strong>Genetic effects on gene expression across human tissues</strong></div><div><em>Nature.</em> 2017; <strong>550</strong>:204-213</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature24277" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-85031313737" target="_blank" aria-label="Open Scopus in new tab">Scopus (1546)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29022597/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature24277&amp;pmid=29022597" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), and conserved regions from GERP and Phastcons to identify more regulatory annotations of genetic variants. RefSeq (<span class="dropBlock reference-citations"><a id="body-ref-sref132-2" href="#" role="doc-biblioref" data-xml-rid="bib132" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref132-2" aria-controls="sref132-2">Pruitt et al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref132-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">104.</div><div>Pruitt, K.D. ∙ Brown, G.R. ∙ Hiatt, S.M. ...</div><div><strong>RefSeq: an update on mammalian reference sequences</strong></div><div><em>Nucleic Acids Res.</em> 2014; <strong>42</strong>:D756-D763</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gkt1114" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_104_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-84891767394" target="_blank" aria-label="Open Scopus in new tab">Scopus (651)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24259432/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgkt1114&amp;pmid=24259432" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and GENCODE v19 (<span class="dropBlock reference-citations"><a id="body-ref-sref133-2" href="#" role="doc-biblioref" data-xml-rid="bib133" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref133-2" aria-controls="sref133-2">Harrow et al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref133-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">45.</div><div>Harrow, J. ∙ Frankish, A. ∙ Gonzalez, J.M. ...</div><div><strong>GENCODE: the reference human genome annotation for The ENCODE Project</strong></div><div><em>Genome research.</em> 2012; <strong>22</strong>:1760-1774</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/gr.135350.111" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_45_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-84865760395" target="_blank" aria-label="Open Scopus in new tab">Scopus (2415)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22955987/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2Fgr.135350.111&amp;pmid=22955987" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) databases were used to obtain gene annotations. 3D genome browser (<a id="intref0245" href="http://3dgenome.fsm.northwestern.edu" target="_blank">http://3dgenome.fsm.northwestern.edu</a>) were used to visualize the chromatin interaction of genome (<span class="dropBlock reference-citations"><a id="body-ref-sref118" href="#" role="doc-biblioref" data-xml-rid="bib118" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref118" aria-controls="sref118">Wang et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref118"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">128.</div><div>Wang, Y. ∙ Song, F. ∙ Zhang, B. ...</div><div><strong>The 3D Genome Browser: a web-based browser for visualizing 3D genome organization and long-range chromatin interactions</strong></div><div><em>Genome Biol.</em> 2018; <strong>19</strong>:151</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13059-018-1519-9" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_128_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-85054460140" target="_blank" aria-label="Open Scopus in new tab">Scopus (171)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30286773/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13059-018-1519-9&amp;pmid=30286773" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Further, we performed genomic enrichment analyses by using the Genomic Regions Enrichment of Annotation Tool (GREAT, abbreviated as G) to identify whether the mapped genes (<a id="crosref1495" href="#mmc5">Table S5</a>) nearby both coding and non-coding genomic regions within 1000kb of top SNPs were enriched with relevant annotations across GO biological processes, human or mouse morphology, and gene expression (<span class="dropBlock reference-citations"><a id="body-ref-sref76-3" href="#" role="doc-biblioref" data-xml-rid="bib76" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref76-3" aria-controls="sref76-3">McLean et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref76-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">83.</div><div>McLean, C.Y. ∙ Bristor, D. ∙ Hiller, M. ...</div><div><strong>GREAT improves functional interpretation of cis-regulatory regions</strong></div><div><em>Nat. Biotechnol.</em> 2010; <strong>28</strong>:495-501</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nbt.1630" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_83_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-77952214662" target="_blank" aria-label="Open Scopus in new tab">Scopus (2170)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20436461/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnbt.1630&amp;pmid=20436461" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Gene regulatory domains utilized for region annotation were defined as the two nearest genes, and extended up to 1000 kb to the nearest gene’s Transcription Start Site (‘Two nearest genes’ option). The Gene Ontology (GO) Biological Process had 13,145 terms (significant threshold after Bonferroni correction 3.80&nbsp;× 10<sup>−6</sup>) (<span class="dropBlock reference-citations"><a id="body-ref-sref8" href="#" role="doc-biblioref" data-xml-rid="bib8" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref8" aria-controls="sref8">Ashburner et&nbsp;al., 2000</a><div class="dropBlock__holder" data-db-target-of="sref8"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">8.</div><div>Ashburner, M. ∙ Ball, C.A. ∙ Blake, J.A. ..., The Gene Ontology Consortium</div><div><strong>Gene ontology: tool for the unification of biology</strong></div><div><em>Nat. Genet.</em> 2000; <strong>25</strong>:25-29</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/75556" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-0034069495" target="_blank" aria-label="Open Scopus in new tab">Scopus (24474)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/10802651/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2F75556&amp;pmid=10802651" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), Human Phenotypes 6,672 terms (7.49&nbsp;× 10<sup>−6</sup>), Mouse Phenotypes 9,554 terms (5.23&nbsp;× 10<sup>−6</sup>) (<span class="dropBlock reference-citations"><a id="body-ref-sref26" href="#" role="doc-biblioref" data-xml-rid="bib26" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref26" aria-controls="sref26">Dickinson et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref26"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">27.</div><div>Dickinson, M.E. ∙ Flenniken, A.M. ∙ Ji, X. ..., International Mouse Phenotyping Consortium, Jackson Laboratory, Infrastructure Nationale PHENOMIN, Institut Clinique de la Souris (ICS), Charles River Laboratories, MRC Harwell, Toronto Centre for Phenogenomics, Wellcome Trust Sanger Institute, RIKEN BioResource Center</div><div><strong>High-throughput discovery of novel developmental phenotypes</strong></div><div><em>Nature.</em> 2016; <strong>537</strong>:508-514</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature19356" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-84989204804" target="_blank" aria-label="Open Scopus in new tab">Scopus (513)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27626380/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature19356&amp;pmid=27626380" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), and gene expression from the MGI (Mouse Genome Informatics) database 9,337 terms (5.35&nbsp;× 10<sup>−6</sup>) (<span class="dropBlock reference-citations"><a id="body-ref-sref29" href="#" role="doc-biblioref" data-xml-rid="bib29" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref29" aria-controls="sref29">Eppig et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref29"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">30.</div><div>Eppig, J.T. ∙ Smith, C.L. ∙ Blake, J.A. ...</div><div><strong>Mouse Genome Informatics (MGI): Resources for Mining Mouse Genetic, Genomic, and Biological Data in Support of Primary and Translational Research</strong></div><div>Springer New York, 2017</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.T.EppigC.L.SmithJ.A.BlakeM.RingwaldC.J.BultMouse+Genome+Informatics+%28MGI%29%3A+Resources+for+Mining+Mouse+Genetic%2C+Genomic%2C+and+Biological+Data+in+Support+of+Primary+and+Translational+Research2017Springer+New+York" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Enrichment was tested against the whole human genome (hg19) using standard parameters.</div></section><section id="sec-4-4-11"><h4>Protein network</h4><div id="p0315" role="paragraph">To build a meaningful network, the genes (<a id="crosref1520" href="#mmc5">Table S5</a>) which showed limb related by literature for each locus and all notable genes if none of them has been shown to be limb-related for that locus were used as input into the STRING (<span class="dropBlock reference-citations"><a id="body-ref-sref111-3" href="#" role="doc-biblioref" data-xml-rid="bib111" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref111-3" aria-controls="sref111-3">Szklarczyk et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref111-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">120.</div><div>Szklarczyk, D. ∙ Gable, A.L. ∙ Lyon, D. ...</div><div><strong>STRING v11: protein-protein association networks with increased coverage, supporting functional discovery in genome-wide experimental datasets</strong></div><div><em>Nucleic Acids Res.</em> 2019; <strong>47</strong>:D607-D613</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gky1131" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_120_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-85059797047" target="_blank" aria-label="Open Scopus in new tab">Scopus (4748)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30476243/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgky1131&amp;pmid=30476243" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) (<a id="intref0250" href="https://string-db.org/" target="_blank">https://string-db.org/</a>) with specifying <i>Homo sapiens</i> in organism. STRING tool was used to construct a PPI (protein-protein interaction) network with an interacting confidence scores of &gt; 0.4 designated as the cutoff and limiting the number of interactions that directly connect with input by setting the 1st shell to 10. STRING is one of the largest databases of known and predicted protein-protein interactions. In STRING, the functional associations are derived from four sources: genomic context, high-throughput experiments, conserved coexpression, and previous knowledge. The network was produced in STRING and recolored in Cytoscape 3.8.1 (<span class="dropBlock reference-citations"><a id="body-ref-sref57-2" href="#" role="doc-biblioref" data-xml-rid="bib57" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref57-2" aria-controls="sref57-2">Kohl et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref57-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">62.</div><div>Kohl, M. ∙ Wiese, S. ∙ Warscheid, B.</div><div><strong>Cytoscape: software for visualization and analysis of biological networks</strong></div><div><strong>Data Mining in Proteomics</strong></div><div>Springer, 2011; 291-303</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/978-1-60761-987-1_18" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-79952202448" target="_blank" aria-label="Open Scopus in new tab">Scopus (668)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2F978-1-60761-987-1_18" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) (<a id="intref0255" href="https://apps.cytoscape.org/" target="_blank">https://apps.cytoscape.org/</a>), which is an open source platform for visualizing complex networks. cytoHubba plugin in Cytoscape was used to extract the top 2 hub genes (not include extended additional ones) from the PPI network based on maximal clique centrality (MCC) algorithm.</div></section><section id="sec-4-4-12"><h4>Gene-phenotype associations</h4><div id="p0320" role="paragraph">Genes and their associated disorders in human were taken from Online Mendelian Inheritance in Man (OMIM) database. Gene name was used to search OMIM and if associated with a Mendelian condition then the description of phenotypic condition and OMIM code were collected. Literature was searched for embryonic expression and mouse phenotypes using Google Scholar. Gene names were searched together with digit, finger, limb, skin, or dermatoglyph. Embryonic limb expression data were summarized for the table if expression was determined using a spatially explicit method (RNA <i>in situ</i> hybridization, immunodetection, or <i>in&nbsp;vivo</i> reporter) only.</div></section><section id="sec-4-4-13"><h4>Genetic correlation estimation</h4><div id="p0325" role="paragraph">The pairwise genetic correlation (<i>r</i><sub><i>g</i></sub>) values among fingerprint patterns on ten digits in discovery cohorts (genotype available) and genetic correlation between fingerprint and hand traits in NSPT and JD cohorts were calculated using bivariate GCTA-GREML (<span class="dropBlock reference-citations"><a id="body-ref-sref61-2" href="#" role="doc-biblioref" data-xml-rid="bib61" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref61-2" aria-controls="sref61-2">Lee et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref61-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">66.</div><div>Lee, S.H. ∙ Yang, J. ∙ Goddard, M.E. ...</div><div><strong>Estimation of pleiotropy between complex diseases using single-nucleotide polymorphism-derived genomic relationships and restricted maximum likelihood</strong></div><div><em>Bioinformatics.</em> 2012; <strong>28</strong>:2540-2542</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/bts474" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-84867284056" target="_blank" aria-label="Open Scopus in new tab">Scopus (344)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22843982/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbts474&amp;pmid=22843982" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref125-3" href="#" role="doc-biblioref" data-xml-rid="bib125" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref125-3" aria-controls="sref125-3">Yang et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref125-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">135.</div><div>Yang, J. ∙ Lee, S.H. ∙ Goddard, M.E. ...</div><div><strong>GCTA: a tool for genome-wide complex trait analysis</strong></div><div><em>Am. J. Hum. Genet.</em> 2011; <strong>88</strong>:76-82</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_135_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=10.1016%2Fj.ajhg.2010.11.011&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_135_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=10.1016%2Fj.ajhg.2010.11.011&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_135_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-78650856517" target="_blank" aria-label="Open Scopus in new tab">Scopus (3187)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21167468/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.ajhg.2010.11.011&amp;pmid=21167468" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) where phenotypes are measured in the same sample. This approach estimates the extent to which genetic similarities correlate with phenotypic similarities. Then we calculated the average <i>r</i><sub><i>g</i></sub> values of overall pairs (<mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper C 10 squared" tabindex="0" ctxtmenu_counter="2" style="font-size: 130%;"><mjx-math data-semantic-structure="(4 (3 0 1) 2)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="4" data-semantic-children="0,1,2" data-semantic-collapsed="(4 (3 0 1) 2)" data-semantic-owns="0 1 2" data-semantic-speech="upper C 10 squared"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="4"><mjx-c class="mjx-c1D436">𝐶</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.288em; margin-left: -0.045em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="4" style="margin-left: 0.138em;"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="4"><mjx-c class="mjx-c31" noic="true" style="padding-top: 0.666em;">1</mjx-c><mjx-c class="mjx-c30" style="padding-top: 0.666em;">0</mjx-c></mjx-mn></mjx-script></mjx-msubsup></mjx-mrow></mjx-math></mjx-container>) and the middle three digits pairs (<mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper C 6 squared" tabindex="0" ctxtmenu_counter="3" style="font-size: 130%;"><mjx-math data-semantic-structure="(4 (3 0 1) 2)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="4" data-semantic-children="0,1,2" data-semantic-collapsed="(4 (3 0 1) 2)" data-semantic-owns="0 1 2" data-semantic-speech="upper C 6 squared"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="4"><mjx-c class="mjx-c1D436">𝐶</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.288em; margin-left: -0.045em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="4" style="margin-left: 0.138em;"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="4"><mjx-c class="mjx-c36">6</mjx-c></mjx-mn></mjx-script></mjx-msubsup></mjx-mrow></mjx-math></mjx-container>). The significance test between genetic correlations values of overall pairs and the middle three digits (D2, D3, and D4) pairs was conducted using t test in R.</div></section><section id="sec-4-4-14"><h4>Polygenic Risk Score (PRS) Analysis</h4><div id="p0330" role="paragraph">We calculated the polygenic risk score for fingerprint patterns of ten digits by PRSice-2 software (<span class="dropBlock reference-citations"><a id="body-ref-sref19" href="#" role="doc-biblioref" data-xml-rid="bib19" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref19" aria-controls="sref19">Choi and O’Reilly, 2019</a><div class="dropBlock__holder" data-db-target-of="sref19"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">19.</div><div>Choi, S.W. ∙ O’Reilly, P.F.</div><div><strong>PRSice-2: Polygenic Risk Score software for biobank-scale data</strong></div><div><em>Gigascience.</em> 2019; <strong>8</strong>:giz082</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/gigascience/giz082" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-85069904441" target="_blank" aria-label="Open Scopus in new tab">Scopus (190)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31307061/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fgigascience%2Fgiz082&amp;pmid=31307061" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The PRS is a method which is the sum of the trait-related SNPs at multiple genetic loci and weighted according to the effect size estimated by genome-wide association studies. In this study, the “base” data was obtained from GWAS summary statistics of fingerprint-associated 43 SNPs of meta-analysis. The “target” data was individual-level genotype (PLINK format) and ordinal or binary fingerprint phenotypes from three EAS cohorts (TZL, NSPT and JD). Sex was included as a covariate, for p value thresholds using a lower bound of p&nbsp;= 0.0001, an upper bound of&nbsp;= 0.5 and an increment of 5e-05.</div></section><section id="sec-4-4-15"><h4>Correlation between fingerprint patterns and hand traits</h4><div id="p0335" role="paragraph">We used linear regression to analyze the correlation between digit or distal phalanx length and fingerprints on eight digits (statistical significance level at p&nbsp;&lt; 0.05). Wilcoxon tests were performed to compare the difference of digit five (D5) length and distal phalanx length of digit two (DP2) and three (DP3) between whorl types and non-whorl types fingerprint (significant threshold at p&nbsp;&lt; 0.00083 (i.e., 0.05/60) after FDR correction). To evaluate the performance of 43 independent genomic signals significantly associated with fingerprints in the GWAS results of hand phenotypes, we use the p value, 0.0042 (i.e., 0.05/12), adjusted by eigen as the threshold. All statistical analysis was done with R.</div></section><section id="sec-4-4-16"><h4>Plasmids construction, cell culture and luciferase assay</h4><div id="p0340" role="paragraph">The promoter region of <i>EVI1</i> gene was amplified from genomic DNA of an unspecified participant in TZL cohort using the primers EVI1 pro-318-F (5′-ATCGAGATCTAAAGTCTGGGCGATGTG-3′) and EVI1 pro-318-R (5′- ATCGAAGCTTAAACCGACGGACAGAGACA −3′). The 303&nbsp;bp fragment was cloned into pGL3-promoter vector digested with BglII and Hind III, resulting in the replacement of the SV40 promoter with the human <i>EVI1</i> promoter, yielding the <i>EVI1</i> promoter vector pGL3-318. The human TERC promoter fragment was subcloned in the luciferase reporter pGL3-promoter with BglII/Nco I, which resulted in the replacement of SV40 promoter with the 867&nbsp;bp TERC promoter, namely pGL3-TERC. The construction of TERC promoter is based on the previous study (<span class="dropBlock reference-citations"><a id="body-ref-sref129" href="#" role="doc-biblioref" data-xml-rid="bib129" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref129" aria-controls="sref129">Zhao et&nbsp;al., 2005</a><div class="dropBlock__holder" data-db-target-of="sref129"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">139.</div><div>Zhao, J. ∙ Bilsland, A. ∙ Jackson, K. ...</div><div><strong>MDM2 negatively regulates the human telomerase RNA gene promoter</strong></div><div><em>BMC Cancer.</em> 2005; <strong>5</strong>:6</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/1471-2407-5-6" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_139_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-13144291643" target="_blank" aria-label="Open Scopus in new tab">Scopus (22)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15656906/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2F1471-2407-5-6&amp;pmid=15656906" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) which has been proven to be functional. The human GOLIM4 promoter fragment was subcloned in the luciferase reporter pGL3-promoter with Sac I and Mlu I, which resulted in the replacement of SV40 promoter with the 1021&nbsp;bp GOLIM4 promoter (upstream of transcription start site), namely pGL3-GOLIM4.</div><div id="p0345" role="paragraph">The genomic fragment that contains the SNP rs7646897 was amplified from, genomic DNA of an unspecified participant, who was verified as homozygous at rs7646897 by Sanger sequencing using the primers rs7646897-cloning-F (5′-ATCGACGCGTACTGCCATCTCAAGACTAAGC-3′) and rs7646897-cloning-R (5′-AGCTAGATCTCATCCTGCACATGTACCTCTG-3′), then the 1469&nbsp;bp fragment was cloned into pGL3-318 vector with restriction enzymes MluI and BglII. The whole same approach was applied with SNP rs7623083. The genomic fragment containing the rs7623083 SNP was amplified using the primers rs7623083-cloning-F (5′-ATCGACGCGTGAGATGACCCCAAAGGATGGG-3′) and rs7623083-cloning-R (5′-AGCTCTCGAGGTCACTGCCTTAATAGCTCCCC-3′), then the 1415&nbsp;bp fragment was cloned into pGL3-promoter vector with restriction enzymes MluI and XhoI. SNP rs7646897 is at the position of 786bp in its fragment and SNP rs7623083 is at the position of 752bp in its fragment. Individual mutations were incorporated using site-directed mutagenesis (Yeasen). The inserts in each construct were verified by Sanger sequencing. The amplified, cloned sequence carrying SNP rs7646897 was verified by Sanger sequencing and was consistent with the NCBI primary assembly sequence (NCBI: gi|224514994: 168695100-168696800 <i>Homo sapiens</i> chromosome 3, GRCh37.p13 Primary Assembly), though the target site (red box) was different due to site-directed mutagenesis. The actual sequence carrying rs7623083 (red box) was different from the NCBI primary assembly sequence (NCBI: NC_000003.11: 168709002-168710504 <i>Homo sapiens</i> chromosome 3, GRCh37.p13 Primary Assembly) at three sites (green box), which could be explained by the individual mutation of donor. The BLAST alignment for each fragment is presented at Data S1.</div><div id="p0350" role="paragraph">Luciferase reporter plasmids were transfected into HEK293T cells (200ng), using Lipofectamine2000 (Invitrogen) according to manufacturer’s instructions. HEK293T cells were cultured in DMEM (GIBCO) supplemented with 10% fetal bovine serum (FBS, GIBCO) and 1% Penicillin-Streptomycin (GIBCO) and incubated at 37°C in 5% CO2. Firefly luciferase expression was normalized to values from co-transfected Renilla luciferase plasmid (10&nbsp;ng pRL-TK). Cells were harvested 48&nbsp;h after transfection. Luminescence activity was measured with a Lumat LB 9508 Single Tube Luminometer. Data were represented at least three independent experiments.</div></section><section id="sec-4-4-17"><h4>Visualization and measurement of dermal structure of mouse forelimbs</h4><div id="p0355" role="paragraph">Mouse forelimbs were collected and fixed in 4% formaldehyde for 72 hours. The fixed limbs were treated with 5% KOH at 37°C overnight to remove the epidermis. Limbs were washed using 1X PBS and tubes containing the forelimb vortexed to remove residual epidermis. Each forelimb was dipped in 0.05% Toluidine Blue solution for 45 s, placed immediately in 1X PBS and visualized using Stereo microscope (Olympus SZX10) and images recorded. Images were used for measurement of digit length using ImageJ.</div></section><section id="sec-4-4-18"><h4>Categorization of transverse digital ridges</h4><div id="p0360" role="paragraph">We observed four different categories of transverse digital ridges and they were defined as follows (<a id="crosref1555" href="#fig3">Figures 3</a>B and 3C). Pads running continuously across the digit were referred as continuous (c), those running discontinuously across the digit along a single line were referred to as discontinuous (d), and pads originating only at one side of the digit were coded as incomplete (i). Scoring was done for transverse ridges of the middle three digits (D2, D3 and D4) for each forelimb. Both right (R) and left (L) forelimb was scored for each individual mouse. In total, 4 litters were analyzed (<a id="crosref1560" href="#mmc4">Table S4</a>). Two pups of FVB mice were collected at postnatal day 0 (P0), P4, P8 and P12 to study the development of transverse digital ridge patterning (<a id="crosref1565" href="#figs3">Figure&nbsp;S3</a>B).</div></section><section id="sec-4-4-19"><h4>Statistical analyses on transverse digital ridge patterns</h4><div id="p0365" role="paragraph">The effect of genotype on three transverse digit ridge patterns (continuous(c), discontinuous (d), and incomplete (i)/half (h)) was analyzed using a mixed ordinal logistic regression model fitting: genotype, digit, side (L/R), digit × genotype and digit × side interaction was included as fixed effects and litter and line were fitted as random effects. All the statistical analyses were performed using statistical package SAS.</div></section><section id="sec-4-4-20"><h4>Gene expression analyses</h4><div id="p0370" role="paragraph">For quantitative RT-PCR, total RNA was isolated from mouse embryonic limb buds using the RNEasy micro kit (QIAGEN) following disruption of the tissue in RLT buffer (QIAGEN) using a handheld homogenizer. cDNA was synthesized from 1&nbsp;μg of total RNA using random primers and Superscript III reverse transcriptase (Life Technologies). cDNA was diluted 20-fold and 3&nbsp;μl used as a template for each qRT-PCR using the Universal SYBR Green Master Mix (Life Technologies), according to manufacturer’s instructions. qRT-PCRs were performed using a Stratagene MX 3000p with primer annealing temperatures of 60°C for 40 cycles. Reactions were performed in triplicate, with at least three biological replicates used to determine each data point. Relative expression levels were determined from cDNA dilution standard curves, and normalized to <i>Gapdh</i> values. Sequences of oligonucleotides used as primers were:<div id="ulist0020" role="list"><div id="u0045" role="listitem"><div class="content"><div id="p0375" role="paragraph">Evi1 F: 5′-GCTATGATCAGCACAACCTTGTTG-3′</div></div></div><div id="u0050" role="listitem"><div class="content"><div id="p0380" role="paragraph">Evi1 R: 5′-TGTCTGCGACTACTCGGTAGAATATC-3′</div></div></div><div id="u0055" role="listitem"><div class="content"><div id="p0385" role="paragraph">Gapdh F: 5′-CGTATTGGGCGCCTGGTCAC-3′</div></div></div><div id="u0060" role="listitem"><div class="content"><div id="p0390" role="paragraph">Gapdh R: 5′-ATGATGACCCTTTTGGCTCC-3′</div></div></div></div></div><div id="p0395" role="paragraph">For <i>Evi1</i> whole mount <i>in situ</i> hybridization, mouse embryos were fixed in 4% PFA overnight at 4°C. Samples were dehydrated into 100% methanol, bleached in 5:1 MeOH: 30% H<sub>2</sub>O<sub>2</sub>, rehydrated, treated with 20&nbsp;μg/ml proteinase K then fixed in 4% PFA, 0.2% glutaraldehyde. Samples were hybridized with a digoxigenin-labeled riboprobe synthesized from expressed sequence tag clone IMAGp998F2411055Q (Source Bioscience) at 60°C overnight in 50% formamide, 5 X saline sodium citrate (SSC), 1% sodium dodecyl sulfate (SDS), 50&nbsp;μg/mL heparin, and 50&nbsp;μg/mL yeast RNA. Samples were washed to remove unbound probe and signal detected by incubating with alkaline phosphatase conjugated anti-digoxigenin antibody (Sigma Aldrich, 1:1000) overnight at 4°C followed by a subsequent 5-bromo-4-chloro-3′-indolylphosphate/nitro-blue-tetrazolium (BCIP/NBT) (Sigma) color reaction.</div></section><section id="sec-4-4-21"><h4>RNA <i>in situ</i> hybridization</h4><div id="p0400" role="paragraph">Formaldehyde-fixed forelimb or digit samples embedded in paraffin were sectioned at 6&nbsp;μm and processed using the RNAscope Multiplex fluorescent reagent kit according to manufacturer’s instructions. Probes were species-specific <i>Evi1</i>(<i>Mecom)</i> and <i>Prrx1</i> (see table in <a id="crosref1570" href="#sec-4">STAR Methods</a>). Digits 1 and 5 were excluded from analyses. Positive and negative (dapB) control probes were used for both mouse and human sections. After <i>in situ</i> hybridization sections were blocked (5% goat serum/TBST) and incubated overnight at 4°C with primary antibodies (see table in <a id="crosref1575" href="#sec-4">STAR Methods</a>) in blocking buffer. Detection was with fluorescent secondary antibodies diluted in blocking buffer, followed by counterstaining with DAPI and mounting in Prolong Gold. Sections were imaged using a Zeiss LSM 880 confocal microscope.</div></section><section id="sec-4-4-22"><h4>Immunofluorescence</h4><div id="p0405" role="paragraph">Fetal tissue samples were fixed overnight at 4°C in 10% neutral buffered formalin, or, for CS14 and CS17 specimens, were embedded in OCT for cryosectioning. 10-, 13- and 16-&nbsp;week estimated gestational age samples were embedded in paraffin. Digits 1 and 5 were not included in analyses. For immunofluorescence, cryosections were incubated in PBS for 30&nbsp;minutes at 37°C to remove gelatin; paraffin sections were dewaxed, rehydrated and antigen retrieved in sodium citrate (pH 6) using Bio-retriever2000 (Aptum Bio, Southampton, UK). Tissue sections were permeabilized in 0.1% Triton X-100/PBS, washed with PBS and treated with TrueBlack (Biotium). Sections were blocked in 5% goat serum/PBS then incubated at 4°C with primary antibody. Sections were washed with PBS, incubated with fluorescent secondary antibody at room temperature and washed with PBS, then counterstained with DAPI (Sigma Aldrich) and mounted in Prolong Gold (ThermoFisher Scientific). Fluorescent images were collected using a Zeiss LSM710 inverted confocal microscope.</div></section></section><section id="sec-4-5"><h3>Quantification and statistical analysis</h3><div id="p0410" role="paragraph">Information on specific quantification methods such as collection and specification of fingerprint patterns, hand traits in human and quantification of dermatoglyph in mice are described in associated <a id="crosref1580" href="#sec-4-4">Method details</a>, or main texts. Statistical tests were performed using R software, parameters such as number of sample size, the number of mice, the number of independent experiments, measures of center, dispersion, and precision (mean ± SEM and 95% confidence interval), statistical test and significance, are reported in Figures and Figure&nbsp;Legends or <a id="crosref1585" href="#sec-4-4">Method details</a>.</div></section></section><section id="acknowledgments" role="doc-acknowledgments"><h2>Acknowledgments</h2><div id="p0415" role="paragraph">We would like to thank the participants of the TZL, NSPT, JD, CKB, WeGene, ALSPAC, QIMR, and Pittsburgh study who consented to participate in research, and the whole TZL, NSPT, and JD teams, which include interviewers, computer and laboratory technicians, clerical workers, research scientists, volunteers, managers, receptionists, and nurses. We also thank F.W. for her help with <a id="crosref1590" href="#fig1">Figure&nbsp;1</a>A and W.S. for her help with graphic abstract. We thank Dida Biotech for performing the Luciferase assay experiments. This project was funded by the “Strategic Priority Research Program” of the Chinese Academy of Sciences (XDB38020400 to S. Wang); Science and Tecnology Commission of Shanghai Municipality; the Ministry of Science and Technology of the People's Republic of China; Shanghai Municipal Science and Technology Major Project (2017SHZDZX01 to L.J. and S. Wang); National Key Research and Development Project (2018YFC0910403 to S. Wang); Max Planck-CAS Paul Gerson Unna Independent Research Group Leadership Award (to S. Wang); National Natural Science Foundation of China (NSFC) (31521003 to L.J., 81622003 to L.W., 81373082 to J. Lv, and 31771325 to J.T.); China Postdoctoral Science Foundation (2019M651351 to J. Li); National Science &amp; Technology Basic Research Project (2015FY111700 to L.J.); CAMS Innovation Fund for Medical Sciences (2019-I2M-5-066 to L.J.); The 111 Project (B13016 to L.J.); Major Research Program of National Natural Science Foundation of China (91731303 to J.T.); the National Institute for Dental and Craniofacial Research (NIDCR) (R01-DE016148 to M.L.M and S.M.W and R21-DE016930 to M.L.M); Biotechnology and Biological Sciences Research Council (BBS/E/D/10002071 and BB/T007788/1 to D.J.H.); MRC (G1100357 to R.A.A and MR/N022556/1 [MRC Centre for Reproductive Health]; European Commission; Marie Sklodowska-Curie Actions Individual fellowship 706429; Wellcome Trust/DBT India Alliance Early Career Fellowship (IA/E/18/1/504338 to C.B.M); Australian Research Council (A7960034, A79906588, A79801419, DP0212016, and DP0343921 to S.E.M); Australian National Health and Medical Research Council (NHMRC) (241944, 339462, 389927, 389875, 389891, 389892, 389938, 443036, 442915, 442981, 496739, 552485, and 552498 to S.E.M); the UK Medical Research Council; the Wellcome Trust (102215/2/13/2); the University of Bristol; Medical Research Council (MC_UU_12013/4 to D.M.E. and MC_UU_12013/3 to N.J.T.); Australian Research Council Future Fellowship (FT130101709 to D.M.E.); the Hong Kong Kadoorie Charitable Foundation; UK Wellcome Trust (202922/Z/16/Z, 104085/Z/14/Z, and 088158/Z/09/Z); UK Medical Research Council (MC-PC-13049 and MC-PC-14135); and core funding to the Clinical Trial Service Unit and Epidemiological Studies Unit at Oxford University from British Heart Foundation, UK MRC, and Cancer Research UK (to R.G.W. and Z.C.). S.E.M. is supported by NHMRC (SRF 1103623).</div><section id="sec-5"><h3>Author contributions</h3><div id="p0420" role="paragraph">S. Wang, D.J.H., and L.J. conceived the project and provided main resources. For the discovery, GWAS, H.Z., J. Li, J.T., Y.Y., S.Z., J.Z., H.Y., and W.Z. performed data and sample collections. H.Z., J. Li, S.Z., and Y.X. contributed to generating the fingerprint and hand phenotype data. S. Wu, Y. Liu, and Q.P. contributed to generating the SNP array data. J. Li and M.P. performed computational analysis. For the replication and meta-analysis, J. Lv, R.M., R.G.W., K.L., and Z.C. performed data analysis and provided summary statistics results and relevant descriptions for the CKB cohort; Lizhong Wang, Y. Li, S.T., X.W., and G.C. performed data analysis and provided summary statistics results and relevant descriptions for the WeGene cohort; D.M.E. and J.P.K. performed data analysis and provided summary statistics results and relevant descriptions for the ALSPAC; S.E.M., N.G.M., Y.Y.W.H., and D.Z.L. performed data analysis and provided summary statistics results and relevant descriptions for the QIMR; and J.M.C., K.N., E.F., S.M.W., and M.L.M. performed data analysis and provided summary statistics results and relevant descriptions for the Pittsburg cohort. J.L. and M.P. performed the overall analysis. For the functional experiments, D.J.H., J.D.G., C.B.M., and M.C. performed the experiments on mouse models and D.J.H., J.D.G., A.M., R.A.A., and E.I.C. did the gene and protein expression. H.B. aided in data analysis from mouse models. D.H. and Lan Wang performed the luciferase reporter assay. J. Li, D.J.H., and S. Wang wrote the manuscript with input from J.M.C., E.F., S.M.W., M.L.M., M.C., S.E.M., and other co-authors.</div></section><section id="sec-6"><h3>Declaration of interests</h3><div id="p0425" role="paragraph">Lizhong Wang, Y. Li, S.T., X.W., and G.C. are employees of WeGene Inc. The other authors declare no competing interests.</div></section><section id="sec-7"><h3>Inclusion and diversity</h3><div id="p0430" role="paragraph">We worked to ensure gender balance in the recruitment of human subjects. We worked to ensure sex balance in the selection of non-human subjects. We worked to ensure ethnic or other types of diversity in the recruitment of human subjects. The author list of this paper includes contributors from the location where the research was conducted who participated in the data collection, design, analysis, and/or interpretation of the work.</div></section></section><section id="supplementary-material" data-type="supplementary-materials"><h2>Supplemental information (8)</h2><div id="p0440" role="paragraph"><div class="core-supplementary-material"><div class="core-link"><button class="downloadAllSupplementary"><span>Download all</span></button></div></div><div id="mmc1" class="core-supplementary-material" data-category="pdf"><div class="core-link"><a href="/cms/10.1016/j.cell.2021.12.008/attachment/939eb111-67d0-411b-8429-80c611a7259e/mmc1.pdf" download="mmc1.pdf" aria-labelledby="mmc1-heading mmc1-link"><span id="mmc1-link">PDF (56.97 KB)</span></a></div><div class="core-description"><div id="mmc1-heading" class="heading">Methods S1. Justifications for transverse digital ridges as human dermatoglyphs and the BLAST alignment for each fragment, related to Figures 2 and 3</div></div></div><div id="mmc2" class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="/cms/10.1016/j.cell.2021.12.008/attachment/bbfa0b5b-0911-48b8-b662-73216206e52d/mmc2.xlsx" download="mmc2.xlsx" aria-labelledby="mmc2-heading mmc2-link"><span id="mmc2-link">Spreadsheet (15.94 KB)</span></a></div><div class="core-description"><div id="mmc2-heading" class="heading">Table S1. Characteristics of cohorts and summary of imputation methods and quality control filters, related to Table 1</div></div></div><div id="mmc3" class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="/cms/10.1016/j.cell.2021.12.008/attachment/b64d17ea-c19c-4014-8c1e-c12193025bba/mmc3.xlsx" download="mmc3.xlsx" aria-labelledby="mmc3-heading mmc3-link"><span id="mmc3-link">Spreadsheet (5.32 MB)</span></a></div><div class="core-description"><div id="mmc3-heading" class="heading">Table S2. The performance of signals in Han Chinese discovery and replication populations based on ordinal and binary fingerprint patterns, related to Figure&nbsp;1</div></div></div><div id="mmc4" class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="/cms/10.1016/j.cell.2021.12.008/attachment/0b9fbc26-48ba-4149-984b-c625cf0fac75/mmc4.xlsx" download="mmc4.xlsx" aria-labelledby="mmc4-heading mmc4-link"><span id="mmc4-link">Spreadsheet (22.64 KB)</span></a></div><div class="core-description"><div id="mmc4-heading" class="heading">Table S3. Association results of the composite phenotype (extracted from the middle three digits of both hands) and 66 additional phenotypes (derived from fingerprint patterns) used in this study, related to Figures 1 and 2</div></div></div><div id="mmc5" class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="/cms/10.1016/j.cell.2021.12.008/attachment/af0c4b43-986d-4e56-b4b1-b3564b4a4ef0/mmc5.xlsx" download="mmc5.xlsx" aria-labelledby="mmc5-heading mmc5-link"><span id="mmc5-link">Spreadsheet (17.13 KB)</span></a></div><div class="core-description"><div id="mmc5-heading" class="heading">Table S4. Transverse digit pad patterning in Evi1Jbo mice, related to Figure&nbsp;3</div></div></div><div id="mmc6" class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="/cms/10.1016/j.cell.2021.12.008/attachment/09f2ff24-8289-45b5-a357-71c13b0b51b4/mmc6.xlsx" download="mmc6.xlsx" aria-labelledby="mmc6-heading mmc6-link"><span id="mmc6-link">Spreadsheet (24.31 KB)</span></a></div><div class="core-description"><div id="mmc6-heading" class="heading">Table S5. The performance of 43 signals after meta-analysis of eight populations (the top SNP in each locus was displayed), related to Figure&nbsp;4</div></div></div><div id="mmc7" class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="/cms/10.1016/j.cell.2021.12.008/attachment/5e04db61-0bdb-439c-b9bf-74f223864f82/mmc7.xlsx" download="mmc7.xlsx" aria-labelledby="mmc7-heading mmc7-link"><span id="mmc7-link">Spreadsheet (54.92 KB)</span></a></div><div class="core-description"><div id="mmc7-heading" class="heading">Table S6. Single-factor regression analysis and genetic correlation (rg) based on SNP between hand traits and fingerprint patterns of eight digits (except the thumbs) in the Chinese Han populations (n&nbsp;= 6,318, JD+NSPT), related to Figure&nbsp;5</div></div></div><div class="core-supplementary-material" data-category="pdf"><div class="core-link"><a href="/cms/10.1016/j.cell.2021.12.008/attachment/db0bedd0-2ff3-48ec-bd34-d4969f85f25d/mmc8.pdf" download="mmc8.pdf" aria-labelledby="mmc8-heading mmc8-link"><span id="mmc8-link">PDF (10.61 MB)</span></a></div><div class="core-description"><div id="mmc8-heading" class="heading">Data S1. Alignment view of the cloned sequences used in luciferase reporter assay and regional plots of 43 loci associated with fingerprint patterns, related to Figures 2 and 4</div></div></div></div></section><section id="app-2"><h2>Supporting citations</h2><div id="p0445" role="paragraph">The following references appear in the supplemental information: <span class="dropBlock reference-citations"><a id="body-ref-sref43" href="#" role="doc-biblioref" data-xml-rid="bib43" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref43" aria-controls="sref43">Headon and Overbeek (1999)</a><div class="dropBlock__holder" data-db-target-of="sref43"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">46.</div><div>Headon, D.J. ∙ Overbeek, P.A.</div><div><strong>Involvement of a novel Tnf receptor homologue in hair follicle induction</strong></div><div><em>Nat. Genet.</em> 1999; <strong>22</strong>:370-374</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/11943" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-0032775933" target="_blank" aria-label="Open Scopus in new tab">Scopus (298)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/10431242/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2F11943&amp;pmid=10431242" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref58" href="#" role="doc-biblioref" data-xml-rid="bib58" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref58" aria-controls="sref58">Kowalczyk-Quintas and Schneider (2014)</a><div class="dropBlock__holder" data-db-target-of="sref58"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">63.</div><div>Kowalczyk-Quintas, C. ∙ Schneider, P.</div><div><strong>Ectodysplasin A (EDA) - EDA receptor signalling and its pharmacological modulation</strong></div><div><em>Cytokine Growth Factor Rev.</em> 2014; <strong>25</strong>:195-203</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.cytogfr.2014.01.004" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24508088/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cytogfr.2014.01.004&amp;pmid=24508088" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref81" href="#" role="doc-biblioref" data-xml-rid="bib81" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref81" aria-controls="sref81">Monreal et&nbsp;al. (1999)</a><div class="dropBlock__holder" data-db-target-of="sref81"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">88.</div><div>Monreal, A.W. ∙ Ferguson, B.M. ∙ Headon, D.J. ...</div><div><strong>Mutations in the human homologue of mouse dl cause autosomal recessive and dominant hypohidrotic ectodermal dysplasia</strong></div><div><em>Nat. Genet.</em> 1999; <strong>22</strong>:366-369</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/11937" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-0032811085" target="_blank" aria-label="Open Scopus in new tab">Scopus (319)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/10431241/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2F11937&amp;pmid=10431241" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref109" href="#" role="doc-biblioref" data-xml-rid="bib109" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref109" aria-controls="sref109">Srivastava et&nbsp;al. (1997)</a><div class="dropBlock__holder" data-db-target-of="sref109"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">118.</div><div>Srivastava, A.K. ∙ Pispa, J. ∙ Hartung, A.J. ...</div><div><strong>The Tabby phenotype is caused by mutation in a mouse homologue of the EDA gene that reveals novel mouse and human exons and encodes a protein (ectodysplasin-A) with collagenous domains</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 1997; <strong>94</strong>:13069-13074</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.94.24.13069" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_118_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-12644310324" target="_blank" aria-label="Open Scopus in new tab">Scopus (246)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/9371801/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.94.24.13069&amp;pmid=9371801" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref113" href="#" role="doc-biblioref" data-xml-rid="bib113" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref113" aria-controls="sref113">Tsugane and Yasuda (1995)</a><div class="dropBlock__holder" data-db-target-of="sref113"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">123.</div><div>Tsugane, M. ∙ Yasuda, M.</div><div><strong>Dermatoglyphics on volar skin of mice: the normal pattern</strong></div><div><em>Anat. Rec.</em> 1995; <strong>242</strong>:225-232</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/ar.1092420212" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_123_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-0029027841" target="_blank" aria-label="Open Scopus in new tab">Scopus (19)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/7668408/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Far.1092420212&amp;pmid=7668408" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>.</div></section></div></section><div class="core-linked-content">
    

        

</div><section id="backmatter" data-extent="backmatter"><div class="core-container"><section id="data-availability" class="core-data-availability"><h2>Data and code availability</h2><div id="p0030" role="paragraph"><div id="ulist0015" role="list"><div id="u0030" role="listitem"><div class="label">•</div><div class="content"><div id="p0035" role="paragraph">Full summary statistics for all SNPs in the eight cohorts (TZL, NSPT, JD, CKB, WeGene, ALSPAC, QIMR, and Pittsburgh) GWAS datasets can be found in the National Omics Data Encyclopedia (NODE) (<a id="intref0010" href="http://www.biosino.org/node" target="_blank">http://www.biosino.org/node</a>). The accession number for the data analyzed in this paper is NODE: OEP000198 (<a id="interref0010" href="http://www.biosino.org/node/project/detail/OEP000198" target="_blank">http://www.biosino.org/node/project/detail/OEP000198</a>), and also listed in the <a id="crosref0380" href="#sec-4-1">Key Resources Table</a>. Individual genotype and phenotype data cannot be shared due to privacy concerns. All other relevant data, such as mice dermatoglyph, are within the paper and its Supporting Information files.</div></div></div><div id="u0035" role="listitem"><div class="label">•</div><div class="content"><div id="p0040" role="paragraph">This paper does not report original code.</div></div></div><div id="u0040" role="listitem"><div class="label">•</div><div class="content"><div id="p0045" role="paragraph">Any additional information required to reanalyze the data reported in this paper is available from the Lead Contact upon request.</div></div></div></div></div></section><section id="references"><h2>References</h2><div id="bibliography" role="doc-bibliography"><div role="list"><div role="listitem" data-has="label"><div id="bib1" class="citations"><div id="sref1" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref1-1" title="View in article">1.</a></div><div>Abecasis, G.R. ∙ Cherny, S.S. ∙ Cookson, W.O. ...</div><div><strong>Merlin--rapid analysis of dense genetic maps using sparse gene flow trees</strong></div><div><em>Nat. Genet.</em> 2002; <strong>30</strong>:97-101</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng786" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_1_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-0036338150" target="_blank" aria-label="Open Scopus in new tab">Scopus (2712)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11731797/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng786&amp;pmid=11731797" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib2" class="citations"><div id="sref2" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref2-1" title="View in article">2.</a></div><div>Aken, B.L. ∙ Ayling, S. ∙ Barrell, D. ...</div><div><strong>The Ensembl gene annotation system</strong></div><div><em>Database (Oxford).</em> 2016; <strong>2016</strong>:baw093</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/database/baw093" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27337980/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fdatabase%2Fbaw093&amp;pmid=27337980" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib3" class="citations"><div id="sref3" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref3" title="View in article">3.</a></div><div>Ali, S. ∙ Amina, B. ∙ Anwar, S. ...</div><div><strong>Genomic features of human limb specific enhancers</strong></div><div><em>Genomics.</em> 2016; <strong>108</strong>:143-150</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.ygeno.2016.08.003" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-84994065473" target="_blank" aria-label="Open Scopus in new tab">Scopus (2)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27580967/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.ygeno.2016.08.003&amp;pmid=27580967" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib4" class="citations"><div id="sref4" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref4-1" title="View in article">4.</a></div><div>Altshuler, D.M. ∙ Gibbs, R.A. ∙ Peltonen, L. ..., International HapMap 3 Consortium</div><div><strong>Integrating common and rare genetic variation in diverse human populations</strong></div><div><em>Nature.</em> 2010; <strong>467</strong>:52-58</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature09298" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-77956331627" target="_blank" aria-label="Open Scopus in new tab">Scopus (1889)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20811451/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature09298&amp;pmid=20811451" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib5" class="citations"><div id="sref5" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref5" title="View in article">5.</a></div><div>André, T. ∙ Lefèvre, P. ∙ Thonnard, J.L.</div><div><strong>Fingertip moisture is optimally modulated during object manipulation</strong></div><div><em>J.&nbsp;Neurophysiol.</em> 2010; <strong>103</strong>:402-408</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1152/jn.00901.2009" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-74049123806" target="_blank" aria-label="Open Scopus in new tab">Scopus (49)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19906876/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1152%2Fjn.00901.2009&amp;pmid=19906876" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib6" class="citations"><div id="sref6" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref6" title="View in article">6.</a></div><div>Arrieta, M.I. ∙ Salazar, L. ∙ Criado, B. ...</div><div><strong>Twin study of digital dermatoglyphic traits: Investigation of heritability</strong></div><div><em>Am. J. Hum. Biol.</em> 1991; <strong>3</strong>:11-15</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/ajhb.1310030104" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-84995256188" target="_blank" aria-label="Open Scopus in new tab">Scopus (7)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28520314/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fajhb.1310030104&amp;pmid=28520314" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib7" class="citations"><div id="sref7" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref7" title="View in article">7.</a></div><div>Arts, H.H. ∙ Doherty, D. ∙ van Beersum, S.E. ...</div><div><strong>Mutations in the gene encoding the basal body protein RPGRIP1L, a nephrocystin-4 interactor, cause Joubert syndrome</strong></div><div><em>Nat. Genet.</em> 2007; <strong>39</strong>:882-888</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng2069" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-34347356500" target="_blank" aria-label="Open Scopus in new tab">Scopus (245)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17558407/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng2069&amp;pmid=17558407" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib8" class="citations"><div id="sref8" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref8" title="View in article">8.</a></div><div>Ashburner, M. ∙ Ball, C.A. ∙ Blake, J.A. ..., The Gene Ontology Consortium</div><div><strong>Gene ontology: tool for the unification of biology</strong></div><div><em>Nat. Genet.</em> 2000; <strong>25</strong>:25-29</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/75556" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-0034069495" target="_blank" aria-label="Open Scopus in new tab">Scopus (24474)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/10802651/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2F75556&amp;pmid=10802651" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib9" class="citations"><div id="sref9" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref9" title="View in article">9.</a></div><div>Babler, W.</div><div><strong>Prenatal development of dermatoglyphic digital patterns: Associations with epidermal ridge, volar pad and bone morphology</strong></div><div><em>Coll. Antropol.</em> 1987; <strong>11</strong>:297-303</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=W.BablerPrenatal+development+of+dermatoglyphic+digital+patterns%3A+Associations+with+epidermal+ridge%2C+volar+pad+and+bone+morphologyColl.+Antropol.111987297303" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib10" class="citations"><div id="sref10" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref10-1" title="View in article">10.</a></div><div>Babler, W.J.</div><div><strong>Embryologic development of epidermal ridges and their configurations</strong></div><div><em>Birth Defects Orig. Artic. Ser.</em> 1991; <strong>27</strong>:95-112</div></div><div class="external-links"><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/1786361/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?pmid=1786361" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib11" class="citations"><div id="sref11" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref11" title="View in article">11.</a></div><div>Beccari, L. ∙ Jaquier, G. ∙ Lopez-Delisle, L. ...</div><div><strong>Dbx2 regulation in limbs suggests interTAD sharing of enhancers</strong></div><div><em>Dev. Dyn.</em> 2021; <strong>250</strong>:1280-1299</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/dvdy.303" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-85101792011" target="_blank" aria-label="Open Scopus in new tab">Scopus (4)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33497014/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fdvdy.303&amp;pmid=33497014" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib12" class="citations"><div id="sref12" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref12-1" title="View in article">12.</a></div><div>Bernstein, B.E. ∙ Stamatoyannopoulos, J.A. ∙ Costello, J.F. ...</div><div><strong>The NIH Roadmap Epigenomics Mapping Consortium</strong></div><div><em>Nat. Biotechnol.</em> 2010; <strong>28</strong>:1045-1048</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nbt1010-1045" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-77957940722" target="_blank" aria-label="Open Scopus in new tab">Scopus (1125)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20944595/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnbt1010-1045&amp;pmid=20944595" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib13" class="citations"><div id="sref13" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref13" title="View in article">13.</a></div><div>Bonnevie, K.</div><div><strong>Studies on papillary patterns of human fingers</strong></div><div><em>J.&nbsp;Genet.</em> 1924; <strong>15</strong>:1-111</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/BF02983100" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2FBF02983100" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib14" class="citations"><div id="sref14" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref14" title="View in article">14.</a></div><div>Boyd, A. ∙ Golding, J. ∙ Macleod, J. ...</div><div><strong>Cohort Profile: the ‘children of the 90s’--the index offspring of the Avon Longitudinal Study of Parents and Children</strong></div><div><em>Int. J. Epidemiol.</em> 2013; <strong>42</strong>:111-127</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/ije/dys064" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-84875577737" target="_blank" aria-label="Open Scopus in new tab">Scopus (1470)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22507743/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fije%2Fdys064&amp;pmid=22507743" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib15" class="citations"><div id="sref15" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref15" title="View in article">15.</a></div><div>Carlsen, E.Ø. ∙ Frengen, E. ∙ Fannemel, M. ...</div><div><strong>Haploinsufficiency of ANO6, NELL2 and DBX2 in a boy with intellectual disability and growth delay</strong></div><div><em>Am. J. Med. Genet. A.</em> 2015; <strong>167A</strong>:1890-1896</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/ajmg.a.37079" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-84937975018" target="_blank" aria-label="Open Scopus in new tab">Scopus (11)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25846056/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fajmg.a.37079&amp;pmid=25846056" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib16" class="citations"><div id="sref16" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref16" title="View in article">16.</a></div><div>Chaudhry, A.Z. ∙ Lyons, G.E. ∙ Gronostajski, R.M.</div><div><strong>Expression patterns of the four nuclear factor I genes during mouse embryogenesis indicate a potential role in development</strong></div><div><em>Dev. Dyn.</em> 1997; <strong>208</strong>:313-325</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/(SICI)1097-0177(199703)208:3%3C313::AID-AJA3%3E3.0.CO;2-L" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-0031050274" target="_blank" aria-label="Open Scopus in new tab">Scopus (171)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/9056636/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2F%28SICI%291097-0177%28199703%29208%3A3%253C313%3A%3AAID-AJA3%253E3.0.CO%3B2-L&amp;pmid=9056636" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib17" class="citations"><div id="sref17" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref17" title="View in article">17.</a></div><div>Chesterman, E.S. ∙ Gainey, G.D. ∙ Varn, A.C. ...</div><div><strong>Investigation of Prx1 protein expression provides evidence for conservation of cardiac-specific posttranscriptional regulation in vertebrates</strong></div><div><em>Dev. Dyn.</em> 2001; <strong>222</strong>:459-470</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/dvdy.1198" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-0034759688" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11747080/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fdvdy.1198&amp;pmid=11747080" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib18" class="citations"><div id="sref18" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref18" title="View in article">18.</a></div><div>Cho, E.A. ∙ Dressler, G.R.</div><div><strong>TCF-4 binds beta-catenin and is expressed in distinct regions of the embryonic brain and limbs</strong></div><div><em>Mech. Dev.</em> 1998; <strong>77</strong>:9-18</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/S0925-4773(98)00131-2" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-0032170317" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/9784592/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0925-4773%2898%2900131-2&amp;pmid=9784592" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib19" class="citations"><div id="sref19" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref19" title="View in article">19.</a></div><div>Choi, S.W. ∙ O’Reilly, P.F.</div><div><strong>PRSice-2: Polygenic Risk Score software for biobank-scale data</strong></div><div><em>Gigascience.</em> 2019; <strong>8</strong>:giz082</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/gigascience/giz082" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-85069904441" target="_blank" aria-label="Open Scopus in new tab">Scopus (190)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31307061/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fgigascience%2Fgiz082&amp;pmid=31307061" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib20" class="citations"><div id="sref20" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref20" title="View in article">20.</a></div><div>Claes, P. ∙ Roosenboom, J. ∙ White, J.D. ...</div><div><strong>Genome-wide mapping of global-to-local genetic effects on human facial shape</strong></div><div><em>Nat. Genet.</em> 2018; <strong>50</strong>:414-423</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41588-018-0057-4" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-85042196407" target="_blank" aria-label="Open Scopus in new tab">Scopus (89)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29459680/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41588-018-0057-4&amp;pmid=29459680" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib21" class="citations"><div id="sref21" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref21" title="View in article">21.</a></div><div>Compagni, A. ∙ Logan, M. ∙ Klein, R. ...</div><div><strong>Control of skeletal patterning by ephrinB1-EphB interactions</strong></div><div><em>Dev. Cell.</em> 2003; <strong>5</strong>:217-230</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=10.1016%2FS1534-5807%2803%2900198-9&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=10.1016%2FS1534-5807%2803%2900198-9&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-0042442460" target="_blank" aria-label="Open Scopus in new tab">Scopus (188)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/12919674/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS1534-5807%2803%2900198-9&amp;pmid=12919674" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib22" class="citations"><div id="sref22" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref22" title="View in article">22.</a></div><div>Cummins, H.</div><div><strong>The genetics of dermal ridges</strong></div><div><em>Am. J. Hum. Genet.</em> 1969; <strong>21</strong>:516</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=H.CumminsThe+genetics+of+dermal+ridgesAm.+J.+Hum.+Genet.211969516" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib23" class="citations"><div id="sref23" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref23" title="View in article">23.</a></div><div>Cummins, H. ∙ Midlo, C.</div><div><strong>Palmar and plantar epidermal ridge configurations (dermatoglyphics) in European-Americans</strong></div><div><em>Am. J. Phys. Anthropol.</em> 1926; <strong>9</strong>:471-502</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/ajpa.1330090422" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_23_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-0347914832" target="_blank" aria-label="Open Scopus in new tab">Scopus (64)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fajpa.1330090422" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib135" class="citations"><div id="sref135" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref135-1" title="View in article">24.</a></div><div>Davis, C.A. ∙ Hitz, B.C. ∙ Sloan, C.A. ...</div><div><strong>The Encyclopedia of DNA elements (ENCODE): data portal update</strong></div><div><em>Nucleic Acids Res.</em> 2018; <strong>46</strong>:D794-D801</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gkx1081" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-85040942752" target="_blank" aria-label="Open Scopus in new tab">Scopus (573)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29126249/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgkx1081&amp;pmid=29126249" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib24" class="citations"><div id="sref24" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref24-1" title="View in article">25.</a></div><div>Delaneau, O. ∙ Marchini, J. ∙ Zagury, J.F.</div><div><strong>A linear complexity phasing method for thousands of genomes</strong></div><div><em>Nat. Methods.</em> 2011; <strong>9</strong>:179-181</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.1785" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-84856478855" target="_blank" aria-label="Open Scopus in new tab">Scopus (1024)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22138821/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.1785&amp;pmid=22138821" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib25" class="citations"><div id="sref25" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref25" title="View in article">26.</a></div><div>Delous, M. ∙ Baala, L. ∙ Salomon, R. ...</div><div><strong>The ciliary gene RPGRIP1L is mutated in cerebello-oculo-renal syndrome (Joubert syndrome type B) and Meckel syndrome</strong></div><div><em>Nat. Genet.</em> 2007; <strong>39</strong>:875-881</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng2039" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_26_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-34347324031" target="_blank" aria-label="Open Scopus in new tab">Scopus (341)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17558409/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng2039&amp;pmid=17558409" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib26" class="citations"><div id="sref26" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref26" title="View in article">27.</a></div><div>Dickinson, M.E. ∙ Flenniken, A.M. ∙ Ji, X. ..., International Mouse Phenotyping Consortium, Jackson Laboratory, Infrastructure Nationale PHENOMIN, Institut Clinique de la Souris (ICS), Charles River Laboratories, MRC Harwell, Toronto Centre for Phenogenomics, Wellcome Trust Sanger Institute, RIKEN BioResource Center</div><div><strong>High-throughput discovery of novel developmental phenotypes</strong></div><div><em>Nature.</em> 2016; <strong>537</strong>:508-514</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature19356" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-84989204804" target="_blank" aria-label="Open Scopus in new tab">Scopus (513)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27626380/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature19356&amp;pmid=27626380" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib27" class="citations"><div id="sref27" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref27" title="View in article">28.</a></div><div>Dong, Y. ∙ Jesse, A.M. ∙ Kohn, A. ...</div><div><strong>RBPjkappa-dependent Notch signaling regulates mesenchymal progenitor cell proliferation and differentiation during skeletal development</strong></div><div><em>Development.</em> 2010; <strong>137</strong>:1461-1471</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1242/dev.042911" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-77951229591" target="_blank" aria-label="Open Scopus in new tab">Scopus (134)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20335360/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1242%2Fdev.042911&amp;pmid=20335360" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib28" class="citations"><div id="sref28" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref28" title="View in article">29.</a></div><div>Dubail, J. ∙ Aramaki-Hattori, N. ∙ Bader, H.L. ...</div><div><strong>A new Adamts9 conditional mouse allele identifies its non-redundant role in interdigital web regression</strong></div><div><em>Genesis.</em> 2014; <strong>52</strong>:702-712</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/dvg.22784" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24753090/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fdvg.22784&amp;pmid=24753090" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib29" class="citations"><div id="sref29" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref29" title="View in article">30.</a></div><div>Eppig, J.T. ∙ Smith, C.L. ∙ Blake, J.A. ...</div><div><strong>Mouse Genome Informatics (MGI): Resources for Mining Mouse Genetic, Genomic, and Biological Data in Support of Primary and Translational Research</strong></div><div>Springer New York, 2017</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.T.EppigC.L.SmithJ.A.BlakeM.RingwaldC.J.BultMouse+Genome+Informatics+%28MGI%29%3A+Resources+for+Mining+Mouse+Genetic%2C+Genomic%2C+and+Biological+Data+in+Support+of+Primary+and+Translational+Research2017Springer+New+York" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib30" class="citations"><div id="sref30" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref30" title="View in article">31.</a></div><div>Fatemifar, G. ∙ Hoggart, C.J. ∙ Paternoster, L. ...</div><div><strong>Genome-wide association study of primary tooth eruption identifies pleiotropic loci associated with height and craniofacial distances</strong></div><div><em>Hum. Mol. Genet.</em> 2013; <strong>22</strong>:3807-3817</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/hmg/ddt231" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-84883035112" target="_blank" aria-label="Open Scopus in new tab">Scopus (52)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23704328/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fhmg%2Fddt231&amp;pmid=23704328" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib31" class="citations"><div id="sref31" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref31" title="View in article">32.</a></div><div>Feng, W. ∙ Huang, J. ∙ Zhang, J. ...</div><div><strong>Identification and analysis of a conserved Tcfap2a intronic enhancer element required for expression in facial and limb bud mesenchyme</strong></div><div><em>Mol. Cell. Biol.</em> 2008; <strong>28</strong>:315-325</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/MCB.01168-07" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-37549020745" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17984226/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FMCB.01168-07&amp;pmid=17984226" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib32" class="citations"><div id="sref32" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref32" title="View in article">33.</a></div><div>Ferrer-Vaquer, A. ∙ Piliszek, A. ∙ Tian, G. ...</div><div><strong>A sensitive and bright single-cell resolution live imaging reporter of Wnt/ß-catenin signaling in the mouse</strong></div><div><em>BMC Dev. Biol.</em> 2010; <strong>10</strong>:121</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/1471-213X-10-121" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-78650255796" target="_blank" aria-label="Open Scopus in new tab">Scopus (185)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21176145/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2F1471-213X-10-121&amp;pmid=21176145" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib33" class="citations"><div id="sref33" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref33" title="View in article">34.</a></div><div>Fraser, A. ∙ Macdonald-Wallis, C. ∙ Tilling, K. ...</div><div><strong>Cohort Profile: the Avon Longitudinal Study of Parents and Children: ALSPAC mothers cohort</strong></div><div><em>Int. J. Epidemiol.</em> 2013; <strong>42</strong>:97-110</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/ije/dys066" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-84875619270" target="_blank" aria-label="Open Scopus in new tab">Scopus (1066)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22507742/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fije%2Fdys066&amp;pmid=22507742" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib34" class="citations"><div id="sref34" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref34-1" title="View in article">35.</a></div><div>Fujita, P.A. ∙ Rhead, B. ∙ Zweig, A.S. ...</div><div><strong>The UCSC Genome Browser database: update 2011</strong></div><div><em>Nucleic Acids Res.</em> 2011; <strong>39</strong>:D876-D882</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gkq963" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-78651320424" target="_blank" aria-label="Open Scopus in new tab">Scopus (815)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20959295/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgkq963&amp;pmid=20959295" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib35" class="citations"><div id="sref35" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref35" title="View in article">36.</a></div><div>Galton, F.</div><div><strong>Finger Prints</strong></div><div>Macmillan and Company, 1892</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=F.GaltonFinger+Prints1892Macmillan+and+Company" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib36" class="citations"><div id="sref36" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref36" title="View in article">37.</a></div><div>Gao, B. ∙ Yang, Y.</div><div><strong>Planar cell polarity in vertebrate limb morphogenesis</strong></div><div><em>Curr. Opin. Genet. Dev.</em> 2013; <strong>23</strong>:438-444</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.gde.2013.05.003" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-84883455409" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23747034/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.gde.2013.05.003&amp;pmid=23747034" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib37" class="citations"><div id="sref37" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref37" title="View in article">38.</a></div><div>Garzón-Alvarado, D.A. ∙ Ramírez Martinez, A.M.</div><div><strong>A biochemical hypothesis on the formation of fingerprints using a turing patterns approach</strong></div><div><em>Theor. Biol. Med. Model.</em> 2011; <strong>8</strong>:24</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/1742-4682-8-24" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21711537/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2F1742-4682-8-24&amp;pmid=21711537" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib38" class="citations"><div id="sref38" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref38" title="View in article">39.</a></div><div>Gerhardt, C. ∙ Lier, J.M. ∙ Burmühl, S. ...</div><div><strong>The transition zone protein Rpgrip1l regulates proteasomal activity at the primary cilium</strong></div><div><em>J.&nbsp;Cell Biol.</em> 2015; <strong>210</strong>:115-133</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1083/jcb.201408060" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-84957560272" target="_blank" aria-label="Open Scopus in new tab">Scopus (49)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26150391/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1083%2Fjcb.201408060&amp;pmid=26150391" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib39" class="citations"><div id="sref39" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref39" title="View in article">40.</a></div><div>Gibson-Brown, J.J. ∙ Agulnik, S.I. ∙ Chapman, D.L. ...</div><div><strong>Evidence of a role for T-box genes in the evolution of limb morphogenesis and the specification of forelimb/hindlimb identity</strong></div><div><em>Mech. Dev.</em> 1996; <strong>56</strong>:93-101</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/0925-4773(96)00514-X" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_40_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-0029977495" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/8798150/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2F0925-4773%2896%2900514-X&amp;pmid=8798150" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib40" class="citations"><div id="sref40" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref40" title="View in article">41.</a></div><div>Glass, C. ∙ Wilson, M. ∙ Gonzalez, R. ...</div><div><strong>The role of EVI1 in myeloid malignancies</strong></div><div><em>Blood Cells Mol. Dis.</em> 2014; <strong>53</strong>:67-76</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.bcmd.2014.01.002" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-84900033263" target="_blank" aria-label="Open Scopus in new tab">Scopus (46)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24495476/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.bcmd.2014.01.002&amp;pmid=24495476" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib134" class="citations"><div id="sref134" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref134-1" title="View in article">42.</a></div><div>GTEx Consortium</div><div><strong>Genetic effects on gene expression across human tissues</strong></div><div><em>Nature.</em> 2017; <strong>550</strong>:204-213</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature24277" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-85031313737" target="_blank" aria-label="Open Scopus in new tab">Scopus (1546)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29022597/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature24277&amp;pmid=29022597" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib41" class="citations"><div id="sref41" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref41" title="View in article">43.</a></div><div>Hamrick, M.W.</div><div><strong>Primate origins: evolutionary change in digital ray patterning and segmentation</strong></div><div><em>J.&nbsp;Hum. Evol.</em> 2001; <strong>40</strong>:339-351</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1006/jhev.2001.0467" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-0035074837" target="_blank" aria-label="Open Scopus in new tab">Scopus (72)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11312586/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1006%2Fjhev.2001.0467&amp;pmid=11312586" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib42" class="citations"><div id="sref42" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref42-1" title="View in article">44.</a></div><div>Han, B. ∙ Eskin, E.</div><div><strong>Random-effects model aimed at discovering associations in meta-analysis of genome-wide association studies</strong></div><div><em>Am. J. Hum. Genet.</em> 2011; <strong>88</strong>:586-598</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=10.1016%2Fj.ajhg.2011.04.014&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=10.1016%2Fj.ajhg.2011.04.014&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-79955860273" target="_blank" aria-label="Open Scopus in new tab">Scopus (297)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21565292/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.ajhg.2011.04.014&amp;pmid=21565292" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib133" class="citations"><div id="sref133" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref133-1" title="View in article">45.</a></div><div>Harrow, J. ∙ Frankish, A. ∙ Gonzalez, J.M. ...</div><div><strong>GENCODE: the reference human genome annotation for The ENCODE Project</strong></div><div><em>Genome research.</em> 2012; <strong>22</strong>:1760-1774</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/gr.135350.111" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_45_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-84865760395" target="_blank" aria-label="Open Scopus in new tab">Scopus (2415)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22955987/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2Fgr.135350.111&amp;pmid=22955987" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib43" class="citations"><div id="sref43" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref43" title="View in article">46.</a></div><div>Headon, D.J. ∙ Overbeek, P.A.</div><div><strong>Involvement of a novel Tnf receptor homologue in hair follicle induction</strong></div><div><em>Nat. Genet.</em> 1999; <strong>22</strong>:370-374</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/11943" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-0032775933" target="_blank" aria-label="Open Scopus in new tab">Scopus (298)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/10431242/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2F11943&amp;pmid=10431242" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib44" class="citations"><div id="sref44" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref44" title="View in article">47.</a></div><div>Hill, T.P. ∙ Taketo, M.M. ∙ Birchmeier, W. ...</div><div><strong>Multiple roles of mesenchymal beta-catenin during murine limb patterning</strong></div><div><em>Development.</em> 2006; <strong>133</strong>:1219-1229</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1242/dev.02298" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-33646130168" target="_blank" aria-label="Open Scopus in new tab">Scopus (104)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16495310/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1242%2Fdev.02298&amp;pmid=16495310" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib45" class="citations"><div id="sref45" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref45" title="View in article">48.</a></div><div>Hirsch, W. ∙ Schweichel, J.U.</div><div><strong>Morphological evidence concerning the problem of skin ridge formation</strong></div><div><em>J.&nbsp;Ment. Defic. Res.</em> 1973; <strong>17</strong>:58-72</div></div><div class="external-links"><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/4794986/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?pmid=4794986" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib46" class="citations"><div id="sref46" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref46-1" title="View in article">49.</a></div><div>Ho, Y.Y.W. ∙ Evans, D.M. ∙ Montgomery, G.W. ...</div><div><strong>Common Genetic Variants Influence Whorls in Fingerprint Patterns</strong></div><div><em>J.&nbsp;Invest. Dermatol.</em> 2016; <strong>136</strong>:859-862</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=10.1016%2Fj.jid.2015.10.062&amp;cf=fulltext&amp;site=jid-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=10.1016%2Fj.jid.2015.10.062&amp;cf=pdf&amp;site=jid-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-84978086948" target="_blank" aria-label="Open Scopus in new tab">Scopus (11)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27045867/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.jid.2015.10.062&amp;pmid=27045867" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib47" class="citations"><div id="sref47" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref47" title="View in article">50.</a></div><div>Holmes, G.P. ∙ Negus, K. ∙ Burridge, L. ...</div><div><strong>Distinct but overlapping expression patterns of two vertebrate slit homologs implies functional roles in CNS development and organogenesis</strong></div><div><em>Mech. Dev.</em> 1998; <strong>79</strong>:57-72</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/S0925-4773(98)00174-9" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-0032416836" target="_blank" aria-label="Open Scopus in new tab">Scopus (128)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/10349621/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0925-4773%2898%2900174-9&amp;pmid=10349621" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib48" class="citations"><div id="sref48" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref48" title="View in article">51.</a></div><div>Holt, S.B. ∙ Penrose, L.S.</div><div><strong>The Genetics of Dermal Ridges</strong></div><div>Charles C. Thomas, 1968</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=S.B.HoltL.S.PenroseThe+Genetics+of+Dermal+Ridges1968Charles+C.+Thomas" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib49" class="citations"><div id="sref49" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref49-1" title="View in article">52.</a></div><div>Howie, B.N. ∙ Donnelly, P. ∙ Marchini, J.</div><div><strong>A flexible and accurate genotype imputation method for the next generation of genome-wide association studies</strong></div><div><em>PLoS Genet.</em> 2009; <strong>5</strong>:e1000529</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pgen.1000529" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-67651222400" target="_blank" aria-label="Open Scopus in new tab">Scopus (2615)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19543373/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pgen.1000529&amp;pmid=19543373" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib50" class="citations"><div id="sref50" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref50-1" title="View in article">53.</a></div><div>Hoyt, P.R. ∙ Bartholomew, C. ∙ Davis, A.J. ...</div><div><strong>The Evi1 proto-oncogene is required at midgestation for neural, heart, and paraxial mesenchyme development</strong></div><div><em>Mech. Dev.</em> 1997; <strong>65</strong>:55-70</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/S0925-4773(97)00057-9" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-0031173051" target="_blank" aria-label="Open Scopus in new tab">Scopus (129)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/9256345/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0925-4773%2897%2900057-9&amp;pmid=9256345" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib130" class="citations"><div id="sref130" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref130" title="View in article">54.</a></div><div>The International HapMap Consortium</div><div><strong>The International HapMap Project</strong></div><div><em>Nature.</em> 2003; <strong>426</strong>:789-796</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature02168" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-79959503826" target="_blank" aria-label="Open Scopus in new tab">Scopus (4812)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/14685227/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature02168&amp;pmid=14685227" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib131" class="citations"><div id="sref131" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref131" title="View in article">55.</a></div><div>International HapMap ∙ Altshuler, C. ∙ Gibbs, D.M. ...</div><div><strong>Integrating common and rare genetic variation in diverse human populations</strong></div><div><em>Nature.</em> 2010; <strong>467</strong>:52-58</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature09298" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-77956331627" target="_blank" aria-label="Open Scopus in new tab">Scopus (1883)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20811451/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature09298&amp;pmid=20811451" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib51" class="citations"><div id="sref51" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref51-1" title="View in article">56.</a></div><div>Kang, H.M. ∙ Sul, J.H. ∙ Service, S.K. ...</div><div><strong>Variance component model to account for sample structure in genome-wide association studies</strong></div><div><em>Nat. Genet.</em> 2010; <strong>42</strong>:348-354</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng.548" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_56_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-77950301214" target="_blank" aria-label="Open Scopus in new tab">Scopus (1414)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20208533/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng.548&amp;pmid=20208533" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib52" class="citations"><div id="sref52" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref52" title="View in article">57.</a></div><div>Kargül, B. ∙ Alcan, T. ∙ Kabalay, U. ...</div><div><strong>Hypohidrotic ectodermal dysplasia: dental, clinical, genetic and dermatoglyphic findings of three cases</strong></div><div><em>J.&nbsp;Clin. Pediatr. Dent.</em> 2001; <strong>26</strong>:5-12</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.17796/jcpd.26.1.g156r02u66w30729" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11688814/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.17796%2Fjcpd.26.1.g156r02u66w30729&amp;pmid=11688814" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib53" class="citations"><div id="sref53" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref53" title="View in article">58.</a></div><div>Karmakar, B. ∙ Malkin, I. ∙ Kobyliansky, E.</div><div><strong>Inheritance of 18 quantitative dermatoglyphic traits based on factors in MZ and DZ twins</strong></div><div><em>Anthropol. Anz.</em> 2010; <strong>68</strong>:185-193</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1127/0003-5548/2011/0083" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-79952960577" target="_blank" aria-label="Open Scopus in new tab">Scopus (5)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21452683/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1127%2F0003-5548%2F2011%2F0083&amp;pmid=21452683" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib54" class="citations"><div id="sref54" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref54" title="View in article">59.</a></div><div>Kawakami, Y. ∙ Esteban, C.R. ∙ Matsui, T. ...</div><div><strong>Sp8 and Sp9, two closely related buttonhead-like transcription factors, regulate Fgf8 expression and limb outgrowth in vertebrate embryos</strong></div><div><em>Development.</em> 2004; <strong>131</strong>:4763-4774</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1242/dev.01331" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-7244248753" target="_blank" aria-label="Open Scopus in new tab">Scopus (123)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15358670/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1242%2Fdev.01331&amp;pmid=15358670" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib55" class="citations"><div id="sref55" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref55-1" title="View in article">60.</a></div><div>Kawakami, Y. ∙ Uchiyama, Y. ∙ Rodriguez Esteban, C. ...</div><div><strong>Sall genes regulate region-specific morphogenesis in the mouse limb by modulating Hox activities</strong></div><div><em>Development.</em> 2009; <strong>136</strong>:585-594</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1242/dev.027748" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-64549145826" target="_blank" aria-label="Open Scopus in new tab">Scopus (58)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19168674/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1242%2Fdev.027748&amp;pmid=19168674" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib56" class="citations"><div id="sref56" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref56-1" title="View in article">61.</a></div><div>Kichaev, G. ∙ Yang, W.Y. ∙ Lindstrom, S. ...</div><div><strong>Integrating functional data to prioritize causal variants in statistical fine-mapping studies</strong></div><div><em>PLoS Genet.</em> 2014; <strong>10</strong>:e1004722</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pgen.1004722" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-84908324508" target="_blank" aria-label="Open Scopus in new tab">Scopus (227)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25357204/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pgen.1004722&amp;pmid=25357204" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib57" class="citations"><div id="sref57" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref57-1" title="View in article">62.</a></div><div>Kohl, M. ∙ Wiese, S. ∙ Warscheid, B.</div><div><strong>Cytoscape: software for visualization and analysis of biological networks</strong></div><div><strong>Data Mining in Proteomics</strong></div><div>Springer, 2011; 291-303</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/978-1-60761-987-1_18" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-79952202448" target="_blank" aria-label="Open Scopus in new tab">Scopus (668)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2F978-1-60761-987-1_18" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib58" class="citations"><div id="sref58" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref58" title="View in article">63.</a></div><div>Kowalczyk-Quintas, C. ∙ Schneider, P.</div><div><strong>Ectodysplasin A (EDA) - EDA receptor signalling and its pharmacological modulation</strong></div><div><em>Cytokine Growth Factor Rev.</em> 2014; <strong>25</strong>:195-203</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.cytogfr.2014.01.004" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24508088/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cytogfr.2014.01.004&amp;pmid=24508088" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib59" class="citations"><div id="sref59" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref59-1" title="View in article">64.</a></div><div>Kücken, M.</div><div><strong>Models for fingerprint pattern formation</strong></div><div><em>Forensic Sci. Int.</em> 2007; <strong>171</strong>:85-96</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.forsciint.2007.02.025" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_64_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-34547691328" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17459625/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.forsciint.2007.02.025&amp;pmid=17459625" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib60" class="citations"><div id="sref60" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref60-1" title="View in article">65.</a></div><div>Lancman, J.J. ∙ Caruccio, N.C. ∙ Harfe, B.D. ...</div><div><strong>Analysis of the regulation of lin-41 during chick and mouse limb development</strong></div><div><em>Dev. Dyn.</em> 2005; <strong>234</strong>:948-960</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/dvdy.20591" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-28444491759" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16245339/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fdvdy.20591&amp;pmid=16245339" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib61" class="citations"><div id="sref61" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref61-1" title="View in article">66.</a></div><div>Lee, S.H. ∙ Yang, J. ∙ Goddard, M.E. ...</div><div><strong>Estimation of pleiotropy between complex diseases using single-nucleotide polymorphism-derived genomic relationships and restricted maximum likelihood</strong></div><div><em>Bioinformatics.</em> 2012; <strong>28</strong>:2540-2542</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/bts474" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-84867284056" target="_blank" aria-label="Open Scopus in new tab">Scopus (344)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22843982/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbts474&amp;pmid=22843982" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib62" class="citations"><div id="sref62" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref62" title="View in article">67.</a></div><div>Lehoczky, J.A. ∙ Williams, M.E. ∙ Innis, J.W.</div><div><strong>Conserved expression domains for genes upstream and within the HoxA and HoxD clusters suggests a long-range enhancer existed before cluster duplication</strong></div><div><em>Evol. Dev.</em> 2004; <strong>6</strong>:423-430</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1111/j.1525-142X.2004.04050.x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_67_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-9344246836" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15509224/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1111%2Fj.1525-142X.2004.04050.x&amp;pmid=15509224" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib63" class="citations"><div id="sref63" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref63" title="View in article">68.</a></div><div>Lewandowski, J.P. ∙ Du, F. ∙ Zhang, S. ...</div><div><strong>Spatiotemporal regulation of GLI target genes in the mammalian limb bud</strong></div><div><em>Dev. Biol.</em> 2015; <strong>406</strong>:92-103</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.ydbio.2015.07.022" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-84942195571" target="_blank" aria-label="Open Scopus in new tab">Scopus (32)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26238476/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.ydbio.2015.07.022&amp;pmid=26238476" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib64" class="citations"><div id="sref64" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref64" title="View in article">69.</a></div><div>Li, J. ∙ Ji, L.</div><div><strong>Adjusting multiple testing in multilocus analyses using the eigenvalues of a correlation matrix</strong></div><div><em>Heredity.</em> 2005; <strong>95</strong>:221-227</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/sj.hdy.6800717" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-30744434862" target="_blank" aria-label="Open Scopus in new tab">Scopus (913)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16077740/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fsj.hdy.6800717&amp;pmid=16077740" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib65" class="citations"><div id="sref65" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref65-1" title="View in article">70.</a></div><div>Li, D. ∙ Sakuma, R. ∙ Vakili, N.A. ...</div><div><strong>Formation of proximal and anterior limb skeleton requires early function of Irx3 and Irx5 and is negatively regulated by Shh signaling</strong></div><div><em>Dev. Cell.</em> 2014; <strong>29</strong>:233-240</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=10.1016%2Fj.devcel.2014.03.001&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=10.1016%2Fj.devcel.2014.03.001&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-84899634426" target="_blank" aria-label="Open Scopus in new tab">Scopus (67)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24726282/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.devcel.2014.03.001&amp;pmid=24726282" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib138" class="citations"><div id="sref138" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref138-1" title="View in article">71.</a></div><div>Li, Y. ∙ Willer, C.J. ∙ Ding, J. ...</div><div><strong>MaCH: using sequence and genotype data to estimate haplotypes and unobserved genotypes</strong></div><div><em>Genet Epidemiol.</em> 2010; <strong>34</strong>:816-834</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/gepi.20533" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-78649508578" target="_blank" aria-label="Open Scopus in new tab">Scopus (1447)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21058334/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fgepi.20533&amp;pmid=21058334" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib66" class="citations"><div id="sref66" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref66-1" title="View in article">72.</a></div><div>Liu, J. ∙ Li, Q. ∙ Kuehn, M.R. ...</div><div><strong>Sonic hedgehog signaling directly targets Hyaluronic Acid Synthase 2, an essential regulator of phalangeal joint patterning</strong></div><div><em>Dev. Biol.</em> 2013; <strong>375</strong>:160-171</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.ydbio.2012.12.018" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-84874251615" target="_blank" aria-label="Open Scopus in new tab">Scopus (17)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23313125/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.ydbio.2012.12.018&amp;pmid=23313125" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib67" class="citations"><div id="sref67" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref67" title="View in article">73.</a></div><div>Loesch, D.Z.</div><div><strong>Quantitative Dermatoglyphics. Classification, Genetics, and Pathology</strong></div><div>Oxford University Press, 1983</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=D.Z.LoeschQuantitative+Dermatoglyphics.+Classification%2C+Genetics%2C+and+Pathology1983Oxford+University+Press" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib68" class="citations"><div id="sref68" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref68" title="View in article">74.</a></div><div>Loesch, D.Z. ∙ Martin, N.G.</div><div><strong>Finger ridge patterns and tactile sensitivity</strong></div><div><em>Ann. Hum. Biol.</em> 1984; <strong>11</strong>:113-124</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1080/03014468400006961" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/6732178/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1080%2F03014468400006961&amp;pmid=6732178" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib137" class="citations"><div id="sref137" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref137-1" title="View in article">75.</a></div><div>Loh, P.R. ∙ Tucker, G. ∙ Bulik-Sullivan, B.K. ...</div><div><strong>Efficient Bayesian mixed-model analysis increases association power in large cohorts</strong></div><div><em>. Nat Genet.</em> 2015; <strong>47</strong>:284-290</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng.3190" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_75_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-84924060689" target="_blank" aria-label="Open Scopus in new tab">Scopus (693)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25642633/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng.3190&amp;pmid=25642633" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib69" class="citations"><div id="sref69" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref69" title="View in article">76.</a></div><div>Ma, P. ∙ Song, N.N. ∙ Li, Y. ...</div><div><strong>Fine-Tuning of Shh/Gli Signaling Gradient by Non-proteolytic Ubiquitination during Neural Patterning</strong></div><div><em>Cell Rep.</em> 2019; <strong>28</strong>:541-553.e4</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_76_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=10.1016%2Fj.celrep.2019.06.017&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_76_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=10.1016%2Fj.celrep.2019.06.017&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_76_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-85068110805" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31291587/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.celrep.2019.06.017&amp;pmid=31291587" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib70" class="citations"><div id="sref70" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref70" title="View in article">77.</a></div><div>Machado, J.F. ∙ Fernandes, P.R. ∙ Roquetti, R.W. ...</div><div><strong>Digital dermatoglyphic heritability differences as evidenced by a female twin study</strong></div><div><em>Twin Res. Hum. Genet.</em> 2010; <strong>13</strong>:482-489</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1375/twin.13.5.482" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20874471/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1375%2Ftwin.13.5.482&amp;pmid=20874471" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib71" class="citations"><div id="sref71" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref71" title="View in article">78.</a></div><div>Mankoo, B.S. ∙ Collins, N.S. ∙ Ashby, P. ...</div><div><strong>Mox2 is a component of the genetic hierarchy controlling limb muscle development</strong></div><div><em>Nature.</em> 1999; <strong>400</strong>:69-73</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/21892" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_78_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-0033167972" target="_blank" aria-label="Open Scopus in new tab">Scopus (145)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/10403250/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2F21892&amp;pmid=10403250" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib72" class="citations"><div id="sref72" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref72" title="View in article">79.</a></div><div>Mariani, F.V. ∙ Ahn, C.P. ∙ Martin, G.R.</div><div><strong>Genetic evidence that FGFs have an instructive role in limb proximal-distal patterning</strong></div><div><em>Nature.</em> 2008; <strong>453</strong>:401-405</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature06876" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_79_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-43749110077" target="_blank" aria-label="Open Scopus in new tab">Scopus (253)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/18449196/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature06876&amp;pmid=18449196" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib73" class="citations"><div id="sref73" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref73" title="View in article">80.</a></div><div>Martin, N.G. ∙ Eaves, L.J. ∙ Loesch, D.Z.</div><div><strong>A genetical analysis of covariation between finger ridge counts</strong></div><div><em>Ann. Hum. Biol.</em> 1982; <strong>9</strong>:539-552</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1080/03014468200006061" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/7181445/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1080%2F03014468200006061&amp;pmid=7181445" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib74" class="citations"><div id="sref74" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref74" title="View in article">81.</a></div><div>Matsuda, T. ∙ Nomi, M. ∙ Ikeya, M. ...</div><div><strong>Expression of the receptor tyrosine kinase genes, Ror1 and Ror2, during mouse development</strong></div><div><em>Mech. Dev.</em> 2001; <strong>105</strong>:153-156</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/S0925-4773(01)00383-5" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-0034948866" target="_blank" aria-label="Open Scopus in new tab">Scopus (111)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11429290/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0925-4773%2801%2900383-5&amp;pmid=11429290" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib75" class="citations"><div id="sref75" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref75" title="View in article">82.</a></div><div>McCulloch, D.R. ∙ Nelson, C.M. ∙ Dixon, L.J. ...</div><div><strong>ADAMTS metalloproteases generate active versican fragments that regulate interdigital web regression</strong></div><div><em>Dev. Cell.</em> 2009; <strong>17</strong>:687-698</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_82_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=10.1016%2Fj.devcel.2009.09.008&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_82_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=10.1016%2Fj.devcel.2009.09.008&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_82_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-71549163138" target="_blank" aria-label="Open Scopus in new tab">Scopus (178)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19922873/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.devcel.2009.09.008&amp;pmid=19922873" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib76" class="citations"><div id="sref76" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref76-1" title="View in article">83.</a></div><div>McLean, C.Y. ∙ Bristor, D. ∙ Hiller, M. ...</div><div><strong>GREAT improves functional interpretation of cis-regulatory regions</strong></div><div><em>Nat. Biotechnol.</em> 2010; <strong>28</strong>:495-501</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nbt.1630" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_83_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-77952214662" target="_blank" aria-label="Open Scopus in new tab">Scopus (2170)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20436461/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnbt.1630&amp;pmid=20436461" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib77" class="citations"><div id="sref77" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref77-1" title="View in article">84.</a></div><div>Medland, S.E. ∙ Loesch, D.Z. ∙ Mdzewski, B. ...</div><div><strong>Linkage analysis of a model quantitative trait in humans: finger ridge count shows significant multivariate linkage to 5q14.1</strong></div><div><em>PLoS Genet.</em> 2007; <strong>3</strong>:1736-1744</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pgen.0030165" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_84_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-34848862129" target="_blank" aria-label="Open Scopus in new tab">Scopus (39)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17907812/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pgen.0030165&amp;pmid=17907812" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib78" class="citations"><div id="sref78" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref78" title="View in article">85.</a></div><div>Medland, S.E. ∙ Zayats, T. ∙ Glaser, B. ...</div><div><strong>A variant in LIN28B is associated with 2D:4D finger-length ratio, a putative retrospective biomarker of prenatal testosterone exposure</strong></div><div><em>Am. J. Hum. Genet.</em> 2010; <strong>86</strong>:519-525</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=10.1016%2Fj.ajhg.2010.02.017&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=10.1016%2Fj.ajhg.2010.02.017&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-77950304213" target="_blank" aria-label="Open Scopus in new tab">Scopus (78)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20303062/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.ajhg.2010.02.017&amp;pmid=20303062" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib79" class="citations"><div id="sref79" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref79" title="View in article">86.</a></div><div>Menser, M.A. ∙ Purvis-Smith, S.G.</div><div><strong>Dermatoglyphic defects in children with leukaemia</strong></div><div><em>Lancet.</em> 1969; <strong>1</strong>:1076-1078</div></div><div class="external-links"><div class="core-xlink-abstract"><a href="/servlet/linkout?suffix=e_1_5_1_2_86_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=10.1016%2FS0140-6736%2869%2991710-3&amp;cf=abstract&amp;site=lancet-site" target="_blank" aria-label="Open Abstract in new tab">Abstract</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/4181736/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0140-6736%2869%2991710-3&amp;pmid=4181736" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib80" class="citations"><div id="sref80" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref80" title="View in article">87.</a></div><div>Mikels, A.J. ∙ Nusse, R.</div><div><strong>Purified Wnt5a protein activates or inhibits beta-catenin-TCF signaling depending on receptor context</strong></div><div><em>PLoS Biol.</em> 2006; <strong>4</strong>:e115</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pbio.0040115" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_87_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-33646241997" target="_blank" aria-label="Open Scopus in new tab">Scopus (961)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16602827/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pbio.0040115&amp;pmid=16602827" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib81" class="citations"><div id="sref81" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref81" title="View in article">88.</a></div><div>Monreal, A.W. ∙ Ferguson, B.M. ∙ Headon, D.J. ...</div><div><strong>Mutations in the human homologue of mouse dl cause autosomal recessive and dominant hypohidrotic ectodermal dysplasia</strong></div><div><em>Nat. Genet.</em> 1999; <strong>22</strong>:366-369</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/11937" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-0032811085" target="_blank" aria-label="Open Scopus in new tab">Scopus (319)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/10431241/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2F11937&amp;pmid=10431241" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib82" class="citations"><div id="sref82" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref82" title="View in article">89.</a></div><div>Mou, C. ∙ Jackson, B. ∙ Schneider, P. ...</div><div><strong>Generation of the primary hair follicle pattern</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2006; <strong>103</strong>:9075-9080</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.0600825103" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_89_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-33745121169" target="_blank" aria-label="Open Scopus in new tab">Scopus (149)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16769906/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.0600825103&amp;pmid=16769906" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib83" class="citations"><div id="sref83" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref83" title="View in article">90.</a></div><div>Mulvihill, J.J. ∙ Smith, D.W.</div><div><strong>The genesis of dermatoglyphics</strong></div><div><em>J.&nbsp;Pediatr.</em> 1969; <strong>75</strong>:579-589</div></div><div class="external-links"><div class="core-xlink-abstract"><a href="/servlet/linkout?suffix=e_1_5_1_2_90_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=10.1016%2FS0022-3476%2869%2980453-1&amp;cf=abstract&amp;site=ympd-site" target="_blank" aria-label="Open Abstract in new tab">Abstract</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_90_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=10.1016%2FS0022-3476%2869%2980453-1&amp;cf=pdf&amp;site=ympd-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/4309281/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0022-3476%2869%2980453-1&amp;pmid=4309281" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib84" class="citations"><div id="sref84" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref84" title="View in article">91.</a></div><div>Nagy, A.S. ∙ Pap, M.</div><div><strong>Pattern influence on the fingers</strong></div><div><em>Homo.</em> 2005; <strong>56</strong>:51-67</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.jchb.2004.10.003" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-16844367875" target="_blank" aria-label="Open Scopus in new tab">Scopus (4)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15901118/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.jchb.2004.10.003&amp;pmid=15901118" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib85" class="citations"><div id="sref85" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref85" title="View in article">92.</a></div><div>Nandadasa, S. ∙ Kraft, C.M. ∙ Wang, L.W. ...</div><div><strong>Secreted metalloproteases ADAMTS9 and ADAMTS20 have a non-canonical role in ciliary vesicle growth during ciliogenesis</strong></div><div><em>Nat. Commun.</em> 2019; <strong>10</strong>:953</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41467-019-08520-7" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_92_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-85062261861" target="_blank" aria-label="Open Scopus in new tab">Scopus (27)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30814516/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41467-019-08520-7&amp;pmid=30814516" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib86" class="citations"><div id="sref86" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref86" title="View in article">93.</a></div><div>Nohno, T. ∙ Koyama, E. ∙ Myokai, F. ...</div><div><strong>A chicken homeobox gene related to Drosophila paired is predominantly expressed in the developing limb</strong></div><div><em>Dev. Biol.</em> 1993; <strong>158</strong>:254-264</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1006/dbio.1993.1184" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_93_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-0027249822" target="_blank" aria-label="Open Scopus in new tab">Scopus (65)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/8101172/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1006%2Fdbio.1993.1184&amp;pmid=8101172" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib87" class="citations"><div id="sref87" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref87" title="View in article">94.</a></div><div>Okajima, M.</div><div><strong>Development of dermal ridges in the fetus</strong></div><div><em>J.&nbsp;Med. Genet.</em> 1975; <strong>12</strong>:243-250</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1136/jmg.12.3.243" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/51929/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1136%2Fjmg.12.3.243&amp;pmid=51929" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib88" class="citations"><div id="sref88" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref88" title="View in article">95.</a></div><div>Ota, S. ∙ Zhou, Z.Q. ∙ Keene, D.R. ...</div><div><strong>Activities of N-Myc in the developing limb link control of skeletal size with digit separation</strong></div><div><em>Development.</em> 2007; <strong>134</strong>:1583-1592</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1242/dev.000703" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_95_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-34248589385" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17360777/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1242%2Fdev.000703&amp;pmid=17360777" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib89" class="citations"><div id="sref89" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref89" title="View in article">96.</a></div><div>Painter, K.J. ∙ Hunt, G.S. ∙ Wells, K.L. ...</div><div><strong>Towards an integrated experimental-theoretical approach for assessing the mechanistic basis of hair and feather morphogenesis</strong></div><div><em>Interface Focus.</em> 2012; <strong>2</strong>:433-450</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1098/rsfs.2011.0122" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_96_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-84862663863" target="_blank" aria-label="Open Scopus in new tab">Scopus (23)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23919127/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1098%2Frsfs.2011.0122&amp;pmid=23919127" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib90" class="citations"><div id="sref90" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref90" title="View in article">97.</a></div><div>Parkinson, N. ∙ Hardisty-Hughes, R.E. ∙ Tateossian, H. ...</div><div><strong>Mutation at the Evi1 locus in Junbo mice causes susceptibility to otitis media</strong></div><div><em>PLoS Genet.</em> 2006; <strong>2</strong>:e149</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pgen.0020149" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_97_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-39049182374" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17029558/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pgen.0020149&amp;pmid=17029558" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib91" class="citations"><div id="sref91" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref91" title="View in article">98.</a></div><div>Penrose, L.S.</div><div><strong>Dermatoglyphic Topology</strong></div><div><em>Nature.</em> 1965; <strong>205</strong>:544-546</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/205544a0" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_98_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-0010505869" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2F205544a0" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib92" class="citations"><div id="sref92" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref92" title="View in article">99.</a></div><div>Penrose, L.S.</div><div><strong>On the geometry of loops and deltas</strong></div><div><em>Ann. Hum. Genet.</em> 1965; <strong>29</strong>:207-211</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1111/j.1469-1809.1965.tb00515.x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_99_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-0013815364" target="_blank" aria-label="Open Scopus in new tab">Scopus (5)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/5863842/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1111%2Fj.1469-1809.1965.tb00515.x&amp;pmid=5863842" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib93" class="citations"><div id="sref93" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref93" title="View in article">100.</a></div><div>Penrose, L.S. ∙ Ohara, P.T.</div><div><strong>The development of the epidermal ridges</strong></div><div><em>J.&nbsp;Med. Genet.</em> 1973; <strong>10</strong>:201-208</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1136/jmg.10.3.201" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/4774533/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1136%2Fjmg.10.3.201&amp;pmid=4774533" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib94" class="citations"><div id="sref94" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref94" title="View in article">101.</a></div><div>Pierani, A. ∙ Brenner-Morton, S. ∙ Chiang, C. ...</div><div><strong>A sonic hedgehog-independent, retinoid-activated pathway of neurogenesis in the ventral spinal cord</strong></div><div><em>Cell.</em> 1999; <strong>97</strong>:903-915</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_101_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=10.1016%2FS0092-8674%2800%2980802-8&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_101_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=10.1016%2FS0092-8674%2800%2980802-8&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_101_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-0033603209" target="_blank" aria-label="Open Scopus in new tab">Scopus (279)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/10399918/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0092-8674%2800%2980802-8&amp;pmid=10399918" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib95" class="citations"><div id="sref95" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref95-1" title="View in article">102.</a></div><div>Price, A.L. ∙ Patterson, N.J. ∙ Plenge, R.M. ...</div><div><strong>Principal components analysis corrects for stratification in genome-wide association studies</strong></div><div><em>Nat. Genet.</em> 2006; <strong>38</strong>:904-909</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng1847" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_102_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-33746512512" target="_blank" aria-label="Open Scopus in new tab">Scopus (6332)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16862161/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng1847&amp;pmid=16862161" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib139" class="citations"><div id="sref139" class="citation"><div class="citation-content"><div class="label"><a href="#fig2" title="View in article">103.</a></div><div>Pruim, R.J. ∙ Welch, R.P. ∙ Sanna, S. ...</div><div><strong>LocusZoom: regional visualization of genome-wide association scan results</strong></div><div><em>Bioinformatics.</em> 2010; <strong>26</strong>:2336-2337</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btq419" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_103_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-77956586071" target="_blank" aria-label="Open Scopus in new tab">Scopus (1476)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20634204/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtq419&amp;pmid=20634204" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib132" class="citations"><div id="sref132" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref132-1" title="View in article">104.</a></div><div>Pruitt, K.D. ∙ Brown, G.R. ∙ Hiatt, S.M. ...</div><div><strong>RefSeq: an update on mammalian reference sequences</strong></div><div><em>Nucleic Acids Res.</em> 2014; <strong>42</strong>:D756-D763</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gkt1114" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_104_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-84891767394" target="_blank" aria-label="Open Scopus in new tab">Scopus (651)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24259432/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgkt1114&amp;pmid=24259432" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib96" class="citations"><div id="sref96" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref96-1" title="View in article">105.</a></div><div>Purcell, S. ∙ Neale, B. ∙ Todd-Brown, K. ...</div><div><strong>PLINK: a tool set for whole-genome association and population-based linkage analyses</strong></div><div><em>Am. J. Hum. Genet.</em> 2007; <strong>81</strong>:559-575</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_105_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=10.1086%2F519795&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_105_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=10.1086%2F519795&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_105_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-34548292504" target="_blank" aria-label="Open Scopus in new tab">Scopus (17562)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17701901/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1086%2F519795&amp;pmid=17701901" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib97" class="citations"><div id="sref97" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref97-1" title="View in article">106.</a></div><div>Rao, R. ∙ Gornbein, J. ∙ Afshar, Y. ...</div><div><strong>A new biometric: In utero growth curves for metacarpal and phalangeal lengths reveal an embryonic patterning ratio</strong></div><div><em>Prenat. Diagn.</em> 2019; <strong>39</strong>:200-208</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/pd.5397" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_106_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-85061354603" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30488473/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fpd.5397&amp;pmid=30488473" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib98" class="citations"><div id="sref98" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref98" title="View in article">107.</a></div><div>Rathee, R. ∙ Kamal, N. ∙ Kumar, A. ...</div><div><strong>Dermatoglyphic Patterns of Acute Leukemia Patients</strong></div><div><em>Int. Res. J. Biol. Sci.</em> 2014; <strong>3</strong>:90-93</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=R.RatheeN.KamalA.KumarM.VashistR.YadavDermatoglyphic+Patterns+of+Acute+Leukemia+PatientsInt.+Res.+J.+Biol.+Sci.320149093" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib99" class="citations"><div id="sref99" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref99" title="View in article">108.</a></div><div>Reed, T. ∙ Sprague, F.R. ∙ Kang, K.W. ...</div><div><strong>Genetic analysis of dermatoglyphic patterns in twins</strong></div><div><em>Hum. Hered.</em> 1975; <strong>25</strong>:263-275</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1159/000152734" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/1237463/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1159%2F000152734&amp;pmid=1237463" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib100" class="citations"><div id="sref100" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref100" title="View in article">109.</a></div><div>Robertson, E.J. ∙ Charatsi, I. ∙ Joyner, C.J. ...</div><div><strong>Blimp1 regulates development of the posterior forelimb, caudal pharyngeal arches, heart and sensory vibrissae in mice</strong></div><div><em>Development.</em> 2007; <strong>134</strong>:4335-4345</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1242/dev.012047" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_109_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-38349080867" target="_blank" aria-label="Open Scopus in new tab">Scopus (98)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/18039967/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1242%2Fdev.012047&amp;pmid=18039967" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib101" class="citations"><div id="sref101" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref101" title="View in article">110.</a></div><div>Rosner, F.</div><div><strong>Dermatoglyphics of leukaemic children</strong></div><div><em>Lancet.</em> 1969; <strong>2</strong>:272-273</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="/servlet/linkout?suffix=e_1_5_1_2_110_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=10.1016%2FS0140-6736%2869%2990043-9&amp;site=lancet-site" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/4184135/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0140-6736%2869%2990043-9&amp;pmid=4184135" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib102" class="citations"><div id="sref102" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref102" title="View in article">111.</a></div><div>Saad, K. ∙ Theis, S. ∙ Otto, A. ...</div><div><strong>Detailed expression profile of the six Glypicans and their modifying enzyme, Notum during chick limb and feather development</strong></div><div><em>Gene.</em> 2017; <strong>610</strong>:71-79</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.gene.2017.02.012" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_111_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-85014296035" target="_blank" aria-label="Open Scopus in new tab">Scopus (3)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28192166/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.gene.2017.02.012&amp;pmid=28192166" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib103" class="citations"><div id="sref103" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref103" title="View in article">112.</a></div><div>Scheibert, J. ∙ Leurent, S. ∙ Prevost, A. ...</div><div><strong>The role of fingerprints in the coding of tactile information probed with a biomimetic sensor</strong></div><div><em>Science.</em> 2009; <strong>323</strong>:1503-1506</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.1166467" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_112_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-62449256629" target="_blank" aria-label="Open Scopus in new tab">Scopus (251)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19179493/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.1166467&amp;pmid=19179493" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib104" class="citations"><div id="sref104" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref104" title="View in article">113.</a></div><div>Sengupta, M. ∙ Karmakar, B.</div><div><strong>Mode of inheritance of finger dermatoglyphic traits among Vaidyas of West Bengal, India</strong></div><div><em>Ann. Hum. Biol.</em> 2004; <strong>31</strong>:526-540</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1080/03014460412331287164" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_113_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-10344236018" target="_blank" aria-label="Open Scopus in new tab">Scopus (25)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15739382/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1080%2F03014460412331287164&amp;pmid=15739382" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib105" class="citations"><div id="sref105" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref105" title="View in article">114.</a></div><div>Sheehan-Rooney, K. ∙ Pálinkášová, B. ∙ Eberhart, J.K. ...</div><div><strong>A cross-species analysis of Satb2 expression suggests deep conservation across vertebrate lineages</strong></div><div><em>Dev. Dyn.</em> 2010; <strong>239</strong>:3481-3491</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/dvdy.22483" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_114_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-78649712329" target="_blank" aria-label="Open Scopus in new tab">Scopus (50)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21089028/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fdvdy.22483&amp;pmid=21089028" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib106" class="citations"><div id="sref106" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref106-1" title="View in article">115.</a></div><div>Sherry, S.T. ∙ Ward, M.-H. ∙ Kholodov, M. ...</div><div><strong>dbSNP: the NCBI database of genetic variation</strong></div><div><em>Nucleic Acids Res.</em> 2001; <strong>29</strong>:308-311</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/29.1.308" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11125122/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2F29.1.308&amp;pmid=11125122" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib107" class="citations"><div id="sref107" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref107" title="View in article">116.</a></div><div>Shima, Y. ∙ Copeland, N.G. ∙ Gilbert, D.J. ...</div><div><strong>Differential expression of the seven-pass transmembrane cadherin genes Celsr1-3 and distribution of the Celsr2 protein during mouse development</strong></div><div><em>Dev. Dyn.</em> 2002; <strong>223</strong>:321-332</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/dvdy.10054" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_116_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-0036198940" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11891983/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fdvdy.10054&amp;pmid=11891983" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib108" class="citations"><div id="sref108" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref108" title="View in article">117.</a></div><div>Spielmann, M. ∙ Kakar, N. ∙ Tayebi, N. ...</div><div><strong>Exome sequencing and CRISPR/Cas genome editing identify mutations of ZAK as a cause of limb defects in humans and mice</strong></div><div><em>Genome Res.</em> 2016; <strong>26</strong>:183-191</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/gr.199430.115" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_117_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-84956905665" target="_blank" aria-label="Open Scopus in new tab">Scopus (31)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26755636/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2Fgr.199430.115&amp;pmid=26755636" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib109" class="citations"><div id="sref109" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref109" title="View in article">118.</a></div><div>Srivastava, A.K. ∙ Pispa, J. ∙ Hartung, A.J. ...</div><div><strong>The Tabby phenotype is caused by mutation in a mouse homologue of the EDA gene that reveals novel mouse and human exons and encodes a protein (ectodysplasin-A) with collagenous domains</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 1997; <strong>94</strong>:13069-13074</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.94.24.13069" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_118_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-12644310324" target="_blank" aria-label="Open Scopus in new tab">Scopus (246)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/9371801/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.94.24.13069&amp;pmid=9371801" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib110" class="citations"><div id="sref110" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref110" title="View in article">119.</a></div><div>Szenker-Ravi, E. ∙ Altunoglu, U. ∙ Leushacke, M. ...</div><div><strong>RSPO2 inhibition of RNF43 and ZNRF3 governs limb development independently of LGR4/5/6</strong></div><div><em>Nature.</em> 2018; <strong>557</strong>:564-569</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-018-0118-y" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_119_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-85047253329" target="_blank" aria-label="Open Scopus in new tab">Scopus (76)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29769720/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-018-0118-y&amp;pmid=29769720" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib111" class="citations"><div id="sref111" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref111-1" title="View in article">120.</a></div><div>Szklarczyk, D. ∙ Gable, A.L. ∙ Lyon, D. ...</div><div><strong>STRING v11: protein-protein association networks with increased coverage, supporting functional discovery in genome-wide experimental datasets</strong></div><div><em>Nucleic Acids Res.</em> 2019; <strong>47</strong>:D607-D613</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gky1131" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_120_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-85059797047" target="_blank" aria-label="Open Scopus in new tab">Scopus (4748)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30476243/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgky1131&amp;pmid=30476243" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib136" class="citations"><div id="sref136" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref136" title="View in article">121.</a></div><div>Tenenhaus, M. ∙ Vinzi, V.E. ∙ Chatelin, Y.M. ...</div><div><strong>PLS path modeling</strong></div><div><em>Comput Stat Data Anal.</em> 2005; <strong>48</strong>:159-205</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.csda.2004.03.005" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_121_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-10144243373" target="_blank" aria-label="Open Scopus in new tab">Scopus (3076)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.csda.2004.03.005" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib112" class="citations"><div id="sref112" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref112-1" title="View in article">122.</a></div><div>Towers, M. ∙ Tickle, C.</div><div><strong>Growing models of vertebrate limb development</strong></div><div><em>Development.</em> 2009; <strong>136</strong>:179-190</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1242/dev.024158" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_122_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-62349111082" target="_blank" aria-label="Open Scopus in new tab">Scopus (133)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19103802/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1242%2Fdev.024158&amp;pmid=19103802" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib113" class="citations"><div id="sref113" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref113" title="View in article">123.</a></div><div>Tsugane, M. ∙ Yasuda, M.</div><div><strong>Dermatoglyphics on volar skin of mice: the normal pattern</strong></div><div><em>Anat. Rec.</em> 1995; <strong>242</strong>:225-232</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/ar.1092420212" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_123_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-0029027841" target="_blank" aria-label="Open Scopus in new tab">Scopus (19)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/7668408/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Far.1092420212&amp;pmid=7668408" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib114" class="citations"><div id="sref114" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref114-1" title="View in article">124.</a></div><div>Turner, S.D.</div><div><strong>qqman: an R package for visualizing GWAS results using QQ and manhattan plots</strong></div><div><em>bioRxiv.</em> 2014;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/005165" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2F005165" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib115" class="citations"><div id="sref115" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref115" title="View in article">125.</a></div><div>van Kleffens, M. ∙ Groffen, C. ∙ Rosato, R.R. ...</div><div><strong>mRNA expression patterns of the IGF system during mouse limb bud development, determined by whole mount in&nbsp;situ hybridization</strong></div><div><em>Mol. Cell. Endocrinol.</em> 1998; <strong>138</strong>:151-161</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/S0303-7207(98)00007-0" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_125_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-0032536742" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/9685224/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0303-7207%2898%2900007-0&amp;pmid=9685224" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib116" class="citations"><div id="sref116" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref116" title="View in article">126.</a></div><div>Verbov, J.</div><div><strong>Hypohidrotic (or anhidrotic) ectodermal dysplasia--an appraisal of diagnostic methods</strong></div><div><em>Br. J. Dermatol.</em> 1970; <strong>83</strong>:341-348</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1111/j.1365-2133.1970.tb15713.x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_126_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-0014845184" target="_blank" aria-label="Open Scopus in new tab">Scopus (27)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/5479305/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1111%2Fj.1365-2133.1970.tb15713.x&amp;pmid=5479305" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib117" class="citations"><div id="sref117" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref117" title="View in article">127.</a></div><div>Wang, X. ∙ Lu, M. ∙ Qian, J. ...</div><div><strong>Rationales, design and recruitment of the Taizhou Longitudinal Study</strong></div><div><em>BMC Public Health.</em> 2009; <strong>9</strong>:223</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/1471-2458-9-223" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_127_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-68049137338" target="_blank" aria-label="Open Scopus in new tab">Scopus (89)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19589173/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2F1471-2458-9-223&amp;pmid=19589173" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib118" class="citations"><div id="sref118" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref118" title="View in article">128.</a></div><div>Wang, Y. ∙ Song, F. ∙ Zhang, B. ...</div><div><strong>The 3D Genome Browser: a web-based browser for visualizing 3D genome organization and long-range chromatin interactions</strong></div><div><em>Genome Biol.</em> 2018; <strong>19</strong>:151</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13059-018-1519-9" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_128_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-85054460140" target="_blank" aria-label="Open Scopus in new tab">Scopus (171)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30286773/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13059-018-1519-9&amp;pmid=30286773" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib119" class="citations"><div id="sref119" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref119-1" title="View in article">129.</a></div><div>Ward, L.D. ∙ Kellis, M.</div><div><strong>HaploReg: a resource for exploring chromatin states, conservation, and regulatory motif alterations within sets of genetically linked variants</strong></div><div><em>Nucleic Acids Res.</em> 2012; <strong>40</strong>:D930-D934</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gkr917" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_129_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-84858779229" target="_blank" aria-label="Open Scopus in new tab">Scopus (1526)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22064851/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgkr917&amp;pmid=22064851" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib120" class="citations"><div id="sref120" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref120-1" title="View in article">130.</a></div><div>Weinberg, S.M. ∙ Neiswanger, K. ∙ Martin, R.A. ...</div><div><strong>The Pittsburgh Oral-Facial Cleft study: expanding the cleft phenotype. Background and justification</strong></div><div><em>Cleft Palate Craniofac. J.</em> 2006; <strong>43</strong>:7-20</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1597/04-122R1.1" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_130_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-32044438969" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16405378/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1597%2F04-122R1.1&amp;pmid=16405378" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib121" class="citations"><div id="sref121" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref121" title="View in article">131.</a></div><div>Wertheim, K. ∙ Maceo, A.</div><div><strong>The critical stage of friction ridge and pattern formation</strong></div><div><em>J.&nbsp;Forensic Identif.</em> 2002; <strong>52</strong>:35</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=K.WertheimA.MaceoThe+critical+stage+of+friction+ridge+and+pattern+formationJ.%C2%A0Forensic+Identif.52200235" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib122" class="citations"><div id="sref122" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref122-1" title="View in article">132.</a></div><div>Willer, C.J. ∙ Li, Y. ∙ Abecasis, G.R.</div><div><strong>METAL: fast and efficient meta-analysis of genomewide association scans</strong></div><div><em>Bioinformatics.</em> 2010; <strong>26</strong>:2190-2191</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btq340" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_132_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-77955894071" target="_blank" aria-label="Open Scopus in new tab">Scopus (2264)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20616382/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtq340&amp;pmid=20616382" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib123" class="citations"><div id="sref123" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref123-1" title="View in article">133.</a></div><div>Yamaguchi, T.P. ∙ Bradley, A. ∙ McMahon, A.P. ...</div><div><strong>A Wnt5a pathway underlies outgrowth of multiple structures in the vertebrate embryo</strong></div><div><em>Development.</em> 1999; <strong>126</strong>:1211-1223</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1242/dev.126.6.1211" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/10021340/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1242%2Fdev.126.6.1211&amp;pmid=10021340" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib124" class="citations"><div id="sref124" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref124" title="View in article">134.</a></div><div>Yamamoto, S. ∙ Uchida, Y. ∙ Ohtani, T. ...</div><div><strong>Hoxa13 regulates expression of common Hox target genes involved in cartilage development to coordinate the expansion of the autopodal anlage</strong></div><div><em>Dev. Growth Differ.</em> 2019; <strong>61</strong>:228-251</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1111/dgd.12601" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_134_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-85063132528" target="_blank" aria-label="Open Scopus in new tab">Scopus (4)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30895612/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1111%2Fdgd.12601&amp;pmid=30895612" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib125" class="citations"><div id="sref125" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref125-1" title="View in article">135.</a></div><div>Yang, J. ∙ Lee, S.H. ∙ Goddard, M.E. ...</div><div><strong>GCTA: a tool for genome-wide complex trait analysis</strong></div><div><em>Am. J. Hum. Genet.</em> 2011; <strong>88</strong>:76-82</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_135_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=10.1016%2Fj.ajhg.2010.11.011&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_135_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=10.1016%2Fj.ajhg.2010.11.011&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_135_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-78650856517" target="_blank" aria-label="Open Scopus in new tab">Scopus (3187)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21167468/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.ajhg.2010.11.011&amp;pmid=21167468" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib126" class="citations"><div id="sref126" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref126" title="View in article">136.</a></div><div>Yum, S.M. ∙ Baek, I.K. ∙ Hong, D. ...</div><div><strong>Fingerprint ridges allow primates to regulate grip</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2020; <strong>117</strong>:31665-31673</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.2001055117" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_136_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-85097560390" target="_blank" aria-label="Open Scopus in new tab">Scopus (5)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33257543/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.2001055117&amp;pmid=33257543" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib127" class="citations"><div id="sref127" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref127" title="View in article">137.</a></div><div>Zeng, H. ∙ Hoover, A.N. ∙ Liu, A.</div><div><strong>PCP effector gene Inturned is an important regulator of cilia formation and embryonic development in mammals</strong></div><div><em>Dev. Biol.</em> 2010; <strong>339</strong>:418-428</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.ydbio.2010.01.003" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_137_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-77149146791" target="_blank" aria-label="Open Scopus in new tab">Scopus (73)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20067783/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.ydbio.2010.01.003&amp;pmid=20067783" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib128" class="citations"><div id="sref128" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref128" title="View in article">138.</a></div><div>Zhang, H.-G. ∙ Chen, Y.-F. ∙ Ding, M. ...</div><div><strong>Dermatoglyphics from all Chinese ethnic groups reveal geographic patterning</strong></div><div><em>PLoS ONE.</em> 2010; <strong>5</strong>:e8783</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pone.0008783" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_138_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-77649306234" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20098698/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pone.0008783&amp;pmid=20098698" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib129" class="citations"><div id="sref129" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref129" title="View in article">139.</a></div><div>Zhao, J. ∙ Bilsland, A. ∙ Jackson, K. ...</div><div><strong>MDM2 negatively regulates the human telomerase RNA gene promoter</strong></div><div><em>BMC Cancer.</em> 2005; <strong>5</strong>:6</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/1471-2407-5-6" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_139_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.12.008&amp;key=2-s2.0-13144291643" target="_blank" aria-label="Open Scopus in new tab">Scopus (22)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15656906/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2F1471-2407-5-6&amp;pmid=15656906" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div></div></div></section></div></section><div class="core-linked-content">
    

        


    

            

            

            <div data-widget-id="8633d695-b258-4908-b86f-e2c268ec3c6b" data-widget-name="UX3HTMLWidget"><script type="text/javascript">
if($('.content-navigation__body__title').length){
$('.content-navigation__body__title').each(function(){
var html=this.innerHTML;
var div = document.createElement("div");
div.innerHTML = html;
var text = div.textContent || div.innerText || "";
this.innerHTML=text;
});
}
</script></div>

            

        

    

        


    

            

            

            <div data-widget-id="393effff-5b44-4681-a9b4-4f029b3540b7" data-widget-name="UX3HTMLWidget"><style>
.linkback-ad .pb-ad {
    display: block;
   height: auto;
    margin-left: 0px;
    margin-right: auto;
}
</style></div>

            

        

    

        

</div></div></div><div class="core-collateral-aside"><section id="core-collateral-figures"><h2>Figures (11)<button class="js--openFigureViewer"><span>Figure Viewer</span></button></h2><div class="core-collateral-aside-wrapper"><div class="grid"><a class="figure__open__ctrl" href="#fig1" aria-label="Go to Figure&amp;nbsp;1"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.12.008/asset/936ee3ba-2c16-49ec-92cc-c2411453bed5/main.assets/gr1.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig2" aria-label="Go to Figure&amp;nbsp;2"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.12.008/asset/96e0cbec-0b1c-4711-a03e-f182465db123/main.assets/gr2.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs1" aria-label="Go to Figure&amp;nbsp;S1"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.12.008/asset/caf8acfa-021b-498d-9498-fdd3b39626d2/main.assets/figs1.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs2" aria-label="Go to Figure&amp;nbsp;S2"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.12.008/asset/fed7c50d-cf6c-4713-be22-a60beca981ef/main.assets/figs2.jpg" aria-hidden="true"></a></div><div id="collapsible-figures" data-expandable="item"><div class="content grid" style="display:none;" role="region" id="collapsible-figures_content"><a class="figure__open__ctrl" href="#figs3" aria-label="Go to Figure&amp;nbsp;S3"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.12.008/asset/65f595d1-bedd-479a-9a0c-9132cd645731/main.assets/figs3.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig3" aria-label="Go to Figure&amp;nbsp;3"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.12.008/asset/0b0db4e4-60d7-4b2a-82e5-a67ec235e310/main.assets/gr3.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs4" aria-label="Go to Figure&amp;nbsp;S4"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.12.008/asset/d8495a7e-34fe-49b0-95e9-cdaab04c548b/main.assets/figs4.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig4" aria-label="Go to Figure&amp;nbsp;4"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.12.008/asset/29542a0e-957f-46c2-a880-aa6406278697/main.assets/gr4.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig5" aria-label="Go to Figure&amp;nbsp;5"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.12.008/asset/abd3aead-8aa8-4cd4-9bdb-f22760165c9f/main.assets/gr5.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs5" aria-label="Go to Figure&amp;nbsp;S5"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.12.008/asset/c5187c16-fff0-486f-a85c-3e655c722e32/main.assets/figs5.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs6" aria-label="Go to Figure&amp;nbsp;S6"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.12.008/asset/720f1ed8-7dec-4194-8a28-a0f0a7184c73/main.assets/figs6.jpg" aria-hidden="true"></a></div><button class="trigger" aria-controls="collapsible-figures_content" aria-expanded="false" tabindex="0" role="button"><span class="expand">Show all figures</span><span class="collapse">Hide figures</span><i class="icon-section_arrow_d" aria-hidden="true"></i></button></div></div></section>
    

        


    

            

            

            <section id="core-collateral-metrics" class="publication-list__axel-section"><h2>Article metrics</h2><div><ul class="publication-list rlist publication-list--list plum-x--lazy-load publication-list--axel" data-plum-x-url="/pb/widgets/ux3/publicationList/plumx?widgetId=66dd7e11-7768-4006-93d8-a72b60e2a264&amp;pbContext=%3Bproduct%3Aproduct%3Aelsevier%5C%3Aproduct%5C%3Acell%3Bwebsite%3Awebsite%3Acell-site%3Bctype%3Astring%3AJournal+Content%3Bjournal%3Ajournal%3Acell%3BpageGroup%3Astring%3AArticle+View%3Bpage%3Astring%3AShow+Full+Text%3BrequestedJournal%3Ajournal%3Acell%3Bissue%3Aissue%3Apii%5C%3AS0092867421X00025%3Bwgroup%3Astring%3ADefault+Website+Group%3Barticle%3Aarticle%3Apii%5C%3AS009286742101446X%3BsubPage%3Astring%3AFull+Text" data-pb-mode-enabled="false"><li aria-label="loading articles" role="alert" class="js--loading plum-x--spinner hidden"></li></ul></div></section>

            

        
<section id="core-collateral-supplementary"><h2>Supplemental information (8)</h2><div class="core-collateral-aside-wrapper"><div class="core-supplementary-material"><div class="core-link"><button class="downloadAllSupplementary"><span>Download all</span></button></div></div><div class="core-supplementary-material" data-category="pdf"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2021.12.008/attachment/939eb111-67d0-411b-8429-80c611a7259e/mmc1.pdf" download="mmc1.pdf"><span id="mmc1-link">PDF (56.97 KB)</span></a><div class="core-description">Methods S1. Justifications for transverse digital ridges as human dermatoglyphs and the BLAST alignment for each fragment, related to Figures 2 and 3</div></div></div><div class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2021.12.008/attachment/bbfa0b5b-0911-48b8-b662-73216206e52d/mmc2.xlsx" download="mmc2.xlsx"><span id="mmc2-link">Spreadsheet (15.94 KB)</span></a><div class="core-description">Table S1. Characteristics of cohorts and summary of imputation methods and quality control filters, related to Table 1</div></div></div><div class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2021.12.008/attachment/b64d17ea-c19c-4014-8c1e-c12193025bba/mmc3.xlsx" download="mmc3.xlsx"><span id="mmc3-link">Spreadsheet (5.32 MB)</span></a><div class="core-description">Table S2. The performance of signals in Han Chinese discovery and replication populations based on ordinal and binary fingerprint patterns, related to Figure&nbsp;1</div></div></div><div class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2021.12.008/attachment/0b9fbc26-48ba-4149-984b-c625cf0fac75/mmc4.xlsx" download="mmc4.xlsx"><span id="mmc4-link">Spreadsheet (22.64 KB)</span></a><div class="core-description">Table S3. Association results of the composite phenotype (extracted from the middle three digits of both hands) and 66 additional phenotypes (derived from fingerprint patterns) used in this study, related to Figures 1 and 2</div></div></div><div class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2021.12.008/attachment/af0c4b43-986d-4e56-b4b1-b3564b4a4ef0/mmc5.xlsx" download="mmc5.xlsx"><span id="mmc5-link">Spreadsheet (17.13 KB)</span></a><div class="core-description">Table S4. Transverse digit pad patterning in Evi1Jbo mice, related to Figure&nbsp;3</div></div></div><div id="collapsible-supplementary" data-expandable="item"><div class="content" style="display:none;" role="region" id="collapsible-supplementary_content"><div class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2021.12.008/attachment/09f2ff24-8289-45b5-a357-71c13b0b51b4/mmc6.xlsx" download="mmc6.xlsx"><span id="mmc6-link">Spreadsheet (24.31 KB)</span></a><div class="core-description">Table S5. The performance of 43 signals after meta-analysis of eight populations (the top SNP in each locus was displayed), related to Figure&nbsp;4</div></div></div><div class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2021.12.008/attachment/5e04db61-0bdb-439c-b9bf-74f223864f82/mmc7.xlsx" download="mmc7.xlsx"><span id="mmc7-link">Spreadsheet (54.92 KB)</span></a><div class="core-description">Table S6. Single-factor regression analysis and genetic correlation (rg) based on SNP between hand traits and fingerprint patterns of eight digits (except the thumbs) in the Chinese Han populations (n&nbsp;= 6,318, JD+NSPT), related to Figure&nbsp;5</div></div></div><div class="core-supplementary-material" data-category="pdf"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2021.12.008/attachment/db0bedd0-2ff3-48ec-bd34-d4969f85f25d/mmc8.pdf" download="mmc8.pdf"><span id="mmc8-link">PDF (10.61 MB)</span></a><div class="core-description">Data S1. Alignment view of the cloned sequences used in luciferase reporter assay and regional plots of 43 loci associated with fingerprint patterns, related to Figures 2 and 4</div></div></div></div><button class="trigger" aria-controls="collapsible-supplementary_content" aria-expanded="false" tabindex="0" role="button"><span class="expand">Show more</span><span class="collapse">Show less</span><i class="icon-section_arrow_d" aria-hidden="true"></i></button></div></div></section>
    

        


    

            

            

            <section id="core-collateral-relatedArticles" class="axel-related-articles"><h2 id="relatedArticles" class="top axel-related-articles__title" data-title="Related articles">Related Articles</h2><div data-pb-widget-type="ux3-relatedArticles" data-widget-id="8d3844d1-450e-497c-ab75-a63fc41750ac" data-widget-name="AtyponTempUX3RelatedArticlesWidget" data-article-pii="S009286742101446X" class="ux3DeferredOnScrollWidget axel-related-articles__body" data-show-more="true"></div></section>

            

        
</div></div><aside data-core-aside="right-rail"><div class="inner-wrapper">

    

        


    

            

            

            <div data-widget-id="dca7470d-9085-4d21-a6f4-a66397718146" data-widget-name="UX3HTMLWidget"><input type="hidden" id="gptPage" value="cell-videos"></div>

            

        











        <div data-widget-def="literatumAd" data-widget-id="481d10c2-bbe0-470e-a5ba-207a81ecc8dc" class="adplaceholders" style="right: 0px; position: static;">
        
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd alignCenter  widget-verticalAd" id="481d10c2-bbe0-470e-a5ba-207a81ecc8dc">
                
                
                    
                

                <div class="wrapped " id="Marketplace-Recs">
                    
                    <div class="widget-body body body-verticalAd "><div class="verticalAdContainer">
    <div class="verticalAdLabelContainer">
        <div class="verticalAdLabel">
                <img src="/templates/jsp/_style2/_marlin/images/bg_adVert.gif" alt="Advertisement">
        </div>
    </div>
    <div class="verticalAd">
        





        <div class="pb-ad">
            <script src="//cdn.pbgrd.com/core-cell.js" async=""></script>
<div id="pbgrd-skyr-c"><div id="pbgrd-skyr" style="width: inherit; position: relative; background: transparent;"></div></div>
        </div>
    

    </div>
</div></div>
                </div>
                </div>

        

        </div>
    

</div></aside><div class="core-authors-details" style="display:none"><section class="core-authors"></section></div></article><div data-extent="article-wrapper"><div class="core-container"></div></div><script id="axel-publication-metadata" type="application/json">{"doi":"10.1016/j.cell.2021.12.008","data-is-retraction":false,"data-is-retired":false}
</script><script type="text/javascript" data-ot-ignore="data-ot-ignore" defer="defer" src="https://static.addtoany.com/menu/page.js" class="optanon-category-C0004 ot-vscat-C0004"></script><template id="figure_nav_template"><a href="#" class="open-in-viewer btn btn-dark figure-pop-btn"><i aria-hidden="true" class="icon-export_citation"></i><span>Open in viewer</span></a></template><div id="figure-viewer" role="dialog" aria-modal="true" aria-label="Figure Viewer" class="showCaption showThumbstrip figure-viewer"><nav class="nav-bar figure-viewer__top"><a href="#" title="Close Figure Viewer" role="button" class="figure-viewer__close"><i aria-hidden="true" class="icon-tools_close"></i></a><div class="figure-viewer__title">Limb development genes underlie variation in human fingerprint patterns</div></nav><div class="frames"><ul class="frames__list"><li><div class="thumbnail__image"><img id="gr1" src="/cms/10.1016/j.cell.2021.12.008/asset/936ee3ba-2c16-49ec-92cc-c2411453bed5/main.assets/gr1.jpg" height="444" width="657" aria-labelledby="fig1-title fig1-content" data-viewer-src="/cms/10.1016/j.cell.2021.12.008/asset/48cc46d0-af80-4e9e-8324-179d32c4b9c4/main.assets/gr1_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr2" src="/cms/10.1016/j.cell.2021.12.008/asset/96e0cbec-0b1c-4711-a03e-f182465db123/main.assets/gr2.jpg" height="726" width="503" aria-labelledby="fig2-title fig2-content" data-viewer-src="/cms/10.1016/j.cell.2021.12.008/asset/95e35782-f618-45a9-912c-80006ed62b6e/main.assets/gr2_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs1" src="/cms/10.1016/j.cell.2021.12.008/asset/caf8acfa-021b-498d-9498-fdd3b39626d2/main.assets/figs1.jpg" height="293" width="654" aria-labelledby="figs1-title figs1-content" data-viewer-src="/cms/10.1016/j.cell.2021.12.008/asset/40210a55-1a22-4289-aea3-0fbac9f44ef6/main.assets/figs1_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs2" src="/cms/10.1016/j.cell.2021.12.008/asset/fed7c50d-cf6c-4713-be22-a60beca981ef/main.assets/figs2.jpg" height="657" width="657" aria-labelledby="figs2-title figs2-content" data-viewer-src="/cms/10.1016/j.cell.2021.12.008/asset/64bde2b2-b6ff-4c5f-b61f-4a1a22b83702/main.assets/figs2_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs3" src="/cms/10.1016/j.cell.2021.12.008/asset/65f595d1-bedd-479a-9a0c-9132cd645731/main.assets/figs3.jpg" height="430" width="506" aria-labelledby="figs3-title figs3-content" data-viewer-src="/cms/10.1016/j.cell.2021.12.008/asset/3b8f137b-117e-46af-82f0-acaf7c2c4070/main.assets/figs3_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr3" src="/cms/10.1016/j.cell.2021.12.008/asset/0b0db4e4-60d7-4b2a-82e5-a67ec235e310/main.assets/gr3.jpg" height="611" width="663" aria-labelledby="fig3-title fig3-content" data-viewer-src="/cms/10.1016/j.cell.2021.12.008/asset/f3827bdd-47dc-468d-ad22-5930a8971e67/main.assets/gr3_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs4" src="/cms/10.1016/j.cell.2021.12.008/asset/d8495a7e-34fe-49b0-95e9-cdaab04c548b/main.assets/figs4.jpg" height="926" width="657" aria-labelledby="figs4-title figs4-content" data-viewer-src="/cms/10.1016/j.cell.2021.12.008/asset/c2e0655f-a09b-44c1-8b83-649f24ee9b1f/main.assets/figs4_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr4" src="/cms/10.1016/j.cell.2021.12.008/asset/29542a0e-957f-46c2-a880-aa6406278697/main.assets/gr4.jpg" height="840" width="657" aria-labelledby="fig4-title fig4-content" data-viewer-src="/cms/10.1016/j.cell.2021.12.008/asset/fb1d51d0-1ccf-48a3-a4e1-817e292dbfaf/main.assets/gr4_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr5" src="/cms/10.1016/j.cell.2021.12.008/asset/abd3aead-8aa8-4cd4-9bdb-f22760165c9f/main.assets/gr5.jpg" height="392" width="506" aria-labelledby="fig5-title fig5-content" data-viewer-src="/cms/10.1016/j.cell.2021.12.008/asset/4a580541-3d9e-4238-9200-24eb52782f2f/main.assets/gr5_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs5" src="/cms/10.1016/j.cell.2021.12.008/asset/c5187c16-fff0-486f-a85c-3e655c722e32/main.assets/figs5.jpg" height="398" width="657" aria-labelledby="figs5-title figs5-content" data-viewer-src="/cms/10.1016/j.cell.2021.12.008/asset/2a1c0e81-ac83-49a4-99f9-4494ce3de105/main.assets/figs5_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs6" src="/cms/10.1016/j.cell.2021.12.008/asset/720f1ed8-7dec-4194-8a28-a0f0a7184c73/main.assets/figs6.jpg" height="725" width="506" aria-labelledby="figs6-title figs6-content" data-viewer-src="/cms/10.1016/j.cell.2021.12.008/asset/47983249-8bd7-4ac8-a3c8-5ad3ca90c5ad/main.assets/figs6_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li></ul><a href="#" title="Previous Frame" class="frameButton previousFrameButton frames__prev"><!--span Previous Frame--><i class="icon-arrow_l"></i></a><a href="#" title="Next Frame" class="frameButton nextFrameButton frames__next"><!--span Next Frame--><i class="icon-arrow_r"></i></a></div><div class="caption"><div class="figure-viewer__caption-body"></div></div><div class="toolbar figure-viewer__toolbar"><span class="figure-viewer__counter"></span><a href="#" class="button toggle-caption-button figure-viewer__caption__ctrl"><i aria-hidden="true" class="icon icon-info"></i><span class="label">Hide Caption</span></a><a href="#" class="button download-options-button"><i aria-hidden="true" class="icon icon-gizmo-folder-in download-size-icon"></i><span class="label">Download</span></a><a href="#figures" class="button see-figures figure-viewer__see-figure__ctrl"><i aria-hidden="true" class="icon icon-gizmo-image_figure viewer-size-icon"></i><span class="label">See figure in Article</span></a></div><div class="thumbstrip"><a href="#" class="toggleButton thumbstrip__ctrl"><span>Toggle Thumbstrip</span></a><a href="#" title="Previous Thumbnails" class="thumbstripButton previousThumbstripButton thumbstrip__prev"><i aria-hidden="true" class="icon-arrow_l"></i></a><a href="#" title="Next Thumbnails" class="thumbstripButton nextThumbstripButton thumbstrip__next"><i aria-hidden="true" class="icon-arrow_r"></i></a><div class="thumbstrip__wrapper"><ul class="rlist figure-viewer__thumbslist"><li><div class="thumbnail__label">Figure&nbsp;1</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr1" src="/cms/10.1016/j.cell.2021.12.008/asset/936ee3ba-2c16-49ec-92cc-c2411453bed5/main.assets/gr1.jpg" height="444" width="657" aria-labelledby="fig1-title fig1-content" data-viewer-src="/cms/10.1016/j.cell.2021.12.008/asset/48cc46d0-af80-4e9e-8324-179d32c4b9c4/main.assets/gr1_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;2</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr2" src="/cms/10.1016/j.cell.2021.12.008/asset/96e0cbec-0b1c-4711-a03e-f182465db123/main.assets/gr2.jpg" height="726" width="503" aria-labelledby="fig2-title fig2-content" data-viewer-src="/cms/10.1016/j.cell.2021.12.008/asset/95e35782-f618-45a9-912c-80006ed62b6e/main.assets/gr2_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S1</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs1" src="/cms/10.1016/j.cell.2021.12.008/asset/caf8acfa-021b-498d-9498-fdd3b39626d2/main.assets/figs1.jpg" height="293" width="654" aria-labelledby="figs1-title figs1-content" data-viewer-src="/cms/10.1016/j.cell.2021.12.008/asset/40210a55-1a22-4289-aea3-0fbac9f44ef6/main.assets/figs1_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S2</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs2" src="/cms/10.1016/j.cell.2021.12.008/asset/fed7c50d-cf6c-4713-be22-a60beca981ef/main.assets/figs2.jpg" height="657" width="657" aria-labelledby="figs2-title figs2-content" data-viewer-src="/cms/10.1016/j.cell.2021.12.008/asset/64bde2b2-b6ff-4c5f-b61f-4a1a22b83702/main.assets/figs2_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S3</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs3" src="/cms/10.1016/j.cell.2021.12.008/asset/65f595d1-bedd-479a-9a0c-9132cd645731/main.assets/figs3.jpg" height="430" width="506" aria-labelledby="figs3-title figs3-content" data-viewer-src="/cms/10.1016/j.cell.2021.12.008/asset/3b8f137b-117e-46af-82f0-acaf7c2c4070/main.assets/figs3_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;3</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr3" src="/cms/10.1016/j.cell.2021.12.008/asset/0b0db4e4-60d7-4b2a-82e5-a67ec235e310/main.assets/gr3.jpg" height="611" width="663" aria-labelledby="fig3-title fig3-content" data-viewer-src="/cms/10.1016/j.cell.2021.12.008/asset/f3827bdd-47dc-468d-ad22-5930a8971e67/main.assets/gr3_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S4</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs4" src="/cms/10.1016/j.cell.2021.12.008/asset/d8495a7e-34fe-49b0-95e9-cdaab04c548b/main.assets/figs4.jpg" height="926" width="657" aria-labelledby="figs4-title figs4-content" data-viewer-src="/cms/10.1016/j.cell.2021.12.008/asset/c2e0655f-a09b-44c1-8b83-649f24ee9b1f/main.assets/figs4_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;4</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr4" src="/cms/10.1016/j.cell.2021.12.008/asset/29542a0e-957f-46c2-a880-aa6406278697/main.assets/gr4.jpg" height="840" width="657" aria-labelledby="fig4-title fig4-content" data-viewer-src="/cms/10.1016/j.cell.2021.12.008/asset/fb1d51d0-1ccf-48a3-a4e1-817e292dbfaf/main.assets/gr4_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;5</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr5" src="/cms/10.1016/j.cell.2021.12.008/asset/abd3aead-8aa8-4cd4-9bdb-f22760165c9f/main.assets/gr5.jpg" height="392" width="506" aria-labelledby="fig5-title fig5-content" data-viewer-src="/cms/10.1016/j.cell.2021.12.008/asset/4a580541-3d9e-4238-9200-24eb52782f2f/main.assets/gr5_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S5</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs5" src="/cms/10.1016/j.cell.2021.12.008/asset/c5187c16-fff0-486f-a85c-3e655c722e32/main.assets/figs5.jpg" height="398" width="657" aria-labelledby="figs5-title figs5-content" data-viewer-src="/cms/10.1016/j.cell.2021.12.008/asset/2a1c0e81-ac83-49a4-99f9-4494ce3de105/main.assets/figs5_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S6</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs6" src="/cms/10.1016/j.cell.2021.12.008/asset/720f1ed8-7dec-4194-8a28-a0f0a7184c73/main.assets/figs6.jpg" height="725" width="506" aria-labelledby="figs6-title figs6-content" data-viewer-src="/cms/10.1016/j.cell.2021.12.008/asset/47983249-8bd7-4ac8-a3c8-5ad3ca90c5ad/main.assets/figs6_lrg.jpg" loading="lazy" tabindex="0"></div></div></li></ul></div></div><div class="download figure-viewer__download"><div class="pointer"></div><ul class="rlist downlad__wrapper"><li class="hidden-xs hidden-sm"><a href="/action/downloadFigures?pii=e00489&amp;id=" class="downloadAllButton"><i class="icon icon-gizmo-ppt figure-viewer__download__viewer-size-icon figure-viewer__download__viewer-size-icon--ppt"></i><span class="figure-viewer__download__text"> Download .PPT</span></a></li></ul></div></div><template id="toCitationLink" data-citation="Citation" data-footnote="Footnote"><div class="to-citation__wrapper"><a class="to-citation"><i aria-hidden="true" class="icon-subdirectory_arrow_left"></i><span>Go to</span></a></div></template><template id="toCitationButton" data-citation="Citation" data-footnote="Footnote"><div class="to-citation__wrapper"><button class="to-citation"><i aria-hidden="true" class="icon-subdirectory_arrow_left"></i><span>Go to</span></button></div></template><template id="toCitationAccordion" data-collapse-citation="Hide Citations" data-expand-citation="Show Citations" data-collapse-footnote="Hide Footnotes" data-expand-footnote="Show Footnotes"><div class="to-citation__wrapper"><button aria-controls="aria-controls" aria-expanded="false" aria-label="Toggle citations menu" class="to-citation__toggle"><i aria-hidden="true" class="icon-plus"></i><span data-expand-title="data-expand-title" data-collapsed-title="data-collapsed-title" class="accordion__toggle__title"></span></button><div role="menu" aria-label="links" class="to-citation__accordion no-separator"><ul></ul></div></div></template><template id="toCitationAccordionItem"><li><a class="to-citation"><i aria-hidden="true" class="icon-subdirectory_arrow_left"></i><span></span></a></li></template><template id="citations_truncate_template"><div class="citations-truncation"><button data-label-expand="Show all references" data-label-collapse="Show fewer" data-label-remaining="references remaining" class="btn btn--inverse"><span>Show all references</span></button></div></template><template id="collateral_texts_template" data-do="Digital Object" data-figures="Figures" data-metrics="Article Metrics" data-related="Related articles" data-supplementary="Supplemental information"></template><template id="collapsible_authors_template"><button aria-expanded="false" data-expandable="all" data-label-expand="Expand All" data-label-collapse="Collapse All" class="collateral-contributors-control"><span>Expand All</span></button></template>
<template id="collapsible_tables_collapse_template"><div class="collapsible-figure-btn__wrapper expanded"><button aria-expanded="true" class="btn collapsible-figure-btn btn--inverse"><i aria-hidden="true" class="icon-arrow-up"></i><span class="text-uppercase">Collapse</span></button></div></template>
<template id="collapsible_tables_expand_template"><div class="collapsible-figure-btn__wrapper collapsed"><button aria-expanded="false" class="btn collapsible-figure-btn btn--inverse"><i aria-hidden="true" class="icon-arrow-down"></i><span class="text-uppercase">Expand Table</span></button></div></template><template id="authorsAffiliationsLink"><a href="#tab-contributors" class="to-authors-affiliations">Authors Info &amp; Affiliations</a></template></div>
                </div>
                </div>

        

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none  widget-  widget-compact-all" id="756cf18d-e910-4468-936c-0225ae7c669f">
                
                
                    
                

                <div class="wrapped " id="ad for nucleus-single-molecule-technologies">
                    
                    <div class="widget-body body body-  body-compact-all"><div class="pb-ad">
            <script type="text/javascript">
URL_var = 'https://www.cell.com/cell/fulltext/S0092-8674(21)01446-X';

if(URL_var.indexOf("S0165-6147(17)30205-5") > -1
|| URL_var.indexOf("S0167-7799(18)30204-X") > -1 
|| URL_var.indexOf("S0092-8674(18)31649-0") > -1 
|| URL_var.indexOf("S0092-8674(19)30404-0") > -1 
|| URL_var.indexOf("S1097-2765(18)30756-1") > -1 
|| URL_var.indexOf("S0092-8674(18)31452-1") > -1 
|| URL_var.indexOf("S1097-2765(19)30175-3") > -1 
|| URL_var.indexOf("S0092-8674(18)30852-3") > -1 
|| URL_var.indexOf("S0006-3495(18)30295-9") > -1 
|| URL_var.indexOf("S1097-2765(19)30063-2") > -1 
|| URL_var.indexOf("S0006-3495(17)33208-3") > -1 
|| URL_var.indexOf("S2211-1247(18)31639-5") > -1 
|| URL_var.indexOf("S1097-2765(18)30978-X") > -1 
)
    {
document.write('<input type="hidden" id="gpttc" value="nucleus-single-molecule-technologies" class="gpt-Target">')
}
</script>
        </div></div>
                </div>
                </div>

        

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none  widget-" id="754b1780-1117-4371-9285-9cd7020963d5">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body- "><div class="pb-ad">
            <script type="text/javascript">
URL_var = 'https://www.cell.com/cell/fulltext/S0092-8674(21)01446-X';

if(URL_var.indexOf("S1535-6108(19)30104-7") > -1
|| URL_var.indexOf("S0092-8674(18)31394-1") > -1
|| URL_var.indexOf("S1074-7613(19)30033-0") > -1
|| URL_var.indexOf("S1535-6108(18)30473-2") > -1
|| URL_var.indexOf("S0092-8674(19)30154-0") > -1
|| URL_var.indexOf("S1550-4131(18)30178-5") > -1
|| URL_var.indexOf("S1535-6108(19)30242-9") > -1
|| URL_var.indexOf("S1074-7613(18)30431-X") > -1
|| URL_var.indexOf("S0092-8674(19)30618-X") > -1
|| URL_var.indexOf("S1535-6108(18)30472-0") > -1
|| URL_var.indexOf("S1535-6108(18)30120-X") > -1
|| URL_var.indexOf("S0092-8674(18)31247-9") > -1
|| URL_var.indexOf("S1074-7613(18)30085-2") > -1
|| URL_var.indexOf("S1074-7613(18)30080-3") > -1
|| URL_var.indexOf("S1471-4906(18)30210-2") > -1
|| URL_var.indexOf("S1550-4131(19)30304-3") > -1
|| URL_var.indexOf("S1934-5909(18)30599-X") > -1
|| URL_var.indexOf("S2405-8033(19)30024-X") > -1
|| URL_var.indexOf("S1535-6108(18)30109-0") > -1
|| URL_var.indexOf("S0962-8924(18)30127-2") > -1
)
    {
document.write('<input type="hidden" id="gpttc" value="nucleus-tumor-immune-microenvironment" class="gpt-Target">')
}
</script>
        </div></div>
                </div>
                </div>

        

    

        


    

            

            

            <div data-widget-id="9c9ff7a1-a035-49f6-a336-600c9419c518" data-widget-name="UX3HTMLWidget"><!-- Start of HubSpot Embed Code --> 

<script type="text/plain" class="optanon-category-4" id="hs-script-loader" async="" defer="" src="//js.hs-scripts.com/519224.js"></script> 

<!-- End of HubSpot Embed Code --></div>

            

        

    

        


    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget sessionValidationWidget none  widget-none  widget-compact-all" id="0257aef7-6074-4898-a989-1c3f693225a2">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none  body-compact-all"><script>
    $.ajax({
        url: "https://www.cell.com/ssoSessionCheck",
        type: "GET",
        dataType: "jsonp",
        success: function (data) {
            if(data.RefreshSession) {
                location.reload();
            }
        },
        error : function (){
            console.warn("can't validate CSAS session.");
        }
    });
</script></div>
                </div>
                </div>

        

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget csasSsoUrls none  widget-none  widget-compact-all" id="948bc5c4-8b0d-47be-b5ad-6570559802d1">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none  body-compact-all"><!-- clear SSO URLs after all iFrames are rendered or after passing a time out-->
<script>
    setTimeout(clearSsoUrlsFun,10000);
    var iFramesList = document.getElementById("sso-urls-div").getElementsByTagName("iframe");
    var renderedIFramesCount = 0;
    var numberOfIFrames = iFramesList.length;
    var skipRefresh = false;
    for (var i = 0; i < iFramesList.length; i++) {
        var iFrame = iFramesList[i];
        iFrame.addEventListener("load", function() {
            renderedIFramesCount = renderedIFramesCount + 1;
            if (renderedIFramesCount >= numberOfIFrames)
            {
                clearSsoUrlsFun();
            }
        });
    }

    function clearSsoUrlsFun() {
        if (skipRefresh == false && numberOfIFrames > 0)
        {
            skipRefresh = true;
            $.ajax({
                url: "https://www.cell.com/action/clearSsoUrls",
                type: "GET",
                dataType: "jsonp",
                error : errorCallingAjx
            });
        }
    }

    function errorCallingAjx(){
        console.log("can't delete sso urls .")
    }
</script></div>
                </div>
                </div>

        










        <div data-widget-def="ux3-layout-widget" data-widget-id="d95b4a46-dc4d-4ca8-83f9-8b6724c649ec" class="ux3 ">
        
    

        


    

            

            

            <div data-widget-id="d95b4a46-dc4d-4ca8-83f9-8b6724c649ec" data-widget-name="ux3-layout-widget" class="fonts-loaded"><div class="footer clearfix">
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none adplaceholder adplaceholder--header widget-horizontalAd" id="a07ee92b-3942-49f8-9186-a2a08fc3fc56">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-horizontalAd "><div class="horizontalAdContainer">
    <div class="horizontalAdLabelContainer">
        <div class="horizontalAdLabel">
                <img src="/templates/jsp/_style2/_marlin/images/bg_adHoriz.gif" alt="Advertisement">
        </div>
    </div>
    <div class="horizontalAd">
        





        <div class="pb-ad">
            <!-- Beginning Async AdSlot 4 for Ad unit els.cellpress/cell  ### size: [[728,90]] -->
<div class="gpt-ad" data-adsize="cell-leaderboard" data-page="" data-site="cell" data-pos="4"></div>
<!-- End AdSlot 4 -->
        </div>
    

    </div>
</div></div>
                </div>
                </div>

        

    

        


    

            

            

            <div data-widget-id="23703bbc-af46-4f3f-a5bb-dd52891ff0ff" data-widget-name="UX3HTMLWidget"><div class="footer__top">
    <div class="footer__logo centered"><a href="/home" title="Cell Press"><!-- Generator: Adobe Illustrator 25.2.1, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 300 72" style="enable-background:new 0 0 300 72;" xml:space="preserve">
<style type="text/css">
	.st0{fill:#007DBC;}
</style>
<g>
	<path class="st0" d="M39.4,7.5c-2.3-0.3-4.4,1.1-4.9,3.3l-1.8,7.4c-0.4,1.8-1.9,3.1-3.8,3.3l-10.5,1.2C7.1,24-1,34.3,0.7,45.6
		c1.3,9,8.6,15.7,17.2,17c2.3,0.3,4.4-1.1,4.9-3.3l1.8-7.4c0.4-1.8,1.9-3.1,3.8-3.3l10.5-1.2c11.3-1.2,19.4-11.6,17.8-22.9
		C55.3,15.4,48.1,8.8,39.4,7.5z M37.1,34l-6.6,0.8c-1.1,0.1-2,0.9-2.3,2l-1.9,8.1c-0.6,2.4-2.5,4.2-5,4.5c-3.8,0.4-7.1-2.4-7.4-6.2
		c-0.3-3.7,2.6-6.8,6.3-7.2l6.6-0.8c1.1-0.1,2-0.9,2.3-2l1.9-8.1c0.6-2.4,2.5-4.2,5-4.5c3.8-0.4,7.1,2.4,7.4,6.2
		C43.7,30.5,40.8,33.6,37.1,34z"></path>
	<rect x="153.1" y="13.9" class="st0" width="8.2" height="42.3"></rect>
	<path class="st0" d="M90.2,20.4c7.3,0,8.6,4.8,9.4,7.3h8.8c-0.5-6.2-5.6-14.9-18.2-14.9c-10.9,0-19.4,7.5-19.4,22.3
		c0,14.6,8.1,22.2,19.3,22.2c10.6,0,16.6-6.4,18.4-15.6h-8.8c-1.1,5.2-4.5,7.9-9.4,7.9c-7.5,0-10.5-6.8-10.5-14.4
		C79.7,23.3,85.9,20.4,90.2,20.4z"></path>
	<path class="st0" d="M125.2,24c-13.7,0-15,12.1-15,15.8c0,12.9,6.8,17.1,15.6,17.1c5.4,0,11.6-2.5,13.8-10H131
		c-0.7,2.2-3,3.4-5.3,3.4c-6.6,0-7.1-5.3-7.3-7.7h21.5v-1.5C139.9,27,131.3,24,125.2,24z M118.6,37.4c0.5-4.1,2.5-6.6,6.4-6.6
		c2.6,0,5.9,1.5,6.4,6.6H118.6z"></path>
	<rect x="142" y="13.9" class="st0" width="8.2" height="42.3"></rect>
	<path class="st0" d="M174,13.8h-9.8V21h7.2c4.3,0,8.2,0.9,8.2,6.4c0,5.4-3.9,6.4-8.2,6.4h-7.2V41h9.8c10.6,0,14.7-6.7,14.7-13.6
		C188.7,20.5,184.6,13.8,174,13.8z"></path>
	<path class="st0" d="M210.5,24.9c-0.5-0.2-1.2-0.3-1.8-0.3c-4.1,0-8.3,2.7-9.8,6.5h-0.1v-5.7h-8v30.7h8.4V42.3
		c0-5.4,2.1-9.9,8.3-9.9c1,0,2.3,0.1,3,0.3V24.9z"></path>
	<path class="st0" d="M225.6,24.6c-9.3,0-15.7,7-15.7,16.2c0,9.5,6,16.1,15.7,16.1c6.9,0,12-3.1,14.4-10.3h-7.4
		c-0.5,1.9-3.3,4-6.6,4c-4.7,0-7.3-2.4-7.5-7.8h22.1C241.1,33.4,236,24.6,225.6,24.6z M218.4,37.5c0.1-2.4,1.7-6.5,7-6.5
		c4.1,0,5.9,2.3,6.7,6.5H218.4z"></path>
	<path class="st0" d="M260.3,37.7c-4.8-1.1-9.6-1.5-9.6-4.6c0-2.5,2.8-2.9,4.7-2.9c3,0,5.5,0.9,5.8,4.2h8c-0.7-7.5-6.8-9.8-13.5-9.8
		c-6.6,0-13.4,2-13.4,9.9c0,5.4,4.9,7.1,9.7,8.2c6.1,1.3,9.6,2.2,9.6,4.8c0,3-3.1,3.9-5.5,3.9c-3.3,0-6.4-1.5-6.5-5.2h-8
		C242,54.3,248.9,57,256,57c7,0,14-2.6,14-10.7C270.1,40.6,265.2,38.8,260.3,37.7z"></path>
	<path class="st0" d="M289.8,37.7c-4.8-1.1-9.6-1.5-9.6-4.6c0-2.5,2.8-2.9,4.7-2.9c3,0,5.5,0.9,5.8,4.2h8c-0.7-7.5-6.8-9.8-13.5-9.8
		c-6.6,0-13.4,2-13.4,9.9c0,5.4,4.9,7.1,9.7,8.2c6.1,1.3,9.6,2.2,9.6,4.8c0,3-3.1,3.9-5.5,3.9c-3.3,0-6.4-1.5-6.5-5.2h-8
		c0.4,8.1,7.3,10.8,14.4,10.8c7,0,14-2.6,14-10.7C299.6,40.6,294.7,38.8,289.8,37.7z"></path>
	<path class="st0" d="M34.1,57.6c0.3-0.2,0.4-0.6,0.4-1c0-0.5-0.2-0.9-0.5-1.1c-0.3-0.2-0.8-0.3-1.4-0.3h-1.9v5.1h0.8V58h0.8
		l1.3,2.2h0.9l-1.4-2.2C33.6,57.9,33.9,57.8,34.1,57.6z M32.5,57.3h-0.9v-1.5h1c0.1,0,0.3,0,0.4,0c0.1,0,0.3,0,0.4,0.1
		c0.1,0,0.2,0.1,0.3,0.2c0.1,0.1,0.1,0.2,0.1,0.4c0,0.2,0,0.3-0.1,0.4c-0.1,0.1-0.2,0.2-0.3,0.2c-0.1,0.1-0.2,0.1-0.4,0.1
		C32.8,57.3,32.6,57.3,32.5,57.3z"></path>
	<path class="st0" d="M35.6,54.2c-0.4-0.4-0.9-0.7-1.4-0.9c-0.5-0.2-1.1-0.3-1.7-0.3c-0.6,0-1.2,0.1-1.7,0.3c-0.6,0.2-1,0.5-1.4,0.9
		c-0.4,0.4-0.7,0.9-1,1.5c-0.2,0.6-0.4,1.2-0.4,1.9c0,0.7,0.1,1.3,0.3,1.8c0.2,0.6,0.5,1,0.9,1.5c0.4,0.4,0.9,0.7,1.4,1
		c0.6,0.2,1.2,0.4,1.8,0.4c0.6,0,1.2-0.1,1.7-0.3c0.5-0.2,1-0.5,1.4-0.9c0.4-0.4,0.7-0.9,1-1.5c0.2-0.6,0.4-1.2,0.4-1.9
		c0-0.7-0.1-1.3-0.4-1.9C36.4,55.1,36,54.6,35.6,54.2z M35.7,59.1c-0.2,0.5-0.4,0.9-0.8,1.2c-0.3,0.3-0.7,0.6-1.1,0.8
		c-0.4,0.2-0.9,0.3-1.4,0.3c-0.6,0-1-0.1-1.5-0.3c-0.4-0.2-0.8-0.5-1.1-0.8c-0.3-0.3-0.5-0.7-0.7-1.2c-0.2-0.5-0.2-1-0.2-1.5
		c0-0.6,0.1-1.1,0.3-1.5c0.2-0.5,0.4-0.9,0.8-1.2c0.3-0.3,0.7-0.6,1.1-0.8c0.4-0.2,0.9-0.3,1.4-0.3c0.5,0,1,0.1,1.4,0.3
		c0.4,0.2,0.8,0.4,1.1,0.8c0.3,0.3,0.6,0.7,0.8,1.2c0.2,0.5,0.3,1,0.3,1.5C36,58.1,35.9,58.6,35.7,59.1z"></path>
</g>
</svg></a></div>
    <div class="footer__social-links">
        <ul class="rlist rlist--inline centered">
            <li><a href="http://on.fb.me/1tgHakh" title="Facebook" target="_blank"><i aria-hidden="true" class="icon-Icon_Facebook"></i></a></li>

            <li><a href="https://www.instagram.com/cellpress/" title="Instagram" target="_blank"><svg width="18px" height="18px" viewBox="0 0 18 18" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-147.000000, -16.000000)" fill-rule="nonzero" fill="#969696"> <g id="Instagram-Logo" transform="translate(147.000000, 16.000000)"> <path d="M9,1.62580645 C11.4096774,1.62580645 11.7,1.62580645 12.6290323,1.68387097 C13.5,1.71290323 13.9935484,1.85806452 14.3129032,2.00322581 C14.7193548,2.17741935 15.0387097,2.3516129 15.3580645,2.67096774 C15.6774194,2.99032258 15.8806452,3.28064516 16.0258065,3.71612903 C16.1419355,4.03548387 16.2870968,4.5 16.3451613,5.4 C16.3741935,6.35806452 16.4032258,6.61935484 16.4032258,9.02903226 C16.4032258,11.4387097 16.4032258,11.7290323 16.3451613,12.6580645 C16.316129,13.5290323 16.1709677,14.0225806 16.0258065,14.3419355 C15.8516129,14.7483871 15.6774194,15.0677419 15.3580645,15.3870968 C15.0387097,15.7064516 14.7483871,15.9096774 14.3129032,16.0548387 C13.9935484,16.1709677 13.5290323,16.316129 12.6290323,16.3741935 C11.6709677,16.4032258 11.4096774,16.4322581 9,16.4322581 C6.59032258,16.4322581 6.3,16.4322581 5.37096774,16.3741935 C4.5,16.3451613 4.00645161,16.2 3.68709677,16.0548387 C3.28064516,15.8806452 2.96129032,15.7064516 2.64193548,15.3870968 C2.32258065,15.0677419 2.11935484,14.7774194 1.97419355,14.3419355 C1.85806452,14.0225806 1.71290323,13.5580645 1.65483871,12.6580645 C1.62580645,11.7 1.62580645,11.4096774 1.62580645,9 C1.62580645,6.59032258 1.62580645,6.3 1.68387097,5.37096774 C1.71290323,4.5 1.85806452,4.00645161 2.00322581,3.68709677 C2.17741935,3.28064516 2.3516129,2.96129032 2.67096774,2.64193548 C2.99032258,2.32258065 3.28064516,2.11935484 3.71612903,1.97419355 C4.03548387,1.85806452 4.5,1.71290323 5.4,1.65483871 C6.3,1.62580645 6.59032258,1.62580645 9,1.62580645 L9,1.62580645 Z M9,0 C6.56129032,0 6.24193548,0 5.28387097,0.0580645161 C4.32580645,0.116129032 3.65806452,0.261290323 3.10645161,0.464516129 C2.52580645,0.696774194 2.00322581,1.01612903 1.50967742,1.50967742 C1.01612903,2.00322581 0.696774194,2.52580645 0.464516129,3.10645161 C0.232258065,3.68709677 0.0870967742,4.32580645 0.0580645161,5.28387097 C0,6.24193548 0,6.56129032 0,9 C0,11.4387097 0,11.7580645 0.0580645161,12.716129 C0.116129032,13.6741935 0.261290323,14.3419355 0.464516129,14.8935484 C0.696774194,15.4741935 1.01612903,15.9967742 1.50967742,16.4903226 C2.00322581,16.983871 2.52580645,17.3032258 3.10645161,17.5354839 C3.68709677,17.7677419 4.32580645,17.9129032 5.28387097,17.9419355 C6.24193548,18 6.56129032,18 9,18 C11.4387097,18 11.7580645,18 12.716129,17.9419355 C13.6741935,17.883871 14.3419355,17.7387097 14.8935484,17.5354839 C15.4741935,17.3032258 15.9967742,16.983871 16.4903226,16.4903226 C16.983871,15.9967742 17.3032258,15.4741935 17.5354839,14.8935484 C17.7677419,14.3129032 17.9129032,13.6741935 17.9419355,12.716129 C18,11.7580645 18,11.4387097 18,9 C18,6.56129032 18,6.24193548 17.9419355,5.28387097 C17.883871,4.32580645 17.7387097,3.65806452 17.5354839,3.10645161 C17.3032258,2.52580645 16.983871,2.00322581 16.4903226,1.50967742 C15.9967742,1.01612903 15.4741935,0.696774194 14.8935484,0.464516129 C14.3129032,0.232258065 13.6741935,0.0870967742 12.716129,0.0580645161 C11.7580645,0 11.4387097,0 9,0 L9,0 Z"></path> <path d="M9,4.38387097 C6.44516129,4.38387097 4.38387097,6.44516129 4.38387097,9 C4.38387097,11.5548387 6.44516129,13.616129 9,13.616129 C11.5548387,13.616129 13.616129,11.5548387 13.616129,9 C13.616129,6.44516129 11.5548387,4.38387097 9,4.38387097 Z M9,11.9903226 C7.34516129,11.9903226 6.00967742,10.6548387 6.00967742,9 C6.00967742,7.34516129 7.34516129,6.00967742 9,6.00967742 C10.6548387,6.00967742 11.9903226,7.34516129 11.9903226,9 C11.9903226,10.6548387 10.6548387,11.9903226 9,11.9903226 Z"></path> <circle id="Oval" cx="13.7903226" cy="4.20967742" r="1.07419355"></circle> </g> </g> </g></svg></a></li>
            
            <li><a href="https://www.linkedin.com/company/cell-press/" title="LinkedIn" target="_blank"><i aria-hidden="true" class="icon-linkedin"></i></a></li>
            
                        <li><a href="http://bit.ly/1zq7Q5T" title="Twitter" target="_blank">
                <!--<i aria-hidden="true" class="icon-Icon_Twitter"></i></a>-->
                
                <svg id="xicon" viewBox="0 0 1200 1227" xmlns="http://www.w3.org/2000/svg" aria-hidden="true" role="none"><path fill="#969696" d="M714.163 519.284L1160.89 0H1055.03L667.137 450.887L357.328 0H0L468.492 681.821L0 1226.37H105.866L515.491 750.218L842.672 1226.37H1200L714.137 519.284H714.163ZM569.165 687.828L521.697 619.934L144.011 79.6944H306.615L611.412 515.685L658.88 583.579L1055.08 1150.3H892.476L569.165 687.854V687.828Z"></path></svg> 

                    <style>
                    .footer svg#xicon {
                      height: 16px;
                    }
                    </style>
            </a></li>

            
            <li><a href="/pb-assets/marketing/WeChat/WeChat-1557753222727.jpg" title="WeChat" target="_blank"><svg width="18px" height="15px" viewBox="0 0 18 15" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-122.000000, -17.000000)" fill-rule="nonzero" fill="#969696"> <g id="wechat" transform="translate(122.000000, 17.000000)"> <path d="M12.375,4.5 L12.375,4.5 L12.4875,4.5 L12.4875,4.5 C12.6898579,4.48604473 12.8520408,4.32704187 12.87,4.125 C12.8705905,4.04725572 12.8470025,3.9712499 12.8025,3.9075 C11.94,1.5675 9.4425,0 6.5625,0 C2.9475,0 0,2.52 0,5.625 C0.0313772094,7.34410122 0.895513317,8.94093526 2.3175,9.9075 L1.5675,11.46 C1.49009368,11.6087816 1.52088865,11.7908702 1.64291494,11.9059236 C1.76494122,12.0209769 1.94852501,12.0410168 2.0925,11.955 L4.08,10.8225 C4.61861992,11.0129999 5.17724298,11.1413322 5.745,11.205 C5.86564124,11.2178337 5.98503738,11.1715098 6.06545426,11.0806685 C6.14587113,10.9898272 6.17737153,10.865694 6.15,10.7475 C6.06148727,10.4219419 6.01114891,10.0871918 6,9.75 C6,6.855 8.8575,4.5 12.375,4.5 Z M8.25,3 C8.66421356,3 9,3.33578644 9,3.75 C9,4.16421356 8.66421356,4.5 8.25,4.5 C7.83578644,4.5 7.5,4.16421356 7.5,3.75 C7.5,3.33578644 7.83578644,3 8.25,3 Z M4.5,4.5 C4.08578644,4.5 3.75,4.16421356 3.75,3.75 C3.75,3.33578644 4.08578644,3 4.5,3 C4.91421356,3 5.25,3.33578644 5.25,3.75 C5.25,4.16421356 4.91421356,4.5 4.5,4.5 Z"></path> <path d="M18,9.75 C18,7.2675 15.48,5.25 12.375,5.25 C9.27,5.25 6.75,7.2675 6.75,9.75 C6.75,12.2325 9.27,14.25 12.375,14.25 C12.9568557,14.2502837 13.5364258,14.1772075 14.1,14.0325 L15.96,14.9625 C16.0114248,14.9874074 16.0678613,15.0002339 16.125,15 C16.2497193,14.9993787 16.3659682,14.9367912 16.4351508,14.8330172 C16.5043335,14.7292432 16.5173955,14.5978644 16.47,14.4825 L15.96,13.2075 C17.1784092,12.4716325 17.9450305,11.1723221 18,9.75 L18,9.75 Z M10.5,9 C10.0857864,9 9.75,8.66421356 9.75,8.25 C9.75,7.83578644 10.0857864,7.5 10.5,7.5 C10.9142136,7.5 11.25,7.83578644 11.25,8.25 C11.25,8.66421356 10.9142136,9 10.5,9 Z M14.25,9 C13.8357864,9 13.5,8.66421356 13.5,8.25 C13.5,7.83578644 13.8357864,7.5 14.25,7.5 C14.6642136,7.5 15,7.83578644 15,8.25 C15,8.66421356 14.6642136,9 14.25,9 Z" opacity="0.959999979"></path> </g> </g> </g></svg></a></li>
            
            <li><a href="https://www.weibo.com/CellPress" title="Weibo" target="_blank"><svg width="18px" height="15px" viewBox="0 0 18 15" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-99.000000, -17.000000)" fill-rule="nonzero" fill="#969696"> <g id="weibo" transform="translate(99.000000, 14.000000)"> <g transform="translate(0.000000, 3.000000)"> <path d="M13.2402211,7.47054172 C12.9918392,7.39866484 12.821608,7.34902326 12.9514975,7.03290287 C13.232804,6.34602189 13.2622915,5.75359781 12.9571055,5.33095486 C12.3847236,4.5380684 10.8191759,4.58064295 9.02514573,5.30975376 C9.02514573,5.30889193 8.46144724,5.54882627 8.60544724,5.11515185 C8.88150754,4.25435294 8.83953769,3.53386047 8.41043216,3.11759508 C7.43716583,2.17285363 4.84824121,3.15344733 2.62745729,5.3059617 C0.964944724,6.9186238 0,8.62798358 0,10.105855 C0,12.9324952 3.73748744,14.8235294 7.39429146,14.8235294 C12.1877186,14.8235294 15.3768844,11.9501778 15.3768844,9.80576471 C15.3768844,8.50974008 14.2514774,7.77442408 13.2402211,7.47054172 Z M7.40424121,13.5561176 C4.48625126,13.8315595 1.9678794,12.5686293 1.77811055,10.7327524 C1.58870352,8.89773735 3.80098492,7.18579207 6.71861307,6.90948837 C9.63678392,6.63266758 12.1556985,7.89663201 12.3447437,9.73164706 C12.533608,11.5675239 10.3227739,13.2796416 7.40424121,13.5561176 Z M18,5.16875787 C17.9985528,2.31402189 15.5695477,0 12.5728643,0 C12.222995,0 11.9396985,0.270098495 11.9396985,0.603283174 C11.9396985,0.936640219 12.222995,1.20656635 12.5728643,1.20656635 C14.8708945,1.20656635 16.7336683,2.98159781 16.7336683,5.17099863 C16.7336683,5.50435568 17.0173266,5.77428181 17.3668342,5.77428181 C17.7167035,5.77428181 18,5.50435568 18,5.17099863 C18,5.17099863 18,5.1696197 18,5.16875787 Z M15.4841608,5.08016142 C15.1948945,3.73415048 14.0812462,2.67323393 12.6683819,2.39744733 C12.3262915,2.33039672 11.9914372,2.54051163 11.9212462,2.86680164 C11.8506935,3.19309166 12.0712161,3.51179754 12.4133065,3.57919289 C13.3319397,3.75845417 14.0559196,4.44774829 14.2438794,5.32302599 C14.3146131,5.64931601 14.6491055,5.85925855 14.9915578,5.7923803 C15.3338291,5.72498495 15.5541709,5.40627907 15.4841608,5.08016142 Z M5.90761809,8.7069275 C4.7321005,8.9320383 3.93901508,9.87126402 4.1358392,10.8044569 C4.33248241,11.7378222 5.44504523,12.311803 6.62056281,12.0865198 C7.79571859,11.861409 8.58916583,10.922528 8.3921608,9.98899042 C8.19569849,9.0559699 7.08277387,8.48181669 5.90761809,8.7069275 Z"></path> </g> </g> </g> </g></svg></a></li>
            
            <li><a href="https://www.youtube.com/user/cellvideoabstracts" title="YouTube" target="_blank"><svg width="18px" height="13px" viewBox="0 0 18 13" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-76.000000, -19.000000)" fill-rule="nonzero" fill="#969696"> <g id="youtube" transform="translate(76.000000, 14.000000)"> <path d="M17.2984219,6.41974219 C16.6486641,5.64735937 15.4490625,5.33232422 13.1580703,5.33232422 L4.84175391,5.33232422 C2.49834375,5.33232422 1.27838672,5.66771484 0.631089844,6.49001953 C0,7.29172266 0,8.47304297 0,10.1080547 L0,13.2243047 C0,16.3917773 0.748792969,18 4.84175391,18 L13.1581055,18 C15.1448555,18 16.2457383,17.7219844 16.9579688,17.040375 C17.688375,16.3413984 18,15.2001211 18,13.2243047 L18,10.1080547 C18,8.38381641 17.951168,7.19553516 17.2984219,6.41974219 Z M11.5560703,12.0963867 L7.77965625,14.0700586 C7.69524609,14.1141797 7.60289062,14.136082 7.51071094,14.136082 C7.40629687,14.136082 7.30212891,14.1079922 7.20987891,14.0521289 C7.03613672,13.9468711 6.93007031,13.7585391 6.93007031,13.5554414 L6.93007031,9.62075391 C6.93007031,9.41800781 7.03585547,9.22985156 7.20924609,9.12452344 C7.38267187,9.01919531 7.59839062,9.01198828 7.77832031,9.10546875 L11.5547344,11.0664141 C11.7468633,11.1661523 11.8675195,11.3645039 11.8678008,11.5809258 C11.8680469,11.7975586 11.747918,11.9961914 11.5560703,12.0963867 Z"></path> </g> </g> </g></svg></a></li>
            
        </ul>
    </div>
</div>

<style>

.ux3 .footer__logo svg {
    max-width: 160px !important;
}

.ux3 .footer__social-links a svg {
    max-width: 26px;
    height: 1.15rem;
    width: auto;
}

.ux3 .footer__social-links a:hover svg path {
    fill: #353535;
}

</style></div>

            

        

    

        


    

            

            
                <div class="footer__bottom text-onDark">
            

            <div data-widget-id="06227714-c621-4651-9879-b0ce5176cccc" data-widget-name="ux3-layout-widget"><div class="col-sm-4 col-md-5 clearfix">
    

        


    

            

            

            <div data-widget-id="07e1d276-b4dd-4870-b858-ed74e70d64ef" data-widget-name="UX3HTMLWidget"><div class="groups">
	<div class="group">
		<h2 class="group__title">Life &amp; medical sciences journals</h2>
		<ul class="rlist clearfix sided">
		<li>
			<a alt="Cell" href="/cell/home">Cell</a>
		</li>
		<li>
			<a alt="Cancer Cell" href="/cancer-cell/home">Cancer Cell</a>
		</li>
		<li>
			<a alt="Cell Chemical Biology" href="/cell-chemical-biology">Cell Chemical Biology</a>
		</li>
		<li>
			<a alt="Cell Genomics" href="/cell-genomics/home">Cell Genomics</a>
		</li>
		<li>
			<a alt="Cell Host and Microbe" href="/cell-host-microbe/home">Cell Host &amp; Microbe</a>
		</li>
		<li>
			<a alt="Cell Metabolism" href="/cell-metabolism/home">Cell Metabolism</a>
		</li>
		<li>
			<a alt="Cell Reports" href="/cell-reports/home">Cell Reports</a>
		</li>
		<li>
			<a alt="Cell Reports Medicine" href="/cell-reports-medicine/home">Cell Reports Medicine</a>
		</li>
		<li>
			<a alt="Cell Stem Cell" href="/cell-stem-cell/home">Cell Stem Cell</a>
		</li>
		<li>
			<a alt="Cell Systems" href="/cell-systems/home">Cell Systems</a>
		</li>
		<li>
			<a alt="Current Biology" href="/current-biology/home">Current Biology</a>
		</li>
		<li>
			<a alt="Developmental Cell" href="/developmental-cell/home">Developmental Cell</a>
		</li>
		<li>
			<a alt="Immunity" href="/immunity/home">Immunity</a>
		</li>
		<li>
			<a alt="Med" href="/med/home">Med</a>
		</li>
		<li>
			<a alt="Molecular Cell" href="/molecular-cell/home">Molecular Cell</a>
		</li>
		<li>
			<a alt="Neuron" href="/neuron/home">Neuron</a>
		</li>
		<li>
			<a alt="Structure" href="/structure/home">Structure</a>
		</li>
		<li>
			<a alt="American Journal of Human Genetics" href="/ajhg/home">American Journal of Human Genetics (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Biophysical Journal" href="/biophysj/home">Biophysical Journal (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Biophysical Reports" href="/biophysreports/home">Biophysical Reports (<i>partner</i>)</a>
		</li>	
		<li>
			<a alt="HGG Advances" href="/hgg-advances/home">Human Genetics and Genomics Advances (<i>partner</i>)</a></li>
		<li>
			<a alt="Molecular Plant" href="/molecular-plant/home">Molecular Plant (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy" href="/molecular-therapy-family/molecular-therapy/home">Molecular Therapy (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy Methods and Clinical Development" href="/molecular-therapy-family/methods/home">Molecular Therapy Methods &amp; Clinical Development (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy Nucleic Acids" href="/molecular-therapy-family/nucleic-acids/home">Molecular Therapy Nucleic Acids (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy Oncology" href="/molecular-therapy-family/oncology/home">Molecular Therapy Oncology (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Plant Communications" href="/plant-communications/home">Plant Communications (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Stem Cell Reports" href="/stem-cell-reports/home">Stem Cell Reports (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Trends in Biochemical Sciences" href="/trends/biochemical-sciences/home">Trends in Biochemical Sciences</a>
		</li>
		<li>
		</li><li>
			<a alt="Trends in Cancer" href="/trends/cancer/home">Trends in Cancer</a>
		</li>
		<li>
			<a alt="Trends in Cell Biology" href="/trends/cell-biology/home">Trends in Cell Biology</a>
		</li>
		<li>
			<a alt="Trends in Ecology and Evolution" href="/trends/ecology-evolution/home">Trends in Ecology &amp; Evolution</a>
		</li>
		<li>
			<a alt="Trends in Endocrinology and Metabolism" href="/trends/endocrinology-metabolism/home">Trends in Endocrinology &amp; Metabolism</a>
		</li>
		<li>
			<a alt="Trends in Genetics" href="/trends/genetics/home">Trends in Genetics</a>
		</li>
		<li>
			<a alt="Trends in Immunology" href="/trends/immunology/home">Trends in Immunology</a>
		</li>
		<li>
			<a alt="Trends in Microbiology" href="/trends/microbiology/home">Trends in Microbiology</a>
		</li>
		<li>
			<a alt="Trends in Molecular Medicine" href="/trends/molecular-medicine/home">Trends in Molecular Medicine</a>
		</li>
		<li>
			<a alt="Trends in Neurosciences" href="/trends/neurosciences/home">Trends in Neurosciences</a>
		</li>
		<li>
			<a alt="Trends in Parasitology" href="/trends/parasitology/home">Trends in Parasitology</a>
		</li>
		<li>
			<a alt="Trends in Pharmacological Sciences" href="/trends/pharmacological-sciences/home">Trends in Pharmacological Sciences</a>
		</li>
		<li>
			<a alt="Trends in Plant Science" href="/trends/plant-science/home">Trends in Plant Science</a>
		</li>
		</ul>
	</div>

	<div class="group">
		<h2 class="group__title">Physical sciences &amp; engineering journals</h2>
		<ul class="rlist clearfix sided">
		  <li><a alt="Cell Biomaterials" href="/cell-biomaterials/home">Cell Biomaterials</a></li>
		  <li>
			<a alt="Cell Reports Physical Science" href="/cell-reports-physical-science/home">Cell Reports Physical Science</a>
		</li>
		<li>
			<a alt="Chem" href="/chem/home">Chem</a>
		</li>
		<li>
			<a alt="Chem Catalysis" href="/chem-catalysis/home">Chem Catalysis</a>
		</li>
		<li>
			<a alt="Device" href="/device/home">Device</a>
		</li>
		<li>
			<a alt="Joule" href="/joule/home">Joule</a>
		</li>
		<li>
			<a alt="Matter" href="/matter/home">Matter</a>
		</li>	
		<li><a alt="Matter" href="/newton/home">Newton</a></li>
		<li>
			<a alt="Trends in Chemistry" href="/trends/chemistry/home">Trends in Chemistry</a>
		</li>			
		</ul>
	</div>

<div class="group">
	<h2 class="group__title">Multidisciplinary journals</h2>
	<ul class="rlist clearfix sided">
	  		<li>
			<a alt="Cell Reports Methods" href="/cell-reports-methods/home">Cell Reports Methods</a>
		</li>
		<li>
			<a alt="Cell Reports Sustainability" href="/cell-reports-sustainability/home">Cell Reports Sustainability</a>
		</li>
		<li>
			<a alt="Heliyon" href="/heliyon/home">Heliyon</a>
		</li>	
		<li>
			<a alt="iScience" href="/iscience/home">iScience</a>
		</li>
		<li>
			<a alt="One Earth" href="/one-earth/home">One Earth</a>
		</li>
		<li>
			<a alt="Patterns" href="/patterns/home">Patterns</a>
		</li>
		<li>
			<a alt="STAR Protocols" href="/star-protocols/home">STAR Protocols</a>
		</li>
		<li>
			<a alt="Nexus" href="/nexus/home">Nexus (<i>partner</i>)</a>
		</li>			
        <li>
			<a alt="The Innovation" href="/the-innovation/home">The Innovation (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Trends in Biotechnology" href="/trends/biotechnology/home">Trends in Biotechnology</a>
		</li>
		<li>
			<a alt="Trends in Cognitive Sciences" href="/trends/cognitive-sciences/home">Trends in Cognitive Sciences</a>
		</li>
	</ul>
</div>

</div>
</div>

            

        
</div><div class="col-sm-4 col-md-5 clearfix">
    

        


    

            

            

            <div data-widget-id="850476de-00cb-4c0d-943c-bf1881da24f9" data-widget-name="UX3HTMLWidget"><div class="groups">
	<div class="clearfix">
		<div class="group sided">
			<h2 class="group__title">Authors</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Submit article" href="/submit-to-cell-press">Submit article</a>
				</li>
				<li>
					<a alt="Multi-Journal Submission" href="/multi-journal-submission">Multi-Journal Submission</a>
				</li>
				<li>
					<a alt="STAR Methods" href="/star-authors-guide">STAR Methods</a>
				</li>
				<li>
					<a alt="Sneak Peek – Preprints" href="/sneakpeek">Sneak Peek – Preprints</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
		<h2 class="group__title">Reviewers</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Reviewers" href="/reviewers">Information for reviewers</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
		<h2 class="group__title">News &amp; events</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Newsroom" href="/newsroom">Newsroom</a>
				</li>
				<li>
					<a alt="Events" href="/events">Events</a>
				</li>
				<li>
					<a alt="Cell Symposia" href="/symposia">Cell Symposia</a>
				</li>
				<li>
					<a alt="Consortia Hub" href="/consortia">Consortia Hub</a>
				</li>
				<li>
					<a alt="Webinars" href="/webinars">Webinars</a>
				</li>
								<li>
					&nbsp;
				</li>
			</ul>
		<h2 class="group__title">Multimedia</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Cell Press Podcast" href="/podcast">Cell Press Podcast</a>
				</li>
				<li>
					<a alt="Cell Press Videos" href="/video">Cell Press Videos</a>
				</li>
				<li>
					<a alt="Color and Comics" href="/comics">Coloring and Comics</a>
				</li>
				<li>
					<a alt="Figure360" href="/video-guidelines">Figure360</a>
				</li>
				<li>
					<a alt="Cell Picture Show" href="/pictureshow">Cell Picture Show</a>
				</li>
				<li>
					<a alt="Research Arc" href="/research-arc">Research Arc</a>
				</li>
			</ul>
		</div>

		<div class="group sided">
			<h2 class="group__title">About</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="About Cell Press" href="/about">About Cell Press</a>
				</li>

				<li>
					<a alt="Open access" href="/open-access">Open access</a>
				</li>
				<li>
					<a alt="COVID Hub" href="/covid-19">COVID Hub</a>
				</li>
				<li>
					<a alt="Sustainability" href="/sustainability">Sustainability hub</a>
				</li>
				<li>
					<a alt="Inclusion and diversity" href="/diversity">Inclusion and diversity</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
			<h2 class="group__title">Contact</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Contact us" href="/contact">Contact us</a>
				</li>
				<li>
					<a alt="Help and Support" href="https://service.elsevier.com/app/overview/cell-press/" target="_blank">Help &amp; Support</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
			<h2 class="group__title">Careers</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Cell Press Careers" href="https://relx.wd3.myworkdayjobs.com/en-US/ElsevierJobs" target="_blank">Cell Press Careers</a>
				</li>
			</ul>
		</div>
	</div>
</div></div>

            

        
</div><div class="col-sm-4 col-md-2 clearfix">
    

        


    

            

            

            <div data-widget-id="beafac41-1cfc-440b-9b55-74aba4a5612c" data-widget-name="UX3HTMLWidget"><div class="groups">
	<div class="group">
		<h2 class="group__title">Access</h2>
		<ul class="rlist clearfix">
			<li>
				<a alt="Subscribe" href="/subscribe">Subscribe</a>
			</li>
			<li>
				<a alt="Claim" href="/activate">Claim</a>
			</li>
			<li>
				<a alt="Read It Now" href="/read-it-now">Read-It-Now</a>
			</li>
			<li>
				<a alt="Recommend to Librarian" href="http://info.cell.com/request-access-to-cell-press-content" target="_blank">Recommend to Librarian</a>
			</li>
			<li>
				<a alt="Publication Alerts" href="/action/showPreferences?menuTab=Alerts">Publication Alerts</a>
			</li>
		</ul>
	</div>
	<div class="group">
		<h2 class="group__title">Collections</h2>
		<ul class="rlist clearfix">
			<li>
				<a alt="Best of Cell Press" href="/bestof">Best of Cell Press</a>
			</li>
			<li>
				<a alt="Cell Press Reviews" href="/reviews">Cell Press Reviews</a>
			</li>
			<li>
				<a alt="Cell Selections" href="/selections">Cell Press Selections</a>
			</li>
			<li>
				<a alt="Nucleus Collections" href="/nucleus">Nucleus Collections</a>
			</li>
			<li>
				<a alt="SnapShots" href="/snapshots">SnapShot Archive</a>
			</li>
		</ul>
	</div>
	<div class="group">
		<h2 class="group__title">Information</h2>
		<ul class="rlist clearfix">
			<li>
				<a alt="For Advertisters" href="/media-kit">For Advertisers</a>
			</li>
			<li>
				<a alt="For Recruiters" target="_blank" href="/pb-assets/Cell/common/CP-RecruitmentKit-US-2018-LowRes-1712070648.pdf">For Recruiters</a>
			</li>
			<li>
				<a alt="For Librarians" target="_blank" href="https://www.elsevier.com/products/sciencedirect/journals/direct-subscription-journals">For Librarians</a>
			</li>
			<li>
				<a alt="Privacy Policy" target="_blank" href="https://www.elsevier.com/legal/privacy-policy">Privacy Policy</a>
			</li>
			<li>
				<a alt="Terms and Conditions" target="_blank" href="https://www.elsevier.com/legal/elsevier-website-terms-and-conditions">Terms and Conditions</a>
			</li>
			<li>
				<a alt="Accessibility" href="/accessibility">Accessibility</a>
			</li>
		</ul>
	</div>
</div></div>

            

        
</div></div>

            
                </div>
            

        

    

        


    

            

            

            <div data-widget-id="779ad11f-7bcd-4a2a-a2cd-8c67b649cbb2" data-widget-name="UX3HTMLWidget"><style>
 
button#cookieBtn {
   text-decoration: none;
   background-color: transparent;
   color: #005789;
   font-weight: bold;
   padding: 0;
   margin: 0;
   font-size: 14px;
}
 
button#cookieBtn:hover {
   text-decoration: underline;
}

</style>
<script>
// HIDE ONETRUST BACKGROUND AFTER USER PRESSES ESCAPE KEY
setTimeout(function(){ 
   $("#onetrust-pc-sdk").on("keydown", function(e) { 
      if (e.keyCode == 27) {  // ESC key
         setTimeout(function(){ 
            $(".onetrust-pc-dark-filter").css("display","none");
         }, 100); 
      }
   });
}, 5000);
</script>

<span class="align-center copy-right">
<hr>
<p align="center"><b>The content on this site is intended for healthcare professionals and researchers across all fields of science.</b></p>
<hr>
<div>
<p>We use cookies to help provide and enhance our service and tailor content. To update your cookie settings, please visit the 
      <b><button id="cookieBtn" class="ot-sdk-show-settings" type="button" aria-label="Manage cookie consent preferences">Cookie Settings</button></b> for this site.<br>

All content on this site: Copyright © 2025 Elsevier Inc., its licensors, and contributors. <br>All rights are reserved, including those for text and data mining, AI training, and similar technologies. <br>For all open access content, the relevant licensing terms apply.
</p>
<ul class="rlist--inline">
			<li>
				<a alt="Privacy Policy" target="_blank" href="https://www.elsevier.com/legal/privacy-policy">Privacy Policy</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Terms and Conditions" target="_blank" href="https://www.elsevier.com/legal/elsevier-website-terms-and-conditions">Terms &amp; Conditions</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Accessibility" href="/accessibility">Accessibility</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Help and Support" target="_blank" href="https://service.elsevier.com/app/overview/cell-press/">Help &amp; Support</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Contact" href="/contact">Contact</a>
			</li>
			</ul>
<p>
<a href="https://www.relx.com"><img src="/pb/assets/raw/Health%20Advance/images/relx-1554393210983.png" alt="RELX" rel="nofollow" target="_new"></a></p>
</div>

</span></div>

            

        

    

        


    

            

            

            <div data-widget-id="c5e179c1-156e-41e3-af7f-5e307efa8acf" data-widget-name="UX3HTMLWidget"><style>#brandingRow{
    background-color:#fff;
    padding:7px 26px 7px 30px;
    color:#007DBC;
    font-size:14px;
    clear:both;
    width:100%
}
#brandingRow a{
    text-decoration:none;
    color:#007DBC
}
#brandingRow a:hover{
    text-decoration:underline
}
#brandingRowRight{
    display:inline;
    float:right;
    width:auto;
    padding:0;
    margin:0
}
svg#cp_logo {
  float: none;
  height: 36px;
  width: 36px;
  margin-left: 0px;
  margin-top: 0px;
}

footer hr {
  display: block;
}

@media screen and (max-width: 600px){
    #brandingRow{
        font-size:12px
    }
}
@media screen and (max-width: 750px){
    #brandingRow{
        padding:7px 18px 7px 16px
    }
    svg#cp_logo{
        margin-right:10px
    }
}
@media screen and (max-width: 800px) {
   a#cpIconLnk {
      display: none;
   }
}

.header__wrapper.clearfix .col-xs-6:first-of-type{
    width:55%
}
.header__wrapper.clearfix .col-xs-6:nth-of-type(2){
    width:45%
}
@media screen and (min-width: 800px) and (max-width: 1000px){
    .header__wrapper.clearfix .col-xs-6:first-of-type{
        width:75%
    }
    .header__wrapper.clearfix .col-xs-6:nth-of-type(2){
        width:25%
    }
}

</style>

<script>
var partnerx = false;
var overrideEndorsement = "";
var tmpj = $("input[name='journalCode']").attr("value");
	
if (window.location.pathname == "/molecular-therapy-family/home") {
	tmpj = "ymthe";
}
	
var ribbonx = "";
if (tmpj == "ajhg" || tmpj == "bpj" || tmpj == "molp" || tmpj == "xplc" || tmpj == "stemcr" || tmpj == "ymthe" || tmpj == "xhgg" || tmpj == "omtn" || tmpj == "omto" || tmpj == "omtm" || tmpj == "biophysreports" ||  tmpj == "bpr" || tmpj == "xinn" || tmpj == "omton") {
	partnerx = true;
	ribbonx = "<a href='https://www.cell.com' data-aa-name='cell press endorsement' data-aa-region='cp global header' title='Visit Cell.com'>A Cell Press partner journal</a>";
} else if (tmpj != "" && tmpj != "undefined" && tmpj != undefined) {
	ribbonx = "<a href='/' data-aa-name='cell press endorsement' data-aa-region='cp global header' title='Visit Cell.com'>A Cell Press journal</a>";
} else {
	ribbonx = "<a href='/' data-aa-name='cell press endorsement' data-aa-region='cp global header' title='Visit Cell.com'>A Cell Press resource</a>";
}
	
if (window.location.pathname == "/") {
	ribbonx = "";
}
if (overrideEndorsement != "") {
	ribbonx = overrideEndorsement;
}

$(".header__wrapper.clearfix").prepend("<div id='brandingRow'><a href='/' data-aa-name='science that inspires' data-aa-region='cp global header' title='Visit Cell.com'>Science that inspires</a><p id='brandingRowRight'>" + ribbonx + "</p></div>");
	
// handle scrolling effect on article pages
if ($("article").length > 0) {
	var scrollx;
	$(window).scroll(function() {
		scrollx = $(window).scrollTop();
		if (scrollx > 0) { 
			$("#brandingRow").css("display","none"); 
		} else {
			$("#brandingRow").css("display","block");
		}
	});
}

if (window.location.pathname == "/" || partnerx == false) {
               $("<a href='/home' id='cpIconLnk' data-aa-name='cell press logo' data-aa-region='cp global header' title='Cell Press' target='_blank'><svg id='cp_logo' version='1.1' xmlns='http://www.w3.org/2000/svg' xmlns:xlink='http://www.w3.org/1999/xlink' x='0px' y='0px' viewBox='0 0 100 98.1' style='enable-background:new 0 0 100 98.1;' xml:space='preserve' height='100' width='98.1'><path style='fill:#FFFFFF;' d='M69.09,0.09c-4.01-0.6-7.83,1.94-8.78,5.89l-3.14,13.11c-0.76,3.19-3.44,5.56-6.7,5.93l-18.6,2.13          C11.83,29.31-2.53,47.66,0.37,67.78c2.3,15.96,15.18,27.77,30.54,30.08c4.01,0.6,7.83-1.94,8.78-5.89l3.14-13.11 c0.76-3.19,3.44-5.56,6.69-5.93l18.6-2.13c20.04-2.17,34.41-20.52,31.5-40.65C97.32,14.21,84.45,2.39,69.09,0.09z M65.06,47.2  l-11.75,1.4c-1.98,0.21-3.62,1.65-4.08,3.59l-3.43,14.3c-1.01,4.22-4.5,7.39-8.8,8c-6.66,0.72-12.63-4.2-13.15-10.92                c-0.51-6.5,4.6-12.11,11.08-12.81l11.75-1.39c1.98-0.22,3.62-1.65,4.08-3.59l3.43-14.3c1.01-4.22,4.5-7.39,8.8-8          c6.66-0.72,12.63,4.19,13.15,10.92C76.66,40.89,71.55,46.5,65.06,47.2z'/></svg></a>").insertBefore(".header__wrapper.clearfix .journal-logos:eq(0)");
}


</script></div>

            

        

    

        


    

            

            

            <div data-widget-id="1af759b7-9bb6-4413-ada9-51a8333e8fb8" data-widget-name="UX3HTMLWidget"><style>
    /*This was added temporarily until the PB changelog tab issue got fixed. [LIT-444969 - c#4]*/
    [data-tab-id="changelogTab"] {
        display: none !important;
    }
    
    /*This was added temporarily until the PB changelog tab issue got fixed. [LIT-518839]*/
    @media (min-width: 769px) {
        .ux3 .quick-search .dropBlock [data-db-target-of],
        .quick-search .dropBlock [data-db-target-of] {
            width: 100vw !important;
        }
    }
</style></div>

            

        
</div></div>

            

        

        </div>
    

    

        


    

            

            

            <div data-widget-id="d7e0813b-a300-497a-ab37-9c3ddaf03e71" data-widget-name="UX3HTMLWidget"><!--
<div class="session-reminder hidden">
    <div class="session-reminder__overlay">
        <div class="session-reminder__dialog">
            <div class="session-reminder__dialog-inner">
                <div class="dialog-header" role="dialog" aria-labelledby="dialog-title" aria-describedby="dialog-desc">
                    <h2 class="dialog-header-title" id="dialog-title">
                        <span>Session Timeout</span>
                        <div class="dialog-timer">(2:00)</div>
                    </h2>
                    <button class="dialog-close" type="button" aria-label="close dialog">
                        <i class="icon-tools_close" aria-hidden="true"></i>
                    </button>
                </div>
                <div class="dialog-body">
                    <p class="dialog-desc" id="dialog-desc">Your session will expire shortly. If you are still working, click the ‘Keep Me Logged In’ button below. If you do not respond within the next minute, you will be automatically logged out.</p>
                    <p class="dialog-expired sr-only" aria-live="polite"></p>
                </div>
                <div class="dialog-footer">
                    <button class="session-reset" type="button">Keep Me Logged In</button>
                </div>
            </div>
        </div>
    </div>
</div>
<a class="sr-only" href="javascript:void(0)" aria-hidden="true"></a>
--></div>

            

        
        </div>
    </div>


































    



















        
        
            
                <script src="/products/marlin/cell/releasedAssets/js/main.bundle-90475670808d5d3eee41.js"></script>
            
        
        
        

    




<script type="text/javascript">
    var pageData =  {"journal":{"specialty":"Cell Press","issn":"0092-8674","name":"Cell"},"page":{"loadTimestamp":"1749592284792","server":"cell","environment":"prod","businessUnit":"ELS:RAP:JBS","noTracking":"","name":"fulltext:html","section":"Cell","language":"en","type":"CP-CA","productName":"JB"},"visitor":{"departmentName":"|ae:ScienceDirect Guests","accessType":"ae:ANON_GUEST|ae:ANON_GUEST","accountId":"ae:838949|ae:228598","accountName":"ae:Cell Guest Account|ae:ScienceDirect Guests","departmentId":"ae:977212|ae:291352","ipAddress":"189.120.74.179","loginStatus":"anonymous","userId":"ae:246228266|ae:12975512"},"content":[{"accessType":"article:free:openarchive","volumeNumber":"185","issueNumber":"1","viewState":"fulltext","format":"MIME-HTML","id":"pii:S009286742101446X","coverDate":"202201","type":"XOCS-JOURNAL:SCOPE-FULL","title":"Limb development genes underlie variation in human fingerprint patterns","status":"S300"}]};
</script>

<script type="text/javascript">
    if(typeof pageDataTracker !== "undefined") {
        pageDataTracker.trackPageLoad();
    }
</script>
<script>(function(){function c(){var b=a.contentDocument||a.contentWindow.document;if(b){var d=b.createElement('script');d.innerHTML="window.__CF$cv$params={r:'94dc16fc6a244e0b',t:'MTc0OTU5MjI4NC4wMDAwMDA='};var a=document.createElement('script');a.nonce='';a.src='/cdn-cgi/challenge-platform/scripts/jsd/main.js';document.getElementsByTagName('head')[0].appendChild(a);";b.getElementsByTagName('head')[0].appendChild(d)}}if(document.body){var a=document.createElement('iframe');a.height=1;a.width=1;a.style.position='absolute';a.style.top=0;a.style.left=0;a.style.border='none';a.style.visibility='hidden';document.body.appendChild(a);if('loading'!==document.readyState)c();else if(window.addEventListener)document.addEventListener('DOMContentLoaded',c);else{var e=document.onreadystatechange||function(){};document.onreadystatechange=function(b){e(b);'loading'!==document.readyState&&(document.onreadystatechange=e,c())}}}})();</script><iframe height="1" width="1" style="position: absolute; top: 0px; left: 0px; border: none; visibility: hidden;"></iframe><script defer="" src="https://static.cloudflareinsights.com/beacon.min.js/vcd15cbe7772f49c399c6a5babf22c1241717689176015" integrity="sha512-ZpsOmlRQV6y907TI0dKBHq9Md29nnaEIPlkf84rnaERnq6zvWvPUqr2ft8M1aS28oN72PdrCzSjY4U6VaAw1EQ==" data-cf-beacon="{&quot;rayId&quot;:&quot;94dc16fc6a244e0b&quot;,&quot;serverTiming&quot;:{&quot;name&quot;:{&quot;cfExtPri&quot;:true,&quot;cfEdge&quot;:true,&quot;cfOrigin&quot;:true,&quot;cfL4&quot;:true,&quot;cfSpeedBrain&quot;:true,&quot;cfCacheStatus&quot;:true}},&quot;version&quot;:&quot;2025.5.0&quot;,&quot;token&quot;:&quot;57a02315956f4b2e82817b3dfa99b1a2&quot;}" crossorigin="anonymous"></script>


<div id="addtoany" style="position: static;"><div class="a2a_overlay" id="a2a_overlay"></div><div id="a2a_modal" class="a2a_modal a2a_hide" role="dialog" tabindex="-1" aria-label=""><div class="a2a_modal_body a2a_menu a2a_hide" id="a2a_copy_link"><label for="a2a_copy_link_text" id="a2a_copy_link_icon" class="a2a_svg a2a_s_link a2a_logo_color"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"><title>Copy link</title><path fill="#FFF" d="M7.591 21.177c0-.36.126-.665.377-.917l2.804-2.804a1.235 1.235 0 0 1 .913-.378c.377 0 .7.144.97.43-.026.028-.11.11-.255.25-.144.14-.24.236-.29.29a2.82 2.82 0 0 0-.2.256 1.056 1.056 0 0 0-.177.344 1.43 1.43 0 0 0-.046.37c0 .36.126.666.377.918a1.25 1.25 0 0 0 .918.377c.126.001.251-.015.373-.047.125-.037.242-.096.345-.175.09-.06.176-.127.256-.2.1-.094.196-.19.29-.29.14-.142.223-.23.25-.254.297.28.445.607.445.984 0 .36-.126.664-.377.916l-2.778 2.79a1.242 1.242 0 0 1-.917.364c-.36 0-.665-.118-.917-.35l-1.982-1.97a1.223 1.223 0 0 1-.378-.9l-.001-.004Zm9.477-9.504c0-.36.126-.665.377-.917l2.777-2.79a1.235 1.235 0 0 1 .913-.378c.35 0 .656.12.917.364l1.984 1.968c.254.252.38.553.38.903 0 .36-.126.665-.38.917l-2.802 2.804a1.238 1.238 0 0 1-.916.364c-.377 0-.7-.14-.97-.418.026-.027.11-.11.255-.25a7.5 7.5 0 0 0 .29-.29c.072-.08.139-.166.2-.255.08-.103.14-.22.176-.344.032-.12.048-.245.047-.37 0-.36-.126-.662-.377-.914a1.247 1.247 0 0 0-.917-.377c-.136 0-.26.015-.37.046-.114.03-.23.09-.346.175a3.868 3.868 0 0 0-.256.2c-.054.05-.15.148-.29.29-.14.146-.222.23-.25.258-.294-.278-.442-.606-.442-.983v-.003ZM5.003 21.177c0 1.078.382 1.99 1.146 2.736l1.982 1.968c.745.75 1.658 1.12 2.736 1.12 1.087 0 2.004-.38 2.75-1.143l2.777-2.79c.75-.747 1.12-1.66 1.12-2.737 0-1.106-.392-2.046-1.183-2.818l1.186-1.185c.774.79 1.708 1.186 2.805 1.186 1.078 0 1.995-.376 2.75-1.13l2.803-2.81c.751-.754 1.128-1.671 1.128-2.748 0-1.08-.382-1.993-1.146-2.738L23.875 6.12C23.13 5.372 22.218 5 21.139 5c-1.087 0-2.004.382-2.75 1.146l-2.777 2.79c-.75.747-1.12 1.66-1.12 2.737 0 1.105.392 2.045 1.183 2.817l-1.186 1.186c-.774-.79-1.708-1.186-2.805-1.186-1.078 0-1.995.377-2.75 1.132L6.13 18.426c-.754.755-1.13 1.672-1.13 2.75l.003.001Z"></path></svg></label><input id="a2a_copy_link_text" type="text" title="Copy link" readonly=""><div id="a2a_copy_link_copied">✓</div></div><div class="a2a_modal_body a2a_menu a2a_thanks a2a_hide" id="a2a_thanks"><div class="a2a_localize" data-a2a-localize="inner,ThanksForSharing">Thanks for sharing!</div></div></div><div class="a2a_menu a2a_full a2a_localize" id="a2apage_full" role="dialog" tabindex="-1" aria-label="Share" data-a2a-localize="title,Share"><div class="a2a_full_header"><div id="a2apage_find_container" class="a2a_menu_find_container"><label for="a2apage_find" id="a2apage_find_icon" class="a2a_svg a2a_s_find"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" aria-hidden="true"><title>Find any service</title><path fill="#CCC" d="M19.7 18.2l-4.5-4.5c.7-1.1 1.2-2.3 1.2-3.6 0-3.5-2.8-6.3-6.3-6.3s-6.3 2.8-6.3 6.3 2.8 6.3 6.3 6.3c1.4 0 2.6-.4 3.6-1.2l4.5 4.5c.6.6 1.3.7 1.7.2.5-.4.4-1.1-.2-1.7zm-9.6-3.6c-2.5 0-4.5-2.1-4.5-4.5 0-2.5 2.1-4.5 4.5-4.5 2.5 0 4.5 2.1 4.5 4.5s-2 4.5-4.5 4.5z"></path></svg></label><input id="a2apage_find" class="a2a_menu_find a2a_localize" type="text" autocomplete="off" title="Find any service" data-a2a-localize="title,FindAnyServiceToAddTo"></div></div><div class="a2a_full_services" id="a2apage_full_services" role="presentation"></div><div class="a2a_full_footer"><a href="https://www.addtoany.com" title="Share Buttons" rel="noopener" target="_blank"><span class="a2a_svg a2a_s__default a2a_s_a2a a2a_logo_color"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"><g fill="#FFF"><path d="M14 7h4v18h-4z"></path><path d="M7 14h18v4H7z"></path></g></svg></span>AddToAny</a></div></div><div id="a2apage_dropdown" class="a2a_menu a2a_mini a2a_localize a2a_hide" tabindex="-1" aria-label="Share" data-a2a-localize="label,Share"><div class="a2a_mini_services" id="a2apage_mini_services"></div><div id="a2apage_cols_container" class="a2a_cols_container"><div class="a2a_col1" id="a2apage_col1"></div><div id="a2apage_2_col1" class="a2a_hide"></div><div class="a2a_clear"></div></div><div class="a2apage_wide a2a_wide"><a href="#addtoany" id="a2apage_show_more_less" class="a2a_more a2a_localize" title="Show all" data-a2a-localize="title,ShowAll"><span class="a2a_svg a2a_s__default a2a_s_a2a a2a_logo_color"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"><g fill="#FFF"><path d="M14 7h4v18h-4z"></path><path d="M7 14h18v4H7z"></path></g></svg></span><span class="a2a_localize" data-a2a-localize="inner,More">More…</span></a></div></div><div style="height: 1px; width: 1px; position: absolute; z-index: 100000; top: 0px; visibility: hidden;"><iframe id="a2a_sm_ifr" title="AddToAny Utility Frame" aria-hidden="true" src="https://static.addtoany.com/menu/sm.25.html#type=core&amp;event=load" style="height: 1px; width: 1px; border: 0px; left: 0px; top: 0px; position: absolute; z-index: 100000; display: none;"></iframe></div></div><div role="status" aria-live="assertive" aria-relevant="additions" class="ui-helper-hidden-accessible"></div><div id="crossmark-widget" style="display: none;"><div class="crossmark-reset crossmark-overlay"></div><div class="crossmark-reset crossmark-popup"><div class="crossmark-reset crossmark-popup__offset"><div class="crossmark-reset crossmark-popup__inner"><div class="crossmark-reset crossmark-popup__header"><button class="crossmark-reset crossmark-popup__btn-close"><span class="sr-only">__("articleCrossmark.closePopup")</span></button></div><div class="crossmark-reset crossmark-popup__content-wrapper"><iframe class="crossmark-reset crossmark-popup__content" title="Crossmark service - Intentionally blank" aria-hidden="false" tabindex="0" src="https://crossmark.crossref.org/dialog?doi=10.1016%2Fj.cell.2021.12.008&amp;domain=www.cell.com&amp;uri_scheme=https%3A&amp;cm_version=v2.0"></iframe></div></div></div></div></div><div id="onetrust-consent-sdk" data-nosnippet="true"><div class="onetrust-pc-dark-filter ot-fade-in" style="z-index:2147483645;"></div><div id="onetrust-banner-sdk" class="otFlat bottom ot-wo-title ot-buttons-fw" tabindex="0" role="region" aria-label="Cookie banner"><div role="dialog" aria-modal="true" aria-label="Privacy"><div class="ot-sdk-container"><div class="ot-sdk-row"><div id="onetrust-group-container" class="ot-sdk-eight ot-sdk-columns"><div class="banner_logo"></div><div id="onetrust-policy"><div id="onetrust-policy-text">We use cookies that are necessary to make our site work. We may also use additional cookies to analyze, improve, and personalize our content and your digital experience. For more information, see our<a class="ot-cookie-policy-link" href="https://www.elsevier.com/legal/cookienotice" target="_blank" aria-label=", opens in a new tab" rel="noopener">Cookie Policy</a></div></div></div><div id="onetrust-button-group-parent" class="ot-sdk-three ot-sdk-columns"><div id="onetrust-button-group"><button id="onetrust-pc-btn-handler" class="cookie-setting-link">Cookie Settings</button>  <button id="onetrust-accept-btn-handler">Accept all cookies</button></div></div></div></div><!-- Close Button --><div id="onetrust-close-btn-container"></div><!-- Close Button END--></div></div><div id="onetrust-pc-sdk" class="otPcCenter ot-hide ot-fade-in" lang="en" aria-label="Preference center" role="region"><div role="dialog" aria-modal="true" style="height: 100%;" aria-label="Cookie Preference Center"><!-- Close Button --><div class="ot-pc-header"><!-- Logo Tag --><div class="ot-pc-logo" role="img" aria-label="Company Logo"><img alt="Company Logo" src="https://cdn.cookielaw.org/logos/static/ot_company_logo.png"></div></div><!-- Close Button --><div id="ot-pc-content" class="ot-pc-scrollbar"><div class="ot-optout-signal ot-hide"><div class="ot-optout-icon"><svg xmlns="http://www.w3.org/2000/svg"><path class="ot-floating-button__svg-fill" d="M14.588 0l.445.328c1.807 1.303 3.961 2.533 6.461 3.688 2.015.93 4.576 1.746 7.682 2.446 0 14.178-4.73 24.133-14.19 29.864l-.398.236C4.863 30.87 0 20.837 0 6.462c3.107-.7 5.668-1.516 7.682-2.446 2.709-1.251 5.01-2.59 6.906-4.016zm5.87 13.88a.75.75 0 00-.974.159l-5.475 6.625-3.005-2.997-.077-.067a.75.75 0 00-.983 1.13l4.172 4.16 6.525-7.895.06-.083a.75.75 0 00-.16-.973z" fill="#FFF" fill-rule="evenodd"></path></svg></div><span></span></div><h2 id="ot-pc-title">Cookie Preference Center</h2><div id="ot-pc-desc">We use cookies which are necessary to make our site work. We may also use additional cookies to analyse, improve and personalise our content and your digital experience. For more information, see our <a href="https://www.elsevier.com/legal/cookienotice/_nocache" target="_blank">Cookie Policy</a> and the list of <a href="https://support.google.com/admanager/answer/9012903" target="_blank">Google Ad-Tech Vendors</a>.
<br>
<br>
You may choose not to allow some types of cookies. However, blocking some types may impact your experience of our site and the services we are able to offer. See the different category headings below to find out more or change your settings.
<br>
</div><button id="accept-recommended-btn-handler">Allow all</button><section class="ot-sdk-row ot-cat-grp"><h3 id="ot-category-title"> Manage Consent Preferences</h3><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="1"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-1" aria-labelledby="ot-header-id-1 ot-status-id-1"></button><!-- Accordion header --><div class="ot-acc-hdr ot-always-active-group"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-1">Strictly Necessary Cookies</h4><div id="ot-status-id-1" class="ot-always-active">Always active</div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-1">These cookies are necessary for the website to function and cannot be switched off in our systems. They are usually only set in response to actions made by you which amount to a request for services, such as setting your privacy preferences, logging in or filling in forms. You can set your browser to block or alert you about these cookies, but some parts of the site will not then work. These cookies do not store any personally identifiable information.
<br><br></p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="1">Cookie Details List‎</button></div></div></div><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="3"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-3" aria-labelledby="ot-header-id-3"></button><!-- Accordion header --><div class="ot-acc-hdr"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-3">Functional Cookies</h4><div class="ot-tgl"><input type="checkbox" name="ot-group-id-3" id="ot-group-id-3" role="switch" class="category-switch-handler" data-optanongroupid="3" aria-labelledby="ot-header-id-3"> <label class="ot-switch" for="ot-group-id-3"><span class="ot-switch-nob"></span> <span class="ot-label-txt">Functional Cookies</span></label> </div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-3">These cookies enable the website to provide enhanced functionality and personalisation. They may be set by us or by third party providers whose services we have added to our pages. If you do not allow these cookies then some or all of these services may not function properly.</p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="3">Cookie Details List‎</button></div></div></div><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="2"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-2" aria-labelledby="ot-header-id-2"></button><!-- Accordion header --><div class="ot-acc-hdr"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-2">Performance Cookies</h4><div class="ot-tgl"><input type="checkbox" name="ot-group-id-2" id="ot-group-id-2" role="switch" class="category-switch-handler" data-optanongroupid="2" aria-labelledby="ot-header-id-2"> <label class="ot-switch" for="ot-group-id-2"><span class="ot-switch-nob"></span> <span class="ot-label-txt">Performance Cookies</span></label> </div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-2">These cookies allow us to count visits and traffic sources so we can measure and improve the performance of our site. They help us to know which pages are the most and least popular and see how visitors move around the site.</p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="2">Cookie Details List‎</button></div></div></div><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="4"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-4" aria-labelledby="ot-header-id-4"></button><!-- Accordion header --><div class="ot-acc-hdr"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-4">Targeting Cookies</h4><div class="ot-tgl"><input type="checkbox" name="ot-group-id-4" id="ot-group-id-4" role="switch" class="category-switch-handler" data-optanongroupid="4" aria-labelledby="ot-header-id-4"> <label class="ot-switch" for="ot-group-id-4"><span class="ot-switch-nob"></span> <span class="ot-label-txt">Targeting Cookies</span></label> </div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-4">These cookies may be set through our site by our advertising partners. They may be used by those companies to build a profile of your interests and show you relevant adverts on other sites. If you do not allow these cookies, you will experience less targeted advertising.</p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="4">Cookie Details List‎</button></div></div></div><!-- Groups sections starts --><!-- Group section ends --><!-- Accordion Group section starts --><!-- Accordion Group section ends --></section></div><section id="ot-pc-lst" class="ot-hide ot-hosts-ui ot-pc-scrollbar"><div id="ot-pc-hdr"><div id="ot-lst-title"><button class="ot-link-btn back-btn-handler" aria-label="Back"><svg id="ot-back-arw" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 444.531 444.531" xml:space="preserve"><title>Back Button</title><g><path fill="#656565" d="M213.13,222.409L351.88,83.653c7.05-7.043,10.567-15.657,10.567-25.841c0-10.183-3.518-18.793-10.567-25.835
                    l-21.409-21.416C323.432,3.521,314.817,0,304.637,0s-18.791,3.521-25.841,10.561L92.649,196.425
                    c-7.044,7.043-10.566,15.656-10.566,25.841s3.521,18.791,10.566,25.837l186.146,185.864c7.05,7.043,15.66,10.564,25.841,10.564
                    s18.795-3.521,25.834-10.564l21.409-21.412c7.05-7.039,10.567-15.604,10.567-25.697c0-10.085-3.518-18.746-10.567-25.978
                    L213.13,222.409z"></path></g></svg></button><h3>Cookie List</h3></div><div class="ot-lst-subhdr"><div class="ot-search-cntr"><p role="status" class="ot-scrn-rdr"></p><input id="vendor-search-handler" type="text" name="vendor-search-handler" placeholder="Search…" aria-label="Cookie list search"> <svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 -30 110 110" aria-hidden="true"><title>Search Icon</title><path fill="#2e3644" d="M55.146,51.887L41.588,37.786c3.486-4.144,5.396-9.358,5.396-14.786c0-12.682-10.318-23-23-23s-23,10.318-23,23
            s10.318,23,23,23c4.761,0,9.298-1.436,13.177-4.162l13.661,14.208c0.571,0.593,1.339,0.92,2.162,0.92
            c0.779,0,1.518-0.297,2.079-0.837C56.255,54.982,56.293,53.08,55.146,51.887z M23.984,6c9.374,0,17,7.626,17,17s-7.626,17-17,17
            s-17-7.626-17-17S14.61,6,23.984,6z"></path></svg></div><div class="ot-fltr-cntr"><button id="filter-btn-handler" aria-label="Filter" aria-haspopup="true"><svg role="presentation" aria-hidden="true" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 402.577 402.577" xml:space="preserve"><title>Filter Icon</title><g><path fill="#fff" d="M400.858,11.427c-3.241-7.421-8.85-11.132-16.854-11.136H18.564c-7.993,0-13.61,3.715-16.846,11.136
      c-3.234,7.801-1.903,14.467,3.999,19.985l140.757,140.753v138.755c0,4.955,1.809,9.232,5.424,12.854l73.085,73.083
      c3.429,3.614,7.71,5.428,12.851,5.428c2.282,0,4.66-0.479,7.135-1.43c7.426-3.238,11.14-8.851,11.14-16.845V172.166L396.861,31.413
      C402.765,25.895,404.093,19.231,400.858,11.427z"></path></g></svg></button></div><div id="ot-anchor"></div><section id="ot-fltr-modal"><div id="ot-fltr-cnt"><button id="clear-filters-handler">Clear</button><div class="ot-fltr-scrlcnt ot-pc-scrollbar"><div class="ot-fltr-opts" role="list"><div class="ot-fltr-opt" role="listitem"><div class="ot-chkbox"><input id="chkbox-id" type="checkbox" class="category-filter-handler"> <label for="chkbox-id"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div></div></div><div class="ot-fltr-btns"><button id="filter-apply-handler">Apply</button> <button id="filter-cancel-handler">Cancel</button></div></div></div></section></div></div><section id="ot-lst-cnt" class="ot-host-cnt ot-pc-scrollbar"><div id="ot-sel-blk"><div class="ot-sel-all"><div class="ot-sel-all-hdr"><span class="ot-consent-hdr">Consent</span> <span class="ot-li-hdr">Leg.Interest</span></div><div class="ot-sel-all-chkbox"><div class="ot-chkbox" id="ot-selall-hostcntr"><input id="select-all-hosts-groups-handler" type="checkbox"> <label for="select-all-hosts-groups-handler"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div><div class="ot-chkbox" id="ot-selall-vencntr"><input id="select-all-vendor-groups-handler" type="checkbox"> <label for="select-all-vendor-groups-handler"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div><div class="ot-chkbox" id="ot-selall-licntr"><input id="select-all-vendor-leg-handler" type="checkbox"> <label for="select-all-vendor-leg-handler"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div></div></div></div><div class="ot-sdk-row"><div class="ot-sdk-column"><ul id="ot-host-lst"></ul></div></div></section></section><div class="ot-pc-footer ot-pc-scrollbar"><div class="ot-btn-container"> <button class="save-preference-btn-handler onetrust-close-btn-handler">Confirm my choices</button></div><!-- Footer logo --><div class="ot-pc-footer-logo"><a href="https://www.onetrust.com/products/cookie-consent/" target="_blank" rel="noopener noreferrer" aria-label="Powered by OneTrust Opens in a new Tab"><img alt="Powered by Onetrust" src="https://cdn.cookielaw.org/logos/static/powered_by_logo.svg" title="Powered by OneTrust Opens in a new Tab"></a></div></div><!-- Cookie subgroup container --><!-- Vendor list link --><!-- Cookie lost link --><!-- Toggle HTML element --><!-- Checkbox HTML --><!-- plus minus--><!-- Arrow SVG element --><!-- Accordion basic element --><span class="ot-scrn-rdr" aria-atomic="true" aria-live="polite"></span><!-- Vendor Service container and item template --></div><iframe class="ot-text-resize" sandbox="allow-same-origin" title="onetrust-text-resize" style="position: absolute; top: -50000px; width: 100em;" aria-hidden="true"></iframe></div></div></body></html>