<html lang="en" class="pb-page fonts-loaded fonts-loaded" data-request-id="1db1bc18-4c72-42d1-8745-f99decae6297" style="--article-sticky-top: 82.00019999999999px; --scroll-padding: 163.594px auto auto;"><head data-pb-dropzone="head"><meta name="pbContext" content=";product:product:elsevier\:product\:cell;website:website:cell-site;ctype:string:Journal Content;article:article:pii\:S0092867422000794;issue:issue:pii\:S0092867421X00062;journal:journal:cell;pageGroup:string:Article View;page:string:Show Full Text;requestedJournal:journal:cell;wgroup:string:Default Website Group;subPage:string:Full Text">




<!-- if there is any customization for Responsive Project widget  -->


























    
    







    
            
        



        
                <meta name="robots" content="noarchive,nocache">
            
    






    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=1, user-scalable=1">











        <meta property="og:title" content="Transcription factor protein interactomes reveal genetic determinants in heart disease">
    


        <meta property="og:type" content="Article">
    


        <meta property="og:url" content="https://www.cell.com/cell/abstract/S0092-8674(22)00079-4">
    



        <meta property="og:site_name" content="Cell">
    


        <meta property="og:image" content="https://www.cell.com/cms/asset/25d819ef-2ea7-4696-9e40-f5f967339f81/fx1.jpg">
    


        <meta property="og:description" content="The integration of human protein-protein interactome networks of endogenous transcription
factors known to be involved in cardiac malformations with the largest whole-exome-sequencing
dataset of trios with congenital heart disease identified an enrichment of rare de
novo variants among interactome proteins, pointing to candidate disease genes.">
    











        <meta name="twitter:title" content="Transcription factor protein interactomes reveal genetic determinants in heart disease">
    




        <meta name="twitter:image" content="https://www.cell.com/cms/asset/25d819ef-2ea7-4696-9e40-f5f967339f81/fx1.jpg">
    


        <meta name="twitter:description" content="The integration of human protein-protein interactome networks of endogenous transcription
factors known to be involved in cardiac malformations with the largest whole-exome-sequencing
dataset of trios with congenital heart disease identified an enrichment of rare de
novo variants among interactome proteins, pointing to candidate disease genes.">
    

        <meta name="twitter:card" content="summary_large_image">
    











    
    
            
            
        <link rel="canonical" href="https://www.cell.com/cell/fulltext/S0092-8674(22)00079-4">
    


<meta name="tdm-reservation" content="1">
<meta name="tdm-policy" content="https://www.elsevier.com/tdm/tdmrep-policy.json">

<title>Transcription factor protein interactomes reveal genetic determinants in heart disease: Cell</title><meta name="citation_journal_title" content="Cell"> <meta name="citation_journal_abbrev" content="Cell"> <meta name="citation_publisher" content="Elsevier"> <meta name="citation_title" content="Transcription factor protein interactomes reveal genetic determinants in heart disease"> <meta name="citation_date" content="2022/03/03"> <meta name="citation_online_date" content="2022/02/18"> <meta name="citation_volume" content="185"> <meta name="citation_issue" content="5"> <meta name="citation_firstpage" content="794"> <meta name="citation_lastpage" content="814.e30"> <meta name="citation_pmid" content="35182466"> <meta name="citation_abstract_html_url" content="https://www.cell.com/action/showAbstract?pii=S0092867422000794"> <meta name="citation_fulltext_html_url" content="https://www.cell.com/action/showFullText?pii=S0092867422000794"> <meta name="citation_pdf_url" content="https://www.cell.com/action/showPdf?pii=S0092867422000794"> <meta name="citation_issn" content="0092-8674"> <meta name="citation_issn" content="1097-4172"> <meta name="citation_language" content="English"> <meta name="citation_doi" content="10.1016/j.cell.2022.01.021"> <meta name="citation_pmid" content="35182466"> <meta name="citation_author" content="Barbara Gonzalez-Teran"><meta name="citation_author_orcid" content="0000-0002-4336-8644"><meta name="citation_author_institution" content="Gladstone Institutes, San Francisco, CA, USA"><meta name="citation_author_institution" content="Roddenberry Center for Stem Cell Biology and Medicine at Gladstone, San Francisco, CA, USA"><meta name="citation_author" content="Maureen Pittman"><meta name="citation_author_orcid" content="0000-0003-3714-991X"><meta name="citation_author_institution" content="Gladstone Institutes, San Francisco, CA, USA"><meta name="citation_author_institution" content="Department of Epidemiology &amp; Biostatistics, Institute for Computational Health Sciences, and Institute for Human Genetics, University of California San Francisco, San Francisco, CA, USA"><meta name="citation_author" content="Franco Felix"><meta name="citation_author_orcid" content="0000-0002-4297-7149"><meta name="citation_author_institution" content="Gladstone Institutes, San Francisco, CA, USA"><meta name="citation_author_institution" content="Roddenberry Center for Stem Cell Biology and Medicine at Gladstone, San Francisco, CA, USA"><meta name="citation_author" content="Reuben Thomas"><meta name="citation_author_institution" content="Gladstone Institutes, San Francisco, CA, USA"><meta name="citation_author" content="Desmond Richmond-Buccola"><meta name="citation_author_orcid" content="0000-0001-5574-1891"><meta name="citation_author_institution" content="Gladstone Institutes, San Francisco, CA, USA"><meta name="citation_author_institution" content="Roddenberry Center for Stem Cell Biology and Medicine at Gladstone, San Francisco, CA, USA"><meta name="citation_author" content="Ruth Hüttenhain"><meta name="citation_author_orcid" content="0000-0002-0896-5910"><meta name="citation_author_institution" content="Gladstone Institutes, San Francisco, CA, USA"><meta name="citation_author_institution" content="Department of Cellular and Molecular Pharmacology, University of California San Francisco, San Francisco, CA, USA"><meta name="citation_author_institution" content="Quantitative Biosciences Institute (QBI), University of California San Francisco, San Francisco, CA, USA"><meta name="citation_author" content="Krishna Choudhary"><meta name="citation_author_orcid" content="0000-0002-7966-1527"><meta name="citation_author_institution" content="Gladstone Institutes, San Francisco, CA, USA"><meta name="citation_author" content="Elisabetta Moroni"><meta name="citation_author_orcid" content="0000-0002-7705-7457"><meta name="citation_author_institution" content="SCITEC-CNR, Milano, Italy"><meta name="citation_author" content="Mauro W. Costa"><meta name="citation_author_orcid" content="0000-0003-3579-8625"><meta name="citation_author_institution" content="Gladstone Institutes, San Francisco, CA, USA"><meta name="citation_author_institution" content="Roddenberry Center for Stem Cell Biology and Medicine at Gladstone, San Francisco, CA, USA"><meta name="citation_author" content="Yu Huang"><meta name="citation_author_institution" content="Gladstone Institutes, San Francisco, CA, USA"><meta name="citation_author_institution" content="Roddenberry Center for Stem Cell Biology and Medicine at Gladstone, San Francisco, CA, USA"><meta name="citation_author" content="Arun Padmanabhan"><meta name="citation_author_institution" content="Gladstone Institutes, San Francisco, CA, USA"><meta name="citation_author_institution" content="Roddenberry Center for Stem Cell Biology and Medicine at Gladstone, San Francisco, CA, USA"><meta name="citation_author_institution" content="Division of Cardiology, Department of Medicine, University of California, San Francisco, CA, USA"><meta name="citation_author" content="Michael Alexanian"><meta name="citation_author_institution" content="Gladstone Institutes, San Francisco, CA, USA"><meta name="citation_author_institution" content="Roddenberry Center for Stem Cell Biology and Medicine at Gladstone, San Francisco, CA, USA"><meta name="citation_author" content="Clara Youngna Lee"><meta name="citation_author_institution" content="Gladstone Institutes, San Francisco, CA, USA"><meta name="citation_author_institution" content="Roddenberry Center for Stem Cell Biology and Medicine at Gladstone, San Francisco, CA, USA"><meta name="citation_author" content="Bonnie E.J. Maven"><meta name="citation_author_institution" content="Gladstone Institutes, San Francisco, CA, USA"><meta name="citation_author_institution" content="Roddenberry Center for Stem Cell Biology and Medicine at Gladstone, San Francisco, CA, USA"><meta name="citation_author_institution" content="Developmental and Stem Cell Biology Graduate Program, University of California San Francisco, San Francisco, CA, USA"><meta name="citation_author" content="Kaitlen Samse-Knapp"><meta name="citation_author_orcid" content="0000-0002-4116-7697"><meta name="citation_author_institution" content="Gladstone Institutes, San Francisco, CA, USA"><meta name="citation_author_institution" content="Roddenberry Center for Stem Cell Biology and Medicine at Gladstone, San Francisco, CA, USA"><meta name="citation_author" content="Sarah U. Morton"><meta name="citation_author_orcid" content="0000-0002-7816-2646"><meta name="citation_author_institution" content="Division of Newborn Medicine, Department of Medicine, Boston Children’s Hospital, Boston, MA, USA"><meta name="citation_author_institution" content="Department of Pediatrics, Harvard Medical School, Boston, MA, USA"><meta name="citation_author" content="Michael McGregor"><meta name="citation_author_institution" content="Gladstone Institutes, San Francisco, CA, USA"><meta name="citation_author_institution" content="Department of Cellular and Molecular Pharmacology, University of California San Francisco, San Francisco, CA, USA"><meta name="citation_author_institution" content="Quantitative Biosciences Institute (QBI), University of California San Francisco, San Francisco, CA, USA"><meta name="citation_author" content="Casey A. Gifford"><meta name="citation_author_orcid" content="0000-0002-7757-573X"><meta name="citation_author_institution" content="Gladstone Institutes, San Francisco, CA, USA"><meta name="citation_author_institution" content="Roddenberry Center for Stem Cell Biology and Medicine at Gladstone, San Francisco, CA, USA"><meta name="citation_author" content="J.G. Seidman"><meta name="citation_author_institution" content="Department of Genetics, Harvard Medical School, Boston, MA, USA"><meta name="citation_author" content="Christine E. Seidman"><meta name="citation_author_institution" content="Department of Genetics, Harvard Medical School, Boston, MA, USA"><meta name="citation_author_institution" content="Howard Hughes Medical Institute, Harvard Medical School, Boston, MA, USA"><meta name="citation_author_institution" content="Cardiovascular Division, Brigham and Women’s Hospital, Boston, MA, USA"><meta name="citation_author" content="Bruce D. Gelb"><meta name="citation_author_orcid" content="0000-0001-8527-5027"><meta name="citation_author_institution" content="Department of Genetics and Genomic Sciences, Icahn School of Medicine at Mount Sinai, New York, NY, USA"><meta name="citation_author_institution" content="Mindich Child Health and Development Institute, Icahn School of Medicine at Mount Sinai, New York, NY, USA"><meta name="citation_author_institution" content="Department of Pediatrics, Icahn School of Medicine at Mount Sinai, New York, NY, USA"><meta name="citation_author" content="Giorgio Colombo"><meta name="citation_author_orcid" content="0000-0002-1318-668X"><meta name="citation_author_institution" content="University of Pavia, Department of Chemistry, Pavia, Italy"><meta name="citation_author" content="Bruce R. Conklin"><meta name="citation_author_institution" content="Gladstone Institutes, San Francisco, CA, USA"><meta name="citation_author_institution" content="Roddenberry Center for Stem Cell Biology and Medicine at Gladstone, San Francisco, CA, USA"><meta name="citation_author" content="Brian L. Black"><meta name="citation_author_orcid" content="0000-0002-6664-8913"><meta name="citation_author_institution" content="Cardiovascular Research Institute, University of California San Francisco, San Francisco, CA, USA"><meta name="citation_author" content="Benoit G. Bruneau"><meta name="citation_author_orcid" content="0000-0002-0804-7597"><meta name="citation_author_institution" content="Gladstone Institutes, San Francisco, CA, USA"><meta name="citation_author_institution" content="Roddenberry Center for Stem Cell Biology and Medicine at Gladstone, San Francisco, CA, USA"><meta name="citation_author_institution" content="Cardiovascular Research Institute, University of California San Francisco, San Francisco, CA, USA"><meta name="citation_author_institution" content="Division of Cardiology, Department of Pediatrics, UCSF School of Medicine, San Francisco, CA, USA"><meta name="citation_author" content="Nevan J. Krogan"><meta name="citation_author_institution" content="Gladstone Institutes, San Francisco, CA, USA"><meta name="citation_author_institution" content="Department of Cellular and Molecular Pharmacology, University of California San Francisco, San Francisco, CA, USA"><meta name="citation_author_institution" content="Quantitative Biosciences Institute (QBI), University of California San Francisco, San Francisco, CA, USA"><meta name="citation_author" content="Katherine S. Pollard"><meta name="citation_author_email" content="katherine.pollard@gladstone.ucsf.edu"><meta name="citation_author_orcid" content="0000-0002-9870-6196"><meta name="citation_author_institution" content="Gladstone Institutes, San Francisco, CA, USA"><meta name="citation_author_institution" content="Chan Zuckerberg Biohub, San Francisco, CA, USA"><meta name="citation_author_institution" content="Department of Epidemiology &amp; Biostatistics, Institute for Computational Health Sciences, and Institute for Human Genetics, University of California San Francisco, San Francisco, CA, USA"><meta name="citation_author" content="Deepak Srivastava"><meta name="citation_author_email" content="deepak.srivastava@gladstone.ucsf.edu"><meta name="citation_author_institution" content="Gladstone Institutes, San Francisco, CA, USA"><meta name="citation_author_institution" content="Roddenberry Center for Stem Cell Biology and Medicine at Gladstone, San Francisco, CA, USA"><meta name="citation_author_institution" content="Division of Cardiology, Department of Pediatrics, UCSF School of Medicine, San Francisco, CA, USA"><meta name="citation_author_institution" content="Department of Biochemistry and Biophysics, University of California San Francisco, San Francisco, CA, USA"><meta name="citation_keywords" content="genetics; protein interactome networks; congenital heart disease; disease variants; de novo variants; GATA4; TBX5; GLYR1; NPAC"><meta name="citation_reference" content="citation_title=A reference map of murine cardiac transcription factor chromatin occupancy identifies dynamic and conserved enhancers; citation_author=B.N. Akerberg; citation_author=F. Gu; citation_author=N.J. VanDusen; citation_author=X. Zhang; citation_author=R. Dong; citation_author=K. Li; citation_author=B. Zhang; citation_author=B. Zhou; citation_author=I. Sethi; citation_author=Q. Ma; citation_author=et al.; citation_journal_title=Nat. Commun.; citation_volume=10; citation_firstpage=4907; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=A transcriptional switch governs fibroblast activation in heart disease; citation_author=M. Alexanian; citation_author=P.F. Przytycki; citation_author=R. Micheletti; citation_author=A. Padmanabhan; citation_author=L. Ye; citation_author=J.G. Travers; citation_author=B. Gonzalez-Teran; citation_author=A.C. Silva; citation_author=Q. Duan; citation_author=S.S. Ranade; citation_author=et al.; citation_journal_title=Nature; citation_volume=595; citation_firstpage=438; citation_lastpage=443; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=A quality control tool for high throughput sequence data; citation_author=S. Andrews; citation_journal_title=babraham bioinformatics; citation_publication_date=2007; "><meta name="citation_reference" content="citation_title=Disease model of GATA4 mutation reveals transcription factor cooperativity in human cardiogenesis; citation_author=Y.-S. Ang; citation_author=R.N. Rivas; citation_author=A.J.S. Ribeiro; citation_author=R. Srivas; citation_author=J. Rivera; citation_author=N.R. Stone; citation_author=K. Pratt; citation_author=T.M.A. Mohamed; citation_author=J.-D. Fu; citation_author=C.I. Spencer; citation_author=et al.; citation_journal_title=Cell; citation_volume=167; citation_firstpage=1734; citation_lastpage=1749.e22; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Comparison of Sequencing Utility Programs; citation_author=E. Aronesty; citation_journal_title=The Open Bioinformatics Journal; citation_volume=7; citation_firstpage=1; citation_lastpage=8; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=NCBI GEO: archive for functional genomics data sets--update; citation_author=T. Barrett; citation_author=S.E. Wilhite; citation_author=P. Ledoux; citation_author=C. Evangelista; citation_author=I.F. Kim; citation_author=M. Tomashevsky; citation_author=K.A. Marshall; citation_author=K.H. Phillippy; citation_author=P.M. Sherman; citation_author=M. Holko; citation_author=et al.; citation_journal_title=Nucleic Acids Res.; citation_volume=41; citation_firstpage=D991; citation_lastpage=D995; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=Comparative analysis of human tissue interactomes reveals factors leading to tissue-specific manifestation of hereditary diseases; citation_author=R. Barshir; citation_author=O. Shwartz; citation_author=I.Y. Smoly; citation_author=E. Yeger-Lotem; citation_journal_title=PLoS Comput. Biol.; citation_volume=10; citation_firstpage=e1003632; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Mutations in human TBX5 [corrected] cause limb and cardiac malformation in Holt-Oram syndrome; citation_author=C.T. Basson; citation_author=D.R. Bachinsky; citation_author=R.C. Lin; citation_author=T. Levi; citation_author=J.A. Elkins; citation_author=J. Soults; citation_author=D. Grayzel; citation_author=E. Kroumpouzou; citation_author=T.A. Traill; citation_author=J. Leblanc-Straceski; citation_author=et al.; citation_journal_title=Nat. Genet.; citation_volume=15; citation_firstpage=30; citation_lastpage=35; citation_publication_date=1997; "><meta name="citation_reference" content="citation_title=Different TBX5 interactions in heart and limb defined by Holt-Oram syndrome mutations; citation_author=C.T. Basson; citation_author=T. Huang; citation_author=R.C. Lin; citation_author=D.R. Bachinsky; citation_author=S. Weremowicz; citation_author=A. Vaglio; citation_author=R. Bruzzone; citation_author=R. Quadrelli; citation_author=M. Lerone; citation_author=G. Romeo; citation_author=et al.; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=96; citation_firstpage=2919; citation_lastpage=2924; citation_publication_date=1999; "><meta name="citation_reference" content="citation_title=Fitting linear mixed-effects models using lme4; citation_author=D. Bates; citation_author=M. Mächler; citation_author=B. Bolker; citation_author=S. Walker; citation_journal_title=J.&nbsp;Stat. Softw.; citation_volume=67; citation_firstpage=1; citation_lastpage=48; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Genetic effects on gene expression across human tissues; citation_author=GTEx Consortium; citation_journal_title=Nature; citation_volume=550; citation_firstpage=204; citation_lastpage=213; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Gromacs: A parallel computer for molecular dynamics simulations; citation_author=H. Bekker; citation_author=H.J.C. Berendsen; citation_author=E.J. Dijkstra; citation_author=S. Achterop; citation_author=R. van Drunen; citation_author=D. van der Spoel; citation_author=A. Sijbers; citation_author=H. Keegstra; citation_author=B. Reitsma; citation_author=M.K.R. Renardus; citation_publication_date=1993; "><meta name="citation_reference" content="citation_title=Oral-facial-digital syndrome type 1 in males: Congenital heart defects are included in its phenotypic spectrum; citation_author=A. Bouman; citation_author=M. Alders; citation_author=R.J. Oostra; citation_author=E. van Leeuwen; citation_author=N. Thuijs; citation_author=A.-M. van der Kevie-Kersemaekers; citation_author=M. van Maarle; citation_journal_title=Am. J. Med. Genet. A.; citation_volume=173; citation_firstpage=1383; citation_lastpage=1389; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Chamber-specific cardiac expression of Tbx5 and heart defects in Holt-Oram syndrome; citation_author=B.G. Bruneau; citation_author=M. Logan; citation_author=N. Davis; citation_author=T. Levi; citation_author=C.J. Tabin; citation_author=J.G. Seidman; citation_author=C.E. Seidman; citation_journal_title=Dev. Biol.; citation_volume=211; citation_firstpage=100; citation_lastpage=108; citation_publication_date=1999; "><meta name="citation_reference" content="citation_title=A murine model of Holt-Oram syndrome defines roles of the T-box transcription factor Tbx5 in cardiogenesis and disease; citation_author=B.G. Bruneau; citation_author=G. Nemer; citation_author=J.P. Schmitt; citation_author=F. Charron; citation_author=L. Robitaille; citation_author=S. Caron; citation_author=D.A. Conner; citation_author=M. Gessler; citation_author=M. Nemer; citation_author=C.E. Seidman; citation_author=J.G. Seidman; citation_journal_title=Cell; citation_volume=106; citation_firstpage=709; citation_lastpage=721; citation_publication_date=2001; "><meta name="citation_reference" content="citation_title=Genetic identification of cell types underlying brain complex traits yields insights into the etiology of Parkinson’s disease; citation_author=J. Bryois; citation_author=N.G. Skene; citation_author=T.F. Hansen; citation_author=L.J.A. Kogelman; citation_author=H.J. Watson; citation_author=Z. Liu; citation_author=L. Brueggeman; citation_author=G. Breen; citation_author=C.M. Bulik; citation_author=E. Arenas; citation_author=Eating Disorders Working Group of the Psychiatric Genomics Consortium; citation_author=International Headache Genetics Consortium; citation_author=23andMe Research Team; citation_author=et al.; citation_journal_title=Nat. Genet.; citation_volume=52; citation_firstpage=482; citation_lastpage=493; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Human induced pluripotent stem cell-derived cardiomyocytes recapitulate the predilection of breast cancer patients to doxorubicin-induced cardiotoxicity; citation_author=P.W. Burridge; citation_author=Y.F. Li; citation_author=E. Matsa; citation_author=H. Wu; citation_author=S.G. Ong; citation_author=A. Sharma; citation_author=A. Holmström; citation_author=A.C. Chang; citation_author=M.J. Coronado; citation_author=A.D. Ebert; citation_author=et al.; citation_journal_title=Nat. Med.; citation_volume=22; citation_firstpage=547; citation_lastpage=556; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=A human cell atlas of fetal gene expression; citation_author=J. Cao; citation_author=D.R. O’Day; citation_author=H.A. Pliner; citation_author=P.D. Kingsley; citation_author=M. Deng; citation_author=R.M. Daza; citation_author=M.A. Zager; citation_author=K.A. Aldinger; citation_author=R. Blecher-Gonen; citation_author=F. Zhang; citation_author=et al.; citation_journal_title=Science; citation_volume=370; citation_doi=10.1126/science.aba7721; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=AMBER 2018; citation_author=D.A. Case; citation_author=I.Y. Ben-Shalom; citation_author=S.R. Brozell; citation_author=D.S. Cerutti; citation_author=T.E. Cheatham, III; citation_author=V.W.D. Cruzeiro; citation_author=T.A. Darden; citation_author=R.E. Duke; citation_author=D. Ghoreishi; citation_author=M.K. Gilson; citation_author=et al.; citation_publication_date=2018; citation_publisher=University of California, San Francisco; "><meta name="citation_reference" content="citation_title=annotatr: genomic regions in context; citation_author=R.G. Cavalcante; citation_author=M.A. Sartor; citation_journal_title=Bioinformatics; citation_volume=33; citation_firstpage=2381; citation_lastpage=2383; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Recessive TTN truncating mutations define novel forms of core myopathy with heart disease; citation_author=C. Chauveau; citation_author=C.G. Bonnemann; citation_author=C. Julien; citation_author=A.L. Kho; citation_author=H. Marks; citation_author=B. Talim; citation_author=P. Maury; citation_author=M.C. Arne-Bes; citation_author=E. Uro-Coste; citation_author=A. Alexandrovich; citation_author=et al.; citation_journal_title=Hum. Mol. Genet.; citation_volume=23; citation_firstpage=980; citation_lastpage=991; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=A novel TAB2 nonsense mutation (p.S149X) causing autosomal dominant congenital heart defects: a case report of a Chinese family; citation_author=J. Chen; citation_author=H. Yuan; citation_author=K. Xie; citation_author=X. Wang; citation_author=L. Tan; citation_author=Y. Zou; citation_author=Y. Yang; citation_author=L. Pan; citation_author=J. Xiao; citation_author=G. Chen; citation_author=Y. Liu; citation_journal_title=BMC Cardiovasc. Disord.; citation_volume=20; citation_firstpage=27; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=March of dimes; citation_author=A. Christianson; citation_author=C.P. Howson; citation_publication_date=2006; citation_publisher=Global Report on Birth.; "><meta name="citation_reference" content="citation_title=Cranial and cardiac neural crest defects in endothelin-A receptor-deficient mice; citation_author=D.E. Clouthier; citation_author=K. Hosoda; citation_author=J.A. Richardson; citation_author=S.C. Williams; citation_author=H. Yanagisawa; citation_author=T. Kuwaki; citation_author=M. Kumada; citation_author=R.E. Hammer; citation_author=M. Yanagisawa; citation_journal_title=Development; citation_volume=125; citation_firstpage=813; citation_lastpage=824; citation_publication_date=1998; "><meta name="citation_reference" content="citation_title=MaxQuant enables high peptide identification rates, individualized p.p.b.-range mass accuracies and proteome-wide protein quantification; citation_author=J. Cox; citation_author=M. Mann; citation_journal_title=Nat. Biotechnol.; citation_volume=26; citation_firstpage=1367; citation_lastpage=1372; citation_publication_date=2008; "><meta name="citation_reference" content="citation_title=Particle mesh Ewald: An N×log(N) method for Ewald sums in large systems; citation_author=T. Darden; citation_author=D. York; citation_author=L. Pedersen; citation_journal_title=J.&nbsp;Chem. Phys.; citation_volume=98; citation_firstpage=10089; citation_publication_date=1993; "><meta name="citation_reference" content="citation_title=Single-cell analysis of cardiogenesis reveals basis for organ-level developmental defects; citation_author=T.Y. de Soysa; citation_author=S.S. Ranade; citation_author=S. Okawa; citation_author=S. Ravichandran; citation_author=Y. Huang; citation_author=H.T. Salunga; citation_author=A. Schricker; citation_author=A. Del Sol; citation_author=C.A. Gifford; citation_author=D. Srivastava; citation_journal_title=Nature; citation_volume=572; citation_firstpage=120; citation_lastpage=124; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Large-scale discovery of novel genetic causes of developmental disorders; citation_author=Deciphering Developmental Disorders Study; citation_journal_title=Nature; citation_volume=519; citation_firstpage=223; citation_lastpage=228; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=STAR: ultrafast universal RNA-seq aligner; citation_author=A. Dobin; citation_author=C.A. Davis; citation_author=F. Schlesinger; citation_author=J. Drenkow; citation_author=C. Zaleski; citation_author=S. Jha; citation_author=P. Batut; citation_author=M. Chaisson; citation_author=T.R. Gingeras; citation_journal_title=Bioinformatics; citation_volume=29; citation_firstpage=15; citation_lastpage=21; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=Sequential Binding of MEIS1 and NKX2-5 on the Popdc2 Gene: A Mechanism for Spatiotemporal Regulation of Enhancers during Cardiogenesis; citation_author=L. Dupays; citation_author=C. Shang; citation_author=R. Wilson; citation_author=S. Kotecha; citation_author=S. Wood; citation_author=N. Towers; citation_author=T. Mohun; citation_journal_title=Cell Rep.; citation_volume=13; citation_firstpage=183; citation_lastpage=195; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Settling the score: variant prioritization and Mendelian disease; citation_author=K. Eilbeck; citation_author=A. Quinlan; citation_author=M. Yandell; citation_journal_title=Nat. Rev. Genet.; citation_volume=18; citation_firstpage=599; citation_lastpage=612; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=GATA4 loss of function in liver cancer impedes precursor to hepatocyte transition; citation_author=F.O. Enane; citation_author=W.H. Shuen; citation_author=X. Gu; citation_author=E. Quteba; citation_author=B. Przychodzen; citation_author=H. Makishima; citation_author=J. Bodo; citation_author=J. Ng; citation_author=C.L. Chee; citation_author=R. Ba; citation_author=et al.; citation_journal_title=J.&nbsp;Clin. Invest.; citation_volume=127; citation_firstpage=3527; citation_lastpage=3542; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=LSD2/KDM1B and its cofactor NPAC/GLYR1 endow a structural and molecular model for regulation of H3K4 demethylation; citation_author=R. Fang; citation_author=F. Chen; citation_author=Z. Dong; citation_author=D. Hu; citation_author=A.J. Barbera; citation_author=E.A. Clark; citation_author=J. Fang; citation_author=Y. Yang; citation_author=P. Mei; citation_author=M. Rutenberg; citation_author=et al.; citation_journal_title=Mol. Cell; citation_volume=49; citation_firstpage=558; citation_lastpage=570; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=Enhanced utility of family-centered diagnostic exome sequencing with inheritance model-based analysis: results from 500 unselected families with undiagnosed genetic conditions; citation_author=K.D. Farwell; citation_author=L. Shahmirzadi; citation_author=D. El-Khechen; citation_author=Z. Powis; citation_author=E.C. Chao; citation_author=B. Tippin Davis; citation_author=R.M. Baxter; citation_author=W. Zeng; citation_author=C. Mroske; citation_author=M.C. Parra; citation_author=et al.; citation_journal_title=Genet. Med.; citation_volume=17; citation_firstpage=578; citation_lastpage=586; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=NDF, a nucleosome-destabilizing factor that facilitates transcription through nucleosomes; citation_author=J. Fei; citation_author=H. Ishii; citation_author=M.A. Hoeksema; citation_author=F. Meitinger; citation_author=G.A. Kassavetis; citation_author=C.K. Glass; citation_author=B. Ren; citation_author=J.T. Kadonaga; citation_journal_title=Genes Dev.; citation_volume=32; citation_firstpage=682; citation_lastpage=694; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=MAST: a flexible statistical framework for assessing transcriptional changes and characterizing heterogeneity in single-cell RNA sequencing data; citation_author=G. Finak; citation_author=A. McDavid; citation_author=M. Yajima; citation_author=J. Deng; citation_author=V. Gersuk; citation_author=A.K. Shalek; citation_author=C.K. Slichter; citation_author=H.W. Miller; citation_author=M.J. McElrath; citation_author=M. Prlic; citation_author=et al.; citation_journal_title=Genome Biol.; citation_volume=16; citation_firstpage=278; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Nuclear protein NP60 regulates p38 MAPK activity; citation_author=J. Fu; citation_author=Z. Yang; citation_author=J. Wei; citation_author=J. Han; citation_author=J. Gu; citation_journal_title=J.&nbsp;Cell Sci.; citation_volume=119; citation_firstpage=115; citation_lastpage=123; citation_publication_date=2006; "><meta name="citation_reference" content="citation_title=Measuring intolerance to mutation in human genetics; citation_author=Z.L. Fuller; citation_author=J.J. Berg; citation_author=H. Mostafavi; citation_author=G. Sella; citation_author=M. Przeworski; citation_journal_title=Nat. Genet.; citation_volume=51; citation_firstpage=772; citation_lastpage=776; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Point mutations in murine Nkx2-5 phenocopy human congenital heart disease and induce pathogenic Wnt signaling; citation_author=M.B. Furtado; citation_author=J.C. Wilmanns; citation_author=A. Chandran; citation_author=J. Perera; citation_author=O. Hon; citation_author=C. Biben; citation_author=T.J. Willow; citation_author=H.T. Nim; citation_author=G. Kaur; citation_author=S. Simonds; citation_author=et al.; citation_journal_title=JCI Insight; citation_volume=2; citation_firstpage=e88271; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=GATA4 mutations cause human congenital heart defects and reveal an interaction with TBX5; citation_author=V. Garg; citation_author=I.S. Kathiriya; citation_author=R. Barnes; citation_author=M.K. Schluterman; citation_author=I.N. King; citation_author=C.A. Butler; citation_author=C.R. Rothrock; citation_author=R.S. Eapen; citation_author=K. Hirayama-Yamada; citation_author=K. Joo; citation_author=et al.; citation_journal_title=Nature; citation_volume=424; citation_firstpage=443; citation_lastpage=447; citation_publication_date=2003; "><meta name="citation_reference" content="citation_title=Oligogenic inheritance of a human heart disease involving a genetic modifier; citation_author=C.A. Gifford; citation_author=S.S. Ranade; citation_author=R. Samarakoon; citation_author=H.T. Salunga; citation_author=T.Y. de Soysa; citation_author=Y. Huang; citation_author=P. Zhou; citation_author=A. Elfenbein; citation_author=S.K. Wyman; citation_author=Y.K. Bui; citation_author=et al.; citation_journal_title=Science; citation_volume=364; citation_firstpage=865; citation_lastpage=870; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=The human disease network; citation_author=K.-I. Goh; citation_author=M.E. Cusick; citation_author=D. Valle; citation_author=B. Childs; citation_author=M. Vidal; citation_author=A.-L. Barabási; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=104; citation_firstpage=8685; citation_lastpage=8690; citation_publication_date=2007; "><meta name="citation_reference" content="citation_title=p38γ and δ promote heart hypertrophy by targeting the mTOR-inhibitory protein DEPTOR for degradation; citation_author=B. González-Terán; citation_author=J.A. López; citation_author=E. Rodríguez; citation_author=L. Leiva; citation_author=S. Martínez-Martínez; citation_author=J.A. Bernal; citation_author=L.J. Jiménez-Borreguero; citation_author=J.M. Redondo; citation_author=J. Vazquez; citation_author=G. Sabio; citation_journal_title=Nat. Commun.; citation_volume=7; citation_firstpage=10477; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=PRROC: computing and visualizing precision-recall and receiver operating characteristic curves in R; citation_author=J. Grau; citation_author=I. Grosse; citation_author=J. Keilwagen; citation_journal_title=Bioinformatics; citation_volume=31; citation_firstpage=2595; citation_lastpage=2597; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Understanding multicellular function and disease with human tissue-specific networks; citation_author=C.S. Greene; citation_author=A. Krishnan; citation_author=A.K. Wong; citation_author=E. Ricciotti; citation_author=R.A. Zelaya; citation_author=D.S. Himmelstein; citation_author=R. Zhang; citation_author=B.M. Hartmann; citation_author=E. Zaslavsky; citation_author=S.C. Sealfon; citation_author=et al.; citation_journal_title=Nat. Genet.; citation_volume=47; citation_firstpage=569; citation_lastpage=576; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=High resolution genome wide binding event finding and motif discovery reveals transcription factor spatial binding constraints; citation_author=Y. Guo; citation_author=S. Mahony; citation_author=D.K. Gifford; citation_journal_title=PLoS Comput. Biol.; citation_volume=8; citation_firstpage=e1002638; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=Integrated analysis of multimodal single-cell data; citation_author=Y. Hao; citation_author=S. Hao; citation_author=E. Andersen-Nissen; citation_author=W.M. Mauck; citation_author=S. Zheng; citation_author=A. Butler; citation_author=M.J. Lee; citation_author=A.J. Wilk; citation_author=C. Darby; citation_author=M. Zager; citation_author=et al.; citation_journal_title=Cell; citation_volume=184; citation_firstpage=3573; citation_lastpage=3587.e29; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=Simple combinations of lineage-determining transcription factors prime cis-regulatory elements required for macrophage and B cell identities; citation_author=S. Heinz; citation_author=C. Benner; citation_author=N. Spann; citation_author=E. Bertolino; citation_author=Y.C. Lin; citation_author=P. Laslo; citation_author=J.X. Cheng; citation_author=C. Murre; citation_author=H. Singh; citation_author=C.K. Glass; citation_journal_title=Mol. Cell; citation_volume=38; citation_firstpage=576; citation_lastpage=589; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=Mechanisms of tissue and cell-type specificity in heritable traits and diseases; citation_author=I. Hekselman; citation_author=E. Yeger-Lotem; citation_journal_title=Nat. Rev. Genet.; citation_volume=21; citation_firstpage=137; citation_lastpage=150; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Cardiovascular manifestations of tuberous sclerosis complex and summary of the revised diagnostic criteria and surveillance and management recommendations from the International Tuberous Sclerosis Consensus Group; citation_author=R.B. Hinton; citation_author=A. Prakash; citation_author=R.L. Romp; citation_author=D.A. Krueger; citation_author=T.K. Knilans; citation_author=International Tuberous Sclerosis Consensus Group; citation_journal_title=J.&nbsp;Am. Heart Assoc.; citation_volume=3; citation_firstpage=e001493; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=De novo mutations in congenital heart disease with neurodevelopmental and other congenital anomalies; citation_author=J. Homsy; citation_author=S. Zaidi; citation_author=Y. Shen; citation_author=J.S. Ware; citation_author=K.E. Samocha; citation_author=K.J. Karczewski; citation_author=S.R. DePalma; citation_author=D. McKean; citation_author=H. Wakimoto; citation_author=J. Gorham; citation_author=et al.; citation_journal_title=Science; citation_volume=350; citation_firstpage=1262; citation_lastpage=1266; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=ATP-dependent chromatin remodeling during mammalian development; citation_author=S.K. Hota; citation_author=B.G. Bruneau; citation_journal_title=Development; citation_volume=143; citation_firstpage=2882; citation_lastpage=2897; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Characterising and predicting haploinsufficiency in the human genome; citation_author=N. Huang; citation_author=I. Lee; citation_author=E.M. Marcotte; citation_author=M.E. Hurles; citation_journal_title=PLoS Genet.; citation_volume=6; citation_firstpage=e1001154; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=Systems genetics analysis identifies calcium-signaling defects as novel cause of congenital heart disease; citation_author=J.M.G. Izarzugaza; citation_author=S.G. Ellesøe; citation_author=C. Doganli; citation_author=N.S. Ehlers; citation_author=M.D. Dalgaard; citation_author=E. Audain; citation_author=G. Dombrowsky; citation_author=K. Banasik; citation_author=A. Sifrim; citation_author=A. Wilsdon; citation_author=Competence Network for Congenital Heart Defects, Germany; citation_author=et al.; citation_journal_title=Genome Med.; citation_volume=12; citation_firstpage=76; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=artMS: Analytical R tools for Mass Spectrometry version 1.6.5 from Bioconductor; citation_author=D. Jimenez-Morales; citation_author=A. Rosa Campos; citation_author=J. Von Dollen; citation_author=D. Swaney; citation_journal_title=Bioconductor; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Contribution of rare inherited and de novo variants in 2,871 congenital heart disease probands; citation_author=S.C. Jin; citation_author=J. Homsy; citation_author=S. Zaidi; citation_author=Q. Lu; citation_author=S. Morton; citation_author=S.R. DePalma; citation_author=X. Zeng; citation_author=H. Qi; citation_author=W. Chang; citation_author=M.C. Sierant; citation_author=et al.; citation_journal_title=Nat. Genet.; citation_volume=49; citation_firstpage=1593; citation_lastpage=1601; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=De novo mutations in MLL cause Wiedemann-Steiner syndrome; citation_author=W.D. Jones; citation_author=D. Dafou; citation_author=M. McEntagart; citation_author=W.J. Woollard; citation_author=F.V. Elmslie; citation_author=M. Holder-Espinasse; citation_author=M. Irving; citation_author=A.K. Saggar; citation_author=S. Smithson; citation_author=R.C. Trembath; citation_author=et al.; citation_journal_title=Am. J. Hum. Genet.; citation_volume=91; citation_firstpage=358; citation_lastpage=364; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=Comparison of simple potential functions for simulating liquid water; citation_author=W.L. Jorgensen; citation_author=J. Chandrasekhar; citation_author=J.D. Madura; citation_author=R.W. Impey; citation_author=M.L. Klein; citation_journal_title=J.&nbsp;Chem. Phys.; citation_volume=79; citation_firstpage=926; citation_publication_date=1983; "><meta name="citation_reference" content="citation_title=The mutational constraint spectrum quantified from variation in 141,456 humans; citation_author=K.J. Karczewski; citation_author=L.C. Francioli; citation_author=G. Tiao; citation_author=B.B. Cummings; citation_author=J. Alföldi; citation_author=Q. Wang; citation_author=R.L. Collins; citation_author=K.M. Laricchia; citation_author=A. Ganna; citation_author=D.P. Birnbaum; citation_author=Genome Aggregation Database Consortium; citation_author=et al.; citation_journal_title=Nature; citation_volume=581; citation_firstpage=434; citation_lastpage=443; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Modeling human TBX5 haploinsufficiency predicts regulatory networks for congenital heart disease; citation_author=I.S. Kathiriya; citation_author=K.S. Rao; citation_author=G. Iacono; citation_author=W.P. Devine; citation_author=A.P. Blair; citation_author=S.K. Hota; citation_author=M.H. Lai; citation_author=B.I. Garay; citation_author=R. Thomas; citation_author=H.Z. Gong; citation_author=et al.; citation_journal_title=Dev. Cell; citation_volume=56; citation_firstpage=292; citation_lastpage=309.e9; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=Histone modifications for human epigenome analysis; citation_author=H. Kimura; citation_journal_title=J.&nbsp;Hum. Genet.; citation_volume=58; citation_firstpage=439; citation_lastpage=445; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=Co-regulation of the atrial natriuretic factor and cardiac myosin light chain-2 genes during alpha-adrenergic stimulation of neonatal rat ventricular cells. Identification of cis sequences within an embryonic and a constitutive contractile protein gene which mediate inducible expression; citation_author=K.U. Knowlton; citation_author=E. Baracchini; citation_author=R.S. Ross; citation_author=A.N. Harris; citation_author=S.A. Henderson; citation_author=S.M. Evans; citation_author=C.C. Glembotski; citation_author=K.R. Chien; citation_journal_title=J.&nbsp;Biol. Chem.; citation_volume=266; citation_firstpage=7759; citation_lastpage=7768; citation_publication_date=1991; "><meta name="citation_reference" content="citation_title=Walking the interactome for prioritization of candidate disease genes; citation_author=S. Köhler; citation_author=S. Bauer; citation_author=D. Horn; citation_author=P.N. Robinson; citation_journal_title=Am. J. Hum. Genet.; citation_volume=82; citation_firstpage=949; citation_lastpage=958; citation_publication_date=2008; "><meta name="citation_reference" content="citation_title=Fast, sensitive and accurate integration of single-cell data with Harmony; citation_author=I. Korsunsky; citation_author=N. Millard; citation_author=J. Fan; citation_author=K. Slowikowski; citation_author=F. Zhang; citation_author=K. Wei; citation_author=Y. Baglaenko; citation_author=M. Brenner; citation_author=P.R. Loh; citation_author=S. Raychaudhuri; citation_journal_title=Nat. Methods; citation_volume=16; citation_firstpage=1289; citation_lastpage=1296; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=GATA4 transcription factor is required for ventral morphogenesis and heart tube formation; citation_author=C.T. Kuo; citation_author=E.E. Morrisey; citation_author=R. Anandappa; citation_author=K. Sigrist; citation_author=M.M. Lu; citation_author=M.S. Parmacek; citation_author=C. Soudais; citation_author=J.M. Leiden; citation_journal_title=Genes Dev.; citation_volume=11; citation_firstpage=1048; citation_lastpage=1060; citation_publication_date=1997; "><meta name="citation_reference" content="citation_title=The human transcription factors; citation_author=S.A. Lambert; citation_author=A. Jolma; citation_author=L.F. Campitelli; citation_author=P.K. Das; citation_author=Y. Yin; citation_author=M. Albu; citation_author=X. Chen; citation_author=J. Taipale; citation_author=T.R. Hughes; citation_author=M.T. Weirauch; citation_journal_title=Cell; citation_volume=172; citation_firstpage=650; citation_lastpage=665; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Fast gapped-read alignment with Bowtie 2; citation_author=B. Langmead; citation_author=S.L. Salzberg; citation_journal_title=Nat. Methods; citation_volume=9; citation_firstpage=357; citation_lastpage=359; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=Splice-Junction-Based Mapping of Alternative Isoforms in the Human Proteome; citation_author=E. Lau; citation_author=Y. Han; citation_author=D.R. Williams; citation_author=C.T. Thomas; citation_author=R. Shrestha; citation_author=J.C. Wu; citation_author=M.P.Y. Lam; citation_journal_title=Cell Rep.; citation_volume=29; citation_firstpage=3751; citation_lastpage=3765.e5; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Holt-Oram syndrome is caused by mutations in TBX5, a member of the Brachyury (T) gene family; citation_author=Q.Y. Li; citation_author=R.A. Newbury-Ecob; citation_author=J.A. Terrett; citation_author=D.I. Wilson; citation_author=A.R. Curtis; citation_author=C.H. Yi; citation_author=T. Gebuhr; citation_author=P.J. Bullen; citation_author=S.C. Robson; citation_author=T. Strachan; citation_author=et al.; citation_journal_title=Nat. Genet.; citation_volume=15; citation_firstpage=21; citation_lastpage=29; citation_publication_date=1997; "><meta name="citation_reference" content="citation_title=Directed cardiomyocyte differentiation from human pluripotent stem cells by modulating Wnt/β-catenin signaling under fully defined conditions; citation_author=X. Lian; citation_author=J. Zhang; citation_author=S.M. Azarin; citation_author=K. Zhu; citation_author=L.B. Hazeltine; citation_author=X. Bao; citation_author=C. Hsiao; citation_author=T.J. Kamp; citation_author=S.P. Palecek; citation_journal_title=Nat. Protoc.; citation_volume=8; citation_firstpage=162; citation_lastpage=175; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=featureCounts: an efficient general purpose program for assigning sequence reads to genomic features; citation_author=Y. Liao; citation_author=G.K. Smyth; citation_author=W. Shi; citation_journal_title=Bioinformatics; citation_volume=30; citation_firstpage=923; citation_lastpage=930; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=CHD7 interacts with BMP R-SMADs to epigenetically regulate cardiogenesis in mice; citation_author=Y. Liu; citation_author=C. Harmelink; citation_author=Y. Peng; citation_author=Y. Chen; citation_author=Q. Wang; citation_author=K. Jiao; citation_journal_title=Hum. Mol. Genet.; citation_volume=23; citation_firstpage=2145; citation_lastpage=2156; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=It’s DE-licious: A Recipe for Differential Expression Analyses of RNA-seq Experiments Using Quasi-Likelihood Methods in edgeR; citation_author=A.T.L. Lun; citation_author=Y. Chen; citation_author=G.K. Smyth; citation_journal_title=Methods Mol. Biol.; citation_volume=1418; citation_firstpage=391; citation_lastpage=416; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Complex interdependence regulates heterotypic transcription factor distribution and coordinates cardiogenesis; citation_author=L. Luna-Zurita; citation_author=C.U. Stirnimann; citation_author=S. Glatt; citation_author=B.L. Kaynak; citation_author=S. Thomas; citation_author=F. Baudin; citation_author=M.A.H. Samee; citation_author=D. He; citation_author=E.M. Small; citation_author=M. Mileikovsky; citation_author=et al.; citation_journal_title=Cell; citation_volume=164; citation_firstpage=999; citation_lastpage=1014; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=A non-invasive platform for functional characterization of stem-cell-derived cardiomyocytes with applications in cardiotoxicity testing; citation_author=M. Maddah; citation_author=J.D. Heidmann; citation_author=M.A. Mandegar; citation_author=C.D. Walker; citation_author=S. Bolouki; citation_author=B.R. Conklin; citation_author=K.E. Loewke; citation_journal_title=Stem Cell Reports; citation_volume=4; citation_firstpage=621; citation_lastpage=631; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=BiNGO: a Cytoscape plugin to assess overrepresentation of gene ontology categories in biological networks; citation_author=S. Maere; citation_author=K. Heymans; citation_author=M. Kuiper; citation_journal_title=Bioinformatics; citation_volume=21; citation_firstpage=3448; citation_lastpage=3449; citation_publication_date=2005; "><meta name="citation_reference" content="citation_title=Maestro Suite of Programs (v. 2019-4); citation_author=Maestro Schrödinger, LLC; citation_publication_date=2019; citation_publisher=Maestro Schrödinger, LLC; "><meta name="citation_reference" content="citation_title=Enhancing the prioritization of disease-causing genes through tissue specific protein interaction networks; citation_author=O. Magger; citation_author=Y.Y. Waldman; citation_author=E. Ruppin; citation_author=R. Sharan; citation_journal_title=PLoS Comput. Biol.; citation_volume=8; citation_firstpage=e1002690; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=Interaction of Gata4 and Gata6 with Tbx5 is critical for normal cardiac development; citation_author=M. Maitra; citation_author=M.K. Schluterman; citation_author=H.A. Nichols; citation_author=J.A. Richardson; citation_author=C.W. Lo; citation_author=D. Srivastava; citation_author=V. Garg; citation_journal_title=Dev. Biol.; citation_volume=326; citation_firstpage=368; citation_lastpage=377; citation_publication_date=2009; "><meta name="citation_reference" content="citation_title=Identification of GATA6 sequence variants in patients with congenital heart defects; citation_author=M. Maitra; citation_author=S.N. Koenig; citation_author=D. Srivastava; citation_author=V. Garg; citation_journal_title=Pediatr. Res.; citation_volume=68; citation_firstpage=281; citation_lastpage=285; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=A Tail-Based Mechanism Drives Nucleosome Demethylation by the LSD2/NPAC Multimeric Complex; citation_author=C. Marabelli; citation_author=B. Marrocco; citation_author=S. Pilotto; citation_author=S. Chittori; citation_author=S. Picaud; citation_author=S. Marchese; citation_author=G. Ciossani; citation_author=F. Forneris; citation_author=P. Filippakopoulos; citation_author=G. Schoehn; citation_author=et al.; citation_journal_title=Cell Rep.; citation_volume=27; citation_firstpage=387; citation_lastpage=399.e7; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Settle: An analytical version of the SHAKE and RATTLE algorithm for rigid water models; citation_author=S. Miyamoto; citation_author=P.A. Kollman; citation_journal_title=J.&nbsp;Comput. Chem.; citation_volume=13; citation_firstpage=952; citation_lastpage=962; citation_publication_date=1992; "><meta name="citation_reference" content="citation_title=Isolation of single-base genome-edited human iPS cells without antibiotic selection; citation_author=Y. Miyaoka; citation_author=A.H. Chan; citation_author=L.M. Judge; citation_author=J. Yoo; citation_author=M. Huang; citation_author=T.D. Nguyen; citation_author=P.P. Lizarraga; citation_author=P.L. So; citation_author=B.R. Conklin; citation_journal_title=Nat. Methods; citation_volume=11; citation_firstpage=291; citation_lastpage=293; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Requirement of the transcription factor GATA4 for heart tube formation and ventral morphogenesis; citation_author=J.D. Molkentin; citation_author=Q. Lin; citation_author=S.A. Duncan; citation_author=E.N. Olson; citation_journal_title=Genes Dev.; citation_volume=11; citation_firstpage=1061; citation_lastpage=1072; citation_publication_date=1997; "><meta name="citation_reference" content="citation_title=Impact of Mutations on NPAC Structural Dynamics: Mechanistic Insights from MD Simulations; citation_author=M. Montefiori; citation_author=S. Pilotto; citation_author=C. Marabelli; citation_author=E. Moroni; citation_author=M. Ferraro; citation_author=S.A. Serapian; citation_author=A. Mattevi; citation_author=G. Colombo; citation_journal_title=J.&nbsp;Chem. Inf. Model.; citation_volume=59; citation_firstpage=3927; citation_lastpage=3937; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Tbx5-dependent rheostatic control of cardiac gene expression and morphogenesis; citation_author=A.D. Mori; citation_author=Y. Zhu; citation_author=I. Vahora; citation_author=B. Nieman; citation_author=K. Koshiba-Takeuchi; citation_author=L. Davidson; citation_author=A. Pizard; citation_author=J.G. Seidman; citation_author=C.E. Seidman; citation_author=X.J. Chen; citation_author=et al.; citation_journal_title=Dev. Biol.; citation_volume=297; citation_firstpage=566; citation_lastpage=586; citation_publication_date=2006; "><meta name="citation_reference" content="citation_title=Transcription factor genes Smad4 and Gata4 cooperatively regulate cardiac valve development. [corrected]; citation_author=I.P. Moskowitz; citation_author=J. Wang; citation_author=M.A. Peterson; citation_author=W.T. Pu; citation_author=A.C. Mackinnon; citation_author=L. Oxburgh; citation_author=G.C. Chu; citation_author=M. Sarkar; citation_author=C. Berul; citation_author=L. Smoot; citation_author=et al.; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=108; citation_firstpage=4006; citation_lastpage=4011; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=Mouse Tenm4 is required for mesoderm induction; citation_author=H. Nakamura; citation_author=R.N. Cook; citation_author=M.J. Justice; citation_journal_title=BMC Dev. Biol.; citation_volume=13; citation_firstpage=9; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=Cardiomyocyte differentiation by GATA-4-deficient embryonic stem cells; citation_author=N. Narita; citation_author=M. Bielinska; citation_author=D.B. Wilson; citation_journal_title=Development; citation_volume=124; citation_firstpage=3755; citation_lastpage=3764; citation_publication_date=1997; "><meta name="citation_reference" content="citation_title=The BioGRID database: A comprehensive biomedical resource of curated protein, genetic, and chemical interactions; citation_author=R. Oughtred; citation_author=J. Rust; citation_author=C. Chang; citation_author=B.-J. Breitkreutz; citation_author=C. Stark; citation_author=A. Willems; citation_author=L. Boucher; citation_author=G. Leung; citation_author=N. Kolas; citation_author=F. Zhang; citation_author=et al.; citation_journal_title=Protein Sci.; citation_volume=30; citation_firstpage=187; citation_lastpage=200; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=BRD4 (Bromodomain-Containing Protein 4) Interacts with GATA4 (GATA Binding Protein 4) to Govern Mitochondrial Homeostasis in Adult Cardiomyocytes; citation_author=A. Padmanabhan; citation_author=M. Alexanian; citation_author=R. Linares-Saldana; citation_author=B. González-Terán; citation_author=G. Andreoletti; citation_author=Y. Huang; citation_author=A.J. Connolly; citation_author=W. Kim; citation_author=A. Hsu; citation_author=Q. Duan; citation_author=et al.; citation_journal_title=Circulation; citation_volume=142; citation_firstpage=2338; citation_lastpage=2355; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=321 Congenital heart defects in CHARGE syndrome patients with CHD7 mutations; citation_author=P. Parisot; citation_author=F. Bajolle; citation_author=T. Attié-Bittach; citation_author=S. Thomas; citation_author=G. Goudefroye; citation_author=V. Abadie; citation_author=S. Lyonnet; citation_author=D. Bonnet; citation_journal_title=Archives of Cardiovascular Diseases Supplements; citation_volume=2; citation_firstpage=104; citation_lastpage=105; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=The PRIDE database and related tools and resources in 2019: improving support for quantification data; citation_author=Y. Perez-Riverol; citation_author=A. Csordas; citation_author=J. Bai; citation_author=M. Bernal-Llinares; citation_author=S. Hewapathirana; citation_author=D.J. Kundu; citation_author=A. Inuganti; citation_author=J. Griss; citation_author=G. Mayer; citation_author=M. Eisenacher; citation_author=et al.; citation_journal_title=Nucleic Acids Res.; citation_volume=47; citation_number=D1; citation_firstpage=D442; citation_lastpage=D450; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Genetic basis for congenital heart disease: revisited: A scientific statement from the american heart association; citation_author=M.E. Pierpont; citation_author=M. Brueckner; citation_author=W.K. Chung; citation_author=V. Garg; citation_author=R.V. Lacro; citation_author=A.L. McGuire; citation_author=S. Mital; citation_author=J.R. Priest; citation_author=W.T. Pu; citation_author=A. Roberts; citation_author=American Heart Association Council on Cardiovascular Disease in the Young; Council on Cardiovascular and Stroke Nursing; and Council on Genomic and Precision Medicine; citation_author=et al.; citation_journal_title=Circulation; citation_volume=138; citation_firstpage=e653; citation_lastpage=e711; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=De novo and rare variants at multiple loci support the oligogenic origins of atrioventricular septal heart defects; citation_author=J.R. Priest; citation_author=K. Osoegawa; citation_author=N. Mohammed; citation_author=V. Nanda; citation_author=R. Kundu; citation_author=K. Schultz; citation_author=E.J. Lammer; citation_author=S. Girirajan; citation_author=T. Scheetz; citation_author=D. Waggott; citation_author=et al.; citation_journal_title=PLoS Genet.; citation_volume=12; citation_firstpage=e1005963; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=R: A language and environment for statistical computing; citation_author=R Core Team; citation_publication_date=2020; citation_publisher=R Foundation for Statistical Computing, Vienna, Austria; "><meta name="citation_reference" content="citation_title=The Brd4 extraterminal domain confers transcription activation independent of pTEFb by recruiting multiple proteins, including NSD3; citation_author=S. Rahman; citation_author=M.E. Sowa; citation_author=M. Ottinger; citation_author=J.A. Smith; citation_author=Y. Shi; citation_author=J.W. Harper; citation_author=P.M. Howley; citation_journal_title=Mol. Cell. Biol.; citation_volume=31; citation_firstpage=2641; citation_lastpage=2652; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=Spectrum of heart disease associated with murine and human GATA4 mutation; citation_author=S.K. Rajagopal; citation_author=Q. Ma; citation_author=D. Obler; citation_author=J. Shen; citation_author=A. Manichaikul; citation_author=A. Tomita-Mitchell; citation_author=K. Boardman; citation_author=C. Briggs; citation_author=V. Garg; citation_author=D. Srivastava; citation_author=et al.; citation_journal_title=J.&nbsp;Mol. Cell. Cardiol.; citation_volume=43; citation_firstpage=677; citation_lastpage=685; citation_publication_date=2007; "><meta name="citation_reference" content="citation_title=deepTools2: a next generation web server for deep-sequencing data analysis; citation_author=F. Ramírez; citation_author=D.P. Ryan; citation_author=B. Grüning; citation_author=V. Bhardwaj; citation_author=F. Kilpert; citation_author=A.S. Richter; citation_author=S. Heyne; citation_author=F. Dündar; citation_author=T. Manke; citation_journal_title=Nucleic Acids Res.; citation_volume=44; citation_number=W1; citation_firstpage=W160-5; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=iRefIndex: a consolidated protein interaction database with provenance; citation_author=S. Razick; citation_author=G. Magklaras; citation_author=I.M. Donaldson; citation_journal_title=BMC Bioinformatics; citation_volume=9; citation_firstpage=405; citation_publication_date=2008; "><meta name="citation_reference" content="citation_title=CADD: predicting the deleteriousness of variants throughout the human genome; citation_author=P. Rentzsch; citation_author=D. Witten; citation_author=G.M. Cooper; citation_author=J. Shendure; citation_author=M. Kircher; citation_journal_title=Nucleic Acids Res.; citation_volume=47; citation_number=D1; citation_firstpage=D886; citation_lastpage=D894; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Genomic analyses implicate noncoding de novo variants in congenital heart disease; citation_author=F. Richter; citation_author=S.U. Morton; citation_author=S.W. Kim; citation_author=A. Kitaygorodsky; citation_author=L.K. Wasson; citation_author=K.M. Chen; citation_author=J. Zhou; citation_author=H. Qi; citation_author=N. Patel; citation_author=S.R. DePalma; citation_author=et al.; citation_journal_title=Nat. Genet.; citation_volume=52; citation_firstpage=769; citation_lastpage=777; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=A scaling normalization method for differential expression analysis of RNA-seq data; citation_author=M.D. Robinson; citation_author=A. Oshlack; citation_journal_title=Genome Biol.; citation_volume=11; citation_firstpage=R25; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=Moderated statistical tests for assessing differences in tag abundance; citation_author=M.D. Robinson; citation_author=G.K. Smyth; citation_journal_title=Bioinformatics; citation_volume=23; citation_firstpage=2881; citation_lastpage=2887; citation_publication_date=2007; "><meta name="citation_reference" content="citation_title=Small-sample estimation of negative binomial dispersion, with applications to SAGE data; citation_author=M.D. Robinson; citation_author=G.K. Smyth; citation_journal_title=Biostatistics; citation_volume=9; citation_firstpage=321; citation_lastpage=332; citation_publication_date=2008; "><meta name="citation_reference" content="citation_title=edgeR: a Bioconductor package for differential expression analysis of digital gene expression data; citation_author=M.D. Robinson; citation_author=D.J. McCarthy; citation_author=G.K. Smyth; citation_journal_title=Bioinformatics; citation_volume=26; citation_firstpage=139; citation_lastpage=140; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=A framework for the interpretation of de novo mutation in human disease; citation_author=K.E. Samocha; citation_author=E.B. Robinson; citation_author=S.J. Sanders; citation_author=C. Stevens; citation_author=A. Sabo; citation_author=L.M. McGrath; citation_author=J.A. Kosmicki; citation_author=K. Rehnström; citation_author=S. Mallick; citation_author=A. Kirby; citation_author=et al.; citation_journal_title=Nat. Genet.; citation_volume=46; citation_firstpage=944; citation_lastpage=950; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Protein and ligand preparation: parameters, protocols, and influence on virtual screening enrichments; citation_author=G.M. Sastry; citation_author=M. Adzhigirey; citation_author=T. Day; citation_author=R. Annabhimoju; citation_author=W. Sherman; citation_journal_title=J.&nbsp;Comput. Aided Mol. Des.; citation_volume=27; citation_firstpage=221; citation_lastpage=234; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=De novo variants in exomes of congenital heart disease patients identify risk genes and pathways; citation_author=C. Sevim Bayrak; citation_author=P. Zhang; citation_author=M. Tristani-Firouzi; citation_author=B.D. Gelb; citation_author=Y. Itan; citation_journal_title=Genome Med.; citation_volume=12; citation_firstpage=9; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Cytoscape: a software environment for integrated models of biomolecular interaction networks; citation_author=P. Shannon; citation_author=A. Markiel; citation_author=O. Ozier; citation_author=N.S. Baliga; citation_author=J.T. Wang; citation_author=D. Ramage; citation_author=N. Amin; citation_author=B. Schwikowski; citation_author=T. Ideker; citation_journal_title=Genome Res.; citation_volume=13; citation_firstpage=2498; citation_lastpage=2504; citation_publication_date=2003; "><meta name="citation_reference" content="citation_title=Distinct genetic architectures for syndromic and nonsyndromic congenital heart defects identified by exome sequencing; citation_author=A. Sifrim; citation_author=M.-P. Hitz; citation_author=A. Wilsdon; citation_author=J. Breckpot; citation_author=S.H.A. Turki; citation_author=B. Thienpont; citation_author=J. McRae; citation_author=T.W. Fitzgerald; citation_author=T. Singh; citation_author=G.J. Swaminathan; citation_author=INTERVAL Study; citation_author=UK10K Consortium; citation_author=Deciphering Developmental Disorders Study; citation_author=et al.; citation_journal_title=Nat. Genet.; citation_volume=48; citation_firstpage=1060; citation_lastpage=1065; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=A conditional approach to residual maximum likelihood estimation in generalized linear models; citation_author=G.K. Smyth; citation_publication_date=1996; citation_publisher=R. Stat. Soc. B.; "><meta name="citation_reference" content="citation_title=A clinically driven variant prioritization framework outperforms purely computational approaches for the diagnostic analysis of singleton WES data; citation_author=Z. Stark; citation_author=H. Dashnow; citation_author=S. Lunke; citation_author=T.Y. Tan; citation_author=A. Yeung; citation_author=S. Sadedin; citation_author=N. Thorne; citation_author=I. Macciocca; citation_author=C. Gaff; citation_author=A. Oshlack; citation_author=Melbourne Genomics Health Alliance; citation_author=et al.; citation_journal_title=Eur. J. Hum. Genet.; citation_volume=25; citation_firstpage=1268; citation_lastpage=1272; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=STRING v11: protein-protein association networks with increased coverage, supporting functional discovery in genome-wide experimental datasets; citation_author=D. Szklarczyk; citation_author=A.L. Gable; citation_author=D. Lyon; citation_author=A. Junge; citation_author=S. Wyder; citation_author=J. Huerta-Cepas; citation_author=M. Simonovic; citation_author=N.T. Doncheva; citation_author=J.H. Morris; citation_author=P. Bork; citation_author=et al.; citation_journal_title=Nucleic Acids Res.; citation_volume=47; citation_number=D1; citation_firstpage=D607; citation_lastpage=D613; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=SAINTq: Scoring protein-protein interactions in affinity purification - mass spectrometry experiments with fragment or peptide intensity data; citation_author=G. Teo; citation_author=H. Koh; citation_author=D. Fermin; citation_author=J.-P. Lambert; citation_author=J.D.R. Knight; citation_author=A.-C. Gingras; citation_author=H. Choi; citation_journal_title=Proteomics; citation_volume=16; citation_firstpage=2238; citation_lastpage=2245; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Patient-specific genomics and cross-species functional alalysis implicate LRP2 in hypoplastic left heart syndrome; citation_author=J.L. Theis; citation_author=G. Vogler; citation_author=M.A. Missinato; citation_author=X. Li; citation_author=T. Nielsen; citation_author=X.-X.I. Zeng; citation_author=A. Martinez-Fernandez; citation_author=S.M. Walls; citation_author=A. Kervadec; citation_author=J.A. Kezos; citation_author=et al.; citation_journal_title=eLife; citation_volume=9; citation_doi=10.7554/eLife.59554; citation_firstpage=359554; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Haploinsufficiency of TAB2 causes congenital heart defects in humans; citation_author=B. Thienpont; citation_author=L. Zhang; citation_author=A.V. Postma; citation_author=J. Breckpot; citation_author=L.-C. Tranchevent; citation_author=P. Van Loo; citation_author=K. Møllgård; citation_author=N. Tommerup; citation_author=I. Bache; citation_author=Z. Tümer; citation_author=et al.; citation_journal_title=Am. J. Hum. Genet.; citation_volume=86; citation_firstpage=839; citation_lastpage=849; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=Structure of the cytokine-like nuclear factor n-pac; citation_author=J. Tickle; citation_author=E.S. Pilka; citation_author=G. Bunkoczi; citation_author=G. Berridge; citation_author=C. Smee; citation_author=K.L. Kavanagh; citation_author=V. Hozjan; citation_author=F.H. Niesen; citation_author=E. Papagrigoriou; citation_author=A.C.W. Pike; citation_author=et al.; citation_publication_date=2007; "><meta name="citation_reference" content="citation_title=Distinct metabolic flow enables large-scale purification of mouse and human pluripotent stem cell-derived cardiomyocytes; citation_author=S. Tohyama; citation_author=F. Hattori; citation_author=M. Sano; citation_author=T. Hishiki; citation_author=Y. Nagahata; citation_author=T. Matsuura; citation_author=H. Hashimoto; citation_author=T. Suzuki; citation_author=H. Yamashita; citation_author=Y. Satoh; citation_author=et al.; citation_journal_title=Cell Stem Cell; citation_volume=12; citation_firstpage=127; citation_lastpage=137; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=GATA4 sequence variants in patients with congenital heart disease; citation_author=A. Tomita-Mitchell; citation_author=C.L. Maslen; citation_author=C.D. Morris; citation_author=V. Garg; citation_author=E. Goldmuntz; citation_journal_title=J.&nbsp;Med. Genet.; citation_volume=44; citation_firstpage=779; citation_lastpage=783; citation_publication_date=2007; "><meta name="citation_reference" content="citation_title=Proteomics. Tissue-based map of the human proteome; citation_author=M. Uhlén; citation_author=L. Fagerberg; citation_author=B.M. Hallström; citation_author=C. Lindskog; citation_author=P. Oksvold; citation_author=A. Mardinoglu; citation_author=Å. Sivertsson; citation_author=C. Kampf; citation_author=E. Sjöstedt; citation_author=A. Asplund; citation_author=et al.; citation_journal_title=Science; citation_volume=347; citation_firstpage=1260419; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=The cardiac TBX5 interactome reveals a chromatin remodeling network essential for cardiac septation; citation_author=L. Waldron; citation_author=J.D. Steimle; citation_author=T.M. Greco; citation_author=N.C. Gomez; citation_author=K.M. Dorr; citation_author=J. Kweon; citation_author=B. Temple; citation_author=X.H. Yang; citation_author=C.M. Wilczewski; citation_author=I.J. Davis; citation_author=et al.; citation_journal_title=Dev. Cell; citation_volume=36; citation_firstpage=262; citation_lastpage=275; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=ANNOVAR: functional annotation of genetic variants from high-throughput sequencing data; citation_author=K. Wang; citation_author=M. Li; citation_author=H. Hakonarson; citation_journal_title=Nucleic Acids Res.; citation_volume=38; citation_doi=10.1093/nar/gkq603; citation_firstpage=e164; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=RSeQC: quality control of RNA-seq experiments; citation_author=L. Wang; citation_author=S. Wang; citation_author=W. Li; citation_journal_title=Bioinformatics; citation_volume=28; citation_firstpage=2184; citation_lastpage=2185; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=GATA4 is essential for formation of the proepicardium and regulates cardiogenesis; citation_author=A.J. Watt; citation_author=M.A. Battle; citation_author=J. Li; citation_author=S.A. Duncan; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=101; citation_firstpage=12573; citation_lastpage=12578; citation_publication_date=2004; "><meta name="citation_reference" content="citation_title=A Bayesian framework for de novo mutation calling in parents-offspring trios; citation_author=Q. Wei; citation_author=X. Zhan; citation_author=X. Zhong; citation_author=Y. Liu; citation_author=Y. Han; citation_author=W. Chen; citation_author=B. Li; citation_journal_title=Bioinformatics; citation_volume=31; citation_firstpage=1375; citation_lastpage=1381; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=A threshold of GATA4 and GATA6 expression is required for cardiovascular development; citation_author=M. Xin; citation_author=C.A. Davis; citation_author=J.D. Molkentin; citation_author=C.-L. Lien; citation_author=S.A. Duncan; citation_author=J.A. Richardson; citation_author=E.N. Olson; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=103; citation_firstpage=11189; citation_lastpage=11194; citation_publication_date=2006; "><meta name="citation_reference" content="citation_title=Genome-wide localization of protein-DNA binding and histone modification by a Bayesian change-point method with ChIP-seq data; citation_author=H. Xing; citation_author=Y. Mo; citation_author=W. Liao; citation_author=M.Q. Zhang; citation_journal_title=PLoS Comput. Biol.; citation_volume=8; citation_firstpage=e1002613; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=Npac Is a Co-factor of Histone H3K36me3 and Regulates Transcriptional Elongation in Mouse Embryonic Stem Cells; citation_author=S. Yu; citation_author=J. Li; citation_author=G. Ji; citation_author=Z.L. Ng; citation_author=J. Siew; citation_author=W.N. Lo; citation_author=Y. Ye; citation_author=Y.Y. Chew; citation_author=Y.C. Long; citation_author=W. Zhang; citation_author=et al.; citation_journal_title=Genomics Proteomics Bioinformatics; citation_volume=3; citation_doi=10.1016/j.gpb.2020.08.004; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=Genetics and genomics of congenital heart disease; citation_author=S. Zaidi; citation_author=M. Brueckner; citation_journal_title=Circ. Res.; citation_volume=120; citation_firstpage=923; citation_lastpage=940; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=De novo mutations in histone-modifying genes in congenital heart disease; citation_author=S. Zaidi; citation_author=M. Choi; citation_author=H. Wakimoto; citation_author=L. Ma; citation_author=J. Jiang; citation_author=J.D. Overton; citation_author=A. Romano-Adesman; citation_author=R.D. Bjornson; citation_author=R.E. Breitbart; citation_author=K.K. Brown; citation_author=et al.; citation_journal_title=Nature; citation_volume=498; citation_firstpage=220; citation_lastpage=223; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=Structural characterization of a β-hydroxyacid dehydrogenase from Geobacter sulfurreducens and Geobacter metallireducens with succinic semialdehyde reductase activity; citation_author=Y. Zhang; citation_author=Y. Zheng; citation_author=L. Qin; citation_author=S. Wang; citation_author=G.W. Buchko; citation_author=R.M. Garavito; citation_journal_title=Biochimie; citation_volume=104; citation_firstpage=61; citation_lastpage=69; citation_publication_date=2014; "><meta name="citation_abstract" content="<h2>Summary</h2><p>Congenital heart disease (CHD) is present in 1% of live births, yet identification of causal mutations remains challenging. We hypothesized that genetic determinants for CHDs may lie in the protein interactomes of transcription factors whose mutations cause CHDs. Defining the interactomes of two transcription factors haplo-insufficient in CHD, GATA4 and TBX5, within human cardiac progenitors, and integrating the results with nearly 9,000 exomes from proband-parent trios revealed an enrichment of <i>de novo</i> missense variants associated with CHD within the interactomes. Scoring variants of interactome members based on residue, gene, and proband features identified likely CHD-causing genes, including the epigenetic reader <i>GLYR1</i>. GLYR1 and GATA4 widely co-occupied and co-activated cardiac developmental genes, and the identified <i>GLYR1</i> missense variant disrupted interaction with GATA4, impairing <i>in&nbsp;vitro</i> and <i>in&nbsp;vivo</i> function in mice. This integrative proteomic and genetic approach provides a framework for prioritizing and interrogating genetic variants in heart disease.</p>"><meta charset="UTF-8"><!--

    Pagebuild admin UI
-->



<script type="text/javascript" src="https://bam.nr-data.net/1/16e2192192?a=916957862&amp;sa=1&amp;v=1177.96a4d39&amp;t=Unnamed%20Transaction&amp;rst=10015&amp;ck=1&amp;ref=https://www.cell.com/cell/fulltext/S0092-8674(22)00079-4&amp;be=3720&amp;fe=9840&amp;dc=7010&amp;af=err,xhr,stn,ins,spa&amp;perf=%7B%22timing%22:%7B%22of%22:1749592693764,%22n%22:0,%22f%22:2987,%22dn%22:2988,%22dne%22:2988,%22c%22:2988,%22s%22:2988,%22ce%22:2988,%22rq%22:2988,%22rp%22:3704,%22rpe%22:4592,%22dl%22:3709,%22di%22:5120,%22ds%22:7010,%22de%22:7295,%22dc%22:9839,%22l%22:9839,%22le%22:9994%7D,%22navigation%22:%7B%7D%7D&amp;jsonp=NREUM.setToken"></script><script src="https://js-agent.newrelic.com/nr-spa-1177.min.js"></script><script type="text/javascript">
    ;window.NREUM||(NREUM={});NREUM.init={privacy:{cookies_enabled:true}};
    window.NREUM||(NREUM={}),__nr_require=function(t,e,n){function r(n){if(!e[n]){var o=e[n]={exports:{}};t[n][0].call(o.exports,function(e){var o=t[n][1][e];return r(o||e)},o,o.exports)}return e[n].exports}if("function"==typeof __nr_require)return __nr_require;for(var o=0;o<n.length;o++)r(n[o]);return r}({1:[function(t,e,n){function r(t){try{c.console&&console.log(t)}catch(e){}}var o,i=t("ee"),a=t(26),c={};try{o=localStorage.getItem("__nr_flags").split(","),console&&"function"==typeof console.log&&(c.console=!0,o.indexOf("dev")!==-1&&(c.dev=!0),o.indexOf("nr_dev")!==-1&&(c.nrDev=!0))}catch(s){}c.nrDev&&i.on("internal-error",function(t){r(t.stack)}),c.dev&&i.on("fn-err",function(t,e,n){r(n.stack)}),c.dev&&(r("NR AGENT IN DEVELOPMENT MODE"),r("flags: "+a(c,function(t,e){return t}).join(", ")))},{}],2:[function(t,e,n){function r(t,e,n,r,c){try{l?l-=1:o(c||new UncaughtException(t,e,n),!0)}catch(f){try{i("ierr",[f,s.now(),!0])}catch(d){}}return"function"==typeof u&&u.apply(this,a(arguments))}function UncaughtException(t,e,n){this.message=t||"Uncaught error with no additional information",this.sourceURL=e,this.line=n}function o(t,e){var n=e?null:s.now();i("err",[t,n])}var i=t("handle"),a=t(27),c=t("ee"),s=t("loader"),f=t("gos"),u=window.onerror,d=!1,p="nr@seenError",l=0;s.features.err=!0,t(1),window.onerror=r;try{throw new Error}catch(h){"stack"in h&&(t(13),t(12),"addEventListener"in window&&t(6),s.xhrWrappable&&t(14),d=!0)}c.on("fn-start",function(t,e,n){d&&(l+=1)}),c.on("fn-err",function(t,e,n){d&&!n[p]&&(f(n,p,function(){return!0}),this.thrown=!0,o(n))}),c.on("fn-end",function(){d&&!this.thrown&&l>0&&(l-=1)}),c.on("internal-error",function(t){i("ierr",[t,s.now(),!0])})},{}],3:[function(t,e,n){t("loader").features.ins=!0},{}],4:[function(t,e,n){function r(){_++,T=g.hash,this[u]=y.now()}function o(){_--,g.hash!==T&&i(0,!0);var t=y.now();this[h]=~~this[h]+t-this[u],this[d]=t}function i(t,e){E.emit("newURL",[""+g,e])}function a(t,e){t.on(e,function(){this[e]=y.now()})}var c="-start",s="-end",f="-body",u="fn"+c,d="fn"+s,p="cb"+c,l="cb"+s,h="jsTime",m="fetch",v="addEventListener",w=window,g=w.location,y=t("loader");if(w[v]&&y.xhrWrappable){var x=t(10),b=t(11),E=t(8),R=t(6),O=t(13),N=t(7),M=t(14),P=t(9),S=t("ee"),C=S.get("tracer");t(16),y.features.spa=!0;var T,_=0;S.on(u,r),S.on(p,r),S.on(d,o),S.on(l,o),S.buffer([u,d,"xhr-done","xhr-resolved"]),R.buffer([u]),O.buffer(["setTimeout"+s,"clearTimeout"+c,u]),M.buffer([u,"new-xhr","send-xhr"+c]),N.buffer([m+c,m+"-done",m+f+c,m+f+s]),E.buffer(["newURL"]),x.buffer([u]),b.buffer(["propagate",p,l,"executor-err","resolve"+c]),C.buffer([u,"no-"+u]),P.buffer(["new-jsonp","cb-start","jsonp-error","jsonp-end"]),a(M,"send-xhr"+c),a(S,"xhr-resolved"),a(S,"xhr-done"),a(N,m+c),a(N,m+"-done"),a(P,"new-jsonp"),a(P,"jsonp-end"),a(P,"cb-start"),E.on("pushState-end",i),E.on("replaceState-end",i),w[v]("hashchange",i,!0),w[v]("load",i,!0),w[v]("popstate",function(){i(0,_>1)},!0)}},{}],5:[function(t,e,n){function r(t){}if(window.performance&&window.performance.timing&&window.performance.getEntriesByType){var o=t("ee"),i=t("handle"),a=t(13),c=t(12),s="learResourceTimings",f="addEventListener",u="resourcetimingbufferfull",d="bstResource",p="resource",l="-start",h="-end",m="fn"+l,v="fn"+h,w="bstTimer",g="pushState",y=t("loader");y.features.stn=!0,t(8),"addEventListener"in window&&t(6);var x=NREUM.o.EV;o.on(m,function(t,e){var n=t[0];n instanceof x&&(this.bstStart=y.now())}),o.on(v,function(t,e){var n=t[0];n instanceof x&&i("bst",[n,e,this.bstStart,y.now()])}),a.on(m,function(t,e,n){this.bstStart=y.now(),this.bstType=n}),a.on(v,function(t,e){i(w,[e,this.bstStart,y.now(),this.bstType])}),c.on(m,function(){this.bstStart=y.now()}),c.on(v,function(t,e){i(w,[e,this.bstStart,y.now(),"requestAnimationFrame"])}),o.on(g+l,function(t){this.time=y.now(),this.startPath=location.pathname+location.hash}),o.on(g+h,function(t){i("bstHist",[location.pathname+location.hash,this.startPath,this.time])}),f in window.performance&&(window.performance["c"+s]?window.performance[f](u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["c"+s]()},!1):window.performance[f]("webkit"+u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["webkitC"+s]()},!1)),document[f]("scroll",r,{passive:!0}),document[f]("keypress",r,!1),document[f]("click",r,!1)}},{}],6:[function(t,e,n){function r(t){for(var e=t;e&&!e.hasOwnProperty(u);)e=Object.getPrototypeOf(e);e&&o(e)}function o(t){c.inPlace(t,[u,d],"-",i)}function i(t,e){return t[1]}var a=t("ee").get("events"),c=t("wrap-function")(a,!0),s=t("gos"),f=XMLHttpRequest,u="addEventListener",d="removeEventListener";e.exports=a,"getPrototypeOf"in Object?(r(document),r(window),r(f.prototype)):f.prototype.hasOwnProperty(u)&&(o(window),o(f.prototype)),a.on(u+"-start",function(t,e){var n=t[1],r=s(n,"nr@wrapped",function(){function t(){if("function"==typeof n.handleEvent)return n.handleEvent.apply(n,arguments)}var e={object:t,"function":n}[typeof n];return e?c(e,"fn-",null,e.name||"anonymous"):n});this.wrapped=t[1]=r}),a.on(d+"-start",function(t){t[1]=this.wrapped||t[1]})},{}],7:[function(t,e,n){function r(t,e,n){var r=t[e];"function"==typeof r&&(t[e]=function(){var t=i(arguments),e={};o.emit(n+"before-start",[t],e);var a;e[m]&&e[m].dt&&(a=e[m].dt);var c=r.apply(this,t);return o.emit(n+"start",[t,a],c),c.then(function(t){return o.emit(n+"end",[null,t],c),t},function(t){throw o.emit(n+"end",[t],c),t})})}var o=t("ee").get("fetch"),i=t(27),a=t(26);e.exports=o;var c=window,s="fetch-",f=s+"body-",u=["arrayBuffer","blob","json","text","formData"],d=c.Request,p=c.Response,l=c.fetch,h="prototype",m="nr@context";d&&p&&l&&(a(u,function(t,e){r(d[h],e,f),r(p[h],e,f)}),r(c,"fetch",s),o.on(s+"end",function(t,e){var n=this;if(e){var r=e.headers.get("content-length");null!==r&&(n.rxSize=r),o.emit(s+"done",[null,e],n)}else o.emit(s+"done",[t],n)}))},{}],8:[function(t,e,n){var r=t("ee").get("history"),o=t("wrap-function")(r);e.exports=r;var i=window.history&&window.history.constructor&&window.history.constructor.prototype,a=window.history;i&&i.pushState&&i.replaceState&&(a=i),o.inPlace(a,["pushState","replaceState"],"-")},{}],9:[function(t,e,n){function r(t){function e(){s.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}function n(){s.emit("jsonp-error",[],p),s.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}var r=t&&"string"==typeof t.nodeName&&"script"===t.nodeName.toLowerCase();if(r){var o="function"==typeof t.addEventListener;if(o){var a=i(t.src);if(a){var u=c(a),d="function"==typeof u.parent[u.key];if(d){var p={};f.inPlace(u.parent,[u.key],"cb-",p),t.addEventListener("load",e,!1),t.addEventListener("error",n,!1),s.emit("new-jsonp",[t.src],p)}}}}}function o(){return"addEventListener"in window}function i(t){var e=t.match(u);return e?e[1]:null}function a(t,e){var n=t.match(p),r=n[1],o=n[3];return o?a(o,e[r]):e[r]}function c(t){var e=t.match(d);return e&&e.length>=3?{key:e[2],parent:a(e[1],window)}:{key:t,parent:window}}var s=t("ee").get("jsonp"),f=t("wrap-function")(s);if(e.exports=s,o()){var u=/[?&](?:callback|cb)=([^&#]+)/,d=/(.*)\.([^.]+)/,p=/^(\w+)(\.|$)(.*)$/,l=["appendChild","insertBefore","replaceChild"];Node&&Node.prototype&&Node.prototype.appendChild?f.inPlace(Node.prototype,l,"dom-"):(f.inPlace(HTMLElement.prototype,l,"dom-"),f.inPlace(HTMLHeadElement.prototype,l,"dom-"),f.inPlace(HTMLBodyElement.prototype,l,"dom-")),s.on("dom-start",function(t){r(t[0])})}},{}],10:[function(t,e,n){var r=t("ee").get("mutation"),o=t("wrap-function")(r),i=NREUM.o.MO;e.exports=r,i&&(window.MutationObserver=function(t){return this instanceof i?new i(o(t,"fn-")):i.apply(this,arguments)},MutationObserver.prototype=i.prototype)},{}],11:[function(t,e,n){function r(t){var e=a.context(),n=c(t,"executor-",e),r=new f(n);return a.context(r).getCtx=function(){return e},a.emit("new-promise",[r,e],e),r}function o(t,e){return e}var i=t("wrap-function"),a=t("ee").get("promise"),c=i(a),s=t(26),f=NREUM.o.PR;e.exports=a,f&&(window.Promise=r,["all","race"].forEach(function(t){var e=f[t];f[t]=function(n){function r(t){return function(){a.emit("propagate",[null,!o],i),o=o||!t}}var o=!1;s(n,function(e,n){Promise.resolve(n).then(r("all"===t),r(!1))});var i=e.apply(f,arguments),c=f.resolve(i);return c}}),["resolve","reject"].forEach(function(t){var e=f[t];f[t]=function(t){var n=e.apply(f,arguments);return t!==n&&a.emit("propagate",[t,!0],n),n}}),f.prototype["catch"]=function(t){return this.then(null,t)},f.prototype=Object.create(f.prototype,{constructor:{value:r}}),s(Object.getOwnPropertyNames(f),function(t,e){try{r[e]=f[e]}catch(n){}}),a.on("executor-start",function(t){t[0]=c(t[0],"resolve-",this),t[1]=c(t[1],"resolve-",this)}),a.on("executor-err",function(t,e,n){t[1](n)}),c.inPlace(f.prototype,["then"],"then-",o),a.on("then-start",function(t,e){this.promise=e,t[0]=c(t[0],"cb-",this),t[1]=c(t[1],"cb-",this)}),a.on("then-end",function(t,e,n){this.nextPromise=n;var r=this.promise;a.emit("propagate",[r,!0],n)}),a.on("cb-end",function(t,e,n){a.emit("propagate",[n,!0],this.nextPromise)}),a.on("propagate",function(t,e,n){this.getCtx&&!e||(this.getCtx=function(){if(t instanceof Promise)var e=a.context(t);return e&&e.getCtx?e.getCtx():this})}),r.toString=function(){return""+f})},{}],12:[function(t,e,n){var r=t("ee").get("raf"),o=t("wrap-function")(r),i="equestAnimationFrame";e.exports=r,o.inPlace(window,["r"+i,"mozR"+i,"webkitR"+i,"msR"+i],"raf-"),r.on("raf-start",function(t){t[0]=o(t[0],"fn-")})},{}],13:[function(t,e,n){function r(t,e,n){t[0]=a(t[0],"fn-",null,n)}function o(t,e,n){this.method=n,this.timerDuration=isNaN(t[1])?0:+t[1],t[0]=a(t[0],"fn-",this,n)}var i=t("ee").get("timer"),a=t("wrap-function")(i),c="setTimeout",s="setInterval",f="clearTimeout",u="-start",d="-";e.exports=i,a.inPlace(window,[c,"setImmediate"],c+d),a.inPlace(window,[s],s+d),a.inPlace(window,[f,"clearImmediate"],f+d),i.on(s+u,r),i.on(c+u,o)},{}],14:[function(t,e,n){function r(t,e){d.inPlace(e,["onreadystatechange"],"fn-",c)}function o(){var t=this,e=u.context(t);t.readyState>3&&!e.resolved&&(e.resolved=!0,u.emit("xhr-resolved",[],t)),d.inPlace(t,g,"fn-",c)}function i(t){y.push(t),h&&(b?b.then(a):v?v(a):(E=-E,R.data=E))}function a(){for(var t=0;t<y.length;t++)r([],y[t]);y.length&&(y=[])}function c(t,e){return e}function s(t,e){for(var n in t)e[n]=t[n];return e}t(6);var f=t("ee"),u=f.get("xhr"),d=t("wrap-function")(u),p=NREUM.o,l=p.XHR,h=p.MO,m=p.PR,v=p.SI,w="readystatechange",g=["onload","onerror","onabort","onloadstart","onloadend","onprogress","ontimeout"],y=[];e.exports=u;var x=window.XMLHttpRequest=function(t){var e=new l(t);try{u.emit("new-xhr",[e],e),e.addEventListener(w,o,!1)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}return e};if(s(l,x),x.prototype=l.prototype,d.inPlace(x.prototype,["open","send"],"-xhr-",c),u.on("send-xhr-start",function(t,e){r(t,e),i(e)}),u.on("open-xhr-start",r),h){var b=m&&m.resolve();if(!v&&!m){var E=1,R=document.createTextNode(E);new h(a).observe(R,{characterData:!0})}}else f.on("fn-end",function(t){t[0]&&t[0].type===w||a()})},{}],15:[function(t,e,n){function r(t){if(!c(t))return null;var e=window.NREUM;if(!e.loader_config)return null;var n=(e.loader_config.accountID||"").toString()||null,r=(e.loader_config.agentID||"").toString()||null,f=(e.loader_config.trustKey||"").toString()||null;if(!n||!r)return null;var h=l.generateSpanId(),m=l.generateTraceId(),v=Date.now(),w={spanId:h,traceId:m,timestamp:v};return(t.sameOrigin||s(t)&&p())&&(w.traceContextParentHeader=o(h,m),w.traceContextStateHeader=i(h,v,n,r,f)),(t.sameOrigin&&!u()||!t.sameOrigin&&s(t)&&d())&&(w.newrelicHeader=a(h,m,v,n,r,f)),w}function o(t,e){return"00-"+e+"-"+t+"-01"}function i(t,e,n,r,o){var i=0,a="",c=1,s="",f="";return o+"@nr="+i+"-"+c+"-"+n+"-"+r+"-"+t+"-"+a+"-"+s+"-"+f+"-"+e}function a(t,e,n,r,o,i){var a="btoa"in window&&"function"==typeof window.btoa;if(!a)return null;var c={v:[0,1],d:{ty:"Browser",ac:r,ap:o,id:t,tr:e,ti:n}};return i&&r!==i&&(c.d.tk=i),btoa(JSON.stringify(c))}function c(t){return f()&&s(t)}function s(t){var e=!1,n={};if("init"in NREUM&&"distributed_tracing"in NREUM.init&&(n=NREUM.init.distributed_tracing),t.sameOrigin)e=!0;else if(n.allowed_origins instanceof Array)for(var r=0;r<n.allowed_origins.length;r++){var o=h(n.allowed_origins[r]);if(t.hostname===o.hostname&&t.protocol===o.protocol&&t.port===o.port){e=!0;break}}return e}function f(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.enabled}function u(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.exclude_newrelic_header}function d(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&NREUM.init.distributed_tracing.cors_use_newrelic_header!==!1}function p(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.cors_use_tracecontext_headers}var l=t(23),h=t(17);e.exports={generateTracePayload:r,shouldGenerateTrace:c}},{}],16:[function(t,e,n){function r(t){var e=this.params,n=this.metrics;if(!this.ended){this.ended=!0;for(var r=0;r<p;r++)t.removeEventListener(d[r],this.listener,!1);e.aborted||(n.duration=a.now()-this.startTime,this.loadCaptureCalled||4!==t.readyState?null==e.status&&(e.status=0):i(this,t),n.cbTime=this.cbTime,u.emit("xhr-done",[t],t),c("xhr",[e,n,this.startTime]))}}function o(t,e){var n=s(e),r=t.params;r.host=n.hostname+":"+n.port,r.pathname=n.pathname,t.parsedOrigin=s(e),t.sameOrigin=t.parsedOrigin.sameOrigin}function i(t,e){t.params.status=e.status;var n=v(e,t.lastSize);if(n&&(t.metrics.rxSize=n),t.sameOrigin){var r=e.getResponseHeader("X-NewRelic-App-Data");r&&(t.params.cat=r.split(", ").pop())}t.loadCaptureCalled=!0}var a=t("loader");if(a.xhrWrappable){var c=t("handle"),s=t(17),f=t(15).generateTracePayload,u=t("ee"),d=["load","error","abort","timeout"],p=d.length,l=t("id"),h=t(21),m=t(20),v=t(18),w=window.XMLHttpRequest;a.features.xhr=!0,t(14),t(7),u.on("new-xhr",function(t){var e=this;e.totalCbs=0,e.called=0,e.cbTime=0,e.end=r,e.ended=!1,e.xhrGuids={},e.lastSize=null,e.loadCaptureCalled=!1,t.addEventListener("load",function(n){i(e,t)},!1),h&&(h>34||h<10)||window.opera||t.addEventListener("progress",function(t){e.lastSize=t.loaded},!1)}),u.on("open-xhr-start",function(t){this.params={method:t[0]},o(this,t[1]),this.metrics={}}),u.on("open-xhr-end",function(t,e){"loader_config"in NREUM&&"xpid"in NREUM.loader_config&&this.sameOrigin&&e.setRequestHeader("X-NewRelic-ID",NREUM.loader_config.xpid);var n=f(this.parsedOrigin);if(n){var r=!1;n.newrelicHeader&&(e.setRequestHeader("newrelic",n.newrelicHeader),r=!0),n.traceContextParentHeader&&(e.setRequestHeader("traceparent",n.traceContextParentHeader),n.traceContextStateHeader&&e.setRequestHeader("tracestate",n.traceContextStateHeader),r=!0),r&&(this.dt=n)}}),u.on("send-xhr-start",function(t,e){var n=this.metrics,r=t[0],o=this;if(n&&r){var i=m(r);i&&(n.txSize=i)}this.startTime=a.now(),this.listener=function(t){try{"abort"!==t.type||o.loadCaptureCalled||(o.params.aborted=!0),("load"!==t.type||o.called===o.totalCbs&&(o.onloadCalled||"function"!=typeof e.onload))&&o.end(e)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}};for(var c=0;c<p;c++)e.addEventListener(d[c],this.listener,!1)}),u.on("xhr-cb-time",function(t,e,n){this.cbTime+=t,e?this.onloadCalled=!0:this.called+=1,this.called!==this.totalCbs||!this.onloadCalled&&"function"==typeof n.onload||this.end(n)}),u.on("xhr-load-added",function(t,e){var n=""+l(t)+!!e;this.xhrGuids&&!this.xhrGuids[n]&&(this.xhrGuids[n]=!0,this.totalCbs+=1)}),u.on("xhr-load-removed",function(t,e){var n=""+l(t)+!!e;this.xhrGuids&&this.xhrGuids[n]&&(delete this.xhrGuids[n],this.totalCbs-=1)}),u.on("addEventListener-end",function(t,e){e instanceof w&&"load"===t[0]&&u.emit("xhr-load-added",[t[1],t[2]],e)}),u.on("removeEventListener-end",function(t,e){e instanceof w&&"load"===t[0]&&u.emit("xhr-load-removed",[t[1],t[2]],e)}),u.on("fn-start",function(t,e,n){e instanceof w&&("onload"===n&&(this.onload=!0),("load"===(t[0]&&t[0].type)||this.onload)&&(this.xhrCbStart=a.now()))}),u.on("fn-end",function(t,e){this.xhrCbStart&&u.emit("xhr-cb-time",[a.now()-this.xhrCbStart,this.onload,e],e)}),u.on("fetch-before-start",function(t){function e(t,e){var n=!1;return e.newrelicHeader&&(t.set("newrelic",e.newrelicHeader),n=!0),e.traceContextParentHeader&&(t.set("traceparent",e.traceContextParentHeader),e.traceContextStateHeader&&t.set("tracestate",e.traceContextStateHeader),n=!0),n}var n,r=t[1]||{};"string"==typeof t[0]?n=t[0]:t[0]&&t[0].url&&(n=t[0].url),n&&(this.parsedOrigin=s(n),this.sameOrigin=this.parsedOrigin.sameOrigin);var o=f(this.parsedOrigin);if(o&&(o.newrelicHeader||o.traceContextParentHeader))if("string"==typeof t[0]){var i={};for(var a in r)i[a]=r[a];i.headers=new Headers(r.headers||{}),e(i.headers,o)&&(this.dt=o),t.length>1?t[1]=i:t.push(i)}else t[0]&&t[0].headers&&e(t[0].headers,o)&&(this.dt=o)})}},{}],17:[function(t,e,n){var r={};e.exports=function(t){if(t in r)return r[t];var e=document.createElement("a"),n=window.location,o={};e.href=t,o.port=e.port;var i=e.href.split("://");!o.port&&i[1]&&(o.port=i[1].split("/")[0].split("@").pop().split(":")[1]),o.port&&"0"!==o.port||(o.port="https"===i[0]?"443":"80"),o.hostname=e.hostname||n.hostname,o.pathname=e.pathname,o.protocol=i[0],"/"!==o.pathname.charAt(0)&&(o.pathname="/"+o.pathname);var a=!e.protocol||":"===e.protocol||e.protocol===n.protocol,c=e.hostname===document.domain&&e.port===n.port;return o.sameOrigin=a&&(!e.hostname||c),"/"===o.pathname&&(r[t]=o),o}},{}],18:[function(t,e,n){function r(t,e){var n=t.responseType;return"json"===n&&null!==e?e:"arraybuffer"===n||"blob"===n||"json"===n?o(t.response):"text"===n||""===n||void 0===n?o(t.responseText):void 0}var o=t(20);e.exports=r},{}],19:[function(t,e,n){function r(){}function o(t,e,n){return function(){return i(t,[f.now()].concat(c(arguments)),e?null:this,n),e?void 0:this}}var i=t("handle"),a=t(26),c=t(27),s=t("ee").get("tracer"),f=t("loader"),u=NREUM;"undefined"==typeof window.newrelic&&(newrelic=u);var d=["setPageViewName","setCustomAttribute","setErrorHandler","finished","addToTrace","inlineHit","addRelease"],p="api-",l=p+"ixn-";a(d,function(t,e){u[e]=o(p+e,!0,"api")}),u.addPageAction=o(p+"addPageAction",!0),u.setCurrentRouteName=o(p+"routeName",!0),e.exports=newrelic,u.interaction=function(){return(new r).get()};var h=r.prototype={createTracer:function(t,e){var n={},r=this,o="function"==typeof e;return i(l+"tracer",[f.now(),t,n],r),function(){if(s.emit((o?"":"no-")+"fn-start",[f.now(),r,o],n),o)try{return e.apply(this,arguments)}catch(t){throw s.emit("fn-err",[arguments,this,t],n),t}finally{s.emit("fn-end",[f.now()],n)}}}};a("actionText,setName,setAttribute,save,ignore,onEnd,getContext,end,get".split(","),function(t,e){h[e]=o(l+e)}),newrelic.noticeError=function(t,e){"string"==typeof t&&(t=new Error(t)),i("err",[t,f.now(),!1,e])}},{}],20:[function(t,e,n){e.exports=function(t){if("string"==typeof t&&t.length)return t.length;if("object"==typeof t){if("undefined"!=typeof ArrayBuffer&&t instanceof ArrayBuffer&&t.byteLength)return t.byteLength;if("undefined"!=typeof Blob&&t instanceof Blob&&t.size)return t.size;if(!("undefined"!=typeof FormData&&t instanceof FormData))try{return JSON.stringify(t).length}catch(e){return}}}},{}],21:[function(t,e,n){var r=0,o=navigator.userAgent.match(/Firefox[\/\s](\d+\.\d+)/);o&&(r=+o[1]),e.exports=r},{}],22:[function(t,e,n){function r(t,e){var n=t.getEntries();n.forEach(function(t){"first-paint"===t.name?d("timing",["fp",Math.floor(t.startTime)]):"first-contentful-paint"===t.name&&d("timing",["fcp",Math.floor(t.startTime)])})}function o(t,e){var n=t.getEntries();n.length>0&&d("lcp",[n[n.length-1]])}function i(t){t.getEntries().forEach(function(t){t.hadRecentInput||d("cls",[t])})}function a(t){if(t instanceof h&&!v){var e=Math.round(t.timeStamp),n={type:t.type};e<=p.now()?n.fid=p.now()-e:e>p.offset&&e<=Date.now()?(e-=p.offset,n.fid=p.now()-e):e=p.now(),v=!0,d("timing",["fi",e,n])}}function c(t){d("pageHide",[p.now(),t])}if(!("init"in NREUM&&"page_view_timing"in NREUM.init&&"enabled"in NREUM.init.page_view_timing&&NREUM.init.page_view_timing.enabled===!1)){var s,f,u,d=t("handle"),p=t("loader"),l=t(25),h=NREUM.o.EV;if("PerformanceObserver"in window&&"function"==typeof window.PerformanceObserver){s=new PerformanceObserver(r);try{s.observe({entryTypes:["paint"]})}catch(m){}f=new PerformanceObserver(o);try{f.observe({entryTypes:["largest-contentful-paint"]})}catch(m){}u=new PerformanceObserver(i);try{u.observe({type:"layout-shift",buffered:!0})}catch(m){}}if("addEventListener"in document){var v=!1,w=["click","keydown","mousedown","pointerdown","touchstart"];w.forEach(function(t){document.addEventListener(t,a,!1)})}l(c)}},{}],23:[function(t,e,n){function r(){function t(){return e?15&e[n++]:16*Math.random()|0}var e=null,n=0,r=window.crypto||window.msCrypto;r&&r.getRandomValues&&(e=r.getRandomValues(new Uint8Array(31)));for(var o,i="xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx",a="",c=0;c<i.length;c++)o=i[c],"x"===o?a+=t().toString(16):"y"===o?(o=3&t()|8,a+=o.toString(16)):a+=o;return a}function o(){return a(16)}function i(){return a(32)}function a(t){function e(){return n?15&n[r++]:16*Math.random()|0}var n=null,r=0,o=window.crypto||window.msCrypto;o&&o.getRandomValues&&Uint8Array&&(n=o.getRandomValues(new Uint8Array(31)));for(var i=[],a=0;a<t;a++)i.push(e().toString(16));return i.join("")}e.exports={generateUuid:r,generateSpanId:o,generateTraceId:i}},{}],24:[function(t,e,n){function r(t,e){if(!o)return!1;if(t!==o)return!1;if(!e)return!0;if(!i)return!1;for(var n=i.split("."),r=e.split("."),a=0;a<r.length;a++)if(r[a]!==n[a])return!1;return!0}var o=null,i=null,a=/Version\/(\S+)\s+Safari/;if(navigator.userAgent){var c=navigator.userAgent,s=c.match(a);s&&c.indexOf("Chrome")===-1&&c.indexOf("Chromium")===-1&&(o="Safari",i=s[1])}e.exports={agent:o,version:i,match:r}},{}],25:[function(t,e,n){function r(t){function e(){t(a&&document[a]?document[a]:document[o]?"hidden":"visible")}"addEventListener"in document&&i&&document.addEventListener(i,e,!1)}e.exports=r;var o,i,a;"undefined"!=typeof document.hidden?(o="hidden",i="visibilitychange",a="visibilityState"):"undefined"!=typeof document.msHidden?(o="msHidden",i="msvisibilitychange"):"undefined"!=typeof document.webkitHidden&&(o="webkitHidden",i="webkitvisibilitychange",a="webkitVisibilityState")},{}],26:[function(t,e,n){function r(t,e){var n=[],r="",i=0;for(r in t)o.call(t,r)&&(n[i]=e(r,t[r]),i+=1);return n}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],27:[function(t,e,n){function r(t,e,n){e||(e=0),"undefined"==typeof n&&(n=t?t.length:0);for(var r=-1,o=n-e||0,i=Array(o<0?0:o);++r<o;)i[r]=t[e+r];return i}e.exports=r},{}],28:[function(t,e,n){e.exports={exists:"undefined"!=typeof window.performance&&window.performance.timing&&"undefined"!=typeof window.performance.timing.navigationStart}},{}],ee:[function(t,e,n){function r(){}function o(t){function e(t){return t&&t instanceof r?t:t?s(t,c,i):i()}function n(n,r,o,i){if(!p.aborted||i){t&&t(n,r,o);for(var a=e(o),c=m(n),s=c.length,f=0;f<s;f++)c[f].apply(a,r);var d=u[y[n]];return d&&d.push([x,n,r,a]),a}}function l(t,e){g[t]=m(t).concat(e)}function h(t,e){var n=g[t];if(n)for(var r=0;r<n.length;r++)n[r]===e&&n.splice(r,1)}function m(t){return g[t]||[]}function v(t){return d[t]=d[t]||o(n)}function w(t,e){f(t,function(t,n){e=e||"feature",y[n]=e,e in u||(u[e]=[])})}var g={},y={},x={on:l,addEventListener:l,removeEventListener:h,emit:n,get:v,listeners:m,context:e,buffer:w,abort:a,aborted:!1};return x}function i(){return new r}function a(){(u.api||u.feature)&&(p.aborted=!0,u=p.backlog={})}var c="nr@context",s=t("gos"),f=t(26),u={},d={},p=e.exports=o();p.backlog=u},{}],gos:[function(t,e,n){function r(t,e,n){if(o.call(t,e))return t[e];var r=n();if(Object.defineProperty&&Object.keys)try{return Object.defineProperty(t,e,{value:r,writable:!0,enumerable:!1}),r}catch(i){}return t[e]=r,r}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],handle:[function(t,e,n){function r(t,e,n,r){o.buffer([t],r),o.emit(t,e,n)}var o=t("ee").get("handle");e.exports=r,r.ee=o},{}],id:[function(t,e,n){function r(t){var e=typeof t;return!t||"object"!==e&&"function"!==e?-1:t===window?0:a(t,i,function(){return o++})}var o=1,i="nr@id",a=t("gos");e.exports=r},{}],loader:[function(t,e,n){function r(){if(!E++){var t=b.info=NREUM.info,e=l.getElementsByTagName("script")[0];if(setTimeout(u.abort,3e4),!(t&&t.licenseKey&&t.applicationID&&e))return u.abort();f(y,function(e,n){t[e]||(t[e]=n)});var n=a();s("mark",["onload",n+b.offset],null,"api"),s("timing",["load",n]);var r=l.createElement("script");r.src="https://"+t.agent,e.parentNode.insertBefore(r,e)}}function o(){"complete"===l.readyState&&i()}function i(){s("mark",["domContent",a()+b.offset],null,"api")}function a(){return R.exists&&performance.now?Math.round(performance.now()):(c=Math.max((new Date).getTime(),c))-b.offset}var c=(new Date).getTime(),s=t("handle"),f=t(26),u=t("ee"),d=t(24),p=window,l=p.document,h="addEventListener",m="attachEvent",v=p.XMLHttpRequest,w=v&&v.prototype;NREUM.o={ST:setTimeout,SI:p.setImmediate,CT:clearTimeout,XHR:v,REQ:p.Request,EV:p.Event,PR:p.Promise,MO:p.MutationObserver};var g=""+location,y={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",agent:"js-agent.newrelic.com/nr-spa-1177.min.js"},x=v&&w&&w[h]&&!/CriOS/.test(navigator.userAgent),b=e.exports={offset:c,now:a,origin:g,features:{},xhrWrappable:x,userAgent:d};t(19),t(22),l[h]?(l[h]("DOMContentLoaded",i,!1),p[h]("load",r,!1)):(l[m]("onreadystatechange",o),p[m]("onload",r)),s("mark",["firstbyte",c],null,"api");var E=0,R=t(28)},{}],"wrap-function":[function(t,e,n){function r(t){return!(t&&t instanceof Function&&t.apply&&!t[a])}var o=t("ee"),i=t(27),a="nr@original",c=Object.prototype.hasOwnProperty,s=!1;e.exports=function(t,e){function n(t,e,n,o){function nrWrapper(){var r,a,c,s;try{a=this,r=i(arguments),c="function"==typeof n?n(r,a):n||{}}catch(f){p([f,"",[r,a,o],c])}u(e+"start",[r,a,o],c);try{return s=t.apply(a,r)}catch(d){throw u(e+"err",[r,a,d],c),d}finally{u(e+"end",[r,a,s],c)}}return r(t)?t:(e||(e=""),nrWrapper[a]=t,d(t,nrWrapper),nrWrapper)}function f(t,e,o,i){o||(o="");var a,c,s,f="-"===o.charAt(0);for(s=0;s<e.length;s++)c=e[s],a=t[c],r(a)||(t[c]=n(a,f?c+o:o,i,c))}function u(n,r,o){if(!s||e){var i=s;s=!0;try{t.emit(n,r,o,e)}catch(a){p([a,n,r,o])}s=i}}function d(t,e){if(Object.defineProperty&&Object.keys)try{var n=Object.keys(t);return n.forEach(function(n){Object.defineProperty(e,n,{get:function(){return t[n]},set:function(e){return t[n]=e,e}})}),e}catch(r){p([r])}for(var o in t)c.call(t,o)&&(e[o]=t[o]);return e}function p(e){try{t.emit("internal-error",e)}catch(n){}}return t||(t=o),n.inPlace=f,n.flag=a,n}},{}]},{},["loader",2,16,5,3,4]);
    ;NREUM.loader_config={accountID:"2095297",trustKey:"2038175",agentID:"916957862",licenseKey:"16e2192192",applicationID:"916957862"}
    ;NREUM.info={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",licenseKey:"16e2192192",applicationID:"916957862",sa:1}
</script>
    <script src="https://cdn.cookielaw.org/scripttemplates/otSDKStub.js" type="text/javascript" charset="UTF-8" data-domain-script="dada9834-04e2-4bcd-b5ba-cae44d498dfa"></script>
    <script type="text/javascript">
        function getCookie(name) {
            var value = "; " + document.cookie;
            var parts = value.split("; " + name + "=");
            if (parts.length == 2) {
                return true;
            }
        }
            function OptanonWrapper() {
                var optCookie = getCookie("OptanonAlertBoxClosed");
                var isActiveCategory = "false";
                var isFoundCookieBox = "false";
                if(isActiveCategory == 'false' && isFoundCookieBox == 'false' && optCookie) {
                    OneTrust.InsertScript('/pb-assets/adobe/prod/adobedtm-1616606406233.js', 'head', null, null, '2');
                }
            }
    </script>


<noscript>
    <style>.requiresJS {
        display: none
    }

    .dartAd {
        display: block !important
    }</style>
</noscript>
        <script type="text/javascript" src="/wro/o4ny~selected-legacy.js"></script>
        <link rel="stylesheet" type="text/css" href="/wro/o4ny~selected-legacy.css">
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Open+Sans">
        <script>window.MSInputMethodContext && document.documentMode && document.write('<script src="https://cdn.jsdelivr.net/gh/nuxodin/ie11CustomProperties@4.1.0/ie11CustomProperties.min.js"><\x2fscript>');</script><link rel="stylesheet" href="/products/marlin/cell/releasedAssets/css/build-b26f3342b722959c7931.css"><link rel="stylesheet" href="/products/marlin/cell/releasedAssets/css/print-b26f3342b722959c7931.css" media="print"><link rel="stylesheet" href="/products/marlin/cell/releasedAssets/css/themes/theme-cell.css">







        
        
        <script class="optanon-category-4" type="text/plain" async="" src="https://www.youtube.com/iframe_api"></script>
        
    





<!---->
<meta http-equiv="X-UA-Compatible" content="IE=edge">












<script src="https://cdn.cookielaw.org/scripttemplates/202411.2.0/otBannerSdk.js" async="" type="text/javascript"></script><style id="onetrust-style">#onetrust-banner-sdk .onetrust-vendors-list-handler{cursor:pointer;color:#1f96db;font-size:inherit;font-weight:bold;text-decoration:none;margin-left:5px}#onetrust-banner-sdk .onetrust-vendors-list-handler:hover{color:#1f96db}#onetrust-banner-sdk:focus{outline:2px solid #000;outline-offset:-2px}#onetrust-banner-sdk a:focus{outline:2px solid #000}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{outline-offset:1px}#onetrust-banner-sdk.ot-bnr-w-logo .ot-bnr-logo{height:64px;width:64px}#onetrust-banner-sdk .ot-tcf2-vendor-count.ot-text-bold{font-weight:bold}#onetrust-banner-sdk .ot-close-icon,#onetrust-pc-sdk .ot-close-icon,#ot-sync-ntfy .ot-close-icon{background-size:contain;background-repeat:no-repeat;background-position:center;height:12px;width:12px}#onetrust-banner-sdk .powered-by-logo,#onetrust-banner-sdk .ot-pc-footer-logo a,#onetrust-pc-sdk .powered-by-logo,#onetrust-pc-sdk .ot-pc-footer-logo a,#ot-sync-ntfy .powered-by-logo,#ot-sync-ntfy .ot-pc-footer-logo a{background-size:contain;background-repeat:no-repeat;background-position:center;height:25px;width:152px;display:block;text-decoration:none;font-size:.75em}#onetrust-banner-sdk .powered-by-logo:hover,#onetrust-banner-sdk .ot-pc-footer-logo a:hover,#onetrust-pc-sdk .powered-by-logo:hover,#onetrust-pc-sdk .ot-pc-footer-logo a:hover,#ot-sync-ntfy .powered-by-logo:hover,#ot-sync-ntfy .ot-pc-footer-logo a:hover{color:#565656}#onetrust-banner-sdk h3 *,#onetrust-banner-sdk h4 *,#onetrust-banner-sdk h6 *,#onetrust-banner-sdk button *,#onetrust-banner-sdk a[data-parent-id] *,#onetrust-pc-sdk h3 *,#onetrust-pc-sdk h4 *,#onetrust-pc-sdk h6 *,#onetrust-pc-sdk button *,#onetrust-pc-sdk a[data-parent-id] *,#ot-sync-ntfy h3 *,#ot-sync-ntfy h4 *,#ot-sync-ntfy h6 *,#ot-sync-ntfy button *,#ot-sync-ntfy a[data-parent-id] *{font-size:inherit;font-weight:inherit;color:inherit}#onetrust-banner-sdk .ot-hide,#onetrust-pc-sdk .ot-hide,#ot-sync-ntfy .ot-hide{display:none !important}#onetrust-banner-sdk button.ot-link-btn:hover,#onetrust-pc-sdk button.ot-link-btn:hover,#ot-sync-ntfy button.ot-link-btn:hover{text-decoration:underline;opacity:1}#onetrust-pc-sdk .ot-sdk-row .ot-sdk-column{padding:0}#onetrust-pc-sdk .ot-sdk-container{padding-right:0}#onetrust-pc-sdk .ot-sdk-row{flex-direction:initial;width:100%}#onetrust-pc-sdk [type=checkbox]:checked,#onetrust-pc-sdk [type=checkbox]:not(:checked){pointer-events:initial}#onetrust-pc-sdk [type=checkbox]:disabled+label::before,#onetrust-pc-sdk [type=checkbox]:disabled+label:after,#onetrust-pc-sdk [type=checkbox]:disabled+label{pointer-events:none;opacity:.8}#onetrust-pc-sdk #vendor-list-content{transform:translate3d(0, 0, 0)}#onetrust-pc-sdk li input[type=checkbox]{z-index:1}#onetrust-pc-sdk li .ot-checkbox label{z-index:2}#onetrust-pc-sdk li .ot-checkbox input[type=checkbox]{height:auto;width:auto}#onetrust-pc-sdk li .host-title a,#onetrust-pc-sdk li .ot-host-name a,#onetrust-pc-sdk li .accordion-text,#onetrust-pc-sdk li .ot-acc-txt{z-index:2;position:relative}#onetrust-pc-sdk input{margin:3px .1ex}#onetrust-pc-sdk .pc-logo,#onetrust-pc-sdk .ot-pc-logo{height:60px;width:180px;background-position:center;background-size:contain;background-repeat:no-repeat;display:inline-flex;justify-content:center;align-items:center}#onetrust-pc-sdk .pc-logo img,#onetrust-pc-sdk .ot-pc-logo img{max-height:100%;max-width:100%}#onetrust-pc-sdk .screen-reader-only,#onetrust-pc-sdk .ot-scrn-rdr,.ot-sdk-cookie-policy .screen-reader-only,.ot-sdk-cookie-policy .ot-scrn-rdr{border:0;clip:rect(0 0 0 0);height:1px;margin:-1px;overflow:hidden;padding:0;position:absolute;width:1px}#onetrust-pc-sdk.ot-fade-in,.onetrust-pc-dark-filter.ot-fade-in,#onetrust-banner-sdk.ot-fade-in{animation-name:onetrust-fade-in;animation-duration:400ms;animation-timing-function:ease-in-out}#onetrust-pc-sdk.ot-hide{display:none !important}.onetrust-pc-dark-filter.ot-hide{display:none !important}#ot-sdk-btn.ot-sdk-show-settings,#ot-sdk-btn.optanon-show-settings{color:#fff;background-color:#468254;height:auto;white-space:normal;word-wrap:break-word;padding:.8em 2em;font-size:.8em;line-height:1.2;cursor:pointer;-moz-transition:.1s ease;-o-transition:.1s ease;-webkit-transition:1s ease;transition:.1s ease}#ot-sdk-btn.ot-sdk-show-settings:hover,#ot-sdk-btn.optanon-show-settings:hover{color:#fff;background-color:#2c6415}#ot-sdk-btn.ot-sdk-show-settings:active,#ot-sdk-btn.optanon-show-settings:active{color:#fff;background-color:#2c6415;border:1px solid rgba(162,192,169,.5)}.onetrust-pc-dark-filter{background:rgba(0,0,0,.5);z-index:2147483646;width:100%;height:100%;overflow:hidden;position:fixed;top:0;bottom:0;left:0}@keyframes onetrust-fade-in{0%{opacity:0}100%{opacity:1}}.ot-cookie-label{text-decoration:underline}@media only screen and (min-width: 426px)and (max-width: 896px)and (orientation: landscape){#onetrust-pc-sdk p{font-size:.75em}}#onetrust-banner-sdk .banner-option-input:focus+label{outline:1px solid #000;outline-style:auto}.category-vendors-list-handler+a:focus,.category-vendors-list-handler+a:focus-visible{outline:2px solid #000}#onetrust-pc-sdk .ot-userid-title{margin-top:10px}#onetrust-pc-sdk .ot-userid-title>span,#onetrust-pc-sdk .ot-userid-timestamp>span{font-weight:700}#onetrust-pc-sdk .ot-userid-desc{font-style:italic}#onetrust-pc-sdk .ot-host-desc a{pointer-events:initial}#onetrust-pc-sdk .ot-ven-hdr>p a{position:relative;z-index:2;pointer-events:initial}#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info a,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info a{margin-right:auto}#onetrust-pc-sdk .ot-pc-footer-logo img{width:136px;height:16px}#onetrust-pc-sdk .ot-pur-vdr-count{font-weight:400;font-size:.7rem;padding-top:3px;display:block}#onetrust-banner-sdk .ot-optout-signal,#onetrust-pc-sdk .ot-optout-signal{border:1px solid #32ae88;border-radius:3px;padding:5px;margin-bottom:10px;background-color:#f9fffa;font-size:.85rem;line-height:2}#onetrust-banner-sdk .ot-optout-signal .ot-optout-icon,#onetrust-pc-sdk .ot-optout-signal .ot-optout-icon{display:inline;margin-right:5px}#onetrust-banner-sdk .ot-optout-signal svg,#onetrust-pc-sdk .ot-optout-signal svg{height:20px;width:30px;transform:scale(0.5)}#onetrust-banner-sdk .ot-optout-signal svg path,#onetrust-pc-sdk .ot-optout-signal svg path{fill:#32ae88}#onetrust-consent-sdk .ot-general-modal{overflow:hidden;position:fixed;margin:0 auto;top:50%;left:50%;width:40%;padding:1.5rem;max-width:575px;min-width:575px;z-index:2147483647;border-radius:2.5px;transform:translate(-50%, -50%)}#onetrust-consent-sdk .ot-signature-health-group{margin-top:1rem;padding-left:1.25rem;padding-right:1.25rem;margin-bottom:.625rem;width:calc(100% - 2.5rem)}#onetrust-consent-sdk .ot-signature-health-group .ot-signature-health-form{gap:.5rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-health-form{width:70%;gap:.35rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-input{height:38px;padding:6px 10px;background-color:#fff;border:1px solid #d1d1d1;border-radius:4px;box-shadow:none;box-sizing:border-box}#onetrust-consent-sdk .ot-signature-health .ot-signature-subtitle{font-size:1.125rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-group-title{font-size:1.25rem;font-weight:bold}#onetrust-consent-sdk .ot-signature-health,#onetrust-consent-sdk .ot-signature-health-group{display:flex;flex-direction:column;gap:1rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-cont,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-cont{display:flex;flex-direction:column;gap:.25rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-paragraph,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-paragraph{margin:0;line-height:20px;font-size:max(14px,.875rem)}#onetrust-consent-sdk .ot-signature-health .ot-health-signature-error,#onetrust-consent-sdk .ot-signature-health-group .ot-health-signature-error{color:#4d4d4d;font-size:min(12px,.75rem)}#onetrust-consent-sdk .ot-signature-health .ot-signature-buttons-cont,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-buttons-cont{margin-top:max(.75rem,2%);gap:1rem;display:flex;justify-content:flex-end}#onetrust-consent-sdk .ot-signature-health .ot-signature-button,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-button{flex:1;height:auto;color:#fff;cursor:pointer;line-height:1.2;min-width:125px;font-weight:600;font-size:.813em;border-radius:2px;padding:12px 10px;white-space:normal;word-wrap:break-word;word-break:break-word;background-color:#68b631;border:2px solid #68b631}#onetrust-consent-sdk .ot-signature-health .ot-signature-button.reject,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-button.reject{background-color:#fff}#onetrust-consent-sdk .ot-input-field-cont{display:flex;flex-direction:column;gap:.5rem}#onetrust-consent-sdk .ot-input-field-cont .ot-signature-input{width:65%}#onetrust-consent-sdk .ot-signature-health-form{display:flex;flex-direction:column}#onetrust-consent-sdk .ot-signature-health-form .ot-signature-label{margin-bottom:0;line-height:20px;font-size:max(14px,.875rem)}@media only screen and (max-width: 600px){#onetrust-consent-sdk .ot-general-modal{min-width:100%}#onetrust-consent-sdk .ot-signature-health .ot-signature-health-form{width:100%}#onetrust-consent-sdk .ot-input-field-cont .ot-signature-input{width:100%}}#onetrust-banner-sdk,#onetrust-pc-sdk,#ot-sdk-cookie-policy,#ot-sync-ntfy{font-size:16px}#onetrust-banner-sdk *,#onetrust-banner-sdk ::after,#onetrust-banner-sdk ::before,#onetrust-pc-sdk *,#onetrust-pc-sdk ::after,#onetrust-pc-sdk ::before,#ot-sdk-cookie-policy *,#ot-sdk-cookie-policy ::after,#ot-sdk-cookie-policy ::before,#ot-sync-ntfy *,#ot-sync-ntfy ::after,#ot-sync-ntfy ::before{-webkit-box-sizing:content-box;-moz-box-sizing:content-box;box-sizing:content-box}#onetrust-banner-sdk div,#onetrust-banner-sdk span,#onetrust-banner-sdk h1,#onetrust-banner-sdk h2,#onetrust-banner-sdk h3,#onetrust-banner-sdk h4,#onetrust-banner-sdk h5,#onetrust-banner-sdk h6,#onetrust-banner-sdk p,#onetrust-banner-sdk img,#onetrust-banner-sdk svg,#onetrust-banner-sdk button,#onetrust-banner-sdk section,#onetrust-banner-sdk a,#onetrust-banner-sdk label,#onetrust-banner-sdk input,#onetrust-banner-sdk ul,#onetrust-banner-sdk li,#onetrust-banner-sdk nav,#onetrust-banner-sdk table,#onetrust-banner-sdk thead,#onetrust-banner-sdk tr,#onetrust-banner-sdk td,#onetrust-banner-sdk tbody,#onetrust-banner-sdk .ot-main-content,#onetrust-banner-sdk .ot-toggle,#onetrust-banner-sdk #ot-content,#onetrust-banner-sdk #ot-pc-content,#onetrust-banner-sdk .checkbox,#onetrust-pc-sdk div,#onetrust-pc-sdk span,#onetrust-pc-sdk h1,#onetrust-pc-sdk h2,#onetrust-pc-sdk h3,#onetrust-pc-sdk h4,#onetrust-pc-sdk h5,#onetrust-pc-sdk h6,#onetrust-pc-sdk p,#onetrust-pc-sdk img,#onetrust-pc-sdk svg,#onetrust-pc-sdk button,#onetrust-pc-sdk section,#onetrust-pc-sdk a,#onetrust-pc-sdk label,#onetrust-pc-sdk input,#onetrust-pc-sdk ul,#onetrust-pc-sdk li,#onetrust-pc-sdk nav,#onetrust-pc-sdk table,#onetrust-pc-sdk thead,#onetrust-pc-sdk tr,#onetrust-pc-sdk td,#onetrust-pc-sdk tbody,#onetrust-pc-sdk .ot-main-content,#onetrust-pc-sdk .ot-toggle,#onetrust-pc-sdk #ot-content,#onetrust-pc-sdk #ot-pc-content,#onetrust-pc-sdk .checkbox,#ot-sdk-cookie-policy div,#ot-sdk-cookie-policy span,#ot-sdk-cookie-policy h1,#ot-sdk-cookie-policy h2,#ot-sdk-cookie-policy h3,#ot-sdk-cookie-policy h4,#ot-sdk-cookie-policy h5,#ot-sdk-cookie-policy h6,#ot-sdk-cookie-policy p,#ot-sdk-cookie-policy img,#ot-sdk-cookie-policy svg,#ot-sdk-cookie-policy button,#ot-sdk-cookie-policy section,#ot-sdk-cookie-policy a,#ot-sdk-cookie-policy label,#ot-sdk-cookie-policy input,#ot-sdk-cookie-policy ul,#ot-sdk-cookie-policy li,#ot-sdk-cookie-policy nav,#ot-sdk-cookie-policy table,#ot-sdk-cookie-policy thead,#ot-sdk-cookie-policy tr,#ot-sdk-cookie-policy td,#ot-sdk-cookie-policy tbody,#ot-sdk-cookie-policy .ot-main-content,#ot-sdk-cookie-policy .ot-toggle,#ot-sdk-cookie-policy #ot-content,#ot-sdk-cookie-policy #ot-pc-content,#ot-sdk-cookie-policy .checkbox,#ot-sync-ntfy div,#ot-sync-ntfy span,#ot-sync-ntfy h1,#ot-sync-ntfy h2,#ot-sync-ntfy h3,#ot-sync-ntfy h4,#ot-sync-ntfy h5,#ot-sync-ntfy h6,#ot-sync-ntfy p,#ot-sync-ntfy img,#ot-sync-ntfy svg,#ot-sync-ntfy button,#ot-sync-ntfy section,#ot-sync-ntfy a,#ot-sync-ntfy label,#ot-sync-ntfy input,#ot-sync-ntfy ul,#ot-sync-ntfy li,#ot-sync-ntfy nav,#ot-sync-ntfy table,#ot-sync-ntfy thead,#ot-sync-ntfy tr,#ot-sync-ntfy td,#ot-sync-ntfy tbody,#ot-sync-ntfy .ot-main-content,#ot-sync-ntfy .ot-toggle,#ot-sync-ntfy #ot-content,#ot-sync-ntfy #ot-pc-content,#ot-sync-ntfy .checkbox{font-family:inherit;font-weight:normal;-webkit-font-smoothing:auto;letter-spacing:normal;line-height:normal;padding:0;margin:0;height:auto;min-height:0;max-height:none;width:auto;min-width:0;max-width:none;border-radius:0;border:none;clear:none;float:none;position:static;bottom:auto;left:auto;right:auto;top:auto;text-align:left;text-decoration:none;text-indent:0;text-shadow:none;text-transform:none;white-space:normal;background:none;overflow:visible;vertical-align:baseline;visibility:visible;z-index:auto;box-shadow:none}#onetrust-banner-sdk label:before,#onetrust-banner-sdk label:after,#onetrust-banner-sdk .checkbox:after,#onetrust-banner-sdk .checkbox:before,#onetrust-pc-sdk label:before,#onetrust-pc-sdk label:after,#onetrust-pc-sdk .checkbox:after,#onetrust-pc-sdk .checkbox:before,#ot-sdk-cookie-policy label:before,#ot-sdk-cookie-policy label:after,#ot-sdk-cookie-policy .checkbox:after,#ot-sdk-cookie-policy .checkbox:before,#ot-sync-ntfy label:before,#ot-sync-ntfy label:after,#ot-sync-ntfy .checkbox:after,#ot-sync-ntfy .checkbox:before{content:"";content:none}#onetrust-banner-sdk .ot-sdk-container,#onetrust-pc-sdk .ot-sdk-container,#ot-sdk-cookie-policy .ot-sdk-container{position:relative;width:100%;max-width:100%;margin:0 auto;padding:0 20px;box-sizing:border-box}#onetrust-banner-sdk .ot-sdk-column,#onetrust-banner-sdk .ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-column,#onetrust-pc-sdk .ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-column,#ot-sdk-cookie-policy .ot-sdk-columns{width:100%;float:left;box-sizing:border-box;padding:0;display:initial}@media(min-width: 400px){#onetrust-banner-sdk .ot-sdk-container,#onetrust-pc-sdk .ot-sdk-container,#ot-sdk-cookie-policy .ot-sdk-container{width:90%;padding:0}}@media(min-width: 550px){#onetrust-banner-sdk .ot-sdk-container,#onetrust-pc-sdk .ot-sdk-container,#ot-sdk-cookie-policy .ot-sdk-container{width:100%}#onetrust-banner-sdk .ot-sdk-column,#onetrust-banner-sdk .ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-column,#onetrust-pc-sdk .ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-column,#ot-sdk-cookie-policy .ot-sdk-columns{margin-left:4%}#onetrust-banner-sdk .ot-sdk-column:first-child,#onetrust-banner-sdk .ot-sdk-columns:first-child,#onetrust-pc-sdk .ot-sdk-column:first-child,#onetrust-pc-sdk .ot-sdk-columns:first-child,#ot-sdk-cookie-policy .ot-sdk-column:first-child,#ot-sdk-cookie-policy .ot-sdk-columns:first-child{margin-left:0}#onetrust-banner-sdk .ot-sdk-two.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-two.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-two.ot-sdk-columns{width:13.3333333333%}#onetrust-banner-sdk .ot-sdk-three.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-three.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-three.ot-sdk-columns{width:22%}#onetrust-banner-sdk .ot-sdk-four.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-four.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-four.ot-sdk-columns{width:30.6666666667%}#onetrust-banner-sdk .ot-sdk-eight.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-eight.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-eight.ot-sdk-columns{width:65.3333333333%}#onetrust-banner-sdk .ot-sdk-nine.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-nine.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-nine.ot-sdk-columns{width:74%}#onetrust-banner-sdk .ot-sdk-ten.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-ten.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-ten.ot-sdk-columns{width:82.6666666667%}#onetrust-banner-sdk .ot-sdk-eleven.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-eleven.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-eleven.ot-sdk-columns{width:91.3333333333%}#onetrust-banner-sdk .ot-sdk-twelve.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-twelve.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-twelve.ot-sdk-columns{width:100%;margin-left:0}}#onetrust-banner-sdk h1,#onetrust-banner-sdk h2,#onetrust-banner-sdk h3,#onetrust-banner-sdk h4,#onetrust-banner-sdk h5,#onetrust-banner-sdk h6,#onetrust-pc-sdk h1,#onetrust-pc-sdk h2,#onetrust-pc-sdk h3,#onetrust-pc-sdk h4,#onetrust-pc-sdk h5,#onetrust-pc-sdk h6,#ot-sdk-cookie-policy h1,#ot-sdk-cookie-policy h2,#ot-sdk-cookie-policy h3,#ot-sdk-cookie-policy h4,#ot-sdk-cookie-policy h5,#ot-sdk-cookie-policy h6{margin-top:0;font-weight:600;font-family:inherit}#onetrust-banner-sdk h1,#onetrust-pc-sdk h1,#ot-sdk-cookie-policy h1{font-size:1.5rem;line-height:1.2}#onetrust-banner-sdk h2,#onetrust-pc-sdk h2,#ot-sdk-cookie-policy h2{font-size:1.5rem;line-height:1.25}#onetrust-banner-sdk h3,#onetrust-pc-sdk h3,#ot-sdk-cookie-policy h3{font-size:1.5rem;line-height:1.3}#onetrust-banner-sdk h4,#onetrust-pc-sdk h4,#ot-sdk-cookie-policy h4{font-size:1.5rem;line-height:1.35}#onetrust-banner-sdk h5,#onetrust-pc-sdk h5,#ot-sdk-cookie-policy h5{font-size:1.5rem;line-height:1.5}#onetrust-banner-sdk h6,#onetrust-pc-sdk h6,#ot-sdk-cookie-policy h6{font-size:1.5rem;line-height:1.6}@media(min-width: 550px){#onetrust-banner-sdk h1,#onetrust-pc-sdk h1,#ot-sdk-cookie-policy h1{font-size:1.5rem}#onetrust-banner-sdk h2,#onetrust-pc-sdk h2,#ot-sdk-cookie-policy h2{font-size:1.5rem}#onetrust-banner-sdk h3,#onetrust-pc-sdk h3,#ot-sdk-cookie-policy h3{font-size:1.5rem}#onetrust-banner-sdk h4,#onetrust-pc-sdk h4,#ot-sdk-cookie-policy h4{font-size:1.5rem}#onetrust-banner-sdk h5,#onetrust-pc-sdk h5,#ot-sdk-cookie-policy h5{font-size:1.5rem}#onetrust-banner-sdk h6,#onetrust-pc-sdk h6,#ot-sdk-cookie-policy h6{font-size:1.5rem}}#onetrust-banner-sdk p,#onetrust-pc-sdk p,#ot-sdk-cookie-policy p{margin:0 0 1em 0;font-family:inherit;line-height:normal}#onetrust-banner-sdk a,#onetrust-pc-sdk a,#ot-sdk-cookie-policy a{color:#565656;text-decoration:underline}#onetrust-banner-sdk a:hover,#onetrust-pc-sdk a:hover,#ot-sdk-cookie-policy a:hover{color:#565656;text-decoration:none}#onetrust-banner-sdk .ot-sdk-button,#onetrust-banner-sdk button,#onetrust-pc-sdk .ot-sdk-button,#onetrust-pc-sdk button,#ot-sdk-cookie-policy .ot-sdk-button,#ot-sdk-cookie-policy button{margin-bottom:1rem;font-family:inherit}#onetrust-banner-sdk .ot-sdk-button,#onetrust-banner-sdk button,#onetrust-pc-sdk .ot-sdk-button,#onetrust-pc-sdk button,#ot-sdk-cookie-policy .ot-sdk-button,#ot-sdk-cookie-policy button{display:inline-block;height:38px;padding:0 30px;color:#555;text-align:center;font-size:.9em;font-weight:400;line-height:38px;letter-spacing:.01em;text-decoration:none;white-space:nowrap;background-color:rgba(0,0,0,0);border-radius:2px;border:1px solid #bbb;cursor:pointer;box-sizing:border-box}#onetrust-banner-sdk .ot-sdk-button:hover,#onetrust-banner-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):hover,#onetrust-banner-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):focus,#onetrust-pc-sdk .ot-sdk-button:hover,#onetrust-pc-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):hover,#onetrust-pc-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):focus,#ot-sdk-cookie-policy .ot-sdk-button:hover,#ot-sdk-cookie-policy :not(.ot-leg-btn-container)>button:not(.ot-link-btn):hover,#ot-sdk-cookie-policy :not(.ot-leg-btn-container)>button:not(.ot-link-btn):focus{color:#333;border-color:#888;opacity:.9}#onetrust-banner-sdk .ot-sdk-button:focus,#onetrust-banner-sdk :not(.ot-leg-btn-container)>button:focus,#onetrust-pc-sdk .ot-sdk-button:focus,#onetrust-pc-sdk :not(.ot-leg-btn-container)>button:focus,#ot-sdk-cookie-policy .ot-sdk-button:focus,#ot-sdk-cookie-policy :not(.ot-leg-btn-container)>button:focus{outline:2px solid #000}#onetrust-banner-sdk .ot-sdk-button.ot-sdk-button-primary,#onetrust-banner-sdk button.ot-sdk-button-primary,#onetrust-banner-sdk input[type=submit].ot-sdk-button-primary,#onetrust-banner-sdk input[type=reset].ot-sdk-button-primary,#onetrust-banner-sdk input[type=button].ot-sdk-button-primary,#onetrust-pc-sdk .ot-sdk-button.ot-sdk-button-primary,#onetrust-pc-sdk button.ot-sdk-button-primary,#onetrust-pc-sdk input[type=submit].ot-sdk-button-primary,#onetrust-pc-sdk input[type=reset].ot-sdk-button-primary,#onetrust-pc-sdk input[type=button].ot-sdk-button-primary,#ot-sdk-cookie-policy .ot-sdk-button.ot-sdk-button-primary,#ot-sdk-cookie-policy button.ot-sdk-button-primary,#ot-sdk-cookie-policy input[type=submit].ot-sdk-button-primary,#ot-sdk-cookie-policy input[type=reset].ot-sdk-button-primary,#ot-sdk-cookie-policy input[type=button].ot-sdk-button-primary{color:#fff;background-color:#33c3f0;border-color:#33c3f0}#onetrust-banner-sdk .ot-sdk-button.ot-sdk-button-primary:hover,#onetrust-banner-sdk button.ot-sdk-button-primary:hover,#onetrust-banner-sdk input[type=submit].ot-sdk-button-primary:hover,#onetrust-banner-sdk input[type=reset].ot-sdk-button-primary:hover,#onetrust-banner-sdk input[type=button].ot-sdk-button-primary:hover,#onetrust-banner-sdk .ot-sdk-button.ot-sdk-button-primary:focus,#onetrust-banner-sdk button.ot-sdk-button-primary:focus,#onetrust-banner-sdk input[type=submit].ot-sdk-button-primary:focus,#onetrust-banner-sdk input[type=reset].ot-sdk-button-primary:focus,#onetrust-banner-sdk input[type=button].ot-sdk-button-primary:focus,#onetrust-pc-sdk .ot-sdk-button.ot-sdk-button-primary:hover,#onetrust-pc-sdk button.ot-sdk-button-primary:hover,#onetrust-pc-sdk input[type=submit].ot-sdk-button-primary:hover,#onetrust-pc-sdk input[type=reset].ot-sdk-button-primary:hover,#onetrust-pc-sdk input[type=button].ot-sdk-button-primary:hover,#onetrust-pc-sdk .ot-sdk-button.ot-sdk-button-primary:focus,#onetrust-pc-sdk button.ot-sdk-button-primary:focus,#onetrust-pc-sdk input[type=submit].ot-sdk-button-primary:focus,#onetrust-pc-sdk input[type=reset].ot-sdk-button-primary:focus,#onetrust-pc-sdk input[type=button].ot-sdk-button-primary:focus,#ot-sdk-cookie-policy .ot-sdk-button.ot-sdk-button-primary:hover,#ot-sdk-cookie-policy button.ot-sdk-button-primary:hover,#ot-sdk-cookie-policy input[type=submit].ot-sdk-button-primary:hover,#ot-sdk-cookie-policy input[type=reset].ot-sdk-button-primary:hover,#ot-sdk-cookie-policy input[type=button].ot-sdk-button-primary:hover,#ot-sdk-cookie-policy .ot-sdk-button.ot-sdk-button-primary:focus,#ot-sdk-cookie-policy button.ot-sdk-button-primary:focus,#ot-sdk-cookie-policy input[type=submit].ot-sdk-button-primary:focus,#ot-sdk-cookie-policy input[type=reset].ot-sdk-button-primary:focus,#ot-sdk-cookie-policy input[type=button].ot-sdk-button-primary:focus{color:#fff;background-color:#1eaedb;border-color:#1eaedb}#onetrust-banner-sdk input[type=text],#onetrust-pc-sdk input[type=text],#ot-sdk-cookie-policy input[type=text]{height:38px;padding:6px 10px;background-color:#fff;border:1px solid #d1d1d1;border-radius:4px;box-shadow:none;box-sizing:border-box}#onetrust-banner-sdk input[type=text],#onetrust-pc-sdk input[type=text],#ot-sdk-cookie-policy input[type=text]{-webkit-appearance:none;-moz-appearance:none;appearance:none}#onetrust-banner-sdk input[type=text]:focus,#onetrust-pc-sdk input[type=text]:focus,#ot-sdk-cookie-policy input[type=text]:focus{border:1px solid #000;outline:0}#onetrust-banner-sdk label,#onetrust-pc-sdk label,#ot-sdk-cookie-policy label{display:block;margin-bottom:.5rem;font-weight:600}#onetrust-banner-sdk input[type=checkbox],#onetrust-pc-sdk input[type=checkbox],#ot-sdk-cookie-policy input[type=checkbox]{display:inline}#onetrust-banner-sdk ul,#onetrust-pc-sdk ul,#ot-sdk-cookie-policy ul{list-style:circle inside}#onetrust-banner-sdk ul,#onetrust-pc-sdk ul,#ot-sdk-cookie-policy ul{padding-left:0;margin-top:0}#onetrust-banner-sdk ul ul,#onetrust-pc-sdk ul ul,#ot-sdk-cookie-policy ul ul{margin:1.5rem 0 1.5rem 3rem;font-size:90%}#onetrust-banner-sdk li,#onetrust-pc-sdk li,#ot-sdk-cookie-policy li{margin-bottom:1rem}#onetrust-banner-sdk th,#onetrust-banner-sdk td,#onetrust-pc-sdk th,#onetrust-pc-sdk td,#ot-sdk-cookie-policy th,#ot-sdk-cookie-policy td{padding:12px 15px;text-align:left;border-bottom:1px solid #e1e1e1}#onetrust-banner-sdk button,#onetrust-pc-sdk button,#ot-sdk-cookie-policy button{margin-bottom:1rem;font-family:inherit}#onetrust-banner-sdk .ot-sdk-container:after,#onetrust-banner-sdk .ot-sdk-row:after,#onetrust-pc-sdk .ot-sdk-container:after,#onetrust-pc-sdk .ot-sdk-row:after,#ot-sdk-cookie-policy .ot-sdk-container:after,#ot-sdk-cookie-policy .ot-sdk-row:after{content:"";display:table;clear:both}#onetrust-banner-sdk .ot-sdk-row,#onetrust-pc-sdk .ot-sdk-row,#ot-sdk-cookie-policy .ot-sdk-row{margin:0;max-width:none;display:block}#onetrust-banner-sdk{box-shadow:0 0 18px rgba(0,0,0,.2)}#onetrust-banner-sdk.otFlat{position:fixed;z-index:2147483645;bottom:0;right:0;left:0;background-color:#fff;max-height:90%;overflow-x:hidden;overflow-y:auto}#onetrust-banner-sdk.otFlat.top{top:0px;bottom:auto}#onetrust-banner-sdk.otRelFont{font-size:1rem}#onetrust-banner-sdk>.ot-sdk-container{overflow:hidden}#onetrust-banner-sdk::-webkit-scrollbar{width:11px}#onetrust-banner-sdk::-webkit-scrollbar-thumb{border-radius:10px;background:#c1c1c1}#onetrust-banner-sdk{scrollbar-arrow-color:#c1c1c1;scrollbar-darkshadow-color:#c1c1c1;scrollbar-face-color:#c1c1c1;scrollbar-shadow-color:#c1c1c1}#onetrust-banner-sdk #onetrust-policy{margin:1.25em 0 .625em 2em;overflow:hidden}#onetrust-banner-sdk #onetrust-policy .ot-gv-list-handler{float:left;font-size:.82em;padding:0;margin-bottom:0;border:0;line-height:normal;height:auto;width:auto}#onetrust-banner-sdk #onetrust-policy-title{font-size:1.2em;line-height:1.3;margin-bottom:10px}#onetrust-banner-sdk #onetrust-policy-text{clear:both;text-align:left;font-size:.88em;line-height:1.4}#onetrust-banner-sdk #onetrust-policy-text *{font-size:inherit;line-height:inherit}#onetrust-banner-sdk #onetrust-policy-text a{font-weight:bold;margin-left:5px}#onetrust-banner-sdk #onetrust-policy-title,#onetrust-banner-sdk #onetrust-policy-text{color:dimgray;float:left}#onetrust-banner-sdk #onetrust-button-group-parent{min-height:1px;text-align:center}#onetrust-banner-sdk #onetrust-button-group{display:inline-block}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{background-color:#68b631;color:#fff;border-color:#68b631;margin-right:1em;min-width:125px;height:auto;white-space:normal;word-break:break-word;word-wrap:break-word;padding:12px 10px;line-height:1.2;font-size:.813em;font-weight:600}#onetrust-banner-sdk #onetrust-pc-btn-handler.cookie-setting-link{background-color:#fff;border:none;color:#68b631;text-decoration:underline;padding-left:0;padding-right:0}#onetrust-banner-sdk .onetrust-close-btn-ui{width:44px;height:44px;background-size:12px;border:none;position:relative;margin:auto;padding:0}#onetrust-banner-sdk .banner_logo{display:none}#onetrust-banner-sdk.ot-bnr-w-logo .ot-bnr-logo{position:absolute;top:50%;transform:translateY(-50%);left:0px}#onetrust-banner-sdk.ot-bnr-w-logo #onetrust-policy{margin-left:65px}#onetrust-banner-sdk .ot-b-addl-desc{clear:both;float:left;display:block}#onetrust-banner-sdk #banner-options{float:left;display:table;margin-right:0;margin-left:1em;width:calc(100% - 1em)}#onetrust-banner-sdk .banner-option-input{cursor:pointer;width:auto;height:auto;border:none;padding:0;padding-right:3px;margin:0 0 10px;font-size:.82em;line-height:1.4}#onetrust-banner-sdk .banner-option-input *{pointer-events:none;font-size:inherit;line-height:inherit}#onetrust-banner-sdk .banner-option-input[aria-expanded=true]~.banner-option-details{display:block;height:auto}#onetrust-banner-sdk .banner-option-input[aria-expanded=true] .ot-arrow-container{transform:rotate(90deg)}#onetrust-banner-sdk .banner-option{margin-bottom:12px;margin-left:0;border:none;float:left;padding:0}#onetrust-banner-sdk .banner-option:first-child{padding-left:2px}#onetrust-banner-sdk .banner-option:not(:first-child){padding:0;border:none}#onetrust-banner-sdk .banner-option-header{cursor:pointer;display:inline-block}#onetrust-banner-sdk .banner-option-header :first-child{color:dimgray;font-weight:bold;float:left}#onetrust-banner-sdk .banner-option-header .ot-arrow-container{display:inline-block;border-top:6px solid rgba(0,0,0,0);border-bottom:6px solid rgba(0,0,0,0);border-left:6px solid dimgray;margin-left:10px;vertical-align:middle}#onetrust-banner-sdk .banner-option-details{display:none;font-size:.83em;line-height:1.5;padding:10px 0px 5px 10px;margin-right:10px;height:0px}#onetrust-banner-sdk .banner-option-details *{font-size:inherit;line-height:inherit;color:dimgray}#onetrust-banner-sdk .ot-arrow-container,#onetrust-banner-sdk .banner-option-details{transition:all 300ms ease-in 0s;-webkit-transition:all 300ms ease-in 0s;-moz-transition:all 300ms ease-in 0s;-o-transition:all 300ms ease-in 0s}#onetrust-banner-sdk .ot-dpd-container{float:left}#onetrust-banner-sdk .ot-dpd-title{margin-bottom:10px}#onetrust-banner-sdk .ot-dpd-title,#onetrust-banner-sdk .ot-dpd-desc{font-size:.88em;line-height:1.4;color:dimgray}#onetrust-banner-sdk .ot-dpd-title *,#onetrust-banner-sdk .ot-dpd-desc *{font-size:inherit;line-height:inherit}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-text *{margin-bottom:0}#onetrust-banner-sdk.ot-iab-2 .onetrust-vendors-list-handler{display:block;margin-left:0;margin-top:5px;clear:both;margin-bottom:0;padding:0;border:0;height:auto;width:auto}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group button{display:block}#onetrust-banner-sdk.ot-close-btn-link{padding-top:25px}#onetrust-banner-sdk.ot-close-btn-link #onetrust-close-btn-container{top:15px;transform:none;right:15px}#onetrust-banner-sdk.ot-close-btn-link #onetrust-close-btn-container button{padding:0;white-space:pre-wrap;border:none;height:auto;line-height:1.5;text-decoration:underline;font-size:.69em}#onetrust-banner-sdk #onetrust-policy-text,#onetrust-banner-sdk .ot-dpd-desc,#onetrust-banner-sdk .ot-b-addl-desc{font-size:.813em;line-height:1.5}#onetrust-banner-sdk .ot-dpd-desc{margin-bottom:10px}#onetrust-banner-sdk .ot-dpd-desc>.ot-b-addl-desc{margin-top:10px;margin-bottom:10px;font-size:1em}@media only screen and (max-width: 425px){#onetrust-banner-sdk #onetrust-close-btn-container{position:absolute;top:6px;right:2px}#onetrust-banner-sdk #onetrust-policy{margin-left:0;margin-top:3em}#onetrust-banner-sdk #onetrust-button-group{display:block}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{width:100%}#onetrust-banner-sdk .onetrust-close-btn-ui{top:auto;transform:none}#onetrust-banner-sdk #onetrust-policy-title{display:inline;float:none}#onetrust-banner-sdk #banner-options{margin:0;padding:0;width:100%}}@media only screen and (min-width: 426px)and (max-width: 896px){#onetrust-banner-sdk #onetrust-close-btn-container{position:absolute;top:0;right:0}#onetrust-banner-sdk #onetrust-policy{margin-left:1em;margin-right:1em}#onetrust-banner-sdk .onetrust-close-btn-ui{top:10px;right:10px}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{width:95%}#onetrust-banner-sdk.ot-iab-2 #onetrust-group-container{width:100%}#onetrust-banner-sdk.ot-bnr-w-logo #onetrust-button-group-parent{padding-left:50px}#onetrust-banner-sdk #onetrust-button-group-parent{width:100%;position:relative;margin-left:0}#onetrust-banner-sdk #onetrust-button-group button{display:inline-block}#onetrust-banner-sdk #onetrust-button-group{margin-right:0;text-align:center}#onetrust-banner-sdk .has-reject-all-button #onetrust-pc-btn-handler{float:left}#onetrust-banner-sdk .has-reject-all-button #onetrust-reject-all-handler,#onetrust-banner-sdk .has-reject-all-button #onetrust-accept-btn-handler{float:right}#onetrust-banner-sdk .has-reject-all-button #onetrust-button-group{width:calc(100% - 2em);margin-right:0}#onetrust-banner-sdk .has-reject-all-button #onetrust-pc-btn-handler.cookie-setting-link{padding-left:0px;text-align:left}#onetrust-banner-sdk.ot-buttons-fw .ot-sdk-three button{width:100%;text-align:center}#onetrust-banner-sdk.ot-buttons-fw #onetrust-button-group-parent button{float:none}#onetrust-banner-sdk.ot-buttons-fw #onetrust-pc-btn-handler.cookie-setting-link{text-align:center}}@media only screen and (min-width: 550px){#onetrust-banner-sdk .banner-option:not(:first-child){border-left:1px solid #d8d8d8;padding-left:25px}}@media only screen and (min-width: 425px)and (max-width: 550px){#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group,#onetrust-banner-sdk.ot-iab-2 #onetrust-policy,#onetrust-banner-sdk.ot-iab-2 .banner-option{width:100%}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-accept-btn-handler,#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-reject-all-handler,#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-pc-btn-handler{width:100%}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-accept-btn-handler,#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-reject-all-handler{float:left}}@media only screen and (min-width: 769px){#onetrust-banner-sdk #onetrust-button-group{margin-right:30%}#onetrust-banner-sdk #banner-options{margin-left:2em;margin-right:5em;margin-bottom:1.25em;width:calc(100% - 7em)}}@media only screen and (min-width: 897px)and (max-width: 1023px){#onetrust-banner-sdk.vertical-align-content #onetrust-button-group-parent{position:absolute;top:50%;left:75%;transform:translateY(-50%)}#onetrust-banner-sdk #onetrust-close-btn-container{top:50%;margin:auto;transform:translate(-50%, -50%);position:absolute;padding:0;right:0}#onetrust-banner-sdk #onetrust-close-btn-container button{position:relative;margin:0;right:-22px;top:2px}}@media only screen and (min-width: 1024px){#onetrust-banner-sdk #onetrust-close-btn-container{top:50%;margin:auto;transform:translate(-50%, -50%);position:absolute;right:0}#onetrust-banner-sdk #onetrust-close-btn-container button{right:-12px}#onetrust-banner-sdk #onetrust-policy{margin-left:2em}#onetrust-banner-sdk.vertical-align-content #onetrust-button-group-parent{position:absolute;top:50%;left:60%;transform:translateY(-50%)}#onetrust-banner-sdk .ot-optout-signal{width:50%}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-title{width:50%}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-text,#onetrust-banner-sdk.ot-iab-2 :not(.ot-dpd-desc)>.ot-b-addl-desc{margin-bottom:1em;width:50%;border-right:1px solid #d8d8d8;padding-right:1rem}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-text{margin-bottom:0;padding-bottom:1em}#onetrust-banner-sdk.ot-iab-2 :not(.ot-dpd-desc)>.ot-b-addl-desc{margin-bottom:0;padding-bottom:1em}#onetrust-banner-sdk.ot-iab-2 .ot-dpd-container{width:45%;padding-left:1rem;display:inline-block;float:none}#onetrust-banner-sdk.ot-iab-2 .ot-dpd-title{line-height:1.7}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group-parent{left:auto;right:4%;margin-left:0}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group button{display:block}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{margin:auto;width:30%}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{width:60%}#onetrust-banner-sdk #onetrust-button-group{margin-right:auto}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{margin-top:1em}}@media only screen and (min-width: 890px){#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group-parent{padding-left:3%;padding-right:4%;margin-left:0}#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group{margin-right:0;margin-top:1.25em;width:100%}#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button{width:100%;margin-bottom:5px;margin-top:5px}#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button:last-of-type{margin-bottom:20px}}@media only screen and (min-width: 1280px){#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{width:55%}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{width:44%;padding-left:2%;padding-right:2%}#onetrust-banner-sdk:not(.ot-iab-2).vertical-align-content #onetrust-button-group-parent{position:absolute;left:55%}}
        #onetrust-consent-sdk #onetrust-banner-sdk {background-color: #FFF;}
            #onetrust-consent-sdk #onetrust-policy-title,
                    #onetrust-consent-sdk #onetrust-policy-text,
                    #onetrust-consent-sdk .ot-b-addl-desc,
                    #onetrust-consent-sdk .ot-dpd-desc,
                    #onetrust-consent-sdk .ot-dpd-title,
                    #onetrust-consent-sdk #onetrust-policy-text *:not(.onetrust-vendors-list-handler),
                    #onetrust-consent-sdk .ot-dpd-desc *:not(.onetrust-vendors-list-handler),
                    #onetrust-consent-sdk #onetrust-banner-sdk #banner-options *,
                    #onetrust-banner-sdk .ot-cat-header,
                    #onetrust-banner-sdk .ot-optout-signal
                    {
                        color: #2E2E2E;
                    }
            #onetrust-consent-sdk #onetrust-banner-sdk .banner-option-details {
                    background-color: #E9E9E9;}
             #onetrust-consent-sdk #onetrust-banner-sdk a[href],
                    #onetrust-consent-sdk #onetrust-banner-sdk a[href] font,
                    #onetrust-consent-sdk #onetrust-banner-sdk .ot-link-btn
                        {
                            color: #007398;
                        }#onetrust-consent-sdk #onetrust-accept-btn-handler,
                         #onetrust-banner-sdk #onetrust-reject-all-handler {
                            background-color: #007398;border-color: #007398;
                color: #FFF;
            }
            #onetrust-consent-sdk #onetrust-banner-sdk *:focus,
            #onetrust-consent-sdk #onetrust-banner-sdk:focus {
               outline-color: #000000;
               outline-width: 1px;
            }
            #onetrust-consent-sdk #onetrust-pc-btn-handler,
            #onetrust-consent-sdk #onetrust-pc-btn-handler.cookie-setting-link {
                color: #6CC04A; border-color: #6CC04A;
                background-color:
                #FFF;
            }/*! v6.10.0 2021-01-18 */
div#onetrust-consent-sdk #onetrust-banner-sdk{border-top:2px solid #d0d3d4!important;box-shadow:none;padding:24px}div#onetrust-consent-sdk button{border-radius:0!important;box-shadow:none!important;box-sizing:border-box!important;font-size:20px!important;font-weight:400!important;letter-spacing:0!important;max-width:none!important;white-space:nowrap!important}div#onetrust-consent-sdk button:not(.ot-link-btn){background-color:#007dbc!important;border:2px solid #007dbc!important;color:#fff!important;height:48px!important;padding:0 1em!important;width:auto!important}div#onetrust-consent-sdk button:hover{background-color:#fff!important;border-color:#949494!important;color:#2e2e2e!important}div#onetrust-consent-sdk button.ot-link-btn{color:#007dbc!important;font-size:16px!important;text-decoration:underline}div#onetrust-consent-sdk button.ot-link-btn:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-consent-sdk a,div#onetrust-pc-sdk a{color:#007dbc!important;text-decoration:underline!important}div#onetrust-consent-sdk a,div#onetrust-consent-sdk button,div#onetrust-consent-sdk p:hover{opacity:1!important}div#onetrust-consent-sdk a:focus,div#onetrust-consent-sdk button:focus,div#onetrust-consent-sdk input:focus{outline:2px solid #949494!important;outline-offset:1px!important}div#onetrust-banner-sdk .ot-sdk-container{padding:0;width:auto}div#onetrust-banner-sdk .ot-sdk-row{align-items:flex-start;box-sizing:border-box;display:flex;flex-direction:column;justify-content:space-between;margin:auto;max-width:1152px}div#onetrust-banner-sdk .ot-sdk-row:after{display:none}div#onetrust-banner-sdk #onetrust-group-container,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-group-container,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{flex-grow:1;width:auto}div#onetrust-banner-sdk #onetrust-policy,div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy{margin:0;overflow:visible}div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy-text,div#onetrust-consent-sdk #onetrust-policy-text{font-size:16px;line-height:24px;max-width:44em;margin:0}div#onetrust-consent-sdk #onetrust-policy-text a[href]{font-weight:400;margin-left:8px}div#onetrust-banner-sdk #onetrust-button-group-parent{flex:0 0 auto;margin:32px 0 0;width:100%}div#onetrust-banner-sdk #onetrust-button-group{display:flex;flex-direction:row;flex-wrap:wrap;justify-content:flex-end;margin:-8px}div#onetrust-banner-sdk .banner-actions-container{display:flex;flex:1 0 auto}div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button:last-of-type,div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto;margin:8px;width:auto}div#onetrust-consent-sdk #onetrust-pc-btn-handler{background-color:#fff!important;color:inherit!important}div#onetrust-banner-sdk #onetrust-close-btn-container{display:none}@media only screen and (min-width:556px){div#onetrust-consent-sdk #onetrust-banner-sdk{padding:40px}div#onetrust-banner-sdk #onetrust-policy{margin:0 40px 0 0}div#onetrust-banner-sdk .ot-sdk-row{align-items:center;flex-direction:row}div#onetrust-banner-sdk #onetrust-button-group-parent,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-button-group-parent,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{margin:0;padding:0;width:auto}div#onetrust-banner-sdk #onetrust-button-group,div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group{align-items:stretch;flex-direction:column-reverse;margin:0}div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto}}@media only screen and (min-width:768px){div#onetrust-banner-sdk #onetrust-policy{margin:0 48px 0 0}div#onetrust-consent-sdk #onetrust-banner-sdk{padding:48px}}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-size:16px;line-height:24px}div#onetrust-consent-sdk #onetrust-pc-sdk p,div#onetrust-pc-sdk #ot-pc-desc,div#onetrust-pc-sdk .category-host-list-handler,div#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{font-size:16px;font-weight:400;line-height:24px}div#onetrust-consent-sdk a:hover,div#onetrust-pc-sdk a:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-pc-sdk{border-radius:0;bottom:0;height:auto;left:0;margin:auto;max-width:100%;overflow:hidden;right:0;top:0;width:512px;max-height:800px}div#onetrust-pc-sdk .ot-pc-header{display:none}div#onetrust-pc-sdk #ot-pc-content{overscroll-behavior:contain;padding:0 12px 0 24px;margin:16px 4px 0 0;top:0;right:16px;left:0;width:auto}div#onetrust-pc-sdk #ot-category-title,div#onetrust-pc-sdk #ot-pc-title{font-size:24px;font-weight:400;line-height:32px;margin:0 0 16px}div#onetrust-pc-sdk #ot-pc-desc{padding:0}div#onetrust-pc-sdk #ot-pc-desc a{display:inline}div#onetrust-pc-sdk #accept-recommended-btn-handler{display:none!important}div#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:2px solid #007dbc;outline-offset:-1px;transition:none}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item{border-width:0 0 2px}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item:first-of-type{border-width:2px 0}div#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{padding:8px 0;width:100%}div#onetrust-pc-sdk .ot-plus-minus{transform:translateY(2px)}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item>button{background:0 0!important;border:0!important;height:44px!important;max-width:none!important;width:calc(100% - 48px)!important}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-weight:700}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr{padding:0}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item .ot-acc-grpdesc{padding:0;width:100%}div#onetrust-pc-sdk .ot-acc-grpcntr .ot-subgrp-cntr{border:0;padding:0}div#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp{margin:0}div#onetrust-pc-sdk .ot-always-active-group .ot-cat-header{width:calc(100% - 160px)}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{width:calc(100% - 88px)}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active{color:inherit;font-size:12px;font-weight:400;line-height:1.5;padding-right:48px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:before{border-radius:12px;position:absolute;right:0;top:0;content:'';background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:after{border-radius:50%;position:absolute;right:5px;top:4px;content:'';background-color:#007dbc;height:12px;width:12px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active,div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl{right:2px}div#onetrust-pc-sdk .ot-switch{display:block;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob,div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob{background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob:before{background-color:#737373;height:8px;left:4px;top:4px;width:8px}div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob:before{background-color:#007dbc;height:12px;left:0;top:2px;width:12px}div#onetrust-pc-sdk .ot-tgl input:focus+.ot-switch .ot-switch-nob{box-shadow:0 0;outline:2px solid #007dbc!important;outline-offset:1px;transition:none}div#onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-grpcntr.ot-acc-txt{background-color:transparent;padding-left:3px}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr,div#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr{overflow:visible;width:100%}div#onetrust-pc-sdk .ot-pc-footer{border-top:0 solid}div#onetrust-pc-sdk .ot-btn-container{padding-top:24px;text-align:center}div#onetrust-pc-sdk .ot-pc-footer button{margin:8px 0;background-color:#fff}div#onetrust-pc-sdk .ot-pc-footer-logo{background-color:#fff}div#onetrust-pc-sdk #ot-lst-title span{font-size:24px;font-weight:400;line-height:32px}div#onetrust-pc-sdk #ot-host-lst .ot-host-desc,div#onetrust-pc-sdk #ot-host-lst .ot-host-expand,div#onetrust-pc-sdk #ot-host-lst .ot-host-name,div#onetrust-pc-sdk #ot-host-lst .ot-host-name a,div#onetrust-pc-sdk .back-btn-handler,div#onetrust-pc-sdk .ot-host-opt li>div div{font-size:16px;font-weight:400;line-height:24px}div#onetrust-pc-sdk #ot-pc-lst{top:0}div#onetrust-pc-sdk .back-btn-handler{text-decoration:none!important}div#onetrust-pc-sdk #filter-btn-handler:hover svg{filter:invert(1)}div#onetrust-pc-sdk .back-btn-handler svg{width:16px;height:16px}div#onetrust-pc-sdk .ot-host-item>button{background:0 0!important;border:0!important;height:66px!important;max-width:none!important;width:calc(100% - 5px)!important;transform:translate(2px,2px)}div#onetrust-pc-sdk .ot-host-item{border-bottom:2px solid #b9b9b9;padding:0}div#onetrust-pc-sdk .ot-host-item .ot-acc-hdr{margin:0 0 -6px;padding:8px 0}div#onetrust-pc-sdk ul li:first-child{border-top:2px solid #b9b9b9}div#onetrust-pc-sdk .ot-host-item .ot-plus-minus{margin:0 8px 0 0}div#onetrust-pc-sdk .ot-search-cntr{width:calc(100% - 48px)}div#onetrust-pc-sdk .ot-host-opt .ot-host-info{background-color:transparent}div#onetrust-pc-sdk .ot-host-opt li>div div{padding:0}div#onetrust-pc-sdk #vendor-search-handler{border-radius:0;border-color:#939393;border-style:solid;border-width:2px 0 2px 2px;font-size:20px;height:48px;margin:0}div#onetrust-pc-sdk .ot-lst-subhdr svg{right:0;top:8px}div#onetrust-pc-sdk .ot-fltr-cntr{box-sizing:border-box;right:0;width:48px}div#onetrust-pc-sdk #filter-btn-handler{width:48px!important;padding:8px!important}div#onetrust-pc-sdk #ot-fltr-modal{max-height:400px}
#onetrust-pc-sdk.otPcCenter{overflow:hidden;position:fixed;margin:0 auto;top:5%;right:0;left:0;width:40%;max-width:575px;min-width:575px;border-radius:2.5px;z-index:2147483647;background-color:#fff;-webkit-box-shadow:0px 2px 10px -3px #999;-moz-box-shadow:0px 2px 10px -3px #999;box-shadow:0px 2px 10px -3px #999}#onetrust-pc-sdk.otPcCenter[dir=rtl]{right:0;left:0}#onetrust-pc-sdk.otRelFont{font-size:1rem}#onetrust-pc-sdk .ot-optout-signal{margin-top:.625rem}#onetrust-pc-sdk #ot-addtl-venlst .ot-arw-cntr,#onetrust-pc-sdk #ot-addtl-venlst .ot-plus-minus,#onetrust-pc-sdk .ot-hide-tgl{visibility:hidden}#onetrust-pc-sdk #ot-addtl-venlst .ot-arw-cntr *,#onetrust-pc-sdk #ot-addtl-venlst .ot-plus-minus *,#onetrust-pc-sdk .ot-hide-tgl *{visibility:hidden}#onetrust-pc-sdk #ot-gn-venlst .ot-ven-item .ot-acc-hdr{min-height:40px}#onetrust-pc-sdk .ot-pc-header{height:39px;padding:10px 0 10px 30px;border-bottom:1px solid #e9e9e9}#onetrust-pc-sdk #ot-pc-title,#onetrust-pc-sdk #ot-category-title,#onetrust-pc-sdk .ot-cat-header,#onetrust-pc-sdk #ot-lst-title,#onetrust-pc-sdk .ot-ven-hdr .ot-ven-name,#onetrust-pc-sdk .ot-always-active{font-weight:bold;color:dimgray}#onetrust-pc-sdk .ot-always-active-group .ot-cat-header{width:55%;font-weight:700}#onetrust-pc-sdk .ot-cat-item p{clear:both;float:left;margin-top:10px;margin-bottom:5px;line-height:1.5;font-size:.812em;color:dimgray}#onetrust-pc-sdk .ot-close-icon{height:44px;width:44px;background-size:10px}#onetrust-pc-sdk #ot-pc-title{float:left;font-size:1em;line-height:1.5;margin-bottom:10px;margin-top:10px;width:100%}#onetrust-pc-sdk #accept-recommended-btn-handler{margin-right:10px;margin-bottom:25px;outline-offset:-1px}#onetrust-pc-sdk #ot-pc-desc{clear:both;width:100%;font-size:.812em;line-height:1.5;margin-bottom:25px}#onetrust-pc-sdk #ot-pc-desc *{font-size:inherit;line-height:inherit}#onetrust-pc-sdk #ot-pc-desc ul li{padding:10px 0px}#onetrust-pc-sdk a{color:#656565;cursor:pointer}#onetrust-pc-sdk a:hover{color:#3860be}#onetrust-pc-sdk label{margin-bottom:0}#onetrust-pc-sdk #vdr-lst-dsc{font-size:.812em;line-height:1.5;padding:10px 15px 5px 15px}#onetrust-pc-sdk button{max-width:394px;padding:12px 30px;line-height:1;word-break:break-word;word-wrap:break-word;white-space:normal;font-weight:bold;height:auto}#onetrust-pc-sdk .ot-link-btn{padding:0;margin-bottom:0;border:0;font-weight:normal;line-height:normal;width:auto;height:auto}#onetrust-pc-sdk #ot-pc-content{position:absolute;overflow-y:scroll;padding-left:2px;padding-right:30px;top:60px;bottom:110px;margin:1px 3px 0 30px;width:calc(100% - 63px)}#onetrust-pc-sdk .ot-vs-list .ot-always-active,#onetrust-pc-sdk .ot-cat-grp .ot-always-active{float:right;clear:none;color:#3860be;margin:0;font-size:.813em;line-height:1.3}#onetrust-pc-sdk .ot-pc-scrollbar::-webkit-scrollbar-track{margin-right:20px}#onetrust-pc-sdk .ot-pc-scrollbar::-webkit-scrollbar{width:11px}#onetrust-pc-sdk .ot-pc-scrollbar::-webkit-scrollbar-thumb{border-radius:10px;background:#d8d8d8}#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:#000 1px solid}#onetrust-pc-sdk .ot-pc-scrollbar{scrollbar-arrow-color:#d8d8d8;scrollbar-darkshadow-color:#d8d8d8;scrollbar-face-color:#d8d8d8;scrollbar-shadow-color:#d8d8d8}#onetrust-pc-sdk .save-preference-btn-handler{margin-right:20px}#onetrust-pc-sdk .ot-pc-refuse-all-handler{margin-right:10px}#onetrust-pc-sdk #ot-pc-desc .privacy-notice-link{margin-left:0;margin-right:8px}#onetrust-pc-sdk #ot-pc-desc .ot-imprint-handler{margin-left:0;margin-right:8px}#onetrust-pc-sdk .ot-subgrp-cntr{display:inline-block;clear:both;width:100%;padding-top:15px}#onetrust-pc-sdk .ot-switch+.ot-subgrp-cntr{padding-top:10px}#onetrust-pc-sdk ul.ot-subgrps{margin:0;font-size:initial}#onetrust-pc-sdk ul.ot-subgrps li p,#onetrust-pc-sdk ul.ot-subgrps li h5{font-size:.813em;line-height:1.4;color:dimgray}#onetrust-pc-sdk ul.ot-subgrps .ot-switch{min-height:auto}#onetrust-pc-sdk ul.ot-subgrps .ot-switch-nob{top:0}#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr{display:inline-block;width:100%}#onetrust-pc-sdk ul.ot-subgrps .ot-acc-txt{margin:0}#onetrust-pc-sdk ul.ot-subgrps li{padding:0;border:none}#onetrust-pc-sdk ul.ot-subgrps li h5{position:relative;top:5px;font-weight:bold;margin-bottom:0;float:left}#onetrust-pc-sdk li.ot-subgrp{margin-left:20px;overflow:auto}#onetrust-pc-sdk li.ot-subgrp>h5{width:calc(100% - 100px)}#onetrust-pc-sdk .ot-cat-item p>ul,#onetrust-pc-sdk li.ot-subgrp p>ul{margin:0px;list-style:disc;margin-left:15px;font-size:inherit}#onetrust-pc-sdk .ot-cat-item p>ul li,#onetrust-pc-sdk li.ot-subgrp p>ul li{font-size:inherit;padding-top:10px;padding-left:0px;padding-right:0px;border:none}#onetrust-pc-sdk .ot-cat-item p>ul li:last-child,#onetrust-pc-sdk li.ot-subgrp p>ul li:last-child{padding-bottom:10px}#onetrust-pc-sdk .ot-pc-logo{height:40px;width:120px}#onetrust-pc-sdk .ot-pc-footer{position:absolute;bottom:0px;width:100%;max-height:160px;border-top:1px solid #d8d8d8}#onetrust-pc-sdk.ot-ftr-stacked .ot-pc-refuse-all-handler{margin-bottom:0px}#onetrust-pc-sdk.ot-ftr-stacked #ot-pc-content{bottom:160px}#onetrust-pc-sdk.ot-ftr-stacked .ot-pc-footer button{width:100%;max-width:none}#onetrust-pc-sdk.ot-ftr-stacked .ot-btn-container{margin:0 30px;width:calc(100% - 60px);padding-right:0}#onetrust-pc-sdk .ot-pc-footer-logo{height:30px;width:100%;text-align:right;background:#f4f4f4}#onetrust-pc-sdk .ot-pc-footer-logo a{display:inline-block;margin-top:5px;margin-right:10px}#onetrust-pc-sdk[dir=rtl] .ot-pc-footer-logo{direction:rtl}#onetrust-pc-sdk[dir=rtl] .ot-pc-footer-logo a{margin-right:25px}#onetrust-pc-sdk .ot-tgl{float:right;position:relative;z-index:1}#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob{background-color:#468254;border:1px solid #fff}#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob:before{-webkit-transform:translateX(20px);-ms-transform:translateX(20px);transform:translateX(20px);background-color:#fff;border-color:#fff}#onetrust-pc-sdk .ot-tgl input:focus+.ot-switch{outline:#000 solid 1px}#onetrust-pc-sdk .ot-switch{position:relative;display:inline-block;width:45px;height:25px}#onetrust-pc-sdk .ot-switch-nob{position:absolute;cursor:pointer;top:0;left:0;right:0;bottom:0;background-color:#767676;border:1px solid #ddd;transition:all .2s ease-in 0s;-moz-transition:all .2s ease-in 0s;-o-transition:all .2s ease-in 0s;-webkit-transition:all .2s ease-in 0s;border-radius:20px}#onetrust-pc-sdk .ot-switch-nob:before{position:absolute;content:"";height:18px;width:18px;bottom:3px;left:3px;background-color:#fff;-webkit-transition:.4s;transition:.4s;border-radius:20px}#onetrust-pc-sdk .ot-chkbox input:checked~label::before{background-color:#3860be}#onetrust-pc-sdk .ot-chkbox input+label::after{content:none;color:#fff}#onetrust-pc-sdk .ot-chkbox input:checked+label::after{content:""}#onetrust-pc-sdk .ot-chkbox input:focus+label::before{outline-style:solid;outline-width:2px;outline-style:auto}#onetrust-pc-sdk .ot-chkbox input[aria-checked=mixed]~label::before{background-color:#3860be}#onetrust-pc-sdk .ot-chkbox input[aria-checked=mixed]+label::after{content:""}#onetrust-pc-sdk .ot-chkbox label{position:relative;display:inline-block;padding-left:30px;cursor:pointer;font-weight:500}#onetrust-pc-sdk .ot-chkbox label::before,#onetrust-pc-sdk .ot-chkbox label::after{position:absolute;content:"";display:inline-block;border-radius:3px}#onetrust-pc-sdk .ot-chkbox label::before{height:18px;width:18px;border:1px solid #3860be;left:0px;top:auto}#onetrust-pc-sdk .ot-chkbox label::after{height:5px;width:9px;border-left:3px solid;border-bottom:3px solid;transform:rotate(-45deg);-o-transform:rotate(-45deg);-ms-transform:rotate(-45deg);-webkit-transform:rotate(-45deg);left:4px;top:5px}#onetrust-pc-sdk .ot-label-txt{display:none}#onetrust-pc-sdk .ot-chkbox input,#onetrust-pc-sdk .ot-tgl input{position:absolute;opacity:0;width:0;height:0}#onetrust-pc-sdk .ot-arw-cntr{float:right;position:relative;pointer-events:none}#onetrust-pc-sdk .ot-arw-cntr .ot-arw{width:16px;height:16px;margin-left:5px;color:dimgray;display:inline-block;vertical-align:middle;-webkit-transition:all 150ms ease-in 0s;-moz-transition:all 150ms ease-in 0s;-o-transition:all 150ms ease-in 0s;transition:all 150ms ease-in 0s}#onetrust-pc-sdk input:checked~.ot-acc-hdr .ot-arw,#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-arw-cntr svg{transform:rotate(90deg);-o-transform:rotate(90deg);-ms-transform:rotate(90deg);-webkit-transform:rotate(90deg)}#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:#000 1px solid}#onetrust-pc-sdk .ot-tgl-cntr,#onetrust-pc-sdk .ot-arw-cntr{display:inline-block}#onetrust-pc-sdk .ot-tgl-cntr{width:45px;float:right;margin-top:2px}#onetrust-pc-sdk #ot-lst-cnt .ot-tgl-cntr{margin-top:10px}#onetrust-pc-sdk .ot-always-active-subgroup{width:auto;padding-left:0px !important;top:3px;position:relative}#onetrust-pc-sdk .ot-label-status{font-size:.75em;display:none;font-size:.75em;position:relative;top:2px;padding-right:5px;float:left}#onetrust-pc-sdk .ot-arw-cntr{margin-top:-1px}#onetrust-pc-sdk .ot-arw-cntr svg{-webkit-transition:all 300ms ease-in 0s;-moz-transition:all 300ms ease-in 0s;-o-transition:all 300ms ease-in 0s;transition:all 300ms ease-in 0s;height:10px;width:10px}#onetrust-pc-sdk input:checked~.ot-acc-hdr .ot-arw{transform:rotate(90deg);-o-transform:rotate(90deg);-ms-transform:rotate(90deg);-webkit-transform:rotate(90deg)}#onetrust-pc-sdk .ot-arw{width:10px;margin-left:15px;transition:all 300ms ease-in 0s;-webkit-transition:all 300ms ease-in 0s;-moz-transition:all 300ms ease-in 0s;-o-transition:all 300ms ease-in 0s}#onetrust-pc-sdk .ot-vlst-cntr{margin-bottom:0}#onetrust-pc-sdk .ot-hlst-cntr{margin-top:5px;display:inline-block;width:100%}#onetrust-pc-sdk .category-vendors-list-handler,#onetrust-pc-sdk .category-vendors-list-handler+a,#onetrust-pc-sdk .category-host-list-handler{clear:both;color:#3860be;margin-left:0;font-size:.813em;text-decoration:none;float:left;overflow:hidden}#onetrust-pc-sdk .category-vendors-list-handler:hover,#onetrust-pc-sdk .category-vendors-list-handler+a:hover,#onetrust-pc-sdk .category-host-list-handler:hover{text-decoration-line:underline}#onetrust-pc-sdk .category-vendors-list-handler+a{clear:none}#onetrust-pc-sdk .ot-vlst-cntr .ot-ext-lnk,#onetrust-pc-sdk .ot-ven-hdr .ot-ext-lnk{display:inline-block;height:13px;width:13px;background-repeat:no-repeat;margin-left:1px;margin-top:6px;cursor:pointer}#onetrust-pc-sdk .ot-ven-hdr .ot-ext-lnk{margin-bottom:-1px}#onetrust-pc-sdk .back-btn-handler{font-size:1em;text-decoration:none}#onetrust-pc-sdk .back-btn-handler:hover{opacity:.6}#onetrust-pc-sdk #ot-lst-title h3{display:inline-block;word-break:break-word;word-wrap:break-word;margin-bottom:0;color:#656565;font-size:1em;font-weight:bold;margin-left:15px}#onetrust-pc-sdk #ot-lst-title{margin:10px 0 10px 0px;font-size:1em;text-align:left}#onetrust-pc-sdk #ot-pc-hdr{margin:0 0 0 30px;height:auto;width:auto}#onetrust-pc-sdk #ot-pc-hdr input::placeholder{color:#707070;font-style:italic}#onetrust-pc-sdk #vendor-search-handler{height:31px;width:100%;border-radius:50px;font-size:.8em;padding-right:35px;padding-left:15px;float:left;margin-left:15px}#onetrust-pc-sdk .ot-ven-name{display:block;width:auto;padding-right:5px}#onetrust-pc-sdk #ot-lst-cnt{overflow-y:auto;margin-left:20px;margin-right:7px;width:calc(100% - 27px);max-height:calc(100% - 80px);height:100%;transform:translate3d(0, 0, 0)}#onetrust-pc-sdk #ot-pc-lst{width:100%;bottom:100px;position:absolute;top:60px}#onetrust-pc-sdk #ot-pc-lst:not(.ot-enbl-chr) .ot-tgl-cntr .ot-arw-cntr,#onetrust-pc-sdk #ot-pc-lst:not(.ot-enbl-chr) .ot-tgl-cntr .ot-arw-cntr *{visibility:hidden}#onetrust-pc-sdk #ot-pc-lst .ot-tgl-cntr{right:12px;position:absolute}#onetrust-pc-sdk #ot-pc-lst .ot-arw-cntr{float:right;position:relative}#onetrust-pc-sdk #ot-pc-lst .ot-arw{margin-left:10px}#onetrust-pc-sdk #ot-pc-lst .ot-acc-hdr{overflow:hidden;cursor:pointer}#onetrust-pc-sdk .ot-vlst-cntr{overflow:hidden}#onetrust-pc-sdk #ot-sel-blk{overflow:hidden;width:100%;position:sticky;position:-webkit-sticky;top:0;z-index:3}#onetrust-pc-sdk #ot-back-arw{height:12px;width:12px}#onetrust-pc-sdk .ot-lst-subhdr{width:100%;display:inline-block}#onetrust-pc-sdk .ot-search-cntr{float:left;width:78%;position:relative}#onetrust-pc-sdk .ot-search-cntr>svg{width:30px;height:30px;position:absolute;float:left;right:-15px}#onetrust-pc-sdk .ot-fltr-cntr{float:right;right:50px;position:relative}#onetrust-pc-sdk #filter-btn-handler{background-color:#3860be;border-radius:17px;display:inline-block;position:relative;width:32px;height:32px;-moz-transition:.1s ease;-o-transition:.1s ease;-webkit-transition:1s ease;transition:.1s ease;padding:0;margin:0}#onetrust-pc-sdk #filter-btn-handler:hover{background-color:#3860be}#onetrust-pc-sdk #filter-btn-handler svg{width:12px;height:12px;margin:3px 10px 0 10px;display:block;position:static;right:auto;top:auto}#onetrust-pc-sdk .ot-ven-link,#onetrust-pc-sdk .ot-ven-legclaim-link{color:#3860be;text-decoration:none;font-weight:100;display:inline-block;padding-top:10px;transform:translate(0, 1%);-o-transform:translate(0, 1%);-ms-transform:translate(0, 1%);-webkit-transform:translate(0, 1%);position:relative;z-index:2}#onetrust-pc-sdk .ot-ven-link *,#onetrust-pc-sdk .ot-ven-legclaim-link *{font-size:inherit}#onetrust-pc-sdk .ot-ven-link:hover,#onetrust-pc-sdk .ot-ven-legclaim-link:hover{text-decoration:underline}#onetrust-pc-sdk .ot-ven-hdr{width:calc(100% - 160px);height:auto;float:left;word-break:break-word;word-wrap:break-word;vertical-align:middle;padding-bottom:3px}#onetrust-pc-sdk .ot-ven-link,#onetrust-pc-sdk .ot-ven-legclaim-link{letter-spacing:.03em;font-size:.75em;font-weight:400}#onetrust-pc-sdk .ot-ven-dets{border-radius:2px;background-color:#f8f8f8}#onetrust-pc-sdk .ot-ven-dets li:first-child p:first-child{border-top:none}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:not(:first-child){border-top:1px solid #ddd !important}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:nth-child(n+3) p{display:inline-block}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:nth-child(n+3) p:nth-of-type(odd){width:30%}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:nth-child(n+3) p:nth-of-type(even){width:50%;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc p,#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc h5{padding-top:5px;padding-bottom:5px;display:block}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc h5{display:inline-block}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc p:nth-last-child(-n+1){padding-bottom:10px}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc p:nth-child(-n+2):not(.disc-pur){padding-top:10px}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc .disc-pur-cont{display:inline}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc .disc-pur{position:relative;width:50% !important;word-break:break-word;word-wrap:break-word;left:calc(30% + 17px)}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc .disc-pur:nth-child(-n+1){position:static}#onetrust-pc-sdk .ot-ven-dets p,#onetrust-pc-sdk .ot-ven-dets h5,#onetrust-pc-sdk .ot-ven-dets span{font-size:.69em;text-align:left;vertical-align:middle;word-break:break-word;word-wrap:break-word;margin:0;padding-bottom:10px;padding-left:15px;color:#2e3644}#onetrust-pc-sdk .ot-ven-dets h5{padding-top:5px}#onetrust-pc-sdk .ot-ven-dets span{color:dimgray;padding:0;vertical-align:baseline}#onetrust-pc-sdk .ot-ven-dets .ot-ven-pur h5{border-top:1px solid #e9e9e9;border-bottom:1px solid #e9e9e9;padding-bottom:5px;margin-bottom:5px;font-weight:bold}#onetrust-pc-sdk #ot-host-lst .ot-sel-all{float:right;position:relative;margin-right:42px;top:10px}#onetrust-pc-sdk #ot-host-lst .ot-sel-all input[type=checkbox]{width:auto;height:auto}#onetrust-pc-sdk #ot-host-lst .ot-sel-all label{height:20px;width:20px;padding-left:0px}#onetrust-pc-sdk #ot-host-lst .ot-acc-txt{overflow:hidden;width:95%}#onetrust-pc-sdk .ot-host-hdr{position:relative;z-index:1;pointer-events:none;width:calc(100% - 125px);float:left}#onetrust-pc-sdk .ot-host-name,#onetrust-pc-sdk .ot-host-desc{display:inline-block;width:90%}#onetrust-pc-sdk .ot-host-name{pointer-events:none}#onetrust-pc-sdk .ot-host-hdr>a{text-decoration:underline;font-size:.82em;position:relative;z-index:2;float:left;margin-bottom:5px;pointer-events:initial}#onetrust-pc-sdk .ot-host-name+a{margin-top:5px}#onetrust-pc-sdk .ot-host-name,#onetrust-pc-sdk .ot-host-name a,#onetrust-pc-sdk .ot-host-desc,#onetrust-pc-sdk .ot-host-info{color:dimgray;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-host-name,#onetrust-pc-sdk .ot-host-name a{font-weight:bold;font-size:.82em;line-height:1.3}#onetrust-pc-sdk .ot-host-name a{font-size:1em}#onetrust-pc-sdk .ot-host-expand{margin-top:3px;margin-bottom:3px;clear:both;display:block;color:#3860be;font-size:.72em;font-weight:normal}#onetrust-pc-sdk .ot-host-expand *{font-size:inherit}#onetrust-pc-sdk .ot-host-desc,#onetrust-pc-sdk .ot-host-info{font-size:.688em;line-height:1.4;font-weight:normal}#onetrust-pc-sdk .ot-host-desc{margin-top:10px}#onetrust-pc-sdk .ot-host-opt{margin:0;font-size:inherit;display:inline-block;width:100%}#onetrust-pc-sdk .ot-host-opt li>div div{font-size:.8em;padding:5px 0}#onetrust-pc-sdk .ot-host-opt li>div div:nth-child(1){width:30%;float:left}#onetrust-pc-sdk .ot-host-opt li>div div:nth-child(2){width:70%;float:left;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-host-info{border:none;display:inline-block;width:calc(100% - 10px);padding:10px;margin-bottom:10px;background-color:#f8f8f8}#onetrust-pc-sdk .ot-host-info>div{overflow:auto}#onetrust-pc-sdk #no-results{text-align:center;margin-top:30px}#onetrust-pc-sdk #no-results p{font-size:1em;color:#2e3644;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk #no-results p span{font-weight:bold}#onetrust-pc-sdk #ot-fltr-modal{width:100%;height:auto;display:none;-moz-transition:.2s ease;-o-transition:.2s ease;-webkit-transition:2s ease;transition:.2s ease;overflow:hidden;opacity:1;right:0}#onetrust-pc-sdk #ot-fltr-modal .ot-label-txt{display:inline-block;font-size:.85em;color:dimgray}#onetrust-pc-sdk #ot-fltr-cnt{z-index:2147483646;background-color:#fff;position:absolute;height:90%;max-height:300px;width:325px;left:210px;margin-top:10px;margin-bottom:20px;padding-right:10px;border-radius:3px;-webkit-box-shadow:0px 0px 12px 2px #c7c5c7;-moz-box-shadow:0px 0px 12px 2px #c7c5c7;box-shadow:0px 0px 12px 2px #c7c5c7}#onetrust-pc-sdk .ot-fltr-scrlcnt{overflow-y:auto;overflow-x:hidden;clear:both;max-height:calc(100% - 60px)}#onetrust-pc-sdk #ot-anchor{border:12px solid rgba(0,0,0,0);display:none;position:absolute;z-index:2147483647;right:55px;top:75px;transform:rotate(45deg);-o-transform:rotate(45deg);-ms-transform:rotate(45deg);-webkit-transform:rotate(45deg);background-color:#fff;-webkit-box-shadow:-3px -3px 5px -2px #c7c5c7;-moz-box-shadow:-3px -3px 5px -2px #c7c5c7;box-shadow:-3px -3px 5px -2px #c7c5c7}#onetrust-pc-sdk .ot-fltr-btns{margin-left:15px}#onetrust-pc-sdk #filter-apply-handler{margin-right:15px}#onetrust-pc-sdk .ot-fltr-opt{margin-bottom:5px;margin-left:15px;min-height:20px;width:75%;position:relative}#onetrust-pc-sdk .ot-fltr-opt p{display:inline-block;margin:0;font-size:.9em;color:#2e3644}#onetrust-pc-sdk .ot-chkbox label span{font-size:.85em;color:dimgray}#onetrust-pc-sdk .ot-chkbox input[type=checkbox]+label::after{content:none;color:#fff}#onetrust-pc-sdk .ot-chkbox input[type=checkbox]:checked+label::after{content:""}#onetrust-pc-sdk .ot-chkbox input[type=checkbox][aria-checked=mixed]+label::after{content:""}#onetrust-pc-sdk .ot-chkbox input[type=checkbox]:focus+label::before{outline-style:solid;outline-width:2px;outline-style:auto}#onetrust-pc-sdk #ot-selall-vencntr,#onetrust-pc-sdk #ot-selall-adtlvencntr,#onetrust-pc-sdk #ot-selall-hostcntr,#onetrust-pc-sdk #ot-selall-licntr,#onetrust-pc-sdk #ot-selall-gnvencntr{right:15px;position:relative;width:20px;height:20px;float:right}#onetrust-pc-sdk #ot-selall-vencntr label,#onetrust-pc-sdk #ot-selall-adtlvencntr label,#onetrust-pc-sdk #ot-selall-hostcntr label,#onetrust-pc-sdk #ot-selall-licntr label,#onetrust-pc-sdk #ot-selall-gnvencntr label{float:left;padding-left:0}#onetrust-pc-sdk #ot-ven-lst:first-child{border-top:1px solid #e2e2e2}#onetrust-pc-sdk ul{list-style:none;padding:0}#onetrust-pc-sdk ul li{position:relative;margin:0;padding:15px 15px 15px 10px;border-bottom:1px solid #e2e2e2}#onetrust-pc-sdk ul li h3,#onetrust-pc-sdk ul li h4{font-size:.75em;color:#656565;margin:0;display:inline-block;width:70%;height:auto;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk ul li p{margin:0;font-size:.7em}#onetrust-pc-sdk ul li input[type=checkbox]{position:absolute;cursor:pointer;width:100%;height:100%;opacity:0;margin:0;top:0;left:0}#onetrust-pc-sdk .ot-cat-item>button:focus,#onetrust-pc-sdk .ot-acc-cntr>button:focus,#onetrust-pc-sdk li>button:focus{outline:#000 solid 2px}#onetrust-pc-sdk .ot-cat-item>button,#onetrust-pc-sdk .ot-acc-cntr>button,#onetrust-pc-sdk li>button{position:absolute;cursor:pointer;width:100%;height:100%;margin:0;top:0;left:0;z-index:1;max-width:none;border:none}#onetrust-pc-sdk .ot-cat-item>button[aria-expanded=false]~.ot-acc-txt,#onetrust-pc-sdk .ot-acc-cntr>button[aria-expanded=false]~.ot-acc-txt,#onetrust-pc-sdk li>button[aria-expanded=false]~.ot-acc-txt{margin-top:0;max-height:0;opacity:0;overflow:hidden;width:100%;transition:.25s ease-out;display:none}#onetrust-pc-sdk .ot-cat-item>button[aria-expanded=true]~.ot-acc-txt,#onetrust-pc-sdk .ot-acc-cntr>button[aria-expanded=true]~.ot-acc-txt,#onetrust-pc-sdk li>button[aria-expanded=true]~.ot-acc-txt{transition:.1s ease-in;margin-top:10px;width:100%;overflow:auto;display:block}#onetrust-pc-sdk .ot-cat-item>button[aria-expanded=true]~.ot-acc-grpcntr,#onetrust-pc-sdk .ot-acc-cntr>button[aria-expanded=true]~.ot-acc-grpcntr,#onetrust-pc-sdk li>button[aria-expanded=true]~.ot-acc-grpcntr{width:auto;margin-top:0px;padding-bottom:10px}#onetrust-pc-sdk .ot-host-item>button:focus,#onetrust-pc-sdk .ot-ven-item>button:focus{outline:0;border:2px solid #000}#onetrust-pc-sdk .ot-hide-acc>button{pointer-events:none}#onetrust-pc-sdk .ot-hide-acc .ot-plus-minus>*,#onetrust-pc-sdk .ot-hide-acc .ot-arw-cntr>*{visibility:hidden}#onetrust-pc-sdk .ot-hide-acc .ot-acc-hdr{min-height:30px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt){padding-right:10px;width:calc(100% - 37px);margin-top:10px;max-height:calc(100% - 90px)}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk{background-color:#f9f9fc;border:1px solid #e2e2e2;width:calc(100% - 2px);padding-bottom:5px;padding-top:5px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt{border:unset;background-color:unset}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt .ot-sel-all-hdr{display:none}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt .ot-sel-all{padding-right:.5rem}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt .ot-sel-all .ot-chkbox{right:0}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) .ot-sel-all{padding-right:34px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) .ot-sel-all-chkbox{width:auto}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) ul li{border:1px solid #e2e2e2;margin-bottom:10px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) .ot-acc-cntr>.ot-acc-hdr{padding:10px 0 10px 15px}#onetrust-pc-sdk.ot-addtl-vendors .ot-sel-all-chkbox{float:right}#onetrust-pc-sdk.ot-addtl-vendors .ot-plus-minus~.ot-sel-all-chkbox{right:34px}#onetrust-pc-sdk.ot-addtl-vendors #ot-ven-lst:first-child{border-top:none}#onetrust-pc-sdk .ot-acc-cntr{position:relative;border-left:1px solid #e2e2e2;border-right:1px solid #e2e2e2;border-bottom:1px solid #e2e2e2}#onetrust-pc-sdk .ot-acc-cntr input{z-index:1}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr{background-color:#f9f9fc;padding:5px 0 5px 15px;width:auto}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr .ot-plus-minus{vertical-align:middle;top:auto}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr .ot-arw-cntr{right:10px}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr input{z-index:2}#onetrust-pc-sdk .ot-acc-cntr.ot-add-tech .ot-acc-hdr{padding:10px 0 10px 15px}#onetrust-pc-sdk .ot-acc-cntr>input[type=checkbox]:checked~.ot-acc-hdr{border-bottom:1px solid #e2e2e2}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-txt{padding-left:10px;padding-right:10px}#onetrust-pc-sdk .ot-acc-cntr button[aria-expanded=true]~.ot-acc-txt{width:auto}#onetrust-pc-sdk .ot-acc-cntr .ot-addtl-venbox{display:none}#onetrust-pc-sdk .ot-vlst-cntr{margin-bottom:0;width:100%}#onetrust-pc-sdk .ot-vensec-title{font-size:.813em;vertical-align:middle;display:inline-block}#onetrust-pc-sdk .category-vendors-list-handler,#onetrust-pc-sdk .category-vendors-list-handler+a{margin-left:0;margin-top:10px}#onetrust-pc-sdk #ot-selall-vencntr.line-through label::after,#onetrust-pc-sdk #ot-selall-adtlvencntr.line-through label::after,#onetrust-pc-sdk #ot-selall-licntr.line-through label::after,#onetrust-pc-sdk #ot-selall-hostcntr.line-through label::after,#onetrust-pc-sdk #ot-selall-gnvencntr.line-through label::after{height:auto;border-left:0;transform:none;-o-transform:none;-ms-transform:none;-webkit-transform:none;left:5px;top:9px}#onetrust-pc-sdk #ot-category-title{float:left;padding-bottom:10px;font-size:1em;width:100%}#onetrust-pc-sdk .ot-cat-grp{margin-top:10px}#onetrust-pc-sdk .ot-cat-item{line-height:1.1;margin-top:10px;display:inline-block;width:100%}#onetrust-pc-sdk .ot-btn-container{text-align:right}#onetrust-pc-sdk .ot-btn-container button{display:inline-block;font-size:.75em;letter-spacing:.08em;margin-top:19px}#onetrust-pc-sdk #close-pc-btn-handler.ot-close-icon{position:absolute;top:10px;right:0;z-index:1;padding:0;background-color:rgba(0,0,0,0);border:none}#onetrust-pc-sdk #close-pc-btn-handler.ot-close-icon svg{display:block;height:10px;width:10px}#onetrust-pc-sdk #clear-filters-handler{margin-top:20px;margin-bottom:10px;float:right;max-width:200px;text-decoration:none;color:#3860be;font-size:.9em;font-weight:bold;background-color:rgba(0,0,0,0);border-color:rgba(0,0,0,0);padding:1px}#onetrust-pc-sdk #clear-filters-handler:hover{color:#2285f7}#onetrust-pc-sdk #clear-filters-handler:focus{outline:#000 solid 1px}#onetrust-pc-sdk .ot-enbl-chr h4~.ot-tgl,#onetrust-pc-sdk .ot-enbl-chr h4~.ot-always-active{right:45px}#onetrust-pc-sdk .ot-enbl-chr h4~.ot-tgl+.ot-tgl{right:120px}#onetrust-pc-sdk .ot-enbl-chr .ot-pli-hdr.ot-leg-border-color span:first-child{width:90px}#onetrust-pc-sdk .ot-enbl-chr li.ot-subgrp>h5+.ot-tgl-cntr{padding-right:25px}#onetrust-pc-sdk .ot-plus-minus{width:20px;height:20px;font-size:1.5em;position:relative;display:inline-block;margin-right:5px;top:3px}#onetrust-pc-sdk .ot-plus-minus span{position:absolute;background:#27455c;border-radius:1px}#onetrust-pc-sdk .ot-plus-minus span:first-of-type{top:25%;bottom:25%;width:10%;left:45%}#onetrust-pc-sdk .ot-plus-minus span:last-of-type{left:25%;right:25%;height:10%;top:45%}#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-arw,#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-plus-minus span:first-of-type,#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-plus-minus span:last-of-type{transform:rotate(90deg)}#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-plus-minus span:last-of-type{left:50%;right:50%}#onetrust-pc-sdk #ot-selall-vencntr label,#onetrust-pc-sdk #ot-selall-adtlvencntr label,#onetrust-pc-sdk #ot-selall-hostcntr label,#onetrust-pc-sdk #ot-selall-licntr label{position:relative;display:inline-block;width:20px;height:20px}#onetrust-pc-sdk .ot-host-item .ot-plus-minus,#onetrust-pc-sdk .ot-ven-item .ot-plus-minus{float:left;margin-right:8px;top:10px}#onetrust-pc-sdk .ot-ven-item ul{list-style:none inside;font-size:100%;margin:0}#onetrust-pc-sdk .ot-ven-item ul li{margin:0 !important;padding:0;border:none !important}#onetrust-pc-sdk .ot-pli-hdr{color:#77808e;overflow:hidden;padding-top:7.5px;padding-bottom:7.5px;width:calc(100% - 2px);border-top-left-radius:3px;border-top-right-radius:3px}#onetrust-pc-sdk .ot-pli-hdr span:first-child{top:50%;transform:translateY(50%);max-width:90px}#onetrust-pc-sdk .ot-pli-hdr span:last-child{padding-right:10px;max-width:95px;text-align:center}#onetrust-pc-sdk .ot-li-title{float:right;font-size:.813em}#onetrust-pc-sdk .ot-pli-hdr.ot-leg-border-color{background-color:#f4f4f4;border:1px solid #d8d8d8}#onetrust-pc-sdk .ot-pli-hdr.ot-leg-border-color span:first-child{text-align:left;width:70px}#onetrust-pc-sdk li.ot-subgrp>h5,#onetrust-pc-sdk .ot-cat-header{width:calc(100% - 130px)}#onetrust-pc-sdk li.ot-subgrp>h5+.ot-tgl-cntr{padding-left:13px}#onetrust-pc-sdk .ot-acc-grpcntr .ot-acc-grpdesc{margin-bottom:5px}#onetrust-pc-sdk .ot-acc-grpcntr .ot-subgrp-cntr{border-top:1px solid #d8d8d8}#onetrust-pc-sdk .ot-acc-grpcntr .ot-vlst-cntr+.ot-subgrp-cntr{border-top:none}#onetrust-pc-sdk .ot-acc-hdr .ot-arw-cntr+.ot-tgl-cntr,#onetrust-pc-sdk .ot-acc-txt h4+.ot-tgl-cntr{padding-left:13px}#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item .ot-subgrp>h5,#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item .ot-cat-header{width:calc(100% - 145px)}#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item h5+.ot-tgl-cntr,#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item .ot-cat-header+.ot-tgl{padding-left:28px}#onetrust-pc-sdk .ot-sel-all-hdr,#onetrust-pc-sdk .ot-sel-all-chkbox{display:inline-block;width:100%;position:relative}#onetrust-pc-sdk .ot-sel-all-chkbox{z-index:1}#onetrust-pc-sdk .ot-sel-all{margin:0;position:relative;padding-right:23px;float:right}#onetrust-pc-sdk .ot-consent-hdr,#onetrust-pc-sdk .ot-li-hdr{float:right;font-size:.812em;line-height:normal;text-align:center;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-li-hdr{max-width:100px;padding-right:10px}#onetrust-pc-sdk .ot-consent-hdr{max-width:55px}#onetrust-pc-sdk #ot-selall-licntr{display:block;width:21px;height:auto;float:right;position:relative;right:80px}#onetrust-pc-sdk #ot-selall-licntr label{position:absolute}#onetrust-pc-sdk .ot-ven-ctgl{margin-left:66px}#onetrust-pc-sdk .ot-ven-litgl+.ot-arw-cntr{margin-left:81px}#onetrust-pc-sdk .ot-enbl-chr .ot-host-cnt .ot-tgl-cntr{width:auto}#onetrust-pc-sdk #ot-lst-cnt:not(.ot-host-cnt) .ot-tgl-cntr{width:auto;top:auto;height:20px}#onetrust-pc-sdk #ot-lst-cnt .ot-chkbox{position:relative;display:inline-block;width:20px;height:20px}#onetrust-pc-sdk #ot-lst-cnt .ot-chkbox label{position:absolute;padding:0;width:20px;height:20px}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info-cntr{border:1px solid #d8d8d8;padding:.75rem 2rem;padding-bottom:0;width:auto;margin-top:.5rem}#onetrust-pc-sdk .ot-acc-grpdesc+.ot-leg-btn-container{padding-left:20px;padding-right:20px;width:calc(100% - 40px);margin-bottom:5px}#onetrust-pc-sdk .ot-subgrp .ot-leg-btn-container{margin-bottom:5px}#onetrust-pc-sdk #ot-ven-lst .ot-leg-btn-container{margin-top:10px}#onetrust-pc-sdk .ot-leg-btn-container{display:inline-block;width:100%;margin-bottom:10px}#onetrust-pc-sdk .ot-leg-btn-container button{height:auto;padding:6.5px 8px;margin-bottom:0;letter-spacing:0;font-size:.75em;line-height:normal}#onetrust-pc-sdk .ot-leg-btn-container svg{display:none;height:14px;width:14px;padding-right:5px;vertical-align:sub}#onetrust-pc-sdk .ot-active-leg-btn{cursor:default;pointer-events:none}#onetrust-pc-sdk .ot-active-leg-btn svg{display:inline-block}#onetrust-pc-sdk .ot-remove-objection-handler{text-decoration:underline;padding:0;font-size:.75em;font-weight:600;line-height:1;padding-left:10px}#onetrust-pc-sdk .ot-obj-leg-btn-handler span{font-weight:bold;text-align:center;font-size:inherit;line-height:1.5}#onetrust-pc-sdk.ot-close-btn-link #close-pc-btn-handler{border:none;height:auto;line-height:1.5;text-decoration:underline;font-size:.69em;background:none;right:15px;top:15px;width:auto;font-weight:normal}#onetrust-pc-sdk .ot-pgph-link{font-size:.813em !important;margin-top:5px;position:relative}#onetrust-pc-sdk .ot-pgph-link.ot-pgph-link-subgroup{margin-bottom:1rem}#onetrust-pc-sdk .ot-pgph-contr{margin:0 2.5rem}#onetrust-pc-sdk .ot-pgph-title{font-size:1.18rem;margin-bottom:2rem}#onetrust-pc-sdk .ot-pgph-desc{font-size:1rem;font-weight:400;margin-bottom:2rem;line-height:1.5rem}#onetrust-pc-sdk .ot-pgph-desc:not(:last-child):after{content:"";width:96%;display:block;margin:0 auto;padding-bottom:2rem;border-bottom:1px solid #e9e9e9}#onetrust-pc-sdk .ot-cat-header{float:left;font-weight:600;font-size:.875em;line-height:1.5;max-width:90%;vertical-align:middle}#onetrust-pc-sdk .ot-vnd-item>button:focus{outline:#000 solid 2px}#onetrust-pc-sdk .ot-vnd-item>button{position:absolute;cursor:pointer;width:100%;height:100%;margin:0;top:0;left:0;z-index:1;max-width:none;border:none}#onetrust-pc-sdk .ot-vnd-item>button[aria-expanded=false]~.ot-acc-txt{margin-top:0;max-height:0;opacity:0;overflow:hidden;width:100%;transition:.25s ease-out;display:none}#onetrust-pc-sdk .ot-vnd-item>button[aria-expanded=true]~.ot-acc-txt{transition:.1s ease-in;margin-top:10px;width:100%;overflow:auto;display:block}#onetrust-pc-sdk .ot-vnd-item>button[aria-expanded=true]~.ot-acc-grpcntr{width:auto;margin-top:0px;padding-bottom:10px}#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item{position:relative;border-radius:2px;margin:0;padding:0;border:1px solid #d8d8d8;border-top:none;width:calc(100% - 2px);float:left}#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item:first-of-type{margin-top:10px;border-top:1px solid #d8d8d8}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpdesc{padding-left:20px;padding-right:20px;width:calc(100% - 40px);font-size:.812em;margin-bottom:10px;margin-top:15px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpdesc>ul{padding-top:10px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpdesc>ul li{padding-top:0;line-height:1.5;padding-bottom:10px}#onetrust-pc-sdk .ot-accordion-layout div+.ot-acc-grpdesc{margin-top:5px}#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr:first-child{margin-top:10px}#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr:last-child,#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr:last-child{margin-bottom:5px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{padding-top:11.5px;padding-bottom:11.5px;padding-left:20px;padding-right:20px;width:calc(100% - 40px);display:inline-block}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-txt{width:100%;padding:0}#onetrust-pc-sdk .ot-accordion-layout .ot-subgrp-cntr{padding-left:20px;padding-right:15px;padding-bottom:0;width:calc(100% - 35px)}#onetrust-pc-sdk .ot-accordion-layout .ot-subgrp{padding-right:5px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpcntr{z-index:1;position:relative}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header+.ot-arw-cntr{position:absolute;top:50%;transform:translateY(-50%);right:20px;margin-top:-2px}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header+.ot-arw-cntr .ot-arw{width:15px;height:20px;margin-left:5px;color:dimgray}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{float:none;color:#2e3644;margin:0;display:inline-block;height:auto;word-wrap:break-word;min-height:inherit}#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr,#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr{padding-left:20px;width:calc(100% - 20px);display:inline-block;margin-top:0;padding-bottom:2px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{position:relative;min-height:25px}#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl,#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active{position:absolute;top:50%;transform:translateY(-50%);right:20px}#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl+.ot-tgl{right:95px}#onetrust-pc-sdk .ot-accordion-layout .category-vendors-list-handler,#onetrust-pc-sdk .ot-accordion-layout .category-vendors-list-handler+a{margin-top:5px}#onetrust-pc-sdk #ot-lst-cnt{margin-top:1rem;max-height:calc(100% - 96px)}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info-cntr{border:1px solid #d8d8d8;padding:.75rem 2rem;padding-bottom:0;width:auto;margin-top:.5rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info{margin-bottom:1rem;padding-left:.75rem;padding-right:.75rem;display:flex;flex-direction:column}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info[data-vnd-info-key*=DPOEmail]{border-top:1px solid #d8d8d8;padding-top:1rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info[data-vnd-info-key*=DPOLink]{border-bottom:1px solid #d8d8d8;padding-bottom:1rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info .ot-vnd-lbl{font-weight:bold;font-size:.85em;margin-bottom:.5rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info .ot-vnd-cnt{margin-left:.5rem;font-weight:500;font-size:.85rem}#onetrust-pc-sdk .ot-vs-list,#onetrust-pc-sdk .ot-vnd-serv{width:auto;padding:1rem 1.25rem;padding-bottom:0}#onetrust-pc-sdk .ot-vs-list .ot-vnd-serv-hdr-cntr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-serv-hdr-cntr{padding-bottom:.75rem;border-bottom:1px solid #d8d8d8}#onetrust-pc-sdk .ot-vs-list .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr{font-weight:600;font-size:.95em;line-height:2;margin-left:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item{border:none;margin:0;padding:0}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item button,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item button{outline:none;border-bottom:1px solid #d8d8d8}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item button[aria-expanded=true],#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item button[aria-expanded=true]{border-bottom:none}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item:first-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item:first-child{margin-top:.25rem;border-top:unset}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item:last-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item:last-child{margin-bottom:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item:last-child button,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item:last-child button{border-bottom:none}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info-cntr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info-cntr{border:1px solid #d8d8d8;padding:.75rem 1.75rem;padding-bottom:0;width:auto;margin-top:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info{margin-bottom:1rem;padding-left:.75rem;padding-right:.75rem;display:flex;flex-direction:column}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOEmail],#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOEmail]{border-top:1px solid #d8d8d8;padding-top:1rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOLink],#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOLink]{border-bottom:1px solid #d8d8d8;padding-bottom:1rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info .ot-vnd-lbl,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info .ot-vnd-lbl{font-weight:bold;font-size:.85em;margin-bottom:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info .ot-vnd-cnt,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info .ot-vnd-cnt{margin-left:.5rem;font-weight:500;font-size:.85rem}#onetrust-pc-sdk .ot-vs-list.ot-vnd-subgrp-cnt,#onetrust-pc-sdk .ot-vnd-serv.ot-vnd-subgrp-cnt{padding-left:40px}#onetrust-pc-sdk .ot-vs-list.ot-vnd-subgrp-cnt .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr,#onetrust-pc-sdk .ot-vnd-serv.ot-vnd-subgrp-cnt .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr{font-size:.8em}#onetrust-pc-sdk .ot-vs-list.ot-vnd-subgrp-cnt .ot-cat-header,#onetrust-pc-sdk .ot-vnd-serv.ot-vnd-subgrp-cnt .ot-cat-header{font-size:.8em}#onetrust-pc-sdk .ot-subgrp-cntr .ot-vnd-serv{margin-bottom:1rem;padding:1rem .95rem}#onetrust-pc-sdk .ot-subgrp-cntr .ot-vnd-serv .ot-vnd-serv-hdr-cntr{padding-bottom:.75rem;border-bottom:1px solid #d8d8d8}#onetrust-pc-sdk .ot-subgrp-cntr .ot-vnd-serv .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr{font-weight:700;font-size:.8em;line-height:20px;margin-left:.82rem}#onetrust-pc-sdk .ot-subgrp-cntr .ot-cat-header{font-weight:700;font-size:.8em;line-height:20px}#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-vnd-serv .ot-vnd-lst-cont .ot-accordion-layout .ot-acc-hdr div.ot-chkbox{margin-left:.82rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr{padding:.7rem 0;margin:0;display:flex;width:100%;align-items:center;justify-content:space-between}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr div:first-child,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr div:first-child,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr div:first-child{margin-left:.5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr div:last-child,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr div:last-child,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr div:last-child{margin-right:.5rem;margin-left:.5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-always-active{position:relative;right:unset;top:unset;transform:unset}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-plus-minus{top:0}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-arw-cntr{float:none;top:unset;right:unset;transform:unset;margin-top:-2px;position:relative}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-cat-header{flex:1;margin:0 .5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-tgl{position:relative;transform:none;right:0;top:0;float:none}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox{position:relative;margin:0 .5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox label{padding:0}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox label::before{position:relative}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox input{position:absolute;cursor:pointer;width:100%;height:100%;opacity:0;margin:0;top:0;left:0;z-index:1}#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp .ot-acc-hdr h5.ot-cat-header,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp .ot-acc-hdr h4.ot-cat-header{margin:0}#onetrust-pc-sdk .ot-vs-config .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp h5{top:0;line-height:20px}#onetrust-pc-sdk .ot-vs-list{display:flex;flex-direction:column;padding:0;margin:.5rem 4px}#onetrust-pc-sdk .ot-vs-selc-all{display:flex;padding:0;float:unset;align-items:center;justify-content:flex-start}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf{justify-content:flex-end}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf.ot-caret-conf .ot-sel-all-chkbox{margin-right:48px}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf .ot-sel-all-chkbox{margin:0;padding:0;margin-right:14px;justify-content:flex-end}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf #ot-selall-vencntr.ot-chkbox,#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf #ot-selall-vencntr.ot-tgl{display:inline-block;right:unset;width:auto;height:auto;float:none}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf #ot-selall-vencntr label{width:45px;height:25px}#onetrust-pc-sdk .ot-vs-selc-all .ot-sel-all-chkbox{margin-right:11px;margin-left:.75rem;display:flex;align-items:center}#onetrust-pc-sdk .ot-vs-selc-all .sel-all-hdr{margin:0 1.25rem;font-size:.812em;line-height:normal;text-align:center;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-vnd-list-cnt #ot-selall-vencntr.ot-chkbox{float:unset;right:0}#onetrust-pc-sdk[dir=rtl] #ot-back-arw,#onetrust-pc-sdk[dir=rtl] input~.ot-acc-hdr .ot-arw{transform:rotate(180deg);-o-transform:rotate(180deg);-ms-transform:rotate(180deg);-webkit-transform:rotate(180deg)}#onetrust-pc-sdk[dir=rtl] input:checked~.ot-acc-hdr .ot-arw{transform:rotate(270deg);-o-transform:rotate(270deg);-ms-transform:rotate(270deg);-webkit-transform:rotate(270deg)}#onetrust-pc-sdk[dir=rtl] .ot-chkbox label::after{transform:rotate(45deg);-webkit-transform:rotate(45deg);-o-transform:rotate(45deg);-ms-transform:rotate(45deg);border-left:0;border-right:3px solid}#onetrust-pc-sdk[dir=rtl] .ot-search-cntr>svg{right:0}@media only screen and (max-width: 600px){#onetrust-pc-sdk.otPcCenter{left:0;min-width:100%;height:100%;top:0;border-radius:0}#onetrust-pc-sdk #ot-pc-content,#onetrust-pc-sdk.ot-ftr-stacked .ot-btn-container{margin:1px 3px 0 10px;padding-right:10px;width:calc(100% - 23px)}#onetrust-pc-sdk .ot-btn-container button{max-width:none;letter-spacing:.01em}#onetrust-pc-sdk #close-pc-btn-handler{top:10px;right:17px}#onetrust-pc-sdk p{font-size:.7em}#onetrust-pc-sdk #ot-pc-hdr{margin:10px 10px 0 5px;width:calc(100% - 15px)}#onetrust-pc-sdk .vendor-search-handler{font-size:1em}#onetrust-pc-sdk #ot-back-arw{margin-left:12px}#onetrust-pc-sdk #ot-lst-cnt{margin:0;padding:0 5px 0 10px;min-width:95%}#onetrust-pc-sdk .switch+p{max-width:80%}#onetrust-pc-sdk .ot-ftr-stacked button{width:100%}#onetrust-pc-sdk #ot-fltr-cnt{max-width:320px;width:90%;border-top-right-radius:0;border-bottom-right-radius:0;margin:0;margin-left:15px;left:auto;right:40px;top:85px}#onetrust-pc-sdk .ot-fltr-opt{margin-left:25px;margin-bottom:10px}#onetrust-pc-sdk .ot-pc-refuse-all-handler{margin-bottom:0}#onetrust-pc-sdk #ot-fltr-cnt{right:40px}}@media only screen and (max-width: 476px){#onetrust-pc-sdk .ot-fltr-cntr,#onetrust-pc-sdk #ot-fltr-cnt{right:10px}#onetrust-pc-sdk #ot-anchor{right:25px}#onetrust-pc-sdk button{width:100%}#onetrust-pc-sdk:not(.ot-addtl-vendors) #ot-pc-lst:not(.ot-enbl-chr) .ot-sel-all{padding-right:9px}#onetrust-pc-sdk:not(.ot-addtl-vendors) #ot-pc-lst:not(.ot-enbl-chr) .ot-tgl-cntr{right:0}}@media(min-width: 768px){#onetrust-pc-sdk.ot-tgl-with-label .ot-label-status{display:inline}#onetrust-pc-sdk.ot-tgl-with-label #ot-pc-lst .ot-label-status{display:none}}@media only screen and (max-width: 896px)and (max-height: 425px)and (orientation: landscape){#onetrust-pc-sdk.otPcCenter{left:0;top:0;min-width:100%;height:100%;border-radius:0}#onetrust-pc-sdk .ot-pc-header{height:auto;min-height:20px}#onetrust-pc-sdk .ot-pc-header .ot-pc-logo{max-height:30px}#onetrust-pc-sdk .ot-pc-footer{max-height:60px;overflow-y:auto}#onetrust-pc-sdk #ot-pc-content,#onetrust-pc-sdk #ot-pc-lst{bottom:70px}#onetrust-pc-sdk.ot-ftr-stacked #ot-pc-content{bottom:70px}#onetrust-pc-sdk #ot-anchor{left:initial;right:50px}#onetrust-pc-sdk #ot-lst-title{margin-top:12px}#onetrust-pc-sdk #ot-lst-title *{font-size:inherit}#onetrust-pc-sdk #ot-pc-hdr input{margin-right:0;padding-right:45px}#onetrust-pc-sdk .switch+p{max-width:85%}#onetrust-pc-sdk #ot-sel-blk{position:static}#onetrust-pc-sdk #ot-pc-lst{overflow:auto}#onetrust-pc-sdk #ot-lst-cnt{max-height:none;overflow:initial}#onetrust-pc-sdk #ot-lst-cnt.no-results{height:auto}#onetrust-pc-sdk input{font-size:1em !important}#onetrust-pc-sdk p{font-size:.6em}#onetrust-pc-sdk #ot-fltr-modal{width:100%;top:0}#onetrust-pc-sdk ul li p,#onetrust-pc-sdk .category-vendors-list-handler,#onetrust-pc-sdk .category-vendors-list-handler+a,#onetrust-pc-sdk .category-host-list-handler{font-size:.6em}#onetrust-pc-sdk.ot-shw-fltr #ot-anchor{display:none !important}#onetrust-pc-sdk.ot-shw-fltr #ot-pc-lst{height:100% !important;overflow:hidden;top:0px}#onetrust-pc-sdk.ot-shw-fltr #ot-fltr-cnt{margin:0;height:100%;max-height:none;padding:10px;top:0;width:calc(100% - 20px);position:absolute;right:0;left:0;max-width:none}#onetrust-pc-sdk.ot-shw-fltr .ot-fltr-scrlcnt{max-height:calc(100% - 65px)}}
            #onetrust-consent-sdk #onetrust-pc-sdk,
                #onetrust-consent-sdk #ot-search-cntr,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-switch.ot-toggle,
                #onetrust-consent-sdk #onetrust-pc-sdk ot-grp-hdr1 .checkbox,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-title:after
                ,#onetrust-consent-sdk #onetrust-pc-sdk #ot-sel-blk,
                        #onetrust-consent-sdk #onetrust-pc-sdk #ot-fltr-cnt,
                        #onetrust-consent-sdk #onetrust-pc-sdk #ot-anchor {
                    background-color: #FFF;
                }
               
            #onetrust-consent-sdk #onetrust-pc-sdk h3,
                #onetrust-consent-sdk #onetrust-pc-sdk h4,
                #onetrust-consent-sdk #onetrust-pc-sdk h5,
                #onetrust-consent-sdk #onetrust-pc-sdk h6,
                #onetrust-consent-sdk #onetrust-pc-sdk p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-ven-lst .ot-ven-opts p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-desc,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-title,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-li-title,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-sel-all-hdr span,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-info,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-fltr-modal #modal-header,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-checkbox label span,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst #ot-sel-blk p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst #ot-lst-title h3,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst .back-btn-handler p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst .ot-ven-name,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst #ot-ven-lst .consent-category,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-inactive-leg-btn,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-label-status,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-chkbox label span,
                #onetrust-consent-sdk #onetrust-pc-sdk #clear-filters-handler,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-optout-signal
                {
                    color: #2E2E2E;
                }
             #onetrust-consent-sdk #onetrust-pc-sdk .privacy-notice-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-pgph-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk .category-vendors-list-handler,
                    #onetrust-consent-sdk #onetrust-pc-sdk .category-vendors-list-handler + a,
                    #onetrust-consent-sdk #onetrust-pc-sdk .category-host-list-handler,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-ven-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-ven-legclaim-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-name a,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-acc-hdr .ot-host-expand,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-info a,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-content #ot-pc-desc .ot-link-btn,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info a,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info a
                    {
                        color: #007398;
                    }
            #onetrust-consent-sdk #onetrust-pc-sdk .category-vendors-list-handler:hover { text-decoration: underline;}
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-grpcntr.ot-acc-txt,
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-txt .ot-subgrp-tgl .ot-switch.ot-toggle
             {
                background-color: #F8F8F8;
            }
             #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-info,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-txt .ot-ven-dets
                            {
                                background-color: #F8F8F8;
                            }
        #onetrust-consent-sdk #onetrust-pc-sdk
            button:not(#clear-filters-handler):not(.ot-close-icon):not(#filter-btn-handler):not(.ot-remove-objection-handler):not(.ot-obj-leg-btn-handler):not([aria-expanded]):not(.ot-link-btn),
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-active-leg-btn {
                background-color: #007398;border-color: #007398;
                color: #FFF;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-active-menu {
                border-color: #007398;
            }
            
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-remove-objection-handler{
                background-color: transparent;
                border: 1px solid transparent;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-inactive-leg-btn {
                background-color: #FFFFFF;
                color: #4D4D4D; border-color: #4D4D4D;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-tgl input:focus + .ot-switch, .ot-switch .ot-switch-nob, .ot-switch .ot-switch-nob:before,
            #onetrust-pc-sdk .ot-checkbox input[type="checkbox"]:focus + label::before,
            #onetrust-pc-sdk .ot-chkbox input[type="checkbox"]:focus + label::before {
                outline-color: #000000;
                outline-width: 1px;
                outline-offset: 1px;
            }
            #onetrust-pc-sdk .ot-host-item > button:focus, #onetrust-pc-sdk .ot-ven-item > button:focus {
                border: 1px solid #000000;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk *:focus,
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-vlst-cntr > a:focus {
               outline: 1px solid #000000;
               outline-offset: 1px;
            }#onetrust-pc-sdk .ot-vlst-cntr .ot-ext-lnk,  #onetrust-pc-sdk .ot-ven-hdr .ot-ext-lnk{
                    background-image: url('https://cdn.cookielaw.org/logos/static/ot_external_link.svg');
                }
            /*! v6.10.0 2021-01-18 */
div#onetrust-consent-sdk #onetrust-banner-sdk{border-top:2px solid #d0d3d4!important;box-shadow:none;padding:24px}div#onetrust-consent-sdk button{border-radius:0!important;box-shadow:none!important;box-sizing:border-box!important;font-size:20px!important;font-weight:400!important;letter-spacing:0!important;max-width:none!important;white-space:nowrap!important}div#onetrust-consent-sdk button:not(.ot-link-btn){background-color:#007dbc!important;border:2px solid #007dbc!important;color:#fff!important;height:48px!important;padding:0 1em!important;width:auto!important}div#onetrust-consent-sdk button:hover{background-color:#fff!important;border-color:#949494!important;color:#2e2e2e!important}div#onetrust-consent-sdk button.ot-link-btn{color:#007dbc!important;font-size:16px!important;text-decoration:underline}div#onetrust-consent-sdk button.ot-link-btn:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-consent-sdk a,div#onetrust-pc-sdk a{color:#007dbc!important;text-decoration:underline!important}div#onetrust-consent-sdk a,div#onetrust-consent-sdk button,div#onetrust-consent-sdk p:hover{opacity:1!important}div#onetrust-consent-sdk a:focus,div#onetrust-consent-sdk button:focus,div#onetrust-consent-sdk input:focus{outline:2px solid #949494!important;outline-offset:1px!important}div#onetrust-banner-sdk .ot-sdk-container{padding:0;width:auto}div#onetrust-banner-sdk .ot-sdk-row{align-items:flex-start;box-sizing:border-box;display:flex;flex-direction:column;justify-content:space-between;margin:auto;max-width:1152px}div#onetrust-banner-sdk .ot-sdk-row:after{display:none}div#onetrust-banner-sdk #onetrust-group-container,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-group-container,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{flex-grow:1;width:auto}div#onetrust-banner-sdk #onetrust-policy,div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy{margin:0;overflow:visible}div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy-text,div#onetrust-consent-sdk #onetrust-policy-text{font-size:16px;line-height:24px;max-width:44em;margin:0}div#onetrust-consent-sdk #onetrust-policy-text a[href]{font-weight:400;margin-left:8px}div#onetrust-banner-sdk #onetrust-button-group-parent{flex:0 0 auto;margin:32px 0 0;width:100%}div#onetrust-banner-sdk #onetrust-button-group{display:flex;flex-direction:row;flex-wrap:wrap;justify-content:flex-end;margin:-8px}div#onetrust-banner-sdk .banner-actions-container{display:flex;flex:1 0 auto}div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button:last-of-type,div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto;margin:8px;width:auto}div#onetrust-consent-sdk #onetrust-pc-btn-handler{background-color:#fff!important;color:inherit!important}div#onetrust-banner-sdk #onetrust-close-btn-container{display:none}@media only screen and (min-width:556px){div#onetrust-consent-sdk #onetrust-banner-sdk{padding:40px}div#onetrust-banner-sdk #onetrust-policy{margin:0 40px 0 0}div#onetrust-banner-sdk .ot-sdk-row{align-items:center;flex-direction:row}div#onetrust-banner-sdk #onetrust-button-group-parent,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-button-group-parent,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{margin:0;padding:0;width:auto}div#onetrust-banner-sdk #onetrust-button-group,div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group{align-items:stretch;flex-direction:column-reverse;margin:0}div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto}}@media only screen and (min-width:768px){div#onetrust-banner-sdk #onetrust-policy{margin:0 48px 0 0}div#onetrust-consent-sdk #onetrust-banner-sdk{padding:48px}}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-size:16px;line-height:24px}div#onetrust-consent-sdk #onetrust-pc-sdk p,div#onetrust-pc-sdk #ot-pc-desc,div#onetrust-pc-sdk .category-host-list-handler,div#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{font-size:16px;font-weight:400;line-height:24px}div#onetrust-consent-sdk a:hover,div#onetrust-pc-sdk a:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-pc-sdk{border-radius:0;bottom:0;height:auto;left:0;margin:auto;max-width:100%;overflow:hidden;right:0;top:0;width:512px;max-height:800px}div#onetrust-pc-sdk .ot-pc-header{display:none}div#onetrust-pc-sdk #ot-pc-content{overscroll-behavior:contain;padding:0 12px 0 24px;margin:16px 4px 0 0;top:0;right:16px;left:0;width:auto}div#onetrust-pc-sdk #ot-category-title,div#onetrust-pc-sdk #ot-pc-title{font-size:24px;font-weight:400;line-height:32px;margin:0 0 16px}div#onetrust-pc-sdk #ot-pc-desc{padding:0}div#onetrust-pc-sdk #ot-pc-desc a{display:inline}div#onetrust-pc-sdk #accept-recommended-btn-handler{display:none!important}div#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:2px solid #007dbc;outline-offset:-1px;transition:none}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item{border-width:0 0 2px}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item:first-of-type{border-width:2px 0}div#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{padding:8px 0;width:100%}div#onetrust-pc-sdk .ot-plus-minus{transform:translateY(2px)}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item>button{background:0 0!important;border:0!important;height:44px!important;max-width:none!important;width:calc(100% - 48px)!important}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-weight:700}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr{padding:0}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item .ot-acc-grpdesc{padding:0;width:100%}div#onetrust-pc-sdk .ot-acc-grpcntr .ot-subgrp-cntr{border:0;padding:0}div#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp{margin:0}div#onetrust-pc-sdk .ot-always-active-group .ot-cat-header{width:calc(100% - 160px)}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{width:calc(100% - 88px)}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active{color:inherit;font-size:12px;font-weight:400;line-height:1.5;padding-right:48px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:before{border-radius:12px;position:absolute;right:0;top:0;content:'';background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:after{border-radius:50%;position:absolute;right:5px;top:4px;content:'';background-color:#007dbc;height:12px;width:12px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active,div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl{right:2px}div#onetrust-pc-sdk .ot-switch{display:block;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob,div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob{background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob:before{background-color:#737373;height:8px;left:4px;top:4px;width:8px}div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob:before{background-color:#007dbc;height:12px;left:0;top:2px;width:12px}div#onetrust-pc-sdk .ot-tgl input:focus+.ot-switch .ot-switch-nob{box-shadow:0 0;outline:2px solid #007dbc!important;outline-offset:1px;transition:none}div#onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-grpcntr.ot-acc-txt{background-color:transparent;padding-left:3px}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr,div#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr{overflow:visible;width:100%}div#onetrust-pc-sdk .ot-pc-footer{border-top:0 solid}div#onetrust-pc-sdk .ot-btn-container{padding-top:24px;text-align:center}div#onetrust-pc-sdk .ot-pc-footer button{margin:8px 0;background-color:#fff}div#onetrust-pc-sdk .ot-pc-footer-logo{background-color:#fff}div#onetrust-pc-sdk #ot-lst-title span{font-size:24px;font-weight:400;line-height:32px}div#onetrust-pc-sdk #ot-host-lst .ot-host-desc,div#onetrust-pc-sdk #ot-host-lst .ot-host-expand,div#onetrust-pc-sdk #ot-host-lst .ot-host-name,div#onetrust-pc-sdk #ot-host-lst .ot-host-name a,div#onetrust-pc-sdk .back-btn-handler,div#onetrust-pc-sdk .ot-host-opt li>div div{font-size:16px;font-weight:400;line-height:24px}div#onetrust-pc-sdk #ot-pc-lst{top:0}div#onetrust-pc-sdk .back-btn-handler{text-decoration:none!important}div#onetrust-pc-sdk #filter-btn-handler:hover svg{filter:invert(1)}div#onetrust-pc-sdk .back-btn-handler svg{width:16px;height:16px}div#onetrust-pc-sdk .ot-host-item>button{background:0 0!important;border:0!important;height:66px!important;max-width:none!important;width:calc(100% - 5px)!important;transform:translate(2px,2px)}div#onetrust-pc-sdk .ot-host-item{border-bottom:2px solid #b9b9b9;padding:0}div#onetrust-pc-sdk .ot-host-item .ot-acc-hdr{margin:0 0 -6px;padding:8px 0}div#onetrust-pc-sdk ul li:first-child{border-top:2px solid #b9b9b9}div#onetrust-pc-sdk .ot-host-item .ot-plus-minus{margin:0 8px 0 0}div#onetrust-pc-sdk .ot-search-cntr{width:calc(100% - 48px)}div#onetrust-pc-sdk .ot-host-opt .ot-host-info{background-color:transparent}div#onetrust-pc-sdk .ot-host-opt li>div div{padding:0}div#onetrust-pc-sdk #vendor-search-handler{border-radius:0;border-color:#939393;border-style:solid;border-width:2px 0 2px 2px;font-size:20px;height:48px;margin:0}div#onetrust-pc-sdk .ot-lst-subhdr svg{right:0;top:8px}div#onetrust-pc-sdk .ot-fltr-cntr{box-sizing:border-box;right:0;width:48px}div#onetrust-pc-sdk #filter-btn-handler{width:48px!important;padding:8px!important}div#onetrust-pc-sdk #ot-fltr-modal{max-height:400px}
.ot-sdk-cookie-policy{font-family:inherit;font-size:16px}.ot-sdk-cookie-policy.otRelFont{font-size:1rem}.ot-sdk-cookie-policy h3,.ot-sdk-cookie-policy h4,.ot-sdk-cookie-policy h6,.ot-sdk-cookie-policy p,.ot-sdk-cookie-policy li,.ot-sdk-cookie-policy a,.ot-sdk-cookie-policy th,.ot-sdk-cookie-policy #cookie-policy-description,.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group,.ot-sdk-cookie-policy #cookie-policy-title{color:dimgray}.ot-sdk-cookie-policy #cookie-policy-description{margin-bottom:1em}.ot-sdk-cookie-policy h4{font-size:1.2em}.ot-sdk-cookie-policy h6{font-size:1em;margin-top:2em}.ot-sdk-cookie-policy th{min-width:75px}.ot-sdk-cookie-policy a,.ot-sdk-cookie-policy a:hover{background:#fff}.ot-sdk-cookie-policy thead{background-color:#f6f6f4;font-weight:bold}.ot-sdk-cookie-policy .ot-mobile-border{display:none}.ot-sdk-cookie-policy section{margin-bottom:2em}.ot-sdk-cookie-policy table{border-collapse:inherit}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy{font-family:inherit;font-size:1rem}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h3,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h4,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h6,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy p,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy li,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-title{color:dimgray}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description{margin-bottom:1em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-subgroup{margin-left:1.5em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group-desc,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-table-header,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy span,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td{font-size:.9em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td span,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td a{font-size:inherit}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group{font-size:1em;margin-bottom:.6em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-title{margin-bottom:1.2em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy>section{margin-bottom:1em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th{min-width:75px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a:hover{background:#fff}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy thead{background-color:#f6f6f4;font-weight:bold}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-mobile-border{display:none}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy section{margin-bottom:2em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-subgroup ul li{list-style:disc;margin-left:1.5em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-subgroup ul li h4{display:inline-block}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table{border-collapse:inherit;margin:auto;border:1px solid #d7d7d7;border-radius:5px;border-spacing:initial;width:100%;overflow:hidden}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table th,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table td{border-bottom:1px solid #d7d7d7;border-right:1px solid #d7d7d7}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr:last-child td{border-bottom:0px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr th:last-child,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr td:last-child{border-right:0px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-host,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-cookies-type{width:25%}.ot-sdk-cookie-policy[dir=rtl]{text-align:left}#ot-sdk-cookie-policy h3{font-size:1.5em}@media only screen and (max-width: 530px){.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) table,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) thead,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tbody,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) th,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr{display:block}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) thead tr{position:absolute;top:-9999px;left:-9999px}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr{margin:0 0 1em 0}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr:nth-child(odd),.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr:nth-child(odd) a{background:#f6f6f4}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td{border:none;border-bottom:1px solid #eee;position:relative;padding-left:50%}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td:before{position:absolute;height:100%;left:6px;width:40%;padding-right:10px}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) .ot-mobile-border{display:inline-block;background-color:#e4e4e4;position:absolute;height:100%;top:0;left:45%;width:2px}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td:before{content:attr(data-label);font-weight:bold}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) li{word-break:break-word;word-wrap:break-word}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table{overflow:hidden}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table td{border:none;border-bottom:1px solid #d7d7d7}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy thead,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy tbody,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy tr{display:block}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-host,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-cookies-type{width:auto}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy tr{margin:0 0 1em 0}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td:before{height:100%;width:40%;padding-right:10px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td:before{content:attr(data-label);font-weight:bold}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy li{word-break:break-word;word-wrap:break-word}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy thead tr{position:absolute;top:-9999px;left:-9999px;z-index:-9999}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr:last-child td{border-bottom:1px solid #d7d7d7;border-right:0px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr:last-child td:last-child{border-bottom:0px}}
                
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h5,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h6,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy li,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy p,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy span,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description {
                        color: #696969;
                    }
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th {
                        color: #696969;
                    }
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group {
                        color: #696969;
                    }
                    
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-title {
                            color: #696969;
                        }
                    
            
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table th {
                            background-color: #F8F8F8;
                        }
                    
            .ot-floating-button__front{background-image:url('https://cdn.cookielaw.org/logos/static/ot_persistent_cookie_icon.png')}</style><script src="https://static.addtoany.com/menu/modules/core.pt5ow5lr.js" type="module"></script><style type="text/css">.a2a_hide{display:none}.a2a_logo_color{background-color:#0166ff}.a2a_menu,.a2a_menu *{-moz-box-sizing:content-box;-webkit-box-sizing:content-box;box-sizing:content-box;float:none;margin:0;padding:0;position:static;height:auto;width:auto}.a2a_menu{border-radius:6px;display:none;direction:ltr;background:#FFF;font:16px sans-serif-light,HelveticaNeue-Light,"Helvetica Neue Light","Helvetica Neue",Arial,Helvetica,"Liberation Sans",sans-serif;color:#000;line-height:12px;border:1px solid #CCC;vertical-align:baseline;overflow:hidden}.a2a_mini{min-width:200px;position:absolute;width:300px;z-index:9999997}.a2a_overlay{display:none;background:#616c7deb;backdrop-filter:blur(4px);-webkit-backdrop-filter:blur(4px);position:fixed;top:0;right:0;left:0;bottom:0;z-index:9999998;-webkit-tap-highlight-color:transparent;transition:opacity .14s,backdrop-filter .14s}.a2a_full{background:#FFF;border:1px solid #FFF;box-shadow:#2a2a2a1a 0 0 20px 10px;height:auto;height:calc(320px);top:15%;left:50%;margin-left:-320px;position:fixed;text-align:center;width:640px;z-index:9999999;transition:transform .14s,opacity .14s}.a2a_full_footer,.a2a_full_header,.a2a_full_services{border:0;margin:0;padding:12px;box-sizing:border-box}.a2a_full_header{padding-bottom:8px}.a2a_full_services{height:280px;overflow-y:scroll;padding:0 12px;-webkit-overflow-scrolling:touch}.a2a_full_services .a2a_i{display:inline-block;float:none;width:181px;width:calc(33.334% - 18px)}div.a2a_full_footer{font-size:12px;text-align:center;padding:8px 14px}div.a2a_full_footer a,div.a2a_full_footer a:visited{display:inline;font-size:12px;line-height:14px;padding:8px 14px}div.a2a_full_footer a:focus,div.a2a_full_footer a:hover{background:0 0;border:0;color:#0166FF}div.a2a_full_footer a span.a2a_s_a2a,div.a2a_full_footer a span.a2a_w_a2a{background-size:14px;border-radius:3px;display:inline-block;height:14px;line-height:14px;margin:0 3px 0 0;vertical-align:top;width:14px}.a2a_modal{height:0;left:50%;margin-left:-320px;position:fixed;text-align:center;top:15%;width:640px;z-index:9999999;transition:transform .14s,opacity .14s;-webkit-tap-highlight-color:transparent}.a2a_modal_body{background:0 0;border:0;font:24px sans-serif-light,HelveticaNeue-Light,"Helvetica Neue Light","Helvetica Neue",Arial,Helvetica,"Liberation Sans",sans-serif;position:relative;height:auto;width:auto}.a2a_thanks{color:#fff;height:auto;margin-top:20px;width:auto}.a2a_thanks>div:first-child{margin:0 0 40px 0}.a2a_thanks div *{height:inherit}#a2a_copy_link{background:#FFF;border:1px solid #FFF;cursor:pointer;margin-top:15%}label.a2a_s_link#a2a_copy_link_icon,label.a2a_w_link#a2a_copy_link_icon{background-size:48px;border-radius:0;display:inline-block;height:48px;left:0;line-height:48px;margin:0 3px 0 0;position:absolute;vertical-align:top;width:48px}#a2a_modal input#a2a_copy_link_text{background-color:transparent;border:0;color:#2A2A2A;cursor:pointer;font:inherit;height:48px;left:62px;max-width:initial;min-height:auto;padding:0;position:relative;width:564px;width:calc(100% - 76px)}#a2a_copy_link_copied{background-color:#0166ff;color:#fff;display:none;font:inherit;font-size:16px;margin-top:1px;padding:3px 8px}@media (forced-colors:active){.a2a_color_buttons a,.a2a_svg{forced-color-adjust:none}}@media (prefers-color-scheme:dark){.a2a_menu a,.a2a_menu a.a2a_i,.a2a_menu a.a2a_i:visited,.a2a_menu a.a2a_more,i.a2a_i{border-color:#2a2a2a!important;color:#fff!important}.a2a_menu a.a2a_i:active,.a2a_menu a.a2a_i:focus,.a2a_menu a.a2a_i:hover,.a2a_menu a.a2a_more:active,.a2a_menu a.a2a_more:focus,.a2a_menu a.a2a_more:hover,.a2a_menu_find_container{border-color:#444!important;background-color:#444!important}.a2a_menu:not(.a2a_thanks){background-color:#2a2a2a;border-color:#2a2a2a}.a2a_menu_find{color:#fff!important}.a2a_menu label.a2a_s_find svg{background-color:transparent!important}.a2a_menu label.a2a_s_find svg path{fill:#fff!important}.a2a_full{box-shadow:#00000066 0 0 20px 10px}.a2a_overlay{background-color:#373737eb}}@media print{.a2a_floating_style,.a2a_menu,.a2a_overlay{visibility:hidden}}@keyframes a2aFadeIn{from{opacity:0}to{opacity:1}}.a2a_starting{opacity:0}.a2a_starting.a2a_full,.a2a_starting.a2a_modal{transform:scale(.8)}@media (max-width:639px){.a2a_full{border-radius:0;top:15%;left:0;margin-left:auto;width:100%}.a2a_modal{left:0;margin-left:10px;width:calc(100% - 20px)}}@media (min-width:318px) and (max-width:437px){.a2a_full .a2a_full_services .a2a_i{width:calc(50% - 18px)}}@media (max-width:317px){.a2a_full .a2a_full_services .a2a_i{width:calc(100% - 18px)}}@media (max-height:436px){.a2a_full{bottom:40px;height:auto;top:40px}}@media (max-height:550px){.a2a_modal{top:30px}}@media (max-height:360px){.a2a_modal{top:20px}.a2a_thanks>div:first-child{margin-bottom:20px}}.a2a_menu a{color:#0166FF;text-decoration:none;font:16px sans-serif-light,HelveticaNeue-Light,"Helvetica Neue Light","Helvetica Neue",Arial,Helvetica,"Liberation Sans",sans-serif;line-height:14px;height:auto;width:auto;outline:0}.a2a_menu a.a2a_i:visited,.a2a_menu a.a2a_more{color:#0166FF}.a2a_menu a.a2a_i:active,.a2a_menu a.a2a_i:focus,.a2a_menu a.a2a_i:hover,.a2a_menu a.a2a_more:active,.a2a_menu a.a2a_more:focus,.a2a_menu a.a2a_more:hover{color:#2A2A2A;border-color:#EEE;border-style:solid;background-color:#EEE;text-decoration:none}.a2a_menu label.a2a_s_find{background-size:24px;height:24px;left:8px;pointer-events:auto;position:absolute;top:7px;width:24px}.a2a_menu label.a2a_s_find svg{background-color:#FFF}.a2a_menu label.a2a_s_find svg path{fill:#CCC}#a2a_menu_container{display:inline-block}.a2a_menu_find_container{border:1px solid #CCC;border-radius:6px;padding:2px 24px 2px 0;position:relative;text-align:left}.a2a_cols_container .a2a_col1{overflow-x:hidden;overflow-y:auto;-webkit-overflow-scrolling:touch}#a2a_modal input,#a2a_modal input[type=text],.a2a_menu input,.a2a_menu input[type=text]{display:block;background-image:none;box-shadow:none;line-height:100%;margin:0;outline:0;overflow:hidden;padding:0;-moz-box-shadow:none;-webkit-box-shadow:none;-webkit-appearance:none}#a2afeed_find_container input,#a2afeed_find_container input[type=text],#a2apage_find_container input,#a2apage_find_container input[type=text]{background-color:transparent;border:0;box-sizing:content-box;color:#2A2A2A;float:none;font:inherit;font-size:16px;height:28px;line-height:20px;left:38px;outline:0;margin:0;max-width:initial;min-height:initial;padding:2px 0;position:relative;width:99%}.a2a_clear{clear:both}.a2a_svg{background-repeat:no-repeat;display:block;overflow:hidden;height:32px;line-height:32px;padding:0;pointer-events:none;width:32px}.a2a_svg svg{background-repeat:no-repeat;background-position:50% 50%;border:none;display:block;left:0;margin:0 auto;overflow:hidden;padding:0;position:relative;top:0;width:auto;height:auto}a.a2a_i,i.a2a_i{display:block;float:left;border:1px solid #FFF;line-height:24px;padding:6px 8px;text-align:left;white-space:nowrap;overflow:hidden;text-overflow:ellipsis;width:132px}a.a2a_i span,a.a2a_more span{display:inline-block;overflow:hidden;vertical-align:top}a.a2a_i .a2a_svg{margin:0 6px 0 0}a.a2a_i .a2a_svg,a.a2a_more .a2a_svg{background-size:24px;height:24px;line-height:24px;width:24px}a.a2a_sss:hover{border-left:1px solid #CCC}a.a2a_more{border-bottom:1px solid #FFF;border-left:0;border-right:0;line-height:24px;margin:6px 0 0;padding:6px;-webkit-touch-callout:none}a.a2a_more span{height:24px;margin:0 6px 0 0}.a2a_kit .a2a_svg{background-repeat:repeat}.a2a_default_style a:empty,.a2a_flex_style a:empty,.a2a_floating_style a:empty,.a2a_overlay_style a:empty{display:none}.a2a_color_buttons a,.a2a_floating_style a{text-decoration:none}.a2a_default_style:not(.a2a_flex_style) a{float:left;line-height:16px;padding:0 2px}.a2a_default_style a:hover .a2a_svg,.a2a_floating_style a:hover .a2a_svg,.a2a_overlay_style a:hover .a2a_svg svg{opacity:.7}.a2a_overlay_style.a2a_default_style a:hover .a2a_svg{opacity:1}.a2a_default_style .a2a_count,.a2a_default_style .a2a_svg,.a2a_floating_style .a2a_svg,.a2a_menu .a2a_svg,.a2a_vertical_style .a2a_count,.a2a_vertical_style .a2a_svg{border-radius:4px}.a2a_default_style .a2a_counter img,.a2a_default_style .a2a_dd,.a2a_default_style .a2a_svg{float:left}.a2a_default_style .a2a_img_text{margin-right:4px}.a2a_default_style .a2a_divider{border-left:1px solid #000;display:inline;float:left;height:16px;line-height:16px;margin:0 5px}.a2a_kit a{cursor:pointer;transition:none}.a2a_floating_style{background-color:#fff;border-radius:6px;position:fixed;z-index:9999995}.a2a_overlay_style{z-index:2147483647}.a2a_floating_style,.a2a_overlay_style{animation:a2aFadeIn .2s ease-in;padding:4px}.a2a_vertical_style:not(.a2a_flex_style) a{clear:left;display:block;overflow:hidden;padding:4px}.a2a_floating_style.a2a_default_style{bottom:0}.a2a_floating_style.a2a_default_style a,.a2a_overlay_style.a2a_default_style a{padding:4px}.a2a_count{background-color:#fff;border:1px solid #ccc;box-sizing:border-box;color:#2a2a2a;display:block;float:left;font:12px Arial,Helvetica,sans-serif;height:16px;margin-left:4px;position:relative;text-align:center;width:50px}.a2a_count:after,.a2a_count:before{border:solid transparent;border-width:4px 4px 4px 0;content:"";height:0;left:0;line-height:0;margin:-4px 0 0 -4px;position:absolute;top:50%;width:0}.a2a_count:before{border-right-color:#ccc}.a2a_count:after{border-right-color:#fff;margin-left:-3px}.a2a_count span{animation:a2aFadeIn .14s ease-in}.a2a_vertical_style .a2a_counter img{display:block}.a2a_vertical_style .a2a_count{float:none;margin-left:0;margin-top:6px}.a2a_vertical_style .a2a_count:after,.a2a_vertical_style .a2a_count:before{border:solid transparent;border-width:0 4px 4px 4px;content:"";height:0;left:50%;line-height:0;margin:-4px 0 0 -4px;position:absolute;top:0;width:0}.a2a_vertical_style .a2a_count:before{border-bottom-color:#ccc}.a2a_vertical_style .a2a_count:after{border-bottom-color:#fff;margin-top:-3px}.a2a_color_buttons .a2a_count,.a2a_color_buttons .a2a_count:after,.a2a_color_buttons .a2a_count:before,.a2a_color_buttons.a2a_vertical_style .a2a_count:after,.a2a_color_buttons.a2a_vertical_style .a2a_count:before{background-color:transparent;border:none;color:#fff;float:none;width:auto}.a2a_color_buttons.a2a_vertical_style .a2a_count{margin-top:0}.a2a_flex_style{display:flex;align-items:flex-start;gap:0}.a2a_default_style.a2a_flex_style{left:0;right:0;width:100%}.a2a_vertical_style.a2a_flex_style{flex-direction:column;top:0;bottom:0}.a2a_flex_style a{display:flex;justify-content:center;flex:1;padding:4px}.a2a_flex_style.a2a_vertical_style a{flex-direction:column}.a2a_floating_style.a2a_color_buttons,.a2a_floating_style.a2a_flex_style{border-radius:0;padding:0}.a2a_floating_style.a2a_default_style.a2a_flex_style{bottom:0}.a2a_kit.a2a_flex_style .a2a_counter img,.a2a_kit.a2a_flex_style .a2a_dd,.a2a_kit.a2a_flex_style .a2a_svg{float:none}.a2a_nowrap{white-space:nowrap}.a2a_note{margin:0 auto;padding:9px;font-size:12px;text-align:center}.a2a_note .a2a_note_note{margin:0;color:#2A2A2A}.a2a_wide a{display:block;margin-top:3px;border-top:1px solid #EEE;text-align:center}.a2a_label{position:absolute!important;clip-path:polygon(0px 0px,0px 0px,0px 0px);-webkit-clip-path:polygon(0px 0px,0px 0px,0px 0px);overflow:hidden;height:1px;width:1px}.a2a_kit,.a2a_menu,.a2a_modal,.a2a_overlay{-ms-touch-action:manipulation;touch-action:manipulation;outline:0}.a2a_dd{-webkit-user-drag:none}.a2a_dd img{border:0}.a2a_button_facebook_like iframe{max-width:none}</style><style></style><style>stream {
  position: relative;
  display: block;
}
</style><script type="text/javascript" src="https://www.cell.com/products/marlin/cell/releasedAssets/public/mathjax-511ffcc670c4d93d695bf67802ba7aad/tex-mml-chtml.js" async=""></script><link href="https://crossmark-cdn.crossref.org/widget/v2.0/style.css" type="text/css" rel="stylesheet"><style type="text/css">.CtxtMenu_InfoClose {  top:.2em; right:.2em;}
.CtxtMenu_InfoContent {  overflow:auto; text-align:left; font-size:80%;  padding:.4em .6em; border:1px inset; margin:1em 0px;  max-height:20em; max-width:30em; background-color:#EEEEEE;  white-space:normal;}
.CtxtMenu_Info.CtxtMenu_MousePost {outline:none;}
.CtxtMenu_Info {  position:fixed; left:50%; width:auto; text-align:center;  border:3px outset; padding:1em 2em; background-color:#DDDDDD;  color:black;  cursor:default; font-family:message-box; font-size:120%;  font-style:normal; text-indent:0; text-transform:none;  line-height:normal; letter-spacing:normal; word-spacing:normal;  word-wrap:normal; white-space:nowrap; float:none; z-index:201;  border-radius: 15px;                     /* Opera 10.5 and IE9 */  -webkit-border-radius:15px;               /* Safari and Chrome */  -moz-border-radius:15px;                  /* Firefox */  -khtml-border-radius:15px;                /* Konqueror */  box-shadow:0px 10px 20px #808080;         /* Opera 10.5 and IE9 */  -webkit-box-shadow:0px 10px 20px #808080; /* Safari 3 & Chrome */  -moz-box-shadow:0px 10px 20px #808080;    /* Forefox 3.5 */  -khtml-box-shadow:0px 10px 20px #808080;  /* Konqueror */  filter:progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color="gray", Positive="true"); /* IE */}
</style><style type="text/css">.CtxtMenu_MenuClose {  position:absolute;  cursor:pointer;  display:inline-block;  border:2px solid #AAA;  border-radius:18px;  -webkit-border-radius: 18px;             /* Safari and Chrome */  -moz-border-radius: 18px;                /* Firefox */  -khtml-border-radius: 18px;              /* Konqueror */  font-family: "Courier New", Courier;  font-size:24px;  color:#F0F0F0}
.CtxtMenu_MenuClose span {  display:block; background-color:#AAA; border:1.5px solid;  border-radius:18px;  -webkit-border-radius: 18px;             /* Safari and Chrome */  -moz-border-radius: 18px;                /* Firefox */  -khtml-border-radius: 18px;              /* Konqueror */  line-height:0;  padding:8px 0 6px     /* may need to be browser-specific */}
.CtxtMenu_MenuClose:hover {  color:white!important;  border:2px solid #CCC!important}
.CtxtMenu_MenuClose:hover span {  background-color:#CCC!important}
.CtxtMenu_MenuClose:hover:focus {  outline:none}
</style><style type="text/css">.CtxtMenu_Menu {  position:absolute;  background-color:white;  color:black;  width:auto; padding:5px 0px;  border:1px solid #CCCCCC; margin:0; cursor:default;  font: menu; text-align:left; text-indent:0; text-transform:none;  line-height:normal; letter-spacing:normal; word-spacing:normal;  word-wrap:normal; white-space:nowrap; float:none; z-index:201;  border-radius: 5px;                     /* Opera 10.5 and IE9 */  -webkit-border-radius: 5px;             /* Safari and Chrome */  -moz-border-radius: 5px;                /* Firefox */  -khtml-border-radius: 5px;              /* Konqueror */  box-shadow:0px 10px 20px #808080;         /* Opera 10.5 and IE9 */  -webkit-box-shadow:0px 10px 20px #808080; /* Safari 3 & Chrome */  -moz-box-shadow:0px 10px 20px #808080;    /* Forefox 3.5 */  -khtml-box-shadow:0px 10px 20px #808080;  /* Konqueror */}
.CtxtMenu_MenuItem {  padding: 1px 2em;  background:transparent;}
.CtxtMenu_MenuArrow {  position:absolute; right:.5em; padding-top:.25em; color:#666666;  font-family: null; font-size: .75em}
.CtxtMenu_MenuActive .CtxtMenu_MenuArrow {color:white}
.CtxtMenu_MenuArrow.CtxtMenu_RTL {left:.5em; right:auto}
.CtxtMenu_MenuCheck {  position:absolute; left:.7em;  font-family: null}
.CtxtMenu_MenuCheck.CtxtMenu_RTL { right:.7em; left:auto }
.CtxtMenu_MenuRadioCheck {  position:absolute; left: .7em;}
.CtxtMenu_MenuRadioCheck.CtxtMenu_RTL {  right: .7em; left:auto}
.CtxtMenu_MenuInputBox {  padding-left: 1em; right:.5em; color:#666666;  font-family: null;}
.CtxtMenu_MenuInputBox.CtxtMenu_RTL {  left: .1em;}
.CtxtMenu_MenuComboBox {  left:.1em; padding-bottom:.5em;}
.CtxtMenu_MenuSlider {  left: .1em;}
.CtxtMenu_SliderValue {  position:absolute; right:.1em; padding-top:.25em; color:#333333;  font-size: .75em}
.CtxtMenu_SliderBar {  outline: none; background: #d3d3d3}
.CtxtMenu_MenuLabel {  padding: 1px 2em 3px 1.33em;  font-style:italic}
.CtxtMenu_MenuRule {  border-top: 1px solid #DDDDDD;  margin: 4px 3px;}
.CtxtMenu_MenuDisabled {  color:GrayText}
.CtxtMenu_MenuActive {  background-color: #606872;  color: white;}
.CtxtMenu_MenuDisabled:focus {  background-color: #E8E8E8}
.CtxtMenu_MenuLabel:focus {  background-color: #E8E8E8}
.CtxtMenu_ContextMenu:focus {  outline:none}
.CtxtMenu_ContextMenu .CtxtMenu_MenuItem:focus {  outline:none}
.CtxtMenu_SelectionMenu {  position:relative; float:left;  border-bottom: none; -webkit-box-shadow:none; -webkit-border-radius:0px; }
.CtxtMenu_SelectionItem {  padding-right: 1em;}
.CtxtMenu_Selection {  right: 40%; width:50%; }
.CtxtMenu_SelectionBox {  padding: 0em; max-height:20em; max-width: none;  background-color:#FFFFFF;}
.CtxtMenu_SelectionDivider {  clear: both; border-top: 2px solid #000000;}
.CtxtMenu_Menu .CtxtMenu_MenuClose {  top:-10px; left:-10px}
</style><style id="MJX-CHTML-styles">
mjx-container[overflow="scroll"][display] {
  overflow-x: auto;
  min-width: initial !important;
}

mjx-container[overflow="truncate"][display] {
  overflow-x: hidden;
  min-width: initial !important;
}

mjx-container[jax="CHTML"] {
  white-space: nowrap;
}

mjx-container [space="1"] {
  margin-left: .111em;
}

mjx-container [space="2"] {
  margin-left: .167em;
}

mjx-container [space="3"] {
  margin-left: .222em;
}

mjx-container [space="4"] {
  margin-left: .278em;
}

mjx-container [space="5"] {
  margin-left: .333em;
}

mjx-container [rspace="1"] {
  margin-right: .111em;
}

mjx-container [rspace="2"] {
  margin-right: .167em;
}

mjx-container [rspace="3"] {
  margin-right: .222em;
}

mjx-container [rspace="4"] {
  margin-right: .278em;
}

mjx-container [rspace="5"] {
  margin-right: .333em;
}

mjx-container [size="s"] {
  font-size: 70.7%;
}

mjx-container [size="ss"] {
  font-size: 50%;
}

mjx-container [size="Tn"] {
  font-size: 60%;
}

mjx-container [size="sm"] {
  font-size: 85%;
}

mjx-container [size="lg"] {
  font-size: 120%;
}

mjx-container [size="Lg"] {
  font-size: 144%;
}

mjx-container [size="LG"] {
  font-size: 173%;
}

mjx-container [size="hg"] {
  font-size: 207%;
}

mjx-container [size="HG"] {
  font-size: 249%;
}

mjx-container [width="full"] {
  width: 100%;
}

mjx-box {
  display: inline-block;
}

mjx-block {
  display: block;
}

mjx-itable {
  display: inline-table;
}

mjx-row {
  display: table-row;
}

mjx-row > * {
  display: table-cell;
}

mjx-container [inline-breaks] {
  display: inline;
}

mjx-mtext {
  display: inline-block;
}

mjx-mstyle {
  display: inline-block;
}

mjx-merror {
  display: inline-block;
  color: red;
  background-color: yellow;
}

mjx-mphantom {
  visibility: hidden;
}

mjx-math {
  display: inline-block;
  text-align: left;
  line-height: 0;
  text-indent: 0;
  font-style: normal;
  font-weight: normal;
  font-size: 100%;
  font-size-adjust: none;
  letter-spacing: normal;
  word-wrap: normal;
  word-spacing: normal;
  direction: ltr;
  padding: 1px 0;
}

mjx-container[jax="CHTML"][display="true"] {
  display: block;
  text-align: center;
  justify-content: center;
  margin: 1em 0;
}

mjx-container[jax="CHTML"][display="true"][width="full"] {
  display: flex;
}

mjx-container[jax="CHTML"][display="true"] mjx-math {
  padding: 0;
}

mjx-container[jax="CHTML"][justify="left"] {
  text-align: left;
  justify-content: left;
}

mjx-container[jax="CHTML"][justify="right"] {
  text-align: right;
  justify-content: right;
}

mjx-container[jax="CHTML"] mjx-break::after {
  content: " ";
  white-space: normal;
  font-family: MJX-BRK;
}

mjx-break[size="0"] {
  letter-spacing: -0.999em;
}

mjx-break[size="1"] {
  letter-spacing: -0.889em;
}

mjx-break[size="2"] {
  letter-spacing: -0.833em;
}

mjx-break[size="3"] {
  letter-spacing: -0.778em;
}

mjx-break[size="4"] {
  letter-spacing: -0.722em;
}

mjx-break[size="5"] {
  letter-spacing: -0.667em;
}

mjx-math[breakable] {
  display: inline;
}

mjx-mrow {
  display: inline-block;
  text-align: left;
}

mjx-linestack, mjx-mrow[break-top] {
  display: inline-table;
  width: 100%;
}

mjx-linestack[break-align="bottom"], mjx-mrow[break-top][break-align="bottom"] {
  display: inline-block;
}

mjx-linestack[break-align="middle"], mjx-mrow[break-top][break-align="middle"] {
  vertical-align: middle;
}

mjx-linestack[break-align="center"], mjx-mrow[break-top][break-align="center"] {
  vertical-align: middle;
}

mjx-linestack[breakable] {
  display: inline;
}

mjx-linestack[breakable] > mjx-linebox {
  display: inline;
}

mjx-linestack[breakable] > mjx-linebox::before {
  white-space: pre;
  content: "\A";
}

mjx-linestack[breakable] > mjx-linebox::after {
  white-space: normal;
  content: " ";
  letter-spacing: -.999em;
  font-family: MJX-BRK;
}

mjx-linestack[breakable] > mjx-linebox:first-of-type::before {
  display: none;
}

mjx-linestack[breakable] > mjx-linebox:last-of-type::after {
  display: none;
}

mjx-linebox {
  display: block;
}

mjx-linebox[align="left"] {
  text-align: left;
}

mjx-linebox[align="center"] {
  text-align: center;
}

mjx-linebox[align="right"] {
  text-align: right;
}

mjx-linestrut {
  display: inline-block;
  height: 1em;
  vertical-align: -.25em;
}

mjx-msup {
  display: inline-block;
  text-align: left;
}

mjx-msubsup {
  display: inline-block;
  text-align: left;
}

mjx-script {
  display: inline-block;
  padding-right: .05em;
  padding-left: .033em;
}

mjx-script > mjx-spacer {
  display: block;
}

mjx-mi {
  display: inline-block;
  text-align: left;
}

mjx-c {
  display: inline-block;
  width: 0;
  text-align: right;
}

mjx-utext {
  display: inline-block;
  padding: .75em 0 .2em 0;
}

mjx-mn {
  display: inline-block;
  text-align: left;
}

mjx-mo {
  display: inline-block;
  text-align: left;
}

mjx-stretchy-h {
  display: inline-block;
}

mjx-stretchy-h > * {
  display: inline-block;
  width: 0;
}

mjx-stretchy-h > mjx-ext {
  /* IE */ overflow: hidden;
  /* others */ overflow: clip visible;
  width: 100%;
  border: 0px solid transparent;
  box-sizing: border-box;
  text-align: center;
}

mjx-stretchy-h > mjx-ext > mjx-c {
  transform: scalex(500);
  width: 0;
}

mjx-stretchy-v {
  display: inline-block;
  text-align: center;
}

mjx-stretchy-v > * {
  display: block;
  height: 0;
}

mjx-stretchy-v > mjx-ext {
  /* IE */ overflow: hidden;
  /* others */ overflow: visible clip;
  height: 100%;
  border: 0px solid transparent;
  box-sizing: border-box;
}

mjx-mark {
  display: inline-block;
  height: 0;
}

mjx-msqrt {
  display: inline-block;
  text-align: left;
}

mjx-root {
  display: inline-block;
  white-space: nowrap;
}

mjx-surd {
  display: inline-block;
  vertical-align: top;
}

mjx-sqrt {
  display: inline-block;
  padding-top: .075em;
}

mjx-sqrt > mjx-box {
  border-top: .075em solid;
  padding-left: .03em;
  margin-left: -.03em;
}

mjx-sqrt.mjx-tall > mjx-box {
  padding-left: .3em;
  margin-left: -.3em;
}

mjx-mfrac {
  display: inline-block;
  text-align: left;
}

mjx-frac {
  display: inline-block;
  vertical-align: 0.17em;
  padding: 0 .22em;
}

mjx-frac[type="d"] {
  vertical-align: .04em;
}

mjx-frac[delims] {
  padding: 0 .1em;
}

mjx-frac[atop] {
  padding: 0 .12em;
}

mjx-frac[atop][delims] {
  padding: 0;
}

mjx-dtable {
  display: inline-table;
  width: 100%;
}

mjx-dtable > * {
  font-size: 2000%;
}

mjx-dbox {
  display: block;
  font-size: 5%;
}

mjx-num {
  display: block;
  text-align: center;
}

mjx-den {
  display: block;
  text-align: center;
}

mjx-mfrac[bevelled] > mjx-num {
  display: inline-block;
}

mjx-mfrac[bevelled] > mjx-den {
  display: inline-block;
}

mjx-den[align="right"], mjx-num[align="right"] {
  text-align: right;
}

mjx-den[align="left"], mjx-num[align="left"] {
  text-align: left;
}

mjx-nstrut {
  display: inline-block;
  height: .054em;
  width: 0;
  vertical-align: -.054em;
}

mjx-nstrut[type="d"] {
  height: .217em;
  vertical-align: -.217em;
}

mjx-dstrut {
  display: inline-block;
  height: .505em;
  width: 0;
}

mjx-dstrut[type="d"] {
  height: .726em;
}

mjx-line {
  display: block;
  box-sizing: border-box;
  min-height: 1px;
  height: .06em;
  border-top: .075em solid;
  margin: .06em -.1em;
  overflow: hidden;
}

mjx-line[type="d"] {
  margin: .18em -.1em;
}

mjx-msub {
  display: inline-block;
  text-align: left;
}

mjx-mspace {
  display: inline-block;
  text-align: left;
}

.MM-N {
  font-family: MJX-MM-ZERO, MJX-MM-N;
}

.MM-B {
  font-family: MJX-MM-ZERO, MJX-MM-B;
}

.MM-I {
  font-family: MJX-MM-ZERO, MJX-MM-I;
}

.MM-BI {
  font-family: MJX-MM-ZERO, MJX-MM-BI;
}

.MM-DS {
  font-family: MJX-MM-ZERO, MJX-MM-DS;
}

.MM-F {
  font-family: MJX-MM-ZERO, MJX-MM-F;
}

.MM-FB {
  font-family: MJX-MM-ZERO, MJX-MM-FB;
}

.MM-S {
  font-family: MJX-MM-ZERO, MJX-MM-S;
}

.MM-SB {
  font-family: MJX-MM-ZERO, MJX-MM-SB;
}

.MM-SS {
  font-family: MJX-MM-ZERO, MJX-MM-SS;
}

.MM-SSB {
  font-family: MJX-MM-ZERO, MJX-MM-SSB;
}

.MM-SSI {
  font-family: MJX-MM-ZERO, MJX-MM-SSI;
}

.MM-SSBI {
  font-family: MJX-MM-ZERO, MJX-MM-SSBI;
}

.MM-M {
  font-family: MJX-MM-ZERO, MJX-MM-M;
}

.MM-SO {
  font-family: MJX-MM-ZERO, MJX-MM-SO;
}

.MM-LO {
  font-family: MJX-MM-ZERO, MJX-MM-LO;
}

.MM-S3 {
  font-family: MJX-MM-ZERO, MJX-MM-S3;
}

.MM-S4 {
  font-family: MJX-MM-ZERO, MJX-MM-S4;
}

.MM-S5 {
  font-family: MJX-MM-ZERO, MJX-MM-S5;
}

.MM-S6 {
  font-family: MJX-MM-ZERO, MJX-MM-S6;
}

.MM-S7 {
  font-family: MJX-MM-ZERO, MJX-MM-S7;
}

.MM-MI {
  font-family: MJX-MM-ZERO, MJX-MM-MI;
}

.MM-C {
  font-family: MJX-MM-ZERO, MJX-MM-C;
}

.MM-CB {
  font-family: MJX-MM-ZERO, MJX-MM-CB;
}

.MM-OS {
  font-family: MJX-MM-ZERO, MJX-MM-OS;
}

.MM-OB {
  font-family: MJX-MM-ZERO, MJX-MM-OB;
}

.MM-V {
  font-family: MJX-MM-ZERO, MJX-MM-V;
}

.MM-LT {
  font-family: MJX-MM-ZERO, MJX-MM-LT;
}

.MM-RB {
  font-family: MJX-MM-ZERO, MJX-MM-RB;
}

.MM-EM {
  font-family: MJX-MM-ZERO, MJX-MM-EM;
}

@font-face /* MJX-MM-ZERO */ {
  font-family: MJX-MM-ZERO;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-zero.woff") format("woff");
}

@font-face /* MJX-BRK */ {
  font-family: MJX-BRK;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-brk.woff") format("woff");
}

@font-face /* MJX-MM-N */ {
  font-family: MJX-MM-N;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-n.woff") format("woff");
}

@font-face /* MJX-MM-B */ {
  font-family: MJX-MM-B;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-b.woff") format("woff");
}

@font-face /* MJX-MM-I */ {
  font-family: MJX-MM-I;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-i.woff") format("woff");
}

@font-face /* MJX-MM-BI */ {
  font-family: MJX-MM-BI;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-bi.woff") format("woff");
}

@font-face /* MJX-MM-DS */ {
  font-family: MJX-MM-DS;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-ds.woff") format("woff");
}

@font-face /* MJX-MM-F */ {
  font-family: MJX-MM-F;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-f.woff") format("woff");
}

@font-face /* MJX-MM-FB */ {
  font-family: MJX-MM-FB;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-fb.woff") format("woff");
}

@font-face /* MJX-MM-S */ {
  font-family: MJX-MM-S;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-s.woff") format("woff");
}

@font-face /* MJX-MM-SB */ {
  font-family: MJX-MM-SB;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-sb.woff") format("woff");
}

@font-face /* MJX-MM-SS */ {
  font-family: MJX-MM-SS;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-ss.woff") format("woff");
}

@font-face /* MJX-MM-SSB */ {
  font-family: MJX-MM-SSB;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-ssb.woff") format("woff");
}

@font-face /* MJX-MM-SSI */ {
  font-family: MJX-MM-SSI;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-ssi.woff") format("woff");
}

@font-face /* MJX-MM-SSBI */ {
  font-family: MJX-MM-SSBI;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-ssbi.woff") format("woff");
}

@font-face /* MJX-MM-M */ {
  font-family: MJX-MM-M;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-m.woff") format("woff");
}

@font-face /* MJX-MM-SO */ {
  font-family: MJX-MM-SO;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-so.woff") format("woff");
}

@font-face /* MJX-MM-LO */ {
  font-family: MJX-MM-LO;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-lo.woff") format("woff");
}

@font-face /* MJX-MM-S3 */ {
  font-family: MJX-MM-S3;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-s3.woff") format("woff");
}

@font-face /* MJX-MM-S4 */ {
  font-family: MJX-MM-S4;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-s4.woff") format("woff");
}

@font-face /* MJX-MM-S5 */ {
  font-family: MJX-MM-S5;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-s5.woff") format("woff");
}

@font-face /* MJX-MM-S6 */ {
  font-family: MJX-MM-S6;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-s6.woff") format("woff");
}

@font-face /* MJX-MM-S7 */ {
  font-family: MJX-MM-S7;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-s7.woff") format("woff");
}

@font-face /* MJX-MM-MI */ {
  font-family: MJX-MM-MI;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-mi.woff") format("woff");
}

@font-face /* MJX-MM-C */ {
  font-family: MJX-MM-C;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-c.woff") format("woff");
}

@font-face /* MJX-MM-CB */ {
  font-family: MJX-MM-CB;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-cb.woff") format("woff");
}

@font-face /* MJX-MM-OS */ {
  font-family: MJX-MM-OS;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-os.woff") format("woff");
}

@font-face /* MJX-MM-OB */ {
  font-family: MJX-MM-OB;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-ob.woff") format("woff");
}

@font-face /* MJX-MM-V */ {
  font-family: MJX-MM-V;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-v.woff") format("woff");
}

@font-face /* MJX-MM-LT */ {
  font-family: MJX-MM-LT;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-lt.woff") format("woff");
}

@font-face /* MJX-MM-RB */ {
  font-family: MJX-MM-RB;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-rb.woff") format("woff");
}

@font-face /* MJX-MM-EM */ {
  font-family: MJX-MM-EM;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-em.woff") format("woff");
}

mjx-c.mjx-c3C7 {
  padding: 0.442em 0.641em 0.205em 0;
}

mjx-c.mjx-c32 {
  padding: 0.666em 0.5em 0 0;
}

mjx-c.mjx-c52 {
  padding: 0.683em 0.736em 0.022em 0;
}

mjx-c.mjx-c2062 {
  padding: 0 0 0 0;
}

mjx-c.mjx-c4D {
  padding: 0.683em 0.917em 0 0;
}

mjx-c.mjx-c53 {
  padding: 0.705em 0.556em 0.022em 0;
}

mjx-c.mjx-c44 {
  padding: 0.683em 0.764em 0 0;
}

mjx-c.mjx-c2061 {
  padding: 0 0 0 0;
}

mjx-c.mjx-c28 {
  padding: 0.748em 0.389em 0.248em 0;
}

mjx-c.mjx-c74 {
  padding: 0.615em 0.389em 0.011em 0;
}

mjx-c.mjx-c31 {
  padding: 0.666em 0.5em 0 0;
}

mjx-c.mjx-c2C {
  padding: 0.106em 0.278em 0.193em 0;
}

mjx-c.mjx-c29 {
  padding: 0.748em 0.389em 0.248em 0;
}

mjx-c.mjx-c3D {
  padding: 0.367em 0.778em 0 0;
}

mjx-c.mjx-c221A.MM-LO {
  padding: 1.15em 1.02em 0.65em 0;
}

mjx-c.mjx-c5B.MM-S3 {
  padding: 0.975em 0.444em 0.475em 0;
}

mjx-c.mjx-c1D440 {
  padding: 0.683em 1.044em 0 0;
}

mjx-c.mjx-c2211 {
  padding: 0.75em 1.056em 0.25em 0;
}

mjx-c.mjx-c1D441 {
  padding: 0.683em 0.881em 0 0;
}

mjx-c.mjx-c1D456 {
  padding: 0.661em 0.345em 0.011em 0;
}

mjx-c.mjx-c1D45A {
  padding: 0.442em 0.878em 0.011em 0;
}

mjx-c.mjx-c2016 {
  padding: 0.75em 0.398em 0.25em 0;
}

mjx-c.mjx-c1D45F {
  padding: 0.442em 0.451em 0.011em 0;
}

mjx-c.mjx-c1D461 {
  padding: 0.626em 0.361em 0.011em 0;
}

mjx-c.mjx-c2212 {
  padding: 0.583em 0.778em 0.083em 0;
}

mjx-c.mjx-c5D.MM-S3 {
  padding: 0.975em 0.444em 0.475em 0;
}

mjx-c.mjx-c23 {
  padding: 0.694em 0.833em 0.194em 0;
}

mjx-c.mjx-c1D435 {
  padding: 0.683em 0.759em 0 0;
}

mjx-c.mjx-c1D45B {
  padding: 0.442em 0.6em 0.011em 0;
}

mjx-c.mjx-c1D460 {
  padding: 0.442em 0.469em 0.011em 0;
}

mjx-c.mjx-c1D43C {
  padding: 0.683em 0.497em 0 0;
}

mjx-c.mjx-c1D43A {
  padding: 0.705em 0.786em 0.022em 0;
}

mjx-c.mjx-c1D452 {
  padding: 0.442em 0.466em 0.011em 0;
}

mjx-c.mjx-c1D45C {
  padding: 0.442em 0.485em 0.011em 0;
}

mjx-c.mjx-cD7 {
  padding: 0.491em 0.778em 0 0;
}

mjx-c.mjx-c28.MM-S3 {
  padding: 0.972em 0.523em 0.472em 0;
}

mjx-c.mjx-c1D450 {
  padding: 0.442em 0.433em 0.011em 0;
}

mjx-c.mjx-c210E {
  padding: 0.694em 0.576em 0.011em 0;
}

mjx-c.mjx-c1D45D {
  padding: 0.442em 0.503em 0.194em 0;
}

mjx-c.mjx-c65 {
  padding: 0.448em 0.444em 0.011em 0;
}

mjx-c.mjx-c1D44E {
  padding: 0.442em 0.529em 0.011em 0;
}

mjx-c.mjx-c1D451 {
  padding: 0.694em 0.52em 0.011em 0;
}

mjx-c.mjx-c1D459 {
  padding: 0.694em 0.298em 0.011em 0;
}

mjx-c.mjx-c1D436 {
  padding: 0.705em 0.76em 0.022em 0;
}

mjx-c.mjx-c1D462 {
  padding: 0.442em 0.572em 0.011em 0;
}

mjx-c.mjx-c29.MM-S3 {
  padding: 0.972em 0.523em 0.472em 0;
}
</style></head>
<body class="pb-ui">
    <div class="skip">
        <a href="#page-body-id" id="skipNavigationLink" title="Skip to Main Content" tabindex="0" class="skipToContent">Skip to Main Content</a>
        <a href="#mega-menu-nav" id="skipToMainMenu" title="Skip to Main Menu" tabindex="0" class="skipToContent">Skip to Main Menu</a>
    </div>



    <div id="pb-page-content" data-ng-non-bindable="">
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="ux3-layout-widget" data-widget-id="832ad3f3-97ac-45ab-9af1-90c6f7bff54a" class="ux3">
        
    

        


    

            

            
                <div class="cell fonts-loaded">
            

            <div data-widget-id="832ad3f3-97ac-45ab-9af1-90c6f7bff54a" data-widget-name="ux3-layout-widget"><div>
    

        


    

            

            

            <div id="a1f1ba25-79af-4194-aaa9-66206104a800" class="widget loginPopup none widget-none widget-compact-all"><div class="wrapped"><div class="widget-body body body-none body-compact-all"><div role="dialog" aria-modal="true" aria-label="Login" class="popup login-popup hidden"><div class="content"><a href="#" aria-label="Close login popup" title="Close login popup" role="button" class="close"><i class="icon-close_thin"></i></a><h2 class="login-title">Login to your account</h2><div class="login-form"><p tabindex="-1" class="message error"></p><form action="/action/doLogin?redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2822%2900079-4" method="post"><input type="hidden" name="redirectUri" value="https://www.cell.com/cell/fulltext/S0092-8674(22)00079-4"><input type="hidden" name="referer" value="/cell/fulltext/S0092-8674(22)00079-4"><input type="hidden" name="ecommRedirect"><input type="hidden" name="code" value="cell-site"><input type="hidden" name="UX3LoginWidget" value="true"><input type="hidden" name="isPopup" value="true"><div class="input-group login-input--wrapper"><div class="label"><label for="login">Email/Username</label></div><input id="login" type="text" name="login" value="" size="15" class="login"><div role="alert" class="error-hint js--hidden"><div class="error-hint__text"><div class="error-text">Your email address is a required field.</div><div class="email-example">E.g., j.smith@mail.com</div></div></div></div><div class="input-group login-input--wrapper"><div class="label"><label for="password">Password</label></div><input id="password" type="password" name="password" value="" autocomplete="off" class="password"><a href="#" class="login__field__action js-show-password">Show</a><div role="alert" class="error-hint js--hidden"><span class="error-hint__text">Your password is a required field.</span></div><div class="actions"><a href="https://www.cell.com/action/requestResetPassword" class="link show-request-reset-password">Forgot password?</a></div></div><div class="remember"><input type="checkbox" name="savePassword" id="a1f1ba25-79af-4194-aaa9-66206104a800-remember" value="1" checked="checked"><label for="a1f1ba25-79af-4194-aaa9-66206104a800-remember">Remember me</label></div><div class="submit"><input type="submit" name="submit" value="Log in" class="button submit primary"></div></form></div><div class="create-account"><div>Don’t have an account?</div><a href="/action/registration?redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2822%2900079-4" class="show-registration">Create a Free Account</a></div></div></div></div></div></div>

            

        

    

        


    

            

            

            <div role="dialog" aria-modal="true" aria-label="Reset password" class="popup top-drawer request-reset-password-drawer hidden"><div class="content"><a href="#" aria-label="Close reset password popup" title="Close reset password popup" role="button" class="close"><i class="icon-close_thin"></i></a><div class="request-reset"><div class="form"><p class="sub">If you don't remember your password, you can reset it by entering your email address and clicking the Reset Password button. You will then receive an email that contains a secure link for resetting your password</p><div tabindex="-1" class="message error"></div><form action="/action/requestResetPassword" method="post" class="request-reset-password-form"><input type="hidden" name="platform"><input type="hidden" name="requestResetPassword" value="true"><input type="hidden" name="codeSite" value="cell-site"><div class="input-group"><div class="label"><label for="reset-password-email">Email*</label></div><input id="reset-password-email" type="text" name="email" value="" size="15" class="email"></div><div class="submit" disabled="disabled"><input type="submit" name="submit" value="Submit" disabled="disabled" class="button primary submit"></div></form></div><div class="success-template hidden"><p>If the address matches a valid account an email will be sent to __email__ with instructions for resetting your password</p></div></div><div class="request-reset-cancel"><a href="#" aria-label="Close reset password popup" title="Close reset password popup" class="cancel">Cancel</a></div></div></div>

            

        

    

        


    

            

            
                <div class="ux3-ads">
            

            <div class="pb-ad">
            <!-- Do Not Modify without contacting ELS-HSOnlineCampaign@elsevier.com -->
<!-- To be kept in sync with Global Commercial Publishing Operations-->
<!-- Start: GPT Async -->
<div id="gpt-Variables" data-adfile="CELLJOURNAL">
	<input type="hidden" id="gptSite" value="cell">
	<input type="hidden" id="gptLoggedIn" value="no">
	<input type="hidden" id="gptAccessType" value="ae:ANON_GUEST|ae:ANON_GUEST">
	<input type="hidden" id="gptDefaultSite" value="cell">
</div>
<div id="gpt-Interscroller" class="gpt-InScrollBox" style="display:none;">
	<div class="gpt-InScrollLabel">ADVERTISEMENT</div>
	<div class="gpt-InScrollContainer">
		<div class="gpt-InScrollWrapper">
			<div class="gpt-InScrollInner">
				<div class="gpt-adinterscroller" data-adsize="interscroller"></div>
			</div>
		</div>
	</div>
	<div class="gpt-InScrollLabel">SCROLL TO CONTINUE WITH CONTENT</div>
</div>
<div id="gpt-DebugDiv" class="gpt-DebugDiv" style="display:none;">
	<div style="width: 100%; display: table;">
		<div style="display: table-row">
			<div style="width: 200px; display: table-cell;">
				<button class="gpt-DebugBtn" onclick="gpt_ShowConsole()">Open GPT Console</button><br>
				<button id="gpt_GrapeShotDebugBtn" class="gpt-DebugBtn" onclick="gpt_GrapeShotDebug();">Open Oracle Keywords</button><br>
				<br>
				<button class="gpt-DebugBtn" onclick="gpt_LoadDebugValues()">Refresh Values</button>
			</div>
			<div style="display: table-cell;">
				<table class="gpt-DebugTable">
					<thead>
						<tr>
							<th>Property</th>
							<th>Value</th>
						</tr>
					</thead>
					<tbody>
						<tr><td class="gpt-DebugTitle">Status</td><td><div class="gptStatus_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Version</td><td><div class="gptVersion_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Ad File</td><td><div class="gptAdFile_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Disable Ads Flag</td><td><div class="gptDisableAds_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Environment</td><td><div class="gptTestENV_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Moat Init</td><td><div class="gptMoatInit_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Moat Ready</td><td><div class="gptMoatReady_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual Ready</td><td><div class="gptContextualReady_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual URL</td><td><div class="gptContextualURL_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual Initial Segments</td><td><div class="gptContextualInit_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual Used Segments</td><td><div class="gptContextualResult_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">AdUnit</td><td><div class="gptSite_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">SubAdUnit</td><td><div class="gptPage_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Custom Targeting</td><td><div class="gptCustomTargeting_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Ad Events</td><td><div class="gptEvent_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Invalid Ad Sizes</td><td><div class="gptInvalidAdSize_Debug"></div></td></tr>
					</tbody>
				</table>
			</div>
		</div>
	</div>
</div>
<style type="text/css">
	.gpt-DebugBtn {
		display: inline-block;
		text-align: center;
		text-decoration: none;
		margin: 2px 0;
		border: solid 1px #122b40;
		border-radius: 0.4em;
		padding: 0.5em 1em;
		color: #ffffff;
		background-color: #204d74;
		font-weight: normal;
		font-size: 12px;
	}

	.gpt-DebugDiv {
		margin: 5px;
	}

	.gpt-DebugTitle {
		font-weight: bold;
	}

	.gpt-DebugTable {
		border: 1px solid #dddddd;
		border-collapse: collapse;
		margin: 5px;
		font-size: 0.9em;
		font-family: sans-serif;
		min-width: 400px;
		margin-bottom: 25px;
	}

		.gpt-DebugTable td:first-child {
			padding: 5px;
		}

		.gpt-DebugTable thead tr th {
			background-color: #d9edf7;
			color: #ffffff;
			text-align: left;
			padding: 5px;
			border-bottom: 2px solid #204d74;
		}

		.gpt-DebugTable th,
		.gpt-DebugTable td {
			padding: 5px;
		}

		.gpt-DebugTable tbody tr {
			border-bottom: 1px solid #dddddd;
		}

			.gpt-DebugTable tbody tr:nth-of-type(even) {
				background-color: #f3f3f3;
			}

			.gpt-DebugTable tbody tr:last-of-type {
				border-bottom: 2px solid #204d74;
			}

	.gpt-InScrollLabel {
		background-color: rgb(0, 0, 0);
		color: rgb(255, 255, 255);
		position: relative;
		max-width: 100vw;
		left: 50%;
		right: 50%;
		margin-left: -50vw;
		margin-right: -50vw;
		padding: 0;
		text-align: center;
		width: 100vw !important;
		font-size: 10px;
	}

	.gpt-InScrollBox {
		height: 100vh !important;
		width: 100vw;
		cursor: pointer;
		max-width: 100vw;
		position: relative;
		left: 50%;
		right: 50%;
		margin-left: -50vw;
		margin-right: -50vw;
		margin-bottom: 50px;
		padding-top: 5px;
		padding-bottom: 5px;
		z-index: 10001;
		background-color: white;
	}

		.gpt-InScrollBox .gpt-InScrollContainer {
			height: 95vh !important;
			position: relative;
		}

		.gpt-InScrollBox .gpt-InScrollWrapper {
			position: absolute !important;
			top: 0 !important;
			left: 0;
			width: 100%;
			height: 95vh !important;
			border: 0 !important;
			margin: 0 !important;
			padding: 0 !important;
			clip: rect(0, auto, auto, 0) !important;
			-webkit-clip-path: polygon(0px 0px, 100% 0px, 100% 100%, 0px 100%) !important;
			clip-path: polygon(0px 0px, 100% 0px, 100% 100%, 0px 100%) !important;
		}

		.gpt-InScrollBox .gpt-InScrollInner {
			position: fixed !important;
			top: 0 !important;
			left: 0 !important;
			bottom: 0;
			margin: auto;
			width: 100vw;
			height: 95vh;
			-webkit-transform: translateZ(0) !important;
			display: -webkit-box;
			display: -ms-flexbox;
			display: flex;
			-webkit-box-orient: vertical;
			-webkit-box-direction: normal;
			-ms-flex-direction: column;
			flex-direction: column;
			-webkit-box-align: center;
			-ms-flex-align: center;
			align-items: center;
			-webkit-box-pack: center;
			-ms-flex-pack: center;
			justify-content: center;
			text-align: center;
		}
</style>
<script class="optanon-category-4" type="text/plain">
	//--------------------------------------------------------------------------
	var gpt_Version = '04072023';
	var gpt_AdFile = $('#gpt-Variables').data('adfile');
	var gpt_MaxMOATInitAttempts = 11;
	var gpt_MaxMOATReadyAttempts = 11;
	var gpt_MaxContextualReadyAttempts = 21;
	var gpt_MOATInitAttempts = 0;
	var gpt_MOATReadyAttempts = 0;
	var gpt_ContextualReadyAttempts = 0;
	var gpt_IsProd = true;
	var gpt_StartTime;
	var gs_channels = 'DEFAULT';
	var gpt_ContextualReady = false;
	var gpt_ContextualURL = 'Not enabled outside of Production';
	var gpt_DisableAdsFlagFound = false;

	var gptEvent_Debug = '';
	var gptCustomTargeting_Debug = '';
	var gptSite_Debug = "";
	var gptPage_Debug = "";
	var gptContextualEnabled = false;
	var gptContextualResult_Debug = '';
	var gptContextualInit_Debug = '';
	var gptContextualURL_Debug = '';
	var gptContextualReady_Debug = '';
	var gptMoatReady_Debug = '';
	var gptMoatInit_Debug = '';
	var gptTestENV_Debug = '';
	var gptDisableAds_Debug = '';
	var gptStatus_Debug = 'GPT Still Loading - Refresh Values';
	var gptInvalidAdSize_Debug = '';
	//--------------------------------------------------------------------------
	$(document).ready(function () {
		//Check to see if we are on a development env
		gpt_CheckEnvironment();

		if (!gpt_DisableAdsFlagFound) {
			gpt_StartTime = new Date();
			if (gpt_AdFile !== "PU") {
				//Once the dom has loaded, begin the process to init the ads.
				gpt_InitScripts();
				gpt_CheckForMOATInit();
			}
		}
	});
</script>
<script type="text/javascript">
	function gpt_CheckEnvironment() {
		if ($('#gptDisableAds').length > 0) {
			gptDisableAds_Debug = 'Flag Detected - Ads Disabled';
			gpt_DisableAdsFlagFound = true;
		}
		else {
			gptDisableAds_Debug = 'Ads Enabled';
			gpt_DisableAdsFlagFound = false;
		}
		switch (gpt_AdFile) {
			case 'PU':
				gptContextualEnabled = true;
				if ($("#Environment").val() !== "Prod") {
					gpt_IsProd = false;
					gptTestENV_Debug = 'Test Environment Detected';
				}
				else {
					gptTestENV_Debug = 'Production Environment Detected';
				}
				break;
			case 'JBS':
			case 'LANCET':
				gptContextualEnabled = true;
				if ($(".gpt-ad").length > 0) {
					//Other ads exist, allow the Prestitial
					$(document.body).append("<div class='gpt-ad' data-adsize='prestitial'></div>");
				}
				if (window.location.hostname.toLowerCase().indexOf('.literatumonline.') >= 0) {
					gpt_IsProd = false;
					gptTestENV_Debug = 'Test Environment Detected';
				}
				else {
					gptTestENV_Debug = 'Production Environment Detected';
				}
				break;
			default:
				//Cell
				gptContextualEnabled = false;
				if (window.location.hostname.toLowerCase().indexOf('.literatumonline.') >= 0) {
					gpt_IsProd = false;
					gptTestENV_Debug = 'Test Environment Detected';
				}
				else {
					gptTestENV_Debug = 'Production Environment Detected';
				}
				break;
		}
	}
	//--------------------------------------------------------------------------
    function gpt_GetNonCMEArticle() {
        var url;
        $('.articleCitation').each(function (i, obj) {
            if ($(obj).find('.toc__item__detials').length > 0) {
                var itemDetails = $(obj).first().find('.toc__item__detials');
                var cmeFound = 0;
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='CME']").length; }
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='(CME)']").length; }
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='Education']").length; }
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='education']").length; }
                if (cmeFound == 0) {
                    var itemLink = $(itemDetails).first().find('.toc__item__title');
                    var linkCount = $(itemLink).first().find("a:contains('CME')").length;
                    if (linkCount == 0) {
                        url = $(itemLink).first().find('a').attr('href');
                        return false;   //Found a URL, break the loop
                    }
                }
            }
        });
        return url;
    }
	//--------------------------------------------------------------------------
	function gpt_GetElapsedTime() {
		var endTime = new Date();
		var timeDiff = endTime - gpt_StartTime; //in ms
		// strip the ms
		timeDiff /= 1000;

		// get seconds
		return seconds = Math.round(timeDiff) + " seconds";
	}
	//--------------------------------------------------------------------------
	function gpt_CheckSSC() {
		//Check for ScreenShot Client
		var gpt_SSC = gpt_GetQuerystring('gpt_ssc', '');
		if (gpt_SSC == '1') {
			//it's the ScreenShotClient, disable MOAT IVT Detection
			$('#gpt-Variables').append('<input type="hidden" id="gptm_data">');
			$('#gptm_data').val(0);
			gpt_Target("m_data");
		}
	}
	//--------------------------------------------------------------------------
	function gpt_LocalSS(adTypeId) {
		window.scrollTo(0, 0);
		gpt_Sleep(1000);
		gpt_ScrollToAd(adTypeId);
		gpt_PreTearsheet(adTypeId);
	}
	//--------------------------------------------------------------------------
	function gpt_InitScripts() {
		if (gpt_AdFile == "PU") {
			if (gpt_IsProd) {
				gpt_ContextualURL = gpt_CheckForNULL($('#canonicalurl').val());
				if (gpt_ContextualURL === null) {
					gpt_ContextualURL = window.location.href.split('?')[0].split('#')[0];
				}
				else {
					gpt_ContextualURL = window.location.origin + gpt_ContextualURL;
				}
				var useSSL = 'https:' == document.location.protocol;
				var gs = document.createElement('script');
				gs.async = true;
				gs.type = 'text/javascript';
				gs.src = (useSSL ? 'https:' : 'http:') + '//elsevier.gscontxt.net/multizone/channels.cgi?url=' + encodeURIComponent(gpt_ContextualURL);
				gs.onload = function () {
					gpt_ContextualReady = true;
				};
				document.head.appendChild(gs);
			}
		}
		else {
			//JBS-Cell-Lancet
			var useSSL = 'https:' == document.location.protocol;

			var gads = document.createElement('script');
			gads.async = true;
			gads.type = 'text/javascript';
			gads.src = (useSSL ? 'https:' : 'http:') + '//securepubads.g.doubleclick.net/tag/js/gpt.js';
			document.head.appendChild(gads);

			if (gpt_IsProd && gptContextualEnabled) {
				gpt_ContextualURL = window.location.href.split('?')[0].split('#')[0];
				var gs = document.createElement('script');
				gs.async = true;
				gs.type = 'text/javascript';
				gs.src = (useSSL ? 'https:' : 'http:') + '//elsevier.gscontxt.net/multizone/channels.cgi?url=' + encodeURIComponent(gpt_ContextualURL);
				gs.onload = function () {
					gpt_ContextualReady = true;
				};
				document.head.appendChild(gs);
			}

			var elsevierMoatHeader = document.createElement('link');
			elsevierMoatHeader.rel = 'preload';
			elsevierMoatHeader.as = 'script';
			elsevierMoatHeader.src = (useSSL ? 'https:' : 'http:') + '//z.moatads.com/elsevierheader150204004183/moatheader.js';
			document.head.appendChild(elsevierMoatHeader);

			var elsevierMoatAds = document.createElement('link');
			elsevierMoatAds.rel = 'preconnect';
			elsevierMoatAds.src = (useSSL ? 'https:' : 'http:') + '//mb.moatads.com';
			document.head.appendChild(elsevierMoatAds);

			var moatHeader = document.createElement('script');
			moatHeader.async = true; gads.type = 'text/javascript';
			moatHeader.src = (useSSL ? 'https:' : 'http:') + '//z.moatads.com/elsevierheader150204004183/moatheader.js';
			document.head.appendChild(moatHeader);
		}
	}
	//--------------------------------------------------------------------------
	async function gpt_CheckForMOATInit() {
		//Check every 1/10 of a second until MOAT is Loaded or 1 second timeout
		if (gpt_MOATInitAttempts < gpt_MaxMOATInitAttempts) {
			gpt_MOATInitAttempts += 1;
			if (typeof window.moatPrebidApi === "object") {
				//MOAT LOADED
				gptMoatInit_Debug = '0.0' + gpt_MOATInitAttempts + ' seconds';
				gpt_CheckForMOATReady();
			}
			else {
				setTimeout(() => { gpt_CheckForMOATInit(); }, 100);
			}
		}
		else {
			//TIMEOUT: MOAT Not LOADED
			gptMoatInit_Debug = 'Timeout Expired';
			var googletag = googletag || {};
			googletag.cmd = googletag.cmd || [];
			gpt_CheckForContextualReady();
		}
	}
	//--------------------------------------------------------------------------
	function gpt_CheckForMOATReady() {
		//Check every 1/10 of a second until MOAT is Ready or 1 second timeout
		if (gpt_MOATReadyAttempts < gpt_MaxMOATReadyAttempts) {
			gpt_MOATReadyAttempts += 1;
			if (window.moatPrebidApi.safetyDataAvailable()) {
				//MOAT Ready
				gptMoatReady_Debug = '0.0' + gpt_MOATReadyAttempts + " seconds";
				gpt_CheckForContextualReady();
			}
			else {
				setTimeout(() => { gpt_CheckForMOATReady(); }, 100);
			}
		}
		else {
			//TIMEOUT: MOAT Not Ready
			gptMoatReady_Debug = 'Timeout Expired';
			gpt_CheckForContextualReady();
		}
	}
	//--------------------------------------------------------------------------
	function gpt_CheckForContextualReady() {
		//Check every 1/10 of a second until Contextual is Ready or 2 second timeout
		if (gpt_IsProd && gptContextualEnabled) {
			//Enabled
			if (gpt_ContextualReadyAttempts < gpt_MaxContextualReadyAttempts) {
				gpt_ContextualReadyAttempts += 1;
				if (gpt_ContextualReady) {
					//Contextual Ready
					gptContextualURL_Debug = gpt_ContextualURL;
					gptContextualReady_Debug = '0.0' + gpt_ContextualReadyAttempts + " seconds";
					gptContextualInit_Debug = gs_channels;
					gpt_InitAds();
				}
				else {
					setTimeout(() => { gpt_CheckForContextualReady(); }, 100);
				}
			}
			else {
				//LOG TIMEOUT error
				//REMOVE FOR NOW (SHOULD LOG TO A DIFF LOCATION)
				//TIMEOUT: Contextual Not Ready
				$('#gpt_GrapeShotDebugBtn').remove();
				gptContextualURL_Debug = gpt_ContextualURL;
				gptContextualReady_Debug = 'Timeout Expired';
				gptContextualInit_Debug = "";
				gpt_InitAds();
			}
		}
		else {
			//Not Enabled
			$('#gpt_GrapeShotDebugBtn').remove();
			switch (gpt_AdFile) {
				case 'PU':
				case 'LANCET':
				case 'JBS':
					gptContextualURL_Debug = 'Not enabled outside of Production';
					gptContextualReady_Debug = 'Not enabled outside of Production';
					gptContextualInit_Debug = 'Not enabled outside of Production';
					break;
				case "CELLJOURNAL":
				case "CELLBUCKET":
					gptContextualReady_Debug = 'Not enabled on Cell';
					gptContextualInit_Debug = 'Not enabled on Cell';
					gptContextualURL_Debug = 'Not enabled on Cell';
					break;
			}
			gpt_InitAds();
		}
	}
	//--------------------------------------------------------------------------
	function GPT_GetPageByURL() {
		var host = window.location.host.toLocaleLowerCase()
		var localpath = window.location.pathname.toLocaleLowerCase();

		///Check for Cell.com
		if (host.indexOf('cell.com') >= 0) {
			return null;
		}

		///Check for Cell.com (marlin-stag)
		if (host.indexOf('www-cell-com') >= 0) {
			return null;
		}

		///Check for Lancet
		if (host.indexOf('thelancet') >= 0) {
			return null;
		}

		if (localpath.indexOf('/article/') >= 0) {
			return 'article.fulltext';
		}

		if (localpath.indexOf('/action/dosearch') >= 0) {
			return 'search.searchResults';
		}

		if (localpath.indexOf('/content/') >= 0) {
			return 'periodical.editorial';
		}

		return 'periodical.home';
	}
	//--------------------------------------------------------------------------
	function gpt_CheckForNULL(x, valueifnull) {
		if (typeof valueifnull == "undefined") {
			valueifnull = null;
		}

		if (typeof x == 'undefined') {
			return valueifnull;
		}
		if (x == null) {
			return valueifnull;
		}
		if (x == "null") {
			return valueifnull;
		}
		if ($.trim(x) == '') {
			return valueifnull;
		}

		return x;
	}
	//--------------------------------------------------------------------------
	function gpt_GetAdditionalTargeting() {
		//adds page level targeting of custom variables
		//the id (minus gpt) and the value will be used as a key-value pair
		//eg: <input type="hidden" id="gptissuepii" value="x" class="gpt-Target">
		//eg: target issuepii=x
		$(".gpt-Target").each(function (index) {
			var gptTargetId = $(this).attr('id');
			gptTargetId = gptTargetId.replace('gpt', '').toLowerCase();
			if (gptTargetId !== '') {
				gpt_Target(gptTargetId);
			}
		})
	}
	//--------------------------------------------------------------------------
	function gpt_GetQuerystring(key, default_) {
		//function to read querystring variables
		if (default_ === null) default_ = '';
		key = key.replace(/[\[]/, "\\\[").replace(/[\]]/, "\\\]");
		var regex = new RegExp("[\\?&]" + key + "=([^&#]*)");
		var qs = regex.exec(window.location.href);
		if (qs === null)
			return default_;
		else
			return qs[1];
	}
	//--------------------------------------------------------------------------
	function gpt_GetDFPTestId() {
		//Get any request for a test id from querystring and set custom targeting parameter
		var gptDFPTestId = gpt_GetQuerystring('dfptestid', '');
		if (gptDFPTestId !== '') {
			$('#gpt-Variables').append('<input type="hidden" id="gptDFPTestId">');
			$('#gptDFPTestId').val(gptDFPTestId);
			gpt_Target("DFPTestId");
		}
	}
	//--------------------------------------------------------------------------
	function gpt_Target(targetName) {
		//Set page level target if value is found
		if ($('#gpt' + targetName).length > 0) {
			var targetValue = $('#gpt' + targetName).val();

			if (targetValue !== '') {
				targetValue = targetValue.replace(/[^-a-z0-9]/ig, '');	//Only Allow a-z, 0-9
				googletag.pubads().setTargeting(targetName.toLowerCase(), targetValue.toLowerCase());
				gptCustomTargeting_Debug = gptCustomTargeting_Debug + targetName.toLowerCase() + ' = ' + targetValue.toLowerCase() + '<br>';
			}
		}
	}
	//--------------------------------------------------------------------------
	function gpt_TargetAccessType() {
		var currentAccessType = $('#gptAccessType').val();
		if (typeof currentAccessType == "undefined") {
			currentAccessType = "";
		}
		if (currentAccessType.indexOf('ANON_IP') >= 0) {
			googletag.pubads().setTargeting("loggedin", "yes");
		}
		else {
			gpt_Target("LoggedIn");
		}
	}
	//--------------------------------------------------------------------------
	async function gpt_PreTearsheet(adTypeId) {
		if (typeof adTypeId === "undefined") {
			adTypeId = 0;
		}
		//Remove Elsevier Cookie Policy Notice
		if ($("#onetrust-consent-sdk").length > 0) {
			$("#onetrust-consent-sdk").remove();
		}

        //Mobile Nav
        $('#skip').remove();
        $('skipNavigationLink').remove();

		//JBS Modals
		if ($('.leadinModal').length > 0) {
			$('.leadinModal').remove();
		}
		if ($('.leadinModal').length > 0) {
			$('.leadinModal').remove();
		}
		if ($('.usabilla__overlay').length > 0) {
			$('.usabilla__overlay').remove();
		}

		//Pendo
		if ($('._pendo-step-container').length > 0) {
			$('._pendo-step-container').remove();
		}
		if ($('._pendo-guide-tt_').length > 0) {
			$('._pendo-guide-tt_').remove();
		}
		if ($('._pendo-image').length > 0) {
			$('._pendo-image').remove();
		}
		if ($('._pendo-badge-image').length > 0) {
			$('._pendo-badge-image').remove();
		}

		switch (gpt_AdFile) {
			case "CELLJOURNAL":
			case "CELLBUCKET":
				//Mobile Bug Fix on Cell
				if (parseInt(adTypeId) == 5) {
					$('.widget-horizontalAd').css('position', 'static');
					$('.body-horizontalAd').css('display', 'block');
				}
				break;
			case "JBS":
			case "LANCET":
				objLeaderboard = ".container-leaderboard";
				offset = 53;
				switch (parseInt(adTypeId)) {
					case 3: //MPU
					case 4: //SkyScraper
						//BUGFIX: Show Sidebar ad for JBS
						//skyscraper & mpu
						if ($('.widget-verticalAd').length > 0) {
							$('.widget-verticalAd').css('display', 'block');
						}
						$('#backgroundLayer').css('display', 'none');   //Hide any Prestitial
						break;
					case 6:     //Prestitial
						$('#backgroundLayer').css('display', 'block');
						break;
					default:
						$('#backgroundLayer').css('display', 'none');   //Hide any Prestitial
						break;
				}
				break;
		}
	}
	//--------------------------------------------------------------------------
	function gpt_Sleep(ms) {
		return new Promise(resolve => setTimeout(resolve, ms));
	}
	//--------------------------------------------------------------------------
	async function gpt_ScrollToAd(adTypeId) {
		if (typeof adTypeId === "undefined") {
			adTypeId = 0;
		}
		var adId = "";
		var offset = 0;
		var objLeaderboard = "";
		var gpt_ScrollToAd = true;

		switch (gpt_AdFile) {
			case "PU": //PracticeUpdate
				var hideOtherAds = true;
				$('#modalPrestitial').modal('hide');
				$('.jumbotron-welcome').hide();
				objLeaderboard = ".container-leaderboard";
				offset = 53;
				switch (parseInt(adTypeId)) {
					case 2:
						//LeaderBoard, no Scroll Needed
						gpt_ScrollToAd = false;
						adId = "adSlot1";
						break;
					case 3:
						//MPU
						adId = "adSlot2";
						break;
					case 4:
						//SkyScraper
						adId = "adSlot3";
						$('.most-read').hide();
						break;
					case 5:
						//Mobile, no Scroll Needed
						gpt_ScrollToAd = false;
						adId = "adSlot1";
						break;
					case 6:
						//Interstitial
						gpt_ScrollToAd = false;
						adId = "adSlot15";
						await gpt_Sleep(1000);
						$(objLeaderboard).hide();
						$('#modalPrestitial').unbind();
						$('#modalPrestitial').modal();
						break;
					case 7:
						//Native Ad
						break;
					case 10:
						//Master Companion
						hideOtherAds = false;
						gpt_ScrollToAd = false;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						adId = "adSlot1";
						break;
				}
				if (hideOtherAds) {
					//Hide all other ads
					$('.ad-container').each(function (i, obj) {
						if (this.id !== adId) {
							var tempParent = this.id;
							tempParent = tempParent.replace("adSlot", "adParent");
							$("#" + tempParent).addClass('hide');
						}
					});
				}
				break;
			case "JBS":
				objLeaderboard = ".js__adToHide";
				offset = 80;
				switch (parseInt(adTypeId)) {
					case 2:
						//Leaderboard
						gpt_ScrollToAd = false;
						$(".widget-verticalAd").hide();
						break;
					case 5:
						//Mobile
						//Scroll to Top
						gpt_ScrollToAd = false;
						break;
					case 3: //MPU
					case 4: //SkyScraper
						$(objLeaderboard).hide();
						adId = $('.gpt-ad-defined[data-adsize="sidebar"').attr('id');
						break;
					case 10:
						//MasterCompanion
						gpt_ScrollToAd = false;
						break;
					case 13:	    //Phone - Portrait
					case 14:        //Phone - Landscape
						$(objLeaderboard).hide();
						offset = 60;
						adId = 'gpt-Interscroller';
						gpt_ScrollToAd = true;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						break;
				}
				break;
			case "CELLJOURNAL":
			case "CELLBUCKET":
				objLeaderboard = ".js__adToHide";
				offset = 260;
				switch (parseInt(adTypeId)) {
					case 2:
						//Leaderboard
						gpt_ScrollToAd = false;
						$('.body-verticalAd').hide();
						break;
					case 5:
						//Mobile
						//Scroll to Top
						gpt_ScrollToAd = false;
						break;
					case 3: //MPU
					case 4: //SkyScraper
						$(objLeaderboard).hide();
						adId = $('.gpt-ad-defined[data-adsize="boombox"').attr('id');
						break;
					case 10:
						//MasterCompanion
						gpt_ScrollToAd = false;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						break;
				}
				//Hide the second ad if found
				var tCount = 0;
				$(".widget.literatumAd.none.GPT-ad-placeholder.widget-horizontalAd.widget-compact-all").each(function (index) {
					tCount += 1;
					if (tCount !== 1) {
						$(this).hide();
					}
				});
				break;
			case "LANCET":
				objLeaderboard = ".js__adToHide";
				offset = 80;
				switch (parseInt(adTypeId)) {
					case 2:
						//Leaderboard
						gpt_ScrollToAd = false;
						$('.body-verticalAd').hide();
						break;
					case 5:
						//Mobile
						//Scroll to Top
						gpt_ScrollToAd = false;
						break;
					case 3: //MPU
					case 4: //SkyScraper
						$(objLeaderboard).hide();
						adId = $('.gpt-ad-defined[data-adsize="skyscraper"').attr('id');
						break;
					case 10:
						//MasterCompanion
						gpt_ScrollToAd = false;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						break;
				}
				break;
		}

		//Scroll to Top
		var gpt_container = $("html,body");
		gpt_container.animate({ scrollTop: 0, scrollLeft: 0 });

		if (gpt_ScrollToAd) {
			//If Adexists, scroll to it
			var gpt_scrollTo = $('#' + adId);
			if (gpt_scrollTo.length > 0) {
				//Scroll to the Ad
				var gpt_topPosition = gpt_scrollTo.offset().top - gpt_container.offset().top + gpt_container.scrollTop();
				if (gpt_topPosition > 0) {
					gpt_container.animate({ scrollTop: gpt_topPosition, scrollLeft: 0 });
					if (gpt_topPosition - offset > 0 && offset > 0) {
						gpt_container.animate({ scrollTop: gpt_topPosition - offset, scrollLeft: 0 });
					}
				}
			}
		}
	}
	//--------------------------------------------------------------------------
	async function gpt_CheckForCookiePolicy() {
		var returnVal = false;
		if ($('#onetrust-accept-btn-handler').length > 0) {
			$('#onetrust-accept-btn-handler').click();
			returnVal = true;
		}
		return returnVal;
	}
	//--------------------------------------------------------------------------
	function gpt_InitAds() {
		var gpt_ResponsiveHomepage = false;

		switch (gpt_AdFile) {
			case "JBS":
			case "CELLJOURNAL":
			case "CELLBUCKET":
				//Responsive Check
				if ($('#hdrHomeBrand').length > 0) {
					gpt_ResponsiveHomepage = true;
				}
				break;
		}

		//InterScroller Check
		if (gpt_AdFile === "JBS") {
			var gpt_InterScrollerEnabled = false;
			var gpt_InterScrollerInsertObj;

			if ($("#articleHeader").length > 0) {
				//It's an article Page
				if ($('.article__sections > section').length > 0) {
					gpt_InterScrollerEnabled = true;
					gpt_InterScrollerInsertObj = $(".article__sections > section");
				}
			}
			else {
				if ($('.toc__section').length > 0) {
					//It's a TOC Page
					gpt_InterScrollerEnabled = true;
					gpt_InterScrollerInsertObj = $(".toc__section").first();
				}
			}

			//Add the Interscroller
			if (gpt_InterScrollerEnabled) {
				gpt_InterScrollerInsertObj.first().append($("#gpt-Interscroller").clone());
				$("#gpt-Interscroller").remove();
				$(".gpt-adinterscroller").addClass("gpt-ad");
			}
		}

		googletag.cmd.push(function () {
			var gptBoomBoxSizeMapping;
			var gptFluidMapping;
			var gptHomePageSidebarMapping;
			var gptCellHomePageSidebarMapping;
			var gptInterscrollerMapping;
			var gptLeaderboardSizeMapping;
			var gptMobileSizeMapping;
			var gptPrestitialMapping;
			var gptRectangleShortSizeMapping;
			var gptRectangleSizeMapping;
			var gptSideBarSizeMapping;
			var gptTallSizeMapping;
			var gptWideSizeMapping;
			var gptWideRectangleSizeMapping;

			//***********************************************
			//Start Define Size mapping based on client Viewport
			//***********************************************
			if (gpt_AdFile === 'PU') {
				gptTallSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[300, 600], [160, 600], [120, 600]]).
					addSize([0, 0], []).
					build();

				gptRectangleSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[300, 250]]).
					addSize([0, 0], []).
					build();

				gptRectangleShortSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[300, 250], [320, 50], [300, 50]]).
					addSize([0, 0], [[300, 250], [320, 50], [300, 50]]).
					build();

				gptWideSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[728, 90]]).
					addSize([760, 0], [[728, 90], [320, 50], [300, 50]]).
					addSize([0, 0], [[320, 50], [300, 50]]).
					build();

				gptWideRectangleSizeMapping = googletag.sizeMapping().
					addSize([992, 0], []).
					addSize([760, 0], [[320, 50], [300, 50], [300, 250]]).
					addSize([0, 0], [[320, 50], [300, 50], [300, 250]]).
					build();

				gptMobileSizeMapping = googletag.sizeMapping().
					addSize([0, 0], [[320, 50], [300, 50]]).
					build();

				gptFluidMapping = googletag.sizeMapping().
					addSize([0, 0], ['fluid']).
					build();

				gptPrestitialMapping = googletag.sizeMapping().
					addSize([760, 0], [[1, 1]]).
					build();
			}
			else {
				//JBS-CELL-LANCET
				gptLeaderboardSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[728, 90]]).
					addSize([768, 0], [[728, 90]]).
					addSize([0, 0], [[320, 50], [300, 50]]).
					build();

				gptHomePageSidebarMapping = googletag.sizeMapping().
					addSize([2090, 795], [[300, 600], [300, 250], [160, 600], [120, 600]]).
					addSize([1910, 795], [[160, 600], [120, 600]]).
                    addSize([1740, 795], [[120, 600]]).
					addSize([0, 0], []).
					build();

				gptCellHomePageSidebarMapping = googletag.sizeMapping().
                    addSize([2180, 795], [[300, 600], [160, 600], [120, 600], [300, 250], [336, 280]]).
					addSize([2180, 445], [[336, 280]]).
                    addSize([2090, 795], [[300, 600], [160, 600], [120, 600], [300, 250]]).
                    addSize([2090, 445], [[300, 250]]).
					addSize([1830, 795], [[160, 600], [120, 600]]).
                    addSize([1740, 795], [[120, 600]]).
					addSize([0, 0], []).
					build();

				gptSideBarSizeMapping = googletag.sizeMapping().
					addSize([1140, 795], [[300, 600], [160, 600], [120, 600], [300, 250]]).
					addSize([1140, 445], [[300, 250]]).
					addSize([0, 0], []). 				
					build();

				gptBoomBoxSizeMapping = googletag.sizeMapping().
					addSize([768, 0], [[300, 250], [336, 280]]).
					addSize([0, 0], []). 						
					build();

				gptFluidMapping = googletag.sizeMapping().
					addSize([0, 0], ['fluid']).
					build();

				gptPrestitialMapping = googletag.sizeMapping().
					addSize([600, 700], [[1, 1]]).
					addSize([0, 0], []).
					build();

				gptInterscrollerMapping = googletag.sizeMapping().
					addSize([1140, 445], []).
					addSize([1024, 768], [[1024, 768]]).
					addSize([768, 1024], [[768, 1024]]).
					addSize([480, 320], [[480, 320]]).
					addSize([320, 480], [[320, 480]]).
					build();
			}
			//***********************************************
			//End Define Size mapping based on client Viewport
			//***********************************************

			//***********************************************
			//Start detection for ad slots on page
			//***********************************************
			var gptMasterPage = gpt_CheckForNULL($('#gptPage').val());
			var gptMasterSite;

			if (gpt_AdFile === 'PU') {
				gptMasterSite = $('#gptSite').val();
			}
			else {
				//JBS-CELL-LANCET
				gptMasterSite = "dev";

				if (gpt_IsProd) {
					gptMasterSite = gpt_CheckForNULL($('#gptSite').val());

					//If there is no master site, check for a defaultsite
					if (gptMasterSite == null) {
						gptMasterSite = gpt_CheckForNULL($('#gptDefaultSite').val());
					}
				}
				else {
					//Dev ENV
					//If the page has no gptSite Value, assume no value which will disable ads
					if (gpt_CheckForNULL($('#gptSite').val()) === null) {
						gptMasterSite = null;
					}
				}

				//If there is no master Page, check for a Page using the URL
				if (gptMasterPage === null) {
					gptMasterPage = GPT_GetPageByURL();
				}
			}

			$(".gpt-ad").each(function (index) {
				var gptPage = '';
				var adSizeMapping = '';
				var gptPOS;
				var gptTC;
				var gptNative;
				var adId;
				var gptSite;
				var tempSlot = null;


                if (gpt_IsProd) {
					if (gpt_AdFile !== "CELLJOURNAL") {
						//Check if the ad slot has it's own site defined
						if (typeof $(this).data('site') === 'undefined') {
							gptSite = gptMasterSite;
						}
						else {
							gptSite = gpt_CheckForNULL($(this).data('site'));
						}
					}
					else {
						gptSite = gptMasterSite;
					}
                }
                else {
					gptSite = gptMasterSite;
                }



				if (gpt_AdFile === 'PU') {
					adId = $(this).attr('id');
				}
				else {
					//JBS-CELL-LANCET
					adId = 'gpt-ad-' + index + 1;
				}

				if (gpt_CheckForNULL(gptSite) !== null) {
					gptSite_Debug = gptSite;

					gptPOS = parseInt($(this).data('pos')) || -1;
					if (gptPOS === -1) {
						gptPOS = index + 1;
					}

					//Check if the ad slot has it's own page defined
					gptPage = $(this).data('page');
					if (gpt_CheckForNULL(gptPage) == null) {
						//Use the Master Page
						gptPage = gptMasterPage;
					}

					//If the page is not null, preceed it with a slash
					if (gptPage !== null) {
						gptPage = '/' + gptPage;
					}
					else {
						gptPage = '';
					}
					gptPage_Debug = gptPage;


					gptTC = $(this).data('tc');
					if (typeof gptTC === "undefined") {
						gptTC = '';
					}

					gptNative = $(this).data('native');
					if (typeof gptNative === "undefined") {
						gptNative = '';
					}

					//Determine sizemapping based on data-adsize
					if (gpt_AdFile === 'PU') {
						switch ($(this).data('adsize')) {
							case 'tall':
								adSizeMapping = gptTallSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600]], adId);
								break;
							case 'rectangle':
								adSizeMapping = gptRectangleSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250]], adId);
								break;
							case 'rectangleshort':
								adSizeMapping = gptRectangleShortSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250], [320, 50], [300, 50]], adId);
								break;
							case 'wide':
								adSizeMapping = gptWideSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[728, 90], [320, 50], [300, 50]], adId);
								break;
							case 'widerectangle':
								adSizeMapping = gptWideRectangleSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[320, 50], [300, 50], [300, 250]], adId);
								break;
							case 'mobile':
								adSizeMapping = gptMobileSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[320, 50], [300, 50]], adId);
								break;
							case 'fluid':
								adSizeMapping = gptFluidMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, 'fluid', adId);
								break;
							case 'prestitial':
								adSizeMapping = gptPrestitialMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[1, 1]], adId);
								break;
						}
					}
					else {
						//JBS-CELL-LANCET
						switch ($(this).data('adsize')) {
							case 'cell-leaderboard':
							case 'leaderboard':
								adSizeMapping = gptLeaderboardSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[728, 90], [320, 50], [300, 50]], adId);
								break;
							case 'bottom':
								if (gpt_ResponsiveHomepage) {
									adSizeMapping = gptHomePageSidebarMapping;
									tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600], [300, 250]], adId);
								}
								else {
									adSizeMapping = gptBoomBoxSizeMapping;
									tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250]], adId);
								}
								break;
							case 'boombox':
								if (gpt_AdFile === "CELLJOURNAL" || gpt_AdFile == "CELLBUCKET") {
									if (gpt_ResponsiveHomepage) {
										adSizeMapping = gptCellHomePageSidebarMapping;
										tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600], [300, 250], [336, 280]], adId);
									}
									else {
										adSizeMapping = gptBoomBoxSizeMapping;
										tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250], [336, 280]], adId);
									}
								}
								else {
									adSizeMapping = gptBoomBoxSizeMapping;
									tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250]], adId);
								}
								break;
							case 'interscroller':
								adSizeMapping = gptInterscrollerMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[320, 480], [480, 320], [768, 1024], [1024, 768]], adId);
								break;
							case 'sidebar':
							case 'skyscraper':
								adSizeMapping = gptSideBarSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600], [300, 250]], adId);
								break;
							case 'native':
							case 'fluid':
								adSizeMapping = gptFluidMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, 'fluid', adId);
								break;
							case 'prestitial':
								adSizeMapping = gptPrestitialMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, 'fluid', adId);
								break;
						}
					}
				}

				//Ignore any invalid adsizes that could be on the page
				if (adSizeMapping !== '') {
					$(this).addClass('gpt-ad-defined');
					if (gpt_AdFile !== 'PU') {
						$(this).attr('id', adId);
					}
					tempSlot.defineSizeMapping(adSizeMapping).addService(googletag.pubads()).setTargeting("pos", gptPOS);

					if (gptTC !== '') {
						tempSlot.setTargeting("tc", gptTC);
					}
					if (gptNative !== '') {
						tempSlot.setTargeting("native", gptNative);
					}
					gptEvent_Debug = gptEvent_Debug + $(this).data('adsize') + ' Requested<br>';
				}
				else {
					gptInvalidAdSize_Debug = gptInvalidAdSize_Debug + "Invalid Size Found, Id: " + $(this).prop('id') + ', adSize: ' + $(this).data('adsize') + '<br>';
					$(this).remove();
				}
			});
			if (gpt_CheckForNULL(gptInvalidAdSize_Debug) === null) {
				gptInvalidAdSize_Debug = "None Detected";
			}
			//***********************************************
			//End detection for ad slots on page
			//***********************************************

			//***********************************************
			//Start page level targeting
			//***********************************************
			if (gpt_AdFile !== 'PU') {
				//Logged in or IP Access
				gpt_TargetAccessType();

				//Check for Screenshot Client
				gpt_CheckSSC();
			}

			//Check for DFP Test
			gpt_GetDFPTestId();

			//Check for Additional Targeting
			gpt_GetAdditionalTargeting();

			if (gpt_IsProd && gptContextualEnabled) {
				var gpt_ContextualRetry = false;
				//Check for Contextual Targeting
				if (gs_channels !== "DEFAULT") {
					var gpt_ContextualCategories = Array.from(gs_channels);
					for (var i = 0; i < gpt_ContextualCategories.length; i++) {
						if (gpt_ContextualCategories[i] == 'gx_retry') {
							gpt_ContextualRetry = true;
						}
						if (!gpt_ContextualCategories[i].startsWith("custom_")) {
							//Remove any tag that is not custom
							gpt_ContextualCategories.splice(i, 1);
							i--;
						}
						else {
							//Strip the 'custom_' from the tag
							gpt_ContextualCategories[i] = gpt_ContextualCategories[i].replace("custom_", "")
						}
					}
					if (gpt_ContextualCategories.length > 0) {
						googletag.pubads().setTargeting("gs_cat", gpt_ContextualCategories);
					}
					else {
						gpt_ContextualCategories = 'none';
					}
					if (gpt_ContextualRetry) {
						gpt_ContextualCategories = "*Indexing*";
					}
					gptContextualResult_Debug = gpt_ContextualCategories;
				}
				else {
					gptContextualResult_Debug = 'No segments found';
				}
			}
			else {
				//Not Prod || Contextual not enabled
				gptContextualResult_Debug = 'Not enabled';
				
				if (!gpt_IsProd) {
                    var gpt_DevSite = gpt_CheckForNULL($('#gptSite').val(), "dev");
                    $('#gpt-Variables').append('<input type="hidden" id="gptEnvironment" value="' + gpt_DevSite + '">');
                    gpt_Target("Environment");
                }

			}
			//***********************************************
			//End page level targeting
			//***********************************************

			//Enable google dfp services
			if (parseInt($('#gptLazyLoading').val()) === 1) {
				googletag.pubads().enableLazyLoad({
					fetchMarginPercent: 200,
					renderMarginPercent: 0,
					mobileScaling: 2.0
				});
			}
			else {
				googletag.pubads().enableSingleRequest();
			}

			//Add Listener for RenderEnded
			googletag.pubads().addEventListener('slotRenderEnded', function (event) {
				if (!event.isEmpty) {
					if (typeof LocalAdLoaded === "function") {
						LocalAdLoaded(event);
					}
					if (gpt_ResponsiveHomepage) {
						if (typeof positionAd === "function") {
							positionAd();
						}
					}
					if (event.size[0] == "480" || event.size[1] == "480") {
						$('.gpt-InScrollBox').css('display', 'block');
						gptEvent_Debug = gptEvent_Debug + 'Interscroller ' + event.size + ' rendered: ' + gpt_GetElapsedTime() + '<br>';
					}
					else {
						gptEvent_Debug = gptEvent_Debug + event.size + ' rendered: ' + gpt_GetElapsedTime() + '<br>';
					}

				}
			});

			if (gpt_AdFile !== 'PU') {
				googletag.pubads().collapseEmptyDivs();
			}
			googletag.enableServices();

			//Populate Defined Ad Slots
			$(".gpt-ad-defined").each(function (index) {
				var adId = $(this).attr('id');
				googletag.cmd.push(function () { googletag.display(adId); });
			});

			gptStatus_Debug = 'GPT Complete';
		});
	}
	//--------------------------------------------------------------------------
	function gpt_DisplayDebug() {
		//Clone the template and append to Body
		$(document.body).append($('#gpt-DebugDiv').clone());
		$('#gpt-DebugDiv').remove();

		//Populate the Values
		gpt_LoadDebugValues();

		//Display and scroll to debug section
		$('.gpt-DebugDiv').css('display', 'block');
		window.scrollTo(0, document.body.scrollHeight);
	}
	//--------------------------------------------------------------------------
	function gpt_LoadDebugValues() {
		$('.gpt-DebugBtn').blur();
		$('.gptVersion_Debug').text(gpt_Version);
		$('.gptAdFile_Debug').html(gpt_AdFile);
		$('.gptEvent_Debug').html(gptEvent_Debug);
		$('.gptCustomTargeting_Debug').html(gptCustomTargeting_Debug);
		$('.gptSite_Debug').text(gptSite_Debug);
		$('.gptContextualResult_Debug').text(gptContextualResult_Debug);
		$('.gptContextualInit_Debug').text(gptContextualInit_Debug);
		$('.gptContextualURL_Debug').text(gptContextualURL_Debug);
		$('.gptContextualReady_Debug').text(gptContextualReady_Debug);
		$('.gptMoatReady_Debug').text(gptMoatReady_Debug);
		$('.gptMoatInit_Debug').text(gptMoatInit_Debug);
		$('.gptTestENV_Debug').text(gptTestENV_Debug);
		$('.gptDisableAds_Debug').text(gptDisableAds_Debug);
		$('.gptStatus_Debug').text(gptStatus_Debug);
		$('.gptPage_Debug').text(gptPage_Debug);
		$('.gptInvalidAdSize_Debug').html(gptInvalidAdSize_Debug);
	}
	//--------------------------------------------------------------------------
	function gpt_GrapeShotDebug() {
		$('.gpt-DebugBtn').blur();
		window.open("http://admin-elsevier.grapeshot.co.uk/custom/bookmarklet.cgi?blocking=1&vn=2&url=" + encodeURIComponent(document.location), "GRAPESHOT_POPUP", "width=650,height=600,scrollbars=yes,screenX=40,screenY=50,menubar=no,toolbar=no,resizable=yes", "Custom").focus();
	}
	//--------------------------------------------------------------------------
	function gpt_ShowConsole() {
		$('.gpt-DebugBtn').blur();
		googletag.openConsole();
	}
	//--------------------------------------------------------------------------
</script>
        </div>

            
                </div>
            

        

    

        


    

            

            

            <div data-widget-id="912135d6-7135-4d45-b457-b503cacc3c8b" data-widget-name="ux3-layout-widget"><header class="header base fixed">
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none adplaceholder adplaceholder--header js__adToHide widget-horizontalAd" id="9d6062d7-fad6-4e64-81cc-7dd25d99b23d">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-horizontalAd "><div class="horizontalAdContainer">
    <div class="horizontalAdLabelContainer">
        <div class="horizontalAdLabel">
                <img src="/templates/jsp/_style2/_marlin/images/bg_adHoriz.gif" alt="Advertisement">
        </div>
    </div>
    <div class="horizontalAd">
        





        <div class="pb-ad">
            <!-- Beginning Async AdSlot 1 for Ad unit els.cellpress/cell  ### size: [[728,90]] -->
<div class="gpt-ad" data-adsize="cell-leaderboard" data-page="" data-site="cell" data-pos="1"></div>
<!-- End AdSlot 1 -->
        </div>
    

    </div>
</div></div>
                </div>
                </div>

        










        <div data-widget-def="ux3-layout-widget" data-widget-id="3a83e2a6-2e7e-4931-892c-4ef43b6e3312" class="header__wrapper clearfix" data-db-parent-of="quickSearch"><div id="brandingRow"><a href="/" data-aa-name="science that inspires" data-aa-region="cp global header" title="Visit Cell.com">Science that inspires</a><p id="brandingRowRight"><a href="/" data-aa-name="cell press endorsement" data-aa-region="cp global header" title="Visit Cell.com">A Cell Press journal</a></p></div>
        
    

        


    

            

            

            <div data-widget-id="3a83e2a6-2e7e-4931-892c-4ef43b6e3312" data-widget-name="ux3-layout-widget"><div class="col-xs-6 header__left-side">
    

        


    

            

            
                <a href="/home" id="cpIconLnk" data-aa-name="cell press logo" data-aa-region="cp global header" title="Cell Press" target="_blank"><svg id="cp_logo" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 100 98.1" style="enable-background:new 0 0 100 98.1;" xml:space="preserve" height="100" width="98.1"><path style="fill:#FFFFFF;" d="M69.09,0.09c-4.01-0.6-7.83,1.94-8.78,5.89l-3.14,13.11c-0.76,3.19-3.44,5.56-6.7,5.93l-18.6,2.13          C11.83,29.31-2.53,47.66,0.37,67.78c2.3,15.96,15.18,27.77,30.54,30.08c4.01,0.6,7.83-1.94,8.78-5.89l3.14-13.11 c0.76-3.19,3.44-5.56,6.69-5.93l18.6-2.13c20.04-2.17,34.41-20.52,31.5-40.65C97.32,14.21,84.45,2.39,69.09,0.09z M65.06,47.2  l-11.75,1.4c-1.98,0.21-3.62,1.65-4.08,3.59l-3.43,14.3c-1.01,4.22-4.5,7.39-8.8,8c-6.66,0.72-12.63-4.2-13.15-10.92                c-0.51-6.5,4.6-12.11,11.08-12.81l11.75-1.39c1.98-0.22,3.62-1.65,4.08-3.59l3.43-14.3c1.01-4.22,4.5-7.39,8.8-8          c6.66-0.72,12.63,4.19,13.15,10.92C76.66,40.89,71.55,46.5,65.06,47.2z"></path></svg></a><div class="journal-logos">
            

            <span data-widget-id="35d8d734-3aa0-4ec8-9e2d-14babe803594" data-widget-name="ux3-general-image"><a href="/cell/home" title="Cell"><img alt="Cell" src="/pb-assets/ux3/logos/cell/Cell_white-1532516611593.svg"></a></span>

            
                </div>
            

        

    

        


    

            

            
                <div class="journal-logos">
            

            <span data-widget-id="3ffcbf1f-79fd-4ba3-85b5-bef10a63475f" data-widget-name="ux3-general-image"><a href="https://www.cell.com/open-access?utm_source=oa-label&amp;__hstc=109691226.e4e812b53ff324eccbdbdb2e4fb9a6ab.1644503737405.1644524234622.1644587109507.4&amp;__hssc=109691226.7.1644587109507&amp;__hsfp=3253153019" title="This journal offers authors two options (open access or subscription) to publish research"><img alt="This journal offers authors two options (open access or subscription) to publish research" src="/pb-assets/ux3/logos/cell/Supports_open_access_border-1645554937897.svg"></a></span>

            
                </div>
            

        
</div><div class="col-xs-6">
    

        


    

            

            

            <div data-widget-id="2555f6db-b333-4edb-8b15-ef9213f1f0a5" data-widget-name="ux3-layout-widget"><div class="header__nav row">
    

        


    

            

            

            <ul data-widget-id="52432d24-0331-4c9d-85f3-2033edc7208b" data-widget-name="UX3ProfileLinksWidget" class="rlist rlist--inline profile-links"><li class="profile-links__submit"><a data-adobe-name="Publish" class="profile-links__submit__btn" title="Submit" href="https://www.editorialmanager.com/cell" target="_self" data-aa-conversiondriver="jbs:global-header:submitpage" data-profile-link="true">Submit</a></li><li class="profile-links__login-dropdown"><div class="profile-links__user-list__wrapper"><button data-db-target-for="loginmenu-179338" aria-expanded="false" aria-controls="login-links-179338" title="Log in links dropdown" class="profile-links__username"><span>Log in</span></button><div data-db-target-of="loginmenu-179338" id="login-links--179338" class="profile-links__user-list__holder dropBlock__holder"><ul class="rlist profile-links__list"><li class="profile-links__login"><a data-adobe-name="Log In" class="profile-links__login__btn" title="Log in to Cell.com" href="https://www.cell.com/action/idLogin?type=login&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2822%2900079-4" target="_self" data-profile-link="true">Log in</a></li><li class="profile-links__institution"><a data-adobe-name="Get institutional access" class="profile-links__institution__btn" title="Institutional access" href="https://www.cell.com/action/idLogin?type=institution&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2822%2900079-4" target="_self" data-profile-link="true">Get Institutional Access</a></li></ul></div></div></li><li class="profile-links__register"><a data-adobe-name="Register" class="profile-links__register__btn" title="Register a username and password for free" href="https://www.cell.com/action/idLogin?type=registration&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2822%2900079-4" target="_self" data-profile-link="true">Register</a></li><li class="hidden-lg"> <div class="profile-links__controls"> <button aria-expanded="false" aria-controls="links-dialog-179338" data-opens="#links-dialog-179338" class="profile-links__btn--open"><span class="sr-only">open links dialog</span><i aria-hidden="true" class="icon-gizmo-person"></i></button></div><dialog id="links-dialog-179338" aria-modal="true"><button data-closes="#links-dialog-179338" class="profile-links__btn--close"><span class="sr-only">close links dialog</span><i aria-hidden="true" class="icon-close-solid"></i></button><ul>   <li class="profile-links__submit"><a data-adobe-name="Publish" class="profile-links__submit__btn" title="Submit" href="https://www.editorialmanager.com/cell" target="_self" data-aa-conversiondriver="jbs:global-header:submitpage" data-profile-link="true">Submit</a></li><li class="profile-links__login"><a data-adobe-name="Log In" class="profile-links__login__btn" title="Log in to Cell.com" href="https://www.cell.com/action/idLogin?type=login&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2822%2900079-4" target="_self" data-profile-link="true">Log in</a></li><li class="profile-links__institution"><a data-adobe-name="Get institutional access" class="profile-links__institution__btn" title="Institutional access" href="https://www.cell.com/action/idLogin?type=institution&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2822%2900079-4" target="_self" data-profile-link="true">Get Institutional Access</a></li><li class="profile-links__register"><a data-adobe-name="Register" class="profile-links__register__btn" title="Register a username and password for free" href="https://www.cell.com/action/idLogin?type=registration&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2822%2900079-4" target="_self" data-profile-link="true">Register</a></li></ul></dialog></li></ul>

            

        
</div></div>

            

        
</div></div>

            

        

        </div>
    

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget csasInstitutionBanner none  widget-none" id="51b7fa33-2137-4f58-9e98-9746c4400513" style="display: none;">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none "><div class="ui-state-highlight ui-corner-all csas-institution-banner hidden" data-doi="10.1016/j.cell.2022.01.021">
    <span class="ui-icon ui-icon-close banner-close"></span>
    <p>
        Access provided by <span class="accountName"></span>
    </p>
</div>

<style>
    .csas-institution-banner {
        padding: 10px 0 0 10px;
        margin: 5px;
    }

    .csas-institution-banner.ui-state-highlight .ui-icon {
        background-image: url("/templates/jsp/css/jquery-ui-1.10.2/base/images/ui-icons_454545_256x240.png");
        margin-right: 5px;
        margin-top: -4px;
    }

    .csas-institution-banner .banner-close {
        float: right;
    }
</style>

<script>

    function resetBanner(doi) {
        $.get('/action/getAccount?doi=' + doi, function(data) {
            if (data) {
                $(".csas-institution-banner p .accountName").html(data);
                $(".csasInstitutionBanner").fadeIn();
                $(".csas-institution-banner").fadeIn();
            }
        });
    }

    var flag = 0;
    $(document).ready(function() {
        if(!$(".csas-institution-banner").is(":visible")){
            $(".csasInstitutionBanner").hide();
        }

        $(".banner-close").click(function() {
            $.get("/action/dismissBanner", function(data) {
                $(".csasInstitutionBanner").fadeOut(function () {
                    if(typeof UX !== 'undefined') {
                        UX.pageBody.init();
                    }
                });
            });
        });

        $(".articleContent .tabs li[role='tab'].showFullText").click(function() {
           flag = 1;
        });

        $(document).on("reloadTabsContent", function() {
            var isSelected = $(".articleContent .tabs li[role='tab'].showFullText").hasClass("active");
            if (isSelected) {
                var isHidden =  $(".csas-institution-banner").is(".hidden");
                if (!isHidden) {
                    resetBanner($(".csas-institution-banner").data("doi"));
                    console.log("fadeIn");
                }
            } else {
                $(".csas-institution-banner").fadeOut();
                console.log("fadeOut");
            }
            flag = 0;
        });

        $(document).on("click", ".articleContent .tabs li[role='tab']", function() {

        });
    });
</script></div>
                </div>
                </div>

        
</header></div>

            

        










        <div data-widget-def="graphQueryWidget" data-widget-id="2a125dae-6dbc-4e76-a75e-3e563f1d3a1d" class="ux3">
        
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget graphQueryWidget none top-padding-handler noMargin axel-mega-menu widget-none widget-compact-all fonts-loaded" id="2a125dae-6dbc-4e76-a75e-3e563f1d3a1d" style="padding-top: 115.594px;">
                
                
                    
                

                <div class="wrapped " id="CP TOP LEVEL NAV MENU">
                    
                    <div class="widget-body body body-none  body-compact-all"><nav id="mega-menu-nav" role="navigation" class="mega-menu-nav">
  <!--.skip//a(href='#searchText' aria-label='Skip menu') Skip menu
  -->
  <div class="mega-menu-nav__wrapper">
    <button class="mega-menu__toggle"><span class="sr-only">Main menu</span><i aria-hidden="true" class="icon-gizmo-hamburger"></i></button>
    <ul role="menubar" class="mega-menu rlist">
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-1" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-1" data-item-link="main-menu-link-1" data-index="0" class="mega-menu__item-link menu-link"><span>This journal</span></a>
        <div id="main-menu-panel-1" aria-labelledby="main-menu-link-1" data-item-wrapper="main-menu-link-1" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <div aria-label="Menu dropzone section" tabindex="-1" class="listDropZone">
    

        


    

            

            

            <div class="cover-image"><div class="cover-image__thumb"><a href="/issue/S0092-8674(24)X0012-4" class="menu-link"><img src="/cms/asset/atypon:cms:attachment:img:d5e6:rev:1748270185670-389:pii:S0092867424X00124/cover.tif.jpg" alt="Cover Image - Cell, Volume 188, Issue 11"></a></div><div class="cover-image__meta"></div></div>

            

        
</div>
            </li>
            <li style="grid-column: 2/3" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-1-2" class="sub-menu__heading">Read
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/newarticles" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Online now</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/current" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Current issue</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/archive" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Archive</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/action/showPreferences?menuTab=Alerts" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Publication alerts</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/sneakpeek" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Sneak Peek - preprints</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 3/5" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-1-3" class="sub-menu__heading">About
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-2">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/advisory-board" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Advisory board</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/aims" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Aims and scope</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/contact" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Contact</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/editors-and-staff" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Editors and staff</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/editorial-ethics" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Ethics</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/information-for-authors/permissions" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Permissions</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/reviewers" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>Reviewer instructions</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-2" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-2" data-item-link="main-menu-link-2" data-index="1" class="mega-menu__item-link menu-link"><span>Journals</span></a>
        <div id="main-menu-panel-2" aria-labelledby="main-menu-link-2" data-item-wrapper="main-menu-link-2" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/5" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-2-1" class="sub-menu__heading">Life &amp; medical sciences
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-4">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/home" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cancer-cell/home" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Cancer Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-chemical-biology" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Chemical Biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-genomics/home" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Genomics</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-host-microbe/home" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Host &amp; Microbe</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-metabolism/home" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Metabolism</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports/home" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-medicine/home" data-index="7" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Medicine</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-stem-cell/home" data-index="8" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Stem Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-systems/home" data-index="9" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Systems</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/current-biology/home" data-index="10" role="menuitem" class="sub-menu__item-link menu-link"><span>Current Biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/developmental-cell/home" data-index="11" role="menuitem" class="sub-menu__item-link menu-link"><span>Developmental Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/immunity/home" data-index="12" role="menuitem" class="sub-menu__item-link menu-link"><span>Immunity</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/med/home" data-index="13" role="menuitem" class="sub-menu__item-link menu-link"><span>Med</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-cell/home" data-index="14" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/neuron/home" data-index="15" role="menuitem" class="sub-menu__item-link menu-link"><span>Neuron</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/structure/home" data-index="16" role="menuitem" class="sub-menu__item-link menu-link"><span>Structure</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/ajhg/home" data-index="17" role="menuitem" class="sub-menu__item-link menu-link"><span>American Journal of Human Genetics (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/biophysj/home" data-index="18" role="menuitem" class="sub-menu__item-link menu-link"><span>Biophysical Journal (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/biophysreports/home" data-index="19" role="menuitem" class="sub-menu__item-link menu-link"><span>Biophysical Reports (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/hgg-advances/home" data-index="20" role="menuitem" class="sub-menu__item-link menu-link"><span>Human Genetics and Genomics Advances (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-plant/home" data-index="21" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Plant (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/molecular-therapy/home" data-index="22" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/methods/home" data-index="23" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy Methods &amp; Clinical Development (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/nucleic-acids/home" data-index="24" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy Nucleic Acids (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/oncology/home" data-index="25" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy Oncology (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/plant-communications/home" data-index="26" role="menuitem" class="sub-menu__item-link menu-link"><span>Plant Communications (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/stem-cell-reports/home" data-index="27" role="menuitem" class="sub-menu__item-link menu-link"><span>Stem Cell Reports (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/biochemical-sciences/home" data-index="28" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Biochemical Sciences</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/cancer/home" data-index="29" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Cancer</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/cell-biology/home" data-index="30" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Cell Biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/ecology-evolution/home" data-index="31" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Ecology &amp; Evolution</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/endocrinology-metabolism/home" data-index="32" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Endocrinology &amp; Metabolism</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/genetics/home" data-index="33" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Genetics</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/immunology/home" data-index="34" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Immunology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/microbiology/home" data-index="35" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Microbiology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/molecular-medicine/home" data-index="36" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Molecular Medicine</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/neurosciences/home" data-index="37" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Neurosciences</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/parasitology/home" data-index="38" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Parasitology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/pharmacological-sciences/home" data-index="39" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Pharmacological Sciences</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/plant-science/home" data-index="40" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Plant Science</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 5/6" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-2-2" class="sub-menu__heading">Physical sciences &amp; engineering
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-biomaterials/home" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Biomaterials</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-physical-science/home" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Physical Science</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/chem/home" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Chem</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/chem-catalysis/home" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Chem Catalysis</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/device/home" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Device</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/joule/home" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Joule</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/matter/home" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>Matter</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/newton/home" data-index="7" role="menuitem" class="sub-menu__item-link menu-link"><span>Newton</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/chemistry/home" data-index="8" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Chemistry</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 6/7" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-2-3" class="sub-menu__heading">Multidisciplinary
              </h4>
              <ul role="menubar" data-index="2" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-methods/home" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Methods</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-sustainability/home" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Sustainability</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/heliyon/home" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Heliyon</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/iscience/home" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>iScience</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/one-earth/home" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>One Earth</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/patterns/home" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Patterns</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/star-protocols/home" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>STAR Protocols</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/nexus/home" data-index="7" role="menuitem" class="sub-menu__item-link menu-link"><span>Nexus (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/the-innovation/home" data-index="8" role="menuitem" class="sub-menu__item-link menu-link"><span>The Innovation (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/biotechnology/home" data-index="9" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Biotechnology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/cognitive-sciences/home" data-index="10" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Cognitive Sciences</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-3" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-3" data-item-link="main-menu-link-3" data-index="2" class="mega-menu__item-link menu-link"><span>Publish</span></a>
        <div id="main-menu-panel-3" aria-labelledby="main-menu-link-3" data-item-wrapper="main-menu-link-3" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-3-1" class="sub-menu__heading">Authors
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="https://www.editorialmanager.com/cell?__hstc=25856994.7201670a4397fa03148684920d4a8d38.1672762013831.1694716659709.1694718909620.146&amp;__hssc=25856994.15.1694718909620&amp;__hsfp=1366844671" data-index="0" target="_blank" role="menuitem" class="sub-menu__item-link menu-link"><span>Submit article
                      <div class="external-icon"></div><span class="sr-only">External Link</span></span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/information-for-authors" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Information for authors</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/open-access" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Open access</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/multi-journal-submission" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Multi-Journal Submission</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/information-for-authors/star-authors-guide" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>STAR Methods</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 2/3" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-3-2" class="sub-menu__heading">Reviewers
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/reviewers" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Information for reviewers</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-4" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-4" data-item-link="main-menu-link-4" data-index="3" class="mega-menu__item-link menu-link"><span>News &amp; events</span></a>
        <div id="main-menu-panel-4" aria-labelledby="main-menu-link-4" data-item-wrapper="main-menu-link-4" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/newsroom" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Newsroom</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/events" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Events</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/symposia" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Symposia</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/webinars" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Webinars</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/consortia" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Consortia Hub</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-5" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-5" data-item-link="main-menu-link-5" data-index="4" class="mega-menu__item-link menu-link"><span>About Cell Press</span></a>
        <div id="main-menu-panel-5" aria-labelledby="main-menu-link-5" data-item-wrapper="main-menu-link-5" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-5-1" class="sub-menu__heading">Cell Press
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/about" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>About Cell Press</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/contact" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Contact us</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="https://service.elsevier.com/app/overview/cell-press/" data-index="2" target="_blank" role="menuitem" class="sub-menu__item-link menu-link"><span>Help &amp; support
                      <div class="external-icon"></div><span class="sr-only">External Link</span></span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/open-access" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Open access at Cell Press</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/diversity" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Inclusion and diversity</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-press-50" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>50th anniversary</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 2/3" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-5-2" class="sub-menu__heading">Hubs
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/biotechnology-bioengineering" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Biotechnology and bioengineering</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-press-chemistry" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Chemistry</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/clinical-translational" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Clinical and translational</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/covid-19" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>COVID-19</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/fundamental-biology" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Fundamental biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/sustainability" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Sustainability</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 3/4" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-5-3" class="sub-menu__heading">Careers
              </h4>
              <ul role="menubar" data-index="2" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="https://relx.wd3.myworkdayjobs.com/en-US/ElsevierJobs" data-index="0" target="_blank" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Press careers
                      <div class="external-icon"></div><span class="sr-only">External Link</span></span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
    </ul>
    <div class="mega-menu-nav__quick-search">
    

        


    

            

            

            <div data-widget-id="a1f05748-166f-41e0-b01a-ca8ac1cc95d4" data-widget-name="UX3QuickSearchWidget" class="quick-search"><div class="dropBlock"><form action="https://www.cell.com/action/doSearch?journalCode=cell" name="quickSearchBar" method="get" title="Quick Search"><input id="type" type="hidden" value="quicksearch" name="type"><div class="quick-search__toggle autoCompleteWrapper"><input id="searchText" type="text" name="text1" placeholder="Search for..." enterkeyhint="search" autocomplete="off" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" data-group-titles-conf="3" data-publication-items-conf="3" aria-label="Enter a search term" class="searchText searchText-autocomplete ui-autocomplete-input"><button type="submit" title="Search" aria-label="Submit the search form" class="btn quick-search__button icon-search"><span>Go search</span></button><a href="https://www.cell.com/search/advanced?SeriesKey=cell&amp;ISSN=0092-8674&amp;journalCode=cell" title="Advanced search" class="quick-search__advanced-link">Advanced search</a><ul id="ui-id-1" tabindex="0" class="ui-menu ui-widget ui-widget-content ui-autocomplete ui-front" unselectable="on" style="display: none;"></ul></div><div class="quick-search__box"><div class="quick-search__box-inner"><div class="quick-search__box-wrapper"><select name="field1" aria-label="Select a search area"><option value="AllField">All content</option><option value="Title">Article title</option><option value="Contrib">Authors</option><option value="Keyword">Keywords</option><option value="Abstract">Abstract</option><option value="AbstractTitleKeywordFilterField">Article title, abstract, keywords</option></select><input type="hidden" name="journalCode" value="cell"><div class="quick-search__scope-options"><span>Search within:</span><label title="This journal" class="radio--primary"><input type="radio" name="SeriesKey" id="searchScopeSeries-a1f05748-166f-41e0-b01a-ca8ac1cc95d4" title="This journal" value="cell" checked="checked" aria-label="Search within current journal"><span>This journal</span></label><label title="Full site" class="radio--primary"><input type="radio" name="SeriesKey" id="searchScopeFullSite-a1f05748-166f-41e0-b01a-ca8ac1cc95d4" value="" title="Full site" aria-label="Search within full site"><span>Full site</span></label></div><a href="https://www.cell.com/search/advanced?SeriesKey=cell&amp;ISSN=0092-8674&amp;journalCode=cell" title="Advanced search" class="quick-search__advanced-link">Advanced search</a></div><a href="javascript:void(0)" aria-label="Close search options" title="Close search options" class="quick-search__box-close"><i aria-hidden="true" class="icon-tools_close"></i></a></div></div><div class="quick-search__alert hidden"><p>Please enter a term before submitting your search.</p><a href="#" class="quick-search__alert--close"> Ok</a></div></form></div></div>

            

        
</div>
  </div>
</nav>
<div id="page-body-id" class="page-body-flag"></div></div>
                </div>
                </div>

        

        </div>
    

    

        


    

            

            

            <div data-widget-id="f8c6daf9-9df1-4fd9-ba3a-7f5d7b3d4466" data-widget-name="UX3HTMLWidget"><style>
.journal-logos:nth-of-type(2) img {
max-height: 30px !important;
}
.journal-logos:nth-of-type(3) img {
max-height: 30px !important;
}
.ux3 .journal-logos + .journal-logos {
margin-left: 30px;
}
.header__wrapper.clearfix .col-xs-6:first-of-type {
width: 55%;
}
.header__wrapper.clearfix .col-xs-6:nth-of-type(2) {
width: 45%;
}
@media screen and (max-width: 1000px) {
.ux3 .journal-logos + .journal-logos {
display: none;
}
.header__wrapper.clearfix .col-xs-6:first-of-type {
width: 50%;
}
.header__wrapper.clearfix .col-xs-6:nth-of-type(2) {
width: 50%;
}
}

.ux3 .journal-logos:first-of-type {
   max-width: 500px;
}

.profile-links {
   white-space: nowrap;
}

@media only screen and (min-width: 740px) and (max-width: 1600px) {
   .journal-logos:nth-of-type(2) img {
      max-width: 250px !important;
   }
}

@media only screen and (min-width: 1024px) and (max-width: 1600px) {
   .header__wrapper.clearfix .col-xs-6:first-of-type {
      width: 60% !important;
   }
   .header__wrapper.clearfix .col-xs-6:nth-of-type(2) {
      width: 40% !important;
   }
}

@media only screen and (max-width: 1240px) {
   .ux3 .journal-logos + .journal-logos {
      display: none !important;
   }
}

</style></div>

            

        
</div></div>

            
                </div>
            

        

        </div>
    

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget axelPublicationContent none  widget-none" id="cadc9b41-a542-448a-9de4-8d0803320174">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none "><link id="build-style-article" rel="stylesheet" type="text/css" href="/products/marlin/cell/releasedAssets/css/build-article-b26f3342b722959c7931.css"><article xmlns="http://www.w3.org/1999/xhtml" data-design="pill" data-has="right-rail" vocab="http://schema.org/" typeof="ScholarlyArticle" data-pii="S0092-8674(22)00079-4" lang="en" dir="ltr"><div data-core-wrapper="header"><header data-extent="frontmatter"><div class="core-container"><div data-article-access="free" data-article-access-type="open" class="meta-panel"><span class="meta-panel__left-content"><span class="meta-panel__type"><span>Article</span></span></span><span class="meta-panel__middle-content"><span class="meta-panel__volumeIssue"><a href="/cell/issue?pii=S0092-8674(21)X0006-2"><span>Volume 185</span>, <span>Issue 5</span></a></span><span class="meta-panel__pages"><span>p794-814.e30</span></span><span class="meta-panel__onlineDate">March 03, 2022</span></span><span class="meta-panel__right-content"><span class="meta-panel__access meta-panel__access--archive"><span>Open Archive</span></span></span><div class="article-header__download-full-issue"><button title="Download full issue" type="submit" formaction="/action/downloadMultiplePdf?action=downloadIssuePdf&amp;journal=cell&amp;volume=185&amp;issue=5&amp;pii=S0092-8674%2821%29X0006-2" class="btn--download-issue btn--cancel download-full-issue"><i class="icon-download-new"></i><span>Download Full Issue</span></button><div class="table-of-content__alert table-of-content__alert--download"><p>Download started</p><a href="#" class="table-of-content__alert--close">Ok</a></div></div></div><h1 property="name">Transcription factor protein interactomes reveal genetic determinants in heart disease</h1><div class="contributors"><span class="authors"><span role="list"><span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au1" aria-controls="axel_author_au1"><span property="givenName">Barbara</span> <span property="familyName">Gonzalez-Teran</span></a><div id="au1" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Barbara</span> <span property="familyName">Gonzalez-Teran</span> <a class="orcid-id" href="https://orcid.org/0000-0002-4336-8644" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0002-4336-8644</a></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Gladstone Institutes, San Francisco, CA, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Roddenberry Center for Stem Cell Biology and Medicine at Gladstone, San Francisco, CA, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Gonzalez-Teran/Barbara" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">2</a></sup></span> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au2" aria-controls="axel_author_au2"><span property="givenName">Maureen</span> <span property="familyName">Pittman</span></a><div id="au2" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Maureen</span> <span property="familyName">Pittman</span> <a class="orcid-id" href="https://orcid.org/0000-0003-3714-991X" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0003-3714-991X</a></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Gladstone Institutes, San Francisco, CA, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Epidemiology &amp; Biostatistics, Institute for Computational Health Sciences, and Institute for Human Genetics, University of California San Francisco, San Francisco, CA, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Pittman/Maureen" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">19</a></sup></span> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au3" aria-controls="axel_author_au3"><span property="givenName">Franco</span> <span property="familyName">Felix</span></a><div id="au3" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Franco</span> <span property="familyName">Felix</span> <a class="orcid-id" href="https://orcid.org/0000-0002-4297-7149" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0002-4297-7149</a></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Gladstone Institutes, San Francisco, CA, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Roddenberry Center for Stem Cell Biology and Medicine at Gladstone, San Francisco, CA, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Felix/Franco" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">2</a></sup></span><span data-displayed-on="all"> ∙ … </span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au4" aria-controls="axel_author_au4"><span property="givenName">Reuben</span> <span property="familyName">Thomas</span></a><div id="au4" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Reuben</span> <span property="familyName">Thomas</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Gladstone Institutes, San Francisco, CA, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Thomas/Reuben" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au5" aria-controls="axel_author_au5"><span property="givenName">Desmond</span> <span property="familyName">Richmond-Buccola</span></a><div id="au5" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Desmond</span> <span property="familyName">Richmond-Buccola</span> <a class="orcid-id" href="https://orcid.org/0000-0001-5574-1891" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0001-5574-1891</a></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Gladstone Institutes, San Francisco, CA, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Roddenberry Center for Stem Cell Biology and Medicine at Gladstone, San Francisco, CA, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Richmond-Buccola/Desmond" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">2</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au6" aria-controls="axel_author_au6"><span property="givenName">Ruth</span> <span property="familyName">Hüttenhain</span></a><div id="au6" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Ruth</span> <span property="familyName">Hüttenhain</span> <a class="orcid-id" href="https://orcid.org/0000-0002-0896-5910" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0002-0896-5910</a></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Gladstone Institutes, San Francisco, CA, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Cellular and Molecular Pharmacology, University of California San Francisco, San Francisco, CA, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Quantitative Biosciences Institute (QBI), University of California San Francisco, San Francisco, CA, USA</span></div></div><div class="core-author-link"><a href="/authored-by/H%C3%BCttenhain/Ruth" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">3</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">4</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au7" aria-controls="axel_author_au7"><span property="givenName">Krishna</span> <span property="familyName">Choudhary</span></a><div id="au7" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au7"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Krishna</span> <span property="familyName">Choudhary</span> <a class="orcid-id" href="https://orcid.org/0000-0002-7966-1527" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0002-7966-1527</a></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Gladstone Institutes, San Francisco, CA, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Choudhary/Krishna" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au8" aria-controls="axel_author_au8"><span property="givenName">Elisabetta</span> <span property="familyName">Moroni</span></a><div id="au8" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au8"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Elisabetta</span> <span property="familyName">Moroni</span> <a class="orcid-id" href="https://orcid.org/0000-0002-7705-7457" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0002-7705-7457</a></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">SCITEC-CNR, Milano, Italy</span></div></div><div class="core-author-link"><a href="/authored-by/Moroni/Elisabetta" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">5</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au9" aria-controls="axel_author_au9"><span property="givenName">Mauro W.</span> <span property="familyName">Costa</span></a><div id="au9" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au9"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Mauro W.</span> <span property="familyName">Costa</span> <a class="orcid-id" href="https://orcid.org/0000-0003-3579-8625" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0003-3579-8625</a></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Gladstone Institutes, San Francisco, CA, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Roddenberry Center for Stem Cell Biology and Medicine at Gladstone, San Francisco, CA, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Costa/Mauro+W" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">2</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au10" aria-controls="axel_author_au10"><span property="givenName">Yu</span> <span property="familyName">Huang</span></a><div id="au10" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au10"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Yu</span> <span property="familyName">Huang</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Gladstone Institutes, San Francisco, CA, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Roddenberry Center for Stem Cell Biology and Medicine at Gladstone, San Francisco, CA, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Huang/Yu" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">2</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au11" aria-controls="axel_author_au11"><span property="givenName">Arun</span> <span property="familyName">Padmanabhan</span></a><div id="au11" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au11"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Arun</span> <span property="familyName">Padmanabhan</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Gladstone Institutes, San Francisco, CA, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Roddenberry Center for Stem Cell Biology and Medicine at Gladstone, San Francisco, CA, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Division of Cardiology, Department of Medicine, University of California, San Francisco, CA, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Padmanabhan/Arun" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">2</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">6</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au12" aria-controls="axel_author_au12"><span property="givenName">Michael</span> <span property="familyName">Alexanian</span></a><div id="au12" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au12"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Michael</span> <span property="familyName">Alexanian</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Gladstone Institutes, San Francisco, CA, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Roddenberry Center for Stem Cell Biology and Medicine at Gladstone, San Francisco, CA, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Alexanian/Michael" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">2</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au13" aria-controls="axel_author_au13"><span property="givenName">Clara Youngna</span> <span property="familyName">Lee</span></a><div id="au13" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au13"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Clara Youngna</span> <span property="familyName">Lee</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Gladstone Institutes, San Francisco, CA, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Roddenberry Center for Stem Cell Biology and Medicine at Gladstone, San Francisco, CA, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Lee/Clara+Youngna" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">2</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au14" aria-controls="axel_author_au14"><span property="givenName">Bonnie E.J.</span> <span property="familyName">Maven</span></a><div id="au14" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au14"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Bonnie E.J.</span> <span property="familyName">Maven</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Gladstone Institutes, San Francisco, CA, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Roddenberry Center for Stem Cell Biology and Medicine at Gladstone, San Francisco, CA, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Developmental and Stem Cell Biology Graduate Program, University of California San Francisco, San Francisco, CA, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Maven/Bonnie+E+J" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">2</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">7</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au15" aria-controls="axel_author_au15"><span property="givenName">Kaitlen</span> <span property="familyName">Samse-Knapp</span></a><div id="au15" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au15"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Kaitlen</span> <span property="familyName">Samse-Knapp</span> <a class="orcid-id" href="https://orcid.org/0000-0002-4116-7697" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0002-4116-7697</a></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Gladstone Institutes, San Francisco, CA, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Roddenberry Center for Stem Cell Biology and Medicine at Gladstone, San Francisco, CA, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Samse-Knapp/Kaitlen" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">2</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au16" aria-controls="axel_author_au16"><span property="givenName">Sarah U.</span> <span property="familyName">Morton</span></a><div id="au16" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au16"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Sarah U.</span> <span property="familyName">Morton</span> <a class="orcid-id" href="https://orcid.org/0000-0002-7816-2646" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0002-7816-2646</a></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Division of Newborn Medicine, Department of Medicine, Boston Children’s Hospital, Boston, MA, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Pediatrics, Harvard Medical School, Boston, MA, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Morton/Sarah+U" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">8</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">21</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au17" aria-controls="axel_author_au17"><span property="givenName">Michael</span> <span property="familyName">McGregor</span></a><div id="au17" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au17"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Michael</span> <span property="familyName">McGregor</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Gladstone Institutes, San Francisco, CA, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Cellular and Molecular Pharmacology, University of California San Francisco, San Francisco, CA, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Quantitative Biosciences Institute (QBI), University of California San Francisco, San Francisco, CA, USA</span></div></div><div class="core-author-link"><a href="/authored-by/McGregor/Michael" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">3</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">4</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au18" aria-controls="axel_author_au18"><span property="givenName">Casey A.</span> <span property="familyName">Gifford</span></a><div id="au18" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au18"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Casey A.</span> <span property="familyName">Gifford</span> <a class="orcid-id" href="https://orcid.org/0000-0002-7757-573X" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0002-7757-573X</a></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Gladstone Institutes, San Francisco, CA, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Roddenberry Center for Stem Cell Biology and Medicine at Gladstone, San Francisco, CA, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Gifford/Casey+A" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">2</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au19" aria-controls="axel_author_au19"><span property="givenName">J.G.</span> <span property="familyName">Seidman</span></a><div id="au19" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au19"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">J.G.</span> <span property="familyName">Seidman</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Genetics, Harvard Medical School, Boston, MA, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Seidman/J+G" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">9</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au20" aria-controls="axel_author_au20"><span property="givenName">Christine E.</span> <span property="familyName">Seidman</span></a><div id="au20" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au20"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Christine E.</span> <span property="familyName">Seidman</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Genetics, Harvard Medical School, Boston, MA, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Howard Hughes Medical Institute, Harvard Medical School, Boston, MA, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Cardiovascular Division, Brigham and Women’s Hospital, Boston, MA, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Seidman/Christine+E" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">9</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">10</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">11</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au21" aria-controls="axel_author_au21"><span property="givenName">Bruce D.</span> <span property="familyName">Gelb</span></a><div id="au21" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au21"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Bruce D.</span> <span property="familyName">Gelb</span> <a class="orcid-id" href="https://orcid.org/0000-0001-8527-5027" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0001-8527-5027</a></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Genetics and Genomic Sciences, Icahn School of Medicine at Mount Sinai, New York, NY, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Mindich Child Health and Development Institute, Icahn School of Medicine at Mount Sinai, New York, NY, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Pediatrics, Icahn School of Medicine at Mount Sinai, New York, NY, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Gelb/Bruce+D" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">12</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">13</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">14</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au22" aria-controls="axel_author_au22"><span property="givenName">Giorgio</span> <span property="familyName">Colombo</span></a><div id="au22" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au22"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Giorgio</span> <span property="familyName">Colombo</span> <a class="orcid-id" href="https://orcid.org/0000-0002-1318-668X" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0002-1318-668X</a></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">University of Pavia, Department of Chemistry, Pavia, Italy</span></div></div><div class="core-author-link"><a href="/authored-by/Colombo/Giorgio" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">15</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au23" aria-controls="axel_author_au23"><span property="givenName">Bruce R.</span> <span property="familyName">Conklin</span></a><div id="au23" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au23"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Bruce R.</span> <span property="familyName">Conklin</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Gladstone Institutes, San Francisco, CA, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Roddenberry Center for Stem Cell Biology and Medicine at Gladstone, San Francisco, CA, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Conklin/Bruce+R" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">2</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au24" aria-controls="axel_author_au24"><span property="givenName">Brian L.</span> <span property="familyName">Black</span></a><div id="au24" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au24"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Brian L.</span> <span property="familyName">Black</span> <a class="orcid-id" href="https://orcid.org/0000-0002-6664-8913" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0002-6664-8913</a></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Cardiovascular Research Institute, University of California San Francisco, San Francisco, CA, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Black/Brian+L" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">16</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au25" aria-controls="axel_author_au25"><span property="givenName">Benoit G.</span> <span property="familyName">Bruneau</span></a><div id="au25" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au25"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Benoit G.</span> <span property="familyName">Bruneau</span> <a class="orcid-id" href="https://orcid.org/0000-0002-0804-7597" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0002-0804-7597</a></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Gladstone Institutes, San Francisco, CA, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Roddenberry Center for Stem Cell Biology and Medicine at Gladstone, San Francisco, CA, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Cardiovascular Research Institute, University of California San Francisco, San Francisco, CA, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Division of Cardiology, Department of Pediatrics, UCSF School of Medicine, San Francisco, CA, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Bruneau/Benoit+G" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">2</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">16</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">17</a></sup></span></span> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au26" aria-controls="axel_author_au26"><span property="givenName">Nevan J.</span> <span property="familyName">Krogan</span></a><div id="au26" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au26"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Nevan J.</span> <span property="familyName">Krogan</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Gladstone Institutes, San Francisco, CA, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Cellular and Molecular Pharmacology, University of California San Francisco, San Francisco, CA, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Quantitative Biosciences Institute (QBI), University of California San Francisco, San Francisco, CA, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Krogan/Nevan+J" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">3</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">4</a></sup></span> ∙ <span property="author" typeof="Person" role="listitem" class="corresponding-author"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au27" aria-controls="axel_author_au27"><span property="givenName">Katherine S.</span> <span property="familyName">Pollard</span></a><div id="au27" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au27"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Katherine S.</span> <span property="familyName">Pollard</span> <a class="orcid-id" href="https://orcid.org/0000-0002-9870-6196" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0002-9870-6196</a></div><div class="content"><div class="core-author-emails"><div class="label">Correspondence</div><div>Corresponding author</div><div><a id="eadd0010" href="mailto:katherine.pollard@gladstone.ucsf.edu" property="email" title="Send email to Katherine S. Pollard">katherine.pollard@gladstone.ucsf.edu</a></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Gladstone Institutes, San Francisco, CA, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Chan Zuckerberg Biohub, San Francisco, CA, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Epidemiology &amp; Biostatistics, Institute for Computational Health Sciences, and Institute for Human Genetics, University of California San Francisco, San Francisco, CA, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Pollard/Katherine+S" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">18</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">19</a></sup> <a id="eadd0010" href="mailto:katherine.pollard@gladstone.ucsf.edu" property="email" title="Send email to Katherine S. Pollard">katherine.pollard@gladstone.ucsf.edu</a></span> ∙ <span property="author" typeof="Person" role="listitem" class="corresponding-author"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au28" aria-controls="axel_author_au28"><span property="givenName">Deepak</span> <span property="familyName">Srivastava</span></a><div id="au28" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au28"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Deepak</span> <span property="familyName">Srivastava</span></div><div class="content"><div class="core-author-emails"><div class="label">Correspondence</div><div>Corresponding author</div><div><a id="eadd0015" href="mailto:deepak.srivastava@gladstone.ucsf.edu" property="email" title="Send email to Deepak Srivastava">deepak.srivastava@gladstone.ucsf.edu</a></div></div><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">22</div><div id="fn1"><div id="ntpara0010" role="paragraph">Lead contact</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Gladstone Institutes, San Francisco, CA, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Roddenberry Center for Stem Cell Biology and Medicine at Gladstone, San Francisco, CA, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Division of Cardiology, Department of Pediatrics, UCSF School of Medicine, San Francisco, CA, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Biochemistry and Biophysics, University of California San Francisco, San Francisco, CA, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Srivastava/Deepak" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">2</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">17</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">20</a>,<a role="doc-noteref">22</a></sup> <a id="eadd0015" href="mailto:deepak.srivastava@gladstone.ucsf.edu" property="email" title="Send email to Deepak Srivastava">deepak.srivastava@gladstone.ucsf.edu</a></span><span data-displayed-on="all"> … <span><span data-action="reveal" tabindex="0" role="listitem">Show more</span></span> </span><span data-displayed-on="none" data-on-display="all" aria-hidden="true"> <span data-action="hide" tabindex="0" role="listitem">Show less</span></span></span></span></div><div class="core-self-citation"><div class="expandable-triggers"><button aria-controls="core-affiliations-notes" aria-expanded="false" tabindex="0" role="button"><span>Affiliations &amp; Notes</span><i class="icon-arrow_d_n" aria-hidden="true"></i></button><button aria-controls="core-content-info" aria-expanded="false" tabindex="0" role="button"><span>Article Info</span><i class="icon-arrow_d_n" aria-hidden="true"></i></button></div><div id="core-linked-content">
    

        

</div><div class="expandable-content"><div id="core-affiliations-notes" role="region" data-expandable="item" style="display: none;"><div class="affiliations"><div property="affiliation" typeof="Organization"><span class="label">1</span><span id="aff1" property="name">Gladstone Institutes, San Francisco, CA, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">2</span><span id="aff2" property="name">Roddenberry Center for Stem Cell Biology and Medicine at Gladstone, San Francisco, CA, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">3</span><span id="aff3" property="name">Department of Cellular and Molecular Pharmacology, University of California San Francisco, San Francisco, CA, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">4</span><span id="aff4" property="name">Quantitative Biosciences Institute (QBI), University of California San Francisco, San Francisco, CA, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">5</span><span id="aff5" property="name">SCITEC-CNR, Milano, Italy</span></div><div property="affiliation" typeof="Organization"><span class="label">6</span><span id="aff6" property="name">Division of Cardiology, Department of Medicine, University of California, San Francisco, CA, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">7</span><span id="aff7" property="name">Developmental and Stem Cell Biology Graduate Program, University of California San Francisco, San Francisco, CA, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">8</span><span id="aff8" property="name">Division of Newborn Medicine, Department of Medicine, Boston Children’s Hospital, Boston, MA, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">9</span><span id="aff9" property="name">Department of Genetics, Harvard Medical School, Boston, MA, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">10</span><span id="aff10" property="name">Howard Hughes Medical Institute, Harvard Medical School, Boston, MA, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">11</span><span id="aff11" property="name">Cardiovascular Division, Brigham and Women’s Hospital, Boston, MA, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">12</span><span id="aff12" property="name">Department of Genetics and Genomic Sciences, Icahn School of Medicine at Mount Sinai, New York, NY, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">13</span><span id="aff13" property="name">Mindich Child Health and Development Institute, Icahn School of Medicine at Mount Sinai, New York, NY, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">14</span><span id="aff14" property="name">Department of Pediatrics, Icahn School of Medicine at Mount Sinai, New York, NY, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">15</span><span id="aff15" property="name">University of Pavia, Department of Chemistry, Pavia, Italy</span></div><div property="affiliation" typeof="Organization"><span class="label">16</span><span id="aff16" property="name">Cardiovascular Research Institute, University of California San Francisco, San Francisco, CA, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">17</span><span id="aff17" property="name">Division of Cardiology, Department of Pediatrics, UCSF School of Medicine, San Francisco, CA, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">18</span><span id="aff18" property="name">Chan Zuckerberg Biohub, San Francisco, CA, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">19</span><span id="aff19" property="name">Department of Epidemiology &amp; Biostatistics, Institute for Computational Health Sciences, and Institute for Human Genetics, University of California San Francisco, San Francisco, CA, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">20</span><span id="aff20" property="name">Department of Biochemistry and Biophysics, University of California San Francisco, San Francisco, CA, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">21</span><span id="aff21" property="name">Department of Pediatrics, Harvard Medical School, Boston, MA, USA</span></div></div><div class="core-authors-notes"><div role="doc-footnote" data-has="label"><div class="label">22</div><div id="fn1"><div id="ntpara0010" role="paragraph">Lead contact</div></div></div></div></div><div id="core-content-info" role="region" data-expandable="item" style="display: none;"><div class="history"><div>Publication History:</div><div><span>Received November 25, 2020</span>; <span>Revised August 20, 2021</span>; <span>Accepted January 25, 2022</span>; <span>Published online February 18, 2022</span></div></div><div class="core-links"><span class="doi">DOI: <a href="https://doi.org/10.1016/j.cell.2022.01.021" property="sameAs" target="_blank">10.1016/j.cell.2022.01.021 <span class="sr-only">External Link</span></a></span><span class="science-direct">Also available on <a href="https://www.sciencedirect.com/science/article/pii/S0092867422000794" target="_blank">ScienceDirect <span class="sr-only">External Link</span></a></span></div><div role="paragraph">Copyright: © 2022 Elsevier Inc.</div><div class="core-links"><span>User License: <a href="http://www.elsevier.com/open-access/userlicense/1.0/" target="_blank">Elsevier user license</a> | <a href="https://www.elsevier.com/open-access" target="_blank">Elsevier's open access license policy</a></span></div><div class="core-crossmark-badge"><a data-target="crossmark" href="#" title="Check for updates on crossmark" data-doi="10.1016/j.cell.2022.01.021" data-id="article-info-crossmark" class="crossmark__link" style="cursor: pointer;"><svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 290 58" height="22px" role="presentation"><path fill="url(#horizontal_crossmark)" stroke="#948F8F" stroke-miterlimit="10" stroke-width=".75" d="M287 1H3a2 2 0 0 0-2 2v52a2 2 0 0 0 2 2h284a2 2 0 0 0 2-2V3a2 2 0 0 0-2-2Z"></path><path fill="#535353" d="M80.3 23.58a6.395 6.395 0 0 0-.55-1.43 3.92 3.92 0 0 0-2.19-1.86 5.099 5.099 0 0 0-1.71-.29 5 5 0 0 0-2.46.58 4.831 4.831 0 0 0-1.69 1.54 6.921 6.921 0 0 0-1 2.19 10.16 10.16 0 0 0 0 5 6.921 6.921 0 0 0 1 2.19 4.83 4.83 0 0 0 1.69 1.5 5 5 0 0 0 2.46.58 4.44 4.44 0 0 0 1.85-.36 4.12 4.12 0 0 0 1.38-1c.39-.436.695-.942.9-1.49A7.094 7.094 0 0 0 80.4 29h3a8.46 8.46 0 0 1-.69 2.92 7.179 7.179 0 0 1-1.59 2.29 7 7 0 0 1-2.35 1.49 8.14 8.14 0 0 1-3 .52 8.43 8.43 0 0 1-3.59-.74 7.88 7.88 0 0 1-2.69-2 9.001 9.001 0 0 1-1.69-3 11.41 11.41 0 0 1 0-7.23 9.001 9.001 0 0 1 1.69-3 8.001 8.001 0 0 1 2.69-2 8.3 8.3 0 0 1 3.59-.75 9.44 9.44 0 0 1 2.83.41c.866.242 1.68.642 2.4 1.18A6.229 6.229 0 0 1 82.65 21a6.66 6.66 0 0 1 .78 2.58H80.3ZM85.9 17.9h2.85v6.63h.05a4.15 4.15 0 0 1 1.56-1.46 4.61 4.61 0 0 1 2.31-.59 4.83 4.83 0 0 1 3.33 1.1 4.21 4.21 0 0 1 1.22 3.3v8.88H94.4v-8.13a3.51 3.51 0 0 0-.65-2.21 2.36 2.36 0 0 0-1.87-.69 3.1 3.1 0 0 0-1.3.26 2.821 2.821 0 0 0-1 .73A3.41 3.41 0 0 0 89 26.8a3.88 3.88 0 0 0-.23 1.33v7.63H85.9V17.9ZM102.47 30c-.002.491.069.98.21 1.45.131.448.348.866.64 1.23a3.08 3.08 0 0 0 1.08.84 3.543 3.543 0 0 0 1.55.31 3.431 3.431 0 0 0 2-.54c.541-.402.94-.966 1.14-1.61h2.7A5.349 5.349 0 0 1 109.7 35a5.715 5.715 0 0 1-1.74.85 7 7 0 0 1-2 .29 6.848 6.848 0 0 1-2.7-.5 5.577 5.577 0 0 1-2-1.4A5.953 5.953 0 0 1 100 32a8.748 8.748 0 0 1-.41-2.75 7.768 7.768 0 0 1 .44-2.61 6.67 6.67 0 0 1 1.25-2.17 5.863 5.863 0 0 1 4.56-2 5.79 5.79 0 0 1 2.74.64c.783.41 1.466.986 2 1.69a6.824 6.824 0 0 1 1.16 2.41c.264.905.352 1.852.26 2.79h-9.53Zm6.65-1.87a4.605 4.605 0 0 0-.29-1.3c-.148-.4-.368-.77-.65-1.09a3.22 3.22 0 0 0-1-.75 3 3 0 0 0-1.31-.29 3.431 3.431 0 0 0-1.36.26 3.108 3.108 0 0 0-1 .73 3.594 3.594 0 0 0-.7 1.09 3.798 3.798 0 0 0-.29 1.35h6.65-.05ZM122.88 27.15a2.666 2.666 0 0 0-1-1.81 3.14 3.14 0 0 0-2-.61c-.4.005-.798.07-1.18.19-.435.136-.83.376-1.15.7a3.92 3.92 0 0 0-.87 1.43 6.783 6.783 0 0 0-.35 2.39c.002.539.065 1.076.19 1.6.113.509.316.993.6 1.43.273.41.635.752 1.06 1a3 3 0 0 0 1.58.39 2.872 2.872 0 0 0 2.06-.77 3.596 3.596 0 0 0 1-2.18h2.85a6.26 6.26 0 0 1-1.91 3.86 5.83 5.83 0 0 1-4 1.34 6.61 6.61 0 0 1-2.69-.51 5.695 5.695 0 0 1-2-1.4 6.001 6.001 0 0 1-1.21-2.11 8.279 8.279 0 0 1-.41-2.65 8.987 8.987 0 0 1 .4-2.72 6.237 6.237 0 0 1 1.2-2.21 5.614 5.614 0 0 1 2-1.47 6.7 6.7 0 0 1 2.79-.54 7.801 7.801 0 0 1 2.14.29 5.634 5.634 0 0 1 1.8.86c.529.386.969.88 1.29 1.45a5 5 0 0 1 .6 2.07h-2.85l.06-.02ZM128.13 17.9H131V28l5.15-5.22h3.5l-5 4.75 5.42 8.17h-3.47l-4-6.27-1.6 1.7v4.63h-2.85V17.9h-.02ZM147.15 22.83h2.13v-1.08a5.505 5.505 0 0 1 .3-2c.161-.456.436-.862.8-1.18.33-.273.721-.462 1.14-.55.44-.094.89-.141 1.34-.14a8.158 8.158 0 0 1 2 .18v2.24a4.113 4.113 0 0 0-.59-.11 6.343 6.343 0 0 0-.74 0 1.74 1.74 0 0 0-1 .27 1.204 1.204 0 0 0-.41 1.08v1.33h2.42V25h-2.42v10.8h-2.85V25h-2.12v-2.17ZM162.23 36.1a6.999 6.999 0 0 1-2.76-.51 5.877 5.877 0 0 1-2-1.41 5.998 5.998 0 0 1-1.33-2.18 8.775 8.775 0 0 1 0-5.48 6.008 6.008 0 0 1 1.27-2.15 5.894 5.894 0 0 1 2-1.41A7.71 7.71 0 0 1 165 23a5.894 5.894 0 0 1 2 1.41 6.003 6.003 0 0 1 1.28 2.15c.585 1.78.585 3.7 0 5.48a5.99 5.99 0 0 1-3.28 3.54 7.002 7.002 0 0 1-2.77.52Zm0-2.25a3.268 3.268 0 0 0 1.65-.4c.447-.262.83-.621 1.12-1.05.304-.444.528-.938.66-1.46a6.581 6.581 0 0 0 0-3.31 4.416 4.416 0 0 0-.66-1.46 3.526 3.526 0 0 0-1.15-1 3.6 3.6 0 0 0-3.3 0 3.526 3.526 0 0 0-1.15 1 4.414 4.414 0 0 0-.66 1.46 6.558 6.558 0 0 0 0 3.31c.132.522.356 1.016.66 1.46a3.47 3.47 0 0 0 1.15 1.05 3.26 3.26 0 0 0 1.68.4ZM171.13 22.83h2.67v2.5c.097-.362.264-.701.49-1 .251-.341.547-.647.88-.91a4.404 4.404 0 0 1 1.14-.66 3.41 3.41 0 0 1 1.28-.25c.23-.013.46-.013.69 0h.39v2.79l-.61-.09a5.173 5.173 0 0 0-.61 0 3.278 3.278 0 0 0-1.36.29 3.131 3.131 0 0 0-1.11.85 4.171 4.171 0 0 0-.75 1.39 6 6 0 0 0-.28 1.9v6.15h-2.85V22.83h.03ZM198.22 35.75h-2.8V34a3.907 3.907 0 0 1-1.56 1.56 4.25 4.25 0 0 1-2.11.59 4.742 4.742 0 0 1-3.75-1.31 5.537 5.537 0 0 1-1.12-3.84v-8.2h2.85v7.92a3.503 3.503 0 0 0 .65 2.4 2.38 2.38 0 0 0 1.83.7 3.599 3.599 0 0 0 1.5-.27 2.66 2.66 0 0 0 1-.74c.259-.326.444-.705.54-1.11a5.79 5.79 0 0 0 .16-1.4v-7.5h2.85v12.95h-.04ZM201.3 22.83h2.7v1.75a3.523 3.523 0 0 1 1.72-1.58 5.568 5.568 0 0 1 2.33-.49 6.17 6.17 0 0 1 2.66.54 5.246 5.246 0 0 1 1.89 1.47c.515.65.899 1.394 1.13 2.19.257.87.385 1.773.38 2.68.001.851-.114 1.7-.34 2.52a6.568 6.568 0 0 1-1 2.16 5.002 5.002 0 0 1-4.21 2.06c-.419 0-.837-.036-1.25-.11a5.39 5.39 0 0 1-1.2-.36A4.737 4.737 0 0 1 205 35a3.567 3.567 0 0 1-.81-.91v6.45h-2.89V22.83Zm10 6.47c0-.574-.078-1.146-.23-1.7a4.606 4.606 0 0 0-.67-1.46 3.475 3.475 0 0 0-1.12-1 3.06 3.06 0 0 0-1.55-.39A3.141 3.141 0 0 0 205 26a5.518 5.518 0 0 0-1 3.3c-.004.611.077 1.22.24 1.81.138.523.38 1.012.71 1.44a3.55 3.55 0 0 0 1.14 1 3.242 3.242 0 0 0 1.54.35 3.18 3.18 0 0 0 1.65-.4c.438-.25.817-.59 1.11-1 .299-.44.513-.931.63-1.45a7.38 7.38 0 0 0 .23-1.75h.05ZM228.67 35.75H226V34a3.38 3.38 0 0 1-1.67 1.61 5.668 5.668 0 0 1-2.33.49 6.17 6.17 0 0 1-2.66-.54 5.307 5.307 0 0 1-1.89-1.46 6.254 6.254 0 0 1-1.12-2.19 9.48 9.48 0 0 1-.37-2.71 8.67 8.67 0 0 1 .47-3 6.058 6.058 0 0 1 1.26-2.1 4.89 4.89 0 0 1 1.8-1.21 5.738 5.738 0 0 1 2.06-.39c.412 0 .824.037 1.23.11.413.073.815.194 1.2.36.38.166.736.381 1.06.64.32.255.594.562.81.91V17.9h2.85v17.85h-.03Zm-10-6.33c-.001.55.069 1.098.21 1.63.13.511.35.996.65 1.43.288.414.668.757 1.11 1a3.241 3.241 0 0 0 1.6.38 3.199 3.199 0 0 0 1.64-.4 3.398 3.398 0 0 0 1.13-1.05c.297-.447.517-.94.65-1.46.14-.542.211-1.1.21-1.66a5.227 5.227 0 0 0-1-3.35 3.175 3.175 0 0 0-2.61-1.2 3.23 3.23 0 0 0-1.69.41 3.478 3.478 0 0 0-1.12 1.07 4.494 4.494 0 0 0-.62 1.5 7.871 7.871 0 0 0-.11 1.7h-.05ZM242.6 32.88c-.018.258.03.516.14.75a.577.577 0 0 0 .54.23h.3c.135-.002.269-.019.4-.05v2l-.39.11-.49.08-.5.08h-.42a2.75 2.75 0 0 1-1.45-.35 1.731 1.731 0 0 1-.75-1.23 4.998 4.998 0 0 1-2.09 1.2 8.224 8.224 0 0 1-2.39.38 5.861 5.861 0 0 1-1.67-.24 4.394 4.394 0 0 1-1.41-.7 3.35 3.35 0 0 1-1-1.18 3.624 3.624 0 0 1-.36-1.66 3.84 3.84 0 0 1 .44-2 3.246 3.246 0 0 1 1.15-1.17 5.195 5.195 0 0 1 1.6-.61c.593-.127 1.19-.223 1.79-.29a14.77 14.77 0 0 1 1.47-.21 6.896 6.896 0 0 0 1.24-.21c.318-.077.611-.236.85-.46a1.286 1.286 0 0 0 .31-.94 1.458 1.458 0 0 0-.26-.9A1.759 1.759 0 0 0 239 25a2.807 2.807 0 0 0-.86-.25 6.941 6.941 0 0 0-.9-.06 3.58 3.58 0 0 0-2 .5 2 2 0 0 0-.87 1.55h-2.85c.021-.73.227-1.442.6-2.07.338-.54.797-.996 1.34-1.33a5.475 5.475 0 0 1 1.84-.7c.692-.134 1.395-.2 2.1-.2.632 0 1.262.067 1.88.2a5.337 5.337 0 0 1 1.66.65 3.6 3.6 0 0 1 1.19 1.16c.314.525.47 1.129.45 1.74v6.65l.02.04Zm-2.85-3.6a3.45 3.45 0 0 1-1.6.51c-.667.06-1.3.147-1.9.26a6.137 6.137 0 0 0-.87.21 2.64 2.64 0 0 0-.75.38c-.22.164-.395.38-.51.63a2.129 2.129 0 0 0-.19.94c-.011.29.085.575.27.8.182.215.407.388.66.51.269.128.555.216.85.26.277.05.558.076.84.08a4.63 4.63 0 0 0 1-.12 3.418 3.418 0 0 0 1-.43 2.73 2.73 0 0 0 .81-.76 1.93 1.93 0 0 0 .32-1.14v-2.14l.07.01ZM244.37 22.83h2.15V19h2.85v3.88h2.58V25h-2.58v6.9a7.064 7.064 0 0 0 0 .77c.016.194.074.381.17.55a.84.84 0 0 0 .41.34 2 2 0 0 0 .75.11h.6c.203-.01.404-.04.6-.09v2.2l-.93.1a8.403 8.403 0 0 1-.92 0 6.33 6.33 0 0 1-1.81-.21 2.421 2.421 0 0 1-1.08-.62 2.15 2.15 0 0 1-.52-1 7.791 7.791 0 0 1-.16-1.42V25h-2.15v-2.17h.04ZM256.22 30c-.002.491.069.98.21 1.45.131.448.348.866.64 1.23a3.08 3.08 0 0 0 1.08.84 3.543 3.543 0 0 0 1.55.31 3.431 3.431 0 0 0 2-.54c.541-.402.94-.966 1.14-1.61h2.7a5.352 5.352 0 0 1-2.1 3.26 5.715 5.715 0 0 1-1.74.85 7 7 0 0 1-2 .29 6.848 6.848 0 0 1-2.7-.5 5.577 5.577 0 0 1-2-1.4 5.947 5.947 0 0 1-1.21-2.18 8.754 8.754 0 0 1-.41-2.75 7.758 7.758 0 0 1 .44-2.61 6.67 6.67 0 0 1 1.25-2.17 5.863 5.863 0 0 1 4.56-2 5.79 5.79 0 0 1 2.74.64c.783.41 1.466.986 2 1.69a6.824 6.824 0 0 1 1.16 2.41c.243.916.308 1.87.19 2.81h-9.5V30Zm6.65-1.87a4.605 4.605 0 0 0-.29-1.3c-.148-.4-.368-.77-.65-1.09a3.22 3.22 0 0 0-1-.75 3 3 0 0 0-1.31-.29 3.431 3.431 0 0 0-1.36.26 3.108 3.108 0 0 0-1 .73 3.575 3.575 0 0 0-.7 1.09 3.798 3.798 0 0 0-.29 1.35h6.65-.05ZM270 31.6a2.154 2.154 0 0 0 .95 1.75c.607.35 1.3.524 2 .5.304-.002.608-.022.91-.06a3.943 3.943 0 0 0 1-.24c.286-.105.54-.28.74-.51a1.23 1.23 0 0 0 .26-.89 1.261 1.261 0 0 0-.4-.9 2.779 2.779 0 0 0-1-.56 9.315 9.315 0 0 0-1.34-.36l-1.5-.33a14.219 14.219 0 0 1-1.54-.42 4.937 4.937 0 0 1-1.33-.68 3.12 3.12 0 0 1-.94-1.09 3.472 3.472 0 0 1-.35-1.64 2.94 2.94 0 0 1 .51-1.76 3.913 3.913 0 0 1 1.3-1.15 5.66 5.66 0 0 1 1.75-.61 10.316 10.316 0 0 1 1.84-.17 8.418 8.418 0 0 1 1.91.21 5.19 5.19 0 0 1 1.65.69c.496.318.916.741 1.23 1.24.341.559.55 1.188.61 1.84h-3a1.783 1.783 0 0 0-.94-1.37 3.892 3.892 0 0 0-1.69-.35 6.093 6.093 0 0 0-.71 0 3.327 3.327 0 0 0-.78.19 1.642 1.642 0 0 0-.61.4 1 1 0 0 0-.25.69 1.093 1.093 0 0 0 .36.85c.294.245.634.428 1 .54.437.153.885.274 1.34.36l1.55.33a20.9 20.9 0 0 1 1.53.42c.479.157.931.386 1.34.68.41.277.751.643 1 1.07.255.494.379 1.045.36 1.6a3.48 3.48 0 0 1-.52 2 4.15 4.15 0 0 1-1.36 1.3 6.19 6.19 0 0 1-1.86.73 9.435 9.435 0 0 1-2 .23 8.823 8.823 0 0 1-2.26-.27 5.265 5.265 0 0 1-1.8-.84 4 4 0 0 1-1.2-1.4 4.445 4.445 0 0 1-.46-2h2.7v-.02Z"></path><path fill="#C72914" d="m21.94 38.74 18.12-12.08v-14.5H21.94v26.58Z"></path><path fill="#EF3340" d="M40.06 38.74 21.94 26.66v-14.5h18.12v26.58Z"></path><path fill="#3EB1C8" d="M31 9a20 20 0 1 0 0 40 20 20 0 0 0 0-40Zm0 35.1a15.1 15.1 0 1 1 0-30.2 15.1 15.1 0 0 1 0 30.2Z"></path><path fill="#FFC72C" d="M42.09 39.23A15.09 15.09 0 0 1 17 23.38l-4.2-2.64a20 20 0 0 0 33.49 21.14l-4.2-2.65Z"></path><defs><linearGradient id="horizontal_crossmark" x1="145" x2="145" y1="7.33" y2="56.82" gradientUnits="userSpaceOnUse"><stop stop-color="#fff"></stop><stop offset="1" stop-color="#c4c4c4"></stop></linearGradient></defs></svg></a></div><div class="core-content-info"></div></div></div></div></div></header><div class="core-header-aside">
    

        


    

            

            

            <div class="cover-image cover-image--axel"><div class="cover-image__thumb"><a href="https://www.cell.com/cell/issue?pii=S0092-8674(21)X0006-2"><img src="/cms/asset/0de86198-e5d8-4af8-8ddf-367f4a9918c1/cov200h.gif" alt="Cover Image - Cell, Volume 185, Issue 5"></a></div></div>

            

        
</div></div><div data-core-wrapper="content"><div id="dummy-sticky-observer"></div><div data-core-nav="header" data-extent="frontmatter"><div class="core-nav-wrapper core-container" aria-label="Article navigation"><div class="navigation-menu-pdf">
    

        


    

            

            

            <div data-widget-id="ecff07e8-0bc4-4533-b1f1-49365e2ead20" data-widget-name="UX3ArticleToolsWidget" class="article-tools"><a href="/action/showPdf?pii=S0092-8674%2822%2900079-4" title="Standard PDF (opens in a new tab)" target="_blank" aria-hidden="false" role="menuitem" class="article-tools__item__pdf"><svg focusable="false" viewBox="0 0 32 32" width="20" height="20" title="PDF" class="pdf-icon"><path id="pdfwhite" d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path id="pdfda0000" d="M.167 2.592H22.39V9.72H.166z" fill="#da0000"></path><path id="pdffff9f9" fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path id="pdff91d0a" d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="basic">Download PDF</span><span class="alternative">Download PDF</span></a></div>

            

        
</div><div class="core-sections-menu"><button data-id="article-toolbar-showhide" aria-controls="article_sections_menu" aria-expanded="false" aria-haspopup="true" aria-label="Toggle section navigation menu" style="margin-top: 0px;"><i class="icon-list" aria-hidden="true"></i><span>Outline</span></button><nav id="article_sections_menu" aria-label="Contents" data-core-nav="article" style="margin-top: 0px;"><div class="heading">Outline<button aria-haspopup="true" aria-controls="article_sections_menu" title="Close outline menu"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div><ul><li><a href="#author-highlights-abstract">Highlights</a></li><li><a href="#author-abstract">Summary</a></li><li><a href="#graphical-abstract">Graphical abstract</a></li><li><a href="#keywords">Keywords</a></li><li><a href="#sec-1">Introduction</a></li><li><a href="#sec-2">Results</a></li><li><a href="#sec-3">Discussion</a></li><li><a href="#sec-4">STAR★Methods</a></li><li><a href="#acknowledgments">Acknowledgments</a></li><li><a href="#supplementary-material">Supplemental information (1)</a></li><li><a href="#data-availability">Data and code availability</a></li><li><a href="#references">References</a></li><li role="none" class="mobile-only"><a role="menuitem" href="#core-collateral-metrics">Article metrics</a></li><li role="none" class="mobile-only"><a role="menuitem" href="#core-collateral-relatedArticles">Related Articles</a></li></ul></nav></div><div class="navigation-menu-share">
    

        


    

            

            

            <!-- Go to https://www.addtoany.com/buttons/customize/ to customize your tools --><script type="text/javascript" defer="defer" src="https://static.addtoany.com/menu/page.js"></script><script type="text/javascript">var a2a_config = a2a_config || {};
a2a_config.thanks = {
    postShare: false,
    ad: false,
};</script><script>var a2a_config = a2a_config || {};
let res = window.location.href.replace(/\(/g, "%28").replace(/\)/g, "%29");
a2a_config.templates = a2a_config.templates || {};

a2a_config.templates.email = {
    body: res
};
</script><div data-widget-id="5f978446-c80a-450c-a185-94d633df2db2" data-widget-name="UX3share" aria-hidden="false" role="group" class="article-tools__dropblock article-tools__share"><button aria-label="Toggle share menu" aria-expanded="false" aria-haspopup="true" role="menuitem" aria-controls="navigation_menu_share_block" aria-labelledby="share-label" class="article-tools__item__share"><i aria-hidden="true" class="icon-article-share"></i><span id="share-label">Share</span></button><div id="navigation_menu_share_block" role="menu" class="share__block axel-share article-tools__dropblock__holder"><h4 class="axel-share__title heading">Share<button title="Close share menu" aria-expanded="false" aria-haspopup="true" aria-controls="navigation_menu_share_block"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></h4><div class="axel-share__inner"><span class="sr-only">Share on</span><ul class="a2a__share-list rlist w-slide--list a2a a2a_kit a2a_kit_size_32 a2a_default_style" data-a2a-url="https://www.cell.com/cell/fulltext/S0092-8674%2822%2900079-4" style="line-height: 32px;"><li class="a2a_listitem_custom"><a role="link" title="Email" class="a2a_button_email" target="_blank" rel="nofollow noopener" href="/#email"><i aria-hidden="true" class="at-icon-wrapper icon-gizmo-email-plus"></i><span>Email</span></a></li><li class="a2a_listitem_custom"><a role="link" title="formerly known as Twitter" class="a2a_button_twitter" target="_blank" rel="nofollow noopener" href="/#twitter"><i aria-hidden="true" class="at-icon-wrapper icon-x-default"></i><span>X</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Facebook" class="a2a_button_facebook" target="_blank" rel="nofollow noopener" href="/#facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook-square-2"></i><span>Facebook</span></a></li><li class="a2a_listitem_custom"><a role="link" title="LinkedIn" class="a2a_button_linkedin" target="_blank" rel="nofollow noopener" href="/#linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin-square"></i><span>LinkedIn</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Sina Weibo" class="a2a_button_sina_weibo" target="_blank" rel="nofollow noopener" href="/#sina_weibo"><i aria-hidden="true" class="at-icon-wrapper icon-weibo"></i><span>Sina Weibo</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Mendeley" class="a2a_button_mendeley" target="_blank" rel="nofollow noopener" href="/#mendeley"><i aria-hidden="true" class="at-icon-wrapper icon-mendeley-2"></i><span>Mendeley</span></a></li><li class="a2a_listitem_custom"><a role="link" title="bluesky" class="a2a_button_bluesky" target="_blank" rel="nofollow noopener" href="/#bluesky"><i aria-hidden="true" class="at-icon-wrapper icon-bluesky-logo"></i><span>bluesky</span></a></li><div style="clear: both;"></div></ul><div class="extra-links-drop-zone">
    

        


    

            

            

            <div data-widget-id="f1a208d3-ac5e-4db2-9c3a-40ef97ddfa10" data-widget-name="UX3HTMLWidget"><a class="axel-share__extra" target="_blank" href="/personalize/addFavoritePublication?doi=10.1016/j.cell.2022.01.021">
	<i class="icon-reading-list" aria-hidden="true"></i>
	<span>Add to my reading list</span>
</a></div>

            

        
</div></div></div></div>

            

        
</div><div><button data-id="article-nav-menubar-more" aria-controls="article_more_menu" aria-expanded="false" aria-haspopup="true" aria-label="Toggle more menu"><i aria-hidden="true" class="icon-more-mobile"></i><span>More</span></button><div class="info-panel"><div class="info-panel__left-content"><div class="info-panel__article_tools info-panel__item"><div id="article_more_menu" aria-label="More menu" data-core-nav="more"><div class="heading">More<button title="Close more menu" aria-controls="article_more_menu" aria-haspopup="true"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div>
    

        


    

            

            

            <ul data-widget-id="183a17a2-87b5-4e6f-bb85-6b5cda068c89" data-widget-name="UX3ArticleToolsWidget" class="article-tools"><li class="article-tools__item article-tools__pdf"><a href="/action/showPdf?pii=S0092-8674%2822%2900079-4" title="Standard PDF (opens in a new tab)" target="_blank" aria-hidden="false" class="article-tools__ctrl article-tools__item__pdf"><svg focusable="false" viewBox="0 0 32 32" width="20" height="20" title="PDF" class="pdf-icon"><path id="pdfwhite" d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path id="pdfda0000" d="M.167 2.592H22.39V9.72H.166z" fill="#da0000"></path><path id="pdffff9f9" fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path id="pdff91d0a" d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="basic">Download PDF</span><span class="alternative">Download PDF</span></a></li><li><a href="https://www.cell.com/action/showCitFormats?doi=10.1016%2Fj.cell.2022.01.021&amp;pii=S0092-8674%2822%2900079-4" title="Cite" target="_blank" aria-label="Open Cite in new tab" class="article-tools__item__displayExportCitation"><i aria-hidden="true" class="icon-export-Citation"></i><span>Cite</span></a></li><li id="article_tools_share">
    

        


    

            

            

            <!-- Go to https://www.addtoany.com/buttons/customize/ to customize your tools --><script type="text/javascript" defer="defer" src="https://static.addtoany.com/menu/page.js"></script><script type="text/javascript">var a2a_config = a2a_config || {};
a2a_config.thanks = {
    postShare: false,
    ad: false,
};</script><script>var a2a_config = a2a_config || {};
let res = window.location.href.replace(/\(/g, "%28").replace(/\)/g, "%29");
a2a_config.templates = a2a_config.templates || {};

a2a_config.templates.email = {
    body: res
};
</script><div data-widget-id="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da" data-widget-name="UX3share" class="dropBlock article-tools__dropblock article-tools__share"><button data-db-target-for="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da" aria-expanded="false" title="open Share menu" class="article-tools__ctrl share__ctrl"><i aria-hidden="true" class="icon-article-share"></i><span>Share</span></button><div data-db-target-of="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da" id="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da_Pop" class="share__block axel-share dropBlock__holder article-tools__dropblock__holder"><h4 class="axel-share__title">Share</h4><div class="axel-share__inner"><span class="sr-only">Share on</span><ul class="a2a__share-list rlist w-slide--list a2a a2a_kit a2a_kit_size_32 a2a_default_style" data-a2a-url="https://www.cell.com/cell/fulltext/S0092-8674%2822%2900079-4" style="line-height: 32px;"><li class="a2a_listitem_custom"><a role="link" title="Email" class="a2a_button_email" target="_blank" rel="nofollow noopener" href="/#email"><i aria-hidden="true" class="at-icon-wrapper icon-gizmo-email-plus"></i><span>Email</span></a></li><li class="a2a_listitem_custom"><a role="link" title="formerly known as Twitter" class="a2a_button_twitter" target="_blank" rel="nofollow noopener" href="/#twitter"><i aria-hidden="true" class="at-icon-wrapper icon-x-default"></i><span>X</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Facebook" class="a2a_button_facebook" target="_blank" rel="nofollow noopener" href="/#facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook-square-2"></i><span>Facebook</span></a></li><li class="a2a_listitem_custom"><a role="link" title="LinkedIn" class="a2a_button_linkedin" target="_blank" rel="nofollow noopener" href="/#linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin-square"></i><span>LinkedIn</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Sina Weibo" class="a2a_button_sina_weibo" target="_blank" rel="nofollow noopener" href="/#sina_weibo"><i aria-hidden="true" class="at-icon-wrapper icon-weibo"></i><span>Sina Weibo</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Mendeley" class="a2a_button_mendeley" target="_blank" rel="nofollow noopener" href="/#mendeley"><i aria-hidden="true" class="at-icon-wrapper icon-mendeley-2"></i><span>Mendeley</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Bluesky" class="a2a_button_bluesky" target="_blank" rel="nofollow noopener" href="/#bluesky"><i aria-hidden="true" class="at-icon-wrapper icon-bluesky-logo"></i><span>Bluesky</span></a></li><div style="clear: both;"></div></ul><div class="extra-links-drop-zone">
    

        


    

            

            

            <div data-widget-id="ae9bf5c5-a2ee-4f5c-8e41-9b21638b1a9f" data-widget-name="UX3HTMLWidget"><a class="axel-share__extra" target="_blank" href="/personalize/addFavoritePublication?doi=10.1016/j.cell.2022.01.021">
	<i class="icon-reading-list" aria-hidden="true"></i>
	<span>Add to my reading list</span>
</a></div>

            

        
</div></div></div></div>

            

        
</li><li><a href="https://www.cell.com/action/addCitationAlert?doi=10.1016%2Fj.cell.2022.01.021" title="Set Alert" target="_blank" aria-label="Open Set Alert in new tab" class="article-tools__item__displayCreateCitationAlert"><i aria-hidden="true" class="icon-set-alert"></i><span>Set Alert</span></a></li><li><a href="https://s100.copyright.com/AppDispatchServlet?publisherName=ELS&amp;contentID=S0092867422000794&amp;orderBeanReset=true&amp;orderSource=Phoenix" title="Get Rights" target="_blank" aria-label="Open Get Rights in new tab" class="article-tools__item__request"><i aria-hidden="true" class="icon-get-rights"></i><span>Get Rights</span></a></li><li><a href="https://www.contentednet.com/salesforce-form/?pii=S0092867422000794&amp;issn=00928674&amp;source=jbs" title="Reprints" target="_blank" aria-label="Open Reprints in new tab" class="article-tools__item__reprints"><i aria-hidden="true" class="icon-article-reprints"></i><span>Reprints</span></a></li></ul>

            

        
</div></div><div class="info-panel__download-issue info-panel__item"><div class="article-header__download-full-issue"><button title="Download full issue" type="submit" formaction="/action/downloadMultiplePdf?action=downloadIssuePdf&amp;journal=cell&amp;volume=185&amp;issue=5&amp;pii=S0092-8674%2821%29X0006-2" class="btn--download-issue btn--cancel download-full-issue"><i class="icon-download-new"></i><span>Download Full Issue</span></button><div class="table-of-content__alert table-of-content__alert--download"><p>Download started</p><a href="#" class="table-of-content__alert--close">Ok</a></div></div></div></div><div class="info-panel__right-content"><div class="info-panel__navigation info-panel__item"><div class="content-navigation"><a href="/cell/fulltext/S0092-8674(22)00123-4" aria-label="Previous article" aria-disabled="false" class="content-navigation__prev"><i aria-hidden="true" class="icon-arrow_l"></i><span>Previous article</span></a><a href="/cell/fulltext/S0092-8674(22)00068-X" aria-label="Next article" aria-disabled="false" class="content-navigation__next"><span>Next article</span><i aria-hidden="true" class="icon-arrow_r"></i></a></div></div></div></div></div></div></div><div class="core-sections-menu-outer"><div class="core-sections-menu"><button aria-controls="article_sections_menu-desktop" aria-haspopup="true" aria-expanded="false" aria-label="Toggle sections menu" data-id="article-toolbar-showhide"><i class="icon-show-outline" aria-hidden="true"></i><span>Show Outline</span></button><nav id="article_sections_menu-desktop" aria-label="Contents" data-core-nav="article" style="margin-top: 0px;"><button aria-controls="article_sections_menu-desktop" aria-haspopup="true" aria-expanded="false" aria-label="Toggle sections menu" data-id="article-toolbar-showhide"><i class="icon-hide-outline" aria-hidden="true"></i><span>Hide Outline</span></button><ul><li><a href="#author-highlights-abstract">Highlights</a></li><li><a href="#author-abstract">Summary</a></li><li><a href="#graphical-abstract">Graphical abstract</a></li><li><a href="#keywords">Keywords</a></li><li><a href="#sec-1">Introduction</a></li><li><a href="#sec-2">Results</a></li><li><a href="#sec-3">Discussion</a></li><li><a href="#sec-4">STAR★Methods</a></li><li><a href="#acknowledgments">Acknowledgments</a></li><li><a href="#supplementary-material">Supplemental information (1)</a></li><li><a href="#data-availability">Data and code availability</a></li><li><a href="#references">References</a></li><li role="none" class="mobile-only"><a role="menuitem" href="#core-collateral-metrics">Article metrics</a></li><li role="none" class="mobile-only"><a role="menuitem" href="#core-collateral-relatedArticles">Related Articles</a></li></ul></nav></div></div><div><div class="core-content-inner"><div class="core-linked-content"></div><div id="abstracts" data-extent="frontmatter"><div class="core-container"><section id="author-highlights-abstract" property="abstract" typeof="Text" role="doc-abstract"><h2 property="name">Highlights</h2><div id="abspara0020" role="paragraph"><div id="ulist0010" role="list"><div id="u0010" role="listitem"><div class="label">•</div><div class="content"><div id="p0010" role="paragraph">GATA4:TBX5 interactome in CPs is enriched in <i>de novo</i> variants associated with CHD</div></div></div><div id="u0015" role="listitem"><div class="label">•</div><div class="content"><div id="p0015" role="paragraph">A method for scoring interactome variants identified <i>GLYR1</i> as a candidate CHD gene</div></div></div><div id="u0020" role="listitem"><div class="label">•</div><div class="content"><div id="p0020" role="paragraph">GLYR1 and GATA4 widely co-occupied and co-activated cardiac developmental genes</div></div></div><div id="u0025" role="listitem"><div class="label">•</div><div class="content"><div id="p0025" role="paragraph">The <i>GLYR1</i> CHD variant disrupted interaction with GATA4 and impaired cardiogenesis</div></div></div></div></div></section><section id="author-abstract" property="abstract" typeof="Text" role="doc-abstract"><h2 property="name">Summary</h2><div id="abspara0010" role="paragraph">Congenital heart disease (CHD) is present in 1% of live births, yet identification of causal mutations remains challenging. We hypothesized that genetic determinants for CHDs may lie in the protein interactomes of transcription factors whose mutations cause CHDs. Defining the interactomes of two transcription factors haplo-insufficient in CHD, GATA4 and TBX5, within human cardiac progenitors, and integrating the results with nearly 9,000 exomes from proband-parent trios revealed an enrichment of <i>de novo</i> missense variants associated with CHD within the interactomes. Scoring variants of interactome members based on residue, gene, and proband features identified likely CHD-causing genes, including the epigenetic reader <i>GLYR1</i>. GLYR1 and GATA4 widely co-occupied and co-activated cardiac developmental genes, and the identified <i>GLYR1</i> missense variant disrupted interaction with GATA4, impairing <i>in&nbsp;vitro</i> and <i>in&nbsp;vivo</i> function in mice. This integrative proteomic and genetic approach provides a framework for prioritizing and interrogating genetic variants in heart disease.</div></section><section id="graphical-abstract" property="abstract" typeof="Text" role="doc-abstract"><h2 property="name">Graphical abstract</h2><div id="abspara0015" role="paragraph"><div class="figure-wrap"><figure id="undfig1" class="graphic" data-figureviewer-ignore="true"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2022.01.021/asset/dd34289d-4540-40b8-8175-4b62530f3457/main.assets/fx1_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="fx1" alt="Graphical abstract undfig1" src="/cms/10.1016/j.cell.2022.01.021/asset/25d819ef-2ea7-4696-9e40-f5f967339f81/main.assets/fx1.jpg" height="375" width="375" data-viewer-src="/cms/10.1016/j.cell.2022.01.021/asset/dd34289d-4540-40b8-8175-4b62530f3457/main.assets/fx1_lrg.jpg" loading="lazy"></a></figure></div></div></section></div></div><div class="core-linked-content"></div><section id="keywords" property="keywords"><h2>Keywords</h2><ol><li><a href="/action/doSearch?AllField=&quot;genetics&quot;" alt="Search on this keyword" rel="nofollow">genetics</a></li><li><a href="/action/doSearch?AllField=&quot;protein+interactome+networks&quot;" alt="Search on this keyword" rel="nofollow">protein interactome networks</a></li><li><a href="/action/doSearch?AllField=&quot;congenital+heart+disease&quot;" alt="Search on this keyword" rel="nofollow">congenital heart disease</a></li><li><a href="/action/doSearch?AllField=&quot;disease+variants&quot;" alt="Search on this keyword" rel="nofollow">disease variants</a></li><li><a href="/action/doSearch?AllField=&quot;de+novo+variants&quot;" alt="Search on this keyword" rel="nofollow"><i>de novo</i> variants</a></li><li><a href="/action/doSearch?AllField=&quot;GATA4&quot;" alt="Search on this keyword" rel="nofollow">GATA4</a></li><li><a href="/action/doSearch?AllField=&quot;TBX5&quot;" alt="Search on this keyword" rel="nofollow">TBX5</a></li><li><a href="/action/doSearch?AllField=&quot;GLYR1&quot;" alt="Search on this keyword" rel="nofollow">GLYR1</a></li><li><a href="/action/doSearch?AllField=&quot;NPAC&quot;" alt="Search on this keyword" rel="nofollow">NPAC</a></li></ol></section><div class="core-linked-content"></div><div class="core-linked-content"></div><section id="bodymatter" data-extent="bodymatter" property="articleBody" typeof="Text"><div class="core-container"><section id="sec-1"><h2>Introduction</h2><div id="p0035" role="paragraph">Birth defects are complex developmental phenotypes affecting ∼6% of births worldwide, yet their genetic roots are multifarious and difficult to ascertain (<span class="dropBlock reference-citations"><a id="body-ref-sref24" href="#" role="doc-biblioref" data-xml-rid="bib24" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref24" aria-controls="sref24">Christianson and Howson, 2006</a><div class="dropBlock__holder" data-db-target-of="sref24"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">23.</div><div>Christianson, A. ∙ Howson, C.P.</div><div><strong>March of dimes</strong></div><div>Global Report on Birth., 2006</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.ChristiansonC.P.HowsonMarch+of+dimes2006Global+Report+on+Birth." target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref29" href="#" role="doc-biblioref" data-xml-rid="bib29" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref29" aria-controls="sref29">Deciphering Developmental Disorders Study, 2015</a><div class="dropBlock__holder" data-db-target-of="sref29"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">28.</div><div>Deciphering Developmental Disorders Study</div><div><strong>Large-scale discovery of novel genetic causes of developmental disorders</strong></div><div><em>Nature.</em> 2015; <strong>519</strong>:223-228</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature14135" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-84924666082" target="_blank" aria-label="Open Scopus in new tab">Scopus (685)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25533962/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature14135&amp;pmid=25533962" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Particularly challenging are rare disorders and more common complex defects with high allelic and locus heterogeneity. In recent years, whole-exome sequencing has accelerated our understanding of such disorders, including the most common birth defect, congenital heart disease (CHD) (<span class="dropBlock reference-citations"><a id="body-ref-sref145-1" href="#" role="doc-biblioref" data-xml-rid="bib145" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref145-1" aria-controls="sref145-1">Zaidi et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref145-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">131.</div><div>Zaidi, S. ∙ Choi, M. ∙ Wakimoto, H. ...</div><div><strong>De novo mutations in histone-modifying genes in congenital heart disease</strong></div><div><em>Nature.</em> 2013; <strong>498</strong>:220-223</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature12141" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_131_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-84879001958" target="_blank" aria-label="Open Scopus in new tab">Scopus (616)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23665959/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature12141&amp;pmid=23665959" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref56-1" href="#" role="doc-biblioref" data-xml-rid="bib56" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref56-1" aria-controls="sref56-1">Homsy et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref56-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">51.</div><div>Homsy, J. ∙ Zaidi, S. ∙ Shen, Y. ...</div><div><strong>De novo mutations in congenital heart disease with neurodevelopmental and other congenital anomalies</strong></div><div><em>Science.</em> 2015; <strong>350</strong>:1262-1266</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aac9396" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_51_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-84949008129" target="_blank" aria-label="Open Scopus in new tab">Scopus (412)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26785492/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aac9396&amp;pmid=26785492" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref63-1" href="#" role="doc-biblioref" data-xml-rid="bib63" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref63-1" aria-controls="sref63-1">Jin et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref63-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">56.</div><div>Jin, S.C. ∙ Homsy, J. ∙ Zaidi, S. ...</div><div><strong>Contribution of rare inherited and de novo variants in 2,871 congenital heart disease probands</strong></div><div><em>Nat. Genet.</em> 2017; <strong>49</strong>:1593-1601</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng.3970" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_56_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-85032451853" target="_blank" aria-label="Open Scopus in new tab">Scopus (358)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28991257/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng.3970&amp;pmid=28991257" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref114" href="#" role="doc-biblioref" data-xml-rid="bib114" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref114" aria-controls="sref114">Richter et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref114"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">102.</div><div>Richter, F. ∙ Morton, S.U. ∙ Kim, S.W. ...</div><div><strong>Genomic analyses implicate noncoding de novo variants in congenital heart disease</strong></div><div><em>Nat. Genet.</em> 2020; <strong>52</strong>:769-777</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41588-020-0652-z" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_102_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-85087032848" target="_blank" aria-label="Open Scopus in new tab">Scopus (46)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32601476/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41588-020-0652-z&amp;pmid=32601476" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). <i>De novo</i> monogenic aberrations were found to collectively contribute to ∼10% of CHD cases, whereas rare inherited and copy number variants have been identified in ∼1% and 25% of cases, respectively (<span class="dropBlock reference-citations"><a id="body-ref-sref144-1" href="#" role="doc-biblioref" data-xml-rid="bib144" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref144-1" aria-controls="sref144-1">Zaidi and Brueckner, 2017</a><div class="dropBlock__holder" data-db-target-of="sref144-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">130.</div><div>Zaidi, S. ∙ Brueckner, M.</div><div><strong>Genetics and genomics of congenital heart disease</strong></div><div><em>Circ. Res.</em> 2017; <strong>120</strong>:923-940</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1161/CIRCRESAHA.116.309140" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_130_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-85015361244" target="_blank" aria-label="Open Scopus in new tab">Scopus (212)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28302740/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1161%2FCIRCRESAHA.116.309140&amp;pmid=28302740" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Additionally, polygenic and oligogenic inheritance models, where multiple genetic variants with epistatic relationships are implicated, have been proposed as mechanistic explanations for certain complex phenotypes. A recent study from our group highlighted the involvement of genetic modifiers in human cardiac disease (<span class="dropBlock reference-citations"><a id="body-ref-sref45-1" href="#" role="doc-biblioref" data-xml-rid="bib45" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref45-1" aria-controls="sref45-1">Gifford et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref45-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">41.</div><div>Gifford, C.A. ∙ Ranade, S.S. ∙ Samarakoon, R. ...</div><div><strong>Oligogenic inheritance of a human heart disease involving a genetic modifier</strong></div><div><em>Science.</em> 2019; <strong>364</strong>:865-870</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aat5056" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-85066940786" target="_blank" aria-label="Open Scopus in new tab">Scopus (84)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31147515/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aat5056&amp;pmid=31147515" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), but the net contribution of oligogenic inheritance remains to be determined. Despite the growing catalog of human genome variants, the cause of over 50% of CHD cases remains unknown (<span class="dropBlock reference-citations"><a id="body-ref-sref144-2" href="#" role="doc-biblioref" data-xml-rid="bib144" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref144-2" aria-controls="sref144-2">Zaidi and Brueckner, 2017</a><div class="dropBlock__holder" data-db-target-of="sref144-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">130.</div><div>Zaidi, S. ∙ Brueckner, M.</div><div><strong>Genetics and genomics of congenital heart disease</strong></div><div><em>Circ. Res.</em> 2017; <strong>120</strong>:923-940</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1161/CIRCRESAHA.116.309140" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_130_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-85015361244" target="_blank" aria-label="Open Scopus in new tab">Scopus (212)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28302740/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1161%2FCIRCRESAHA.116.309140&amp;pmid=28302740" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="p0040" role="paragraph">A barrier to a complete understanding of CHD’s etiology is its immense genetic heterogeneity. Estimates based on <i>de novo</i> mutations alone indicate that more than 390 genes may contribute to CHD pathogenesis (<span class="dropBlock reference-citations"><a id="body-ref-sref56-2" href="#" role="doc-biblioref" data-xml-rid="bib56" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref56-2" aria-controls="sref56-2">Homsy et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref56-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">51.</div><div>Homsy, J. ∙ Zaidi, S. ∙ Shen, Y. ...</div><div><strong>De novo mutations in congenital heart disease with neurodevelopmental and other congenital anomalies</strong></div><div><em>Science.</em> 2015; <strong>350</strong>:1262-1266</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aac9396" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_51_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-84949008129" target="_blank" aria-label="Open Scopus in new tab">Scopus (412)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26785492/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aac9396&amp;pmid=26785492" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). This heterogeneity reduces the statistical power of CHD risk gene analysis with the cohorts currently available. It is estimated that cohorts of approximately 10,000 parent-proband trios would be needed for whole-exome sequencing to detect ∼80% of genes contributing to haplo-insufficient syndromic CHD (<span class="dropBlock reference-citations"><a id="body-ref-sref123" href="#" role="doc-biblioref" data-xml-rid="bib123" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref123" aria-controls="sref123">Sifrim et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref123"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">111.</div><div>Sifrim, A. ∙ Hitz, M.-P. ∙ Wilsdon, A. ..., INTERVAL Study, UK10K Consortium, Deciphering Developmental Disorders Study</div><div><strong>Distinct genetic architectures for syndromic and nonsyndromic congenital heart defects identified by exome sequencing</strong></div><div><em>Nat. Genet.</em> 2016; <strong>48</strong>:1060-1065</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng.3627" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_111_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-84980370998" target="_blank" aria-label="Open Scopus in new tab">Scopus (215)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27479907/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng.3627&amp;pmid=27479907" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), highlighting the need for alternative strategies to identify CHD risk genes and to prioritize for potentially causative variants.</div><div id="p0045" role="paragraph">Many diseases display tissue-restricted phenotypes but are rarely explained by mutations in genes with tissue-specific expression (<span class="dropBlock reference-citations"><a id="body-ref-sref54" href="#" role="doc-biblioref" data-xml-rid="bib54" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref54" aria-controls="sref54">Hekselman and Yeger-Lotem, 2020</a><div class="dropBlock__holder" data-db-target-of="sref54"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">49.</div><div>Hekselman, I. ∙ Yeger-Lotem, E.</div><div><strong>Mechanisms of tissue and cell-type specificity in heritable traits and diseases</strong></div><div><em>Nat. Rev. Genet.</em> 2020; <strong>21</strong>:137-150</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41576-019-0200-9" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-85077632359" target="_blank" aria-label="Open Scopus in new tab">Scopus (43)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31913361/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41576-019-0200-9&amp;pmid=31913361" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). For example, cardiac malformations have been linked to variants in tissue-enriched cardiac transcription factors (cTFs) that are expressed more widely. Such cTFs typically form complexes with other tissue-enriched and ubiquitous proteins to orchestrate specific developmental gene programs (<span class="dropBlock reference-citations"><a id="body-ref-sref74" href="#" role="doc-biblioref" data-xml-rid="bib74" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref74" aria-controls="sref74">Lambert et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref74"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">66.</div><div>Lambert, S.A. ∙ Jolma, A. ∙ Campitelli, L.F. ...</div><div><strong>The human transcription factors</strong></div><div><em>Cell.</em> 2018; <strong>172</strong>:650-665</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=10.1016%2Fj.cell.2018.01.029&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=10.1016%2Fj.cell.2018.01.029&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-85041527619" target="_blank" aria-label="Open Scopus in new tab">Scopus (848)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29425488/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2018.01.029&amp;pmid=29425488" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). cTF missense variants may disrupt specific interactions with other proteins, affecting their transcriptional cooperativity and causing disease (<span class="dropBlock reference-citations"><a id="body-ref-sref4-1" href="#" role="doc-biblioref" data-xml-rid="bib4" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref4-1" aria-controls="sref4-1">Ang et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref4-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">4.</div><div>Ang, Y.-S. ∙ Rivas, R.N. ∙ Ribeiro, A.J.S. ...</div><div><strong>Disease model of GATA4 mutation reveals transcription factor cooperativity in human cardiogenesis</strong></div><div><em>Cell.</em> 2016; <strong>167</strong>:1734-1749.e22</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=10.1016%2Fj.cell.2016.11.033&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=10.1016%2Fj.cell.2016.11.033&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-85006302151" target="_blank" aria-label="Open Scopus in new tab">Scopus (127)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27984724/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2016.11.033&amp;pmid=27984724" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref99" href="#" role="doc-biblioref" data-xml-rid="bib99" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref99" aria-controls="sref99">Moskowitz et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref99"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">87.</div><div>Moskowitz, I.P. ∙ Wang, J. ∙ Peterson, M.A. ...</div><div><strong>Transcription factor genes Smad4 and Gata4 cooperatively regulate cardiac valve development. [corrected]</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2011; <strong>108</strong>:4006-4011</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1019025108" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_87_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-79952760100" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21330551/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1019025108&amp;pmid=21330551" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref136-1" href="#" role="doc-biblioref" data-xml-rid="bib136" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref136-1" aria-controls="sref136-1">Waldron et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref136-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">122.</div><div>Waldron, L. ∙ Steimle, J.D. ∙ Greco, T.M. ...</div><div><strong>The cardiac TBX5 interactome reveals a chromatin remodeling network essential for cardiac septation</strong></div><div><em>Dev. Cell.</em> 2016; <strong>36</strong>:262-275</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_122_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=10.1016%2Fj.devcel.2016.01.009&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_122_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=10.1016%2Fj.devcel.2016.01.009&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_122_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-84958661999" target="_blank" aria-label="Open Scopus in new tab">Scopus (56)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26859351/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.devcel.2016.01.009&amp;pmid=26859351" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). This observation suggests a functional relevance for cTF interactors in genetic disorders, including CHD. In agreement, <span class="dropBlock reference-citations"><a id="body-ref-sref7-1" href="#" role="doc-biblioref" data-xml-rid="bib7" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref7-1" aria-controls="sref7-1">Barshir et&nbsp;al. (2014)</a><div class="dropBlock__holder" data-db-target-of="sref7-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">7.</div><div>Barshir, R. ∙ Shwartz, O. ∙ Smoly, I.Y. ...</div><div><strong>Comparative analysis of human tissue interactomes reveals factors leading to tissue-specific manifestation of hereditary diseases</strong></div><div><em>PLoS Comput. Biol.</em> 2014; <strong>10</strong>:e1003632</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pcbi.1003632" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-84903384845" target="_blank" aria-label="Open Scopus in new tab">Scopus (49)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24921629/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pcbi.1003632&amp;pmid=24921629" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span> observed that disease causal genes are often widely expressed across tissues but with a tendency to exhibit more tissue-specific protein-protein interactions in diseased versus unaffected tissues. In CHD specifically, an excess of protein-altering <i>de novo</i> variants from the Pediatric Cardiac Genomic Consortium’s (PCGC’s) cohort were found in ubiquitously expressed chromatin regulators that partner with cTFs to regulate the expression of key developmental genes (<span class="dropBlock reference-citations"><a id="body-ref-sref145-2" href="#" role="doc-biblioref" data-xml-rid="bib145" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref145-2" aria-controls="sref145-2">Zaidi et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref145-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">131.</div><div>Zaidi, S. ∙ Choi, M. ∙ Wakimoto, H. ...</div><div><strong>De novo mutations in histone-modifying genes in congenital heart disease</strong></div><div><em>Nature.</em> 2013; <strong>498</strong>:220-223</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature12141" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_131_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-84879001958" target="_blank" aria-label="Open Scopus in new tab">Scopus (616)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23665959/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature12141&amp;pmid=23665959" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). This led us to hypothesize that protein-protein interactors of cTFs associated with CHD may be enriched in disease-associated proteins, even if these proteins are not tissue specific.</div><div id="p0050" role="paragraph">GATA4 and TBX5 are two essential cTFs (<span class="dropBlock reference-citations"><a id="body-ref-sref73" href="#" role="doc-biblioref" data-xml-rid="bib73" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref73" aria-controls="sref73">Kuo et&nbsp;al., 1997</a><div class="dropBlock__holder" data-db-target-of="sref73"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">65.</div><div>Kuo, C.T. ∙ Morrisey, E.E. ∙ Anandappa, R. ...</div><div><strong>GATA4 transcription factor is required for ventral morphogenesis and heart tube formation</strong></div><div><em>Genes Dev.</em> 1997; <strong>11</strong>:1048-1060</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/gad.11.8.1048" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/9136932/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2Fgad.11.8.1048&amp;pmid=9136932" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref14" href="#" role="doc-biblioref" data-xml-rid="bib14" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref14" aria-controls="sref14">Bruneau et&nbsp;al., 1999</a><div class="dropBlock__holder" data-db-target-of="sref14"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">14.</div><div>Bruneau, B.G. ∙ Logan, M. ∙ Davis, N. ...</div><div><strong>Chamber-specific cardiac expression of Tbx5 and heart defects in Holt-Oram syndrome</strong></div><div><em>Dev. Biol.</em> 1999; <strong>211</strong>:100-108</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1006/dbio.1999.9298" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-0033168070" target="_blank" aria-label="Open Scopus in new tab">Scopus (385)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/10373308/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1006%2Fdbio.1999.9298&amp;pmid=10373308" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>, <span class="dropBlock reference-citations"><a id="body-ref-sref15" href="#" role="doc-biblioref" data-xml-rid="bib15" href-manipulated="true" aria-label="Reference 2001" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref15" aria-controls="sref15">2001</a><div class="dropBlock__holder" data-db-target-of="sref15"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">15.</div><div>Bruneau, B.G. ∙ Nemer, G. ∙ Schmitt, J.P. ...</div><div><strong>A murine model of Holt-Oram syndrome defines roles of the T-box transcription factor Tbx5 in cardiogenesis and disease</strong></div><div><em>Cell.</em> 2001; <strong>106</strong>:709-721</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=10.1016%2FS0092-8674%2801%2900493-7&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=10.1016%2FS0092-8674%2801%2900493-7&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-17944378083" target="_blank" aria-label="Open Scopus in new tab">Scopus (832)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11572777/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0092-8674%2801%2900493-7&amp;pmid=11572777" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref96" href="#" role="doc-biblioref" data-xml-rid="bib96" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref96" aria-controls="sref96">Molkentin et&nbsp;al., 1997</a><div class="dropBlock__holder" data-db-target-of="sref96"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">84.</div><div>Molkentin, J.D. ∙ Lin, Q. ∙ Duncan, S.A. ...</div><div><strong>Requirement of the transcription factor GATA4 for heart tube formation and ventral morphogenesis</strong></div><div><em>Genes Dev.</em> 1997; <strong>11</strong>:1061-1072</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/gad.11.8.1061" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/9136933/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2Fgad.11.8.1061&amp;pmid=9136933" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref98" href="#" role="doc-biblioref" data-xml-rid="bib98" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref98" aria-controls="sref98">Mori et&nbsp;al., 2006</a><div class="dropBlock__holder" data-db-target-of="sref98"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">86.</div><div>Mori, A.D. ∙ Zhu, Y. ∙ Vahora, I. ...</div><div><strong>Tbx5-dependent rheostatic control of cardiac gene expression and morphogenesis</strong></div><div><em>Dev. Biol.</em> 2006; <strong>297</strong>:566-586</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.ydbio.2006.05.023" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_86_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-33748489726" target="_blank" aria-label="Open Scopus in new tab">Scopus (144)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16870172/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.ydbio.2006.05.023&amp;pmid=16870172" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and among the first identified monogenic etiologies of familial CHD. Heterozygous pathogenic variations in <i>TBX5</i> are a cause of septation defects and other forms of CHD in the setting of Holt-Oram syndrome (<span class="dropBlock reference-citations"><a id="body-ref-sref8" href="#" role="doc-biblioref" data-xml-rid="bib8" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref8" aria-controls="sref8">Basson et&nbsp;al., 1997</a><div class="dropBlock__holder" data-db-target-of="sref8"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">8.</div><div>Basson, C.T. ∙ Bachinsky, D.R. ∙ Lin, R.C. ...</div><div><strong>Mutations in human TBX5 [corrected] cause limb and cardiac malformation in Holt-Oram syndrome</strong></div><div><em>Nat. Genet.</em> 1997; <strong>15</strong>:30-35</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng0197-30" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-0030636780" target="_blank" aria-label="Open Scopus in new tab">Scopus (893)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/8988165/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng0197-30&amp;pmid=8988165" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref80" href="#" role="doc-biblioref" data-xml-rid="bib80" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref80" aria-controls="sref80">Li et&nbsp;al., 1997</a><div class="dropBlock__holder" data-db-target-of="sref80"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">69.</div><div>Li, Q.Y. ∙ Newbury-Ecob, R.A. ∙ Terrett, J.A. ...</div><div><strong>Holt-Oram syndrome is caused by mutations in TBX5, a member of the Brachyury (T) gene family</strong></div><div><em>Nat. Genet.</em> 1997; <strong>15</strong>:21-29</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng0197-21" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-1842413728" target="_blank" aria-label="Open Scopus in new tab">Scopus (748)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/8988164/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng0197-21&amp;pmid=8988164" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Heterozygous variations in <i>GATA4</i> also cause atrial and ventricular septal defects, as well as pulmonary stenosis and outflow tract abnormalities (<span class="dropBlock reference-citations"><a id="body-ref-sref44-1" href="#" role="doc-biblioref" data-xml-rid="bib44" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref44-1" aria-controls="sref44-1">Garg et&nbsp;al., 2003</a><div class="dropBlock__holder" data-db-target-of="sref44-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">40.</div><div>Garg, V. ∙ Kathiriya, I.S. ∙ Barnes, R. ...</div><div><strong>GATA4 mutations cause human congenital heart defects and reveal an interaction with TBX5</strong></div><div><em>Nature.</em> 2003; <strong>424</strong>:443-447</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature01827" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_40_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-0043267988" target="_blank" aria-label="Open Scopus in new tab">Scopus (953)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/12845333/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature01827&amp;pmid=12845333" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref110-1" href="#" role="doc-biblioref" data-xml-rid="bib110" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref110-1" aria-controls="sref110-1">Rajagopal et&nbsp;al., 2007</a><div class="dropBlock__holder" data-db-target-of="sref110-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">98.</div><div>Rajagopal, S.K. ∙ Ma, Q. ∙ Obler, D. ...</div><div><strong>Spectrum of heart disease associated with murine and human GATA4 mutation</strong></div><div><em>J.&nbsp;Mol. Cell. Cardiol.</em> 2007; <strong>43</strong>:677-685</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_98_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=10.1016%2Fj.yjmcc.2007.06.004&amp;cf=fulltext&amp;site=yjmcc-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_98_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=10.1016%2Fj.yjmcc.2007.06.004&amp;cf=pdf&amp;site=yjmcc-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_98_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-35048838310" target="_blank" aria-label="Open Scopus in new tab">Scopus (197)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17643447/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.yjmcc.2007.06.004&amp;pmid=17643447" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref133" href="#" role="doc-biblioref" data-xml-rid="bib133" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref133" aria-controls="sref133">Tomita-Mitchell et&nbsp;al., 2007</a><div class="dropBlock__holder" data-db-target-of="sref133"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">120.</div><div>Tomita-Mitchell, A. ∙ Maslen, C.L. ∙ Morris, C.D. ...</div><div><strong>GATA4 sequence variants in patients with congenital heart disease</strong></div><div><em>J.&nbsp;Med. Genet.</em> 2007; <strong>44</strong>:779-783</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1136/jmg.2007.052183" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_120_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-37249090635" target="_blank" aria-label="Open Scopus in new tab">Scopus (165)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/18055909/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1136%2Fjmg.2007.052183&amp;pmid=18055909" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Subsequent studies have demonstrated that TBX5 and GATA4 cooperatively interact on DNA throughout the genome to regulate heart development (<span class="dropBlock reference-citations"><a id="body-ref-sref4-2" href="#" role="doc-biblioref" data-xml-rid="bib4" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref4-2" aria-controls="sref4-2">Ang et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref4-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">4.</div><div>Ang, Y.-S. ∙ Rivas, R.N. ∙ Ribeiro, A.J.S. ...</div><div><strong>Disease model of GATA4 mutation reveals transcription factor cooperativity in human cardiogenesis</strong></div><div><em>Cell.</em> 2016; <strong>167</strong>:1734-1749.e22</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=10.1016%2Fj.cell.2016.11.033&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=10.1016%2Fj.cell.2016.11.033&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-85006302151" target="_blank" aria-label="Open Scopus in new tab">Scopus (127)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27984724/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2016.11.033&amp;pmid=27984724" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref86-1" href="#" role="doc-biblioref" data-xml-rid="bib86" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref86-1" aria-controls="sref86-1">Luna-Zurita et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref86-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">74.</div><div>Luna-Zurita, L. ∙ Stirnimann, C.U. ∙ Glatt, S. ...</div><div><strong>Complex interdependence regulates heterotypic transcription factor distribution and coordinates cardiogenesis</strong></div><div><em>Cell.</em> 2016; <strong>164</strong>:999-1014</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=10.1016%2Fj.cell.2016.01.004&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=10.1016%2Fj.cell.2016.01.004&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-84959387425" target="_blank" aria-label="Open Scopus in new tab">Scopus (122)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26875865/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2016.01.004&amp;pmid=26875865" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Disruption of the physical interaction between these cTFs or with other specific co-factors by missense variants can impair transcriptional cooperativity and lineage specification, and ultimately cause cardiac malformations (<span class="dropBlock reference-citations"><a id="body-ref-sref4-3" href="#" role="doc-biblioref" data-xml-rid="bib4" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref4-3" aria-controls="sref4-3">Ang et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref4-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">4.</div><div>Ang, Y.-S. ∙ Rivas, R.N. ∙ Ribeiro, A.J.S. ...</div><div><strong>Disease model of GATA4 mutation reveals transcription factor cooperativity in human cardiogenesis</strong></div><div><em>Cell.</em> 2016; <strong>167</strong>:1734-1749.e22</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=10.1016%2Fj.cell.2016.11.033&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=10.1016%2Fj.cell.2016.11.033&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-85006302151" target="_blank" aria-label="Open Scopus in new tab">Scopus (127)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27984724/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2016.11.033&amp;pmid=27984724" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref44-2" href="#" role="doc-biblioref" data-xml-rid="bib44" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref44-2" aria-controls="sref44-2">Garg et&nbsp;al., 2003</a><div class="dropBlock__holder" data-db-target-of="sref44-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">40.</div><div>Garg, V. ∙ Kathiriya, I.S. ∙ Barnes, R. ...</div><div><strong>GATA4 mutations cause human congenital heart defects and reveal an interaction with TBX5</strong></div><div><em>Nature.</em> 2003; <strong>424</strong>:443-447</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature01827" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_40_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-0043267988" target="_blank" aria-label="Open Scopus in new tab">Scopus (953)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/12845333/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature01827&amp;pmid=12845333" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref91" href="#" role="doc-biblioref" data-xml-rid="bib91" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref91" aria-controls="sref91">Maitra et&nbsp;al., 2009</a><div class="dropBlock__holder" data-db-target-of="sref91"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">79.</div><div>Maitra, M. ∙ Schluterman, M.K. ∙ Nichols, H.A. ...</div><div><strong>Interaction of Gata4 and Gata6 with Tbx5 is critical for normal cardiac development</strong></div><div><em>Dev. Biol.</em> 2009; <strong>326</strong>:368-377</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.ydbio.2008.11.004" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_79_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-58649091740" target="_blank" aria-label="Open Scopus in new tab">Scopus (149)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19084512/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.ydbio.2008.11.004&amp;pmid=19084512" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref136-2" href="#" role="doc-biblioref" data-xml-rid="bib136" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref136-2" aria-controls="sref136-2">Waldron et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref136-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">122.</div><div>Waldron, L. ∙ Steimle, J.D. ∙ Greco, T.M. ...</div><div><strong>The cardiac TBX5 interactome reveals a chromatin remodeling network essential for cardiac septation</strong></div><div><em>Dev. Cell.</em> 2016; <strong>36</strong>:262-275</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_122_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=10.1016%2Fj.devcel.2016.01.009&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_122_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=10.1016%2Fj.devcel.2016.01.009&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_122_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-84958661999" target="_blank" aria-label="Open Scopus in new tab">Scopus (56)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26859351/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.devcel.2016.01.009&amp;pmid=26859351" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Therefore, the identification of human GATA4 and TBX5 (GT) protein interactors during cardiogenesis could highlight disease mechanisms and aid in predicting the impact of protein-coding variants in CHD.</div><div id="p0055" role="paragraph">Here, we leveraged an integrated proteomics and human genetics approach that dissects the protein-protein interactors of endogenous GATA4 and TBX5 in human cardiac progenitor cells to identify and prioritize potential disease genes harboring CHD-associated variants, revealing aspects of cardiac gene regulation. This approach can be leveraged to study the genetic underpinnings of many human diseases.</div></section><section id="sec-2"><h2>Results</h2><section id="sec-2-1"><h3>Identification of the GATA4 and TBX5 protein interactomes in cardiac progenitors</h3><div id="p0060" role="paragraph">To identify the GATA4 and TBX5 protein interactome (GT-PPI) in human induced pluripotent stem-cell-derived cardiac progenitors (CPs), we used antibodies against each endogenous cTF for affinity purification and mass spectrometry (AP-MS) (<a id="crosref0505" href="#fig1">Figure&nbsp;1</a>A). Using CRISPR-Cas9-gRNA ribonucleoproteins, we generated clonal <i>TBX5</i> or <i>GATA4</i> homozygous knockout (KO) hiPSC lines as negative controls. These control lines were differentiated to CP and cardiomyocyte (CM) stages, and the absence of the respective cTF expression was confirmed (<a id="crosref0510" href="#figs1">Figure&nbsp;S1</a>A–S1E). Consistent with previous reports (<span class="dropBlock reference-citations"><a id="body-ref-sref67" href="#" role="doc-biblioref" data-xml-rid="bib67" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref67" aria-controls="sref67">Kathiriya et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref67"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">60.</div><div>Kathiriya, I.S. ∙ Rao, K.S. ∙ Iacono, G. ...</div><div><strong>Modeling human TBX5 haploinsufficiency predicts regulatory networks for congenital heart disease</strong></div><div><em>Dev. Cell.</em> 2021; <strong>56</strong>:292-309.e9</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=10.1016%2Fj.devcel.2020.11.020&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=10.1016%2Fj.devcel.2020.11.020&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-85099131520" target="_blank" aria-label="Open Scopus in new tab">Scopus (20)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33321106/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.devcel.2020.11.020&amp;pmid=33321106" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref86-2" href="#" role="doc-biblioref" data-xml-rid="bib86" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref86-2" aria-controls="sref86-2">Luna-Zurita et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref86-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">74.</div><div>Luna-Zurita, L. ∙ Stirnimann, C.U. ∙ Glatt, S. ...</div><div><strong>Complex interdependence regulates heterotypic transcription factor distribution and coordinates cardiogenesis</strong></div><div><em>Cell.</em> 2016; <strong>164</strong>:999-1014</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=10.1016%2Fj.cell.2016.01.004&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=10.1016%2Fj.cell.2016.01.004&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-84959387425" target="_blank" aria-label="Open Scopus in new tab">Scopus (122)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26875865/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2016.01.004&amp;pmid=26875865" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref101" href="#" role="doc-biblioref" data-xml-rid="bib101" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref101" aria-controls="sref101">Narita et&nbsp;al., 1997</a><div class="dropBlock__holder" data-db-target-of="sref101"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">89.</div><div>Narita, N. ∙ Bielinska, M. ∙ Wilson, D.B.</div><div><strong>Cardiomyocyte differentiation by GATA-4-deficient embryonic stem cells</strong></div><div><em>Development.</em> 1997; <strong>124</strong>:3755-3764</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1242/dev.124.19.3755" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/9367431/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1242%2Fdev.124.19.3755&amp;pmid=9367431" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), <i>GATA4</i> and <i>TBX5</i> KO cells were able to differentiate into CMs, albeit with delayed beating and reduced differentiation efficiency (<a id="crosref0530" href="#figs1">Figures S1</a>E–S1G; <a id="crosref0535" href="#mmc1">Tables S1</a>A and S1B).</div><div class="figure-wrap"><figure id="fig1" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2022.01.021/asset/79910616-b6cf-4d6e-b3f0-222d933b217c/main.assets/gr1_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr1" src="/cms/10.1016/j.cell.2022.01.021/asset/681c6e80-acbe-4d42-ae97-8376ffc8cdf3/main.assets/gr1.jpg" height="858" width="659" aria-labelledby="fig1-title fig1-content" data-viewer-src="/cms/10.1016/j.cell.2022.01.021/asset/79910616-b6cf-4d6e-b3f0-222d933b217c/main.assets/gr1_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig1-title" class="figure__title"><span class="label figure__label">Figure&nbsp;1</span> <span class="figure__title__text">Generation of GATA4 and TBX5 protein interactomes in human iPSC-derived cardiac progenitors</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig1-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0015" role="paragraph">(A) GATA4 and TBX5 AP-MS strategy from hiPSC-derived cardiac progenitors with gene knockout lines as negative controls.</div><div id="fspara0020" role="paragraph">(B and C) GATA4 (B) and TBX5 (C) interacting protein categories with boxed areas proportional to the number of interactors in each. Proteins interacting with both GATA4 and TBX5 (blue) or previously reported interactors (red) are highlighted.</div><div id="fspara0025" role="paragraph">(D) Distribution of GATA4 and TBX5 PPIs in biological processes, as annotated in (B and C).</div><div id="fspara0030" role="paragraph">(E) Tissue expression distribution of GATA4 and TBX5 interactors across the six Human Protein Atlas categories based on transcript detection (NX<sup>3</sup>1) in all 37 analyzed tissues (see <a id="crosref2255" href="#sec-4">STAR Methods</a>).</div><div id="fspara0035" role="paragraph">See also <a id="crosref2260" href="#figs1">Figures S1</a> and <a id="crosref2265" href="#figs2">S2</a> and <a id="crosref2270" href="#mmc1">Tables S1</a>A–S1D.</div></div></div></div></figcaption></figure></div><div class="figure-wrap"><figure id="figs1" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2022.01.021/asset/9ba95259-d77d-4f24-af9d-ae63bc64de7d/main.assets/figs1_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs1" src="/cms/10.1016/j.cell.2022.01.021/asset/4fe8939e-df96-46ac-b4ed-d03ed368cb07/main.assets/figs1.jpg" height="750" width="497" aria-labelledby="figs1-title figs1-content" data-viewer-src="/cms/10.1016/j.cell.2022.01.021/asset/9ba95259-d77d-4f24-af9d-ae63bc64de7d/main.assets/figs1_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs1-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S1</span> <span class="figure__title__text">Differentiation of <i>GATA4-KO</i> and <i>TBX5-KO</i> hiPSC clonal lines into cardiomyocytes, related to <a id="crosref2385" href="#fig1">Figure&nbsp;1</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs1-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0265" role="paragraph">(A) Representative immunostaining micrographs for cTNT (green), TBX5 (red) or DAPI (blue) in WT or <i>TBX5-KO</i> hiPSC-derived cardiomyocytes (CMs) at day 15 of differentiation. Scale (100&nbsp;μm).</div><div id="fspara0270" role="paragraph">(B) Immunoprecipitation of TBX5 from enriched nuclear lysates of WT or <i>TBX5-KO</i> hiPSC-derived CPs (differentiation day 6), followed by immunoblotting with anti-TBX5 or anti-vinculin antibodies.</div><div id="fspara0275" role="paragraph">(C) Representative immunostaining micrographs for cTNT (green), GATA4 (red) or DAPI (blue) in WT or <i>GATA4-KO</i> hiPSC-derived CMs at day 15 of differentiation.</div><div id="fspara0280" role="paragraph">(D) Immunoprecipitation of GATA4 from enriched nuclear lysates of WT or <i>GATA4-KO</i> hiPSC-derived CPs (differentiation day 6), followed by immunoblotting with anti-GATA4 or anti-vinculin antibodies</div><div id="fspara0285" role="paragraph">(E) Percentage of cells positive for the indicated proteins at the CP (day 6) and CM (day 15) stages of differentiation as measured by flow cytometry. (n&nbsp;= 10-4)</div><div id="fspara0290" role="paragraph">(F) Beating rates of the WT, <i>TBX5-KO</i> and <i>GATA4-KO</i> CMs as measured by Pulse automated measurement video analysis. (n&nbsp;= 4-6)</div><div id="fspara0295" role="paragraph">(G) Beating onset for WT, <i>TBX5-KO</i> and <i>GATA4-KO</i> CMs. (n&nbsp;= 5)</div><div id="fspara0300" role="paragraph">For E and F One-way ANOVA coupled with Tukey post hoc test: <sup>∗∗∗</sup>&nbsp;= p value&nbsp;&lt; 0.001.</div></div></div></div></figcaption></figure></div><div id="p0065" role="paragraph">GATA4 or TBX5 mass spectrometry data were generated from three replicates of nuclei-enriched day 6 hiPSC-derived CPs from wild-type (WT) or KO samples treated with RNase and DNase to focus on nucleic acid independent interactions (<a id="crosref0540" href="#fig1">Figure&nbsp;1</a>A). An initial list of GT interactors in WT CPs was obtained by scoring the proteins identified in WT AP-MS experiments to their corresponding KO control line using the established protein-protein interaction algorithm SAINTq (<span class="dropBlock reference-citations"><a id="body-ref-sref128-1" href="#" role="doc-biblioref" data-xml-rid="bib128" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref128-1" aria-controls="sref128-1">Teo et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref128-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">115.</div><div>Teo, G. ∙ Koh, H. ∙ Fermin, D. ...</div><div><strong>SAINTq: Scoring protein-protein interactions in affinity purification - mass spectrometry experiments with fragment or peptide intensity data</strong></div><div><em>Proteomics.</em> 2016; <strong>16</strong>:2238-2245</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/pmic.201500499" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_115_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-84983261035" target="_blank" aria-label="Open Scopus in new tab">Scopus (24)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27119218/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fpmic.201500499&amp;pmid=27119218" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). For further stringency, additional filtering was applied for the high-scoring interactors determined by SAINTq based on nuclear localization and co-expression in the same cells as the bait protein. Proteins whose mRNA was downregulated in the KO cells compared to WT were excluded (see <a id="crosref0550" href="#sec-5">STAR Methods: Selection of interactome proteins</a>). This approach yielded 272 proteins in total, which comprised several of the previously reported GATA4 and TBX5 interactors as well as novel interactors (<span class="dropBlock reference-citations"><a id="body-ref-sref35-1" href="#" role="doc-biblioref" data-xml-rid="bib35" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref35-1" aria-controls="sref35-1">Enane et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref35-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">32.</div><div>Enane, F.O. ∙ Shuen, W.H. ∙ Gu, X. ...</div><div><strong>GATA4 loss of function in liver cancer impedes precursor to hepatocyte transition</strong></div><div><em>J.&nbsp;Clin. Invest.</em> 2017; <strong>127</strong>:3527-3542</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1172/JCI93488" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-85028908653" target="_blank" aria-label="Open Scopus in new tab">Scopus (24)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28758902/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1172%2FJCI93488&amp;pmid=28758902" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref103-1" href="#" role="doc-biblioref" data-xml-rid="bib103" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref103-1" aria-controls="sref103-1">Padmanabhan et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref103-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">91.</div><div>Padmanabhan, A. ∙ Alexanian, M. ∙ Linares-Saldana, R. ...</div><div><strong>BRD4 (Bromodomain-Containing Protein 4) Interacts with GATA4 (GATA Binding Protein 4) to Govern Mitochondrial Homeostasis in Adult Cardiomyocytes</strong></div><div><em>Circulation.</em> 2020; <strong>142</strong>:2338-2355</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1161/CIRCULATIONAHA.120.047753" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-85097828347" target="_blank" aria-label="Open Scopus in new tab">Scopus (14)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33094644/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1161%2FCIRCULATIONAHA.120.047753&amp;pmid=33094644" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref136-3" href="#" role="doc-biblioref" data-xml-rid="bib136" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref136-3" aria-controls="sref136-3">Waldron et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref136-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">122.</div><div>Waldron, L. ∙ Steimle, J.D. ∙ Greco, T.M. ...</div><div><strong>The cardiac TBX5 interactome reveals a chromatin remodeling network essential for cardiac septation</strong></div><div><em>Dev. Cell.</em> 2016; <strong>36</strong>:262-275</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_122_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=10.1016%2Fj.devcel.2016.01.009&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_122_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=10.1016%2Fj.devcel.2016.01.009&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_122_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-84958661999" target="_blank" aria-label="Open Scopus in new tab">Scopus (56)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26859351/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.devcel.2016.01.009&amp;pmid=26859351" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Mutations in several of these interactors have been previously associated with human or mouse cardiac malformations, highlighting the potential of our approach for disease-gene discovery (<a id="crosref0570" href="#fig1">Figures 1</a>B, 1C, <a id="crosref0575" href="#figs2">S2</a>A, and S2B; <a id="crosref0580" href="#mmc1">Tables S1</a>C and S1D).</div><div id="p0070" role="paragraph">Consistent with the interdependence of GATA4 and TBX5 during cardiac development, their networks showed some overlap, but the bulk of the detected interactors were unique to each cTF (<a id="crosref0585" href="#figs2">Figure&nbsp;S2</a>C). Both networks were enriched in proteins involved in similar biological processes (<a id="crosref0590" href="#fig1">Figures 1</a>B–1D, <a id="crosref0595" href="#figs2">S2</a>A, and S2B). The top two most represented processes were transcription regulation and chromatin modification (<a id="crosref0600" href="#fig1">Figure&nbsp;1</a>D), as expected from the cTFs’ well-established functions in gene regulation. Both known and previously unreported low-abundance TFs were found to interact with GATA4 and/or TBX5 (e.g., ZFPM1, ZNF787, SALL3, ZNF219, and MAB21L2) demonstrating the sensitivity of the AP-MS approach (<a id="crosref0605" href="#fig1">Figures 1</a>B, 1C, <a id="crosref0610" href="#figs2">S2</a>A, and S2B). Chromatin modifiers (∼25% or 15% of GATA4 or TBX5 interactors, respectively) predominantly belonged to ATP-dependent complexes, and we found several histone-modifying enzymes in the GATA4-PPI (<a id="crosref0615" href="#fig1">Figures 1</a>B, 1C, <a id="crosref0620" href="#figs2">S2</a>A, and S2B) (<span class="dropBlock reference-citations"><a id="body-ref-sref35-2" href="#" role="doc-biblioref" data-xml-rid="bib35" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref35-2" aria-controls="sref35-2">Enane et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref35-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">32.</div><div>Enane, F.O. ∙ Shuen, W.H. ∙ Gu, X. ...</div><div><strong>GATA4 loss of function in liver cancer impedes precursor to hepatocyte transition</strong></div><div><em>J.&nbsp;Clin. Invest.</em> 2017; <strong>127</strong>:3527-3542</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1172/JCI93488" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-85028908653" target="_blank" aria-label="Open Scopus in new tab">Scopus (24)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28758902/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1172%2FJCI93488&amp;pmid=28758902" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). A number of RNA processing and splicing proteins, as well as members of the nuclear pore complex, were also identified (<a id="crosref0630" href="#fig1">Figures 1</a>B, 1C, <a id="crosref0635" href="#figs2">S2</a>A, and S2B). The GT-PPIs mostly included proteins expressed ubiquitously, with a small number of tissue-enriched and cell-type enriched interactors (<a id="crosref0640" href="#fig1">Figures 1</a>E and <a id="crosref0645" href="#figs2">S2</a>D).</div><div class="figure-wrap"><figure id="figs2" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2022.01.021/asset/f4724dba-fa6d-403f-9381-95a42dbeac04/main.assets/figs2_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs2" src="/cms/10.1016/j.cell.2022.01.021/asset/24acb0d0-9390-4050-be1b-1de14a473e55/main.assets/figs2.jpg" height="938" width="603" aria-labelledby="figs2-title figs2-content" data-viewer-src="/cms/10.1016/j.cell.2022.01.021/asset/f4724dba-fa6d-403f-9381-95a42dbeac04/main.assets/figs2_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs2-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S2</span> <span class="figure__title__text">Complete GATA4 and TBX5 PPIs in hiPSC-derived cardiac progenitors, related to <a id="crosref2390" href="#fig1">Figure&nbsp;1</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs2-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0310" role="paragraph">(A and B) GATA4-PPI (A) or TBX5-PPI (B). Interactors were manually annotated for biological processes and protein complexes based on literature available. Boxed areas are roughly proportional to the number of interactors they represent. Enriched proteins with a Bayesian false discovery rate (BFDR)&lt;0.001 for GATA4-PPI and BFDR&nbsp;&lt; 0.05 for TBX5-PPI are shown. Proteins interacting with both GATA4 and TBX5, previously reported interactors, and genes involved in mouse/human cardiac development (<span class="dropBlock reference-citations"><a id="body-ref-sref63-2" href="#" role="doc-biblioref" data-xml-rid="bib63" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref63-2" aria-controls="sref63-2">Jin et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref63-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">56.</div><div>Jin, S.C. ∙ Homsy, J. ∙ Zaidi, S. ...</div><div><strong>Contribution of rare inherited and de novo variants in 2,871 congenital heart disease probands</strong></div><div><em>Nat. Genet.</em> 2017; <strong>49</strong>:1593-1601</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng.3970" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_56_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-85032451853" target="_blank" aria-label="Open Scopus in new tab">Scopus (358)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28991257/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng.3970&amp;pmid=28991257" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) are highlighted in blue, red, and underline, respectively. 3-4 replicates from independent differentiations were analyzed per condition.</div><div id="fspara0315" role="paragraph">(C) Venn diagram representing the overlap of GATA4 and TBX5 PPIs generated in CPs.</div><div id="fspara0320" role="paragraph">(D) Interactome gene expression distribution in fetal human heart cell identities from DESCARTES human cell atlas of fetal gene expression (<span class="dropBlock reference-citations"><a id="body-ref-sref18-1" href="#" role="doc-biblioref" data-xml-rid="bib18" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref18-1" aria-controls="sref18-1">Cao et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref18-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">18.</div><div>Cao, J. ∙ O’Day, D.R. ∙ Pliner, H.A. ...</div><div><strong>A human cell atlas of fetal gene expression</strong></div><div><em>Science.</em> 2020; <strong>370</strong></div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aba7721" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-85096082939" target="_blank" aria-label="Open Scopus in new tab">Scopus (135)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aba7721" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div></div></div></div></figcaption></figure></div></section><section id="sec-2-2"><h3>GATA4:TBX5 interactome is enriched in proteins harboring <i>de novo</i> variants in CHD</h3><div id="p0075" role="paragraph">To determine whether the GT interactors identified in human CPs might help predict genetic risk factors for CHD, we assessed their intersection with <i>de novo</i> variants (DNVs) and very rare (minor allele frequency [MAF]&nbsp;&lt; 10<sup>−5</sup>) inherited loss-of-function (LoF) variants found in CHD probands from the PCGC. In addition to a previously published cohort of parent-offspring CHD trios and control trios (<span class="dropBlock reference-citations"><a id="body-ref-sref63-3" href="#" role="doc-biblioref" data-xml-rid="bib63" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref63-3" aria-controls="sref63-3">Jin et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref63-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">56.</div><div>Jin, S.C. ∙ Homsy, J. ∙ Zaidi, S. ...</div><div><strong>Contribution of rare inherited and de novo variants in 2,871 congenital heart disease probands</strong></div><div><em>Nat. Genet.</em> 2017; <strong>49</strong>:1593-1601</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng.3970" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_56_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-85032451853" target="_blank" aria-label="Open Scopus in new tab">Scopus (358)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28991257/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng.3970&amp;pmid=28991257" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), we included variant data from an additional 419 CHD probands and their parents for a total of over 3,000 trios. We used a permutation-based statistical test to analyze the frequency of variants in GT-interacting proteins among the CHD probands compared to the control group (see <a id="crosref0655" href="#sec-5">STAR Methods: Permutation-based test</a>). Briefly, the observed odds ratio (OR) of finding a DNV in an interactome gene was adjusted by a factor correcting for synonymous mutation frequency (adjusted OR), then compared to a distribution of odds ratios in which the case/control status of the dataset was permuted (permuted ORs) (<a id="crosref0660" href="#fig2">Figure&nbsp;2</a>A). The analysis indicated that protein-altering DNVs were significantly more likely to be found within GT interactors in the CHD cohort relative to the control cohort (adjusted OR GATA4-PPI: 5.59 and adjusted p value 0.001; TBX5-PPI: 4.34 and adjusted p value 0.0096). By contrast, very rare inherited LoF variants occurred in GT-PPI proteins with the same frequency in control and CHD groups (<a id="crosref0665" href="#fig2">Figure&nbsp;2</a>B; <a id="crosref0670" href="#mmc1">Table S1</a>E).</div><div class="figure-wrap"><figure id="fig2" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2022.01.021/asset/3b704776-a9be-4b5f-9794-ebc6525e7603/main.assets/gr2_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr2" src="/cms/10.1016/j.cell.2022.01.021/asset/25214844-0c30-4134-bed6-69da20f060e5/main.assets/gr2.jpg" height="799" width="497" aria-labelledby="fig2-title fig2-content" data-viewer-src="/cms/10.1016/j.cell.2022.01.021/asset/3b704776-a9be-4b5f-9794-ebc6525e7603/main.assets/gr2_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig2-title" class="figure__title"><span class="label figure__label">Figure&nbsp;2</span> <span class="figure__title__text">Enrichment of <i>de novo</i> variants in CHD trios among GATA4 and TBX5 interactome proteins</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig2-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0045" role="paragraph">(A) Permutation-based statistical test design to analyze enrichment in genetic variants from a CHD cohort relative to a control cohort in GATA4 or TBX5 PPIs (odds ratio, OR) (see <a id="crosref2275" href="#sec-5">STAR Methods: Permutation-based test</a>).</div><div id="fspara0050" role="paragraph">(B) Results of permutation-based test in (A) for genomic variation indicated from PCGC CHD and control cohorts within the GATA4 or TBX5 interactomes in cardiac progenitors (CP interactome), or after removing proteins involved in human or mouse cardiac malformations (CP interactome heart dev. unknown) (See <a id="crosref2280" href="#mmc1">Table S1</a>F). The same analysis shown for HEK293s (HEK293 interactome).</div><div id="fspara0055" role="paragraph">(C) Violin plot for the Combined Annotation-Dependent Depletion (CADD) scores of protein-altering or synonymous (Syn) variants found in the CHD cohort affecting proteins within the GT-PPI or proteins outside the interactome. White dot&nbsp;= median; black lines&nbsp;= interquartile range (thick) or 1.5× the interquartile range (thin). Two-sided Mann-Whitney-Wilcoxon test with Bonferroni correction for p values; <sup>∗∗∗</sup>p value&nbsp;&lt; 0.001.</div><div id="fspara0060" role="paragraph">See also <a id="crosref2285" href="#figs3">Figure&nbsp;S3</a> and <a id="crosref2290" href="#mmc1">Tables S1</a>E–S1I.</div></div></div></div></figcaption></figure></div><div id="p0080" role="paragraph">To determine whether the enrichment was predominately driven by genes previously known to be involved in cardiac development (<a id="crosref0675" href="#figs3">Figure&nbsp;S3</a>A), we removed a published curated list of genes involved in human or mouse cardiac malformations from the dataset (<span class="dropBlock reference-citations"><a id="body-ref-sref63-4" href="#" role="doc-biblioref" data-xml-rid="bib63" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref63-4" aria-controls="sref63-4">Jin et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref63-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">56.</div><div>Jin, S.C. ∙ Homsy, J. ∙ Zaidi, S. ...</div><div><strong>Contribution of rare inherited and de novo variants in 2,871 congenital heart disease probands</strong></div><div><em>Nat. Genet.</em> 2017; <strong>49</strong>:1593-1601</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng.3970" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_56_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-85032451853" target="_blank" aria-label="Open Scopus in new tab">Scopus (358)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28991257/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng.3970&amp;pmid=28991257" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and repeated the permutation-based analysis (<a id="crosref0685" href="#mmc1">Table S1</a>F). We still found an enrichment in proteins harboring protein-altering DNVs from CHD probands in both GATA4 and TBX5 interactomes (<a id="crosref0690" href="#fig2">Figure&nbsp;2</a>B; <a id="crosref0695" href="#mmc1">Table S1</a>E). Similar trends were observed holding out a smaller list of 144 published human CHD genes (<a id="crosref0700" href="#mmc1">Table S1</a>E) (<span class="dropBlock reference-citations"><a id="body-ref-sref60-1" href="#" role="doc-biblioref" data-xml-rid="bib60" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref60-1" aria-controls="sref60-1">Izarzugaza et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref60-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">54.</div><div>Izarzugaza, J.M.G. ∙ Ellesøe, S.G. ∙ Doganli, C. ..., Competence Network for Congenital Heart Defects, Germany</div><div><strong>Systems genetics analysis identifies calcium-signaling defects as novel cause of congenital heart disease</strong></div><div><em>Genome Med.</em> 2020; <strong>12</strong>:76</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13073-020-00772-z" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-85090171445" target="_blank" aria-label="Open Scopus in new tab">Scopus (5)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32859249/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13073-020-00772-z&amp;pmid=32859249" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div class="figure-wrap"><figure id="figs3" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2022.01.021/asset/255aedf0-9e92-4e01-9c5f-5c99083f4f28/main.assets/figs3_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs3" src="/cms/10.1016/j.cell.2022.01.021/asset/3f397664-29f3-4b2c-9eb4-ebd5a92519bf/main.assets/figs3.jpg" height="939" width="748" aria-labelledby="figs3-title figs3-content" data-viewer-src="/cms/10.1016/j.cell.2022.01.021/asset/255aedf0-9e92-4e01-9c5f-5c99083f4f28/main.assets/figs3_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs3-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S3</span> <span class="figure__title__text">GT-PPIs in the kidney cell line HEK293 and features of CHD candidate genes in the GT interactome from CPs, related to <a id="crosref2405" href="#fig2">Figures 2</a> and <a id="crosref2410" href="#fig3">3</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs3-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0330" role="paragraph">(A) GT interactors with CHD-associated DNVs previously implicated in human cardiac malformations (<span class="dropBlock reference-citations"><a id="body-ref-sref13" href="#" role="doc-biblioref" data-xml-rid="bib13" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref13" aria-controls="sref13">Bouman et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref13"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">13.</div><div>Bouman, A. ∙ Alders, M. ∙ Oostra, R.J. ...</div><div><strong>Oral-facial-digital syndrome type 1 in males: Congenital heart defects are included in its phenotypic spectrum</strong></div><div><em>Am. J. Med. Genet. A.</em> 2017; <strong>173</strong>:1383-1389</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/ajmg.a.38179" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_13_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-85017332855" target="_blank" aria-label="Open Scopus in new tab">Scopus (8)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28371265/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fajmg.a.38179&amp;pmid=28371265" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref23" href="#" role="doc-biblioref" data-xml-rid="bib23" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref23" aria-controls="sref23">Chen et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref23"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">22.</div><div>Chen, J. ∙ Yuan, H. ∙ Xie, K. ...</div><div><strong>A novel TAB2 nonsense mutation (p.S149X) causing autosomal dominant congenital heart defects: a case report of a Chinese family</strong></div><div><em>BMC Cardiovasc. Disord.</em> 2020; <strong>20</strong>:27</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s12872-019-01322-1" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-85078306887" target="_blank" aria-label="Open Scopus in new tab">Scopus (6)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31959127/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs12872-019-01322-1&amp;pmid=31959127" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref63-5" href="#" role="doc-biblioref" data-xml-rid="bib63" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref63-5" aria-controls="sref63-5">Jin et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref63-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">56.</div><div>Jin, S.C. ∙ Homsy, J. ∙ Zaidi, S. ...</div><div><strong>Contribution of rare inherited and de novo variants in 2,871 congenital heart disease probands</strong></div><div><em>Nat. Genet.</em> 2017; <strong>49</strong>:1593-1601</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng.3970" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_56_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-85032451853" target="_blank" aria-label="Open Scopus in new tab">Scopus (358)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28991257/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng.3970&amp;pmid=28991257" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref64" href="#" role="doc-biblioref" data-xml-rid="bib64" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref64" aria-controls="sref64">Jones et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref64"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">57.</div><div>Jones, W.D. ∙ Dafou, D. ∙ McEntagart, M. ...</div><div><strong>De novo mutations in MLL cause Wiedemann-Steiner syndrome</strong></div><div><em>Am. J. Hum. Genet.</em> 2012; <strong>91</strong>:358-364</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=10.1016%2Fj.ajhg.2012.06.008&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=10.1016%2Fj.ajhg.2012.06.008&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-84864958596" target="_blank" aria-label="Open Scopus in new tab">Scopus (171)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22795537/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.ajhg.2012.06.008&amp;pmid=22795537" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref92" href="#" role="doc-biblioref" data-xml-rid="bib92" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref92" aria-controls="sref92">Maitra et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref92"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">80.</div><div>Maitra, M. ∙ Koenig, S.N. ∙ Srivastava, D. ...</div><div><strong>Identification of GATA6 sequence variants in patients with congenital heart defects</strong></div><div><em>Pediatr. Res.</em> 2010; <strong>68</strong>:281-285</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1203/PDR.0b013e3181ed17e4" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_80_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-77957274392" target="_blank" aria-label="Open Scopus in new tab">Scopus (85)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20581743/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1203%2FPDR.0b013e3181ed17e4&amp;pmid=20581743" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref104" href="#" role="doc-biblioref" data-xml-rid="bib104" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref104" aria-controls="sref104">Parisot et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref104"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">92.</div><div>Parisot, P. ∙ Bajolle, F. ∙ Attié-Bittach, T. ...</div><div><strong>321 Congenital heart defects in CHARGE syndrome patients with CHD7 mutations</strong></div><div><em>Archives of Cardiovascular Diseases Supplements.</em> 2010; <strong>2</strong>:104-105</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/S1878-6480(10)70323-2" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS1878-6480%2810%2970323-2" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref106-1" href="#" role="doc-biblioref" data-xml-rid="bib106" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref106-1" aria-controls="sref106-1">Pierpont et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref106-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">94.</div><div>Pierpont, M.E. ∙ Brueckner, M. ∙ Chung, W.K. ..., American Heart Association Council on Cardiovascular Disease in the Young; Council on Cardiovascular and Stroke Nursing; and Council on Genomic and Precision Medicine</div><div><strong>Genetic basis for congenital heart disease: revisited: A scientific statement from the american heart association</strong></div><div><em>Circulation.</em> 2018; <strong>138</strong>:e653-e711</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1161/CIR.0000000000000606" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_94_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-85058908137" target="_blank" aria-label="Open Scopus in new tab">Scopus (203)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30571578/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1161%2FCIR.0000000000000606&amp;pmid=30571578" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref130" href="#" role="doc-biblioref" data-xml-rid="bib130" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref130" aria-controls="sref130">Thienpont et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref130"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">117.</div><div>Thienpont, B. ∙ Zhang, L. ∙ Postma, A.V. ...</div><div><strong>Haploinsufficiency of TAB2 causes congenital heart defects in humans</strong></div><div><em>Am. J. Hum. Genet.</em> 2010; <strong>86</strong>:839-849</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_117_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=10.1016%2Fj.ajhg.2010.04.011&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_117_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=10.1016%2Fj.ajhg.2010.04.011&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_117_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-77953231502" target="_blank" aria-label="Open Scopus in new tab">Scopus (75)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20493459/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.ajhg.2010.04.011&amp;pmid=20493459" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="fspara0335" role="paragraph">(B and C) Venn diagram representing the overlap of the GATA4 or TBX5 PPIs between hiPS cell-derived CPs and HEK293 cells.</div><div id="fspara0340" role="paragraph">(D) GT-PPI reconstructed in HEK293 kidney cells. FLAG tagged GATA4 or TBX5 proteins were ectopically expressed in HEK293 cells and the cells collected 48&nbsp;h after transfection; an empty vector was used as negative control. Nuclear-enriched lysates treated with benzonase (DNase/RNase enzyme) were subjected to affinity purification (AP) with anti-FLAG antibodies. For each AP condition, replicates from three independent transfections were analyzed by mass spectrometry (LC/MS). AP-MS results from the negative controls were used to remove antibody-specific background from the experimental samples’ signal; data were subjected to the same filtering steps as the CP AP-MS data to identify high-confidence GATA4 and TBX5 PPIs. Enriched proteins with a BFDR&nbsp;&lt; 0.05 are represented in the network. CP and HEK293 overlapping TBX5, GATA4 and TBX5 &amp; GATA4 interactors are highlighted with a colored node border in brown, black and green respectively.</div><div id="fspara0345" role="paragraph">(E) Violin plot of the haploinsufficiency scores for synonymous (Syn) or protein-altering DNVs found in the CHD cohort and affecting proteins inside the GT interactome (GT-PPI) compared to outside the interactome (Non-Interactome). The white dot represents the median, the black lines the interquartile range (thick) and 1.5x the interquartile range (thin). P values were determined using a two-sided Mann-Whitney-Wilcoxon test with Bonferroni correction; the number of asterisks indicate significance level (<sup>∗∗∗</sup>p value&nbsp;&lt; 0.001).</div><div id="fspara0350" role="paragraph">(F and G) Dot plot representing the expression patterns of interactome genes harboring CHD-associated protein-altering DNVs in the (B) human developing heart from DESCARTES gene expression atlas (<span class="dropBlock reference-citations"><a id="body-ref-sref18-2" href="#" role="doc-biblioref" data-xml-rid="bib18" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref18-2" aria-controls="sref18-2">Cao et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref18-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">18.</div><div>Cao, J. ∙ O’Day, D.R. ∙ Pliner, H.A. ...</div><div><strong>A human cell atlas of fetal gene expression</strong></div><div><em>Science.</em> 2020; <strong>370</strong></div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aba7721" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-85096082939" target="_blank" aria-label="Open Scopus in new tab">Scopus (135)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aba7721" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) or (C) DNVs in the mouse developing heart (average of E7.75, E8.25 and E9.25) based on published single-cell RNaseq data (<span class="dropBlock reference-citations"><a id="body-ref-sref28-1" href="#" role="doc-biblioref" data-xml-rid="bib28" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref28-1" aria-controls="sref28-1">de Soysa et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref28-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">27.</div><div>de Soysa, T.Y. ∙ Ranade, S.S. ∙ Okawa, S. ...</div><div><strong>Single-cell analysis of cardiogenesis reveals basis for organ-level developmental defects</strong></div><div><em>Nature.</em> 2019; <strong>572</strong>:120-124</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-1414-x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-85069751389" target="_blank" aria-label="Open Scopus in new tab">Scopus (91)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31341279/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-1414-x&amp;pmid=31341279" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The size of the dot indicates the percentage of cells expressing that gene within a cluster and the color indicates the average expression level of that gene within a cluster.</div><div id="fspara0355" role="paragraph">(H) Distribution of GT-PPI and Non-Interactome genes harboring CHD-associated protein-altering DNVs across the five Human Protein Atlas categories based on transcript specificity in 37 analyzed tissues (See Methods). Tissue enriched: At least four-fold higher mRNA level in a particular tissue compared to any other tissues; Group enriched: At least four-fold higher average mRNA level in a group of 2-5 tissues compared to any other tissue; Tissue enhanced: At least four-fold higher mRNA level in a particular tissue compared to the average level in all other tissues; Low tissue specificity: detected and not within the other categories; Non detected.</div><div id="fspara0360" role="paragraph">(I) Violin plot representing the distribution of Heart Enriched Expression (Log<sub>2</sub> Heart GTEX RPKM/ Average RPKM in 18 non-heart tissues) for synonymous (Syn) and protein-altering DNVs found in the CHD cohort and affecting proteins inside the GT interactome (GT-PPI) or outside the interactome (Non-Interactome). The white dot represents the median, the black lines the interquartile range (thick) and 1.5x the interquartile range (thin). P values were determined using a two-sided Mann-Whitney-Wilcoxon test with Bonferroni correction; the number of asterisks indicate significance level (<sup>∗∗</sup>p value&nbsp;&lt; 0.01, <sup>∗</sup>p value&nbsp;&lt; 0.05).</div><div id="fspara0365" role="paragraph">(J) Venn diagram representing the number of interactome genes with protein-altering DNVs found in probands suffering from “isolated CHD,” CHD with concomitant extra-cardiac defects (extracardiac abnormalities and/or neurodevelopmental defects), or in both types of CHD.</div><div id="fspara0370" role="paragraph">(K) Number of mutations per cDNA kilobase, based on the number of mutations per gene corrected by the gene’s length, for synonymous (Syn) and protein-altering DNVs found in the CHD cohort and affecting proteins inside the GT interactome (GT-PPI) or outside the interactome (Non-Interactome). The white dot represents the median, the black lines the interquartile range (thick) and 1.5x the interquartile range (thin).</div></div></div></div></figcaption></figure></div><div id="p0085" role="paragraph">Although our AP-MS analysis was conducted in human CP cells for endogenous TBX5 and GATA4, most PPIs have been identified in less biologically relevant cells and upon overexpression. To assess the importance of biological context, we generated GT-PPIs in kidney cells (HEK293) overexpressing human GATA4 or TBX5 and subjected them to the same permutation analysis with the CHD and control cohorts (<a id="crosref0710" href="#figs3">Figures S3</a>B–S3D; <a id="crosref0715" href="#mmc1">Tables S1</a>G and S1H). There was no significant enrichment in proteins harboring CHD-associated protein-altering DNVs (<a id="crosref0720" href="#fig2">Figure&nbsp;2</a>B; <a id="crosref0725" href="#mmc1">Table S1</a>E). The GT-PPI overlap between cell types was small, with only 20 GATA4 and 13 TBX5-interactors shared (<a id="crosref0730" href="#figs3">Figures S3</a>B–S3D; <a id="crosref0735" href="#mmc1">Tables S1</a>G and S1H), highlighting the importance of endogenous tissue-specific protein-protein interactions in elucidating the genetic underpinnings of human diseases.</div><div id="p0090" role="paragraph">In a complementary analysis to test whether genes in the GT-PPI were enriched for protein-altering DNVs in CHD probands, we permuted the list of interactors and tallied the number of variants found in each gene set. This allowed us to compare the null distribution of the number of variants found in otherwise-comparable non-GT-PPI genes to what we observed in interactome genes. For each gene in the GT-PPI, we identified other genes that had comparable <i>de novo</i> mutability scores (<span class="dropBlock reference-citations"><a id="body-ref-sref119" href="#" role="doc-biblioref" data-xml-rid="bib119" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref119" aria-controls="sref119">Samocha et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref119"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">107.</div><div>Samocha, K.E. ∙ Robinson, E.B. ∙ Sanders, S.J. ...</div><div><strong>A framework for the interpretation of de novo mutation in human disease</strong></div><div><em>Nat. Genet.</em> 2014; <strong>46</strong>:944-950</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng.3050" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_107_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-84905309176" target="_blank" aria-label="Open Scopus in new tab">Scopus (591)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25086666/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng.3050&amp;pmid=25086666" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). We further narrowed the list of matches based on similarity of expression levels in WT CP cells. The observed number of protein-altering DNVs was significantly higher in GT-PPI genes compared to permuted selections of non-interactome genes with similar mutability and expression (Bonferroni-adjusted p&nbsp;= 0.009). Conversely, there was no enrichment of non-synonymous or rare inherited LoF variants among GT-PPI genes (<a id="crosref0745" href="#mmc1">Table S1</a>I).</div><div id="p0095" role="paragraph">Having demonstrated that the GT-PPI was enriched in protein-altering variants found in CHD patients, we aimed to assess the likelihood that the GT-PPI variants contribute to disease. Using combined annotation-dependent depletion (CADD) scores, we found that GT-PPI protein-altering variants found in the CHD cases were more likely predicted to be deleterious than the rest of protein-altering DNVs in CHD cases outside the GT interactome (<a id="crosref0750" href="#fig2">Figure&nbsp;2</a>C).</div></section><section id="sec-2-3"><h3>GATA4:TBX5 interactors with protein-altering DNVs unveil CHD candidate genes with characteristic features of disease genes</h3><div id="p0100" role="paragraph">We next asked whether the candidate CHD genes identified in the GT-PPI exhibited features that could increase their likelihood of causing disease compared to the remaining non-interactome genes mutated in CHD probands. Extreme intolerance to LoF variation and haplo-insufficiency are common features of genes associated with developmental disorders (<span class="dropBlock reference-citations"><a id="body-ref-sref42" href="#" role="doc-biblioref" data-xml-rid="bib42" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref42" aria-controls="sref42">Fuller et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref42"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">38.</div><div>Fuller, Z.L. ∙ Berg, J.J. ∙ Mostafavi, H. ...</div><div><strong>Measuring intolerance to mutation in human genetics</strong></div><div><em>Nat. Genet.</em> 2019; <strong>51</strong>:772-776</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41588-019-0383-1" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_38_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-85064090845" target="_blank" aria-label="Open Scopus in new tab">Scopus (43)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30962618/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41588-019-0383-1&amp;pmid=30962618" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Remarkably, most candidate CHD genes in the PPI were extremely intolerant to LoF variation (probability of being intolerant to LoF [pLI] &gt; 0.9) and exhibited significantly higher pLI and haplo-insufficiency scores than genes outside the interactome with protein-altering DNVs (<a id="crosref0760" href="#fig3">Figures 3</a>A and <a id="crosref0765" href="#figs3">S3</a>E). Another feature of disease genes is an increased tendency for their products to interact with one another when their mutations result in similar phenotypes (<span class="dropBlock reference-citations"><a id="body-ref-sref46" href="#" role="doc-biblioref" data-xml-rid="bib46" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref46" aria-controls="sref46">Goh et&nbsp;al., 2007</a><div class="dropBlock__holder" data-db-target-of="sref46"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">42.</div><div>Goh, K.-I. ∙ Cusick, M.E. ∙ Valle, D. ...</div><div><strong>The human disease network</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2007; <strong>104</strong>:8685-8690</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.0701361104" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-34547140875" target="_blank" aria-label="Open Scopus in new tab">Scopus (2412)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17502601/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.0701361104&amp;pmid=17502601" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Based on iRefIndex database information (<span class="dropBlock reference-citations"><a id="body-ref-sref112-1" href="#" role="doc-biblioref" data-xml-rid="bib112" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref112-1" aria-controls="sref112-1">Razick et&nbsp;al., 2008</a><div class="dropBlock__holder" data-db-target-of="sref112-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">100.</div><div>Razick, S. ∙ Magklaras, G. ∙ Donaldson, I.M.</div><div><strong>iRefIndex: a consolidated protein interaction database with provenance</strong></div><div><em>BMC Bioinformatics.</em> 2008; <strong>9</strong>:405</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/1471-2105-9-405" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_100_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-54949127160" target="_blank" aria-label="Open Scopus in new tab">Scopus (405)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/18823568/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2F1471-2105-9-405&amp;pmid=18823568" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), the proteins encoded by our candidate genes had a higher connectivity degree with other proteins found to be mutated in the CHD cohort as well as with a curated list of proteins involved in mouse/human cardiac malformations (<span class="dropBlock reference-citations"><a id="body-ref-sref63-6" href="#" role="doc-biblioref" data-xml-rid="bib63" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref63-6" aria-controls="sref63-6">Jin et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref63-6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">56.</div><div>Jin, S.C. ∙ Homsy, J. ∙ Zaidi, S. ...</div><div><strong>Contribution of rare inherited and de novo variants in 2,871 congenital heart disease probands</strong></div><div><em>Nat. Genet.</em> 2017; <strong>49</strong>:1593-1601</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng.3970" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_56_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-85032451853" target="_blank" aria-label="Open Scopus in new tab">Scopus (358)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28991257/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng.3970&amp;pmid=28991257" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) than proteins outside the interactome with protein-altering DNVs (<a id="crosref0785" href="#fig3">Figures 3</a>B and 3C).</div><div class="figure-wrap"><figure id="fig3" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2022.01.021/asset/6e326926-f3cc-414b-bf98-b05d902e43af/main.assets/gr3_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr3" src="/cms/10.1016/j.cell.2022.01.021/asset/20bea442-66aa-4897-a122-7420118503ba/main.assets/gr3.jpg" height="760" width="656" aria-labelledby="fig3-title fig3-content" data-viewer-src="/cms/10.1016/j.cell.2022.01.021/asset/6e326926-f3cc-414b-bf98-b05d902e43af/main.assets/gr3_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig3-title" class="figure__title"><span class="label figure__label">Figure&nbsp;3</span> <span class="figure__title__text"><i>De novo</i> variants in GATA4 and TBX5 interactomes exhibit features typical of disease genes</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig3-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0070" role="paragraph">(A–D) Violin plots for the distribution of (A) intolerance to LoF (pLI Score), (B) degree of connectivity with all protein-altering DNVs found in the CHD cohort, (C) degree of connectivity with proteins encoded by genes involved in mouse/human cardiac malformations (<span class="dropBlock reference-citations"><a id="body-ref-sref63-7" href="#" role="doc-biblioref" data-xml-rid="bib63" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref63-7" aria-controls="sref63-7">Jin et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref63-7"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">56.</div><div>Jin, S.C. ∙ Homsy, J. ∙ Zaidi, S. ...</div><div><strong>Contribution of rare inherited and de novo variants in 2,871 congenital heart disease probands</strong></div><div><em>Nat. Genet.</em> 2017; <strong>49</strong>:1593-1601</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng.3970" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_56_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-85032451853" target="_blank" aria-label="Open Scopus in new tab">Scopus (358)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28991257/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng.3970&amp;pmid=28991257" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), and (D) expression percentile rank in the developing heart (E14.5) for genes harboring synonymous (Syn) or protein-altering DNVs found in the CHD cohort and affecting proteins inside the GT interactome (GT-PPI) or outside the interactome (non-interactome). White dot&nbsp;= median; black lines&nbsp;= interquartile range (thick) and 1.5× the interquartile range (thin). Two-sided Mann-Whitney-Wilcoxon test with Bonferroni correction for p values; <sup>∗∗∗</sup>p value&nbsp;&lt; 0.001, <sup>∗∗</sup>p value&nbsp;&lt; 0.01, <sup>∗</sup>p value&nbsp;&lt; 0.05, and ns: non-significant.</div><div id="fspara0075" role="paragraph">(E) Pie chart of tissue expression distribution of GT-PPI or non-interactome genes harboring CHD-associated protein-altering DNVs across the six Human Protein Atlas categories (see <a id="crosref2300" href="#sec-5">STAR Methods</a>).</div><div id="fspara0080" role="paragraph">(F) Interactome CHD candidate genes represented as a network after integration with PPI information from iRefIndex database. Nodes colored based on manually annotated biological processes and protein families/complexes grouped in boxed areas. Node size reflects probability of loss-of-function intolerance (pLI) scores. Node shape reflects belonging to TBX5 (triangle), GATA4 (circle), or GATA4&amp;TBX5 (square) networks. Red highlights proteins encoded by genes involved in human CHD. Edges represent protein-protein interactions from iRefIndex database (<span class="dropBlock reference-citations"><a id="body-ref-sref112-2" href="#" role="doc-biblioref" data-xml-rid="bib112" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref112-2" aria-controls="sref112-2">Razick et&nbsp;al., 2008</a><div class="dropBlock__holder" data-db-target-of="sref112-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">100.</div><div>Razick, S. ∙ Magklaras, G. ∙ Donaldson, I.M.</div><div><strong>iRefIndex: a consolidated protein interaction database with provenance</strong></div><div><em>BMC Bioinformatics.</em> 2008; <strong>9</strong>:405</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/1471-2105-9-405" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_100_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-54949127160" target="_blank" aria-label="Open Scopus in new tab">Scopus (405)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/18823568/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2F1471-2105-9-405&amp;pmid=18823568" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="fspara0085" role="paragraph">See also <a id="crosref2310" href="#figs3">Figure&nbsp;S3</a> and <a id="crosref2315" href="#mmc1">Tables S1</a>J and S1K.</div></div></div></div></figcaption></figure></div><div id="p0105" role="paragraph">GT interactors with protein-altering DNVs among those with CHD exhibited higher expression in the developing heart than genes with protein-altering DNVs outside the GT-PPIs (<a id="crosref0790" href="#fig3">Figure&nbsp;3</a>D), but they generally displayed a broad expression pattern across most cell types (<a id="crosref0795" href="#fig3">Figures 3</a>E and <a id="crosref0800" href="#figs3">S3</a>F–S3I) and largely involved proteins relevant to chromatin biology (<a id="crosref0805" href="#fig3">Figure&nbsp;3</a>F). Other biological processes with unexplored roles in CHD were affected, such as RNA splicing and protein folding (<a id="crosref0810" href="#fig3">Figure&nbsp;3</a>F). Furthermore, although many DNVs in GT interactors were detected in probands suffering from CHD with extracardiac abnormalities and/or neurodevelopmental defects, a sizeable number were also found in “isolated” CHD cases (<a id="crosref0815" href="#figs3">Figure&nbsp;S3</a>J; <a id="crosref0820" href="#mmc1">Table S1</a>J).</div><div id="p0110" role="paragraph">We next investigated the specific types of protein-altering <i>de novo</i> CHD variants corresponding to proteins in the GT-PPIs. Among the 272 proteins in the GT-PPI, we identified 20 LoF DNVs and 53 missense DNVs present in CHD cases (<a id="crosref0825" href="#mmc1">Table S1</a>J). The odds of a DNV occurring in a GT-PPI gene was substantially greater in CHD probands compared to controls for both LoF (adj. OR 4.96) and missense DNVs (adj. OR: 3.76) (<a id="crosref0830" href="#mmc1">Table S1</a>K). LoF DNVs preferentially affected genes involved in human and mouse cardiac malformations, whereas the bulk of GT-PPI genes with CHD-missense DNVs had not previously been linked to cardiac development or CHD (<a id="crosref0835" href="#mmc1">Tables S1</a>J and S1K). The contribution of <i>de novo</i> splice variants could not be determined due to their low counts in interactome genes from cases and controls (<a id="crosref0840" href="#mmc1">Tables S1</a>J and S1K).</div></section><section id="sec-2-4"><h3>An integrative method for scoring variants identifies specific GT interactors as candidate genes for CHD</h3><div id="p0115" role="paragraph">The GT-PPI framework combined with trio sequencing allowed us to significantly reduce the number of candidate variants in individual genomes to 20 LoF and 53 missense DNVs in genes encoding protein partners of cTFs that may contribute to CHD. However, even after this significant filtering step, the interpretation of missense variants remains a challenge and requires methods to prioritize those that could substantially impact human phenotypes.</div><div id="p0120" role="paragraph">Many variant prioritization methods have been described to date, and most integrate widely accepted variant and gene features to rank potential candidate variants based on the combined evidence of the variant’s predicted deleterious effect on protein function, the harboring gene’s accumulated mutational damage, and its biological relatedness to known CHD-causing genes (<span class="dropBlock reference-citations"><a id="body-ref-sref34-1" href="#" role="doc-biblioref" data-xml-rid="bib34" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref34-1" aria-controls="sref34-1">Eilbeck et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref34-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">31.</div><div>Eilbeck, K. ∙ Quinlan, A. ∙ Yandell, M.</div><div><strong>Settling the score: variant prioritization and Mendelian disease</strong></div><div><em>Nat. Rev. Genet.</em> 2017; <strong>18</strong>:599-612</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nrg.2017.52" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-85029470345" target="_blank" aria-label="Open Scopus in new tab">Scopus (133)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28804138/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnrg.2017.52&amp;pmid=28804138" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref71-1" href="#" role="doc-biblioref" data-xml-rid="bib71" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref71-1" aria-controls="sref71-1">Köhler et&nbsp;al., 2008</a><div class="dropBlock__holder" data-db-target-of="sref71-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">63.</div><div>Köhler, S. ∙ Bauer, S. ∙ Horn, D. ...</div><div><strong>Walking the interactome for prioritization of candidate disease genes</strong></div><div><em>Am. J. Hum. Genet.</em> 2008; <strong>82</strong>:949-958</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=10.1016%2Fj.ajhg.2008.02.013&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=10.1016%2Fj.ajhg.2008.02.013&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-41549139527" target="_blank" aria-label="Open Scopus in new tab">Scopus (910)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/18371930/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.ajhg.2008.02.013&amp;pmid=18371930" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref113-1" href="#" role="doc-biblioref" data-xml-rid="bib113" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref113-1" aria-controls="sref113-1">Rentzsch et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref113-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">101.</div><div>Rentzsch, P. ∙ Witten, D. ∙ Cooper, G.M. ...</div><div><strong>CADD: predicting the deleteriousness of variants throughout the human genome</strong></div><div><em>Nucleic Acids Res.</em> 2019; <strong>47</strong>:D886-D894</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gky1016" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_101_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-85059798241" target="_blank" aria-label="Open Scopus in new tab">Scopus (1261)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30371827/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgky1016&amp;pmid=30371827" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref121-1" href="#" role="doc-biblioref" data-xml-rid="bib121" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref121-1" aria-controls="sref121-1">Sevim Bayrak et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref121-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">109.</div><div>Sevim Bayrak, C. ∙ Zhang, P. ∙ Tristani-Firouzi, M. ...</div><div><strong>De novo variants in exomes of congenital heart disease patients identify risk genes and pathways</strong></div><div><em>Genome Med.</em> 2020; <strong>12</strong>:9</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13073-019-0709-8" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_109_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-85077940747" target="_blank" aria-label="Open Scopus in new tab">Scopus (25)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31941532/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13073-019-0709-8&amp;pmid=31941532" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). However, the development of a score that would work universally is theoretically difficult, and a common finding of many genetic studies is that gene-set specific rules for pathogenicity are required for proper evaluation (<span class="dropBlock reference-citations"><a id="body-ref-sref34-2" href="#" role="doc-biblioref" data-xml-rid="bib34" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref34-2" aria-controls="sref34-2">Eilbeck et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref34-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">31.</div><div>Eilbeck, K. ∙ Quinlan, A. ∙ Yandell, M.</div><div><strong>Settling the score: variant prioritization and Mendelian disease</strong></div><div><em>Nat. Rev. Genet.</em> 2017; <strong>18</strong>:599-612</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nrg.2017.52" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-85029470345" target="_blank" aria-label="Open Scopus in new tab">Scopus (133)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28804138/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnrg.2017.52&amp;pmid=28804138" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). In addition, most of these methods were designed for singleton sequencing studies and fail to incorporate proband pedigree information that can aid prioritizing variants with potential greater effect within an individual (<span class="dropBlock reference-citations"><a id="body-ref-sref37" href="#" role="doc-biblioref" data-xml-rid="bib37" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref37" aria-controls="sref37">Farwell et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref37"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">34.</div><div>Farwell, K.D. ∙ Shahmirzadi, L. ∙ El-Khechen, D. ...</div><div><strong>Enhanced utility of family-centered diagnostic exome sequencing with inheritance model-based analysis: results from 500 unselected families with undiagnosed genetic conditions</strong></div><div><em>Genet. Med.</em> 2015; <strong>17</strong>:578-586</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=10.1038%2Fgim.2014.154&amp;cf=fulltext&amp;site=gim-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=10.1038%2Fgim.2014.154&amp;cf=pdf&amp;site=gim-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-84939635642" target="_blank" aria-label="Open Scopus in new tab">Scopus (332)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25356970/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fgim.2014.154&amp;pmid=25356970" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref125" href="#" role="doc-biblioref" data-xml-rid="bib125" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref125" aria-controls="sref125">Stark et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref125"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">113.</div><div>Stark, Z. ∙ Dashnow, H. ∙ Lunke, S. ..., Melbourne Genomics Health Alliance</div><div><strong>A clinically driven variant prioritization framework outperforms purely computational approaches for the diagnostic analysis of singleton WES data</strong></div><div><em>Eur. J. Hum. Genet.</em> 2017; <strong>25</strong>:1268-1272</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ejhg.2017.123" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_113_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-85031307052" target="_blank" aria-label="Open Scopus in new tab">Scopus (7)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28832562/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fejhg.2017.123&amp;pmid=28832562" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Thus, we developed an integrative pipeline customized for the CHD trio whole-exome-sequencing dataset to calculate a variant prioritization score for the 53 missense DNVs mapped to our GT-PPI. This scoring method has two steps: (1) variant prioritization based on the consolidation of annotations from a combination of widely used gene and variant metrics to assess variant deleteriousness, together with the gene’s frequency of mutation within our dataset (<a id="crosref0880" href="#figs3">Figure&nbsp;S3</a>K), and (2) re-weighting based on occurrence at a known functional residue/domain and on the presence of other potentially causal variants in the same proband (<a id="crosref0885" href="#fig4">Figures 4</a>A and <a id="crosref0890" href="#figs4">S4</a>A). Specifically, at the gene level, a higher score indicates (1) a gene’s low tolerance to LoF variation, (2) connectivity to a high number of proteins involved in cardiac malformations (<span class="dropBlock reference-citations"><a id="body-ref-sref63-8" href="#" role="doc-biblioref" data-xml-rid="bib63" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref63-8" aria-controls="sref63-8">Jin et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref63-8"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">56.</div><div>Jin, S.C. ∙ Homsy, J. ∙ Zaidi, S. ...</div><div><strong>Contribution of rare inherited and de novo variants in 2,871 congenital heart disease probands</strong></div><div><em>Nat. Genet.</em> 2017; <strong>49</strong>:1593-1601</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng.3970" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_56_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-85032451853" target="_blank" aria-label="Open Scopus in new tab">Scopus (358)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28991257/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng.3970&amp;pmid=28991257" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and to PCGC CHD proband variant-harboring proteins based on publicly available PPI information, (3) high cardiac expression compared to other tissues, and (4) high number of PCGC variants within the gene relative to CDS length. At the residue level, a higher score indicates (5) a variant’s increased likelihood of being deleterious to protein function (CADD score) and (6) occurrence at a functional residue or protein domain. At the proband level, through the application of a weighted correction factor, a higher score indicates that (7) the background genetic variation of this individual does not include DNVs or very rare inherited LoF variants in genes known to be involved in cardiac malformations (<span class="dropBlock reference-citations"><a id="body-ref-sref63-9" href="#" role="doc-biblioref" data-xml-rid="bib63" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref63-9" aria-controls="sref63-9">Jin et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref63-9"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">56.</div><div>Jin, S.C. ∙ Homsy, J. ∙ Zaidi, S. ...</div><div><strong>Contribution of rare inherited and de novo variants in 2,871 congenital heart disease probands</strong></div><div><em>Nat. Genet.</em> 2017; <strong>49</strong>:1593-1601</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng.3970" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_56_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-85032451853" target="_blank" aria-label="Open Scopus in new tab">Scopus (358)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28991257/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng.3970&amp;pmid=28991257" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and includes none or fewer variants in other GT-PPI genes. The individual features are combined by rank sum and weighted where applicable (see <a id="crosref0905" href="#sec-5">STAR Methods: Variant scoring</a>) (<a id="crosref0910" href="#fig4">Figures 4</a>A and <a id="crosref0915" href="#figs4">S4</a>A). The resulting score is represented with respect to the gene’s percentile of expression in the developing heart (E14.5), a feature previously shown to be effective for variant filtering in CHD by the PCGC (<span class="dropBlock reference-citations"><a id="body-ref-sref145-3" href="#" role="doc-biblioref" data-xml-rid="bib145" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref145-3" aria-controls="sref145-3">Zaidi et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref145-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">131.</div><div>Zaidi, S. ∙ Choi, M. ∙ Wakimoto, H. ...</div><div><strong>De novo mutations in histone-modifying genes in congenital heart disease</strong></div><div><em>Nature.</em> 2013; <strong>498</strong>:220-223</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature12141" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_131_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-84879001958" target="_blank" aria-label="Open Scopus in new tab">Scopus (616)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23665959/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature12141&amp;pmid=23665959" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref56-3" href="#" role="doc-biblioref" data-xml-rid="bib56" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref56-3" aria-controls="sref56-3">Homsy et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref56-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">51.</div><div>Homsy, J. ∙ Zaidi, S. ∙ Shen, Y. ...</div><div><strong>De novo mutations in congenital heart disease with neurodevelopmental and other congenital anomalies</strong></div><div><em>Science.</em> 2015; <strong>350</strong>:1262-1266</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aac9396" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_51_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-84949008129" target="_blank" aria-label="Open Scopus in new tab">Scopus (412)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26785492/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aac9396&amp;pmid=26785492" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref63-10" href="#" role="doc-biblioref" data-xml-rid="bib63" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref63-10" aria-controls="sref63-10">Jin et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref63-10"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">56.</div><div>Jin, S.C. ∙ Homsy, J. ∙ Zaidi, S. ...</div><div><strong>Contribution of rare inherited and de novo variants in 2,871 congenital heart disease probands</strong></div><div><em>Nat. Genet.</em> 2017; <strong>49</strong>:1593-1601</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng.3970" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_56_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-85032451853" target="_blank" aria-label="Open Scopus in new tab">Scopus (358)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28991257/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng.3970&amp;pmid=28991257" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref121-2" href="#" role="doc-biblioref" data-xml-rid="bib121" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref121-2" aria-controls="sref121-2">Sevim Bayrak et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref121-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">109.</div><div>Sevim Bayrak, C. ∙ Zhang, P. ∙ Tristani-Firouzi, M. ...</div><div><strong>De novo variants in exomes of congenital heart disease patients identify risk genes and pathways</strong></div><div><em>Genome Med.</em> 2020; <strong>12</strong>:9</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13073-019-0709-8" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_109_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-85077940747" target="_blank" aria-label="Open Scopus in new tab">Scopus (25)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31941532/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13073-019-0709-8&amp;pmid=31941532" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div class="figure-wrap"><figure id="fig4" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2022.01.021/asset/3dc15568-216c-484e-9940-2322dcbe46ef/main.assets/gr4_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr4" src="/cms/10.1016/j.cell.2022.01.021/asset/0f48180d-f39d-413b-9dd6-33d0ed8c54d5/main.assets/gr4.jpg" height="504" width="656" aria-labelledby="fig4-title fig4-content" data-viewer-src="/cms/10.1016/j.cell.2022.01.021/asset/3dc15568-216c-484e-9940-2322dcbe46ef/main.assets/gr4_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig4-title" class="figure__title"><span class="label figure__label">Figure&nbsp;4</span> <span class="figure__title__text">Integrative variant prioritization scoring to rank CHD-associated variants</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig4-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0095" role="paragraph">(A) Variant prioritization score strategy (see <a id="crosref2320" href="#sec-5">STAR Methods: Variant scoring</a> and <a id="crosref2325" href="#figs5">Figure&nbsp;S5</a>A).</div><div id="fspara0100" role="paragraph">(B) Variant prioritization scores for interactome missense DNVs in described CHD genes (red) or in CHD candidate genes (green) plotted against the corresponding genes’ expression percentile rank in the developing heart (E14.5). Published mutations with strong contribution (blue) or partial contribution (orange) to CHD are included as references.</div><div id="fspara0105" role="paragraph">(C) Biochemical evaluation by luciferase assays of the functional impact for variant alleles with different prioritization scores in (B) within <i>NKX2-5</i>, <i>CHD7</i>, <i>BRD4</i>, or <i>SMARCC1</i>. The CHD7 ATPase mutant is used as positive control for <i>CHD7</i> loss of function (<span class="dropBlock reference-citations"><a id="body-ref-sref84-1" href="#" role="doc-biblioref" data-xml-rid="bib84" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref84-1" aria-controls="sref84-1">Liu et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref84-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">72.</div><div>Liu, Y. ∙ Harmelink, C. ∙ Peng, Y. ...</div><div><strong>CHD7 interacts with BMP R-SMADs to epigenetically regulate cardiogenesis in mice</strong></div><div><em>Hum. Mol. Genet.</em> 2014; <strong>23</strong>:2145-2156</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/hmg/ddt610" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-84897835537" target="_blank" aria-label="Open Scopus in new tab">Scopus (41)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24293546/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fhmg%2Fddt610&amp;pmid=24293546" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). One-way ANOVA coupled with Tukey post hoc test: <sup>∗∗∗</sup>p&nbsp;value&nbsp;&lt; 0.001; <sup>∗∗</sup>p value&nbsp;&lt; 0.01.</div><div id="fspara0110" role="paragraph">See also <a id="crosref2335" href="#figs4">Figure&nbsp;S4</a> and <a id="crosref2340" href="#mmc1">Tables S1</a>L and S1M.</div></div></div></div></figcaption></figure></div><div class="figure-wrap"><figure id="figs4" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2022.01.021/asset/2bcda251-c61a-4435-924f-5f90b0205bfa/main.assets/figs4_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs4" src="/cms/10.1016/j.cell.2022.01.021/asset/f68cd285-332a-4a21-8330-7245923cb700/main.assets/figs4.jpg" height="872" width="656" aria-labelledby="figs4-title figs4-content" data-viewer-src="/cms/10.1016/j.cell.2022.01.021/asset/2bcda251-c61a-4435-924f-5f90b0205bfa/main.assets/figs4_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs4-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S4</span> <span class="figure__title__text">Benefit of the GT-PPI approach to identify variants likely to contribute to CHD and protein-damaging effect of the CHD missense DNV in GLYR1, related to <a id="crosref2465" href="#fig4">Figures 4</a> and <a id="crosref2470" href="#fig5">5</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs4-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0380" role="paragraph">(A) Variant prioritization score customized for our trio dataset of coding variants based on a combination of widely used gene and variant features together with proband pedigree information. The indicated annotations were consolidated into a unique score by rank sum and weighted as indicated in the diagram (see <a id="crosref2475" href="#sec-5">STAR Methods</a>: Variant scoring and <a id="crosref2480" href="#figs5">Figure&nbsp;S5</a>A).</div><div id="fspara0385" role="paragraph">(B) Variant prioritization scores for all <i>de novo</i> missense variants from probands found in both interactome (green) and non-interactome (gray) genes plotted against the corresponding genes’ expression percentile rank in the developing heart (E14.5), (<span class="dropBlock reference-citations"><a id="body-ref-sref145-4" href="#" role="doc-biblioref" data-xml-rid="bib145" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref145-4" aria-controls="sref145-4">Zaidi et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref145-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">131.</div><div>Zaidi, S. ∙ Choi, M. ∙ Wakimoto, H. ...</div><div><strong>De novo mutations in histone-modifying genes in congenital heart disease</strong></div><div><em>Nature.</em> 2013; <strong>498</strong>:220-223</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature12141" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_131_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-84879001958" target="_blank" aria-label="Open Scopus in new tab">Scopus (616)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23665959/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature12141&amp;pmid=23665959" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Published mutations with monogenic contribution (blue) or partial contribution (orange) to CHD are included as references. Variant prioritization score’s 75<sup>th</sup> percentile is higher for GT-PPI missense DNVs than for non-interactome variants (NON-GT-PPI) and all unfiltered missense DNVs. Genes within the top quartile of expression in the developing heart are indicated as High Heart expressed (HHE).</div><div id="fspara0390" role="paragraph">(C) Percentage of (All) versus interactome (GT-PPI) missense DNVs (misDNVs) in genes within the top quartile of Developing Heart Expression (High Heart Expressed genes, HHE) and the top quartile of Variant Prioritization Score (VPS) (green), the top quartile of Developing Heart Expression and the top half of VPS (gray), or below the 75th percentile of Developing Heart Expression or in the bottom half of VPS (orange).</div><div id="fspara0395" role="paragraph">(D) Average VPS for all misDNVs and GT-PPI misDNVs within the top quartile of Developing Heart Expression and Variant Prioritization Score. The white line represents the median, the black lines the interquartile range. Unpaired Student’s t test: <sup>∗∗</sup>p value&nbsp;&lt; 0.01.</div><div id="fspara0400" role="paragraph">(E and F) Precision Recall (PR) curves demonstrating the ability of the variant prioritization scoring (VPS) to predict known CHD causing variants among all observed missense DNVs (All misDNVs) or among all observed missense DNVs in the GT-PPI interactome (GT-PPI misDNVs). Analysis using (E) the original VPS or (F) a modified VPS where no re-weighting factor was applied to variants co-occurring with other variants in GT-PPI genes. Only a penalization factor was applied to those variants occurring in patients with other <i>de novo</i> or inherited variants in known CHD genes. The Area-Under the Curve (AUC) estimates for these two situations are provided next to the legend. The expected AUC from a random classifier using data for all observed variants&nbsp;= 113/2155&nbsp;= 0.052, while the corresponding expected AUC using data for variants in the GT-PPI interactome is 18/55&nbsp;= 0.327. The PR curves are generated by varying the threshold applied to the respective VPS. Observed missense variants with VPS greater than a selected threshold are predicted to be CHD-causing ones. At each threshold, Precision refers to the fraction of variants predicted to cause CHD that were known to cause CHD, while Recall refers to the fraction of known CHD causing variants that are predicted as such.</div><div id="fspara0405" role="paragraph">(G–I) The ability of the proteins encoded by three top-scored interactome CHD candidate genes encoded proteins, SMARCC1 (G), GLYR1 (H) and BRD4 (I), to interact with GATA4 as assessed by ectopic expression of their MYC- or HA-tagged WT proteins in HEK293 cells followed by immunoprecipitation (IP) with anti-MYC or anti-HA antibodies. Enriched nuclear lysates prior to IP (Inputs) were set aside and analyzed by immunoblotting with the indicated antibodies in parallel with IP samples to verify similar protein ectopic expression levels across samples.</div><div id="fspara0410" role="paragraph">(J) Immunoprecipitation (IP) for endogenous GATA4 protein and its protein complexes from enriched nuclear lysates of WT and <i>GATA4-KO</i> (G4KO) CPs, followed by immunoblot for indicated antibodies. Aliquots of CP-enriched nuclear lysates were put aside prior to IP (Inputs). IP and Inputs were subsequently subjected to immunoblotting with the indicated antibodies.</div><div id="fspara0415" role="paragraph">(K) Evolution of the root mean square deviation (RMSD) of the structural dynamic frames visited by WT (blue) or GLYR1 P496L (green) beta-DH domains over time, taking the starting protein structure as reference.</div><div id="fspara0420" role="paragraph">(L) The ability of GLYR1 WT or P496L mutant to interact with GATA4 as assessed by ectopic expression in HEK293 cells and immunoprecipitation (IP) of GFP-GATA4 followed by immunoblotting with the indicated antibodies. Enriched nuclear lysates prior to IP (Inputs) were set aside and analyzed in parallel with IP samples to verify similar protein ectopic expression levels across samples.</div><div id="fspara0425" role="paragraph">(M) The ability of GLYR1-WT or P496L mutant to interact with previously described interactors (<span class="dropBlock reference-citations"><a id="body-ref-sref36-1" href="#" role="doc-biblioref" data-xml-rid="bib36" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref36-1" aria-controls="sref36-1">Fang et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref36-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">33.</div><div>Fang, R. ∙ Chen, F. ∙ Dong, Z. ...</div><div><strong>LSD2/KDM1B and its cofactor NPAC/GLYR1 endow a structural and molecular model for regulation of H3K4 demethylation</strong></div><div><em>Mol. Cell.</em> 2013; <strong>49</strong>:558-570</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=10.1016%2Fj.molcel.2012.11.019&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=10.1016%2Fj.molcel.2012.11.019&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-84873420044" target="_blank" aria-label="Open Scopus in new tab">Scopus (64)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23260659/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.molcel.2012.11.019&amp;pmid=23260659" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref143-1" href="#" role="doc-biblioref" data-xml-rid="bib143" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref143-1" aria-controls="sref143-1">Yu et al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref143-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">129.</div><div>Yu, S. ∙ Li, J. ∙ Ji, G. ...</div><div><strong>Npac Is a Co-factor of Histone H3K36me3 and Regulates Transcriptional Elongation in Mouse Embryonic Stem Cells</strong></div><div><em>Genomics Proteomics Bioinformatics.</em> 2021; <strong>3</strong></div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.gpb.2020.08.004" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_129_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-85118281452" target="_blank" aria-label="Open Scopus in new tab">Scopus (2)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.gpb.2020.08.004" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) as assessed by ectopic expression of MYC-tagged GLYR1<sup>WT</sup> or GLYR1<sup>P496L</sup> in HEK293 cells followed by MYC immunoprecipitation (IP) and immunoblotting with the indicated antibodies. Enriched nuclear lysates prior to IP (Inputs) were set aside and analyzed in parallel with IP samples to verify similar protein ectopic expression levels across samples.</div><div id="fspara0430" role="paragraph">(N) Luciferase reporter assay in HeLa cells showing activation of the luciferase reporter upon addition of plasmids encoding indicated proteins. Equal amount of&nbsp;total transfected DNA per condition was adjusted with empty vector. (n&nbsp;= 3 independent experiments). One-way ANOVA coupled with Tukey post hoc test: <sup>∗∗</sup>p&nbsp;value&nbsp;&lt; 0.01, <sup>∗∗∗</sup>p value&nbsp;&lt; 0.001.</div></div></div></div></figcaption></figure></div><div id="p0125" role="paragraph">We applied this scoring method to previously identified variants implicated in CHD (<span class="dropBlock reference-citations"><a id="body-ref-sref9" href="#" role="doc-biblioref" data-xml-rid="bib9" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref9" aria-controls="sref9">Basson et&nbsp;al., 1999</a><div class="dropBlock__holder" data-db-target-of="sref9"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">9.</div><div>Basson, C.T. ∙ Huang, T. ∙ Lin, R.C. ...</div><div><strong>Different TBX5 interactions in heart and limb defined by Holt-Oram syndrome mutations</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 1999; <strong>96</strong>:2919-2924</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.96.6.2919" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-13044287363" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/10077612/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.96.6.2919&amp;pmid=10077612" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref43" href="#" role="doc-biblioref" data-xml-rid="bib43" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref43" aria-controls="sref43">Furtado et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref43"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">39.</div><div>Furtado, M.B. ∙ Wilmanns, J.C. ∙ Chandran, A. ...</div><div><strong>Point mutations in murine <i>Nkx2-5</i> phenocopy human congenital heart disease and induce pathogenic Wnt signaling</strong></div><div><em>JCI Insight.</em> 2017; <strong>2</strong>:e88271</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1172/jci.insight.88271" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-85054931220" target="_blank" aria-label="Open Scopus in new tab">Scopus (14)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28352650/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1172%2Fjci.insight.88271&amp;pmid=28352650" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref44-3" href="#" role="doc-biblioref" data-xml-rid="bib44" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref44-3" aria-controls="sref44-3">Garg et&nbsp;al., 2003</a><div class="dropBlock__holder" data-db-target-of="sref44-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">40.</div><div>Garg, V. ∙ Kathiriya, I.S. ∙ Barnes, R. ...</div><div><strong>GATA4 mutations cause human congenital heart defects and reveal an interaction with TBX5</strong></div><div><em>Nature.</em> 2003; <strong>424</strong>:443-447</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature01827" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_40_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-0043267988" target="_blank" aria-label="Open Scopus in new tab">Scopus (953)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/12845333/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature01827&amp;pmid=12845333" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and found that the method ranked these reference monogenic variants more highly than the few mutations demonstrated to partially contribute to CHD and cause oligogenic disease (<a id="crosref0955" href="#fig4">Figure&nbsp;4</a>B) (<span class="dropBlock reference-citations"><a id="body-ref-sref45-2" href="#" role="doc-biblioref" data-xml-rid="bib45" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref45-2" aria-controls="sref45-2">Gifford et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref45-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">41.</div><div>Gifford, C.A. ∙ Ranade, S.S. ∙ Samarakoon, R. ...</div><div><strong>Oligogenic inheritance of a human heart disease involving a genetic modifier</strong></div><div><em>Science.</em> 2019; <strong>364</strong>:865-870</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aat5056" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-85066940786" target="_blank" aria-label="Open Scopus in new tab">Scopus (84)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31147515/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aat5056&amp;pmid=31147515" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), even when the mutations affected the same gene. Furthermore, among the top-scoring interactome variants, there were several in proteins known to cause cardiac malformations, consistent with the relevance of this score for identifying gene variants with potential for contributing to disease (<a id="crosref0965" href="#fig4">Figure&nbsp;4</a>B; <a id="crosref0970" href="#mmc1">Table S1</a>L).</div><div id="p0130" role="paragraph">In order to test whether higher variant prioritization scores indeed translated to greater functional impact of variants, we evaluated the effect of multiple variants on cofactor activity in a luciferase reporter assay using a luciferase reporter containing the <i>PPARGC1a</i> promoter, which is strongly activated by GATA4 (<span class="dropBlock reference-citations"><a id="body-ref-sref103-2" href="#" role="doc-biblioref" data-xml-rid="bib103" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref103-2" aria-controls="sref103-2">Padmanabhan et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref103-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">91.</div><div>Padmanabhan, A. ∙ Alexanian, M. ∙ Linares-Saldana, R. ...</div><div><strong>BRD4 (Bromodomain-Containing Protein 4) Interacts with GATA4 (GATA Binding Protein 4) to Govern Mitochondrial Homeostasis in Adult Cardiomyocytes</strong></div><div><em>Circulation.</em> 2020; <strong>142</strong>:2338-2355</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1161/CIRCULATIONAHA.120.047753" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-85097828347" target="_blank" aria-label="Open Scopus in new tab">Scopus (14)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33094644/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1161%2FCIRCULATIONAHA.120.047753&amp;pmid=33094644" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). We selected <i>NKX2-5</i>, a reference gene with one high and one low scored variant; <i>CHD7</i>, a CHD gene encoding a GATA4 interactor with four identified missense DNVs; and <i>BRD4</i> and <i>SMARCC1</i>, CHD candidate genes and GATA4 interactors, each with two identified missense DNVs in CHD patients. For the GATA4 interactors—CHD7, SMARCC1, and BRD4—each variant’s impact on transcriptional activity was tested in the presence of GATA4. We found that variants with a higher prioritization score exerted a greater effect on the encoded protein’s transcriptional activity (<a id="crosref0980" href="#fig4">Figure&nbsp;4</a>C).</div><div id="p0135" role="paragraph">Next, we aimed to evaluate the benefit of the GATA4 and TBX5 PPI incorporation as a filtering strategy for the identification of CHD candidate genes. To test this, we applied the variant prioritization scoring to all <i>de novo</i> missense variants from probands found in both interactome and non-interactome genes (<a id="crosref0985" href="#mmc1">Table S1</a>M). We found that the variant prioritization score’s 75<sup>th</sup> percentile was 23 or 22 points higher (score range 0–99) for GT-PPI missense DNVs than for variants in genes outside the GT-PPI network or all unfiltered missense DNVs, respectively (<a id="crosref0990" href="#figs4">Figure&nbsp;S4</a>B). Moreover, 41.5% of interactome missense DNVs ranked within the top quartile of all DNV prioritization scores, and within the top quartile of Developing Heart Expression percentile (<span class="dropBlock reference-citations"><a id="body-ref-sref145-5" href="#" role="doc-biblioref" data-xml-rid="bib145" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref145-5" aria-controls="sref145-5">Zaidi et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref145-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">131.</div><div>Zaidi, S. ∙ Choi, M. ∙ Wakimoto, H. ...</div><div><strong>De novo mutations in histone-modifying genes in congenital heart disease</strong></div><div><em>Nature.</em> 2013; <strong>498</strong>:220-223</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature12141" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_131_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-84879001958" target="_blank" aria-label="Open Scopus in new tab">Scopus (616)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23665959/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature12141&amp;pmid=23665959" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), compared to just 12.4% of unfiltered missense DNVs (<a id="crosref1000" href="#figs4">Figure&nbsp;S4</a>C). Additionally, among the missense DNVs within the top quartile, the average prioritization score was significantly higher for GT-PPI variants than for the unfiltered missense DNVs (<a id="crosref1005" href="#figs4">Figure&nbsp;S4</a>D). In agreement, the proportion of missense DNVs in known CHD-causing genes that belonged to the GT-PPI network (∼30% GT-PPI missense DNVs) was an order of magnitude higher than the corresponding proportion across all observed missense DNVs (∼3% of all missense DNVs) observed in CHD probands. To most-directly test the value of filtering the DNVs with the GT-PPI, we performed a Precision-Recall analysis that provides a performance metric, area under the curve (AUC), estimated for the variant prioritization scoring with or without incorporating information from the GT-PPI. This approach demonstrated that incorporation of the GT-PPI variant-filtering strategy improved the ability of the variant prioritization scoring to predict known CHD-causing variants (GT-PPI filtered variants using VPS, AUC&nbsp;= 0.61 versus Random Classifier, AUC&nbsp;= 0.3; all variants using VPS, AUC&nbsp;= 0.18 versus Random Classifier, AUC&nbsp;= 0.05); furthermore, this outcome was not driven by a confounding effect of GT-PPI-dependent parameters included in the variant prioritization scoring (VPS) re-weighting step, as a modified scoring excluding this feature performed similarly (<a id="crosref1010" href="#figs4">Figures S4</a>E and S4F).</div><div id="p0140" role="paragraph">Among the CHD candidate missense DNVs, the majority affected interactome proteins highly expressed in the developing heart, with only 25% occurring in GT interactors outside the top quartile of expression (<a id="crosref1015" href="#fig4">Figure&nbsp;4</a>B). The genes with lower heart expression generally also exhibited low variant prioritization scores, except for the tuberous sclerosis gene, <i>TSC1</i>, which is associated with cardiac rhabdomyomas (<a id="crosref1020" href="#fig4">Figure&nbsp;4</a>B) (<span class="dropBlock reference-citations"><a id="body-ref-sref55" href="#" role="doc-biblioref" data-xml-rid="bib55" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref55" aria-controls="sref55">Hinton et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref55"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">50.</div><div>Hinton, R.B. ∙ Prakash, A. ∙ Romp, R.L. ..., International Tuberous Sclerosis Consensus Group</div><div><strong>Cardiovascular manifestations of tuberous sclerosis complex and summary of the revised diagnostic criteria and surveillance and management recommendations from the International Tuberous Sclerosis Consensus Group</strong></div><div><em>J.&nbsp;Am. Heart Assoc.</em> 2014; <strong>3</strong>:e001493</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1161/JAHA.114.001493" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-84935004324" target="_blank" aria-label="Open Scopus in new tab">Scopus (75)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25424575/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1161%2FJAHA.114.001493&amp;pmid=25424575" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). On the other hand, missense DNVs in GT interactors highly expressed in the developing heart exhibited a broad range of prioritization scores, with a potentially highly pathogenic cluster of variants ranking close to the published reference variants with known strong contribution to CHD, and a more disperse group of variants scoring similarly to the few reference variants with known partial contribution to CHD. Among the missense DNVs with the highest scores, which we hypothesized to be more significant contributors, there were four variants in GT interactors with previously described monogenic contribution to human cardiac defects (<i>TBX5</i>, <i>GATA6</i>, <i>CHD4</i>, and <i>CHD7</i>), and six variants within proteins with yet undescribed functions in human congenital heart malformations (<i>BRD4 x2</i>, <i>SMARCC1</i>, <i>GLYR1</i>, <i>CSNK2A1</i>, <i>and SAP18</i>) (<a id="crosref1030" href="#fig4">Figure&nbsp;4</a>B; <a id="crosref1035" href="#mmc1">Table S1</a>L).</div><div id="p0145" role="paragraph">BRD4, GLYR1, and SMARCC1 are chromatin modifiers, in concordance with the observed enrichment of CHD-associated DNVs in genes involved in this process (<span class="dropBlock reference-citations"><a id="body-ref-sref145-6" href="#" role="doc-biblioref" data-xml-rid="bib145" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref145-6" aria-controls="sref145-6">Zaidi et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref145-6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">131.</div><div>Zaidi, S. ∙ Choi, M. ∙ Wakimoto, H. ...</div><div><strong>De novo mutations in histone-modifying genes in congenital heart disease</strong></div><div><em>Nature.</em> 2013; <strong>498</strong>:220-223</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature12141" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_131_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-84879001958" target="_blank" aria-label="Open Scopus in new tab">Scopus (616)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23665959/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature12141&amp;pmid=23665959" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). These CHD candidate genes were detected as GATA4 interactors, which was validated by co-immunoprecipitation (<a id="crosref1045" href="#figs4">Figures S4</a>G–S4J). While GLYR1 and SMARCC1 were previously unknown to interact with GATA4, we recently reported a role for a BRD4-GATA4 protein module in the regulation of cardiac mitochondrial homeostasis and showed that deletion of <i>BRD4</i> during embryonic development (<i>Tnnt2-Cre</i>; <i>Brd4</i><sup><i>flox/flox</i></sup>) resulted in embryonic lethality with signs of cardiac dysfunction (<span class="dropBlock reference-citations"><a id="body-ref-sref103-3" href="#" role="doc-biblioref" data-xml-rid="bib103" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref103-3" aria-controls="sref103-3">Padmanabhan et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref103-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">91.</div><div>Padmanabhan, A. ∙ Alexanian, M. ∙ Linares-Saldana, R. ...</div><div><strong>BRD4 (Bromodomain-Containing Protein 4) Interacts with GATA4 (GATA Binding Protein 4) to Govern Mitochondrial Homeostasis in Adult Cardiomyocytes</strong></div><div><em>Circulation.</em> 2020; <strong>142</strong>:2338-2355</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1161/CIRCULATIONAHA.120.047753" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-85097828347" target="_blank" aria-label="Open Scopus in new tab">Scopus (14)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33094644/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1161%2FCIRCULATIONAHA.120.047753&amp;pmid=33094644" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Although the specific contribution of <i>SMARCC1</i> to CHD is yet uncertain, its encoded protein, BAF155, is a component of the BAF complex, which orchestrates many aspects of heart development (<span class="dropBlock reference-citations"><a id="body-ref-sref57" href="#" role="doc-biblioref" data-xml-rid="bib57" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref57" aria-controls="sref57">Hota and Bruneau, 2016</a><div class="dropBlock__holder" data-db-target-of="sref57"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">52.</div><div>Hota, S.K. ∙ Bruneau, B.G.</div><div><strong>ATP-dependent chromatin remodeling during mammalian development</strong></div><div><em>Development.</em> 2016; <strong>143</strong>:2882-2897</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1242/dev.128892" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-84983085494" target="_blank" aria-label="Open Scopus in new tab">Scopus (123)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27531948/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1242%2Fdev.128892&amp;pmid=27531948" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The <i>GLYR1</i> DNV occurred in a patient with atrioventricular septal defects, left ventricle outflow tract obstruction, and pulmonary stenosis, a spectrum of cardiac malformations observed in humans with <i>GATA4</i> mutations. However, the role of GLYR1 in most tissues, including the heart, remains unexplored.</div><div id="p0150" role="paragraph">We therefore investigated the genetic landscape of the <i>de novo GLYR1</i> variant carrier and identified three LoF and 62 missense variants inherited from their asymptomatic parents, while no other DNVs were found in this proband (<a id="crosref1060" href="#mmc1">Table S1</a>N). Interestingly, one of these inherited missense variants occurred in <i>GATA6</i>, encoding a GATA factor that genetically interacts, and is partially redundant, with GATA4 in cardiac development (<span class="dropBlock reference-citations"><a id="body-ref-sref141" href="#" role="doc-biblioref" data-xml-rid="bib141" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref141" aria-controls="sref141">Xin et&nbsp;al., 2006</a><div class="dropBlock__holder" data-db-target-of="sref141"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">127.</div><div>Xin, M. ∙ Davis, C.A. ∙ Molkentin, J.D. ...</div><div><strong>A threshold of GATA4 and GATA6 expression is required for cardiovascular development</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2006; <strong>103</strong>:11189-11194</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.0604604103" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_127_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-33746630840" target="_blank" aria-label="Open Scopus in new tab">Scopus (141)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16847256/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.0604604103&amp;pmid=16847256" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Although these inherited LoF and missense variants are present in the asymptomatic parents and, therefore, are unlikely to be sufficient to cause cardiac malformations, future studies may assess if any could contribute together with GLYR1 to the cardiac malformations observed in this patient.</div></section><section id="sec-2-5"><h3>The CHD variant in <i>GLYR1</i> impacts structural dynamics and destabilizes its physical interaction with GATA4</h3><div id="p0155" role="paragraph">GLYR1, also known as NDF, NPAC, or NP60, is a chromatin reader involved in chromatin modification and regulation of gene expression through nucleosome demethylation (<span class="dropBlock reference-citations"><a id="body-ref-sref36-2" href="#" role="doc-biblioref" data-xml-rid="bib36" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref36-2" aria-controls="sref36-2">Fang et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref36-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">33.</div><div>Fang, R. ∙ Chen, F. ∙ Dong, Z. ...</div><div><strong>LSD2/KDM1B and its cofactor NPAC/GLYR1 endow a structural and molecular model for regulation of H3K4 demethylation</strong></div><div><em>Mol. Cell.</em> 2013; <strong>49</strong>:558-570</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=10.1016%2Fj.molcel.2012.11.019&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=10.1016%2Fj.molcel.2012.11.019&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-84873420044" target="_blank" aria-label="Open Scopus in new tab">Scopus (64)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23260659/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.molcel.2012.11.019&amp;pmid=23260659" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref38-1" href="#" role="doc-biblioref" data-xml-rid="bib38" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref38-1" aria-controls="sref38-1">Fei et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref38-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">35.</div><div>Fei, J. ∙ Ishii, H. ∙ Hoeksema, M.A. ...</div><div><strong>NDF, a nucleosome-destabilizing factor that facilitates transcription through nucleosomes</strong></div><div><em>Genes Dev.</em> 2018; <strong>32</strong>:682-694</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/gad.313973.118" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-85047604949" target="_blank" aria-label="Open Scopus in new tab">Scopus (24)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29759984/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2Fgad.313973.118&amp;pmid=29759984" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref41" href="#" role="doc-biblioref" data-xml-rid="bib41" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref41" aria-controls="sref41">Fu et&nbsp;al., 2006</a><div class="dropBlock__holder" data-db-target-of="sref41"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">37.</div><div>Fu, J. ∙ Yang, Z. ∙ Wei, J. ...</div><div><strong>Nuclear protein NP60 regulates p38 MAPK activity</strong></div><div><em>J.&nbsp;Cell Sci.</em> 2006; <strong>119</strong>:115-123</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1242/jcs.02699" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-31644442352" target="_blank" aria-label="Open Scopus in new tab">Scopus (13)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16352664/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1242%2Fjcs.02699&amp;pmid=16352664" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref93-1" href="#" role="doc-biblioref" data-xml-rid="bib93" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref93-1" aria-controls="sref93-1">Marabelli et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref93-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">81.</div><div>Marabelli, C. ∙ Marrocco, B. ∙ Pilotto, S. ...</div><div><strong>A Tail-Based Mechanism Drives Nucleosome Demethylation by the LSD2/NPAC Multimeric Complex</strong></div><div><em>Cell Rep.</em> 2019; <strong>27</strong>:387-399.e7</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=10.1016%2Fj.celrep.2019.03.061&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=10.1016%2Fj.celrep.2019.03.061&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-85063864248" target="_blank" aria-label="Open Scopus in new tab">Scopus (12)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30970244/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.celrep.2019.03.061&amp;pmid=30970244" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref143-2" href="#" role="doc-biblioref" data-xml-rid="bib143" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref143-2" aria-controls="sref143-2">Yu et al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref143-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">129.</div><div>Yu, S. ∙ Li, J. ∙ Ji, G. ...</div><div><strong>Npac Is a Co-factor of Histone H3K36me3 and Regulates Transcriptional Elongation in Mouse Embryonic Stem Cells</strong></div><div><em>Genomics Proteomics Bioinformatics.</em> 2021; <strong>3</strong></div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.gpb.2020.08.004" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_129_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-85118281452" target="_blank" aria-label="Open Scopus in new tab">Scopus (2)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.gpb.2020.08.004" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The <i>GLYR1</i> missense CHD DNV we detected involved the substitution of a highly conserved proline with a leucine at amino acid (aa) 496 within the β-hydroxyacid dehydrogenase (β-HAD) domain, described to mediate the interaction between GLYR1 monomers (<span class="dropBlock reference-citations"><a id="body-ref-sref93-2" href="#" role="doc-biblioref" data-xml-rid="bib93" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref93-2" aria-controls="sref93-2">Marabelli et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref93-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">81.</div><div>Marabelli, C. ∙ Marrocco, B. ∙ Pilotto, S. ...</div><div><strong>A Tail-Based Mechanism Drives Nucleosome Demethylation by the LSD2/NPAC Multimeric Complex</strong></div><div><em>Cell Rep.</em> 2019; <strong>27</strong>:387-399.e7</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=10.1016%2Fj.celrep.2019.03.061&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=10.1016%2Fj.celrep.2019.03.061&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-85063864248" target="_blank" aria-label="Open Scopus in new tab">Scopus (12)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30970244/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.celrep.2019.03.061&amp;pmid=30970244" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref97" href="#" role="doc-biblioref" data-xml-rid="bib97" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref97" aria-controls="sref97">Montefiori et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref97"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">85.</div><div>Montefiori, M. ∙ Pilotto, S. ∙ Marabelli, C. ...</div><div><strong>Impact of Mutations on NPAC Structural Dynamics: Mechanistic Insights from MD Simulations</strong></div><div><em>J.&nbsp;Chem. Inf. Model.</em> 2019; <strong>59</strong>:3927-3937</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/acs.jcim.9b00588" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-85071873088" target="_blank" aria-label="Open Scopus in new tab">Scopus (19)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31408337/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Facs.jcim.9b00588&amp;pmid=31408337" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Since Proline<sup>496</sup> is located within a rigid loop enriched in aromatic residues connecting two tetramerization domains (<a id="crosref1105" href="#fig5">Figures 5</a>A–5C), we hypothesized that its substitution by a leucine would impact the structural dynamics of the GLYR1 β-HAD domain and therefore its ability to acquire certain functional states.</div><div class="figure-wrap"><figure id="fig5" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2022.01.021/asset/12063954-16ad-467f-bb5d-6e9c8e169f48/main.assets/gr5_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr5" src="/cms/10.1016/j.cell.2022.01.021/asset/f4ab8d64-bad4-45c6-bffd-e1918f0cc8b8/main.assets/gr5.jpg" height="711" width="659" aria-labelledby="fig5-title fig5-content" data-viewer-src="/cms/10.1016/j.cell.2022.01.021/asset/12063954-16ad-467f-bb5d-6e9c8e169f48/main.assets/gr5_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig5-title" class="figure__title"><span class="label figure__label">Figure&nbsp;5</span> <span class="figure__title__text">Functional impact of a highly scored CHD variant in <i>GLYR1</i></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig5-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0120" role="paragraph">(A) Simplified protein schematic depicting the domain organization of human GLYR1. Black rectangle indicates zoomed-in protein region in (B).</div><div id="fspara0125" role="paragraph">(B) Protein sequence conservation across vertebrate species for the GLYR1 rigid loop region containing the CHD-associated P496L DNV.</div><div id="fspara0130" role="paragraph">(C) GLYR1 dehydrogenase domains: Rossman-fold globular domain (green), the linking α9-helix (red), and the α-helical bundle (dark blue). Right panels: zoom into the WT and mutant forms of the rigid loop with aromatic residues in beige and Proline 496 in orange.</div><div id="fspara0135" role="paragraph">(D) Distribution of the root mean square deviation (RMSD) of frames visited during the trajectories from the reference state represented by the starting structure of the WT (blue) and the P496L mutant (green) GLYR1 dehydrogenase domains within the measured time.</div><div id="fspara0140" role="paragraph">(E) Residue flexibility analysis based on the standard deviations of the atomic positions in the simulations (RMSF) after fitting to the starting structure of the WT form (blue) and the mutant (green) GLYR1 dehydrogenase domains. F-statistic shows lower flexibility of the mutant compared to the Rossman-fold domain (residues 262–437).</div><div id="fspara0145" role="paragraph">(F) The ability of GLYR1 WT or P496L mutant to interact with GATA4 by immunoprecipitation (IP) of GLYR1-MYC and immunoblotting with indicated antibodies.</div><div id="fspara0150" role="paragraph">(G) Luciferase reporter assay in HeLa cells showing activation of the GATA4-dependent <i>Nppa</i> luciferase reporter upon addition of plasmids encoding indicated proteins. (n&nbsp;= 3 independent experiments). One-way ANOVA coupled with Tukey post hoc test: <sup>∗∗∗</sup>p value&nbsp;&lt; 0.001.</div><div id="fspara0155" role="paragraph">See also <a id="crosref2345" href="#figs4">Figure&nbsp;S4</a> and <a id="crosref2350" href="#mmc1">Table S1</a>N.</div></div></div></div></figcaption></figure></div><div id="p0160" role="paragraph">Molecular dynamics (MD) computational simulations predicted that the mutant (GLYR1<sup>P496L</sup>) β-HAD would explore a narrower set of structural conformations than the WT, as shown by the time-dependent evolution of the root mean square deviation (RMSD) of frames visited during the trajectories from the reference structure (<a id="crosref1110" href="#figs4">Figure&nbsp;S4</a>K). This result was confirmed by the distribution of the RMSD calculated for every pair of states sampled during the simulations (<a id="crosref1115" href="#fig5">Figure&nbsp;5</a>D). Furthermore, GLYR1 structural dynamics at the local level, measured by the standard deviations of the atomic positions in the simulations (RMSF), indicated an overall lower flexibility of GLYR1<sup>P496L</sup> compared to the WT protein, which was evident in the Rossman-fold domain (262-437 aa) (<a id="crosref1120" href="#fig5">Figure&nbsp;5</a>E). These data indicated that the P496L variant in GLYR1 induces significant differences in the structural dynamics of the β-HAD domain, at the global and local levels, predicting a general increase in the structural rigidity of this region in GLYR1.</div><div id="p0165" role="paragraph">Increased rigidity within the β-HAD domain could affect GLYR1’s capacity to adapt to interacting partner proteins through conformational selection. Co-immunoprecipitation assays demonstrated that the <i>GLYR1</i> P496L DNV destabilized its physical interaction with GATA4 (<a id="crosref1125" href="#fig5">Figures 5</a>F and <a id="crosref1130" href="#figs4">S4</a>L) but not with previously described GLYR1 interactors LSD2, CDK9, or Cyclin T1 (<a id="crosref1135" href="#figs4">Figure&nbsp;S4</a>M) (<span class="dropBlock reference-citations"><a id="body-ref-sref36-3" href="#" role="doc-biblioref" data-xml-rid="bib36" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref36-3" aria-controls="sref36-3">Fang et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref36-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">33.</div><div>Fang, R. ∙ Chen, F. ∙ Dong, Z. ...</div><div><strong>LSD2/KDM1B and its cofactor NPAC/GLYR1 endow a structural and molecular model for regulation of H3K4 demethylation</strong></div><div><em>Mol. Cell.</em> 2013; <strong>49</strong>:558-570</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=10.1016%2Fj.molcel.2012.11.019&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=10.1016%2Fj.molcel.2012.11.019&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-84873420044" target="_blank" aria-label="Open Scopus in new tab">Scopus (64)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23260659/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.molcel.2012.11.019&amp;pmid=23260659" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref143-3" href="#" role="doc-biblioref" data-xml-rid="bib143" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref143-3" aria-controls="sref143-3">Yu et al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref143-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">129.</div><div>Yu, S. ∙ Li, J. ∙ Ji, G. ...</div><div><strong>Npac Is a Co-factor of Histone H3K36me3 and Regulates Transcriptional Elongation in Mouse Embryonic Stem Cells</strong></div><div><em>Genomics Proteomics Bioinformatics.</em> 2021; <strong>3</strong></div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.gpb.2020.08.004" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_129_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-85118281452" target="_blank" aria-label="Open Scopus in new tab">Scopus (2)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.gpb.2020.08.004" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Since previous studies indicated a role for GLYR1 in transcriptional regulation (<span class="dropBlock reference-citations"><a id="body-ref-sref38-2" href="#" role="doc-biblioref" data-xml-rid="bib38" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref38-2" aria-controls="sref38-2">Fei et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref38-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">35.</div><div>Fei, J. ∙ Ishii, H. ∙ Hoeksema, M.A. ...</div><div><strong>NDF, a nucleosome-destabilizing factor that facilitates transcription through nucleosomes</strong></div><div><em>Genes Dev.</em> 2018; <strong>32</strong>:682-694</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/gad.313973.118" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-85047604949" target="_blank" aria-label="Open Scopus in new tab">Scopus (24)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29759984/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2Fgad.313973.118&amp;pmid=29759984" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref143-4" href="#" role="doc-biblioref" data-xml-rid="bib143" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref143-4" aria-controls="sref143-4">Yu et al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref143-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">129.</div><div>Yu, S. ∙ Li, J. ∙ Ji, G. ...</div><div><strong>Npac Is a Co-factor of Histone H3K36me3 and Regulates Transcriptional Elongation in Mouse Embryonic Stem Cells</strong></div><div><em>Genomics Proteomics Bioinformatics.</em> 2021; <strong>3</strong></div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.gpb.2020.08.004" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_129_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-85118281452" target="_blank" aria-label="Open Scopus in new tab">Scopus (2)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.gpb.2020.08.004" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), we probed whether GLYR1 co-regulates gene expression together with GATA4 and found that co-transfection of GATA4 and GLYR1 increased <i>Nppa</i>-luciferase reporter activity by approximately 15-fold, compared with an 8-fold activation with GATA4 alone. Synergistic transactivation of the <i>Ccnd2</i>-luciferase reporter by GLYR1 and GATA4 was similarly observed and in both cases was attenuated by the GLYR1 P496L mutation (<a id="crosref1160" href="#fig5">Figures 5</a>G and <a id="crosref1165" href="#figs4">S4</a>N).</div></section><section id="sec-2-6"><h3>GATA4 and GLYR1 co-bind a defined set of heart development genes and co-regulate their expression</h3><div id="p0170" role="paragraph">GLYR1 localizes within chromatin regions rich in histone H3 trimethylated on Lys36 (H3K36me3) at actively transcribed gene bodies to regulate transcription elongation (<span class="dropBlock reference-citations"><a id="body-ref-sref36-4" href="#" role="doc-biblioref" data-xml-rid="bib36" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref36-4" aria-controls="sref36-4">Fang et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref36-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">33.</div><div>Fang, R. ∙ Chen, F. ∙ Dong, Z. ...</div><div><strong>LSD2/KDM1B and its cofactor NPAC/GLYR1 endow a structural and molecular model for regulation of H3K4 demethylation</strong></div><div><em>Mol. Cell.</em> 2013; <strong>49</strong>:558-570</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=10.1016%2Fj.molcel.2012.11.019&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=10.1016%2Fj.molcel.2012.11.019&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-84873420044" target="_blank" aria-label="Open Scopus in new tab">Scopus (64)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23260659/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.molcel.2012.11.019&amp;pmid=23260659" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref38-3" href="#" role="doc-biblioref" data-xml-rid="bib38" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref38-3" aria-controls="sref38-3">Fei et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref38-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">35.</div><div>Fei, J. ∙ Ishii, H. ∙ Hoeksema, M.A. ...</div><div><strong>NDF, a nucleosome-destabilizing factor that facilitates transcription through nucleosomes</strong></div><div><em>Genes Dev.</em> 2018; <strong>32</strong>:682-694</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/gad.313973.118" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-85047604949" target="_blank" aria-label="Open Scopus in new tab">Scopus (24)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29759984/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2Fgad.313973.118&amp;pmid=29759984" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref93-3" href="#" role="doc-biblioref" data-xml-rid="bib93" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref93-3" aria-controls="sref93-3">Marabelli et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref93-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">81.</div><div>Marabelli, C. ∙ Marrocco, B. ∙ Pilotto, S. ...</div><div><strong>A Tail-Based Mechanism Drives Nucleosome Demethylation by the LSD2/NPAC Multimeric Complex</strong></div><div><em>Cell Rep.</em> 2019; <strong>27</strong>:387-399.e7</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=10.1016%2Fj.celrep.2019.03.061&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=10.1016%2Fj.celrep.2019.03.061&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-85063864248" target="_blank" aria-label="Open Scopus in new tab">Scopus (12)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30970244/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.celrep.2019.03.061&amp;pmid=30970244" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref143-5" href="#" role="doc-biblioref" data-xml-rid="bib143" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref143-5" aria-controls="sref143-5">Yu et al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref143-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">129.</div><div>Yu, S. ∙ Li, J. ∙ Ji, G. ...</div><div><strong>Npac Is a Co-factor of Histone H3K36me3 and Regulates Transcriptional Elongation in Mouse Embryonic Stem Cells</strong></div><div><em>Genomics Proteomics Bioinformatics.</em> 2021; <strong>3</strong></div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.gpb.2020.08.004" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_129_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-85118281452" target="_blank" aria-label="Open Scopus in new tab">Scopus (2)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.gpb.2020.08.004" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). However, knowledge about how GLYR1 is recruited to specific loci or its function in homeostasis and disease is limited. Analysis of gene expression along with GLYR1 genome-wide occupancy during CM differentiation together with H3K36me3 genome-wide distribution by chromatin immunoprecipitation sequencing (ChIP-seq) in hiPSCs and CPs revealed dynamic relocalization of GLYR1 during differentiation of hiPSCs to CPs, with ∼7,400 differentially bound genes (FDR&nbsp;&lt; 0.1) between the two stages (<a id="crosref1190" href="#fig6">Figures 6</a>A and <a id="crosref1195" href="#figs5">S5</a>A). K-means clustering of genes differentially bound by GLYR1 based on the three measured variables—GLYR1 ChIP-seq, H3K36me3 ChIP-seq, and RNA expression—highlighted GLYR1 recruitment to 4,246 gene bodies upon differentiation of hiPSCs to CPs (clusters 2 and 3). Gene ontology (GO) analysis revealed that gene programs associated with heart development were enriched in cluster 2, which showed the highest levels of GLYR1 ChIP signal in CPs, whereas cluster 3 was enriched for genes involved in general cellular processes. On the other hand, cluster 1 contained ∼3,155 GLYR1-bound genes in hiPSCs that were lost in CPs, mainly associated with cell cycle and ribosome biogenesis terms (<a id="crosref1200" href="#fig6">Figure&nbsp;6</a>A; <a id="crosref1205" href="#mmc1">Tables S1</a>O and S1P). Overall, GLYR1 preferentially bound to transcribed regions of active genes and co-localized with H3K36me3 (<a id="crosref1210" href="#fig6">Figures 6</a>A and <a id="crosref1215" href="#figs5">S5</a>A–S5C). Interestingly, GLYR1 occupied ∼50% of the genes upregulated in CPs and marked with H3K36me3 (<a id="crosref1220" href="#figs5">Figures S5</a>A and S5B), suggesting that GLYR1 is recruited to a large set of cardiac genes during CM differentiation.</div><div class="figure-wrap"><figure id="fig6" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2022.01.021/asset/707cff8f-db79-41b5-97cf-d2d599dee5f1/main.assets/gr6_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr6" src="/cms/10.1016/j.cell.2022.01.021/asset/6bcddf09-7630-4aa7-b219-9505156bcc7f/main.assets/gr6.jpg" height="825" width="656" aria-labelledby="fig6-title fig6-content" data-viewer-src="/cms/10.1016/j.cell.2022.01.021/asset/707cff8f-db79-41b5-97cf-d2d599dee5f1/main.assets/gr6_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig6-title" class="figure__title"><span class="label figure__label">Figure&nbsp;6</span> <span class="figure__title__text">GATA4-associated roles for GLYR1 in transcription regulation during cardiomyocyte differentiation</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig6-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0165" role="paragraph">(A) Heatmap of genes differentially bound by GLYR1 (FDR&nbsp;&lt; 0.1) between hiPSCs and CPs subjected to k-means clustering based on GLYR1 ChIP-seq signal (3 representative replicates plotted; n&nbsp;= 5), H3K36me3 ChIP-seq signal (n&nbsp;= 2), and gene expression levels (GSE137920; n&nbsp;= 3). Statistically enriched GO Biological Process terms and example genes per cluster on the right panel.</div><div id="fspara0170" role="paragraph">(B) Overlap of genes bound by GLYR1 in CPs from clusters 2 and 3 (FDR&nbsp;&lt; 0.1, LogFC &gt; 0.5) with genes occupied by GATA4 within the gene body (1<sup>st</sup> intron-TES). The odds of GATA4 binding to gene bodies enriched GLYR1 signal versus no GATA4 binding is 2.38 (<sup>∗∗∗</sup>p value&nbsp;&lt; 2.2e<sup>−16</sup>, Fisher’s exact test).</div><div id="fspara0175" role="paragraph">(C) Gene Ontology enrichment analysis of biological process for genes up- or downregulated in CPs compared to hiPSCs (FDR&nbsp;&lt; 0.05) and bound by GATA4:GLYR1, GLYR1-only, and GATA4-only. Prot., protein; dev., development.</div><div id="fspara0180" role="paragraph">(D) Heatmap of GATA4:GLYR1 co-bound genes differentially expressed (FDR&nbsp;&lt; 0.05) upon independent knockdown of GATA4 or GLYR1 at the CP stage by RNA-seq.</div><div id="fspara0185" role="paragraph">(E) Metagene plots for GATA4:GLYR1 co-bound genes plotting the normalized ChIP-seq signal for the indicated histone marks (publicly available data GSE85631 and GSM2047027), and other cardiac transcription factors centered on GATA4 peaks within the gene body (1<sup>st</sup> Intron-TES). One representative replicate plotted.</div><div id="fspara0190" role="paragraph">(F) Transcriptional activity of three putative intronic regulatory elements co-bound by GATA4 and GLYR1 in the presence of indicated regulatory proteins. One-way ANOVA coupled with Tukey post hoc test (n&nbsp;= 3): <sup>∗∗∗</sup>p value&nbsp;&lt; 0.001.</div><div id="fspara0195" role="paragraph">See also <a id="crosref2355" href="#figs5">Figure&nbsp;S5</a> and <a id="crosref2360" href="#mmc1">Tables S1</a>O–S1T.</div></div></div></div></figcaption></figure></div><div class="figure-wrap"><figure id="figs5" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2022.01.021/asset/643a7477-dbcd-4afa-9888-484617ced8da/main.assets/figs5_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs5" src="/cms/10.1016/j.cell.2022.01.021/asset/6bbed26a-105a-4539-b8f8-b1051541272e/main.assets/figs5.jpg" height="832" width="656" aria-labelledby="figs5-title figs5-content" data-viewer-src="/cms/10.1016/j.cell.2022.01.021/asset/643a7477-dbcd-4afa-9888-484617ced8da/main.assets/figs5_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs5-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S5</span> <span class="figure__title__text">GLYR1 genome-wide occupancy and transcriptional regulation during cardiomyocyte differentiation, related to <a id="crosref2500" href="#fig6">Figure&nbsp;6</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs5-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0440" role="paragraph">(A) Scatterplots showing the correlations between indicated ChIPseq signals (log 2 RPKM) at the indicated CP or hiPSC stages for genes classified as not differentially expressed (Not DE genes; light gray), upregulated (Up-reg genes; red) and downregulated (Down-reg genes; dark gray) based on publicly available hiPSCs versus CPs RNaseq data (GSE137920). Dotted lines represent y&nbsp;= x line. ChIPseq GLYR1 hiPSC, H3K36me3 hiPSCs and CPs n&nbsp;= 2; GLYR1 CPs ChIPseq n&nbsp;= 3.</div><div id="fspara0445" role="paragraph">(B) Ven diagram for genes upregulated in CPs versus hiPSCs by RNaseq (GSE137920) (FDR&nbsp;&lt; 0.05 &amp; LogFC &gt; 0.5, n&nbsp;= 3) and which gained H3K36me3 (n&nbsp;= 2) or GLYR1 ChIP seq signal (n&nbsp;= 5) (CP versus hiPSC logFC &gt; 0.2).</div><div id="fspara0450" role="paragraph">(C) Metagene plot representing the normalized ChIP tag densities for GLYR1, H3K36me3 and GATA4 centered on gene bodies and extending one kilobase upstream of the transcription start sites (TSS) and downstream of the transcription end sites (TES). Curves represent a single representative replicate per ChIP condition.</div><div id="fspara0455" role="paragraph">(D) Distribution of GATA4 and GLYR1 genome-wide occupancy across indicated features as assessed by ChIPseq in CPs. GLYR1 CPs ChIPseq n&nbsp;= 5; GATA4 ChIPseq n&nbsp;= 3.</div><div id="fspara0460" role="paragraph">(E) Volcano plots from RNaseq differential expression analysis (FDR&nbsp;&lt; 0.05, n&nbsp;= 3) in CPs versus hiPSCs (GSE137920) for GATA4:GLYR1, GLYR1-Only and GATA4-Only bound genes defined in <a id="crosref2505" href="#fig6">Figure&nbsp;6</a>B.</div><div id="fspara0465" role="paragraph">(F) Genes differentially expressed (DE) upon <i>GLYR1</i> knockdown at CP stage (FDR&nbsp;&lt; 0.05, LogFC&nbsp;&lt; −0.25; n&nbsp;= 2). Cells were transfected with Control or <i>GLYR1</i> siRNAs at day 4 of differentiation and CPs collected 72&nbsp;h later for RNaseq. Bar graphs represent enriched Biological Process terms from Gene Ontology (GO) for downregulated (gray) genes and upregulated genes (red) in si<i>GLYR1</i> compared to siControl treated cells. The number of DE genes and the total number of genes in each GO category are indicated in each bar graph.</div><div id="fspara0470" role="paragraph">(G) Pie charts showing the percentage of genes differentially expressed (DE; FDR&nbsp;&lt; 0.05, LogFC&nbsp;&lt; −0.25) upon <i>GATA4</i> knockdown (si<i>GATA4</i>), <i>GLYR1</i> knockdown (si<i>GLYR1</i>), downregulated in both independent knockdown experiments (blue), upon si<i>GATA4</i> only (green) and upon si<i>GLYR1</i> only (orange), as well as non-DE genes (unchanged; gray) for GATA4:GLYR1-bound genes and Not co-bound genes. siControl versus si<i>GATA4</i> RNaseq (n&nbsp;= 3); siControl versus siGLYR1 RNASeq (n&nbsp;= 2). Each replicate corresponds to independent CM differentiations.</div><div id="fspara0475" role="paragraph">(H) Metagene plots for GATA4:GLYR1 and GATA4-Only-bound genes centered on GATA4 peaks and GATA4:GLYR1 and GLYR1-Only-bound genes centered on GLYR1 broad peaks inside the gene body (1<sup>st</sup> Intron-TES) and showing one representative replicate for the CPs normalized ChIPseq signal for GATA4 (n&nbsp;= 3), GLYR1 (n&nbsp;= 5) and H3K36me3 (n&nbsp;= 2) (lower panels), the indicated histone marks (middle panels; public available data GSE85631 and GSM2047027) and the GATA4 (n&nbsp;= 3), TBX5 (n&nbsp;= 2), NKX2-5 (n&nbsp;= 2), MEIS1 (n&nbsp;= 1) and ISL1 (n&nbsp;= 1) (upper panels).</div><div id="fspara0480" role="paragraph">(I) The ability of GLYR1 to interact with cardiac TFs that co-localized with GATA4-bound regions inside the gene body (1<sup>st</sup> Intron-TES) in CPs was assessed by endogenous GLYR1 or IgG immunoprecipitation (IP) followed by immunoblotting with the indicated antibodies against the endogenous TFs. Enriched nuclear lysates prior to IP (Inputs) were set aside and analyzed in parallel with IP samples to verify similar protein ectopic expression levels across samples.</div></div></div></div></figcaption></figure></div><div id="p0175" role="paragraph">In CPs, as described in other cell types (<span class="dropBlock reference-citations"><a id="body-ref-sref38-4" href="#" role="doc-biblioref" data-xml-rid="bib38" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref38-4" aria-controls="sref38-4">Fei et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref38-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">35.</div><div>Fei, J. ∙ Ishii, H. ∙ Hoeksema, M.A. ...</div><div><strong>NDF, a nucleosome-destabilizing factor that facilitates transcription through nucleosomes</strong></div><div><em>Genes Dev.</em> 2018; <strong>32</strong>:682-694</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/gad.313973.118" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-85047604949" target="_blank" aria-label="Open Scopus in new tab">Scopus (24)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29759984/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2Fgad.313973.118&amp;pmid=29759984" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref143-6" href="#" role="doc-biblioref" data-xml-rid="bib143" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref143-6" aria-controls="sref143-6">Yu et al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref143-6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">129.</div><div>Yu, S. ∙ Li, J. ∙ Ji, G. ...</div><div><strong>Npac Is a Co-factor of Histone H3K36me3 and Regulates Transcriptional Elongation in Mouse Embryonic Stem Cells</strong></div><div><em>Genomics Proteomics Bioinformatics.</em> 2021; <strong>3</strong></div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.gpb.2020.08.004" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_129_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-85118281452" target="_blank" aria-label="Open Scopus in new tab">Scopus (2)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.gpb.2020.08.004" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), GLYR1 broadly occupied gene bodies, from the first intron to the transcription end site (TES) on average (<a id="crosref1235" href="#figs5">Figures S5</a>C and S5D). On the other hand, GATA4 preferentially occupied distal regulatory elements, though some peaks were found at introns inside gene bodies, similar to GLYR1 (<a id="crosref1240" href="#figs5">Figures S5</a>C and S5D). To investigate GATA4-GLYR1 genomic co-occupancy in CPs, we overlapped the genes where GLYR1 was recruited in CPs (GLYR1<sup>CP</sup>: clusters 2 and 3, FDR&nbsp;&lt; 0.1 and Log2FC &gt; 0.5) with genes bound by GATA4 within the gene body window where GLYR1 typically binds (1<sup>st</sup> Intron-TES). This analysis found a statistically significant overlap between GLYR1<sup>CP</sup> and GATA4-bound gene bodies (Fisher exact p value&nbsp;&lt; 2.2E<sup>−16</sup>; OR: 2.38), identifying a defined subset of GATA4 and GLYR1-bound genes, mostly upregulated in CPs versus hiPSCs (FDR&nbsp;&lt; 0.05) and with greater enrichment in heart development GO terms compared to GLYR1-only and GATA4-only occupied gene bodies (<a id="crosref1245" href="#fig6">Figures 6</a>B, 6C, and <a id="crosref1250" href="#figs5">S5</a>E; <a id="crosref1255" href="#mmc1">Tables S1</a>Q and S1R).</div><div id="p0180" role="paragraph">To directly evaluate if GATA4 and GLYR1 regulate the transcription of the genes they co-occupy, we analyzed the effect of silencing <i>GATA4</i> or <i>GLYR1</i> on the expression of genes in CPs by bulk RNA-seq (<a id="crosref1260" href="#mmc1">Tables S1</a>S and S1T). <i>GLYR1</i> silencing led to reduced expression of more than 800 genes associated with embryonic development and heart development terms compared to a control siRNA, which suggested a functional relevance for GLYR1 in the transcriptional regulation of the CM differentiation process (<a id="crosref1265" href="#figs5">Figure&nbsp;S5</a>F; <a id="crosref1270" href="#mmc1">Tables S1</a>S and S1U). Genes co-bound by GATA4 and GLYR1 were about seven times more likely to be significantly downregulated by both GATA4 and GLYR1 independent knockdowns compared to those not co-bound (<a id="crosref1275" href="#figs5">Figure&nbsp;S5</a>G). Several co-occupied and co-regulated loci (<i>GATA4</i>, <i>GATA6</i>, <i>TBX5</i>, <i>LRP2</i>, <i>TEMN4</i>, <i>CC2C2A</i>, <i>TTN</i>, and <i>ENDRA</i>) are involved in human or mouse cardiac malformations (<a id="crosref1280" href="#fig6">Figure&nbsp;6</a>D) (<span class="dropBlock reference-citations"><a id="body-ref-sref22" href="#" role="doc-biblioref" data-xml-rid="bib22" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref22" aria-controls="sref22">Chauveau et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref22"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">21.</div><div>Chauveau, C. ∙ Bonnemann, C.G. ∙ Julien, C. ...</div><div><strong>Recessive TTN truncating mutations define novel forms of core myopathy with heart disease</strong></div><div><em>Hum. Mol. Genet.</em> 2014; <strong>23</strong>:980-991</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/hmg/ddt494" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-84892953505" target="_blank" aria-label="Open Scopus in new tab">Scopus (121)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24105469/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fhmg%2Fddt494&amp;pmid=24105469" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref25" href="#" role="doc-biblioref" data-xml-rid="bib25" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref25" aria-controls="sref25">Clouthier et&nbsp;al., 1998</a><div class="dropBlock__holder" data-db-target-of="sref25"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">24.</div><div>Clouthier, D.E. ∙ Hosoda, K. ∙ Richardson, J.A. ...</div><div><strong>Cranial and cardiac neural crest defects in endothelin-A receptor-deficient mice</strong></div><div><em>Development.</em> 1998; <strong>125</strong>:813-824</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1242/dev.125.5.813" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/9449664/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1242%2Fdev.125.5.813&amp;pmid=9449664" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref100" href="#" role="doc-biblioref" data-xml-rid="bib100" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref100" aria-controls="sref100">Nakamura et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref100"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">88.</div><div>Nakamura, H. ∙ Cook, R.N. ∙ Justice, M.J.</div><div><strong>Mouse Tenm4 is required for mesoderm induction</strong></div><div><em>BMC Dev. Biol.</em> 2013; <strong>13</strong>:9</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/1471-213X-13-9" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-84875327009" target="_blank" aria-label="Open Scopus in new tab">Scopus (16)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23521771/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2F1471-213X-13-9&amp;pmid=23521771" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref129" href="#" role="doc-biblioref" data-xml-rid="bib129" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref129" aria-controls="sref129">Theis et al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref129"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">116.</div><div>Theis, J.L. ∙ Vogler, G. ∙ Missinato, M.A. ...</div><div><strong>Patient-specific genomics and cross-species functional alalysis implicate LRP2 in hypoplastic left heart syndrome</strong></div><div><em>eLife.</em> 2020; <strong>9</strong>:359554</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.7554/eLife.59554" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_116_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-85094219857" target="_blank" aria-label="Open Scopus in new tab">Scopus (10)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.7554%2FeLife.59554" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref106-2" href="#" role="doc-biblioref" data-xml-rid="bib106" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref106-2" aria-controls="sref106-2">Pierpont et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref106-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">94.</div><div>Pierpont, M.E. ∙ Brueckner, M. ∙ Chung, W.K. ..., American Heart Association Council on Cardiovascular Disease in the Young; Council on Cardiovascular and Stroke Nursing; and Council on Genomic and Precision Medicine</div><div><strong>Genetic basis for congenital heart disease: revisited: A scientific statement from the american heart association</strong></div><div><em>Circulation.</em> 2018; <strong>138</strong>:e653-e711</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1161/CIR.0000000000000606" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_94_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-85058908137" target="_blank" aria-label="Open Scopus in new tab">Scopus (203)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30571578/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1161%2FCIR.0000000000000606&amp;pmid=30571578" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The observation that GATA4 mainly occupies intronic regions within GATA4:GLYR1-bound gene bodies led us to examine features characteristic of active or repressed gene regulatory elements (<span class="dropBlock reference-citations"><a id="body-ref-sref1-1" href="#" role="doc-biblioref" data-xml-rid="bib1" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref1-1" aria-controls="sref1-1">Akerberg et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref1-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">1.</div><div>Akerberg, B.N. ∙ Gu, F. ∙ VanDusen, N.J. ...</div><div><strong>A reference map of murine cardiac transcription factor chromatin occupancy identifies dynamic and conserved enhancers</strong></div><div><em>Nat. Commun.</em> 2019; <strong>10</strong>:4907</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41467-019-12812-3" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_1_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-85074196402" target="_blank" aria-label="Open Scopus in new tab">Scopus (39)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31659164/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41467-019-12812-3&amp;pmid=31659164" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref68" href="#" role="doc-biblioref" data-xml-rid="bib68" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref68" aria-controls="sref68">Kimura, 2013</a><div class="dropBlock__holder" data-db-target-of="sref68"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">61.</div><div>Kimura, H.</div><div><strong>Histone modifications for human epigenome analysis</strong></div><div><em>J.&nbsp;Hum. Genet.</em> 2013; <strong>58</strong>:439-445</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/jhg.2013.66" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-84880864749" target="_blank" aria-label="Open Scopus in new tab">Scopus (253)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23739122/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fjhg.2013.66&amp;pmid=23739122" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) within these regions. GATA4 occupancy within GATA4:GLYR1-bound&nbsp;gene bodies co-localized with high levels of marks associated with active regulatory elements (H3K27ac, H3K4me3, H3K4me1, and MED1), as well as with TFs TBX5, MEIS1, ISL1, and NKX2-5, but with undetectable levels of the repressive mark H3K27me3 (<a id="crosref1320" href="#fig6">Figures 6</a>E and <a id="crosref1325" href="#figs5">S5</a>H). Similarly, GATA4 in GATA4-only genes occupied multi-TF intronic regions, but, on average, with lower levels of marks associated with active regulatory elements and higher levels of the repressive mark H3K27me3, in line with GATA4 acting as a repressor in about half of GATA4-only bound neural genes (<a id="crosref1330" href="#fig6">Figures 6</a>C, <a id="crosref1335" href="#figs5">S5</a>E, and S5H). The GATA4 co-localization with multiple TFs in GATA4:GLYR1 co-bound genes raised the possibility that GLYR1 interacted with other cTFs. However, GLYR1 co-immunoprecipitation assays in CPs demonstrated that while GLYR1 interacted with GATA4, it did not interact with the other cTFs tested, including NKX2-5, TBX5, ISL1, and MEIS1 (<a id="crosref1340" href="#figs5">Figure&nbsp;S5</a>I), each of which had motif enrichment in GATA4 occupied regions (<a id="crosref1345" href="#mmc1">Table S1</a>V) (<span class="dropBlock reference-citations"><a id="body-ref-sref1-2" href="#" role="doc-biblioref" data-xml-rid="bib1" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref1-2" aria-controls="sref1-2">Akerberg et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref1-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">1.</div><div>Akerberg, B.N. ∙ Gu, F. ∙ VanDusen, N.J. ...</div><div><strong>A reference map of murine cardiac transcription factor chromatin occupancy identifies dynamic and conserved enhancers</strong></div><div><em>Nat. Commun.</em> 2019; <strong>10</strong>:4907</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41467-019-12812-3" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_1_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-85074196402" target="_blank" aria-label="Open Scopus in new tab">Scopus (39)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31659164/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41467-019-12812-3&amp;pmid=31659164" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="p0185" role="paragraph">To test whether GATA4 and GLYR1 interact to positively regulate gene expression, we cloned several intronic regions with the features described above into a luciferase reporter vector under control of a minimal promoter and tested the transactivation ability of GLYR1 and GATA4. Transfection of GATA4 alone resulted in activation of all three reporters, whereas GLYR1 alone induced luciferase activity of two out of three tested reporters, indicating that these intronic locations could function as response elements (<a id="crosref1355" href="#fig6">Figure&nbsp;6</a>F). Importantly, co-transfection of GATA4 and GLYR1 increased activity of each reporter, consistent with functional co-regulation. The synergistic/additive activation induced by GLYR1 WT was strongly attenuated in the context of the GLYR1 P496L mutation (<a id="crosref1360" href="#fig6">Figure&nbsp;6</a>F).</div></section><section id="sec-2-7"><h3>The P496L variant in GLYR1 affects cardiomyocyte differentiation</h3><div id="p0190" role="paragraph">In order to better characterize the impact of the P496L missense variant in GLYR1 protein function and gene regulation during CM differentiation, we created an hiPSC line homozygous for the missense variant P496L (<i>GLYR1</i><sup><i>P496L</i></sup>) by CRISPR-Cas9 homology-directed repair (<a id="crosref1365" href="#figs6">Figures S6</a>A and S6B). In parallel, we generated a <i>GLYR1</i><sup><i>KO</i></sup> hiPSC line and verified the reduction of RNA expression and absence of detectable protein (<a id="crosref1370" href="#figs6">Figures S6</a>C and S6D).</div><div class="figure-wrap"><figure id="figs6" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2022.01.021/asset/46fb3b43-59d9-4e21-9d14-c8ce6397599d/main.assets/figs6_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs6" src="/cms/10.1016/j.cell.2022.01.021/asset/65dd2a63-1059-453b-87f6-fa9cd51148bb/main.assets/figs6.jpg" height="761" width="656" aria-labelledby="figs6-title figs6-content" data-viewer-src="/cms/10.1016/j.cell.2022.01.021/asset/46fb3b43-59d9-4e21-9d14-c8ce6397599d/main.assets/figs6_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs6-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S6</span> <span class="figure__title__text">Impact of the P496L missense variant in GLYR1 protein function in hiPSC and in hiPSC-derived cardiac progenitors, related to <a id="crosref2510" href="#fig7">Figure&nbsp;7</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs6-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0490" role="paragraph">(A) DNA sequencing traces for region of GLYR1 locus that encodes for the amino acids 493 to 499 from <i>GLYR1</i><sup><i>WT</i></sup> and <i>GLYR1</i><sup><i>P496L</i></sup> hiPSC lines.</div><div id="fspara0495" role="paragraph">(B) Immunoblotting for GLYR1 protein levels in <i>GLYR1</i><sup><i>WT</i></sup> and <i>GLYR1</i><sup><i>P496L</i></sup> hiPSC lysates.</div><div id="fspara0500" role="paragraph">(C) GLYR1 expression by qPCR from <i>GLYR1</i><sup><i>WT</i></sup> and <i>GLYR1</i><sup><i>KO</i></sup> hiPSC lines (n&nbsp;= 3). Unpaired Student’s t test: <sup>∗∗∗</sup>p value&nbsp;&lt; 0.001.</div><div id="fspara0505" role="paragraph">(D) Immunoblotting for GLYR1 protein levels from <i>GLYR1</i><sup><i>WT</i></sup> and <i>GLYR1</i><sup><i>KO</i></sup> hiPSC lysates. GLYR1 knockdown (siGLYR1) and siControl in <i>GLYR1</i><sup><i>WT</i></sup> hiPSCs were included as controls.</div><div id="fspara0510" role="paragraph">(E) UMAP plot of all captured hiPS cells colored by genotype. WT (n&nbsp;= 2), <i>GLYR1</i><sup><i>P496L</i></sup> (n&nbsp;= 2) and <i>GLYR1</i><sup><i>KO</i></sup> (n&nbsp;= 1).</div><div id="fspara0515" role="paragraph">(F) Violin plots for the expression of pluripotency genes, cell cycle genes, tumor suppressors and apoptosis genes in <i>GLYR1</i><sup><i>WT</i></sup>, and <i>GLYR1</i><sup><i>P496L</i></sup>, and <i>GLYR1</i><sup><i>KO</i></sup> hiPSCs.</div><div id="fspara0520" role="paragraph">(G) Selected marker genes expression for each of the identity clusters identified in <i>GLYR1</i><sup><i>WT</i></sup> and <i>GLYR1</i><sup><i>P496L</i></sup> at CM differentiation day 6 by scRNaseq (n&nbsp;= 3). Refers to clusters described in <a id="crosref2515" href="#fig7">Figure&nbsp;7</a>A.</div><div id="fspara0525" role="paragraph">(H) <i>GATA4</i> and <i>GLYR1</i> expression per cluster identified <i>GLYR1</i><sup><i>WT</i></sup> and <i>GLYR1</i><sup><i>P496L</i></sup> CM differentiation day 6 by scRNaseq (n&nbsp;= 3). Refers to clusters described in <a id="crosref2520" href="#fig7">Figure&nbsp;7</a>A.</div><div id="fspara0530" role="paragraph">(I) Percentage of genes driving identity clusters and GATA4:GLYR1 co-bound for each cluster identified in <i>GLYR1</i><sup><i>WT</i></sup> and <i>GLYR1</i><sup><i>P496L</i></sup> CM differentiation day 6 by scRNaseq. Refers to clusters described in <a id="crosref2525" href="#fig7">Figure&nbsp;7</a>A.</div><div id="fspara0535" role="paragraph">(J) Coverage of GLYR1 ChIPseq and expression violin plots within CP-like cells (cluster 0) for representative GATA4:GLYR1 bound loci found in <a id="crosref2530" href="#fig7">Figures 7</a>D and 7E to be downregulated in CP-like cells and had reduced GLYR1 occupancy in <i>GLYR1</i><sup><i>P496L</i></sup> compared to <i>GLYR1</i><sup><i>WT</i></sup> at differentiation day 6 cells. GLYR1 ChIP tracks for 1 representative <i>GLYR1</i><sup><i>WT</i></sup> (n&nbsp;= 5) and <i>GLYR1</i><sup><i>P496L</i></sup> (n&nbsp;= 3) replicate are shown.</div><div id="fspara0540" role="paragraph">(K and L) Scatterplots for GLYR1 ChIPseq log2 average signal across bio-replicates in <i>GLYR1</i><sup><i>WT</i></sup> (n&nbsp;= 5) and <i>GLYR1</i><sup><i>P496L</i></sup> (n&nbsp;= 3) differentiation day 6 cells (K) for GATA4:GLYR1 co-bound genes and not differentially expressed in <a id="crosref2535" href="#fig7">Figure&nbsp;7</a>D; (K) for all GLYR1 bound genes in <i>GLYR1</i><sup><i>WT</i></sup> at CM differentiation day 6. Dash red line&nbsp;= identity line; gray line&nbsp;= data trend line.</div></div></div></div></figcaption></figure></div><div id="p0195" role="paragraph">In mouse embryonic stem cells (mESCs), GLYR1 is essential for pluripotency, and depletion of GLYR1 leads to differentiation, cell cycle arrest, and apoptosis (<span class="dropBlock reference-citations"><a id="body-ref-sref143-7" href="#" role="doc-biblioref" data-xml-rid="bib143" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref143-7" aria-controls="sref143-7">Yu et al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref143-7"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">129.</div><div>Yu, S. ∙ Li, J. ∙ Ji, G. ...</div><div><strong>Npac Is a Co-factor of Histone H3K36me3 and Regulates Transcriptional Elongation in Mouse Embryonic Stem Cells</strong></div><div><em>Genomics Proteomics Bioinformatics.</em> 2021; <strong>3</strong></div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.gpb.2020.08.004" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_129_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-85118281452" target="_blank" aria-label="Open Scopus in new tab">Scopus (2)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.gpb.2020.08.004" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Similar to mESCs, GLYR1 deficiency in hiPSCs resulted in a dramatic shift in the transcriptional landscape (<a id="crosref1380" href="#figs6">Figure&nbsp;S6</a>E), characterized by a reduction in pluripotency and cell cycle gene expression and upregulation of tumor suppressors and apoptosis genes compared to <i>GLYR1</i><sup><i>WT</i></sup>. However, these changes were not observed in <i>GLYR1</i><sup><i>P496L</i></sup> hiPSCs, which were more transcriptionally similar to <i>GLYR1</i><sup><i>WT</i></sup> cells (<a id="crosref1385" href="#figs6">Figures S6</a>E and S6F). These results suggested that GLYR1 is essential for the maintenance of pluripotency in hiPSCs and that the P496L variant does not lead to a complete protein loss of function.</div><div id="p0200" role="paragraph">We then subjected the <i>GLYR1</i><sup><i>WT</i></sup> and <i>GLYR1</i><sup><i>P496L</i></sup> lines to CM differentiation and investigated their transcriptional landscapes at the CP (day 6) and CM stages (day 18) by scRNA-seq. At day 6 of differentiation, we observed three distinct subpopulations comprising both <i>GLYR1</i><sup><i>WT</i></sup> and <i>GLYR1</i><sup><i>P496L</i></sup> cells clustering together and expressing marker genes associated with CPs (cluster 0), endoderm (cluster 1), and vascular/endothelial cells (cluster 4), as well as a neural progenitor-like cluster mainly comprising <i>GLYR1</i><sup><i>P496L</i></sup> cells (cluster 2) (<a id="crosref1390" href="#fig7">Figures 7</a>A, 7B, and <a id="crosref1395" href="#figs6">S6</a>G; <a id="crosref1400" href="#mmc1">Table S1</a>W). <i>GLYR1</i> expression was detected in all clusters, whereas <i>GATA4</i> was only detected in the CP (cluster 0), endoderm (cluster 1), and the vascular/endothelial (cluster 4)-like clusters (<a id="crosref1405" href="#figs6">Figure&nbsp;S6</a>H). The intersection of the scRNA-seq analysis with GLYR1 and GATA4 ChIP-seq at differentiation day 6 from <i>GLYR1</i><sup><i>WT</i></sup> revealed that the CP-like cluster was enriched for genes co-bound by GATA4 and GLYR1 (<a id="crosref1410" href="#figs6">Figure&nbsp;S6</a>I). Differential expression analysis (FDR&nbsp;&lt; 0.05, LogFC &gt; 0.125) between <i>GLYR1</i><sup><i>P496L</i></sup> and <i>GLYR1</i><sup><i>WT</i></sup> CP subpopulations (cluster 0) identified 1,458 downregulated genes involved in heart development, cytoskeletal organization, cell cycle, response to hypoxia, and ATP metabolic processes; 1,025 genes were upregulated and associated with non-cardiomyocyte developmental processes (<a id="crosref1415" href="#fig7">Figure&nbsp;7</a>C; <a id="crosref1420" href="#mmc1">Tables S1</a>X and S1Y). Remarkably, more than 35% of the GATA4:GLYR1 co-bound genes were differentially expressed between <i>GLYR1</i><sup><i>P496L</i></sup> and <i>GLYR1</i><sup><i>WT</i></sup> CPs, whereas less than 15% of GATA4-only and GLYR1-only bound were differentially expressed (<a id="crosref1425" href="#fig7">Figure&nbsp;7</a>D). ChIP-seq revealed that GLYR1 occupancy was reduced in the <i>GLYR1</i><sup><i>P496L</i></sup> CP-like cluster among GATA4:GLYR1 co-bound genes whose expression was shown to be downregulated, but not on those whose expression was upregulated or not changed (<a id="crosref1430" href="#fig7">Figures 7</a>E, 7F, <a id="crosref1435" href="#figs6">S6</a>J, and S6K). Importantly, GLYR1 genome-wide binding was not affected by the P496L variant (<a id="crosref1440" href="#figs6">Figure&nbsp;S6</a>L). These data suggest that the P496L variant affects GLYR1 occupancy and transcriptional regulation of a discrete set of target genes co-bound by GATA4, several of which have been involved in human cardiac malformations and cardiomyopathies.</div><div class="figure-wrap"><figure id="fig7" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2022.01.021/asset/13ad87ad-364c-45ce-9258-cc9b3160629e/main.assets/gr7_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr7" src="/cms/10.1016/j.cell.2022.01.021/asset/d48839aa-b97d-4a39-9793-197954f3b6dc/main.assets/gr7.jpg" height="935" width="656" aria-labelledby="fig7-title fig7-content" data-viewer-src="/cms/10.1016/j.cell.2022.01.021/asset/13ad87ad-364c-45ce-9258-cc9b3160629e/main.assets/gr7_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig7-title" class="figure__title"><span class="label figure__label">Figure&nbsp;7</span> <span class="figure__title__text">Impact of the GLYR1 P496L missense variant in human iPS-derived cardiac cells and in mouse cardiogenesis</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig7-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0205" role="paragraph">(A and B) UMAP plot from three independent human iPS-CP differentiations at day 6 colored by (A) cluster identity and (B) genotype. Bar plot and natural log odds ratio (LogOR) reflect the <i>GLYR1</i><sup><i>P496L</i></sup> cells contribution to each of the clusters compared to <i>GLYR1</i><sup><i>WT</i></sup> cells. None reach statistical significance.</div><div id="fspara0210" role="paragraph">(C) Gene Ontology (GO) Biological Process enrichment analysis for genes upregulated or downregulated (<i>GLYR1</i><sup><i>P496L</i></sup> versus <i>GLYR1</i><sup><i>WT</i></sup>, FDR&nbsp;&lt; 0.05) within CP-like cells (cluster 0) at differentiation day 6.</div><div id="fspara0215" role="paragraph">(D) Percentage of GATA4:GLYR1, GLYR1-only, or GATA4-only bound genes in CPs based on ChIP-seq that were differentially expressed (<i>GLYR1</i><sup><i>P496L</i></sup> versus <i>GLYR1</i><sup><i>WT</i></sup>, FDR&nbsp;&lt; 0.05) in CP-like cells (cluster 0). Numbers within the bars: absolute numbers of genes involved.</div><div id="fspara0220" role="paragraph">(E) Scatterplots for GLYR1 ChIP-seq signal among biological replicates from <i>GLYR1</i><sup><i>WT</i></sup> (n&nbsp;= 5) or <i>GLYR1</i><sup><i>P496L</i></sup> (n&nbsp;= 3) CP differentiation at day 6 for GATA4:GLYR1 co-bound genes and downregulated or upregulated (FDR&nbsp;&lt; 0.05) in (D). Dashed red line: identity line; gray line: data trend line.</div><div id="fspara0225" role="paragraph">(F) Representative GLYR1 ChIP-seq coverage tracks and expression violin plots for two representative GATA4:GLYR1 bound loci found in (D and E) to be downregulated in CP-like cells (cluster 0) and had reduced GLYR1 occupancy in <i>GLYR1</i><sup><i>P496L</i></sup> compared to <i>GLYR1</i><sup><i>WT</i></sup> at differentiation day 6.</div><div id="fspara0230" role="paragraph">(G and H) UMAP plot from three independent day 18 CM differentiations colored by (G) cluster identity and (H) genotype. Bar plot and natural log odds ratio (LogOR) reflect <i>GLYR1</i><sup><i>P496L</i></sup> cells contribution to each of the identity clusters compared to <i>GLYR1</i><sup><i>WT</i></sup> cells (<a id="crosref2365" href="#mmc1">Table S1</a>AB). The LogOR of all clusters between <i>GLYR1</i><sup><i>P496L</i></sup> versus <i>GLYR1</i><sup><i>WT</i></sup> are statistically significant, except for clusters 7 and 10 (FDR&nbsp;&lt; 0.05).</div><div id="fspara0235" role="paragraph">(I) Percentage of cTNT positive cells in <i>GLYR1</i><sup><i>WT</i></sup> and <i>GLYR1</i><sup><i>P496L</i></sup> CM differentiation day 18 as measured by flow cytometry (n&nbsp;= 3). Unpaired Student’s t test: <sup>∗∗∗</sup>p&nbsp;value&nbsp;&lt; 0.001.</div><div id="fspara0240" role="paragraph">(J) Representative immunostaining micrographs for cTNT (red), GLYR1 (green), or DAPI (blue) in <i>GLYR1</i><sup><i>WT</i></sup> and <i>GLYR1</i><sup><i>P496L</i></sup> at CM differentiation day 18. Scale, 50&nbsp;μm.</div><div id="fspara0245" role="paragraph">(K) Whole-mount images (scale 1&nbsp;mm) and hematoxylin and eosin (H&amp;E) images of cross-sections (scale 300&nbsp;μm) from WT, <i>Glyr1</i><sup><i>+/P495L</i></sup>, <i>Gata4</i><sup><i>+/−</i></sup>, <i>and</i> Glyr<i>1</i><sup><i>+/P495L</i></sup><i>:Gata4</i><sup><i>+/−</i></sup> representative hearts at postnatal day 1. The AVSD incidence per genotype is indicated as a percentage of the total number of hearts analyzed by histology.</div><div id="fspara0250" role="paragraph">(L) Echocardiography detection of ventricular septal defects (VSD) in Glyr<i>1</i><sup><i>+/P495L</i></sup><i>:Gata4</i><sup><i>+/−</i></sup> compound heterozygous hearts at postnatal day 0. On apical four-chamber view, in Glyr<i>1</i><sup><i>+/P495L</i></sup><i>:Gata4</i><sup><i>+/−</i></sup>, red flow in the right ventricle (RV) indicates blood that has crossed the intraventricular septum (IVS) from the left ventricle (LV) and is flowing toward the transducer.</div><div id="fspara0255" role="paragraph">See also <a id="crosref2370" href="#figs6">Figures S6</a> and <a id="crosref2375" href="#figs7">S7</a> and <a id="crosref2380" href="#mmc1">Tables S1</a>U–S1AC.</div></div></div></div></figcaption></figure></div><div id="p0205" role="paragraph">At day 18 of differentiation, we observed a heterogenous group of cellular subpopulations with significantly imbalanced contributions from <i>GLYR1</i><sup><i>P496L</i></sup> and <i>GLYR1</i><sup><i>WT</i></sup> cells. <i>GLYR1</i><sup><i>P496L</i></sup> cells mainly contributed to neural-like (clusters 2, 3, 5, and 9) and hepatocyte-like (cluster 7) subpopulations, to the detriment of CM-like clusters 0 and 6 (<a id="crosref1445" href="#fig7">Figures 7</a>G, 7H, and <a id="crosref1450" href="#figs7">S7</a>A; <a id="crosref1455" href="#mmc1">Tables S1</a>Z and S1AA). The impaired CM-like differentiation of the <i>GLYR1</i><sup><i>P496L</i></sup> line was also evident in the diminished cTNT levels detected by FACs and immunostaining (<a id="crosref1460" href="#fig7">Figures 7</a>I and 7J). Within the CM-like clusters, <i>GLYR1</i><sup><i>P496L</i></sup> versus <i>GLYR1</i><sup><i>WT</i></sup> cells were homogenously distributed, and no segregation between genotypes was observed based on maturity markers (<a id="crosref1465" href="#figs7">Figure&nbsp;S7</a>B). However, differential expression analysis identified 544 downregulated genes in <i>GLYR1</i><sup><i>P496L</i></sup> CMs (FDR&nbsp;&lt; 0.05; logFC &gt; 0.125 <i>GLYR1</i><sup><i>P496L</i></sup> versus <i>GLYR1</i><sup><i>WT</i></sup>), mainly associated with precursor metabolites and energy generation, particularly ATP metabolic processes. Additionally, 803 genes were upregulated in GLYR1<sup>P496L</sup> CMs (FDR&nbsp;&lt; 0.05; logFC &gt; 0.125 <i>GLYR1</i><sup><i>P496L</i></sup> versus <i>GLYR1</i><sup><i>WT</i></sup>) involved in cell adhesion and migration, protein phosphorylation, and cell contraction (<a id="crosref1470" href="#figs7">Figure&nbsp;S7</a>C; <a id="crosref1475" href="#mmc1">Tables S1</a>AB and S1AC). Moreover, <i>GLYR1</i><sup><i>P496L</i></sup> CMs exhibited altered beating dynamics characterized by a significant prolongation of physical contraction and relaxation, accompanied by a decrease in the beating rate compared to control cardiomyocytes (<a id="crosref1480" href="#figs7">Figure&nbsp;S7</a>D; <a id="crosref1485" href="#mmc2">Videos S1</a> and <a id="crosref1490" href="#mmc3">S2</a>). Overall, these data demonstrated a detrimental impact of the GLYR1 P496L variant in CM differentiation, associated with altered GLYR1 genomic occupancy and gene regulation at a discrete set of loci co-bound by GATA4.</div><div class="figure-wrap"><figure id="figs7" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2022.01.021/asset/fd9aee73-2c6b-4cf6-ae04-920e1d3957a0/main.assets/figs7_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs7" src="/cms/10.1016/j.cell.2022.01.021/asset/40aa3d08-29bb-48cc-95b9-96c9c6240685/main.assets/figs7.jpg" height="903" width="656" aria-labelledby="figs7-title figs7-content" data-viewer-src="/cms/10.1016/j.cell.2022.01.021/asset/fd9aee73-2c6b-4cf6-ae04-920e1d3957a0/main.assets/figs7_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs7-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S7</span> <span class="figure__title__text">Impact of the P496L missense variant in GLYR1 protein function in hiPSC-derived cardiomyocytes and during mouse development, related to <a id="crosref2540" href="#fig7">Figure&nbsp;7</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs7-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0550" role="paragraph">(A) Selected marker gene expression for each of the identity clusters identified in <i>GLYR1</i><sup><i>WT</i></sup> and <i>GLYR1</i><sup><i>P496L</i></sup> at CM differentiation day 18 by scRNaseq (n&nbsp;= 3). Refers to clusters described in <a id="crosref2545" href="#fig7">Figure&nbsp;7</a>G.</div><div id="fspara0555" role="paragraph">(B) UMAP plot for CM-like cells subclustered and colored by genotype and expression of known CM genes associated with different maturity levels.</div><div id="fspara0560" role="paragraph">(C) Gene Ontology (GO) Biological Process enrichment analysis for genes upregulated and downregulated (<i>GLYR1</i><sup><i>P496L</i></sup> versus <i>GLYR1</i><sup><i>WT</i></sup>, FDR&nbsp;&lt; 0.05) within the CM-like subpopulation (cluster 0 and 6) at differentiation day 18.</div><div id="fspara0565" role="paragraph">(D) <i>GLYR1</i><sup><i>WT</i></sup> and <i>GLYR1</i><sup><i>P496L</i></sup> CM differentiation day 18 contractility parameters measured by PULSE automated software, which captures and quantifies the biomechanical beating of cardiomyocytes by performing motion analysis on the image sequence to capture changes in the image intensity due to cardiomyocyte contraction and relaxation. Data from three WT and four <i>GLYR1</i><sup><i>P496L</i></sup> independent differentiations; 3-4 wells were analyzed per differentiation.</div><div id="fspara0570" role="paragraph">(E) DNA Sequencing traces for region of the mouse <i>Glyr1</i> locus that encodes for the amino acids 493 to 498 from WT and <i>Glyr1</i><sup><i>+/P495L</i></sup> and <i>Glyr1</i><sup><i>P495L/P495L</i></sup> mice.</div><div id="fspara0575" role="paragraph">(F) Genotyping data from parental intercross of <i>Glyr1</i><sup><i>+/P495</i></sup> animals demonstrating postnatal lethality between day 0-1 after birth in <i>Glyr1</i><sup><i>P495L/P495L</i></sup> offspring. Chi-square statistic: <sup>∗∗</sup>p value&nbsp;&lt; 0.01.</div><div id="fspara0580" role="paragraph">(G) Echocardiography detection of ventricular septal defects (VSD) by color flow Doppler in Glyr<i>1</i><sup><i>P495L/P495L</i></sup> hearts at postnatal day 0.</div><div id="fspara0585" role="paragraph">(H) Hematoxylin and eosin (H&amp;E) images of cross-sections from a representative WT heart and a Glyr<i>1</i><sup><i>P495L/P495L</i></sup> heart with a muscular VSD at postnatal day 1.</div><div id="fspara0590" role="paragraph">(I) Genotyping data from parental intercross of <i>Glyr1</i><sup><i>+/P495L</i></sup><i>and Gata4</i><sup><i>+/−</i></sup> animals demonstrating embryonic lethality at birth in <i>Glyr1</i><sup><i>+/P495L</i></sup><i>;Gata4</i><sup><i>+/−</i></sup> compound heterozygous offspring. Chi-square statistic: <sup>∗∗∗</sup>p value&nbsp;&lt; 0.001.</div><div id="fspara0595" role="paragraph">(J) Representative hematoxylin and eosin (H&amp;E) heart cross-section (scale 300&nbsp;μm) and whole mount image (scale 1&nbsp;mm) from a Glyr<i>1</i><sup><i>+/P495L</i></sup><i>:Gata4</i><sup><i>+/−</i></sup> mouse that died by postnatal day 1 showing a dysmorphic heart displaying an atrio-ventricular septal defect (dotted circle).</div></div></div></div></figcaption></figure></div><div id="p0210" role="paragraph"><div id="mmc2" class="core-supplementary-material" data-category="video"><div class="core-link"><div class="cloudflare-stream-container" id="mmc2" data-type="mp4"><span id="video_stream_mmc2_src" class="video-stream-source" style="display:none">eyJraWQiOiI4ZjUxYWNhY2IzYjhiNjNlNzFlYmIzYWFmYTU5NmZmYyIsImFsZyI6IlJTMjU2In0.eyJzdWIiOiJiN2NhMzEyMjAwZjk1NjA5Y2I3MzgxYTFhOWU5MDJjNyIsImtpZCI6IjhmNTFhY2FjYjNiOGI2M2U3MWViYjNhYWZhNTk2ZmZjIiwiZXhwIjoxNzQ5NjE0Mjk4fQ.HGCCttsfP_T7jOaVt0l81e8k3K6kv6KMzdfYDzQYPRXqs_Wu8HxYI11-uSQ_WHxpzDMxIkKLP6pCVIFkiY1QANJa20-YtsGrDkzbsm_rV7kPmNuUbLw3TzktyQUZLbLud7uco9_jvU1OTOgzp5X_fq6LDS9Nr2dcroiUozr6RBfn80BsJSo0YDeXF9dgLmCsaXLh29sXN5WQgGILnirD8FwGoMIWPAcgyF1XvTc5KYyKpJ87t7kkWmtJME8jlyh3usMLTg9s5PYYiucNJLOpzplPcHaM2xAqLhnlPkLnC1hi6qUsShMcNfUuWVJ4NvZt-O8gKd706An1rYjdoNdsWQ</span><stream src="eyJraWQiOiI4ZjUxYWNhY2IzYjhiNjNlNzFlYmIzYWFmYTU5NmZmYyIsImFsZyI6IlJTMjU2In0.eyJzdWIiOiJiN2NhMzEyMjAwZjk1NjA5Y2I3MzgxYTFhOWU5MDJjNyIsImtpZCI6IjhmNTFhY2FjYjNiOGI2M2U3MWViYjNhYWZhNTk2ZmZjIiwiZXhwIjoxNzQ5NjE0Mjk4fQ.HGCCttsfP_T7jOaVt0l81e8k3K6kv6KMzdfYDzQYPRXqs_Wu8HxYI11-uSQ_WHxpzDMxIkKLP6pCVIFkiY1QANJa20-YtsGrDkzbsm_rV7kPmNuUbLw3TzktyQUZLbLud7uco9_jvU1OTOgzp5X_fq6LDS9Nr2dcroiUozr6RBfn80BsJSo0YDeXF9dgLmCsaXLh29sXN5WQgGILnirD8FwGoMIWPAcgyF1XvTc5KYyKpJ87t7kkWmtJME8jlyh3usMLTg9s5PYYiucNJLOpzplPcHaM2xAqLhnlPkLnC1hi6qUsShMcNfUuWVJ4NvZt-O8gKd706An1rYjdoNdsWQ" class="cloudflare-stream-player ready tracked" id="video_stream_mmc2" controls="true"><div class="player-share-button" title="Share Video"><div class="share-player-icon"></div></div><div class="player-share-container hidden"><span class="player-close-share"></span><div class="social-links-container"><ul class="social-links-list"><li class="html5-share-facebook-icon" data-share-url="https://www.facebook.com/sharer/sharer.php?u=https://www.cell.com/cell/fulltext/S0092-8674(22)00079-4" target="_blank"></li><li class="html5-share-twitter-icon" data-share-url="https://twitter.com/share?url=https://www.cell.com/cell/fulltext/S0092-8674(22)00079-4" target="_blank"></li><li class="html5-share-mail-icon" data-share-url="mailto:?body=https://www.cell.com/cell/fulltext/S0092-8674(22)00079-4" target="_blank"></li></ul><input type="text" value="https://www.cell.com/cell/fulltext/S0092-8674(22)00079-4" readonly="readonly" class="share-input"></div></div><div style="position: relative; height: 0px; width: 100%; padding-top: 56.25%;"><iframe src="https://embed.cloudflarestream.com/embed/iframe.fla9.823df22.html?videoId=eyJraWQiOiI4ZjUxYWNhY2IzYjhiNjNlNzFlYmIzYWFmYTU5NmZmYyIsImFsZyI6IlJTMjU2In0.eyJzdWIiOiJiN2NhMzEyMjAwZjk1NjA5Y2I3MzgxYTFhOWU5MDJjNyIsImtpZCI6IjhmNTFhY2FjYjNiOGI2M2U3MWViYjNhYWZhNTk2ZmZjIiwiZXhwIjoxNzQ5NjE0Mjk4fQ.HGCCttsfP_T7jOaVt0l81e8k3K6kv6KMzdfYDzQYPRXqs_Wu8HxYI11-uSQ_WHxpzDMxIkKLP6pCVIFkiY1QANJa20-YtsGrDkzbsm_rV7kPmNuUbLw3TzktyQUZLbLud7uco9_jvU1OTOgzp5X_fq6LDS9Nr2dcroiUozr6RBfn80BsJSo0YDeXF9dgLmCsaXLh29sXN5WQgGILnirD8FwGoMIWPAcgyF1XvTc5KYyKpJ87t7kkWmtJME8jlyh3usMLTg9s5PYYiucNJLOpzplPcHaM2xAqLhnlPkLnC1hi6qUsShMcNfUuWVJ4NvZt-O8gKd706An1rYjdoNdsWQ" allowfullscreen="true" allow="autoplay; picture-in-picture" style="height: 100%; width: 100%; position: absolute; top: 0px; left: 0px; border: none;"></iframe></div></stream></div><a href="/cms/10.1016/j.cell.2022.01.021/attachment/814b488b-372c-4620-908a-8de10b1f3822/mmc2.mp4" download="mmc2.mp4" aria-labelledby="mmc2-heading mmc2-link"><span id="mmc2-link">Video (30.97 MB)</span></a></div><div class="core-description"><div id="mmc2-heading" class="heading">Video S1. <i>GLYR1</i><sup><i>WT</i></sup> cardiomyocyte differentiation day 18 representative beating video, related to Figure&nbsp;7</div></div></div></div><div id="p0215" role="paragraph"><div id="mmc3" class="core-supplementary-material" data-category="video"><div class="core-link"><div class="cloudflare-stream-container" id="mmc3" data-type="mp4"><span id="video_stream_mmc3_src" class="video-stream-source" style="display:none">eyJraWQiOiI4ZjUxYWNhY2IzYjhiNjNlNzFlYmIzYWFmYTU5NmZmYyIsImFsZyI6IlJTMjU2In0.eyJzdWIiOiI5NjZiZTJmMDBmMGMzZWIzMDU3MDU2Nzk0YjlmZTg2MiIsImtpZCI6IjhmNTFhY2FjYjNiOGI2M2U3MWViYjNhYWZhNTk2ZmZjIiwiZXhwIjoxNzQ5NjE0Mjk4fQ.PK_JM8MOia2c5mv45OKDesAHqp2agd3C8jNbsi4AvfNGHwo79MsRG7Y4CnVei8MXY3MRlY9XXPgMKH2G8F98lXVex_ZlJkMHltD7zlB6b5IrUFVIDrXqu2YBEuY82POmnaYTY3JyCVxpXeaLAwhbXpYVjoio8J5b3kjldAZHh70Wn-tdakWJc5_zvWreofOohrDkRJz8a7Jaz2sInhV5bw81GPq06Y7uARRQCyIJ6Bj3xNyPMSx7aNRFhOWapzNgfJynz00eSneNemBoYkjyF_-wjlc4vS243fCL4tKql-BA5XdkgcQh2hgks0kdDjSNUuznEh9x67AGjqbiheqcrw</span><stream src="eyJraWQiOiI4ZjUxYWNhY2IzYjhiNjNlNzFlYmIzYWFmYTU5NmZmYyIsImFsZyI6IlJTMjU2In0.eyJzdWIiOiI5NjZiZTJmMDBmMGMzZWIzMDU3MDU2Nzk0YjlmZTg2MiIsImtpZCI6IjhmNTFhY2FjYjNiOGI2M2U3MWViYjNhYWZhNTk2ZmZjIiwiZXhwIjoxNzQ5NjE0Mjk4fQ.PK_JM8MOia2c5mv45OKDesAHqp2agd3C8jNbsi4AvfNGHwo79MsRG7Y4CnVei8MXY3MRlY9XXPgMKH2G8F98lXVex_ZlJkMHltD7zlB6b5IrUFVIDrXqu2YBEuY82POmnaYTY3JyCVxpXeaLAwhbXpYVjoio8J5b3kjldAZHh70Wn-tdakWJc5_zvWreofOohrDkRJz8a7Jaz2sInhV5bw81GPq06Y7uARRQCyIJ6Bj3xNyPMSx7aNRFhOWapzNgfJynz00eSneNemBoYkjyF_-wjlc4vS243fCL4tKql-BA5XdkgcQh2hgks0kdDjSNUuznEh9x67AGjqbiheqcrw" class="cloudflare-stream-player ready tracked" id="video_stream_mmc3" controls="true"><div class="player-share-button" title="Share Video"><div class="share-player-icon"></div></div><div class="player-share-container hidden"><span class="player-close-share"></span><div class="social-links-container"><ul class="social-links-list"><li class="html5-share-facebook-icon" data-share-url="https://www.facebook.com/sharer/sharer.php?u=https://www.cell.com/cell/fulltext/S0092-8674(22)00079-4" target="_blank"></li><li class="html5-share-twitter-icon" data-share-url="https://twitter.com/share?url=https://www.cell.com/cell/fulltext/S0092-8674(22)00079-4" target="_blank"></li><li class="html5-share-mail-icon" data-share-url="mailto:?body=https://www.cell.com/cell/fulltext/S0092-8674(22)00079-4" target="_blank"></li></ul><input type="text" value="https://www.cell.com/cell/fulltext/S0092-8674(22)00079-4" readonly="readonly" class="share-input"></div></div><div style="position: relative; height: 0px; width: 100%; padding-top: 56.25%;"><iframe src="https://embed.cloudflarestream.com/embed/iframe.fla9.823df22.html?videoId=eyJraWQiOiI4ZjUxYWNhY2IzYjhiNjNlNzFlYmIzYWFmYTU5NmZmYyIsImFsZyI6IlJTMjU2In0.eyJzdWIiOiI5NjZiZTJmMDBmMGMzZWIzMDU3MDU2Nzk0YjlmZTg2MiIsImtpZCI6IjhmNTFhY2FjYjNiOGI2M2U3MWViYjNhYWZhNTk2ZmZjIiwiZXhwIjoxNzQ5NjE0Mjk4fQ.PK_JM8MOia2c5mv45OKDesAHqp2agd3C8jNbsi4AvfNGHwo79MsRG7Y4CnVei8MXY3MRlY9XXPgMKH2G8F98lXVex_ZlJkMHltD7zlB6b5IrUFVIDrXqu2YBEuY82POmnaYTY3JyCVxpXeaLAwhbXpYVjoio8J5b3kjldAZHh70Wn-tdakWJc5_zvWreofOohrDkRJz8a7Jaz2sInhV5bw81GPq06Y7uARRQCyIJ6Bj3xNyPMSx7aNRFhOWapzNgfJynz00eSneNemBoYkjyF_-wjlc4vS243fCL4tKql-BA5XdkgcQh2hgks0kdDjSNUuznEh9x67AGjqbiheqcrw" allowfullscreen="true" allow="autoplay; picture-in-picture" style="height: 100%; width: 100%; position: absolute; top: 0px; left: 0px; border: none;"></iframe></div></stream></div><a href="/cms/10.1016/j.cell.2022.01.021/attachment/c1601eeb-6e5f-4430-bf4e-02c5244e8e61/mmc3.mp4" download="mmc3.mp4" aria-labelledby="mmc3-heading mmc3-link"><span id="mmc3-link">Video (30.95 MB)</span></a></div><div class="core-description"><div id="mmc3-heading" class="heading">Video S2. <i>GLYR1</i><sup><i>P496L</i></sup> cardiomyocyte differentiation day 18 representative beating video, related to Figure&nbsp;7</div></div></div></div></section><section id="sec-2-8"><h3>GLYR1 mutation disrupts mouse cardiac development</h3><div id="p0220" role="paragraph">In order to assess the biological importance of the GLYR1 P496L variant <i>in&nbsp;vivo</i>, we generated a mouse line harboring a P495L single nucleotide variant in <i>GLYR1</i> (<i>Glyr1</i><sup><i>P495L/+</i></sup>), homologous to human P496L, using CRISPR-Cas9 mediated genome editing (<a id="crosref1495" href="#figs7">Figure&nbsp;S7</a>E). After backcrossing four generations into the C57BL6/J background, we intercrossed <i>Glyr1</i><sup><i>P495L/+</i></sup> mice and collected 92 pups for genotyping and heart histology at day 1 after birth (P1). Although all genotypes were born at the expected mendelian ratios (<mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="normal chi squared" tabindex="0" ctxtmenu_counter="0" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msup data-semantic-type="superscript" data-semantic-role="greekletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="normal chi squared"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c3C7">χ</mjx-c></mjx-mi><mjx-script style="vertical-align: 0.363em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn></mjx-script></mjx-msup></mjx-mrow></mjx-math></mjx-container>&nbsp;= 0.96), 54% of the <i>Glyr1</i><sup><i>P495L/P495L</i></sup> and 15.5% of the heterozygous (<i>Glyr1</i><sup><i>P495L/+</i></sup>) mice displayed postnatal lethality between days 0 and 1, compared to only 4.4% of the WT littermates (<mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="normal chi squared" tabindex="0" ctxtmenu_counter="1" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msup data-semantic-type="superscript" data-semantic-role="greekletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="normal chi squared"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c3C7">χ</mjx-c></mjx-mi><mjx-script style="vertical-align: 0.363em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn></mjx-script></mjx-msup></mjx-mrow></mjx-math></mjx-container>&nbsp;= 0.02 at P1) (<a id="crosref1500" href="#figs7">Figure&nbsp;S7</a>F). Echocardiography and histological analysis revealed ventricular septal defects (VSDs) in ∼15% of the <i>Glyr1</i><sup><i>P495L/P495L</i></sup> mice (<a id="crosref1505" href="#figs7">Figures S7</a>G and S7H; <a id="crosref1510" href="#mmc4">Videos S3</a> and <a id="crosref1515" href="#mmc5">S4</a>). Thus, this model provides evidence for the biological importance of GLYR1 in cardiac development and demonstrates a deleterious effect of the P495L variant <i>in&nbsp;vivo</i>.</div><div id="p0225" role="paragraph"><div id="mmc4" class="core-supplementary-material" data-category="video"><div class="core-link"><div class="cloudflare-stream-container" id="mmc4" data-type="mp4"><span id="video_stream_mmc4_src" class="video-stream-source" style="display:none">eyJraWQiOiI4ZjUxYWNhY2IzYjhiNjNlNzFlYmIzYWFmYTU5NmZmYyIsImFsZyI6IlJTMjU2In0.eyJzdWIiOiJjOTMwNjllZDk4Nzk4YzA1ZmQ2OWZiMmY4YjRkYjI2NyIsImtpZCI6IjhmNTFhY2FjYjNiOGI2M2U3MWViYjNhYWZhNTk2ZmZjIiwiZXhwIjoxNzQ5NjE0Mjk4fQ.Ga1JwRZS5FCqNTZWa0ifiQns6abvfYcja4n0feM1db6c5OeR8ckzIDjtJKIC0WE4MmKIGt9uZ75641N7RBoIy2NfRnZlYAlFb9mT70apm-XEhRF7g73Rrpj4W5XHZ90-EvvATzMJtoB2NaCuPwh4C8lggL0h-7NdlNADt85iQh-RKNSKhvZomPu0j4o6uiCfpzCWw5hJqrcV0HFoBDUvq4YiM1GomLpLaA2LKarI8RyrS_DONzx2NMvW9gCIdwusjYAz6nYFOCbGI9fqDtpMD-eFzhdAApOeMkF1vTsufK7zYq_EY6fTrmXp0jWE5_S4lq1NO-EOVvRKqqJuLgCpeQ</span><stream src="eyJraWQiOiI4ZjUxYWNhY2IzYjhiNjNlNzFlYmIzYWFmYTU5NmZmYyIsImFsZyI6IlJTMjU2In0.eyJzdWIiOiJjOTMwNjllZDk4Nzk4YzA1ZmQ2OWZiMmY4YjRkYjI2NyIsImtpZCI6IjhmNTFhY2FjYjNiOGI2M2U3MWViYjNhYWZhNTk2ZmZjIiwiZXhwIjoxNzQ5NjE0Mjk4fQ.Ga1JwRZS5FCqNTZWa0ifiQns6abvfYcja4n0feM1db6c5OeR8ckzIDjtJKIC0WE4MmKIGt9uZ75641N7RBoIy2NfRnZlYAlFb9mT70apm-XEhRF7g73Rrpj4W5XHZ90-EvvATzMJtoB2NaCuPwh4C8lggL0h-7NdlNADt85iQh-RKNSKhvZomPu0j4o6uiCfpzCWw5hJqrcV0HFoBDUvq4YiM1GomLpLaA2LKarI8RyrS_DONzx2NMvW9gCIdwusjYAz6nYFOCbGI9fqDtpMD-eFzhdAApOeMkF1vTsufK7zYq_EY6fTrmXp0jWE5_S4lq1NO-EOVvRKqqJuLgCpeQ" class="cloudflare-stream-player ready tracked" id="video_stream_mmc4" controls="true"><div class="player-share-button" title="Share Video"><div class="share-player-icon"></div></div><div class="player-share-container hidden"><span class="player-close-share"></span><div class="social-links-container"><ul class="social-links-list"><li class="html5-share-facebook-icon" data-share-url="https://www.facebook.com/sharer/sharer.php?u=https://www.cell.com/cell/fulltext/S0092-8674(22)00079-4" target="_blank"></li><li class="html5-share-twitter-icon" data-share-url="https://twitter.com/share?url=https://www.cell.com/cell/fulltext/S0092-8674(22)00079-4" target="_blank"></li><li class="html5-share-mail-icon" data-share-url="mailto:?body=https://www.cell.com/cell/fulltext/S0092-8674(22)00079-4" target="_blank"></li></ul><input type="text" value="https://www.cell.com/cell/fulltext/S0092-8674(22)00079-4" readonly="readonly" class="share-input"></div></div><div style="position: relative; height: 0px; width: 100%; padding-top: 69.6314%;"><iframe src="https://embed.cloudflarestream.com/embed/iframe.fla9.823df22.html?videoId=eyJraWQiOiI4ZjUxYWNhY2IzYjhiNjNlNzFlYmIzYWFmYTU5NmZmYyIsImFsZyI6IlJTMjU2In0.eyJzdWIiOiJjOTMwNjllZDk4Nzk4YzA1ZmQ2OWZiMmY4YjRkYjI2NyIsImtpZCI6IjhmNTFhY2FjYjNiOGI2M2U3MWViYjNhYWZhNTk2ZmZjIiwiZXhwIjoxNzQ5NjE0Mjk4fQ.Ga1JwRZS5FCqNTZWa0ifiQns6abvfYcja4n0feM1db6c5OeR8ckzIDjtJKIC0WE4MmKIGt9uZ75641N7RBoIy2NfRnZlYAlFb9mT70apm-XEhRF7g73Rrpj4W5XHZ90-EvvATzMJtoB2NaCuPwh4C8lggL0h-7NdlNADt85iQh-RKNSKhvZomPu0j4o6uiCfpzCWw5hJqrcV0HFoBDUvq4YiM1GomLpLaA2LKarI8RyrS_DONzx2NMvW9gCIdwusjYAz6nYFOCbGI9fqDtpMD-eFzhdAApOeMkF1vTsufK7zYq_EY6fTrmXp0jWE5_S4lq1NO-EOVvRKqqJuLgCpeQ" allowfullscreen="true" allow="autoplay; picture-in-picture" style="height: 100%; width: 100%; position: absolute; top: 0px; left: 0px; border: none;"></iframe></div></stream></div><a href="/cms/10.1016/j.cell.2022.01.021/attachment/a8dfd94a-e0e7-4df1-8070-d092cab1f73d/mmc4.mp4" download="mmc4.mp4" aria-labelledby="mmc4-heading mmc4-link"><span id="mmc4-link">Video (397.78 KB)</span></a></div><div class="core-description"><div id="mmc4-heading" class="heading">Video S3. Echocardiography colored flow Doppler in WT littermate hearts at postnatal day 0 from parental intercross of <i>Glyr1</i><sup><i>+/P495</i></sup> animals, related to Figure&nbsp;7</div></div></div></div><div id="p0230" role="paragraph"><div id="mmc5" class="core-supplementary-material" data-category="video"><div class="core-link"><div class="cloudflare-stream-container" id="mmc5" data-type="mp4"><span id="video_stream_mmc5_src" class="video-stream-source" style="display:none">eyJraWQiOiI4ZjUxYWNhY2IzYjhiNjNlNzFlYmIzYWFmYTU5NmZmYyIsImFsZyI6IlJTMjU2In0.eyJzdWIiOiIzNDVhYzZjMDVmYWNkNzk5NzMzYzY5NDJjYjFmMmI0NyIsImtpZCI6IjhmNTFhY2FjYjNiOGI2M2U3MWViYjNhYWZhNTk2ZmZjIiwiZXhwIjoxNzQ5NjE0Mjk4fQ.W4yZrBFDnX1UC1ewwpHl35RPtGVvRn8Ko-GbCoTPpYXuApjM6MNI70ciYJETeKoFxdulyn_KKMAqwI6wx5Y2bCiR4Lkeor_Hds5PqrAknLPGx-xGwig9-Qr3AyNYITYH3eQqmT1LUTubHCnefF89vLhr1FDuG2trRGEA626nh7xnJZvNt_WE7wCbjPVc7SvsnLmUJvj0lwc5UN_tWNzAfbDnOQyD3P-D6wbyZouIG8AQ0AgSrpkuPu_1hqk-vtoP7QmI5niJKqjJHXVula7s7UuKa5oQAdhJwnE2NUBhb8uDZ3F7xuFkfTRPUC1bjmf8qMohoB9Bgt9P_TlO9zQ_Ew</span><stream src="eyJraWQiOiI4ZjUxYWNhY2IzYjhiNjNlNzFlYmIzYWFmYTU5NmZmYyIsImFsZyI6IlJTMjU2In0.eyJzdWIiOiIzNDVhYzZjMDVmYWNkNzk5NzMzYzY5NDJjYjFmMmI0NyIsImtpZCI6IjhmNTFhY2FjYjNiOGI2M2U3MWViYjNhYWZhNTk2ZmZjIiwiZXhwIjoxNzQ5NjE0Mjk4fQ.W4yZrBFDnX1UC1ewwpHl35RPtGVvRn8Ko-GbCoTPpYXuApjM6MNI70ciYJETeKoFxdulyn_KKMAqwI6wx5Y2bCiR4Lkeor_Hds5PqrAknLPGx-xGwig9-Qr3AyNYITYH3eQqmT1LUTubHCnefF89vLhr1FDuG2trRGEA626nh7xnJZvNt_WE7wCbjPVc7SvsnLmUJvj0lwc5UN_tWNzAfbDnOQyD3P-D6wbyZouIG8AQ0AgSrpkuPu_1hqk-vtoP7QmI5niJKqjJHXVula7s7UuKa5oQAdhJwnE2NUBhb8uDZ3F7xuFkfTRPUC1bjmf8qMohoB9Bgt9P_TlO9zQ_Ew" class="cloudflare-stream-player ready tracked" id="video_stream_mmc5" controls="true"><div class="player-share-button" title="Share Video"><div class="share-player-icon"></div></div><div class="player-share-container hidden"><span class="player-close-share"></span><div class="social-links-container"><ul class="social-links-list"><li class="html5-share-facebook-icon" data-share-url="https://www.facebook.com/sharer/sharer.php?u=https://www.cell.com/cell/fulltext/S0092-8674(22)00079-4" target="_blank"></li><li class="html5-share-twitter-icon" data-share-url="https://twitter.com/share?url=https://www.cell.com/cell/fulltext/S0092-8674(22)00079-4" target="_blank"></li><li class="html5-share-mail-icon" data-share-url="mailto:?body=https://www.cell.com/cell/fulltext/S0092-8674(22)00079-4" target="_blank"></li></ul><input type="text" value="https://www.cell.com/cell/fulltext/S0092-8674(22)00079-4" readonly="readonly" class="share-input"></div></div><div style="position: relative; height: 0px; width: 100%; padding-top: 69.6314%;"><iframe src="https://embed.cloudflarestream.com/embed/iframe.fla9.823df22.html?videoId=eyJraWQiOiI4ZjUxYWNhY2IzYjhiNjNlNzFlYmIzYWFmYTU5NmZmYyIsImFsZyI6IlJTMjU2In0.eyJzdWIiOiIzNDVhYzZjMDVmYWNkNzk5NzMzYzY5NDJjYjFmMmI0NyIsImtpZCI6IjhmNTFhY2FjYjNiOGI2M2U3MWViYjNhYWZhNTk2ZmZjIiwiZXhwIjoxNzQ5NjE0Mjk4fQ.W4yZrBFDnX1UC1ewwpHl35RPtGVvRn8Ko-GbCoTPpYXuApjM6MNI70ciYJETeKoFxdulyn_KKMAqwI6wx5Y2bCiR4Lkeor_Hds5PqrAknLPGx-xGwig9-Qr3AyNYITYH3eQqmT1LUTubHCnefF89vLhr1FDuG2trRGEA626nh7xnJZvNt_WE7wCbjPVc7SvsnLmUJvj0lwc5UN_tWNzAfbDnOQyD3P-D6wbyZouIG8AQ0AgSrpkuPu_1hqk-vtoP7QmI5niJKqjJHXVula7s7UuKa5oQAdhJwnE2NUBhb8uDZ3F7xuFkfTRPUC1bjmf8qMohoB9Bgt9P_TlO9zQ_Ew" allowfullscreen="true" allow="autoplay; picture-in-picture" style="height: 100%; width: 100%; position: absolute; top: 0px; left: 0px; border: none;"></iframe></div></stream></div><a href="/cms/10.1016/j.cell.2022.01.021/attachment/1e894436-2d05-49f6-832e-235de33b6c79/mmc5.mp4" download="mmc5.mp4" aria-labelledby="mmc5-heading mmc5-link"><span id="mmc5-link">Video (582.66 KB)</span></a></div><div class="core-description"><div id="mmc5-heading" class="heading">Video S4. Echocardiography detection of ventricular septal defects (VSD) by colored flow Doppler in Glyr1<sup>P495L/P495</sup> homozygous hearts at postnatal day 0, related to Figure&nbsp;7</div></div></div></div><div id="p0235" role="paragraph">To assess whether there is a GATA4-GLYR1 genetic interaction in mice, we crossed <i>Glyr1</i><sup><i>P495L/+</i></sup> mice to GATA4 mutant mice (<span class="dropBlock reference-citations"><a id="body-ref-sref138" href="#" role="doc-biblioref" data-xml-rid="bib138" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref138" aria-controls="sref138">Watt et&nbsp;al., 2004</a><div class="dropBlock__holder" data-db-target-of="sref138"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">125.</div><div>Watt, A.J. ∙ Battle, M.A. ∙ Li, J. ...</div><div><strong>GATA4 is essential for formation of the proepicardium and regulates cardiogenesis</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2004; <strong>101</strong>:12573-12578</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.0400752101" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_125_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-4344565788" target="_blank" aria-label="Open Scopus in new tab">Scopus (222)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15310850/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.0400752101&amp;pmid=15310850" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). As expected, <i>Gata4</i><sup><i>+/−</i></sup> mice exhibited partially penetrant cardiac defects with a 22% incidence of VSDs, and 50% died between day 0 and day 1, consistent with previous reports for Gata4 deletion in C57BL6/J background (<a id="crosref1525" href="#fig7">Figures 7</a>K and <a id="crosref1530" href="#figs7">S7</a>I) (<span class="dropBlock reference-citations"><a id="body-ref-sref110-2" href="#" role="doc-biblioref" data-xml-rid="bib110" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref110-2" aria-controls="sref110-2">Rajagopal et&nbsp;al., 2007</a><div class="dropBlock__holder" data-db-target-of="sref110-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">98.</div><div>Rajagopal, S.K. ∙ Ma, Q. ∙ Obler, D. ...</div><div><strong>Spectrum of heart disease associated with murine and human GATA4 mutation</strong></div><div><em>J.&nbsp;Mol. Cell. Cardiol.</em> 2007; <strong>43</strong>:677-685</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_98_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=10.1016%2Fj.yjmcc.2007.06.004&amp;cf=fulltext&amp;site=yjmcc-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_98_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=10.1016%2Fj.yjmcc.2007.06.004&amp;cf=pdf&amp;site=yjmcc-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_98_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-35048838310" target="_blank" aria-label="Open Scopus in new tab">Scopus (197)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17643447/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.yjmcc.2007.06.004&amp;pmid=17643447" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). <i>Glyr1</i><sup><i>P495L/+</i></sup> mice exhibited the expected low penetrance of cardiac alterations, most frequently a persistent interatrial communication (patent foramen ovale), whereas no defects were detected in the 23 WT littermate hearts analyzed (<a id="crosref1540" href="#fig7">Figures 7</a>K and <a id="crosref1545" href="#figs7">S7</a>I). We only identified eight compound heterozygote (<i>Glyr1</i><sup><i>P495L/+</i></sup><i>:Gata4</i><sup><i>+/−</i></sup>) animals out of 88 pups at birth (∼9.1% observed versus 25% expected, <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="normal chi squared" tabindex="0" ctxtmenu_counter="2" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msup data-semantic-type="superscript" data-semantic-role="greekletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="normal chi squared"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c3C7">χ</mjx-c></mjx-mi><mjx-script style="vertical-align: 0.363em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn></mjx-script></mjx-msup></mjx-mrow></mjx-math></mjx-container>&nbsp;= 0.0086), and five of those died in the first 24&nbsp;h of life (<a id="crosref1550" href="#figs7">Figure&nbsp;S7</a>I). Whole-mount histology and echocardiography analysis showed that all compound <i>Glyr1</i><sup><i>P495L/+</i></sup><i>:Gata4</i><sup><i>+/−</i></sup> hearts were dysmorphic, with complete penetrance of cardiac septal defects, including about 80% represented by atrio-ventricular septal defects (AVSDs) (<a id="crosref1555" href="#fig7">Figures 7</a>K, <a id="crosref1560" href="#figs7">S7</a>I, and S7J; <a id="crosref1565" href="#mmc6">Videos S5</a> and <a id="crosref1570" href="#mmc7">S6</a>). These data provide <i>in&nbsp;vivo</i> evidence for the biological relevance of the <i>GLYR1</i> P495L variant and its interaction with GATA4 in human disease.</div><div id="p0240" role="paragraph"><div id="mmc6" class="core-supplementary-material" data-category="video"><div class="core-link"><div class="cloudflare-stream-container" id="mmc6" data-type="mp4"><span id="video_stream_mmc6_src" class="video-stream-source" style="display:none">eyJraWQiOiI4ZjUxYWNhY2IzYjhiNjNlNzFlYmIzYWFmYTU5NmZmYyIsImFsZyI6IlJTMjU2In0.eyJzdWIiOiJmMmJiMzE3NGQ1YWU3NWQwMTZmNDI5NzRhY2RmZjUxOCIsImtpZCI6IjhmNTFhY2FjYjNiOGI2M2U3MWViYjNhYWZhNTk2ZmZjIiwiZXhwIjoxNzQ5NjE0Mjk4fQ.WKv8kkUMbqCTSgenbgt5BCNFewmZmz_tooyoR4Mubm-CtTqZBxcbnNUZcGkTnEN4O2OkUBx-ymR3qgck2at-d0Zwi9x9XMLmFI0CkSwdBtf1k7Fwgysb20FtTXUoRfWs5aJl_-Y_4XV4R_KLTj8DWofDkFjjtnKSsyxrT3TlN2c221aCFIbGA444qY6U8AAdfVIB-4SjGlNL5AvyxfFw8K-XHVvduyuF9HO72p-zqHAvZa43gAiqweXjJz7ER04PoV_3sfJvYCRTjg4K8sVhWNTynJ2TGKdQY8NeyY-0Fcx6h84mnwl4Dh_gjd4K3QHDi-w3DhBp7qv-iGV3Xt5rIQ</span><stream src="eyJraWQiOiI4ZjUxYWNhY2IzYjhiNjNlNzFlYmIzYWFmYTU5NmZmYyIsImFsZyI6IlJTMjU2In0.eyJzdWIiOiJmMmJiMzE3NGQ1YWU3NWQwMTZmNDI5NzRhY2RmZjUxOCIsImtpZCI6IjhmNTFhY2FjYjNiOGI2M2U3MWViYjNhYWZhNTk2ZmZjIiwiZXhwIjoxNzQ5NjE0Mjk4fQ.WKv8kkUMbqCTSgenbgt5BCNFewmZmz_tooyoR4Mubm-CtTqZBxcbnNUZcGkTnEN4O2OkUBx-ymR3qgck2at-d0Zwi9x9XMLmFI0CkSwdBtf1k7Fwgysb20FtTXUoRfWs5aJl_-Y_4XV4R_KLTj8DWofDkFjjtnKSsyxrT3TlN2c221aCFIbGA444qY6U8AAdfVIB-4SjGlNL5AvyxfFw8K-XHVvduyuF9HO72p-zqHAvZa43gAiqweXjJz7ER04PoV_3sfJvYCRTjg4K8sVhWNTynJ2TGKdQY8NeyY-0Fcx6h84mnwl4Dh_gjd4K3QHDi-w3DhBp7qv-iGV3Xt5rIQ" class="cloudflare-stream-player ready tracked" id="video_stream_mmc6" controls="true"><div class="player-share-button" title="Share Video"><div class="share-player-icon"></div></div><div class="player-share-container hidden"><span class="player-close-share"></span><div class="social-links-container"><ul class="social-links-list"><li class="html5-share-facebook-icon" data-share-url="https://www.facebook.com/sharer/sharer.php?u=https://www.cell.com/cell/fulltext/S0092-8674(22)00079-4" target="_blank"></li><li class="html5-share-twitter-icon" data-share-url="https://twitter.com/share?url=https://www.cell.com/cell/fulltext/S0092-8674(22)00079-4" target="_blank"></li><li class="html5-share-mail-icon" data-share-url="mailto:?body=https://www.cell.com/cell/fulltext/S0092-8674(22)00079-4" target="_blank"></li></ul><input type="text" value="https://www.cell.com/cell/fulltext/S0092-8674(22)00079-4" readonly="readonly" class="share-input"></div></div><div style="position: relative; height: 0px; width: 100%; padding-top: 69.6314%;"><iframe src="https://embed.cloudflarestream.com/embed/iframe.fla9.823df22.html?videoId=eyJraWQiOiI4ZjUxYWNhY2IzYjhiNjNlNzFlYmIzYWFmYTU5NmZmYyIsImFsZyI6IlJTMjU2In0.eyJzdWIiOiJmMmJiMzE3NGQ1YWU3NWQwMTZmNDI5NzRhY2RmZjUxOCIsImtpZCI6IjhmNTFhY2FjYjNiOGI2M2U3MWViYjNhYWZhNTk2ZmZjIiwiZXhwIjoxNzQ5NjE0Mjk4fQ.WKv8kkUMbqCTSgenbgt5BCNFewmZmz_tooyoR4Mubm-CtTqZBxcbnNUZcGkTnEN4O2OkUBx-ymR3qgck2at-d0Zwi9x9XMLmFI0CkSwdBtf1k7Fwgysb20FtTXUoRfWs5aJl_-Y_4XV4R_KLTj8DWofDkFjjtnKSsyxrT3TlN2c221aCFIbGA444qY6U8AAdfVIB-4SjGlNL5AvyxfFw8K-XHVvduyuF9HO72p-zqHAvZa43gAiqweXjJz7ER04PoV_3sfJvYCRTjg4K8sVhWNTynJ2TGKdQY8NeyY-0Fcx6h84mnwl4Dh_gjd4K3QHDi-w3DhBp7qv-iGV3Xt5rIQ" allowfullscreen="true" allow="autoplay; picture-in-picture" style="height: 100%; width: 100%; position: absolute; top: 0px; left: 0px; border: none;"></iframe></div></stream></div><a href="/cms/10.1016/j.cell.2022.01.021/attachment/cf784197-391d-4d88-9e47-794f1860eab5/mmc6.mp4" download="mmc6.mp4" aria-labelledby="mmc6-heading mmc6-link"><span id="mmc6-link">Video (431.21 KB)</span></a></div><div class="core-description"><div id="mmc6-heading" class="heading">Video S5. Echocardiography colored flow Doppler in WT littermate hearts at postnatal day 0 from parental intercross of <i>Glyr1</i><sup><i>+/P495</i></sup> and <i>Gata4</i><sup><i>+/−</i></sup> animals, related to Figure&nbsp;7</div></div></div></div><div id="p0245" role="paragraph"><div id="mmc7" class="core-supplementary-material" data-category="video"><div class="core-link"><div class="cloudflare-stream-container" id="mmc7" data-type="mp4"><span id="video_stream_mmc7_src" class="video-stream-source" style="display:none">eyJraWQiOiI4ZjUxYWNhY2IzYjhiNjNlNzFlYmIzYWFmYTU5NmZmYyIsImFsZyI6IlJTMjU2In0.eyJzdWIiOiIxMzFkYjQ3ZjAzZjdjYzUzNGZhMzUzOGU5OGI2NGRkYSIsImtpZCI6IjhmNTFhY2FjYjNiOGI2M2U3MWViYjNhYWZhNTk2ZmZjIiwiZXhwIjoxNzQ5NjE0Mjk4fQ.RhcIYFPMiVA36oPEARZpNy1FJ0S6_RiUF03UlH2kMRG6t45Rv_8M_sZFGEXHALnMujiOdV4oAs9QR3CASN7K_Uu_DbgqMollw6wFvdo31A6n_uzw1QooTkoYvjGi5gKT19tAIstQ2F_iNFkwnmLzoqhQeUL-jKSwD8pQwpPoM7vwJfo_pdRYgtgToI2a9FbTeywQcKgjuI6Gf_COhy2oWk6gI3joGaD-QaYjiAuTTmdisATQNsVjOKUx6N4WMqXQRQI4130tH5Sq5eMoeOYpzvtiXYl-jwWA8bKmamAryTRYdzG0r9-uP1Qd2PDqmp2mM41gXckweRMMJpE8bZQ2IQ</span><stream src="eyJraWQiOiI4ZjUxYWNhY2IzYjhiNjNlNzFlYmIzYWFmYTU5NmZmYyIsImFsZyI6IlJTMjU2In0.eyJzdWIiOiIxMzFkYjQ3ZjAzZjdjYzUzNGZhMzUzOGU5OGI2NGRkYSIsImtpZCI6IjhmNTFhY2FjYjNiOGI2M2U3MWViYjNhYWZhNTk2ZmZjIiwiZXhwIjoxNzQ5NjE0Mjk4fQ.RhcIYFPMiVA36oPEARZpNy1FJ0S6_RiUF03UlH2kMRG6t45Rv_8M_sZFGEXHALnMujiOdV4oAs9QR3CASN7K_Uu_DbgqMollw6wFvdo31A6n_uzw1QooTkoYvjGi5gKT19tAIstQ2F_iNFkwnmLzoqhQeUL-jKSwD8pQwpPoM7vwJfo_pdRYgtgToI2a9FbTeywQcKgjuI6Gf_COhy2oWk6gI3joGaD-QaYjiAuTTmdisATQNsVjOKUx6N4WMqXQRQI4130tH5Sq5eMoeOYpzvtiXYl-jwWA8bKmamAryTRYdzG0r9-uP1Qd2PDqmp2mM41gXckweRMMJpE8bZQ2IQ" class="cloudflare-stream-player ready tracked" id="video_stream_mmc7" controls="true"><div class="player-share-button" title="Share Video"><div class="share-player-icon"></div></div><div class="player-share-container hidden"><span class="player-close-share"></span><div class="social-links-container"><ul class="social-links-list"><li class="html5-share-facebook-icon" data-share-url="https://www.facebook.com/sharer/sharer.php?u=https://www.cell.com/cell/fulltext/S0092-8674(22)00079-4" target="_blank"></li><li class="html5-share-twitter-icon" data-share-url="https://twitter.com/share?url=https://www.cell.com/cell/fulltext/S0092-8674(22)00079-4" target="_blank"></li><li class="html5-share-mail-icon" data-share-url="mailto:?body=https://www.cell.com/cell/fulltext/S0092-8674(22)00079-4" target="_blank"></li></ul><input type="text" value="https://www.cell.com/cell/fulltext/S0092-8674(22)00079-4" readonly="readonly" class="share-input"></div></div><div style="position: relative; height: 0px; width: 100%; padding-top: 69.6314%;"><iframe src="https://embed.cloudflarestream.com/embed/iframe.fla9.823df22.html?videoId=eyJraWQiOiI4ZjUxYWNhY2IzYjhiNjNlNzFlYmIzYWFmYTU5NmZmYyIsImFsZyI6IlJTMjU2In0.eyJzdWIiOiIxMzFkYjQ3ZjAzZjdjYzUzNGZhMzUzOGU5OGI2NGRkYSIsImtpZCI6IjhmNTFhY2FjYjNiOGI2M2U3MWViYjNhYWZhNTk2ZmZjIiwiZXhwIjoxNzQ5NjE0Mjk4fQ.RhcIYFPMiVA36oPEARZpNy1FJ0S6_RiUF03UlH2kMRG6t45Rv_8M_sZFGEXHALnMujiOdV4oAs9QR3CASN7K_Uu_DbgqMollw6wFvdo31A6n_uzw1QooTkoYvjGi5gKT19tAIstQ2F_iNFkwnmLzoqhQeUL-jKSwD8pQwpPoM7vwJfo_pdRYgtgToI2a9FbTeywQcKgjuI6Gf_COhy2oWk6gI3joGaD-QaYjiAuTTmdisATQNsVjOKUx6N4WMqXQRQI4130tH5Sq5eMoeOYpzvtiXYl-jwWA8bKmamAryTRYdzG0r9-uP1Qd2PDqmp2mM41gXckweRMMJpE8bZQ2IQ" allowfullscreen="true" allow="autoplay; picture-in-picture" style="height: 100%; width: 100%; position: absolute; top: 0px; left: 0px; border: none;"></iframe></div></stream></div><a href="/cms/10.1016/j.cell.2022.01.021/attachment/66da8ab5-363b-48fd-8db1-e22ae24f3a6e/mmc7.mp4" download="mmc7.mp4" aria-labelledby="mmc7-heading mmc7-link"><span id="mmc7-link">Video (445.85 KB)</span></a></div><div class="core-description"><div id="mmc7-heading" class="heading">Video S6. Echocardiography detection of ventricular septal defects (VSD) by colored flow Doppler in Glyr1<sup>+/P495</sup>:Gata4<sup>+/−</sup> compound heterozygous hearts at postnatal day 0, related to Figure&nbsp;7</div></div></div></div></section></section><section id="sec-3"><h2>Discussion</h2><div id="p0250" role="paragraph">Here, we integrated an analysis of the protein-protein interaction network of CHD-associated TFs with human whole-exome-sequencing data to inform the genetic underpinnings of CHD. An unbiased PPI reconstruction for two essential cTFs, GATA4 and TBX5, identified known and previously unreported functional relationships. DNVs in GT-PPIs occurred with significantly greater frequency in CHD patients than healthy controls. Additionally, a consolidative computational framework devised to prioritize variants in GT-interacting proteins identified numerous candidate disease genes, including <i>GLYR1</i>, a ubiquitously expressed epigenetic reader. GLYR1 widely co-occupied cardiac regulatory elements with GATA4, and the <i>GLYR1</i> disease variant P496L disrupted the interaction with GATA4 and co-activation of cardiac developmental genes. The importance of the GLYR1 variant and the GATA4-GLRY1 interaction in cardiac development was further confirmed in a mouse model. These findings indicate that the use of tissue- and disease-specific PPIs may partially overcome the genetic heterogeneity of CHDs and help prioritize the potential impact of <i>de novo</i> missense variants present in disease.</div><section id="sec-3-1"><h3>Integration of tissue-specific TF-PPIs with human variant data highlights disease mechanisms</h3><div id="p0255" role="paragraph">The integration of PPI information from publicly available databases with human genetic data has been previously used to prioritize disease candidate variants based on network topological measures (<span class="dropBlock reference-citations"><a id="body-ref-sref16" href="#" role="doc-biblioref" data-xml-rid="bib16" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref16" aria-controls="sref16">Bryois et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref16"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">16.</div><div>Bryois, J. ∙ Skene, N.G. ∙ Hansen, T.F. ..., Eating Disorders Working Group of the Psychiatric Genomics Consortium, International Headache Genetics Consortium, 23andMe Research Team</div><div><strong>Genetic identification of cell types underlying brain complex traits yields insights into the etiology of Parkinson’s disease</strong></div><div><em>Nat. Genet.</em> 2020; <strong>52</strong>:482-493</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41588-020-0610-9" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-85084195663" target="_blank" aria-label="Open Scopus in new tab">Scopus (92)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32341526/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41588-020-0610-9&amp;pmid=32341526" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref50" href="#" role="doc-biblioref" data-xml-rid="bib50" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref50" aria-controls="sref50">Greene et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref50"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">45.</div><div>Greene, C.S. ∙ Krishnan, A. ∙ Wong, A.K. ...</div><div><strong>Understanding multicellular function and disease with human tissue-specific networks</strong></div><div><em>Nat. Genet.</em> 2015; <strong>47</strong>:569-576</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng.3259" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_45_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-84930083700" target="_blank" aria-label="Open Scopus in new tab">Scopus (461)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25915600/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng.3259&amp;pmid=25915600" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref60-2" href="#" role="doc-biblioref" data-xml-rid="bib60" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref60-2" aria-controls="sref60-2">Izarzugaza et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref60-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">54.</div><div>Izarzugaza, J.M.G. ∙ Ellesøe, S.G. ∙ Doganli, C. ..., Competence Network for Congenital Heart Defects, Germany</div><div><strong>Systems genetics analysis identifies calcium-signaling defects as novel cause of congenital heart disease</strong></div><div><em>Genome Med.</em> 2020; <strong>12</strong>:76</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13073-020-00772-z" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-85090171445" target="_blank" aria-label="Open Scopus in new tab">Scopus (5)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32859249/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13073-020-00772-z&amp;pmid=32859249" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref71-2" href="#" role="doc-biblioref" data-xml-rid="bib71" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref71-2" aria-controls="sref71-2">Köhler et&nbsp;al., 2008</a><div class="dropBlock__holder" data-db-target-of="sref71-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">63.</div><div>Köhler, S. ∙ Bauer, S. ∙ Horn, D. ...</div><div><strong>Walking the interactome for prioritization of candidate disease genes</strong></div><div><em>Am. J. Hum. Genet.</em> 2008; <strong>82</strong>:949-958</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=10.1016%2Fj.ajhg.2008.02.013&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=10.1016%2Fj.ajhg.2008.02.013&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-41549139527" target="_blank" aria-label="Open Scopus in new tab">Scopus (910)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/18371930/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.ajhg.2008.02.013&amp;pmid=18371930" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref107" href="#" role="doc-biblioref" data-xml-rid="bib107" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref107" aria-controls="sref107">Priest et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref107"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">95.</div><div>Priest, J.R. ∙ Osoegawa, K. ∙ Mohammed, N. ...</div><div><strong>De novo and rare variants at multiple loci support the oligogenic origins of atrioventricular septal heart defects</strong></div><div><em>PLoS Genet.</em> 2016; <strong>12</strong>:e1005963</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pgen.1005963" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27058611/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pgen.1005963&amp;pmid=27058611" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Since most of the available PPI datasets have been reconstructed in non-physiological settings and cell types not relevant to the disease of interest, some of these methods incorporated RNA expression information to generate predicted “tissue-specific” networks to reduce the number of candidate variants (<span class="dropBlock reference-citations"><a id="body-ref-sref7-2" href="#" role="doc-biblioref" data-xml-rid="bib7" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref7-2" aria-controls="sref7-2">Barshir et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref7-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">7.</div><div>Barshir, R. ∙ Shwartz, O. ∙ Smoly, I.Y. ...</div><div><strong>Comparative analysis of human tissue interactomes reveals factors leading to tissue-specific manifestation of hereditary diseases</strong></div><div><em>PLoS Comput. Biol.</em> 2014; <strong>10</strong>:e1003632</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pcbi.1003632" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-84903384845" target="_blank" aria-label="Open Scopus in new tab">Scopus (49)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24921629/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pcbi.1003632&amp;pmid=24921629" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref90" href="#" role="doc-biblioref" data-xml-rid="bib90" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref90" aria-controls="sref90">Magger et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref90"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">78.</div><div>Magger, O. ∙ Waldman, Y.Y. ∙ Ruppin, E. ...</div><div><strong>Enhancing the prioritization of disease-causing genes through tissue specific protein interaction networks</strong></div><div><em>PLoS Comput. Biol.</em> 2012; <strong>8</strong>:e1002690</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pcbi.1002690" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_78_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-84866888440" target="_blank" aria-label="Open Scopus in new tab">Scopus (117)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23028288/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pcbi.1002690&amp;pmid=23028288" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). However, whether a protein-protein interaction indeed occurs in the tissue depends on additional factors, and co-expression of both partners is only a necessary initial requirement but not a guarantee for the interaction to occur. Even after the application of these prioritization strategies, the large number of highly ranked candidate variants makes it challenging to identify likely contributing mutations in the absence of additional biologically meaningful information. In contrast, the approach described here allowed us to capture ubiquitously expressed CHD candidate genes that might have tissue-specific effects due to their interaction with tissue-enriched factors. This is of importance, as the majority of known disease genes are broadly expressed across multiple human tissues.</div><div id="p0260" role="paragraph">In contrast to single-gene enrichment approaches, the network-enrichment analysis allows the detection of rare CHD candidate genes, but it does so without resolving the relative contributions of specific variants. Hence, downstream prioritization of candidate disease variants is needed to rank the likelihood that specific variants contribute to CHD. For this purpose, the integrative scoring method we developed combines commonly used disease-variant prioritization metrics, including diverse and complementary biological information at the gene and variant levels, together with proband pedigree information. The integration of proband genomic information regarding the co-occurrence of variants in known CHD genes with metrics that predict variant deleteriousness and gene-level parameters allowed prioritization of variants with potentially higher contribution to CHD. Functional investigation will be needed to test whether the identified CHD candidate genes are essential in heart development and to determine the causal nature of the associated variants, as we have done here with <i>GLYR1</i>. In the future, high-throughput screening methods similar to our integrative PPI-genetic variant scoring pipeline will aid in assessing the vast genomic variation catalog provided by the increasing number of large-scale sequencing studies.</div></section><section id="sec-3-2"><h3>GLYR1 co-regulates heart development genes with GATA4 and is mutated in CHD</h3><div id="p0265" role="paragraph">Our integrative proteomics and human genetics approach revealed GLYR1 as a GATA4 interactor in CPs that constitutes a strong candidate gene for CHD. Since the GLYR1 P496L variant impacted the protein structural dynamics and reduced its interaction with GATA4, it suggests that destabilization of tissue-specific protein-protein interactions could result in cardiac-restricted phenotypic manifestation associated with the mutation of a ubiquitously expressed chromatin reader. Evidence from human iPSC-derived CPs and <i>in&nbsp;vivo</i> mouse studies suggests that the GLYR1 P496L functions as a hypomorph with&nbsp;discrete alterations in genome occupancy at a subset of&nbsp;H3K36me3-enriched regions. During CM differentiation, GATA4 physical interaction with GLYR1 may be one of the mechanisms explaining how GLYR1 can bind a specific subset of heart development genes. Disruption of this co-regulation in the context of the P496L variant has detrimental effects in CM differentiation that may contribute to cardiac malformations. Indeed, the genetic interaction observed in mice compound heterozygous for GATA4 and GLYR1 P496L, with a high incidence of atrioventricular septal defects, is in agreement with the GLYR1 variant playing a role in CHD. Given the overlapping functions of GATA4 and GATA6, it will be interesting to determine if the inherited heterozygous missense variant in GATA6 in this proband functions together with the GLYR1 variant to cause disease.</div><div id="p0270" role="paragraph">Overall, this work has identified interactors of TFs essential for cardiac development, provided a ranked list of candidate disease variants potentially contributing to CHD, and revealed biology of gene regulation related to cardiac disease. Notably, this tissue- and disease-specific TF network-based approach could be applied with slight modifications of the variant prioritization scoring to other genetic disorders for which large-scale sequencing data are available to highlight disease mechanisms and provide a powerful filter for interrogating the genetic basis of disease.</div></section><section id="sec-3-3"><h3>Limitations of the study</h3><div id="p0275" role="paragraph">The variant prioritization scoring developed in this study has been customized specifically for the CHD variant dataset from trio whole-exome sequencing and designed as a complementary method to our interactome filtering approach. Although the principles could be widely applicable to other genetic diseases, disease- and dataset-specific modifications would be necessary for its application in different disease contexts. Further, this study focuses on very rare variants, which are normally depleted from the population, as is the case for the P496L variant in GLYR1. Indeed, as a <i>de novo</i> variant, the likelihood of observing P496L in another sequenced individual is small. Future studies will be needed to determine whether variants in GLYR1 contribute to other cases of CHD or to other diseases in humans.</div></section></section><section id="sec-4"><h2>STAR★Methods</h2><section id="sec-4-1"><h3>Key resources table</h3><div id="p0285" role="paragraph"><div class="figure-wrap"><figure id="undtbl1" class="table" data-figureviewer-ignore="true"><div class="table-wrap"><table data-figureviewer-ignore="true" tabindex="0"><thead><tr class="rowsep"><th>REAGENT or RESOURCE</th><th>SOURCE</th><th>IDENTIFIER</th></tr></thead><tbody><tr class="rowsep"><td colspan="3"><b>Antibodies</b></td></tr><tr><td>Mouse monoclonal anti-Troponin T Ab-1 (13-11)</td><td>Thermo Scientific</td><td>MA5-12960,&nbsp;RRID:<a id="interref275" href="https://antibodyregistry.org/AB_11000742" target="_blank">AB_11000742</a></td></tr><tr><td>Mouse monoclonal anti-Cardiac Troponin T 1C11</td><td>Abcam</td><td>ab8295, RRID:<a id="interref280" href="https://antibodyregistry.org/AB_306445" target="_blank">AB_306445</a></td></tr><tr><td>Goat polyclonal C-20 anti-TBX5</td><td>Santa Cruz</td><td>sc-17866, RRID:<a id="interref285" href="https://antibodyregistry.org/AB_2200827" target="_blank">AB_2200827</a></td></tr><tr><td>Goat polyclonal C-20 anti-GATA4</td><td>Santa Cruz</td><td>sc-1237, RRID:<a id="interref290" href="https://antibodyregistry.org/AB_2108747" target="_blank">AB_2108747</a></td></tr><tr><td>Mouse monoclonal A-6 anti-TBX5</td><td>Santa Cruz</td><td>sc-515536</td></tr><tr><td>Mouse monoclonal G-4 anti-GATA4</td><td>Santa Cruz</td><td>sc-25310</td></tr><tr><td>Goat polyclonal anti-NKX2-5</td><td>Santa Cruz</td><td>sc-8697, RRID:<a id="interref295" href="https://antibodyregistry.org/AB_650280" target="_blank">AB_650280</a></td></tr><tr><td>Goat polyclonal anti-Myc tag - ChIP Grade</td><td>Abcam</td><td>ab9132, RRID:<a id="interref300" href="https://antibodyregistry.org/AB_307033" target="_blank">AB_307033</a></td></tr><tr><td>Rabbit hGLYR1 (NDF) anti-sera 7</td><td>James T. Kadonaga Laboratory</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref38-5" href="#" role="doc-biblioref" data-xml-rid="bib38" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref38-5" aria-controls="sref38-5">Fei et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref38-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">35.</div><div>Fei, J. ∙ Ishii, H. ∙ Hoeksema, M.A. ...</div><div><strong>NDF, a nucleosome-destabilizing factor that facilitates transcription through nucleosomes</strong></div><div><em>Genes Dev.</em> 2018; <strong>32</strong>:682-694</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/gad.313973.118" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-85047604949" target="_blank" aria-label="Open Scopus in new tab">Scopus (24)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29759984/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2Fgad.313973.118&amp;pmid=29759984" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td></tr><tr><td>Rabbit polyclonal anti-HA tag - ChIP Grade</td><td>Abcam</td><td>ab9110, RRID:<a id="interref305" href="https://antibodyregistry.org/AB_307019" target="_blank">AB_307019</a></td></tr><tr><td>Rabbit polyclonal anti-H3K36me3– ChIP Grade</td><td>Abcam</td><td>ab9050, RRID:<a id="interref310" href="https://antibodyregistry.org/AB_306966" target="_blank">AB_306966</a></td></tr><tr><td>Rabbit polyclonal anti-MEIS1</td><td>Abcam</td><td>ab19867, RRID:<a id="interref315" href="https://antibodyregistry.org/AB_776272" target="_blank">AB_776272</a></td></tr><tr><td>Rabbit polyclonal anti-ISL1</td><td>Abcam</td><td>ab20670, RRID:<a id="interref320" href="https://antibodyregistry.org/AB_881306" target="_blank">AB_881306</a></td></tr><tr><td>Monoclonal anti-Vinculin VLN01</td><td>Thermo Fisher Scientific</td><td>MA5-11690, RRID:<a id="interref325" href="https://antibodyregistry.org/AB_10976821" target="_blank">AB_10976821</a></td></tr><tr><td>Rabbit Anti-LSD2 antibody</td><td>MyBioSource</td><td>MBS4751131</td></tr><tr><td>Mouse Anti-CDK9 antibody</td><td>Santa Cruz</td><td>sc-13130, RRID:<a id="interref330" href="https://antibodyregistry.org/AB_627245" target="_blank">AB_627245</a></td></tr><tr><td>Mouse Anti-Cyclin T1 antibody</td><td>Santa Cruz</td><td>sc-271348, RRID:<a id="interref335" href="https://antibodyregistry.org/AB_10608086" target="_blank">AB_10608086</a></td></tr><tr><td>Donkey anti-Goat Alexa Fluor 488</td><td>Thermo Fisher Scientific</td><td>A11055, RRID:<a id="interref340" href="https://antibodyregistry.org/AB_2534102" target="_blank">AB_2534102</a></td></tr><tr><td>Donkey anti-Mouse Alexa Fluor 647</td><td>Thermo Fisher Scientific</td><td>A-31571, RRID:<a id="interref345" href="https://antibodyregistry.org/AB_162542" target="_blank">AB_162542</a></td></tr><tr><td>Donkey Anti-Mouse Alexa Fluor 568</td><td>Thermo Fisher Scientific</td><td>A10037, RRID:<a id="interref350" href="https://antibodyregistry.org/AB_2534013" target="_blank">AB_2534013</a></td></tr><tr class="rowsep"><td>Donkey anti-Rabbit Alexa Fluor 488</td><td>Thermo Fisher Scientific</td><td>A21206, RRID:<a id="interref355" href="https://antibodyregistry.org/AB_2535792" target="_blank">AB_2535792</a></td></tr><tr class="rowsep"><td colspan="3"><b>Bacterial and virus strains</b></td></tr><tr><td>Ad-GFP-EF1-h-GLYR1</td><td>Vector Biolabs</td><td>NA</td></tr><tr><td>Ad-GFP-EF1-h-GLYR1 P496L</td><td>Vector Biolabs</td><td>NA</td></tr><tr class="rowsep"><td>Ad-EF1a-eGFP</td><td>Vector Biolabs</td><td>NA</td></tr><tr class="rowsep"><td colspan="3"><b>Chemicals, peptides, and recombinant proteins</b></td></tr><tr><td>SpCas9-NLS purified protein</td><td>QB3 Macrolab, UCB</td><td>NA</td></tr><tr><td>ROCK inhibitor Y-27632 2HCl</td><td>Selleckchem.com</td><td>S1049</td></tr><tr><td>hESC-qualified LDEV-free matrigel</td><td>Corning</td><td>354277</td></tr><tr><td>CHIR99021</td><td>Tocris</td><td>4423</td></tr><tr><td>IWP4</td><td>Tocris</td><td>5214</td></tr><tr><td>B27-supplemented (without insulin)</td><td>Life Technologies</td><td>A1895601</td></tr><tr><td>B27-supplemented (with insulin)</td><td>Life Technologies</td><td>A1895601</td></tr><tr><td>Fibronectin bovine plasma solution</td><td>Sigma-Aldrich</td><td>F1141</td></tr><tr><td>Lipofectamine RNAimax</td><td>Invitrogen</td><td>13778075</td></tr><tr><td>FuGENE HD</td><td>Promega</td><td>E2311</td></tr><tr><td>Benzonase</td><td>Millipore</td><td>E1014</td></tr><tr><td>DynabeadsTM Protein G</td><td>Invitrogen</td><td>10004D</td></tr><tr><td>Dynabeads™ Protein A/Protein G</td><td>Invitrogen</td><td>10015D</td></tr><tr><td>AMPure XP beads</td><td>Beckman Coulter</td><td>A63881</td></tr><tr><td>Dulbecco’s Modified Eagle Medium (DMEM), high glucose, GlutaMAX™ Supplement</td><td>Thermo Fisher Scientific</td><td>10566016</td></tr><tr><td>RPMI1640 media</td><td>Life Technologies</td><td>11875-119</td></tr><tr><td>Essential 8 medium (E8)</td><td>Life Technologies</td><td>A1517001</td></tr><tr><td>Novex 4-12% Tris-Glycine gels</td><td>Invitrogen</td><td>XP04122BOX</td></tr><tr><td>ECL Prime Western Blotting Detection Reagent</td><td>GELife Sciences</td><td>RPN2232</td></tr><tr><td>TRIzolTM LS reagent</td><td>Invitogen</td><td>10296010</td></tr><tr class="rowsep"><td>Opti-MEM</td><td>Thermo Fisher Scientific</td><td>31985070</td></tr><tr class="rowsep"><td colspan="3"><b>Critical commercial assays</b></td></tr><tr><td>Primary Cell Nucleofection P3 Kit</td><td>Lonza</td><td>V4XP-3960</td></tr><tr><td>Titanium Taq DNA Polymerase &amp; Titanium Buffer</td><td>Takara Bio</td><td>639209</td></tr><tr><td>Phusion® High-Fidelity PCR Master Mix with GC Buffer</td><td>New England BioLabs</td><td>M0532S</td></tr><tr><td>Cold Fusion Cloning kit</td><td>Systems Biosciences</td><td>MC101B-1</td></tr><tr><td>iBlot® Transfer Stack PVDF mini</td><td>ThermoScientific</td><td>IB4010-32</td></tr><tr><td>OMIX C18 pipette tips</td><td>Agilent</td><td>A57003100</td></tr><tr><td>Direct-Zol RNA kit</td><td>Zymo Research</td><td>R2052</td></tr><tr><td>SuperScriptTM III First-strand Synthesis SuperMix</td><td>Invitrogen</td><td>18080400</td></tr><tr><td>Taqman Universal PCR master mix</td><td>Life technologies</td><td>4305719</td></tr><tr><td>Ovation RNA-seq System V2 Kit</td><td>NuGEN</td><td>7102-08</td></tr><tr><td>Illumina Library Quantification Kit</td><td>KAPA Biosystems</td><td>KK4824</td></tr><tr><td>NEBNext Ultra II DNA Library Prep Kit for Illumina</td><td>New England BioLabs</td><td>E7645</td></tr><tr><td>NEBNext® Multiplex Oligos for Illumina® (Index Primers Set 1)</td><td>New England BioLabs</td><td>E7335</td></tr><tr><td>Dual Luciferase Assay System</td><td>Promega</td><td>E1960</td></tr><tr><td>Quick Start™ Bradford Protein Assay Kit 1</td><td>Bio-Rad</td><td>5000201</td></tr><tr><td>Chromium Next GEM Single Cell 3′ GEM, Library &amp; Gel Bead Kit v3.1</td><td>10X Genomics</td><td>PN-1000121</td></tr><tr><td>Chromium Next GEM Chip G Single Cell Kit</td><td>10X Genomics</td><td>PN-1000120</td></tr><tr class="rowsep"><td>3′v3.1 Single Index Kit</td><td>10X Genomics</td><td>PN-1000213</td></tr><tr class="rowsep"><td colspan="3"><b>Deposited data</b></td></tr><tr><td>RNaseq</td><td>GEO Database</td><td>GSE159411</td></tr><tr><td>Single-cell RNaseq</td><td>GEO Database</td><td>GSE159411</td></tr><tr><td>ChIPseq</td><td>GEO Database</td><td>GSE159411</td></tr><tr><td>Proteomics</td><td>PRIDE Database</td><td>PXD022091</td></tr><tr class="rowsep"><td>Code</td><td>GitHub</td><td><a id="interref0360" href="https://github.com/mepittman/ctf-apms" target="_blank">https://github.com/mepittman/ctf-apms</a></td></tr><tr class="rowsep"><td colspan="3"><b>Experimental models: Cell lines</b></td></tr><tr><td>WTC11 hiPS cell line</td><td>Gladstone Stem Cell Core (Coriell)</td><td>GM25256</td></tr><tr><td>TBX5-KO hiPS cell line</td><td>This paper</td><td>NA</td></tr><tr><td>GATA4-KO hiPS cell line</td><td>This paper</td><td>NA</td></tr><tr><td>GLYR1-KO hiPS cell line</td><td>This paper</td><td>NA</td></tr><tr><td>GLYR1-P496L hiPS cell line</td><td>This paper</td><td>NA</td></tr><tr><td>HeLa cell line</td><td>ATCC</td><td>ATCC® CCL-2™</td></tr><tr class="rowsep"><td>HEK293 cell line</td><td>ATCC</td><td>ATCC® CRL-1573™</td></tr><tr class="rowsep"><td colspan="3"><b>Experimental models: Organisms/strains</b></td></tr><tr class="rowsep"><td>GLYR1 P495L mouse line</td><td>This paper</td><td>NA</td></tr><tr class="rowsep"><td colspan="3"><b>Oligonucleotides</b></td></tr><tr><td>GATA4 sgRNA GAGGCCCACUCGGCGGGAGG</td><td>Synthego</td><td>NA</td></tr><tr><td>TBX5 sgRNA GCTTACCTTGTGGTTCTGGTAGG</td><td>Synthego</td><td>NA</td></tr><tr><td>CRISPR GATA4-KO FW primer 5′AGAGATCTCATGCAGGGTCG3′</td><td>This paper</td><td>NA</td></tr><tr><td>CRISPR GATA4-KO REV primer 5′TCATGATGCCTGGCCTTACT3′</td><td>This paper</td><td>NA</td></tr><tr><td>CRISPR TBX5-KO FW primer 5′GCAGAAACAGTTGCCCAGAA3′</td><td>This paper</td><td>NA</td></tr><tr><td>CRISPR TBX5-KO REV primer 5′CAAGGCGAATTTAGAGGGCG3′</td><td>This paper</td><td>NA</td></tr><tr><td>hGATA4 Cold Fusion cloning FW primer 5′TGGTGGATCCACCGGTATGTATCAGA<br>GCTTGGCCATGG3′</td><td>This paper</td><td>NA</td></tr><tr><td>hGATA4 Cold Fusion cloning REV primer 5′TGAGCGGCCGCGTTTAAACTTACGC<br>AGTGATTATGTCCCCGTG3′</td><td>This paper</td><td>NA</td></tr><tr><td>hGLYR1 sgRNA ATGTATTTCAGGTAGAAATCAGG</td><td>This paper</td><td>NA</td></tr><tr><td>hGLYR1-P496L HDR template: CCTCAGATATCCTGCAAGGAAACTT<br>TAAGCTTGATTTCTACCTGAAATACATTCAGAAGGA</td><td>This paper</td><td>NA</td></tr><tr><td>CRISPR GLYR1 FW Primer 5′CACCAGTGCACTCTAGCCT3′</td><td>This paper</td><td>NA</td></tr><tr><td>CRISPR GLYR1 RV Primer 5′TGCAGCAAATGAGGTAGGGT3′</td><td>This paper</td><td>NA</td></tr><tr><td>P495L Mouse Genotyping FW Primer FW 5′TTCCAGTCATTCCTTGCCCC3′</td><td>This paper</td><td>NA</td></tr><tr><td>P495L Mouse Genotyping FW Primer RV 5′TGATCAGAAGGGTCGGCAAG3′</td><td>This paper</td><td>NA</td></tr><tr><td>GATA4 Silencer Select Pre-designed SiRNA ID s535120</td><td>Thermo Fisher Scientific</td><td>4392420</td></tr><tr class="rowsep"><td>GLYR1 siRNA SASI_Hs01_00116796</td><td>Millipore-Sigma</td><td>NA</td></tr><tr class="rowsep"><td colspan="3"><b>Recombinant DNA</b></td></tr><tr><td>pCMV-T7-<i>hGLYR1</i>-MYC-IRES2-mCherry-pA</td><td>GeneCopoeia</td><td>EX-Z0806-M73</td></tr><tr><td>pCMV-T7-<i>hGLYR1</i> P496L-MYC-IRES2-mCherry-pA</td><td>This paper</td><td>NA</td></tr><tr><td>pEN563-pCAGG-eGFP-<i>GATA4</i></td><td>This paper</td><td>NA</td></tr><tr><td>pCMV-T7-<i>hSMARCC1</i>-MYC-IRES2-mCherry-pA</td><td>GeneCopoeia</td><td>EX-A6386-M73</td></tr><tr><td>pCMV-T7-<i>hSMARCC1</i>-W279G-MYC-IRES2-mCherry-pA</td><td>This paper</td><td>NA</td></tr><tr><td>pCMV-T7-<i>hSMARCC1</i>-M958T-MYC-IRES2-mCherry-pA</td><td>This paper</td><td>NA</td></tr><tr><td>p6344 pcDNA4-TO-HA-<i>Brd4FL</i></td><td><span class="dropBlock reference-citations"><a id="body-ref-sref109-1" href="#" role="doc-biblioref" data-xml-rid="bib109" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref109-1" aria-controls="sref109-1">Rahman et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref109-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">97.</div><div>Rahman, S. ∙ Sowa, M.E. ∙ Ottinger, M. ...</div><div><strong>The Brd4 extraterminal domain confers transcription activation independent of pTEFb by recruiting multiple proteins, including NSD3</strong></div><div><em>Mol. Cell. Biol.</em> 2011; <strong>31</strong>:2641-2652</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/MCB.01341-10" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_97_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-79959483607" target="_blank" aria-label="Open Scopus in new tab">Scopus (356)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21555454/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FMCB.01341-10&amp;pmid=21555454" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>Addgene-31351</td></tr><tr><td>p6344 pcDNA4-TO-HA-<i>Brd4FL</i>-S494L</td><td>This paper</td><td>NA</td></tr><tr><td>p6344 pcDNA4-TO-HA-<i>Brd4FL</i>-R616W</td><td>This paper</td><td>NA</td></tr><tr><td>pCIneo-<i>hCHD7</i>-Kozak ATG 3′ HA-bGH polyA</td><td>Peter Scacheri Lab</td><td>Addgene-89460</td></tr><tr><td>pCIneo-<i>hCHD7</i>-S2272R-Kozak ATG 3′ HA-bGH polyA</td><td>This paper</td><td>NA</td></tr><tr><td>pCIneo-<i>hCHD7</i>-L1745P-Kozak ATG 3′ HA-bGH polyA</td><td>This paper</td><td>NA</td></tr><tr><td>pCIneo-<i>hCHD7</i>-D2355N-Kozak ATG 3′ HA-bGH polyA</td><td>This paper</td><td>NA</td></tr><tr><td>pCIneo-<i>hCHD7</i>-R2111W-Kozak ATG 3′ HA-bGH polyA</td><td>This paper</td><td>NA</td></tr><tr><td>EX-Y4729-M06- h<i>CHD7</i>- del ATPase domain</td><td>Yuelong <span class="dropBlock reference-citations"><a id="body-ref-sref84-2" href="#" role="doc-biblioref" data-xml-rid="bib84" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref84-2" aria-controls="sref84-2">Liu et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref84-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">72.</div><div>Liu, Y. ∙ Harmelink, C. ∙ Peng, Y. ...</div><div><strong>CHD7 interacts with BMP R-SMADs to epigenetically regulate cardiogenesis in mice</strong></div><div><em>Hum. Mol. Genet.</em> 2014; <strong>23</strong>:2145-2156</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/hmg/ddt610" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-84897835537" target="_blank" aria-label="Open Scopus in new tab">Scopus (41)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24293546/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fhmg%2Fddt610&amp;pmid=24293546" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>GeneCopoeia and mod. by Kai Jiao Lab</td></tr><tr><td>pcDNA3.1-CMV-<i>NKX2-5-</i>3xFLAG-bGH polyA</td><td>This paper</td><td>NA</td></tr><tr><td>pcDNA3.1-CMV-<i>NKX2-5-</i>I184M-3xFLAG-bGH polyA</td><td>This paper</td><td>NA</td></tr><tr><td>pcDNA3.1-CMV-<i>NKX2-5</i>-A119S-3xFLAG-bGH polyA</td><td>This paper</td><td>NA</td></tr><tr><td>pANF638 Luc vector</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref69-1" href="#" role="doc-biblioref" data-xml-rid="bib69" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref69-1" aria-controls="sref69-1">Knowlton et&nbsp;al., 1991</a><div class="dropBlock__holder" data-db-target-of="sref69-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">62.</div><div>Knowlton, K.U. ∙ Baracchini, E. ∙ Ross, R.S. ...</div><div><strong>Co-regulation of the atrial natriuretic factor and cardiac myosin light chain-2 genes during alpha-adrenergic stimulation of neonatal rat ventricular cells. Identification of cis sequences within an embryonic and a constitutive contractile protein gene which mediate inducible expression</strong></div><div><em>J.&nbsp;Biol. Chem.</em> 1991; <strong>266</strong>:7759-7768</div></div><div class="external-links"><div class="core-xlink-abstract"><a href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=10.1016%2FS0021-9258%2820%2989515-5&amp;cf=abstract&amp;site=jbc-site" target="_blank" aria-label="Open Abstract in new tab">Abstract</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=10.1016%2FS0021-9258%2820%2989515-5&amp;cf=pdf&amp;site=jbc-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/1850419/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0021-9258%2820%2989515-5&amp;pmid=1850419" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>NA</td></tr><tr><td>pGL4.23[luc2/minP]</td><td>Promega</td><td>E8411</td></tr><tr><td>pGL4.23[luc2/minP] <i>Ppargc1a</i> Promoter</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref103-4" href="#" role="doc-biblioref" data-xml-rid="bib103" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref103-4" aria-controls="sref103-4">Padmanabhan et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref103-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">91.</div><div>Padmanabhan, A. ∙ Alexanian, M. ∙ Linares-Saldana, R. ...</div><div><strong>BRD4 (Bromodomain-Containing Protein 4) Interacts with GATA4 (GATA Binding Protein 4) to Govern Mitochondrial Homeostasis in Adult Cardiomyocytes</strong></div><div><em>Circulation.</em> 2020; <strong>142</strong>:2338-2355</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1161/CIRCULATIONAHA.120.047753" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-85097828347" target="_blank" aria-label="Open Scopus in new tab">Scopus (14)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33094644/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1161%2FCIRCULATIONAHA.120.047753&amp;pmid=33094644" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>NA</td></tr><tr><td>pGL4.23[luc2/minP] <i>GATA6</i> intronic RE</td><td>This paper</td><td>NA</td></tr><tr><td>pGL4.23[luc2/minP] <i>TTN</i> intronic RE</td><td>This paper</td><td>NA</td></tr><tr><td>pGL4.23[luc2/minP] <i>MYL4</i> intronic RE</td><td>This paper</td><td>NA</td></tr><tr><td>pRL-TK</td><td>Promega</td><td>E2241</td></tr><tr class="rowsep"><td>pRL-SV40</td><td>Promega</td><td>E2231</td></tr><tr class="rowsep"><td colspan="3"><b>Software and algorithms</b></td></tr><tr><td>PRISM v10 graphing and statistical software</td><td>GraphPad Software</td><td><a id="intref0020" href="http://www.graphpad.com/scientific-software/prism/" target="_blank">http://www.graphpad.com/scientific-software/prism/</a></td></tr><tr><td>Flowjo-v10</td><td>FlowJo LLC</td><td><a id="intref0025" href="http://www.flowjo.com" target="_blank">http://www.flowjo.com</a></td></tr><tr><td>Pulse automated analysis software</td><td>PULSE sofware</td><td><a id="intref0030" href="https://www.pulsevideoanalysis.com" target="_blank">https://www.pulsevideoanalysis.com</a></td></tr><tr><td>Cytoscape v3.7.1</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref122-1" href="#" role="doc-biblioref" data-xml-rid="bib122" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref122-1" aria-controls="sref122-1">Shannon et&nbsp;al., 2003</a><div class="dropBlock__holder" data-db-target-of="sref122-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">110.</div><div>Shannon, P. ∙ Markiel, A. ∙ Ozier, O. ...</div><div><strong>Cytoscape: a software environment for integrated models of biomolecular interaction networks</strong></div><div><em>Genome Res.</em> 2003; <strong>13</strong>:2498-2504</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/gr.1239303" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_110_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-0242490780" target="_blank" aria-label="Open Scopus in new tab">Scopus (23829)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/14597658/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2Fgr.1239303&amp;pmid=14597658" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0035" href="https://cytoscape.org" target="_blank">https://cytoscape.org</a></td></tr><tr><td>R v.3.6.1</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref108-1" href="#" role="doc-biblioref" data-xml-rid="bib108" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref108-1" aria-controls="sref108-1">R Core Team, 2020</a><div class="dropBlock__holder" data-db-target-of="sref108-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">96.</div><div>R Core Team</div><div><strong>R: A language and environment for statistical computing</strong></div><div>R Foundation for Statistical Computing, Vienna, Austria, 2020</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=R+Core+TeamR%3A+A+language+and+environment+for+statistical+computing2020R+Foundation+for+Statistical+ComputingVienna%2C+Austria" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0040" href="https://www.R-project.org/" target="_blank">https://www.R-project.org/</a></td></tr><tr><td>artMS v1.4.0</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref62-1" href="#" role="doc-biblioref" data-xml-rid="bib62" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref62-1" aria-controls="sref62-1">Jimenez-Morales et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref62-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">55.</div><div>Jimenez-Morales, D. ∙ Rosa Campos, A. ∙ Von Dollen, J. ...</div><div><strong>artMS: Analytical R tools for Mass Spectrometry version 1.6.5 from Bioconductor</strong></div><div><em>Bioconductor.</em> 2020;</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=D.Jimenez-MoralesA.Rosa+CamposJ.Von+DollenD.SwaneyartMS%3A+Analytical+R+tools+for+Mass+Spectrometry+version+1.6.5+from+BioconductorBioconductor2020" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0045" href="https://bioconductor.org/packages/release/bioc/html/artMS.html" target="_blank">https://bioconductor.org/packages/release/bioc/html/artMS.html</a></td></tr><tr><td>SAINTq v0.0.4</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref128-2" href="#" role="doc-biblioref" data-xml-rid="bib128" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref128-2" aria-controls="sref128-2">Teo et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref128-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">115.</div><div>Teo, G. ∙ Koh, H. ∙ Fermin, D. ...</div><div><strong>SAINTq: Scoring protein-protein interactions in affinity purification - mass spectrometry experiments with fragment or peptide intensity data</strong></div><div><em>Proteomics.</em> 2016; <strong>16</strong>:2238-2245</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/pmic.201500499" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_115_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-84983261035" target="_blank" aria-label="Open Scopus in new tab">Scopus (24)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27119218/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fpmic.201500499&amp;pmid=27119218" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0050" href="http://saint-apms.sourceforge.net/" target="_blank">http://saint-apms.sourceforge.net/</a></td></tr><tr><td>fastq-mcf</td><td>ea-utils 1.1.2-537</td><td><a id="intref0055" href="http://code.google.com/p/ea-utils" target="_blank">http://code.google.com/p/ea-utils</a></td></tr><tr><td>bowtie 2.2.4</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref75-1" href="#" role="doc-biblioref" data-xml-rid="bib75" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref75-1" aria-controls="sref75-1">Langmead and Salzberg, 2012</a><div class="dropBlock__holder" data-db-target-of="sref75-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">67.</div><div>Langmead, B. ∙ Salzberg, S.L.</div><div><strong>Fast gapped-read alignment with Bowtie 2</strong></div><div><em>Nat. Methods.</em> 2012; <strong>9</strong>:357-359</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.1923" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_67_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-84859210032" target="_blank" aria-label="Open Scopus in new tab">Scopus (24558)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22388286/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.1923&amp;pmid=22388286" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0060" href="https://sourceforge.net/projects/bowtie-bio/files/bowtie2/2.1.0/" target="_blank">https://sourceforge.net/projects/bowtie-bio/files/bowtie2/2.1.0/</a></td></tr><tr><td>GEM</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref51-1" href="#" role="doc-biblioref" data-xml-rid="bib51" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref51-1" aria-controls="sref51-1">Guo et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref51-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">46.</div><div>Guo, Y. ∙ Mahony, S. ∙ Gifford, D.K.</div><div><strong>High resolution genome wide binding event finding and motif discovery reveals transcription factor spatial binding constraints</strong></div><div><em>PLoS Comput. Biol.</em> 2012; <strong>8</strong>:e1002638</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pcbi.1002638" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-84866086842" target="_blank" aria-label="Open Scopus in new tab">Scopus (176)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22912568/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pcbi.1002638&amp;pmid=22912568" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0065" href="http://cgs.csail.mit.edu/gem" target="_blank">http://cgs.csail.mit.edu/gem</a></td></tr><tr><td>BCP</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref142-1" href="#" role="doc-biblioref" data-xml-rid="bib142" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref142-1" aria-controls="sref142-1">Xing et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref142-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">128.</div><div>Xing, H. ∙ Mo, Y. ∙ Liao, W. ...</div><div><strong>Genome-wide localization of protein-DNA binding and histone modification by a Bayesian change-point method with ChIP-seq data</strong></div><div><em>PLoS Comput. Biol.</em> 2012; <strong>8</strong>:e1002613</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pcbi.1002613" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_128_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-84864587283" target="_blank" aria-label="Open Scopus in new tab">Scopus (40)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22844240/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pcbi.1002613&amp;pmid=22844240" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0070" href="http://rulai.cshl.edu" target="_blank">http://rulai.cshl.edu</a></td></tr><tr><td>AMBER 18</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref19-1" href="#" role="doc-biblioref" data-xml-rid="bib19" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref19-1" aria-controls="sref19-1">Case et al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref19-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">19.</div><div>Case, D.A. ∙ Ben-Shalom, I.Y. ∙ Brozell, S.R. ...</div><div><strong>AMBER 2018</strong></div><div>University of California, San Francisco, 2018</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=D.A.CaseI.Y.Ben-ShalomS.R.BrozellD.S.CeruttiT.E.Cheatham%2C+IIIV.W.D.CruzeiroT.A.DardenR.E.DukeD.GhoreishiM.K.GilsonAMBER+20182018University+of+California%2C+San+Francisco" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0075" href="http://ambermd.org/" target="_blank">http://ambermd.org/</a></td></tr><tr><td>PyMOL, Molecular Graphics System, Version 1.2r3pre.</td><td>Schrödinger, LLC.</td><td><a id="intref0080" href="https://pymol.org/2/" target="_blank">https://pymol.org/2/</a></td></tr><tr><td>TrioDeNovo v0.06</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref139-1" href="#" role="doc-biblioref" data-xml-rid="bib139" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref139-1" aria-controls="sref139-1">Wei et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref139-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">126.</div><div>Wei, Q. ∙ Zhan, X. ∙ Zhong, X. ...</div><div><strong>A Bayesian framework for de novo mutation calling in parents-offspring trios</strong></div><div><em>Bioinformatics.</em> 2015; <strong>31</strong>:1375-1381</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btu839" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25535243/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtu839&amp;pmid=25535243" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0085" href="https://genome.sph.umich.edu/wiki/Triodenovo" target="_blank">https://genome.sph.umich.edu/wiki/Triodenovo</a></td></tr><tr><td>annovar</td><td><span class="dropBlock reference-citations"><a id="body-ref-optG9Ymi7TVGH" href="#" role="doc-biblioref" data-xml-rid="bib147" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="optG9Ymi7TVGH" aria-controls="optG9Ymi7TVGH">Wang et al., 2010</a><div class="dropBlock__holder" data-db-target-of="optG9Ymi7TVGH"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">123.</div><div>Wang, K. ∙ Li, M. ∙ Hakonarson, H.</div><div><strong>ANNOVAR: functional annotation of genetic variants from high-throughput sequencing data</strong></div><div><em>Nucleic Acids Res.</em> 2010; <strong>38</strong>:e164</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gkq603" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_123_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-77956534324" target="_blank" aria-label="Open Scopus in new tab">Scopus (7498)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20601685/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgkq603&amp;pmid=20601685" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0090" href="https://annovar.openbioinformatics.org/en/latest/" target="_blank">https://annovar.openbioinformatics.org/en/latest/</a></td></tr><tr><td>Seurat v4.0.1</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref52-1" href="#" role="doc-biblioref" data-xml-rid="bib52" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref52-1" aria-controls="sref52-1">Hao et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref52-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">47.</div><div>Hao, Y. ∙ Hao, S. ∙ Andersen-Nissen, E. ...</div><div><strong>Integrated analysis of multimodal single-cell data</strong></div><div><em>Cell.</em> 2021; <strong>184</strong>:3573-3587.e29</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=10.1016%2Fj.cell.2021.04.048&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=10.1016%2Fj.cell.2021.04.048&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-85108179646" target="_blank" aria-label="Open Scopus in new tab">Scopus (800)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/34062119/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2021.04.048&amp;pmid=34062119" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0095" href="https://satijalab.org/seurat/articles/install.html" target="_blank">https://satijalab.org/seurat/articles/install.html</a></td></tr><tr><td>harmony v1.0</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref72-1" href="#" role="doc-biblioref" data-xml-rid="bib72" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref72-1" aria-controls="sref72-1">Korsunsky et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref72-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">64.</div><div>Korsunsky, I. ∙ Millard, N. ∙ Fan, J. ...</div><div><strong>Fast, sensitive and accurate integration of single-cell data with Harmony</strong></div><div><em>Nat. Methods.</em> 2019; <strong>16</strong>:1289-1296</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41592-019-0619-0" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_64_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-85070698712" target="_blank" aria-label="Open Scopus in new tab">Scopus (806)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31740819/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41592-019-0619-0&amp;pmid=31740819" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0100" href="https://portals.broadinstitute.org/harmony/articles/quickstart.html" target="_blank">https://portals.broadinstitute.org/harmony/articles/quickstart.html</a></td></tr><tr><td>lme4 v1.1-26</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref10-1" href="#" role="doc-biblioref" data-xml-rid="bib10" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref10-1" aria-controls="sref10-1">Bates et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref10-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">10.</div><div>Bates, D. ∙ Mächler, M. ∙ Bolker, B. ...</div><div><strong>Fitting linear mixed-effects models using lme4</strong></div><div><em>J.&nbsp;Stat. Softw.</em> 2015; <strong>67</strong>:1-48</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.18637/jss.v067.i01" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-84943645306" target="_blank" aria-label="Open Scopus in new tab">Scopus (36021)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.18637%2Fjss.v067.i01" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0105" href="https://cran.r-project.org/web/packages/lme4/index.html" target="_blank">https://cran.r-project.org/web/packages/lme4/index.html</a></td></tr><tr><td>PRROC_1.3.1</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref49-1" href="#" role="doc-biblioref" data-xml-rid="bib49" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref49-1" aria-controls="sref49-1">Grau et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref49-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">44.</div><div>Grau, J. ∙ Grosse, I. ∙ Keilwagen, J.</div><div><strong>PRROC: computing and visualizing precision-recall and receiver operating characteristic curves in R</strong></div><div><em>Bioinformatics.</em> 2015; <strong>31</strong>:2595-2597</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btv153" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-84943599060" target="_blank" aria-label="Open Scopus in new tab">Scopus (170)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25810428/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtv153&amp;pmid=25810428" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0110" href="https://cran.r-project.org/web/packages/PRROC/index.html" target="_blank">https://cran.r-project.org/web/packages/PRROC/index.html</a></td></tr><tr><td>MAST v1.16</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref40-1" href="#" role="doc-biblioref" data-xml-rid="bib40" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref40-1" aria-controls="sref40-1">Finak et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref40-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">36.</div><div>Finak, G. ∙ McDavid, A. ∙ Yajima, M. ...</div><div><strong>MAST: a flexible statistical framework for assessing transcriptional changes and characterizing heterogeneity in single-cell RNA sequencing data</strong></div><div><em>Genome Biol.</em> 2015; <strong>16</strong>:278</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13059-015-0844-5" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-84951574149" target="_blank" aria-label="Open Scopus in new tab">Scopus (870)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26653891/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13059-015-0844-5&amp;pmid=26653891" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0115" href="https://www.bioconductor.org/packages/release/bioc/html/MAST.html" target="_blank">https://www.bioconductor.org/packages/release/bioc/html/MAST.html</a></td></tr><tr class="rowsep"><td>homer v4.11.1</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref53-1" href="#" role="doc-biblioref" data-xml-rid="bib53" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref53-1" aria-controls="sref53-1">Heinz et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref53-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">48.</div><div>Heinz, S. ∙ Benner, C. ∙ Spann, N. ...</div><div><strong>Simple combinations of lineage-determining transcription factors prime cis-regulatory elements required for macrophage and B cell identities</strong></div><div><em>Mol. Cell.</em> 2010; <strong>38</strong>:576-589</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_48_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=10.1016%2Fj.molcel.2010.05.004&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_48_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=10.1016%2Fj.molcel.2010.05.004&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_48_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-77952567987" target="_blank" aria-label="Open Scopus in new tab">Scopus (6049)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20513432/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.molcel.2010.05.004&amp;pmid=20513432" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0120" href="http://homer.ucsd.edu/homer/download.html" target="_blank">http://homer.ucsd.edu/homer/download.html</a></td></tr><tr class="rowsep"><td colspan="3"><b>Other</b></td></tr><tr><td>GTEx v8.1.1.9</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref11-1" href="#" role="doc-biblioref" data-xml-rid="bib11" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref11-1" aria-controls="sref11-1">GTEx Consortium, 2017</a><div class="dropBlock__holder" data-db-target-of="sref11-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">11.</div><div>GTEx Consortium</div><div><strong>Genetic effects on gene expression across human tissues</strong></div><div><em>Nature.</em> 2017; <strong>550</strong>:204-213</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature24277" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-85031313737" target="_blank" aria-label="Open Scopus in new tab">Scopus (1802)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29022597/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature24277&amp;pmid=29022597" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0125" href="https://www.gtexportal.org/home/datasets" target="_blank">https://www.gtexportal.org/home/datasets</a></td></tr><tr><td>iRefIndex v15.0</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref112-3" href="#" role="doc-biblioref" data-xml-rid="bib112" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref112-3" aria-controls="sref112-3">Razick et&nbsp;al., 2008</a><div class="dropBlock__holder" data-db-target-of="sref112-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">100.</div><div>Razick, S. ∙ Magklaras, G. ∙ Donaldson, I.M.</div><div><strong>iRefIndex: a consolidated protein interaction database with provenance</strong></div><div><em>BMC Bioinformatics.</em> 2008; <strong>9</strong>:405</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/1471-2105-9-405" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_100_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-54949127160" target="_blank" aria-label="Open Scopus in new tab">Scopus (405)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/18823568/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2F1471-2105-9-405&amp;pmid=18823568" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0130" href="https://irefindex.vib.be/wiki/index.php/iRefIndex" target="_blank">https://irefindex.vib.be/wiki/index.php/iRefIndex</a></td></tr><tr><td>STRING-db</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref126" href="#" role="doc-biblioref" data-xml-rid="bib126" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref126" aria-controls="sref126">Szklarczyk et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref126"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">114.</div><div>Szklarczyk, D. ∙ Gable, A.L. ∙ Lyon, D. ...</div><div><strong>STRING v11: protein-protein association networks with increased coverage, supporting functional discovery in genome-wide experimental datasets</strong></div><div><em>Nucleic Acids Res.</em> 2019; <strong>47</strong>:D607-D613</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gky1131" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_114_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-85059797047" target="_blank" aria-label="Open Scopus in new tab">Scopus (6765)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30476243/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgky1131&amp;pmid=30476243" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0135" href="https://string-db.org/" target="_blank">https://string-db.org/</a> Accessed April 22, 2021</td></tr><tr><td>BIOGRID</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref102" href="#" role="doc-biblioref" data-xml-rid="bib102" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref102" aria-controls="sref102">Oughtred et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref102"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">90.</div><div>Oughtred, R. ∙ Rust, J. ∙ Chang, C. ...</div><div><strong>The BioGRID database: A comprehensive biomedical resource of curated protein, genetic, and chemical interactions</strong></div><div><em>Protein Sci.</em> 2021; <strong>30</strong>:187-200</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/pro.3978" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_90_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-85096803916" target="_blank" aria-label="Open Scopus in new tab">Scopus (215)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33070389/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fpro.3978&amp;pmid=33070389" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0140" href="https://thebiogrid.org/" target="_blank">https://thebiogrid.org/</a> Accessed April 22, 2021</td></tr><tr><td>Human Protein Atlas v13</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref134-1" href="#" role="doc-biblioref" data-xml-rid="bib134" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref134-1" aria-controls="sref134-1">Uhlén et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref134-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">121.</div><div>Uhlén, M. ∙ Fagerberg, L. ∙ Hallström, B.M. ...</div><div><strong>Proteomics. Tissue-based map of the human proteome</strong></div><div><em>Science.</em> 2015; <strong>347</strong>:1260419</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.1260419" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25613900/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.1260419&amp;pmid=25613900" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0145" href="https://www.proteinatlas.org/humanproteome/tissue/tissue+specific" target="_blank">https://www.proteinatlas.org/humanproteome/tissue/tissue+specific</a></td></tr><tr><td>Combined Annotation Dependent Depletion (CADD) v1.6</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref113-2" href="#" role="doc-biblioref" data-xml-rid="bib113" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref113-2" aria-controls="sref113-2">Rentzsch et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref113-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">101.</div><div>Rentzsch, P. ∙ Witten, D. ∙ Cooper, G.M. ...</div><div><strong>CADD: predicting the deleteriousness of variants throughout the human genome</strong></div><div><em>Nucleic Acids Res.</em> 2019; <strong>47</strong>:D886-D894</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gky1016" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_101_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-85059798241" target="_blank" aria-label="Open Scopus in new tab">Scopus (1261)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30371827/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgky1016&amp;pmid=30371827" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0150" href="https://cadd.gs.washington.edu/score" target="_blank">https://cadd.gs.washington.edu/score</a></td></tr><tr><td>Haploinsufficiency prediction with imputation</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref59-1" href="#" role="doc-biblioref" data-xml-rid="bib59" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref59-1" aria-controls="sref59-1">Huang et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref59-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">53.</div><div>Huang, N. ∙ Lee, I. ∙ Marcotte, E.M. ...</div><div><strong>Characterising and predicting haploinsufficiency in the human genome</strong></div><div><em>PLoS Genet.</em> 2010; <strong>6</strong>:e1001154</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pgen.1001154" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-78449263023" target="_blank" aria-label="Open Scopus in new tab">Scopus (438)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20976243/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pgen.1001154&amp;pmid=20976243" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0155" href="https://doi.org/10.1371/journal.pgen.1001154.s002" target="_blank">https://doi.org/10.1371/journal.pgen.1001154.s002</a></td></tr><tr><td>gnomAD v2.1.1</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref66-1" href="#" role="doc-biblioref" data-xml-rid="bib66" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref66-1" aria-controls="sref66-1">Karczewski et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref66-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">59.</div><div>Karczewski, K.J. ∙ Francioli, L.C. ∙ Tiao, G. ..., Genome Aggregation Database Consortium</div><div><strong>The mutational constraint spectrum quantified from variation in 141,456 humans</strong></div><div><em>Nature.</em> 2020; <strong>581</strong>:434-443</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-020-2308-7" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-85085542423" target="_blank" aria-label="Open Scopus in new tab">Scopus (2715)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32461654/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-020-2308-7&amp;pmid=32461654" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0160" href="https://gnomad.broadinstitute.org/downloads#v2-constraint" target="_blank">https://gnomad.broadinstitute.org/downloads#v2-constraint</a></td></tr></tbody></table></div><figcaption><div class="core-table-tools"><ul><li><a class="core-table-show-action" href="/action/showFullTableHTML?isHtml=true&amp;tableId=undtbl1&amp;pii=S0092-8674%2822%2900079-4" target="_blank">Open table in a new tab</a></li></ul></div></figcaption></figure></div></div></section><section id="sec-4-2"><h3>Resource availability</h3><section id="sec-4-2-1"><h4>Lead contact</h4><div id="p0290" role="paragraph">Further information and requests for resources and reagents should be directed to and will be fulfilled by the Lead Contact, Deepak Srivastava (<a id="intref0165" href="mailto:Deepak.srivastava@gladstone.ucsf.edu">Deepak.srivastava@gladstone.ucsf.edu</a>).</div></section><section id="sec-4-2-2"><h4>Materials availability</h4><div id="p0295" role="paragraph">All resources and materials reported in this paper will be shared by the lead contact upon request.</div></section></section><section id="sec-4-3"><h3>Experimental model and subject details</h3><section id="sec-4-3-1"><h4>Cell lines</h4><div id="p0300" role="paragraph">HEK293 (human [<i>Homo sapiens</i>] fetal kidney) and HeLa (human [<i>Homo sapiens</i>] cervical cancer) cells were all obtained from ATCC (<a id="intref0170" href="https://www.atcc.org/" target="_blank">https://www.atcc.org/</a>). HEK293 and HeLa cells were cultured in Dulbecco’s Modified Eagle Medium (DMEM), high glucose, GlutaMAX Supplement (Cat.10566016, Thermo Fisher Scientific) supplemented with 10% fetal bovine serum, 2mM sodium pyruvate, 2mM non-essential amino acids and 100 I.U./mL penicillin and 100&nbsp;μg/mL streptomycin.</div><div id="p0305" role="paragraph">The WTC11 human iPSC line (Cat. GM25256, Coriell) was obtained from the Gladstone Stem Cell Core (<a id="intref0175" href="https://labs.gladstone.org/stem_cell" target="_blank">https://labs.gladstone.org/stem_cell</a>). The WTC11 line is used as a normal control by research groups all over the world. This hiPSC line was derived from a skin biopsy from a healthy adult Asian male donor in his early thirties, who showed normal function in a battery of tests. The original fibroblasts were reprogrammed using episomal methods with the following factors: LIN28A, MYC (c-MYC), POU5F1 (OCT4), and SOX2 and pluripotency state and differentiation potential characterized as previously described (<span class="dropBlock reference-citations"><a id="body-ref-sref95" href="#" role="doc-biblioref" data-xml-rid="bib95" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref95" aria-controls="sref95">Miyaoka et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref95"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">83.</div><div>Miyaoka, Y. ∙ Chan, A.H. ∙ Judge, L.M. ...</div><div><strong>Isolation of single-base genome-edited human iPS cells without antibiotic selection</strong></div><div><em>Nat. Methods.</em> 2014; <strong>11</strong>:291-293</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.2840" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_83_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-84895528015" target="_blank" aria-label="Open Scopus in new tab">Scopus (161)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24509632/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.2840&amp;pmid=24509632" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="p0310" role="paragraph">All hiPSC clones were recurrently verified free of mycoplasma contamination and checked for normal karyotype. Karyotyping analyses were performed by Cell Line Genetics.</div></section><section id="sec-4-3-2"><h4>Human iPS cell line generation by CRISPR-Cas9 editing</h4><div id="p0315" role="paragraph">To generate the <i>GATA4</i>, <i>TBX5</i> and <i>GLYR1</i> knockout lines as well as the <i>GLYR1</i><sup><i>P496L</i></sup> hiPSC line, WTC11 hiPSCs were dissociated in accutase (Cat. 07920, Stem Cell Technologies), and 250.000 cells were aliquoted per condition and nucleofected with Cas9-ribonucleoprotein complexes (Cas9-RNP) following the Primary Cell Nucleofection P3 Kit manufacturer’s instructions (V4XP-3960, Lonza). For Cas9-RNP complex preparation 180pmol of each synthetic modified sgRNA (Synthego) to target exon4 of <i>GATA4</i> (GAGGCCCACUCGGCGGGAGG), exon6 <i>TBX5</i> (GCTTACCTTGTGGTTCTGGTAGG) or the <i>GLYR1</i> locus at chr16:4811270-4811248 (ATGTATTTCAGGTAGAAATCAGG) and 20pmol of SpCas9-NLS purified protein (QB3 Macrolab, UCB) were diluted into 20&nbsp;μL of nucleofection buffer prepared as indicated in Primary Cell Nucleofection P3 Kit, mixed and incubated at room temperature for 10&nbsp;min. After 10&nbsp;min. of incubation, the aliquoted 250.000 cells were resuspended in 20&nbsp;μL nucleofection buffer containing the corresponding Cas9-RNP complexes, mixed 5-6 times and then transferred to the bottom of a nucleofector well (nucleofector 96 well cassette from the Primary Cell Nucleofection P3 Kit). For the generation of the <i>GLYR1</i><sup><i>P496L</i></sup> line, the HDR template (12nM): CCTCAGATATCCTGCAAGGAAACTTTAAGCTTGATTTCTACCTGAAATACATTCAGAAGGA, was added to the Cas9-RNP complexes, prior to transferring into the nucleofector wells. This was repeated for every Cas9-RNP condition. The nucleofector cassette was placed in the nucleofector instrument (Nucleofector 4D system, Lonza) and cells nucleofector using the preset program DS-138. The nucleofection cassette was brought back to sterile hood and 80&nbsp;μL of Essential 8 medium (E8) (Cat. A1517001, Life Technologies) with 5&nbsp;μM rho kinase (ROCK) inhibitor (Y-27632 2HCl, Cat. S1049, <a id="intref0180" href="http://Selleckchem.com" target="_blank">Selleckchem.com</a>) added into each well and incubated for 10&nbsp;min. at 37°C. During the incubation time, we removed the hESC-qualified LDEV-free matrigel (Cat. 354277, Corning) from 12 well plates (Corning) pre-coated for 1 h. at 37°C with 0.5ml per well of matrigel and added 2mL of pre-warmed E8 medium plus 5&nbsp;μM ROCK inhibitor (Y-27632 2HCl) in each well. Then pipetted the nucleofected cells to each of the pre-coted wells with media. After ∼3-5&nbsp;days, wells with surviving clones were expanded to isolate gDNA for screening. A genomic fragment spanning the gRNA target sites was amplified using primers FW 5′AGAGATCTCATGCAGGGTCG3′ and REV 5′TCATGATGCCTGGCCTTACT3′ for GATA4 with Titanium Taq DNA Polymerase (Cat. 639209, Takara Bio) and primers FW 5′GCAGAAACAGTTGCCCAGAA3′ and REV 5′CAAGGCGAATTTAGAGGGCG3′ for TBX5 or FW 5′ CACCAGTGCACTCTAGCCT 3′ and REV 5′ TGCAGCAAATGAGGTAGGGT 3′ for GLYR1 with Phusion® High-Fidelity PCR Master Mix with GC Buffer (Cat. M0532S, New England BioLabs) and Sanger sequenced (Quintara Biosciences or MCLAB) to identify clones with frameshift insertions/deletions. Synthego ICE analysis was&nbsp;run to identify clones with highest knock-out efficiency (<a id="intref0185" href="https://ice.synthego.com/#/" target="_blank">https://ice.synthego.com/#/</a>), which were subsequently subjected to&nbsp;the colony picking and clone sequencing until monoclonal lines were generated. The top 5 sgRNA predicted off-targets (<a id="intref0190" href="https://horizondiscovery.com/en/products/tools/crispr-specificity-analysis" target="_blank">https://horizondiscovery.com/en/products/tools/crispr-specificity-analysis</a>) were verified to be intact by sanger sequencing in the final monoclonal lines and checked for normal karyotype (Cell Line Genetics).</div></section><section id="sec-4-3-3"><h4>Human iPSC culture</h4><div id="p0320" role="paragraph">Human iPSCs were maintained on tissue culture-treated polystyrene plates (Cat. 430630, Corning) with hESC-qualified LDEV-free&nbsp;matrigel (Cat. 354277, Corning) in Essential 8 medium (E8) (Cat. A1517001, Life Technologies). The medium was changed daily&nbsp;and the hiPSCs were split every 4 – 6&nbsp;days using Accutase (Cat. 07920, Stem Cell Technologies). The rho kinase (ROCK) inhibitor&nbsp;(Y-27632 2HCl, Cat. S1049, <a id="intref0195" href="http://Selleckchem.com" target="_blank">Selleckchem.com</a>) was included in the medium at 5&nbsp;μM final concentration on the day of passaging.</div></section><section id="sec-4-3-4"><h4>Induced cardiomyocyte differentiation</h4><div id="p0325" role="paragraph">For human cardiac differentiation into CPs and CMs, we modified the protocols originally developed by Lian et&nbsp;al. and Tohyama et&nbsp;al. to achieve stage-specific, high yield, high-purity cardiac commitment <i>in&nbsp;vitro</i> (<span class="dropBlock reference-citations"><a id="body-ref-sref82" href="#" role="doc-biblioref" data-xml-rid="bib82" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref82" aria-controls="sref82">Lian et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref82"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">70.</div><div>Lian, X. ∙ Zhang, J. ∙ Azarin, S.M. ...</div><div><strong>Directed cardiomyocyte differentiation from human pluripotent stem cells by modulating Wnt/β-catenin signaling under fully defined conditions</strong></div><div><em>Nat. Protoc.</em> 2013; <strong>8</strong>:162-175</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nprot.2012.150" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-84872016692" target="_blank" aria-label="Open Scopus in new tab">Scopus (941)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23257984/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnprot.2012.150&amp;pmid=23257984" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref132" href="#" role="doc-biblioref" data-xml-rid="bib132" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref132" aria-controls="sref132">Tohyama et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref132"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">119.</div><div>Tohyama, S. ∙ Hattori, F. ∙ Sano, M. ...</div><div><strong>Distinct metabolic flow enables large-scale purification of mouse and human pluripotent stem cell-derived cardiomyocytes</strong></div><div><em>Cell Stem Cell.</em> 2013; <strong>12</strong>:127-137</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_119_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=10.1016%2Fj.stem.2012.09.013&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_119_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=10.1016%2Fj.stem.2012.09.013&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_119_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-84872048038" target="_blank" aria-label="Open Scopus in new tab">Scopus (591)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23168164/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.stem.2012.09.013&amp;pmid=23168164" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Briefly hiPSCs were detached from hESC-qualified LDEV-free matrigel (Cat. 354277, Corning) with accutase (Cat. 07920, Stem Cell Technologies) and reseeded on matrigel at 0.6–1.2x10<sup>5</sup> cells per 12well in Essential 8 medium (E8) (Cat. A1517001, Life Technologies) with 5&nbsp;μM ROCK inhibitor (Y-27632 2HCl, Cat. S1049, <a id="intref0200" href="http://Selleckchem.com" target="_blank">Selleckchem.com</a>) (day-3). We optimized each hiPSC clone individually to identify the best cell seeding density that resulted in high levels of cTNT, NKX2-5, TBX5 and GATA4-positive sheet-like beating CMs at day15 of differentiation. On the next two days, media was changed daily with fresh E8 medium without ROCK inhibitor. On the day of cardiac induction (day0), 6&nbsp;μM CHIR99021 (Cat. 4423, Tocris) was added for 24 h. in 1ml per well of B27-supplemented (without insulin) RPMI1640 media (Cat. 11875-119, Life Technologies). On day1 media was changed for freshly prepared 6&nbsp;μM CHIR99021 in B27-supplemented (without insulin) (Cat. A1895601, Life Technologies) RPMI1640 media. At day 2 and 3 IWP4 (Cat. 5214, Tocris) in B27-supplemented (without insulin) RPMI1640 media was added to activate Wnt signaling at a final concentration of 5&nbsp;μM. At day 4, IWP4 was removed, and 1ml of B27-supplemented (without insulin) RPMI1640 media per was daily added per well (days 4-9). At day 10 the media was changed to regular B27-supplemented (with insulin) (Cat. A1895601, Life Technologies) RPMI1640 hereafter. Typically, in parallel differentiations under identical conditions, WT cells started spontaneous contraction as early as days 7-8 while <i>TBX5-KO</i> and <i>GATA4-KO</i> lines tend to be delayed by 48-96 h., and <i>GLYR1</i><sup><i>P496L</i></sup> line showed a delayed beating onset of 24-48 h. To ensure consistency across the differentiations used for experimental proposes, from each differentiation cells were collected at day6 and day15-18 for cTNT, TBX5, NKX2-5 and GATA4 FACS analysis. WT hiPSC-CP samples used for AP-MS corresponded to differentiations with sheet-like beating cardiomyocytes and a minimum of 75% cTNT<sup>+</sup>, 70% TBX5<sup>+</sup>, 70% GATA4<sup>+</sup>, 70% NKX2-5<sup>+</sup> cells analyzed by FACS at day15 of differentiation. GATA4-KO and TBX5-KO CP samples corresponded to differentiations with sheet-like beating cardiomyocytes, were TBX5 or GATA4 protein absence had been confirmed, and had a minimum of 40% cTNT positive cells at day15 of differentiation. Similarly, <i>GLYR1</i><sup><i>P496L</i></sup> samples corresponded to differentiations with sheet-like beating cardiomyocytes and had a minimum of 40% cTNT positive cells.</div></section><section id="sec-4-3-5"><h4>Adenovirus</h4><div id="p0330" role="paragraph">Adenoviral – Human Type 5 (dE1/E3) viral particles expressing GLYR1 wild-type (Ad-GFP-EF1-h-GLYR1; PFU titer: 1x10<sup>10</sup> PFU/mL), GLYR1 P496L mutant (Ad-GFP-EF1-h-GLYR1 P496L PFU titer: 2x10<sup>9</sup> PFU/mL) or negative control (Ad-EF1a-eGFP; PFU titer: 1.2x10<sup>10</sup> PFU/mL) viral particles, were obtained from Vector Biolabs.</div></section><section id="sec-4-3-6"><h4><i>Glyr1</i> P495L mouse generation</h4><div id="p0335" role="paragraph">Mice were generated by blastocyst injection of ribonucleoprotein (RNP) complexes consisting of purified Cas9 protein (IDT) and guide RNAs targeting the GLYR1 locus along with a single-stranded oligonucleotide DNA template for homology-directed repair (IDT) that leads to insertion of the P495L mutation along with introduction of a novel BsrI restriction endonuclease site. The single stranded guide RNA (ATGTATTTCAGGTAGAAGTC) was used to target the Glyr1 locus together with the HDR template with the <i>Glyr1</i> P495L substitution indicated below. Blastocysts were transferred to pseudo-pregnant females and pups were weaned at approximately 4&nbsp;weeks of age. Founder animals were screened for introduction of the P495L mutation by PCR amplification and restriction digestion with BsrI and confirmed by sequencing. Founder animals were outcrossed to wild types and gremlin transmission of the P495L mutation was confirmed PCR/restriction digestion and sequencing. Animals were outcrossed to C57BL6 wild type mice for 4 generations. A genomic fragment spanning the gRNA target sites was amplified using primers FW 5′TTCCAGTCATTCCTTGCCCC3′ and REV 5′TGATCAGAAGGGTCGGCAAG3′ for <i>Glyr1</i> with Phusion® High-Fidelity PCR Master Mix with GC Buffer (Cat. M0532S, New England BioLabs) and Sanger sequenced (Quintara Biosciences or MCLAB) for genotyping.</div><div id="p0340" role="paragraph">gRNA<sup>+</sup>5: ATGTATTTCAGGTAGAAGTC</div><div id="p0345" role="paragraph">HDR template: AGGTGAGCCTGATACTCGGCGGGCAATTTTCATGTAGATCTTTTAAACTTCTAATGAATGGCTTTCCCTTCTCAGATATCCTACAAGGAAACTTTAAACTGGACTTCTACCTGAAATACATTCAGAAGGATCTCCGCCTCGCCATTGCATTGGGTGATGCAGTCAACCACCCCACTCCCATGGCAGCTGCAGCCAATGAG</div></section></section><section id="sec-4-4"><h3>Method details</h3><section id="sec-4-4-1"><h4>Immunocytochemistry</h4><div id="p0350" role="paragraph">Media was removed from day 15 CMs and fixed on 12 well plates by adding 1ml 4% formaldehyde (Cat. 28906, ThermoScientific) followed by a 15&nbsp;min. incubation at room temperature (RT). After the incubation, fixed cells were washed 3 times with 1ml of PBS (without Ca<sup>2+</sup> and Mg2+) and stored at 4°C in 1ml of PBS until processed for immunostaining. At first, cells were permeabilized for 45&nbsp;min. at RT in 1ml of permeabilization/blocking buffer (5% donkey serum, 0.2% triton-X, PBS 1x). CMs were stained overnight at 4°C on gentle agitation with the following indicated primary antibodies: mouse monoclonal anti-Troponin T, Cardiac Isoform Ab-1 Clone 13-11 (REF MS-295-P, Thermo Scientific) (1:100) together with goat polyclonal C-20 anti-TBX5 (Cat. Sc-17866, Santa Cruz) (1:50) or goat polyclonal C-20 anti-GATA4 (Cat. Sc-1237, Santa Cruz) (1:50) or anti-GLYR1 (14833-1-AP, Proteintech Group) (1:50) diluted in permeabilization/blocking buffer. The next day wells were washed 3 times with 1ml of PBS per well followed by a 10&nbsp;min. incubation on gentle agitation at RT. Cells were stained with Donkey Anti-Mouse Alexa Fluor 647 (Cat. A-31571, Thermo Fisher Scientific) and Donkey Anti-Goat or -Rabbit Alexa Fluor 488 (Cat. A11055 and A21206, Thermo Fisher Scientific) secondary fluorophore conjugated antibodies (1:1000) for 45&nbsp;min. on gentle agitation at RT, protected from light exposure. After the incubation, cells were washed 3 times with 1ml of PBS per well followed by a 10&nbsp;min. incubation on gentle agitation at RT. Immunostained samples were counterstained with DAPI (Cat. 422801, BioLegend) and visualized in a Zeiss Z1 microscope and associated ZEN software.</div></section><section id="sec-4-4-2"><h4>Flow cytometry</h4><div id="p0355" role="paragraph">At CP (day6) or CM (day 15-18) stages of differentiation cells were dissociated in accutase (Cat. 07920, Stem Cell Technologies), and fixed in 1.5ml Eppendorf tubes with 1ml of 4% formaldehyde (Cat. 28906, ThermoScientific) for 15&nbsp;min. on rotation at RT. After the incubation, fixed cells were washed 3 times with 1ml of PBS (without Ca<sup>2+</sup> and Mg2+) followed by centrifugation at 1700&nbsp;rpm for 3&nbsp;min and stored at 4°C in 1ml of PBS until processed for staining. For staining, cells were pelleted by centrifugation at 1700rpm 3min and PBS removed. Cell pellets were resuspended in 200&nbsp;μL of FACS buffer [1x PBS, 5% BSA, 5mM EDTA, 0.25% triton-X] and incubated at RT for 1 h. The permeabilized cells were then pelleted resuspended in 30&nbsp;μL of staining-A in FACS buffer: mouse monoclonal anti-Cardiac Troponin T [1C11] (1:100) (Cat. Ab8295, Abcam) and goat polyclonal anti-NKX2-5 (1:50) (Cat. Sc-8697, Santa Cruz); or staining-B: mouse monoclonal A-6 anti-TBX5 (1:50) (Cat. Sc-515536, Santa Cruz) and goat polyclonal C-20 anti-GATA4 (1:50) (Cat. Sc-1237, Santa Cruz) and incubated at RT for 1 h. Cells were then washed/spined down 3 times with 200&nbsp;μL of FACS buffer, resuspended in 30&nbsp;μL of FACS buffer with the secondary antibodies diluted 1:1000: Donkey Anti-Mouse Alexa Fluor 647 or 568 (Cat. A-31571 or A10037, Thermo Fisher Scientific) and Donkey Anti-Goat 488 Alexa Fluor (Cat. A11055, Thermo Fisher Scientific) and incubated for 1&nbsp;h. protected from the light at RT. The stained cells were washed/spined down 3 times with 200&nbsp;μL of FACS buffer and finally resuspended in 200&nbsp;μL of FACS buffer until analyzed. FACS stained samples were measured using FACSCalibur (BD Biosciences) or LSRII (BD Biosciences) and further analyzed using Flowjo software (<a id="intref0205" href="https://www.flowjo.com/" target="_blank">https://www.flowjo.com/</a>, FlowJo LLC).</div></section><section id="sec-4-4-3"><h4>Cardiomyocyte beating characterization</h4><div id="p0360" role="paragraph">WT, <i>TBX5-KO</i>, <i>GATA4-KO</i> and <i>GLYR1</i><sup><i>P496L</i></sup> CMs contractility beat rate parameters were measured from brightfield acquired videos with the Pulse automated analysis software (<a id="intref0210" href="https://www.pulsevideoanalysis.com" target="_blank">https://www.pulsevideoanalysis.com</a>). Pulse applies patented computer vision algorithms to measure beating signals and their parameters from videos of beating cardiomyocytes. It uses deep learning for detection of noisy signals, enabling robust and accurate measurement of parameters of beating frequency, beat duration and amplitude. Specifically, it captures and quantifies the biomechanical beating of cardiomyocytes by performing motion analysis on the image sequence to capture changes in the image intensity due to cardiomyocyte contraction and relaxation. This platform has been previously validated in hiPSC-derived cardiomyocytes and showed high correlation with calcium transients and patch-clamp experimental outcomes (<span class="dropBlock reference-citations"><a id="body-ref-sref87" href="#" role="doc-biblioref" data-xml-rid="bib87" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref87" aria-controls="sref87">Maddah et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref87"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">75.</div><div>Maddah, M. ∙ Heidmann, J.D. ∙ Mandegar, M.A. ...</div><div><strong>A non-invasive platform for functional characterization of stem-cell-derived cardiomyocytes with applications in cardiotoxicity testing</strong></div><div><em>Stem Cell Reports.</em> 2015; <strong>4</strong>:621-631</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_75_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=10.1016%2Fj.stemcr.2015.02.007&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_75_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=10.1016%2Fj.stemcr.2015.02.007&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_75_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-84933672526" target="_blank" aria-label="Open Scopus in new tab">Scopus (87)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25801505/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.stemcr.2015.02.007&amp;pmid=25801505" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref17" href="#" role="doc-biblioref" data-xml-rid="bib17" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref17" aria-controls="sref17">Burridge et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref17"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">17.</div><div>Burridge, P.W. ∙ Li, Y.F. ∙ Matsa, E. ...</div><div><strong>Human induced pluripotent stem cell-derived cardiomyocytes recapitulate the predilection of breast cancer patients to doxorubicin-induced cardiotoxicity</strong></div><div><em>Nat. Med.</em> 2016; <strong>22</strong>:547-556</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nm.4087" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-84964409142" target="_blank" aria-label="Open Scopus in new tab">Scopus (424)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27089514/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnm.4087&amp;pmid=27089514" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The onset of beating was determined by careful visual daily examination of multiple independent differentiations.</div></section><section id="sec-4-4-4"><h4>Plasmid generation: Cloning and mutagenesis</h4><div id="p0365" role="paragraph">The GFP-GATA4 plasmid (pEN563-pCAGG-eGFP-GATA4) was generated by PCR amplification of the <i>hGATA4</i> ORF from the precursor vector pAAV2.1CAG-hGATA4 (FW primer: TGGTGGATCCACCGGTATGTATCAGAGCTTGGCCATGG; REV primer: TGAGCGGCCGCGTTTAAACTTACGCAGTGATTATGTCCCCGTG) and cloned following the Cold Fusion Cloning kit (Cat. MC101B-1, Systems Biosciences). Followed manufacturer’s recommendations to linearize the pEN563-CAGG-eGFP vector with PmeI and AgeI (Cat. R0560S and R0552S, New England BioLabs) enzymes to replace the previous contained ORF for the <i>hGATA4</i>. The hGLYR1-MYC and hSMARCC1-MYC plasmids (pCMV-T7-cDNA-MYC-IRES2-mCherry-pA; Cat. EX-Z0806-M73 and EX-A6386-M73) were obtained from GeneCopoeia™. The HA-hBRD4 plasmid (p6344 pcDNA4-TO-HA-Brd4FL) was a gift from Peter Howley (Addgene plasmid # 31351; <a id="intref0215" href="http://n2t.net/addgene:31351" target="_blank">http://n2t.net/addgene:31351</a>; RRID: Addgene_31351) (<span class="dropBlock reference-citations"><a id="body-ref-sref109-2" href="#" role="doc-biblioref" data-xml-rid="bib109" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref109-2" aria-controls="sref109-2">Rahman et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref109-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">97.</div><div>Rahman, S. ∙ Sowa, M.E. ∙ Ottinger, M. ...</div><div><strong>The Brd4 extraterminal domain confers transcription activation independent of pTEFb by recruiting multiple proteins, including NSD3</strong></div><div><em>Mol. Cell. Biol.</em> 2011; <strong>31</strong>:2641-2652</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/MCB.01341-10" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_97_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-79959483607" target="_blank" aria-label="Open Scopus in new tab">Scopus (356)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21555454/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FMCB.01341-10&amp;pmid=21555454" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and pCIneo-hCHD7-Kozak ATG 3′ HA-bGH polyA was a gift from Peter Scacheri (Addgene plasmid # 89460; <a id="intref0220" href="http://n2t.net/addgene:89460" target="_blank">http://n2t.net/addgene:89460</a> ; RRID:Addgene_89460). The NKX2-5-3xFlag plasmid was generated by modifying the pcDNA3.1 backbone, where the human <i>NKX2-5</i> ORF with a C terminus fused 3xFLAG. The allele variants for <i>NKX2-5</i>, <i>CHD7</i>, <i>BRD4</i>, <i>SMARCC1</i> and <i>GLYR1</i> were generated by QuikChange II XL Site Directed Mutagenesis (Cat. 200521-5, Agilent Technologies) of the aforementioned plasmids. The EX-Y4729-M06- CHD7- del ATPase domain plasmid was obtained from Kai Jiao laboratory (<span class="dropBlock reference-citations"><a id="body-ref-sref84-3" href="#" role="doc-biblioref" data-xml-rid="bib84" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref84-3" aria-controls="sref84-3">Liu et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref84-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">72.</div><div>Liu, Y. ∙ Harmelink, C. ∙ Peng, Y. ...</div><div><strong>CHD7 interacts with BMP R-SMADs to epigenetically regulate cardiogenesis in mice</strong></div><div><em>Hum. Mol. Genet.</em> 2014; <strong>23</strong>:2145-2156</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/hmg/ddt610" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-84897835537" target="_blank" aria-label="Open Scopus in new tab">Scopus (41)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24293546/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fhmg%2Fddt610&amp;pmid=24293546" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div></section><section id="sec-4-4-5"><h4>Nuclear enriched lysis, immunoprecipitation, and immunoblotting</h4><div id="p0370" role="paragraph">WT, GATA4-KO and TBX5-KO hIPSC derived CPs or HEK293 cells transfected with the indicated expression vectors following manufacturer instructions (FuGene HD, Promega) were lysed in Cell Lysis buffer [20mM Tris-HCl pH 8, 85mM KCl, 0.5% NP-40, freshly added protease and phosphatase inhibitors (Cat. 4693132001 Roche and Cat. 4906837001 Sigma-Aldrich)] and incubated on rotator at 4°C for 10&nbsp;min. Cells were spun down at 2500&nbsp;rpm for 5min. at 4°C to pellet nuclei and supernatants containing the cytosolic fraction were removed and stored for quality control proposes. Nuclei were resuspended in Nuclear Extraction Buffer (NEB) [20&nbsp;mM HEPES, pH 7.4, 0.5&nbsp;M NaCl, 2&nbsp;mM MgCl2, 1&nbsp;mM CaCl2, 0.5% NP-40, K-Acetate 110mM, 1&nbsp;μM ZnCl2, and freshly added Benzonase 2&nbsp;μL enzyme/mL buffer (Cat. E1014, Millipore), protease and phosphatase inhibitors] and incubated on rotation for 30&nbsp;min. at 4°C. For CPs, 600&nbsp;μL NEB were used per 4x 12 well plates of initial CPs (∼100x10<sup>6</sup>), whereas the nuclei resulting from one 10&nbsp;mm confluent HEK293 plate were lysed in 300&nbsp;μL of NEB. After incubation, samples were centrifuged at max speed for 10&nbsp;min. and the nuclear enriched lysates (supernatants) were moved to clean tubes. Nuclear lysates were then diluted 1:3 in Nuclear Dilution Buffer (NDB) [20&nbsp;mM HEPES, pH 7.9, 1&nbsp;mM EDTA, 0.2% NP-40, freshly added protease and phosphatase inhibitors], 1200&nbsp;μL of NDB for CPs; 600&nbsp;μL NDB for HEK293. At this step samples can be stored and later used for immunoprecipitation or for western blotting.</div><div id="p0375" role="paragraph">The immunoprecipitations (IP) were done as described before (<span class="dropBlock reference-citations"><a id="body-ref-sref47" href="#" role="doc-biblioref" data-xml-rid="bib47" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref47" aria-controls="sref47">González-Terán et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref47"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">43.</div><div>González-Terán, B. ∙ López, J.A. ∙ Rodríguez, E. ...</div><div><strong>p38γ and δ promote heart hypertrophy by targeting the mTOR-inhibitory protein DEPTOR for degradation</strong></div><div><em>Nat. Commun.</em> 2016; <strong>7</strong>:10477</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ncomms10477" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-84955456248" target="_blank" aria-label="Open Scopus in new tab">Scopus (47)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26795633/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fncomms10477&amp;pmid=26795633" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) with modifications. Briefly, 50&nbsp;μL of Dynabeads™ Protein G magnetic (Cat. 10004D, Invitrogen) per IP sample were washed twice with 1ml PBS using a magnetic stand, and resuspended in an equivalent volume to the original volume of beads of PBS. Magnetic beads were then conjugated with a primary antibody: per 50&nbsp;μL of magnetic Dynabeads 4&nbsp;μg of mouse anti- GATA-4 Antibody (G-4) (Cat. Sc-25310 X, Santa Cruz), 4&nbsp;μg mouse anti- TBX5 Antibody (A-6) (Cat. Sc-515536 X, Santa Cruz), 2&nbsp;μg of anti-Myc tag antibody – ChIP Grade (Cat. Ab9132, Abcam) or 2&nbsp;μL of GLYR1 anti-sera 7 (James T. Kadonaga Laboratory) were added and incubated on rotation for 1 h. at 4°C. Then, the extra non-conjugated antibody was removed by washing the incubated Dynabeads with 1ml of PBS and twice with 1ml of Nuclear Dilution Buffer (NDB) using the magnetic stand, and resuspended in the initial volume of NDB. At this point the coated Dynabeads are ready for IP. Protein content was quantified by Quick Start Bradford Protein Assay Kit 1 (Cat. 5000201, BioRad) from nuclear enriched lysates and 1mg (immunoblot) or 3mg (mass spectrometry) of total protein per IP of endogenous proteins or 150&nbsp;μg (immunoblot) or 1mg (mass spectrometry) per IP of ectopically expressed proteins were aliquoted per condition and volumes were equalized with NDB. Prior to adding 50&nbsp;μL of coated beads per IP sample, 50&nbsp;μL of the prepared nuclear enriched lysates were set aside as “inputs.” IP samples were incubated with coated Dynabeads 4 h. for endogenous IP or 1 h. for IP of overexpressed proteins, on rotator at 4°C. After incubation, samples were placed in the magnetic stand and supernatants removed and saved as “unbound-fraction” for quality control proposes. Beads were washed 2x with 1ml of NDB, and 3 times with 1ml of NDB buffer without detergent. After the last wash, beads were spun down and tubes put on the magnet to remove the remaining liquid. At this point the IP samples can be processed for mass spectrometry or subjected to immunoblotting.</div><div id="p0380" role="paragraph">For immunoblotting, samples were subjected to PAGE-SDS. First, 1x of NuPAGE LDS Sample Buffer (4X) (Cat. NP0007, Thermo Fisher Scientific) was added to IP samples or to 25-50&nbsp;μg of nuclear enriched lysates and boiled at 95°C for 10&nbsp;min. and 5&nbsp;min. respectively. Samples were resolved in pre-cast Novex 4%–12% Tris-Glycine gels (Cat. XP04122BOX, Invitrogen) and transferred over night at 35V into polyvinylidene difluoride (PVDF) membranes for endogenous proteins. For ectopically expressed proteins gels were transferred using iBlot® Transfer Stack, PVDF, mini (Cat. IB4010-32, ThermoScientific) and iBlot™ gel transfer device (Thermo Fisher Scientific) for 8&nbsp;min. at 200V. Membranes were blocked with LI-COR blocking buffer (Cat. 927-40010, LI-COR Biosciences) or 10% BSA in PBS-T (PBS with 0.1% Tween) for 20&nbsp;min., and incubated overnight on agitation with the primary antibody. Appropriate secondary HRP-conjugated antibody (Abcam) or fluorophore conjugated secondary antibody (LI-COR) was added for 1 h. at a dilution of 1:5000 followed by detection with ECL Prime Western Blotting Detection Reagent (Cat. RPN2232, GELife Sciences) and exposure to autoradiography film at various time intervals or by digital imaging (LI-COR Odyssey). The following primary antibodies were used for immunoblotting: mouse anti- GATA-4 Antibody (G-4) (Cat. Sc-25310 X, Santa Cruz), mouse anti- TBX5 Antibody (A-6) (Cat. Sc-515536 X, Santa Cruz), anti-Myc tag antibody – ChIP Grade (Cat. Ab9132, Abcam), GLYR1 anti-sera 7 (James T. Kadonaga Laboratory), anti-HA tag antibody – ChIP Grade (Cat. Ab9110, Abcam), anti-MEIS1 antibody (Cat. Ab19867, Abcam), anti-ISL1 antibody (Cat. Ab20670, Abcam), anti- NKX2-5 antibody (Cat. Sc-8697, Santa Cruz), anti-LSD2 antibody (Cat. MBS4751131, MyBioSource), anti-CDK9 antibody (Cat. Sc-13130, Santa Cruz), anti-Cyclin T1 antibody (Cat. Sc-271348, Santa Cruz), and anti-Vinculin Monoclonal Antibody VLN01 (Cat. MA5-11690, Thermo Fisher Scientific).</div></section><section id="sec-4-4-6"><h4>Mass spectrometry</h4><div id="p0385" role="paragraph">For mass spectrometry all the IP steps described were processed in protein low-bind sterile Eppendorf tubes (Cat. 022431102, Eppendorf), aliquots specific for mass spectrometry were made for each of the IP buffers and carefully managed to avoid contaminations with filter tips. After the magnetic beads were spun down and the excess of liquid removed, to each IP sample tube another 1ml of NDB was added, beads resuspended and transferred using wide orifice tips to new protein low-bind sterile Eppendorf tubes. IP samples were returned to the magnetic stand and supernatant carefully removed by aspiration. IP samples were spun down again to collect the remaining liquid, placed in the magnet and the remaining liquid removed. Next, we proceeded with on-bead protein digestion. One bead volume (25&nbsp;μl) of freshly prepared Alkylation Buffer [2M Urea; 50mM Tris, pH8.0; 1mM DTT; 3mM IODO; resuspended in LC-MS high grade water] was added to each IP sample and incubated at RT in the dark for 45&nbsp;min. while shaking to ensure bead suspension. After incubation, an additional 3mM DTT and 750ng of trypsin per 10&nbsp;μL of bead volume were added to each tube. IP samples were on-bead digested overnight at 37°C on agitation. The next morning, beads were pelleted in a microcentrifuge at 2000&nbsp;rpm for 4&nbsp;min. IP samples were placed in a magnetic stand and supernatants carefully transferred to a fresh 0.5ml protein lo-bind tubes. Beads can be saved at 4°C for quality control. To each 0.5ml tube containing the digested IP supernatants, formic acid at a final concentration of 1% was added to stop the digestion process. The processed samples can be stored at this point at −80°C until desalting. Samples were subjected to desalting with OMICS tips (Agilent) and lyophilization in a speed-vacuum concentrator for 30&nbsp;min. Lyophilized samples were stored at −20°C until proceeding with mass spec analysis. When ready, lyophilized samples were resuspended in 10&nbsp;μL of 0.2% formic acid/ 2%acetonitrile immediately before loading into the mass spectrometry instrument. All APMS samples were measured using the Q Exactive Hybrid Quadrupole-Orbitrap Mass Spectrometrer.</div></section><section id="sec-4-4-7"><h4>RNA extraction, RT-PCR, and real-time PCR analysis</h4><div id="p0390" role="paragraph">Cells were harvested in TRIzolTM LS reagent (Cat. 10296010, Invitogen) and total RNA was extracted using the Direct-Zol RNA kit (Cat. R2052, Zymo Research) according to manufacturer instruction. 1000ng of RNA were converted to cDNA using SuperScriptTM III First-strand Synthesis SuperMix for qRT-PCR (Cat. 18080400, Invitrogen). For Taqman real-time PCR, 1/50 cDNA was applied for quantitative PCR reaction using Taqman Universal PCR master mix (Cat. 4305719, Life technologies). The PCR was conducted in&nbsp;7900HT Fast Real-Time system (Applied Biosystem). The Taqman probe for enhanced GFP (eGFP) quantification (Mr04097229_mr, ThermoFisher Scientific). All gene expressions were normalized with human GAPDH levels (Hs99999905_m1, ThermoFisher Scientific).</div></section><section id="sec-4-4-8"><h4>RNA-seq assay</h4><div id="p0395" role="paragraph">Total RNA was TRIzol-extracted (Cat. 10296010, Thermo Fisher Scientific) and further purified using the Direct-Zol RNA kit (Cat. R2052, Zymo Research) with DnaseI in-column treatment according to the manufacturer’s instructions and quantified with Nanodrop (Thermo scientific). After RNA quality control with bioanalyzer Agilent 2100 (Agilent Technologies), Paired-end Poly(A)-enriched RNA libraries were prepared with the Ovation RNA-seq System V2 Kit (Cat. 7102-08, NuGEN; strand specific) from the Gladstone Genomic core. The mRNA-seq libraries were analyzed by Agilent Bioanalyzer and quantified using an Illumina Library Quantification Kit (Cat. KK4824, KAPA Biosystems). Libraries were prepared by the Gladstone Genomics Core (<a id="intref0225" href="http://labs.gladstone.org/genomics/home" target="_blank">http://labs.gladstone.org/genomics/home</a>). High-throughput sequencing was done using an Illumina HiSeq 2500 instrument (<a id="intref0230" href="http://humangenetics.ucsf.edu/genomics-services/sample-processing/" target="_blank">http://humangenetics.ucsf.edu/genomics-services/sample-processing/</a>).</div></section><section id="sec-4-4-9"><h4>ChIP-seq assay, library preparation, and sequencing</h4><div id="p0400" role="paragraph">ChIPseq was performed as previously described (<span class="dropBlock reference-citations"><a id="body-ref-sref2" href="#" role="doc-biblioref" data-xml-rid="bib2" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref2" aria-controls="sref2">Alexanian et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">2.</div><div>Alexanian, M. ∙ Przytycki, P.F. ∙ Micheletti, R. ...</div><div><strong>A transcriptional switch governs fibroblast activation in heart disease</strong></div><div><em>Nature.</em> 2021; <strong>595</strong>:438-443</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-021-03674-1" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_2_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-85108376829" target="_blank" aria-label="Open Scopus in new tab">Scopus (35)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/34163071/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-021-03674-1&amp;pmid=34163071" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) with minor modifications. Briefly, CP or hiPSC cells (30&nbsp;× 10<sup>6</sup> for cTFs and 10&nbsp;× 10<sup>6</sup> for GLYR1 and histone marks) were pelleted and suspended in 10ml DMEM and cross-linked in 1% Formaldehyde solution (Cat. 28906, Thermo Fisher Scientific) by rocking in room temperature for 10&nbsp;min. Then glycine (final concentration 0.125M) was added to quench the cross-link for 5&nbsp;min. Samples were centrifuged at 1000 rcf for 5&nbsp;min. at 4°C. Cells were washed with 10ml of cold 1x PBS supplemented with proteinase inhibitors and phosphatase inhibitors (Cat. 4693132001 Roche and Cat. 4906837001 Sigma-Aldrich) and the pellets were snap frozen in liquid nitrogen. All samples were stored at −80°C until use. When ready, cell pellets were incubated in cell lysis buffer (20&nbsp;mM Tris-HCl, pH 8, 85&nbsp;mM KCl, 0.5% NP-40, protease/phosphatase inhibitors) for 10&nbsp;min. on a rotator at 4°C. Nuclei were isolated by centrifugation (2,500 x g, 5&nbsp;min., 4°C), resuspended in nuclear lysis buffer (50&nbsp;mM Tris-HCl, pH 8, 10&nbsp;mM EDTA, pH 8, 1% SDS, protease/phosphatase inhibitors) and incubated on a rotator for 30&nbsp;min. at 4°C. Chromatin was sheared using a Covaris S2 sonicator (Covaris Inc) for 15&nbsp;min. (60&nbsp;s cycles, 5% duty cycle, 200 cycles/burst, intensity&nbsp;= 6) until DNA was in the 200–700 base pair range. Chromatin was diluted 3-fold in ChIP dilution buffer (0.01% SDS, 1.1% Triton X-100, 1.2mMEDTA, 16.7mMTris-HCl, pH 8, 167&nbsp;mM NaCl, protease/phosphatase inhibitors) and incubated with the corresponding primary antibody at 4°C overnight under rotation. Antibody-protein complexes were immunoprecipitated using 50&nbsp;μL of Dynabeads Protein A/Protein G (Cat. 10015D, Invitrogen) per sample at 4°C for 2&nbsp;h under rotation. After incubation, beads were washed five times (2&nbsp;min./wash under rotation) with cold RIPA buffer [50&nbsp;mM HEPES-KOH, pH 7.5, 500&nbsp;mM LiCl, 1&nbsp;mM EDTA, 1% NP-40, 0.7% Na-deoxycholate], followed by one wash in cold final wash buffer [1xTE, 50&nbsp;mM NaCl]. Immunoprecipitated chromatin was eluted at 65°C with agitation for 30&nbsp;min. in elution buffer [50mMTris-HCl pH 8.0, 10mMEDTA, 1% SDS]. High-salt buffer [250mM Tris-HCl, pH 7.5, 32.5&nbsp;mM EDTA, pH 8, 1.25M NaCl] and Proteinase K (Cat. P8107s, New England Biolabs Inc (NEB)) were added and crosslinks were reversed overnight at 65°C. Samples were treated with Rnase A, and DNA was purified with AMPure XP beads (Cat. A63881, Beckman Coulter). Fragmented ChIP and input DNA were end-repaired, 5′-phosphorylated and dA-tailed with NEBNext Ultra II DNA Library Prep Kit for Illumina (Cat. E7645, New England BioLabs). Samples were ligated to adaptor oligos for multiplex sequencing (Cat. E7335, New England BioLabs), PCR amplified, and sequenced on an Illumina NextSeq 500 at the Gladstone Institutes. Primary antibodies used for ChIP were: GATA4 (Cat. Sc-1237 X, Santa Cruz), TBX5 (Cat. Sc-17866 X, Santa Cruz), H3K36me3 (Cat. Ab9050, Abcam), GLYR1 (anti-serum 7; provided by James T. Kadonaga’s laboratory), NKX2-5 (Cat. Sc-8697 X, Santa Cruz), MEIS1 (Cat. Ab19867, Abcam), ISL1 (Cat. AF1837, R&amp;D Systems). The specificity of the antibodies was validated in previous publications (<span class="dropBlock reference-citations"><a id="body-ref-sref86-3" href="#" role="doc-biblioref" data-xml-rid="bib86" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref86-3" aria-controls="sref86-3">Luna-Zurita et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref86-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">74.</div><div>Luna-Zurita, L. ∙ Stirnimann, C.U. ∙ Glatt, S. ...</div><div><strong>Complex interdependence regulates heterotypic transcription factor distribution and coordinates cardiogenesis</strong></div><div><em>Cell.</em> 2016; <strong>164</strong>:999-1014</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=10.1016%2Fj.cell.2016.01.004&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=10.1016%2Fj.cell.2016.01.004&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-84959387425" target="_blank" aria-label="Open Scopus in new tab">Scopus (122)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26875865/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2016.01.004&amp;pmid=26875865" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref33" href="#" role="doc-biblioref" data-xml-rid="bib33" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref33" aria-controls="sref33">Dupays et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref33"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">30.</div><div>Dupays, L. ∙ Shang, C. ∙ Wilson, R. ...</div><div><strong>Sequential Binding of MEIS1 and NKX2-5 on the Popdc2 Gene: A Mechanism for Spatiotemporal Regulation of Enhancers during Cardiogenesis</strong></div><div><em>Cell Rep.</em> 2015; <strong>13</strong>:183-195</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=10.1016%2Fj.celrep.2015.08.065&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=10.1016%2Fj.celrep.2015.08.065&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-84943235008" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26411676/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.celrep.2015.08.065&amp;pmid=26411676" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref38-6" href="#" role="doc-biblioref" data-xml-rid="bib38" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref38-6" aria-controls="sref38-6">Fei et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref38-6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">35.</div><div>Fei, J. ∙ Ishii, H. ∙ Hoeksema, M.A. ...</div><div><strong>NDF, a nucleosome-destabilizing factor that facilitates transcription through nucleosomes</strong></div><div><em>Genes Dev.</em> 2018; <strong>32</strong>:682-694</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/gad.313973.118" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-85047604949" target="_blank" aria-label="Open Scopus in new tab">Scopus (24)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29759984/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2Fgad.313973.118&amp;pmid=29759984" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div></section><section id="sec-4-4-10"><h4>siRNA transfection on CPs</h4><div id="p0405" role="paragraph">For siRNA knockdown experiments during in cardiac progenitor cells, at day 4 of differentiation cells were detached from the plates with 1ml of accutase (Cat. 07920, Stem Cell Technologies) and quenched with 1ml of B27 (minus insulin) RPMI1640 media (Cat.&nbsp;11875-119, Life Technologies) per well. All cells were combined and centrifuge at 300xg, supernatant removed and the pellet resuspended in a volume of B27 (minus insulin) RPMI with 5uM ROCK inhibitor (Y-27632 2HCl, Cat. S1049, <a id="intref0235" href="http://Selleckchem.com" target="_blank">Selleckchem.com</a>) necessary for 2x the number of wells initially collected. Cells were then seeded in twice the number of 12 well plates originally collected that were pre-coated with fibronectin bovine plasma solution (Cat. F1141, Sigma-Aldrich). Cells were immediately transfected in solution, prior attachment to the well surface using lipofectamine RNAiMax (Cat. 13778075, Invitrogen). For one well of a 12 well plate, mix A (75&nbsp;μL Opti-MEM (Cat. 31985070, Thermo Fisher Scientific) were combined with 3&nbsp;μL of a 10&nbsp;μM siRNA stock) and mix B (75&nbsp;μL Opti-MEM) with 7&nbsp;μL of lipofectamine RNAiMax were prepared. Mix A and B were combined and incubated at RT for 5-10&nbsp;min. 160&nbsp;μL of lipofectamine siRNA complexes were added dropwise to each well. At day 7 of differentiation, ∼72 h. after transfection cell were collected, washed, supernatants removed and pellets snap frozen and stored at −80°C until processed. The following siRNA were used: <i>GATA4</i> Silencer Select Pre-designed SiRNA (Cat. 4392420, ID s535120, Lot. AS02F2E2, Thermo Fisher Scientific), si<i>GLYR1</i> (siRNA ID: SASI_Hs01_00116796, Millipore-Sigma) and Silencer Select Negative control #1 siRNA (Cat. 4390843, Thermo Fisher Scientific).</div></section><section id="sec-4-4-11"><h4>Luciferase assay</h4><div id="p0410" role="paragraph">Transcriptional activity/synergy reporter assays for variant alleles within the reference gene <i>NKX2-5</i> and the GATA4 interactors: CHD7, SMARCC1 and BRD4 were performed employing the previously described <i>Ppargc1a</i> promoter PGL4.23 vector (<span class="dropBlock reference-citations"><a id="body-ref-sref103-5" href="#" role="doc-biblioref" data-xml-rid="bib103" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref103-5" aria-controls="sref103-5">Padmanabhan et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref103-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">91.</div><div>Padmanabhan, A. ∙ Alexanian, M. ∙ Linares-Saldana, R. ...</div><div><strong>BRD4 (Bromodomain-Containing Protein 4) Interacts with GATA4 (GATA Binding Protein 4) to Govern Mitochondrial Homeostasis in Adult Cardiomyocytes</strong></div><div><em>Circulation.</em> 2020; <strong>142</strong>:2338-2355</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1161/CIRCULATIONAHA.120.047753" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-85097828347" target="_blank" aria-label="Open Scopus in new tab">Scopus (14)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33094644/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1161%2FCIRCULATIONAHA.120.047753&amp;pmid=33094644" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). GATA4–GLYR1 transcriptional synergy reporter assay was performed using the pANF638L vector (<span class="dropBlock reference-citations"><a id="body-ref-sref69-2" href="#" role="doc-biblioref" data-xml-rid="bib69" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref69-2" aria-controls="sref69-2">Knowlton et&nbsp;al., 1991</a><div class="dropBlock__holder" data-db-target-of="sref69-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">62.</div><div>Knowlton, K.U. ∙ Baracchini, E. ∙ Ross, R.S. ...</div><div><strong>Co-regulation of the atrial natriuretic factor and cardiac myosin light chain-2 genes during alpha-adrenergic stimulation of neonatal rat ventricular cells. Identification of cis sequences within an embryonic and a constitutive contractile protein gene which mediate inducible expression</strong></div><div><em>J.&nbsp;Biol. Chem.</em> 1991; <strong>266</strong>:7759-7768</div></div><div class="external-links"><div class="core-xlink-abstract"><a href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=10.1016%2FS0021-9258%2820%2989515-5&amp;cf=abstract&amp;site=jbc-site" target="_blank" aria-label="Open Abstract in new tab">Abstract</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=10.1016%2FS0021-9258%2820%2989515-5&amp;cf=pdf&amp;site=jbc-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/1850419/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0021-9258%2820%2989515-5&amp;pmid=1850419" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) or pGL4.23[luc2/minP] (Cat. E8411, Promega) modified reporters in which putative intronic RE co-bound by GATA4 &amp; GLYR1 were cloned. Briefly, HeLa cells were cultured in 24-well plates at 10ˆ5 cells per well and transfected within 24 h. of seeding. Cells were co-transfected with 200&nbsp;ng of luciferase reporter vector and 20&nbsp;ng of Renilla luciferase control vector pRL-TK or SV40 (Cat. E2241 or E2231, Promega) in 2.4&nbsp;μL FuGENE HD (Cat. E2311, Promega) and 43&nbsp;μl Opti-MEM (Cat. 31985070, Thermo Fisher Scientific). The transfection mix was aliquoted in 5 tubes (1 per condition) and the following conditions were prepared for the luciferase assay using the pANF638L or <i>Ppargc1a</i> promoter PGL4.23 vector: 1.) Control: 600ng of empty vector (EV); 2.) GATA4: 200ng of GFP-GATA4 vector plus 400ng EV; 3.) co-factor (GLYR1, CHD7, SMARCC1, BRD4): 400ng co-factor vector plus 200ng EV and 4.) GATA4+ co-factor: 200ng of GFP-GATA4 vector with 400ng co-factor vector and 5.) GATA4+ mutant co-factor vector: 200ng of GFP-GATA4 vector with 400ng mutant co-factor vector. Cells were collected at 24 h, or 48 h. following transfection. Samples were processed with the Dual Luciferase Assay System (Cat. E1960, Promega) following manufacturer’s instructions and measured with a luminometer (SpectraMax i3).</div><div id="p0415" role="paragraph">For analyzing the GATA4–GLYR1 transcriptional synergy within putative intronic REs, GATA4 -bound intronic regions within GATA4 &amp; GLYR1 -bound cardiac development genes that co-localized with H3K27ac, H3K4me1 or H3K4me3, MED1 and at least co-occupied by two cTFs, were cloned with Cold Phusion (Systems Biosciences) by designing gBlocks (IDT) for each of the selected putative REs (GATA6: chr18:19,773,958-19,774,419; MYL4: chr17:45,296,131-45,296,972; TTN: chr2:179,493,152-179,494,179) flanked by homology arms complementary to the pGL4.23[luc2/minP] luciferase reporter vector (Cat. E8411, Promega). HeLa cells were plated as indicated for the pANF638L reporter and transfection mixed prepared as indicated above. The transfection mix was aliquoted in 5 tubes (1 per condition) and the following conditions were prepared for each of the cloned pGL4.23 luciferase reporter vectors: 1.) Control: 200ng of empty vector (EV); 2.) GATA4: 200ng of GFP-GATA4 vector plus Adenovirus control (Ad-EF1a-eGFP, MOI 25); 3.) GLYR1: Adenovirus GLYR1 WT (Ad-eGFP-EF1-h-GLYR1, MOI 25) and 4.) GATA4+GLYR1: 200ng of GFP-GATA4 vector and Adenovirus GLYR1 WT (MOI 25); 5.) GATA4+GLYR1 P496L: 200ng of GFP-GATA4 vector Adenovirus GLYR1 P496L (Ad-eGFP-EF1-h-GLYR1 P496L, MOI 15). The MOIs were determined by HeLa cell infection followed by quantitative PCR amplification with the eGFP Taqman probe (Mr04097229_mr, ThermoFisher Scientific). MOIs rendering comparable eGFP expression levels were chosen for the luciferase experiments (Relative to GAPDH Avg levels for Ad GLYR1 WT MOI 25: 50.134 and Ad GLYR1 P496L MOI 15: 57.333; n&nbsp;= 3). HeLa cells were collected 48&nbsp;h after transfection/infection and processed as indicated for the pANF638L vector.</div></section><section id="sec-4-4-12"><h4>Single-cell RNA-seq cell preparation</h4><div id="p0420" role="paragraph">Cells were wash with PBS and detached with 0.25% Trypsin (CMs) or accutase (hiPSCs and CPs) incubated for 5&nbsp;min. Quenched with 1&nbsp;mL of PBS&nbsp;+ 1% FBS and spin down at 800-1000&nbsp;rpms for 3&nbsp;min. Cells were washed twice with PBS&nbsp;+ 1% FBS, resuspend in cold PBS&nbsp;+ 1% FBS (use 0.5mL PBS per well collected from a 12-well-plate) and put on ice. Cells were filtered to avoid clumps and manually count with hemocytometer. Both live and dead cells were included in the count and cells were diluted to 1e6 cells/mL. A total of 60,000 cells were used to proceed with the scRNaseq library construction protocol.</div></section><section id="sec-4-4-13"><h4>Single-cell transcriptome library preparation and sequencing</h4><div id="p0425" role="paragraph">Single-cell droplet libraries from the hiPSC (1x <i>GLYR1</i><sup><i>KO</i></sup>, 2x <i>GLYR1</i><sup><i>WT</i></sup>, 2x <i>GLYR1</i><sup><i>P496L</i></sup>), CM-differentiation day 6 (3x GLYR1<sup>WT</sup>, 3x <i>GLYR1</i><sup><i>P496L</i></sup>) and day 18 (3x GLYR1<sup>WT</sup>, 3x <i>GLYR1</i><sup><i>P496L</i></sup>) cell suspensions (<a id="crosref1895" href="#fig7">Figures 7</a> and <a id="crosref1900" href="#figs6">S6</a>) were generated in the 10X Genomics Chromium controller according to the manufacturer’s instructions in the Chromium Single Cell 3′ Reagent Kit v.3&nbsp;User Guide. Additional components used for library preparation include the Chromium Next GEM Single Cell 3′ GEM, Library &amp; Gel Bead Kit v3.1, (PN-1000121, 10X Genomics) and the Chromium Next GEM Chip G Single Cell Kit (PN-1000120, 10X Genomics). Libraries were prepared according to the manufacturer’s instructions using the Chromium Single Cell 3′ Library and Gel Bead Kit v.3.1 (PN-1000121, 10X Genomics) and 3′v3.1 Single Index Kit (PN-1000213, 10X Genomics). Final libraries were sequenced on the NovaSeq (Illumina, software v1.5). Sequencing parameters were selected according to the Chromium Single Cell v.3.1 specifications. All libraries were sequenced to a mean read depth of at least 50,000 total aligned reads per cell.</div></section><section id="sec-4-4-14"><h4>Heart histology</h4><div id="p0430" role="paragraph">To examine hearts at postnatal day 1, hearts were dissected from the animals and fixed overnight at 4°C in 4% paraformaldehyde. They were then washed twice in PBS and stored in 70% ethanol until processing. Hearts were paraffin embedded and sectioned to obtained a four-chamber view. Heart sections were stained with hematoxylin and eosin and imaged with a slide scanner.</div></section><section id="sec-4-4-15"><h4>Echocardiography</h4><div id="p0435" role="paragraph">For echocardiography, newborn mice were imaged using the Vevo 3100 High Resolution Imaging System (FujiFilm VisualSonics Inc.), an ultra-high frequency linear array transducer (MX700). All the echocardiogram analyses were performed blinded (mice assigned to an alphanumeric code) until statistical analysis.</div></section><section id="sec-4-4-16"><h4>Computational analysis</h4><section id="sec-4-4-16-1"><h5>Mass spectrometry analysis of affinity purifications</h5><div id="p0440" role="paragraph">Peptides from affinity purifications were analyzed on a Q-Exactive Plus (Thermo Fisher) mass spectrometer. The Q-Exactive Plus system was equipped with an Easy1200 nLC system (Thermo Fisher) and an analytical column (25&nbsp;cm x 75 um I.D. packed with ReproSil Pur C18 1.9&nbsp;μm, 120Å particles, Dr. Maisch). A gradient was delivered from 2% to 30% acetonitrile over 53&nbsp;min at a flow rate of 300 nL/min. All MS spectra were collected with Orbitrap detection, while the 20 most abundant ions were fragmented by HCD and detected in the Orbitrap. Peptide and protein identification searches, as well as label-free quantitation were performed using the MaxQuant data analysis algorithm (version 1.5.8.0) (<span class="dropBlock reference-citations"><a id="body-ref-sref26" href="#" role="doc-biblioref" data-xml-rid="bib26" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref26" aria-controls="sref26">Cox and Mann 2008</a><div class="dropBlock__holder" data-db-target-of="sref26"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">25.</div><div>Cox, J. ∙ Mann, M.</div><div><strong>MaxQuant enables high peptide identification rates, individualized p.p.b.-range mass accuracies and proteome-wide protein quantification</strong></div><div><em>Nat. Biotechnol.</em> 2008; <strong>26</strong>:1367-1372</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nbt.1511" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-57449099865" target="_blank" aria-label="Open Scopus in new tab">Scopus (8817)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19029910/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnbt.1511&amp;pmid=19029910" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Data were searched against a database containing SwissProt Human sequences (downloaded 02/2017) concatenated to a decoy database where each sequence was randomized in order to estimate the false discovery rate (FDR).</div><div id="p0445" role="paragraph">Variable modifications were allowed for methionine oxidation and protein N terminus acetylation. A fixed modification was indicated for cysteine carbamidomethylation. Full trypsin specificity was required. The first search was performed with a mass accuracy of ± 20 parts per million (ppm) and the main search was performed with a mass accuracy of ± 4.5 parts per million. A maximum of 5 modifications were allowed per peptide. A maximum of 2 missed cleavages were allowed. The maximum charge allowed was 7+. Individual peptide mass tolerances were allowed. For MS/MS matching, a mass tolerance of ± 20 ppm was allowed and the top 12 peaks per 100&nbsp;Da were analyzed. MS/MS matching was allowed for higher charge states, water and ammonia loss events. The data were filtered to obtain a peptide, protein, and site-level false discovery rate of 0.01. The minimum peptide length was 7 amino acids.</div></section><section id="sec-4-4-16-2"><h5>Selection of interactome proteins</h5><div id="p0450" role="paragraph">APMS data was analyzed using the artMS package (<span class="dropBlock reference-citations"><a id="body-ref-sref62-2" href="#" role="doc-biblioref" data-xml-rid="bib62" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref62-2" aria-controls="sref62-2">Jimenez-Morales et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref62-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">55.</div><div>Jimenez-Morales, D. ∙ Rosa Campos, A. ∙ Von Dollen, J. ...</div><div><strong>artMS: Analytical R tools for Mass Spectrometry version 1.6.5 from Bioconductor</strong></div><div><em>Bioconductor.</em> 2020;</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=D.Jimenez-MoralesA.Rosa+CamposJ.Von+DollenD.SwaneyartMS%3A+Analytical+R+tools+for+Mass+Spectrometry+version+1.6.5+from+BioconductorBioconductor2020" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) in R followed by protein-protein interaction scoring by the SAINTq software (<span class="dropBlock reference-citations"><a id="body-ref-sref128-3" href="#" role="doc-biblioref" data-xml-rid="bib128" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref128-3" aria-controls="sref128-3">Teo et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref128-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">115.</div><div>Teo, G. ∙ Koh, H. ∙ Fermin, D. ...</div><div><strong>SAINTq: Scoring protein-protein interactions in affinity purification - mass spectrometry experiments with fragment or peptide intensity data</strong></div><div><em>Proteomics.</em> 2016; <strong>16</strong>:2238-2245</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/pmic.201500499" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_115_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-84983261035" target="_blank" aria-label="Open Scopus in new tab">Scopus (24)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27119218/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fpmic.201500499&amp;pmid=27119218" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) to identify significantly-interacting proteins for GATA4 and TBX5 baits. Default parameters for both softwares were used except where indicated here: To create the GATA4 interactome, we analyze at the protein level and select proteins that interact at a BFDR cutoff of £ 0.001; to create the TBX5 interactome, we analyze at the peptide level and select those that interact at a BFDR cutoff of £ 0.05. Intensity data from the control (knockout) cell lines was normalized per SAINTq configuration options such that the average total intensity in each bait purification was equal to the average total intensity across the control experiments.</div><div id="p0455" role="paragraph">To focus on transcriptionally-relevant interactions, we additionally filter proteins by those that appear in the nuclear compartment, those that are expressed at detectable levels in at least one of the same cell types as the bait, and proteins whose gene expression was significantly lower in the control line but did not have a greater than 0.5 log-fold change drop in intensity. <b>Nuclear compartment</b>: nuclear compartment genes were identified using the Cytoscape package BiNGO (<span class="dropBlock reference-citations"><a id="body-ref-sref88" href="#" role="doc-biblioref" data-xml-rid="bib88" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref88" aria-controls="sref88">Maere et&nbsp;al., 2005</a><div class="dropBlock__holder" data-db-target-of="sref88"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">76.</div><div>Maere, S. ∙ Heymans, K. ∙ Kuiper, M.</div><div><strong>BiNGO: a Cytoscape plugin to assess overrepresentation of gene ontology categories in biological networks</strong></div><div><em>Bioinformatics.</em> 2005; <strong>21</strong>:3448-3449</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/bti551" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_76_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-24044440971" target="_blank" aria-label="Open Scopus in new tab">Scopus (3028)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15972284/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbti551&amp;pmid=15972284" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref122-2" href="#" role="doc-biblioref" data-xml-rid="bib122" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref122-2" aria-controls="sref122-2">Shannon et&nbsp;al., 2003</a><div class="dropBlock__holder" data-db-target-of="sref122-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">110.</div><div>Shannon, P. ∙ Markiel, A. ∙ Ozier, O. ...</div><div><strong>Cytoscape: a software environment for integrated models of biomolecular interaction networks</strong></div><div><em>Genome Res.</em> 2003; <strong>13</strong>:2498-2504</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/gr.1239303" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_110_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-0242490780" target="_blank" aria-label="Open Scopus in new tab">Scopus (23829)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/14597658/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2Fgr.1239303&amp;pmid=14597658" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) with additional manual curation from literature. <b>Cell type co-expression</b>: single-cell RNA-seq data from deSoysa et&nbsp;al., 2019 was used to determine if an interaction was likely to occur, given co-expression in the same cell type. Briefly, mesoderm and neural crest cells in the developing heart were used to identify seven cell type populations (multipotent Isl1+ progenitors, endothelial or endocardial cells, epicardium, myocardium, neural crest-derived mesenchyme, paraxial mesoderm and lateral plate mesoderm) (<span class="dropBlock reference-citations"><a id="body-ref-sref28-2" href="#" role="doc-biblioref" data-xml-rid="bib28" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref28-2" aria-controls="sref28-2">de Soysa et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref28-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">27.</div><div>de Soysa, T.Y. ∙ Ranade, S.S. ∙ Okawa, S. ...</div><div><strong>Single-cell analysis of cardiogenesis reveals basis for organ-level developmental defects</strong></div><div><em>Nature.</em> 2019; <strong>572</strong>:120-124</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-1414-x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-85069751389" target="_blank" aria-label="Open Scopus in new tab">Scopus (91)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31341279/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-1414-x&amp;pmid=31341279" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). A bait protein is considered to be expressed in one of these cell types if the transcripts per million (tpm) for the bait gene were greater than 0.05 tpm. Prey proteins were considered to be potentially physiologically relevant interactors if they were detected at any level in one of the same cell types as the bait. <b>Controlling for differential gene expression</b>: protein hits that were considered likely false positives based on lower expression in the control cell lines, without concomitant reduction in protein intensity, were removed from the interactome list. This is intended to control for genes that are expressed less in the controls due to bait knockout, but whose APMS protein intensities do not change (suggesting the protein pulled down was background rather than an interactor). Significant differential gene expression was determined in R using the edgeR package <span data-style="underline">(</span><span class="dropBlock reference-citations"><a id="body-ref-sref118-1" href="#" role="doc-biblioref" data-xml-rid="bib118" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref118-1" aria-controls="sref118-1">Robinson et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref118-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">106.</div><div>Robinson, M.D. ∙ McCarthy, D.J. ∙ Smyth, G.K.</div><div><strong>edgeR: a Bioconductor package for differential expression analysis of digital gene expression data</strong></div><div><em>Bioinformatics.</em> 2010; <strong>26</strong>:139-140</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btp616" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_106_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-75249087100" target="_blank" aria-label="Open Scopus in new tab">Scopus (19590)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19910308/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtp616&amp;pmid=19910308" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><span data-style="underline">);</span> normalized protein intensities were averaged in all control experiments and bait experiments. Proteins with significantly reduced expression in control (FDR £ 0.05) with less than a 0.5 log-fold change drop in intensity were not considered to be interactors.</div></section><section id="sec-4-4-16-3"><h5>Gene expression tissue distribution and specificity</h5><div id="p0460" role="paragraph">The categories for gene expression tissue distribution and tissue specificity defined by the <i>Tissue Atlas</i> within the <i>Human Protein Atlas</i> were used to classify the specified gene groups (<a id="intref0240" href="https://www.proteinatlas.org/humanproteome/tissue/tissue+specific" target="_blank">https://www.proteinatlas.org/humanproteome/tissue/tissue+specific</a>). These classifications are based on transcriptomics analysis across all major organs and tissue types in the human body, where all putative 19670 protein coding genes have been classified with regard to abundance and distribution of transcribed mRNA molecules (<span class="dropBlock reference-citations"><a id="body-ref-sref134-2" href="#" role="doc-biblioref" data-xml-rid="bib134" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref134-2" aria-controls="sref134-2">Uhlén et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref134-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">121.</div><div>Uhlén, M. ∙ Fagerberg, L. ∙ Hallström, B.M. ...</div><div><strong>Proteomics. Tissue-based map of the human proteome</strong></div><div><em>Science.</em> 2015; <strong>347</strong>:1260419</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.1260419" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25613900/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.1260419&amp;pmid=25613900" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="p0465" role="paragraph">Specificity illustrates the number of genes with elevated or non-elevated expression. Elevated expression includes three subcategory types:<div id="ulist0015" role="list"><div id="u0030" role="listitem"><div class="label">•</div><div class="content"><div id="p0470" role="paragraph">Tissue enriched: At least four-fold higher mRNA level in a particular tissue compared to any other tissues.</div></div></div><div id="u0035" role="listitem"><div class="label">•</div><div class="content"><div id="p0475" role="paragraph">Group enriched: At least four-fold higher average mRNA level in a group of 2-5 tissues compared to any other tissue.</div></div></div><div id="u0040" role="listitem"><div class="label">•</div><div class="content"><div id="p0480" role="paragraph">Tissue enhanced: At least four-fold higher mRNA level in a particular tissue compared to the average level in all other tissues.</div></div></div></div></div><div id="p0485" role="paragraph">Distribution, on the other hand, visualizes how many genes that have, or do not have, detectable levels (NX<sup>3</sup>1) of transcribed mRNA molecules. All elevated genes are categorized as:<div id="ulist0020" role="list"><div id="u0045" role="listitem"><div class="label">•</div><div class="content"><div id="p0490" role="paragraph">Detected in single: Detected in a single tissue</div></div></div><div id="u0050" role="listitem"><div class="label">•</div><div class="content"><div id="p0495" role="paragraph">Detected in some: Detected in more than one but less than one third of tissues</div></div></div><div id="u0055" role="listitem"><div class="label">•</div><div class="content"><div id="p0500" role="paragraph">Detected in many: Detected in at least a third but not all tissues</div></div></div><div id="u0060" role="listitem"><div class="label">•</div><div class="content"><div id="p0505" role="paragraph">Detected in all: Detected in all tissues</div></div></div></div></div></section><section id="sec-4-4-16-4"><h5>Variant calling</h5><div id="p0510" role="paragraph">Whole Exome Sequencing data from 2645 CHD trios and 1789 control trios was processed as described and published in Jin et&nbsp;al., 2017. We include Whole Exome Sequencing data from 419 additional CHD trios recruited to the Pediatric Cardiac Genomics Consortium (PCGC), processed by the HMS pipeline as described in Jin et&nbsp;al., 2017. Protein-coding mutations were filtered based on a Mapping Quality score &gt; 59 and Genotype Quality &gt; 90, then annotated using ANNOVAR. <i>De novo</i> variants were called using the TrioDeNovo program (<span class="dropBlock reference-citations"><a id="body-ref-sref139-2" href="#" role="doc-biblioref" data-xml-rid="bib139" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref139-2" aria-controls="sref139-2">Wei et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref139-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">126.</div><div>Wei, Q. ∙ Zhan, X. ∙ Zhong, X. ...</div><div><strong>A Bayesian framework for de novo mutation calling in parents-offspring trios</strong></div><div><em>Bioinformatics.</em> 2015; <strong>31</strong>:1375-1381</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btu839" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25535243/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtu839&amp;pmid=25535243" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), and accepted if the minor allele frequency (MAF) and read-depth criteria described in Homsy et&nbsp;al., 2015 are met. Namely, the in-cohort MAF of the variant must be below 4x10<sup>−4</sup>, with a minimum of 5 alternative reads and 10 total reads in the proband, and a minimum of 10 reference reads in the parents (with a maximum alternate allele ratio of 3.5%).</div></section><section id="sec-4-4-16-5"><h5>Permutation-based test</h5><div id="p0515" role="paragraph"><i>Case-control permutation.</i> We tested the adjusted odds ratio of observing a <i>de novo</i> mutation in an interactome gene in CHD probands relative to controls. We ran 10,000 permutations in which case/control status was randomly shuffled to generate a null distribution of permuted odds ratios (Ors). This was performed for protein-altering (non-synonymous) <i>de novo</i> mutations, synonymous <i>de novo</i> mutations, and rare inherited loss-of-function mutations (at minor allele frequency 10<sup>−5</sup>) (<span class="dropBlock reference-citations"><a id="body-ref-sref63-11" href="#" role="doc-biblioref" data-xml-rid="bib63" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref63-11" aria-controls="sref63-11">Jin et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref63-11"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">56.</div><div>Jin, S.C. ∙ Homsy, J. ∙ Zaidi, S. ...</div><div><strong>Contribution of rare inherited and de novo variants in 2,871 congenital heart disease probands</strong></div><div><em>Nat. Genet.</em> 2017; <strong>49</strong>:1593-1601</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng.3970" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_56_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-85032451853" target="_blank" aria-label="Open Scopus in new tab">Scopus (358)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28991257/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng.3970&amp;pmid=28991257" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) on the GATA4 and TBX5 interactomes generated from both cardiac progenitor and HEK293 APMS experiments. The raw p value for each test is equal to the proportion of random shuffles with a permuted OR greater than or equal to the observed OR. P values were adjusted for multiple testing using the Bonferroni method. We observed that some genes appeared to have been more deeply sequenced in control individuals, while other genes showed the opposite trend. This is not unexpected, as control individuals in the Jin et&nbsp;al. dataset were sequenced for a different study and at different institutions from PCGC individuals. Therefore, to control for regional biases in sequencing between the case and control studies, we adjusted the odds ratios of the synonymous and protein-altering variant data by a factor that restricts the synonymous odds ratio to 1 (the null expectation). This correction was performed for the observed odds ratio and the odds ratios calculated in each permutation of <i>de novo</i> variants. To determine whether this signal was driven by already-identified CHD risk genes, we repeated the analysis after removing <i>de novo</i> variants occurring in known known Human/Mouse CHD genes (sourced from <span class="dropBlock reference-citations"><a id="body-ref-sref63-12" href="#" role="doc-biblioref" data-xml-rid="bib63" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref63-12" aria-controls="sref63-12">Jin et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref63-12"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">56.</div><div>Jin, S.C. ∙ Homsy, J. ∙ Zaidi, S. ...</div><div><strong>Contribution of rare inherited and de novo variants in 2,871 congenital heart disease probands</strong></div><div><em>Nat. Genet.</em> 2017; <strong>49</strong>:1593-1601</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng.3970" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_56_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-85032451853" target="_blank" aria-label="Open Scopus in new tab">Scopus (358)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28991257/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng.3970&amp;pmid=28991257" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>, Supplemental Dataset 2: 253 Curated) (<span class="dropBlock reference-citations"><a id="body-ref-sref63-13" href="#" role="doc-biblioref" data-xml-rid="bib63" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref63-13" aria-controls="sref63-13">Jin et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref63-13"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">56.</div><div>Jin, S.C. ∙ Homsy, J. ∙ Zaidi, S. ...</div><div><strong>Contribution of rare inherited and de novo variants in 2,871 congenital heart disease probands</strong></div><div><em>Nat. Genet.</em> 2017; <strong>49</strong>:1593-1601</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng.3970" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_56_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-85032451853" target="_blank" aria-label="Open Scopus in new tab">Scopus (358)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28991257/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng.3970&amp;pmid=28991257" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) as well as after removing a curated list of 144 human CHD-genes (<span class="dropBlock reference-citations"><a id="body-ref-sref60-3" href="#" role="doc-biblioref" data-xml-rid="bib60" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref60-3" aria-controls="sref60-3">Izarzugaza et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref60-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">54.</div><div>Izarzugaza, J.M.G. ∙ Ellesøe, S.G. ∙ Doganli, C. ..., Competence Network for Congenital Heart Defects, Germany</div><div><strong>Systems genetics analysis identifies calcium-signaling defects as novel cause of congenital heart disease</strong></div><div><em>Genome Med.</em> 2020; <strong>12</strong>:76</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13073-020-00772-z" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-85090171445" target="_blank" aria-label="Open Scopus in new tab">Scopus (5)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32859249/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13073-020-00772-z&amp;pmid=32859249" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="p0520" role="paragraph"><i>Gene-set permutation.</i> For each gene in the GT-PPI, we identify non-interactome genes that are expressed at similar levels in WT CP cells, and have comparable mutability scores as calculated by Samocha et&nbsp;al., 2014. A gene is considered a match if its mutability score (expected number of <i>de novo</i> mutations in this gene per chromosome per generation) is equivalent when rounded to the order of one hundred-thousandth. We further filter the list of matches based on similarity of expression levels in wildtype cardiac progenitor cells, such that the measured transcripts per million (tpm) is equivalent to the order of one one-hundredth. For genes with fewer than 100 possible matches, we relax these requirements by ± 0.5&nbsp;× 10<sup>N</sup> (where N is the relevant order of magnitude), and remove any genes from the analysis in the case of&nbsp;&lt; 10 matches. For 1000 permutations, we permute each interactome gene into one from its list of comparable non-interactome genes to compare the total count of variants found in CHD cases from the GT-PPI interactome versus those across all permuted gene-sets.</div></section><section id="sec-4-4-16-6"><h5>Characterization of CHD candidate variants and genes</h5><div id="p0525" role="paragraph">All <i>de novo</i> variants and harboring genes observed in CHD probands and matched controls were assessed for the following properties: CADD score, pLI score, variant degree, CHD-gene degree, heart expression percentile rank, haploinsufficiency, and number of mutations per kilobase. The residue-level CADD score (<span class="dropBlock reference-citations"><a id="body-ref-sref113-3" href="#" role="doc-biblioref" data-xml-rid="bib113" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref113-3" aria-controls="sref113-3">Rentzsch et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref113-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">101.</div><div>Rentzsch, P. ∙ Witten, D. ∙ Cooper, G.M. ...</div><div><strong>CADD: predicting the deleteriousness of variants throughout the human genome</strong></div><div><em>Nucleic Acids Res.</em> 2019; <strong>47</strong>:D886-D894</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gky1016" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_101_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-85059798241" target="_blank" aria-label="Open Scopus in new tab">Scopus (1261)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30371827/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgky1016&amp;pmid=30371827" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) estimates the likely deleteriousness of a variant based on conservation data. pLI score indicates the predicted loss-of-function intolerance of the gene, scaled between 0 and 1, and was sourced from gnomAD version 2.1.1 (<span class="dropBlock reference-citations"><a id="body-ref-sref66-2" href="#" role="doc-biblioref" data-xml-rid="bib66" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref66-2" aria-controls="sref66-2">Karczewski et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref66-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">59.</div><div>Karczewski, K.J. ∙ Francioli, L.C. ∙ Tiao, G. ..., Genome Aggregation Database Consortium</div><div><strong>The mutational constraint spectrum quantified from variation in 141,456 humans</strong></div><div><em>Nature.</em> 2020; <strong>581</strong>:434-443</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-020-2308-7" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-85085542423" target="_blank" aria-label="Open Scopus in new tab">Scopus (2715)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32461654/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-020-2308-7&amp;pmid=32461654" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Similarly, haploinsufficiency predicts the deleteriousness of having only a single functional copy of a gene. We use the predicted haploinsufficiency values from Huang et&nbsp;al., 2010 (<span class="dropBlock reference-citations"><a id="body-ref-sref59-2" href="#" role="doc-biblioref" data-xml-rid="bib59" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref59-2" aria-controls="sref59-2">Huang et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref59-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">53.</div><div>Huang, N. ∙ Lee, I. ∙ Marcotte, E.M. ...</div><div><strong>Characterising and predicting haploinsufficiency in the human genome</strong></div><div><em>PLoS Genet.</em> 2010; <strong>6</strong>:e1001154</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pgen.1001154" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-78449263023" target="_blank" aria-label="Open Scopus in new tab">Scopus (438)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20976243/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pgen.1001154&amp;pmid=20976243" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The CHD-gene degree counts the number of protein-protein interactions that the gene shares with previously-identified CHD risk genes, while the variant degree counts the number of protein-protein interactions shared with other genes that had <i>de novo</i> variants (DNVs) in a CHD proband. These node degree counts were normalized by the total number of connections observed in the gene, and are based on known mammalian protein-protein interactions in iRefIndex version 15.0 (<span class="dropBlock reference-citations"><a id="body-ref-sref112-4" href="#" role="doc-biblioref" data-xml-rid="bib112" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref112-4" aria-controls="sref112-4">Razick et&nbsp;al., 2008</a><div class="dropBlock__holder" data-db-target-of="sref112-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">100.</div><div>Razick, S. ∙ Magklaras, G. ∙ Donaldson, I.M.</div><div><strong>iRefIndex: a consolidated protein interaction database with provenance</strong></div><div><em>BMC Bioinformatics.</em> 2008; <strong>9</strong>:405</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/1471-2105-9-405" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_100_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-54949127160" target="_blank" aria-label="Open Scopus in new tab">Scopus (405)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/18823568/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2F1471-2105-9-405&amp;pmid=18823568" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Finally, the number of mutations per kilobase measures the number of times a <i>de novo</i> or rare loss of function variant was observed in a CHD proband, normalized by the coding length of that gene. We use a Mann-Whitney U test with Bonferroni correction to assess whether protein-altering DNVs in interactome genes differ significantly from those in non-interactome genes with respect to these properties, as well as whether protein-altering DNVs in cases differ from those found in controls.</div></section><section id="sec-4-4-16-7"><h5>Variant scoring</h5><div id="p0530" role="paragraph">All protein-altering <i>de novo</i> missense variants occurring in GT-interacting genes and observed in CHD probands were ranked based on a series of gene-level, residue-level, and patient-level properties. A mutations per kilobase value was determined for each gene, based on the number of protein-altering <i>de novo</i> and rare loss-of-function mutations found in CHD probands in the PCGC, normalized by the CDS length of that gene in the gnomAD database (<span class="dropBlock reference-citations"><a id="body-ref-sref66-3" href="#" role="doc-biblioref" data-xml-rid="bib66" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref66-3" aria-controls="sref66-3">Karczewski et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref66-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">59.</div><div>Karczewski, K.J. ∙ Francioli, L.C. ∙ Tiao, G. ..., Genome Aggregation Database Consortium</div><div><strong>The mutational constraint spectrum quantified from variation in 141,456 humans</strong></div><div><em>Nature.</em> 2020; <strong>581</strong>:434-443</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-020-2308-7" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-85085542423" target="_blank" aria-label="Open Scopus in new tab">Scopus (2715)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32461654/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-020-2308-7&amp;pmid=32461654" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). pLI score indicates the predicted loss-of-function intolerance of the gene, scaled between 0 and 1 where 1 is more intolerant. pLI data was sourced from gnomAD version 2.1.1 (<span class="dropBlock reference-citations"><a id="body-ref-sref66-4" href="#" role="doc-biblioref" data-xml-rid="bib66" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref66-4" aria-controls="sref66-4">Karczewski et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref66-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">59.</div><div>Karczewski, K.J. ∙ Francioli, L.C. ∙ Tiao, G. ..., Genome Aggregation Database Consortium</div><div><strong>The mutational constraint spectrum quantified from variation in 141,456 humans</strong></div><div><em>Nature.</em> 2020; <strong>581</strong>:434-443</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-020-2308-7" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-85085542423" target="_blank" aria-label="Open Scopus in new tab">Scopus (2715)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32461654/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-020-2308-7&amp;pmid=32461654" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). CHD-gene degree, variant degree, and mutations per kilobase values were calculated as described above based on known mammalian protein-protein interactions in iRefIndex version 15.0 (<span class="dropBlock reference-citations"><a id="body-ref-sref112-5" href="#" role="doc-biblioref" data-xml-rid="bib112" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref112-5" aria-controls="sref112-5">Razick et&nbsp;al., 2008</a><div class="dropBlock__holder" data-db-target-of="sref112-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">100.</div><div>Razick, S. ∙ Magklaras, G. ∙ Donaldson, I.M.</div><div><strong>iRefIndex: a consolidated protein interaction database with provenance</strong></div><div><em>BMC Bioinformatics.</em> 2008; <strong>9</strong>:405</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/1471-2105-9-405" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_100_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-54949127160" target="_blank" aria-label="Open Scopus in new tab">Scopus (405)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/18823568/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2F1471-2105-9-405&amp;pmid=18823568" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) (see Methods: Interactome CHD Candidate Variant Characterization). Expression specificity was calculated using data from median transcripts-per-million (tpm) as published in GTEx version 8.1.1.9 (<span class="dropBlock reference-citations"><a id="body-ref-sref11-2" href="#" role="doc-biblioref" data-xml-rid="bib11" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref11-2" aria-controls="sref11-2">GTEx Consortium, 2017</a><div class="dropBlock__holder" data-db-target-of="sref11-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">11.</div><div>GTEx Consortium</div><div><strong>Genetic effects on gene expression across human tissues</strong></div><div><em>Nature.</em> 2017; <strong>550</strong>:204-213</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature24277" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-85031313737" target="_blank" aria-label="Open Scopus in new tab">Scopus (1802)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29022597/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature24277&amp;pmid=29022597" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Average median tpm was calculated for heart tissues (adult atrium, adult left ventricle) and all other available tissues with the exception of testis. The specificity score is then defined as the average tpm in heart tissues normalized by average tpm across all tissues.</div><div id="p0535" role="paragraph">For each of these properties, the variants were ranked based on their relative scores. Ties were resolved by taking the average value of the would-be ranks. Missing data was imputed to the median value of the given property. Gene-level rankings (mutations per kilobase, pLI score, CHD-gene degree, variant degree, and expression specificity) and residue-level rankings (CADD score) were separately averaged and then summed. This average rank sum was then additionally weighted by two factors to capture aspects of their proband-level and protein contexts.</div><div id="p0540" role="paragraph">First, if the proband had additional mutations in other interactome genes or other previously-identified CHD genes, we reduced the variant’s weight. Specifically, we multiply the rank-sum score by the lowest-applicable factor if they meet any of these conditions:<div class="figure-wrap"><figure id="undtbl2" class="table" data-figureviewer-ignore="true"><div class="table-wrap"><table data-figureviewer-ignore="true"><thead><tr class="rowsep"><th>Factor</th><th>Conditions</th></tr></thead><tbody><tr><td>0.75</td><td>Proband has another rare (MAF 10<sup>−5</sup>) inherited loss-of-function OR missense <i>de novo</i> variant in an interactome gene<br>OR proband has an inherited missense damaging variant in a known CHD gene</td></tr><tr><td>0.50</td><td>Proband has a predicted-damaging <i>de novo</i> mutation in an interactome gene or rare inherited loss-of-function mutation in a previously-identified CHD gene</td></tr><tr><td>0.25</td><td>Proband has a <i>de novo</i> missense mutation in a previously-identified CHD gene</td></tr><tr><td>0.10</td><td>Proband had a <i>de novo</i> missense mutation in a previously-identified CHD gene, and that variant was predicted-damaging or led to protein loss-of-function.</td></tr></tbody></table></div><figcaption><div class="core-table-tools"><ul><li><a class="core-table-show-action" href="/action/showFullTableHTML?isHtml=true&amp;tableId=undtbl2&amp;pii=S0092-8674%2822%2900079-4" target="_blank">Open table in a new tab</a></li></ul></div></figcaption></figure></div></div><div id="p0545" role="paragraph">To summarize, the variant is down-weighted in cases where it is likely that another mutation in the proband is causing or contributing to the CHD phenotype.</div><div id="p0550" role="paragraph">Second, if the <i>de novo</i> variant leads to protein loss-of-function, or if it occurred in a known protein domain (and therefore is suspected to interfere with protein activity), the variant rank-sum was transmitted as-is. Otherwise, the variant’s rank-sum was multiplied by 0.5.</div></section><section id="sec-4-4-16-8"><h5>Precision-recall analysis</h5><div id="p0555" role="paragraph">The ability of the Variant Prioritization Score to predict variants known to result in CHD was performed under two scenarios using a Precision-Recall (PR) Analysis. In the first scenario, prediction was performed over all missense DNVs, and in the second one, prediction was restricted to the GT-PPI missense DNVs. The PR analysis was performed using the pr.curve function that is part of the PRROC (<span class="dropBlock reference-citations"><a id="body-ref-sref49-2" href="#" role="doc-biblioref" data-xml-rid="bib49" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref49-2" aria-controls="sref49-2">Grau et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref49-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">44.</div><div>Grau, J. ∙ Grosse, I. ∙ Keilwagen, J.</div><div><strong>PRROC: computing and visualizing precision-recall and receiver operating characteristic curves in R</strong></div><div><em>Bioinformatics.</em> 2015; <strong>31</strong>:2595-2597</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btv153" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-84943599060" target="_blank" aria-label="Open Scopus in new tab">Scopus (170)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25810428/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtv153&amp;pmid=25810428" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) package in R. The performance of the score was quantified in terms of the Area-Under the Curve (AUC) for the PR curves generated under the two scenarios, relative to the expected AUC from a random classifier under the corresponding scenarios. The expected AUC for the PR curve from a random classifier is equal to the fraction of variants known to cause CHD among all variants used in the prediction.</div></section><section id="sec-4-4-16-9"><h5>GLYR1 model organisms alignment</h5><div id="p0560" role="paragraph">Sequences of several vertebrate model organisms containing the rigid loop (bridging the two tetramerization alpha helice bundles) of the NPAC proteins’ dehydrogenase domain were aligned using CLC Sequence Viewer 8.0. Amino acids 490-529 were aligned, partially spanning exons 14 and 15 (490-495, 496-5229 respectively) in the <i>H.&nbsp;sapiens</i> sequence. Alignments were created with the “Alignment” function with a gap open cost of 10.0, gap extension cost of 1.0, end gap cost as any other, and the very accurate (slow parameter).</div><div id="p0565" role="paragraph">Used mRNA (NM) and predicted mRNA (XM) Sequences<div id="ulist0025" role="list"><div id="u0065" role="listitem"><div class="content"><div id="p0570" role="paragraph">Chimp (<i>Pan Traglodyte</i>): XM_016929357</div></div></div><div id="u0070" role="listitem"><div class="content"><div id="p0575" role="paragraph">Gorilla (<i>Gorilla gorilla</i>): XM_019012358</div></div></div><div id="u0075" role="listitem"><div class="content"><div id="p0580" role="paragraph">Human (<i>Homo sapien</i>): NM_032569</div></div></div><div id="u0080" role="listitem"><div class="content"><div id="p0585" role="paragraph">Mouse (<i>Mus musculus</i>): NM_001359747.1</div></div></div><div id="u0085" role="listitem"><div class="content"><div id="p0590" role="paragraph">Rat (<i>Rattus norvegicus</i>): NM_001007800</div></div></div><div id="u0090" role="listitem"><div class="content"><div id="p0595" role="paragraph">Chicken (<i>Gallus gallus</i>): NM_001006572</div></div></div><div id="u0095" role="listitem"><div class="content"><div id="p0600" role="paragraph">Frog (<i>Xenopus laevis</i>): NM_001030494</div></div></div><div id="u0100" role="listitem"><div class="content"><div id="p0605" role="paragraph">Zebrafish (<i>Danio rerio</i>): XM_005164104</div></div></div></div></div></section><section id="sec-4-4-16-10"><h5>GLYR1 structural model</h5><div id="p0610" role="paragraph">The wildtype GLYR1 structure was imported from the RCSB Protein Data Bank, entry 2UYY, the structure being elucidated through X-ray diffraction (<span class="dropBlock reference-citations"><a id="body-ref-sref131" href="#" role="doc-biblioref" data-xml-rid="bib131" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref131" aria-controls="sref131">Tickle. et&nbsp;al., 2007</a><div class="dropBlock__holder" data-db-target-of="sref131"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">118.</div><div>Tickle, J. ∙ Pilka, E.S. ∙ Bunkoczi, G. ...</div><div><strong>Structure of the cytokine-like nuclear factor n-pac</strong></div><div><a id="intref0270" href="https://www.wwpdb.org/pdb?id=pdb_00002uyy" target="_blank">https://www.wwpdb.org/pdb?id=pdb_00002uyy</a><div>Date: 2007</div></div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.TickleE.S.PilkaG.BunkocziG.BerridgeC.SmeeK.L.KavanaghV.HozjanF.H.NiesenE.PapagrigoriouA.C.W.PikeStructure+of+the+cytokine-like+nuclear+factor+n-pachttps%3A%2F%2Fwww.wwpdb.org%2Fpdb%3Fid%3Dpdb_00002uyy2007" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The structure and domains of the NPAC monomer were edited using the PYMOL Molecular Graphics System Version 2.3.5. Domains of the NPAC dehydrogenase domain are defined as by Zhang et&nbsp;al., 2014 (<span class="dropBlock reference-citations"><a id="body-ref-sref146" href="#" role="doc-biblioref" data-xml-rid="bib146" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref146" aria-controls="sref146">Zhang et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref146"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">132.</div><div>Zhang, Y. ∙ Zheng, Y. ∙ Qin, L. ...</div><div><strong>Structural characterization of a β-hydroxyacid dehydrogenase from Geobacter sulfurreducens and Geobacter metallireducens with succinic semialdehyde reductase activity</strong></div><div><em>Biochimie.</em> 2014; <strong>104</strong>:61-69</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.biochi.2014.05.002" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_132_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-84905507214" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24878278/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.biochi.2014.05.002&amp;pmid=24878278" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The protein is shown through the cartoon function, displaying the general tertiary structure of the protein. Amino acid proline 496 and its mutant proline 496 leucine are shown in gold and through the stick function displaying the secondary structure of the amino acids to delineate the significance in change of structure. In the focused images of the rigid loop of the alpha helices tetramerization bundle, the amino acids are again shown through the stick function to delineate the secondary structure interactions of the amino acids.</div></section><section id="sec-4-4-16-11"><h5>Molecular structural dynamics methods</h5><div id="p0615" role="paragraph">The initial protein structure for all-atom MD simulations in explicit water of NPAC was downloaded from the Protein Data Bank, code 2uyy.pdb.</div><div id="p0620" role="paragraph">Missing atoms and side chains were added using the Protein Preparation Wizard of the Maestro Suite of Programs (v. 2019−4). Proline to Leucine mutation was also performed using Maestro (<span class="dropBlock reference-citations"><a id="body-ref-sref89" href="#" role="doc-biblioref" data-xml-rid="bib89" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref89" aria-controls="sref89">Maestro Schrödinger, LLC, 2019</a><div class="dropBlock__holder" data-db-target-of="sref89"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">77.</div><div>Maestro Schrödinger, LLC</div><div><strong>Maestro Suite of Programs (v. 2019-4)</strong></div><div>Maestro Schrödinger, LLC, 2019</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=Maestro+Schr%C3%B6dinger%2C+LLCMaestro+Suite+of+Programs+%28v.+2019-4%292019Maestro+Schr%C3%B6dinger%2C+LLC" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref120" href="#" role="doc-biblioref" data-xml-rid="bib120" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref120" aria-controls="sref120">Sastry et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref120"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">108.</div><div>Sastry, G.M. ∙ Adzhigirey, M. ∙ Day, T. ...</div><div><strong>Protein and ligand preparation: parameters, protocols, and influence on virtual screening enrichments</strong></div><div><em>J.&nbsp;Comput. Aided Mol. Des.</em> 2013; <strong>27</strong>:221-234</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/s10822-013-9644-8" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_108_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-84880529288" target="_blank" aria-label="Open Scopus in new tab">Scopus (2784)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23579614/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2Fs10822-013-9644-8&amp;pmid=23579614" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The simulations were run using the same protocol for both the WT and mutated monomer (subunit A).</div><div id="p0625" role="paragraph">All systems were allowed to relax with 2000 steps of steepest descent followed by another 2000 steps of conjugate gradient energy minimization. The temperature of the systems was gradually raised to 300 K in the NVT ensemble in 1.2&nbsp;ns at 1&nbsp;fs time-step, using the Langevin thermostat. In particular, six runs of 200&nbsp;ps were performed increasing the temperature of 50 K at each step (T&nbsp;= 50, 100, 150, 200, 250, and 300 K, respectively). At 300 K, the density of the system was adjusted with 1&nbsp;ns at 2&nbsp;fs time-step under NPT conditions by weak coupling to a bath of constant pressure (P<sub>0</sub>&nbsp;= 1 bar, coupling time t<sub>p</sub>&nbsp;= 0.5 ps). The production runs were thus carried out in the NVT ensemble. Bonds involving hydrogen atoms were constrained with the SHAKE algorithm (<span class="dropBlock reference-citations"><a id="body-ref-sref94" href="#" role="doc-biblioref" data-xml-rid="bib94" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref94" aria-controls="sref94">Miyamoto and Kollman 1992</a><div class="dropBlock__holder" data-db-target-of="sref94"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">82.</div><div>Miyamoto, S. ∙ Kollman, P.A.</div><div><strong>Settle: An analytical version of the SHAKE and RATTLE algorithm for rigid water models</strong></div><div><em>J.&nbsp;Comput. Chem.</em> 1992; <strong>13</strong>:952-962</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/jcc.540130805" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fjcc.540130805" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), allowing a time step of 2 fs. Electrostatic forces were computed using the particle mesh Ewald algorithm with a truncation cut-off of 10A° (<span class="dropBlock reference-citations"><a id="body-ref-sref27" href="#" role="doc-biblioref" data-xml-rid="bib27" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref27" aria-controls="sref27">Darden et&nbsp;al., 1993</a><div class="dropBlock__holder" data-db-target-of="sref27"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">26.</div><div>Darden, T. ∙ York, D. ∙ Pedersen, L.</div><div><strong>Particle mesh Ewald: An N×log(N) method for Ewald sums in large systems</strong></div><div><em>J.&nbsp;Chem. Phys.</em> 1993; <strong>98</strong>:10089</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1063/1.464397" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1063%2F1.464397" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The initial velocity of all atoms was obtained from a Maxwellian distribution at the initial temperature of 300 K.</div><div id="p0630" role="paragraph">MD simulations where run in 3 independent replicas of 500&nbsp;ns each (1.5&nbsp;μs in total per system),</div><div id="p0635" role="paragraph">Specifically, MD simulations were performed using Amber18 pmemd.CUDA with the all atom ff14SB force field under periodic boundary conditions (<span class="dropBlock reference-citations"><a id="body-ref-sref19-2" href="#" role="doc-biblioref" data-xml-rid="bib19" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref19-2" aria-controls="sref19-2">Case et al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref19-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">19.</div><div>Case, D.A. ∙ Ben-Shalom, I.Y. ∙ Brozell, S.R. ...</div><div><strong>AMBER 2018</strong></div><div>University of California, San Francisco, 2018</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=D.A.CaseI.Y.Ben-ShalomS.R.BrozellD.S.CeruttiT.E.Cheatham%2C+IIIV.W.D.CruzeiroT.A.DardenR.E.DukeD.GhoreishiM.K.GilsonAMBER+20182018University+of+California%2C+San+Francisco" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The triclinic simulative box, filled with TIP3P (<span class="dropBlock reference-citations"><a id="body-ref-sref65" href="#" role="doc-biblioref" data-xml-rid="bib65" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref65" aria-controls="sref65">Jorgensen et&nbsp;al., 1983</a><div class="dropBlock__holder" data-db-target-of="sref65"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">58.</div><div>Jorgensen, W.L. ∙ Chandrasekhar, J. ∙ Madura, J.D. ...</div><div><strong>Comparison of simple potential functions for simulating liquid water</strong></div><div><em>J.&nbsp;Chem. Phys.</em> 1983; <strong>79</strong>:926</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1063/1.445869" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1063%2F1.445869" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) water molecules and rendered electroneutral by addition of Na<sup>+</sup> counterions consists of a final number of atoms of about 41 300 (monomer WT and P496L mutant), particles for each system.</div><div id="p0640" role="paragraph">The atomic positions were saved every 10 ps. The equilibrated parts of the trajectories were used for subsequent analyses. Equilibration of the trajectories was checked by monitoring the equilibration of the RMSD with respect to the initial structure and of the internal protein energy. The equilibrated parts of each trajectory for the two systems were next combined into a meta-trajectory, which was subsequently used for all the reported characterizations. Classical structural analyses were carried out with the tools in the Amber18 and Gromacs 4.5.5 package (<span class="dropBlock reference-citations"><a id="body-ref-sref12" href="#" role="doc-biblioref" data-xml-rid="bib12" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref12" aria-controls="sref12">Bekker et&nbsp;al., 1993</a><div class="dropBlock__holder" data-db-target-of="sref12"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">12.</div><div>Bekker, H. ∙ Berendsen, H.J.C. ∙ Dijkstra, E.J. ...</div><div><strong>Gromacs: A parallel computer for molecular dynamics simulations</strong></div><div><a id="intref0265" href="https://www.scienceopen.com/document?vid=59290415-039c-4900-8e95-d649687a2473" target="_blank">https://www.scienceopen.com/document?vid=59290415-039c-4900-8e95-d649687a2473</a><div>Date: 1993</div></div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=H.BekkerH.J.C.BerendsenE.J.DijkstraS.AchteropR.van+DrunenD.van+der+SpoelA.SijbersH.KeegstraB.ReitsmaM.K.R.RenardusGromacs%3A+A+parallel+computer+for+molecular+dynamics+simulationshttps%3A%2F%2Fwww.scienceopen.com%2Fdocument%3Fvid%3D59290415-039c-4900-8e95-d649687a24731993" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) or with code written in-house.</div><div id="p0645" role="paragraph">The <i>root mean square deviation</i> (RMSD) of the backbone of the protein with respect to first frame of the trajectory along the simulation time has been calculated by least-square fitting the structure to the reference structure (t2&nbsp;= 0) and subsequently calculating the RMSD<div id="ufd1" class="display-formula"><div class="equation" role="math"><div class="inner"><mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="normal upper R normal upper M normal upper S normal upper D left parenthesis normal t Baseline 1 comma normal t Baseline 2 right parenthesis equals StartRoot left bracket StartFraction 1 Over upper M EndFraction sigma summation Underscript i equals 1 Overscript upper N Endscripts m i StartMetric r i left parenthesis t Baseline 1 right parenthesis minus r i left parenthesis t Baseline 2 right parenthesis EndMetric 2 right bracket EndRoot" tabindex="0" ctxtmenu_counter="3" style="font-size: 130%;"><mjx-math data-semantic-structure="(81 (80 0 77 1 78 2 79 (76 3 75 (16 4 (14 (11 5 10 6) 7 (13 8 12 9)) 15))) 17 (74 (73 18 (71 (21 19 20) 70 (69 (28 22 (26 23 24 25) 27) (68 29 64 (63 30 62 (61 31 (59 (56 32 55 (54 33 53 (40 34 (38 35 37 36) 39))) 41 (58 42 57 (52 43 51 (50 44 (48 45 47 46) 49)))) 60)) 67 66))) 72)))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="81" data-semantic-children="80,74" data-semantic-content="17" data-semantic-owns="80 17 74" data-semantic-speech="normal upper R normal upper M normal upper S normal upper D left parenthesis normal t Baseline 1 comma normal t Baseline 2 right parenthesis equals StartRoot left bracket StartFraction 1 Over upper M EndFraction sigma summation Underscript i equals 1 Overscript upper N Endscripts m i StartMetric r i left parenthesis t Baseline 1 right parenthesis minus r i left parenthesis t Baseline 2 right parenthesis EndMetric 2 right bracket EndRoot"><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="80" data-semantic-children="0,1,2,76" data-semantic-content="77,78,79" data-semantic-parent="81" data-semantic-owns="0 77 1 78 2 79 76"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="80"><mjx-c class="mjx-c52">R</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="77" data-semantic-parent="80" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="80"><mjx-c class="mjx-c4D">M</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="78" data-semantic-parent="80" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="80"><mjx-c class="mjx-c53">S</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="79" data-semantic-parent="80" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mrow data-semantic-added="true" data-semantic-type="appl" data-semantic-role="simple function" data-semantic-id="76" data-semantic-children="3,16" data-semantic-content="75,3" data-semantic-parent="80" data-semantic-owns="3 75 16"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="76" data-semantic-operator="appl"><mjx-c class="mjx-c44">D</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="75" data-semantic-parent="76" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="16" data-semantic-children="14" data-semantic-content="4,15" data-semantic-parent="76" data-semantic-owns="4 14 15"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="4" data-semantic-parent="16" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="14" data-semantic-children="11,7,13" data-semantic-content="7" data-semantic-parent="16" data-semantic-owns="11 7 13"><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="11" data-semantic-children="5,6" data-semantic-content="10" data-semantic-parent="14" data-semantic-owns="5 10 6"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="5" data-semantic-parent="11"><mjx-c class="mjx-c74">t</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="10" data-semantic-parent="11" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="6" data-semantic-parent="11"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-mrow><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="7" data-semantic-parent="14" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="13" data-semantic-children="8,9" data-semantic-content="12" data-semantic-parent="14" data-semantic-owns="8 12 9"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="8" data-semantic-parent="13"><mjx-c class="mjx-c74">t</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="12" data-semantic-parent="13" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="9" data-semantic-parent="13"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn></mjx-mrow></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="15" data-semantic-parent="16" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-mrow><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="17" data-semantic-parent="81" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-msqrt space="4" data-semantic-type="sqrt" data-semantic-role="unknown" data-semantic-id="74" data-semantic-children="73" data-semantic-parent="81" data-semantic-owns="73"><mjx-sqrt><mjx-surd><mjx-mo><mjx-c class="mjx-c221A MM-LO">√</mjx-c></mjx-mo></mjx-surd><mjx-box style="padding-top: 0.182em;"><mjx-mrow data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="73" data-semantic-children="71" data-semantic-content="18,72" data-semantic-parent="74" data-semantic-owns="18 71 72"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="18" data-semantic-parent="73" data-semantic-operator="fenced"><mjx-c class="mjx-c5B MM-S3">[</mjx-c></mjx-mo><mjx-mrow><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="71" data-semantic-children="21,69" data-semantic-content="70" data-semantic-parent="73" data-semantic-owns="21 70 69"><mjx-mfrac data-semantic-type="fraction" data-semantic-role="division" data-semantic-id="21" data-semantic-children="19,20" data-semantic-parent="71" data-semantic-owns="19 20"><mjx-frac><mjx-num><mjx-nstrut></mjx-nstrut><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="19" data-semantic-parent="21"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-num><mjx-dbox><mjx-dtable><mjx-line></mjx-line><mjx-row><mjx-den><mjx-dstrut></mjx-dstrut><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="20" data-semantic-parent="21"><mjx-c class="mjx-c1D440">𝑀</mjx-c></mjx-mi></mjx-den></mjx-row></mjx-dtable></mjx-dbox></mjx-frac></mjx-mfrac><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="70" data-semantic-parent="71" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-added="true" data-semantic-type="bigop" data-semantic-role="sum" data-semantic-id="69" data-semantic-children="28,68" data-semantic-content="22" data-semantic-parent="71" data-semantic-owns="28 68"><mjx-msubsup data-semantic-type="limboth" data-semantic-role="sum" data-semantic-id="28" data-semantic-children="22,26,27" data-semantic-parent="69" data-semantic-owns="22 26 27"><mjx-mo data-semantic-type="largeop" data-semantic-role="sum" data-semantic-id="22" data-semantic-parent="28" data-semantic-operator="bigop"><mjx-c class="mjx-c2211">∑</mjx-c></mjx-mo><mjx-script style="vertical-align: -0.285em; margin-left: 0px;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="27" data-semantic-parent="28"><mjx-c class="mjx-c1D441">𝑁</mjx-c></mjx-mi><mjx-spacer style="margin-top: 0.291em;"></mjx-spacer><mjx-mrow size="s" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="26" data-semantic-children="23,25" data-semantic-content="24" data-semantic-parent="28" data-semantic-owns="23 24 25"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="23" data-semantic-parent="26"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-mo data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="24" data-semantic-parent="26" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="25" data-semantic-parent="26"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-mrow></mjx-script></mjx-msubsup><mjx-mrow space="2"><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="68" data-semantic-children="29,63,66" data-semantic-content="64,67" data-semantic-parent="69" data-semantic-owns="29 64 63 67 66"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="29" data-semantic-parent="68"><mjx-c class="mjx-c1D45A">𝑚</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="64" data-semantic-parent="68" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mrow data-semantic-added="true" data-semantic-type="appl" data-semantic-role="simple function" data-semantic-id="63" data-semantic-children="30,61" data-semantic-content="62,30" data-semantic-parent="68" data-semantic-owns="30 62 61"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="30" data-semantic-parent="63" data-semantic-operator="appl"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="62" data-semantic-parent="63" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="metric" data-semantic-id="61" data-semantic-children="59" data-semantic-content="31,60" data-semantic-parent="63" data-semantic-owns="31 59 60"><mjx-mo data-semantic-type="fence" data-semantic-role="metric" data-semantic-id="31" data-semantic-parent="61" data-semantic-operator="fenced"><mjx-c class="mjx-c2016">‖</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="subtraction" data-semantic-id="59" data-semantic-children="56,58" data-semantic-content="41" data-semantic-parent="61" data-semantic-owns="56 41 58"><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="56" data-semantic-children="32,54" data-semantic-content="55" data-semantic-parent="59" data-semantic-owns="32 55 54"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="32" data-semantic-parent="56"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="55" data-semantic-parent="56" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mrow data-semantic-added="true" data-semantic-type="appl" data-semantic-role="simple function" data-semantic-id="54" data-semantic-children="33,40" data-semantic-content="53,33" data-semantic-parent="56" data-semantic-owns="33 53 40"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="33" data-semantic-parent="54" data-semantic-operator="appl"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="53" data-semantic-parent="54" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="40" data-semantic-children="38" data-semantic-content="34,39" data-semantic-parent="54" data-semantic-owns="34 38 39"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="34" data-semantic-parent="40" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="38" data-semantic-children="35,36" data-semantic-content="37" data-semantic-parent="40" data-semantic-owns="35 37 36"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="35" data-semantic-parent="38"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="37" data-semantic-parent="38" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="36" data-semantic-parent="38"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="39" data-semantic-parent="40" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-mrow><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="41" data-semantic-parent="59" data-semantic-operator="infixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mrow space="3" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="58" data-semantic-children="42,52" data-semantic-content="57" data-semantic-parent="59" data-semantic-owns="42 57 52"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="42" data-semantic-parent="58"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="57" data-semantic-parent="58" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mrow data-semantic-added="true" data-semantic-type="appl" data-semantic-role="simple function" data-semantic-id="52" data-semantic-children="43,50" data-semantic-content="51,43" data-semantic-parent="58" data-semantic-owns="43 51 50"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="43" data-semantic-parent="52" data-semantic-operator="appl"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="51" data-semantic-parent="52" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="50" data-semantic-children="48" data-semantic-content="44,49" data-semantic-parent="52" data-semantic-owns="44 48 49"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="44" data-semantic-parent="50" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="48" data-semantic-children="45,46" data-semantic-content="47" data-semantic-parent="50" data-semantic-owns="45 47 46"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="45" data-semantic-parent="48"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="47" data-semantic-parent="48" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="46" data-semantic-parent="48"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="49" data-semantic-parent="50" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-mrow></mjx-mrow><mjx-mo space="2" data-semantic-type="fence" data-semantic-role="metric" data-semantic-id="60" data-semantic-parent="61" data-semantic-operator="fenced"><mjx-c class="mjx-c2016">‖</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-mrow></mjx-mrow></mjx-mrow></mjx-mrow><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="67" data-semantic-parent="68" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mn space="2" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="66" data-semantic-parent="68"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="72" data-semantic-parent="73" data-semantic-operator="fenced"><mjx-c class="mjx-c5D MM-S3">]</mjx-c></mjx-mo></mjx-mrow></mjx-box></mjx-sqrt></mjx-msqrt></mjx-mrow></mjx-math></mjx-container></div></div></div>where <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper M equals sigma summation Underscript i equals 1 Overscript upper N Endscripts m Subscript i" tabindex="0" ctxtmenu_counter="4" style="font-size: 130%;"><mjx-math data-semantic-structure="(13 0 1 (12 (8 2 (6 3 4 5) 7) (11 9 10)))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="13" data-semantic-children="0,12" data-semantic-content="1" data-semantic-owns="0 1 12" data-semantic-speech="upper M equals sigma summation Underscript i equals 1 Overscript upper N Endscripts m Subscript i"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="13"><mjx-c class="mjx-c1D440">𝑀</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="1" data-semantic-parent="13" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mrow space="4" data-semantic-added="true" data-semantic-type="bigop" data-semantic-role="sum" data-semantic-id="12" data-semantic-children="8,11" data-semantic-content="2" data-semantic-parent="13" data-semantic-owns="8 11"><mjx-msubsup data-semantic-type="limboth" data-semantic-role="sum" data-semantic-id="8" data-semantic-children="2,6,7" data-semantic-parent="12" data-semantic-owns="2 6 7"><mjx-mo data-semantic-type="largeop" data-semantic-role="sum" data-semantic-id="2" data-semantic-parent="8" data-semantic-operator="bigop"><mjx-c class="mjx-c2211">∑</mjx-c></mjx-mo><mjx-script style="vertical-align: -0.285em; margin-left: 0px;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="7" data-semantic-parent="8"><mjx-c class="mjx-c1D441">𝑁</mjx-c></mjx-mi><mjx-spacer style="margin-top: 0.291em;"></mjx-spacer><mjx-mrow size="s" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="6" data-semantic-children="3,5" data-semantic-content="4" data-semantic-parent="8" data-semantic-owns="3 4 5"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="6"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-mo data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="4" data-semantic-parent="6" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="5" data-semantic-parent="6"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-mrow></mjx-script></mjx-msubsup><mjx-msub space="2" data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="11" data-semantic-children="9,10" data-semantic-parent="12" data-semantic-owns="9 10"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="9" data-semantic-parent="11"><mjx-c class="mjx-c1D45A">𝑚</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="10" data-semantic-parent="11"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-mrow></mjx-math></mjx-container> and r<sub>i</sub>(t) is the position of atom <i>I</i> at time <i>t</i>.</div><div id="p0650" role="paragraph">The <i>RMSF</i> which is a measure of the displacement of each residue averaged over the number of atoms considered, has been calculated relative to the average structure, in the equilibrated part of the simulation.</div></section><section id="sec-4-4-16-12"><h5>Differential gene expression</h5><div id="p0655" role="paragraph">In order to identify genes differentially expressed between WT CP versus GATA4-KO or TBX5 CPs (n&nbsp;= 5); siControl versus siGATA4 CPs (n&nbsp;= 3) and siControl.2 versus si<i>GLYR1</i> CPs (n&nbsp;= 2), the analyses start with raw reads/sequences in FASTQ format. Trimming of known adapters and low-quality regions of reads was performed using Fastq-mcf (<span class="dropBlock reference-citations"><a id="body-ref-sref5" href="#" role="doc-biblioref" data-xml-rid="bib5" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref5" aria-controls="sref5">Aronesty 2013</a><div class="dropBlock__holder" data-db-target-of="sref5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">5.</div><div>Aronesty, E.</div><div><strong>Comparison of Sequencing Utility Programs</strong></div><div><em>The Open Bioinformatics Journal.</em> 2013; <strong>7</strong>:1-8</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.2174/1875036201307010001" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-84875098273" target="_blank" aria-label="Open Scopus in new tab">Scopus (722)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.2174%2F1875036201307010001" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Sequence quality control was assessed using the program FastQC (<span class="dropBlock reference-citations"><a id="body-ref-sref3" href="#" role="doc-biblioref" data-xml-rid="bib3" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref3" aria-controls="sref3">Andrews 2007</a><div class="dropBlock__holder" data-db-target-of="sref3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">3.</div><div>Andrews, S.</div><div><strong>A quality control tool for high throughput sequence data</strong></div><div><em>babraham bioinformatics.</em> 2007;</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=S.AndrewsA+quality+control+tool+for+high+throughput+sequence+datababraham+bioinformatics2007" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and rSeQC (<span class="dropBlock reference-citations"><a id="body-ref-sref137" href="#" role="doc-biblioref" data-xml-rid="bib137" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref137" aria-controls="sref137">Wang et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref137"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">124.</div><div>Wang, L. ∙ Wang, S. ∙ Li, W.</div><div><strong>RSeQC: quality control of RNA-seq experiments</strong></div><div><em>Bioinformatics.</em> 2012; <strong>28</strong>:2184-2185</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/bts356" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_124_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-84865074382" target="_blank" aria-label="Open Scopus in new tab">Scopus (1107)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22743226/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbts356&amp;pmid=22743226" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Alignment of the provided samples to the reference genome was performed using STAR 2.5.2a (<span class="dropBlock reference-citations"><a id="body-ref-sref31" href="#" role="doc-biblioref" data-xml-rid="bib31" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref31" aria-controls="sref31">Dobin et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref31"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">29.</div><div>Dobin, A. ∙ Davis, C.A. ∙ Schlesinger, F. ...</div><div><strong>STAR: ultrafast universal RNA-seq aligner</strong></div><div><em>Bioinformatics.</em> 2013; <strong>29</strong>:15-21</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/bts635" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-84871809302" target="_blank" aria-label="Open Scopus in new tab">Scopus (17570)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23104886/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbts635&amp;pmid=23104886" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Reads were aligned to the human hg19 reference assembly indicated in the header of the differential expression file. Reads were assigned to genes using <i>featureCounts</i> (<span class="dropBlock reference-citations"><a id="body-ref-sref83-1" href="#" role="doc-biblioref" data-xml-rid="bib83" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref83-1" aria-controls="sref83-1">Liao et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref83-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">71.</div><div>Liao, Y. ∙ Smyth, G.K. ∙ Shi, W.</div><div><strong>featureCounts: an efficient general purpose program for assigning sequence reads to genomic features</strong></div><div><em>Bioinformatics.</em> 2014; <strong>30</strong>:923-930</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btt656" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-84897397058" target="_blank" aria-label="Open Scopus in new tab">Scopus (7989)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24227677/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtt656&amp;pmid=24227677" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), part of the Subread suite (<a id="intref0245" href="http://subread.sourceforge.net/" target="_blank">http://subread.sourceforge.net/</a>). Gene-level counts were arrived at using Ensembl gene annotation, in GTF format. Differential expression was assessed using edgeR (<span class="dropBlock reference-citations"><a id="body-ref-sref118-2" href="#" role="doc-biblioref" data-xml-rid="bib118" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref118-2" aria-controls="sref118-2">Robinson et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref118-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">106.</div><div>Robinson, M.D. ∙ McCarthy, D.J. ∙ Smyth, G.K.</div><div><strong>edgeR: a Bioconductor package for differential expression analysis of digital gene expression data</strong></div><div><em>Bioinformatics.</em> 2010; <strong>26</strong>:139-140</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btp616" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_106_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-75249087100" target="_blank" aria-label="Open Scopus in new tab">Scopus (19590)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19910308/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtp616&amp;pmid=19910308" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), an R package available through Bioconductor. Genes where there were not at least two samples with at least 5 (raw) reads were filtered out from further analyses. The reads counts of remaining ones are normalized for sample-to-sample variation using <i>calcNormFactors</i> in edgeR (<span class="dropBlock reference-citations"><a id="body-ref-sref118-3" href="#" role="doc-biblioref" data-xml-rid="bib118" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref118-3" aria-controls="sref118-3">Robinson et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref118-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">106.</div><div>Robinson, M.D. ∙ McCarthy, D.J. ∙ Smyth, G.K.</div><div><strong>edgeR: a Bioconductor package for differential expression analysis of digital gene expression data</strong></div><div><em>Bioinformatics.</em> 2010; <strong>26</strong>:139-140</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btp616" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_106_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-75249087100" target="_blank" aria-label="Open Scopus in new tab">Scopus (19590)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19910308/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtp616&amp;pmid=19910308" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The mean gene expression was modeled as a function of siRNA status (siRNA treatment versus scramble control) and sample id. Genes whose expression is associated with siRNA status were determined by the likelihood ratio test (<span class="dropBlock reference-citations"><a id="body-ref-sref124" href="#" role="doc-biblioref" data-xml-rid="bib124" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref124" aria-controls="sref124">Smyth 1996</a><div class="dropBlock__holder" data-db-target-of="sref124"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">112.</div><div>Smyth, G.K.</div><div><strong>A conditional approach to residual maximum likelihood estimation in generalized linear models</strong></div><div>R. Stat. Soc. B., 1996</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1111/j.2517-6161.1996.tb02101.x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1111%2Fj.2517-6161.1996.tb02101.x" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref116" href="#" role="doc-biblioref" data-xml-rid="bib116" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref116" aria-controls="sref116">Robinson and Smyth 2007</a><div class="dropBlock__holder" data-db-target-of="sref116"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">104.</div><div>Robinson, M.D. ∙ Smyth, G.K.</div><div><strong>Moderated statistical tests for assessing differences in tag abundance</strong></div><div><em>Bioinformatics.</em> 2007; <strong>23</strong>:2881-2887</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btm453" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_104_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-36448981743" target="_blank" aria-label="Open Scopus in new tab">Scopus (554)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17881408/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtm453&amp;pmid=17881408" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref117" href="#" role="doc-biblioref" data-xml-rid="bib117" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref117" aria-controls="sref117">Robinson and Smyth 2008</a><div class="dropBlock__holder" data-db-target-of="sref117"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">105.</div><div>Robinson, M.D. ∙ Smyth, G.K.</div><div><strong>Small-sample estimation of negative binomial dispersion, with applications to SAGE data</strong></div><div><em>Biostatistics.</em> 2008; <strong>9</strong>:321-332</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/biostatistics/kxm030" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_105_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-41149085992" target="_blank" aria-label="Open Scopus in new tab">Scopus (673)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17728317/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbiostatistics%2Fkxm030&amp;pmid=17728317" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) implemented in edgeR using a FDR&nbsp;&lt; 0.05 and LogFC&nbsp;&lt; −0.25 threshold.</div></section><section id="sec-4-4-16-13"><h5>Pathway enrichment analysis</h5><div id="p0660" role="paragraph">Functional enrichment gene-set analysis for GO (Gene Ontology) terms was performed using ToppGene Suite (<a id="intref0250" href="https://toppgene.cchmc.org/enrichment.jsp" target="_blank">https://toppgene.cchmc.org/enrichment.jsp</a>) using all <i>Homo sapiens</i> genes as background. Statistically significant (Bonferroni q-value&nbsp;&lt; 0.05) categories within the GO:Biological Process section were extracted and replotted.</div></section><section id="sec-4-4-16-14"><h5>ChIP-seq analysis</h5><div id="p0665" role="paragraph">For the ChIPseq analysis, trimming of known adapters and low-quality regions of reads was performed using Fastq-mcf. Sequence quality control was assessed using FastQC (<a id="intref0255" href="http://www.bioinformatics.babraham.ac.uk/projects/fastqc/" target="_blank">http://www.bioinformatics.babraham.ac.uk/projects/fastqc/</a>). Alignment to the hg19 reference genome was performed using Bowtie 2.2.4 (<span class="dropBlock reference-citations"><a id="body-ref-sref75-2" href="#" role="doc-biblioref" data-xml-rid="bib75" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref75-2" aria-controls="sref75-2">Langmead and Salzberg 2012</a><div class="dropBlock__holder" data-db-target-of="sref75-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">67.</div><div>Langmead, B. ∙ Salzberg, S.L.</div><div><strong>Fast gapped-read alignment with Bowtie 2</strong></div><div><em>Nat. Methods.</em> 2012; <strong>9</strong>:357-359</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.1923" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_67_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-84859210032" target="_blank" aria-label="Open Scopus in new tab">Scopus (24558)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22388286/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.1923&amp;pmid=22388286" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Peaks were called using GEM (<span class="dropBlock reference-citations"><a id="body-ref-sref51-2" href="#" role="doc-biblioref" data-xml-rid="bib51" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref51-2" aria-controls="sref51-2">Guo et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref51-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">46.</div><div>Guo, Y. ∙ Mahony, S. ∙ Gifford, D.K.</div><div><strong>High resolution genome wide binding event finding and motif discovery reveals transcription factor spatial binding constraints</strong></div><div><em>PLoS Comput. Biol.</em> 2012; <strong>8</strong>:e1002638</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pcbi.1002638" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-84866086842" target="_blank" aria-label="Open Scopus in new tab">Scopus (176)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22912568/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pcbi.1002638&amp;pmid=22912568" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) for TFs and BCP (<span class="dropBlock reference-citations"><a id="body-ref-sref142-2" href="#" role="doc-biblioref" data-xml-rid="bib142" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref142-2" aria-controls="sref142-2">Xing et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref142-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">128.</div><div>Xing, H. ∙ Mo, Y. ∙ Liao, W. ...</div><div><strong>Genome-wide localization of protein-DNA binding and histone modification by a Bayesian change-point method with ChIP-seq data</strong></div><div><em>PLoS Comput. Biol.</em> 2012; <strong>8</strong>:e1002613</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pcbi.1002613" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_128_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-84864587283" target="_blank" aria-label="Open Scopus in new tab">Scopus (40)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22844240/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pcbi.1002613&amp;pmid=22844240" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) for GLYR1 and H3K36me3 ChIPseq signals. Read counts per peak were generated with featureCounts (<span class="dropBlock reference-citations"><a id="body-ref-sref83-2" href="#" role="doc-biblioref" data-xml-rid="bib83" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref83-2" aria-controls="sref83-2">Liao et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref83-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">71.</div><div>Liao, Y. ∙ Smyth, G.K. ∙ Shi, W.</div><div><strong>featureCounts: an efficient general purpose program for assigning sequence reads to genomic features</strong></div><div><em>Bioinformatics.</em> 2014; <strong>30</strong>:923-930</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btt656" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-84897397058" target="_blank" aria-label="Open Scopus in new tab">Scopus (7989)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24227677/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtt656&amp;pmid=24227677" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and normalized to account for differences in sequencing depth between samples using upper quartile normalization separately for the ChIP and input sample. Bound regions were determined using empirical Bayes F-tests for a quasi-likelihood negative binomial generalized log-linear model of the count data as implemented in edgeR. Specifically, we tested for a significant (i.e., non-zero at FDR&nbsp;&lt; 5%) log2 fold-increase in normalized peak signal for ChIP versus the corresponding input sample. 2 or 3 separate samples (and relative inputs) were ran from independent ChIP assays.</div><div id="p0670" role="paragraph"><i>GATA4 and GLYR1 ChIP-seq genomic features (</i><a id="crosref2130" href="#figs5">Figure&nbsp;S5</a>E<i>).</i> We obtained the genomic features associated with GATA4 and GLYR1 ChIP-seq peaks using the annotatr (<span class="dropBlock reference-citations"><a id="body-ref-sref21" href="#" role="doc-biblioref" data-xml-rid="bib21" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref21" aria-controls="sref21">Cavalcante and Sartor 2017</a><div class="dropBlock__holder" data-db-target-of="sref21"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">20.</div><div>Cavalcante, R.G. ∙ Sartor, M.A.</div><div><strong>annotatr: genomic regions in context</strong></div><div><em>Bioinformatics.</em> 2017; <strong>33</strong>:2381-2383</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btx183" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-85026423268" target="_blank" aria-label="Open Scopus in new tab">Scopus (130)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28369316/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtx183&amp;pmid=28369316" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) package in R.</div><div id="p0675" role="paragraph"><i>Metagene plot analysis (</i><a id="crosrefs0010" href="#fig1">Figure&nbsp;S</a>5D, S5J, <i>and</i> <a id="crosref2140" href="#fig6">6</a>D<i>).</i> To generate metagene plots, BED files were generated containing regions of interest. The <i>computeMatrix scale-regions</i> module of deepTools (<span class="dropBlock reference-citations"><a id="body-ref-sref111" href="#" role="doc-biblioref" data-xml-rid="bib111" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref111" aria-controls="sref111">Ramírez et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref111"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">99.</div><div>Ramírez, F. ∙ Ryan, D.P. ∙ Grüning, B. ...</div><div><strong>deepTools2: a next generation web server for deep-sequencing data analysis</strong></div><div><em>Nucleic Acids Res.</em> 2016; <strong>44</strong>:W160-5</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gkw257" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_99_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-85017133693" target="_blank" aria-label="Open Scopus in new tab">Scopus (1768)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27079975/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgkw257&amp;pmid=27079975" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), which shrinks or stretches all regions in the input BED file to the same length was used to summarize the ChIP signal profile for each region. The ChIP signal was defined in terms of input subtracted tag densities. Specifically, the human genome, hg19 is divided into 20-bp bins. The tag density or normalized difference between the number of the reads in the ChIP sample and the input sample is computed as:<div id="ufd2" class="display-formula"><div class="equation" role="math"><div class="inner"><mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="number sign upper B i n s upper I n upper G e n o m e times left parenthesis StartFraction number sign c h i p re a d s Over number sign t o t a l upper C h i p re a d s EndFraction minus StartFraction number sign i n p u t re a d s Over number sign t o t a l i n p u t re a d s EndFraction right parenthesis" tabindex="0" ctxtmenu_counter="5" style="font-size: 130%;"><mjx-math data-semantic-structure="(140 0 (139 (138 1 127 2 128 3 129 4 130 5 131 6 132 7 133 8 134 9 135 10 136 11 137 12) 13 (126 14 (124 (66 (34 15 (33 16 29 17 30 18 31 19 32 (28 20 27 (26 21 24 22 25 23)))) (65 35 (64 36 55 37 56 38 57 39 58 40 59 42 60 43 61 44 62 45 63 (54 46 53 (52 47 50 48 51 49))))) 67 (123 (89 68 (88 69 83 70 84 71 85 72 86 73 87 (82 74 81 (80 75 78 76 79 77)))) (122 90 (121 91 111 92 112 93 113 94 114 95 115 97 116 98 117 99 118 100 119 101 120 (110 102 109 (108 103 106 104 107 105)))))) 125)))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="punctuated" data-semantic-role="startpunct" data-semantic-id="140" data-semantic-children="0,139" data-semantic-content="0" data-semantic-owns="0 139" data-semantic-speech="number sign upper B i n s upper I n upper G e n o m e times left parenthesis StartFraction number sign c h i p re a d s Over number sign t o t a l upper C h i p re a d s EndFraction minus StartFraction number sign i n p u t re a d s Over number sign t o t a l i n p u t re a d s EndFraction right parenthesis"><mjx-mo data-semantic-type="punctuation" data-semantic-role="unknown" data-semantic-id="0" data-semantic-parent="140" data-semantic-operator="punctuated"><mjx-c class="mjx-c23">#</mjx-c></mjx-mo><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="multiplication" data-semantic-id="139" data-semantic-children="138,126" data-semantic-content="13" data-semantic-parent="140" data-semantic-owns="138 13 126"><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="138" data-semantic-children="1,2,3,4,5,6,7,8,9,10,11,12" data-semantic-content="127,128,129,130,131,132,133,134,135,136,137" data-semantic-parent="139" data-semantic-owns="1 127 2 128 3 129 4 130 5 131 6 132 7 133 8 134 9 135 10 136 11 137 12"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="138"><mjx-c class="mjx-c1D435">𝐵</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="127" data-semantic-parent="138" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="138"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="128" data-semantic-parent="138" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="138"><mjx-c class="mjx-c1D45B">𝑛</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="129" data-semantic-parent="138" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="138"><mjx-c class="mjx-c1D460">𝑠</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="130" data-semantic-parent="138" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="5" data-semantic-parent="138"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="131" data-semantic-parent="138" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="6" data-semantic-parent="138"><mjx-c class="mjx-c1D45B">𝑛</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="132" data-semantic-parent="138" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="7" data-semantic-parent="138"><mjx-c class="mjx-c1D43A">𝐺</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="133" data-semantic-parent="138" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="8" data-semantic-parent="138"><mjx-c class="mjx-c1D452">𝑒</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="134" data-semantic-parent="138" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="9" data-semantic-parent="138"><mjx-c class="mjx-c1D45B">𝑛</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="135" data-semantic-parent="138" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="10" data-semantic-parent="138"><mjx-c class="mjx-c1D45C">𝑜</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="136" data-semantic-parent="138" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="11" data-semantic-parent="138"><mjx-c class="mjx-c1D45A">𝑚</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="137" data-semantic-parent="138" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="12" data-semantic-parent="138"><mjx-c class="mjx-c1D452">𝑒</mjx-c></mjx-mi></mjx-mrow><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="13" data-semantic-parent="139" data-semantic-operator="infixop,×"><mjx-c class="mjx-cD7">×</mjx-c></mjx-mo><mjx-mrow space="3" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="126" data-semantic-children="124" data-semantic-content="14,125" data-semantic-parent="139" data-semantic-owns="14 124 125"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="14" data-semantic-parent="126" data-semantic-operator="fenced"><mjx-c class="mjx-c28 MM-S3">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="subtraction" data-semantic-id="124" data-semantic-children="66,123" data-semantic-content="67" data-semantic-parent="126" data-semantic-owns="66 67 123"><mjx-mfrac data-semantic-type="fraction" data-semantic-role="division" data-semantic-id="66" data-semantic-children="34,65" data-semantic-parent="124" data-semantic-owns="34 65"><mjx-frac><mjx-num><mjx-nstrut></mjx-nstrut><mjx-mrow size="s" data-semantic-type="punctuated" data-semantic-role="startpunct" data-semantic-id="34" data-semantic-children="15,33" data-semantic-content="15" data-semantic-parent="66" data-semantic-owns="15 33"><mjx-mo data-semantic-type="punctuation" data-semantic-role="unknown" data-semantic-id="15" data-semantic-parent="34" data-semantic-operator="punctuated"><mjx-c class="mjx-c23">#</mjx-c></mjx-mo><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="33" data-semantic-children="16,17,18,19,28" data-semantic-content="29,30,31,32" data-semantic-parent="34" data-semantic-owns="16 29 17 30 18 31 19 32 28"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="16" data-semantic-parent="33"><mjx-c class="mjx-c1D450">𝑐</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="29" data-semantic-parent="33" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="17" data-semantic-parent="33"><mjx-c class="mjx-c210E">ℎ</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="30" data-semantic-parent="33" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="18" data-semantic-parent="33"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="31" data-semantic-parent="33" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="19" data-semantic-parent="33"><mjx-c class="mjx-c1D45D">𝑝</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="32" data-semantic-parent="33" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-added="true" data-semantic-type="appl" data-semantic-role="prefix function" data-semantic-id="28" data-semantic-children="20,26" data-semantic-content="27,20" data-semantic-parent="33" data-semantic-owns="20 27 26"><mjx-mi data-semantic-type="function" data-semantic-role="prefix function" data-semantic-font="normal" data-semantic-id="20" data-semantic-parent="28" data-semantic-operator="appl"><mjx-c class="mjx-c52" noic="true" style="padding-top: 0.683em;">R</mjx-c><mjx-c class="mjx-c65" style="padding-top: 0.683em;">e</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="27" data-semantic-parent="28" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="26" data-semantic-children="21,22,23" data-semantic-content="24,25" data-semantic-parent="28" data-semantic-owns="21 24 22 25 23"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="21" data-semantic-parent="26"><mjx-c class="mjx-c1D44E">𝑎</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="24" data-semantic-parent="26" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="22" data-semantic-parent="26"><mjx-c class="mjx-c1D451">𝑑</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="25" data-semantic-parent="26" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="23" data-semantic-parent="26"><mjx-c class="mjx-c1D460">𝑠</mjx-c></mjx-mi></mjx-mrow></mjx-mrow></mjx-mrow></mjx-mrow></mjx-num><mjx-dbox><mjx-dtable><mjx-line></mjx-line><mjx-row><mjx-den><mjx-dstrut></mjx-dstrut><mjx-mrow size="s" data-semantic-type="punctuated" data-semantic-role="startpunct" data-semantic-id="65" data-semantic-children="35,64" data-semantic-content="35" data-semantic-parent="66" data-semantic-owns="35 64"><mjx-mo data-semantic-type="punctuation" data-semantic-role="unknown" data-semantic-id="35" data-semantic-parent="65" data-semantic-operator="punctuated"><mjx-c class="mjx-c23">#</mjx-c></mjx-mo><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="64" data-semantic-children="36,37,38,39,40,42,43,44,45,54" data-semantic-content="55,56,57,58,59,60,61,62,63" data-semantic-parent="65" data-semantic-owns="36 55 37 56 38 57 39 58 40 59 42 60 43 61 44 62 45 63 54"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="36" data-semantic-parent="64"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="55" data-semantic-parent="64" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="37" data-semantic-parent="64"><mjx-c class="mjx-c1D45C">𝑜</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="56" data-semantic-parent="64" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="38" data-semantic-parent="64"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="57" data-semantic-parent="64" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="39" data-semantic-parent="64"><mjx-c class="mjx-c1D44E">𝑎</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="58" data-semantic-parent="64" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="40" data-semantic-parent="64"><mjx-c class="mjx-c1D459">𝑙</mjx-c></mjx-mi><mjx-mspace data-semantic-type="operator" data-semantic-role="space" data-semantic-id="59" data-semantic-parent="64" data-semantic-operator="infixop,⁢" style="width: 0.25em;"></mjx-mspace><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="42" data-semantic-parent="64"><mjx-c class="mjx-c1D436">𝐶</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="60" data-semantic-parent="64" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="43" data-semantic-parent="64"><mjx-c class="mjx-c210E">ℎ</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="61" data-semantic-parent="64" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="44" data-semantic-parent="64"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="62" data-semantic-parent="64" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="45" data-semantic-parent="64"><mjx-c class="mjx-c1D45D">𝑝</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="63" data-semantic-parent="64" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-added="true" data-semantic-type="appl" data-semantic-role="prefix function" data-semantic-id="54" data-semantic-children="46,52" data-semantic-content="53,46" data-semantic-parent="64" data-semantic-owns="46 53 52"><mjx-mi data-semantic-type="function" data-semantic-role="prefix function" data-semantic-font="normal" data-semantic-id="46" data-semantic-parent="54" data-semantic-operator="appl"><mjx-c class="mjx-c52" noic="true" style="padding-top: 0.683em;">R</mjx-c><mjx-c class="mjx-c65" style="padding-top: 0.683em;">e</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="53" data-semantic-parent="54" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="52" data-semantic-children="47,48,49" data-semantic-content="50,51" data-semantic-parent="54" data-semantic-owns="47 50 48 51 49"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="47" data-semantic-parent="52"><mjx-c class="mjx-c1D44E">𝑎</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="50" data-semantic-parent="52" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="48" data-semantic-parent="52"><mjx-c class="mjx-c1D451">𝑑</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="51" data-semantic-parent="52" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="49" data-semantic-parent="52"><mjx-c class="mjx-c1D460">𝑠</mjx-c></mjx-mi></mjx-mrow></mjx-mrow></mjx-mrow></mjx-mrow></mjx-den></mjx-row></mjx-dtable></mjx-dbox></mjx-frac></mjx-mfrac><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="67" data-semantic-parent="124" data-semantic-operator="infixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mfrac space="3" data-semantic-type="fraction" data-semantic-role="division" data-semantic-id="123" data-semantic-children="89,122" data-semantic-parent="124" data-semantic-owns="89 122"><mjx-frac><mjx-num><mjx-nstrut></mjx-nstrut><mjx-mrow size="s" data-semantic-type="punctuated" data-semantic-role="startpunct" data-semantic-id="89" data-semantic-children="68,88" data-semantic-content="68" data-semantic-parent="123" data-semantic-owns="68 88"><mjx-mo data-semantic-type="punctuation" data-semantic-role="unknown" data-semantic-id="68" data-semantic-parent="89" data-semantic-operator="punctuated"><mjx-c class="mjx-c23">#</mjx-c></mjx-mo><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="88" data-semantic-children="69,70,71,72,73,82" data-semantic-content="83,84,85,86,87" data-semantic-parent="89" data-semantic-owns="69 83 70 84 71 85 72 86 73 87 82"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="69" data-semantic-parent="88"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="83" data-semantic-parent="88" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="70" data-semantic-parent="88"><mjx-c class="mjx-c1D45B">𝑛</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="84" data-semantic-parent="88" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="71" data-semantic-parent="88"><mjx-c class="mjx-c1D45D">𝑝</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="85" data-semantic-parent="88" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="72" data-semantic-parent="88"><mjx-c class="mjx-c1D462">𝑢</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="86" data-semantic-parent="88" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="73" data-semantic-parent="88"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="87" data-semantic-parent="88" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-added="true" data-semantic-type="appl" data-semantic-role="prefix function" data-semantic-id="82" data-semantic-children="74,80" data-semantic-content="81,74" data-semantic-parent="88" data-semantic-owns="74 81 80"><mjx-mi data-semantic-type="function" data-semantic-role="prefix function" data-semantic-font="normal" data-semantic-id="74" data-semantic-parent="82" data-semantic-operator="appl"><mjx-c class="mjx-c52" noic="true" style="padding-top: 0.683em;">R</mjx-c><mjx-c class="mjx-c65" style="padding-top: 0.683em;">e</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="81" data-semantic-parent="82" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="80" data-semantic-children="75,76,77" data-semantic-content="78,79" data-semantic-parent="82" data-semantic-owns="75 78 76 79 77"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="75" data-semantic-parent="80"><mjx-c class="mjx-c1D44E">𝑎</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="78" data-semantic-parent="80" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="76" data-semantic-parent="80"><mjx-c class="mjx-c1D451">𝑑</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="79" data-semantic-parent="80" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="77" data-semantic-parent="80"><mjx-c class="mjx-c1D460">𝑠</mjx-c></mjx-mi></mjx-mrow></mjx-mrow></mjx-mrow></mjx-mrow></mjx-num><mjx-dbox><mjx-dtable><mjx-line></mjx-line><mjx-row><mjx-den><mjx-dstrut></mjx-dstrut><mjx-mrow size="s" data-semantic-type="punctuated" data-semantic-role="startpunct" data-semantic-id="122" data-semantic-children="90,121" data-semantic-content="90" data-semantic-parent="123" data-semantic-owns="90 121"><mjx-mo data-semantic-type="punctuation" data-semantic-role="unknown" data-semantic-id="90" data-semantic-parent="122" data-semantic-operator="punctuated"><mjx-c class="mjx-c23">#</mjx-c></mjx-mo><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="121" data-semantic-children="91,92,93,94,95,97,98,99,100,101,110" data-semantic-content="111,112,113,114,115,116,117,118,119,120" data-semantic-parent="122" data-semantic-owns="91 111 92 112 93 113 94 114 95 115 97 116 98 117 99 118 100 119 101 120 110"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="91" data-semantic-parent="121"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="111" data-semantic-parent="121" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="92" data-semantic-parent="121"><mjx-c class="mjx-c1D45C">𝑜</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="112" data-semantic-parent="121" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="93" data-semantic-parent="121"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="113" data-semantic-parent="121" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="94" data-semantic-parent="121"><mjx-c class="mjx-c1D44E">𝑎</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="114" data-semantic-parent="121" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="95" data-semantic-parent="121"><mjx-c class="mjx-c1D459">𝑙</mjx-c></mjx-mi><mjx-mspace data-semantic-type="operator" data-semantic-role="space" data-semantic-id="115" data-semantic-parent="121" data-semantic-operator="infixop,⁢" style="width: 0.25em;"></mjx-mspace><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="97" data-semantic-parent="121"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="116" data-semantic-parent="121" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="98" data-semantic-parent="121"><mjx-c class="mjx-c1D45B">𝑛</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="117" data-semantic-parent="121" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="99" data-semantic-parent="121"><mjx-c class="mjx-c1D45D">𝑝</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="118" data-semantic-parent="121" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="100" data-semantic-parent="121"><mjx-c class="mjx-c1D462">𝑢</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="119" data-semantic-parent="121" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="101" data-semantic-parent="121"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="120" data-semantic-parent="121" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-added="true" data-semantic-type="appl" data-semantic-role="prefix function" data-semantic-id="110" data-semantic-children="102,108" data-semantic-content="109,102" data-semantic-parent="121" data-semantic-owns="102 109 108"><mjx-mi data-semantic-type="function" data-semantic-role="prefix function" data-semantic-font="normal" data-semantic-id="102" data-semantic-parent="110" data-semantic-operator="appl"><mjx-c class="mjx-c52" noic="true" style="padding-top: 0.683em;">R</mjx-c><mjx-c class="mjx-c65" style="padding-top: 0.683em;">e</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="109" data-semantic-parent="110" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="108" data-semantic-children="103,104,105" data-semantic-content="106,107" data-semantic-parent="110" data-semantic-owns="103 106 104 107 105"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="103" data-semantic-parent="108"><mjx-c class="mjx-c1D44E">𝑎</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="106" data-semantic-parent="108" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="104" data-semantic-parent="108"><mjx-c class="mjx-c1D451">𝑑</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="107" data-semantic-parent="108" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="105" data-semantic-parent="108"><mjx-c class="mjx-c1D460">𝑠</mjx-c></mjx-mi></mjx-mrow></mjx-mrow></mjx-mrow></mjx-mrow></mjx-den></mjx-row></mjx-dtable></mjx-dbox></mjx-frac></mjx-mfrac></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="125" data-semantic-parent="126" data-semantic-operator="fenced"><mjx-c class="mjx-c29 MM-S3">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-mrow></mjx-math></mjx-container></div></div></div></div><div id="p0680" role="paragraph"><i>Analysis of differential ChIP-seq signal.</i> The counts of reads mapping to genes for each of the replicates for each of the ChIPs (GLYR1 and H3K36me3) at the hiPSC and CPC stages under wild-type conditions and at the CPC stage under P496L mutant conditions were obtained using featureCounts (<span class="dropBlock reference-citations"><a id="body-ref-sref83-3" href="#" role="doc-biblioref" data-xml-rid="bib83" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref83-3" aria-controls="sref83-3">Liao et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref83-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">71.</div><div>Liao, Y. ∙ Smyth, G.K. ∙ Shi, W.</div><div><strong>featureCounts: an efficient general purpose program for assigning sequence reads to genomic features</strong></div><div><em>Bioinformatics.</em> 2014; <strong>30</strong>:923-930</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btt656" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-84897397058" target="_blank" aria-label="Open Scopus in new tab">Scopus (7989)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24227677/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtt656&amp;pmid=24227677" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) using their corresponding aligned reads in bam files. The counts of reads for each of replicates used for assaying gene expression at the two stages in the GSE137920 (<span class="dropBlock reference-citations"><a id="body-ref-sref76-1" href="#" role="doc-biblioref" data-xml-rid="bib76" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref76-1" aria-controls="sref76-1">Lau et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref76-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">68.</div><div>Lau, E. ∙ Han, Y. ∙ Williams, D.R. ...</div><div><strong>Splice-Junction-Based Mapping of Alternative Isoforms in the Human Proteome</strong></div><div><em>Cell Rep.</em> 2019; <strong>29</strong>:3751-3765.e5</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=10.1016%2Fj.celrep.2019.11.026&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=10.1016%2Fj.celrep.2019.11.026&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-85076033361" target="_blank" aria-label="Open Scopus in new tab">Scopus (26)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31825849/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.celrep.2019.11.026&amp;pmid=31825849" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) dataset were downloaded from GEO (<span class="dropBlock reference-citations"><a id="body-ref-sref6" href="#" role="doc-biblioref" data-xml-rid="bib6" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref6" aria-controls="sref6">Barrett et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">6.</div><div>Barrett, T. ∙ Wilhite, S.E. ∙ Ledoux, P. ...</div><div><strong>NCBI GEO: archive for functional genomics data sets--update</strong></div><div><em>Nucleic Acids Res.</em> 2013; <strong>41</strong>:D991-D995</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gks1193" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-84874271270" target="_blank" aria-label="Open Scopus in new tab">Scopus (4723)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23193258/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgks1193&amp;pmid=23193258" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Genes where there were not at least two samples with at least 5 (raw) reads in the GLYR1 ChIPs were filtered out from further analyses. The read counts for the remaining genes corresponding to each of the three signals (GLYR1, H3K36m3 and Gene expression) are separately normalized using <i>calcNormFactors</i> in edgeR (<span class="dropBlock reference-citations"><a id="body-ref-sref115" href="#" role="doc-biblioref" data-xml-rid="bib115" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref115" aria-controls="sref115">Robinson and Oshlack 2010</a><div class="dropBlock__holder" data-db-target-of="sref115"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">103.</div><div>Robinson, M.D. ∙ Oshlack, A.</div><div><strong>A scaling normalization method for differential expression analysis of RNA-seq data</strong></div><div><em>Genome Biol.</em> 2010; <strong>11</strong>:R25</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/gb-2010-11-3-r25" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_103_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-77953176036" target="_blank" aria-label="Open Scopus in new tab">Scopus (3916)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20196867/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fgb-2010-11-3-r25&amp;pmid=20196867" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Genes for which the mean GLYR1 signal in their bodies were significantly changed from CPC stage relative to hiPSC stage (or in the P496L mutant versus wild-type conditions) were determined the likelihood ratio test implemented in edgeR using FDR&nbsp;&lt; 0.1 threshold. The row-normalized log2 transformed Counts-Per-Million (CPM) of GLYR1 signal for these significantly associated (with changing GLYR1 signal) genes were clustered using <i>kmeans</i> with 3 clusters implemented in R (<span class="dropBlock reference-citations"><a id="body-ref-sref108-2" href="#" role="doc-biblioref" data-xml-rid="bib108" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref108-2" aria-controls="sref108-2">R Core Team 2020</a><div class="dropBlock__holder" data-db-target-of="sref108-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">96.</div><div>R Core Team</div><div><strong>R: A language and environment for statistical computing</strong></div><div>R Foundation for Statistical Computing, Vienna, Austria, 2020</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=R+Core+TeamR%3A+A+language+and+environment+for+statistical+computing2020R+Foundation+for+Statistical+ComputingVienna%2C+Austria" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The resulting cluster definitions (using the GLYR1 signal) and order of genes were used to visualize the signals (in row-normalized log2 CPM units) in the H3K36me3 and RNA-seq data.</div><div id="p0685" role="paragraph"><i>Definition of GATA4 and GLYR1-bound gene categories (list used in most of the panel</i> <a id="crosref2175" href="#fig6"><i>Figure&nbsp;6</i></a><i>).</i> GLYR1 bound genes in <a id="crosref2180" href="#fig6">Figure&nbsp;6</a>B are defined as those genes in clusters 2 and 3 in <a id="crosref2185" href="#fig6">Figure&nbsp;6</a>A which displayed enriched binding signal at the CPC stage relative to the hiPSC stage. Genes with Gata4 ChIP peaks from the first intron to Transcription End Site (TES) were defined as Gata4 bound genes.</div><div id="p0690" role="paragraph"><i>Scatterplot analysis (ChIP-seq/RNA-seq) (</i><a id="crosref2190" href="#figs5">Figure&nbsp;S5</a>A<i>).</i> Differential gene expression between the hiPSC (day 0) and the CPC stage (day 7) was determined using quasi-likelihood F-test implemented in the <i>glmQLFTest</i> in edgeR (<span class="dropBlock reference-citations"><a id="body-ref-sref85" href="#" role="doc-biblioref" data-xml-rid="bib85" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref85" aria-controls="sref85">Lun et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref85"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">73.</div><div>Lun, A.T.L. ∙ Chen, Y. ∙ Smyth, G.K.</div><div><strong>It’s DE-licious: A Recipe for Differential Expression Analyses of RNA-seq Experiments Using Quasi-Likelihood Methods in edgeR</strong></div><div><em>Methods Mol. Biol.</em> 2016; <strong>1418</strong>:391-416</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/978-1-4939-3578-9_19" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-84961614091" target="_blank" aria-label="Open Scopus in new tab">Scopus (213)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27008025/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2F978-1-4939-3578-9_19&amp;pmid=27008025" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) using the count matrix association with the GSE137920 (<span class="dropBlock reference-citations"><a id="body-ref-sref76-2" href="#" role="doc-biblioref" data-xml-rid="bib76" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref76-2" aria-controls="sref76-2">Lau et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref76-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">68.</div><div>Lau, E. ∙ Han, Y. ∙ Williams, D.R. ...</div><div><strong>Splice-Junction-Based Mapping of Alternative Isoforms in the Human Proteome</strong></div><div><em>Cell Rep.</em> 2019; <strong>29</strong>:3751-3765.e5</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=10.1016%2Fj.celrep.2019.11.026&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=10.1016%2Fj.celrep.2019.11.026&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-85076033361" target="_blank" aria-label="Open Scopus in new tab">Scopus (26)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31825849/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.celrep.2019.11.026&amp;pmid=31825849" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) filtered for low counts (at least two samples with at least 5 (raw) reads) genes normalized using the <i>calcNormFactors</i>. Upregulated genes were determined using thresholds of 1.5 for log2 fold-change (log2FC &gt; 1.5) and 0.05 for FDR (FDR&nbsp;&lt; 0.05), while downregulated genes were determined using thresholds of −1.5 for log2 fold-change (log2FC&nbsp;&lt; −1.5) and 0.05 for FDR. The raw read counts for all replicates of GLYR1 at the two stages for all genes that were part of the differential gene expression analyses above, were normalized using <i>calcNormFactors</i> and Reads Per Kilobase of transcript, per Million mapped reads (RPKM) was calculated using the <i>rpkm</i> function along with the gene lengths based on the Ensembl gene annotation. Similarly, RPKM values were estimated for all replicates of the H3K36me3 ChIP at the two stages. The scatterplots in <a id="crosref2205" href="#figs5">Figure&nbsp;S5</a> use the mean log2-transformed RPKM values across replicates at a given stage.</div><div id="p0695" role="paragraph"><i>Statistical analysis of GATA4 and GLYR1 ChIP-seq overlap (</i><a id="crosref2210" href="#fig6"><i>Figure&nbsp;6</i></a><i>B).</i> The significance of the overlap of genes bound within their bodies (first intron to Transcription End Site (TES)) by Gata4 and GLYR1 was determined using the Fisher’s exact test implemented in R (<span class="dropBlock reference-citations"><a id="body-ref-sref108-3" href="#" role="doc-biblioref" data-xml-rid="bib108" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref108-3" aria-controls="sref108-3">R Core Team 2020</a><div class="dropBlock__holder" data-db-target-of="sref108-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">96.</div><div>R Core Team</div><div><strong>R: A language and environment for statistical computing</strong></div><div>R Foundation for Statistical Computing, Vienna, Austria, 2020</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=R+Core+TeamR%3A+A+language+and+environment+for+statistical+computing2020R+Foundation+for+Statistical+ComputingVienna%2C+Austria" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) on 30,611 genes which had detectable reads (5 reads in at least two samples) at the day 7 in the GSE137920 (<span class="dropBlock reference-citations"><a id="body-ref-sref76-3" href="#" role="doc-biblioref" data-xml-rid="bib76" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref76-3" aria-controls="sref76-3">Lau et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref76-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">68.</div><div>Lau, E. ∙ Han, Y. ∙ Williams, D.R. ...</div><div><strong>Splice-Junction-Based Mapping of Alternative Isoforms in the Human Proteome</strong></div><div><em>Cell Rep.</em> 2019; <strong>29</strong>:3751-3765.e5</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=10.1016%2Fj.celrep.2019.11.026&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=10.1016%2Fj.celrep.2019.11.026&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-85076033361" target="_blank" aria-label="Open Scopus in new tab">Scopus (26)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31825849/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.celrep.2019.11.026&amp;pmid=31825849" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) dataset.</div><div id="p0700" role="paragraph"><i>Motif analyses.</i> Motif enrichment in the ChIP peak regions were performed using the <i>findMotifsGenome</i> function implemented in homer (version v4.11.1) (<span class="dropBlock reference-citations"><a id="body-ref-sref53-2" href="#" role="doc-biblioref" data-xml-rid="bib53" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref53-2" aria-controls="sref53-2">Heinz et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref53-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">48.</div><div>Heinz, S. ∙ Benner, C. ∙ Spann, N. ...</div><div><strong>Simple combinations of lineage-determining transcription factors prime cis-regulatory elements required for macrophage and B cell identities</strong></div><div><em>Mol. Cell.</em> 2010; <strong>38</strong>:576-589</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_48_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=10.1016%2Fj.molcel.2010.05.004&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_48_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=10.1016%2Fj.molcel.2010.05.004&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_48_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-77952567987" target="_blank" aria-label="Open Scopus in new tab">Scopus (6049)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20513432/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.molcel.2010.05.004&amp;pmid=20513432" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div></section><section id="sec-4-4-16-15"><h5>Single-cell RNA-seq analysis</h5><div id="p0705" role="paragraph"><i>Preprocessing.</i> 6-8 replicate samples (from independent differentiation runs involving either wild-type or P496L mutant cells) were each processed in 3 batches. Reads from each sample were aligned to the hg38 human reference (version 2020-A from 10X Genomics website) using Cell Ranger version 5.0.1. The resulting count matrices for all samples within each batch were aggregated without depth normalization. The aggregated matrix was analyzed with the Seurat package (version 4.0.1) in R (<span class="dropBlock reference-citations"><a id="body-ref-sref52-2" href="#" role="doc-biblioref" data-xml-rid="bib52" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref52-2" aria-controls="sref52-2">Hao et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref52-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">47.</div><div>Hao, Y. ∙ Hao, S. ∙ Andersen-Nissen, E. ...</div><div><strong>Integrated analysis of multimodal single-cell data</strong></div><div><em>Cell.</em> 2021; <strong>184</strong>:3573-3587.e29</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=10.1016%2Fj.cell.2021.04.048&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=10.1016%2Fj.cell.2021.04.048&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-85108179646" target="_blank" aria-label="Open Scopus in new tab">Scopus (800)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/34062119/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2021.04.048&amp;pmid=34062119" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The Seurat object was created keeping cells with the number of detected features between 30<sup>th</sup> and 95<sup>th</sup> quantiles of detected features across all cells in the batch and whose percent mitochondrial reads were below 30%. The remaining counts were normalized with the <i>SCTranform</i> function after setting the number of variable features to 3000, using the <i>glmGamPoi</i> method to estimate parameters of the fitted negative binomial distributions and regressing out the effects of the percent mitochondrial reads per cell. The <i>SCTranform</i> based normalized counts per sample were subject to correction for systematic differences between the three batches using the <i>RunHarmony</i> function as implemented in the harmony (<span class="dropBlock reference-citations"><a id="body-ref-sref72-2" href="#" role="doc-biblioref" data-xml-rid="bib72" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref72-2" aria-controls="sref72-2">Korsunsky et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref72-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">64.</div><div>Korsunsky, I. ∙ Millard, N. ∙ Fan, J. ...</div><div><strong>Fast, sensitive and accurate integration of single-cell data with Harmony</strong></div><div><em>Nat. Methods.</em> 2019; <strong>16</strong>:1289-1296</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41592-019-0619-0" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_64_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-85070698712" target="_blank" aria-label="Open Scopus in new tab">Scopus (806)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31740819/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41592-019-0619-0&amp;pmid=31740819" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) R package. Clustering was done using a shared nearest neighbor graph built with the top 30 dimensions of the harmony-based correction and the original Louvain algorithm for modularity optimization with resolution parameter set to 0.4. UMAP embedding was then generated using the top 30 dimensions of the harmony-based correction.</div><div id="p0710" role="paragraph"><i>Association with cell state.</i> The differences in the proportion of P496L mutant cells versus WT cells in each of the identified cell clusters at each of the sampled time-points in the differentiation protocol was quantified using an odds ratio estimate. The log odds ratios were estimated in the context of a generalized linear mixed effects model assuming the binomial family of probability distribution for the numbers of cells, as implemented in the <i>glmer</i> function in the lme4 (<span class="dropBlock reference-citations"><a id="body-ref-sref10-2" href="#" role="doc-biblioref" data-xml-rid="bib10" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref10-2" aria-controls="sref10-2">Bates et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref10-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">10.</div><div>Bates, D. ∙ Mächler, M. ∙ Bolker, B. ...</div><div><strong>Fitting linear mixed-effects models using lme4</strong></div><div><em>J.&nbsp;Stat. Softw.</em> 2015; <strong>67</strong>:1-48</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.18637/jss.v067.i01" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-84943645306" target="_blank" aria-label="Open Scopus in new tab">Scopus (36021)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.18637%2Fjss.v067.i01" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) package in R. The sample or differentiation run of origin of each cell was modeled as a random effect in these models.</div><div id="p0715" role="paragraph"><i>Differential expression.</i> The association of gene expression with differences between wild-type and mutant cells were estimated using the <i>FindMarkers</i> function in Seurat (<span class="dropBlock reference-citations"><a id="body-ref-sref52-3" href="#" role="doc-biblioref" data-xml-rid="bib52" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref52-3" aria-controls="sref52-3">Hao et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref52-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">47.</div><div>Hao, Y. ∙ Hao, S. ∙ Andersen-Nissen, E. ...</div><div><strong>Integrated analysis of multimodal single-cell data</strong></div><div><em>Cell.</em> 2021; <strong>184</strong>:3573-3587.e29</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=10.1016%2Fj.cell.2021.04.048&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=10.1016%2Fj.cell.2021.04.048&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-85108179646" target="_blank" aria-label="Open Scopus in new tab">Scopus (800)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/34062119/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2021.04.048&amp;pmid=34062119" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) that implemented the MAST (<span class="dropBlock reference-citations"><a id="body-ref-sref40-2" href="#" role="doc-biblioref" data-xml-rid="bib40" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref40-2" aria-controls="sref40-2">Finak et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref40-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">36.</div><div>Finak, G. ∙ McDavid, A. ∙ Yajima, M. ...</div><div><strong>MAST: a flexible statistical framework for assessing transcriptional changes and characterizing heterogeneity in single-cell RNA sequencing data</strong></div><div><em>Genome Biol.</em> 2015; <strong>16</strong>:278</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13059-015-0844-5" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-84951574149" target="_blank" aria-label="Open Scopus in new tab">Scopus (870)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26653891/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13059-015-0844-5&amp;pmid=26653891" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) method where the batch of the origin of each cell was modeled as a latent variable.</div><div id="p0720" role="paragraph"><i>Association with ChIP-seq data.</i> The differences between the average (across replicates from independent differentiation runs) ChIP signal (quantified in Transcripts-Per-Million (TPM) units) within gene bodies among the wild-type cells and the P496L cells were visualized as scatterplots separately for genes that were upregulated (logFC &gt; 0.125 and FDR&nbsp;&lt; 0.05 in the output of the <i>FindMarkers</i> function) and downregulated (logFC&nbsp;&lt; −0.125 and FDR&nbsp;&lt; 0.05).</div></section></section></section><section id="sec-4-5"><h3>Quantification and statistical analysis</h3><div id="p0725" role="paragraph">Statistical parameters including the exact value of n, precision measures (mean ± SEM) and statistical significance are reported in the Figures and the Figure&nbsp;Legends. All calculations were performed using R or GraphPad Prism software. When several conditions were to compare, we performed a one-way ANOVA, followed by Tukey range test to assess the significance among pairs of conditions. The significance of the PPIN enrichment in CHD-associated DNVs was calculated with a permutation-based test as explained in the Computational Analysis Methods section. All the p values related to the violin plots showing features typical of disease genes were obtained using a two-sided Mann-Whitney-Wilcoxon test with Bonferroni correction. The significance of the GATA4 &amp; GLYR1 ChIPseq overlap was estimated using the Fisher.Exact function in R. The level of significance in all graphs is represented as follow: <sup>∗</sup> p&nbsp;&lt; 0.05, <sup>∗∗</sup> p&nbsp;&lt; 0.01, <sup>∗∗∗</sup> p&nbsp;&lt; 0.001, <sup>∗∗∗∗</sup> p&nbsp;&lt; 0.0001.</div></section></section><section id="acknowledgments" role="doc-acknowledgments"><h2>Acknowledgments</h2><div id="p0730" role="paragraph">We thank the Srivastava laboratory and Gladstone colleagues for critical discussions and feedback; Guadalupe Sabio for critical reading of the manuscript; B. Taylor for editorial assistance; Jim Kadonaga for kindly sharing GLYR1 anti-serum; Irfan Kathiriya and Kai Jiao for kindly sharing the <i>Nppa</i> luciferase reporter plasmid and the CHD7 ATP mutant plasmid, respectively; the Gladstone Genomics Core, Bioinformatic Core, Stem Cell Core, Microscopy &amp; Histology Core, Mouse Transgenics Core, and Flow Cytometry Core for their technical expertise and the Gladstone Animal Facility for support with mouse colony maintenance; David E. Gordon for sharing an optimized CRISPR-Cas9 RNPs hiPSCs knockout generation protocol. We thank Francoise Chanut for manuscript editorial support and Ana Catarina Silva (ana@anasilvaillustrations.com) for helping with figure editing and design. B.G.-T. is supported by the American Heart Association (18POST34080175) and AHA/CHF Congenital Heart Defect Research Award (818798). M.A. is supported by the Swiss National Science Foundation (P400PM_186704). K.S.P. is supported by NIH P01 HL098707, P01 HL146366, and UM1 HL098179; Gladstone Institutes; and the San Simeon Fund. D.S. is supported by NIH/NHLBI P01 HL098707, P01 HL146366, R01 HL057181, and R01 HL127240, and by the Roddenberry Foundation, the L. K. Whittier Foundation, Dario and Irina Sattui, and the Younger Family Fund. N.J.K. is supported by grants from the National Institutes of Health (P01 HL146366 and 1U01MH115747). B.G.B. was supported by NIH/NHLBI P01 HL098707 and P01 HL146366, and by the Younger Family Fund. A.P. is supported by the NIH (K08HL157700), Tobacco-Related Disease Research Program (578649), A.P. Giannini Foundation (P0527061), Michael Antonov Charitable Foundation, and Sarnoff Cardiovascular Research Foundation. S.U.M. was supported by the American Heart Association (20POST35210452) and the Boston Children’s Hospital Office of Faculty Development. C.E.S. and J.G.S. are supported by NIH U01 HL098147 and UM1 HL098166. This work was also supported by NIH/NCRR grant C06 RR018928 to the Gladstone Institutes.</div><section id="sec-5"><h3>Author contributions</h3><div id="p0735" role="paragraph">B.G.-T., K.S.P., and D.S. conceived and directed the study with input from B.G.B., B.R.C., B.L.B., and N.J.K. as part of an NIH/NHLBI-sponsored Program Project Grant and B.D.G., S.U.M., C.E.S., and J.G.S. as part of the NIH/NHLBI-sponsored Bench to Bassinet Pediatric Cardiac Genomics Consortium. D.R.-B. and B.G.-T. generated the <i>GATA4-KO</i> hiPSC line, and B.E.J.M. and B.G.-T. generated the <i>TBX5-KO</i> hiPSC line. D.R.-B. and B.G.-T. performed WT, GATA4, and TBX5 knockout CP and CM differentiations and characterization. B.R.C., R.H., and B.G.-T. defined the appropriate affinity purification strategy for cTFs. B.G.-T. performed GATA4 and TBX5 (GT) affinity purification and sample preparation for mass spectrometry and GT-PPI classification. M.M. prepared the AP-MS buffers and performed the desalting and lyophilization of the AP-MS samples. M.P. and K.S.P. performed the AP-MS statistical analysis, and M.P., K.S.P., and B.G.-T. the PPI filtering. S.U.M. and M.P. performed variant processing; M.P. and K.S.P. designed and applied GT-PPI variant enrichment analysis. M.P. and B.G.-T. performed interactome gene features analysis and developed the integrative pathogenicity scoring method. M.P., R.T., B.G.-T., and K.S.P. analyzed the benefit of variant filtering with the GT-PPI. C.A.G. analyzed the expression of GT-PPI genes with CHD variants mouse heart development cell types. F.F. performed the GLYR1 protein sequence alignment, GLYR1 protein structure modeling, and recombinant DNA cloning. E.M. and G.C. performed the molecular dynamics GLYR1 simulations. F.F. and B.G.-T. performed GATA4 and GLYR1 silencing in CPs and luciferase reporter assays. K.S.-K. generated the cTFs ChIP-seq data. B.G.-T. and M.A. performed GLYR1 and H3K36me3 ChIP-seq. B.G.-T. and F.F. generated the GLYR1<sup>P496L</sup> hiPSC line, GLYR1<sup>P496L</sup> cardiomyocyte differentiations, scRNA-seq, GLYR1 ChIP-seq, and beating analysis. R.T. and K.C. performed RNA-seq and ChIP-seq computational analysis. R.T. and B.G.-T. performed the scRNA-seq analysis. A.P. designed and generated the Glyr1<sup>P495L</sup> mouse line. C.Y.L. assisted with mouse colony maintenance and experimentation. Y.H. performed echocardiography. M.W.C. and B.G.-T performed Glyr1<sup>P495L</sup> mouse studies.</div></section><section id="sec-6"><h3>Declaration of interests</h3><div id="p0740" role="paragraph">D.S. is scientific co-founder, shareholder, and director of Tenaya Therapeutics. B.G.B. and B.R.C. are scientific co-founders and shareholders of Tenaya Therapeutics. K.S.P. is a shareholder of Tenaya Therapeutics. N.J.K. has received research support from Vir Biotechnology and F. Hoffmann-La Roche. N.J.K. has consulting agreements with the Icahn School of Medicine at Mount Sinai, New York, Maze Therapeutics, and Interline Therapeutics; is a shareholder of Tenaya Therapeutics, Maze Therapeutics, and Interline Therapeutics; and is financially compensated by GEn1E Lifesciences, Inc. and Twist Bioscience Corp.</div></section><section id="sec-7"><h3>Inclusion and diversity</h3><div id="p0745" role="paragraph">One or more of the authors of this paper self-identifies as a member of the LGBTQ+ community. One or more of the authors of this paper self-identifies as an underrepresented ethnic minority in science. While citing references scientifically relevant for this work, we also actively worked to promote gender balance in our reference list.</div></section></section><section id="supplementary-material" data-type="supplementary-materials"><h2>Supplemental information (1)</h2><div id="p0755" role="paragraph"><div id="mmc1" class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="/cms/10.1016/j.cell.2022.01.021/attachment/a2e3c901-53ea-49f3-b6e3-5f6389b4e3f4/mmc1.xlsx" download="mmc1.xlsx" aria-labelledby="mmc1-heading mmc1-link"><span id="mmc1-link">Spreadsheet (27.86 MB)</span></a></div><div class="core-description"><div id="mmc1-heading" class="heading">Table S1. Compendium of all supplemental tables, related to Figures 1, 2, 3, 4, 5, 6, 7, S1, S2, S3, S4, S5, S6, and S7</div><div id="espara0010" role="paragraph">Table S1A: DE analysis between WT and <i>GATA4-KO</i> CPs (day 6 differentiation). Related to Figures 1, S1, and S2. Table S1B: DE analysis between WT and <i>TBX5-KO</i> CPs (day 6 differentiation). Related to Figures 1, S1, and S2. Table S1C: AP-MS data and filtering criteria for the GATA4 interactome in CPs. Related to Figures 1, S1, and S2. Table S1D: AP-MS data and filtering criteria for the TBX5 interactome in CPs. Related to Figures 1, S1, and S2 Table S1E: <i>De novo</i> and inherited loss-of-function variant counts in cases and controls, with odds ratio and p value. Related to Figure&nbsp;2. Table S1F: Proband variants in genes involved in mouse/human heart development (Jin et&nbsp;al., 2017) removed from the Permutation analysis in Figure&nbsp;2B. Related to Figure&nbsp;2. Table S1G: AP-MS data and filtering criteria for the GATA4 interactome in HEK293 cells. Related to Figures 2 and S3. Table S1H: AP-MS data and filtering criteria for the TBX5 interactome in HEK293 cells. Related to Figures 2 and S3. Table S1I: Null distribution of the number of variants found in GT-PPI genes compared to a comparable non-GT-PPI gene-set expressed in CPs (see STAR Methods). Related to Figures 2 and S3. Table S1J: <i>De novo</i> missense, loss-of-function and splice PCGC variants found in interactome genes. Related to Figure&nbsp;3 and Figure&nbsp;S3. Table S1K: Counts for the different types of <i>de novo</i> variants (DNVs) found in CHD and control cohorts in GT-PPIs and CHD variant network enrichment analysis. Related to Figure&nbsp;3 and Figure&nbsp;S3. Table S1L: DNV scoring of missense PCGC variants found in interactome genes. Related to Figure&nbsp;4. Table S1M: DNV scoring of all missense variants in interactome genes and non-interactome genes found in CHD Cases. Related to Figures 4 and S4. Table S1N: Rare inherited variants found in <i>GLYR1</i> P496L patient. Related to Figure&nbsp;5. Table S1O: GLYR1 differentially bound genes between hiPSCs and CPs; Heatmap 6A. Related to Figure&nbsp;6. Table S1P: GO term enrichment analysis for gene clusters FDR&lt;0.1 defined in Table S1O and Figure&nbsp;6A. Related to Figure&nbsp;6. Table S1Q: Lists for GLYR1-Only, GATA4&amp;GLYR1 and GATA4-Only bound genes from Figure&nbsp;6B. Related to Figure&nbsp;6. Table S1R: Top three Biological Process GO term terms significantly enriched for gene clusters FDR&lt;0.1 defined in Table S1O and Figure&nbsp;6A. Related to Figure&nbsp;6. Table S1S: DE genes in CPs upon siControl versus si<i>GATA4</i>; Figure&nbsp;6D. Related to Figure&nbsp;6. Table S1T: DE genes in CPs upon siControl versus si<i>GLYR1</i>; Figures 6D and S5F. Related to Figure&nbsp;6 and Figure&nbsp;S5. Table S1U: GO term enrichment analysis for DE genes in si<i>GLYR1</i> versus siControl CPs, FDR&lt;0.05 and LogFC&nbsp;&lt; −0.25 or &gt; 0.25; Figure&nbsp;S5F. Related to Figures 6 and S5. Table S1V: Homer Known Motif Enrichment Results for GATA4-occupied genomic regions within GATA4:GLYR1 co-bound genes. Related to Figure&nbsp;6 and Figure&nbsp;S5. Table S1W: Cluster identity marker genes from single cell RNaseq at CM differentiation day 6 from GLYR1-WT and GLYRYR1-P496L lines. Related to Figures 7 and S6. Table S1X: Differential expression analysis <i>GLYR1-P496L</i> versus <i>GLYR1-WT</i> cardiac progenitor-like cells (cluster 0) at CM differentiation day 6. Related to Figure&nbsp;7. Table S1Y: Gene Ontology enrichment analysis for DE-genes <i>GLYR1-P496L</i> versus <i>GLYR1-WT</i> (FDR&lt;0.05; Log2FC&gt;0.125) CP-like cells (cluster 0) at CM differentiation day 6. Related to Figure&nbsp;7. Table S1Z: Cluster identity marker genes from single cell RNaseq at CM differentiation day 18 from <i>GLYR1-WT</i> and <i>GLYR1-P496L</i> lines. Related to Figures 7 and S7. Table S1AA: Cluster identity contribution odds per genotype at CM differentiation day 18. Related to Figure&nbsp;7. Table S1AB: Differential expression (DE) analysis <i>GLYR1-P496L</i> versus <i>GLYR1-WT</i> cardiomyocyte-like cells (cluster 0 and 6) at CM differentiation day 18. Related to Figure&nbsp;7. Table S1AC: Gene Ontology enrichment analysis for DE-genes <i>GLYR1-P496L</i> versus <i>GLYR1-WT</i> (FDR&lt;0.05; Log2FC&gt;0.125) CM-like cells (cluster 0 and 6) at CM differentiation day 18. Related to Figure&nbsp;7.</div></div></div></div></section></div></section><div class="core-linked-content">
    

        

</div><section id="backmatter" data-extent="backmatter"><div class="core-container"><section id="data-availability" class="core-data-availability"><h2>Data and code availability</h2><div id="p0030" role="paragraph">The RNa-seq, scRNa-seq and ChIP-seq datasets generated during this study are available at GEO:GSE159411[<a id="intref0010" href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE159411" target="_blank">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE159411</a>]. The mass spectrometry proteomics data have been deposited to the ProteomeXchange Consortium via the PRIDE (<span class="dropBlock reference-citations"><a id="body-ref-sref105" href="#" role="doc-biblioref" data-xml-rid="bib105" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref105" aria-controls="sref105">Perez-Riverol et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref105"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">93.</div><div>Perez-Riverol, Y. ∙ Csordas, A. ∙ Bai, J. ...</div><div><strong>The PRIDE database and related tools and resources in 2019: improving support for quantification data</strong></div><div><em>Nucleic Acids Res.</em> 2019; <strong>47</strong>:D442-D450</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gky1106" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_93_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-85058095584" target="_blank" aria-label="Open Scopus in new tab">Scopus (4040)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30395289/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgky1106&amp;pmid=30395289" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) partner repository with the dataset identifier PRIDE: PXD022091. Code is available at <a id="intref0015" href="https://github.com/mepittman/ctf-apms" target="_blank">https://github.com/mepittman/ctf-apms</a>. Any additional information required to reanalyze the data reported in this paper is available from the lead contact upon request.</div></section><section id="references"><h2>References</h2><div id="bibliography" role="doc-bibliography"><div role="list"><div role="listitem" data-has="label"><div id="bib1" class="citations"><div id="sref1" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref1-1" title="View in article">1.</a></div><div>Akerberg, B.N. ∙ Gu, F. ∙ VanDusen, N.J. ...</div><div><strong>A reference map of murine cardiac transcription factor chromatin occupancy identifies dynamic and conserved enhancers</strong></div><div><em>Nat. Commun.</em> 2019; <strong>10</strong>:4907</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41467-019-12812-3" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_1_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-85074196402" target="_blank" aria-label="Open Scopus in new tab">Scopus (39)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31659164/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41467-019-12812-3&amp;pmid=31659164" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib2" class="citations"><div id="sref2" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref2" title="View in article">2.</a></div><div>Alexanian, M. ∙ Przytycki, P.F. ∙ Micheletti, R. ...</div><div><strong>A transcriptional switch governs fibroblast activation in heart disease</strong></div><div><em>Nature.</em> 2021; <strong>595</strong>:438-443</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-021-03674-1" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_2_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-85108376829" target="_blank" aria-label="Open Scopus in new tab">Scopus (35)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/34163071/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-021-03674-1&amp;pmid=34163071" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib3" class="citations"><div id="sref3" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref3" title="View in article">3.</a></div><div>Andrews, S.</div><div><strong>A quality control tool for high throughput sequence data</strong></div><div><em>babraham bioinformatics.</em> 2007;</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=S.AndrewsA+quality+control+tool+for+high+throughput+sequence+datababraham+bioinformatics2007" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib4" class="citations"><div id="sref4" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref4-1" title="View in article">4.</a></div><div>Ang, Y.-S. ∙ Rivas, R.N. ∙ Ribeiro, A.J.S. ...</div><div><strong>Disease model of GATA4 mutation reveals transcription factor cooperativity in human cardiogenesis</strong></div><div><em>Cell.</em> 2016; <strong>167</strong>:1734-1749.e22</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=10.1016%2Fj.cell.2016.11.033&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=10.1016%2Fj.cell.2016.11.033&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-85006302151" target="_blank" aria-label="Open Scopus in new tab">Scopus (127)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27984724/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2016.11.033&amp;pmid=27984724" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib5" class="citations"><div id="sref5" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref5" title="View in article">5.</a></div><div>Aronesty, E.</div><div><strong>Comparison of Sequencing Utility Programs</strong></div><div><em>The Open Bioinformatics Journal.</em> 2013; <strong>7</strong>:1-8</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.2174/1875036201307010001" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-84875098273" target="_blank" aria-label="Open Scopus in new tab">Scopus (722)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.2174%2F1875036201307010001" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib6" class="citations"><div id="sref6" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref6" title="View in article">6.</a></div><div>Barrett, T. ∙ Wilhite, S.E. ∙ Ledoux, P. ...</div><div><strong>NCBI GEO: archive for functional genomics data sets--update</strong></div><div><em>Nucleic Acids Res.</em> 2013; <strong>41</strong>:D991-D995</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gks1193" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-84874271270" target="_blank" aria-label="Open Scopus in new tab">Scopus (4723)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23193258/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgks1193&amp;pmid=23193258" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib7" class="citations"><div id="sref7" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref7-1" title="View in article">7.</a></div><div>Barshir, R. ∙ Shwartz, O. ∙ Smoly, I.Y. ...</div><div><strong>Comparative analysis of human tissue interactomes reveals factors leading to tissue-specific manifestation of hereditary diseases</strong></div><div><em>PLoS Comput. Biol.</em> 2014; <strong>10</strong>:e1003632</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pcbi.1003632" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-84903384845" target="_blank" aria-label="Open Scopus in new tab">Scopus (49)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24921629/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pcbi.1003632&amp;pmid=24921629" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib8" class="citations"><div id="sref8" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref8" title="View in article">8.</a></div><div>Basson, C.T. ∙ Bachinsky, D.R. ∙ Lin, R.C. ...</div><div><strong>Mutations in human TBX5 [corrected] cause limb and cardiac malformation in Holt-Oram syndrome</strong></div><div><em>Nat. Genet.</em> 1997; <strong>15</strong>:30-35</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng0197-30" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-0030636780" target="_blank" aria-label="Open Scopus in new tab">Scopus (893)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/8988165/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng0197-30&amp;pmid=8988165" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib9" class="citations"><div id="sref9" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref9" title="View in article">9.</a></div><div>Basson, C.T. ∙ Huang, T. ∙ Lin, R.C. ...</div><div><strong>Different TBX5 interactions in heart and limb defined by Holt-Oram syndrome mutations</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 1999; <strong>96</strong>:2919-2924</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.96.6.2919" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-13044287363" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/10077612/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.96.6.2919&amp;pmid=10077612" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib10" class="citations"><div id="sref10" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref10-1" title="View in article">10.</a></div><div>Bates, D. ∙ Mächler, M. ∙ Bolker, B. ...</div><div><strong>Fitting linear mixed-effects models using lme4</strong></div><div><em>J.&nbsp;Stat. Softw.</em> 2015; <strong>67</strong>:1-48</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.18637/jss.v067.i01" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-84943645306" target="_blank" aria-label="Open Scopus in new tab">Scopus (36021)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.18637%2Fjss.v067.i01" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib11" class="citations"><div id="sref11" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref11-1" title="View in article">11.</a></div><div>GTEx Consortium</div><div><strong>Genetic effects on gene expression across human tissues</strong></div><div><em>Nature.</em> 2017; <strong>550</strong>:204-213</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature24277" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-85031313737" target="_blank" aria-label="Open Scopus in new tab">Scopus (1802)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29022597/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature24277&amp;pmid=29022597" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib12" class="citations"><div id="sref12" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref12" title="View in article">12.</a></div><div>Bekker, H. ∙ Berendsen, H.J.C. ∙ Dijkstra, E.J. ...</div><div><strong>Gromacs: A parallel computer for molecular dynamics simulations</strong></div><div><a id="intref0265" href="https://www.scienceopen.com/document?vid=59290415-039c-4900-8e95-d649687a2473" target="_blank">https://www.scienceopen.com/document?vid=59290415-039c-4900-8e95-d649687a2473</a><div>Date: 1993</div></div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=H.BekkerH.J.C.BerendsenE.J.DijkstraS.AchteropR.van+DrunenD.van+der+SpoelA.SijbersH.KeegstraB.ReitsmaM.K.R.RenardusGromacs%3A+A+parallel+computer+for+molecular+dynamics+simulationshttps%3A%2F%2Fwww.scienceopen.com%2Fdocument%3Fvid%3D59290415-039c-4900-8e95-d649687a24731993" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib13" class="citations"><div id="sref13" class="citation"><div class="citation-content"><div class="label"><a href="#figs3" title="View in article">13.</a></div><div>Bouman, A. ∙ Alders, M. ∙ Oostra, R.J. ...</div><div><strong>Oral-facial-digital syndrome type 1 in males: Congenital heart defects are included in its phenotypic spectrum</strong></div><div><em>Am. J. Med. Genet. A.</em> 2017; <strong>173</strong>:1383-1389</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/ajmg.a.38179" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_13_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-85017332855" target="_blank" aria-label="Open Scopus in new tab">Scopus (8)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28371265/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fajmg.a.38179&amp;pmid=28371265" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib14" class="citations"><div id="sref14" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref14" title="View in article">14.</a></div><div>Bruneau, B.G. ∙ Logan, M. ∙ Davis, N. ...</div><div><strong>Chamber-specific cardiac expression of Tbx5 and heart defects in Holt-Oram syndrome</strong></div><div><em>Dev. Biol.</em> 1999; <strong>211</strong>:100-108</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1006/dbio.1999.9298" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-0033168070" target="_blank" aria-label="Open Scopus in new tab">Scopus (385)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/10373308/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1006%2Fdbio.1999.9298&amp;pmid=10373308" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib15" class="citations"><div id="sref15" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref15" title="View in article">15.</a></div><div>Bruneau, B.G. ∙ Nemer, G. ∙ Schmitt, J.P. ...</div><div><strong>A murine model of Holt-Oram syndrome defines roles of the T-box transcription factor Tbx5 in cardiogenesis and disease</strong></div><div><em>Cell.</em> 2001; <strong>106</strong>:709-721</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=10.1016%2FS0092-8674%2801%2900493-7&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=10.1016%2FS0092-8674%2801%2900493-7&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-17944378083" target="_blank" aria-label="Open Scopus in new tab">Scopus (832)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11572777/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0092-8674%2801%2900493-7&amp;pmid=11572777" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib16" class="citations"><div id="sref16" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref16" title="View in article">16.</a></div><div>Bryois, J. ∙ Skene, N.G. ∙ Hansen, T.F. ..., Eating Disorders Working Group of the Psychiatric Genomics Consortium, International Headache Genetics Consortium, 23andMe Research Team</div><div><strong>Genetic identification of cell types underlying brain complex traits yields insights into the etiology of Parkinson’s disease</strong></div><div><em>Nat. Genet.</em> 2020; <strong>52</strong>:482-493</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41588-020-0610-9" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-85084195663" target="_blank" aria-label="Open Scopus in new tab">Scopus (92)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32341526/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41588-020-0610-9&amp;pmid=32341526" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib17" class="citations"><div id="sref17" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref17" title="View in article">17.</a></div><div>Burridge, P.W. ∙ Li, Y.F. ∙ Matsa, E. ...</div><div><strong>Human induced pluripotent stem cell-derived cardiomyocytes recapitulate the predilection of breast cancer patients to doxorubicin-induced cardiotoxicity</strong></div><div><em>Nat. Med.</em> 2016; <strong>22</strong>:547-556</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nm.4087" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-84964409142" target="_blank" aria-label="Open Scopus in new tab">Scopus (424)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27089514/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnm.4087&amp;pmid=27089514" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib18" class="citations"><div id="sref18" class="citation"><div class="citation-content"><div class="label"><a href="#figs2" title="View in article">18.</a></div><div>Cao, J. ∙ O’Day, D.R. ∙ Pliner, H.A. ...</div><div><strong>A human cell atlas of fetal gene expression</strong></div><div><em>Science.</em> 2020; <strong>370</strong></div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aba7721" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-85096082939" target="_blank" aria-label="Open Scopus in new tab">Scopus (135)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aba7721" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib19" class="citations"><div id="sref19" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref19-1" title="View in article">19.</a></div><div>Case, D.A. ∙ Ben-Shalom, I.Y. ∙ Brozell, S.R. ...</div><div><strong>AMBER 2018</strong></div><div>University of California, San Francisco, 2018</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=D.A.CaseI.Y.Ben-ShalomS.R.BrozellD.S.CeruttiT.E.Cheatham%2C+IIIV.W.D.CruzeiroT.A.DardenR.E.DukeD.GhoreishiM.K.GilsonAMBER+20182018University+of+California%2C+San+Francisco" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib21" class="citations"><div id="sref21" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref21" title="View in article">20.</a></div><div>Cavalcante, R.G. ∙ Sartor, M.A.</div><div><strong>annotatr: genomic regions in context</strong></div><div><em>Bioinformatics.</em> 2017; <strong>33</strong>:2381-2383</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btx183" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-85026423268" target="_blank" aria-label="Open Scopus in new tab">Scopus (130)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28369316/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtx183&amp;pmid=28369316" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib22" class="citations"><div id="sref22" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref22" title="View in article">21.</a></div><div>Chauveau, C. ∙ Bonnemann, C.G. ∙ Julien, C. ...</div><div><strong>Recessive TTN truncating mutations define novel forms of core myopathy with heart disease</strong></div><div><em>Hum. Mol. Genet.</em> 2014; <strong>23</strong>:980-991</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/hmg/ddt494" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-84892953505" target="_blank" aria-label="Open Scopus in new tab">Scopus (121)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24105469/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fhmg%2Fddt494&amp;pmid=24105469" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib23" class="citations"><div id="sref23" class="citation"><div class="citation-content"><div class="label"><a href="#figs3" title="View in article">22.</a></div><div>Chen, J. ∙ Yuan, H. ∙ Xie, K. ...</div><div><strong>A novel TAB2 nonsense mutation (p.S149X) causing autosomal dominant congenital heart defects: a case report of a Chinese family</strong></div><div><em>BMC Cardiovasc. Disord.</em> 2020; <strong>20</strong>:27</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s12872-019-01322-1" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-85078306887" target="_blank" aria-label="Open Scopus in new tab">Scopus (6)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31959127/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs12872-019-01322-1&amp;pmid=31959127" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib24" class="citations"><div id="sref24" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref24" title="View in article">23.</a></div><div>Christianson, A. ∙ Howson, C.P.</div><div><strong>March of dimes</strong></div><div>Global Report on Birth., 2006</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.ChristiansonC.P.HowsonMarch+of+dimes2006Global+Report+on+Birth." target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib25" class="citations"><div id="sref25" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref25" title="View in article">24.</a></div><div>Clouthier, D.E. ∙ Hosoda, K. ∙ Richardson, J.A. ...</div><div><strong>Cranial and cardiac neural crest defects in endothelin-A receptor-deficient mice</strong></div><div><em>Development.</em> 1998; <strong>125</strong>:813-824</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1242/dev.125.5.813" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/9449664/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1242%2Fdev.125.5.813&amp;pmid=9449664" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib26" class="citations"><div id="sref26" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref26" title="View in article">25.</a></div><div>Cox, J. ∙ Mann, M.</div><div><strong>MaxQuant enables high peptide identification rates, individualized p.p.b.-range mass accuracies and proteome-wide protein quantification</strong></div><div><em>Nat. Biotechnol.</em> 2008; <strong>26</strong>:1367-1372</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nbt.1511" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-57449099865" target="_blank" aria-label="Open Scopus in new tab">Scopus (8817)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19029910/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnbt.1511&amp;pmid=19029910" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib27" class="citations"><div id="sref27" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref27" title="View in article">26.</a></div><div>Darden, T. ∙ York, D. ∙ Pedersen, L.</div><div><strong>Particle mesh Ewald: An N×log(N) method for Ewald sums in large systems</strong></div><div><em>J.&nbsp;Chem. Phys.</em> 1993; <strong>98</strong>:10089</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1063/1.464397" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1063%2F1.464397" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib28" class="citations"><div id="sref28" class="citation"><div class="citation-content"><div class="label"><a href="#figs3" title="View in article">27.</a></div><div>de Soysa, T.Y. ∙ Ranade, S.S. ∙ Okawa, S. ...</div><div><strong>Single-cell analysis of cardiogenesis reveals basis for organ-level developmental defects</strong></div><div><em>Nature.</em> 2019; <strong>572</strong>:120-124</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-1414-x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-85069751389" target="_blank" aria-label="Open Scopus in new tab">Scopus (91)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31341279/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-1414-x&amp;pmid=31341279" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib29" class="citations"><div id="sref29" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref29" title="View in article">28.</a></div><div>Deciphering Developmental Disorders Study</div><div><strong>Large-scale discovery of novel genetic causes of developmental disorders</strong></div><div><em>Nature.</em> 2015; <strong>519</strong>:223-228</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature14135" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-84924666082" target="_blank" aria-label="Open Scopus in new tab">Scopus (685)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25533962/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature14135&amp;pmid=25533962" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib31" class="citations"><div id="sref31" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref31" title="View in article">29.</a></div><div>Dobin, A. ∙ Davis, C.A. ∙ Schlesinger, F. ...</div><div><strong>STAR: ultrafast universal RNA-seq aligner</strong></div><div><em>Bioinformatics.</em> 2013; <strong>29</strong>:15-21</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/bts635" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-84871809302" target="_blank" aria-label="Open Scopus in new tab">Scopus (17570)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23104886/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbts635&amp;pmid=23104886" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib33" class="citations"><div id="sref33" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref33" title="View in article">30.</a></div><div>Dupays, L. ∙ Shang, C. ∙ Wilson, R. ...</div><div><strong>Sequential Binding of MEIS1 and NKX2-5 on the Popdc2 Gene: A Mechanism for Spatiotemporal Regulation of Enhancers during Cardiogenesis</strong></div><div><em>Cell Rep.</em> 2015; <strong>13</strong>:183-195</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=10.1016%2Fj.celrep.2015.08.065&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=10.1016%2Fj.celrep.2015.08.065&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-84943235008" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26411676/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.celrep.2015.08.065&amp;pmid=26411676" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib34" class="citations"><div id="sref34" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref34-1" title="View in article">31.</a></div><div>Eilbeck, K. ∙ Quinlan, A. ∙ Yandell, M.</div><div><strong>Settling the score: variant prioritization and Mendelian disease</strong></div><div><em>Nat. Rev. Genet.</em> 2017; <strong>18</strong>:599-612</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nrg.2017.52" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-85029470345" target="_blank" aria-label="Open Scopus in new tab">Scopus (133)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28804138/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnrg.2017.52&amp;pmid=28804138" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib35" class="citations"><div id="sref35" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref35-1" title="View in article">32.</a></div><div>Enane, F.O. ∙ Shuen, W.H. ∙ Gu, X. ...</div><div><strong>GATA4 loss of function in liver cancer impedes precursor to hepatocyte transition</strong></div><div><em>J.&nbsp;Clin. Invest.</em> 2017; <strong>127</strong>:3527-3542</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1172/JCI93488" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-85028908653" target="_blank" aria-label="Open Scopus in new tab">Scopus (24)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28758902/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1172%2FJCI93488&amp;pmid=28758902" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib36" class="citations"><div id="sref36" class="citation"><div class="citation-content"><div class="label"><a href="#figs4" title="View in article">33.</a></div><div>Fang, R. ∙ Chen, F. ∙ Dong, Z. ...</div><div><strong>LSD2/KDM1B and its cofactor NPAC/GLYR1 endow a structural and molecular model for regulation of H3K4 demethylation</strong></div><div><em>Mol. Cell.</em> 2013; <strong>49</strong>:558-570</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=10.1016%2Fj.molcel.2012.11.019&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=10.1016%2Fj.molcel.2012.11.019&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-84873420044" target="_blank" aria-label="Open Scopus in new tab">Scopus (64)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23260659/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.molcel.2012.11.019&amp;pmid=23260659" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib37" class="citations"><div id="sref37" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref37" title="View in article">34.</a></div><div>Farwell, K.D. ∙ Shahmirzadi, L. ∙ El-Khechen, D. ...</div><div><strong>Enhanced utility of family-centered diagnostic exome sequencing with inheritance model-based analysis: results from 500 unselected families with undiagnosed genetic conditions</strong></div><div><em>Genet. Med.</em> 2015; <strong>17</strong>:578-586</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=10.1038%2Fgim.2014.154&amp;cf=fulltext&amp;site=gim-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=10.1038%2Fgim.2014.154&amp;cf=pdf&amp;site=gim-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-84939635642" target="_blank" aria-label="Open Scopus in new tab">Scopus (332)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25356970/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fgim.2014.154&amp;pmid=25356970" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib38" class="citations"><div id="sref38" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref38-1" title="View in article">35.</a></div><div>Fei, J. ∙ Ishii, H. ∙ Hoeksema, M.A. ...</div><div><strong>NDF, a nucleosome-destabilizing factor that facilitates transcription through nucleosomes</strong></div><div><em>Genes Dev.</em> 2018; <strong>32</strong>:682-694</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/gad.313973.118" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-85047604949" target="_blank" aria-label="Open Scopus in new tab">Scopus (24)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29759984/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2Fgad.313973.118&amp;pmid=29759984" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib40" class="citations"><div id="sref40" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref40-1" title="View in article">36.</a></div><div>Finak, G. ∙ McDavid, A. ∙ Yajima, M. ...</div><div><strong>MAST: a flexible statistical framework for assessing transcriptional changes and characterizing heterogeneity in single-cell RNA sequencing data</strong></div><div><em>Genome Biol.</em> 2015; <strong>16</strong>:278</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13059-015-0844-5" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-84951574149" target="_blank" aria-label="Open Scopus in new tab">Scopus (870)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26653891/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13059-015-0844-5&amp;pmid=26653891" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib41" class="citations"><div id="sref41" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref41" title="View in article">37.</a></div><div>Fu, J. ∙ Yang, Z. ∙ Wei, J. ...</div><div><strong>Nuclear protein NP60 regulates p38 MAPK activity</strong></div><div><em>J.&nbsp;Cell Sci.</em> 2006; <strong>119</strong>:115-123</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1242/jcs.02699" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-31644442352" target="_blank" aria-label="Open Scopus in new tab">Scopus (13)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16352664/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1242%2Fjcs.02699&amp;pmid=16352664" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib42" class="citations"><div id="sref42" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref42" title="View in article">38.</a></div><div>Fuller, Z.L. ∙ Berg, J.J. ∙ Mostafavi, H. ...</div><div><strong>Measuring intolerance to mutation in human genetics</strong></div><div><em>Nat. Genet.</em> 2019; <strong>51</strong>:772-776</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41588-019-0383-1" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_38_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-85064090845" target="_blank" aria-label="Open Scopus in new tab">Scopus (43)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30962618/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41588-019-0383-1&amp;pmid=30962618" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib43" class="citations"><div id="sref43" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref43" title="View in article">39.</a></div><div>Furtado, M.B. ∙ Wilmanns, J.C. ∙ Chandran, A. ...</div><div><strong>Point mutations in murine <i>Nkx2-5</i> phenocopy human congenital heart disease and induce pathogenic Wnt signaling</strong></div><div><em>JCI Insight.</em> 2017; <strong>2</strong>:e88271</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1172/jci.insight.88271" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-85054931220" target="_blank" aria-label="Open Scopus in new tab">Scopus (14)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28352650/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1172%2Fjci.insight.88271&amp;pmid=28352650" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib44" class="citations"><div id="sref44" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref44-1" title="View in article">40.</a></div><div>Garg, V. ∙ Kathiriya, I.S. ∙ Barnes, R. ...</div><div><strong>GATA4 mutations cause human congenital heart defects and reveal an interaction with TBX5</strong></div><div><em>Nature.</em> 2003; <strong>424</strong>:443-447</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature01827" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_40_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-0043267988" target="_blank" aria-label="Open Scopus in new tab">Scopus (953)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/12845333/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature01827&amp;pmid=12845333" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib45" class="citations"><div id="sref45" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref45-1" title="View in article">41.</a></div><div>Gifford, C.A. ∙ Ranade, S.S. ∙ Samarakoon, R. ...</div><div><strong>Oligogenic inheritance of a human heart disease involving a genetic modifier</strong></div><div><em>Science.</em> 2019; <strong>364</strong>:865-870</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aat5056" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-85066940786" target="_blank" aria-label="Open Scopus in new tab">Scopus (84)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31147515/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aat5056&amp;pmid=31147515" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib46" class="citations"><div id="sref46" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref46" title="View in article">42.</a></div><div>Goh, K.-I. ∙ Cusick, M.E. ∙ Valle, D. ...</div><div><strong>The human disease network</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2007; <strong>104</strong>:8685-8690</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.0701361104" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-34547140875" target="_blank" aria-label="Open Scopus in new tab">Scopus (2412)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17502601/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.0701361104&amp;pmid=17502601" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib47" class="citations"><div id="sref47" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref47" title="View in article">43.</a></div><div>González-Terán, B. ∙ López, J.A. ∙ Rodríguez, E. ...</div><div><strong>p38γ and δ promote heart hypertrophy by targeting the mTOR-inhibitory protein DEPTOR for degradation</strong></div><div><em>Nat. Commun.</em> 2016; <strong>7</strong>:10477</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ncomms10477" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-84955456248" target="_blank" aria-label="Open Scopus in new tab">Scopus (47)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26795633/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fncomms10477&amp;pmid=26795633" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib49" class="citations"><div id="sref49" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref49-1" title="View in article">44.</a></div><div>Grau, J. ∙ Grosse, I. ∙ Keilwagen, J.</div><div><strong>PRROC: computing and visualizing precision-recall and receiver operating characteristic curves in R</strong></div><div><em>Bioinformatics.</em> 2015; <strong>31</strong>:2595-2597</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btv153" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-84943599060" target="_blank" aria-label="Open Scopus in new tab">Scopus (170)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25810428/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtv153&amp;pmid=25810428" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib50" class="citations"><div id="sref50" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref50" title="View in article">45.</a></div><div>Greene, C.S. ∙ Krishnan, A. ∙ Wong, A.K. ...</div><div><strong>Understanding multicellular function and disease with human tissue-specific networks</strong></div><div><em>Nat. Genet.</em> 2015; <strong>47</strong>:569-576</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng.3259" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_45_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-84930083700" target="_blank" aria-label="Open Scopus in new tab">Scopus (461)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25915600/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng.3259&amp;pmid=25915600" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib51" class="citations"><div id="sref51" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref51-1" title="View in article">46.</a></div><div>Guo, Y. ∙ Mahony, S. ∙ Gifford, D.K.</div><div><strong>High resolution genome wide binding event finding and motif discovery reveals transcription factor spatial binding constraints</strong></div><div><em>PLoS Comput. Biol.</em> 2012; <strong>8</strong>:e1002638</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pcbi.1002638" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-84866086842" target="_blank" aria-label="Open Scopus in new tab">Scopus (176)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22912568/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pcbi.1002638&amp;pmid=22912568" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib52" class="citations"><div id="sref52" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref52-1" title="View in article">47.</a></div><div>Hao, Y. ∙ Hao, S. ∙ Andersen-Nissen, E. ...</div><div><strong>Integrated analysis of multimodal single-cell data</strong></div><div><em>Cell.</em> 2021; <strong>184</strong>:3573-3587.e29</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=10.1016%2Fj.cell.2021.04.048&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=10.1016%2Fj.cell.2021.04.048&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-85108179646" target="_blank" aria-label="Open Scopus in new tab">Scopus (800)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/34062119/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2021.04.048&amp;pmid=34062119" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib53" class="citations"><div id="sref53" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref53-1" title="View in article">48.</a></div><div>Heinz, S. ∙ Benner, C. ∙ Spann, N. ...</div><div><strong>Simple combinations of lineage-determining transcription factors prime cis-regulatory elements required for macrophage and B cell identities</strong></div><div><em>Mol. Cell.</em> 2010; <strong>38</strong>:576-589</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_48_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=10.1016%2Fj.molcel.2010.05.004&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_48_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=10.1016%2Fj.molcel.2010.05.004&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_48_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-77952567987" target="_blank" aria-label="Open Scopus in new tab">Scopus (6049)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20513432/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.molcel.2010.05.004&amp;pmid=20513432" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib54" class="citations"><div id="sref54" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref54" title="View in article">49.</a></div><div>Hekselman, I. ∙ Yeger-Lotem, E.</div><div><strong>Mechanisms of tissue and cell-type specificity in heritable traits and diseases</strong></div><div><em>Nat. Rev. Genet.</em> 2020; <strong>21</strong>:137-150</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41576-019-0200-9" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-85077632359" target="_blank" aria-label="Open Scopus in new tab">Scopus (43)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31913361/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41576-019-0200-9&amp;pmid=31913361" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib55" class="citations"><div id="sref55" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref55" title="View in article">50.</a></div><div>Hinton, R.B. ∙ Prakash, A. ∙ Romp, R.L. ..., International Tuberous Sclerosis Consensus Group</div><div><strong>Cardiovascular manifestations of tuberous sclerosis complex and summary of the revised diagnostic criteria and surveillance and management recommendations from the International Tuberous Sclerosis Consensus Group</strong></div><div><em>J.&nbsp;Am. Heart Assoc.</em> 2014; <strong>3</strong>:e001493</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1161/JAHA.114.001493" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-84935004324" target="_blank" aria-label="Open Scopus in new tab">Scopus (75)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25424575/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1161%2FJAHA.114.001493&amp;pmid=25424575" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib56" class="citations"><div id="sref56" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref56-1" title="View in article">51.</a></div><div>Homsy, J. ∙ Zaidi, S. ∙ Shen, Y. ...</div><div><strong>De novo mutations in congenital heart disease with neurodevelopmental and other congenital anomalies</strong></div><div><em>Science.</em> 2015; <strong>350</strong>:1262-1266</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aac9396" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_51_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-84949008129" target="_blank" aria-label="Open Scopus in new tab">Scopus (412)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26785492/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aac9396&amp;pmid=26785492" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib57" class="citations"><div id="sref57" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref57" title="View in article">52.</a></div><div>Hota, S.K. ∙ Bruneau, B.G.</div><div><strong>ATP-dependent chromatin remodeling during mammalian development</strong></div><div><em>Development.</em> 2016; <strong>143</strong>:2882-2897</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1242/dev.128892" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-84983085494" target="_blank" aria-label="Open Scopus in new tab">Scopus (123)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27531948/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1242%2Fdev.128892&amp;pmid=27531948" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib59" class="citations"><div id="sref59" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref59-1" title="View in article">53.</a></div><div>Huang, N. ∙ Lee, I. ∙ Marcotte, E.M. ...</div><div><strong>Characterising and predicting haploinsufficiency in the human genome</strong></div><div><em>PLoS Genet.</em> 2010; <strong>6</strong>:e1001154</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pgen.1001154" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-78449263023" target="_blank" aria-label="Open Scopus in new tab">Scopus (438)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20976243/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pgen.1001154&amp;pmid=20976243" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib60" class="citations"><div id="sref60" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref60-1" title="View in article">54.</a></div><div>Izarzugaza, J.M.G. ∙ Ellesøe, S.G. ∙ Doganli, C. ..., Competence Network for Congenital Heart Defects, Germany</div><div><strong>Systems genetics analysis identifies calcium-signaling defects as novel cause of congenital heart disease</strong></div><div><em>Genome Med.</em> 2020; <strong>12</strong>:76</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13073-020-00772-z" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-85090171445" target="_blank" aria-label="Open Scopus in new tab">Scopus (5)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32859249/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13073-020-00772-z&amp;pmid=32859249" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib62" class="citations"><div id="sref62" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref62-1" title="View in article">55.</a></div><div>Jimenez-Morales, D. ∙ Rosa Campos, A. ∙ Von Dollen, J. ...</div><div><strong>artMS: Analytical R tools for Mass Spectrometry version 1.6.5 from Bioconductor</strong></div><div><em>Bioconductor.</em> 2020;</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=D.Jimenez-MoralesA.Rosa+CamposJ.Von+DollenD.SwaneyartMS%3A+Analytical+R+tools+for+Mass+Spectrometry+version+1.6.5+from+BioconductorBioconductor2020" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib63" class="citations"><div id="sref63" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref63-1" title="View in article">56.</a></div><div>Jin, S.C. ∙ Homsy, J. ∙ Zaidi, S. ...</div><div><strong>Contribution of rare inherited and de novo variants in 2,871 congenital heart disease probands</strong></div><div><em>Nat. Genet.</em> 2017; <strong>49</strong>:1593-1601</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng.3970" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_56_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-85032451853" target="_blank" aria-label="Open Scopus in new tab">Scopus (358)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28991257/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng.3970&amp;pmid=28991257" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib64" class="citations"><div id="sref64" class="citation"><div class="citation-content"><div class="label"><a href="#figs3" title="View in article">57.</a></div><div>Jones, W.D. ∙ Dafou, D. ∙ McEntagart, M. ...</div><div><strong>De novo mutations in MLL cause Wiedemann-Steiner syndrome</strong></div><div><em>Am. J. Hum. Genet.</em> 2012; <strong>91</strong>:358-364</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=10.1016%2Fj.ajhg.2012.06.008&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=10.1016%2Fj.ajhg.2012.06.008&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-84864958596" target="_blank" aria-label="Open Scopus in new tab">Scopus (171)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22795537/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.ajhg.2012.06.008&amp;pmid=22795537" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib65" class="citations"><div id="sref65" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref65" title="View in article">58.</a></div><div>Jorgensen, W.L. ∙ Chandrasekhar, J. ∙ Madura, J.D. ...</div><div><strong>Comparison of simple potential functions for simulating liquid water</strong></div><div><em>J.&nbsp;Chem. Phys.</em> 1983; <strong>79</strong>:926</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1063/1.445869" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1063%2F1.445869" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib66" class="citations"><div id="sref66" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref66-1" title="View in article">59.</a></div><div>Karczewski, K.J. ∙ Francioli, L.C. ∙ Tiao, G. ..., Genome Aggregation Database Consortium</div><div><strong>The mutational constraint spectrum quantified from variation in 141,456 humans</strong></div><div><em>Nature.</em> 2020; <strong>581</strong>:434-443</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-020-2308-7" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-85085542423" target="_blank" aria-label="Open Scopus in new tab">Scopus (2715)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32461654/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-020-2308-7&amp;pmid=32461654" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib67" class="citations"><div id="sref67" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref67" title="View in article">60.</a></div><div>Kathiriya, I.S. ∙ Rao, K.S. ∙ Iacono, G. ...</div><div><strong>Modeling human TBX5 haploinsufficiency predicts regulatory networks for congenital heart disease</strong></div><div><em>Dev. Cell.</em> 2021; <strong>56</strong>:292-309.e9</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=10.1016%2Fj.devcel.2020.11.020&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=10.1016%2Fj.devcel.2020.11.020&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-85099131520" target="_blank" aria-label="Open Scopus in new tab">Scopus (20)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33321106/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.devcel.2020.11.020&amp;pmid=33321106" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib68" class="citations"><div id="sref68" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref68" title="View in article">61.</a></div><div>Kimura, H.</div><div><strong>Histone modifications for human epigenome analysis</strong></div><div><em>J.&nbsp;Hum. Genet.</em> 2013; <strong>58</strong>:439-445</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/jhg.2013.66" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-84880864749" target="_blank" aria-label="Open Scopus in new tab">Scopus (253)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23739122/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fjhg.2013.66&amp;pmid=23739122" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib69" class="citations"><div id="sref69" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref69-1" title="View in article">62.</a></div><div>Knowlton, K.U. ∙ Baracchini, E. ∙ Ross, R.S. ...</div><div><strong>Co-regulation of the atrial natriuretic factor and cardiac myosin light chain-2 genes during alpha-adrenergic stimulation of neonatal rat ventricular cells. Identification of cis sequences within an embryonic and a constitutive contractile protein gene which mediate inducible expression</strong></div><div><em>J.&nbsp;Biol. Chem.</em> 1991; <strong>266</strong>:7759-7768</div></div><div class="external-links"><div class="core-xlink-abstract"><a href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=10.1016%2FS0021-9258%2820%2989515-5&amp;cf=abstract&amp;site=jbc-site" target="_blank" aria-label="Open Abstract in new tab">Abstract</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=10.1016%2FS0021-9258%2820%2989515-5&amp;cf=pdf&amp;site=jbc-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/1850419/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0021-9258%2820%2989515-5&amp;pmid=1850419" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib71" class="citations"><div id="sref71" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref71-1" title="View in article">63.</a></div><div>Köhler, S. ∙ Bauer, S. ∙ Horn, D. ...</div><div><strong>Walking the interactome for prioritization of candidate disease genes</strong></div><div><em>Am. J. Hum. Genet.</em> 2008; <strong>82</strong>:949-958</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=10.1016%2Fj.ajhg.2008.02.013&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=10.1016%2Fj.ajhg.2008.02.013&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-41549139527" target="_blank" aria-label="Open Scopus in new tab">Scopus (910)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/18371930/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.ajhg.2008.02.013&amp;pmid=18371930" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib72" class="citations"><div id="sref72" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref72-1" title="View in article">64.</a></div><div>Korsunsky, I. ∙ Millard, N. ∙ Fan, J. ...</div><div><strong>Fast, sensitive and accurate integration of single-cell data with Harmony</strong></div><div><em>Nat. Methods.</em> 2019; <strong>16</strong>:1289-1296</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41592-019-0619-0" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_64_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-85070698712" target="_blank" aria-label="Open Scopus in new tab">Scopus (806)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31740819/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41592-019-0619-0&amp;pmid=31740819" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib73" class="citations"><div id="sref73" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref73" title="View in article">65.</a></div><div>Kuo, C.T. ∙ Morrisey, E.E. ∙ Anandappa, R. ...</div><div><strong>GATA4 transcription factor is required for ventral morphogenesis and heart tube formation</strong></div><div><em>Genes Dev.</em> 1997; <strong>11</strong>:1048-1060</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/gad.11.8.1048" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/9136932/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2Fgad.11.8.1048&amp;pmid=9136932" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib74" class="citations"><div id="sref74" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref74" title="View in article">66.</a></div><div>Lambert, S.A. ∙ Jolma, A. ∙ Campitelli, L.F. ...</div><div><strong>The human transcription factors</strong></div><div><em>Cell.</em> 2018; <strong>172</strong>:650-665</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=10.1016%2Fj.cell.2018.01.029&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=10.1016%2Fj.cell.2018.01.029&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-85041527619" target="_blank" aria-label="Open Scopus in new tab">Scopus (848)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29425488/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2018.01.029&amp;pmid=29425488" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib75" class="citations"><div id="sref75" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref75-1" title="View in article">67.</a></div><div>Langmead, B. ∙ Salzberg, S.L.</div><div><strong>Fast gapped-read alignment with Bowtie 2</strong></div><div><em>Nat. Methods.</em> 2012; <strong>9</strong>:357-359</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.1923" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_67_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-84859210032" target="_blank" aria-label="Open Scopus in new tab">Scopus (24558)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22388286/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.1923&amp;pmid=22388286" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib76" class="citations"><div id="sref76" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref76-1" title="View in article">68.</a></div><div>Lau, E. ∙ Han, Y. ∙ Williams, D.R. ...</div><div><strong>Splice-Junction-Based Mapping of Alternative Isoforms in the Human Proteome</strong></div><div><em>Cell Rep.</em> 2019; <strong>29</strong>:3751-3765.e5</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=10.1016%2Fj.celrep.2019.11.026&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=10.1016%2Fj.celrep.2019.11.026&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-85076033361" target="_blank" aria-label="Open Scopus in new tab">Scopus (26)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31825849/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.celrep.2019.11.026&amp;pmid=31825849" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib80" class="citations"><div id="sref80" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref80" title="View in article">69.</a></div><div>Li, Q.Y. ∙ Newbury-Ecob, R.A. ∙ Terrett, J.A. ...</div><div><strong>Holt-Oram syndrome is caused by mutations in TBX5, a member of the Brachyury (T) gene family</strong></div><div><em>Nat. Genet.</em> 1997; <strong>15</strong>:21-29</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng0197-21" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-1842413728" target="_blank" aria-label="Open Scopus in new tab">Scopus (748)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/8988164/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng0197-21&amp;pmid=8988164" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib82" class="citations"><div id="sref82" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref82" title="View in article">70.</a></div><div>Lian, X. ∙ Zhang, J. ∙ Azarin, S.M. ...</div><div><strong>Directed cardiomyocyte differentiation from human pluripotent stem cells by modulating Wnt/β-catenin signaling under fully defined conditions</strong></div><div><em>Nat. Protoc.</em> 2013; <strong>8</strong>:162-175</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nprot.2012.150" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-84872016692" target="_blank" aria-label="Open Scopus in new tab">Scopus (941)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23257984/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnprot.2012.150&amp;pmid=23257984" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib83" class="citations"><div id="sref83" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref83-1" title="View in article">71.</a></div><div>Liao, Y. ∙ Smyth, G.K. ∙ Shi, W.</div><div><strong>featureCounts: an efficient general purpose program for assigning sequence reads to genomic features</strong></div><div><em>Bioinformatics.</em> 2014; <strong>30</strong>:923-930</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btt656" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-84897397058" target="_blank" aria-label="Open Scopus in new tab">Scopus (7989)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24227677/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtt656&amp;pmid=24227677" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib84" class="citations"><div id="sref84" class="citation"><div class="citation-content"><div class="label"><a href="#fig4" title="View in article">72.</a></div><div>Liu, Y. ∙ Harmelink, C. ∙ Peng, Y. ...</div><div><strong>CHD7 interacts with BMP R-SMADs to epigenetically regulate cardiogenesis in mice</strong></div><div><em>Hum. Mol. Genet.</em> 2014; <strong>23</strong>:2145-2156</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/hmg/ddt610" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-84897835537" target="_blank" aria-label="Open Scopus in new tab">Scopus (41)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24293546/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fhmg%2Fddt610&amp;pmid=24293546" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib85" class="citations"><div id="sref85" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref85" title="View in article">73.</a></div><div>Lun, A.T.L. ∙ Chen, Y. ∙ Smyth, G.K.</div><div><strong>It’s DE-licious: A Recipe for Differential Expression Analyses of RNA-seq Experiments Using Quasi-Likelihood Methods in edgeR</strong></div><div><em>Methods Mol. Biol.</em> 2016; <strong>1418</strong>:391-416</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/978-1-4939-3578-9_19" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-84961614091" target="_blank" aria-label="Open Scopus in new tab">Scopus (213)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27008025/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2F978-1-4939-3578-9_19&amp;pmid=27008025" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib86" class="citations"><div id="sref86" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref86-1" title="View in article">74.</a></div><div>Luna-Zurita, L. ∙ Stirnimann, C.U. ∙ Glatt, S. ...</div><div><strong>Complex interdependence regulates heterotypic transcription factor distribution and coordinates cardiogenesis</strong></div><div><em>Cell.</em> 2016; <strong>164</strong>:999-1014</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=10.1016%2Fj.cell.2016.01.004&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=10.1016%2Fj.cell.2016.01.004&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-84959387425" target="_blank" aria-label="Open Scopus in new tab">Scopus (122)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26875865/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2016.01.004&amp;pmid=26875865" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib87" class="citations"><div id="sref87" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref87" title="View in article">75.</a></div><div>Maddah, M. ∙ Heidmann, J.D. ∙ Mandegar, M.A. ...</div><div><strong>A non-invasive platform for functional characterization of stem-cell-derived cardiomyocytes with applications in cardiotoxicity testing</strong></div><div><em>Stem Cell Reports.</em> 2015; <strong>4</strong>:621-631</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_75_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=10.1016%2Fj.stemcr.2015.02.007&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_75_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=10.1016%2Fj.stemcr.2015.02.007&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_75_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-84933672526" target="_blank" aria-label="Open Scopus in new tab">Scopus (87)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25801505/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.stemcr.2015.02.007&amp;pmid=25801505" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib88" class="citations"><div id="sref88" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref88" title="View in article">76.</a></div><div>Maere, S. ∙ Heymans, K. ∙ Kuiper, M.</div><div><strong>BiNGO: a Cytoscape plugin to assess overrepresentation of gene ontology categories in biological networks</strong></div><div><em>Bioinformatics.</em> 2005; <strong>21</strong>:3448-3449</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/bti551" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_76_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-24044440971" target="_blank" aria-label="Open Scopus in new tab">Scopus (3028)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15972284/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbti551&amp;pmid=15972284" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib89" class="citations"><div id="sref89" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref89" title="View in article">77.</a></div><div>Maestro Schrödinger, LLC</div><div><strong>Maestro Suite of Programs (v. 2019-4)</strong></div><div>Maestro Schrödinger, LLC, 2019</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=Maestro+Schr%C3%B6dinger%2C+LLCMaestro+Suite+of+Programs+%28v.+2019-4%292019Maestro+Schr%C3%B6dinger%2C+LLC" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib90" class="citations"><div id="sref90" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref90" title="View in article">78.</a></div><div>Magger, O. ∙ Waldman, Y.Y. ∙ Ruppin, E. ...</div><div><strong>Enhancing the prioritization of disease-causing genes through tissue specific protein interaction networks</strong></div><div><em>PLoS Comput. Biol.</em> 2012; <strong>8</strong>:e1002690</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pcbi.1002690" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_78_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-84866888440" target="_blank" aria-label="Open Scopus in new tab">Scopus (117)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23028288/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pcbi.1002690&amp;pmid=23028288" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib91" class="citations"><div id="sref91" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref91" title="View in article">79.</a></div><div>Maitra, M. ∙ Schluterman, M.K. ∙ Nichols, H.A. ...</div><div><strong>Interaction of Gata4 and Gata6 with Tbx5 is critical for normal cardiac development</strong></div><div><em>Dev. Biol.</em> 2009; <strong>326</strong>:368-377</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.ydbio.2008.11.004" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_79_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-58649091740" target="_blank" aria-label="Open Scopus in new tab">Scopus (149)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19084512/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.ydbio.2008.11.004&amp;pmid=19084512" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib92" class="citations"><div id="sref92" class="citation"><div class="citation-content"><div class="label"><a href="#figs3" title="View in article">80.</a></div><div>Maitra, M. ∙ Koenig, S.N. ∙ Srivastava, D. ...</div><div><strong>Identification of GATA6 sequence variants in patients with congenital heart defects</strong></div><div><em>Pediatr. Res.</em> 2010; <strong>68</strong>:281-285</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1203/PDR.0b013e3181ed17e4" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_80_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-77957274392" target="_blank" aria-label="Open Scopus in new tab">Scopus (85)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20581743/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1203%2FPDR.0b013e3181ed17e4&amp;pmid=20581743" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib93" class="citations"><div id="sref93" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref93-1" title="View in article">81.</a></div><div>Marabelli, C. ∙ Marrocco, B. ∙ Pilotto, S. ...</div><div><strong>A Tail-Based Mechanism Drives Nucleosome Demethylation by the LSD2/NPAC Multimeric Complex</strong></div><div><em>Cell Rep.</em> 2019; <strong>27</strong>:387-399.e7</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=10.1016%2Fj.celrep.2019.03.061&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=10.1016%2Fj.celrep.2019.03.061&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-85063864248" target="_blank" aria-label="Open Scopus in new tab">Scopus (12)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30970244/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.celrep.2019.03.061&amp;pmid=30970244" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib94" class="citations"><div id="sref94" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref94" title="View in article">82.</a></div><div>Miyamoto, S. ∙ Kollman, P.A.</div><div><strong>Settle: An analytical version of the SHAKE and RATTLE algorithm for rigid water models</strong></div><div><em>J.&nbsp;Comput. Chem.</em> 1992; <strong>13</strong>:952-962</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/jcc.540130805" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fjcc.540130805" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib95" class="citations"><div id="sref95" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref95" title="View in article">83.</a></div><div>Miyaoka, Y. ∙ Chan, A.H. ∙ Judge, L.M. ...</div><div><strong>Isolation of single-base genome-edited human iPS cells without antibiotic selection</strong></div><div><em>Nat. Methods.</em> 2014; <strong>11</strong>:291-293</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.2840" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_83_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-84895528015" target="_blank" aria-label="Open Scopus in new tab">Scopus (161)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24509632/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.2840&amp;pmid=24509632" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib96" class="citations"><div id="sref96" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref96" title="View in article">84.</a></div><div>Molkentin, J.D. ∙ Lin, Q. ∙ Duncan, S.A. ...</div><div><strong>Requirement of the transcription factor GATA4 for heart tube formation and ventral morphogenesis</strong></div><div><em>Genes Dev.</em> 1997; <strong>11</strong>:1061-1072</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/gad.11.8.1061" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/9136933/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2Fgad.11.8.1061&amp;pmid=9136933" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib97" class="citations"><div id="sref97" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref97" title="View in article">85.</a></div><div>Montefiori, M. ∙ Pilotto, S. ∙ Marabelli, C. ...</div><div><strong>Impact of Mutations on NPAC Structural Dynamics: Mechanistic Insights from MD Simulations</strong></div><div><em>J.&nbsp;Chem. Inf. Model.</em> 2019; <strong>59</strong>:3927-3937</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/acs.jcim.9b00588" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-85071873088" target="_blank" aria-label="Open Scopus in new tab">Scopus (19)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31408337/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Facs.jcim.9b00588&amp;pmid=31408337" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib98" class="citations"><div id="sref98" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref98" title="View in article">86.</a></div><div>Mori, A.D. ∙ Zhu, Y. ∙ Vahora, I. ...</div><div><strong>Tbx5-dependent rheostatic control of cardiac gene expression and morphogenesis</strong></div><div><em>Dev. Biol.</em> 2006; <strong>297</strong>:566-586</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.ydbio.2006.05.023" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_86_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-33748489726" target="_blank" aria-label="Open Scopus in new tab">Scopus (144)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16870172/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.ydbio.2006.05.023&amp;pmid=16870172" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib99" class="citations"><div id="sref99" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref99" title="View in article">87.</a></div><div>Moskowitz, I.P. ∙ Wang, J. ∙ Peterson, M.A. ...</div><div><strong>Transcription factor genes Smad4 and Gata4 cooperatively regulate cardiac valve development. [corrected]</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2011; <strong>108</strong>:4006-4011</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1019025108" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_87_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-79952760100" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21330551/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1019025108&amp;pmid=21330551" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib100" class="citations"><div id="sref100" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref100" title="View in article">88.</a></div><div>Nakamura, H. ∙ Cook, R.N. ∙ Justice, M.J.</div><div><strong>Mouse Tenm4 is required for mesoderm induction</strong></div><div><em>BMC Dev. Biol.</em> 2013; <strong>13</strong>:9</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/1471-213X-13-9" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-84875327009" target="_blank" aria-label="Open Scopus in new tab">Scopus (16)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23521771/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2F1471-213X-13-9&amp;pmid=23521771" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib101" class="citations"><div id="sref101" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref101" title="View in article">89.</a></div><div>Narita, N. ∙ Bielinska, M. ∙ Wilson, D.B.</div><div><strong>Cardiomyocyte differentiation by GATA-4-deficient embryonic stem cells</strong></div><div><em>Development.</em> 1997; <strong>124</strong>:3755-3764</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1242/dev.124.19.3755" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/9367431/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1242%2Fdev.124.19.3755&amp;pmid=9367431" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib102" class="citations"><div id="sref102" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref102" title="View in article">90.</a></div><div>Oughtred, R. ∙ Rust, J. ∙ Chang, C. ...</div><div><strong>The BioGRID database: A comprehensive biomedical resource of curated protein, genetic, and chemical interactions</strong></div><div><em>Protein Sci.</em> 2021; <strong>30</strong>:187-200</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/pro.3978" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_90_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-85096803916" target="_blank" aria-label="Open Scopus in new tab">Scopus (215)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33070389/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fpro.3978&amp;pmid=33070389" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib103" class="citations"><div id="sref103" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref103-1" title="View in article">91.</a></div><div>Padmanabhan, A. ∙ Alexanian, M. ∙ Linares-Saldana, R. ...</div><div><strong>BRD4 (Bromodomain-Containing Protein 4) Interacts with GATA4 (GATA Binding Protein 4) to Govern Mitochondrial Homeostasis in Adult Cardiomyocytes</strong></div><div><em>Circulation.</em> 2020; <strong>142</strong>:2338-2355</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1161/CIRCULATIONAHA.120.047753" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-85097828347" target="_blank" aria-label="Open Scopus in new tab">Scopus (14)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33094644/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1161%2FCIRCULATIONAHA.120.047753&amp;pmid=33094644" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib104" class="citations"><div id="sref104" class="citation"><div class="citation-content"><div class="label"><a href="#figs3" title="View in article">92.</a></div><div>Parisot, P. ∙ Bajolle, F. ∙ Attié-Bittach, T. ...</div><div><strong>321 Congenital heart defects in CHARGE syndrome patients with CHD7 mutations</strong></div><div><em>Archives of Cardiovascular Diseases Supplements.</em> 2010; <strong>2</strong>:104-105</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/S1878-6480(10)70323-2" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS1878-6480%2810%2970323-2" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib105" class="citations"><div id="sref105" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref105" title="View in article">93.</a></div><div>Perez-Riverol, Y. ∙ Csordas, A. ∙ Bai, J. ...</div><div><strong>The PRIDE database and related tools and resources in 2019: improving support for quantification data</strong></div><div><em>Nucleic Acids Res.</em> 2019; <strong>47</strong>:D442-D450</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gky1106" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_93_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-85058095584" target="_blank" aria-label="Open Scopus in new tab">Scopus (4040)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30395289/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgky1106&amp;pmid=30395289" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib106" class="citations"><div id="sref106" class="citation"><div class="citation-content"><div class="label"><a href="#figs3" title="View in article">94.</a></div><div>Pierpont, M.E. ∙ Brueckner, M. ∙ Chung, W.K. ..., American Heart Association Council on Cardiovascular Disease in the Young; Council on Cardiovascular and Stroke Nursing; and Council on Genomic and Precision Medicine</div><div><strong>Genetic basis for congenital heart disease: revisited: A scientific statement from the american heart association</strong></div><div><em>Circulation.</em> 2018; <strong>138</strong>:e653-e711</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1161/CIR.0000000000000606" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_94_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-85058908137" target="_blank" aria-label="Open Scopus in new tab">Scopus (203)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30571578/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1161%2FCIR.0000000000000606&amp;pmid=30571578" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib107" class="citations"><div id="sref107" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref107" title="View in article">95.</a></div><div>Priest, J.R. ∙ Osoegawa, K. ∙ Mohammed, N. ...</div><div><strong>De novo and rare variants at multiple loci support the oligogenic origins of atrioventricular septal heart defects</strong></div><div><em>PLoS Genet.</em> 2016; <strong>12</strong>:e1005963</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pgen.1005963" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27058611/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pgen.1005963&amp;pmid=27058611" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib108" class="citations"><div id="sref108" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref108-1" title="View in article">96.</a></div><div>R Core Team</div><div><strong>R: A language and environment for statistical computing</strong></div><div>R Foundation for Statistical Computing, Vienna, Austria, 2020</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=R+Core+TeamR%3A+A+language+and+environment+for+statistical+computing2020R+Foundation+for+Statistical+ComputingVienna%2C+Austria" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib109" class="citations"><div id="sref109" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref109-1" title="View in article">97.</a></div><div>Rahman, S. ∙ Sowa, M.E. ∙ Ottinger, M. ...</div><div><strong>The Brd4 extraterminal domain confers transcription activation independent of pTEFb by recruiting multiple proteins, including NSD3</strong></div><div><em>Mol. Cell. Biol.</em> 2011; <strong>31</strong>:2641-2652</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/MCB.01341-10" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_97_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-79959483607" target="_blank" aria-label="Open Scopus in new tab">Scopus (356)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21555454/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FMCB.01341-10&amp;pmid=21555454" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib110" class="citations"><div id="sref110" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref110-1" title="View in article">98.</a></div><div>Rajagopal, S.K. ∙ Ma, Q. ∙ Obler, D. ...</div><div><strong>Spectrum of heart disease associated with murine and human GATA4 mutation</strong></div><div><em>J.&nbsp;Mol. Cell. Cardiol.</em> 2007; <strong>43</strong>:677-685</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_98_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=10.1016%2Fj.yjmcc.2007.06.004&amp;cf=fulltext&amp;site=yjmcc-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_98_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=10.1016%2Fj.yjmcc.2007.06.004&amp;cf=pdf&amp;site=yjmcc-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_98_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-35048838310" target="_blank" aria-label="Open Scopus in new tab">Scopus (197)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17643447/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.yjmcc.2007.06.004&amp;pmid=17643447" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib111" class="citations"><div id="sref111" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref111" title="View in article">99.</a></div><div>Ramírez, F. ∙ Ryan, D.P. ∙ Grüning, B. ...</div><div><strong>deepTools2: a next generation web server for deep-sequencing data analysis</strong></div><div><em>Nucleic Acids Res.</em> 2016; <strong>44</strong>:W160-5</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gkw257" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_99_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-85017133693" target="_blank" aria-label="Open Scopus in new tab">Scopus (1768)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27079975/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgkw257&amp;pmid=27079975" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib112" class="citations"><div id="sref112" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref112-1" title="View in article">100.</a></div><div>Razick, S. ∙ Magklaras, G. ∙ Donaldson, I.M.</div><div><strong>iRefIndex: a consolidated protein interaction database with provenance</strong></div><div><em>BMC Bioinformatics.</em> 2008; <strong>9</strong>:405</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/1471-2105-9-405" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_100_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-54949127160" target="_blank" aria-label="Open Scopus in new tab">Scopus (405)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/18823568/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2F1471-2105-9-405&amp;pmid=18823568" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib113" class="citations"><div id="sref113" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref113-1" title="View in article">101.</a></div><div>Rentzsch, P. ∙ Witten, D. ∙ Cooper, G.M. ...</div><div><strong>CADD: predicting the deleteriousness of variants throughout the human genome</strong></div><div><em>Nucleic Acids Res.</em> 2019; <strong>47</strong>:D886-D894</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gky1016" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_101_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-85059798241" target="_blank" aria-label="Open Scopus in new tab">Scopus (1261)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30371827/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgky1016&amp;pmid=30371827" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib114" class="citations"><div id="sref114" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref114" title="View in article">102.</a></div><div>Richter, F. ∙ Morton, S.U. ∙ Kim, S.W. ...</div><div><strong>Genomic analyses implicate noncoding de novo variants in congenital heart disease</strong></div><div><em>Nat. Genet.</em> 2020; <strong>52</strong>:769-777</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41588-020-0652-z" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_102_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-85087032848" target="_blank" aria-label="Open Scopus in new tab">Scopus (46)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32601476/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41588-020-0652-z&amp;pmid=32601476" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib115" class="citations"><div id="sref115" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref115" title="View in article">103.</a></div><div>Robinson, M.D. ∙ Oshlack, A.</div><div><strong>A scaling normalization method for differential expression analysis of RNA-seq data</strong></div><div><em>Genome Biol.</em> 2010; <strong>11</strong>:R25</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/gb-2010-11-3-r25" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_103_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-77953176036" target="_blank" aria-label="Open Scopus in new tab">Scopus (3916)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20196867/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fgb-2010-11-3-r25&amp;pmid=20196867" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib116" class="citations"><div id="sref116" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref116" title="View in article">104.</a></div><div>Robinson, M.D. ∙ Smyth, G.K.</div><div><strong>Moderated statistical tests for assessing differences in tag abundance</strong></div><div><em>Bioinformatics.</em> 2007; <strong>23</strong>:2881-2887</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btm453" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_104_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-36448981743" target="_blank" aria-label="Open Scopus in new tab">Scopus (554)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17881408/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtm453&amp;pmid=17881408" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib117" class="citations"><div id="sref117" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref117" title="View in article">105.</a></div><div>Robinson, M.D. ∙ Smyth, G.K.</div><div><strong>Small-sample estimation of negative binomial dispersion, with applications to SAGE data</strong></div><div><em>Biostatistics.</em> 2008; <strong>9</strong>:321-332</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/biostatistics/kxm030" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_105_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-41149085992" target="_blank" aria-label="Open Scopus in new tab">Scopus (673)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17728317/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbiostatistics%2Fkxm030&amp;pmid=17728317" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib118" class="citations"><div id="sref118" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref118-1" title="View in article">106.</a></div><div>Robinson, M.D. ∙ McCarthy, D.J. ∙ Smyth, G.K.</div><div><strong>edgeR: a Bioconductor package for differential expression analysis of digital gene expression data</strong></div><div><em>Bioinformatics.</em> 2010; <strong>26</strong>:139-140</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btp616" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_106_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-75249087100" target="_blank" aria-label="Open Scopus in new tab">Scopus (19590)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19910308/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtp616&amp;pmid=19910308" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib119" class="citations"><div id="sref119" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref119" title="View in article">107.</a></div><div>Samocha, K.E. ∙ Robinson, E.B. ∙ Sanders, S.J. ...</div><div><strong>A framework for the interpretation of de novo mutation in human disease</strong></div><div><em>Nat. Genet.</em> 2014; <strong>46</strong>:944-950</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng.3050" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_107_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-84905309176" target="_blank" aria-label="Open Scopus in new tab">Scopus (591)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25086666/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng.3050&amp;pmid=25086666" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib120" class="citations"><div id="sref120" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref120" title="View in article">108.</a></div><div>Sastry, G.M. ∙ Adzhigirey, M. ∙ Day, T. ...</div><div><strong>Protein and ligand preparation: parameters, protocols, and influence on virtual screening enrichments</strong></div><div><em>J.&nbsp;Comput. Aided Mol. Des.</em> 2013; <strong>27</strong>:221-234</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/s10822-013-9644-8" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_108_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-84880529288" target="_blank" aria-label="Open Scopus in new tab">Scopus (2784)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23579614/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2Fs10822-013-9644-8&amp;pmid=23579614" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib121" class="citations"><div id="sref121" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref121-1" title="View in article">109.</a></div><div>Sevim Bayrak, C. ∙ Zhang, P. ∙ Tristani-Firouzi, M. ...</div><div><strong>De novo variants in exomes of congenital heart disease patients identify risk genes and pathways</strong></div><div><em>Genome Med.</em> 2020; <strong>12</strong>:9</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13073-019-0709-8" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_109_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-85077940747" target="_blank" aria-label="Open Scopus in new tab">Scopus (25)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31941532/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13073-019-0709-8&amp;pmid=31941532" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib122" class="citations"><div id="sref122" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref122-1" title="View in article">110.</a></div><div>Shannon, P. ∙ Markiel, A. ∙ Ozier, O. ...</div><div><strong>Cytoscape: a software environment for integrated models of biomolecular interaction networks</strong></div><div><em>Genome Res.</em> 2003; <strong>13</strong>:2498-2504</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/gr.1239303" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_110_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-0242490780" target="_blank" aria-label="Open Scopus in new tab">Scopus (23829)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/14597658/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2Fgr.1239303&amp;pmid=14597658" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib123" class="citations"><div id="sref123" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref123" title="View in article">111.</a></div><div>Sifrim, A. ∙ Hitz, M.-P. ∙ Wilsdon, A. ..., INTERVAL Study, UK10K Consortium, Deciphering Developmental Disorders Study</div><div><strong>Distinct genetic architectures for syndromic and nonsyndromic congenital heart defects identified by exome sequencing</strong></div><div><em>Nat. Genet.</em> 2016; <strong>48</strong>:1060-1065</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng.3627" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_111_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-84980370998" target="_blank" aria-label="Open Scopus in new tab">Scopus (215)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27479907/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng.3627&amp;pmid=27479907" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib124" class="citations"><div id="sref124" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref124" title="View in article">112.</a></div><div>Smyth, G.K.</div><div><strong>A conditional approach to residual maximum likelihood estimation in generalized linear models</strong></div><div>R. Stat. Soc. B., 1996</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1111/j.2517-6161.1996.tb02101.x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1111%2Fj.2517-6161.1996.tb02101.x" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib125" class="citations"><div id="sref125" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref125" title="View in article">113.</a></div><div>Stark, Z. ∙ Dashnow, H. ∙ Lunke, S. ..., Melbourne Genomics Health Alliance</div><div><strong>A clinically driven variant prioritization framework outperforms purely computational approaches for the diagnostic analysis of singleton WES data</strong></div><div><em>Eur. J. Hum. Genet.</em> 2017; <strong>25</strong>:1268-1272</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ejhg.2017.123" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_113_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-85031307052" target="_blank" aria-label="Open Scopus in new tab">Scopus (7)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28832562/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fejhg.2017.123&amp;pmid=28832562" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib126" class="citations"><div id="sref126" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref126" title="View in article">114.</a></div><div>Szklarczyk, D. ∙ Gable, A.L. ∙ Lyon, D. ...</div><div><strong>STRING v11: protein-protein association networks with increased coverage, supporting functional discovery in genome-wide experimental datasets</strong></div><div><em>Nucleic Acids Res.</em> 2019; <strong>47</strong>:D607-D613</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gky1131" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_114_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-85059797047" target="_blank" aria-label="Open Scopus in new tab">Scopus (6765)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30476243/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgky1131&amp;pmid=30476243" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib128" class="citations"><div id="sref128" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref128-1" title="View in article">115.</a></div><div>Teo, G. ∙ Koh, H. ∙ Fermin, D. ...</div><div><strong>SAINTq: Scoring protein-protein interactions in affinity purification - mass spectrometry experiments with fragment or peptide intensity data</strong></div><div><em>Proteomics.</em> 2016; <strong>16</strong>:2238-2245</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/pmic.201500499" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_115_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-84983261035" target="_blank" aria-label="Open Scopus in new tab">Scopus (24)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27119218/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fpmic.201500499&amp;pmid=27119218" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib129" class="citations"><div id="sref129" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref129" title="View in article">116.</a></div><div>Theis, J.L. ∙ Vogler, G. ∙ Missinato, M.A. ...</div><div><strong>Patient-specific genomics and cross-species functional alalysis implicate LRP2 in hypoplastic left heart syndrome</strong></div><div><em>eLife.</em> 2020; <strong>9</strong>:359554</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.7554/eLife.59554" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_116_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-85094219857" target="_blank" aria-label="Open Scopus in new tab">Scopus (10)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.7554%2FeLife.59554" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib130" class="citations"><div id="sref130" class="citation"><div class="citation-content"><div class="label"><a href="#figs3" title="View in article">117.</a></div><div>Thienpont, B. ∙ Zhang, L. ∙ Postma, A.V. ...</div><div><strong>Haploinsufficiency of TAB2 causes congenital heart defects in humans</strong></div><div><em>Am. J. Hum. Genet.</em> 2010; <strong>86</strong>:839-849</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_117_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=10.1016%2Fj.ajhg.2010.04.011&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_117_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=10.1016%2Fj.ajhg.2010.04.011&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_117_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-77953231502" target="_blank" aria-label="Open Scopus in new tab">Scopus (75)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20493459/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.ajhg.2010.04.011&amp;pmid=20493459" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib131" class="citations"><div id="sref131" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref131" title="View in article">118.</a></div><div>Tickle, J. ∙ Pilka, E.S. ∙ Bunkoczi, G. ...</div><div><strong>Structure of the cytokine-like nuclear factor n-pac</strong></div><div><a id="intref0270" href="https://www.wwpdb.org/pdb?id=pdb_00002uyy" target="_blank">https://www.wwpdb.org/pdb?id=pdb_00002uyy</a><div>Date: 2007</div></div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.TickleE.S.PilkaG.BunkocziG.BerridgeC.SmeeK.L.KavanaghV.HozjanF.H.NiesenE.PapagrigoriouA.C.W.PikeStructure+of+the+cytokine-like+nuclear+factor+n-pachttps%3A%2F%2Fwww.wwpdb.org%2Fpdb%3Fid%3Dpdb_00002uyy2007" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib132" class="citations"><div id="sref132" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref132" title="View in article">119.</a></div><div>Tohyama, S. ∙ Hattori, F. ∙ Sano, M. ...</div><div><strong>Distinct metabolic flow enables large-scale purification of mouse and human pluripotent stem cell-derived cardiomyocytes</strong></div><div><em>Cell Stem Cell.</em> 2013; <strong>12</strong>:127-137</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_119_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=10.1016%2Fj.stem.2012.09.013&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_119_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=10.1016%2Fj.stem.2012.09.013&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_119_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-84872048038" target="_blank" aria-label="Open Scopus in new tab">Scopus (591)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23168164/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.stem.2012.09.013&amp;pmid=23168164" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib133" class="citations"><div id="sref133" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref133" title="View in article">120.</a></div><div>Tomita-Mitchell, A. ∙ Maslen, C.L. ∙ Morris, C.D. ...</div><div><strong>GATA4 sequence variants in patients with congenital heart disease</strong></div><div><em>J.&nbsp;Med. Genet.</em> 2007; <strong>44</strong>:779-783</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1136/jmg.2007.052183" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_120_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-37249090635" target="_blank" aria-label="Open Scopus in new tab">Scopus (165)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/18055909/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1136%2Fjmg.2007.052183&amp;pmid=18055909" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib134" class="citations"><div id="sref134" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref134-1" title="View in article">121.</a></div><div>Uhlén, M. ∙ Fagerberg, L. ∙ Hallström, B.M. ...</div><div><strong>Proteomics. Tissue-based map of the human proteome</strong></div><div><em>Science.</em> 2015; <strong>347</strong>:1260419</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.1260419" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25613900/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.1260419&amp;pmid=25613900" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib136" class="citations"><div id="sref136" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref136-1" title="View in article">122.</a></div><div>Waldron, L. ∙ Steimle, J.D. ∙ Greco, T.M. ...</div><div><strong>The cardiac TBX5 interactome reveals a chromatin remodeling network essential for cardiac septation</strong></div><div><em>Dev. Cell.</em> 2016; <strong>36</strong>:262-275</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_122_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=10.1016%2Fj.devcel.2016.01.009&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_122_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=10.1016%2Fj.devcel.2016.01.009&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_122_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-84958661999" target="_blank" aria-label="Open Scopus in new tab">Scopus (56)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26859351/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.devcel.2016.01.009&amp;pmid=26859351" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib147" class="citations"><div id="optG9Ymi7TVGH" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-optG9Ymi7TVGH" title="View in article">123.</a></div><div>Wang, K. ∙ Li, M. ∙ Hakonarson, H.</div><div><strong>ANNOVAR: functional annotation of genetic variants from high-throughput sequencing data</strong></div><div><em>Nucleic Acids Res.</em> 2010; <strong>38</strong>:e164</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gkq603" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_123_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-77956534324" target="_blank" aria-label="Open Scopus in new tab">Scopus (7498)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20601685/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgkq603&amp;pmid=20601685" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib137" class="citations"><div id="sref137" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref137" title="View in article">124.</a></div><div>Wang, L. ∙ Wang, S. ∙ Li, W.</div><div><strong>RSeQC: quality control of RNA-seq experiments</strong></div><div><em>Bioinformatics.</em> 2012; <strong>28</strong>:2184-2185</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/bts356" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_124_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-84865074382" target="_blank" aria-label="Open Scopus in new tab">Scopus (1107)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22743226/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbts356&amp;pmid=22743226" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib138" class="citations"><div id="sref138" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref138" title="View in article">125.</a></div><div>Watt, A.J. ∙ Battle, M.A. ∙ Li, J. ...</div><div><strong>GATA4 is essential for formation of the proepicardium and regulates cardiogenesis</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2004; <strong>101</strong>:12573-12578</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.0400752101" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_125_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-4344565788" target="_blank" aria-label="Open Scopus in new tab">Scopus (222)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15310850/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.0400752101&amp;pmid=15310850" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib139" class="citations"><div id="sref139" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref139-1" title="View in article">126.</a></div><div>Wei, Q. ∙ Zhan, X. ∙ Zhong, X. ...</div><div><strong>A Bayesian framework for de novo mutation calling in parents-offspring trios</strong></div><div><em>Bioinformatics.</em> 2015; <strong>31</strong>:1375-1381</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btu839" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25535243/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtu839&amp;pmid=25535243" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib141" class="citations"><div id="sref141" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref141" title="View in article">127.</a></div><div>Xin, M. ∙ Davis, C.A. ∙ Molkentin, J.D. ...</div><div><strong>A threshold of GATA4 and GATA6 expression is required for cardiovascular development</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2006; <strong>103</strong>:11189-11194</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.0604604103" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_127_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-33746630840" target="_blank" aria-label="Open Scopus in new tab">Scopus (141)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16847256/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.0604604103&amp;pmid=16847256" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib142" class="citations"><div id="sref142" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref142-1" title="View in article">128.</a></div><div>Xing, H. ∙ Mo, Y. ∙ Liao, W. ...</div><div><strong>Genome-wide localization of protein-DNA binding and histone modification by a Bayesian change-point method with ChIP-seq data</strong></div><div><em>PLoS Comput. Biol.</em> 2012; <strong>8</strong>:e1002613</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pcbi.1002613" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_128_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-84864587283" target="_blank" aria-label="Open Scopus in new tab">Scopus (40)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22844240/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pcbi.1002613&amp;pmid=22844240" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib143" class="citations"><div id="sref143" class="citation"><div class="citation-content"><div class="label"><a href="#figs4" title="View in article">129.</a></div><div>Yu, S. ∙ Li, J. ∙ Ji, G. ...</div><div><strong>Npac Is a Co-factor of Histone H3K36me3 and Regulates Transcriptional Elongation in Mouse Embryonic Stem Cells</strong></div><div><em>Genomics Proteomics Bioinformatics.</em> 2021; <strong>3</strong></div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.gpb.2020.08.004" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_129_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-85118281452" target="_blank" aria-label="Open Scopus in new tab">Scopus (2)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.gpb.2020.08.004" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib144" class="citations"><div id="sref144" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref144-1" title="View in article">130.</a></div><div>Zaidi, S. ∙ Brueckner, M.</div><div><strong>Genetics and genomics of congenital heart disease</strong></div><div><em>Circ. Res.</em> 2017; <strong>120</strong>:923-940</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1161/CIRCRESAHA.116.309140" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_130_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-85015361244" target="_blank" aria-label="Open Scopus in new tab">Scopus (212)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28302740/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1161%2FCIRCRESAHA.116.309140&amp;pmid=28302740" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib145" class="citations"><div id="sref145" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref145-1" title="View in article">131.</a></div><div>Zaidi, S. ∙ Choi, M. ∙ Wakimoto, H. ...</div><div><strong>De novo mutations in histone-modifying genes in congenital heart disease</strong></div><div><em>Nature.</em> 2013; <strong>498</strong>:220-223</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature12141" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_131_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-84879001958" target="_blank" aria-label="Open Scopus in new tab">Scopus (616)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23665959/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature12141&amp;pmid=23665959" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib146" class="citations"><div id="sref146" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref146" title="View in article">132.</a></div><div>Zhang, Y. ∙ Zheng, Y. ∙ Qin, L. ...</div><div><strong>Structural characterization of a β-hydroxyacid dehydrogenase from Geobacter sulfurreducens and Geobacter metallireducens with succinic semialdehyde reductase activity</strong></div><div><em>Biochimie.</em> 2014; <strong>104</strong>:61-69</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.biochi.2014.05.002" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_132_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.01.021&amp;key=2-s2.0-84905507214" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24878278/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.biochi.2014.05.002&amp;pmid=24878278" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div></div></div></section></div></section><div class="core-linked-content">
    

        


    

            

            

            <div data-widget-id="8633d695-b258-4908-b86f-e2c268ec3c6b" data-widget-name="UX3HTMLWidget"><script type="text/javascript">
if($('.content-navigation__body__title').length){
$('.content-navigation__body__title').each(function(){
var html=this.innerHTML;
var div = document.createElement("div");
div.innerHTML = html;
var text = div.textContent || div.innerText || "";
this.innerHTML=text;
});
}
</script></div>

            

        

    

        


    

            

            

            <div data-widget-id="393effff-5b44-4681-a9b4-4f029b3540b7" data-widget-name="UX3HTMLWidget"><style>
.linkback-ad .pb-ad {
    display: block;
   height: auto;
    margin-left: 0px;
    margin-right: auto;
}
</style></div>

            

        

    

        

</div></div></div><div class="core-collateral-aside"><section id="core-collateral-figures"><h2>Figures (14)<button class="js--openFigureViewer"><span>Figure Viewer</span></button></h2><div class="core-collateral-aside-wrapper"><div class="grid"><a class="figure__open__ctrl" href="#fig1" aria-label="Go to Figure&amp;nbsp;1"><img src="https://www.cell.com/cms/10.1016/j.cell.2022.01.021/asset/681c6e80-acbe-4d42-ae97-8376ffc8cdf3/main.assets/gr1.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs1" aria-label="Go to Figure&amp;nbsp;S1"><img src="https://www.cell.com/cms/10.1016/j.cell.2022.01.021/asset/4fe8939e-df96-46ac-b4ed-d03ed368cb07/main.assets/figs1.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs2" aria-label="Go to Figure&amp;nbsp;S2"><img src="https://www.cell.com/cms/10.1016/j.cell.2022.01.021/asset/24acb0d0-9390-4050-be1b-1de14a473e55/main.assets/figs2.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig2" aria-label="Go to Figure&amp;nbsp;2"><img src="https://www.cell.com/cms/10.1016/j.cell.2022.01.021/asset/25214844-0c30-4134-bed6-69da20f060e5/main.assets/gr2.jpg" aria-hidden="true"></a></div><div id="collapsible-figures" data-expandable="item"><div class="content grid" style="display:none;" role="region" id="collapsible-figures_content"><a class="figure__open__ctrl" href="#figs3" aria-label="Go to Figure&amp;nbsp;S3"><img src="https://www.cell.com/cms/10.1016/j.cell.2022.01.021/asset/3f397664-29f3-4b2c-9eb4-ebd5a92519bf/main.assets/figs3.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig3" aria-label="Go to Figure&amp;nbsp;3"><img src="https://www.cell.com/cms/10.1016/j.cell.2022.01.021/asset/20bea442-66aa-4897-a122-7420118503ba/main.assets/gr3.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig4" aria-label="Go to Figure&amp;nbsp;4"><img src="https://www.cell.com/cms/10.1016/j.cell.2022.01.021/asset/0f48180d-f39d-413b-9dd6-33d0ed8c54d5/main.assets/gr4.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs4" aria-label="Go to Figure&amp;nbsp;S4"><img src="https://www.cell.com/cms/10.1016/j.cell.2022.01.021/asset/f68cd285-332a-4a21-8330-7245923cb700/main.assets/figs4.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig5" aria-label="Go to Figure&amp;nbsp;5"><img src="https://www.cell.com/cms/10.1016/j.cell.2022.01.021/asset/f4ab8d64-bad4-45c6-bffd-e1918f0cc8b8/main.assets/gr5.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig6" aria-label="Go to Figure&amp;nbsp;6"><img src="https://www.cell.com/cms/10.1016/j.cell.2022.01.021/asset/6bcddf09-7630-4aa7-b219-9505156bcc7f/main.assets/gr6.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs5" aria-label="Go to Figure&amp;nbsp;S5"><img src="https://www.cell.com/cms/10.1016/j.cell.2022.01.021/asset/6bbed26a-105a-4539-b8f8-b1051541272e/main.assets/figs5.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs6" aria-label="Go to Figure&amp;nbsp;S6"><img src="https://www.cell.com/cms/10.1016/j.cell.2022.01.021/asset/65dd2a63-1059-453b-87f6-fa9cd51148bb/main.assets/figs6.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig7" aria-label="Go to Figure&amp;nbsp;7"><img src="https://www.cell.com/cms/10.1016/j.cell.2022.01.021/asset/d48839aa-b97d-4a39-9793-197954f3b6dc/main.assets/gr7.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs7" aria-label="Go to Figure&amp;nbsp;S7"><img src="https://www.cell.com/cms/10.1016/j.cell.2022.01.021/asset/40aa3d08-29bb-48cc-95b9-96c9c6240685/main.assets/figs7.jpg" aria-hidden="true"></a></div><button class="trigger" aria-controls="collapsible-figures_content" aria-expanded="false" tabindex="0" role="button"><span class="expand">Show all figures</span><span class="collapse">Hide figures</span><i class="icon-section_arrow_d" aria-hidden="true"></i></button></div></div></section>
    

        


    

            

            

            <section id="core-collateral-metrics" class="publication-list__axel-section"><h2>Article metrics</h2><div><ul class="publication-list rlist publication-list--list plum-x--lazy-load publication-list--axel" data-plum-x-url="/pb/widgets/ux3/publicationList/plumx?widgetId=66dd7e11-7768-4006-93d8-a72b60e2a264&amp;pbContext=%3Bproduct%3Aproduct%3Aelsevier%5C%3Aproduct%5C%3Acell%3Bwebsite%3Awebsite%3Acell-site%3Bctype%3Astring%3AJournal+Content%3Barticle%3Aarticle%3Apii%5C%3AS0092867422000794%3Bissue%3Aissue%3Apii%5C%3AS0092867421X00062%3Bjournal%3Ajournal%3Acell%3BpageGroup%3Astring%3AArticle+View%3Bpage%3Astring%3AShow+Full+Text%3BrequestedJournal%3Ajournal%3Acell%3Bwgroup%3Astring%3ADefault+Website+Group%3BsubPage%3Astring%3AFull+Text" data-pb-mode-enabled="false"><li aria-label="loading articles" role="alert" class="js--loading plum-x--spinner hidden"></li></ul></div></section>

            

        
<section id="core-collateral-supplementary"><h2>Supplemental information (1)</h2><div class="core-collateral-aside-wrapper"><div class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2022.01.021/attachment/a2e3c901-53ea-49f3-b6e3-5f6389b4e3f4/mmc1.xlsx" download="mmc1.xlsx"><span id="mmc1-link">Spreadsheet (27.86 MB)</span></a><div class="core-description">Table S1. Compendium of all supplemental tables, related to Figures 1, 2, 3, 4, 5, 6, 7, S1, S2, S3, S4, S5, S6, and S7</div></div></div></div></section>
    

        


    

            

            

            <section id="core-collateral-relatedArticles" class="axel-related-articles"><h2 id="relatedArticles" class="top axel-related-articles__title" data-title="Related articles">Related Articles</h2><div data-pb-widget-type="ux3-relatedArticles" data-widget-id="8d3844d1-450e-497c-ab75-a63fc41750ac" data-widget-name="AtyponTempUX3RelatedArticlesWidget" data-article-pii="S0092867422000794" class="ux3DeferredOnScrollWidget axel-related-articles__body" data-show-more="true"></div></section>

            

        
</div></div><aside data-core-aside="right-rail"><div class="inner-wrapper">

    

        


    

            

            

            <div data-widget-id="dca7470d-9085-4d21-a6f4-a66397718146" data-widget-name="UX3HTMLWidget"><input type="hidden" id="gptPage" value="cell-videos"></div>

            

        











        <div data-widget-def="literatumAd" data-widget-id="481d10c2-bbe0-470e-a5ba-207a81ecc8dc" class="adplaceholders" style="right: 0px; position: static;">
        
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd alignCenter  widget-verticalAd" id="481d10c2-bbe0-470e-a5ba-207a81ecc8dc">
                
                
                    
                

                <div class="wrapped " id="Marketplace-Recs">
                    
                    <div class="widget-body body body-verticalAd "><div class="verticalAdContainer">
    <div class="verticalAdLabelContainer">
        <div class="verticalAdLabel">
                <img src="/templates/jsp/_style2/_marlin/images/bg_adVert.gif" alt="Advertisement">
        </div>
    </div>
    <div class="verticalAd">
        





        <div class="pb-ad">
            <script src="//cdn.pbgrd.com/core-cell.js" async=""></script>
<div id="pbgrd-skyr-c"><div id="pbgrd-skyr" style="width: inherit; position: relative; background: transparent;"></div></div>
        </div>
    

    </div>
</div></div>
                </div>
                </div>

        

        </div>
    

</div></aside><div class="core-authors-details" style="display:none"><section class="core-authors"></section></div></article><div data-extent="article-wrapper"><div class="core-container"></div></div><script id="axel-publication-metadata" type="application/json">{"doi":"10.1016/j.cell.2022.01.021","data-is-retraction":false,"data-is-retired":false}
</script><script type="text/javascript" data-ot-ignore="data-ot-ignore" defer="defer" src="https://static.addtoany.com/menu/page.js" class="optanon-category-C0004 ot-vscat-C0004"></script><template id="figure_nav_template"><a href="#" class="open-in-viewer btn btn-dark figure-pop-btn"><i aria-hidden="true" class="icon-export_citation"></i><span>Open in viewer</span></a></template><div id="figure-viewer" role="dialog" aria-modal="true" aria-label="Figure Viewer" class="showCaption showThumbstrip figure-viewer"><nav class="nav-bar figure-viewer__top"><a href="#" title="Close Figure Viewer" role="button" class="figure-viewer__close"><i aria-hidden="true" class="icon-tools_close"></i></a><div class="figure-viewer__title">Transcription factor protein interactomes reveal genetic determinants in heart disease</div></nav><div class="frames"><ul class="frames__list"><li><div class="thumbnail__image"><img id="gr1" src="/cms/10.1016/j.cell.2022.01.021/asset/681c6e80-acbe-4d42-ae97-8376ffc8cdf3/main.assets/gr1.jpg" height="858" width="659" aria-labelledby="fig1-title fig1-content" data-viewer-src="/cms/10.1016/j.cell.2022.01.021/asset/79910616-b6cf-4d6e-b3f0-222d933b217c/main.assets/gr1_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs1" src="/cms/10.1016/j.cell.2022.01.021/asset/4fe8939e-df96-46ac-b4ed-d03ed368cb07/main.assets/figs1.jpg" height="750" width="497" aria-labelledby="figs1-title figs1-content" data-viewer-src="/cms/10.1016/j.cell.2022.01.021/asset/9ba95259-d77d-4f24-af9d-ae63bc64de7d/main.assets/figs1_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs2" src="/cms/10.1016/j.cell.2022.01.021/asset/24acb0d0-9390-4050-be1b-1de14a473e55/main.assets/figs2.jpg" height="938" width="603" aria-labelledby="figs2-title figs2-content" data-viewer-src="/cms/10.1016/j.cell.2022.01.021/asset/f4724dba-fa6d-403f-9381-95a42dbeac04/main.assets/figs2_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr2" src="/cms/10.1016/j.cell.2022.01.021/asset/25214844-0c30-4134-bed6-69da20f060e5/main.assets/gr2.jpg" height="799" width="497" aria-labelledby="fig2-title fig2-content" data-viewer-src="/cms/10.1016/j.cell.2022.01.021/asset/3b704776-a9be-4b5f-9794-ebc6525e7603/main.assets/gr2_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs3" src="/cms/10.1016/j.cell.2022.01.021/asset/3f397664-29f3-4b2c-9eb4-ebd5a92519bf/main.assets/figs3.jpg" height="939" width="748" aria-labelledby="figs3-title figs3-content" data-viewer-src="/cms/10.1016/j.cell.2022.01.021/asset/255aedf0-9e92-4e01-9c5f-5c99083f4f28/main.assets/figs3_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr3" src="/cms/10.1016/j.cell.2022.01.021/asset/20bea442-66aa-4897-a122-7420118503ba/main.assets/gr3.jpg" height="760" width="656" aria-labelledby="fig3-title fig3-content" data-viewer-src="/cms/10.1016/j.cell.2022.01.021/asset/6e326926-f3cc-414b-bf98-b05d902e43af/main.assets/gr3_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr4" src="/cms/10.1016/j.cell.2022.01.021/asset/0f48180d-f39d-413b-9dd6-33d0ed8c54d5/main.assets/gr4.jpg" height="504" width="656" aria-labelledby="fig4-title fig4-content" data-viewer-src="/cms/10.1016/j.cell.2022.01.021/asset/3dc15568-216c-484e-9940-2322dcbe46ef/main.assets/gr4_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs4" src="/cms/10.1016/j.cell.2022.01.021/asset/f68cd285-332a-4a21-8330-7245923cb700/main.assets/figs4.jpg" height="872" width="656" aria-labelledby="figs4-title figs4-content" data-viewer-src="/cms/10.1016/j.cell.2022.01.021/asset/2bcda251-c61a-4435-924f-5f90b0205bfa/main.assets/figs4_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr5" src="/cms/10.1016/j.cell.2022.01.021/asset/f4ab8d64-bad4-45c6-bffd-e1918f0cc8b8/main.assets/gr5.jpg" height="711" width="659" aria-labelledby="fig5-title fig5-content" data-viewer-src="/cms/10.1016/j.cell.2022.01.021/asset/12063954-16ad-467f-bb5d-6e9c8e169f48/main.assets/gr5_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr6" src="/cms/10.1016/j.cell.2022.01.021/asset/6bcddf09-7630-4aa7-b219-9505156bcc7f/main.assets/gr6.jpg" height="825" width="656" aria-labelledby="fig6-title fig6-content" data-viewer-src="/cms/10.1016/j.cell.2022.01.021/asset/707cff8f-db79-41b5-97cf-d2d599dee5f1/main.assets/gr6_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs5" src="/cms/10.1016/j.cell.2022.01.021/asset/6bbed26a-105a-4539-b8f8-b1051541272e/main.assets/figs5.jpg" height="832" width="656" aria-labelledby="figs5-title figs5-content" data-viewer-src="/cms/10.1016/j.cell.2022.01.021/asset/643a7477-dbcd-4afa-9888-484617ced8da/main.assets/figs5_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs6" src="/cms/10.1016/j.cell.2022.01.021/asset/65dd2a63-1059-453b-87f6-fa9cd51148bb/main.assets/figs6.jpg" height="761" width="656" aria-labelledby="figs6-title figs6-content" data-viewer-src="/cms/10.1016/j.cell.2022.01.021/asset/46fb3b43-59d9-4e21-9d14-c8ce6397599d/main.assets/figs6_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr7" src="/cms/10.1016/j.cell.2022.01.021/asset/d48839aa-b97d-4a39-9793-197954f3b6dc/main.assets/gr7.jpg" height="935" width="656" aria-labelledby="fig7-title fig7-content" data-viewer-src="/cms/10.1016/j.cell.2022.01.021/asset/13ad87ad-364c-45ce-9258-cc9b3160629e/main.assets/gr7_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs7" src="/cms/10.1016/j.cell.2022.01.021/asset/40aa3d08-29bb-48cc-95b9-96c9c6240685/main.assets/figs7.jpg" height="903" width="656" aria-labelledby="figs7-title figs7-content" data-viewer-src="/cms/10.1016/j.cell.2022.01.021/asset/fd9aee73-2c6b-4cf6-ae04-920e1d3957a0/main.assets/figs7_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li></ul><a href="#" title="Previous Frame" class="frameButton previousFrameButton frames__prev"><!--span Previous Frame--><i class="icon-arrow_l"></i></a><a href="#" title="Next Frame" class="frameButton nextFrameButton frames__next"><!--span Next Frame--><i class="icon-arrow_r"></i></a></div><div class="caption"><div class="figure-viewer__caption-body"></div></div><div class="toolbar figure-viewer__toolbar"><span class="figure-viewer__counter"></span><a href="#" class="button toggle-caption-button figure-viewer__caption__ctrl"><i aria-hidden="true" class="icon icon-info"></i><span class="label">Hide Caption</span></a><a href="#" class="button download-options-button"><i aria-hidden="true" class="icon icon-gizmo-folder-in download-size-icon"></i><span class="label">Download</span></a><a href="#figures" class="button see-figures figure-viewer__see-figure__ctrl"><i aria-hidden="true" class="icon icon-gizmo-image_figure viewer-size-icon"></i><span class="label">See figure in Article</span></a></div><div class="thumbstrip"><a href="#" class="toggleButton thumbstrip__ctrl"><span>Toggle Thumbstrip</span></a><a href="#" title="Previous Thumbnails" class="thumbstripButton previousThumbstripButton thumbstrip__prev"><i aria-hidden="true" class="icon-arrow_l"></i></a><a href="#" title="Next Thumbnails" class="thumbstripButton nextThumbstripButton thumbstrip__next"><i aria-hidden="true" class="icon-arrow_r"></i></a><div class="thumbstrip__wrapper"><ul class="rlist figure-viewer__thumbslist"><li><div class="thumbnail__label">Figure&nbsp;1</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr1" src="/cms/10.1016/j.cell.2022.01.021/asset/681c6e80-acbe-4d42-ae97-8376ffc8cdf3/main.assets/gr1.jpg" height="858" width="659" aria-labelledby="fig1-title fig1-content" data-viewer-src="/cms/10.1016/j.cell.2022.01.021/asset/79910616-b6cf-4d6e-b3f0-222d933b217c/main.assets/gr1_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S1</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs1" src="/cms/10.1016/j.cell.2022.01.021/asset/4fe8939e-df96-46ac-b4ed-d03ed368cb07/main.assets/figs1.jpg" height="750" width="497" aria-labelledby="figs1-title figs1-content" data-viewer-src="/cms/10.1016/j.cell.2022.01.021/asset/9ba95259-d77d-4f24-af9d-ae63bc64de7d/main.assets/figs1_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S2</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs2" src="/cms/10.1016/j.cell.2022.01.021/asset/24acb0d0-9390-4050-be1b-1de14a473e55/main.assets/figs2.jpg" height="938" width="603" aria-labelledby="figs2-title figs2-content" data-viewer-src="/cms/10.1016/j.cell.2022.01.021/asset/f4724dba-fa6d-403f-9381-95a42dbeac04/main.assets/figs2_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;2</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr2" src="/cms/10.1016/j.cell.2022.01.021/asset/25214844-0c30-4134-bed6-69da20f060e5/main.assets/gr2.jpg" height="799" width="497" aria-labelledby="fig2-title fig2-content" data-viewer-src="/cms/10.1016/j.cell.2022.01.021/asset/3b704776-a9be-4b5f-9794-ebc6525e7603/main.assets/gr2_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S3</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs3" src="/cms/10.1016/j.cell.2022.01.021/asset/3f397664-29f3-4b2c-9eb4-ebd5a92519bf/main.assets/figs3.jpg" height="939" width="748" aria-labelledby="figs3-title figs3-content" data-viewer-src="/cms/10.1016/j.cell.2022.01.021/asset/255aedf0-9e92-4e01-9c5f-5c99083f4f28/main.assets/figs3_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;3</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr3" src="/cms/10.1016/j.cell.2022.01.021/asset/20bea442-66aa-4897-a122-7420118503ba/main.assets/gr3.jpg" height="760" width="656" aria-labelledby="fig3-title fig3-content" data-viewer-src="/cms/10.1016/j.cell.2022.01.021/asset/6e326926-f3cc-414b-bf98-b05d902e43af/main.assets/gr3_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;4</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr4" src="/cms/10.1016/j.cell.2022.01.021/asset/0f48180d-f39d-413b-9dd6-33d0ed8c54d5/main.assets/gr4.jpg" height="504" width="656" aria-labelledby="fig4-title fig4-content" data-viewer-src="/cms/10.1016/j.cell.2022.01.021/asset/3dc15568-216c-484e-9940-2322dcbe46ef/main.assets/gr4_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S4</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs4" src="/cms/10.1016/j.cell.2022.01.021/asset/f68cd285-332a-4a21-8330-7245923cb700/main.assets/figs4.jpg" height="872" width="656" aria-labelledby="figs4-title figs4-content" data-viewer-src="/cms/10.1016/j.cell.2022.01.021/asset/2bcda251-c61a-4435-924f-5f90b0205bfa/main.assets/figs4_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;5</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr5" src="/cms/10.1016/j.cell.2022.01.021/asset/f4ab8d64-bad4-45c6-bffd-e1918f0cc8b8/main.assets/gr5.jpg" height="711" width="659" aria-labelledby="fig5-title fig5-content" data-viewer-src="/cms/10.1016/j.cell.2022.01.021/asset/12063954-16ad-467f-bb5d-6e9c8e169f48/main.assets/gr5_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;6</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr6" src="/cms/10.1016/j.cell.2022.01.021/asset/6bcddf09-7630-4aa7-b219-9505156bcc7f/main.assets/gr6.jpg" height="825" width="656" aria-labelledby="fig6-title fig6-content" data-viewer-src="/cms/10.1016/j.cell.2022.01.021/asset/707cff8f-db79-41b5-97cf-d2d599dee5f1/main.assets/gr6_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S5</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs5" src="/cms/10.1016/j.cell.2022.01.021/asset/6bbed26a-105a-4539-b8f8-b1051541272e/main.assets/figs5.jpg" height="832" width="656" aria-labelledby="figs5-title figs5-content" data-viewer-src="/cms/10.1016/j.cell.2022.01.021/asset/643a7477-dbcd-4afa-9888-484617ced8da/main.assets/figs5_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S6</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs6" src="/cms/10.1016/j.cell.2022.01.021/asset/65dd2a63-1059-453b-87f6-fa9cd51148bb/main.assets/figs6.jpg" height="761" width="656" aria-labelledby="figs6-title figs6-content" data-viewer-src="/cms/10.1016/j.cell.2022.01.021/asset/46fb3b43-59d9-4e21-9d14-c8ce6397599d/main.assets/figs6_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;7</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr7" src="/cms/10.1016/j.cell.2022.01.021/asset/d48839aa-b97d-4a39-9793-197954f3b6dc/main.assets/gr7.jpg" height="935" width="656" aria-labelledby="fig7-title fig7-content" data-viewer-src="/cms/10.1016/j.cell.2022.01.021/asset/13ad87ad-364c-45ce-9258-cc9b3160629e/main.assets/gr7_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S7</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs7" src="/cms/10.1016/j.cell.2022.01.021/asset/40aa3d08-29bb-48cc-95b9-96c9c6240685/main.assets/figs7.jpg" height="903" width="656" aria-labelledby="figs7-title figs7-content" data-viewer-src="/cms/10.1016/j.cell.2022.01.021/asset/fd9aee73-2c6b-4cf6-ae04-920e1d3957a0/main.assets/figs7_lrg.jpg" loading="lazy" tabindex="0"></div></div></li></ul></div></div><div class="download figure-viewer__download"><div class="pointer"></div><ul class="rlist downlad__wrapper"><li class="hidden-xs hidden-sm"><a href="/action/downloadFigures?pii=e00489&amp;id=" class="downloadAllButton"><i class="icon icon-gizmo-ppt figure-viewer__download__viewer-size-icon figure-viewer__download__viewer-size-icon--ppt"></i><span class="figure-viewer__download__text"> Download .PPT</span></a></li></ul></div></div><template id="toCitationLink" data-citation="Citation" data-footnote="Footnote"><div class="to-citation__wrapper"><a class="to-citation"><i aria-hidden="true" class="icon-subdirectory_arrow_left"></i><span>Go to</span></a></div></template><template id="toCitationButton" data-citation="Citation" data-footnote="Footnote"><div class="to-citation__wrapper"><button class="to-citation"><i aria-hidden="true" class="icon-subdirectory_arrow_left"></i><span>Go to</span></button></div></template><template id="toCitationAccordion" data-collapse-citation="Hide Citations" data-expand-citation="Show Citations" data-collapse-footnote="Hide Footnotes" data-expand-footnote="Show Footnotes"><div class="to-citation__wrapper"><button aria-controls="aria-controls" aria-expanded="false" aria-label="Toggle citations menu" class="to-citation__toggle"><i aria-hidden="true" class="icon-plus"></i><span data-expand-title="data-expand-title" data-collapsed-title="data-collapsed-title" class="accordion__toggle__title"></span></button><div role="menu" aria-label="links" class="to-citation__accordion no-separator"><ul></ul></div></div></template><template id="toCitationAccordionItem"><li><a class="to-citation"><i aria-hidden="true" class="icon-subdirectory_arrow_left"></i><span></span></a></li></template><template id="citations_truncate_template"><div class="citations-truncation"><button data-label-expand="Show all references" data-label-collapse="Show fewer" data-label-remaining="references remaining" class="btn btn--inverse"><span>Show all references</span></button></div></template><template id="collateral_texts_template" data-do="Digital Object" data-figures="Figures" data-metrics="Article Metrics" data-related="Related articles" data-supplementary="Supplemental information"></template><template id="collapsible_authors_template"><button aria-expanded="false" data-expandable="all" data-label-expand="Expand All" data-label-collapse="Collapse All" class="collateral-contributors-control"><span>Expand All</span></button></template>
<template id="collapsible_tables_collapse_template"><div class="collapsible-figure-btn__wrapper expanded"><button aria-expanded="true" class="btn collapsible-figure-btn btn--inverse"><i aria-hidden="true" class="icon-arrow-up"></i><span class="text-uppercase">Collapse</span></button></div></template>
<template id="collapsible_tables_expand_template"><div class="collapsible-figure-btn__wrapper collapsed"><button aria-expanded="false" class="btn collapsible-figure-btn btn--inverse"><i aria-hidden="true" class="icon-arrow-down"></i><span class="text-uppercase">Expand Table</span></button></div></template><template id="authorsAffiliationsLink"><a href="#tab-contributors" class="to-authors-affiliations">Authors Info &amp; Affiliations</a></template></div>
                </div>
                </div>

        

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none  widget-  widget-compact-all" id="756cf18d-e910-4468-936c-0225ae7c669f">
                
                
                    
                

                <div class="wrapped " id="ad for nucleus-single-molecule-technologies">
                    
                    <div class="widget-body body body-  body-compact-all"><div class="pb-ad">
            <script type="text/javascript">
URL_var = 'https://www.cell.com/cell/fulltext/S0092-8674(22)00079-4';

if(URL_var.indexOf("S0165-6147(17)30205-5") > -1
|| URL_var.indexOf("S0167-7799(18)30204-X") > -1 
|| URL_var.indexOf("S0092-8674(18)31649-0") > -1 
|| URL_var.indexOf("S0092-8674(19)30404-0") > -1 
|| URL_var.indexOf("S1097-2765(18)30756-1") > -1 
|| URL_var.indexOf("S0092-8674(18)31452-1") > -1 
|| URL_var.indexOf("S1097-2765(19)30175-3") > -1 
|| URL_var.indexOf("S0092-8674(18)30852-3") > -1 
|| URL_var.indexOf("S0006-3495(18)30295-9") > -1 
|| URL_var.indexOf("S1097-2765(19)30063-2") > -1 
|| URL_var.indexOf("S0006-3495(17)33208-3") > -1 
|| URL_var.indexOf("S2211-1247(18)31639-5") > -1 
|| URL_var.indexOf("S1097-2765(18)30978-X") > -1 
)
    {
document.write('<input type="hidden" id="gpttc" value="nucleus-single-molecule-technologies" class="gpt-Target">')
}
</script>
        </div></div>
                </div>
                </div>

        

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none  widget-" id="754b1780-1117-4371-9285-9cd7020963d5">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body- "><div class="pb-ad">
            <script type="text/javascript">
URL_var = 'https://www.cell.com/cell/fulltext/S0092-8674(22)00079-4';

if(URL_var.indexOf("S1535-6108(19)30104-7") > -1
|| URL_var.indexOf("S0092-8674(18)31394-1") > -1
|| URL_var.indexOf("S1074-7613(19)30033-0") > -1
|| URL_var.indexOf("S1535-6108(18)30473-2") > -1
|| URL_var.indexOf("S0092-8674(19)30154-0") > -1
|| URL_var.indexOf("S1550-4131(18)30178-5") > -1
|| URL_var.indexOf("S1535-6108(19)30242-9") > -1
|| URL_var.indexOf("S1074-7613(18)30431-X") > -1
|| URL_var.indexOf("S0092-8674(19)30618-X") > -1
|| URL_var.indexOf("S1535-6108(18)30472-0") > -1
|| URL_var.indexOf("S1535-6108(18)30120-X") > -1
|| URL_var.indexOf("S0092-8674(18)31247-9") > -1
|| URL_var.indexOf("S1074-7613(18)30085-2") > -1
|| URL_var.indexOf("S1074-7613(18)30080-3") > -1
|| URL_var.indexOf("S1471-4906(18)30210-2") > -1
|| URL_var.indexOf("S1550-4131(19)30304-3") > -1
|| URL_var.indexOf("S1934-5909(18)30599-X") > -1
|| URL_var.indexOf("S2405-8033(19)30024-X") > -1
|| URL_var.indexOf("S1535-6108(18)30109-0") > -1
|| URL_var.indexOf("S0962-8924(18)30127-2") > -1
)
    {
document.write('<input type="hidden" id="gpttc" value="nucleus-tumor-immune-microenvironment" class="gpt-Target">')
}
</script>
        </div></div>
                </div>
                </div>

        

    

        


    

            

            

            <div data-widget-id="9c9ff7a1-a035-49f6-a336-600c9419c518" data-widget-name="UX3HTMLWidget"><!-- Start of HubSpot Embed Code --> 

<script type="text/plain" class="optanon-category-4" id="hs-script-loader" async="" defer="" src="//js.hs-scripts.com/519224.js"></script> 

<!-- End of HubSpot Embed Code --></div>

            

        

    

        


    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget sessionValidationWidget none  widget-none  widget-compact-all" id="0257aef7-6074-4898-a989-1c3f693225a2">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none  body-compact-all"><script>
    $.ajax({
        url: "https://www.cell.com/ssoSessionCheck",
        type: "GET",
        dataType: "jsonp",
        success: function (data) {
            if(data.RefreshSession) {
                location.reload();
            }
        },
        error : function (){
            console.warn("can't validate CSAS session.");
        }
    });
</script></div>
                </div>
                </div>

        

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget csasSsoUrls none  widget-none  widget-compact-all" id="948bc5c4-8b0d-47be-b5ad-6570559802d1">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none  body-compact-all"><!-- clear SSO URLs after all iFrames are rendered or after passing a time out-->
<script>
    setTimeout(clearSsoUrlsFun,10000);
    var iFramesList = document.getElementById("sso-urls-div").getElementsByTagName("iframe");
    var renderedIFramesCount = 0;
    var numberOfIFrames = iFramesList.length;
    var skipRefresh = false;
    for (var i = 0; i < iFramesList.length; i++) {
        var iFrame = iFramesList[i];
        iFrame.addEventListener("load", function() {
            renderedIFramesCount = renderedIFramesCount + 1;
            if (renderedIFramesCount >= numberOfIFrames)
            {
                clearSsoUrlsFun();
            }
        });
    }

    function clearSsoUrlsFun() {
        if (skipRefresh == false && numberOfIFrames > 0)
        {
            skipRefresh = true;
            $.ajax({
                url: "https://www.cell.com/action/clearSsoUrls",
                type: "GET",
                dataType: "jsonp",
                error : errorCallingAjx
            });
        }
    }

    function errorCallingAjx(){
        console.log("can't delete sso urls .")
    }
</script></div>
                </div>
                </div>

        










        <div data-widget-def="ux3-layout-widget" data-widget-id="d95b4a46-dc4d-4ca8-83f9-8b6724c649ec" class="ux3 ">
        
    

        


    

            

            

            <div data-widget-id="d95b4a46-dc4d-4ca8-83f9-8b6724c649ec" data-widget-name="ux3-layout-widget" class="fonts-loaded"><div class="footer clearfix">
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none adplaceholder adplaceholder--header widget-horizontalAd" id="a07ee92b-3942-49f8-9186-a2a08fc3fc56">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-horizontalAd "><div class="horizontalAdContainer">
    <div class="horizontalAdLabelContainer">
        <div class="horizontalAdLabel">
                <img src="/templates/jsp/_style2/_marlin/images/bg_adHoriz.gif" alt="Advertisement">
        </div>
    </div>
    <div class="horizontalAd">
        





        <div class="pb-ad">
            <!-- Beginning Async AdSlot 4 for Ad unit els.cellpress/cell  ### size: [[728,90]] -->
<div class="gpt-ad" data-adsize="cell-leaderboard" data-page="" data-site="cell" data-pos="4"></div>
<!-- End AdSlot 4 -->
        </div>
    

    </div>
</div></div>
                </div>
                </div>

        

    

        


    

            

            

            <div data-widget-id="23703bbc-af46-4f3f-a5bb-dd52891ff0ff" data-widget-name="UX3HTMLWidget"><div class="footer__top">
    <div class="footer__logo centered"><a href="/home" title="Cell Press"><!-- Generator: Adobe Illustrator 25.2.1, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 300 72" style="enable-background:new 0 0 300 72;" xml:space="preserve">
<style type="text/css">
	.st0{fill:#007DBC;}
</style>
<g>
	<path class="st0" d="M39.4,7.5c-2.3-0.3-4.4,1.1-4.9,3.3l-1.8,7.4c-0.4,1.8-1.9,3.1-3.8,3.3l-10.5,1.2C7.1,24-1,34.3,0.7,45.6
		c1.3,9,8.6,15.7,17.2,17c2.3,0.3,4.4-1.1,4.9-3.3l1.8-7.4c0.4-1.8,1.9-3.1,3.8-3.3l10.5-1.2c11.3-1.2,19.4-11.6,17.8-22.9
		C55.3,15.4,48.1,8.8,39.4,7.5z M37.1,34l-6.6,0.8c-1.1,0.1-2,0.9-2.3,2l-1.9,8.1c-0.6,2.4-2.5,4.2-5,4.5c-3.8,0.4-7.1-2.4-7.4-6.2
		c-0.3-3.7,2.6-6.8,6.3-7.2l6.6-0.8c1.1-0.1,2-0.9,2.3-2l1.9-8.1c0.6-2.4,2.5-4.2,5-4.5c3.8-0.4,7.1,2.4,7.4,6.2
		C43.7,30.5,40.8,33.6,37.1,34z"></path>
	<rect x="153.1" y="13.9" class="st0" width="8.2" height="42.3"></rect>
	<path class="st0" d="M90.2,20.4c7.3,0,8.6,4.8,9.4,7.3h8.8c-0.5-6.2-5.6-14.9-18.2-14.9c-10.9,0-19.4,7.5-19.4,22.3
		c0,14.6,8.1,22.2,19.3,22.2c10.6,0,16.6-6.4,18.4-15.6h-8.8c-1.1,5.2-4.5,7.9-9.4,7.9c-7.5,0-10.5-6.8-10.5-14.4
		C79.7,23.3,85.9,20.4,90.2,20.4z"></path>
	<path class="st0" d="M125.2,24c-13.7,0-15,12.1-15,15.8c0,12.9,6.8,17.1,15.6,17.1c5.4,0,11.6-2.5,13.8-10H131
		c-0.7,2.2-3,3.4-5.3,3.4c-6.6,0-7.1-5.3-7.3-7.7h21.5v-1.5C139.9,27,131.3,24,125.2,24z M118.6,37.4c0.5-4.1,2.5-6.6,6.4-6.6
		c2.6,0,5.9,1.5,6.4,6.6H118.6z"></path>
	<rect x="142" y="13.9" class="st0" width="8.2" height="42.3"></rect>
	<path class="st0" d="M174,13.8h-9.8V21h7.2c4.3,0,8.2,0.9,8.2,6.4c0,5.4-3.9,6.4-8.2,6.4h-7.2V41h9.8c10.6,0,14.7-6.7,14.7-13.6
		C188.7,20.5,184.6,13.8,174,13.8z"></path>
	<path class="st0" d="M210.5,24.9c-0.5-0.2-1.2-0.3-1.8-0.3c-4.1,0-8.3,2.7-9.8,6.5h-0.1v-5.7h-8v30.7h8.4V42.3
		c0-5.4,2.1-9.9,8.3-9.9c1,0,2.3,0.1,3,0.3V24.9z"></path>
	<path class="st0" d="M225.6,24.6c-9.3,0-15.7,7-15.7,16.2c0,9.5,6,16.1,15.7,16.1c6.9,0,12-3.1,14.4-10.3h-7.4
		c-0.5,1.9-3.3,4-6.6,4c-4.7,0-7.3-2.4-7.5-7.8h22.1C241.1,33.4,236,24.6,225.6,24.6z M218.4,37.5c0.1-2.4,1.7-6.5,7-6.5
		c4.1,0,5.9,2.3,6.7,6.5H218.4z"></path>
	<path class="st0" d="M260.3,37.7c-4.8-1.1-9.6-1.5-9.6-4.6c0-2.5,2.8-2.9,4.7-2.9c3,0,5.5,0.9,5.8,4.2h8c-0.7-7.5-6.8-9.8-13.5-9.8
		c-6.6,0-13.4,2-13.4,9.9c0,5.4,4.9,7.1,9.7,8.2c6.1,1.3,9.6,2.2,9.6,4.8c0,3-3.1,3.9-5.5,3.9c-3.3,0-6.4-1.5-6.5-5.2h-8
		C242,54.3,248.9,57,256,57c7,0,14-2.6,14-10.7C270.1,40.6,265.2,38.8,260.3,37.7z"></path>
	<path class="st0" d="M289.8,37.7c-4.8-1.1-9.6-1.5-9.6-4.6c0-2.5,2.8-2.9,4.7-2.9c3,0,5.5,0.9,5.8,4.2h8c-0.7-7.5-6.8-9.8-13.5-9.8
		c-6.6,0-13.4,2-13.4,9.9c0,5.4,4.9,7.1,9.7,8.2c6.1,1.3,9.6,2.2,9.6,4.8c0,3-3.1,3.9-5.5,3.9c-3.3,0-6.4-1.5-6.5-5.2h-8
		c0.4,8.1,7.3,10.8,14.4,10.8c7,0,14-2.6,14-10.7C299.6,40.6,294.7,38.8,289.8,37.7z"></path>
	<path class="st0" d="M34.1,57.6c0.3-0.2,0.4-0.6,0.4-1c0-0.5-0.2-0.9-0.5-1.1c-0.3-0.2-0.8-0.3-1.4-0.3h-1.9v5.1h0.8V58h0.8
		l1.3,2.2h0.9l-1.4-2.2C33.6,57.9,33.9,57.8,34.1,57.6z M32.5,57.3h-0.9v-1.5h1c0.1,0,0.3,0,0.4,0c0.1,0,0.3,0,0.4,0.1
		c0.1,0,0.2,0.1,0.3,0.2c0.1,0.1,0.1,0.2,0.1,0.4c0,0.2,0,0.3-0.1,0.4c-0.1,0.1-0.2,0.2-0.3,0.2c-0.1,0.1-0.2,0.1-0.4,0.1
		C32.8,57.3,32.6,57.3,32.5,57.3z"></path>
	<path class="st0" d="M35.6,54.2c-0.4-0.4-0.9-0.7-1.4-0.9c-0.5-0.2-1.1-0.3-1.7-0.3c-0.6,0-1.2,0.1-1.7,0.3c-0.6,0.2-1,0.5-1.4,0.9
		c-0.4,0.4-0.7,0.9-1,1.5c-0.2,0.6-0.4,1.2-0.4,1.9c0,0.7,0.1,1.3,0.3,1.8c0.2,0.6,0.5,1,0.9,1.5c0.4,0.4,0.9,0.7,1.4,1
		c0.6,0.2,1.2,0.4,1.8,0.4c0.6,0,1.2-0.1,1.7-0.3c0.5-0.2,1-0.5,1.4-0.9c0.4-0.4,0.7-0.9,1-1.5c0.2-0.6,0.4-1.2,0.4-1.9
		c0-0.7-0.1-1.3-0.4-1.9C36.4,55.1,36,54.6,35.6,54.2z M35.7,59.1c-0.2,0.5-0.4,0.9-0.8,1.2c-0.3,0.3-0.7,0.6-1.1,0.8
		c-0.4,0.2-0.9,0.3-1.4,0.3c-0.6,0-1-0.1-1.5-0.3c-0.4-0.2-0.8-0.5-1.1-0.8c-0.3-0.3-0.5-0.7-0.7-1.2c-0.2-0.5-0.2-1-0.2-1.5
		c0-0.6,0.1-1.1,0.3-1.5c0.2-0.5,0.4-0.9,0.8-1.2c0.3-0.3,0.7-0.6,1.1-0.8c0.4-0.2,0.9-0.3,1.4-0.3c0.5,0,1,0.1,1.4,0.3
		c0.4,0.2,0.8,0.4,1.1,0.8c0.3,0.3,0.6,0.7,0.8,1.2c0.2,0.5,0.3,1,0.3,1.5C36,58.1,35.9,58.6,35.7,59.1z"></path>
</g>
</svg></a></div>
    <div class="footer__social-links">
        <ul class="rlist rlist--inline centered">
            <li><a href="http://on.fb.me/1tgHakh" title="Facebook" target="_blank"><i aria-hidden="true" class="icon-Icon_Facebook"></i></a></li>

            <li><a href="https://www.instagram.com/cellpress/" title="Instagram" target="_blank"><svg width="18px" height="18px" viewBox="0 0 18 18" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-147.000000, -16.000000)" fill-rule="nonzero" fill="#969696"> <g id="Instagram-Logo" transform="translate(147.000000, 16.000000)"> <path d="M9,1.62580645 C11.4096774,1.62580645 11.7,1.62580645 12.6290323,1.68387097 C13.5,1.71290323 13.9935484,1.85806452 14.3129032,2.00322581 C14.7193548,2.17741935 15.0387097,2.3516129 15.3580645,2.67096774 C15.6774194,2.99032258 15.8806452,3.28064516 16.0258065,3.71612903 C16.1419355,4.03548387 16.2870968,4.5 16.3451613,5.4 C16.3741935,6.35806452 16.4032258,6.61935484 16.4032258,9.02903226 C16.4032258,11.4387097 16.4032258,11.7290323 16.3451613,12.6580645 C16.316129,13.5290323 16.1709677,14.0225806 16.0258065,14.3419355 C15.8516129,14.7483871 15.6774194,15.0677419 15.3580645,15.3870968 C15.0387097,15.7064516 14.7483871,15.9096774 14.3129032,16.0548387 C13.9935484,16.1709677 13.5290323,16.316129 12.6290323,16.3741935 C11.6709677,16.4032258 11.4096774,16.4322581 9,16.4322581 C6.59032258,16.4322581 6.3,16.4322581 5.37096774,16.3741935 C4.5,16.3451613 4.00645161,16.2 3.68709677,16.0548387 C3.28064516,15.8806452 2.96129032,15.7064516 2.64193548,15.3870968 C2.32258065,15.0677419 2.11935484,14.7774194 1.97419355,14.3419355 C1.85806452,14.0225806 1.71290323,13.5580645 1.65483871,12.6580645 C1.62580645,11.7 1.62580645,11.4096774 1.62580645,9 C1.62580645,6.59032258 1.62580645,6.3 1.68387097,5.37096774 C1.71290323,4.5 1.85806452,4.00645161 2.00322581,3.68709677 C2.17741935,3.28064516 2.3516129,2.96129032 2.67096774,2.64193548 C2.99032258,2.32258065 3.28064516,2.11935484 3.71612903,1.97419355 C4.03548387,1.85806452 4.5,1.71290323 5.4,1.65483871 C6.3,1.62580645 6.59032258,1.62580645 9,1.62580645 L9,1.62580645 Z M9,0 C6.56129032,0 6.24193548,0 5.28387097,0.0580645161 C4.32580645,0.116129032 3.65806452,0.261290323 3.10645161,0.464516129 C2.52580645,0.696774194 2.00322581,1.01612903 1.50967742,1.50967742 C1.01612903,2.00322581 0.696774194,2.52580645 0.464516129,3.10645161 C0.232258065,3.68709677 0.0870967742,4.32580645 0.0580645161,5.28387097 C0,6.24193548 0,6.56129032 0,9 C0,11.4387097 0,11.7580645 0.0580645161,12.716129 C0.116129032,13.6741935 0.261290323,14.3419355 0.464516129,14.8935484 C0.696774194,15.4741935 1.01612903,15.9967742 1.50967742,16.4903226 C2.00322581,16.983871 2.52580645,17.3032258 3.10645161,17.5354839 C3.68709677,17.7677419 4.32580645,17.9129032 5.28387097,17.9419355 C6.24193548,18 6.56129032,18 9,18 C11.4387097,18 11.7580645,18 12.716129,17.9419355 C13.6741935,17.883871 14.3419355,17.7387097 14.8935484,17.5354839 C15.4741935,17.3032258 15.9967742,16.983871 16.4903226,16.4903226 C16.983871,15.9967742 17.3032258,15.4741935 17.5354839,14.8935484 C17.7677419,14.3129032 17.9129032,13.6741935 17.9419355,12.716129 C18,11.7580645 18,11.4387097 18,9 C18,6.56129032 18,6.24193548 17.9419355,5.28387097 C17.883871,4.32580645 17.7387097,3.65806452 17.5354839,3.10645161 C17.3032258,2.52580645 16.983871,2.00322581 16.4903226,1.50967742 C15.9967742,1.01612903 15.4741935,0.696774194 14.8935484,0.464516129 C14.3129032,0.232258065 13.6741935,0.0870967742 12.716129,0.0580645161 C11.7580645,0 11.4387097,0 9,0 L9,0 Z"></path> <path d="M9,4.38387097 C6.44516129,4.38387097 4.38387097,6.44516129 4.38387097,9 C4.38387097,11.5548387 6.44516129,13.616129 9,13.616129 C11.5548387,13.616129 13.616129,11.5548387 13.616129,9 C13.616129,6.44516129 11.5548387,4.38387097 9,4.38387097 Z M9,11.9903226 C7.34516129,11.9903226 6.00967742,10.6548387 6.00967742,9 C6.00967742,7.34516129 7.34516129,6.00967742 9,6.00967742 C10.6548387,6.00967742 11.9903226,7.34516129 11.9903226,9 C11.9903226,10.6548387 10.6548387,11.9903226 9,11.9903226 Z"></path> <circle id="Oval" cx="13.7903226" cy="4.20967742" r="1.07419355"></circle> </g> </g> </g></svg></a></li>
            
            <li><a href="https://www.linkedin.com/company/cell-press/" title="LinkedIn" target="_blank"><i aria-hidden="true" class="icon-linkedin"></i></a></li>
            
                        <li><a href="http://bit.ly/1zq7Q5T" title="Twitter" target="_blank">
                <!--<i aria-hidden="true" class="icon-Icon_Twitter"></i></a>-->
                
                <svg id="xicon" viewBox="0 0 1200 1227" xmlns="http://www.w3.org/2000/svg" aria-hidden="true" role="none"><path fill="#969696" d="M714.163 519.284L1160.89 0H1055.03L667.137 450.887L357.328 0H0L468.492 681.821L0 1226.37H105.866L515.491 750.218L842.672 1226.37H1200L714.137 519.284H714.163ZM569.165 687.828L521.697 619.934L144.011 79.6944H306.615L611.412 515.685L658.88 583.579L1055.08 1150.3H892.476L569.165 687.854V687.828Z"></path></svg> 

                    <style>
                    .footer svg#xicon {
                      height: 16px;
                    }
                    </style>
            </a></li>

            
            <li><a href="/pb-assets/marketing/WeChat/WeChat-1557753222727.jpg" title="WeChat" target="_blank"><svg width="18px" height="15px" viewBox="0 0 18 15" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-122.000000, -17.000000)" fill-rule="nonzero" fill="#969696"> <g id="wechat" transform="translate(122.000000, 17.000000)"> <path d="M12.375,4.5 L12.375,4.5 L12.4875,4.5 L12.4875,4.5 C12.6898579,4.48604473 12.8520408,4.32704187 12.87,4.125 C12.8705905,4.04725572 12.8470025,3.9712499 12.8025,3.9075 C11.94,1.5675 9.4425,0 6.5625,0 C2.9475,0 0,2.52 0,5.625 C0.0313772094,7.34410122 0.895513317,8.94093526 2.3175,9.9075 L1.5675,11.46 C1.49009368,11.6087816 1.52088865,11.7908702 1.64291494,11.9059236 C1.76494122,12.0209769 1.94852501,12.0410168 2.0925,11.955 L4.08,10.8225 C4.61861992,11.0129999 5.17724298,11.1413322 5.745,11.205 C5.86564124,11.2178337 5.98503738,11.1715098 6.06545426,11.0806685 C6.14587113,10.9898272 6.17737153,10.865694 6.15,10.7475 C6.06148727,10.4219419 6.01114891,10.0871918 6,9.75 C6,6.855 8.8575,4.5 12.375,4.5 Z M8.25,3 C8.66421356,3 9,3.33578644 9,3.75 C9,4.16421356 8.66421356,4.5 8.25,4.5 C7.83578644,4.5 7.5,4.16421356 7.5,3.75 C7.5,3.33578644 7.83578644,3 8.25,3 Z M4.5,4.5 C4.08578644,4.5 3.75,4.16421356 3.75,3.75 C3.75,3.33578644 4.08578644,3 4.5,3 C4.91421356,3 5.25,3.33578644 5.25,3.75 C5.25,4.16421356 4.91421356,4.5 4.5,4.5 Z"></path> <path d="M18,9.75 C18,7.2675 15.48,5.25 12.375,5.25 C9.27,5.25 6.75,7.2675 6.75,9.75 C6.75,12.2325 9.27,14.25 12.375,14.25 C12.9568557,14.2502837 13.5364258,14.1772075 14.1,14.0325 L15.96,14.9625 C16.0114248,14.9874074 16.0678613,15.0002339 16.125,15 C16.2497193,14.9993787 16.3659682,14.9367912 16.4351508,14.8330172 C16.5043335,14.7292432 16.5173955,14.5978644 16.47,14.4825 L15.96,13.2075 C17.1784092,12.4716325 17.9450305,11.1723221 18,9.75 L18,9.75 Z M10.5,9 C10.0857864,9 9.75,8.66421356 9.75,8.25 C9.75,7.83578644 10.0857864,7.5 10.5,7.5 C10.9142136,7.5 11.25,7.83578644 11.25,8.25 C11.25,8.66421356 10.9142136,9 10.5,9 Z M14.25,9 C13.8357864,9 13.5,8.66421356 13.5,8.25 C13.5,7.83578644 13.8357864,7.5 14.25,7.5 C14.6642136,7.5 15,7.83578644 15,8.25 C15,8.66421356 14.6642136,9 14.25,9 Z" opacity="0.959999979"></path> </g> </g> </g></svg></a></li>
            
            <li><a href="https://www.weibo.com/CellPress" title="Weibo" target="_blank"><svg width="18px" height="15px" viewBox="0 0 18 15" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-99.000000, -17.000000)" fill-rule="nonzero" fill="#969696"> <g id="weibo" transform="translate(99.000000, 14.000000)"> <g transform="translate(0.000000, 3.000000)"> <path d="M13.2402211,7.47054172 C12.9918392,7.39866484 12.821608,7.34902326 12.9514975,7.03290287 C13.232804,6.34602189 13.2622915,5.75359781 12.9571055,5.33095486 C12.3847236,4.5380684 10.8191759,4.58064295 9.02514573,5.30975376 C9.02514573,5.30889193 8.46144724,5.54882627 8.60544724,5.11515185 C8.88150754,4.25435294 8.83953769,3.53386047 8.41043216,3.11759508 C7.43716583,2.17285363 4.84824121,3.15344733 2.62745729,5.3059617 C0.964944724,6.9186238 0,8.62798358 0,10.105855 C0,12.9324952 3.73748744,14.8235294 7.39429146,14.8235294 C12.1877186,14.8235294 15.3768844,11.9501778 15.3768844,9.80576471 C15.3768844,8.50974008 14.2514774,7.77442408 13.2402211,7.47054172 Z M7.40424121,13.5561176 C4.48625126,13.8315595 1.9678794,12.5686293 1.77811055,10.7327524 C1.58870352,8.89773735 3.80098492,7.18579207 6.71861307,6.90948837 C9.63678392,6.63266758 12.1556985,7.89663201 12.3447437,9.73164706 C12.533608,11.5675239 10.3227739,13.2796416 7.40424121,13.5561176 Z M18,5.16875787 C17.9985528,2.31402189 15.5695477,0 12.5728643,0 C12.222995,0 11.9396985,0.270098495 11.9396985,0.603283174 C11.9396985,0.936640219 12.222995,1.20656635 12.5728643,1.20656635 C14.8708945,1.20656635 16.7336683,2.98159781 16.7336683,5.17099863 C16.7336683,5.50435568 17.0173266,5.77428181 17.3668342,5.77428181 C17.7167035,5.77428181 18,5.50435568 18,5.17099863 C18,5.17099863 18,5.1696197 18,5.16875787 Z M15.4841608,5.08016142 C15.1948945,3.73415048 14.0812462,2.67323393 12.6683819,2.39744733 C12.3262915,2.33039672 11.9914372,2.54051163 11.9212462,2.86680164 C11.8506935,3.19309166 12.0712161,3.51179754 12.4133065,3.57919289 C13.3319397,3.75845417 14.0559196,4.44774829 14.2438794,5.32302599 C14.3146131,5.64931601 14.6491055,5.85925855 14.9915578,5.7923803 C15.3338291,5.72498495 15.5541709,5.40627907 15.4841608,5.08016142 Z M5.90761809,8.7069275 C4.7321005,8.9320383 3.93901508,9.87126402 4.1358392,10.8044569 C4.33248241,11.7378222 5.44504523,12.311803 6.62056281,12.0865198 C7.79571859,11.861409 8.58916583,10.922528 8.3921608,9.98899042 C8.19569849,9.0559699 7.08277387,8.48181669 5.90761809,8.7069275 Z"></path> </g> </g> </g> </g></svg></a></li>
            
            <li><a href="https://www.youtube.com/user/cellvideoabstracts" title="YouTube" target="_blank"><svg width="18px" height="13px" viewBox="0 0 18 13" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-76.000000, -19.000000)" fill-rule="nonzero" fill="#969696"> <g id="youtube" transform="translate(76.000000, 14.000000)"> <path d="M17.2984219,6.41974219 C16.6486641,5.64735937 15.4490625,5.33232422 13.1580703,5.33232422 L4.84175391,5.33232422 C2.49834375,5.33232422 1.27838672,5.66771484 0.631089844,6.49001953 C0,7.29172266 0,8.47304297 0,10.1080547 L0,13.2243047 C0,16.3917773 0.748792969,18 4.84175391,18 L13.1581055,18 C15.1448555,18 16.2457383,17.7219844 16.9579688,17.040375 C17.688375,16.3413984 18,15.2001211 18,13.2243047 L18,10.1080547 C18,8.38381641 17.951168,7.19553516 17.2984219,6.41974219 Z M11.5560703,12.0963867 L7.77965625,14.0700586 C7.69524609,14.1141797 7.60289062,14.136082 7.51071094,14.136082 C7.40629687,14.136082 7.30212891,14.1079922 7.20987891,14.0521289 C7.03613672,13.9468711 6.93007031,13.7585391 6.93007031,13.5554414 L6.93007031,9.62075391 C6.93007031,9.41800781 7.03585547,9.22985156 7.20924609,9.12452344 C7.38267187,9.01919531 7.59839062,9.01198828 7.77832031,9.10546875 L11.5547344,11.0664141 C11.7468633,11.1661523 11.8675195,11.3645039 11.8678008,11.5809258 C11.8680469,11.7975586 11.747918,11.9961914 11.5560703,12.0963867 Z"></path> </g> </g> </g></svg></a></li>
            
        </ul>
    </div>
</div>

<style>

.ux3 .footer__logo svg {
    max-width: 160px !important;
}

.ux3 .footer__social-links a svg {
    max-width: 26px;
    height: 1.15rem;
    width: auto;
}

.ux3 .footer__social-links a:hover svg path {
    fill: #353535;
}

</style></div>

            

        

    

        


    

            

            
                <div class="footer__bottom text-onDark">
            

            <div data-widget-id="06227714-c621-4651-9879-b0ce5176cccc" data-widget-name="ux3-layout-widget"><div class="col-sm-4 col-md-5 clearfix">
    

        


    

            

            

            <div data-widget-id="07e1d276-b4dd-4870-b858-ed74e70d64ef" data-widget-name="UX3HTMLWidget"><div class="groups">
	<div class="group">
		<h2 class="group__title">Life &amp; medical sciences journals</h2>
		<ul class="rlist clearfix sided">
		<li>
			<a alt="Cell" href="/cell/home">Cell</a>
		</li>
		<li>
			<a alt="Cancer Cell" href="/cancer-cell/home">Cancer Cell</a>
		</li>
		<li>
			<a alt="Cell Chemical Biology" href="/cell-chemical-biology">Cell Chemical Biology</a>
		</li>
		<li>
			<a alt="Cell Genomics" href="/cell-genomics/home">Cell Genomics</a>
		</li>
		<li>
			<a alt="Cell Host and Microbe" href="/cell-host-microbe/home">Cell Host &amp; Microbe</a>
		</li>
		<li>
			<a alt="Cell Metabolism" href="/cell-metabolism/home">Cell Metabolism</a>
		</li>
		<li>
			<a alt="Cell Reports" href="/cell-reports/home">Cell Reports</a>
		</li>
		<li>
			<a alt="Cell Reports Medicine" href="/cell-reports-medicine/home">Cell Reports Medicine</a>
		</li>
		<li>
			<a alt="Cell Stem Cell" href="/cell-stem-cell/home">Cell Stem Cell</a>
		</li>
		<li>
			<a alt="Cell Systems" href="/cell-systems/home">Cell Systems</a>
		</li>
		<li>
			<a alt="Current Biology" href="/current-biology/home">Current Biology</a>
		</li>
		<li>
			<a alt="Developmental Cell" href="/developmental-cell/home">Developmental Cell</a>
		</li>
		<li>
			<a alt="Immunity" href="/immunity/home">Immunity</a>
		</li>
		<li>
			<a alt="Med" href="/med/home">Med</a>
		</li>
		<li>
			<a alt="Molecular Cell" href="/molecular-cell/home">Molecular Cell</a>
		</li>
		<li>
			<a alt="Neuron" href="/neuron/home">Neuron</a>
		</li>
		<li>
			<a alt="Structure" href="/structure/home">Structure</a>
		</li>
		<li>
			<a alt="American Journal of Human Genetics" href="/ajhg/home">American Journal of Human Genetics (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Biophysical Journal" href="/biophysj/home">Biophysical Journal (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Biophysical Reports" href="/biophysreports/home">Biophysical Reports (<i>partner</i>)</a>
		</li>	
		<li>
			<a alt="HGG Advances" href="/hgg-advances/home">Human Genetics and Genomics Advances (<i>partner</i>)</a></li>
		<li>
			<a alt="Molecular Plant" href="/molecular-plant/home">Molecular Plant (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy" href="/molecular-therapy-family/molecular-therapy/home">Molecular Therapy (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy Methods and Clinical Development" href="/molecular-therapy-family/methods/home">Molecular Therapy Methods &amp; Clinical Development (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy Nucleic Acids" href="/molecular-therapy-family/nucleic-acids/home">Molecular Therapy Nucleic Acids (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy Oncology" href="/molecular-therapy-family/oncology/home">Molecular Therapy Oncology (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Plant Communications" href="/plant-communications/home">Plant Communications (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Stem Cell Reports" href="/stem-cell-reports/home">Stem Cell Reports (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Trends in Biochemical Sciences" href="/trends/biochemical-sciences/home">Trends in Biochemical Sciences</a>
		</li>
		<li>
		</li><li>
			<a alt="Trends in Cancer" href="/trends/cancer/home">Trends in Cancer</a>
		</li>
		<li>
			<a alt="Trends in Cell Biology" href="/trends/cell-biology/home">Trends in Cell Biology</a>
		</li>
		<li>
			<a alt="Trends in Ecology and Evolution" href="/trends/ecology-evolution/home">Trends in Ecology &amp; Evolution</a>
		</li>
		<li>
			<a alt="Trends in Endocrinology and Metabolism" href="/trends/endocrinology-metabolism/home">Trends in Endocrinology &amp; Metabolism</a>
		</li>
		<li>
			<a alt="Trends in Genetics" href="/trends/genetics/home">Trends in Genetics</a>
		</li>
		<li>
			<a alt="Trends in Immunology" href="/trends/immunology/home">Trends in Immunology</a>
		</li>
		<li>
			<a alt="Trends in Microbiology" href="/trends/microbiology/home">Trends in Microbiology</a>
		</li>
		<li>
			<a alt="Trends in Molecular Medicine" href="/trends/molecular-medicine/home">Trends in Molecular Medicine</a>
		</li>
		<li>
			<a alt="Trends in Neurosciences" href="/trends/neurosciences/home">Trends in Neurosciences</a>
		</li>
		<li>
			<a alt="Trends in Parasitology" href="/trends/parasitology/home">Trends in Parasitology</a>
		</li>
		<li>
			<a alt="Trends in Pharmacological Sciences" href="/trends/pharmacological-sciences/home">Trends in Pharmacological Sciences</a>
		</li>
		<li>
			<a alt="Trends in Plant Science" href="/trends/plant-science/home">Trends in Plant Science</a>
		</li>
		</ul>
	</div>

	<div class="group">
		<h2 class="group__title">Physical sciences &amp; engineering journals</h2>
		<ul class="rlist clearfix sided">
		  <li><a alt="Cell Biomaterials" href="/cell-biomaterials/home">Cell Biomaterials</a></li>
		  <li>
			<a alt="Cell Reports Physical Science" href="/cell-reports-physical-science/home">Cell Reports Physical Science</a>
		</li>
		<li>
			<a alt="Chem" href="/chem/home">Chem</a>
		</li>
		<li>
			<a alt="Chem Catalysis" href="/chem-catalysis/home">Chem Catalysis</a>
		</li>
		<li>
			<a alt="Device" href="/device/home">Device</a>
		</li>
		<li>
			<a alt="Joule" href="/joule/home">Joule</a>
		</li>
		<li>
			<a alt="Matter" href="/matter/home">Matter</a>
		</li>	
		<li><a alt="Matter" href="/newton/home">Newton</a></li>
		<li>
			<a alt="Trends in Chemistry" href="/trends/chemistry/home">Trends in Chemistry</a>
		</li>			
		</ul>
	</div>

<div class="group">
	<h2 class="group__title">Multidisciplinary journals</h2>
	<ul class="rlist clearfix sided">
	  		<li>
			<a alt="Cell Reports Methods" href="/cell-reports-methods/home">Cell Reports Methods</a>
		</li>
		<li>
			<a alt="Cell Reports Sustainability" href="/cell-reports-sustainability/home">Cell Reports Sustainability</a>
		</li>
		<li>
			<a alt="Heliyon" href="/heliyon/home">Heliyon</a>
		</li>	
		<li>
			<a alt="iScience" href="/iscience/home">iScience</a>
		</li>
		<li>
			<a alt="One Earth" href="/one-earth/home">One Earth</a>
		</li>
		<li>
			<a alt="Patterns" href="/patterns/home">Patterns</a>
		</li>
		<li>
			<a alt="STAR Protocols" href="/star-protocols/home">STAR Protocols</a>
		</li>
		<li>
			<a alt="Nexus" href="/nexus/home">Nexus (<i>partner</i>)</a>
		</li>			
        <li>
			<a alt="The Innovation" href="/the-innovation/home">The Innovation (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Trends in Biotechnology" href="/trends/biotechnology/home">Trends in Biotechnology</a>
		</li>
		<li>
			<a alt="Trends in Cognitive Sciences" href="/trends/cognitive-sciences/home">Trends in Cognitive Sciences</a>
		</li>
	</ul>
</div>

</div>
</div>

            

        
</div><div class="col-sm-4 col-md-5 clearfix">
    

        


    

            

            

            <div data-widget-id="850476de-00cb-4c0d-943c-bf1881da24f9" data-widget-name="UX3HTMLWidget"><div class="groups">
	<div class="clearfix">
		<div class="group sided">
			<h2 class="group__title">Authors</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Submit article" href="/submit-to-cell-press">Submit article</a>
				</li>
				<li>
					<a alt="Multi-Journal Submission" href="/multi-journal-submission">Multi-Journal Submission</a>
				</li>
				<li>
					<a alt="STAR Methods" href="/star-authors-guide">STAR Methods</a>
				</li>
				<li>
					<a alt="Sneak Peek – Preprints" href="/sneakpeek">Sneak Peek – Preprints</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
		<h2 class="group__title">Reviewers</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Reviewers" href="/reviewers">Information for reviewers</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
		<h2 class="group__title">News &amp; events</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Newsroom" href="/newsroom">Newsroom</a>
				</li>
				<li>
					<a alt="Events" href="/events">Events</a>
				</li>
				<li>
					<a alt="Cell Symposia" href="/symposia">Cell Symposia</a>
				</li>
				<li>
					<a alt="Consortia Hub" href="/consortia">Consortia Hub</a>
				</li>
				<li>
					<a alt="Webinars" href="/webinars">Webinars</a>
				</li>
								<li>
					&nbsp;
				</li>
			</ul>
		<h2 class="group__title">Multimedia</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Cell Press Podcast" href="/podcast">Cell Press Podcast</a>
				</li>
				<li>
					<a alt="Cell Press Videos" href="/video">Cell Press Videos</a>
				</li>
				<li>
					<a alt="Color and Comics" href="/comics">Coloring and Comics</a>
				</li>
				<li>
					<a alt="Figure360" href="/video-guidelines">Figure360</a>
				</li>
				<li>
					<a alt="Cell Picture Show" href="/pictureshow">Cell Picture Show</a>
				</li>
				<li>
					<a alt="Research Arc" href="/research-arc">Research Arc</a>
				</li>
			</ul>
		</div>

		<div class="group sided">
			<h2 class="group__title">About</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="About Cell Press" href="/about">About Cell Press</a>
				</li>

				<li>
					<a alt="Open access" href="/open-access">Open access</a>
				</li>
				<li>
					<a alt="COVID Hub" href="/covid-19">COVID Hub</a>
				</li>
				<li>
					<a alt="Sustainability" href="/sustainability">Sustainability hub</a>
				</li>
				<li>
					<a alt="Inclusion and diversity" href="/diversity">Inclusion and diversity</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
			<h2 class="group__title">Contact</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Contact us" href="/contact">Contact us</a>
				</li>
				<li>
					<a alt="Help and Support" href="https://service.elsevier.com/app/overview/cell-press/" target="_blank">Help &amp; Support</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
			<h2 class="group__title">Careers</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Cell Press Careers" href="https://relx.wd3.myworkdayjobs.com/en-US/ElsevierJobs" target="_blank">Cell Press Careers</a>
				</li>
			</ul>
		</div>
	</div>
</div></div>

            

        
</div><div class="col-sm-4 col-md-2 clearfix">
    

        


    

            

            

            <div data-widget-id="beafac41-1cfc-440b-9b55-74aba4a5612c" data-widget-name="UX3HTMLWidget"><div class="groups">
	<div class="group">
		<h2 class="group__title">Access</h2>
		<ul class="rlist clearfix">
			<li>
				<a alt="Subscribe" href="/subscribe">Subscribe</a>
			</li>
			<li>
				<a alt="Claim" href="/activate">Claim</a>
			</li>
			<li>
				<a alt="Read It Now" href="/read-it-now">Read-It-Now</a>
			</li>
			<li>
				<a alt="Recommend to Librarian" href="http://info.cell.com/request-access-to-cell-press-content" target="_blank">Recommend to Librarian</a>
			</li>
			<li>
				<a alt="Publication Alerts" href="/action/showPreferences?menuTab=Alerts">Publication Alerts</a>
			</li>
		</ul>
	</div>
	<div class="group">
		<h2 class="group__title">Collections</h2>
		<ul class="rlist clearfix">
			<li>
				<a alt="Best of Cell Press" href="/bestof">Best of Cell Press</a>
			</li>
			<li>
				<a alt="Cell Press Reviews" href="/reviews">Cell Press Reviews</a>
			</li>
			<li>
				<a alt="Cell Selections" href="/selections">Cell Press Selections</a>
			</li>
			<li>
				<a alt="Nucleus Collections" href="/nucleus">Nucleus Collections</a>
			</li>
			<li>
				<a alt="SnapShots" href="/snapshots">SnapShot Archive</a>
			</li>
		</ul>
	</div>
	<div class="group">
		<h2 class="group__title">Information</h2>
		<ul class="rlist clearfix">
			<li>
				<a alt="For Advertisters" href="/media-kit">For Advertisers</a>
			</li>
			<li>
				<a alt="For Recruiters" target="_blank" href="/pb-assets/Cell/common/CP-RecruitmentKit-US-2018-LowRes-1712070648.pdf">For Recruiters</a>
			</li>
			<li>
				<a alt="For Librarians" target="_blank" href="https://www.elsevier.com/products/sciencedirect/journals/direct-subscription-journals">For Librarians</a>
			</li>
			<li>
				<a alt="Privacy Policy" target="_blank" href="https://www.elsevier.com/legal/privacy-policy">Privacy Policy</a>
			</li>
			<li>
				<a alt="Terms and Conditions" target="_blank" href="https://www.elsevier.com/legal/elsevier-website-terms-and-conditions">Terms and Conditions</a>
			</li>
			<li>
				<a alt="Accessibility" href="/accessibility">Accessibility</a>
			</li>
		</ul>
	</div>
</div></div>

            

        
</div></div>

            
                </div>
            

        

    

        


    

            

            

            <div data-widget-id="779ad11f-7bcd-4a2a-a2cd-8c67b649cbb2" data-widget-name="UX3HTMLWidget"><style>
 
button#cookieBtn {
   text-decoration: none;
   background-color: transparent;
   color: #005789;
   font-weight: bold;
   padding: 0;
   margin: 0;
   font-size: 14px;
}
 
button#cookieBtn:hover {
   text-decoration: underline;
}

</style>
<script>
// HIDE ONETRUST BACKGROUND AFTER USER PRESSES ESCAPE KEY
setTimeout(function(){ 
   $("#onetrust-pc-sdk").on("keydown", function(e) { 
      if (e.keyCode == 27) {  // ESC key
         setTimeout(function(){ 
            $(".onetrust-pc-dark-filter").css("display","none");
         }, 100); 
      }
   });
}, 5000);
</script>

<span class="align-center copy-right">
<hr>
<p align="center"><b>The content on this site is intended for healthcare professionals and researchers across all fields of science.</b></p>
<hr>
<div>
<p>We use cookies to help provide and enhance our service and tailor content. To update your cookie settings, please visit the 
      <b><button id="cookieBtn" class="ot-sdk-show-settings" type="button" aria-label="Manage cookie consent preferences">Cookie Settings</button></b> for this site.<br>

All content on this site: Copyright © 2025 Elsevier Inc., its licensors, and contributors. <br>All rights are reserved, including those for text and data mining, AI training, and similar technologies. <br>For all open access content, the relevant licensing terms apply.
</p>
<ul class="rlist--inline">
			<li>
				<a alt="Privacy Policy" target="_blank" href="https://www.elsevier.com/legal/privacy-policy">Privacy Policy</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Terms and Conditions" target="_blank" href="https://www.elsevier.com/legal/elsevier-website-terms-and-conditions">Terms &amp; Conditions</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Accessibility" href="/accessibility">Accessibility</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Help and Support" target="_blank" href="https://service.elsevier.com/app/overview/cell-press/">Help &amp; Support</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Contact" href="/contact">Contact</a>
			</li>
			</ul>
<p>
<a href="https://www.relx.com"><img src="/pb/assets/raw/Health%20Advance/images/relx-1554393210983.png" alt="RELX" rel="nofollow" target="_new"></a></p>
</div>

</span></div>

            

        

    

        


    

            

            

            <div data-widget-id="c5e179c1-156e-41e3-af7f-5e307efa8acf" data-widget-name="UX3HTMLWidget"><style>#brandingRow{
    background-color:#fff;
    padding:7px 26px 7px 30px;
    color:#007DBC;
    font-size:14px;
    clear:both;
    width:100%
}
#brandingRow a{
    text-decoration:none;
    color:#007DBC
}
#brandingRow a:hover{
    text-decoration:underline
}
#brandingRowRight{
    display:inline;
    float:right;
    width:auto;
    padding:0;
    margin:0
}
svg#cp_logo {
  float: none;
  height: 36px;
  width: 36px;
  margin-left: 0px;
  margin-top: 0px;
}

footer hr {
  display: block;
}

@media screen and (max-width: 600px){
    #brandingRow{
        font-size:12px
    }
}
@media screen and (max-width: 750px){
    #brandingRow{
        padding:7px 18px 7px 16px
    }
    svg#cp_logo{
        margin-right:10px
    }
}
@media screen and (max-width: 800px) {
   a#cpIconLnk {
      display: none;
   }
}

.header__wrapper.clearfix .col-xs-6:first-of-type{
    width:55%
}
.header__wrapper.clearfix .col-xs-6:nth-of-type(2){
    width:45%
}
@media screen and (min-width: 800px) and (max-width: 1000px){
    .header__wrapper.clearfix .col-xs-6:first-of-type{
        width:75%
    }
    .header__wrapper.clearfix .col-xs-6:nth-of-type(2){
        width:25%
    }
}

</style>

<script>
var partnerx = false;
var overrideEndorsement = "";
var tmpj = $("input[name='journalCode']").attr("value");
	
if (window.location.pathname == "/molecular-therapy-family/home") {
	tmpj = "ymthe";
}
	
var ribbonx = "";
if (tmpj == "ajhg" || tmpj == "bpj" || tmpj == "molp" || tmpj == "xplc" || tmpj == "stemcr" || tmpj == "ymthe" || tmpj == "xhgg" || tmpj == "omtn" || tmpj == "omto" || tmpj == "omtm" || tmpj == "biophysreports" ||  tmpj == "bpr" || tmpj == "xinn" || tmpj == "omton") {
	partnerx = true;
	ribbonx = "<a href='https://www.cell.com' data-aa-name='cell press endorsement' data-aa-region='cp global header' title='Visit Cell.com'>A Cell Press partner journal</a>";
} else if (tmpj != "" && tmpj != "undefined" && tmpj != undefined) {
	ribbonx = "<a href='/' data-aa-name='cell press endorsement' data-aa-region='cp global header' title='Visit Cell.com'>A Cell Press journal</a>";
} else {
	ribbonx = "<a href='/' data-aa-name='cell press endorsement' data-aa-region='cp global header' title='Visit Cell.com'>A Cell Press resource</a>";
}
	
if (window.location.pathname == "/") {
	ribbonx = "";
}
if (overrideEndorsement != "") {
	ribbonx = overrideEndorsement;
}

$(".header__wrapper.clearfix").prepend("<div id='brandingRow'><a href='/' data-aa-name='science that inspires' data-aa-region='cp global header' title='Visit Cell.com'>Science that inspires</a><p id='brandingRowRight'>" + ribbonx + "</p></div>");
	
// handle scrolling effect on article pages
if ($("article").length > 0) {
	var scrollx;
	$(window).scroll(function() {
		scrollx = $(window).scrollTop();
		if (scrollx > 0) { 
			$("#brandingRow").css("display","none"); 
		} else {
			$("#brandingRow").css("display","block");
		}
	});
}

if (window.location.pathname == "/" || partnerx == false) {
               $("<a href='/home' id='cpIconLnk' data-aa-name='cell press logo' data-aa-region='cp global header' title='Cell Press' target='_blank'><svg id='cp_logo' version='1.1' xmlns='http://www.w3.org/2000/svg' xmlns:xlink='http://www.w3.org/1999/xlink' x='0px' y='0px' viewBox='0 0 100 98.1' style='enable-background:new 0 0 100 98.1;' xml:space='preserve' height='100' width='98.1'><path style='fill:#FFFFFF;' d='M69.09,0.09c-4.01-0.6-7.83,1.94-8.78,5.89l-3.14,13.11c-0.76,3.19-3.44,5.56-6.7,5.93l-18.6,2.13          C11.83,29.31-2.53,47.66,0.37,67.78c2.3,15.96,15.18,27.77,30.54,30.08c4.01,0.6,7.83-1.94,8.78-5.89l3.14-13.11 c0.76-3.19,3.44-5.56,6.69-5.93l18.6-2.13c20.04-2.17,34.41-20.52,31.5-40.65C97.32,14.21,84.45,2.39,69.09,0.09z M65.06,47.2  l-11.75,1.4c-1.98,0.21-3.62,1.65-4.08,3.59l-3.43,14.3c-1.01,4.22-4.5,7.39-8.8,8c-6.66,0.72-12.63-4.2-13.15-10.92                c-0.51-6.5,4.6-12.11,11.08-12.81l11.75-1.39c1.98-0.22,3.62-1.65,4.08-3.59l3.43-14.3c1.01-4.22,4.5-7.39,8.8-8          c6.66-0.72,12.63,4.19,13.15,10.92C76.66,40.89,71.55,46.5,65.06,47.2z'/></svg></a>").insertBefore(".header__wrapper.clearfix .journal-logos:eq(0)");
}


</script></div>

            

        

    

        


    

            

            

            <div data-widget-id="1af759b7-9bb6-4413-ada9-51a8333e8fb8" data-widget-name="UX3HTMLWidget"><style>
    /*This was added temporarily until the PB changelog tab issue got fixed. [LIT-444969 - c#4]*/
    [data-tab-id="changelogTab"] {
        display: none !important;
    }
    
    /*This was added temporarily until the PB changelog tab issue got fixed. [LIT-518839]*/
    @media (min-width: 769px) {
        .ux3 .quick-search .dropBlock [data-db-target-of],
        .quick-search .dropBlock [data-db-target-of] {
            width: 100vw !important;
        }
    }
</style></div>

            

        
</div></div>

            

        

        </div>
    

    

        


    

            

            

            <div data-widget-id="d7e0813b-a300-497a-ab37-9c3ddaf03e71" data-widget-name="UX3HTMLWidget"><!--
<div class="session-reminder hidden">
    <div class="session-reminder__overlay">
        <div class="session-reminder__dialog">
            <div class="session-reminder__dialog-inner">
                <div class="dialog-header" role="dialog" aria-labelledby="dialog-title" aria-describedby="dialog-desc">
                    <h2 class="dialog-header-title" id="dialog-title">
                        <span>Session Timeout</span>
                        <div class="dialog-timer">(2:00)</div>
                    </h2>
                    <button class="dialog-close" type="button" aria-label="close dialog">
                        <i class="icon-tools_close" aria-hidden="true"></i>
                    </button>
                </div>
                <div class="dialog-body">
                    <p class="dialog-desc" id="dialog-desc">Your session will expire shortly. If you are still working, click the ‘Keep Me Logged In’ button below. If you do not respond within the next minute, you will be automatically logged out.</p>
                    <p class="dialog-expired sr-only" aria-live="polite"></p>
                </div>
                <div class="dialog-footer">
                    <button class="session-reset" type="button">Keep Me Logged In</button>
                </div>
            </div>
        </div>
    </div>
</div>
<a class="sr-only" href="javascript:void(0)" aria-hidden="true"></a>
--></div>

            

        
        </div>
    </div>


































    



















        
        
            
                <script src="/products/marlin/cell/releasedAssets/js/main.bundle-90475670808d5d3eee41.js"></script>
            
        
        
        

    




<script type="text/javascript">
    var pageData =  {"journal":{"specialty":"Cell Press","issn":"0092-8674","name":"Cell"},"page":{"loadTimestamp":"1749592698296","server":"cell","environment":"prod","businessUnit":"ELS:RAP:JBS","noTracking":"","name":"fulltext:html","section":"Cell","language":"en","type":"CP-CA","productName":"JB"},"visitor":{"departmentName":"|ae:ScienceDirect Guests","accessType":"ae:ANON_GUEST|ae:ANON_GUEST","accountId":"ae:838949|ae:228598","accountName":"ae:Cell Guest Account|ae:ScienceDirect Guests","departmentId":"ae:977212|ae:291352","ipAddress":"189.120.74.179","loginStatus":"anonymous","userId":"ae:246228266|ae:12975512"},"content":[{"accessType":"article:free:openarchive","volumeNumber":"185","issueNumber":"5","viewState":"fulltext","format":"MIME-HTML","id":"pii:S0092867422000794","coverDate":"202203","type":"XOCS-JOURNAL:SCOPE-FULL","title":"Transcription factor protein interactomes reveal genetic determinants in heart disease","status":"S300"}]};
</script>

<script type="text/javascript">
    if(typeof pageDataTracker !== "undefined") {
        pageDataTracker.trackPageLoad();
    }
</script>
<script>(function(){function c(){var b=a.contentDocument||a.contentWindow.document;if(b){var d=b.createElement('script');d.innerHTML="window.__CF$cv$params={r:'94dc211368ad4ee3',t:'MTc0OTU5MjY5Ny4wMDAwMDA='};var a=document.createElement('script');a.nonce='';a.src='/cdn-cgi/challenge-platform/scripts/jsd/main.js';document.getElementsByTagName('head')[0].appendChild(a);";b.getElementsByTagName('head')[0].appendChild(d)}}if(document.body){var a=document.createElement('iframe');a.height=1;a.width=1;a.style.position='absolute';a.style.top=0;a.style.left=0;a.style.border='none';a.style.visibility='hidden';document.body.appendChild(a);if('loading'!==document.readyState)c();else if(window.addEventListener)document.addEventListener('DOMContentLoaded',c);else{var e=document.onreadystatechange||function(){};document.onreadystatechange=function(b){e(b);'loading'!==document.readyState&&(document.onreadystatechange=e,c())}}}})();</script><iframe height="1" width="1" style="position: absolute; top: 0px; left: 0px; border: none; visibility: hidden;"></iframe><script defer="" src="https://static.cloudflareinsights.com/beacon.min.js/vcd15cbe7772f49c399c6a5babf22c1241717689176015" integrity="sha512-ZpsOmlRQV6y907TI0dKBHq9Md29nnaEIPlkf84rnaERnq6zvWvPUqr2ft8M1aS28oN72PdrCzSjY4U6VaAw1EQ==" data-cf-beacon="{&quot;rayId&quot;:&quot;94dc211368ad4ee3&quot;,&quot;serverTiming&quot;:{&quot;name&quot;:{&quot;cfExtPri&quot;:true,&quot;cfEdge&quot;:true,&quot;cfOrigin&quot;:true,&quot;cfL4&quot;:true,&quot;cfSpeedBrain&quot;:true,&quot;cfCacheStatus&quot;:true}},&quot;version&quot;:&quot;2025.6.2&quot;,&quot;token&quot;:&quot;57a02315956f4b2e82817b3dfa99b1a2&quot;}" crossorigin="anonymous"></script>


<div id="addtoany" style="position: static;"><div class="a2a_overlay" id="a2a_overlay"></div><div id="a2a_modal" class="a2a_modal a2a_hide" role="dialog" tabindex="-1" aria-label=""><div class="a2a_modal_body a2a_menu a2a_hide" id="a2a_copy_link"><label for="a2a_copy_link_text" id="a2a_copy_link_icon" class="a2a_svg a2a_s_link a2a_logo_color"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"><title>Copy link</title><path fill="#FFF" d="M7.591 21.177c0-.36.126-.665.377-.917l2.804-2.804a1.235 1.235 0 0 1 .913-.378c.377 0 .7.144.97.43-.026.028-.11.11-.255.25-.144.14-.24.236-.29.29a2.82 2.82 0 0 0-.2.256 1.056 1.056 0 0 0-.177.344 1.43 1.43 0 0 0-.046.37c0 .36.126.666.377.918a1.25 1.25 0 0 0 .918.377c.126.001.251-.015.373-.047.125-.037.242-.096.345-.175.09-.06.176-.127.256-.2.1-.094.196-.19.29-.29.14-.142.223-.23.25-.254.297.28.445.607.445.984 0 .36-.126.664-.377.916l-2.778 2.79a1.242 1.242 0 0 1-.917.364c-.36 0-.665-.118-.917-.35l-1.982-1.97a1.223 1.223 0 0 1-.378-.9l-.001-.004Zm9.477-9.504c0-.36.126-.665.377-.917l2.777-2.79a1.235 1.235 0 0 1 .913-.378c.35 0 .656.12.917.364l1.984 1.968c.254.252.38.553.38.903 0 .36-.126.665-.38.917l-2.802 2.804a1.238 1.238 0 0 1-.916.364c-.377 0-.7-.14-.97-.418.026-.027.11-.11.255-.25a7.5 7.5 0 0 0 .29-.29c.072-.08.139-.166.2-.255.08-.103.14-.22.176-.344.032-.12.048-.245.047-.37 0-.36-.126-.662-.377-.914a1.247 1.247 0 0 0-.917-.377c-.136 0-.26.015-.37.046-.114.03-.23.09-.346.175a3.868 3.868 0 0 0-.256.2c-.054.05-.15.148-.29.29-.14.146-.222.23-.25.258-.294-.278-.442-.606-.442-.983v-.003ZM5.003 21.177c0 1.078.382 1.99 1.146 2.736l1.982 1.968c.745.75 1.658 1.12 2.736 1.12 1.087 0 2.004-.38 2.75-1.143l2.777-2.79c.75-.747 1.12-1.66 1.12-2.737 0-1.106-.392-2.046-1.183-2.818l1.186-1.185c.774.79 1.708 1.186 2.805 1.186 1.078 0 1.995-.376 2.75-1.13l2.803-2.81c.751-.754 1.128-1.671 1.128-2.748 0-1.08-.382-1.993-1.146-2.738L23.875 6.12C23.13 5.372 22.218 5 21.139 5c-1.087 0-2.004.382-2.75 1.146l-2.777 2.79c-.75.747-1.12 1.66-1.12 2.737 0 1.105.392 2.045 1.183 2.817l-1.186 1.186c-.774-.79-1.708-1.186-2.805-1.186-1.078 0-1.995.377-2.75 1.132L6.13 18.426c-.754.755-1.13 1.672-1.13 2.75l.003.001Z"></path></svg></label><input id="a2a_copy_link_text" type="text" title="Copy link" readonly=""><div id="a2a_copy_link_copied">✓</div></div><div class="a2a_modal_body a2a_menu a2a_thanks a2a_hide" id="a2a_thanks"><div class="a2a_localize" data-a2a-localize="inner,ThanksForSharing">Thanks for sharing!</div></div></div><div class="a2a_menu a2a_full a2a_localize" id="a2apage_full" role="dialog" tabindex="-1" aria-label="Share" data-a2a-localize="title,Share"><div class="a2a_full_header"><div id="a2apage_find_container" class="a2a_menu_find_container"><label for="a2apage_find" id="a2apage_find_icon" class="a2a_svg a2a_s_find"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" aria-hidden="true"><title>Find any service</title><path fill="#CCC" d="M19.7 18.2l-4.5-4.5c.7-1.1 1.2-2.3 1.2-3.6 0-3.5-2.8-6.3-6.3-6.3s-6.3 2.8-6.3 6.3 2.8 6.3 6.3 6.3c1.4 0 2.6-.4 3.6-1.2l4.5 4.5c.6.6 1.3.7 1.7.2.5-.4.4-1.1-.2-1.7zm-9.6-3.6c-2.5 0-4.5-2.1-4.5-4.5 0-2.5 2.1-4.5 4.5-4.5 2.5 0 4.5 2.1 4.5 4.5s-2 4.5-4.5 4.5z"></path></svg></label><input id="a2apage_find" class="a2a_menu_find a2a_localize" type="text" autocomplete="off" title="Find any service" data-a2a-localize="title,FindAnyServiceToAddTo"></div></div><div class="a2a_full_services" id="a2apage_full_services" role="presentation"></div><div class="a2a_full_footer"><a href="https://www.addtoany.com" title="Share Buttons" rel="noopener" target="_blank"><span class="a2a_svg a2a_s__default a2a_s_a2a a2a_logo_color"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"><g fill="#FFF"><path d="M14 7h4v18h-4z"></path><path d="M7 14h18v4H7z"></path></g></svg></span>AddToAny</a></div></div><div id="a2apage_dropdown" class="a2a_menu a2a_mini a2a_localize a2a_hide" tabindex="-1" aria-label="Share" data-a2a-localize="label,Share"><div class="a2a_mini_services" id="a2apage_mini_services"></div><div id="a2apage_cols_container" class="a2a_cols_container"><div class="a2a_col1" id="a2apage_col1"></div><div id="a2apage_2_col1" class="a2a_hide"></div><div class="a2a_clear"></div></div><div class="a2apage_wide a2a_wide"><a href="#addtoany" id="a2apage_show_more_less" class="a2a_more a2a_localize" title="Show all" data-a2a-localize="title,ShowAll"><span class="a2a_svg a2a_s__default a2a_s_a2a a2a_logo_color"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"><g fill="#FFF"><path d="M14 7h4v18h-4z"></path><path d="M7 14h18v4H7z"></path></g></svg></span><span class="a2a_localize" data-a2a-localize="inner,More">More…</span></a></div></div><div style="height: 1px; width: 1px; position: absolute; z-index: 100000; top: 0px; visibility: hidden;"><iframe id="a2a_sm_ifr" title="AddToAny Utility Frame" aria-hidden="true" src="https://static.addtoany.com/menu/sm.25.html#type=core&amp;event=load" style="height: 1px; width: 1px; border: 0px; left: 0px; top: 0px; position: absolute; z-index: 100000; display: none;"></iframe></div></div><div role="status" aria-live="assertive" aria-relevant="additions" class="ui-helper-hidden-accessible"></div><div id="crossmark-widget" style="display: none;"><div class="crossmark-reset crossmark-overlay"></div><div class="crossmark-reset crossmark-popup"><div class="crossmark-reset crossmark-popup__offset"><div class="crossmark-reset crossmark-popup__inner"><div class="crossmark-reset crossmark-popup__header"><button class="crossmark-reset crossmark-popup__btn-close"><span class="sr-only">__("articleCrossmark.closePopup")</span></button></div><div class="crossmark-reset crossmark-popup__content-wrapper"><iframe class="crossmark-reset crossmark-popup__content" title="Crossmark service - Intentionally blank" aria-hidden="false" tabindex="0" src="https://crossmark.crossref.org/dialog?doi=10.1016%2Fj.cell.2022.01.021&amp;domain=www.cell.com&amp;uri_scheme=https%3A&amp;cm_version=v2.0"></iframe></div></div></div></div></div><div id="onetrust-consent-sdk" data-nosnippet="true"><div class="onetrust-pc-dark-filter ot-fade-in" style="z-index:2147483645;"></div><div id="onetrust-banner-sdk" class="otFlat bottom ot-wo-title ot-buttons-fw" tabindex="0" role="region" aria-label="Cookie banner"><div role="dialog" aria-modal="true" aria-label="Privacy"><div class="ot-sdk-container"><div class="ot-sdk-row"><div id="onetrust-group-container" class="ot-sdk-eight ot-sdk-columns"><div class="banner_logo"></div><div id="onetrust-policy"><div id="onetrust-policy-text">We use cookies that are necessary to make our site work. We may also use additional cookies to analyze, improve, and personalize our content and your digital experience. For more information, see our<a class="ot-cookie-policy-link" href="https://www.elsevier.com/legal/cookienotice" target="_blank" aria-label=", opens in a new tab" rel="noopener">Cookie Policy</a></div></div></div><div id="onetrust-button-group-parent" class="ot-sdk-three ot-sdk-columns"><div id="onetrust-button-group"><button id="onetrust-pc-btn-handler" class="cookie-setting-link">Cookie Settings</button>  <button id="onetrust-accept-btn-handler">Accept all cookies</button></div></div></div></div><!-- Close Button --><div id="onetrust-close-btn-container"></div><!-- Close Button END--></div></div><div id="onetrust-pc-sdk" class="otPcCenter ot-hide ot-fade-in" lang="en" aria-label="Preference center" role="region"><div role="dialog" aria-modal="true" style="height: 100%;" aria-label="Cookie Preference Center"><!-- Close Button --><div class="ot-pc-header"><!-- Logo Tag --><div class="ot-pc-logo" role="img" aria-label="Company Logo"><img alt="Company Logo" src="https://cdn.cookielaw.org/logos/static/ot_company_logo.png"></div></div><!-- Close Button --><div id="ot-pc-content" class="ot-pc-scrollbar"><div class="ot-optout-signal ot-hide"><div class="ot-optout-icon"><svg xmlns="http://www.w3.org/2000/svg"><path class="ot-floating-button__svg-fill" d="M14.588 0l.445.328c1.807 1.303 3.961 2.533 6.461 3.688 2.015.93 4.576 1.746 7.682 2.446 0 14.178-4.73 24.133-14.19 29.864l-.398.236C4.863 30.87 0 20.837 0 6.462c3.107-.7 5.668-1.516 7.682-2.446 2.709-1.251 5.01-2.59 6.906-4.016zm5.87 13.88a.75.75 0 00-.974.159l-5.475 6.625-3.005-2.997-.077-.067a.75.75 0 00-.983 1.13l4.172 4.16 6.525-7.895.06-.083a.75.75 0 00-.16-.973z" fill="#FFF" fill-rule="evenodd"></path></svg></div><span></span></div><h2 id="ot-pc-title">Cookie Preference Center</h2><div id="ot-pc-desc">We use cookies which are necessary to make our site work. We may also use additional cookies to analyse, improve and personalise our content and your digital experience. For more information, see our <a href="https://www.elsevier.com/legal/cookienotice/_nocache" target="_blank">Cookie Policy</a> and the list of <a href="https://support.google.com/admanager/answer/9012903" target="_blank">Google Ad-Tech Vendors</a>.
<br>
<br>
You may choose not to allow some types of cookies. However, blocking some types may impact your experience of our site and the services we are able to offer. See the different category headings below to find out more or change your settings.
<br>
</div><button id="accept-recommended-btn-handler">Allow all</button><section class="ot-sdk-row ot-cat-grp"><h3 id="ot-category-title"> Manage Consent Preferences</h3><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="1"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-1" aria-labelledby="ot-header-id-1 ot-status-id-1"></button><!-- Accordion header --><div class="ot-acc-hdr ot-always-active-group"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-1">Strictly Necessary Cookies</h4><div id="ot-status-id-1" class="ot-always-active">Always active</div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-1">These cookies are necessary for the website to function and cannot be switched off in our systems. They are usually only set in response to actions made by you which amount to a request for services, such as setting your privacy preferences, logging in or filling in forms. You can set your browser to block or alert you about these cookies, but some parts of the site will not then work. These cookies do not store any personally identifiable information.
<br><br></p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="1">Cookie Details List‎</button></div></div></div><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="3"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-3" aria-labelledby="ot-header-id-3"></button><!-- Accordion header --><div class="ot-acc-hdr"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-3">Functional Cookies</h4><div class="ot-tgl"><input type="checkbox" name="ot-group-id-3" id="ot-group-id-3" role="switch" class="category-switch-handler" data-optanongroupid="3" aria-labelledby="ot-header-id-3"> <label class="ot-switch" for="ot-group-id-3"><span class="ot-switch-nob"></span> <span class="ot-label-txt">Functional Cookies</span></label> </div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-3">These cookies enable the website to provide enhanced functionality and personalisation. They may be set by us or by third party providers whose services we have added to our pages. If you do not allow these cookies then some or all of these services may not function properly.</p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="3">Cookie Details List‎</button></div></div></div><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="2"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-2" aria-labelledby="ot-header-id-2"></button><!-- Accordion header --><div class="ot-acc-hdr"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-2">Performance Cookies</h4><div class="ot-tgl"><input type="checkbox" name="ot-group-id-2" id="ot-group-id-2" role="switch" class="category-switch-handler" data-optanongroupid="2" aria-labelledby="ot-header-id-2"> <label class="ot-switch" for="ot-group-id-2"><span class="ot-switch-nob"></span> <span class="ot-label-txt">Performance Cookies</span></label> </div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-2">These cookies allow us to count visits and traffic sources so we can measure and improve the performance of our site. They help us to know which pages are the most and least popular and see how visitors move around the site.</p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="2">Cookie Details List‎</button></div></div></div><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="4"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-4" aria-labelledby="ot-header-id-4"></button><!-- Accordion header --><div class="ot-acc-hdr"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-4">Targeting Cookies</h4><div class="ot-tgl"><input type="checkbox" name="ot-group-id-4" id="ot-group-id-4" role="switch" class="category-switch-handler" data-optanongroupid="4" aria-labelledby="ot-header-id-4"> <label class="ot-switch" for="ot-group-id-4"><span class="ot-switch-nob"></span> <span class="ot-label-txt">Targeting Cookies</span></label> </div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-4">These cookies may be set through our site by our advertising partners. They may be used by those companies to build a profile of your interests and show you relevant adverts on other sites. If you do not allow these cookies, you will experience less targeted advertising.</p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="4">Cookie Details List‎</button></div></div></div><!-- Groups sections starts --><!-- Group section ends --><!-- Accordion Group section starts --><!-- Accordion Group section ends --></section></div><section id="ot-pc-lst" class="ot-hide ot-hosts-ui ot-pc-scrollbar"><div id="ot-pc-hdr"><div id="ot-lst-title"><button class="ot-link-btn back-btn-handler" aria-label="Back"><svg id="ot-back-arw" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 444.531 444.531" xml:space="preserve"><title>Back Button</title><g><path fill="#656565" d="M213.13,222.409L351.88,83.653c7.05-7.043,10.567-15.657,10.567-25.841c0-10.183-3.518-18.793-10.567-25.835
                    l-21.409-21.416C323.432,3.521,314.817,0,304.637,0s-18.791,3.521-25.841,10.561L92.649,196.425
                    c-7.044,7.043-10.566,15.656-10.566,25.841s3.521,18.791,10.566,25.837l186.146,185.864c7.05,7.043,15.66,10.564,25.841,10.564
                    s18.795-3.521,25.834-10.564l21.409-21.412c7.05-7.039,10.567-15.604,10.567-25.697c0-10.085-3.518-18.746-10.567-25.978
                    L213.13,222.409z"></path></g></svg></button><h3>Cookie List</h3></div><div class="ot-lst-subhdr"><div class="ot-search-cntr"><p role="status" class="ot-scrn-rdr"></p><input id="vendor-search-handler" type="text" name="vendor-search-handler" placeholder="Search…" aria-label="Cookie list search"> <svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 -30 110 110" aria-hidden="true"><title>Search Icon</title><path fill="#2e3644" d="M55.146,51.887L41.588,37.786c3.486-4.144,5.396-9.358,5.396-14.786c0-12.682-10.318-23-23-23s-23,10.318-23,23
            s10.318,23,23,23c4.761,0,9.298-1.436,13.177-4.162l13.661,14.208c0.571,0.593,1.339,0.92,2.162,0.92
            c0.779,0,1.518-0.297,2.079-0.837C56.255,54.982,56.293,53.08,55.146,51.887z M23.984,6c9.374,0,17,7.626,17,17s-7.626,17-17,17
            s-17-7.626-17-17S14.61,6,23.984,6z"></path></svg></div><div class="ot-fltr-cntr"><button id="filter-btn-handler" aria-label="Filter" aria-haspopup="true"><svg role="presentation" aria-hidden="true" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 402.577 402.577" xml:space="preserve"><title>Filter Icon</title><g><path fill="#fff" d="M400.858,11.427c-3.241-7.421-8.85-11.132-16.854-11.136H18.564c-7.993,0-13.61,3.715-16.846,11.136
      c-3.234,7.801-1.903,14.467,3.999,19.985l140.757,140.753v138.755c0,4.955,1.809,9.232,5.424,12.854l73.085,73.083
      c3.429,3.614,7.71,5.428,12.851,5.428c2.282,0,4.66-0.479,7.135-1.43c7.426-3.238,11.14-8.851,11.14-16.845V172.166L396.861,31.413
      C402.765,25.895,404.093,19.231,400.858,11.427z"></path></g></svg></button></div><div id="ot-anchor"></div><section id="ot-fltr-modal"><div id="ot-fltr-cnt"><button id="clear-filters-handler">Clear</button><div class="ot-fltr-scrlcnt ot-pc-scrollbar"><div class="ot-fltr-opts" role="list"><div class="ot-fltr-opt" role="listitem"><div class="ot-chkbox"><input id="chkbox-id" type="checkbox" class="category-filter-handler"> <label for="chkbox-id"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div></div></div><div class="ot-fltr-btns"><button id="filter-apply-handler">Apply</button> <button id="filter-cancel-handler">Cancel</button></div></div></div></section></div></div><section id="ot-lst-cnt" class="ot-host-cnt ot-pc-scrollbar"><div id="ot-sel-blk"><div class="ot-sel-all"><div class="ot-sel-all-hdr"><span class="ot-consent-hdr">Consent</span> <span class="ot-li-hdr">Leg.Interest</span></div><div class="ot-sel-all-chkbox"><div class="ot-chkbox" id="ot-selall-hostcntr"><input id="select-all-hosts-groups-handler" type="checkbox"> <label for="select-all-hosts-groups-handler"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div><div class="ot-chkbox" id="ot-selall-vencntr"><input id="select-all-vendor-groups-handler" type="checkbox"> <label for="select-all-vendor-groups-handler"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div><div class="ot-chkbox" id="ot-selall-licntr"><input id="select-all-vendor-leg-handler" type="checkbox"> <label for="select-all-vendor-leg-handler"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div></div></div></div><div class="ot-sdk-row"><div class="ot-sdk-column"><ul id="ot-host-lst"></ul></div></div></section></section><div class="ot-pc-footer ot-pc-scrollbar"><div class="ot-btn-container"> <button class="save-preference-btn-handler onetrust-close-btn-handler">Confirm my choices</button></div><!-- Footer logo --><div class="ot-pc-footer-logo"><a href="https://www.onetrust.com/products/cookie-consent/" target="_blank" rel="noopener noreferrer" aria-label="Powered by OneTrust Opens in a new Tab"><img alt="Powered by Onetrust" src="https://cdn.cookielaw.org/logos/static/powered_by_logo.svg" title="Powered by OneTrust Opens in a new Tab"></a></div></div><!-- Cookie subgroup container --><!-- Vendor list link --><!-- Cookie lost link --><!-- Toggle HTML element --><!-- Checkbox HTML --><!-- plus minus--><!-- Arrow SVG element --><!-- Accordion basic element --><span class="ot-scrn-rdr" aria-atomic="true" aria-live="polite"></span><!-- Vendor Service container and item template --></div><iframe class="ot-text-resize" sandbox="allow-same-origin" title="onetrust-text-resize" style="position: absolute; top: -50000px; width: 100em;" aria-hidden="true"></iframe></div></div></body></html>