<html lang="en" class="pb-page fonts-loaded fonts-loaded" data-request-id="69d462e3-135d-49a9-a21e-63bbfc3be62d" style="--article-sticky-top: 82.00019999999999px; --scroll-padding: 163.594px auto auto;"><head data-pb-dropzone="head"><meta name="pbContext" content=";product:product:elsevier\:product\:cell;article:article:pii\:S0092867421013374;issue:issue:pii\:S0092867420X00274;website:website:cell-site;ctype:string:Journal Content;journal:journal:cell;pageGroup:string:Article View;page:string:Show Full Text;requestedJournal:journal:cell;wgroup:string:Default Website Group;subPage:string:Full Text">




<!-- if there is any customization for Responsive Project widget  -->


























    
    







    
            
        



        
                <meta name="robots" content="noarchive,nocache">
            
    






    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=1, user-scalable=1">











        <meta property="og:title" content="A transcriptional rheostat couples past activity to future sensory responses">
    


        <meta property="og:type" content="Article">
    


        <meta property="og:url" content="https://www.cell.com/cell/abstract/S0092-8674(21)01337-4">
    



        <meta property="og:site_name" content="Cell">
    


        <meta property="og:image" content="https://www.cell.com/cms/asset/e8773704-36aa-49dd-89f3-453586e8cd4f/fx1.jpg">
    


        <meta property="og:description" content="Olfactory sensory neurons adaptively shape their odor responses via environment-dependent
changes in gene expression, thereby separating salient olfactory cues from predictable
background.">
    











        <meta name="twitter:title" content="A transcriptional rheostat couples past activity to future sensory responses">
    




        <meta name="twitter:image" content="https://www.cell.com/cms/asset/e8773704-36aa-49dd-89f3-453586e8cd4f/fx1.jpg">
    


        <meta name="twitter:description" content="Olfactory sensory neurons adaptively shape their odor responses via environment-dependent
changes in gene expression, thereby separating salient olfactory cues from predictable
background.">
    

        <meta name="twitter:card" content="summary_large_image">
    











    
    
            
            
        <link rel="canonical" href="https://www.cell.com/cell/fulltext/S0092-8674(21)01337-4">
    


<meta name="tdm-reservation" content="1">
<meta name="tdm-policy" content="https://www.elsevier.com/tdm/tdmrep-policy.json">

<title>A transcriptional rheostat couples past activity to future sensory responses: Cell</title><meta name="citation_journal_title" content="Cell"> <meta name="citation_journal_abbrev" content="Cell"> <meta name="citation_publisher" content="Elsevier"> <meta name="citation_title" content="A transcriptional rheostat couples past activity to future sensory responses"> <meta name="citation_date" content="2021/12/22"> <meta name="citation_online_date" content="2021/12/07"> <meta name="citation_volume" content="184"> <meta name="citation_issue" content="26"> <meta name="citation_firstpage" content="6326"> <meta name="citation_lastpage" content="6343.e32"> <meta name="citation_pmid" content="34879231"> <meta name="citation_abstract_html_url" content="https://www.cell.com/action/showAbstract?pii=S0092867421013374"> <meta name="citation_fulltext_html_url" content="https://www.cell.com/action/showFullText?pii=S0092867421013374"> <meta name="citation_pdf_url" content="https://www.cell.com/action/showPdf?pii=S0092867421013374"> <meta name="citation_issn" content="0092-8674"> <meta name="citation_issn" content="1097-4172"> <meta name="citation_language" content="English"> <meta name="citation_doi" content="10.1016/j.cell.2021.11.022"> <meta name="citation_pmid" content="34879231"> <meta name="citation_author" content="Tatsuya Tsukahara"><meta name="citation_author_orcid" content="0000-0002-6928-8562"><meta name="citation_author_institution" content="Department of Neurobiology, Harvard Medical School, Boston, MA 02115, USA"><meta name="citation_author" content="David H. Brann"><meta name="citation_author_orcid" content="0000-0001-8844-8886"><meta name="citation_author_institution" content="Department of Neurobiology, Harvard Medical School, Boston, MA 02115, USA"><meta name="citation_author" content="Stan L. Pashkovski"><meta name="citation_author_institution" content="Department of Neurobiology, Harvard Medical School, Boston, MA 02115, USA"><meta name="citation_author" content="Grigori Guitchounts"><meta name="citation_author_orcid" content="0000-0003-0142-0149"><meta name="citation_author_institution" content="Department of Neurobiology, Harvard Medical School, Boston, MA 02115, USA"><meta name="citation_author" content="Thomas Bozza"><meta name="citation_author_institution" content="Department of Neurobiology, Northwestern University, Evanston, IL 60208, USA"><meta name="citation_author" content="Sandeep Robert Datta"><meta name="citation_author_email" content="srdatta@hms.harvard.edu"><meta name="citation_author_orcid" content="0000-0002-8068-3862"><meta name="citation_author_institution" content="Department of Neurobiology, Harvard Medical School, Boston, MA 02115, USA"><meta name="citation_keywords" content="olfaction; sensory neurons; homeostasis; transcription; adaptation; single-cell RNA sequencing; gene expression programs; odor coding; functional imaging; odorant receptor; Act-seq"><meta name="citation_reference" content="citation_title=SCENIC: Single-cell regulatory network inference and clustering; citation_author=S. Aibar; citation_author=C.B. González-Blas; citation_author=T. Moerman; citation_author=V.A. Huynh-Thu; citation_author=H. Imrichova; citation_author=G. Hulselmans; citation_author=F. Rambow; citation_author=J.-C. Marine; citation_author=P. Geurts; citation_author=J. Aerts; citation_author=et al.; citation_journal_title=Nat. Methods; citation_volume=14; citation_firstpage=1083; citation_lastpage=1086; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Stimulus dependent diversity and stereotypy in the output of an olfactory functional unit; citation_author=E.M. Arneodo; citation_author=K.B. Penikis; citation_author=N. Rabinowitz; citation_author=A. Licata; citation_author=A. Cichy; citation_author=J. Zhang; citation_author=T. Bozza; citation_author=D. Rinberg; citation_journal_title=Nat. Commun.; citation_volume=9; citation_firstpage=1347; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Some informational aspects of visual perception; citation_author=F. Attneave; citation_journal_title=Psychol. Rev.; citation_volume=61; citation_firstpage=183; citation_lastpage=193; citation_publication_date=1954; "><meta name="citation_reference" content="citation_title=Compensatory plasticity in the olfactory epithelium: Age, timing, and reversibility; citation_author=C.N. Barber; citation_author=D.M. Coppola; citation_journal_title=J.&nbsp;Neurophysiol.; citation_volume=114; citation_firstpage=2023; citation_lastpage=2032; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Possible principles underlying the transformation of sensory messages; citation_inbook_title=Sensory Communication; citation_author=H. Barlow; citation_firstpage=217; citation_lastpage=234; citation_publication_date=1961; citation_publisher=MIT press; "><meta name="citation_reference" content="citation_title=Neural adaptation; citation_author=J. Benda; citation_journal_title=Curr. Biol.; citation_volume=31; citation_firstpage=R110; citation_lastpage=R116; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=Cortical metabotropic glutamate receptors contribute to habituation of a simple odor-evoked behavior; citation_author=A.R. Best; citation_author=J.V. Thompson; citation_author=M.L. Fletcher; citation_author=D.A. Wilson; citation_journal_title=J.&nbsp;Neurosci.; citation_volume=25; citation_firstpage=2513; citation_lastpage=2517; citation_publication_date=2005; "><meta name="citation_reference" content="citation_title=Odorant receptor expression defines functional units in the mouse olfactory system; citation_author=T. Bozza; citation_author=P. Feinstein; citation_author=C. Zheng; citation_author=P. Mombaerts; citation_journal_title=J.&nbsp;Neurosci.; citation_volume=22; citation_firstpage=3033; citation_lastpage=3043; citation_publication_date=2002; "><meta name="citation_reference" content="citation_title=In&nbsp;vivo imaging of neuronal activity by targeted expression of a genetically encoded probe in the mouse; citation_author=T. Bozza; citation_author=J.P. McGann; citation_author=P. Mombaerts; citation_author=M. Wachowiak; citation_journal_title=Neuron; citation_volume=42; citation_firstpage=9; citation_lastpage=21; citation_publication_date=2004; "><meta name="citation_reference" content="citation_title=Mapping of class I and class II odorant receptors to glomerular domains by two distinct types of olfactory sensory neurons in the mouse; citation_author=T. Bozza; citation_author=A. Vassalli; citation_author=S. Fuss; citation_author=J.-J. Zhang; citation_author=B. Weiland; citation_author=R. Pacifico; citation_author=P. Feinstein; citation_author=P. Mombaerts; citation_journal_title=Neuron; citation_volume=61; citation_firstpage=220; citation_lastpage=233; citation_publication_date=2009; "><meta name="citation_reference" content="citation_title=Finding the brain in the nose; citation_author=D.H. Brann; citation_author=S.R. Datta; citation_journal_title=Annu. Rev. Neurosci.; citation_volume=43; citation_firstpage=277; citation_lastpage=295; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Information coding in the vertebrate olfactory system; citation_author=L.B. Buck; citation_journal_title=Annu. Rev. Neurosci.; citation_volume=19; citation_firstpage=517; citation_lastpage=544; citation_publication_date=1996; "><meta name="citation_reference" content="citation_title=Activation, deactivation, and adaptation in vertebrate photoreceptor cells; citation_author=M.E. Burns; citation_author=D.A. Baylor; citation_journal_title=Annu. Rev. Neurosci.; citation_volume=24; citation_firstpage=779; citation_lastpage=805; citation_publication_date=2001; "><meta name="citation_reference" content="citation_title=Postnatal odorant exposure induces peripheral olfactory plasticity at the cellular level; citation_author=H. Cadiou; citation_author=I. Aoudé; citation_author=B. Tazir; citation_author=A. Molinas; citation_author=C. Fenech; citation_author=N. Meunier; citation_author=X. Grosmaitre; citation_journal_title=J.&nbsp;Neurosci.; citation_volume=34; citation_firstpage=4857; citation_lastpage=4870; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Parallel encoding of sensory history and behavioral preference during Caenorhabditis elegans olfactory learning; citation_author=C.E. Cho; citation_author=C. Brueggemann; citation_author=N.D. L’Etoile; citation_author=C.I. Bargmann; citation_journal_title=eLife; citation_volume=5; citation_firstpage=e14000; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Manipulating synthetic optogenetic odors reveals the coding logic of olfactory perception; citation_author=E. Chong; citation_author=M. Moroni; citation_author=C. Wilson; citation_author=S. Shoham; citation_author=S. Panzeri; citation_author=D. Rinberg; citation_journal_title=Science; citation_volume=368; citation_firstpage=eaba2357; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Lack of pattern separation in sensory inputs to the olfactory bulb during perceptual learning; citation_author=M.W. Chu; citation_author=W.L. Li; citation_author=T. Komiyama; citation_journal_title=eNeuro; citation_volume=4; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Computation in the olfactory system; citation_author=T.A. Cleland; citation_author=C. Linster; citation_journal_title=Chem. Senses; citation_volume=30; citation_firstpage=801; citation_lastpage=813; citation_publication_date=2005; "><meta name="citation_reference" content="citation_title=The effects of unilateral naris occlusion on gene expression profiles in mouse olfactory mucosa; citation_author=D.M. Coppola; citation_author=C.T. Waggener; citation_journal_title=J.&nbsp;Mol. Neurosci.; citation_volume=47; citation_firstpage=604; citation_lastpage=618; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=Naris occlusion alters transductory protein immunoreactivity in olfactory epithelium; citation_author=D.M. Coppola; citation_author=A.M. Waguespack; citation_author=M.R. Reems; citation_author=M.L. Butman; citation_author=J.A. Cherry; citation_journal_title=Histol. Histopathol.; citation_volume=21; citation_firstpage=487; citation_lastpage=501; citation_publication_date=2006; "><meta name="citation_reference" content="citation_title=Homeostatic control of neural activity: From phenomenology to molecular design; citation_author=G.W. Davis; citation_journal_title=Annu. Rev. Neurosci.; citation_volume=29; citation_firstpage=307; citation_lastpage=323; citation_publication_date=2006; "><meta name="citation_reference" content="citation_title=Homeostatic signaling and the stabilization of neural function; citation_author=G.W. Davis; citation_journal_title=Neuron; citation_volume=80; citation_firstpage=718; citation_lastpage=728; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=Modulation of the combinatorial code of odorant receptor response patterns in odorant mixtures; citation_author=C.A. de March; citation_author=W.B. Titlow; citation_author=T. Sengoku; citation_author=P. Breheny; citation_author=H. Matsunami; citation_author=T.S. McClintock; citation_journal_title=Mol. Cell. Neurosci.; citation_volume=104; citation_firstpage=103469; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Cyclic regulation of sensory perception by a female hormone alters behavior; citation_author=S. Dey; citation_author=P. Chamero; citation_author=J.K. Pru; citation_author=M.-S. Chien; citation_author=X. Ibarra-Soria; citation_author=K.R. Spencer; citation_author=D.W. Logan; citation_author=H. Matsunami; citation_author=J.J. Peluso; citation_author=L. Stowers; citation_journal_title=Cell; citation_volume=161; citation_firstpage=1334; citation_lastpage=1344; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Mice cloned from olfactory sensory neurons; citation_author=K. Eggan; citation_author=K. Baldwin; citation_author=M. Tackett; citation_author=J. Osborne; citation_author=J. Gogos; citation_author=A. Chess; citation_author=R. Axel; citation_author=R. Jaenisch; citation_journal_title=Nature; citation_volume=428; citation_firstpage=44; citation_lastpage=49; citation_publication_date=2004; "><meta name="citation_reference" content="citation_title=A contextual model for axonal sorting into glomeruli in the mouse olfactory system; citation_author=P. Feinstein; citation_author=P. Mombaerts; citation_journal_title=Cell; citation_volume=117; citation_firstpage=817; citation_lastpage=831; citation_publication_date=2004; "><meta name="citation_reference" content="citation_title=Adaptation in auditory hair cells; citation_author=R. Fettiplace; citation_author=A.J. Ricci; citation_journal_title=Curr. Opin. Neurobiol.; citation_volume=13; citation_firstpage=446; citation_lastpage=451; citation_publication_date=2003; "><meta name="citation_reference" content="citation_title=How the olfactory system makes sense of scents; citation_author=S. Firestein; citation_journal_title=Nature; citation_volume=413; citation_firstpage=211; citation_lastpage=218; citation_publication_date=2001; "><meta name="citation_reference" content="citation_title=Activity-dependent genes in mouse olfactory sensory neurons; citation_author=A.M. Fischl; citation_author=P.M. Heron; citation_author=A.J. Stromberg; citation_author=T.S. McClintock; citation_journal_title=Chem. Senses; citation_volume=39; citation_firstpage=439; citation_lastpage=449; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Olfactory deprivation and enrichment: An identity of opposites?; citation_author=E. Fitzwater; citation_author=D.M. Coppola; citation_journal_title=Chem. Senses; citation_volume=46; citation_firstpage=bjaa071; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=Deconstructing olfactory stem cell trajectories at single-cell resolution; citation_author=R.B. Fletcher; citation_author=D. Das; citation_author=L. Gadye; citation_author=K.N. Street; citation_author=A. Baudhuin; citation_author=A. Wagner; citation_author=M.B. Cole; citation_author=Q. Flores; citation_author=Y.G. Choi; citation_author=N. Yosef; citation_author=et al.; citation_journal_title=Cell Stem Cell; citation_volume=20; citation_firstpage=817; citation_lastpage=830.e8; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Brief sensory deprivation triggers cell type-specific structural and functional plasticity in olfactory bulb neurons; citation_author=E. Galliano; citation_author=C. Hahn; citation_author=L.P.R. Browne; citation_author=P. R Villamayor; citation_author=C. Tufo; citation_author=A. Crespo; citation_author=M.S. Grubb; citation_journal_title=J.&nbsp;Neurosci.; citation_volume=41; citation_firstpage=2135; citation_lastpage=2151; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=Global structure, robustness, and modulation of neuronal models; citation_author=M.S. Goldman; citation_author=J. Golowasch; citation_author=E. Marder; citation_author=L.F. Abbott; citation_journal_title=J.&nbsp;Neurosci.; citation_volume=21; citation_firstpage=5229; citation_lastpage=5238; citation_publication_date=2001; "><meta name="citation_reference" content="citation_title=Integrated morphoelectric and transcriptomic classification of cortical GABAergic cells; citation_author=N.W. Gouwens; citation_author=S.A. Sorensen; citation_author=F. Baftizadeh; citation_author=A. Budzillo; citation_author=B.R. Lee; citation_author=T. Jarsky; citation_author=L. Alfiler; citation_author=K. Baker; citation_author=E. Barkan; citation_author=K. Berry; citation_author=et al.; citation_journal_title=Cell; citation_volume=183; citation_firstpage=935; citation_lastpage=953.e19; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=A probabilistic approach to demixing odors; citation_author=A. Grabska-Barwińska; citation_author=S. Barthelmé; citation_author=J. Beck; citation_author=Z.F. Mainen; citation_author=A. Pouget; citation_author=P.E. Latham; citation_journal_title=Nat. Neurosci.; citation_volume=20; citation_firstpage=98; citation_lastpage=106; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Homeostatic control of sensory output in basal vomeronasal neurons: activity-dependent expression of ether-à-go-go-related gene potassium channels; citation_author=S. Hagendorf; citation_author=D. Fluegge; citation_author=C. Engelhardt; citation_author=M. Spehr; citation_journal_title=J.&nbsp;Neurosci.; citation_volume=29; citation_firstpage=206; citation_lastpage=221; citation_publication_date=2009; "><meta name="citation_reference" content="citation_title=Single-cell transcriptomics reveals receptor transformations during olfactory neurogenesis; citation_author=N.K. Hanchate; citation_author=K. Kondoh; citation_author=Z. Lu; citation_author=D. Kuang; citation_author=X. Ye; citation_author=X. Qiu; citation_author=L. Pachter; citation_author=C. Trapnell; citation_author=L.B. Buck; citation_journal_title=Science; citation_volume=350; citation_firstpage=1251; citation_lastpage=1255; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Olfactory computation and object perception; citation_author=J.J. Hopfield; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=88; citation_firstpage=6462; citation_lastpage=6466; citation_publication_date=1991; "><meta name="citation_reference" content="citation_title=Concentration-dependent recruitment of mammalian odorant receptors; citation_author=X.S. Hu; citation_author=K. Ikegami; citation_author=A. Vihani; citation_author=K.W. Zhu; citation_author=M. Zapata; citation_author=C.A. de March; citation_author=M. Do; citation_author=N. Vaidya; citation_author=G. Kucera; citation_author=C. Bock; citation_author=et al.; citation_journal_title=eNeuro; citation_volume=7; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Variation in olfactory neuron repertoires is genetically controlled and environmentally modulated; citation_author=X. Ibarra-Soria; citation_author=T.S. Nakahara; citation_author=J. Lilue; citation_author=Y. Jiang; citation_author=C. Trimmer; citation_author=M.A. Souza; citation_author=P.H. Netto; citation_author=K. Ikegami; citation_author=N.R. Murphy; citation_author=M. Kusma; citation_author=et al.; citation_journal_title=eLife; citation_volume=6; citation_firstpage=1083; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Topographic mapping—The olfactory system; citation_author=T. Imai; citation_author=H. Sakano; citation_author=L.B. Vosshall; citation_journal_title=Cold Spring Harb. Perspect. Biol.; citation_volume=2; citation_firstpage=a001776; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=Molecular profiling of activated olfactory neurons identifies odorant receptors for odors in&nbsp;vivo; citation_author=Y. Jiang; citation_author=N.N. Gong; citation_author=X.S. Hu; citation_author=M.J. Ni; citation_author=R. Pasi; citation_author=H. Matsunami; citation_journal_title=Nat. Neurosci.; citation_volume=18; citation_firstpage=1446; citation_lastpage=1454; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Endogenous nuclear RNAi mediates behavioral adaptation to odor; citation_author=B.-T. Juang; citation_author=C. Gu; citation_author=L. Starnes; citation_author=F. Palladino; citation_author=A. Goga; citation_author=S. Kennedy; citation_author=N.D. L’Etoile; citation_journal_title=Cell; citation_volume=154; citation_firstpage=1010; citation_lastpage=1022; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=Olfactory cortical adaptation facilitates detection of odors against background; citation_author=M. Kadohisa; citation_author=D.A. Wilson; citation_journal_title=J.&nbsp;Neurophysiol.; citation_volume=95; citation_firstpage=1888; citation_lastpage=1896; citation_publication_date=2006; "><meta name="citation_reference" content="citation_title=Changes in Olfactory Sensory Neuron Physiology and Olfactory Perceptual Learning After Odorant Exposure in Adult Mice; citation_author=M.D. Kass; citation_author=S.A. Guang; citation_author=A.H. Moberly; citation_author=J.P. McGann; citation_journal_title=Chem. Senses; citation_volume=41; citation_firstpage=123; citation_lastpage=133; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Changes in the neural representation of odorants after olfactory deprivation in the adult mouse olfactory bulb; citation_author=M.D. Kass; citation_author=J. Pottackal; citation_author=D.J. Turkel; citation_author=J.P. McGann; citation_journal_title=Chem. Senses; citation_volume=38; citation_firstpage=77; citation_lastpage=89; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=Dynamic sensory representations in the olfactory bulb: Modulation by wakefulness and experience; citation_author=H.K. Kato; citation_author=M.W. Chu; citation_author=J.S. Isaacson; citation_author=T. Komiyama; citation_journal_title=Neuron; citation_volume=76; citation_firstpage=962; citation_lastpage=975; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=Multimodal analysis of cell types in a hypothalamic node controlling social behavior; citation_author=D.-W. Kim; citation_author=Z. Yao; citation_author=L.T. Graybuck; citation_author=T.-K. Kim; citation_author=T.N. Nguyen; citation_author=K.A. Smith; citation_author=O. Fong; citation_author=L. Yi; citation_author=N. Koulena; citation_author=N. Pierson; citation_author=et al.; citation_journal_title=Cell; citation_volume=179; citation_firstpage=713; citation_lastpage=728.e17; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Extraction of distinct neuronal cell types from within a genetically continuous population; citation_author=E.J. Kim; citation_author=Z. Zhang; citation_author=L. Huang; citation_author=T. Ito-Cole; citation_author=M.W. Jacobs; citation_author=A.L. Juavinett; citation_author=G. Senturk; citation_author=M. Hu; citation_author=M. Ku; citation_author=J.R. Ecker; citation_author=E.M. Callaway; citation_journal_title=Neuron; citation_volume=107; citation_firstpage=274; citation_lastpage=282.e6; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Innate versus learned odour processing in the mouse olfactory bulb; citation_author=K. Kobayakawa; citation_author=R. Kobayakawa; citation_author=H. Matsumoto; citation_author=Y. Oka; citation_author=T. Imai; citation_author=M. Ikawa; citation_author=M. Okabe; citation_author=T. Ikeda; citation_author=S. Itohara; citation_author=T. Kikusui; citation_author=K. Mori; citation_author=H. Sakano; citation_journal_title=Nature; citation_volume=450; citation_firstpage=503; citation_lastpage=508; citation_publication_date=2007; "><meta name="citation_reference" content="citation_title=Efficient olfactory coding in the pheromone receptor neuron of a moth; citation_author=L. Kostal; citation_author=P. Lansky; citation_author=J.-P. Rospars; citation_journal_title=PLoS Comput. Biol.; citation_volume=4; citation_firstpage=e1000053; citation_publication_date=2008; "><meta name="citation_reference" content="citation_title=Identifying gene expression programs of cell-type identity and cellular activity with single-cell RNA-seq; citation_author=D. Kotliar; citation_author=A. Veres; citation_author=M.A. Nagy; citation_author=S. Tabrizi; citation_author=E. Hodis; citation_author=D.A. Melton; citation_author=P.C. Sabeti; citation_journal_title=eLife; citation_volume=8; citation_firstpage=e43803; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=The cyclic GMP-dependent protein kinase EGL-4 regulates olfactory adaptation in C.&nbsp;elegans; citation_author=N.D. L’Etoile; citation_author=C.M. Coburn; citation_author=J. Eastham; citation_author=A. Kistler; citation_author=G. Gallegos; citation_author=C.I. Bargmann; citation_journal_title=Neuron; citation_volume=36; citation_firstpage=1079; citation_lastpage=1089; citation_publication_date=2002; "><meta name="citation_reference" content="citation_title=RNA velocity of single cells; citation_author=G. La Manno; citation_author=R. Soldatov; citation_author=A. Zeisel; citation_author=E. Braun; citation_author=H. Hochgerner; citation_author=V. Petukhov; citation_author=K. Lidschreiber; citation_author=M.E. Kastriti; citation_author=P. Lönnerberg; citation_author=A. Furlan; citation_author=et al.; citation_journal_title=Nature; citation_volume=560; citation_firstpage=494; citation_lastpage=498; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Peripheral adaptation codes for high odor concentration in glomeruli; citation_author=J. Lecoq; citation_author=P. Tiret; citation_author=S. Charpak; citation_journal_title=J.&nbsp;Neurosci.; citation_volume=29; citation_firstpage=3067; citation_lastpage=3072; citation_publication_date=2009; "><meta name="citation_reference" content="citation_title=Precise detection of direct glomerular input duration by the olfactory bulb; citation_author=A. Li; citation_author=D.H. Gire; citation_author=T. Bozza; citation_author=D. Restrepo; citation_journal_title=J.&nbsp;Neurosci.; citation_volume=34; citation_firstpage=16058; citation_lastpage=16064; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Classifying Drosophila olfactory projection neuron subtypes by single-cell RNA sequencing; citation_author=H. Li; citation_author=F. Horns; citation_author=B. Wu; citation_author=Q. Xie; citation_author=J. Li; citation_author=T. Li; citation_author=D.J. Luginbuhl; citation_author=S.R. Quake; citation_author=L. Luo; citation_journal_title=Cell; citation_volume=171; citation_firstpage=1206; citation_lastpage=1220.e22; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Optimal degrees of synaptic connectivity; citation_author=A. Litwin-Kumar; citation_author=K.D. Harris; citation_author=R. Axel; citation_author=H. Sompolinsky; citation_author=L.F. Abbott; citation_journal_title=Neuron; citation_volume=93; citation_firstpage=1153; citation_lastpage=1164.e7; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=A developmental switch of axon targeting in the continuously regenerating mouse olfactory system; citation_author=L. Ma; citation_author=Y. Wu; citation_author=Q. Qiu; citation_author=H. Scheerer; citation_author=A. Moran; citation_author=C.R. Yu; citation_journal_title=Science; citation_volume=344; citation_firstpage=194; citation_lastpage=197; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Variability, compensation, and modulation in neurons and circuits; citation_author=E. Marder; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=108; citation_number=Suppl 3; citation_firstpage=15542; citation_lastpage=15548; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=Variability, compensation and homeostasis in neuron and network function; citation_author=E. Marder; citation_author=J.-M. Goaillard; citation_journal_title=Nat. Rev. Neurosci.; citation_volume=7; citation_firstpage=563; citation_lastpage=574; citation_publication_date=2006; "><meta name="citation_reference" content="citation_title=Stimulus driven functional transformations in the early olfactory system; citation_author=C. Martelli; citation_author=D.A. Storace; citation_journal_title=Front. Cell. Neurosci.; citation_volume=15; citation_firstpage=684742; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=β-arrestin2-mediated internalization of mammalian odorant receptors; citation_author=A. Mashukova; citation_author=M. Spehr; citation_author=H. Hatt; citation_author=E.M. Neuhaus; citation_journal_title=J.&nbsp;Neurosci.; citation_volume=26; citation_firstpage=9902; citation_lastpage=9912; citation_publication_date=2006; "><meta name="citation_reference" content="McInnes, L., Healy, J., and Melville, J. (2018). UMAP: Uniform manifold approximation and projection for dimension reduction. arXiv, https://arxiv.org/pdf/1802.03426v3.pdf."><meta name="citation_reference" content="citation_title=Distinct neural mechanisms mediate olfactory memory formation at different timescales; citation_author=A.M. McNamara; citation_author=P.D. Magidson; citation_author=C. Linster; citation_author=D.A. Wilson; citation_author=T.A. Cleland; citation_journal_title=Learn. Mem.; citation_volume=15; citation_firstpage=117; citation_lastpage=125; citation_publication_date=2008; "><meta name="citation_reference" content="citation_title=Continuous and overlapping expression domains of odorant receptor genes in the olfactory epithelium determine the dorsal/ventral positioning of glomeruli in the olfactory bulb; citation_author=K. Miyamichi; citation_author=S. Serizawa; citation_author=H.M. Kimura; citation_author=H. Sakano; citation_journal_title=J.&nbsp;Neurosci.; citation_volume=25; citation_firstpage=3586; citation_lastpage=3592; citation_publication_date=2005; "><meta name="citation_reference" content="citation_title=Monoallelic expression of olfactory receptors; citation_author=K. Monahan; citation_author=S. Lomvardas; citation_journal_title=Annu. Rev. Cell Dev. Biol.; citation_volume=31; citation_firstpage=721; citation_lastpage=740; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Cooperative interactions enable singular olfactory receptor expression in mouse olfactory neurons; citation_author=K. Monahan; citation_author=I. Schieren; citation_author=J. Cheung; citation_author=A. Mumbey-Wafula; citation_author=E.S. Monuki; citation_author=S. Lomvardas; citation_journal_title=eLife; citation_volume=6; citation_firstpage=e28620; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=A model of the role of adaptation and disadaptation in olfactory receptor neurons: Implications for the coding of temporal and intensity patterns in odor signals; citation_author=P.A. Moore; citation_journal_title=Chem. Senses; citation_volume=19; citation_firstpage=71; citation_lastpage=86; citation_publication_date=1994; "><meta name="citation_reference" content="citation_title=How is the olfactory map formed and interpreted in the mammalian brain?; citation_author=K. Mori; citation_author=H. Sakano; citation_journal_title=Annu. Rev. Neurosci.; citation_volume=34; citation_firstpage=467; citation_lastpage=499; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=Biophysical mechanisms underlying olfactory receptor neuron dynamics; citation_author=K.I. Nagel; citation_author=R.I. Wilson; citation_journal_title=Nat. Neurosci.; citation_volume=14; citation_firstpage=208; citation_lastpage=216; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=Agonist-independent GPCR activity regulates anterior-posterior targeting of olfactory sensory neurons; citation_author=A. Nakashima; citation_author=H. Takeuchi; citation_author=T. Imai; citation_author=H. Saito; citation_author=H. Kiyonari; citation_author=T. Abe; citation_author=M. Chen; citation_author=L.S. Weinstein; citation_author=C.R. Yu; citation_author=D.R. Storm; citation_author=et al.; citation_journal_title=Cell; citation_volume=154; citation_firstpage=1314; citation_lastpage=1325; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=Amplification of Drosophila olfactory responses by a DEG/ENaC channel; citation_author=R. Ng; citation_author=S.S. Salem; citation_author=S.-T. Wu; citation_author=M. Wu; citation_author=H.-H. Lin; citation_author=A.K. Shepherd; citation_author=W.J. Joiner; citation_author=J.W. Wang; citation_author=C.-Y. Su; citation_journal_title=Neuron; citation_volume=104; citation_firstpage=947; citation_lastpage=959.e5; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Correlations in ion channel expression emerge from homeostatic tuning rules; citation_author=T. O’Leary; citation_author=A.H. Williams; citation_author=J.S. Caplan; citation_author=E. Marder; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=110; citation_firstpage=E2645; citation_lastpage=E2654; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=CD36 is involved in oleic acid detection by the murine olfactory system; citation_author=S. Oberland; citation_author=T. Ackels; citation_author=S. Gaab; citation_author=T. Pelz; citation_author=J. Spehr; citation_author=M. Spehr; citation_author=E.M. Neuhaus; citation_journal_title=Front. Cell. Neurosci.; citation_volume=9; citation_firstpage=366; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Suite2p: Beyond 10,000 neurons with standard two-photon microscopy; citation_author=M. Pachitariu; citation_author=C. Stringer; citation_author=M. Dipoppa; citation_author=S. Schröder; citation_author=L.F. Rossi; citation_author=H. Dalgleish; citation_author=M. Carandini; citation_author=K.D. Harris; citation_journal_title=bioRxiv; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=An olfactory subsystem that mediates high-sensitivity detection of volatile amines; citation_author=R. Pacifico; citation_author=A. Dewan; citation_author=D. Cawley; citation_author=C. Guo; citation_author=T. Bozza; citation_journal_title=Cell Rep.; citation_volume=2; citation_firstpage=76; citation_lastpage=88; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=Expression of homeobox genes in the mouse olfactory epithelium; citation_author=M. Parrilla; citation_author=I. Chang; citation_author=A. Degl’Innocenti; citation_author=M. Omura; citation_journal_title=J.&nbsp;Comp. Neurol.; citation_volume=524; citation_firstpage=2713; citation_lastpage=2739; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Structure and flexibility in cortical representations of odour space; citation_author=S.L. Pashkovski; citation_author=G. Iurilli; citation_author=D. Brann; citation_author=D. Chicharro; citation_author=K. Drummey; citation_author=K.M. Franks; citation_author=S. Panzeri; citation_author=S.R. Datta; citation_journal_title=Nature; citation_volume=583; citation_firstpage=253; citation_lastpage=258; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Molecular receptive range variation among mouse odorant receptors for aliphatic carboxylic acids; citation_author=S.E. Repicky; citation_author=C.W. Luetje; citation_journal_title=J.&nbsp;Neurochem.; citation_volume=109; citation_firstpage=193; citation_lastpage=202; citation_publication_date=2009; "><meta name="citation_reference" content="citation_title=Optical imaging of odorant representations in the mammalian olfactory bulb; citation_author=B.D. Rubin; citation_author=L.C. Katz; citation_journal_title=Neuron; citation_volume=23; citation_firstpage=499; citation_lastpage=511; citation_publication_date=1999; "><meta name="citation_reference" content="citation_title=Odor coding by a mammalian receptor repertoire; citation_author=H. Saito; citation_author=Q. Chi; citation_author=H. Zhuang; citation_author=H. Matsunami; citation_author=J.D. Mainland; citation_journal_title=Sci. Signal.; citation_volume=2; citation_firstpage=ra9; citation_publication_date=2009; "><meta name="citation_reference" content="citation_title=Phenotypic variation of transcriptomic cell types in mouse motor cortex; citation_author=F. Scala; citation_author=D. Kobak; citation_author=M. Bernabucci; citation_author=Y. Bernaerts; citation_author=C.R. Cadwell; citation_author=J.R. Castro; citation_author=L. Hartmanis; citation_author=X. Jiang; citation_author=S. Laturnus; citation_author=E. Miranda; citation_author=et al.; citation_journal_title=Nature; citation_volume=598; citation_firstpage=144; citation_lastpage=150; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=Odor perception on the two sides of the brain: Consistency despite randomness; citation_author=E.S. Schaffer; citation_author=D.D. Stettler; citation_author=D. Kato; citation_author=G.B. Choi; citation_author=R. Axel; citation_author=L.F. Abbott; citation_journal_title=Neuron; citation_volume=98; citation_firstpage=736; citation_lastpage=742.e3; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Transcriptome analysis of murine olfactory sensory neurons during development using single cell RNA-seq; citation_author=P. Scholz; citation_author=B. Kalbe; citation_author=F. Jansen; citation_author=J. Altmueller; citation_author=C. Becker; citation_author=J. Mohrhardt; citation_author=B. Schreiner; citation_author=G. Gisselmann; citation_author=H. Hatt; citation_author=S. Osterloh; citation_journal_title=Chem. Senses; citation_volume=41; citation_firstpage=313; citation_lastpage=323; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Quantitative expression profiling of identified neurons reveals cell-specific constraints on highly variable levels of gene expression; citation_author=D.J. Schulz; citation_author=J.-M. Goaillard; citation_author=E.E. Marder; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=104; citation_firstpage=13187; citation_lastpage=13191; citation_publication_date=2007; "><meta name="citation_reference" content="citation_title=Precision and diversity in an odor map on the olfactory bulb; citation_author=E.R. Soucy; citation_author=D.F. Albeanu; citation_author=A.L. Fantana; citation_author=V.N. Murthy; citation_author=M. Meister; citation_journal_title=Nat. Neurosci.; citation_volume=12; citation_firstpage=210; citation_lastpage=220; citation_publication_date=2009; "><meta name="citation_reference" content="citation_title=Spatio-temporal dynamics of odor representations in the mammalian olfactory bulb; citation_author=H. Spors; citation_author=A. Grinvald; citation_journal_title=Neuron; citation_volume=34; citation_firstpage=301; citation_lastpage=315; citation_publication_date=2002; "><meta name="citation_reference" content="citation_title=Spatial patterning and information coding in the olfactory system; citation_author=S.L. Sullivan; citation_author=K.J. Ressler; citation_author=L.B. Buck; citation_journal_title=Curr. Opin. Genet. Dev.; citation_volume=5; citation_firstpage=516; citation_lastpage=523; citation_publication_date=1995; "><meta name="citation_reference" content="citation_title=A Near-Complete Spatial Map of Olfactory Receptors in the Mouse Main Olfactory Epithelium; citation_author=L. Tan; citation_author=X.S. Xie; citation_journal_title=Chem. Senses; citation_volume=43; citation_firstpage=427; citation_lastpage=432; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Shared and distinct transcriptomic cell types across neocortical areas; citation_author=B. Tasic; citation_author=Z. Yao; citation_author=L.T. Graybuck; citation_author=K.A. Smith; citation_author=T.N. Nguyen; citation_author=D. Bertagnolli; citation_author=J. Goldy; citation_author=E. Garren; citation_author=M.N. Economo; citation_author=S. Viswanathan; citation_author=et al.; citation_journal_title=Nature; citation_volume=563; citation_firstpage=72; citation_lastpage=78; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Adaptation of olfactory receptor abundances for efficient coding; citation_author=T. Teşileanu; citation_author=S. Cocco; citation_author=R. Monasson; citation_author=V. Balasubramanian; citation_journal_title=eLife; citation_volume=8; citation_firstpage=308; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Odor discrimination by G protein-coupled olfactory receptors; citation_author=K. Touhara; citation_journal_title=Microsc. Res. Tech.; citation_volume=58; citation_firstpage=135; citation_lastpage=141; citation_publication_date=2002; "><meta name="citation_reference" content="citation_title=Homeostatic plasticity in neuronal networks: The more things change, the more they stay the same; citation_author=G.G. Turrigiano; citation_journal_title=Trends Neurosci.; citation_volume=22; citation_firstpage=221; citation_lastpage=227; citation_publication_date=1999; "><meta name="citation_reference" content="citation_title=The self-tuning neuron: Synaptic scaling of excitatory synapses; citation_author=G.G. Turrigiano; citation_journal_title=Cell; citation_volume=135; citation_firstpage=422; citation_lastpage=435; citation_publication_date=2008; "><meta name="citation_reference" content="citation_title=Too many cooks? Intrinsic and synaptic homeostatic mechanisms in cortical circuit refinement; citation_author=G. Turrigiano; citation_journal_title=Annu. Rev. Neurosci.; citation_volume=34; citation_firstpage=89; citation_lastpage=103; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=The dialectic of Hebb and homeostasis; citation_author=G.G. Turrigiano; citation_journal_title=Philos. Trans. R. Soc. Lond. B Biol. Sci.; citation_volume=372; citation_firstpage=20160258; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=The neuronal stimulation-transcription coupling map; citation_author=K.M. Tyssowski; citation_author=J.M. Gray; citation_journal_title=Curr. Opin. Neurobiol.; citation_volume=59; citation_firstpage=87; citation_lastpage=94; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Semiochemical responsive olfactory sensory neurons are sexually dimorphic and plastic; citation_author=A. Vihani; citation_author=X.S. Hu; citation_author=S. Gundala; citation_author=S. Koyama; citation_author=E. Block; citation_author=H. Matsunami; citation_journal_title=eLife; citation_volume=9; citation_firstpage=e54501; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Large-scale transcriptional profiling of chemosensory neurons identifies receptor-ligand pairs in&nbsp;vivo; citation_author=B. von der Weid; citation_author=D. Rossier; citation_author=M. Lindup; citation_author=J. Tuberosa; citation_author=A. Widmer; citation_author=J.D. Col; citation_author=C. Kan; citation_author=A. Carleton; citation_author=I. Rodriguez; citation_journal_title=Nat. Neurosci.; citation_volume=18; citation_firstpage=1455; citation_lastpage=1463; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Representation of odorants by receptor neuron input to the mouse olfactory bulb; citation_author=M. Wachowiak; citation_author=L.B. Cohen; citation_journal_title=Neuron; citation_volume=32; citation_firstpage=723; citation_lastpage=735; citation_publication_date=2001; "><meta name="citation_reference" content="citation_title=Naris occlusion alters the electro-olfactogram: Evidence for compensatory plasticity in the olfactory system; citation_author=C.T. Waggener; citation_author=D.M. Coppola; citation_journal_title=Neurosci. Lett.; citation_volume=427; citation_firstpage=112; citation_lastpage=116; citation_publication_date=2007; "><meta name="citation_reference" content="citation_title=Induction of olfactory receptor sensitivity in mice; citation_author=H.W. Wang; citation_author=C.J. Wysocki; citation_author=G.H. Gold; citation_journal_title=Science; citation_volume=260; citation_firstpage=998; citation_lastpage=1000; citation_publication_date=1993; "><meta name="citation_reference" content="citation_title=Activity-dependent gene expression in the mammalian olfactory epithelium; citation_author=Q. Wang; citation_author=W.B. Titlow; citation_author=D.A. McClintock; citation_author=A.J. Stromberg; citation_author=T.S. McClintock; citation_journal_title=Chem. Senses; citation_volume=42; citation_firstpage=611; citation_lastpage=624; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Sensory adaptation; citation_author=B. Wark; citation_author=B.N. Lundstrom; citation_author=A. Fairhall; citation_journal_title=Curr. Opin. Neurobiol.; citation_volume=17; citation_firstpage=423; citation_lastpage=429; citation_publication_date=2007; "><meta name="citation_reference" content="citation_title=Coding principles in adaptation; citation_author=A.I. Weber; citation_author=K. Krishnamurthy; citation_author=A.L. Fairhall; citation_journal_title=Annu. Rev. Vis. Sci.; citation_volume=5; citation_firstpage=427; citation_lastpage=449; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=SPRING: A kinetic interface for visualizing high dimensional single-cell expression data; citation_author=C. Weinreb; citation_author=S. Wolock; citation_author=A.M. Klein; citation_journal_title=Bioinformatics; citation_volume=34; citation_firstpage=1246; citation_lastpage=1248; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Unsupervised discovery of demixed, low-dimensional neural dynamics across multiple timescales through tensor component analysis; citation_author=A.H. Williams; citation_author=T.H. Kim; citation_author=F. Wang; citation_author=S. Vyas; citation_author=S.I. Ryu; citation_author=K.V. Shenoy; citation_author=M. Schnitzer; citation_author=T.G. Kolda; citation_author=S. Ganguli; citation_journal_title=Neuron; citation_volume=98; citation_firstpage=1099; citation_lastpage=1115.e8; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Habituation of odor responses in the rat anterior piriform cortex; citation_author=D.A. Wilson; citation_journal_title=J.&nbsp;Neurophysiol.; citation_volume=79; citation_firstpage=1425; citation_lastpage=1440; citation_publication_date=1998; "><meta name="citation_reference" content="citation_title=Comparison of odor receptive field plasticity in the rat olfactory bulb and anterior piriform cortex; citation_author=D.A. Wilson; citation_journal_title=J.&nbsp;Neurophysiol.; citation_volume=84; citation_firstpage=3036; citation_lastpage=3042; citation_publication_date=2000; "><meta name="citation_reference" content="citation_title=Olfaction as a model system for the neurobiology of mammalian short-term habituation; citation_author=D.A. Wilson; citation_journal_title=Neurobiol. Learn. Mem.; citation_volume=92; citation_firstpage=199; citation_lastpage=205; citation_publication_date=2009; "><meta name="citation_reference" content="citation_title=A primacy code for odor identity; citation_author=C.D. Wilson; citation_author=G.O. Serrano; citation_author=A.A. Koulakov; citation_author=D. Rinberg; citation_journal_title=Nat. Commun.; citation_volume=8; citation_firstpage=1477; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=SCANPY: Large-scale single-cell gene expression data analysis; citation_author=F.A. Wolf; citation_author=P. Angerer; citation_author=F.J. Theis; citation_journal_title=Genome Biol.; citation_volume=19; citation_firstpage=15; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Detecting activated cell populations using single-cell RNA-seq; citation_author=Y.E. Wu; citation_author=L. Pan; citation_author=Y. Zuo; citation_author=X. Li; citation_author=W. Hong; citation_journal_title=Neuron; citation_volume=96; citation_firstpage=313; citation_lastpage=329.e6; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=A population of navigator neurons is essential for olfactory map formation during the critical period; citation_author=Y. Wu; citation_author=L. Ma; citation_author=K. Duyck; citation_author=C.C. Long; citation_author=A. Moran; citation_author=H. Scheerer; citation_author=J. Blanck; citation_author=A. Peak; citation_author=A. Box; citation_author=A. Perera; citation_author=C.R. Yu; citation_journal_title=Neuron; citation_volume=100; citation_firstpage=1066; citation_lastpage=1082.e6; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=CD36 is expressed in a defined subpopulation of neurons in the olfactory epithelium; citation_author=A.M. Xavier; citation_author=R.G. Ludwig; citation_author=M.H. Nagai; citation_author=T.J. de Almeida; citation_author=H.M. Watanabe; citation_author=M.Y. Hirata; citation_author=T.R. Rosenstock; citation_author=F. Papes; citation_author=B. Malnic; citation_author=I. Glezer; citation_journal_title=Sci. Rep.; citation_volume=6; citation_firstpage=25507; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=The role of metabotropic glutamate receptors and cortical adaptation in habituation of odor-guided behavior; citation_author=C.A. Yadon; citation_author=D.A. Wilson; citation_journal_title=Learn. Mem.; citation_volume=12; citation_firstpage=601; citation_lastpage=605; citation_publication_date=2005; "><meta name="citation_reference" content="citation_title=Activity-regulated transcription: Bridging the gap between neural activity and behavior; citation_author=E.-L. Yap; citation_author=M.E. Greenberg; citation_journal_title=Neuron; citation_volume=100; citation_firstpage=330; citation_lastpage=348; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=The olfactory receptor gene superfamily of the mouse; citation_author=X. Zhang; citation_author=S. Firestein; citation_journal_title=Nat. Neurosci.; citation_volume=5; citation_firstpage=124; citation_lastpage=133; citation_publication_date=2002; "><meta name="citation_reference" content="citation_title=Uncoupling stimulus specificity and glomerular position in the mouse olfactory system; citation_author=J. Zhang; citation_author=G. Huang; citation_author=A. Dewan; citation_author=P. Feinstein; citation_author=T. Bozza; citation_journal_title=Mol. Cell. Neurosci.; citation_volume=51; citation_firstpage=79; citation_lastpage=88; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=The cellular and molecular basis of odor adaptation; citation_author=F. Zufall; citation_author=T. Leinders-Zufall; citation_journal_title=Chem. Senses; citation_volume=25; citation_firstpage=473; citation_lastpage=481; citation_publication_date=2000; "><meta name="citation_reference" content="citation_title=Receptor arrays optimized for natural odor statistics; citation_author=D. Zwicker; citation_author=A. Murugan; citation_author=M.P. Brenner; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=113; citation_firstpage=5570; citation_lastpage=5575; citation_publication_date=2016; "><meta name="citation_abstract" content="<h2>Summary</h2><p>Animals traversing different environments encounter both stable background stimuli and novel cues, which are thought to be detected by primary sensory neurons and then distinguished by downstream brain circuits. Here, we show that each of the ∼1,000 olfactory sensory neuron (OSN) subtypes in the mouse harbors a distinct transcriptome whose content is precisely determined by interactions between its odorant receptor and the environment. This transcriptional variation is systematically organized to support sensory adaptation: expression levels of more than 70 genes relevant to transforming odors into spikes continuously vary across OSN subtypes, dynamically adjust to new environments over hours, and accurately predict acute OSN-specific odor responses. The sensory periphery therefore separates salient signals from predictable background via a transcriptional rheostat whose moment-to-moment state reflects the past and constrains the future; these findings suggest a general model in which structured transcriptional variation within a cell type reflects individual experience.</p>"><meta charset="UTF-8"><!--

    Pagebuild admin UI
-->



<script type="text/javascript" src="https://bam.nr-data.net/1/16e2192192?a=916957862&amp;sa=1&amp;v=1177.96a4d39&amp;t=Unnamed%20Transaction&amp;rst=7472&amp;ck=1&amp;ref=https://www.cell.com/cell/fulltext/S0092-8674(21)01337-4&amp;be=4006&amp;fe=7433&amp;dc=7025&amp;af=err,xhr,stn,ins,spa&amp;perf=%7B%22timing%22:%7B%22of%22:1749592248858,%22n%22:0,%22f%22:3478,%22dn%22:3479,%22dne%22:3479,%22c%22:3479,%22s%22:3479,%22ce%22:3479,%22rq%22:3479,%22rp%22:3990,%22rpe%22:4719,%22dl%22:3996,%22di%22:4996,%22ds%22:7025,%22de%22:7036,%22dc%22:7433,%22l%22:7433,%22le%22:7464%7D,%22navigation%22:%7B%7D%7D&amp;jsonp=NREUM.setToken"></script><script src="https://js-agent.newrelic.com/nr-spa-1177.min.js"></script><script type="text/javascript">
    ;window.NREUM||(NREUM={});NREUM.init={privacy:{cookies_enabled:true}};
    window.NREUM||(NREUM={}),__nr_require=function(t,e,n){function r(n){if(!e[n]){var o=e[n]={exports:{}};t[n][0].call(o.exports,function(e){var o=t[n][1][e];return r(o||e)},o,o.exports)}return e[n].exports}if("function"==typeof __nr_require)return __nr_require;for(var o=0;o<n.length;o++)r(n[o]);return r}({1:[function(t,e,n){function r(t){try{c.console&&console.log(t)}catch(e){}}var o,i=t("ee"),a=t(26),c={};try{o=localStorage.getItem("__nr_flags").split(","),console&&"function"==typeof console.log&&(c.console=!0,o.indexOf("dev")!==-1&&(c.dev=!0),o.indexOf("nr_dev")!==-1&&(c.nrDev=!0))}catch(s){}c.nrDev&&i.on("internal-error",function(t){r(t.stack)}),c.dev&&i.on("fn-err",function(t,e,n){r(n.stack)}),c.dev&&(r("NR AGENT IN DEVELOPMENT MODE"),r("flags: "+a(c,function(t,e){return t}).join(", ")))},{}],2:[function(t,e,n){function r(t,e,n,r,c){try{l?l-=1:o(c||new UncaughtException(t,e,n),!0)}catch(f){try{i("ierr",[f,s.now(),!0])}catch(d){}}return"function"==typeof u&&u.apply(this,a(arguments))}function UncaughtException(t,e,n){this.message=t||"Uncaught error with no additional information",this.sourceURL=e,this.line=n}function o(t,e){var n=e?null:s.now();i("err",[t,n])}var i=t("handle"),a=t(27),c=t("ee"),s=t("loader"),f=t("gos"),u=window.onerror,d=!1,p="nr@seenError",l=0;s.features.err=!0,t(1),window.onerror=r;try{throw new Error}catch(h){"stack"in h&&(t(13),t(12),"addEventListener"in window&&t(6),s.xhrWrappable&&t(14),d=!0)}c.on("fn-start",function(t,e,n){d&&(l+=1)}),c.on("fn-err",function(t,e,n){d&&!n[p]&&(f(n,p,function(){return!0}),this.thrown=!0,o(n))}),c.on("fn-end",function(){d&&!this.thrown&&l>0&&(l-=1)}),c.on("internal-error",function(t){i("ierr",[t,s.now(),!0])})},{}],3:[function(t,e,n){t("loader").features.ins=!0},{}],4:[function(t,e,n){function r(){_++,T=g.hash,this[u]=y.now()}function o(){_--,g.hash!==T&&i(0,!0);var t=y.now();this[h]=~~this[h]+t-this[u],this[d]=t}function i(t,e){E.emit("newURL",[""+g,e])}function a(t,e){t.on(e,function(){this[e]=y.now()})}var c="-start",s="-end",f="-body",u="fn"+c,d="fn"+s,p="cb"+c,l="cb"+s,h="jsTime",m="fetch",v="addEventListener",w=window,g=w.location,y=t("loader");if(w[v]&&y.xhrWrappable){var x=t(10),b=t(11),E=t(8),R=t(6),O=t(13),N=t(7),M=t(14),P=t(9),S=t("ee"),C=S.get("tracer");t(16),y.features.spa=!0;var T,_=0;S.on(u,r),S.on(p,r),S.on(d,o),S.on(l,o),S.buffer([u,d,"xhr-done","xhr-resolved"]),R.buffer([u]),O.buffer(["setTimeout"+s,"clearTimeout"+c,u]),M.buffer([u,"new-xhr","send-xhr"+c]),N.buffer([m+c,m+"-done",m+f+c,m+f+s]),E.buffer(["newURL"]),x.buffer([u]),b.buffer(["propagate",p,l,"executor-err","resolve"+c]),C.buffer([u,"no-"+u]),P.buffer(["new-jsonp","cb-start","jsonp-error","jsonp-end"]),a(M,"send-xhr"+c),a(S,"xhr-resolved"),a(S,"xhr-done"),a(N,m+c),a(N,m+"-done"),a(P,"new-jsonp"),a(P,"jsonp-end"),a(P,"cb-start"),E.on("pushState-end",i),E.on("replaceState-end",i),w[v]("hashchange",i,!0),w[v]("load",i,!0),w[v]("popstate",function(){i(0,_>1)},!0)}},{}],5:[function(t,e,n){function r(t){}if(window.performance&&window.performance.timing&&window.performance.getEntriesByType){var o=t("ee"),i=t("handle"),a=t(13),c=t(12),s="learResourceTimings",f="addEventListener",u="resourcetimingbufferfull",d="bstResource",p="resource",l="-start",h="-end",m="fn"+l,v="fn"+h,w="bstTimer",g="pushState",y=t("loader");y.features.stn=!0,t(8),"addEventListener"in window&&t(6);var x=NREUM.o.EV;o.on(m,function(t,e){var n=t[0];n instanceof x&&(this.bstStart=y.now())}),o.on(v,function(t,e){var n=t[0];n instanceof x&&i("bst",[n,e,this.bstStart,y.now()])}),a.on(m,function(t,e,n){this.bstStart=y.now(),this.bstType=n}),a.on(v,function(t,e){i(w,[e,this.bstStart,y.now(),this.bstType])}),c.on(m,function(){this.bstStart=y.now()}),c.on(v,function(t,e){i(w,[e,this.bstStart,y.now(),"requestAnimationFrame"])}),o.on(g+l,function(t){this.time=y.now(),this.startPath=location.pathname+location.hash}),o.on(g+h,function(t){i("bstHist",[location.pathname+location.hash,this.startPath,this.time])}),f in window.performance&&(window.performance["c"+s]?window.performance[f](u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["c"+s]()},!1):window.performance[f]("webkit"+u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["webkitC"+s]()},!1)),document[f]("scroll",r,{passive:!0}),document[f]("keypress",r,!1),document[f]("click",r,!1)}},{}],6:[function(t,e,n){function r(t){for(var e=t;e&&!e.hasOwnProperty(u);)e=Object.getPrototypeOf(e);e&&o(e)}function o(t){c.inPlace(t,[u,d],"-",i)}function i(t,e){return t[1]}var a=t("ee").get("events"),c=t("wrap-function")(a,!0),s=t("gos"),f=XMLHttpRequest,u="addEventListener",d="removeEventListener";e.exports=a,"getPrototypeOf"in Object?(r(document),r(window),r(f.prototype)):f.prototype.hasOwnProperty(u)&&(o(window),o(f.prototype)),a.on(u+"-start",function(t,e){var n=t[1],r=s(n,"nr@wrapped",function(){function t(){if("function"==typeof n.handleEvent)return n.handleEvent.apply(n,arguments)}var e={object:t,"function":n}[typeof n];return e?c(e,"fn-",null,e.name||"anonymous"):n});this.wrapped=t[1]=r}),a.on(d+"-start",function(t){t[1]=this.wrapped||t[1]})},{}],7:[function(t,e,n){function r(t,e,n){var r=t[e];"function"==typeof r&&(t[e]=function(){var t=i(arguments),e={};o.emit(n+"before-start",[t],e);var a;e[m]&&e[m].dt&&(a=e[m].dt);var c=r.apply(this,t);return o.emit(n+"start",[t,a],c),c.then(function(t){return o.emit(n+"end",[null,t],c),t},function(t){throw o.emit(n+"end",[t],c),t})})}var o=t("ee").get("fetch"),i=t(27),a=t(26);e.exports=o;var c=window,s="fetch-",f=s+"body-",u=["arrayBuffer","blob","json","text","formData"],d=c.Request,p=c.Response,l=c.fetch,h="prototype",m="nr@context";d&&p&&l&&(a(u,function(t,e){r(d[h],e,f),r(p[h],e,f)}),r(c,"fetch",s),o.on(s+"end",function(t,e){var n=this;if(e){var r=e.headers.get("content-length");null!==r&&(n.rxSize=r),o.emit(s+"done",[null,e],n)}else o.emit(s+"done",[t],n)}))},{}],8:[function(t,e,n){var r=t("ee").get("history"),o=t("wrap-function")(r);e.exports=r;var i=window.history&&window.history.constructor&&window.history.constructor.prototype,a=window.history;i&&i.pushState&&i.replaceState&&(a=i),o.inPlace(a,["pushState","replaceState"],"-")},{}],9:[function(t,e,n){function r(t){function e(){s.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}function n(){s.emit("jsonp-error",[],p),s.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}var r=t&&"string"==typeof t.nodeName&&"script"===t.nodeName.toLowerCase();if(r){var o="function"==typeof t.addEventListener;if(o){var a=i(t.src);if(a){var u=c(a),d="function"==typeof u.parent[u.key];if(d){var p={};f.inPlace(u.parent,[u.key],"cb-",p),t.addEventListener("load",e,!1),t.addEventListener("error",n,!1),s.emit("new-jsonp",[t.src],p)}}}}}function o(){return"addEventListener"in window}function i(t){var e=t.match(u);return e?e[1]:null}function a(t,e){var n=t.match(p),r=n[1],o=n[3];return o?a(o,e[r]):e[r]}function c(t){var e=t.match(d);return e&&e.length>=3?{key:e[2],parent:a(e[1],window)}:{key:t,parent:window}}var s=t("ee").get("jsonp"),f=t("wrap-function")(s);if(e.exports=s,o()){var u=/[?&](?:callback|cb)=([^&#]+)/,d=/(.*)\.([^.]+)/,p=/^(\w+)(\.|$)(.*)$/,l=["appendChild","insertBefore","replaceChild"];Node&&Node.prototype&&Node.prototype.appendChild?f.inPlace(Node.prototype,l,"dom-"):(f.inPlace(HTMLElement.prototype,l,"dom-"),f.inPlace(HTMLHeadElement.prototype,l,"dom-"),f.inPlace(HTMLBodyElement.prototype,l,"dom-")),s.on("dom-start",function(t){r(t[0])})}},{}],10:[function(t,e,n){var r=t("ee").get("mutation"),o=t("wrap-function")(r),i=NREUM.o.MO;e.exports=r,i&&(window.MutationObserver=function(t){return this instanceof i?new i(o(t,"fn-")):i.apply(this,arguments)},MutationObserver.prototype=i.prototype)},{}],11:[function(t,e,n){function r(t){var e=a.context(),n=c(t,"executor-",e),r=new f(n);return a.context(r).getCtx=function(){return e},a.emit("new-promise",[r,e],e),r}function o(t,e){return e}var i=t("wrap-function"),a=t("ee").get("promise"),c=i(a),s=t(26),f=NREUM.o.PR;e.exports=a,f&&(window.Promise=r,["all","race"].forEach(function(t){var e=f[t];f[t]=function(n){function r(t){return function(){a.emit("propagate",[null,!o],i),o=o||!t}}var o=!1;s(n,function(e,n){Promise.resolve(n).then(r("all"===t),r(!1))});var i=e.apply(f,arguments),c=f.resolve(i);return c}}),["resolve","reject"].forEach(function(t){var e=f[t];f[t]=function(t){var n=e.apply(f,arguments);return t!==n&&a.emit("propagate",[t,!0],n),n}}),f.prototype["catch"]=function(t){return this.then(null,t)},f.prototype=Object.create(f.prototype,{constructor:{value:r}}),s(Object.getOwnPropertyNames(f),function(t,e){try{r[e]=f[e]}catch(n){}}),a.on("executor-start",function(t){t[0]=c(t[0],"resolve-",this),t[1]=c(t[1],"resolve-",this)}),a.on("executor-err",function(t,e,n){t[1](n)}),c.inPlace(f.prototype,["then"],"then-",o),a.on("then-start",function(t,e){this.promise=e,t[0]=c(t[0],"cb-",this),t[1]=c(t[1],"cb-",this)}),a.on("then-end",function(t,e,n){this.nextPromise=n;var r=this.promise;a.emit("propagate",[r,!0],n)}),a.on("cb-end",function(t,e,n){a.emit("propagate",[n,!0],this.nextPromise)}),a.on("propagate",function(t,e,n){this.getCtx&&!e||(this.getCtx=function(){if(t instanceof Promise)var e=a.context(t);return e&&e.getCtx?e.getCtx():this})}),r.toString=function(){return""+f})},{}],12:[function(t,e,n){var r=t("ee").get("raf"),o=t("wrap-function")(r),i="equestAnimationFrame";e.exports=r,o.inPlace(window,["r"+i,"mozR"+i,"webkitR"+i,"msR"+i],"raf-"),r.on("raf-start",function(t){t[0]=o(t[0],"fn-")})},{}],13:[function(t,e,n){function r(t,e,n){t[0]=a(t[0],"fn-",null,n)}function o(t,e,n){this.method=n,this.timerDuration=isNaN(t[1])?0:+t[1],t[0]=a(t[0],"fn-",this,n)}var i=t("ee").get("timer"),a=t("wrap-function")(i),c="setTimeout",s="setInterval",f="clearTimeout",u="-start",d="-";e.exports=i,a.inPlace(window,[c,"setImmediate"],c+d),a.inPlace(window,[s],s+d),a.inPlace(window,[f,"clearImmediate"],f+d),i.on(s+u,r),i.on(c+u,o)},{}],14:[function(t,e,n){function r(t,e){d.inPlace(e,["onreadystatechange"],"fn-",c)}function o(){var t=this,e=u.context(t);t.readyState>3&&!e.resolved&&(e.resolved=!0,u.emit("xhr-resolved",[],t)),d.inPlace(t,g,"fn-",c)}function i(t){y.push(t),h&&(b?b.then(a):v?v(a):(E=-E,R.data=E))}function a(){for(var t=0;t<y.length;t++)r([],y[t]);y.length&&(y=[])}function c(t,e){return e}function s(t,e){for(var n in t)e[n]=t[n];return e}t(6);var f=t("ee"),u=f.get("xhr"),d=t("wrap-function")(u),p=NREUM.o,l=p.XHR,h=p.MO,m=p.PR,v=p.SI,w="readystatechange",g=["onload","onerror","onabort","onloadstart","onloadend","onprogress","ontimeout"],y=[];e.exports=u;var x=window.XMLHttpRequest=function(t){var e=new l(t);try{u.emit("new-xhr",[e],e),e.addEventListener(w,o,!1)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}return e};if(s(l,x),x.prototype=l.prototype,d.inPlace(x.prototype,["open","send"],"-xhr-",c),u.on("send-xhr-start",function(t,e){r(t,e),i(e)}),u.on("open-xhr-start",r),h){var b=m&&m.resolve();if(!v&&!m){var E=1,R=document.createTextNode(E);new h(a).observe(R,{characterData:!0})}}else f.on("fn-end",function(t){t[0]&&t[0].type===w||a()})},{}],15:[function(t,e,n){function r(t){if(!c(t))return null;var e=window.NREUM;if(!e.loader_config)return null;var n=(e.loader_config.accountID||"").toString()||null,r=(e.loader_config.agentID||"").toString()||null,f=(e.loader_config.trustKey||"").toString()||null;if(!n||!r)return null;var h=l.generateSpanId(),m=l.generateTraceId(),v=Date.now(),w={spanId:h,traceId:m,timestamp:v};return(t.sameOrigin||s(t)&&p())&&(w.traceContextParentHeader=o(h,m),w.traceContextStateHeader=i(h,v,n,r,f)),(t.sameOrigin&&!u()||!t.sameOrigin&&s(t)&&d())&&(w.newrelicHeader=a(h,m,v,n,r,f)),w}function o(t,e){return"00-"+e+"-"+t+"-01"}function i(t,e,n,r,o){var i=0,a="",c=1,s="",f="";return o+"@nr="+i+"-"+c+"-"+n+"-"+r+"-"+t+"-"+a+"-"+s+"-"+f+"-"+e}function a(t,e,n,r,o,i){var a="btoa"in window&&"function"==typeof window.btoa;if(!a)return null;var c={v:[0,1],d:{ty:"Browser",ac:r,ap:o,id:t,tr:e,ti:n}};return i&&r!==i&&(c.d.tk=i),btoa(JSON.stringify(c))}function c(t){return f()&&s(t)}function s(t){var e=!1,n={};if("init"in NREUM&&"distributed_tracing"in NREUM.init&&(n=NREUM.init.distributed_tracing),t.sameOrigin)e=!0;else if(n.allowed_origins instanceof Array)for(var r=0;r<n.allowed_origins.length;r++){var o=h(n.allowed_origins[r]);if(t.hostname===o.hostname&&t.protocol===o.protocol&&t.port===o.port){e=!0;break}}return e}function f(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.enabled}function u(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.exclude_newrelic_header}function d(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&NREUM.init.distributed_tracing.cors_use_newrelic_header!==!1}function p(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.cors_use_tracecontext_headers}var l=t(23),h=t(17);e.exports={generateTracePayload:r,shouldGenerateTrace:c}},{}],16:[function(t,e,n){function r(t){var e=this.params,n=this.metrics;if(!this.ended){this.ended=!0;for(var r=0;r<p;r++)t.removeEventListener(d[r],this.listener,!1);e.aborted||(n.duration=a.now()-this.startTime,this.loadCaptureCalled||4!==t.readyState?null==e.status&&(e.status=0):i(this,t),n.cbTime=this.cbTime,u.emit("xhr-done",[t],t),c("xhr",[e,n,this.startTime]))}}function o(t,e){var n=s(e),r=t.params;r.host=n.hostname+":"+n.port,r.pathname=n.pathname,t.parsedOrigin=s(e),t.sameOrigin=t.parsedOrigin.sameOrigin}function i(t,e){t.params.status=e.status;var n=v(e,t.lastSize);if(n&&(t.metrics.rxSize=n),t.sameOrigin){var r=e.getResponseHeader("X-NewRelic-App-Data");r&&(t.params.cat=r.split(", ").pop())}t.loadCaptureCalled=!0}var a=t("loader");if(a.xhrWrappable){var c=t("handle"),s=t(17),f=t(15).generateTracePayload,u=t("ee"),d=["load","error","abort","timeout"],p=d.length,l=t("id"),h=t(21),m=t(20),v=t(18),w=window.XMLHttpRequest;a.features.xhr=!0,t(14),t(7),u.on("new-xhr",function(t){var e=this;e.totalCbs=0,e.called=0,e.cbTime=0,e.end=r,e.ended=!1,e.xhrGuids={},e.lastSize=null,e.loadCaptureCalled=!1,t.addEventListener("load",function(n){i(e,t)},!1),h&&(h>34||h<10)||window.opera||t.addEventListener("progress",function(t){e.lastSize=t.loaded},!1)}),u.on("open-xhr-start",function(t){this.params={method:t[0]},o(this,t[1]),this.metrics={}}),u.on("open-xhr-end",function(t,e){"loader_config"in NREUM&&"xpid"in NREUM.loader_config&&this.sameOrigin&&e.setRequestHeader("X-NewRelic-ID",NREUM.loader_config.xpid);var n=f(this.parsedOrigin);if(n){var r=!1;n.newrelicHeader&&(e.setRequestHeader("newrelic",n.newrelicHeader),r=!0),n.traceContextParentHeader&&(e.setRequestHeader("traceparent",n.traceContextParentHeader),n.traceContextStateHeader&&e.setRequestHeader("tracestate",n.traceContextStateHeader),r=!0),r&&(this.dt=n)}}),u.on("send-xhr-start",function(t,e){var n=this.metrics,r=t[0],o=this;if(n&&r){var i=m(r);i&&(n.txSize=i)}this.startTime=a.now(),this.listener=function(t){try{"abort"!==t.type||o.loadCaptureCalled||(o.params.aborted=!0),("load"!==t.type||o.called===o.totalCbs&&(o.onloadCalled||"function"!=typeof e.onload))&&o.end(e)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}};for(var c=0;c<p;c++)e.addEventListener(d[c],this.listener,!1)}),u.on("xhr-cb-time",function(t,e,n){this.cbTime+=t,e?this.onloadCalled=!0:this.called+=1,this.called!==this.totalCbs||!this.onloadCalled&&"function"==typeof n.onload||this.end(n)}),u.on("xhr-load-added",function(t,e){var n=""+l(t)+!!e;this.xhrGuids&&!this.xhrGuids[n]&&(this.xhrGuids[n]=!0,this.totalCbs+=1)}),u.on("xhr-load-removed",function(t,e){var n=""+l(t)+!!e;this.xhrGuids&&this.xhrGuids[n]&&(delete this.xhrGuids[n],this.totalCbs-=1)}),u.on("addEventListener-end",function(t,e){e instanceof w&&"load"===t[0]&&u.emit("xhr-load-added",[t[1],t[2]],e)}),u.on("removeEventListener-end",function(t,e){e instanceof w&&"load"===t[0]&&u.emit("xhr-load-removed",[t[1],t[2]],e)}),u.on("fn-start",function(t,e,n){e instanceof w&&("onload"===n&&(this.onload=!0),("load"===(t[0]&&t[0].type)||this.onload)&&(this.xhrCbStart=a.now()))}),u.on("fn-end",function(t,e){this.xhrCbStart&&u.emit("xhr-cb-time",[a.now()-this.xhrCbStart,this.onload,e],e)}),u.on("fetch-before-start",function(t){function e(t,e){var n=!1;return e.newrelicHeader&&(t.set("newrelic",e.newrelicHeader),n=!0),e.traceContextParentHeader&&(t.set("traceparent",e.traceContextParentHeader),e.traceContextStateHeader&&t.set("tracestate",e.traceContextStateHeader),n=!0),n}var n,r=t[1]||{};"string"==typeof t[0]?n=t[0]:t[0]&&t[0].url&&(n=t[0].url),n&&(this.parsedOrigin=s(n),this.sameOrigin=this.parsedOrigin.sameOrigin);var o=f(this.parsedOrigin);if(o&&(o.newrelicHeader||o.traceContextParentHeader))if("string"==typeof t[0]){var i={};for(var a in r)i[a]=r[a];i.headers=new Headers(r.headers||{}),e(i.headers,o)&&(this.dt=o),t.length>1?t[1]=i:t.push(i)}else t[0]&&t[0].headers&&e(t[0].headers,o)&&(this.dt=o)})}},{}],17:[function(t,e,n){var r={};e.exports=function(t){if(t in r)return r[t];var e=document.createElement("a"),n=window.location,o={};e.href=t,o.port=e.port;var i=e.href.split("://");!o.port&&i[1]&&(o.port=i[1].split("/")[0].split("@").pop().split(":")[1]),o.port&&"0"!==o.port||(o.port="https"===i[0]?"443":"80"),o.hostname=e.hostname||n.hostname,o.pathname=e.pathname,o.protocol=i[0],"/"!==o.pathname.charAt(0)&&(o.pathname="/"+o.pathname);var a=!e.protocol||":"===e.protocol||e.protocol===n.protocol,c=e.hostname===document.domain&&e.port===n.port;return o.sameOrigin=a&&(!e.hostname||c),"/"===o.pathname&&(r[t]=o),o}},{}],18:[function(t,e,n){function r(t,e){var n=t.responseType;return"json"===n&&null!==e?e:"arraybuffer"===n||"blob"===n||"json"===n?o(t.response):"text"===n||""===n||void 0===n?o(t.responseText):void 0}var o=t(20);e.exports=r},{}],19:[function(t,e,n){function r(){}function o(t,e,n){return function(){return i(t,[f.now()].concat(c(arguments)),e?null:this,n),e?void 0:this}}var i=t("handle"),a=t(26),c=t(27),s=t("ee").get("tracer"),f=t("loader"),u=NREUM;"undefined"==typeof window.newrelic&&(newrelic=u);var d=["setPageViewName","setCustomAttribute","setErrorHandler","finished","addToTrace","inlineHit","addRelease"],p="api-",l=p+"ixn-";a(d,function(t,e){u[e]=o(p+e,!0,"api")}),u.addPageAction=o(p+"addPageAction",!0),u.setCurrentRouteName=o(p+"routeName",!0),e.exports=newrelic,u.interaction=function(){return(new r).get()};var h=r.prototype={createTracer:function(t,e){var n={},r=this,o="function"==typeof e;return i(l+"tracer",[f.now(),t,n],r),function(){if(s.emit((o?"":"no-")+"fn-start",[f.now(),r,o],n),o)try{return e.apply(this,arguments)}catch(t){throw s.emit("fn-err",[arguments,this,t],n),t}finally{s.emit("fn-end",[f.now()],n)}}}};a("actionText,setName,setAttribute,save,ignore,onEnd,getContext,end,get".split(","),function(t,e){h[e]=o(l+e)}),newrelic.noticeError=function(t,e){"string"==typeof t&&(t=new Error(t)),i("err",[t,f.now(),!1,e])}},{}],20:[function(t,e,n){e.exports=function(t){if("string"==typeof t&&t.length)return t.length;if("object"==typeof t){if("undefined"!=typeof ArrayBuffer&&t instanceof ArrayBuffer&&t.byteLength)return t.byteLength;if("undefined"!=typeof Blob&&t instanceof Blob&&t.size)return t.size;if(!("undefined"!=typeof FormData&&t instanceof FormData))try{return JSON.stringify(t).length}catch(e){return}}}},{}],21:[function(t,e,n){var r=0,o=navigator.userAgent.match(/Firefox[\/\s](\d+\.\d+)/);o&&(r=+o[1]),e.exports=r},{}],22:[function(t,e,n){function r(t,e){var n=t.getEntries();n.forEach(function(t){"first-paint"===t.name?d("timing",["fp",Math.floor(t.startTime)]):"first-contentful-paint"===t.name&&d("timing",["fcp",Math.floor(t.startTime)])})}function o(t,e){var n=t.getEntries();n.length>0&&d("lcp",[n[n.length-1]])}function i(t){t.getEntries().forEach(function(t){t.hadRecentInput||d("cls",[t])})}function a(t){if(t instanceof h&&!v){var e=Math.round(t.timeStamp),n={type:t.type};e<=p.now()?n.fid=p.now()-e:e>p.offset&&e<=Date.now()?(e-=p.offset,n.fid=p.now()-e):e=p.now(),v=!0,d("timing",["fi",e,n])}}function c(t){d("pageHide",[p.now(),t])}if(!("init"in NREUM&&"page_view_timing"in NREUM.init&&"enabled"in NREUM.init.page_view_timing&&NREUM.init.page_view_timing.enabled===!1)){var s,f,u,d=t("handle"),p=t("loader"),l=t(25),h=NREUM.o.EV;if("PerformanceObserver"in window&&"function"==typeof window.PerformanceObserver){s=new PerformanceObserver(r);try{s.observe({entryTypes:["paint"]})}catch(m){}f=new PerformanceObserver(o);try{f.observe({entryTypes:["largest-contentful-paint"]})}catch(m){}u=new PerformanceObserver(i);try{u.observe({type:"layout-shift",buffered:!0})}catch(m){}}if("addEventListener"in document){var v=!1,w=["click","keydown","mousedown","pointerdown","touchstart"];w.forEach(function(t){document.addEventListener(t,a,!1)})}l(c)}},{}],23:[function(t,e,n){function r(){function t(){return e?15&e[n++]:16*Math.random()|0}var e=null,n=0,r=window.crypto||window.msCrypto;r&&r.getRandomValues&&(e=r.getRandomValues(new Uint8Array(31)));for(var o,i="xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx",a="",c=0;c<i.length;c++)o=i[c],"x"===o?a+=t().toString(16):"y"===o?(o=3&t()|8,a+=o.toString(16)):a+=o;return a}function o(){return a(16)}function i(){return a(32)}function a(t){function e(){return n?15&n[r++]:16*Math.random()|0}var n=null,r=0,o=window.crypto||window.msCrypto;o&&o.getRandomValues&&Uint8Array&&(n=o.getRandomValues(new Uint8Array(31)));for(var i=[],a=0;a<t;a++)i.push(e().toString(16));return i.join("")}e.exports={generateUuid:r,generateSpanId:o,generateTraceId:i}},{}],24:[function(t,e,n){function r(t,e){if(!o)return!1;if(t!==o)return!1;if(!e)return!0;if(!i)return!1;for(var n=i.split("."),r=e.split("."),a=0;a<r.length;a++)if(r[a]!==n[a])return!1;return!0}var o=null,i=null,a=/Version\/(\S+)\s+Safari/;if(navigator.userAgent){var c=navigator.userAgent,s=c.match(a);s&&c.indexOf("Chrome")===-1&&c.indexOf("Chromium")===-1&&(o="Safari",i=s[1])}e.exports={agent:o,version:i,match:r}},{}],25:[function(t,e,n){function r(t){function e(){t(a&&document[a]?document[a]:document[o]?"hidden":"visible")}"addEventListener"in document&&i&&document.addEventListener(i,e,!1)}e.exports=r;var o,i,a;"undefined"!=typeof document.hidden?(o="hidden",i="visibilitychange",a="visibilityState"):"undefined"!=typeof document.msHidden?(o="msHidden",i="msvisibilitychange"):"undefined"!=typeof document.webkitHidden&&(o="webkitHidden",i="webkitvisibilitychange",a="webkitVisibilityState")},{}],26:[function(t,e,n){function r(t,e){var n=[],r="",i=0;for(r in t)o.call(t,r)&&(n[i]=e(r,t[r]),i+=1);return n}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],27:[function(t,e,n){function r(t,e,n){e||(e=0),"undefined"==typeof n&&(n=t?t.length:0);for(var r=-1,o=n-e||0,i=Array(o<0?0:o);++r<o;)i[r]=t[e+r];return i}e.exports=r},{}],28:[function(t,e,n){e.exports={exists:"undefined"!=typeof window.performance&&window.performance.timing&&"undefined"!=typeof window.performance.timing.navigationStart}},{}],ee:[function(t,e,n){function r(){}function o(t){function e(t){return t&&t instanceof r?t:t?s(t,c,i):i()}function n(n,r,o,i){if(!p.aborted||i){t&&t(n,r,o);for(var a=e(o),c=m(n),s=c.length,f=0;f<s;f++)c[f].apply(a,r);var d=u[y[n]];return d&&d.push([x,n,r,a]),a}}function l(t,e){g[t]=m(t).concat(e)}function h(t,e){var n=g[t];if(n)for(var r=0;r<n.length;r++)n[r]===e&&n.splice(r,1)}function m(t){return g[t]||[]}function v(t){return d[t]=d[t]||o(n)}function w(t,e){f(t,function(t,n){e=e||"feature",y[n]=e,e in u||(u[e]=[])})}var g={},y={},x={on:l,addEventListener:l,removeEventListener:h,emit:n,get:v,listeners:m,context:e,buffer:w,abort:a,aborted:!1};return x}function i(){return new r}function a(){(u.api||u.feature)&&(p.aborted=!0,u=p.backlog={})}var c="nr@context",s=t("gos"),f=t(26),u={},d={},p=e.exports=o();p.backlog=u},{}],gos:[function(t,e,n){function r(t,e,n){if(o.call(t,e))return t[e];var r=n();if(Object.defineProperty&&Object.keys)try{return Object.defineProperty(t,e,{value:r,writable:!0,enumerable:!1}),r}catch(i){}return t[e]=r,r}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],handle:[function(t,e,n){function r(t,e,n,r){o.buffer([t],r),o.emit(t,e,n)}var o=t("ee").get("handle");e.exports=r,r.ee=o},{}],id:[function(t,e,n){function r(t){var e=typeof t;return!t||"object"!==e&&"function"!==e?-1:t===window?0:a(t,i,function(){return o++})}var o=1,i="nr@id",a=t("gos");e.exports=r},{}],loader:[function(t,e,n){function r(){if(!E++){var t=b.info=NREUM.info,e=l.getElementsByTagName("script")[0];if(setTimeout(u.abort,3e4),!(t&&t.licenseKey&&t.applicationID&&e))return u.abort();f(y,function(e,n){t[e]||(t[e]=n)});var n=a();s("mark",["onload",n+b.offset],null,"api"),s("timing",["load",n]);var r=l.createElement("script");r.src="https://"+t.agent,e.parentNode.insertBefore(r,e)}}function o(){"complete"===l.readyState&&i()}function i(){s("mark",["domContent",a()+b.offset],null,"api")}function a(){return R.exists&&performance.now?Math.round(performance.now()):(c=Math.max((new Date).getTime(),c))-b.offset}var c=(new Date).getTime(),s=t("handle"),f=t(26),u=t("ee"),d=t(24),p=window,l=p.document,h="addEventListener",m="attachEvent",v=p.XMLHttpRequest,w=v&&v.prototype;NREUM.o={ST:setTimeout,SI:p.setImmediate,CT:clearTimeout,XHR:v,REQ:p.Request,EV:p.Event,PR:p.Promise,MO:p.MutationObserver};var g=""+location,y={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",agent:"js-agent.newrelic.com/nr-spa-1177.min.js"},x=v&&w&&w[h]&&!/CriOS/.test(navigator.userAgent),b=e.exports={offset:c,now:a,origin:g,features:{},xhrWrappable:x,userAgent:d};t(19),t(22),l[h]?(l[h]("DOMContentLoaded",i,!1),p[h]("load",r,!1)):(l[m]("onreadystatechange",o),p[m]("onload",r)),s("mark",["firstbyte",c],null,"api");var E=0,R=t(28)},{}],"wrap-function":[function(t,e,n){function r(t){return!(t&&t instanceof Function&&t.apply&&!t[a])}var o=t("ee"),i=t(27),a="nr@original",c=Object.prototype.hasOwnProperty,s=!1;e.exports=function(t,e){function n(t,e,n,o){function nrWrapper(){var r,a,c,s;try{a=this,r=i(arguments),c="function"==typeof n?n(r,a):n||{}}catch(f){p([f,"",[r,a,o],c])}u(e+"start",[r,a,o],c);try{return s=t.apply(a,r)}catch(d){throw u(e+"err",[r,a,d],c),d}finally{u(e+"end",[r,a,s],c)}}return r(t)?t:(e||(e=""),nrWrapper[a]=t,d(t,nrWrapper),nrWrapper)}function f(t,e,o,i){o||(o="");var a,c,s,f="-"===o.charAt(0);for(s=0;s<e.length;s++)c=e[s],a=t[c],r(a)||(t[c]=n(a,f?c+o:o,i,c))}function u(n,r,o){if(!s||e){var i=s;s=!0;try{t.emit(n,r,o,e)}catch(a){p([a,n,r,o])}s=i}}function d(t,e){if(Object.defineProperty&&Object.keys)try{var n=Object.keys(t);return n.forEach(function(n){Object.defineProperty(e,n,{get:function(){return t[n]},set:function(e){return t[n]=e,e}})}),e}catch(r){p([r])}for(var o in t)c.call(t,o)&&(e[o]=t[o]);return e}function p(e){try{t.emit("internal-error",e)}catch(n){}}return t||(t=o),n.inPlace=f,n.flag=a,n}},{}]},{},["loader",2,16,5,3,4]);
    ;NREUM.loader_config={accountID:"2095297",trustKey:"2038175",agentID:"916957862",licenseKey:"16e2192192",applicationID:"916957862"}
    ;NREUM.info={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",licenseKey:"16e2192192",applicationID:"916957862",sa:1}
</script>
    <script src="https://cdn.cookielaw.org/scripttemplates/otSDKStub.js" type="text/javascript" charset="UTF-8" data-domain-script="dada9834-04e2-4bcd-b5ba-cae44d498dfa"></script>
    <script type="text/javascript">
        function getCookie(name) {
            var value = "; " + document.cookie;
            var parts = value.split("; " + name + "=");
            if (parts.length == 2) {
                return true;
            }
        }
            function OptanonWrapper() {
                var optCookie = getCookie("OptanonAlertBoxClosed");
                var isActiveCategory = "false";
                var isFoundCookieBox = "false";
                if(isActiveCategory == 'false' && isFoundCookieBox == 'false' && optCookie) {
                    OneTrust.InsertScript('/pb-assets/adobe/prod/adobedtm-1616606406233.js', 'head', null, null, '2');
                }
            }
    </script>


<noscript>
    <style>.requiresJS {
        display: none
    }

    .dartAd {
        display: block !important
    }</style>
</noscript>
        <script type="text/javascript" src="/wro/o4ny~selected-legacy.js"></script>
        <link rel="stylesheet" type="text/css" href="/wro/o4ny~selected-legacy.css">
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Open+Sans">
        <script>window.MSInputMethodContext && document.documentMode && document.write('<script src="https://cdn.jsdelivr.net/gh/nuxodin/ie11CustomProperties@4.1.0/ie11CustomProperties.min.js"><\x2fscript>');</script><link rel="stylesheet" href="/products/marlin/cell/releasedAssets/css/build-b26f3342b722959c7931.css"><link rel="stylesheet" href="/products/marlin/cell/releasedAssets/css/print-b26f3342b722959c7931.css" media="print"><link rel="stylesheet" href="/products/marlin/cell/releasedAssets/css/themes/theme-cell.css">







        
        
        <script class="optanon-category-4" type="text/plain" async="" src="https://www.youtube.com/iframe_api"></script>
        
    





<!---->
<meta http-equiv="X-UA-Compatible" content="IE=edge">












<script src="https://cdn.cookielaw.org/scripttemplates/202411.2.0/otBannerSdk.js" async="" type="text/javascript"></script><style id="onetrust-style">#onetrust-banner-sdk .onetrust-vendors-list-handler{cursor:pointer;color:#1f96db;font-size:inherit;font-weight:bold;text-decoration:none;margin-left:5px}#onetrust-banner-sdk .onetrust-vendors-list-handler:hover{color:#1f96db}#onetrust-banner-sdk:focus{outline:2px solid #000;outline-offset:-2px}#onetrust-banner-sdk a:focus{outline:2px solid #000}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{outline-offset:1px}#onetrust-banner-sdk.ot-bnr-w-logo .ot-bnr-logo{height:64px;width:64px}#onetrust-banner-sdk .ot-tcf2-vendor-count.ot-text-bold{font-weight:bold}#onetrust-banner-sdk .ot-close-icon,#onetrust-pc-sdk .ot-close-icon,#ot-sync-ntfy .ot-close-icon{background-size:contain;background-repeat:no-repeat;background-position:center;height:12px;width:12px}#onetrust-banner-sdk .powered-by-logo,#onetrust-banner-sdk .ot-pc-footer-logo a,#onetrust-pc-sdk .powered-by-logo,#onetrust-pc-sdk .ot-pc-footer-logo a,#ot-sync-ntfy .powered-by-logo,#ot-sync-ntfy .ot-pc-footer-logo a{background-size:contain;background-repeat:no-repeat;background-position:center;height:25px;width:152px;display:block;text-decoration:none;font-size:.75em}#onetrust-banner-sdk .powered-by-logo:hover,#onetrust-banner-sdk .ot-pc-footer-logo a:hover,#onetrust-pc-sdk .powered-by-logo:hover,#onetrust-pc-sdk .ot-pc-footer-logo a:hover,#ot-sync-ntfy .powered-by-logo:hover,#ot-sync-ntfy .ot-pc-footer-logo a:hover{color:#565656}#onetrust-banner-sdk h3 *,#onetrust-banner-sdk h4 *,#onetrust-banner-sdk h6 *,#onetrust-banner-sdk button *,#onetrust-banner-sdk a[data-parent-id] *,#onetrust-pc-sdk h3 *,#onetrust-pc-sdk h4 *,#onetrust-pc-sdk h6 *,#onetrust-pc-sdk button *,#onetrust-pc-sdk a[data-parent-id] *,#ot-sync-ntfy h3 *,#ot-sync-ntfy h4 *,#ot-sync-ntfy h6 *,#ot-sync-ntfy button *,#ot-sync-ntfy a[data-parent-id] *{font-size:inherit;font-weight:inherit;color:inherit}#onetrust-banner-sdk .ot-hide,#onetrust-pc-sdk .ot-hide,#ot-sync-ntfy .ot-hide{display:none !important}#onetrust-banner-sdk button.ot-link-btn:hover,#onetrust-pc-sdk button.ot-link-btn:hover,#ot-sync-ntfy button.ot-link-btn:hover{text-decoration:underline;opacity:1}#onetrust-pc-sdk .ot-sdk-row .ot-sdk-column{padding:0}#onetrust-pc-sdk .ot-sdk-container{padding-right:0}#onetrust-pc-sdk .ot-sdk-row{flex-direction:initial;width:100%}#onetrust-pc-sdk [type=checkbox]:checked,#onetrust-pc-sdk [type=checkbox]:not(:checked){pointer-events:initial}#onetrust-pc-sdk [type=checkbox]:disabled+label::before,#onetrust-pc-sdk [type=checkbox]:disabled+label:after,#onetrust-pc-sdk [type=checkbox]:disabled+label{pointer-events:none;opacity:.8}#onetrust-pc-sdk #vendor-list-content{transform:translate3d(0, 0, 0)}#onetrust-pc-sdk li input[type=checkbox]{z-index:1}#onetrust-pc-sdk li .ot-checkbox label{z-index:2}#onetrust-pc-sdk li .ot-checkbox input[type=checkbox]{height:auto;width:auto}#onetrust-pc-sdk li .host-title a,#onetrust-pc-sdk li .ot-host-name a,#onetrust-pc-sdk li .accordion-text,#onetrust-pc-sdk li .ot-acc-txt{z-index:2;position:relative}#onetrust-pc-sdk input{margin:3px .1ex}#onetrust-pc-sdk .pc-logo,#onetrust-pc-sdk .ot-pc-logo{height:60px;width:180px;background-position:center;background-size:contain;background-repeat:no-repeat;display:inline-flex;justify-content:center;align-items:center}#onetrust-pc-sdk .pc-logo img,#onetrust-pc-sdk .ot-pc-logo img{max-height:100%;max-width:100%}#onetrust-pc-sdk .screen-reader-only,#onetrust-pc-sdk .ot-scrn-rdr,.ot-sdk-cookie-policy .screen-reader-only,.ot-sdk-cookie-policy .ot-scrn-rdr{border:0;clip:rect(0 0 0 0);height:1px;margin:-1px;overflow:hidden;padding:0;position:absolute;width:1px}#onetrust-pc-sdk.ot-fade-in,.onetrust-pc-dark-filter.ot-fade-in,#onetrust-banner-sdk.ot-fade-in{animation-name:onetrust-fade-in;animation-duration:400ms;animation-timing-function:ease-in-out}#onetrust-pc-sdk.ot-hide{display:none !important}.onetrust-pc-dark-filter.ot-hide{display:none !important}#ot-sdk-btn.ot-sdk-show-settings,#ot-sdk-btn.optanon-show-settings{color:#fff;background-color:#468254;height:auto;white-space:normal;word-wrap:break-word;padding:.8em 2em;font-size:.8em;line-height:1.2;cursor:pointer;-moz-transition:.1s ease;-o-transition:.1s ease;-webkit-transition:1s ease;transition:.1s ease}#ot-sdk-btn.ot-sdk-show-settings:hover,#ot-sdk-btn.optanon-show-settings:hover{color:#fff;background-color:#2c6415}#ot-sdk-btn.ot-sdk-show-settings:active,#ot-sdk-btn.optanon-show-settings:active{color:#fff;background-color:#2c6415;border:1px solid rgba(162,192,169,.5)}.onetrust-pc-dark-filter{background:rgba(0,0,0,.5);z-index:2147483646;width:100%;height:100%;overflow:hidden;position:fixed;top:0;bottom:0;left:0}@keyframes onetrust-fade-in{0%{opacity:0}100%{opacity:1}}.ot-cookie-label{text-decoration:underline}@media only screen and (min-width: 426px)and (max-width: 896px)and (orientation: landscape){#onetrust-pc-sdk p{font-size:.75em}}#onetrust-banner-sdk .banner-option-input:focus+label{outline:1px solid #000;outline-style:auto}.category-vendors-list-handler+a:focus,.category-vendors-list-handler+a:focus-visible{outline:2px solid #000}#onetrust-pc-sdk .ot-userid-title{margin-top:10px}#onetrust-pc-sdk .ot-userid-title>span,#onetrust-pc-sdk .ot-userid-timestamp>span{font-weight:700}#onetrust-pc-sdk .ot-userid-desc{font-style:italic}#onetrust-pc-sdk .ot-host-desc a{pointer-events:initial}#onetrust-pc-sdk .ot-ven-hdr>p a{position:relative;z-index:2;pointer-events:initial}#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info a,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info a{margin-right:auto}#onetrust-pc-sdk .ot-pc-footer-logo img{width:136px;height:16px}#onetrust-pc-sdk .ot-pur-vdr-count{font-weight:400;font-size:.7rem;padding-top:3px;display:block}#onetrust-banner-sdk .ot-optout-signal,#onetrust-pc-sdk .ot-optout-signal{border:1px solid #32ae88;border-radius:3px;padding:5px;margin-bottom:10px;background-color:#f9fffa;font-size:.85rem;line-height:2}#onetrust-banner-sdk .ot-optout-signal .ot-optout-icon,#onetrust-pc-sdk .ot-optout-signal .ot-optout-icon{display:inline;margin-right:5px}#onetrust-banner-sdk .ot-optout-signal svg,#onetrust-pc-sdk .ot-optout-signal svg{height:20px;width:30px;transform:scale(0.5)}#onetrust-banner-sdk .ot-optout-signal svg path,#onetrust-pc-sdk .ot-optout-signal svg path{fill:#32ae88}#onetrust-consent-sdk .ot-general-modal{overflow:hidden;position:fixed;margin:0 auto;top:50%;left:50%;width:40%;padding:1.5rem;max-width:575px;min-width:575px;z-index:2147483647;border-radius:2.5px;transform:translate(-50%, -50%)}#onetrust-consent-sdk .ot-signature-health-group{margin-top:1rem;padding-left:1.25rem;padding-right:1.25rem;margin-bottom:.625rem;width:calc(100% - 2.5rem)}#onetrust-consent-sdk .ot-signature-health-group .ot-signature-health-form{gap:.5rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-health-form{width:70%;gap:.35rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-input{height:38px;padding:6px 10px;background-color:#fff;border:1px solid #d1d1d1;border-radius:4px;box-shadow:none;box-sizing:border-box}#onetrust-consent-sdk .ot-signature-health .ot-signature-subtitle{font-size:1.125rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-group-title{font-size:1.25rem;font-weight:bold}#onetrust-consent-sdk .ot-signature-health,#onetrust-consent-sdk .ot-signature-health-group{display:flex;flex-direction:column;gap:1rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-cont,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-cont{display:flex;flex-direction:column;gap:.25rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-paragraph,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-paragraph{margin:0;line-height:20px;font-size:max(14px,.875rem)}#onetrust-consent-sdk .ot-signature-health .ot-health-signature-error,#onetrust-consent-sdk .ot-signature-health-group .ot-health-signature-error{color:#4d4d4d;font-size:min(12px,.75rem)}#onetrust-consent-sdk .ot-signature-health .ot-signature-buttons-cont,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-buttons-cont{margin-top:max(.75rem,2%);gap:1rem;display:flex;justify-content:flex-end}#onetrust-consent-sdk .ot-signature-health .ot-signature-button,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-button{flex:1;height:auto;color:#fff;cursor:pointer;line-height:1.2;min-width:125px;font-weight:600;font-size:.813em;border-radius:2px;padding:12px 10px;white-space:normal;word-wrap:break-word;word-break:break-word;background-color:#68b631;border:2px solid #68b631}#onetrust-consent-sdk .ot-signature-health .ot-signature-button.reject,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-button.reject{background-color:#fff}#onetrust-consent-sdk .ot-input-field-cont{display:flex;flex-direction:column;gap:.5rem}#onetrust-consent-sdk .ot-input-field-cont .ot-signature-input{width:65%}#onetrust-consent-sdk .ot-signature-health-form{display:flex;flex-direction:column}#onetrust-consent-sdk .ot-signature-health-form .ot-signature-label{margin-bottom:0;line-height:20px;font-size:max(14px,.875rem)}@media only screen and (max-width: 600px){#onetrust-consent-sdk .ot-general-modal{min-width:100%}#onetrust-consent-sdk .ot-signature-health .ot-signature-health-form{width:100%}#onetrust-consent-sdk .ot-input-field-cont .ot-signature-input{width:100%}}#onetrust-banner-sdk,#onetrust-pc-sdk,#ot-sdk-cookie-policy,#ot-sync-ntfy{font-size:16px}#onetrust-banner-sdk *,#onetrust-banner-sdk ::after,#onetrust-banner-sdk ::before,#onetrust-pc-sdk *,#onetrust-pc-sdk ::after,#onetrust-pc-sdk ::before,#ot-sdk-cookie-policy *,#ot-sdk-cookie-policy ::after,#ot-sdk-cookie-policy ::before,#ot-sync-ntfy *,#ot-sync-ntfy ::after,#ot-sync-ntfy ::before{-webkit-box-sizing:content-box;-moz-box-sizing:content-box;box-sizing:content-box}#onetrust-banner-sdk div,#onetrust-banner-sdk span,#onetrust-banner-sdk h1,#onetrust-banner-sdk h2,#onetrust-banner-sdk h3,#onetrust-banner-sdk h4,#onetrust-banner-sdk h5,#onetrust-banner-sdk h6,#onetrust-banner-sdk p,#onetrust-banner-sdk img,#onetrust-banner-sdk svg,#onetrust-banner-sdk button,#onetrust-banner-sdk section,#onetrust-banner-sdk a,#onetrust-banner-sdk label,#onetrust-banner-sdk input,#onetrust-banner-sdk ul,#onetrust-banner-sdk li,#onetrust-banner-sdk nav,#onetrust-banner-sdk table,#onetrust-banner-sdk thead,#onetrust-banner-sdk tr,#onetrust-banner-sdk td,#onetrust-banner-sdk tbody,#onetrust-banner-sdk .ot-main-content,#onetrust-banner-sdk .ot-toggle,#onetrust-banner-sdk #ot-content,#onetrust-banner-sdk #ot-pc-content,#onetrust-banner-sdk .checkbox,#onetrust-pc-sdk div,#onetrust-pc-sdk span,#onetrust-pc-sdk h1,#onetrust-pc-sdk h2,#onetrust-pc-sdk h3,#onetrust-pc-sdk h4,#onetrust-pc-sdk h5,#onetrust-pc-sdk h6,#onetrust-pc-sdk p,#onetrust-pc-sdk img,#onetrust-pc-sdk svg,#onetrust-pc-sdk button,#onetrust-pc-sdk section,#onetrust-pc-sdk a,#onetrust-pc-sdk label,#onetrust-pc-sdk input,#onetrust-pc-sdk ul,#onetrust-pc-sdk li,#onetrust-pc-sdk nav,#onetrust-pc-sdk table,#onetrust-pc-sdk thead,#onetrust-pc-sdk tr,#onetrust-pc-sdk td,#onetrust-pc-sdk tbody,#onetrust-pc-sdk .ot-main-content,#onetrust-pc-sdk .ot-toggle,#onetrust-pc-sdk #ot-content,#onetrust-pc-sdk #ot-pc-content,#onetrust-pc-sdk .checkbox,#ot-sdk-cookie-policy div,#ot-sdk-cookie-policy span,#ot-sdk-cookie-policy h1,#ot-sdk-cookie-policy h2,#ot-sdk-cookie-policy h3,#ot-sdk-cookie-policy h4,#ot-sdk-cookie-policy h5,#ot-sdk-cookie-policy h6,#ot-sdk-cookie-policy p,#ot-sdk-cookie-policy img,#ot-sdk-cookie-policy svg,#ot-sdk-cookie-policy button,#ot-sdk-cookie-policy section,#ot-sdk-cookie-policy a,#ot-sdk-cookie-policy label,#ot-sdk-cookie-policy input,#ot-sdk-cookie-policy ul,#ot-sdk-cookie-policy li,#ot-sdk-cookie-policy nav,#ot-sdk-cookie-policy table,#ot-sdk-cookie-policy thead,#ot-sdk-cookie-policy tr,#ot-sdk-cookie-policy td,#ot-sdk-cookie-policy tbody,#ot-sdk-cookie-policy .ot-main-content,#ot-sdk-cookie-policy .ot-toggle,#ot-sdk-cookie-policy #ot-content,#ot-sdk-cookie-policy #ot-pc-content,#ot-sdk-cookie-policy .checkbox,#ot-sync-ntfy div,#ot-sync-ntfy span,#ot-sync-ntfy h1,#ot-sync-ntfy h2,#ot-sync-ntfy h3,#ot-sync-ntfy h4,#ot-sync-ntfy h5,#ot-sync-ntfy h6,#ot-sync-ntfy p,#ot-sync-ntfy img,#ot-sync-ntfy svg,#ot-sync-ntfy button,#ot-sync-ntfy section,#ot-sync-ntfy a,#ot-sync-ntfy label,#ot-sync-ntfy input,#ot-sync-ntfy ul,#ot-sync-ntfy li,#ot-sync-ntfy nav,#ot-sync-ntfy table,#ot-sync-ntfy thead,#ot-sync-ntfy tr,#ot-sync-ntfy td,#ot-sync-ntfy tbody,#ot-sync-ntfy .ot-main-content,#ot-sync-ntfy .ot-toggle,#ot-sync-ntfy #ot-content,#ot-sync-ntfy #ot-pc-content,#ot-sync-ntfy .checkbox{font-family:inherit;font-weight:normal;-webkit-font-smoothing:auto;letter-spacing:normal;line-height:normal;padding:0;margin:0;height:auto;min-height:0;max-height:none;width:auto;min-width:0;max-width:none;border-radius:0;border:none;clear:none;float:none;position:static;bottom:auto;left:auto;right:auto;top:auto;text-align:left;text-decoration:none;text-indent:0;text-shadow:none;text-transform:none;white-space:normal;background:none;overflow:visible;vertical-align:baseline;visibility:visible;z-index:auto;box-shadow:none}#onetrust-banner-sdk label:before,#onetrust-banner-sdk label:after,#onetrust-banner-sdk .checkbox:after,#onetrust-banner-sdk .checkbox:before,#onetrust-pc-sdk label:before,#onetrust-pc-sdk label:after,#onetrust-pc-sdk .checkbox:after,#onetrust-pc-sdk .checkbox:before,#ot-sdk-cookie-policy label:before,#ot-sdk-cookie-policy label:after,#ot-sdk-cookie-policy .checkbox:after,#ot-sdk-cookie-policy .checkbox:before,#ot-sync-ntfy label:before,#ot-sync-ntfy label:after,#ot-sync-ntfy .checkbox:after,#ot-sync-ntfy .checkbox:before{content:"";content:none}#onetrust-banner-sdk .ot-sdk-container,#onetrust-pc-sdk .ot-sdk-container,#ot-sdk-cookie-policy .ot-sdk-container{position:relative;width:100%;max-width:100%;margin:0 auto;padding:0 20px;box-sizing:border-box}#onetrust-banner-sdk .ot-sdk-column,#onetrust-banner-sdk .ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-column,#onetrust-pc-sdk .ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-column,#ot-sdk-cookie-policy .ot-sdk-columns{width:100%;float:left;box-sizing:border-box;padding:0;display:initial}@media(min-width: 400px){#onetrust-banner-sdk .ot-sdk-container,#onetrust-pc-sdk .ot-sdk-container,#ot-sdk-cookie-policy .ot-sdk-container{width:90%;padding:0}}@media(min-width: 550px){#onetrust-banner-sdk .ot-sdk-container,#onetrust-pc-sdk .ot-sdk-container,#ot-sdk-cookie-policy .ot-sdk-container{width:100%}#onetrust-banner-sdk .ot-sdk-column,#onetrust-banner-sdk .ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-column,#onetrust-pc-sdk .ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-column,#ot-sdk-cookie-policy .ot-sdk-columns{margin-left:4%}#onetrust-banner-sdk .ot-sdk-column:first-child,#onetrust-banner-sdk .ot-sdk-columns:first-child,#onetrust-pc-sdk .ot-sdk-column:first-child,#onetrust-pc-sdk .ot-sdk-columns:first-child,#ot-sdk-cookie-policy .ot-sdk-column:first-child,#ot-sdk-cookie-policy .ot-sdk-columns:first-child{margin-left:0}#onetrust-banner-sdk .ot-sdk-two.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-two.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-two.ot-sdk-columns{width:13.3333333333%}#onetrust-banner-sdk .ot-sdk-three.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-three.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-three.ot-sdk-columns{width:22%}#onetrust-banner-sdk .ot-sdk-four.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-four.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-four.ot-sdk-columns{width:30.6666666667%}#onetrust-banner-sdk .ot-sdk-eight.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-eight.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-eight.ot-sdk-columns{width:65.3333333333%}#onetrust-banner-sdk .ot-sdk-nine.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-nine.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-nine.ot-sdk-columns{width:74%}#onetrust-banner-sdk .ot-sdk-ten.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-ten.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-ten.ot-sdk-columns{width:82.6666666667%}#onetrust-banner-sdk .ot-sdk-eleven.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-eleven.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-eleven.ot-sdk-columns{width:91.3333333333%}#onetrust-banner-sdk .ot-sdk-twelve.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-twelve.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-twelve.ot-sdk-columns{width:100%;margin-left:0}}#onetrust-banner-sdk h1,#onetrust-banner-sdk h2,#onetrust-banner-sdk h3,#onetrust-banner-sdk h4,#onetrust-banner-sdk h5,#onetrust-banner-sdk h6,#onetrust-pc-sdk h1,#onetrust-pc-sdk h2,#onetrust-pc-sdk h3,#onetrust-pc-sdk h4,#onetrust-pc-sdk h5,#onetrust-pc-sdk h6,#ot-sdk-cookie-policy h1,#ot-sdk-cookie-policy h2,#ot-sdk-cookie-policy h3,#ot-sdk-cookie-policy h4,#ot-sdk-cookie-policy h5,#ot-sdk-cookie-policy h6{margin-top:0;font-weight:600;font-family:inherit}#onetrust-banner-sdk h1,#onetrust-pc-sdk h1,#ot-sdk-cookie-policy h1{font-size:1.5rem;line-height:1.2}#onetrust-banner-sdk h2,#onetrust-pc-sdk h2,#ot-sdk-cookie-policy h2{font-size:1.5rem;line-height:1.25}#onetrust-banner-sdk h3,#onetrust-pc-sdk h3,#ot-sdk-cookie-policy h3{font-size:1.5rem;line-height:1.3}#onetrust-banner-sdk h4,#onetrust-pc-sdk h4,#ot-sdk-cookie-policy h4{font-size:1.5rem;line-height:1.35}#onetrust-banner-sdk h5,#onetrust-pc-sdk h5,#ot-sdk-cookie-policy h5{font-size:1.5rem;line-height:1.5}#onetrust-banner-sdk h6,#onetrust-pc-sdk h6,#ot-sdk-cookie-policy h6{font-size:1.5rem;line-height:1.6}@media(min-width: 550px){#onetrust-banner-sdk h1,#onetrust-pc-sdk h1,#ot-sdk-cookie-policy h1{font-size:1.5rem}#onetrust-banner-sdk h2,#onetrust-pc-sdk h2,#ot-sdk-cookie-policy h2{font-size:1.5rem}#onetrust-banner-sdk h3,#onetrust-pc-sdk h3,#ot-sdk-cookie-policy h3{font-size:1.5rem}#onetrust-banner-sdk h4,#onetrust-pc-sdk h4,#ot-sdk-cookie-policy h4{font-size:1.5rem}#onetrust-banner-sdk h5,#onetrust-pc-sdk h5,#ot-sdk-cookie-policy h5{font-size:1.5rem}#onetrust-banner-sdk h6,#onetrust-pc-sdk h6,#ot-sdk-cookie-policy h6{font-size:1.5rem}}#onetrust-banner-sdk p,#onetrust-pc-sdk p,#ot-sdk-cookie-policy p{margin:0 0 1em 0;font-family:inherit;line-height:normal}#onetrust-banner-sdk a,#onetrust-pc-sdk a,#ot-sdk-cookie-policy a{color:#565656;text-decoration:underline}#onetrust-banner-sdk a:hover,#onetrust-pc-sdk a:hover,#ot-sdk-cookie-policy a:hover{color:#565656;text-decoration:none}#onetrust-banner-sdk .ot-sdk-button,#onetrust-banner-sdk button,#onetrust-pc-sdk .ot-sdk-button,#onetrust-pc-sdk button,#ot-sdk-cookie-policy .ot-sdk-button,#ot-sdk-cookie-policy button{margin-bottom:1rem;font-family:inherit}#onetrust-banner-sdk .ot-sdk-button,#onetrust-banner-sdk button,#onetrust-pc-sdk .ot-sdk-button,#onetrust-pc-sdk button,#ot-sdk-cookie-policy .ot-sdk-button,#ot-sdk-cookie-policy button{display:inline-block;height:38px;padding:0 30px;color:#555;text-align:center;font-size:.9em;font-weight:400;line-height:38px;letter-spacing:.01em;text-decoration:none;white-space:nowrap;background-color:rgba(0,0,0,0);border-radius:2px;border:1px solid #bbb;cursor:pointer;box-sizing:border-box}#onetrust-banner-sdk .ot-sdk-button:hover,#onetrust-banner-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):hover,#onetrust-banner-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):focus,#onetrust-pc-sdk .ot-sdk-button:hover,#onetrust-pc-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):hover,#onetrust-pc-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):focus,#ot-sdk-cookie-policy .ot-sdk-button:hover,#ot-sdk-cookie-policy :not(.ot-leg-btn-container)>button:not(.ot-link-btn):hover,#ot-sdk-cookie-policy :not(.ot-leg-btn-container)>button:not(.ot-link-btn):focus{color:#333;border-color:#888;opacity:.9}#onetrust-banner-sdk .ot-sdk-button:focus,#onetrust-banner-sdk :not(.ot-leg-btn-container)>button:focus,#onetrust-pc-sdk .ot-sdk-button:focus,#onetrust-pc-sdk :not(.ot-leg-btn-container)>button:focus,#ot-sdk-cookie-policy .ot-sdk-button:focus,#ot-sdk-cookie-policy :not(.ot-leg-btn-container)>button:focus{outline:2px solid #000}#onetrust-banner-sdk .ot-sdk-button.ot-sdk-button-primary,#onetrust-banner-sdk button.ot-sdk-button-primary,#onetrust-banner-sdk input[type=submit].ot-sdk-button-primary,#onetrust-banner-sdk input[type=reset].ot-sdk-button-primary,#onetrust-banner-sdk input[type=button].ot-sdk-button-primary,#onetrust-pc-sdk .ot-sdk-button.ot-sdk-button-primary,#onetrust-pc-sdk button.ot-sdk-button-primary,#onetrust-pc-sdk input[type=submit].ot-sdk-button-primary,#onetrust-pc-sdk input[type=reset].ot-sdk-button-primary,#onetrust-pc-sdk input[type=button].ot-sdk-button-primary,#ot-sdk-cookie-policy .ot-sdk-button.ot-sdk-button-primary,#ot-sdk-cookie-policy button.ot-sdk-button-primary,#ot-sdk-cookie-policy input[type=submit].ot-sdk-button-primary,#ot-sdk-cookie-policy input[type=reset].ot-sdk-button-primary,#ot-sdk-cookie-policy input[type=button].ot-sdk-button-primary{color:#fff;background-color:#33c3f0;border-color:#33c3f0}#onetrust-banner-sdk .ot-sdk-button.ot-sdk-button-primary:hover,#onetrust-banner-sdk button.ot-sdk-button-primary:hover,#onetrust-banner-sdk input[type=submit].ot-sdk-button-primary:hover,#onetrust-banner-sdk input[type=reset].ot-sdk-button-primary:hover,#onetrust-banner-sdk input[type=button].ot-sdk-button-primary:hover,#onetrust-banner-sdk .ot-sdk-button.ot-sdk-button-primary:focus,#onetrust-banner-sdk button.ot-sdk-button-primary:focus,#onetrust-banner-sdk input[type=submit].ot-sdk-button-primary:focus,#onetrust-banner-sdk input[type=reset].ot-sdk-button-primary:focus,#onetrust-banner-sdk input[type=button].ot-sdk-button-primary:focus,#onetrust-pc-sdk .ot-sdk-button.ot-sdk-button-primary:hover,#onetrust-pc-sdk button.ot-sdk-button-primary:hover,#onetrust-pc-sdk input[type=submit].ot-sdk-button-primary:hover,#onetrust-pc-sdk input[type=reset].ot-sdk-button-primary:hover,#onetrust-pc-sdk input[type=button].ot-sdk-button-primary:hover,#onetrust-pc-sdk .ot-sdk-button.ot-sdk-button-primary:focus,#onetrust-pc-sdk button.ot-sdk-button-primary:focus,#onetrust-pc-sdk input[type=submit].ot-sdk-button-primary:focus,#onetrust-pc-sdk input[type=reset].ot-sdk-button-primary:focus,#onetrust-pc-sdk input[type=button].ot-sdk-button-primary:focus,#ot-sdk-cookie-policy .ot-sdk-button.ot-sdk-button-primary:hover,#ot-sdk-cookie-policy button.ot-sdk-button-primary:hover,#ot-sdk-cookie-policy input[type=submit].ot-sdk-button-primary:hover,#ot-sdk-cookie-policy input[type=reset].ot-sdk-button-primary:hover,#ot-sdk-cookie-policy input[type=button].ot-sdk-button-primary:hover,#ot-sdk-cookie-policy .ot-sdk-button.ot-sdk-button-primary:focus,#ot-sdk-cookie-policy button.ot-sdk-button-primary:focus,#ot-sdk-cookie-policy input[type=submit].ot-sdk-button-primary:focus,#ot-sdk-cookie-policy input[type=reset].ot-sdk-button-primary:focus,#ot-sdk-cookie-policy input[type=button].ot-sdk-button-primary:focus{color:#fff;background-color:#1eaedb;border-color:#1eaedb}#onetrust-banner-sdk input[type=text],#onetrust-pc-sdk input[type=text],#ot-sdk-cookie-policy input[type=text]{height:38px;padding:6px 10px;background-color:#fff;border:1px solid #d1d1d1;border-radius:4px;box-shadow:none;box-sizing:border-box}#onetrust-banner-sdk input[type=text],#onetrust-pc-sdk input[type=text],#ot-sdk-cookie-policy input[type=text]{-webkit-appearance:none;-moz-appearance:none;appearance:none}#onetrust-banner-sdk input[type=text]:focus,#onetrust-pc-sdk input[type=text]:focus,#ot-sdk-cookie-policy input[type=text]:focus{border:1px solid #000;outline:0}#onetrust-banner-sdk label,#onetrust-pc-sdk label,#ot-sdk-cookie-policy label{display:block;margin-bottom:.5rem;font-weight:600}#onetrust-banner-sdk input[type=checkbox],#onetrust-pc-sdk input[type=checkbox],#ot-sdk-cookie-policy input[type=checkbox]{display:inline}#onetrust-banner-sdk ul,#onetrust-pc-sdk ul,#ot-sdk-cookie-policy ul{list-style:circle inside}#onetrust-banner-sdk ul,#onetrust-pc-sdk ul,#ot-sdk-cookie-policy ul{padding-left:0;margin-top:0}#onetrust-banner-sdk ul ul,#onetrust-pc-sdk ul ul,#ot-sdk-cookie-policy ul ul{margin:1.5rem 0 1.5rem 3rem;font-size:90%}#onetrust-banner-sdk li,#onetrust-pc-sdk li,#ot-sdk-cookie-policy li{margin-bottom:1rem}#onetrust-banner-sdk th,#onetrust-banner-sdk td,#onetrust-pc-sdk th,#onetrust-pc-sdk td,#ot-sdk-cookie-policy th,#ot-sdk-cookie-policy td{padding:12px 15px;text-align:left;border-bottom:1px solid #e1e1e1}#onetrust-banner-sdk button,#onetrust-pc-sdk button,#ot-sdk-cookie-policy button{margin-bottom:1rem;font-family:inherit}#onetrust-banner-sdk .ot-sdk-container:after,#onetrust-banner-sdk .ot-sdk-row:after,#onetrust-pc-sdk .ot-sdk-container:after,#onetrust-pc-sdk .ot-sdk-row:after,#ot-sdk-cookie-policy .ot-sdk-container:after,#ot-sdk-cookie-policy .ot-sdk-row:after{content:"";display:table;clear:both}#onetrust-banner-sdk .ot-sdk-row,#onetrust-pc-sdk .ot-sdk-row,#ot-sdk-cookie-policy .ot-sdk-row{margin:0;max-width:none;display:block}#onetrust-banner-sdk{box-shadow:0 0 18px rgba(0,0,0,.2)}#onetrust-banner-sdk.otFlat{position:fixed;z-index:2147483645;bottom:0;right:0;left:0;background-color:#fff;max-height:90%;overflow-x:hidden;overflow-y:auto}#onetrust-banner-sdk.otFlat.top{top:0px;bottom:auto}#onetrust-banner-sdk.otRelFont{font-size:1rem}#onetrust-banner-sdk>.ot-sdk-container{overflow:hidden}#onetrust-banner-sdk::-webkit-scrollbar{width:11px}#onetrust-banner-sdk::-webkit-scrollbar-thumb{border-radius:10px;background:#c1c1c1}#onetrust-banner-sdk{scrollbar-arrow-color:#c1c1c1;scrollbar-darkshadow-color:#c1c1c1;scrollbar-face-color:#c1c1c1;scrollbar-shadow-color:#c1c1c1}#onetrust-banner-sdk #onetrust-policy{margin:1.25em 0 .625em 2em;overflow:hidden}#onetrust-banner-sdk #onetrust-policy .ot-gv-list-handler{float:left;font-size:.82em;padding:0;margin-bottom:0;border:0;line-height:normal;height:auto;width:auto}#onetrust-banner-sdk #onetrust-policy-title{font-size:1.2em;line-height:1.3;margin-bottom:10px}#onetrust-banner-sdk #onetrust-policy-text{clear:both;text-align:left;font-size:.88em;line-height:1.4}#onetrust-banner-sdk #onetrust-policy-text *{font-size:inherit;line-height:inherit}#onetrust-banner-sdk #onetrust-policy-text a{font-weight:bold;margin-left:5px}#onetrust-banner-sdk #onetrust-policy-title,#onetrust-banner-sdk #onetrust-policy-text{color:dimgray;float:left}#onetrust-banner-sdk #onetrust-button-group-parent{min-height:1px;text-align:center}#onetrust-banner-sdk #onetrust-button-group{display:inline-block}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{background-color:#68b631;color:#fff;border-color:#68b631;margin-right:1em;min-width:125px;height:auto;white-space:normal;word-break:break-word;word-wrap:break-word;padding:12px 10px;line-height:1.2;font-size:.813em;font-weight:600}#onetrust-banner-sdk #onetrust-pc-btn-handler.cookie-setting-link{background-color:#fff;border:none;color:#68b631;text-decoration:underline;padding-left:0;padding-right:0}#onetrust-banner-sdk .onetrust-close-btn-ui{width:44px;height:44px;background-size:12px;border:none;position:relative;margin:auto;padding:0}#onetrust-banner-sdk .banner_logo{display:none}#onetrust-banner-sdk.ot-bnr-w-logo .ot-bnr-logo{position:absolute;top:50%;transform:translateY(-50%);left:0px}#onetrust-banner-sdk.ot-bnr-w-logo #onetrust-policy{margin-left:65px}#onetrust-banner-sdk .ot-b-addl-desc{clear:both;float:left;display:block}#onetrust-banner-sdk #banner-options{float:left;display:table;margin-right:0;margin-left:1em;width:calc(100% - 1em)}#onetrust-banner-sdk .banner-option-input{cursor:pointer;width:auto;height:auto;border:none;padding:0;padding-right:3px;margin:0 0 10px;font-size:.82em;line-height:1.4}#onetrust-banner-sdk .banner-option-input *{pointer-events:none;font-size:inherit;line-height:inherit}#onetrust-banner-sdk .banner-option-input[aria-expanded=true]~.banner-option-details{display:block;height:auto}#onetrust-banner-sdk .banner-option-input[aria-expanded=true] .ot-arrow-container{transform:rotate(90deg)}#onetrust-banner-sdk .banner-option{margin-bottom:12px;margin-left:0;border:none;float:left;padding:0}#onetrust-banner-sdk .banner-option:first-child{padding-left:2px}#onetrust-banner-sdk .banner-option:not(:first-child){padding:0;border:none}#onetrust-banner-sdk .banner-option-header{cursor:pointer;display:inline-block}#onetrust-banner-sdk .banner-option-header :first-child{color:dimgray;font-weight:bold;float:left}#onetrust-banner-sdk .banner-option-header .ot-arrow-container{display:inline-block;border-top:6px solid rgba(0,0,0,0);border-bottom:6px solid rgba(0,0,0,0);border-left:6px solid dimgray;margin-left:10px;vertical-align:middle}#onetrust-banner-sdk .banner-option-details{display:none;font-size:.83em;line-height:1.5;padding:10px 0px 5px 10px;margin-right:10px;height:0px}#onetrust-banner-sdk .banner-option-details *{font-size:inherit;line-height:inherit;color:dimgray}#onetrust-banner-sdk .ot-arrow-container,#onetrust-banner-sdk .banner-option-details{transition:all 300ms ease-in 0s;-webkit-transition:all 300ms ease-in 0s;-moz-transition:all 300ms ease-in 0s;-o-transition:all 300ms ease-in 0s}#onetrust-banner-sdk .ot-dpd-container{float:left}#onetrust-banner-sdk .ot-dpd-title{margin-bottom:10px}#onetrust-banner-sdk .ot-dpd-title,#onetrust-banner-sdk .ot-dpd-desc{font-size:.88em;line-height:1.4;color:dimgray}#onetrust-banner-sdk .ot-dpd-title *,#onetrust-banner-sdk .ot-dpd-desc *{font-size:inherit;line-height:inherit}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-text *{margin-bottom:0}#onetrust-banner-sdk.ot-iab-2 .onetrust-vendors-list-handler{display:block;margin-left:0;margin-top:5px;clear:both;margin-bottom:0;padding:0;border:0;height:auto;width:auto}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group button{display:block}#onetrust-banner-sdk.ot-close-btn-link{padding-top:25px}#onetrust-banner-sdk.ot-close-btn-link #onetrust-close-btn-container{top:15px;transform:none;right:15px}#onetrust-banner-sdk.ot-close-btn-link #onetrust-close-btn-container button{padding:0;white-space:pre-wrap;border:none;height:auto;line-height:1.5;text-decoration:underline;font-size:.69em}#onetrust-banner-sdk #onetrust-policy-text,#onetrust-banner-sdk .ot-dpd-desc,#onetrust-banner-sdk .ot-b-addl-desc{font-size:.813em;line-height:1.5}#onetrust-banner-sdk .ot-dpd-desc{margin-bottom:10px}#onetrust-banner-sdk .ot-dpd-desc>.ot-b-addl-desc{margin-top:10px;margin-bottom:10px;font-size:1em}@media only screen and (max-width: 425px){#onetrust-banner-sdk #onetrust-close-btn-container{position:absolute;top:6px;right:2px}#onetrust-banner-sdk #onetrust-policy{margin-left:0;margin-top:3em}#onetrust-banner-sdk #onetrust-button-group{display:block}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{width:100%}#onetrust-banner-sdk .onetrust-close-btn-ui{top:auto;transform:none}#onetrust-banner-sdk #onetrust-policy-title{display:inline;float:none}#onetrust-banner-sdk #banner-options{margin:0;padding:0;width:100%}}@media only screen and (min-width: 426px)and (max-width: 896px){#onetrust-banner-sdk #onetrust-close-btn-container{position:absolute;top:0;right:0}#onetrust-banner-sdk #onetrust-policy{margin-left:1em;margin-right:1em}#onetrust-banner-sdk .onetrust-close-btn-ui{top:10px;right:10px}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{width:95%}#onetrust-banner-sdk.ot-iab-2 #onetrust-group-container{width:100%}#onetrust-banner-sdk.ot-bnr-w-logo #onetrust-button-group-parent{padding-left:50px}#onetrust-banner-sdk #onetrust-button-group-parent{width:100%;position:relative;margin-left:0}#onetrust-banner-sdk #onetrust-button-group button{display:inline-block}#onetrust-banner-sdk #onetrust-button-group{margin-right:0;text-align:center}#onetrust-banner-sdk .has-reject-all-button #onetrust-pc-btn-handler{float:left}#onetrust-banner-sdk .has-reject-all-button #onetrust-reject-all-handler,#onetrust-banner-sdk .has-reject-all-button #onetrust-accept-btn-handler{float:right}#onetrust-banner-sdk .has-reject-all-button #onetrust-button-group{width:calc(100% - 2em);margin-right:0}#onetrust-banner-sdk .has-reject-all-button #onetrust-pc-btn-handler.cookie-setting-link{padding-left:0px;text-align:left}#onetrust-banner-sdk.ot-buttons-fw .ot-sdk-three button{width:100%;text-align:center}#onetrust-banner-sdk.ot-buttons-fw #onetrust-button-group-parent button{float:none}#onetrust-banner-sdk.ot-buttons-fw #onetrust-pc-btn-handler.cookie-setting-link{text-align:center}}@media only screen and (min-width: 550px){#onetrust-banner-sdk .banner-option:not(:first-child){border-left:1px solid #d8d8d8;padding-left:25px}}@media only screen and (min-width: 425px)and (max-width: 550px){#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group,#onetrust-banner-sdk.ot-iab-2 #onetrust-policy,#onetrust-banner-sdk.ot-iab-2 .banner-option{width:100%}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-accept-btn-handler,#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-reject-all-handler,#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-pc-btn-handler{width:100%}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-accept-btn-handler,#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-reject-all-handler{float:left}}@media only screen and (min-width: 769px){#onetrust-banner-sdk #onetrust-button-group{margin-right:30%}#onetrust-banner-sdk #banner-options{margin-left:2em;margin-right:5em;margin-bottom:1.25em;width:calc(100% - 7em)}}@media only screen and (min-width: 897px)and (max-width: 1023px){#onetrust-banner-sdk.vertical-align-content #onetrust-button-group-parent{position:absolute;top:50%;left:75%;transform:translateY(-50%)}#onetrust-banner-sdk #onetrust-close-btn-container{top:50%;margin:auto;transform:translate(-50%, -50%);position:absolute;padding:0;right:0}#onetrust-banner-sdk #onetrust-close-btn-container button{position:relative;margin:0;right:-22px;top:2px}}@media only screen and (min-width: 1024px){#onetrust-banner-sdk #onetrust-close-btn-container{top:50%;margin:auto;transform:translate(-50%, -50%);position:absolute;right:0}#onetrust-banner-sdk #onetrust-close-btn-container button{right:-12px}#onetrust-banner-sdk #onetrust-policy{margin-left:2em}#onetrust-banner-sdk.vertical-align-content #onetrust-button-group-parent{position:absolute;top:50%;left:60%;transform:translateY(-50%)}#onetrust-banner-sdk .ot-optout-signal{width:50%}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-title{width:50%}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-text,#onetrust-banner-sdk.ot-iab-2 :not(.ot-dpd-desc)>.ot-b-addl-desc{margin-bottom:1em;width:50%;border-right:1px solid #d8d8d8;padding-right:1rem}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-text{margin-bottom:0;padding-bottom:1em}#onetrust-banner-sdk.ot-iab-2 :not(.ot-dpd-desc)>.ot-b-addl-desc{margin-bottom:0;padding-bottom:1em}#onetrust-banner-sdk.ot-iab-2 .ot-dpd-container{width:45%;padding-left:1rem;display:inline-block;float:none}#onetrust-banner-sdk.ot-iab-2 .ot-dpd-title{line-height:1.7}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group-parent{left:auto;right:4%;margin-left:0}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group button{display:block}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{margin:auto;width:30%}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{width:60%}#onetrust-banner-sdk #onetrust-button-group{margin-right:auto}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{margin-top:1em}}@media only screen and (min-width: 890px){#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group-parent{padding-left:3%;padding-right:4%;margin-left:0}#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group{margin-right:0;margin-top:1.25em;width:100%}#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button{width:100%;margin-bottom:5px;margin-top:5px}#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button:last-of-type{margin-bottom:20px}}@media only screen and (min-width: 1280px){#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{width:55%}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{width:44%;padding-left:2%;padding-right:2%}#onetrust-banner-sdk:not(.ot-iab-2).vertical-align-content #onetrust-button-group-parent{position:absolute;left:55%}}
        #onetrust-consent-sdk #onetrust-banner-sdk {background-color: #FFF;}
            #onetrust-consent-sdk #onetrust-policy-title,
                    #onetrust-consent-sdk #onetrust-policy-text,
                    #onetrust-consent-sdk .ot-b-addl-desc,
                    #onetrust-consent-sdk .ot-dpd-desc,
                    #onetrust-consent-sdk .ot-dpd-title,
                    #onetrust-consent-sdk #onetrust-policy-text *:not(.onetrust-vendors-list-handler),
                    #onetrust-consent-sdk .ot-dpd-desc *:not(.onetrust-vendors-list-handler),
                    #onetrust-consent-sdk #onetrust-banner-sdk #banner-options *,
                    #onetrust-banner-sdk .ot-cat-header,
                    #onetrust-banner-sdk .ot-optout-signal
                    {
                        color: #2E2E2E;
                    }
            #onetrust-consent-sdk #onetrust-banner-sdk .banner-option-details {
                    background-color: #E9E9E9;}
             #onetrust-consent-sdk #onetrust-banner-sdk a[href],
                    #onetrust-consent-sdk #onetrust-banner-sdk a[href] font,
                    #onetrust-consent-sdk #onetrust-banner-sdk .ot-link-btn
                        {
                            color: #007398;
                        }#onetrust-consent-sdk #onetrust-accept-btn-handler,
                         #onetrust-banner-sdk #onetrust-reject-all-handler {
                            background-color: #007398;border-color: #007398;
                color: #FFF;
            }
            #onetrust-consent-sdk #onetrust-banner-sdk *:focus,
            #onetrust-consent-sdk #onetrust-banner-sdk:focus {
               outline-color: #000000;
               outline-width: 1px;
            }
            #onetrust-consent-sdk #onetrust-pc-btn-handler,
            #onetrust-consent-sdk #onetrust-pc-btn-handler.cookie-setting-link {
                color: #6CC04A; border-color: #6CC04A;
                background-color:
                #FFF;
            }/*! v6.10.0 2021-01-18 */
div#onetrust-consent-sdk #onetrust-banner-sdk{border-top:2px solid #d0d3d4!important;box-shadow:none;padding:24px}div#onetrust-consent-sdk button{border-radius:0!important;box-shadow:none!important;box-sizing:border-box!important;font-size:20px!important;font-weight:400!important;letter-spacing:0!important;max-width:none!important;white-space:nowrap!important}div#onetrust-consent-sdk button:not(.ot-link-btn){background-color:#007dbc!important;border:2px solid #007dbc!important;color:#fff!important;height:48px!important;padding:0 1em!important;width:auto!important}div#onetrust-consent-sdk button:hover{background-color:#fff!important;border-color:#949494!important;color:#2e2e2e!important}div#onetrust-consent-sdk button.ot-link-btn{color:#007dbc!important;font-size:16px!important;text-decoration:underline}div#onetrust-consent-sdk button.ot-link-btn:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-consent-sdk a,div#onetrust-pc-sdk a{color:#007dbc!important;text-decoration:underline!important}div#onetrust-consent-sdk a,div#onetrust-consent-sdk button,div#onetrust-consent-sdk p:hover{opacity:1!important}div#onetrust-consent-sdk a:focus,div#onetrust-consent-sdk button:focus,div#onetrust-consent-sdk input:focus{outline:2px solid #949494!important;outline-offset:1px!important}div#onetrust-banner-sdk .ot-sdk-container{padding:0;width:auto}div#onetrust-banner-sdk .ot-sdk-row{align-items:flex-start;box-sizing:border-box;display:flex;flex-direction:column;justify-content:space-between;margin:auto;max-width:1152px}div#onetrust-banner-sdk .ot-sdk-row:after{display:none}div#onetrust-banner-sdk #onetrust-group-container,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-group-container,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{flex-grow:1;width:auto}div#onetrust-banner-sdk #onetrust-policy,div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy{margin:0;overflow:visible}div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy-text,div#onetrust-consent-sdk #onetrust-policy-text{font-size:16px;line-height:24px;max-width:44em;margin:0}div#onetrust-consent-sdk #onetrust-policy-text a[href]{font-weight:400;margin-left:8px}div#onetrust-banner-sdk #onetrust-button-group-parent{flex:0 0 auto;margin:32px 0 0;width:100%}div#onetrust-banner-sdk #onetrust-button-group{display:flex;flex-direction:row;flex-wrap:wrap;justify-content:flex-end;margin:-8px}div#onetrust-banner-sdk .banner-actions-container{display:flex;flex:1 0 auto}div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button:last-of-type,div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto;margin:8px;width:auto}div#onetrust-consent-sdk #onetrust-pc-btn-handler{background-color:#fff!important;color:inherit!important}div#onetrust-banner-sdk #onetrust-close-btn-container{display:none}@media only screen and (min-width:556px){div#onetrust-consent-sdk #onetrust-banner-sdk{padding:40px}div#onetrust-banner-sdk #onetrust-policy{margin:0 40px 0 0}div#onetrust-banner-sdk .ot-sdk-row{align-items:center;flex-direction:row}div#onetrust-banner-sdk #onetrust-button-group-parent,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-button-group-parent,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{margin:0;padding:0;width:auto}div#onetrust-banner-sdk #onetrust-button-group,div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group{align-items:stretch;flex-direction:column-reverse;margin:0}div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto}}@media only screen and (min-width:768px){div#onetrust-banner-sdk #onetrust-policy{margin:0 48px 0 0}div#onetrust-consent-sdk #onetrust-banner-sdk{padding:48px}}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-size:16px;line-height:24px}div#onetrust-consent-sdk #onetrust-pc-sdk p,div#onetrust-pc-sdk #ot-pc-desc,div#onetrust-pc-sdk .category-host-list-handler,div#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{font-size:16px;font-weight:400;line-height:24px}div#onetrust-consent-sdk a:hover,div#onetrust-pc-sdk a:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-pc-sdk{border-radius:0;bottom:0;height:auto;left:0;margin:auto;max-width:100%;overflow:hidden;right:0;top:0;width:512px;max-height:800px}div#onetrust-pc-sdk .ot-pc-header{display:none}div#onetrust-pc-sdk #ot-pc-content{overscroll-behavior:contain;padding:0 12px 0 24px;margin:16px 4px 0 0;top:0;right:16px;left:0;width:auto}div#onetrust-pc-sdk #ot-category-title,div#onetrust-pc-sdk #ot-pc-title{font-size:24px;font-weight:400;line-height:32px;margin:0 0 16px}div#onetrust-pc-sdk #ot-pc-desc{padding:0}div#onetrust-pc-sdk #ot-pc-desc a{display:inline}div#onetrust-pc-sdk #accept-recommended-btn-handler{display:none!important}div#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:2px solid #007dbc;outline-offset:-1px;transition:none}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item{border-width:0 0 2px}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item:first-of-type{border-width:2px 0}div#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{padding:8px 0;width:100%}div#onetrust-pc-sdk .ot-plus-minus{transform:translateY(2px)}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item>button{background:0 0!important;border:0!important;height:44px!important;max-width:none!important;width:calc(100% - 48px)!important}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-weight:700}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr{padding:0}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item .ot-acc-grpdesc{padding:0;width:100%}div#onetrust-pc-sdk .ot-acc-grpcntr .ot-subgrp-cntr{border:0;padding:0}div#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp{margin:0}div#onetrust-pc-sdk .ot-always-active-group .ot-cat-header{width:calc(100% - 160px)}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{width:calc(100% - 88px)}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active{color:inherit;font-size:12px;font-weight:400;line-height:1.5;padding-right:48px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:before{border-radius:12px;position:absolute;right:0;top:0;content:'';background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:after{border-radius:50%;position:absolute;right:5px;top:4px;content:'';background-color:#007dbc;height:12px;width:12px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active,div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl{right:2px}div#onetrust-pc-sdk .ot-switch{display:block;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob,div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob{background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob:before{background-color:#737373;height:8px;left:4px;top:4px;width:8px}div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob:before{background-color:#007dbc;height:12px;left:0;top:2px;width:12px}div#onetrust-pc-sdk .ot-tgl input:focus+.ot-switch .ot-switch-nob{box-shadow:0 0;outline:2px solid #007dbc!important;outline-offset:1px;transition:none}div#onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-grpcntr.ot-acc-txt{background-color:transparent;padding-left:3px}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr,div#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr{overflow:visible;width:100%}div#onetrust-pc-sdk .ot-pc-footer{border-top:0 solid}div#onetrust-pc-sdk .ot-btn-container{padding-top:24px;text-align:center}div#onetrust-pc-sdk .ot-pc-footer button{margin:8px 0;background-color:#fff}div#onetrust-pc-sdk .ot-pc-footer-logo{background-color:#fff}div#onetrust-pc-sdk #ot-lst-title span{font-size:24px;font-weight:400;line-height:32px}div#onetrust-pc-sdk #ot-host-lst .ot-host-desc,div#onetrust-pc-sdk #ot-host-lst .ot-host-expand,div#onetrust-pc-sdk #ot-host-lst .ot-host-name,div#onetrust-pc-sdk #ot-host-lst .ot-host-name a,div#onetrust-pc-sdk .back-btn-handler,div#onetrust-pc-sdk .ot-host-opt li>div div{font-size:16px;font-weight:400;line-height:24px}div#onetrust-pc-sdk #ot-pc-lst{top:0}div#onetrust-pc-sdk .back-btn-handler{text-decoration:none!important}div#onetrust-pc-sdk #filter-btn-handler:hover svg{filter:invert(1)}div#onetrust-pc-sdk .back-btn-handler svg{width:16px;height:16px}div#onetrust-pc-sdk .ot-host-item>button{background:0 0!important;border:0!important;height:66px!important;max-width:none!important;width:calc(100% - 5px)!important;transform:translate(2px,2px)}div#onetrust-pc-sdk .ot-host-item{border-bottom:2px solid #b9b9b9;padding:0}div#onetrust-pc-sdk .ot-host-item .ot-acc-hdr{margin:0 0 -6px;padding:8px 0}div#onetrust-pc-sdk ul li:first-child{border-top:2px solid #b9b9b9}div#onetrust-pc-sdk .ot-host-item .ot-plus-minus{margin:0 8px 0 0}div#onetrust-pc-sdk .ot-search-cntr{width:calc(100% - 48px)}div#onetrust-pc-sdk .ot-host-opt .ot-host-info{background-color:transparent}div#onetrust-pc-sdk .ot-host-opt li>div div{padding:0}div#onetrust-pc-sdk #vendor-search-handler{border-radius:0;border-color:#939393;border-style:solid;border-width:2px 0 2px 2px;font-size:20px;height:48px;margin:0}div#onetrust-pc-sdk .ot-lst-subhdr svg{right:0;top:8px}div#onetrust-pc-sdk .ot-fltr-cntr{box-sizing:border-box;right:0;width:48px}div#onetrust-pc-sdk #filter-btn-handler{width:48px!important;padding:8px!important}div#onetrust-pc-sdk #ot-fltr-modal{max-height:400px}
#onetrust-pc-sdk.otPcCenter{overflow:hidden;position:fixed;margin:0 auto;top:5%;right:0;left:0;width:40%;max-width:575px;min-width:575px;border-radius:2.5px;z-index:2147483647;background-color:#fff;-webkit-box-shadow:0px 2px 10px -3px #999;-moz-box-shadow:0px 2px 10px -3px #999;box-shadow:0px 2px 10px -3px #999}#onetrust-pc-sdk.otPcCenter[dir=rtl]{right:0;left:0}#onetrust-pc-sdk.otRelFont{font-size:1rem}#onetrust-pc-sdk .ot-optout-signal{margin-top:.625rem}#onetrust-pc-sdk #ot-addtl-venlst .ot-arw-cntr,#onetrust-pc-sdk #ot-addtl-venlst .ot-plus-minus,#onetrust-pc-sdk .ot-hide-tgl{visibility:hidden}#onetrust-pc-sdk #ot-addtl-venlst .ot-arw-cntr *,#onetrust-pc-sdk #ot-addtl-venlst .ot-plus-minus *,#onetrust-pc-sdk .ot-hide-tgl *{visibility:hidden}#onetrust-pc-sdk #ot-gn-venlst .ot-ven-item .ot-acc-hdr{min-height:40px}#onetrust-pc-sdk .ot-pc-header{height:39px;padding:10px 0 10px 30px;border-bottom:1px solid #e9e9e9}#onetrust-pc-sdk #ot-pc-title,#onetrust-pc-sdk #ot-category-title,#onetrust-pc-sdk .ot-cat-header,#onetrust-pc-sdk #ot-lst-title,#onetrust-pc-sdk .ot-ven-hdr .ot-ven-name,#onetrust-pc-sdk .ot-always-active{font-weight:bold;color:dimgray}#onetrust-pc-sdk .ot-always-active-group .ot-cat-header{width:55%;font-weight:700}#onetrust-pc-sdk .ot-cat-item p{clear:both;float:left;margin-top:10px;margin-bottom:5px;line-height:1.5;font-size:.812em;color:dimgray}#onetrust-pc-sdk .ot-close-icon{height:44px;width:44px;background-size:10px}#onetrust-pc-sdk #ot-pc-title{float:left;font-size:1em;line-height:1.5;margin-bottom:10px;margin-top:10px;width:100%}#onetrust-pc-sdk #accept-recommended-btn-handler{margin-right:10px;margin-bottom:25px;outline-offset:-1px}#onetrust-pc-sdk #ot-pc-desc{clear:both;width:100%;font-size:.812em;line-height:1.5;margin-bottom:25px}#onetrust-pc-sdk #ot-pc-desc *{font-size:inherit;line-height:inherit}#onetrust-pc-sdk #ot-pc-desc ul li{padding:10px 0px}#onetrust-pc-sdk a{color:#656565;cursor:pointer}#onetrust-pc-sdk a:hover{color:#3860be}#onetrust-pc-sdk label{margin-bottom:0}#onetrust-pc-sdk #vdr-lst-dsc{font-size:.812em;line-height:1.5;padding:10px 15px 5px 15px}#onetrust-pc-sdk button{max-width:394px;padding:12px 30px;line-height:1;word-break:break-word;word-wrap:break-word;white-space:normal;font-weight:bold;height:auto}#onetrust-pc-sdk .ot-link-btn{padding:0;margin-bottom:0;border:0;font-weight:normal;line-height:normal;width:auto;height:auto}#onetrust-pc-sdk #ot-pc-content{position:absolute;overflow-y:scroll;padding-left:2px;padding-right:30px;top:60px;bottom:110px;margin:1px 3px 0 30px;width:calc(100% - 63px)}#onetrust-pc-sdk .ot-vs-list .ot-always-active,#onetrust-pc-sdk .ot-cat-grp .ot-always-active{float:right;clear:none;color:#3860be;margin:0;font-size:.813em;line-height:1.3}#onetrust-pc-sdk .ot-pc-scrollbar::-webkit-scrollbar-track{margin-right:20px}#onetrust-pc-sdk .ot-pc-scrollbar::-webkit-scrollbar{width:11px}#onetrust-pc-sdk .ot-pc-scrollbar::-webkit-scrollbar-thumb{border-radius:10px;background:#d8d8d8}#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:#000 1px solid}#onetrust-pc-sdk .ot-pc-scrollbar{scrollbar-arrow-color:#d8d8d8;scrollbar-darkshadow-color:#d8d8d8;scrollbar-face-color:#d8d8d8;scrollbar-shadow-color:#d8d8d8}#onetrust-pc-sdk .save-preference-btn-handler{margin-right:20px}#onetrust-pc-sdk .ot-pc-refuse-all-handler{margin-right:10px}#onetrust-pc-sdk #ot-pc-desc .privacy-notice-link{margin-left:0;margin-right:8px}#onetrust-pc-sdk #ot-pc-desc .ot-imprint-handler{margin-left:0;margin-right:8px}#onetrust-pc-sdk .ot-subgrp-cntr{display:inline-block;clear:both;width:100%;padding-top:15px}#onetrust-pc-sdk .ot-switch+.ot-subgrp-cntr{padding-top:10px}#onetrust-pc-sdk ul.ot-subgrps{margin:0;font-size:initial}#onetrust-pc-sdk ul.ot-subgrps li p,#onetrust-pc-sdk ul.ot-subgrps li h5{font-size:.813em;line-height:1.4;color:dimgray}#onetrust-pc-sdk ul.ot-subgrps .ot-switch{min-height:auto}#onetrust-pc-sdk ul.ot-subgrps .ot-switch-nob{top:0}#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr{display:inline-block;width:100%}#onetrust-pc-sdk ul.ot-subgrps .ot-acc-txt{margin:0}#onetrust-pc-sdk ul.ot-subgrps li{padding:0;border:none}#onetrust-pc-sdk ul.ot-subgrps li h5{position:relative;top:5px;font-weight:bold;margin-bottom:0;float:left}#onetrust-pc-sdk li.ot-subgrp{margin-left:20px;overflow:auto}#onetrust-pc-sdk li.ot-subgrp>h5{width:calc(100% - 100px)}#onetrust-pc-sdk .ot-cat-item p>ul,#onetrust-pc-sdk li.ot-subgrp p>ul{margin:0px;list-style:disc;margin-left:15px;font-size:inherit}#onetrust-pc-sdk .ot-cat-item p>ul li,#onetrust-pc-sdk li.ot-subgrp p>ul li{font-size:inherit;padding-top:10px;padding-left:0px;padding-right:0px;border:none}#onetrust-pc-sdk .ot-cat-item p>ul li:last-child,#onetrust-pc-sdk li.ot-subgrp p>ul li:last-child{padding-bottom:10px}#onetrust-pc-sdk .ot-pc-logo{height:40px;width:120px}#onetrust-pc-sdk .ot-pc-footer{position:absolute;bottom:0px;width:100%;max-height:160px;border-top:1px solid #d8d8d8}#onetrust-pc-sdk.ot-ftr-stacked .ot-pc-refuse-all-handler{margin-bottom:0px}#onetrust-pc-sdk.ot-ftr-stacked #ot-pc-content{bottom:160px}#onetrust-pc-sdk.ot-ftr-stacked .ot-pc-footer button{width:100%;max-width:none}#onetrust-pc-sdk.ot-ftr-stacked .ot-btn-container{margin:0 30px;width:calc(100% - 60px);padding-right:0}#onetrust-pc-sdk .ot-pc-footer-logo{height:30px;width:100%;text-align:right;background:#f4f4f4}#onetrust-pc-sdk .ot-pc-footer-logo a{display:inline-block;margin-top:5px;margin-right:10px}#onetrust-pc-sdk[dir=rtl] .ot-pc-footer-logo{direction:rtl}#onetrust-pc-sdk[dir=rtl] .ot-pc-footer-logo a{margin-right:25px}#onetrust-pc-sdk .ot-tgl{float:right;position:relative;z-index:1}#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob{background-color:#468254;border:1px solid #fff}#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob:before{-webkit-transform:translateX(20px);-ms-transform:translateX(20px);transform:translateX(20px);background-color:#fff;border-color:#fff}#onetrust-pc-sdk .ot-tgl input:focus+.ot-switch{outline:#000 solid 1px}#onetrust-pc-sdk .ot-switch{position:relative;display:inline-block;width:45px;height:25px}#onetrust-pc-sdk .ot-switch-nob{position:absolute;cursor:pointer;top:0;left:0;right:0;bottom:0;background-color:#767676;border:1px solid #ddd;transition:all .2s ease-in 0s;-moz-transition:all .2s ease-in 0s;-o-transition:all .2s ease-in 0s;-webkit-transition:all .2s ease-in 0s;border-radius:20px}#onetrust-pc-sdk .ot-switch-nob:before{position:absolute;content:"";height:18px;width:18px;bottom:3px;left:3px;background-color:#fff;-webkit-transition:.4s;transition:.4s;border-radius:20px}#onetrust-pc-sdk .ot-chkbox input:checked~label::before{background-color:#3860be}#onetrust-pc-sdk .ot-chkbox input+label::after{content:none;color:#fff}#onetrust-pc-sdk .ot-chkbox input:checked+label::after{content:""}#onetrust-pc-sdk .ot-chkbox input:focus+label::before{outline-style:solid;outline-width:2px;outline-style:auto}#onetrust-pc-sdk .ot-chkbox input[aria-checked=mixed]~label::before{background-color:#3860be}#onetrust-pc-sdk .ot-chkbox input[aria-checked=mixed]+label::after{content:""}#onetrust-pc-sdk .ot-chkbox label{position:relative;display:inline-block;padding-left:30px;cursor:pointer;font-weight:500}#onetrust-pc-sdk .ot-chkbox label::before,#onetrust-pc-sdk .ot-chkbox label::after{position:absolute;content:"";display:inline-block;border-radius:3px}#onetrust-pc-sdk .ot-chkbox label::before{height:18px;width:18px;border:1px solid #3860be;left:0px;top:auto}#onetrust-pc-sdk .ot-chkbox label::after{height:5px;width:9px;border-left:3px solid;border-bottom:3px solid;transform:rotate(-45deg);-o-transform:rotate(-45deg);-ms-transform:rotate(-45deg);-webkit-transform:rotate(-45deg);left:4px;top:5px}#onetrust-pc-sdk .ot-label-txt{display:none}#onetrust-pc-sdk .ot-chkbox input,#onetrust-pc-sdk .ot-tgl input{position:absolute;opacity:0;width:0;height:0}#onetrust-pc-sdk .ot-arw-cntr{float:right;position:relative;pointer-events:none}#onetrust-pc-sdk .ot-arw-cntr .ot-arw{width:16px;height:16px;margin-left:5px;color:dimgray;display:inline-block;vertical-align:middle;-webkit-transition:all 150ms ease-in 0s;-moz-transition:all 150ms ease-in 0s;-o-transition:all 150ms ease-in 0s;transition:all 150ms ease-in 0s}#onetrust-pc-sdk input:checked~.ot-acc-hdr .ot-arw,#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-arw-cntr svg{transform:rotate(90deg);-o-transform:rotate(90deg);-ms-transform:rotate(90deg);-webkit-transform:rotate(90deg)}#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:#000 1px solid}#onetrust-pc-sdk .ot-tgl-cntr,#onetrust-pc-sdk .ot-arw-cntr{display:inline-block}#onetrust-pc-sdk .ot-tgl-cntr{width:45px;float:right;margin-top:2px}#onetrust-pc-sdk #ot-lst-cnt .ot-tgl-cntr{margin-top:10px}#onetrust-pc-sdk .ot-always-active-subgroup{width:auto;padding-left:0px !important;top:3px;position:relative}#onetrust-pc-sdk .ot-label-status{font-size:.75em;display:none;font-size:.75em;position:relative;top:2px;padding-right:5px;float:left}#onetrust-pc-sdk .ot-arw-cntr{margin-top:-1px}#onetrust-pc-sdk .ot-arw-cntr svg{-webkit-transition:all 300ms ease-in 0s;-moz-transition:all 300ms ease-in 0s;-o-transition:all 300ms ease-in 0s;transition:all 300ms ease-in 0s;height:10px;width:10px}#onetrust-pc-sdk input:checked~.ot-acc-hdr .ot-arw{transform:rotate(90deg);-o-transform:rotate(90deg);-ms-transform:rotate(90deg);-webkit-transform:rotate(90deg)}#onetrust-pc-sdk .ot-arw{width:10px;margin-left:15px;transition:all 300ms ease-in 0s;-webkit-transition:all 300ms ease-in 0s;-moz-transition:all 300ms ease-in 0s;-o-transition:all 300ms ease-in 0s}#onetrust-pc-sdk .ot-vlst-cntr{margin-bottom:0}#onetrust-pc-sdk .ot-hlst-cntr{margin-top:5px;display:inline-block;width:100%}#onetrust-pc-sdk .category-vendors-list-handler,#onetrust-pc-sdk .category-vendors-list-handler+a,#onetrust-pc-sdk .category-host-list-handler{clear:both;color:#3860be;margin-left:0;font-size:.813em;text-decoration:none;float:left;overflow:hidden}#onetrust-pc-sdk .category-vendors-list-handler:hover,#onetrust-pc-sdk .category-vendors-list-handler+a:hover,#onetrust-pc-sdk .category-host-list-handler:hover{text-decoration-line:underline}#onetrust-pc-sdk .category-vendors-list-handler+a{clear:none}#onetrust-pc-sdk .ot-vlst-cntr .ot-ext-lnk,#onetrust-pc-sdk .ot-ven-hdr .ot-ext-lnk{display:inline-block;height:13px;width:13px;background-repeat:no-repeat;margin-left:1px;margin-top:6px;cursor:pointer}#onetrust-pc-sdk .ot-ven-hdr .ot-ext-lnk{margin-bottom:-1px}#onetrust-pc-sdk .back-btn-handler{font-size:1em;text-decoration:none}#onetrust-pc-sdk .back-btn-handler:hover{opacity:.6}#onetrust-pc-sdk #ot-lst-title h3{display:inline-block;word-break:break-word;word-wrap:break-word;margin-bottom:0;color:#656565;font-size:1em;font-weight:bold;margin-left:15px}#onetrust-pc-sdk #ot-lst-title{margin:10px 0 10px 0px;font-size:1em;text-align:left}#onetrust-pc-sdk #ot-pc-hdr{margin:0 0 0 30px;height:auto;width:auto}#onetrust-pc-sdk #ot-pc-hdr input::placeholder{color:#707070;font-style:italic}#onetrust-pc-sdk #vendor-search-handler{height:31px;width:100%;border-radius:50px;font-size:.8em;padding-right:35px;padding-left:15px;float:left;margin-left:15px}#onetrust-pc-sdk .ot-ven-name{display:block;width:auto;padding-right:5px}#onetrust-pc-sdk #ot-lst-cnt{overflow-y:auto;margin-left:20px;margin-right:7px;width:calc(100% - 27px);max-height:calc(100% - 80px);height:100%;transform:translate3d(0, 0, 0)}#onetrust-pc-sdk #ot-pc-lst{width:100%;bottom:100px;position:absolute;top:60px}#onetrust-pc-sdk #ot-pc-lst:not(.ot-enbl-chr) .ot-tgl-cntr .ot-arw-cntr,#onetrust-pc-sdk #ot-pc-lst:not(.ot-enbl-chr) .ot-tgl-cntr .ot-arw-cntr *{visibility:hidden}#onetrust-pc-sdk #ot-pc-lst .ot-tgl-cntr{right:12px;position:absolute}#onetrust-pc-sdk #ot-pc-lst .ot-arw-cntr{float:right;position:relative}#onetrust-pc-sdk #ot-pc-lst .ot-arw{margin-left:10px}#onetrust-pc-sdk #ot-pc-lst .ot-acc-hdr{overflow:hidden;cursor:pointer}#onetrust-pc-sdk .ot-vlst-cntr{overflow:hidden}#onetrust-pc-sdk #ot-sel-blk{overflow:hidden;width:100%;position:sticky;position:-webkit-sticky;top:0;z-index:3}#onetrust-pc-sdk #ot-back-arw{height:12px;width:12px}#onetrust-pc-sdk .ot-lst-subhdr{width:100%;display:inline-block}#onetrust-pc-sdk .ot-search-cntr{float:left;width:78%;position:relative}#onetrust-pc-sdk .ot-search-cntr>svg{width:30px;height:30px;position:absolute;float:left;right:-15px}#onetrust-pc-sdk .ot-fltr-cntr{float:right;right:50px;position:relative}#onetrust-pc-sdk #filter-btn-handler{background-color:#3860be;border-radius:17px;display:inline-block;position:relative;width:32px;height:32px;-moz-transition:.1s ease;-o-transition:.1s ease;-webkit-transition:1s ease;transition:.1s ease;padding:0;margin:0}#onetrust-pc-sdk #filter-btn-handler:hover{background-color:#3860be}#onetrust-pc-sdk #filter-btn-handler svg{width:12px;height:12px;margin:3px 10px 0 10px;display:block;position:static;right:auto;top:auto}#onetrust-pc-sdk .ot-ven-link,#onetrust-pc-sdk .ot-ven-legclaim-link{color:#3860be;text-decoration:none;font-weight:100;display:inline-block;padding-top:10px;transform:translate(0, 1%);-o-transform:translate(0, 1%);-ms-transform:translate(0, 1%);-webkit-transform:translate(0, 1%);position:relative;z-index:2}#onetrust-pc-sdk .ot-ven-link *,#onetrust-pc-sdk .ot-ven-legclaim-link *{font-size:inherit}#onetrust-pc-sdk .ot-ven-link:hover,#onetrust-pc-sdk .ot-ven-legclaim-link:hover{text-decoration:underline}#onetrust-pc-sdk .ot-ven-hdr{width:calc(100% - 160px);height:auto;float:left;word-break:break-word;word-wrap:break-word;vertical-align:middle;padding-bottom:3px}#onetrust-pc-sdk .ot-ven-link,#onetrust-pc-sdk .ot-ven-legclaim-link{letter-spacing:.03em;font-size:.75em;font-weight:400}#onetrust-pc-sdk .ot-ven-dets{border-radius:2px;background-color:#f8f8f8}#onetrust-pc-sdk .ot-ven-dets li:first-child p:first-child{border-top:none}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:not(:first-child){border-top:1px solid #ddd !important}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:nth-child(n+3) p{display:inline-block}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:nth-child(n+3) p:nth-of-type(odd){width:30%}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:nth-child(n+3) p:nth-of-type(even){width:50%;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc p,#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc h5{padding-top:5px;padding-bottom:5px;display:block}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc h5{display:inline-block}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc p:nth-last-child(-n+1){padding-bottom:10px}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc p:nth-child(-n+2):not(.disc-pur){padding-top:10px}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc .disc-pur-cont{display:inline}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc .disc-pur{position:relative;width:50% !important;word-break:break-word;word-wrap:break-word;left:calc(30% + 17px)}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc .disc-pur:nth-child(-n+1){position:static}#onetrust-pc-sdk .ot-ven-dets p,#onetrust-pc-sdk .ot-ven-dets h5,#onetrust-pc-sdk .ot-ven-dets span{font-size:.69em;text-align:left;vertical-align:middle;word-break:break-word;word-wrap:break-word;margin:0;padding-bottom:10px;padding-left:15px;color:#2e3644}#onetrust-pc-sdk .ot-ven-dets h5{padding-top:5px}#onetrust-pc-sdk .ot-ven-dets span{color:dimgray;padding:0;vertical-align:baseline}#onetrust-pc-sdk .ot-ven-dets .ot-ven-pur h5{border-top:1px solid #e9e9e9;border-bottom:1px solid #e9e9e9;padding-bottom:5px;margin-bottom:5px;font-weight:bold}#onetrust-pc-sdk #ot-host-lst .ot-sel-all{float:right;position:relative;margin-right:42px;top:10px}#onetrust-pc-sdk #ot-host-lst .ot-sel-all input[type=checkbox]{width:auto;height:auto}#onetrust-pc-sdk #ot-host-lst .ot-sel-all label{height:20px;width:20px;padding-left:0px}#onetrust-pc-sdk #ot-host-lst .ot-acc-txt{overflow:hidden;width:95%}#onetrust-pc-sdk .ot-host-hdr{position:relative;z-index:1;pointer-events:none;width:calc(100% - 125px);float:left}#onetrust-pc-sdk .ot-host-name,#onetrust-pc-sdk .ot-host-desc{display:inline-block;width:90%}#onetrust-pc-sdk .ot-host-name{pointer-events:none}#onetrust-pc-sdk .ot-host-hdr>a{text-decoration:underline;font-size:.82em;position:relative;z-index:2;float:left;margin-bottom:5px;pointer-events:initial}#onetrust-pc-sdk .ot-host-name+a{margin-top:5px}#onetrust-pc-sdk .ot-host-name,#onetrust-pc-sdk .ot-host-name a,#onetrust-pc-sdk .ot-host-desc,#onetrust-pc-sdk .ot-host-info{color:dimgray;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-host-name,#onetrust-pc-sdk .ot-host-name a{font-weight:bold;font-size:.82em;line-height:1.3}#onetrust-pc-sdk .ot-host-name a{font-size:1em}#onetrust-pc-sdk .ot-host-expand{margin-top:3px;margin-bottom:3px;clear:both;display:block;color:#3860be;font-size:.72em;font-weight:normal}#onetrust-pc-sdk .ot-host-expand *{font-size:inherit}#onetrust-pc-sdk .ot-host-desc,#onetrust-pc-sdk .ot-host-info{font-size:.688em;line-height:1.4;font-weight:normal}#onetrust-pc-sdk .ot-host-desc{margin-top:10px}#onetrust-pc-sdk .ot-host-opt{margin:0;font-size:inherit;display:inline-block;width:100%}#onetrust-pc-sdk .ot-host-opt li>div div{font-size:.8em;padding:5px 0}#onetrust-pc-sdk .ot-host-opt li>div div:nth-child(1){width:30%;float:left}#onetrust-pc-sdk .ot-host-opt li>div div:nth-child(2){width:70%;float:left;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-host-info{border:none;display:inline-block;width:calc(100% - 10px);padding:10px;margin-bottom:10px;background-color:#f8f8f8}#onetrust-pc-sdk .ot-host-info>div{overflow:auto}#onetrust-pc-sdk #no-results{text-align:center;margin-top:30px}#onetrust-pc-sdk #no-results p{font-size:1em;color:#2e3644;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk #no-results p span{font-weight:bold}#onetrust-pc-sdk #ot-fltr-modal{width:100%;height:auto;display:none;-moz-transition:.2s ease;-o-transition:.2s ease;-webkit-transition:2s ease;transition:.2s ease;overflow:hidden;opacity:1;right:0}#onetrust-pc-sdk #ot-fltr-modal .ot-label-txt{display:inline-block;font-size:.85em;color:dimgray}#onetrust-pc-sdk #ot-fltr-cnt{z-index:2147483646;background-color:#fff;position:absolute;height:90%;max-height:300px;width:325px;left:210px;margin-top:10px;margin-bottom:20px;padding-right:10px;border-radius:3px;-webkit-box-shadow:0px 0px 12px 2px #c7c5c7;-moz-box-shadow:0px 0px 12px 2px #c7c5c7;box-shadow:0px 0px 12px 2px #c7c5c7}#onetrust-pc-sdk .ot-fltr-scrlcnt{overflow-y:auto;overflow-x:hidden;clear:both;max-height:calc(100% - 60px)}#onetrust-pc-sdk #ot-anchor{border:12px solid rgba(0,0,0,0);display:none;position:absolute;z-index:2147483647;right:55px;top:75px;transform:rotate(45deg);-o-transform:rotate(45deg);-ms-transform:rotate(45deg);-webkit-transform:rotate(45deg);background-color:#fff;-webkit-box-shadow:-3px -3px 5px -2px #c7c5c7;-moz-box-shadow:-3px -3px 5px -2px #c7c5c7;box-shadow:-3px -3px 5px -2px #c7c5c7}#onetrust-pc-sdk .ot-fltr-btns{margin-left:15px}#onetrust-pc-sdk #filter-apply-handler{margin-right:15px}#onetrust-pc-sdk .ot-fltr-opt{margin-bottom:5px;margin-left:15px;min-height:20px;width:75%;position:relative}#onetrust-pc-sdk .ot-fltr-opt p{display:inline-block;margin:0;font-size:.9em;color:#2e3644}#onetrust-pc-sdk .ot-chkbox label span{font-size:.85em;color:dimgray}#onetrust-pc-sdk .ot-chkbox input[type=checkbox]+label::after{content:none;color:#fff}#onetrust-pc-sdk .ot-chkbox input[type=checkbox]:checked+label::after{content:""}#onetrust-pc-sdk .ot-chkbox input[type=checkbox][aria-checked=mixed]+label::after{content:""}#onetrust-pc-sdk .ot-chkbox input[type=checkbox]:focus+label::before{outline-style:solid;outline-width:2px;outline-style:auto}#onetrust-pc-sdk #ot-selall-vencntr,#onetrust-pc-sdk #ot-selall-adtlvencntr,#onetrust-pc-sdk #ot-selall-hostcntr,#onetrust-pc-sdk #ot-selall-licntr,#onetrust-pc-sdk #ot-selall-gnvencntr{right:15px;position:relative;width:20px;height:20px;float:right}#onetrust-pc-sdk #ot-selall-vencntr label,#onetrust-pc-sdk #ot-selall-adtlvencntr label,#onetrust-pc-sdk #ot-selall-hostcntr label,#onetrust-pc-sdk #ot-selall-licntr label,#onetrust-pc-sdk #ot-selall-gnvencntr label{float:left;padding-left:0}#onetrust-pc-sdk #ot-ven-lst:first-child{border-top:1px solid #e2e2e2}#onetrust-pc-sdk ul{list-style:none;padding:0}#onetrust-pc-sdk ul li{position:relative;margin:0;padding:15px 15px 15px 10px;border-bottom:1px solid #e2e2e2}#onetrust-pc-sdk ul li h3,#onetrust-pc-sdk ul li h4{font-size:.75em;color:#656565;margin:0;display:inline-block;width:70%;height:auto;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk ul li p{margin:0;font-size:.7em}#onetrust-pc-sdk ul li input[type=checkbox]{position:absolute;cursor:pointer;width:100%;height:100%;opacity:0;margin:0;top:0;left:0}#onetrust-pc-sdk .ot-cat-item>button:focus,#onetrust-pc-sdk .ot-acc-cntr>button:focus,#onetrust-pc-sdk li>button:focus{outline:#000 solid 2px}#onetrust-pc-sdk .ot-cat-item>button,#onetrust-pc-sdk .ot-acc-cntr>button,#onetrust-pc-sdk li>button{position:absolute;cursor:pointer;width:100%;height:100%;margin:0;top:0;left:0;z-index:1;max-width:none;border:none}#onetrust-pc-sdk .ot-cat-item>button[aria-expanded=false]~.ot-acc-txt,#onetrust-pc-sdk .ot-acc-cntr>button[aria-expanded=false]~.ot-acc-txt,#onetrust-pc-sdk li>button[aria-expanded=false]~.ot-acc-txt{margin-top:0;max-height:0;opacity:0;overflow:hidden;width:100%;transition:.25s ease-out;display:none}#onetrust-pc-sdk .ot-cat-item>button[aria-expanded=true]~.ot-acc-txt,#onetrust-pc-sdk .ot-acc-cntr>button[aria-expanded=true]~.ot-acc-txt,#onetrust-pc-sdk li>button[aria-expanded=true]~.ot-acc-txt{transition:.1s ease-in;margin-top:10px;width:100%;overflow:auto;display:block}#onetrust-pc-sdk .ot-cat-item>button[aria-expanded=true]~.ot-acc-grpcntr,#onetrust-pc-sdk .ot-acc-cntr>button[aria-expanded=true]~.ot-acc-grpcntr,#onetrust-pc-sdk li>button[aria-expanded=true]~.ot-acc-grpcntr{width:auto;margin-top:0px;padding-bottom:10px}#onetrust-pc-sdk .ot-host-item>button:focus,#onetrust-pc-sdk .ot-ven-item>button:focus{outline:0;border:2px solid #000}#onetrust-pc-sdk .ot-hide-acc>button{pointer-events:none}#onetrust-pc-sdk .ot-hide-acc .ot-plus-minus>*,#onetrust-pc-sdk .ot-hide-acc .ot-arw-cntr>*{visibility:hidden}#onetrust-pc-sdk .ot-hide-acc .ot-acc-hdr{min-height:30px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt){padding-right:10px;width:calc(100% - 37px);margin-top:10px;max-height:calc(100% - 90px)}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk{background-color:#f9f9fc;border:1px solid #e2e2e2;width:calc(100% - 2px);padding-bottom:5px;padding-top:5px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt{border:unset;background-color:unset}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt .ot-sel-all-hdr{display:none}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt .ot-sel-all{padding-right:.5rem}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt .ot-sel-all .ot-chkbox{right:0}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) .ot-sel-all{padding-right:34px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) .ot-sel-all-chkbox{width:auto}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) ul li{border:1px solid #e2e2e2;margin-bottom:10px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) .ot-acc-cntr>.ot-acc-hdr{padding:10px 0 10px 15px}#onetrust-pc-sdk.ot-addtl-vendors .ot-sel-all-chkbox{float:right}#onetrust-pc-sdk.ot-addtl-vendors .ot-plus-minus~.ot-sel-all-chkbox{right:34px}#onetrust-pc-sdk.ot-addtl-vendors #ot-ven-lst:first-child{border-top:none}#onetrust-pc-sdk .ot-acc-cntr{position:relative;border-left:1px solid #e2e2e2;border-right:1px solid #e2e2e2;border-bottom:1px solid #e2e2e2}#onetrust-pc-sdk .ot-acc-cntr input{z-index:1}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr{background-color:#f9f9fc;padding:5px 0 5px 15px;width:auto}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr .ot-plus-minus{vertical-align:middle;top:auto}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr .ot-arw-cntr{right:10px}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr input{z-index:2}#onetrust-pc-sdk .ot-acc-cntr.ot-add-tech .ot-acc-hdr{padding:10px 0 10px 15px}#onetrust-pc-sdk .ot-acc-cntr>input[type=checkbox]:checked~.ot-acc-hdr{border-bottom:1px solid #e2e2e2}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-txt{padding-left:10px;padding-right:10px}#onetrust-pc-sdk .ot-acc-cntr button[aria-expanded=true]~.ot-acc-txt{width:auto}#onetrust-pc-sdk .ot-acc-cntr .ot-addtl-venbox{display:none}#onetrust-pc-sdk .ot-vlst-cntr{margin-bottom:0;width:100%}#onetrust-pc-sdk .ot-vensec-title{font-size:.813em;vertical-align:middle;display:inline-block}#onetrust-pc-sdk .category-vendors-list-handler,#onetrust-pc-sdk .category-vendors-list-handler+a{margin-left:0;margin-top:10px}#onetrust-pc-sdk #ot-selall-vencntr.line-through label::after,#onetrust-pc-sdk #ot-selall-adtlvencntr.line-through label::after,#onetrust-pc-sdk #ot-selall-licntr.line-through label::after,#onetrust-pc-sdk #ot-selall-hostcntr.line-through label::after,#onetrust-pc-sdk #ot-selall-gnvencntr.line-through label::after{height:auto;border-left:0;transform:none;-o-transform:none;-ms-transform:none;-webkit-transform:none;left:5px;top:9px}#onetrust-pc-sdk #ot-category-title{float:left;padding-bottom:10px;font-size:1em;width:100%}#onetrust-pc-sdk .ot-cat-grp{margin-top:10px}#onetrust-pc-sdk .ot-cat-item{line-height:1.1;margin-top:10px;display:inline-block;width:100%}#onetrust-pc-sdk .ot-btn-container{text-align:right}#onetrust-pc-sdk .ot-btn-container button{display:inline-block;font-size:.75em;letter-spacing:.08em;margin-top:19px}#onetrust-pc-sdk #close-pc-btn-handler.ot-close-icon{position:absolute;top:10px;right:0;z-index:1;padding:0;background-color:rgba(0,0,0,0);border:none}#onetrust-pc-sdk #close-pc-btn-handler.ot-close-icon svg{display:block;height:10px;width:10px}#onetrust-pc-sdk #clear-filters-handler{margin-top:20px;margin-bottom:10px;float:right;max-width:200px;text-decoration:none;color:#3860be;font-size:.9em;font-weight:bold;background-color:rgba(0,0,0,0);border-color:rgba(0,0,0,0);padding:1px}#onetrust-pc-sdk #clear-filters-handler:hover{color:#2285f7}#onetrust-pc-sdk #clear-filters-handler:focus{outline:#000 solid 1px}#onetrust-pc-sdk .ot-enbl-chr h4~.ot-tgl,#onetrust-pc-sdk .ot-enbl-chr h4~.ot-always-active{right:45px}#onetrust-pc-sdk .ot-enbl-chr h4~.ot-tgl+.ot-tgl{right:120px}#onetrust-pc-sdk .ot-enbl-chr .ot-pli-hdr.ot-leg-border-color span:first-child{width:90px}#onetrust-pc-sdk .ot-enbl-chr li.ot-subgrp>h5+.ot-tgl-cntr{padding-right:25px}#onetrust-pc-sdk .ot-plus-minus{width:20px;height:20px;font-size:1.5em;position:relative;display:inline-block;margin-right:5px;top:3px}#onetrust-pc-sdk .ot-plus-minus span{position:absolute;background:#27455c;border-radius:1px}#onetrust-pc-sdk .ot-plus-minus span:first-of-type{top:25%;bottom:25%;width:10%;left:45%}#onetrust-pc-sdk .ot-plus-minus span:last-of-type{left:25%;right:25%;height:10%;top:45%}#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-arw,#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-plus-minus span:first-of-type,#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-plus-minus span:last-of-type{transform:rotate(90deg)}#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-plus-minus span:last-of-type{left:50%;right:50%}#onetrust-pc-sdk #ot-selall-vencntr label,#onetrust-pc-sdk #ot-selall-adtlvencntr label,#onetrust-pc-sdk #ot-selall-hostcntr label,#onetrust-pc-sdk #ot-selall-licntr label{position:relative;display:inline-block;width:20px;height:20px}#onetrust-pc-sdk .ot-host-item .ot-plus-minus,#onetrust-pc-sdk .ot-ven-item .ot-plus-minus{float:left;margin-right:8px;top:10px}#onetrust-pc-sdk .ot-ven-item ul{list-style:none inside;font-size:100%;margin:0}#onetrust-pc-sdk .ot-ven-item ul li{margin:0 !important;padding:0;border:none !important}#onetrust-pc-sdk .ot-pli-hdr{color:#77808e;overflow:hidden;padding-top:7.5px;padding-bottom:7.5px;width:calc(100% - 2px);border-top-left-radius:3px;border-top-right-radius:3px}#onetrust-pc-sdk .ot-pli-hdr span:first-child{top:50%;transform:translateY(50%);max-width:90px}#onetrust-pc-sdk .ot-pli-hdr span:last-child{padding-right:10px;max-width:95px;text-align:center}#onetrust-pc-sdk .ot-li-title{float:right;font-size:.813em}#onetrust-pc-sdk .ot-pli-hdr.ot-leg-border-color{background-color:#f4f4f4;border:1px solid #d8d8d8}#onetrust-pc-sdk .ot-pli-hdr.ot-leg-border-color span:first-child{text-align:left;width:70px}#onetrust-pc-sdk li.ot-subgrp>h5,#onetrust-pc-sdk .ot-cat-header{width:calc(100% - 130px)}#onetrust-pc-sdk li.ot-subgrp>h5+.ot-tgl-cntr{padding-left:13px}#onetrust-pc-sdk .ot-acc-grpcntr .ot-acc-grpdesc{margin-bottom:5px}#onetrust-pc-sdk .ot-acc-grpcntr .ot-subgrp-cntr{border-top:1px solid #d8d8d8}#onetrust-pc-sdk .ot-acc-grpcntr .ot-vlst-cntr+.ot-subgrp-cntr{border-top:none}#onetrust-pc-sdk .ot-acc-hdr .ot-arw-cntr+.ot-tgl-cntr,#onetrust-pc-sdk .ot-acc-txt h4+.ot-tgl-cntr{padding-left:13px}#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item .ot-subgrp>h5,#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item .ot-cat-header{width:calc(100% - 145px)}#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item h5+.ot-tgl-cntr,#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item .ot-cat-header+.ot-tgl{padding-left:28px}#onetrust-pc-sdk .ot-sel-all-hdr,#onetrust-pc-sdk .ot-sel-all-chkbox{display:inline-block;width:100%;position:relative}#onetrust-pc-sdk .ot-sel-all-chkbox{z-index:1}#onetrust-pc-sdk .ot-sel-all{margin:0;position:relative;padding-right:23px;float:right}#onetrust-pc-sdk .ot-consent-hdr,#onetrust-pc-sdk .ot-li-hdr{float:right;font-size:.812em;line-height:normal;text-align:center;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-li-hdr{max-width:100px;padding-right:10px}#onetrust-pc-sdk .ot-consent-hdr{max-width:55px}#onetrust-pc-sdk #ot-selall-licntr{display:block;width:21px;height:auto;float:right;position:relative;right:80px}#onetrust-pc-sdk #ot-selall-licntr label{position:absolute}#onetrust-pc-sdk .ot-ven-ctgl{margin-left:66px}#onetrust-pc-sdk .ot-ven-litgl+.ot-arw-cntr{margin-left:81px}#onetrust-pc-sdk .ot-enbl-chr .ot-host-cnt .ot-tgl-cntr{width:auto}#onetrust-pc-sdk #ot-lst-cnt:not(.ot-host-cnt) .ot-tgl-cntr{width:auto;top:auto;height:20px}#onetrust-pc-sdk #ot-lst-cnt .ot-chkbox{position:relative;display:inline-block;width:20px;height:20px}#onetrust-pc-sdk #ot-lst-cnt .ot-chkbox label{position:absolute;padding:0;width:20px;height:20px}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info-cntr{border:1px solid #d8d8d8;padding:.75rem 2rem;padding-bottom:0;width:auto;margin-top:.5rem}#onetrust-pc-sdk .ot-acc-grpdesc+.ot-leg-btn-container{padding-left:20px;padding-right:20px;width:calc(100% - 40px);margin-bottom:5px}#onetrust-pc-sdk .ot-subgrp .ot-leg-btn-container{margin-bottom:5px}#onetrust-pc-sdk #ot-ven-lst .ot-leg-btn-container{margin-top:10px}#onetrust-pc-sdk .ot-leg-btn-container{display:inline-block;width:100%;margin-bottom:10px}#onetrust-pc-sdk .ot-leg-btn-container button{height:auto;padding:6.5px 8px;margin-bottom:0;letter-spacing:0;font-size:.75em;line-height:normal}#onetrust-pc-sdk .ot-leg-btn-container svg{display:none;height:14px;width:14px;padding-right:5px;vertical-align:sub}#onetrust-pc-sdk .ot-active-leg-btn{cursor:default;pointer-events:none}#onetrust-pc-sdk .ot-active-leg-btn svg{display:inline-block}#onetrust-pc-sdk .ot-remove-objection-handler{text-decoration:underline;padding:0;font-size:.75em;font-weight:600;line-height:1;padding-left:10px}#onetrust-pc-sdk .ot-obj-leg-btn-handler span{font-weight:bold;text-align:center;font-size:inherit;line-height:1.5}#onetrust-pc-sdk.ot-close-btn-link #close-pc-btn-handler{border:none;height:auto;line-height:1.5;text-decoration:underline;font-size:.69em;background:none;right:15px;top:15px;width:auto;font-weight:normal}#onetrust-pc-sdk .ot-pgph-link{font-size:.813em !important;margin-top:5px;position:relative}#onetrust-pc-sdk .ot-pgph-link.ot-pgph-link-subgroup{margin-bottom:1rem}#onetrust-pc-sdk .ot-pgph-contr{margin:0 2.5rem}#onetrust-pc-sdk .ot-pgph-title{font-size:1.18rem;margin-bottom:2rem}#onetrust-pc-sdk .ot-pgph-desc{font-size:1rem;font-weight:400;margin-bottom:2rem;line-height:1.5rem}#onetrust-pc-sdk .ot-pgph-desc:not(:last-child):after{content:"";width:96%;display:block;margin:0 auto;padding-bottom:2rem;border-bottom:1px solid #e9e9e9}#onetrust-pc-sdk .ot-cat-header{float:left;font-weight:600;font-size:.875em;line-height:1.5;max-width:90%;vertical-align:middle}#onetrust-pc-sdk .ot-vnd-item>button:focus{outline:#000 solid 2px}#onetrust-pc-sdk .ot-vnd-item>button{position:absolute;cursor:pointer;width:100%;height:100%;margin:0;top:0;left:0;z-index:1;max-width:none;border:none}#onetrust-pc-sdk .ot-vnd-item>button[aria-expanded=false]~.ot-acc-txt{margin-top:0;max-height:0;opacity:0;overflow:hidden;width:100%;transition:.25s ease-out;display:none}#onetrust-pc-sdk .ot-vnd-item>button[aria-expanded=true]~.ot-acc-txt{transition:.1s ease-in;margin-top:10px;width:100%;overflow:auto;display:block}#onetrust-pc-sdk .ot-vnd-item>button[aria-expanded=true]~.ot-acc-grpcntr{width:auto;margin-top:0px;padding-bottom:10px}#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item{position:relative;border-radius:2px;margin:0;padding:0;border:1px solid #d8d8d8;border-top:none;width:calc(100% - 2px);float:left}#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item:first-of-type{margin-top:10px;border-top:1px solid #d8d8d8}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpdesc{padding-left:20px;padding-right:20px;width:calc(100% - 40px);font-size:.812em;margin-bottom:10px;margin-top:15px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpdesc>ul{padding-top:10px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpdesc>ul li{padding-top:0;line-height:1.5;padding-bottom:10px}#onetrust-pc-sdk .ot-accordion-layout div+.ot-acc-grpdesc{margin-top:5px}#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr:first-child{margin-top:10px}#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr:last-child,#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr:last-child{margin-bottom:5px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{padding-top:11.5px;padding-bottom:11.5px;padding-left:20px;padding-right:20px;width:calc(100% - 40px);display:inline-block}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-txt{width:100%;padding:0}#onetrust-pc-sdk .ot-accordion-layout .ot-subgrp-cntr{padding-left:20px;padding-right:15px;padding-bottom:0;width:calc(100% - 35px)}#onetrust-pc-sdk .ot-accordion-layout .ot-subgrp{padding-right:5px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpcntr{z-index:1;position:relative}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header+.ot-arw-cntr{position:absolute;top:50%;transform:translateY(-50%);right:20px;margin-top:-2px}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header+.ot-arw-cntr .ot-arw{width:15px;height:20px;margin-left:5px;color:dimgray}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{float:none;color:#2e3644;margin:0;display:inline-block;height:auto;word-wrap:break-word;min-height:inherit}#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr,#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr{padding-left:20px;width:calc(100% - 20px);display:inline-block;margin-top:0;padding-bottom:2px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{position:relative;min-height:25px}#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl,#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active{position:absolute;top:50%;transform:translateY(-50%);right:20px}#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl+.ot-tgl{right:95px}#onetrust-pc-sdk .ot-accordion-layout .category-vendors-list-handler,#onetrust-pc-sdk .ot-accordion-layout .category-vendors-list-handler+a{margin-top:5px}#onetrust-pc-sdk #ot-lst-cnt{margin-top:1rem;max-height:calc(100% - 96px)}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info-cntr{border:1px solid #d8d8d8;padding:.75rem 2rem;padding-bottom:0;width:auto;margin-top:.5rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info{margin-bottom:1rem;padding-left:.75rem;padding-right:.75rem;display:flex;flex-direction:column}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info[data-vnd-info-key*=DPOEmail]{border-top:1px solid #d8d8d8;padding-top:1rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info[data-vnd-info-key*=DPOLink]{border-bottom:1px solid #d8d8d8;padding-bottom:1rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info .ot-vnd-lbl{font-weight:bold;font-size:.85em;margin-bottom:.5rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info .ot-vnd-cnt{margin-left:.5rem;font-weight:500;font-size:.85rem}#onetrust-pc-sdk .ot-vs-list,#onetrust-pc-sdk .ot-vnd-serv{width:auto;padding:1rem 1.25rem;padding-bottom:0}#onetrust-pc-sdk .ot-vs-list .ot-vnd-serv-hdr-cntr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-serv-hdr-cntr{padding-bottom:.75rem;border-bottom:1px solid #d8d8d8}#onetrust-pc-sdk .ot-vs-list .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr{font-weight:600;font-size:.95em;line-height:2;margin-left:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item{border:none;margin:0;padding:0}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item button,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item button{outline:none;border-bottom:1px solid #d8d8d8}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item button[aria-expanded=true],#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item button[aria-expanded=true]{border-bottom:none}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item:first-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item:first-child{margin-top:.25rem;border-top:unset}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item:last-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item:last-child{margin-bottom:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item:last-child button,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item:last-child button{border-bottom:none}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info-cntr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info-cntr{border:1px solid #d8d8d8;padding:.75rem 1.75rem;padding-bottom:0;width:auto;margin-top:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info{margin-bottom:1rem;padding-left:.75rem;padding-right:.75rem;display:flex;flex-direction:column}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOEmail],#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOEmail]{border-top:1px solid #d8d8d8;padding-top:1rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOLink],#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOLink]{border-bottom:1px solid #d8d8d8;padding-bottom:1rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info .ot-vnd-lbl,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info .ot-vnd-lbl{font-weight:bold;font-size:.85em;margin-bottom:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info .ot-vnd-cnt,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info .ot-vnd-cnt{margin-left:.5rem;font-weight:500;font-size:.85rem}#onetrust-pc-sdk .ot-vs-list.ot-vnd-subgrp-cnt,#onetrust-pc-sdk .ot-vnd-serv.ot-vnd-subgrp-cnt{padding-left:40px}#onetrust-pc-sdk .ot-vs-list.ot-vnd-subgrp-cnt .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr,#onetrust-pc-sdk .ot-vnd-serv.ot-vnd-subgrp-cnt .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr{font-size:.8em}#onetrust-pc-sdk .ot-vs-list.ot-vnd-subgrp-cnt .ot-cat-header,#onetrust-pc-sdk .ot-vnd-serv.ot-vnd-subgrp-cnt .ot-cat-header{font-size:.8em}#onetrust-pc-sdk .ot-subgrp-cntr .ot-vnd-serv{margin-bottom:1rem;padding:1rem .95rem}#onetrust-pc-sdk .ot-subgrp-cntr .ot-vnd-serv .ot-vnd-serv-hdr-cntr{padding-bottom:.75rem;border-bottom:1px solid #d8d8d8}#onetrust-pc-sdk .ot-subgrp-cntr .ot-vnd-serv .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr{font-weight:700;font-size:.8em;line-height:20px;margin-left:.82rem}#onetrust-pc-sdk .ot-subgrp-cntr .ot-cat-header{font-weight:700;font-size:.8em;line-height:20px}#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-vnd-serv .ot-vnd-lst-cont .ot-accordion-layout .ot-acc-hdr div.ot-chkbox{margin-left:.82rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr{padding:.7rem 0;margin:0;display:flex;width:100%;align-items:center;justify-content:space-between}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr div:first-child,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr div:first-child,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr div:first-child{margin-left:.5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr div:last-child,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr div:last-child,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr div:last-child{margin-right:.5rem;margin-left:.5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-always-active{position:relative;right:unset;top:unset;transform:unset}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-plus-minus{top:0}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-arw-cntr{float:none;top:unset;right:unset;transform:unset;margin-top:-2px;position:relative}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-cat-header{flex:1;margin:0 .5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-tgl{position:relative;transform:none;right:0;top:0;float:none}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox{position:relative;margin:0 .5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox label{padding:0}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox label::before{position:relative}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox input{position:absolute;cursor:pointer;width:100%;height:100%;opacity:0;margin:0;top:0;left:0;z-index:1}#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp .ot-acc-hdr h5.ot-cat-header,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp .ot-acc-hdr h4.ot-cat-header{margin:0}#onetrust-pc-sdk .ot-vs-config .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp h5{top:0;line-height:20px}#onetrust-pc-sdk .ot-vs-list{display:flex;flex-direction:column;padding:0;margin:.5rem 4px}#onetrust-pc-sdk .ot-vs-selc-all{display:flex;padding:0;float:unset;align-items:center;justify-content:flex-start}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf{justify-content:flex-end}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf.ot-caret-conf .ot-sel-all-chkbox{margin-right:48px}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf .ot-sel-all-chkbox{margin:0;padding:0;margin-right:14px;justify-content:flex-end}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf #ot-selall-vencntr.ot-chkbox,#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf #ot-selall-vencntr.ot-tgl{display:inline-block;right:unset;width:auto;height:auto;float:none}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf #ot-selall-vencntr label{width:45px;height:25px}#onetrust-pc-sdk .ot-vs-selc-all .ot-sel-all-chkbox{margin-right:11px;margin-left:.75rem;display:flex;align-items:center}#onetrust-pc-sdk .ot-vs-selc-all .sel-all-hdr{margin:0 1.25rem;font-size:.812em;line-height:normal;text-align:center;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-vnd-list-cnt #ot-selall-vencntr.ot-chkbox{float:unset;right:0}#onetrust-pc-sdk[dir=rtl] #ot-back-arw,#onetrust-pc-sdk[dir=rtl] input~.ot-acc-hdr .ot-arw{transform:rotate(180deg);-o-transform:rotate(180deg);-ms-transform:rotate(180deg);-webkit-transform:rotate(180deg)}#onetrust-pc-sdk[dir=rtl] input:checked~.ot-acc-hdr .ot-arw{transform:rotate(270deg);-o-transform:rotate(270deg);-ms-transform:rotate(270deg);-webkit-transform:rotate(270deg)}#onetrust-pc-sdk[dir=rtl] .ot-chkbox label::after{transform:rotate(45deg);-webkit-transform:rotate(45deg);-o-transform:rotate(45deg);-ms-transform:rotate(45deg);border-left:0;border-right:3px solid}#onetrust-pc-sdk[dir=rtl] .ot-search-cntr>svg{right:0}@media only screen and (max-width: 600px){#onetrust-pc-sdk.otPcCenter{left:0;min-width:100%;height:100%;top:0;border-radius:0}#onetrust-pc-sdk #ot-pc-content,#onetrust-pc-sdk.ot-ftr-stacked .ot-btn-container{margin:1px 3px 0 10px;padding-right:10px;width:calc(100% - 23px)}#onetrust-pc-sdk .ot-btn-container button{max-width:none;letter-spacing:.01em}#onetrust-pc-sdk #close-pc-btn-handler{top:10px;right:17px}#onetrust-pc-sdk p{font-size:.7em}#onetrust-pc-sdk #ot-pc-hdr{margin:10px 10px 0 5px;width:calc(100% - 15px)}#onetrust-pc-sdk .vendor-search-handler{font-size:1em}#onetrust-pc-sdk #ot-back-arw{margin-left:12px}#onetrust-pc-sdk #ot-lst-cnt{margin:0;padding:0 5px 0 10px;min-width:95%}#onetrust-pc-sdk .switch+p{max-width:80%}#onetrust-pc-sdk .ot-ftr-stacked button{width:100%}#onetrust-pc-sdk #ot-fltr-cnt{max-width:320px;width:90%;border-top-right-radius:0;border-bottom-right-radius:0;margin:0;margin-left:15px;left:auto;right:40px;top:85px}#onetrust-pc-sdk .ot-fltr-opt{margin-left:25px;margin-bottom:10px}#onetrust-pc-sdk .ot-pc-refuse-all-handler{margin-bottom:0}#onetrust-pc-sdk #ot-fltr-cnt{right:40px}}@media only screen and (max-width: 476px){#onetrust-pc-sdk .ot-fltr-cntr,#onetrust-pc-sdk #ot-fltr-cnt{right:10px}#onetrust-pc-sdk #ot-anchor{right:25px}#onetrust-pc-sdk button{width:100%}#onetrust-pc-sdk:not(.ot-addtl-vendors) #ot-pc-lst:not(.ot-enbl-chr) .ot-sel-all{padding-right:9px}#onetrust-pc-sdk:not(.ot-addtl-vendors) #ot-pc-lst:not(.ot-enbl-chr) .ot-tgl-cntr{right:0}}@media(min-width: 768px){#onetrust-pc-sdk.ot-tgl-with-label .ot-label-status{display:inline}#onetrust-pc-sdk.ot-tgl-with-label #ot-pc-lst .ot-label-status{display:none}}@media only screen and (max-width: 896px)and (max-height: 425px)and (orientation: landscape){#onetrust-pc-sdk.otPcCenter{left:0;top:0;min-width:100%;height:100%;border-radius:0}#onetrust-pc-sdk .ot-pc-header{height:auto;min-height:20px}#onetrust-pc-sdk .ot-pc-header .ot-pc-logo{max-height:30px}#onetrust-pc-sdk .ot-pc-footer{max-height:60px;overflow-y:auto}#onetrust-pc-sdk #ot-pc-content,#onetrust-pc-sdk #ot-pc-lst{bottom:70px}#onetrust-pc-sdk.ot-ftr-stacked #ot-pc-content{bottom:70px}#onetrust-pc-sdk #ot-anchor{left:initial;right:50px}#onetrust-pc-sdk #ot-lst-title{margin-top:12px}#onetrust-pc-sdk #ot-lst-title *{font-size:inherit}#onetrust-pc-sdk #ot-pc-hdr input{margin-right:0;padding-right:45px}#onetrust-pc-sdk .switch+p{max-width:85%}#onetrust-pc-sdk #ot-sel-blk{position:static}#onetrust-pc-sdk #ot-pc-lst{overflow:auto}#onetrust-pc-sdk #ot-lst-cnt{max-height:none;overflow:initial}#onetrust-pc-sdk #ot-lst-cnt.no-results{height:auto}#onetrust-pc-sdk input{font-size:1em !important}#onetrust-pc-sdk p{font-size:.6em}#onetrust-pc-sdk #ot-fltr-modal{width:100%;top:0}#onetrust-pc-sdk ul li p,#onetrust-pc-sdk .category-vendors-list-handler,#onetrust-pc-sdk .category-vendors-list-handler+a,#onetrust-pc-sdk .category-host-list-handler{font-size:.6em}#onetrust-pc-sdk.ot-shw-fltr #ot-anchor{display:none !important}#onetrust-pc-sdk.ot-shw-fltr #ot-pc-lst{height:100% !important;overflow:hidden;top:0px}#onetrust-pc-sdk.ot-shw-fltr #ot-fltr-cnt{margin:0;height:100%;max-height:none;padding:10px;top:0;width:calc(100% - 20px);position:absolute;right:0;left:0;max-width:none}#onetrust-pc-sdk.ot-shw-fltr .ot-fltr-scrlcnt{max-height:calc(100% - 65px)}}
            #onetrust-consent-sdk #onetrust-pc-sdk,
                #onetrust-consent-sdk #ot-search-cntr,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-switch.ot-toggle,
                #onetrust-consent-sdk #onetrust-pc-sdk ot-grp-hdr1 .checkbox,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-title:after
                ,#onetrust-consent-sdk #onetrust-pc-sdk #ot-sel-blk,
                        #onetrust-consent-sdk #onetrust-pc-sdk #ot-fltr-cnt,
                        #onetrust-consent-sdk #onetrust-pc-sdk #ot-anchor {
                    background-color: #FFF;
                }
               
            #onetrust-consent-sdk #onetrust-pc-sdk h3,
                #onetrust-consent-sdk #onetrust-pc-sdk h4,
                #onetrust-consent-sdk #onetrust-pc-sdk h5,
                #onetrust-consent-sdk #onetrust-pc-sdk h6,
                #onetrust-consent-sdk #onetrust-pc-sdk p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-ven-lst .ot-ven-opts p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-desc,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-title,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-li-title,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-sel-all-hdr span,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-info,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-fltr-modal #modal-header,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-checkbox label span,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst #ot-sel-blk p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst #ot-lst-title h3,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst .back-btn-handler p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst .ot-ven-name,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst #ot-ven-lst .consent-category,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-inactive-leg-btn,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-label-status,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-chkbox label span,
                #onetrust-consent-sdk #onetrust-pc-sdk #clear-filters-handler,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-optout-signal
                {
                    color: #2E2E2E;
                }
             #onetrust-consent-sdk #onetrust-pc-sdk .privacy-notice-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-pgph-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk .category-vendors-list-handler,
                    #onetrust-consent-sdk #onetrust-pc-sdk .category-vendors-list-handler + a,
                    #onetrust-consent-sdk #onetrust-pc-sdk .category-host-list-handler,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-ven-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-ven-legclaim-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-name a,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-acc-hdr .ot-host-expand,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-info a,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-content #ot-pc-desc .ot-link-btn,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info a,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info a
                    {
                        color: #007398;
                    }
            #onetrust-consent-sdk #onetrust-pc-sdk .category-vendors-list-handler:hover { text-decoration: underline;}
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-grpcntr.ot-acc-txt,
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-txt .ot-subgrp-tgl .ot-switch.ot-toggle
             {
                background-color: #F8F8F8;
            }
             #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-info,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-txt .ot-ven-dets
                            {
                                background-color: #F8F8F8;
                            }
        #onetrust-consent-sdk #onetrust-pc-sdk
            button:not(#clear-filters-handler):not(.ot-close-icon):not(#filter-btn-handler):not(.ot-remove-objection-handler):not(.ot-obj-leg-btn-handler):not([aria-expanded]):not(.ot-link-btn),
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-active-leg-btn {
                background-color: #007398;border-color: #007398;
                color: #FFF;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-active-menu {
                border-color: #007398;
            }
            
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-remove-objection-handler{
                background-color: transparent;
                border: 1px solid transparent;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-inactive-leg-btn {
                background-color: #FFFFFF;
                color: #4D4D4D; border-color: #4D4D4D;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-tgl input:focus + .ot-switch, .ot-switch .ot-switch-nob, .ot-switch .ot-switch-nob:before,
            #onetrust-pc-sdk .ot-checkbox input[type="checkbox"]:focus + label::before,
            #onetrust-pc-sdk .ot-chkbox input[type="checkbox"]:focus + label::before {
                outline-color: #000000;
                outline-width: 1px;
                outline-offset: 1px;
            }
            #onetrust-pc-sdk .ot-host-item > button:focus, #onetrust-pc-sdk .ot-ven-item > button:focus {
                border: 1px solid #000000;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk *:focus,
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-vlst-cntr > a:focus {
               outline: 1px solid #000000;
               outline-offset: 1px;
            }#onetrust-pc-sdk .ot-vlst-cntr .ot-ext-lnk,  #onetrust-pc-sdk .ot-ven-hdr .ot-ext-lnk{
                    background-image: url('https://cdn.cookielaw.org/logos/static/ot_external_link.svg');
                }
            /*! v6.10.0 2021-01-18 */
div#onetrust-consent-sdk #onetrust-banner-sdk{border-top:2px solid #d0d3d4!important;box-shadow:none;padding:24px}div#onetrust-consent-sdk button{border-radius:0!important;box-shadow:none!important;box-sizing:border-box!important;font-size:20px!important;font-weight:400!important;letter-spacing:0!important;max-width:none!important;white-space:nowrap!important}div#onetrust-consent-sdk button:not(.ot-link-btn){background-color:#007dbc!important;border:2px solid #007dbc!important;color:#fff!important;height:48px!important;padding:0 1em!important;width:auto!important}div#onetrust-consent-sdk button:hover{background-color:#fff!important;border-color:#949494!important;color:#2e2e2e!important}div#onetrust-consent-sdk button.ot-link-btn{color:#007dbc!important;font-size:16px!important;text-decoration:underline}div#onetrust-consent-sdk button.ot-link-btn:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-consent-sdk a,div#onetrust-pc-sdk a{color:#007dbc!important;text-decoration:underline!important}div#onetrust-consent-sdk a,div#onetrust-consent-sdk button,div#onetrust-consent-sdk p:hover{opacity:1!important}div#onetrust-consent-sdk a:focus,div#onetrust-consent-sdk button:focus,div#onetrust-consent-sdk input:focus{outline:2px solid #949494!important;outline-offset:1px!important}div#onetrust-banner-sdk .ot-sdk-container{padding:0;width:auto}div#onetrust-banner-sdk .ot-sdk-row{align-items:flex-start;box-sizing:border-box;display:flex;flex-direction:column;justify-content:space-between;margin:auto;max-width:1152px}div#onetrust-banner-sdk .ot-sdk-row:after{display:none}div#onetrust-banner-sdk #onetrust-group-container,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-group-container,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{flex-grow:1;width:auto}div#onetrust-banner-sdk #onetrust-policy,div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy{margin:0;overflow:visible}div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy-text,div#onetrust-consent-sdk #onetrust-policy-text{font-size:16px;line-height:24px;max-width:44em;margin:0}div#onetrust-consent-sdk #onetrust-policy-text a[href]{font-weight:400;margin-left:8px}div#onetrust-banner-sdk #onetrust-button-group-parent{flex:0 0 auto;margin:32px 0 0;width:100%}div#onetrust-banner-sdk #onetrust-button-group{display:flex;flex-direction:row;flex-wrap:wrap;justify-content:flex-end;margin:-8px}div#onetrust-banner-sdk .banner-actions-container{display:flex;flex:1 0 auto}div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button:last-of-type,div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto;margin:8px;width:auto}div#onetrust-consent-sdk #onetrust-pc-btn-handler{background-color:#fff!important;color:inherit!important}div#onetrust-banner-sdk #onetrust-close-btn-container{display:none}@media only screen and (min-width:556px){div#onetrust-consent-sdk #onetrust-banner-sdk{padding:40px}div#onetrust-banner-sdk #onetrust-policy{margin:0 40px 0 0}div#onetrust-banner-sdk .ot-sdk-row{align-items:center;flex-direction:row}div#onetrust-banner-sdk #onetrust-button-group-parent,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-button-group-parent,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{margin:0;padding:0;width:auto}div#onetrust-banner-sdk #onetrust-button-group,div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group{align-items:stretch;flex-direction:column-reverse;margin:0}div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto}}@media only screen and (min-width:768px){div#onetrust-banner-sdk #onetrust-policy{margin:0 48px 0 0}div#onetrust-consent-sdk #onetrust-banner-sdk{padding:48px}}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-size:16px;line-height:24px}div#onetrust-consent-sdk #onetrust-pc-sdk p,div#onetrust-pc-sdk #ot-pc-desc,div#onetrust-pc-sdk .category-host-list-handler,div#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{font-size:16px;font-weight:400;line-height:24px}div#onetrust-consent-sdk a:hover,div#onetrust-pc-sdk a:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-pc-sdk{border-radius:0;bottom:0;height:auto;left:0;margin:auto;max-width:100%;overflow:hidden;right:0;top:0;width:512px;max-height:800px}div#onetrust-pc-sdk .ot-pc-header{display:none}div#onetrust-pc-sdk #ot-pc-content{overscroll-behavior:contain;padding:0 12px 0 24px;margin:16px 4px 0 0;top:0;right:16px;left:0;width:auto}div#onetrust-pc-sdk #ot-category-title,div#onetrust-pc-sdk #ot-pc-title{font-size:24px;font-weight:400;line-height:32px;margin:0 0 16px}div#onetrust-pc-sdk #ot-pc-desc{padding:0}div#onetrust-pc-sdk #ot-pc-desc a{display:inline}div#onetrust-pc-sdk #accept-recommended-btn-handler{display:none!important}div#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:2px solid #007dbc;outline-offset:-1px;transition:none}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item{border-width:0 0 2px}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item:first-of-type{border-width:2px 0}div#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{padding:8px 0;width:100%}div#onetrust-pc-sdk .ot-plus-minus{transform:translateY(2px)}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item>button{background:0 0!important;border:0!important;height:44px!important;max-width:none!important;width:calc(100% - 48px)!important}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-weight:700}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr{padding:0}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item .ot-acc-grpdesc{padding:0;width:100%}div#onetrust-pc-sdk .ot-acc-grpcntr .ot-subgrp-cntr{border:0;padding:0}div#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp{margin:0}div#onetrust-pc-sdk .ot-always-active-group .ot-cat-header{width:calc(100% - 160px)}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{width:calc(100% - 88px)}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active{color:inherit;font-size:12px;font-weight:400;line-height:1.5;padding-right:48px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:before{border-radius:12px;position:absolute;right:0;top:0;content:'';background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:after{border-radius:50%;position:absolute;right:5px;top:4px;content:'';background-color:#007dbc;height:12px;width:12px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active,div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl{right:2px}div#onetrust-pc-sdk .ot-switch{display:block;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob,div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob{background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob:before{background-color:#737373;height:8px;left:4px;top:4px;width:8px}div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob:before{background-color:#007dbc;height:12px;left:0;top:2px;width:12px}div#onetrust-pc-sdk .ot-tgl input:focus+.ot-switch .ot-switch-nob{box-shadow:0 0;outline:2px solid #007dbc!important;outline-offset:1px;transition:none}div#onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-grpcntr.ot-acc-txt{background-color:transparent;padding-left:3px}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr,div#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr{overflow:visible;width:100%}div#onetrust-pc-sdk .ot-pc-footer{border-top:0 solid}div#onetrust-pc-sdk .ot-btn-container{padding-top:24px;text-align:center}div#onetrust-pc-sdk .ot-pc-footer button{margin:8px 0;background-color:#fff}div#onetrust-pc-sdk .ot-pc-footer-logo{background-color:#fff}div#onetrust-pc-sdk #ot-lst-title span{font-size:24px;font-weight:400;line-height:32px}div#onetrust-pc-sdk #ot-host-lst .ot-host-desc,div#onetrust-pc-sdk #ot-host-lst .ot-host-expand,div#onetrust-pc-sdk #ot-host-lst .ot-host-name,div#onetrust-pc-sdk #ot-host-lst .ot-host-name a,div#onetrust-pc-sdk .back-btn-handler,div#onetrust-pc-sdk .ot-host-opt li>div div{font-size:16px;font-weight:400;line-height:24px}div#onetrust-pc-sdk #ot-pc-lst{top:0}div#onetrust-pc-sdk .back-btn-handler{text-decoration:none!important}div#onetrust-pc-sdk #filter-btn-handler:hover svg{filter:invert(1)}div#onetrust-pc-sdk .back-btn-handler svg{width:16px;height:16px}div#onetrust-pc-sdk .ot-host-item>button{background:0 0!important;border:0!important;height:66px!important;max-width:none!important;width:calc(100% - 5px)!important;transform:translate(2px,2px)}div#onetrust-pc-sdk .ot-host-item{border-bottom:2px solid #b9b9b9;padding:0}div#onetrust-pc-sdk .ot-host-item .ot-acc-hdr{margin:0 0 -6px;padding:8px 0}div#onetrust-pc-sdk ul li:first-child{border-top:2px solid #b9b9b9}div#onetrust-pc-sdk .ot-host-item .ot-plus-minus{margin:0 8px 0 0}div#onetrust-pc-sdk .ot-search-cntr{width:calc(100% - 48px)}div#onetrust-pc-sdk .ot-host-opt .ot-host-info{background-color:transparent}div#onetrust-pc-sdk .ot-host-opt li>div div{padding:0}div#onetrust-pc-sdk #vendor-search-handler{border-radius:0;border-color:#939393;border-style:solid;border-width:2px 0 2px 2px;font-size:20px;height:48px;margin:0}div#onetrust-pc-sdk .ot-lst-subhdr svg{right:0;top:8px}div#onetrust-pc-sdk .ot-fltr-cntr{box-sizing:border-box;right:0;width:48px}div#onetrust-pc-sdk #filter-btn-handler{width:48px!important;padding:8px!important}div#onetrust-pc-sdk #ot-fltr-modal{max-height:400px}
.ot-sdk-cookie-policy{font-family:inherit;font-size:16px}.ot-sdk-cookie-policy.otRelFont{font-size:1rem}.ot-sdk-cookie-policy h3,.ot-sdk-cookie-policy h4,.ot-sdk-cookie-policy h6,.ot-sdk-cookie-policy p,.ot-sdk-cookie-policy li,.ot-sdk-cookie-policy a,.ot-sdk-cookie-policy th,.ot-sdk-cookie-policy #cookie-policy-description,.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group,.ot-sdk-cookie-policy #cookie-policy-title{color:dimgray}.ot-sdk-cookie-policy #cookie-policy-description{margin-bottom:1em}.ot-sdk-cookie-policy h4{font-size:1.2em}.ot-sdk-cookie-policy h6{font-size:1em;margin-top:2em}.ot-sdk-cookie-policy th{min-width:75px}.ot-sdk-cookie-policy a,.ot-sdk-cookie-policy a:hover{background:#fff}.ot-sdk-cookie-policy thead{background-color:#f6f6f4;font-weight:bold}.ot-sdk-cookie-policy .ot-mobile-border{display:none}.ot-sdk-cookie-policy section{margin-bottom:2em}.ot-sdk-cookie-policy table{border-collapse:inherit}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy{font-family:inherit;font-size:1rem}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h3,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h4,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h6,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy p,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy li,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-title{color:dimgray}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description{margin-bottom:1em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-subgroup{margin-left:1.5em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group-desc,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-table-header,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy span,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td{font-size:.9em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td span,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td a{font-size:inherit}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group{font-size:1em;margin-bottom:.6em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-title{margin-bottom:1.2em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy>section{margin-bottom:1em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th{min-width:75px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a:hover{background:#fff}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy thead{background-color:#f6f6f4;font-weight:bold}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-mobile-border{display:none}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy section{margin-bottom:2em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-subgroup ul li{list-style:disc;margin-left:1.5em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-subgroup ul li h4{display:inline-block}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table{border-collapse:inherit;margin:auto;border:1px solid #d7d7d7;border-radius:5px;border-spacing:initial;width:100%;overflow:hidden}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table th,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table td{border-bottom:1px solid #d7d7d7;border-right:1px solid #d7d7d7}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr:last-child td{border-bottom:0px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr th:last-child,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr td:last-child{border-right:0px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-host,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-cookies-type{width:25%}.ot-sdk-cookie-policy[dir=rtl]{text-align:left}#ot-sdk-cookie-policy h3{font-size:1.5em}@media only screen and (max-width: 530px){.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) table,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) thead,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tbody,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) th,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr{display:block}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) thead tr{position:absolute;top:-9999px;left:-9999px}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr{margin:0 0 1em 0}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr:nth-child(odd),.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr:nth-child(odd) a{background:#f6f6f4}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td{border:none;border-bottom:1px solid #eee;position:relative;padding-left:50%}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td:before{position:absolute;height:100%;left:6px;width:40%;padding-right:10px}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) .ot-mobile-border{display:inline-block;background-color:#e4e4e4;position:absolute;height:100%;top:0;left:45%;width:2px}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td:before{content:attr(data-label);font-weight:bold}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) li{word-break:break-word;word-wrap:break-word}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table{overflow:hidden}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table td{border:none;border-bottom:1px solid #d7d7d7}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy thead,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy tbody,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy tr{display:block}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-host,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-cookies-type{width:auto}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy tr{margin:0 0 1em 0}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td:before{height:100%;width:40%;padding-right:10px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td:before{content:attr(data-label);font-weight:bold}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy li{word-break:break-word;word-wrap:break-word}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy thead tr{position:absolute;top:-9999px;left:-9999px;z-index:-9999}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr:last-child td{border-bottom:1px solid #d7d7d7;border-right:0px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr:last-child td:last-child{border-bottom:0px}}
                
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h5,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h6,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy li,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy p,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy span,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description {
                        color: #696969;
                    }
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th {
                        color: #696969;
                    }
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group {
                        color: #696969;
                    }
                    
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-title {
                            color: #696969;
                        }
                    
            
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table th {
                            background-color: #F8F8F8;
                        }
                    
            .ot-floating-button__front{background-image:url('https://cdn.cookielaw.org/logos/static/ot_persistent_cookie_icon.png')}</style><script src="https://static.addtoany.com/menu/modules/core.pt5ow5lr.js" type="module"></script><style type="text/css">.a2a_hide{display:none}.a2a_logo_color{background-color:#0166ff}.a2a_menu,.a2a_menu *{-moz-box-sizing:content-box;-webkit-box-sizing:content-box;box-sizing:content-box;float:none;margin:0;padding:0;position:static;height:auto;width:auto}.a2a_menu{border-radius:6px;display:none;direction:ltr;background:#FFF;font:16px sans-serif-light,HelveticaNeue-Light,"Helvetica Neue Light","Helvetica Neue",Arial,Helvetica,"Liberation Sans",sans-serif;color:#000;line-height:12px;border:1px solid #CCC;vertical-align:baseline;overflow:hidden}.a2a_mini{min-width:200px;position:absolute;width:300px;z-index:9999997}.a2a_overlay{display:none;background:#616c7deb;backdrop-filter:blur(4px);-webkit-backdrop-filter:blur(4px);position:fixed;top:0;right:0;left:0;bottom:0;z-index:9999998;-webkit-tap-highlight-color:transparent;transition:opacity .14s,backdrop-filter .14s}.a2a_full{background:#FFF;border:1px solid #FFF;box-shadow:#2a2a2a1a 0 0 20px 10px;height:auto;height:calc(320px);top:15%;left:50%;margin-left:-320px;position:fixed;text-align:center;width:640px;z-index:9999999;transition:transform .14s,opacity .14s}.a2a_full_footer,.a2a_full_header,.a2a_full_services{border:0;margin:0;padding:12px;box-sizing:border-box}.a2a_full_header{padding-bottom:8px}.a2a_full_services{height:280px;overflow-y:scroll;padding:0 12px;-webkit-overflow-scrolling:touch}.a2a_full_services .a2a_i{display:inline-block;float:none;width:181px;width:calc(33.334% - 18px)}div.a2a_full_footer{font-size:12px;text-align:center;padding:8px 14px}div.a2a_full_footer a,div.a2a_full_footer a:visited{display:inline;font-size:12px;line-height:14px;padding:8px 14px}div.a2a_full_footer a:focus,div.a2a_full_footer a:hover{background:0 0;border:0;color:#0166FF}div.a2a_full_footer a span.a2a_s_a2a,div.a2a_full_footer a span.a2a_w_a2a{background-size:14px;border-radius:3px;display:inline-block;height:14px;line-height:14px;margin:0 3px 0 0;vertical-align:top;width:14px}.a2a_modal{height:0;left:50%;margin-left:-320px;position:fixed;text-align:center;top:15%;width:640px;z-index:9999999;transition:transform .14s,opacity .14s;-webkit-tap-highlight-color:transparent}.a2a_modal_body{background:0 0;border:0;font:24px sans-serif-light,HelveticaNeue-Light,"Helvetica Neue Light","Helvetica Neue",Arial,Helvetica,"Liberation Sans",sans-serif;position:relative;height:auto;width:auto}.a2a_thanks{color:#fff;height:auto;margin-top:20px;width:auto}.a2a_thanks>div:first-child{margin:0 0 40px 0}.a2a_thanks div *{height:inherit}#a2a_copy_link{background:#FFF;border:1px solid #FFF;cursor:pointer;margin-top:15%}label.a2a_s_link#a2a_copy_link_icon,label.a2a_w_link#a2a_copy_link_icon{background-size:48px;border-radius:0;display:inline-block;height:48px;left:0;line-height:48px;margin:0 3px 0 0;position:absolute;vertical-align:top;width:48px}#a2a_modal input#a2a_copy_link_text{background-color:transparent;border:0;color:#2A2A2A;cursor:pointer;font:inherit;height:48px;left:62px;max-width:initial;min-height:auto;padding:0;position:relative;width:564px;width:calc(100% - 76px)}#a2a_copy_link_copied{background-color:#0166ff;color:#fff;display:none;font:inherit;font-size:16px;margin-top:1px;padding:3px 8px}@media (forced-colors:active){.a2a_color_buttons a,.a2a_svg{forced-color-adjust:none}}@media (prefers-color-scheme:dark){.a2a_menu a,.a2a_menu a.a2a_i,.a2a_menu a.a2a_i:visited,.a2a_menu a.a2a_more,i.a2a_i{border-color:#2a2a2a!important;color:#fff!important}.a2a_menu a.a2a_i:active,.a2a_menu a.a2a_i:focus,.a2a_menu a.a2a_i:hover,.a2a_menu a.a2a_more:active,.a2a_menu a.a2a_more:focus,.a2a_menu a.a2a_more:hover,.a2a_menu_find_container{border-color:#444!important;background-color:#444!important}.a2a_menu:not(.a2a_thanks){background-color:#2a2a2a;border-color:#2a2a2a}.a2a_menu_find{color:#fff!important}.a2a_menu label.a2a_s_find svg{background-color:transparent!important}.a2a_menu label.a2a_s_find svg path{fill:#fff!important}.a2a_full{box-shadow:#00000066 0 0 20px 10px}.a2a_overlay{background-color:#373737eb}}@media print{.a2a_floating_style,.a2a_menu,.a2a_overlay{visibility:hidden}}@keyframes a2aFadeIn{from{opacity:0}to{opacity:1}}.a2a_starting{opacity:0}.a2a_starting.a2a_full,.a2a_starting.a2a_modal{transform:scale(.8)}@media (max-width:639px){.a2a_full{border-radius:0;top:15%;left:0;margin-left:auto;width:100%}.a2a_modal{left:0;margin-left:10px;width:calc(100% - 20px)}}@media (min-width:318px) and (max-width:437px){.a2a_full .a2a_full_services .a2a_i{width:calc(50% - 18px)}}@media (max-width:317px){.a2a_full .a2a_full_services .a2a_i{width:calc(100% - 18px)}}@media (max-height:436px){.a2a_full{bottom:40px;height:auto;top:40px}}@media (max-height:550px){.a2a_modal{top:30px}}@media (max-height:360px){.a2a_modal{top:20px}.a2a_thanks>div:first-child{margin-bottom:20px}}.a2a_menu a{color:#0166FF;text-decoration:none;font:16px sans-serif-light,HelveticaNeue-Light,"Helvetica Neue Light","Helvetica Neue",Arial,Helvetica,"Liberation Sans",sans-serif;line-height:14px;height:auto;width:auto;outline:0}.a2a_menu a.a2a_i:visited,.a2a_menu a.a2a_more{color:#0166FF}.a2a_menu a.a2a_i:active,.a2a_menu a.a2a_i:focus,.a2a_menu a.a2a_i:hover,.a2a_menu a.a2a_more:active,.a2a_menu a.a2a_more:focus,.a2a_menu a.a2a_more:hover{color:#2A2A2A;border-color:#EEE;border-style:solid;background-color:#EEE;text-decoration:none}.a2a_menu label.a2a_s_find{background-size:24px;height:24px;left:8px;pointer-events:auto;position:absolute;top:7px;width:24px}.a2a_menu label.a2a_s_find svg{background-color:#FFF}.a2a_menu label.a2a_s_find svg path{fill:#CCC}#a2a_menu_container{display:inline-block}.a2a_menu_find_container{border:1px solid #CCC;border-radius:6px;padding:2px 24px 2px 0;position:relative;text-align:left}.a2a_cols_container .a2a_col1{overflow-x:hidden;overflow-y:auto;-webkit-overflow-scrolling:touch}#a2a_modal input,#a2a_modal input[type=text],.a2a_menu input,.a2a_menu input[type=text]{display:block;background-image:none;box-shadow:none;line-height:100%;margin:0;outline:0;overflow:hidden;padding:0;-moz-box-shadow:none;-webkit-box-shadow:none;-webkit-appearance:none}#a2afeed_find_container input,#a2afeed_find_container input[type=text],#a2apage_find_container input,#a2apage_find_container input[type=text]{background-color:transparent;border:0;box-sizing:content-box;color:#2A2A2A;float:none;font:inherit;font-size:16px;height:28px;line-height:20px;left:38px;outline:0;margin:0;max-width:initial;min-height:initial;padding:2px 0;position:relative;width:99%}.a2a_clear{clear:both}.a2a_svg{background-repeat:no-repeat;display:block;overflow:hidden;height:32px;line-height:32px;padding:0;pointer-events:none;width:32px}.a2a_svg svg{background-repeat:no-repeat;background-position:50% 50%;border:none;display:block;left:0;margin:0 auto;overflow:hidden;padding:0;position:relative;top:0;width:auto;height:auto}a.a2a_i,i.a2a_i{display:block;float:left;border:1px solid #FFF;line-height:24px;padding:6px 8px;text-align:left;white-space:nowrap;overflow:hidden;text-overflow:ellipsis;width:132px}a.a2a_i span,a.a2a_more span{display:inline-block;overflow:hidden;vertical-align:top}a.a2a_i .a2a_svg{margin:0 6px 0 0}a.a2a_i .a2a_svg,a.a2a_more .a2a_svg{background-size:24px;height:24px;line-height:24px;width:24px}a.a2a_sss:hover{border-left:1px solid #CCC}a.a2a_more{border-bottom:1px solid #FFF;border-left:0;border-right:0;line-height:24px;margin:6px 0 0;padding:6px;-webkit-touch-callout:none}a.a2a_more span{height:24px;margin:0 6px 0 0}.a2a_kit .a2a_svg{background-repeat:repeat}.a2a_default_style a:empty,.a2a_flex_style a:empty,.a2a_floating_style a:empty,.a2a_overlay_style a:empty{display:none}.a2a_color_buttons a,.a2a_floating_style a{text-decoration:none}.a2a_default_style:not(.a2a_flex_style) a{float:left;line-height:16px;padding:0 2px}.a2a_default_style a:hover .a2a_svg,.a2a_floating_style a:hover .a2a_svg,.a2a_overlay_style a:hover .a2a_svg svg{opacity:.7}.a2a_overlay_style.a2a_default_style a:hover .a2a_svg{opacity:1}.a2a_default_style .a2a_count,.a2a_default_style .a2a_svg,.a2a_floating_style .a2a_svg,.a2a_menu .a2a_svg,.a2a_vertical_style .a2a_count,.a2a_vertical_style .a2a_svg{border-radius:4px}.a2a_default_style .a2a_counter img,.a2a_default_style .a2a_dd,.a2a_default_style .a2a_svg{float:left}.a2a_default_style .a2a_img_text{margin-right:4px}.a2a_default_style .a2a_divider{border-left:1px solid #000;display:inline;float:left;height:16px;line-height:16px;margin:0 5px}.a2a_kit a{cursor:pointer;transition:none}.a2a_floating_style{background-color:#fff;border-radius:6px;position:fixed;z-index:9999995}.a2a_overlay_style{z-index:2147483647}.a2a_floating_style,.a2a_overlay_style{animation:a2aFadeIn .2s ease-in;padding:4px}.a2a_vertical_style:not(.a2a_flex_style) a{clear:left;display:block;overflow:hidden;padding:4px}.a2a_floating_style.a2a_default_style{bottom:0}.a2a_floating_style.a2a_default_style a,.a2a_overlay_style.a2a_default_style a{padding:4px}.a2a_count{background-color:#fff;border:1px solid #ccc;box-sizing:border-box;color:#2a2a2a;display:block;float:left;font:12px Arial,Helvetica,sans-serif;height:16px;margin-left:4px;position:relative;text-align:center;width:50px}.a2a_count:after,.a2a_count:before{border:solid transparent;border-width:4px 4px 4px 0;content:"";height:0;left:0;line-height:0;margin:-4px 0 0 -4px;position:absolute;top:50%;width:0}.a2a_count:before{border-right-color:#ccc}.a2a_count:after{border-right-color:#fff;margin-left:-3px}.a2a_count span{animation:a2aFadeIn .14s ease-in}.a2a_vertical_style .a2a_counter img{display:block}.a2a_vertical_style .a2a_count{float:none;margin-left:0;margin-top:6px}.a2a_vertical_style .a2a_count:after,.a2a_vertical_style .a2a_count:before{border:solid transparent;border-width:0 4px 4px 4px;content:"";height:0;left:50%;line-height:0;margin:-4px 0 0 -4px;position:absolute;top:0;width:0}.a2a_vertical_style .a2a_count:before{border-bottom-color:#ccc}.a2a_vertical_style .a2a_count:after{border-bottom-color:#fff;margin-top:-3px}.a2a_color_buttons .a2a_count,.a2a_color_buttons .a2a_count:after,.a2a_color_buttons .a2a_count:before,.a2a_color_buttons.a2a_vertical_style .a2a_count:after,.a2a_color_buttons.a2a_vertical_style .a2a_count:before{background-color:transparent;border:none;color:#fff;float:none;width:auto}.a2a_color_buttons.a2a_vertical_style .a2a_count{margin-top:0}.a2a_flex_style{display:flex;align-items:flex-start;gap:0}.a2a_default_style.a2a_flex_style{left:0;right:0;width:100%}.a2a_vertical_style.a2a_flex_style{flex-direction:column;top:0;bottom:0}.a2a_flex_style a{display:flex;justify-content:center;flex:1;padding:4px}.a2a_flex_style.a2a_vertical_style a{flex-direction:column}.a2a_floating_style.a2a_color_buttons,.a2a_floating_style.a2a_flex_style{border-radius:0;padding:0}.a2a_floating_style.a2a_default_style.a2a_flex_style{bottom:0}.a2a_kit.a2a_flex_style .a2a_counter img,.a2a_kit.a2a_flex_style .a2a_dd,.a2a_kit.a2a_flex_style .a2a_svg{float:none}.a2a_nowrap{white-space:nowrap}.a2a_note{margin:0 auto;padding:9px;font-size:12px;text-align:center}.a2a_note .a2a_note_note{margin:0;color:#2A2A2A}.a2a_wide a{display:block;margin-top:3px;border-top:1px solid #EEE;text-align:center}.a2a_label{position:absolute!important;clip-path:polygon(0px 0px,0px 0px,0px 0px);-webkit-clip-path:polygon(0px 0px,0px 0px,0px 0px);overflow:hidden;height:1px;width:1px}.a2a_kit,.a2a_menu,.a2a_modal,.a2a_overlay{-ms-touch-action:manipulation;touch-action:manipulation;outline:0}.a2a_dd{-webkit-user-drag:none}.a2a_dd img{border:0}.a2a_button_facebook_like iframe{max-width:none}</style><style></style><link href="https://crossmark-cdn.crossref.org/widget/v2.0/style.css" type="text/css" rel="stylesheet"></head>
<body class="pb-ui">
    <div class="skip">
        <a href="#page-body-id" id="skipNavigationLink" title="Skip to Main Content" tabindex="0" class="skipToContent">Skip to Main Content</a>
        <a href="#mega-menu-nav" id="skipToMainMenu" title="Skip to Main Menu" tabindex="0" class="skipToContent">Skip to Main Menu</a>
    </div>



    <div id="pb-page-content" data-ng-non-bindable="">
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="ux3-layout-widget" data-widget-id="832ad3f3-97ac-45ab-9af1-90c6f7bff54a" class="ux3">
        
    

        


    

            

            
                <div class="cell fonts-loaded">
            

            <div data-widget-id="832ad3f3-97ac-45ab-9af1-90c6f7bff54a" data-widget-name="ux3-layout-widget"><div>
    

        


    

            

            

            <div id="a1f1ba25-79af-4194-aaa9-66206104a800" class="widget loginPopup none widget-none widget-compact-all"><div class="wrapped"><div class="widget-body body body-none body-compact-all"><div role="dialog" aria-modal="true" aria-label="Login" class="popup login-popup hidden"><div class="content"><a href="#" aria-label="Close login popup" title="Close login popup" role="button" class="close"><i class="icon-close_thin"></i></a><h2 class="login-title">Login to your account</h2><div class="login-form"><p tabindex="-1" class="message error"></p><form action="/action/doLogin?redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2901337-4" method="post"><input type="hidden" name="redirectUri" value="https://www.cell.com/cell/fulltext/S0092-8674(21)01337-4"><input type="hidden" name="referer" value="/cell/fulltext/S0092-8674(21)01337-4"><input type="hidden" name="ecommRedirect"><input type="hidden" name="code" value="cell-site"><input type="hidden" name="UX3LoginWidget" value="true"><input type="hidden" name="isPopup" value="true"><div class="input-group login-input--wrapper"><div class="label"><label for="login">Email/Username</label></div><input id="login" type="text" name="login" value="" size="15" class="login"><div role="alert" class="error-hint js--hidden"><div class="error-hint__text"><div class="error-text">Your email address is a required field.</div><div class="email-example">E.g., j.smith@mail.com</div></div></div></div><div class="input-group login-input--wrapper"><div class="label"><label for="password">Password</label></div><input id="password" type="password" name="password" value="" autocomplete="off" class="password"><a href="#" class="login__field__action js-show-password">Show</a><div role="alert" class="error-hint js--hidden"><span class="error-hint__text">Your password is a required field.</span></div><div class="actions"><a href="https://www.cell.com/action/requestResetPassword" class="link show-request-reset-password">Forgot password?</a></div></div><div class="remember"><input type="checkbox" name="savePassword" id="a1f1ba25-79af-4194-aaa9-66206104a800-remember" value="1" checked="checked"><label for="a1f1ba25-79af-4194-aaa9-66206104a800-remember">Remember me</label></div><div class="submit"><input type="submit" name="submit" value="Log in" class="button submit primary"></div></form></div><div class="create-account"><div>Don’t have an account?</div><a href="/action/registration?redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2901337-4" class="show-registration">Create a Free Account</a></div></div></div></div></div></div>

            

        

    

        


    

            

            

            <div role="dialog" aria-modal="true" aria-label="Reset password" class="popup top-drawer request-reset-password-drawer hidden"><div class="content"><a href="#" aria-label="Close reset password popup" title="Close reset password popup" role="button" class="close"><i class="icon-close_thin"></i></a><div class="request-reset"><div class="form"><p class="sub">If you don't remember your password, you can reset it by entering your email address and clicking the Reset Password button. You will then receive an email that contains a secure link for resetting your password</p><div tabindex="-1" class="message error"></div><form action="/action/requestResetPassword" method="post" class="request-reset-password-form"><input type="hidden" name="platform"><input type="hidden" name="requestResetPassword" value="true"><input type="hidden" name="codeSite" value="cell-site"><div class="input-group"><div class="label"><label for="reset-password-email">Email*</label></div><input id="reset-password-email" type="text" name="email" value="" size="15" class="email"></div><div class="submit" disabled="disabled"><input type="submit" name="submit" value="Submit" disabled="disabled" class="button primary submit"></div></form></div><div class="success-template hidden"><p>If the address matches a valid account an email will be sent to __email__ with instructions for resetting your password</p></div></div><div class="request-reset-cancel"><a href="#" aria-label="Close reset password popup" title="Close reset password popup" class="cancel">Cancel</a></div></div></div>

            

        

    

        


    

            

            
                <div class="ux3-ads">
            

            <div class="pb-ad">
            <!-- Do Not Modify without contacting ELS-HSOnlineCampaign@elsevier.com -->
<!-- To be kept in sync with Global Commercial Publishing Operations-->
<!-- Start: GPT Async -->
<div id="gpt-Variables" data-adfile="CELLJOURNAL">
	<input type="hidden" id="gptSite" value="cell">
	<input type="hidden" id="gptLoggedIn" value="no">
	<input type="hidden" id="gptAccessType" value="ae:ANON_GUEST|ae:ANON_GUEST">
	<input type="hidden" id="gptDefaultSite" value="cell">
</div>
<div id="gpt-Interscroller" class="gpt-InScrollBox" style="display:none;">
	<div class="gpt-InScrollLabel">ADVERTISEMENT</div>
	<div class="gpt-InScrollContainer">
		<div class="gpt-InScrollWrapper">
			<div class="gpt-InScrollInner">
				<div class="gpt-adinterscroller" data-adsize="interscroller"></div>
			</div>
		</div>
	</div>
	<div class="gpt-InScrollLabel">SCROLL TO CONTINUE WITH CONTENT</div>
</div>
<div id="gpt-DebugDiv" class="gpt-DebugDiv" style="display:none;">
	<div style="width: 100%; display: table;">
		<div style="display: table-row">
			<div style="width: 200px; display: table-cell;">
				<button class="gpt-DebugBtn" onclick="gpt_ShowConsole()">Open GPT Console</button><br>
				<button id="gpt_GrapeShotDebugBtn" class="gpt-DebugBtn" onclick="gpt_GrapeShotDebug();">Open Oracle Keywords</button><br>
				<br>
				<button class="gpt-DebugBtn" onclick="gpt_LoadDebugValues()">Refresh Values</button>
			</div>
			<div style="display: table-cell;">
				<table class="gpt-DebugTable">
					<thead>
						<tr>
							<th>Property</th>
							<th>Value</th>
						</tr>
					</thead>
					<tbody>
						<tr><td class="gpt-DebugTitle">Status</td><td><div class="gptStatus_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Version</td><td><div class="gptVersion_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Ad File</td><td><div class="gptAdFile_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Disable Ads Flag</td><td><div class="gptDisableAds_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Environment</td><td><div class="gptTestENV_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Moat Init</td><td><div class="gptMoatInit_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Moat Ready</td><td><div class="gptMoatReady_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual Ready</td><td><div class="gptContextualReady_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual URL</td><td><div class="gptContextualURL_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual Initial Segments</td><td><div class="gptContextualInit_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual Used Segments</td><td><div class="gptContextualResult_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">AdUnit</td><td><div class="gptSite_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">SubAdUnit</td><td><div class="gptPage_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Custom Targeting</td><td><div class="gptCustomTargeting_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Ad Events</td><td><div class="gptEvent_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Invalid Ad Sizes</td><td><div class="gptInvalidAdSize_Debug"></div></td></tr>
					</tbody>
				</table>
			</div>
		</div>
	</div>
</div>
<style type="text/css">
	.gpt-DebugBtn {
		display: inline-block;
		text-align: center;
		text-decoration: none;
		margin: 2px 0;
		border: solid 1px #122b40;
		border-radius: 0.4em;
		padding: 0.5em 1em;
		color: #ffffff;
		background-color: #204d74;
		font-weight: normal;
		font-size: 12px;
	}

	.gpt-DebugDiv {
		margin: 5px;
	}

	.gpt-DebugTitle {
		font-weight: bold;
	}

	.gpt-DebugTable {
		border: 1px solid #dddddd;
		border-collapse: collapse;
		margin: 5px;
		font-size: 0.9em;
		font-family: sans-serif;
		min-width: 400px;
		margin-bottom: 25px;
	}

		.gpt-DebugTable td:first-child {
			padding: 5px;
		}

		.gpt-DebugTable thead tr th {
			background-color: #d9edf7;
			color: #ffffff;
			text-align: left;
			padding: 5px;
			border-bottom: 2px solid #204d74;
		}

		.gpt-DebugTable th,
		.gpt-DebugTable td {
			padding: 5px;
		}

		.gpt-DebugTable tbody tr {
			border-bottom: 1px solid #dddddd;
		}

			.gpt-DebugTable tbody tr:nth-of-type(even) {
				background-color: #f3f3f3;
			}

			.gpt-DebugTable tbody tr:last-of-type {
				border-bottom: 2px solid #204d74;
			}

	.gpt-InScrollLabel {
		background-color: rgb(0, 0, 0);
		color: rgb(255, 255, 255);
		position: relative;
		max-width: 100vw;
		left: 50%;
		right: 50%;
		margin-left: -50vw;
		margin-right: -50vw;
		padding: 0;
		text-align: center;
		width: 100vw !important;
		font-size: 10px;
	}

	.gpt-InScrollBox {
		height: 100vh !important;
		width: 100vw;
		cursor: pointer;
		max-width: 100vw;
		position: relative;
		left: 50%;
		right: 50%;
		margin-left: -50vw;
		margin-right: -50vw;
		margin-bottom: 50px;
		padding-top: 5px;
		padding-bottom: 5px;
		z-index: 10001;
		background-color: white;
	}

		.gpt-InScrollBox .gpt-InScrollContainer {
			height: 95vh !important;
			position: relative;
		}

		.gpt-InScrollBox .gpt-InScrollWrapper {
			position: absolute !important;
			top: 0 !important;
			left: 0;
			width: 100%;
			height: 95vh !important;
			border: 0 !important;
			margin: 0 !important;
			padding: 0 !important;
			clip: rect(0, auto, auto, 0) !important;
			-webkit-clip-path: polygon(0px 0px, 100% 0px, 100% 100%, 0px 100%) !important;
			clip-path: polygon(0px 0px, 100% 0px, 100% 100%, 0px 100%) !important;
		}

		.gpt-InScrollBox .gpt-InScrollInner {
			position: fixed !important;
			top: 0 !important;
			left: 0 !important;
			bottom: 0;
			margin: auto;
			width: 100vw;
			height: 95vh;
			-webkit-transform: translateZ(0) !important;
			display: -webkit-box;
			display: -ms-flexbox;
			display: flex;
			-webkit-box-orient: vertical;
			-webkit-box-direction: normal;
			-ms-flex-direction: column;
			flex-direction: column;
			-webkit-box-align: center;
			-ms-flex-align: center;
			align-items: center;
			-webkit-box-pack: center;
			-ms-flex-pack: center;
			justify-content: center;
			text-align: center;
		}
</style>
<script class="optanon-category-4" type="text/plain">
	//--------------------------------------------------------------------------
	var gpt_Version = '04072023';
	var gpt_AdFile = $('#gpt-Variables').data('adfile');
	var gpt_MaxMOATInitAttempts = 11;
	var gpt_MaxMOATReadyAttempts = 11;
	var gpt_MaxContextualReadyAttempts = 21;
	var gpt_MOATInitAttempts = 0;
	var gpt_MOATReadyAttempts = 0;
	var gpt_ContextualReadyAttempts = 0;
	var gpt_IsProd = true;
	var gpt_StartTime;
	var gs_channels = 'DEFAULT';
	var gpt_ContextualReady = false;
	var gpt_ContextualURL = 'Not enabled outside of Production';
	var gpt_DisableAdsFlagFound = false;

	var gptEvent_Debug = '';
	var gptCustomTargeting_Debug = '';
	var gptSite_Debug = "";
	var gptPage_Debug = "";
	var gptContextualEnabled = false;
	var gptContextualResult_Debug = '';
	var gptContextualInit_Debug = '';
	var gptContextualURL_Debug = '';
	var gptContextualReady_Debug = '';
	var gptMoatReady_Debug = '';
	var gptMoatInit_Debug = '';
	var gptTestENV_Debug = '';
	var gptDisableAds_Debug = '';
	var gptStatus_Debug = 'GPT Still Loading - Refresh Values';
	var gptInvalidAdSize_Debug = '';
	//--------------------------------------------------------------------------
	$(document).ready(function () {
		//Check to see if we are on a development env
		gpt_CheckEnvironment();

		if (!gpt_DisableAdsFlagFound) {
			gpt_StartTime = new Date();
			if (gpt_AdFile !== "PU") {
				//Once the dom has loaded, begin the process to init the ads.
				gpt_InitScripts();
				gpt_CheckForMOATInit();
			}
		}
	});
</script>
<script type="text/javascript">
	function gpt_CheckEnvironment() {
		if ($('#gptDisableAds').length > 0) {
			gptDisableAds_Debug = 'Flag Detected - Ads Disabled';
			gpt_DisableAdsFlagFound = true;
		}
		else {
			gptDisableAds_Debug = 'Ads Enabled';
			gpt_DisableAdsFlagFound = false;
		}
		switch (gpt_AdFile) {
			case 'PU':
				gptContextualEnabled = true;
				if ($("#Environment").val() !== "Prod") {
					gpt_IsProd = false;
					gptTestENV_Debug = 'Test Environment Detected';
				}
				else {
					gptTestENV_Debug = 'Production Environment Detected';
				}
				break;
			case 'JBS':
			case 'LANCET':
				gptContextualEnabled = true;
				if ($(".gpt-ad").length > 0) {
					//Other ads exist, allow the Prestitial
					$(document.body).append("<div class='gpt-ad' data-adsize='prestitial'></div>");
				}
				if (window.location.hostname.toLowerCase().indexOf('.literatumonline.') >= 0) {
					gpt_IsProd = false;
					gptTestENV_Debug = 'Test Environment Detected';
				}
				else {
					gptTestENV_Debug = 'Production Environment Detected';
				}
				break;
			default:
				//Cell
				gptContextualEnabled = false;
				if (window.location.hostname.toLowerCase().indexOf('.literatumonline.') >= 0) {
					gpt_IsProd = false;
					gptTestENV_Debug = 'Test Environment Detected';
				}
				else {
					gptTestENV_Debug = 'Production Environment Detected';
				}
				break;
		}
	}
	//--------------------------------------------------------------------------
    function gpt_GetNonCMEArticle() {
        var url;
        $('.articleCitation').each(function (i, obj) {
            if ($(obj).find('.toc__item__detials').length > 0) {
                var itemDetails = $(obj).first().find('.toc__item__detials');
                var cmeFound = 0;
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='CME']").length; }
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='(CME)']").length; }
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='Education']").length; }
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='education']").length; }
                if (cmeFound == 0) {
                    var itemLink = $(itemDetails).first().find('.toc__item__title');
                    var linkCount = $(itemLink).first().find("a:contains('CME')").length;
                    if (linkCount == 0) {
                        url = $(itemLink).first().find('a').attr('href');
                        return false;   //Found a URL, break the loop
                    }
                }
            }
        });
        return url;
    }
	//--------------------------------------------------------------------------
	function gpt_GetElapsedTime() {
		var endTime = new Date();
		var timeDiff = endTime - gpt_StartTime; //in ms
		// strip the ms
		timeDiff /= 1000;

		// get seconds
		return seconds = Math.round(timeDiff) + " seconds";
	}
	//--------------------------------------------------------------------------
	function gpt_CheckSSC() {
		//Check for ScreenShot Client
		var gpt_SSC = gpt_GetQuerystring('gpt_ssc', '');
		if (gpt_SSC == '1') {
			//it's the ScreenShotClient, disable MOAT IVT Detection
			$('#gpt-Variables').append('<input type="hidden" id="gptm_data">');
			$('#gptm_data').val(0);
			gpt_Target("m_data");
		}
	}
	//--------------------------------------------------------------------------
	function gpt_LocalSS(adTypeId) {
		window.scrollTo(0, 0);
		gpt_Sleep(1000);
		gpt_ScrollToAd(adTypeId);
		gpt_PreTearsheet(adTypeId);
	}
	//--------------------------------------------------------------------------
	function gpt_InitScripts() {
		if (gpt_AdFile == "PU") {
			if (gpt_IsProd) {
				gpt_ContextualURL = gpt_CheckForNULL($('#canonicalurl').val());
				if (gpt_ContextualURL === null) {
					gpt_ContextualURL = window.location.href.split('?')[0].split('#')[0];
				}
				else {
					gpt_ContextualURL = window.location.origin + gpt_ContextualURL;
				}
				var useSSL = 'https:' == document.location.protocol;
				var gs = document.createElement('script');
				gs.async = true;
				gs.type = 'text/javascript';
				gs.src = (useSSL ? 'https:' : 'http:') + '//elsevier.gscontxt.net/multizone/channels.cgi?url=' + encodeURIComponent(gpt_ContextualURL);
				gs.onload = function () {
					gpt_ContextualReady = true;
				};
				document.head.appendChild(gs);
			}
		}
		else {
			//JBS-Cell-Lancet
			var useSSL = 'https:' == document.location.protocol;

			var gads = document.createElement('script');
			gads.async = true;
			gads.type = 'text/javascript';
			gads.src = (useSSL ? 'https:' : 'http:') + '//securepubads.g.doubleclick.net/tag/js/gpt.js';
			document.head.appendChild(gads);

			if (gpt_IsProd && gptContextualEnabled) {
				gpt_ContextualURL = window.location.href.split('?')[0].split('#')[0];
				var gs = document.createElement('script');
				gs.async = true;
				gs.type = 'text/javascript';
				gs.src = (useSSL ? 'https:' : 'http:') + '//elsevier.gscontxt.net/multizone/channels.cgi?url=' + encodeURIComponent(gpt_ContextualURL);
				gs.onload = function () {
					gpt_ContextualReady = true;
				};
				document.head.appendChild(gs);
			}

			var elsevierMoatHeader = document.createElement('link');
			elsevierMoatHeader.rel = 'preload';
			elsevierMoatHeader.as = 'script';
			elsevierMoatHeader.src = (useSSL ? 'https:' : 'http:') + '//z.moatads.com/elsevierheader150204004183/moatheader.js';
			document.head.appendChild(elsevierMoatHeader);

			var elsevierMoatAds = document.createElement('link');
			elsevierMoatAds.rel = 'preconnect';
			elsevierMoatAds.src = (useSSL ? 'https:' : 'http:') + '//mb.moatads.com';
			document.head.appendChild(elsevierMoatAds);

			var moatHeader = document.createElement('script');
			moatHeader.async = true; gads.type = 'text/javascript';
			moatHeader.src = (useSSL ? 'https:' : 'http:') + '//z.moatads.com/elsevierheader150204004183/moatheader.js';
			document.head.appendChild(moatHeader);
		}
	}
	//--------------------------------------------------------------------------
	async function gpt_CheckForMOATInit() {
		//Check every 1/10 of a second until MOAT is Loaded or 1 second timeout
		if (gpt_MOATInitAttempts < gpt_MaxMOATInitAttempts) {
			gpt_MOATInitAttempts += 1;
			if (typeof window.moatPrebidApi === "object") {
				//MOAT LOADED
				gptMoatInit_Debug = '0.0' + gpt_MOATInitAttempts + ' seconds';
				gpt_CheckForMOATReady();
			}
			else {
				setTimeout(() => { gpt_CheckForMOATInit(); }, 100);
			}
		}
		else {
			//TIMEOUT: MOAT Not LOADED
			gptMoatInit_Debug = 'Timeout Expired';
			var googletag = googletag || {};
			googletag.cmd = googletag.cmd || [];
			gpt_CheckForContextualReady();
		}
	}
	//--------------------------------------------------------------------------
	function gpt_CheckForMOATReady() {
		//Check every 1/10 of a second until MOAT is Ready or 1 second timeout
		if (gpt_MOATReadyAttempts < gpt_MaxMOATReadyAttempts) {
			gpt_MOATReadyAttempts += 1;
			if (window.moatPrebidApi.safetyDataAvailable()) {
				//MOAT Ready
				gptMoatReady_Debug = '0.0' + gpt_MOATReadyAttempts + " seconds";
				gpt_CheckForContextualReady();
			}
			else {
				setTimeout(() => { gpt_CheckForMOATReady(); }, 100);
			}
		}
		else {
			//TIMEOUT: MOAT Not Ready
			gptMoatReady_Debug = 'Timeout Expired';
			gpt_CheckForContextualReady();
		}
	}
	//--------------------------------------------------------------------------
	function gpt_CheckForContextualReady() {
		//Check every 1/10 of a second until Contextual is Ready or 2 second timeout
		if (gpt_IsProd && gptContextualEnabled) {
			//Enabled
			if (gpt_ContextualReadyAttempts < gpt_MaxContextualReadyAttempts) {
				gpt_ContextualReadyAttempts += 1;
				if (gpt_ContextualReady) {
					//Contextual Ready
					gptContextualURL_Debug = gpt_ContextualURL;
					gptContextualReady_Debug = '0.0' + gpt_ContextualReadyAttempts + " seconds";
					gptContextualInit_Debug = gs_channels;
					gpt_InitAds();
				}
				else {
					setTimeout(() => { gpt_CheckForContextualReady(); }, 100);
				}
			}
			else {
				//LOG TIMEOUT error
				//REMOVE FOR NOW (SHOULD LOG TO A DIFF LOCATION)
				//TIMEOUT: Contextual Not Ready
				$('#gpt_GrapeShotDebugBtn').remove();
				gptContextualURL_Debug = gpt_ContextualURL;
				gptContextualReady_Debug = 'Timeout Expired';
				gptContextualInit_Debug = "";
				gpt_InitAds();
			}
		}
		else {
			//Not Enabled
			$('#gpt_GrapeShotDebugBtn').remove();
			switch (gpt_AdFile) {
				case 'PU':
				case 'LANCET':
				case 'JBS':
					gptContextualURL_Debug = 'Not enabled outside of Production';
					gptContextualReady_Debug = 'Not enabled outside of Production';
					gptContextualInit_Debug = 'Not enabled outside of Production';
					break;
				case "CELLJOURNAL":
				case "CELLBUCKET":
					gptContextualReady_Debug = 'Not enabled on Cell';
					gptContextualInit_Debug = 'Not enabled on Cell';
					gptContextualURL_Debug = 'Not enabled on Cell';
					break;
			}
			gpt_InitAds();
		}
	}
	//--------------------------------------------------------------------------
	function GPT_GetPageByURL() {
		var host = window.location.host.toLocaleLowerCase()
		var localpath = window.location.pathname.toLocaleLowerCase();

		///Check for Cell.com
		if (host.indexOf('cell.com') >= 0) {
			return null;
		}

		///Check for Cell.com (marlin-stag)
		if (host.indexOf('www-cell-com') >= 0) {
			return null;
		}

		///Check for Lancet
		if (host.indexOf('thelancet') >= 0) {
			return null;
		}

		if (localpath.indexOf('/article/') >= 0) {
			return 'article.fulltext';
		}

		if (localpath.indexOf('/action/dosearch') >= 0) {
			return 'search.searchResults';
		}

		if (localpath.indexOf('/content/') >= 0) {
			return 'periodical.editorial';
		}

		return 'periodical.home';
	}
	//--------------------------------------------------------------------------
	function gpt_CheckForNULL(x, valueifnull) {
		if (typeof valueifnull == "undefined") {
			valueifnull = null;
		}

		if (typeof x == 'undefined') {
			return valueifnull;
		}
		if (x == null) {
			return valueifnull;
		}
		if (x == "null") {
			return valueifnull;
		}
		if ($.trim(x) == '') {
			return valueifnull;
		}

		return x;
	}
	//--------------------------------------------------------------------------
	function gpt_GetAdditionalTargeting() {
		//adds page level targeting of custom variables
		//the id (minus gpt) and the value will be used as a key-value pair
		//eg: <input type="hidden" id="gptissuepii" value="x" class="gpt-Target">
		//eg: target issuepii=x
		$(".gpt-Target").each(function (index) {
			var gptTargetId = $(this).attr('id');
			gptTargetId = gptTargetId.replace('gpt', '').toLowerCase();
			if (gptTargetId !== '') {
				gpt_Target(gptTargetId);
			}
		})
	}
	//--------------------------------------------------------------------------
	function gpt_GetQuerystring(key, default_) {
		//function to read querystring variables
		if (default_ === null) default_ = '';
		key = key.replace(/[\[]/, "\\\[").replace(/[\]]/, "\\\]");
		var regex = new RegExp("[\\?&]" + key + "=([^&#]*)");
		var qs = regex.exec(window.location.href);
		if (qs === null)
			return default_;
		else
			return qs[1];
	}
	//--------------------------------------------------------------------------
	function gpt_GetDFPTestId() {
		//Get any request for a test id from querystring and set custom targeting parameter
		var gptDFPTestId = gpt_GetQuerystring('dfptestid', '');
		if (gptDFPTestId !== '') {
			$('#gpt-Variables').append('<input type="hidden" id="gptDFPTestId">');
			$('#gptDFPTestId').val(gptDFPTestId);
			gpt_Target("DFPTestId");
		}
	}
	//--------------------------------------------------------------------------
	function gpt_Target(targetName) {
		//Set page level target if value is found
		if ($('#gpt' + targetName).length > 0) {
			var targetValue = $('#gpt' + targetName).val();

			if (targetValue !== '') {
				targetValue = targetValue.replace(/[^-a-z0-9]/ig, '');	//Only Allow a-z, 0-9
				googletag.pubads().setTargeting(targetName.toLowerCase(), targetValue.toLowerCase());
				gptCustomTargeting_Debug = gptCustomTargeting_Debug + targetName.toLowerCase() + ' = ' + targetValue.toLowerCase() + '<br>';
			}
		}
	}
	//--------------------------------------------------------------------------
	function gpt_TargetAccessType() {
		var currentAccessType = $('#gptAccessType').val();
		if (typeof currentAccessType == "undefined") {
			currentAccessType = "";
		}
		if (currentAccessType.indexOf('ANON_IP') >= 0) {
			googletag.pubads().setTargeting("loggedin", "yes");
		}
		else {
			gpt_Target("LoggedIn");
		}
	}
	//--------------------------------------------------------------------------
	async function gpt_PreTearsheet(adTypeId) {
		if (typeof adTypeId === "undefined") {
			adTypeId = 0;
		}
		//Remove Elsevier Cookie Policy Notice
		if ($("#onetrust-consent-sdk").length > 0) {
			$("#onetrust-consent-sdk").remove();
		}

        //Mobile Nav
        $('#skip').remove();
        $('skipNavigationLink').remove();

		//JBS Modals
		if ($('.leadinModal').length > 0) {
			$('.leadinModal').remove();
		}
		if ($('.leadinModal').length > 0) {
			$('.leadinModal').remove();
		}
		if ($('.usabilla__overlay').length > 0) {
			$('.usabilla__overlay').remove();
		}

		//Pendo
		if ($('._pendo-step-container').length > 0) {
			$('._pendo-step-container').remove();
		}
		if ($('._pendo-guide-tt_').length > 0) {
			$('._pendo-guide-tt_').remove();
		}
		if ($('._pendo-image').length > 0) {
			$('._pendo-image').remove();
		}
		if ($('._pendo-badge-image').length > 0) {
			$('._pendo-badge-image').remove();
		}

		switch (gpt_AdFile) {
			case "CELLJOURNAL":
			case "CELLBUCKET":
				//Mobile Bug Fix on Cell
				if (parseInt(adTypeId) == 5) {
					$('.widget-horizontalAd').css('position', 'static');
					$('.body-horizontalAd').css('display', 'block');
				}
				break;
			case "JBS":
			case "LANCET":
				objLeaderboard = ".container-leaderboard";
				offset = 53;
				switch (parseInt(adTypeId)) {
					case 3: //MPU
					case 4: //SkyScraper
						//BUGFIX: Show Sidebar ad for JBS
						//skyscraper & mpu
						if ($('.widget-verticalAd').length > 0) {
							$('.widget-verticalAd').css('display', 'block');
						}
						$('#backgroundLayer').css('display', 'none');   //Hide any Prestitial
						break;
					case 6:     //Prestitial
						$('#backgroundLayer').css('display', 'block');
						break;
					default:
						$('#backgroundLayer').css('display', 'none');   //Hide any Prestitial
						break;
				}
				break;
		}
	}
	//--------------------------------------------------------------------------
	function gpt_Sleep(ms) {
		return new Promise(resolve => setTimeout(resolve, ms));
	}
	//--------------------------------------------------------------------------
	async function gpt_ScrollToAd(adTypeId) {
		if (typeof adTypeId === "undefined") {
			adTypeId = 0;
		}
		var adId = "";
		var offset = 0;
		var objLeaderboard = "";
		var gpt_ScrollToAd = true;

		switch (gpt_AdFile) {
			case "PU": //PracticeUpdate
				var hideOtherAds = true;
				$('#modalPrestitial').modal('hide');
				$('.jumbotron-welcome').hide();
				objLeaderboard = ".container-leaderboard";
				offset = 53;
				switch (parseInt(adTypeId)) {
					case 2:
						//LeaderBoard, no Scroll Needed
						gpt_ScrollToAd = false;
						adId = "adSlot1";
						break;
					case 3:
						//MPU
						adId = "adSlot2";
						break;
					case 4:
						//SkyScraper
						adId = "adSlot3";
						$('.most-read').hide();
						break;
					case 5:
						//Mobile, no Scroll Needed
						gpt_ScrollToAd = false;
						adId = "adSlot1";
						break;
					case 6:
						//Interstitial
						gpt_ScrollToAd = false;
						adId = "adSlot15";
						await gpt_Sleep(1000);
						$(objLeaderboard).hide();
						$('#modalPrestitial').unbind();
						$('#modalPrestitial').modal();
						break;
					case 7:
						//Native Ad
						break;
					case 10:
						//Master Companion
						hideOtherAds = false;
						gpt_ScrollToAd = false;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						adId = "adSlot1";
						break;
				}
				if (hideOtherAds) {
					//Hide all other ads
					$('.ad-container').each(function (i, obj) {
						if (this.id !== adId) {
							var tempParent = this.id;
							tempParent = tempParent.replace("adSlot", "adParent");
							$("#" + tempParent).addClass('hide');
						}
					});
				}
				break;
			case "JBS":
				objLeaderboard = ".js__adToHide";
				offset = 80;
				switch (parseInt(adTypeId)) {
					case 2:
						//Leaderboard
						gpt_ScrollToAd = false;
						$(".widget-verticalAd").hide();
						break;
					case 5:
						//Mobile
						//Scroll to Top
						gpt_ScrollToAd = false;
						break;
					case 3: //MPU
					case 4: //SkyScraper
						$(objLeaderboard).hide();
						adId = $('.gpt-ad-defined[data-adsize="sidebar"').attr('id');
						break;
					case 10:
						//MasterCompanion
						gpt_ScrollToAd = false;
						break;
					case 13:	    //Phone - Portrait
					case 14:        //Phone - Landscape
						$(objLeaderboard).hide();
						offset = 60;
						adId = 'gpt-Interscroller';
						gpt_ScrollToAd = true;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						break;
				}
				break;
			case "CELLJOURNAL":
			case "CELLBUCKET":
				objLeaderboard = ".js__adToHide";
				offset = 260;
				switch (parseInt(adTypeId)) {
					case 2:
						//Leaderboard
						gpt_ScrollToAd = false;
						$('.body-verticalAd').hide();
						break;
					case 5:
						//Mobile
						//Scroll to Top
						gpt_ScrollToAd = false;
						break;
					case 3: //MPU
					case 4: //SkyScraper
						$(objLeaderboard).hide();
						adId = $('.gpt-ad-defined[data-adsize="boombox"').attr('id');
						break;
					case 10:
						//MasterCompanion
						gpt_ScrollToAd = false;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						break;
				}
				//Hide the second ad if found
				var tCount = 0;
				$(".widget.literatumAd.none.GPT-ad-placeholder.widget-horizontalAd.widget-compact-all").each(function (index) {
					tCount += 1;
					if (tCount !== 1) {
						$(this).hide();
					}
				});
				break;
			case "LANCET":
				objLeaderboard = ".js__adToHide";
				offset = 80;
				switch (parseInt(adTypeId)) {
					case 2:
						//Leaderboard
						gpt_ScrollToAd = false;
						$('.body-verticalAd').hide();
						break;
					case 5:
						//Mobile
						//Scroll to Top
						gpt_ScrollToAd = false;
						break;
					case 3: //MPU
					case 4: //SkyScraper
						$(objLeaderboard).hide();
						adId = $('.gpt-ad-defined[data-adsize="skyscraper"').attr('id');
						break;
					case 10:
						//MasterCompanion
						gpt_ScrollToAd = false;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						break;
				}
				break;
		}

		//Scroll to Top
		var gpt_container = $("html,body");
		gpt_container.animate({ scrollTop: 0, scrollLeft: 0 });

		if (gpt_ScrollToAd) {
			//If Adexists, scroll to it
			var gpt_scrollTo = $('#' + adId);
			if (gpt_scrollTo.length > 0) {
				//Scroll to the Ad
				var gpt_topPosition = gpt_scrollTo.offset().top - gpt_container.offset().top + gpt_container.scrollTop();
				if (gpt_topPosition > 0) {
					gpt_container.animate({ scrollTop: gpt_topPosition, scrollLeft: 0 });
					if (gpt_topPosition - offset > 0 && offset > 0) {
						gpt_container.animate({ scrollTop: gpt_topPosition - offset, scrollLeft: 0 });
					}
				}
			}
		}
	}
	//--------------------------------------------------------------------------
	async function gpt_CheckForCookiePolicy() {
		var returnVal = false;
		if ($('#onetrust-accept-btn-handler').length > 0) {
			$('#onetrust-accept-btn-handler').click();
			returnVal = true;
		}
		return returnVal;
	}
	//--------------------------------------------------------------------------
	function gpt_InitAds() {
		var gpt_ResponsiveHomepage = false;

		switch (gpt_AdFile) {
			case "JBS":
			case "CELLJOURNAL":
			case "CELLBUCKET":
				//Responsive Check
				if ($('#hdrHomeBrand').length > 0) {
					gpt_ResponsiveHomepage = true;
				}
				break;
		}

		//InterScroller Check
		if (gpt_AdFile === "JBS") {
			var gpt_InterScrollerEnabled = false;
			var gpt_InterScrollerInsertObj;

			if ($("#articleHeader").length > 0) {
				//It's an article Page
				if ($('.article__sections > section').length > 0) {
					gpt_InterScrollerEnabled = true;
					gpt_InterScrollerInsertObj = $(".article__sections > section");
				}
			}
			else {
				if ($('.toc__section').length > 0) {
					//It's a TOC Page
					gpt_InterScrollerEnabled = true;
					gpt_InterScrollerInsertObj = $(".toc__section").first();
				}
			}

			//Add the Interscroller
			if (gpt_InterScrollerEnabled) {
				gpt_InterScrollerInsertObj.first().append($("#gpt-Interscroller").clone());
				$("#gpt-Interscroller").remove();
				$(".gpt-adinterscroller").addClass("gpt-ad");
			}
		}

		googletag.cmd.push(function () {
			var gptBoomBoxSizeMapping;
			var gptFluidMapping;
			var gptHomePageSidebarMapping;
			var gptCellHomePageSidebarMapping;
			var gptInterscrollerMapping;
			var gptLeaderboardSizeMapping;
			var gptMobileSizeMapping;
			var gptPrestitialMapping;
			var gptRectangleShortSizeMapping;
			var gptRectangleSizeMapping;
			var gptSideBarSizeMapping;
			var gptTallSizeMapping;
			var gptWideSizeMapping;
			var gptWideRectangleSizeMapping;

			//***********************************************
			//Start Define Size mapping based on client Viewport
			//***********************************************
			if (gpt_AdFile === 'PU') {
				gptTallSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[300, 600], [160, 600], [120, 600]]).
					addSize([0, 0], []).
					build();

				gptRectangleSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[300, 250]]).
					addSize([0, 0], []).
					build();

				gptRectangleShortSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[300, 250], [320, 50], [300, 50]]).
					addSize([0, 0], [[300, 250], [320, 50], [300, 50]]).
					build();

				gptWideSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[728, 90]]).
					addSize([760, 0], [[728, 90], [320, 50], [300, 50]]).
					addSize([0, 0], [[320, 50], [300, 50]]).
					build();

				gptWideRectangleSizeMapping = googletag.sizeMapping().
					addSize([992, 0], []).
					addSize([760, 0], [[320, 50], [300, 50], [300, 250]]).
					addSize([0, 0], [[320, 50], [300, 50], [300, 250]]).
					build();

				gptMobileSizeMapping = googletag.sizeMapping().
					addSize([0, 0], [[320, 50], [300, 50]]).
					build();

				gptFluidMapping = googletag.sizeMapping().
					addSize([0, 0], ['fluid']).
					build();

				gptPrestitialMapping = googletag.sizeMapping().
					addSize([760, 0], [[1, 1]]).
					build();
			}
			else {
				//JBS-CELL-LANCET
				gptLeaderboardSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[728, 90]]).
					addSize([768, 0], [[728, 90]]).
					addSize([0, 0], [[320, 50], [300, 50]]).
					build();

				gptHomePageSidebarMapping = googletag.sizeMapping().
					addSize([2090, 795], [[300, 600], [300, 250], [160, 600], [120, 600]]).
					addSize([1910, 795], [[160, 600], [120, 600]]).
                    addSize([1740, 795], [[120, 600]]).
					addSize([0, 0], []).
					build();

				gptCellHomePageSidebarMapping = googletag.sizeMapping().
                    addSize([2180, 795], [[300, 600], [160, 600], [120, 600], [300, 250], [336, 280]]).
					addSize([2180, 445], [[336, 280]]).
                    addSize([2090, 795], [[300, 600], [160, 600], [120, 600], [300, 250]]).
                    addSize([2090, 445], [[300, 250]]).
					addSize([1830, 795], [[160, 600], [120, 600]]).
                    addSize([1740, 795], [[120, 600]]).
					addSize([0, 0], []).
					build();

				gptSideBarSizeMapping = googletag.sizeMapping().
					addSize([1140, 795], [[300, 600], [160, 600], [120, 600], [300, 250]]).
					addSize([1140, 445], [[300, 250]]).
					addSize([0, 0], []). 				
					build();

				gptBoomBoxSizeMapping = googletag.sizeMapping().
					addSize([768, 0], [[300, 250], [336, 280]]).
					addSize([0, 0], []). 						
					build();

				gptFluidMapping = googletag.sizeMapping().
					addSize([0, 0], ['fluid']).
					build();

				gptPrestitialMapping = googletag.sizeMapping().
					addSize([600, 700], [[1, 1]]).
					addSize([0, 0], []).
					build();

				gptInterscrollerMapping = googletag.sizeMapping().
					addSize([1140, 445], []).
					addSize([1024, 768], [[1024, 768]]).
					addSize([768, 1024], [[768, 1024]]).
					addSize([480, 320], [[480, 320]]).
					addSize([320, 480], [[320, 480]]).
					build();
			}
			//***********************************************
			//End Define Size mapping based on client Viewport
			//***********************************************

			//***********************************************
			//Start detection for ad slots on page
			//***********************************************
			var gptMasterPage = gpt_CheckForNULL($('#gptPage').val());
			var gptMasterSite;

			if (gpt_AdFile === 'PU') {
				gptMasterSite = $('#gptSite').val();
			}
			else {
				//JBS-CELL-LANCET
				gptMasterSite = "dev";

				if (gpt_IsProd) {
					gptMasterSite = gpt_CheckForNULL($('#gptSite').val());

					//If there is no master site, check for a defaultsite
					if (gptMasterSite == null) {
						gptMasterSite = gpt_CheckForNULL($('#gptDefaultSite').val());
					}
				}
				else {
					//Dev ENV
					//If the page has no gptSite Value, assume no value which will disable ads
					if (gpt_CheckForNULL($('#gptSite').val()) === null) {
						gptMasterSite = null;
					}
				}

				//If there is no master Page, check for a Page using the URL
				if (gptMasterPage === null) {
					gptMasterPage = GPT_GetPageByURL();
				}
			}

			$(".gpt-ad").each(function (index) {
				var gptPage = '';
				var adSizeMapping = '';
				var gptPOS;
				var gptTC;
				var gptNative;
				var adId;
				var gptSite;
				var tempSlot = null;


                if (gpt_IsProd) {
					if (gpt_AdFile !== "CELLJOURNAL") {
						//Check if the ad slot has it's own site defined
						if (typeof $(this).data('site') === 'undefined') {
							gptSite = gptMasterSite;
						}
						else {
							gptSite = gpt_CheckForNULL($(this).data('site'));
						}
					}
					else {
						gptSite = gptMasterSite;
					}
                }
                else {
					gptSite = gptMasterSite;
                }



				if (gpt_AdFile === 'PU') {
					adId = $(this).attr('id');
				}
				else {
					//JBS-CELL-LANCET
					adId = 'gpt-ad-' + index + 1;
				}

				if (gpt_CheckForNULL(gptSite) !== null) {
					gptSite_Debug = gptSite;

					gptPOS = parseInt($(this).data('pos')) || -1;
					if (gptPOS === -1) {
						gptPOS = index + 1;
					}

					//Check if the ad slot has it's own page defined
					gptPage = $(this).data('page');
					if (gpt_CheckForNULL(gptPage) == null) {
						//Use the Master Page
						gptPage = gptMasterPage;
					}

					//If the page is not null, preceed it with a slash
					if (gptPage !== null) {
						gptPage = '/' + gptPage;
					}
					else {
						gptPage = '';
					}
					gptPage_Debug = gptPage;


					gptTC = $(this).data('tc');
					if (typeof gptTC === "undefined") {
						gptTC = '';
					}

					gptNative = $(this).data('native');
					if (typeof gptNative === "undefined") {
						gptNative = '';
					}

					//Determine sizemapping based on data-adsize
					if (gpt_AdFile === 'PU') {
						switch ($(this).data('adsize')) {
							case 'tall':
								adSizeMapping = gptTallSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600]], adId);
								break;
							case 'rectangle':
								adSizeMapping = gptRectangleSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250]], adId);
								break;
							case 'rectangleshort':
								adSizeMapping = gptRectangleShortSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250], [320, 50], [300, 50]], adId);
								break;
							case 'wide':
								adSizeMapping = gptWideSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[728, 90], [320, 50], [300, 50]], adId);
								break;
							case 'widerectangle':
								adSizeMapping = gptWideRectangleSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[320, 50], [300, 50], [300, 250]], adId);
								break;
							case 'mobile':
								adSizeMapping = gptMobileSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[320, 50], [300, 50]], adId);
								break;
							case 'fluid':
								adSizeMapping = gptFluidMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, 'fluid', adId);
								break;
							case 'prestitial':
								adSizeMapping = gptPrestitialMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[1, 1]], adId);
								break;
						}
					}
					else {
						//JBS-CELL-LANCET
						switch ($(this).data('adsize')) {
							case 'cell-leaderboard':
							case 'leaderboard':
								adSizeMapping = gptLeaderboardSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[728, 90], [320, 50], [300, 50]], adId);
								break;
							case 'bottom':
								if (gpt_ResponsiveHomepage) {
									adSizeMapping = gptHomePageSidebarMapping;
									tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600], [300, 250]], adId);
								}
								else {
									adSizeMapping = gptBoomBoxSizeMapping;
									tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250]], adId);
								}
								break;
							case 'boombox':
								if (gpt_AdFile === "CELLJOURNAL" || gpt_AdFile == "CELLBUCKET") {
									if (gpt_ResponsiveHomepage) {
										adSizeMapping = gptCellHomePageSidebarMapping;
										tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600], [300, 250], [336, 280]], adId);
									}
									else {
										adSizeMapping = gptBoomBoxSizeMapping;
										tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250], [336, 280]], adId);
									}
								}
								else {
									adSizeMapping = gptBoomBoxSizeMapping;
									tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250]], adId);
								}
								break;
							case 'interscroller':
								adSizeMapping = gptInterscrollerMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[320, 480], [480, 320], [768, 1024], [1024, 768]], adId);
								break;
							case 'sidebar':
							case 'skyscraper':
								adSizeMapping = gptSideBarSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600], [300, 250]], adId);
								break;
							case 'native':
							case 'fluid':
								adSizeMapping = gptFluidMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, 'fluid', adId);
								break;
							case 'prestitial':
								adSizeMapping = gptPrestitialMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, 'fluid', adId);
								break;
						}
					}
				}

				//Ignore any invalid adsizes that could be on the page
				if (adSizeMapping !== '') {
					$(this).addClass('gpt-ad-defined');
					if (gpt_AdFile !== 'PU') {
						$(this).attr('id', adId);
					}
					tempSlot.defineSizeMapping(adSizeMapping).addService(googletag.pubads()).setTargeting("pos", gptPOS);

					if (gptTC !== '') {
						tempSlot.setTargeting("tc", gptTC);
					}
					if (gptNative !== '') {
						tempSlot.setTargeting("native", gptNative);
					}
					gptEvent_Debug = gptEvent_Debug + $(this).data('adsize') + ' Requested<br>';
				}
				else {
					gptInvalidAdSize_Debug = gptInvalidAdSize_Debug + "Invalid Size Found, Id: " + $(this).prop('id') + ', adSize: ' + $(this).data('adsize') + '<br>';
					$(this).remove();
				}
			});
			if (gpt_CheckForNULL(gptInvalidAdSize_Debug) === null) {
				gptInvalidAdSize_Debug = "None Detected";
			}
			//***********************************************
			//End detection for ad slots on page
			//***********************************************

			//***********************************************
			//Start page level targeting
			//***********************************************
			if (gpt_AdFile !== 'PU') {
				//Logged in or IP Access
				gpt_TargetAccessType();

				//Check for Screenshot Client
				gpt_CheckSSC();
			}

			//Check for DFP Test
			gpt_GetDFPTestId();

			//Check for Additional Targeting
			gpt_GetAdditionalTargeting();

			if (gpt_IsProd && gptContextualEnabled) {
				var gpt_ContextualRetry = false;
				//Check for Contextual Targeting
				if (gs_channels !== "DEFAULT") {
					var gpt_ContextualCategories = Array.from(gs_channels);
					for (var i = 0; i < gpt_ContextualCategories.length; i++) {
						if (gpt_ContextualCategories[i] == 'gx_retry') {
							gpt_ContextualRetry = true;
						}
						if (!gpt_ContextualCategories[i].startsWith("custom_")) {
							//Remove any tag that is not custom
							gpt_ContextualCategories.splice(i, 1);
							i--;
						}
						else {
							//Strip the 'custom_' from the tag
							gpt_ContextualCategories[i] = gpt_ContextualCategories[i].replace("custom_", "")
						}
					}
					if (gpt_ContextualCategories.length > 0) {
						googletag.pubads().setTargeting("gs_cat", gpt_ContextualCategories);
					}
					else {
						gpt_ContextualCategories = 'none';
					}
					if (gpt_ContextualRetry) {
						gpt_ContextualCategories = "*Indexing*";
					}
					gptContextualResult_Debug = gpt_ContextualCategories;
				}
				else {
					gptContextualResult_Debug = 'No segments found';
				}
			}
			else {
				//Not Prod || Contextual not enabled
				gptContextualResult_Debug = 'Not enabled';
				
				if (!gpt_IsProd) {
                    var gpt_DevSite = gpt_CheckForNULL($('#gptSite').val(), "dev");
                    $('#gpt-Variables').append('<input type="hidden" id="gptEnvironment" value="' + gpt_DevSite + '">');
                    gpt_Target("Environment");
                }

			}
			//***********************************************
			//End page level targeting
			//***********************************************

			//Enable google dfp services
			if (parseInt($('#gptLazyLoading').val()) === 1) {
				googletag.pubads().enableLazyLoad({
					fetchMarginPercent: 200,
					renderMarginPercent: 0,
					mobileScaling: 2.0
				});
			}
			else {
				googletag.pubads().enableSingleRequest();
			}

			//Add Listener for RenderEnded
			googletag.pubads().addEventListener('slotRenderEnded', function (event) {
				if (!event.isEmpty) {
					if (typeof LocalAdLoaded === "function") {
						LocalAdLoaded(event);
					}
					if (gpt_ResponsiveHomepage) {
						if (typeof positionAd === "function") {
							positionAd();
						}
					}
					if (event.size[0] == "480" || event.size[1] == "480") {
						$('.gpt-InScrollBox').css('display', 'block');
						gptEvent_Debug = gptEvent_Debug + 'Interscroller ' + event.size + ' rendered: ' + gpt_GetElapsedTime() + '<br>';
					}
					else {
						gptEvent_Debug = gptEvent_Debug + event.size + ' rendered: ' + gpt_GetElapsedTime() + '<br>';
					}

				}
			});

			if (gpt_AdFile !== 'PU') {
				googletag.pubads().collapseEmptyDivs();
			}
			googletag.enableServices();

			//Populate Defined Ad Slots
			$(".gpt-ad-defined").each(function (index) {
				var adId = $(this).attr('id');
				googletag.cmd.push(function () { googletag.display(adId); });
			});

			gptStatus_Debug = 'GPT Complete';
		});
	}
	//--------------------------------------------------------------------------
	function gpt_DisplayDebug() {
		//Clone the template and append to Body
		$(document.body).append($('#gpt-DebugDiv').clone());
		$('#gpt-DebugDiv').remove();

		//Populate the Values
		gpt_LoadDebugValues();

		//Display and scroll to debug section
		$('.gpt-DebugDiv').css('display', 'block');
		window.scrollTo(0, document.body.scrollHeight);
	}
	//--------------------------------------------------------------------------
	function gpt_LoadDebugValues() {
		$('.gpt-DebugBtn').blur();
		$('.gptVersion_Debug').text(gpt_Version);
		$('.gptAdFile_Debug').html(gpt_AdFile);
		$('.gptEvent_Debug').html(gptEvent_Debug);
		$('.gptCustomTargeting_Debug').html(gptCustomTargeting_Debug);
		$('.gptSite_Debug').text(gptSite_Debug);
		$('.gptContextualResult_Debug').text(gptContextualResult_Debug);
		$('.gptContextualInit_Debug').text(gptContextualInit_Debug);
		$('.gptContextualURL_Debug').text(gptContextualURL_Debug);
		$('.gptContextualReady_Debug').text(gptContextualReady_Debug);
		$('.gptMoatReady_Debug').text(gptMoatReady_Debug);
		$('.gptMoatInit_Debug').text(gptMoatInit_Debug);
		$('.gptTestENV_Debug').text(gptTestENV_Debug);
		$('.gptDisableAds_Debug').text(gptDisableAds_Debug);
		$('.gptStatus_Debug').text(gptStatus_Debug);
		$('.gptPage_Debug').text(gptPage_Debug);
		$('.gptInvalidAdSize_Debug').html(gptInvalidAdSize_Debug);
	}
	//--------------------------------------------------------------------------
	function gpt_GrapeShotDebug() {
		$('.gpt-DebugBtn').blur();
		window.open("http://admin-elsevier.grapeshot.co.uk/custom/bookmarklet.cgi?blocking=1&vn=2&url=" + encodeURIComponent(document.location), "GRAPESHOT_POPUP", "width=650,height=600,scrollbars=yes,screenX=40,screenY=50,menubar=no,toolbar=no,resizable=yes", "Custom").focus();
	}
	//--------------------------------------------------------------------------
	function gpt_ShowConsole() {
		$('.gpt-DebugBtn').blur();
		googletag.openConsole();
	}
	//--------------------------------------------------------------------------
</script>
        </div>

            
                </div>
            

        

    

        


    

            

            

            <div data-widget-id="912135d6-7135-4d45-b457-b503cacc3c8b" data-widget-name="ux3-layout-widget"><header class="header base fixed">
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none adplaceholder adplaceholder--header js__adToHide widget-horizontalAd" id="9d6062d7-fad6-4e64-81cc-7dd25d99b23d">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-horizontalAd "><div class="horizontalAdContainer">
    <div class="horizontalAdLabelContainer">
        <div class="horizontalAdLabel">
                <img src="/templates/jsp/_style2/_marlin/images/bg_adHoriz.gif" alt="Advertisement">
        </div>
    </div>
    <div class="horizontalAd">
        





        <div class="pb-ad">
            <!-- Beginning Async AdSlot 1 for Ad unit els.cellpress/cell  ### size: [[728,90]] -->
<div class="gpt-ad" data-adsize="cell-leaderboard" data-page="" data-site="cell" data-pos="1"></div>
<!-- End AdSlot 1 -->
        </div>
    

    </div>
</div></div>
                </div>
                </div>

        










        <div data-widget-def="ux3-layout-widget" data-widget-id="3a83e2a6-2e7e-4931-892c-4ef43b6e3312" class="header__wrapper clearfix" data-db-parent-of="quickSearch"><div id="brandingRow"><a href="/" data-aa-name="science that inspires" data-aa-region="cp global header" title="Visit Cell.com">Science that inspires</a><p id="brandingRowRight"><a href="/" data-aa-name="cell press endorsement" data-aa-region="cp global header" title="Visit Cell.com">A Cell Press journal</a></p></div>
        
    

        


    

            

            

            <div data-widget-id="3a83e2a6-2e7e-4931-892c-4ef43b6e3312" data-widget-name="ux3-layout-widget"><div class="col-xs-6 header__left-side">
    

        


    

            

            
                <a href="/home" id="cpIconLnk" data-aa-name="cell press logo" data-aa-region="cp global header" title="Cell Press" target="_blank"><svg id="cp_logo" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 100 98.1" style="enable-background:new 0 0 100 98.1;" xml:space="preserve" height="100" width="98.1"><path style="fill:#FFFFFF;" d="M69.09,0.09c-4.01-0.6-7.83,1.94-8.78,5.89l-3.14,13.11c-0.76,3.19-3.44,5.56-6.7,5.93l-18.6,2.13          C11.83,29.31-2.53,47.66,0.37,67.78c2.3,15.96,15.18,27.77,30.54,30.08c4.01,0.6,7.83-1.94,8.78-5.89l3.14-13.11 c0.76-3.19,3.44-5.56,6.69-5.93l18.6-2.13c20.04-2.17,34.41-20.52,31.5-40.65C97.32,14.21,84.45,2.39,69.09,0.09z M65.06,47.2  l-11.75,1.4c-1.98,0.21-3.62,1.65-4.08,3.59l-3.43,14.3c-1.01,4.22-4.5,7.39-8.8,8c-6.66,0.72-12.63-4.2-13.15-10.92                c-0.51-6.5,4.6-12.11,11.08-12.81l11.75-1.39c1.98-0.22,3.62-1.65,4.08-3.59l3.43-14.3c1.01-4.22,4.5-7.39,8.8-8          c6.66-0.72,12.63,4.19,13.15,10.92C76.66,40.89,71.55,46.5,65.06,47.2z"></path></svg></a><div class="journal-logos">
            

            <span data-widget-id="35d8d734-3aa0-4ec8-9e2d-14babe803594" data-widget-name="ux3-general-image"><a href="/cell/home" title="Cell"><img alt="Cell" src="/pb-assets/ux3/logos/cell/Cell_white-1532516611593.svg"></a></span>

            
                </div>
            

        

    

        


    

            

            
                <div class="journal-logos">
            

            <span data-widget-id="3ffcbf1f-79fd-4ba3-85b5-bef10a63475f" data-widget-name="ux3-general-image"><a href="https://www.cell.com/open-access?utm_source=oa-label&amp;__hstc=109691226.e4e812b53ff324eccbdbdb2e4fb9a6ab.1644503737405.1644524234622.1644587109507.4&amp;__hssc=109691226.7.1644587109507&amp;__hsfp=3253153019" title="This journal offers authors two options (open access or subscription) to publish research"><img alt="This journal offers authors two options (open access or subscription) to publish research" src="/pb-assets/ux3/logos/cell/Supports_open_access_border-1645554937897.svg"></a></span>

            
                </div>
            

        
</div><div class="col-xs-6">
    

        


    

            

            

            <div data-widget-id="2555f6db-b333-4edb-8b15-ef9213f1f0a5" data-widget-name="ux3-layout-widget"><div class="header__nav row">
    

        


    

            

            

            <ul data-widget-id="52432d24-0331-4c9d-85f3-2033edc7208b" data-widget-name="UX3ProfileLinksWidget" class="rlist rlist--inline profile-links"><li class="profile-links__submit"><a data-adobe-name="Publish" class="profile-links__submit__btn" title="Submit" href="https://www.editorialmanager.com/cell" target="_self" data-aa-conversiondriver="jbs:global-header:submitpage" data-profile-link="true">Submit</a></li><li class="profile-links__login-dropdown"><div class="profile-links__user-list__wrapper"><button data-db-target-for="loginmenu-875665" aria-expanded="false" aria-controls="login-links-875665" title="Log in links dropdown" class="profile-links__username"><span>Log in</span></button><div data-db-target-of="loginmenu-875665" id="login-links--875665" class="profile-links__user-list__holder dropBlock__holder"><ul class="rlist profile-links__list"><li class="profile-links__login"><a data-adobe-name="Log In" class="profile-links__login__btn" title="Log in to Cell.com" href="https://www.cell.com/action/idLogin?type=login&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2901337-4" target="_self" data-profile-link="true">Log in</a></li><li class="profile-links__institution"><a data-adobe-name="Get institutional access" class="profile-links__institution__btn" title="Institutional access" href="https://www.cell.com/action/idLogin?type=institution&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2901337-4" target="_self" data-profile-link="true">Get Institutional Access</a></li></ul></div></div></li><li class="profile-links__register"><a data-adobe-name="Register" class="profile-links__register__btn" title="Register a username and password for free" href="https://www.cell.com/action/idLogin?type=registration&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2901337-4" target="_self" data-profile-link="true">Register</a></li><li class="hidden-lg"> <div class="profile-links__controls"> <button aria-expanded="false" aria-controls="links-dialog-875665" data-opens="#links-dialog-875665" class="profile-links__btn--open"><span class="sr-only">open links dialog</span><i aria-hidden="true" class="icon-gizmo-person"></i></button></div><dialog id="links-dialog-875665" aria-modal="true"><button data-closes="#links-dialog-875665" class="profile-links__btn--close"><span class="sr-only">close links dialog</span><i aria-hidden="true" class="icon-close-solid"></i></button><ul>   <li class="profile-links__submit"><a data-adobe-name="Publish" class="profile-links__submit__btn" title="Submit" href="https://www.editorialmanager.com/cell" target="_self" data-aa-conversiondriver="jbs:global-header:submitpage" data-profile-link="true">Submit</a></li><li class="profile-links__login"><a data-adobe-name="Log In" class="profile-links__login__btn" title="Log in to Cell.com" href="https://www.cell.com/action/idLogin?type=login&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2901337-4" target="_self" data-profile-link="true">Log in</a></li><li class="profile-links__institution"><a data-adobe-name="Get institutional access" class="profile-links__institution__btn" title="Institutional access" href="https://www.cell.com/action/idLogin?type=institution&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2901337-4" target="_self" data-profile-link="true">Get Institutional Access</a></li><li class="profile-links__register"><a data-adobe-name="Register" class="profile-links__register__btn" title="Register a username and password for free" href="https://www.cell.com/action/idLogin?type=registration&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2901337-4" target="_self" data-profile-link="true">Register</a></li></ul></dialog></li></ul>

            

        
</div></div>

            

        
</div></div>

            

        

        </div>
    

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget csasInstitutionBanner none  widget-none" id="51b7fa33-2137-4f58-9e98-9746c4400513" style="display: none;">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none "><div class="ui-state-highlight ui-corner-all csas-institution-banner hidden" data-doi="10.1016/j.cell.2021.11.022">
    <span class="ui-icon ui-icon-close banner-close"></span>
    <p>
        Access provided by <span class="accountName"></span>
    </p>
</div>

<style>
    .csas-institution-banner {
        padding: 10px 0 0 10px;
        margin: 5px;
    }

    .csas-institution-banner.ui-state-highlight .ui-icon {
        background-image: url("/templates/jsp/css/jquery-ui-1.10.2/base/images/ui-icons_454545_256x240.png");
        margin-right: 5px;
        margin-top: -4px;
    }

    .csas-institution-banner .banner-close {
        float: right;
    }
</style>

<script>

    function resetBanner(doi) {
        $.get('/action/getAccount?doi=' + doi, function(data) {
            if (data) {
                $(".csas-institution-banner p .accountName").html(data);
                $(".csasInstitutionBanner").fadeIn();
                $(".csas-institution-banner").fadeIn();
            }
        });
    }

    var flag = 0;
    $(document).ready(function() {
        if(!$(".csas-institution-banner").is(":visible")){
            $(".csasInstitutionBanner").hide();
        }

        $(".banner-close").click(function() {
            $.get("/action/dismissBanner", function(data) {
                $(".csasInstitutionBanner").fadeOut(function () {
                    if(typeof UX !== 'undefined') {
                        UX.pageBody.init();
                    }
                });
            });
        });

        $(".articleContent .tabs li[role='tab'].showFullText").click(function() {
           flag = 1;
        });

        $(document).on("reloadTabsContent", function() {
            var isSelected = $(".articleContent .tabs li[role='tab'].showFullText").hasClass("active");
            if (isSelected) {
                var isHidden =  $(".csas-institution-banner").is(".hidden");
                if (!isHidden) {
                    resetBanner($(".csas-institution-banner").data("doi"));
                    console.log("fadeIn");
                }
            } else {
                $(".csas-institution-banner").fadeOut();
                console.log("fadeOut");
            }
            flag = 0;
        });

        $(document).on("click", ".articleContent .tabs li[role='tab']", function() {

        });
    });
</script></div>
                </div>
                </div>

        
</header></div>

            

        










        <div data-widget-def="graphQueryWidget" data-widget-id="2a125dae-6dbc-4e76-a75e-3e563f1d3a1d" class="ux3">
        
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget graphQueryWidget none top-padding-handler noMargin axel-mega-menu widget-none widget-compact-all fonts-loaded" id="2a125dae-6dbc-4e76-a75e-3e563f1d3a1d" style="padding-top: 115.594px;">
                
                
                    
                

                <div class="wrapped " id="CP TOP LEVEL NAV MENU">
                    
                    <div class="widget-body body body-none  body-compact-all"><nav id="mega-menu-nav" role="navigation" class="mega-menu-nav">
  <!--.skip//a(href='#searchText' aria-label='Skip menu') Skip menu
  -->
  <div class="mega-menu-nav__wrapper">
    <button class="mega-menu__toggle"><span class="sr-only">Main menu</span><i aria-hidden="true" class="icon-gizmo-hamburger"></i></button>
    <ul role="menubar" class="mega-menu rlist">
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-1" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-1" data-item-link="main-menu-link-1" data-index="0" class="mega-menu__item-link menu-link"><span>This journal</span></a>
        <div id="main-menu-panel-1" aria-labelledby="main-menu-link-1" data-item-wrapper="main-menu-link-1" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <div aria-label="Menu dropzone section" tabindex="-1" class="listDropZone">
    

        


    

            

            

            <div class="cover-image"><div class="cover-image__thumb"><a href="/issue/S0092-8674(24)X0012-4" class="menu-link"><img src="/cms/asset/atypon:cms:attachment:img:d5e6:rev:1748270185670-389:pii:S0092867424X00124/cover.tif.jpg" alt="Cover Image - Cell, Volume 188, Issue 11"></a></div><div class="cover-image__meta"></div></div>

            

        
</div>
            </li>
            <li style="grid-column: 2/3" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-1-2" class="sub-menu__heading">Read
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/newarticles" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Online now</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/current" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Current issue</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/archive" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Archive</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/action/showPreferences?menuTab=Alerts" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Publication alerts</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/sneakpeek" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Sneak Peek - preprints</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 3/5" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-1-3" class="sub-menu__heading">About
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-2">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/advisory-board" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Advisory board</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/aims" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Aims and scope</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/contact" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Contact</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/editors-and-staff" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Editors and staff</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/editorial-ethics" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Ethics</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/information-for-authors/permissions" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Permissions</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/reviewers" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>Reviewer instructions</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-2" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-2" data-item-link="main-menu-link-2" data-index="1" class="mega-menu__item-link menu-link"><span>Journals</span></a>
        <div id="main-menu-panel-2" aria-labelledby="main-menu-link-2" data-item-wrapper="main-menu-link-2" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/5" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-2-1" class="sub-menu__heading">Life &amp; medical sciences
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-4">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/home" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cancer-cell/home" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Cancer Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-chemical-biology" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Chemical Biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-genomics/home" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Genomics</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-host-microbe/home" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Host &amp; Microbe</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-metabolism/home" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Metabolism</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports/home" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-medicine/home" data-index="7" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Medicine</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-stem-cell/home" data-index="8" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Stem Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-systems/home" data-index="9" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Systems</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/current-biology/home" data-index="10" role="menuitem" class="sub-menu__item-link menu-link"><span>Current Biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/developmental-cell/home" data-index="11" role="menuitem" class="sub-menu__item-link menu-link"><span>Developmental Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/immunity/home" data-index="12" role="menuitem" class="sub-menu__item-link menu-link"><span>Immunity</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/med/home" data-index="13" role="menuitem" class="sub-menu__item-link menu-link"><span>Med</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-cell/home" data-index="14" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/neuron/home" data-index="15" role="menuitem" class="sub-menu__item-link menu-link"><span>Neuron</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/structure/home" data-index="16" role="menuitem" class="sub-menu__item-link menu-link"><span>Structure</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/ajhg/home" data-index="17" role="menuitem" class="sub-menu__item-link menu-link"><span>American Journal of Human Genetics (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/biophysj/home" data-index="18" role="menuitem" class="sub-menu__item-link menu-link"><span>Biophysical Journal (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/biophysreports/home" data-index="19" role="menuitem" class="sub-menu__item-link menu-link"><span>Biophysical Reports (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/hgg-advances/home" data-index="20" role="menuitem" class="sub-menu__item-link menu-link"><span>Human Genetics and Genomics Advances (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-plant/home" data-index="21" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Plant (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/molecular-therapy/home" data-index="22" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/methods/home" data-index="23" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy Methods &amp; Clinical Development (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/nucleic-acids/home" data-index="24" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy Nucleic Acids (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/oncology/home" data-index="25" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy Oncology (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/plant-communications/home" data-index="26" role="menuitem" class="sub-menu__item-link menu-link"><span>Plant Communications (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/stem-cell-reports/home" data-index="27" role="menuitem" class="sub-menu__item-link menu-link"><span>Stem Cell Reports (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/biochemical-sciences/home" data-index="28" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Biochemical Sciences</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/cancer/home" data-index="29" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Cancer</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/cell-biology/home" data-index="30" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Cell Biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/ecology-evolution/home" data-index="31" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Ecology &amp; Evolution</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/endocrinology-metabolism/home" data-index="32" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Endocrinology &amp; Metabolism</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/genetics/home" data-index="33" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Genetics</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/immunology/home" data-index="34" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Immunology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/microbiology/home" data-index="35" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Microbiology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/molecular-medicine/home" data-index="36" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Molecular Medicine</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/neurosciences/home" data-index="37" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Neurosciences</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/parasitology/home" data-index="38" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Parasitology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/pharmacological-sciences/home" data-index="39" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Pharmacological Sciences</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/plant-science/home" data-index="40" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Plant Science</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 5/6" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-2-2" class="sub-menu__heading">Physical sciences &amp; engineering
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-biomaterials/home" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Biomaterials</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-physical-science/home" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Physical Science</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/chem/home" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Chem</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/chem-catalysis/home" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Chem Catalysis</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/device/home" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Device</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/joule/home" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Joule</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/matter/home" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>Matter</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/newton/home" data-index="7" role="menuitem" class="sub-menu__item-link menu-link"><span>Newton</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/chemistry/home" data-index="8" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Chemistry</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 6/7" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-2-3" class="sub-menu__heading">Multidisciplinary
              </h4>
              <ul role="menubar" data-index="2" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-methods/home" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Methods</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-sustainability/home" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Sustainability</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/heliyon/home" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Heliyon</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/iscience/home" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>iScience</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/one-earth/home" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>One Earth</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/patterns/home" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Patterns</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/star-protocols/home" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>STAR Protocols</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/nexus/home" data-index="7" role="menuitem" class="sub-menu__item-link menu-link"><span>Nexus (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/the-innovation/home" data-index="8" role="menuitem" class="sub-menu__item-link menu-link"><span>The Innovation (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/biotechnology/home" data-index="9" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Biotechnology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/cognitive-sciences/home" data-index="10" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Cognitive Sciences</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-3" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-3" data-item-link="main-menu-link-3" data-index="2" class="mega-menu__item-link menu-link"><span>Publish</span></a>
        <div id="main-menu-panel-3" aria-labelledby="main-menu-link-3" data-item-wrapper="main-menu-link-3" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-3-1" class="sub-menu__heading">Authors
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="https://www.editorialmanager.com/cell?__hstc=25856994.7201670a4397fa03148684920d4a8d38.1672762013831.1694716659709.1694718909620.146&amp;__hssc=25856994.15.1694718909620&amp;__hsfp=1366844671" data-index="0" target="_blank" role="menuitem" class="sub-menu__item-link menu-link"><span>Submit article
                      <div class="external-icon"></div><span class="sr-only">External Link</span></span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/information-for-authors" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Information for authors</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/open-access" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Open access</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/multi-journal-submission" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Multi-Journal Submission</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/information-for-authors/star-authors-guide" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>STAR Methods</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 2/3" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-3-2" class="sub-menu__heading">Reviewers
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/reviewers" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Information for reviewers</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-4" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-4" data-item-link="main-menu-link-4" data-index="3" class="mega-menu__item-link menu-link"><span>News &amp; events</span></a>
        <div id="main-menu-panel-4" aria-labelledby="main-menu-link-4" data-item-wrapper="main-menu-link-4" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/newsroom" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Newsroom</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/events" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Events</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/symposia" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Symposia</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/webinars" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Webinars</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/consortia" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Consortia Hub</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-5" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-5" data-item-link="main-menu-link-5" data-index="4" class="mega-menu__item-link menu-link"><span>About Cell Press</span></a>
        <div id="main-menu-panel-5" aria-labelledby="main-menu-link-5" data-item-wrapper="main-menu-link-5" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-5-1" class="sub-menu__heading">Cell Press
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/about" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>About Cell Press</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/contact" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Contact us</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="https://service.elsevier.com/app/overview/cell-press/" data-index="2" target="_blank" role="menuitem" class="sub-menu__item-link menu-link"><span>Help &amp; support
                      <div class="external-icon"></div><span class="sr-only">External Link</span></span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/open-access" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Open access at Cell Press</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/diversity" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Inclusion and diversity</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-press-50" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>50th anniversary</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 2/3" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-5-2" class="sub-menu__heading">Hubs
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/biotechnology-bioengineering" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Biotechnology and bioengineering</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-press-chemistry" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Chemistry</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/clinical-translational" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Clinical and translational</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/covid-19" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>COVID-19</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/fundamental-biology" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Fundamental biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/sustainability" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Sustainability</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 3/4" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-5-3" class="sub-menu__heading">Careers
              </h4>
              <ul role="menubar" data-index="2" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="https://relx.wd3.myworkdayjobs.com/en-US/ElsevierJobs" data-index="0" target="_blank" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Press careers
                      <div class="external-icon"></div><span class="sr-only">External Link</span></span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
    </ul>
    <div class="mega-menu-nav__quick-search">
    

        


    

            

            

            <div data-widget-id="a1f05748-166f-41e0-b01a-ca8ac1cc95d4" data-widget-name="UX3QuickSearchWidget" class="quick-search"><div class="dropBlock"><form action="https://www.cell.com/action/doSearch?journalCode=cell" name="quickSearchBar" method="get" title="Quick Search"><input id="type" type="hidden" value="quicksearch" name="type"><div class="quick-search__toggle autoCompleteWrapper"><input id="searchText" type="text" name="text1" placeholder="Search for..." enterkeyhint="search" autocomplete="off" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" data-group-titles-conf="3" data-publication-items-conf="3" aria-label="Enter a search term" class="searchText searchText-autocomplete ui-autocomplete-input"><button type="submit" title="Search" aria-label="Submit the search form" class="btn quick-search__button icon-search"><span>Go search</span></button><a href="https://www.cell.com/search/advanced?SeriesKey=cell&amp;ISSN=0092-8674&amp;journalCode=cell" title="Advanced search" class="quick-search__advanced-link">Advanced search</a><ul id="ui-id-1" tabindex="0" class="ui-menu ui-widget ui-widget-content ui-autocomplete ui-front" unselectable="on" style="display: none;"></ul></div><div class="quick-search__box"><div class="quick-search__box-inner"><div class="quick-search__box-wrapper"><select name="field1" aria-label="Select a search area"><option value="AllField">All content</option><option value="Title">Article title</option><option value="Contrib">Authors</option><option value="Keyword">Keywords</option><option value="Abstract">Abstract</option><option value="AbstractTitleKeywordFilterField">Article title, abstract, keywords</option></select><input type="hidden" name="journalCode" value="cell"><div class="quick-search__scope-options"><span>Search within:</span><label title="This journal" class="radio--primary"><input type="radio" name="SeriesKey" id="searchScopeSeries-a1f05748-166f-41e0-b01a-ca8ac1cc95d4" title="This journal" value="cell" checked="checked" aria-label="Search within current journal"><span>This journal</span></label><label title="Full site" class="radio--primary"><input type="radio" name="SeriesKey" id="searchScopeFullSite-a1f05748-166f-41e0-b01a-ca8ac1cc95d4" value="" title="Full site" aria-label="Search within full site"><span>Full site</span></label></div><a href="https://www.cell.com/search/advanced?SeriesKey=cell&amp;ISSN=0092-8674&amp;journalCode=cell" title="Advanced search" class="quick-search__advanced-link">Advanced search</a></div><a href="javascript:void(0)" aria-label="Close search options" title="Close search options" class="quick-search__box-close"><i aria-hidden="true" class="icon-tools_close"></i></a></div></div><div class="quick-search__alert hidden"><p>Please enter a term before submitting your search.</p><a href="#" class="quick-search__alert--close"> Ok</a></div></form></div></div>

            

        
</div>
  </div>
</nav>
<div id="page-body-id" class="page-body-flag"></div></div>
                </div>
                </div>

        

        </div>
    

    

        


    

            

            

            <div data-widget-id="f8c6daf9-9df1-4fd9-ba3a-7f5d7b3d4466" data-widget-name="UX3HTMLWidget"><style>
.journal-logos:nth-of-type(2) img {
max-height: 30px !important;
}
.journal-logos:nth-of-type(3) img {
max-height: 30px !important;
}
.ux3 .journal-logos + .journal-logos {
margin-left: 30px;
}
.header__wrapper.clearfix .col-xs-6:first-of-type {
width: 55%;
}
.header__wrapper.clearfix .col-xs-6:nth-of-type(2) {
width: 45%;
}
@media screen and (max-width: 1000px) {
.ux3 .journal-logos + .journal-logos {
display: none;
}
.header__wrapper.clearfix .col-xs-6:first-of-type {
width: 50%;
}
.header__wrapper.clearfix .col-xs-6:nth-of-type(2) {
width: 50%;
}
}

.ux3 .journal-logos:first-of-type {
   max-width: 500px;
}

.profile-links {
   white-space: nowrap;
}

@media only screen and (min-width: 740px) and (max-width: 1600px) {
   .journal-logos:nth-of-type(2) img {
      max-width: 250px !important;
   }
}

@media only screen and (min-width: 1024px) and (max-width: 1600px) {
   .header__wrapper.clearfix .col-xs-6:first-of-type {
      width: 60% !important;
   }
   .header__wrapper.clearfix .col-xs-6:nth-of-type(2) {
      width: 40% !important;
   }
}

@media only screen and (max-width: 1240px) {
   .ux3 .journal-logos + .journal-logos {
      display: none !important;
   }
}

</style></div>

            

        
</div></div>

            
                </div>
            

        

        </div>
    

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget axelPublicationContent none  widget-none" id="cadc9b41-a542-448a-9de4-8d0803320174">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none "><link id="build-style-article" rel="stylesheet" type="text/css" href="/products/marlin/cell/releasedAssets/css/build-article-b26f3342b722959c7931.css"><article xmlns="http://www.w3.org/1999/xhtml" data-design="pill" data-has="right-rail" vocab="http://schema.org/" typeof="ScholarlyArticle" data-pii="S0092-8674(21)01337-4" lang="en" dir="ltr"><div data-core-wrapper="header"><header data-extent="frontmatter"><div class="core-container"><div data-article-access="full" data-article-access-type="open" class="meta-panel"><span class="meta-panel__left-content"><span class="meta-panel__type"><span>Article</span></span></span><span class="meta-panel__middle-content"><span class="meta-panel__volumeIssue"><a href="/cell/issue?pii=S0092-8674(20)X0027-4"><span>Volume 184</span>, <span>Issue 26</span></a></span><span class="meta-panel__pages"><span>p6326-6343.e32</span></span><span class="meta-panel__onlineDate">December 22, 2021</span></span><span class="meta-panel__right-content"><span class="meta-panel__access meta-panel__access--open"><span>Open access</span></span></span><div class="article-header__download-full-issue"><button title="Download full issue" type="submit" formaction="/action/downloadMultiplePdf?action=downloadIssuePdf&amp;journal=cell&amp;volume=184&amp;issue=26&amp;pii=S0092-8674%2820%29X0027-4" class="btn--download-issue btn--cancel download-full-issue"><i class="icon-download-new"></i><span>Download Full Issue</span></button><div class="table-of-content__alert table-of-content__alert--download"><p>Download started</p><a href="#" class="table-of-content__alert--close">Ok</a></div></div></div><h1 property="name">A transcriptional rheostat couples past activity to future sensory responses</h1><div class="contributors"><span class="authors"><span role="list"><span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au1" aria-controls="axel_author_au1"><span property="givenName">Tatsuya</span> <span property="familyName">Tsukahara</span></a><div id="au1" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Tatsuya</span> <span property="familyName">Tsukahara</span> <a class="orcid-id" href="https://orcid.org/0000-0002-6928-8562" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0002-6928-8562</a></div><div class="content"><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">3</div><div id="fn1"><div id="ntpara0010" role="paragraph">These authors contributed equally</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Neurobiology, Harvard Medical School, Boston, MA 02115, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Tsukahara/Tatsuya" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a role="doc-noteref">3</a></sup></span> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au2" aria-controls="axel_author_au2"><span property="givenName">David H.</span> <span property="familyName">Brann</span></a><div id="au2" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">David H.</span> <span property="familyName">Brann</span> <a class="orcid-id" href="https://orcid.org/0000-0001-8844-8886" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0001-8844-8886</a></div><div class="content"><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">3</div><div id="fn1"><div id="ntpara0010" role="paragraph">These authors contributed equally</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Neurobiology, Harvard Medical School, Boston, MA 02115, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Brann/David+H" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a role="doc-noteref">3</a></sup></span> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au3" aria-controls="axel_author_au3"><span property="givenName">Stan L.</span> <span property="familyName">Pashkovski</span></a><div id="au3" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Stan L.</span> <span property="familyName">Pashkovski</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Neurobiology, Harvard Medical School, Boston, MA 02115, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Pashkovski/Stan+L" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a></sup></span> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au4" aria-controls="axel_author_au4"><span property="givenName">Grigori</span> <span property="familyName">Guitchounts</span></a><div id="au4" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Grigori</span> <span property="familyName">Guitchounts</span> <a class="orcid-id" href="https://orcid.org/0000-0003-0142-0149" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0003-0142-0149</a></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Neurobiology, Harvard Medical School, Boston, MA 02115, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Guitchounts/Grigori" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a></sup></span> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au5" aria-controls="axel_author_au5"><span property="givenName">Thomas</span> <span property="familyName">Bozza</span></a><div id="au5" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Thomas</span> <span property="familyName">Bozza</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Neurobiology, Northwestern University, Evanston, IL 60208, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Bozza/Thomas" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">2</a></sup></span> ∙ <span property="author" typeof="Person" role="listitem" class="corresponding-author"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au6" aria-controls="axel_author_au6"><span property="givenName">Sandeep Robert</span> <span property="familyName">Datta</span></a><div id="au6" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Sandeep Robert</span> <span property="familyName">Datta</span> <a class="orcid-id" href="https://orcid.org/0000-0002-8068-3862" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0002-8068-3862</a></div><div class="content"><div class="core-author-emails"><div class="label">Correspondence</div><div>Corresponding author</div><div><a id="eadd0010" href="mailto:srdatta@hms.harvard.edu" property="email" title="Send email to Sandeep Robert Datta">srdatta@hms.harvard.edu</a></div></div><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">4</div><div id="fn2"><div id="ntpara0015" role="paragraph">Lead contact</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Neurobiology, Harvard Medical School, Boston, MA 02115, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Datta/Sandeep+Robert" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a role="doc-noteref">4</a></sup> <a id="eadd0010" href="mailto:srdatta@hms.harvard.edu" property="email" title="Send email to Sandeep Robert Datta">srdatta@hms.harvard.edu</a></span></span></span></div><div class="core-self-citation"><div class="expandable-triggers"><button aria-controls="core-affiliations-notes" aria-expanded="false" tabindex="0" role="button"><span>Affiliations &amp; Notes</span><i class="icon-arrow_d_n" aria-hidden="true"></i></button><button aria-controls="core-content-info" aria-expanded="false" tabindex="0" role="button"><span>Article Info</span><i class="icon-arrow_d_n" aria-hidden="true"></i></button></div><div id="core-linked-content">
    

        

</div><div class="expandable-content"><div id="core-affiliations-notes" role="region" data-expandable="item" style="display: none;"><div class="affiliations"><div property="affiliation" typeof="Organization"><span class="label">1</span><span id="aff1" property="name">Department of Neurobiology, Harvard Medical School, Boston, MA 02115, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">2</span><span id="aff2" property="name">Department of Neurobiology, Northwestern University, Evanston, IL 60208, USA</span></div></div><div class="core-authors-notes"><div role="doc-footnote" data-has="label"><div class="label">3</div><div id="fn1"><div id="ntpara0010" role="paragraph">These authors contributed equally</div></div></div><div role="doc-footnote" data-has="label"><div class="label">4</div><div id="fn2"><div id="ntpara0015" role="paragraph">Lead contact</div></div></div></div></div><div id="core-content-info" role="region" data-expandable="item" style="display: none;"><div class="history"><div>Publication History:</div><div><span>Received May 3, 2021</span>; <span>Revised October 7, 2021</span>; <span>Accepted November 11, 2021</span>; <span>Published online December 7, 2021</span></div></div><div class="core-links"><span class="doi">DOI: <a href="https://doi.org/10.1016/j.cell.2021.11.022" property="sameAs" target="_blank">10.1016/j.cell.2021.11.022 <span class="sr-only">External Link</span></a></span><span class="science-direct">Also available on <a href="https://www.sciencedirect.com/science/article/pii/S0092867421013374" target="_blank">ScienceDirect <span class="sr-only">External Link</span></a></span></div><div role="paragraph">Copyright: © 2021 The Author(s). Published by Elsevier Inc.</div><div class="core-links"><span>User License: <a href="http://creativecommons.org/licenses/by-nc-nd/4.0/" target="_blank">Creative Commons Attribution – NonCommercial – NoDerivs (CC BY-NC-ND 4.0)</a> | <a href="https://www.elsevier.com/open-access" target="_blank">Elsevier's open access license policy</a></span></div><div class="core-crossmark-badge"><a data-target="crossmark" href="#" title="Check for updates on crossmark" data-doi="10.1016/j.cell.2021.11.022" data-id="article-info-crossmark" class="crossmark__link" style="cursor: pointer;"><svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 290 58" height="22px" role="presentation"><path fill="url(#horizontal_crossmark)" stroke="#948F8F" stroke-miterlimit="10" stroke-width=".75" d="M287 1H3a2 2 0 0 0-2 2v52a2 2 0 0 0 2 2h284a2 2 0 0 0 2-2V3a2 2 0 0 0-2-2Z"></path><path fill="#535353" d="M80.3 23.58a6.395 6.395 0 0 0-.55-1.43 3.92 3.92 0 0 0-2.19-1.86 5.099 5.099 0 0 0-1.71-.29 5 5 0 0 0-2.46.58 4.831 4.831 0 0 0-1.69 1.54 6.921 6.921 0 0 0-1 2.19 10.16 10.16 0 0 0 0 5 6.921 6.921 0 0 0 1 2.19 4.83 4.83 0 0 0 1.69 1.5 5 5 0 0 0 2.46.58 4.44 4.44 0 0 0 1.85-.36 4.12 4.12 0 0 0 1.38-1c.39-.436.695-.942.9-1.49A7.094 7.094 0 0 0 80.4 29h3a8.46 8.46 0 0 1-.69 2.92 7.179 7.179 0 0 1-1.59 2.29 7 7 0 0 1-2.35 1.49 8.14 8.14 0 0 1-3 .52 8.43 8.43 0 0 1-3.59-.74 7.88 7.88 0 0 1-2.69-2 9.001 9.001 0 0 1-1.69-3 11.41 11.41 0 0 1 0-7.23 9.001 9.001 0 0 1 1.69-3 8.001 8.001 0 0 1 2.69-2 8.3 8.3 0 0 1 3.59-.75 9.44 9.44 0 0 1 2.83.41c.866.242 1.68.642 2.4 1.18A6.229 6.229 0 0 1 82.65 21a6.66 6.66 0 0 1 .78 2.58H80.3ZM85.9 17.9h2.85v6.63h.05a4.15 4.15 0 0 1 1.56-1.46 4.61 4.61 0 0 1 2.31-.59 4.83 4.83 0 0 1 3.33 1.1 4.21 4.21 0 0 1 1.22 3.3v8.88H94.4v-8.13a3.51 3.51 0 0 0-.65-2.21 2.36 2.36 0 0 0-1.87-.69 3.1 3.1 0 0 0-1.3.26 2.821 2.821 0 0 0-1 .73A3.41 3.41 0 0 0 89 26.8a3.88 3.88 0 0 0-.23 1.33v7.63H85.9V17.9ZM102.47 30c-.002.491.069.98.21 1.45.131.448.348.866.64 1.23a3.08 3.08 0 0 0 1.08.84 3.543 3.543 0 0 0 1.55.31 3.431 3.431 0 0 0 2-.54c.541-.402.94-.966 1.14-1.61h2.7A5.349 5.349 0 0 1 109.7 35a5.715 5.715 0 0 1-1.74.85 7 7 0 0 1-2 .29 6.848 6.848 0 0 1-2.7-.5 5.577 5.577 0 0 1-2-1.4A5.953 5.953 0 0 1 100 32a8.748 8.748 0 0 1-.41-2.75 7.768 7.768 0 0 1 .44-2.61 6.67 6.67 0 0 1 1.25-2.17 5.863 5.863 0 0 1 4.56-2 5.79 5.79 0 0 1 2.74.64c.783.41 1.466.986 2 1.69a6.824 6.824 0 0 1 1.16 2.41c.264.905.352 1.852.26 2.79h-9.53Zm6.65-1.87a4.605 4.605 0 0 0-.29-1.3c-.148-.4-.368-.77-.65-1.09a3.22 3.22 0 0 0-1-.75 3 3 0 0 0-1.31-.29 3.431 3.431 0 0 0-1.36.26 3.108 3.108 0 0 0-1 .73 3.594 3.594 0 0 0-.7 1.09 3.798 3.798 0 0 0-.29 1.35h6.65-.05ZM122.88 27.15a2.666 2.666 0 0 0-1-1.81 3.14 3.14 0 0 0-2-.61c-.4.005-.798.07-1.18.19-.435.136-.83.376-1.15.7a3.92 3.92 0 0 0-.87 1.43 6.783 6.783 0 0 0-.35 2.39c.002.539.065 1.076.19 1.6.113.509.316.993.6 1.43.273.41.635.752 1.06 1a3 3 0 0 0 1.58.39 2.872 2.872 0 0 0 2.06-.77 3.596 3.596 0 0 0 1-2.18h2.85a6.26 6.26 0 0 1-1.91 3.86 5.83 5.83 0 0 1-4 1.34 6.61 6.61 0 0 1-2.69-.51 5.695 5.695 0 0 1-2-1.4 6.001 6.001 0 0 1-1.21-2.11 8.279 8.279 0 0 1-.41-2.65 8.987 8.987 0 0 1 .4-2.72 6.237 6.237 0 0 1 1.2-2.21 5.614 5.614 0 0 1 2-1.47 6.7 6.7 0 0 1 2.79-.54 7.801 7.801 0 0 1 2.14.29 5.634 5.634 0 0 1 1.8.86c.529.386.969.88 1.29 1.45a5 5 0 0 1 .6 2.07h-2.85l.06-.02ZM128.13 17.9H131V28l5.15-5.22h3.5l-5 4.75 5.42 8.17h-3.47l-4-6.27-1.6 1.7v4.63h-2.85V17.9h-.02ZM147.15 22.83h2.13v-1.08a5.505 5.505 0 0 1 .3-2c.161-.456.436-.862.8-1.18.33-.273.721-.462 1.14-.55.44-.094.89-.141 1.34-.14a8.158 8.158 0 0 1 2 .18v2.24a4.113 4.113 0 0 0-.59-.11 6.343 6.343 0 0 0-.74 0 1.74 1.74 0 0 0-1 .27 1.204 1.204 0 0 0-.41 1.08v1.33h2.42V25h-2.42v10.8h-2.85V25h-2.12v-2.17ZM162.23 36.1a6.999 6.999 0 0 1-2.76-.51 5.877 5.877 0 0 1-2-1.41 5.998 5.998 0 0 1-1.33-2.18 8.775 8.775 0 0 1 0-5.48 6.008 6.008 0 0 1 1.27-2.15 5.894 5.894 0 0 1 2-1.41A7.71 7.71 0 0 1 165 23a5.894 5.894 0 0 1 2 1.41 6.003 6.003 0 0 1 1.28 2.15c.585 1.78.585 3.7 0 5.48a5.99 5.99 0 0 1-3.28 3.54 7.002 7.002 0 0 1-2.77.52Zm0-2.25a3.268 3.268 0 0 0 1.65-.4c.447-.262.83-.621 1.12-1.05.304-.444.528-.938.66-1.46a6.581 6.581 0 0 0 0-3.31 4.416 4.416 0 0 0-.66-1.46 3.526 3.526 0 0 0-1.15-1 3.6 3.6 0 0 0-3.3 0 3.526 3.526 0 0 0-1.15 1 4.414 4.414 0 0 0-.66 1.46 6.558 6.558 0 0 0 0 3.31c.132.522.356 1.016.66 1.46a3.47 3.47 0 0 0 1.15 1.05 3.26 3.26 0 0 0 1.68.4ZM171.13 22.83h2.67v2.5c.097-.362.264-.701.49-1 .251-.341.547-.647.88-.91a4.404 4.404 0 0 1 1.14-.66 3.41 3.41 0 0 1 1.28-.25c.23-.013.46-.013.69 0h.39v2.79l-.61-.09a5.173 5.173 0 0 0-.61 0 3.278 3.278 0 0 0-1.36.29 3.131 3.131 0 0 0-1.11.85 4.171 4.171 0 0 0-.75 1.39 6 6 0 0 0-.28 1.9v6.15h-2.85V22.83h.03ZM198.22 35.75h-2.8V34a3.907 3.907 0 0 1-1.56 1.56 4.25 4.25 0 0 1-2.11.59 4.742 4.742 0 0 1-3.75-1.31 5.537 5.537 0 0 1-1.12-3.84v-8.2h2.85v7.92a3.503 3.503 0 0 0 .65 2.4 2.38 2.38 0 0 0 1.83.7 3.599 3.599 0 0 0 1.5-.27 2.66 2.66 0 0 0 1-.74c.259-.326.444-.705.54-1.11a5.79 5.79 0 0 0 .16-1.4v-7.5h2.85v12.95h-.04ZM201.3 22.83h2.7v1.75a3.523 3.523 0 0 1 1.72-1.58 5.568 5.568 0 0 1 2.33-.49 6.17 6.17 0 0 1 2.66.54 5.246 5.246 0 0 1 1.89 1.47c.515.65.899 1.394 1.13 2.19.257.87.385 1.773.38 2.68.001.851-.114 1.7-.34 2.52a6.568 6.568 0 0 1-1 2.16 5.002 5.002 0 0 1-4.21 2.06c-.419 0-.837-.036-1.25-.11a5.39 5.39 0 0 1-1.2-.36A4.737 4.737 0 0 1 205 35a3.567 3.567 0 0 1-.81-.91v6.45h-2.89V22.83Zm10 6.47c0-.574-.078-1.146-.23-1.7a4.606 4.606 0 0 0-.67-1.46 3.475 3.475 0 0 0-1.12-1 3.06 3.06 0 0 0-1.55-.39A3.141 3.141 0 0 0 205 26a5.518 5.518 0 0 0-1 3.3c-.004.611.077 1.22.24 1.81.138.523.38 1.012.71 1.44a3.55 3.55 0 0 0 1.14 1 3.242 3.242 0 0 0 1.54.35 3.18 3.18 0 0 0 1.65-.4c.438-.25.817-.59 1.11-1 .299-.44.513-.931.63-1.45a7.38 7.38 0 0 0 .23-1.75h.05ZM228.67 35.75H226V34a3.38 3.38 0 0 1-1.67 1.61 5.668 5.668 0 0 1-2.33.49 6.17 6.17 0 0 1-2.66-.54 5.307 5.307 0 0 1-1.89-1.46 6.254 6.254 0 0 1-1.12-2.19 9.48 9.48 0 0 1-.37-2.71 8.67 8.67 0 0 1 .47-3 6.058 6.058 0 0 1 1.26-2.1 4.89 4.89 0 0 1 1.8-1.21 5.738 5.738 0 0 1 2.06-.39c.412 0 .824.037 1.23.11.413.073.815.194 1.2.36.38.166.736.381 1.06.64.32.255.594.562.81.91V17.9h2.85v17.85h-.03Zm-10-6.33c-.001.55.069 1.098.21 1.63.13.511.35.996.65 1.43.288.414.668.757 1.11 1a3.241 3.241 0 0 0 1.6.38 3.199 3.199 0 0 0 1.64-.4 3.398 3.398 0 0 0 1.13-1.05c.297-.447.517-.94.65-1.46.14-.542.211-1.1.21-1.66a5.227 5.227 0 0 0-1-3.35 3.175 3.175 0 0 0-2.61-1.2 3.23 3.23 0 0 0-1.69.41 3.478 3.478 0 0 0-1.12 1.07 4.494 4.494 0 0 0-.62 1.5 7.871 7.871 0 0 0-.11 1.7h-.05ZM242.6 32.88c-.018.258.03.516.14.75a.577.577 0 0 0 .54.23h.3c.135-.002.269-.019.4-.05v2l-.39.11-.49.08-.5.08h-.42a2.75 2.75 0 0 1-1.45-.35 1.731 1.731 0 0 1-.75-1.23 4.998 4.998 0 0 1-2.09 1.2 8.224 8.224 0 0 1-2.39.38 5.861 5.861 0 0 1-1.67-.24 4.394 4.394 0 0 1-1.41-.7 3.35 3.35 0 0 1-1-1.18 3.624 3.624 0 0 1-.36-1.66 3.84 3.84 0 0 1 .44-2 3.246 3.246 0 0 1 1.15-1.17 5.195 5.195 0 0 1 1.6-.61c.593-.127 1.19-.223 1.79-.29a14.77 14.77 0 0 1 1.47-.21 6.896 6.896 0 0 0 1.24-.21c.318-.077.611-.236.85-.46a1.286 1.286 0 0 0 .31-.94 1.458 1.458 0 0 0-.26-.9A1.759 1.759 0 0 0 239 25a2.807 2.807 0 0 0-.86-.25 6.941 6.941 0 0 0-.9-.06 3.58 3.58 0 0 0-2 .5 2 2 0 0 0-.87 1.55h-2.85c.021-.73.227-1.442.6-2.07.338-.54.797-.996 1.34-1.33a5.475 5.475 0 0 1 1.84-.7c.692-.134 1.395-.2 2.1-.2.632 0 1.262.067 1.88.2a5.337 5.337 0 0 1 1.66.65 3.6 3.6 0 0 1 1.19 1.16c.314.525.47 1.129.45 1.74v6.65l.02.04Zm-2.85-3.6a3.45 3.45 0 0 1-1.6.51c-.667.06-1.3.147-1.9.26a6.137 6.137 0 0 0-.87.21 2.64 2.64 0 0 0-.75.38c-.22.164-.395.38-.51.63a2.129 2.129 0 0 0-.19.94c-.011.29.085.575.27.8.182.215.407.388.66.51.269.128.555.216.85.26.277.05.558.076.84.08a4.63 4.63 0 0 0 1-.12 3.418 3.418 0 0 0 1-.43 2.73 2.73 0 0 0 .81-.76 1.93 1.93 0 0 0 .32-1.14v-2.14l.07.01ZM244.37 22.83h2.15V19h2.85v3.88h2.58V25h-2.58v6.9a7.064 7.064 0 0 0 0 .77c.016.194.074.381.17.55a.84.84 0 0 0 .41.34 2 2 0 0 0 .75.11h.6c.203-.01.404-.04.6-.09v2.2l-.93.1a8.403 8.403 0 0 1-.92 0 6.33 6.33 0 0 1-1.81-.21 2.421 2.421 0 0 1-1.08-.62 2.15 2.15 0 0 1-.52-1 7.791 7.791 0 0 1-.16-1.42V25h-2.15v-2.17h.04ZM256.22 30c-.002.491.069.98.21 1.45.131.448.348.866.64 1.23a3.08 3.08 0 0 0 1.08.84 3.543 3.543 0 0 0 1.55.31 3.431 3.431 0 0 0 2-.54c.541-.402.94-.966 1.14-1.61h2.7a5.352 5.352 0 0 1-2.1 3.26 5.715 5.715 0 0 1-1.74.85 7 7 0 0 1-2 .29 6.848 6.848 0 0 1-2.7-.5 5.577 5.577 0 0 1-2-1.4 5.947 5.947 0 0 1-1.21-2.18 8.754 8.754 0 0 1-.41-2.75 7.758 7.758 0 0 1 .44-2.61 6.67 6.67 0 0 1 1.25-2.17 5.863 5.863 0 0 1 4.56-2 5.79 5.79 0 0 1 2.74.64c.783.41 1.466.986 2 1.69a6.824 6.824 0 0 1 1.16 2.41c.243.916.308 1.87.19 2.81h-9.5V30Zm6.65-1.87a4.605 4.605 0 0 0-.29-1.3c-.148-.4-.368-.77-.65-1.09a3.22 3.22 0 0 0-1-.75 3 3 0 0 0-1.31-.29 3.431 3.431 0 0 0-1.36.26 3.108 3.108 0 0 0-1 .73 3.575 3.575 0 0 0-.7 1.09 3.798 3.798 0 0 0-.29 1.35h6.65-.05ZM270 31.6a2.154 2.154 0 0 0 .95 1.75c.607.35 1.3.524 2 .5.304-.002.608-.022.91-.06a3.943 3.943 0 0 0 1-.24c.286-.105.54-.28.74-.51a1.23 1.23 0 0 0 .26-.89 1.261 1.261 0 0 0-.4-.9 2.779 2.779 0 0 0-1-.56 9.315 9.315 0 0 0-1.34-.36l-1.5-.33a14.219 14.219 0 0 1-1.54-.42 4.937 4.937 0 0 1-1.33-.68 3.12 3.12 0 0 1-.94-1.09 3.472 3.472 0 0 1-.35-1.64 2.94 2.94 0 0 1 .51-1.76 3.913 3.913 0 0 1 1.3-1.15 5.66 5.66 0 0 1 1.75-.61 10.316 10.316 0 0 1 1.84-.17 8.418 8.418 0 0 1 1.91.21 5.19 5.19 0 0 1 1.65.69c.496.318.916.741 1.23 1.24.341.559.55 1.188.61 1.84h-3a1.783 1.783 0 0 0-.94-1.37 3.892 3.892 0 0 0-1.69-.35 6.093 6.093 0 0 0-.71 0 3.327 3.327 0 0 0-.78.19 1.642 1.642 0 0 0-.61.4 1 1 0 0 0-.25.69 1.093 1.093 0 0 0 .36.85c.294.245.634.428 1 .54.437.153.885.274 1.34.36l1.55.33a20.9 20.9 0 0 1 1.53.42c.479.157.931.386 1.34.68.41.277.751.643 1 1.07.255.494.379 1.045.36 1.6a3.48 3.48 0 0 1-.52 2 4.15 4.15 0 0 1-1.36 1.3 6.19 6.19 0 0 1-1.86.73 9.435 9.435 0 0 1-2 .23 8.823 8.823 0 0 1-2.26-.27 5.265 5.265 0 0 1-1.8-.84 4 4 0 0 1-1.2-1.4 4.445 4.445 0 0 1-.46-2h2.7v-.02Z"></path><path fill="#C72914" d="m21.94 38.74 18.12-12.08v-14.5H21.94v26.58Z"></path><path fill="#EF3340" d="M40.06 38.74 21.94 26.66v-14.5h18.12v26.58Z"></path><path fill="#3EB1C8" d="M31 9a20 20 0 1 0 0 40 20 20 0 0 0 0-40Zm0 35.1a15.1 15.1 0 1 1 0-30.2 15.1 15.1 0 0 1 0 30.2Z"></path><path fill="#FFC72C" d="M42.09 39.23A15.09 15.09 0 0 1 17 23.38l-4.2-2.64a20 20 0 0 0 33.49 21.14l-4.2-2.65Z"></path><defs><linearGradient id="horizontal_crossmark" x1="145" x2="145" y1="7.33" y2="56.82" gradientUnits="userSpaceOnUse"><stop stop-color="#fff"></stop><stop offset="1" stop-color="#c4c4c4"></stop></linearGradient></defs></svg></a></div><div class="core-content-info"></div></div></div></div></div></header><div class="core-header-aside">
    

        


    

            

            

            <div class="cover-image cover-image--axel"><div class="cover-image__thumb"><a href="https://www.cell.com/cell/issue?pii=S0092-8674(20)X0027-4"><img src="/cms/asset/e60548fe-1f2a-409e-9e57-7dd962cd2a11/cov200h.gif" alt="Cover Image - Cell, Volume 184, Issue 26"></a></div></div>

            

        
</div></div><div data-core-wrapper="content"><div id="dummy-sticky-observer"></div><div data-core-nav="header" data-extent="frontmatter"><div class="core-nav-wrapper core-container" aria-label="Article navigation"><div class="navigation-menu-pdf">
    

        


    

            

            

            <div data-widget-id="ecff07e8-0bc4-4533-b1f1-49365e2ead20" data-widget-name="UX3ArticleToolsWidget" class="article-tools"><a href="/action/showPdf?pii=S0092-8674%2821%2901337-4" title="Standard PDF (opens in a new tab)" target="_blank" aria-hidden="false" role="menuitem" class="article-tools__item__pdf"><svg focusable="false" viewBox="0 0 32 32" width="20" height="20" title="PDF" class="pdf-icon"><path id="pdfwhite" d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path id="pdfda0000" d="M.167 2.592H22.39V9.72H.166z" fill="#da0000"></path><path id="pdffff9f9" fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path id="pdff91d0a" d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="basic">Download PDF</span><span class="alternative">Download PDF</span></a></div>

            

        
</div><div class="core-sections-menu"><button data-id="article-toolbar-showhide" aria-controls="article_sections_menu" aria-expanded="false" aria-haspopup="true" aria-label="Toggle section navigation menu" style="margin-top: 0px;"><i class="icon-list" aria-hidden="true"></i><span>Outline</span></button><nav id="article_sections_menu" aria-label="Contents" data-core-nav="article" style="margin-top: 0px;"><div class="heading">Outline<button aria-haspopup="true" aria-controls="article_sections_menu" title="Close outline menu"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div><ul><li><a href="#author-highlights-abstract">Highlights</a></li><li><a href="#author-abstract">Summary</a></li><li><a href="#graphical-abstract">Graphical abstract</a></li><li><a href="#keywords">Keywords</a></li><li><a href="#sec-1">Introduction</a></li><li><a href="#sec-2">Results</a></li><li><a href="#sec-3">Discussion</a></li><li><a href="#sec-4">STAR★Methods</a></li><li><a href="#acknowledgments">Acknowledgments</a></li><li><a href="#supplementary-material">Supplemental information (5)</a></li><li><a href="#data-availability">Data and code availability</a></li><li><a href="#references">References</a></li><li role="none" class="mobile-only"><a role="menuitem" href="#core-collateral-metrics">Article metrics</a></li><li role="none" class="mobile-only"><a role="menuitem" href="#core-collateral-relatedArticles">Related Articles</a></li></ul></nav></div><div class="navigation-menu-share">
    

        


    

            

            

            <!-- Go to https://www.addtoany.com/buttons/customize/ to customize your tools --><script type="text/javascript" defer="defer" src="https://static.addtoany.com/menu/page.js"></script><script type="text/javascript">var a2a_config = a2a_config || {};
a2a_config.thanks = {
    postShare: false,
    ad: false,
};</script><script>var a2a_config = a2a_config || {};
let res = window.location.href.replace(/\(/g, "%28").replace(/\)/g, "%29");
a2a_config.templates = a2a_config.templates || {};

a2a_config.templates.email = {
    body: res
};
</script><div data-widget-id="5f978446-c80a-450c-a185-94d633df2db2" data-widget-name="UX3share" aria-hidden="false" role="group" class="article-tools__dropblock article-tools__share"><button aria-label="Toggle share menu" aria-expanded="false" aria-haspopup="true" role="menuitem" aria-controls="navigation_menu_share_block" aria-labelledby="share-label" class="article-tools__item__share"><i aria-hidden="true" class="icon-article-share"></i><span id="share-label">Share</span></button><div id="navigation_menu_share_block" role="menu" class="share__block axel-share article-tools__dropblock__holder"><h4 class="axel-share__title heading">Share<button title="Close share menu" aria-expanded="false" aria-haspopup="true" aria-controls="navigation_menu_share_block"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></h4><div class="axel-share__inner"><span class="sr-only">Share on</span><ul class="a2a__share-list rlist w-slide--list a2a a2a_kit a2a_kit_size_32 a2a_default_style" data-a2a-url="https://www.cell.com/cell/fulltext/S0092-8674%2821%2901337-4" style="line-height: 32px;"><li class="a2a_listitem_custom"><a role="link" title="Email" class="a2a_button_email" target="_blank" rel="nofollow noopener" href="/#email"><i aria-hidden="true" class="at-icon-wrapper icon-gizmo-email-plus"></i><span>Email</span></a></li><li class="a2a_listitem_custom"><a role="link" title="formerly known as Twitter" class="a2a_button_twitter" target="_blank" rel="nofollow noopener" href="/#twitter"><i aria-hidden="true" class="at-icon-wrapper icon-x-default"></i><span>X</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Facebook" class="a2a_button_facebook" target="_blank" rel="nofollow noopener" href="/#facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook-square-2"></i><span>Facebook</span></a></li><li class="a2a_listitem_custom"><a role="link" title="LinkedIn" class="a2a_button_linkedin" target="_blank" rel="nofollow noopener" href="/#linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin-square"></i><span>LinkedIn</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Sina Weibo" class="a2a_button_sina_weibo" target="_blank" rel="nofollow noopener" href="/#sina_weibo"><i aria-hidden="true" class="at-icon-wrapper icon-weibo"></i><span>Sina Weibo</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Mendeley" class="a2a_button_mendeley" target="_blank" rel="nofollow noopener" href="/#mendeley"><i aria-hidden="true" class="at-icon-wrapper icon-mendeley-2"></i><span>Mendeley</span></a></li><li class="a2a_listitem_custom"><a role="link" title="bluesky" class="a2a_button_bluesky" target="_blank" rel="nofollow noopener" href="/#bluesky"><i aria-hidden="true" class="at-icon-wrapper icon-bluesky-logo"></i><span>bluesky</span></a></li><div style="clear: both;"></div></ul><div class="extra-links-drop-zone">
    

        


    

            

            

            <div data-widget-id="f1a208d3-ac5e-4db2-9c3a-40ef97ddfa10" data-widget-name="UX3HTMLWidget"><a class="axel-share__extra" target="_blank" href="/personalize/addFavoritePublication?doi=10.1016/j.cell.2021.11.022">
	<i class="icon-reading-list" aria-hidden="true"></i>
	<span>Add to my reading list</span>
</a></div>

            

        
</div></div></div></div>

            

        
</div><div><button data-id="article-nav-menubar-more" aria-controls="article_more_menu" aria-expanded="false" aria-haspopup="true" aria-label="Toggle more menu"><i aria-hidden="true" class="icon-more-mobile"></i><span>More</span></button><div class="info-panel"><div class="info-panel__left-content"><div class="info-panel__article_tools info-panel__item"><div id="article_more_menu" aria-label="More menu" data-core-nav="more"><div class="heading">More<button title="Close more menu" aria-controls="article_more_menu" aria-haspopup="true"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div>
    

        


    

            

            

            <ul data-widget-id="183a17a2-87b5-4e6f-bb85-6b5cda068c89" data-widget-name="UX3ArticleToolsWidget" class="article-tools"><li class="article-tools__item article-tools__pdf"><a href="/action/showPdf?pii=S0092-8674%2821%2901337-4" title="Standard PDF (opens in a new tab)" target="_blank" aria-hidden="false" class="article-tools__ctrl article-tools__item__pdf"><svg focusable="false" viewBox="0 0 32 32" width="20" height="20" title="PDF" class="pdf-icon"><path id="pdfwhite" d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path id="pdfda0000" d="M.167 2.592H22.39V9.72H.166z" fill="#da0000"></path><path id="pdffff9f9" fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path id="pdff91d0a" d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="basic">Download PDF</span><span class="alternative">Download PDF</span></a></li><li><a href="https://www.cell.com/action/showCitFormats?doi=10.1016%2Fj.cell.2021.11.022&amp;pii=S0092-8674%2821%2901337-4" title="Cite" target="_blank" aria-label="Open Cite in new tab" class="article-tools__item__displayExportCitation"><i aria-hidden="true" class="icon-export-Citation"></i><span>Cite</span></a></li><li id="article_tools_share">
    

        


    

            

            

            <!-- Go to https://www.addtoany.com/buttons/customize/ to customize your tools --><script type="text/javascript" defer="defer" src="https://static.addtoany.com/menu/page.js"></script><script type="text/javascript">var a2a_config = a2a_config || {};
a2a_config.thanks = {
    postShare: false,
    ad: false,
};</script><script>var a2a_config = a2a_config || {};
let res = window.location.href.replace(/\(/g, "%28").replace(/\)/g, "%29");
a2a_config.templates = a2a_config.templates || {};

a2a_config.templates.email = {
    body: res
};
</script><div data-widget-id="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da" data-widget-name="UX3share" class="dropBlock article-tools__dropblock article-tools__share"><button data-db-target-for="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da" aria-expanded="false" title="open Share menu" class="article-tools__ctrl share__ctrl"><i aria-hidden="true" class="icon-article-share"></i><span>Share</span></button><div data-db-target-of="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da" id="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da_Pop" class="share__block axel-share dropBlock__holder article-tools__dropblock__holder"><h4 class="axel-share__title">Share</h4><div class="axel-share__inner"><span class="sr-only">Share on</span><ul class="a2a__share-list rlist w-slide--list a2a a2a_kit a2a_kit_size_32 a2a_default_style" data-a2a-url="https://www.cell.com/cell/fulltext/S0092-8674%2821%2901337-4" style="line-height: 32px;"><li class="a2a_listitem_custom"><a role="link" title="Email" class="a2a_button_email" target="_blank" rel="nofollow noopener" href="/#email"><i aria-hidden="true" class="at-icon-wrapper icon-gizmo-email-plus"></i><span>Email</span></a></li><li class="a2a_listitem_custom"><a role="link" title="formerly known as Twitter" class="a2a_button_twitter" target="_blank" rel="nofollow noopener" href="/#twitter"><i aria-hidden="true" class="at-icon-wrapper icon-x-default"></i><span>X</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Facebook" class="a2a_button_facebook" target="_blank" rel="nofollow noopener" href="/#facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook-square-2"></i><span>Facebook</span></a></li><li class="a2a_listitem_custom"><a role="link" title="LinkedIn" class="a2a_button_linkedin" target="_blank" rel="nofollow noopener" href="/#linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin-square"></i><span>LinkedIn</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Sina Weibo" class="a2a_button_sina_weibo" target="_blank" rel="nofollow noopener" href="/#sina_weibo"><i aria-hidden="true" class="at-icon-wrapper icon-weibo"></i><span>Sina Weibo</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Mendeley" class="a2a_button_mendeley" target="_blank" rel="nofollow noopener" href="/#mendeley"><i aria-hidden="true" class="at-icon-wrapper icon-mendeley-2"></i><span>Mendeley</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Bluesky" class="a2a_button_bluesky" target="_blank" rel="nofollow noopener" href="/#bluesky"><i aria-hidden="true" class="at-icon-wrapper icon-bluesky-logo"></i><span>Bluesky</span></a></li><div style="clear: both;"></div></ul><div class="extra-links-drop-zone">
    

        


    

            

            

            <div data-widget-id="ae9bf5c5-a2ee-4f5c-8e41-9b21638b1a9f" data-widget-name="UX3HTMLWidget"><a class="axel-share__extra" target="_blank" href="/personalize/addFavoritePublication?doi=10.1016/j.cell.2021.11.022">
	<i class="icon-reading-list" aria-hidden="true"></i>
	<span>Add to my reading list</span>
</a></div>

            

        
</div></div></div></div>

            

        
</li><li><a href="https://www.cell.com/action/addCitationAlert?doi=10.1016%2Fj.cell.2021.11.022" title="Set Alert" target="_blank" aria-label="Open Set Alert in new tab" class="article-tools__item__displayCreateCitationAlert"><i aria-hidden="true" class="icon-set-alert"></i><span>Set Alert</span></a></li><li><a href="https://s100.copyright.com/AppDispatchServlet?publisherName=ELS&amp;contentID=S0092867421013374&amp;orderBeanReset=true&amp;orderSource=Phoenix" title="Get Rights" target="_blank" aria-label="Open Get Rights in new tab" class="article-tools__item__request"><i aria-hidden="true" class="icon-get-rights"></i><span>Get Rights</span></a></li><li><a href="https://www.contentednet.com/salesforce-form/?pii=S0092867421013374&amp;issn=00928674&amp;source=jbs" title="Reprints" target="_blank" aria-label="Open Reprints in new tab" class="article-tools__item__reprints"><i aria-hidden="true" class="icon-article-reprints"></i><span>Reprints</span></a></li></ul>

            

        
</div></div><div class="info-panel__download-issue info-panel__item"><div class="article-header__download-full-issue"><button title="Download full issue" type="submit" formaction="/action/downloadMultiplePdf?action=downloadIssuePdf&amp;journal=cell&amp;volume=184&amp;issue=26&amp;pii=S0092-8674%2820%29X0027-4" class="btn--download-issue btn--cancel download-full-issue"><i class="icon-download-new"></i><span>Download Full Issue</span></button><div class="table-of-content__alert table-of-content__alert--download"><p>Download started</p><a href="#" class="table-of-content__alert--close">Ok</a></div></div></div></div><div class="info-panel__right-content"><div class="info-panel__navigation info-panel__item"><div class="content-navigation"><a href="/cell/fulltext/S0092-8674(21)01373-8" aria-label="Previous article" aria-disabled="false" class="content-navigation__prev"><i aria-hidden="true" class="icon-arrow_l"></i><span>Previous article</span></a><a href="/cell/fulltext/S0092-8674(21)01334-9" aria-label="Next article" aria-disabled="false" class="content-navigation__next"><span>Next article</span><i aria-hidden="true" class="icon-arrow_r"></i></a></div></div></div></div></div></div></div><div class="core-sections-menu-outer"><div class="core-sections-menu"><button aria-controls="article_sections_menu-desktop" aria-haspopup="true" aria-expanded="false" aria-label="Toggle sections menu" data-id="article-toolbar-showhide"><i class="icon-show-outline" aria-hidden="true"></i><span>Show Outline</span></button><nav id="article_sections_menu-desktop" aria-label="Contents" data-core-nav="article" style="margin-top: 0px;"><button aria-controls="article_sections_menu-desktop" aria-haspopup="true" aria-expanded="false" aria-label="Toggle sections menu" data-id="article-toolbar-showhide"><i class="icon-hide-outline" aria-hidden="true"></i><span>Hide Outline</span></button><ul><li><a href="#author-highlights-abstract">Highlights</a></li><li><a href="#author-abstract">Summary</a></li><li><a href="#graphical-abstract">Graphical abstract</a></li><li><a href="#keywords">Keywords</a></li><li><a href="#sec-1">Introduction</a></li><li><a href="#sec-2">Results</a></li><li><a href="#sec-3">Discussion</a></li><li><a href="#sec-4">STAR★Methods</a></li><li><a href="#acknowledgments">Acknowledgments</a></li><li><a href="#supplementary-material">Supplemental information (5)</a></li><li><a href="#data-availability">Data and code availability</a></li><li><a href="#references">References</a></li><li role="none" class="mobile-only"><a role="menuitem" href="#core-collateral-metrics">Article metrics</a></li><li role="none" class="mobile-only"><a role="menuitem" href="#core-collateral-relatedArticles">Related Articles</a></li></ul></nav></div></div><div><div class="core-content-inner"><div class="core-linked-content"></div><div id="abstracts" data-extent="frontmatter"><div class="core-container"><section id="author-highlights-abstract" property="abstract" typeof="Text" role="doc-abstract"><h2 property="name">Highlights</h2><div id="abspara0020" role="paragraph"><div id="ulist0010" role="list"><div id="u0010" role="listitem"><div class="label">•</div><div class="content"><div id="p0010" role="paragraph">Olfactory sensory neurons harbor unique, odorant receptor-specific transcriptomes</div></div></div><div id="u0015" role="listitem"><div class="label">•</div><div class="content"><div id="p0015" role="paragraph">Receptor engagement by environmental odors drives transcriptional variation</div></div></div><div id="u0020" role="listitem"><div class="label">•</div><div class="content"><div id="p0020" role="paragraph">Environment-dependent changes in gene expression adaptively modify odor responses</div></div></div><div id="u0025" role="listitem"><div class="label">•</div><div class="content"><div id="p0025" role="paragraph"><i>In&nbsp;vivo</i> imaging reveals that peripheral odor codes are altered across environments</div></div></div></div></div></section><section id="author-abstract" property="abstract" typeof="Text" role="doc-abstract"><h2 property="name">Summary</h2><div id="abspara0010" role="paragraph">Animals traversing different environments encounter both stable background stimuli and novel cues, which are thought to be detected by primary sensory neurons and then distinguished by downstream brain circuits. Here, we show that each of the ∼1,000 olfactory sensory neuron (OSN) subtypes in the mouse harbors a distinct transcriptome whose content is precisely determined by interactions between its odorant receptor and the environment. This transcriptional variation is systematically organized to support sensory adaptation: expression levels of more than 70 genes relevant to transforming odors into spikes continuously vary across OSN subtypes, dynamically adjust to new environments over hours, and accurately predict acute OSN-specific odor responses. The sensory periphery therefore separates salient signals from predictable background via a transcriptional rheostat whose moment-to-moment state reflects the past and constrains the future; these findings suggest a general model in which structured transcriptional variation within a cell type reflects individual experience.</div></section><section id="graphical-abstract" property="abstract" typeof="Text" role="doc-abstract"><h2 property="name">Graphical abstract</h2><div id="abspara0015" role="paragraph"><div class="figure-wrap"><figure id="undfig1" class="graphic" data-figureviewer-ignore="true"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.11.022/asset/55c8b10a-2dd7-4374-b48f-085d4232b48a/main.assets/fx1_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="fx1" alt="Graphical abstract undfig1" src="/cms/10.1016/j.cell.2021.11.022/asset/e8773704-36aa-49dd-89f3-453586e8cd4f/main.assets/fx1.jpg" height="375" width="375" data-viewer-src="/cms/10.1016/j.cell.2021.11.022/asset/55c8b10a-2dd7-4374-b48f-085d4232b48a/main.assets/fx1_lrg.jpg" loading="lazy"></a></figure></div></div></section></div></div><div class="core-linked-content"></div><section id="keywords" property="keywords"><h2>Keywords</h2><ol><li><a href="/action/doSearch?AllField=&quot;olfaction&quot;" alt="Search on this keyword" rel="nofollow">olfaction</a></li><li><a href="/action/doSearch?AllField=&quot;sensory+neurons&quot;" alt="Search on this keyword" rel="nofollow">sensory neurons</a></li><li><a href="/action/doSearch?AllField=&quot;homeostasis&quot;" alt="Search on this keyword" rel="nofollow">homeostasis</a></li><li><a href="/action/doSearch?AllField=&quot;transcription&quot;" alt="Search on this keyword" rel="nofollow">transcription</a></li><li><a href="/action/doSearch?AllField=&quot;adaptation&quot;" alt="Search on this keyword" rel="nofollow">adaptation</a></li><li><a href="/action/doSearch?AllField=&quot;single-cell+RNA+sequencing&quot;" alt="Search on this keyword" rel="nofollow">single-cell RNA sequencing</a></li><li><a href="/action/doSearch?AllField=&quot;gene+expression+programs&quot;" alt="Search on this keyword" rel="nofollow">gene expression programs</a></li><li><a href="/action/doSearch?AllField=&quot;odor+coding&quot;" alt="Search on this keyword" rel="nofollow">odor coding</a></li><li><a href="/action/doSearch?AllField=&quot;functional+imaging&quot;" alt="Search on this keyword" rel="nofollow">functional imaging</a></li><li><a href="/action/doSearch?AllField=&quot;odorant+receptor&quot;" alt="Search on this keyword" rel="nofollow">odorant receptor</a></li><li><a href="/action/doSearch?AllField=&quot;Act-seq&quot;" alt="Search on this keyword" rel="nofollow">Act-seq</a></li></ol></section><div class="core-linked-content"></div><div class="core-linked-content"></div><section id="bodymatter" data-extent="bodymatter" property="articleBody" typeof="Text"><div class="core-container"><section id="sec-1"><h2>Introduction</h2><div id="p0045" role="paragraph">Sensory adaptation allows neurons and networks to minimize responses to background stimuli, thereby building more efficient neural codes that emphasize surprising or novel information (<span class="dropBlock reference-citations"><a id="body-ref-sref3" href="#" role="doc-biblioref" data-xml-rid="bib3" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref3" aria-controls="sref3">Attneave, 1954</a><div class="dropBlock__holder" data-db-target-of="sref3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">3.</div><div>Attneave, F.</div><div><strong>Some informational aspects of visual perception</strong></div><div><em>Psychol. Rev.</em> 1954; <strong>61</strong>:183-193</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1037/h0054663" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-0000728115" target="_blank" aria-label="Open Scopus in new tab">Scopus (1752)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/13167245/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1037%2Fh0054663&amp;pmid=13167245" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref5" href="#" role="doc-biblioref" data-xml-rid="bib5" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref5" aria-controls="sref5">Barlow, 1961</a><div class="dropBlock__holder" data-db-target-of="sref5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">5.</div><div>Barlow, H.</div><div><strong>Possible principles underlying the transformation of sensory messages</strong></div><div><div>Rosenblith, W.A. (Editor)</div></div><div><strong>Sensory Communication</strong></div><div>MIT press, 1961; 217-234</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=H.BarlowPossible+principles+underlying+the+transformation+of+sensory+messagesW.A.RosenblithSensory+Communication1961MIT+press217234" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref6-1" href="#" role="doc-biblioref" data-xml-rid="bib6" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref6-1" aria-controls="sref6-1">Benda, 2021</a><div class="dropBlock__holder" data-db-target-of="sref6-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">6.</div><div>Benda, J.</div><div><strong>Neural adaptation</strong></div><div><em>Curr. Biol.</em> 2021; <strong>31</strong>:R110-R116</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=10.1016%2Fj.cub.2020.11.054&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=10.1016%2Fj.cub.2020.11.054&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-85100374082" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33561404/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cub.2020.11.054&amp;pmid=33561404" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref103" href="#" role="doc-biblioref" data-xml-rid="bib103" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref103" aria-controls="sref103">Weber et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref103"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">106.</div><div>Weber, A.I. ∙ Krishnamurthy, K. ∙ Fairhall, A.L.</div><div><strong>Coding principles in adaptation</strong></div><div><em>Annu. Rev. Vis. Sci.</em> 2019; <strong>5</strong>:427-449</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1146/annurev-vision-091718-014818" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_106_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-85072266526" target="_blank" aria-label="Open Scopus in new tab">Scopus (18)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31283447/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1146%2Fannurev-vision-091718-014818&amp;pmid=31283447" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Rapid (milliseconds to seconds) sensory adaptation via post-translational mechanisms occurs in most sensory neurons, and it plays an important role in shaping peripheral responses to highly dynamic stimuli (<span class="dropBlock reference-citations"><a id="body-ref-sref49-1" href="#" role="doc-biblioref" data-xml-rid="bib49" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref49-1" aria-controls="sref49-1">Kostal et&nbsp;al., 2008</a><div class="dropBlock__holder" data-db-target-of="sref49-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">51.</div><div>Kostal, L. ∙ Lansky, P. ∙ Rospars, J.-P.</div><div><strong>Efficient olfactory coding in the pheromone receptor neuron of a moth</strong></div><div><em>PLoS Comput. Biol.</em> 2008; <strong>4</strong>:e1000053</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pcbi.1000053" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_51_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-43249110822" target="_blank" aria-label="Open Scopus in new tab">Scopus (29)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/18437217/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pcbi.1000053&amp;pmid=18437217" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref60-1" href="#" role="doc-biblioref" data-xml-rid="bib60" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref60-1" aria-controls="sref60-1">Martelli and Storace, 2021</a><div class="dropBlock__holder" data-db-target-of="sref60-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">62.</div><div>Martelli, C. ∙ Storace, D.A.</div><div><strong>Stimulus driven functional transformations in the early olfactory system</strong></div><div><em>Front. Cell. Neurosci.</em> 2021; <strong>15</strong>:684742</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.3389/fncel.2021.684742" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-85113135443" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/34413724/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.3389%2Ffncel.2021.684742&amp;pmid=34413724" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref67-1" href="#" role="doc-biblioref" data-xml-rid="bib67" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref67-1" aria-controls="sref67-1">Moore, 1994</a><div class="dropBlock__holder" data-db-target-of="sref67-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">69.</div><div>Moore, P.A.</div><div><strong>A model of the role of adaptation and disadaptation in olfactory receptor neurons: Implications for the coding of temporal and intensity patterns in odor signals</strong></div><div><em>Chem. Senses.</em> 1994; <strong>19</strong>:71-86</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/chemse/19.1.71" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-0028153852" target="_blank" aria-label="Open Scopus in new tab">Scopus (42)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/8055260/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fchemse%2F19.1.71&amp;pmid=8055260" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). However, it is not clear how sensory neurons adapt at the longer timescales over which animals traverse different environments—and therefore encounter different background stimuli—as is common during a typical circadian cycle.</div><div id="p0050" role="paragraph">Activity-dependent transcription evolves over minutes to hours and therefore in principle could support longer-term adaptation (<span class="dropBlock reference-citations"><a id="body-ref-sref115" href="#" role="doc-biblioref" data-xml-rid="bib115" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref115" aria-controls="sref115">Yap and Greenberg, 2018</a><div class="dropBlock__holder" data-db-target-of="sref115"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">118.</div><div>Yap, E.-L. ∙ Greenberg, M.E.</div><div><strong>Activity-regulated transcription: Bridging the gap between neural activity and behavior</strong></div><div><em>Neuron.</em> 2018; <strong>100</strong>:330-348</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_118_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=10.1016%2Fj.neuron.2018.10.013&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_118_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=10.1016%2Fj.neuron.2018.10.013&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_118_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-85054841240" target="_blank" aria-label="Open Scopus in new tab">Scopus (137)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30359600/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.neuron.2018.10.013&amp;pmid=30359600" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). For example, mature brain neurons are thought to use the regulated transcription of synaptic proteins and ion channels to homeostatically maintain their firing rates within a narrow target range (<span class="dropBlock reference-citations"><a id="body-ref-sref21-1" href="#" role="doc-biblioref" data-xml-rid="bib21" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref21-1" aria-controls="sref21-1">Davis, 2006</a><div class="dropBlock__holder" data-db-target-of="sref21-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">21.</div><div>Davis, G.W.</div><div><strong>Homeostatic control of neural activity: From phenomenology to molecular design</strong></div><div><em>Annu. Rev. Neurosci.</em> 2006; <strong>29</strong>:307-323</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1146/annurev.neuro.28.061604.135751" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-33745734146" target="_blank" aria-label="Open Scopus in new tab">Scopus (400)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16776588/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1146%2Fannurev.neuro.28.061604.135751&amp;pmid=16776588" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>, <span class="dropBlock reference-citations"><a id="body-ref-sref22-1" href="#" role="doc-biblioref" data-xml-rid="bib22" href-manipulated="true" aria-label="Reference 2013" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref22-1" aria-controls="sref22-1">2013</a><div class="dropBlock__holder" data-db-target-of="sref22-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">22.</div><div>Davis, G.W.</div><div><strong>Homeostatic signaling and the stabilization of neural function</strong></div><div><em>Neuron.</em> 2013; <strong>80</strong>:718-728</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=10.1016%2Fj.neuron.2013.09.044&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=10.1016%2Fj.neuron.2013.09.044&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-84887011923" target="_blank" aria-label="Open Scopus in new tab">Scopus (165)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24183022/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.neuron.2013.09.044&amp;pmid=24183022" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref58-1" href="#" role="doc-biblioref" data-xml-rid="bib58" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref58-1" aria-controls="sref58-1">Marder, 2011</a><div class="dropBlock__holder" data-db-target-of="sref58-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">60.</div><div>Marder, E.</div><div><strong>Variability, compensation, and modulation in neurons and circuits</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2011; <strong>108</strong>:15542-15548</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1010674108" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-80053141839" target="_blank" aria-label="Open Scopus in new tab">Scopus (197)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21383190/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1010674108&amp;pmid=21383190" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref59-1" href="#" role="doc-biblioref" data-xml-rid="bib59" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref59-1" aria-controls="sref59-1">Marder and Goaillard, 2006</a><div class="dropBlock__holder" data-db-target-of="sref59-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">61.</div><div>Marder, E. ∙ Goaillard, J.-M.</div><div><strong>Variability, compensation and homeostasis in neuron and network function</strong></div><div><em>Nat. Rev. Neurosci.</em> 2006; <strong>7</strong>:563-574</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nrn1949" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-33745712893" target="_blank" aria-label="Open Scopus in new tab">Scopus (723)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16791145/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnrn1949&amp;pmid=16791145" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref93-1" href="#" role="doc-biblioref" data-xml-rid="bib93" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref93-1" aria-controls="sref93-1">Turrigiano, 2011</a><div class="dropBlock__holder" data-db-target-of="sref93-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">96.</div><div>Turrigiano, G.</div><div><strong>Too many cooks? Intrinsic and synaptic homeostatic mechanisms in cortical circuit refinement</strong></div><div><em>Annu. Rev. Neurosci.</em> 2011; <strong>34</strong>:89-103</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1146/annurev-neuro-060909-153238" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_96_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-79959889965" target="_blank" aria-label="Open Scopus in new tab">Scopus (425)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21438687/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1146%2Fannurev-neuro-060909-153238&amp;pmid=21438687" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref91-1" href="#" role="doc-biblioref" data-xml-rid="bib91" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref91-1" aria-controls="sref91-1">Turrigiano, 1999</a><div class="dropBlock__holder" data-db-target-of="sref91-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">94.</div><div>Turrigiano, G.G.</div><div><strong>Homeostatic plasticity in neuronal networks: The more things change, the more they stay the same</strong></div><div><em>Trends Neurosci.</em> 1999; <strong>22</strong>:221-227</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_94_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=10.1016%2FS0166-2236%2898%2901341-1&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_94_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=10.1016%2FS0166-2236%2898%2901341-1&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_94_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-0033135312" target="_blank" aria-label="Open Scopus in new tab">Scopus (585)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/10322495/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0166-2236%2898%2901341-1&amp;pmid=10322495" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref94-1" href="#" role="doc-biblioref" data-xml-rid="bib94" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref94-1" aria-controls="sref94-1">Turrigiano, 2017</a><div class="dropBlock__holder" data-db-target-of="sref94-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">97.</div><div>Turrigiano, G.G.</div><div><strong>The dialectic of Hebb and homeostasis</strong></div><div><em>Philos. Trans. R. Soc. Lond. B Biol. Sci.</em> 2017; <strong>372</strong>:20160258</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1098/rstb.2016.0258" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_97_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-85009958893" target="_blank" aria-label="Open Scopus in new tab">Scopus (97)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28093556/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1098%2Frstb.2016.0258&amp;pmid=28093556" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). If similar transcriptional mechanisms underlie adaptation in the periphery, then sensory neurons of a given type would be expected to occupy a range of transcriptional states, with the specific state adopted by any given neuron both reflecting its prior activity history and predicting its future responses to sensory inputs (<span class="dropBlock reference-citations"><a id="body-ref-sref95" href="#" role="doc-biblioref" data-xml-rid="bib95" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref95" aria-controls="sref95">Tyssowski and Gray, 2019</a><div class="dropBlock__holder" data-db-target-of="sref95"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">98.</div><div>Tyssowski, K.M. ∙ Gray, J.M.</div><div><strong>The neuronal stimulation-transcription coupling map</strong></div><div><em>Curr. Opin. Neurobiol.</em> 2019; <strong>59</strong>:87-94</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.conb.2019.05.001" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_98_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-85066322540" target="_blank" aria-label="Open Scopus in new tab">Scopus (7)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31163285/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.conb.2019.05.001&amp;pmid=31163285" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). However, the relationship between activity-dependent gene regulation and functional adaptation remains elusive, in part because it has not yet been shown in either peripheral or central neurons that analog changes in activity yield proportional, bidirectional, and systematic changes in the expression of genes that tune neural function (<span class="dropBlock reference-citations"><a id="body-ref-sref22-2" href="#" role="doc-biblioref" data-xml-rid="bib22" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref22-2" aria-controls="sref22-2">Davis, 2013</a><div class="dropBlock__holder" data-db-target-of="sref22-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">22.</div><div>Davis, G.W.</div><div><strong>Homeostatic signaling and the stabilization of neural function</strong></div><div><em>Neuron.</em> 2013; <strong>80</strong>:718-728</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=10.1016%2Fj.neuron.2013.09.044&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=10.1016%2Fj.neuron.2013.09.044&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-84887011923" target="_blank" aria-label="Open Scopus in new tab">Scopus (165)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24183022/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.neuron.2013.09.044&amp;pmid=24183022" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="p0055" role="paragraph">There are ∼1,000 different olfactory sensory neuron (OSN) subtypes in the mouse, each of which can be identified across individuals based upon its expression of a single odorant receptor (OR) (<span class="dropBlock reference-citations"><a id="body-ref-sref65-1" href="#" role="doc-biblioref" data-xml-rid="bib65" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref65-1" aria-controls="sref65-1">Monahan and Lomvardas, 2015</a><div class="dropBlock__holder" data-db-target-of="sref65-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">67.</div><div>Monahan, K. ∙ Lomvardas, S.</div><div><strong>Monoallelic expression of olfactory receptors</strong></div><div><em>Annu. Rev. Cell Dev. Biol.</em> 2015; <strong>31</strong>:721-740</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1146/annurev-cellbio-100814-125308" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_67_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-84947294591" target="_blank" aria-label="Open Scopus in new tab">Scopus (71)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26359778/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1146%2Fannurev-cellbio-100814-125308&amp;pmid=26359778" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). ORs are G protein-coupled receptors that transduce odor binding into an influx of calcium, which leads to action potentials. The expressed OR endows each OSN with a specific level of neural activity that depends on the odors present within a given environment. Although post-translational mechanisms sculpt OSN responses to rapidly fluctuating odor plumes, at longer timescales it is thought that OSNs faithfully report odor-receptor interactions to the brain. For example, OSN responses to odors (as assessed by functional imaging of OSN axons in the olfactory bulb, the first waystation for odor information in the brain) appear similar in individual mice tested at different times, across different mice, and in mice before and after training to associate a test odorant with a reward (<span class="dropBlock reference-citations"><a id="body-ref-sref9" href="#" role="doc-biblioref" data-xml-rid="bib9" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref9" aria-controls="sref9">Bozza et&nbsp;al., 2004</a><div class="dropBlock__holder" data-db-target-of="sref9"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">9.</div><div>Bozza, T. ∙ McGann, J.P. ∙ Mombaerts, P. ...</div><div><strong>In&nbsp;vivo imaging of neuronal activity by targeted expression of a genetically encoded probe in the mouse</strong></div><div><em>Neuron.</em> 2004; <strong>42</strong>:9-21</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=10.1016%2FS0896-6273%2804%2900144-8&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=10.1016%2FS0896-6273%2804%2900144-8&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-1842505551" target="_blank" aria-label="Open Scopus in new tab">Scopus (265)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15066261/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0896-6273%2804%2900144-8&amp;pmid=15066261" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref17" href="#" role="doc-biblioref" data-xml-rid="bib17" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref17" aria-controls="sref17">Chu et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref17"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">17.</div><div>Chu, M.W. ∙ Li, W.L. ∙ Komiyama, T.</div><div><strong>Lack of pattern separation in sensory inputs to the olfactory bulb during perceptual learning</strong></div><div><em>eNeuro.</em> 2017; <strong>4</strong></div><div>ENEURO.0287-17.2017</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1523/ENEURO.0287-17.2017" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-85032145385" target="_blank" aria-label="Open Scopus in new tab">Scopus (6)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28955724/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1523%2FENEURO.0287-17.2017&amp;pmid=28955724" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref46" href="#" role="doc-biblioref" data-xml-rid="bib46" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref46" aria-controls="sref46">Kato et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref46"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">47.</div><div>Kato, H.K. ∙ Chu, M.W. ∙ Isaacson, J.S. ...</div><div><strong>Dynamic sensory representations in the olfactory bulb: Modulation by wakefulness and experience</strong></div><div><em>Neuron.</em> 2012; <strong>76</strong>:962-975</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=10.1016%2Fj.neuron.2012.09.037&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=10.1016%2Fj.neuron.2012.09.037&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-84872745558" target="_blank" aria-label="Open Scopus in new tab">Scopus (150)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23217744/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.neuron.2012.09.037&amp;pmid=23217744" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref79" href="#" role="doc-biblioref" data-xml-rid="bib79" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref79" aria-controls="sref79">Rubin and Katz, 1999</a><div class="dropBlock__holder" data-db-target-of="sref79"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">81.</div><div>Rubin, B.D. ∙ Katz, L.C.</div><div><strong>Optical imaging of odorant representations in the mammalian olfactory bulb</strong></div><div><em>Neuron.</em> 1999; <strong>23</strong>:499-511</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=10.1016%2FS0896-6273%2800%2980803-X&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=10.1016%2FS0896-6273%2800%2980803-X&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-0033166536" target="_blank" aria-label="Open Scopus in new tab">Scopus (536)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/10433262/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0896-6273%2800%2980803-X&amp;pmid=10433262" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref85" href="#" role="doc-biblioref" data-xml-rid="bib85" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref85" aria-controls="sref85">Soucy&nbsp;et&nbsp;al., 2009</a><div class="dropBlock__holder" data-db-target-of="sref85"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">87.</div><div>Soucy, E.R. ∙ Albeanu, D.F. ∙ Fantana, A.L. ...</div><div><strong>Precision and diversity in an odor map on the olfactory bulb</strong></div><div><em>Nat. Neurosci.</em> 2009; <strong>12</strong>:210-220</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nn.2262" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_87_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-58849109461" target="_blank" aria-label="Open Scopus in new tab">Scopus (217)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19151709/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnn.2262&amp;pmid=19151709" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref86" href="#" role="doc-biblioref" data-xml-rid="bib86" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref86" aria-controls="sref86">Spors and Grinvald, 2002</a><div class="dropBlock__holder" data-db-target-of="sref86"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">88.</div><div>Spors, H. ∙ Grinvald, A.</div><div><strong>Spatio-temporal dynamics of odor representations in the mammalian olfactory bulb</strong></div><div><em>Neuron.</em> 2002; <strong>34</strong>:301-315</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=10.1016%2FS0896-6273%2802%2900644-X&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=10.1016%2FS0896-6273%2802%2900644-X&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-0037061682" target="_blank" aria-label="Open Scopus in new tab">Scopus (262)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11970871/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0896-6273%2802%2900644-X&amp;pmid=11970871" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref98" href="#" role="doc-biblioref" data-xml-rid="bib98" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref98" aria-controls="sref98">Wachowiak and Cohen, 2001</a><div class="dropBlock__holder" data-db-target-of="sref98"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">101.</div><div>Wachowiak, M. ∙ Cohen, L.B.</div><div><strong>Representation of odorants by receptor neuron input to the mouse olfactory bulb</strong></div><div><em>Neuron.</em> 2001; <strong>32</strong>:723-735</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_101_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=10.1016%2FS0896-6273%2801%2900506-2&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_101_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=10.1016%2FS0896-6273%2801%2900506-2&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_101_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-0035923747" target="_blank" aria-label="Open Scopus in new tab">Scopus (370)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11719211/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0896-6273%2801%2900506-2&amp;pmid=11719211" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). In contrast, olfactory bulb and cortical neurons adapt to the repeated presentation of an odor on timescales of minutes (<span class="dropBlock reference-citations"><a id="body-ref-sref7" href="#" role="doc-biblioref" data-xml-rid="bib7" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref7" aria-controls="sref7">Best et&nbsp;al., 2005</a><div class="dropBlock__holder" data-db-target-of="sref7"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">7.</div><div>Best, A.R. ∙ Thompson, J.V. ∙ Fletcher, M.L. ...</div><div><strong>Cortical metabotropic glutamate receptors contribute to habituation of a simple odor-evoked behavior</strong></div><div><em>J.&nbsp;Neurosci.</em> 2005; <strong>25</strong>:2513-2517</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1523/JNEUROSCI.5298-04.2005" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-14944364759" target="_blank" aria-label="Open Scopus in new tab">Scopus (48)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15758159/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1523%2FJNEUROSCI.5298-04.2005&amp;pmid=15758159" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref18-1" href="#" role="doc-biblioref" data-xml-rid="bib18" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref18-1" aria-controls="sref18-1">Cleland and Linster, 2005</a><div class="dropBlock__holder" data-db-target-of="sref18-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">18.</div><div>Cleland, T.A. ∙ Linster, C.</div><div><strong>Computation in the olfactory system</strong></div><div><em>Chem. Senses.</em> 2005; <strong>30</strong>:801-813</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/chemse/bji072" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-27944461858" target="_blank" aria-label="Open Scopus in new tab">Scopus (101)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16267161/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fchemse%2Fbji072&amp;pmid=16267161" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref63" href="#" role="doc-biblioref" data-xml-rid="bib63" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref63" aria-controls="sref63">McNamara et&nbsp;al., 2008</a><div class="dropBlock__holder" data-db-target-of="sref63"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">65.</div><div>McNamara, A.M. ∙ Magidson, P.D. ∙ Linster, C. ...</div><div><strong>Distinct neural mechanisms mediate olfactory memory formation at different timescales</strong></div><div><em>Learn. Mem.</em> 2008; <strong>15</strong>:117-125</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/lm.785608" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-40949119862" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/18299438/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2Flm.785608&amp;pmid=18299438" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref106" href="#" role="doc-biblioref" data-xml-rid="bib106" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref106" aria-controls="sref106">Wilson, 1998</a><div class="dropBlock__holder" data-db-target-of="sref106"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">109.</div><div>Wilson, D.A.</div><div><strong>Habituation of odor responses in the rat anterior piriform cortex</strong></div><div><em>J.&nbsp;Neurophysiol.</em> 1998; <strong>79</strong>:1425-1440</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1152/jn.1998.79.3.1425" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_109_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-0031952787" target="_blank" aria-label="Open Scopus in new tab">Scopus (181)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/9497422/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1152%2Fjn.1998.79.3.1425&amp;pmid=9497422" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref107" href="#" role="doc-biblioref" data-xml-rid="bib107" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref107" aria-controls="sref107">Wilson, 2000</a><div class="dropBlock__holder" data-db-target-of="sref107"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">110.</div><div>Wilson, D.A.</div><div><strong>Comparison of odor receptive field plasticity in the rat olfactory bulb and anterior piriform cortex</strong></div><div><em>J.&nbsp;Neurophysiol.</em> 2000; <strong>84</strong>:3036-3042</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1152/jn.2000.84.6.3036" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11110830/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1152%2Fjn.2000.84.6.3036&amp;pmid=11110830" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref114" href="#" role="doc-biblioref" data-xml-rid="bib114" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref114" aria-controls="sref114">Yadon and Wilson, 2005</a><div class="dropBlock__holder" data-db-target-of="sref114"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">117.</div><div>Yadon, C.A. ∙ Wilson, D.A.</div><div><strong>The role of metabotropic glutamate receptors and cortical adaptation in habituation of odor-guided behavior</strong></div><div><em>Learn. Mem.</em> 2005; <strong>12</strong>:601-605</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/lm.41405" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_117_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-28744459570" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16322361/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2Flm.41405&amp;pmid=16322361" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). These findings suggest that ORs confer stable odor response properties upon OSNs, while the brain separates salient odors from background; this idea has pervasively influenced theoretical and computational models of odor coding (<span class="dropBlock reference-citations"><a id="body-ref-sref11" href="#" role="doc-biblioref" data-xml-rid="bib11" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref11" aria-controls="sref11">Brann and Datta, 2020</a><div class="dropBlock__holder" data-db-target-of="sref11"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">11.</div><div>Brann, D.H. ∙ Datta, S.R.</div><div><strong>Finding the brain in the nose</strong></div><div><em>Annu. Rev. Neurosci.</em> 2020; <strong>43</strong>:277-295</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1146/annurev-neuro-102119-103452" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-85087696971" target="_blank" aria-label="Open Scopus in new tab">Scopus (3)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32640927/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1146%2Fannurev-neuro-102119-103452&amp;pmid=32640927" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref16" href="#" role="doc-biblioref" data-xml-rid="bib16" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref16" aria-controls="sref16">Chong et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref16"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">16.</div><div>Chong, E. ∙ Moroni, M. ∙ Wilson, C. ...</div><div><strong>Manipulating synthetic optogenetic odors reveals the coding logic of olfactory perception</strong></div><div><em>Science.</em> 2020; <strong>368</strong>:eaba2357</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aba2357" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-85086692136" target="_blank" aria-label="Open Scopus in new tab">Scopus (17)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32554567/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aba2357&amp;pmid=32554567" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref35" href="#" role="doc-biblioref" data-xml-rid="bib35" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref35" aria-controls="sref35">Grabska-Barwińska et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref35"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">35.</div><div>Grabska-Barwińska, A. ∙ Barthelmé, S. ∙ Beck, J. ...</div><div><strong>A probabilistic approach to demixing odors</strong></div><div><em>Nat. Neurosci.</em> 2017; <strong>20</strong>:98-106</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nn.4444" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27918530/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnn.4444&amp;pmid=27918530" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref38" href="#" role="doc-biblioref" data-xml-rid="bib38" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref38" aria-controls="sref38">Hopfield, 1991</a><div class="dropBlock__holder" data-db-target-of="sref38"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">38.</div><div>Hopfield, J.J.</div><div><strong>Olfactory computation and object perception</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 1991; <strong>88</strong>:6462-6466</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.88.15.6462" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_38_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-0026003099" target="_blank" aria-label="Open Scopus in new tab">Scopus (93)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/1862075/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.88.15.6462&amp;pmid=1862075" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref56" href="#" role="doc-biblioref" data-xml-rid="bib56" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref56" aria-controls="sref56">Litwin-Kumar et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref56"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">58.</div><div>Litwin-Kumar, A. ∙ Harris, K.D. ∙ Axel, R. ...</div><div><strong>Optimal degrees of synaptic connectivity</strong></div><div><em>Neuron.</em> 2017; <strong>93</strong>:1153-1164.e7</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=10.1016%2Fj.neuron.2017.01.030&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=10.1016%2Fj.neuron.2017.01.030&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-85012934211" target="_blank" aria-label="Open Scopus in new tab">Scopus (97)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28215558/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.neuron.2017.01.030&amp;pmid=28215558" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref82" href="#" role="doc-biblioref" data-xml-rid="bib82" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref82" aria-controls="sref82">Schaffer et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref82"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">84.</div><div>Schaffer, E.S. ∙ Stettler, D.D. ∙ Kato, D. ...</div><div><strong>Odor perception on the two sides of the brain: Consistency despite randomness</strong></div><div><em>Neuron.</em> 2018; <strong>98</strong>:736-742.e3</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_84_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=10.1016%2Fj.neuron.2018.04.004&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_84_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=10.1016%2Fj.neuron.2018.04.004&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_84_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-85046149960" target="_blank" aria-label="Open Scopus in new tab">Scopus (21)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29706585/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.neuron.2018.04.004&amp;pmid=29706585" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref89" href="#" role="doc-biblioref" data-xml-rid="bib89" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref89" aria-controls="sref89">Teşileanu et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref89"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">92.</div><div>Teşileanu, T. ∙ Cocco, S. ∙ Monasson, R. ...</div><div><strong>Adaptation of olfactory receptor abundances for efficient coding</strong></div><div><em>eLife.</em> 2019; <strong>8</strong>:308</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.7554/eLife.39279" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_92_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-85062409446" target="_blank" aria-label="Open Scopus in new tab">Scopus (8)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.7554%2FeLife.39279" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref109" href="#" role="doc-biblioref" data-xml-rid="bib109" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref109" aria-controls="sref109">Wilson et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref109"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">112.</div><div>Wilson, C.D. ∙ Serrano, G.O. ∙ Koulakov, A.A. ...</div><div><strong>A primacy code for odor identity</strong></div><div><em>Nat. Commun.</em> 2017; <strong>8</strong>:1477</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41467-017-01432-4" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_112_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-85034246616" target="_blank" aria-label="Open Scopus in new tab">Scopus (56)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29133907/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41467-017-01432-4&amp;pmid=29133907" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref119" href="#" role="doc-biblioref" data-xml-rid="bib119" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref119" aria-controls="sref119">Zwicker et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref119"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">122.</div><div>Zwicker, D. ∙ Murugan, A. ∙ Brenner, M.P.</div><div><strong>Receptor arrays optimized for natural odor statistics</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2016; <strong>113</strong>:5570-5575</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1600357113" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_122_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-84969872374" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27102871/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1600357113&amp;pmid=27102871" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="p0060" role="paragraph">Consistent with this view, transcriptional analyses suggest that the genes expressed by mature OSNs are largely similar across subtypes, with the exception of the ORs themselves, a handful of genes related to dorsoventral position in the olfactory epithelium, and ∼10 axon guidance genes that enable glomerular targeting in the olfactory bulb (<span class="dropBlock reference-citations"><a id="body-ref-sref31" href="#" role="doc-biblioref" data-xml-rid="bib31" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref31" aria-controls="sref31">Fletcher et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref31"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">31.</div><div>Fletcher, R.B. ∙ Das, D. ∙ Gadye, L. ...</div><div><strong>Deconstructing olfactory stem cell trajectories at single-cell resolution</strong></div><div><em>Cell Stem Cell.</em> 2017; <strong>20</strong>:817-830.e8</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=10.1016%2Fj.stem.2017.04.003&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=10.1016%2Fj.stem.2017.04.003&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-85019062024" target="_blank" aria-label="Open Scopus in new tab">Scopus (84)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28506465/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.stem.2017.04.003&amp;pmid=28506465" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref37-1" href="#" role="doc-biblioref" data-xml-rid="bib37" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref37-1" aria-controls="sref37-1">Hanchate et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref37-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">37.</div><div>Hanchate, N.K. ∙ Kondoh, K. ∙ Lu, Z. ...</div><div><strong>Single-cell transcriptomics reveals receptor transformations during olfactory neurogenesis</strong></div><div><em>Science.</em> 2015; <strong>350</strong>:1251-1255</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aad2456" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-84949035291" target="_blank" aria-label="Open Scopus in new tab">Scopus (116)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26541607/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aad2456&amp;pmid=26541607" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref68-1" href="#" role="doc-biblioref" data-xml-rid="bib68" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref68-1" aria-controls="sref68-1">Mori and Sakano, 2011</a><div class="dropBlock__holder" data-db-target-of="sref68-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">70.</div><div>Mori, K. ∙ Sakano, H.</div><div><strong>How is the olfactory map formed and interpreted in the mammalian brain?</strong></div><div><em>Annu. Rev. Neurosci.</em> 2011; <strong>34</strong>:467-499</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1146/annurev-neuro-112210-112917" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-79959891730" target="_blank" aria-label="Open Scopus in new tab">Scopus (227)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21469960/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1146%2Fannurev-neuro-112210-112917&amp;pmid=21469960" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref70" href="#" role="doc-biblioref" data-xml-rid="bib70" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref70" aria-controls="sref70">Nakashima et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref70"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">72.</div><div>Nakashima, A. ∙ Takeuchi, H. ∙ Imai, T. ...</div><div><strong>Agonist-independent GPCR activity regulates anterior-posterior targeting of olfactory sensory neurons</strong></div><div><em>Cell.</em> 2013; <strong>154</strong>:1314-1325</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=10.1016%2Fj.cell.2013.08.033&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=10.1016%2Fj.cell.2013.08.033&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-84884249630" target="_blank" aria-label="Open Scopus in new tab">Scopus (92)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24034253/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2013.08.033&amp;pmid=24034253" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref83" href="#" role="doc-biblioref" data-xml-rid="bib83" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref83" aria-controls="sref83">Scholz et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref83"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">85.</div><div>Scholz, P. ∙ Kalbe, B. ∙ Jansen, F. ...</div><div><strong>Transcriptome analysis of murine olfactory sensory neurons during development using single cell RNA-seq</strong></div><div><em>Chem. Senses.</em> 2016; <strong>41</strong>:313-323</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/chemse/bjw003" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-84964898686" target="_blank" aria-label="Open Scopus in new tab">Scopus (25)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26839357/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fchemse%2Fbjw003&amp;pmid=26839357" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref96" href="#" role="doc-biblioref" data-xml-rid="bib96" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref96" aria-controls="sref96">Vihani et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref96"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">99.</div><div>Vihani, A. ∙ Hu, X.S. ∙ Gundala, S. ...</div><div><strong>Semiochemical responsive olfactory sensory neurons are sexually dimorphic and plastic</strong></div><div><em>eLife.</em> 2020; <strong>9</strong>:e54501</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.7554/eLife.54501" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_99_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-85098288126" target="_blank" aria-label="Open Scopus in new tab">Scopus (4)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33231170/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.7554%2FeLife.54501&amp;pmid=33231170" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref112" href="#" role="doc-biblioref" data-xml-rid="bib112" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref112" aria-controls="sref112">Wu et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref112"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">115.</div><div>Wu, Y. ∙ Ma, L. ∙ Duyck, K. ...</div><div><strong>A population of navigator neurons is essential for olfactory map formation during the critical period</strong></div><div><em>Neuron.</em> 2018; <strong>100</strong>:1066-1082.e6</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_115_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=10.1016%2Fj.neuron.2018.09.051&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_115_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=10.1016%2Fj.neuron.2018.09.051&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_115_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-85059320843" target="_blank" aria-label="Open Scopus in new tab">Scopus (9)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30482691/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.neuron.2018.09.051&amp;pmid=30482691" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). This homogeneity suggests that individual OSN subtypes, which vary widely in their level of activity within a given environment, do not adapt their odor responses by actively modulating gene expression. In contrast, bulk RNA and activity measurements that pool across all OSNs in mice have hinted that OSNs may use transcription to compensate for the chronic absence or presence of activity, although results and interpretations have varied across experiments (<span class="dropBlock reference-citations"><a id="body-ref-sref4" href="#" role="doc-biblioref" data-xml-rid="bib4" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref4" aria-controls="sref4">Barber and Coppola, 2015</a><div class="dropBlock__holder" data-db-target-of="sref4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">4.</div><div>Barber, C.N. ∙ Coppola, D.M.</div><div><strong>Compensatory plasticity in the olfactory epithelium: Age, timing, and reversibility</strong></div><div><em>J.&nbsp;Neurophysiol.</em> 2015; <strong>114</strong>:2023-2032</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1152/jn.00076.2015" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-84943175800" target="_blank" aria-label="Open Scopus in new tab">Scopus (4)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26269548/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1152%2Fjn.00076.2015&amp;pmid=26269548" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref14" href="#" role="doc-biblioref" data-xml-rid="bib14" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref14" aria-controls="sref14">Cadiou et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref14"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">14.</div><div>Cadiou, H. ∙ Aoudé, I. ∙ Tazir, B. ...</div><div><strong>Postnatal odorant exposure induces peripheral olfactory plasticity at the cellular level</strong></div><div><em>J.&nbsp;Neurosci.</em> 2014; <strong>34</strong>:4857-4870</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1523/JNEUROSCI.0688-13.2014" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-84899413440" target="_blank" aria-label="Open Scopus in new tab">Scopus (43)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24695705/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1523%2FJNEUROSCI.0688-13.2014&amp;pmid=24695705" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref19" href="#" role="doc-biblioref" data-xml-rid="bib19" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref19" aria-controls="sref19">Coppola and Waggener, 2012</a><div class="dropBlock__holder" data-db-target-of="sref19"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">19.</div><div>Coppola, D.M. ∙ Waggener, C.T.</div><div><strong>The effects of unilateral naris occlusion on gene expression profiles in mouse olfactory mucosa</strong></div><div><em>J.&nbsp;Mol. Neurosci.</em> 2012; <strong>47</strong>:604-618</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/s12031-011-9690-4" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-84863876802" target="_blank" aria-label="Open Scopus in new tab">Scopus (25)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22187364/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2Fs12031-011-9690-4&amp;pmid=22187364" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref20" href="#" role="doc-biblioref" data-xml-rid="bib20" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref20" aria-controls="sref20">Coppola et&nbsp;al., 2006</a><div class="dropBlock__holder" data-db-target-of="sref20"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">20.</div><div>Coppola, D.M. ∙ Waguespack, A.M. ∙ Reems, M.R. ...</div><div><strong>Naris occlusion alters transductory protein immunoreactivity in olfactory epithelium</strong></div><div><em>Histol. Histopathol.</em> 2006; <strong>21</strong>:487-501</div></div><div class="external-links"><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16493579/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?pmid=16493579" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref29-1" href="#" role="doc-biblioref" data-xml-rid="bib29" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref29-1" aria-controls="sref29-1">Fischl et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref29-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">29.</div><div>Fischl, A.M. ∙ Heron, P.M. ∙ Stromberg, A.J. ...</div><div><strong>Activity-dependent genes in mouse olfactory sensory neurons</strong></div><div><em>Chem. Senses.</em> 2014; <strong>39</strong>:439-449</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/chemse/bju015" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-84901414304" target="_blank" aria-label="Open Scopus in new tab">Scopus (13)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24692514/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fchemse%2Fbju015&amp;pmid=24692514" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref30" href="#" role="doc-biblioref" data-xml-rid="bib30" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref30" aria-controls="sref30">Fitzwater and Coppola, 2021</a><div class="dropBlock__holder" data-db-target-of="sref30"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">30.</div><div>Fitzwater, E. ∙ Coppola, D.M.</div><div><strong>Olfactory deprivation and enrichment: An identity of opposites?</strong></div><div><em>Chem. Senses.</em> 2021; <strong>46</strong>:bjaa071</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/chemse/bjaa071" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-85102212835" target="_blank" aria-label="Open Scopus in new tab">Scopus (1)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33103187/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fchemse%2Fbjaa071&amp;pmid=33103187" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref36" href="#" role="doc-biblioref" data-xml-rid="bib36" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref36" aria-controls="sref36">Hagendorf et&nbsp;al., 2009</a><div class="dropBlock__holder" data-db-target-of="sref36"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">36.</div><div>Hagendorf, S. ∙ Fluegge, D. ∙ Engelhardt, C. ...</div><div><strong>Homeostatic control of sensory output in basal vomeronasal neurons: activity-dependent expression of ether-à-go-go-related gene potassium channels</strong></div><div><em>J.&nbsp;Neurosci.</em> 2009; <strong>29</strong>:206-221</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1523/JNEUROSCI.3656-08.2009" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-58149397652" target="_blank" aria-label="Open Scopus in new tab">Scopus (24)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19129398/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1523%2FJNEUROSCI.3656-08.2009&amp;pmid=19129398" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref45-1" href="#" role="doc-biblioref" data-xml-rid="bib45" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref45-1" aria-controls="sref45-1">Kass et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref45-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">46.</div><div>Kass, M.D. ∙ Pottackal, J. ∙ Turkel, D.J. ...</div><div><strong>Changes in the neural representation of odorants after olfactory deprivation in the adult mouse olfactory bulb</strong></div><div><em>Chem. Senses.</em> 2013; <strong>38</strong>:77-89</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/chemse/bjs081" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-84871317604" target="_blank" aria-label="Open Scopus in new tab">Scopus (12)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23125347/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fchemse%2Fbjs081&amp;pmid=23125347" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref99" href="#" role="doc-biblioref" data-xml-rid="bib99" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref99" aria-controls="sref99">Waggener and Coppola, 2007</a><div class="dropBlock__holder" data-db-target-of="sref99"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">102.</div><div>Waggener, C.T. ∙ Coppola, D.M.</div><div><strong>Naris occlusion alters the electro-olfactogram: Evidence for compensatory plasticity in the olfactory system</strong></div><div><em>Neurosci. Lett.</em> 2007; <strong>427</strong>:112-116</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.neulet.2007.09.013" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_102_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-35448943484" target="_blank" aria-label="Open Scopus in new tab">Scopus (27)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17931777/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.neulet.2007.09.013&amp;pmid=17931777" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref100" href="#" role="doc-biblioref" data-xml-rid="bib100" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref100" aria-controls="sref100">Wang et&nbsp;al., 1993</a><div class="dropBlock__holder" data-db-target-of="sref100"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">103.</div><div>Wang, H.W. ∙ Wysocki, C.J. ∙ Gold, G.H.</div><div><strong>Induction of olfactory receptor sensitivity in mice</strong></div><div><em>Science.</em> 1993; <strong>260</strong>:998-1000</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.8493539" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_103_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-0027237421" target="_blank" aria-label="Open Scopus in new tab">Scopus (154)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/8493539/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.8493539&amp;pmid=8493539" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref101-1" href="#" role="doc-biblioref" data-xml-rid="bib101" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref101-1" aria-controls="sref101-1">Wang et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref101-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">104.</div><div>Wang, Q. ∙ Titlow, W.B. ∙ McClintock, D.A. ...</div><div><strong>Activity-dependent gene expression in the mammalian olfactory epithelium</strong></div><div><em>Chem. Senses.</em> 2017; <strong>42</strong>:611-624</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/chemse/bjx028" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_104_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-85030719241" target="_blank" aria-label="Open Scopus in new tab">Scopus (7)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28525560/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fchemse%2Fbjx028&amp;pmid=28525560" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Resolving these discrepancies—and ultimately testing the hypothesis that dynamic gene expression mediates long-term adaptation in sensory neurons—requires asking how odor-evoked activity influences gene expression within particular OSN subtypes; this level of specificity is essential to clarify whether OSN transcriptomes change in an OR- and environment-specific manner; whether these changes are limited to small numbers of genes (as appears to be the case in <i>C.&nbsp;elegans</i> [<span class="dropBlock reference-citations"><a id="body-ref-sref15" href="#" role="doc-biblioref" data-xml-rid="bib15" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref15" aria-controls="sref15">Cho et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref15"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">15.</div><div>Cho, C.E. ∙ Brueggemann, C. ∙ L’Etoile, N.D. ...</div><div><strong>Parallel encoding of sensory history and behavioral preference during <i>Caenorhabditis elegans</i> olfactory learning</strong></div><div><em>eLife.</em> 2016; <strong>5</strong>:e14000</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.7554/eLife.14000" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-84979501960" target="_blank" aria-label="Open Scopus in new tab">Scopus (26)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27383131/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.7554%2FeLife.14000&amp;pmid=27383131" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref43" href="#" role="doc-biblioref" data-xml-rid="bib43" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref43" aria-controls="sref43">Juang et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref43"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">43.</div><div>Juang, B.-T. ∙ Gu, C. ∙ Starnes, L. ...</div><div><strong>Endogenous nuclear RNAi mediates behavioral adaptation to odor</strong></div><div><em>Cell.</em> 2013; <strong>154</strong>:1010-1022</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=10.1016%2Fj.cell.2013.08.006&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=10.1016%2Fj.cell.2013.08.006&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-84883426407" target="_blank" aria-label="Open Scopus in new tab">Scopus (53)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23993094/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2013.08.006&amp;pmid=23993094" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref51" href="#" role="doc-biblioref" data-xml-rid="bib51" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref51" aria-controls="sref51">L’Etoile et&nbsp;al., 2002</a><div class="dropBlock__holder" data-db-target-of="sref51"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">53.</div><div>L’Etoile, N.D. ∙ Coburn, C.M. ∙ Eastham, J. ...</div><div><strong>The cyclic GMP-dependent protein kinase EGL-4 regulates olfactory adaptation in <i>C.&nbsp;elegans</i></strong></div><div><em>Neuron.</em> 2002; <strong>36</strong>:1079-1089</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=10.1016%2FS0896-6273%2802%2901066-8&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=10.1016%2FS0896-6273%2802%2901066-8&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-0037137698" target="_blank" aria-label="Open Scopus in new tab">Scopus (144)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/12495623/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0896-6273%2802%2901066-8&amp;pmid=12495623" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>]) or organized into a systematic program of gene expression; and whether environment-dependent transcriptional changes actually adapt OR-specific odor responses.</div><div id="p0065" role="paragraph">To address these questions we characterized, via single-cell RNA sequencing (scRNA-seq), transcriptional variation across the extraordinarily large array of identifiable OSN subtypes. Our experiments reveal that each OSN subtype expresses a distinguishable set of transcripts that is stereotyped across mice within a given environment. The main axis of transcriptional variability includes more than 70 functionally relevant genes that adaptively attenuate or amplify the transformation of odors into spikes. Furthermore, the expression of these genes varies in a continuous and coordinated manner across OSN subtypes, is&nbsp;modulated by specific interactions between ORs and the environment, and predicts OSN responses to new odors. Transcriptional variation among OSNs is therefore systematically organized through a rheostat-like mechanism (akin to a balance control on a stereo) whose setting in each OSN is defined by OR-environment interactions. Our findings suggest that the olfactory system uses this transcriptional rheostat to proportionally and bidirectionally adapt to persistent background odors—thereby enabling OSNs to engage in a form of sensory predictive coding—before odor information is transmitted to neural circuits responsible for perception. These data reveal that peripheral odor codes are flexible rather than fixed and support a broad model in which neurons continuously individualize their transcriptomes to facilitate functional adaptation.</div></section><section id="sec-2"><h2>Results</h2><div id="p0070" role="paragraph">To assess relationships between transcription and function in the peripheral olfactory system we performed scRNA-seq on ∼770,000 mature mouse OSNs (<a id="crosref0365" href="#mmc1">Table S1</a>). Our initial analysis focused on ∼40,000 mature OSNs derived from adult mice housed in a typical home cage environment; collectively, these OSNs expressed more than 1,000 ORs, with nearly every mature neuron expressing only one OR (<a id="crosref0370" href="#fig1">Figures 1</a>A and <a id="crosref0375" href="#figs1">S1</a>A–S1D). Uniform manifold approximation and projection (UMAP) embeddings of ∼1,350 highly variable genes (HVGs, here and in all analyses excluding the OR genes themselves, see <a id="crosref0380" href="#sec-4">STAR Methods</a>) revealed that OSNs are partitioned into several broad subtypes related to positional identity and the expression of the lipid receptor CD36 (<a id="crosref0385" href="#fig1">Figure&nbsp;1</a>B).</div><div class="figure-wrap"><figure id="fig1" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.11.022/asset/53c5d6ca-f460-4fab-a293-a9cd15be464d/main.assets/gr1_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr1" src="/cms/10.1016/j.cell.2021.11.022/asset/5e8dc58a-bc0e-4528-bf87-ce55c278fed4/main.assets/gr1.jpg" height="527" width="662" aria-labelledby="fig1-title fig1-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.022/asset/53c5d6ca-f460-4fab-a293-a9cd15be464d/main.assets/gr1_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig1-title" class="figure__title"><span class="label figure__label">Figure&nbsp;1</span> <span class="figure__title__text">Each odorant receptor (OR) is associated with a distinct OSN transcriptome</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig1-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0015" role="paragraph">(A) Schematic of single-cell RNA sequencing (scRNA-seq) experiments. OR genes, which identify each OSN subtype, were excluded from the highly variable genes used in downstream analyses of OSN gene expression.</div><div id="fspara0020" role="paragraph">(B) UMAP plots visualizing gene expression in mature OSNs, with normalized expression of known identity-related marker genes (38,345 mature OSNs from six mice depicted).</div><div id="fspara0025" role="paragraph">(C) UMAP plot depicting OSN subtypes expressing the indicated ORs.</div><div id="fspara0030" role="paragraph">(D) (Left) Transcriptome distances between individual OSNs expressing <i>Olfr727</i>, <i>Olfr728</i>, or <i>Olfr729</i> (color bars indicate individual mice). (Right) Distributions of distances between cells expressing the same OR (within-OR, mean and 2.5th–97.5th percentile across 1,000 restarts indicated) or different ORs (between-OR, mean and interquartile range across OSN subtypes of the pairwise distances between a given OSN subtype and all others indicated).</div><div id="fspara0035" role="paragraph">(E) Accuracy of pairwise linear classifiers at predicting via OSN transcriptomes that OR is expressed by a given OSN (black dashed line indicates median, boxes represent 25th/75th percentile, 12.5th/87.5th percentile, and so forth across <sub>831</sub>C<sub>2</sub>&nbsp;= 344,865 pairs). Classification performance is at chance levels upon shuffling OR labels across OSNs.</div><div id="fspara0040" role="paragraph">(F) Accuracy of linear classifiers predicting which OR (out of 831) is expressed, at varying levels of prediction accuracy, from perfect (0.0%) to the rate at which the correct OR was within the top 2.5% of predicted ORs. Distributions depict the mean accuracy across the 831 OR-defined OSN subtypes (each averaged across 1,000 restarts). Performance is at chance levels upon shuffling OR labels across OSNs.</div><div id="fspara0045" role="paragraph">(G) Accuracy of linear classifiers predicting which OR is expressed in an OSN from a held-out mouse, with training data provided from five separate mice. Both perfect (dark blue) and the top 1% (light blue) of predictions are shown.</div><div id="fspara0050" role="paragraph">(H) Schematic of the genomic loci for the OR-swap and P2-IRES-GFP mouse lines.</div><div id="fspara0055" role="paragraph">(I) UMAP plots of mature OSNs from wild-type and the mice shown in (H) highlighting OSNs expressing M72, S50, or P2 receptors.</div><div id="fspara0060" role="paragraph">(J) Accuracy of linear classifiers predicting OR identity for OSNs expressing M72, S50, or P2, trained and tested as indicated. Error bars, here and throughout unless otherwise stated, depict the mean and the bootstrapped 95% confidence intervals of the mean.</div><div id="fspara0065" role="paragraph">See also <a id="crosref1575" href="#figs1">Figure&nbsp;S1</a>.</div></div></div></div></figcaption></figure></div><div class="figure-wrap"><figure id="figs1" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.11.022/asset/4aa93467-d7ff-4b38-8b1a-2053a78eab4a/main.assets/figs1_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs1" src="/cms/10.1016/j.cell.2021.11.022/asset/8e871440-af04-4b66-88b7-e8ef0410af22/main.assets/figs1.jpg" height="832" width="657" aria-labelledby="figs1-title figs1-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.022/asset/4aa93467-d7ff-4b38-8b1a-2053a78eab4a/main.assets/figs1_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs1-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S1</span> <span class="figure__title__text">OSN transcriptomes are predictable across individuals, and variation is distributed across hundreds of genes, related to <a id="crosref1680" href="#fig1">Figure&nbsp;1</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs1-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0500" role="paragraph">(A) Histogram of the number of cells expressing each OR, from wild-type (WT) mice housed in the home cage.</div><div id="fspara0505" role="paragraph">(B) The cumulative percent of ORs observed in OSNs (sorted by how often each OR is observed) as a function of the cumulative fraction of OSNs that express any given OR. The top 200 ORs (of the 1,016 detected) account for &gt;50% of OSNs.</div><div id="fspara0510" role="paragraph">(C) The number of OSN subtypes identified, thresholded by the number of OSNs expressing a given OR; colors represent numbers of OSNs used for this analysis; a threshold of 10 OSNs yielded 831 OSN subtypes.</div><div id="fspara0515" role="paragraph">(D) Percent of OSNs expressing 0, 1, or &gt;1 ORs thresholded by the number of transcripts (unique molecular identifiers [UMIs]) required for an OR gene to be considered “detected.” Many of the rare 2 OR cells were clearly doublets and contained higher than average total numbers of UMIs compared to a typical 1 OR cell. Therefore, downstream analyses used a threshold of 3&nbsp;UMIs and only considered cells in which only 1 OR was detected.</div><div id="fspara0520" role="paragraph">(E) UMAP plots highlighting OSNs expressing the indicated ORs.</div><div id="fspara0525" role="paragraph">(F) The mean fraction of all OSNs expressing a given OR (averaged across 1,000 restarts) as a function of the rank-ordered transcriptome distance from each cell expressing that OR. Shaded error indicates the interquartile range across ORs.</div><div id="fspara0530" role="paragraph">(G) Distribution of within-OR transcriptome distances across OSN subtypes using cells from the same or different mice, or after shuffling OR labels.</div><div id="fspara0535" role="paragraph">(H) Distribution of within-OR transcriptome distances as a function of the number of OSNs expressing each OR.</div><div id="fspara0540" role="paragraph">(I) Separability of the distributions of within-OR and between-OR transcriptome distances, as measured by the Jensen-Shannon divergence (JSD), for various numbers of genes and increasing numbers of principal components (PCs). JSD was normalized to the maximum JSD value across all conditions. Highly variable genes (HVGs, used to compute the PCs) were selected based on their variability across OSNs expressing different ORs. Stars represent the number of HVGs used in the analyses shown in the paper. The JSD is low, and separability is at chance levels, when OR labels are shuffled.</div><div id="fspara0545" role="paragraph">(J) Distribution of accuracies for linear classifiers predicting which OR is expressed in a given OSN, as a function of the number of OSNs expressing each OR.</div><div id="fspara0550" role="paragraph">(K) Mean accuracy across OSN subtypes of linear classifiers predicting which OR is expressed as a function of the number of HVGs included (sorted by their informativeness, see <a id="crosref1685" href="#sec-4">STAR Methods</a>). Colors indicate classification thresholds, and error bars depict the mean and SD across 10 restarts.</div><div id="fspara0555" role="paragraph">(L) Similar to (K), but as a function of the number of genes removed.</div><div id="fspara0560" role="paragraph">(M) (Left) Accuracy of linear classifiers predicting which OR (out of 831) is expressed, after excluding all transcription factor (TF) and axon guidance genes. Distributions depict the mean accuracy across the 831 OR-defined OSN subtypes (each averaged across 1,000 restarts). (Right) Similar to left, but for various gene sets, including axon guidance genes and TF genes (which both contain HVGs and non-HVGs), and the 100 most predictive or all HVGs (reproduced from <a id="crosref1690" href="#fig1">Figure&nbsp;1</a>F). Classifier hyperparameters were optimized for each gene set.</div><div id="fspara0565" role="paragraph">(N) Accuracy of pairwise linear classifiers predicting which OR is expressed, with respect to (left) anatomical identity or (middle) OR class. (Right) Accuracy of linear classifiers to predict which of dorsal class I, dorsal class II, or ventral class II OR is expressed.</div><div id="fspara0570" role="paragraph">(O) Similar to left and middle of (N), but with respect to CD36 positivity.</div><div id="fspara0575" role="paragraph">(P) Similar to left and middle of (N), but with respect to genomic distance (distance between transcriptional start sites [TSSs], in base pairs).</div><div id="fspara0580" role="paragraph">(Q) Similar to left and middle of (N), but with respect to the phylogenetic distance.</div></div></div></div></figcaption></figure></div><section id="sec-2-1"><h3>OSN transcriptomes are stereotyped and uniquely specified by the expressed OR</h3><div id="p0075" role="paragraph">Given the few known transcriptional differences among mature OSNs, we expected that OR-defined OSN subpopulations would overlap in UMAP space; instead, OSNs expressing each OR were locally clustered and separated from those expressing different ORs, suggesting that each OSN subtype is transcriptionally unique (<a id="crosref0390" href="#fig1">Figures 1</a>C and <a id="crosref0395" href="#figs1">S1</a>E). For example, the transcriptomes of OSNs expressing <i>Olfr727</i> were self-similar and distinct from OSNs expressing <i>Olfr728</i> and <i>Olfr729</i> (<a id="crosref0400" href="#fig1">Figure&nbsp;1</a>D). The transcriptomes of all OR-identified OSN subtypes were distinguishable, regardless of how transcriptome distances were computed or the frequency with which each OR was expressed (<a id="crosref0405" href="#fig1">Figures 1</a>D and <a id="crosref0410" href="#figs1">S1</a>F–S1I). Linear classifiers distinguished OSNs expressing different ORs with near-perfect accuracy, and predicted the specific expressed OR (of the 831 queried) with &gt;50% accuracy (versus 0.1% at chance) (<a id="crosref0415" href="#fig1">Figures 1</a>E, 1F, and <a id="crosref0420" href="#figs1">S1</a>J). Approximately 1,000 HVGs were required for maximum classification accuracy, although individual genes contributed marginally to overall performance (<a id="crosref0425" href="#figs1">Figures S1</a>K and S1L). Importantly, classifiers accurately predicted the identity of the expressed OR in OSNs derived from held-out mice, indicating that OR-transcriptome relationships generalize across individuals housed in a similar environment (<a id="crosref0430" href="#fig1">Figure&nbsp;1</a>G).</div><div id="p0080" role="paragraph">The unexpectedly close relationship between OR expression and OSN transcriptomes could result from shared regulatory mechanisms, or from each OR determining its associated transcriptome. To distinguish these possibilities, we performed scRNA-seq in mice with swapped coding regions for the M72 (<i>Olfr160</i>) and S50 (<i>Olfr545</i>) receptor genes (<a id="crosref0435" href="#fig1">Figure&nbsp;1</a>H). OSNs expressing the M72 receptor from the S50 gene locus aligned with wild-type M72 OSNs; likewise, OSNs expressing S50 from the M72 gene locus resembled wild-type S50 OSNs (<a id="crosref0440" href="#fig1">Figures 1</a>I and 1J). The unique gene expression signature associated with each OSN is therefore largely specified by the expressed OR protein.</div><div id="p0085" role="paragraph">OSNs express a limited number of transcription factors and axon guidance genes in an OR-dependent manner during development (<span class="dropBlock reference-citations"><a id="body-ref-sref68-2" href="#" role="doc-biblioref" data-xml-rid="bib68" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref68-2" aria-controls="sref68-2">Mori and Sakano, 2011</a><div class="dropBlock__holder" data-db-target-of="sref68-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">70.</div><div>Mori, K. ∙ Sakano, H.</div><div><strong>How is the olfactory map formed and interpreted in the mammalian brain?</strong></div><div><em>Annu. Rev. Neurosci.</em> 2011; <strong>34</strong>:467-499</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1146/annurev-neuro-112210-112917" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-79959891730" target="_blank" aria-label="Open Scopus in new tab">Scopus (227)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21469960/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1146%2Fannurev-neuro-112210-112917&amp;pmid=21469960" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref76" href="#" role="doc-biblioref" data-xml-rid="bib76" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref76" aria-controls="sref76">Parrilla et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref76"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">78.</div><div>Parrilla, M. ∙ Chang, I. ∙ Degl’Innocenti, A. ...</div><div><strong>Expression of homeobox genes in the mouse olfactory epithelium</strong></div><div><em>J.&nbsp;Comp. Neurol.</em> 2016; <strong>524</strong>:2713-2739</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/cne.24051" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_78_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-85028257873" target="_blank" aria-label="Open Scopus in new tab">Scopus (7)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27243442/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fcne.24051&amp;pmid=27243442" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). However, genes from these categories were neither required for accurate OR predictions nor were they as predictive as either the complete set or the most predictive subset of HVGs (<a id="crosref0455" href="#figs1">Figure&nbsp;S1</a>M). Each OSN can also be categorized by the expression of CD36, the specific class of its expressed OR (class I versus class II), or the dorsal or ventral location of its associated glomeruli in the olfactory bulb; classifiers accurately distinguished ORs from OSNs within or between each of these categories (<a id="crosref0460" href="#figs1">Figures S1</a>N and S1O). Furthermore, classifiers distinguished OSNs independently of the genomic or phylogenetic relationships between ORs (<a id="crosref0465" href="#figs1">Figures S1</a>P and S1Q). Therefore, known differences relating to OSN identity or genomic/phylogenetic features of ORs do not explain the transcriptional diversity of mature OSNs.</div></section><section id="sec-2-2"><h3>Decomposing OSN transcriptomes into identity and activity gene expression programs</h3><div id="p0090" role="paragraph">Alternatively, the diversity in OSN transcriptomes might reflect OR-associated differences in neural activity. To test this possibility, we performed consensus non-negative matrix factorization (cNMF), which decomposes gene expression patterns observed across OSNs into sets of co-expressed genes called gene expression programs (GEPs) (<a id="crosref0470" href="#fig2">Figure&nbsp;2</a>A, see <a id="crosref0475" href="#sec-4">STAR Methods</a>); importantly, cNMF successfully identified both identity-associated and activity-related GEPs in a prior study (<span class="dropBlock reference-citations"><a id="body-ref-sref50-1" href="#" role="doc-biblioref" data-xml-rid="bib50" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref50-1" aria-controls="sref50-1">Kotliar et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref50-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">52.</div><div>Kotliar, D. ∙ Veres, A. ∙ Nagy, M.A. ...</div><div><strong>Identifying gene expression programs of cell-type identity and cellular activity with single-cell RNA-seq</strong></div><div><em>eLife.</em> 2019; <strong>8</strong>:e43803</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.7554/eLife.43803" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-85070056080" target="_blank" aria-label="Open Scopus in new tab">Scopus (43)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31282856/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.7554%2FeLife.43803&amp;pmid=31282856" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). cNMF identified 10 GEPs—each composed of 100s of largely non-overlapping genes—that effectively captured OR-driven transcriptional variability, that varied in usage across OSNs in an OR-specific manner, and that accurately predicted the expressed OR in each OSN (<a id="crosref0485" href="#fig2">Figures 2</a>B and <a id="crosref0490" href="#figs2">S2</a>A–S2D).</div><div class="figure-wrap"><figure id="fig2" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.11.022/asset/08eeda0b-eb45-4077-ba2c-b7e9e368e6ff/main.assets/gr2_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr2" src="/cms/10.1016/j.cell.2021.11.022/asset/21aa6332-12b7-49a1-99c5-807c5cc3468f/main.assets/gr2.jpg" height="415" width="769" aria-labelledby="fig2-title fig2-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.022/asset/08eeda0b-eb45-4077-ba2c-b7e9e368e6ff/main.assets/gr2_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig2-title" class="figure__title"><span class="label figure__label">Figure&nbsp;2</span> <span class="figure__title__text">OSN transcriptional variation can be decomposed into identity and activity gene expression programs</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig2-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0075" role="paragraph">(A) Consensus non-negative matrix factorization (cNMF) identifies coherent gene expression programs (GEPs). Gene expression in each cell can be decomposed into a set of usages across a small number of GEPs, each of which is defined by its loadings for each gene.</div><div id="fspara0080" role="paragraph">(B) UMAP plots showing the usage of each functionally annotated GEP in mature OSNs from wild-type mice housed in the home cage.</div><div id="fspara0085" role="paragraph">(C) GEP<sub>High</sub> and GEP<sub>Low</sub> usages by the OSNs shown in (B). Each point indicates the mean GEP usage for all OSNs expressing the same OR for each of the 831 ORs detected in at least 10 OSNs.</div><div id="fspara0090" role="paragraph">(D) Environmental state (ES) scores for each OSN are calculated by taking the difference between GEP<sub>High</sub> and GEP<sub>Low</sub>.</div><div id="fspara0095" role="paragraph">(E) Mean ES score for each OSN subtype. ES scores were shuffled 1,000 times across cells, and the mean interquartile range across shuffles is depicted in gray for each OSN subtype, with the mean across sorted shuffles depicted by the dashed line.</div><div id="fspara0100" role="paragraph">(F) As in (B) but for ES scores.</div><div id="fspara0105" role="paragraph">(G) Between-mouse correlation coefficients of the mean ES score for each OSN subtype.</div><div id="fspara0110" role="paragraph">(H) Accuracy of pairwise linear classifiers (as in <a id="crosref1580" href="#fig1">Figure&nbsp;1</a>E) predicting OR identity using only the usages of GEP<sub>High</sub> and GEP<sub>Low</sub>.</div><div id="fspara0115" role="paragraph">See also <a id="crosref1585" href="#figs2">Figure&nbsp;S2</a> and <a id="crosref1590" href="#mmc2">Table S2</a>.</div></div></div></div></figcaption></figure></div><div class="figure-wrap"><figure id="figs2" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.11.022/asset/ce789880-7a50-4357-843f-dccc31404e9f/main.assets/figs2_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs2" src="/cms/10.1016/j.cell.2021.11.022/asset/23fe8d03-ec8e-4594-8ea1-85673dc7a6da/main.assets/figs2.jpg" height="864" width="657" aria-labelledby="figs2-title figs2-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.022/asset/ce789880-7a50-4357-843f-dccc31404e9f/main.assets/figs2_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs2-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S2</span> <span class="figure__title__text">GEP<sub>High</sub> and GEP<sub>Low</sub> are aligned with the main axis of transcriptional variation across OSNs, related to <a id="crosref1695" href="#fig2">Figure&nbsp;2</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs2-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0590" role="paragraph">(A) Consensus non-negative matrix factorization (cNMF) identifies 10 OR-specific gene expression programs (GEPs) whose usages are correlated among OSNs that express the same OR. OSNs expressing the same OR were randomly split into two populations and the correlation in GEP usage between the two populations was computed for each GEP. Error bars depict the mean correlation with SD across 10,000 restarts. Six GEPs (including one with many genes related to the unfolded protein response [UPR]) only showed weak correlation, indicating that the usages for such GEPs are not specific to ORs in mature OSNs.</div><div id="fspara0595" role="paragraph">(B) UMAP plots showing the usages of identified GEPs, together with the expression of genes with high loadings for each GEP.</div><div id="fspara0600" role="paragraph">(C) Gene loadings for the subset of HVGs whose loadings was higher than 2 of 1,350 (total number of HVGs) in any of the 10 OR-specific GEPs; genes were hierarchically clustered by their loadings.</div><div id="fspara0605" role="paragraph">(D) Accuracy of linear classifiers predicting OR identity using GEPs across (left) OR pairs or (right) across all ORs. Performance upon shuffling OR labels is at chance&nbsp;= 1/831.</div><div id="fspara0610" role="paragraph">(E) Correlation coefficients between mean GEP usages for each OSN subtype. Strong anticorrelation between dorsal/ventral (ρ&nbsp;= −0.85), anterior/posterior (ρ&nbsp;= −0.70), and high/low GEPs (ρ&nbsp;= −0.87) is observed.</div><div id="fspara0615" role="paragraph">(F) Mean usages of (left) GEP<sub>Dorsal</sub> and GEP<sub>Ventral</sub> or (right) GEP<sub>Anterior</sub> and GEP<sub>Posterior</sub> for each OSN subtype, from WT mice housed in the home cage.</div><div id="fspara0620" role="paragraph">(G) Accuracy of pairwise linear classifiers predicting OR identity using only the usages of either the dorsal/ventral (D/V) or the anterior/posterior (A/P) GEPs.</div><div id="fspara0625" role="paragraph">(H) Correlation coefficients of environmental state (ES), dorsal/ventral (DV), and anterior/posterior (AP) scores. DV and AP scores are calculated by GEP<sub>Dorsal</sub> − GEP<sub>Ventral</sub> and GEP<sub>Anterior</sub> − GEP<sub>Posterior</sub>, respectively. Individual mice are indicated as colored on top.</div><div id="fspara0630" role="paragraph">(I) Correlation coefficients, computed across OSN subtypes, between the GEP usages and PC scores (calculated from the top 1,350 HVGs).</div><div id="fspara0635" role="paragraph">(J) Absolute values of the correlation coefficient, computed across OSN subtypes, between the ES score and the PC score for each of top 10 PCs. The ES score is highly correlated with the first PC (ρ&nbsp;= 0.89), indicating that the ES score is a major axis of transcriptional variation across OSNs derived from mice housed in the home cage. Error bars depict the mean and SD across 1,000 restarts.</div><div id="fspara0640" role="paragraph">(K) Mean <i>Z</i>-scored expression of immature (<i>Gng8</i>, <i>Gap43</i>) and mature (<i>Gnal</i>, <i>Omp</i>) OSN marker genes as a function of the developmental pseudotime from the onset of OR expression in late-immature OSNs (see <a id="crosref1700" href="#sec-4">STAR Methods</a>).</div><div id="fspara0645" role="paragraph">(L) (Left) GEP<sub>Low</sub>, (middle) GEP<sub>High</sub>, and (right) ES scores as a function of the developmental pseudotime from the onset of OR expression in late-immature OSNs. OSN subtypes are fractionated by their ES scores in mature OSNs.</div></div></div></div></figcaption></figure></div><div id="p0095" role="paragraph">Of the 10 identified GEPs, we focused on the 7 whose constituent genes suggested putative functions (<a id="crosref0495" href="#figs2">Figure&nbsp;S2</a>E). Five GEPs encoded known aspects of OSN identity (dorsal, ventral, anterior, posterior, and CD36 positivity, <a id="crosref0500" href="#figs2">Figures S2</a>B and S2F–S2H; <a id="crosref0505" href="#mmc2">Table S2</a>). The other two GEPs, which we refer to as GEP<sub>High</sub> and GEP<sub>Low</sub>, included genes associated with neural activity in OSNs: GEP<sub>High</sub> included genes downregulated by chronic sensory deprivation (e.g., <i>S100a5</i>, <i>Pcp4l1</i>, and <i>Kirrel2</i>) while GEP<sub>Low</sub> included genes upregulated after chronic sensory deprivation (e.g., <i>Calb2</i>, <i>Kirrel3</i>, and <i>Ppp3ca</i>) (<span class="dropBlock reference-citations"><a id="body-ref-sref29-2" href="#" role="doc-biblioref" data-xml-rid="bib29" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref29-2" aria-controls="sref29-2">Fischl et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref29-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">29.</div><div>Fischl, A.M. ∙ Heron, P.M. ∙ Stromberg, A.J. ...</div><div><strong>Activity-dependent genes in mouse olfactory sensory neurons</strong></div><div><em>Chem. Senses.</em> 2014; <strong>39</strong>:439-449</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/chemse/bju015" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-84901414304" target="_blank" aria-label="Open Scopus in new tab">Scopus (13)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24692514/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fchemse%2Fbju015&amp;pmid=24692514" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref101-2" href="#" role="doc-biblioref" data-xml-rid="bib101" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref101-2" aria-controls="sref101-2">Wang et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref101-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">104.</div><div>Wang, Q. ∙ Titlow, W.B. ∙ McClintock, D.A. ...</div><div><strong>Activity-dependent gene expression in the mammalian olfactory epithelium</strong></div><div><em>Chem. Senses.</em> 2017; <strong>42</strong>:611-624</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/chemse/bjx028" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_104_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-85030719241" target="_blank" aria-label="Open Scopus in new tab">Scopus (7)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28525560/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fchemse%2Fbjx028&amp;pmid=28525560" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Each OSN primarily used either GEP<sub>Low</sub> or GEP<sub>High</sub>, and the usage of each GEP varied continuously across OSN subtypes (<a id="crosref0520" href="#fig2">Figures 2</a>C and <a id="crosref0525" href="#figs2">S2</a>B).</div><div id="p0100" role="paragraph">We hypothesized that persistent interactions between each OR and the wide variety of odors in a given environment (such as the food, bedding, and semiochemical odors present in a typical home cage) might explain the continuous variation of GEP<sub>High</sub> or GEP<sub>Low</sub> across OSN subtypes. To explore this idea, we developed a metric (simply the difference between GEP<sub>High</sub> and GEP<sub>Low</sub>) that we provisionally refer to as the environmental state (ES) score (<a id="crosref0530" href="#fig2">Figure&nbsp;2</a>D). ES scores for OSNs derived from mice housed in the home cage varied continuously across OSN subtypes; however, OSN subtype-specific ES scores were remarkably consistent across mice and enabled accurate predictions of OR identity (<a id="crosref0535" href="#fig2">Figures 2</a>E–2H). Both ES scores and GEP<sub>High</sub> and GEP<sub>Low</sub> usages aligned with the first principal component of the OSN transcriptomes (<a id="crosref0540" href="#figs2">Figures S2</a>I and S2J). GEP<sub>High</sub> and GEP<sub>Low</sub> genes were not expressed in immature OSNs but their expression increased and diversified after the onset of OR expression (<a id="crosref0545" href="#figs2">Figures S2</a>K and S2L).</div></section><section id="sec-2-3"><h3>OSN transcriptomes systematically reflect environment-dependent activity</h3><div id="p0105" role="paragraph">Consistent with the hypothesis that GEP<sub>High</sub> and GEP<sub>Low</sub> genes, and thus ES scores, are sensitive to the chronic activity state of each OSN, artificially lowering (via nares occlusion for a month) or raising (via pulsed optogenetic stimulation for 12 h) OSN activity systematically decreased or increased ES scores, respectively (<a id="crosref0550" href="#fig3">Figures 3</a>A–3H and <a id="crosref0555" href="#figs3">S3</a>A–S3E). To address whether OSN ES scores are sensitive to chronic activity levels under more physiological conditions, we compared OSN gene expression between mice housed in either a regular home cage or in two distinct naturalistic odor environments (<a id="crosref0560" href="#fig3">Figure&nbsp;3</a>I). ES scores shifted significantly and bidirectionally in ∼45% of OSN subtypes across environments, consistent with specific odor environments differentially ligating or disengaging distinct ORs (<a id="crosref0565" href="#fig3">Figures 3</a>J–3N and <a id="crosref0570" href="#figs3">S3</a>F–S3H).</div><div class="figure-wrap"><figure id="fig3" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.11.022/asset/7df42adf-e1d0-4a14-8ac3-9eec2d53102c/main.assets/gr3_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr3" src="/cms/10.1016/j.cell.2021.11.022/asset/383db28a-790b-45b4-969c-7399942335b2/main.assets/gr3.jpg" height="845" width="654" aria-labelledby="fig3-title fig3-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.022/asset/7df42adf-e1d0-4a14-8ac3-9eec2d53102c/main.assets/gr3_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig3-title" class="figure__title"><span class="label figure__label">Figure&nbsp;3</span> <span class="figure__title__text">OSN transcriptomes are shaped by the odor environment</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig3-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0125" role="paragraph">(A) Schematic of the chronic naris occlusion experiment.</div><div id="fspara0130" role="paragraph">(B) UMAP plots of mature OSNs depicting the nostril of origin, ES scores, and OSNs expressing example ORs.</div><div id="fspara0135" role="paragraph">(C) Mean occlusion-dependent changes in GEP usage across OSN subtypes, as defined by OR expression.</div><div id="fspara0140" role="paragraph">(D) Mean ES score for each OSN subtype for each nostril. ES scores decreased by occlusion for 770 out of 797 OSN subtypes (false discovery rate [FDR] ≤ 0.01).</div><div id="fspara0145" role="paragraph">(E) Correlation of the mean ES scores for each OSN subtype from the indicated nostril pairs (p&nbsp;= 1.68&nbsp;× 10<sup>−5</sup>, Mann-Whitney U test, for both within- versus between-nostril comparisons).</div><div id="fspara0150" role="paragraph">(F) Schematic of the chronic optogenetic stimulation experiment.</div><div id="fspara0155" role="paragraph">(G) Mean ES score for each dorsal OSN subtype from stimulated and control (no-light) mice. ES scores increased following optogenetic stimulation for 56 out of 90 subtypes (FDR ≤ 0.01).</div><div id="fspara0160" role="paragraph">(H) Distribution of mean ES scores across OSN subtypes from each condition. ES scores following optogenetic stimulation are right-shifted (p&nbsp;= 1.33&nbsp;× 10<sup>−34</sup>, Kolmogorov-Smirnov [KS] test).</div><div id="fspara0165" role="paragraph">(I) Schematic of the environment switch experiment.</div><div id="fspara0170" role="paragraph">(J) UMAP plots highlighting (left) OSNs expressing example ORs from mice housed in each of the three environments. (Right) OSN ES scores.</div><div id="fspara0175" role="paragraph">(K) Pairwise correlations of the mean ES scores for each OSN subtype, across mice from the indicated environments.</div><div id="fspara0180" role="paragraph">(L) Pairwise correlations from (K) separated by the type of comparison. Correlations are higher for mice housed within the same environment (p&nbsp;= 7.17&nbsp;× 10<sup>−7</sup>, Mann-Whitney U test).</div><div id="fspara0185" role="paragraph">(M) Percent of OSN subtypes whose ES scores change significantly (FDR ≤ 0.01), for animals from the same or different environments.</div><div id="fspara0190" role="paragraph">(N) Distribution of ES score changes, for OSN subtypes from the same or different environments. ES score changes are larger between environments (p&nbsp;= 5.16&nbsp;× 10<sup>−140</sup>, KS test).</div><div id="fspara0195" role="paragraph">(O) The observed occlusion-induced change in ES scores for each OSN subtype, as a function of ES scores from open nostrils.</div><div id="fspara0200" role="paragraph">(P) Distribution of the mean ES scores across OSN subtypes from each nostril. ES scores following occlusion are left-shifted (p&nbsp;= 3.74&nbsp;× 10<sup>−217</sup>, KS test).</div><div id="fspara0205" role="paragraph">(Q) Accuracy of a minimum distance classification procedure (see <a id="crosref1595" href="#sec-4">STAR Methods</a>) to predict in which environment a mouse was housed, based on ES scores for each OSN subtype (observed or shuffled environment as labeled, curves depict the mean and interquartile range of classification accuracy for each environment across 1,000 restarts).</div><div id="fspara0210" role="paragraph">(R) Cumulative distribution of ES scores in a given reference environment for OSN subtypes whose ES scores were significantly increased, decreased, or unchanged across environments. OSN subtypes whose ES scores decreased tended to have higher ES scores in the reference environment (p&nbsp;= 4.03&nbsp;× 10<sup>−8</sup>, KS test); the opposite was true for those whose ES scores increased (p&nbsp;= 2.32&nbsp;× 10<sup>−14</sup>).</div><div id="fspara0215" role="paragraph">See also <a id="crosref1600" href="#figs3">Figure&nbsp;S3</a>.</div></div></div></div></figcaption></figure></div><div class="figure-wrap"><figure id="figs3" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.11.022/asset/072e79d1-1eef-4c7c-bb06-1c28c5e29095/main.assets/figs3_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs3" src="/cms/10.1016/j.cell.2021.11.022/asset/12e7d552-6583-4708-8a1a-b228e4e281ca/main.assets/figs3.jpg" height="699" width="657" aria-labelledby="figs3-title figs3-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.022/asset/072e79d1-1eef-4c7c-bb06-1c28c5e29095/main.assets/figs3_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs3-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S3</span> <span class="figure__title__text">The dependence of OSN transcriptomes on long-term interactions with odors and air, related to <a id="crosref1705" href="#fig3">Figure&nbsp;3</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs3-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0655" role="paragraph">(A) Accuracy of linear classifiers predicting OR identity based on HVG expression (see <a id="crosref1710" href="#fig1">Figures 1</a>E and 1F) from open or occluded nostrils across OR pairs (left) or across all ORs (right).</div><div id="fspara0660" role="paragraph">(B) Accuracy of linear classifiers predicting whether a given cell for each OSN subtype is from the open or closed nostril, using the usage of either (left) all 10 OR-specific GEPs or (right) only GEP<sub>High</sub>/GEP<sub>Low</sub>.</div><div id="fspara0665" role="paragraph">(C) Gene loadings for each GEP for HVGs that are differentially expressed (DE), or are unchanged (non-DE, in gray) by chronic naris occlusion. HVGs that decrease upon occlusion (in blue) have higher than average loadings compared to non-DE genes for GEP<sub>High</sub> (p&nbsp;&lt; 1&nbsp;× 10<sup>−57</sup>, KS test, adjusted by the Holm-Bonferroni method) and lower than average loadings for GEP<sub>Low</sub>, and vice versa for genes that increased (in red, p&nbsp;&lt; 1&nbsp;× 10<sup>−48</sup>, KS test, adjusted by the Holm-Bonferroni method).</div><div id="fspara0670" role="paragraph">(D) Mean usages of GEP<sub>High</sub> and GEP<sub>Low</sub> for 797 OSN subtypes for each nostril, using data from all mice.</div><div id="fspara0675" role="paragraph">(E) Mean ES score for each OSN subtype for each nostril. Data from each mouse are shown separately. ES scores decreased by occlusion for 314 of 341, 356 of 391, and 320 of 340 of OSN subtypes, respectively (FDR ≤ 0.01).</div><div id="fspara0680" role="paragraph">(F) Mean ES scores for each OSN subtype for mouse pairs from the same or different environments.</div><div id="fspara0685" role="paragraph">(G) Cumulative percent of OSN subtypes whose ES scores differ significantly (FDR ≤ 0.01) in different numbers of environment pairs.</div><div id="fspara0690" role="paragraph">(H) Mean environment-dependent changes in GEP usage across OSN subtypes, categorized based on the sign and significance of their ES score changes across environments.</div><div id="fspara0695" role="paragraph">(I) Accuracy of pairwise linear classifiers predicting OR identity using only the usages of the two ES GEPs (GEP<sub>High</sub> and GEP<sub>Low</sub>) of OSNs from each nostril in the chronic occlusion data.</div><div id="fspara0700" role="paragraph">(J) UMAP plot highlighting OSNs expressing the indicated ORs, using only cells from occluded nostrils.</div><div id="fspara0705" role="paragraph">(K) UMAP plot depicting the <i>Z</i>-scored ES scores, using only cells from occluded nostrils.</div><div id="fspara0710" role="paragraph">(L) Accuracy of pairwise linear classifiers predicting OR identity using only the usages of the two ES GEPs of OSNs from each of the three environments.</div><div id="fspara0715" role="paragraph">(M) Accuracy of pairwise linear classifiers predicting in which environment a mouse was housed, based on usages of either all GEPs or ES GEPs. Classification was performed for each OSN subtype, and distributions of the mean accuracies (across 1,000 restarts) across OSN subtypes with or without significant ES score changes across environments are shown (shuffled indicates environment labels permuted).</div><div id="fspara0720" role="paragraph">(N) OR expression levels (as quantified by the number of transcripts of the expressed OR normalized by the total number of transcripts per cell) in each nostril, for each OSN subtype.</div><div id="fspara0725" role="paragraph">(O) Similar to (N), but for optogenetically stimulated (12 h) and control (no light) mice.</div><div id="fspara0730" role="paragraph">(P) Pairwise correlations across all environments of the (left) mean OR expression levels or (right) the fraction of OSNs in which each OR was detected.</div><div id="fspara0735" role="paragraph">(Q) As in the right panel of (F), but for dorsal/ventral (DV) and anterior/posterior (AP) scores.</div><div id="fspara0740" role="paragraph">(R) ES scores for each OSN subtype from WT and β-Arrestin2 mutant mice housed in the home cage.</div><div id="fspara0745" role="paragraph">(S) Similar to (R), but for OR expression levels.</div><div id="fspara0750" role="paragraph">(T) (Left) Mean change in ES scores (relative to the mean across all OSN subtypes) 5&nbsp;days after a shift to environment A (envA), for OSN subtypes from β-Arrestin2 mutant mice. OSN subtypes whose ES score rose or fell after 2&nbsp;weeks in envA in WT data show an increase or decrease in ES scores in mutants (p&nbsp;&lt; 1&nbsp;× 10<sup>−15</sup>, Wilcoxon signed-rank test). (Right) Correlation coefficient of changes in ES score for each OSN subtype between WT and β-Arrestin2 mutant mice (p&nbsp;&lt; 1&nbsp;× 10<sup>−4</sup> via permutation test across 10,000 shuffles).</div></div></div></div></figcaption></figure></div><div id="p0110" role="paragraph">Three additional lines of evidence argue that ES scores reflect specific interactions between each OR and each environment. First, occlusion-dependent changes in ES scores for each OR-defined OSN subtype were negatively correlated with the ES score in the open nostril (<a id="crosref0575" href="#fig3">Figure&nbsp;3</a>O). ES scores after occlusion were sufficiently diverse to enable accurate predictions of OR identity; this diversity might reflect a variety of OR-specific spontaneous activity patterns (<a id="crosref0580" href="#fig3">Figures 3</a>P and <a id="crosref0585" href="#figs3">S3</a>I–S3K). Second, after switching environments, ES scores for each OSN subtype were consistent across mice and accurately predicted the specific environment in which each mouse was housed (<a id="crosref0590" href="#fig3">Figures 3</a>Q, <a id="crosref0595" href="#figs3">S3</a>L, and S3M). Finally, OSN subtypes whose ES scores increased across odor environments tended to have relatively low initial ES scores, and conversely those that fell tended to have relatively high initial ES scores (<a id="crosref0600" href="#fig3">Figure&nbsp;3</a>R).</div><div id="p0115" role="paragraph">We ruled out several alternative explanations for environment-dependent changes in OSN ES scores. Odor-dependent modulation in OR expression levels or β-Arrestin2-mediated OR endocytosis could, in principle, influence ES scores (<span class="dropBlock reference-citations"><a id="body-ref-sref40" href="#" role="doc-biblioref" data-xml-rid="bib40" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref40" aria-controls="sref40">Ibarra-Soria et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref40"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">40.</div><div>Ibarra-Soria, X. ∙ Nakahara, T.S. ∙ Lilue, J. ...</div><div><strong>Variation in olfactory neuron repertoires is genetically controlled and environmentally modulated</strong></div><div><em>eLife.</em> 2017; <strong>6</strong>:1083</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.7554/eLife.21476" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_40_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-85018344872" target="_blank" aria-label="Open Scopus in new tab">Scopus (38)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.7554%2FeLife.21476" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref61" href="#" role="doc-biblioref" data-xml-rid="bib61" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref61" aria-controls="sref61">Mashukova et&nbsp;al., 2006</a><div class="dropBlock__holder" data-db-target-of="sref61"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">63.</div><div>Mashukova, A. ∙ Spehr, M. ∙ Hatt, H. ...</div><div><strong>β-arrestin2-mediated internalization of mammalian odorant receptors</strong></div><div><em>J.&nbsp;Neurosci.</em> 2006; <strong>26</strong>:9902-9912</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1523/JNEUROSCI.2897-06.2006" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-33749170169" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17005854/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1523%2FJNEUROSCI.2897-06.2006&amp;pmid=17005854" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref97-1" href="#" role="doc-biblioref" data-xml-rid="bib97" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref97-1" aria-controls="sref97-1">von der Weid et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref97-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">100.</div><div>von der Weid, B. ∙ Rossier, D. ∙ Lindup, M. ...</div><div><strong>Large-scale transcriptional profiling of chemosensory neurons identifies receptor-ligand pairs in&nbsp;vivo</strong></div><div><em>Nat. Neurosci.</em> 2015; <strong>18</strong>:1455-1463</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nn.4100" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_100_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-84942456919" target="_blank" aria-label="Open Scopus in new tab">Scopus (61)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26322926/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnn.4100&amp;pmid=26322926" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). However, across all three of our experimental manipulations, OR gene expression and the usage of identity-related GEPs remained constant (<a id="crosref0620" href="#fig3">Figures 3</a>C, <a id="crosref0625" href="#figs3">S3</a>C, S3H, and S3N–S3Q). Furthermore, β-Arrestin2 knockout mice exhibited largely normal ES scores that adapted to a new environment (<a id="crosref0630" href="#figs3">Figures S3</a>R–S3T). Taken together, these results reveal that specific interactions between each OR and the environment differentially drive activity in each OSN, which in turn specifies the particular level at which GEP<sub>High</sub> and GEP<sub>Low</sub> genes are expressed. Our results also demonstrate that, under physiological conditions and in realistic odor environments, odors and air chronically engage the entire OSN array.</div></section><section id="sec-2-4"><h3>Structured correlations and flexibility in genes influencing odor-activity coupling</h3><div id="p0120" role="paragraph">What functions might environment-dependent gene expression confer upon mature OSNs? Inspection revealed that GEP<sub>High</sub> and GEP<sub>Low</sub> include 73 genes with known or likely roles in shaping OSN sensory responses (e.g., calcium homeostasis, OR signaling, and intrinsic excitability), as well as an additional ∼40 genes putatively involved in axon guidance or synaptic transmission (<a id="crosref0635" href="#fig4">Figure&nbsp;4</a>A; <a id="crosref0640" href="#mmc3">Table S3</a>). The expression of these genes smoothly co-varied across OSN subtypes and exhibited structured correlations and anticorrelations with other GEP<sub>High</sub> and GEP<sub>Low</sub> genes (<a id="crosref0645" href="#fig4">Figures 4</a>B and <a id="crosref0650" href="#figs4">S4</a>A–S4C). Similar to the ES score itself, the expression of “functional” genes (here and throughout, defined as the set of 73 GEP<sub>High</sub> and GEP<sub>Low</sub> genes relevant to sensory responses) both distinguished all OSN subtypes and systematically fell after naris occlusion (<a id="crosref0655" href="#figs4">Figures S4</a>D–S4F). The observed pattern of functional gene expression in each OSN suggested a role in compensating for ongoing activity; the expression of genes that likely attenuate odor responses was higher in OSNs with high ES scores than in those with low ES scores, whereas genes that likely amplify responses showed the opposite trend (<a id="crosref0660" href="#fig4">Figures 4</a>C and 4D). Furthermore, persistently changing OSN activity by chronic occlusion, optogenetic stimulation, and switching environments caused functional gene expression to change bidirectionally in an OR-dependent manner (<a id="crosref0665" href="#fig4">Figures 4</a>E, 4F, and <a id="crosref0670" href="#figs4">S4</a>G–S4K; <a id="crosref0675" href="#mmc4">Table S4</a>).</div><div class="figure-wrap"><figure id="fig4" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.11.022/asset/cea09ef2-7df1-404a-8174-414ae324afb5/main.assets/gr4_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr4" src="/cms/10.1016/j.cell.2021.11.022/asset/7ac5b6c9-2170-4147-b3c2-b80e7808115e/main.assets/gr4.jpg" height="784" width="659" aria-labelledby="fig4-title fig4-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.022/asset/cea09ef2-7df1-404a-8174-414ae324afb5/main.assets/gr4_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig4-title" class="figure__title"><span class="label figure__label">Figure&nbsp;4</span> <span class="figure__title__text">Large-scale transcriptional variation is organized into an environment-dependent rheostat</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig4-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0225" role="paragraph">(A) Categorization of ES neuronal genes with high loadings in either GEP<sub>High</sub> or GEP<sub>Low</sub>, including “functional” genes that may regulate sensory responses. A subset of the 117 genes used in (B), (E), and (F) is shown.</div><div id="fspara0230" role="paragraph">(B) Heatmap of the <i>Z</i>-scored expression across OSN subtypes of 117 ES neuronal genes (including 73 functional genes).</div><div id="fspara0235" role="paragraph">(C) Expression (normalized by the total number of transcripts per cell and averaged across cells for each OSN subtype) of example functional genes associated with either GEP<sub>High</sub> (red) or GEP<sub>Low</sub> (blue), as a function of ES scores for each OSN subtype.</div><div id="fspara0240" role="paragraph">(D) Expression of 73 functional genes as a function of ES scores (binned into 50 quantiles) for each OSN subtype; plots depict the mean and standard deviation of the <i>Z</i>-scored expression of functional genes associated with GEP<sub>High</sub> (red) and GEP<sub>Low</sub> (blue).</div><div id="fspara0245" role="paragraph">(E) (Top) Heatmap of the chronic occlusion-dependent change in <i>Z</i>-scored expression of ES neuronal genes for each OSN subtype. (Bottom) Similar to (D), but for changes in both functional gene expression and ES scores.</div><div id="fspara0250" role="paragraph">(F) Similar to (E), but for environment-dependent changes in functional gene expression.</div><div id="fspara0255" role="paragraph">See also <a id="crosref1605" href="#figs4">Figure&nbsp;S4</a> and <a id="crosref1610" href="#mmc3">Tables S3</a> and <a id="crosref1615" href="#mmc4">S4</a>.</div></div></div></div></figcaption></figure></div><div class="figure-wrap"><figure id="figs4" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.11.022/asset/f656a5ec-a595-4af7-8478-5b6d198e77c0/main.assets/figs4_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs4" src="/cms/10.1016/j.cell.2021.11.022/asset/5ba6d8e4-6dc8-4d1f-9d56-06db1e1a68c8/main.assets/figs4.jpg" height="569" width="657" aria-labelledby="figs4-title figs4-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.022/asset/f656a5ec-a595-4af7-8478-5b6d198e77c0/main.assets/figs4_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs4-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S4</span> <span class="figure__title__text">Functional gene expression exhibits organized correlation structure, related to <a id="crosref1715" href="#fig4">Figure&nbsp;4</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs4-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0760" role="paragraph">(A) Heatmap showing pairwise correlations between the expression levels, across OSN subtypes, of the 117 ES neuronal genes, in data from mice housed in the home cage.</div><div id="fspara0765" role="paragraph">(B) Summary of the correlations between pairs of the 73 functional genes (see <a id="crosref1720" href="#fig4">Figure&nbsp;4</a>A and <a id="crosref1725" href="#mmc3">Table S3</a>), separated by pairs from either GEP<sub>High</sub> alone, GEP<sub>Low</sub> alone, or both GEP<sub>High</sub> and GEP<sub>Low</sub>.</div><div id="fspara0770" role="paragraph">(C) Correlation of the expression of individual functional genes and the ES score, computed across OSN subtypes, compared to correlations observed upon shuffling ES scores across OSN subtypes.</div><div id="fspara0775" role="paragraph">(D) Mean ES score and mean “functional” ES score (using GEP<sub>High</sub> and GEP<sub>Low</sub> usages calculated from the loadings of the 73 functional genes) for each OSN subtype from mice housed in the home cage.</div><div id="fspara0780" role="paragraph">(E) Accuracy of pairwise linear classifiers trained to predict OR identity using the GEP<sub>High</sub> and GEP<sub>Low</sub> usages calculated from the loadings of the 73 functional genes.</div><div id="fspara0785" role="paragraph">(F) Mean occlusion-dependent change in ES score and functional ES score for each OSN subtype.</div><div id="fspara0790" role="paragraph">(G) Similar to (B), but for occlusion-dependent changes in functional gene expression.</div><div id="fspara0795" role="paragraph">(H) Similar to (C), but for occlusion-dependent changes in ES scores and functional gene expression.</div><div id="fspara0800" role="paragraph">(I) Optogenetic stimulation-dependent changes in expression of 73 functional genes as a function of changes in ES scores (binned into 10 quantiles); plots depict the mean and SD of the <i>Z</i>-scored expression of functional genes associated with GEP<sub>High</sub> (red) and GEP<sub>Low</sub> (blue).</div><div id="fspara0805" role="paragraph">(J) Similar to (B), but for environment-dependent changes in functional gene expression.</div><div id="fspara0810" role="paragraph">(K) Similar to (C), but for environment-dependent changes in ES scores and functional gene expression.</div></div></div></div></figcaption></figure></div></section><section id="sec-2-5"><h3>Act-seq quantifies odor-evoked OSN activation</h3><div id="p0125" role="paragraph">To test the possibility that the continuous variation in functional gene expression uniquely adapts the odor responses of each OR-defined OSN subtype to each environment, we subjected the olfactory epithelium to Act-seq, a scRNA-seq variant in which neural activity is read out as a rapid change in gene expression (<span class="dropBlock reference-citations"><a id="body-ref-sref111-1" href="#" role="doc-biblioref" data-xml-rid="bib111" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref111-1" aria-controls="sref111-1">Wu et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref111-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">114.</div><div>Wu, Y.E. ∙ Pan, L. ∙ Zuo, Y. ...</div><div><strong>Detecting activated cell populations using single-cell RNA-seq</strong></div><div><em>Neuron.</em> 2017; <strong>96</strong>:313-329.e6</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_114_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=10.1016%2Fj.neuron.2017.09.026&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_114_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=10.1016%2Fj.neuron.2017.09.026&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_114_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-85031794820" target="_blank" aria-label="Open Scopus in new tab">Scopus (141)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29024657/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.neuron.2017.09.026&amp;pmid=29024657" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Act-seq reliably identified a subset of OSNs (and therefore ORs) that acutely responded after a 2-h exposure to the volatile odorants acetophenone and octanal, as assessed by immediate early gene (IEG) expression (with similar results obtained using unsupervised analysis of odor-evoked transcriptional changes; <a id="crosref0685" href="#fig5">Figures 5</a>A, 5B, and <a id="crosref0690" href="#figs5">S5</a>A; see <a id="crosref0695" href="#sec-4">STAR Methods</a>). Act-seq identified a subset of ORs (∼12%–13% of total ORs) activated by either acetophenone or octanal; these included most known responsive ORs for each odor, as well as many new acetophenone and octanal receptors (<a id="crosref0700" href="#figs5">Figures S5</a>B–S5D) (<span class="dropBlock reference-citations"><a id="body-ref-sref8" href="#" role="doc-biblioref" data-xml-rid="bib8" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref8" aria-controls="sref8">Bozza et&nbsp;al., 2002</a><div class="dropBlock__holder" data-db-target-of="sref8"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">8.</div><div>Bozza, T. ∙ Feinstein, P. ∙ Zheng, C. ...</div><div><strong>Odorant receptor expression defines functional units in the mouse olfactory system</strong></div><div><em>J.&nbsp;Neurosci.</em> 2002; <strong>22</strong>:3033-3043</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1523/JNEUROSCI.22-08-03033.2002" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11943806/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1523%2FJNEUROSCI.22-08-03033.2002&amp;pmid=11943806" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref42-1" href="#" role="doc-biblioref" data-xml-rid="bib42" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref42-1" aria-controls="sref42-1">Jiang et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref42-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">42.</div><div>Jiang, Y. ∙ Gong, N.N. ∙ Hu, X.S. ...</div><div><strong>Molecular profiling of activated olfactory neurons identifies odorant receptors for odors in&nbsp;vivo</strong></div><div><em>Nat. Neurosci.</em> 2015; <strong>18</strong>:1446-1454</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nn.4104" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-84942414428" target="_blank" aria-label="Open Scopus in new tab">Scopus (62)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26322927/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnn.4104&amp;pmid=26322927" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref78" href="#" role="doc-biblioref" data-xml-rid="bib78" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref78" aria-controls="sref78">Repicky and Luetje, 2009</a><div class="dropBlock__holder" data-db-target-of="sref78"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">80.</div><div>Repicky, S.E. ∙ Luetje, C.W.</div><div><strong>Molecular receptive range variation among mouse odorant receptors for aliphatic carboxylic acids</strong></div><div><em>J.&nbsp;Neurochem.</em> 2009; <strong>109</strong>:193-202</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1111/j.1471-4159.2009.05925.x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_80_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-61849133441" target="_blank" aria-label="Open Scopus in new tab">Scopus (14)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19166503/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1111%2Fj.1471-4159.2009.05925.x&amp;pmid=19166503" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref97-2" href="#" role="doc-biblioref" data-xml-rid="bib97" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref97-2" aria-controls="sref97-2">von der Weid et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref97-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">100.</div><div>von der Weid, B. ∙ Rossier, D. ∙ Lindup, M. ...</div><div><strong>Large-scale transcriptional profiling of chemosensory neurons identifies receptor-ligand pairs in&nbsp;vivo</strong></div><div><em>Nat. Neurosci.</em> 2015; <strong>18</strong>:1455-1463</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nn.4100" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_100_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-84942456919" target="_blank" aria-label="Open Scopus in new tab">Scopus (61)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26322926/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnn.4100&amp;pmid=26322926" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div class="figure-wrap"><figure id="fig5" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.11.022/asset/c09e0323-761b-4bc6-95e5-3c611584159d/main.assets/gr5_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr5" src="/cms/10.1016/j.cell.2021.11.022/asset/c778f799-8294-4060-a727-85fa7364af42/main.assets/gr5.jpg" height="426" width="657" aria-labelledby="fig5-title fig5-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.022/asset/c09e0323-761b-4bc6-95e5-3c611584159d/main.assets/gr5_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig5-title" class="figure__title"><span class="label figure__label">Figure&nbsp;5</span> <span class="figure__title__text">Act-seq identifies odor-responsive receptors and reveals that OSN transcriptomes determine acute <i>in&nbsp;vivo</i> odor responses</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig5-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0265" role="paragraph">(A) Schematic of the Act-seq experiment.</div><div id="fspara0270" role="paragraph">(B) <i>Z</i>-scored expression of 10 immediate early genes (IEGs, listed in <a id="crosref1620" href="#sec-4">STAR Methods</a>) for each OSN subtype from control or odor conditions (acetophenone [ACE] or octanal [OCT]), sorted for each odor. IEG expression was higher in the odor conditions (p&nbsp;&lt; 1&nbsp;× 10<sup>−18</sup> for each odor via Wilcoxon signed-rank test).</div><div id="fspara0275" role="paragraph">(C) Odor-evoked activation score for each OSN subtype, as a function of that evoked by control solvent (dipropylene glycol [DPG]). Odor-evoked activation scores are higher for responsive OSN subtypes (in color, p&nbsp;&lt; 1&nbsp;× 10<sup>−14</sup>, one-sided Wilcoxon signed-rank test for each odor).</div><div id="fspara0280" role="paragraph">(D) Schematic of the acute (2 h) optogenetic stimulation experiment.</div><div id="fspara0285" role="paragraph">(E) Distribution of mean activation scores for OSN subtypes expressing dorsal ORs in optogenetically stimulated and control conditions. Activation scores increase with stimulation frequency (p&nbsp;&lt; 1&nbsp;× 10<sup>−5</sup>, one-sided Jonckheere-Terpstra trend test).</div><div id="fspara0290" role="paragraph">(F) Percent of cells activated by acetophenone, for OSN subtypes that were responsive at 10% (and colored dark to light by the number of concentrations that activated each OSN subtype), increase with concentration (p&nbsp;&lt; 1&nbsp;× 10<sup>−5</sup>, one-sided Jonckheere-Terpstra trend test).</div><div id="fspara0295" role="paragraph">(G) Activation scores, for OSN subtypes that were responsive at 10%, increase with increasing concentrations of acetophenone (p&nbsp;&lt; 1&nbsp;× 10<sup>−5</sup>, one-sided Jonckheere-Terpstra trend test).</div><div id="fspara0300" role="paragraph">(H) Activation scores for OSNs expressing either M72 (<i>Olfr160</i>) or <i>Olfr923</i> to either acetophenone or 2-hydroxyacetophenone (2-HA) at the indicated concentrations.</div><div id="fspara0305" role="paragraph">(I) Activation scores for each odor as a function of control ES scores, for odor-responsive OSN subtypes.</div><div id="fspara0310" role="paragraph">(J) Mean-squared error of predicted activation scores via linear regression models fit on either ES scores or expression of 73 functional genes, for odor-responsive OSN subtypes. Error bars represent mean and SD across 1,000 restarts, normalized to the models fit on ES scores.</div><div id="fspara0315" role="paragraph">(K) Activation scores for acetophenone-responsive OSN subtypes whose ORs have known <i>in&nbsp;vitro</i> EC<sub>50</sub> values (<span class="dropBlock reference-citations"><a id="body-ref-sref42-2" href="#" role="doc-biblioref" data-xml-rid="bib42" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref42-2" aria-controls="sref42-2">Jiang et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref42-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">42.</div><div>Jiang, Y. ∙ Gong, N.N. ∙ Hu, X.S. ...</div><div><strong>Molecular profiling of activated olfactory neurons identifies odorant receptors for odors in&nbsp;vivo</strong></div><div><em>Nat. Neurosci.</em> 2015; <strong>18</strong>:1446-1454</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nn.4104" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-84942414428" target="_blank" aria-label="Open Scopus in new tab">Scopus (62)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26322927/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnn.4104&amp;pmid=26322927" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref97-3" href="#" role="doc-biblioref" data-xml-rid="bib97" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref97-3" aria-controls="sref97-3">von der Weid et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref97-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">100.</div><div>von der Weid, B. ∙ Rossier, D. ∙ Lindup, M. ...</div><div><strong>Large-scale transcriptional profiling of chemosensory neurons identifies receptor-ligand pairs in&nbsp;vivo</strong></div><div><em>Nat. Neurosci.</em> 2015; <strong>18</strong>:1455-1463</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nn.4100" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_100_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-84942456919" target="_blank" aria-label="Open Scopus in new tab">Scopus (61)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26322926/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnn.4100&amp;pmid=26322926" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="fspara0320" role="paragraph">(L) Similar to (I), but for optogenetically activated dorsal OSN subtypes (medium intensity in D).</div><div id="fspara0325" role="paragraph">See also <a id="crosref1635" href="#figs5">Figure&nbsp;S5</a> and <a id="crosref1640" href="#mmc5">Table S5</a>.</div></div></div></div></figcaption></figure></div><div class="figure-wrap"><figure id="figs5" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.11.022/asset/0e74ed20-4fda-4990-8300-0c5b2afc7a37/main.assets/figs5_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs5" src="/cms/10.1016/j.cell.2021.11.022/asset/23938dac-554e-4d31-9861-02715dca58f3/main.assets/figs5.jpg" height="832" width="657" aria-labelledby="figs5-title figs5-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.022/asset/0e74ed20-4fda-4990-8300-0c5b2afc7a37/main.assets/figs5_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs5-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S5</span> <span class="figure__title__text">Act-seq identified odor-activated OSNs and quantified the degree of transcriptional odor response, related to <a id="crosref1730" href="#fig5">Figure&nbsp;5</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs5-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0820" role="paragraph">(A) (Top left) UMAP plot showing mean <i>Z</i>-scored expression of 10 immediate early genes (IEGs, see <a id="crosref1735" href="#sec-4">STAR Methods</a> for list of IEGs). (Top right) Fraction of cells per OSN subtype that were activated by acetophenone (ACE) or octanal (OCT) based on IEG expression. OSN subtypes were defined as odor-responsive when at least 70 percent of cells expressing the given OR for that subtype were activated (dashed line), and these ORs were considered as activated. (Middle left) UMAP plot highlighting activated cells identified by unsupervised Leiden clustering (see <a id="crosref1740" href="#sec-4">STAR Methods</a>). (Middle right) Fraction of cells per OSN subtype that were activated based on Leiden clustering, colored by odor responsiveness as assessed by IEG expression. (Bottom left) UMAP plot depicting odor-driven activation as quantified by unsupervised tensor component analysis (TCA). (Bottom right) Distribution of TCA loadings of a factor representing odor activation (with 11-fold higher weight on the odor source compared to the control condition) for OSN subtypes for mice exposed to ACE and OCT, colored by odor responsiveness as assessed by IEG expression.</div><div id="fspara0825" role="paragraph">(B) Distribution of the mean expression of 10 IEGs across OSN subtypes, separated by odor responsiveness.</div><div id="fspara0830" role="paragraph">(C) Venn diagram depicting the overlap between ORs identified as acetophenone receptors via Act-seq, the DREAM method, and phospho-S6 immunoprecipitation (from <span class="dropBlock reference-citations"><a id="body-ref-sref97-4" href="#" role="doc-biblioref" data-xml-rid="bib97" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref97-4" aria-controls="sref97-4">von der Weid et&nbsp;al. 2015</a><div class="dropBlock__holder" data-db-target-of="sref97-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">100.</div><div>von der Weid, B. ∙ Rossier, D. ∙ Lindup, M. ...</div><div><strong>Large-scale transcriptional profiling of chemosensory neurons identifies receptor-ligand pairs in&nbsp;vivo</strong></div><div><em>Nat. Neurosci.</em> 2015; <strong>18</strong>:1455-1463</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nn.4100" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_100_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-84942456919" target="_blank" aria-label="Open Scopus in new tab">Scopus (61)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26322926/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnn.4100&amp;pmid=26322926" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref42-3" href="#" role="doc-biblioref" data-xml-rid="bib42" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref42-3" aria-controls="sref42-3">Jiang et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref42-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">42.</div><div>Jiang, Y. ∙ Gong, N.N. ∙ Hu, X.S. ...</div><div><strong>Molecular profiling of activated olfactory neurons identifies odorant receptors for odors in&nbsp;vivo</strong></div><div><em>Nat. Neurosci.</em> 2015; <strong>18</strong>:1446-1454</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nn.4104" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-84942414428" target="_blank" aria-label="Open Scopus in new tab">Scopus (62)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26322927/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnn.4104&amp;pmid=26322927" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). For the eight ORs identified by only the phospho-S6 method, three of them were not activated in any cells via Act-seq, and the remaining five were partially activated (activated in 43.8% of cells per OR on average). This analysis only includes the 717 ORs that were present in four or more OSNs in these odor-activation experiments. Although Act-seq captured nearly all such receptors that were also captured by DREAM and phospho-S6 (and many missed by both of these techniques), the 717 observed ORs included only 16 of 22 DREAM and 25 of 49 phospho-S6 ACE ORs; these “missed” receptors possibly reflect ORs that are expressed in particularly small numbers of OSNs.</div><div id="fspara0835" role="paragraph">(D) Venn diagram depicting the overlap between ORs activated by ACE and OCT. The overlap is at chance levels (odds ratio&nbsp;= 1.42, p&nbsp;= 0.29, Fisher’s exact test).</div><div id="fspara0840" role="paragraph">(E) Comparisons of acute gene expression changes induced by ACE or OCT compared to solvent-only (DPG) condition for 996 genes with significant changes (FDR ≤ 1&nbsp;× 10<sup>−3</sup>) in at least one odor.</div><div id="fspara0845" role="paragraph">(F) Mean percent of ORs activated by the indicated concentration of acetophenone.</div><div id="fspara0850" role="paragraph">(G) Mean percent of ORs activated by 10% acetophenone and its analogs (2-hydroxyacetophenone [2-HA], 4-methylacetophenone [4-MA], methyl salicylate [MS]).</div><div id="fspara0855" role="paragraph">(H) Heatmap depicting percent overlap of activated ORs among four acetophenone-related odors at 10% concentration. ORs activated by acetophenone analogs are also consistently activated by acetophenone (odds ratio &gt; 22, p&nbsp;&lt; 1&nbsp;× 10<sup>−11</sup> via Fisher’s exact test for each of three analogs).</div><div id="fspara0860" role="paragraph">(I) Correlation coefficients across individual mice of activation scores for OSN subtypes responsive to either ACE or OCT (ρ ≥ 0.89 between mice exposed to the same odor).</div><div id="fspara0865" role="paragraph">(J) Correlation coefficients comparing activation scores for OSN subtypes activated by acetophenone-related odors, shown for pairs of mice exposed to the same or different odors. The correlation is higher in mice exposed to the same odor (p&nbsp;= 0.002, Mann-Whitney U test).</div><div id="fspara0870" role="paragraph">(K) Activation scores for four acetophenone-related odors, for OSN subtypes responsive to each odor.</div><div id="fspara0875" role="paragraph">(L) Accuracy of pairwise linear classifiers trained to distinguish pairs of acetophenone-related odors based on activation scores from odor-responsive OSN subtypes (using only OSN subtypes responsive to both odors in each pair). Observed and shuffled (odor labels are shuffled 1,000 times) classification accuracies are shown.</div><div id="fspara0880" role="paragraph">(M) Accuracy of a minimum distance classification procedure to predict odor identity of the four acetophenone-related odors using OSN subtype-specific activation scores (see <a id="crosref1755" href="#sec-4">STAR Methods</a>), as a function of the number of OSN subtypes (from the set of OSN subtypes responsive to any of the four odors). The curves depict the mean and interquartile range of classifier accuracy for each odor across 1,000 restarts for the observed data, and for data in which odor labels were shuffled for each OSN subtype.</div><div id="fspara0885" role="paragraph">(N) Correlation coefficient between the activation score and the control ES score for each OSN subtype, evaluated at various thresholds used for identifying activated cells based on IEG expression and for identifying responsive ORs (fraction of cells activated). The threshold used in <a id="crosref1760" href="#fig5">Figure&nbsp;5</a>I (70% of cells per OR and IEG expression ≥0.2) is highlighted with stars.</div><div id="fspara0890" role="paragraph">(O) Correlation coefficient between the activation score and the control ES score for each OSN subtype, within and between individual mice. For each odor, the observed negative correlation between activation and control ES score is higher (p&nbsp;&lt; 1&nbsp;× 10<sup>−4</sup> via permutation test) than any value in the distribution of correlation coefficients measured after permuting the activation score across OSN subtypes for 10,000 shuffles.</div><div id="fspara0895" role="paragraph">(P) Similar to (O), but for four acetophenone-related odors (p&nbsp;&lt; 8&nbsp;× 10<sup>−3</sup> via permutation test for each odor).</div><div id="fspara0900" role="paragraph">(Q) Mean activation scores at the indicated concentration for each of the acetophenone-responsive OSN subtypes, as a function of the <i>in&nbsp;vitro</i> EC<sub>50</sub> for each OR, as reported in <span class="dropBlock reference-citations"><a id="body-ref-sref42-4" href="#" role="doc-biblioref" data-xml-rid="bib42" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref42-4" aria-controls="sref42-4">Jiang et&nbsp;al. (2015)</a><div class="dropBlock__holder" data-db-target-of="sref42-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">42.</div><div>Jiang, Y. ∙ Gong, N.N. ∙ Hu, X.S. ...</div><div><strong>Molecular profiling of activated olfactory neurons identifies odorant receptors for odors in&nbsp;vivo</strong></div><div><em>Nat. Neurosci.</em> 2015; <strong>18</strong>:1446-1454</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nn.4104" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-84942414428" target="_blank" aria-label="Open Scopus in new tab">Scopus (62)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26322927/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnn.4104&amp;pmid=26322927" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span> and <span class="dropBlock reference-citations"><a id="body-ref-sref80-1" href="#" role="doc-biblioref" data-xml-rid="bib80" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref80-1" aria-controls="sref80-1">Saito et&nbsp;al. (2009)</a><div class="dropBlock__holder" data-db-target-of="sref80-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">82.</div><div>Saito, H. ∙ Chi, Q. ∙ Zhuang, H. ...</div><div><strong>Odor coding by a mammalian receptor repertoire</strong></div><div><em>Sci. Signal.</em> 2009; <strong>2</strong>:ra9</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/scisignal.2000016" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_82_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-65949098470" target="_blank" aria-label="Open Scopus in new tab">Scopus (363)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19261596/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscisignal.2000016&amp;pmid=19261596" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span> (see also <a id="crosref1775" href="#fig5">Figure&nbsp;5</a>K).</div><div id="fspara0905" role="paragraph">(R) Normalized error for predictions (see <a id="crosref1780" href="#sec-4">STAR Methods</a>) of activation scores (both observed and shuffled) for OSN subtypes responsive to ACE at 10%, based on control ES scores, the <i>in&nbsp;vitro</i> EC<sub>50</sub> for each OR (from <span class="dropBlock reference-citations"><a id="body-ref-sref42-5" href="#" role="doc-biblioref" data-xml-rid="bib42" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref42-5" aria-controls="sref42-5">Jiang et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref42-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">42.</div><div>Jiang, Y. ∙ Gong, N.N. ∙ Hu, X.S. ...</div><div><strong>Molecular profiling of activated olfactory neurons identifies odorant receptors for odors in&nbsp;vivo</strong></div><div><em>Nat. Neurosci.</em> 2015; <strong>18</strong>:1446-1454</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nn.4104" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-84942414428" target="_blank" aria-label="Open Scopus in new tab">Scopus (62)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26322927/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnn.4104&amp;pmid=26322927" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span> and <span class="dropBlock reference-citations"><a id="body-ref-sref80-2" href="#" role="doc-biblioref" data-xml-rid="bib80" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref80-2" aria-controls="sref80-2">Saito et&nbsp;al., 2009</a><div class="dropBlock__holder" data-db-target-of="sref80-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">82.</div><div>Saito, H. ∙ Chi, Q. ∙ Zhuang, H. ...</div><div><strong>Odor coding by a mammalian receptor repertoire</strong></div><div><em>Sci. Signal.</em> 2009; <strong>2</strong>:ra9</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/scisignal.2000016" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_82_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-65949098470" target="_blank" aria-label="Open Scopus in new tab">Scopus (363)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19261596/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscisignal.2000016&amp;pmid=19261596" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), or both. Error bars represent mean and SD across 1,000 restarts.</div><div id="fspara0910" role="paragraph">(S) Similar to (O), but for data derived from the acute optogenetic stimulation experiment (p&nbsp;&lt; 1&nbsp;× 10<sup>−4</sup> via permutation test for each stimulation frequency).</div></div></div></div></figcaption></figure></div><div id="p0130" role="paragraph">Acetophenone and octanal elicited similar acute transcriptional changes across many genes (<a id="crosref0725" href="#figs5">Figure&nbsp;S5</a>E). We therefore developed an analog measure of odor response (referred to herein as the activation score) to quantitatively summarize information from the ∼500 genes (including IEGs) that were reliably and rapidly changed by odor exposure (and which lacked strong ES score correlations; <a id="crosref0730" href="#fig5">Figure&nbsp;5</a>C; <a id="crosref0735" href="#mmc5">Table S5</a>; see <a id="crosref0740" href="#sec-4">STAR Methods</a>). To test whether activation scores capture meaningful variation in activity, we subjected OSNs to optogenetic stimulation, which revealed a tonic relationship between stimulation frequency and activation scores (<a id="crosref0745" href="#fig5">Figures 5</a>D and 5E). Similarly, increasing concentrations of acetophenone tonically increased both the number of ORs activated by acetophenone and the activation scores of acetophenone-responsive OSNs (<a id="crosref0750" href="#fig5">Figures 5</a>F, 5G, and <a id="crosref0755" href="#figs5">S5</a>F).</div><div id="p0135" role="paragraph">To test whether activation scores also capture differences in binding affinities between different ligands for the same receptor, we took advantage of 2-hydroxyacetophenone (2-HA), which, relative to acetophenone, binds more strongly to the M72 receptor (<span class="dropBlock reference-citations"><a id="body-ref-sref2" href="#" role="doc-biblioref" data-xml-rid="bib2" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref2" aria-controls="sref2">Arneodo et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">2.</div><div>Arneodo, E.M. ∙ Penikis, K.B. ∙ Rabinowitz, N. ...</div><div><strong>Stimulus dependent diversity and stereotypy in the output of an olfactory functional unit</strong></div><div><em>Nat. Commun.</em> 2018; <strong>9</strong>:1347</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41467-018-03837-1" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_2_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-85045210680" target="_blank" aria-label="Open Scopus in new tab">Scopus (15)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29632302/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41467-018-03837-1&amp;pmid=29632302" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref117" href="#" role="doc-biblioref" data-xml-rid="bib117" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref117" aria-controls="sref117">Zhang et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref117"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">120.</div><div>Zhang, J. ∙ Huang, G. ∙ Dewan, A. ...</div><div><strong>Uncoupling stimulus specificity and glomerular position in the mouse olfactory system</strong></div><div><em>Mol. Cell. Neurosci.</em> 2012; <strong>51</strong>:79-88</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.mcn.2012.08.006" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_120_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-84866146006" target="_blank" aria-label="Open Scopus in new tab">Scopus (25)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22926192/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.mcn.2012.08.006&amp;pmid=22926192" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Act-seq revealed that 2-HA indeed elicits higher activation scores than acetophenone in OSNs expressing M72 (<a id="crosref0770" href="#fig5">Figure&nbsp;5</a>H). Conversely, activation scores for OSNs expressing <i>Olfr923</i>, a high-affinity acetophenone receptor, were higher for acetophenone than 2-HA (<span class="dropBlock reference-citations"><a id="body-ref-sref39" href="#" role="doc-biblioref" data-xml-rid="bib39" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref39" aria-controls="sref39">Hu et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref39"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">39.</div><div>Hu, X.S. ∙ Ikegami, K. ∙ Vihani, A. ...</div><div><strong>Concentration-dependent recruitment of mammalian odorant receptors</strong></div><div><em>eNeuro.</em> 2020; <strong>7</strong></div><div>ENEURO.0103-19.2019</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1523/ENEURO.0103-19.2019" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-85084028188" target="_blank" aria-label="Open Scopus in new tab">Scopus (5)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1523%2FENEURO.0103-19.2019" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref42-6" href="#" role="doc-biblioref" data-xml-rid="bib42" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref42-6" aria-controls="sref42-6">Jiang et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref42-6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">42.</div><div>Jiang, Y. ∙ Gong, N.N. ∙ Hu, X.S. ...</div><div><strong>Molecular profiling of activated olfactory neurons identifies odorant receptors for odors in&nbsp;vivo</strong></div><div><em>Nat. Neurosci.</em> 2015; <strong>18</strong>:1446-1454</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nn.4104" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-84942414428" target="_blank" aria-label="Open Scopus in new tab">Scopus (62)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26322927/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnn.4104&amp;pmid=26322927" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Furthermore, 2-HA, acetophenone, and two related odors each elicited a broad range of OR- and odor-specific activation scores that were stereotyped across mice and accurately predicted the specific odor each mouse experienced (<a id="crosref0785" href="#figs5">Figures S5</a>G–S5M). Taken together, these results demonstrate that Act-seq can identify the OSN subtypes (and thus the ORs) that can bind to and respond to any odorant, and that the activation score affords a quantitative transcriptional measure of the degree to which a given OSN has responded to an odor <i>in&nbsp;vivo</i>.</div></section><section id="sec-2-6"><h3>OSN transcriptomes predict acute odor responses</h3><div id="p0140" role="paragraph">Consistent with the hypothesis that the transcriptome of each OSN shapes its acute odor responses, ES scores were negatively correlated with odor activation: the higher the ES score for a given acetophenone-responsive OSN subtype (identified across experiments via its OR), the lower its acute, acetophenone-evoked response (<a id="crosref0790" href="#fig5">Figures 5</a>I, <a id="optlxpk64oRWg" href="#figs5">S5</a>N, and S5O). This negative correlation was also observed in OSNs responding to octanal and to the three acetophenone-related odors (<a id="crosref0800" href="#fig5">Figures 5</a>I, <a id="crosref0805" href="#figs5">S5</a>O, and S5P). The 73 functional genes were as effective as the ES score at predicting acute OSN odor responses, but the <i>in&nbsp;vitro</i> half-maximal effective concentration (EC<sub>50</sub>) of each acetophenone OR was unable to support <i>in&nbsp;vivo</i> odor response predictions (<a id="crosref0810" href="#fig5">Figures 5</a>J, 5K, <a id="crosref0815" href="#figs5">S5</a>Q, and S5R) (<span class="dropBlock reference-citations"><a id="body-ref-sref42-7" href="#" role="doc-biblioref" data-xml-rid="bib42" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref42-7" aria-controls="sref42-7">Jiang et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref42-7"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">42.</div><div>Jiang, Y. ∙ Gong, N.N. ∙ Hu, X.S. ...</div><div><strong>Molecular profiling of activated olfactory neurons identifies odorant receptors for odors in&nbsp;vivo</strong></div><div><em>Nat. Neurosci.</em> 2015; <strong>18</strong>:1446-1454</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nn.4104" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-84942414428" target="_blank" aria-label="Open Scopus in new tab">Scopus (62)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26322927/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnn.4104&amp;pmid=26322927" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref80-3" href="#" role="doc-biblioref" data-xml-rid="bib80" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref80-3" aria-controls="sref80-3">Saito et&nbsp;al., 2009</a><div class="dropBlock__holder" data-db-target-of="sref80-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">82.</div><div>Saito, H. ∙ Chi, Q. ∙ Zhuang, H. ...</div><div><strong>Odor coding by a mammalian receptor repertoire</strong></div><div><em>Sci. Signal.</em> 2009; <strong>2</strong>:ra9</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/scisignal.2000016" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_82_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-65949098470" target="_blank" aria-label="Open Scopus in new tab">Scopus (363)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19261596/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscisignal.2000016&amp;pmid=19261596" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Interestingly, OSN activation scores following 2&nbsp;h of acute optogenetic stimulation were also negatively correlated with control ES scores, indicating that the ability of light to transcriptionally activate each OSN also depends on the expression levels of functional genes (<a id="crosref0830" href="#fig5">Figures 5</a>L and <a id="crosref0835" href="#figs5">S5</a>S).</div><div id="p0145" role="paragraph">If functional gene expression indeed plays a causal role in determining odor responses, then experimentally raising or lowering ES scores should predictably change acute odor responses (<a id="crosref0840" href="#fig6">Figure&nbsp;6</a>A). We tested this hypothesis by lowering OSN activity via transient unilateral naris occlusion using removable plugs, and then performing Act-seq after acute odor exposure (<a id="crosref0845" href="#fig6">Figure&nbsp;6</a>B). Transient occlusion reduced ES scores in an OR-specific manner but did not change the correlation structure between functional genes (<a id="crosref0850" href="#fig6">Figures 6</a>C, <a id="crosref0855" href="#figs6">S6</a>A, and S6B). Similar sets of ORs were activated by acetophenone in control conditions and after occlusion, but acetophenone receptor-expressing OSNs derived from occluded nostrils exhibited higher activation scores (<a id="crosref0860" href="#fig6">Figures 6</a>D and <a id="crosref0865" href="#figs6">S6</a>C). Critically, the occlusion-dependent changes in ES scores and functional gene expression predicted the occlusion-dependent changes in odor-evoked activation (<a id="crosref0870" href="#fig6">Figures 6</a>E, <a id="crosref0875" href="#figs6">S6</a>D, and S6E).</div><div class="figure-wrap"><figure id="fig6" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.11.022/asset/11c2fa33-f387-4576-85c1-2b49023671a5/main.assets/gr6_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr6" src="/cms/10.1016/j.cell.2021.11.022/asset/79ac34c9-9c02-42a1-b475-25b93cb2def2/main.assets/gr6.jpg" height="817" width="657" aria-labelledby="fig6-title fig6-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.022/asset/11c2fa33-f387-4576-85c1-2b49023671a5/main.assets/gr6_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig6-title" class="figure__title"><span class="label figure__label">Figure&nbsp;6</span> <span class="figure__title__text">Changes in OSN transcriptomes predict changes in acute odor responses, and acute odor responses predict long-term transcriptome changes</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig6-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0335" role="paragraph">(A) Model depicting how occlusion- or environment-dependent changes in ES scores could affect acute odor responses.</div><div id="fspara0340" role="paragraph">(B) Schematic of transient naris occlusion followed by Act-seq.</div><div id="fspara0345" role="paragraph">(C) ES score, for the 745 OSN subtypes identified in transiently occluded (but not unplugged) mice. Occlusion decreased ES scores for 726 OSN subtypes (554 significantly at FDR ≤ 0.01).</div><div id="fspara0350" role="paragraph">(D) Activation scores for acetophenone-responsive OSN subtypes are higher in unplugged than open nostrils (p&nbsp;= 2.29&nbsp;× 10<sup>−7</sup>, Wilcoxon signed-rank test).</div><div id="fspara0355" role="paragraph">(E) Changes in activation scores between acetophenone-responsive OSN subtypes from open or unplugged nostrils, as a function of the occlusion-dependent change in ES scores for each subtype (as measured in the data in C).</div><div id="fspara0360" role="paragraph">(F) Correlation of the activation scores for acetophenone-responsive OSN subtypes, using data from either the same or different environments. Activation scores are more consistent within a given environment (p&nbsp;= 0.014, Mann-Whitney U test).</div><div id="fspara0365" role="paragraph">(G) Changes in activation scores across environments, as a function of changes in ES scores across environments, for acetophenone-responsive OSN subtypes.</div><div id="fspara0370" role="paragraph">(H) Model depicting how activation upon a shift to a novel odor environment such as environment A (envA) could subsequently change ES scores.</div><div id="fspara0375" role="paragraph">(I) Activation scores (relative to the mean across all OSN subtypes) 2&nbsp;h after the indicated environment shift. Compared to shifts between the same environment, home-envA and envA-home shifts induced opposing changes in activation scores, specifically for OSN subtypes with either significant increases or decreases in ES scores after 2&nbsp;weeks (p&nbsp;&lt; 1&nbsp;× 10<sup>−5</sup>, one-sided Jonckheere-Terpstra trend test for both; p&nbsp;= 0.662 for OSN subtypes that remained constant after 2&nbsp;weeks).</div><div id="fspara0380" role="paragraph">(J) Changes in ES scores as a function of activation scores observed 2&nbsp;h after a shift to envA, for each OSN subtype (relative to their mean across all OSN subtypes).</div><div id="fspara0385" role="paragraph">(K) Accuracy of classifiers predicting whether ES scores would rise or fall after 2&nbsp;weeks, based on the activation scores observed 2&nbsp;h after a shift to envA. For 1,000 restarts, classifiers were fit using only OSN subtypes with significant ES scores changes after 2&nbsp;weeks in envA, and the mean accuracy was summarized across OSN subtypes (error bars depict the mean and SD across restarts, accuracy is at chance levels after permuting the signs of the ES score changes).</div><div id="fspara0390" role="paragraph">(L) Mean change in ES scores (relative to the mean across all OSN subtypes) after 5 days of chronic exposure to the indicated concentration of acetophenone for OSN subtypes that were acutely responsive to 10% odor (as identified in the experiments depicted in <a id="crosref1645" href="#fig5">Figure&nbsp;5</a>). Subtypes were separated into those responding to either 10% alone or at least two concentrations (significant effect of OR responsivity [10% only versus at least two concentrations, F(1,270)&nbsp;= 43.04, p&nbsp;= 2.72&nbsp;× 10<sup>−10</sup>] and chronic acetophenone concentration [F(2,270)&nbsp;= 4.48, p&nbsp;= 0.012] on ES score changes, but the interaction was not significant [F(2,270)&nbsp;= 0.0085, p&nbsp;= 0.99], via two-way ANOVA).</div><div id="fspara0395" role="paragraph">(M) Change in ES scores (relative to the mean across all OSN subtypes) after chronic exposure to acetophenone, as a function of the acute activation score, for all acetophenone-responsive OSN subtypes identified at each concentration.</div><div id="fspara0400" role="paragraph">(N) Same as (M), but for ES score changes as a function of activation scores after acute optogenetic stimulation (medium intensity, see <a id="crosref1650" href="#fig5">Figure&nbsp;5</a>D).</div><div id="fspara0405" role="paragraph">(O) Change in ES scores (relative to the mean across all OSN subtypes), at various time points after a home-envA shift, for OSN subtypes categorized based on the sign and significance of their 2-week ES score changes.</div><div id="fspara0410" role="paragraph">(P) Similar to (M), but for changes in ES scores (relative to the mean across all OSN subtypes) induced by chronic exposure to acetophenone as a function of ES scores from mice housed in the home cage.</div><div id="fspara0415" role="paragraph">(Q) Similar to (P), but for optogenetic stimulation and plotting changes in ES scores.</div><div id="fspara0420" role="paragraph">See also <a id="crosref1655" href="#figs6">Figure&nbsp;S6</a>.</div></div></div></div></figcaption></figure></div><div class="figure-wrap"><figure id="figs6" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.11.022/asset/3390e7f4-192b-40c5-8a4e-132582c0cfe7/main.assets/figs6_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs6" src="/cms/10.1016/j.cell.2021.11.022/asset/94bdef7c-07ab-47dc-8b47-914b34ef41cc/main.assets/figs6.jpg" height="787" width="657" aria-labelledby="figs6-title figs6-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.022/asset/3390e7f4-192b-40c5-8a4e-132582c0cfe7/main.assets/figs6_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs6-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S6</span> <span class="figure__title__text">Odor responses are determined by OSN ES scores and changes in ES score are influenced by acute odor responses, related to <a id="crosref1795" href="#fig6">Figure&nbsp;6</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs6-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0920" role="paragraph">(A) Summary of the correlations between pairs of the 73 functional genes, separated by pairs from either GEP<sub>High</sub> alone, GEP<sub>Low</sub> alone, or both GEP<sub>High</sub> and GEP<sub>Low</sub>, using data from transiently occluded (but not unplugged) nostrils (see <a id="crosref1800" href="#fig6">Figure&nbsp;6</a>B).</div><div id="fspara0925" role="paragraph">(B) Mean transient occlusion-dependent changes in GEP usage across OSN subtypes.</div><div id="fspara0930" role="paragraph">(C) Fraction of OSN subtypes responsive to acetophenone (ACE) from unplugged nostrils, for OSN subtypes responsive to 10% ACE (as shown in the top right of <a id="crosref1805" href="#figs5">Figure&nbsp;S5</a>A).</div><div id="fspara0935" role="paragraph">(D) Mean changes in ES (left) or activation scores (right) between acetophenone-responsive OSN subtypes from open or unplugged nostrils, as a function of the difference in ES scores (binned into quartiles), as measured in control mice that were transiently occluded (but not unplugged). OSN subtypes whose ES scores fell more showed increased activation (p&nbsp;= 2.63&nbsp;× 10<sup>−4</sup>, Mann-Whitney U test).</div><div id="fspara0940" role="paragraph">(E) Normalized mean-squared error for predictions of the occlusion-dependent change in activation scores for OSN subtypes responsive to acetophenone, with predictions based on the occlusion-dependent changes in functional gene expression. Model errors increase if the nostril type (open versus transiently occluded) of the control mice was shuffled, or if the activation scores are shuffled across the OSN subtype. Errors were normalized to the value from the observed data, and the error bars depict the mean and SD across 1,000 restarts.</div><div id="fspara0945" role="paragraph">(F) Venn diagram, showing that ORs identified as activated by acetophenone in each environment are largely consistent but not fully overlapped; modulation of odor responses across environments may contribute to the changes in activated ORs identified in each environment via our relatively conservative thresholds (see G).</div><div id="fspara0950" role="paragraph">(G) Activation score distributions for OSN subtypes that were responsive to acetophenone in different numbers of environments (0–3). OSN subtypes that responded to acetophenone in 1–2 environments were still activated more in the other environments in which they were not considered as responsive (shown in gray) than those that were not responsive in any environment, consistent with the possibility that these receptors were activated to some extent in all environments.</div><div id="fspara0955" role="paragraph">(H) The overlap in identified OSN subtypes responsive to acetophenone between pairs of mice housed within the same or different olfactory environments. Pairs of mice exposed to acetophenone exhibit activation in overlapping sets of OSN subtypes (positive log odds), but the overlap is greater between mice that were housed within the same olfactory environment (p&nbsp;= 0.026, Mann-Whitney U test). The overlap and log odds are at chance levels after permuting the identity of ACE ORs across 1,000 restarts for each mouse pair.</div><div id="fspara0960" role="paragraph">(I) The cumulative percent of acetophenone-responsive OSN subtypes identified in each environment, colored by the number of environments in which each subtype responded.</div><div id="fspara0965" role="paragraph">(J) Mean activation scores for each acetophenone-responsive OSN subtype, sorted by each environment, for OSN subtypes responsive in all three environments.</div><div id="fspara0970" role="paragraph">(K) Correlation coefficient between the activation score and the control ES score for each acetophenone-responsive OSN subtypes, using data from mice housed in the indicated environments. The observed value is stronger than any value in the distribution of correlation coefficients measured after permuting the activation score across OSN subtypes for 10,000 shuffles (p&nbsp;&lt; 1&nbsp;× 10<sup>−4</sup> via permutation test for each environment).</div><div id="fspara0975" role="paragraph">(L) Classification accuracy for a pairwise linear classifier predicting the environment from which a given set of OSNs derived. Classification was performed for each acetophenone-responsive OSN subtype with significant ES score changes across environments, using the activation scores in each environment for that OSN subtype. Shuffle indicates permuted environment.</div><div id="fspara0980" role="paragraph">(M) Mean environment-dependent changes in activation scores, for acetophenone-responsive OSN subtypes, separated in OSN subtypes whose ES scores rose or fell across environments. OSNs whose ES scores increased in one environment relative to another exhibited lower acetophenone-driven activation scores, while those whose ES scores decreased showed the opposite trend (p&nbsp;= 1.77&nbsp;× 10<sup>−11</sup>, Mann-Whitney U test).</div><div id="fspara0985" role="paragraph">(N) Similar to (E), but for the environment-dependent changes in activation scores.</div><div id="fspara0990" role="paragraph">(O) OR expression levels after chronic exposure to the indicated concentration of acetophenone for 5&nbsp;days, normalized to the control condition (exposure to DPG for 5&nbsp;days). ORs identified as activated in none of the concentrations, at 10% alone, or at two or more concentrations by acute (2-h) Act-seq are shown separately.</div><div id="fspara0995" role="paragraph">(P) Correlation between environment-dependent changes in ES scores (relative to their mean across all OSN subtypes) after 24&nbsp;h and 2&nbsp;weeks, for individual OSN subtypes.</div><div id="fspara1000" role="paragraph">(Q) Mean change in <i>Z</i>-scored expression of 73 functional genes (relative to the mean change observed across all OSN subtypes and shown separately for genes associated with GEP<sub>High</sub> and GEP<sub>Low</sub>), at various time points after a shift to envA, for OSN subtypes categorized based on the sign and significance of their 2-week ES score changes.</div><div id="fspara1005" role="paragraph">(R) Mean ES scores for OSN subtypes from mice exposed to acetophenone or DPG for 2&nbsp;h and housed in home cages for 22 h. Acetophenone-induced changes in the ES scores were not significantly different for acetophenone-responsive OSN subtypes compared to those that did not respond to acetophenone (p&nbsp;= 0.214, Mann-Whitney U test).</div><div id="fspara1010" role="paragraph">(S) Mean expression of <i>Nr4a1</i> and <i>Fos</i> normalized by the total number of transcripts per cell for OSN subtypes, as a function of the ES scores (binned by steps of 50) for each OSN subtype in mice housed in the home cage, compared to that observed in acetophenone-responsive OSN subtypes following 2-hour exposure to acetophenone.</div></div></div></div></figcaption></figure></div><div id="p0150" role="paragraph">Similarly, switching mice from the home cage into a new odor environment (which bidirectionally modulates ES scores [<a id="crosref0880" href="#fig3">Figures 3</a>I–3N]) altered acetophenone-driven OSN activation scores in an environment-dependent manner (<a id="crosref0885" href="#fig6">Figures 6</a>F and <a id="crosref0890" href="#figs6">S6</a>F–S6I); activation scores were sufficiently different across environments and reliable across mice that they could be used to identify the environment in which a given mouse was housed (<a id="crosref0895" href="#fig6">Figures 6</a>F and <a id="crosref0900" href="#figs6">S6</a>J–S6L). Activation scores for acetophenone-responsive OSN subtypes were also negatively correlated with ES scores within a given environment, and the bidirectional changes in ES scores or functional gene expression following environment shifts predicted the corresponding change in the acetophenone response for each OSN subtype (<a id="crosref0905" href="#fig6">Figures 6</a>G, <a id="crosref0910" href="#figs6">S6</a>M, and S6N). Taken together, these results demonstrate that odor-evoked activation is not determined solely by the affinities of odors for their receptors, but rather is flexibly tuned in each OSN by functional genes, which adapt OSNs in an OR-specific manner to the specific contents of each odor environment.</div></section><section id="sec-2-7"><h3>Acute odor responses predict future changes in functional gene expression</h3><div id="p0155" role="paragraph">We wondered how different environments induce changes in OSN transcriptomes and hypothesized that acute activity triggered by new odor environments sculpts long-term functional gene expression (<a id="crosref0915" href="#fig6">Figure&nbsp;6</a>H). Simply switching mice between different environments caused OSNs to be activated, while OSN activation was negligible in mice moved between similar environments, demonstrating that OSN activation reflects salient changes in the odor environment (<a id="crosref0920" href="#fig6">Figure&nbsp;6</a>I). Furthermore, OSN-specific activation scores measured 2&nbsp;h after an environment shift predicted OSN ES score changes measured after 2&nbsp;weeks (<a id="crosref0925" href="#fig6">Figures 6</a>J and 6K). Similarly, OSN activation scores observed after acute 2-h exposure to acetophenone or optogenetic stimulation predicted ES score changes after chronic acetophenone exposure or prolonged optogenetic stimulation, respectively (<a id="crosref0930" href="#fig6">Figures 6</a>L–6N and <a id="crosref0935" href="#figs6">S6</a>O). These data demonstrate that ES scores adjust proportionally to acute perturbations in activity. Interestingly, much of the long-term change in ES scores and functional genes was apparent 4&nbsp;h after switching environments, suggesting that ES scores integrate activity on timescales of hours (<a id="crosref0940" href="#fig6">Figures 6</a>O, <a id="crosref0945" href="#figs6">S6</a>P, and S6Q). In contrast, transient 2-h exposure of mice to acetophenone did not change ES scores after 24&nbsp;h (<a id="crosref0950" href="#figs6">Figure&nbsp;S6</a>R), indicating that the ongoing presence of odors is required for persistent changes in functional gene expression.</div><div id="p0160" role="paragraph">We observed that the ES score changes observed after chronic acetophenone exposure could be predicted by the baseline ES score of each acetophenone-responsive OSN subtype in the home cage; a similar relationship was observed following optogenetic stimulation (<a id="crosref0955" href="#fig6">Figures 6</a>P and 6Q). Given that baseline ES scores also determine acute responses after odor or light stimulation (<a id="crosref0960" href="#fig5">Figures 5</a>I and 5L), these results are consistent with odor-evoked activity and ES scores being linked together in a closed loop. We noted that OSNs with high ES scores exhibited some residual IEG expression in the home cage (<a id="crosref0965" href="#figs6">Figure&nbsp;S6</a>S). This finding suggests that transcription-based adaptation in OSNs compensates for but does not fully eliminate activity driven by background odors, suggesting that under normal circumstances OSNs are continuously re-adapting to their environment.</div></section><section id="sec-2-8"><h3>Odor environments influence odor codes accessed by the brain</h3><div id="p0165" role="paragraph">To test whether the ambient environment also shapes odor codes in the brain, we performed presynaptic functional imaging of OSN axons in the olfactory bulb (<a id="crosref0970" href="#fig7">Figure&nbsp;7</a>A). OSN axons expressing the same OR converge on one or more insular structures known as glomeruli; imaging pre-synaptic OSN neural activity in olfactory bulb glomeruli can therefore characterize odor responses across different OSN subtypes. We imaged responses to a panel of purified odors before, during, and after a 2-week period in which mice were housed in one of the alternative odor environments described above (<a id="crosref0975" href="#fig7">Figures 7</a>B, <a id="crosref0980" href="#figs7">S7</a>A, and S7B). Response amplitudes to the same odor stimulus changed bidirectionally in ∼50% of glomeruli after mice were switched from the standard home cage into a new environment, consistent with prior work exploring the influence of a single odor on subsequent responses (<span class="dropBlock reference-citations"><a id="body-ref-optfKwddODQsj" href="#" role="doc-biblioref" data-xml-rid="bib120" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="optfKwddODQsj" aria-controls="optfKwddODQsj">Kass et al., 2016</a><div class="dropBlock__holder" data-db-target-of="optfKwddODQsj"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">45.</div><div>Kass, M.D. ∙ Guang, S.A. ∙ Moberly, A.H. ...</div><div><strong>Changes in Olfactory Sensory Neuron Physiology and Olfactory Perceptual Learning After Odorant Exposure in Adult Mice</strong></div><div><em>Chem. Senses.</em> 2016; <strong>41</strong>:123-133</div></div><div class="external-links"><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26514410/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?pmid=26514410" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>); these environment-dependent changes were consistently observed across both mice and odorants, and reverted when mice were later returned to standard home cages (<a id="crosref0985" href="#fig7">Figures 7</a>C–7E and <a id="crosref0990" href="#figs7">S7</a>C–S7F).</div><div class="figure-wrap"><figure id="fig7" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.11.022/asset/71fc961d-9b67-4b39-a030-034c586193ba/main.assets/gr7_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr7" src="/cms/10.1016/j.cell.2021.11.022/asset/11160ab1-5272-49e9-9661-cf78d2d34513/main.assets/gr7.jpg" height="710" width="657" aria-labelledby="fig7-title fig7-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.022/asset/71fc961d-9b67-4b39-a030-034c586193ba/main.assets/gr7_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig7-title" class="figure__title"><span class="label figure__label">Figure&nbsp;7</span> <span class="figure__title__text">The ambient odor environment determines functional odor responses</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig7-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0430" role="paragraph">(A) Schematic of the olfactory bulb imaging experiment. Scale bar, 200&nbsp;μm. Data shown in (A)–(D) are from mouse 1 (see <a id="crosref1660" href="#figs7">Figure&nbsp;S7</a>B).</div><div id="fspara0435" role="paragraph">(B) Mean <i>Z</i>-scored dF/F responses to pentanal for example glomeruli and the mean fluorescence image (in gray) for each environment. Data from glomeruli marked (a)–(e) are depicted in (C) and (D). Scale bars, 200&nbsp;μm.</div><div id="fspara0440" role="paragraph">(C) <i>Z</i>-scored dF/F responses for example glomeruli. Heatmaps depict average across trials within each session, and the means ± SEM across sessions for each environment are shown above.</div><div id="fspara0445" role="paragraph">(D) Glomerular responses to pentanal for each environment (mean ± SEM across sessions), normalized to the home-cage (home1) response, for pentanal-responsive glomeruli. Asterisks indicate glomeruli whose responses in environment A (envA) differ from those of home1 (FDR ≤ 0.01, permutation test).</div><div id="fspara0450" role="paragraph">(E) Distance between the population response to pentanal in home1 and the response in either envA or home2, as a function of time (see <a id="crosref1665" href="#sec-4">STAR Methods</a>). Shaded error bars depict the mean and SD across 1,000 restarts.</div><div id="fspara0455" role="paragraph">(F) Accuracy of pairwise linear classifiers predicting in which environment (home-cage or envA) a mouse was housed from the mean odor responses of increasing populations of glomeruli for each odor (as colored in H).</div><div id="fspara0460" role="paragraph">(G) Accuracy of linear classifiers predicting odor identity from mean glomerular responses, to each of 16 odors, using increasing populations of glomeruli. Classification in which training and test data are from either the same or different environments, and from data with shuffled environment labels, are shown separately.</div><div id="fspara0465" role="paragraph">(H) Correlation matrix summarizing pairwise relationships between mean odor responses across the population of glomeruli from mouse 5 (see <a id="crosref1670" href="#figs7">Figure&nbsp;S7</a>B).</div><div id="fspara0470" role="paragraph">(I) The difference between the correlation matrices shown in (H).</div><div id="fspara0475" role="paragraph">(J) Procedure for evaluating changes in odor codes across environments, by assessing how well a given pairwise odor classifier generalizes, across all other odor pairs, to test data from the same or different environments.</div><div id="fspara0480" role="paragraph">(K) Classification accuracy for each of the test × train pairs (120&nbsp;× 119 pairs), sorted by the accuracy when training and test data are from the same environment (within-environment). Colors depict whether training and test data are either from same or different environments, or from data with shuffled environment labels.</div><div id="fspara0485" role="paragraph">(L) Summary of (K) showing the cumulative fraction of the absolute change in generalization accuracy, relative to within-environment accuracy.</div><div id="fspara0490" role="paragraph">See also <a id="crosref1675" href="#figs7">Figure&nbsp;S7</a>.</div></div></div></div></figcaption></figure></div><div class="figure-wrap"><figure id="figs7" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.11.022/asset/a99c68d8-8f4a-4174-b172-974b118f45da/main.assets/figs7_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs7" src="/cms/10.1016/j.cell.2021.11.022/asset/cfad3233-f6e4-41d0-b5bb-a1bd18e36ff1/main.assets/figs7.jpg" height="562" width="657" aria-labelledby="figs7-title figs7-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.022/asset/a99c68d8-8f4a-4174-b172-974b118f45da/main.assets/figs7_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs7-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S7</span> <span class="figure__title__text">Odor environments determine acute functional odor responses in the olfactory bulb, related to <a id="crosref1810" href="#fig7">Figure&nbsp;7</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs7-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara1020" role="paragraph">(A) Maximum-normalized mean fluorescence for individual sessions from a single mouse (mouse 1 in B), imaged for 5&nbsp;weeks across environments (rows). Scale bar, 200&nbsp;μm.</div><div id="fspara1025" role="paragraph">(B) Pairwise enhanced correlation coefficients for the entire field of view (FOV) for all imaging sessions from each mouse (after registering FOVs across sessions, see <a id="crosref1815" href="#sec-4">STAR Methods</a>).</div><div id="fspara1030" role="paragraph">(C) Percent of odor-responsive glomeruli (see <a id="crosref1820" href="#sec-4">STAR Methods</a>), for the three indicated odors at two separate concentrations. Each circle indicates a single mouse.</div><div id="fspara1035" role="paragraph">(D) Percent of environment-sensitive glomeruli (of the glomeruli responsive to each odor) whose odor responses significantly differed (FDR ≤ 0.01 via permutation test), when mice were housed in home1 compared to environment A (envA). Odors were tested at two concentrations; pentanal-high corresponds to the concentration used in <a id="crosref1825" href="#fig7">Figures 7</a>B–7E. Each point represents a single mouse.</div><div id="fspara1040" role="paragraph">(E) Fold change in the number of glomeruli that differ significantly when comparing odor responses between home1/envA and home1/home2; each circle indicates a single mouse. A larger number of glomeruli change significantly across environments (see also F).</div><div id="fspara1045" role="paragraph">(F) Percent of glomeruli whose difference in odor response between home1/envA has the opposite sign as that observed between envA/home2, for all glomeruli significantly different between home1/envA (dots refer to individual mice); upon return to the home cage after experiencing envA, the odor responses of nearly all glomeruli are at least partially reverted.</div><div id="fspara1050" role="paragraph">(G) Percent of odor-responsive glomeruli for the mice used for decoding analysis (mouse 5 and 6 in B). Each circle indicates an odor, and colors correspond to those in <a id="crosref1830" href="#fig7">Figure&nbsp;7</a>H.</div><div id="fspara1055" role="paragraph">(H) Percent of environment-sensitive glomeruli (of the glomeruli responsive to each odor, colored as in <a id="crosref1835" href="#fig7">Figure&nbsp;7</a>H) whose odor responses significantly differed (FDR ≤ 0.01 via permutation test), when mice were housed in home1 compared to envA.</div><div id="fspara1060" role="paragraph">(I) Absolute changes in pairwise correlations for mean odor responses (see <a id="crosref1840" href="#fig7">Figures 7</a>H and 7I). Observed values and distribution of means after permuting environment labels across 10,000 shuffles are shown for each mouse (p&nbsp;&lt; 1&nbsp;× 10<sup>−4</sup> for each mouse).</div><div id="fspara1065" role="paragraph">(J) Similar to <a id="crosref1845" href="#fig7">Figure&nbsp;7</a>K. Classification accuracy for each of the test × train pairs (120&nbsp;× 119 pairs), colored by whether training and test data are from either the same or different environments, or from data with shuffled environment labels (and sorted independently for each of the three conditions).</div></div></div></div></figcaption></figure></div><div id="p0170" role="paragraph">To determine how environmental history influences odor coding, we imaged responses to a panel of 16 odors that broadly activated dorsal glomeruli (<a id="crosref0995" href="#figs7">Figure&nbsp;S7</a>G). A large fraction of glomeruli exhibited environment-specific odor response amplitudes, and linear decoders for each odor accurately distinguished between environments (<a id="crosref1000" href="#fig7">Figures 7</a>F and <a id="crosref1005" href="#figs7">S7</a>H). Odor identity could also be accurately predicted based on the glomerular responses within any given environment; however, classifiers trained in one environment performed worse across environments (<a id="crosref1010" href="#fig7">Figure&nbsp;7</a>G). Quantification of pairwise correlations in glomerular odor responses revealed that relationships for many odor pairs were modulated across environments, while close odor relationships were often preserved (<a id="crosref1015" href="#fig7">Figures 7</a>H, 7I, and <a id="crosref1020" href="#figs7">S7</a>I); similarly, classification analysis designed to quantify the relational structure in odor responses revealed diffuse changes in odor relationships across odor environments (<a id="crosref1025" href="#fig7">Figures 7</a>J–7L and <a id="crosref1030" href="#figs7">S7</a>J). Taken together, these data demonstrate that changes in OSN transcriptomes evoked by complex naturalistic environments are paralleled by pervasive environment-dependent changes in both pre-synaptic glomerular odor responses and population-level odor codes in the brain.</div></section></section><section id="sec-3"><h2>Discussion</h2><div id="p0175" role="paragraph">OSNs are thought to primarily communicate information about odor-OR interactions, which is used by the brain to facilitate odor perception (<span class="dropBlock reference-citations"><a id="body-ref-sref12" href="#" role="doc-biblioref" data-xml-rid="bib12" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref12" aria-controls="sref12">Buck, 1996</a><div class="dropBlock__holder" data-db-target-of="sref12"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">12.</div><div>Buck, L.B.</div><div><strong>Information coding in the vertebrate olfactory system</strong></div><div><em>Annu. Rev. Neurosci.</em> 1996; <strong>19</strong>:517-544</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1146/annurev.ne.19.030196.002505" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/8833453/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1146%2Fannurev.ne.19.030196.002505&amp;pmid=8833453" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref18-2" href="#" role="doc-biblioref" data-xml-rid="bib18" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref18-2" aria-controls="sref18-2">Cleland and Linster, 2005</a><div class="dropBlock__holder" data-db-target-of="sref18-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">18.</div><div>Cleland, T.A. ∙ Linster, C.</div><div><strong>Computation in the olfactory system</strong></div><div><em>Chem. Senses.</em> 2005; <strong>30</strong>:801-813</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/chemse/bji072" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-27944461858" target="_blank" aria-label="Open Scopus in new tab">Scopus (101)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16267161/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fchemse%2Fbji072&amp;pmid=16267161" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref23" href="#" role="doc-biblioref" data-xml-rid="bib23" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref23" aria-controls="sref23">de March et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref23"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">23.</div><div>de March, C.A. ∙ Titlow, W.B. ∙ Sengoku, T. ...</div><div><strong>Modulation of the combinatorial code of odorant receptor response patterns in odorant mixtures</strong></div><div><em>Mol. Cell. Neurosci.</em> 2020; <strong>104</strong>:103469</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.mcn.2020.103469" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_23_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-85080982427" target="_blank" aria-label="Open Scopus in new tab">Scopus (14)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32061665/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.mcn.2020.103469&amp;pmid=32061665" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref28" href="#" role="doc-biblioref" data-xml-rid="bib28" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref28" aria-controls="sref28">Firestein, 2001</a><div class="dropBlock__holder" data-db-target-of="sref28"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">28.</div><div>Firestein, S.</div><div><strong>How the olfactory system makes sense of scents</strong></div><div><em>Nature.</em> 2001; <strong>413</strong>:211-218</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/35093026" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-0035855833" target="_blank" aria-label="Open Scopus in new tab">Scopus (808)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11557990/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2F35093026&amp;pmid=11557990" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref41" href="#" role="doc-biblioref" data-xml-rid="bib41" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref41" aria-controls="sref41">Imai et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref41"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">41.</div><div>Imai, T. ∙ Sakano, H. ∙ Vosshall, L.B.</div><div><strong>Topographic mapping—The olfactory system</strong></div><div><em>Cold Spring Harb. Perspect. Biol.</em> 2010; <strong>2</strong>:a001776</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/cshperspect.a001776" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-78349286342" target="_blank" aria-label="Open Scopus in new tab">Scopus (55)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20554703/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2Fcshperspect.a001776&amp;pmid=20554703" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref87" href="#" role="doc-biblioref" data-xml-rid="bib87" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref87" aria-controls="sref87">Sullivan et&nbsp;al., 1995</a><div class="dropBlock__holder" data-db-target-of="sref87"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">89.</div><div>Sullivan, S.L. ∙ Ressler, K.J. ∙ Buck, L.B.</div><div><strong>Spatial patterning and information coding in the olfactory system</strong></div><div><em>Curr. Opin. Genet. Dev.</em> 1995; <strong>5</strong>:516-523</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/0959-437X(95)90057-N" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/7580145/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2F0959-437X%2895%2990057-N&amp;pmid=7580145" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref90" href="#" role="doc-biblioref" data-xml-rid="bib90" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref90" aria-controls="sref90">Touhara, 2002</a><div class="dropBlock__holder" data-db-target-of="sref90"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">93.</div><div>Touhara, K.</div><div><strong>Odor discrimination by G protein-coupled olfactory receptors</strong></div><div><em>Microsc. Res. Tech.</em> 2002; <strong>58</strong>:135-141</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/jemt.10131" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_93_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-0036668321" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/12203691/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fjemt.10131&amp;pmid=12203691" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Here, we find that each of the ∼1,000 OR-defined OSN subtypes in the mouse olfactory system adapts to the environment through a common mechanism: a transcriptional rheostat. This rheostat is composed of more than 70 genes relevant to OSN function, whose expression levels vary continuously across OSNs in an OR-dependent manner and adapt dynamically as mice traverse distinct environments. The specific position adopted by this rheostat (i.e., the OR-specific pattern of functional gene expression) predicts the odor response amplitude of a given OSN; furthermore, acute odor responses triggered by novel odor environments predict future patterns of functional gene expression (i.e., the setting of the rheostat).</div><div id="p0180" role="paragraph">These observations suggest a closed regulatory loop in which analog changes in environment-driven activity yield proportional transcriptional changes that predictably influence future neural responses to odors. Thus, rather than faithfully reporting the extent of odor-OR interactions, the peripheral olfactory system uses gene expression to instantiate expectation, thereby building odor codes that are personalized by each animal’s experience. While fast, post-translational mechanisms for sensory adaptation have been observed across modalities, and many circuit-level mechanisms have been characterized that adapt central sensory responses, our data reveal a systematic, large-scale adaptive transcriptional program that operates at the level of sensory neurons themselves (<span class="dropBlock reference-citations"><a id="body-ref-sref6-2" href="#" role="doc-biblioref" data-xml-rid="bib6" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref6-2" aria-controls="sref6-2">Benda, 2021</a><div class="dropBlock__holder" data-db-target-of="sref6-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">6.</div><div>Benda, J.</div><div><strong>Neural adaptation</strong></div><div><em>Curr. Biol.</em> 2021; <strong>31</strong>:R110-R116</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=10.1016%2Fj.cub.2020.11.054&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=10.1016%2Fj.cub.2020.11.054&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-85100374082" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33561404/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cub.2020.11.054&amp;pmid=33561404" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref13" href="#" role="doc-biblioref" data-xml-rid="bib13" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref13" aria-controls="sref13">Burns and Baylor, 2001</a><div class="dropBlock__holder" data-db-target-of="sref13"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">13.</div><div>Burns, M.E. ∙ Baylor, D.A.</div><div><strong>Activation, deactivation, and adaptation in vertebrate photoreceptor cells</strong></div><div><em>Annu. Rev. Neurosci.</em> 2001; <strong>24</strong>:779-805</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1146/annurev.neuro.24.1.779" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_13_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-0034917522" target="_blank" aria-label="Open Scopus in new tab">Scopus (317)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11520918/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1146%2Fannurev.neuro.24.1.779&amp;pmid=11520918" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref27" href="#" role="doc-biblioref" data-xml-rid="bib27" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref27" aria-controls="sref27">Fettiplace and Ricci, 2003</a><div class="dropBlock__holder" data-db-target-of="sref27"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">27.</div><div>Fettiplace, R. ∙ Ricci, A.J.</div><div><strong>Adaptation in auditory hair cells</strong></div><div><em>Curr. Opin. Neurobiol.</em> 2003; <strong>13</strong>:446-451</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/S0959-4388(03)00094-1" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-0041377628" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/12965292/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0959-4388%2803%2900094-1&amp;pmid=12965292" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref44-1" href="#" role="doc-biblioref" data-xml-rid="bib44" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref44-1" aria-controls="sref44-1">Kadohisa and Wilson, 2006</a><div class="dropBlock__holder" data-db-target-of="sref44-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">44.</div><div>Kadohisa, M. ∙ Wilson, D.A.</div><div><strong>Olfactory cortical adaptation facilitates detection of odors against background</strong></div><div><em>J.&nbsp;Neurophysiol.</em> 2006; <strong>95</strong>:1888-1896</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1152/jn.00812.2005" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-33646004295" target="_blank" aria-label="Open Scopus in new tab">Scopus (88)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16251260/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1152%2Fjn.00812.2005&amp;pmid=16251260" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref60-2" href="#" role="doc-biblioref" data-xml-rid="bib60" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref60-2" aria-controls="sref60-2">Martelli and Storace, 2021</a><div class="dropBlock__holder" data-db-target-of="sref60-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">62.</div><div>Martelli, C. ∙ Storace, D.A.</div><div><strong>Stimulus driven functional transformations in the early olfactory system</strong></div><div><em>Front. Cell. Neurosci.</em> 2021; <strong>15</strong>:684742</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.3389/fncel.2021.684742" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-85113135443" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/34413724/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.3389%2Ffncel.2021.684742&amp;pmid=34413724" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref102" href="#" role="doc-biblioref" data-xml-rid="bib102" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref102" aria-controls="sref102">Wark et&nbsp;al., 2007</a><div class="dropBlock__holder" data-db-target-of="sref102"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">105.</div><div>Wark, B. ∙ Lundstrom, B.N. ∙ Fairhall, A.</div><div><strong>Sensory adaptation</strong></div><div><em>Curr. Opin. Neurobiol.</em> 2007; <strong>17</strong>:423-429</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.conb.2007.07.001" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_105_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-34948900353" target="_blank" aria-label="Open Scopus in new tab">Scopus (324)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17714934/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.conb.2007.07.001&amp;pmid=17714934" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref118" href="#" role="doc-biblioref" data-xml-rid="bib118" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref118" aria-controls="sref118">Zufall and Leinders-Zufall, 2000</a><div class="dropBlock__holder" data-db-target-of="sref118"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">121.</div><div>Zufall, F. ∙ Leinders-Zufall, T.</div><div><strong>The cellular and molecular basis of odor adaptation</strong></div><div><em>Chem. Senses.</em> 2000; <strong>25</strong>:473-481</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/chemse/25.4.473" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/10944513/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fchemse%2F25.4.473&amp;pmid=10944513" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="p0185" role="paragraph">Our data demonstrate that the OR repertoire is densely activated by airflow and odors in our environments, suggesting that every OSN must contend with some degree of chronic activation. Because experience bidirectionally alters ES gene expression on timescales of hours, transcription-mediated adaptation most likely helps to center OSNs in their dynamic range as animals traverse different odor contexts, or as odor environments evolve during circadian cycles. As such, the adaptive mechanism identified here likely serves a distinct purpose from those that rapidly truncate responses to odor filaments (and which support, e.g., odor-guided navigation), and from central habituation mechanisms that sparsen odor representations on the minutes-long timescale but cannot restore information that is lost when OSNs operate outside their dynamic range (<span class="dropBlock reference-citations"><a id="body-ref-sref44-2" href="#" role="doc-biblioref" data-xml-rid="bib44" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref44-2" aria-controls="sref44-2">Kadohisa and Wilson, 2006</a><div class="dropBlock__holder" data-db-target-of="sref44-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">44.</div><div>Kadohisa, M. ∙ Wilson, D.A.</div><div><strong>Olfactory cortical adaptation facilitates detection of odors against background</strong></div><div><em>J.&nbsp;Neurophysiol.</em> 2006; <strong>95</strong>:1888-1896</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1152/jn.00812.2005" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-33646004295" target="_blank" aria-label="Open Scopus in new tab">Scopus (88)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16251260/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1152%2Fjn.00812.2005&amp;pmid=16251260" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref49-2" href="#" role="doc-biblioref" data-xml-rid="bib49" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref49-2" aria-controls="sref49-2">Kostal et&nbsp;al., 2008</a><div class="dropBlock__holder" data-db-target-of="sref49-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">51.</div><div>Kostal, L. ∙ Lansky, P. ∙ Rospars, J.-P.</div><div><strong>Efficient olfactory coding in the pheromone receptor neuron of a moth</strong></div><div><em>PLoS Comput. Biol.</em> 2008; <strong>4</strong>:e1000053</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pcbi.1000053" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_51_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-43249110822" target="_blank" aria-label="Open Scopus in new tab">Scopus (29)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/18437217/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pcbi.1000053&amp;pmid=18437217" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref53" href="#" role="doc-biblioref" data-xml-rid="bib53" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref53" aria-controls="sref53">Lecoq et&nbsp;al., 2009</a><div class="dropBlock__holder" data-db-target-of="sref53"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">55.</div><div>Lecoq, J. ∙ Tiret, P. ∙ Charpak, S.</div><div><strong>Peripheral adaptation codes for high odor concentration in glomeruli</strong></div><div><em>J.&nbsp;Neurosci.</em> 2009; <strong>29</strong>:3067-3072</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1523/JNEUROSCI.6187-08.2009" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-63849232987" target="_blank" aria-label="Open Scopus in new tab">Scopus (44)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19279243/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1523%2FJNEUROSCI.6187-08.2009&amp;pmid=19279243" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref60-3" href="#" role="doc-biblioref" data-xml-rid="bib60" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref60-3" aria-controls="sref60-3">Martelli and Storace, 2021</a><div class="dropBlock__holder" data-db-target-of="sref60-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">62.</div><div>Martelli, C. ∙ Storace, D.A.</div><div><strong>Stimulus driven functional transformations in the early olfactory system</strong></div><div><em>Front. Cell. Neurosci.</em> 2021; <strong>15</strong>:684742</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.3389/fncel.2021.684742" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-85113135443" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/34413724/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.3389%2Ffncel.2021.684742&amp;pmid=34413724" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref67-2" href="#" role="doc-biblioref" data-xml-rid="bib67" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref67-2" aria-controls="sref67-2">Moore, 1994</a><div class="dropBlock__holder" data-db-target-of="sref67-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">69.</div><div>Moore, P.A.</div><div><strong>A model of the role of adaptation and disadaptation in olfactory receptor neurons: Implications for the coding of temporal and intensity patterns in odor signals</strong></div><div><em>Chem. Senses.</em> 1994; <strong>19</strong>:71-86</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/chemse/19.1.71" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-0028153852" target="_blank" aria-label="Open Scopus in new tab">Scopus (42)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/8055260/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fchemse%2F19.1.71&amp;pmid=8055260" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref69" href="#" role="doc-biblioref" data-xml-rid="bib69" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref69" aria-controls="sref69">Nagel and Wilson, 2011</a><div class="dropBlock__holder" data-db-target-of="sref69"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">71.</div><div>Nagel, K.I. ∙ Wilson, R.I.</div><div><strong>Biophysical mechanisms underlying olfactory receptor neuron dynamics</strong></div><div><em>Nat. Neurosci.</em> 2011; <strong>14</strong>:208-216</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nn.2725" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-79251594106" target="_blank" aria-label="Open Scopus in new tab">Scopus (108)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21217763/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnn.2725&amp;pmid=21217763" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref108" href="#" role="doc-biblioref" data-xml-rid="bib108" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref108" aria-controls="sref108">Wilson, 2009</a><div class="dropBlock__holder" data-db-target-of="sref108"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">111.</div><div>Wilson, D.A.</div><div><strong>Olfaction as a model system for the neurobiology of mammalian short-term habituation</strong></div><div><em>Neurobiol. Learn. Mem.</em> 2009; <strong>92</strong>:199-205</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.nlm.2008.07.010" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_111_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-67349191494" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/18678264/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.nlm.2008.07.010&amp;pmid=18678264" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="p0190" role="paragraph">Transcription factors and axon guidance genes appear insufficient to effectively distinguish OSN transcriptomes or predict OR expression. Although known identity markers have been thought to only assign OSNs to broad dorsal or ventral domains (<span class="dropBlock reference-citations"><a id="body-ref-sref10-1" href="#" role="doc-biblioref" data-xml-rid="bib10" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref10-1" aria-controls="sref10-1">Bozza et al., 2009</a><div class="dropBlock__holder" data-db-target-of="sref10-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">10.</div><div>Bozza, T. ∙ Vassalli, A. ∙ Fuss, S. ...</div><div><strong>Mapping of class I and class II odorant receptors to glomerular domains by two distinct types of olfactory sensory neurons in the mouse</strong></div><div><em>Neuron.</em> 2009; <strong>61</strong>:220-233</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=10.1016%2Fj.neuron.2008.11.010&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=10.1016%2Fj.neuron.2008.11.010&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-58549092612" target="_blank" aria-label="Open Scopus in new tab">Scopus (135)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19186165/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.neuron.2008.11.010&amp;pmid=19186165" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-optUxWGoN4vCM" href="#" role="doc-biblioref" data-xml-rid="bib122" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="optUxWGoN4vCM" aria-controls="optUxWGoN4vCM">Kobayakawa et al., 2007</a><div class="dropBlock__holder" data-db-target-of="optUxWGoN4vCM"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">50.</div><div>Kobayakawa, K. ∙ Kobayakawa, R. ∙ Matsumoto, H. ...</div><div><strong>Innate versus learned odour processing in the mouse olfactory bulb</strong></div><div><em>Nature.</em> 2007; <strong>450</strong>:503-508</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature06281" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-36549040013" target="_blank" aria-label="Open Scopus in new tab">Scopus (444)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17989651/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature06281&amp;pmid=17989651" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-optwGxKihF1PF" href="#" role="doc-biblioref" data-xml-rid="bib121" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="optwGxKihF1PF" aria-controls="optwGxKihF1PF">Tan and Xie, 2018</a><div class="dropBlock__holder" data-db-target-of="optwGxKihF1PF"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">90.</div><div>Tan, L. ∙ Xie, X.S.</div><div><strong>A Near-Complete Spatial Map of Olfactory Receptors in the Mouse Main Olfactory Epithelium</strong></div><div><em>Chem. Senses.</em> 2018; <strong>43</strong>:427-432</div></div><div class="external-links"><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29796642/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?pmid=29796642" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), the dorsal, ventral, anterior, and posterior GEPs are sufficiently diverse and organized to support OR predictions (<a id="crosref1145" href="#figs2">Figures S2</a>F–S2H and <a id="crosref1150" href="#figs3">S3</a>Q). These observations suggest a fine gradation in OSN cell identities organized across the two main anatomical axes (<span class="dropBlock reference-citations"><a id="body-ref-sref10-2" href="#" role="doc-biblioref" data-xml-rid="bib10" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref10-2" aria-controls="sref10-2">Bozza et&nbsp;al., 2009</a><div class="dropBlock__holder" data-db-target-of="sref10-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">10.</div><div>Bozza, T. ∙ Vassalli, A. ∙ Fuss, S. ...</div><div><strong>Mapping of class I and class II odorant receptors to glomerular domains by two distinct types of olfactory sensory neurons in the mouse</strong></div><div><em>Neuron.</em> 2009; <strong>61</strong>:220-233</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=10.1016%2Fj.neuron.2008.11.010&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=10.1016%2Fj.neuron.2008.11.010&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-58549092612" target="_blank" aria-label="Open Scopus in new tab">Scopus (135)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19186165/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.neuron.2008.11.010&amp;pmid=19186165" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref64" href="#" role="doc-biblioref" data-xml-rid="bib64" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref64" aria-controls="sref64">Miyamichi et&nbsp;al., 2005</a><div class="dropBlock__holder" data-db-target-of="sref64"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">66.</div><div>Miyamichi, K. ∙ Serizawa, S. ∙ Kimura, H.M. ...</div><div><strong>Continuous and overlapping expression domains of odorant receptor genes in the olfactory epithelium determine the dorsal/ventral positioning of glomeruli in the olfactory bulb</strong></div><div><em>J.&nbsp;Neurosci.</em> 2005; <strong>25</strong>:3586-3592</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1523/JNEUROSCI.0324-05.2005" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-17044395103" target="_blank" aria-label="Open Scopus in new tab">Scopus (207)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15814789/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1523%2FJNEUROSCI.0324-05.2005&amp;pmid=15814789" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref75" href="#" role="doc-biblioref" data-xml-rid="bib75" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref75" aria-controls="sref75">Pacifico et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref75"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">77.</div><div>Pacifico, R. ∙ Dewan, A. ∙ Cawley, D. ...</div><div><strong>An olfactory subsystem that mediates high-sensitivity detection of volatile amines</strong></div><div><em>Cell Rep.</em> 2012; <strong>2</strong>:76-88</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=10.1016%2Fj.celrep.2012.06.006&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=10.1016%2Fj.celrep.2012.06.006&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-84864313337" target="_blank" aria-label="Open Scopus in new tab">Scopus (90)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22840399/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.celrep.2012.06.006&amp;pmid=22840399" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). OR expression levels and the usages of identity-related GEPs remained largely constant across all our experimental manipulations, indicating that functional genes are uniquely sensitive to the environment. Although the mechanisms that render ES genes sensitive to neural activity, and which coordinate expression levels across ES genes, remain to be determined, mature OSNs express several transcription factors (including <i>Ebf1</i> and <i>Lhx2</i>) that in principle could be sensitive to activity; the prominent role of heterochromatin in regulating OR expression suggests that epigenetic mechanisms involving chromatin or DNA modifications might also coordinate ES gene expression (<span class="dropBlock reference-citations"><a id="body-ref-sref65-2" href="#" role="doc-biblioref" data-xml-rid="bib65" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref65-2" aria-controls="sref65-2">Monahan and Lomvardas, 2015</a><div class="dropBlock__holder" data-db-target-of="sref65-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">67.</div><div>Monahan, K. ∙ Lomvardas, S.</div><div><strong>Monoallelic expression of olfactory receptors</strong></div><div><em>Annu. Rev. Cell Dev. Biol.</em> 2015; <strong>31</strong>:721-740</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1146/annurev-cellbio-100814-125308" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_67_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-84947294591" target="_blank" aria-label="Open Scopus in new tab">Scopus (71)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26359778/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1146%2Fannurev-cellbio-100814-125308&amp;pmid=26359778" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref66-1" href="#" role="doc-biblioref" data-xml-rid="bib66" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref66-1" aria-controls="sref66-1">Monahan et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref66-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">68.</div><div>Monahan, K. ∙ Schieren, I. ∙ Cheung, J. ...</div><div><strong>Cooperative interactions enable singular olfactory receptor expression in mouse olfactory neurons</strong></div><div><em>eLife.</em> 2017; <strong>6</strong>:e28620</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.7554/eLife.28620" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-85032933643" target="_blank" aria-label="Open Scopus in new tab">Scopus (42)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28933695/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.7554%2FeLife.28620&amp;pmid=28933695" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="p0195" role="paragraph">Environment-driven variation in functional gene expression both highlights new odor information and dynamically sculpts population-level odor representations. Given that peripheral odors codes are modified by ongoing experience, it is not clear how the brain supports odor identification and generalization across different environments. It is possible that some features of population OSN activity (e.g., timing, dynamics) are relatively insensitive to differences in functional gene expression, and/or that circuit-level attractor dynamics in the olfactory bulb and cortex cause central representations for odors to converge, despite differing inputs. Conversely, the brain may take advantage of environment-specific differences in peripheral odor codes to better understand the context in which a given stimulus was encountered. Transcription-dependent changes in both vomeronasal sensory neurons and fly olfactory receptor neurons render their odor responses sensitive to sex hormones (<span class="dropBlock reference-citations"><a id="body-ref-sref24" href="#" role="doc-biblioref" data-xml-rid="bib24" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref24" aria-controls="sref24">Dey et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref24"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">24.</div><div>Dey, S. ∙ Chamero, P. ∙ Pru, J.K. ...</div><div><strong>Cyclic regulation of sensory perception by a female hormone alters behavior</strong></div><div><em>Cell.</em> 2015; <strong>161</strong>:1334-1344</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=10.1016%2Fj.cell.2015.04.052&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=10.1016%2Fj.cell.2015.04.052&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-84930664111" target="_blank" aria-label="Open Scopus in new tab">Scopus (100)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26046438/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2015.04.052&amp;pmid=26046438" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref71" href="#" role="doc-biblioref" data-xml-rid="bib71" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref71" aria-controls="sref71">Ng et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref71"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">73.</div><div>Ng, R. ∙ Salem, S.S. ∙ Wu, S.-T. ...</div><div><strong>Amplification of <i>Drosophila</i> olfactory responses by a DEG/ENaC channel</strong></div><div><em>Neuron.</em> 2019; <strong>104</strong>:947-959.e5</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=10.1016%2Fj.neuron.2019.08.041&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=10.1016%2Fj.neuron.2019.08.041&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-85074767200" target="_blank" aria-label="Open Scopus in new tab">Scopus (20)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31629603/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.neuron.2019.08.041&amp;pmid=31629603" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), raising the possibility that internal states may similarly influence conventional mammalian OSNs and thereby further modify odor codes.</div><div id="p0200" role="paragraph">There is a close conceptual relationship between the bidirectional sensory adaptation we describe herein and firing rate homeostasis (FRH), the process through which neurons in a network maintain stable firing rates by adjusting both synaptic weights and intrinsic neural properties (<span class="dropBlock reference-citations"><a id="body-ref-sref21-2" href="#" role="doc-biblioref" data-xml-rid="bib21" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref21-2" aria-controls="sref21-2">Davis, 2006</a><div class="dropBlock__holder" data-db-target-of="sref21-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">21.</div><div>Davis, G.W.</div><div><strong>Homeostatic control of neural activity: From phenomenology to molecular design</strong></div><div><em>Annu. Rev. Neurosci.</em> 2006; <strong>29</strong>:307-323</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1146/annurev.neuro.28.061604.135751" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-33745734146" target="_blank" aria-label="Open Scopus in new tab">Scopus (400)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16776588/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1146%2Fannurev.neuro.28.061604.135751&amp;pmid=16776588" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>, <span class="dropBlock reference-citations"><a id="body-ref-sref22-3" href="#" role="doc-biblioref" data-xml-rid="bib22" href-manipulated="true" aria-label="Reference 2013" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref22-3" aria-controls="sref22-3">2013</a><div class="dropBlock__holder" data-db-target-of="sref22-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">22.</div><div>Davis, G.W.</div><div><strong>Homeostatic signaling and the stabilization of neural function</strong></div><div><em>Neuron.</em> 2013; <strong>80</strong>:718-728</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=10.1016%2Fj.neuron.2013.09.044&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=10.1016%2Fj.neuron.2013.09.044&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-84887011923" target="_blank" aria-label="Open Scopus in new tab">Scopus (165)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24183022/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.neuron.2013.09.044&amp;pmid=24183022" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref58-2" href="#" role="doc-biblioref" data-xml-rid="bib58" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref58-2" aria-controls="sref58-2">Marder, 2011</a><div class="dropBlock__holder" data-db-target-of="sref58-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">60.</div><div>Marder, E.</div><div><strong>Variability, compensation, and modulation in neurons and circuits</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2011; <strong>108</strong>:15542-15548</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1010674108" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-80053141839" target="_blank" aria-label="Open Scopus in new tab">Scopus (197)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21383190/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1010674108&amp;pmid=21383190" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref59-2" href="#" role="doc-biblioref" data-xml-rid="bib59" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref59-2" aria-controls="sref59-2">Marder and Goaillard, 2006</a><div class="dropBlock__holder" data-db-target-of="sref59-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">61.</div><div>Marder, E. ∙ Goaillard, J.-M.</div><div><strong>Variability, compensation and homeostasis in neuron and network function</strong></div><div><em>Nat. Rev. Neurosci.</em> 2006; <strong>7</strong>:563-574</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nrn1949" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-33745712893" target="_blank" aria-label="Open Scopus in new tab">Scopus (723)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16791145/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnrn1949&amp;pmid=16791145" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref93-2" href="#" role="doc-biblioref" data-xml-rid="bib93" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref93-2" aria-controls="sref93-2">Turrigiano, 2011</a><div class="dropBlock__holder" data-db-target-of="sref93-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">96.</div><div>Turrigiano, G.</div><div><strong>Too many cooks? Intrinsic and synaptic homeostatic mechanisms in cortical circuit refinement</strong></div><div><em>Annu. Rev. Neurosci.</em> 2011; <strong>34</strong>:89-103</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1146/annurev-neuro-060909-153238" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_96_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-79959889965" target="_blank" aria-label="Open Scopus in new tab">Scopus (425)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21438687/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1146%2Fannurev-neuro-060909-153238&amp;pmid=21438687" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref91-2" href="#" role="doc-biblioref" data-xml-rid="bib91" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref91-2" aria-controls="sref91-2">Turrigiano, 1999</a><div class="dropBlock__holder" data-db-target-of="sref91-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">94.</div><div>Turrigiano, G.G.</div><div><strong>Homeostatic plasticity in neuronal networks: The more things change, the more they stay the same</strong></div><div><em>Trends Neurosci.</em> 1999; <strong>22</strong>:221-227</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_94_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=10.1016%2FS0166-2236%2898%2901341-1&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_94_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=10.1016%2FS0166-2236%2898%2901341-1&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_94_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-0033135312" target="_blank" aria-label="Open Scopus in new tab">Scopus (585)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/10322495/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0166-2236%2898%2901341-1&amp;pmid=10322495" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref92" href="#" role="doc-biblioref" data-xml-rid="bib92" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref92" aria-controls="sref92">Turrigiano, 2008</a><div class="dropBlock__holder" data-db-target-of="sref92"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">95.</div><div>Turrigiano, G.G.</div><div><strong>The self-tuning neuron: Synaptic scaling of excitatory synapses</strong></div><div><em>Cell.</em> 2008; <strong>135</strong>:422-435</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_95_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=10.1016%2Fj.cell.2008.10.008&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_95_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=10.1016%2Fj.cell.2008.10.008&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_95_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-54549125798" target="_blank" aria-label="Open Scopus in new tab">Scopus (942)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/18984155/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2008.10.008&amp;pmid=18984155" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>, <span class="dropBlock reference-citations"><a id="body-ref-sref94-2" href="#" role="doc-biblioref" data-xml-rid="bib94" href-manipulated="true" aria-label="Reference 2017" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref94-2" aria-controls="sref94-2">2017</a><div class="dropBlock__holder" data-db-target-of="sref94-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">97.</div><div>Turrigiano, G.G.</div><div><strong>The dialectic of Hebb and homeostasis</strong></div><div><em>Philos. Trans. R. Soc. Lond. B Biol. Sci.</em> 2017; <strong>372</strong>:20160258</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1098/rstb.2016.0258" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_97_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-85009958893" target="_blank" aria-label="Open Scopus in new tab">Scopus (97)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28093556/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1098%2Frstb.2016.0258&amp;pmid=28093556" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Invertebrate neurons of a given type with identical firing properties can exhibit significant variation in ion channel expression, suggesting that there are many transcriptional means to achieving the same functional end (<span class="dropBlock reference-citations"><a id="body-ref-sref33" href="#" role="doc-biblioref" data-xml-rid="bib33" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref33" aria-controls="sref33">Goldman et&nbsp;al., 2001</a><div class="dropBlock__holder" data-db-target-of="sref33"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">33.</div><div>Goldman, M.S. ∙ Golowasch, J. ∙ Marder, E. ...</div><div><strong>Global structure, robustness, and modulation of neuronal models</strong></div><div><em>J.&nbsp;Neurosci.</em> 2001; <strong>21</strong>:5229-5238</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1523/JNEUROSCI.21-14-05229.2001" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11438598/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1523%2FJNEUROSCI.21-14-05229.2001&amp;pmid=11438598" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref58-3" href="#" role="doc-biblioref" data-xml-rid="bib58" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref58-3" aria-controls="sref58-3">Marder, 2011</a><div class="dropBlock__holder" data-db-target-of="sref58-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">60.</div><div>Marder, E.</div><div><strong>Variability, compensation, and modulation in neurons and circuits</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2011; <strong>108</strong>:15542-15548</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1010674108" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-80053141839" target="_blank" aria-label="Open Scopus in new tab">Scopus (197)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21383190/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1010674108&amp;pmid=21383190" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref72-1" href="#" role="doc-biblioref" data-xml-rid="bib72" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref72-1" aria-controls="sref72-1">O’Leary et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref72-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">74.</div><div>O’Leary, T. ∙ Williams, A.H. ∙ Caplan, J.S. ...</div><div><strong>Correlations in ion channel expression emerge from homeostatic tuning rules</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2013; <strong>110</strong>:E2645-E2654</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1309966110" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-84879927251" target="_blank" aria-label="Open Scopus in new tab">Scopus (111)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23798391/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1309966110&amp;pmid=23798391" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref84-1" href="#" role="doc-biblioref" data-xml-rid="bib84" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref84-1" aria-controls="sref84-1">Schulz et&nbsp;al., 2007</a><div class="dropBlock__holder" data-db-target-of="sref84-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">86.</div><div>Schulz, D.J. ∙ Goaillard, J.-M. ∙ Marder, E.E.</div><div><strong>Quantitative expression profiling of identified neurons reveals cell-specific constraints on highly variable levels of gene expression</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2007; <strong>104</strong>:13187-13191</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.0705827104" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_86_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-34548760721" target="_blank" aria-label="Open Scopus in new tab">Scopus (184)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17652510/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.0705827104&amp;pmid=17652510" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Our experiments in OSNs identify a deterministic and proportional interdependence between environment-dependent activity and functional gene expression patterns. We speculate that this tight closed-loop relationship emerges because OSNs (unlike neurons in networks) cannot manipulate synaptic weights to adapt the strength of their inputs, as input strength in OSNs is entirely determined by odor-receptor interactions. It has been proposed that structured correlations in the expression of functional genes can be used to define different cell types (<span class="dropBlock reference-citations"><a id="body-ref-sref72-2" href="#" role="doc-biblioref" data-xml-rid="bib72" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref72-2" aria-controls="sref72-2">O’Leary et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref72-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">74.</div><div>O’Leary, T. ∙ Williams, A.H. ∙ Caplan, J.S. ...</div><div><strong>Correlations in ion channel expression emerge from homeostatic tuning rules</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2013; <strong>110</strong>:E2645-E2654</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1309966110" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-84879927251" target="_blank" aria-label="Open Scopus in new tab">Scopus (111)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23798391/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1309966110&amp;pmid=23798391" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref84-2" href="#" role="doc-biblioref" data-xml-rid="bib84" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref84-2" aria-controls="sref84-2">Schulz et&nbsp;al., 2007</a><div class="dropBlock__holder" data-db-target-of="sref84-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">86.</div><div>Schulz, D.J. ∙ Goaillard, J.-M. ∙ Marder, E.E.</div><div><strong>Quantitative expression profiling of identified neurons reveals cell-specific constraints on highly variable levels of gene expression</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2007; <strong>104</strong>:13187-13191</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.0705827104" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_86_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-34548760721" target="_blank" aria-label="Open Scopus in new tab">Scopus (184)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17652510/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.0705827104&amp;pmid=17652510" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>); OSNs, whose functional genes exhibit near-perfect expression correlations along a continuum of activity levels, clearly meet this operational definition.</div><div id="p0205" role="paragraph">scRNA-seq analysis reveals a startling diversity of transcriptomes associated with neurons of a single putative type (<span class="dropBlock reference-citations"><a id="body-ref-sref47" href="#" role="doc-biblioref" data-xml-rid="bib47" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref47" aria-controls="sref47">Kim et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref47"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">48.</div><div>Kim, D.-W. ∙ Yao, Z. ∙ Graybuck, L.T. ...</div><div><strong>Multimodal analysis of cell types in a hypothalamic node controlling social behavior</strong></div><div><em>Cell.</em> 2019; <strong>179</strong>:713-728.e17</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_48_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=10.1016%2Fj.cell.2019.09.020&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_48_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=10.1016%2Fj.cell.2019.09.020&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_48_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-85073612261" target="_blank" aria-label="Open Scopus in new tab">Scopus (54)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31626771/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2019.09.020&amp;pmid=31626771" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref55" href="#" role="doc-biblioref" data-xml-rid="bib55" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref55" aria-controls="sref55">Li et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref55"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">57.</div><div>Li, H. ∙ Horns, F. ∙ Wu, B. ...</div><div><strong>Classifying <i>Drosophila</i> olfactory projection neuron subtypes by single-cell RNA sequencing</strong></div><div><em>Cell.</em> 2017; <strong>171</strong>:1206-1220.e22</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=10.1016%2Fj.cell.2017.10.019&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=10.1016%2Fj.cell.2017.10.019&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-85034065086" target="_blank" aria-label="Open Scopus in new tab">Scopus (104)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29149607/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2017.10.019&amp;pmid=29149607" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref88" href="#" role="doc-biblioref" data-xml-rid="bib88" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref88" aria-controls="sref88">Tasic et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref88"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">91.</div><div>Tasic, B. ∙ Yao, Z. ∙ Graybuck, L.T. ...</div><div><strong>Shared and distinct transcriptomic cell types across neocortical areas</strong></div><div><em>Nature.</em> 2018; <strong>563</strong>:72-78</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-018-0654-5" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-85055855595" target="_blank" aria-label="Open Scopus in new tab">Scopus (420)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30382198/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-018-0654-5&amp;pmid=30382198" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). It remains unclear whether these transcriptional differences reflect an occult diversity of neural subtypes or whether they instead reflect the ongoing activity-dependent modulation of a single underlying cell type (<span class="dropBlock reference-citations"><a id="body-ref-sref34" href="#" role="doc-biblioref" data-xml-rid="bib34" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref34" aria-controls="sref34">Gouwens et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref34"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">34.</div><div>Gouwens, N.W. ∙ Sorensen, S.A. ∙ Baftizadeh, F. ...</div><div><strong>Integrated morphoelectric and transcriptomic classification of cortical GABAergic cells</strong></div><div><em>Cell.</em> 2020; <strong>183</strong>:935-953.e19</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=10.1016%2Fj.cell.2020.09.057&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=10.1016%2Fj.cell.2020.09.057&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-85095742671" target="_blank" aria-label="Open Scopus in new tab">Scopus (50)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33186530/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2020.09.057&amp;pmid=33186530" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref48" href="#" role="doc-biblioref" data-xml-rid="bib48" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref48" aria-controls="sref48">Kim et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref48"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">49.</div><div>Kim, E.J. ∙ Zhang, Z. ∙ Huang, L. ...</div><div><strong>Extraction of distinct neuronal cell types from within a genetically continuous population</strong></div><div><em>Neuron.</em> 2020; <strong>107</strong>:274-282.e6</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=10.1016%2Fj.neuron.2020.04.018&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=10.1016%2Fj.neuron.2020.04.018&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-85085319359" target="_blank" aria-label="Open Scopus in new tab">Scopus (28)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32396852/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.neuron.2020.04.018&amp;pmid=32396852" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref81" href="#" role="doc-biblioref" data-xml-rid="bib81" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref81" aria-controls="sref81">Scala et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref81"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">83.</div><div>Scala, F. ∙ Kobak, D. ∙ Bernabucci, M. ...</div><div><strong>Phenotypic variation of transcriptomic cell types in mouse motor cortex</strong></div><div><em>Nature.</em> 2021; <strong>598</strong>:144-150</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-020-2907-3" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_83_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-85095987674" target="_blank" aria-label="Open Scopus in new tab">Scopus (18)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33184512/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-020-2907-3&amp;pmid=33184512" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Taken together, our findings—enabled by the ability to query gene expression and activity in ∼1,000 different OSN subtypes across experiments—are consistent with a general model in which neurons systematically modulate their transcriptomes to continuously adapt to their inputs.</div><section id="sec-3-1"><h3>Limitations of study</h3><div id="p0210" role="paragraph">Act-seq can reliably identify analog odor-OR relationships across much of the OR repertoire <i>in&nbsp;vivo</i>, suggesting its general utility for deorphanizing odors and receptors; however, OSNs expressing rare ORs may be missed by scRNA-seq. Our <i>in&nbsp;vivo</i> measures of acute odor-evoked neural activity depend on transcription, rather than directly querying either calcium or action potentials, and may not fully capture the activity state of each OSN. While our data strongly suggest that functional gene expression alters odor-spike coupling, it may also alter the gain of activity-transcription coupling. Furthermore, the mechanisms through which these functional genes, as well as potentially other genes, adapt OSNs remain to be determined. The significant environment-dependent changes in axonal calcium responses in the olfactory bulb roughly match observed transcriptional changes in OSNs; however, it is possible that some aspect of these changes is caused by pre-synaptic inhibition.</div></section></section><section id="sec-4"><h2>STAR★Methods</h2><section id="sec-4-1"><h3>Key resources table</h3><div id="p0225" role="paragraph"><div class="figure-wrap"><figure id="undtbl1" class="table" data-figureviewer-ignore="true"><div class="table-wrap"><table data-figureviewer-ignore="true" tabindex="0"><thead><tr class="rowsep"><th>REAGENT or RESOURCE</th><th>SOURCE</th><th>IDENTIFIER</th></tr></thead><tbody><tr class="rowsep"><td colspan="3"><b>Biological samples</b></td></tr><tr class="rowsep"><td>Mouse olfactory epithelium</td><td>This paper</td><td>N/A</td></tr><tr class="rowsep"><td colspan="3"><b>Chemicals, peptides, and recombinant proteins</b></td></tr><tr><td>Pentanal</td><td>Sigma-Aldrich</td><td>Cat# 110132</td></tr><tr><td>Pentanone</td><td>Sigma-Aldrich</td><td>Cat# 471194</td></tr><tr><td>Ethyl butyrate</td><td>Sigma-Aldrich</td><td>Cat# W242713</td></tr><tr><td>Heptanal</td><td>Sigma-Aldrich</td><td>Cat# W254002</td></tr><tr><td>Hexanal</td><td>Sigma-Aldrich</td><td>Cat# 115606</td></tr><tr><td>Propanal</td><td>Sigma-Aldrich</td><td>Cat# 538124</td></tr><tr><td>Butanal</td><td>Sigma-Aldrich</td><td>Cat# W217018</td></tr><tr><td>Butanone</td><td>Sigma-Aldrich</td><td>Cat# W217018</td></tr><tr><td>Hexanone</td><td>Sigma-Aldrich</td><td>Cat# 103004</td></tr><tr><td>Ethyl acetate</td><td>Sigma-Aldrich</td><td>Cat# 270989</td></tr><tr><td>Propyl acetate</td><td>Sigma-Aldrich</td><td>Cat# 537438</td></tr><tr><td>Butyl acetate</td><td>Sigma-Aldrich</td><td>Cat# 287725</td></tr><tr><td>Benzaldehyde</td><td>Sigma-Aldrich</td><td>Cat# B1334</td></tr><tr><td>Tiglaldehyde</td><td>Sigma-Aldrich</td><td>Cat# W340707</td></tr><tr><td>Methyl tiglate</td><td>Thermo Fisher</td><td>Cat# T024825ML</td></tr><tr><td>Methyl valerate</td><td>Sigma-Aldrich</td><td>Cat# W275204</td></tr><tr><td>Acetophonone</td><td>Sigma-Aldrich</td><td>Cat# A10701</td></tr><tr><td>2-Hydroxyacetophenone</td><td>Sigma-Aldrich</td><td>Cat# H18607</td></tr><tr><td>4-Methylacetophenone</td><td>Sigma-Aldrich</td><td>Cat# W267708</td></tr><tr><td>Methyl salicylate</td><td>Sigma-Aldrich</td><td>Cat# M6752</td></tr><tr><td>Octanal</td><td>Sigma-Aldrich</td><td>Cat# O5608</td></tr><tr><td>Mineral oil</td><td>Sigma-Aldrich</td><td>Cat# M5904</td></tr><tr><td>Dipropylene glycol</td><td>Sigma-Aldrich</td><td>Cat# D215554</td></tr><tr><td>Actinomycin D</td><td>Sigma-Aldrich</td><td>Cat# A9415</td></tr><tr><td>Triptolide</td><td>Sigma-Aldrich</td><td>Cat# T3652</td></tr><tr><td>Anisomycin</td><td>Sigma-Aldrich</td><td>Cat# A9789</td></tr><tr><td>Papain</td><td>Worthington</td><td>Cat# LK003178</td></tr><tr><td>DNase I</td><td>Worthington</td><td>Cat# LK003172</td></tr><tr><td>EBSS</td><td>Worthington</td><td>Cat# LK003188</td></tr><tr><td>Hibernate-A</td><td>Thermo Fisher</td><td>Cat# A1247501</td></tr><tr><td>Heat-inactivated Fetal Bovine Serum</td><td>Thermo Fisher</td><td>Cat# 16140071</td></tr><tr><td>Propidium iodide</td><td>Thermo Fisher</td><td>Cat# P1304MP</td></tr><tr class="rowsep"><td>PBS</td><td>Thermo Fisher</td><td>Cat# 70011044</td></tr><tr class="rowsep"><td colspan="3"><b>Critical commercial assays</b></td></tr><tr><td>Chromium Next GEM Single Cell 3′ Kit v3.1</td><td>10x genomics</td><td>Cat# 1000268</td></tr><tr><td>Chromium Next GEM Chip G Single Cell Kit</td><td>10x genomics</td><td>Cat# 1000120</td></tr><tr><td>Dual Index Kit TT Set A</td><td>10x genomics</td><td>Cat# 1000215</td></tr><tr><td>Agilent High Sensitivity DNA kit</td><td>Agilent</td><td>Cat# 5067-4626</td></tr><tr class="rowsep"><td>KAPA Library Quantification Kit, illumina platform, Complete kit (Universal)</td><td>Roche</td><td>Cat# KK4824</td></tr><tr class="rowsep"><td colspan="3"><b>Deposited data</b></td></tr><tr class="rowsep"><td>Raw and analyzed sequencing data</td><td>This paper</td><td>GEO: <a id="intref0010" href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE173947" target="_blank">GSE173947</a></td></tr><tr class="rowsep"><td colspan="3"><b>Experimental models: Organisms/strains</b></td></tr><tr><td>Mouse: C57BL/6J</td><td>The Jackson Laboratory</td><td>JAX: 000664, RRID:IMSR_JAX:000664</td></tr><tr><td>Mouse: OMP-IRES-GFP</td><td>The Jackson Laboratory</td><td>JAX: 006667, RRID:IMSR_JAX:006667</td></tr><tr><td>Mouse: B6J.Cg-Gt(ROSA)26Sor<sup>tm95.1(CAG-GCaMP6f)Hze</sup>/MwarJ</td><td>The Jackson Laboratory</td><td>JAX:028865, RRID:IMSR_JAX:028865</td></tr><tr><td>Mouse: Arrb2<sup>tm1Rjl</sup>/J</td><td>The Jackson Laboratory</td><td>JAX:011130, RRID:IMSR_JAX:011130</td></tr><tr><td>Mouse: B6;129P2-Olfr160<sup>tm11(Olfr545)Mom</sup>/MomJ (S50<sub>(M72 locus)</sub>)</td><td>Thomas Bozza</td><td>JAX:006714, RRID:IMSR_JAX:006714</td></tr><tr><td>Mouse: B6;129P2-Olfr545<sup>tm3(Olfr160)Mom</sup>/MomJ (M72<sub>(S50 locus)</sub>)</td><td>Thomas Bozza</td><td>JAX:006715, RRID:IMSR_JAX:006715</td></tr><tr><td>Mouse: Olfr17<sup>tm7Mom</sup>/MomJ</td><td>Stavros Lomvardas</td><td>JAX:006669, RRID:IMSR_JAX:006669</td></tr><tr><td>Mouse: OMP-IRES-Cre</td><td>Richard Axel</td><td>MGI:<a id="intref0015" href="https://www.informatics.jax.org/accession/MGI:3045687" target="_blank">3045687</a></td></tr><tr class="rowsep"><td>Mouse: OMP-ChR2(H134R)-Venus</td><td>Thomas Bozza and Dmitry Rinberg</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref54-1" href="#" role="doc-biblioref" data-xml-rid="bib54" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref54-1" aria-controls="sref54-1">Li et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref54-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">56.</div><div>Li, A. ∙ Gire, D.H. ∙ Bozza, T. ...</div><div><strong>Precise detection of direct glomerular input duration by the olfactory bulb</strong></div><div><em>J.&nbsp;Neurosci.</em> 2014; <strong>34</strong>:16058-16064</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1523/JNEUROSCI.3382-14.2014" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_56_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-84911870900" target="_blank" aria-label="Open Scopus in new tab">Scopus (37)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25429146/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1523%2FJNEUROSCI.3382-14.2014&amp;pmid=25429146" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td></tr><tr class="rowsep"><td colspan="3"><b>Software and algorithms</b></td></tr><tr><td>Python 3.6–3.8</td><td>Python</td><td><a id="intref0020" href="https://www.python.org/" target="_blank">https://www.python.org/</a></td></tr><tr><td>MATLAB (versions 2014a and 2015a)</td><td>Mathworks</td><td><a id="intref0025" href="https://www.mathworks.com/" target="_blank">https://www.mathworks.com/</a></td></tr><tr><td>ScanImage 5</td><td>Vidirio Technologies</td><td><a id="intref0030" href="http://scanimage.vidriotechnologies.com/display/SIH/ScanImage+Home" target="_blank">http://scanimage.vidriotechnologies.com/display/SIH/ScanImage+Home</a></td></tr><tr><td>Wavesurfer 0.775</td><td>Howard Hughes Medical Institute Janelia Research Campus</td><td><a id="intref0035" href="https://wavesurfer.janelia.org/" target="_blank">https://wavesurfer.janelia.org/</a></td></tr><tr><td>Suite2p</td><td>Howard Hughes Medical Institute Janelia Research Campus</td><td><a id="intref0040" href="https://github.com/mouseland/suite2p" target="_blank">https://github.com/mouseland/suite2p</a></td></tr><tr><td>Adobe Illustrator</td><td>Adobe</td><td>RRID:<a id="intref0045" href="https://scicrunch.org/resources/Any/search?q=undefined&amp;l=SCR_010279" target="_blank">SCR_010279</a>, <a id="intref0050" href="https://www.adobe.com/products/illustrator.html" target="_blank">https://www.adobe.com/products/illustrator.html</a></td></tr><tr><td>Cell Ranger (2.2.0 for v2 and v3 data and 4.0.0 for v3.1 data)</td><td>10x genomics</td><td><a id="intref0055" href="https://support.10xgenomics.com/single-cell-gene-expression/software/overview/welcome" target="_blank">https://support.10xgenomics.com/single-cell-gene-expression/software/overview/welcome</a></td></tr><tr><td>Scanpy 1.6.0</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref110-1" href="#" role="doc-biblioref" data-xml-rid="bib110" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref110-1" aria-controls="sref110-1">Wolf et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref110-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">113.</div><div>Wolf, F.A. ∙ Angerer, P. ∙ Theis, F.J.</div><div><strong>SCANPY: Large-scale single-cell gene expression data analysis</strong></div><div><em>Genome Biol.</em> 2018; <strong>19</strong>:15</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13059-017-1382-0" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_113_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-85041394976" target="_blank" aria-label="Open Scopus in new tab">Scopus (758)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29409532/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13059-017-1382-0&amp;pmid=29409532" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0060" href="https://github.com/theislab/scanpy" target="_blank">https://github.com/theislab/scanpy</a></td></tr><tr><td rowspan="2">Python analysis scripts</td><td rowspan="2">This paper</td><td>Zenodo: <a id="intref0065" href="https://doi.org/10.5281/zenodo.5550454" target="_blank">https://doi.org/10.5281/zenodo.5550454</a></td></tr><tr class="rowsep"><td>GitHub: <a id="intref0070" href="https://github.com/dattalab/Tsukahara_Brann_OSN" target="_blank">https://github.com/dattalab/Tsukahara_Brann_OSN</a></td></tr><tr class="rowsep"><td colspan="3"><b>Other</b></td></tr><tr><td>Teensy 3.2</td><td>Adafruit</td><td>Cat# 2756</td></tr><tr><td>blue LED</td><td>Digikey</td><td>Cat# 1416-1052-6-ND</td></tr></tbody></table></div><figcaption><div class="core-table-tools"><ul><li><a class="core-table-show-action" href="/action/showFullTableHTML?isHtml=true&amp;tableId=undtbl1&amp;pii=S0092-8674%2821%2901337-4" target="_blank">Open table in a new tab</a></li></ul></div></figcaption></figure></div></div></section><section id="sec-4-2"><h3>Resource availability</h3><section id="sec-4-2-1"><h4>Lead contact</h4><div id="p0230" role="paragraph">Further information and requests for resources and reagents should be directed to and will be fulfilled by the Lead Contact, Sandeep R. Datta (<a id="intref0075" href="mailto:srdatta@hms.harvard.edu">srdatta@hms.harvard.edu</a>).</div></section><section id="sec-4-2-2"><h4>Materials availability</h4><div id="p0235" role="paragraph">This study did not generate new unique reagents.</div></section></section><section id="sec-4-3"><h3>Experimental model and subject details</h3><section id="sec-4-3-1"><h4>Mice</h4><div id="p0240" role="paragraph">C57BL/6J, OMP-IRES-GEP, Ai95D (GCaMP6f) and β-arrestin2 knockout mice were obtained from Jackson Laboratory (stock number 000664, 006667, 028865, 011130). “OR-swap” mice (M72<sub>(S50 locus)</sub> and S50<sub>(M72 locus)</sub>) were maintained in the Bozza laboratory (and are available from Jackson Laboratory with stock numbers 006715, 006714) (<span class="dropBlock reference-citations"><a id="body-ref-sref10-3" href="#" role="doc-biblioref" data-xml-rid="bib10" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref10-3" aria-controls="sref10-3">Bozza et&nbsp;al., 2009</a><div class="dropBlock__holder" data-db-target-of="sref10-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">10.</div><div>Bozza, T. ∙ Vassalli, A. ∙ Fuss, S. ...</div><div><strong>Mapping of class I and class II odorant receptors to glomerular domains by two distinct types of olfactory sensory neurons in the mouse</strong></div><div><em>Neuron.</em> 2009; <strong>61</strong>:220-233</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=10.1016%2Fj.neuron.2008.11.010&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=10.1016%2Fj.neuron.2008.11.010&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-58549092612" target="_blank" aria-label="Open Scopus in new tab">Scopus (135)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19186165/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.neuron.2008.11.010&amp;pmid=19186165" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). P2-IRES-GFP mice (P2<sub>(P2 locus)</sub>) were obtained from the Lomvardas laboratory (and are available from Jackson Laboratory with stock number 006669) (<span class="dropBlock reference-citations"><a id="body-ref-sref26" href="#" role="doc-biblioref" data-xml-rid="bib26" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref26" aria-controls="sref26">Feinstein and Mombaerts, 2004</a><div class="dropBlock__holder" data-db-target-of="sref26"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">26.</div><div>Feinstein, P. ∙ Mombaerts, P.</div><div><strong>A contextual model for axonal sorting into glomeruli in the mouse olfactory system</strong></div><div><em>Cell.</em> 2004; <strong>117</strong>:817-831</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_26_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=10.1016%2Fj.cell.2004.05.011&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_26_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=10.1016%2Fj.cell.2004.05.011&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_26_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-2942604879" target="_blank" aria-label="Open Scopus in new tab">Scopus (246)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15186781/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2004.05.011&amp;pmid=15186781" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). OMP-IRES-Cre mice were obtained from the Axel laboratory (<span class="dropBlock reference-citations"><a id="body-ref-sref25" href="#" role="doc-biblioref" data-xml-rid="bib25" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref25" aria-controls="sref25">Eggan et&nbsp;al., 2004</a><div class="dropBlock__holder" data-db-target-of="sref25"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">25.</div><div>Eggan, K. ∙ Baldwin, K. ∙ Tackett, M. ...</div><div><strong>Mice cloned from olfactory sensory neurons</strong></div><div><em>Nature.</em> 2004; <strong>428</strong>:44-49</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature02375" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-1542347544" target="_blank" aria-label="Open Scopus in new tab">Scopus (298)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/14990966/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature02375&amp;pmid=14990966" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). OMP-ChR2(H134R)-Venus mice were maintained in the Bozza laboratory and obtained from the Rinberg laboratory (<span class="dropBlock reference-citations"><a id="body-ref-sref54-2" href="#" role="doc-biblioref" data-xml-rid="bib54" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref54-2" aria-controls="sref54-2">Li et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref54-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">56.</div><div>Li, A. ∙ Gire, D.H. ∙ Bozza, T. ...</div><div><strong>Precise detection of direct glomerular input duration by the olfactory bulb</strong></div><div><em>J.&nbsp;Neurosci.</em> 2014; <strong>34</strong>:16058-16064</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1523/JNEUROSCI.3382-14.2014" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_56_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-84911870900" target="_blank" aria-label="Open Scopus in new tab">Scopus (37)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25429146/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1523%2FJNEUROSCI.3382-14.2014&amp;pmid=25429146" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Mice of either sex between 6–16&nbsp;weeks-old were used for experiments. Mouse husbandry and experiments were performed following institutional and federal guidelines and were approved by Harvard Medical School’s Institutional Animal Care and Use Committee.</div></section></section><section id="sec-4-4"><h3>Method details</h3><section id="sec-4-4-1"><h4>Chronic naris occlusion</h4><div id="p0245" role="paragraph">To identify gene expression programs sensitive to ongoing activity from environmental odorants, 7&nbsp;day old mice were anesthetized on ice, and one of the two nostrils was occluded by cautery, as previously described (<span class="dropBlock reference-citations"><a id="body-ref-sref29-3" href="#" role="doc-biblioref" data-xml-rid="bib29" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref29-3" aria-controls="sref29-3">Fischl et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref29-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">29.</div><div>Fischl, A.M. ∙ Heron, P.M. ∙ Stromberg, A.J. ...</div><div><strong>Activity-dependent genes in mouse olfactory sensory neurons</strong></div><div><em>Chem. Senses.</em> 2014; <strong>39</strong>:439-449</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/chemse/bju015" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-84901414304" target="_blank" aria-label="Open Scopus in new tab">Scopus (13)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24692514/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fchemse%2Fbju015&amp;pmid=24692514" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). This age was chosen to minimize deficits in axon targeting that occur from manipulating activity levels at earlier post-natal time points (<span class="dropBlock reference-citations"><a id="body-ref-sref57" href="#" role="doc-biblioref" data-xml-rid="bib57" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref57" aria-controls="sref57">Ma et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref57"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">59.</div><div>Ma, L. ∙ Wu, Y. ∙ Qiu, Q. ...</div><div><strong>A developmental switch of axon targeting in the continuously regenerating mouse olfactory system</strong></div><div><em>Science.</em> 2014; <strong>344</strong>:194-197</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.1248805" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-84897975541" target="_blank" aria-label="Open Scopus in new tab">Scopus (45)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24723610/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.1248805&amp;pmid=24723610" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Unilateral naris occlusion was confirmed using a dissection microscope. After ∼1&nbsp;month, mice were used for scRNA-seq experiments. Cells isolated from occluded and open nostrils were processed and analyzed separately.</div></section><section id="sec-4-4-2"><h4>Odor exposure (Act-seq)</h4><div id="p0250" role="paragraph">Act-seq (<span class="dropBlock reference-citations"><a id="body-ref-sref111-2" href="#" role="doc-biblioref" data-xml-rid="bib111" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref111-2" aria-controls="sref111-2">Wu et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref111-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">114.</div><div>Wu, Y.E. ∙ Pan, L. ∙ Zuo, Y. ...</div><div><strong>Detecting activated cell populations using single-cell RNA-seq</strong></div><div><em>Neuron.</em> 2017; <strong>96</strong>:313-329.e6</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_114_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=10.1016%2Fj.neuron.2017.09.026&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_114_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=10.1016%2Fj.neuron.2017.09.026&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_114_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-85031794820" target="_blank" aria-label="Open Scopus in new tab">Scopus (141)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29024657/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.neuron.2017.09.026&amp;pmid=29024657" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) was performed following odor exposure to quantify odor-evoked responses in each OSN subtype (defined by their expressed OR) across the entire OR repertoire. Mice were habituated to a reversed light cycle for at least for 1&nbsp;week before odor exposure, transferred to a new disposable cage with regular bedding/food and kept overnight in a satellite animal facility. On the experimental day, each mouse was first transferred to a new empty disposable cage during the dark cycle and habituated for 20&nbsp;minutes. Empty cages were used to avoid any activation by odorants present within the environments in which the mice were housed. Odors were added to a piece of filter paper (100&nbsp;μL of odorant) in a 35&nbsp;mm Petri dish and a cotton ball (200&nbsp;μL of odorant) and placed in each cage. After 2 hours, mice were euthanized, and dissociation was performed as described below. Odors used in this study were as follows. Dipropylene glycol (DPG, control solvent), and 10% of acetophenone, octanal, 2-hydroxy acetophenone, 4-methyl acetophenone, and methyl salicylate. As a control, mice were exposed to DPG alone for 30&nbsp;min. This control condition was used to account for any drift in transcriptomes during the overnight housing and habituation periods prior to odor exposure and designed to capture the transcriptome in the state it would have been in the odor-exposed animals prior to odor exposure. The 30&nbsp;min DPG control was used for all Act-seq experiments, except following transient naris occlusion (see below); in the Act-seq experiments our results did not depend upon the specific control condition we used, which likely reflects the relative stability of the transcriptome for each OSN subtype. Act-seq was also performed using lower concentrations of acetophenone (1.0, 0.1, 0.01%) and 2-hydroxy acetophenone (0.1%, 0.01%). All odor dilutions were made with DPG, and all odors and DPG were obtained from Sigma. To assess the persistence of any transcriptional changes observed as a result of odor exposure, a cohort of mice was exposed to acetophenone or DPG for 2 hours, transferred to new clean regular home cages, and subjected to scRNA-seq after an additional 22 hours.</div></section><section id="sec-4-4-3"><h4>Transient naris occlusion</h4><div id="p0255" role="paragraph">Because odor responses in occluded OSNs cannot be examined in the chronic occlusion experiment due to permanent cautery, transient unilateral naris occlusion was performed using removable nasal plugs. Adult mice (6-12&nbsp;weeks old) were anesthetized by isoflurane and removal nasal plugs were inserted into one of the two nostrils, as previously described (<span class="dropBlock reference-citations"><a id="body-ref-sref32" href="#" role="doc-biblioref" data-xml-rid="bib32" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref32" aria-controls="sref32">Galliano et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref32"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">32.</div><div>Galliano, E. ∙ Hahn, C. ∙ Browne, L.P.R. ...</div><div><strong>Brief sensory deprivation triggers cell type-specific structural and functional plasticity in olfactory bulb neurons</strong></div><div><em>J.&nbsp;Neurosci.</em> 2021; <strong>41</strong>:2135-2151</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1523/JNEUROSCI.1606-20.2020" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-85102909527" target="_blank" aria-label="Open Scopus in new tab">Scopus (2)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33483429/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1523%2FJNEUROSCI.1606-20.2020&amp;pmid=33483429" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref45-2" href="#" role="doc-biblioref" data-xml-rid="bib45" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref45-2" aria-controls="sref45-2">Kass et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref45-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">46.</div><div>Kass, M.D. ∙ Pottackal, J. ∙ Turkel, D.J. ...</div><div><strong>Changes in the neural representation of odorants after olfactory deprivation in the adult mouse olfactory bulb</strong></div><div><em>Chem. Senses.</em> 2013; <strong>38</strong>:77-89</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/chemse/bjs081" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-84871317604" target="_blank" aria-label="Open Scopus in new tab">Scopus (12)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23125347/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fchemse%2Fbjs081&amp;pmid=23125347" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The occlusion was confirmed by measuring airflow from the occluded nostril via a thermocouple. After 5&nbsp;days, a subset of mice was used for scRNA-seq experiment directly (control mice) to assess the effects of naris occlusion on changes in gene expression. Under brief anesthesia by isoflurane, the remaining mice were unplugged and transferred to a new empty disposable cage containing either DPG alone or 10% acetophenone. After 2 hours, mice were subjected to scRNA-seq. Each of the two nostrils was used separately in both control (open and plugged) and odor-exposed mice (open and unplugged). Mice whose nostrils did not experience airflow after being unplugged, as assessed transcriptionally, were excluded from any analyses.</div></section><section id="sec-4-4-4"><h4>Environment switch experiments</h4><div id="p0260" role="paragraph">To examine environment-dependent changes in gene expression and their effects on odor-evoked responses, Act-Seq was performed 2&nbsp;weeks after environment switches. Adult mice (6–12&nbsp;weeks old) were group-housed in the regular home-cage environment for at least one week before the transfer to their novel environments. Individual mice were transferred to a new disposable cage containing materials for a novel environment in the satellite animal facility. Two different novel environments were used, with each containing non-overlapping contents: Environment A (paper bedding, hay, dried flowers (such as marigold), peanuts, seeds (e.g.,&nbsp;sunflower seeds, pumpkin seeds), puffed rice), and Environment B (garden soil, aspen shavings, coconut husks, dried fruits (e.g., berries, mango, pineapple), dried vegetables (e.g., green peas, corn, bell peppers), millet, fresh fruits (banana, apple, peach), corn flakes). Mice were singly-housed throughout the entire 2&nbsp;weeks and were transferred to newly prepared environmental cages every 12 hours to minimize the contributions of any murine odors, which are likely the predominant odorants present within standard home-cage housing environments. After 2&nbsp;weeks, individual mice were transferred to new empty disposable cages, and Act-Seq using 10% acetophenone was performed as above.</div><div id="p0265" role="paragraph">To assess the dynamics of gene expression changes in response to novel environments, mice were housed overnight in a new disposable cage with regular bedding/food in a satellite animal facility. On the experimental day, each mouse was first transferred to Environment A cages and subjected to scRNA-seq after 45&nbsp;minutes, 2, 4, 12, 24 hours, or 3, 5, 14&nbsp;days (with cages refreshed every 12 hours for durations longer than 12 hours). To account for any non-specific effects resulting from transferring mice from the animal facility to disposable cages within the satellite facility, data from mice transferred and housed in new disposable cages with regular bedding and food overnight were used as controls for downstream analyses of the effects of Environment A on gene expression.</div><div id="p0270" role="paragraph">For β-arrestin2 knockout mice, scRNA-seq was performed on cohorts of mice that were either housed in home cages, housed in new disposable cages with regular bedding and food overnight and then transferred to Environment A cages for 5&nbsp;days, or housed in new disposable cages with regular bedding and food overnight as controls, as described above. Each of the three knockout datasets was compared to its respective dataset from wild-type mice housed in the same condition.</div><div id="p0275" role="paragraph">To determine whether OSN activation reflects salient differences between environments, one set of mice were housed in a new disposable cage with regular bedding/food overnight in a satellite animal facility and then transferred to Environment A for 5&nbsp;days. On the experimental day, mice were transferred to either a new Environment A cage or a home cage for two hours before being subjected to scRNA-seq. An additional cohort of mice was housed in the home cage. The day before the experiment, they were transferred to a new disposable cage with regular bedding/food overnight in a satellite animal facility. On the experimental day they were transferred to a new home cage for two hours before being subjected to scRNA-seq. These three conditions were compared to mice transferred to Environment A cages for 2 hours as part of the time course described above, thus generating all four combinations of environmental switches between mice housed in Environment A and home cages.</div></section><section id="sec-4-4-5"><h4>Chronic acetophenone exposure</h4><div id="p0280" role="paragraph">To compare the effects of chronic odor exposure with the acute activation observed via Act-seq, scRNA-seq was performed after exposure to DPG alone or 0.1, 1, 10% of acetopheone (diluted by DPG) for 5&nbsp;days. Adult mice were individually housed in regular home cages in a satellite animal facility overnight, then two cotton balls, each of which was soaked with 300&nbsp;μL of DPG or diluted acetophenone solution, were added into each cage. The odorized cotton balls were replaced every 12 hours and cages were replaced every 2&nbsp;days.</div></section><section id="sec-4-4-6"><h4>Optogenetic stimulation</h4><div id="p0285" role="paragraph">To characterize quantitative changes in gene expression after activation of OSNs, OMP-ChR2(H134R)-Venus mice were subject to optogenetic activation and analyzed via scRNA-seq. Mice (N&nbsp;= 14) were anesthetized with 2% isoflurane and injected with bupivacaine (1.25&nbsp;mg/kg) under the scalp. An incision was then made to expose the dorsal skull and a scalpel (Aspen Surgical 372615 Bard-Parker) was used to thin the bone overlying the right olfactory bulb, after which cyanoacrylate glue (Loctite Glass Glue) was applied over the thinned bone. A blue LED (470&nbsp;nm, Lumileds LXZ1-PB01, Digikey) was soldered to a two-pin connector (Millmax, Digikey ED8450-ND) and manually placed over the bulb. A titanium head bar was then placed over the caudal end of the skull, and the whole dorsal surface was covered with dental cement (Metabond, Parkell). After the dental cement dried, a layer of black nail polish was applied to minimize the leak of blue light during stimulation. Postoperative care included a subcutaneous injection of buprenorphine SR (1&nbsp;mg/kg, given 1 hour prior to surgery start) and carprofen (5&nbsp;mg/kg) administered through drinking water. Before experiments, mice were tethered to a cable connecting the LED to a microcontroller (Teensy 3.2, Adafruit) that provided a custom stimulation pattern. The LED emitted 100mW at 470nm, measured with a light meter (Thorlabs PM100D).</div><div id="p0290" role="paragraph">For chronic activation, OSNs were activated by 5 pulses (50&nbsp;msec on, 50&nbsp;msec off cycle) with a 10&nbsp;s inter-pulse interval for 12 hours before being subjected to scRNA-seq. For acute activation, OSNs were activated by the same cycle as above with a 5, 10, or 20&nbsp;s inter-pulse interval for 2 hours (denoted as High, Medium, and Low, respectively) and then subjected to scRNA-seq. For both chronic and acute activation, control mice underwent the same surgery and tethering, but were implanted with a dummy connector and did not receive LED stimulation. Home cage control samples for optogenetic experiments were mice that underwent the same surgery but were subjected to scRNA-seq directly from the home cage without any optogenetic stimulation. Because the LED was placed over the right olfactory bulb, only the right nostril was dissociated and used for scRNA-seq for all optogenetic stimulation and control experiments.</div></section><section id="sec-4-4-7"><h4>Preparation of single-cell suspensions for scRNA-seq</h4><div id="p0295" role="paragraph">The main olfactory epithelium was dissected in Earle’s Balanced Salt Solution (EBSS, pre-treated with carbogen for at least 5&nbsp;minutes before each use, Worthington), then transferred to a round-bottomed glass dish containing 750&nbsp;μL of papain solution (one vial of Papain (Worthington) dissolved in 5&nbsp;mL of EBSS and then equilibrated 10&nbsp;minutes at 37°C) and 50&nbsp;μL of DNase solution (one vial of DNase-I (Worthington) dissolved in 500&nbsp;μL of EBSS). Bone was removed from the epithelium under a dissecting microscope and the resulting epithelial tissue was placed in a 5&nbsp;mL tube (Becton Dickinson) with an additional 1.75&nbsp;mL of Papain solution and 200&nbsp;μL of DNase-I solution and rocked gently for ∼60&nbsp;minutes at 37°C. The tissue was then gently triturated with a 5&nbsp;mL pipette 10-15 times, passed through a 40&nbsp;μm cell strainer (Becton Dickinson), and washed with 1&nbsp;mL of Hibernate-A medium containing 10% Fetal Bovine Serum (FBS, GIBCO). Filtered cells were transferred to a new 5&nbsp;mL tube and centrifuged 5&nbsp;minutes at 300 x g. The supernatant was decanted, the cells were washed once with 4&nbsp;mL Hibernate-A containing 10% FBS and resuspended in 1&nbsp;mL Hibernate-A containing 0.2% FBS. Importantly, each of the solutions above contained transcriptional inhibitors (5&nbsp;μg/mL of Actinomycin D, 10&nbsp;μg/mL of Anisomycin and 10&nbsp;μM of Triptolide, all obtained from Sigma) for all experiments, especially the Act-Seq experiments, except where noted (<a id="crosref1345" href="#mmc1">Table S1</a>).</div></section><section id="sec-4-4-8"><h4>Fluorescence-activated cell sorting</h4><div id="p0300" role="paragraph">Dissociated cells were stained with propidium iodide (final concentration 1.65&nbsp;μg/mL) and subjected to FACS to remove dead cells and obvious doublets. For fluorescent reporter-expressing mouse lines (OMP-IRES-GFP, P2<sub>(P2 locus)</sub>, M72<sub>(S50 locus)</sub>, S50<sub>(M72 locus)</sub>, OMP-ChR2(H134R)-Venus), fluorescence positive and negative cells were sorted separately using a FITC filter. Cells were sorted into Hibernate-A containing 5% FBS (and 0.1x concentration of transcriptional inhibitors), centrifuged for 5&nbsp;minutes at 300 g, and resuspended with PBS. For samples from OMP-IRES-GFP and OMP-ChR2(H134R)-Venus mice, fluorescence positive cells were used for scRNA-seq experiments. For samples from OR lines (P2<sub>(P2 locus)</sub>, M72<sub>(S50 locus)</sub> and S50<sub>(M72 locus)</sub>), fluorescence positive and negative cells were combined for scRNA-seq experiments.</div></section><section id="sec-4-4-9"><h4>Single-cell library preparation</h4><div id="p0305" role="paragraph">Single cell RNA-seq library were prepared from the single cell suspensions via the chromium single cell gene expression system (chromium single cell 3′ reagents and GEM v2, v3 or v3.1 dual index, 10x genomics), using the default protocols provided by 10x genomics. Each replicate, other than GFP-positive ones, was loaded at a concentration predicted to yield 10,000 cells, for which the expected multiplet rate is ∼8.0%.</div></section><section id="sec-4-4-10"><h4>Sequencing</h4><div id="p0310" role="paragraph">Sequencing library fragments were examined using the Agilent High Sensitivity DNA kit (Agilent) and quantified via qPCR by the KAPA library quantification kit (Roche). NextSeq and NovaSeq platforms were used for sequencing libraries. 75 cycle High output kit was used for NextSeq (Read1&nbsp;= 26 cycle, Index (i7)&nbsp;= 8 cycle, Read2&nbsp;= 58 cycle). Full flowcells of 100 cycle SP/S1/S2 kits, full flowcells or single lanes of 200 cycle S4 kit were used for NovaSeq (minimum read lengths: Read1&nbsp;= 28 cycle, Index (i7)&nbsp;= 8 cycle, Read2&nbsp;= 89 cycle for v2 and v3 kits and Read1&nbsp;= 28 cycle, Index (i7)&nbsp;= 10 cycle, Index (i5)&nbsp;= 10 cycle, Read2&nbsp;= 77 cycle for v3.1 dual index kits). Demultiplexed fastq files were generated by mkfastq function in Cell Ranger software.</div></section><section id="sec-4-4-11"><h4>Generating gene expression matrix (raw UMI count) from sequencing data</h4><div id="p0315" role="paragraph">Demultiplexed fastq files were aligned to the mouse reference genome mm10 (Ensembl 93) and converted into gene expression matrices using the 10x Genomics Cell Ranger software (version 2.2.0 (v2 and v3 samples) or 4.0.0 (v3.1 samples)) with two key modifications. First, despite the fact that the 10x genomic platforms have relatively high rates of intronic priming (<span class="dropBlock reference-citations"><a id="body-ref-sref52" href="#" role="doc-biblioref" data-xml-rid="bib52" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref52" aria-controls="sref52">La Manno et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref52"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">54.</div><div>La Manno, G. ∙ Soldatov, R. ∙ Zeisel, A. ...</div><div><strong>RNA velocity of single cells</strong></div><div><em>Nature.</em> 2018; <strong>560</strong>:494-498</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-018-0414-6" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-85052109231" target="_blank" aria-label="Open Scopus in new tab">Scopus (690)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30089906/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-018-0414-6&amp;pmid=30089906" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), Cell Ranger by default considers multi-mapped reads that map to a single exonic locus as well as to non-exonic (e.g., intronic) locations as being uniquely mapped to that gene (and modifies the MAPQ scores accordingly), even when the read may have better alignment to the non-exonic loci. These multimapped reads (typically 5%–10% of all reads) were filtered out by removing any read in the bam file with a MM:i:1 tag. Second, the output BAM files from Cell Ranger also contained many reads that were uniquely mapped to separate genes despite sharing the same cell barcode and unique molecular identifier (UMI), which is biologically implausible given that the cell barcode and UMI should uniquely identify each individual transcript. This could be either the result of UMI collision or misalignment. In support of the latter possibility, different reads of individual UMIs were often each uniquely aligned to multiple related genes, such as the large family of olfactory receptors (ORs). To avoid double-counting UMIs and inflating the numbers of genes, ORs, or UMIs detected in each cell, all ambiguous UMIs that mapped to multiple genes were removed from the BAM file with custom scripts, using samtools and pysam. The gene expression matrix was then recomputed by counting the number of distinct UMIs for each cell barcode for each gene.</div></section><section id="sec-4-4-12"><h4>Extracting mature olfactory sensory neurons from all cells</h4><div id="p0320" role="paragraph">Mature OSNs were identified using an iterative subclustering procedure via the Scanpy python package (<span class="dropBlock reference-citations"><a id="body-ref-sref110-2" href="#" role="doc-biblioref" data-xml-rid="bib110" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref110-2" aria-controls="sref110-2">Wolf et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref110-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">113.</div><div>Wolf, F.A. ∙ Angerer, P. ∙ Theis, F.J.</div><div><strong>SCANPY: Large-scale single-cell gene expression data analysis</strong></div><div><em>Genome Biol.</em> 2018; <strong>19</strong>:15</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13059-017-1382-0" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_113_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-85041394976" target="_blank" aria-label="Open Scopus in new tab">Scopus (758)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29409532/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13059-017-1382-0&amp;pmid=29409532" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). In each cell, the UMIs were total-count normalized and scaled by 10,000 (TPT normalized). Variable genes that were overdispersed relative to their mean were identified using the SPRING gene filtering function “filter_genes” with parameters (90, 3, 10; see (<span class="dropBlock reference-citations"><a id="body-ref-sref104" href="#" role="doc-biblioref" data-xml-rid="bib104" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref104" aria-controls="sref104">Weinreb et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref104"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">107.</div><div>Weinreb, C. ∙ Wolock, S. ∙ Klein, A.M.</div><div><strong>SPRING: A kinetic interface for visualizing high dimensional single-cell expression data</strong></div><div><em>Bioinformatics.</em> 2018; <strong>34</strong>:1246-1248</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btx792" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_107_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-85041411084" target="_blank" aria-label="Open Scopus in new tab">Scopus (112)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29228172/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtx792&amp;pmid=29228172" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)). Ribosomal, mitrochondrial, and OR genes were excluded from the variable genes. To identify mature OSNs, the gene expression of the set of variable genes was log-normalized, and for each gene, the residuals from linear regression models using the total number of UMIs and percent of UMIs for mitochondrial genes as predictors were then scaled via z-scoring and reduced to a smaller number of dimensions via principal component analysis (PCA). Cells were clustered using the top 35 PCs via the Leiden algorithm (resolution&nbsp;= 1.2). Clusters containing mature OSNs were identified based on their expression of known mature OSN marker genes (e.g., <i>Omp</i>, <i>Stoml3</i>, <i>Cnga2</i>, <i>Adcy3</i>), the presence of cells expressing OR genes, as well as the absence of immature OSN marker genes or genes found in non-neuronal cells (e.g., <i>Sox11</i>, <i>Gap43</i>, <i>Cbr2</i>, <i>C1qb</i>, <i>Aqp3</i>). A small cluster containing the non-canonical <i>Gucy2d</i> (GCD) and <i>Gucy1b2</i> expressing neurons was routinely identified but not used for any analyses in this paper. Clusters containing dying cells with low numbers of total counts, low numbers of genes, and high percentages of mitochondrial genes were removed. This clustering procedure typically revealed a cluster of OSNs that had higher than average total counts (often close to double the mean of other OSN clusters) and the majority of cells in such clusters often expressed multiple ORs; these cells are likely OSN-OSN doublets and were not considered further. Similarly, clusters containing mixtures of OSN and non-neuronal markers were also likely doublets between OSNs and other cell types and were not considered further. After excluding immature OSNs and non-neuronal cells (~30% of cells) as well as doublets (&lt;5% of cells), the above procedure (starting from variable gene identification but using only 20 PCs) was repeated multiple times for the mature OSN clusters that passed these initial filtering steps to remove any additional immature or unhealthy OSNs (with low counts or &gt; 10% of total counts in mitochondrial genes) until stable results were obtained, yielding the mature OSNs (60–65% of all cells) used for downstream analyses. Importantly, although immature OSNs (e.g., <i>Gap43+</i> and weakly <i>Omp+</i>) routinely express OR genes and other mature OSN markers, these OSNs were excluded by the conservative subclustering procedure described above so that any comparisons between cells expressing the same OR would not be confounded by any differences in OSN maturity.</div></section><section id="sec-4-4-13"><h4>OR gene annotation (including functional or pseudogenes, class I or II, dorsal or ventral, and CD36-positive or -negative), OR protein similarity, and OSN positional subtypes</h4><div id="p0325" role="paragraph">The Ensembl database (Release 93) was used for OR gene annotation. Functional OR genes were identified as genes that are associated with GO term “olfactory receptor activity” (GO:0004984); the TAAR family of receptors was also included (GO:0001594, GO:1990081). To identify the OR class for each mouse OR, phylogenetic analysis was performed using all mouse and zebrafish OR proteins in MEGA7 with default parameters. As previously described, OR proteins cluster into two main phylogenetic clusters (<span class="dropBlock reference-citations"><a id="body-ref-sref116" href="#" role="doc-biblioref" data-xml-rid="bib116" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref116" aria-controls="sref116">Zhang and Firestein, 2002</a><div class="dropBlock__holder" data-db-target-of="sref116"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">119.</div><div>Zhang, X. ∙ Firestein, S.</div><div><strong>The olfactory receptor gene superfamily of the mouse</strong></div><div><em>Nat. Neurosci.</em> 2002; <strong>5</strong>:124-133</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nn800" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_119_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-0036163689" target="_blank" aria-label="Open Scopus in new tab">Scopus (642)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11802173/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnn800&amp;pmid=11802173" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The ORs that clustered together with zebrafish ORs are the class I ORs and the remaining ORs are class II ORs. To compare OR protein similarity, the pairwise phylogenetic distance matrix between OR protein sequences was calculated by taking the cophenetic distance of the phylogenetic tree of all ORs. For each pair of ORs, the cophenetic distance is equal to the height of a dendrogram in which the two branches of the phylogenetic tree containing both ORs first merge into a single cluster. Unsupervised graph-based clustering of cells identified a cluster of OSNs expressing <i>Cd36</i>, as previously described (<span class="dropBlock reference-citations"><a id="body-ref-sref73" href="#" role="doc-biblioref" data-xml-rid="bib73" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref73" aria-controls="sref73">Oberland et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref73"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">75.</div><div>Oberland, S. ∙ Ackels, T. ∙ Gaab, S. ...</div><div><strong>CD36 is involved in oleic acid detection by the murine olfactory system</strong></div><div><em>Front. Cell. Neurosci.</em> 2015; <strong>9</strong>:366</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.3389/fncel.2015.00366" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_75_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-84942904692" target="_blank" aria-label="Open Scopus in new tab">Scopus (24)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26441537/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.3389%2Ffncel.2015.00366&amp;pmid=26441537" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref113" href="#" role="doc-biblioref" data-xml-rid="bib113" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref113" aria-controls="sref113">Xavier et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref113"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">116.</div><div>Xavier, A.M. ∙ Ludwig, R.G. ∙ Nagai, M.H. ...</div><div><strong>CD36 is expressed in a defined subpopulation of neurons in the olfactory epithelium</strong></div><div><em>Sci. Rep.</em> 2016; <strong>6</strong>:25507</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/srep25507" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_116_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-84966277054" target="_blank" aria-label="Open Scopus in new tab">Scopus (19)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27145700/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fsrep25507&amp;pmid=27145700" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). ORs were identified as <i>Cd36</i>-positive if the majority of OSNs expressing that OR were found within the <i>Cd36</i>-positive clusters. OSNs were categorized as dorsal or ventral based on their expression of known dorsal (<i>Nqo1</i>, <i>Acsm4</i>) and ventral (<i>Ncam2</i>, <i>Nfix</i>) marker genes. ORs expressed in dorsal or ventral OSNs were referred to as dorsal or ventral ORs. With the exception of three known ventral class I ORs, all other class I ORs are dorsal, whereas OSNs expressing individual class II OR expressed either dorsal or ventral marker genes. The OR genes are located in gene clusters across most chromosomes in the genome; OR genes whose transcription start sites (TSS) were within 3 megabases of any other OR gene were considered part of the same gene cluster (<span class="dropBlock reference-citations"><a id="body-ref-sref66-2" href="#" role="doc-biblioref" data-xml-rid="bib66" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref66-2" aria-controls="sref66-2">Monahan et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref66-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">68.</div><div>Monahan, K. ∙ Schieren, I. ∙ Cheung, J. ...</div><div><strong>Cooperative interactions enable singular olfactory receptor expression in mouse olfactory neurons</strong></div><div><em>eLife.</em> 2017; <strong>6</strong>:e28620</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.7554/eLife.28620" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-85032933643" target="_blank" aria-label="Open Scopus in new tab">Scopus (42)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28933695/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.7554%2FeLife.28620&amp;pmid=28933695" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div></section><section id="sec-4-4-14"><h4>Identification of the OR expressed in each OSN</h4><div id="p0330" role="paragraph">Of the 1,172 identified functional ORs, an OR was considered expressed in any given OSN if at least 3&nbsp;UMIs were detected (except for the experiments using the 10x v2 chemistry, where a threshold of 2&nbsp;UMIs was used). Mature OSNs expressing single ORs were identified based on the OR that they expressed, and all cells expressing the same OR were considered as the same OSN subtype for downstream analyses. The vast majority of mature OSNs expressed a single OR (<a id="crosref1385" href="#figs1">Figure&nbsp;S1</a>D). OSNs expressing multiple ORs at high levels typically had higher numbers of total counts and were likely cell doublets that remained even after the previous filtering steps. OSN expressing multiple ORs with higher expression for a single OR gene also sometimes expressed other OR genes at low levels, as seen during development (<span class="dropBlock reference-citations"><a id="body-ref-sref37-2" href="#" role="doc-biblioref" data-xml-rid="bib37" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref37-2" aria-controls="sref37-2">Hanchate et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref37-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">37.</div><div>Hanchate, N.K. ∙ Kondoh, K. ∙ Lu, Z. ...</div><div><strong>Single-cell transcriptomics reveals receptor transformations during olfactory neurogenesis</strong></div><div><em>Science.</em> 2015; <strong>350</strong>:1251-1255</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aad2456" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-84949035291" target="_blank" aria-label="Open Scopus in new tab">Scopus (116)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26541607/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aad2456&amp;pmid=26541607" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), or due to contamination from ambient RNA or errors in gene alignment. Regardless, any OSN expressing either zero or multiple OR genes was not considered further for any downstream analyses. OR frequency was defined empirically based on the fraction of all mature OSNs expressing a given OR. The expression level for each OR was determined based on the normalized expression (see below) of that OR gene within the OSNs singly-expressing that OR. In the “OR-swap” mice, in which the coding sequence for each OR was knocked into the locus of the other, cells expressing swapped ORs were identified based on the gene counts for the locus rather than the expressed OR (e.g., in M72<sub>(S50 locus)</sub> mice, OR transcripts that originated from the S50 locus indicate the expression of M72 swap cells while OSNs expressing M72 from the endogenous locus were not considered and vice versa for S50<sub>(M72 locus)</sub> mice) due to fact that only coding sequences were swapped whereas the 3′ UTR for the original OR remain and are detected given the 3′ bias of the 10x data that arises from the use of oligo(dT) primers. The median number of UMIs/genes detected per mature OSNs and number of mature OSNs used for each replicate are summarized in <a id="crosref1395" href="#mmc1">Table S1</a>.</div></section><section id="sec-4-4-15"><h4>Accounting for OR frequency</h4><div id="p0335" role="paragraph">ORs were expressed at a wide range of frequencies, spanning multiple orders of magnitude (<a id="crosref1400" href="#figs1">Figures S1</a>A and S1B). For instance, given that the 200 OR genes with the highest frequencies are expressed in ∼50% of all OSNs, if standard algorithms like PCA are applied on all OSNs at once, they would weight ORs expressed in 100 cells 100x more than an OR expressed in a single cell, and the resulting top PCs would capture the axes of transcriptional variation that distinguished highly-expressed ORs rather than all&nbsp;OSN subtypes. Similarly, classifiers trained on all cells would more easily distinguish highly-expressed ORs (from their higher influence on the PCs, increased presence in training data, and higher likelihood of accurate predictions by chance), and one could achieve high levels of overall prediction accuracy even with models that were unable to distinguish among the majority of ORs. Therefore, to account for OR frequency, most analyses were performed using equal numbers of OSNs per OR by subsampling from the population of OSNs that expressed the set of ORs each detected in at least a certain number of cells (e.g., at least 4-10, as specified, depending on the analysis). This subsampling procedure was performed 1,000 times and results were summarized for each OSN subtype (as defined by the expressed OR) across the 1,000 restarts. Analyses at the OSN subtype level were performed by first averaging across all OSNs expressing the same OR within a given condition.</div></section><section id="sec-4-4-16"><h4>Normalization and scaling</h4><div id="p0340" role="paragraph">The filtered mature OSN datasets containing only cells expressing single ORs were renormalized, starting again from the raw UMI counts for each gene. Normalization was performed by dividing the counts for each gene in each cell by the total counts across all genes (excluding mitochondrial and ribosomal genes, as well as highly-expressed lncRNAs and lincRNAs like <i>Malat1</i> and <i>Gm42418,</i> and sex-specific transcripts like <i>Xist</i>) in that cell and multiplying by 10,000 to yield tags per ten thousand (TPT) normalized data. Normalized expression in the figures and throughout refers to TPT-normalized data, where a value of 1 indicates on average 1&nbsp;UMI for that gene per 10,000&nbsp;UMIs. For analyses that required scaled data (e.g., PCA and classification), the logarithm of the TPT-normalized data was used (log(TPT&nbsp;+ 1)). Scaling via z-scoring was performed for each gene, using means and standard deviations identified by taking the mean of the values from 100 restarts containing equal numbers of cells per OR. The home-cage data consisted of six replicates distributed across 2 batches; separate scaling was used for each batch, but similar results were obtained by scaling both batches together. For the “OR-swap” experiments, the means and standard deviations were fit using only cells from wild-type mice collected with the same 10x v2 chemistry platform, and these were applied to the “OR-swap” and P2<sub>(P2 locus)</sub> OSNs. In the chronic occlusion data, the means and standard deviations were recomputed using data from both nostrils. In the environment switch experiments, the gene scaling was recomputed using data from all environments.</div></section><section id="sec-4-4-17"><h4>Dataset integration</h4><div id="p0345" role="paragraph">Importantly, besides scaling and normalizing the gene expression data, no methods for batch correction or dataset integration were performed for any dataset. The mature OSNs were readily identified and separately isolated from each dataset (obviating the need for any additional alignment of cell types across datasets), and batch effects were rarely observed when comparing the normalized expression between datasets (apart from changes in the levels of mitochondrial and ribosomal genes, which were excluded from normalization procedures and downstream analyses). This analytical choice was deliberate, as data integration procedures are affected by changes in OSN composition (due to differences in the set and frequency of ORs detected across datasets and the wide range of OR frequencies), can modify the transcriptomes by averaging and leaking data across cells (which may express different ORs), and could make OSNs transcriptomes more or less similar to each other in a manner that would hinder and bias downstream comparisons between cells expressing the same OR across experimental conditions. For instance, in the Act-Seq experiments, our expectation was that a small fraction of OSNs would exhibit difference in their transcriptomes upon being activated by odorants; however, dataset integration procedures typically attempt to normalize such differences, and thus tend to make control OSNs appear more activated and odor-exposed OSNs appear less activated. Some datasets were collected with different 10x kits (v2, v3, and v3.1; see <a id="crosref1405" href="#mmc1">Table S1</a>); as these kits differ in their sensitivities, each sample was compared with appropriate control samples collected with the same platform.</div></section><section id="sec-4-4-18"><h4>Gene selection</h4><div id="p0350" role="paragraph">A consensus set of genes whose expression varied across mature OSNs were identified and used as the basis of most downstream analyses where transcriptomes were compared across OSNs and conditions. After using the OR genes to identify the OR expressed in each cell, the OR genes were not considered further, and were subsequently removed from all downstream analyses. Mitochondrial and ribosomal genes, and the lncRNAs mentioned above were also excluded. Using the home-cage data, the population of OSNs that expressed 831 ORs in at least 10 cells was used. To identify genes whose expression varied across cells expressing different ORs, 10 cells were selected for each OR and the F-score between the scaled gene expression and OR identity was computed for each gene, using the set of ∼15,000 genes that were expressed in at least 0.2% of OSNs. This procedure was performed 1,000 times and genes that were consistently strongly significant (Benjamini-Hochberg FDR-corrected p value ≤ 1&nbsp;× 10<sup>−7</sup> for at least 90% of restarts) were considered highly-variable genes (HVGs). This procedure identified a set of 1,350 non-OR genes that were used for downstream analyses. Varying thresholds, such as the number of restarts to consider a gene as a HVG, yielded different numbers of HVGs but gave consistent results with respect to other analyses (<a id="crosref1410" href="#figs1">Figure&nbsp;S1</a>I).</div></section><section id="sec-4-4-19"><h4>Principal-component analysis</h4><div id="p0355" role="paragraph">Principal component analysis (PCA) was performed using equal numbers of cells for each OR to avoid any OR having undue weights on any PC based on its frequency of expression. All versions of PCA shown used the scaled expression of the set of 1,350 HVGs, and these HVGs were reduced to the top 20 PCs; however, stable results were obtained with different numbers of HVGs and PCs (<a id="crosref1415" href="#figs1">Figure&nbsp;S1</a>I). After fitting PCA in this manner using equal numbers of cells for each OR the fitted gene loadings (V<sup>T</sup>) were applied to the scaled HVG expression for all cells to obtain PCA scores for the remaining cells. In analyses in which this subsampling procedure was repeated across restarts, PCA was fit on the subset of cells sampled for each restart. For UMAP visualization, the results from a single restart are shown (although qualitatively similar results were obtained across restarts). In the “OR-swap” experiments, PCA was fit&nbsp;using only OSNs from wild-type mice (with equal numbers of cells per OR) and then applied to the cells from the “OR-swap” and P2<sub>(P2 locus)</sub> mouse lines. In the chronic occlusion experiments, PCA was refit using equal numbers of cells per each OR for each nostril, and in the environment switch experiments, PCA was fit using equal cells for each OR from each of the DPG-exposed control mice from each of the three environments.</div></section><section id="sec-4-4-20"><h4>Visualization via UMAP</h4><div id="p0360" role="paragraph">UMAP was used solely for visualization purposes (<span class="dropBlock reference-citations"><a id="body-ref-sref62" href="#" role="doc-biblioref" data-xml-rid="bib62" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref62" aria-controls="sref62">McInnes et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref62"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">64.</div>McInnes, L., Healy, J., and Melville, J. (2018). UMAP: Uniform manifold approximation and projection for dimension reduction. arXiv, <a id="intref0105" href="https://arxiv.org/pdf/1802.03426v3.pdf" target="_blank">https://arxiv.org/pdf/1802.03426v3.pdf</a>.</div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=McInnes%2C+L.%2C+Healy%2C+J.%2C+and+Melville%2C+J.+%282018%29.+UMAP%3A+Uniform+manifold+approximation+and+projection+for+dimension+reduction.+arXiv%2C+https%3A%2F%2Farxiv.org%2Fpdf%2F1802.03426v3.pdf." target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). To generate UMAP visualizations, the gene expression of each cell was further reduced from 20 PCs to 2 dimensions using the Uniform Manifold Approximation and Projection (UMAP) technique with parameters n_neighbors&nbsp;= 25, min_dist&nbsp;= 0.6, and metric&nbsp;= “euclidean.” As for PCA, UMAP embeddings were fit using balanced numbers of cells for each OR (and only for wild-type OSNs in the “OR-swap” experiments), and these embeddings were then used to transform to the PCA scores for all cells of a given dataset into the same UMAP space. UMAP was fit using 1,000 epochs and a learning rate of 0.25 to improve the stability of this transformation step. The UMAP embeddings were refit separately using the PC scores from each dataset, yielding embeddings that differed qualitatively in appearance yet retained similar features (dorsal, ventral, and <i>Cd36</i>-positive OSNs were separated, the main axis was activity-dependent, cells expressing the same OR were locally clustered). UMAP and PCA were also refit using only cells from the occluded nostrils to determine if OSN subtypes cluster based on intrinsic activity in the absence of odor-evoked activity.</div></section><section id="sec-4-4-21"><h4>Transcriptome distance</h4><div id="p0365" role="paragraph">To measure the similarity of cells based on their gene expression, the transcriptome distance was defined as the cosine distance (1 – cosine similarity) between the PC scores of pairs of OSNs. The cosine distance is bounded between 0 (an angle of 0° indicating similar transcriptomes), 1 (an angle of 90° indicating orthogonality) and 2 (an angle of 180° indicating opposing transcriptomes).</div><div id="p0370" role="paragraph">Equal numbers of cells (10) were selected for each OR (of the 831 ORs detected in at least 10 cells in the home-cage dataset), PCA was refit based on the scaled HVG expression of these 8,310 cells and the pairwise transcriptome distance matrix was then evaluated. Within-OR distances we calculated, for each OR, as the median pairwise distance between all pairs of cells sharing that OR. For between-OR distances, the median pairwise distance between all pairs of ORs was calculated. Then, since each OR is more similar and dissimilar to specific subsets of other ORs, for each OR, the distribution of between-OR distances were summarized across all pairs of ORs sharing that OR. Starting from subsampling OSNs, the within- and between-OR distances were recomputed across 1,000 restarts, and the distribution of within-OR distances across ORs was compared to the distribution of pairwise distances between each OR and all other ORs. On each restart, the OR labels were also shuffled across cells, and within-OR distances were recomputed (and were close to 1 indicating that the transcriptomes of random cells were dissimilar). Similar results were also obtained using the Euclidean and Correlation distance metrics. To evaluate whether these distributions of within- and between-OR transcriptome distances remained separable (even though distances increase with increasing dimensionality) across various choices of gene sets and numbers of PCs, the Jensen–Shannon divergence was computed by comparing the discretized versions of these two distributions on each restart (with 111 evenly spaced bins between 0 and 2) and comparing the entropy of the average distribution to that of the average of the entropies of each individual distribution.</div></section><section id="sec-4-4-22"><h4>k-nearest neighbor analysis</h4><div id="p0375" role="paragraph">To determine whether cells sharing the same OR are clustered locally within gene expression space, a <i>k</i>-nearest neighbor (KNN) approach was used. For each of 1,000 restarts, using the pairwise distance matrix between cells described above, for each cell for each OR, the fraction of all the other cells that share the same OR within <i>k</i> neighbors was computed empirically for progressively larger groups of <i>k</i> neighbors. The fraction of same-OR cells found within <i>k</i> neighbors was averaged for each OR across restarts and was then summarized across the entire set of 831 ORs. The local clustering of cells sharing the same-OR was not apparent when shuffling the OR labels across the cells from each restart.</div></section><section id="sec-4-4-23"><h4>Classification of ORs expressed in home cage data</h4><section id="sec-4-4-23-1"><h5>Classification pipeline</h5><div id="p0380" role="paragraph">In all instances, classification was performed in a cross-validated manner using linear classifiers fit on equal numbers of cells for each OR. Equal numbers of cells were randomly selected for each OR (10 cells for each of the 831 ORs expressed in at least 10 OSNs in the home-cage data). For both pairwise and OR identity classification, the number of cross-validation folds was chosen to be equal to the number of subsampled cells (10) for each OR. For each fold, one cell expressing each OR was held out and the remained cells were used to train linear support vector machine (SVM) models (SVC with kernel&nbsp;= “linear,” decision_function_shape&nbsp;= “one-vs-one,” and regularization parameter <i>C</i>&nbsp;= 0.03). The hyperparameters for the classification pipeline were identified via a random search, but similar results were obtained across a range of hyperparameters. The input to the classifiers was the scaled gene expression (excluding any OR, ribosomal, or mitochondrial genes), using all genes (∼13,000) that were expressed in at least 0.5% of all cells. During each cross-validation fold, feature selection was performed by keeping the top 10% of genes with the highest F-score with the OR labels of the training fold cells. Importantly, although this is the same procedure used for HVG selection described above and the 10% threshold was designed to yield a similar number of genes, because feature selection was performed in a cross-validated manner, the resulting set of 1,300 genes identified for each fold may differ slightly from the set of HVGs used for other analyses). Next, the dimensionality of these 1,300 genes was further reduced to 20 dimensions with linear transformations via PCA and Linear Discriminant Analysis (LDA), also using only the cells in the training fold. Importantly, in addition to using equal numbers of cells per OR (to make predictions at similar chance levels for each OR), the feature selection, PCA and LDA transformations, and SVM decision boundaries were all fit using only the data in the training folds and then applied to the held-out cells in the test fold to generate predictions of the expressed OR in each held-out cell. This classification pipeline therefore avoids data leakage (which could aid in the prediction of OR identities and potentially bias results) between training and test folds.</div></section><section id="sec-4-4-23-2"><h5>Classification between OR pairs</h5><div id="p0385" role="paragraph">Pairwise classification was performed using the same classification pipeline. For each training fold, as described above, the gene sets, PCA and LDA transformations were fit using all cells in the training fold, and then applied to transform all held-out cells into the same 20D subspace. Then, using the training data, separate SVM models were fit for each pair of ORs (<sub>831</sub>C<sub>2</sub>&nbsp;= 344,865 pairs) and these decisions boundaries were applied to the held-out cells of that OR pair to predict which OR within that pair each held-out cell expressed. This cross-validation procedure was repeated for each cross-validation fold, and the mean classification accuracy was computed for each OR pair. The entire pairwise classification procedure (starting from subsampling OSNs) was repeated across 1,000 restarts (yielding in total 344,865&nbsp;× 10&nbsp;× 1,000&nbsp;= 3.45 billion models). The mean accuracy for each OR pair was summarized across the 1,000 restarts, and the distributions of classifications accuracies were shown for the 344,865 OR pairs. The median prediction accuracy was 100% (compared to a chance accuracy of 50%), indicating that in the majority of OR pairs, the OR identity for every OSN of that OR pair was correctly predicted in each of the 1,000 restarts. The same classification procedure was performed separately for both nostrils for the 436 ORs detected in at least 10 cells in each nostril in the chronic occlusion data.</div></section><section id="sec-4-4-23-3"><h5>Classification of OR identity across all ORs</h5><div id="p0390" role="paragraph">The same cross-validation and classification procedure was used to predict which OR (out of 831) was expressed in each held-out cell. “One-vs-one” SVM models were first fit on the PCA and LDA-transformed gene expression of all OR pairs in the training set and then these transformations and models were applied to each cell in the test fold; this procedure was repeated for each fold. The “decision function” from this procedure thus returned for each held-out cell the predictions from the model fitted on each pair of ORs in the respective training fold, yielding a matrix of 8,310 cells x 344,865 models that indicates for each of the 344,865 “one-vs-one” binary pairwise classifiers which OR within that pair was more likely to be expressed. Predictions were generated using the standard voting procedure; for each cell, for each OR, the number of times that the subset of 831 pairwise models containing that OR predicted that specific OR (rather than the other OR in each pair) as more likely to be expressed were summed. The OR with the highest number of “+1” votes (out of a maximum possible 831) was predicted as the OR that was most likely to be expressed in a given cell. Since this procedure calculated the total number of votes for each OR for each cell, it also generated the likelihood (based on the ranks of the number of “+1” votes for each OR) any given OR was expressed in any given cell. Cells whose OR identity was correctly predicted were those in which the actual OR expressed in that cell was the one that received the most votes (or on rare occasions tied for the most votes).</div><div id="p0395" role="paragraph">Given that correctly predicting the exact OR (out of 831) of each held-out cell given only 9 training cells expressing the same OR and 7470 (830&nbsp;× 9) cells expressing other ORs is a difficult task (with chance accuracy 1/831), the frequency in which the OR expressed in each held-out cell was within the top-N% of predicted ORs was also calculated using the ranks of the number of votes for each OR described above. In this case, predictions were considered as accurate at increasing percent thresholds (e.g., 0.25%&nbsp;= the actual OR was within the top 2 ORs with the most votes and 1%&nbsp;= within the top 8 ORs out of 831). For both the exact and top-N% classification procedures, the classification accuracy for each OR was summarized across the 10 cells expressing that OR in each restart and the accuracy for each OR was then averaged across 1,000 restarts yielding the average classification accuracy for each of the 831 ORs, which are shown in the figures. Across the 831 ORs, the median classification accuracy was ∼50%. Given the balanced number of cells per OR, classification accuracy did not depend on OR frequency (<a id="crosref1425" href="#figs1">Figure&nbsp;S1</a>J) and was at chance levels for classifiers fit on the same data in each restart but with shuffled OR labels (1/831&nbsp;= 0.1% for the exact predictions and 1% for the top-1% predictions). This same OR identity classification procedure was performed separately for both nostrils for the 436 ORs detected in at least 10 cells in each nostril in the chronic occlusion data.</div></section><section id="sec-4-4-23-4"><h5>Classification with different numbers of genes</h5><div id="p0400" role="paragraph">OR identity classification was performed as described above, but the % threshold for the F-score feature selection procedure of the classification pipeline was modified to assess the classification performance with varying numbers of genes. First, the high threshold was progressively increased from 0.5 to 12.5% of the ∼13,000 expressed genes (compared to the default of 10%). Next, starting from the default threshold of 10%, the most informative genes were progressively excluded by increasing the low threshold in 0.5% intervals such that the top 0.5%, then the top 1%, were excluded until only genes in the top 9.5%–10% remained. 10 restarts were performed for each gene threshold, and the mean accuracy across ORs was calculated.</div></section><section id="sec-4-4-23-5"><h5>Classification with different gene sets</h5><div id="p0405" role="paragraph">OR identity classification was performed as above for the different gene sets, using genes from each set that were expressed in at least 0.5% of cells. These gene sets included the set of 543 mouse transcription factors detected in the OSNs (<span class="dropBlock reference-citations"><a id="body-ref-sref1" href="#" role="doc-biblioref" data-xml-rid="bib1" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref1" aria-controls="sref1">Aibar et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">1.</div><div>Aibar, S. ∙ González-Blas, C.B. ∙ Moerman, T. ...</div><div><strong>SCENIC: Single-cell regulatory network inference and clustering</strong></div><div><em>Nat. Methods.</em> 2017; <strong>14</strong>:1083-1086</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.4463" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_1_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-85032583384" target="_blank" aria-label="Open Scopus in new tab">Scopus (656)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28991892/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.4463&amp;pmid=28991892" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), and a set of 119 axon guidance genes (defined as known cell adhesion molecules with the following prefixes: <i>Slit, Robo, Epha, Ephb, Efn, Nrp, Sema, Plx, Kirrel, Pcdh, Cdh, Tenm;</i> using these prefixes was important because common GO terms for axon guidance genes also include OSN-specific transcription factors like <i>Lhx2</i> and <i>Bcl11b</i> and other genes like ion channels that are not directly involved in axon guidance). As a comparison, classification was performed using only the top 0.75% of highly variable genes based on their F-score (rather than the default used above of 10%, thus yielding ∼100 rather than 1,300 genes). Optimal hyperparameters for feature selection (based on their F-score in the training folds, as before) for the percent of genes from each gene set to keep, as well as the number of dimensions for PCA and LDA and SVM regularization strength were identified via randomized searches for each gene set. Classification was also performed after removing all transcription factors and axon guidance genes (and increasing the F-score threshold to 10.5% to again use the top ∼1,300 genes). Models trained with only axon guidance genes or transcription factors performed substantially worse than those with the top 100 F-score genes, and excluding all axon guidance genes and transcription factors genes had little effect on classification accuracy (<a id="crosref1435" href="#figs1">Figure&nbsp;S1</a>M).</div></section><section id="sec-4-4-23-6"><h5>Classification generalization across mice</h5><div id="p0410" role="paragraph">Pairwise and OR identity classification was performed with the same pipeline described above. Rather than just generalizing across held-out cells (which may have been from either the same or other mice), here classification was performed by leaving out all of the OSNs from single mouse, while the remaining 5 out of 6 mice were used for training to fit the classification pipeline (gene selection, PCA and LDA transformations, and the SVM decision boundaries). For each mouse, ORs found in at least 10 cells in the 5 training mice and at least 4 cells in the test mouse were used, and classification accuracies for each OR in the test mouse were evaluated across 1,000 restarts. Similar accuracies were observed when shuffling mouse identities, demonstrating that models successful generalize across mice. Generalization performance was at chance levels when trained on models with shuffled OR labels.</div></section><section id="sec-4-4-23-7"><h5>Classification of OR identity in “OR-swap” mice</h5><div id="p0415" role="paragraph">Classification was performed for the “OR-swap” mice using the PCs fit on the set of 1,350 HVGs. The top 10 PCs were used as inputs to “one-vs-one” SVM linear classifiers trained to distinguish between M72, S50, and P2. Using the 42/18/84 cells for M72/S50/P2 in the wild-type data, the models were trained using 15-fold cross validation (subsampling 15 cells for each OR) to evaluate their ability to distinguish between wild-type OSNs expressing the three ORs. Models fit on only the 15 wild-type cells subsampled for each OR also successfully generalized to accurately predict the OR identities in the more than 500 held-out cells for each OR from the “OR-swap” and P2<sub>(P2 locus)</sub> mouse lines. Classification accuracy was summarized across 1,000 restarts.</div></section><section id="sec-4-4-23-8"><h5>Classification of OR classes</h5><div id="p0420" role="paragraph">Classification of OR class was performed at the OSN subtype level (by taking the means of all the OSNs expressing each OR) between the 504/207/105 ventral class II, dorsal class II, and dorsal class I ORs. For each of 1,000 restarts, 80 OSN subtypes were selected for each OR type and classification was performed using 10-fold cross-validation with a pipeline that used linear SVMs (SVC with kernel&nbsp;= “linear,” decision_function_shape&nbsp;= “one-vs-one,” and regularization parameter C&nbsp;= 0.03) fit on the top 10 PCs in each training fold to predict the OR class of the held-out ORs. Classification accuracies were at chance performance (33.3%) upon shuffling the OR type labels across OSN subtypes.</div></section></section><section id="sec-4-4-24"><h4>Consensus non-negative matrix factorization (cNMF)</h4><div id="p0425" role="paragraph">Non-negative Matrix Factorization (NMF) was used to decompose the transcriptomes of each OSN into a smaller set of interpretable factors. NMF decomposes the gene expression matrix (cells x HVGs) into a combination of two matrices (W <sup>∗</sup> H), one containing gene loadings (H) for each factor and another containing the usages (W) of these factors in each cell (<a id="crosref1440" href="#fig2">Figure&nbsp;2</a>A). In contrast to PCA, these NMF factors often represent biologically meaningful and interpretable gene sets, and therefore were referred to as gene expression programs (GEPs). Because NMF is non-deterministic, a recently-described consensus NMF (cNMF) strategy was used, which has successfully found identity- and activity-related GEPs in a range of scRNA-seq datasets (<span class="dropBlock reference-citations"><a id="body-ref-sref50-2" href="#" role="doc-biblioref" data-xml-rid="bib50" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref50-2" aria-controls="sref50-2">Kotliar et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref50-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">52.</div><div>Kotliar, D. ∙ Veres, A. ∙ Nagy, M.A. ...</div><div><strong>Identifying gene expression programs of cell-type identity and cellular activity with single-cell RNA-seq</strong></div><div><em>eLife.</em> 2019; <strong>8</strong>:e43803</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.7554/eLife.43803" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-85070056080" target="_blank" aria-label="Open Scopus in new tab">Scopus (43)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31282856/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.7554%2FeLife.43803&amp;pmid=31282856" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). TPT-normalized data rather than raw counts (as used previously) were used as inputs to the cNMF procedure, since biological variation in the number of counts per cell was not expected given that all cells were mature OSNs. Each gene was scaled by its standard deviation using the standard deviations described above.</div><div id="p0430" role="paragraph">In brief, as previously described (<span class="dropBlock reference-citations"><a id="body-ref-sref50-3" href="#" role="doc-biblioref" data-xml-rid="bib50" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref50-3" aria-controls="sref50-3">Kotliar et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref50-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">52.</div><div>Kotliar, D. ∙ Veres, A. ∙ Nagy, M.A. ...</div><div><strong>Identifying gene expression programs of cell-type identity and cellular activity with single-cell RNA-seq</strong></div><div><em>eLife.</em> 2019; <strong>8</strong>:e43803</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.7554/eLife.43803" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-85070056080" target="_blank" aria-label="Open Scopus in new tab">Scopus (43)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31282856/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.7554%2FeLife.43803&amp;pmid=31282856" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), cNMF applies NMF multiple times and, after excluding outliers (based on their distance to nearest neighbors, density_threshold&nbsp;= 0.3), clusters the gene loadings across restarts with <i>k</i>-means clustering and uses the median of these clusters as the new gene loadings. In each of 20 restarts in which 10 cells were resampled for each of the 831 ORs, NMF was refit 150 times across a range of factors. A rank 18 decomposition, which balanced the tradeoff between reconstruction error (as measured by the Frobenius norm) and the consistency of the identified GEPs across restarts (as measured by the silhouette score), was used for the analyses presented here, but similar GEPs were observed with factorizations of other ranks. To match factors across restarts, the 54,000&nbsp;× 54,000 distance matrix (18&nbsp;× 150&nbsp;× 20) was subject to <i>k-</i>means clustering using the value of <i>k</i> (19) that maximized the silhouette score. Clusters for stable factors should have close to 3,000 members (20&nbsp;× 150). For each of the 19 clusters, the median loading across cluster members for each gene were used as the consensus gene loadings for that GEP. Three of these 19 clusters either appeared in few restarts or had non-zero usages only in small numbers of cells and were not considered further. For the 16 remaining GEPs, the loadings were normalized to sum to 1 for each GEP; these loadings can be found in <a id="crosref1455" href="#mmc2">Table S2</a>. Using this set of 16 GEPs as the new H matrix, the usages of these GEPs for each OSN were calculated by reapplying NMF while keeping the gene loadings constant (with update_H&nbsp;= False).</div><section id="sec-4-4-24-1"><h5>Identification of OR-specific GEPs, the environmental state (ES) GEPs, and calculation of the ES score</h5><div id="p0435" role="paragraph">Individual GEPs may explain variation across ORs, allowing for accurate classification of OR identities, or they may explain other,&nbsp;independent, aspects of variation. To distinguish between these possibilities, across 10,000 restarts 10 cells were randomly subsampled for each OR, and the correlation between the mean GEP usages for cells expressing each OR (vectors of length 831 for each of the 16 GEPs) were compared between the first and second half of cells for each OR. The 10 GEPs with consistently-high correlations (R &gt; 0.9) were considered “OR-specific,” and each had higher gene loadings in a restricted set of HVGs (<a id="crosref1460" href="#figs2">Figures S2</a>A and S2B).</div><div id="p0440" role="paragraph">GEPs were identified based on manual inspection of genes with high loadings and high correlations with the usages for that GEP; four GEPs included known dorsal, ventral, anterior or posterior identity marker genes, while one GEP was specific to genes associated with CD36 positivity. One GEP contained genes like <i>Ddit3</i> (CHOP) that are associated with ER stress and the unfolded protein response, but this GEP was not identified as OR-specific in our mature OSN dataset. GEPs that were not OR-specific (GEPs 11-16) and whose means across cells expressing the same OR are therefore not meaningful, or GEPs whose functions were unable to be identified based on their gene loadings (GEPs 8-10, which also had usages that were weakly correlated with other GEPs) were not considered for downstream analyses.</div><div id="p0445" role="paragraph">Two of the OR-specific GEPs encoded many genes associated with neural activity in OSNs (reviewed in (<span class="dropBlock reference-citations"><a id="body-ref-sref101-3" href="#" role="doc-biblioref" data-xml-rid="bib101" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref101-3" aria-controls="sref101-3">Wang et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref101-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">104.</div><div>Wang, Q. ∙ Titlow, W.B. ∙ McClintock, D.A. ...</div><div><strong>Activity-dependent gene expression in the mammalian olfactory epithelium</strong></div><div><em>Chem. Senses.</em> 2017; <strong>42</strong>:611-624</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/chemse/bjx028" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_104_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-85030719241" target="_blank" aria-label="Open Scopus in new tab">Scopus (7)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28525560/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fchemse%2Fbjx028&amp;pmid=28525560" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)). One&nbsp;included <i>S100a5</i>, <i>Pcp4l1</i>, and <i>Kirrel2</i> (which are downregulated by chronic sensory deprivation), while another included <i>Calb2</i>, <i>Kirrel3</i>, <i>Ppp3ca</i> (which are upregulated after chronic sensory deprivation; see <a id="crosref1470" href="#fig4">Figure&nbsp;4</a> and <a id="crosref1850" href="#mmc3">Table S3</a>). These two GEPs were referred to as GEP<sub>High</sub> and GEP<sub>Low</sub>. The usages of GEP<sub>High</sub> and GEP<sub>Low</sub> were anti-correlated and expressed in a mutually-exclusive manner across a continuous range of values (<a id="crosref1475" href="#fig2">Figures 2</a> and <a id="crosref1480" href="#figs2">S2</a>). The difference in their usages was therefore summarized as a single metric to define a single continuous axis of gene expression. This difference (GEP<sub>High</sub> - GEP<sub>Low</sub>) was referred to as the Environmental State (ES) score, and GEP<sub>High</sub> and GEP<sub>Low</sub> were collectively referred to as ES GEPs, given the hypothesis that variation in ES GEP usages and thus ES scores across OSNs expressing different ORs is a consequence of the differential contribution of environmental odors on the activity levels and expression of ES-related genes across OSNs.</div><div id="p0450" role="paragraph">Among the identity GEPs, GEP<sub>Dorsal</sub> and GEP<sub>Ventral</sub>, as well as GEP<sub>Anterior</sub> and GEP<sub>Posterior</sub> also varied smoothly and in a similar mutually-exclusive manner across ORs; their differences in usages were therefor also summarized into single metrics (DV score&nbsp;= GEP<sub>Dorsal</sub> - GEP<sub>Ventral</sub> and AP score&nbsp;= GEP<sub>Anterior</sub> - GEP<sub>Posterior</sub>) to capture the continuous variation across these 3 independent axes (DV, AP, and ES; see <a id="crosref1485" href="#figs2">Figure&nbsp;S2</a>). The DV scores were distributed bimodally, and OSNs whose DV score&nbsp;was at least 40 were considered as dorsal OSNs, yielding consistent results as the manual gene-based identification of dorsal OSNs described above.</div></section><section id="sec-4-4-24-2"><h5>Identification of neuronal and functional genes in ES GEPs</h5><div id="p0455" role="paragraph">Genes associated with each of GEP<sub>High</sub> and GEP<sub>Low</sub> were identified by two criteria: 1) the gene was within the top 200 genes with the highest loading for that GEP and 2) the expression of that gene was correlated with the GEP usage on OR-by-OR basis in the home cage data (the Spearman’s rank correlation between the vectors of GEP means and gene means across the 831 ORs was 0.25). This approach identified 168 and 177 genes in GEP<sub>High</sub> and GEP<sub>Low</sub>, respectively. The potential functions/ontologies of these genes were manually annotated, and 63 and 54 “neuronal” genes with known or proposed function in neurons were identified for the two ES GEPs. As listed in <a id="crosref1490" href="#mmc3">Table S3</a>, these neuronal genes were categorized according to their putative roles in either calcium homeostasis (calcium binding, calcium signaling, inositol phosphatide related) ion transfer (ion channel, ion pump/transporter), axon guidance (axon guidance, cell adhesion), synapse (synapse, secretory peptide), or protein transport (ER/Golgi/cilia related). The remaining genes with high loadings and correlations with the ES GEPs had similar patterns of gene expression in OSNs as the above genes, but their functions in OSNs remain unannotated. The “neuronal” genes were further restricted to include only the “functional” gene categories that are predicted to directly impact OSN odor responses, and which included the 73 genes that were part of the calcium homeostasis, ion transfer and OR signaling categories (see <a id="crosref1495" href="#mmc3">Table S3</a>). Of note, while many known components of OR signaling are included in this list, the expression of <i>Ano2</i> and <i>Adcy3</i> remained constant across OSN subtypes and did not meet the above correlation thresholds (ρ&nbsp;&lt; 0.2 for GEP<sub>High</sub>, GEP<sub>Low</sub>, and the ES score for both genes). Furthermore, although <i>Omp</i> was weakly correlated with the ES score (ρ&nbsp;= 0.36), because it was not part of the HVGs it did not contribute to the usages of any GEP.</div></section><section id="sec-4-4-24-3"><h5>Applying cNMF from home-cage data to other datasets</h5><div id="p0460" role="paragraph">Although cNMF gene loadings were identified using the home-cage data, they were subsequently applied to other datasets to thus compare, on an OR-by-OR basis, changes in gene expression and GEP usages across experimental conditions. For all datasets, the TPT-normalized HVG expression from that dataset was scaled by the standard deviations from the home-cage data that were used when initially fitting cNMF. Using the fixed set of gene loadings (H), new usages (W) were solved by refitting NMF (update_H&nbsp;= False). To compare GEP usages across conditions, the mean GEP usage in that condition was calculated for each OSN subtype for all cells expressing that OR, and then differences between conditions were computed for each OSn subtype; distributions depict the change across OSN subtypes.</div></section><section id="sec-4-4-24-4"><h5>Comparisons of ES scores across mice and conditions</h5><div id="p0465" role="paragraph">Changes in ES scores were evaluated by comparing the ES scores for each OSN subtype across experimental conditions. For chronic and transient occlusion ES scores from occluded nostrils were compared to those from open nostrils. For environment switches ES scores were compared for each OSN subtype between pairs of environments. For optogenetic stimulation, the change in ES scores was compared between mice housed in the home cages and those that either were or were not optogenetically-stimulated. In the chronic acetophenone experiments, ES scores were compared between mice that were chronically exposed to various concentrations of acetophenone and those that were exposed to the control solvent DPG. The changes in ES scores and z-scored functional gene expression across conditions for each OSN subtype are described in <a id="crosref1500" href="#mmc4">Table S4</a>. The relative ES score change for each OSN subtype was calculated by subtracting the mean across all OSN subtypes, to normalize for any global changes across conditions that were present across the population of OSNs.</div><div id="p0470" role="paragraph">The Spearman’s rank correlation was also used to compare the consistency of ES scores in OSNs expressing the same OR across datasets. ORs identified in each dataset were compared, and the correlation coefficient was calculated on an OR-by-OR basis between the vectors of mean usages for each OSN subtype in each dataset (where each value is the mean ES score across all OSNs that express that OR in each dataset). On-diagonal values for pairwise correlation matrices were calculated via bootstrapping, by resampling with replacement the ES scores for each OSN subtype from the OSNs expressing that OR and taking the mean of the pairwise correlations (of the OR-by-OR ES score means) between all pairs (10,000) from 100 bootstrap restarts.</div><div id="p0475" role="paragraph">In all experiments, identity-related GEP usages were stable for each OR across conditions (ρ &gt; 0.95), whereas the ES GEPs and ES scores moved predictably with changes in activity. To assess how the change in ES scores related to the overall change in transcriptomes across conditions, equal numbers of cells were subsampled for each OR, and the difference in the mean ES score for each OR was correlated with the PCs of the Δ HVG versus OR matrix (the difference in mean expression of each HVG between cells expressing each OR in each condition). In the chronic occlusion, optogenetic stimulation, and environment switch experiments, the changes in ES scores were strongly correlated (ρ &gt; 0.8) with the top PC of this matrix, and thus well aligned with the main axis of overall gene expression changes. Given that ES scores are also aligned with the top PC of the transcriptomes in the home cage, these data together suggest that activity-regulated ES genes define a main axis of transcriptional variation, and experimental manipulation causes restricted changes in gene expression along this axis.</div></section><section id="sec-4-4-24-5"><h5>Identifying ORs with significant ES score changes across datasets</h5><div id="p0480" role="paragraph">ORs with significant changes in their associated ES scores across datasets were identified empirically using the d-prime metric, which measured the separability of the ES score distributions for each OSN subtype. Within each OSN subtype, the d-prime statistic was calculated by taking the difference in the mean ES scores for each condition and dividing by the square-root of the average of the variance in the two conditions. This observed d-prime statistic was compared to the distribution of resulting d-prime values from permuting the condition labels across all cells of a given OSN subtype 10,000 times. P values were calculated empirically, corrections for multiple comparisons were performed via the Benjamini-Hochberg FDR procedure, and FDR-corrected p values ≤ 0.01 were considered as statistically significant. ORs with significant shifts in the chronic occlusion and environment switch experiments had mean d-prime values of 5.63 and 2.50, respectively, indicating excellent separability. OSN subtypes with significant ES score changes were separated into those whose mean ES scores increased and decreases based on the sign of the difference in mean ES scores across datasets. In the environment switch experiments, each pair of environments was analyzed separately, but results are concatenated across the 3 environment pairs (and therefore in plots depicting OSN subtypes there are 3 datapoints for OSN subtypes identified in all 3 environments).</div></section><section id="sec-4-4-24-6"><h5>Classification of OR identity with NMF GEPs</h5><div id="p0485" role="paragraph">Pairwise OR classification and classification of OR identity was performed using the same cross-validation and balanced subsampling procedures described above. However, rather than performing gene selection and dimensionality reduction during training, the direct input to the linear SVM classifiers was the GEP usages for each OSN for either all 10 OR-specific GEPs or only the two ES GEPs, as specified. Classification was performed using the 831 ORs found in at least 10 cells in the home-cage data, the 436 ORs found in at least 10 cells in both nostrils in the chronic occlusion data, and in the 488/496/522 ORs found in at least 8 cells in home-cage, environment A and environment B datasets from the environment switch experiments. Despite using only 2 or 10 dimensions to distinguish the entire set of ORs, classification accuracies were above chance for all datasets, and at chance levels upon shuffling OR labels for each OSN.</div></section><section id="sec-4-4-24-7"><h5>Classification of nostril and environment identities using the NMF GEPs and ES scores</h5><div id="p0490" role="paragraph">Rather than identifying the OR expressed in each cell, linear SVM classifiers were also used to test whether OSNs expressing the same OR were distinguishable across datasets based on their GEP usages. Separate classifiers were used for each OSN subtype and <i>k</i>-fold cross-validation was used, with <i>k</i> set to be equal to the numbers of cells subsampled per condition. As expected, classification accuracy was significantly above chance for the chronic occlusion and environment switch experiments, especially for the ORs with significant ES shifts (but ORs with smaller but non-significant shifts were still discriminable at above-chance levels). Classification performance was similar using all 10 GEPs or only the 2 ES GEPs, and at chance levels when the dataset label was permuted across OSNs for each of 1,000 restarts, further demonstrating that the main change across conditions is restricted to changes in the ES score axis.</div><div id="p0495" role="paragraph">Classification of environment identity was also performed across randomly selected subpopulations of OSN subtypes, using a minimum distance classification procedure. Across 1,000 restarts, six OSNs were subsampled for each OSN subtype for each environment (from the 477 subtypes present with enough cells in all environments), and five OSNs were used for training. The mean ES score for each subtype for each of the three environments was calculated (yielding three training vectors containing the mean ES scores across the population of 477 subtypes). The ES scores of the held-out cells for each subtype were similarly concatenated into three test vectors. For each of the 1,000 restarts, subpopulations of the OSN subtypes (containing 1–50 subtypes) were randomly sub-selected 1,000 times and the training vector for that subpopulation with the minimum distance to the same subpopulation from each test vector was the predicted environment label; predictions were considered accurate if the training vector with the minimum distance came from the same environment as the test vector. Accuracies were summarized across restarts, and classification performance was at chance levels when the environment labels were shuffled across training cells for each OSN subtype.</div></section><section id="sec-4-4-24-8"><h5>Modified ES scores with HVG subsets</h5><div id="p0500" role="paragraph">Modified versions of the ES GEPs were constructed to assess the contributions of individual genes within the ES GEPs. Although many genes have low loadings for each GEP and a restricted subset have higher loadings, each GEP was defined based on their loadings across the entire set of 1,350 HVGs. To test whether the 73 functional genes could substitute for the entire set of HVGs, the loadings for all of the remaining HVGs were set to 0, the total loadings for each GEP were rescaled to sum to 1 across the 73 functional genes, and this new gene loading (H) matrix was used to calculate updated usages (W) by reapplying NMF (with update_H&nbsp;= False).</div><div id="p0505" role="paragraph">The “functional” ES score, constructed again as GEP<sub>High</sub> - GEP<sub>Low</sub> from the new set of usages was highly correlated (ρ &gt; 0.99) with the ES score from all HVGs, decreased as expected upon occlusion, and could distinguish between OSNs expressing different pairs of ORs using the classification procedure described above, thus indicating that the “functional” genes and their expression can substitute for the entire set of HVGs (<a id="crosref1505" href="#figs4">Figure&nbsp;S4</a>). We also observed that the levels of some of the activation genes varied slightly across environments. To assess the contribution of these activation genes on ES scores and on the ES scores changes measured after environmental shifts, the loadings for the 169 activation genes that were part of the HVGs were set to 0 for all GEPs and the GEP usage and ES score was recalculated as described above. The ES scores and changes in ES scores calculated using all HVGs or excluding the activation genes were highly correlated (ρ &gt; 0.99), suggesting that activation genes (whose expression primarily changes during periods of acute activation) do not account for the significant shifts in ES scores observed across environments.</div></section></section><section id="sec-4-4-25"><h4>Identification and pseudotime analysis of immature OSNs</h4><div id="p0510" role="paragraph">Pseudotime analysis was performed on the immature OSNs identified from mice housed in the home cages. The Leiden clustering procedure that was used to identify mature OSNs also identified clusters of immature OSNs, which were used for pseudotime analysis. Diffusion pseudotime (n_dcs&nbsp;= 10) was calculated in Scanpy on the nearest neighbor graph (n_pcs&nbsp;= 10, n_neighbors&nbsp;= 10), and normalized to the lowest pseudotime values of immature OSNs that had undergone OR choice and were expressing ORs. To compare gene expression patterns and GEP usages in immature OSNs expressing given ORs with those of their respective mature OSN subtype, only immature OSNs that had chosen and expressed ORs were used. Gene expression and GEP usage as a function of pseudotime were evaluated using linear generalized additive models (GAMs) in pyGAM. A linear GAM was fit using 15 basis splines (lambda&nbsp;= 1) for individual genes and fit using 50 splines (lambda&nbsp;= 5) for GEP usages, and the mean and 95% confidence intervals of the mean were evaluated across a grid of 500 evenly-spaced pseudotime values. The mature ES scores across OSN subtypes were discretized into five quantiles, and separate GAMs were fit using the immature OSNs expressing ORs from each mature ES score quantile.</div></section><section id="sec-4-4-26"><h4>Differential expression testing</h4><div id="p0515" role="paragraph">Given the wide variation in gene expression across OSNs in the home-cage condition, differential expression testing was performed at the OR-by-OR level to identify genes that consistently changed across multiple OSN subtypes. Differential expression (DE) testing was performed using ORs found in at least 6 cells in each dataset to balance the trade-off between the number of OSN subtypes and within-subtype variance. Testing was performed via the Wilcoxon signed-rank test (comparing, for each gene, the differences between means for each OSN subtype across conditions). P values were corrected for multiple comparisons across genes via the Benjamini-Hochberg FDR procedure.</div><div id="p0520" role="paragraph">For chronic occlusion, the DE genes with the most significant changes were defined as those among the top 500 genes with the lowest p values that had consistent changes across ORs (same sign of change in at least 50% of the 673 OSN subtypes detected in at least 6 cells in each nostril and absolute value of the median log<sub>2</sub>-fold change across OSN subtypes of at least 0.5); however, consistent results were obtained across a wide range of thresholds given that most DE genes changed in the majority of the OSN subtypes. As expected, genes that increased or decreased during occlusion had high loadings for GEP<sub>Low</sub> and GEP<sub>High</sub>, respectively.</div><div id="p0525" role="paragraph">DE testing via the Wilcoxon signed-rank test was also performed to identify genes that were acutely responsive to odor exposure. For each odor, the same DE testing procedure was performed separately for activated and non-activated OSN subtypes (as defined based on the percentage of activated cells expressing that OR; see below). To exclude the small subsets of genes that changed indiscriminately across all OSN subtypes regardless of whether they had been activated by odor, genes whose p value was higher for non-activated ORs or those whose mean change in expression across OSN subtypes was less than 1.5 times higher in the activated than non-activated OSN subtypes were removed and not considered for downstream analyses.</div></section><section id="sec-4-4-27"><h4>Identification of activated cells and ORs by Act-seq</h4><div id="p0530" role="paragraph">OSN subtypes that were responsive to odorants were identified via a two-step process. First, individual OSNs (as defined by the expressed OR) were determined as activated based on their mean z-scored immediate early genes (IEG) expression. A set of 10 IEGs were used (<i>Btg2, Egr1, Fos, Fosb, Gm13889, Junb, Nr4a1, Nr4a2, Pcdh10,</i> and <i>Srxn1</i>). These genes were reliably and strongly induced in odor-exposed mice and, unlike many of the activity-dependent genes that are part of the ES GEPs and whose expression varies widely across OSNs expressing different ORs, these IEGs had consistently low expression across OSNs in control mice (average expression ∼0.15 in the control conditions and average log<sub>2</sub> fold-change of ∼5 in the odor conditions). The set of 10 IEGs was used since even though all of the IEGs were specific to acute activation by odors, they were still only detected in a minority of cells due to the sparse nature of scRNA-seq. The means and standard deviations for IEG z-scoring were fit on the TPT-normalized IEG expression (without the log transformation due to the sparsity of expression) across 100 restarts containing equal numbers of subsampled cells for each OR from mice that were exposed to either acetophenone or octanal. OSNs whose mean z-scored expression was above 0.2 (∼15% of OSNs in odor-exposed mice but less than 2% of OSNs in control mice exposed to the DPG solvent alone).</div><div id="p0535" role="paragraph">Second, as expected, the distribution of the percent of OSNs per OSN subtype was bimodal, with a majority of OSN subtypes showing low percentages of activated OSNs with a smaller subset showing high percentages of activated cells. Using the ORs detected in at least 4 cells in both control and odor-exposed conditions, activated ORs were defined as those in which less than 20% of cells expressing that OR were activated in the control conditions and more than 70% of cells were activated in the odor-exposed conditions, with the additional requirement of non-zero expression of at least 4 IEGs (to avoid false positives from OSNs activated by semiochemicals in the empty cages used for Act-Seq or those that strongly induced a small number of IEGs). Given that the set of ORs and the number of cells expressing each OR differed across datasets, odor-responsive OSN subtypes were re-evaluated for each dataset. Overall, odor-responsive OSN subtypes that were detected in multiple datasets were consistently identified as odor-responsive, though a small fraction of them (and thus their expressed ORs) were identified as activated in some datasets but only partially activated in others. These results are likely due to both the consequences of experimental manipulations, which were designed to affect odor responses, as well as the relatively conservative thresholds used above to identify ORs as activated.</div><div id="p0540" role="paragraph">For the environment switch experiments and the Act-Seq experiments using the four acetophenone-related odors, activated ORs were identified as described above, except using an IEG threshold of −0.05 because of the higher levels of IEG expression in the 10x v3.1 data. For the acetophenone concentration series experiment, activated ORs were either identified for each concentration, or the ORs identified as activated at 10% were also examined at lower concentrations, as specified in the text. In the optogenetic stimulation experiments, no IEG thresholds were used to identify activated cells, and instead all dorsal OSN subtypes were used for downstream analyses. In the chronic acetophenone exposure experiments, the set of acetophenone-responsive ORs identified via the acute Act-Seq experiments were used.</div></section><section id="sec-4-4-28"><h4>Quantifying the magnitude of odor responses via the activation score</h4><div id="p0545" role="paragraph">Although IEG induction could identify activated OSNs in a binary manner, the sparse nature of the IEGs made them less suitable for measuring differences in the degree of activation. Therefore, a larger set of odor-responsive genes (from the DE genes identified above) were used to construct an analog metric for activation and therefore compare differences in the amount of activation across OSN subtypes (as defined by their expressed ORs). The set of “activation” genes used for this metric satisfied the following criteria 1) significant responses (FDR ≤ 1&nbsp;× 10<sup>−3</sup>) to both acetophenone and octanal, and 2) average TPT-expression of at least 0.8 in activated ORs (such that the metric wouldn’t be biased by the presence of zeros). Acetophenone and octanal induced similar changes in gene expression in their respective responsive OSN subtypes. Of the ∼500 genes passing these criteria, the small number of genes that had high (ρ &gt; 0.4) correlations with the ES score in the control mice were removed, leaving a set of 472 “activation” genes that were used for all Act-Seq experiments (<a id="crosref1510" href="#mmc5">Table S5</a>).</div><div id="p0550" role="paragraph">Consistent with activation score genes generally not being differentially used in the home cage, the majority of these activation genes were not part of the 1,350 HVGs in the home-cage data, and thus do not contribute to any GEP usages. The remaining 169 activation genes that were also considered as HVGs often had weak loadings across multiple GEPs. Only 16 of these 169 genes had high loadings for GEP<sub>High</sub>, and removing all activation genes from the HVGs and recomputing GEP usages yielded similar results. Together, these results suggest that activation gene expression is specific to the acute effects of odor exposure and does not contribute significantly to baseline variation in transcriptional identities or ES scores across OSN subtypes. To summarize the effects of odor activation across this set of activation genes, the TPT-expression of these activation genes were z-scored (using means and standard deviations for these genes in the control, acetophenone, and octanal datasets, identified by subsampling equal numbers of cells across all ORs expressed in at least 6 cells across 100 restarts). Next, using the combined set of activated acetophenone- and octanal-responsive OSN subtypes, PCA was performed on the differences in z-scored activation gene expression between control and odor conditions. The top PC of this activation gene delta matrix was kept, and the activation score was defined as the product of the weights for this top PC and the difference in z-scored activation gene expression between odor and control conditions.</div><div id="p0555" role="paragraph">Thus, the activation score summarizes for each OSN subtype the overall movement of the activation genes between control and odor conditions along this main activation axis. Although the activation score weights were fit using the combined set of acetophenone- and octanal-responsive OSN subtypes expressed in at least 6 cells for stability purposes, all subsequent Act-Seq results are shown using the larger sets of ORs detected in at least 4 cells in both control and odor conditions. To obtain activation scores for OSN subtypes from other datasets (such as those in the transient occlusion and environment switch experiments), the activation score was evaluated in a similar manner by multiplying the activation PC weights and the delta activation gene expression (odor – control) for each OSN subtype. Only activated OSNs from the odor-exposed mice were used for calculating the activation score. The weights were kept constant across experiments, but three separate versions of the gene scaling was used for the ACE and OCT Act-Seq experiments, the transient occlusion experiments, and all other Act-Seq and environment switch experiments, respectively, to account for changes in the gene means due to differences in 10x kit versions and sequencing depth across these datasets. In all experiments, activation scores were higher in odor-exposed and odor-responsive OSN subtypes. Across replicates from the same condition, OSN subtypes had consistent levels of activation, demonstrating that the activation score successfully captures differences in the amount of activation in each condition across OSNs expressing different ORs.</div></section><section id="sec-4-4-29"><h4>Unsupervised methods for identifying activated cells and ORs</h4><div id="p0560" role="paragraph">Two additional complementary methods were used to identify activated cells and measure activation in the ACE and OCT Act-Seq experiments. First unsupervised graph-based Leiden clustering (resolution&nbsp;= 0.6) was fit on the top 20 PCs from the scaled HVGs (where PCA was fit using equal numbers of cells from control, acetophenone, and octanal mice and applied to all cells). A single cluster bore the hallmarks of activated cells, and primarily contained cells from the odor condition with increased IEG expression. Additionally, nearly all of the OSNs from odor-exposed mice that expressed activated ORs, as identified via the IEG approach described above, were part of this activated cluster.</div><div id="p0565" role="paragraph">Second, activated ORs and weights on genes affected by odor exposure were simultaneously identified via unsupervised tensor component analysis (TCA). Nonnegative tensor decomposition was fit using the hierarchical alternating least-squares algorithm (ncp_hals) in the tensortools python package (<span class="dropBlock reference-citations"><a id="body-ref-sref105" href="#" role="doc-biblioref" data-xml-rid="bib105" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref105" aria-controls="sref105">Williams et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref105"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">108.</div><div>Williams, A.H. ∙ Kim, T.H. ∙ Wang, F. ...</div><div><strong>Unsupervised discovery of demixed, low-dimensional neural dynamics across multiple timescales through tensor component analysis</strong></div><div><em>Neuron.</em> 2018; <strong>98</strong>:1099-1115.e8</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_108_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=10.1016%2Fj.neuron.2018.05.015&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_108_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=10.1016%2Fj.neuron.2018.05.015&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_108_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-85047777491" target="_blank" aria-label="Open Scopus in new tab">Scopus (63)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29887338/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.neuron.2018.05.015&amp;pmid=29887338" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), and the tensor of conditions x activation genes x OSN subtypes was reduced with a rank 4 decomposition into 4 rank-1 tensors. One of these tensors represented activation and had 1) high weights in the odor but not control condition, 2) high weights for activation genes affected by acute odor exposure (like <i>Pcdh10</i> and <i>Fos)</i>, as well as 3) high weights for the odor-responsive OSN subtypes identified above based on their IEG expressions; furthermore, the TCA weights for the activation factor for each OSN subtype was correlated with the mean activation score for that subtype (ρ &gt; 0.85 for both ACE and OCT).</div></section><section id="sec-4-4-30"><h4>Comparison of ES scores and activation scores for Act-seq experiments</h4><div id="p0570" role="paragraph">ES scores were compared to the activation scores for each OSN subtype to test how OSN transcriptional variation affects odor responses. All correlations were calculated between the ES scores in control mice and the activation scores from separate odor-exposed mice, using the set of odor-responsive OSN subtypes detected in at least 4 cells in both control and odor conditions. Comparisons were made at the OSN subtype level, between the mean ES score from the control condition and that OSN subtype’s activation score. When comparing the change in ES score and change in activation scores across datasets, only OSN subtypes present in at least 4 cells in both control and odor conditions for each dataset being compared were used.</div><section id="sec-4-4-30-1"><h5>Predictions of activation scores from control ES scores and functional gene expression via linear regression models</h5><div id="p0575" role="paragraph">Cross-validated linear regression models were constructed using the TPT-normalized expression of the functional genes in the control mice as predictors of the activation score in odor-exposed mice. Regularization was performed using elastic-net regularized linear models, where the optimal hyperparameters (alpha, the l1-ratio, and standardization of regressors) were identified through grid searches for each dataset. 5-fold cross-validation was used, and model performance was evaluated via the mean squared error (MSE) between predicated and observed activation scores from each test fold. This cross-validation procedure was repeated 1,000 times and the distribution of observed MSEs was compared to the predictions obtained when the same models were trained and tested on data in which the activation scores were permuted across OSN subtypes; this shuffling procedure represents performance similar to that of null models that predict the mean activation score for all subtypes. Reported errors were normalized as a percentage of the observed data to facilitate comparisons across datasets. For a subset of datasets, the predictions of the activation score from the functional genes were also compared to predictions from ordinary least-squares (OLS) linear regression models that used the control ES score as a single predictor. Predictions of the changes in activation scores were evaluated in a similar way to those of the activation score, except the changes in functional gene expression for each OSN subtype were used as predictors. In addition to shuffling the change in activation score across OSN subtypes, these models were also evaluated on data where the condition label was permuted across OSNs for each OSN subtype before calculating the changes in the functional gene expression for that subtype.</div></section></section><section id="sec-4-4-31"><h4>Classification of odors and environments from activation scores</h4><div id="p0580" role="paragraph">Classification of odor identity for the acetophenone analogs based on activation scores was performed using a minimum distance procedure in a similar manner to classification of environmental identity via ES scores. The input was the set of 106 OSN subtypes whose ORs were activated by at least one of the four acetophenone-related odors and were detected in at least four OSNs for each odor (since few ORs were present and activated in enough cells across all four datasets; but see below). Across 1,000 restarts, three OSNs per odor were used for training and the fourth was used for testing. The mean activation score for each subtype for each odor was calculated. For each of the 1,000 restarts, subpopulations of the OSN subtypes were randomly sub-selected 1,000 times and predictions were considered accurate if, across the subpopulation of activation scores, the training vector with the minimum distance for each test vector shared the same odor label. Predictions were at chance levels upon shuffling odor labels across cells for each OSN subtype. The same classification procedure was also used for each pair of acetophenone odors, but the input was the set of ORs activated by both analogs for each pair to determine if the pattern of activation scores across OSN subtypes can distinguish between odors, even for ORs activated by both odors.</div><div id="p0585" role="paragraph">Classification of environments using OSN activation scores after odor exposure as predictors was performed using cross-validated linear SVM classifiers. Activation scores were computed for each OSN (rather than the OSN subtype as described above) by applying the activation gene weights to the difference in expression between the activation gene expression in each cell in the odor condition and that of the mean of its respective OSN subtype in the control condition. Classification was performed as before by subsampling equal numbers of cells per OR on each of 1,000 restarts, and prediction accuracies were at chance levels when the OR labels were shuffled across OR pairs or the environment pair labels were shuffled across the OSNs expression each OR.</div></section><section id="sec-4-4-32"><h4><i>in&nbsp;vitro</i> odor-receptor interaction metrics</h4><div id="p0590" role="paragraph">The set of activated ORs identified via Act-seq were compared to those identified via the DREAM and phospho-S6 immunoprecipitation methods (<span class="dropBlock reference-citations"><a id="body-ref-sref42-8" href="#" role="doc-biblioref" data-xml-rid="bib42" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref42-8" aria-controls="sref42-8">Jiang et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref42-8"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">42.</div><div>Jiang, Y. ∙ Gong, N.N. ∙ Hu, X.S. ...</div><div><strong>Molecular profiling of activated olfactory neurons identifies odorant receptors for odors in&nbsp;vivo</strong></div><div><em>Nat. Neurosci.</em> 2015; <strong>18</strong>:1446-1454</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nn.4104" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-84942414428" target="_blank" aria-label="Open Scopus in new tab">Scopus (62)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26322927/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnn.4104&amp;pmid=26322927" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref97-5" href="#" role="doc-biblioref" data-xml-rid="bib97" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref97-5" aria-controls="sref97-5">von der Weid et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref97-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">100.</div><div>von der Weid, B. ∙ Rossier, D. ∙ Lindup, M. ...</div><div><strong>Large-scale transcriptional profiling of chemosensory neurons identifies receptor-ligand pairs in&nbsp;vivo</strong></div><div><em>Nat. Neurosci.</em> 2015; <strong>18</strong>:1455-1463</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nn.4100" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_100_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-84942456919" target="_blank" aria-label="Open Scopus in new tab">Scopus (61)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26322926/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnn.4100&amp;pmid=26322926" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). <i>In&nbsp;vitro</i> EC<sub>50</sub> values to acetophenone for its receptors were obtained from (<span class="dropBlock reference-citations"><a id="body-ref-sref42-9" href="#" role="doc-biblioref" data-xml-rid="bib42" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref42-9" aria-controls="sref42-9">Jiang et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref42-9"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">42.</div><div>Jiang, Y. ∙ Gong, N.N. ∙ Hu, X.S. ...</div><div><strong>Molecular profiling of activated olfactory neurons identifies odorant receptors for odors in&nbsp;vivo</strong></div><div><em>Nat. Neurosci.</em> 2015; <strong>18</strong>:1446-1454</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nn.4104" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-84942414428" target="_blank" aria-label="Open Scopus in new tab">Scopus (62)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26322927/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnn.4104&amp;pmid=26322927" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref80-4" href="#" role="doc-biblioref" data-xml-rid="bib80" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref80-4" aria-controls="sref80-4">Saito et&nbsp;al., 2009</a><div class="dropBlock__holder" data-db-target-of="sref80-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">82.</div><div>Saito, H. ∙ Chi, Q. ∙ Zhuang, H. ...</div><div><strong>Odor coding by a mammalian receptor repertoire</strong></div><div><em>Sci. Signal.</em> 2009; <strong>2</strong>:ra9</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/scisignal.2000016" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_82_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-65949098470" target="_blank" aria-label="Open Scopus in new tab">Scopus (363)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19261596/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscisignal.2000016&amp;pmid=19261596" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Only ORs from these prior papers that were also identified in the Act-seq experiments described above were used for comparisons. The 717 ORs detected in four or more OSNs in both DPG and acetophenone-exposed animals included 16/22 DREAM and 25/49 phospho-S6 ACE ORs. ORs that were not identified by Act-seq were also not identified in more than 4 OSNs in these experiments, suggesting that these receptors might be “missing” because they are expressed in particularly small numbers of OSNs; note that this observation raises the possibility that these receptors are also near the noise floor in the DREAM and pS6 methods, since these approaches rely on bulk sequencing approaches that are also likely affected by OR frequency. In contrast to the relationship observed between the control ES score and activation scores, there was no relationship between <i>in&nbsp;vitro</i> EC<sub>50</sub> values and activation scores from any concentration of ACE and OLS linear regression models were unable to predict activation scores from the EC<sub>50</sub> values alone.</div></section><section id="sec-4-4-33"><h4>Comparisons of acute and long-term ES changes upon environment shifts</h4><div id="p0595" role="paragraph">OSN specific activation scores were measured after mice were shifted from the home-cage to environment A (envA). The activation score at these time points was then compared to the two-week changes in the ES score (as measured as the difference in the ES score between home-cage and envA-housed mice in the environment switch experiments). The activation score was calculated by applying the activation gene weights to the activation gene deltas between mice from each time point and control mice housed in disposable cages with regular food/bedding overnight. The activation scores were normalized by subtracting the mean activation score across OSN subtypes, to account for any non-specific changes in activation across the entire population (e.g., due to changes in odor sampling that may occur upon the transfer to new environment cages). The resulting “relative activation scores” were compared between OSN subtypes with significant increases and decreases in their ES scores after two weeks in envA, for subtypes present in at least 4 cells in all 4 conditions (home-cage and envA after two-weeks, as well as in the overnight control data and in the envA two hour dataset). As expected, OSN subtypes with significant increases/decreases in their ES scores after two weeks showed increased/decreased activation (relative to the mean) at each time point. In contrast, the opposite effects were observed for mice switched from envA back to home cages, and little activation was observed between mice switched to cages of the same environment.</div><div id="p0600" role="paragraph">The correlation between the activation scores at two hours and the two-week ES score shifts was evaluated using all OSN subtypes, and the two-week ES score shifts were also compared to the change in the ES score after 24 hours. Predictions of the sign of the two-week ES score based on the activation score at each time point was performed using 10-fold cross-validated linear SVMs (fit at the OSN subtype level using a random subset of 80 of the OSN subtypes whose ES scores significantly increased and 80 that decreased after two-weeks for classification on each of 1,000 restarts). Classification accuracy was consistently above chance levels but was at chance levels upon shuffling the sign of the two-week ES score change for each restart.</div></section><section id="sec-4-4-34"><h4>Craniotomy and cranial window for chronic olfactory bulb imaging</h4><div id="p0605" role="paragraph">All surgical procedures were performed in accordance with the guidelines provided by IACUC at Harvard Medical School. Prior to surgery, 4-6&nbsp;weeks old <i>Omp-IRES-Cre;Ai95D</i> (GCaMP6f) mice, were subcutaneously injected with the analgesic Buprenorphine-SR (1mg/kg) and anesthetized 1-2 hours later with isoflurane (3% induction, 1%–2% maintenance). The skin overlying the left olfactory bulb was retracted and the underlying periosteum was scraped off with a scalpel. The skull was subsequently cleaned with saline and allowed to dry. A 2.5&nbsp;mm circular craniotomy was made using a biopsy punch (Integra Miltex) taking care not to damage the underlying dura. Gelfoam saturated with cold saline was placed over the craniotomy to reduce swelling and suppress minor bleeds. A custom cranial window was constructed using two circular coverslips (Warner Instruments, 3mm, #1 thickness). The first coverslip was ground down to 2.5&nbsp;mm in diameter using a diamond scribe and subsequently bonded to the second 3mm coverslip using Norland 71 optical epoxy. With the smaller coverslip in contact with the dura, the exposed edges of the 3mm coverslip were gently depressed until flush with the surrounding skull and secured in place with glass glue (Loctite, P.N. 233841). This “top hat” design ensured a tight seal around the craniotomy. For head-fixation, a custom titanium head plate was positioned over the skull and, using a manual manipulator, aligned to be coplanar with the cranial window. Following alignment, the head-plate was secured to the skull using dental cement (C&amp;B Metabond, Parkell) and the animal was returned to the home cage for recovery. The supplementary analgesic Carprofen (5mg/kg) was provided in drinking water for 4&nbsp;days following surgery. Throughout the 1-2&nbsp;weeks prior to chronic imaging, the craniotomy was regularly inspected for any changes in clarity. Animals with excessive overgrowth of dura and skull tissue were not imaged and were removed from the study. In order to keep the cranial window clear of debris between imaging sessions, the dorsal surface of the skull was covered with silicone sealant (Kwik-Cast, World Precision Instruments).</div></section><section id="sec-4-4-35"><h4>Odor delivery</h4><div id="p0610" role="paragraph">A 23-valve olfactometer was used to present odors, as previously described (Island Motion Systems, see <span class="dropBlock reference-citations"><a id="body-ref-sref77-1" href="#" role="doc-biblioref" data-xml-rid="bib77" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref77-1" aria-controls="sref77-1">Pashkovski et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref77-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">79.</div><div>Pashkovski, S.L. ∙ Iurilli, G. ∙ Brann, D. ...</div><div><strong>Structure and flexibility in cortical representations of odour space</strong></div><div><em>Nature.</em> 2020; <strong>583</strong>:253-258</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-020-2451-1" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_79_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-85087290240" target="_blank" aria-label="Open Scopus in new tab">Scopus (24)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32612230/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-020-2451-1&amp;pmid=32612230" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). In brief, custom Arduino software was used to control valve opening and closing, thereby enabling switching between odor vials and the blank vial. This software also controlled the output of two mass flow controllers (MFC). The first MFC delivered a constant carrier flow at 0.75&nbsp;L min<sup>−1</sup> of purified air into a common channel; the second MFC supplied a constant flow at 0.25&nbsp;L min<sup>−1</sup> of clean air that was injected into an odor vial (see below) and then merged with the carrier flow 1 inch (2.54&nbsp;cm) in front of the mouse’s nose. A larger exhaust fan drew air from the cage enclosing the imaging rig to prevent any cross-contamination between odors. Monomolecular odors were diluted in Mineral oil (Sigma), and this vapor-phase concentration was further diluted 1:4 by the carrier airflow. Odor presentations lasted for two seconds and were interleaved by 30&nbsp;s of blank (Mineral oil) delivery. The order of presentation of odors was pseudo-randomized for each experiment, such that on any given trial, odors were presented once in no predictable order.</div></section><section id="sec-4-4-36"><h4><i>In&nbsp;vivo</i> olfactory bulb imaging</h4><div id="p0615" role="paragraph">Chronic imaging of the dorsal surface of the olfactory bulb was performed after recovery from the surgery (1–2&nbsp;weeks) under ketamine (100mg/kg) and xylazine (10&nbsp;mg/kg) induced anesthesia, to specifically isolate responses in OSN axons with less modulation from local interneurons and long-range cortical feedback. Depth of anesthesia was assessed every 15&nbsp;minutes via the toe-pinch reflex and intraperitoneal ketamine was administered at 1/3 concentration of the induction dose for anesthesia maintenance. Multiphoton imaging was performed at 60&nbsp;Hz using a 16 kHz resonant galvo scanner (Cambridge Technologies) with a chameleon laser tuned to 920&nbsp;nm delivered to 30-50 mW of excited power at the front end of the objective (10X Olympus XLUMPLFL10XW-SP lens, NA 0.6), as previously described (<span class="dropBlock reference-citations"><a id="body-ref-sref77-2" href="#" role="doc-biblioref" data-xml-rid="bib77" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref77-2" aria-controls="sref77-2">Pashkovski et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref77-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">79.</div><div>Pashkovski, S.L. ∙ Iurilli, G. ∙ Brann, D. ...</div><div><strong>Structure and flexibility in cortical representations of odour space</strong></div><div><em>Nature.</em> 2020; <strong>583</strong>:253-258</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-020-2451-1" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_79_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-85087290240" target="_blank" aria-label="Open Scopus in new tab">Scopus (24)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32612230/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-020-2451-1&amp;pmid=32612230" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Emitted fluorescence was detected using Hamamatsu H10770PA-40 PMTs. Scanimage 5 was used for hardware control and image data acquisition; WaveSurfer was used for recording of acquisition, odor and frame triggers, as well as sniffing data in a subset of mice. To obtain consistent field of views across sessions, 0.5&nbsp;mg of Texas-Red conjugated Dextran (3000 MW, D3328 Thermo Fisher) dye was injected subcutaneously, and the field of view was aligned using both red and green channels based on patterns of vasculature and glomeruli, respectively (using the first session as a template). Animals were monitored throughout each imaging session for any drift in the field of view across trials.</div></section><section id="sec-4-4-37"><h4>Experimental designs for olfactory bulb imaging across environment shifts</h4><div id="p0620" role="paragraph">Initial experiments consisted of a cohort of four mice imaged across multiple environment shifts. Each imaging session consisted of at least 8 trials, in which each of the 7 odors were presented. The odors used included 3 odors that activate broad sets of the dorsal glomeruli and which were presented at both a high/low concentration (pentanone [25/2.5%], ethyl butyrate [2.5/0.25%], and pentanal [25/2.5%]) as well as a 7th blank odorant (Mineral oil). When mice were not being imaged, they were housed in either regular home-cage or Environment A (envA) cages, consisting of the same contents as used for the scRNA-seq experiments. Throughout the course of the experiment, mice were sequentially housed in the regular home cages (home1), envA cages (envA), and then regular home cages again (home2). 4–6 sessions were obtained for home1, 4–6 sessions were obtained for envA (across 14&nbsp;days of housing in envA), and 2–3 sessions were obtained for home2 (across an additional 14&nbsp;days of housing in the regular home-cage environments).</div><div id="p0625" role="paragraph">In an additional cohort of two mice, the responses to a broader panel of 16 odors (to facilitate decoding analyses) that densely activated the dorsal olfactory bulb were imaged. Each session consisted of at least 6 trials, in which a blank odor and each of the 16 odors (2.5% of pentanal, pentanone, ethyl butyrate, heptanal, hexanal, propanal, butanal, butanone, hexanone, ethyl acetate, propyl acetate, butyl acetate, benzaldehyde, tiglaldehyde, methyl tiglate, methyl valerate, all of which were obtained from Sigma except for methyl tiglate, which was obtained from Thermo Fisher) were presented pseudo-randomly. Throughout the course of the experiment, mice were sequentially housed in the regular home cages (home-cage) and envA cages (envA).</div></section><section id="sec-4-4-38"><h4>Registration, ROI detection, and signal extraction for olfactory bulb imaging data</h4><div id="p0630" role="paragraph">For alignment of the FOV across sessions, baseline-subtracted images were downsampled (from 60&nbsp;Hz to 10&nbsp;Hz) and the mean image from each session was used to register the field of view across imaging sessions. Registration across sessions was performed using an iterative approach to align each session to a common template session. First, the optimal geometric transformation that maximized the overall cross correlation between mean images was identified using OpenCV (findTransformECC with warp_mode&nbsp;= MOTION_HOMOGRAPHY and gaussFiltSize&nbsp;= 1). Next, local deformations were further aligned with Dipy (SymmetricDiffeomorphicRegistration with metric&nbsp;= CCMetric, level_iters&nbsp;= [250, 100, 50, 25], and inv_iter&nbsp;= 50). The resulting procedure gave an invertible warping for each session that was subsequently applied to all frames from that session. Throughout a session, the field of view was stable, and no motion correction was applied; therefore, after registration, glomeruli were stably aligned across all trials and sessions. The z-depth of the image plane was aligned for every imaging session based on the specific pattern of vasculature and glomeruli found in each mouse, and this single plane was imaged throughout each session. <a id="crosref1550" href="#figs7">Figure&nbsp;S7</a>B shows that FOVs are stable across sessions and environments, as assessed by the pairwise enhanced correlation coefficient between mean images of the entire FOVs (computeECC function in OpenCV). Regions of interest (ROIs, glomeruli) were identified via Suite2p (skipping the registration and deconvolution steps) from the entire set of registered baseline-subtracted and down-sampled images from all sessions (<span class="dropBlock reference-citations"><a id="body-ref-sref74-1" href="#" role="doc-biblioref" data-xml-rid="bib74" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref74-1" aria-controls="sref74-1">Pachitariu et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref74-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">76.</div><div>Pachitariu, M. ∙ Stringer, C. ∙ Dipoppa, M. ...</div><div><strong>Suite2p: Beyond 10,000 neurons with standard two-photon microscopy</strong></div><div><em>bioRxiv.</em> 2017;</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.PachitariuC.StringerM.DipoppaS.Schr%C3%B6derL.F.RossiH.DalgleishM.CarandiniK.D.HarrisSuite2p%3A+Beyond+10%2C000+neurons+with+standard+two-photon+microscopybioRxiv2017" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). ROIs that covered multiple glomeruli or masks glomeruli split into multiple ROIs were manually corrected. Using the set of identified ROIs, fluorescence traces were manually extracted from the raw movies from each session by applying the session-specific warping described above to each movie and then taking the dot product of the fluorescence in the aligned movies with the pixel weights of each ROI mask. Neuropil subtraction was not performed since annuli from each glomerulus would overlap with other glomeruli and there was little background fluorescence outside of activated glomeruli.</div></section><section id="sec-4-4-39"><h4>Analysis of GCaMP signals</h4><section id="sec-4-4-39-1"><h5>Normalization of GCaMP signals</h5><div id="p0635" role="paragraph">For each trial and glomerulus, GCaMP signals were baseline-subtracted and normalized relative to their baseline (dF/F). The 10<sup>th</sup> percentile of the ten seconds prior to each odor presentation was considered as the baseline. dF/F values were interpolated to 50&nbsp;Hz and smoothed by convolving with a Gaussian filter (length&nbsp;= 21, standard deviation&nbsp;= 5). Because of the long duration of the initial cohort, in which mice were imaged across multiple environment shifts over more than a month, dF/F traces were z-scored for each trial; dF/F values were not z-scored for the later cohort, which was imaged for a shorter duration across a single environment shift.</div></section><section id="sec-4-4-39-2"><h5>Identifying odor-responsive glomeruli</h5><div id="p0640" role="paragraph">Responsive glomeruli were identified separately for each environment. Responsive glomeruli were defined as those whose mean response from 0.0 to 3.0&nbsp;s after odor valve onset was significantly higher than that of the prior 10&nbsp;s (p ≤ 1&nbsp;× 10<sup>−3</sup>, via the Wilcoxon signed-rank test and adjusted via the Holm–Bonferroni method across odors for each glomerulus) and whose mean z-scored dF/F was greater than 0.5 (or mean dF/F was greater than 0.125 for the second cohort).</div></section><section id="sec-4-4-39-3"><h5>Aligning responses to the population mean</h5><div id="p0645" role="paragraph">Responses were aligned across trials using the population mean to account for the variability in response onset present because odors were given in open loop without controlling for sniffing. The set of glomeruli that responded in at least two environments with a positive mean odor response were used for aligning odor responses across trials. The population mean was computed across these responsive glomeruli for each trial and the population onset was defined as the first time point in which the population exceeded a threshold (0.75 for z-scored dF/F data and 0.25 for dF/F); traces were aligned across trials by shifting them based on the identified population onset. This alignment procedure increased the correlation of the odor response across trials for each odor, suggesting that responses are more reliable once aligned to population, rather than odor, onset. Sniffing was also recorded with a pressure sensor (Honeywell AWM3100V) in a subset of mice to further assess the performance of this alignment procedure. Sniffing signals were uncorrelated across trials when using the odor valve times but were correlated across trials when using the shifts identified via the population onset. After alignment, the population response reliably increased following inspiration, further suggesting that this alignment procedure helps to correct for variability in timing of odor responses across trials due to sniffing. After alignment relative to the population mean, responsive glomeruli were re-identified used the same thresholds and procedure described above, except now using the first 3&nbsp;s after population onset, and responsive glomeruli kept for downstream analyses were those that were identified as responsive in at least two environments, and whose response to a given odor was a least twice that of the response to the blank.</div></section><section id="sec-4-4-39-4"><h5>Identifying environment-sensitive glomeruli</h5><div id="p0650" role="paragraph">For the responsive glomeruli for each odor, the mean response amplitudes for each trial (the mean dF/F or z-scored dF/F values in the first 3&nbsp;s following population onset) were averaged across trials within each environment. Glomeruli whose odor response amplitude differed across environment pairs were identified via permutation testing. The observed differences in means across environments was compared to a null distribution calculated by following the same procedure after permuting environment labels across trials 100,000 times. P values were calculated empirically for each glomerulus-odor pair and were adjusted across pairs by the Benjamini-Hochberg FDR procedure. Pairs whose FDR-corrected p value was ≤ 0.01 were considered as significantly different across environments. Across mice and odors, 20%–80% of responsive glomeruli demonstrated significant shifts in response amplitudes across the shift from home-cage to envA. In contrast, fewer glomerular-odor pairs were significantly different between home1 and home2. Glomerular responses were considered as reverted if the response amplitudes in home2 were more similar to home1 than envA. While one would naively expect reversions via regression to the mean if data were independent, no glomeruli were identified as environmentally sensitive when environmental labels were shuffled before calculating the observed difference, indicating that the observed differences in response amplitudes between home1 and envA (as well as those between envA and home2) are not the results of spurious partitioning of responses. Together, these results suggest that a large fraction of glomeruli responsive to a given odor are stable within environments but vary across environments.</div></section><section id="sec-4-4-39-5"><h5>Population level analysis</h5><div id="p0655" role="paragraph">Population odor responses were analyzed using pseudo-populations of glomeruli imaged across mice. Pentanal elicited the broadest responses across the population of glomeruli and was therefore used to assess how the population response differed between environments. To test whether differences at the population level in the pentanal responses in envA compared to home1 were reverted in home2, an “environmental axis” that separated the home1 and envA responses was constructed. For each mouse, two testing sessions were held out for both home1 and envA, and the remaining sessions were used for training, and the response at each time point in these training sessions was averaged across all training trials. PCA was fit on the population responses in the training data to reduce the dimensionality of the glomerular population to 10 dimensions, and the mean response was calculated in 30 overlapping windows in time (each 2&nbsp;s long starting from −0.2&nbsp;s and each subsequent one offset by 0.05&nbsp;s from odor onset). The differences across the 10 PCs between the response in envA and home1 was calculated for each window, and the weights were averaged across the 30 windows and then normalized by their L-2 norm. This procedure thus returns the projection vector designed to maximally separate the odor responses in the two environments. To test whether this projection generalized to held-out sessions, the PCA transformation was applied to the mean response in the held-out sessions for home1 and envA, as well as to the home2 sessions. The difference between the population responses along this environment axis between the held-out home1 and envA session sessions was compared to the difference between the home2 and held-out home1 sessions. This cross-validation procedure was repeated across 1,000 restarts, and the mean and standard deviations across restarts were evaluated. Across all restarts, the response in envA readily diverged after odor onset, whereas the response in home2 remained similar to that in home1 at all time points after odor onset.</div></section><section id="sec-4-4-39-6"><h5>Pairwise odor correlations within and across environments</h5><div id="p0660" role="paragraph">Pairwise correlations for each odor pair were calculated for each environment using the population vectors of mean odor responses (averaged across the first 3&nbsp;s following population onset). For each odor pair, the union of responsive glomeruli was used. The pairwise correlation distance between all trials for each odor was computed and the median across trial pairs was computed for each odor pair. This same procedure also applied to pairs of the same odor, and the on-diagonal of the resulting correlation matrix represents the median across trials pairs and demonstrates the stability of the population response to the same odor across trials. The difference in correlation matrices across environments was computed, and the observed difference in the mean absolute change in pairwise correlations of the lower triangle (excluding the diagonal) was compared to a null distribution of means from shuffling the environment identity of each trial across 10,000 shuffles. For both mice, the observed difference was greater than that observed in any shuffle. Correlation matrices are shown after hierarchical clustering odors using Ward’s method.</div></section></section><section id="sec-4-4-40"><h4>Classifying environment and odor identities via glomerular populations</h4><section id="sec-4-4-40-1"><h5>Environment identity classification</h5><div id="p0665" role="paragraph">Classification of environment identity was performed for each odor separately using the responsive glomeruli for each odor. Classification was performed using pseudo-populations of glomeruli combined across mice, using the mean response for each glomerulus across the first 3&nbsp;s following population onset. For each of 100 restarts, 18 trials were randomly chosen for each mouse for each environment and were split into 12 training and 6 test trials. Glomerular responses were permuted across trials within each mouse and then concatenated across mice to generate pseudo-trials for model training and testing. On each restart, subpopulations of glomeruli were randomly selected 100 times, thus yielding 10,000 models. Standardization across trials for each glomerulus via z-scoring and linear SVM classifiers (regularization parameter <i>C</i>&nbsp;= 0.1) were both fit on the training data and applied to predict the environment label of the held-out test pseudo-trials, and the accuracy was summarized for each subpopulation across restarts. Performance was a chance levels upon shuffling the environment labels of the training and test data.</div></section><section id="sec-4-4-40-2"><h5>Odor identity classification and classification generalization</h5><div id="p0670" role="paragraph">Odor identity was predicted in a similar manner, using the mean responses across the first 3&nbsp;s following population onset in pseudo-populations of glomeruli. For each of 100 restarts, for each environment, 6 test trials (containing the responses to each of the 16 odors) were randomly held out, and the glomeruli were randomly permuted across test trials 10 times to generate pseudo-populations for 60 test pseudo-trials for each of the 16 odors. The remaining training trials were randomly subsampled from each environment to yield 24 training trials for each odor. On each restart, subpopulations of glomeruli were randomly selected 20 times, thus yielding 2,000 (20&nbsp;× 100) models for each threshold. A classification pipeline was fit on the training data, consisting of standardization across trials for each glomerulus via z-scoring, reducing the dimensionality via PCA (n_components&nbsp;= 5 when the number of glomeruli was&nbsp;&lt; 10, else n_components&nbsp;= 10), and fitting linear SVM classifiers (“one-vs-one” configuration with regularization parameter <i>C</i>&nbsp;= 0.5), and then applied to the test data. The training data from each environment and the test data from held-out trials from each environment were each used for training and testing, thus generating all four combinations (2&nbsp;× 2) of training and testing data from either the same or different environments. Classification performance was summarized across restarts for each threshold for each of the within-environment and between-environment training/testing combinations and compared to 1) performance when environment labels were shuffled across the training trials and across the test trials from each environment and 2) performance when the odor labels were shuffled. Classification performance quickly saturated with increasing number of glomeruli, was higher for held-out trials from the same environment, and was at chance performance when odor labels were shuffled.</div><div id="p0675" role="paragraph">Odor generalization was performed to determine how the changes in pairwise odor correlations observed across environments impacted odor coding. The same pseudo-populations were constructed for training and test trials from each environment as described above for odor decoding. The entire pseudo-population of glomeruli was used without subsampling. The training and test data was first transformed with the z-scoring and PCA transformation (n_components&nbsp;= 10) described above and fit on the training data. However, rather than fitting a single classifier to all trials from all odors, a separate classifier was fit on each odor pair in the training data (<sub>16</sub>C<sub>2</sub>&nbsp;= 120 pairs) and applied to the remaining odor pairs (119) in the test data, thus yielding 120&nbsp;× 119&nbsp;= 14,280 predictions. This same procedure was repeated for each of the four within- and between-environment training/testing combinations, and the entire procedure was repeated when the environment labels were shuffled across the training trials and across the test trials from each environment. Classification accuracy from classifiers trained and tested on different odor pairs from the same environment (within-environment) was sorted across the 14,280 pairs and the accuracy for each pair in the within-environment condition was compared to that of the between-environment and shuffled-environment conditions. Although the overall distribution of classification accuracies was similar for each of the three conditions, the mean absolute difference between within-environment and between-environment accuracies for each pair was greater for the observed, rather than the shuffled environmental labels, further demonstrating that pairwise relationships between odors are altered across environments.</div></section></section><section id="sec-4-4-41"><h4>Software used for data analysis</h4><div id="p0680" role="paragraph">Single-cell RNA-seq analyses were performed in python (versions 3.6-3.8) using the Scanpy package (<span class="dropBlock reference-citations"><a id="body-ref-sref110-3" href="#" role="doc-biblioref" data-xml-rid="bib110" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref110-3" aria-controls="sref110-3">Wolf et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref110-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">113.</div><div>Wolf, F.A. ∙ Angerer, P. ∙ Theis, F.J.</div><div><strong>SCANPY: Large-scale single-cell gene expression data analysis</strong></div><div><em>Genome Biol.</em> 2018; <strong>19</strong>:15</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13059-017-1382-0" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_113_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-85041394976" target="_blank" aria-label="Open Scopus in new tab">Scopus (758)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29409532/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13059-017-1382-0&amp;pmid=29409532" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) as well as custom-written scripts using the open-source python scientific stack (pysam, SciPy, NumPy, scikit-learn, umap-learn, pandas, statsmodels, matplotlib, seaborn, Numba, Dask). cNMF was performed using modified versions of the code described in (<span class="dropBlock reference-citations"><a id="body-ref-sref50-4" href="#" role="doc-biblioref" data-xml-rid="bib50" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref50-4" aria-controls="sref50-4">Kotliar et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref50-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">52.</div><div>Kotliar, D. ∙ Veres, A. ∙ Nagy, M.A. ...</div><div><strong>Identifying gene expression programs of cell-type identity and cellular activity with single-cell RNA-seq</strong></div><div><em>eLife.</em> 2019; <strong>8</strong>:e43803</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.7554/eLife.43803" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-85070056080" target="_blank" aria-label="Open Scopus in new tab">Scopus (43)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31282856/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.7554%2FeLife.43803&amp;pmid=31282856" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and accessed at <a id="intref0080" href="https://github.com/dylkot/cNMF" target="_blank">https://github.com/dylkot/cNMF</a>. Imaging data was analyzed using additional packages, including OpenCV and Suite2p (<span class="dropBlock reference-citations"><a id="body-ref-sref74-2" href="#" role="doc-biblioref" data-xml-rid="bib74" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref74-2" aria-controls="sref74-2">Pachitariu et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref74-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">76.</div><div>Pachitariu, M. ∙ Stringer, C. ∙ Dipoppa, M. ...</div><div><strong>Suite2p: Beyond 10,000 neurons with standard two-photon microscopy</strong></div><div><em>bioRxiv.</em> 2017;</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.PachitariuC.StringerM.DipoppaS.Schr%C3%B6derL.F.RossiH.DalgleishM.CarandiniK.D.HarrisSuite2p%3A+Beyond+10%2C000+neurons+with+standard+two-photon+microscopybioRxiv2017" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Scripts to replicate data analyses are available at <a id="intref0085" href="https://github.com/dattalab/Tsukahara_Brann_OSN" target="_blank">https://github.com/dattalab/Tsukahara_Brann_OSN</a>.</div></section></section><section id="sec-4-5"><h3>Quantification and statistical analysis</h3><div id="p0685" role="paragraph">Hypothesis testing was performed using non-parametric statistical tests, including the Mann-Whitney U Test and the Wilcoxon signed-rank test, except in the cases where p values were calculated empirically using resampling-based permutation tests. Distributions were compared via the Kolmogorov–Smirnov (KS) test. Experiments with multiple factors were tested with either a two-way ANOVA or with the Jonckheere-Terpstra trend test (calculated empirically via 10,000 permutations), a non-parametric extension of the Kruskal–Wallis test to evaluate trends across groups. Statistical tests that were performed for multiple genes, ORs, GEPs, glomeruli, and odors were corrected for multiple comparisons using the Benjamini-Hochberg FDR Procedure or the Holm–Bonferroni method, as specified in the text. The results of statistical tests are listed in the figure legends. The precision of sample statistics and regression trend lines were evaluated using bootstrapping, and, except where noted, plots and error bars depict the mean and the 95% confidence intervals of the mean across 1,000-10,000 bootstraps. Results from classification, regression, and other analyses were repeated across 1,000 restarts, using balanced subsamples of the data in analyses at the OSN-level to account for differences in OR frequencies. Throughout the paper, a non-parametric version of the boxplot (also known as a letter-value plot) was used to represent multiple quantiles and tails of large distributions of data (e.g., to summarize across OSN subtypes or sets of ORs or OR pairs) in an agnostic way that does require setting bandwidth parameters as in violin plots or kernel density estimates. Like a conventional boxplot, the largest box represents the interquartile range (25–75 percentile) and the median (dotted-line). Subsequent boxes recursively represent exponentially-smaller quantiles (the 12.5–25 and 75–87.5 percentiles, then 6.25–12.5 and 87.5–93.75 percentiles, then 3.125-6.25 and 93.75–96.875 percentiles, and so forth).</div></section></section><section id="acknowledgments" role="doc-acknowledgments"><h2>Acknowledgments</h2><div id="p0690" role="paragraph">We thank John McGann and Matthew Grubb for technical advice on naris occlusion; Venkatesh Murthy, Dima Rinberg, and Catherine Dulac for mice; and Kelsey Tyssowski, Vanessa Ruta, Rachel Wilson, Bernardo Sabatini, David Ginty, Michael Greenberg, Piali Sengupta, and the Datta lab for useful comments on the manuscript. We thank Neha Bhagat for laboratory assistance and Janet Wallace for administrative assistance. We thank the Bauer Core Facility at Harvard University for sequencing and the Flow Cytometry Core at Beth Israel Deaconess Medical Center. Portions of this research were conducted on the O2 High Performance Compute Cluster at Harvard Medical School. Illustrations were created in part with <a id="intref0090" href="http://BioRender.com" target="_blank">BioRender.com</a> and <a id="intref0095" href="http://Vecteezy.com" target="_blank">Vecteezy.com</a>. S.R.D. is supported by NIH grants R011DC016222 and U19 NS112953 and by grants from the Simons Collaboration on the Global Brain, the Brain Research Foundation, and the Tan Yang Center at Harvard Medical School. T.T. is supported by postdoctoral fellowships from the Japan Society for the Promotion of Science, the Louis Perry Jones Fund, and the Tan Yang Center. D.H.B. is supported by an NSF Graduate Research Fellowship and by NIH grant F31DC019017.</div><section id="sec-5"><h3>Author contributions</h3><div id="p0695" role="paragraph">T.T., D.H.B., and S.R.D. designed the study and interpreted the data. T.T. and D.H.B. performed and analyzed scRNA-seq experiments; T.T., D.H.B., and G.G. performed optogenetic stimulation experiments; and T.T., D.H.B., and S.L.P. performed and analyzed functional imaging experiments. T.B. provided resources and assisted in experimental design and interpretation. T.T., D.H.B., and S.R.D. wrote the manuscript with input from all authors.</div></section><section id="sec-6"><h3>Declaration of interests</h3><div id="p0700" role="paragraph">The authors declare no competing interests.</div></section></section><section id="supplementary-material" data-type="supplementary-materials"><h2>Supplemental information (5)</h2><div id="p0710" role="paragraph"><div class="core-supplementary-material"><div class="core-link"><button class="downloadAllSupplementary"><span>Download all</span></button></div></div><div id="mmc1" class="core-supplementary-material" data-category="file"><div class="core-link"><a href="/cms/10.1016/j.cell.2021.11.022/attachment/1bf6643c-5639-4ea6-a15b-ab9397372a25/mmc1.csv" download="mmc1.csv" aria-labelledby="mmc1-heading mmc1-link"><span id="mmc1-link">File (11.80 KB)</span></a></div><div class="core-description"><div id="mmc1-heading" class="heading">Table S1. Summary of single-cell RNA-seq datasets for olfactory sensory neurons used in this study, related to Figures 1, 2, 3, 4, 5, and 6</div></div></div><div id="mmc2" class="core-supplementary-material" data-category="file"><div class="core-link"><a href="/cms/10.1016/j.cell.2021.11.022/attachment/7347e101-4d48-4870-b3a0-5155460b91ad/mmc2.csv" download="mmc2.csv" aria-labelledby="mmc2-heading mmc2-link"><span id="mmc2-link">File (325.13 KB)</span></a></div><div class="core-description"><div id="mmc2-heading" class="heading">Table S2. Loadings of 1,350 highly variable genes for 16 gene expression programs identified by consensus non-negative matrix factorization of the home cage dataset, related to Figure&nbsp;2</div></div></div><div id="mmc3" class="core-supplementary-material" data-category="file"><div class="core-link"><a href="/cms/10.1016/j.cell.2021.11.022/attachment/268c326a-9741-4a55-9092-151eae4468c6/mmc3.csv" download="mmc3.csv" aria-labelledby="mmc3-heading mmc3-link"><span id="mmc3-link">File (4.98 KB)</span></a></div><div class="core-description"><div id="mmc3-heading" class="heading">Table S3. 117 environmental state neuronal genes, including 73 functional genes that may regulate sensory responses, related to Figure&nbsp;4</div></div></div><div id="mmc4" class="core-supplementary-material" data-category="file"><div class="core-link"><a href="/cms/10.1016/j.cell.2021.11.022/attachment/b0625282-5363-4fa7-b60c-52a6d021aa1e/mmc4.csv" download="mmc4.csv" aria-labelledby="mmc4-heading mmc4-link"><span id="mmc4-link">File (2.18 MB)</span></a></div><div class="core-description"><div id="mmc4-heading" class="heading">Table S4. Changes in <i>Z</i>-scored expression of 117 environmental state neuronal genes across conditions, for each OSN subtype, related to Figure&nbsp;4</div></div></div><div id="mmc5" class="core-supplementary-material" data-category="file"><div class="core-link"><a href="/cms/10.1016/j.cell.2021.11.022/attachment/7b2c1591-af79-4def-8266-56b347729f67/mmc5.csv" download="mmc5.csv" aria-labelledby="mmc5-heading mmc5-link"><span id="mmc5-link">File (61.09 KB)</span></a></div><div class="core-description"><div id="mmc5-heading" class="heading">Table S5. 472 genes used to calculate activation score that were acutely odor-sensitive in responsive OSNs, related to Figure&nbsp;5</div><div id="espara0010" role="paragraph">Eight values characterizing the 472 genes used to calculate the activation score are shown. Columns are as follows: “activation_score_loading”: gene loading for calculating activation score. “ess_correlation”: Spearman’s rho between normalized expression and environmental state score in the home cage dataset on OR-by-OR basis. “p_adj_ACE2h”: FDR corrected p values for the differential expression (via Wilcoxon signed-rank test) between control and acetophenone 2h samples for acetophenone responsive ORs on OR-by-OR basis. “ctl_expr_ACE2h”: Normalized mean expression for OSNs expressing-acetophenone responsive ORs in the control data. “expression_ACE2h”: Normalized mean expression for OSNs expressing acetophenone responsive ORs in the acetophenone 2h data. “p_adj_OCT2h,” “ctl_expr_OCT2h,” “expression_OCT2h”: same as the previous three columns for acetophenone responsive ORs, but for OSNs expressing octanal-responsive ORs.</div></div></div></div></section></div></section><div class="core-linked-content">
    

        

</div><section id="backmatter" data-extent="backmatter"><div class="core-container"><section id="data-availability" class="core-data-availability"><h2>Data and code availability</h2><div id="p0030" role="paragraph">Single-cell RNA-seq data have been deposited at GEO and are publicly available as of the date of publication. The accession number is listed in the key resources table.</div><div id="p0035" role="paragraph">All original code has been deposited at GitHub and Zenodo, and the links and DOI are listed in the key resources table.</div><div id="p0040" role="paragraph">Any additional information required to reanalyze the data reported in this paper is available from the lead contact upon request.</div></section><section id="references"><h2>References</h2><div id="bibliography" role="doc-bibliography"><div role="list"><div role="listitem" data-has="label"><div id="bib1" class="citations"><div id="sref1" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref1" title="View in article">1.</a></div><div>Aibar, S. ∙ González-Blas, C.B. ∙ Moerman, T. ...</div><div><strong>SCENIC: Single-cell regulatory network inference and clustering</strong></div><div><em>Nat. Methods.</em> 2017; <strong>14</strong>:1083-1086</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.4463" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_1_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-85032583384" target="_blank" aria-label="Open Scopus in new tab">Scopus (656)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28991892/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.4463&amp;pmid=28991892" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib2" class="citations"><div id="sref2" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref2" title="View in article">2.</a></div><div>Arneodo, E.M. ∙ Penikis, K.B. ∙ Rabinowitz, N. ...</div><div><strong>Stimulus dependent diversity and stereotypy in the output of an olfactory functional unit</strong></div><div><em>Nat. Commun.</em> 2018; <strong>9</strong>:1347</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41467-018-03837-1" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_2_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-85045210680" target="_blank" aria-label="Open Scopus in new tab">Scopus (15)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29632302/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41467-018-03837-1&amp;pmid=29632302" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib3" class="citations"><div id="sref3" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref3" title="View in article">3.</a></div><div>Attneave, F.</div><div><strong>Some informational aspects of visual perception</strong></div><div><em>Psychol. Rev.</em> 1954; <strong>61</strong>:183-193</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1037/h0054663" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-0000728115" target="_blank" aria-label="Open Scopus in new tab">Scopus (1752)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/13167245/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1037%2Fh0054663&amp;pmid=13167245" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib4" class="citations"><div id="sref4" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref4" title="View in article">4.</a></div><div>Barber, C.N. ∙ Coppola, D.M.</div><div><strong>Compensatory plasticity in the olfactory epithelium: Age, timing, and reversibility</strong></div><div><em>J.&nbsp;Neurophysiol.</em> 2015; <strong>114</strong>:2023-2032</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1152/jn.00076.2015" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-84943175800" target="_blank" aria-label="Open Scopus in new tab">Scopus (4)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26269548/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1152%2Fjn.00076.2015&amp;pmid=26269548" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib5" class="citations"><div id="sref5" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref5" title="View in article">5.</a></div><div>Barlow, H.</div><div><strong>Possible principles underlying the transformation of sensory messages</strong></div><div><div>Rosenblith, W.A. (Editor)</div></div><div><strong>Sensory Communication</strong></div><div>MIT press, 1961; 217-234</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=H.BarlowPossible+principles+underlying+the+transformation+of+sensory+messagesW.A.RosenblithSensory+Communication1961MIT+press217234" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib6" class="citations"><div id="sref6" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref6-1" title="View in article">6.</a></div><div>Benda, J.</div><div><strong>Neural adaptation</strong></div><div><em>Curr. Biol.</em> 2021; <strong>31</strong>:R110-R116</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=10.1016%2Fj.cub.2020.11.054&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=10.1016%2Fj.cub.2020.11.054&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-85100374082" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33561404/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cub.2020.11.054&amp;pmid=33561404" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib7" class="citations"><div id="sref7" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref7" title="View in article">7.</a></div><div>Best, A.R. ∙ Thompson, J.V. ∙ Fletcher, M.L. ...</div><div><strong>Cortical metabotropic glutamate receptors contribute to habituation of a simple odor-evoked behavior</strong></div><div><em>J.&nbsp;Neurosci.</em> 2005; <strong>25</strong>:2513-2517</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1523/JNEUROSCI.5298-04.2005" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-14944364759" target="_blank" aria-label="Open Scopus in new tab">Scopus (48)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15758159/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1523%2FJNEUROSCI.5298-04.2005&amp;pmid=15758159" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib8" class="citations"><div id="sref8" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref8" title="View in article">8.</a></div><div>Bozza, T. ∙ Feinstein, P. ∙ Zheng, C. ...</div><div><strong>Odorant receptor expression defines functional units in the mouse olfactory system</strong></div><div><em>J.&nbsp;Neurosci.</em> 2002; <strong>22</strong>:3033-3043</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1523/JNEUROSCI.22-08-03033.2002" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11943806/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1523%2FJNEUROSCI.22-08-03033.2002&amp;pmid=11943806" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib9" class="citations"><div id="sref9" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref9" title="View in article">9.</a></div><div>Bozza, T. ∙ McGann, J.P. ∙ Mombaerts, P. ...</div><div><strong>In&nbsp;vivo imaging of neuronal activity by targeted expression of a genetically encoded probe in the mouse</strong></div><div><em>Neuron.</em> 2004; <strong>42</strong>:9-21</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=10.1016%2FS0896-6273%2804%2900144-8&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=10.1016%2FS0896-6273%2804%2900144-8&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-1842505551" target="_blank" aria-label="Open Scopus in new tab">Scopus (265)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15066261/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0896-6273%2804%2900144-8&amp;pmid=15066261" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib10" class="citations"><div id="sref10" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref10-1" title="View in article">10.</a></div><div>Bozza, T. ∙ Vassalli, A. ∙ Fuss, S. ...</div><div><strong>Mapping of class I and class II odorant receptors to glomerular domains by two distinct types of olfactory sensory neurons in the mouse</strong></div><div><em>Neuron.</em> 2009; <strong>61</strong>:220-233</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=10.1016%2Fj.neuron.2008.11.010&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=10.1016%2Fj.neuron.2008.11.010&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-58549092612" target="_blank" aria-label="Open Scopus in new tab">Scopus (135)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19186165/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.neuron.2008.11.010&amp;pmid=19186165" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib11" class="citations"><div id="sref11" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref11" title="View in article">11.</a></div><div>Brann, D.H. ∙ Datta, S.R.</div><div><strong>Finding the brain in the nose</strong></div><div><em>Annu. Rev. Neurosci.</em> 2020; <strong>43</strong>:277-295</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1146/annurev-neuro-102119-103452" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-85087696971" target="_blank" aria-label="Open Scopus in new tab">Scopus (3)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32640927/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1146%2Fannurev-neuro-102119-103452&amp;pmid=32640927" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib12" class="citations"><div id="sref12" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref12" title="View in article">12.</a></div><div>Buck, L.B.</div><div><strong>Information coding in the vertebrate olfactory system</strong></div><div><em>Annu. Rev. Neurosci.</em> 1996; <strong>19</strong>:517-544</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1146/annurev.ne.19.030196.002505" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/8833453/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1146%2Fannurev.ne.19.030196.002505&amp;pmid=8833453" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib13" class="citations"><div id="sref13" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref13" title="View in article">13.</a></div><div>Burns, M.E. ∙ Baylor, D.A.</div><div><strong>Activation, deactivation, and adaptation in vertebrate photoreceptor cells</strong></div><div><em>Annu. Rev. Neurosci.</em> 2001; <strong>24</strong>:779-805</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1146/annurev.neuro.24.1.779" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_13_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-0034917522" target="_blank" aria-label="Open Scopus in new tab">Scopus (317)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11520918/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1146%2Fannurev.neuro.24.1.779&amp;pmid=11520918" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib14" class="citations"><div id="sref14" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref14" title="View in article">14.</a></div><div>Cadiou, H. ∙ Aoudé, I. ∙ Tazir, B. ...</div><div><strong>Postnatal odorant exposure induces peripheral olfactory plasticity at the cellular level</strong></div><div><em>J.&nbsp;Neurosci.</em> 2014; <strong>34</strong>:4857-4870</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1523/JNEUROSCI.0688-13.2014" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-84899413440" target="_blank" aria-label="Open Scopus in new tab">Scopus (43)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24695705/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1523%2FJNEUROSCI.0688-13.2014&amp;pmid=24695705" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib15" class="citations"><div id="sref15" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref15" title="View in article">15.</a></div><div>Cho, C.E. ∙ Brueggemann, C. ∙ L’Etoile, N.D. ...</div><div><strong>Parallel encoding of sensory history and behavioral preference during <i>Caenorhabditis elegans</i> olfactory learning</strong></div><div><em>eLife.</em> 2016; <strong>5</strong>:e14000</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.7554/eLife.14000" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-84979501960" target="_blank" aria-label="Open Scopus in new tab">Scopus (26)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27383131/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.7554%2FeLife.14000&amp;pmid=27383131" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib16" class="citations"><div id="sref16" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref16" title="View in article">16.</a></div><div>Chong, E. ∙ Moroni, M. ∙ Wilson, C. ...</div><div><strong>Manipulating synthetic optogenetic odors reveals the coding logic of olfactory perception</strong></div><div><em>Science.</em> 2020; <strong>368</strong>:eaba2357</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aba2357" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-85086692136" target="_blank" aria-label="Open Scopus in new tab">Scopus (17)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32554567/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aba2357&amp;pmid=32554567" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib17" class="citations"><div id="sref17" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref17" title="View in article">17.</a></div><div>Chu, M.W. ∙ Li, W.L. ∙ Komiyama, T.</div><div><strong>Lack of pattern separation in sensory inputs to the olfactory bulb during perceptual learning</strong></div><div><em>eNeuro.</em> 2017; <strong>4</strong></div><div>ENEURO.0287-17.2017</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1523/ENEURO.0287-17.2017" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-85032145385" target="_blank" aria-label="Open Scopus in new tab">Scopus (6)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28955724/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1523%2FENEURO.0287-17.2017&amp;pmid=28955724" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib18" class="citations"><div id="sref18" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref18-1" title="View in article">18.</a></div><div>Cleland, T.A. ∙ Linster, C.</div><div><strong>Computation in the olfactory system</strong></div><div><em>Chem. Senses.</em> 2005; <strong>30</strong>:801-813</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/chemse/bji072" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-27944461858" target="_blank" aria-label="Open Scopus in new tab">Scopus (101)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16267161/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fchemse%2Fbji072&amp;pmid=16267161" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib19" class="citations"><div id="sref19" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref19" title="View in article">19.</a></div><div>Coppola, D.M. ∙ Waggener, C.T.</div><div><strong>The effects of unilateral naris occlusion on gene expression profiles in mouse olfactory mucosa</strong></div><div><em>J.&nbsp;Mol. Neurosci.</em> 2012; <strong>47</strong>:604-618</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/s12031-011-9690-4" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-84863876802" target="_blank" aria-label="Open Scopus in new tab">Scopus (25)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22187364/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2Fs12031-011-9690-4&amp;pmid=22187364" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib20" class="citations"><div id="sref20" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref20" title="View in article">20.</a></div><div>Coppola, D.M. ∙ Waguespack, A.M. ∙ Reems, M.R. ...</div><div><strong>Naris occlusion alters transductory protein immunoreactivity in olfactory epithelium</strong></div><div><em>Histol. Histopathol.</em> 2006; <strong>21</strong>:487-501</div></div><div class="external-links"><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16493579/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?pmid=16493579" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib21" class="citations"><div id="sref21" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref21-1" title="View in article">21.</a></div><div>Davis, G.W.</div><div><strong>Homeostatic control of neural activity: From phenomenology to molecular design</strong></div><div><em>Annu. Rev. Neurosci.</em> 2006; <strong>29</strong>:307-323</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1146/annurev.neuro.28.061604.135751" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-33745734146" target="_blank" aria-label="Open Scopus in new tab">Scopus (400)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16776588/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1146%2Fannurev.neuro.28.061604.135751&amp;pmid=16776588" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib22" class="citations"><div id="sref22" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref22-1" title="View in article">22.</a></div><div>Davis, G.W.</div><div><strong>Homeostatic signaling and the stabilization of neural function</strong></div><div><em>Neuron.</em> 2013; <strong>80</strong>:718-728</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=10.1016%2Fj.neuron.2013.09.044&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=10.1016%2Fj.neuron.2013.09.044&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-84887011923" target="_blank" aria-label="Open Scopus in new tab">Scopus (165)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24183022/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.neuron.2013.09.044&amp;pmid=24183022" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib23" class="citations"><div id="sref23" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref23" title="View in article">23.</a></div><div>de March, C.A. ∙ Titlow, W.B. ∙ Sengoku, T. ...</div><div><strong>Modulation of the combinatorial code of odorant receptor response patterns in odorant mixtures</strong></div><div><em>Mol. Cell. Neurosci.</em> 2020; <strong>104</strong>:103469</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.mcn.2020.103469" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_23_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-85080982427" target="_blank" aria-label="Open Scopus in new tab">Scopus (14)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32061665/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.mcn.2020.103469&amp;pmid=32061665" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib24" class="citations"><div id="sref24" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref24" title="View in article">24.</a></div><div>Dey, S. ∙ Chamero, P. ∙ Pru, J.K. ...</div><div><strong>Cyclic regulation of sensory perception by a female hormone alters behavior</strong></div><div><em>Cell.</em> 2015; <strong>161</strong>:1334-1344</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=10.1016%2Fj.cell.2015.04.052&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=10.1016%2Fj.cell.2015.04.052&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-84930664111" target="_blank" aria-label="Open Scopus in new tab">Scopus (100)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26046438/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2015.04.052&amp;pmid=26046438" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib25" class="citations"><div id="sref25" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref25" title="View in article">25.</a></div><div>Eggan, K. ∙ Baldwin, K. ∙ Tackett, M. ...</div><div><strong>Mice cloned from olfactory sensory neurons</strong></div><div><em>Nature.</em> 2004; <strong>428</strong>:44-49</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature02375" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-1542347544" target="_blank" aria-label="Open Scopus in new tab">Scopus (298)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/14990966/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature02375&amp;pmid=14990966" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib26" class="citations"><div id="sref26" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref26" title="View in article">26.</a></div><div>Feinstein, P. ∙ Mombaerts, P.</div><div><strong>A contextual model for axonal sorting into glomeruli in the mouse olfactory system</strong></div><div><em>Cell.</em> 2004; <strong>117</strong>:817-831</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_26_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=10.1016%2Fj.cell.2004.05.011&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_26_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=10.1016%2Fj.cell.2004.05.011&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_26_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-2942604879" target="_blank" aria-label="Open Scopus in new tab">Scopus (246)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15186781/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2004.05.011&amp;pmid=15186781" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib27" class="citations"><div id="sref27" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref27" title="View in article">27.</a></div><div>Fettiplace, R. ∙ Ricci, A.J.</div><div><strong>Adaptation in auditory hair cells</strong></div><div><em>Curr. Opin. Neurobiol.</em> 2003; <strong>13</strong>:446-451</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/S0959-4388(03)00094-1" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-0041377628" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/12965292/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0959-4388%2803%2900094-1&amp;pmid=12965292" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib28" class="citations"><div id="sref28" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref28" title="View in article">28.</a></div><div>Firestein, S.</div><div><strong>How the olfactory system makes sense of scents</strong></div><div><em>Nature.</em> 2001; <strong>413</strong>:211-218</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/35093026" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-0035855833" target="_blank" aria-label="Open Scopus in new tab">Scopus (808)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11557990/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2F35093026&amp;pmid=11557990" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib29" class="citations"><div id="sref29" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref29-1" title="View in article">29.</a></div><div>Fischl, A.M. ∙ Heron, P.M. ∙ Stromberg, A.J. ...</div><div><strong>Activity-dependent genes in mouse olfactory sensory neurons</strong></div><div><em>Chem. Senses.</em> 2014; <strong>39</strong>:439-449</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/chemse/bju015" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-84901414304" target="_blank" aria-label="Open Scopus in new tab">Scopus (13)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24692514/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fchemse%2Fbju015&amp;pmid=24692514" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib30" class="citations"><div id="sref30" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref30" title="View in article">30.</a></div><div>Fitzwater, E. ∙ Coppola, D.M.</div><div><strong>Olfactory deprivation and enrichment: An identity of opposites?</strong></div><div><em>Chem. Senses.</em> 2021; <strong>46</strong>:bjaa071</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/chemse/bjaa071" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-85102212835" target="_blank" aria-label="Open Scopus in new tab">Scopus (1)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33103187/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fchemse%2Fbjaa071&amp;pmid=33103187" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib31" class="citations"><div id="sref31" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref31" title="View in article">31.</a></div><div>Fletcher, R.B. ∙ Das, D. ∙ Gadye, L. ...</div><div><strong>Deconstructing olfactory stem cell trajectories at single-cell resolution</strong></div><div><em>Cell Stem Cell.</em> 2017; <strong>20</strong>:817-830.e8</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=10.1016%2Fj.stem.2017.04.003&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=10.1016%2Fj.stem.2017.04.003&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-85019062024" target="_blank" aria-label="Open Scopus in new tab">Scopus (84)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28506465/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.stem.2017.04.003&amp;pmid=28506465" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib32" class="citations"><div id="sref32" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref32" title="View in article">32.</a></div><div>Galliano, E. ∙ Hahn, C. ∙ Browne, L.P.R. ...</div><div><strong>Brief sensory deprivation triggers cell type-specific structural and functional plasticity in olfactory bulb neurons</strong></div><div><em>J.&nbsp;Neurosci.</em> 2021; <strong>41</strong>:2135-2151</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1523/JNEUROSCI.1606-20.2020" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-85102909527" target="_blank" aria-label="Open Scopus in new tab">Scopus (2)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33483429/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1523%2FJNEUROSCI.1606-20.2020&amp;pmid=33483429" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib33" class="citations"><div id="sref33" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref33" title="View in article">33.</a></div><div>Goldman, M.S. ∙ Golowasch, J. ∙ Marder, E. ...</div><div><strong>Global structure, robustness, and modulation of neuronal models</strong></div><div><em>J.&nbsp;Neurosci.</em> 2001; <strong>21</strong>:5229-5238</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1523/JNEUROSCI.21-14-05229.2001" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11438598/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1523%2FJNEUROSCI.21-14-05229.2001&amp;pmid=11438598" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib34" class="citations"><div id="sref34" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref34" title="View in article">34.</a></div><div>Gouwens, N.W. ∙ Sorensen, S.A. ∙ Baftizadeh, F. ...</div><div><strong>Integrated morphoelectric and transcriptomic classification of cortical GABAergic cells</strong></div><div><em>Cell.</em> 2020; <strong>183</strong>:935-953.e19</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=10.1016%2Fj.cell.2020.09.057&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=10.1016%2Fj.cell.2020.09.057&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-85095742671" target="_blank" aria-label="Open Scopus in new tab">Scopus (50)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33186530/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2020.09.057&amp;pmid=33186530" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib35" class="citations"><div id="sref35" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref35" title="View in article">35.</a></div><div>Grabska-Barwińska, A. ∙ Barthelmé, S. ∙ Beck, J. ...</div><div><strong>A probabilistic approach to demixing odors</strong></div><div><em>Nat. Neurosci.</em> 2017; <strong>20</strong>:98-106</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nn.4444" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27918530/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnn.4444&amp;pmid=27918530" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib36" class="citations"><div id="sref36" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref36" title="View in article">36.</a></div><div>Hagendorf, S. ∙ Fluegge, D. ∙ Engelhardt, C. ...</div><div><strong>Homeostatic control of sensory output in basal vomeronasal neurons: activity-dependent expression of ether-à-go-go-related gene potassium channels</strong></div><div><em>J.&nbsp;Neurosci.</em> 2009; <strong>29</strong>:206-221</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1523/JNEUROSCI.3656-08.2009" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-58149397652" target="_blank" aria-label="Open Scopus in new tab">Scopus (24)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19129398/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1523%2FJNEUROSCI.3656-08.2009&amp;pmid=19129398" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib37" class="citations"><div id="sref37" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref37-1" title="View in article">37.</a></div><div>Hanchate, N.K. ∙ Kondoh, K. ∙ Lu, Z. ...</div><div><strong>Single-cell transcriptomics reveals receptor transformations during olfactory neurogenesis</strong></div><div><em>Science.</em> 2015; <strong>350</strong>:1251-1255</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aad2456" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-84949035291" target="_blank" aria-label="Open Scopus in new tab">Scopus (116)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26541607/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aad2456&amp;pmid=26541607" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib38" class="citations"><div id="sref38" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref38" title="View in article">38.</a></div><div>Hopfield, J.J.</div><div><strong>Olfactory computation and object perception</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 1991; <strong>88</strong>:6462-6466</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.88.15.6462" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_38_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-0026003099" target="_blank" aria-label="Open Scopus in new tab">Scopus (93)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/1862075/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.88.15.6462&amp;pmid=1862075" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib39" class="citations"><div id="sref39" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref39" title="View in article">39.</a></div><div>Hu, X.S. ∙ Ikegami, K. ∙ Vihani, A. ...</div><div><strong>Concentration-dependent recruitment of mammalian odorant receptors</strong></div><div><em>eNeuro.</em> 2020; <strong>7</strong></div><div>ENEURO.0103-19.2019</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1523/ENEURO.0103-19.2019" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-85084028188" target="_blank" aria-label="Open Scopus in new tab">Scopus (5)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1523%2FENEURO.0103-19.2019" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib40" class="citations"><div id="sref40" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref40" title="View in article">40.</a></div><div>Ibarra-Soria, X. ∙ Nakahara, T.S. ∙ Lilue, J. ...</div><div><strong>Variation in olfactory neuron repertoires is genetically controlled and environmentally modulated</strong></div><div><em>eLife.</em> 2017; <strong>6</strong>:1083</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.7554/eLife.21476" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_40_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-85018344872" target="_blank" aria-label="Open Scopus in new tab">Scopus (38)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.7554%2FeLife.21476" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib41" class="citations"><div id="sref41" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref41" title="View in article">41.</a></div><div>Imai, T. ∙ Sakano, H. ∙ Vosshall, L.B.</div><div><strong>Topographic mapping—The olfactory system</strong></div><div><em>Cold Spring Harb. Perspect. Biol.</em> 2010; <strong>2</strong>:a001776</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/cshperspect.a001776" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-78349286342" target="_blank" aria-label="Open Scopus in new tab">Scopus (55)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20554703/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2Fcshperspect.a001776&amp;pmid=20554703" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib42" class="citations"><div id="sref42" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref42-1" title="View in article">42.</a></div><div>Jiang, Y. ∙ Gong, N.N. ∙ Hu, X.S. ...</div><div><strong>Molecular profiling of activated olfactory neurons identifies odorant receptors for odors in&nbsp;vivo</strong></div><div><em>Nat. Neurosci.</em> 2015; <strong>18</strong>:1446-1454</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nn.4104" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-84942414428" target="_blank" aria-label="Open Scopus in new tab">Scopus (62)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26322927/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnn.4104&amp;pmid=26322927" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib43" class="citations"><div id="sref43" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref43" title="View in article">43.</a></div><div>Juang, B.-T. ∙ Gu, C. ∙ Starnes, L. ...</div><div><strong>Endogenous nuclear RNAi mediates behavioral adaptation to odor</strong></div><div><em>Cell.</em> 2013; <strong>154</strong>:1010-1022</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=10.1016%2Fj.cell.2013.08.006&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=10.1016%2Fj.cell.2013.08.006&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-84883426407" target="_blank" aria-label="Open Scopus in new tab">Scopus (53)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23993094/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2013.08.006&amp;pmid=23993094" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib44" class="citations"><div id="sref44" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref44-1" title="View in article">44.</a></div><div>Kadohisa, M. ∙ Wilson, D.A.</div><div><strong>Olfactory cortical adaptation facilitates detection of odors against background</strong></div><div><em>J.&nbsp;Neurophysiol.</em> 2006; <strong>95</strong>:1888-1896</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1152/jn.00812.2005" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-33646004295" target="_blank" aria-label="Open Scopus in new tab">Scopus (88)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16251260/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1152%2Fjn.00812.2005&amp;pmid=16251260" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib120" class="citations"><div id="optfKwddODQsj" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-optfKwddODQsj" title="View in article">45.</a></div><div>Kass, M.D. ∙ Guang, S.A. ∙ Moberly, A.H. ...</div><div><strong>Changes in Olfactory Sensory Neuron Physiology and Olfactory Perceptual Learning After Odorant Exposure in Adult Mice</strong></div><div><em>Chem. Senses.</em> 2016; <strong>41</strong>:123-133</div></div><div class="external-links"><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26514410/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?pmid=26514410" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib45" class="citations"><div id="sref45" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref45-1" title="View in article">46.</a></div><div>Kass, M.D. ∙ Pottackal, J. ∙ Turkel, D.J. ...</div><div><strong>Changes in the neural representation of odorants after olfactory deprivation in the adult mouse olfactory bulb</strong></div><div><em>Chem. Senses.</em> 2013; <strong>38</strong>:77-89</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/chemse/bjs081" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-84871317604" target="_blank" aria-label="Open Scopus in new tab">Scopus (12)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23125347/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fchemse%2Fbjs081&amp;pmid=23125347" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib46" class="citations"><div id="sref46" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref46" title="View in article">47.</a></div><div>Kato, H.K. ∙ Chu, M.W. ∙ Isaacson, J.S. ...</div><div><strong>Dynamic sensory representations in the olfactory bulb: Modulation by wakefulness and experience</strong></div><div><em>Neuron.</em> 2012; <strong>76</strong>:962-975</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=10.1016%2Fj.neuron.2012.09.037&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=10.1016%2Fj.neuron.2012.09.037&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-84872745558" target="_blank" aria-label="Open Scopus in new tab">Scopus (150)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23217744/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.neuron.2012.09.037&amp;pmid=23217744" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib47" class="citations"><div id="sref47" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref47" title="View in article">48.</a></div><div>Kim, D.-W. ∙ Yao, Z. ∙ Graybuck, L.T. ...</div><div><strong>Multimodal analysis of cell types in a hypothalamic node controlling social behavior</strong></div><div><em>Cell.</em> 2019; <strong>179</strong>:713-728.e17</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_48_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=10.1016%2Fj.cell.2019.09.020&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_48_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=10.1016%2Fj.cell.2019.09.020&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_48_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-85073612261" target="_blank" aria-label="Open Scopus in new tab">Scopus (54)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31626771/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2019.09.020&amp;pmid=31626771" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib48" class="citations"><div id="sref48" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref48" title="View in article">49.</a></div><div>Kim, E.J. ∙ Zhang, Z. ∙ Huang, L. ...</div><div><strong>Extraction of distinct neuronal cell types from within a genetically continuous population</strong></div><div><em>Neuron.</em> 2020; <strong>107</strong>:274-282.e6</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=10.1016%2Fj.neuron.2020.04.018&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=10.1016%2Fj.neuron.2020.04.018&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-85085319359" target="_blank" aria-label="Open Scopus in new tab">Scopus (28)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32396852/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.neuron.2020.04.018&amp;pmid=32396852" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib122" class="citations"><div id="optUxWGoN4vCM" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-optUxWGoN4vCM" title="View in article">50.</a></div><div>Kobayakawa, K. ∙ Kobayakawa, R. ∙ Matsumoto, H. ...</div><div><strong>Innate versus learned odour processing in the mouse olfactory bulb</strong></div><div><em>Nature.</em> 2007; <strong>450</strong>:503-508</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature06281" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-36549040013" target="_blank" aria-label="Open Scopus in new tab">Scopus (444)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17989651/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature06281&amp;pmid=17989651" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib49" class="citations"><div id="sref49" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref49-1" title="View in article">51.</a></div><div>Kostal, L. ∙ Lansky, P. ∙ Rospars, J.-P.</div><div><strong>Efficient olfactory coding in the pheromone receptor neuron of a moth</strong></div><div><em>PLoS Comput. Biol.</em> 2008; <strong>4</strong>:e1000053</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pcbi.1000053" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_51_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-43249110822" target="_blank" aria-label="Open Scopus in new tab">Scopus (29)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/18437217/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pcbi.1000053&amp;pmid=18437217" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib50" class="citations"><div id="sref50" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref50-1" title="View in article">52.</a></div><div>Kotliar, D. ∙ Veres, A. ∙ Nagy, M.A. ...</div><div><strong>Identifying gene expression programs of cell-type identity and cellular activity with single-cell RNA-seq</strong></div><div><em>eLife.</em> 2019; <strong>8</strong>:e43803</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.7554/eLife.43803" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-85070056080" target="_blank" aria-label="Open Scopus in new tab">Scopus (43)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31282856/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.7554%2FeLife.43803&amp;pmid=31282856" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib51" class="citations"><div id="sref51" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref51" title="View in article">53.</a></div><div>L’Etoile, N.D. ∙ Coburn, C.M. ∙ Eastham, J. ...</div><div><strong>The cyclic GMP-dependent protein kinase EGL-4 regulates olfactory adaptation in <i>C.&nbsp;elegans</i></strong></div><div><em>Neuron.</em> 2002; <strong>36</strong>:1079-1089</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=10.1016%2FS0896-6273%2802%2901066-8&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=10.1016%2FS0896-6273%2802%2901066-8&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-0037137698" target="_blank" aria-label="Open Scopus in new tab">Scopus (144)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/12495623/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0896-6273%2802%2901066-8&amp;pmid=12495623" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib52" class="citations"><div id="sref52" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref52" title="View in article">54.</a></div><div>La Manno, G. ∙ Soldatov, R. ∙ Zeisel, A. ...</div><div><strong>RNA velocity of single cells</strong></div><div><em>Nature.</em> 2018; <strong>560</strong>:494-498</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-018-0414-6" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-85052109231" target="_blank" aria-label="Open Scopus in new tab">Scopus (690)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30089906/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-018-0414-6&amp;pmid=30089906" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib53" class="citations"><div id="sref53" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref53" title="View in article">55.</a></div><div>Lecoq, J. ∙ Tiret, P. ∙ Charpak, S.</div><div><strong>Peripheral adaptation codes for high odor concentration in glomeruli</strong></div><div><em>J.&nbsp;Neurosci.</em> 2009; <strong>29</strong>:3067-3072</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1523/JNEUROSCI.6187-08.2009" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-63849232987" target="_blank" aria-label="Open Scopus in new tab">Scopus (44)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19279243/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1523%2FJNEUROSCI.6187-08.2009&amp;pmid=19279243" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib54" class="citations"><div id="sref54" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref54-1" title="View in article">56.</a></div><div>Li, A. ∙ Gire, D.H. ∙ Bozza, T. ...</div><div><strong>Precise detection of direct glomerular input duration by the olfactory bulb</strong></div><div><em>J.&nbsp;Neurosci.</em> 2014; <strong>34</strong>:16058-16064</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1523/JNEUROSCI.3382-14.2014" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_56_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-84911870900" target="_blank" aria-label="Open Scopus in new tab">Scopus (37)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25429146/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1523%2FJNEUROSCI.3382-14.2014&amp;pmid=25429146" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib55" class="citations"><div id="sref55" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref55" title="View in article">57.</a></div><div>Li, H. ∙ Horns, F. ∙ Wu, B. ...</div><div><strong>Classifying <i>Drosophila</i> olfactory projection neuron subtypes by single-cell RNA sequencing</strong></div><div><em>Cell.</em> 2017; <strong>171</strong>:1206-1220.e22</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=10.1016%2Fj.cell.2017.10.019&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=10.1016%2Fj.cell.2017.10.019&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-85034065086" target="_blank" aria-label="Open Scopus in new tab">Scopus (104)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29149607/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2017.10.019&amp;pmid=29149607" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib56" class="citations"><div id="sref56" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref56" title="View in article">58.</a></div><div>Litwin-Kumar, A. ∙ Harris, K.D. ∙ Axel, R. ...</div><div><strong>Optimal degrees of synaptic connectivity</strong></div><div><em>Neuron.</em> 2017; <strong>93</strong>:1153-1164.e7</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=10.1016%2Fj.neuron.2017.01.030&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=10.1016%2Fj.neuron.2017.01.030&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-85012934211" target="_blank" aria-label="Open Scopus in new tab">Scopus (97)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28215558/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.neuron.2017.01.030&amp;pmid=28215558" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib57" class="citations"><div id="sref57" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref57" title="View in article">59.</a></div><div>Ma, L. ∙ Wu, Y. ∙ Qiu, Q. ...</div><div><strong>A developmental switch of axon targeting in the continuously regenerating mouse olfactory system</strong></div><div><em>Science.</em> 2014; <strong>344</strong>:194-197</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.1248805" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-84897975541" target="_blank" aria-label="Open Scopus in new tab">Scopus (45)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24723610/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.1248805&amp;pmid=24723610" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib58" class="citations"><div id="sref58" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref58-1" title="View in article">60.</a></div><div>Marder, E.</div><div><strong>Variability, compensation, and modulation in neurons and circuits</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2011; <strong>108</strong>:15542-15548</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1010674108" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-80053141839" target="_blank" aria-label="Open Scopus in new tab">Scopus (197)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21383190/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1010674108&amp;pmid=21383190" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib59" class="citations"><div id="sref59" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref59-1" title="View in article">61.</a></div><div>Marder, E. ∙ Goaillard, J.-M.</div><div><strong>Variability, compensation and homeostasis in neuron and network function</strong></div><div><em>Nat. Rev. Neurosci.</em> 2006; <strong>7</strong>:563-574</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nrn1949" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-33745712893" target="_blank" aria-label="Open Scopus in new tab">Scopus (723)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16791145/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnrn1949&amp;pmid=16791145" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib60" class="citations"><div id="sref60" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref60-1" title="View in article">62.</a></div><div>Martelli, C. ∙ Storace, D.A.</div><div><strong>Stimulus driven functional transformations in the early olfactory system</strong></div><div><em>Front. Cell. Neurosci.</em> 2021; <strong>15</strong>:684742</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.3389/fncel.2021.684742" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-85113135443" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/34413724/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.3389%2Ffncel.2021.684742&amp;pmid=34413724" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib61" class="citations"><div id="sref61" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref61" title="View in article">63.</a></div><div>Mashukova, A. ∙ Spehr, M. ∙ Hatt, H. ...</div><div><strong>β-arrestin2-mediated internalization of mammalian odorant receptors</strong></div><div><em>J.&nbsp;Neurosci.</em> 2006; <strong>26</strong>:9902-9912</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1523/JNEUROSCI.2897-06.2006" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-33749170169" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17005854/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1523%2FJNEUROSCI.2897-06.2006&amp;pmid=17005854" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib62" class="citations"><div id="sref62" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref62" title="View in article">64.</a></div>McInnes, L., Healy, J., and Melville, J. (2018). UMAP: Uniform manifold approximation and projection for dimension reduction. arXiv, <a id="intref0105" href="https://arxiv.org/pdf/1802.03426v3.pdf" target="_blank">https://arxiv.org/pdf/1802.03426v3.pdf</a>.</div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=McInnes%2C+L.%2C+Healy%2C+J.%2C+and+Melville%2C+J.+%282018%29.+UMAP%3A+Uniform+manifold+approximation+and+projection+for+dimension+reduction.+arXiv%2C+https%3A%2F%2Farxiv.org%2Fpdf%2F1802.03426v3.pdf." target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib63" class="citations"><div id="sref63" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref63" title="View in article">65.</a></div><div>McNamara, A.M. ∙ Magidson, P.D. ∙ Linster, C. ...</div><div><strong>Distinct neural mechanisms mediate olfactory memory formation at different timescales</strong></div><div><em>Learn. Mem.</em> 2008; <strong>15</strong>:117-125</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/lm.785608" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-40949119862" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/18299438/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2Flm.785608&amp;pmid=18299438" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib64" class="citations"><div id="sref64" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref64" title="View in article">66.</a></div><div>Miyamichi, K. ∙ Serizawa, S. ∙ Kimura, H.M. ...</div><div><strong>Continuous and overlapping expression domains of odorant receptor genes in the olfactory epithelium determine the dorsal/ventral positioning of glomeruli in the olfactory bulb</strong></div><div><em>J.&nbsp;Neurosci.</em> 2005; <strong>25</strong>:3586-3592</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1523/JNEUROSCI.0324-05.2005" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-17044395103" target="_blank" aria-label="Open Scopus in new tab">Scopus (207)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15814789/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1523%2FJNEUROSCI.0324-05.2005&amp;pmid=15814789" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib65" class="citations"><div id="sref65" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref65-1" title="View in article">67.</a></div><div>Monahan, K. ∙ Lomvardas, S.</div><div><strong>Monoallelic expression of olfactory receptors</strong></div><div><em>Annu. Rev. Cell Dev. Biol.</em> 2015; <strong>31</strong>:721-740</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1146/annurev-cellbio-100814-125308" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_67_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-84947294591" target="_blank" aria-label="Open Scopus in new tab">Scopus (71)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26359778/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1146%2Fannurev-cellbio-100814-125308&amp;pmid=26359778" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib66" class="citations"><div id="sref66" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref66-1" title="View in article">68.</a></div><div>Monahan, K. ∙ Schieren, I. ∙ Cheung, J. ...</div><div><strong>Cooperative interactions enable singular olfactory receptor expression in mouse olfactory neurons</strong></div><div><em>eLife.</em> 2017; <strong>6</strong>:e28620</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.7554/eLife.28620" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-85032933643" target="_blank" aria-label="Open Scopus in new tab">Scopus (42)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28933695/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.7554%2FeLife.28620&amp;pmid=28933695" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib67" class="citations"><div id="sref67" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref67-1" title="View in article">69.</a></div><div>Moore, P.A.</div><div><strong>A model of the role of adaptation and disadaptation in olfactory receptor neurons: Implications for the coding of temporal and intensity patterns in odor signals</strong></div><div><em>Chem. Senses.</em> 1994; <strong>19</strong>:71-86</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/chemse/19.1.71" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-0028153852" target="_blank" aria-label="Open Scopus in new tab">Scopus (42)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/8055260/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fchemse%2F19.1.71&amp;pmid=8055260" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib68" class="citations"><div id="sref68" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref68-1" title="View in article">70.</a></div><div>Mori, K. ∙ Sakano, H.</div><div><strong>How is the olfactory map formed and interpreted in the mammalian brain?</strong></div><div><em>Annu. Rev. Neurosci.</em> 2011; <strong>34</strong>:467-499</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1146/annurev-neuro-112210-112917" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-79959891730" target="_blank" aria-label="Open Scopus in new tab">Scopus (227)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21469960/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1146%2Fannurev-neuro-112210-112917&amp;pmid=21469960" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib69" class="citations"><div id="sref69" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref69" title="View in article">71.</a></div><div>Nagel, K.I. ∙ Wilson, R.I.</div><div><strong>Biophysical mechanisms underlying olfactory receptor neuron dynamics</strong></div><div><em>Nat. Neurosci.</em> 2011; <strong>14</strong>:208-216</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nn.2725" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-79251594106" target="_blank" aria-label="Open Scopus in new tab">Scopus (108)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21217763/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnn.2725&amp;pmid=21217763" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib70" class="citations"><div id="sref70" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref70" title="View in article">72.</a></div><div>Nakashima, A. ∙ Takeuchi, H. ∙ Imai, T. ...</div><div><strong>Agonist-independent GPCR activity regulates anterior-posterior targeting of olfactory sensory neurons</strong></div><div><em>Cell.</em> 2013; <strong>154</strong>:1314-1325</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=10.1016%2Fj.cell.2013.08.033&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=10.1016%2Fj.cell.2013.08.033&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-84884249630" target="_blank" aria-label="Open Scopus in new tab">Scopus (92)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24034253/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2013.08.033&amp;pmid=24034253" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib71" class="citations"><div id="sref71" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref71" title="View in article">73.</a></div><div>Ng, R. ∙ Salem, S.S. ∙ Wu, S.-T. ...</div><div><strong>Amplification of <i>Drosophila</i> olfactory responses by a DEG/ENaC channel</strong></div><div><em>Neuron.</em> 2019; <strong>104</strong>:947-959.e5</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=10.1016%2Fj.neuron.2019.08.041&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=10.1016%2Fj.neuron.2019.08.041&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-85074767200" target="_blank" aria-label="Open Scopus in new tab">Scopus (20)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31629603/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.neuron.2019.08.041&amp;pmid=31629603" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib72" class="citations"><div id="sref72" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref72-1" title="View in article">74.</a></div><div>O’Leary, T. ∙ Williams, A.H. ∙ Caplan, J.S. ...</div><div><strong>Correlations in ion channel expression emerge from homeostatic tuning rules</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2013; <strong>110</strong>:E2645-E2654</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1309966110" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-84879927251" target="_blank" aria-label="Open Scopus in new tab">Scopus (111)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23798391/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1309966110&amp;pmid=23798391" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib73" class="citations"><div id="sref73" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref73" title="View in article">75.</a></div><div>Oberland, S. ∙ Ackels, T. ∙ Gaab, S. ...</div><div><strong>CD36 is involved in oleic acid detection by the murine olfactory system</strong></div><div><em>Front. Cell. Neurosci.</em> 2015; <strong>9</strong>:366</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.3389/fncel.2015.00366" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_75_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-84942904692" target="_blank" aria-label="Open Scopus in new tab">Scopus (24)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26441537/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.3389%2Ffncel.2015.00366&amp;pmid=26441537" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib74" class="citations"><div id="sref74" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref74-1" title="View in article">76.</a></div><div>Pachitariu, M. ∙ Stringer, C. ∙ Dipoppa, M. ...</div><div><strong>Suite2p: Beyond 10,000 neurons with standard two-photon microscopy</strong></div><div><em>bioRxiv.</em> 2017;</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.PachitariuC.StringerM.DipoppaS.Schr%C3%B6derL.F.RossiH.DalgleishM.CarandiniK.D.HarrisSuite2p%3A+Beyond+10%2C000+neurons+with+standard+two-photon+microscopybioRxiv2017" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib75" class="citations"><div id="sref75" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref75" title="View in article">77.</a></div><div>Pacifico, R. ∙ Dewan, A. ∙ Cawley, D. ...</div><div><strong>An olfactory subsystem that mediates high-sensitivity detection of volatile amines</strong></div><div><em>Cell Rep.</em> 2012; <strong>2</strong>:76-88</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=10.1016%2Fj.celrep.2012.06.006&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=10.1016%2Fj.celrep.2012.06.006&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-84864313337" target="_blank" aria-label="Open Scopus in new tab">Scopus (90)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22840399/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.celrep.2012.06.006&amp;pmid=22840399" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib76" class="citations"><div id="sref76" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref76" title="View in article">78.</a></div><div>Parrilla, M. ∙ Chang, I. ∙ Degl’Innocenti, A. ...</div><div><strong>Expression of homeobox genes in the mouse olfactory epithelium</strong></div><div><em>J.&nbsp;Comp. Neurol.</em> 2016; <strong>524</strong>:2713-2739</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/cne.24051" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_78_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-85028257873" target="_blank" aria-label="Open Scopus in new tab">Scopus (7)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27243442/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fcne.24051&amp;pmid=27243442" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib77" class="citations"><div id="sref77" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref77-1" title="View in article">79.</a></div><div>Pashkovski, S.L. ∙ Iurilli, G. ∙ Brann, D. ...</div><div><strong>Structure and flexibility in cortical representations of odour space</strong></div><div><em>Nature.</em> 2020; <strong>583</strong>:253-258</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-020-2451-1" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_79_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-85087290240" target="_blank" aria-label="Open Scopus in new tab">Scopus (24)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32612230/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-020-2451-1&amp;pmid=32612230" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib78" class="citations"><div id="sref78" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref78" title="View in article">80.</a></div><div>Repicky, S.E. ∙ Luetje, C.W.</div><div><strong>Molecular receptive range variation among mouse odorant receptors for aliphatic carboxylic acids</strong></div><div><em>J.&nbsp;Neurochem.</em> 2009; <strong>109</strong>:193-202</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1111/j.1471-4159.2009.05925.x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_80_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-61849133441" target="_blank" aria-label="Open Scopus in new tab">Scopus (14)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19166503/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1111%2Fj.1471-4159.2009.05925.x&amp;pmid=19166503" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib79" class="citations"><div id="sref79" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref79" title="View in article">81.</a></div><div>Rubin, B.D. ∙ Katz, L.C.</div><div><strong>Optical imaging of odorant representations in the mammalian olfactory bulb</strong></div><div><em>Neuron.</em> 1999; <strong>23</strong>:499-511</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=10.1016%2FS0896-6273%2800%2980803-X&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=10.1016%2FS0896-6273%2800%2980803-X&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-0033166536" target="_blank" aria-label="Open Scopus in new tab">Scopus (536)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/10433262/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0896-6273%2800%2980803-X&amp;pmid=10433262" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib80" class="citations"><div id="sref80" class="citation"><div class="citation-content"><div class="label"><a href="#figs5" title="View in article">82.</a></div><div>Saito, H. ∙ Chi, Q. ∙ Zhuang, H. ...</div><div><strong>Odor coding by a mammalian receptor repertoire</strong></div><div><em>Sci. Signal.</em> 2009; <strong>2</strong>:ra9</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/scisignal.2000016" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_82_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-65949098470" target="_blank" aria-label="Open Scopus in new tab">Scopus (363)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19261596/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscisignal.2000016&amp;pmid=19261596" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib81" class="citations"><div id="sref81" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref81" title="View in article">83.</a></div><div>Scala, F. ∙ Kobak, D. ∙ Bernabucci, M. ...</div><div><strong>Phenotypic variation of transcriptomic cell types in mouse motor cortex</strong></div><div><em>Nature.</em> 2021; <strong>598</strong>:144-150</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-020-2907-3" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_83_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-85095987674" target="_blank" aria-label="Open Scopus in new tab">Scopus (18)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33184512/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-020-2907-3&amp;pmid=33184512" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib82" class="citations"><div id="sref82" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref82" title="View in article">84.</a></div><div>Schaffer, E.S. ∙ Stettler, D.D. ∙ Kato, D. ...</div><div><strong>Odor perception on the two sides of the brain: Consistency despite randomness</strong></div><div><em>Neuron.</em> 2018; <strong>98</strong>:736-742.e3</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_84_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=10.1016%2Fj.neuron.2018.04.004&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_84_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=10.1016%2Fj.neuron.2018.04.004&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_84_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-85046149960" target="_blank" aria-label="Open Scopus in new tab">Scopus (21)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29706585/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.neuron.2018.04.004&amp;pmid=29706585" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib83" class="citations"><div id="sref83" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref83" title="View in article">85.</a></div><div>Scholz, P. ∙ Kalbe, B. ∙ Jansen, F. ...</div><div><strong>Transcriptome analysis of murine olfactory sensory neurons during development using single cell RNA-seq</strong></div><div><em>Chem. Senses.</em> 2016; <strong>41</strong>:313-323</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/chemse/bjw003" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-84964898686" target="_blank" aria-label="Open Scopus in new tab">Scopus (25)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26839357/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fchemse%2Fbjw003&amp;pmid=26839357" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib84" class="citations"><div id="sref84" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref84-1" title="View in article">86.</a></div><div>Schulz, D.J. ∙ Goaillard, J.-M. ∙ Marder, E.E.</div><div><strong>Quantitative expression profiling of identified neurons reveals cell-specific constraints on highly variable levels of gene expression</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2007; <strong>104</strong>:13187-13191</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.0705827104" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_86_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-34548760721" target="_blank" aria-label="Open Scopus in new tab">Scopus (184)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17652510/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.0705827104&amp;pmid=17652510" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib85" class="citations"><div id="sref85" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref85" title="View in article">87.</a></div><div>Soucy, E.R. ∙ Albeanu, D.F. ∙ Fantana, A.L. ...</div><div><strong>Precision and diversity in an odor map on the olfactory bulb</strong></div><div><em>Nat. Neurosci.</em> 2009; <strong>12</strong>:210-220</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nn.2262" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_87_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-58849109461" target="_blank" aria-label="Open Scopus in new tab">Scopus (217)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19151709/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnn.2262&amp;pmid=19151709" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib86" class="citations"><div id="sref86" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref86" title="View in article">88.</a></div><div>Spors, H. ∙ Grinvald, A.</div><div><strong>Spatio-temporal dynamics of odor representations in the mammalian olfactory bulb</strong></div><div><em>Neuron.</em> 2002; <strong>34</strong>:301-315</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=10.1016%2FS0896-6273%2802%2900644-X&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=10.1016%2FS0896-6273%2802%2900644-X&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-0037061682" target="_blank" aria-label="Open Scopus in new tab">Scopus (262)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11970871/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0896-6273%2802%2900644-X&amp;pmid=11970871" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib87" class="citations"><div id="sref87" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref87" title="View in article">89.</a></div><div>Sullivan, S.L. ∙ Ressler, K.J. ∙ Buck, L.B.</div><div><strong>Spatial patterning and information coding in the olfactory system</strong></div><div><em>Curr. Opin. Genet. Dev.</em> 1995; <strong>5</strong>:516-523</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/0959-437X(95)90057-N" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/7580145/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2F0959-437X%2895%2990057-N&amp;pmid=7580145" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib121" class="citations"><div id="optwGxKihF1PF" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-optwGxKihF1PF" title="View in article">90.</a></div><div>Tan, L. ∙ Xie, X.S.</div><div><strong>A Near-Complete Spatial Map of Olfactory Receptors in the Mouse Main Olfactory Epithelium</strong></div><div><em>Chem. Senses.</em> 2018; <strong>43</strong>:427-432</div></div><div class="external-links"><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29796642/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?pmid=29796642" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib88" class="citations"><div id="sref88" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref88" title="View in article">91.</a></div><div>Tasic, B. ∙ Yao, Z. ∙ Graybuck, L.T. ...</div><div><strong>Shared and distinct transcriptomic cell types across neocortical areas</strong></div><div><em>Nature.</em> 2018; <strong>563</strong>:72-78</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-018-0654-5" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-85055855595" target="_blank" aria-label="Open Scopus in new tab">Scopus (420)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30382198/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-018-0654-5&amp;pmid=30382198" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib89" class="citations"><div id="sref89" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref89" title="View in article">92.</a></div><div>Teşileanu, T. ∙ Cocco, S. ∙ Monasson, R. ...</div><div><strong>Adaptation of olfactory receptor abundances for efficient coding</strong></div><div><em>eLife.</em> 2019; <strong>8</strong>:308</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.7554/eLife.39279" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_92_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-85062409446" target="_blank" aria-label="Open Scopus in new tab">Scopus (8)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.7554%2FeLife.39279" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib90" class="citations"><div id="sref90" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref90" title="View in article">93.</a></div><div>Touhara, K.</div><div><strong>Odor discrimination by G protein-coupled olfactory receptors</strong></div><div><em>Microsc. Res. Tech.</em> 2002; <strong>58</strong>:135-141</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/jemt.10131" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_93_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-0036668321" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/12203691/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fjemt.10131&amp;pmid=12203691" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib91" class="citations"><div id="sref91" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref91-1" title="View in article">94.</a></div><div>Turrigiano, G.G.</div><div><strong>Homeostatic plasticity in neuronal networks: The more things change, the more they stay the same</strong></div><div><em>Trends Neurosci.</em> 1999; <strong>22</strong>:221-227</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_94_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=10.1016%2FS0166-2236%2898%2901341-1&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_94_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=10.1016%2FS0166-2236%2898%2901341-1&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_94_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-0033135312" target="_blank" aria-label="Open Scopus in new tab">Scopus (585)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/10322495/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0166-2236%2898%2901341-1&amp;pmid=10322495" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib92" class="citations"><div id="sref92" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref92" title="View in article">95.</a></div><div>Turrigiano, G.G.</div><div><strong>The self-tuning neuron: Synaptic scaling of excitatory synapses</strong></div><div><em>Cell.</em> 2008; <strong>135</strong>:422-435</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_95_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=10.1016%2Fj.cell.2008.10.008&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_95_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=10.1016%2Fj.cell.2008.10.008&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_95_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-54549125798" target="_blank" aria-label="Open Scopus in new tab">Scopus (942)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/18984155/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2008.10.008&amp;pmid=18984155" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib93" class="citations"><div id="sref93" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref93-1" title="View in article">96.</a></div><div>Turrigiano, G.</div><div><strong>Too many cooks? Intrinsic and synaptic homeostatic mechanisms in cortical circuit refinement</strong></div><div><em>Annu. Rev. Neurosci.</em> 2011; <strong>34</strong>:89-103</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1146/annurev-neuro-060909-153238" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_96_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-79959889965" target="_blank" aria-label="Open Scopus in new tab">Scopus (425)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21438687/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1146%2Fannurev-neuro-060909-153238&amp;pmid=21438687" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib94" class="citations"><div id="sref94" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref94-1" title="View in article">97.</a></div><div>Turrigiano, G.G.</div><div><strong>The dialectic of Hebb and homeostasis</strong></div><div><em>Philos. Trans. R. Soc. Lond. B Biol. Sci.</em> 2017; <strong>372</strong>:20160258</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1098/rstb.2016.0258" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_97_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-85009958893" target="_blank" aria-label="Open Scopus in new tab">Scopus (97)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28093556/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1098%2Frstb.2016.0258&amp;pmid=28093556" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib95" class="citations"><div id="sref95" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref95" title="View in article">98.</a></div><div>Tyssowski, K.M. ∙ Gray, J.M.</div><div><strong>The neuronal stimulation-transcription coupling map</strong></div><div><em>Curr. Opin. Neurobiol.</em> 2019; <strong>59</strong>:87-94</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.conb.2019.05.001" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_98_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-85066322540" target="_blank" aria-label="Open Scopus in new tab">Scopus (7)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31163285/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.conb.2019.05.001&amp;pmid=31163285" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib96" class="citations"><div id="sref96" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref96" title="View in article">99.</a></div><div>Vihani, A. ∙ Hu, X.S. ∙ Gundala, S. ...</div><div><strong>Semiochemical responsive olfactory sensory neurons are sexually dimorphic and plastic</strong></div><div><em>eLife.</em> 2020; <strong>9</strong>:e54501</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.7554/eLife.54501" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_99_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-85098288126" target="_blank" aria-label="Open Scopus in new tab">Scopus (4)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33231170/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.7554%2FeLife.54501&amp;pmid=33231170" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib97" class="citations"><div id="sref97" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref97-1" title="View in article">100.</a></div><div>von der Weid, B. ∙ Rossier, D. ∙ Lindup, M. ...</div><div><strong>Large-scale transcriptional profiling of chemosensory neurons identifies receptor-ligand pairs in&nbsp;vivo</strong></div><div><em>Nat. Neurosci.</em> 2015; <strong>18</strong>:1455-1463</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nn.4100" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_100_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-84942456919" target="_blank" aria-label="Open Scopus in new tab">Scopus (61)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26322926/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnn.4100&amp;pmid=26322926" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib98" class="citations"><div id="sref98" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref98" title="View in article">101.</a></div><div>Wachowiak, M. ∙ Cohen, L.B.</div><div><strong>Representation of odorants by receptor neuron input to the mouse olfactory bulb</strong></div><div><em>Neuron.</em> 2001; <strong>32</strong>:723-735</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_101_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=10.1016%2FS0896-6273%2801%2900506-2&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_101_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=10.1016%2FS0896-6273%2801%2900506-2&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_101_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-0035923747" target="_blank" aria-label="Open Scopus in new tab">Scopus (370)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11719211/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0896-6273%2801%2900506-2&amp;pmid=11719211" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib99" class="citations"><div id="sref99" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref99" title="View in article">102.</a></div><div>Waggener, C.T. ∙ Coppola, D.M.</div><div><strong>Naris occlusion alters the electro-olfactogram: Evidence for compensatory plasticity in the olfactory system</strong></div><div><em>Neurosci. Lett.</em> 2007; <strong>427</strong>:112-116</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.neulet.2007.09.013" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_102_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-35448943484" target="_blank" aria-label="Open Scopus in new tab">Scopus (27)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17931777/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.neulet.2007.09.013&amp;pmid=17931777" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib100" class="citations"><div id="sref100" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref100" title="View in article">103.</a></div><div>Wang, H.W. ∙ Wysocki, C.J. ∙ Gold, G.H.</div><div><strong>Induction of olfactory receptor sensitivity in mice</strong></div><div><em>Science.</em> 1993; <strong>260</strong>:998-1000</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.8493539" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_103_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-0027237421" target="_blank" aria-label="Open Scopus in new tab">Scopus (154)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/8493539/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.8493539&amp;pmid=8493539" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib101" class="citations"><div id="sref101" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref101-1" title="View in article">104.</a></div><div>Wang, Q. ∙ Titlow, W.B. ∙ McClintock, D.A. ...</div><div><strong>Activity-dependent gene expression in the mammalian olfactory epithelium</strong></div><div><em>Chem. Senses.</em> 2017; <strong>42</strong>:611-624</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/chemse/bjx028" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_104_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-85030719241" target="_blank" aria-label="Open Scopus in new tab">Scopus (7)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28525560/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fchemse%2Fbjx028&amp;pmid=28525560" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib102" class="citations"><div id="sref102" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref102" title="View in article">105.</a></div><div>Wark, B. ∙ Lundstrom, B.N. ∙ Fairhall, A.</div><div><strong>Sensory adaptation</strong></div><div><em>Curr. Opin. Neurobiol.</em> 2007; <strong>17</strong>:423-429</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.conb.2007.07.001" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_105_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-34948900353" target="_blank" aria-label="Open Scopus in new tab">Scopus (324)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17714934/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.conb.2007.07.001&amp;pmid=17714934" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib103" class="citations"><div id="sref103" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref103" title="View in article">106.</a></div><div>Weber, A.I. ∙ Krishnamurthy, K. ∙ Fairhall, A.L.</div><div><strong>Coding principles in adaptation</strong></div><div><em>Annu. Rev. Vis. Sci.</em> 2019; <strong>5</strong>:427-449</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1146/annurev-vision-091718-014818" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_106_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-85072266526" target="_blank" aria-label="Open Scopus in new tab">Scopus (18)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31283447/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1146%2Fannurev-vision-091718-014818&amp;pmid=31283447" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib104" class="citations"><div id="sref104" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref104" title="View in article">107.</a></div><div>Weinreb, C. ∙ Wolock, S. ∙ Klein, A.M.</div><div><strong>SPRING: A kinetic interface for visualizing high dimensional single-cell expression data</strong></div><div><em>Bioinformatics.</em> 2018; <strong>34</strong>:1246-1248</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btx792" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_107_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-85041411084" target="_blank" aria-label="Open Scopus in new tab">Scopus (112)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29228172/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtx792&amp;pmid=29228172" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib105" class="citations"><div id="sref105" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref105" title="View in article">108.</a></div><div>Williams, A.H. ∙ Kim, T.H. ∙ Wang, F. ...</div><div><strong>Unsupervised discovery of demixed, low-dimensional neural dynamics across multiple timescales through tensor component analysis</strong></div><div><em>Neuron.</em> 2018; <strong>98</strong>:1099-1115.e8</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_108_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=10.1016%2Fj.neuron.2018.05.015&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_108_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=10.1016%2Fj.neuron.2018.05.015&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_108_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-85047777491" target="_blank" aria-label="Open Scopus in new tab">Scopus (63)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29887338/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.neuron.2018.05.015&amp;pmid=29887338" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib106" class="citations"><div id="sref106" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref106" title="View in article">109.</a></div><div>Wilson, D.A.</div><div><strong>Habituation of odor responses in the rat anterior piriform cortex</strong></div><div><em>J.&nbsp;Neurophysiol.</em> 1998; <strong>79</strong>:1425-1440</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1152/jn.1998.79.3.1425" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_109_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-0031952787" target="_blank" aria-label="Open Scopus in new tab">Scopus (181)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/9497422/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1152%2Fjn.1998.79.3.1425&amp;pmid=9497422" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib107" class="citations"><div id="sref107" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref107" title="View in article">110.</a></div><div>Wilson, D.A.</div><div><strong>Comparison of odor receptive field plasticity in the rat olfactory bulb and anterior piriform cortex</strong></div><div><em>J.&nbsp;Neurophysiol.</em> 2000; <strong>84</strong>:3036-3042</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1152/jn.2000.84.6.3036" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11110830/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1152%2Fjn.2000.84.6.3036&amp;pmid=11110830" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib108" class="citations"><div id="sref108" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref108" title="View in article">111.</a></div><div>Wilson, D.A.</div><div><strong>Olfaction as a model system for the neurobiology of mammalian short-term habituation</strong></div><div><em>Neurobiol. Learn. Mem.</em> 2009; <strong>92</strong>:199-205</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.nlm.2008.07.010" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_111_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-67349191494" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/18678264/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.nlm.2008.07.010&amp;pmid=18678264" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib109" class="citations"><div id="sref109" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref109" title="View in article">112.</a></div><div>Wilson, C.D. ∙ Serrano, G.O. ∙ Koulakov, A.A. ...</div><div><strong>A primacy code for odor identity</strong></div><div><em>Nat. Commun.</em> 2017; <strong>8</strong>:1477</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41467-017-01432-4" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_112_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-85034246616" target="_blank" aria-label="Open Scopus in new tab">Scopus (56)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29133907/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41467-017-01432-4&amp;pmid=29133907" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib110" class="citations"><div id="sref110" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref110-1" title="View in article">113.</a></div><div>Wolf, F.A. ∙ Angerer, P. ∙ Theis, F.J.</div><div><strong>SCANPY: Large-scale single-cell gene expression data analysis</strong></div><div><em>Genome Biol.</em> 2018; <strong>19</strong>:15</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13059-017-1382-0" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_113_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-85041394976" target="_blank" aria-label="Open Scopus in new tab">Scopus (758)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29409532/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13059-017-1382-0&amp;pmid=29409532" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib111" class="citations"><div id="sref111" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref111-1" title="View in article">114.</a></div><div>Wu, Y.E. ∙ Pan, L. ∙ Zuo, Y. ...</div><div><strong>Detecting activated cell populations using single-cell RNA-seq</strong></div><div><em>Neuron.</em> 2017; <strong>96</strong>:313-329.e6</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_114_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=10.1016%2Fj.neuron.2017.09.026&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_114_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=10.1016%2Fj.neuron.2017.09.026&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_114_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-85031794820" target="_blank" aria-label="Open Scopus in new tab">Scopus (141)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29024657/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.neuron.2017.09.026&amp;pmid=29024657" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib112" class="citations"><div id="sref112" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref112" title="View in article">115.</a></div><div>Wu, Y. ∙ Ma, L. ∙ Duyck, K. ...</div><div><strong>A population of navigator neurons is essential for olfactory map formation during the critical period</strong></div><div><em>Neuron.</em> 2018; <strong>100</strong>:1066-1082.e6</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_115_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=10.1016%2Fj.neuron.2018.09.051&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_115_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=10.1016%2Fj.neuron.2018.09.051&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_115_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-85059320843" target="_blank" aria-label="Open Scopus in new tab">Scopus (9)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30482691/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.neuron.2018.09.051&amp;pmid=30482691" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib113" class="citations"><div id="sref113" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref113" title="View in article">116.</a></div><div>Xavier, A.M. ∙ Ludwig, R.G. ∙ Nagai, M.H. ...</div><div><strong>CD36 is expressed in a defined subpopulation of neurons in the olfactory epithelium</strong></div><div><em>Sci. Rep.</em> 2016; <strong>6</strong>:25507</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/srep25507" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_116_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-84966277054" target="_blank" aria-label="Open Scopus in new tab">Scopus (19)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27145700/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fsrep25507&amp;pmid=27145700" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib114" class="citations"><div id="sref114" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref114" title="View in article">117.</a></div><div>Yadon, C.A. ∙ Wilson, D.A.</div><div><strong>The role of metabotropic glutamate receptors and cortical adaptation in habituation of odor-guided behavior</strong></div><div><em>Learn. Mem.</em> 2005; <strong>12</strong>:601-605</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/lm.41405" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_117_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-28744459570" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16322361/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2Flm.41405&amp;pmid=16322361" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib115" class="citations"><div id="sref115" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref115" title="View in article">118.</a></div><div>Yap, E.-L. ∙ Greenberg, M.E.</div><div><strong>Activity-regulated transcription: Bridging the gap between neural activity and behavior</strong></div><div><em>Neuron.</em> 2018; <strong>100</strong>:330-348</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_118_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=10.1016%2Fj.neuron.2018.10.013&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_118_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=10.1016%2Fj.neuron.2018.10.013&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_118_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-85054841240" target="_blank" aria-label="Open Scopus in new tab">Scopus (137)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30359600/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.neuron.2018.10.013&amp;pmid=30359600" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib116" class="citations"><div id="sref116" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref116" title="View in article">119.</a></div><div>Zhang, X. ∙ Firestein, S.</div><div><strong>The olfactory receptor gene superfamily of the mouse</strong></div><div><em>Nat. Neurosci.</em> 2002; <strong>5</strong>:124-133</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nn800" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_119_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-0036163689" target="_blank" aria-label="Open Scopus in new tab">Scopus (642)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11802173/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnn800&amp;pmid=11802173" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib117" class="citations"><div id="sref117" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref117" title="View in article">120.</a></div><div>Zhang, J. ∙ Huang, G. ∙ Dewan, A. ...</div><div><strong>Uncoupling stimulus specificity and glomerular position in the mouse olfactory system</strong></div><div><em>Mol. Cell. Neurosci.</em> 2012; <strong>51</strong>:79-88</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.mcn.2012.08.006" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_120_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-84866146006" target="_blank" aria-label="Open Scopus in new tab">Scopus (25)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22926192/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.mcn.2012.08.006&amp;pmid=22926192" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib118" class="citations"><div id="sref118" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref118" title="View in article">121.</a></div><div>Zufall, F. ∙ Leinders-Zufall, T.</div><div><strong>The cellular and molecular basis of odor adaptation</strong></div><div><em>Chem. Senses.</em> 2000; <strong>25</strong>:473-481</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/chemse/25.4.473" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/10944513/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fchemse%2F25.4.473&amp;pmid=10944513" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib119" class="citations"><div id="sref119" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref119" title="View in article">122.</a></div><div>Zwicker, D. ∙ Murugan, A. ∙ Brenner, M.P.</div><div><strong>Receptor arrays optimized for natural odor statistics</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2016; <strong>113</strong>:5570-5575</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1600357113" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_122_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.022&amp;key=2-s2.0-84969872374" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27102871/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1600357113&amp;pmid=27102871" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div></div></div></section></div></section><div class="core-linked-content">
    

        


    

            

            

            <div data-widget-id="8633d695-b258-4908-b86f-e2c268ec3c6b" data-widget-name="UX3HTMLWidget"><script type="text/javascript">
if($('.content-navigation__body__title').length){
$('.content-navigation__body__title').each(function(){
var html=this.innerHTML;
var div = document.createElement("div");
div.innerHTML = html;
var text = div.textContent || div.innerText || "";
this.innerHTML=text;
});
}
</script></div>

            

        

    

        


    

            

            

            <div data-widget-id="393effff-5b44-4681-a9b4-4f029b3540b7" data-widget-name="UX3HTMLWidget"><style>
.linkback-ad .pb-ad {
    display: block;
   height: auto;
    margin-left: 0px;
    margin-right: auto;
}
</style></div>

            

        

    

        

</div></div></div><div class="core-collateral-aside"><section id="core-collateral-figures"><h2>Figures (14)<button class="js--openFigureViewer"><span>Figure Viewer</span></button></h2><div class="core-collateral-aside-wrapper"><div class="grid"><a class="figure__open__ctrl" href="#fig1" aria-label="Go to Figure&amp;nbsp;1"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.11.022/asset/5e8dc58a-bc0e-4528-bf87-ce55c278fed4/main.assets/gr1.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs1" aria-label="Go to Figure&amp;nbsp;S1"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.11.022/asset/8e871440-af04-4b66-88b7-e8ef0410af22/main.assets/figs1.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig2" aria-label="Go to Figure&amp;nbsp;2"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.11.022/asset/21aa6332-12b7-49a1-99c5-807c5cc3468f/main.assets/gr2.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs2" aria-label="Go to Figure&amp;nbsp;S2"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.11.022/asset/23fe8d03-ec8e-4594-8ea1-85673dc7a6da/main.assets/figs2.jpg" aria-hidden="true"></a></div><div id="collapsible-figures" data-expandable="item"><div class="content grid" style="display:none;" role="region" id="collapsible-figures_content"><a class="figure__open__ctrl" href="#fig3" aria-label="Go to Figure&amp;nbsp;3"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.11.022/asset/383db28a-790b-45b4-969c-7399942335b2/main.assets/gr3.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs3" aria-label="Go to Figure&amp;nbsp;S3"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.11.022/asset/12e7d552-6583-4708-8a1a-b228e4e281ca/main.assets/figs3.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig4" aria-label="Go to Figure&amp;nbsp;4"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.11.022/asset/7ac5b6c9-2170-4147-b3c2-b80e7808115e/main.assets/gr4.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs4" aria-label="Go to Figure&amp;nbsp;S4"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.11.022/asset/5ba6d8e4-6dc8-4d1f-9d56-06db1e1a68c8/main.assets/figs4.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig5" aria-label="Go to Figure&amp;nbsp;5"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.11.022/asset/c778f799-8294-4060-a727-85fa7364af42/main.assets/gr5.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs5" aria-label="Go to Figure&amp;nbsp;S5"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.11.022/asset/23938dac-554e-4d31-9861-02715dca58f3/main.assets/figs5.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig6" aria-label="Go to Figure&amp;nbsp;6"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.11.022/asset/79ac34c9-9c02-42a1-b475-25b93cb2def2/main.assets/gr6.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs6" aria-label="Go to Figure&amp;nbsp;S6"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.11.022/asset/94bdef7c-07ab-47dc-8b47-914b34ef41cc/main.assets/figs6.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig7" aria-label="Go to Figure&amp;nbsp;7"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.11.022/asset/11160ab1-5272-49e9-9661-cf78d2d34513/main.assets/gr7.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs7" aria-label="Go to Figure&amp;nbsp;S7"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.11.022/asset/cfad3233-f6e4-41d0-b5bb-a1bd18e36ff1/main.assets/figs7.jpg" aria-hidden="true"></a></div><button class="trigger" aria-controls="collapsible-figures_content" aria-expanded="false" tabindex="0" role="button"><span class="expand">Show all figures</span><span class="collapse">Hide figures</span><i class="icon-section_arrow_d" aria-hidden="true"></i></button></div></div></section>
    

        


    

            

            

            <section id="core-collateral-metrics" class="publication-list__axel-section"><h2>Article metrics</h2><div><ul class="publication-list rlist publication-list--list plum-x--lazy-load publication-list--axel" data-plum-x-url="/pb/widgets/ux3/publicationList/plumx?widgetId=66dd7e11-7768-4006-93d8-a72b60e2a264&amp;pbContext=%3Bproduct%3Aproduct%3Aelsevier%5C%3Aproduct%5C%3Acell%3Barticle%3Aarticle%3Apii%5C%3AS0092867421013374%3Bissue%3Aissue%3Apii%5C%3AS0092867420X00274%3Bwebsite%3Awebsite%3Acell-site%3Bctype%3Astring%3AJournal+Content%3Bjournal%3Ajournal%3Acell%3BpageGroup%3Astring%3AArticle+View%3Bpage%3Astring%3AShow+Full+Text%3BrequestedJournal%3Ajournal%3Acell%3Bwgroup%3Astring%3ADefault+Website+Group%3BsubPage%3Astring%3AFull+Text" data-pb-mode-enabled="false"><li aria-label="loading articles" role="alert" class="js--loading plum-x--spinner hidden"></li></ul></div></section>

            

        
<section id="core-collateral-supplementary"><h2>Supplemental information (5)</h2><div class="core-collateral-aside-wrapper"><div class="core-supplementary-material"><div class="core-link"><button class="downloadAllSupplementary"><span>Download all</span></button></div></div><div class="core-supplementary-material" data-category="file"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2021.11.022/attachment/1bf6643c-5639-4ea6-a15b-ab9397372a25/mmc1.csv" download="mmc1.csv"><span id="mmc1-link">File (11.80 KB)</span></a><div class="core-description">Table S1. Summary of single-cell RNA-seq datasets for olfactory sensory neurons used in this study, related to Figures 1, 2, 3, 4, 5, and 6</div></div></div><div class="core-supplementary-material" data-category="file"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2021.11.022/attachment/7347e101-4d48-4870-b3a0-5155460b91ad/mmc2.csv" download="mmc2.csv"><span id="mmc2-link">File (325.13 KB)</span></a><div class="core-description">Table S2. Loadings of 1,350 highly variable genes for 16 gene expression programs identified by consensus non-negative matrix factorization of the home cage dataset, related to Figure&nbsp;2</div></div></div><div class="core-supplementary-material" data-category="file"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2021.11.022/attachment/268c326a-9741-4a55-9092-151eae4468c6/mmc3.csv" download="mmc3.csv"><span id="mmc3-link">File (4.98 KB)</span></a><div class="core-description">Table S3. 117 environmental state neuronal genes, including 73 functional genes that may regulate sensory responses, related to Figure&nbsp;4</div></div></div><div class="core-supplementary-material" data-category="file"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2021.11.022/attachment/b0625282-5363-4fa7-b60c-52a6d021aa1e/mmc4.csv" download="mmc4.csv"><span id="mmc4-link">File (2.18 MB)</span></a><div class="core-description">Table S4. Changes in <i>Z</i>-scored expression of 117 environmental state neuronal genes across conditions, for each OSN subtype, related to Figure&nbsp;4</div></div></div><div class="core-supplementary-material" data-category="file"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2021.11.022/attachment/7b2c1591-af79-4def-8266-56b347729f67/mmc5.csv" download="mmc5.csv"><span id="mmc5-link">File (61.09 KB)</span></a><div class="core-description">Table S5. 472 genes used to calculate activation score that were acutely odor-sensitive in responsive OSNs, related to Figure&nbsp;5</div></div></div></div></section>
    

        


    

            

            

            <section id="core-collateral-relatedArticles" class="axel-related-articles"><h2 id="relatedArticles" class="top axel-related-articles__title" data-title="Related articles">Related Articles</h2><div data-pb-widget-type="ux3-relatedArticles" data-widget-id="8d3844d1-450e-497c-ab75-a63fc41750ac" data-widget-name="AtyponTempUX3RelatedArticlesWidget" data-article-pii="S0092867421013374" class="ux3DeferredOnScrollWidget axel-related-articles__body ux3DeferredLoaded" data-show-more="true">We haven't found any related articles.</div></section>

            

        
</div></div><aside data-core-aside="right-rail"><div class="inner-wrapper">

    

        


    

            

            

            <div data-widget-id="dca7470d-9085-4d21-a6f4-a66397718146" data-widget-name="UX3HTMLWidget"><input type="hidden" id="gptPage" value="cell-videos"></div>

            

        











        <div data-widget-def="literatumAd" data-widget-id="481d10c2-bbe0-470e-a5ba-207a81ecc8dc" class="adplaceholders" style="right: 0px; position: static;">
        
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd alignCenter  widget-verticalAd" id="481d10c2-bbe0-470e-a5ba-207a81ecc8dc">
                
                
                    
                

                <div class="wrapped " id="Marketplace-Recs">
                    
                    <div class="widget-body body body-verticalAd "><div class="verticalAdContainer">
    <div class="verticalAdLabelContainer">
        <div class="verticalAdLabel">
                <img src="/templates/jsp/_style2/_marlin/images/bg_adVert.gif" alt="Advertisement">
        </div>
    </div>
    <div class="verticalAd">
        





        <div class="pb-ad">
            <script src="//cdn.pbgrd.com/core-cell.js" async=""></script>
<div id="pbgrd-skyr-c"><div id="pbgrd-skyr" style="width: inherit; position: relative; background: transparent;"></div></div>
        </div>
    

    </div>
</div></div>
                </div>
                </div>

        

        </div>
    

</div></aside><div class="core-authors-details" style="display:none"><section class="core-authors"></section></div></article><div data-extent="article-wrapper"><div class="core-container"></div></div><script id="axel-publication-metadata" type="application/json">{"doi":"10.1016/j.cell.2021.11.022","data-is-retraction":false,"data-is-retired":false}
</script><script type="text/javascript" data-ot-ignore="data-ot-ignore" defer="defer" src="https://static.addtoany.com/menu/page.js" class="optanon-category-C0004 ot-vscat-C0004"></script><template id="figure_nav_template"><a href="#" class="open-in-viewer btn btn-dark figure-pop-btn"><i aria-hidden="true" class="icon-export_citation"></i><span>Open in viewer</span></a></template><div id="figure-viewer" role="dialog" aria-modal="true" aria-label="Figure Viewer" class="showCaption showThumbstrip figure-viewer"><nav class="nav-bar figure-viewer__top"><a href="#" title="Close Figure Viewer" role="button" class="figure-viewer__close"><i aria-hidden="true" class="icon-tools_close"></i></a><div class="figure-viewer__title">A transcriptional rheostat couples past activity to future sensory responses</div></nav><div class="frames"><ul class="frames__list"><li><div class="thumbnail__image"><img id="gr1" src="/cms/10.1016/j.cell.2021.11.022/asset/5e8dc58a-bc0e-4528-bf87-ce55c278fed4/main.assets/gr1.jpg" height="527" width="662" aria-labelledby="fig1-title fig1-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.022/asset/53c5d6ca-f460-4fab-a293-a9cd15be464d/main.assets/gr1_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs1" src="/cms/10.1016/j.cell.2021.11.022/asset/8e871440-af04-4b66-88b7-e8ef0410af22/main.assets/figs1.jpg" height="832" width="657" aria-labelledby="figs1-title figs1-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.022/asset/4aa93467-d7ff-4b38-8b1a-2053a78eab4a/main.assets/figs1_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr2" src="/cms/10.1016/j.cell.2021.11.022/asset/21aa6332-12b7-49a1-99c5-807c5cc3468f/main.assets/gr2.jpg" height="415" width="769" aria-labelledby="fig2-title fig2-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.022/asset/08eeda0b-eb45-4077-ba2c-b7e9e368e6ff/main.assets/gr2_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs2" src="/cms/10.1016/j.cell.2021.11.022/asset/23fe8d03-ec8e-4594-8ea1-85673dc7a6da/main.assets/figs2.jpg" height="864" width="657" aria-labelledby="figs2-title figs2-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.022/asset/ce789880-7a50-4357-843f-dccc31404e9f/main.assets/figs2_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr3" src="/cms/10.1016/j.cell.2021.11.022/asset/383db28a-790b-45b4-969c-7399942335b2/main.assets/gr3.jpg" height="845" width="654" aria-labelledby="fig3-title fig3-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.022/asset/7df42adf-e1d0-4a14-8ac3-9eec2d53102c/main.assets/gr3_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs3" src="/cms/10.1016/j.cell.2021.11.022/asset/12e7d552-6583-4708-8a1a-b228e4e281ca/main.assets/figs3.jpg" height="699" width="657" aria-labelledby="figs3-title figs3-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.022/asset/072e79d1-1eef-4c7c-bb06-1c28c5e29095/main.assets/figs3_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr4" src="/cms/10.1016/j.cell.2021.11.022/asset/7ac5b6c9-2170-4147-b3c2-b80e7808115e/main.assets/gr4.jpg" height="784" width="659" aria-labelledby="fig4-title fig4-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.022/asset/cea09ef2-7df1-404a-8174-414ae324afb5/main.assets/gr4_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs4" src="/cms/10.1016/j.cell.2021.11.022/asset/5ba6d8e4-6dc8-4d1f-9d56-06db1e1a68c8/main.assets/figs4.jpg" height="569" width="657" aria-labelledby="figs4-title figs4-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.022/asset/f656a5ec-a595-4af7-8478-5b6d198e77c0/main.assets/figs4_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr5" src="/cms/10.1016/j.cell.2021.11.022/asset/c778f799-8294-4060-a727-85fa7364af42/main.assets/gr5.jpg" height="426" width="657" aria-labelledby="fig5-title fig5-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.022/asset/c09e0323-761b-4bc6-95e5-3c611584159d/main.assets/gr5_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs5" src="/cms/10.1016/j.cell.2021.11.022/asset/23938dac-554e-4d31-9861-02715dca58f3/main.assets/figs5.jpg" height="832" width="657" aria-labelledby="figs5-title figs5-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.022/asset/0e74ed20-4fda-4990-8300-0c5b2afc7a37/main.assets/figs5_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr6" src="/cms/10.1016/j.cell.2021.11.022/asset/79ac34c9-9c02-42a1-b475-25b93cb2def2/main.assets/gr6.jpg" height="817" width="657" aria-labelledby="fig6-title fig6-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.022/asset/11c2fa33-f387-4576-85c1-2b49023671a5/main.assets/gr6_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs6" src="/cms/10.1016/j.cell.2021.11.022/asset/94bdef7c-07ab-47dc-8b47-914b34ef41cc/main.assets/figs6.jpg" height="787" width="657" aria-labelledby="figs6-title figs6-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.022/asset/3390e7f4-192b-40c5-8a4e-132582c0cfe7/main.assets/figs6_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr7" src="/cms/10.1016/j.cell.2021.11.022/asset/11160ab1-5272-49e9-9661-cf78d2d34513/main.assets/gr7.jpg" height="710" width="657" aria-labelledby="fig7-title fig7-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.022/asset/71fc961d-9b67-4b39-a030-034c586193ba/main.assets/gr7_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs7" src="/cms/10.1016/j.cell.2021.11.022/asset/cfad3233-f6e4-41d0-b5bb-a1bd18e36ff1/main.assets/figs7.jpg" height="562" width="657" aria-labelledby="figs7-title figs7-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.022/asset/a99c68d8-8f4a-4174-b172-974b118f45da/main.assets/figs7_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li></ul><a href="#" title="Previous Frame" class="frameButton previousFrameButton frames__prev"><!--span Previous Frame--><i class="icon-arrow_l"></i></a><a href="#" title="Next Frame" class="frameButton nextFrameButton frames__next"><!--span Next Frame--><i class="icon-arrow_r"></i></a></div><div class="caption"><div class="figure-viewer__caption-body"></div></div><div class="toolbar figure-viewer__toolbar"><span class="figure-viewer__counter"></span><a href="#" class="button toggle-caption-button figure-viewer__caption__ctrl"><i aria-hidden="true" class="icon icon-info"></i><span class="label">Hide Caption</span></a><a href="#" class="button download-options-button"><i aria-hidden="true" class="icon icon-gizmo-folder-in download-size-icon"></i><span class="label">Download</span></a><a href="#figures" class="button see-figures figure-viewer__see-figure__ctrl"><i aria-hidden="true" class="icon icon-gizmo-image_figure viewer-size-icon"></i><span class="label">See figure in Article</span></a></div><div class="thumbstrip"><a href="#" class="toggleButton thumbstrip__ctrl"><span>Toggle Thumbstrip</span></a><a href="#" title="Previous Thumbnails" class="thumbstripButton previousThumbstripButton thumbstrip__prev"><i aria-hidden="true" class="icon-arrow_l"></i></a><a href="#" title="Next Thumbnails" class="thumbstripButton nextThumbstripButton thumbstrip__next"><i aria-hidden="true" class="icon-arrow_r"></i></a><div class="thumbstrip__wrapper"><ul class="rlist figure-viewer__thumbslist"><li><div class="thumbnail__label">Figure&nbsp;1</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr1" src="/cms/10.1016/j.cell.2021.11.022/asset/5e8dc58a-bc0e-4528-bf87-ce55c278fed4/main.assets/gr1.jpg" height="527" width="662" aria-labelledby="fig1-title fig1-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.022/asset/53c5d6ca-f460-4fab-a293-a9cd15be464d/main.assets/gr1_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S1</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs1" src="/cms/10.1016/j.cell.2021.11.022/asset/8e871440-af04-4b66-88b7-e8ef0410af22/main.assets/figs1.jpg" height="832" width="657" aria-labelledby="figs1-title figs1-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.022/asset/4aa93467-d7ff-4b38-8b1a-2053a78eab4a/main.assets/figs1_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;2</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr2" src="/cms/10.1016/j.cell.2021.11.022/asset/21aa6332-12b7-49a1-99c5-807c5cc3468f/main.assets/gr2.jpg" height="415" width="769" aria-labelledby="fig2-title fig2-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.022/asset/08eeda0b-eb45-4077-ba2c-b7e9e368e6ff/main.assets/gr2_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S2</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs2" src="/cms/10.1016/j.cell.2021.11.022/asset/23fe8d03-ec8e-4594-8ea1-85673dc7a6da/main.assets/figs2.jpg" height="864" width="657" aria-labelledby="figs2-title figs2-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.022/asset/ce789880-7a50-4357-843f-dccc31404e9f/main.assets/figs2_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;3</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr3" src="/cms/10.1016/j.cell.2021.11.022/asset/383db28a-790b-45b4-969c-7399942335b2/main.assets/gr3.jpg" height="845" width="654" aria-labelledby="fig3-title fig3-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.022/asset/7df42adf-e1d0-4a14-8ac3-9eec2d53102c/main.assets/gr3_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S3</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs3" src="/cms/10.1016/j.cell.2021.11.022/asset/12e7d552-6583-4708-8a1a-b228e4e281ca/main.assets/figs3.jpg" height="699" width="657" aria-labelledby="figs3-title figs3-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.022/asset/072e79d1-1eef-4c7c-bb06-1c28c5e29095/main.assets/figs3_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;4</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr4" src="/cms/10.1016/j.cell.2021.11.022/asset/7ac5b6c9-2170-4147-b3c2-b80e7808115e/main.assets/gr4.jpg" height="784" width="659" aria-labelledby="fig4-title fig4-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.022/asset/cea09ef2-7df1-404a-8174-414ae324afb5/main.assets/gr4_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S4</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs4" src="/cms/10.1016/j.cell.2021.11.022/asset/5ba6d8e4-6dc8-4d1f-9d56-06db1e1a68c8/main.assets/figs4.jpg" height="569" width="657" aria-labelledby="figs4-title figs4-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.022/asset/f656a5ec-a595-4af7-8478-5b6d198e77c0/main.assets/figs4_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;5</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr5" src="/cms/10.1016/j.cell.2021.11.022/asset/c778f799-8294-4060-a727-85fa7364af42/main.assets/gr5.jpg" height="426" width="657" aria-labelledby="fig5-title fig5-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.022/asset/c09e0323-761b-4bc6-95e5-3c611584159d/main.assets/gr5_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S5</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs5" src="/cms/10.1016/j.cell.2021.11.022/asset/23938dac-554e-4d31-9861-02715dca58f3/main.assets/figs5.jpg" height="832" width="657" aria-labelledby="figs5-title figs5-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.022/asset/0e74ed20-4fda-4990-8300-0c5b2afc7a37/main.assets/figs5_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;6</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr6" src="/cms/10.1016/j.cell.2021.11.022/asset/79ac34c9-9c02-42a1-b475-25b93cb2def2/main.assets/gr6.jpg" height="817" width="657" aria-labelledby="fig6-title fig6-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.022/asset/11c2fa33-f387-4576-85c1-2b49023671a5/main.assets/gr6_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S6</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs6" src="/cms/10.1016/j.cell.2021.11.022/asset/94bdef7c-07ab-47dc-8b47-914b34ef41cc/main.assets/figs6.jpg" height="787" width="657" aria-labelledby="figs6-title figs6-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.022/asset/3390e7f4-192b-40c5-8a4e-132582c0cfe7/main.assets/figs6_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;7</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr7" src="/cms/10.1016/j.cell.2021.11.022/asset/11160ab1-5272-49e9-9661-cf78d2d34513/main.assets/gr7.jpg" height="710" width="657" aria-labelledby="fig7-title fig7-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.022/asset/71fc961d-9b67-4b39-a030-034c586193ba/main.assets/gr7_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S7</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs7" src="/cms/10.1016/j.cell.2021.11.022/asset/cfad3233-f6e4-41d0-b5bb-a1bd18e36ff1/main.assets/figs7.jpg" height="562" width="657" aria-labelledby="figs7-title figs7-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.022/asset/a99c68d8-8f4a-4174-b172-974b118f45da/main.assets/figs7_lrg.jpg" loading="lazy" tabindex="0"></div></div></li></ul></div></div><div class="download figure-viewer__download"><div class="pointer"></div><ul class="rlist downlad__wrapper"><li class="hidden-xs hidden-sm"><a href="/action/downloadFigures?pii=e00489&amp;id=" class="downloadAllButton"><i class="icon icon-gizmo-ppt figure-viewer__download__viewer-size-icon figure-viewer__download__viewer-size-icon--ppt"></i><span class="figure-viewer__download__text"> Download .PPT</span></a></li></ul></div></div><template id="toCitationLink" data-citation="Citation" data-footnote="Footnote"><div class="to-citation__wrapper"><a class="to-citation"><i aria-hidden="true" class="icon-subdirectory_arrow_left"></i><span>Go to</span></a></div></template><template id="toCitationButton" data-citation="Citation" data-footnote="Footnote"><div class="to-citation__wrapper"><button class="to-citation"><i aria-hidden="true" class="icon-subdirectory_arrow_left"></i><span>Go to</span></button></div></template><template id="toCitationAccordion" data-collapse-citation="Hide Citations" data-expand-citation="Show Citations" data-collapse-footnote="Hide Footnotes" data-expand-footnote="Show Footnotes"><div class="to-citation__wrapper"><button aria-controls="aria-controls" aria-expanded="false" aria-label="Toggle citations menu" class="to-citation__toggle"><i aria-hidden="true" class="icon-plus"></i><span data-expand-title="data-expand-title" data-collapsed-title="data-collapsed-title" class="accordion__toggle__title"></span></button><div role="menu" aria-label="links" class="to-citation__accordion no-separator"><ul></ul></div></div></template><template id="toCitationAccordionItem"><li><a class="to-citation"><i aria-hidden="true" class="icon-subdirectory_arrow_left"></i><span></span></a></li></template><template id="citations_truncate_template"><div class="citations-truncation"><button data-label-expand="Show all references" data-label-collapse="Show fewer" data-label-remaining="references remaining" class="btn btn--inverse"><span>Show all references</span></button></div></template><template id="collateral_texts_template" data-do="Digital Object" data-figures="Figures" data-metrics="Article Metrics" data-related="Related articles" data-supplementary="Supplemental information"></template><template id="collapsible_authors_template"><button aria-expanded="false" data-expandable="all" data-label-expand="Expand All" data-label-collapse="Collapse All" class="collateral-contributors-control"><span>Expand All</span></button></template>
<template id="collapsible_tables_collapse_template"><div class="collapsible-figure-btn__wrapper expanded"><button aria-expanded="true" class="btn collapsible-figure-btn btn--inverse"><i aria-hidden="true" class="icon-arrow-up"></i><span class="text-uppercase">Collapse</span></button></div></template>
<template id="collapsible_tables_expand_template"><div class="collapsible-figure-btn__wrapper collapsed"><button aria-expanded="false" class="btn collapsible-figure-btn btn--inverse"><i aria-hidden="true" class="icon-arrow-down"></i><span class="text-uppercase">Expand Table</span></button></div></template><template id="authorsAffiliationsLink"><a href="#tab-contributors" class="to-authors-affiliations">Authors Info &amp; Affiliations</a></template></div>
                </div>
                </div>

        

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none  widget-  widget-compact-all" id="756cf18d-e910-4468-936c-0225ae7c669f">
                
                
                    
                

                <div class="wrapped " id="ad for nucleus-single-molecule-technologies">
                    
                    <div class="widget-body body body-  body-compact-all"><div class="pb-ad">
            <script type="text/javascript">
URL_var = 'https://www.cell.com/cell/fulltext/S0092-8674(21)01337-4';

if(URL_var.indexOf("S0165-6147(17)30205-5") > -1
|| URL_var.indexOf("S0167-7799(18)30204-X") > -1 
|| URL_var.indexOf("S0092-8674(18)31649-0") > -1 
|| URL_var.indexOf("S0092-8674(19)30404-0") > -1 
|| URL_var.indexOf("S1097-2765(18)30756-1") > -1 
|| URL_var.indexOf("S0092-8674(18)31452-1") > -1 
|| URL_var.indexOf("S1097-2765(19)30175-3") > -1 
|| URL_var.indexOf("S0092-8674(18)30852-3") > -1 
|| URL_var.indexOf("S0006-3495(18)30295-9") > -1 
|| URL_var.indexOf("S1097-2765(19)30063-2") > -1 
|| URL_var.indexOf("S0006-3495(17)33208-3") > -1 
|| URL_var.indexOf("S2211-1247(18)31639-5") > -1 
|| URL_var.indexOf("S1097-2765(18)30978-X") > -1 
)
    {
document.write('<input type="hidden" id="gpttc" value="nucleus-single-molecule-technologies" class="gpt-Target">')
}
</script>
        </div></div>
                </div>
                </div>

        

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none  widget-" id="754b1780-1117-4371-9285-9cd7020963d5">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body- "><div class="pb-ad">
            <script type="text/javascript">
URL_var = 'https://www.cell.com/cell/fulltext/S0092-8674(21)01337-4';

if(URL_var.indexOf("S1535-6108(19)30104-7") > -1
|| URL_var.indexOf("S0092-8674(18)31394-1") > -1
|| URL_var.indexOf("S1074-7613(19)30033-0") > -1
|| URL_var.indexOf("S1535-6108(18)30473-2") > -1
|| URL_var.indexOf("S0092-8674(19)30154-0") > -1
|| URL_var.indexOf("S1550-4131(18)30178-5") > -1
|| URL_var.indexOf("S1535-6108(19)30242-9") > -1
|| URL_var.indexOf("S1074-7613(18)30431-X") > -1
|| URL_var.indexOf("S0092-8674(19)30618-X") > -1
|| URL_var.indexOf("S1535-6108(18)30472-0") > -1
|| URL_var.indexOf("S1535-6108(18)30120-X") > -1
|| URL_var.indexOf("S0092-8674(18)31247-9") > -1
|| URL_var.indexOf("S1074-7613(18)30085-2") > -1
|| URL_var.indexOf("S1074-7613(18)30080-3") > -1
|| URL_var.indexOf("S1471-4906(18)30210-2") > -1
|| URL_var.indexOf("S1550-4131(19)30304-3") > -1
|| URL_var.indexOf("S1934-5909(18)30599-X") > -1
|| URL_var.indexOf("S2405-8033(19)30024-X") > -1
|| URL_var.indexOf("S1535-6108(18)30109-0") > -1
|| URL_var.indexOf("S0962-8924(18)30127-2") > -1
)
    {
document.write('<input type="hidden" id="gpttc" value="nucleus-tumor-immune-microenvironment" class="gpt-Target">')
}
</script>
        </div></div>
                </div>
                </div>

        

    

        


    

            

            

            <div data-widget-id="9c9ff7a1-a035-49f6-a336-600c9419c518" data-widget-name="UX3HTMLWidget"><!-- Start of HubSpot Embed Code --> 

<script type="text/plain" class="optanon-category-4" id="hs-script-loader" async="" defer="" src="//js.hs-scripts.com/519224.js"></script> 

<!-- End of HubSpot Embed Code --></div>

            

        

    

        


    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget sessionValidationWidget none  widget-none  widget-compact-all" id="0257aef7-6074-4898-a989-1c3f693225a2">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none  body-compact-all"><script>
    $.ajax({
        url: "https://www.cell.com/ssoSessionCheck",
        type: "GET",
        dataType: "jsonp",
        success: function (data) {
            if(data.RefreshSession) {
                location.reload();
            }
        },
        error : function (){
            console.warn("can't validate CSAS session.");
        }
    });
</script></div>
                </div>
                </div>

        

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget csasSsoUrls none  widget-none  widget-compact-all" id="948bc5c4-8b0d-47be-b5ad-6570559802d1">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none  body-compact-all"><!-- clear SSO URLs after all iFrames are rendered or after passing a time out-->
<script>
    setTimeout(clearSsoUrlsFun,10000);
    var iFramesList = document.getElementById("sso-urls-div").getElementsByTagName("iframe");
    var renderedIFramesCount = 0;
    var numberOfIFrames = iFramesList.length;
    var skipRefresh = false;
    for (var i = 0; i < iFramesList.length; i++) {
        var iFrame = iFramesList[i];
        iFrame.addEventListener("load", function() {
            renderedIFramesCount = renderedIFramesCount + 1;
            if (renderedIFramesCount >= numberOfIFrames)
            {
                clearSsoUrlsFun();
            }
        });
    }

    function clearSsoUrlsFun() {
        if (skipRefresh == false && numberOfIFrames > 0)
        {
            skipRefresh = true;
            $.ajax({
                url: "https://www.cell.com/action/clearSsoUrls",
                type: "GET",
                dataType: "jsonp",
                error : errorCallingAjx
            });
        }
    }

    function errorCallingAjx(){
        console.log("can't delete sso urls .")
    }
</script></div>
                </div>
                </div>

        










        <div data-widget-def="ux3-layout-widget" data-widget-id="d95b4a46-dc4d-4ca8-83f9-8b6724c649ec" class="ux3 ">
        
    

        


    

            

            

            <div data-widget-id="d95b4a46-dc4d-4ca8-83f9-8b6724c649ec" data-widget-name="ux3-layout-widget" class="fonts-loaded"><div class="footer clearfix">
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none adplaceholder adplaceholder--header widget-horizontalAd" id="a07ee92b-3942-49f8-9186-a2a08fc3fc56">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-horizontalAd "><div class="horizontalAdContainer">
    <div class="horizontalAdLabelContainer">
        <div class="horizontalAdLabel">
                <img src="/templates/jsp/_style2/_marlin/images/bg_adHoriz.gif" alt="Advertisement">
        </div>
    </div>
    <div class="horizontalAd">
        





        <div class="pb-ad">
            <!-- Beginning Async AdSlot 4 for Ad unit els.cellpress/cell  ### size: [[728,90]] -->
<div class="gpt-ad" data-adsize="cell-leaderboard" data-page="" data-site="cell" data-pos="4"></div>
<!-- End AdSlot 4 -->
        </div>
    

    </div>
</div></div>
                </div>
                </div>

        

    

        


    

            

            

            <div data-widget-id="23703bbc-af46-4f3f-a5bb-dd52891ff0ff" data-widget-name="UX3HTMLWidget"><div class="footer__top">
    <div class="footer__logo centered"><a href="/home" title="Cell Press"><!-- Generator: Adobe Illustrator 25.2.1, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 300 72" style="enable-background:new 0 0 300 72;" xml:space="preserve">
<style type="text/css">
	.st0{fill:#007DBC;}
</style>
<g>
	<path class="st0" d="M39.4,7.5c-2.3-0.3-4.4,1.1-4.9,3.3l-1.8,7.4c-0.4,1.8-1.9,3.1-3.8,3.3l-10.5,1.2C7.1,24-1,34.3,0.7,45.6
		c1.3,9,8.6,15.7,17.2,17c2.3,0.3,4.4-1.1,4.9-3.3l1.8-7.4c0.4-1.8,1.9-3.1,3.8-3.3l10.5-1.2c11.3-1.2,19.4-11.6,17.8-22.9
		C55.3,15.4,48.1,8.8,39.4,7.5z M37.1,34l-6.6,0.8c-1.1,0.1-2,0.9-2.3,2l-1.9,8.1c-0.6,2.4-2.5,4.2-5,4.5c-3.8,0.4-7.1-2.4-7.4-6.2
		c-0.3-3.7,2.6-6.8,6.3-7.2l6.6-0.8c1.1-0.1,2-0.9,2.3-2l1.9-8.1c0.6-2.4,2.5-4.2,5-4.5c3.8-0.4,7.1,2.4,7.4,6.2
		C43.7,30.5,40.8,33.6,37.1,34z"></path>
	<rect x="153.1" y="13.9" class="st0" width="8.2" height="42.3"></rect>
	<path class="st0" d="M90.2,20.4c7.3,0,8.6,4.8,9.4,7.3h8.8c-0.5-6.2-5.6-14.9-18.2-14.9c-10.9,0-19.4,7.5-19.4,22.3
		c0,14.6,8.1,22.2,19.3,22.2c10.6,0,16.6-6.4,18.4-15.6h-8.8c-1.1,5.2-4.5,7.9-9.4,7.9c-7.5,0-10.5-6.8-10.5-14.4
		C79.7,23.3,85.9,20.4,90.2,20.4z"></path>
	<path class="st0" d="M125.2,24c-13.7,0-15,12.1-15,15.8c0,12.9,6.8,17.1,15.6,17.1c5.4,0,11.6-2.5,13.8-10H131
		c-0.7,2.2-3,3.4-5.3,3.4c-6.6,0-7.1-5.3-7.3-7.7h21.5v-1.5C139.9,27,131.3,24,125.2,24z M118.6,37.4c0.5-4.1,2.5-6.6,6.4-6.6
		c2.6,0,5.9,1.5,6.4,6.6H118.6z"></path>
	<rect x="142" y="13.9" class="st0" width="8.2" height="42.3"></rect>
	<path class="st0" d="M174,13.8h-9.8V21h7.2c4.3,0,8.2,0.9,8.2,6.4c0,5.4-3.9,6.4-8.2,6.4h-7.2V41h9.8c10.6,0,14.7-6.7,14.7-13.6
		C188.7,20.5,184.6,13.8,174,13.8z"></path>
	<path class="st0" d="M210.5,24.9c-0.5-0.2-1.2-0.3-1.8-0.3c-4.1,0-8.3,2.7-9.8,6.5h-0.1v-5.7h-8v30.7h8.4V42.3
		c0-5.4,2.1-9.9,8.3-9.9c1,0,2.3,0.1,3,0.3V24.9z"></path>
	<path class="st0" d="M225.6,24.6c-9.3,0-15.7,7-15.7,16.2c0,9.5,6,16.1,15.7,16.1c6.9,0,12-3.1,14.4-10.3h-7.4
		c-0.5,1.9-3.3,4-6.6,4c-4.7,0-7.3-2.4-7.5-7.8h22.1C241.1,33.4,236,24.6,225.6,24.6z M218.4,37.5c0.1-2.4,1.7-6.5,7-6.5
		c4.1,0,5.9,2.3,6.7,6.5H218.4z"></path>
	<path class="st0" d="M260.3,37.7c-4.8-1.1-9.6-1.5-9.6-4.6c0-2.5,2.8-2.9,4.7-2.9c3,0,5.5,0.9,5.8,4.2h8c-0.7-7.5-6.8-9.8-13.5-9.8
		c-6.6,0-13.4,2-13.4,9.9c0,5.4,4.9,7.1,9.7,8.2c6.1,1.3,9.6,2.2,9.6,4.8c0,3-3.1,3.9-5.5,3.9c-3.3,0-6.4-1.5-6.5-5.2h-8
		C242,54.3,248.9,57,256,57c7,0,14-2.6,14-10.7C270.1,40.6,265.2,38.8,260.3,37.7z"></path>
	<path class="st0" d="M289.8,37.7c-4.8-1.1-9.6-1.5-9.6-4.6c0-2.5,2.8-2.9,4.7-2.9c3,0,5.5,0.9,5.8,4.2h8c-0.7-7.5-6.8-9.8-13.5-9.8
		c-6.6,0-13.4,2-13.4,9.9c0,5.4,4.9,7.1,9.7,8.2c6.1,1.3,9.6,2.2,9.6,4.8c0,3-3.1,3.9-5.5,3.9c-3.3,0-6.4-1.5-6.5-5.2h-8
		c0.4,8.1,7.3,10.8,14.4,10.8c7,0,14-2.6,14-10.7C299.6,40.6,294.7,38.8,289.8,37.7z"></path>
	<path class="st0" d="M34.1,57.6c0.3-0.2,0.4-0.6,0.4-1c0-0.5-0.2-0.9-0.5-1.1c-0.3-0.2-0.8-0.3-1.4-0.3h-1.9v5.1h0.8V58h0.8
		l1.3,2.2h0.9l-1.4-2.2C33.6,57.9,33.9,57.8,34.1,57.6z M32.5,57.3h-0.9v-1.5h1c0.1,0,0.3,0,0.4,0c0.1,0,0.3,0,0.4,0.1
		c0.1,0,0.2,0.1,0.3,0.2c0.1,0.1,0.1,0.2,0.1,0.4c0,0.2,0,0.3-0.1,0.4c-0.1,0.1-0.2,0.2-0.3,0.2c-0.1,0.1-0.2,0.1-0.4,0.1
		C32.8,57.3,32.6,57.3,32.5,57.3z"></path>
	<path class="st0" d="M35.6,54.2c-0.4-0.4-0.9-0.7-1.4-0.9c-0.5-0.2-1.1-0.3-1.7-0.3c-0.6,0-1.2,0.1-1.7,0.3c-0.6,0.2-1,0.5-1.4,0.9
		c-0.4,0.4-0.7,0.9-1,1.5c-0.2,0.6-0.4,1.2-0.4,1.9c0,0.7,0.1,1.3,0.3,1.8c0.2,0.6,0.5,1,0.9,1.5c0.4,0.4,0.9,0.7,1.4,1
		c0.6,0.2,1.2,0.4,1.8,0.4c0.6,0,1.2-0.1,1.7-0.3c0.5-0.2,1-0.5,1.4-0.9c0.4-0.4,0.7-0.9,1-1.5c0.2-0.6,0.4-1.2,0.4-1.9
		c0-0.7-0.1-1.3-0.4-1.9C36.4,55.1,36,54.6,35.6,54.2z M35.7,59.1c-0.2,0.5-0.4,0.9-0.8,1.2c-0.3,0.3-0.7,0.6-1.1,0.8
		c-0.4,0.2-0.9,0.3-1.4,0.3c-0.6,0-1-0.1-1.5-0.3c-0.4-0.2-0.8-0.5-1.1-0.8c-0.3-0.3-0.5-0.7-0.7-1.2c-0.2-0.5-0.2-1-0.2-1.5
		c0-0.6,0.1-1.1,0.3-1.5c0.2-0.5,0.4-0.9,0.8-1.2c0.3-0.3,0.7-0.6,1.1-0.8c0.4-0.2,0.9-0.3,1.4-0.3c0.5,0,1,0.1,1.4,0.3
		c0.4,0.2,0.8,0.4,1.1,0.8c0.3,0.3,0.6,0.7,0.8,1.2c0.2,0.5,0.3,1,0.3,1.5C36,58.1,35.9,58.6,35.7,59.1z"></path>
</g>
</svg></a></div>
    <div class="footer__social-links">
        <ul class="rlist rlist--inline centered">
            <li><a href="http://on.fb.me/1tgHakh" title="Facebook" target="_blank"><i aria-hidden="true" class="icon-Icon_Facebook"></i></a></li>

            <li><a href="https://www.instagram.com/cellpress/" title="Instagram" target="_blank"><svg width="18px" height="18px" viewBox="0 0 18 18" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-147.000000, -16.000000)" fill-rule="nonzero" fill="#969696"> <g id="Instagram-Logo" transform="translate(147.000000, 16.000000)"> <path d="M9,1.62580645 C11.4096774,1.62580645 11.7,1.62580645 12.6290323,1.68387097 C13.5,1.71290323 13.9935484,1.85806452 14.3129032,2.00322581 C14.7193548,2.17741935 15.0387097,2.3516129 15.3580645,2.67096774 C15.6774194,2.99032258 15.8806452,3.28064516 16.0258065,3.71612903 C16.1419355,4.03548387 16.2870968,4.5 16.3451613,5.4 C16.3741935,6.35806452 16.4032258,6.61935484 16.4032258,9.02903226 C16.4032258,11.4387097 16.4032258,11.7290323 16.3451613,12.6580645 C16.316129,13.5290323 16.1709677,14.0225806 16.0258065,14.3419355 C15.8516129,14.7483871 15.6774194,15.0677419 15.3580645,15.3870968 C15.0387097,15.7064516 14.7483871,15.9096774 14.3129032,16.0548387 C13.9935484,16.1709677 13.5290323,16.316129 12.6290323,16.3741935 C11.6709677,16.4032258 11.4096774,16.4322581 9,16.4322581 C6.59032258,16.4322581 6.3,16.4322581 5.37096774,16.3741935 C4.5,16.3451613 4.00645161,16.2 3.68709677,16.0548387 C3.28064516,15.8806452 2.96129032,15.7064516 2.64193548,15.3870968 C2.32258065,15.0677419 2.11935484,14.7774194 1.97419355,14.3419355 C1.85806452,14.0225806 1.71290323,13.5580645 1.65483871,12.6580645 C1.62580645,11.7 1.62580645,11.4096774 1.62580645,9 C1.62580645,6.59032258 1.62580645,6.3 1.68387097,5.37096774 C1.71290323,4.5 1.85806452,4.00645161 2.00322581,3.68709677 C2.17741935,3.28064516 2.3516129,2.96129032 2.67096774,2.64193548 C2.99032258,2.32258065 3.28064516,2.11935484 3.71612903,1.97419355 C4.03548387,1.85806452 4.5,1.71290323 5.4,1.65483871 C6.3,1.62580645 6.59032258,1.62580645 9,1.62580645 L9,1.62580645 Z M9,0 C6.56129032,0 6.24193548,0 5.28387097,0.0580645161 C4.32580645,0.116129032 3.65806452,0.261290323 3.10645161,0.464516129 C2.52580645,0.696774194 2.00322581,1.01612903 1.50967742,1.50967742 C1.01612903,2.00322581 0.696774194,2.52580645 0.464516129,3.10645161 C0.232258065,3.68709677 0.0870967742,4.32580645 0.0580645161,5.28387097 C0,6.24193548 0,6.56129032 0,9 C0,11.4387097 0,11.7580645 0.0580645161,12.716129 C0.116129032,13.6741935 0.261290323,14.3419355 0.464516129,14.8935484 C0.696774194,15.4741935 1.01612903,15.9967742 1.50967742,16.4903226 C2.00322581,16.983871 2.52580645,17.3032258 3.10645161,17.5354839 C3.68709677,17.7677419 4.32580645,17.9129032 5.28387097,17.9419355 C6.24193548,18 6.56129032,18 9,18 C11.4387097,18 11.7580645,18 12.716129,17.9419355 C13.6741935,17.883871 14.3419355,17.7387097 14.8935484,17.5354839 C15.4741935,17.3032258 15.9967742,16.983871 16.4903226,16.4903226 C16.983871,15.9967742 17.3032258,15.4741935 17.5354839,14.8935484 C17.7677419,14.3129032 17.9129032,13.6741935 17.9419355,12.716129 C18,11.7580645 18,11.4387097 18,9 C18,6.56129032 18,6.24193548 17.9419355,5.28387097 C17.883871,4.32580645 17.7387097,3.65806452 17.5354839,3.10645161 C17.3032258,2.52580645 16.983871,2.00322581 16.4903226,1.50967742 C15.9967742,1.01612903 15.4741935,0.696774194 14.8935484,0.464516129 C14.3129032,0.232258065 13.6741935,0.0870967742 12.716129,0.0580645161 C11.7580645,0 11.4387097,0 9,0 L9,0 Z"></path> <path d="M9,4.38387097 C6.44516129,4.38387097 4.38387097,6.44516129 4.38387097,9 C4.38387097,11.5548387 6.44516129,13.616129 9,13.616129 C11.5548387,13.616129 13.616129,11.5548387 13.616129,9 C13.616129,6.44516129 11.5548387,4.38387097 9,4.38387097 Z M9,11.9903226 C7.34516129,11.9903226 6.00967742,10.6548387 6.00967742,9 C6.00967742,7.34516129 7.34516129,6.00967742 9,6.00967742 C10.6548387,6.00967742 11.9903226,7.34516129 11.9903226,9 C11.9903226,10.6548387 10.6548387,11.9903226 9,11.9903226 Z"></path> <circle id="Oval" cx="13.7903226" cy="4.20967742" r="1.07419355"></circle> </g> </g> </g></svg></a></li>
            
            <li><a href="https://www.linkedin.com/company/cell-press/" title="LinkedIn" target="_blank"><i aria-hidden="true" class="icon-linkedin"></i></a></li>
            
                        <li><a href="http://bit.ly/1zq7Q5T" title="Twitter" target="_blank">
                <!--<i aria-hidden="true" class="icon-Icon_Twitter"></i></a>-->
                
                <svg id="xicon" viewBox="0 0 1200 1227" xmlns="http://www.w3.org/2000/svg" aria-hidden="true" role="none"><path fill="#969696" d="M714.163 519.284L1160.89 0H1055.03L667.137 450.887L357.328 0H0L468.492 681.821L0 1226.37H105.866L515.491 750.218L842.672 1226.37H1200L714.137 519.284H714.163ZM569.165 687.828L521.697 619.934L144.011 79.6944H306.615L611.412 515.685L658.88 583.579L1055.08 1150.3H892.476L569.165 687.854V687.828Z"></path></svg> 

                    <style>
                    .footer svg#xicon {
                      height: 16px;
                    }
                    </style>
            </a></li>

            
            <li><a href="/pb-assets/marketing/WeChat/WeChat-1557753222727.jpg" title="WeChat" target="_blank"><svg width="18px" height="15px" viewBox="0 0 18 15" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-122.000000, -17.000000)" fill-rule="nonzero" fill="#969696"> <g id="wechat" transform="translate(122.000000, 17.000000)"> <path d="M12.375,4.5 L12.375,4.5 L12.4875,4.5 L12.4875,4.5 C12.6898579,4.48604473 12.8520408,4.32704187 12.87,4.125 C12.8705905,4.04725572 12.8470025,3.9712499 12.8025,3.9075 C11.94,1.5675 9.4425,0 6.5625,0 C2.9475,0 0,2.52 0,5.625 C0.0313772094,7.34410122 0.895513317,8.94093526 2.3175,9.9075 L1.5675,11.46 C1.49009368,11.6087816 1.52088865,11.7908702 1.64291494,11.9059236 C1.76494122,12.0209769 1.94852501,12.0410168 2.0925,11.955 L4.08,10.8225 C4.61861992,11.0129999 5.17724298,11.1413322 5.745,11.205 C5.86564124,11.2178337 5.98503738,11.1715098 6.06545426,11.0806685 C6.14587113,10.9898272 6.17737153,10.865694 6.15,10.7475 C6.06148727,10.4219419 6.01114891,10.0871918 6,9.75 C6,6.855 8.8575,4.5 12.375,4.5 Z M8.25,3 C8.66421356,3 9,3.33578644 9,3.75 C9,4.16421356 8.66421356,4.5 8.25,4.5 C7.83578644,4.5 7.5,4.16421356 7.5,3.75 C7.5,3.33578644 7.83578644,3 8.25,3 Z M4.5,4.5 C4.08578644,4.5 3.75,4.16421356 3.75,3.75 C3.75,3.33578644 4.08578644,3 4.5,3 C4.91421356,3 5.25,3.33578644 5.25,3.75 C5.25,4.16421356 4.91421356,4.5 4.5,4.5 Z"></path> <path d="M18,9.75 C18,7.2675 15.48,5.25 12.375,5.25 C9.27,5.25 6.75,7.2675 6.75,9.75 C6.75,12.2325 9.27,14.25 12.375,14.25 C12.9568557,14.2502837 13.5364258,14.1772075 14.1,14.0325 L15.96,14.9625 C16.0114248,14.9874074 16.0678613,15.0002339 16.125,15 C16.2497193,14.9993787 16.3659682,14.9367912 16.4351508,14.8330172 C16.5043335,14.7292432 16.5173955,14.5978644 16.47,14.4825 L15.96,13.2075 C17.1784092,12.4716325 17.9450305,11.1723221 18,9.75 L18,9.75 Z M10.5,9 C10.0857864,9 9.75,8.66421356 9.75,8.25 C9.75,7.83578644 10.0857864,7.5 10.5,7.5 C10.9142136,7.5 11.25,7.83578644 11.25,8.25 C11.25,8.66421356 10.9142136,9 10.5,9 Z M14.25,9 C13.8357864,9 13.5,8.66421356 13.5,8.25 C13.5,7.83578644 13.8357864,7.5 14.25,7.5 C14.6642136,7.5 15,7.83578644 15,8.25 C15,8.66421356 14.6642136,9 14.25,9 Z" opacity="0.959999979"></path> </g> </g> </g></svg></a></li>
            
            <li><a href="https://www.weibo.com/CellPress" title="Weibo" target="_blank"><svg width="18px" height="15px" viewBox="0 0 18 15" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-99.000000, -17.000000)" fill-rule="nonzero" fill="#969696"> <g id="weibo" transform="translate(99.000000, 14.000000)"> <g transform="translate(0.000000, 3.000000)"> <path d="M13.2402211,7.47054172 C12.9918392,7.39866484 12.821608,7.34902326 12.9514975,7.03290287 C13.232804,6.34602189 13.2622915,5.75359781 12.9571055,5.33095486 C12.3847236,4.5380684 10.8191759,4.58064295 9.02514573,5.30975376 C9.02514573,5.30889193 8.46144724,5.54882627 8.60544724,5.11515185 C8.88150754,4.25435294 8.83953769,3.53386047 8.41043216,3.11759508 C7.43716583,2.17285363 4.84824121,3.15344733 2.62745729,5.3059617 C0.964944724,6.9186238 0,8.62798358 0,10.105855 C0,12.9324952 3.73748744,14.8235294 7.39429146,14.8235294 C12.1877186,14.8235294 15.3768844,11.9501778 15.3768844,9.80576471 C15.3768844,8.50974008 14.2514774,7.77442408 13.2402211,7.47054172 Z M7.40424121,13.5561176 C4.48625126,13.8315595 1.9678794,12.5686293 1.77811055,10.7327524 C1.58870352,8.89773735 3.80098492,7.18579207 6.71861307,6.90948837 C9.63678392,6.63266758 12.1556985,7.89663201 12.3447437,9.73164706 C12.533608,11.5675239 10.3227739,13.2796416 7.40424121,13.5561176 Z M18,5.16875787 C17.9985528,2.31402189 15.5695477,0 12.5728643,0 C12.222995,0 11.9396985,0.270098495 11.9396985,0.603283174 C11.9396985,0.936640219 12.222995,1.20656635 12.5728643,1.20656635 C14.8708945,1.20656635 16.7336683,2.98159781 16.7336683,5.17099863 C16.7336683,5.50435568 17.0173266,5.77428181 17.3668342,5.77428181 C17.7167035,5.77428181 18,5.50435568 18,5.17099863 C18,5.17099863 18,5.1696197 18,5.16875787 Z M15.4841608,5.08016142 C15.1948945,3.73415048 14.0812462,2.67323393 12.6683819,2.39744733 C12.3262915,2.33039672 11.9914372,2.54051163 11.9212462,2.86680164 C11.8506935,3.19309166 12.0712161,3.51179754 12.4133065,3.57919289 C13.3319397,3.75845417 14.0559196,4.44774829 14.2438794,5.32302599 C14.3146131,5.64931601 14.6491055,5.85925855 14.9915578,5.7923803 C15.3338291,5.72498495 15.5541709,5.40627907 15.4841608,5.08016142 Z M5.90761809,8.7069275 C4.7321005,8.9320383 3.93901508,9.87126402 4.1358392,10.8044569 C4.33248241,11.7378222 5.44504523,12.311803 6.62056281,12.0865198 C7.79571859,11.861409 8.58916583,10.922528 8.3921608,9.98899042 C8.19569849,9.0559699 7.08277387,8.48181669 5.90761809,8.7069275 Z"></path> </g> </g> </g> </g></svg></a></li>
            
            <li><a href="https://www.youtube.com/user/cellvideoabstracts" title="YouTube" target="_blank"><svg width="18px" height="13px" viewBox="0 0 18 13" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-76.000000, -19.000000)" fill-rule="nonzero" fill="#969696"> <g id="youtube" transform="translate(76.000000, 14.000000)"> <path d="M17.2984219,6.41974219 C16.6486641,5.64735937 15.4490625,5.33232422 13.1580703,5.33232422 L4.84175391,5.33232422 C2.49834375,5.33232422 1.27838672,5.66771484 0.631089844,6.49001953 C0,7.29172266 0,8.47304297 0,10.1080547 L0,13.2243047 C0,16.3917773 0.748792969,18 4.84175391,18 L13.1581055,18 C15.1448555,18 16.2457383,17.7219844 16.9579688,17.040375 C17.688375,16.3413984 18,15.2001211 18,13.2243047 L18,10.1080547 C18,8.38381641 17.951168,7.19553516 17.2984219,6.41974219 Z M11.5560703,12.0963867 L7.77965625,14.0700586 C7.69524609,14.1141797 7.60289062,14.136082 7.51071094,14.136082 C7.40629687,14.136082 7.30212891,14.1079922 7.20987891,14.0521289 C7.03613672,13.9468711 6.93007031,13.7585391 6.93007031,13.5554414 L6.93007031,9.62075391 C6.93007031,9.41800781 7.03585547,9.22985156 7.20924609,9.12452344 C7.38267187,9.01919531 7.59839062,9.01198828 7.77832031,9.10546875 L11.5547344,11.0664141 C11.7468633,11.1661523 11.8675195,11.3645039 11.8678008,11.5809258 C11.8680469,11.7975586 11.747918,11.9961914 11.5560703,12.0963867 Z"></path> </g> </g> </g></svg></a></li>
            
        </ul>
    </div>
</div>

<style>

.ux3 .footer__logo svg {
    max-width: 160px !important;
}

.ux3 .footer__social-links a svg {
    max-width: 26px;
    height: 1.15rem;
    width: auto;
}

.ux3 .footer__social-links a:hover svg path {
    fill: #353535;
}

</style></div>

            

        

    

        


    

            

            
                <div class="footer__bottom text-onDark">
            

            <div data-widget-id="06227714-c621-4651-9879-b0ce5176cccc" data-widget-name="ux3-layout-widget"><div class="col-sm-4 col-md-5 clearfix">
    

        


    

            

            

            <div data-widget-id="07e1d276-b4dd-4870-b858-ed74e70d64ef" data-widget-name="UX3HTMLWidget"><div class="groups">
	<div class="group">
		<h2 class="group__title">Life &amp; medical sciences journals</h2>
		<ul class="rlist clearfix sided">
		<li>
			<a alt="Cell" href="/cell/home">Cell</a>
		</li>
		<li>
			<a alt="Cancer Cell" href="/cancer-cell/home">Cancer Cell</a>
		</li>
		<li>
			<a alt="Cell Chemical Biology" href="/cell-chemical-biology">Cell Chemical Biology</a>
		</li>
		<li>
			<a alt="Cell Genomics" href="/cell-genomics/home">Cell Genomics</a>
		</li>
		<li>
			<a alt="Cell Host and Microbe" href="/cell-host-microbe/home">Cell Host &amp; Microbe</a>
		</li>
		<li>
			<a alt="Cell Metabolism" href="/cell-metabolism/home">Cell Metabolism</a>
		</li>
		<li>
			<a alt="Cell Reports" href="/cell-reports/home">Cell Reports</a>
		</li>
		<li>
			<a alt="Cell Reports Medicine" href="/cell-reports-medicine/home">Cell Reports Medicine</a>
		</li>
		<li>
			<a alt="Cell Stem Cell" href="/cell-stem-cell/home">Cell Stem Cell</a>
		</li>
		<li>
			<a alt="Cell Systems" href="/cell-systems/home">Cell Systems</a>
		</li>
		<li>
			<a alt="Current Biology" href="/current-biology/home">Current Biology</a>
		</li>
		<li>
			<a alt="Developmental Cell" href="/developmental-cell/home">Developmental Cell</a>
		</li>
		<li>
			<a alt="Immunity" href="/immunity/home">Immunity</a>
		</li>
		<li>
			<a alt="Med" href="/med/home">Med</a>
		</li>
		<li>
			<a alt="Molecular Cell" href="/molecular-cell/home">Molecular Cell</a>
		</li>
		<li>
			<a alt="Neuron" href="/neuron/home">Neuron</a>
		</li>
		<li>
			<a alt="Structure" href="/structure/home">Structure</a>
		</li>
		<li>
			<a alt="American Journal of Human Genetics" href="/ajhg/home">American Journal of Human Genetics (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Biophysical Journal" href="/biophysj/home">Biophysical Journal (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Biophysical Reports" href="/biophysreports/home">Biophysical Reports (<i>partner</i>)</a>
		</li>	
		<li>
			<a alt="HGG Advances" href="/hgg-advances/home">Human Genetics and Genomics Advances (<i>partner</i>)</a></li>
		<li>
			<a alt="Molecular Plant" href="/molecular-plant/home">Molecular Plant (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy" href="/molecular-therapy-family/molecular-therapy/home">Molecular Therapy (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy Methods and Clinical Development" href="/molecular-therapy-family/methods/home">Molecular Therapy Methods &amp; Clinical Development (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy Nucleic Acids" href="/molecular-therapy-family/nucleic-acids/home">Molecular Therapy Nucleic Acids (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy Oncology" href="/molecular-therapy-family/oncology/home">Molecular Therapy Oncology (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Plant Communications" href="/plant-communications/home">Plant Communications (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Stem Cell Reports" href="/stem-cell-reports/home">Stem Cell Reports (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Trends in Biochemical Sciences" href="/trends/biochemical-sciences/home">Trends in Biochemical Sciences</a>
		</li>
		<li>
		</li><li>
			<a alt="Trends in Cancer" href="/trends/cancer/home">Trends in Cancer</a>
		</li>
		<li>
			<a alt="Trends in Cell Biology" href="/trends/cell-biology/home">Trends in Cell Biology</a>
		</li>
		<li>
			<a alt="Trends in Ecology and Evolution" href="/trends/ecology-evolution/home">Trends in Ecology &amp; Evolution</a>
		</li>
		<li>
			<a alt="Trends in Endocrinology and Metabolism" href="/trends/endocrinology-metabolism/home">Trends in Endocrinology &amp; Metabolism</a>
		</li>
		<li>
			<a alt="Trends in Genetics" href="/trends/genetics/home">Trends in Genetics</a>
		</li>
		<li>
			<a alt="Trends in Immunology" href="/trends/immunology/home">Trends in Immunology</a>
		</li>
		<li>
			<a alt="Trends in Microbiology" href="/trends/microbiology/home">Trends in Microbiology</a>
		</li>
		<li>
			<a alt="Trends in Molecular Medicine" href="/trends/molecular-medicine/home">Trends in Molecular Medicine</a>
		</li>
		<li>
			<a alt="Trends in Neurosciences" href="/trends/neurosciences/home">Trends in Neurosciences</a>
		</li>
		<li>
			<a alt="Trends in Parasitology" href="/trends/parasitology/home">Trends in Parasitology</a>
		</li>
		<li>
			<a alt="Trends in Pharmacological Sciences" href="/trends/pharmacological-sciences/home">Trends in Pharmacological Sciences</a>
		</li>
		<li>
			<a alt="Trends in Plant Science" href="/trends/plant-science/home">Trends in Plant Science</a>
		</li>
		</ul>
	</div>

	<div class="group">
		<h2 class="group__title">Physical sciences &amp; engineering journals</h2>
		<ul class="rlist clearfix sided">
		  <li><a alt="Cell Biomaterials" href="/cell-biomaterials/home">Cell Biomaterials</a></li>
		  <li>
			<a alt="Cell Reports Physical Science" href="/cell-reports-physical-science/home">Cell Reports Physical Science</a>
		</li>
		<li>
			<a alt="Chem" href="/chem/home">Chem</a>
		</li>
		<li>
			<a alt="Chem Catalysis" href="/chem-catalysis/home">Chem Catalysis</a>
		</li>
		<li>
			<a alt="Device" href="/device/home">Device</a>
		</li>
		<li>
			<a alt="Joule" href="/joule/home">Joule</a>
		</li>
		<li>
			<a alt="Matter" href="/matter/home">Matter</a>
		</li>	
		<li><a alt="Matter" href="/newton/home">Newton</a></li>
		<li>
			<a alt="Trends in Chemistry" href="/trends/chemistry/home">Trends in Chemistry</a>
		</li>			
		</ul>
	</div>

<div class="group">
	<h2 class="group__title">Multidisciplinary journals</h2>
	<ul class="rlist clearfix sided">
	  		<li>
			<a alt="Cell Reports Methods" href="/cell-reports-methods/home">Cell Reports Methods</a>
		</li>
		<li>
			<a alt="Cell Reports Sustainability" href="/cell-reports-sustainability/home">Cell Reports Sustainability</a>
		</li>
		<li>
			<a alt="Heliyon" href="/heliyon/home">Heliyon</a>
		</li>	
		<li>
			<a alt="iScience" href="/iscience/home">iScience</a>
		</li>
		<li>
			<a alt="One Earth" href="/one-earth/home">One Earth</a>
		</li>
		<li>
			<a alt="Patterns" href="/patterns/home">Patterns</a>
		</li>
		<li>
			<a alt="STAR Protocols" href="/star-protocols/home">STAR Protocols</a>
		</li>
		<li>
			<a alt="Nexus" href="/nexus/home">Nexus (<i>partner</i>)</a>
		</li>			
        <li>
			<a alt="The Innovation" href="/the-innovation/home">The Innovation (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Trends in Biotechnology" href="/trends/biotechnology/home">Trends in Biotechnology</a>
		</li>
		<li>
			<a alt="Trends in Cognitive Sciences" href="/trends/cognitive-sciences/home">Trends in Cognitive Sciences</a>
		</li>
	</ul>
</div>

</div>
</div>

            

        
</div><div class="col-sm-4 col-md-5 clearfix">
    

        


    

            

            

            <div data-widget-id="850476de-00cb-4c0d-943c-bf1881da24f9" data-widget-name="UX3HTMLWidget"><div class="groups">
	<div class="clearfix">
		<div class="group sided">
			<h2 class="group__title">Authors</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Submit article" href="/submit-to-cell-press">Submit article</a>
				</li>
				<li>
					<a alt="Multi-Journal Submission" href="/multi-journal-submission">Multi-Journal Submission</a>
				</li>
				<li>
					<a alt="STAR Methods" href="/star-authors-guide">STAR Methods</a>
				</li>
				<li>
					<a alt="Sneak Peek – Preprints" href="/sneakpeek">Sneak Peek – Preprints</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
		<h2 class="group__title">Reviewers</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Reviewers" href="/reviewers">Information for reviewers</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
		<h2 class="group__title">News &amp; events</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Newsroom" href="/newsroom">Newsroom</a>
				</li>
				<li>
					<a alt="Events" href="/events">Events</a>
				</li>
				<li>
					<a alt="Cell Symposia" href="/symposia">Cell Symposia</a>
				</li>
				<li>
					<a alt="Consortia Hub" href="/consortia">Consortia Hub</a>
				</li>
				<li>
					<a alt="Webinars" href="/webinars">Webinars</a>
				</li>
								<li>
					&nbsp;
				</li>
			</ul>
		<h2 class="group__title">Multimedia</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Cell Press Podcast" href="/podcast">Cell Press Podcast</a>
				</li>
				<li>
					<a alt="Cell Press Videos" href="/video">Cell Press Videos</a>
				</li>
				<li>
					<a alt="Color and Comics" href="/comics">Coloring and Comics</a>
				</li>
				<li>
					<a alt="Figure360" href="/video-guidelines">Figure360</a>
				</li>
				<li>
					<a alt="Cell Picture Show" href="/pictureshow">Cell Picture Show</a>
				</li>
				<li>
					<a alt="Research Arc" href="/research-arc">Research Arc</a>
				</li>
			</ul>
		</div>

		<div class="group sided">
			<h2 class="group__title">About</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="About Cell Press" href="/about">About Cell Press</a>
				</li>

				<li>
					<a alt="Open access" href="/open-access">Open access</a>
				</li>
				<li>
					<a alt="COVID Hub" href="/covid-19">COVID Hub</a>
				</li>
				<li>
					<a alt="Sustainability" href="/sustainability">Sustainability hub</a>
				</li>
				<li>
					<a alt="Inclusion and diversity" href="/diversity">Inclusion and diversity</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
			<h2 class="group__title">Contact</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Contact us" href="/contact">Contact us</a>
				</li>
				<li>
					<a alt="Help and Support" href="https://service.elsevier.com/app/overview/cell-press/" target="_blank">Help &amp; Support</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
			<h2 class="group__title">Careers</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Cell Press Careers" href="https://relx.wd3.myworkdayjobs.com/en-US/ElsevierJobs" target="_blank">Cell Press Careers</a>
				</li>
			</ul>
		</div>
	</div>
</div></div>

            

        
</div><div class="col-sm-4 col-md-2 clearfix">
    

        


    

            

            

            <div data-widget-id="beafac41-1cfc-440b-9b55-74aba4a5612c" data-widget-name="UX3HTMLWidget"><div class="groups">
	<div class="group">
		<h2 class="group__title">Access</h2>
		<ul class="rlist clearfix">
			<li>
				<a alt="Subscribe" href="/subscribe">Subscribe</a>
			</li>
			<li>
				<a alt="Claim" href="/activate">Claim</a>
			</li>
			<li>
				<a alt="Read It Now" href="/read-it-now">Read-It-Now</a>
			</li>
			<li>
				<a alt="Recommend to Librarian" href="http://info.cell.com/request-access-to-cell-press-content" target="_blank">Recommend to Librarian</a>
			</li>
			<li>
				<a alt="Publication Alerts" href="/action/showPreferences?menuTab=Alerts">Publication Alerts</a>
			</li>
		</ul>
	</div>
	<div class="group">
		<h2 class="group__title">Collections</h2>
		<ul class="rlist clearfix">
			<li>
				<a alt="Best of Cell Press" href="/bestof">Best of Cell Press</a>
			</li>
			<li>
				<a alt="Cell Press Reviews" href="/reviews">Cell Press Reviews</a>
			</li>
			<li>
				<a alt="Cell Selections" href="/selections">Cell Press Selections</a>
			</li>
			<li>
				<a alt="Nucleus Collections" href="/nucleus">Nucleus Collections</a>
			</li>
			<li>
				<a alt="SnapShots" href="/snapshots">SnapShot Archive</a>
			</li>
		</ul>
	</div>
	<div class="group">
		<h2 class="group__title">Information</h2>
		<ul class="rlist clearfix">
			<li>
				<a alt="For Advertisters" href="/media-kit">For Advertisers</a>
			</li>
			<li>
				<a alt="For Recruiters" target="_blank" href="/pb-assets/Cell/common/CP-RecruitmentKit-US-2018-LowRes-1712070648.pdf">For Recruiters</a>
			</li>
			<li>
				<a alt="For Librarians" target="_blank" href="https://www.elsevier.com/products/sciencedirect/journals/direct-subscription-journals">For Librarians</a>
			</li>
			<li>
				<a alt="Privacy Policy" target="_blank" href="https://www.elsevier.com/legal/privacy-policy">Privacy Policy</a>
			</li>
			<li>
				<a alt="Terms and Conditions" target="_blank" href="https://www.elsevier.com/legal/elsevier-website-terms-and-conditions">Terms and Conditions</a>
			</li>
			<li>
				<a alt="Accessibility" href="/accessibility">Accessibility</a>
			</li>
		</ul>
	</div>
</div></div>

            

        
</div></div>

            
                </div>
            

        

    

        


    

            

            

            <div data-widget-id="779ad11f-7bcd-4a2a-a2cd-8c67b649cbb2" data-widget-name="UX3HTMLWidget"><style>
 
button#cookieBtn {
   text-decoration: none;
   background-color: transparent;
   color: #005789;
   font-weight: bold;
   padding: 0;
   margin: 0;
   font-size: 14px;
}
 
button#cookieBtn:hover {
   text-decoration: underline;
}

</style>
<script>
// HIDE ONETRUST BACKGROUND AFTER USER PRESSES ESCAPE KEY
setTimeout(function(){ 
   $("#onetrust-pc-sdk").on("keydown", function(e) { 
      if (e.keyCode == 27) {  // ESC key
         setTimeout(function(){ 
            $(".onetrust-pc-dark-filter").css("display","none");
         }, 100); 
      }
   });
}, 5000);
</script>

<span class="align-center copy-right">
<hr>
<p align="center"><b>The content on this site is intended for healthcare professionals and researchers across all fields of science.</b></p>
<hr>
<div>
<p>We use cookies to help provide and enhance our service and tailor content. To update your cookie settings, please visit the 
      <b><button id="cookieBtn" class="ot-sdk-show-settings" type="button" aria-label="Manage cookie consent preferences">Cookie Settings</button></b> for this site.<br>

All content on this site: Copyright © 2025 Elsevier Inc., its licensors, and contributors. <br>All rights are reserved, including those for text and data mining, AI training, and similar technologies. <br>For all open access content, the relevant licensing terms apply.
</p>
<ul class="rlist--inline">
			<li>
				<a alt="Privacy Policy" target="_blank" href="https://www.elsevier.com/legal/privacy-policy">Privacy Policy</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Terms and Conditions" target="_blank" href="https://www.elsevier.com/legal/elsevier-website-terms-and-conditions">Terms &amp; Conditions</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Accessibility" href="/accessibility">Accessibility</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Help and Support" target="_blank" href="https://service.elsevier.com/app/overview/cell-press/">Help &amp; Support</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Contact" href="/contact">Contact</a>
			</li>
			</ul>
<p>
<a href="https://www.relx.com"><img src="/pb/assets/raw/Health%20Advance/images/relx-1554393210983.png" alt="RELX" rel="nofollow" target="_new"></a></p>
</div>

</span></div>

            

        

    

        


    

            

            

            <div data-widget-id="c5e179c1-156e-41e3-af7f-5e307efa8acf" data-widget-name="UX3HTMLWidget"><style>#brandingRow{
    background-color:#fff;
    padding:7px 26px 7px 30px;
    color:#007DBC;
    font-size:14px;
    clear:both;
    width:100%
}
#brandingRow a{
    text-decoration:none;
    color:#007DBC
}
#brandingRow a:hover{
    text-decoration:underline
}
#brandingRowRight{
    display:inline;
    float:right;
    width:auto;
    padding:0;
    margin:0
}
svg#cp_logo {
  float: none;
  height: 36px;
  width: 36px;
  margin-left: 0px;
  margin-top: 0px;
}

footer hr {
  display: block;
}

@media screen and (max-width: 600px){
    #brandingRow{
        font-size:12px
    }
}
@media screen and (max-width: 750px){
    #brandingRow{
        padding:7px 18px 7px 16px
    }
    svg#cp_logo{
        margin-right:10px
    }
}
@media screen and (max-width: 800px) {
   a#cpIconLnk {
      display: none;
   }
}

.header__wrapper.clearfix .col-xs-6:first-of-type{
    width:55%
}
.header__wrapper.clearfix .col-xs-6:nth-of-type(2){
    width:45%
}
@media screen and (min-width: 800px) and (max-width: 1000px){
    .header__wrapper.clearfix .col-xs-6:first-of-type{
        width:75%
    }
    .header__wrapper.clearfix .col-xs-6:nth-of-type(2){
        width:25%
    }
}

</style>

<script>
var partnerx = false;
var overrideEndorsement = "";
var tmpj = $("input[name='journalCode']").attr("value");
	
if (window.location.pathname == "/molecular-therapy-family/home") {
	tmpj = "ymthe";
}
	
var ribbonx = "";
if (tmpj == "ajhg" || tmpj == "bpj" || tmpj == "molp" || tmpj == "xplc" || tmpj == "stemcr" || tmpj == "ymthe" || tmpj == "xhgg" || tmpj == "omtn" || tmpj == "omto" || tmpj == "omtm" || tmpj == "biophysreports" ||  tmpj == "bpr" || tmpj == "xinn" || tmpj == "omton") {
	partnerx = true;
	ribbonx = "<a href='https://www.cell.com' data-aa-name='cell press endorsement' data-aa-region='cp global header' title='Visit Cell.com'>A Cell Press partner journal</a>";
} else if (tmpj != "" && tmpj != "undefined" && tmpj != undefined) {
	ribbonx = "<a href='/' data-aa-name='cell press endorsement' data-aa-region='cp global header' title='Visit Cell.com'>A Cell Press journal</a>";
} else {
	ribbonx = "<a href='/' data-aa-name='cell press endorsement' data-aa-region='cp global header' title='Visit Cell.com'>A Cell Press resource</a>";
}
	
if (window.location.pathname == "/") {
	ribbonx = "";
}
if (overrideEndorsement != "") {
	ribbonx = overrideEndorsement;
}

$(".header__wrapper.clearfix").prepend("<div id='brandingRow'><a href='/' data-aa-name='science that inspires' data-aa-region='cp global header' title='Visit Cell.com'>Science that inspires</a><p id='brandingRowRight'>" + ribbonx + "</p></div>");
	
// handle scrolling effect on article pages
if ($("article").length > 0) {
	var scrollx;
	$(window).scroll(function() {
		scrollx = $(window).scrollTop();
		if (scrollx > 0) { 
			$("#brandingRow").css("display","none"); 
		} else {
			$("#brandingRow").css("display","block");
		}
	});
}

if (window.location.pathname == "/" || partnerx == false) {
               $("<a href='/home' id='cpIconLnk' data-aa-name='cell press logo' data-aa-region='cp global header' title='Cell Press' target='_blank'><svg id='cp_logo' version='1.1' xmlns='http://www.w3.org/2000/svg' xmlns:xlink='http://www.w3.org/1999/xlink' x='0px' y='0px' viewBox='0 0 100 98.1' style='enable-background:new 0 0 100 98.1;' xml:space='preserve' height='100' width='98.1'><path style='fill:#FFFFFF;' d='M69.09,0.09c-4.01-0.6-7.83,1.94-8.78,5.89l-3.14,13.11c-0.76,3.19-3.44,5.56-6.7,5.93l-18.6,2.13          C11.83,29.31-2.53,47.66,0.37,67.78c2.3,15.96,15.18,27.77,30.54,30.08c4.01,0.6,7.83-1.94,8.78-5.89l3.14-13.11 c0.76-3.19,3.44-5.56,6.69-5.93l18.6-2.13c20.04-2.17,34.41-20.52,31.5-40.65C97.32,14.21,84.45,2.39,69.09,0.09z M65.06,47.2  l-11.75,1.4c-1.98,0.21-3.62,1.65-4.08,3.59l-3.43,14.3c-1.01,4.22-4.5,7.39-8.8,8c-6.66,0.72-12.63-4.2-13.15-10.92                c-0.51-6.5,4.6-12.11,11.08-12.81l11.75-1.39c1.98-0.22,3.62-1.65,4.08-3.59l3.43-14.3c1.01-4.22,4.5-7.39,8.8-8          c6.66-0.72,12.63,4.19,13.15,10.92C76.66,40.89,71.55,46.5,65.06,47.2z'/></svg></a>").insertBefore(".header__wrapper.clearfix .journal-logos:eq(0)");
}


</script></div>

            

        

    

        


    

            

            

            <div data-widget-id="1af759b7-9bb6-4413-ada9-51a8333e8fb8" data-widget-name="UX3HTMLWidget"><style>
    /*This was added temporarily until the PB changelog tab issue got fixed. [LIT-444969 - c#4]*/
    [data-tab-id="changelogTab"] {
        display: none !important;
    }
    
    /*This was added temporarily until the PB changelog tab issue got fixed. [LIT-518839]*/
    @media (min-width: 769px) {
        .ux3 .quick-search .dropBlock [data-db-target-of],
        .quick-search .dropBlock [data-db-target-of] {
            width: 100vw !important;
        }
    }
</style></div>

            

        
</div></div>

            

        

        </div>
    

    

        


    

            

            

            <div data-widget-id="d7e0813b-a300-497a-ab37-9c3ddaf03e71" data-widget-name="UX3HTMLWidget"><!--
<div class="session-reminder hidden">
    <div class="session-reminder__overlay">
        <div class="session-reminder__dialog">
            <div class="session-reminder__dialog-inner">
                <div class="dialog-header" role="dialog" aria-labelledby="dialog-title" aria-describedby="dialog-desc">
                    <h2 class="dialog-header-title" id="dialog-title">
                        <span>Session Timeout</span>
                        <div class="dialog-timer">(2:00)</div>
                    </h2>
                    <button class="dialog-close" type="button" aria-label="close dialog">
                        <i class="icon-tools_close" aria-hidden="true"></i>
                    </button>
                </div>
                <div class="dialog-body">
                    <p class="dialog-desc" id="dialog-desc">Your session will expire shortly. If you are still working, click the ‘Keep Me Logged In’ button below. If you do not respond within the next minute, you will be automatically logged out.</p>
                    <p class="dialog-expired sr-only" aria-live="polite"></p>
                </div>
                <div class="dialog-footer">
                    <button class="session-reset" type="button">Keep Me Logged In</button>
                </div>
            </div>
        </div>
    </div>
</div>
<a class="sr-only" href="javascript:void(0)" aria-hidden="true"></a>
--></div>

            

        
        </div>
    </div>


































    



















        
        
            
                <script src="/products/marlin/cell/releasedAssets/js/main.bundle-90475670808d5d3eee41.js"></script>
            
        
        
        

    




<script type="text/javascript">
    var pageData =  {"journal":{"specialty":"Cell Press","issn":"0092-8674","name":"Cell"},"page":{"loadTimestamp":"1749592253496","server":"cell","environment":"prod","businessUnit":"ELS:RAP:JBS","noTracking":"","name":"fulltext:html","section":"Cell","language":"en","type":"CP-CA","productName":"JB"},"visitor":{"departmentName":"|ae:ScienceDirect Guests","accessType":"ae:ANON_GUEST|ae:ANON_GUEST","accountId":"ae:838949|ae:228598","accountName":"ae:Cell Guest Account|ae:ScienceDirect Guests","departmentId":"ae:977212|ae:291352","ipAddress":"189.120.74.179","loginStatus":"anonymous","userId":"ae:246228266|ae:12975512"},"content":[{"accessType":"article:free:openaccess","volumeNumber":"184","issueNumber":"26","viewState":"fulltext","format":"MIME-HTML","id":"pii:S0092867421013374","coverDate":"202112","type":"XOCS-JOURNAL:SCOPE-FULL","title":"A transcriptional rheostat couples past activity to future sensory responses","status":"S300"}]};
</script>

<script type="text/javascript">
    if(typeof pageDataTracker !== "undefined") {
        pageDataTracker.trackPageLoad();
    }
</script>
<script>(function(){function c(){var b=a.contentDocument||a.contentWindow.document;if(b){var d=b.createElement('script');d.innerHTML="window.__CF$cv$params={r:'94dc1639da684292',t:'MTc0OTU5MjI1Mi4wMDAwMDA='};var a=document.createElement('script');a.nonce='';a.src='/cdn-cgi/challenge-platform/scripts/jsd/main.js';document.getElementsByTagName('head')[0].appendChild(a);";b.getElementsByTagName('head')[0].appendChild(d)}}if(document.body){var a=document.createElement('iframe');a.height=1;a.width=1;a.style.position='absolute';a.style.top=0;a.style.left=0;a.style.border='none';a.style.visibility='hidden';document.body.appendChild(a);if('loading'!==document.readyState)c();else if(window.addEventListener)document.addEventListener('DOMContentLoaded',c);else{var e=document.onreadystatechange||function(){};document.onreadystatechange=function(b){e(b);'loading'!==document.readyState&&(document.onreadystatechange=e,c())}}}})();</script><iframe height="1" width="1" style="position: absolute; top: 0px; left: 0px; border: none; visibility: hidden;"></iframe><script defer="" src="https://static.cloudflareinsights.com/beacon.min.js/vcd15cbe7772f49c399c6a5babf22c1241717689176015" integrity="sha512-ZpsOmlRQV6y907TI0dKBHq9Md29nnaEIPlkf84rnaERnq6zvWvPUqr2ft8M1aS28oN72PdrCzSjY4U6VaAw1EQ==" data-cf-beacon="{&quot;rayId&quot;:&quot;94dc1639da684292&quot;,&quot;serverTiming&quot;:{&quot;name&quot;:{&quot;cfExtPri&quot;:true,&quot;cfEdge&quot;:true,&quot;cfOrigin&quot;:true,&quot;cfL4&quot;:true,&quot;cfSpeedBrain&quot;:true,&quot;cfCacheStatus&quot;:true}},&quot;version&quot;:&quot;2025.6.2&quot;,&quot;token&quot;:&quot;57a02315956f4b2e82817b3dfa99b1a2&quot;}" crossorigin="anonymous"></script>


<div id="addtoany" style="position: static;"><div class="a2a_overlay" id="a2a_overlay"></div><div id="a2a_modal" class="a2a_modal a2a_hide" role="dialog" tabindex="-1" aria-label=""><div class="a2a_modal_body a2a_menu a2a_hide" id="a2a_copy_link"><label for="a2a_copy_link_text" id="a2a_copy_link_icon" class="a2a_svg a2a_s_link a2a_logo_color"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"><title>Copy link</title><path fill="#FFF" d="M7.591 21.177c0-.36.126-.665.377-.917l2.804-2.804a1.235 1.235 0 0 1 .913-.378c.377 0 .7.144.97.43-.026.028-.11.11-.255.25-.144.14-.24.236-.29.29a2.82 2.82 0 0 0-.2.256 1.056 1.056 0 0 0-.177.344 1.43 1.43 0 0 0-.046.37c0 .36.126.666.377.918a1.25 1.25 0 0 0 .918.377c.126.001.251-.015.373-.047.125-.037.242-.096.345-.175.09-.06.176-.127.256-.2.1-.094.196-.19.29-.29.14-.142.223-.23.25-.254.297.28.445.607.445.984 0 .36-.126.664-.377.916l-2.778 2.79a1.242 1.242 0 0 1-.917.364c-.36 0-.665-.118-.917-.35l-1.982-1.97a1.223 1.223 0 0 1-.378-.9l-.001-.004Zm9.477-9.504c0-.36.126-.665.377-.917l2.777-2.79a1.235 1.235 0 0 1 .913-.378c.35 0 .656.12.917.364l1.984 1.968c.254.252.38.553.38.903 0 .36-.126.665-.38.917l-2.802 2.804a1.238 1.238 0 0 1-.916.364c-.377 0-.7-.14-.97-.418.026-.027.11-.11.255-.25a7.5 7.5 0 0 0 .29-.29c.072-.08.139-.166.2-.255.08-.103.14-.22.176-.344.032-.12.048-.245.047-.37 0-.36-.126-.662-.377-.914a1.247 1.247 0 0 0-.917-.377c-.136 0-.26.015-.37.046-.114.03-.23.09-.346.175a3.868 3.868 0 0 0-.256.2c-.054.05-.15.148-.29.29-.14.146-.222.23-.25.258-.294-.278-.442-.606-.442-.983v-.003ZM5.003 21.177c0 1.078.382 1.99 1.146 2.736l1.982 1.968c.745.75 1.658 1.12 2.736 1.12 1.087 0 2.004-.38 2.75-1.143l2.777-2.79c.75-.747 1.12-1.66 1.12-2.737 0-1.106-.392-2.046-1.183-2.818l1.186-1.185c.774.79 1.708 1.186 2.805 1.186 1.078 0 1.995-.376 2.75-1.13l2.803-2.81c.751-.754 1.128-1.671 1.128-2.748 0-1.08-.382-1.993-1.146-2.738L23.875 6.12C23.13 5.372 22.218 5 21.139 5c-1.087 0-2.004.382-2.75 1.146l-2.777 2.79c-.75.747-1.12 1.66-1.12 2.737 0 1.105.392 2.045 1.183 2.817l-1.186 1.186c-.774-.79-1.708-1.186-2.805-1.186-1.078 0-1.995.377-2.75 1.132L6.13 18.426c-.754.755-1.13 1.672-1.13 2.75l.003.001Z"></path></svg></label><input id="a2a_copy_link_text" type="text" title="Copy link" readonly=""><div id="a2a_copy_link_copied">✓</div></div><div class="a2a_modal_body a2a_menu a2a_thanks a2a_hide" id="a2a_thanks"><div class="a2a_localize" data-a2a-localize="inner,ThanksForSharing">Thanks for sharing!</div></div></div><div class="a2a_menu a2a_full a2a_localize" id="a2apage_full" role="dialog" tabindex="-1" aria-label="Share" data-a2a-localize="title,Share"><div class="a2a_full_header"><div id="a2apage_find_container" class="a2a_menu_find_container"><label for="a2apage_find" id="a2apage_find_icon" class="a2a_svg a2a_s_find"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" aria-hidden="true"><title>Find any service</title><path fill="#CCC" d="M19.7 18.2l-4.5-4.5c.7-1.1 1.2-2.3 1.2-3.6 0-3.5-2.8-6.3-6.3-6.3s-6.3 2.8-6.3 6.3 2.8 6.3 6.3 6.3c1.4 0 2.6-.4 3.6-1.2l4.5 4.5c.6.6 1.3.7 1.7.2.5-.4.4-1.1-.2-1.7zm-9.6-3.6c-2.5 0-4.5-2.1-4.5-4.5 0-2.5 2.1-4.5 4.5-4.5 2.5 0 4.5 2.1 4.5 4.5s-2 4.5-4.5 4.5z"></path></svg></label><input id="a2apage_find" class="a2a_menu_find a2a_localize" type="text" autocomplete="off" title="Find any service" data-a2a-localize="title,FindAnyServiceToAddTo"></div></div><div class="a2a_full_services" id="a2apage_full_services" role="presentation"></div><div class="a2a_full_footer"><a href="https://www.addtoany.com" title="Share Buttons" rel="noopener" target="_blank"><span class="a2a_svg a2a_s__default a2a_s_a2a a2a_logo_color"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"><g fill="#FFF"><path d="M14 7h4v18h-4z"></path><path d="M7 14h18v4H7z"></path></g></svg></span>AddToAny</a></div></div><div id="a2apage_dropdown" class="a2a_menu a2a_mini a2a_localize a2a_hide" tabindex="-1" aria-label="Share" data-a2a-localize="label,Share"><div class="a2a_mini_services" id="a2apage_mini_services"></div><div id="a2apage_cols_container" class="a2a_cols_container"><div class="a2a_col1" id="a2apage_col1"></div><div id="a2apage_2_col1" class="a2a_hide"></div><div class="a2a_clear"></div></div><div class="a2apage_wide a2a_wide"><a href="#addtoany" id="a2apage_show_more_less" class="a2a_more a2a_localize" title="Show all" data-a2a-localize="title,ShowAll"><span class="a2a_svg a2a_s__default a2a_s_a2a a2a_logo_color"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"><g fill="#FFF"><path d="M14 7h4v18h-4z"></path><path d="M7 14h18v4H7z"></path></g></svg></span><span class="a2a_localize" data-a2a-localize="inner,More">More…</span></a></div></div><div style="height: 1px; width: 1px; position: absolute; z-index: 100000; top: 0px; visibility: hidden;"><iframe id="a2a_sm_ifr" title="AddToAny Utility Frame" aria-hidden="true" src="https://static.addtoany.com/menu/sm.25.html#type=core&amp;event=load" style="height: 1px; width: 1px; border: 0px; left: 0px; top: 0px; position: absolute; z-index: 100000; display: none;"></iframe></div></div><div role="status" aria-live="assertive" aria-relevant="additions" class="ui-helper-hidden-accessible"></div><div id="onetrust-consent-sdk" data-nosnippet="true"><div class="onetrust-pc-dark-filter ot-fade-in" style="z-index:2147483645;"></div><div id="onetrust-banner-sdk" class="otFlat bottom ot-wo-title ot-buttons-fw" tabindex="0" role="region" aria-label="Cookie banner"><div role="dialog" aria-modal="true" aria-label="Privacy"><div class="ot-sdk-container"><div class="ot-sdk-row"><div id="onetrust-group-container" class="ot-sdk-eight ot-sdk-columns"><div class="banner_logo"></div><div id="onetrust-policy"><div id="onetrust-policy-text">We use cookies that are necessary to make our site work. We may also use additional cookies to analyze, improve, and personalize our content and your digital experience. For more information, see our<a class="ot-cookie-policy-link" href="https://www.elsevier.com/legal/cookienotice" target="_blank" aria-label=", opens in a new tab" rel="noopener">Cookie Policy</a></div></div></div><div id="onetrust-button-group-parent" class="ot-sdk-three ot-sdk-columns"><div id="onetrust-button-group"><button id="onetrust-pc-btn-handler" class="cookie-setting-link">Cookie Settings</button>  <button id="onetrust-accept-btn-handler">Accept all cookies</button></div></div></div></div><!-- Close Button --><div id="onetrust-close-btn-container"></div><!-- Close Button END--></div></div><div id="onetrust-pc-sdk" class="otPcCenter ot-hide ot-fade-in" lang="en" aria-label="Preference center" role="region"><div role="dialog" aria-modal="true" style="height: 100%;" aria-label="Cookie Preference Center"><!-- Close Button --><div class="ot-pc-header"><!-- Logo Tag --><div class="ot-pc-logo" role="img" aria-label="Company Logo"><img alt="Company Logo" src="https://cdn.cookielaw.org/logos/static/ot_company_logo.png"></div></div><!-- Close Button --><div id="ot-pc-content" class="ot-pc-scrollbar"><div class="ot-optout-signal ot-hide"><div class="ot-optout-icon"><svg xmlns="http://www.w3.org/2000/svg"><path class="ot-floating-button__svg-fill" d="M14.588 0l.445.328c1.807 1.303 3.961 2.533 6.461 3.688 2.015.93 4.576 1.746 7.682 2.446 0 14.178-4.73 24.133-14.19 29.864l-.398.236C4.863 30.87 0 20.837 0 6.462c3.107-.7 5.668-1.516 7.682-2.446 2.709-1.251 5.01-2.59 6.906-4.016zm5.87 13.88a.75.75 0 00-.974.159l-5.475 6.625-3.005-2.997-.077-.067a.75.75 0 00-.983 1.13l4.172 4.16 6.525-7.895.06-.083a.75.75 0 00-.16-.973z" fill="#FFF" fill-rule="evenodd"></path></svg></div><span></span></div><h2 id="ot-pc-title">Cookie Preference Center</h2><div id="ot-pc-desc">We use cookies which are necessary to make our site work. We may also use additional cookies to analyse, improve and personalise our content and your digital experience. For more information, see our <a href="https://www.elsevier.com/legal/cookienotice/_nocache" target="_blank">Cookie Policy</a> and the list of <a href="https://support.google.com/admanager/answer/9012903" target="_blank">Google Ad-Tech Vendors</a>.
<br>
<br>
You may choose not to allow some types of cookies. However, blocking some types may impact your experience of our site and the services we are able to offer. See the different category headings below to find out more or change your settings.
<br>
</div><button id="accept-recommended-btn-handler">Allow all</button><section class="ot-sdk-row ot-cat-grp"><h3 id="ot-category-title"> Manage Consent Preferences</h3><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="1"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-1" aria-labelledby="ot-header-id-1 ot-status-id-1"></button><!-- Accordion header --><div class="ot-acc-hdr ot-always-active-group"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-1">Strictly Necessary Cookies</h4><div id="ot-status-id-1" class="ot-always-active">Always active</div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-1">These cookies are necessary for the website to function and cannot be switched off in our systems. They are usually only set in response to actions made by you which amount to a request for services, such as setting your privacy preferences, logging in or filling in forms. You can set your browser to block or alert you about these cookies, but some parts of the site will not then work. These cookies do not store any personally identifiable information.
<br><br></p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="1">Cookie Details List‎</button></div></div></div><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="3"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-3" aria-labelledby="ot-header-id-3"></button><!-- Accordion header --><div class="ot-acc-hdr"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-3">Functional Cookies</h4><div class="ot-tgl"><input type="checkbox" name="ot-group-id-3" id="ot-group-id-3" role="switch" class="category-switch-handler" data-optanongroupid="3" aria-labelledby="ot-header-id-3"> <label class="ot-switch" for="ot-group-id-3"><span class="ot-switch-nob"></span> <span class="ot-label-txt">Functional Cookies</span></label> </div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-3">These cookies enable the website to provide enhanced functionality and personalisation. They may be set by us or by third party providers whose services we have added to our pages. If you do not allow these cookies then some or all of these services may not function properly.</p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="3">Cookie Details List‎</button></div></div></div><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="2"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-2" aria-labelledby="ot-header-id-2"></button><!-- Accordion header --><div class="ot-acc-hdr"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-2">Performance Cookies</h4><div class="ot-tgl"><input type="checkbox" name="ot-group-id-2" id="ot-group-id-2" role="switch" class="category-switch-handler" data-optanongroupid="2" aria-labelledby="ot-header-id-2"> <label class="ot-switch" for="ot-group-id-2"><span class="ot-switch-nob"></span> <span class="ot-label-txt">Performance Cookies</span></label> </div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-2">These cookies allow us to count visits and traffic sources so we can measure and improve the performance of our site. They help us to know which pages are the most and least popular and see how visitors move around the site.</p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="2">Cookie Details List‎</button></div></div></div><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="4"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-4" aria-labelledby="ot-header-id-4"></button><!-- Accordion header --><div class="ot-acc-hdr"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-4">Targeting Cookies</h4><div class="ot-tgl"><input type="checkbox" name="ot-group-id-4" id="ot-group-id-4" role="switch" class="category-switch-handler" data-optanongroupid="4" aria-labelledby="ot-header-id-4"> <label class="ot-switch" for="ot-group-id-4"><span class="ot-switch-nob"></span> <span class="ot-label-txt">Targeting Cookies</span></label> </div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-4">These cookies may be set through our site by our advertising partners. They may be used by those companies to build a profile of your interests and show you relevant adverts on other sites. If you do not allow these cookies, you will experience less targeted advertising.</p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="4">Cookie Details List‎</button></div></div></div><!-- Groups sections starts --><!-- Group section ends --><!-- Accordion Group section starts --><!-- Accordion Group section ends --></section></div><section id="ot-pc-lst" class="ot-hide ot-hosts-ui ot-pc-scrollbar"><div id="ot-pc-hdr"><div id="ot-lst-title"><button class="ot-link-btn back-btn-handler" aria-label="Back"><svg id="ot-back-arw" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 444.531 444.531" xml:space="preserve"><title>Back Button</title><g><path fill="#656565" d="M213.13,222.409L351.88,83.653c7.05-7.043,10.567-15.657,10.567-25.841c0-10.183-3.518-18.793-10.567-25.835
                    l-21.409-21.416C323.432,3.521,314.817,0,304.637,0s-18.791,3.521-25.841,10.561L92.649,196.425
                    c-7.044,7.043-10.566,15.656-10.566,25.841s3.521,18.791,10.566,25.837l186.146,185.864c7.05,7.043,15.66,10.564,25.841,10.564
                    s18.795-3.521,25.834-10.564l21.409-21.412c7.05-7.039,10.567-15.604,10.567-25.697c0-10.085-3.518-18.746-10.567-25.978
                    L213.13,222.409z"></path></g></svg></button><h3>Cookie List</h3></div><div class="ot-lst-subhdr"><div class="ot-search-cntr"><p role="status" class="ot-scrn-rdr"></p><input id="vendor-search-handler" type="text" name="vendor-search-handler" placeholder="Search…" aria-label="Cookie list search"> <svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 -30 110 110" aria-hidden="true"><title>Search Icon</title><path fill="#2e3644" d="M55.146,51.887L41.588,37.786c3.486-4.144,5.396-9.358,5.396-14.786c0-12.682-10.318-23-23-23s-23,10.318-23,23
            s10.318,23,23,23c4.761,0,9.298-1.436,13.177-4.162l13.661,14.208c0.571,0.593,1.339,0.92,2.162,0.92
            c0.779,0,1.518-0.297,2.079-0.837C56.255,54.982,56.293,53.08,55.146,51.887z M23.984,6c9.374,0,17,7.626,17,17s-7.626,17-17,17
            s-17-7.626-17-17S14.61,6,23.984,6z"></path></svg></div><div class="ot-fltr-cntr"><button id="filter-btn-handler" aria-label="Filter" aria-haspopup="true"><svg role="presentation" aria-hidden="true" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 402.577 402.577" xml:space="preserve"><title>Filter Icon</title><g><path fill="#fff" d="M400.858,11.427c-3.241-7.421-8.85-11.132-16.854-11.136H18.564c-7.993,0-13.61,3.715-16.846,11.136
      c-3.234,7.801-1.903,14.467,3.999,19.985l140.757,140.753v138.755c0,4.955,1.809,9.232,5.424,12.854l73.085,73.083
      c3.429,3.614,7.71,5.428,12.851,5.428c2.282,0,4.66-0.479,7.135-1.43c7.426-3.238,11.14-8.851,11.14-16.845V172.166L396.861,31.413
      C402.765,25.895,404.093,19.231,400.858,11.427z"></path></g></svg></button></div><div id="ot-anchor"></div><section id="ot-fltr-modal"><div id="ot-fltr-cnt"><button id="clear-filters-handler">Clear</button><div class="ot-fltr-scrlcnt ot-pc-scrollbar"><div class="ot-fltr-opts" role="list"><div class="ot-fltr-opt" role="listitem"><div class="ot-chkbox"><input id="chkbox-id" type="checkbox" class="category-filter-handler"> <label for="chkbox-id"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div></div></div><div class="ot-fltr-btns"><button id="filter-apply-handler">Apply</button> <button id="filter-cancel-handler">Cancel</button></div></div></div></section></div></div><section id="ot-lst-cnt" class="ot-host-cnt ot-pc-scrollbar"><div id="ot-sel-blk"><div class="ot-sel-all"><div class="ot-sel-all-hdr"><span class="ot-consent-hdr">Consent</span> <span class="ot-li-hdr">Leg.Interest</span></div><div class="ot-sel-all-chkbox"><div class="ot-chkbox" id="ot-selall-hostcntr"><input id="select-all-hosts-groups-handler" type="checkbox"> <label for="select-all-hosts-groups-handler"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div><div class="ot-chkbox" id="ot-selall-vencntr"><input id="select-all-vendor-groups-handler" type="checkbox"> <label for="select-all-vendor-groups-handler"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div><div class="ot-chkbox" id="ot-selall-licntr"><input id="select-all-vendor-leg-handler" type="checkbox"> <label for="select-all-vendor-leg-handler"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div></div></div></div><div class="ot-sdk-row"><div class="ot-sdk-column"><ul id="ot-host-lst"></ul></div></div></section></section><div class="ot-pc-footer ot-pc-scrollbar"><div class="ot-btn-container"> <button class="save-preference-btn-handler onetrust-close-btn-handler">Confirm my choices</button></div><!-- Footer logo --><div class="ot-pc-footer-logo"><a href="https://www.onetrust.com/products/cookie-consent/" target="_blank" rel="noopener noreferrer" aria-label="Powered by OneTrust Opens in a new Tab"><img alt="Powered by Onetrust" src="https://cdn.cookielaw.org/logos/static/powered_by_logo.svg" title="Powered by OneTrust Opens in a new Tab"></a></div></div><!-- Cookie subgroup container --><!-- Vendor list link --><!-- Cookie lost link --><!-- Toggle HTML element --><!-- Checkbox HTML --><!-- plus minus--><!-- Arrow SVG element --><!-- Accordion basic element --><span class="ot-scrn-rdr" aria-atomic="true" aria-live="polite"></span><!-- Vendor Service container and item template --></div><iframe class="ot-text-resize" sandbox="allow-same-origin" title="onetrust-text-resize" style="position: absolute; top: -50000px; width: 100em;" aria-hidden="true"></iframe></div></div><div id="crossmark-widget" style="display: none;"><div class="crossmark-reset crossmark-overlay"></div><div class="crossmark-reset crossmark-popup"><div class="crossmark-reset crossmark-popup__offset"><div class="crossmark-reset crossmark-popup__inner"><div class="crossmark-reset crossmark-popup__header"><button class="crossmark-reset crossmark-popup__btn-close"><span class="sr-only">__("articleCrossmark.closePopup")</span></button></div><div class="crossmark-reset crossmark-popup__content-wrapper"><iframe class="crossmark-reset crossmark-popup__content" title="Crossmark service - Intentionally blank" aria-hidden="false" tabindex="0" src="https://crossmark.crossref.org/dialog?doi=10.1016%2Fj.cell.2021.11.022&amp;domain=www.cell.com&amp;uri_scheme=https%3A&amp;cm_version=v2.0"></iframe></div></div></div></div></div></body></html>