<html lang="en" class="pb-page fonts-loaded fonts-loaded" data-request-id="f3873365-12e5-464c-b7b0-143c72928c03" style="--article-sticky-top: 82.00019999999999px; --scroll-padding: 163.594px auto auto;"><head data-pb-dropzone="head"><meta name="pbContext" content=";product:product:elsevier\:product\:cell;website:website:cell-site;ctype:string:Journal Content;issue:issue:pii\:S0092867420X0016X;journal:journal:cell;pageGroup:string:Article View;page:string:Show Full Text;requestedJournal:journal:cell;wgroup:string:Default Website Group;article:article:pii\:S0092867421007534;subPage:string:Full Text">




<!-- if there is any customization for Responsive Project widget  -->


























    
    







    
            
        



        
                <meta name="robots" content="noarchive,nocache">
            
    






    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=1, user-scalable=1">











        <meta property="og:title" content="GABA-receptive microglia selectively sculpt developing inhibitory circuits">
    


        <meta property="og:type" content="Article">
    


        <meta property="og:url" content="https://www.cell.com/cell/abstract/S0092-8674(21)00753-4">
    



        <meta property="og:site_name" content="Cell">
    


        <meta property="og:image" content="https://www.cell.com/cms/asset/10709c4c-a8de-43f7-b674-b66116c03d5b/fx1.jpg">
    


        <meta property="og:description" content="GABA-receptive microglia selectively prune inhibitory synapses in development and
disruption of this microglial response leads to behavioral abnormalities highlighting
a critical function of selective communication between microglia and neuronal cell
types.">
    











        <meta name="twitter:title" content="GABA-receptive microglia selectively sculpt developing inhibitory circuits">
    




        <meta name="twitter:image" content="https://www.cell.com/cms/asset/10709c4c-a8de-43f7-b674-b66116c03d5b/fx1.jpg">
    


        <meta name="twitter:description" content="GABA-receptive microglia selectively prune inhibitory synapses in development and
disruption of this microglial response leads to behavioral abnormalities highlighting
a critical function of selective communication between microglia and neuronal cell
types.">
    

        <meta name="twitter:card" content="summary_large_image">
    











    
    
            
            
        <link rel="canonical" href="https://www.cell.com/cell/fulltext/S0092-8674(21)00753-4">
    


<meta name="tdm-reservation" content="1">
<meta name="tdm-policy" content="https://www.elsevier.com/tdm/tdmrep-policy.json">

<title>GABA-receptive microglia selectively sculpt developing inhibitory circuits: Cell</title><meta name="citation_journal_title" content="Cell"> <meta name="citation_journal_abbrev" content="Cell"> <meta name="citation_publisher" content="Elsevier"> <meta name="citation_title" content="GABA-receptive microglia selectively sculpt developing inhibitory circuits"> <meta name="citation_date" content="2021/07/22"> <meta name="citation_online_date" content="2021/07/06"> <meta name="citation_volume" content="184"> <meta name="citation_issue" content="15"> <meta name="citation_firstpage" content="4048"> <meta name="citation_lastpage" content="4063.e32"> <meta name="citation_pmid" content="34233165"> <meta name="citation_abstract_html_url" content="https://www.cell.com/action/showAbstract?pii=S0092867421007534"> <meta name="citation_fulltext_html_url" content="https://www.cell.com/action/showFullText?pii=S0092867421007534"> <meta name="citation_pdf_url" content="https://www.cell.com/action/showPdf?pii=S0092867421007534"> <meta name="citation_issn" content="0092-8674"> <meta name="citation_issn" content="1097-4172"> <meta name="citation_language" content="English"> <meta name="citation_doi" content="10.1016/j.cell.2021.06.018"> <meta name="citation_pmid" content="34233165"> <meta name="citation_author" content="Emilia Favuzzi"><meta name="citation_author_orcid" content="0000-0003-4256-1490"><meta name="citation_author_institution" content="Department of Neurobiology, Blavatnik Institute, Harvard Medical School, Boston, MA 02115, USA"><meta name="citation_author_institution" content="Stanley Center for Psychiatric Research, Broad Institute of Harvard and MIT, Cambridge, MA 02142, USA"><meta name="citation_author" content="Shuhan Huang"><meta name="citation_author_orcid" content="0000-0002-3139-7296"><meta name="citation_author_institution" content="Department of Neurobiology, Blavatnik Institute, Harvard Medical School, Boston, MA 02115, USA"><meta name="citation_author_institution" content="Stanley Center for Psychiatric Research, Broad Institute of Harvard and MIT, Cambridge, MA 02142, USA"><meta name="citation_author" content="Giuseppe A. Saldi"><meta name="citation_author_orcid" content="0000-0002-9994-7354"><meta name="citation_author_institution" content="Stanley Center for Psychiatric Research, Broad Institute of Harvard and MIT, Cambridge, MA 02142, USA"><meta name="citation_author_institution" content="Department of Biology, New York University, New York, NY 10003, USA"><meta name="citation_author" content="Loïc Binan"><meta name="citation_author_institution" content="Klarman Cell Observatory, Broad Institute of Harvard and MIT, Cambridge, MA 02142, USA"><meta name="citation_author" content="Leena A. Ibrahim"><meta name="citation_author_orcid" content="0000-0001-8255-3423"><meta name="citation_author_institution" content="Department of Neurobiology, Blavatnik Institute, Harvard Medical School, Boston, MA 02115, USA"><meta name="citation_author_institution" content="Stanley Center for Psychiatric Research, Broad Institute of Harvard and MIT, Cambridge, MA 02142, USA"><meta name="citation_author" content="Marian Fernández-Otero"><meta name="citation_author_institution" content="Department of Neurobiology, Blavatnik Institute, Harvard Medical School, Boston, MA 02115, USA"><meta name="citation_author_institution" content="Stanley Center for Psychiatric Research, Broad Institute of Harvard and MIT, Cambridge, MA 02142, USA"><meta name="citation_author" content="Yuqing Cao"><meta name="citation_author_institution" content="Department of Neurobiology, Blavatnik Institute, Harvard Medical School, Boston, MA 02115, USA"><meta name="citation_author_institution" content="Stanley Center for Psychiatric Research, Broad Institute of Harvard and MIT, Cambridge, MA 02142, USA"><meta name="citation_author" content="Ayman Zeine"><meta name="citation_author_institution" content="Department of Neurobiology, Blavatnik Institute, Harvard Medical School, Boston, MA 02115, USA"><meta name="citation_author" content="Adwoa Sefah"><meta name="citation_author_orcid" content="0000-0002-1258-5486"><meta name="citation_author_institution" content="Department of Neurobiology, Blavatnik Institute, Harvard Medical School, Boston, MA 02115, USA"><meta name="citation_author_institution" content="Stanley Center for Psychiatric Research, Broad Institute of Harvard and MIT, Cambridge, MA 02142, USA"><meta name="citation_author" content="Karen Zheng"><meta name="citation_author_orcid" content="0000-0003-4974-1136"><meta name="citation_author_institution" content="Broad Institute of Harvard and MIT, Cambridge, MA 02142, USA"><meta name="citation_author" content="Qing Xu"><meta name="citation_author_orcid" content="0000-0001-9479-470X"><meta name="citation_author_institution" content="New York University Abu Dhabi, Abu Dhabi, UAE"><meta name="citation_author" content="Elizaveta Khlestova"><meta name="citation_author_institution" content="Department of Neurobiology, Blavatnik Institute, Harvard Medical School, Boston, MA 02115, USA"><meta name="citation_author" content="Samouil L. Farhi"><meta name="citation_author_orcid" content="0000-0003-1359-4568"><meta name="citation_author_institution" content="Klarman Cell Observatory, Broad Institute of Harvard and MIT, Cambridge, MA 02142, USA"><meta name="citation_author" content="Richard Bonneau"><meta name="citation_author_institution" content="Department of Biology, New York University, New York, NY 10003, USA"><meta name="citation_author_institution" content="Center for Computational Biology, Flatiron Institute, New York, NY 10010, USA"><meta name="citation_author_institution" content="Center for Data Science, New York University, New York, NY 10011, USA"><meta name="citation_author" content="Sandeep Robert Datta"><meta name="citation_author_institution" content="Department of Neurobiology, Blavatnik Institute, Harvard Medical School, Boston, MA 02115, USA"><meta name="citation_author" content="Beth Stevens"><meta name="citation_author_institution" content="Stanley Center for Psychiatric Research, Broad Institute of Harvard and MIT, Cambridge, MA 02142, USA"><meta name="citation_author_institution" content="Boston Children’s Hospital, F.M. Kirby Neurobiology Center, Boston, MA 02115, USA"><meta name="citation_author" content="Gord Fishell"><meta name="citation_author_email" content="gordon_fishell@hms.harvard.edu"><meta name="citation_author_institution" content="Department of Neurobiology, Blavatnik Institute, Harvard Medical School, Boston, MA 02115, USA"><meta name="citation_author_institution" content="Stanley Center for Psychiatric Research, Broad Institute of Harvard and MIT, Cambridge, MA 02142, USA"><meta name="citation_keywords" content="microglia; interneurons; inhibitory synapses; development; neurodevelopmental disorders; glia-neuron interactions; cerebral cortex"><meta name="citation_reference" content="citation_title=Diagnostic and Statistical Manual of Mental Disorders (DSM-5); citation_author=American Psychiatric Association; citation_publication_date=2013; citation_publisher=APA Publishing; "><meta name="citation_reference" content="citation_title=Orchestrating single-cell analysis with Bioconductor; citation_author=R.A. Amezquita; citation_author=A.T.L. Lun; citation_author=E. Becht; citation_author=V.J. Carey; citation_author=L.N. Carpp; citation_author=L. Geistlinger; citation_author=F. Marini; citation_author=K. Rue-Albrecht; citation_author=D. Risso; citation_author=C. Soneson; citation_author=et al.; citation_journal_title=Nat. Methods; citation_volume=17; citation_firstpage=137; citation_lastpage=145; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=viSNE enables visualization of high dimensional single-cell data and reveals phenotypic heterogeneity of leukemia; citation_author=A.D. Amir; citation_author=K.L. Davis; citation_author=M.D. Tadmor; citation_author=E.F. Simonds; citation_author=J.H. Levine; citation_author=S.C. Bendall; citation_author=D.K. Shenfeld; citation_author=S. Krishnaswamy; citation_author=G.P. Nolan; citation_author=D. Pe’er; citation_journal_title=Nat. Biotechnol.; citation_volume=31; citation_firstpage=545; citation_lastpage=552; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=Negative feedback control of neuronal activity by microglia; citation_author=A. Badimon; citation_author=H.J. Strasburger; citation_author=P. Ayata; citation_author=X. Chen; citation_author=A. Nair; citation_author=A. Ikegami; citation_author=P. Hwang; citation_author=A.T. Chan; citation_author=S.M. Graves; citation_author=J.O. Uweru; citation_author=et al.; citation_journal_title=Nature; citation_volume=586; citation_firstpage=417; citation_lastpage=423; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Microglia in Brain Development, Homeostasis, and Neurodegeneration; citation_author=C.J. Bohlen; citation_author=B.A. Friedman; citation_author=B. Dejanovic; citation_author=M. Sheng; citation_journal_title=Annu. Rev. Genet.; citation_volume=53; citation_firstpage=263; citation_lastpage=288; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Homozygous C1q deficiency causes glomerulonephritis associated with multiple apoptotic bodies; citation_author=M. Botto; citation_author=C. Dell’Agnola; citation_author=A.E. Bygrave; citation_author=E.M. Thompson; citation_author=H.T. Cook; citation_author=F. Petry; citation_author=M. Loos; citation_author=P.P. Pandolfi; citation_author=M.J. Walport; citation_journal_title=Nat. Genet.; citation_volume=19; citation_firstpage=56; citation_lastpage=59; citation_publication_date=1998; "><meta name="citation_reference" content="citation_title=The GABAA Receptor as a Therapeutic Target for Neurodevelopmental Disorders; citation_author=S. Braat; citation_author=R.F. Kooy; citation_journal_title=Neuron; citation_volume=86; citation_firstpage=1119; citation_lastpage=1130; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=The single-cell transcriptional landscape of mammalian organogenesis; citation_author=J. Cao; citation_author=M. Spielmann; citation_author=X. Qiu; citation_author=X. Huang; citation_author=D.M. Ibrahim; citation_author=A.J. Hill; citation_author=F. Zhang; citation_author=S. Mundlos; citation_author=L. Christiansen; citation_author=F.J. Steemers; citation_author=et al.; citation_journal_title=Nature; citation_volume=566; citation_firstpage=496; citation_lastpage=502; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Functional role of ambient GABA in refining neuronal circuits early in postnatal development; citation_author=G. Cellot; citation_author=E. Cherubini; citation_journal_title=Front. Neural Circuits; citation_volume=7; citation_firstpage=136; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=Tetraspanins at a glance; citation_author=S. Charrin; citation_author=S. Jouannet; citation_author=C. Boucheix; citation_author=E. Rubinstein; citation_journal_title=J.&nbsp;Cell Sci.; citation_volume=127; citation_firstpage=3641; citation_lastpage=3648; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Sensory Experience Engages Microglia to Shape Neural Connectivity through a Non-Phagocytic Mechanism; citation_author=L. Cheadle; citation_author=S.A. Rivera; citation_author=J.S. Phelps; citation_author=K.A. Ennis; citation_author=B. Stevens; citation_author=L.C. Burkly; citation_author=W.A. Lee; citation_author=M.E. Greenberg; citation_journal_title=Neuron; citation_volume=108; citation_firstpage=451; citation_lastpage=468.e9; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Microglial displacement of inhibitory synapses provides neuroprotection in the adult brain; citation_author=Z. Chen; citation_author=W. Jalabi; citation_author=W. Hu; citation_author=H.-J. Park; citation_author=J.T. Gale; citation_author=G.J. Kidd; citation_author=R. Bernatowicz; citation_author=Z.C. Gossman; citation_author=J.T. Chen; citation_author=R. Dutta; citation_author=B.D. Trapp; citation_journal_title=Nat. Commun.; citation_volume=5; citation_firstpage=4486; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=The emerging picture of autism spectrum disorder: genetics and pathology; citation_author=J.A. Chen; citation_author=O. Peñagarikano; citation_author=T.G. Belgard; citation_author=V. Swarup; citation_author=D.H. Geschwind; citation_journal_title=Annu. Rev. Pathol.; citation_volume=10; citation_firstpage=111; citation_lastpage=144; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=RNA imaging. Spatially resolved, highly multiplexed RNA profiling in single cells; citation_author=K.H. Chen; citation_author=A.N. Boettiger; citation_author=J.R. Moffitt; citation_author=S. Wang; citation_author=X. Zhuang; citation_journal_title=Science; citation_volume=348; citation_firstpage=aaa6090; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Microglial depletion prevents extracellular matrix changes and striatal volume reduction in a model of Huntington’s disease.; citation_author=J. Crapser; citation_author=J. Ochaba; citation_author=N. Soni; citation_author=J. Reidling; citation_author=L. Thompson; citation_author=K. Green; citation_journal_title=Brain; citation_volume=143; citation_firstpage=266; citation_lastpage=288; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=The igraph software package for complex network research; citation_author=G. Csardi; citation_author=T. Nepusz; citation_journal_title=InterJournal Complex Syst.; citation_volume=1695; citation_firstpage=1; citation_lastpage=9; citation_publication_date=2006; "><meta name="citation_reference" content="citation_title=Microglia monitor and protect neuronal function through specialized somatic purinergic junctions; citation_author=C. Cserép; citation_author=B. Pósfai; citation_author=N. Lénárt; citation_author=R. Fekete; citation_author=Z.I. László; citation_author=Z. Lele; citation_author=B. Orsolits; citation_author=G. Molnár; citation_author=S. Heindl; citation_author=A.D. Schwarcz; citation_author=et al.; citation_journal_title=Science; citation_volume=367; citation_firstpage=528; citation_lastpage=537; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=A review of fronto-striatal and fronto-cortical brain abnormalities in children and adults with Attention Deficit Hyperactivity Disorder (ADHD) and new evidence for dysfunction in adults with ADHD during motivation and attention; citation_author=A. Cubillo; citation_author=R. Halari; citation_author=A. Smith; citation_author=E. Taylor; citation_author=K. Rubia; citation_journal_title=Cortex; citation_volume=48; citation_firstpage=194; citation_lastpage=215; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=Disease-Associated Microglia: A Universal Immune Sensor of Neurodegeneration; citation_author=A. Deczkowska; citation_author=H. Keren-Shaul; citation_author=A. Weiner; citation_author=M. Colonna; citation_author=M. Schwartz; citation_author=I. Amit; citation_journal_title=Cell; citation_volume=173; citation_firstpage=1073; citation_lastpage=1081; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Complement 3a receptor in dorsal horn microglia mediates pronociceptive neuropeptide signaling; citation_author=S. Doolen; citation_author=J. Cook; citation_author=M. Riedl; citation_author=K. Kitto; citation_author=S. Kohsaka; citation_author=C.N. Honda; citation_author=C.A. Fairbanks; citation_author=B.K. Taylor; citation_author=L. Vulchanova; citation_journal_title=Glia; citation_volume=65; citation_firstpage=1976; citation_lastpage=1989; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Reduced GABA concentration in attention-deficit/hyperactivity disorder; citation_author=R.A.E. Edden; citation_author=D. Crocetti; citation_author=H. Zhu; citation_author=D.L. Gilbert; citation_author=S.H. Mostofsky; citation_journal_title=Arch. Gen. Psychiatry; citation_volume=69; citation_firstpage=750; citation_lastpage=753; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=ZhuangLab/MERlin: MERlin v0.1.6; citation_author=G. Emanuel; citation_author=H. Babcock; citation_publication_date=2020; citation_publisher=Zenodo; "><meta name="citation_reference" content="citation_title=Molecular diversity underlying cortical excitatory and inhibitory synapse development; citation_author=E. Favuzzi; citation_author=B. Rico; citation_journal_title=Curr. Opin. Neurobiol.; citation_volume=53; citation_firstpage=8; citation_lastpage=15; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Activity-Dependent Gating of Parvalbumin Interneuron Function by the Perineuronal Net Protein Brevican; citation_author=E. Favuzzi; citation_author=A. Marques-Smith; citation_author=R. Deogracias; citation_author=C.M. Winterflood; citation_author=A. Sánchez-Aguilera; citation_author=L. Mantoan; citation_author=P. Maeso; citation_author=C. Fernandes; citation_author=H. Ewers; citation_author=B. Rico; citation_journal_title=Neuron; citation_volume=95; citation_firstpage=639; citation_lastpage=655.e10; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Distinct molecular programs regulate synapse specificity in cortical inhibitory circuits; citation_author=E. Favuzzi; citation_author=R. Deogracias; citation_author=A. Marques-Smith; citation_author=P. Maeso; citation_author=J. Jezequel; citation_author=D. Exposito-Alonso; citation_author=M. Balia; citation_author=T. Kroon; citation_author=A.J. Hinojosa; citation_author=E. F Maraver; citation_author=B. Rico; citation_journal_title=Science; citation_volume=363; citation_firstpage=413; citation_lastpage=417; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=The Microglial Innate Immune Receptor TREM2 Is Required for Synapse Elimination and Normal Brain Connectivity; citation_author=F. Filipello; citation_author=R. Morini; citation_author=I. Corradini; citation_author=V. Zerbi; citation_author=A. Canzi; citation_author=B. Michalski; citation_author=M. Erreni; citation_author=M. Markicevic; citation_author=C. Starvaggi-Cucuzza; citation_author=K. Otero; citation_author=et al.; citation_journal_title=Immunity; citation_volume=48; citation_firstpage=979; citation_lastpage=991.e8; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Microglial morphology and dynamic behavior is regulated by ionotropic glutamatergic and GABAergic neurotransmission; citation_author=A.M. Fontainhas; citation_author=M. Wang; citation_author=K.J. Liang; citation_author=S. Chen; citation_author=P. Mettu; citation_author=M. Damani; citation_author=R.N. Fariss; citation_author=W. Li; citation_author=W.T. Wong; citation_journal_title=PLoS ONE; citation_volume=6; citation_firstpage=e15973; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=Emerging neurotrophic role of GABAB receptors in neuronal circuit development; citation_author=J.-L. Gaiarsa; citation_author=C. Porcher; citation_journal_title=Front. Cell. Neurosci.; citation_volume=7; citation_firstpage=206; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=The splicing regulator Rbfox1 (A2BP1) controls neuronal excitation in the mammalian brain; citation_author=L.T. Gehman; citation_author=P. Stoilov; citation_author=J. Maguire; citation_author=A. Damianov; citation_author=C.-H. Lin; citation_author=L. Shiue; citation_author=M. Ares; citation_author=I. Mody; citation_author=D.L. Black; citation_journal_title=Nat. Genet.; citation_volume=43; citation_firstpage=706; citation_lastpage=711; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=Motor cortex inhibition: a marker of ADHD behavior and motor development in children; citation_author=D.L. Gilbert; citation_author=K.M. Isaacs; citation_author=M. Augusta; citation_author=L.K. Macneil; citation_author=S.H. Mostofsky; citation_journal_title=Neurology; citation_volume=76; citation_firstpage=615; citation_lastpage=621; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=Floxed allele for conditional inactivation of the GABAB(1) gene; citation_author=C. Haller; citation_author=E. Casanova; citation_author=M. Müller; citation_author=C.-M. Vacher; citation_author=R. Vigot; citation_author=T. Doll; citation_author=S. Barbieri; citation_author=M. Gassmann; citation_author=B. Bettler; citation_journal_title=Genesis; citation_volume=40; citation_firstpage=125; citation_lastpage=130; citation_publication_date=2004; "><meta name="citation_reference" content="citation_title=Single-Cell RNA Sequencing of Microglia throughout the Mouse Lifespan and in the Injured Brain Reveals Complex Cell-State Changes; citation_author=T.R. Hammond; citation_author=C. Dufort; citation_author=L. Dissing-Olesen; citation_author=S. Giera; citation_author=A. Young; citation_author=A. Wysoker; citation_author=A.J. Walker; citation_author=F. Gergits; citation_author=M. Segel; citation_author=J. Nemesh; citation_author=et al.; citation_journal_title=Immunity; citation_volume=50; citation_firstpage=253; citation_lastpage=271.e6; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=GABA(B) receptors function as a heteromeric assembly of the subunits GABA(B)R1 and GABA(B)R2; citation_author=K.A. Jones; citation_author=B. Borowsky; citation_author=J.A. Tamm; citation_author=D.A. Craig; citation_author=M.M. Durkin; citation_author=M. Dai; citation_author=W.-J. Yao; citation_author=M. Johnson; citation_author=C. Gunwaldsen; citation_author=L.-Y. Huang; citation_author=et al.; citation_journal_title=Nature; citation_volume=396; citation_firstpage=674; citation_lastpage=679; citation_publication_date=1998; "><meta name="citation_reference" content="citation_title=Tmem119-EGFP and Tmem119-CreERT2 Transgenic Mice for Labeling and Manipulating Microglia; citation_author=T. Kaiser; citation_author=G. Feng; citation_journal_title=eNeuro; citation_volume=6; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=GFP-like proteins stably accumulate in lysosomes; citation_author=H. Katayama; citation_author=A. Yamamoto; citation_author=N. Mizushima; citation_author=T. Yoshimori; citation_author=A. Miyawaki; citation_journal_title=Cell Struct. Funct.; citation_volume=33; citation_firstpage=1; citation_lastpage=12; citation_publication_date=2008; "><meta name="citation_reference" content="citation_title=A Unique Microglia Type Associated with Restricting Development of Alzheimer’s Disease; citation_author=H. Keren-Shaul; citation_author=A. Spinrad; citation_author=A. Weiner; citation_author=O. Matcovitch-Natan; citation_author=R. Dvir-Szternfeld; citation_author=T.K. Ulland; citation_author=E. David; citation_author=K. Baruch; citation_author=D. Lara-Astaiso; citation_author=B. Toth; citation_author=et al.; citation_journal_title=Cell; citation_volume=169; citation_firstpage=1276; citation_lastpage=1290.e17; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=The TREM2-APOE pathway drives the transcriptional phenotype of dysfunctional microglia in neurodegenerative diseases; citation_author=S. Krasemann; citation_author=C. Madore; citation_author=R. Cialic; citation_author=C. Baufeld; citation_author=N. Calcagno; citation_author=R. El Fatimy; citation_author=L. Beckers; citation_author=E. O’Loughlin; citation_author=Y. Xu; citation_author=Z. Fanek; citation_author=et al.; citation_journal_title=Immunity; citation_volume=47; citation_firstpage=566; citation_lastpage=581.e9; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Microglia express GABA(B) receptors to modulate interleukin release; citation_author=S.A. Kuhn; citation_author=F.K.H. van Landeghem; citation_author=R. Zacharias; citation_author=K. Färber; citation_author=A. Rappert; citation_author=S. Pavlovic; citation_author=A. Hoffmann; citation_author=C. Nolte; citation_author=H. Kettenmann; citation_journal_title=Mol. Cell. Neurosci.; citation_volume=25; citation_firstpage=312; citation_lastpage=322; citation_publication_date=2004; "><meta name="citation_reference" content="citation_title=Developmental Heterogeneity of Microglia and Brain Myeloid Cells Revealed by Deep Single-Cell RNA Sequencing; citation_author=Q. Li; citation_author=Z. Cheng; citation_author=L. Zhou; citation_author=S. Darmanis; citation_author=N.F. Neff; citation_author=J. Okamoto; citation_author=G. Gulati; citation_author=M.L. Bennett; citation_author=L.O. Sun; citation_author=L.E. Clarke; citation_author=et al.; citation_journal_title=Neuron; citation_volume=101; citation_firstpage=207; citation_lastpage=223.e10; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Neuronal network activity controls microglial process surveillance in awake mice via norepinephrine signaling; citation_author=Y.U. Liu; citation_author=Y. Ying; citation_author=Y. Li; citation_author=U.B. Eyo; citation_author=T. Chen; citation_author=J. Zheng; citation_author=A.D. Umpierre; citation_author=J. Zhu; citation_author=D.B. Bosco; citation_author=H. Dong; citation_author=L.J. Wu; citation_journal_title=Nat. Neurosci.; citation_volume=22; citation_firstpage=1771; citation_lastpage=1781; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Microglia elimination increases neural circuit connectivity and activity in adult mouse cortex; citation_author=Y.-J. Liu; citation_author=E. Spangenberg; citation_author=B. Tang; citation_author=T.C. Holmes; citation_author=K.N. Green; citation_author=X. Xu; citation_journal_title=J.&nbsp;Neurosci.; citation_volume=41; citation_firstpage=1274; citation_lastpage=1287; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=Progranulin Deficiency Promotes Circuit-Specific Synaptic Pruning by Microglia via Complement Activation; citation_author=H. Lui; citation_author=J. Zhang; citation_author=S.R. Makinson; citation_author=M.K. Cahill; citation_author=K.W. Kelley; citation_author=H.-Y. Huang; citation_author=Y. Shang; citation_author=M.C. Oldham; citation_author=L.H. Martens; citation_author=F. Gao; citation_author=et al.; citation_journal_title=Cell; citation_volume=165; citation_firstpage=921; citation_lastpage=935; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Pooling across cells to normalize single-cell RNA sequencing data with many zero counts; citation_author=A.T. Lun; citation_author=K. Bach; citation_author=J.C. Marioni; citation_journal_title=Genome Biol.; citation_volume=17; citation_firstpage=75; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=A step-by-step workflow for low-level analysis of single-cell RNA-seq data with Bioconductor; citation_author=A.T.L. Lun; citation_author=D.J. McCarthy; citation_author=J.C. Marioni; citation_journal_title=F1000Res.; citation_volume=5; citation_firstpage=2122; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=A toolbox of Cre-dependent optogenetic transgenic mice for light-induced activation and silencing; citation_author=L. Madisen; citation_author=T. Mao; citation_author=H. Koch; citation_author=J.M. Zhuo; citation_author=A. Berenyi; citation_author=S. Fujisawa; citation_author=Y.-W.A. Hsu; citation_author=A.J. Garcia; citation_author=X. Gu; citation_author=S. Zanella; citation_author=et al.; citation_journal_title=Nat. Neurosci.; citation_volume=15; citation_firstpage=793; citation_lastpage=802; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=Receptors, ion channels, and signaling mechanisms underlying microglial dynamics; citation_author=C. Madry; citation_author=D. Attwell; citation_journal_title=J.&nbsp;Biol. Chem.; citation_volume=290; citation_firstpage=12443; citation_lastpage=12450; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=The Striatum Organizes 3D Behavior via Moment-to-Moment Action Selection; citation_author=J.E. Markowitz; citation_author=W.F. Gillis; citation_author=C.C. Beron; citation_author=S.Q. Neufeld; citation_author=K. Robertson; citation_author=N.D. Bhagat; citation_author=R.E. Peterson; citation_author=E. Peterson; citation_author=M. Hyun; citation_author=S.W. Linderman; citation_author=et al.; citation_journal_title=Cell; citation_volume=174; citation_firstpage=44; citation_lastpage=58.e17; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Spatial and temporal heterogeneity of mouse and human microglia at single-cell resolution; citation_author=T. Masuda; citation_author=R. Sankowski; citation_author=O. Staszewski; citation_author=C. Böttcher; citation_author=L. Amann; citation_author=Sagar; citation_author=C. Scheiwe; citation_author=S. Nessler; citation_author=P. Kunz; citation_author=G. van Loo; citation_author=et al.; citation_journal_title=Nature; citation_volume=566; citation_firstpage=388; citation_lastpage=392; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Microglia development follows a stepwise program to regulate brain homeostasis; citation_author=O. Matcovitch-Natan; citation_author=D.R. Winter; citation_author=A. Giladi; citation_author=S. Vargas Aguilar; citation_author=A. Spinrad; citation_author=S. Sarrazin; citation_author=H. Ben-Yehuda; citation_author=E. David; citation_author=F. Zelada González; citation_author=P. Perrin; citation_author=et al.; citation_journal_title=Science; citation_volume=353; citation_firstpage=aad8670; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Scater: pre-processing, quality control, normalization and visualization of single-cell RNA-seq data in R; citation_author=D.J. McCarthy; citation_author=K.R. Campbell; citation_author=A.T.L. Lun; citation_author=Q.F. Wills; citation_journal_title=Bioinformatics; citation_volume=33; citation_firstpage=1179; citation_lastpage=1186; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=GABAergic-astrocyte signaling: A refinement of inhibitory brain networks; citation_author=S. Mederos; citation_author=G. Perea; citation_journal_title=Glia; citation_volume=67; citation_firstpage=1842; citation_lastpage=1851; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Molecular, spatial, and functional single-cell profiling of the hypothalamic preoptic region; citation_author=J.R. Moffitt; citation_author=D. Bambah-Mukku; citation_author=S.W. Eichhorn; citation_author=E. Vaughn; citation_author=K. Shekhar; citation_author=J.D. Perez; citation_author=N.D. Rubinstein; citation_author=J. Hao; citation_author=A. Regev; citation_author=C. Dulac; citation_author=X. Zhuang; citation_journal_title=Science; citation_volume=362; citation_firstpage=eaau5324; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Hyperactivity with Disrupted Attention by Activation of an Astrocyte Synaptogenic Cue; citation_author=J. Nagai; citation_author=A.K. Rajbhandari; citation_author=M.R. Gangwani; citation_author=A. Hachisuka; citation_author=G. Coppola; citation_author=S.C. Masmanidis; citation_author=M.S. Fanselow; citation_author=B.S. Khakh; citation_journal_title=Cell; citation_volume=177; citation_firstpage=1280; citation_lastpage=1292.e20; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Errant gardeners: glial-cell-dependent synaptic pruning and neurodevelopmental disorders; citation_author=U. Neniskyte; citation_author=C.T. Gross; citation_journal_title=Nat. Rev. Neurosci.; citation_volume=18; citation_firstpage=658; citation_lastpage=670; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Microglial Remodeling of the Extracellular Matrix Promotes Synapse Plasticity; citation_author=P. Nguyen; citation_author=L. Dorman; citation_author=Pan; citation_author=I. Vainchtein; citation_author=R. Han; citation_author=H. Nakao-Inoue; citation_author=S. Taloma; citation_author=J. Barron; citation_author=A. Molofsky; citation_author=M. Kheirbek; citation_author=et al.; citation_journal_title=Cell; citation_volume=182; citation_firstpage=388; citation_lastpage=403; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=De novo synaptogenesis induced by GABA in the developing mouse cortex; citation_author=W.C. Oh; citation_author=S. Lutzu; citation_author=P.E. Castillo; citation_author=H.-B. Kwon; citation_journal_title=Science; citation_volume=353; citation_firstpage=1037; citation_lastpage=1040; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Synaptic pruning by microglia is necessary for normal brain development; citation_author=R.C. Paolicelli; citation_author=G. Bolasco; citation_author=F. Pagani; citation_author=L. Maggi; citation_author=M. Scianni; citation_author=P. Panzanelli; citation_author=M. Giustetto; citation_author=T.A. Ferreira; citation_author=E. Guiducci; citation_author=L. Dumas; citation_author=et al.; citation_journal_title=Science; citation_volume=333; citation_firstpage=1456; citation_lastpage=1458; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=Sample Drift Correction Following 4D Confocal Time-lapse Imaging; citation_author=A. Parslow; citation_author=A. Cardona; citation_author=R.J. Bryson-Richardson; citation_journal_title=J.&nbsp;Vis. Exp.; citation_number=86; citation_firstpage=51086; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Comparing development of synaptic proteins in rat visual, somatosensory, and frontal cortex; citation_author=J.G.A. Pinto; citation_author=D.G. Jones; citation_author=K.M. Murphy; citation_journal_title=Front. Neural Circuits; citation_volume=7; citation_firstpage=97; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=Neurotransmitter receptors on microglia; citation_author=J.M. Pocock; citation_author=H. Kettenmann; citation_journal_title=Trends Neurosci.; citation_volume=30; citation_firstpage=527; citation_lastpage=535; citation_publication_date=2007; "><meta name="citation_reference" content="citation_title=BBKNN: fast batch alignment of single cell transcriptomes; citation_author=K. Polański; citation_author=M.D. Young; citation_author=Z. Miao; citation_author=K.B. Meyer; citation_author=S.A. Teichmann; citation_author=J.-E. Park; citation_journal_title=Bioinformatics; citation_volume=36; citation_firstpage=964; citation_lastpage=965; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Modification of seizure activity by electrical stimulation. II. Motor seizure; citation_author=R.J. Racine; citation_journal_title=Electroencephalogr. Clin. Neurophysiol.; citation_volume=32; citation_firstpage=281; citation_lastpage=294; citation_publication_date=1972; "><meta name="citation_reference" content="citation_title=Microglial physiology: unique stimuli, specialized responses; citation_author=R.M. Ransohoff; citation_author=V.H. Perry; citation_journal_title=Annu. Rev. Immunol.; citation_volume=27; citation_firstpage=119; citation_lastpage=145; citation_publication_date=2009; "><meta name="citation_reference" content="citation_title=Spatial reconstruction of single-cell gene expression data; citation_author=R. Satija; citation_author=J.A. Farrell; citation_author=D. Gennert; citation_author=A.F. Schier; citation_author=A. Regev; citation_journal_title=Nat. Biotechnol.; citation_volume=33; citation_firstpage=495; citation_lastpage=502; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Molecular Diversity and Specializations among the Cells of the Adult Mouse Brain; citation_author=A. Saunders; citation_author=E.Z. Macosko; citation_author=A. Wysoker; citation_author=M. Goldman; citation_author=F.M. Krienen; citation_author=H. de Rivera; citation_author=E. Bien; citation_author=M. Baum; citation_author=L. Bortolin; citation_author=S. Wang; citation_author=et al.; citation_journal_title=Cell; citation_volume=174; citation_firstpage=1015; citation_lastpage=1030.e16; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Microglia sculpt postnatal neural circuits in an activity and complement-dependent manner; citation_author=D.P. Schafer; citation_author=E.K. Lehrman; citation_author=A.G. Kautzman; citation_author=R. Koyama; citation_author=A.R. Mardinly; citation_author=R. Yamasaki; citation_author=R.M. Ransohoff; citation_author=M.E. Greenberg; citation_author=B.A. Barres; citation_author=B. Stevens; citation_journal_title=Neuron; citation_volume=74; citation_firstpage=691; citation_lastpage=705; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=NIH Image to ImageJ: 25 years of image analysis; citation_author=C.A. Schneider; citation_author=W.S. Rasband; citation_author=K.W. Eliceiri; citation_journal_title=Nat. Methods; citation_volume=9; citation_firstpage=671; citation_lastpage=675; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=Acid-Tolerant Monomeric GFP from Olindias formosa; citation_author=H. Shinoda; citation_author=Y. Ma; citation_author=R. Nakashima; citation_author=K. Sakurai; citation_author=T. Matsuda; citation_author=T. Nagai; citation_journal_title=Cell Chem. Biol.; citation_volume=25; citation_firstpage=330; citation_lastpage=338.e7; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Fluorescent Proteins for Investigating Biological Events in Acidic Environments; citation_author=H. Shinoda; citation_author=M. Shannon; citation_author=T. Nagai; citation_journal_title=Int. J. Mol. Sci.; citation_volume=19; citation_firstpage=1548; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Excitation-inhibition balance as a framework for investigating mechanisms in neuropsychiatric disorders; citation_author=V.S. Sohal; citation_author=J.L.R. Rubenstein; citation_journal_title=Mol. Psychiatry; citation_volume=24; citation_firstpage=1248; citation_lastpage=1257; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Sustained microglial depletion with CSF1R inhibitor impairs parenchymal plaque development in an Alzheimer’s disease model; citation_author=E. Spangenberg; citation_author=P.L. Severson; citation_author=L.A. Hohsfield; citation_author=J. Crapser; citation_author=J. Zhang; citation_author=E.A. Burton; citation_author=Y. Zhang; citation_author=W. Spevak; citation_author=J. Lin; citation_author=N.Y. Phan; citation_author=et al.; citation_journal_title=Nat. Commun.; citation_volume=10; citation_firstpage=3758; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Microglia modulate wiring of the embryonic forebrain; citation_author=P. Squarzoni; citation_author=G. Oller; citation_author=G. Hoeffel; citation_author=L. Pont-Lezica; citation_author=P. Rostaing; citation_author=D. Low; citation_author=A. Bessis; citation_author=F. Ginhoux; citation_author=S. Garel; citation_journal_title=Cell Rep.; citation_volume=8; citation_firstpage=1271; citation_lastpage=1279; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=BioGRID: a general repository for interaction datasets; citation_author=C. Stark; citation_author=B.-J. Breitkreutz; citation_author=T. Reguly; citation_author=L. Boucher; citation_author=A. Breitkreutz; citation_author=M. Tyers; citation_journal_title=Nucleic Acids Res.; citation_volume=34; citation_firstpage=D535; citation_lastpage=D539; citation_publication_date=2006; "><meta name="citation_reference" content="citation_title=The classical complement cascade mediates CNS synapse elimination; citation_author=B. Stevens; citation_author=N.J. Allen; citation_author=L.E. Vazquez; citation_author=G.R. Howell; citation_author=K.S. Christopherson; citation_author=N. Nouri; citation_author=K.D. Micheva; citation_author=A.K. Mehalow; citation_author=A.D. Huberman; citation_author=B. Stafford; citation_author=et al.; citation_journal_title=Cell; citation_volume=131; citation_firstpage=1164; citation_lastpage=1178; citation_publication_date=2007; "><meta name="citation_reference" content="citation_title=Noradrenergic signaling in the wakeful state inhibits microglial surveillance and synaptic plasticity in the mouse visual cortex; citation_author=R.D. Stowell; citation_author=G.O. Sipe; citation_author=R.P. Dawes; citation_author=H.N. Batchelor; citation_author=K.A. Lordy; citation_author=B.S. Whitelaw; citation_author=M.B. Stoessel; citation_author=J.M. Bidlack; citation_author=E. Brown; citation_author=M. Sur; citation_author=A.K. Majewska; citation_journal_title=Nat. Neurosci.; citation_volume=22; citation_firstpage=1782; citation_lastpage=1792; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=UniProt: the universal protein knowledgebase; citation_author=The UniProt Consortium; citation_journal_title=Nucleic Acids Res.; citation_volume=45; citation_number=D1; citation_firstpage=D158; citation_lastpage=D169; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Microglia and early brain development: An intimate journey; citation_author=M.S. Thion; citation_author=F. Ginhoux; citation_author=S. Garel; citation_journal_title=Science; citation_volume=362; citation_firstpage=185; citation_lastpage=189; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Biphasic Impact of Prenatal Inflammation and Macrophage Depletion on the Wiring of Neocortical Inhibitory Circuits; citation_author=M.S. Thion; citation_author=C.-A. Mosser; citation_author=I. Férézou; citation_author=P. Grisel; citation_author=S. Baptista; citation_author=D. Low; citation_author=F. Ginhoux; citation_author=S. Garel; citation_author=E. Audinat; citation_journal_title=Cell Rep.; citation_volume=28; citation_firstpage=1119; citation_lastpage=1126.e4; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Preparation of Acute Brain Slices Using an Optimized N-Methyl-D-glucamine Protective Recovery Method; citation_author=J.T. Ting; citation_author=B.R. Lee; citation_author=P. Chong; citation_author=G. Soler-Llavina; citation_author=C. Cobbs; citation_author=C. Koch; citation_author=H. Zeng; citation_author=E. Lein; citation_journal_title=J.&nbsp;Vis. Exp.; citation_number=132; citation_firstpage=53825; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Optimal tuning of weighted kNN- and diffusion-based methods for denoising single cell genomics data; citation_author=A. Tjärnberg; citation_author=O. Mahmood; citation_author=C.A. Jackson; citation_author=G.-A. Saldi; citation_author=K. Cho; citation_author=L.A. Christiaen; citation_author=R.A. Bonneau; citation_journal_title=bioRxiv; citation_doi=10.1101/2020.02.28.970202; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=A single-cell atlas of mouse brain macrophages reveals unique transcriptional identities shaped by ontogeny and tissue environment; citation_author=H. Van Hove; citation_author=L. Martens; citation_author=I. Scheyltjens; citation_author=K. De Vlaminck; citation_author=A.R. Pombo Antunes; citation_author=S. De Prijck; citation_author=N. Vandamme; citation_author=S. De Schepper; citation_author=G. Van Isterdael; citation_author=C.L. Scott; citation_author=et al.; citation_journal_title=Nat. Neurosci.; citation_volume=22; citation_firstpage=1021; citation_lastpage=1035; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Gating multiple signals through detailed balance of excitation and inhibition in spiking networks; citation_author=T.P. Vogels; citation_author=L.F. Abbott; citation_journal_title=Nat. Neurosci.; citation_volume=12; citation_firstpage=483; citation_lastpage=491; citation_publication_date=2009; "><meta name="citation_reference" content="citation_title=Inhibition of Microglial TGFβ Signaling Increases Expression of Mrc1; citation_author=A. von Ehr; citation_author=A. Attaai; citation_author=N. Neidert; citation_author=P.S. Potru; citation_author=T. Ruß; citation_author=T. Zöller; citation_author=B. Spittau; citation_journal_title=Front. Cell. Neurosci.; citation_volume=14; citation_firstpage=66; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Viral manipulation of functionally distinct interneurons in mice, non-human primates and humans; citation_author=D. Vormstein-Schneider; citation_author=J.D. Lin; citation_author=K.A. Pelkey; citation_author=R. Chittajallu; citation_author=B. Guo; citation_author=M.A. Arias-Garcia; citation_author=K. Allaway; citation_author=S. Sakopoulos; citation_author=G. Schneider; citation_author=O. Stevenson; citation_author=et al.; citation_journal_title=Nat. Neurosci.; citation_volume=23; citation_firstpage=1629; citation_lastpage=1636; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Rbfox1 Mediates Cell-type-Specific Splicing in Cortical Interneurons; citation_author=B. Wamsley; citation_author=X.H. Jaglin; citation_author=E. Favuzzi; citation_author=G. Quattrocolo; citation_author=M.J. Nigro; citation_author=N. Yusuf; citation_author=A. Khodadadi-Jamayran; citation_author=B. Rudy; citation_author=G. Fishell; citation_journal_title=Neuron; citation_volume=100; citation_firstpage=846; citation_lastpage=859.e7; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Defining the role of GABA in cortical development; citation_author=D.D. Wang; citation_author=A.R. Kriegstein; citation_journal_title=J.&nbsp;Physiol.; citation_volume=587; citation_firstpage=1873; citation_lastpage=1879; citation_publication_date=2009; "><meta name="citation_reference" content="citation_title=Neuron-Glia Signaling in Synapse Elimination; citation_author=D.K. Wilton; citation_author=L. Dissing-Olesen; citation_author=B. Stevens; citation_journal_title=Annu. Rev. Neurosci.; citation_volume=42; citation_firstpage=107; citation_lastpage=127; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Mapping Sub-Second Structure in Mouse Behavior; citation_author=A.B. Wiltschko; citation_author=M.J. Johnson; citation_author=G. Iurilli; citation_author=R.E. Peterson; citation_author=J.M. Katon; citation_author=S.L. Pashkovski; citation_author=V.E. Abraira; citation_author=R.P. Adams; citation_author=S.R. Datta; citation_journal_title=Neuron; citation_volume=88; citation_firstpage=1121; citation_lastpage=1135; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=SCANPY: large-scale single-cell gene expression data analysis; citation_author=F.A. Wolf; citation_author=P. Angerer; citation_author=F.J. Theis; citation_journal_title=Genome Biol.; citation_volume=19; citation_firstpage=15; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=GABA signaling promotes synapse elimination and axon pruning in developing cortical inhibitory interneurons; citation_author=X. Wu; citation_author=Y. Fu; citation_author=G. Knott; citation_author=J. Lu; citation_author=G. Di Cristo; citation_author=Z.J. Huang; citation_journal_title=J.&nbsp;Neurosci.; citation_volume=32; citation_firstpage=331; citation_lastpage=343; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=Functional characterization of mannose receptor expressed by immunocompetent mouse microglia; citation_author=H. Zimmer; citation_author=S. Riese; citation_author=A. Régnier-Vigouroux; citation_journal_title=Glia; citation_volume=42; citation_firstpage=89; citation_lastpage=100; citation_publication_date=2003; "><meta name="citation_abstract" content="<h2>Summary</h2><p>Microglia, the resident immune cells of the brain, have emerged as crucial regulators of synaptic refinement and brain wiring. However, whether the remodeling of distinct synapse types during development is mediated by specialized microglia is unknown. Here, we show that GABA-receptive microglia selectively interact with inhibitory cortical synapses during a critical window of mouse postnatal development. GABA initiates a transcriptional synapse remodeling program within these specialized microglia, which in turn sculpt inhibitory connectivity without impacting excitatory synapses. Ablation of GABA<sub>B</sub> receptors within microglia impairs this process and leads to behavioral abnormalities. These findings demonstrate that brain wiring relies on the selective communication between matched neuronal and glial cell types.</p>"><meta charset="UTF-8"><!--

    Pagebuild admin UI
-->



<script type="text/javascript" src="https://bam.nr-data.net/1/16e2192192?a=916957862&amp;sa=1&amp;v=1177.96a4d39&amp;t=Unnamed%20Transaction&amp;rst=8044&amp;ck=1&amp;ref=https://www.cell.com/cell/fulltext/S0092-8674(21)00753-4&amp;be=4422&amp;fe=7983&amp;dc=5869&amp;af=err,xhr,stn,ins,spa&amp;perf=%7B%22timing%22:%7B%22of%22:1749590954275,%22n%22:0,%22f%22:3857,%22dn%22:3858,%22dne%22:3858,%22c%22:3858,%22s%22:3858,%22ce%22:3858,%22rq%22:3858,%22rp%22:4398,%22rpe%22:5390,%22dl%22:4403,%22di%22:5713,%22ds%22:5868,%22de%22:5886,%22dc%22:7981,%22l%22:7982,%22le%22:8018%7D,%22navigation%22:%7B%7D%7D&amp;fp=4972&amp;fcp=4972&amp;jsonp=NREUM.setToken"></script><script src="https://js-agent.newrelic.com/nr-spa-1177.min.js"></script><script type="text/javascript">
    ;window.NREUM||(NREUM={});NREUM.init={privacy:{cookies_enabled:true}};
    window.NREUM||(NREUM={}),__nr_require=function(t,e,n){function r(n){if(!e[n]){var o=e[n]={exports:{}};t[n][0].call(o.exports,function(e){var o=t[n][1][e];return r(o||e)},o,o.exports)}return e[n].exports}if("function"==typeof __nr_require)return __nr_require;for(var o=0;o<n.length;o++)r(n[o]);return r}({1:[function(t,e,n){function r(t){try{c.console&&console.log(t)}catch(e){}}var o,i=t("ee"),a=t(26),c={};try{o=localStorage.getItem("__nr_flags").split(","),console&&"function"==typeof console.log&&(c.console=!0,o.indexOf("dev")!==-1&&(c.dev=!0),o.indexOf("nr_dev")!==-1&&(c.nrDev=!0))}catch(s){}c.nrDev&&i.on("internal-error",function(t){r(t.stack)}),c.dev&&i.on("fn-err",function(t,e,n){r(n.stack)}),c.dev&&(r("NR AGENT IN DEVELOPMENT MODE"),r("flags: "+a(c,function(t,e){return t}).join(", ")))},{}],2:[function(t,e,n){function r(t,e,n,r,c){try{l?l-=1:o(c||new UncaughtException(t,e,n),!0)}catch(f){try{i("ierr",[f,s.now(),!0])}catch(d){}}return"function"==typeof u&&u.apply(this,a(arguments))}function UncaughtException(t,e,n){this.message=t||"Uncaught error with no additional information",this.sourceURL=e,this.line=n}function o(t,e){var n=e?null:s.now();i("err",[t,n])}var i=t("handle"),a=t(27),c=t("ee"),s=t("loader"),f=t("gos"),u=window.onerror,d=!1,p="nr@seenError",l=0;s.features.err=!0,t(1),window.onerror=r;try{throw new Error}catch(h){"stack"in h&&(t(13),t(12),"addEventListener"in window&&t(6),s.xhrWrappable&&t(14),d=!0)}c.on("fn-start",function(t,e,n){d&&(l+=1)}),c.on("fn-err",function(t,e,n){d&&!n[p]&&(f(n,p,function(){return!0}),this.thrown=!0,o(n))}),c.on("fn-end",function(){d&&!this.thrown&&l>0&&(l-=1)}),c.on("internal-error",function(t){i("ierr",[t,s.now(),!0])})},{}],3:[function(t,e,n){t("loader").features.ins=!0},{}],4:[function(t,e,n){function r(){_++,T=g.hash,this[u]=y.now()}function o(){_--,g.hash!==T&&i(0,!0);var t=y.now();this[h]=~~this[h]+t-this[u],this[d]=t}function i(t,e){E.emit("newURL",[""+g,e])}function a(t,e){t.on(e,function(){this[e]=y.now()})}var c="-start",s="-end",f="-body",u="fn"+c,d="fn"+s,p="cb"+c,l="cb"+s,h="jsTime",m="fetch",v="addEventListener",w=window,g=w.location,y=t("loader");if(w[v]&&y.xhrWrappable){var x=t(10),b=t(11),E=t(8),R=t(6),O=t(13),N=t(7),M=t(14),P=t(9),S=t("ee"),C=S.get("tracer");t(16),y.features.spa=!0;var T,_=0;S.on(u,r),S.on(p,r),S.on(d,o),S.on(l,o),S.buffer([u,d,"xhr-done","xhr-resolved"]),R.buffer([u]),O.buffer(["setTimeout"+s,"clearTimeout"+c,u]),M.buffer([u,"new-xhr","send-xhr"+c]),N.buffer([m+c,m+"-done",m+f+c,m+f+s]),E.buffer(["newURL"]),x.buffer([u]),b.buffer(["propagate",p,l,"executor-err","resolve"+c]),C.buffer([u,"no-"+u]),P.buffer(["new-jsonp","cb-start","jsonp-error","jsonp-end"]),a(M,"send-xhr"+c),a(S,"xhr-resolved"),a(S,"xhr-done"),a(N,m+c),a(N,m+"-done"),a(P,"new-jsonp"),a(P,"jsonp-end"),a(P,"cb-start"),E.on("pushState-end",i),E.on("replaceState-end",i),w[v]("hashchange",i,!0),w[v]("load",i,!0),w[v]("popstate",function(){i(0,_>1)},!0)}},{}],5:[function(t,e,n){function r(t){}if(window.performance&&window.performance.timing&&window.performance.getEntriesByType){var o=t("ee"),i=t("handle"),a=t(13),c=t(12),s="learResourceTimings",f="addEventListener",u="resourcetimingbufferfull",d="bstResource",p="resource",l="-start",h="-end",m="fn"+l,v="fn"+h,w="bstTimer",g="pushState",y=t("loader");y.features.stn=!0,t(8),"addEventListener"in window&&t(6);var x=NREUM.o.EV;o.on(m,function(t,e){var n=t[0];n instanceof x&&(this.bstStart=y.now())}),o.on(v,function(t,e){var n=t[0];n instanceof x&&i("bst",[n,e,this.bstStart,y.now()])}),a.on(m,function(t,e,n){this.bstStart=y.now(),this.bstType=n}),a.on(v,function(t,e){i(w,[e,this.bstStart,y.now(),this.bstType])}),c.on(m,function(){this.bstStart=y.now()}),c.on(v,function(t,e){i(w,[e,this.bstStart,y.now(),"requestAnimationFrame"])}),o.on(g+l,function(t){this.time=y.now(),this.startPath=location.pathname+location.hash}),o.on(g+h,function(t){i("bstHist",[location.pathname+location.hash,this.startPath,this.time])}),f in window.performance&&(window.performance["c"+s]?window.performance[f](u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["c"+s]()},!1):window.performance[f]("webkit"+u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["webkitC"+s]()},!1)),document[f]("scroll",r,{passive:!0}),document[f]("keypress",r,!1),document[f]("click",r,!1)}},{}],6:[function(t,e,n){function r(t){for(var e=t;e&&!e.hasOwnProperty(u);)e=Object.getPrototypeOf(e);e&&o(e)}function o(t){c.inPlace(t,[u,d],"-",i)}function i(t,e){return t[1]}var a=t("ee").get("events"),c=t("wrap-function")(a,!0),s=t("gos"),f=XMLHttpRequest,u="addEventListener",d="removeEventListener";e.exports=a,"getPrototypeOf"in Object?(r(document),r(window),r(f.prototype)):f.prototype.hasOwnProperty(u)&&(o(window),o(f.prototype)),a.on(u+"-start",function(t,e){var n=t[1],r=s(n,"nr@wrapped",function(){function t(){if("function"==typeof n.handleEvent)return n.handleEvent.apply(n,arguments)}var e={object:t,"function":n}[typeof n];return e?c(e,"fn-",null,e.name||"anonymous"):n});this.wrapped=t[1]=r}),a.on(d+"-start",function(t){t[1]=this.wrapped||t[1]})},{}],7:[function(t,e,n){function r(t,e,n){var r=t[e];"function"==typeof r&&(t[e]=function(){var t=i(arguments),e={};o.emit(n+"before-start",[t],e);var a;e[m]&&e[m].dt&&(a=e[m].dt);var c=r.apply(this,t);return o.emit(n+"start",[t,a],c),c.then(function(t){return o.emit(n+"end",[null,t],c),t},function(t){throw o.emit(n+"end",[t],c),t})})}var o=t("ee").get("fetch"),i=t(27),a=t(26);e.exports=o;var c=window,s="fetch-",f=s+"body-",u=["arrayBuffer","blob","json","text","formData"],d=c.Request,p=c.Response,l=c.fetch,h="prototype",m="nr@context";d&&p&&l&&(a(u,function(t,e){r(d[h],e,f),r(p[h],e,f)}),r(c,"fetch",s),o.on(s+"end",function(t,e){var n=this;if(e){var r=e.headers.get("content-length");null!==r&&(n.rxSize=r),o.emit(s+"done",[null,e],n)}else o.emit(s+"done",[t],n)}))},{}],8:[function(t,e,n){var r=t("ee").get("history"),o=t("wrap-function")(r);e.exports=r;var i=window.history&&window.history.constructor&&window.history.constructor.prototype,a=window.history;i&&i.pushState&&i.replaceState&&(a=i),o.inPlace(a,["pushState","replaceState"],"-")},{}],9:[function(t,e,n){function r(t){function e(){s.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}function n(){s.emit("jsonp-error",[],p),s.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}var r=t&&"string"==typeof t.nodeName&&"script"===t.nodeName.toLowerCase();if(r){var o="function"==typeof t.addEventListener;if(o){var a=i(t.src);if(a){var u=c(a),d="function"==typeof u.parent[u.key];if(d){var p={};f.inPlace(u.parent,[u.key],"cb-",p),t.addEventListener("load",e,!1),t.addEventListener("error",n,!1),s.emit("new-jsonp",[t.src],p)}}}}}function o(){return"addEventListener"in window}function i(t){var e=t.match(u);return e?e[1]:null}function a(t,e){var n=t.match(p),r=n[1],o=n[3];return o?a(o,e[r]):e[r]}function c(t){var e=t.match(d);return e&&e.length>=3?{key:e[2],parent:a(e[1],window)}:{key:t,parent:window}}var s=t("ee").get("jsonp"),f=t("wrap-function")(s);if(e.exports=s,o()){var u=/[?&](?:callback|cb)=([^&#]+)/,d=/(.*)\.([^.]+)/,p=/^(\w+)(\.|$)(.*)$/,l=["appendChild","insertBefore","replaceChild"];Node&&Node.prototype&&Node.prototype.appendChild?f.inPlace(Node.prototype,l,"dom-"):(f.inPlace(HTMLElement.prototype,l,"dom-"),f.inPlace(HTMLHeadElement.prototype,l,"dom-"),f.inPlace(HTMLBodyElement.prototype,l,"dom-")),s.on("dom-start",function(t){r(t[0])})}},{}],10:[function(t,e,n){var r=t("ee").get("mutation"),o=t("wrap-function")(r),i=NREUM.o.MO;e.exports=r,i&&(window.MutationObserver=function(t){return this instanceof i?new i(o(t,"fn-")):i.apply(this,arguments)},MutationObserver.prototype=i.prototype)},{}],11:[function(t,e,n){function r(t){var e=a.context(),n=c(t,"executor-",e),r=new f(n);return a.context(r).getCtx=function(){return e},a.emit("new-promise",[r,e],e),r}function o(t,e){return e}var i=t("wrap-function"),a=t("ee").get("promise"),c=i(a),s=t(26),f=NREUM.o.PR;e.exports=a,f&&(window.Promise=r,["all","race"].forEach(function(t){var e=f[t];f[t]=function(n){function r(t){return function(){a.emit("propagate",[null,!o],i),o=o||!t}}var o=!1;s(n,function(e,n){Promise.resolve(n).then(r("all"===t),r(!1))});var i=e.apply(f,arguments),c=f.resolve(i);return c}}),["resolve","reject"].forEach(function(t){var e=f[t];f[t]=function(t){var n=e.apply(f,arguments);return t!==n&&a.emit("propagate",[t,!0],n),n}}),f.prototype["catch"]=function(t){return this.then(null,t)},f.prototype=Object.create(f.prototype,{constructor:{value:r}}),s(Object.getOwnPropertyNames(f),function(t,e){try{r[e]=f[e]}catch(n){}}),a.on("executor-start",function(t){t[0]=c(t[0],"resolve-",this),t[1]=c(t[1],"resolve-",this)}),a.on("executor-err",function(t,e,n){t[1](n)}),c.inPlace(f.prototype,["then"],"then-",o),a.on("then-start",function(t,e){this.promise=e,t[0]=c(t[0],"cb-",this),t[1]=c(t[1],"cb-",this)}),a.on("then-end",function(t,e,n){this.nextPromise=n;var r=this.promise;a.emit("propagate",[r,!0],n)}),a.on("cb-end",function(t,e,n){a.emit("propagate",[n,!0],this.nextPromise)}),a.on("propagate",function(t,e,n){this.getCtx&&!e||(this.getCtx=function(){if(t instanceof Promise)var e=a.context(t);return e&&e.getCtx?e.getCtx():this})}),r.toString=function(){return""+f})},{}],12:[function(t,e,n){var r=t("ee").get("raf"),o=t("wrap-function")(r),i="equestAnimationFrame";e.exports=r,o.inPlace(window,["r"+i,"mozR"+i,"webkitR"+i,"msR"+i],"raf-"),r.on("raf-start",function(t){t[0]=o(t[0],"fn-")})},{}],13:[function(t,e,n){function r(t,e,n){t[0]=a(t[0],"fn-",null,n)}function o(t,e,n){this.method=n,this.timerDuration=isNaN(t[1])?0:+t[1],t[0]=a(t[0],"fn-",this,n)}var i=t("ee").get("timer"),a=t("wrap-function")(i),c="setTimeout",s="setInterval",f="clearTimeout",u="-start",d="-";e.exports=i,a.inPlace(window,[c,"setImmediate"],c+d),a.inPlace(window,[s],s+d),a.inPlace(window,[f,"clearImmediate"],f+d),i.on(s+u,r),i.on(c+u,o)},{}],14:[function(t,e,n){function r(t,e){d.inPlace(e,["onreadystatechange"],"fn-",c)}function o(){var t=this,e=u.context(t);t.readyState>3&&!e.resolved&&(e.resolved=!0,u.emit("xhr-resolved",[],t)),d.inPlace(t,g,"fn-",c)}function i(t){y.push(t),h&&(b?b.then(a):v?v(a):(E=-E,R.data=E))}function a(){for(var t=0;t<y.length;t++)r([],y[t]);y.length&&(y=[])}function c(t,e){return e}function s(t,e){for(var n in t)e[n]=t[n];return e}t(6);var f=t("ee"),u=f.get("xhr"),d=t("wrap-function")(u),p=NREUM.o,l=p.XHR,h=p.MO,m=p.PR,v=p.SI,w="readystatechange",g=["onload","onerror","onabort","onloadstart","onloadend","onprogress","ontimeout"],y=[];e.exports=u;var x=window.XMLHttpRequest=function(t){var e=new l(t);try{u.emit("new-xhr",[e],e),e.addEventListener(w,o,!1)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}return e};if(s(l,x),x.prototype=l.prototype,d.inPlace(x.prototype,["open","send"],"-xhr-",c),u.on("send-xhr-start",function(t,e){r(t,e),i(e)}),u.on("open-xhr-start",r),h){var b=m&&m.resolve();if(!v&&!m){var E=1,R=document.createTextNode(E);new h(a).observe(R,{characterData:!0})}}else f.on("fn-end",function(t){t[0]&&t[0].type===w||a()})},{}],15:[function(t,e,n){function r(t){if(!c(t))return null;var e=window.NREUM;if(!e.loader_config)return null;var n=(e.loader_config.accountID||"").toString()||null,r=(e.loader_config.agentID||"").toString()||null,f=(e.loader_config.trustKey||"").toString()||null;if(!n||!r)return null;var h=l.generateSpanId(),m=l.generateTraceId(),v=Date.now(),w={spanId:h,traceId:m,timestamp:v};return(t.sameOrigin||s(t)&&p())&&(w.traceContextParentHeader=o(h,m),w.traceContextStateHeader=i(h,v,n,r,f)),(t.sameOrigin&&!u()||!t.sameOrigin&&s(t)&&d())&&(w.newrelicHeader=a(h,m,v,n,r,f)),w}function o(t,e){return"00-"+e+"-"+t+"-01"}function i(t,e,n,r,o){var i=0,a="",c=1,s="",f="";return o+"@nr="+i+"-"+c+"-"+n+"-"+r+"-"+t+"-"+a+"-"+s+"-"+f+"-"+e}function a(t,e,n,r,o,i){var a="btoa"in window&&"function"==typeof window.btoa;if(!a)return null;var c={v:[0,1],d:{ty:"Browser",ac:r,ap:o,id:t,tr:e,ti:n}};return i&&r!==i&&(c.d.tk=i),btoa(JSON.stringify(c))}function c(t){return f()&&s(t)}function s(t){var e=!1,n={};if("init"in NREUM&&"distributed_tracing"in NREUM.init&&(n=NREUM.init.distributed_tracing),t.sameOrigin)e=!0;else if(n.allowed_origins instanceof Array)for(var r=0;r<n.allowed_origins.length;r++){var o=h(n.allowed_origins[r]);if(t.hostname===o.hostname&&t.protocol===o.protocol&&t.port===o.port){e=!0;break}}return e}function f(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.enabled}function u(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.exclude_newrelic_header}function d(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&NREUM.init.distributed_tracing.cors_use_newrelic_header!==!1}function p(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.cors_use_tracecontext_headers}var l=t(23),h=t(17);e.exports={generateTracePayload:r,shouldGenerateTrace:c}},{}],16:[function(t,e,n){function r(t){var e=this.params,n=this.metrics;if(!this.ended){this.ended=!0;for(var r=0;r<p;r++)t.removeEventListener(d[r],this.listener,!1);e.aborted||(n.duration=a.now()-this.startTime,this.loadCaptureCalled||4!==t.readyState?null==e.status&&(e.status=0):i(this,t),n.cbTime=this.cbTime,u.emit("xhr-done",[t],t),c("xhr",[e,n,this.startTime]))}}function o(t,e){var n=s(e),r=t.params;r.host=n.hostname+":"+n.port,r.pathname=n.pathname,t.parsedOrigin=s(e),t.sameOrigin=t.parsedOrigin.sameOrigin}function i(t,e){t.params.status=e.status;var n=v(e,t.lastSize);if(n&&(t.metrics.rxSize=n),t.sameOrigin){var r=e.getResponseHeader("X-NewRelic-App-Data");r&&(t.params.cat=r.split(", ").pop())}t.loadCaptureCalled=!0}var a=t("loader");if(a.xhrWrappable){var c=t("handle"),s=t(17),f=t(15).generateTracePayload,u=t("ee"),d=["load","error","abort","timeout"],p=d.length,l=t("id"),h=t(21),m=t(20),v=t(18),w=window.XMLHttpRequest;a.features.xhr=!0,t(14),t(7),u.on("new-xhr",function(t){var e=this;e.totalCbs=0,e.called=0,e.cbTime=0,e.end=r,e.ended=!1,e.xhrGuids={},e.lastSize=null,e.loadCaptureCalled=!1,t.addEventListener("load",function(n){i(e,t)},!1),h&&(h>34||h<10)||window.opera||t.addEventListener("progress",function(t){e.lastSize=t.loaded},!1)}),u.on("open-xhr-start",function(t){this.params={method:t[0]},o(this,t[1]),this.metrics={}}),u.on("open-xhr-end",function(t,e){"loader_config"in NREUM&&"xpid"in NREUM.loader_config&&this.sameOrigin&&e.setRequestHeader("X-NewRelic-ID",NREUM.loader_config.xpid);var n=f(this.parsedOrigin);if(n){var r=!1;n.newrelicHeader&&(e.setRequestHeader("newrelic",n.newrelicHeader),r=!0),n.traceContextParentHeader&&(e.setRequestHeader("traceparent",n.traceContextParentHeader),n.traceContextStateHeader&&e.setRequestHeader("tracestate",n.traceContextStateHeader),r=!0),r&&(this.dt=n)}}),u.on("send-xhr-start",function(t,e){var n=this.metrics,r=t[0],o=this;if(n&&r){var i=m(r);i&&(n.txSize=i)}this.startTime=a.now(),this.listener=function(t){try{"abort"!==t.type||o.loadCaptureCalled||(o.params.aborted=!0),("load"!==t.type||o.called===o.totalCbs&&(o.onloadCalled||"function"!=typeof e.onload))&&o.end(e)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}};for(var c=0;c<p;c++)e.addEventListener(d[c],this.listener,!1)}),u.on("xhr-cb-time",function(t,e,n){this.cbTime+=t,e?this.onloadCalled=!0:this.called+=1,this.called!==this.totalCbs||!this.onloadCalled&&"function"==typeof n.onload||this.end(n)}),u.on("xhr-load-added",function(t,e){var n=""+l(t)+!!e;this.xhrGuids&&!this.xhrGuids[n]&&(this.xhrGuids[n]=!0,this.totalCbs+=1)}),u.on("xhr-load-removed",function(t,e){var n=""+l(t)+!!e;this.xhrGuids&&this.xhrGuids[n]&&(delete this.xhrGuids[n],this.totalCbs-=1)}),u.on("addEventListener-end",function(t,e){e instanceof w&&"load"===t[0]&&u.emit("xhr-load-added",[t[1],t[2]],e)}),u.on("removeEventListener-end",function(t,e){e instanceof w&&"load"===t[0]&&u.emit("xhr-load-removed",[t[1],t[2]],e)}),u.on("fn-start",function(t,e,n){e instanceof w&&("onload"===n&&(this.onload=!0),("load"===(t[0]&&t[0].type)||this.onload)&&(this.xhrCbStart=a.now()))}),u.on("fn-end",function(t,e){this.xhrCbStart&&u.emit("xhr-cb-time",[a.now()-this.xhrCbStart,this.onload,e],e)}),u.on("fetch-before-start",function(t){function e(t,e){var n=!1;return e.newrelicHeader&&(t.set("newrelic",e.newrelicHeader),n=!0),e.traceContextParentHeader&&(t.set("traceparent",e.traceContextParentHeader),e.traceContextStateHeader&&t.set("tracestate",e.traceContextStateHeader),n=!0),n}var n,r=t[1]||{};"string"==typeof t[0]?n=t[0]:t[0]&&t[0].url&&(n=t[0].url),n&&(this.parsedOrigin=s(n),this.sameOrigin=this.parsedOrigin.sameOrigin);var o=f(this.parsedOrigin);if(o&&(o.newrelicHeader||o.traceContextParentHeader))if("string"==typeof t[0]){var i={};for(var a in r)i[a]=r[a];i.headers=new Headers(r.headers||{}),e(i.headers,o)&&(this.dt=o),t.length>1?t[1]=i:t.push(i)}else t[0]&&t[0].headers&&e(t[0].headers,o)&&(this.dt=o)})}},{}],17:[function(t,e,n){var r={};e.exports=function(t){if(t in r)return r[t];var e=document.createElement("a"),n=window.location,o={};e.href=t,o.port=e.port;var i=e.href.split("://");!o.port&&i[1]&&(o.port=i[1].split("/")[0].split("@").pop().split(":")[1]),o.port&&"0"!==o.port||(o.port="https"===i[0]?"443":"80"),o.hostname=e.hostname||n.hostname,o.pathname=e.pathname,o.protocol=i[0],"/"!==o.pathname.charAt(0)&&(o.pathname="/"+o.pathname);var a=!e.protocol||":"===e.protocol||e.protocol===n.protocol,c=e.hostname===document.domain&&e.port===n.port;return o.sameOrigin=a&&(!e.hostname||c),"/"===o.pathname&&(r[t]=o),o}},{}],18:[function(t,e,n){function r(t,e){var n=t.responseType;return"json"===n&&null!==e?e:"arraybuffer"===n||"blob"===n||"json"===n?o(t.response):"text"===n||""===n||void 0===n?o(t.responseText):void 0}var o=t(20);e.exports=r},{}],19:[function(t,e,n){function r(){}function o(t,e,n){return function(){return i(t,[f.now()].concat(c(arguments)),e?null:this,n),e?void 0:this}}var i=t("handle"),a=t(26),c=t(27),s=t("ee").get("tracer"),f=t("loader"),u=NREUM;"undefined"==typeof window.newrelic&&(newrelic=u);var d=["setPageViewName","setCustomAttribute","setErrorHandler","finished","addToTrace","inlineHit","addRelease"],p="api-",l=p+"ixn-";a(d,function(t,e){u[e]=o(p+e,!0,"api")}),u.addPageAction=o(p+"addPageAction",!0),u.setCurrentRouteName=o(p+"routeName",!0),e.exports=newrelic,u.interaction=function(){return(new r).get()};var h=r.prototype={createTracer:function(t,e){var n={},r=this,o="function"==typeof e;return i(l+"tracer",[f.now(),t,n],r),function(){if(s.emit((o?"":"no-")+"fn-start",[f.now(),r,o],n),o)try{return e.apply(this,arguments)}catch(t){throw s.emit("fn-err",[arguments,this,t],n),t}finally{s.emit("fn-end",[f.now()],n)}}}};a("actionText,setName,setAttribute,save,ignore,onEnd,getContext,end,get".split(","),function(t,e){h[e]=o(l+e)}),newrelic.noticeError=function(t,e){"string"==typeof t&&(t=new Error(t)),i("err",[t,f.now(),!1,e])}},{}],20:[function(t,e,n){e.exports=function(t){if("string"==typeof t&&t.length)return t.length;if("object"==typeof t){if("undefined"!=typeof ArrayBuffer&&t instanceof ArrayBuffer&&t.byteLength)return t.byteLength;if("undefined"!=typeof Blob&&t instanceof Blob&&t.size)return t.size;if(!("undefined"!=typeof FormData&&t instanceof FormData))try{return JSON.stringify(t).length}catch(e){return}}}},{}],21:[function(t,e,n){var r=0,o=navigator.userAgent.match(/Firefox[\/\s](\d+\.\d+)/);o&&(r=+o[1]),e.exports=r},{}],22:[function(t,e,n){function r(t,e){var n=t.getEntries();n.forEach(function(t){"first-paint"===t.name?d("timing",["fp",Math.floor(t.startTime)]):"first-contentful-paint"===t.name&&d("timing",["fcp",Math.floor(t.startTime)])})}function o(t,e){var n=t.getEntries();n.length>0&&d("lcp",[n[n.length-1]])}function i(t){t.getEntries().forEach(function(t){t.hadRecentInput||d("cls",[t])})}function a(t){if(t instanceof h&&!v){var e=Math.round(t.timeStamp),n={type:t.type};e<=p.now()?n.fid=p.now()-e:e>p.offset&&e<=Date.now()?(e-=p.offset,n.fid=p.now()-e):e=p.now(),v=!0,d("timing",["fi",e,n])}}function c(t){d("pageHide",[p.now(),t])}if(!("init"in NREUM&&"page_view_timing"in NREUM.init&&"enabled"in NREUM.init.page_view_timing&&NREUM.init.page_view_timing.enabled===!1)){var s,f,u,d=t("handle"),p=t("loader"),l=t(25),h=NREUM.o.EV;if("PerformanceObserver"in window&&"function"==typeof window.PerformanceObserver){s=new PerformanceObserver(r);try{s.observe({entryTypes:["paint"]})}catch(m){}f=new PerformanceObserver(o);try{f.observe({entryTypes:["largest-contentful-paint"]})}catch(m){}u=new PerformanceObserver(i);try{u.observe({type:"layout-shift",buffered:!0})}catch(m){}}if("addEventListener"in document){var v=!1,w=["click","keydown","mousedown","pointerdown","touchstart"];w.forEach(function(t){document.addEventListener(t,a,!1)})}l(c)}},{}],23:[function(t,e,n){function r(){function t(){return e?15&e[n++]:16*Math.random()|0}var e=null,n=0,r=window.crypto||window.msCrypto;r&&r.getRandomValues&&(e=r.getRandomValues(new Uint8Array(31)));for(var o,i="xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx",a="",c=0;c<i.length;c++)o=i[c],"x"===o?a+=t().toString(16):"y"===o?(o=3&t()|8,a+=o.toString(16)):a+=o;return a}function o(){return a(16)}function i(){return a(32)}function a(t){function e(){return n?15&n[r++]:16*Math.random()|0}var n=null,r=0,o=window.crypto||window.msCrypto;o&&o.getRandomValues&&Uint8Array&&(n=o.getRandomValues(new Uint8Array(31)));for(var i=[],a=0;a<t;a++)i.push(e().toString(16));return i.join("")}e.exports={generateUuid:r,generateSpanId:o,generateTraceId:i}},{}],24:[function(t,e,n){function r(t,e){if(!o)return!1;if(t!==o)return!1;if(!e)return!0;if(!i)return!1;for(var n=i.split("."),r=e.split("."),a=0;a<r.length;a++)if(r[a]!==n[a])return!1;return!0}var o=null,i=null,a=/Version\/(\S+)\s+Safari/;if(navigator.userAgent){var c=navigator.userAgent,s=c.match(a);s&&c.indexOf("Chrome")===-1&&c.indexOf("Chromium")===-1&&(o="Safari",i=s[1])}e.exports={agent:o,version:i,match:r}},{}],25:[function(t,e,n){function r(t){function e(){t(a&&document[a]?document[a]:document[o]?"hidden":"visible")}"addEventListener"in document&&i&&document.addEventListener(i,e,!1)}e.exports=r;var o,i,a;"undefined"!=typeof document.hidden?(o="hidden",i="visibilitychange",a="visibilityState"):"undefined"!=typeof document.msHidden?(o="msHidden",i="msvisibilitychange"):"undefined"!=typeof document.webkitHidden&&(o="webkitHidden",i="webkitvisibilitychange",a="webkitVisibilityState")},{}],26:[function(t,e,n){function r(t,e){var n=[],r="",i=0;for(r in t)o.call(t,r)&&(n[i]=e(r,t[r]),i+=1);return n}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],27:[function(t,e,n){function r(t,e,n){e||(e=0),"undefined"==typeof n&&(n=t?t.length:0);for(var r=-1,o=n-e||0,i=Array(o<0?0:o);++r<o;)i[r]=t[e+r];return i}e.exports=r},{}],28:[function(t,e,n){e.exports={exists:"undefined"!=typeof window.performance&&window.performance.timing&&"undefined"!=typeof window.performance.timing.navigationStart}},{}],ee:[function(t,e,n){function r(){}function o(t){function e(t){return t&&t instanceof r?t:t?s(t,c,i):i()}function n(n,r,o,i){if(!p.aborted||i){t&&t(n,r,o);for(var a=e(o),c=m(n),s=c.length,f=0;f<s;f++)c[f].apply(a,r);var d=u[y[n]];return d&&d.push([x,n,r,a]),a}}function l(t,e){g[t]=m(t).concat(e)}function h(t,e){var n=g[t];if(n)for(var r=0;r<n.length;r++)n[r]===e&&n.splice(r,1)}function m(t){return g[t]||[]}function v(t){return d[t]=d[t]||o(n)}function w(t,e){f(t,function(t,n){e=e||"feature",y[n]=e,e in u||(u[e]=[])})}var g={},y={},x={on:l,addEventListener:l,removeEventListener:h,emit:n,get:v,listeners:m,context:e,buffer:w,abort:a,aborted:!1};return x}function i(){return new r}function a(){(u.api||u.feature)&&(p.aborted=!0,u=p.backlog={})}var c="nr@context",s=t("gos"),f=t(26),u={},d={},p=e.exports=o();p.backlog=u},{}],gos:[function(t,e,n){function r(t,e,n){if(o.call(t,e))return t[e];var r=n();if(Object.defineProperty&&Object.keys)try{return Object.defineProperty(t,e,{value:r,writable:!0,enumerable:!1}),r}catch(i){}return t[e]=r,r}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],handle:[function(t,e,n){function r(t,e,n,r){o.buffer([t],r),o.emit(t,e,n)}var o=t("ee").get("handle");e.exports=r,r.ee=o},{}],id:[function(t,e,n){function r(t){var e=typeof t;return!t||"object"!==e&&"function"!==e?-1:t===window?0:a(t,i,function(){return o++})}var o=1,i="nr@id",a=t("gos");e.exports=r},{}],loader:[function(t,e,n){function r(){if(!E++){var t=b.info=NREUM.info,e=l.getElementsByTagName("script")[0];if(setTimeout(u.abort,3e4),!(t&&t.licenseKey&&t.applicationID&&e))return u.abort();f(y,function(e,n){t[e]||(t[e]=n)});var n=a();s("mark",["onload",n+b.offset],null,"api"),s("timing",["load",n]);var r=l.createElement("script");r.src="https://"+t.agent,e.parentNode.insertBefore(r,e)}}function o(){"complete"===l.readyState&&i()}function i(){s("mark",["domContent",a()+b.offset],null,"api")}function a(){return R.exists&&performance.now?Math.round(performance.now()):(c=Math.max((new Date).getTime(),c))-b.offset}var c=(new Date).getTime(),s=t("handle"),f=t(26),u=t("ee"),d=t(24),p=window,l=p.document,h="addEventListener",m="attachEvent",v=p.XMLHttpRequest,w=v&&v.prototype;NREUM.o={ST:setTimeout,SI:p.setImmediate,CT:clearTimeout,XHR:v,REQ:p.Request,EV:p.Event,PR:p.Promise,MO:p.MutationObserver};var g=""+location,y={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",agent:"js-agent.newrelic.com/nr-spa-1177.min.js"},x=v&&w&&w[h]&&!/CriOS/.test(navigator.userAgent),b=e.exports={offset:c,now:a,origin:g,features:{},xhrWrappable:x,userAgent:d};t(19),t(22),l[h]?(l[h]("DOMContentLoaded",i,!1),p[h]("load",r,!1)):(l[m]("onreadystatechange",o),p[m]("onload",r)),s("mark",["firstbyte",c],null,"api");var E=0,R=t(28)},{}],"wrap-function":[function(t,e,n){function r(t){return!(t&&t instanceof Function&&t.apply&&!t[a])}var o=t("ee"),i=t(27),a="nr@original",c=Object.prototype.hasOwnProperty,s=!1;e.exports=function(t,e){function n(t,e,n,o){function nrWrapper(){var r,a,c,s;try{a=this,r=i(arguments),c="function"==typeof n?n(r,a):n||{}}catch(f){p([f,"",[r,a,o],c])}u(e+"start",[r,a,o],c);try{return s=t.apply(a,r)}catch(d){throw u(e+"err",[r,a,d],c),d}finally{u(e+"end",[r,a,s],c)}}return r(t)?t:(e||(e=""),nrWrapper[a]=t,d(t,nrWrapper),nrWrapper)}function f(t,e,o,i){o||(o="");var a,c,s,f="-"===o.charAt(0);for(s=0;s<e.length;s++)c=e[s],a=t[c],r(a)||(t[c]=n(a,f?c+o:o,i,c))}function u(n,r,o){if(!s||e){var i=s;s=!0;try{t.emit(n,r,o,e)}catch(a){p([a,n,r,o])}s=i}}function d(t,e){if(Object.defineProperty&&Object.keys)try{var n=Object.keys(t);return n.forEach(function(n){Object.defineProperty(e,n,{get:function(){return t[n]},set:function(e){return t[n]=e,e}})}),e}catch(r){p([r])}for(var o in t)c.call(t,o)&&(e[o]=t[o]);return e}function p(e){try{t.emit("internal-error",e)}catch(n){}}return t||(t=o),n.inPlace=f,n.flag=a,n}},{}]},{},["loader",2,16,5,3,4]);
    ;NREUM.loader_config={accountID:"2095297",trustKey:"2038175",agentID:"916957862",licenseKey:"16e2192192",applicationID:"916957862"}
    ;NREUM.info={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",licenseKey:"16e2192192",applicationID:"916957862",sa:1}
</script>
    <script src="https://cdn.cookielaw.org/scripttemplates/otSDKStub.js" type="text/javascript" charset="UTF-8" data-domain-script="dada9834-04e2-4bcd-b5ba-cae44d498dfa"></script>
    <script type="text/javascript">
        function getCookie(name) {
            var value = "; " + document.cookie;
            var parts = value.split("; " + name + "=");
            if (parts.length == 2) {
                return true;
            }
        }
            function OptanonWrapper() {
                var optCookie = getCookie("OptanonAlertBoxClosed");
                var isActiveCategory = "false";
                var isFoundCookieBox = "false";
                if(isActiveCategory == 'false' && isFoundCookieBox == 'false' && optCookie) {
                    OneTrust.InsertScript('/pb-assets/adobe/prod/adobedtm-1616606406233.js', 'head', null, null, '2');
                }
            }
    </script>


<noscript>
    <style>.requiresJS {
        display: none
    }

    .dartAd {
        display: block !important
    }</style>
</noscript>
        <script type="text/javascript" src="/wro/o4ny~selected-legacy.js"></script>
        <link rel="stylesheet" type="text/css" href="/wro/o4ny~selected-legacy.css">
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Open+Sans">
        <script>window.MSInputMethodContext && document.documentMode && document.write('<script src="https://cdn.jsdelivr.net/gh/nuxodin/ie11CustomProperties@4.1.0/ie11CustomProperties.min.js"><\x2fscript>');</script><script src="https://cdn.cookielaw.org/scripttemplates/202411.2.0/otBannerSdk.js" async="" type="text/javascript"></script><link rel="stylesheet" href="/products/marlin/cell/releasedAssets/css/build-b26f3342b722959c7931.css"><link rel="stylesheet" href="/products/marlin/cell/releasedAssets/css/print-b26f3342b722959c7931.css" media="print"><link rel="stylesheet" href="/products/marlin/cell/releasedAssets/css/themes/theme-cell.css">







        
        
        <script class="optanon-category-4" type="text/plain" async="" src="https://www.youtube.com/iframe_api"></script>
        
    





<!---->
<meta http-equiv="X-UA-Compatible" content="IE=edge">












<style id="onetrust-style">#onetrust-banner-sdk .onetrust-vendors-list-handler{cursor:pointer;color:#1f96db;font-size:inherit;font-weight:bold;text-decoration:none;margin-left:5px}#onetrust-banner-sdk .onetrust-vendors-list-handler:hover{color:#1f96db}#onetrust-banner-sdk:focus{outline:2px solid #000;outline-offset:-2px}#onetrust-banner-sdk a:focus{outline:2px solid #000}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{outline-offset:1px}#onetrust-banner-sdk.ot-bnr-w-logo .ot-bnr-logo{height:64px;width:64px}#onetrust-banner-sdk .ot-tcf2-vendor-count.ot-text-bold{font-weight:bold}#onetrust-banner-sdk .ot-close-icon,#onetrust-pc-sdk .ot-close-icon,#ot-sync-ntfy .ot-close-icon{background-size:contain;background-repeat:no-repeat;background-position:center;height:12px;width:12px}#onetrust-banner-sdk .powered-by-logo,#onetrust-banner-sdk .ot-pc-footer-logo a,#onetrust-pc-sdk .powered-by-logo,#onetrust-pc-sdk .ot-pc-footer-logo a,#ot-sync-ntfy .powered-by-logo,#ot-sync-ntfy .ot-pc-footer-logo a{background-size:contain;background-repeat:no-repeat;background-position:center;height:25px;width:152px;display:block;text-decoration:none;font-size:.75em}#onetrust-banner-sdk .powered-by-logo:hover,#onetrust-banner-sdk .ot-pc-footer-logo a:hover,#onetrust-pc-sdk .powered-by-logo:hover,#onetrust-pc-sdk .ot-pc-footer-logo a:hover,#ot-sync-ntfy .powered-by-logo:hover,#ot-sync-ntfy .ot-pc-footer-logo a:hover{color:#565656}#onetrust-banner-sdk h3 *,#onetrust-banner-sdk h4 *,#onetrust-banner-sdk h6 *,#onetrust-banner-sdk button *,#onetrust-banner-sdk a[data-parent-id] *,#onetrust-pc-sdk h3 *,#onetrust-pc-sdk h4 *,#onetrust-pc-sdk h6 *,#onetrust-pc-sdk button *,#onetrust-pc-sdk a[data-parent-id] *,#ot-sync-ntfy h3 *,#ot-sync-ntfy h4 *,#ot-sync-ntfy h6 *,#ot-sync-ntfy button *,#ot-sync-ntfy a[data-parent-id] *{font-size:inherit;font-weight:inherit;color:inherit}#onetrust-banner-sdk .ot-hide,#onetrust-pc-sdk .ot-hide,#ot-sync-ntfy .ot-hide{display:none !important}#onetrust-banner-sdk button.ot-link-btn:hover,#onetrust-pc-sdk button.ot-link-btn:hover,#ot-sync-ntfy button.ot-link-btn:hover{text-decoration:underline;opacity:1}#onetrust-pc-sdk .ot-sdk-row .ot-sdk-column{padding:0}#onetrust-pc-sdk .ot-sdk-container{padding-right:0}#onetrust-pc-sdk .ot-sdk-row{flex-direction:initial;width:100%}#onetrust-pc-sdk [type=checkbox]:checked,#onetrust-pc-sdk [type=checkbox]:not(:checked){pointer-events:initial}#onetrust-pc-sdk [type=checkbox]:disabled+label::before,#onetrust-pc-sdk [type=checkbox]:disabled+label:after,#onetrust-pc-sdk [type=checkbox]:disabled+label{pointer-events:none;opacity:.8}#onetrust-pc-sdk #vendor-list-content{transform:translate3d(0, 0, 0)}#onetrust-pc-sdk li input[type=checkbox]{z-index:1}#onetrust-pc-sdk li .ot-checkbox label{z-index:2}#onetrust-pc-sdk li .ot-checkbox input[type=checkbox]{height:auto;width:auto}#onetrust-pc-sdk li .host-title a,#onetrust-pc-sdk li .ot-host-name a,#onetrust-pc-sdk li .accordion-text,#onetrust-pc-sdk li .ot-acc-txt{z-index:2;position:relative}#onetrust-pc-sdk input{margin:3px .1ex}#onetrust-pc-sdk .pc-logo,#onetrust-pc-sdk .ot-pc-logo{height:60px;width:180px;background-position:center;background-size:contain;background-repeat:no-repeat;display:inline-flex;justify-content:center;align-items:center}#onetrust-pc-sdk .pc-logo img,#onetrust-pc-sdk .ot-pc-logo img{max-height:100%;max-width:100%}#onetrust-pc-sdk .screen-reader-only,#onetrust-pc-sdk .ot-scrn-rdr,.ot-sdk-cookie-policy .screen-reader-only,.ot-sdk-cookie-policy .ot-scrn-rdr{border:0;clip:rect(0 0 0 0);height:1px;margin:-1px;overflow:hidden;padding:0;position:absolute;width:1px}#onetrust-pc-sdk.ot-fade-in,.onetrust-pc-dark-filter.ot-fade-in,#onetrust-banner-sdk.ot-fade-in{animation-name:onetrust-fade-in;animation-duration:400ms;animation-timing-function:ease-in-out}#onetrust-pc-sdk.ot-hide{display:none !important}.onetrust-pc-dark-filter.ot-hide{display:none !important}#ot-sdk-btn.ot-sdk-show-settings,#ot-sdk-btn.optanon-show-settings{color:#fff;background-color:#468254;height:auto;white-space:normal;word-wrap:break-word;padding:.8em 2em;font-size:.8em;line-height:1.2;cursor:pointer;-moz-transition:.1s ease;-o-transition:.1s ease;-webkit-transition:1s ease;transition:.1s ease}#ot-sdk-btn.ot-sdk-show-settings:hover,#ot-sdk-btn.optanon-show-settings:hover{color:#fff;background-color:#2c6415}#ot-sdk-btn.ot-sdk-show-settings:active,#ot-sdk-btn.optanon-show-settings:active{color:#fff;background-color:#2c6415;border:1px solid rgba(162,192,169,.5)}.onetrust-pc-dark-filter{background:rgba(0,0,0,.5);z-index:2147483646;width:100%;height:100%;overflow:hidden;position:fixed;top:0;bottom:0;left:0}@keyframes onetrust-fade-in{0%{opacity:0}100%{opacity:1}}.ot-cookie-label{text-decoration:underline}@media only screen and (min-width: 426px)and (max-width: 896px)and (orientation: landscape){#onetrust-pc-sdk p{font-size:.75em}}#onetrust-banner-sdk .banner-option-input:focus+label{outline:1px solid #000;outline-style:auto}.category-vendors-list-handler+a:focus,.category-vendors-list-handler+a:focus-visible{outline:2px solid #000}#onetrust-pc-sdk .ot-userid-title{margin-top:10px}#onetrust-pc-sdk .ot-userid-title>span,#onetrust-pc-sdk .ot-userid-timestamp>span{font-weight:700}#onetrust-pc-sdk .ot-userid-desc{font-style:italic}#onetrust-pc-sdk .ot-host-desc a{pointer-events:initial}#onetrust-pc-sdk .ot-ven-hdr>p a{position:relative;z-index:2;pointer-events:initial}#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info a,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info a{margin-right:auto}#onetrust-pc-sdk .ot-pc-footer-logo img{width:136px;height:16px}#onetrust-pc-sdk .ot-pur-vdr-count{font-weight:400;font-size:.7rem;padding-top:3px;display:block}#onetrust-banner-sdk .ot-optout-signal,#onetrust-pc-sdk .ot-optout-signal{border:1px solid #32ae88;border-radius:3px;padding:5px;margin-bottom:10px;background-color:#f9fffa;font-size:.85rem;line-height:2}#onetrust-banner-sdk .ot-optout-signal .ot-optout-icon,#onetrust-pc-sdk .ot-optout-signal .ot-optout-icon{display:inline;margin-right:5px}#onetrust-banner-sdk .ot-optout-signal svg,#onetrust-pc-sdk .ot-optout-signal svg{height:20px;width:30px;transform:scale(0.5)}#onetrust-banner-sdk .ot-optout-signal svg path,#onetrust-pc-sdk .ot-optout-signal svg path{fill:#32ae88}#onetrust-consent-sdk .ot-general-modal{overflow:hidden;position:fixed;margin:0 auto;top:50%;left:50%;width:40%;padding:1.5rem;max-width:575px;min-width:575px;z-index:2147483647;border-radius:2.5px;transform:translate(-50%, -50%)}#onetrust-consent-sdk .ot-signature-health-group{margin-top:1rem;padding-left:1.25rem;padding-right:1.25rem;margin-bottom:.625rem;width:calc(100% - 2.5rem)}#onetrust-consent-sdk .ot-signature-health-group .ot-signature-health-form{gap:.5rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-health-form{width:70%;gap:.35rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-input{height:38px;padding:6px 10px;background-color:#fff;border:1px solid #d1d1d1;border-radius:4px;box-shadow:none;box-sizing:border-box}#onetrust-consent-sdk .ot-signature-health .ot-signature-subtitle{font-size:1.125rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-group-title{font-size:1.25rem;font-weight:bold}#onetrust-consent-sdk .ot-signature-health,#onetrust-consent-sdk .ot-signature-health-group{display:flex;flex-direction:column;gap:1rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-cont,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-cont{display:flex;flex-direction:column;gap:.25rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-paragraph,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-paragraph{margin:0;line-height:20px;font-size:max(14px,.875rem)}#onetrust-consent-sdk .ot-signature-health .ot-health-signature-error,#onetrust-consent-sdk .ot-signature-health-group .ot-health-signature-error{color:#4d4d4d;font-size:min(12px,.75rem)}#onetrust-consent-sdk .ot-signature-health .ot-signature-buttons-cont,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-buttons-cont{margin-top:max(.75rem,2%);gap:1rem;display:flex;justify-content:flex-end}#onetrust-consent-sdk .ot-signature-health .ot-signature-button,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-button{flex:1;height:auto;color:#fff;cursor:pointer;line-height:1.2;min-width:125px;font-weight:600;font-size:.813em;border-radius:2px;padding:12px 10px;white-space:normal;word-wrap:break-word;word-break:break-word;background-color:#68b631;border:2px solid #68b631}#onetrust-consent-sdk .ot-signature-health .ot-signature-button.reject,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-button.reject{background-color:#fff}#onetrust-consent-sdk .ot-input-field-cont{display:flex;flex-direction:column;gap:.5rem}#onetrust-consent-sdk .ot-input-field-cont .ot-signature-input{width:65%}#onetrust-consent-sdk .ot-signature-health-form{display:flex;flex-direction:column}#onetrust-consent-sdk .ot-signature-health-form .ot-signature-label{margin-bottom:0;line-height:20px;font-size:max(14px,.875rem)}@media only screen and (max-width: 600px){#onetrust-consent-sdk .ot-general-modal{min-width:100%}#onetrust-consent-sdk .ot-signature-health .ot-signature-health-form{width:100%}#onetrust-consent-sdk .ot-input-field-cont .ot-signature-input{width:100%}}#onetrust-banner-sdk,#onetrust-pc-sdk,#ot-sdk-cookie-policy,#ot-sync-ntfy{font-size:16px}#onetrust-banner-sdk *,#onetrust-banner-sdk ::after,#onetrust-banner-sdk ::before,#onetrust-pc-sdk *,#onetrust-pc-sdk ::after,#onetrust-pc-sdk ::before,#ot-sdk-cookie-policy *,#ot-sdk-cookie-policy ::after,#ot-sdk-cookie-policy ::before,#ot-sync-ntfy *,#ot-sync-ntfy ::after,#ot-sync-ntfy ::before{-webkit-box-sizing:content-box;-moz-box-sizing:content-box;box-sizing:content-box}#onetrust-banner-sdk div,#onetrust-banner-sdk span,#onetrust-banner-sdk h1,#onetrust-banner-sdk h2,#onetrust-banner-sdk h3,#onetrust-banner-sdk h4,#onetrust-banner-sdk h5,#onetrust-banner-sdk h6,#onetrust-banner-sdk p,#onetrust-banner-sdk img,#onetrust-banner-sdk svg,#onetrust-banner-sdk button,#onetrust-banner-sdk section,#onetrust-banner-sdk a,#onetrust-banner-sdk label,#onetrust-banner-sdk input,#onetrust-banner-sdk ul,#onetrust-banner-sdk li,#onetrust-banner-sdk nav,#onetrust-banner-sdk table,#onetrust-banner-sdk thead,#onetrust-banner-sdk tr,#onetrust-banner-sdk td,#onetrust-banner-sdk tbody,#onetrust-banner-sdk .ot-main-content,#onetrust-banner-sdk .ot-toggle,#onetrust-banner-sdk #ot-content,#onetrust-banner-sdk #ot-pc-content,#onetrust-banner-sdk .checkbox,#onetrust-pc-sdk div,#onetrust-pc-sdk span,#onetrust-pc-sdk h1,#onetrust-pc-sdk h2,#onetrust-pc-sdk h3,#onetrust-pc-sdk h4,#onetrust-pc-sdk h5,#onetrust-pc-sdk h6,#onetrust-pc-sdk p,#onetrust-pc-sdk img,#onetrust-pc-sdk svg,#onetrust-pc-sdk button,#onetrust-pc-sdk section,#onetrust-pc-sdk a,#onetrust-pc-sdk label,#onetrust-pc-sdk input,#onetrust-pc-sdk ul,#onetrust-pc-sdk li,#onetrust-pc-sdk nav,#onetrust-pc-sdk table,#onetrust-pc-sdk thead,#onetrust-pc-sdk tr,#onetrust-pc-sdk td,#onetrust-pc-sdk tbody,#onetrust-pc-sdk .ot-main-content,#onetrust-pc-sdk .ot-toggle,#onetrust-pc-sdk #ot-content,#onetrust-pc-sdk #ot-pc-content,#onetrust-pc-sdk .checkbox,#ot-sdk-cookie-policy div,#ot-sdk-cookie-policy span,#ot-sdk-cookie-policy h1,#ot-sdk-cookie-policy h2,#ot-sdk-cookie-policy h3,#ot-sdk-cookie-policy h4,#ot-sdk-cookie-policy h5,#ot-sdk-cookie-policy h6,#ot-sdk-cookie-policy p,#ot-sdk-cookie-policy img,#ot-sdk-cookie-policy svg,#ot-sdk-cookie-policy button,#ot-sdk-cookie-policy section,#ot-sdk-cookie-policy a,#ot-sdk-cookie-policy label,#ot-sdk-cookie-policy input,#ot-sdk-cookie-policy ul,#ot-sdk-cookie-policy li,#ot-sdk-cookie-policy nav,#ot-sdk-cookie-policy table,#ot-sdk-cookie-policy thead,#ot-sdk-cookie-policy tr,#ot-sdk-cookie-policy td,#ot-sdk-cookie-policy tbody,#ot-sdk-cookie-policy .ot-main-content,#ot-sdk-cookie-policy .ot-toggle,#ot-sdk-cookie-policy #ot-content,#ot-sdk-cookie-policy #ot-pc-content,#ot-sdk-cookie-policy .checkbox,#ot-sync-ntfy div,#ot-sync-ntfy span,#ot-sync-ntfy h1,#ot-sync-ntfy h2,#ot-sync-ntfy h3,#ot-sync-ntfy h4,#ot-sync-ntfy h5,#ot-sync-ntfy h6,#ot-sync-ntfy p,#ot-sync-ntfy img,#ot-sync-ntfy svg,#ot-sync-ntfy button,#ot-sync-ntfy section,#ot-sync-ntfy a,#ot-sync-ntfy label,#ot-sync-ntfy input,#ot-sync-ntfy ul,#ot-sync-ntfy li,#ot-sync-ntfy nav,#ot-sync-ntfy table,#ot-sync-ntfy thead,#ot-sync-ntfy tr,#ot-sync-ntfy td,#ot-sync-ntfy tbody,#ot-sync-ntfy .ot-main-content,#ot-sync-ntfy .ot-toggle,#ot-sync-ntfy #ot-content,#ot-sync-ntfy #ot-pc-content,#ot-sync-ntfy .checkbox{font-family:inherit;font-weight:normal;-webkit-font-smoothing:auto;letter-spacing:normal;line-height:normal;padding:0;margin:0;height:auto;min-height:0;max-height:none;width:auto;min-width:0;max-width:none;border-radius:0;border:none;clear:none;float:none;position:static;bottom:auto;left:auto;right:auto;top:auto;text-align:left;text-decoration:none;text-indent:0;text-shadow:none;text-transform:none;white-space:normal;background:none;overflow:visible;vertical-align:baseline;visibility:visible;z-index:auto;box-shadow:none}#onetrust-banner-sdk label:before,#onetrust-banner-sdk label:after,#onetrust-banner-sdk .checkbox:after,#onetrust-banner-sdk .checkbox:before,#onetrust-pc-sdk label:before,#onetrust-pc-sdk label:after,#onetrust-pc-sdk .checkbox:after,#onetrust-pc-sdk .checkbox:before,#ot-sdk-cookie-policy label:before,#ot-sdk-cookie-policy label:after,#ot-sdk-cookie-policy .checkbox:after,#ot-sdk-cookie-policy .checkbox:before,#ot-sync-ntfy label:before,#ot-sync-ntfy label:after,#ot-sync-ntfy .checkbox:after,#ot-sync-ntfy .checkbox:before{content:"";content:none}#onetrust-banner-sdk .ot-sdk-container,#onetrust-pc-sdk .ot-sdk-container,#ot-sdk-cookie-policy .ot-sdk-container{position:relative;width:100%;max-width:100%;margin:0 auto;padding:0 20px;box-sizing:border-box}#onetrust-banner-sdk .ot-sdk-column,#onetrust-banner-sdk .ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-column,#onetrust-pc-sdk .ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-column,#ot-sdk-cookie-policy .ot-sdk-columns{width:100%;float:left;box-sizing:border-box;padding:0;display:initial}@media(min-width: 400px){#onetrust-banner-sdk .ot-sdk-container,#onetrust-pc-sdk .ot-sdk-container,#ot-sdk-cookie-policy .ot-sdk-container{width:90%;padding:0}}@media(min-width: 550px){#onetrust-banner-sdk .ot-sdk-container,#onetrust-pc-sdk .ot-sdk-container,#ot-sdk-cookie-policy .ot-sdk-container{width:100%}#onetrust-banner-sdk .ot-sdk-column,#onetrust-banner-sdk .ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-column,#onetrust-pc-sdk .ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-column,#ot-sdk-cookie-policy .ot-sdk-columns{margin-left:4%}#onetrust-banner-sdk .ot-sdk-column:first-child,#onetrust-banner-sdk .ot-sdk-columns:first-child,#onetrust-pc-sdk .ot-sdk-column:first-child,#onetrust-pc-sdk .ot-sdk-columns:first-child,#ot-sdk-cookie-policy .ot-sdk-column:first-child,#ot-sdk-cookie-policy .ot-sdk-columns:first-child{margin-left:0}#onetrust-banner-sdk .ot-sdk-two.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-two.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-two.ot-sdk-columns{width:13.3333333333%}#onetrust-banner-sdk .ot-sdk-three.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-three.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-three.ot-sdk-columns{width:22%}#onetrust-banner-sdk .ot-sdk-four.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-four.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-four.ot-sdk-columns{width:30.6666666667%}#onetrust-banner-sdk .ot-sdk-eight.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-eight.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-eight.ot-sdk-columns{width:65.3333333333%}#onetrust-banner-sdk .ot-sdk-nine.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-nine.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-nine.ot-sdk-columns{width:74%}#onetrust-banner-sdk .ot-sdk-ten.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-ten.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-ten.ot-sdk-columns{width:82.6666666667%}#onetrust-banner-sdk .ot-sdk-eleven.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-eleven.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-eleven.ot-sdk-columns{width:91.3333333333%}#onetrust-banner-sdk .ot-sdk-twelve.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-twelve.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-twelve.ot-sdk-columns{width:100%;margin-left:0}}#onetrust-banner-sdk h1,#onetrust-banner-sdk h2,#onetrust-banner-sdk h3,#onetrust-banner-sdk h4,#onetrust-banner-sdk h5,#onetrust-banner-sdk h6,#onetrust-pc-sdk h1,#onetrust-pc-sdk h2,#onetrust-pc-sdk h3,#onetrust-pc-sdk h4,#onetrust-pc-sdk h5,#onetrust-pc-sdk h6,#ot-sdk-cookie-policy h1,#ot-sdk-cookie-policy h2,#ot-sdk-cookie-policy h3,#ot-sdk-cookie-policy h4,#ot-sdk-cookie-policy h5,#ot-sdk-cookie-policy h6{margin-top:0;font-weight:600;font-family:inherit}#onetrust-banner-sdk h1,#onetrust-pc-sdk h1,#ot-sdk-cookie-policy h1{font-size:1.5rem;line-height:1.2}#onetrust-banner-sdk h2,#onetrust-pc-sdk h2,#ot-sdk-cookie-policy h2{font-size:1.5rem;line-height:1.25}#onetrust-banner-sdk h3,#onetrust-pc-sdk h3,#ot-sdk-cookie-policy h3{font-size:1.5rem;line-height:1.3}#onetrust-banner-sdk h4,#onetrust-pc-sdk h4,#ot-sdk-cookie-policy h4{font-size:1.5rem;line-height:1.35}#onetrust-banner-sdk h5,#onetrust-pc-sdk h5,#ot-sdk-cookie-policy h5{font-size:1.5rem;line-height:1.5}#onetrust-banner-sdk h6,#onetrust-pc-sdk h6,#ot-sdk-cookie-policy h6{font-size:1.5rem;line-height:1.6}@media(min-width: 550px){#onetrust-banner-sdk h1,#onetrust-pc-sdk h1,#ot-sdk-cookie-policy h1{font-size:1.5rem}#onetrust-banner-sdk h2,#onetrust-pc-sdk h2,#ot-sdk-cookie-policy h2{font-size:1.5rem}#onetrust-banner-sdk h3,#onetrust-pc-sdk h3,#ot-sdk-cookie-policy h3{font-size:1.5rem}#onetrust-banner-sdk h4,#onetrust-pc-sdk h4,#ot-sdk-cookie-policy h4{font-size:1.5rem}#onetrust-banner-sdk h5,#onetrust-pc-sdk h5,#ot-sdk-cookie-policy h5{font-size:1.5rem}#onetrust-banner-sdk h6,#onetrust-pc-sdk h6,#ot-sdk-cookie-policy h6{font-size:1.5rem}}#onetrust-banner-sdk p,#onetrust-pc-sdk p,#ot-sdk-cookie-policy p{margin:0 0 1em 0;font-family:inherit;line-height:normal}#onetrust-banner-sdk a,#onetrust-pc-sdk a,#ot-sdk-cookie-policy a{color:#565656;text-decoration:underline}#onetrust-banner-sdk a:hover,#onetrust-pc-sdk a:hover,#ot-sdk-cookie-policy a:hover{color:#565656;text-decoration:none}#onetrust-banner-sdk .ot-sdk-button,#onetrust-banner-sdk button,#onetrust-pc-sdk .ot-sdk-button,#onetrust-pc-sdk button,#ot-sdk-cookie-policy .ot-sdk-button,#ot-sdk-cookie-policy button{margin-bottom:1rem;font-family:inherit}#onetrust-banner-sdk .ot-sdk-button,#onetrust-banner-sdk button,#onetrust-pc-sdk .ot-sdk-button,#onetrust-pc-sdk button,#ot-sdk-cookie-policy .ot-sdk-button,#ot-sdk-cookie-policy button{display:inline-block;height:38px;padding:0 30px;color:#555;text-align:center;font-size:.9em;font-weight:400;line-height:38px;letter-spacing:.01em;text-decoration:none;white-space:nowrap;background-color:rgba(0,0,0,0);border-radius:2px;border:1px solid #bbb;cursor:pointer;box-sizing:border-box}#onetrust-banner-sdk .ot-sdk-button:hover,#onetrust-banner-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):hover,#onetrust-banner-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):focus,#onetrust-pc-sdk .ot-sdk-button:hover,#onetrust-pc-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):hover,#onetrust-pc-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):focus,#ot-sdk-cookie-policy .ot-sdk-button:hover,#ot-sdk-cookie-policy :not(.ot-leg-btn-container)>button:not(.ot-link-btn):hover,#ot-sdk-cookie-policy :not(.ot-leg-btn-container)>button:not(.ot-link-btn):focus{color:#333;border-color:#888;opacity:.9}#onetrust-banner-sdk .ot-sdk-button:focus,#onetrust-banner-sdk :not(.ot-leg-btn-container)>button:focus,#onetrust-pc-sdk .ot-sdk-button:focus,#onetrust-pc-sdk :not(.ot-leg-btn-container)>button:focus,#ot-sdk-cookie-policy .ot-sdk-button:focus,#ot-sdk-cookie-policy :not(.ot-leg-btn-container)>button:focus{outline:2px solid #000}#onetrust-banner-sdk .ot-sdk-button.ot-sdk-button-primary,#onetrust-banner-sdk button.ot-sdk-button-primary,#onetrust-banner-sdk input[type=submit].ot-sdk-button-primary,#onetrust-banner-sdk input[type=reset].ot-sdk-button-primary,#onetrust-banner-sdk input[type=button].ot-sdk-button-primary,#onetrust-pc-sdk .ot-sdk-button.ot-sdk-button-primary,#onetrust-pc-sdk button.ot-sdk-button-primary,#onetrust-pc-sdk input[type=submit].ot-sdk-button-primary,#onetrust-pc-sdk input[type=reset].ot-sdk-button-primary,#onetrust-pc-sdk input[type=button].ot-sdk-button-primary,#ot-sdk-cookie-policy .ot-sdk-button.ot-sdk-button-primary,#ot-sdk-cookie-policy button.ot-sdk-button-primary,#ot-sdk-cookie-policy input[type=submit].ot-sdk-button-primary,#ot-sdk-cookie-policy input[type=reset].ot-sdk-button-primary,#ot-sdk-cookie-policy input[type=button].ot-sdk-button-primary{color:#fff;background-color:#33c3f0;border-color:#33c3f0}#onetrust-banner-sdk .ot-sdk-button.ot-sdk-button-primary:hover,#onetrust-banner-sdk button.ot-sdk-button-primary:hover,#onetrust-banner-sdk input[type=submit].ot-sdk-button-primary:hover,#onetrust-banner-sdk input[type=reset].ot-sdk-button-primary:hover,#onetrust-banner-sdk input[type=button].ot-sdk-button-primary:hover,#onetrust-banner-sdk .ot-sdk-button.ot-sdk-button-primary:focus,#onetrust-banner-sdk button.ot-sdk-button-primary:focus,#onetrust-banner-sdk input[type=submit].ot-sdk-button-primary:focus,#onetrust-banner-sdk input[type=reset].ot-sdk-button-primary:focus,#onetrust-banner-sdk input[type=button].ot-sdk-button-primary:focus,#onetrust-pc-sdk .ot-sdk-button.ot-sdk-button-primary:hover,#onetrust-pc-sdk button.ot-sdk-button-primary:hover,#onetrust-pc-sdk input[type=submit].ot-sdk-button-primary:hover,#onetrust-pc-sdk input[type=reset].ot-sdk-button-primary:hover,#onetrust-pc-sdk input[type=button].ot-sdk-button-primary:hover,#onetrust-pc-sdk .ot-sdk-button.ot-sdk-button-primary:focus,#onetrust-pc-sdk button.ot-sdk-button-primary:focus,#onetrust-pc-sdk input[type=submit].ot-sdk-button-primary:focus,#onetrust-pc-sdk input[type=reset].ot-sdk-button-primary:focus,#onetrust-pc-sdk input[type=button].ot-sdk-button-primary:focus,#ot-sdk-cookie-policy .ot-sdk-button.ot-sdk-button-primary:hover,#ot-sdk-cookie-policy button.ot-sdk-button-primary:hover,#ot-sdk-cookie-policy input[type=submit].ot-sdk-button-primary:hover,#ot-sdk-cookie-policy input[type=reset].ot-sdk-button-primary:hover,#ot-sdk-cookie-policy input[type=button].ot-sdk-button-primary:hover,#ot-sdk-cookie-policy .ot-sdk-button.ot-sdk-button-primary:focus,#ot-sdk-cookie-policy button.ot-sdk-button-primary:focus,#ot-sdk-cookie-policy input[type=submit].ot-sdk-button-primary:focus,#ot-sdk-cookie-policy input[type=reset].ot-sdk-button-primary:focus,#ot-sdk-cookie-policy input[type=button].ot-sdk-button-primary:focus{color:#fff;background-color:#1eaedb;border-color:#1eaedb}#onetrust-banner-sdk input[type=text],#onetrust-pc-sdk input[type=text],#ot-sdk-cookie-policy input[type=text]{height:38px;padding:6px 10px;background-color:#fff;border:1px solid #d1d1d1;border-radius:4px;box-shadow:none;box-sizing:border-box}#onetrust-banner-sdk input[type=text],#onetrust-pc-sdk input[type=text],#ot-sdk-cookie-policy input[type=text]{-webkit-appearance:none;-moz-appearance:none;appearance:none}#onetrust-banner-sdk input[type=text]:focus,#onetrust-pc-sdk input[type=text]:focus,#ot-sdk-cookie-policy input[type=text]:focus{border:1px solid #000;outline:0}#onetrust-banner-sdk label,#onetrust-pc-sdk label,#ot-sdk-cookie-policy label{display:block;margin-bottom:.5rem;font-weight:600}#onetrust-banner-sdk input[type=checkbox],#onetrust-pc-sdk input[type=checkbox],#ot-sdk-cookie-policy input[type=checkbox]{display:inline}#onetrust-banner-sdk ul,#onetrust-pc-sdk ul,#ot-sdk-cookie-policy ul{list-style:circle inside}#onetrust-banner-sdk ul,#onetrust-pc-sdk ul,#ot-sdk-cookie-policy ul{padding-left:0;margin-top:0}#onetrust-banner-sdk ul ul,#onetrust-pc-sdk ul ul,#ot-sdk-cookie-policy ul ul{margin:1.5rem 0 1.5rem 3rem;font-size:90%}#onetrust-banner-sdk li,#onetrust-pc-sdk li,#ot-sdk-cookie-policy li{margin-bottom:1rem}#onetrust-banner-sdk th,#onetrust-banner-sdk td,#onetrust-pc-sdk th,#onetrust-pc-sdk td,#ot-sdk-cookie-policy th,#ot-sdk-cookie-policy td{padding:12px 15px;text-align:left;border-bottom:1px solid #e1e1e1}#onetrust-banner-sdk button,#onetrust-pc-sdk button,#ot-sdk-cookie-policy button{margin-bottom:1rem;font-family:inherit}#onetrust-banner-sdk .ot-sdk-container:after,#onetrust-banner-sdk .ot-sdk-row:after,#onetrust-pc-sdk .ot-sdk-container:after,#onetrust-pc-sdk .ot-sdk-row:after,#ot-sdk-cookie-policy .ot-sdk-container:after,#ot-sdk-cookie-policy .ot-sdk-row:after{content:"";display:table;clear:both}#onetrust-banner-sdk .ot-sdk-row,#onetrust-pc-sdk .ot-sdk-row,#ot-sdk-cookie-policy .ot-sdk-row{margin:0;max-width:none;display:block}#onetrust-banner-sdk{box-shadow:0 0 18px rgba(0,0,0,.2)}#onetrust-banner-sdk.otFlat{position:fixed;z-index:2147483645;bottom:0;right:0;left:0;background-color:#fff;max-height:90%;overflow-x:hidden;overflow-y:auto}#onetrust-banner-sdk.otFlat.top{top:0px;bottom:auto}#onetrust-banner-sdk.otRelFont{font-size:1rem}#onetrust-banner-sdk>.ot-sdk-container{overflow:hidden}#onetrust-banner-sdk::-webkit-scrollbar{width:11px}#onetrust-banner-sdk::-webkit-scrollbar-thumb{border-radius:10px;background:#c1c1c1}#onetrust-banner-sdk{scrollbar-arrow-color:#c1c1c1;scrollbar-darkshadow-color:#c1c1c1;scrollbar-face-color:#c1c1c1;scrollbar-shadow-color:#c1c1c1}#onetrust-banner-sdk #onetrust-policy{margin:1.25em 0 .625em 2em;overflow:hidden}#onetrust-banner-sdk #onetrust-policy .ot-gv-list-handler{float:left;font-size:.82em;padding:0;margin-bottom:0;border:0;line-height:normal;height:auto;width:auto}#onetrust-banner-sdk #onetrust-policy-title{font-size:1.2em;line-height:1.3;margin-bottom:10px}#onetrust-banner-sdk #onetrust-policy-text{clear:both;text-align:left;font-size:.88em;line-height:1.4}#onetrust-banner-sdk #onetrust-policy-text *{font-size:inherit;line-height:inherit}#onetrust-banner-sdk #onetrust-policy-text a{font-weight:bold;margin-left:5px}#onetrust-banner-sdk #onetrust-policy-title,#onetrust-banner-sdk #onetrust-policy-text{color:dimgray;float:left}#onetrust-banner-sdk #onetrust-button-group-parent{min-height:1px;text-align:center}#onetrust-banner-sdk #onetrust-button-group{display:inline-block}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{background-color:#68b631;color:#fff;border-color:#68b631;margin-right:1em;min-width:125px;height:auto;white-space:normal;word-break:break-word;word-wrap:break-word;padding:12px 10px;line-height:1.2;font-size:.813em;font-weight:600}#onetrust-banner-sdk #onetrust-pc-btn-handler.cookie-setting-link{background-color:#fff;border:none;color:#68b631;text-decoration:underline;padding-left:0;padding-right:0}#onetrust-banner-sdk .onetrust-close-btn-ui{width:44px;height:44px;background-size:12px;border:none;position:relative;margin:auto;padding:0}#onetrust-banner-sdk .banner_logo{display:none}#onetrust-banner-sdk.ot-bnr-w-logo .ot-bnr-logo{position:absolute;top:50%;transform:translateY(-50%);left:0px}#onetrust-banner-sdk.ot-bnr-w-logo #onetrust-policy{margin-left:65px}#onetrust-banner-sdk .ot-b-addl-desc{clear:both;float:left;display:block}#onetrust-banner-sdk #banner-options{float:left;display:table;margin-right:0;margin-left:1em;width:calc(100% - 1em)}#onetrust-banner-sdk .banner-option-input{cursor:pointer;width:auto;height:auto;border:none;padding:0;padding-right:3px;margin:0 0 10px;font-size:.82em;line-height:1.4}#onetrust-banner-sdk .banner-option-input *{pointer-events:none;font-size:inherit;line-height:inherit}#onetrust-banner-sdk .banner-option-input[aria-expanded=true]~.banner-option-details{display:block;height:auto}#onetrust-banner-sdk .banner-option-input[aria-expanded=true] .ot-arrow-container{transform:rotate(90deg)}#onetrust-banner-sdk .banner-option{margin-bottom:12px;margin-left:0;border:none;float:left;padding:0}#onetrust-banner-sdk .banner-option:first-child{padding-left:2px}#onetrust-banner-sdk .banner-option:not(:first-child){padding:0;border:none}#onetrust-banner-sdk .banner-option-header{cursor:pointer;display:inline-block}#onetrust-banner-sdk .banner-option-header :first-child{color:dimgray;font-weight:bold;float:left}#onetrust-banner-sdk .banner-option-header .ot-arrow-container{display:inline-block;border-top:6px solid rgba(0,0,0,0);border-bottom:6px solid rgba(0,0,0,0);border-left:6px solid dimgray;margin-left:10px;vertical-align:middle}#onetrust-banner-sdk .banner-option-details{display:none;font-size:.83em;line-height:1.5;padding:10px 0px 5px 10px;margin-right:10px;height:0px}#onetrust-banner-sdk .banner-option-details *{font-size:inherit;line-height:inherit;color:dimgray}#onetrust-banner-sdk .ot-arrow-container,#onetrust-banner-sdk .banner-option-details{transition:all 300ms ease-in 0s;-webkit-transition:all 300ms ease-in 0s;-moz-transition:all 300ms ease-in 0s;-o-transition:all 300ms ease-in 0s}#onetrust-banner-sdk .ot-dpd-container{float:left}#onetrust-banner-sdk .ot-dpd-title{margin-bottom:10px}#onetrust-banner-sdk .ot-dpd-title,#onetrust-banner-sdk .ot-dpd-desc{font-size:.88em;line-height:1.4;color:dimgray}#onetrust-banner-sdk .ot-dpd-title *,#onetrust-banner-sdk .ot-dpd-desc *{font-size:inherit;line-height:inherit}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-text *{margin-bottom:0}#onetrust-banner-sdk.ot-iab-2 .onetrust-vendors-list-handler{display:block;margin-left:0;margin-top:5px;clear:both;margin-bottom:0;padding:0;border:0;height:auto;width:auto}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group button{display:block}#onetrust-banner-sdk.ot-close-btn-link{padding-top:25px}#onetrust-banner-sdk.ot-close-btn-link #onetrust-close-btn-container{top:15px;transform:none;right:15px}#onetrust-banner-sdk.ot-close-btn-link #onetrust-close-btn-container button{padding:0;white-space:pre-wrap;border:none;height:auto;line-height:1.5;text-decoration:underline;font-size:.69em}#onetrust-banner-sdk #onetrust-policy-text,#onetrust-banner-sdk .ot-dpd-desc,#onetrust-banner-sdk .ot-b-addl-desc{font-size:.813em;line-height:1.5}#onetrust-banner-sdk .ot-dpd-desc{margin-bottom:10px}#onetrust-banner-sdk .ot-dpd-desc>.ot-b-addl-desc{margin-top:10px;margin-bottom:10px;font-size:1em}@media only screen and (max-width: 425px){#onetrust-banner-sdk #onetrust-close-btn-container{position:absolute;top:6px;right:2px}#onetrust-banner-sdk #onetrust-policy{margin-left:0;margin-top:3em}#onetrust-banner-sdk #onetrust-button-group{display:block}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{width:100%}#onetrust-banner-sdk .onetrust-close-btn-ui{top:auto;transform:none}#onetrust-banner-sdk #onetrust-policy-title{display:inline;float:none}#onetrust-banner-sdk #banner-options{margin:0;padding:0;width:100%}}@media only screen and (min-width: 426px)and (max-width: 896px){#onetrust-banner-sdk #onetrust-close-btn-container{position:absolute;top:0;right:0}#onetrust-banner-sdk #onetrust-policy{margin-left:1em;margin-right:1em}#onetrust-banner-sdk .onetrust-close-btn-ui{top:10px;right:10px}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{width:95%}#onetrust-banner-sdk.ot-iab-2 #onetrust-group-container{width:100%}#onetrust-banner-sdk.ot-bnr-w-logo #onetrust-button-group-parent{padding-left:50px}#onetrust-banner-sdk #onetrust-button-group-parent{width:100%;position:relative;margin-left:0}#onetrust-banner-sdk #onetrust-button-group button{display:inline-block}#onetrust-banner-sdk #onetrust-button-group{margin-right:0;text-align:center}#onetrust-banner-sdk .has-reject-all-button #onetrust-pc-btn-handler{float:left}#onetrust-banner-sdk .has-reject-all-button #onetrust-reject-all-handler,#onetrust-banner-sdk .has-reject-all-button #onetrust-accept-btn-handler{float:right}#onetrust-banner-sdk .has-reject-all-button #onetrust-button-group{width:calc(100% - 2em);margin-right:0}#onetrust-banner-sdk .has-reject-all-button #onetrust-pc-btn-handler.cookie-setting-link{padding-left:0px;text-align:left}#onetrust-banner-sdk.ot-buttons-fw .ot-sdk-three button{width:100%;text-align:center}#onetrust-banner-sdk.ot-buttons-fw #onetrust-button-group-parent button{float:none}#onetrust-banner-sdk.ot-buttons-fw #onetrust-pc-btn-handler.cookie-setting-link{text-align:center}}@media only screen and (min-width: 550px){#onetrust-banner-sdk .banner-option:not(:first-child){border-left:1px solid #d8d8d8;padding-left:25px}}@media only screen and (min-width: 425px)and (max-width: 550px){#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group,#onetrust-banner-sdk.ot-iab-2 #onetrust-policy,#onetrust-banner-sdk.ot-iab-2 .banner-option{width:100%}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-accept-btn-handler,#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-reject-all-handler,#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-pc-btn-handler{width:100%}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-accept-btn-handler,#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-reject-all-handler{float:left}}@media only screen and (min-width: 769px){#onetrust-banner-sdk #onetrust-button-group{margin-right:30%}#onetrust-banner-sdk #banner-options{margin-left:2em;margin-right:5em;margin-bottom:1.25em;width:calc(100% - 7em)}}@media only screen and (min-width: 897px)and (max-width: 1023px){#onetrust-banner-sdk.vertical-align-content #onetrust-button-group-parent{position:absolute;top:50%;left:75%;transform:translateY(-50%)}#onetrust-banner-sdk #onetrust-close-btn-container{top:50%;margin:auto;transform:translate(-50%, -50%);position:absolute;padding:0;right:0}#onetrust-banner-sdk #onetrust-close-btn-container button{position:relative;margin:0;right:-22px;top:2px}}@media only screen and (min-width: 1024px){#onetrust-banner-sdk #onetrust-close-btn-container{top:50%;margin:auto;transform:translate(-50%, -50%);position:absolute;right:0}#onetrust-banner-sdk #onetrust-close-btn-container button{right:-12px}#onetrust-banner-sdk #onetrust-policy{margin-left:2em}#onetrust-banner-sdk.vertical-align-content #onetrust-button-group-parent{position:absolute;top:50%;left:60%;transform:translateY(-50%)}#onetrust-banner-sdk .ot-optout-signal{width:50%}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-title{width:50%}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-text,#onetrust-banner-sdk.ot-iab-2 :not(.ot-dpd-desc)>.ot-b-addl-desc{margin-bottom:1em;width:50%;border-right:1px solid #d8d8d8;padding-right:1rem}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-text{margin-bottom:0;padding-bottom:1em}#onetrust-banner-sdk.ot-iab-2 :not(.ot-dpd-desc)>.ot-b-addl-desc{margin-bottom:0;padding-bottom:1em}#onetrust-banner-sdk.ot-iab-2 .ot-dpd-container{width:45%;padding-left:1rem;display:inline-block;float:none}#onetrust-banner-sdk.ot-iab-2 .ot-dpd-title{line-height:1.7}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group-parent{left:auto;right:4%;margin-left:0}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group button{display:block}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{margin:auto;width:30%}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{width:60%}#onetrust-banner-sdk #onetrust-button-group{margin-right:auto}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{margin-top:1em}}@media only screen and (min-width: 890px){#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group-parent{padding-left:3%;padding-right:4%;margin-left:0}#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group{margin-right:0;margin-top:1.25em;width:100%}#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button{width:100%;margin-bottom:5px;margin-top:5px}#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button:last-of-type{margin-bottom:20px}}@media only screen and (min-width: 1280px){#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{width:55%}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{width:44%;padding-left:2%;padding-right:2%}#onetrust-banner-sdk:not(.ot-iab-2).vertical-align-content #onetrust-button-group-parent{position:absolute;left:55%}}
        #onetrust-consent-sdk #onetrust-banner-sdk {background-color: #FFF;}
            #onetrust-consent-sdk #onetrust-policy-title,
                    #onetrust-consent-sdk #onetrust-policy-text,
                    #onetrust-consent-sdk .ot-b-addl-desc,
                    #onetrust-consent-sdk .ot-dpd-desc,
                    #onetrust-consent-sdk .ot-dpd-title,
                    #onetrust-consent-sdk #onetrust-policy-text *:not(.onetrust-vendors-list-handler),
                    #onetrust-consent-sdk .ot-dpd-desc *:not(.onetrust-vendors-list-handler),
                    #onetrust-consent-sdk #onetrust-banner-sdk #banner-options *,
                    #onetrust-banner-sdk .ot-cat-header,
                    #onetrust-banner-sdk .ot-optout-signal
                    {
                        color: #2E2E2E;
                    }
            #onetrust-consent-sdk #onetrust-banner-sdk .banner-option-details {
                    background-color: #E9E9E9;}
             #onetrust-consent-sdk #onetrust-banner-sdk a[href],
                    #onetrust-consent-sdk #onetrust-banner-sdk a[href] font,
                    #onetrust-consent-sdk #onetrust-banner-sdk .ot-link-btn
                        {
                            color: #007398;
                        }#onetrust-consent-sdk #onetrust-accept-btn-handler,
                         #onetrust-banner-sdk #onetrust-reject-all-handler {
                            background-color: #007398;border-color: #007398;
                color: #FFF;
            }
            #onetrust-consent-sdk #onetrust-banner-sdk *:focus,
            #onetrust-consent-sdk #onetrust-banner-sdk:focus {
               outline-color: #000000;
               outline-width: 1px;
            }
            #onetrust-consent-sdk #onetrust-pc-btn-handler,
            #onetrust-consent-sdk #onetrust-pc-btn-handler.cookie-setting-link {
                color: #6CC04A; border-color: #6CC04A;
                background-color:
                #FFF;
            }/*! v6.10.0 2021-01-18 */
div#onetrust-consent-sdk #onetrust-banner-sdk{border-top:2px solid #d0d3d4!important;box-shadow:none;padding:24px}div#onetrust-consent-sdk button{border-radius:0!important;box-shadow:none!important;box-sizing:border-box!important;font-size:20px!important;font-weight:400!important;letter-spacing:0!important;max-width:none!important;white-space:nowrap!important}div#onetrust-consent-sdk button:not(.ot-link-btn){background-color:#007dbc!important;border:2px solid #007dbc!important;color:#fff!important;height:48px!important;padding:0 1em!important;width:auto!important}div#onetrust-consent-sdk button:hover{background-color:#fff!important;border-color:#949494!important;color:#2e2e2e!important}div#onetrust-consent-sdk button.ot-link-btn{color:#007dbc!important;font-size:16px!important;text-decoration:underline}div#onetrust-consent-sdk button.ot-link-btn:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-consent-sdk a,div#onetrust-pc-sdk a{color:#007dbc!important;text-decoration:underline!important}div#onetrust-consent-sdk a,div#onetrust-consent-sdk button,div#onetrust-consent-sdk p:hover{opacity:1!important}div#onetrust-consent-sdk a:focus,div#onetrust-consent-sdk button:focus,div#onetrust-consent-sdk input:focus{outline:2px solid #949494!important;outline-offset:1px!important}div#onetrust-banner-sdk .ot-sdk-container{padding:0;width:auto}div#onetrust-banner-sdk .ot-sdk-row{align-items:flex-start;box-sizing:border-box;display:flex;flex-direction:column;justify-content:space-between;margin:auto;max-width:1152px}div#onetrust-banner-sdk .ot-sdk-row:after{display:none}div#onetrust-banner-sdk #onetrust-group-container,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-group-container,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{flex-grow:1;width:auto}div#onetrust-banner-sdk #onetrust-policy,div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy{margin:0;overflow:visible}div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy-text,div#onetrust-consent-sdk #onetrust-policy-text{font-size:16px;line-height:24px;max-width:44em;margin:0}div#onetrust-consent-sdk #onetrust-policy-text a[href]{font-weight:400;margin-left:8px}div#onetrust-banner-sdk #onetrust-button-group-parent{flex:0 0 auto;margin:32px 0 0;width:100%}div#onetrust-banner-sdk #onetrust-button-group{display:flex;flex-direction:row;flex-wrap:wrap;justify-content:flex-end;margin:-8px}div#onetrust-banner-sdk .banner-actions-container{display:flex;flex:1 0 auto}div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button:last-of-type,div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto;margin:8px;width:auto}div#onetrust-consent-sdk #onetrust-pc-btn-handler{background-color:#fff!important;color:inherit!important}div#onetrust-banner-sdk #onetrust-close-btn-container{display:none}@media only screen and (min-width:556px){div#onetrust-consent-sdk #onetrust-banner-sdk{padding:40px}div#onetrust-banner-sdk #onetrust-policy{margin:0 40px 0 0}div#onetrust-banner-sdk .ot-sdk-row{align-items:center;flex-direction:row}div#onetrust-banner-sdk #onetrust-button-group-parent,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-button-group-parent,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{margin:0;padding:0;width:auto}div#onetrust-banner-sdk #onetrust-button-group,div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group{align-items:stretch;flex-direction:column-reverse;margin:0}div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto}}@media only screen and (min-width:768px){div#onetrust-banner-sdk #onetrust-policy{margin:0 48px 0 0}div#onetrust-consent-sdk #onetrust-banner-sdk{padding:48px}}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-size:16px;line-height:24px}div#onetrust-consent-sdk #onetrust-pc-sdk p,div#onetrust-pc-sdk #ot-pc-desc,div#onetrust-pc-sdk .category-host-list-handler,div#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{font-size:16px;font-weight:400;line-height:24px}div#onetrust-consent-sdk a:hover,div#onetrust-pc-sdk a:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-pc-sdk{border-radius:0;bottom:0;height:auto;left:0;margin:auto;max-width:100%;overflow:hidden;right:0;top:0;width:512px;max-height:800px}div#onetrust-pc-sdk .ot-pc-header{display:none}div#onetrust-pc-sdk #ot-pc-content{overscroll-behavior:contain;padding:0 12px 0 24px;margin:16px 4px 0 0;top:0;right:16px;left:0;width:auto}div#onetrust-pc-sdk #ot-category-title,div#onetrust-pc-sdk #ot-pc-title{font-size:24px;font-weight:400;line-height:32px;margin:0 0 16px}div#onetrust-pc-sdk #ot-pc-desc{padding:0}div#onetrust-pc-sdk #ot-pc-desc a{display:inline}div#onetrust-pc-sdk #accept-recommended-btn-handler{display:none!important}div#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:2px solid #007dbc;outline-offset:-1px;transition:none}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item{border-width:0 0 2px}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item:first-of-type{border-width:2px 0}div#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{padding:8px 0;width:100%}div#onetrust-pc-sdk .ot-plus-minus{transform:translateY(2px)}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item>button{background:0 0!important;border:0!important;height:44px!important;max-width:none!important;width:calc(100% - 48px)!important}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-weight:700}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr{padding:0}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item .ot-acc-grpdesc{padding:0;width:100%}div#onetrust-pc-sdk .ot-acc-grpcntr .ot-subgrp-cntr{border:0;padding:0}div#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp{margin:0}div#onetrust-pc-sdk .ot-always-active-group .ot-cat-header{width:calc(100% - 160px)}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{width:calc(100% - 88px)}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active{color:inherit;font-size:12px;font-weight:400;line-height:1.5;padding-right:48px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:before{border-radius:12px;position:absolute;right:0;top:0;content:'';background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:after{border-radius:50%;position:absolute;right:5px;top:4px;content:'';background-color:#007dbc;height:12px;width:12px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active,div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl{right:2px}div#onetrust-pc-sdk .ot-switch{display:block;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob,div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob{background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob:before{background-color:#737373;height:8px;left:4px;top:4px;width:8px}div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob:before{background-color:#007dbc;height:12px;left:0;top:2px;width:12px}div#onetrust-pc-sdk .ot-tgl input:focus+.ot-switch .ot-switch-nob{box-shadow:0 0;outline:2px solid #007dbc!important;outline-offset:1px;transition:none}div#onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-grpcntr.ot-acc-txt{background-color:transparent;padding-left:3px}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr,div#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr{overflow:visible;width:100%}div#onetrust-pc-sdk .ot-pc-footer{border-top:0 solid}div#onetrust-pc-sdk .ot-btn-container{padding-top:24px;text-align:center}div#onetrust-pc-sdk .ot-pc-footer button{margin:8px 0;background-color:#fff}div#onetrust-pc-sdk .ot-pc-footer-logo{background-color:#fff}div#onetrust-pc-sdk #ot-lst-title span{font-size:24px;font-weight:400;line-height:32px}div#onetrust-pc-sdk #ot-host-lst .ot-host-desc,div#onetrust-pc-sdk #ot-host-lst .ot-host-expand,div#onetrust-pc-sdk #ot-host-lst .ot-host-name,div#onetrust-pc-sdk #ot-host-lst .ot-host-name a,div#onetrust-pc-sdk .back-btn-handler,div#onetrust-pc-sdk .ot-host-opt li>div div{font-size:16px;font-weight:400;line-height:24px}div#onetrust-pc-sdk #ot-pc-lst{top:0}div#onetrust-pc-sdk .back-btn-handler{text-decoration:none!important}div#onetrust-pc-sdk #filter-btn-handler:hover svg{filter:invert(1)}div#onetrust-pc-sdk .back-btn-handler svg{width:16px;height:16px}div#onetrust-pc-sdk .ot-host-item>button{background:0 0!important;border:0!important;height:66px!important;max-width:none!important;width:calc(100% - 5px)!important;transform:translate(2px,2px)}div#onetrust-pc-sdk .ot-host-item{border-bottom:2px solid #b9b9b9;padding:0}div#onetrust-pc-sdk .ot-host-item .ot-acc-hdr{margin:0 0 -6px;padding:8px 0}div#onetrust-pc-sdk ul li:first-child{border-top:2px solid #b9b9b9}div#onetrust-pc-sdk .ot-host-item .ot-plus-minus{margin:0 8px 0 0}div#onetrust-pc-sdk .ot-search-cntr{width:calc(100% - 48px)}div#onetrust-pc-sdk .ot-host-opt .ot-host-info{background-color:transparent}div#onetrust-pc-sdk .ot-host-opt li>div div{padding:0}div#onetrust-pc-sdk #vendor-search-handler{border-radius:0;border-color:#939393;border-style:solid;border-width:2px 0 2px 2px;font-size:20px;height:48px;margin:0}div#onetrust-pc-sdk .ot-lst-subhdr svg{right:0;top:8px}div#onetrust-pc-sdk .ot-fltr-cntr{box-sizing:border-box;right:0;width:48px}div#onetrust-pc-sdk #filter-btn-handler{width:48px!important;padding:8px!important}div#onetrust-pc-sdk #ot-fltr-modal{max-height:400px}
#onetrust-pc-sdk.otPcCenter{overflow:hidden;position:fixed;margin:0 auto;top:5%;right:0;left:0;width:40%;max-width:575px;min-width:575px;border-radius:2.5px;z-index:2147483647;background-color:#fff;-webkit-box-shadow:0px 2px 10px -3px #999;-moz-box-shadow:0px 2px 10px -3px #999;box-shadow:0px 2px 10px -3px #999}#onetrust-pc-sdk.otPcCenter[dir=rtl]{right:0;left:0}#onetrust-pc-sdk.otRelFont{font-size:1rem}#onetrust-pc-sdk .ot-optout-signal{margin-top:.625rem}#onetrust-pc-sdk #ot-addtl-venlst .ot-arw-cntr,#onetrust-pc-sdk #ot-addtl-venlst .ot-plus-minus,#onetrust-pc-sdk .ot-hide-tgl{visibility:hidden}#onetrust-pc-sdk #ot-addtl-venlst .ot-arw-cntr *,#onetrust-pc-sdk #ot-addtl-venlst .ot-plus-minus *,#onetrust-pc-sdk .ot-hide-tgl *{visibility:hidden}#onetrust-pc-sdk #ot-gn-venlst .ot-ven-item .ot-acc-hdr{min-height:40px}#onetrust-pc-sdk .ot-pc-header{height:39px;padding:10px 0 10px 30px;border-bottom:1px solid #e9e9e9}#onetrust-pc-sdk #ot-pc-title,#onetrust-pc-sdk #ot-category-title,#onetrust-pc-sdk .ot-cat-header,#onetrust-pc-sdk #ot-lst-title,#onetrust-pc-sdk .ot-ven-hdr .ot-ven-name,#onetrust-pc-sdk .ot-always-active{font-weight:bold;color:dimgray}#onetrust-pc-sdk .ot-always-active-group .ot-cat-header{width:55%;font-weight:700}#onetrust-pc-sdk .ot-cat-item p{clear:both;float:left;margin-top:10px;margin-bottom:5px;line-height:1.5;font-size:.812em;color:dimgray}#onetrust-pc-sdk .ot-close-icon{height:44px;width:44px;background-size:10px}#onetrust-pc-sdk #ot-pc-title{float:left;font-size:1em;line-height:1.5;margin-bottom:10px;margin-top:10px;width:100%}#onetrust-pc-sdk #accept-recommended-btn-handler{margin-right:10px;margin-bottom:25px;outline-offset:-1px}#onetrust-pc-sdk #ot-pc-desc{clear:both;width:100%;font-size:.812em;line-height:1.5;margin-bottom:25px}#onetrust-pc-sdk #ot-pc-desc *{font-size:inherit;line-height:inherit}#onetrust-pc-sdk #ot-pc-desc ul li{padding:10px 0px}#onetrust-pc-sdk a{color:#656565;cursor:pointer}#onetrust-pc-sdk a:hover{color:#3860be}#onetrust-pc-sdk label{margin-bottom:0}#onetrust-pc-sdk #vdr-lst-dsc{font-size:.812em;line-height:1.5;padding:10px 15px 5px 15px}#onetrust-pc-sdk button{max-width:394px;padding:12px 30px;line-height:1;word-break:break-word;word-wrap:break-word;white-space:normal;font-weight:bold;height:auto}#onetrust-pc-sdk .ot-link-btn{padding:0;margin-bottom:0;border:0;font-weight:normal;line-height:normal;width:auto;height:auto}#onetrust-pc-sdk #ot-pc-content{position:absolute;overflow-y:scroll;padding-left:2px;padding-right:30px;top:60px;bottom:110px;margin:1px 3px 0 30px;width:calc(100% - 63px)}#onetrust-pc-sdk .ot-vs-list .ot-always-active,#onetrust-pc-sdk .ot-cat-grp .ot-always-active{float:right;clear:none;color:#3860be;margin:0;font-size:.813em;line-height:1.3}#onetrust-pc-sdk .ot-pc-scrollbar::-webkit-scrollbar-track{margin-right:20px}#onetrust-pc-sdk .ot-pc-scrollbar::-webkit-scrollbar{width:11px}#onetrust-pc-sdk .ot-pc-scrollbar::-webkit-scrollbar-thumb{border-radius:10px;background:#d8d8d8}#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:#000 1px solid}#onetrust-pc-sdk .ot-pc-scrollbar{scrollbar-arrow-color:#d8d8d8;scrollbar-darkshadow-color:#d8d8d8;scrollbar-face-color:#d8d8d8;scrollbar-shadow-color:#d8d8d8}#onetrust-pc-sdk .save-preference-btn-handler{margin-right:20px}#onetrust-pc-sdk .ot-pc-refuse-all-handler{margin-right:10px}#onetrust-pc-sdk #ot-pc-desc .privacy-notice-link{margin-left:0;margin-right:8px}#onetrust-pc-sdk #ot-pc-desc .ot-imprint-handler{margin-left:0;margin-right:8px}#onetrust-pc-sdk .ot-subgrp-cntr{display:inline-block;clear:both;width:100%;padding-top:15px}#onetrust-pc-sdk .ot-switch+.ot-subgrp-cntr{padding-top:10px}#onetrust-pc-sdk ul.ot-subgrps{margin:0;font-size:initial}#onetrust-pc-sdk ul.ot-subgrps li p,#onetrust-pc-sdk ul.ot-subgrps li h5{font-size:.813em;line-height:1.4;color:dimgray}#onetrust-pc-sdk ul.ot-subgrps .ot-switch{min-height:auto}#onetrust-pc-sdk ul.ot-subgrps .ot-switch-nob{top:0}#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr{display:inline-block;width:100%}#onetrust-pc-sdk ul.ot-subgrps .ot-acc-txt{margin:0}#onetrust-pc-sdk ul.ot-subgrps li{padding:0;border:none}#onetrust-pc-sdk ul.ot-subgrps li h5{position:relative;top:5px;font-weight:bold;margin-bottom:0;float:left}#onetrust-pc-sdk li.ot-subgrp{margin-left:20px;overflow:auto}#onetrust-pc-sdk li.ot-subgrp>h5{width:calc(100% - 100px)}#onetrust-pc-sdk .ot-cat-item p>ul,#onetrust-pc-sdk li.ot-subgrp p>ul{margin:0px;list-style:disc;margin-left:15px;font-size:inherit}#onetrust-pc-sdk .ot-cat-item p>ul li,#onetrust-pc-sdk li.ot-subgrp p>ul li{font-size:inherit;padding-top:10px;padding-left:0px;padding-right:0px;border:none}#onetrust-pc-sdk .ot-cat-item p>ul li:last-child,#onetrust-pc-sdk li.ot-subgrp p>ul li:last-child{padding-bottom:10px}#onetrust-pc-sdk .ot-pc-logo{height:40px;width:120px}#onetrust-pc-sdk .ot-pc-footer{position:absolute;bottom:0px;width:100%;max-height:160px;border-top:1px solid #d8d8d8}#onetrust-pc-sdk.ot-ftr-stacked .ot-pc-refuse-all-handler{margin-bottom:0px}#onetrust-pc-sdk.ot-ftr-stacked #ot-pc-content{bottom:160px}#onetrust-pc-sdk.ot-ftr-stacked .ot-pc-footer button{width:100%;max-width:none}#onetrust-pc-sdk.ot-ftr-stacked .ot-btn-container{margin:0 30px;width:calc(100% - 60px);padding-right:0}#onetrust-pc-sdk .ot-pc-footer-logo{height:30px;width:100%;text-align:right;background:#f4f4f4}#onetrust-pc-sdk .ot-pc-footer-logo a{display:inline-block;margin-top:5px;margin-right:10px}#onetrust-pc-sdk[dir=rtl] .ot-pc-footer-logo{direction:rtl}#onetrust-pc-sdk[dir=rtl] .ot-pc-footer-logo a{margin-right:25px}#onetrust-pc-sdk .ot-tgl{float:right;position:relative;z-index:1}#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob{background-color:#468254;border:1px solid #fff}#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob:before{-webkit-transform:translateX(20px);-ms-transform:translateX(20px);transform:translateX(20px);background-color:#fff;border-color:#fff}#onetrust-pc-sdk .ot-tgl input:focus+.ot-switch{outline:#000 solid 1px}#onetrust-pc-sdk .ot-switch{position:relative;display:inline-block;width:45px;height:25px}#onetrust-pc-sdk .ot-switch-nob{position:absolute;cursor:pointer;top:0;left:0;right:0;bottom:0;background-color:#767676;border:1px solid #ddd;transition:all .2s ease-in 0s;-moz-transition:all .2s ease-in 0s;-o-transition:all .2s ease-in 0s;-webkit-transition:all .2s ease-in 0s;border-radius:20px}#onetrust-pc-sdk .ot-switch-nob:before{position:absolute;content:"";height:18px;width:18px;bottom:3px;left:3px;background-color:#fff;-webkit-transition:.4s;transition:.4s;border-radius:20px}#onetrust-pc-sdk .ot-chkbox input:checked~label::before{background-color:#3860be}#onetrust-pc-sdk .ot-chkbox input+label::after{content:none;color:#fff}#onetrust-pc-sdk .ot-chkbox input:checked+label::after{content:""}#onetrust-pc-sdk .ot-chkbox input:focus+label::before{outline-style:solid;outline-width:2px;outline-style:auto}#onetrust-pc-sdk .ot-chkbox input[aria-checked=mixed]~label::before{background-color:#3860be}#onetrust-pc-sdk .ot-chkbox input[aria-checked=mixed]+label::after{content:""}#onetrust-pc-sdk .ot-chkbox label{position:relative;display:inline-block;padding-left:30px;cursor:pointer;font-weight:500}#onetrust-pc-sdk .ot-chkbox label::before,#onetrust-pc-sdk .ot-chkbox label::after{position:absolute;content:"";display:inline-block;border-radius:3px}#onetrust-pc-sdk .ot-chkbox label::before{height:18px;width:18px;border:1px solid #3860be;left:0px;top:auto}#onetrust-pc-sdk .ot-chkbox label::after{height:5px;width:9px;border-left:3px solid;border-bottom:3px solid;transform:rotate(-45deg);-o-transform:rotate(-45deg);-ms-transform:rotate(-45deg);-webkit-transform:rotate(-45deg);left:4px;top:5px}#onetrust-pc-sdk .ot-label-txt{display:none}#onetrust-pc-sdk .ot-chkbox input,#onetrust-pc-sdk .ot-tgl input{position:absolute;opacity:0;width:0;height:0}#onetrust-pc-sdk .ot-arw-cntr{float:right;position:relative;pointer-events:none}#onetrust-pc-sdk .ot-arw-cntr .ot-arw{width:16px;height:16px;margin-left:5px;color:dimgray;display:inline-block;vertical-align:middle;-webkit-transition:all 150ms ease-in 0s;-moz-transition:all 150ms ease-in 0s;-o-transition:all 150ms ease-in 0s;transition:all 150ms ease-in 0s}#onetrust-pc-sdk input:checked~.ot-acc-hdr .ot-arw,#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-arw-cntr svg{transform:rotate(90deg);-o-transform:rotate(90deg);-ms-transform:rotate(90deg);-webkit-transform:rotate(90deg)}#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:#000 1px solid}#onetrust-pc-sdk .ot-tgl-cntr,#onetrust-pc-sdk .ot-arw-cntr{display:inline-block}#onetrust-pc-sdk .ot-tgl-cntr{width:45px;float:right;margin-top:2px}#onetrust-pc-sdk #ot-lst-cnt .ot-tgl-cntr{margin-top:10px}#onetrust-pc-sdk .ot-always-active-subgroup{width:auto;padding-left:0px !important;top:3px;position:relative}#onetrust-pc-sdk .ot-label-status{font-size:.75em;display:none;font-size:.75em;position:relative;top:2px;padding-right:5px;float:left}#onetrust-pc-sdk .ot-arw-cntr{margin-top:-1px}#onetrust-pc-sdk .ot-arw-cntr svg{-webkit-transition:all 300ms ease-in 0s;-moz-transition:all 300ms ease-in 0s;-o-transition:all 300ms ease-in 0s;transition:all 300ms ease-in 0s;height:10px;width:10px}#onetrust-pc-sdk input:checked~.ot-acc-hdr .ot-arw{transform:rotate(90deg);-o-transform:rotate(90deg);-ms-transform:rotate(90deg);-webkit-transform:rotate(90deg)}#onetrust-pc-sdk .ot-arw{width:10px;margin-left:15px;transition:all 300ms ease-in 0s;-webkit-transition:all 300ms ease-in 0s;-moz-transition:all 300ms ease-in 0s;-o-transition:all 300ms ease-in 0s}#onetrust-pc-sdk .ot-vlst-cntr{margin-bottom:0}#onetrust-pc-sdk .ot-hlst-cntr{margin-top:5px;display:inline-block;width:100%}#onetrust-pc-sdk .category-vendors-list-handler,#onetrust-pc-sdk .category-vendors-list-handler+a,#onetrust-pc-sdk .category-host-list-handler{clear:both;color:#3860be;margin-left:0;font-size:.813em;text-decoration:none;float:left;overflow:hidden}#onetrust-pc-sdk .category-vendors-list-handler:hover,#onetrust-pc-sdk .category-vendors-list-handler+a:hover,#onetrust-pc-sdk .category-host-list-handler:hover{text-decoration-line:underline}#onetrust-pc-sdk .category-vendors-list-handler+a{clear:none}#onetrust-pc-sdk .ot-vlst-cntr .ot-ext-lnk,#onetrust-pc-sdk .ot-ven-hdr .ot-ext-lnk{display:inline-block;height:13px;width:13px;background-repeat:no-repeat;margin-left:1px;margin-top:6px;cursor:pointer}#onetrust-pc-sdk .ot-ven-hdr .ot-ext-lnk{margin-bottom:-1px}#onetrust-pc-sdk .back-btn-handler{font-size:1em;text-decoration:none}#onetrust-pc-sdk .back-btn-handler:hover{opacity:.6}#onetrust-pc-sdk #ot-lst-title h3{display:inline-block;word-break:break-word;word-wrap:break-word;margin-bottom:0;color:#656565;font-size:1em;font-weight:bold;margin-left:15px}#onetrust-pc-sdk #ot-lst-title{margin:10px 0 10px 0px;font-size:1em;text-align:left}#onetrust-pc-sdk #ot-pc-hdr{margin:0 0 0 30px;height:auto;width:auto}#onetrust-pc-sdk #ot-pc-hdr input::placeholder{color:#707070;font-style:italic}#onetrust-pc-sdk #vendor-search-handler{height:31px;width:100%;border-radius:50px;font-size:.8em;padding-right:35px;padding-left:15px;float:left;margin-left:15px}#onetrust-pc-sdk .ot-ven-name{display:block;width:auto;padding-right:5px}#onetrust-pc-sdk #ot-lst-cnt{overflow-y:auto;margin-left:20px;margin-right:7px;width:calc(100% - 27px);max-height:calc(100% - 80px);height:100%;transform:translate3d(0, 0, 0)}#onetrust-pc-sdk #ot-pc-lst{width:100%;bottom:100px;position:absolute;top:60px}#onetrust-pc-sdk #ot-pc-lst:not(.ot-enbl-chr) .ot-tgl-cntr .ot-arw-cntr,#onetrust-pc-sdk #ot-pc-lst:not(.ot-enbl-chr) .ot-tgl-cntr .ot-arw-cntr *{visibility:hidden}#onetrust-pc-sdk #ot-pc-lst .ot-tgl-cntr{right:12px;position:absolute}#onetrust-pc-sdk #ot-pc-lst .ot-arw-cntr{float:right;position:relative}#onetrust-pc-sdk #ot-pc-lst .ot-arw{margin-left:10px}#onetrust-pc-sdk #ot-pc-lst .ot-acc-hdr{overflow:hidden;cursor:pointer}#onetrust-pc-sdk .ot-vlst-cntr{overflow:hidden}#onetrust-pc-sdk #ot-sel-blk{overflow:hidden;width:100%;position:sticky;position:-webkit-sticky;top:0;z-index:3}#onetrust-pc-sdk #ot-back-arw{height:12px;width:12px}#onetrust-pc-sdk .ot-lst-subhdr{width:100%;display:inline-block}#onetrust-pc-sdk .ot-search-cntr{float:left;width:78%;position:relative}#onetrust-pc-sdk .ot-search-cntr>svg{width:30px;height:30px;position:absolute;float:left;right:-15px}#onetrust-pc-sdk .ot-fltr-cntr{float:right;right:50px;position:relative}#onetrust-pc-sdk #filter-btn-handler{background-color:#3860be;border-radius:17px;display:inline-block;position:relative;width:32px;height:32px;-moz-transition:.1s ease;-o-transition:.1s ease;-webkit-transition:1s ease;transition:.1s ease;padding:0;margin:0}#onetrust-pc-sdk #filter-btn-handler:hover{background-color:#3860be}#onetrust-pc-sdk #filter-btn-handler svg{width:12px;height:12px;margin:3px 10px 0 10px;display:block;position:static;right:auto;top:auto}#onetrust-pc-sdk .ot-ven-link,#onetrust-pc-sdk .ot-ven-legclaim-link{color:#3860be;text-decoration:none;font-weight:100;display:inline-block;padding-top:10px;transform:translate(0, 1%);-o-transform:translate(0, 1%);-ms-transform:translate(0, 1%);-webkit-transform:translate(0, 1%);position:relative;z-index:2}#onetrust-pc-sdk .ot-ven-link *,#onetrust-pc-sdk .ot-ven-legclaim-link *{font-size:inherit}#onetrust-pc-sdk .ot-ven-link:hover,#onetrust-pc-sdk .ot-ven-legclaim-link:hover{text-decoration:underline}#onetrust-pc-sdk .ot-ven-hdr{width:calc(100% - 160px);height:auto;float:left;word-break:break-word;word-wrap:break-word;vertical-align:middle;padding-bottom:3px}#onetrust-pc-sdk .ot-ven-link,#onetrust-pc-sdk .ot-ven-legclaim-link{letter-spacing:.03em;font-size:.75em;font-weight:400}#onetrust-pc-sdk .ot-ven-dets{border-radius:2px;background-color:#f8f8f8}#onetrust-pc-sdk .ot-ven-dets li:first-child p:first-child{border-top:none}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:not(:first-child){border-top:1px solid #ddd !important}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:nth-child(n+3) p{display:inline-block}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:nth-child(n+3) p:nth-of-type(odd){width:30%}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:nth-child(n+3) p:nth-of-type(even){width:50%;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc p,#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc h5{padding-top:5px;padding-bottom:5px;display:block}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc h5{display:inline-block}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc p:nth-last-child(-n+1){padding-bottom:10px}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc p:nth-child(-n+2):not(.disc-pur){padding-top:10px}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc .disc-pur-cont{display:inline}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc .disc-pur{position:relative;width:50% !important;word-break:break-word;word-wrap:break-word;left:calc(30% + 17px)}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc .disc-pur:nth-child(-n+1){position:static}#onetrust-pc-sdk .ot-ven-dets p,#onetrust-pc-sdk .ot-ven-dets h5,#onetrust-pc-sdk .ot-ven-dets span{font-size:.69em;text-align:left;vertical-align:middle;word-break:break-word;word-wrap:break-word;margin:0;padding-bottom:10px;padding-left:15px;color:#2e3644}#onetrust-pc-sdk .ot-ven-dets h5{padding-top:5px}#onetrust-pc-sdk .ot-ven-dets span{color:dimgray;padding:0;vertical-align:baseline}#onetrust-pc-sdk .ot-ven-dets .ot-ven-pur h5{border-top:1px solid #e9e9e9;border-bottom:1px solid #e9e9e9;padding-bottom:5px;margin-bottom:5px;font-weight:bold}#onetrust-pc-sdk #ot-host-lst .ot-sel-all{float:right;position:relative;margin-right:42px;top:10px}#onetrust-pc-sdk #ot-host-lst .ot-sel-all input[type=checkbox]{width:auto;height:auto}#onetrust-pc-sdk #ot-host-lst .ot-sel-all label{height:20px;width:20px;padding-left:0px}#onetrust-pc-sdk #ot-host-lst .ot-acc-txt{overflow:hidden;width:95%}#onetrust-pc-sdk .ot-host-hdr{position:relative;z-index:1;pointer-events:none;width:calc(100% - 125px);float:left}#onetrust-pc-sdk .ot-host-name,#onetrust-pc-sdk .ot-host-desc{display:inline-block;width:90%}#onetrust-pc-sdk .ot-host-name{pointer-events:none}#onetrust-pc-sdk .ot-host-hdr>a{text-decoration:underline;font-size:.82em;position:relative;z-index:2;float:left;margin-bottom:5px;pointer-events:initial}#onetrust-pc-sdk .ot-host-name+a{margin-top:5px}#onetrust-pc-sdk .ot-host-name,#onetrust-pc-sdk .ot-host-name a,#onetrust-pc-sdk .ot-host-desc,#onetrust-pc-sdk .ot-host-info{color:dimgray;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-host-name,#onetrust-pc-sdk .ot-host-name a{font-weight:bold;font-size:.82em;line-height:1.3}#onetrust-pc-sdk .ot-host-name a{font-size:1em}#onetrust-pc-sdk .ot-host-expand{margin-top:3px;margin-bottom:3px;clear:both;display:block;color:#3860be;font-size:.72em;font-weight:normal}#onetrust-pc-sdk .ot-host-expand *{font-size:inherit}#onetrust-pc-sdk .ot-host-desc,#onetrust-pc-sdk .ot-host-info{font-size:.688em;line-height:1.4;font-weight:normal}#onetrust-pc-sdk .ot-host-desc{margin-top:10px}#onetrust-pc-sdk .ot-host-opt{margin:0;font-size:inherit;display:inline-block;width:100%}#onetrust-pc-sdk .ot-host-opt li>div div{font-size:.8em;padding:5px 0}#onetrust-pc-sdk .ot-host-opt li>div div:nth-child(1){width:30%;float:left}#onetrust-pc-sdk .ot-host-opt li>div div:nth-child(2){width:70%;float:left;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-host-info{border:none;display:inline-block;width:calc(100% - 10px);padding:10px;margin-bottom:10px;background-color:#f8f8f8}#onetrust-pc-sdk .ot-host-info>div{overflow:auto}#onetrust-pc-sdk #no-results{text-align:center;margin-top:30px}#onetrust-pc-sdk #no-results p{font-size:1em;color:#2e3644;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk #no-results p span{font-weight:bold}#onetrust-pc-sdk #ot-fltr-modal{width:100%;height:auto;display:none;-moz-transition:.2s ease;-o-transition:.2s ease;-webkit-transition:2s ease;transition:.2s ease;overflow:hidden;opacity:1;right:0}#onetrust-pc-sdk #ot-fltr-modal .ot-label-txt{display:inline-block;font-size:.85em;color:dimgray}#onetrust-pc-sdk #ot-fltr-cnt{z-index:2147483646;background-color:#fff;position:absolute;height:90%;max-height:300px;width:325px;left:210px;margin-top:10px;margin-bottom:20px;padding-right:10px;border-radius:3px;-webkit-box-shadow:0px 0px 12px 2px #c7c5c7;-moz-box-shadow:0px 0px 12px 2px #c7c5c7;box-shadow:0px 0px 12px 2px #c7c5c7}#onetrust-pc-sdk .ot-fltr-scrlcnt{overflow-y:auto;overflow-x:hidden;clear:both;max-height:calc(100% - 60px)}#onetrust-pc-sdk #ot-anchor{border:12px solid rgba(0,0,0,0);display:none;position:absolute;z-index:2147483647;right:55px;top:75px;transform:rotate(45deg);-o-transform:rotate(45deg);-ms-transform:rotate(45deg);-webkit-transform:rotate(45deg);background-color:#fff;-webkit-box-shadow:-3px -3px 5px -2px #c7c5c7;-moz-box-shadow:-3px -3px 5px -2px #c7c5c7;box-shadow:-3px -3px 5px -2px #c7c5c7}#onetrust-pc-sdk .ot-fltr-btns{margin-left:15px}#onetrust-pc-sdk #filter-apply-handler{margin-right:15px}#onetrust-pc-sdk .ot-fltr-opt{margin-bottom:5px;margin-left:15px;min-height:20px;width:75%;position:relative}#onetrust-pc-sdk .ot-fltr-opt p{display:inline-block;margin:0;font-size:.9em;color:#2e3644}#onetrust-pc-sdk .ot-chkbox label span{font-size:.85em;color:dimgray}#onetrust-pc-sdk .ot-chkbox input[type=checkbox]+label::after{content:none;color:#fff}#onetrust-pc-sdk .ot-chkbox input[type=checkbox]:checked+label::after{content:""}#onetrust-pc-sdk .ot-chkbox input[type=checkbox][aria-checked=mixed]+label::after{content:""}#onetrust-pc-sdk .ot-chkbox input[type=checkbox]:focus+label::before{outline-style:solid;outline-width:2px;outline-style:auto}#onetrust-pc-sdk #ot-selall-vencntr,#onetrust-pc-sdk #ot-selall-adtlvencntr,#onetrust-pc-sdk #ot-selall-hostcntr,#onetrust-pc-sdk #ot-selall-licntr,#onetrust-pc-sdk #ot-selall-gnvencntr{right:15px;position:relative;width:20px;height:20px;float:right}#onetrust-pc-sdk #ot-selall-vencntr label,#onetrust-pc-sdk #ot-selall-adtlvencntr label,#onetrust-pc-sdk #ot-selall-hostcntr label,#onetrust-pc-sdk #ot-selall-licntr label,#onetrust-pc-sdk #ot-selall-gnvencntr label{float:left;padding-left:0}#onetrust-pc-sdk #ot-ven-lst:first-child{border-top:1px solid #e2e2e2}#onetrust-pc-sdk ul{list-style:none;padding:0}#onetrust-pc-sdk ul li{position:relative;margin:0;padding:15px 15px 15px 10px;border-bottom:1px solid #e2e2e2}#onetrust-pc-sdk ul li h3,#onetrust-pc-sdk ul li h4{font-size:.75em;color:#656565;margin:0;display:inline-block;width:70%;height:auto;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk ul li p{margin:0;font-size:.7em}#onetrust-pc-sdk ul li input[type=checkbox]{position:absolute;cursor:pointer;width:100%;height:100%;opacity:0;margin:0;top:0;left:0}#onetrust-pc-sdk .ot-cat-item>button:focus,#onetrust-pc-sdk .ot-acc-cntr>button:focus,#onetrust-pc-sdk li>button:focus{outline:#000 solid 2px}#onetrust-pc-sdk .ot-cat-item>button,#onetrust-pc-sdk .ot-acc-cntr>button,#onetrust-pc-sdk li>button{position:absolute;cursor:pointer;width:100%;height:100%;margin:0;top:0;left:0;z-index:1;max-width:none;border:none}#onetrust-pc-sdk .ot-cat-item>button[aria-expanded=false]~.ot-acc-txt,#onetrust-pc-sdk .ot-acc-cntr>button[aria-expanded=false]~.ot-acc-txt,#onetrust-pc-sdk li>button[aria-expanded=false]~.ot-acc-txt{margin-top:0;max-height:0;opacity:0;overflow:hidden;width:100%;transition:.25s ease-out;display:none}#onetrust-pc-sdk .ot-cat-item>button[aria-expanded=true]~.ot-acc-txt,#onetrust-pc-sdk .ot-acc-cntr>button[aria-expanded=true]~.ot-acc-txt,#onetrust-pc-sdk li>button[aria-expanded=true]~.ot-acc-txt{transition:.1s ease-in;margin-top:10px;width:100%;overflow:auto;display:block}#onetrust-pc-sdk .ot-cat-item>button[aria-expanded=true]~.ot-acc-grpcntr,#onetrust-pc-sdk .ot-acc-cntr>button[aria-expanded=true]~.ot-acc-grpcntr,#onetrust-pc-sdk li>button[aria-expanded=true]~.ot-acc-grpcntr{width:auto;margin-top:0px;padding-bottom:10px}#onetrust-pc-sdk .ot-host-item>button:focus,#onetrust-pc-sdk .ot-ven-item>button:focus{outline:0;border:2px solid #000}#onetrust-pc-sdk .ot-hide-acc>button{pointer-events:none}#onetrust-pc-sdk .ot-hide-acc .ot-plus-minus>*,#onetrust-pc-sdk .ot-hide-acc .ot-arw-cntr>*{visibility:hidden}#onetrust-pc-sdk .ot-hide-acc .ot-acc-hdr{min-height:30px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt){padding-right:10px;width:calc(100% - 37px);margin-top:10px;max-height:calc(100% - 90px)}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk{background-color:#f9f9fc;border:1px solid #e2e2e2;width:calc(100% - 2px);padding-bottom:5px;padding-top:5px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt{border:unset;background-color:unset}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt .ot-sel-all-hdr{display:none}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt .ot-sel-all{padding-right:.5rem}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt .ot-sel-all .ot-chkbox{right:0}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) .ot-sel-all{padding-right:34px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) .ot-sel-all-chkbox{width:auto}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) ul li{border:1px solid #e2e2e2;margin-bottom:10px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) .ot-acc-cntr>.ot-acc-hdr{padding:10px 0 10px 15px}#onetrust-pc-sdk.ot-addtl-vendors .ot-sel-all-chkbox{float:right}#onetrust-pc-sdk.ot-addtl-vendors .ot-plus-minus~.ot-sel-all-chkbox{right:34px}#onetrust-pc-sdk.ot-addtl-vendors #ot-ven-lst:first-child{border-top:none}#onetrust-pc-sdk .ot-acc-cntr{position:relative;border-left:1px solid #e2e2e2;border-right:1px solid #e2e2e2;border-bottom:1px solid #e2e2e2}#onetrust-pc-sdk .ot-acc-cntr input{z-index:1}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr{background-color:#f9f9fc;padding:5px 0 5px 15px;width:auto}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr .ot-plus-minus{vertical-align:middle;top:auto}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr .ot-arw-cntr{right:10px}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr input{z-index:2}#onetrust-pc-sdk .ot-acc-cntr.ot-add-tech .ot-acc-hdr{padding:10px 0 10px 15px}#onetrust-pc-sdk .ot-acc-cntr>input[type=checkbox]:checked~.ot-acc-hdr{border-bottom:1px solid #e2e2e2}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-txt{padding-left:10px;padding-right:10px}#onetrust-pc-sdk .ot-acc-cntr button[aria-expanded=true]~.ot-acc-txt{width:auto}#onetrust-pc-sdk .ot-acc-cntr .ot-addtl-venbox{display:none}#onetrust-pc-sdk .ot-vlst-cntr{margin-bottom:0;width:100%}#onetrust-pc-sdk .ot-vensec-title{font-size:.813em;vertical-align:middle;display:inline-block}#onetrust-pc-sdk .category-vendors-list-handler,#onetrust-pc-sdk .category-vendors-list-handler+a{margin-left:0;margin-top:10px}#onetrust-pc-sdk #ot-selall-vencntr.line-through label::after,#onetrust-pc-sdk #ot-selall-adtlvencntr.line-through label::after,#onetrust-pc-sdk #ot-selall-licntr.line-through label::after,#onetrust-pc-sdk #ot-selall-hostcntr.line-through label::after,#onetrust-pc-sdk #ot-selall-gnvencntr.line-through label::after{height:auto;border-left:0;transform:none;-o-transform:none;-ms-transform:none;-webkit-transform:none;left:5px;top:9px}#onetrust-pc-sdk #ot-category-title{float:left;padding-bottom:10px;font-size:1em;width:100%}#onetrust-pc-sdk .ot-cat-grp{margin-top:10px}#onetrust-pc-sdk .ot-cat-item{line-height:1.1;margin-top:10px;display:inline-block;width:100%}#onetrust-pc-sdk .ot-btn-container{text-align:right}#onetrust-pc-sdk .ot-btn-container button{display:inline-block;font-size:.75em;letter-spacing:.08em;margin-top:19px}#onetrust-pc-sdk #close-pc-btn-handler.ot-close-icon{position:absolute;top:10px;right:0;z-index:1;padding:0;background-color:rgba(0,0,0,0);border:none}#onetrust-pc-sdk #close-pc-btn-handler.ot-close-icon svg{display:block;height:10px;width:10px}#onetrust-pc-sdk #clear-filters-handler{margin-top:20px;margin-bottom:10px;float:right;max-width:200px;text-decoration:none;color:#3860be;font-size:.9em;font-weight:bold;background-color:rgba(0,0,0,0);border-color:rgba(0,0,0,0);padding:1px}#onetrust-pc-sdk #clear-filters-handler:hover{color:#2285f7}#onetrust-pc-sdk #clear-filters-handler:focus{outline:#000 solid 1px}#onetrust-pc-sdk .ot-enbl-chr h4~.ot-tgl,#onetrust-pc-sdk .ot-enbl-chr h4~.ot-always-active{right:45px}#onetrust-pc-sdk .ot-enbl-chr h4~.ot-tgl+.ot-tgl{right:120px}#onetrust-pc-sdk .ot-enbl-chr .ot-pli-hdr.ot-leg-border-color span:first-child{width:90px}#onetrust-pc-sdk .ot-enbl-chr li.ot-subgrp>h5+.ot-tgl-cntr{padding-right:25px}#onetrust-pc-sdk .ot-plus-minus{width:20px;height:20px;font-size:1.5em;position:relative;display:inline-block;margin-right:5px;top:3px}#onetrust-pc-sdk .ot-plus-minus span{position:absolute;background:#27455c;border-radius:1px}#onetrust-pc-sdk .ot-plus-minus span:first-of-type{top:25%;bottom:25%;width:10%;left:45%}#onetrust-pc-sdk .ot-plus-minus span:last-of-type{left:25%;right:25%;height:10%;top:45%}#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-arw,#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-plus-minus span:first-of-type,#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-plus-minus span:last-of-type{transform:rotate(90deg)}#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-plus-minus span:last-of-type{left:50%;right:50%}#onetrust-pc-sdk #ot-selall-vencntr label,#onetrust-pc-sdk #ot-selall-adtlvencntr label,#onetrust-pc-sdk #ot-selall-hostcntr label,#onetrust-pc-sdk #ot-selall-licntr label{position:relative;display:inline-block;width:20px;height:20px}#onetrust-pc-sdk .ot-host-item .ot-plus-minus,#onetrust-pc-sdk .ot-ven-item .ot-plus-minus{float:left;margin-right:8px;top:10px}#onetrust-pc-sdk .ot-ven-item ul{list-style:none inside;font-size:100%;margin:0}#onetrust-pc-sdk .ot-ven-item ul li{margin:0 !important;padding:0;border:none !important}#onetrust-pc-sdk .ot-pli-hdr{color:#77808e;overflow:hidden;padding-top:7.5px;padding-bottom:7.5px;width:calc(100% - 2px);border-top-left-radius:3px;border-top-right-radius:3px}#onetrust-pc-sdk .ot-pli-hdr span:first-child{top:50%;transform:translateY(50%);max-width:90px}#onetrust-pc-sdk .ot-pli-hdr span:last-child{padding-right:10px;max-width:95px;text-align:center}#onetrust-pc-sdk .ot-li-title{float:right;font-size:.813em}#onetrust-pc-sdk .ot-pli-hdr.ot-leg-border-color{background-color:#f4f4f4;border:1px solid #d8d8d8}#onetrust-pc-sdk .ot-pli-hdr.ot-leg-border-color span:first-child{text-align:left;width:70px}#onetrust-pc-sdk li.ot-subgrp>h5,#onetrust-pc-sdk .ot-cat-header{width:calc(100% - 130px)}#onetrust-pc-sdk li.ot-subgrp>h5+.ot-tgl-cntr{padding-left:13px}#onetrust-pc-sdk .ot-acc-grpcntr .ot-acc-grpdesc{margin-bottom:5px}#onetrust-pc-sdk .ot-acc-grpcntr .ot-subgrp-cntr{border-top:1px solid #d8d8d8}#onetrust-pc-sdk .ot-acc-grpcntr .ot-vlst-cntr+.ot-subgrp-cntr{border-top:none}#onetrust-pc-sdk .ot-acc-hdr .ot-arw-cntr+.ot-tgl-cntr,#onetrust-pc-sdk .ot-acc-txt h4+.ot-tgl-cntr{padding-left:13px}#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item .ot-subgrp>h5,#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item .ot-cat-header{width:calc(100% - 145px)}#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item h5+.ot-tgl-cntr,#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item .ot-cat-header+.ot-tgl{padding-left:28px}#onetrust-pc-sdk .ot-sel-all-hdr,#onetrust-pc-sdk .ot-sel-all-chkbox{display:inline-block;width:100%;position:relative}#onetrust-pc-sdk .ot-sel-all-chkbox{z-index:1}#onetrust-pc-sdk .ot-sel-all{margin:0;position:relative;padding-right:23px;float:right}#onetrust-pc-sdk .ot-consent-hdr,#onetrust-pc-sdk .ot-li-hdr{float:right;font-size:.812em;line-height:normal;text-align:center;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-li-hdr{max-width:100px;padding-right:10px}#onetrust-pc-sdk .ot-consent-hdr{max-width:55px}#onetrust-pc-sdk #ot-selall-licntr{display:block;width:21px;height:auto;float:right;position:relative;right:80px}#onetrust-pc-sdk #ot-selall-licntr label{position:absolute}#onetrust-pc-sdk .ot-ven-ctgl{margin-left:66px}#onetrust-pc-sdk .ot-ven-litgl+.ot-arw-cntr{margin-left:81px}#onetrust-pc-sdk .ot-enbl-chr .ot-host-cnt .ot-tgl-cntr{width:auto}#onetrust-pc-sdk #ot-lst-cnt:not(.ot-host-cnt) .ot-tgl-cntr{width:auto;top:auto;height:20px}#onetrust-pc-sdk #ot-lst-cnt .ot-chkbox{position:relative;display:inline-block;width:20px;height:20px}#onetrust-pc-sdk #ot-lst-cnt .ot-chkbox label{position:absolute;padding:0;width:20px;height:20px}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info-cntr{border:1px solid #d8d8d8;padding:.75rem 2rem;padding-bottom:0;width:auto;margin-top:.5rem}#onetrust-pc-sdk .ot-acc-grpdesc+.ot-leg-btn-container{padding-left:20px;padding-right:20px;width:calc(100% - 40px);margin-bottom:5px}#onetrust-pc-sdk .ot-subgrp .ot-leg-btn-container{margin-bottom:5px}#onetrust-pc-sdk #ot-ven-lst .ot-leg-btn-container{margin-top:10px}#onetrust-pc-sdk .ot-leg-btn-container{display:inline-block;width:100%;margin-bottom:10px}#onetrust-pc-sdk .ot-leg-btn-container button{height:auto;padding:6.5px 8px;margin-bottom:0;letter-spacing:0;font-size:.75em;line-height:normal}#onetrust-pc-sdk .ot-leg-btn-container svg{display:none;height:14px;width:14px;padding-right:5px;vertical-align:sub}#onetrust-pc-sdk .ot-active-leg-btn{cursor:default;pointer-events:none}#onetrust-pc-sdk .ot-active-leg-btn svg{display:inline-block}#onetrust-pc-sdk .ot-remove-objection-handler{text-decoration:underline;padding:0;font-size:.75em;font-weight:600;line-height:1;padding-left:10px}#onetrust-pc-sdk .ot-obj-leg-btn-handler span{font-weight:bold;text-align:center;font-size:inherit;line-height:1.5}#onetrust-pc-sdk.ot-close-btn-link #close-pc-btn-handler{border:none;height:auto;line-height:1.5;text-decoration:underline;font-size:.69em;background:none;right:15px;top:15px;width:auto;font-weight:normal}#onetrust-pc-sdk .ot-pgph-link{font-size:.813em !important;margin-top:5px;position:relative}#onetrust-pc-sdk .ot-pgph-link.ot-pgph-link-subgroup{margin-bottom:1rem}#onetrust-pc-sdk .ot-pgph-contr{margin:0 2.5rem}#onetrust-pc-sdk .ot-pgph-title{font-size:1.18rem;margin-bottom:2rem}#onetrust-pc-sdk .ot-pgph-desc{font-size:1rem;font-weight:400;margin-bottom:2rem;line-height:1.5rem}#onetrust-pc-sdk .ot-pgph-desc:not(:last-child):after{content:"";width:96%;display:block;margin:0 auto;padding-bottom:2rem;border-bottom:1px solid #e9e9e9}#onetrust-pc-sdk .ot-cat-header{float:left;font-weight:600;font-size:.875em;line-height:1.5;max-width:90%;vertical-align:middle}#onetrust-pc-sdk .ot-vnd-item>button:focus{outline:#000 solid 2px}#onetrust-pc-sdk .ot-vnd-item>button{position:absolute;cursor:pointer;width:100%;height:100%;margin:0;top:0;left:0;z-index:1;max-width:none;border:none}#onetrust-pc-sdk .ot-vnd-item>button[aria-expanded=false]~.ot-acc-txt{margin-top:0;max-height:0;opacity:0;overflow:hidden;width:100%;transition:.25s ease-out;display:none}#onetrust-pc-sdk .ot-vnd-item>button[aria-expanded=true]~.ot-acc-txt{transition:.1s ease-in;margin-top:10px;width:100%;overflow:auto;display:block}#onetrust-pc-sdk .ot-vnd-item>button[aria-expanded=true]~.ot-acc-grpcntr{width:auto;margin-top:0px;padding-bottom:10px}#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item{position:relative;border-radius:2px;margin:0;padding:0;border:1px solid #d8d8d8;border-top:none;width:calc(100% - 2px);float:left}#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item:first-of-type{margin-top:10px;border-top:1px solid #d8d8d8}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpdesc{padding-left:20px;padding-right:20px;width:calc(100% - 40px);font-size:.812em;margin-bottom:10px;margin-top:15px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpdesc>ul{padding-top:10px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpdesc>ul li{padding-top:0;line-height:1.5;padding-bottom:10px}#onetrust-pc-sdk .ot-accordion-layout div+.ot-acc-grpdesc{margin-top:5px}#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr:first-child{margin-top:10px}#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr:last-child,#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr:last-child{margin-bottom:5px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{padding-top:11.5px;padding-bottom:11.5px;padding-left:20px;padding-right:20px;width:calc(100% - 40px);display:inline-block}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-txt{width:100%;padding:0}#onetrust-pc-sdk .ot-accordion-layout .ot-subgrp-cntr{padding-left:20px;padding-right:15px;padding-bottom:0;width:calc(100% - 35px)}#onetrust-pc-sdk .ot-accordion-layout .ot-subgrp{padding-right:5px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpcntr{z-index:1;position:relative}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header+.ot-arw-cntr{position:absolute;top:50%;transform:translateY(-50%);right:20px;margin-top:-2px}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header+.ot-arw-cntr .ot-arw{width:15px;height:20px;margin-left:5px;color:dimgray}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{float:none;color:#2e3644;margin:0;display:inline-block;height:auto;word-wrap:break-word;min-height:inherit}#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr,#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr{padding-left:20px;width:calc(100% - 20px);display:inline-block;margin-top:0;padding-bottom:2px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{position:relative;min-height:25px}#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl,#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active{position:absolute;top:50%;transform:translateY(-50%);right:20px}#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl+.ot-tgl{right:95px}#onetrust-pc-sdk .ot-accordion-layout .category-vendors-list-handler,#onetrust-pc-sdk .ot-accordion-layout .category-vendors-list-handler+a{margin-top:5px}#onetrust-pc-sdk #ot-lst-cnt{margin-top:1rem;max-height:calc(100% - 96px)}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info-cntr{border:1px solid #d8d8d8;padding:.75rem 2rem;padding-bottom:0;width:auto;margin-top:.5rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info{margin-bottom:1rem;padding-left:.75rem;padding-right:.75rem;display:flex;flex-direction:column}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info[data-vnd-info-key*=DPOEmail]{border-top:1px solid #d8d8d8;padding-top:1rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info[data-vnd-info-key*=DPOLink]{border-bottom:1px solid #d8d8d8;padding-bottom:1rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info .ot-vnd-lbl{font-weight:bold;font-size:.85em;margin-bottom:.5rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info .ot-vnd-cnt{margin-left:.5rem;font-weight:500;font-size:.85rem}#onetrust-pc-sdk .ot-vs-list,#onetrust-pc-sdk .ot-vnd-serv{width:auto;padding:1rem 1.25rem;padding-bottom:0}#onetrust-pc-sdk .ot-vs-list .ot-vnd-serv-hdr-cntr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-serv-hdr-cntr{padding-bottom:.75rem;border-bottom:1px solid #d8d8d8}#onetrust-pc-sdk .ot-vs-list .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr{font-weight:600;font-size:.95em;line-height:2;margin-left:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item{border:none;margin:0;padding:0}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item button,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item button{outline:none;border-bottom:1px solid #d8d8d8}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item button[aria-expanded=true],#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item button[aria-expanded=true]{border-bottom:none}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item:first-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item:first-child{margin-top:.25rem;border-top:unset}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item:last-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item:last-child{margin-bottom:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item:last-child button,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item:last-child button{border-bottom:none}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info-cntr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info-cntr{border:1px solid #d8d8d8;padding:.75rem 1.75rem;padding-bottom:0;width:auto;margin-top:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info{margin-bottom:1rem;padding-left:.75rem;padding-right:.75rem;display:flex;flex-direction:column}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOEmail],#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOEmail]{border-top:1px solid #d8d8d8;padding-top:1rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOLink],#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOLink]{border-bottom:1px solid #d8d8d8;padding-bottom:1rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info .ot-vnd-lbl,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info .ot-vnd-lbl{font-weight:bold;font-size:.85em;margin-bottom:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info .ot-vnd-cnt,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info .ot-vnd-cnt{margin-left:.5rem;font-weight:500;font-size:.85rem}#onetrust-pc-sdk .ot-vs-list.ot-vnd-subgrp-cnt,#onetrust-pc-sdk .ot-vnd-serv.ot-vnd-subgrp-cnt{padding-left:40px}#onetrust-pc-sdk .ot-vs-list.ot-vnd-subgrp-cnt .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr,#onetrust-pc-sdk .ot-vnd-serv.ot-vnd-subgrp-cnt .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr{font-size:.8em}#onetrust-pc-sdk .ot-vs-list.ot-vnd-subgrp-cnt .ot-cat-header,#onetrust-pc-sdk .ot-vnd-serv.ot-vnd-subgrp-cnt .ot-cat-header{font-size:.8em}#onetrust-pc-sdk .ot-subgrp-cntr .ot-vnd-serv{margin-bottom:1rem;padding:1rem .95rem}#onetrust-pc-sdk .ot-subgrp-cntr .ot-vnd-serv .ot-vnd-serv-hdr-cntr{padding-bottom:.75rem;border-bottom:1px solid #d8d8d8}#onetrust-pc-sdk .ot-subgrp-cntr .ot-vnd-serv .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr{font-weight:700;font-size:.8em;line-height:20px;margin-left:.82rem}#onetrust-pc-sdk .ot-subgrp-cntr .ot-cat-header{font-weight:700;font-size:.8em;line-height:20px}#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-vnd-serv .ot-vnd-lst-cont .ot-accordion-layout .ot-acc-hdr div.ot-chkbox{margin-left:.82rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr{padding:.7rem 0;margin:0;display:flex;width:100%;align-items:center;justify-content:space-between}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr div:first-child,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr div:first-child,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr div:first-child{margin-left:.5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr div:last-child,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr div:last-child,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr div:last-child{margin-right:.5rem;margin-left:.5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-always-active{position:relative;right:unset;top:unset;transform:unset}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-plus-minus{top:0}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-arw-cntr{float:none;top:unset;right:unset;transform:unset;margin-top:-2px;position:relative}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-cat-header{flex:1;margin:0 .5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-tgl{position:relative;transform:none;right:0;top:0;float:none}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox{position:relative;margin:0 .5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox label{padding:0}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox label::before{position:relative}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox input{position:absolute;cursor:pointer;width:100%;height:100%;opacity:0;margin:0;top:0;left:0;z-index:1}#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp .ot-acc-hdr h5.ot-cat-header,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp .ot-acc-hdr h4.ot-cat-header{margin:0}#onetrust-pc-sdk .ot-vs-config .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp h5{top:0;line-height:20px}#onetrust-pc-sdk .ot-vs-list{display:flex;flex-direction:column;padding:0;margin:.5rem 4px}#onetrust-pc-sdk .ot-vs-selc-all{display:flex;padding:0;float:unset;align-items:center;justify-content:flex-start}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf{justify-content:flex-end}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf.ot-caret-conf .ot-sel-all-chkbox{margin-right:48px}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf .ot-sel-all-chkbox{margin:0;padding:0;margin-right:14px;justify-content:flex-end}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf #ot-selall-vencntr.ot-chkbox,#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf #ot-selall-vencntr.ot-tgl{display:inline-block;right:unset;width:auto;height:auto;float:none}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf #ot-selall-vencntr label{width:45px;height:25px}#onetrust-pc-sdk .ot-vs-selc-all .ot-sel-all-chkbox{margin-right:11px;margin-left:.75rem;display:flex;align-items:center}#onetrust-pc-sdk .ot-vs-selc-all .sel-all-hdr{margin:0 1.25rem;font-size:.812em;line-height:normal;text-align:center;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-vnd-list-cnt #ot-selall-vencntr.ot-chkbox{float:unset;right:0}#onetrust-pc-sdk[dir=rtl] #ot-back-arw,#onetrust-pc-sdk[dir=rtl] input~.ot-acc-hdr .ot-arw{transform:rotate(180deg);-o-transform:rotate(180deg);-ms-transform:rotate(180deg);-webkit-transform:rotate(180deg)}#onetrust-pc-sdk[dir=rtl] input:checked~.ot-acc-hdr .ot-arw{transform:rotate(270deg);-o-transform:rotate(270deg);-ms-transform:rotate(270deg);-webkit-transform:rotate(270deg)}#onetrust-pc-sdk[dir=rtl] .ot-chkbox label::after{transform:rotate(45deg);-webkit-transform:rotate(45deg);-o-transform:rotate(45deg);-ms-transform:rotate(45deg);border-left:0;border-right:3px solid}#onetrust-pc-sdk[dir=rtl] .ot-search-cntr>svg{right:0}@media only screen and (max-width: 600px){#onetrust-pc-sdk.otPcCenter{left:0;min-width:100%;height:100%;top:0;border-radius:0}#onetrust-pc-sdk #ot-pc-content,#onetrust-pc-sdk.ot-ftr-stacked .ot-btn-container{margin:1px 3px 0 10px;padding-right:10px;width:calc(100% - 23px)}#onetrust-pc-sdk .ot-btn-container button{max-width:none;letter-spacing:.01em}#onetrust-pc-sdk #close-pc-btn-handler{top:10px;right:17px}#onetrust-pc-sdk p{font-size:.7em}#onetrust-pc-sdk #ot-pc-hdr{margin:10px 10px 0 5px;width:calc(100% - 15px)}#onetrust-pc-sdk .vendor-search-handler{font-size:1em}#onetrust-pc-sdk #ot-back-arw{margin-left:12px}#onetrust-pc-sdk #ot-lst-cnt{margin:0;padding:0 5px 0 10px;min-width:95%}#onetrust-pc-sdk .switch+p{max-width:80%}#onetrust-pc-sdk .ot-ftr-stacked button{width:100%}#onetrust-pc-sdk #ot-fltr-cnt{max-width:320px;width:90%;border-top-right-radius:0;border-bottom-right-radius:0;margin:0;margin-left:15px;left:auto;right:40px;top:85px}#onetrust-pc-sdk .ot-fltr-opt{margin-left:25px;margin-bottom:10px}#onetrust-pc-sdk .ot-pc-refuse-all-handler{margin-bottom:0}#onetrust-pc-sdk #ot-fltr-cnt{right:40px}}@media only screen and (max-width: 476px){#onetrust-pc-sdk .ot-fltr-cntr,#onetrust-pc-sdk #ot-fltr-cnt{right:10px}#onetrust-pc-sdk #ot-anchor{right:25px}#onetrust-pc-sdk button{width:100%}#onetrust-pc-sdk:not(.ot-addtl-vendors) #ot-pc-lst:not(.ot-enbl-chr) .ot-sel-all{padding-right:9px}#onetrust-pc-sdk:not(.ot-addtl-vendors) #ot-pc-lst:not(.ot-enbl-chr) .ot-tgl-cntr{right:0}}@media(min-width: 768px){#onetrust-pc-sdk.ot-tgl-with-label .ot-label-status{display:inline}#onetrust-pc-sdk.ot-tgl-with-label #ot-pc-lst .ot-label-status{display:none}}@media only screen and (max-width: 896px)and (max-height: 425px)and (orientation: landscape){#onetrust-pc-sdk.otPcCenter{left:0;top:0;min-width:100%;height:100%;border-radius:0}#onetrust-pc-sdk .ot-pc-header{height:auto;min-height:20px}#onetrust-pc-sdk .ot-pc-header .ot-pc-logo{max-height:30px}#onetrust-pc-sdk .ot-pc-footer{max-height:60px;overflow-y:auto}#onetrust-pc-sdk #ot-pc-content,#onetrust-pc-sdk #ot-pc-lst{bottom:70px}#onetrust-pc-sdk.ot-ftr-stacked #ot-pc-content{bottom:70px}#onetrust-pc-sdk #ot-anchor{left:initial;right:50px}#onetrust-pc-sdk #ot-lst-title{margin-top:12px}#onetrust-pc-sdk #ot-lst-title *{font-size:inherit}#onetrust-pc-sdk #ot-pc-hdr input{margin-right:0;padding-right:45px}#onetrust-pc-sdk .switch+p{max-width:85%}#onetrust-pc-sdk #ot-sel-blk{position:static}#onetrust-pc-sdk #ot-pc-lst{overflow:auto}#onetrust-pc-sdk #ot-lst-cnt{max-height:none;overflow:initial}#onetrust-pc-sdk #ot-lst-cnt.no-results{height:auto}#onetrust-pc-sdk input{font-size:1em !important}#onetrust-pc-sdk p{font-size:.6em}#onetrust-pc-sdk #ot-fltr-modal{width:100%;top:0}#onetrust-pc-sdk ul li p,#onetrust-pc-sdk .category-vendors-list-handler,#onetrust-pc-sdk .category-vendors-list-handler+a,#onetrust-pc-sdk .category-host-list-handler{font-size:.6em}#onetrust-pc-sdk.ot-shw-fltr #ot-anchor{display:none !important}#onetrust-pc-sdk.ot-shw-fltr #ot-pc-lst{height:100% !important;overflow:hidden;top:0px}#onetrust-pc-sdk.ot-shw-fltr #ot-fltr-cnt{margin:0;height:100%;max-height:none;padding:10px;top:0;width:calc(100% - 20px);position:absolute;right:0;left:0;max-width:none}#onetrust-pc-sdk.ot-shw-fltr .ot-fltr-scrlcnt{max-height:calc(100% - 65px)}}
            #onetrust-consent-sdk #onetrust-pc-sdk,
                #onetrust-consent-sdk #ot-search-cntr,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-switch.ot-toggle,
                #onetrust-consent-sdk #onetrust-pc-sdk ot-grp-hdr1 .checkbox,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-title:after
                ,#onetrust-consent-sdk #onetrust-pc-sdk #ot-sel-blk,
                        #onetrust-consent-sdk #onetrust-pc-sdk #ot-fltr-cnt,
                        #onetrust-consent-sdk #onetrust-pc-sdk #ot-anchor {
                    background-color: #FFF;
                }
               
            #onetrust-consent-sdk #onetrust-pc-sdk h3,
                #onetrust-consent-sdk #onetrust-pc-sdk h4,
                #onetrust-consent-sdk #onetrust-pc-sdk h5,
                #onetrust-consent-sdk #onetrust-pc-sdk h6,
                #onetrust-consent-sdk #onetrust-pc-sdk p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-ven-lst .ot-ven-opts p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-desc,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-title,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-li-title,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-sel-all-hdr span,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-info,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-fltr-modal #modal-header,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-checkbox label span,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst #ot-sel-blk p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst #ot-lst-title h3,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst .back-btn-handler p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst .ot-ven-name,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst #ot-ven-lst .consent-category,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-inactive-leg-btn,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-label-status,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-chkbox label span,
                #onetrust-consent-sdk #onetrust-pc-sdk #clear-filters-handler,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-optout-signal
                {
                    color: #2E2E2E;
                }
             #onetrust-consent-sdk #onetrust-pc-sdk .privacy-notice-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-pgph-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk .category-vendors-list-handler,
                    #onetrust-consent-sdk #onetrust-pc-sdk .category-vendors-list-handler + a,
                    #onetrust-consent-sdk #onetrust-pc-sdk .category-host-list-handler,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-ven-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-ven-legclaim-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-name a,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-acc-hdr .ot-host-expand,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-info a,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-content #ot-pc-desc .ot-link-btn,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info a,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info a
                    {
                        color: #007398;
                    }
            #onetrust-consent-sdk #onetrust-pc-sdk .category-vendors-list-handler:hover { text-decoration: underline;}
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-grpcntr.ot-acc-txt,
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-txt .ot-subgrp-tgl .ot-switch.ot-toggle
             {
                background-color: #F8F8F8;
            }
             #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-info,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-txt .ot-ven-dets
                            {
                                background-color: #F8F8F8;
                            }
        #onetrust-consent-sdk #onetrust-pc-sdk
            button:not(#clear-filters-handler):not(.ot-close-icon):not(#filter-btn-handler):not(.ot-remove-objection-handler):not(.ot-obj-leg-btn-handler):not([aria-expanded]):not(.ot-link-btn),
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-active-leg-btn {
                background-color: #007398;border-color: #007398;
                color: #FFF;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-active-menu {
                border-color: #007398;
            }
            
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-remove-objection-handler{
                background-color: transparent;
                border: 1px solid transparent;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-inactive-leg-btn {
                background-color: #FFFFFF;
                color: #4D4D4D; border-color: #4D4D4D;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-tgl input:focus + .ot-switch, .ot-switch .ot-switch-nob, .ot-switch .ot-switch-nob:before,
            #onetrust-pc-sdk .ot-checkbox input[type="checkbox"]:focus + label::before,
            #onetrust-pc-sdk .ot-chkbox input[type="checkbox"]:focus + label::before {
                outline-color: #000000;
                outline-width: 1px;
                outline-offset: 1px;
            }
            #onetrust-pc-sdk .ot-host-item > button:focus, #onetrust-pc-sdk .ot-ven-item > button:focus {
                border: 1px solid #000000;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk *:focus,
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-vlst-cntr > a:focus {
               outline: 1px solid #000000;
               outline-offset: 1px;
            }#onetrust-pc-sdk .ot-vlst-cntr .ot-ext-lnk,  #onetrust-pc-sdk .ot-ven-hdr .ot-ext-lnk{
                    background-image: url('https://cdn.cookielaw.org/logos/static/ot_external_link.svg');
                }
            /*! v6.10.0 2021-01-18 */
div#onetrust-consent-sdk #onetrust-banner-sdk{border-top:2px solid #d0d3d4!important;box-shadow:none;padding:24px}div#onetrust-consent-sdk button{border-radius:0!important;box-shadow:none!important;box-sizing:border-box!important;font-size:20px!important;font-weight:400!important;letter-spacing:0!important;max-width:none!important;white-space:nowrap!important}div#onetrust-consent-sdk button:not(.ot-link-btn){background-color:#007dbc!important;border:2px solid #007dbc!important;color:#fff!important;height:48px!important;padding:0 1em!important;width:auto!important}div#onetrust-consent-sdk button:hover{background-color:#fff!important;border-color:#949494!important;color:#2e2e2e!important}div#onetrust-consent-sdk button.ot-link-btn{color:#007dbc!important;font-size:16px!important;text-decoration:underline}div#onetrust-consent-sdk button.ot-link-btn:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-consent-sdk a,div#onetrust-pc-sdk a{color:#007dbc!important;text-decoration:underline!important}div#onetrust-consent-sdk a,div#onetrust-consent-sdk button,div#onetrust-consent-sdk p:hover{opacity:1!important}div#onetrust-consent-sdk a:focus,div#onetrust-consent-sdk button:focus,div#onetrust-consent-sdk input:focus{outline:2px solid #949494!important;outline-offset:1px!important}div#onetrust-banner-sdk .ot-sdk-container{padding:0;width:auto}div#onetrust-banner-sdk .ot-sdk-row{align-items:flex-start;box-sizing:border-box;display:flex;flex-direction:column;justify-content:space-between;margin:auto;max-width:1152px}div#onetrust-banner-sdk .ot-sdk-row:after{display:none}div#onetrust-banner-sdk #onetrust-group-container,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-group-container,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{flex-grow:1;width:auto}div#onetrust-banner-sdk #onetrust-policy,div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy{margin:0;overflow:visible}div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy-text,div#onetrust-consent-sdk #onetrust-policy-text{font-size:16px;line-height:24px;max-width:44em;margin:0}div#onetrust-consent-sdk #onetrust-policy-text a[href]{font-weight:400;margin-left:8px}div#onetrust-banner-sdk #onetrust-button-group-parent{flex:0 0 auto;margin:32px 0 0;width:100%}div#onetrust-banner-sdk #onetrust-button-group{display:flex;flex-direction:row;flex-wrap:wrap;justify-content:flex-end;margin:-8px}div#onetrust-banner-sdk .banner-actions-container{display:flex;flex:1 0 auto}div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button:last-of-type,div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto;margin:8px;width:auto}div#onetrust-consent-sdk #onetrust-pc-btn-handler{background-color:#fff!important;color:inherit!important}div#onetrust-banner-sdk #onetrust-close-btn-container{display:none}@media only screen and (min-width:556px){div#onetrust-consent-sdk #onetrust-banner-sdk{padding:40px}div#onetrust-banner-sdk #onetrust-policy{margin:0 40px 0 0}div#onetrust-banner-sdk .ot-sdk-row{align-items:center;flex-direction:row}div#onetrust-banner-sdk #onetrust-button-group-parent,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-button-group-parent,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{margin:0;padding:0;width:auto}div#onetrust-banner-sdk #onetrust-button-group,div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group{align-items:stretch;flex-direction:column-reverse;margin:0}div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto}}@media only screen and (min-width:768px){div#onetrust-banner-sdk #onetrust-policy{margin:0 48px 0 0}div#onetrust-consent-sdk #onetrust-banner-sdk{padding:48px}}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-size:16px;line-height:24px}div#onetrust-consent-sdk #onetrust-pc-sdk p,div#onetrust-pc-sdk #ot-pc-desc,div#onetrust-pc-sdk .category-host-list-handler,div#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{font-size:16px;font-weight:400;line-height:24px}div#onetrust-consent-sdk a:hover,div#onetrust-pc-sdk a:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-pc-sdk{border-radius:0;bottom:0;height:auto;left:0;margin:auto;max-width:100%;overflow:hidden;right:0;top:0;width:512px;max-height:800px}div#onetrust-pc-sdk .ot-pc-header{display:none}div#onetrust-pc-sdk #ot-pc-content{overscroll-behavior:contain;padding:0 12px 0 24px;margin:16px 4px 0 0;top:0;right:16px;left:0;width:auto}div#onetrust-pc-sdk #ot-category-title,div#onetrust-pc-sdk #ot-pc-title{font-size:24px;font-weight:400;line-height:32px;margin:0 0 16px}div#onetrust-pc-sdk #ot-pc-desc{padding:0}div#onetrust-pc-sdk #ot-pc-desc a{display:inline}div#onetrust-pc-sdk #accept-recommended-btn-handler{display:none!important}div#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:2px solid #007dbc;outline-offset:-1px;transition:none}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item{border-width:0 0 2px}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item:first-of-type{border-width:2px 0}div#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{padding:8px 0;width:100%}div#onetrust-pc-sdk .ot-plus-minus{transform:translateY(2px)}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item>button{background:0 0!important;border:0!important;height:44px!important;max-width:none!important;width:calc(100% - 48px)!important}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-weight:700}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr{padding:0}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item .ot-acc-grpdesc{padding:0;width:100%}div#onetrust-pc-sdk .ot-acc-grpcntr .ot-subgrp-cntr{border:0;padding:0}div#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp{margin:0}div#onetrust-pc-sdk .ot-always-active-group .ot-cat-header{width:calc(100% - 160px)}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{width:calc(100% - 88px)}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active{color:inherit;font-size:12px;font-weight:400;line-height:1.5;padding-right:48px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:before{border-radius:12px;position:absolute;right:0;top:0;content:'';background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:after{border-radius:50%;position:absolute;right:5px;top:4px;content:'';background-color:#007dbc;height:12px;width:12px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active,div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl{right:2px}div#onetrust-pc-sdk .ot-switch{display:block;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob,div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob{background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob:before{background-color:#737373;height:8px;left:4px;top:4px;width:8px}div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob:before{background-color:#007dbc;height:12px;left:0;top:2px;width:12px}div#onetrust-pc-sdk .ot-tgl input:focus+.ot-switch .ot-switch-nob{box-shadow:0 0;outline:2px solid #007dbc!important;outline-offset:1px;transition:none}div#onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-grpcntr.ot-acc-txt{background-color:transparent;padding-left:3px}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr,div#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr{overflow:visible;width:100%}div#onetrust-pc-sdk .ot-pc-footer{border-top:0 solid}div#onetrust-pc-sdk .ot-btn-container{padding-top:24px;text-align:center}div#onetrust-pc-sdk .ot-pc-footer button{margin:8px 0;background-color:#fff}div#onetrust-pc-sdk .ot-pc-footer-logo{background-color:#fff}div#onetrust-pc-sdk #ot-lst-title span{font-size:24px;font-weight:400;line-height:32px}div#onetrust-pc-sdk #ot-host-lst .ot-host-desc,div#onetrust-pc-sdk #ot-host-lst .ot-host-expand,div#onetrust-pc-sdk #ot-host-lst .ot-host-name,div#onetrust-pc-sdk #ot-host-lst .ot-host-name a,div#onetrust-pc-sdk .back-btn-handler,div#onetrust-pc-sdk .ot-host-opt li>div div{font-size:16px;font-weight:400;line-height:24px}div#onetrust-pc-sdk #ot-pc-lst{top:0}div#onetrust-pc-sdk .back-btn-handler{text-decoration:none!important}div#onetrust-pc-sdk #filter-btn-handler:hover svg{filter:invert(1)}div#onetrust-pc-sdk .back-btn-handler svg{width:16px;height:16px}div#onetrust-pc-sdk .ot-host-item>button{background:0 0!important;border:0!important;height:66px!important;max-width:none!important;width:calc(100% - 5px)!important;transform:translate(2px,2px)}div#onetrust-pc-sdk .ot-host-item{border-bottom:2px solid #b9b9b9;padding:0}div#onetrust-pc-sdk .ot-host-item .ot-acc-hdr{margin:0 0 -6px;padding:8px 0}div#onetrust-pc-sdk ul li:first-child{border-top:2px solid #b9b9b9}div#onetrust-pc-sdk .ot-host-item .ot-plus-minus{margin:0 8px 0 0}div#onetrust-pc-sdk .ot-search-cntr{width:calc(100% - 48px)}div#onetrust-pc-sdk .ot-host-opt .ot-host-info{background-color:transparent}div#onetrust-pc-sdk .ot-host-opt li>div div{padding:0}div#onetrust-pc-sdk #vendor-search-handler{border-radius:0;border-color:#939393;border-style:solid;border-width:2px 0 2px 2px;font-size:20px;height:48px;margin:0}div#onetrust-pc-sdk .ot-lst-subhdr svg{right:0;top:8px}div#onetrust-pc-sdk .ot-fltr-cntr{box-sizing:border-box;right:0;width:48px}div#onetrust-pc-sdk #filter-btn-handler{width:48px!important;padding:8px!important}div#onetrust-pc-sdk #ot-fltr-modal{max-height:400px}
.ot-sdk-cookie-policy{font-family:inherit;font-size:16px}.ot-sdk-cookie-policy.otRelFont{font-size:1rem}.ot-sdk-cookie-policy h3,.ot-sdk-cookie-policy h4,.ot-sdk-cookie-policy h6,.ot-sdk-cookie-policy p,.ot-sdk-cookie-policy li,.ot-sdk-cookie-policy a,.ot-sdk-cookie-policy th,.ot-sdk-cookie-policy #cookie-policy-description,.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group,.ot-sdk-cookie-policy #cookie-policy-title{color:dimgray}.ot-sdk-cookie-policy #cookie-policy-description{margin-bottom:1em}.ot-sdk-cookie-policy h4{font-size:1.2em}.ot-sdk-cookie-policy h6{font-size:1em;margin-top:2em}.ot-sdk-cookie-policy th{min-width:75px}.ot-sdk-cookie-policy a,.ot-sdk-cookie-policy a:hover{background:#fff}.ot-sdk-cookie-policy thead{background-color:#f6f6f4;font-weight:bold}.ot-sdk-cookie-policy .ot-mobile-border{display:none}.ot-sdk-cookie-policy section{margin-bottom:2em}.ot-sdk-cookie-policy table{border-collapse:inherit}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy{font-family:inherit;font-size:1rem}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h3,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h4,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h6,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy p,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy li,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-title{color:dimgray}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description{margin-bottom:1em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-subgroup{margin-left:1.5em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group-desc,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-table-header,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy span,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td{font-size:.9em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td span,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td a{font-size:inherit}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group{font-size:1em;margin-bottom:.6em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-title{margin-bottom:1.2em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy>section{margin-bottom:1em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th{min-width:75px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a:hover{background:#fff}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy thead{background-color:#f6f6f4;font-weight:bold}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-mobile-border{display:none}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy section{margin-bottom:2em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-subgroup ul li{list-style:disc;margin-left:1.5em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-subgroup ul li h4{display:inline-block}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table{border-collapse:inherit;margin:auto;border:1px solid #d7d7d7;border-radius:5px;border-spacing:initial;width:100%;overflow:hidden}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table th,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table td{border-bottom:1px solid #d7d7d7;border-right:1px solid #d7d7d7}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr:last-child td{border-bottom:0px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr th:last-child,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr td:last-child{border-right:0px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-host,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-cookies-type{width:25%}.ot-sdk-cookie-policy[dir=rtl]{text-align:left}#ot-sdk-cookie-policy h3{font-size:1.5em}@media only screen and (max-width: 530px){.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) table,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) thead,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tbody,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) th,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr{display:block}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) thead tr{position:absolute;top:-9999px;left:-9999px}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr{margin:0 0 1em 0}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr:nth-child(odd),.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr:nth-child(odd) a{background:#f6f6f4}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td{border:none;border-bottom:1px solid #eee;position:relative;padding-left:50%}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td:before{position:absolute;height:100%;left:6px;width:40%;padding-right:10px}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) .ot-mobile-border{display:inline-block;background-color:#e4e4e4;position:absolute;height:100%;top:0;left:45%;width:2px}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td:before{content:attr(data-label);font-weight:bold}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) li{word-break:break-word;word-wrap:break-word}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table{overflow:hidden}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table td{border:none;border-bottom:1px solid #d7d7d7}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy thead,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy tbody,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy tr{display:block}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-host,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-cookies-type{width:auto}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy tr{margin:0 0 1em 0}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td:before{height:100%;width:40%;padding-right:10px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td:before{content:attr(data-label);font-weight:bold}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy li{word-break:break-word;word-wrap:break-word}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy thead tr{position:absolute;top:-9999px;left:-9999px;z-index:-9999}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr:last-child td{border-bottom:1px solid #d7d7d7;border-right:0px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr:last-child td:last-child{border-bottom:0px}}
                
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h5,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h6,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy li,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy p,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy span,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description {
                        color: #696969;
                    }
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th {
                        color: #696969;
                    }
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group {
                        color: #696969;
                    }
                    
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-title {
                            color: #696969;
                        }
                    
            
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table th {
                            background-color: #F8F8F8;
                        }
                    
            .ot-floating-button__front{background-image:url('https://cdn.cookielaw.org/logos/static/ot_persistent_cookie_icon.png')}</style><script src="https://static.addtoany.com/menu/modules/core.pt5ow5lr.js" type="module"></script><style></style><style type="text/css">.a2a_hide{display:none}.a2a_logo_color{background-color:#0166ff}.a2a_menu,.a2a_menu *{-moz-box-sizing:content-box;-webkit-box-sizing:content-box;box-sizing:content-box;float:none;margin:0;padding:0;position:static;height:auto;width:auto}.a2a_menu{border-radius:6px;display:none;direction:ltr;background:#FFF;font:16px sans-serif-light,HelveticaNeue-Light,"Helvetica Neue Light","Helvetica Neue",Arial,Helvetica,"Liberation Sans",sans-serif;color:#000;line-height:12px;border:1px solid #CCC;vertical-align:baseline;overflow:hidden}.a2a_mini{min-width:200px;position:absolute;width:300px;z-index:9999997}.a2a_overlay{display:none;background:#616c7deb;backdrop-filter:blur(4px);-webkit-backdrop-filter:blur(4px);position:fixed;top:0;right:0;left:0;bottom:0;z-index:9999998;-webkit-tap-highlight-color:transparent;transition:opacity .14s,backdrop-filter .14s}.a2a_full{background:#FFF;border:1px solid #FFF;box-shadow:#2a2a2a1a 0 0 20px 10px;height:auto;height:calc(320px);top:15%;left:50%;margin-left:-320px;position:fixed;text-align:center;width:640px;z-index:9999999;transition:transform .14s,opacity .14s}.a2a_full_footer,.a2a_full_header,.a2a_full_services{border:0;margin:0;padding:12px;box-sizing:border-box}.a2a_full_header{padding-bottom:8px}.a2a_full_services{height:280px;overflow-y:scroll;padding:0 12px;-webkit-overflow-scrolling:touch}.a2a_full_services .a2a_i{display:inline-block;float:none;width:181px;width:calc(33.334% - 18px)}div.a2a_full_footer{font-size:12px;text-align:center;padding:8px 14px}div.a2a_full_footer a,div.a2a_full_footer a:visited{display:inline;font-size:12px;line-height:14px;padding:8px 14px}div.a2a_full_footer a:focus,div.a2a_full_footer a:hover{background:0 0;border:0;color:#0166FF}div.a2a_full_footer a span.a2a_s_a2a,div.a2a_full_footer a span.a2a_w_a2a{background-size:14px;border-radius:3px;display:inline-block;height:14px;line-height:14px;margin:0 3px 0 0;vertical-align:top;width:14px}.a2a_modal{height:0;left:50%;margin-left:-320px;position:fixed;text-align:center;top:15%;width:640px;z-index:9999999;transition:transform .14s,opacity .14s;-webkit-tap-highlight-color:transparent}.a2a_modal_body{background:0 0;border:0;font:24px sans-serif-light,HelveticaNeue-Light,"Helvetica Neue Light","Helvetica Neue",Arial,Helvetica,"Liberation Sans",sans-serif;position:relative;height:auto;width:auto}.a2a_thanks{color:#fff;height:auto;margin-top:20px;width:auto}.a2a_thanks>div:first-child{margin:0 0 40px 0}.a2a_thanks div *{height:inherit}#a2a_copy_link{background:#FFF;border:1px solid #FFF;cursor:pointer;margin-top:15%}label.a2a_s_link#a2a_copy_link_icon,label.a2a_w_link#a2a_copy_link_icon{background-size:48px;border-radius:0;display:inline-block;height:48px;left:0;line-height:48px;margin:0 3px 0 0;position:absolute;vertical-align:top;width:48px}#a2a_modal input#a2a_copy_link_text{background-color:transparent;border:0;color:#2A2A2A;cursor:pointer;font:inherit;height:48px;left:62px;max-width:initial;min-height:auto;padding:0;position:relative;width:564px;width:calc(100% - 76px)}#a2a_copy_link_copied{background-color:#0166ff;color:#fff;display:none;font:inherit;font-size:16px;margin-top:1px;padding:3px 8px}@media (forced-colors:active){.a2a_color_buttons a,.a2a_svg{forced-color-adjust:none}}@media (prefers-color-scheme:dark){.a2a_menu a,.a2a_menu a.a2a_i,.a2a_menu a.a2a_i:visited,.a2a_menu a.a2a_more,i.a2a_i{border-color:#2a2a2a!important;color:#fff!important}.a2a_menu a.a2a_i:active,.a2a_menu a.a2a_i:focus,.a2a_menu a.a2a_i:hover,.a2a_menu a.a2a_more:active,.a2a_menu a.a2a_more:focus,.a2a_menu a.a2a_more:hover,.a2a_menu_find_container{border-color:#444!important;background-color:#444!important}.a2a_menu:not(.a2a_thanks){background-color:#2a2a2a;border-color:#2a2a2a}.a2a_menu_find{color:#fff!important}.a2a_menu label.a2a_s_find svg{background-color:transparent!important}.a2a_menu label.a2a_s_find svg path{fill:#fff!important}.a2a_full{box-shadow:#00000066 0 0 20px 10px}.a2a_overlay{background-color:#373737eb}}@media print{.a2a_floating_style,.a2a_menu,.a2a_overlay{visibility:hidden}}@keyframes a2aFadeIn{from{opacity:0}to{opacity:1}}.a2a_starting{opacity:0}.a2a_starting.a2a_full,.a2a_starting.a2a_modal{transform:scale(.8)}@media (max-width:639px){.a2a_full{border-radius:0;top:15%;left:0;margin-left:auto;width:100%}.a2a_modal{left:0;margin-left:10px;width:calc(100% - 20px)}}@media (min-width:318px) and (max-width:437px){.a2a_full .a2a_full_services .a2a_i{width:calc(50% - 18px)}}@media (max-width:317px){.a2a_full .a2a_full_services .a2a_i{width:calc(100% - 18px)}}@media (max-height:436px){.a2a_full{bottom:40px;height:auto;top:40px}}@media (max-height:550px){.a2a_modal{top:30px}}@media (max-height:360px){.a2a_modal{top:20px}.a2a_thanks>div:first-child{margin-bottom:20px}}.a2a_menu a{color:#0166FF;text-decoration:none;font:16px sans-serif-light,HelveticaNeue-Light,"Helvetica Neue Light","Helvetica Neue",Arial,Helvetica,"Liberation Sans",sans-serif;line-height:14px;height:auto;width:auto;outline:0}.a2a_menu a.a2a_i:visited,.a2a_menu a.a2a_more{color:#0166FF}.a2a_menu a.a2a_i:active,.a2a_menu a.a2a_i:focus,.a2a_menu a.a2a_i:hover,.a2a_menu a.a2a_more:active,.a2a_menu a.a2a_more:focus,.a2a_menu a.a2a_more:hover{color:#2A2A2A;border-color:#EEE;border-style:solid;background-color:#EEE;text-decoration:none}.a2a_menu label.a2a_s_find{background-size:24px;height:24px;left:8px;pointer-events:auto;position:absolute;top:7px;width:24px}.a2a_menu label.a2a_s_find svg{background-color:#FFF}.a2a_menu label.a2a_s_find svg path{fill:#CCC}#a2a_menu_container{display:inline-block}.a2a_menu_find_container{border:1px solid #CCC;border-radius:6px;padding:2px 24px 2px 0;position:relative;text-align:left}.a2a_cols_container .a2a_col1{overflow-x:hidden;overflow-y:auto;-webkit-overflow-scrolling:touch}#a2a_modal input,#a2a_modal input[type=text],.a2a_menu input,.a2a_menu input[type=text]{display:block;background-image:none;box-shadow:none;line-height:100%;margin:0;outline:0;overflow:hidden;padding:0;-moz-box-shadow:none;-webkit-box-shadow:none;-webkit-appearance:none}#a2afeed_find_container input,#a2afeed_find_container input[type=text],#a2apage_find_container input,#a2apage_find_container input[type=text]{background-color:transparent;border:0;box-sizing:content-box;color:#2A2A2A;float:none;font:inherit;font-size:16px;height:28px;line-height:20px;left:38px;outline:0;margin:0;max-width:initial;min-height:initial;padding:2px 0;position:relative;width:99%}.a2a_clear{clear:both}.a2a_svg{background-repeat:no-repeat;display:block;overflow:hidden;height:32px;line-height:32px;padding:0;pointer-events:none;width:32px}.a2a_svg svg{background-repeat:no-repeat;background-position:50% 50%;border:none;display:block;left:0;margin:0 auto;overflow:hidden;padding:0;position:relative;top:0;width:auto;height:auto}a.a2a_i,i.a2a_i{display:block;float:left;border:1px solid #FFF;line-height:24px;padding:6px 8px;text-align:left;white-space:nowrap;overflow:hidden;text-overflow:ellipsis;width:132px}a.a2a_i span,a.a2a_more span{display:inline-block;overflow:hidden;vertical-align:top}a.a2a_i .a2a_svg{margin:0 6px 0 0}a.a2a_i .a2a_svg,a.a2a_more .a2a_svg{background-size:24px;height:24px;line-height:24px;width:24px}a.a2a_sss:hover{border-left:1px solid #CCC}a.a2a_more{border-bottom:1px solid #FFF;border-left:0;border-right:0;line-height:24px;margin:6px 0 0;padding:6px;-webkit-touch-callout:none}a.a2a_more span{height:24px;margin:0 6px 0 0}.a2a_kit .a2a_svg{background-repeat:repeat}.a2a_default_style a:empty,.a2a_flex_style a:empty,.a2a_floating_style a:empty,.a2a_overlay_style a:empty{display:none}.a2a_color_buttons a,.a2a_floating_style a{text-decoration:none}.a2a_default_style:not(.a2a_flex_style) a{float:left;line-height:16px;padding:0 2px}.a2a_default_style a:hover .a2a_svg,.a2a_floating_style a:hover .a2a_svg,.a2a_overlay_style a:hover .a2a_svg svg{opacity:.7}.a2a_overlay_style.a2a_default_style a:hover .a2a_svg{opacity:1}.a2a_default_style .a2a_count,.a2a_default_style .a2a_svg,.a2a_floating_style .a2a_svg,.a2a_menu .a2a_svg,.a2a_vertical_style .a2a_count,.a2a_vertical_style .a2a_svg{border-radius:4px}.a2a_default_style .a2a_counter img,.a2a_default_style .a2a_dd,.a2a_default_style .a2a_svg{float:left}.a2a_default_style .a2a_img_text{margin-right:4px}.a2a_default_style .a2a_divider{border-left:1px solid #000;display:inline;float:left;height:16px;line-height:16px;margin:0 5px}.a2a_kit a{cursor:pointer;transition:none}.a2a_floating_style{background-color:#fff;border-radius:6px;position:fixed;z-index:9999995}.a2a_overlay_style{z-index:2147483647}.a2a_floating_style,.a2a_overlay_style{animation:a2aFadeIn .2s ease-in;padding:4px}.a2a_vertical_style:not(.a2a_flex_style) a{clear:left;display:block;overflow:hidden;padding:4px}.a2a_floating_style.a2a_default_style{bottom:0}.a2a_floating_style.a2a_default_style a,.a2a_overlay_style.a2a_default_style a{padding:4px}.a2a_count{background-color:#fff;border:1px solid #ccc;box-sizing:border-box;color:#2a2a2a;display:block;float:left;font:12px Arial,Helvetica,sans-serif;height:16px;margin-left:4px;position:relative;text-align:center;width:50px}.a2a_count:after,.a2a_count:before{border:solid transparent;border-width:4px 4px 4px 0;content:"";height:0;left:0;line-height:0;margin:-4px 0 0 -4px;position:absolute;top:50%;width:0}.a2a_count:before{border-right-color:#ccc}.a2a_count:after{border-right-color:#fff;margin-left:-3px}.a2a_count span{animation:a2aFadeIn .14s ease-in}.a2a_vertical_style .a2a_counter img{display:block}.a2a_vertical_style .a2a_count{float:none;margin-left:0;margin-top:6px}.a2a_vertical_style .a2a_count:after,.a2a_vertical_style .a2a_count:before{border:solid transparent;border-width:0 4px 4px 4px;content:"";height:0;left:50%;line-height:0;margin:-4px 0 0 -4px;position:absolute;top:0;width:0}.a2a_vertical_style .a2a_count:before{border-bottom-color:#ccc}.a2a_vertical_style .a2a_count:after{border-bottom-color:#fff;margin-top:-3px}.a2a_color_buttons .a2a_count,.a2a_color_buttons .a2a_count:after,.a2a_color_buttons .a2a_count:before,.a2a_color_buttons.a2a_vertical_style .a2a_count:after,.a2a_color_buttons.a2a_vertical_style .a2a_count:before{background-color:transparent;border:none;color:#fff;float:none;width:auto}.a2a_color_buttons.a2a_vertical_style .a2a_count{margin-top:0}.a2a_flex_style{display:flex;align-items:flex-start;gap:0}.a2a_default_style.a2a_flex_style{left:0;right:0;width:100%}.a2a_vertical_style.a2a_flex_style{flex-direction:column;top:0;bottom:0}.a2a_flex_style a{display:flex;justify-content:center;flex:1;padding:4px}.a2a_flex_style.a2a_vertical_style a{flex-direction:column}.a2a_floating_style.a2a_color_buttons,.a2a_floating_style.a2a_flex_style{border-radius:0;padding:0}.a2a_floating_style.a2a_default_style.a2a_flex_style{bottom:0}.a2a_kit.a2a_flex_style .a2a_counter img,.a2a_kit.a2a_flex_style .a2a_dd,.a2a_kit.a2a_flex_style .a2a_svg{float:none}.a2a_nowrap{white-space:nowrap}.a2a_note{margin:0 auto;padding:9px;font-size:12px;text-align:center}.a2a_note .a2a_note_note{margin:0;color:#2A2A2A}.a2a_wide a{display:block;margin-top:3px;border-top:1px solid #EEE;text-align:center}.a2a_label{position:absolute!important;clip-path:polygon(0px 0px,0px 0px,0px 0px);-webkit-clip-path:polygon(0px 0px,0px 0px,0px 0px);overflow:hidden;height:1px;width:1px}.a2a_kit,.a2a_menu,.a2a_modal,.a2a_overlay{-ms-touch-action:manipulation;touch-action:manipulation;outline:0}.a2a_dd{-webkit-user-drag:none}.a2a_dd img{border:0}.a2a_button_facebook_like iframe{max-width:none}</style><style>stream {
  position: relative;
  display: block;
}
</style><link href="https://crossmark-cdn.crossref.org/widget/v2.0/style.css" type="text/css" rel="stylesheet"></head>
<body class="pb-ui">
    <div class="skip">
        <a href="#page-body-id" id="skipNavigationLink" title="Skip to Main Content" tabindex="0" class="skipToContent">Skip to Main Content</a>
        <a href="#mega-menu-nav" id="skipToMainMenu" title="Skip to Main Menu" tabindex="0" class="skipToContent">Skip to Main Menu</a>
    </div>



    <div id="pb-page-content" data-ng-non-bindable="">
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="ux3-layout-widget" data-widget-id="832ad3f3-97ac-45ab-9af1-90c6f7bff54a" class="ux3">
        
    

        


    

            

            
                <div class="cell fonts-loaded">
            

            <div data-widget-id="832ad3f3-97ac-45ab-9af1-90c6f7bff54a" data-widget-name="ux3-layout-widget"><div>
    

        


    

            

            

            <div id="a1f1ba25-79af-4194-aaa9-66206104a800" class="widget loginPopup none widget-none widget-compact-all"><div class="wrapped"><div class="widget-body body body-none body-compact-all"><div role="dialog" aria-modal="true" aria-label="Login" class="popup login-popup hidden"><div class="content"><a href="#" aria-label="Close login popup" title="Close login popup" role="button" class="close"><i class="icon-close_thin"></i></a><h2 class="login-title">Login to your account</h2><div class="login-form"><p tabindex="-1" class="message error"></p><form action="/action/doLogin?redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2900753-4" method="post"><input type="hidden" name="redirectUri" value="https://www.cell.com/cell/fulltext/S0092-8674(21)00753-4"><input type="hidden" name="referer" value="/cell/fulltext/S0092-8674(21)00753-4"><input type="hidden" name="ecommRedirect"><input type="hidden" name="code" value="cell-site"><input type="hidden" name="UX3LoginWidget" value="true"><input type="hidden" name="isPopup" value="true"><div class="input-group login-input--wrapper"><div class="label"><label for="login">Email/Username</label></div><input id="login" type="text" name="login" value="" size="15" class="login"><div role="alert" class="error-hint js--hidden"><div class="error-hint__text"><div class="error-text">Your email address is a required field.</div><div class="email-example">E.g., j.smith@mail.com</div></div></div></div><div class="input-group login-input--wrapper"><div class="label"><label for="password">Password</label></div><input id="password" type="password" name="password" value="" autocomplete="off" class="password"><a href="#" class="login__field__action js-show-password">Show</a><div role="alert" class="error-hint js--hidden"><span class="error-hint__text">Your password is a required field.</span></div><div class="actions"><a href="https://www.cell.com/action/requestResetPassword" class="link show-request-reset-password">Forgot password?</a></div></div><div class="remember"><input type="checkbox" name="savePassword" id="a1f1ba25-79af-4194-aaa9-66206104a800-remember" value="1" checked="checked"><label for="a1f1ba25-79af-4194-aaa9-66206104a800-remember">Remember me</label></div><div class="submit"><input type="submit" name="submit" value="Log in" class="button submit primary"></div></form></div><div class="create-account"><div>Don’t have an account?</div><a href="/action/registration?redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2900753-4" class="show-registration">Create a Free Account</a></div></div></div></div></div></div>

            

        

    

        


    

            

            

            <div role="dialog" aria-modal="true" aria-label="Reset password" class="popup top-drawer request-reset-password-drawer hidden"><div class="content"><a href="#" aria-label="Close reset password popup" title="Close reset password popup" role="button" class="close"><i class="icon-close_thin"></i></a><div class="request-reset"><div class="form"><p class="sub">If you don't remember your password, you can reset it by entering your email address and clicking the Reset Password button. You will then receive an email that contains a secure link for resetting your password</p><div tabindex="-1" class="message error"></div><form action="/action/requestResetPassword" method="post" class="request-reset-password-form"><input type="hidden" name="platform"><input type="hidden" name="requestResetPassword" value="true"><input type="hidden" name="codeSite" value="cell-site"><div class="input-group"><div class="label"><label for="reset-password-email">Email*</label></div><input id="reset-password-email" type="text" name="email" value="" size="15" class="email"></div><div class="submit" disabled="disabled"><input type="submit" name="submit" value="Submit" disabled="disabled" class="button primary submit"></div></form></div><div class="success-template hidden"><p>If the address matches a valid account an email will be sent to __email__ with instructions for resetting your password</p></div></div><div class="request-reset-cancel"><a href="#" aria-label="Close reset password popup" title="Close reset password popup" class="cancel">Cancel</a></div></div></div>

            

        

    

        


    

            

            
                <div class="ux3-ads">
            

            <div class="pb-ad">
            <!-- Do Not Modify without contacting ELS-HSOnlineCampaign@elsevier.com -->
<!-- To be kept in sync with Global Commercial Publishing Operations-->
<!-- Start: GPT Async -->
<div id="gpt-Variables" data-adfile="CELLJOURNAL">
	<input type="hidden" id="gptSite" value="cell">
	<input type="hidden" id="gptLoggedIn" value="no">
	<input type="hidden" id="gptAccessType" value="ae:ANON_GUEST|ae:ANON_GUEST">
	<input type="hidden" id="gptDefaultSite" value="cell">
</div>
<div id="gpt-Interscroller" class="gpt-InScrollBox" style="display:none;">
	<div class="gpt-InScrollLabel">ADVERTISEMENT</div>
	<div class="gpt-InScrollContainer">
		<div class="gpt-InScrollWrapper">
			<div class="gpt-InScrollInner">
				<div class="gpt-adinterscroller" data-adsize="interscroller"></div>
			</div>
		</div>
	</div>
	<div class="gpt-InScrollLabel">SCROLL TO CONTINUE WITH CONTENT</div>
</div>
<div id="gpt-DebugDiv" class="gpt-DebugDiv" style="display:none;">
	<div style="width: 100%; display: table;">
		<div style="display: table-row">
			<div style="width: 200px; display: table-cell;">
				<button class="gpt-DebugBtn" onclick="gpt_ShowConsole()">Open GPT Console</button><br>
				<button id="gpt_GrapeShotDebugBtn" class="gpt-DebugBtn" onclick="gpt_GrapeShotDebug();">Open Oracle Keywords</button><br>
				<br>
				<button class="gpt-DebugBtn" onclick="gpt_LoadDebugValues()">Refresh Values</button>
			</div>
			<div style="display: table-cell;">
				<table class="gpt-DebugTable">
					<thead>
						<tr>
							<th>Property</th>
							<th>Value</th>
						</tr>
					</thead>
					<tbody>
						<tr><td class="gpt-DebugTitle">Status</td><td><div class="gptStatus_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Version</td><td><div class="gptVersion_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Ad File</td><td><div class="gptAdFile_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Disable Ads Flag</td><td><div class="gptDisableAds_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Environment</td><td><div class="gptTestENV_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Moat Init</td><td><div class="gptMoatInit_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Moat Ready</td><td><div class="gptMoatReady_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual Ready</td><td><div class="gptContextualReady_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual URL</td><td><div class="gptContextualURL_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual Initial Segments</td><td><div class="gptContextualInit_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual Used Segments</td><td><div class="gptContextualResult_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">AdUnit</td><td><div class="gptSite_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">SubAdUnit</td><td><div class="gptPage_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Custom Targeting</td><td><div class="gptCustomTargeting_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Ad Events</td><td><div class="gptEvent_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Invalid Ad Sizes</td><td><div class="gptInvalidAdSize_Debug"></div></td></tr>
					</tbody>
				</table>
			</div>
		</div>
	</div>
</div>
<style type="text/css">
	.gpt-DebugBtn {
		display: inline-block;
		text-align: center;
		text-decoration: none;
		margin: 2px 0;
		border: solid 1px #122b40;
		border-radius: 0.4em;
		padding: 0.5em 1em;
		color: #ffffff;
		background-color: #204d74;
		font-weight: normal;
		font-size: 12px;
	}

	.gpt-DebugDiv {
		margin: 5px;
	}

	.gpt-DebugTitle {
		font-weight: bold;
	}

	.gpt-DebugTable {
		border: 1px solid #dddddd;
		border-collapse: collapse;
		margin: 5px;
		font-size: 0.9em;
		font-family: sans-serif;
		min-width: 400px;
		margin-bottom: 25px;
	}

		.gpt-DebugTable td:first-child {
			padding: 5px;
		}

		.gpt-DebugTable thead tr th {
			background-color: #d9edf7;
			color: #ffffff;
			text-align: left;
			padding: 5px;
			border-bottom: 2px solid #204d74;
		}

		.gpt-DebugTable th,
		.gpt-DebugTable td {
			padding: 5px;
		}

		.gpt-DebugTable tbody tr {
			border-bottom: 1px solid #dddddd;
		}

			.gpt-DebugTable tbody tr:nth-of-type(even) {
				background-color: #f3f3f3;
			}

			.gpt-DebugTable tbody tr:last-of-type {
				border-bottom: 2px solid #204d74;
			}

	.gpt-InScrollLabel {
		background-color: rgb(0, 0, 0);
		color: rgb(255, 255, 255);
		position: relative;
		max-width: 100vw;
		left: 50%;
		right: 50%;
		margin-left: -50vw;
		margin-right: -50vw;
		padding: 0;
		text-align: center;
		width: 100vw !important;
		font-size: 10px;
	}

	.gpt-InScrollBox {
		height: 100vh !important;
		width: 100vw;
		cursor: pointer;
		max-width: 100vw;
		position: relative;
		left: 50%;
		right: 50%;
		margin-left: -50vw;
		margin-right: -50vw;
		margin-bottom: 50px;
		padding-top: 5px;
		padding-bottom: 5px;
		z-index: 10001;
		background-color: white;
	}

		.gpt-InScrollBox .gpt-InScrollContainer {
			height: 95vh !important;
			position: relative;
		}

		.gpt-InScrollBox .gpt-InScrollWrapper {
			position: absolute !important;
			top: 0 !important;
			left: 0;
			width: 100%;
			height: 95vh !important;
			border: 0 !important;
			margin: 0 !important;
			padding: 0 !important;
			clip: rect(0, auto, auto, 0) !important;
			-webkit-clip-path: polygon(0px 0px, 100% 0px, 100% 100%, 0px 100%) !important;
			clip-path: polygon(0px 0px, 100% 0px, 100% 100%, 0px 100%) !important;
		}

		.gpt-InScrollBox .gpt-InScrollInner {
			position: fixed !important;
			top: 0 !important;
			left: 0 !important;
			bottom: 0;
			margin: auto;
			width: 100vw;
			height: 95vh;
			-webkit-transform: translateZ(0) !important;
			display: -webkit-box;
			display: -ms-flexbox;
			display: flex;
			-webkit-box-orient: vertical;
			-webkit-box-direction: normal;
			-ms-flex-direction: column;
			flex-direction: column;
			-webkit-box-align: center;
			-ms-flex-align: center;
			align-items: center;
			-webkit-box-pack: center;
			-ms-flex-pack: center;
			justify-content: center;
			text-align: center;
		}
</style>
<script class="optanon-category-4" type="text/plain">
	//--------------------------------------------------------------------------
	var gpt_Version = '04072023';
	var gpt_AdFile = $('#gpt-Variables').data('adfile');
	var gpt_MaxMOATInitAttempts = 11;
	var gpt_MaxMOATReadyAttempts = 11;
	var gpt_MaxContextualReadyAttempts = 21;
	var gpt_MOATInitAttempts = 0;
	var gpt_MOATReadyAttempts = 0;
	var gpt_ContextualReadyAttempts = 0;
	var gpt_IsProd = true;
	var gpt_StartTime;
	var gs_channels = 'DEFAULT';
	var gpt_ContextualReady = false;
	var gpt_ContextualURL = 'Not enabled outside of Production';
	var gpt_DisableAdsFlagFound = false;

	var gptEvent_Debug = '';
	var gptCustomTargeting_Debug = '';
	var gptSite_Debug = "";
	var gptPage_Debug = "";
	var gptContextualEnabled = false;
	var gptContextualResult_Debug = '';
	var gptContextualInit_Debug = '';
	var gptContextualURL_Debug = '';
	var gptContextualReady_Debug = '';
	var gptMoatReady_Debug = '';
	var gptMoatInit_Debug = '';
	var gptTestENV_Debug = '';
	var gptDisableAds_Debug = '';
	var gptStatus_Debug = 'GPT Still Loading - Refresh Values';
	var gptInvalidAdSize_Debug = '';
	//--------------------------------------------------------------------------
	$(document).ready(function () {
		//Check to see if we are on a development env
		gpt_CheckEnvironment();

		if (!gpt_DisableAdsFlagFound) {
			gpt_StartTime = new Date();
			if (gpt_AdFile !== "PU") {
				//Once the dom has loaded, begin the process to init the ads.
				gpt_InitScripts();
				gpt_CheckForMOATInit();
			}
		}
	});
</script>
<script type="text/javascript">
	function gpt_CheckEnvironment() {
		if ($('#gptDisableAds').length > 0) {
			gptDisableAds_Debug = 'Flag Detected - Ads Disabled';
			gpt_DisableAdsFlagFound = true;
		}
		else {
			gptDisableAds_Debug = 'Ads Enabled';
			gpt_DisableAdsFlagFound = false;
		}
		switch (gpt_AdFile) {
			case 'PU':
				gptContextualEnabled = true;
				if ($("#Environment").val() !== "Prod") {
					gpt_IsProd = false;
					gptTestENV_Debug = 'Test Environment Detected';
				}
				else {
					gptTestENV_Debug = 'Production Environment Detected';
				}
				break;
			case 'JBS':
			case 'LANCET':
				gptContextualEnabled = true;
				if ($(".gpt-ad").length > 0) {
					//Other ads exist, allow the Prestitial
					$(document.body).append("<div class='gpt-ad' data-adsize='prestitial'></div>");
				}
				if (window.location.hostname.toLowerCase().indexOf('.literatumonline.') >= 0) {
					gpt_IsProd = false;
					gptTestENV_Debug = 'Test Environment Detected';
				}
				else {
					gptTestENV_Debug = 'Production Environment Detected';
				}
				break;
			default:
				//Cell
				gptContextualEnabled = false;
				if (window.location.hostname.toLowerCase().indexOf('.literatumonline.') >= 0) {
					gpt_IsProd = false;
					gptTestENV_Debug = 'Test Environment Detected';
				}
				else {
					gptTestENV_Debug = 'Production Environment Detected';
				}
				break;
		}
	}
	//--------------------------------------------------------------------------
    function gpt_GetNonCMEArticle() {
        var url;
        $('.articleCitation').each(function (i, obj) {
            if ($(obj).find('.toc__item__detials').length > 0) {
                var itemDetails = $(obj).first().find('.toc__item__detials');
                var cmeFound = 0;
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='CME']").length; }
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='(CME)']").length; }
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='Education']").length; }
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='education']").length; }
                if (cmeFound == 0) {
                    var itemLink = $(itemDetails).first().find('.toc__item__title');
                    var linkCount = $(itemLink).first().find("a:contains('CME')").length;
                    if (linkCount == 0) {
                        url = $(itemLink).first().find('a').attr('href');
                        return false;   //Found a URL, break the loop
                    }
                }
            }
        });
        return url;
    }
	//--------------------------------------------------------------------------
	function gpt_GetElapsedTime() {
		var endTime = new Date();
		var timeDiff = endTime - gpt_StartTime; //in ms
		// strip the ms
		timeDiff /= 1000;

		// get seconds
		return seconds = Math.round(timeDiff) + " seconds";
	}
	//--------------------------------------------------------------------------
	function gpt_CheckSSC() {
		//Check for ScreenShot Client
		var gpt_SSC = gpt_GetQuerystring('gpt_ssc', '');
		if (gpt_SSC == '1') {
			//it's the ScreenShotClient, disable MOAT IVT Detection
			$('#gpt-Variables').append('<input type="hidden" id="gptm_data">');
			$('#gptm_data').val(0);
			gpt_Target("m_data");
		}
	}
	//--------------------------------------------------------------------------
	function gpt_LocalSS(adTypeId) {
		window.scrollTo(0, 0);
		gpt_Sleep(1000);
		gpt_ScrollToAd(adTypeId);
		gpt_PreTearsheet(adTypeId);
	}
	//--------------------------------------------------------------------------
	function gpt_InitScripts() {
		if (gpt_AdFile == "PU") {
			if (gpt_IsProd) {
				gpt_ContextualURL = gpt_CheckForNULL($('#canonicalurl').val());
				if (gpt_ContextualURL === null) {
					gpt_ContextualURL = window.location.href.split('?')[0].split('#')[0];
				}
				else {
					gpt_ContextualURL = window.location.origin + gpt_ContextualURL;
				}
				var useSSL = 'https:' == document.location.protocol;
				var gs = document.createElement('script');
				gs.async = true;
				gs.type = 'text/javascript';
				gs.src = (useSSL ? 'https:' : 'http:') + '//elsevier.gscontxt.net/multizone/channels.cgi?url=' + encodeURIComponent(gpt_ContextualURL);
				gs.onload = function () {
					gpt_ContextualReady = true;
				};
				document.head.appendChild(gs);
			}
		}
		else {
			//JBS-Cell-Lancet
			var useSSL = 'https:' == document.location.protocol;

			var gads = document.createElement('script');
			gads.async = true;
			gads.type = 'text/javascript';
			gads.src = (useSSL ? 'https:' : 'http:') + '//securepubads.g.doubleclick.net/tag/js/gpt.js';
			document.head.appendChild(gads);

			if (gpt_IsProd && gptContextualEnabled) {
				gpt_ContextualURL = window.location.href.split('?')[0].split('#')[0];
				var gs = document.createElement('script');
				gs.async = true;
				gs.type = 'text/javascript';
				gs.src = (useSSL ? 'https:' : 'http:') + '//elsevier.gscontxt.net/multizone/channels.cgi?url=' + encodeURIComponent(gpt_ContextualURL);
				gs.onload = function () {
					gpt_ContextualReady = true;
				};
				document.head.appendChild(gs);
			}

			var elsevierMoatHeader = document.createElement('link');
			elsevierMoatHeader.rel = 'preload';
			elsevierMoatHeader.as = 'script';
			elsevierMoatHeader.src = (useSSL ? 'https:' : 'http:') + '//z.moatads.com/elsevierheader150204004183/moatheader.js';
			document.head.appendChild(elsevierMoatHeader);

			var elsevierMoatAds = document.createElement('link');
			elsevierMoatAds.rel = 'preconnect';
			elsevierMoatAds.src = (useSSL ? 'https:' : 'http:') + '//mb.moatads.com';
			document.head.appendChild(elsevierMoatAds);

			var moatHeader = document.createElement('script');
			moatHeader.async = true; gads.type = 'text/javascript';
			moatHeader.src = (useSSL ? 'https:' : 'http:') + '//z.moatads.com/elsevierheader150204004183/moatheader.js';
			document.head.appendChild(moatHeader);
		}
	}
	//--------------------------------------------------------------------------
	async function gpt_CheckForMOATInit() {
		//Check every 1/10 of a second until MOAT is Loaded or 1 second timeout
		if (gpt_MOATInitAttempts < gpt_MaxMOATInitAttempts) {
			gpt_MOATInitAttempts += 1;
			if (typeof window.moatPrebidApi === "object") {
				//MOAT LOADED
				gptMoatInit_Debug = '0.0' + gpt_MOATInitAttempts + ' seconds';
				gpt_CheckForMOATReady();
			}
			else {
				setTimeout(() => { gpt_CheckForMOATInit(); }, 100);
			}
		}
		else {
			//TIMEOUT: MOAT Not LOADED
			gptMoatInit_Debug = 'Timeout Expired';
			var googletag = googletag || {};
			googletag.cmd = googletag.cmd || [];
			gpt_CheckForContextualReady();
		}
	}
	//--------------------------------------------------------------------------
	function gpt_CheckForMOATReady() {
		//Check every 1/10 of a second until MOAT is Ready or 1 second timeout
		if (gpt_MOATReadyAttempts < gpt_MaxMOATReadyAttempts) {
			gpt_MOATReadyAttempts += 1;
			if (window.moatPrebidApi.safetyDataAvailable()) {
				//MOAT Ready
				gptMoatReady_Debug = '0.0' + gpt_MOATReadyAttempts + " seconds";
				gpt_CheckForContextualReady();
			}
			else {
				setTimeout(() => { gpt_CheckForMOATReady(); }, 100);
			}
		}
		else {
			//TIMEOUT: MOAT Not Ready
			gptMoatReady_Debug = 'Timeout Expired';
			gpt_CheckForContextualReady();
		}
	}
	//--------------------------------------------------------------------------
	function gpt_CheckForContextualReady() {
		//Check every 1/10 of a second until Contextual is Ready or 2 second timeout
		if (gpt_IsProd && gptContextualEnabled) {
			//Enabled
			if (gpt_ContextualReadyAttempts < gpt_MaxContextualReadyAttempts) {
				gpt_ContextualReadyAttempts += 1;
				if (gpt_ContextualReady) {
					//Contextual Ready
					gptContextualURL_Debug = gpt_ContextualURL;
					gptContextualReady_Debug = '0.0' + gpt_ContextualReadyAttempts + " seconds";
					gptContextualInit_Debug = gs_channels;
					gpt_InitAds();
				}
				else {
					setTimeout(() => { gpt_CheckForContextualReady(); }, 100);
				}
			}
			else {
				//LOG TIMEOUT error
				//REMOVE FOR NOW (SHOULD LOG TO A DIFF LOCATION)
				//TIMEOUT: Contextual Not Ready
				$('#gpt_GrapeShotDebugBtn').remove();
				gptContextualURL_Debug = gpt_ContextualURL;
				gptContextualReady_Debug = 'Timeout Expired';
				gptContextualInit_Debug = "";
				gpt_InitAds();
			}
		}
		else {
			//Not Enabled
			$('#gpt_GrapeShotDebugBtn').remove();
			switch (gpt_AdFile) {
				case 'PU':
				case 'LANCET':
				case 'JBS':
					gptContextualURL_Debug = 'Not enabled outside of Production';
					gptContextualReady_Debug = 'Not enabled outside of Production';
					gptContextualInit_Debug = 'Not enabled outside of Production';
					break;
				case "CELLJOURNAL":
				case "CELLBUCKET":
					gptContextualReady_Debug = 'Not enabled on Cell';
					gptContextualInit_Debug = 'Not enabled on Cell';
					gptContextualURL_Debug = 'Not enabled on Cell';
					break;
			}
			gpt_InitAds();
		}
	}
	//--------------------------------------------------------------------------
	function GPT_GetPageByURL() {
		var host = window.location.host.toLocaleLowerCase()
		var localpath = window.location.pathname.toLocaleLowerCase();

		///Check for Cell.com
		if (host.indexOf('cell.com') >= 0) {
			return null;
		}

		///Check for Cell.com (marlin-stag)
		if (host.indexOf('www-cell-com') >= 0) {
			return null;
		}

		///Check for Lancet
		if (host.indexOf('thelancet') >= 0) {
			return null;
		}

		if (localpath.indexOf('/article/') >= 0) {
			return 'article.fulltext';
		}

		if (localpath.indexOf('/action/dosearch') >= 0) {
			return 'search.searchResults';
		}

		if (localpath.indexOf('/content/') >= 0) {
			return 'periodical.editorial';
		}

		return 'periodical.home';
	}
	//--------------------------------------------------------------------------
	function gpt_CheckForNULL(x, valueifnull) {
		if (typeof valueifnull == "undefined") {
			valueifnull = null;
		}

		if (typeof x == 'undefined') {
			return valueifnull;
		}
		if (x == null) {
			return valueifnull;
		}
		if (x == "null") {
			return valueifnull;
		}
		if ($.trim(x) == '') {
			return valueifnull;
		}

		return x;
	}
	//--------------------------------------------------------------------------
	function gpt_GetAdditionalTargeting() {
		//adds page level targeting of custom variables
		//the id (minus gpt) and the value will be used as a key-value pair
		//eg: <input type="hidden" id="gptissuepii" value="x" class="gpt-Target">
		//eg: target issuepii=x
		$(".gpt-Target").each(function (index) {
			var gptTargetId = $(this).attr('id');
			gptTargetId = gptTargetId.replace('gpt', '').toLowerCase();
			if (gptTargetId !== '') {
				gpt_Target(gptTargetId);
			}
		})
	}
	//--------------------------------------------------------------------------
	function gpt_GetQuerystring(key, default_) {
		//function to read querystring variables
		if (default_ === null) default_ = '';
		key = key.replace(/[\[]/, "\\\[").replace(/[\]]/, "\\\]");
		var regex = new RegExp("[\\?&]" + key + "=([^&#]*)");
		var qs = regex.exec(window.location.href);
		if (qs === null)
			return default_;
		else
			return qs[1];
	}
	//--------------------------------------------------------------------------
	function gpt_GetDFPTestId() {
		//Get any request for a test id from querystring and set custom targeting parameter
		var gptDFPTestId = gpt_GetQuerystring('dfptestid', '');
		if (gptDFPTestId !== '') {
			$('#gpt-Variables').append('<input type="hidden" id="gptDFPTestId">');
			$('#gptDFPTestId').val(gptDFPTestId);
			gpt_Target("DFPTestId");
		}
	}
	//--------------------------------------------------------------------------
	function gpt_Target(targetName) {
		//Set page level target if value is found
		if ($('#gpt' + targetName).length > 0) {
			var targetValue = $('#gpt' + targetName).val();

			if (targetValue !== '') {
				targetValue = targetValue.replace(/[^-a-z0-9]/ig, '');	//Only Allow a-z, 0-9
				googletag.pubads().setTargeting(targetName.toLowerCase(), targetValue.toLowerCase());
				gptCustomTargeting_Debug = gptCustomTargeting_Debug + targetName.toLowerCase() + ' = ' + targetValue.toLowerCase() + '<br>';
			}
		}
	}
	//--------------------------------------------------------------------------
	function gpt_TargetAccessType() {
		var currentAccessType = $('#gptAccessType').val();
		if (typeof currentAccessType == "undefined") {
			currentAccessType = "";
		}
		if (currentAccessType.indexOf('ANON_IP') >= 0) {
			googletag.pubads().setTargeting("loggedin", "yes");
		}
		else {
			gpt_Target("LoggedIn");
		}
	}
	//--------------------------------------------------------------------------
	async function gpt_PreTearsheet(adTypeId) {
		if (typeof adTypeId === "undefined") {
			adTypeId = 0;
		}
		//Remove Elsevier Cookie Policy Notice
		if ($("#onetrust-consent-sdk").length > 0) {
			$("#onetrust-consent-sdk").remove();
		}

        //Mobile Nav
        $('#skip').remove();
        $('skipNavigationLink').remove();

		//JBS Modals
		if ($('.leadinModal').length > 0) {
			$('.leadinModal').remove();
		}
		if ($('.leadinModal').length > 0) {
			$('.leadinModal').remove();
		}
		if ($('.usabilla__overlay').length > 0) {
			$('.usabilla__overlay').remove();
		}

		//Pendo
		if ($('._pendo-step-container').length > 0) {
			$('._pendo-step-container').remove();
		}
		if ($('._pendo-guide-tt_').length > 0) {
			$('._pendo-guide-tt_').remove();
		}
		if ($('._pendo-image').length > 0) {
			$('._pendo-image').remove();
		}
		if ($('._pendo-badge-image').length > 0) {
			$('._pendo-badge-image').remove();
		}

		switch (gpt_AdFile) {
			case "CELLJOURNAL":
			case "CELLBUCKET":
				//Mobile Bug Fix on Cell
				if (parseInt(adTypeId) == 5) {
					$('.widget-horizontalAd').css('position', 'static');
					$('.body-horizontalAd').css('display', 'block');
				}
				break;
			case "JBS":
			case "LANCET":
				objLeaderboard = ".container-leaderboard";
				offset = 53;
				switch (parseInt(adTypeId)) {
					case 3: //MPU
					case 4: //SkyScraper
						//BUGFIX: Show Sidebar ad for JBS
						//skyscraper & mpu
						if ($('.widget-verticalAd').length > 0) {
							$('.widget-verticalAd').css('display', 'block');
						}
						$('#backgroundLayer').css('display', 'none');   //Hide any Prestitial
						break;
					case 6:     //Prestitial
						$('#backgroundLayer').css('display', 'block');
						break;
					default:
						$('#backgroundLayer').css('display', 'none');   //Hide any Prestitial
						break;
				}
				break;
		}
	}
	//--------------------------------------------------------------------------
	function gpt_Sleep(ms) {
		return new Promise(resolve => setTimeout(resolve, ms));
	}
	//--------------------------------------------------------------------------
	async function gpt_ScrollToAd(adTypeId) {
		if (typeof adTypeId === "undefined") {
			adTypeId = 0;
		}
		var adId = "";
		var offset = 0;
		var objLeaderboard = "";
		var gpt_ScrollToAd = true;

		switch (gpt_AdFile) {
			case "PU": //PracticeUpdate
				var hideOtherAds = true;
				$('#modalPrestitial').modal('hide');
				$('.jumbotron-welcome').hide();
				objLeaderboard = ".container-leaderboard";
				offset = 53;
				switch (parseInt(adTypeId)) {
					case 2:
						//LeaderBoard, no Scroll Needed
						gpt_ScrollToAd = false;
						adId = "adSlot1";
						break;
					case 3:
						//MPU
						adId = "adSlot2";
						break;
					case 4:
						//SkyScraper
						adId = "adSlot3";
						$('.most-read').hide();
						break;
					case 5:
						//Mobile, no Scroll Needed
						gpt_ScrollToAd = false;
						adId = "adSlot1";
						break;
					case 6:
						//Interstitial
						gpt_ScrollToAd = false;
						adId = "adSlot15";
						await gpt_Sleep(1000);
						$(objLeaderboard).hide();
						$('#modalPrestitial').unbind();
						$('#modalPrestitial').modal();
						break;
					case 7:
						//Native Ad
						break;
					case 10:
						//Master Companion
						hideOtherAds = false;
						gpt_ScrollToAd = false;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						adId = "adSlot1";
						break;
				}
				if (hideOtherAds) {
					//Hide all other ads
					$('.ad-container').each(function (i, obj) {
						if (this.id !== adId) {
							var tempParent = this.id;
							tempParent = tempParent.replace("adSlot", "adParent");
							$("#" + tempParent).addClass('hide');
						}
					});
				}
				break;
			case "JBS":
				objLeaderboard = ".js__adToHide";
				offset = 80;
				switch (parseInt(adTypeId)) {
					case 2:
						//Leaderboard
						gpt_ScrollToAd = false;
						$(".widget-verticalAd").hide();
						break;
					case 5:
						//Mobile
						//Scroll to Top
						gpt_ScrollToAd = false;
						break;
					case 3: //MPU
					case 4: //SkyScraper
						$(objLeaderboard).hide();
						adId = $('.gpt-ad-defined[data-adsize="sidebar"').attr('id');
						break;
					case 10:
						//MasterCompanion
						gpt_ScrollToAd = false;
						break;
					case 13:	    //Phone - Portrait
					case 14:        //Phone - Landscape
						$(objLeaderboard).hide();
						offset = 60;
						adId = 'gpt-Interscroller';
						gpt_ScrollToAd = true;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						break;
				}
				break;
			case "CELLJOURNAL":
			case "CELLBUCKET":
				objLeaderboard = ".js__adToHide";
				offset = 260;
				switch (parseInt(adTypeId)) {
					case 2:
						//Leaderboard
						gpt_ScrollToAd = false;
						$('.body-verticalAd').hide();
						break;
					case 5:
						//Mobile
						//Scroll to Top
						gpt_ScrollToAd = false;
						break;
					case 3: //MPU
					case 4: //SkyScraper
						$(objLeaderboard).hide();
						adId = $('.gpt-ad-defined[data-adsize="boombox"').attr('id');
						break;
					case 10:
						//MasterCompanion
						gpt_ScrollToAd = false;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						break;
				}
				//Hide the second ad if found
				var tCount = 0;
				$(".widget.literatumAd.none.GPT-ad-placeholder.widget-horizontalAd.widget-compact-all").each(function (index) {
					tCount += 1;
					if (tCount !== 1) {
						$(this).hide();
					}
				});
				break;
			case "LANCET":
				objLeaderboard = ".js__adToHide";
				offset = 80;
				switch (parseInt(adTypeId)) {
					case 2:
						//Leaderboard
						gpt_ScrollToAd = false;
						$('.body-verticalAd').hide();
						break;
					case 5:
						//Mobile
						//Scroll to Top
						gpt_ScrollToAd = false;
						break;
					case 3: //MPU
					case 4: //SkyScraper
						$(objLeaderboard).hide();
						adId = $('.gpt-ad-defined[data-adsize="skyscraper"').attr('id');
						break;
					case 10:
						//MasterCompanion
						gpt_ScrollToAd = false;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						break;
				}
				break;
		}

		//Scroll to Top
		var gpt_container = $("html,body");
		gpt_container.animate({ scrollTop: 0, scrollLeft: 0 });

		if (gpt_ScrollToAd) {
			//If Adexists, scroll to it
			var gpt_scrollTo = $('#' + adId);
			if (gpt_scrollTo.length > 0) {
				//Scroll to the Ad
				var gpt_topPosition = gpt_scrollTo.offset().top - gpt_container.offset().top + gpt_container.scrollTop();
				if (gpt_topPosition > 0) {
					gpt_container.animate({ scrollTop: gpt_topPosition, scrollLeft: 0 });
					if (gpt_topPosition - offset > 0 && offset > 0) {
						gpt_container.animate({ scrollTop: gpt_topPosition - offset, scrollLeft: 0 });
					}
				}
			}
		}
	}
	//--------------------------------------------------------------------------
	async function gpt_CheckForCookiePolicy() {
		var returnVal = false;
		if ($('#onetrust-accept-btn-handler').length > 0) {
			$('#onetrust-accept-btn-handler').click();
			returnVal = true;
		}
		return returnVal;
	}
	//--------------------------------------------------------------------------
	function gpt_InitAds() {
		var gpt_ResponsiveHomepage = false;

		switch (gpt_AdFile) {
			case "JBS":
			case "CELLJOURNAL":
			case "CELLBUCKET":
				//Responsive Check
				if ($('#hdrHomeBrand').length > 0) {
					gpt_ResponsiveHomepage = true;
				}
				break;
		}

		//InterScroller Check
		if (gpt_AdFile === "JBS") {
			var gpt_InterScrollerEnabled = false;
			var gpt_InterScrollerInsertObj;

			if ($("#articleHeader").length > 0) {
				//It's an article Page
				if ($('.article__sections > section').length > 0) {
					gpt_InterScrollerEnabled = true;
					gpt_InterScrollerInsertObj = $(".article__sections > section");
				}
			}
			else {
				if ($('.toc__section').length > 0) {
					//It's a TOC Page
					gpt_InterScrollerEnabled = true;
					gpt_InterScrollerInsertObj = $(".toc__section").first();
				}
			}

			//Add the Interscroller
			if (gpt_InterScrollerEnabled) {
				gpt_InterScrollerInsertObj.first().append($("#gpt-Interscroller").clone());
				$("#gpt-Interscroller").remove();
				$(".gpt-adinterscroller").addClass("gpt-ad");
			}
		}

		googletag.cmd.push(function () {
			var gptBoomBoxSizeMapping;
			var gptFluidMapping;
			var gptHomePageSidebarMapping;
			var gptCellHomePageSidebarMapping;
			var gptInterscrollerMapping;
			var gptLeaderboardSizeMapping;
			var gptMobileSizeMapping;
			var gptPrestitialMapping;
			var gptRectangleShortSizeMapping;
			var gptRectangleSizeMapping;
			var gptSideBarSizeMapping;
			var gptTallSizeMapping;
			var gptWideSizeMapping;
			var gptWideRectangleSizeMapping;

			//***********************************************
			//Start Define Size mapping based on client Viewport
			//***********************************************
			if (gpt_AdFile === 'PU') {
				gptTallSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[300, 600], [160, 600], [120, 600]]).
					addSize([0, 0], []).
					build();

				gptRectangleSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[300, 250]]).
					addSize([0, 0], []).
					build();

				gptRectangleShortSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[300, 250], [320, 50], [300, 50]]).
					addSize([0, 0], [[300, 250], [320, 50], [300, 50]]).
					build();

				gptWideSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[728, 90]]).
					addSize([760, 0], [[728, 90], [320, 50], [300, 50]]).
					addSize([0, 0], [[320, 50], [300, 50]]).
					build();

				gptWideRectangleSizeMapping = googletag.sizeMapping().
					addSize([992, 0], []).
					addSize([760, 0], [[320, 50], [300, 50], [300, 250]]).
					addSize([0, 0], [[320, 50], [300, 50], [300, 250]]).
					build();

				gptMobileSizeMapping = googletag.sizeMapping().
					addSize([0, 0], [[320, 50], [300, 50]]).
					build();

				gptFluidMapping = googletag.sizeMapping().
					addSize([0, 0], ['fluid']).
					build();

				gptPrestitialMapping = googletag.sizeMapping().
					addSize([760, 0], [[1, 1]]).
					build();
			}
			else {
				//JBS-CELL-LANCET
				gptLeaderboardSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[728, 90]]).
					addSize([768, 0], [[728, 90]]).
					addSize([0, 0], [[320, 50], [300, 50]]).
					build();

				gptHomePageSidebarMapping = googletag.sizeMapping().
					addSize([2090, 795], [[300, 600], [300, 250], [160, 600], [120, 600]]).
					addSize([1910, 795], [[160, 600], [120, 600]]).
                    addSize([1740, 795], [[120, 600]]).
					addSize([0, 0], []).
					build();

				gptCellHomePageSidebarMapping = googletag.sizeMapping().
                    addSize([2180, 795], [[300, 600], [160, 600], [120, 600], [300, 250], [336, 280]]).
					addSize([2180, 445], [[336, 280]]).
                    addSize([2090, 795], [[300, 600], [160, 600], [120, 600], [300, 250]]).
                    addSize([2090, 445], [[300, 250]]).
					addSize([1830, 795], [[160, 600], [120, 600]]).
                    addSize([1740, 795], [[120, 600]]).
					addSize([0, 0], []).
					build();

				gptSideBarSizeMapping = googletag.sizeMapping().
					addSize([1140, 795], [[300, 600], [160, 600], [120, 600], [300, 250]]).
					addSize([1140, 445], [[300, 250]]).
					addSize([0, 0], []). 				
					build();

				gptBoomBoxSizeMapping = googletag.sizeMapping().
					addSize([768, 0], [[300, 250], [336, 280]]).
					addSize([0, 0], []). 						
					build();

				gptFluidMapping = googletag.sizeMapping().
					addSize([0, 0], ['fluid']).
					build();

				gptPrestitialMapping = googletag.sizeMapping().
					addSize([600, 700], [[1, 1]]).
					addSize([0, 0], []).
					build();

				gptInterscrollerMapping = googletag.sizeMapping().
					addSize([1140, 445], []).
					addSize([1024, 768], [[1024, 768]]).
					addSize([768, 1024], [[768, 1024]]).
					addSize([480, 320], [[480, 320]]).
					addSize([320, 480], [[320, 480]]).
					build();
			}
			//***********************************************
			//End Define Size mapping based on client Viewport
			//***********************************************

			//***********************************************
			//Start detection for ad slots on page
			//***********************************************
			var gptMasterPage = gpt_CheckForNULL($('#gptPage').val());
			var gptMasterSite;

			if (gpt_AdFile === 'PU') {
				gptMasterSite = $('#gptSite').val();
			}
			else {
				//JBS-CELL-LANCET
				gptMasterSite = "dev";

				if (gpt_IsProd) {
					gptMasterSite = gpt_CheckForNULL($('#gptSite').val());

					//If there is no master site, check for a defaultsite
					if (gptMasterSite == null) {
						gptMasterSite = gpt_CheckForNULL($('#gptDefaultSite').val());
					}
				}
				else {
					//Dev ENV
					//If the page has no gptSite Value, assume no value which will disable ads
					if (gpt_CheckForNULL($('#gptSite').val()) === null) {
						gptMasterSite = null;
					}
				}

				//If there is no master Page, check for a Page using the URL
				if (gptMasterPage === null) {
					gptMasterPage = GPT_GetPageByURL();
				}
			}

			$(".gpt-ad").each(function (index) {
				var gptPage = '';
				var adSizeMapping = '';
				var gptPOS;
				var gptTC;
				var gptNative;
				var adId;
				var gptSite;
				var tempSlot = null;


                if (gpt_IsProd) {
					if (gpt_AdFile !== "CELLJOURNAL") {
						//Check if the ad slot has it's own site defined
						if (typeof $(this).data('site') === 'undefined') {
							gptSite = gptMasterSite;
						}
						else {
							gptSite = gpt_CheckForNULL($(this).data('site'));
						}
					}
					else {
						gptSite = gptMasterSite;
					}
                }
                else {
					gptSite = gptMasterSite;
                }



				if (gpt_AdFile === 'PU') {
					adId = $(this).attr('id');
				}
				else {
					//JBS-CELL-LANCET
					adId = 'gpt-ad-' + index + 1;
				}

				if (gpt_CheckForNULL(gptSite) !== null) {
					gptSite_Debug = gptSite;

					gptPOS = parseInt($(this).data('pos')) || -1;
					if (gptPOS === -1) {
						gptPOS = index + 1;
					}

					//Check if the ad slot has it's own page defined
					gptPage = $(this).data('page');
					if (gpt_CheckForNULL(gptPage) == null) {
						//Use the Master Page
						gptPage = gptMasterPage;
					}

					//If the page is not null, preceed it with a slash
					if (gptPage !== null) {
						gptPage = '/' + gptPage;
					}
					else {
						gptPage = '';
					}
					gptPage_Debug = gptPage;


					gptTC = $(this).data('tc');
					if (typeof gptTC === "undefined") {
						gptTC = '';
					}

					gptNative = $(this).data('native');
					if (typeof gptNative === "undefined") {
						gptNative = '';
					}

					//Determine sizemapping based on data-adsize
					if (gpt_AdFile === 'PU') {
						switch ($(this).data('adsize')) {
							case 'tall':
								adSizeMapping = gptTallSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600]], adId);
								break;
							case 'rectangle':
								adSizeMapping = gptRectangleSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250]], adId);
								break;
							case 'rectangleshort':
								adSizeMapping = gptRectangleShortSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250], [320, 50], [300, 50]], adId);
								break;
							case 'wide':
								adSizeMapping = gptWideSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[728, 90], [320, 50], [300, 50]], adId);
								break;
							case 'widerectangle':
								adSizeMapping = gptWideRectangleSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[320, 50], [300, 50], [300, 250]], adId);
								break;
							case 'mobile':
								adSizeMapping = gptMobileSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[320, 50], [300, 50]], adId);
								break;
							case 'fluid':
								adSizeMapping = gptFluidMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, 'fluid', adId);
								break;
							case 'prestitial':
								adSizeMapping = gptPrestitialMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[1, 1]], adId);
								break;
						}
					}
					else {
						//JBS-CELL-LANCET
						switch ($(this).data('adsize')) {
							case 'cell-leaderboard':
							case 'leaderboard':
								adSizeMapping = gptLeaderboardSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[728, 90], [320, 50], [300, 50]], adId);
								break;
							case 'bottom':
								if (gpt_ResponsiveHomepage) {
									adSizeMapping = gptHomePageSidebarMapping;
									tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600], [300, 250]], adId);
								}
								else {
									adSizeMapping = gptBoomBoxSizeMapping;
									tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250]], adId);
								}
								break;
							case 'boombox':
								if (gpt_AdFile === "CELLJOURNAL" || gpt_AdFile == "CELLBUCKET") {
									if (gpt_ResponsiveHomepage) {
										adSizeMapping = gptCellHomePageSidebarMapping;
										tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600], [300, 250], [336, 280]], adId);
									}
									else {
										adSizeMapping = gptBoomBoxSizeMapping;
										tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250], [336, 280]], adId);
									}
								}
								else {
									adSizeMapping = gptBoomBoxSizeMapping;
									tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250]], adId);
								}
								break;
							case 'interscroller':
								adSizeMapping = gptInterscrollerMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[320, 480], [480, 320], [768, 1024], [1024, 768]], adId);
								break;
							case 'sidebar':
							case 'skyscraper':
								adSizeMapping = gptSideBarSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600], [300, 250]], adId);
								break;
							case 'native':
							case 'fluid':
								adSizeMapping = gptFluidMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, 'fluid', adId);
								break;
							case 'prestitial':
								adSizeMapping = gptPrestitialMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, 'fluid', adId);
								break;
						}
					}
				}

				//Ignore any invalid adsizes that could be on the page
				if (adSizeMapping !== '') {
					$(this).addClass('gpt-ad-defined');
					if (gpt_AdFile !== 'PU') {
						$(this).attr('id', adId);
					}
					tempSlot.defineSizeMapping(adSizeMapping).addService(googletag.pubads()).setTargeting("pos", gptPOS);

					if (gptTC !== '') {
						tempSlot.setTargeting("tc", gptTC);
					}
					if (gptNative !== '') {
						tempSlot.setTargeting("native", gptNative);
					}
					gptEvent_Debug = gptEvent_Debug + $(this).data('adsize') + ' Requested<br>';
				}
				else {
					gptInvalidAdSize_Debug = gptInvalidAdSize_Debug + "Invalid Size Found, Id: " + $(this).prop('id') + ', adSize: ' + $(this).data('adsize') + '<br>';
					$(this).remove();
				}
			});
			if (gpt_CheckForNULL(gptInvalidAdSize_Debug) === null) {
				gptInvalidAdSize_Debug = "None Detected";
			}
			//***********************************************
			//End detection for ad slots on page
			//***********************************************

			//***********************************************
			//Start page level targeting
			//***********************************************
			if (gpt_AdFile !== 'PU') {
				//Logged in or IP Access
				gpt_TargetAccessType();

				//Check for Screenshot Client
				gpt_CheckSSC();
			}

			//Check for DFP Test
			gpt_GetDFPTestId();

			//Check for Additional Targeting
			gpt_GetAdditionalTargeting();

			if (gpt_IsProd && gptContextualEnabled) {
				var gpt_ContextualRetry = false;
				//Check for Contextual Targeting
				if (gs_channels !== "DEFAULT") {
					var gpt_ContextualCategories = Array.from(gs_channels);
					for (var i = 0; i < gpt_ContextualCategories.length; i++) {
						if (gpt_ContextualCategories[i] == 'gx_retry') {
							gpt_ContextualRetry = true;
						}
						if (!gpt_ContextualCategories[i].startsWith("custom_")) {
							//Remove any tag that is not custom
							gpt_ContextualCategories.splice(i, 1);
							i--;
						}
						else {
							//Strip the 'custom_' from the tag
							gpt_ContextualCategories[i] = gpt_ContextualCategories[i].replace("custom_", "")
						}
					}
					if (gpt_ContextualCategories.length > 0) {
						googletag.pubads().setTargeting("gs_cat", gpt_ContextualCategories);
					}
					else {
						gpt_ContextualCategories = 'none';
					}
					if (gpt_ContextualRetry) {
						gpt_ContextualCategories = "*Indexing*";
					}
					gptContextualResult_Debug = gpt_ContextualCategories;
				}
				else {
					gptContextualResult_Debug = 'No segments found';
				}
			}
			else {
				//Not Prod || Contextual not enabled
				gptContextualResult_Debug = 'Not enabled';
				
				if (!gpt_IsProd) {
                    var gpt_DevSite = gpt_CheckForNULL($('#gptSite').val(), "dev");
                    $('#gpt-Variables').append('<input type="hidden" id="gptEnvironment" value="' + gpt_DevSite + '">');
                    gpt_Target("Environment");
                }

			}
			//***********************************************
			//End page level targeting
			//***********************************************

			//Enable google dfp services
			if (parseInt($('#gptLazyLoading').val()) === 1) {
				googletag.pubads().enableLazyLoad({
					fetchMarginPercent: 200,
					renderMarginPercent: 0,
					mobileScaling: 2.0
				});
			}
			else {
				googletag.pubads().enableSingleRequest();
			}

			//Add Listener for RenderEnded
			googletag.pubads().addEventListener('slotRenderEnded', function (event) {
				if (!event.isEmpty) {
					if (typeof LocalAdLoaded === "function") {
						LocalAdLoaded(event);
					}
					if (gpt_ResponsiveHomepage) {
						if (typeof positionAd === "function") {
							positionAd();
						}
					}
					if (event.size[0] == "480" || event.size[1] == "480") {
						$('.gpt-InScrollBox').css('display', 'block');
						gptEvent_Debug = gptEvent_Debug + 'Interscroller ' + event.size + ' rendered: ' + gpt_GetElapsedTime() + '<br>';
					}
					else {
						gptEvent_Debug = gptEvent_Debug + event.size + ' rendered: ' + gpt_GetElapsedTime() + '<br>';
					}

				}
			});

			if (gpt_AdFile !== 'PU') {
				googletag.pubads().collapseEmptyDivs();
			}
			googletag.enableServices();

			//Populate Defined Ad Slots
			$(".gpt-ad-defined").each(function (index) {
				var adId = $(this).attr('id');
				googletag.cmd.push(function () { googletag.display(adId); });
			});

			gptStatus_Debug = 'GPT Complete';
		});
	}
	//--------------------------------------------------------------------------
	function gpt_DisplayDebug() {
		//Clone the template and append to Body
		$(document.body).append($('#gpt-DebugDiv').clone());
		$('#gpt-DebugDiv').remove();

		//Populate the Values
		gpt_LoadDebugValues();

		//Display and scroll to debug section
		$('.gpt-DebugDiv').css('display', 'block');
		window.scrollTo(0, document.body.scrollHeight);
	}
	//--------------------------------------------------------------------------
	function gpt_LoadDebugValues() {
		$('.gpt-DebugBtn').blur();
		$('.gptVersion_Debug').text(gpt_Version);
		$('.gptAdFile_Debug').html(gpt_AdFile);
		$('.gptEvent_Debug').html(gptEvent_Debug);
		$('.gptCustomTargeting_Debug').html(gptCustomTargeting_Debug);
		$('.gptSite_Debug').text(gptSite_Debug);
		$('.gptContextualResult_Debug').text(gptContextualResult_Debug);
		$('.gptContextualInit_Debug').text(gptContextualInit_Debug);
		$('.gptContextualURL_Debug').text(gptContextualURL_Debug);
		$('.gptContextualReady_Debug').text(gptContextualReady_Debug);
		$('.gptMoatReady_Debug').text(gptMoatReady_Debug);
		$('.gptMoatInit_Debug').text(gptMoatInit_Debug);
		$('.gptTestENV_Debug').text(gptTestENV_Debug);
		$('.gptDisableAds_Debug').text(gptDisableAds_Debug);
		$('.gptStatus_Debug').text(gptStatus_Debug);
		$('.gptPage_Debug').text(gptPage_Debug);
		$('.gptInvalidAdSize_Debug').html(gptInvalidAdSize_Debug);
	}
	//--------------------------------------------------------------------------
	function gpt_GrapeShotDebug() {
		$('.gpt-DebugBtn').blur();
		window.open("http://admin-elsevier.grapeshot.co.uk/custom/bookmarklet.cgi?blocking=1&vn=2&url=" + encodeURIComponent(document.location), "GRAPESHOT_POPUP", "width=650,height=600,scrollbars=yes,screenX=40,screenY=50,menubar=no,toolbar=no,resizable=yes", "Custom").focus();
	}
	//--------------------------------------------------------------------------
	function gpt_ShowConsole() {
		$('.gpt-DebugBtn').blur();
		googletag.openConsole();
	}
	//--------------------------------------------------------------------------
</script>
        </div>

            
                </div>
            

        

    

        


    

            

            

            <div data-widget-id="912135d6-7135-4d45-b457-b503cacc3c8b" data-widget-name="ux3-layout-widget"><header class="header base fixed">
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none adplaceholder adplaceholder--header js__adToHide widget-horizontalAd" id="9d6062d7-fad6-4e64-81cc-7dd25d99b23d">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-horizontalAd "><div class="horizontalAdContainer">
    <div class="horizontalAdLabelContainer">
        <div class="horizontalAdLabel">
                <img src="/templates/jsp/_style2/_marlin/images/bg_adHoriz.gif" alt="Advertisement">
        </div>
    </div>
    <div class="horizontalAd">
        





        <div class="pb-ad">
            <!-- Beginning Async AdSlot 1 for Ad unit els.cellpress/cell  ### size: [[728,90]] -->
<div class="gpt-ad" data-adsize="cell-leaderboard" data-page="" data-site="cell" data-pos="1"></div>
<!-- End AdSlot 1 -->
        </div>
    

    </div>
</div></div>
                </div>
                </div>

        










        <div data-widget-def="ux3-layout-widget" data-widget-id="3a83e2a6-2e7e-4931-892c-4ef43b6e3312" class="header__wrapper clearfix" data-db-parent-of="quickSearch"><div id="brandingRow"><a href="/" data-aa-name="science that inspires" data-aa-region="cp global header" title="Visit Cell.com">Science that inspires</a><p id="brandingRowRight"><a href="/" data-aa-name="cell press endorsement" data-aa-region="cp global header" title="Visit Cell.com">A Cell Press journal</a></p></div>
        
    

        


    

            

            

            <div data-widget-id="3a83e2a6-2e7e-4931-892c-4ef43b6e3312" data-widget-name="ux3-layout-widget"><div class="col-xs-6 header__left-side">
    

        


    

            

            
                <a href="/home" id="cpIconLnk" data-aa-name="cell press logo" data-aa-region="cp global header" title="Cell Press" target="_blank"><svg id="cp_logo" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 100 98.1" style="enable-background:new 0 0 100 98.1;" xml:space="preserve" height="100" width="98.1"><path style="fill:#FFFFFF;" d="M69.09,0.09c-4.01-0.6-7.83,1.94-8.78,5.89l-3.14,13.11c-0.76,3.19-3.44,5.56-6.7,5.93l-18.6,2.13          C11.83,29.31-2.53,47.66,0.37,67.78c2.3,15.96,15.18,27.77,30.54,30.08c4.01,0.6,7.83-1.94,8.78-5.89l3.14-13.11 c0.76-3.19,3.44-5.56,6.69-5.93l18.6-2.13c20.04-2.17,34.41-20.52,31.5-40.65C97.32,14.21,84.45,2.39,69.09,0.09z M65.06,47.2  l-11.75,1.4c-1.98,0.21-3.62,1.65-4.08,3.59l-3.43,14.3c-1.01,4.22-4.5,7.39-8.8,8c-6.66,0.72-12.63-4.2-13.15-10.92                c-0.51-6.5,4.6-12.11,11.08-12.81l11.75-1.39c1.98-0.22,3.62-1.65,4.08-3.59l3.43-14.3c1.01-4.22,4.5-7.39,8.8-8          c6.66-0.72,12.63,4.19,13.15,10.92C76.66,40.89,71.55,46.5,65.06,47.2z"></path></svg></a><div class="journal-logos">
            

            <span data-widget-id="35d8d734-3aa0-4ec8-9e2d-14babe803594" data-widget-name="ux3-general-image"><a href="/cell/home" title="Cell"><img alt="Cell" src="/pb-assets/ux3/logos/cell/Cell_white-1532516611593.svg"></a></span>

            
                </div>
            

        

    

        


    

            

            
                <div class="journal-logos">
            

            <span data-widget-id="3ffcbf1f-79fd-4ba3-85b5-bef10a63475f" data-widget-name="ux3-general-image"><a href="https://www.cell.com/open-access?utm_source=oa-label&amp;__hstc=109691226.e4e812b53ff324eccbdbdb2e4fb9a6ab.1644503737405.1644524234622.1644587109507.4&amp;__hssc=109691226.7.1644587109507&amp;__hsfp=3253153019" title="This journal offers authors two options (open access or subscription) to publish research"><img alt="This journal offers authors two options (open access or subscription) to publish research" src="/pb-assets/ux3/logos/cell/Supports_open_access_border-1645554937897.svg"></a></span>

            
                </div>
            

        
</div><div class="col-xs-6">
    

        


    

            

            

            <div data-widget-id="2555f6db-b333-4edb-8b15-ef9213f1f0a5" data-widget-name="ux3-layout-widget"><div class="header__nav row">
    

        


    

            

            

            <ul data-widget-id="52432d24-0331-4c9d-85f3-2033edc7208b" data-widget-name="UX3ProfileLinksWidget" class="rlist rlist--inline profile-links"><li class="profile-links__submit"><a data-adobe-name="Publish" class="profile-links__submit__btn" title="Submit" href="https://www.editorialmanager.com/cell" target="_self" data-aa-conversiondriver="jbs:global-header:submitpage" data-profile-link="true">Submit</a></li><li class="profile-links__login-dropdown"><div class="profile-links__user-list__wrapper"><button data-db-target-for="loginmenu-652873" aria-expanded="false" aria-controls="login-links-652873" title="Log in links dropdown" class="profile-links__username"><span>Log in</span></button><div data-db-target-of="loginmenu-652873" id="login-links--652873" class="profile-links__user-list__holder dropBlock__holder"><ul class="rlist profile-links__list"><li class="profile-links__login"><a data-adobe-name="Log In" class="profile-links__login__btn" title="Log in to Cell.com" href="https://www.cell.com/action/idLogin?type=login&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2900753-4" target="_self" data-profile-link="true">Log in</a></li><li class="profile-links__institution"><a data-adobe-name="Get institutional access" class="profile-links__institution__btn" title="Institutional access" href="https://www.cell.com/action/idLogin?type=institution&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2900753-4" target="_self" data-profile-link="true">Get Institutional Access</a></li></ul></div></div></li><li class="profile-links__register"><a data-adobe-name="Register" class="profile-links__register__btn" title="Register a username and password for free" href="https://www.cell.com/action/idLogin?type=registration&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2900753-4" target="_self" data-profile-link="true">Register</a></li><li class="hidden-lg"> <div class="profile-links__controls"> <button aria-expanded="false" aria-controls="links-dialog-652873" data-opens="#links-dialog-652873" class="profile-links__btn--open"><span class="sr-only">open links dialog</span><i aria-hidden="true" class="icon-gizmo-person"></i></button></div><dialog id="links-dialog-652873" aria-modal="true"><button data-closes="#links-dialog-652873" class="profile-links__btn--close"><span class="sr-only">close links dialog</span><i aria-hidden="true" class="icon-close-solid"></i></button><ul>   <li class="profile-links__submit"><a data-adobe-name="Publish" class="profile-links__submit__btn" title="Submit" href="https://www.editorialmanager.com/cell" target="_self" data-aa-conversiondriver="jbs:global-header:submitpage" data-profile-link="true">Submit</a></li><li class="profile-links__login"><a data-adobe-name="Log In" class="profile-links__login__btn" title="Log in to Cell.com" href="https://www.cell.com/action/idLogin?type=login&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2900753-4" target="_self" data-profile-link="true">Log in</a></li><li class="profile-links__institution"><a data-adobe-name="Get institutional access" class="profile-links__institution__btn" title="Institutional access" href="https://www.cell.com/action/idLogin?type=institution&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2900753-4" target="_self" data-profile-link="true">Get Institutional Access</a></li><li class="profile-links__register"><a data-adobe-name="Register" class="profile-links__register__btn" title="Register a username and password for free" href="https://www.cell.com/action/idLogin?type=registration&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2900753-4" target="_self" data-profile-link="true">Register</a></li></ul></dialog></li></ul>

            

        
</div></div>

            

        
</div></div>

            

        

        </div>
    

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget csasInstitutionBanner none  widget-none" id="51b7fa33-2137-4f58-9e98-9746c4400513" style="display: none;">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none "><div class="ui-state-highlight ui-corner-all csas-institution-banner hidden" data-doi="10.1016/j.cell.2021.06.018">
    <span class="ui-icon ui-icon-close banner-close"></span>
    <p>
        Access provided by <span class="accountName"></span>
    </p>
</div>

<style>
    .csas-institution-banner {
        padding: 10px 0 0 10px;
        margin: 5px;
    }

    .csas-institution-banner.ui-state-highlight .ui-icon {
        background-image: url("/templates/jsp/css/jquery-ui-1.10.2/base/images/ui-icons_454545_256x240.png");
        margin-right: 5px;
        margin-top: -4px;
    }

    .csas-institution-banner .banner-close {
        float: right;
    }
</style>

<script>

    function resetBanner(doi) {
        $.get('/action/getAccount?doi=' + doi, function(data) {
            if (data) {
                $(".csas-institution-banner p .accountName").html(data);
                $(".csasInstitutionBanner").fadeIn();
                $(".csas-institution-banner").fadeIn();
            }
        });
    }

    var flag = 0;
    $(document).ready(function() {
        if(!$(".csas-institution-banner").is(":visible")){
            $(".csasInstitutionBanner").hide();
        }

        $(".banner-close").click(function() {
            $.get("/action/dismissBanner", function(data) {
                $(".csasInstitutionBanner").fadeOut(function () {
                    if(typeof UX !== 'undefined') {
                        UX.pageBody.init();
                    }
                });
            });
        });

        $(".articleContent .tabs li[role='tab'].showFullText").click(function() {
           flag = 1;
        });

        $(document).on("reloadTabsContent", function() {
            var isSelected = $(".articleContent .tabs li[role='tab'].showFullText").hasClass("active");
            if (isSelected) {
                var isHidden =  $(".csas-institution-banner").is(".hidden");
                if (!isHidden) {
                    resetBanner($(".csas-institution-banner").data("doi"));
                    console.log("fadeIn");
                }
            } else {
                $(".csas-institution-banner").fadeOut();
                console.log("fadeOut");
            }
            flag = 0;
        });

        $(document).on("click", ".articleContent .tabs li[role='tab']", function() {

        });
    });
</script></div>
                </div>
                </div>

        
</header></div>

            

        










        <div data-widget-def="graphQueryWidget" data-widget-id="2a125dae-6dbc-4e76-a75e-3e563f1d3a1d" class="ux3">
        
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget graphQueryWidget none top-padding-handler noMargin axel-mega-menu widget-none widget-compact-all fonts-loaded" id="2a125dae-6dbc-4e76-a75e-3e563f1d3a1d" style="padding-top: 115.594px;">
                
                
                    
                

                <div class="wrapped " id="CP TOP LEVEL NAV MENU">
                    
                    <div class="widget-body body body-none  body-compact-all"><nav id="mega-menu-nav" role="navigation" class="mega-menu-nav">
  <!--.skip//a(href='#searchText' aria-label='Skip menu') Skip menu
  -->
  <div class="mega-menu-nav__wrapper">
    <button class="mega-menu__toggle"><span class="sr-only">Main menu</span><i aria-hidden="true" class="icon-gizmo-hamburger"></i></button>
    <ul role="menubar" class="mega-menu rlist">
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-1" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-1" data-item-link="main-menu-link-1" data-index="0" class="mega-menu__item-link menu-link"><span>This journal</span></a>
        <div id="main-menu-panel-1" aria-labelledby="main-menu-link-1" data-item-wrapper="main-menu-link-1" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <div aria-label="Menu dropzone section" tabindex="-1" class="listDropZone">
    

        


    

            

            

            <div class="cover-image"><div class="cover-image__thumb"><a href="/issue/S0092-8674(24)X0012-4" class="menu-link"><img src="/cms/asset/atypon:cms:attachment:img:d5e6:rev:1748270185670-389:pii:S0092867424X00124/cover.tif.jpg" alt="Cover Image - Cell, Volume 188, Issue 11"></a></div><div class="cover-image__meta"></div></div>

            

        
</div>
            </li>
            <li style="grid-column: 2/3" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-1-2" class="sub-menu__heading">Read
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/newarticles" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Online now</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/current" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Current issue</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/archive" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Archive</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/action/showPreferences?menuTab=Alerts" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Publication alerts</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/sneakpeek" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Sneak Peek - preprints</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 3/5" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-1-3" class="sub-menu__heading">About
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-2">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/advisory-board" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Advisory board</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/aims" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Aims and scope</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/contact" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Contact</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/editors-and-staff" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Editors and staff</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/editorial-ethics" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Ethics</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/information-for-authors/permissions" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Permissions</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/reviewers" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>Reviewer instructions</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-2" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-2" data-item-link="main-menu-link-2" data-index="1" class="mega-menu__item-link menu-link"><span>Journals</span></a>
        <div id="main-menu-panel-2" aria-labelledby="main-menu-link-2" data-item-wrapper="main-menu-link-2" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/5" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-2-1" class="sub-menu__heading">Life &amp; medical sciences
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-4">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/home" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cancer-cell/home" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Cancer Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-chemical-biology" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Chemical Biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-genomics/home" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Genomics</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-host-microbe/home" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Host &amp; Microbe</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-metabolism/home" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Metabolism</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports/home" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-medicine/home" data-index="7" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Medicine</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-stem-cell/home" data-index="8" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Stem Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-systems/home" data-index="9" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Systems</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/current-biology/home" data-index="10" role="menuitem" class="sub-menu__item-link menu-link"><span>Current Biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/developmental-cell/home" data-index="11" role="menuitem" class="sub-menu__item-link menu-link"><span>Developmental Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/immunity/home" data-index="12" role="menuitem" class="sub-menu__item-link menu-link"><span>Immunity</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/med/home" data-index="13" role="menuitem" class="sub-menu__item-link menu-link"><span>Med</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-cell/home" data-index="14" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/neuron/home" data-index="15" role="menuitem" class="sub-menu__item-link menu-link"><span>Neuron</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/structure/home" data-index="16" role="menuitem" class="sub-menu__item-link menu-link"><span>Structure</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/ajhg/home" data-index="17" role="menuitem" class="sub-menu__item-link menu-link"><span>American Journal of Human Genetics (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/biophysj/home" data-index="18" role="menuitem" class="sub-menu__item-link menu-link"><span>Biophysical Journal (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/biophysreports/home" data-index="19" role="menuitem" class="sub-menu__item-link menu-link"><span>Biophysical Reports (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/hgg-advances/home" data-index="20" role="menuitem" class="sub-menu__item-link menu-link"><span>Human Genetics and Genomics Advances (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-plant/home" data-index="21" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Plant (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/molecular-therapy/home" data-index="22" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/methods/home" data-index="23" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy Methods &amp; Clinical Development (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/nucleic-acids/home" data-index="24" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy Nucleic Acids (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/oncology/home" data-index="25" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy Oncology (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/plant-communications/home" data-index="26" role="menuitem" class="sub-menu__item-link menu-link"><span>Plant Communications (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/stem-cell-reports/home" data-index="27" role="menuitem" class="sub-menu__item-link menu-link"><span>Stem Cell Reports (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/biochemical-sciences/home" data-index="28" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Biochemical Sciences</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/cancer/home" data-index="29" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Cancer</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/cell-biology/home" data-index="30" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Cell Biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/ecology-evolution/home" data-index="31" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Ecology &amp; Evolution</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/endocrinology-metabolism/home" data-index="32" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Endocrinology &amp; Metabolism</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/genetics/home" data-index="33" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Genetics</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/immunology/home" data-index="34" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Immunology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/microbiology/home" data-index="35" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Microbiology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/molecular-medicine/home" data-index="36" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Molecular Medicine</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/neurosciences/home" data-index="37" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Neurosciences</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/parasitology/home" data-index="38" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Parasitology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/pharmacological-sciences/home" data-index="39" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Pharmacological Sciences</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/plant-science/home" data-index="40" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Plant Science</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 5/6" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-2-2" class="sub-menu__heading">Physical sciences &amp; engineering
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-biomaterials/home" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Biomaterials</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-physical-science/home" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Physical Science</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/chem/home" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Chem</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/chem-catalysis/home" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Chem Catalysis</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/device/home" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Device</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/joule/home" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Joule</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/matter/home" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>Matter</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/newton/home" data-index="7" role="menuitem" class="sub-menu__item-link menu-link"><span>Newton</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/chemistry/home" data-index="8" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Chemistry</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 6/7" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-2-3" class="sub-menu__heading">Multidisciplinary
              </h4>
              <ul role="menubar" data-index="2" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-methods/home" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Methods</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-sustainability/home" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Sustainability</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/heliyon/home" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Heliyon</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/iscience/home" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>iScience</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/one-earth/home" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>One Earth</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/patterns/home" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Patterns</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/star-protocols/home" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>STAR Protocols</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/nexus/home" data-index="7" role="menuitem" class="sub-menu__item-link menu-link"><span>Nexus (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/the-innovation/home" data-index="8" role="menuitem" class="sub-menu__item-link menu-link"><span>The Innovation (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/biotechnology/home" data-index="9" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Biotechnology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/cognitive-sciences/home" data-index="10" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Cognitive Sciences</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-3" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-3" data-item-link="main-menu-link-3" data-index="2" class="mega-menu__item-link menu-link"><span>Publish</span></a>
        <div id="main-menu-panel-3" aria-labelledby="main-menu-link-3" data-item-wrapper="main-menu-link-3" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-3-1" class="sub-menu__heading">Authors
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="https://www.editorialmanager.com/cell?__hstc=25856994.7201670a4397fa03148684920d4a8d38.1672762013831.1694716659709.1694718909620.146&amp;__hssc=25856994.15.1694718909620&amp;__hsfp=1366844671" data-index="0" target="_blank" role="menuitem" class="sub-menu__item-link menu-link"><span>Submit article
                      <div class="external-icon"></div><span class="sr-only">External Link</span></span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/information-for-authors" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Information for authors</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/open-access" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Open access</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/multi-journal-submission" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Multi-Journal Submission</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/information-for-authors/star-authors-guide" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>STAR Methods</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 2/3" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-3-2" class="sub-menu__heading">Reviewers
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/reviewers" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Information for reviewers</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-4" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-4" data-item-link="main-menu-link-4" data-index="3" class="mega-menu__item-link menu-link"><span>News &amp; events</span></a>
        <div id="main-menu-panel-4" aria-labelledby="main-menu-link-4" data-item-wrapper="main-menu-link-4" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/newsroom" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Newsroom</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/events" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Events</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/symposia" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Symposia</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/webinars" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Webinars</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/consortia" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Consortia Hub</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-5" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-5" data-item-link="main-menu-link-5" data-index="4" class="mega-menu__item-link menu-link"><span>About Cell Press</span></a>
        <div id="main-menu-panel-5" aria-labelledby="main-menu-link-5" data-item-wrapper="main-menu-link-5" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-5-1" class="sub-menu__heading">Cell Press
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/about" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>About Cell Press</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/contact" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Contact us</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="https://service.elsevier.com/app/overview/cell-press/" data-index="2" target="_blank" role="menuitem" class="sub-menu__item-link menu-link"><span>Help &amp; support
                      <div class="external-icon"></div><span class="sr-only">External Link</span></span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/open-access" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Open access at Cell Press</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/diversity" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Inclusion and diversity</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-press-50" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>50th anniversary</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 2/3" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-5-2" class="sub-menu__heading">Hubs
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/biotechnology-bioengineering" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Biotechnology and bioengineering</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-press-chemistry" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Chemistry</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/clinical-translational" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Clinical and translational</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/covid-19" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>COVID-19</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/fundamental-biology" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Fundamental biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/sustainability" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Sustainability</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 3/4" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-5-3" class="sub-menu__heading">Careers
              </h4>
              <ul role="menubar" data-index="2" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="https://relx.wd3.myworkdayjobs.com/en-US/ElsevierJobs" data-index="0" target="_blank" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Press careers
                      <div class="external-icon"></div><span class="sr-only">External Link</span></span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
    </ul>
    <div class="mega-menu-nav__quick-search">
    

        


    

            

            

            <div data-widget-id="a1f05748-166f-41e0-b01a-ca8ac1cc95d4" data-widget-name="UX3QuickSearchWidget" class="quick-search"><div class="dropBlock"><form action="https://www.cell.com/action/doSearch?journalCode=cell" name="quickSearchBar" method="get" title="Quick Search"><input id="type" type="hidden" value="quicksearch" name="type"><div class="quick-search__toggle autoCompleteWrapper"><input id="searchText" type="text" name="text1" placeholder="Search for..." enterkeyhint="search" autocomplete="off" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" data-group-titles-conf="3" data-publication-items-conf="3" aria-label="Enter a search term" class="searchText searchText-autocomplete ui-autocomplete-input"><button type="submit" title="Search" aria-label="Submit the search form" class="btn quick-search__button icon-search"><span>Go search</span></button><a href="https://www.cell.com/search/advanced?SeriesKey=cell&amp;ISSN=0092-8674&amp;journalCode=cell" title="Advanced search" class="quick-search__advanced-link">Advanced search</a><ul id="ui-id-1" tabindex="0" class="ui-menu ui-widget ui-widget-content ui-autocomplete ui-front" unselectable="on" style="display: none;"></ul></div><div class="quick-search__box"><div class="quick-search__box-inner"><div class="quick-search__box-wrapper"><select name="field1" aria-label="Select a search area"><option value="AllField">All content</option><option value="Title">Article title</option><option value="Contrib">Authors</option><option value="Keyword">Keywords</option><option value="Abstract">Abstract</option><option value="AbstractTitleKeywordFilterField">Article title, abstract, keywords</option></select><input type="hidden" name="journalCode" value="cell"><div class="quick-search__scope-options"><span>Search within:</span><label title="This journal" class="radio--primary"><input type="radio" name="SeriesKey" id="searchScopeSeries-a1f05748-166f-41e0-b01a-ca8ac1cc95d4" title="This journal" value="cell" checked="checked" aria-label="Search within current journal"><span>This journal</span></label><label title="Full site" class="radio--primary"><input type="radio" name="SeriesKey" id="searchScopeFullSite-a1f05748-166f-41e0-b01a-ca8ac1cc95d4" value="" title="Full site" aria-label="Search within full site"><span>Full site</span></label></div><a href="https://www.cell.com/search/advanced?SeriesKey=cell&amp;ISSN=0092-8674&amp;journalCode=cell" title="Advanced search" class="quick-search__advanced-link">Advanced search</a></div><a href="javascript:void(0)" aria-label="Close search options" title="Close search options" class="quick-search__box-close"><i aria-hidden="true" class="icon-tools_close"></i></a></div></div><div class="quick-search__alert hidden"><p>Please enter a term before submitting your search.</p><a href="#" class="quick-search__alert--close"> Ok</a></div></form></div></div>

            

        
</div>
  </div>
</nav>
<div id="page-body-id" class="page-body-flag"></div></div>
                </div>
                </div>

        

        </div>
    

    

        


    

            

            

            <div data-widget-id="f8c6daf9-9df1-4fd9-ba3a-7f5d7b3d4466" data-widget-name="UX3HTMLWidget"><style>
.journal-logos:nth-of-type(2) img {
max-height: 30px !important;
}
.journal-logos:nth-of-type(3) img {
max-height: 30px !important;
}
.ux3 .journal-logos + .journal-logos {
margin-left: 30px;
}
.header__wrapper.clearfix .col-xs-6:first-of-type {
width: 55%;
}
.header__wrapper.clearfix .col-xs-6:nth-of-type(2) {
width: 45%;
}
@media screen and (max-width: 1000px) {
.ux3 .journal-logos + .journal-logos {
display: none;
}
.header__wrapper.clearfix .col-xs-6:first-of-type {
width: 50%;
}
.header__wrapper.clearfix .col-xs-6:nth-of-type(2) {
width: 50%;
}
}

.ux3 .journal-logos:first-of-type {
   max-width: 500px;
}

.profile-links {
   white-space: nowrap;
}

@media only screen and (min-width: 740px) and (max-width: 1600px) {
   .journal-logos:nth-of-type(2) img {
      max-width: 250px !important;
   }
}

@media only screen and (min-width: 1024px) and (max-width: 1600px) {
   .header__wrapper.clearfix .col-xs-6:first-of-type {
      width: 60% !important;
   }
   .header__wrapper.clearfix .col-xs-6:nth-of-type(2) {
      width: 40% !important;
   }
}

@media only screen and (max-width: 1240px) {
   .ux3 .journal-logos + .journal-logos {
      display: none !important;
   }
}

</style></div>

            

        
</div></div>

            
                </div>
            

        

        </div>
    

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget axelPublicationContent none  widget-none" id="cadc9b41-a542-448a-9de4-8d0803320174">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none "><link id="build-style-article" rel="stylesheet" type="text/css" href="/products/marlin/cell/releasedAssets/css/build-article-b26f3342b722959c7931.css"><article xmlns="http://www.w3.org/1999/xhtml" data-design="pill" data-has="right-rail" vocab="http://schema.org/" typeof="ScholarlyArticle" data-pii="S0092-8674(21)00753-4" lang="en" dir="ltr"><div data-core-wrapper="header"><header data-extent="frontmatter"><div class="core-container"><div data-article-access="free" data-article-access-type="open" class="meta-panel"><span class="meta-panel__left-content"><span class="meta-panel__type"><span>Article</span></span></span><span class="meta-panel__middle-content"><span class="meta-panel__volumeIssue"><a href="/cell/issue?pii=S0092-8674(20)X0016-X"><span>Volume 184</span>, <span>Issue 15</span></a></span><span class="meta-panel__pages"><span>p4048-4063.e32</span></span><span class="meta-panel__onlineDate">July 22, 2021</span></span><span class="meta-panel__right-content"><span class="meta-panel__access meta-panel__access--archive"><span>Open Archive</span></span></span><div class="article-header__download-full-issue"><button title="Download full issue" type="submit" formaction="/action/downloadMultiplePdf?action=downloadIssuePdf&amp;journal=cell&amp;volume=184&amp;issue=15&amp;pii=S0092-8674%2820%29X0016-X" class="btn--download-issue btn--cancel download-full-issue"><i class="icon-download-new"></i><span>Download Full Issue</span></button><div class="table-of-content__alert table-of-content__alert--download"><p>Download started</p><a href="#" class="table-of-content__alert--close">Ok</a></div></div></div><h1 property="name">GABA-receptive microglia selectively sculpt developing inhibitory circuits</h1><div class="contributors"><span class="authors"><span role="list"><span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au1" aria-controls="axel_author_au1"><span property="givenName">Emilia</span> <span property="familyName">Favuzzi</span></a><div id="au1" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Emilia</span> <span property="familyName">Favuzzi</span> <a class="orcid-id" href="https://orcid.org/0000-0003-4256-1490" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0003-4256-1490</a></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Neurobiology, Blavatnik Institute, Harvard Medical School, Boston, MA 02115, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Stanley Center for Psychiatric Research, Broad Institute of Harvard and MIT, Cambridge, MA 02142, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Favuzzi/Emilia" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">2</a></sup></span> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au2" aria-controls="axel_author_au2"><span property="givenName">Shuhan</span> <span property="familyName">Huang</span></a><div id="au2" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Shuhan</span> <span property="familyName">Huang</span> <a class="orcid-id" href="https://orcid.org/0000-0002-3139-7296" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0002-3139-7296</a></div><div class="content"><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">10</div><div id="fn1"><div id="ntpara0010" role="paragraph">These authors contributed equally</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Neurobiology, Blavatnik Institute, Harvard Medical School, Boston, MA 02115, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Stanley Center for Psychiatric Research, Broad Institute of Harvard and MIT, Cambridge, MA 02142, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Huang/Shuhan" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">2</a>,<a role="doc-noteref">10</a></sup></span> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au3" aria-controls="axel_author_au3"><span property="givenName">Giuseppe A.</span> <span property="familyName">Saldi</span></a><div id="au3" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Giuseppe A.</span> <span property="familyName">Saldi</span> <a class="orcid-id" href="https://orcid.org/0000-0002-9994-7354" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0002-9994-7354</a></div><div class="content"><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">10</div><div id="fn1"><div id="ntpara0010" role="paragraph">These authors contributed equally</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Stanley Center for Psychiatric Research, Broad Institute of Harvard and MIT, Cambridge, MA 02142, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Biology, New York University, New York, NY 10003, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Saldi/Giuseppe+A" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">2</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">3</a>,<a role="doc-noteref">10</a></sup></span><span data-displayed-on="all"> ∙ … </span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au4" aria-controls="axel_author_au4"><span property="givenName">Loïc</span> <span property="familyName">Binan</span></a><div id="au4" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Loïc</span> <span property="familyName">Binan</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Klarman Cell Observatory, Broad Institute of Harvard and MIT, Cambridge, MA 02142, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Binan/Lo%C3%AFc" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">4</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au5" aria-controls="axel_author_au5"><span property="givenName">Leena A.</span> <span property="familyName">Ibrahim</span></a><div id="au5" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Leena A.</span> <span property="familyName">Ibrahim</span> <a class="orcid-id" href="https://orcid.org/0000-0001-8255-3423" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0001-8255-3423</a></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Neurobiology, Blavatnik Institute, Harvard Medical School, Boston, MA 02115, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Stanley Center for Psychiatric Research, Broad Institute of Harvard and MIT, Cambridge, MA 02142, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Ibrahim/Leena+A" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">2</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au6" aria-controls="axel_author_au6"><span property="givenName">Marian</span> <span property="familyName">Fernández-Otero</span></a><div id="au6" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Marian</span> <span property="familyName">Fernández-Otero</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Neurobiology, Blavatnik Institute, Harvard Medical School, Boston, MA 02115, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Stanley Center for Psychiatric Research, Broad Institute of Harvard and MIT, Cambridge, MA 02142, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Fern%C3%A1ndez-Otero/Marian" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">2</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au7" aria-controls="axel_author_au7"><span property="givenName">Yuqing</span> <span property="familyName">Cao</span></a><div id="au7" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au7"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Yuqing</span> <span property="familyName">Cao</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Neurobiology, Blavatnik Institute, Harvard Medical School, Boston, MA 02115, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Stanley Center for Psychiatric Research, Broad Institute of Harvard and MIT, Cambridge, MA 02142, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Cao/Yuqing" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">2</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au8" aria-controls="axel_author_au8"><span property="givenName">Ayman</span> <span property="familyName">Zeine</span></a><div id="au8" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au8"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Ayman</span> <span property="familyName">Zeine</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Neurobiology, Blavatnik Institute, Harvard Medical School, Boston, MA 02115, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Zeine/Ayman" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au9" aria-controls="axel_author_au9"><span property="givenName">Adwoa</span> <span property="familyName">Sefah</span></a><div id="au9" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au9"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Adwoa</span> <span property="familyName">Sefah</span> <a class="orcid-id" href="https://orcid.org/0000-0002-1258-5486" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0002-1258-5486</a></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Neurobiology, Blavatnik Institute, Harvard Medical School, Boston, MA 02115, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Stanley Center for Psychiatric Research, Broad Institute of Harvard and MIT, Cambridge, MA 02142, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Sefah/Adwoa" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">2</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au10" aria-controls="axel_author_au10"><span property="givenName">Karen</span> <span property="familyName">Zheng</span></a><div id="au10" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au10"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Karen</span> <span property="familyName">Zheng</span> <a class="orcid-id" href="https://orcid.org/0000-0003-4974-1136" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0003-4974-1136</a></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Broad Institute of Harvard and MIT, Cambridge, MA 02142, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Zheng/Karen" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">5</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au11" aria-controls="axel_author_au11"><span property="givenName">Qing</span> <span property="familyName">Xu</span></a><div id="au11" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au11"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Qing</span> <span property="familyName">Xu</span> <a class="orcid-id" href="https://orcid.org/0000-0001-9479-470X" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0001-9479-470X</a></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">New York University Abu Dhabi, Abu Dhabi, UAE</span></div></div><div class="core-author-link"><a href="/authored-by/Xu/Qing" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">6</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au12" aria-controls="axel_author_au12"><span property="givenName">Elizaveta</span> <span property="familyName">Khlestova</span></a><div id="au12" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au12"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Elizaveta</span> <span property="familyName">Khlestova</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Neurobiology, Blavatnik Institute, Harvard Medical School, Boston, MA 02115, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Khlestova/Elizaveta" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au13" aria-controls="axel_author_au13"><span property="givenName">Samouil L.</span> <span property="familyName">Farhi</span></a><div id="au13" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au13"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Samouil L.</span> <span property="familyName">Farhi</span> <a class="orcid-id" href="https://orcid.org/0000-0003-1359-4568" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0003-1359-4568</a></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Klarman Cell Observatory, Broad Institute of Harvard and MIT, Cambridge, MA 02142, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Farhi/Samouil+L" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">4</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au14" aria-controls="axel_author_au14"><span property="givenName">Richard</span> <span property="familyName">Bonneau</span></a><div id="au14" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au14"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Richard</span> <span property="familyName">Bonneau</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Biology, New York University, New York, NY 10003, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Center for Computational Biology, Flatiron Institute, New York, NY 10010, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Center for Data Science, New York University, New York, NY 10011, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Bonneau/Richard" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">3</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">7</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">8</a></sup></span></span> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au15" aria-controls="axel_author_au15"><span property="givenName">Sandeep Robert</span> <span property="familyName">Datta</span></a><div id="au15" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au15"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Sandeep Robert</span> <span property="familyName">Datta</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Neurobiology, Blavatnik Institute, Harvard Medical School, Boston, MA 02115, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Datta/Sandeep+Robert" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a></sup></span> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au16" aria-controls="axel_author_au16"><span property="givenName">Beth</span> <span property="familyName">Stevens</span></a><div id="au16" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au16"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Beth</span> <span property="familyName">Stevens</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Stanley Center for Psychiatric Research, Broad Institute of Harvard and MIT, Cambridge, MA 02142, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Boston Children’s Hospital, F.M. Kirby Neurobiology Center, Boston, MA 02115, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Stevens/Beth" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">2</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">9</a></sup></span> ∙ <span property="author" typeof="Person" role="listitem" class="corresponding-author"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au17" aria-controls="axel_author_au17"><span property="givenName">Gord</span> <span property="familyName">Fishell</span></a><div id="au17" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au17"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Gord</span> <span property="familyName">Fishell</span></div><div class="content"><div class="core-author-emails"><div class="label">Correspondence</div><div>Corresponding author</div><div><a id="eadd0010" href="mailto:gordon_fishell@hms.harvard.edu" property="email" title="Send email to Gord Fishell">gordon_fishell@hms.harvard.edu</a></div></div><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">11</div><div id="fn2"><div id="ntpara0015" role="paragraph">Lead contact</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Neurobiology, Blavatnik Institute, Harvard Medical School, Boston, MA 02115, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Stanley Center for Psychiatric Research, Broad Institute of Harvard and MIT, Cambridge, MA 02142, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Fishell/Gord" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">2</a>,<a role="doc-noteref">11</a></sup> <a id="eadd0010" href="mailto:gordon_fishell@hms.harvard.edu" property="email" title="Send email to Gord Fishell">gordon_fishell@hms.harvard.edu</a></span><span data-displayed-on="all"> … <span><span data-action="reveal" tabindex="0" role="listitem">Show more</span></span> </span><span data-displayed-on="none" data-on-display="all" aria-hidden="true"> <span data-action="hide" tabindex="0" role="listitem">Show less</span></span></span></span></div><div class="core-self-citation"><div class="expandable-triggers"><button aria-controls="core-affiliations-notes" aria-expanded="false" tabindex="0" role="button"><span>Affiliations &amp; Notes</span><i class="icon-arrow_d_n" aria-hidden="true"></i></button><button aria-controls="core-content-info" aria-expanded="false" tabindex="0" role="button"><span>Article Info</span><i class="icon-arrow_d_n" aria-hidden="true"></i></button></div><div id="core-linked-content">
    

        

</div><div class="expandable-content"><div id="core-affiliations-notes" role="region" data-expandable="item" style="display: none;"><div class="affiliations"><div property="affiliation" typeof="Organization"><span class="label">1</span><span id="aff1" property="name">Department of Neurobiology, Blavatnik Institute, Harvard Medical School, Boston, MA 02115, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">2</span><span id="aff2" property="name">Stanley Center for Psychiatric Research, Broad Institute of Harvard and MIT, Cambridge, MA 02142, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">3</span><span id="aff3" property="name">Department of Biology, New York University, New York, NY 10003, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">4</span><span id="aff4" property="name">Klarman Cell Observatory, Broad Institute of Harvard and MIT, Cambridge, MA 02142, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">5</span><span id="aff5" property="name">Broad Institute of Harvard and MIT, Cambridge, MA 02142, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">6</span><span id="aff6" property="name">New York University Abu Dhabi, Abu Dhabi, UAE</span></div><div property="affiliation" typeof="Organization"><span class="label">7</span><span id="aff7" property="name">Center for Computational Biology, Flatiron Institute, New York, NY 10010, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">8</span><span id="aff8" property="name">Center for Data Science, New York University, New York, NY 10011, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">9</span><span id="aff9" property="name">Boston Children’s Hospital, F.M. Kirby Neurobiology Center, Boston, MA 02115, USA</span></div></div><div class="core-authors-notes"><div role="doc-footnote" data-has="label"><div class="label">10</div><div id="fn1"><div id="ntpara0010" role="paragraph">These authors contributed equally</div></div></div><div role="doc-footnote" data-has="label"><div class="label">11</div><div id="fn2"><div id="ntpara0015" role="paragraph">Lead contact</div></div></div></div></div><div id="core-content-info" role="region" data-expandable="item" style="display: none;"><div class="history"><div>Publication History:</div><div><span>Received September 12, 2020</span>; <span>Revised March 31, 2021</span>; <span>Accepted June 10, 2021</span>; <span>Published online July 6, 2021</span></div></div><div class="core-links"><span class="doi">DOI: <a href="https://doi.org/10.1016/j.cell.2021.06.018" property="sameAs" target="_blank">10.1016/j.cell.2021.06.018 <span class="sr-only">External Link</span></a></span><span class="science-direct">Also available on <a href="https://www.sciencedirect.com/science/article/pii/S0092867421007534" target="_blank">ScienceDirect <span class="sr-only">External Link</span></a></span></div><div role="paragraph">Copyright: © 2021 Elsevier Inc.</div><div class="core-links"><span>User License: <a href="http://www.elsevier.com/open-access/userlicense/1.0/" target="_blank">Elsevier user license</a> | <a href="https://www.elsevier.com/open-access" target="_blank">Elsevier's open access license policy</a></span></div><div class="core-crossmark-badge"><a data-target="crossmark" href="#" title="Check for updates on crossmark" data-doi="10.1016/j.cell.2021.06.018" data-id="article-info-crossmark" class="crossmark__link" style="cursor: pointer;"><svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 290 58" height="22px" role="presentation"><path fill="url(#horizontal_crossmark)" stroke="#948F8F" stroke-miterlimit="10" stroke-width=".75" d="M287 1H3a2 2 0 0 0-2 2v52a2 2 0 0 0 2 2h284a2 2 0 0 0 2-2V3a2 2 0 0 0-2-2Z"></path><path fill="#535353" d="M80.3 23.58a6.395 6.395 0 0 0-.55-1.43 3.92 3.92 0 0 0-2.19-1.86 5.099 5.099 0 0 0-1.71-.29 5 5 0 0 0-2.46.58 4.831 4.831 0 0 0-1.69 1.54 6.921 6.921 0 0 0-1 2.19 10.16 10.16 0 0 0 0 5 6.921 6.921 0 0 0 1 2.19 4.83 4.83 0 0 0 1.69 1.5 5 5 0 0 0 2.46.58 4.44 4.44 0 0 0 1.85-.36 4.12 4.12 0 0 0 1.38-1c.39-.436.695-.942.9-1.49A7.094 7.094 0 0 0 80.4 29h3a8.46 8.46 0 0 1-.69 2.92 7.179 7.179 0 0 1-1.59 2.29 7 7 0 0 1-2.35 1.49 8.14 8.14 0 0 1-3 .52 8.43 8.43 0 0 1-3.59-.74 7.88 7.88 0 0 1-2.69-2 9.001 9.001 0 0 1-1.69-3 11.41 11.41 0 0 1 0-7.23 9.001 9.001 0 0 1 1.69-3 8.001 8.001 0 0 1 2.69-2 8.3 8.3 0 0 1 3.59-.75 9.44 9.44 0 0 1 2.83.41c.866.242 1.68.642 2.4 1.18A6.229 6.229 0 0 1 82.65 21a6.66 6.66 0 0 1 .78 2.58H80.3ZM85.9 17.9h2.85v6.63h.05a4.15 4.15 0 0 1 1.56-1.46 4.61 4.61 0 0 1 2.31-.59 4.83 4.83 0 0 1 3.33 1.1 4.21 4.21 0 0 1 1.22 3.3v8.88H94.4v-8.13a3.51 3.51 0 0 0-.65-2.21 2.36 2.36 0 0 0-1.87-.69 3.1 3.1 0 0 0-1.3.26 2.821 2.821 0 0 0-1 .73A3.41 3.41 0 0 0 89 26.8a3.88 3.88 0 0 0-.23 1.33v7.63H85.9V17.9ZM102.47 30c-.002.491.069.98.21 1.45.131.448.348.866.64 1.23a3.08 3.08 0 0 0 1.08.84 3.543 3.543 0 0 0 1.55.31 3.431 3.431 0 0 0 2-.54c.541-.402.94-.966 1.14-1.61h2.7A5.349 5.349 0 0 1 109.7 35a5.715 5.715 0 0 1-1.74.85 7 7 0 0 1-2 .29 6.848 6.848 0 0 1-2.7-.5 5.577 5.577 0 0 1-2-1.4A5.953 5.953 0 0 1 100 32a8.748 8.748 0 0 1-.41-2.75 7.768 7.768 0 0 1 .44-2.61 6.67 6.67 0 0 1 1.25-2.17 5.863 5.863 0 0 1 4.56-2 5.79 5.79 0 0 1 2.74.64c.783.41 1.466.986 2 1.69a6.824 6.824 0 0 1 1.16 2.41c.264.905.352 1.852.26 2.79h-9.53Zm6.65-1.87a4.605 4.605 0 0 0-.29-1.3c-.148-.4-.368-.77-.65-1.09a3.22 3.22 0 0 0-1-.75 3 3 0 0 0-1.31-.29 3.431 3.431 0 0 0-1.36.26 3.108 3.108 0 0 0-1 .73 3.594 3.594 0 0 0-.7 1.09 3.798 3.798 0 0 0-.29 1.35h6.65-.05ZM122.88 27.15a2.666 2.666 0 0 0-1-1.81 3.14 3.14 0 0 0-2-.61c-.4.005-.798.07-1.18.19-.435.136-.83.376-1.15.7a3.92 3.92 0 0 0-.87 1.43 6.783 6.783 0 0 0-.35 2.39c.002.539.065 1.076.19 1.6.113.509.316.993.6 1.43.273.41.635.752 1.06 1a3 3 0 0 0 1.58.39 2.872 2.872 0 0 0 2.06-.77 3.596 3.596 0 0 0 1-2.18h2.85a6.26 6.26 0 0 1-1.91 3.86 5.83 5.83 0 0 1-4 1.34 6.61 6.61 0 0 1-2.69-.51 5.695 5.695 0 0 1-2-1.4 6.001 6.001 0 0 1-1.21-2.11 8.279 8.279 0 0 1-.41-2.65 8.987 8.987 0 0 1 .4-2.72 6.237 6.237 0 0 1 1.2-2.21 5.614 5.614 0 0 1 2-1.47 6.7 6.7 0 0 1 2.79-.54 7.801 7.801 0 0 1 2.14.29 5.634 5.634 0 0 1 1.8.86c.529.386.969.88 1.29 1.45a5 5 0 0 1 .6 2.07h-2.85l.06-.02ZM128.13 17.9H131V28l5.15-5.22h3.5l-5 4.75 5.42 8.17h-3.47l-4-6.27-1.6 1.7v4.63h-2.85V17.9h-.02ZM147.15 22.83h2.13v-1.08a5.505 5.505 0 0 1 .3-2c.161-.456.436-.862.8-1.18.33-.273.721-.462 1.14-.55.44-.094.89-.141 1.34-.14a8.158 8.158 0 0 1 2 .18v2.24a4.113 4.113 0 0 0-.59-.11 6.343 6.343 0 0 0-.74 0 1.74 1.74 0 0 0-1 .27 1.204 1.204 0 0 0-.41 1.08v1.33h2.42V25h-2.42v10.8h-2.85V25h-2.12v-2.17ZM162.23 36.1a6.999 6.999 0 0 1-2.76-.51 5.877 5.877 0 0 1-2-1.41 5.998 5.998 0 0 1-1.33-2.18 8.775 8.775 0 0 1 0-5.48 6.008 6.008 0 0 1 1.27-2.15 5.894 5.894 0 0 1 2-1.41A7.71 7.71 0 0 1 165 23a5.894 5.894 0 0 1 2 1.41 6.003 6.003 0 0 1 1.28 2.15c.585 1.78.585 3.7 0 5.48a5.99 5.99 0 0 1-3.28 3.54 7.002 7.002 0 0 1-2.77.52Zm0-2.25a3.268 3.268 0 0 0 1.65-.4c.447-.262.83-.621 1.12-1.05.304-.444.528-.938.66-1.46a6.581 6.581 0 0 0 0-3.31 4.416 4.416 0 0 0-.66-1.46 3.526 3.526 0 0 0-1.15-1 3.6 3.6 0 0 0-3.3 0 3.526 3.526 0 0 0-1.15 1 4.414 4.414 0 0 0-.66 1.46 6.558 6.558 0 0 0 0 3.31c.132.522.356 1.016.66 1.46a3.47 3.47 0 0 0 1.15 1.05 3.26 3.26 0 0 0 1.68.4ZM171.13 22.83h2.67v2.5c.097-.362.264-.701.49-1 .251-.341.547-.647.88-.91a4.404 4.404 0 0 1 1.14-.66 3.41 3.41 0 0 1 1.28-.25c.23-.013.46-.013.69 0h.39v2.79l-.61-.09a5.173 5.173 0 0 0-.61 0 3.278 3.278 0 0 0-1.36.29 3.131 3.131 0 0 0-1.11.85 4.171 4.171 0 0 0-.75 1.39 6 6 0 0 0-.28 1.9v6.15h-2.85V22.83h.03ZM198.22 35.75h-2.8V34a3.907 3.907 0 0 1-1.56 1.56 4.25 4.25 0 0 1-2.11.59 4.742 4.742 0 0 1-3.75-1.31 5.537 5.537 0 0 1-1.12-3.84v-8.2h2.85v7.92a3.503 3.503 0 0 0 .65 2.4 2.38 2.38 0 0 0 1.83.7 3.599 3.599 0 0 0 1.5-.27 2.66 2.66 0 0 0 1-.74c.259-.326.444-.705.54-1.11a5.79 5.79 0 0 0 .16-1.4v-7.5h2.85v12.95h-.04ZM201.3 22.83h2.7v1.75a3.523 3.523 0 0 1 1.72-1.58 5.568 5.568 0 0 1 2.33-.49 6.17 6.17 0 0 1 2.66.54 5.246 5.246 0 0 1 1.89 1.47c.515.65.899 1.394 1.13 2.19.257.87.385 1.773.38 2.68.001.851-.114 1.7-.34 2.52a6.568 6.568 0 0 1-1 2.16 5.002 5.002 0 0 1-4.21 2.06c-.419 0-.837-.036-1.25-.11a5.39 5.39 0 0 1-1.2-.36A4.737 4.737 0 0 1 205 35a3.567 3.567 0 0 1-.81-.91v6.45h-2.89V22.83Zm10 6.47c0-.574-.078-1.146-.23-1.7a4.606 4.606 0 0 0-.67-1.46 3.475 3.475 0 0 0-1.12-1 3.06 3.06 0 0 0-1.55-.39A3.141 3.141 0 0 0 205 26a5.518 5.518 0 0 0-1 3.3c-.004.611.077 1.22.24 1.81.138.523.38 1.012.71 1.44a3.55 3.55 0 0 0 1.14 1 3.242 3.242 0 0 0 1.54.35 3.18 3.18 0 0 0 1.65-.4c.438-.25.817-.59 1.11-1 .299-.44.513-.931.63-1.45a7.38 7.38 0 0 0 .23-1.75h.05ZM228.67 35.75H226V34a3.38 3.38 0 0 1-1.67 1.61 5.668 5.668 0 0 1-2.33.49 6.17 6.17 0 0 1-2.66-.54 5.307 5.307 0 0 1-1.89-1.46 6.254 6.254 0 0 1-1.12-2.19 9.48 9.48 0 0 1-.37-2.71 8.67 8.67 0 0 1 .47-3 6.058 6.058 0 0 1 1.26-2.1 4.89 4.89 0 0 1 1.8-1.21 5.738 5.738 0 0 1 2.06-.39c.412 0 .824.037 1.23.11.413.073.815.194 1.2.36.38.166.736.381 1.06.64.32.255.594.562.81.91V17.9h2.85v17.85h-.03Zm-10-6.33c-.001.55.069 1.098.21 1.63.13.511.35.996.65 1.43.288.414.668.757 1.11 1a3.241 3.241 0 0 0 1.6.38 3.199 3.199 0 0 0 1.64-.4 3.398 3.398 0 0 0 1.13-1.05c.297-.447.517-.94.65-1.46.14-.542.211-1.1.21-1.66a5.227 5.227 0 0 0-1-3.35 3.175 3.175 0 0 0-2.61-1.2 3.23 3.23 0 0 0-1.69.41 3.478 3.478 0 0 0-1.12 1.07 4.494 4.494 0 0 0-.62 1.5 7.871 7.871 0 0 0-.11 1.7h-.05ZM242.6 32.88c-.018.258.03.516.14.75a.577.577 0 0 0 .54.23h.3c.135-.002.269-.019.4-.05v2l-.39.11-.49.08-.5.08h-.42a2.75 2.75 0 0 1-1.45-.35 1.731 1.731 0 0 1-.75-1.23 4.998 4.998 0 0 1-2.09 1.2 8.224 8.224 0 0 1-2.39.38 5.861 5.861 0 0 1-1.67-.24 4.394 4.394 0 0 1-1.41-.7 3.35 3.35 0 0 1-1-1.18 3.624 3.624 0 0 1-.36-1.66 3.84 3.84 0 0 1 .44-2 3.246 3.246 0 0 1 1.15-1.17 5.195 5.195 0 0 1 1.6-.61c.593-.127 1.19-.223 1.79-.29a14.77 14.77 0 0 1 1.47-.21 6.896 6.896 0 0 0 1.24-.21c.318-.077.611-.236.85-.46a1.286 1.286 0 0 0 .31-.94 1.458 1.458 0 0 0-.26-.9A1.759 1.759 0 0 0 239 25a2.807 2.807 0 0 0-.86-.25 6.941 6.941 0 0 0-.9-.06 3.58 3.58 0 0 0-2 .5 2 2 0 0 0-.87 1.55h-2.85c.021-.73.227-1.442.6-2.07.338-.54.797-.996 1.34-1.33a5.475 5.475 0 0 1 1.84-.7c.692-.134 1.395-.2 2.1-.2.632 0 1.262.067 1.88.2a5.337 5.337 0 0 1 1.66.65 3.6 3.6 0 0 1 1.19 1.16c.314.525.47 1.129.45 1.74v6.65l.02.04Zm-2.85-3.6a3.45 3.45 0 0 1-1.6.51c-.667.06-1.3.147-1.9.26a6.137 6.137 0 0 0-.87.21 2.64 2.64 0 0 0-.75.38c-.22.164-.395.38-.51.63a2.129 2.129 0 0 0-.19.94c-.011.29.085.575.27.8.182.215.407.388.66.51.269.128.555.216.85.26.277.05.558.076.84.08a4.63 4.63 0 0 0 1-.12 3.418 3.418 0 0 0 1-.43 2.73 2.73 0 0 0 .81-.76 1.93 1.93 0 0 0 .32-1.14v-2.14l.07.01ZM244.37 22.83h2.15V19h2.85v3.88h2.58V25h-2.58v6.9a7.064 7.064 0 0 0 0 .77c.016.194.074.381.17.55a.84.84 0 0 0 .41.34 2 2 0 0 0 .75.11h.6c.203-.01.404-.04.6-.09v2.2l-.93.1a8.403 8.403 0 0 1-.92 0 6.33 6.33 0 0 1-1.81-.21 2.421 2.421 0 0 1-1.08-.62 2.15 2.15 0 0 1-.52-1 7.791 7.791 0 0 1-.16-1.42V25h-2.15v-2.17h.04ZM256.22 30c-.002.491.069.98.21 1.45.131.448.348.866.64 1.23a3.08 3.08 0 0 0 1.08.84 3.543 3.543 0 0 0 1.55.31 3.431 3.431 0 0 0 2-.54c.541-.402.94-.966 1.14-1.61h2.7a5.352 5.352 0 0 1-2.1 3.26 5.715 5.715 0 0 1-1.74.85 7 7 0 0 1-2 .29 6.848 6.848 0 0 1-2.7-.5 5.577 5.577 0 0 1-2-1.4 5.947 5.947 0 0 1-1.21-2.18 8.754 8.754 0 0 1-.41-2.75 7.758 7.758 0 0 1 .44-2.61 6.67 6.67 0 0 1 1.25-2.17 5.863 5.863 0 0 1 4.56-2 5.79 5.79 0 0 1 2.74.64c.783.41 1.466.986 2 1.69a6.824 6.824 0 0 1 1.16 2.41c.243.916.308 1.87.19 2.81h-9.5V30Zm6.65-1.87a4.605 4.605 0 0 0-.29-1.3c-.148-.4-.368-.77-.65-1.09a3.22 3.22 0 0 0-1-.75 3 3 0 0 0-1.31-.29 3.431 3.431 0 0 0-1.36.26 3.108 3.108 0 0 0-1 .73 3.575 3.575 0 0 0-.7 1.09 3.798 3.798 0 0 0-.29 1.35h6.65-.05ZM270 31.6a2.154 2.154 0 0 0 .95 1.75c.607.35 1.3.524 2 .5.304-.002.608-.022.91-.06a3.943 3.943 0 0 0 1-.24c.286-.105.54-.28.74-.51a1.23 1.23 0 0 0 .26-.89 1.261 1.261 0 0 0-.4-.9 2.779 2.779 0 0 0-1-.56 9.315 9.315 0 0 0-1.34-.36l-1.5-.33a14.219 14.219 0 0 1-1.54-.42 4.937 4.937 0 0 1-1.33-.68 3.12 3.12 0 0 1-.94-1.09 3.472 3.472 0 0 1-.35-1.64 2.94 2.94 0 0 1 .51-1.76 3.913 3.913 0 0 1 1.3-1.15 5.66 5.66 0 0 1 1.75-.61 10.316 10.316 0 0 1 1.84-.17 8.418 8.418 0 0 1 1.91.21 5.19 5.19 0 0 1 1.65.69c.496.318.916.741 1.23 1.24.341.559.55 1.188.61 1.84h-3a1.783 1.783 0 0 0-.94-1.37 3.892 3.892 0 0 0-1.69-.35 6.093 6.093 0 0 0-.71 0 3.327 3.327 0 0 0-.78.19 1.642 1.642 0 0 0-.61.4 1 1 0 0 0-.25.69 1.093 1.093 0 0 0 .36.85c.294.245.634.428 1 .54.437.153.885.274 1.34.36l1.55.33a20.9 20.9 0 0 1 1.53.42c.479.157.931.386 1.34.68.41.277.751.643 1 1.07.255.494.379 1.045.36 1.6a3.48 3.48 0 0 1-.52 2 4.15 4.15 0 0 1-1.36 1.3 6.19 6.19 0 0 1-1.86.73 9.435 9.435 0 0 1-2 .23 8.823 8.823 0 0 1-2.26-.27 5.265 5.265 0 0 1-1.8-.84 4 4 0 0 1-1.2-1.4 4.445 4.445 0 0 1-.46-2h2.7v-.02Z"></path><path fill="#C72914" d="m21.94 38.74 18.12-12.08v-14.5H21.94v26.58Z"></path><path fill="#EF3340" d="M40.06 38.74 21.94 26.66v-14.5h18.12v26.58Z"></path><path fill="#3EB1C8" d="M31 9a20 20 0 1 0 0 40 20 20 0 0 0 0-40Zm0 35.1a15.1 15.1 0 1 1 0-30.2 15.1 15.1 0 0 1 0 30.2Z"></path><path fill="#FFC72C" d="M42.09 39.23A15.09 15.09 0 0 1 17 23.38l-4.2-2.64a20 20 0 0 0 33.49 21.14l-4.2-2.65Z"></path><defs><linearGradient id="horizontal_crossmark" x1="145" x2="145" y1="7.33" y2="56.82" gradientUnits="userSpaceOnUse"><stop stop-color="#fff"></stop><stop offset="1" stop-color="#c4c4c4"></stop></linearGradient></defs></svg></a></div><div class="core-content-info"></div></div></div></div></div></header><div class="core-header-aside">
    

        


    

            

            

            <div class="cover-image cover-image--axel"><div class="cover-image__thumb"><a href="https://www.cell.com/cell/issue?pii=S0092-8674(20)X0016-X"><img src="/cms/asset/59a1d9c0-104f-464d-9739-60297f68fd41/cov200h.gif" alt="Cover Image - Cell, Volume 184, Issue 15"></a></div></div>

            

        
</div></div><div data-core-wrapper="content"><div id="dummy-sticky-observer"></div><div data-core-nav="header" data-extent="frontmatter"><div class="core-nav-wrapper core-container" aria-label="Article navigation"><div class="navigation-menu-pdf">
    

        


    

            

            

            <div data-widget-id="ecff07e8-0bc4-4533-b1f1-49365e2ead20" data-widget-name="UX3ArticleToolsWidget" class="article-tools"><a href="/action/showPdf?pii=S0092-8674%2821%2900753-4" title="Standard PDF (opens in a new tab)" target="_blank" aria-hidden="false" role="menuitem" class="article-tools__item__pdf"><svg focusable="false" viewBox="0 0 32 32" width="20" height="20" title="PDF" class="pdf-icon"><path id="pdfwhite" d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path id="pdfda0000" d="M.167 2.592H22.39V9.72H.166z" fill="#da0000"></path><path id="pdffff9f9" fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path id="pdff91d0a" d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="basic">Download PDF</span><span class="alternative">Download PDF</span></a></div>

            

        
</div><div class="core-sections-menu"><button data-id="article-toolbar-showhide" aria-controls="article_sections_menu" aria-expanded="false" aria-haspopup="true" aria-label="Toggle section navigation menu" style="margin-top: 0px;"><i class="icon-list" aria-hidden="true"></i><span>Outline</span></button><nav id="article_sections_menu" aria-label="Contents" data-core-nav="article" style="margin-top: 0px;"><div class="heading">Outline<button aria-haspopup="true" aria-controls="article_sections_menu" title="Close outline menu"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div><ul><li><a href="#author-highlights-abstract">Highlights</a></li><li><a href="#author-abstract">Summary</a></li><li><a href="#graphical-abstract">Graphical abstract</a></li><li><a href="#keywords">Keywords</a></li><li><a href="#sec-1">Introduction</a></li><li><a href="#sec-2">Results</a></li><li><a href="#sec-3">Discussion</a></li><li><a href="#sec-4">STAR★Methods</a></li><li><a href="#acknowledgments">Acknowledgments</a></li><li><a href="#supplementary-material">Supplemental information (2)</a></li><li><a href="#references">References</a></li><li role="none" class="mobile-only"><a role="menuitem" href="#core-collateral-metrics">Article metrics</a></li><li role="none" class="mobile-only"><a role="menuitem" href="#core-collateral-relatedArticles">Related Articles</a></li></ul></nav></div><div class="navigation-menu-share">
    

        


    

            

            

            <!-- Go to https://www.addtoany.com/buttons/customize/ to customize your tools --><script type="text/javascript" defer="defer" src="https://static.addtoany.com/menu/page.js"></script><script type="text/javascript">var a2a_config = a2a_config || {};
a2a_config.thanks = {
    postShare: false,
    ad: false,
};</script><script>var a2a_config = a2a_config || {};
let res = window.location.href.replace(/\(/g, "%28").replace(/\)/g, "%29");
a2a_config.templates = a2a_config.templates || {};

a2a_config.templates.email = {
    body: res
};
</script><div data-widget-id="5f978446-c80a-450c-a185-94d633df2db2" data-widget-name="UX3share" aria-hidden="false" role="group" class="article-tools__dropblock article-tools__share"><button aria-label="Toggle share menu" aria-expanded="false" aria-haspopup="true" role="menuitem" aria-controls="navigation_menu_share_block" aria-labelledby="share-label" class="article-tools__item__share"><i aria-hidden="true" class="icon-article-share"></i><span id="share-label">Share</span></button><div id="navigation_menu_share_block" role="menu" class="share__block axel-share article-tools__dropblock__holder"><h4 class="axel-share__title heading">Share<button title="Close share menu" aria-expanded="false" aria-haspopup="true" aria-controls="navigation_menu_share_block"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></h4><div class="axel-share__inner"><span class="sr-only">Share on</span><ul class="a2a__share-list rlist w-slide--list a2a a2a_kit a2a_kit_size_32 a2a_default_style" data-a2a-url="https://www.cell.com/cell/fulltext/S0092-8674%2821%2900753-4" style="line-height: 32px;"><li class="a2a_listitem_custom"><a role="link" title="Email" class="a2a_button_email" target="_blank" rel="nofollow noopener" href="/#email"><i aria-hidden="true" class="at-icon-wrapper icon-gizmo-email-plus"></i><span>Email</span></a></li><li class="a2a_listitem_custom"><a role="link" title="formerly known as Twitter" class="a2a_button_twitter" target="_blank" rel="nofollow noopener" href="/#twitter"><i aria-hidden="true" class="at-icon-wrapper icon-x-default"></i><span>X</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Facebook" class="a2a_button_facebook" target="_blank" rel="nofollow noopener" href="/#facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook-square-2"></i><span>Facebook</span></a></li><li class="a2a_listitem_custom"><a role="link" title="LinkedIn" class="a2a_button_linkedin" target="_blank" rel="nofollow noopener" href="/#linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin-square"></i><span>LinkedIn</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Sina Weibo" class="a2a_button_sina_weibo" target="_blank" rel="nofollow noopener" href="/#sina_weibo"><i aria-hidden="true" class="at-icon-wrapper icon-weibo"></i><span>Sina Weibo</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Mendeley" class="a2a_button_mendeley" target="_blank" rel="nofollow noopener" href="/#mendeley"><i aria-hidden="true" class="at-icon-wrapper icon-mendeley-2"></i><span>Mendeley</span></a></li><li class="a2a_listitem_custom"><a role="link" title="bluesky" class="a2a_button_bluesky" target="_blank" rel="nofollow noopener" href="/#bluesky"><i aria-hidden="true" class="at-icon-wrapper icon-bluesky-logo"></i><span>bluesky</span></a></li><div style="clear: both;"></div></ul><div class="extra-links-drop-zone">
    

        


    

            

            

            <div data-widget-id="f1a208d3-ac5e-4db2-9c3a-40ef97ddfa10" data-widget-name="UX3HTMLWidget"><a class="axel-share__extra" target="_blank" href="/personalize/addFavoritePublication?doi=10.1016/j.cell.2021.06.018">
	<i class="icon-reading-list" aria-hidden="true"></i>
	<span>Add to my reading list</span>
</a></div>

            

        
</div></div></div></div>

            

        
</div><div><button data-id="article-nav-menubar-more" aria-controls="article_more_menu" aria-expanded="false" aria-haspopup="true" aria-label="Toggle more menu"><i aria-hidden="true" class="icon-more-mobile"></i><span>More</span></button><div class="info-panel"><div class="info-panel__left-content"><div class="info-panel__article_tools info-panel__item"><div id="article_more_menu" aria-label="More menu" data-core-nav="more"><div class="heading">More<button title="Close more menu" aria-controls="article_more_menu" aria-haspopup="true"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div>
    

        


    

            

            

            <ul data-widget-id="183a17a2-87b5-4e6f-bb85-6b5cda068c89" data-widget-name="UX3ArticleToolsWidget" class="article-tools"><li class="article-tools__item article-tools__pdf"><a href="/action/showPdf?pii=S0092-8674%2821%2900753-4" title="Standard PDF (opens in a new tab)" target="_blank" aria-hidden="false" class="article-tools__ctrl article-tools__item__pdf"><svg focusable="false" viewBox="0 0 32 32" width="20" height="20" title="PDF" class="pdf-icon"><path id="pdfwhite" d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path id="pdfda0000" d="M.167 2.592H22.39V9.72H.166z" fill="#da0000"></path><path id="pdffff9f9" fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path id="pdff91d0a" d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="basic">Download PDF</span><span class="alternative">Download PDF</span></a></li><li><a href="https://www.cell.com/action/showCitFormats?doi=10.1016%2Fj.cell.2021.06.018&amp;pii=S0092-8674%2821%2900753-4" title="Cite" target="_blank" aria-label="Open Cite in new tab" class="article-tools__item__displayExportCitation"><i aria-hidden="true" class="icon-export-Citation"></i><span>Cite</span></a></li><li id="article_tools_share">
    

        


    

            

            

            <!-- Go to https://www.addtoany.com/buttons/customize/ to customize your tools --><script type="text/javascript" defer="defer" src="https://static.addtoany.com/menu/page.js"></script><script type="text/javascript">var a2a_config = a2a_config || {};
a2a_config.thanks = {
    postShare: false,
    ad: false,
};</script><script>var a2a_config = a2a_config || {};
let res = window.location.href.replace(/\(/g, "%28").replace(/\)/g, "%29");
a2a_config.templates = a2a_config.templates || {};

a2a_config.templates.email = {
    body: res
};
</script><div data-widget-id="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da" data-widget-name="UX3share" class="dropBlock article-tools__dropblock article-tools__share"><button data-db-target-for="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da" aria-expanded="false" title="open Share menu" class="article-tools__ctrl share__ctrl"><i aria-hidden="true" class="icon-article-share"></i><span>Share</span></button><div data-db-target-of="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da" id="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da_Pop" class="share__block axel-share dropBlock__holder article-tools__dropblock__holder"><h4 class="axel-share__title">Share</h4><div class="axel-share__inner"><span class="sr-only">Share on</span><ul class="a2a__share-list rlist w-slide--list a2a a2a_kit a2a_kit_size_32 a2a_default_style" data-a2a-url="https://www.cell.com/cell/fulltext/S0092-8674%2821%2900753-4" style="line-height: 32px;"><li class="a2a_listitem_custom"><a role="link" title="Email" class="a2a_button_email" target="_blank" rel="nofollow noopener" href="/#email"><i aria-hidden="true" class="at-icon-wrapper icon-gizmo-email-plus"></i><span>Email</span></a></li><li class="a2a_listitem_custom"><a role="link" title="formerly known as Twitter" class="a2a_button_twitter" target="_blank" rel="nofollow noopener" href="/#twitter"><i aria-hidden="true" class="at-icon-wrapper icon-x-default"></i><span>X</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Facebook" class="a2a_button_facebook" target="_blank" rel="nofollow noopener" href="/#facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook-square-2"></i><span>Facebook</span></a></li><li class="a2a_listitem_custom"><a role="link" title="LinkedIn" class="a2a_button_linkedin" target="_blank" rel="nofollow noopener" href="/#linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin-square"></i><span>LinkedIn</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Sina Weibo" class="a2a_button_sina_weibo" target="_blank" rel="nofollow noopener" href="/#sina_weibo"><i aria-hidden="true" class="at-icon-wrapper icon-weibo"></i><span>Sina Weibo</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Mendeley" class="a2a_button_mendeley" target="_blank" rel="nofollow noopener" href="/#mendeley"><i aria-hidden="true" class="at-icon-wrapper icon-mendeley-2"></i><span>Mendeley</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Bluesky" class="a2a_button_bluesky" target="_blank" rel="nofollow noopener" href="/#bluesky"><i aria-hidden="true" class="at-icon-wrapper icon-bluesky-logo"></i><span>Bluesky</span></a></li><div style="clear: both;"></div></ul><div class="extra-links-drop-zone">
    

        


    

            

            

            <div data-widget-id="ae9bf5c5-a2ee-4f5c-8e41-9b21638b1a9f" data-widget-name="UX3HTMLWidget"><a class="axel-share__extra" target="_blank" href="/personalize/addFavoritePublication?doi=10.1016/j.cell.2021.06.018">
	<i class="icon-reading-list" aria-hidden="true"></i>
	<span>Add to my reading list</span>
</a></div>

            

        
</div></div></div></div>

            

        
</li><li><a href="https://www.cell.com/action/addCitationAlert?doi=10.1016%2Fj.cell.2021.06.018" title="Set Alert" target="_blank" aria-label="Open Set Alert in new tab" class="article-tools__item__displayCreateCitationAlert"><i aria-hidden="true" class="icon-set-alert"></i><span>Set Alert</span></a></li><li><a href="https://s100.copyright.com/AppDispatchServlet?publisherName=ELS&amp;contentID=S0092867421007534&amp;orderBeanReset=true&amp;orderSource=Phoenix" title="Get Rights" target="_blank" aria-label="Open Get Rights in new tab" class="article-tools__item__request"><i aria-hidden="true" class="icon-get-rights"></i><span>Get Rights</span></a></li><li><a href="https://www.contentednet.com/salesforce-form/?pii=S0092867421007534&amp;issn=00928674&amp;source=jbs" title="Reprints" target="_blank" aria-label="Open Reprints in new tab" class="article-tools__item__reprints"><i aria-hidden="true" class="icon-article-reprints"></i><span>Reprints</span></a></li></ul>

            

        
</div></div><div class="info-panel__download-issue info-panel__item"><div class="article-header__download-full-issue"><button title="Download full issue" type="submit" formaction="/action/downloadMultiplePdf?action=downloadIssuePdf&amp;journal=cell&amp;volume=184&amp;issue=15&amp;pii=S0092-8674%2820%29X0016-X" class="btn--download-issue btn--cancel download-full-issue"><i class="icon-download-new"></i><span>Download Full Issue</span></button><div class="table-of-content__alert table-of-content__alert--download"><p>Download started</p><a href="#" class="table-of-content__alert--close">Ok</a></div></div></div></div><div class="info-panel__right-content"><div class="info-panel__navigation info-panel__item"><div class="content-navigation"><a href="/cell/fulltext/S0092-8674(21)00690-5" aria-label="Previous article" aria-disabled="false" class="content-navigation__prev"><i aria-hidden="true" class="icon-arrow_l"></i><span>Previous article</span></a><a href="/cell/fulltext/S0092-8674(21)00629-2" aria-label="Next article" aria-disabled="false" class="content-navigation__next"><span>Next article</span><i aria-hidden="true" class="icon-arrow_r"></i></a></div></div></div></div></div></div></div><div class="core-sections-menu-outer"><div class="core-sections-menu"><button aria-controls="article_sections_menu-desktop" aria-haspopup="true" aria-expanded="false" aria-label="Toggle sections menu" data-id="article-toolbar-showhide"><i class="icon-show-outline" aria-hidden="true"></i><span>Show Outline</span></button><nav id="article_sections_menu-desktop" aria-label="Contents" data-core-nav="article" style="margin-top: 0px;"><button aria-controls="article_sections_menu-desktop" aria-haspopup="true" aria-expanded="false" aria-label="Toggle sections menu" data-id="article-toolbar-showhide"><i class="icon-hide-outline" aria-hidden="true"></i><span>Hide Outline</span></button><ul><li><a href="#author-highlights-abstract" aria-current="true">Highlights</a></li><li><a href="#author-abstract" aria-current="false">Summary</a></li><li><a href="#graphical-abstract" aria-current="false">Graphical abstract</a></li><li><a href="#keywords" aria-current="false">Keywords</a></li><li><a href="#sec-1" aria-current="false">Introduction</a></li><li><a href="#sec-2" aria-current="false">Results</a></li><li><a href="#sec-3" aria-current="false">Discussion</a></li><li><a href="#sec-4" aria-current="false">STAR★Methods</a></li><li><a href="#acknowledgments" aria-current="false">Acknowledgments</a></li><li><a href="#supplementary-material" aria-current="false">Supplemental information (2)</a></li><li><a href="#references" aria-current="false">References</a></li><li role="none" class="mobile-only"><a role="menuitem" href="#core-collateral-metrics">Article metrics</a></li><li role="none" class="mobile-only"><a role="menuitem" href="#core-collateral-relatedArticles">Related Articles</a></li></ul></nav></div></div><div><div class="core-relations"><div class="relation"><div class="relation--head">Correction: </div><div class="relation--body">Errata<div class="content--title"><a href="/cell/fulltext/S0092-8674(21)01183-1">GABA-receptive microglia selectively sculpt developing inhibitory circuits</a></div><div class="content--contributors">Emilia Favuzzi, Shuhan Huang, Giuseppe A. Saldi ...</div><div class="content--publishDate"> October 28, 2021</div></div></div></div><div class="core-content-inner"><div class="core-linked-content"></div><div id="abstracts" data-extent="frontmatter"><div class="core-container"><section id="author-highlights-abstract" property="abstract" typeof="Text" role="doc-abstract"><h2 property="name">Highlights</h2><div id="abspara0020" role="paragraph"><div id="ulist0010" role="list"><div id="u0010" role="listitem"><div class="label">•</div><div class="content"><div id="p0010" role="paragraph">GABA-receptive microglia interact with inhibitory synapses during development</div></div></div><div id="u0015" role="listitem"><div class="label">•</div><div class="content"><div id="p0015" role="paragraph">Removing GABA<sub>B</sub>Rs from microglia alters inhibitory but not excitatory connectivity</div></div></div><div id="u0020" role="listitem"><div class="label">•</div><div class="content"><div id="p0020" role="paragraph">Synapse pruning genes are altered in GABA-receptive microglia lacking GABA<sub>B</sub>Rs</div></div></div><div id="u0025" role="listitem"><div class="label">•</div><div class="content"><div id="p0025" role="paragraph">Mice lacking microglial GABA<sub>B1</sub>Rs exhibit behavioral abnormalities</div></div></div></div></div></section><section id="author-abstract" property="abstract" typeof="Text" role="doc-abstract"><h2 property="name">Summary</h2><div id="abspara0010" role="paragraph">Microglia, the resident immune cells of the brain, have emerged as crucial regulators of synaptic refinement and brain wiring. However, whether the remodeling of distinct synapse types during development is mediated by specialized microglia is unknown. Here, we show that GABA-receptive microglia selectively interact with inhibitory cortical synapses during a critical window of mouse postnatal development. GABA initiates a transcriptional synapse remodeling program within these specialized microglia, which in turn sculpt inhibitory connectivity without impacting excitatory synapses. Ablation of GABA<sub>B</sub> receptors within microglia impairs this process and leads to behavioral abnormalities. These findings demonstrate that brain wiring relies on the selective communication between matched neuronal and glial cell types.</div></section><section id="graphical-abstract" property="abstract" typeof="Text" role="doc-abstract"><h2 property="name">Graphical abstract</h2><div id="abspara0015" role="paragraph"><div class="figure-wrap"><figure id="undfig1" class="graphic" data-figureviewer-ignore="true"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.06.018/asset/556f7b72-ab0b-4aa1-9bfc-c937df543e42/main.assets/fx1_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="fx1" alt="Graphical abstract undfig1" src="/cms/10.1016/j.cell.2021.06.018/asset/10709c4c-a8de-43f7-b674-b66116c03d5b/main.assets/fx1.jpg" height="375" width="375" data-viewer-src="/cms/10.1016/j.cell.2021.06.018/asset/556f7b72-ab0b-4aa1-9bfc-c937df543e42/main.assets/fx1_lrg.jpg" loading="lazy"></a></figure></div></div></section></div></div><div class="core-linked-content"></div><section id="keywords" property="keywords"><h2>Keywords</h2><ol><li><a href="/action/doSearch?AllField=&quot;microglia&quot;" alt="Search on this keyword" rel="nofollow">microglia</a></li><li><a href="/action/doSearch?AllField=&quot;interneurons&quot;" alt="Search on this keyword" rel="nofollow">interneurons</a></li><li><a href="/action/doSearch?AllField=&quot;inhibitory+synapses&quot;" alt="Search on this keyword" rel="nofollow">inhibitory synapses</a></li><li><a href="/action/doSearch?AllField=&quot;development&quot;" alt="Search on this keyword" rel="nofollow">development</a></li><li><a href="/action/doSearch?AllField=&quot;neurodevelopmental+disorders&quot;" alt="Search on this keyword" rel="nofollow">neurodevelopmental disorders</a></li><li><a href="/action/doSearch?AllField=&quot;glia-neuron+interactions&quot;" alt="Search on this keyword" rel="nofollow">glia-neuron interactions</a></li><li><a href="/action/doSearch?AllField=&quot;cerebral+cortex&quot;" alt="Search on this keyword" rel="nofollow">cerebral cortex</a></li></ol></section><div class="core-linked-content"></div><div class="core-linked-content"></div><section id="bodymatter" data-extent="bodymatter" property="articleBody" typeof="Text"><div class="core-container"><section id="sec-1"><h2>Introduction</h2><div id="p0030" role="paragraph">Brain function relies on interactions among diverse cell types. Microglia are the primary brain macrophages and play diverse roles in tissue defense during infection and injury (<span class="dropBlock reference-citations"><a id="body-ref-sref61" href="#" role="doc-biblioref" data-xml-rid="bib61" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref61" aria-controls="sref61">Ransohoff and Perry, 2009</a><div class="dropBlock__holder" data-db-target-of="sref61"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">63.</div><div>Ransohoff, R.M. ∙ Perry, V.H.</div><div><strong>Microglial physiology: unique stimuli, specialized responses</strong></div><div><em>Annu. Rev. Immunol.</em> 2009; <strong>27</strong>:119-145</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1146/annurev.immunol.021908.132528" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-67650966680" target="_blank" aria-label="Open Scopus in new tab">Scopus (1455)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19302036/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1146%2Fannurev.immunol.021908.132528&amp;pmid=19302036" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). In the healthy developing brain, microglia regulate a plethora of processes that impact the organization of neural circuits, including synapse pruning (<span class="dropBlock reference-citations"><a id="body-ref-sref75" href="#" role="doc-biblioref" data-xml-rid="bib75" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref75" aria-controls="sref75">Thion et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref75"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">77.</div><div>Thion, M.S. ∙ Ginhoux, F. ∙ Garel, S.</div><div><strong>Microglia and early brain development: An intimate journey</strong></div><div><em>Science.</em> 2018; <strong>362</strong>:185-189</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aat0474" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85054779519" target="_blank" aria-label="Open Scopus in new tab">Scopus (233)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30309946/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aat0474&amp;pmid=30309946" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref5" href="#" role="doc-biblioref" data-xml-rid="bib5" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref5" aria-controls="sref5">Bohlen et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">5.</div><div>Bohlen, C.J. ∙ Friedman, B.A. ∙ Dejanovic, B. ...</div><div><strong>Microglia in Brain Development, Homeostasis, and Neurodegeneration</strong></div><div><em>Annu. Rev. Genet.</em> 2019; <strong>53</strong>:263-288</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1146/annurev-genet-112618-043515" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85076062493" target="_blank" aria-label="Open Scopus in new tab">Scopus (107)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31518519/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1146%2Fannurev-genet-112618-043515&amp;pmid=31518519" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref53-1" href="#" role="doc-biblioref" data-xml-rid="bib53" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref53-1" aria-controls="sref53-1">Neniskyte and Gross, 2017</a><div class="dropBlock__holder" data-db-target-of="sref53-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">54.</div><div>Neniskyte, U. ∙ Gross, C.T.</div><div><strong>Errant gardeners: glial-cell-dependent synaptic pruning and neurodevelopmental disorders</strong></div><div><em>Nat. Rev. Neurosci.</em> 2017; <strong>18</strong>:658-670</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nrn.2017.110" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85031751357" target="_blank" aria-label="Open Scopus in new tab">Scopus (194)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28931944/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnrn.2017.110&amp;pmid=28931944" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref85-1" href="#" role="doc-biblioref" data-xml-rid="bib85" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref85-1" aria-controls="sref85-1">Wilton et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref85-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">87.</div><div>Wilton, D.K. ∙ Dissing-Olesen, L. ∙ Stevens, B.</div><div><strong>Neuron-Glia Signaling in Synapse Elimination</strong></div><div><em>Annu. Rev. Neurosci.</em> 2019; <strong>42</strong>:107-127</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1146/annurev-neuro-070918-050306" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_87_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85068806385" target="_blank" aria-label="Open Scopus in new tab">Scopus (191)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31283900/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1146%2Fannurev-neuro-070918-050306&amp;pmid=31283900" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Synapses exhibit a striking molecular and functional heterogeneity, the best example of which is the dichotomy between excitatory and inhibitory synapses that possess distinct molecular components and properties (<span class="dropBlock reference-citations"><a id="body-ref-sref22" href="#" role="doc-biblioref" data-xml-rid="bib22" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref22" aria-controls="sref22">Favuzzi and Rico, 2018</a><div class="dropBlock__holder" data-db-target-of="sref22"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">23.</div><div>Favuzzi, E. ∙ Rico, B.</div><div><strong>Molecular diversity underlying cortical excitatory and inhibitory synapse development</strong></div><div><em>Curr. Opin. Neurobiol.</em> 2018; <strong>53</strong>:8-15</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.conb.2018.03.011" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_23_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85046459653" target="_blank" aria-label="Open Scopus in new tab">Scopus (27)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29704699/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.conb.2018.03.011&amp;pmid=29704699" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref80" href="#" role="doc-biblioref" data-xml-rid="bib80" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref80" aria-controls="sref80">Vogels and Abbott, 2009</a><div class="dropBlock__holder" data-db-target-of="sref80"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">82.</div><div>Vogels, T.P. ∙ Abbott, L.F.</div><div><strong>Gating multiple signals through detailed balance of excitation and inhibition in spiking networks</strong></div><div><em>Nat. Neurosci.</em> 2009; <strong>12</strong>:483-491</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nn.2276" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_82_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-63649085510" target="_blank" aria-label="Open Scopus in new tab">Scopus (264)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19305402/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnn.2276&amp;pmid=19305402" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). These fundamental differences have profound implications for circuit function (<span class="dropBlock reference-citations"><a id="body-ref-sref68" href="#" role="doc-biblioref" data-xml-rid="bib68" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref68" aria-controls="sref68">Sohal and Rubenstein, 2019</a><div class="dropBlock__holder" data-db-target-of="sref68"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">70.</div><div>Sohal, V.S. ∙ Rubenstein, J.L.R.</div><div><strong>Excitation-inhibition balance as a framework for investigating mechanisms in neuropsychiatric disorders</strong></div><div><em>Mol. Psychiatry.</em> 2019; <strong>24</strong>:1248-1257</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41380-019-0426-0" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85065920522" target="_blank" aria-label="Open Scopus in new tab">Scopus (424)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31089192/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41380-019-0426-0&amp;pmid=31089192" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). However, whether microglia are generic effectors of synapse pruning or specialized microglia are able to discriminate between distinct synapse types is unknown.</div><div id="p0035" role="paragraph">Our understanding of microglia diversity in both development and disease has been greatly enhanced by the examination of their transcriptomic differences at the single cell level (<span class="dropBlock reference-citations"><a id="body-ref-sref31-1" href="#" role="doc-biblioref" data-xml-rid="bib31" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref31-1" aria-controls="sref31-1">Hammond et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref31-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">32.</div><div>Hammond, T.R. ∙ Dufort, C. ∙ Dissing-Olesen, L. ...</div><div><strong>Single-Cell RNA Sequencing of Microglia throughout the Mouse Lifespan and in the Injured Brain Reveals Complex Cell-State Changes</strong></div><div><em>Immunity.</em> 2019; <strong>50</strong>:253-271.e6</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.immuni.2018.11.004&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.immuni.2018.11.004&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85056738677" target="_blank" aria-label="Open Scopus in new tab">Scopus (1123)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30471926/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.immuni.2018.11.004&amp;pmid=30471926" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref35-1" href="#" role="doc-biblioref" data-xml-rid="bib35" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref35-1" aria-controls="sref35-1">Keren-Shaul et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref35-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">36.</div><div>Keren-Shaul, H. ∙ Spinrad, A. ∙ Weiner, A. ...</div><div><strong>A Unique Microglia Type Associated with Restricting Development of Alzheimer’s Disease</strong></div><div><em>Cell.</em> 2017; <strong>169</strong>:1276-1290.e17</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.cell.2017.05.018&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.cell.2017.05.018&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85020287843" target="_blank" aria-label="Open Scopus in new tab">Scopus (2741)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28602351/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2017.05.018&amp;pmid=28602351" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref36" href="#" role="doc-biblioref" data-xml-rid="bib36" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref36" aria-controls="sref36">Krasemann et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref36"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">37.</div><div>Krasemann, S. ∙ Madore, C. ∙ Cialic, R. ...</div><div><strong>The TREM2-APOE pathway drives the transcriptional phenotype of dysfunctional microglia in neurodegenerative diseases</strong></div><div><em>Immunity.</em> 2017; <strong>47</strong>:566-581.e9</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.immuni.2017.08.008&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.immuni.2017.08.008&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85031732784" target="_blank" aria-label="Open Scopus in new tab">Scopus (1491)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28930663/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.immuni.2017.08.008&amp;pmid=28930663" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref38-1" href="#" role="doc-biblioref" data-xml-rid="bib38" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref38-1" aria-controls="sref38-1">Li et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref38-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">39.</div><div>Li, Q. ∙ Cheng, Z. ∙ Zhou, L. ...</div><div><strong>Developmental Heterogeneity of Microglia and Brain Myeloid Cells Revealed by Deep Single-Cell RNA Sequencing</strong></div><div><em>Neuron.</em> 2019; <strong>101</strong>:207-223.e10</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.neuron.2018.12.006&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.neuron.2018.12.006&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85059299323" target="_blank" aria-label="Open Scopus in new tab">Scopus (572)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30606613/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.neuron.2018.12.006&amp;pmid=30606613" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref47-1" href="#" role="doc-biblioref" data-xml-rid="bib47" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref47-1" aria-controls="sref47-1">Masuda et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref47-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">48.</div><div>Masuda, T. ∙ Sankowski, R. ∙ Staszewski, O. ...</div><div><strong>Spatial and temporal heterogeneity of mouse and human microglia at single-cell resolution</strong></div><div><em>Nature.</em> 2019; <strong>566</strong>:388-392</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-0924-x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_48_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85061969567" target="_blank" aria-label="Open Scopus in new tab">Scopus (735)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30760929/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-0924-x&amp;pmid=30760929" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref48-1" href="#" role="doc-biblioref" data-xml-rid="bib48" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref48-1" aria-controls="sref48-1">Matcovitch-Natan et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref48-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">49.</div><div>Matcovitch-Natan, O. ∙ Winter, D.R. ∙ Giladi, A. ...</div><div><strong>Microglia development follows a stepwise program to regulate brain homeostasis</strong></div><div><em>Science.</em> 2016; <strong>353</strong>:aad8670</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aad8670" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-84976883162" target="_blank" aria-label="Open Scopus in new tab">Scopus (792)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27338705/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aad8670&amp;pmid=27338705" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Such analysis led to the discovery of disease-associated microglia (DAM), which act as universal immune sensors of neurodegeneration (<span class="dropBlock reference-citations"><a id="body-ref-sref18" href="#" role="doc-biblioref" data-xml-rid="bib18" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref18" aria-controls="sref18">Deczkowska et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref18"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">19.</div><div>Deczkowska, A. ∙ Keren-Shaul, H. ∙ Weiner, A. ...</div><div><strong>Disease-Associated Microglia: A Universal Immune Sensor of Neurodegeneration</strong></div><div><em>Cell.</em> 2018; <strong>173</strong>:1073-1081</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.cell.2018.05.003&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.cell.2018.05.003&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85046853468" target="_blank" aria-label="Open Scopus in new tab">Scopus (663)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29775591/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2018.05.003&amp;pmid=29775591" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). However, whether variations in microglial transcriptomes map onto differences in function in the healthy developing brain remains poorly understood. We explored the hypothesis that functional microglia diversity has evolved to ensure the selective pruning of excitatory versus inhibitory synapses. Thus far, the examination of microglia-mediated synaptic pruning has focused on excitatory synapses (<span class="dropBlock reference-citations"><a id="body-ref-sref55" href="#" role="doc-biblioref" data-xml-rid="bib55" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref55" aria-controls="sref55">Paolicelli et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref55"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">57.</div><div>Paolicelli, R.C. ∙ Bolasco, G. ∙ Pagani, F. ...</div><div><strong>Synaptic pruning by microglia is necessary for normal brain development</strong></div><div><em>Science.</em> 2011; <strong>333</strong>:1456-1458</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.1202529" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-80052633284" target="_blank" aria-label="Open Scopus in new tab">Scopus (2844)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21778362/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.1202529&amp;pmid=21778362" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref64-1" href="#" role="doc-biblioref" data-xml-rid="bib64" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref64-1" aria-controls="sref64-1">Schafer et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref64-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">66.</div><div>Schafer, D.P. ∙ Lehrman, E.K. ∙ Kautzman, A.G. ...</div><div><strong>Microglia sculpt postnatal neural circuits in an activity and complement-dependent manner</strong></div><div><em>Neuron.</em> 2012; <strong>74</strong>:691-705</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.neuron.2012.03.026&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.neuron.2012.03.026&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-84861427387" target="_blank" aria-label="Open Scopus in new tab">Scopus (2664)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22632727/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.neuron.2012.03.026&amp;pmid=22632727" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). An association between microglia and inhibitory synapses has been suggested in the adult and under pathological conditions (<span class="dropBlock reference-citations"><a id="body-ref-sref12" href="#" role="doc-biblioref" data-xml-rid="bib12" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref12" aria-controls="sref12">Chen et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref12"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">12.</div><div>Chen, Z. ∙ Jalabi, W. ∙ Hu, W. ...</div><div><strong>Microglial displacement of inhibitory synapses provides neuroprotection in the adult brain</strong></div><div><em>Nat. Commun.</em> 2014; <strong>5</strong>:4486</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ncomms5486" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-84919455244" target="_blank" aria-label="Open Scopus in new tab">Scopus (218)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25047355/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fncomms5486&amp;pmid=25047355" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref41" href="#" role="doc-biblioref" data-xml-rid="bib41" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref41" aria-controls="sref41">Lui et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref41"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">42.</div><div>Lui, H. ∙ Zhang, J. ∙ Makinson, S.R. ...</div><div><strong>Progranulin Deficiency Promotes Circuit-Specific Synaptic Pruning by Microglia via Complement Activation</strong></div><div><em>Cell.</em> 2016; <strong>165</strong>:921-935</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.cell.2016.04.001&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.cell.2016.04.001&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-84963969178" target="_blank" aria-label="Open Scopus in new tab">Scopus (490)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27114033/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2016.04.001&amp;pmid=27114033" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>, <span class="dropBlock reference-citations"><a id="body-ref-sref40" href="#" role="doc-biblioref" data-xml-rid="bib40" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref40" aria-controls="sref40">Liu et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref40"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">41.</div><div>Liu, Y.-J. ∙ Spangenberg, E. ∙ Tang, B. ...</div><div><strong>Microglia elimination increases neural circuit connectivity and activity in adult mouse cortex</strong></div><div><em>J.&nbsp;Neurosci.</em> 2021; <strong>41</strong>:1274-1287</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1523/JNEUROSCI.2140-20.2020" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85102093729" target="_blank" aria-label="Open Scopus in new tab">Scopus (74)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33380470/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1523%2FJNEUROSCI.2140-20.2020&amp;pmid=33380470" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), the latter of which are often characterized by an aberrant reactivation of developmental programs (<span class="dropBlock reference-citations"><a id="body-ref-sref85-2" href="#" role="doc-biblioref" data-xml-rid="bib85" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref85-2" aria-controls="sref85-2">Wilton et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref85-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">87.</div><div>Wilton, D.K. ∙ Dissing-Olesen, L. ∙ Stevens, B.</div><div><strong>Neuron-Glia Signaling in Synapse Elimination</strong></div><div><em>Annu. Rev. Neurosci.</em> 2019; <strong>42</strong>:107-127</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1146/annurev-neuro-070918-050306" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_87_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85068806385" target="_blank" aria-label="Open Scopus in new tab">Scopus (191)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31283900/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1146%2Fannurev-neuro-070918-050306&amp;pmid=31283900" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). However, support for this hypothesis has been to date limited to studies in the embryonic brain, where prenatal immune challenges regulate the laminar positioning and connectivity of neocortical parvalbumin (PV) interneurons (<span class="dropBlock reference-citations"><a id="body-ref-sref70" href="#" role="doc-biblioref" data-xml-rid="bib70" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref70" aria-controls="sref70">Squarzoni et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref70"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">72.</div><div>Squarzoni, P. ∙ Oller, G. ∙ Hoeffel, G. ...</div><div><strong>Microglia modulate wiring of the embryonic forebrain</strong></div><div><em>Cell Rep.</em> 2014; <strong>8</strong>:1271-1279</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.celrep.2014.07.042&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.celrep.2014.07.042&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-84922519429" target="_blank" aria-label="Open Scopus in new tab">Scopus (475)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25159150/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.celrep.2014.07.042&amp;pmid=25159150" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref76-1" href="#" role="doc-biblioref" data-xml-rid="bib76" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref76-1" aria-controls="sref76-1">Thion et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref76-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">78.</div><div>Thion, M.S. ∙ Mosser, C.-A. ∙ Férézou, I. ...</div><div><strong>Biphasic Impact of Prenatal Inflammation and Macrophage Depletion on the Wiring of Neocortical Inhibitory Circuits</strong></div><div><em>Cell Rep.</em> 2019; <strong>28</strong>:1119-1126.e4</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_78_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.celrep.2019.06.086&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_78_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.celrep.2019.06.086&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_78_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85069572402" target="_blank" aria-label="Open Scopus in new tab">Scopus (35)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31365857/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.celrep.2019.06.086&amp;pmid=31365857" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Here, we demonstrate that GABA-receptive microglia remodel inhibitory but not excitatory synapses during mouse postnatal cortical development. The selectivity revealed by this process identifies specialized microglia dedicated to remodeling distinct synapse types.</div></section><section id="sec-2"><h2>Results</h2><section id="sec-2-1"><h3>Microglia depletion during cortical development alters inhibitory and excitatory synapse connectivity</h3><div id="p0040" role="paragraph">To investigate whether microglia are required for inhibitory synapse development, we examined inhibitory connectivity after depleting myeloid cells—including microglia—for the first two postnatal weeks. Daily injections of the colony-stimulating factor 1 receptor inhibitor PLX5622 efficiently depleted microglia beginning at postnatal day 4 (P4) (<a id="crosref0280" href="#fig1">Figures 1</a>A and 1B). We focused on the barrels in the mouse somatosensory (S1) cortex and its most abundant interneuron subtype, PV cells. At P15, PV inhibitory synapses onto excitatory neurons were increased in microglia-depleted mice compared to controls (<a id="crosref0285" href="#fig1">Figure&nbsp;1</a>C). Of note, this phenotype was not due to changes in the distribution or number of PV interneurons (<a id="crosref0290" href="#figs1">Figure&nbsp;S1</a>A). Upon microglia depletion, significant changes in PV synapses were detected only after their initial assembly (P12) (<a id="crosref0295" href="#figs1">Figures S1</a>B and S1C), suggesting a role for microglia in the maturation or refinement rather than formation of these connections. Conversely, the depletion of microglia during the third and fourth postnatal weeks did not alter the density of PV synapses (<a id="crosref0300" href="#figs1">Figures S1</a>C and S1D). Synapse development in different neocortical areas follows common principles but is asynchronous (<span class="dropBlock reference-citations"><a id="body-ref-sref57" href="#" role="doc-biblioref" data-xml-rid="bib57" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref57" aria-controls="sref57">Pinto et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref57"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">59.</div><div>Pinto, J.G.A. ∙ Jones, D.G. ∙ Murphy, K.M.</div><div><strong>Comparing development of synaptic proteins in rat visual, somatosensory, and frontal cortex</strong></div><div><em>Front. Neural Circuits.</em> 2013; <strong>7</strong>:97</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.3389/fncir.2013.00097" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-84878252546" target="_blank" aria-label="Open Scopus in new tab">Scopus (38)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23754984/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.3389%2Ffncir.2013.00097&amp;pmid=23754984" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Consistent with this, the increase in PV synapses upon microglia depletion was also observed in the visual cortex (V1), although the exact time window was shifted in accordance with its later development (<a id="crosref0310" href="#figs1">Figure&nbsp;S1</a>E). Notably, inhibitory synapses made by dendrite-targeting somatostatin (SST) interneurons were also increased in P15 microglia-depleted mice (<a id="crosref0315" href="#figs1">Figure&nbsp;S1</a>F). As for PV cells, the density of SST interneurons was unaltered (<a id="crosref0320" href="#figs1">Figure&nbsp;S1</a>G).</div><div class="figure-wrap"><figure id="fig1" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.06.018/asset/f99b38b5-f97a-4706-b4b8-1967246770fc/main.assets/gr1_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr1" src="/cms/10.1016/j.cell.2021.06.018/asset/94a3bdb6-d705-496e-aca0-ad19f4a097b0/main.assets/gr1.jpg" height="789" width="654" aria-labelledby="fig1-title fig1-content" data-viewer-src="/cms/10.1016/j.cell.2021.06.018/asset/f99b38b5-f97a-4706-b4b8-1967246770fc/main.assets/gr1_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig1-title" class="figure__title"><span class="label figure__label">Figure&nbsp;1</span> <span class="figure__title__text">Microglia depletion during cortical development alters inhibitory and excitatory synapse connectivity</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig1-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0015" role="paragraph">(A) Schematic of microglia depletion experiment.</div><div id="fspara0020" role="paragraph">(B) Image and density of Iba1<sup>+</sup> microglia in control and depleted mice (n&nbsp;= 3–8). <sup>∗∗∗</sup>p&nbsp;&lt; 0.001, Student's t test (P4) and one-way ANOVA, Sidak’s multiple comparisons test (P8–P15).</div><div id="fspara0025" role="paragraph">(C) Images, masks, and density of Syt2<sup>+</sup>Gephyrin<sup>+</sup> synapses made by PV cells onto excitatory neurons (NeuN) in P15 control and depleted mice (n&nbsp;= 5). <sup>∗</sup>p&nbsp;&lt; 0.05, Student’s t test.</div><div id="fspara0030" role="paragraph">(D) Schematic of optogenetics experiment, IPSC traces, and IPSC amplitude (n&nbsp;= 16 cells from 4 controls and n&nbsp;= 16 cells from 3 depleted mice). <sup>∗</sup>p&nbsp;&lt; 0.05, Mann-Whitney test.</div><div id="fspara0035" role="paragraph">(E and F) Traces, frequency, and amplitude of mIPSCs (n&nbsp;= 22 cells from 3 control and n&nbsp;= 25 cells from 3 depleted mice) and mEPSCs (n&nbsp;= 25 cells from 3 control and n&nbsp;= 28 cells from 3 depleted mice). <sup>∗</sup>p&nbsp;&lt; 0.05, <sup>∗∗</sup>p&nbsp;&lt; 0.01, ns p &gt; 0.05, Student’s t test.</div><div id="fspara0040" role="paragraph">(G) mEPSC/mIPSC ratio; n&nbsp;= 22 cells from 3 control mice (Ctl) and n&nbsp;= 25 cells from 3 depleted (Dpl) mice. ns p &gt; 0.05, Mann-Whitney test.</div><div id="fspara0045" role="paragraph">(H) Schematic, images, masks, and density of VGlut2<sup>+</sup>Homer1<sup>+</sup> synapses onto excitatory or PV cells in controls (n&nbsp;= 5-11) and depleted mice (n&nbsp;= 4-9) at P15. <sup>∗</sup>p&nbsp;&lt;&nbsp;0.05, Student’s t test.</div><div id="fspara0050" role="paragraph">(I) Schematic and legend for experiment in (J) and (K).</div><div id="fspara0055" role="paragraph">(J) Images and density of Syt2<sup>+</sup>Gephyrin<sup>+</sup> synapses made by PV cells onto excitatory neurons in P30 control and depleted mice (n&nbsp;= 6) after microglia repopulation. <sup>∗</sup>p&nbsp;&lt; 0.05, Student’s t test.</div><div id="fspara0060" role="paragraph">(K) Density of VGlut2<sup>+</sup>Homer1<sup>+</sup> synapses onto excitatory neurons in P30 control (n&nbsp;= 9) and depleted mice (n&nbsp;= 7) after microglia repopulation. <sup>∗</sup>p&nbsp;&lt; 0.05, Mann-Whitney test.</div><div id="fspara0065" role="paragraph">Full and empty arrowheads indicate colocalization and boutons not meeting criteria. Scale bars, 1&nbsp;μm except in (B) where it equals 100&nbsp;μm. Data are mean ± SEM. See also <a id="crosref1745" href="#figs1">Figure&nbsp;S1</a>.</div></div></div></div></figcaption></figure></div><div class="figure-wrap"><figure id="figs1" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.06.018/asset/448fdf3e-af59-45a9-ab83-d00773d20236/main.assets/figs1_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs1" src="/cms/10.1016/j.cell.2021.06.018/asset/ae9b3e47-1e06-4462-947e-fb3e3c768f47/main.assets/figs1.jpg" height="958" width="722" aria-labelledby="figs1-title figs1-content" data-viewer-src="/cms/10.1016/j.cell.2021.06.018/asset/448fdf3e-af59-45a9-ab83-d00773d20236/main.assets/figs1_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs1-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S1</span> <span class="figure__title__text">Consequences of early postnatal microglia depletion for cortical connectivity, related to <a id="crosref1840" href="#fig1">Figure&nbsp;1</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs1-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0410" role="paragraph">(A) Representative images and quantitation of the density and layer distribution of PV cells in control (n&nbsp;= 4) and microglia-depleted (n&nbsp;= 4) mice at P15 after P1-P15 microglia depletion. ns p &gt; 0.05, Student’s t test (for density) and One-Way ANOVA followed by Sidak’s multiple comparisons test (for layer distribution). Scale bar equal 100&nbsp;μm.</div><div id="fspara0415" role="paragraph">(B) Quantitation of Syt2+Gephyrin+ synapses made by PV cells onto L4 excitatory neurons in P10 control (n&nbsp;= 4) or depleted (n&nbsp;= 4) mice after P1-P10 microglia depletion and in P12 control (n&nbsp;= 3) or depleted (n&nbsp;= 5) mice after P1-P12 microglia depletion. ns p &gt; 0.05, One-Way ANOVA followed by Tukey’s multiple comparisons test.</div><div id="fspara0420" role="paragraph">(C) Representative images and quantitation of Iba1+ microglia density in control (n&nbsp;= 4) and microglia-depleted (n&nbsp;= 4) mice at P30 after P15-P30 microglia depletion. <sup>∗∗∗</sup>p&nbsp;&lt; 0.001, Student’s t test. Scale bar equal 100&nbsp;μm.</div><div id="fspara0425" role="paragraph">(D) Quantitation of Syt2+Gephyrin+ synapses made by PV cells onto L4 excitatory neurons in P30 control (n&nbsp;= 4) and depleted (n&nbsp;= 3) mice after P15-P30 microglia depletion. One brain with incomplete depletion was excluded from the analysis. ns p &gt; 0.05, Student’s t test.</div><div id="fspara0430" role="paragraph">(E) Representative images and quantitation of Syt2+Gephyrin+ synapses made by PV cells onto L4 excitatory neurons in the primary visual cortex (V1) of P15 control (n&nbsp;= 6) and depleted (n&nbsp;= 9) mice after P1-P15 microglia depletion and of P25 control (n&nbsp;= 12) and depleted (n&nbsp;= 10) mice after P1-P25 microglia depletion. ns p &gt; 0.05 and <sup>∗</sup>p&nbsp;&lt; 0.05, One-Way ANOVA followed by Tukey’s multiple comparisons test. Scale bar equal 2&nbsp;μm.</div><div id="fspara0435" role="paragraph">(F) Representative images and quantitation of synapses made in L1 of S1 by SST+ cells (<i>SST</i><sup><i>Cre</i></sup><i>;Ai34</i> Synaptophysin-tdTomato labeled SST+ presynaptic terminals colocalizing with Gephyrin) in P15 control (n&nbsp;= 8) and depleted (n&nbsp;= 6) mice after P1-P15 microglia depletion. <sup>∗</sup>p&nbsp;&lt; 0.05, Student’s t test. Scale bar equal 2&nbsp;μm.</div><div id="fspara0440" role="paragraph">(G) Quantitation of the density of SST cells in P15 control (n&nbsp;= 6) and microglia-depleted (n&nbsp;= 5) mice after P1-P15 microglia depletion. ns p &gt; 0.05, Mann-Whitney test.</div><div id="fspara0445" role="paragraph">(H) Traces, frequency and amplitude of sIPSCs (n&nbsp;= 14 cells from 3 control and n&nbsp;= 17 cells from 5 depleted mice). <sup>∗∗∗</sup>p&nbsp;&lt; 0.001, ns p &gt; 0.05, Student’s t test.</div><div id="fspara0450" role="paragraph">(I) Traces, frequency and amplitude of sEPSCs (n&nbsp;= 17 cells from 4 control and n&nbsp;= 18 cells from 5 depleted mice). <sup>∗∗∗</sup>p&nbsp;&lt; 0.001, ns p &gt; 0.05, Student’s t test.</div><div id="fspara0455" role="paragraph">(J) sEPSC/sIPSC ratio (n&nbsp;= 14 cells from 3 control and n&nbsp;= 15 cells from 3 depleted mice). ns p &gt; 0.05, Mann-Whitney test.</div><div id="fspara0460" role="paragraph">(K) Schematic of synapses analyzed in L and M.</div><div id="fspara0465" role="paragraph">(L) Representative images and quantitation of VGlut2+Homer1+ synapses onto excitatory cell dendrites (TdTomato+, labeled by virus injection of pyramidal neurons) in P15 control (n&nbsp;= 6) and depleted (n&nbsp;= 4) mice. <sup>∗∗</sup>p&nbsp;&lt; 0.01, Student’s t test. Scale bar equal 2&nbsp;μm.</div><div id="fspara0470" role="paragraph">(M) Quantitation of VGlut1+Homer1+ synapses made onto L4 excitatory cells (NeuN) in P15 control (n&nbsp;= 6) and depleted (n&nbsp;= 4) mice (left), onto PV cells in P15 control (n&nbsp;= 9) or depleted (n&nbsp;= 10) mice (middle) and onto excitatory cell dendrites (TdTomato+, labeled by virus injection of pyramidal neurons) in P15 control (n&nbsp;=&nbsp;7) or depleted (n&nbsp;= 4) mice. ns p &gt; 0.05, Mann-Whitney test for excitatory neurons and Student’s t test for PV cells and dendrites.</div><div id="fspara0475" role="paragraph">(N) Representative images of Iba1+ microglia in control and P1-P15 microglia-depleted brains at the indicated stages of repopulation and quantitation of microglia density in control and microglia-repopulated brains at P17, P19, P21, P25 and P30 (n&nbsp;= 3-4 mice per condition). <sup>∗∗∗</sup>p&nbsp;&lt; 0.001, ns p &gt; 0.05, One-Way ANOVA followed by Sidak’s multiple comparisons test. Scale bar equal 100&nbsp;μm.</div><div id="fspara0480" role="paragraph">(O) Quantitation of synapses made in L1 of S1 by SST+ cells (<i>SST</i><sup><i>Cre</i></sup><i>;Ai34</i> labeled presynaptic terminals colocalizing with Gephyrin) in P30 control (n&nbsp;= 4) and microglia-repopulated (n&nbsp;= 5) mice after P1-P15 microglia depletion. <sup>∗</sup>p&nbsp;&lt; 0.05, Student’s t test.</div><div id="fspara0485" role="paragraph">(P) Quantitation of Syt2+Gephyrin+ synapses (left), <i>SST</i><sup><i>Cre</i></sup><i>;Ai34</i>+Gephyrin+ synapses (middle) and VGlut2+Homer1+ synapses (right) made by PV, SST and thalamic cells in P60 control (n&nbsp;= 6-8) and microglia-repopulated (n&nbsp;= 4-7) mice after P1-P15 microglia depletion. ExCs: excitatory cell soma. ns p &gt; 0.05, Student’s t test for SST+ and thalamic synapses and Mann-Whitney test for PV synapses.</div><div id="fspara0490" role="paragraph">All data are mean ± SEM, each data point represents one experimental animal except in F, I and J where it represents one cell. Arrowheads indicate colocalization.</div></div></div></div></figcaption></figure></div><div id="p0045" role="paragraph">We next asked whether the structural increase in PV synapses was paralleled by a functional increase in PV inhibition. While recording from excitatory neurons, we stimulated PV cells expressing Channelrhodopsin-2 (<i>PV</i><sup><i>Cre</i>/+</sup><i>;Ai32</i>) and found that the amplitude of optogenetically evoked inhibitory postsynaptic currents (IPSCs) was significantly increased in microglia-depleted animals compared to controls (<a id="crosref0325" href="#fig1">Figure&nbsp;1</a>D). The increased inhibitory connectivity onto excitatory neurons in microglia-depleted mice was also confirmed by a higher frequency of miniature and spontaneous inhibitory synaptic currents (mIPSCs and sIPSCs) (<a id="crosref0330" href="#fig1">Figures 1</a>E and <a id="crosref0335" href="#figs1">S1</a>H). Together, these results demonstrate the existence of sequential and temporally restricted waves during which the maturation of cortical inhibitory circuits is regulated by microglia.</div><div id="p0050" role="paragraph">For comparison, we examined the impact of microglia depletion on glutamatergic connectivity. The frequency of miniature and spontaneous excitatory synaptic currents (mEPSCs and sEPSCs) was higher in P15 microglia-depleted mice compared to controls (<a id="crosref0340" href="#fig1">Figures 1</a>F and <a id="crosref0345" href="#figs1">S1</a>I). Importantly, developmental microglia depletion did not significantly alter miniature or spontaneous EPSC/IPSC frequency ratios (<a id="crosref0350" href="#fig1">Figures 1</a>G and <a id="crosref0355" href="#figs1">S1</a>J). Consistently, structural synapse analyses showed that both PV and excitatory neurons received more thalamocortical synapses in microglia-depleted mice (<a id="crosref0360" href="#fig1">Figures 1</a>H and <a id="crosref0365" href="#figs1">S1</a>K–S1M). Taken together, these experiments suggest that microglia regulate the development of both excitatory and inhibitory synapses.</div><div id="p0055" role="paragraph">We next asked whether the exuberant connectivity recovers once microglia are allowed to repopulate the brain. Within 48&nbsp;h after cessation of the P1–P15 depletion treatment, microglia sequentially repopulated subcortical and cortical regions. Cortical repopulation was complete by P21 (<a id="crosref0370" href="#fig1">Figures 1</a>I and <a id="crosref0375" href="#figs1">S1</a>N). At P30, the supernumerary inhibitory and excitatory synapses persisted (<a id="crosref0380" href="#fig1">Figures 1</a>J, 1K, and <a id="crosref0385" href="#figs1">S1</a>O). However, in the adult (P60), synapse density returned to control levels (<a id="crosref0390" href="#figs1">Figure&nbsp;S1</a>P). This indicates that depleting microglia during development causes long-lasting, albeit not permanent, defects in inhibitory and excitatory connectivity.</div></section><section id="sec-2-2"><h3>Microglia interact with inhibitory synapses during development</h3><div id="p0060" role="paragraph">The previous experiments are consistent with microglia refining inhibitory synapses during development. Microglia-mediated synapse remodeling has been posited to depend on distinct and sequential processes: chemotaxis, target recognition, and phagocytosis (<span class="dropBlock reference-citations"><a id="body-ref-sref53-2" href="#" role="doc-biblioref" data-xml-rid="bib53" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref53-2" aria-controls="sref53-2">Neniskyte and Gross, 2017</a><div class="dropBlock__holder" data-db-target-of="sref53-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">54.</div><div>Neniskyte, U. ∙ Gross, C.T.</div><div><strong>Errant gardeners: glial-cell-dependent synaptic pruning and neurodevelopmental disorders</strong></div><div><em>Nat. Rev. Neurosci.</em> 2017; <strong>18</strong>:658-670</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nrn.2017.110" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85031751357" target="_blank" aria-label="Open Scopus in new tab">Scopus (194)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28931944/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnrn.2017.110&amp;pmid=28931944" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref85-3" href="#" role="doc-biblioref" data-xml-rid="bib85" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref85-3" aria-controls="sref85-3">Wilton et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref85-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">87.</div><div>Wilton, D.K. ∙ Dissing-Olesen, L. ∙ Stevens, B.</div><div><strong>Neuron-Glia Signaling in Synapse Elimination</strong></div><div><em>Annu. Rev. Neurosci.</em> 2019; <strong>42</strong>:107-127</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1146/annurev-neuro-070918-050306" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_87_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85068806385" target="_blank" aria-label="Open Scopus in new tab">Scopus (191)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31283900/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1146%2Fannurev-neuro-070918-050306&amp;pmid=31283900" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). In response to various chemotactic signals, microglia are attracted to and interact with neurons and synapses (<span class="dropBlock reference-citations"><a id="body-ref-sref4-1" href="#" role="doc-biblioref" data-xml-rid="bib4" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref4-1" aria-controls="sref4-1">Badimon et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref4-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">4.</div><div>Badimon, A. ∙ Strasburger, H.J. ∙ Ayata, P. ...</div><div><strong>Negative feedback control of neuronal activity by microglia</strong></div><div><em>Nature.</em> 2020; <strong>586</strong>:417-423</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-020-2777-8" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85091727125" target="_blank" aria-label="Open Scopus in new tab">Scopus (443)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32999463/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-020-2777-8&amp;pmid=32999463" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref16-1" href="#" role="doc-biblioref" data-xml-rid="bib16" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref16-1" aria-controls="sref16-1">Cserép et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref16-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">17.</div><div>Cserép, C. ∙ Pósfai, B. ∙ Lénárt, N. ...</div><div><strong>Microglia monitor and protect neuronal function through specialized somatic purinergic junctions</strong></div><div><em>Science.</em> 2020; <strong>367</strong>:528-537</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aax6752" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85078742646" target="_blank" aria-label="Open Scopus in new tab">Scopus (330)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31831638/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aax6752&amp;pmid=31831638" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref45" href="#" role="doc-biblioref" data-xml-rid="bib45" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref45" aria-controls="sref45">Madry and Attwell, 2015</a><div class="dropBlock__holder" data-db-target-of="sref45"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">46.</div><div>Madry, C. ∙ Attwell, D.</div><div><strong>Receptors, ion channels, and signaling mechanisms underlying microglial dynamics</strong></div><div><em>J.&nbsp;Biol. Chem.</em> 2015; <strong>290</strong>:12443-12450</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1074%2Fjbc.R115.637157&amp;cf=fulltext&amp;site=jbc-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1074%2Fjbc.R115.637157&amp;cf=pdf&amp;site=jbc-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-84929378470" target="_blank" aria-label="Open Scopus in new tab">Scopus (65)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25855789/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1074%2Fjbc.R115.637157&amp;pmid=25855789" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). To test whether microglia directly interact with inhibitory synapses during development, we used confocal and stimulated emission depletion (STED) super-resolution microscopy (<a id="crosref0420" href="#fig2">Figure&nbsp;2</a>A). We generated mice expressing fluorescent reporters in both microglia and PV synaptic terminals. To this end, we injected <i>Cx3cr1</i><sup><i>GFP</i>/+</sup> mice with adeno-associated viruses (AAVs) expressing synaptophysin-tdTomato under the control of a PV-specific enhancer (<span class="dropBlock reference-citations"><a id="body-ref-sref82-1" href="#" role="doc-biblioref" data-xml-rid="bib82" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref82-1" aria-controls="sref82-1">Vormstein-Schneider et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref82-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">84.</div><div>Vormstein-Schneider, D. ∙ Lin, J.D. ∙ Pelkey, K.A. ...</div><div><strong>Viral manipulation of functionally distinct interneurons in mice, non-human primates and humans</strong></div><div><em>Nat. Neurosci.</em> 2020; <strong>23</strong>:1629-1636</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41593-020-0692-9" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_84_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85089867328" target="_blank" aria-label="Open Scopus in new tab">Scopus (100)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32807948/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41593-020-0692-9&amp;pmid=32807948" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). At P15, microglia contacted 10% of PV boutons and their processes ensheathed these presynaptic terminals. Moreover, the fraction of PV boutons contacted by microglia increased between P12 and P15, peaked at P15–P17, and decreased by P30 (<a id="crosref0430" href="#fig2">Figure&nbsp;2</a>B).</div><div class="figure-wrap"><figure id="fig2" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.06.018/asset/9ca2cc10-367f-4d91-aa34-580069e5254d/main.assets/gr2_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr2" src="/cms/10.1016/j.cell.2021.06.018/asset/002dc7fd-a5d8-4690-9442-9152f83f0363/main.assets/gr2.jpg" height="732" width="657" aria-labelledby="fig2-title fig2-content" data-viewer-src="/cms/10.1016/j.cell.2021.06.018/asset/9ca2cc10-367f-4d91-aa34-580069e5254d/main.assets/gr2_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig2-title" class="figure__title"><span class="label figure__label">Figure&nbsp;2</span> <span class="figure__title__text">Microglia interact with inhibitory synapses during development</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig2-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0075" role="paragraph">(A) Schematic of experiments in (B) to (D).</div><div id="fspara0080" role="paragraph">(B) Images and 3D reconstruction of microglia processes (<i>Cx3cr1</i><sup><i>GFP</i>/+</sup>) contacting PV boutons (PVe-Syp-dTom: synaptophysin-tdTomato under the control of a PV-specific enhancer). Fraction of PV boutons contacted by microglia (n&nbsp;= 3–8 mice). <sup>∗</sup>p&nbsp;&lt; 0.05, one-way ANOVA, Holm-Sidak multiple comparisons test.</div><div id="fspara0085" role="paragraph">(C) 3D reconstruction and fraction of PV boutons (PVe-Syp-Gamillus) encapsulated by microglia (<i>Tmem119</i><sup><i>CreER</i>/+</sup><i>;Ai14</i>) at P15 (n&nbsp;= 3 mice).</div><div id="fspara0090" role="paragraph">(D) Confocal image, 3D reconstruction, STED image and fraction of PV boutons (PVe-Syp-Gamillus) engulfed by microglial (<i>Tmem119</i><sup><i>Cre</i>/+</sup><i>;Ai14</i>) lysosomes (CD68) at P15 (n&nbsp;= 4 mice).</div><div id="fspara0095" role="paragraph">(E) Image, mask, and fraction of C1q<sup>+</sup> PV boutons (Syt2) in control and depleted mice (n&nbsp;= 4). ns, p &gt; 0.05; <sup>∗∗∗</sup>p&nbsp;&lt; 0.001, one-way ANOVA, Holm-Sidak multiple comparisons test.</div><div id="fspara0100" role="paragraph">(F) Images, masks, and density of Syt2<sup>+</sup>Gephyrin<sup>+</sup> synapses made by PV cells onto excitatory neurons (NeuN) in P15 control (n&nbsp;= 5) and C1q<sup>−/−</sup> (n&nbsp;= 6) mice. <sup>∗</sup>p&nbsp;&lt;&nbsp;0.05, Student’s t test.</div><div id="fspara0105" role="paragraph">(G and H) Traces, frequency, and amplitude of mIPSCs (n&nbsp;= 22 cells from 3 control and n&nbsp;= 22 cells from 3 C1q<sup>−/−</sup> mice) and mEPSCs (n&nbsp;= 21 cells from 3 control and n&nbsp;= 24 cells from 3 C1q<sup>−/−</sup> mice). <sup>∗∗</sup>p&nbsp;&lt; 0.01, <sup>∗∗∗</sup>p&nbsp;&lt; 0.001, ns p &gt; 0.05, Student’s t test except for mEPSC amplitude where Mann-Whitney test was used.</div><div id="fspara0110" role="paragraph">(I) mEPSC/mIPSC ratio (n&nbsp;= 21 cells from 3 control and n&nbsp;= 22 cells from 3 C1q<sup>−/−</sup> mice). ns, p &gt; 0.05; Student’s t test.</div><div id="fspara0115" role="paragraph">Scale bars, 1&nbsp;μm. Full and empty arrowheads indicate colocalization and boutons not meeting criteria. Data are mean ± SEM. See also <a id="crosref1750" href="#figs2">Figures S2</a> and <a id="crosref1755" href="#figs3">S3</a>.</div></div></div></div></figcaption></figure></div><div id="p0065" role="paragraph">The developmental interactions of microglia with excitatory synapses involve their phagocytic engulfment and elimination (<span class="dropBlock reference-citations"><a id="body-ref-sref53-3" href="#" role="doc-biblioref" data-xml-rid="bib53" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref53-3" aria-controls="sref53-3">Neniskyte and Gross, 2017</a><div class="dropBlock__holder" data-db-target-of="sref53-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">54.</div><div>Neniskyte, U. ∙ Gross, C.T.</div><div><strong>Errant gardeners: glial-cell-dependent synaptic pruning and neurodevelopmental disorders</strong></div><div><em>Nat. Rev. Neurosci.</em> 2017; <strong>18</strong>:658-670</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nrn.2017.110" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85031751357" target="_blank" aria-label="Open Scopus in new tab">Scopus (194)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28931944/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnrn.2017.110&amp;pmid=28931944" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref85-4" href="#" role="doc-biblioref" data-xml-rid="bib85" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref85-4" aria-controls="sref85-4">Wilton et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref85-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">87.</div><div>Wilton, D.K. ∙ Dissing-Olesen, L. ∙ Stevens, B.</div><div><strong>Neuron-Glia Signaling in Synapse Elimination</strong></div><div><em>Annu. Rev. Neurosci.</em> 2019; <strong>42</strong>:107-127</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1146/annurev-neuro-070918-050306" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_87_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85068806385" target="_blank" aria-label="Open Scopus in new tab">Scopus (191)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31283900/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1146%2Fannurev-neuro-070918-050306&amp;pmid=31283900" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). We therefore examined whether similar processes occur at inhibitory synapses. Because fluorescence quenching and protein degradation by lysosomal proteases may affect the detection of fluorescent proteins inside microglial lysosomes, we used the acid-tolerant monomeric green fluorescent protein Gamillus (<span class="dropBlock reference-citations"><a id="body-ref-sref34-1" href="#" role="doc-biblioref" data-xml-rid="bib34" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref34-1" aria-controls="sref34-1">Katayama et&nbsp;al., 2008</a><div class="dropBlock__holder" data-db-target-of="sref34-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">35.</div><div>Katayama, H. ∙ Yamamoto, A. ∙ Mizushima, N. ...</div><div><strong>GFP-like proteins stably accumulate in lysosomes</strong></div><div><em>Cell Struct. Funct.</em> 2008; <strong>33</strong>:1-12</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1247/csf.07011" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-44949131031" target="_blank" aria-label="Open Scopus in new tab">Scopus (179)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/18256512/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1247%2Fcsf.07011&amp;pmid=18256512" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref66-1" href="#" role="doc-biblioref" data-xml-rid="bib66" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref66-1" aria-controls="sref66-1">Shinoda et&nbsp;al., 2018a</a><div class="dropBlock__holder" data-db-target-of="sref66-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">68.</div><div>Shinoda, H. ∙ Ma, Y. ∙ Nakashima, R. ...</div><div><strong>Acid-Tolerant Monomeric GFP from Olindias formosa</strong></div><div><em>Cell Chem. Biol.</em> 2018; <strong>25</strong>:330-338.e7</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.chembiol.2017.12.005&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.chembiol.2017.12.005&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85042356070" target="_blank" aria-label="Open Scopus in new tab">Scopus (53)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29290624/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.chembiol.2017.12.005&amp;pmid=29290624" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>, <span class="dropBlock reference-citations"><a id="body-ref-sref67-1" href="#" role="doc-biblioref" data-xml-rid="bib67" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref67-1" aria-controls="sref67-1">2018b</a><div class="dropBlock__holder" data-db-target-of="sref67-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">69.</div><div>Shinoda, H. ∙ Shannon, M. ∙ Nagai, T.</div><div><strong>Fluorescent Proteins for Investigating Biological Events in Acidic Environments</strong></div><div><em>Int. J. Mol. Sci.</em> 2018; <strong>19</strong>:1548</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.3390/ijms19061548" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85047554713" target="_blank" aria-label="Open Scopus in new tab">Scopus (67)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29789517/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.3390%2Fijms19061548&amp;pmid=29789517" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). We injected AAVs expressing synaptophysin-Gamillus to label PV synaptic&nbsp;terminals in mice with genetically labeled microglia (<i>Tmem119</i><sup><i>CreER</i>/+</sup><i>;Ai14</i>) (<a id="crosref0460" href="#fig2">Figure&nbsp;2</a>A). At P15, a subset of these boutons was encapsulated within microglia and colocalized with microglial lysosomes (<a id="crosref0465" href="#fig2">Figures 2</a>C and 2D).</div><div id="p0070" role="paragraph">Classical complement proteins tag subsets of excitatory synapses for elimination by microglia (<span class="dropBlock reference-citations"><a id="body-ref-sref64-2" href="#" role="doc-biblioref" data-xml-rid="bib64" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref64-2" aria-controls="sref64-2">Schafer et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref64-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">66.</div><div>Schafer, D.P. ∙ Lehrman, E.K. ∙ Kautzman, A.G. ...</div><div><strong>Microglia sculpt postnatal neural circuits in an activity and complement-dependent manner</strong></div><div><em>Neuron.</em> 2012; <strong>74</strong>:691-705</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.neuron.2012.03.026&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.neuron.2012.03.026&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-84861427387" target="_blank" aria-label="Open Scopus in new tab">Scopus (2664)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22632727/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.neuron.2012.03.026&amp;pmid=22632727" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref72-1" href="#" role="doc-biblioref" data-xml-rid="bib72" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref72-1" aria-controls="sref72-1">Stevens et&nbsp;al., 2007</a><div class="dropBlock__holder" data-db-target-of="sref72-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">74.</div><div>Stevens, B. ∙ Allen, N.J. ∙ Vazquez, L.E. ...</div><div><strong>The classical complement cascade mediates CNS synapse elimination</strong></div><div><em>Cell.</em> 2007; <strong>131</strong>:1164-1178</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.cell.2007.10.036&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.cell.2007.10.036&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-36849076770" target="_blank" aria-label="Open Scopus in new tab">Scopus (2260)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/18083105/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2007.10.036&amp;pmid=18083105" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). We examined complement C1q accumulation on PV boutons and found that C1q was deposited on 10% of PV synaptic terminals at P15 (<a id="crosref0480" href="#fig2">Figure&nbsp;2</a>E). Next, we compared inhibitory and excitatory connectivity in controls versus <i>C1qa</i> knockout mice (C1q<sup>−/−</sup>). At P15, PV inhibitory synapses onto excitatory neurons were increased in C1q<sup>−/−</sup> mice (<a id="crosref0485" href="#fig2">Figure&nbsp;2</a>F). The frequency of both mIPSCs and mEPSCs was also increased (<a id="crosref0490" href="#fig2">Figures 2</a>G and 2H). As with microglia depletion, there was no significant change in the ratio of mEPSCs/mIPSC frequency received by each cell (<a id="crosref0495" href="#fig2">Figure&nbsp;2</a>I). Thus, C1q deficiency mimicked the defects observed in microglia-depleted mice, demonstrating that C1q is involved in regulating inhibitory connectivity.</div></section><section id="sec-2-3"><h3>GABA-receptive microglia preferentially interact with inhibitory versus excitatory synapses</h3><div id="p0075" role="paragraph">To visualize real-time interactions, we performed <i>in&nbsp;vivo</i> two-photon imaging of microglia and PV boutons during the peak contact period (P15–P17) in S1 (<a id="crosref0500" href="#fig3">Figures 3</a>A–3D and <a id="crosref0505" href="#figs2">S2</a>). The imaging experiments revealed a bimodal distribution in microglia-PV synapse dynamics. Microglia either contacted few PV puncta (14%) or engaged in interactions with the majority of PV boutons (60%) in their vicinity (<a id="crosref0510" href="#fig3">Figures 3</a>C, <a id="crosref0515" href="#figs2">S2</a>C, and S2I; <a id="crosref0520" href="#mmc3">Videos S1</a> and <a id="crosref0525" href="#mmc4">S2</a>). Of note, 1.8% of interactions involved microglia with phagocytic morphology (<a id="crosref0530" href="#figs2">Figure&nbsp;S2</a>G; <a id="crosref0535" href="#mmc5">Videos S3</a> and <a id="crosref0540" href="#mmc6">S4</a>). Among those microglia that engaged more with PV boutons, most interactions were sustained for periods ranging from 12 to 15&nbsp;min. By contrast, within the population that rarely contacted boutons, the interactions were of significantly shorter duration (<a id="crosref0545" href="#fig3">Figures 3</a>D, <a id="crosref0550" href="#figs2">S2</a>E, and S2K).</div><div class="figure-wrap"><figure id="fig3" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.06.018/asset/aa5a704e-8676-4815-bb39-c6d36b8a9511/main.assets/gr3_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr3" src="/cms/10.1016/j.cell.2021.06.018/asset/7622f4e2-3dc7-453d-8957-4e483babcf7f/main.assets/gr3.jpg" height="483" width="506" aria-labelledby="fig3-title fig3-content" data-viewer-src="/cms/10.1016/j.cell.2021.06.018/asset/aa5a704e-8676-4815-bb39-c6d36b8a9511/main.assets/gr3_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig3-title" class="figure__title"><span class="label figure__label">Figure&nbsp;3</span> <span class="figure__title__text">GABA-receptive microglia preferentially interact with inhibitory versus excitatory synapses</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig3-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0125" role="paragraph">(A) Schematic of <i>in&nbsp;vivo</i> imaging experiments and brain vasculature imaged through the cranial window. Scale bar, 500&nbsp;μm.</div><div id="fspara0130" role="paragraph">(B) Time-lapse images from <a id="crosref1760" href="#mmc3">Video S1</a> showing microglia contacting PV boutons. Scale bar, 10&nbsp;μm.</div><div id="fspara0135" role="paragraph">(C) Distribution of microglia contacting the indicated percentages of PV boutons over 20&nbsp;min (n&nbsp;=&nbsp;88 cells from 12 mice).</div><div id="fspara0140" role="paragraph">(D) Duration of contacts between microglia interacting with a minority (n&nbsp;= 24 cells) or majority (n&nbsp;=&nbsp;49 cells) of local PV boutons. <sup>∗∗</sup>p&nbsp;&lt; 0.01, Mann-Whitney test.</div><div id="fspara0145" role="paragraph">(E) Images (left: single plane; right: 2-μm stack) of microglia (<i>Cx3cr1</i><sup><i>GFP</i>/+</sup>) expressing <i>Gabbr1</i> and <i>Gabbr2</i> mRNA at P15 (smFISH). Scale bars, 10&nbsp;μm.</div><div id="fspara0150" role="paragraph">(F) Fraction of microglia expressing <i>Gabbr1</i>, <i>Gabbr2</i>, and both mRNAs at P15 (n&nbsp;= 7) in layer 4 of the primary somatosensory cortex (S1).</div><div id="fspara0155" role="paragraph">(G) 3D reconstruction and fraction of PV boutons (PVe-Syp-tdTom) contacted by <i>Gabbr2</i><sup>+</sup> and <i>Gabbr2</i><sup>−</sup> microglia (<i>Cx3cr1</i><sup><i>GFP</i>/+</sup>) at P15 (n&nbsp;= 58 <i>Gabbr2</i><sup>+</sup> and 59 <i>Gabbr2</i><sup>−</sup> cells from 4 mice). <sup>∗∗∗</sup>p&nbsp;&lt;&nbsp;0.001, Mann-Whitney test. Scale bar, 8&nbsp;μm.</div><div id="fspara0160" role="paragraph">(H) 3D reconstruction and fraction of VGlut2<sup>+</sup> boutons contacted by <i>Gabbr2</i><sup>+</sup> and <i>Gabbr2</i><sup>−</sup> microglia (<i>Cx3cr1</i><sup><i>GFP</i>/+</sup>) at P15 (n&nbsp;= 23 <i>Gabbr2</i><sup>+</sup> and 36 <i>Gabbr2</i><sup>−</sup> cells from 4 mice). <sup>∗∗</sup>p&nbsp;&lt; 0.01, Student’s t test. Scale bar, 10&nbsp;μm.</div><div id="fspara0165" role="paragraph">Arrowheads indicate colocalization. Data are mean ± SEM. In (D), thick and thin lines are median and quartiles. See also <a id="crosrefs0010" href="#figs2">Figures S2, S3, S4, S5</a> and <a id="crosref1765" href="#mmc3">Videos S1</a>, <a id="crosref1770" href="#mmc4">S2</a>, <a id="crosref1775" href="#mmc5">S3</a>, and <a id="crosref1780" href="#mmc6">S4</a>.</div></div></div></div></figcaption></figure></div><div class="figure-wrap"><figure id="figs2" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.06.018/asset/10b2335f-5822-4141-a820-747ca0869964/main.assets/figs2_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs2" src="/cms/10.1016/j.cell.2021.06.018/asset/48dcebfc-c4ff-436a-a513-1b04f7195189/main.assets/figs2.jpg" height="799" width="506" aria-labelledby="figs2-title figs2-content" data-viewer-src="/cms/10.1016/j.cell.2021.06.018/asset/10b2335f-5822-4141-a820-747ca0869964/main.assets/figs2_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs2-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S2</span> <span class="figure__title__text">Microglia-PV synapse interactions during development, related to <a id="crosref1845" href="#fig2">Figure&nbsp;2</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs2-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0500" role="paragraph">(A) Masks showing the criteria for contact identification in the <i>in&nbsp;vivo</i> imaging experiments. Percentages indicate the fraction of bouton area contacted by microglia.</div><div id="fspara0505" role="paragraph">(B) Proportion of PV boutons contacted by one microglia during the <i>in&nbsp;vivo</i> imaging experiments in deep layer 3 (n&nbsp;= 15 cells contacting 68 boutons) or layer 4 (n&nbsp;= 36 cells contacting 154 boutons). Data are the same as in (C) and (I) where they are shown as pooled dataset. ns p &gt; 0.05, Mann-Whitney test. Data are mean ± SEM, each data point represents one cell.</div><div id="fspara0510" role="paragraph">(C) Distribution of <i>Cx3cr1</i><sup><i>GFP/+</i></sup> microglia contacting the indicated percentages of PV boutons in a 20&nbsp;μm radius around the microglia cell body over 20&nbsp;minutes (n&nbsp;= 51 cells from 6 mice) during the <i>in&nbsp;vivo</i> imaging experiments. PV boutons are labeled with PVe-Syp-tdTomato. The distribution is shown as density histogram and kernel density estimation. The first peak corresponds to 11.7% of PV boutons contacted by microglia and the second peak to 60.9%. The local minimum of the bimodal distribution is 26.4%. These data were pooled with data in (I) to generate the graph in <a id="crosref1850" href="#fig3">Figure&nbsp;3</a>C.</div><div id="fspara0515" role="paragraph">(D) Distribution of microglia-PV boutons interactions for the indicated duration during the <i>in&nbsp;vivo</i> imaging experiments. The distribution is shown as density histogram and kernel density estimation of the proportion of Syp-tdTomato+ PV synaptic boutons contacted by microglia for the indicated duration (n&nbsp;= 222 PV boutons from 6 mice).</div><div id="fspara0520" role="paragraph">(E) Duration of contacts between microglia interacting with a minority (n&nbsp;= 11 cells) or the majority (n&nbsp;= 25 cells) of local PV boutons during the <i>in&nbsp;vivo</i> imaging experiments. <sup>∗</sup>p&nbsp;&lt; 0.01, Mann-Whitney test. These data represent the subset of data shown in (D) for which it was possible to visualize the microglia cell body and they were pooled with data in (K) to generate the graph shown in <a id="crosref1855" href="#fig3">Figure&nbsp;3</a>D. Data were split based on the local minimum of the kernel density estimation in (C). Data are mean ± SEM, each data point represents one cell.</div><div id="fspara0525" role="paragraph">(F) Time-lapse images from <a id="crosref1860" href="#mmc4">Video S2</a> showing a Syp-tdTomato+ PV bouton being enwrapped by a microglia process (<i>Cx3cr1</i><sup><i>GFP/+</i></sup>) and contacted by a second process over 10&nbsp;minutes. Scale bar equal 10&nbsp;μm.</div><div id="fspara0530" role="paragraph">(G) Time-lapse images from <a id="crosref1865" href="#mmc5">Video S3</a> showing a <i>Cx3cr1</i><sup><i>GFP/+</i></sup> microglial process that terminates into what resembles a phagocytic cup engulfing a Syp-tdTomato+&nbsp;PV bouton over 20&nbsp;minutes. Scale bar equal 10&nbsp;μm.</div><div id="fspara0535" role="paragraph">(H) Time-lapse image from <i>in&nbsp;vivo</i> imaging of Syp-Gamillus+ PV boutons and microglia genetically labeled using <i>Tmem119</i><sup><i>CreER/+</i></sup><i>;Ai14</i> mice. Scale bar equal 10&nbsp;μm.</div><div id="fspara0540" role="paragraph">(I) Distribution of <i>Tmem119</i><sup><i>CreER/+</i></sup><i>;Ai14</i> microglia contacting the indicated percentages of PV boutons (labeled with PVe-Syp-Gamillus) in a 20&nbsp;μm radius around the cell body over 20&nbsp;minutes (n&nbsp;= 37 cells from 6 mice). The distribution is shown as density histogram and kernel density estimation. The first peak corresponds to 18.1% of PV boutons contacted by microglia and the second peak to 57.4%. The local minimum of the bimodal distribution is 34.4%. These data were pooled with data in (C) to generate the graph in <a id="crosref1870" href="#fig3">Figure&nbsp;3</a>C.</div><div id="fspara0545" role="paragraph">(J) Distribution of microglia-PV boutons interactions for the indicated duration. The distribution is shown as density histogram and kernel density estimation of the proportion of Syp-Gamillus+ PV synaptic boutons contacted by microglia for the indicated duration (n&nbsp;= 115 PV boutons from 6 mice).</div><div id="fspara0550" role="paragraph">(K) Duration of contacts between microglia interacting with a minority (n&nbsp;= 13 cells) or the majority (n&nbsp;= 23 cells) of local PV boutons. <sup>∗</sup>p&nbsp;&lt; 0.01, Student’s t test. These data represent the subset of data shown in (J) for which it was possible to visualize the microglia cell body and they were pooled with data in E to generate the graph shown in <a id="crosref1875" href="#fig3">Figure&nbsp;3</a>D. Data are mean ± SEM, each data point represents one cell.</div><div id="fspara0555" role="paragraph">(L) Representative images of the same 3D Imaris space in two different orientations. Note that, in the visualization of a 3D volume, boutons can appear closer than they are depending on the orientation. In contrast, “real contacts” appear close regardless of the orientation. Scale bar equal 5&nbsp;μm.</div><div id="fspara0560" role="paragraph">(M) Representative 3D reconstruction of a microglia process and surrounding PV boutons. Contacted PV boutons are reconstructed as red spots whereas PV boutons not contacted by microglia are reconstructed as light blue spots. Full and empty arrowheads indicate positive and negative (i.e., boutons that did not meet the contact criteria) examples, respectively. The yellow empty arrowhead indicates a bouton that is far from the process. In all relevant figures, only contacted boutons (see methods for criteria) have been highlighted as “reconstructed spots.” Scale bar equal 5&nbsp;μm.</div></div></div></div></figcaption></figure></div><div id="p0080" role="paragraph"><div id="mmc3" class="core-supplementary-material" data-category="video"><div class="core-link"><div class="cloudflare-stream-container" id="mmc3" data-type="mp4"><span id="video_stream_mmc3_src" class="video-stream-source" style="display:none">eyJraWQiOiI4ZjUxYWNhY2IzYjhiNjNlNzFlYmIzYWFmYTU5NmZmYyIsImFsZyI6IlJTMjU2In0.eyJzdWIiOiI0MmE1OTgxYzc1MDA4YTk4YTdjZGM5MzUyZTk2ZDRiNCIsImtpZCI6IjhmNTFhY2FjYjNiOGI2M2U3MWViYjNhYWZhNTk2ZmZjIiwiZXhwIjoxNzQ5NjEyNTU5fQ.UUJhMd8M4DIDm78nuP4lHOcppGKB-7ghFF1xFCUILCDs1DVW-TQyqrDazlj3iCfvZbBhAPecXmAM5HG5WgjGtaNImM07HtvLvUjZP2GBg8Mm7y4gUoTh8fPJTT3ZHhu7jN8Py9Nzy76-PgIuxhcfk66G9iL19K-YtCdjJIyXNy-Tks16owZrjQ3RiR1hht4G8mA4tqHtrl-UHgMQuTcVVEsxi94wNPLKpUVK3n81xl8VQz5sotVxSfv93gYPi0m65czAVL6eGDim7OnrfLhNSb2lqcQwJmmnvUXZbIJblMSz6ThUQfgd25_R9RihXice1fOKDSRbEn2cinB401X4IA</span><stream src="eyJraWQiOiI4ZjUxYWNhY2IzYjhiNjNlNzFlYmIzYWFmYTU5NmZmYyIsImFsZyI6IlJTMjU2In0.eyJzdWIiOiI0MmE1OTgxYzc1MDA4YTk4YTdjZGM5MzUyZTk2ZDRiNCIsImtpZCI6IjhmNTFhY2FjYjNiOGI2M2U3MWViYjNhYWZhNTk2ZmZjIiwiZXhwIjoxNzQ5NjEyNTU5fQ.UUJhMd8M4DIDm78nuP4lHOcppGKB-7ghFF1xFCUILCDs1DVW-TQyqrDazlj3iCfvZbBhAPecXmAM5HG5WgjGtaNImM07HtvLvUjZP2GBg8Mm7y4gUoTh8fPJTT3ZHhu7jN8Py9Nzy76-PgIuxhcfk66G9iL19K-YtCdjJIyXNy-Tks16owZrjQ3RiR1hht4G8mA4tqHtrl-UHgMQuTcVVEsxi94wNPLKpUVK3n81xl8VQz5sotVxSfv93gYPi0m65czAVL6eGDim7OnrfLhNSb2lqcQwJmmnvUXZbIJblMSz6ThUQfgd25_R9RihXice1fOKDSRbEn2cinB401X4IA" class="cloudflare-stream-player ready tracked" id="video_stream_mmc3" controls="true"><div class="player-share-button" title="Share Video"><div class="share-player-icon"></div></div><div class="player-share-container hidden"><span class="player-close-share"></span><div class="social-links-container"><ul class="social-links-list"><li class="html5-share-facebook-icon" data-share-url="https://www.facebook.com/sharer/sharer.php?u=https://www.cell.com/cell/fulltext/S0092-8674(21)00753-4" target="_blank"></li><li class="html5-share-twitter-icon" data-share-url="https://twitter.com/share?url=https://www.cell.com/cell/fulltext/S0092-8674(21)00753-4" target="_blank"></li><li class="html5-share-mail-icon" data-share-url="mailto:?body=https://www.cell.com/cell/fulltext/S0092-8674(21)00753-4" target="_blank"></li></ul><input type="text" value="https://www.cell.com/cell/fulltext/S0092-8674(21)00753-4" readonly="readonly" class="share-input"></div></div><div style="position: relative; height: 0px; width: 100%; padding-top: 96.2562%;"><iframe src="https://embed.cloudflarestream.com/embed/iframe.fla9.823df22.html?videoId=eyJraWQiOiI4ZjUxYWNhY2IzYjhiNjNlNzFlYmIzYWFmYTU5NmZmYyIsImFsZyI6IlJTMjU2In0.eyJzdWIiOiI0MmE1OTgxYzc1MDA4YTk4YTdjZGM5MzUyZTk2ZDRiNCIsImtpZCI6IjhmNTFhY2FjYjNiOGI2M2U3MWViYjNhYWZhNTk2ZmZjIiwiZXhwIjoxNzQ5NjEyNTU5fQ.UUJhMd8M4DIDm78nuP4lHOcppGKB-7ghFF1xFCUILCDs1DVW-TQyqrDazlj3iCfvZbBhAPecXmAM5HG5WgjGtaNImM07HtvLvUjZP2GBg8Mm7y4gUoTh8fPJTT3ZHhu7jN8Py9Nzy76-PgIuxhcfk66G9iL19K-YtCdjJIyXNy-Tks16owZrjQ3RiR1hht4G8mA4tqHtrl-UHgMQuTcVVEsxi94wNPLKpUVK3n81xl8VQz5sotVxSfv93gYPi0m65czAVL6eGDim7OnrfLhNSb2lqcQwJmmnvUXZbIJblMSz6ThUQfgd25_R9RihXice1fOKDSRbEn2cinB401X4IA" allowfullscreen="true" allow="autoplay; picture-in-picture" style="height: 100%; width: 100%; position: absolute; top: 0px; left: 0px; border: none;"></iframe></div></stream></div><a href="/cms/10.1016/j.cell.2021.06.018/attachment/9b5d98f8-c489-4d5c-91d7-e3b0747b922f/mmc3.mp4" download="mmc3.mp4" aria-labelledby="mmc3-heading mmc3-link"><span id="mmc3-link">Video (5.55 MB)</span></a></div><div class="core-description"><div id="mmc3-heading" class="heading">Video S1. Time-lapse video showing interactions between microglia and PV boutons <i>in&nbsp;vivo</i>, related to Figure&nbsp;3</div><div id="espara0010" role="paragraph">Multiple contacts between microglial processes (green, <i>Cx3cr1</i><sup><i>GFP/+</i></sup> mice) displaying bulbous endings and PV boutons (red, PVe-Syp-tdTomato AAV) are observed over 30&nbsp;minutes. Representative time-frames from this movie are shown in Figure&nbsp;3B.</div></div></div></div><div id="p0085" role="paragraph"><div id="mmc4" class="core-supplementary-material" data-category="video"><div class="core-link"><div class="cloudflare-stream-container" id="mmc4" data-type="mp4"><span id="video_stream_mmc4_src" class="video-stream-source" style="display:none">eyJraWQiOiI4ZjUxYWNhY2IzYjhiNjNlNzFlYmIzYWFmYTU5NmZmYyIsImFsZyI6IlJTMjU2In0.eyJzdWIiOiI0M2ZmOGZhZWJjNzQ3ZjI3NDBjNzJiMjE3YzFjMmU3NCIsImtpZCI6IjhmNTFhY2FjYjNiOGI2M2U3MWViYjNhYWZhNTk2ZmZjIiwiZXhwIjoxNzQ5NjEyNTU5fQ.pQXZjRCuuNQyaFfkFBK4-eu9CR4NR5qxwxc1zRAufFtZoHPPG0Hl6AP_L5_6rxgFS-UBXk8ajjGZ4VJn-ru9zpcBv_oqsPJNRoR_ikQ2ZKof6pxhiSK0kxPb1A5EZD13ZsL2FL-_6LsViqhXzlpogyR2TLItIOMDWt9p3H7hABTh1iRa7_dXeQMGuCAhYFNUBoGNbrWBgAxDWtYkcZKTIo59LIP97S69Cvj5a7oG635VWXs91ZXcfYkgsgo0ziUxiJ2K4KP-zyKyd-aL5mtUCKgurpfFbeEp_KTOlVRyvQLa87R-IzuSn3z4TUi3dwmXCTMnIKyV6zwnBcxcfS1QDQ</span><stream src="eyJraWQiOiI4ZjUxYWNhY2IzYjhiNjNlNzFlYmIzYWFmYTU5NmZmYyIsImFsZyI6IlJTMjU2In0.eyJzdWIiOiI0M2ZmOGZhZWJjNzQ3ZjI3NDBjNzJiMjE3YzFjMmU3NCIsImtpZCI6IjhmNTFhY2FjYjNiOGI2M2U3MWViYjNhYWZhNTk2ZmZjIiwiZXhwIjoxNzQ5NjEyNTU5fQ.pQXZjRCuuNQyaFfkFBK4-eu9CR4NR5qxwxc1zRAufFtZoHPPG0Hl6AP_L5_6rxgFS-UBXk8ajjGZ4VJn-ru9zpcBv_oqsPJNRoR_ikQ2ZKof6pxhiSK0kxPb1A5EZD13ZsL2FL-_6LsViqhXzlpogyR2TLItIOMDWt9p3H7hABTh1iRa7_dXeQMGuCAhYFNUBoGNbrWBgAxDWtYkcZKTIo59LIP97S69Cvj5a7oG635VWXs91ZXcfYkgsgo0ziUxiJ2K4KP-zyKyd-aL5mtUCKgurpfFbeEp_KTOlVRyvQLa87R-IzuSn3z4TUi3dwmXCTMnIKyV6zwnBcxcfS1QDQ" class="cloudflare-stream-player ready tracked" id="video_stream_mmc4" controls="true"><div class="player-share-button" title="Share Video"><div class="share-player-icon"></div></div><div class="player-share-container hidden"><span class="player-close-share"></span><div class="social-links-container"><ul class="social-links-list"><li class="html5-share-facebook-icon" data-share-url="https://www.facebook.com/sharer/sharer.php?u=https://www.cell.com/cell/fulltext/S0092-8674(21)00753-4" target="_blank"></li><li class="html5-share-twitter-icon" data-share-url="https://twitter.com/share?url=https://www.cell.com/cell/fulltext/S0092-8674(21)00753-4" target="_blank"></li><li class="html5-share-mail-icon" data-share-url="mailto:?body=https://www.cell.com/cell/fulltext/S0092-8674(21)00753-4" target="_blank"></li></ul><input type="text" value="https://www.cell.com/cell/fulltext/S0092-8674(21)00753-4" readonly="readonly" class="share-input"></div></div><div style="position: relative; height: 0px; width: 100%; padding-top: 95.238%;"><iframe src="https://embed.cloudflarestream.com/embed/iframe.fla9.823df22.html?videoId=eyJraWQiOiI4ZjUxYWNhY2IzYjhiNjNlNzFlYmIzYWFmYTU5NmZmYyIsImFsZyI6IlJTMjU2In0.eyJzdWIiOiI0M2ZmOGZhZWJjNzQ3ZjI3NDBjNzJiMjE3YzFjMmU3NCIsImtpZCI6IjhmNTFhY2FjYjNiOGI2M2U3MWViYjNhYWZhNTk2ZmZjIiwiZXhwIjoxNzQ5NjEyNTU5fQ.pQXZjRCuuNQyaFfkFBK4-eu9CR4NR5qxwxc1zRAufFtZoHPPG0Hl6AP_L5_6rxgFS-UBXk8ajjGZ4VJn-ru9zpcBv_oqsPJNRoR_ikQ2ZKof6pxhiSK0kxPb1A5EZD13ZsL2FL-_6LsViqhXzlpogyR2TLItIOMDWt9p3H7hABTh1iRa7_dXeQMGuCAhYFNUBoGNbrWBgAxDWtYkcZKTIo59LIP97S69Cvj5a7oG635VWXs91ZXcfYkgsgo0ziUxiJ2K4KP-zyKyd-aL5mtUCKgurpfFbeEp_KTOlVRyvQLa87R-IzuSn3z4TUi3dwmXCTMnIKyV6zwnBcxcfS1QDQ" allowfullscreen="true" allow="autoplay; picture-in-picture" style="height: 100%; width: 100%; position: absolute; top: 0px; left: 0px; border: none;"></iframe></div></stream></div><a href="/cms/10.1016/j.cell.2021.06.018/attachment/48893a3f-c190-47c2-83b7-8e235d06fb9f/mmc4.mp4" download="mmc4.mp4" aria-labelledby="mmc4-heading mmc4-link"><span id="mmc4-link">Video (6.90 MB)</span></a></div><div class="core-description"><div id="mmc4-heading" class="heading">Video S2. Time-lapse video showing an example of microglia wrapping a PV bouton <i>in&nbsp;vivo</i>, related to Figure&nbsp;3</div><div id="espara0015" role="paragraph">Time-lapse imaging showing a Syp-tdTomato+ PV bouton (red, highlighted by the yellow square) being enwrapped by a microglia process (green, <i>Cx3cr1</i><sup><i>GFP/+</i></sup> mice) and contacted by a second process over 10&nbsp;minutes. Representative time-frames from this movie are shown in Figure&nbsp;S2F.</div></div></div></div><div id="p0090" role="paragraph"><div id="mmc5" class="core-supplementary-material" data-category="video"><div class="core-link"><div class="cloudflare-stream-container" id="mmc5" data-type="mp4"><span id="video_stream_mmc5_src" class="video-stream-source" style="display:none">eyJraWQiOiI4ZjUxYWNhY2IzYjhiNjNlNzFlYmIzYWFmYTU5NmZmYyIsImFsZyI6IlJTMjU2In0.eyJzdWIiOiJmNjJmNmE3N2RmY2Q0ZDZiZmI3OWZkYzFmYmZmNjUzNCIsImtpZCI6IjhmNTFhY2FjYjNiOGI2M2U3MWViYjNhYWZhNTk2ZmZjIiwiZXhwIjoxNzQ5NjEyNTU5fQ.WCqq5mfum9g01UW4aphpqaoODJ4ZqGHYMkA9ZjvEFzhLtLJT2A_ZOfznIOdo8bqw2BT4IQh3IZkDvDYgt_HVEx-JHM9kZfItmW4KHh0Shs8rmETFg6MLQwGKDp9awN0z3R_VPJvH29ejZM9MD1jaVCVrcvqm-b8dG8b8oOgsVmw_0AeF02gLPNH3npw0Uu40Q1ce62mmYwO3I5RL6eoOiONDFMx1XfPaHxPCGobdfeNkH8rbCRGjAN8-_91_wuoTN3ZNTKIg82_gcIs04-cN06JmC_kHFOtFIU7atzin9fDwww8S5rweO8RGKXlcQZT1j3Es-EuWoNVTJPRQhppr_Q</span><stream src="eyJraWQiOiI4ZjUxYWNhY2IzYjhiNjNlNzFlYmIzYWFmYTU5NmZmYyIsImFsZyI6IlJTMjU2In0.eyJzdWIiOiJmNjJmNmE3N2RmY2Q0ZDZiZmI3OWZkYzFmYmZmNjUzNCIsImtpZCI6IjhmNTFhY2FjYjNiOGI2M2U3MWViYjNhYWZhNTk2ZmZjIiwiZXhwIjoxNzQ5NjEyNTU5fQ.WCqq5mfum9g01UW4aphpqaoODJ4ZqGHYMkA9ZjvEFzhLtLJT2A_ZOfznIOdo8bqw2BT4IQh3IZkDvDYgt_HVEx-JHM9kZfItmW4KHh0Shs8rmETFg6MLQwGKDp9awN0z3R_VPJvH29ejZM9MD1jaVCVrcvqm-b8dG8b8oOgsVmw_0AeF02gLPNH3npw0Uu40Q1ce62mmYwO3I5RL6eoOiONDFMx1XfPaHxPCGobdfeNkH8rbCRGjAN8-_91_wuoTN3ZNTKIg82_gcIs04-cN06JmC_kHFOtFIU7atzin9fDwww8S5rweO8RGKXlcQZT1j3Es-EuWoNVTJPRQhppr_Q" class="cloudflare-stream-player ready tracked" id="video_stream_mmc5" controls="true"><div class="player-share-button" title="Share Video"><div class="share-player-icon"></div></div><div class="player-share-container hidden"><span class="player-close-share"></span><div class="social-links-container"><ul class="social-links-list"><li class="html5-share-facebook-icon" data-share-url="https://www.facebook.com/sharer/sharer.php?u=https://www.cell.com/cell/fulltext/S0092-8674(21)00753-4" target="_blank"></li><li class="html5-share-twitter-icon" data-share-url="https://twitter.com/share?url=https://www.cell.com/cell/fulltext/S0092-8674(21)00753-4" target="_blank"></li><li class="html5-share-mail-icon" data-share-url="mailto:?body=https://www.cell.com/cell/fulltext/S0092-8674(21)00753-4" target="_blank"></li></ul><input type="text" value="https://www.cell.com/cell/fulltext/S0092-8674(21)00753-4" readonly="readonly" class="share-input"></div></div><div style="position: relative; height: 0px; width: 100%; padding-top: 94.9862%;"><iframe src="https://embed.cloudflarestream.com/embed/iframe.fla9.823df22.html?videoId=eyJraWQiOiI4ZjUxYWNhY2IzYjhiNjNlNzFlYmIzYWFmYTU5NmZmYyIsImFsZyI6IlJTMjU2In0.eyJzdWIiOiJmNjJmNmE3N2RmY2Q0ZDZiZmI3OWZkYzFmYmZmNjUzNCIsImtpZCI6IjhmNTFhY2FjYjNiOGI2M2U3MWViYjNhYWZhNTk2ZmZjIiwiZXhwIjoxNzQ5NjEyNTU5fQ.WCqq5mfum9g01UW4aphpqaoODJ4ZqGHYMkA9ZjvEFzhLtLJT2A_ZOfznIOdo8bqw2BT4IQh3IZkDvDYgt_HVEx-JHM9kZfItmW4KHh0Shs8rmETFg6MLQwGKDp9awN0z3R_VPJvH29ejZM9MD1jaVCVrcvqm-b8dG8b8oOgsVmw_0AeF02gLPNH3npw0Uu40Q1ce62mmYwO3I5RL6eoOiONDFMx1XfPaHxPCGobdfeNkH8rbCRGjAN8-_91_wuoTN3ZNTKIg82_gcIs04-cN06JmC_kHFOtFIU7atzin9fDwww8S5rweO8RGKXlcQZT1j3Es-EuWoNVTJPRQhppr_Q" allowfullscreen="true" allow="autoplay; picture-in-picture" style="height: 100%; width: 100%; position: absolute; top: 0px; left: 0px; border: none;"></iframe></div></stream></div><a href="/cms/10.1016/j.cell.2021.06.018/attachment/12d91751-89d5-4add-bf71-d3a65015deae/mmc5.mp4" download="mmc5.mp4" aria-labelledby="mmc5-heading mmc5-link"><span id="mmc5-link">Video (4.14 MB)</span></a></div><div class="core-description"><div id="mmc5-heading" class="heading">Video S3. Time-lapse video showing a possible example of microglia engulfing a PV bouton <i>in&nbsp;vivo</i>, related to Figure&nbsp;3</div><div id="espara0020" role="paragraph">A <i>Cx3cr1</i><sup><i>GFP/+</i></sup> microglial process (green) terminates into what resembles a phagocytic cup engulfing a Syp-tdTomato+ PV bouton (red) over 20&nbsp;minutes. The synaptic material appears to move along with the phagocytic cup and is broken down in two smaller fragments. The phagocytic cup is then partially reabsorbed. Given the low intensity of the PV bouton, the video is played twice and, in the second half, PV synaptic material is marked by a red dot. Representative time-frames from this movie are shown in Figure&nbsp;S2G.</div></div></div></div><div id="p0095" role="paragraph"><div id="mmc6" class="core-supplementary-material" data-category="video"><div class="core-link"><div class="cloudflare-stream-container" id="mmc6" data-type="mp4"><span id="video_stream_mmc6_src" class="video-stream-source" style="display:none">eyJraWQiOiI4ZjUxYWNhY2IzYjhiNjNlNzFlYmIzYWFmYTU5NmZmYyIsImFsZyI6IlJTMjU2In0.eyJzdWIiOiIzYmNjMGM1YWNlYTE2YTVlMzU1NmZkNzNiMWE1Y2JkMCIsImtpZCI6IjhmNTFhY2FjYjNiOGI2M2U3MWViYjNhYWZhNTk2ZmZjIiwiZXhwIjoxNzQ5NjEyNTU5fQ.LBhjumpyZdqxHWz8YpSG3vhFzMRQs22eRGL_BifIR9B8Q4KfmeMtnEIAQlAzsJKCSZZfhdu-RHbcUzwjBqZmXZxVmG9Pzhc6QB4Q9pDv3WKIPpawi_8Ur4ccvPgIg-AtdgUdSJMXc5Ydehwdwo9lraXRoAawIPggZ6yMdSRyDdLHJNtZnbhVSFd_GPGYoJMGrrFtdCsRMWapmE5BgHmqKNBgnqj3kwN-wfzOL8IzmPELPy8PJW5BOqIALya1_kzdGzKKcIBIx_Q6DsstE1FkBpOYZSmT_26WLZOofcrSj21cWKrZi-VYYsliBNCrr-uYdFzAnkYu-wrbSn34jojoiw</span><stream src="eyJraWQiOiI4ZjUxYWNhY2IzYjhiNjNlNzFlYmIzYWFmYTU5NmZmYyIsImFsZyI6IlJTMjU2In0.eyJzdWIiOiIzYmNjMGM1YWNlYTE2YTVlMzU1NmZkNzNiMWE1Y2JkMCIsImtpZCI6IjhmNTFhY2FjYjNiOGI2M2U3MWViYjNhYWZhNTk2ZmZjIiwiZXhwIjoxNzQ5NjEyNTU5fQ.LBhjumpyZdqxHWz8YpSG3vhFzMRQs22eRGL_BifIR9B8Q4KfmeMtnEIAQlAzsJKCSZZfhdu-RHbcUzwjBqZmXZxVmG9Pzhc6QB4Q9pDv3WKIPpawi_8Ur4ccvPgIg-AtdgUdSJMXc5Ydehwdwo9lraXRoAawIPggZ6yMdSRyDdLHJNtZnbhVSFd_GPGYoJMGrrFtdCsRMWapmE5BgHmqKNBgnqj3kwN-wfzOL8IzmPELPy8PJW5BOqIALya1_kzdGzKKcIBIx_Q6DsstE1FkBpOYZSmT_26WLZOofcrSj21cWKrZi-VYYsliBNCrr-uYdFzAnkYu-wrbSn34jojoiw" class="cloudflare-stream-player ready tracked" id="video_stream_mmc6" controls="true"><div class="player-share-button" title="Share Video"><div class="share-player-icon"></div></div><div class="player-share-container hidden"><span class="player-close-share"></span><div class="social-links-container"><ul class="social-links-list"><li class="html5-share-facebook-icon" data-share-url="https://www.facebook.com/sharer/sharer.php?u=https://www.cell.com/cell/fulltext/S0092-8674(21)00753-4" target="_blank"></li><li class="html5-share-twitter-icon" data-share-url="https://twitter.com/share?url=https://www.cell.com/cell/fulltext/S0092-8674(21)00753-4" target="_blank"></li><li class="html5-share-mail-icon" data-share-url="mailto:?body=https://www.cell.com/cell/fulltext/S0092-8674(21)00753-4" target="_blank"></li></ul><input type="text" value="https://www.cell.com/cell/fulltext/S0092-8674(21)00753-4" readonly="readonly" class="share-input"></div></div><div style="position: relative; height: 0px; width: 100%; padding-top: 96.7735%;"><iframe src="https://embed.cloudflarestream.com/embed/iframe.fla9.823df22.html?videoId=eyJraWQiOiI4ZjUxYWNhY2IzYjhiNjNlNzFlYmIzYWFmYTU5NmZmYyIsImFsZyI6IlJTMjU2In0.eyJzdWIiOiIzYmNjMGM1YWNlYTE2YTVlMzU1NmZkNzNiMWE1Y2JkMCIsImtpZCI6IjhmNTFhY2FjYjNiOGI2M2U3MWViYjNhYWZhNTk2ZmZjIiwiZXhwIjoxNzQ5NjEyNTU5fQ.LBhjumpyZdqxHWz8YpSG3vhFzMRQs22eRGL_BifIR9B8Q4KfmeMtnEIAQlAzsJKCSZZfhdu-RHbcUzwjBqZmXZxVmG9Pzhc6QB4Q9pDv3WKIPpawi_8Ur4ccvPgIg-AtdgUdSJMXc5Ydehwdwo9lraXRoAawIPggZ6yMdSRyDdLHJNtZnbhVSFd_GPGYoJMGrrFtdCsRMWapmE5BgHmqKNBgnqj3kwN-wfzOL8IzmPELPy8PJW5BOqIALya1_kzdGzKKcIBIx_Q6DsstE1FkBpOYZSmT_26WLZOofcrSj21cWKrZi-VYYsliBNCrr-uYdFzAnkYu-wrbSn34jojoiw" allowfullscreen="true" allow="autoplay; picture-in-picture" style="height: 100%; width: 100%; position: absolute; top: 0px; left: 0px; border: none;"></iframe></div></stream></div><a href="/cms/10.1016/j.cell.2021.06.018/attachment/040b60f5-c0fd-418c-85d9-fce95c467c51/mmc6.mp4" download="mmc6.mp4" aria-labelledby="mmc6-heading mmc6-link"><span id="mmc6-link">Video (1.72 MB)</span></a></div><div class="core-description"><div id="mmc6-heading" class="heading">Video S4. Time-lapse video showing a possible example of intake of PV presynaptic boutons by microglia <i>in&nbsp;vivo</i>, related to Figure&nbsp;3</div><div id="espara0025" role="paragraph">A <i>Cx3cr1</i><sup><i>GFP/+</i></sup> microglia (green) forms what appears as a phagocytic cup closing over a Syp-tdTomato+ PV bouton (red) over 20&nbsp;minutes.</div></div></div></div><div id="p0100" role="paragraph">These differential interactions suggested the existence of at least two microglia subpopulations or states. We therefore mined published transcriptional data (<span class="dropBlock reference-citations"><a id="body-ref-sref24-1" href="#" role="doc-biblioref" data-xml-rid="bib24" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref24-1" aria-controls="sref24-1">Favuzzi et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref24-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">25.</div><div>Favuzzi, E. ∙ Deogracias, R. ∙ Marques-Smith, A. ...</div><div><strong>Distinct molecular programs regulate synapse specificity in cortical inhibitory circuits</strong></div><div><em>Science.</em> 2019; <strong>363</strong>:413-417</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aau8977" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85060511221" target="_blank" aria-label="Open Scopus in new tab">Scopus (110)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30679375/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aau8977&amp;pmid=30679375" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref48-2" href="#" role="doc-biblioref" data-xml-rid="bib48" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref48-2" aria-controls="sref48-2">Matcovitch-Natan et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref48-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">49.</div><div>Matcovitch-Natan, O. ∙ Winter, D.R. ∙ Giladi, A. ...</div><div><strong>Microglia development follows a stepwise program to regulate brain homeostasis</strong></div><div><em>Science.</em> 2016; <strong>353</strong>:aad8670</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aad8670" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-84976883162" target="_blank" aria-label="Open Scopus in new tab">Scopus (792)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27338705/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aad8670&amp;pmid=27338705" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) to seek annotated ligand-receptor pairs expressed in microglia and inhibitory—but not excitatory—neurons during development (<a id="crosref0565" href="#figs3">Figure&nbsp;S3</a>A). GABA-GABA<sub>B</sub> receptors were among the leading candidates that could conceivably mediate signaling from inhibitory synapses to microglia. Such signaling has a strong precedent. In addition to its role as a neurotransmitter, GABA acts as a paracrine signal to control a myriad of developmental events. These include stem cell proliferation, migration, synaptogenesis, synapse pruning and astrocyte activity (<span class="dropBlock reference-citations"><a id="body-ref-sref52-1" href="#" role="doc-biblioref" data-xml-rid="bib52" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref52-1" aria-controls="sref52-1">Nagai et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref52-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">53.</div><div>Nagai, J. ∙ Rajbhandari, A.K. ∙ Gangwani, M.R. ...</div><div><strong>Hyperactivity with Disrupted Attention by Activation of an Astrocyte Synaptogenic Cue</strong></div><div><em>Cell.</em> 2019; <strong>177</strong>:1280-1292.e20</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.cell.2019.03.019&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.cell.2019.03.019&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85065073576" target="_blank" aria-label="Open Scopus in new tab">Scopus (190)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31031006/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2019.03.019&amp;pmid=31031006" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref54" href="#" role="doc-biblioref" data-xml-rid="bib54" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref54" aria-controls="sref54">Oh et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref54"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">56.</div><div>Oh, W.C. ∙ Lutzu, S. ∙ Castillo, P.E. ...</div><div><strong>De novo synaptogenesis induced by GABA in the developing mouse cortex</strong></div><div><em>Science.</em> 2016; <strong>353</strong>:1037-1040</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aaf5206" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_56_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-84983296823" target="_blank" aria-label="Open Scopus in new tab">Scopus (135)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27516412/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aaf5206&amp;pmid=27516412" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref84" href="#" role="doc-biblioref" data-xml-rid="bib84" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref84" aria-controls="sref84">Wang and Kriegstein, 2009</a><div class="dropBlock__holder" data-db-target-of="sref84"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">86.</div><div>Wang, D.D. ∙ Kriegstein, A.R.</div><div><strong>Defining the role of GABA in cortical development</strong></div><div><em>J.&nbsp;Physiol.</em> 2009; <strong>587</strong>:1873-1879</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1113/jphysiol.2008.167635" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_86_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-62549102539" target="_blank" aria-label="Open Scopus in new tab">Scopus (266)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19153158/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1113%2Fjphysiol.2008.167635&amp;pmid=19153158" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref88-1" href="#" role="doc-biblioref" data-xml-rid="bib88" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref88-1" aria-controls="sref88-1">Wu et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref88-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">90.</div><div>Wu, X. ∙ Fu, Y. ∙ Knott, G. ...</div><div><strong>GABA signaling promotes synapse elimination and axon pruning in developing cortical inhibitory interneurons</strong></div><div><em>J.&nbsp;Neurosci.</em> 2012; <strong>32</strong>:331-343</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1523/JNEUROSCI.3189-11.2012" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_90_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-84855913405" target="_blank" aria-label="Open Scopus in new tab">Scopus (84)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22219294/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1523%2FJNEUROSCI.3189-11.2012&amp;pmid=22219294" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), most of which are mediated by GABA<sub>B</sub> receptors (<span class="dropBlock reference-citations"><a id="body-ref-sref27-1" href="#" role="doc-biblioref" data-xml-rid="bib27" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref27-1" aria-controls="sref27-1">Gaiarsa and Porcher, 2013</a><div class="dropBlock__holder" data-db-target-of="sref27-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">28.</div><div>Gaiarsa, J.-L. ∙ Porcher, C.</div><div><strong>Emerging neurotrophic role of GABAB receptors in neuronal circuit development</strong></div><div><em>Front. Cell. Neurosci.</em> 2013; <strong>7</strong>:206</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.3389/fncel.2013.00206" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-84888155744" target="_blank" aria-label="Open Scopus in new tab">Scopus (44)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24282395/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.3389%2Ffncel.2013.00206&amp;pmid=24282395" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref50" href="#" role="doc-biblioref" data-xml-rid="bib50" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref50" aria-controls="sref50">Mederos and Perea, 2019</a><div class="dropBlock__holder" data-db-target-of="sref50"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">51.</div><div>Mederos, S. ∙ Perea, G.</div><div><strong>GABAergic-astrocyte signaling: A refinement of inhibitory brain networks</strong></div><div><em>Glia.</em> 2019; <strong>67</strong>:1842-1851</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/glia.23644" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_51_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85066497252" target="_blank" aria-label="Open Scopus in new tab">Scopus (67)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31145508/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fglia.23644&amp;pmid=31145508" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref52-2" href="#" role="doc-biblioref" data-xml-rid="bib52" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref52-2" aria-controls="sref52-2">Nagai et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref52-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">53.</div><div>Nagai, J. ∙ Rajbhandari, A.K. ∙ Gangwani, M.R. ...</div><div><strong>Hyperactivity with Disrupted Attention by Activation of an Astrocyte Synaptogenic Cue</strong></div><div><em>Cell.</em> 2019; <strong>177</strong>:1280-1292.e20</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.cell.2019.03.019&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.cell.2019.03.019&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85065073576" target="_blank" aria-label="Open Scopus in new tab">Scopus (190)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31031006/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2019.03.019&amp;pmid=31031006" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Moreover, previous work showed that a subset of microglia express GABA<sub>B</sub> receptors (GABA<sub>B</sub>Rs), and GABA elicits chemical, electrical, and morphological responses in microglia (<span class="dropBlock reference-citations"><a id="body-ref-sref26" href="#" role="doc-biblioref" data-xml-rid="bib26" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref26" aria-controls="sref26">Fontainhas et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref26"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">27.</div><div>Fontainhas, A.M. ∙ Wang, M. ∙ Liang, K.J. ...</div><div><strong>Microglial morphology and dynamic behavior is regulated by ionotropic glutamatergic and GABAergic neurotransmission</strong></div><div><em>PLoS ONE.</em> 2011; <strong>6</strong>:e15973</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pone.0015973" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-79551570269" target="_blank" aria-label="Open Scopus in new tab">Scopus (270)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21283568/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pone.0015973&amp;pmid=21283568" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref37-1" href="#" role="doc-biblioref" data-xml-rid="bib37" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref37-1" aria-controls="sref37-1">Kuhn et&nbsp;al., 2004</a><div class="dropBlock__holder" data-db-target-of="sref37-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">38.</div><div>Kuhn, S.A. ∙ van Landeghem, F.K.H. ∙ Zacharias, R. ...</div><div><strong>Microglia express GABA(B) receptors to modulate interleukin release</strong></div><div><em>Mol. Cell. Neurosci.</em> 2004; <strong>25</strong>:312-322</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.mcn.2003.10.023" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_38_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-1542616888" target="_blank" aria-label="Open Scopus in new tab">Scopus (168)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15019947/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.mcn.2003.10.023&amp;pmid=15019947" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div class="figure-wrap"><figure id="figs3" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.06.018/asset/b2709bed-9fcc-448a-be45-984b910432de/main.assets/figs3_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs3" src="/cms/10.1016/j.cell.2021.06.018/asset/28b9fa3e-450a-4f1f-8ac5-720d67d657c8/main.assets/figs3.jpg" height="958" width="481" aria-labelledby="figs3-title figs3-content" data-viewer-src="/cms/10.1016/j.cell.2021.06.018/asset/b2709bed-9fcc-448a-be45-984b910432de/main.assets/figs3_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs3-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S3</span> <span class="figure__title__text">Expression of GABA<sub>B</sub> receptors in microglia, related to <a id="crosref1880" href="#fig2">Figure&nbsp;2</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs3-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0570" role="paragraph">(A) Schematic of analysis and heatmaps showing ligands-receptor pairs that are leading candidates for mediating interneuron-microglia communication. The heatmap on the left is showing the expression (at the indicated developmental time points) of putative ligands enriched in interneurons as compared to pyramidal cells. For P5 and P10 interneurons, the corresponding values in SST and PV cells were averaged. The list is ranked according to specificity and developmental upregulation. For GABA, the expression of the GABA-synthetizing enzyme Gad1 is shown. The heatmap on the right is showing the expression (at the indicated developmental time points) of receptors expressed in microglia that are experimentally validated interactors of the ligands shown on the left. For the P3-P9 time point, values at P3 and P9 were averaged.</div><div id="fspara0575" role="paragraph">(B) Venn diagram depicting the fraction of microglia expressing <i>Gabbr1</i>+ mRNA, <i>Gabbr2</i>+ mRNA as well as double positive for both <i>Gabbr1</i>+ and <i>Gabbr2</i>+ mRNAs at P15 (smFISH: single molecule fluorescent <i>in situ</i> hybridization). The diagram is a visualization of the data shown in <a id="crosref1885" href="#fig3">Figure&nbsp;3</a>F.</div><div id="fspara0580" role="paragraph">(C) Validation of the specificity of GABA<sub>B1</sub>R antibody. Representative images from P15 controls and <i>VGlut1</i><sup><i>Cre</i></sup><i>;Gabbr1</i> cKO mice, lacking GABA<sub>B1</sub>R in cortical excitatory neurons. Note the absence of GABA<sub>B1</sub>R signal within NeuN+ excitatory neurons in <i>VGlut1</i><sup><i>Cre</i></sup> cKO mice. Scale bar equal 10&nbsp;μm.</div><div id="fspara0585" role="paragraph">(D) Left: quantitation of the percentage of microglia (Iba1+) expressing GABA<sub>B1</sub>R protein at P0, P10 and P15 in control and <i>Cx3cr1</i><sup><i>Cre</i></sup> GABA<sub>B1</sub>R cKO mice (n&nbsp;= 3-6 each). ns p &gt; 0.05, <sup>∗∗</sup>p&nbsp;&lt; 0.01, <sup>∗∗∗</sup>p&nbsp;&lt; 0.001, One-Way ANOVA followed by Sidak’s multiple comparisons test. Note the higher expression of GABA<sub>B1</sub>R in controls at P15 compared to P0 or P10. Right: quantitation of the percentage of microglia (Iba1+) expressing GABA<sub>B1</sub>R protein at P15 in control and <i>Tmem119</i><sup><i>CreER</i></sup> GABA<sub>B1</sub>R cKO mice (n&nbsp;= 4 each). <sup>∗</sup>p&nbsp;&lt; 0.05, Student’s t test. Data are mean ± SEM, each data point represents one experimental animal.</div><div id="fspara0590" role="paragraph">(E) Representative images and quantitation of microglia (Iba1+) expressing both GABA<sub>B1</sub>R and GABA<sub>B2</sub>R proteins in P15 control mice (n&nbsp;= 3) within layer 4 of S1. Scale bar equal 5&nbsp;μm. See <a id="crosref1890" href="#sec-4">STAR Methods</a> for details about noise in the IHC experiment versus smFISH. Arrowheads indicate colocalization. Data are mean ± SEM, each data point represents one experimental animal.</div><div id="fspara0595" role="paragraph">(F) <i>Gabbr1</i> and <i>Gabbr2</i> expression in adult microglia/macrophages from different brain regions. Data and visualization are from <span class="dropBlock reference-citations"><a id="body-ref-sref63-1" href="#" role="doc-biblioref" data-xml-rid="bib63" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref63-1" aria-controls="sref63-1">Saunders et&nbsp;al. (2018)</a><div class="dropBlock__holder" data-db-target-of="sref63-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">65.</div><div>Saunders, A. ∙ Macosko, E.Z. ∙ Wysoker, A. ...</div><div><strong>Molecular Diversity and Specializations among the Cells of the Adult Mouse Brain</strong></div><div><em>Cell.</em> 2018; <strong>174</strong>:1015-1030.e16</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.cell.2018.07.028&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.cell.2018.07.028&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85050885385" target="_blank" aria-label="Open Scopus in new tab">Scopus (875)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30096299/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2018.07.028&amp;pmid=30096299" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>. FC: frontal cortex, TH: thalamus, STR: striatum, PC: posterior cortex, CB: cerebellum, SN: substantia nigra, HC: hippocampus, GP: globus pallidus. The reported confidence intervals reflect statistical sampling noise.</div><div id="fspara0600" role="paragraph">(G) Fraction of GABA-receptive microglia (defined as expressing both <i>Gabbr1</i> and <i>Gabbr2</i> mRNAs by single molecule fluorescent <i>in situ</i> hybridization) in layer 4 of S1 and the CA1 region of the hippocampus of <i>Cx3cr1</i><sup><i>GFP/+</i></sup> mice at P15. S1 data are the same as shown in <a id="crosref1900" href="#fig3">Figure&nbsp;3</a>F. Data are mean ± SEM, each data point represents one experimental animal.</div><div id="fspara0605" role="paragraph">(H) 3D reconstruction and fraction of PV boutons (PVe-Syp-tdTom) encased by <i>Gabbr2</i>+ and <i>Gabbr2</i>- microglia (<i>Cx3cr1</i><sup><i>GFP/+</i></sup>) at P15 (n&nbsp;= 58 <i>Gabbr2</i>+ and 59 <i>Gabbr2</i>- cells from 4 mice). <sup>∗∗∗</sup>p&nbsp;&lt; 0.001, Mann-Whitney test. Scale bar equal 1&nbsp;μm. Data are mean ± SEM, each data point represents one cell.</div><div id="fspara0610" role="paragraph">(I) 3D reconstruction and fraction of VGlut2+ boutons encased by <i>Gabbr2</i>+ and <i>Gabbr2</i>- microglia (<i>Cx3cr1</i><sup><i>GFP/+</i></sup>) at P15 (n&nbsp;= 23 <i>Gabbr2</i>+ and 36 <i>Gabbr2</i>- cells from 4 mice). <sup>∗</sup>p&nbsp;&lt; 0.05, Student’s t test. Scale bar equal 1&nbsp;μm. Data are mean ± SEM, each data point represents one cell.</div></div></div></div></figcaption></figure></div><div id="p0105" role="paragraph">For GABA<sub>B</sub>-responsiveness, the expression of both <i>Gabbr1</i> and <i>Gabbr2</i> is required (<span class="dropBlock reference-citations"><a id="body-ref-sref32" href="#" role="doc-biblioref" data-xml-rid="bib32" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref32" aria-controls="sref32">Jones et&nbsp;al., 1998</a><div class="dropBlock__holder" data-db-target-of="sref32"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">33.</div><div>Jones, K.A. ∙ Borowsky, B. ∙ Tamm, J.A. ...</div><div><strong>GABA(B) receptors function as a heteromeric assembly of the subunits GABA(B)R1 and GABA(B)R2</strong></div><div><em>Nature.</em> 1998; <strong>396</strong>:674-679</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/25348" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-0005169503" target="_blank" aria-label="Open Scopus in new tab">Scopus (942)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/9872315/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2F25348&amp;pmid=9872315" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Using used single molecule fluorescent <i>in situ</i> hybridization (smFISH), we determined that 50% of all microglia express <i>Gabbr1,</i> with only half of this population co-expressing <i>Gabbr2.</i> Hence, 25% of all microglia are double-positive for both GABA<sub>B</sub>R subunits within barrels in S1 at P15 (<a id="crosref0620" href="#fig3">Figures 3</a>E, 3F, and <a id="crosref0625" href="#figs3">S3</a>B–S3E). The expression of GABA<sub>B</sub>R subunits in microglia is upregulated at postnatal stages and exhibits regional differences (<span class="dropBlock reference-citations"><a id="body-ref-sref48-3" href="#" role="doc-biblioref" data-xml-rid="bib48" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref48-3" aria-controls="sref48-3">Matcovitch-Natan et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref48-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">49.</div><div>Matcovitch-Natan, O. ∙ Winter, D.R. ∙ Giladi, A. ...</div><div><strong>Microglia development follows a stepwise program to regulate brain homeostasis</strong></div><div><em>Science.</em> 2016; <strong>353</strong>:aad8670</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aad8670" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-84976883162" target="_blank" aria-label="Open Scopus in new tab">Scopus (792)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27338705/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aad8670&amp;pmid=27338705" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref63-2" href="#" role="doc-biblioref" data-xml-rid="bib63" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref63-2" aria-controls="sref63-2">Saunders et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref63-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">65.</div><div>Saunders, A. ∙ Macosko, E.Z. ∙ Wysoker, A. ...</div><div><strong>Molecular Diversity and Specializations among the Cells of the Adult Mouse Brain</strong></div><div><em>Cell.</em> 2018; <strong>174</strong>:1015-1030.e16</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.cell.2018.07.028&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.cell.2018.07.028&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85050885385" target="_blank" aria-label="Open Scopus in new tab">Scopus (875)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30096299/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2018.07.028&amp;pmid=30096299" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) (<a id="crosref0640" href="#figs3">Figures S3</a>D, S3F, and S3G). To test the involvement of microglial GABA<sub>B</sub>Rs in the regulation of inhibitory synapse refinement, we combined smFISH with three-dimensional reconstruction. We found that inhibitory PV boutons were preferentially contacted and ensheathed by <i>Gabbr2</i>-expressing microglia (<a id="crosref0645" href="#fig3">Figures 3</a>G and <a id="crosref0650" href="#figs3">S3</a>H). In contrast, VGlut2<sup>+</sup> excitatory synaptic terminals were preferentially contacted and ensheathed by microglia that did not express <i>Gabbr2</i> (<a id="crosref0655" href="#fig3">Figures 3</a>H and <a id="crosref0660" href="#figs3">S3</a>I).</div></section><section id="sec-2-4"><h3>Removal of GABA<sub>B1</sub>Rs from microglia selectively impacts inhibitory connectivity</h3><div id="p0110" role="paragraph">To directly test the idea that GABA-receptive microglia may be dedicated to remodel inhibitory synapses during development, we generated conditional mutant mice lacking GABA<sub>B1</sub>Rs in microglia using two distinct Cre-driver lines (cKO, <i>Cx3cr1</i><sup><i>Cre</i>/+</sup><i>;GABA</i><sub><i>B1</i></sub><i>R</i><sup><i>fl</i>/<i>fl</i></sup> and <i>Tmem119</i><sup><i>CreER</i>/+</sup><i>;GABA</i><sub><i>B1</i></sub><i>R</i><sup><i>fl</i>/<i>fl</i></sup>). In each, we examined the impact of GABA<sub>B1</sub>R removal on microglia-PV synapse interactions. We found that cKO microglia contacted significantly fewer PV boutons compared to controls (<i>Cx3cr1</i><sup>+/+</sup><i>;GABA</i><sub><i>B1</i></sub><i>R</i><sup><i>fl</i>/<i>fl</i></sup>, <i>Cx3cr1</i><sup><i>Cre</i>/+</sup><i>;GABA</i><sub><i>B1</i></sub><i>R</i><sup>+/+</sup>, and <i>Tmem119</i><sup>+/+</sup><i>;GABA</i><sub><i>B1</i></sub><i>R</i><sup><i>fl</i>/<i>fl</i></sup>) (<a id="crosref0665" href="#fig4">Figures 4</a>A and <a id="crosref0670" href="#figs4">S4</a>A). In contrast, the proportion of contacted VGlut2<sup>+</sup> synaptic terminals was not affected by the removal of GABA<sub>B1</sub>Rs (<a id="crosref0675" href="#fig4">Figures 4</a>B and <a id="crosref0680" href="#figs4">S4</a>B). Consistent with these observations, <i>in&nbsp;vivo</i> two-photon imaging showed that cKO microglia interactions with PV boutons were no longer bimodal and more closely resembled those that interact less with PV synapses (<a id="crosref0685" href="#fig4">Figures 4</a>C–4E and <a id="crosref0690" href="#figs4">S4</a>C).</div><div class="figure-wrap"><figure id="fig4" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.06.018/asset/9b544f10-9a88-44f7-bf69-2a604ca3b2f9/main.assets/gr4_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr4" src="/cms/10.1016/j.cell.2021.06.018/asset/b68fbac5-b9a9-4013-b0e5-715f917a9001/main.assets/gr4.jpg" height="574" width="657" aria-labelledby="fig4-title fig4-content" data-viewer-src="/cms/10.1016/j.cell.2021.06.018/asset/9b544f10-9a88-44f7-bf69-2a604ca3b2f9/main.assets/gr4_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig4-title" class="figure__title"><span class="label figure__label">Figure&nbsp;4</span> <span class="figure__title__text">Removal of GABA<sub>B1</sub>Rs from microglia selectively impacts inhibitory connectivity</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig4-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0175" role="paragraph">(A) 3D reconstruction and fraction of PV boutons (PVe-Syp-tdTom) contacted by microglia (Iba1) in P15 control (n&nbsp;= 7) and GABA<sub>B1</sub>R cKO (n&nbsp;= 5) mice. <sup>∗∗</sup>p&nbsp;&lt; 0.01, Student’s t test. Scale bar, 4&nbsp;μm.</div><div id="fspara0180" role="paragraph">(B) 3D reconstruction and fraction of VGlut2<sup>+</sup> boutons contacted by microglia (Iba1) in P15 control (n&nbsp;= 7) and GABA<sub>B1</sub>R cKO (n&nbsp;= 5) mice. ns, p &gt; 0.05; Student’s t test. Scale bar, 4&nbsp;μm.</div><div id="fspara0185" role="paragraph">(C) Schematic of <i>in&nbsp;vivo</i> imaging experiments in (D)–(F) and brain vasculature imaged through the cranial window. Scale bar, 500&nbsp;μm.</div><div id="fspara0190" role="paragraph">(D) Time-lapse images showing cKO microglia contacting PV boutons. Scale bar, 20&nbsp;μm.</div><div id="fspara0195" role="paragraph">(E) Distribution of microglia contacting the indicated percentages of PV boutons over 20&nbsp;min in cKO mice (n&nbsp;= 62 cells from 6 mice). Control is from <a id="crosref1785" href="#fig3">Figure&nbsp;3</a>C.</div><div id="fspara0200" role="paragraph">(F) Duration of contacts between microglia and PV boutons in cKO mice (n&nbsp;= 37 cells from 3 mice). Control is from <a id="crosref1790" href="#fig3">Figure&nbsp;3</a>D.</div><div id="fspara0205" role="paragraph">(G) Schematic of experiments in (H) to (L).</div><div id="fspara0210" role="paragraph">(H) Images and density of Syt2<sup>+</sup>Gephyrin<sup>+</sup> synapses made by PV cells onto excitatory neurons in P15 control (n&nbsp;= 8) and cKO (n&nbsp;= 6) mice. <sup>∗∗∗</sup>p&nbsp;&lt; 0.001, Mann-Whitney test. Scale bar, 2&nbsp;μm.</div><div id="fspara0215" role="paragraph">(I) Images and density of VGlut2<sup>+</sup>Homer1<sup>+</sup> synapses onto excitatory neurons in P15 control (n&nbsp;= 6) and cKO (n&nbsp;= 7) mice. ns p &gt; 0.05, Student’s t test. Scale bar, 2&nbsp;μm.</div><div id="fspara0220" role="paragraph">(J and K) Representative traces, frequency, and amplitude of mIPSCs and mEPSCs (n&nbsp;= 13 cells from 3 control and n&nbsp;= 14 cells from 3 cKO mice) at P15. <sup>∗</sup>p&nbsp;&lt; 0.05; ns, p &gt; 0.05, Student’s t test.</div><div id="fspara0225" role="paragraph">(L) mEPSC/mIPSC ratio; n&nbsp;= 13 cells from 3 controls (Ctl) and n&nbsp;= 14 cells from 3 cKO mice (cKO). ns, <sup>∗</sup>&nbsp;&lt; 0.05; Student’s t test.</div><div id="fspara0230" role="paragraph">cKO, <i>Cx3cr1</i><sup><i>Cre</i>/+</sup><i>;GABA</i><sub><i>B1</i></sub><i>R</i><sup><i>fl</i>/<i>fl</i></sup>. Full and empty arrowheads indicate colocalization and boutons not meeting criteria. Insets in (H) and (I) show masks. Data are mean ± SEM. See also <a id="crosref1795" href="#figs4">Figures S4</a> and <a id="crosref1800" href="#figs5">S5</a>.</div></div></div></div></figcaption></figure></div><div class="figure-wrap"><figure id="figs4" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.06.018/asset/0b84e907-c8f4-4bba-b696-8ed2afb48ec9/main.assets/figs4_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs4" src="/cms/10.1016/j.cell.2021.06.018/asset/456b417f-43eb-4693-8be1-ca4822c7c844/main.assets/figs4.jpg" height="958" width="623" aria-labelledby="figs4-title figs4-content" data-viewer-src="/cms/10.1016/j.cell.2021.06.018/asset/0b84e907-c8f4-4bba-b696-8ed2afb48ec9/main.assets/figs4_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs4-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S4</span> <span class="figure__title__text">Synaptic phenotype of GABA<sub>B1</sub>R cKO mice, related to <a id="crosref1905" href="#fig3">Figure&nbsp;3</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs4-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0620" role="paragraph">(A) Left: quantitation of PV boutons contacted by microglia at P15 in wild-type control mice (n&nbsp;= 7, <i>Cx3cr1</i><sup><i>+/+</i></sup><i>;GABA</i><sub><i>B1</i></sub><i>R</i><sup><i>fl/fl</i></sup>), GABA<sub>B1</sub>R cKO mice (n&nbsp;= 5, <i>Cx3cr1</i><sup><i>Cre/+</i></sup><i>; GABA</i><sub><i>B1</i></sub><i>R</i><sup><i>fl/fl</i></sup>) and <i>Cx3cr1</i><sup><i>-/+</i></sup> heterozygous control mice (n&nbsp;= 8, <i>Cx3cr1</i><sup><i>Cre/+</i></sup><i>; GABA</i><sub><i>B1</i></sub><i>R</i><sup><i>+/+</i></sup>) mice. One-Way ANOVA followed by Tukey’s multiple comparisons test.&nbsp;Wild-type control and cKO data are the same as shown in <a id="crosref1910" href="#fig4">Figure&nbsp;4</a>A. Right: quantitation of PV boutons contacted by microglia at P15 in control (n&nbsp;= 4, <i>Tmem119</i><sup><i>+/+</i></sup><i>;GABA</i><sub><i>B1</i></sub><i>R</i><sup><i>fl/fl</i></sup>) and cKO (n&nbsp;= 6, <i>Tmem119</i><sup><i>CreER/+</i></sup><i>;GABA</i><sub><i>B1</i></sub><i>R</i><sup><i>fl/fl</i></sup>) mice. Student’s t test. <sup>∗</sup>p&nbsp;&lt; 0.05, <sup>∗∗</sup>p&nbsp;&lt; 0.01.</div><div id="fspara0625" role="paragraph">(B) Left: quantitation of VGlut2 boutons contacted by microglia at P15 in wild-type control mice (n&nbsp;= 7, <i>Cx3cr1</i><sup><i>+/+</i></sup><i>;GABA</i><sub><i>B1</i></sub><i>R</i><sup><i>fl/fl</i></sup>), GABA<sub>B1</sub>R cKO mice (n&nbsp;= 5, <i>Cx3cr1</i><sup><i>Cre/+</i></sup><i>; GABA</i><sub><i>B1</i></sub><i>R</i><sup><i>fl/fl</i></sup>) and <i>Cx3cr1</i><sup><i>-/+</i></sup> heterozygous control mice (n&nbsp;= 6, <i>Cx3cr1</i><sup><i>Cre/+</i></sup><i>; GABA</i><sub><i>B1</i></sub><i>R</i><sup><i>+/+</i></sup>) mice. One-Way ANOVA followed by Tukey’s multiple comparisons test. Wild-type control and cKO data are the same as shown in <a id="crosref1915" href="#fig4">Figure&nbsp;4</a>B. Right: quantitation of VGlut2 boutons contacted by microglia at P15 in control (n&nbsp;= 8, <i>Tmem119</i><sup><i>+/+</i></sup><i>;GABA</i><sub><i>B1</i></sub><i>R</i><sup><i>fl/fl</i></sup>) and cKO (n&nbsp;= 8, <i>Tmem119</i><sup><i>CreER/+</i></sup><i>;GABA</i><sub><i>B1</i></sub><i>R</i><sup><i>fl/fl</i></sup>) mice. Student’s t test. ns p &gt; 0.05.</div><div id="fspara0630" role="paragraph">(C) Distribution of microglia-PV boutons interactions for the indicated duration during the <i>in&nbsp;vivo</i> imaging experiments in controls (n&nbsp;= 181 PV boutons from 6 mice) and cKOs (<i>Cx3cr1</i><sup><i>GFP/+</i></sup><i>;Tmem119</i><sup><i>CreER/+</i></sup><i>;GABA</i><sub><i>B1</i></sub><i>R</i><sup><i>fl/fl</i></sup>, n&nbsp;= 99 PV boutons from 3 mice) cKO mice. The distribution is shown as kernel density estimation of the proportion of PV boutons contacted by microglia for the indicated duration. Control data are the same as in Figure&nbsp;S2D but limited to a total duration of 20&nbsp;minutes to allow comparison with cKO data (see <a id="crosref1920" href="#sec-4">STAR Methods</a>). <sup>∗∗</sup>p&nbsp;&lt; 0.01, Kolmogorov–Smirnov test. Note the complex effect of GABA<sub>B1</sub>R removal on the contact duration, the cKO distribution is bimodal with some contacts having a shorter duration and others being abnormally long.</div><div id="fspara0635" role="paragraph">(D) Left: quantitation of Syt2+Gephyrin+ synapses made by PV cells onto excitatory neurons at P15 in wild-type control mice (n&nbsp;= 8, <i>Cx3cr1</i><sup><i>+/+</i></sup><i>;GABA</i><sub><i>B1</i></sub><i>R</i><sup><i>fl/fl</i></sup>), GABA<sub>B1</sub>R cKO mice (n&nbsp;= 6, <i>Cx3cr1</i><sup><i>Cre/+</i></sup><i>; GABA</i><sub><i>B1</i></sub><i>R</i><sup><i>fl/fl</i></sup>) and <i>Cx3cr1</i><sup><i>-/+</i></sup> heterozygous control mice (n&nbsp;= 6, <i>Cx3cr1</i><sup><i>Cre/+</i></sup><i>; GABA</i><sub><i>B1</i></sub><i>R</i><sup><i>+/+</i></sup>) mice. One-Way ANOVA followed by Holm-Sidak’s multiple comparisons test. Wild-type control and cKO data are the same as shown in <a id="crosref1925" href="#fig4">Figure&nbsp;4</a>H. Right: Syt2+Gephyrin+ synapses made by PV cells onto excitatory neurons at P15 in control (n&nbsp;= 8, <i>Tmem119</i><sup><i>+/+</i></sup><i>;GABA</i><sub><i>B1</i></sub><i>R</i><sup><i>fl/fl</i></sup>) and cKO (n&nbsp;= 6, <i>Tmem119</i><sup><i>CreER/+</i></sup><i>;GABA</i><sub><i>B1</i></sub><i>R</i><sup><i>fl/fl</i></sup>) mice. Student’s t test. <sup>∗</sup>p&nbsp;&lt; 0.05, <sup>∗∗</sup>p&nbsp;&lt; 0.01.</div><div id="fspara0640" role="paragraph">(E) Image, relative frequency histogram and overlaid kernel density estimation of Syt2+Gephyrin+ synapses imaged using STED super-resolution microscopy and made by PV cells onto excitatory neurons at P15 in wild-type controls (n&nbsp;= 58 cells from 3 <i>Cx3cr1</i><sup><i>+/+</i></sup><i>;GABA</i><sub><i>B1</i></sub><i>R</i><sup><i>fl/fl</i></sup> mice) and GABA<sub>B1</sub>R cKO mice (n&nbsp;= 66 cells from 3 <i>Cx3cr1</i><sup><i>Cre/+</i></sup><i>; GABA</i><sub><i>B1</i></sub><i>R</i><sup><i>fl/fl</i></sup> mice). <sup>∗∗∗</sup>p&nbsp;&lt; 0.001, Kolmogorov–Smirnov test. The dotted line indicates the soma outline.</div><div id="fspara0645" role="paragraph">(F) Left: quantitation of VGlut2+Homer1+ synapses made onto excitatory neurons at P15 in wild-type control mice (n&nbsp;= 6, <i>Cx3cr1</i><sup><i>+/+</i></sup><i>;GABA</i><sub><i>B1</i></sub><i>R</i><sup><i>fl/fl</i></sup>), GABA<sub>B1</sub>R cKO mice (n&nbsp;= 7, <i>Cx3cr1</i><sup><i>Cre/+</i></sup><i>; GABA</i><sub><i>B1</i></sub><i>R</i><sup><i>fl/fl</i></sup>) and <i>Cx3cr1</i><sup><i>-/+</i></sup> heterozygous control mice (n&nbsp;= 9, <i>Cx3cr1</i><sup><i>Cre/+</i></sup><i>; GABA</i><sub><i>B1</i></sub><i>R</i><sup><i>+/+</i></sup>) mice. One-Way ANOVA followed by Holm-Sidak’s multiple comparisons test. Wild-type control and cKO data are the same as shown in <a id="crosref1930" href="#fig4">Figure&nbsp;4</a>I. Right: VGlut2+Homer1+ synapses made onto excitatory neurons at P15 in control (n&nbsp;= 9, <i>Tmem119</i><sup><i>+/+</i></sup><i>;GABA</i><sub><i>B1</i></sub><i>R</i><sup><i>fl/fl</i></sup>) and cKO (n&nbsp;= 8, <i>Tmem119</i><sup><i>CreER/+</i></sup><i>;GABA</i><sub><i>B1</i></sub><i>R</i><sup><i>fl/fl</i></sup>) mice. Student’s t test. ns p &gt; 0.05.</div><div id="fspara0650" role="paragraph">(G) Quantitation of VGlut2+Homer1+ synapses onto L4 PV neurons in P15 control (n&nbsp;= 6, <i>Cx3cr1</i><sup><i>+/+</i></sup><i>;GABA</i><sub><i>B1</i></sub><i>R</i><sup><i>fl/fl</i></sup>) and cKO (n&nbsp;= 5, <i>Cx3cr1</i><sup><i>Cre/+</i></sup><i>; GABA</i><sub><i>B1</i></sub><i>R</i><sup><i>fl/fl</i></sup>) mice. ns p &gt; 0.05, Student’s t test.</div><div id="fspara0655" role="paragraph">(H) Quantitation of VGlut1+Homer1+ synapses onto L4 excitatory cells (left) or pyramidal cell dendrites (right) in control (n&nbsp;= 7-9, <i>Cx3cr1</i><sup><i>+/+</i></sup><i>;GABA</i><sub><i>B1</i></sub><i>R</i><sup><i>fl/fl</i></sup>) and GABA<sub>B1</sub>R cKO (n&nbsp;= 6-7, <i>Cx3cr1</i><sup><i>Cre/+</i></sup><i>; GABA</i><sub><i>B1</i></sub><i>R</i><sup><i>fl/fl</i></sup>) mice at P15. ns p &gt; 0.05, Mann-Whitney test for somatic analysis and Student’s t test for dendritic analysis.</div><div id="fspara0660" role="paragraph">(I) and (J) Traces, frequency and amplitude of mIPSCs (n&nbsp;= 26 cells from 3 <i>Tmem119</i><sup><i>+/+</i></sup><i>;GABA</i><sub><i>B1</i></sub><i>R</i><sup><i>fl/fl</i></sup> control mice and n&nbsp;= 28 cells from 4 <i>Tmem119</i><sup><i>CreER/+</i></sup><i>; GABA</i><sub><i>B1</i></sub><i>R</i><sup><i>fl/fl</i></sup> cKO mice) and mEPSCs (n&nbsp;= 26 cells from 3 <i>Tmem119</i><sup><i>+/+</i></sup><i>; GABA</i><sub><i>B1</i></sub><i>R</i><sup><i>fl/fl</i></sup> control mice and n&nbsp;= 31 cells from 4 <i>Tmem119</i><sup><i>CreER/+</i></sup><i>;GABA</i><sub><i>B1</i></sub><i>R</i><sup><i>fl/fl</i></sup> cKO mice) recorded from layer 4 excitatory neurons at P15. <sup>∗∗∗</sup>p&nbsp;&lt; 0.001, ns p &gt; 0.05, Student’s t test except for mEPSC amplitude where Mann-Whitney test was used.</div><div id="fspara0665" role="paragraph">(K) mEPSC/mIPSC frequency ratio (n&nbsp;= 26 cells from 3 <i>Tmem119</i><sup><i>+/+</i></sup><i>; GABA</i><sub><i>B1</i></sub><i>R</i><sup><i>fl/fl</i></sup> control mice and n&nbsp;= 28 cells from 4 <i>Tmem119</i><sup><i>CreER/+</i></sup><i>;GABA</i><sub><i>B1</i></sub><i>R</i><sup><i>fl/fl</i></sup> cKO mice). <sup>∗∗∗</sup>p&nbsp;&lt; 0.001, Mann-Whitney test.</div><div id="fspara0670" role="paragraph">(L) Left: quantitation of Syt2+Gephyrin+ synapses made by PV cells onto S1 L4 excitatory neurons in control (n&nbsp;= 7, <i>Cx3cr1</i><sup><i>+/+</i></sup><i>;GABA</i><sub><i>B1</i></sub><i>R</i><sup><i>fl/fl</i></sup>) and GABA<sub>B1</sub>R cKO (n&nbsp;= 7, <i>Cx3cr1</i><sup><i>Cre/+</i></sup><i>; GABA</i><sub><i>B1</i></sub><i>R</i><sup><i>fl/fl</i></sup>) mice at P30. Right: quantitation of Syt2+Gephyrin+ synapses made by PV cells onto S1 L4 excitatory neurons in control (n&nbsp;= 6, <i>Tmem119</i><sup><i>+/+</i></sup><i>;GABA</i><sub><i>B1</i></sub><i>R</i><sup><i>fl/fl</i></sup>) and GABA<sub>B1</sub>R cKO (n&nbsp;= 7, <i>Tmem119</i><sup><i>CreER/+</i></sup><i>;GABA</i><sub><i>B1</i></sub><i>R</i><sup><i>fl/fl</i></sup>) mice at P30. <sup>∗∗</sup>p&nbsp;&lt; 0.01, Student’s t test.</div><div id="fspara0675" role="paragraph">(M) Left: quantitation of Syt2+Gephyrin+ synapses made by PV cells onto V1 L4 excitatory neurons at P30 in control (n&nbsp;= 6, <i>Cx3cr1</i><sup><i>+/+</i></sup><i>;GABA</i><sub><i>B1</i></sub><i>R</i><sup><i>fl/fl</i></sup>) and GABA<sub>B1</sub>R cKO (n&nbsp;= 7, <i>Cx3cr1</i><sup><i>Cre/+</i></sup><i>; GABA</i><sub><i>B1</i></sub><i>R</i><sup><i>fl/fl</i></sup>) mice. Right: quantitation of PV+ Gephyrin+ synapses made onto NeuN+ cells in the dorsolateral striatum of P30 control (n&nbsp;=&nbsp;5, <i>Cx3cr1</i><sup><i>+/+</i></sup><i>;GABA</i><sub><i>B1</i></sub><i>R</i><sup><i>fl/fl</i></sup>) and cKO (n&nbsp;= 6, <i>Cx3cr1</i><sup><i>Cre/+</i></sup><i>; GABA</i><sub><i>B1</i></sub><i>R</i><sup><i>fl/fl</i></sup>) mice. <sup>∗</sup>p&nbsp;&lt; 0.05, <sup>∗∗</sup>p&nbsp;&lt; 0.01, Student’s t test.</div><div id="fspara0680" role="paragraph">(<b>N</b>) Images, 3D reconstruction and quantitation of SST synapses (GAD65+Gephyrin+ synapses with GAD65 presynaptic boutons completely encased within genetically labeled SST axons) in layer 1 of S1 at P15 in control (n&nbsp;= 5; <i>SST</i><sup><i>Flp/+</i></sup><i>; RCE:FRT; Cx3cr1</i><sup><i>Cre/+</i></sup><i>; GABAB1R</i><sup><i>fl/fl</i></sup>) and GABA<sub>B1</sub>R cKO (n&nbsp;= 4; <i>SST</i><sup><i>Flp/+</i></sup><i>; RCE:FRT; Cx3cr1</i><sup><i>+/+</i></sup><i>; GABAB1R</i><sup><i>fl/fl</i></sup>) mice. <sup>∗</sup>p&nbsp;&lt; 0.05, Student’s t test.</div><div id="fspara0685" role="paragraph">All data are mean ± SEM, each data point represents one experimental animal except in I-K where they represent cells. Scale bars equal 1&nbsp;μm. Arrowheads indicate colocalization.</div></div></div></div></figcaption></figure></div><div id="p0115" role="paragraph">We reasoned that if GABA<sub>B</sub>R signaling contributes to microglia-mediated remodeling of inhibitory synapses, cKO and microglia-depleted mice may display similar abnormalities in connectivity (<a id="crosref0695" href="#fig1">Figures 1</a>, <a id="crosref0700" href="#fig4">4</a>F–4K, and <a id="crosref0705" href="#figs4">S4</a>D–S4K). To test this hypothesis, we quantified the density of PV inhibitory synapses contacting the soma of excitatory neurons in P15 mutants. We found that excitatory cells received significantly more PV synapses in GABA<sub>B</sub>R mutants than were observed in control neurons (<a id="crosref0710" href="#fig4">Figures 4</a>G, <a id="crosref0715" href="#figs4">S4</a>D, and S4E). These data demonstrate that the removal of GABA<sub>B</sub>Rs from microglia phenocopies the changes in PV efferent connectivity observed when microglia are depleted. In contrast, no change in the density of excitatory inputs occurred (<a id="crosref0720" href="#fig4">Figures 4</a>H and <a id="crosref0725" href="#figs4">S4</a>F–S4H). We confirmed this selective effect on inhibitory connectivity by recording synaptic activity from P15 control and cKO mice. Analysis of mIPSCs and mEPSCs demonstrated that while inhibitory events were significantly increased in frequency, excitatory currents were unaffected (<a id="crosref0730" href="#fig4">Figures 4</a>I, 4J, <a id="crosref0735" href="#figs4">S4</a>I, and S4J). As a result, the mEPSC/mIPSC frequency ratio was decreased in cKO mice (<a id="crosref0740" href="#fig4">Figures 4</a>K and <a id="crosref0745" href="#figs4">S4</a>K). Together, these results reveal that removal of GABA<sub>B1</sub>Rs decouples the effects of microglia perturbation upon inhibitory versus excitatory synapses.</div><div id="p0120" role="paragraph">The increased PV innervation of mutants persisted at P30 (<a id="crosref0750" href="#figs4">Figure&nbsp;S4</a>L) and was detected in both V1 and in the dorsolateral striatum (<a id="crosref0755" href="#figs4">Figure&nbsp;S4</a>M). Moreover, disrupting GABA<sub>B</sub>R signaling in microglia also affected the efferent connectivity of SST interneurons (<a id="crosref0760" href="#figs4">Figure&nbsp;S4</a>N). Of note, the presence of supernumerary synapses was not due to off-targets effects such as changes in the density of microglia or PV and SST interneurons, defects in interneuron survival or synapse assembly, changes in the fraction of PV cells surrounded by perineuronal nets (<span class="dropBlock reference-citations"><a id="body-ref-optPooQj0tqrk" href="#" role="doc-biblioref" data-xml-rid="bib91 bib90" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="optPooQj0tqrk" aria-controls="optPooQj0tqrk">Crapser et al., 2020; Nguyen et al., 2020</a><div class="dropBlock__holder" data-db-target-of="optPooQj0tqrk"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">15.</div><div>Crapser, D., J. ∙ Ochaba, J. ∙ Soni, N. ...</div><div><strong>Microglial depletion prevents extracellular matrix changes and striatal volume reduction in a model of Huntington’s disease.</strong></div><div><em>Brain.</em> 2020; <strong>143</strong>:266-288</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/brain/awz363" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85077346689" target="_blank" aria-label="Open Scopus in new tab">Scopus (78)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31848580/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbrain%2Fawz363&amp;pmid=31848580" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">55.</div><div>Nguyen, T., P. ∙ Dorman, C., L. ∙ Pan ...</div><div><strong>Microglial Remodeling of the Extracellular Matrix Promotes Synapse Plasticity</strong></div><div><em>Cell.</em> 2020; <strong>182</strong>:388-403</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.cell.2020.05.050&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.cell.2020.05.050&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85087989807" target="_blank" aria-label="Open Scopus in new tab">Scopus (298)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32615087/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2020.05.050&amp;pmid=32615087" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) or GABA<sub>B1</sub>R deletion from neurons (<a id="crosref0765" href="#figs5">Figures S5</a>A–S5G). In addition, mutant mice did not exhibit increased susceptibility to epilepsy (<a id="crosref0770" href="#figs5">Figures S5</a>H–S5J). Importantly, although excitatory synapses remained unaltered, inhibitory synapses were decreased in P60 cKO mice compared to controls (<a id="crosref0775" href="#figs5">Figures S5</a>K–S5O). Depleting microglia from P30 to P60 did not prevent this phenotype (<a id="crosref0780" href="#figs5">Figure&nbsp;S5</a>M), demonstrating that the late loss of inhibitory synapses in these mutants is not microglia-dependent. Together, these findings indicate that GABA-receptive microglia selectively mediate inhibitory synapse remodeling, and disrupting this process leads to permanent defects in inhibitory connectivity.</div><div class="figure-wrap"><figure id="figs5" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.06.018/asset/247dce6d-4c92-477b-8b3e-b6b7716140bd/main.assets/figs5_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs5" src="/cms/10.1016/j.cell.2021.06.018/asset/dde7322c-2b64-459b-861d-65120efd2326/main.assets/figs5.jpg" height="958" width="619" aria-labelledby="figs5-title figs5-content" data-viewer-src="/cms/10.1016/j.cell.2021.06.018/asset/247dce6d-4c92-477b-8b3e-b6b7716140bd/main.assets/figs5_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs5-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S5</span> <span class="figure__title__text">The synaptic phenotype of GABA<sub>B1</sub>R cKO mice is not due to off-target effects, related to <a id="crosref1935" href="#fig3">Figure&nbsp;3</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs5-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0695" role="paragraph">(A) Representative images and quantitation of the density of microglia in control (n&nbsp;= 7, <i>Cx3cr1</i><sup><i>+/+</i></sup><i>;GABA</i><sub><i>B1</i></sub><i>R</i><sup><i>fl/fl</i></sup>) and GABA<sub>B1</sub>R cKO (n&nbsp;= 5, <i>Cx3cr1</i><sup><i>Cre/+</i></sup><i>; GABA</i><sub><i>B1</i></sub><i>R</i><sup><i>fl/fl</i></sup>) mice at P15. ns p &gt; 0.05, Student’s t test. Scale bar equal 100&nbsp;μm.</div><div id="fspara0700" role="paragraph">(B) Representative images and quantitation of the density of PV cells in control (n&nbsp;= 6, <i>Cx3cr1</i><sup><i>+/+</i></sup><i>;GABA</i><sub><i>B1</i></sub><i>R</i><sup><i>fl/fl</i></sup>) and GABA<sub>B1</sub>R cKO (n&nbsp;= 5, <i>Cx3cr1</i><sup><i>Cre/+</i></sup><i>; GABA</i><sub><i>B1</i></sub><i>R</i><sup><i>fl/fl</i></sup>) mice at P15. ns p &gt; 0.05, Student’s t test. Scale bar equal 100&nbsp;μm.</div><div id="fspara0705" role="paragraph">(C) Representative images and quantitation of the density of SST cells (GFP<sup>+</sup>) in control (n&nbsp;= 8, <i>SST</i><sup><i>Flp/+</i></sup><i>; RCE:FRT; Cx3cr1</i><sup><i>Cre/+</i></sup><i>; GABAB1R</i><sup><i>fl/fl</i></sup>) and GABA<sub>B1</sub>R cKO (n&nbsp;= 7, <i>SST</i><sup><i>Flp/+</i></sup><i>; RCE:FRT; Cx3cr1</i><sup><i>+/+</i></sup><i>; GABAB1R</i><sup><i>fl/fl</i></sup>) mice at P15. ns p &gt; 0.05, Student’s t test. Scale bar equal 100&nbsp;μm.</div><div id="fspara0710" role="paragraph">(D) Quantitation of the density of PV and SST cells in control (n&nbsp;= 6, <i>Lhx6</i><sup><i>GFP/+</i></sup><i>; Cx3cr1</i><sup><i>+/+</i></sup><i>; GABAB1R</i><sup><i>fl/fl</i></sup>) and GABA<sub>B1</sub>R cKO (n&nbsp;= 6, <i>Lhx6</i><sup><i>GFP/+</i></sup><i>; Cx3cr1</i><sup><i>Cre/+</i></sup><i>; GABAB1R</i><sup><i>fl/fl</i></sup>) mice at P10. PV cells were identified as GFP<sup>+</sup>SST<sup>-</sup> cells. ns p &gt; 0.05, Student’s t test.</div><div id="fspara0715" role="paragraph">(E) Quantitation of Syt2+Gephyrin+ synapses made by PV cells onto L4 excitatory neurons at P10 in control (n&nbsp;= 3, <i>Cx3cr1</i><sup><i>+/+</i></sup><i>;GABA</i><sub><i>B1</i></sub><i>R</i><sup><i>fl/fl</i></sup>) and cKO (n&nbsp;= 6, <i>Cx3cr1</i><sup><i>Cre/+</i></sup><i>; GABA</i><sub><i>B1</i></sub><i>R</i><sup><i>fl/fl</i></sup>) mice. ns p &gt; 0.05, Student’s t test.</div><div id="fspara0720" role="paragraph">(F) Representative images and quantitation of the fraction of L4 PV cells surrounded by perineuronal nets (WFA) in control (n&nbsp;= 4, <i>Cx3cr1</i><sup><i>+/+</i></sup><i>;GABA</i><sub><i>B1</i></sub><i>R</i><sup><i>fl/fl</i></sup>) and GABA<sub>B1</sub>R cKO (n&nbsp;= 5, <i>Cx3cr1</i><sup><i>Cre/+</i></sup><i>; GABA</i><sub><i>B1</i></sub><i>R</i><sup><i>fl/fl</i></sup>) mice at P15 and P30. ns p &gt; 0.05, One-Way ANOVA and Holm-Sidak multiple comparisons test. Scale bar equal 20&nbsp;μm.</div><div id="fspara0725" role="paragraph">(G) Quantitation of neurons (labeled with NeuN) expressing <i>Gabbr1</i> mRNA in control (n&nbsp;= 4, <i>Cx3cr1</i><sup><i>+/+</i></sup><i>;GABA</i><sub><i>B1</i></sub><i>R</i><sup><i>fl/fl</i></sup>) and GABA<sub>B1</sub>R cKO (n&nbsp;= 3, <i>Cx3cr1</i><sup><i>Cre/+</i></sup><i>; GABA</i><sub><i>B1</i></sub><i>R</i><sup><i>+/+</i></sup>) mice. <i>Gabbr1</i> mRNA was detected by single-molecule <i>in situ</i> hybridization specifically for the exons deleted in the cKOs (7 and 8). ns p &gt; 0.05, Student’s t test.</div><div id="fspara0730" role="paragraph">(H) Time course of mean Racine scores of seizures induced by intraperitoneal injection of kainic acid (20&nbsp;mg/kg) into control (n&nbsp;= 7, <i>Cx3cr1</i><sup><i>+/+</i></sup><i>;GABA</i><sub><i>B1</i></sub><i>R</i><sup><i>fl/fl</i></sup>) and GABA<sub>B1</sub>R cKO (n&nbsp;= 7, <i>Cx3cr1</i><sup><i>Cre/+</i></sup><i>; GABA</i><sub><i>B1</i></sub><i>R</i><sup><i>fl/fl</i></sup>) mice. Seizures were scored as described in the <a id="crosref1940" href="#sec-4">STAR Methods</a>. Two-Way ANOVA for repeated-measures with genotype (p &gt; 0.05) and time (p&nbsp;&lt; 0.001) as factors. Interaction between time and genotype p &gt; 0.05.</div><div id="fspara0735" role="paragraph">(I) Maximum and cumulative seizure scores over 90&nbsp;minutes from the time of kainic acid injection. ns p &gt; 0.05, Mann-Whitney test.</div><div id="fspara0740" role="paragraph">(J) Latency to reach the status epilepticus and/or death. ns p &gt; 0.05, Mann-Whitney test.</div><div id="fspara0745" role="paragraph">(K) and (L) Traces, frequency and amplitude of mIPSCs (n&nbsp;= 16 cells from 3 control and n&nbsp;= 22 cells from 3 cKO mice) and mEPSCs (n&nbsp;= 18 cells from 3 control and n&nbsp;= 25 cells from 3 cKO mice) recorded from layer 4 excitatory neurons at P60. <sup>∗∗</sup>p&nbsp;&lt; 0.01, ns p &gt; 0.05, Mann-Whitney test for mIPSCs and Student’s t test for mEPSCs.</div><div id="fspara0750" role="paragraph">(M) Images and quantitation of Syt2+Gephyrin+ synapses made by PV cells onto excitatory neurons in P60 control (Ctl, n&nbsp;= 5, <i>Cx3cr1</i><sup><i>+/+</i></sup><i>;GABA</i><sub><i>B1</i></sub><i>R</i><sup><i>fl/fl</i></sup>) and cKO (n&nbsp;= 8, <i>Cx3cr1</i><sup><i>Cre/+</i></sup><i>; GABA</i><sub><i>B1</i></sub><i>R</i><sup><i>fl/fl</i></sup>) mice as well as cKO (n&nbsp;= 5) and control (n&nbsp;= 3) mice that underwent microglia depletion from P30 to P60. ns p &gt; 0.05, <sup>∗</sup>p&nbsp;&lt; 0.05, One-Way ANOVA followed by Sidak’s multiple comparisons test. Scale bar equal 2&nbsp;μm.</div><div id="fspara0755" role="paragraph">(N) Quantitation of VGlut2+Homer1+ synapses made onto excitatory neurons in P60 control (Ctl, n&nbsp;= 5, <i>Cx3cr1</i><sup><i>+/+</i></sup><i>;GABA</i><sub><i>B1</i></sub><i>R</i><sup><i>fl/fl</i></sup>) and cKO (n&nbsp;= 8, <i>Cx3cr1</i><sup><i>Cre/+</i></sup><i>; GABA</i><sub><i>B1</i></sub><i>R</i><sup><i>fl/fl</i></sup>) mice. ns p &gt; 0.05, Student’s t test.</div><div id="fspara0760" role="paragraph">(O) Quantitation of PV+ Gephyrin+ synapses made onto NeuN+ cells in the dorsolateral striatum of P60 control (Ctl, n&nbsp;= 7, <i>Cx3cr1</i><sup><i>+/+</i></sup><i>;GABA</i><sub><i>B1</i></sub><i>R</i><sup><i>fl/fl</i></sup>) and cKO (n&nbsp;= 8, <i>Cx3cr1</i><sup><i>Cre/+</i></sup><i>; GABA</i><sub><i>B1</i></sub><i>R</i><sup><i>fl/fl</i></sup>) mice. <sup>∗</sup>p&nbsp;&lt; 0.05, Student’s t test.</div><div id="fspara0765" role="paragraph">All data are mean ± SEM, each data point represents one experimental animal except in K and L where they represent cells. Arrowheads indicate colocalization.</div></div></div></div></figcaption></figure></div></section><section id="sec-2-5"><h3>Ablation of GABA<sub>B1</sub>Rs within microglia alters genes involved in synapse remodeling</h3><div id="p0125" role="paragraph">To explore the molecular mechanisms downstream of GABA<sub>B1</sub>Rs in microglia, we isolated P15 wild-type (WT) (<i>Cx3cr1</i><sup>+/+</sup><i>;GABA</i><sub><i>B1</i></sub><i>R</i><sup><i>fl</i>/<i>fl</i></sup>) and cKO (<i>Cx3cr1</i><sup><i>Cre</i>/+</sup><i>;GABA</i><sub><i>B1</i></sub><i>R</i><sup><i>fl</i>/<i>fl</i></sup>) microglia from the S1 cortex and performed single-cell RNA sequencing (scRNA-seq) (<a id="crosref0785" href="#fig5">Figures 5</a> and <a id="crosref0790" href="#figs6">S6</a>; <a id="crosref0795" href="#mmc1">Table S1</a>). To identify transcriptional changes due to the loss of one copy of <i>Cx3cr1</i>, we examined <i>Cx3cr1</i><sup><i>Cre</i>/+</sup><i>;GABA</i><sub><i>B1</i></sub><i>R</i><sup>+/+</sup> mice as an additional control (Cre-Het).</div><div class="figure-wrap"><figure id="fig5" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.06.018/asset/8359678e-4eab-4311-9821-fd183de19169/main.assets/gr5_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr5" src="/cms/10.1016/j.cell.2021.06.018/asset/7715c85a-b092-41dc-8ccc-95d979782543/main.assets/gr5.jpg" height="920" width="657" aria-labelledby="fig5-title fig5-content" data-viewer-src="/cms/10.1016/j.cell.2021.06.018/asset/8359678e-4eab-4311-9821-fd183de19169/main.assets/gr5_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig5-title" class="figure__title"><span class="label figure__label">Figure&nbsp;5</span> <span class="figure__title__text">Ablation of GABA<sub>B1</sub>Rs within microglia alters genes involved in synapse remodeling</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig5-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0240" role="paragraph">(A) UMAP plots of WT microglia showing 5 clusters and scaled expression of representative enriched genes.</div><div id="fspara0245" role="paragraph">(B) UMAP plots of WT and cKO integrated scRNA-seq dataset.</div><div id="fspara0250" role="paragraph">(C) Same as (B), showing 8 mixed clusters and representative enriched genes.</div><div id="fspara0255" role="paragraph">(D) Percentage of WT and cKO microglia composing each cluster.</div><div id="fspara0260" role="paragraph">(E) Mixed cluster contributions to total differentially expressed genes (DEGs) between WT and cKO microglia.</div><div id="fspara0265" role="paragraph">(F) Gene Ontology analysis of downregulated genes from cluster 4.</div><div id="fspara0270" role="paragraph">(G) Violin plots of normalized log-expression values for representative genes significantly downregulated in cKO microglia from cluster 4.</div><div id="fspara0275" role="paragraph">(H) Heatmap showing scaled expression of genes downregulated in cKO.</div><div id="fspara0280" role="paragraph">(I) Cluster 4 subclusters and percentage of WT and cKO microglia. Representative downregulated genes are highlighted.</div><div id="fspara0285" role="paragraph">See also <a id="crosref1805" href="#figs6">Figure&nbsp;S6</a> and <a id="crosref1810" href="#mmc1">Tables S1</a> and <a id="crosref1815" href="#mmc2">S2</a>.</div></div></div></div></figcaption></figure></div><div class="figure-wrap"><figure id="figs6" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.06.018/asset/97842390-3569-4192-a4f7-e0c18ca377ed/main.assets/figs6_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs6" src="/cms/10.1016/j.cell.2021.06.018/asset/784573d0-e154-467c-b4e8-db8446d42730/main.assets/figs6.jpg" height="958" width="731" aria-labelledby="figs6-title figs6-content" data-viewer-src="/cms/10.1016/j.cell.2021.06.018/asset/97842390-3569-4192-a4f7-e0c18ca377ed/main.assets/figs6_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs6-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S6</span> <span class="figure__title__text">Loss of GABA<sub>B1</sub>Rs does not fundamentally alter the range of microglial states but causes a downregulation of pruning-related genes, related to <a id="crosref1945" href="#fig5">Figure&nbsp;5</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs6-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0775" role="paragraph">(A) UMAP plots showing the integration of P15 wild-type microglia with embryonic, early postnatal and P30 datasets from <span class="dropBlock reference-citations"><a id="body-ref-sref31-2" href="#" role="doc-biblioref" data-xml-rid="bib31" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref31-2" aria-controls="sref31-2">Hammond et&nbsp;al. (2019)</a><div class="dropBlock__holder" data-db-target-of="sref31-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">32.</div><div>Hammond, T.R. ∙ Dufort, C. ∙ Dissing-Olesen, L. ...</div><div><strong>Single-Cell RNA Sequencing of Microglia throughout the Mouse Lifespan and in the Injured Brain Reveals Complex Cell-State Changes</strong></div><div><em>Immunity.</em> 2019; <strong>50</strong>:253-271.e6</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.immuni.2018.11.004&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.immuni.2018.11.004&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85056738677" target="_blank" aria-label="Open Scopus in new tab">Scopus (1123)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30471926/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.immuni.2018.11.004&amp;pmid=30471926" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>.</div><div id="fspara0780" role="paragraph">(B) UMAP plot of Cre-Het control (<i>Cx3cr1</i><sup><i>Cre/+</i></sup>) microglia showing 5 clusters and representative genes enriched in each cluster.</div><div id="fspara0785" role="paragraph">(C) UMAP plot of Cre-Het control microglia from cluster 1 further subjected to subclustering. Representative enriched genes are shown.</div><div id="fspara0790" role="paragraph">(D) UMAP plot of GABA<sub>B1</sub>R cKO microglia from P15 S1 cortex showing 7 main clusters and representative genes enriched in each cluster.</div><div id="fspara0795" role="paragraph">(E) Pie charts indicating the contribution of each wild-type or cKO cluster to the clusters of the integrated dataset. The overall filled area is proportional to the total number of cells within each cluster.</div><div id="fspara0800" role="paragraph">(F) UMAP plots of Cre-Het control and cKO integrated scRNaseq dataset. See methods for the different integration of this dataset as compared to WT control and cKO data.</div><div id="fspara0805" role="paragraph">(G) Same as (F), showing 10 mixed clusters and representative genes enriched in each cluster.</div><div id="fspara0810" role="paragraph">(H) Mixed cluster contributions to total differentially expressed genes (DEGs) between Cre-Het control and cKO microglia.</div><div id="fspara0815" role="paragraph">(I) Violin plots of normalized log-expression values for representative genes significantly downregulated in cKO microglia from Cre-Het-cKO cluster 3. See methods for the different integration of Cre-Het control and cKO data as compared to WT control and cKO data.</div><div id="fspara0820" role="paragraph">(J) Violin plots of normalized log-expression values for representative genes significantly downregulated in cKO microglia from Cre-Het-cKO cluster 5.</div><div id="fspara0825" role="paragraph">(K) Heatmap showing the output of an elastic net regularized classifier trained to learn which genes are predictive of control wild-type and GABA<sub>B1</sub>R cKO microglia using all genes as features and wild-type or cKO as labels. Genes are ordered by increasing mean expression. The top 5 genes are predictive of wild-type microglia (downregulated in cKO) and the remaining 3 genes are predictive of cKO microglia (upregulated in cKO).</div><div id="fspara0830" role="paragraph">(L) UMAP plots of control and cKO integrated scRNaseq dataset after denoising expression data with a weighted affinity kernel and self-supervision method (DEWÄKSS, see <a id="crosref1955" href="#sec-4">STAR Methods</a>).</div><div id="fspara0835" role="paragraph">(M) Same as (L) followed by unsupervised clustering and showing 8 mixed clusters. See also <a id="crosref1960" href="#mmc2">Table S2</a> for genes enriched in each cluster.</div><div id="fspara0840" role="paragraph">(N) Volcano plot showing differentially expressed genes between control and cKO microglia from cluster 4 after denoising. The negative log<sub>10</sub>-transformed p&nbsp;values are plotted against the log<sub>2</sub> fold change. Differentially expressed genes with an absolute log<sub>2</sub> fold change higher than 0.5 and -log<sub>10</sub> (p value) higher than or equal to 20 are depicted in orange. Top downregulated genes are highlighted. A full list is shown in <a id="crosref1965" href="#mmc2">Table S2</a>.</div></div></div></div></figcaption></figure></div><div id="p0130" role="paragraph">Upon unsupervised clustering, WT microglia segregated in 5 major clusters of which two were notable (<a id="crosref0800" href="#fig5">Figure&nbsp;5</a>A; <a id="crosref0805" href="#mmc1">Table S1</a>). Cluster 1 cells exhibited higher levels of genes previously identified as transcriptional signatures of postnatal immature microglia (e.g., <i>Tmsb4x</i>) (<span class="dropBlock reference-citations"><a id="body-ref-sref38-2" href="#" role="doc-biblioref" data-xml-rid="bib38" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref38-2" aria-controls="sref38-2">Li et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref38-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">39.</div><div>Li, Q. ∙ Cheng, Z. ∙ Zhou, L. ...</div><div><strong>Developmental Heterogeneity of Microglia and Brain Myeloid Cells Revealed by Deep Single-Cell RNA Sequencing</strong></div><div><em>Neuron.</em> 2019; <strong>101</strong>:207-223.e10</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.neuron.2018.12.006&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.neuron.2018.12.006&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85059299323" target="_blank" aria-label="Open Scopus in new tab">Scopus (572)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30606613/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.neuron.2018.12.006&amp;pmid=30606613" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref47-2" href="#" role="doc-biblioref" data-xml-rid="bib47" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref47-2" aria-controls="sref47-2">Masuda et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref47-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">48.</div><div>Masuda, T. ∙ Sankowski, R. ∙ Staszewski, O. ...</div><div><strong>Spatial and temporal heterogeneity of mouse and human microglia at single-cell resolution</strong></div><div><em>Nature.</em> 2019; <strong>566</strong>:388-392</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-0924-x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_48_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85061969567" target="_blank" aria-label="Open Scopus in new tab">Scopus (735)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30760929/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-0924-x&amp;pmid=30760929" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Cells in Cluster 2 expressed higher levels of synapse pruning (e.g., <i>C1qa</i> [<span class="dropBlock reference-citations"><a id="body-ref-sref72-2" href="#" role="doc-biblioref" data-xml-rid="bib72" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref72-2" aria-controls="sref72-2">Stevens et&nbsp;al., 2007</a><div class="dropBlock__holder" data-db-target-of="sref72-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">74.</div><div>Stevens, B. ∙ Allen, N.J. ∙ Vazquez, L.E. ...</div><div><strong>The classical complement cascade mediates CNS synapse elimination</strong></div><div><em>Cell.</em> 2007; <strong>131</strong>:1164-1178</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.cell.2007.10.036&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.cell.2007.10.036&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-36849076770" target="_blank" aria-label="Open Scopus in new tab">Scopus (2260)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/18083105/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2007.10.036&amp;pmid=18083105" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>] and <i>Trem2</i> [<span class="dropBlock reference-citations"><a id="body-ref-sref25-1" href="#" role="doc-biblioref" data-xml-rid="bib25" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref25-1" aria-controls="sref25-1">Filipello et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref25-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">26.</div><div>Filipello, F. ∙ Morini, R. ∙ Corradini, I. ...</div><div><strong>The Microglial Innate Immune Receptor TREM2 Is Required for Synapse Elimination and Normal Brain Connectivity</strong></div><div><em>Immunity.</em> 2018; <strong>48</strong>:979-991.e8</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_26_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.immuni.2018.04.016&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_26_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.immuni.2018.04.016&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_26_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85046738379" target="_blank" aria-label="Open Scopus in new tab">Scopus (391)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29752066/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.immuni.2018.04.016&amp;pmid=29752066" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>]) and homeostatic microglia core (e.g., <i>P2ry12</i>) genes. Of note, WT cells from our dataset could be integrated into a developmental trajectory with microglia ranging from embryonic to P30 (<span class="dropBlock reference-citations"><a id="body-ref-sref31-3" href="#" role="doc-biblioref" data-xml-rid="bib31" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref31-3" aria-controls="sref31-3">Hammond et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref31-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">32.</div><div>Hammond, T.R. ∙ Dufort, C. ∙ Dissing-Olesen, L. ...</div><div><strong>Single-Cell RNA Sequencing of Microglia throughout the Mouse Lifespan and in the Injured Brain Reveals Complex Cell-State Changes</strong></div><div><em>Immunity.</em> 2019; <strong>50</strong>:253-271.e6</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.immuni.2018.11.004&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.immuni.2018.11.004&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85056738677" target="_blank" aria-label="Open Scopus in new tab">Scopus (1123)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30471926/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.immuni.2018.11.004&amp;pmid=30471926" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) (<a id="crosref0835" href="#figs6">Figure&nbsp;S6</a>A). All genetic alleles examined had comparable transcriptional states (<a id="crosref0840" href="#mmc1">Table S1</a>; <a id="crosref0845" href="#sec-4">STAR Methods</a>). However, consistent with the anti-inflammatory role of GABA<sub>B</sub>Rs in microglia (<span class="dropBlock reference-citations"><a id="body-ref-sref37-2" href="#" role="doc-biblioref" data-xml-rid="bib37" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref37-2" aria-controls="sref37-2">Kuhn et&nbsp;al., 2004</a><div class="dropBlock__holder" data-db-target-of="sref37-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">38.</div><div>Kuhn, S.A. ∙ van Landeghem, F.K.H. ∙ Zacharias, R. ...</div><div><strong>Microglia express GABA(B) receptors to modulate interleukin release</strong></div><div><em>Mol. Cell. Neurosci.</em> 2004; <strong>25</strong>:312-322</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.mcn.2003.10.023" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_38_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-1542616888" target="_blank" aria-label="Open Scopus in new tab">Scopus (168)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15019947/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.mcn.2003.10.023&amp;pmid=15019947" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), we found increases in cKO microglia related to DAM (<span class="dropBlock reference-citations"><a id="body-ref-sref35-2" href="#" role="doc-biblioref" data-xml-rid="bib35" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref35-2" aria-controls="sref35-2">Keren-Shaul et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref35-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">36.</div><div>Keren-Shaul, H. ∙ Spinrad, A. ∙ Weiner, A. ...</div><div><strong>A Unique Microglia Type Associated with Restricting Development of Alzheimer’s Disease</strong></div><div><em>Cell.</em> 2017; <strong>169</strong>:1276-1290.e17</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.cell.2017.05.018&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.cell.2017.05.018&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85020287843" target="_blank" aria-label="Open Scopus in new tab">Scopus (2741)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28602351/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2017.05.018&amp;pmid=28602351" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and with inflammatory profiles (clusters 2 and 5) (<a id="crosref0860" href="#figs6">Figure&nbsp;S6</a>D; <a id="crosref0865" href="#mmc1">Table S1</a>).</div><div id="p0135" role="paragraph">We next combined WT and cKO microglia. The aligned dataset segregated into eight mixed clusters, most of which contained microglia from both experimental groups indicating that the loss of GABA<sub>B1</sub>Rs did not fundamentally alter the range of microglial states (<a id="crosref0870" href="#fig5">Figures 5</a>B–5D and <a id="crosref0875" href="#figs6">S6</a>E; <a id="crosref0880" href="#mmc1">Table S1</a>). Clusters 1–3 shared transcriptional signatures of postnatal immature microglia. Cluster 4 had higher levels of synapse pruning and homeostatic microglia core genes. Cells in cluster 5 were transcriptionally related to DAM, and cluster 6 was composed of actively proliferating cells. Microglia in cluster 7 exhibited inflammatory and interferon-responsive profiles, whereas long non-coding RNAs were enriched in cluster 8. In agreement with the role of GABA<sub>B</sub>Rs in attenuating inflammation, cKO cells were more prevalent in clusters 5, 6, and 7 (<a id="crosref0885" href="#fig5">Figures 5</a>D and <a id="crosref0890" href="#figs6">S6</a>E). To query which genes are altered by the loss of GABA<sub>B1</sub>Rs, we compared WT and cKO microglia in the mixed clusters. Only a few genes—involved in inflammation and proliferation—were differentially expressed in clusters 1–3, 6, and 8. In contrast, cluster 4 harbored a large number of differentially expressed genes (<a id="crosref0895" href="#fig5">Figure&nbsp;5</a>E; <a id="crosref0900" href="#mmc2">Table S2</a>). The majority of genes downregulated in cluster 4 are involved in synaptic pruning and related processes such as chemotaxis and phagocytosis (<a id="crosref0905" href="#fig5">Figures 5</a>F and 5G; <a id="crosref0910" href="#mmc2">Table S2</a>). Similar alterations were also observed in cKOs and Cre-Het microglia and hence are not the result of the loss of one <i>Cx3cr1</i> allele (<a id="crosref0915" href="#figs6">Figures S6</a>F–S6J; <a id="crosref0920" href="#mmc2">Table S2</a>). Thus, removal of microglial GABA<sub>B</sub>Rs impacts pruning genes selectively within more mature microglia (cluster 4).</div><div id="p0140" role="paragraph">As an alternative approach, we trained a classifier to identify genes predictive of WT versus cKO microglia across all mixed clusters and observed that the downregulation of pruning genes was a distinctive feature of mutant cells (<a id="crosref0925" href="#figs6">Figure&nbsp;S6</a>K). Further confirming the relevance of the transcriptional changes driven by cluster 4, pruning genes were also detected in a “pseudo-bulk” differential expression analysis (<a id="crosref0930" href="#mmc2">Table S2</a>). Finally, analysis following denoising of gene expression (<span class="dropBlock reference-citations"><a id="body-ref-sref78-1" href="#" role="doc-biblioref" data-xml-rid="bib78" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref78-1" aria-controls="sref78-1">Tjärnberg et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref78-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">80.</div><div>Tjärnberg, A. ∙ Mahmood, O. ∙ Jackson, C.A. ...</div><div><strong>Optimal tuning of weighted kNN- and diffusion-based methods for denoising single cell genomics data</strong></div><div><em>bioRxiv.</em> 2020;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/2020.02.28.970202" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_80_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85098901697" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2F2020.02.28.970202" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) yielded similar results (<a id="crosref0940" href="#figs6">Figures S6</a>L–S6N; <a id="crosref0945" href="#mmc2">Table S2</a>). Importantly, we found that the downregulated genes were altered only in a subset of cluster 4 microglia, which segregated as a transcriptionally defined subgroup (<a id="crosref0950" href="#fig5">Figures 5</a>H and 5I; <a id="crosref0955" href="#mmc2">Table S2</a>). Taken together, these data implicate several genes relevant to synapse remodeling as effectors of GABA<sub>B</sub>R function in microglia.</div></section><section id="sec-2-6"><h3>The transcriptional changes observed in cKOs are restricted to GABA-receptive microglia</h3><div id="p0145" role="paragraph">scRNA-seq technologies often fail to detect low-expressed mRNAs. As a consequence, <i>Gabbr1</i> and <i>Gabbr2</i>—both low expressed genes in microglia—were incompletely detected in our scRNA-seq analysis. We therefore performed imaging-based single-cell gene expression profiling of WT and cKO microglia using multiplexed error-robust fluorescence <i>in situ</i> hybridization (MERFISH) (<span class="dropBlock reference-citations"><a id="body-ref-sref14-1" href="#" role="doc-biblioref" data-xml-rid="bib14" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref14-1" aria-controls="sref14-1">Chen et&nbsp;al., 2015b</a><div class="dropBlock__holder" data-db-target-of="sref14-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">14.</div><div>Chen, K.H. ∙ Boettiger, A.N. ∙ Moffitt, J.R. ...</div><div><strong>RNA imaging. Spatially resolved, highly multiplexed RNA profiling in single cells</strong></div><div><em>Science.</em> 2015; <strong>348</strong>:aaa6090</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aaa6090" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-84928395184" target="_blank" aria-label="Open Scopus in new tab">Scopus (1274)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25858977/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aaa6090&amp;pmid=25858977" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref51-1" href="#" role="doc-biblioref" data-xml-rid="bib51" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref51-1" aria-controls="sref51-1">Moffitt et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref51-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">52.</div><div>Moffitt, J.R. ∙ Bambah-Mukku, D. ∙ Eichhorn, S.W. ...</div><div><strong>Molecular, spatial, and functional single-cell profiling of the hypothalamic preoptic region</strong></div><div><em>Science.</em> 2018; <strong>362</strong>:eaau5324</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aau5324" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85056571109" target="_blank" aria-label="Open Scopus in new tab">Scopus (576)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30385464/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aau5324&amp;pmid=30385464" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) (<a id="crosref0970" href="#fig6">Figure&nbsp;6</a>A). To this end, we selected a 23-plex gene panel focused on assessing the co-expression of <i>Gabbr1</i> and <i>Gabbr2</i> with both microglia markers and pruning genes. Of note, <i>Gabbr1</i> could be detected also in cKOs due to the presence of a non-deleted portion of the gene.</div><div class="figure-wrap"><figure id="fig6" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.06.018/asset/494dd21e-fa0e-4a77-84b5-5a2d0eb08af5/main.assets/gr6_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr6" src="/cms/10.1016/j.cell.2021.06.018/asset/f89949b8-0ecf-4209-a04d-871afd3f3a80/main.assets/gr6.jpg" height="744" width="739" aria-labelledby="fig6-title fig6-content" data-viewer-src="/cms/10.1016/j.cell.2021.06.018/asset/494dd21e-fa0e-4a77-84b5-5a2d0eb08af5/main.assets/gr6_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig6-title" class="figure__title"><span class="label figure__label">Figure&nbsp;6</span> <span class="figure__title__text">The transcriptional changes observed in cKOs are restricted to GABA-receptive microglia</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig6-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0295" role="paragraph">(A) Schematic of MERFISH experiment.</div><div id="fspara0300" role="paragraph">(B) UMAP plots of WT control and cKO microglia in the MERFISH dataset.</div><div id="fspara0305" role="paragraph">(C) Same as (B), showing 6 clusters.</div><div id="fspara0310" role="paragraph">(D) Same as (B), showing scaled expression of <i>Gabbr1</i> and <i>Gabbr2</i>.</div><div id="fspara0315" role="paragraph">(E) Region imaged for MERFISH and cell maps on DAPI signal from a control slice. Scale bar, 100&nbsp;μm.</div><div id="fspara0320" role="paragraph">(F) Layer distribution of microglia. Data are mean ± SD between slices.</div><div id="fspara0325" role="paragraph">(G) Split violin plots of normalized log-expression values for a representative pruning gene (<i>C1qc</i>) enriched in clusters 4 and 4<sup>GG</sup> control (ctl) cells.</div><div id="fspara0330" role="paragraph">(H) Split violin plots of normalized log-expression values for representative genes enriched in GABA-receptive control (ctl) cells.</div><div id="fspara0335" role="paragraph">(I) Volcano plot showing differentially expressed genes (DEGs) between control and cKO microglia for the MERFISH clusters. The negative log<sub>10</sub>-transformed p&nbsp;values are plotted against the log<sub>2</sub> fold change. DEGs with an absolute log<sub>2</sub> fold change higher than 0.25 and an adjusted p value &lt;0.05 are depicted as opaque shapes with gene name, the rest is depicted with transparency. When close to the threshold, <i>Gabbr</i>-genes are also shown with opacity. Four data points with an adjusted p value &gt;0.05 are outside the x axis limit.</div><div id="fspara0340" role="paragraph">(J) Images, masks, and fraction of C1q<sup>+</sup> PV synaptic terminals (Syt2) in P15 control and cKO mice (n&nbsp;= 5 each). <sup>∗</sup>p&nbsp;&lt; 0.05, Student’s t test. Scale bar, 1&nbsp;μm.</div><div id="fspara0345" role="paragraph">(K) Images, masks, and fraction of C1q<sup>+</sup> VGlut2<sup>+</sup> synaptic terminals in P15 control (n&nbsp;= 6) and cKO (n&nbsp;= 7) mice. ns, p &gt; 0.05; Student’s t test. Scale bar, 1&nbsp;μm.</div><div id="fspara0350" role="paragraph">Data in (J) and (K) are mean ± SEM. Full and empty arrowheads indicate colocalization and boutons not meeting criteria. See also <a id="crosref1820" href="#mmc2">Table S2</a>.</div></div></div></div></figcaption></figure></div><div id="p0150" role="paragraph">Unsupervised clustering of microglia based on MERFISH profiling generated 6 cell groups (1, 2<sup>GG</sup>, 3, 3<sup>GG</sup>, 4, and 4<sup>GG</sup>), containing both WT and cKO microglia (<a id="crosref0975" href="#fig6">Figures 6</a>B and 6C; <a id="crosref0980" href="#mmc2">Table S2</a>). Among the MERFISH groups, GABA-receptive microglia were highly enriched in three clusters (i.e., 2<sup>GG</sup>, 3<sup>GG</sup>, and 4<sup>GG</sup>) (<a id="crosref0985" href="#fig6">Figure&nbsp;6</a>D; <a id="crosref0990" href="#mmc2">Table S2</a>), most of which had a corresponding <i>Gabbr</i>-negative population. MERFISH cluster 4 and 4<sup>GG</sup> cells were distinguished from other microglia by their enhanced expression of homeostatic core genes, such as <i>P2ry13</i>, and were hence more closely related to the cluster 4 cells in our scRNA-seq data (<a id="crosref0995" href="#mmc2">Table S2</a>). Notably, GABA-receptive microglia were uniformly distributed across cortical layers and anatomically interspersed with <i>Gabbr</i>-negative cells (<a id="crosref1000" href="#fig6">Figures 6</a>E and 6F). Pruning genes were enriched in both clusters 4 and 4<sup>GG</sup> at comparable levels (<a id="crosref1005" href="#fig6">Figure&nbsp;6</a>G). In addition, GABA-receptive microglia had higher levels of transcripts encoding various transmembrane proteins. These include members of the tetraspanin family involved in cell motility, signaling, and membrane dynamics (<span class="dropBlock reference-citations"><a id="body-ref-sref10" href="#" role="doc-biblioref" data-xml-rid="bib10" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref10" aria-controls="sref10">Charrin et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref10"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">10.</div><div>Charrin, S. ∙ Jouannet, S. ∙ Boucheix, C. ...</div><div><strong>Tetraspanins at a glance</strong></div><div><em>J.&nbsp;Cell Sci.</em> 2014; <strong>127</strong>:3641-3648</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1242/jcs.154906" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-84906875545" target="_blank" aria-label="Open Scopus in new tab">Scopus (327)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25128561/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1242%2Fjcs.154906&amp;pmid=25128561" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) (<a id="crosref1015" href="#fig6">Figure&nbsp;6</a>H; <a id="crosref1020" href="#mmc2">Table S2</a>). Importantly, these same genes were downregulated in cKO microglia within cluster 4 of our scRNA-seq analysis (<a id="crosref1025" href="#mmc2">Table S2</a>), suggesting a reduction of GABA-receptive microglia in cluster 4 rather than these molecules being obligatory effectors of GABA<sub>B</sub>Rs.</div><div id="p0155" role="paragraph">We next determined whether other gene expression changes observed in our scRNA-seq data were cell-autonomously restricted to GABA-receptive cells. Using the same MERFISH gene panel, we compared the expression of pruning genes in WT and cKO microglia. Genes encoding phagocytic receptors and those involved in microglial chemotaxis (<i>C1qc</i>, <i>Trem2, Gpr34</i>, and <i>P2ry12</i>) were only downregulated in cluster 4 GABA-receptive cells (i.e., selectively within 4<sup>GG</sup>). These genes were unaffected in GABA-receptive microglia within other clusters (<a id="crosref1030" href="#fig6">Figure&nbsp;6</a>I; <a id="crosref1035" href="#mmc2">Table S2</a>).</div><div id="p0160" role="paragraph">Among the genes selectively downregulated are those encoding the complement molecule C1q. Consistent with our transcriptomic results, C1q accumulation at PV synapses was significantly decreased in P15 cKO mice compared to controls (<a id="crosref1040" href="#fig6">Figure&nbsp;6</a>J). In contrast, no change in C1q accumulation at VGlut2<sup>+</sup> terminals was observed (<a id="crosref1045" href="#fig6">Figure&nbsp;6</a>K). Taken together, our findings demonstrate that GABA<sub>B</sub>Rs activate a synapse remodeling program selectively in GABA-receptive microglia during development.</div></section><section id="sec-2-7"><h3>Loss of GABA<sub>B1</sub>Rs within microglia causes behavioral defects</h3><div id="p0165" role="paragraph">To test if the loss of microglial GABA<sub>B1</sub>Rs results in behavioral abnormalities, we performed unsupervised analysis using motion sequencing (MoSeq). MoSeq identifies sub-second motifs (syllables) that compose mouse behavior (<span class="dropBlock reference-citations"><a id="body-ref-sref46-1" href="#" role="doc-biblioref" data-xml-rid="bib46" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref46-1" aria-controls="sref46-1">Markowitz et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref46-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">47.</div><div>Markowitz, J.E. ∙ Gillis, W.F. ∙ Beron, C.C. ...</div><div><strong>The Striatum Organizes 3D Behavior via Moment-to-Moment Action Selection</strong></div><div><em>Cell.</em> 2018; <strong>174</strong>:44-58.e17</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.cell.2018.04.019&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.cell.2018.04.019&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85046871902" target="_blank" aria-label="Open Scopus in new tab">Scopus (193)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29779950/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2018.04.019&amp;pmid=29779950" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref86-1" href="#" role="doc-biblioref" data-xml-rid="bib86" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref86-1" aria-controls="sref86-1">Wiltschko et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref86-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">88.</div><div>Wiltschko, A.B. ∙ Johnson, M.J. ∙ Iurilli, G. ...</div><div><strong>Mapping Sub-Second Structure in Mouse Behavior</strong></div><div><em>Neuron.</em> 2015; <strong>88</strong>:1121-1135</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.neuron.2015.11.031&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.neuron.2015.11.031&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-84957565912" target="_blank" aria-label="Open Scopus in new tab">Scopus (373)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26687221/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.neuron.2015.11.031&amp;pmid=26687221" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). We compared the usage of behavioral syllables in control versus GABA<sub>B1</sub>R cKO mice at P30 and P60. Compared to both WTs (<i>Cx3cr1</i><sup>+/+</sup><i>;GABA</i><sub><i>B1</i></sub><i>R</i><sup><i>fl</i>/<i>fl</i></sup>) and Cre-Hets (<i>Cx3cr1</i><sup><i>Cre</i>/+</sup><i>;GABA</i><sub><i>B1</i></sub><i>R</i><sup>+/+</sup>), high velocity syllables (e.g., jump and run) were less frequent in P30 cKOs (<i>Cx3cr1</i><sup><i>Cre</i>/+</sup><i>;GABA</i><sub><i>B1</i></sub><i>R</i><sup><i>fl</i>/<i>fl</i></sup>), although this difference was more pronounced in males than females (<a id="crosref1060" href="#fig7">Figures 7</a>A, <a id="crosref1065" href="#figs7">S7</a>A, and S7B). In addition, P30 cKOs showed an overall decrease in spatial exploration (<a id="crosref1070" href="#fig7">Figure&nbsp;7</a>B).</div><div class="figure-wrap"><figure id="fig7" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.06.018/asset/ee122a21-6524-4849-a7d9-1a7f2ef9e9b5/main.assets/gr7_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr7" src="/cms/10.1016/j.cell.2021.06.018/asset/658c2f1f-8fe6-4439-bd2c-85abf79cad63/main.assets/gr7.jpg" height="798" width="648" aria-labelledby="fig7-title fig7-content" data-viewer-src="/cms/10.1016/j.cell.2021.06.018/asset/ee122a21-6524-4849-a7d9-1a7f2ef9e9b5/main.assets/gr7_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig7-title" class="figure__title"><span class="label figure__label">Figure&nbsp;7</span> <span class="figure__title__text">Loss of GABA<sub>B1</sub>Rs within microglia causes behavioral defects</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig7-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0360" role="paragraph">(A) Illustration of behavioral syllables enriched in P30 WTs or cKOs.</div><div id="fspara0365" role="paragraph">(B) Heatmap depicting the position of P30 WTs (n&nbsp;= 17) and cKOs (n&nbsp;= 29) during MoSeq.</div><div id="fspara0370" role="paragraph">(C) Expression probability of syllable usage (left) and syllable speed (right) in P60 WT control (n&nbsp;= 9), cKO (n&nbsp;= 9), and Cre-Het control (n&nbsp;= 3) female mice. <sup>∗</sup>p&nbsp;&lt; 0.05, z test on bootstrapped syllable usage/speed distribution corrected for false discovery rate (FDR). Data are mean ± SEM.</div><div id="fspara0375" role="paragraph">(D) Expression probability of syllable usage (left) and syllable speed (right) in P60 WT (n&nbsp;= 10), cKO (n&nbsp;= 5), and Cre-Het (n&nbsp;= 5) male mice. <sup>∗</sup>p&nbsp;&lt; 0.05, z test on bootstrapped syllable usage distribution corrected for FDR. Data are mean ± SEM.</div><div id="fspara0380" role="paragraph">(E) Illustration of syllables enriched in P60 WTs or cKOs.</div><div id="fspara0385" role="paragraph">(F) Heatmap depicting the position of P60 WTs (n&nbsp;= 19) and cKOs (n&nbsp;= 14) during MoSeq.</div><div id="fspara0390" role="paragraph">(G) Transition graphs depicting syllables (nodes) and transition probabilities (edges) in P60 WT and cKO mice. Node size proportional to syllable usage, edges weighted by bigram probability. Numbers correspond to syllables in <a id="crosref1825" href="#figs7">Figure&nbsp;S7</a>C.</div><div id="fspara0395" role="paragraph">(H) Transition graph depicting the difference in syllable usage and transition probability in WT and cKO mice. Low probability transitions were removed.</div><div id="fspara0400" role="paragraph">cKO: <i>Cx3cr1</i><sup><i>Cre</i>/+</sup><i>;GABA</i><sub><i>B1</i></sub><i>R</i><sup><i>fl</i>/<i>fl</i></sup>. Syllable labels were assigned by a human observer. In (C) and (D), only relevant or significant syllables are shown. See also <a id="crosref1830" href="#figs7">Figure&nbsp;S7</a> and <a id="crosref1835" href="#mmc7">Video S5</a>.</div></div></div></div></figcaption></figure></div><div class="figure-wrap"><figure id="figs7" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.06.018/asset/79ab9754-a115-4c0c-be3a-2da48fb7f92f/main.assets/figs7_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs7" src="/cms/10.1016/j.cell.2021.06.018/asset/3148320e-8967-45c2-86b6-86c928b31d81/main.assets/figs7.jpg" height="941" width="657" aria-labelledby="figs7-title figs7-content" data-viewer-src="/cms/10.1016/j.cell.2021.06.018/asset/79ab9754-a115-4c0c-be3a-2da48fb7f92f/main.assets/figs7_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs7-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S7</span> <span class="figure__title__text">MoSeq reveals a biphasic behavioral phenotype in GABA<sub>B1</sub>R cKO mice. Related to <a id="crosref1970" href="#fig7">Figure&nbsp;7</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs7-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0850" role="paragraph">(A) Expression probability of syllable usage in P30 wild-type control (n&nbsp;= 11), cKO (n&nbsp;= 15) and Cre-Het control (n&nbsp;= 5) female mice. Syllables are ordered by differential usage (left: cKO-enriched syllables; right: cKO-downregulated syllables). Here and in the rest of the figure, only relevant or significantly differentially used syllables are shown. <sup>∗</sup>p&nbsp;&lt; 0.05, z-test on bootstrapped syllable usage distribution corrected for false discovery rate. Data are mean ± SEM. Here and in the rest of the figure, syllable labels were assigned by a human observer.</div><div id="fspara0855" role="paragraph">(B) Expression probability of syllable usage in P30 wild-type control (n&nbsp;= 6), cKO (n&nbsp;= 14) and Cre-Het control (n&nbsp;= 7) male mice. Syllables are ordered by differential usage. <sup>∗</sup>p&nbsp;&lt; 0.05, z-test on bootstrapped syllable usage distribution corrected for false discovery rate. Data are mean ± SEM.</div><div id="fspara0860" role="paragraph">(C) Expression probability of syllables enriched in P60 control (n&nbsp;= 19, both sexes) and cKO (n&nbsp;= 14, both sexes) mice. Syllables are ordered by differential usage (left: cKO-enriched syllables; right: cKO-downregulated syllables). <sup>∗</sup>p&nbsp;&lt; 0.05, z-test on bootstrapped syllable usage distribution corrected for false discovery rate. Data are mean ± SEM.</div><div id="fspara0865" role="paragraph">(D) Heatmap depicting the position of P60 control (n&nbsp;= 10 males and 9 females) and cKO (n&nbsp;= 5 males and 9 females) mice during MoSeq.</div><div id="fspara0870" role="paragraph">(E) Graph showing the mean syllable speed in control (n&nbsp;= 19) and GABA<sub>B1</sub>R cKO (n&nbsp;= 14) mice. Syllables are ordered by differential speed (left: syllables having a higher speed in cKOs and right: syllables with a lower speed in the cKOs). <sup>∗</sup>p&nbsp;&lt; 0.05, t test on bootstrapped syllable speed distribution corrected for false discovery rate. Data are mean ± SEM.</div><div id="fspara0875" role="paragraph">(F) Expression probability of syllables enriched in P60 <i>Cx3cr1</i><sup><i>Cre</i></sup> cKOs (n&nbsp;= 9 females and 10 males) and <i>Tmem119</i><sup><i>CreER</i></sup> cKOs (n&nbsp;= 3 females and 3 males) mice. Only significantly differentially used syllables are shown. In females, almost all syllables were similarly used. Males from both groups used high velocity syllables although <i>Tmem119</i><sup><i>CreER</i></sup> cKOs appeared even more active than <i>Cx3cr1</i><sup><i>Cre</i></sup> cKOs. <sup>∗</sup>p&nbsp;&lt; 0.05, z-test on bootstrapped syllable usage distribution corrected for false discovery rate. Data are mean ± SEM.</div><div id="fspara0880" role="paragraph">(G) Summary of the scalar information for P60 control (n&nbsp;= 19) and GABA<sub>B1</sub>R cKO (n&nbsp;= 14) mice. Velocity is measured in mm/s; height, width and length are measured in mm. <sup>∗∗∗</sup>p&nbsp;&lt; 0.001, ns p &gt; 0.05, Student’s t test. The inset shows the probability density function (PDF) of the two-dimensional velocity, p&nbsp;&lt; 0.001, Two-sample Kolmogorov–Smirnov test.</div></div></div></div></figcaption></figure></div><div id="p0170" role="paragraph">In contrast to P30, P60 cKOs displayed high velocity syllables with greater frequency and executed behavioral motifs with increased rapidity compared to P60 WT and Cre-Het controls. Conversely, usage of low-velocity syllables (e.g., pause) was decreased in P60 cKOs (<a id="crosref1075" href="#fig7">Figures 7</a>C–7E, <a id="crosref1080" href="#figs7">S7</a>C, S7E, and S7G; <a id="crosref1085" href="#mmc7">Video S5</a>). In addition, P60 cKO mice showed an overall increase in spatial exploration, although this effect was more pronounced in males than females (<a id="crosref1090" href="#fig7">Figures 7</a>F and <a id="crosref1095" href="#figs7">S7</a>D). Of note, P60 cKOs generated using either <i>Cx3cr1</i><sup><i>Cre</i></sup> or <i>Tmem119</i><sup><i>CreER</i></sup> drivers appeared comparable in their usage of high velocity syllables (<a id="crosref1100" href="#figs7">Figure&nbsp;S7</a>F). We also observed an overall decrease in syllabic transitions in P60 cKOs compared to WTs (<a id="crosref1105" href="#fig7">Figure&nbsp;7</a>G and 7H). Of note, the few enhanced transitions observed in cKOs involved high-velocity syllables, further reaffirming their hyperactivity.</div><div id="p0175" role="paragraph"><div id="mmc7" class="core-supplementary-material" data-category="video"><div class="core-link"><div class="cloudflare-stream-container" id="mmc7" data-type="mp4"><span id="video_stream_mmc7_src" class="video-stream-source" style="display:none">eyJraWQiOiI4ZjUxYWNhY2IzYjhiNjNlNzFlYmIzYWFmYTU5NmZmYyIsImFsZyI6IlJTMjU2In0.eyJzdWIiOiJlNDc2NjgwYmM2YmJmNjRkMDIyMDg4YzU4MzM1NDdlMyIsImtpZCI6IjhmNTFhY2FjYjNiOGI2M2U3MWViYjNhYWZhNTk2ZmZjIiwiZXhwIjoxNzQ5NjEyNTU5fQ.GvhwEzH-QADbClaJUiWgZ0ekYIj67Pm9p2Vd1_3890WXzwo6M2d0CmcSXP4OVNOlb87jBpSgE0760QpvDOO_ewhAEFtkKDyOv_Ma6SS6bgwHFeDHpCwCO5d_F0fJ7lea-PC_yn2yMnUetjiOsqEVdq-05oduemsGlQU3TD6UtBCvJ7uHN8h3koglfsBMRGifuPXzLUdZSkoMe4nrLevPy9uV4zUo8j920H59XS8d8Od15zJmrhqLw2yf4VXX2dTjyr-KMxFVTMFyW1J__eqFG1CayUYS9TbhcObu3IExNkg4dyxbCIvS65PnOIXyS2lmEX3RwlsUQAP1yxf3pe9-8g</span><stream src="eyJraWQiOiI4ZjUxYWNhY2IzYjhiNjNlNzFlYmIzYWFmYTU5NmZmYyIsImFsZyI6IlJTMjU2In0.eyJzdWIiOiJlNDc2NjgwYmM2YmJmNjRkMDIyMDg4YzU4MzM1NDdlMyIsImtpZCI6IjhmNTFhY2FjYjNiOGI2M2U3MWViYjNhYWZhNTk2ZmZjIiwiZXhwIjoxNzQ5NjEyNTU5fQ.GvhwEzH-QADbClaJUiWgZ0ekYIj67Pm9p2Vd1_3890WXzwo6M2d0CmcSXP4OVNOlb87jBpSgE0760QpvDOO_ewhAEFtkKDyOv_Ma6SS6bgwHFeDHpCwCO5d_F0fJ7lea-PC_yn2yMnUetjiOsqEVdq-05oduemsGlQU3TD6UtBCvJ7uHN8h3koglfsBMRGifuPXzLUdZSkoMe4nrLevPy9uV4zUo8j920H59XS8d8Od15zJmrhqLw2yf4VXX2dTjyr-KMxFVTMFyW1J__eqFG1CayUYS9TbhcObu3IExNkg4dyxbCIvS65PnOIXyS2lmEX3RwlsUQAP1yxf3pe9-8g" class="cloudflare-stream-player ready tracked" id="video_stream_mmc7" controls="true"><div class="player-share-button" title="Share Video"><div class="share-player-icon"></div></div><div class="player-share-container hidden"><span class="player-close-share"></span><div class="social-links-container"><ul class="social-links-list"><li class="html5-share-facebook-icon" data-share-url="https://www.facebook.com/sharer/sharer.php?u=https://www.cell.com/cell/fulltext/S0092-8674(21)00753-4" target="_blank"></li><li class="html5-share-twitter-icon" data-share-url="https://twitter.com/share?url=https://www.cell.com/cell/fulltext/S0092-8674(21)00753-4" target="_blank"></li><li class="html5-share-mail-icon" data-share-url="mailto:?body=https://www.cell.com/cell/fulltext/S0092-8674(21)00753-4" target="_blank"></li></ul><input type="text" value="https://www.cell.com/cell/fulltext/S0092-8674(21)00753-4" readonly="readonly" class="share-input"></div></div><div style="position: relative; height: 0px; width: 100%; padding-top: 56.25%;"><iframe src="https://embed.cloudflarestream.com/embed/iframe.fla9.823df22.html?videoId=eyJraWQiOiI4ZjUxYWNhY2IzYjhiNjNlNzFlYmIzYWFmYTU5NmZmYyIsImFsZyI6IlJTMjU2In0.eyJzdWIiOiJlNDc2NjgwYmM2YmJmNjRkMDIyMDg4YzU4MzM1NDdlMyIsImtpZCI6IjhmNTFhY2FjYjNiOGI2M2U3MWViYjNhYWZhNTk2ZmZjIiwiZXhwIjoxNzQ5NjEyNTU5fQ.GvhwEzH-QADbClaJUiWgZ0ekYIj67Pm9p2Vd1_3890WXzwo6M2d0CmcSXP4OVNOlb87jBpSgE0760QpvDOO_ewhAEFtkKDyOv_Ma6SS6bgwHFeDHpCwCO5d_F0fJ7lea-PC_yn2yMnUetjiOsqEVdq-05oduemsGlQU3TD6UtBCvJ7uHN8h3koglfsBMRGifuPXzLUdZSkoMe4nrLevPy9uV4zUo8j920H59XS8d8Od15zJmrhqLw2yf4VXX2dTjyr-KMxFVTMFyW1J__eqFG1CayUYS9TbhcObu3IExNkg4dyxbCIvS65PnOIXyS2lmEX3RwlsUQAP1yxf3pe9-8g" allowfullscreen="true" allow="autoplay; picture-in-picture" style="height: 100%; width: 100%; position: absolute; top: 0px; left: 0px; border: none;"></iframe></div></stream></div><a href="/cms/10.1016/j.cell.2021.06.018/attachment/848f9fd2-aac0-4de1-b218-7ae8c064008f/mmc7.mp4" download="mmc7.mp4" aria-labelledby="mmc7-heading mmc7-link"><span id="mmc7-link">Video (1.48 MB)</span></a></div><div class="core-description"><div id="mmc7-heading" class="heading">Video S5. MoSeq identifies behavioral syllables differentially used in GABA<sub>B1</sub>Rs cKO mice, related to Figure&nbsp;7</div><div id="espara0030" role="paragraph">“Crowd” movies depicting four distinct behavioral modules, two upregulated and two downregulated in P60 GABA<sub>B1</sub>Rs cKO mice. Color scale bar indicates the height of any given pixel above the floor of the arena (mm). “Crowd” movies are generated by overlaying individual movies from each mouse. In these movies, the ongoing expression of the syllable is indicated by a dot being placed over each mouse; during expression of the module, the mice in all examples emit a stereotyped motif of three-dimensional motion.</div></div></div></div></section></section><section id="sec-3"><h2>Discussion</h2><div id="p0180" role="paragraph">Multiple lines of evidence support that GABA-receptive microglia selectively remodel inhibitory synapses during development. (1) <i>In&nbsp;vivo</i> two-photon imaging indicates that a subset of microglia directly interacts with inhibitory synapses and these interactions are strongly attenuated in GABA<sub>B1</sub>R cKOs. (2) GABA-receptive microglia preferentially contact inhibitory versus excitatory synapses. (3) The contacts between microglia and inhibitory synapses are reduced upon ablation of GABA<sub>B</sub>Rs within microglia. (4) Orthogonal transcriptomic analyses demonstrate that the conditional removal of microglial GABA<sub>B</sub>Rs alters synapse pruning genes selectively within GABA-receptive microglia. (5) Removing GABA<sub>B</sub>Rs from microglia alters inhibitory connectivity without impacting excitatory synapses. (6) cKOs exhibit behavioral abnormalities that correlate with the inhibitory synaptic alterations.</div><section id="sec-3-1"><h3>A role for microglia in the wiring of inhibitory circuits</h3><div id="p0185" role="paragraph">Distinct microglia-dependent mechanisms play a crucial role in shaping developing excitatory circuits. For example, both phagocytic and non-phagocytic microglia functions contribute to excitatory synapse refinement (<span class="dropBlock reference-citations"><a id="body-ref-sref11" href="#" role="doc-biblioref" data-xml-rid="bib11" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref11" aria-controls="sref11">Cheadle et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref11"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">11.</div><div>Cheadle, L. ∙ Rivera, S.A. ∙ Phelps, J.S. ...</div><div><strong>Sensory Experience Engages Microglia to Shape Neural Connectivity through a Non-Phagocytic Mechanism</strong></div><div><em>Neuron.</em> 2020; <strong>108</strong>:451-468.e9</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.neuron.2020.08.002&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.neuron.2020.08.002&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85091688509" target="_blank" aria-label="Open Scopus in new tab">Scopus (95)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32931754/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.neuron.2020.08.002&amp;pmid=32931754" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref64-3" href="#" role="doc-biblioref" data-xml-rid="bib64" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref64-3" aria-controls="sref64-3">Schafer et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref64-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">66.</div><div>Schafer, D.P. ∙ Lehrman, E.K. ∙ Kautzman, A.G. ...</div><div><strong>Microglia sculpt postnatal neural circuits in an activity and complement-dependent manner</strong></div><div><em>Neuron.</em> 2012; <strong>74</strong>:691-705</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.neuron.2012.03.026&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.neuron.2012.03.026&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-84861427387" target="_blank" aria-label="Open Scopus in new tab">Scopus (2664)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22632727/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.neuron.2012.03.026&amp;pmid=22632727" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Our findings indicate that the same principles hold true for inhibitory synapses and that microglia-mediated synapse engulfment contributes to inhibitory synapse remodeling. Nonetheless, additional pruning mechanisms (e.g., release of factors that induce receptor endocytosis) may also play a role. Alternatively, or in addition to synapse elimination, microglia may retard the maturation of inhibitory synapses. As PV synapse density continues to increase between P12 and P15 in S1, microglia depletion or GABA<sub>B1</sub>R ablation may lead to supernumerary synapses.</div></section><section id="sec-3-2"><h3>GABA acts as a paracrine signal to initiate inhibitory synapse remodeling</h3><div id="p0190" role="paragraph">Our results showed that the genes downregulated in cKOs are largely confined to cluster 4 GABA-receptive microglia. This suggests that microglia-mediated remodeling of inhibitory circuits requires three conditions to be met: (1) GABA release (<span class="dropBlock reference-citations"><a id="body-ref-sref88-2" href="#" role="doc-biblioref" data-xml-rid="bib88" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref88-2" aria-controls="sref88-2">Wu et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref88-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">90.</div><div>Wu, X. ∙ Fu, Y. ∙ Knott, G. ...</div><div><strong>GABA signaling promotes synapse elimination and axon pruning in developing cortical inhibitory interneurons</strong></div><div><em>J.&nbsp;Neurosci.</em> 2012; <strong>32</strong>:331-343</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1523/JNEUROSCI.3189-11.2012" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_90_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-84855913405" target="_blank" aria-label="Open Scopus in new tab">Scopus (84)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22219294/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1523%2FJNEUROSCI.3189-11.2012&amp;pmid=22219294" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), (2) expression of GABA<sub>B</sub>Rs in microglia, and (3) the appearance of pruning competent microglia.</div><div id="p0195" role="paragraph">Our work adds to the evidence demonstrating a non-synaptic role of ambient GABA signaling through GABA<sub>B</sub>Rs in neural circuit development (<span class="dropBlock reference-citations"><a id="body-ref-sref9" href="#" role="doc-biblioref" data-xml-rid="bib9" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref9" aria-controls="sref9">Cellot and Cherubini, 2013</a><div class="dropBlock__holder" data-db-target-of="sref9"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">9.</div><div>Cellot, G. ∙ Cherubini, E.</div><div><strong>Functional role of ambient GABA in refining neuronal circuits early in postnatal development</strong></div><div><em>Front. Neural Circuits.</em> 2013; <strong>7</strong>:136</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.3389/fncir.2013.00136" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-84882684678" target="_blank" aria-label="Open Scopus in new tab">Scopus (75)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23964205/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.3389%2Ffncir.2013.00136&amp;pmid=23964205" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref27-2" href="#" role="doc-biblioref" data-xml-rid="bib27" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref27-2" aria-controls="sref27-2">Gaiarsa and Porcher, 2013</a><div class="dropBlock__holder" data-db-target-of="sref27-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">28.</div><div>Gaiarsa, J.-L. ∙ Porcher, C.</div><div><strong>Emerging neurotrophic role of GABAB receptors in neuronal circuit development</strong></div><div><em>Front. Cell. Neurosci.</em> 2013; <strong>7</strong>:206</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.3389/fncel.2013.00206" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-84888155744" target="_blank" aria-label="Open Scopus in new tab">Scopus (44)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24282395/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.3389%2Ffncel.2013.00206&amp;pmid=24282395" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). We anticipate that GABA binding to microglial GABA<sub>B</sub>Rs triggers a complex cascade, which initiates a series of synapse-specific and activity-dependent processes. P2Y<sub>12</sub> receptors play a crucial role in attracting microglia to cell bodies and synapses (<span class="dropBlock reference-citations"><a id="body-ref-sref4-2" href="#" role="doc-biblioref" data-xml-rid="bib4" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref4-2" aria-controls="sref4-2">Badimon et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref4-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">4.</div><div>Badimon, A. ∙ Strasburger, H.J. ∙ Ayata, P. ...</div><div><strong>Negative feedback control of neuronal activity by microglia</strong></div><div><em>Nature.</em> 2020; <strong>586</strong>:417-423</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-020-2777-8" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85091727125" target="_blank" aria-label="Open Scopus in new tab">Scopus (443)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32999463/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-020-2777-8&amp;pmid=32999463" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref16-2" href="#" role="doc-biblioref" data-xml-rid="bib16" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref16-2" aria-controls="sref16-2">Cserép et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref16-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">17.</div><div>Cserép, C. ∙ Pósfai, B. ∙ Lénárt, N. ...</div><div><strong>Microglia monitor and protect neuronal function through specialized somatic purinergic junctions</strong></div><div><em>Science.</em> 2020; <strong>367</strong>:528-537</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aax6752" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85078742646" target="_blank" aria-label="Open Scopus in new tab">Scopus (330)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31831638/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aax6752&amp;pmid=31831638" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Here, we showed that microglia lacking GABA<sub>B</sub>Rs contact fewer inhibitory synapses and that <i>P2ry12</i> is downregulated in cKO GABA-receptive microglia. In addition, our MERFISH analysis identified transmembrane molecules enriched in GABA-receptive microglia. Some of these molecules may interact with partners selectively found at GABAergic terminals, thereby explaining the marked preference of GABA-receptive microglia for inhibitory synapses. In the adult, microglia are attracted to active synapses and dampen neuronal activity through the generation of adenosine (<span class="dropBlock reference-citations"><a id="body-ref-sref4-3" href="#" role="doc-biblioref" data-xml-rid="bib4" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref4-3" aria-controls="sref4-3">Badimon et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref4-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">4.</div><div>Badimon, A. ∙ Strasburger, H.J. ∙ Ayata, P. ...</div><div><strong>Negative feedback control of neuronal activity by microglia</strong></div><div><em>Nature.</em> 2020; <strong>586</strong>:417-423</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-020-2777-8" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85091727125" target="_blank" aria-label="Open Scopus in new tab">Scopus (443)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32999463/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-020-2777-8&amp;pmid=32999463" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). An intriguing possibility is that a similar adenosine-mediated weakening of synapses triggers pruning during development.</div></section><section id="sec-3-3"><h3>Loss of microglial GABA<sub>B1</sub>Rs trigger a neurodevelopmental disorder-related behavioral phenotype</h3><div id="p0200" role="paragraph">Our results show that disruption of GABA-signaling within microglia causes reduced activity at P30 but hyperactivity in adult animals. Hyperactivity is a hallmark symptom of neurodevelopmental disorders, including attention deficit/hyperactivity disorder (ADHD) (<span class="dropBlock reference-citations"><a id="body-ref-sref1" href="#" role="doc-biblioref" data-xml-rid="bib1" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref1" aria-controls="sref1">American Psychiatric Association, 2013</a><div class="dropBlock__holder" data-db-target-of="sref1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">1.</div><div>American Psychiatric Association</div><div><strong>Diagnostic and Statistical Manual of Mental Disorders (DSM-5)</strong></div><div>APA Publishing, 2013</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1176/appi.books.9780890425596" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1176%2Fappi.books.9780890425596" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Abnormal development of synaptic contacts and an altered excitatory versus inhibitory synapse ratio underlie the pathophysiology of these disorders (<span class="dropBlock reference-citations"><a id="body-ref-sref7" href="#" role="doc-biblioref" data-xml-rid="bib7" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref7" aria-controls="sref7">Braat and Kooy, 2015</a><div class="dropBlock__holder" data-db-target-of="sref7"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">7.</div><div>Braat, S. ∙ Kooy, R.F.</div><div><strong>The GABAA Receptor as a Therapeutic Target for Neurodevelopmental Disorders</strong></div><div><em>Neuron.</em> 2015; <strong>86</strong>:1119-1130</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.neuron.2015.03.042&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.neuron.2015.03.042&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-84930531864" target="_blank" aria-label="Open Scopus in new tab">Scopus (220)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26050032/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.neuron.2015.03.042&amp;pmid=26050032" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref13" href="#" role="doc-biblioref" data-xml-rid="bib13" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref13" aria-controls="sref13">Chen et&nbsp;al., 2015a</a><div class="dropBlock__holder" data-db-target-of="sref13"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">13.</div><div>Chen, J.A. ∙ Peñagarikano, O. ∙ Belgard, T.G. ...</div><div><strong>The emerging picture of autism spectrum disorder: genetics and pathology</strong></div><div><em>Annu. Rev. Pathol.</em> 2015; <strong>10</strong>:111-144</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1146/annurev-pathol-012414-040405" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_13_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-84921917284" target="_blank" aria-label="Open Scopus in new tab">Scopus (196)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25621659/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1146%2Fannurev-pathol-012414-040405&amp;pmid=25621659" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Consistent with previous findings (<span class="dropBlock reference-citations"><a id="body-ref-sref76-2" href="#" role="doc-biblioref" data-xml-rid="bib76" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref76-2" aria-controls="sref76-2">Thion et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref76-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">78.</div><div>Thion, M.S. ∙ Mosser, C.-A. ∙ Férézou, I. ...</div><div><strong>Biphasic Impact of Prenatal Inflammation and Macrophage Depletion on the Wiring of Neocortical Inhibitory Circuits</strong></div><div><em>Cell Rep.</em> 2019; <strong>28</strong>:1119-1126.e4</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_78_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.celrep.2019.06.086&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_78_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.celrep.2019.06.086&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_78_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85069572402" target="_blank" aria-label="Open Scopus in new tab">Scopus (35)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31365857/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.celrep.2019.06.086&amp;pmid=31365857" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), in cortex and striatum, loss of microglial GABA<sub>B1</sub>Rs causes a selective increase in inhibitory synapses at P15 and P30 that shifts to a selective decrease at P60. We showed that this change is not microglia-dependent and is therefore likely a compensatory effect. Finally, the behavioral phenotype probably reflects circuit anomalies distributed across many brain areas. Reduced cortical inhibition has been reported in ADHD (<span class="dropBlock reference-citations"><a id="body-ref-sref20" href="#" role="doc-biblioref" data-xml-rid="bib20" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref20" aria-controls="sref20">Edden et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref20"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">21.</div><div>Edden, R.A.E. ∙ Crocetti, D. ∙ Zhu, H. ...</div><div><strong>Reduced GABA concentration in attention-deficit/hyperactivity disorder</strong></div><div><em>Arch. Gen. Psychiatry.</em> 2012; <strong>69</strong>:750-753</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1001/archgenpsychiatry.2011.2280" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-84863545490" target="_blank" aria-label="Open Scopus in new tab">Scopus (178)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22752239/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1001%2Farchgenpsychiatry.2011.2280&amp;pmid=22752239" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref29" href="#" role="doc-biblioref" data-xml-rid="bib29" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref29" aria-controls="sref29">Gilbert et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref29"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">30.</div><div>Gilbert, D.L. ∙ Isaacs, K.M. ∙ Augusta, M. ...</div><div><strong>Motor cortex inhibition: a marker of ADHD behavior and motor development in children</strong></div><div><em>Neurology.</em> 2011; <strong>76</strong>:615-621</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1212/WNL.0b013e31820c2ebd" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-79951592190" target="_blank" aria-label="Open Scopus in new tab">Scopus (138)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21321335/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1212%2FWNL.0b013e31820c2ebd&amp;pmid=21321335" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Nonetheless, synaptic alterations in the striatum are likely a crucial component. Accordingly, striatal dysfunction is associated with diverse neuropsychiatric conditions involving hyperactivity (<span class="dropBlock reference-citations"><a id="body-ref-sref17" href="#" role="doc-biblioref" data-xml-rid="bib17" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref17" aria-controls="sref17">Cubillo et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref17"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">18.</div><div>Cubillo, A. ∙ Halari, R. ∙ Smith, A. ...</div><div><strong>A review of fronto-striatal and fronto-cortical brain abnormalities in children and adults with Attention Deficit Hyperactivity Disorder (ADHD) and new evidence for dysfunction in adults with ADHD during motivation and attention</strong></div><div><em>Cortex.</em> 2012; <strong>48</strong>:194-215</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.cortex.2011.04.007&amp;cf=fulltext" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.cortex.2011.04.007&amp;cf=pdf" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-84856228224" target="_blank" aria-label="Open Scopus in new tab">Scopus (367)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21575934/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cortex.2011.04.007&amp;pmid=21575934" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref52-3" href="#" role="doc-biblioref" data-xml-rid="bib52" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref52-3" aria-controls="sref52-3">Nagai et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref52-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">53.</div><div>Nagai, J. ∙ Rajbhandari, A.K. ∙ Gangwani, M.R. ...</div><div><strong>Hyperactivity with Disrupted Attention by Activation of an Astrocyte Synaptogenic Cue</strong></div><div><em>Cell.</em> 2019; <strong>177</strong>:1280-1292.e20</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.cell.2019.03.019&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.cell.2019.03.019&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85065073576" target="_blank" aria-label="Open Scopus in new tab">Scopus (190)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31031006/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2019.03.019&amp;pmid=31031006" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="p0205" role="paragraph">Our work indicates that specialized microglia selectively remodel specific synapse types. Microglia express various neurotransmitter and neuromodulators receptors (<span class="dropBlock reference-citations"><a id="body-ref-sref58" href="#" role="doc-biblioref" data-xml-rid="bib58" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref58" aria-controls="sref58">Pocock and Kettenmann, 2007</a><div class="dropBlock__holder" data-db-target-of="sref58"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">60.</div><div>Pocock, J.M. ∙ Kettenmann, H.</div><div><strong>Neurotransmitter receptors on microglia</strong></div><div><em>Trends Neurosci.</em> 2007; <strong>30</strong>:527-535</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.tins.2007.07.007&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.tins.2007.07.007&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-34848843543" target="_blank" aria-label="Open Scopus in new tab">Scopus (526)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17904651/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.tins.2007.07.007&amp;pmid=17904651" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Unveiling whether this allows them to target specific synapses will deepen our knowledge of brain function and dysfunction.</div></section><section id="sec-3-4"><h3>Limitations of study</h3><div id="p0210" role="paragraph">Two additional aspects may contribute to the observed phenotypes: non-autonomous changes in microglial function not involving transcription and nonspecific effects of the enhanced pro-inflammatory microglia profiles.</div><div id="p0215" role="paragraph">Inhibitory synapses are preferentially, but not exclusively, contacted by GABA-receptive microglia, suggesting that additional subsets of microglia might also contribute to inhibitory synapse pruning.</div></section></section><section id="sec-4"><h2>STAR★Methods</h2><section id="sec-4-1"><h3>Key resources table</h3><div id="p0225" role="paragraph"><div class="figure-wrap"><figure id="undtbl1" class="table" data-figureviewer-ignore="true"><div class="table-wrap"><table data-figureviewer-ignore="true" tabindex="0"><thead><tr class="rowsep"><th>REAGENT or RESOURCE</th><th>SOURCE</th><th>IDENTIFIER</th></tr></thead><tbody><tr class="rowsep"><td colspan="3"><b>Antibodies</b></td></tr><tr><td>mouse anti-Synaptotagmin-2</td><td>ZFIN</td><td>Cat #ZDB-ATB-081002-25; RRID:<a id="intref0010" href="https://antibodyregistry.org/AB_10013783" target="_blank">AB_10013783</a></td></tr><tr><td>mouse anti-gephyrin</td><td>Synaptic Systems</td><td>Cat #147 011; RRID:<a id="intref0015" href="https://antibodyregistry.org/AB_887717" target="_blank">AB_887717</a></td></tr><tr><td>rabbit anti-NeuN</td><td>Millipore</td><td>Cat #ABN78; RRID:<a id="intref0020" href="https://antibodyregistry.org/AB_10807945" target="_blank">AB_10807945</a></td></tr><tr><td>guinea-pig anti-NeuN</td><td>Millipore</td><td>Cat #ABN90P; RRID: <a id="intref0025" href="https://antibodyregistry.org/AB_2341095" target="_blank">AB_2341095</a></td></tr><tr><td>guinea-pig anti-VGlut2</td><td>Millipore</td><td>Cat #AB2251; RRID: <a id="intref0030" href="https://antibodyregistry.org/AB_1587626" target="_blank">AB_1587626</a></td></tr><tr><td>guinea-pig anti-VGlut2</td><td>Synaptic Systems</td><td>Cat #135404; RRID: <a id="intref0035" href="https://antibodyregistry.org/AB_887884" target="_blank">AB_887884</a></td></tr><tr><td>guinea-pig anti-VGlut1</td><td>Millipore</td><td>Cat #AB5905; RRID: <a id="intref0040" href="https://antibodyregistry.org/AB_2301751" target="_blank">AB_2301751</a></td></tr><tr><td>rabbit anti-Homer 1b/c</td><td>Synaptic Systems</td><td>Cat #160023; RRID: <a id="intref0045" href="https://antibodyregistry.org/AB_2619858" target="_blank">AB_2619858</a></td></tr><tr><td>mouse anti-parvalbumin</td><td>Sigma</td><td>Cat#P-3088; RRID:<a id="intref0050" href="https://antibodyregistry.org/AB_477329" target="_blank">AB_477329</a></td></tr><tr><td>rabbit anti-DsRed</td><td>Clontech</td><td>Cat #632496; RRID:<a id="intref0055" href="https://antibodyregistry.org/AB_10013483" target="_blank">AB_10013483</a></td></tr><tr><td>chicken anti-GFP</td><td>Aves Lab</td><td>Cat #1020; RRID:<a id="intref0060" href="https://antibodyregistry.org/AB_10000240" target="_blank">AB_10000240</a></td></tr><tr><td>rabbit anti-Iba1</td><td>Wako Chemicals</td><td>Cat #019-19741; RRID: <a id="intref0065" href="https://antibodyregistry.org/AB_839504" target="_blank">AB_839504</a></td></tr><tr><td>guinea-pig anti-Iba1</td><td>Synaptic Systems</td><td>Cat #234 004; RRID: <a id="intref0070" href="https://antibodyregistry.org/AB_2493179" target="_blank">AB_2493179</a></td></tr><tr><td>guinea-pig anti-parvalbumin</td><td>Swant</td><td>Cat #GP72; RRID: <a id="intref0075" href="https://antibodyregistry.org/AB_2665495" target="_blank">AB_2665495</a></td></tr><tr><td>rabbit anti-C1q</td><td>Abcam</td><td>Cat #ab182451; RRID: <a id="intref0080" href="https://antibodyregistry.org/AB_2732849" target="_blank">AB_2732849</a></td></tr><tr><td>rat anti-CD68</td><td>Bio-Rad</td><td>Cat #MCA1957; RRID: <a id="intref0085" href="https://antibodyregistry.org/AB_322219" target="_blank">AB_322219</a></td></tr><tr><td>mouse anti-GABABR1</td><td>NeuroMab</td><td>Cat #73-183; RRID: <a id="intref0090" href="https://antibodyregistry.org/AB_10672843" target="_blank">AB_10672843</a></td></tr><tr><td>guinea-pig anti-GABABR2</td><td>Millipore</td><td>Cat #AB2255; RRID: <a id="intref0095" href="https://antibodyregistry.org/AB_10563515" target="_blank">AB_10563515</a></td></tr><tr><td>Wisteria Floribundas (WFA)</td><td>Sigma</td><td>L1516-2mg; RRID: <a id="intref0100" href="https://antibodyregistry.org/AB_2620171" target="_blank">AB_2620171</a></td></tr><tr><td>rabbit anti-SST</td><td>Peninsula Laboratories</td><td>Cat #T4103.0050; RRID:<a id="intref0105" href="https://antibodyregistry.org/AB_518614" target="_blank">AB_518614</a></td></tr><tr><td>mouse anti-GAD65</td><td>Millipore</td><td>Cat #MAB351R; RRID: <a id="intref0110" href="https://antibodyregistry.org/AB_94905" target="_blank">AB_94905</a></td></tr><tr><td>Cd11b-PE</td><td>BioLegend</td><td>Cat #101208; RRID: <a id="intref0115" href="https://antibodyregistry.org/AB_312791" target="_blank">AB_312791</a></td></tr><tr><td>CD45-APC-Cy7</td><td>BioLegend</td><td>Cat #103116; RRID: <a id="intref0120" href="https://antibodyregistry.org/AB_312981" target="_blank">AB_312981</a></td></tr><tr class="rowsep"><td>Cx3cr1-APC</td><td>BioLegend</td><td>Cat #149008; RRID: <a id="intref0125" href="https://antibodyregistry.org/AB_2564492" target="_blank">AB_2564492</a></td></tr><tr class="rowsep"><td colspan="3"><b>Bacterial and virus strains</b></td></tr><tr><td>AAV-PHP.eB-PVe(S5E2)-Syp-dTomato</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref82-2" href="#" role="doc-biblioref" data-xml-rid="bib82" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref82-2" aria-controls="sref82-2">Vormstein-Schneider et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref82-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">84.</div><div>Vormstein-Schneider, D. ∙ Lin, J.D. ∙ Pelkey, K.A. ...</div><div><strong>Viral manipulation of functionally distinct interneurons in mice, non-human primates and humans</strong></div><div><em>Nat. Neurosci.</em> 2020; <strong>23</strong>:1629-1636</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41593-020-0692-9" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_84_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85089867328" target="_blank" aria-label="Open Scopus in new tab">Scopus (100)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32807948/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41593-020-0692-9&amp;pmid=32807948" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>N/A</td></tr><tr class="rowsep"><td>AAV- PHP.eB-PVe(S5E2)-Syp-Gamillus</td><td>This paper</td><td>N/A</td></tr><tr class="rowsep"><td colspan="3"><b>Chemicals, peptides, and recombinant proteins</b></td></tr><tr><td>PLX5622 powder</td><td>Plexxikon Inc; <span class="dropBlock reference-citations"><a id="body-ref-sref69-1" href="#" role="doc-biblioref" data-xml-rid="bib69" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref69-1" aria-controls="sref69-1">Spangenberg et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref69-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">71.</div><div>Spangenberg, E. ∙ Severson, P.L. ∙ Hohsfield, L.A. ...</div><div><strong>Sustained microglial depletion with CSF1R inhibitor impairs parenchymal plaque development in an Alzheimer’s disease model</strong></div><div><em>Nat. Commun.</em> 2019; <strong>10</strong>:3758</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41467-019-11674-z" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85071120834" target="_blank" aria-label="Open Scopus in new tab">Scopus (418)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31434879/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41467-019-11674-z&amp;pmid=31434879" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>N/A</td></tr><tr><td>PLX5622-formulated AIN-76A diet</td><td>Research Diets</td><td>N/A, Custom made</td></tr><tr><td>Control AIN-76A diet</td><td>Research Diets</td><td>Cat D10001i</td></tr><tr><td>Tamoxifen</td><td>Sigma-Aldrich</td><td>Cat #T5648-1G</td></tr><tr><td>Kainic acid</td><td>Sigma-Aldrich</td><td>Cat #K0250-10MG</td></tr><tr class="rowsep"><td>Percoll® Density Gradient Media</td><td>Fisher Scientific</td><td>Cat #17-0891-01</td></tr><tr class="rowsep"><td colspan="3"><b>Critical commercial assays</b></td></tr><tr><td>Chromium Single Cell 3′ Library and Gel Bead Kit v3.0</td><td>10x Genomics</td><td>Cat # PN-1000075</td></tr><tr><td>Chromium Single Cell 3′ Library Construction Kit v3.0</td><td>10x Genomics</td><td>Cat # PN-1000078</td></tr><tr><td>Chromium Chip B Single Cell kit</td><td>10x Genomics</td><td>Cat # PN-1000154</td></tr><tr><td>Chromium i7 Multiplex Kit</td><td>10x Genomics</td><td>Cat # PN-120262</td></tr><tr class="rowsep"><td>Imaging Reagent Kit</td><td>Vizgen</td><td>Cat # IK-18</td></tr><tr class="rowsep"><td colspan="3"><b>Deposited data</b></td></tr><tr class="rowsep"><td>Raw data files for single-cell RNA-seq</td><td>NCBI Gene Expression Omnibus</td><td>GSE159947</td></tr><tr class="rowsep"><td colspan="3"><b>Experimental models: cell lines</b></td></tr><tr class="rowsep"><td>HEK293FT</td><td>ThermoScientific</td><td>Cat #R70007</td></tr><tr class="rowsep"><td colspan="3"><b>Experimental models: organisms/strains</b></td></tr><tr><td>Mouse: <i>B6.129P2(Cg)-Cx3cr1</i><sup><i>tm1Litt</i></sup><i>/J</i></td><td>Jackson Laboratories</td><td>Cat #005582; RRID: IMSR_JAX:005582</td></tr><tr><td>Mouse: <i>GABA</i><sub><i>B1</i></sub> <i>floxed</i></td><td>A gift from B. Bettler, <span class="dropBlock reference-citations"><a id="body-ref-sref30-1" href="#" role="doc-biblioref" data-xml-rid="bib30" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref30-1" aria-controls="sref30-1">Haller et&nbsp;al., 2004</a><div class="dropBlock__holder" data-db-target-of="sref30-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">31.</div><div>Haller, C. ∙ Casanova, E. ∙ Müller, M. ...</div><div><strong>Floxed allele for conditional inactivation of the GABAB(1) gene</strong></div><div><em>Genesis.</em> 2004; <strong>40</strong>:125-130</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/gene.20073" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-10344244007" target="_blank" aria-label="Open Scopus in new tab">Scopus (45)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15493018/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fgene.20073&amp;pmid=15493018" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>N/A</td></tr><tr><td>Mouse: <i>B6J.B6N(Cg)-Cx3cr1</i><sup><i>tm1.1(cre)Jung</i></sup><i>/J</i></td><td>Jackson Laboratories</td><td>Cat #025524; RRID: IMSR_JAX:025524</td></tr><tr><td>Mouse: <i>C57BL/6-Tmem119</i><sup><i>em1(cre/ERT2)Gfng</i></sup><i>/J</i></td><td>Jackson Laboratories <span class="dropBlock reference-citations"><a id="body-ref-sref33-1" href="#" role="doc-biblioref" data-xml-rid="bib33" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref33-1" aria-controls="sref33-1">Kaiser and Feng, 2019</a><div class="dropBlock__holder" data-db-target-of="sref33-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">34.</div><div>Kaiser, T. ∙ Feng, G.</div><div><strong>Tmem119-EGFP and Tmem119-CreERT2 Transgenic Mice for Labeling and Manipulating Microglia</strong></div><div><em>eNeuro.</em> 2019; <strong>6</strong></div><div>ENEURO.0448-18.2019</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1523/ENEURO.0448-18.2019" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85071714639" target="_blank" aria-label="Open Scopus in new tab">Scopus (136)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31371457/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1523%2FENEURO.0448-18.2019&amp;pmid=31371457" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>Cat #031820; RRID: IMSR_JAX:031820</td></tr><tr><td>Mouse: <i>B6.Cg-Gt(ROSA)26Sor</i><sup><i>tm14(CAG-tdTomato)Hze</i></sup><i>/J</i></td><td>Jackson Laboratories</td><td>Cat #007914; RRID: IMSR_JAX:007914</td></tr><tr><td>Mouse: <i>B6.Cg-Gt(ROSA)26Sor</i><sup><i>tm32(CAG-COP4∗H134R/EYFP)Hze</i></sup><i>/J</i></td><td>Jackson Laboratories</td><td>Cat #024109; RRID: IMSR_JAX:024109</td></tr><tr><td>Mouse: <i>B6;129P2-Pvalb</i><sup><i>tm1(cre)Arbr</i></sup><i>/J</i></td><td>Jackson Laboratories</td><td>Cat #008069; RRID: IMSR_JAX:008069</td></tr><tr><td>Mouse: <i>Tg(Cx3cr1-cre)MW126Gsat/Mmucd</i></td><td>MMRRC</td><td>Cat #036395-UCD; RRID:MMRRC_036395-UCD</td></tr><tr><td>Mouse: <i>B6;129S-Gt(ROSA)26Sor</i><sup><i>tm34.1(CAG-Syp/tdTomato)Hze</i></sup><i>/J</i></td><td>Jackson Laboratories</td><td>Cat #012570; RRID: IMSR_JAX:012570</td></tr><tr><td>Mouse: <i>C57BL/6-Tmem119</i><sup><i>em2(EGFP)Gfng</i></sup><i>/J</i></td><td>Jackson Laboratories; <span class="dropBlock reference-citations"><a id="body-ref-sref33-2" href="#" role="doc-biblioref" data-xml-rid="bib33" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref33-2" aria-controls="sref33-2">Kaiser and Feng, 2019</a><div class="dropBlock__holder" data-db-target-of="sref33-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">34.</div><div>Kaiser, T. ∙ Feng, G.</div><div><strong>Tmem119-EGFP and Tmem119-CreERT2 Transgenic Mice for Labeling and Manipulating Microglia</strong></div><div><em>eNeuro.</em> 2019; <strong>6</strong></div><div>ENEURO.0448-18.2019</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1523/ENEURO.0448-18.2019" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85071714639" target="_blank" aria-label="Open Scopus in new tab">Scopus (136)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31371457/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1523%2FENEURO.0448-18.2019&amp;pmid=31371457" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>Cat #031823; RRID: IMSR_JAX:031823</td></tr><tr><td>Mouse: <i>Sst</i><sup><i>tm3.1(flpo)Zjh</i></sup><i>/J</i></td><td>Jackson Laboratories</td><td>Cat #028579; RRID: IMSR_JAX:028579</td></tr><tr><td>Mouse: <i>Gt(ROSA)26Sor</i><sup><i>tm1.2(CAG-EGFP)Fsh</i></sup><i>/Mmjax</i></td><td>Jackson Laboratories</td><td>Cat #032038; RRID: MMRRC_032038-JAX</td></tr><tr><td>Mouse: <i>C1qa</i><sup>−/−</sup></td><td><span class="dropBlock reference-citations"><a id="body-ref-sref6-1" href="#" role="doc-biblioref" data-xml-rid="bib6" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref6-1" aria-controls="sref6-1">Botto et&nbsp;al., 1998</a><div class="dropBlock__holder" data-db-target-of="sref6-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">6.</div><div>Botto, M. ∙ Dell’Agnola, C. ∙ Bygrave, A.E. ...</div><div><strong>Homozygous C1q deficiency causes glomerulonephritis associated with multiple apoptotic bodies</strong></div><div><em>Nat. Genet.</em> 1998; <strong>19</strong>:56-59</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng0598-56" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-0031778836" target="_blank" aria-label="Open Scopus in new tab">Scopus (1273)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/9590289/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng0598-56&amp;pmid=9590289" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>N/A</td></tr><tr><td>Mouse: <i>Sst</i><sup><i>tm2.1(cre)Zjh</i></sup><i>/J</i></td><td>Jackson Laboratories</td><td>Cat #013044; RRID: IMSR_JAX:013044</td></tr><tr><td>Mouse: <i>B6;129S-Slc17a7</i><sup><i>tm1.1(cre)Hze</i></sup><i>/J</i></td><td>Jackson Laboratories</td><td>Cat #023527; RRID: IMSR_JAX:023527</td></tr><tr><td>Mouse: <i>Gt(ROSA)26Sor</i><sup><i>tm1.1(CAG-EGFP)Fsh</i></sup><i>/Mmjax</i></td><td>Jackson Laboratories</td><td>Cat #032037; RRID: RRID:MMRRC_032037-JAX</td></tr><tr><td>Mouse: <i>Tg(Lhx6-EGFP)BP221Gsat/Mmmh</i></td><td>MMRRC</td><td>Cat #000246-MU; RRID:MMRRC_000246-MU</td></tr><tr><td>Mouse: C57BL/6J</td><td>Jackson Laboratories</td><td>Cat #000664; RRID: IMSR_JAX:000664</td></tr><tr class="rowsep"><td>Mouse: BALB/cJ</td><td>Jackson Laboratories</td><td>Cat #000651; RRID: IMSR_JAX:000651</td></tr><tr class="rowsep"><td colspan="3"><b>Oligonucleotides</b></td></tr><tr><td>RNAscope® Probe- Mm-Gabbr1</td><td>ACDBio</td><td>Cat #425181;</td></tr><tr><td>RNAscope® Probe- Mm-Gabbr2</td><td>ACDBio</td><td>Cat #317971;</td></tr><tr><td>RNAscope® Probe- Mm-Ex7-8-Gabbr1</td><td>ACDBio</td><td>Custom made</td></tr><tr><td>MERFISH 12-bit probe library</td><td>Vizgen</td><td>Cat #VZG114</td></tr><tr><td>Primer WPRE sequence Forward: AGC TCC TTT CCG GGA CTT TC</td><td>This paper</td><td>N/A</td></tr><tr class="rowsep"><td>Primer WPRE sequence Forward: CAC CAC GGA ATT GTC AGT GC</td><td>This paper</td><td>N/A</td></tr><tr class="rowsep"><td colspan="3"><b>Recombinant DNA</b></td></tr><tr><td>AAV-PVe(S5E2)-Syp-dTomato</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref82-3" href="#" role="doc-biblioref" data-xml-rid="bib82" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref82-3" aria-controls="sref82-3">Vormstein-Schneider et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref82-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">84.</div><div>Vormstein-Schneider, D. ∙ Lin, J.D. ∙ Pelkey, K.A. ...</div><div><strong>Viral manipulation of functionally distinct interneurons in mice, non-human primates and humans</strong></div><div><em>Nat. Neurosci.</em> 2020; <strong>23</strong>:1629-1636</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41593-020-0692-9" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_84_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85089867328" target="_blank" aria-label="Open Scopus in new tab">Scopus (100)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32807948/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41593-020-0692-9&amp;pmid=32807948" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>N/A</td></tr><tr class="rowsep"><td>AAV-PVe(S5E2)-Syp-Gamillus</td><td>This paper</td><td>N/A</td></tr><tr class="rowsep"><td colspan="3"><b>Software and algorithms</b></td></tr><tr><td>ImageJ – Fiji 2.0.0</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref65-1" href="#" role="doc-biblioref" data-xml-rid="bib65" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref65-1" aria-controls="sref65-1">Schneider et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref65-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">67.</div><div>Schneider, C.A. ∙ Rasband, W.S. ∙ Eliceiri, K.W.</div><div><strong>NIH Image to ImageJ: 25 years of image analysis</strong></div><div><em>Nat. Methods.</em> 2012; <strong>9</strong>:671-675</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.2089" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_67_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-84863205849" target="_blank" aria-label="Open Scopus in new tab">Scopus (42378)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22930834/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.2089&amp;pmid=22930834" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0130" href="https://imagej.nih.gov/ij/" target="_blank">https://imagej.nih.gov/ij/</a></td></tr><tr><td>MATLAB R2014b/R2017a (Imaris analysis)</td><td>MathWorks</td><td><a id="intref0135" href="https://www.mathworks.com/" target="_blank">https://www.mathworks.com/</a></td></tr><tr><td>MATLAB R2019a (9.6.0.1072779) 64-bit (glnxa64) (MERFISH analysis)</td><td>MathWorks</td><td><a id="intref0140" href="https://www.mathworks.com/" target="_blank">https://www.mathworks.com/</a></td></tr><tr><td>GraphPad Prism 8/9</td><td>GraphPad Software</td><td><a id="intref0145" href="https://www.graphpad.com/scientific-software/prism/" target="_blank">https://www.graphpad.com/scientific-software/prism/</a></td></tr><tr><td>Zen blue 2.6</td><td>Zeiss</td><td><a id="intref0150" href="https://www.zeiss.com/microscopy/int/products/microscope-software/zen.html" target="_blank">https://www.zeiss.com/microscopy/int/products/microscope-software/zen.html</a></td></tr><tr><td>Imaris 8.1.2/9.3.1/9.5.0</td><td>Bitplane</td><td><a id="intref0155" href="https://imaris.oxinst.com" target="_blank">https://imaris.oxinst.com</a></td></tr><tr><td>Clampfit</td><td>Molecular Devices</td><td><a id="intref0160" href="https://mdc.custhelp.com/app/answers/detail/a_id/18779/%7E/axon%E2%84%A2pclamp%E2%84%A2-10-electrophysiology-data-acquisition-%26-analysis-software-download" target="_blank">https://mdc.custhelp.com/app/answers/detail/a_id/18779/∼/axon%E2%84%A2pclamp%E2%84%A2-10-electrophysiology-data-acquisition-%26-analysis-software-download</a></td></tr><tr><td>Minianalysis</td><td>Synaptosoft</td><td><a id="intref0165" href="http://www.synaptosoft.com/MiniAnalysis/" target="_blank">http://www.synaptosoft.com/MiniAnalysis/</a></td></tr><tr><td>ThorImageLS 3.2</td><td>Thorlabs</td><td><a id="intref0170" href="https://www.thorlabs.com/newgrouppage9.cfm?objectgroup_id=9072#ad-image-0" target="_blank">https://www.thorlabs.com/newgrouppage9.cfm?objectgroup_id=9072#ad-image-0</a></td></tr><tr><td>Cell Ranger v3.0.0</td><td>10x Genomics</td><td><a id="intref0175" href="https://support.10xgenomics.com/single-cell-gene-expression/software/pipelines/latest/installation" target="_blank">https://support.10xgenomics.com/single-cell-gene-expression/software/pipelines/latest/installation</a></td></tr><tr><td>Monocle3 v0.2.3.0</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref8-1" href="#" role="doc-biblioref" data-xml-rid="bib8" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref8-1" aria-controls="sref8-1">Cao et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref8-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">8.</div><div>Cao, J. ∙ Spielmann, M. ∙ Qiu, X. ...</div><div><strong>The single-cell transcriptional landscape of mammalian organogenesis</strong></div><div><em>Nature.</em> 2019; <strong>566</strong>:496-502</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-0969-x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85062419120" target="_blank" aria-label="Open Scopus in new tab">Scopus (1641)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30787437/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-0969-x&amp;pmid=30787437" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0180" href="https://cole-trapnell-lab.github.io/monocle3/" target="_blank">https://cole-trapnell-lab.github.io/monocle3/</a></td></tr><tr><td>Seurat v4.0.0</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref62-1" href="#" role="doc-biblioref" data-xml-rid="bib62" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref62-1" aria-controls="sref62-1">Satija et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref62-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">64.</div><div>Satija, R. ∙ Farrell, J.A. ∙ Gennert, D. ...</div><div><strong>Spatial reconstruction of single-cell gene expression data</strong></div><div><em>Nat. Biotechnol.</em> 2015; <strong>33</strong>:495-502</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nbt.3192" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_64_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-84929151009" target="_blank" aria-label="Open Scopus in new tab">Scopus (3012)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25867923/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnbt.3192&amp;pmid=25867923" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0185" href="https://satijalab.org/seurat/" target="_blank">https://satijalab.org/seurat/</a></td></tr><tr><td>Scanpy 1.6.1.dev78+gd6457902</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref3-1" href="#" role="doc-biblioref" data-xml-rid="bib3" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref3-1" aria-controls="sref3-1">Amir et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref3-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">3.</div><div>Amir, A.D. ∙ Davis, K.L. ∙ Tadmor, M.D. ...</div><div><strong>viSNE enables visualization of high dimensional single-cell data and reveals phenotypic heterogeneity of leukemia</strong></div><div><em>Nat. Biotechnol.</em> 2013; <strong>31</strong>:545-552</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nbt.2594" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-84880280631" target="_blank" aria-label="Open Scopus in new tab">Scopus (1197)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23685480/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnbt.2594&amp;pmid=23685480" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref87-1" href="#" role="doc-biblioref" data-xml-rid="bib87" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref87-1" aria-controls="sref87-1">Wolf et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref87-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">89.</div><div>Wolf, F.A. ∙ Angerer, P. ∙ Theis, F.J.</div><div><strong>SCANPY: large-scale single-cell gene expression data analysis</strong></div><div><em>Genome Biol.</em> 2018; <strong>19</strong>:15</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13059-017-1382-0" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_89_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85041394976" target="_blank" aria-label="Open Scopus in new tab">Scopus (2665)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29409532/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13059-017-1382-0&amp;pmid=29409532" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0190" href="https://scanpy.readthedocs.io/en/stable/" target="_blank">https://scanpy.readthedocs.io/en/stable/</a></td></tr><tr><td>R 4.0.0</td><td>The R Foundation</td><td><a id="intref0195" href="http://www.r-project.org/" target="_blank">http://www.r-project.org/</a></td></tr><tr><td>DEWÄKSS</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref78-2" href="#" role="doc-biblioref" data-xml-rid="bib78" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref78-2" aria-controls="sref78-2">Tjärnberg et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref78-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">80.</div><div>Tjärnberg, A. ∙ Mahmood, O. ∙ Jackson, C.A. ...</div><div><strong>Optimal tuning of weighted kNN- and diffusion-based methods for denoising single cell genomics data</strong></div><div><em>bioRxiv.</em> 2020;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/2020.02.28.970202" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_80_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85098901697" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2F2020.02.28.970202" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>N/A</td></tr><tr><td>Python 3.8.5</td><td>Python Software Foundation</td><td><a id="intref0200" href="https://www.python.org/" target="_blank">https://www.python.org/</a></td></tr><tr><td>MERlin</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref21-1" href="#" role="doc-biblioref" data-xml-rid="bib21" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref21-1" aria-controls="sref21-1">Emanuel and Babcock, 2020</a><div class="dropBlock__holder" data-db-target-of="sref21-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">22.</div><div>Emanuel, G. ∙ Babcock, H.</div><div><strong>ZhuangLab/MERlin: MERlin v0.1.6</strong></div><div>Zenodo, 2020</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=G.EmanuelH.BabcockZhuangLab%2FMERlin%3A+MERlin+v0.1.62020Zenodo" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0205" href="https://github.com/emanuega/MERlin" target="_blank">https://github.com/emanuega/MERlin</a></td></tr><tr><td>Scran 1.18.5</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref43-1" href="#" role="doc-biblioref" data-xml-rid="bib43" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref43-1" aria-controls="sref43-1">Lun et&nbsp;al., 2016b</a><div class="dropBlock__holder" data-db-target-of="sref43-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">44.</div><div>Lun, A.T.L. ∙ McCarthy, D.J. ∙ Marioni, J.C.</div><div><strong>A step-by-step workflow for low-level analysis of single-cell RNA-seq data with Bioconductor</strong></div><div><em>F1000Res.</em> 2016; <strong>5</strong>:2122</div></div><div class="external-links"><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27909575/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?pmid=27909575" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0210" href="https://rdrr.io/bioc/scran/" target="_blank">https://rdrr.io/bioc/scran/</a></td></tr><tr><td>Scater 1.18.6</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref49-1" href="#" role="doc-biblioref" data-xml-rid="bib49" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref49-1" aria-controls="sref49-1">McCarthy et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref49-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">50.</div><div>McCarthy, D.J. ∙ Campbell, K.R. ∙ Lun, A.T.L. ...</div><div><strong>Scater: pre-processing, quality control, normalization and visualization of single-cell RNA-seq data in R</strong></div><div><em>Bioinformatics.</em> 2017; <strong>33</strong>:1179-1186</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btw777" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85019072518" target="_blank" aria-label="Open Scopus in new tab">Scopus (861)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28088763/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtw777&amp;pmid=28088763" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0215" href="https://rdrr.io/github/davismcc/scater/" target="_blank">https://rdrr.io/github/davismcc/scater/</a></td></tr><tr><td>SingleCellExperiment 1.12.0</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref2-1" href="#" role="doc-biblioref" data-xml-rid="bib2" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref2-1" aria-controls="sref2-1">Amezquita et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref2-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">2.</div><div>Amezquita, R.A. ∙ Lun, A.T.L. ∙ Becht, E. ...</div><div><strong>Orchestrating single-cell analysis with Bioconductor</strong></div><div><em>Nat. Methods.</em> 2020; <strong>17</strong>:137-145</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41592-019-0654-x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_2_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85076035714" target="_blank" aria-label="Open Scopus in new tab">Scopus (325)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31792435/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41592-019-0654-x&amp;pmid=31792435" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0220" href="https://rdrr.io/bioc/SingleCellExperiment/" target="_blank">https://rdrr.io/bioc/SingleCellExperiment/</a></td></tr><tr><td>Igraph 1.2.6</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref15-1" href="#" role="doc-biblioref" data-xml-rid="bib15" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref15-1" aria-controls="sref15-1">Csardi and Nepusz, 2006</a><div class="dropBlock__holder" data-db-target-of="sref15-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">16.</div><div>Csardi, G. ∙ Nepusz, T.</div><div><strong>The igraph software package for complex network research</strong></div><div><em>InterJournal Complex Syst.</em> 2006; <strong>1695</strong>:1-9</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=G.CsardiT.NepuszThe+igraph+software+package+for+complex+network+researchInterJournal+Complex+Syst.1695200619" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0225" href="https://igraph.org/" target="_blank">https://igraph.org/</a></td></tr><tr class="rowsep"><td>MoSeq</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref86-2" href="#" role="doc-biblioref" data-xml-rid="bib86" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref86-2" aria-controls="sref86-2">Wiltschko et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref86-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">88.</div><div>Wiltschko, A.B. ∙ Johnson, M.J. ∙ Iurilli, G. ...</div><div><strong>Mapping Sub-Second Structure in Mouse Behavior</strong></div><div><em>Neuron.</em> 2015; <strong>88</strong>:1121-1135</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.neuron.2015.11.031&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.neuron.2015.11.031&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-84957565912" target="_blank" aria-label="Open Scopus in new tab">Scopus (373)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26687221/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.neuron.2015.11.031&amp;pmid=26687221" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>N/A</td></tr><tr class="rowsep"><td colspan="3"><b>Deposited data</b></td></tr><tr><td>Raw and analyzed scRNA-seq data</td><td>This paper</td><td>GEO: <a id="intref0230" href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE159947" target="_blank">GSE159947</a></td></tr><tr><td>Raw and analyzed MERFISH data</td><td>This paper</td><td><a id="intref0235" href="https://drive.google.com/drive/folders/1TG4KkAW6-0HBmKnYqHCD4vfbBZXzSXkA" target="_blank">https://drive.google.com/drive/folders/1TG4KkAW6-0HBmKnYqHCD4vfbBZXzSXkA</a></td></tr><tr><td>Code for scRNA-seq analysis</td><td>This paper; <a id="intref0240" href="https://doi.org/10.5281/zenodo.4899812">https://doi.org/10.5281/zenodo.4899812</a></td><td><a id="intref0245" href="https://github.com/emiliafavuzzi/Favuzzi_et_al_2021" target="_blank">https://github.com/emiliafavuzzi/Favuzzi_et_al_2021</a></td></tr><tr><td>Code for synaptic analysis</td><td>This paper; <a id="intref0250" href="https://doi.org/10.5281/zenodo.4899956">https://doi.org/10.5281/zenodo.4899956</a></td><td><a id="intref0255" href="https://github.com/emiliafavuzzi/synaptic-analyses" target="_blank">https://github.com/emiliafavuzzi/synaptic-analyses</a></td></tr><tr><td>Code for MERFISH analysis</td><td>This paper; <a id="intref0260" href="https://doi.org/10.5281/zenodo.4899812">https://doi.org/10.5281/zenodo.4899812</a></td><td><a id="intref0265" href="https://github.com/emiliafavuzzi/Favuzzi_et_al_2021" target="_blank">https://github.com/emiliafavuzzi/Favuzzi_et_al_2021</a></td></tr></tbody></table></div><figcaption><div class="core-table-tools"><ul><li><a class="core-table-show-action" href="/action/showFullTableHTML?isHtml=true&amp;tableId=undtbl1&amp;pii=S0092-8674%2821%2900753-4" target="_blank">Open table in a new tab</a></li></ul></div></figcaption></figure></div></div></section><section id="sec-4-2"><h3>Resource availability</h3><section id="sec-4-2-1"><h4>Lead contact</h4><div id="p0230" role="paragraph">Further information and requests for reagents should be directed to and will be fulfilled by Lead Contact, Gord Fishell (<a id="intref0270" href="mailto:Gordon_Fishell@hms.harvard.edu">Gordon_Fishell@hms.harvard.edu</a>).</div></section><section id="sec-4-2-2"><h4>Materials availability</h4><div id="p0235" role="paragraph">The AAV-PVe-Syp-Gamillus plasmid generated in this study is available from the corresponding author on request.</div></section><section id="sec-4-2-3"><h4>Data and code availability</h4><div id="p0240" role="paragraph">scRNA-seq data have been deposited in NCBI GEO and assigned the following accession number <a id="intref0275" href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE159947" target="_blank">GSE159947</a>.</div><div id="p0245" role="paragraph">MERFISH raw and processed data are available at: <a id="intref0280" href="https://drive.google.com/drive/folders/1TG4KkAW6-0HBmKnYqHCD4vfbBZXzSXkA" target="_blank">https://drive.google.com/drive/folders/1TG4KkAW6-0HBmKnYqHCD4vfbBZXzSXkA</a></div><div id="p0250" role="paragraph">Should this link not be accessible, MERFISH processed data are also available at: <a id="intref0285" href="https://github.com/emiliafavuzzi/Favuzzi_et_al_2021/tree/main/MERFISH" target="_blank">https://github.com/emiliafavuzzi/Favuzzi_et_al_2021/tree/main/MERFISH</a> while the raw data will be shared upon request in a manner determined on a case by case basis (e.g., hard drive, cloud server).</div><div id="p0255" role="paragraph">Scripts for synaptic analyses are available at: <a id="intref0290" href="https://github.com/emiliafavuzzi/synaptic-analyses" target="_blank">https://github.com/emiliafavuzzi/synaptic-analyses</a> (<a id="intref0295" href="https://doi.org/10.5281/zenodo.4899956">https://doi.org/10.5281/zenodo.4899956</a>)</div><div id="p0260" role="paragraph">Scripts reproducing scRNA-seq and MERFISH analyses are available at: <a id="intref0300" href="https://github.com/emiliafavuzzi/Favuzzi_et_al_2021" target="_blank">https://github.com/emiliafavuzzi/Favuzzi_et_al_2021</a> (<a id="intref0305" href="https://doi.org/10.5281/zenodo.4899812">https://doi.org/10.5281/zenodo.4899812</a>)</div></section></section><section id="sec-4-3"><h3>Experimental model and subject details</h3><div id="p0265" role="paragraph"><i>Cx3Cr1</i><sup><i>GFP/+</i></sup> (Jackson Laboratories #005582), <i>Tmem119</i><sup><i>CreER/+</i></sup> (kind gift from G. Feng; Jackson Laboratories #031820) (<span class="dropBlock reference-citations"><a id="body-ref-sref33-3" href="#" role="doc-biblioref" data-xml-rid="bib33" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref33-3" aria-controls="sref33-3">Kaiser and Feng, 2019</a><div class="dropBlock__holder" data-db-target-of="sref33-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">34.</div><div>Kaiser, T. ∙ Feng, G.</div><div><strong>Tmem119-EGFP and Tmem119-CreERT2 Transgenic Mice for Labeling and Manipulating Microglia</strong></div><div><em>eNeuro.</em> 2019; <strong>6</strong></div><div>ENEURO.0448-18.2019</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1523/ENEURO.0448-18.2019" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85071714639" target="_blank" aria-label="Open Scopus in new tab">Scopus (136)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31371457/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1523%2FENEURO.0448-18.2019&amp;pmid=31371457" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), <i>Cx3Cr1</i><sup><i>Cre/+</i></sup> (Jackson Laboratories #025524), <i>Cx3Cr1</i><sup><i>Cre/+</i></sup> (MMRRC #036395-UCD), <i>Ai14</i> (Jackson Laboratories #007914), <i>PV</i><sup><i>Cre/+</i></sup> (Jackson Laboratories #008069), <i>Tmem119</i><sup><i>GFP</i></sup> (kind gift from G. Feng; Jackson Laboratories #031823) (<span class="dropBlock reference-citations"><a id="body-ref-sref33-4" href="#" role="doc-biblioref" data-xml-rid="bib33" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref33-4" aria-controls="sref33-4">Kaiser and Feng, 2019</a><div class="dropBlock__holder" data-db-target-of="sref33-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">34.</div><div>Kaiser, T. ∙ Feng, G.</div><div><strong>Tmem119-EGFP and Tmem119-CreERT2 Transgenic Mice for Labeling and Manipulating Microglia</strong></div><div><em>eNeuro.</em> 2019; <strong>6</strong></div><div>ENEURO.0448-18.2019</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1523/ENEURO.0448-18.2019" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85071714639" target="_blank" aria-label="Open Scopus in new tab">Scopus (136)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31371457/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1523%2FENEURO.0448-18.2019&amp;pmid=31371457" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), <i>SST</i><sup><i>Flp/+</i></sup> (Jackson Laboratories #028579), <i>RCE:FRT</i> (Jackson Laboratories #032038), <i>C1qa</i><sup>−/−</sup> (kind gift from M. Botto), Ai34 (Jackson Laboratories #012570) (<span class="dropBlock reference-citations"><a id="body-ref-sref6-2" href="#" role="doc-biblioref" data-xml-rid="bib6" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref6-2" aria-controls="sref6-2">Botto et&nbsp;al., 1998</a><div class="dropBlock__holder" data-db-target-of="sref6-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">6.</div><div>Botto, M. ∙ Dell’Agnola, C. ∙ Bygrave, A.E. ...</div><div><strong>Homozygous C1q deficiency causes glomerulonephritis associated with multiple apoptotic bodies</strong></div><div><em>Nat. Genet.</em> 1998; <strong>19</strong>:56-59</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng0598-56" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-0031778836" target="_blank" aria-label="Open Scopus in new tab">Scopus (1273)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/9590289/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng0598-56&amp;pmid=9590289" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), <i>SST</i><sup><i>Cre/+</i></sup> (Jackson Laboratories #013044), <i>VGlut1</i><sup><i>Cre/+</i></sup> (Jackson Laboratories #023527) and <i>Ai32</i> (Jackson Laboratories #024109) were maintained in a C57BL/6 background; GABA<sub>B1</sub> floxed mice (kind gift from B. Bettler) (<span class="dropBlock reference-citations"><a id="body-ref-sref30-2" href="#" role="doc-biblioref" data-xml-rid="bib30" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref30-2" aria-controls="sref30-2">Haller et&nbsp;al., 2004</a><div class="dropBlock__holder" data-db-target-of="sref30-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">31.</div><div>Haller, C. ∙ Casanova, E. ∙ Müller, M. ...</div><div><strong>Floxed allele for conditional inactivation of the GABAB(1) gene</strong></div><div><em>Genesis.</em> 2004; <strong>40</strong>:125-130</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/gene.20073" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-10344244007" target="_blank" aria-label="Open Scopus in new tab">Scopus (45)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15493018/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fgene.20073&amp;pmid=15493018" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) were maintained in a BALB/c background; <i>RCE:loxP</i> (Jackson Laboratories #032037) and <i>Lhx6</i><sup><i>GFP</i></sup> (GENSAT, MMRRC #000246-MU) were maintained in a SWR/J background.</div><div id="p0270" role="paragraph">Animals were group housed and maintained under standard, temperature controlled laboratory conditions. Mice were kept on a 12:12 light/dark cycle and received water and food <i>ad libitum</i>. Both male and female mice were used for all experiments. Except for the behavioral experiments, similar results were obtained in both males and females.</div><div id="p0275" role="paragraph">For all experiments involving a comparison between conditions, littermates were used and they were always processed and analyzed together to minimize technical variability. For all experiments involving a comparison between controls (e.g., <i>Cx3cr1</i><sup><i>+/+</i></sup><i>; GABA</i><sub><i>B1</i></sub><i>R</i><sup><i>fl/fl</i></sup>) and GABA<sub>B1</sub>R cKO (e.g., <i>Cx3cr1</i><sup><i>Cre/+</i></sup><i>; GABA</i><sub><i>B1</i></sub><i>R</i><sup><i>fl/fl</i></sup>) mice, wild-type littermates (<i>Cx3cr1</i><sup><i>+/+</i></sup><i>; GABA</i><sub><i>B1</i></sub><i>R</i><sup><i>fl/fl</i></sup>) were used. When specified, Cre-Het controls (<i>Cx3cr1</i><sup><i>Cre/+</i></sup><i>; GABA</i><sub><i>B1</i></sub><i>R</i><sup><i>+/+</i></sup>) were used as additional control.</div><div id="p0280" role="paragraph"><i>Cx3cr1</i><sup><i>Cre</i></sup> mice from Jackson Laboratories were used in most of the experiments while <i>Cx3cr1</i><sup><i>Cre</i></sup> mice from MMRRC were used to confirm most synaptic phenotypes (not shown, except in <a id="crosref1320" href="#figs4">Figure&nbsp;S4</a>E) and in the MERFISH experiment in <a id="crosref1325" href="#fig6">Figure&nbsp;6</a>.</div><div id="p0285" role="paragraph">All animal maintenance and experimental procedures were conducted in accordance with the regulations established by the National Institutes of Health (NIH, USA) and were approved by the Institutional Animal Care and Use Committee at Harvard Medical School (protocol number IS00001269) and The Broad Institute (protocol number 0156-03-17).</div></section><section id="sec-4-4"><h3>Method details</h3><section id="sec-4-4-1"><h4>Microglia Depletion</h4><div id="p0290" role="paragraph">Microglia depletion was achieved by administering the CSF1R inhibitor PLX5622 (generously provided by Plexxikon Inc, <span class="dropBlock reference-citations"><a id="body-ref-sref69-2" href="#" role="doc-biblioref" data-xml-rid="bib69" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref69-2" aria-controls="sref69-2">Spangenberg et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref69-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">71.</div><div>Spangenberg, E. ∙ Severson, P.L. ∙ Hohsfield, L.A. ...</div><div><strong>Sustained microglial depletion with CSF1R inhibitor impairs parenchymal plaque development in an Alzheimer’s disease model</strong></div><div><em>Nat. Commun.</em> 2019; <strong>10</strong>:3758</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41467-019-11674-z" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85071120834" target="_blank" aria-label="Open Scopus in new tab">Scopus (418)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31434879/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41467-019-11674-z&amp;pmid=31434879" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). To deplete microglia in mice older than post-natal (P) day 18, animals were fed PLX5622-formulated AIN-76A diet (1200&nbsp;mg PLX5622 per kilogram added to chow AIN-76A, Research Diets) <i>ad libitum</i>. Control mice received control diet (AIN-76A, Research Diets). To ablate microglia at early postnatal stages (P1-P18), PLX5622 was delivered to pups via intragastric (P1-P7) and intraperitoneal (P8-P18) injections. PLX5622 powder was provided by Plexxikon. A 20&nbsp;× stock was made every week by dissolving PLX5622-FA in DMSO. The 20&nbsp;× stock was stored at room temperature, protected from light. A diluent containing 0.5% hydroxypropyl methyl cellulose (HPMC, Sigma) and 1% Polysorbate 80 (PS80, Sigma) in deionized water was prepared. On each dosing day, a working solution containing 0.5 volumes of the PLX5622-DMSO 20&nbsp;× stock (or DMSO 20&nbsp;× as control) were dissolved to obtain a final concentration of 0.5% HPMC, 1% PS80, 5% DMSO in water. The mixture was placed in a sonicating water bath for 20-45&nbsp;minutes to make a uniform suspension. Care was taken not to allow the compound to settle for more than 10&nbsp;minutes before injection. P1-P18 pups received an intragastric or intraperitoneal injection of 50&nbsp;mg/kg of PLX5622 every 24 hours. PLX5622 early postnatal treatment was well tolerated and resulted in more than 95% survival.</div></section><section id="sec-4-4-2"><h4>Perfusion and Immunohistochemistry</h4><div id="p0295" role="paragraph">Animals were deeply anesthetized with sodium pentobarbital by intraperitoneal injection and transcardially perfused with PBS followed by 4% paraformaldehyde (PFA) in PBS. Brains were dissected out, postfixed for two hours at 4°C and cryoprotected in 30% sucrose-PBS solutions overnight at 4°C. Tissue was sectioned at 40&nbsp;μm on a sliding microtome (Leica). Free-floating brain sections were incubated with 0.3% Triton X-100 in PBS for 4&nbsp;× 15&nbsp;minutes and then blocked for 2 hours (0.3% Triton X-100, 10% Normal Goat and/or Donkey Serum), followed by incubation with primary antibodies in 0.3% Triton X-100, 5% Normal Goat and/or Donkey Serum overnight at 4°C. The next day, sections were rinsed 4&nbsp;× 15&nbsp;minutes in PBS, incubated with the appropriate secondary antibodies in 0.3% Triton X-100, 5% Normal Goat and/or Donkey Serum for 2 hours at room temperature, rinsed 4&nbsp;× 15&nbsp;minutes in PBS and then incubated with DAPI. The following primary antibodies were used: mouse anti-Synaptotagmin-2 (1:1000, ZFIN #ZDB-ATB-081002-25), mouse anti-gephyrin (1:500, Synaptic Systems #147 011), rabbit anti-NeuN (1:500, Millipore #ABN78), guinea-pig anti-NeuN (1:500, Millipore #ABN90P), guinea-pig anti-VGlut2 (1:2000, Millipore #AB2251), guinea-pig anti-VGlut2 (1:1000, Synaptic Systems #135404), guinea-pig anti-VGlut1 (1:1000, Millipore #AB5905), rabbit anti-Homer 1b/c (1:500, Synaptic Systems #160023), mouse anti-parvalbumin (1:1000, Sigma #P-3088), rabbit anti-DsRed (1:500, Clontech #632496), chicken anti-GFP (1:1000, Aves Lab #1020), rabbit anti-Iba1 (1:500, Wako Chemicals #019-19741), guinea-pig anti-Iba1 (1:500, Synaptic Systems #234 004), guinea-pig anti-parvalbumin (1:2000, Swant #GP72), rabbit anti-C1q (1:500, Abcam #ab182451), rat anti-CD68 (1:500, Bio-Rad #MCA1957), mouse anti-GABABR1 (1:500, NeuroMab #73-183), guinea-pig anti-GABABR2 (1:500, Millipore #AB2255), anti-WFA (1:500, L1516-2mg), mouse anti-GAD65 (1:500, Millipore #MAB351R), rabbit anti-SST (1:2,000, Peninsula Laboratories #T4103.0050). Of note, the following anti-GABABR1 antibody exhibited non-specific signal in GABA<sub>B1</sub>R cKO mice and we therefore discourage its use: Synaptic Systems # 322 102. Conversely, the following anti-GABABR2 antibody did not work in our hands: clone N81/2, NeuroMab # 73-124. Sections were mounted in Fluoromount-G (Southern Biotechnology, #100241-874), except for STED imaging where ProLong Diamond Antifade Mountant (Thermo Fisher Scientific, #P36961) was used.</div></section><section id="sec-4-4-3"><h4>Generation of AAV Expression Vectors</h4><div id="p0300" role="paragraph">The <i>pAAV-PVe-Syp-tdTomato</i> (synaptophysin-tdTomato under the control of the S5E2 PV-specific enhancer) was engineered as follows: the <i>pAAV-S5E2-dTomato</i> (<span class="dropBlock reference-citations"><a id="body-ref-sref82-4" href="#" role="doc-biblioref" data-xml-rid="bib82" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref82-4" aria-controls="sref82-4">Vormstein-Schneider et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref82-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">84.</div><div>Vormstein-Schneider, D. ∙ Lin, J.D. ∙ Pelkey, K.A. ...</div><div><strong>Viral manipulation of functionally distinct interneurons in mice, non-human primates and humans</strong></div><div><em>Nat. Neurosci.</em> 2020; <strong>23</strong>:1629-1636</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41593-020-0692-9" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_84_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85089867328" target="_blank" aria-label="Open Scopus in new tab">Scopus (100)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32807948/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41593-020-0692-9&amp;pmid=32807948" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) was used as backbone and the dTomato reporter was replaced by the Syp-dTomato reporter (Ai34, a gift from Hongkui Zeng, Addgene plasmid #34881) (<span class="dropBlock reference-citations"><a id="body-ref-sref44" href="#" role="doc-biblioref" data-xml-rid="bib44" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref44" aria-controls="sref44">Madisen et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref44"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">45.</div><div>Madisen, L. ∙ Mao, T. ∙ Koch, H. ...</div><div><strong>A toolbox of Cre-dependent optogenetic transgenic mice for light-induced activation and silencing</strong></div><div><em>Nat. Neurosci.</em> 2012; <strong>15</strong>:793-802</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nn.3078" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_45_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-84860294289" target="_blank" aria-label="Open Scopus in new tab">Scopus (925)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22446880/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnn.3078&amp;pmid=22446880" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) using Gibson Assembly (primers: Fw- TCC ACA GCG AGC TCG CTA GCG CCA CCA TGG ACG TGGT and Rv-TCC AGA GGT TGA TTA TCG ATA AGC TTC TAC TTG TAC AGC TCG TCC ATG CC). The <i>pAAV-PVe-Syp-Gamillus</i> was engineered as follows: the <i>pAAV-PVe-Syp-tdTomato</i> was used as backbone and the dTomato reporter was replaced by the acid-tolerant monomeric green fluorescent protein Gamillus (Gamillus/pcDNA3, a gift from Takeharu Nagai, Addgene plasmid #124837) (<span class="dropBlock reference-citations"><a id="body-ref-sref66-2" href="#" role="doc-biblioref" data-xml-rid="bib66" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref66-2" aria-controls="sref66-2">Shinoda et&nbsp;al., 2018a</a><div class="dropBlock__holder" data-db-target-of="sref66-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">68.</div><div>Shinoda, H. ∙ Ma, Y. ∙ Nakashima, R. ...</div><div><strong>Acid-Tolerant Monomeric GFP from Olindias formosa</strong></div><div><em>Cell Chem. Biol.</em> 2018; <strong>25</strong>:330-338.e7</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.chembiol.2017.12.005&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.chembiol.2017.12.005&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85042356070" target="_blank" aria-label="Open Scopus in new tab">Scopus (53)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29290624/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.chembiol.2017.12.005&amp;pmid=29290624" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) using Gibson Assembly (primers: Fw- CTT CTC CAA TCA GAT GTC GCG AAT GGT GAG CAA GGG CGAG and Rv- TCC AGA GGT TGA TTA TCG ATA TTA CTT GTA CAG CTC GTC CAT GCC).</div></section><section id="sec-4-4-4"><h4>Cell Culture, transfection and AAV production</h4><div id="p0305" role="paragraph">HEK293FT cells (Thermo Fisher Scientific, #R70007) were cultured in Dulbecco’s Modified Eagle’s medium with high glucose and pyruvate, GlutaMAX Supplement, 10% fetal bovine serum, penicillin (100 units/ml) and streptomycin (100&nbsp;μg/ml). The cultures were incubated at 37°C in a humidified atmosphere containing 5% CO<sub>2</sub>. For AAV production, HEK293FT cells were seeded on 15-cm plates without antibiotics for 24 hours and co-transfected with the following plasmids using Polyethylenimine (100&nbsp;μg/dish, Polysciences, #23966-1): pHGTI-helper (22&nbsp;μg/dish), pAAV2/PHP.eB (cloned from pUCmini-iCAP-PHP.eB, a gift from Viviana Gradinaru, Addgene plasmid #103005; 9&nbsp;μg/dish) and the AAV expression vector (12&nbsp;μg/dish). 72 hours after transfection, transfected cells were harvested and lysed (150&nbsp;mM NaCl, 20&nbsp;mM Tris pH8.0) by three freeze-thaw cycles and Benzonase treatment (375&nbsp;U/dish; Sigma, #E1014) for 15&nbsp;minutes at 37°C. The supernatants were cleared by centrifugation at 4000 RPM for 20&nbsp;minutes at 4°C, then transferred to Iodixanol gradients (OptiPrep Density Gradient Medium, Sigma, #D1556) for ultracentrifugation (VTi50 rotor, Beckman Coulter) at 50,000 RPM for 1.5 hours at 16°C. The 40% iodixanol fraction containing the AAVs was collected, underwent ultrafiltration with PBS in Amicon Ultra (15&nbsp;ml, 100K, Millipore, #UFC910024) for 4 times, aliquoted and stored at −80°C. The number of genomic viral copies was determined by qPCR using the following primers against the WPRE sequence: Fw: AGC TCC TTT CCG GGA CTT TC and Rv: CAC CAC GGA ATT GTC AGT GC.</div></section><section id="sec-4-4-5"><h4>Intracranial injections</h4><div id="p0310" role="paragraph">P8-P10 pups were anesthetized with 5% isoflurane and mounted on a stereotaxic frame. Isoflurane concentration during surgery was kept between 1%–2% and the body temperature was maintained at 37 °C using a heating pad. Surgery was performed using the ‘no-touch’ sterile procedure, and all surgical tools were sterilized prior to surgery. The scalp was cleaned with betadine and ethanol (after shaving the skin for P10 injections) and cut open to expose the skull covering the somatosensory cortex. A small craniotomy (~1&nbsp;mm) was opened over the primary somatosensory cortex using a micro knife (Fine Science Tools). Then, 200 nL of <i>AAV-PHP.B-PVe-Syp-tdTomato</i> or <i>AAV-PHP.B-PVe-Syp-Gamillus</i> were unilaterally injected in the somatosensory cortex (anteroposterior –2.1/ −2.8 mm, mediolateral&nbsp;+2.4/2.6 mm relative to Lambda; dorsoventral –0.34 and –0.44 mm relative to the pial surface) at a rate of 100 nl/minute using a Nanoject III Injector (Drummond Scientific, USA) followed by 2 additional minutes to allow diffusion. After surgery, mice were given Meloxicam (Metacam) subcutaneously at 5mg/kg of body weight (Boehringer-Ingelheim) and, upon recovery, were placed back in the home cage with the mother.</div></section><section id="sec-4-4-6"><h4>Tamoxifen Induction</h4><div id="p0315" role="paragraph">Tamoxifen was dissolved in corn oil (20&nbsp;mg/ml) at 37°C with constant agitation and stored at −80°C. Before injection, a 10&nbsp;mg/ml dilution in corn oil was made. Microglia labeling in <i>Tmem119</i><sup><i>CreER/+</i></sup><i>; Ai14</i> mice or GABA<sub>B1</sub>R removal in <i>Tmem119</i><sup><i>CreER/+</i></sup><i>; GABA</i><sub><i>B1</i></sub><i>R</i><sup><i>fl/fl</i></sup> mice was achieved by performing intra-gastric tamoxifen injections in P2 and P3 post-natal mouse pups at a dose of 0.1mg/g of body weight for 3 consecutive days. Control mice (<i>Tmem119</i><sup><i>+/+</i></sup><i>; GABA</i><sub><i>B1</i></sub><i>R</i><sup><i>fl/fl</i></sup>) also received tamoxifen injections.</div></section><section id="sec-4-4-7"><h4>Single Molecule Fluorescent <i>In Situ</i> Hybridization Histochemistry</h4><div id="p0320" role="paragraph">For single molecule fluorescent <i>in situ</i> hybridization (smFISH) combined with immunohistochemistry, mice were perfused and brains were fixed overnight in 4% PFA in PBS followed by cryoprotection in 30% sucrose PBS. Then, 16&nbsp;μm or (for 3D reconstruction) 40&nbsp;μm thick brain sections were obtained using a Leica cryostat or sliding microtome. Samples were processed according to the ACDBio Multiplex Flourescent v2 Kit protocol (ACDBio #323100). Briefly, tissue was pre-treated with a series of H<sub>2</sub>O<sub>2</sub>, antigen retrieval and protease IV steps before incubation with the probe for 1.5 hours at 40°C. Probes were purchased from ACDBio: RNAscope® Probes- Mm-Gabbr1 (#425181), Mm-Gabbr2 (#317971) and a custom-made probe to specifically detect <i>Gabbr1</i> exons 7 and 8 deleted in GABA<sub>B1</sub>R cKOs (exon number is based on the exons used by both GABA<sub>B1a</sub> and GABA<sub>B1b</sub> isoforms). Three amplification steps were carried out prior to developing the signal with Cy3 fluorophore (Perkin-Elmer #NEL752001KT). Sections were then stained for GFP (chicken anti-GFP, 1:500, Aves lab #1020) or Iba1 (rabbit anti-Iba1, 1:500, Wako Chemicals #019-1974) and tdTomato (rabbit anti-DsRed, 1:500, Clontech #632496) or VGlut2 (guinea-pig anti-VGlut2 (1:2000, Millipore #AB2251). Corresponding secondary antibodies were: 488 anti-Chicken (1:500, JIR #703-545-155), 488 anti-rabbit (1:500, Thermo Scientific #A-21206), 546 anti-rabbit (1:500, Thermo Scientific #A-10040) and 594 anti-guinea pig (1:500, JIR #706-585-148). Samples were counterstained with DAPI (5&nbsp;μM, Sigma #D9542) and mounted using Prolong Gold antifade mounting medium (Molecular Probes #P369300).</div></section><section id="sec-4-4-8"><h4>Image Acquisition</h4><div id="p0325" role="paragraph">For all analyses, images were taken in layer 4 of the somatosensory cortex (or visual cortex when specified), except for SST synapse analysis which was performed in layer 1, the main output layer of SST interneurons.</div><div id="p0330" role="paragraph">For analysis of synapse density, tissue samples were imaged on an upright ZEISS LSM 800 confocal using a 40X oil immersion objective, 1.4 NA, 2.5 digital zoom, 1024&nbsp;× 1024 pixels (~0.22&nbsp;μm resolution using 510&nbsp;nm emission).</div><div id="p0335" role="paragraph">For analysis of microglia-synapse contacts and engulfment, tissue samples (layer 4 of S1 region) were imaged on an upright ZEISS LSM 800 confocal using a 40X oil immersion objective, 1.4 NA, 2.5 digital zoom, 1024&nbsp;× 1024 pixels, 0.2&nbsp;μm step size to produce confocal stacks of ~10-15&nbsp;μm. For the analysis of contacts made by <i>Gabbr2</i>+ microglia, images were acquired with no zoom at 2048&nbsp;× 2048 pixels. For the percentage of tdTomato+ PV synapses contacted or encapsulated by microglia, images were taken only in the center of the infection, where at least 95% of Syt2+ PV synapses were tdTomato+ (<span class="dropBlock reference-citations"><a id="body-ref-sref82-5" href="#" role="doc-biblioref" data-xml-rid="bib82" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref82-5" aria-controls="sref82-5">Vormstein-Schneider et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref82-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">84.</div><div>Vormstein-Schneider, D. ∙ Lin, J.D. ∙ Pelkey, K.A. ...</div><div><strong>Viral manipulation of functionally distinct interneurons in mice, non-human primates and humans</strong></div><div><em>Nat. Neurosci.</em> 2020; <strong>23</strong>:1629-1636</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41593-020-0692-9" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_84_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85089867328" target="_blank" aria-label="Open Scopus in new tab">Scopus (100)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32807948/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41593-020-0692-9&amp;pmid=32807948" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="p0340" role="paragraph">Stimulated Emission Depletion Microscopy (STED) images were acquired with a Leica SP8 Confocal/STED 3&nbsp;× microscope with an oil-immersion 100&nbsp;×, 1.44-N.A. objective, at the Harvard Neurobiology Imaging Core. For contact and engulfment analysis, infection-rich areas were first scanned in confocal mode (zoom factor 4.70, pixel size 24.2&nbsp;× 24.2&nbsp;nm, speed 400&nbsp;Hz) and confocal scans containing putative contacts or engulfed boutons were followed by sequential STED scans with gated detectors. PV synapse STED images were acquired using a similar approach but the region of interest was selected as a NeuN+ cell body. During STED scanning, Alexa Fluor 647, Alexa Fluor 546/555 and Alexa Fluor 488 signals were excited with 653-nm, 553-nm, and 499-nm white light lasers, respectively and in this particular order, and were depleted with 770&nbsp;nm, 660&nbsp;nm and 592&nbsp;nm time-gated depletion lasers.</div><div id="p0345" role="paragraph">For cell colocalization and density, tissue samples were imaged on a ZEISS Axio Imager using or on an upright ZEISS LSM 800 confocal using 10X or 20X dry objectives (with or without tiling mode).</div></section><section id="sec-4-4-9"><h4>Seizure Susceptibility</h4><div id="p0350" role="paragraph">The susceptibility to seizures was evaluated as described before with some modifications (<span class="dropBlock reference-citations"><a id="body-ref-sref83" href="#" role="doc-biblioref" data-xml-rid="bib83" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref83" aria-controls="sref83">Wamsley et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref83"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">85.</div><div>Wamsley, B. ∙ Jaglin, X.H. ∙ Favuzzi, E. ...</div><div><strong>Rbfox1 Mediates Cell-type-Specific Splicing in Cortical Interneurons</strong></div><div><em>Neuron.</em> 2018; <strong>100</strong>:846-859.e7</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.neuron.2018.09.026&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.neuron.2018.09.026&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85057123825" target="_blank" aria-label="Open Scopus in new tab">Scopus (63)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30318414/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.neuron.2018.09.026&amp;pmid=30318414" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Briefly, a 20&nbsp;mg/ml stock of kainic acid (Sigma) was made in PBS 1X. Each mouse received an intraperitoneal injection of kainic acid at a dose of 20&nbsp;mg/kg of mouse.</div></section><section id="sec-4-4-10"><h4><i>In&nbsp;Vivo</i> Two-Photon Imaging</h4><div id="p0355" role="paragraph">All <i>in&nbsp;vivo</i> two-photon imaging experiments were performed at P15-P17.</div><div id="p0360" role="paragraph">For <i>in&nbsp;vivo</i> two-photon imaging in controls, two sets of experiments were performed: <i>Cx3cr1</i><sup><i>GFP/+</i></sup> mice injected with <i>PVe-Syp-tdTomato</i> (6 mice, 3 males and 3 females) and <i>Tmem119</i><sup><i>CreER/+</i></sup><i>;Ai14</i> mice injected with <i>PVe-Syp-Gamillus</i> (6 mice, 3 males and 3 females).</div><div id="p0365" role="paragraph">For <i>in&nbsp;vivo</i> two-photon imaging in GABA<sub>B1</sub>R cKOs, three sets of experiments were performed: <i>Cx3cr1</i><sup><i>GFP/+</i></sup><i>;Tmem119</i><sup><i>CreER/+</i></sup><i>; GABA</i><sub><i>B1</i></sub><i>R</i><sup><i>fl/fl</i></sup> injected with <i>PVe-Syp-tdTomato</i> (3 mice, 1 male and 2 females), <i>Tmem119</i><sup><i>GFP</i></sup><i>; Cx3cr1</i><sup><i>Cre/+</i></sup><i>; GABA</i><sub><i>B1</i></sub><i>R</i><sup><i>fl/fl</i></sup> injected with <i>PVe-Syp-tdTomato</i> (2 mice, 2 females), <i>Tmem119</i><sup><i>CreER/+</i></sup><i>; RCE</i>; <i>GABA</i><sub><i>B1</i></sub><i>R</i><sup><i>fl/fl</i></sup> injected with <i>PVe-Syp-tdTomato</i> (1 male).</div></section><section id="sec-4-4-11"><h4>Surgery</h4><div id="p0370" role="paragraph">P15-P17 <i>Cx3cr1</i><sup><i>GFP/+</i></sup> or <i>Tmem119</i><sup><i>CreER/+</i></sup><i>;Ai14</i> mice that had been previously injected with <i>PVe-Syp-tdTomato</i> or <i>PVe-Syp-Gamillus</i> AAVs (see intracranial injections above for details) were used. On the imaging day, mice were injected with dexamethasone (2&nbsp;μg/g of body weight) to reduce brain swelling during surgery. Mice were then anesthetized with 2% isoflurane and held on a stereotaxic frame. Isoflurane concentration during surgery was kept between 1%–2% and the body temperature was maintained at 37 °C using a heating pad. The eyes were protected with a lubricant ointment (Systane) to prevent drying. Surgery was performed using the ‘no-touch’ sterile procedure, and all surgical tools were sterilized prior to surgery. The skin was shaved, the scalp was cleaned with betadine and ethanol and cut open to expose the skull covering the somatosensory cortex. A 3% hydrogen peroxide solution (Sigma) was applied to the skull and the periosteal tissue surrounding the skull was gently scraped with forceps. Using a biopsy punch (VWR) and a micro knife (Fine Science Tools), a 3-mm craniotomy was opened over the previously injected region of the primary somatosensory cortex. Care was taken not to damage the dura mater and to absolutely avoid any bleeding as it clearly affected microglia behavior and motility (in addition to imaging quality). A glass window comprised of a 5&nbsp;mm round cover glass (Warner Instruments, Harvard Bioscience, Inc.) and a 3&nbsp;mm diameter cover glass (Warner Instruments, Harvard Bioscience, Inc.) attached to it with an ultraviolet curable adhesive (NOA 71) was placed over the craniotomy and its edges were sealed with Vetbond tissue adhesive (3M). A custom-made metal head-bar (Harvard Medical School Machine Shop) was attached over the right hemisphere using an ethyl-based instant gel adhesive (Loctite 409). Dental cement (Metabond, Parkell) was used to reinforce the attachment of the head-bar to the skull and to further seal the edges of the coverslip. The animal was allowed to recover for at least 1 hour in its home cage before starting imaging.</div></section><section id="sec-4-4-12"><h4>Imaging</h4><div id="p0375" role="paragraph">Time lapse imaging of microglia and PV synapses was performed in layer 4 or deep layer 3 of the primary somatosensory cortex using a Bergamo II multiphoton microscope from ThorLabs (software ThorImageLS 3.2), equipped with a 8 kHz galvo-resonant scanner. For control <i>Cx3cr1</i><sup><i>GFP/+</i></sup><i>; PVe-Syp-tdTomato</i> experiments, the majority (64%) of the imaging was done in awake animals and a subset (36%) was performed under light anesthesia. For control <i>Tmem119</i><sup><i>CreER/+</i></sup><i>;Ai14; PVe-Syp-Gamillus</i> experiments, the majority (87.5%) of the imaging was performed under light anesthesia and a subset (12.5%) was performed in awake animals. For cKO experiments, 30% of the sessions were carried out in awake mice and the rest under light anesthesia. Mice were anesthetized with Equitisin because of the short half-life of this anesthetic mixture. A 1&nbsp;mL solution containing 0.02&nbsp;mg of magnesium sulfate, 0.32&nbsp;mL of double-distilled sterile water, 0.1&nbsp;mL of ethanol, 0.4&nbsp;mL of propylene glycol and 0.18&nbsp;mL of Nembutal was prepared and injected intraperitoneally (1.4&nbsp;μl/g) While anesthetized, body temperature was maintained at 36.5–37.5°C using a heating pad. Imaging was done using a dual path Insight X3 laser (Spectra Physics). TdTomato was excited using a λ&nbsp;= 1045&nbsp;nm fixed output while Gamillus and GFP were excited using the tunable output path tuned at λ&nbsp;= 930&nbsp;nm, and signals were collected using two GaAsP PMTs. Imaging was performed using a 16&nbsp;× /0.8 NA water-immersion objective (Nikon) at a zoom 8.2&nbsp;× . For <i>Cx3cr1</i><sup><i>GFP/+</i></sup><i>; PVe-Syp-tdTomato</i> control experiments and all cKO experiments, the field of view measured 99.22&nbsp;× 99.22 μm (1024&nbsp;× 1024 pixels) yielding a 0.097 pixel size. For <i>Tmem119</i><sup><i>CreER/+</i></sup><i>::Ai14; PVe-Syp-Gamillus</i> control experiments, the field of view measured 99.22&nbsp;× 99.22 μm (512&nbsp;× 512 pixels) yielding a 0.194 pixel size. Scan mode was set to unidirectional in averaging mode. Averaging of 5-7 frames resulted in a frame rate of 1.1 frames per seconds. The choice of the field of view was biased toward regions where we could detect some interactions between microglia and synapses prior to starting imaging. To image an entire microglia cell, z stacks of 10-15&nbsp;μm were taken using a 1-3&nbsp;μm step size using a piezo coupled to the objective (ThorLabs). Each imaging session lasted 20-30&nbsp;minutes and multiple imaging sessions were performed in one day with a mouse recovery time of at least 60&nbsp;minutes every hour. Based on the average contact duration observed in controls, all cKO imaging sessions lasted 20&nbsp;minutes.</div></section><section id="sec-4-4-13"><h4>Patch-Clamp Recordings and Optogenetics</h4><div id="p0380" role="paragraph">P15-28 mice were deeply anesthetized with isoflurane, and P60 mice were deeply anesthetized with sodium pentobarbital by intraperitoneal injection. In the case of P25-28 or P60 recordings, mice were first transcardially perfused with the slicing solution. Brains were removed and 300&nbsp;μm coronal slices were cut using a vibratome (Leica). P15-28 brains were cut with ice-cold sucrose artificial cerebrospinal fluid (ACSF) of the following composition (in mM): 87 NaCl, 26 NaHCO<sub>3</sub>, 2.5 KCl, 1.25, NaH<sub>2</sub>PO<sub>4</sub>, 0.5 CaCl, 4 MgCl<sub>2</sub>, 10 glucose, 75 sucrose, saturated with 95% O<sub>2</sub>, 5% CO<sub>2</sub> at pH 7.3-7.4. For P60 brains, the slicing solution was NMDG-ACSF (<span class="dropBlock reference-citations"><a id="body-ref-sref77-1" href="#" role="doc-biblioref" data-xml-rid="bib77" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref77-1" aria-controls="sref77-1">Ting et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref77-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">79.</div><div>Ting, J.T. ∙ Lee, B.R. ∙ Chong, P. ...</div><div><strong>Preparation of Acute Brain Slices Using an Optimized N-Methyl-D-glucamine Protective Recovery Method</strong></div><div><em>J.&nbsp;Vis. Exp.</em> 2018; 53825</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.T.TingB.R.LeeP.ChongG.Soler-LlavinaC.CobbsC.KochH.ZengE.LeinPreparation+of+Acute+Brain+Slices+Using+an+Optimized+N-Methyl-D-glucamine+Protective+Recovery+MethodJ.%C2%A0Vis.+Exp.132201853825" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) of the following composition (in mM): 92 NMDG, 2.5 KCl, 1.25 NaH<sub>2</sub>PO<sub>4</sub>, 30 NaHCO<sub>3</sub>, 20 HEPES, 25 glucose, 2 thiourea, 5 Na-ascorbate, 3 Na-pyruvate, 0.5 CaCl<sub>2</sub>, and 10 MgSO<sub>4</sub>, pH 7.3-7.4. P15-P28 slices were transferred to a heated chamber at 34 C° with oxygenated ACSF recording solution, where they underwent recovery for 30&nbsp;minutes. Slices were then moved to ACSF recording solution at room temperature, where they remained for at least an hour before recording. For P60 recordings, slices were recovered in oxygenated NMDG slicing solution at 34 C° for 25&nbsp;minutes. Na<sup>+</sup> was reintroduced (to 52mM) by gradually adding 2M NaCl-NMDG solution during recovery as previously described (<span class="dropBlock reference-citations"><a id="body-ref-sref77-2" href="#" role="doc-biblioref" data-xml-rid="bib77" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref77-2" aria-controls="sref77-2">Ting et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref77-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">79.</div><div>Ting, J.T. ∙ Lee, B.R. ∙ Chong, P. ...</div><div><strong>Preparation of Acute Brain Slices Using an Optimized N-Methyl-D-glucamine Protective Recovery Method</strong></div><div><em>J.&nbsp;Vis. Exp.</em> 2018; 53825</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.T.TingB.R.LeeP.ChongG.Soler-LlavinaC.CobbsC.KochH.ZengE.LeinPreparation+of+Acute+Brain+Slices+Using+an+Optimized+N-Methyl-D-glucamine+Protective+Recovery+MethodJ.%C2%A0Vis.+Exp.132201853825" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Slices were then transferred to HEPES-holding solution of the following composition (in mM): 92 NaCl, 2.5 KCl, 1.25 NaH<sub>2</sub>PO<sub>4</sub>, 30 NaHCO<sub>3</sub>, 20 HEPES, 25 glucose, 2 thiourea, 5 Na-ascorbate, 3 Na-pyruvate, 2 CaCl<sub>2</sub>, and 1 MgCl<sub>2</sub>, pH 7.3–7.4 at room temperature for at least an hour prior to recordings.</div><div id="p0385" role="paragraph">For recordings, slices were transferred to the recording chamber of an up-right microscope (Zeiss Axioskope II). All recordings were carried out at a constant temperature (30 C°). P15-P28 slices were perfused with ACSF of the following composition (in mM): 125 NaCl, 2.5 KCl, 1 MgCl<sub>2</sub>, 1.25 NaH<sub>2</sub>PO<sub>4</sub>, 25 NaHCO<sub>3</sub>, 20 Glucose, 2 CaCl<sub>2</sub>, saturated with 95% O<sub>2</sub>, 5% CO<sub>2</sub> at pH 7.3-7.4. For P60 recordings, the ACSF had the following composition (in mM): 124 NaCl, 2.5 KCl, 1.25 NaH<sub>2</sub>PO<sub>4</sub>, 24 NaHCO<sub>3</sub>, 12.5 glucose, 5 HEPES, 2 CaCl<sub>2</sub>, and 1 MgCl<sub>2</sub>, saturated with 95% O<sub>2</sub>, 5% CO<sub>2</sub> at pH 7.3-7.4.</div><div id="p0390" role="paragraph">Excitatory neurons were visualized with infrared-differential interference (IR-DIC) optics through a 40x water-immersion objective (Zeiss) and recorded in layer 4 or 5 of S1 barrel field. Recording pipettes were pulled from borosilicate glass (Harvard Apparatus) to obtain a tip resistance of 3-5 MΩ. For any experiment, only cells with access resistance&nbsp;&lt; 30MΩ were accepted. Access resistance was monitored throughout the recording and any cells where access resistance deteriorated and changed more than 20% were discarded. Spontaneous excitatory and inhibitory postsynaptic currents (sEPSCs and sIPSCs) or miniature excitatory and inhibitory postsynaptic currents (mEPSCs and mIPSCs) were recorded by clamping the cells at either −70&nbsp;mV to record EPSCs and at&nbsp;+0&nbsp;mV to record IPSCs. For voltage-clamp recordings from pyramidal cells, the following internal solution was used (in Mm):&nbsp;130 Cs-methanosulfonate, 5 CsCl, 10 HEPES, 0.2 EGTA, 4 MgATP, 0.3 Na-GTP, 8 Phosphocreatine-Tris, 5 QX-314-Cl, equilibrated with CsOH at pH 7.3). Data were acquired at a 20 kHz sampling rate using a MultiClamp 700B amplifier (Molecular Devices) and were filtered at 10 kHz.</div><div id="p0395" role="paragraph">Electrophysiology recordings were carried out at P15-P17 mice for all experiments measuring miniature synaptic currents (recorded with bath application of 1&nbsp;μM TTX, Tocris #1069), P58-61 for P60 experiments and P25-P28 mice (depletion P1-P28) for optogenetics. For spontaneous synaptic currents, data from P15 mice (depletion P1-P15) are shown as pool with data from P26-28 mice (depletion P1-P28), as similar results were observed in the age-specific datasets.</div></section><section id="sec-4-4-14"><h4>Optogenetics</h4><div id="p0400" role="paragraph">All experiments were conducted under wide-field photostimulation through a 40x water-immersion objective. The recorded neuron was centered in the field of view and a 470&nbsp;nm LED was triggered to deliver a square-shaped pulse of 1 or 5&nbsp;ms illumination at two different irradiance values (50% and 100% of the maximum stimulation intensity ~1&nbsp;mW/mm<sup>2</sup>). Light pulses eliciting IPSCs were delivered every 15 s. The LED output was driven using a digital output from the Clampex software of the pCLAMP 9.0 program suite (Molecular Devices) controlling a BioLED controller (Mightex). Optogenetics experiments were performed in P26-P28 (depletion P1-P28) mice to allow functional expression of Channelrhodopsin-2 under the control of the late <i>PV</i><sup><i>Cre</i></sup> promoter. Results are shown only for the 5&nbsp;ms pulse using 100% of the power, however similar results were observed for the 1&nbsp;ms pulse using 50% of the power. To test monosynapticity, TTX (1&nbsp;μM) and 4-aminopyradine (1&nbsp;mM, Tocris #0940) were applied in some of the optogenetics recordings.</div></section><section id="sec-4-4-15"><h4>Tissue dissociation and microglia isolation</h4><div id="p0405" role="paragraph">Microglia single cell suspensions for sequencing were generated as described previously with some modifications (<span class="dropBlock reference-citations"><a id="body-ref-sref31-4" href="#" role="doc-biblioref" data-xml-rid="bib31" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref31-4" aria-controls="sref31-4">Hammond et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref31-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">32.</div><div>Hammond, T.R. ∙ Dufort, C. ∙ Dissing-Olesen, L. ...</div><div><strong>Single-Cell RNA Sequencing of Microglia throughout the Mouse Lifespan and in the Injured Brain Reveals Complex Cell-State Changes</strong></div><div><em>Immunity.</em> 2019; <strong>50</strong>:253-271.e6</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.immuni.2018.11.004&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.immuni.2018.11.004&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85056738677" target="_blank" aria-label="Open Scopus in new tab">Scopus (1123)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30471926/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.immuni.2018.11.004&amp;pmid=30471926" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Centrifuges and tools were all prechilled at 4°C or on ice. P15 wild-type controls (<i>Cx3cr1</i><sup><i>+/+</i></sup><i>; GABA</i><sub><i>B1</i></sub><i>R</i><sup><i>fl/fl</i></sup>; N&nbsp;= 12), Cre-Het controls (<i>Cx3cr1</i><sup><i>Cre/+</i></sup><i>; GABA</i><sub><i>B1</i></sub><i>R</i><sup><i>+/+</i></sup>; N&nbsp;= 10) and GABA<sub>B1</sub>R cKO (<i>Cx3cr1</i><sup><i>Cre/+</i></sup><i>; GABA</i><sub><i>B1</i></sub><i>R</i><sup><i>fl/fl</i></sup>; N&nbsp;= 13) mice were transcardially perfused using ice-cold Hank’s balanced salt solution (1X HBSS, Thermo Fisher Scientific) and the brains were removed and placed on ice-cold HBSS. Coronal slices encompassing the whole somatosensory region were cut and the S1 region was quickly dissected out using a scalpel and placed in ice-cold HBSS. For each experiment, tissue from at least 4 mice/genotype was pooled to obtain a good cell yield and Dounce homogenized in ice cold HBSS 15-20 times each with the loose and tight pestles while simultaneously rotating the pestle. The cell suspension was then transferred to prechilled 50&nbsp;mL tubes and filtered using a pre-wet (with HBSS) 70&nbsp;μm cell strainer. Cell suspensions were transferred into a prechilled 15mL tube and centrifuged at 500&nbsp;g for 5&nbsp;min at 4°C. The cell pellet was resuspended in 5.5&nbsp;mL of HBSS1X. A 90% Percoll (Thermo Fisher Scientific) stock solution was made by adding 10% of HBSS 10x and 4.5&nbsp;mL of the 90% Percoll stock solution were added to the cell suspension in order to obtain a 40% isotonic Percoll solution. The mix was centrifuged at 500&nbsp;g for 1 hour at 4°C with full acceleration and braking. The top layers (myelin and Percoll) were removed by vacuum suction and the microglia pellet at the bottom of the 15mL tube was resuspended in 2&nbsp;mL of ice-cold FACS buffer (2mM EDTA in 1X PBS), transferred to 2&nbsp;mL tubes and centrifuged at 800&nbsp;g for 10&nbsp;minutes at 4°C to remove any excess of Percoll. The pellet was then resuspended in 200&nbsp;μL of FACS buffer containing 4&nbsp;μL of Mouse Fc Block (1:50, 10&nbsp;μg/ml, BD Bioscience #553141) and blocked for 15&nbsp;minutes on ice. Next, 300&nbsp;μL of ice-cold FACS buffer as well as the following antibodies were added to the single cell suspension at a 1:200 final dilution and incubated for 20-30&nbsp;minutes on ice: Cd11b-PE (BioLegend, #101208), CD45-APC-Cy7 (BioLegend, #103116), and Cx3cr1-APC (BioLegend, #149008). DAPI was also added to the mixture at a final dilution of 0.1&nbsp;μg/ml and used to distinguish alive and dead cells. Samples were then washed in 1&nbsp;mL of ice cold FACS buffer, centrifuged for 5&nbsp;minutes at 800 g, resuspended in 500&nbsp;μL of ice-cold FACS buffer containing RNase inhibitor at a concentration of 20 units/10&nbsp;μL (RNaseOUT, Thermo Fisher Scientific #10777019) and transferred to pre-coated FACS tubes. Alive (low DAPI), individual microglia exhibiting high fluorescence levels of Cd11b and Cx3cr1 but low levels of Cd45 were purified from the suspension by fluorescence-activated cell sorting (FACS) using a 100&nbsp;μm chip on a SH800S Sony sorter in purity mode. Of note, cells exhibiting high fluorescence levels of Cd45 were absent or extremely rare. Approximately 11,000-16,000 microglia were collected in each sorting experiment (experiments yielding fewer than 10,000 cells did not produce good quality libraries and were discarded). Samples were kept at 4°C before, during and after sorting.</div><div id="p0410" role="paragraph">Cells from a total of N&nbsp;= 12 wild-type controls (P15 <i>Cx3cr1</i><sup><i>+/+</i></sup><i>; GABA</i><sub><i>B1</i></sub><i>R</i><sup><i>fl/fl</i></sup>; 3 females and 9 males), N&nbsp;= 10 Cre-Het controls (P15 <i>Cx3cr1</i><sup><i>Cre/+</i></sup><i>;GABA</i><sub><i>B1</i></sub><i>R</i><sup><i>+/+</i></sup>; 8 females and 2 males) and N&nbsp;= 13 GABA<sub>B1</sub>R cKOs (P15 <i>Cx3cr1</i><sup><i>Cre/+</i></sup><i>; GABA</i><sub><i>B1</i></sub><i>R</i><sup><i>fl/fl</i></sup>; 9 females and 4 males) were used for library preparation and sequencing. In order to minimize batch effects, (1) in each experiment wild-type control and GABA<sub>B1</sub>R cKO mice from the same litter were used and (2) each experiment consisted in parallel sorting of cells from both wild-type control and GABA<sub>B1</sub>R cKO mice. Cre-Het control microglia were purified using the same experimental procedure. However, these mice were not from the same litter as control or GABA<sub>B1</sub>R cKO mice and the experiment was performed separately. Nevertheless, this additional sample exhibited similar transcriptional states as the wild-type control microglia (see cluster annotation below) and was therefore used to identify genes expression changes that could be attributed to the loss of one copy of <i>Cx3cr1</i>.</div></section><section id="sec-4-4-16"><h4>Single cell library preparation &amp; sequencing</h4><div id="p0415" role="paragraph">After sorting, collected cells were centrifuged at 800&nbsp;g for 5&nbsp;minutes at 4°C and resuspended in 50-60&nbsp;μL of PBS1x-BSA0.04% (aiming at obtaining 700 cells/μl). A small volume of cells was then diluted 1:10 and cells were counted using a hemocytometer to determine the exact number of cells/μl. Single-cell RNA-seq libraries were prepared on the 10x Genomics platform using the Chromium Single Cell 3′ Library and Gel Bead Kit v3.0 (PN-1000075), Chromium Single Cell 3′ Library Construction Kit v3.0 (PN-1000078), Chromium Chip B Single Cell kit (PN-1000154) and Chromium i7 Multiplex Kit (PN-120262) as instructed by the manufacturer. Libraries were sequenced using the Nova-Seq 100 cycle kit (Illumina) by Broad Institute Genomic Services.</div></section><section id="sec-4-4-17"><h4>Design of MERFISH encoding probes</h4><div id="p0420" role="paragraph">In order to examine the co-expression of <i>Gabbr1</i> and <i>Gabbr2</i> with microglia markers, pruning genes, genes enriched in postnatal immature microglia (scRNA-seq clusters 1-3) and genes downregulated in cKO microglia within the scRNA-seq cluster 4, a MERFISH gene panel comprising 23 genes was designed. These were: <i>Gabbr1, Gabbr2, Tmem119, Fcrls, P2ry13, P2ry12, Gpr34, Trem2, C1qc, Tmsb4x, Rps29, Ftl1, Cd164, Clec4a3, Ecscr, Laptm4a, Laptm5, Ppib, Selenok, Selenop, Tmem14c, Tspan3, Tspan4, Tspan7, mt_Nd2, mt_Co3</i>. The <i>Gabbr1</i> probe was designed to specifically avoid exons 7 and 8 deleted in GABA<sub>B1</sub>R cKOs (the exon number is based on the exons used by both GABA<sub>B1a</sub> and GABA<sub>B1b</sub> isoforms). <i>C1qc, Tmsb4x, Rps29, Ftl1, mt_Nd2, mt_Co3</i> did not pass the RNA length and expression criteria to be part of the combinatorial single-molecule FISH (MERFISH) imaging run as described in previous MERFISH work (<span class="dropBlock reference-citations"><a id="body-ref-sref14-2" href="#" role="doc-biblioref" data-xml-rid="bib14" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref14-2" aria-controls="sref14-2">Chen et&nbsp;al., 2015b</a><div class="dropBlock__holder" data-db-target-of="sref14-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">14.</div><div>Chen, K.H. ∙ Boettiger, A.N. ∙ Moffitt, J.R. ...</div><div><strong>RNA imaging. Spatially resolved, highly multiplexed RNA profiling in single cells</strong></div><div><em>Science.</em> 2015; <strong>348</strong>:aaa6090</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aaa6090" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-84928395184" target="_blank" aria-label="Open Scopus in new tab">Scopus (1274)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25858977/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aaa6090&amp;pmid=25858977" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref51-2" href="#" role="doc-biblioref" data-xml-rid="bib51" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref51-2" aria-controls="sref51-2">Moffitt et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref51-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">52.</div><div>Moffitt, J.R. ∙ Bambah-Mukku, D. ∙ Eichhorn, S.W. ...</div><div><strong>Molecular, spatial, and functional single-cell profiling of the hypothalamic preoptic region</strong></div><div><em>Science.</em> 2018; <strong>362</strong>:eaau5324</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aau5324" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85056571109" target="_blank" aria-label="Open Scopus in new tab">Scopus (576)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30385464/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aau5324&amp;pmid=30385464" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and were therefore imaged in a non-combinatorial set of sequential smFISH imaging rounds with one single gene measured in each color channel per round. Thus, our 23-plex MERFISH panel consisted of a 12-bit library for the combinatorial set (Cat #VZG114) and 6 genes imaged in the linear steps. All probes were produced by Vizgen.</div></section><section id="sec-4-4-18"><h4>Tissue sectioning, staining and imaging</h4><div id="p0425" role="paragraph">P15 control (<i>Cx3cr1</i><sup><i>+/+</i></sup><i>; GABA</i><sub><i>B1</i></sub><i>R</i><sup><i>fl/fl</i></sup>) and GABA<sub>B1</sub>R cKO (<i>Cx3cr1</i><sup><i>Cre/+</i></sup><i>; GABA</i><sub><i>B1</i></sub><i>R</i><sup><i>fl/fl</i></sup>) mice were deeply anesthetized with isoflurane and the brains were removed, fresh-frozen in Tissue-Tek O.C.T. (Electron Microscopy Sciences, #62550-01) and stored at −80°C. Functionalized coverslips (Vizgen, #FCS01) for mounting the tissue sections for MERFISH imaging experiments were coated with fiducial beads (Polysciences, #17149-10) by incubation in PBS1X for 15&nbsp;minutes and dried prior to tissue sectioning. 10&nbsp;μm brain slices from the S1 region were sectioned on a cryostat (Leica) and placed onto the coverslip by dropping the glass onto the tissue slice with the bead coated side facing the tissue. Three brain slices from the same brain were mounted on each coverslip. The mounted brain slices were allowed to adhere to the coverslip for 5&nbsp;minutes at −20°C. Slices were then incubated in fixation buffer (4% PFA in 1XPBS) for 15&nbsp;minutes at room temperature and washed 3 times for 5&nbsp;minutes with 1X PBS. Slices were incubated in 70% ethanol for 5&nbsp;minutes and stored in ethanol at 4°C for up to a week prior to staining. MERFISH staining was carried out as instructed by the manufacturer (Vizgen). Briefly, coverslips with brain slices were washed in 2XSSC and incubated first in formamide wash buffer for 30&nbsp;minutes at 37°C and then in ready-to-use Encoding Probe Hybridization Buffer Mix for 36-48 hours at 37°C. After a series of washes in 2XSSC and formamide wash buffer at 47°C, samples were gel embedded at room temperature for 1.5 hours. Next, tissue was cleared by adding proteinase K supplemented clearing solution to each sample and incubating it for 48h at 37°C. After a series of washes in 2XSSC, samples were incubated in hybridization buffer 1 from the Vizgen Imaging Reagent Kit (Vizgen, #IK-18) containing fluorescent probes for the first round of imaging (i.e., the first 3 bits of the codebook). Coverslips were then assembled into the imaging chamber and the bottom of the coverslip was cleaned with 100% methanol to ensure the optical imaging surface was entirely clean. MERFISH imaging of 11 brain slices from 2 controls (1 male and 1 female) and 8 slices from 2 cKOs (1 male and 1 female) was performed on an automated Vizgen Alpha Instrument with parameter files provided by the company and available at <a id="intref0310" href="https://github.com/emiliafavuzzi/Favuzzi_et_al_2021/tree/main/MERFISH" target="_blank">https://github.com/emiliafavuzzi/Favuzzi_et_al_2021/tree/main/MERFISH</a>. Stacks of seven images spaced by 1.5&nbsp;μm in Z and comprising the whole S1 region were acquired with a Nikon 60X/1.4NA objective. Images were acquired in five colors, three encoding readout probes from the library, one serving as a fiducial marker for registration of the images during analysis and a nuclear staining (DAPI) to help downstream segmentation.</div></section><section id="sec-4-4-19"><h4>Motion Sequencing (MoSeq)</h4><div id="p0430" role="paragraph">MoSeq experiments were carried out as described previously (<span class="dropBlock reference-citations"><a id="body-ref-sref86-3" href="#" role="doc-biblioref" data-xml-rid="bib86" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref86-3" aria-controls="sref86-3">Wiltschko et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref86-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">88.</div><div>Wiltschko, A.B. ∙ Johnson, M.J. ∙ Iurilli, G. ...</div><div><strong>Mapping Sub-Second Structure in Mouse Behavior</strong></div><div><em>Neuron.</em> 2015; <strong>88</strong>:1121-1135</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.neuron.2015.11.031&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.neuron.2015.11.031&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-84957565912" target="_blank" aria-label="Open Scopus in new tab">Scopus (373)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26687221/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.neuron.2015.11.031&amp;pmid=26687221" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref46-2" href="#" role="doc-biblioref" data-xml-rid="bib46" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref46-2" aria-controls="sref46-2">Markowitz et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref46-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">47.</div><div>Markowitz, J.E. ∙ Gillis, W.F. ∙ Beron, C.C. ...</div><div><strong>The Striatum Organizes 3D Behavior via Moment-to-Moment Action Selection</strong></div><div><em>Cell.</em> 2018; <strong>174</strong>:44-58.e17</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.cell.2018.04.019&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.cell.2018.04.019&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85046871902" target="_blank" aria-label="Open Scopus in new tab">Scopus (193)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29779950/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2018.04.019&amp;pmid=29779950" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Briefly, P30 or P60 control, Cre-Het or GABA<sub>B1</sub>R cKO mice were placed in a circular open field in the dark (red light) and their 3D pose dynamics were measured during 60&nbsp;minutes at 30&nbsp;Hz through the use of a depth camera (Kinect 2, Microsoft). All mice were tested in the afternoon (i.e., 4-11 pm), when more active. For P60 MoSeq experiments, a total of N&nbsp;= 19 wild-type controls (<i>Cx3cr1</i><sup><i>+/+</i></sup><i>; GABA</i><sub><i>B1</i></sub><i>R</i><sup><i>fl/fl</i></sup>; 9 females and 10 males), N&nbsp;= 8 Cre-Het controls (<i>Cx3cr1</i><sup><i>Cre/+</i></sup><i>; GABA</i><sub><i>B1</i></sub><i>R</i><sup><i>+/+</i></sup>; 3 females and 5 males), N&nbsp;= 14 GABA<sub>B1</sub>R cKOs (<i>Cx3cr1</i><sup><i>Cre/+</i></sup><i>; GABA</i><sub><i>B1</i></sub><i>R</i><sup><i>fl/fl</i></sup>; 9 females and 5 males) and N&nbsp;= 6 Tmem119-GABA<sub>B1</sub>R cKOs (<i>Tmem119</i><sup><i>CreER/+</i></sup><i>; GABA</i><sub><i>B1</i></sub><i>R</i><sup><i>fl/fl</i></sup>; 3 females and 3 males) were tested. For P30 MoSeq experiments, a total of N&nbsp;= 17 wild-type controls (<i>Cx3cr1</i><sup><i>+/+</i></sup><i>; GABA</i><sub><i>B1</i></sub><i>R</i><sup><i>fl/fl</i></sup>; 11 females and 6 males), N&nbsp;=&nbsp;12 Cre-Het controls (<i>Cx3cr1</i><sup><i>Cre/+</i></sup><i>; GABA</i><sub><i>B1</i></sub><i>R</i><sup><i>+/+</i></sup>; 5 females and 7 males) and N&nbsp;= 29 GABA<sub>B1</sub>R cKOs (<i>Cx3cr1</i><sup><i>Cre/+</i></sup><i>; GABA</i><sub><i>B1</i></sub><i>R</i><sup><i>fl/fl</i></sup>; 15 females and 14 males) were tested.</div></section></section><section id="sec-4-5"><h3>Quantification and statistical analysis</h3><div id="p0435" role="paragraph">All of the statistical details of experiments can be found in the figure legends which include the statistical tests used, exact value of n, exclusion of any data, and what n represents.</div><div id="p0440" role="paragraph">All statistical analyses were performed using GraphPad Prism or R software. Unless otherwise stated, parametric data were analyzed by t test or one-way ANOVA followed by Holm-Sidak or Tukey post hoc analysis for comparisons of multiple samples. Non-parametric data were analyzed by the Mann-Whitney rank sum test or Kruskal-Wallis one-way analysis of variance on ranks followed by Holm-Sidak or Dunn post hoc analysis for comparisons of multiple samples. P values&nbsp;&lt; 0.05 were considered statistically significant.</div><section id="sec-4-5-1"><h4>Quantifications of Cell and Synapse Density</h4><div id="p0445" role="paragraph">The quantification of synapse density during development is highly influenced by technical (e.g., perfusion and fixation, temperature during sectioning, antibody batch, etc.) as well as biological (e.g., maternal care, exact time of birth, etc.) factors. To minimize variability, littermates were used in all experiments involving a comparison in synapse density between conditions. Moreover, brains from the two conditions were strictly processed, imaged and analyzed together. When mice from multiple litters were pooled, they were collected simultaneously or within a short period of time (less than a month) and each collection always included both experimental and control groups. To confirm all main phenotypes, analyses were repeated and performed blindly.</div></section><section id="sec-4-5-2"><h4>Synapse density, contacts &amp; engulfment</h4><div id="p0450" role="paragraph">For PV and excitatory synapse analysis, single planes were analyzed using a custom script in Fiji (ImageJ) (<span class="dropBlock reference-citations"><a id="body-ref-sref65-2" href="#" role="doc-biblioref" data-xml-rid="bib65" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref65-2" aria-controls="sref65-2">Schneider et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref65-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">67.</div><div>Schneider, C.A. ∙ Rasband, W.S. ∙ Eliceiri, K.W.</div><div><strong>NIH Image to ImageJ: 25 years of image analysis</strong></div><div><em>Nat. Methods.</em> 2012; <strong>9</strong>:671-675</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.2089" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_67_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-84863205849" target="_blank" aria-label="Open Scopus in new tab">Scopus (42378)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22930834/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.2089&amp;pmid=22930834" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) software as described before (<span class="dropBlock reference-citations"><a id="body-ref-sref23" href="#" role="doc-biblioref" data-xml-rid="bib23" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref23" aria-controls="sref23">Favuzzi et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref23"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">24.</div><div>Favuzzi, E. ∙ Marques-Smith, A. ∙ Deogracias, R. ...</div><div><strong>Activity-Dependent Gating of Parvalbumin Interneuron Function by the Perineuronal Net Protein Brevican</strong></div><div><em>Neuron.</em> 2017; <strong>95</strong>:639-655.e10</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.neuron.2017.06.028&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.neuron.2017.06.028&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85023623383" target="_blank" aria-label="Open Scopus in new tab">Scopus (224)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28712654/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.neuron.2017.06.028&amp;pmid=28712654" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Briefly, processing of all channels included noise reduction and smoothing. All single channel images were converted to RGB. Next, a color threshold was automatically set to identify the cell body (or dendrite), its perimeter was automatically measured and a masked binary image with the cell body (or dendrite) only was created. For bouton segmentation, a watershed-based method was used such that boutons were separated based on the local minima of the pixel gray values. For the presynaptic boutons (Syt2, VGlut1 or VGlut2) or postsynaptic clusters (Gephyrin or Homer1), a color threshold was selected to segment boutons as isolated puncta (i.e., low enough to allow local minima detection and background exclusion but high enough to detect all putative presynaptic boutons or postsynaptic clusters). The comparison between the original images and the masks was used to guide the choice of the threshold value. The same criteria for each channel were applied to all images from the same experiment. The area of each presynaptic bouton was 0.2-0.8&nbsp;μm<sup>2</sup> whereas postsynaptic clusters measured 0.1-0.3&nbsp;μm<sup>2</sup>. The “Analyze Particles” (where the minimum size for presynaptic and postsynaptic structures was 0.20&nbsp;μm and 0.10μm, respectively) and “Watershed” tools were applied and a mask was generated. A merged image from all masks was created, converted to an 8-bit image and, using an automatic threshold, the overlap between presynaptic boutons, postsynaptic clusters and cell body was automatically detected as particles with a size greater than 0.05&nbsp;μm<sup>2</sup> in the “analyze particles” tool (corresponding to at least half of the smallest postsynaptic cluster overlapping with both presynaptic bouton and soma). Sample scripts for such analyses can be found at <a id="intref0315" href="https://github.com/emiliafavuzzi/synaptic-analyses" target="_blank">https://github.com/emiliafavuzzi/synaptic-analyses</a> <a id="intref0320" href="https://doi.org/10.5281/zenodo.4899956">https://doi.org/10.5281/zenodo.4899956</a>. A video showing an example of the workflow for synapse analysis can be found at <a id="intref0325" href="https://www.youtube.com/watch?v=2WHMuz5oMAM&amp;feature=youtu.be" target="_blank">https://www.youtube.com/watch?v=2WHMuz5oMAM&amp;feature=youtu.be</a>.</div><div id="p0455" role="paragraph">For C1q accumulation, the analysis was carried out as described above and the minimum size for the colocalization between C1q puncta and presynaptic boutons was 0.20&nbsp;μm.</div><div id="p0460" role="paragraph">For the analysis of microglia-synapse contacts and engulfment, confocal stacks (~10-15&nbsp;μm) were analyzed with IMARIS 9.3.1 or 9.5.0 software using a MATLAB script to automatize the analysis (an example can be found at <a id="intref0330" href="https://github.com/emiliafavuzzi/synaptic-analyses" target="_blank">https://github.com/emiliafavuzzi/synaptic-analyses</a>) as described before (<span class="dropBlock reference-citations"><a id="body-ref-sref24-2" href="#" role="doc-biblioref" data-xml-rid="bib24" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref24-2" aria-controls="sref24-2">Favuzzi et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref24-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">25.</div><div>Favuzzi, E. ∙ Deogracias, R. ∙ Marques-Smith, A. ...</div><div><strong>Distinct molecular programs regulate synapse specificity in cortical inhibitory circuits</strong></div><div><em>Science.</em> 2019; <strong>363</strong>:413-417</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aau8977" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85060511221" target="_blank" aria-label="Open Scopus in new tab">Scopus (110)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30679375/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aau8977&amp;pmid=30679375" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). All channels were subjected to background subtraction and Gaussian filtering. Then, three-dimensional isosurfaces were created for GFP+ microglia and volume was quantified automatically. A threshold was selected to include as much of the processes as possible while excluding any background and a 0.01&nbsp;μm<sup>3</sup> size filter was applied. The automatic threshold calculated in Imaris based on k-means statistical methods was used in the majority of analyses. Then, presynaptic tdTomato+ PV boutons were reconstructed as “spots” of 0.8&nbsp;μm diameter (corresponding to the largest measured size of presynaptic boutons) and their total number was automatically calculated. Briefly, the built-in spot detection algorithm in Imaris first applies a 3D Mexican Hat filter using the spot size and then locates the spot centroid at the local maxima of the filtered image. Next, the number of spots located at no more than 0.4&nbsp;μm from the microglia surface was automatically determined and indicated as contact. The 0.4&nbsp;μm distance is calculated from the center of mass of the reconstructed “spot” and corresponds to its radius thus identifying as contact only those spots that are completely juxtaposed to the surface. In the representative images only contacted boutons are shown as reconstructed spots.</div><div id="p0465" role="paragraph">For the analysis of contacts made by <i>Gabbr2</i>+ microglia, the surface of all cells was reconstructed. Using the slice view, individual microglia were identified as <i>Gabbr2</i>-positive or <i>Gabbr2</i>-negative in single confocal planes. The surface corresponding only to the cell of interest was then duplicated and spots were reconstructed only within the region of interest containing the selected cell. Contact analysis was then carried out as described above.</div><div id="p0470" role="paragraph">Fluorescence quenching as well as protein degradation by lysosomal proteases may affect the detection of Syp-tdTomato (pKa 4.7) once inside microglial lysosomes (<span class="dropBlock reference-citations"><a id="body-ref-sref34-2" href="#" role="doc-biblioref" data-xml-rid="bib34" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref34-2" aria-controls="sref34-2">Katayama et&nbsp;al., 2008</a><div class="dropBlock__holder" data-db-target-of="sref34-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">35.</div><div>Katayama, H. ∙ Yamamoto, A. ∙ Mizushima, N. ...</div><div><strong>GFP-like proteins stably accumulate in lysosomes</strong></div><div><em>Cell Struct. Funct.</em> 2008; <strong>33</strong>:1-12</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1247/csf.07011" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-44949131031" target="_blank" aria-label="Open Scopus in new tab">Scopus (179)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/18256512/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1247%2Fcsf.07011&amp;pmid=18256512" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref67-2" href="#" role="doc-biblioref" data-xml-rid="bib67" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref67-2" aria-controls="sref67-2">Shinoda et&nbsp;al., 2018b</a><div class="dropBlock__holder" data-db-target-of="sref67-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">69.</div><div>Shinoda, H. ∙ Shannon, M. ∙ Nagai, T.</div><div><strong>Fluorescent Proteins for Investigating Biological Events in Acidic Environments</strong></div><div><em>Int. J. Mol. Sci.</em> 2018; <strong>19</strong>:1548</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.3390/ijms19061548" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85047554713" target="_blank" aria-label="Open Scopus in new tab">Scopus (67)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29789517/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.3390%2Fijms19061548&amp;pmid=29789517" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). For this reason, instead of Syp-tdTomato, the acid-tolerant monomeric green fluorescent protein Gamillus was used for the encapsulation and engulfment analysis. Gamillus exhibits fluorescence stability at acidic pH (pKa 3.4) such as that of the lysosomes (pH ~4.5–5.0) as well as a proven resistance to proteolytic degradation in lysosomes (<span class="dropBlock reference-citations"><a id="body-ref-sref34-3" href="#" role="doc-biblioref" data-xml-rid="bib34" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref34-3" aria-controls="sref34-3">Katayama et&nbsp;al., 2008</a><div class="dropBlock__holder" data-db-target-of="sref34-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">35.</div><div>Katayama, H. ∙ Yamamoto, A. ∙ Mizushima, N. ...</div><div><strong>GFP-like proteins stably accumulate in lysosomes</strong></div><div><em>Cell Struct. Funct.</em> 2008; <strong>33</strong>:1-12</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1247/csf.07011" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-44949131031" target="_blank" aria-label="Open Scopus in new tab">Scopus (179)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/18256512/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1247%2Fcsf.07011&amp;pmid=18256512" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref67-3" href="#" role="doc-biblioref" data-xml-rid="bib67" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref67-3" aria-controls="sref67-3">Shinoda et&nbsp;al., 2018b</a><div class="dropBlock__holder" data-db-target-of="sref67-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">69.</div><div>Shinoda, H. ∙ Shannon, M. ∙ Nagai, T.</div><div><strong>Fluorescent Proteins for Investigating Biological Events in Acidic Environments</strong></div><div><em>Int. J. Mol. Sci.</em> 2018; <strong>19</strong>:1548</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.3390/ijms19061548" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85047554713" target="_blank" aria-label="Open Scopus in new tab">Scopus (67)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29789517/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.3390%2Fijms19061548&amp;pmid=29789517" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). In particular, <i>PVe-Syp-Gamillus</i> was used to label PV boutons and <i>Tmem119</i><sup><i>CreER/+</i></sup><i>;Ai14</i> mice (upon tamoxifen injection) were used to label microglia. For the encapsulation analysis, reconstruction was performed as described above and the “Split spots into surface” tool was used. The number of PV spots encapsulated inside microglia was then automatically quantified. For the engulfment analysis, three-dimensional isosurfaces were created for the lysosome marker CD68 inside tdTomato+ microglia. The subset of Gamillus+ PV boutons that had been detected as encapsulated within microglia was further subjected to the “Split spots into surface” tool using microglial CD68 as surface and the number of PV spots embedded was automatically determined.</div><div id="p0475" role="paragraph">For SST synapses using <i>SST</i><sup><i>Cre</i></sup><i>;Ai34</i> mice, the analysis was performed in Fiji as described above except for the absence of a cell body. Synapses were analyzed in layer 1 and normalized by area.</div><div id="p0480" role="paragraph">For SST synapses in cKOs, since a Cre driver was already included for <i>Gabbr1</i> genetic deletion, we used a four-allele genetic strategy to delete <i>Gabbr1</i> in microglia while also labeling SST axons (<i>Cx3cr1</i><sup><i>Cre/+</i></sup><i>; GABA</i><sub><i>B1</i></sub><i>R</i><sup><i>fl/fl</i></sup>; <i>SST</i><sup><i>Flp/+</i></sup><i>;RCE:FRT</i>) and performed the analysis as described before (<span class="dropBlock reference-citations"><a id="body-ref-sref24-3" href="#" role="doc-biblioref" data-xml-rid="bib24" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref24-3" aria-controls="sref24-3">Favuzzi et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref24-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">25.</div><div>Favuzzi, E. ∙ Deogracias, R. ∙ Marques-Smith, A. ...</div><div><strong>Distinct molecular programs regulate synapse specificity in cortical inhibitory circuits</strong></div><div><em>Science.</em> 2019; <strong>363</strong>:413-417</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aau8977" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85060511221" target="_blank" aria-label="Open Scopus in new tab">Scopus (110)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30679375/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aau8977&amp;pmid=30679375" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Briefly, 0.2&nbsp;μm step size confocal stacks (~5-10&nbsp;μm) were analyzed using IMARIS 9.3.1 or 9.5.0 and a MATLAB script to automatize the analysis. Briefly, SST axons (genetically labeled using <i>SST</i><sup><i>Flp/+</i></sup><i>;RCE:FRT</i> mice) were reconstructed in 3D and GAD65+ presynaptic boutons contained within the axon were reconstructed as spots (size 0.6&nbsp;μm) and selected as SST pre-synapses using the “split into surface” tool. Then, the number of GAD65+ presynaptic boutons within the axon colocalizing with Gephyrin+ postsynaptic clusters (size 0.3&nbsp;μm) was automatically obtained and normalized per axon volume. The threshold for colocalization was 0.45&nbsp;μm, corresponding to the sum of the radius of each spot and indicating that pre- and post- synapses are identified as such only when juxtaposed.</div></section><section id="sec-4-5-3"><h4>Cell colocalization and density</h4><div id="p0485" role="paragraph">PV and microglia cell density was analyzed using a custom script in Fiji (ImageJ) software (an example can be found at <a id="intref0335" href="https://github.com/emiliafavuzzi/synaptic-analyses" target="_blank">https://github.com/emiliafavuzzi/synaptic-analyses</a>). Briefly, a region of interest corresponding to the barrel field of the somatosensory cortex (or individual layers within it) was selected and the area was measured. Next, using an automatic threshold and a size filter of at least 10&nbsp;μm in the “analyze particle” tool the total number of cells in the region was automatically determined. Due to the small soma size and highly ramified nature of microglia, microglia cell density was analyzed manually. Briefly, a region of interest corresponding to the barrel field of the somatosensory cortex was selected and the area was measured. Next, cells within this area were counted using the cell counter plugin in ImageJ (created by Kurt De Vos). For the colocalization of microglia expressing GABA<sub>B</sub> receptor mRNAs or protein, confocal stacks (~10-15&nbsp;μm) were analyzed manually. Analysis was performed in individual confocal planes using the cell counter plugin. For a cell to be considered <i>Gabbr1</i>+ or <i>Gabbr2</i>+, mRNA clusters had to be present in at least three different focal planes. For the colocalization of microglia with GABA<sub>B1</sub>R and GABA<sub>B2</sub>R proteins, the signal in GABA<sub>B1</sub>R cKOs was used as baseline to determine background. Note that a reduction in GABA<sub>B2</sub>R protein has also been reported in GABA<sub>B1</sub>R cKOs (<span class="dropBlock reference-citations"><a id="body-ref-sref30-3" href="#" role="doc-biblioref" data-xml-rid="bib30" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref30-3" aria-controls="sref30-3">Haller et&nbsp;al., 2004</a><div class="dropBlock__holder" data-db-target-of="sref30-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">31.</div><div>Haller, C. ∙ Casanova, E. ∙ Müller, M. ...</div><div><strong>Floxed allele for conditional inactivation of the GABAB(1) gene</strong></div><div><em>Genesis.</em> 2004; <strong>40</strong>:125-130</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/gene.20073" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-10344244007" target="_blank" aria-label="Open Scopus in new tab">Scopus (45)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15493018/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fgene.20073&amp;pmid=15493018" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Using GABA<sub>B1</sub>R cKOs, we determined that clusters having an area higher than 0.15&nbsp;μm were bona fide GABA<sub>B</sub>Rs. Images were analyzed using the same procedure described above for the synaptic analysis and a mask for each channel was created. Then, microglia expressing GABA<sub>B1</sub>R and GABA<sub>B2</sub>R proteins were identified manually in individual confocal planes using the cell counter plugin. Note that, due to the high antibody background, smFISH should be the preferred method for identification of GABA-receptive cells.</div></section><section id="sec-4-5-4"><h4>Seizure Susceptibility</h4><div id="p0490" role="paragraph">A total of N&nbsp;= 7 controls (<i>Cx3cr1</i><sup><i>+/+</i></sup><i>; GABA</i><sub><i>B1</i></sub><i>R</i><sup><i>fl/fl</i></sup>; 1 P16, 3 P20 and 3 P26; 3 females and 4 males) and N&nbsp;= 7 GABA<sub>B1</sub>R cKOs (<i>Cx3cr1</i><sup><i>Cre/+</i></sup><i>; GABA</i><sub><i>B1</i></sub><i>R</i><sup><i>fl/fl</i></sup>; 3 P16, 2 P20 and 2 P26; 3 females and 4 males) was used. Since no significant difference or trend was observed in the susceptibility to seizures based on age or sex, data from all mice of the same genotype were pooled. Immediately after the kainic acid injection, seizure susceptibility was investigated according to a modified Racine Scale (<span class="dropBlock reference-citations"><a id="body-ref-sref28" href="#" role="doc-biblioref" data-xml-rid="bib28" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref28" aria-controls="sref28">Gehman et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref28"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">29.</div><div>Gehman, L.T. ∙ Stoilov, P. ∙ Maguire, J. ...</div><div><strong>The splicing regulator Rbfox1 (A2BP1) controls neuronal excitation in the mammalian brain</strong></div><div><em>Nat. Genet.</em> 2011; <strong>43</strong>:706-711</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng.841" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-79959726422" target="_blank" aria-label="Open Scopus in new tab">Scopus (259)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21623373/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng.841&amp;pmid=21623373" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref60" href="#" role="doc-biblioref" data-xml-rid="bib60" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref60" aria-controls="sref60">Racine, 1972</a><div class="dropBlock__holder" data-db-target-of="sref60"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">62.</div><div>Racine, R.J.</div><div><strong>Modification of seizure activity by electrical stimulation. II. Motor seizure</strong></div><div><em>Electroencephalogr. Clin. Neurophysiol.</em> 1972; <strong>32</strong>:281-294</div></div><div class="external-links"><div class="core-xlink-abstract"><a href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2F0013-4694%2872%2990177-0&amp;cf=abstract" target="_blank" aria-label="Open Abstract in new tab">Abstract</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2F0013-4694%2872%2990177-0&amp;cf=pdf" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-0015309163" target="_blank" aria-label="Open Scopus in new tab">Scopus (6039)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/4110397/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2F0013-4694%2872%2990177-0&amp;pmid=4110397" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>): stage 0, normal behavior; stage 1, immobility; stage 2, mouth and facial movements; stage 3, head bobbing; stage 4, forelimb clonus; stage 5, rearing; stage 6, continuous rearing and falling (tonic-clonic seizures); stage 7, status epilepticus and/or death. Each animal was continuously inspected for up to 90&nbsp;minutes from the time of kainic acid injection. The maximum score of each animal’s behavior every 5&nbsp;minutes was used to determine the average score and standard deviation for both control and GABA<sub>B1</sub>R cKO mice. Of note, two-way ANOVA for repeated-measures revealed a significant effect of time but no significant effect of the genotype factor.</div></section><section id="sec-4-5-5"><h4><i>In&nbsp;Vivo</i> Two-Photon Imaging</h4><div id="p0495" role="paragraph">To correct for motion artifacts, image registration was carried out in ImageJ using a script created by Albert Cardona and Robert Bryson-Richardson (<span class="dropBlock reference-citations"><a id="body-ref-sref56" href="#" role="doc-biblioref" data-xml-rid="bib56" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref56" aria-controls="sref56">Parslow et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref56"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">58.</div><div>Parslow, A. ∙ Cardona, A. ∙ Bryson-Richardson, R.J.</div><div><strong>Sample Drift Correction Following 4D Confocal Time-lapse Imaging</strong></div><div><em>J.&nbsp;Vis. Exp.</em> 2014; 51086</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.ParslowA.CardonaR.J.Bryson-RichardsonSample+Drift+Correction+Following+4D+Confocal+Time-lapse+ImagingJ.%C2%A0Vis.+Exp.86201451086" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). To facilitate data analysis, four-dimensional data were projected into one plane using a standard maximum projection procedure, contrast was enhanced (saturation 0.35, normalize all) and a noise reduction step was performed using an edge-preserving Kuwahara-type filter developed by Wayne Rasband with a radius of 5. Contacts between microglia and presynaptic PV boutons were defined as an overlap between tdTomato+ PV boutons and GFP+ microglia (or Gamillus+ PV boutons and tdTomato+ microglia) fluorescence signals higher or equal to 30% of the synaptic bouton surface that lasted for more than 1&nbsp;minute. Each contact was verified in at least one individual z plane for the time frame at which the corresponding contact started. For the quantification of the percentage of PV boutons contacted by one cell, the total number of PV boutons in a 20&nbsp;μm radius around the microglia cell body was counted; next, the number of PV boutons contacted by microglia over a total of 20&nbsp;minutes was calculated and expressed as a fraction of the total number of PV boutons in the area. For the quantification of the contact duration, each contact was identified and labeled, and the contact start as well as end time were recorded. tdTomato signal (for PV boutons or microglia) exhibited significant bleaching over time, preventing imaging sessions of 30-60&nbsp;minutes to be included in the analysis. For the quantification of the percentage of PV boutons contacted by one cell, synapses or microglia processes that underwent significant bleaching over the 20-30&nbsp;minute imaging session were still considered if contacted by a microglia while still visible. However, such synapses were not considered for the contact duration analysis as it was not possible to determine the end of the contact (because of bleaching or, occasionally, loss of focus). Conversely, the contact duration analysis included both synapses detected in a 20&nbsp;μm radius around the microglia cell body and synapses contacted by microglia processes that did not have a cell body in the field of view. Of note, for <i>Cx3cr1</i><sup><i>GFP/+</i></sup><i>; PVe-Syp-tdTomato</i> experiments, occasional bleed-through of the GFP signal in the tdTomato channel was observed; however, in most of these cases, the GFP signal detected in the tdTomato channel was observed in microglia cell bodies while contacts were detected between boutons and microglia processes. cKO experiments using <i>Cx3cr1</i><sup><i>GFP/+</i></sup><i>;Tmem119</i><sup><i>CreER/+</i></sup><i>; GABA</i><sub><i>B1</i></sub><i>R</i><sup><i>fl/fl</i></sup> mice were analyzed as described above. In the cKO experiments using <i>Tmem119</i><sup><i>GFP</i></sup><i>; Cx3cr1</i><sup><i>Cre/+</i></sup><i>; GABA</i><sub><i>B1</i></sub><i>R</i><sup><i>fl/fl</i></sup> or <i>Tmem119</i><sup><i>CreER/+</i></sup><i>; RCE</i>; <i>GABA</i><sub><i>B1</i></sub><i>R</i><sup><i>fl/fl</i></sup> mice, the dimmer GFP signal required a higher laser power which caused extensive bleaching of the <i>PVe-Syp-tdTomato</i> boutons over time. This is why, when using those reporters, the analysis was carried out using only the first frames for PV boutons (static) and all imaging sessions where motion was not entirely corrected by registration were removed. Since a similar percentage of interactions was observed in these two sets, data were pooled. However, contact duration was significantly lower in the experiments using <i>Tmem119</i><sup><i>GFP</i></sup><i>; Cx3cr1</i><sup><i>Cre/+</i></sup><i>; GABA</i><sub><i>B1</i></sub><i>R</i><sup><i>fl/fl</i></sup> or <i>Tmem119</i><sup><i>CreER/+</i></sup><i>; RCE</i>; <i>GABA</i><sub><i>B1</i></sub><i>R</i><sup><i>fl/fl</i></sup> mice and therefore only data from <i>Cx3cr1</i><sup><i>GFP/+</i></sup><i>;Tmem119</i><sup><i>CreER/+</i></sup><i>; GABA</i><sub><i>B1</i></sub><i>R</i><sup><i>fl/fl</i></sup> mice were included.</div><div id="p0500" role="paragraph">For <i>Cx3cr1</i><sup><i>GFP/+</i></sup><i>; PVe-Syp-tdTomato</i> control experiments, a total of 51 microglia cells and 222 contacts were analyzed from 6 P15-P17 mice (3 males and 3 females). For <i>Tmem119</i><sup><i>CreER/+</i></sup><i>::Ai14; PVe-Syp-Gamillus</i> control experiments, a total of 37 microglia cells and 115 contacts were analyzed from 6 P15-P17 mice (3 males and 3 females). For <i>Cx3cr1</i><sup><i>GFP/+</i></sup><i>;Tmem119</i><sup><i>CreER/+</i></sup><i>; GABA</i><sub><i>B1</i></sub><i>R</i><sup><i>fl/fl</i></sup> cKO experiments a total of 37 microglia cells and 99 contacts were analyzed from 3 P15-P17 mice (1 male and 2 females). For <i>Cx3cr1</i><sup><i>GFP/+</i></sup><i>;Tmem119</i><sup><i>CreER/+</i></sup><i>; GABA</i><sub><i>B1</i></sub><i>R</i><sup><i>fl/fl</i></sup> and <i>Tmem119</i><sup><i>CreER/+</i></sup><i>; RCE</i>; <i>GABA</i><sub><i>B1</i></sub><i>R</i><sup><i>fl/fl</i></sup> cKO experiments a total of 25 microglia cells and 85 contacts were analyzed from 3 P15-P17 mice (1 male and 2 females).</div><div id="p0505" role="paragraph">Data were visualized using the “ggplot2” R package. The peaks of the probability density function were found using the following coding lines in R: d&nbsp;&lt; - density(x) and d$x[c(F, diff(sign(diff(d$y)))&nbsp;&lt; 0)] where x was the vector containing all observations. The two populations of microglia contacting few or most boutons where split based on the local minimum of the Kernel density estimation identified using the following coding line in R (the interval was selected within the two peaks): optimize(approxfun(d$x,d$y), interval&nbsp;= c(12,65))$minimum. Data from L4 and L3 were pooled as no significant differences were detected in the parameters analyzed. As previously reported (<span class="dropBlock reference-citations"><a id="body-ref-sref39" href="#" role="doc-biblioref" data-xml-rid="bib39" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref39" aria-controls="sref39">Liu et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref39"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">40.</div><div>Liu, Y.U. ∙ Ying, Y. ∙ Li, Y. ...</div><div><strong>Neuronal network activity controls microglial process surveillance in awake mice via norepinephrine signaling</strong></div><div><em>Nat. Neurosci.</em> 2019; <strong>22</strong>:1771-1781</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41593-019-0511-3" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_40_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85074203384" target="_blank" aria-label="Open Scopus in new tab">Scopus (201)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31636449/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41593-019-0511-3&amp;pmid=31636449" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref73" href="#" role="doc-biblioref" data-xml-rid="bib73" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref73" aria-controls="sref73">Stowell et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref73"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">75.</div><div>Stowell, R.D. ∙ Sipe, G.O. ∙ Dawes, R.P. ...</div><div><strong>Noradrenergic signaling in the wakeful state inhibits microglial surveillance and synaptic plasticity in the mouse visual cortex</strong></div><div><em>Nat. Neurosci.</em> 2019; <strong>22</strong>:1782-1792</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41593-019-0514-0" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_75_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85074208801" target="_blank" aria-label="Open Scopus in new tab">Scopus (179)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31636451/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41593-019-0514-0&amp;pmid=31636451" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), microglia appeared more active in anesthetized versus awake mice. Under light anesthesia, microglia exhibited a trend for contacting more PV boutons (although the difference did not reach significance). Moreover, the interactions had a small but significant duration increase in anesthetized versus awake mice. However, since similar conclusions could be reached in both the awake and anesthetized datasets with respect to microglia-PV bouton interactions (e.g., compare the <i>PVe-Syp-tdTomato</i> with <i>PVe-Syp-Gamillus</i> experiments) data from anesthetized versus awake imaging were pooled.</div></section><section id="sec-4-5-6"><h4>Patch-Clamp Recordings</h4><div id="p0510" role="paragraph">Miniature synaptic currents were analyzed using the Minianalysis software (Synaptosoft, Decatur, GA) except for mEPSCs/mIPSCs from <i>Cx3cr1</i><sup><i>Cre</i></sup> P15 cKO experiment and sEPSC/sIPSC experiments where a threshold search in Clampfit (Molecular Devices) was used. For the analysis using Minianalysis, the detection parameters were: Threshold 6.72&nbsp;pA; local maximum period 8000&nbsp;μs; baseline period 4000&nbsp;μs; decay time period 8000&nbsp;μs; decay time fraction 0.32; baseline average period 1000&nbsp;μs; area threshold 11&nbsp;pA; peak average points 1. For the analysis using Clampfit, a detection parameter of 4&nbsp;pA for event threshold was used. The analysis was performed within the first 10 and 40&nbsp;s of recording or within the 30&nbsp;s period that showed the most stable baseline. In the case of optogenetically evoked IPSCs from pyramidal cells, monosynaptic currents were identified by the latency of the light mediated response. This was similar to the latency using TTX+4-AP to isolate monosynaptic responses (used in some of the recordings). Peak amplitude was identified using the threshold search in Clampfit. IPSC peak amplitudes were averaged across 10 trials at each irradiance level per cell. Average values per cell were then condensed into a grand average per group.</div></section><section id="sec-4-5-7"><h4><i>In silico</i> Identification Of Ligand-Receptor Pairs</h4><div id="p0515" role="paragraph">Developmental transcriptomic datasets from <span class="dropBlock reference-citations"><a id="body-ref-sref48-4" href="#" role="doc-biblioref" data-xml-rid="bib48" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref48-4" aria-controls="sref48-4">Matcovitch-Natan et&nbsp;al. (2016)</a><div class="dropBlock__holder" data-db-target-of="sref48-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">49.</div><div>Matcovitch-Natan, O. ∙ Winter, D.R. ∙ Giladi, A. ...</div><div><strong>Microglia development follows a stepwise program to regulate brain homeostasis</strong></div><div><em>Science.</em> 2016; <strong>353</strong>:aad8670</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aad8670" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-84976883162" target="_blank" aria-label="Open Scopus in new tab">Scopus (792)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27338705/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aad8670&amp;pmid=27338705" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span> and <span class="dropBlock reference-citations"><a id="body-ref-sref24-4" href="#" role="doc-biblioref" data-xml-rid="bib24" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref24-4" aria-controls="sref24-4">Favuzzi et&nbsp;al. (2019)</a><div class="dropBlock__holder" data-db-target-of="sref24-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">25.</div><div>Favuzzi, E. ∙ Deogracias, R. ∙ Marques-Smith, A. ...</div><div><strong>Distinct molecular programs regulate synapse specificity in cortical inhibitory circuits</strong></div><div><em>Science.</em> 2019; <strong>363</strong>:413-417</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aau8977" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85060511221" target="_blank" aria-label="Open Scopus in new tab">Scopus (110)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30679375/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aau8977&amp;pmid=30679375" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span> were used to identify receptors (or cell adhesion and membrane proteins) within microglia that could interact with ligands (or cell adhesion and membrane proteins) expressed by interneurons but not pyramidal cells. First, the list of genes enriched in cluster 6 (upregulated in postnatal brain) and 7 (upregulated throughout development) from <span class="dropBlock reference-citations"><a id="body-ref-sref48-5" href="#" role="doc-biblioref" data-xml-rid="bib48" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref48-5" aria-controls="sref48-5">Matcovitch-Natan et&nbsp;al. (2016)</a><div class="dropBlock__holder" data-db-target-of="sref48-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">49.</div><div>Matcovitch-Natan, O. ∙ Winter, D.R. ∙ Giladi, A. ...</div><div><strong>Microglia development follows a stepwise program to regulate brain homeostasis</strong></div><div><em>Science.</em> 2016; <strong>353</strong>:aad8670</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aad8670" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-84976883162" target="_blank" aria-label="Open Scopus in new tab">Scopus (792)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27338705/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aad8670&amp;pmid=27338705" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span> were used as input for a Gene Ontology search in Panther. Genes belonging to the following categories were selected: transmembrane signal receptor, cell adhesion molecule, cell junction protein, defense immune protein, extracellular matrix protein, intercellular signal molecule, membrane traffic protein, transmembrane signal receptor (for protein class); cell junction, extracellular region part, extracellular region, membrane part, membrane, synapse part, synapse (for cellular component); biological adhesion, developmental process, immune system process, response to stimulus, signaling (for biological process). Duplicates were removed and the resulting list was further filtered by function and cellular location in <span class="dropBlock reference-citations"><a id="body-ref-sref74" href="#" role="doc-biblioref" data-xml-rid="bib74" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref74" aria-controls="sref74">The UniProt Consortium (2017)</a><div class="dropBlock__holder" data-db-target-of="sref74"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">76.</div><div>The UniProt Consortium</div><div><strong>UniProt: the universal protein knowledgebase</strong></div><div><em>Nucleic Acids Res.</em> 2017; <strong>45</strong>:D158-D169</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gkw1099" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_76_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85016153986" target="_blank" aria-label="Open Scopus in new tab">Scopus (3396)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27899622/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgkw1099&amp;pmid=27899622" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span> and only receptors, membrane and cell-adhesion proteins were kept (e.g., cytokines were removed). The resulting list of genes was used as input in BioGRID (<span class="dropBlock reference-citations"><a id="body-ref-sref71" href="#" role="doc-biblioref" data-xml-rid="bib71" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref71" aria-controls="sref71">Stark et&nbsp;al., 2006</a><div class="dropBlock__holder" data-db-target-of="sref71"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">73.</div><div>Stark, C. ∙ Breitkreutz, B.-J. ∙ Reguly, T. ...</div><div><strong>BioGRID: a general repository for interaction datasets</strong></div><div><em>Nucleic Acids Res.</em> 2006; <strong>34</strong>:D535-D539</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gkj109" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-33644873184" target="_blank" aria-label="Open Scopus in new tab">Scopus (2967)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16381927/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgkj109&amp;pmid=16381927" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), to detect interactors. Intracellular interactions were manually removed and only inter-cellular interactors were kept. An exception was made for the gene <i>Lmbr1l</i>, whose BioGRID output of 939 interactors was filtered using Panther to remove intracellular interactions by keeping genes in the following GO cellular component categories and removing duplicates: cell junction, extracellular region part, extracellular region, membrane part, membrane, synapse part and synapse. Interactors not expressed in interneurons were removed using a cutoff of FPKM&nbsp;&lt; 4 in P10 SST and PV interneurons. The expression in SST and PV interneurons was averaged to obtain the expression values for interneurons at P5 and P10. Then a specificity score was calculated as log<sub>10</sub>(average expression across P0, P5 and P10 interneurons/expression in P12 pyramidal cells) and genes with a score &gt; 0.2 were considered enriched in interneurons as compared to pyramidal cells. The resulting list of interactors was then ranked by specificity score.</div></section><section id="sec-4-5-8"><h4>scRNA-seq data processing</h4><div id="p0520" role="paragraph">Raw scRNA-seq data were processed using the standard CellRanger v3.0.0 pipeline (10x Genomics). Reads were aligned to the GRCm38 (mm10) <i>Mus musculus</i> genome. Doublets, cells with low quality or likely undergoing stress and putative non-microglia were excluded from the analysis. Criteria for exclusion were: fraction of reads mapping to mitochondrial genes greater than 10%, fewer than 800 genes detected, more than 20,000 total UMI detected and no reads mapping to <i>Fcrls</i> and <i>Cx3cr1</i> genes. The almost complete absence of cells expressing <i>Ms4a7</i> confirmed that CNS border-associated macrophages (BAMs) (<span class="dropBlock reference-citations"><a id="body-ref-sref79" href="#" role="doc-biblioref" data-xml-rid="bib79" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref79" aria-controls="sref79">Van Hove et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref79"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">81.</div><div>Van Hove, H. ∙ Martens, L. ∙ Scheyltjens, I. ...</div><div><strong>A single-cell atlas of mouse brain macrophages reveals unique transcriptional identities shaped by ontogeny and tissue environment</strong></div><div><em>Nat. Neurosci.</em> 2019; <strong>22</strong>:1021-1035</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41593-019-0393-4" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85065294555" target="_blank" aria-label="Open Scopus in new tab">Scopus (509)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31061494/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41593-019-0393-4&amp;pmid=31061494" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) were not contaminating the dataset. After these filtering steps, 6,235 wild-type control cells, 9,078 Cre-Het control cells and 9,752 GABA<sub>B1</sub>R KO cells were retained for further analysis.</div></section><section id="sec-4-5-9"><h4>scRNA-seq data clustering &amp; visualization</h4><div id="p0525" role="paragraph">Since previous work has shown that microglia gene expression follows a continuous rather than discrete distribution (<span class="dropBlock reference-citations"><a id="body-ref-sref31-5" href="#" role="doc-biblioref" data-xml-rid="bib31" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref31-5" aria-controls="sref31-5">Hammond et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref31-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">32.</div><div>Hammond, T.R. ∙ Dufort, C. ∙ Dissing-Olesen, L. ...</div><div><strong>Single-Cell RNA Sequencing of Microglia throughout the Mouse Lifespan and in the Injured Brain Reveals Complex Cell-State Changes</strong></div><div><em>Immunity.</em> 2019; <strong>50</strong>:253-271.e6</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.immuni.2018.11.004&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.immuni.2018.11.004&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85056738677" target="_blank" aria-label="Open Scopus in new tab">Scopus (1123)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30471926/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.immuni.2018.11.004&amp;pmid=30471926" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref38-3" href="#" role="doc-biblioref" data-xml-rid="bib38" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref38-3" aria-controls="sref38-3">Li et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref38-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">39.</div><div>Li, Q. ∙ Cheng, Z. ∙ Zhou, L. ...</div><div><strong>Developmental Heterogeneity of Microglia and Brain Myeloid Cells Revealed by Deep Single-Cell RNA Sequencing</strong></div><div><em>Neuron.</em> 2019; <strong>101</strong>:207-223.e10</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.neuron.2018.12.006&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.neuron.2018.12.006&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85059299323" target="_blank" aria-label="Open Scopus in new tab">Scopus (572)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30606613/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.neuron.2018.12.006&amp;pmid=30606613" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref47-3" href="#" role="doc-biblioref" data-xml-rid="bib47" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref47-3" aria-controls="sref47-3">Masuda et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref47-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">48.</div><div>Masuda, T. ∙ Sankowski, R. ∙ Staszewski, O. ...</div><div><strong>Spatial and temporal heterogeneity of mouse and human microglia at single-cell resolution</strong></div><div><em>Nature.</em> 2019; <strong>566</strong>:388-392</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-0924-x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_48_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85061969567" target="_blank" aria-label="Open Scopus in new tab">Scopus (735)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30760929/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-0924-x&amp;pmid=30760929" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), the Monocle3&nbsp;R package was used to perform unsupervised clustering analysis on scRNA-seq data (<span class="dropBlock reference-citations"><a id="body-ref-sref8-2" href="#" role="doc-biblioref" data-xml-rid="bib8" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref8-2" aria-controls="sref8-2">Cao et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref8-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">8.</div><div>Cao, J. ∙ Spielmann, M. ∙ Qiu, X. ...</div><div><strong>The single-cell transcriptional landscape of mammalian organogenesis</strong></div><div><em>Nature.</em> 2019; <strong>566</strong>:496-502</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-0969-x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85062419120" target="_blank" aria-label="Open Scopus in new tab">Scopus (1641)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30787437/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-0969-x&amp;pmid=30787437" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Briefly, a gene by cell expression matrix was generated for wild-type and GABA<sub>B1</sub>R KO and annotated by sample (WT_1 and WT_2, KO_2 and KO_3 respectively), preprocessed using default parameters (normalized by log and size factor, and reduced to 50 dimensions with PCA), batch effect were removed with align_cds using sample as the alignment group and condition as residual_model_formula_str; UMAP was computed using reduce_dimension, clusters were computed using the Leiden community detection with cluster_cells. Cre-Het data were preprocessed in the same manner. The same procedure was carried out for subclustering with the only difference that cells belonging to the cluster of interest were used. All plots from the analysis were generated using Monocle3 or Scanpy (<span class="dropBlock reference-citations"><a id="body-ref-sref3-2" href="#" role="doc-biblioref" data-xml-rid="bib3" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref3-2" aria-controls="sref3-2">Amir et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref3-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">3.</div><div>Amir, A.D. ∙ Davis, K.L. ∙ Tadmor, M.D. ...</div><div><strong>viSNE enables visualization of high dimensional single-cell data and reveals phenotypic heterogeneity of leukemia</strong></div><div><em>Nat. Biotechnol.</em> 2013; <strong>31</strong>:545-552</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nbt.2594" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-84880280631" target="_blank" aria-label="Open Scopus in new tab">Scopus (1197)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23685480/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnbt.2594&amp;pmid=23685480" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref87-2" href="#" role="doc-biblioref" data-xml-rid="bib87" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref87-2" aria-controls="sref87-2">Wolf et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref87-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">89.</div><div>Wolf, F.A. ∙ Angerer, P. ∙ Theis, F.J.</div><div><strong>SCANPY: large-scale single-cell gene expression data analysis</strong></div><div><em>Genome Biol.</em> 2018; <strong>19</strong>:15</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13059-017-1382-0" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_89_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85041394976" target="_blank" aria-label="Open Scopus in new tab">Scopus (2665)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29409532/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13059-017-1382-0&amp;pmid=29409532" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) built in functions as well as custom scripts in R.</div><div id="p0530" role="paragraph">To generate featureplots and heatmaps, data were log-transformed after adding a pseudocount and scaled to unit variance and zero mean. More specifically, UMI count values per cell were transformed by preprocess_cds in Monocle3, normalize_total and Freeman-Tukey transform (FTT) or sc.pp.log1p with Scanpy; they were then mean centered and scaled by genes with maximum and minimum values set to 3 and −3. These values were plotted using a custom R script and ggplot2.</div><div id="p0535" role="paragraph">For the analysis of the cell cycle phase, each cell was assigned a G2M score and an S score, based on its expression of G2/M and S phase markers using CellCycleScoring from <span class="dropBlock reference-citations"><a id="body-ref-sref62-2" href="#" role="doc-biblioref" data-xml-rid="bib62" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref62-2" aria-controls="sref62-2">Satija et&nbsp;al. (2015)</a><div class="dropBlock__holder" data-db-target-of="sref62-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">64.</div><div>Satija, R. ∙ Farrell, J.A. ∙ Gennert, D. ...</div><div><strong>Spatial reconstruction of single-cell gene expression data</strong></div><div><em>Nat. Biotechnol.</em> 2015; <strong>33</strong>:495-502</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nbt.3192" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_64_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-84929151009" target="_blank" aria-label="Open Scopus in new tab">Scopus (3012)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25867923/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnbt.3192&amp;pmid=25867923" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>.</div></section><section id="sec-4-5-10"><h4>Single Cell RNA Sequencing Dataset Integration</h4><div id="p0540" role="paragraph">For integrating the wild-type control and GABA<sub>B1</sub>R KO datasets, a gene by cell expression matrix was generated and annotated by sample (WT_1 and WT_2, KO_2 and KO_3) and condition (WT and KO). Data were then preprocessed using default parameters in Monocle3 (normalized by log and size factor, and reduced to 50 dimensions with PCA). Batch effect were removed with align_cds using condition as the alignment group. UMAP was computed using reduce_dimension and clusters were computed using the Leiden community detection with cluster_cells. To evaluate the validity of wild-type and cKO cell integration, the composition of each mixed cluster was assessed. Briefly, each cell from all integrated clusters (I1 to I8) was traced back to their corresponding wild-type control (C1 to C5) or GABA<sub>B1</sub>R cKO (GC1 to GC7) cluster. Proportions were then computed for each integrated cluster and are shown in the composition charts in <a id="crosref1530" href="#figs6">Figure&nbsp;S6</a>.</div><div id="p0545" role="paragraph">The Cre-Het control and GABA<sub>B1</sub>R KO datasets were generated separately and, as such, exhibited stronger batch effects than the wild-type control and GABA<sub>B1</sub>R KO datasets. For integrating the Cre-Het control and GABA<sub>B1</sub>R KO datasets, DEWÄKSS was used (<span class="dropBlock reference-citations"><a id="body-ref-sref78-3" href="#" role="doc-biblioref" data-xml-rid="bib78" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref78-3" aria-controls="sref78-3">Tjärnberg et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref78-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">80.</div><div>Tjärnberg, A. ∙ Mahmood, O. ∙ Jackson, C.A. ...</div><div><strong>Optimal tuning of weighted kNN- and diffusion-based methods for denoising single cell genomics data</strong></div><div><em>bioRxiv.</em> 2020;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/2020.02.28.970202" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_80_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85098901697" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2F2020.02.28.970202" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Following standard pre-processing (normalization and log1p transformation), the optimal amount of principal components and k for the k-nearest neighbor graph was computed in DEWÄKSS over the BBKNN algorithm to integrate the conditions (<span class="dropBlock reference-citations"><a id="body-ref-sref59" href="#" role="doc-biblioref" data-xml-rid="bib59" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref59" aria-controls="sref59">Polański et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref59"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">61.</div><div>Polański, K. ∙ Young, M.D. ∙ Miao, Z. ...</div><div><strong>BBKNN: fast batch alignment of single cell transcriptomes</strong></div><div><em>Bioinformatics.</em> 2020; <strong>36</strong>:964-965</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btz625" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85074670646" target="_blank" aria-label="Open Scopus in new tab">Scopus (337)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31400197/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtz625&amp;pmid=31400197" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Unsupervised clustering was performed using Leiden clustering with the adjacency matrix computed in DEWÄKSS. Denoised values and clusters were then imported in Seurat for differential expression analysis (see below).</div><div id="p0550" role="paragraph">To integrate our P15 wild-type microglia single-cell data with embryonic, early postnatal (P4-P5) and juvenile (P30) datasets from <span class="dropBlock reference-citations"><a id="body-ref-sref31-6" href="#" role="doc-biblioref" data-xml-rid="bib31" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref31-6" aria-controls="sref31-6">Hammond et&nbsp;al. (2019)</a><div class="dropBlock__holder" data-db-target-of="sref31-6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">32.</div><div>Hammond, T.R. ∙ Dufort, C. ∙ Dissing-Olesen, L. ...</div><div><strong>Single-Cell RNA Sequencing of Microglia throughout the Mouse Lifespan and in the Injured Brain Reveals Complex Cell-State Changes</strong></div><div><em>Immunity.</em> 2019; <strong>50</strong>:253-271.e6</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.immuni.2018.11.004&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.immuni.2018.11.004&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85056738677" target="_blank" aria-label="Open Scopus in new tab">Scopus (1123)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30471926/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.immuni.2018.11.004&amp;pmid=30471926" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>, Monocle3 was run with default parameters as described above. Due to the higher number of cells, the following modifications were introduced: num_dim set to 200 for preprocess_cds and residual_model_formula_str&nbsp;= ‘~sample&nbsp;+ tp’ for align_cds (“sample” indicates each replicates and “tp” indicates each time point). A developmental trajectory was then computed using learn_graph.</div></section><section id="sec-4-5-11"><h4>Single Cell RNA Sequencing Cluster Annotation</h4><div id="p0555" role="paragraph">Lists of genes enriched in each cluster were generated using the top_markers function in the Monocle3&nbsp;R package (marker score ≥ 0.15). Additionally, clusters from Monocle3 were transferred to Seurat (<span class="dropBlock reference-citations"><a id="body-ref-sref62-3" href="#" role="doc-biblioref" data-xml-rid="bib62" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref62-3" aria-controls="sref62-3">Satija et&nbsp;al. (2015)</a><div class="dropBlock__holder" data-db-target-of="sref62-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">64.</div><div>Satija, R. ∙ Farrell, J.A. ∙ Gennert, D. ...</div><div><strong>Spatial reconstruction of single-cell gene expression data</strong></div><div><em>Nat. Biotechnol.</em> 2015; <strong>33</strong>:495-502</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nbt.3192" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_64_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-84929151009" target="_blank" aria-label="Open Scopus in new tab">Scopus (3012)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25867923/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnbt.3192&amp;pmid=25867923" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span> and lists of genes enriched in each cluster were generated using the FindAllMarkers function in the Seurat R package with Wilcoxon rank sum tests (default parameters: absolute value of the average log<sub>2</sub>FC ≥ 0.25 and p value ≤ 0.01). The full lists of cluster-enriched genes for wild-type, Cre-Het, cKO and the integrated datasets are shown in <a id="crosref1555" href="#mmc1">Table S1</a>. Confirming the soundness and reliability of our data and analysis, both the Seurat and Monocle analyses returned the same genes or genes with similar functions. The relative enrichment of gene groups involved in analogous processes as well as the expression of known marker genes was used for cluster identification. The cellular function of each gene was based on gene ontology (GO) terms and Medline search. The functional annotation of individual clusters was based on the roles of cluster-enriched genes in addition to a comparison with annotated clusters from previous work (including but not limited to references <span class="dropBlock reference-citations"><a id="body-ref-sref31-7" href="#" role="doc-biblioref" data-xml-rid="bib31" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref31-7" aria-controls="sref31-7">Hammond et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref31-7"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">32.</div><div>Hammond, T.R. ∙ Dufort, C. ∙ Dissing-Olesen, L. ...</div><div><strong>Single-Cell RNA Sequencing of Microglia throughout the Mouse Lifespan and in the Injured Brain Reveals Complex Cell-State Changes</strong></div><div><em>Immunity.</em> 2019; <strong>50</strong>:253-271.e6</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.immuni.2018.11.004&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.immuni.2018.11.004&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85056738677" target="_blank" aria-label="Open Scopus in new tab">Scopus (1123)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30471926/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.immuni.2018.11.004&amp;pmid=30471926" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref35-3" href="#" role="doc-biblioref" data-xml-rid="bib35" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref35-3" aria-controls="sref35-3">Keren-Shaul et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref35-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">36.</div><div>Keren-Shaul, H. ∙ Spinrad, A. ∙ Weiner, A. ...</div><div><strong>A Unique Microglia Type Associated with Restricting Development of Alzheimer’s Disease</strong></div><div><em>Cell.</em> 2017; <strong>169</strong>:1276-1290.e17</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.cell.2017.05.018&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.cell.2017.05.018&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85020287843" target="_blank" aria-label="Open Scopus in new tab">Scopus (2741)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28602351/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2017.05.018&amp;pmid=28602351" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref38-4" href="#" role="doc-biblioref" data-xml-rid="bib38" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref38-4" aria-controls="sref38-4">Li et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref38-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">39.</div><div>Li, Q. ∙ Cheng, Z. ∙ Zhou, L. ...</div><div><strong>Developmental Heterogeneity of Microglia and Brain Myeloid Cells Revealed by Deep Single-Cell RNA Sequencing</strong></div><div><em>Neuron.</em> 2019; <strong>101</strong>:207-223.e10</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.neuron.2018.12.006&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.neuron.2018.12.006&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85059299323" target="_blank" aria-label="Open Scopus in new tab">Scopus (572)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30606613/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.neuron.2018.12.006&amp;pmid=30606613" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref47-4" href="#" role="doc-biblioref" data-xml-rid="bib47" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref47-4" aria-controls="sref47-4">Masuda et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref47-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">48.</div><div>Masuda, T. ∙ Sankowski, R. ∙ Staszewski, O. ...</div><div><strong>Spatial and temporal heterogeneity of mouse and human microglia at single-cell resolution</strong></div><div><em>Nature.</em> 2019; <strong>566</strong>:388-392</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-0924-x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_48_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85061969567" target="_blank" aria-label="Open Scopus in new tab">Scopus (735)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30760929/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-0924-x&amp;pmid=30760929" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div></section><section id="sec-4-5-12"><h4>Control WT, GABA<sub>B1</sub>R cKO &amp; WT+KO integrated clusters</h4><div id="p0560" role="paragraph">Unsupervised clustering of wild-type microglia returned 5 main clusters. Cells from cluster 1 exhibited higher levels of genes implicated in metal homeostasis (e.g., <i>Fth1, Ftl1, Mt1</i>), actin cytoskeleton dynamics (e.g., <i>Tmsb4x, Pfn1</i>, <i>Cfl1</i>) and ribosomal components (e.g., <i>Rps26, Rps15a, Rpl32</i>, <i>Rps5</i>), all of which have been previously identified as transcriptional signatures of postnatal immature microglia (<span class="dropBlock reference-citations"><a id="body-ref-sref38-5" href="#" role="doc-biblioref" data-xml-rid="bib38" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref38-5" aria-controls="sref38-5">Li et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref38-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">39.</div><div>Li, Q. ∙ Cheng, Z. ∙ Zhou, L. ...</div><div><strong>Developmental Heterogeneity of Microglia and Brain Myeloid Cells Revealed by Deep Single-Cell RNA Sequencing</strong></div><div><em>Neuron.</em> 2019; <strong>101</strong>:207-223.e10</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.neuron.2018.12.006&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.neuron.2018.12.006&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85059299323" target="_blank" aria-label="Open Scopus in new tab">Scopus (572)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30606613/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.neuron.2018.12.006&amp;pmid=30606613" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref47-5" href="#" role="doc-biblioref" data-xml-rid="bib47" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref47-5" aria-controls="sref47-5">Masuda et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref47-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">48.</div><div>Masuda, T. ∙ Sankowski, R. ∙ Staszewski, O. ...</div><div><strong>Spatial and temporal heterogeneity of mouse and human microglia at single-cell resolution</strong></div><div><em>Nature.</em> 2019; <strong>566</strong>:388-392</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-0924-x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_48_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85061969567" target="_blank" aria-label="Open Scopus in new tab">Scopus (735)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30760929/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-0924-x&amp;pmid=30760929" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Consistently, cells from cluster 1 also shared some transcriptional signatures (e.g., <i>Ftl1, Cfl1, Mt1, Rpl30</i>) with cluster 4 P4-P5 microglia from <span class="dropBlock reference-citations"><a id="body-ref-sref31-8" href="#" role="doc-biblioref" data-xml-rid="bib31" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref31-8" aria-controls="sref31-8">Hammond et&nbsp;al. (2019)</a><div class="dropBlock__holder" data-db-target-of="sref31-8"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">32.</div><div>Hammond, T.R. ∙ Dufort, C. ∙ Dissing-Olesen, L. ...</div><div><strong>Single-Cell RNA Sequencing of Microglia throughout the Mouse Lifespan and in the Injured Brain Reveals Complex Cell-State Changes</strong></div><div><em>Immunity.</em> 2019; <strong>50</strong>:253-271.e6</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.immuni.2018.11.004&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.immuni.2018.11.004&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85056738677" target="_blank" aria-label="Open Scopus in new tab">Scopus (1123)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30471926/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.immuni.2018.11.004&amp;pmid=30471926" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>. Subclustering of cluster 1 further revealed an additional 22 subclusters of which two were notable. Microglia from subcluster 1.15 shared some of the transcriptional signatures of DAM (e.g., <i>Cd63, Cd9, Lyz2</i>) (<span class="dropBlock reference-citations"><a id="body-ref-sref35-4" href="#" role="doc-biblioref" data-xml-rid="bib35" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref35-4" aria-controls="sref35-4">Keren-Shaul et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref35-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">36.</div><div>Keren-Shaul, H. ∙ Spinrad, A. ∙ Weiner, A. ...</div><div><strong>A Unique Microglia Type Associated with Restricting Development of Alzheimer’s Disease</strong></div><div><em>Cell.</em> 2017; <strong>169</strong>:1276-1290.e17</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.cell.2017.05.018&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.cell.2017.05.018&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85020287843" target="_blank" aria-label="Open Scopus in new tab">Scopus (2741)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28602351/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2017.05.018&amp;pmid=28602351" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). These are also shared by proliferative-region-associated microglia (<span class="dropBlock reference-citations"><a id="body-ref-sref38-6" href="#" role="doc-biblioref" data-xml-rid="bib38" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref38-6" aria-controls="sref38-6">Li et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref38-6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">39.</div><div>Li, Q. ∙ Cheng, Z. ∙ Zhou, L. ...</div><div><strong>Developmental Heterogeneity of Microglia and Brain Myeloid Cells Revealed by Deep Single-Cell RNA Sequencing</strong></div><div><em>Neuron.</em> 2019; <strong>101</strong>:207-223.e10</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.neuron.2018.12.006&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.neuron.2018.12.006&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85059299323" target="_blank" aria-label="Open Scopus in new tab">Scopus (572)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30606613/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.neuron.2018.12.006&amp;pmid=30606613" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Subcluster 1.22 contained a small group of cells expressing higher levels of many established pro-inflammatory (e.g., <i>Ccl12, Stat1</i>) and interferon response (e.g., <i>Ifitm3, Ifi44, Ifi27l2a</i>) genes. These genes are typically induced in microglia upon injury or pathological conditions (<span class="dropBlock reference-citations"><a id="body-ref-sref31-9" href="#" role="doc-biblioref" data-xml-rid="bib31" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref31-9" aria-controls="sref31-9">Hammond et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref31-9"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">32.</div><div>Hammond, T.R. ∙ Dufort, C. ∙ Dissing-Olesen, L. ...</div><div><strong>Single-Cell RNA Sequencing of Microglia throughout the Mouse Lifespan and in the Injured Brain Reveals Complex Cell-State Changes</strong></div><div><em>Immunity.</em> 2019; <strong>50</strong>:253-271.e6</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.immuni.2018.11.004&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.immuni.2018.11.004&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85056738677" target="_blank" aria-label="Open Scopus in new tab">Scopus (1123)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30471926/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.immuni.2018.11.004&amp;pmid=30471926" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Cluster 2 was composed of cells expressing higher levels of homeostatic microglia core genes (e.g., <i>Fcrls, P2ry13, Hexb</i>), as well as genes involved in synapse pruning (e.g., <i>C1qa</i> (<span class="dropBlock reference-citations"><a id="body-ref-sref72-3" href="#" role="doc-biblioref" data-xml-rid="bib72" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref72-3" aria-controls="sref72-3">Stevens et&nbsp;al., 2007</a><div class="dropBlock__holder" data-db-target-of="sref72-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">74.</div><div>Stevens, B. ∙ Allen, N.J. ∙ Vazquez, L.E. ...</div><div><strong>The classical complement cascade mediates CNS synapse elimination</strong></div><div><em>Cell.</em> 2007; <strong>131</strong>:1164-1178</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.cell.2007.10.036&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.cell.2007.10.036&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-36849076770" target="_blank" aria-label="Open Scopus in new tab">Scopus (2260)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/18083105/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2007.10.036&amp;pmid=18083105" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), <i>Trem2</i> (<span class="dropBlock reference-citations"><a id="body-ref-sref25-2" href="#" role="doc-biblioref" data-xml-rid="bib25" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref25-2" aria-controls="sref25-2">Filipello et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref25-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">26.</div><div>Filipello, F. ∙ Morini, R. ∙ Corradini, I. ...</div><div><strong>The Microglial Innate Immune Receptor TREM2 Is Required for Synapse Elimination and Normal Brain Connectivity</strong></div><div><em>Immunity.</em> 2018; <strong>48</strong>:979-991.e8</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_26_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.immuni.2018.04.016&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_26_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.immuni.2018.04.016&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_26_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85046738379" target="_blank" aria-label="Open Scopus in new tab">Scopus (391)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29752066/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.immuni.2018.04.016&amp;pmid=29752066" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)). Cluster 3 microglia are distinguished by their relatively high expression of <i>Actb</i> and the long non-coding RNA <i>Gm42418</i>. Cells belonging to cluster 3 appeared as a transitional state between cluster 1 and 2. They expressed intermediate levels of genes encoding ribosome components and microglia core genes but lower levels of synapse pruning genes. Cluster 4 cells are characterized by high expression of long non-coding RNAs with unknown function (e.g., <i>Gm26870, Gm10800, Gm21738</i>). Finally, cluster 5 was composed of actively proliferating microglia, as shown by their high expression of genes encoding cyclins (e.g., <i>Ccnb2</i>), proteins involved in chromosome condensation and segregation during mitosis (e.g., <i>Cenpf, Smc2</i>), DNA replication (e.g., <i>Top2a, Mcm5</i>) and proliferation markers (e.g., <i>Mki67</i>).</div><div id="p0565" role="paragraph">The cKO dataset was composed of 7 clusters. Alike microglia from the wild-type cluster 1, cKO microglia from cluster 1 were less distinctive except for a relatively higher expression of genes encoding ribosomal proteins (e.g., <i>Rps15a, Rpl32</i>, <i>Rps5</i>) and, therefore, resembled postnatal immature microglia. Cluster 2 was composed of microglia sharing some of the transcriptional signatures of disease-associated microglia (DAM [<span class="dropBlock reference-citations"><a id="body-ref-sref35-5" href="#" role="doc-biblioref" data-xml-rid="bib35" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref35-5" aria-controls="sref35-5">Keren-Shaul et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref35-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">36.</div><div>Keren-Shaul, H. ∙ Spinrad, A. ∙ Weiner, A. ...</div><div><strong>A Unique Microglia Type Associated with Restricting Development of Alzheimer’s Disease</strong></div><div><em>Cell.</em> 2017; <strong>169</strong>:1276-1290.e17</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.cell.2017.05.018&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.cell.2017.05.018&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85020287843" target="_blank" aria-label="Open Scopus in new tab">Scopus (2741)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28602351/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2017.05.018&amp;pmid=28602351" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>], e.g., <i>Cd63, Cd9, Lyz2</i>). These cells were also characterized by a higher expression of classical pro-inflammatory genes (e.g., <i>Ccl12</i>) or genes induced in microglia under pathological conditions (e.g., <i>C3aR1</i>) (<span class="dropBlock reference-citations"><a id="body-ref-sref19" href="#" role="doc-biblioref" data-xml-rid="bib19" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref19" aria-controls="sref19">Doolen et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref19"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">20.</div><div>Doolen, S. ∙ Cook, J. ∙ Riedl, M. ...</div><div><strong>Complement 3a receptor in dorsal horn microglia mediates pronociceptive neuropeptide signaling</strong></div><div><em>Glia.</em> 2017; <strong>65</strong>:1976-1989</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/glia.23208" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85031504789" target="_blank" aria-label="Open Scopus in new tab">Scopus (28)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28850719/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fglia.23208&amp;pmid=28850719" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). However, these microglia also exhibited an increased expression of the anti-inflammatory gene <i>Mrc1</i>, typically down-regulated by pro-inflammatory and up-regulated by anti-inflammatory cytokines (<span class="dropBlock reference-citations"><a id="body-ref-sref81" href="#" role="doc-biblioref" data-xml-rid="bib81" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref81" aria-controls="sref81">von Ehr et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref81"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">83.</div><div>von Ehr, A. ∙ Attaai, A. ∙ Neidert, N. ...</div><div><strong>Inhibition of Microglial TGFβ Signaling Increases Expression of <i>Mrc1</i></strong></div><div><em>Front. Cell. Neurosci.</em> 2020; <strong>14</strong>:66</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.3389/fncel.2020.00066" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_83_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85083481662" target="_blank" aria-label="Open Scopus in new tab">Scopus (17)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32296307/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.3389%2Ffncel.2020.00066&amp;pmid=32296307" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref89" href="#" role="doc-biblioref" data-xml-rid="bib89" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref89" aria-controls="sref89">Zimmer et&nbsp;al., 2003</a><div class="dropBlock__holder" data-db-target-of="sref89"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">91.</div><div>Zimmer, H. ∙ Riese, S. ∙ Régnier-Vigouroux, A.</div><div><strong>Functional characterization of mannose receptor expressed by immunocompetent mouse microglia</strong></div><div><em>Glia.</em> 2003; <strong>42</strong>:89-100</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/glia.10196" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-0037388257" target="_blank" aria-label="Open Scopus in new tab">Scopus (35)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/12594740/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fglia.10196&amp;pmid=12594740" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Cluster 2 did not seem to have an obvious wild-type counterpart although it shared some transcriptional signatures with subcluster 1.15. In addition, microglia from cKO cluster 2 was extremely similar to Cre-Het cells from subcluster 1.3 (see below). GABA<sub>B1</sub>R cKO microglia from cluster 3 resembled microglia from the wild-type cluster 2. These cells expressed higher levels of homeostatic microglia core genes (e.g., <i>Fcrls, P2ry13, Hexb</i>) and genes involved in synapse pruning (e.g., <i>C1qa</i>). Like microglia from the wild-type cluster 2, cKO cells from cluster 3 upregulated mitochondrial genes (e.g., <i>mt-Co3, mt-Atp6, mt-Cytb</i>) and downregulated numerous genes encoding ribosomal proteins. Cells belonging to cluster 4 appeared as expanding microglia from cluster 1. Like cells from cluster 1, they expressed higher levels of ribosomal subunit genes (e.g., <i>Rps15a, Rpl32</i>) but were distinct in that they expressed a higher level of genes involved in DNA replication and cell cycle progression (e.g., <i>Hells, Cdca7</i>). Cluster 5 was composed of cells expressing higher levels of many classical pro-inflammatory (e.g., <i>Ccl12, Stat1</i>) and interferon response (e.g., <i>Ifitm3, Ifi44</i>) genes. Notably, microglia from cluster 5 shared several gene expression signatures with cells collected from the white matter of adult mice exposed to a focal demyelinating injury caused by lysolecithin (LPC) injection (cluster 9 from <span class="dropBlock reference-citations"><a id="body-ref-sref31-10" href="#" role="doc-biblioref" data-xml-rid="bib31" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref31-10" aria-controls="sref31-10">Hammond et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref31-10"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">32.</div><div>Hammond, T.R. ∙ Dufort, C. ∙ Dissing-Olesen, L. ...</div><div><strong>Single-Cell RNA Sequencing of Microglia throughout the Mouse Lifespan and in the Injured Brain Reveals Complex Cell-State Changes</strong></div><div><em>Immunity.</em> 2019; <strong>50</strong>:253-271.e6</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.immuni.2018.11.004&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.immuni.2018.11.004&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85056738677" target="_blank" aria-label="Open Scopus in new tab">Scopus (1123)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30471926/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.immuni.2018.11.004&amp;pmid=30471926" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Cluster 5 resembled the wild-type subcluster 1.22, although it was considerably more abundant, consistent with the loss of the anti-inflammatory role of GABA<sub>B1</sub>Rs in microglia (<span class="dropBlock reference-citations"><a id="body-ref-sref37-3" href="#" role="doc-biblioref" data-xml-rid="bib37" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref37-3" aria-controls="sref37-3">Kuhn et&nbsp;al., 2004</a><div class="dropBlock__holder" data-db-target-of="sref37-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">38.</div><div>Kuhn, S.A. ∙ van Landeghem, F.K.H. ∙ Zacharias, R. ...</div><div><strong>Microglia express GABA(B) receptors to modulate interleukin release</strong></div><div><em>Mol. Cell. Neurosci.</em> 2004; <strong>25</strong>:312-322</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.mcn.2003.10.023" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_38_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-1542616888" target="_blank" aria-label="Open Scopus in new tab">Scopus (168)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15019947/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.mcn.2003.10.023&amp;pmid=15019947" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Finally, microglia belonging to clusters 6 and 7 were easily identified as cells undergoing mitosis, as shown by the higher expression of genes encoding cyclins (e.g., <i>Ccnb2</i>), proteins involved in chromosome segregation during mitosis (e.g., <i>Cenpf</i>), DNA replication (e.g., <i>Top2a</i>) and proliferation markers (e.g., <i>Mki67</i>). In addition, cells from cluster 6 almost exclusively expressed high levels of several histone proteins (e.g., <i>Hist1h1b</i>).</div><div id="p0570" role="paragraph">The integrated dataset (wild-type+GABA<sub>B1</sub>R cKO) was composed of 8 clusters that had a clear correspondence to the wild-type and cKO clusters. Microglia from clusters 1-3 resembled cells from wild-type or cKO clusters 1: immature postnatal microglia. In fact, these cells displayed a relative upregulation of genes implicated in metal homeostasis (<i>Fth1, Ftl1, and Mt1</i>), actin cytoskeleton dynamics (<i>Tmsb4x, Pfn1</i>, and <i>Cfl1</i>), and encoding ribosomal components (e.g., <i>Rps26, Rps15a, Rpl32</i>, and <i>Rps5</i>). Cells belonging to cluster 2 had a relative enrichment in <i>Macf1</i>, another gene involved in actin cytoskeleton dynamics and microglia from cluster 3 exhibited a small enrichment in ribosomal components and expressed higher levels of the long non-coding RNA Gm42418, resembling the wild-type cluster 3. Overall, microglia from clusters 2 and 3 were less distinctive than cells from cluster 1, although they exhibited slightly higher levels of homeostatic microglia core genes compared to cells from cluster 1. This last observation suggests that cells in clusters 2 and 3 might be transitioning from immature to homeostatic states. In contrast, microglia from cluster 4 were very distinctive. These cells expressed higher levels of homeostatic microglia core genes (e.g., <i>Fcrls, P2ry13, Hexb</i>) as well as genes involved in synapse pruning (e.g., <i>C1qa, Trem2</i>) and corresponded to microglia from wild-type and cKO clusters 2 and 3, respectively. Cluster 5 was the only cluster composed exclusively of GABA<sub>B1</sub>R cKO microglia and fully matched the cKO cluster 2 or Cre-Het subcluster 1.3 (see below). Like the wild-type cluster 5 and the cKO clusters 6 and 7, cluster 6 was composed of actively proliferating microglia as shown by the higher expression of genes involved in proliferation and DNA replication (e.g., <i>Mcm5, Hells, Mki67</i>) and by a cell cycle phase analysis. Microglia belonging to cluster 7 were characterized by a higher expression of classical pro-inflammatory genes (e.g., <i>Ccl12, Stat1</i>) and a large number of genes induced in microglia upon injury or pathological conditions (e.g., <i>Ifi44, Ifi27l2a</i>). These cells corresponded to the pro-inflammatory microglia from the cKO cluster 7 as well as the small wild-type subcluster 1.22. Consistent with this, GABA<sub>B1</sub>R cKO microglia were more abundant in cluster 7 than wild-type microglia, an expected result given the anti-inflammatory role of GABA<sub>B1</sub>Rs in microglia (<span class="dropBlock reference-citations"><a id="body-ref-sref37-4" href="#" role="doc-biblioref" data-xml-rid="bib37" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref37-4" aria-controls="sref37-4">Kuhn et&nbsp;al., 2004</a><div class="dropBlock__holder" data-db-target-of="sref37-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">38.</div><div>Kuhn, S.A. ∙ van Landeghem, F.K.H. ∙ Zacharias, R. ...</div><div><strong>Microglia express GABA(B) receptors to modulate interleukin release</strong></div><div><em>Mol. Cell. Neurosci.</em> 2004; <strong>25</strong>:312-322</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.mcn.2003.10.023" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_38_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-1542616888" target="_blank" aria-label="Open Scopus in new tab">Scopus (168)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15019947/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.mcn.2003.10.023&amp;pmid=15019947" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Finally, cells from cluster 8 (like wild-type cells from cluster 4) uniquely upregulated a set of long non-coding RNAs whose exact function is yet unidentified (e.g., <i>Gm26870, Gm10800, Gm21738</i>).</div></section><section id="sec-4-5-13"><h4>Control (Cre-Het) clusters</h4><div id="p0575" role="paragraph">Unsupervised clustering of <i>Cx3cr1</i><sup><i>Cre/+</i></sup> microglia returned 5 main clusters. Like microglia from wild-type or cKO cluster 1, Cre-Het microglia from cluster 1 shared transcriptional signatures of postnatal immature microglia (<span class="dropBlock reference-citations"><a id="body-ref-sref38-7" href="#" role="doc-biblioref" data-xml-rid="bib38" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref38-7" aria-controls="sref38-7">Li et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref38-7"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">39.</div><div>Li, Q. ∙ Cheng, Z. ∙ Zhou, L. ...</div><div><strong>Developmental Heterogeneity of Microglia and Brain Myeloid Cells Revealed by Deep Single-Cell RNA Sequencing</strong></div><div><em>Neuron.</em> 2019; <strong>101</strong>:207-223.e10</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.neuron.2018.12.006&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.neuron.2018.12.006&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85059299323" target="_blank" aria-label="Open Scopus in new tab">Scopus (572)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30606613/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.neuron.2018.12.006&amp;pmid=30606613" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref47-6" href="#" role="doc-biblioref" data-xml-rid="bib47" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref47-6" aria-controls="sref47-6">Masuda et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref47-6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">48.</div><div>Masuda, T. ∙ Sankowski, R. ∙ Staszewski, O. ...</div><div><strong>Spatial and temporal heterogeneity of mouse and human microglia at single-cell resolution</strong></div><div><em>Nature.</em> 2019; <strong>566</strong>:388-392</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-0924-x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_48_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85061969567" target="_blank" aria-label="Open Scopus in new tab">Scopus (735)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30760929/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-0924-x&amp;pmid=30760929" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). These cells exhibited a relative upregulation of genes implicated in metal homeostasis (e.g., <i>Fth1, Ftl1</i>), actin cytoskeleton dynamics (e.g., <i>Tmsb4x, Pfn1</i>, and <i>Cfl1</i>) and encoding ribosomal components (e.g., <i>Rps26, Rps15a, Rpl32</i>, and <i>Rps5</i>). However, like wild-type microglia, subclustering of Cre-Het cluster 1 revealed additional differences. In particular, subcluster 1.1 could be identified as prototypic postnatal immature microglia. In fact, the main transcriptional feature of microglia from subcluster 1.1 was a lower expression of genes that characterized subclusters 1.2 and 1.3. Like wild-type subcluster 1.22 or wild-type+cKO cluster 7, Cre-Het subcluster 1.2 contained cells expressing higher levels of pro-inflammatory (e.g., <i>Ccl12, Stat1</i>) and interferon response (e.g., <i>Ifitm3, Ifi44, Ifi27l2a</i>) genes typically induced in microglia upon injury or pathological conditions (<span class="dropBlock reference-citations"><a id="body-ref-sref31-11" href="#" role="doc-biblioref" data-xml-rid="bib31" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref31-11" aria-controls="sref31-11">Hammond et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref31-11"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">32.</div><div>Hammond, T.R. ∙ Dufort, C. ∙ Dissing-Olesen, L. ...</div><div><strong>Single-Cell RNA Sequencing of Microglia throughout the Mouse Lifespan and in the Injured Brain Reveals Complex Cell-State Changes</strong></div><div><em>Immunity.</em> 2019; <strong>50</strong>:253-271.e6</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.immuni.2018.11.004&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.immuni.2018.11.004&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85056738677" target="_blank" aria-label="Open Scopus in new tab">Scopus (1123)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30471926/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.immuni.2018.11.004&amp;pmid=30471926" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Similarly to microglia from wild-type subcluster 1.15, cKO cluster 2 or wild-type+cKO cluster 7, microglia belonging to subcluster 1.3 shared some of the transcriptional signatures of disease-associated microglia (DAM [<span class="dropBlock reference-citations"><a id="body-ref-sref35-6" href="#" role="doc-biblioref" data-xml-rid="bib35" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref35-6" aria-controls="sref35-6">Keren-Shaul et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref35-6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">36.</div><div>Keren-Shaul, H. ∙ Spinrad, A. ∙ Weiner, A. ...</div><div><strong>A Unique Microglia Type Associated with Restricting Development of Alzheimer’s Disease</strong></div><div><em>Cell.</em> 2017; <strong>169</strong>:1276-1290.e17</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.cell.2017.05.018&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.cell.2017.05.018&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85020287843" target="_blank" aria-label="Open Scopus in new tab">Scopus (2741)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28602351/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2017.05.018&amp;pmid=28602351" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>], e.g., <i>Cd63, Cd9, Lyz2</i>). The higher expression of the anti-inflammatory gene <i>Mrc1</i> further confirmed the resemblance of these microglia to cells from cKO cluster 2 or wild-type+cKO cluster 7. Like the wild-type cluster 2, Cre-Het clusters 2 and 3 were composed of cells expressing higher levels of homeostatic microglia core genes (e.g., <i>Fcrls, P2ry13, Hexb</i>), genes involved in synapse pruning (e.g., <i>C1qa</i> [<span class="dropBlock reference-citations"><a id="body-ref-sref72-4" href="#" role="doc-biblioref" data-xml-rid="bib72" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref72-4" aria-controls="sref72-4">Stevens et&nbsp;al., 2007</a><div class="dropBlock__holder" data-db-target-of="sref72-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">74.</div><div>Stevens, B. ∙ Allen, N.J. ∙ Vazquez, L.E. ...</div><div><strong>The classical complement cascade mediates CNS synapse elimination</strong></div><div><em>Cell.</em> 2007; <strong>131</strong>:1164-1178</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.cell.2007.10.036&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.cell.2007.10.036&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-36849076770" target="_blank" aria-label="Open Scopus in new tab">Scopus (2260)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/18083105/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2007.10.036&amp;pmid=18083105" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>], <i>Trem2</i> [<span class="dropBlock reference-citations"><a id="body-ref-sref25-3" href="#" role="doc-biblioref" data-xml-rid="bib25" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref25-3" aria-controls="sref25-3">Filipello et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref25-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">26.</div><div>Filipello, F. ∙ Morini, R. ∙ Corradini, I. ...</div><div><strong>The Microglial Innate Immune Receptor TREM2 Is Required for Synapse Elimination and Normal Brain Connectivity</strong></div><div><em>Immunity.</em> 2018; <strong>48</strong>:979-991.e8</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_26_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.immuni.2018.04.016&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_26_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.immuni.2018.04.016&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_26_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85046738379" target="_blank" aria-label="Open Scopus in new tab">Scopus (391)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29752066/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.immuni.2018.04.016&amp;pmid=29752066" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>]). Like the wild-type cluster 3, cells belonging to Cre-Het cluster 4 appeared as a transitional state between clusters 1 and 2-3. These cells expressed intermediate levels of genes characterizing cluster 1 but – like wild-type microglia from cluster 3 – could be distinguished based on their relatively higher expression of <i>Actb</i> and the long non-coding RNA <i>Gm42418</i>. Finally, like wild-type cluster 5, Cre-Het cluster 5 was composed of actively proliferating microglia as shown by the higher expression of genes encoding cyclins (e.g., <i>Ccnb2</i>), proteins involved in chromosome condensation and segregation during mitosis (e.g., <i>Cenpf</i>), DNA replication (e.g., <i>Top2a</i>) and proliferation markers (e.g., <i>Mki67</i>).</div></section><section id="sec-4-5-14"><h4>Differential expression analyses</h4><div id="p0580" role="paragraph">For Cre-Het versus wild-type analysis, all cells from wild-type (<i>Cx3cr1</i><sup><i>+/+</i></sup><i>; GABA</i><sub><i>B1</i></sub><i>R</i><sup><i>fl/fl</i></sup>) and Cre-Het (<i>Cx3cr1</i><sup><i>Cre/+</i></sup><i>; GABA</i><sub><i>B1</i></sub><i>R</i><sup><i>+/+</i></sup><i>)</i> controls were compared to identify genes expression changes that could be attributed to the loss of one copy of <i>Cx3cr1</i>. A differential expression analysis using all genes and all cells (no clusters) was performed using the FindMarkers function in the Seurat R package with Wilcoxon rank sum tests (log<sub>2</sub> fold change&nbsp;= 0) and the identity class set on condition (wild-type or Cre-Het). Differentially expressed genes (DEGs) having an absolute value of the average log<sub>2</sub> fold change ≥ 0.25 and an adjusted p value ≤ 0.01 were flagged as “Cre-Het DEGs” and used as additional filter applied to all differential expression analyses between wild-type and cKO microglia.</div><div id="p0585" role="paragraph">For each cluster composed of both wild-type and GABA<sub>B1</sub>R cKO microglia in similar proportions (i.e., clusters 1, 2, 3, 4, 6 and 8), a differential expression analysis was performed between wild-type and cKO cells. For this “within cluster” wild-type versus cKO differential expression analysis, first, clusters from Monocle were transferred to Seurat. Then, for cells belonging to each cluster, a differential expression analysis was performed using the FindMarkers function in the Seurat R package with Wilcoxon rank sum tests (absolute value of the average log<sub>2</sub> fold change ≥ 0.25) and the identity class set on condition (wild-type or cKO). The list of differentially expressed genes (DEGs) for each cluster was then intersected with the Cre-Het DEG list and overlapping genes were removed. Finally, only DEGs with an adjusted p value ≤ 0.05 were used for further analysis. The fraction of DEGs was calculated dividing the number of DEGs in each cluster (or group of clusters) by the total number of DEGs found across all clusters. The same conclusions (cluster 4 having the highest proportion of DEGs) were reached with or without the filtering steps.</div><div id="p0590" role="paragraph">For the “pseudo-bulk” wild-type versus cKO analysis, a differential expression analysis using all genes and all cells (no clusters) was performed using the FindMarkers function in the Seurat R package with Wilcoxon rank sum tests (absolute value of the average log<sub>2</sub> fold change ≥ 0) and the identity class set on condition (wild-type or cKO). Criteria for DEGs identification were an absolute value of the average log<sub>2</sub> fold change ≥ 0.10 and an adjusted p value ≤ 0.01. A relatively low fold change was used in this analysis to account for dilution of differences occurring within specific clusters only. Cre-Het DEGs were filtered from the final list. Since pseudo-bulk analyses yielded a higher number of DEGs, a more stringent adjusted p value was used (i.e., 0.01 instead of 0.05).</div><div id="p0595" role="paragraph">For differential expression analyses using microglia from only males or females, cells were selected based on the expression of X (<i>Xist</i>) or Y (<i>Ddx3y</i> and <i>Eif2s3y</i>) chromosome genes. Identity was set as female for cells expressing <i>Xist</i> and exhibiting no <i>Ddx3y</i> and <i>Eif2s3y</i> expression while identity was set as male for cells expressing <i>Ddx3y</i> or <i>Eif2s3y</i> and exhibiting no <i>Xist</i> expression. For microglia from either males or females, DEGs between WT and cKO within each cluster were then computed in Seurat as described above.</div></section><section id="sec-4-5-15"><h4>Gene Ontology (GO) enrichment analysis of DEGs</h4><div id="p0600" role="paragraph">A Gene Ontology (GO) enrichment analysis was performed in AmiGO using PANTHER Overrepresentation Test (released 20200407; annotation version and release date: GO Ontology database <a id="intref0340" href="https://doi.org/10.5281/zenodo.3727280">https://doi.org/10.5281/zenodo.3727280</a> Released 2020-03-23). The list of genes differentially expressed in cKO microglia within each cluster was used as input for a Gene Ontology enrichment analysis using all mouse genes as reference list. The annotation datasets used were: GO biological process complete, GO molecular function complete and GO cellular component complete. Enrichment was verified using Fisher’s Exact test with Bonferroni correction for multiple comparisons. The same analysis was carried out for downregulated genes, upregulated genes and for the combined list of upregulated and downregulated genes.</div></section><section id="sec-4-5-16"><h4>Supervised discriminative gene identification via elastic net penalized regression</h4><div id="p0605" role="paragraph">To identify genes predictive of control versus cKO microglia, a normalized expression matrix from the integrated cells composing clusters 1 to 4, 6 and 8 was used as input matrix. A hot-one-encoded response vector was used where wild-type was set to 0 and cKO to 1. 20% of the input matrix and response vector were randomly removed for each class. The remaining input matrix and response vector were used with cross validation (cv.glmnet, family set to Gaussian and ‘a’ ranging from 0 to 1) to determine optimal lambda and alpha hyper parameters. Model evaluation was run on the unseen 20% using the area under the curve and mean squared error. Once the best model was computed, coefficients were refitted using the whole input matrix. Genes were ranked based on their fitted coefficient value and further filtered on their median expression. Cre-Het genes were removed from the final list and a heatmap was generated as described above.</div></section><section id="sec-4-5-17"><h4>Denoising Expression data with a Weighted Affinity Kernel and Self-Supervision (DEWÄKSS)</h4><div id="p0610" role="paragraph">Cells were processed using Scanpy and DEWÄKSS (<span class="dropBlock reference-citations"><a id="body-ref-sref78-4" href="#" role="doc-biblioref" data-xml-rid="bib78" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref78-4" aria-controls="sref78-4">Tjärnberg et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref78-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">80.</div><div>Tjärnberg, A. ∙ Mahmood, O. ∙ Jackson, C.A. ...</div><div><strong>Optimal tuning of weighted kNN- and diffusion-based methods for denoising single cell genomics data</strong></div><div><em>bioRxiv.</em> 2020;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/2020.02.28.970202" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_80_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85098901697" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2F2020.02.28.970202" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Briefly, each sample was aggregated in a single expression matrix and then filtered as described above with the addition of a minimum number of counts required for a gene set to 10, cells were then normalized and transformed using Freeman-Tukey transform (FTT). DEWAKSS was then run on FTT values (n_neighbors ranging from 5 to 250, n_pcs ranging from 10 to 300, use_global_err set to False, modest set to max, neighbor_alg set to bbknn using euclidean metric and samples as batch keys for neighbor_args). Denoised data were then used for further analysis. First, unsupervised clustering was performed using Leiden clustering with the adjacency matrix computed in DEWÄKSS. Then, a list of top markers for each cluster was generated using rank_genes_groups (method&nbsp;= ‘wilcoxon’, use_raw&nbsp;= False, layer&nbsp;= ‘Ms’) (Table S6). Next, a comparison of cell assignment to each cluster before and after denoising (DEWÄKSS clusters versus integrated Monocle clusters) was carried out. This comparison revealed an almost complete overlap between cells in cluster 4. Cells from cluster 4 were then selected and a differential expression analysis between denoised wild-type and cKO microglia was performed using rank_genes_groups as described above.</div></section><section id="sec-4-5-18"><h4>MERFISH image analysis and cell segmentation</h4><div id="p0615" role="paragraph">Encoded MERFISH images were decoded using the MERlin pipeline (<span class="dropBlock reference-citations"><a id="body-ref-sref21-2" href="#" role="doc-biblioref" data-xml-rid="bib21" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref21-2" aria-controls="sref21-2">Emanuel and Babcock, 2020</a><div class="dropBlock__holder" data-db-target-of="sref21-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">22.</div><div>Emanuel, G. ∙ Babcock, H.</div><div><strong>ZhuangLab/MERlin: MERlin v0.1.6</strong></div><div>Zenodo, 2020</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=G.EmanuelH.BabcockZhuangLab%2FMERlin%3A+MERlin+v0.1.62020Zenodo" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) available at <a id="intref0345" href="https://github.com/emanuega/MERlin" target="_blank">https://github.com/emanuega/MERlin</a>, using the codebook provided by Vizgen. Decoded data were used to generate binary mosaics of the tissue where transcripts were represented as a single pixel. A mask was applied to select only the S1 region and microglia were segmented using the signal of the established <i>Tmem119</i> microglia marker. The segmentation script is available at: <a id="intref0350" href="https://github.com/emiliafavuzzi/Favuzzi_et_al_2021/tree/main/MERFISH" target="_blank">https://github.com/emiliafavuzzi/Favuzzi_et_al_2021/tree/main/MERFISH</a>. Briefly, microglia were segmented as <i>Tmem119</i> rich regions: sequential dilations and erosions on the Z projection of the binary mosaics representing <i>Tmem119</i> allowed transcripts that were close together to merge into a larger object. Transcripts for each gene within these cells were then counted by quantifying the number of “on” pixels in their respective binary representation. For genes specific to or highly enriched in microglia, only the Z projection was used to count transcripts. For <i>Gabbr1</i> and <i>Gabbr2</i>, which are also highly expressed in neurons, expression was assessed only in Z planes containing the microglia markers <i>Tmem119</i>, <i>Fcrls</i> and <i>P2ry12</i> (TFP).</div><div id="p0620" role="paragraph">The expression of genes imaged in the non-combinatorial smFISH rounds was measured using a high pass filter followed by a peak-finding algorithm, also available at: <a id="intref0355" href="https://github.com/emiliafavuzzi/Favuzzi_et_al_2021/tree/main/MERFISH" target="_blank">https://github.com/emiliafavuzzi/Favuzzi_et_al_2021/tree/main/MERFISH</a>. Briefly, transcripts were identified as local maxima on the high frequency images. The minimal intensity for one of these local maxima to be considered a transcript was automatically set for each gene as the 95<sup>th</sup> percentile of the intensity values of the filtered image. To avoid counting the same molecule twice, a minimal distance between two spots was set to 4 pixels (~0.44&nbsp;μm).</div><div id="p0625" role="paragraph">In total, 6,886 microglia were detected. The total number of Z planes with a TFP count higher than 7 was recorded and 92 cells that did not reach this threshold in any Z plane were removed. 93 cells located at the edges of the S1 region mask were also removed.</div></section><section id="sec-4-5-19"><h4>Cell clustering and DE analysis of MERFISH data</h4><div id="p0630" role="paragraph">After these filtering steps, a total of 3,563 control and 3,138 cKO microglia were retained for further analysis. However, as reported previously (<span class="dropBlock reference-citations"><a id="body-ref-sref14-3" href="#" role="doc-biblioref" data-xml-rid="bib14" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref14-3" aria-controls="sref14-3">Chen et&nbsp;al., 2015b</a><div class="dropBlock__holder" data-db-target-of="sref14-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">14.</div><div>Chen, K.H. ∙ Boettiger, A.N. ∙ Moffitt, J.R. ...</div><div><strong>RNA imaging. Spatially resolved, highly multiplexed RNA profiling in single cells</strong></div><div><em>Science.</em> 2015; <strong>348</strong>:aaa6090</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aaa6090" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-84928395184" target="_blank" aria-label="Open Scopus in new tab">Scopus (1274)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25858977/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aaa6090&amp;pmid=25858977" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref51-3" href="#" role="doc-biblioref" data-xml-rid="bib51" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref51-3" aria-controls="sref51-3">Moffitt et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref51-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">52.</div><div>Moffitt, J.R. ∙ Bambah-Mukku, D. ∙ Eichhorn, S.W. ...</div><div><strong>Molecular, spatial, and functional single-cell profiling of the hypothalamic preoptic region</strong></div><div><em>Science.</em> 2018; <strong>362</strong>:eaau5324</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aau5324" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85056571109" target="_blank" aria-label="Open Scopus in new tab">Scopus (576)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30385464/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aau5324&amp;pmid=30385464" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), batch effects between MERFISH runs were observed. Since most of the genes in our panel may be affected in cKO microglia and based on the assumption that batch effects are largely not cell-specific, the influence of these batch effects was removed by calculating the total counts for all genes in all cells in each slice (i.e., not only microglia, and largely dominated by <i>Gabbr</i> neuronal signal). The total number of counts was calculated by summing the total spots on each image and the total number of cells in each brain slice was detected by binarizing the DAPI image and counting the number of objects. These average counts per cells were then used to normalize the raw RNA counts in microglia. As previously reported (<span class="dropBlock reference-citations"><a id="body-ref-sref14-4" href="#" role="doc-biblioref" data-xml-rid="bib14" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref14-4" aria-controls="sref14-4">Chen et&nbsp;al., 2015b</a><div class="dropBlock__holder" data-db-target-of="sref14-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">14.</div><div>Chen, K.H. ∙ Boettiger, A.N. ∙ Moffitt, J.R. ...</div><div><strong>RNA imaging. Spatially resolved, highly multiplexed RNA profiling in single cells</strong></div><div><em>Science.</em> 2015; <strong>348</strong>:aaa6090</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aaa6090" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-84928395184" target="_blank" aria-label="Open Scopus in new tab">Scopus (1274)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25858977/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aaa6090&amp;pmid=25858977" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref51-4" href="#" role="doc-biblioref" data-xml-rid="bib51" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref51-4" aria-controls="sref51-4">Moffitt et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref51-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">52.</div><div>Moffitt, J.R. ∙ Bambah-Mukku, D. ∙ Eichhorn, S.W. ...</div><div><strong>Molecular, spatial, and functional single-cell profiling of the hypothalamic preoptic region</strong></div><div><em>Science.</em> 2018; <strong>362</strong>:eaau5324</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aau5324" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85056571109" target="_blank" aria-label="Open Scopus in new tab">Scopus (576)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30385464/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aau5324&amp;pmid=30385464" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), given the slice thickness, some cells were not imaged completely. To remove differences in RNA counts due to imaged cell volumes, RNA counts per cell were normalized by the imaged volume of each microglia. For both batch and volume correction, a scaling factor was used to ensure that the normalized expression values were on the same scale as the original counts.</div><div id="p0635" role="paragraph">Residual technical cell-specific biases that affected all genes equally were removed using the deconvolution strategy for scaling normalization (<span class="dropBlock reference-citations"><a id="body-ref-sref42" href="#" role="doc-biblioref" data-xml-rid="bib42" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref42" aria-controls="sref42">Lun et&nbsp;al., 2016a</a><div class="dropBlock__holder" data-db-target-of="sref42"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">43.</div><div>Lun, A.T. ∙ Bach, K. ∙ Marioni, J.C.</div><div><strong>Pooling across cells to normalize single-cell RNA sequencing data with many zero counts</strong></div><div><em>Genome Biol.</em> 2016; <strong>17</strong>:75</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13059-016-0947-7" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-84964556059" target="_blank" aria-label="Open Scopus in new tab">Scopus (629)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27122128/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13059-016-0947-7&amp;pmid=27122128" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). This involved dividing all RNA density counts for each cell by a “deconvolved” cell-specific scaling “size factor.” The size factor is based on the pool-based total sum of counts across all genes for each cell and such that the mean size factor across all cells is equal to 1. Specifically, RNA density values were imported using the “SingleCellExperiment” R package (<span class="dropBlock reference-citations"><a id="body-ref-sref2-2" href="#" role="doc-biblioref" data-xml-rid="bib2" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref2-2" aria-controls="sref2-2">Amezquita et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref2-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">2.</div><div>Amezquita, R.A. ∙ Lun, A.T.L. ∙ Becht, E. ...</div><div><strong>Orchestrating single-cell analysis with Bioconductor</strong></div><div><em>Nat. Methods.</em> 2020; <strong>17</strong>:137-145</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41592-019-0654-x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_2_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85076035714" target="_blank" aria-label="Open Scopus in new tab">Scopus (325)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31792435/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41592-019-0654-x&amp;pmid=31792435" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and scaling normalization by deconvolving size factors from cell pools was performed using the computeSumFactors function from the scran R package (<span class="dropBlock reference-citations"><a id="body-ref-sref43-2" href="#" role="doc-biblioref" data-xml-rid="bib43" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref43-2" aria-controls="sref43-2">Lun et&nbsp;al., 2016b</a><div class="dropBlock__holder" data-db-target-of="sref43-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">44.</div><div>Lun, A.T.L. ∙ McCarthy, D.J. ∙ Marioni, J.C.</div><div><strong>A step-by-step workflow for low-level analysis of single-cell RNA-seq data with Bioconductor</strong></div><div><em>F1000Res.</em> 2016; <strong>5</strong>:2122</div></div><div class="external-links"><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27909575/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?pmid=27909575" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The logNormCounts function from the scater R package (<span class="dropBlock reference-citations"><a id="body-ref-sref49-2" href="#" role="doc-biblioref" data-xml-rid="bib49" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref49-2" aria-controls="sref49-2">McCarthy et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref49-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">50.</div><div>McCarthy, D.J. ∙ Campbell, K.R. ∙ Lun, A.T.L. ...</div><div><strong>Scater: pre-processing, quality control, normalization and visualization of single-cell RNA-seq data in R</strong></div><div><em>Bioinformatics.</em> 2017; <strong>33</strong>:1179-1186</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btw777" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85019072518" target="_blank" aria-label="Open Scopus in new tab">Scopus (861)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28088763/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtw777&amp;pmid=28088763" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) was then used to compute the log-transformed normalized RNA density values for each cell which were used for downstream analyses. Of note, two mitochondrial genes (<i>mt-Co3, mt-Nd2</i>) imaged in the non-combinatorial rounds were too locally concentrated to count single transcripts. Gene expression analysis using integrated fluorescence intensity failed to clearly mark any defined cell group and therefore these two genes were removed from the subsequent analysis.</div><div id="p0640" role="paragraph">Given the low number of genes in our MERFISH panel, a reduction in the number of included genes or dimensionality reduction were not computationally necessary for clustering. Clustering of both control and cKO microglia was performed on the z-scores of the log-transformed normalized RNA densities per cell for all genes using the Jaccard-Louvain community-based detection with a k value of 20. This was implemented with the buildSNNGraph function in Scran while the Louvain method from the igraph package (<span class="dropBlock reference-citations"><a id="body-ref-sref15-2" href="#" role="doc-biblioref" data-xml-rid="bib15" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref15-2" aria-controls="sref15-2">Csardi and Nepusz, 2006</a><div class="dropBlock__holder" data-db-target-of="sref15-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">16.</div><div>Csardi, G. ∙ Nepusz, T.</div><div><strong>The igraph software package for complex network research</strong></div><div><em>InterJournal Complex Syst.</em> 2006; <strong>1695</strong>:1-9</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=G.CsardiT.NepuszThe+igraph+software+package+for+complex+network+researchInterJournal+Complex+Syst.1695200619" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) was used to identify communities. Next, data were imported in Seurat and cluster-enriched genes were identified using the FindMarkers function. For visualization, Uniform Manifold Approximation and Projection (UMAP) was used to embed cells in two dimensions.</div><div id="p0645" role="paragraph">A small cluster containing only 18 (15 control and 3 cKO) cells characterized by a low-level of pruning genes was not included for visualization or differential expression analysis. Of note, this cluster might correspond to the small fraction of WT cells in the scRNA-seq subcluster 4.2.</div><div id="p0650" role="paragraph">Lastly, differentially expressed genes between control and cKO microglia within each cluster were identified using the FindMarkers function in Seurat (absolute log-fold change threshold 0.25, adjusted p value lower than 0.05, ident cKO/control). Few <i>Gabbr</i>-negative cells were included in the <i>Gabbr</i>-enriched clusters and vice versa. For the differential expression analysis, these cells were removed by selecting only cells with a log-transformed normalized <i>Gabbr1</i> and <i>Gabbr1</i> RNA density higher than 1 (identified as local minimum in the <i>Gabbr1</i> and <i>Gabbr1</i> histogram distribution). A similar approach (removing few <i>Gabbr</i>-positive cells) was used for the non GABA-receptive clusters. Of note, similar conclusions (with, as expected, slightly lower fold changes) were reached with or without this last filtering step. Plots were generated using custom R scripts and ggplot2. For cell map visualization, cells were dilated with a disk of 13-pixel radius. The code for the downstream analysis is also available at <a id="intref0360" href="https://github.com/emiliafavuzzi/Favuzzi_et_al_2021/tree/main/MERFISH" target="_blank">https://github.com/emiliafavuzzi/Favuzzi_et_al_2021/tree/main/MERFISH</a>. <a id="intref0365" href="https://doi.org/10.5281/zenodo.4899812">https://doi.org/10.5281/zenodo.4899812</a>.</div></section><section id="sec-4-5-20"><h4>Motion Sequencing (MoSeq)</h4><div id="p0655" role="paragraph">Pre-processing code automatically identified the mouse in the arena, centered it in a 80&nbsp;× 80 pixel square and aligned the mouse’s nose to the right and tail to the left. Next, principal component analysis (PCA) was automatically performed on the 80&nbsp;× 80 pixel aligned movies of each mouse. Principal component scores for the data were then computed and fit to a generative model for mouse behavior through the use of computational inference techniques (<span class="dropBlock reference-citations"><a id="body-ref-sref86-4" href="#" role="doc-biblioref" data-xml-rid="bib86" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref86-4" aria-controls="sref86-4">Wiltschko et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref86-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">88.</div><div>Wiltschko, A.B. ∙ Johnson, M.J. ∙ Iurilli, G. ...</div><div><strong>Mapping Sub-Second Structure in Mouse Behavior</strong></div><div><em>Neuron.</em> 2015; <strong>88</strong>:1121-1135</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.neuron.2015.11.031&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.neuron.2015.11.031&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-84957565912" target="_blank" aria-label="Open Scopus in new tab">Scopus (373)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26687221/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.neuron.2015.11.031&amp;pmid=26687221" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref46-3" href="#" role="doc-biblioref" data-xml-rid="bib46" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref46-3" aria-controls="sref46-3">Markowitz et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref46-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">47.</div><div>Markowitz, J.E. ∙ Gillis, W.F. ∙ Beron, C.C. ...</div><div><strong>The Striatum Organizes 3D Behavior via Moment-to-Moment Action Selection</strong></div><div><em>Cell.</em> 2018; <strong>174</strong>:44-58.e17</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.cell.2018.04.019&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.cell.2018.04.019&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85046871902" target="_blank" aria-label="Open Scopus in new tab">Scopus (193)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29779950/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2018.04.019&amp;pmid=29779950" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The model returned a set of behavioral syllables that characterizes the expressed behavior of those mice, and the statistics that govern the frequency and the order in which those syllables were expressed in the experiment. Syllable usage was calculated by summing the number of occurrences of each syllable and dividing by total syllable usage across a recording session, converting syllable usage into a percentage. Each syllable was assigned a label by a human observer. Note that out of all syllables discovered, only a subset is shown (including those with relevance for the identified phenotype and all syllables that were significantly different as assessed by a statistical test). Syllable speed was calculated by computing, for all mice within each group, the mean of the mouse centroid speeds (frames were labeled using the respective syllable label). Transition matrices were calculated by counting the total number of occurrences in which syllable A transitions into syllable B (for all syllables). For comparing significant syllables between experimental conditions, a z-test was run on bootstrapped syllable usage distributions (one test per syllable) followed by the non-negative Benjamini-Hochberg procedure with a 10% family-wise error rate to correct for multiple comparisons.</div></section></section><section id="sec-4-6"><h3>Additional resources</h3><div id="p0660" role="paragraph">We have developed a searchable website where users can examine microglia gene expression patterns in our P15 wild-type and cKO scRNA-seq datasets <a id="intref0370" href="http://zoidberg.bio.nyu.edu/index" target="_blank">http://zoidberg.bio.nyu.edu/index</a>.</div></section></section><section id="acknowledgments" role="doc-acknowledgments"><h2>Acknowledgments</h2><div id="p0665" role="paragraph">We thank Plexxikon for PLX5622, B. Bettler for Gabbr1-floxed allele, and G. Feng for Tmem119<sup>CreER</sup> and Tmem119<sup>GFP</sup> alleles. We thank L. Dissing-Olesen for advice on microglia isolation and suggestions on the project. We thank V. Regulapati for help with IHC, J. Dimidschstein for sharing the S5E2 plasmid, and L. Guo for help with AAV production. We are grateful to members of the Fishell laboratory for stimulating discussions. We thank B. Dejanovic, E. Lebois, and J. Dimidschstein for feedback on the manuscript. E.F. was supported by EMBO (ALTF 444-2018) and FY21 Hearst Fellowships. Work in the G.F. laboratory is supported by the NIH (R01 NS081297, R01 MH071679, UG3 MH120096, and P01 NS074972), the Simons Foundation SFARI (566615), and the Harvard’s Dean Initiative. MERFISH work was supported by RF1MH121289 from NIMH and a Joint Broad-Israel Science Foundation Research Grant. The HMS Neuro Imaging facility is funded by NINDS P30 Core Center (NS072030).</div><section id="sec-5"><h3>Author contributions</h3><div id="p0670" role="paragraph">Conceptualization, E.F.; Methodology, E.F., L.B., Q.X., and S.F.; Software, E.F., G.A.S., L.B., A.Z., and K.Z.; Formal Analysis, E.F., S.H., G.A.S., L.B., L.A.I., A.Z., A.S., and E.K.; Investigation, E.F., S.H., L.B., L.A.I., Y.C., and M.F.-O.; Resources, S.F., S.R.D., B.S., and G.F.; Data Curation, E.F. and G.A.S.; Writing – Original Draft, E.F. and G.F.; Writing – Review &amp; Editing, all co-authors; Visualization, E.F. and G.A.S.; Supervision, R.B., S.R.D., B.S., and G.F.; Project Administration, E.F.; Funding Acquisition, E.F. and G.F.</div></section><section id="sec-6"><h3>Declaration of interests</h3><div id="p0675" role="paragraph">B.S. serves on the SAB and is a minor shareholder of Annexon. G.F. is a founder of Regel. The other authors declare no competing interests.</div></section><section id="sec-7"><h3>Inclusion and diversity</h3><div id="p0680" role="paragraph">One or more of the authors of this paper self-identifies as an underrepresented ethnic minority in science. One or more of the authors of this paper self-identifies as a member of the LGBTQ+ community. While citing references scientifically relevant for this work, we also actively worked to promote gender balance in our reference list.</div></section></section><section id="supplementary-material" data-type="supplementary-materials"><h2>Supplemental information (2)</h2><div id="p0690" role="paragraph"><div class="core-supplementary-material"><div class="core-link"><button class="downloadAllSupplementary"><span>Download all</span></button></div></div><div id="mmc1" class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="/cms/10.1016/j.cell.2021.06.018/attachment/e6503fd4-f5ee-4c85-ac5a-b66e1a4e1b4a/mmc1.xlsx" download="mmc1.xlsx" aria-labelledby="mmc1-heading mmc1-link"><span id="mmc1-link">Spreadsheet (8.46 MB)</span></a></div><div class="core-description"><div id="mmc1-heading" class="heading">Table S1. Genes enriched in microglia within each cluster compared to all other clusters for wild-type, GABA<sub>B1</sub>R cKO, and <i>Cx3cr1</i><sup><i>Cre/+</i></sup> (Cre-Het control) and integrated groups, related to Figure&nbsp;5</div><div id="espara0035" role="paragraph">Genes enriched within each cluster are listed based on a differential gene expression analysis carried out using Seurat. Fold changes, p values, FDR corrected adjusted p values, and the percent of cells in the cluster of interest (pct.1) versus all other cells (pct.2) are indicated. Enriched genes based on a marker gene analysis using Monocle are also shown. Marker scores, mean expression values, the fraction of cells expressing the gene, a specificity score, the proportion of variance pseudo R2, p values and q values are indicated.</div></div></div><div id="mmc2" class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="/cms/10.1016/j.cell.2021.06.018/attachment/e0dc1d57-b31b-44cd-8c5b-29f5f871f400/mmc2.xlsx" download="mmc2.xlsx" aria-labelledby="mmc2-heading mmc2-link"><span id="mmc2-link">Spreadsheet (869.82 KB)</span></a></div><div class="core-description"><div id="mmc2-heading" class="heading">Table S2. Differentially expression analyses, function of genes differentially expressed between wild-type and GABA<sub>B1</sub>R cKO microglia, denoising results as well as MERFISH clusters and differentially expressed genes, related to Figures 5 and 6</div><div id="espara0040" role="paragraph">(Sheet 1) Differential gene expression analysis between wild-type (<i>Cx3cr1</i><sup><i>+/+</i></sup><i>; GABA</i><sub><i>B1</i></sub><i>R</i><sup><i>fl/fl</i></sup>) and GABA<sub>B1</sub>R cKO (<i>Cx3cr1</i><sup><i>Cre/+</i></sup><i>; GABA</i><sub><i>B1</i></sub><i>R</i><sup><i>fl/fl</i></sup>) microglia within each mixed cluster of the integrated dataset. (Sheet 2) Same as sheet 1 but differentially expressed genes (DEGs) have been further filtered to include only those exhibiting an adjusted p value lower than 0.05 and to exclude genes expression changes that could be attributed to the loss of one copy of <i>Cx3cr1</i>. Differentially expressed genes between wild-type (<i>Cx3cr1</i><sup><i>+/+</i></sup><i>; GABA</i><sub><i>B1</i></sub><i>R</i><sup><i>fl/fl</i></sup>) and Cre-Het control (<i>Cx3cr1</i><sup><i>Cre/+</i></sup><i>; GABA</i><sub><i>B1</i></sub><i>R</i><sup><i>+/+</i></sup>) microglia are shown on the right (“Flagged Cre-Het DEGs). The output of this analysis (average log<sub>2</sub> fold change &gt; 0.25 and adjusted p value&nbsp;&lt; 0.01) was used to filter all comparisons between wild-type and GABA<sub>B1</sub>R cKO microglia. A manual functional classification of DEGs is also shown. (Sheet 3) Comparison between the DEGs from sheet 1 and DEGs after selecting cells from only males or females. (Sheet 4) Differential gene expression analysis between Cre-Het control (<i>Cx3cr1</i><sup><i>Cre/+</i></sup><i>; GABA</i><sub><i>B1</i></sub><i>R</i><sup><i>+/+</i></sup>) and GABA<sub>B1</sub>R cKO (<i>Cx3cr1</i><sup><i>Cre/+</i></sup><i>; GABA</i><sub><i>B1</i></sub><i>R</i><sup><i>fl/fl</i></sup>) microglia within each mixed cluster of the integrated dataset. (Sheet 5) Differential expression analysis between wild-type and GABA<sub>B1</sub>R cKO microglia using a “pseudo-bulk” approach (i.e., using all microglia from each experimental group irrespective of clustering). (Sheet 6) Top markers for each cluster of the integrated dataset after denoising of gene expression data and differential gene expression analysis between wild-type control and GABA<sub>B1</sub>R cKO microglia within cluster 4 of the integrated dataset after denoising. (Sheet 7) Genes enriched in subcluster 4.1 (and downregulated in subcluster 4.2) of the WT control and cKO integrated dataset. (Sheet 8) MERFISH cluster markers and DEGs. (Sheet 9) Summary statistics for scRNA-seq and MERFISH data.</div></div></div></div></section></div></section><div class="core-linked-content">
    

        

</div><section id="backmatter" data-extent="backmatter"><div class="core-container"><section id="references"><h2>References</h2><div id="bibliography" role="doc-bibliography"><div role="list"><div role="listitem" data-has="label"><div id="bib1" class="citations"><div id="sref1" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref1" title="View in article">1.</a></div><div>American Psychiatric Association</div><div><strong>Diagnostic and Statistical Manual of Mental Disorders (DSM-5)</strong></div><div>APA Publishing, 2013</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1176/appi.books.9780890425596" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1176%2Fappi.books.9780890425596" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib2" class="citations"><div id="sref2" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref2-1" title="View in article">2.</a></div><div>Amezquita, R.A. ∙ Lun, A.T.L. ∙ Becht, E. ...</div><div><strong>Orchestrating single-cell analysis with Bioconductor</strong></div><div><em>Nat. Methods.</em> 2020; <strong>17</strong>:137-145</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41592-019-0654-x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_2_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85076035714" target="_blank" aria-label="Open Scopus in new tab">Scopus (325)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31792435/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41592-019-0654-x&amp;pmid=31792435" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib3" class="citations"><div id="sref3" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref3-1" title="View in article">3.</a></div><div>Amir, A.D. ∙ Davis, K.L. ∙ Tadmor, M.D. ...</div><div><strong>viSNE enables visualization of high dimensional single-cell data and reveals phenotypic heterogeneity of leukemia</strong></div><div><em>Nat. Biotechnol.</em> 2013; <strong>31</strong>:545-552</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nbt.2594" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-84880280631" target="_blank" aria-label="Open Scopus in new tab">Scopus (1197)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23685480/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnbt.2594&amp;pmid=23685480" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib4" class="citations"><div id="sref4" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref4-1" title="View in article">4.</a></div><div>Badimon, A. ∙ Strasburger, H.J. ∙ Ayata, P. ...</div><div><strong>Negative feedback control of neuronal activity by microglia</strong></div><div><em>Nature.</em> 2020; <strong>586</strong>:417-423</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-020-2777-8" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85091727125" target="_blank" aria-label="Open Scopus in new tab">Scopus (443)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32999463/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-020-2777-8&amp;pmid=32999463" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib5" class="citations"><div id="sref5" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref5" title="View in article">5.</a></div><div>Bohlen, C.J. ∙ Friedman, B.A. ∙ Dejanovic, B. ...</div><div><strong>Microglia in Brain Development, Homeostasis, and Neurodegeneration</strong></div><div><em>Annu. Rev. Genet.</em> 2019; <strong>53</strong>:263-288</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1146/annurev-genet-112618-043515" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85076062493" target="_blank" aria-label="Open Scopus in new tab">Scopus (107)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31518519/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1146%2Fannurev-genet-112618-043515&amp;pmid=31518519" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib6" class="citations"><div id="sref6" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref6-1" title="View in article">6.</a></div><div>Botto, M. ∙ Dell’Agnola, C. ∙ Bygrave, A.E. ...</div><div><strong>Homozygous C1q deficiency causes glomerulonephritis associated with multiple apoptotic bodies</strong></div><div><em>Nat. Genet.</em> 1998; <strong>19</strong>:56-59</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng0598-56" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-0031778836" target="_blank" aria-label="Open Scopus in new tab">Scopus (1273)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/9590289/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng0598-56&amp;pmid=9590289" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib7" class="citations"><div id="sref7" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref7" title="View in article">7.</a></div><div>Braat, S. ∙ Kooy, R.F.</div><div><strong>The GABAA Receptor as a Therapeutic Target for Neurodevelopmental Disorders</strong></div><div><em>Neuron.</em> 2015; <strong>86</strong>:1119-1130</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.neuron.2015.03.042&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.neuron.2015.03.042&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-84930531864" target="_blank" aria-label="Open Scopus in new tab">Scopus (220)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26050032/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.neuron.2015.03.042&amp;pmid=26050032" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib8" class="citations"><div id="sref8" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref8-1" title="View in article">8.</a></div><div>Cao, J. ∙ Spielmann, M. ∙ Qiu, X. ...</div><div><strong>The single-cell transcriptional landscape of mammalian organogenesis</strong></div><div><em>Nature.</em> 2019; <strong>566</strong>:496-502</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-0969-x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85062419120" target="_blank" aria-label="Open Scopus in new tab">Scopus (1641)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30787437/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-0969-x&amp;pmid=30787437" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib9" class="citations"><div id="sref9" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref9" title="View in article">9.</a></div><div>Cellot, G. ∙ Cherubini, E.</div><div><strong>Functional role of ambient GABA in refining neuronal circuits early in postnatal development</strong></div><div><em>Front. Neural Circuits.</em> 2013; <strong>7</strong>:136</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.3389/fncir.2013.00136" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-84882684678" target="_blank" aria-label="Open Scopus in new tab">Scopus (75)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23964205/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.3389%2Ffncir.2013.00136&amp;pmid=23964205" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib10" class="citations"><div id="sref10" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref10" title="View in article">10.</a></div><div>Charrin, S. ∙ Jouannet, S. ∙ Boucheix, C. ...</div><div><strong>Tetraspanins at a glance</strong></div><div><em>J.&nbsp;Cell Sci.</em> 2014; <strong>127</strong>:3641-3648</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1242/jcs.154906" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-84906875545" target="_blank" aria-label="Open Scopus in new tab">Scopus (327)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25128561/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1242%2Fjcs.154906&amp;pmid=25128561" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib11" class="citations"><div id="sref11" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref11" title="View in article">11.</a></div><div>Cheadle, L. ∙ Rivera, S.A. ∙ Phelps, J.S. ...</div><div><strong>Sensory Experience Engages Microglia to Shape Neural Connectivity through a Non-Phagocytic Mechanism</strong></div><div><em>Neuron.</em> 2020; <strong>108</strong>:451-468.e9</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.neuron.2020.08.002&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.neuron.2020.08.002&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85091688509" target="_blank" aria-label="Open Scopus in new tab">Scopus (95)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32931754/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.neuron.2020.08.002&amp;pmid=32931754" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib12" class="citations"><div id="sref12" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref12" title="View in article">12.</a></div><div>Chen, Z. ∙ Jalabi, W. ∙ Hu, W. ...</div><div><strong>Microglial displacement of inhibitory synapses provides neuroprotection in the adult brain</strong></div><div><em>Nat. Commun.</em> 2014; <strong>5</strong>:4486</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ncomms5486" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-84919455244" target="_blank" aria-label="Open Scopus in new tab">Scopus (218)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25047355/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fncomms5486&amp;pmid=25047355" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib13" class="citations"><div id="sref13" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref13" title="View in article">13.</a></div><div>Chen, J.A. ∙ Peñagarikano, O. ∙ Belgard, T.G. ...</div><div><strong>The emerging picture of autism spectrum disorder: genetics and pathology</strong></div><div><em>Annu. Rev. Pathol.</em> 2015; <strong>10</strong>:111-144</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1146/annurev-pathol-012414-040405" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_13_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-84921917284" target="_blank" aria-label="Open Scopus in new tab">Scopus (196)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25621659/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1146%2Fannurev-pathol-012414-040405&amp;pmid=25621659" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib14" class="citations"><div id="sref14" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref14-1" title="View in article">14.</a></div><div>Chen, K.H. ∙ Boettiger, A.N. ∙ Moffitt, J.R. ...</div><div><strong>RNA imaging. Spatially resolved, highly multiplexed RNA profiling in single cells</strong></div><div><em>Science.</em> 2015; <strong>348</strong>:aaa6090</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aaa6090" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-84928395184" target="_blank" aria-label="Open Scopus in new tab">Scopus (1274)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25858977/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aaa6090&amp;pmid=25858977" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib91" class="citations"><div id="optuTeD7tznjW" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-optPooQj0tqrk" title="View in article">15.</a></div><div>Crapser, D., J. ∙ Ochaba, J. ∙ Soni, N. ...</div><div><strong>Microglial depletion prevents extracellular matrix changes and striatal volume reduction in a model of Huntington’s disease.</strong></div><div><em>Brain.</em> 2020; <strong>143</strong>:266-288</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/brain/awz363" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85077346689" target="_blank" aria-label="Open Scopus in new tab">Scopus (78)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31848580/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbrain%2Fawz363&amp;pmid=31848580" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib15" class="citations"><div id="sref15" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref15-1" title="View in article">16.</a></div><div>Csardi, G. ∙ Nepusz, T.</div><div><strong>The igraph software package for complex network research</strong></div><div><em>InterJournal Complex Syst.</em> 2006; <strong>1695</strong>:1-9</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=G.CsardiT.NepuszThe+igraph+software+package+for+complex+network+researchInterJournal+Complex+Syst.1695200619" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib16" class="citations"><div id="sref16" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref16-1" title="View in article">17.</a></div><div>Cserép, C. ∙ Pósfai, B. ∙ Lénárt, N. ...</div><div><strong>Microglia monitor and protect neuronal function through specialized somatic purinergic junctions</strong></div><div><em>Science.</em> 2020; <strong>367</strong>:528-537</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aax6752" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85078742646" target="_blank" aria-label="Open Scopus in new tab">Scopus (330)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31831638/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aax6752&amp;pmid=31831638" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib17" class="citations"><div id="sref17" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref17" title="View in article">18.</a></div><div>Cubillo, A. ∙ Halari, R. ∙ Smith, A. ...</div><div><strong>A review of fronto-striatal and fronto-cortical brain abnormalities in children and adults with Attention Deficit Hyperactivity Disorder (ADHD) and new evidence for dysfunction in adults with ADHD during motivation and attention</strong></div><div><em>Cortex.</em> 2012; <strong>48</strong>:194-215</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.cortex.2011.04.007&amp;cf=fulltext" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.cortex.2011.04.007&amp;cf=pdf" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-84856228224" target="_blank" aria-label="Open Scopus in new tab">Scopus (367)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21575934/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cortex.2011.04.007&amp;pmid=21575934" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib18" class="citations"><div id="sref18" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref18" title="View in article">19.</a></div><div>Deczkowska, A. ∙ Keren-Shaul, H. ∙ Weiner, A. ...</div><div><strong>Disease-Associated Microglia: A Universal Immune Sensor of Neurodegeneration</strong></div><div><em>Cell.</em> 2018; <strong>173</strong>:1073-1081</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.cell.2018.05.003&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.cell.2018.05.003&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85046853468" target="_blank" aria-label="Open Scopus in new tab">Scopus (663)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29775591/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2018.05.003&amp;pmid=29775591" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib19" class="citations"><div id="sref19" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref19" title="View in article">20.</a></div><div>Doolen, S. ∙ Cook, J. ∙ Riedl, M. ...</div><div><strong>Complement 3a receptor in dorsal horn microglia mediates pronociceptive neuropeptide signaling</strong></div><div><em>Glia.</em> 2017; <strong>65</strong>:1976-1989</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/glia.23208" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85031504789" target="_blank" aria-label="Open Scopus in new tab">Scopus (28)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28850719/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fglia.23208&amp;pmid=28850719" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib20" class="citations"><div id="sref20" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref20" title="View in article">21.</a></div><div>Edden, R.A.E. ∙ Crocetti, D. ∙ Zhu, H. ...</div><div><strong>Reduced GABA concentration in attention-deficit/hyperactivity disorder</strong></div><div><em>Arch. Gen. Psychiatry.</em> 2012; <strong>69</strong>:750-753</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1001/archgenpsychiatry.2011.2280" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-84863545490" target="_blank" aria-label="Open Scopus in new tab">Scopus (178)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22752239/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1001%2Farchgenpsychiatry.2011.2280&amp;pmid=22752239" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib21" class="citations"><div id="sref21" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref21-1" title="View in article">22.</a></div><div>Emanuel, G. ∙ Babcock, H.</div><div><strong>ZhuangLab/MERlin: MERlin v0.1.6</strong></div><div>Zenodo, 2020</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=G.EmanuelH.BabcockZhuangLab%2FMERlin%3A+MERlin+v0.1.62020Zenodo" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib22" class="citations"><div id="sref22" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref22" title="View in article">23.</a></div><div>Favuzzi, E. ∙ Rico, B.</div><div><strong>Molecular diversity underlying cortical excitatory and inhibitory synapse development</strong></div><div><em>Curr. Opin. Neurobiol.</em> 2018; <strong>53</strong>:8-15</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.conb.2018.03.011" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_23_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85046459653" target="_blank" aria-label="Open Scopus in new tab">Scopus (27)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29704699/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.conb.2018.03.011&amp;pmid=29704699" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib23" class="citations"><div id="sref23" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref23" title="View in article">24.</a></div><div>Favuzzi, E. ∙ Marques-Smith, A. ∙ Deogracias, R. ...</div><div><strong>Activity-Dependent Gating of Parvalbumin Interneuron Function by the Perineuronal Net Protein Brevican</strong></div><div><em>Neuron.</em> 2017; <strong>95</strong>:639-655.e10</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.neuron.2017.06.028&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.neuron.2017.06.028&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85023623383" target="_blank" aria-label="Open Scopus in new tab">Scopus (224)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28712654/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.neuron.2017.06.028&amp;pmid=28712654" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib24" class="citations"><div id="sref24" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref24-1" title="View in article">25.</a></div><div>Favuzzi, E. ∙ Deogracias, R. ∙ Marques-Smith, A. ...</div><div><strong>Distinct molecular programs regulate synapse specificity in cortical inhibitory circuits</strong></div><div><em>Science.</em> 2019; <strong>363</strong>:413-417</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aau8977" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85060511221" target="_blank" aria-label="Open Scopus in new tab">Scopus (110)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30679375/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aau8977&amp;pmid=30679375" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib25" class="citations"><div id="sref25" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref25-1" title="View in article">26.</a></div><div>Filipello, F. ∙ Morini, R. ∙ Corradini, I. ...</div><div><strong>The Microglial Innate Immune Receptor TREM2 Is Required for Synapse Elimination and Normal Brain Connectivity</strong></div><div><em>Immunity.</em> 2018; <strong>48</strong>:979-991.e8</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_26_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.immuni.2018.04.016&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_26_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.immuni.2018.04.016&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_26_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85046738379" target="_blank" aria-label="Open Scopus in new tab">Scopus (391)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29752066/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.immuni.2018.04.016&amp;pmid=29752066" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib26" class="citations"><div id="sref26" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref26" title="View in article">27.</a></div><div>Fontainhas, A.M. ∙ Wang, M. ∙ Liang, K.J. ...</div><div><strong>Microglial morphology and dynamic behavior is regulated by ionotropic glutamatergic and GABAergic neurotransmission</strong></div><div><em>PLoS ONE.</em> 2011; <strong>6</strong>:e15973</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pone.0015973" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-79551570269" target="_blank" aria-label="Open Scopus in new tab">Scopus (270)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21283568/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pone.0015973&amp;pmid=21283568" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib27" class="citations"><div id="sref27" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref27-1" title="View in article">28.</a></div><div>Gaiarsa, J.-L. ∙ Porcher, C.</div><div><strong>Emerging neurotrophic role of GABAB receptors in neuronal circuit development</strong></div><div><em>Front. Cell. Neurosci.</em> 2013; <strong>7</strong>:206</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.3389/fncel.2013.00206" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-84888155744" target="_blank" aria-label="Open Scopus in new tab">Scopus (44)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24282395/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.3389%2Ffncel.2013.00206&amp;pmid=24282395" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib28" class="citations"><div id="sref28" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref28" title="View in article">29.</a></div><div>Gehman, L.T. ∙ Stoilov, P. ∙ Maguire, J. ...</div><div><strong>The splicing regulator Rbfox1 (A2BP1) controls neuronal excitation in the mammalian brain</strong></div><div><em>Nat. Genet.</em> 2011; <strong>43</strong>:706-711</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng.841" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-79959726422" target="_blank" aria-label="Open Scopus in new tab">Scopus (259)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21623373/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng.841&amp;pmid=21623373" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib29" class="citations"><div id="sref29" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref29" title="View in article">30.</a></div><div>Gilbert, D.L. ∙ Isaacs, K.M. ∙ Augusta, M. ...</div><div><strong>Motor cortex inhibition: a marker of ADHD behavior and motor development in children</strong></div><div><em>Neurology.</em> 2011; <strong>76</strong>:615-621</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1212/WNL.0b013e31820c2ebd" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-79951592190" target="_blank" aria-label="Open Scopus in new tab">Scopus (138)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21321335/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1212%2FWNL.0b013e31820c2ebd&amp;pmid=21321335" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib30" class="citations"><div id="sref30" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref30-1" title="View in article">31.</a></div><div>Haller, C. ∙ Casanova, E. ∙ Müller, M. ...</div><div><strong>Floxed allele for conditional inactivation of the GABAB(1) gene</strong></div><div><em>Genesis.</em> 2004; <strong>40</strong>:125-130</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/gene.20073" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-10344244007" target="_blank" aria-label="Open Scopus in new tab">Scopus (45)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15493018/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fgene.20073&amp;pmid=15493018" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib31" class="citations"><div id="sref31" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref31-1" title="View in article">32.</a></div><div>Hammond, T.R. ∙ Dufort, C. ∙ Dissing-Olesen, L. ...</div><div><strong>Single-Cell RNA Sequencing of Microglia throughout the Mouse Lifespan and in the Injured Brain Reveals Complex Cell-State Changes</strong></div><div><em>Immunity.</em> 2019; <strong>50</strong>:253-271.e6</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.immuni.2018.11.004&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.immuni.2018.11.004&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85056738677" target="_blank" aria-label="Open Scopus in new tab">Scopus (1123)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30471926/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.immuni.2018.11.004&amp;pmid=30471926" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib32" class="citations"><div id="sref32" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref32" title="View in article">33.</a></div><div>Jones, K.A. ∙ Borowsky, B. ∙ Tamm, J.A. ...</div><div><strong>GABA(B) receptors function as a heteromeric assembly of the subunits GABA(B)R1 and GABA(B)R2</strong></div><div><em>Nature.</em> 1998; <strong>396</strong>:674-679</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/25348" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-0005169503" target="_blank" aria-label="Open Scopus in new tab">Scopus (942)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/9872315/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2F25348&amp;pmid=9872315" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib33" class="citations"><div id="sref33" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref33-1" title="View in article">34.</a></div><div>Kaiser, T. ∙ Feng, G.</div><div><strong>Tmem119-EGFP and Tmem119-CreERT2 Transgenic Mice for Labeling and Manipulating Microglia</strong></div><div><em>eNeuro.</em> 2019; <strong>6</strong></div><div>ENEURO.0448-18.2019</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1523/ENEURO.0448-18.2019" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85071714639" target="_blank" aria-label="Open Scopus in new tab">Scopus (136)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31371457/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1523%2FENEURO.0448-18.2019&amp;pmid=31371457" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib34" class="citations"><div id="sref34" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref34-1" title="View in article">35.</a></div><div>Katayama, H. ∙ Yamamoto, A. ∙ Mizushima, N. ...</div><div><strong>GFP-like proteins stably accumulate in lysosomes</strong></div><div><em>Cell Struct. Funct.</em> 2008; <strong>33</strong>:1-12</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1247/csf.07011" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-44949131031" target="_blank" aria-label="Open Scopus in new tab">Scopus (179)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/18256512/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1247%2Fcsf.07011&amp;pmid=18256512" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib35" class="citations"><div id="sref35" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref35-1" title="View in article">36.</a></div><div>Keren-Shaul, H. ∙ Spinrad, A. ∙ Weiner, A. ...</div><div><strong>A Unique Microglia Type Associated with Restricting Development of Alzheimer’s Disease</strong></div><div><em>Cell.</em> 2017; <strong>169</strong>:1276-1290.e17</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.cell.2017.05.018&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.cell.2017.05.018&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85020287843" target="_blank" aria-label="Open Scopus in new tab">Scopus (2741)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28602351/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2017.05.018&amp;pmid=28602351" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib36" class="citations"><div id="sref36" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref36" title="View in article">37.</a></div><div>Krasemann, S. ∙ Madore, C. ∙ Cialic, R. ...</div><div><strong>The TREM2-APOE pathway drives the transcriptional phenotype of dysfunctional microglia in neurodegenerative diseases</strong></div><div><em>Immunity.</em> 2017; <strong>47</strong>:566-581.e9</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.immuni.2017.08.008&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.immuni.2017.08.008&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85031732784" target="_blank" aria-label="Open Scopus in new tab">Scopus (1491)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28930663/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.immuni.2017.08.008&amp;pmid=28930663" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib37" class="citations"><div id="sref37" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref37-1" title="View in article">38.</a></div><div>Kuhn, S.A. ∙ van Landeghem, F.K.H. ∙ Zacharias, R. ...</div><div><strong>Microglia express GABA(B) receptors to modulate interleukin release</strong></div><div><em>Mol. Cell. Neurosci.</em> 2004; <strong>25</strong>:312-322</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.mcn.2003.10.023" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_38_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-1542616888" target="_blank" aria-label="Open Scopus in new tab">Scopus (168)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15019947/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.mcn.2003.10.023&amp;pmid=15019947" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib38" class="citations"><div id="sref38" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref38-1" title="View in article">39.</a></div><div>Li, Q. ∙ Cheng, Z. ∙ Zhou, L. ...</div><div><strong>Developmental Heterogeneity of Microglia and Brain Myeloid Cells Revealed by Deep Single-Cell RNA Sequencing</strong></div><div><em>Neuron.</em> 2019; <strong>101</strong>:207-223.e10</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.neuron.2018.12.006&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.neuron.2018.12.006&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85059299323" target="_blank" aria-label="Open Scopus in new tab">Scopus (572)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30606613/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.neuron.2018.12.006&amp;pmid=30606613" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib39" class="citations"><div id="sref39" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref39" title="View in article">40.</a></div><div>Liu, Y.U. ∙ Ying, Y. ∙ Li, Y. ...</div><div><strong>Neuronal network activity controls microglial process surveillance in awake mice via norepinephrine signaling</strong></div><div><em>Nat. Neurosci.</em> 2019; <strong>22</strong>:1771-1781</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41593-019-0511-3" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_40_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85074203384" target="_blank" aria-label="Open Scopus in new tab">Scopus (201)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31636449/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41593-019-0511-3&amp;pmid=31636449" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib40" class="citations"><div id="sref40" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref40" title="View in article">41.</a></div><div>Liu, Y.-J. ∙ Spangenberg, E. ∙ Tang, B. ...</div><div><strong>Microglia elimination increases neural circuit connectivity and activity in adult mouse cortex</strong></div><div><em>J.&nbsp;Neurosci.</em> 2021; <strong>41</strong>:1274-1287</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1523/JNEUROSCI.2140-20.2020" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85102093729" target="_blank" aria-label="Open Scopus in new tab">Scopus (74)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33380470/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1523%2FJNEUROSCI.2140-20.2020&amp;pmid=33380470" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib41" class="citations"><div id="sref41" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref41" title="View in article">42.</a></div><div>Lui, H. ∙ Zhang, J. ∙ Makinson, S.R. ...</div><div><strong>Progranulin Deficiency Promotes Circuit-Specific Synaptic Pruning by Microglia via Complement Activation</strong></div><div><em>Cell.</em> 2016; <strong>165</strong>:921-935</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.cell.2016.04.001&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.cell.2016.04.001&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-84963969178" target="_blank" aria-label="Open Scopus in new tab">Scopus (490)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27114033/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2016.04.001&amp;pmid=27114033" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib42" class="citations"><div id="sref42" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref42" title="View in article">43.</a></div><div>Lun, A.T. ∙ Bach, K. ∙ Marioni, J.C.</div><div><strong>Pooling across cells to normalize single-cell RNA sequencing data with many zero counts</strong></div><div><em>Genome Biol.</em> 2016; <strong>17</strong>:75</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13059-016-0947-7" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-84964556059" target="_blank" aria-label="Open Scopus in new tab">Scopus (629)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27122128/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13059-016-0947-7&amp;pmid=27122128" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib43" class="citations"><div id="sref43" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref43-1" title="View in article">44.</a></div><div>Lun, A.T.L. ∙ McCarthy, D.J. ∙ Marioni, J.C.</div><div><strong>A step-by-step workflow for low-level analysis of single-cell RNA-seq data with Bioconductor</strong></div><div><em>F1000Res.</em> 2016; <strong>5</strong>:2122</div></div><div class="external-links"><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27909575/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?pmid=27909575" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib44" class="citations"><div id="sref44" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref44" title="View in article">45.</a></div><div>Madisen, L. ∙ Mao, T. ∙ Koch, H. ...</div><div><strong>A toolbox of Cre-dependent optogenetic transgenic mice for light-induced activation and silencing</strong></div><div><em>Nat. Neurosci.</em> 2012; <strong>15</strong>:793-802</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nn.3078" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_45_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-84860294289" target="_blank" aria-label="Open Scopus in new tab">Scopus (925)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22446880/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnn.3078&amp;pmid=22446880" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib45" class="citations"><div id="sref45" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref45" title="View in article">46.</a></div><div>Madry, C. ∙ Attwell, D.</div><div><strong>Receptors, ion channels, and signaling mechanisms underlying microglial dynamics</strong></div><div><em>J.&nbsp;Biol. Chem.</em> 2015; <strong>290</strong>:12443-12450</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1074%2Fjbc.R115.637157&amp;cf=fulltext&amp;site=jbc-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1074%2Fjbc.R115.637157&amp;cf=pdf&amp;site=jbc-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-84929378470" target="_blank" aria-label="Open Scopus in new tab">Scopus (65)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25855789/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1074%2Fjbc.R115.637157&amp;pmid=25855789" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib46" class="citations"><div id="sref46" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref46-1" title="View in article">47.</a></div><div>Markowitz, J.E. ∙ Gillis, W.F. ∙ Beron, C.C. ...</div><div><strong>The Striatum Organizes 3D Behavior via Moment-to-Moment Action Selection</strong></div><div><em>Cell.</em> 2018; <strong>174</strong>:44-58.e17</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.cell.2018.04.019&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.cell.2018.04.019&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85046871902" target="_blank" aria-label="Open Scopus in new tab">Scopus (193)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29779950/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2018.04.019&amp;pmid=29779950" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib47" class="citations"><div id="sref47" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref47-1" title="View in article">48.</a></div><div>Masuda, T. ∙ Sankowski, R. ∙ Staszewski, O. ...</div><div><strong>Spatial and temporal heterogeneity of mouse and human microglia at single-cell resolution</strong></div><div><em>Nature.</em> 2019; <strong>566</strong>:388-392</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-0924-x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_48_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85061969567" target="_blank" aria-label="Open Scopus in new tab">Scopus (735)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30760929/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-0924-x&amp;pmid=30760929" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib48" class="citations"><div id="sref48" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref48-1" title="View in article">49.</a></div><div>Matcovitch-Natan, O. ∙ Winter, D.R. ∙ Giladi, A. ...</div><div><strong>Microglia development follows a stepwise program to regulate brain homeostasis</strong></div><div><em>Science.</em> 2016; <strong>353</strong>:aad8670</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aad8670" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-84976883162" target="_blank" aria-label="Open Scopus in new tab">Scopus (792)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27338705/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aad8670&amp;pmid=27338705" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib49" class="citations"><div id="sref49" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref49-1" title="View in article">50.</a></div><div>McCarthy, D.J. ∙ Campbell, K.R. ∙ Lun, A.T.L. ...</div><div><strong>Scater: pre-processing, quality control, normalization and visualization of single-cell RNA-seq data in R</strong></div><div><em>Bioinformatics.</em> 2017; <strong>33</strong>:1179-1186</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btw777" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85019072518" target="_blank" aria-label="Open Scopus in new tab">Scopus (861)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28088763/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtw777&amp;pmid=28088763" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib50" class="citations"><div id="sref50" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref50" title="View in article">51.</a></div><div>Mederos, S. ∙ Perea, G.</div><div><strong>GABAergic-astrocyte signaling: A refinement of inhibitory brain networks</strong></div><div><em>Glia.</em> 2019; <strong>67</strong>:1842-1851</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/glia.23644" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_51_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85066497252" target="_blank" aria-label="Open Scopus in new tab">Scopus (67)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31145508/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fglia.23644&amp;pmid=31145508" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib51" class="citations"><div id="sref51" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref51-1" title="View in article">52.</a></div><div>Moffitt, J.R. ∙ Bambah-Mukku, D. ∙ Eichhorn, S.W. ...</div><div><strong>Molecular, spatial, and functional single-cell profiling of the hypothalamic preoptic region</strong></div><div><em>Science.</em> 2018; <strong>362</strong>:eaau5324</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aau5324" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85056571109" target="_blank" aria-label="Open Scopus in new tab">Scopus (576)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30385464/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aau5324&amp;pmid=30385464" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib52" class="citations"><div id="sref52" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref52-1" title="View in article">53.</a></div><div>Nagai, J. ∙ Rajbhandari, A.K. ∙ Gangwani, M.R. ...</div><div><strong>Hyperactivity with Disrupted Attention by Activation of an Astrocyte Synaptogenic Cue</strong></div><div><em>Cell.</em> 2019; <strong>177</strong>:1280-1292.e20</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.cell.2019.03.019&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.cell.2019.03.019&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85065073576" target="_blank" aria-label="Open Scopus in new tab">Scopus (190)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31031006/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2019.03.019&amp;pmid=31031006" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib53" class="citations"><div id="sref53" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref53-1" title="View in article">54.</a></div><div>Neniskyte, U. ∙ Gross, C.T.</div><div><strong>Errant gardeners: glial-cell-dependent synaptic pruning and neurodevelopmental disorders</strong></div><div><em>Nat. Rev. Neurosci.</em> 2017; <strong>18</strong>:658-670</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nrn.2017.110" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85031751357" target="_blank" aria-label="Open Scopus in new tab">Scopus (194)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28931944/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnrn.2017.110&amp;pmid=28931944" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib90" class="citations"><div id="optPooQj0tqrk" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-optPooQj0tqrk" title="View in article">55.</a></div><div>Nguyen, T., P. ∙ Dorman, C., L. ∙ Pan ...</div><div><strong>Microglial Remodeling of the Extracellular Matrix Promotes Synapse Plasticity</strong></div><div><em>Cell.</em> 2020; <strong>182</strong>:388-403</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.cell.2020.05.050&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.cell.2020.05.050&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85087989807" target="_blank" aria-label="Open Scopus in new tab">Scopus (298)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32615087/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2020.05.050&amp;pmid=32615087" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib54" class="citations"><div id="sref54" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref54" title="View in article">56.</a></div><div>Oh, W.C. ∙ Lutzu, S. ∙ Castillo, P.E. ...</div><div><strong>De novo synaptogenesis induced by GABA in the developing mouse cortex</strong></div><div><em>Science.</em> 2016; <strong>353</strong>:1037-1040</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aaf5206" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_56_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-84983296823" target="_blank" aria-label="Open Scopus in new tab">Scopus (135)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27516412/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aaf5206&amp;pmid=27516412" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib55" class="citations"><div id="sref55" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref55" title="View in article">57.</a></div><div>Paolicelli, R.C. ∙ Bolasco, G. ∙ Pagani, F. ...</div><div><strong>Synaptic pruning by microglia is necessary for normal brain development</strong></div><div><em>Science.</em> 2011; <strong>333</strong>:1456-1458</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.1202529" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-80052633284" target="_blank" aria-label="Open Scopus in new tab">Scopus (2844)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21778362/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.1202529&amp;pmid=21778362" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib56" class="citations"><div id="sref56" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref56" title="View in article">58.</a></div><div>Parslow, A. ∙ Cardona, A. ∙ Bryson-Richardson, R.J.</div><div><strong>Sample Drift Correction Following 4D Confocal Time-lapse Imaging</strong></div><div><em>J.&nbsp;Vis. Exp.</em> 2014; 51086</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.ParslowA.CardonaR.J.Bryson-RichardsonSample+Drift+Correction+Following+4D+Confocal+Time-lapse+ImagingJ.%C2%A0Vis.+Exp.86201451086" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib57" class="citations"><div id="sref57" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref57" title="View in article">59.</a></div><div>Pinto, J.G.A. ∙ Jones, D.G. ∙ Murphy, K.M.</div><div><strong>Comparing development of synaptic proteins in rat visual, somatosensory, and frontal cortex</strong></div><div><em>Front. Neural Circuits.</em> 2013; <strong>7</strong>:97</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.3389/fncir.2013.00097" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-84878252546" target="_blank" aria-label="Open Scopus in new tab">Scopus (38)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23754984/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.3389%2Ffncir.2013.00097&amp;pmid=23754984" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib58" class="citations"><div id="sref58" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref58" title="View in article">60.</a></div><div>Pocock, J.M. ∙ Kettenmann, H.</div><div><strong>Neurotransmitter receptors on microglia</strong></div><div><em>Trends Neurosci.</em> 2007; <strong>30</strong>:527-535</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.tins.2007.07.007&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.tins.2007.07.007&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-34848843543" target="_blank" aria-label="Open Scopus in new tab">Scopus (526)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17904651/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.tins.2007.07.007&amp;pmid=17904651" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib59" class="citations"><div id="sref59" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref59" title="View in article">61.</a></div><div>Polański, K. ∙ Young, M.D. ∙ Miao, Z. ...</div><div><strong>BBKNN: fast batch alignment of single cell transcriptomes</strong></div><div><em>Bioinformatics.</em> 2020; <strong>36</strong>:964-965</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btz625" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85074670646" target="_blank" aria-label="Open Scopus in new tab">Scopus (337)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31400197/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtz625&amp;pmid=31400197" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib60" class="citations"><div id="sref60" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref60" title="View in article">62.</a></div><div>Racine, R.J.</div><div><strong>Modification of seizure activity by electrical stimulation. II. Motor seizure</strong></div><div><em>Electroencephalogr. Clin. Neurophysiol.</em> 1972; <strong>32</strong>:281-294</div></div><div class="external-links"><div class="core-xlink-abstract"><a href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2F0013-4694%2872%2990177-0&amp;cf=abstract" target="_blank" aria-label="Open Abstract in new tab">Abstract</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2F0013-4694%2872%2990177-0&amp;cf=pdf" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-0015309163" target="_blank" aria-label="Open Scopus in new tab">Scopus (6039)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/4110397/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2F0013-4694%2872%2990177-0&amp;pmid=4110397" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib61" class="citations"><div id="sref61" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref61" title="View in article">63.</a></div><div>Ransohoff, R.M. ∙ Perry, V.H.</div><div><strong>Microglial physiology: unique stimuli, specialized responses</strong></div><div><em>Annu. Rev. Immunol.</em> 2009; <strong>27</strong>:119-145</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1146/annurev.immunol.021908.132528" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-67650966680" target="_blank" aria-label="Open Scopus in new tab">Scopus (1455)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19302036/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1146%2Fannurev.immunol.021908.132528&amp;pmid=19302036" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib62" class="citations"><div id="sref62" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref62-1" title="View in article">64.</a></div><div>Satija, R. ∙ Farrell, J.A. ∙ Gennert, D. ...</div><div><strong>Spatial reconstruction of single-cell gene expression data</strong></div><div><em>Nat. Biotechnol.</em> 2015; <strong>33</strong>:495-502</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nbt.3192" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_64_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-84929151009" target="_blank" aria-label="Open Scopus in new tab">Scopus (3012)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25867923/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnbt.3192&amp;pmid=25867923" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib63" class="citations"><div id="sref63" class="citation"><div class="citation-content"><div class="label"><a href="#figs3" title="View in article">65.</a></div><div>Saunders, A. ∙ Macosko, E.Z. ∙ Wysoker, A. ...</div><div><strong>Molecular Diversity and Specializations among the Cells of the Adult Mouse Brain</strong></div><div><em>Cell.</em> 2018; <strong>174</strong>:1015-1030.e16</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.cell.2018.07.028&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.cell.2018.07.028&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85050885385" target="_blank" aria-label="Open Scopus in new tab">Scopus (875)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30096299/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2018.07.028&amp;pmid=30096299" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib64" class="citations"><div id="sref64" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref64-1" title="View in article">66.</a></div><div>Schafer, D.P. ∙ Lehrman, E.K. ∙ Kautzman, A.G. ...</div><div><strong>Microglia sculpt postnatal neural circuits in an activity and complement-dependent manner</strong></div><div><em>Neuron.</em> 2012; <strong>74</strong>:691-705</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.neuron.2012.03.026&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.neuron.2012.03.026&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-84861427387" target="_blank" aria-label="Open Scopus in new tab">Scopus (2664)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22632727/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.neuron.2012.03.026&amp;pmid=22632727" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib65" class="citations"><div id="sref65" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref65-1" title="View in article">67.</a></div><div>Schneider, C.A. ∙ Rasband, W.S. ∙ Eliceiri, K.W.</div><div><strong>NIH Image to ImageJ: 25 years of image analysis</strong></div><div><em>Nat. Methods.</em> 2012; <strong>9</strong>:671-675</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.2089" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_67_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-84863205849" target="_blank" aria-label="Open Scopus in new tab">Scopus (42378)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22930834/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.2089&amp;pmid=22930834" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib66" class="citations"><div id="sref66" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref66-1" title="View in article">68.</a></div><div>Shinoda, H. ∙ Ma, Y. ∙ Nakashima, R. ...</div><div><strong>Acid-Tolerant Monomeric GFP from Olindias formosa</strong></div><div><em>Cell Chem. Biol.</em> 2018; <strong>25</strong>:330-338.e7</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.chembiol.2017.12.005&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.chembiol.2017.12.005&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85042356070" target="_blank" aria-label="Open Scopus in new tab">Scopus (53)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29290624/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.chembiol.2017.12.005&amp;pmid=29290624" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib67" class="citations"><div id="sref67" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref67-1" title="View in article">69.</a></div><div>Shinoda, H. ∙ Shannon, M. ∙ Nagai, T.</div><div><strong>Fluorescent Proteins for Investigating Biological Events in Acidic Environments</strong></div><div><em>Int. J. Mol. Sci.</em> 2018; <strong>19</strong>:1548</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.3390/ijms19061548" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85047554713" target="_blank" aria-label="Open Scopus in new tab">Scopus (67)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29789517/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.3390%2Fijms19061548&amp;pmid=29789517" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib68" class="citations"><div id="sref68" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref68" title="View in article">70.</a></div><div>Sohal, V.S. ∙ Rubenstein, J.L.R.</div><div><strong>Excitation-inhibition balance as a framework for investigating mechanisms in neuropsychiatric disorders</strong></div><div><em>Mol. Psychiatry.</em> 2019; <strong>24</strong>:1248-1257</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41380-019-0426-0" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85065920522" target="_blank" aria-label="Open Scopus in new tab">Scopus (424)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31089192/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41380-019-0426-0&amp;pmid=31089192" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib69" class="citations"><div id="sref69" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref69-1" title="View in article">71.</a></div><div>Spangenberg, E. ∙ Severson, P.L. ∙ Hohsfield, L.A. ...</div><div><strong>Sustained microglial depletion with CSF1R inhibitor impairs parenchymal plaque development in an Alzheimer’s disease model</strong></div><div><em>Nat. Commun.</em> 2019; <strong>10</strong>:3758</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41467-019-11674-z" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85071120834" target="_blank" aria-label="Open Scopus in new tab">Scopus (418)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31434879/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41467-019-11674-z&amp;pmid=31434879" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib70" class="citations"><div id="sref70" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref70" title="View in article">72.</a></div><div>Squarzoni, P. ∙ Oller, G. ∙ Hoeffel, G. ...</div><div><strong>Microglia modulate wiring of the embryonic forebrain</strong></div><div><em>Cell Rep.</em> 2014; <strong>8</strong>:1271-1279</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.celrep.2014.07.042&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.celrep.2014.07.042&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-84922519429" target="_blank" aria-label="Open Scopus in new tab">Scopus (475)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25159150/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.celrep.2014.07.042&amp;pmid=25159150" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib71" class="citations"><div id="sref71" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref71" title="View in article">73.</a></div><div>Stark, C. ∙ Breitkreutz, B.-J. ∙ Reguly, T. ...</div><div><strong>BioGRID: a general repository for interaction datasets</strong></div><div><em>Nucleic Acids Res.</em> 2006; <strong>34</strong>:D535-D539</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gkj109" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-33644873184" target="_blank" aria-label="Open Scopus in new tab">Scopus (2967)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16381927/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgkj109&amp;pmid=16381927" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib72" class="citations"><div id="sref72" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref72-1" title="View in article">74.</a></div><div>Stevens, B. ∙ Allen, N.J. ∙ Vazquez, L.E. ...</div><div><strong>The classical complement cascade mediates CNS synapse elimination</strong></div><div><em>Cell.</em> 2007; <strong>131</strong>:1164-1178</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.cell.2007.10.036&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.cell.2007.10.036&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-36849076770" target="_blank" aria-label="Open Scopus in new tab">Scopus (2260)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/18083105/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2007.10.036&amp;pmid=18083105" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib73" class="citations"><div id="sref73" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref73" title="View in article">75.</a></div><div>Stowell, R.D. ∙ Sipe, G.O. ∙ Dawes, R.P. ...</div><div><strong>Noradrenergic signaling in the wakeful state inhibits microglial surveillance and synaptic plasticity in the mouse visual cortex</strong></div><div><em>Nat. Neurosci.</em> 2019; <strong>22</strong>:1782-1792</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41593-019-0514-0" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_75_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85074208801" target="_blank" aria-label="Open Scopus in new tab">Scopus (179)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31636451/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41593-019-0514-0&amp;pmid=31636451" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib74" class="citations"><div id="sref74" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref74" title="View in article">76.</a></div><div>The UniProt Consortium</div><div><strong>UniProt: the universal protein knowledgebase</strong></div><div><em>Nucleic Acids Res.</em> 2017; <strong>45</strong>:D158-D169</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gkw1099" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_76_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85016153986" target="_blank" aria-label="Open Scopus in new tab">Scopus (3396)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27899622/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgkw1099&amp;pmid=27899622" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib75" class="citations"><div id="sref75" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref75" title="View in article">77.</a></div><div>Thion, M.S. ∙ Ginhoux, F. ∙ Garel, S.</div><div><strong>Microglia and early brain development: An intimate journey</strong></div><div><em>Science.</em> 2018; <strong>362</strong>:185-189</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aat0474" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85054779519" target="_blank" aria-label="Open Scopus in new tab">Scopus (233)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30309946/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aat0474&amp;pmid=30309946" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib76" class="citations"><div id="sref76" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref76-1" title="View in article">78.</a></div><div>Thion, M.S. ∙ Mosser, C.-A. ∙ Férézou, I. ...</div><div><strong>Biphasic Impact of Prenatal Inflammation and Macrophage Depletion on the Wiring of Neocortical Inhibitory Circuits</strong></div><div><em>Cell Rep.</em> 2019; <strong>28</strong>:1119-1126.e4</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_78_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.celrep.2019.06.086&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_78_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.celrep.2019.06.086&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_78_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85069572402" target="_blank" aria-label="Open Scopus in new tab">Scopus (35)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31365857/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.celrep.2019.06.086&amp;pmid=31365857" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib77" class="citations"><div id="sref77" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref77-1" title="View in article">79.</a></div><div>Ting, J.T. ∙ Lee, B.R. ∙ Chong, P. ...</div><div><strong>Preparation of Acute Brain Slices Using an Optimized N-Methyl-D-glucamine Protective Recovery Method</strong></div><div><em>J.&nbsp;Vis. Exp.</em> 2018; 53825</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.T.TingB.R.LeeP.ChongG.Soler-LlavinaC.CobbsC.KochH.ZengE.LeinPreparation+of+Acute+Brain+Slices+Using+an+Optimized+N-Methyl-D-glucamine+Protective+Recovery+MethodJ.%C2%A0Vis.+Exp.132201853825" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib78" class="citations"><div id="sref78" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref78-1" title="View in article">80.</a></div><div>Tjärnberg, A. ∙ Mahmood, O. ∙ Jackson, C.A. ...</div><div><strong>Optimal tuning of weighted kNN- and diffusion-based methods for denoising single cell genomics data</strong></div><div><em>bioRxiv.</em> 2020;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/2020.02.28.970202" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_80_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85098901697" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2F2020.02.28.970202" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib79" class="citations"><div id="sref79" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref79" title="View in article">81.</a></div><div>Van Hove, H. ∙ Martens, L. ∙ Scheyltjens, I. ...</div><div><strong>A single-cell atlas of mouse brain macrophages reveals unique transcriptional identities shaped by ontogeny and tissue environment</strong></div><div><em>Nat. Neurosci.</em> 2019; <strong>22</strong>:1021-1035</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41593-019-0393-4" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85065294555" target="_blank" aria-label="Open Scopus in new tab">Scopus (509)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31061494/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41593-019-0393-4&amp;pmid=31061494" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib80" class="citations"><div id="sref80" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref80" title="View in article">82.</a></div><div>Vogels, T.P. ∙ Abbott, L.F.</div><div><strong>Gating multiple signals through detailed balance of excitation and inhibition in spiking networks</strong></div><div><em>Nat. Neurosci.</em> 2009; <strong>12</strong>:483-491</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nn.2276" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_82_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-63649085510" target="_blank" aria-label="Open Scopus in new tab">Scopus (264)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19305402/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnn.2276&amp;pmid=19305402" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib81" class="citations"><div id="sref81" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref81" title="View in article">83.</a></div><div>von Ehr, A. ∙ Attaai, A. ∙ Neidert, N. ...</div><div><strong>Inhibition of Microglial TGFβ Signaling Increases Expression of <i>Mrc1</i></strong></div><div><em>Front. Cell. Neurosci.</em> 2020; <strong>14</strong>:66</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.3389/fncel.2020.00066" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_83_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85083481662" target="_blank" aria-label="Open Scopus in new tab">Scopus (17)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32296307/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.3389%2Ffncel.2020.00066&amp;pmid=32296307" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib82" class="citations"><div id="sref82" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref82-1" title="View in article">84.</a></div><div>Vormstein-Schneider, D. ∙ Lin, J.D. ∙ Pelkey, K.A. ...</div><div><strong>Viral manipulation of functionally distinct interneurons in mice, non-human primates and humans</strong></div><div><em>Nat. Neurosci.</em> 2020; <strong>23</strong>:1629-1636</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41593-020-0692-9" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_84_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85089867328" target="_blank" aria-label="Open Scopus in new tab">Scopus (100)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32807948/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41593-020-0692-9&amp;pmid=32807948" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib83" class="citations"><div id="sref83" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref83" title="View in article">85.</a></div><div>Wamsley, B. ∙ Jaglin, X.H. ∙ Favuzzi, E. ...</div><div><strong>Rbfox1 Mediates Cell-type-Specific Splicing in Cortical Interneurons</strong></div><div><em>Neuron.</em> 2018; <strong>100</strong>:846-859.e7</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.neuron.2018.09.026&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.neuron.2018.09.026&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85057123825" target="_blank" aria-label="Open Scopus in new tab">Scopus (63)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30318414/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.neuron.2018.09.026&amp;pmid=30318414" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib84" class="citations"><div id="sref84" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref84" title="View in article">86.</a></div><div>Wang, D.D. ∙ Kriegstein, A.R.</div><div><strong>Defining the role of GABA in cortical development</strong></div><div><em>J.&nbsp;Physiol.</em> 2009; <strong>587</strong>:1873-1879</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1113/jphysiol.2008.167635" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_86_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-62549102539" target="_blank" aria-label="Open Scopus in new tab">Scopus (266)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19153158/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1113%2Fjphysiol.2008.167635&amp;pmid=19153158" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib85" class="citations"><div id="sref85" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref85-1" title="View in article">87.</a></div><div>Wilton, D.K. ∙ Dissing-Olesen, L. ∙ Stevens, B.</div><div><strong>Neuron-Glia Signaling in Synapse Elimination</strong></div><div><em>Annu. Rev. Neurosci.</em> 2019; <strong>42</strong>:107-127</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1146/annurev-neuro-070918-050306" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_87_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85068806385" target="_blank" aria-label="Open Scopus in new tab">Scopus (191)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31283900/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1146%2Fannurev-neuro-070918-050306&amp;pmid=31283900" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib86" class="citations"><div id="sref86" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref86-1" title="View in article">88.</a></div><div>Wiltschko, A.B. ∙ Johnson, M.J. ∙ Iurilli, G. ...</div><div><strong>Mapping Sub-Second Structure in Mouse Behavior</strong></div><div><em>Neuron.</em> 2015; <strong>88</strong>:1121-1135</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.neuron.2015.11.031&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=10.1016%2Fj.neuron.2015.11.031&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-84957565912" target="_blank" aria-label="Open Scopus in new tab">Scopus (373)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26687221/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.neuron.2015.11.031&amp;pmid=26687221" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib87" class="citations"><div id="sref87" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref87-1" title="View in article">89.</a></div><div>Wolf, F.A. ∙ Angerer, P. ∙ Theis, F.J.</div><div><strong>SCANPY: large-scale single-cell gene expression data analysis</strong></div><div><em>Genome Biol.</em> 2018; <strong>19</strong>:15</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13059-017-1382-0" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_89_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-85041394976" target="_blank" aria-label="Open Scopus in new tab">Scopus (2665)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29409532/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13059-017-1382-0&amp;pmid=29409532" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib88" class="citations"><div id="sref88" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref88-1" title="View in article">90.</a></div><div>Wu, X. ∙ Fu, Y. ∙ Knott, G. ...</div><div><strong>GABA signaling promotes synapse elimination and axon pruning in developing cortical inhibitory interneurons</strong></div><div><em>J.&nbsp;Neurosci.</em> 2012; <strong>32</strong>:331-343</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1523/JNEUROSCI.3189-11.2012" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_90_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-84855913405" target="_blank" aria-label="Open Scopus in new tab">Scopus (84)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22219294/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1523%2FJNEUROSCI.3189-11.2012&amp;pmid=22219294" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib89" class="citations"><div id="sref89" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref89" title="View in article">91.</a></div><div>Zimmer, H. ∙ Riese, S. ∙ Régnier-Vigouroux, A.</div><div><strong>Functional characterization of mannose receptor expressed by immunocompetent mouse microglia</strong></div><div><em>Glia.</em> 2003; <strong>42</strong>:89-100</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/glia.10196" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.06.018&amp;key=2-s2.0-0037388257" target="_blank" aria-label="Open Scopus in new tab">Scopus (35)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/12594740/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fglia.10196&amp;pmid=12594740" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div></div></div></section></div></section><div class="core-linked-content">
    

        


    

            

            

            <div data-widget-id="8633d695-b258-4908-b86f-e2c268ec3c6b" data-widget-name="UX3HTMLWidget"><script type="text/javascript">
if($('.content-navigation__body__title').length){
$('.content-navigation__body__title').each(function(){
var html=this.innerHTML;
var div = document.createElement("div");
div.innerHTML = html;
var text = div.textContent || div.innerText || "";
this.innerHTML=text;
});
}
</script></div>

            

        

    

        


    

            

            

            <div data-widget-id="393effff-5b44-4681-a9b4-4f029b3540b7" data-widget-name="UX3HTMLWidget"><style>
.linkback-ad .pb-ad {
    display: block;
   height: auto;
    margin-left: 0px;
    margin-right: auto;
}
</style></div>

            

        

    

        

</div></div></div><div class="core-collateral-aside"><section id="core-collateral-figures"><h2>Figures (14)<button class="js--openFigureViewer"><span>Figure Viewer</span></button></h2><div class="core-collateral-aside-wrapper"><div class="grid"><a class="figure__open__ctrl" href="#fig1" aria-label="Go to Figure&amp;nbsp;1"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.06.018/asset/94a3bdb6-d705-496e-aca0-ad19f4a097b0/main.assets/gr1.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs1" aria-label="Go to Figure&amp;nbsp;S1"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.06.018/asset/ae9b3e47-1e06-4462-947e-fb3e3c768f47/main.assets/figs1.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig2" aria-label="Go to Figure&amp;nbsp;2"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.06.018/asset/002dc7fd-a5d8-4690-9442-9152f83f0363/main.assets/gr2.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig3" aria-label="Go to Figure&amp;nbsp;3"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.06.018/asset/7622f4e2-3dc7-453d-8957-4e483babcf7f/main.assets/gr3.jpg" aria-hidden="true"></a></div><div id="collapsible-figures" data-expandable="item"><div class="content grid" style="display:none;" role="region" id="collapsible-figures_content"><a class="figure__open__ctrl" href="#figs2" aria-label="Go to Figure&amp;nbsp;S2"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.06.018/asset/48dcebfc-c4ff-436a-a513-1b04f7195189/main.assets/figs2.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs3" aria-label="Go to Figure&amp;nbsp;S3"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.06.018/asset/28b9fa3e-450a-4f1f-8ac5-720d67d657c8/main.assets/figs3.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig4" aria-label="Go to Figure&amp;nbsp;4"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.06.018/asset/b68fbac5-b9a9-4013-b0e5-715f917a9001/main.assets/gr4.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs4" aria-label="Go to Figure&amp;nbsp;S4"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.06.018/asset/456b417f-43eb-4693-8be1-ca4822c7c844/main.assets/figs4.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs5" aria-label="Go to Figure&amp;nbsp;S5"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.06.018/asset/dde7322c-2b64-459b-861d-65120efd2326/main.assets/figs5.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig5" aria-label="Go to Figure&amp;nbsp;5"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.06.018/asset/7715c85a-b092-41dc-8ccc-95d979782543/main.assets/gr5.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs6" aria-label="Go to Figure&amp;nbsp;S6"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.06.018/asset/784573d0-e154-467c-b4e8-db8446d42730/main.assets/figs6.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig6" aria-label="Go to Figure&amp;nbsp;6"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.06.018/asset/f89949b8-0ecf-4209-a04d-871afd3f3a80/main.assets/gr6.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig7" aria-label="Go to Figure&amp;nbsp;7"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.06.018/asset/658c2f1f-8fe6-4439-bd2c-85abf79cad63/main.assets/gr7.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs7" aria-label="Go to Figure&amp;nbsp;S7"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.06.018/asset/3148320e-8967-45c2-86b6-86c928b31d81/main.assets/figs7.jpg" aria-hidden="true"></a></div><button class="trigger" aria-controls="collapsible-figures_content" aria-expanded="false" tabindex="0" role="button"><span class="expand">Show all figures</span><span class="collapse">Hide figures</span><i class="icon-section_arrow_d" aria-hidden="true"></i></button></div></div></section>
    

        


    

            

            

            <section id="core-collateral-metrics" class="publication-list__axel-section"><h2>Article metrics</h2><div><ul class="publication-list rlist publication-list--list plum-x--lazy-load publication-list--axel js--loaded" data-plum-x-url="/pb/widgets/ux3/publicationList/plumx?widgetId=66dd7e11-7768-4006-93d8-a72b60e2a264&amp;pbContext=%3Bproduct%3Aproduct%3Aelsevier%5C%3Aproduct%5C%3Acell%3Bwebsite%3Awebsite%3Acell-site%3Bctype%3Astring%3AJournal+Content%3Bissue%3Aissue%3Apii%5C%3AS0092867420X0016X%3Bjournal%3Ajournal%3Acell%3BpageGroup%3Astring%3AArticle+View%3Bpage%3Astring%3AShow+Full+Text%3BrequestedJournal%3Ajournal%3Acell%3Bwgroup%3Astring%3ADefault+Website+Group%3Barticle%3Aarticle%3Apii%5C%3AS0092867421007534%3BsubPage%3Astring%3AFull+Text" data-pb-mode-enabled="false">
                <li class="publication-list__box publication-list__box-axel">
                    <ul class="rlist axel-list__box">
                        
                        
            <li class="plum-x-wrapper">
                <div class="plum-x plum-x--axel">
                    <div class="plum-x__items">
                        
                    <div class="plum-x__item plum-x-citation">
                        <div class="plum-x__item-num">174</div>
                        <div class="plum-x__item-title">Citations</div>
                    </div>    
                
                    <div class="plum-x__item plum-x-capture">
                        <div class="plum-x__item-num">363</div>
                        <div class="plum-x__item-title">Captures</div>
                    </div>    
                
                    <div class="plum-x__item plum-x-mention">
                        <div class="plum-x__item-num">15</div>
                        <div class="plum-x__item-title">Mentions</div>
                    </div>    
                
                    <div class="plum-x__item plum-x-socialMedia">
                        <div class="plum-x__item-num">9</div>
                        <div class="plum-x__item-title">Social Media</div>
                    </div>    
                
                    </div>
                    <div class="plum-x__details">
                        <div class="plum-x__logo">
                            <img src="/products/marlin/releasedAssets/images/plumx-logo.png" alt="Plum X logo">
                        </div>
                        <a class="plum-x__view-details" href="https://plu.mx/plum/a/?doi=10.1016/j.cell.2021.06.018&amp;theme=plum-jbs-theme&amp;hideUsage=true" target="_blank">
                            <span>View details</span>
                            <span class="sr-only">opens in a new tab</span>
                            <i class="icon-external-arrow" aria-hidden="true"></i>
                        </a>
                    </div>
                </div>  
            </li>
        
                    </ul>
                </li></ul></div></section>

            

        
<section id="core-collateral-supplementary"><h2>Supplemental information (2)</h2><div class="core-collateral-aside-wrapper"><div class="core-supplementary-material"><div class="core-link"><button class="downloadAllSupplementary"><span>Download all</span></button></div></div><div class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2021.06.018/attachment/e6503fd4-f5ee-4c85-ac5a-b66e1a4e1b4a/mmc1.xlsx" download="mmc1.xlsx"><span id="mmc1-link">Spreadsheet (8.46 MB)</span></a><div class="core-description">Table S1. Genes enriched in microglia within each cluster compared to all other clusters for wild-type, GABA<sub>B1</sub>R cKO, and <i>Cx3cr1</i><sup><i>Cre/+</i></sup> (Cre-Het control) and integrated groups, related to Figure&nbsp;5</div></div></div><div class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2021.06.018/attachment/e0dc1d57-b31b-44cd-8c5b-29f5f871f400/mmc2.xlsx" download="mmc2.xlsx"><span id="mmc2-link">Spreadsheet (869.82 KB)</span></a><div class="core-description">Table S2. Differentially expression analyses, function of genes differentially expressed between wild-type and GABA<sub>B1</sub>R cKO microglia, denoising results as well as MERFISH clusters and differentially expressed genes, related to Figures 5 and 6</div></div></div></div></section>
    

        


    

            

            

            <section id="core-collateral-relatedArticles" class="axel-related-articles"><h2 id="relatedArticles" class="top axel-related-articles__title" data-title="Related articles">Related Articles</h2><div data-pb-widget-type="ux3-relatedArticles" data-widget-id="8d3844d1-450e-497c-ab75-a63fc41750ac" data-widget-name="AtyponTempUX3RelatedArticlesWidget" data-article-pii="S0092867421007534" class="ux3DeferredOnScrollWidget axel-related-articles__body" data-show-more="true"></div></section>

            

        
</div></div><aside data-core-aside="right-rail"><div class="inner-wrapper">

    

        


    

            

            

            <div data-widget-id="dca7470d-9085-4d21-a6f4-a66397718146" data-widget-name="UX3HTMLWidget"><input type="hidden" id="gptPage" value="cell-videos"></div>

            

        











        <div data-widget-def="literatumAd" data-widget-id="481d10c2-bbe0-470e-a5ba-207a81ecc8dc" class="adplaceholders" style="right: 0px; position: static;">
        
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd alignCenter  widget-verticalAd" id="481d10c2-bbe0-470e-a5ba-207a81ecc8dc">
                
                
                    
                

                <div class="wrapped " id="Marketplace-Recs">
                    
                    <div class="widget-body body body-verticalAd "><div class="verticalAdContainer">
    <div class="verticalAdLabelContainer">
        <div class="verticalAdLabel">
                <img src="/templates/jsp/_style2/_marlin/images/bg_adVert.gif" alt="Advertisement">
        </div>
    </div>
    <div class="verticalAd">
        





        <div class="pb-ad">
            <script src="//cdn.pbgrd.com/core-cell.js" async=""></script>
<div id="pbgrd-skyr-c"><div id="pbgrd-skyr" style="width: inherit; position: relative; background: transparent;"></div></div>
        </div>
    

    </div>
</div></div>
                </div>
                </div>

        

        </div>
    

</div></aside><div class="core-authors-details" style="display:none"><section class="core-authors"></section></div></article><div data-extent="article-wrapper"><div class="core-container"></div></div><script id="axel-publication-metadata" type="application/json">{"doi":"10.1016/j.cell.2021.06.018","data-is-retraction":false,"data-is-retired":false}
</script><script type="text/javascript" data-ot-ignore="data-ot-ignore" defer="defer" src="https://static.addtoany.com/menu/page.js" class="optanon-category-C0004 ot-vscat-C0004"></script><template id="figure_nav_template"><a href="#" class="open-in-viewer btn btn-dark figure-pop-btn"><i aria-hidden="true" class="icon-export_citation"></i><span>Open in viewer</span></a></template><div id="figure-viewer" role="dialog" aria-modal="true" aria-label="Figure Viewer" class="showCaption showThumbstrip figure-viewer"><nav class="nav-bar figure-viewer__top"><a href="#" title="Close Figure Viewer" role="button" class="figure-viewer__close"><i aria-hidden="true" class="icon-tools_close"></i></a><div class="figure-viewer__title">GABA-receptive microglia selectively sculpt developing inhibitory circuits</div></nav><div class="frames"><ul class="frames__list"><li><div class="thumbnail__image"><img id="gr1" src="/cms/10.1016/j.cell.2021.06.018/asset/94a3bdb6-d705-496e-aca0-ad19f4a097b0/main.assets/gr1.jpg" height="789" width="654" aria-labelledby="fig1-title fig1-content" data-viewer-src="/cms/10.1016/j.cell.2021.06.018/asset/f99b38b5-f97a-4706-b4b8-1967246770fc/main.assets/gr1_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs1" src="/cms/10.1016/j.cell.2021.06.018/asset/ae9b3e47-1e06-4462-947e-fb3e3c768f47/main.assets/figs1.jpg" height="958" width="722" aria-labelledby="figs1-title figs1-content" data-viewer-src="/cms/10.1016/j.cell.2021.06.018/asset/448fdf3e-af59-45a9-ab83-d00773d20236/main.assets/figs1_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr2" src="/cms/10.1016/j.cell.2021.06.018/asset/002dc7fd-a5d8-4690-9442-9152f83f0363/main.assets/gr2.jpg" height="732" width="657" aria-labelledby="fig2-title fig2-content" data-viewer-src="/cms/10.1016/j.cell.2021.06.018/asset/9ca2cc10-367f-4d91-aa34-580069e5254d/main.assets/gr2_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr3" src="/cms/10.1016/j.cell.2021.06.018/asset/7622f4e2-3dc7-453d-8957-4e483babcf7f/main.assets/gr3.jpg" height="483" width="506" aria-labelledby="fig3-title fig3-content" data-viewer-src="/cms/10.1016/j.cell.2021.06.018/asset/aa5a704e-8676-4815-bb39-c6d36b8a9511/main.assets/gr3_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs2" src="/cms/10.1016/j.cell.2021.06.018/asset/48dcebfc-c4ff-436a-a513-1b04f7195189/main.assets/figs2.jpg" height="799" width="506" aria-labelledby="figs2-title figs2-content" data-viewer-src="/cms/10.1016/j.cell.2021.06.018/asset/10b2335f-5822-4141-a820-747ca0869964/main.assets/figs2_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs3" src="/cms/10.1016/j.cell.2021.06.018/asset/28b9fa3e-450a-4f1f-8ac5-720d67d657c8/main.assets/figs3.jpg" height="958" width="481" aria-labelledby="figs3-title figs3-content" data-viewer-src="/cms/10.1016/j.cell.2021.06.018/asset/b2709bed-9fcc-448a-be45-984b910432de/main.assets/figs3_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr4" src="/cms/10.1016/j.cell.2021.06.018/asset/b68fbac5-b9a9-4013-b0e5-715f917a9001/main.assets/gr4.jpg" height="574" width="657" aria-labelledby="fig4-title fig4-content" data-viewer-src="/cms/10.1016/j.cell.2021.06.018/asset/9b544f10-9a88-44f7-bf69-2a604ca3b2f9/main.assets/gr4_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs4" src="/cms/10.1016/j.cell.2021.06.018/asset/456b417f-43eb-4693-8be1-ca4822c7c844/main.assets/figs4.jpg" height="958" width="623" aria-labelledby="figs4-title figs4-content" data-viewer-src="/cms/10.1016/j.cell.2021.06.018/asset/0b84e907-c8f4-4bba-b696-8ed2afb48ec9/main.assets/figs4_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs5" src="/cms/10.1016/j.cell.2021.06.018/asset/dde7322c-2b64-459b-861d-65120efd2326/main.assets/figs5.jpg" height="958" width="619" aria-labelledby="figs5-title figs5-content" data-viewer-src="/cms/10.1016/j.cell.2021.06.018/asset/247dce6d-4c92-477b-8b3e-b6b7716140bd/main.assets/figs5_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr5" src="/cms/10.1016/j.cell.2021.06.018/asset/7715c85a-b092-41dc-8ccc-95d979782543/main.assets/gr5.jpg" height="920" width="657" aria-labelledby="fig5-title fig5-content" data-viewer-src="/cms/10.1016/j.cell.2021.06.018/asset/8359678e-4eab-4311-9821-fd183de19169/main.assets/gr5_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs6" src="/cms/10.1016/j.cell.2021.06.018/asset/784573d0-e154-467c-b4e8-db8446d42730/main.assets/figs6.jpg" height="958" width="731" aria-labelledby="figs6-title figs6-content" data-viewer-src="/cms/10.1016/j.cell.2021.06.018/asset/97842390-3569-4192-a4f7-e0c18ca377ed/main.assets/figs6_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr6" src="/cms/10.1016/j.cell.2021.06.018/asset/f89949b8-0ecf-4209-a04d-871afd3f3a80/main.assets/gr6.jpg" height="744" width="739" aria-labelledby="fig6-title fig6-content" data-viewer-src="/cms/10.1016/j.cell.2021.06.018/asset/494dd21e-fa0e-4a77-84b5-5a2d0eb08af5/main.assets/gr6_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr7" src="/cms/10.1016/j.cell.2021.06.018/asset/658c2f1f-8fe6-4439-bd2c-85abf79cad63/main.assets/gr7.jpg" height="798" width="648" aria-labelledby="fig7-title fig7-content" data-viewer-src="/cms/10.1016/j.cell.2021.06.018/asset/ee122a21-6524-4849-a7d9-1a7f2ef9e9b5/main.assets/gr7_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs7" src="/cms/10.1016/j.cell.2021.06.018/asset/3148320e-8967-45c2-86b6-86c928b31d81/main.assets/figs7.jpg" height="941" width="657" aria-labelledby="figs7-title figs7-content" data-viewer-src="/cms/10.1016/j.cell.2021.06.018/asset/79ab9754-a115-4c0c-be3a-2da48fb7f92f/main.assets/figs7_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li></ul><a href="#" title="Previous Frame" class="frameButton previousFrameButton frames__prev"><!--span Previous Frame--><i class="icon-arrow_l"></i></a><a href="#" title="Next Frame" class="frameButton nextFrameButton frames__next"><!--span Next Frame--><i class="icon-arrow_r"></i></a></div><div class="caption"><div class="figure-viewer__caption-body"></div></div><div class="toolbar figure-viewer__toolbar"><span class="figure-viewer__counter"></span><a href="#" class="button toggle-caption-button figure-viewer__caption__ctrl"><i aria-hidden="true" class="icon icon-info"></i><span class="label">Hide Caption</span></a><a href="#" class="button download-options-button"><i aria-hidden="true" class="icon icon-gizmo-folder-in download-size-icon"></i><span class="label">Download</span></a><a href="#figures" class="button see-figures figure-viewer__see-figure__ctrl"><i aria-hidden="true" class="icon icon-gizmo-image_figure viewer-size-icon"></i><span class="label">See figure in Article</span></a></div><div class="thumbstrip"><a href="#" class="toggleButton thumbstrip__ctrl"><span>Toggle Thumbstrip</span></a><a href="#" title="Previous Thumbnails" class="thumbstripButton previousThumbstripButton thumbstrip__prev"><i aria-hidden="true" class="icon-arrow_l"></i></a><a href="#" title="Next Thumbnails" class="thumbstripButton nextThumbstripButton thumbstrip__next"><i aria-hidden="true" class="icon-arrow_r"></i></a><div class="thumbstrip__wrapper"><ul class="rlist figure-viewer__thumbslist"><li><div class="thumbnail__label">Figure&nbsp;1</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr1" src="/cms/10.1016/j.cell.2021.06.018/asset/94a3bdb6-d705-496e-aca0-ad19f4a097b0/main.assets/gr1.jpg" height="789" width="654" aria-labelledby="fig1-title fig1-content" data-viewer-src="/cms/10.1016/j.cell.2021.06.018/asset/f99b38b5-f97a-4706-b4b8-1967246770fc/main.assets/gr1_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S1</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs1" src="/cms/10.1016/j.cell.2021.06.018/asset/ae9b3e47-1e06-4462-947e-fb3e3c768f47/main.assets/figs1.jpg" height="958" width="722" aria-labelledby="figs1-title figs1-content" data-viewer-src="/cms/10.1016/j.cell.2021.06.018/asset/448fdf3e-af59-45a9-ab83-d00773d20236/main.assets/figs1_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;2</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr2" src="/cms/10.1016/j.cell.2021.06.018/asset/002dc7fd-a5d8-4690-9442-9152f83f0363/main.assets/gr2.jpg" height="732" width="657" aria-labelledby="fig2-title fig2-content" data-viewer-src="/cms/10.1016/j.cell.2021.06.018/asset/9ca2cc10-367f-4d91-aa34-580069e5254d/main.assets/gr2_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;3</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr3" src="/cms/10.1016/j.cell.2021.06.018/asset/7622f4e2-3dc7-453d-8957-4e483babcf7f/main.assets/gr3.jpg" height="483" width="506" aria-labelledby="fig3-title fig3-content" data-viewer-src="/cms/10.1016/j.cell.2021.06.018/asset/aa5a704e-8676-4815-bb39-c6d36b8a9511/main.assets/gr3_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S2</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs2" src="/cms/10.1016/j.cell.2021.06.018/asset/48dcebfc-c4ff-436a-a513-1b04f7195189/main.assets/figs2.jpg" height="799" width="506" aria-labelledby="figs2-title figs2-content" data-viewer-src="/cms/10.1016/j.cell.2021.06.018/asset/10b2335f-5822-4141-a820-747ca0869964/main.assets/figs2_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S3</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs3" src="/cms/10.1016/j.cell.2021.06.018/asset/28b9fa3e-450a-4f1f-8ac5-720d67d657c8/main.assets/figs3.jpg" height="958" width="481" aria-labelledby="figs3-title figs3-content" data-viewer-src="/cms/10.1016/j.cell.2021.06.018/asset/b2709bed-9fcc-448a-be45-984b910432de/main.assets/figs3_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;4</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr4" src="/cms/10.1016/j.cell.2021.06.018/asset/b68fbac5-b9a9-4013-b0e5-715f917a9001/main.assets/gr4.jpg" height="574" width="657" aria-labelledby="fig4-title fig4-content" data-viewer-src="/cms/10.1016/j.cell.2021.06.018/asset/9b544f10-9a88-44f7-bf69-2a604ca3b2f9/main.assets/gr4_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S4</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs4" src="/cms/10.1016/j.cell.2021.06.018/asset/456b417f-43eb-4693-8be1-ca4822c7c844/main.assets/figs4.jpg" height="958" width="623" aria-labelledby="figs4-title figs4-content" data-viewer-src="/cms/10.1016/j.cell.2021.06.018/asset/0b84e907-c8f4-4bba-b696-8ed2afb48ec9/main.assets/figs4_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S5</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs5" src="/cms/10.1016/j.cell.2021.06.018/asset/dde7322c-2b64-459b-861d-65120efd2326/main.assets/figs5.jpg" height="958" width="619" aria-labelledby="figs5-title figs5-content" data-viewer-src="/cms/10.1016/j.cell.2021.06.018/asset/247dce6d-4c92-477b-8b3e-b6b7716140bd/main.assets/figs5_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;5</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr5" src="/cms/10.1016/j.cell.2021.06.018/asset/7715c85a-b092-41dc-8ccc-95d979782543/main.assets/gr5.jpg" height="920" width="657" aria-labelledby="fig5-title fig5-content" data-viewer-src="/cms/10.1016/j.cell.2021.06.018/asset/8359678e-4eab-4311-9821-fd183de19169/main.assets/gr5_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S6</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs6" src="/cms/10.1016/j.cell.2021.06.018/asset/784573d0-e154-467c-b4e8-db8446d42730/main.assets/figs6.jpg" height="958" width="731" aria-labelledby="figs6-title figs6-content" data-viewer-src="/cms/10.1016/j.cell.2021.06.018/asset/97842390-3569-4192-a4f7-e0c18ca377ed/main.assets/figs6_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;6</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr6" src="/cms/10.1016/j.cell.2021.06.018/asset/f89949b8-0ecf-4209-a04d-871afd3f3a80/main.assets/gr6.jpg" height="744" width="739" aria-labelledby="fig6-title fig6-content" data-viewer-src="/cms/10.1016/j.cell.2021.06.018/asset/494dd21e-fa0e-4a77-84b5-5a2d0eb08af5/main.assets/gr6_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;7</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr7" src="/cms/10.1016/j.cell.2021.06.018/asset/658c2f1f-8fe6-4439-bd2c-85abf79cad63/main.assets/gr7.jpg" height="798" width="648" aria-labelledby="fig7-title fig7-content" data-viewer-src="/cms/10.1016/j.cell.2021.06.018/asset/ee122a21-6524-4849-a7d9-1a7f2ef9e9b5/main.assets/gr7_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S7</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs7" src="/cms/10.1016/j.cell.2021.06.018/asset/3148320e-8967-45c2-86b6-86c928b31d81/main.assets/figs7.jpg" height="941" width="657" aria-labelledby="figs7-title figs7-content" data-viewer-src="/cms/10.1016/j.cell.2021.06.018/asset/79ab9754-a115-4c0c-be3a-2da48fb7f92f/main.assets/figs7_lrg.jpg" loading="lazy" tabindex="0"></div></div></li></ul></div></div><div class="download figure-viewer__download"><div class="pointer"></div><ul class="rlist downlad__wrapper"><li class="hidden-xs hidden-sm"><a href="/action/downloadFigures?pii=e00489&amp;id=" class="downloadAllButton"><i class="icon icon-gizmo-ppt figure-viewer__download__viewer-size-icon figure-viewer__download__viewer-size-icon--ppt"></i><span class="figure-viewer__download__text"> Download .PPT</span></a></li></ul></div></div><template id="toCitationLink" data-citation="Citation" data-footnote="Footnote"><div class="to-citation__wrapper"><a class="to-citation"><i aria-hidden="true" class="icon-subdirectory_arrow_left"></i><span>Go to</span></a></div></template><template id="toCitationButton" data-citation="Citation" data-footnote="Footnote"><div class="to-citation__wrapper"><button class="to-citation"><i aria-hidden="true" class="icon-subdirectory_arrow_left"></i><span>Go to</span></button></div></template><template id="toCitationAccordion" data-collapse-citation="Hide Citations" data-expand-citation="Show Citations" data-collapse-footnote="Hide Footnotes" data-expand-footnote="Show Footnotes"><div class="to-citation__wrapper"><button aria-controls="aria-controls" aria-expanded="false" aria-label="Toggle citations menu" class="to-citation__toggle"><i aria-hidden="true" class="icon-plus"></i><span data-expand-title="data-expand-title" data-collapsed-title="data-collapsed-title" class="accordion__toggle__title"></span></button><div role="menu" aria-label="links" class="to-citation__accordion no-separator"><ul></ul></div></div></template><template id="toCitationAccordionItem"><li><a class="to-citation"><i aria-hidden="true" class="icon-subdirectory_arrow_left"></i><span></span></a></li></template><template id="citations_truncate_template"><div class="citations-truncation"><button data-label-expand="Show all references" data-label-collapse="Show fewer" data-label-remaining="references remaining" class="btn btn--inverse"><span>Show all references</span></button></div></template><template id="collateral_texts_template" data-do="Digital Object" data-figures="Figures" data-metrics="Article Metrics" data-related="Related articles" data-supplementary="Supplemental information"></template><template id="collapsible_authors_template"><button aria-expanded="false" data-expandable="all" data-label-expand="Expand All" data-label-collapse="Collapse All" class="collateral-contributors-control"><span>Expand All</span></button></template>
<template id="collapsible_tables_collapse_template"><div class="collapsible-figure-btn__wrapper expanded"><button aria-expanded="true" class="btn collapsible-figure-btn btn--inverse"><i aria-hidden="true" class="icon-arrow-up"></i><span class="text-uppercase">Collapse</span></button></div></template>
<template id="collapsible_tables_expand_template"><div class="collapsible-figure-btn__wrapper collapsed"><button aria-expanded="false" class="btn collapsible-figure-btn btn--inverse"><i aria-hidden="true" class="icon-arrow-down"></i><span class="text-uppercase">Expand Table</span></button></div></template><template id="authorsAffiliationsLink"><a href="#tab-contributors" class="to-authors-affiliations">Authors Info &amp; Affiliations</a></template></div>
                </div>
                </div>

        

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none  widget-  widget-compact-all" id="756cf18d-e910-4468-936c-0225ae7c669f">
                
                
                    
                

                <div class="wrapped " id="ad for nucleus-single-molecule-technologies">
                    
                    <div class="widget-body body body-  body-compact-all"><div class="pb-ad">
            <script type="text/javascript">
URL_var = 'https://www.cell.com/cell/fulltext/S0092-8674(21)00753-4';

if(URL_var.indexOf("S0165-6147(17)30205-5") > -1
|| URL_var.indexOf("S0167-7799(18)30204-X") > -1 
|| URL_var.indexOf("S0092-8674(18)31649-0") > -1 
|| URL_var.indexOf("S0092-8674(19)30404-0") > -1 
|| URL_var.indexOf("S1097-2765(18)30756-1") > -1 
|| URL_var.indexOf("S0092-8674(18)31452-1") > -1 
|| URL_var.indexOf("S1097-2765(19)30175-3") > -1 
|| URL_var.indexOf("S0092-8674(18)30852-3") > -1 
|| URL_var.indexOf("S0006-3495(18)30295-9") > -1 
|| URL_var.indexOf("S1097-2765(19)30063-2") > -1 
|| URL_var.indexOf("S0006-3495(17)33208-3") > -1 
|| URL_var.indexOf("S2211-1247(18)31639-5") > -1 
|| URL_var.indexOf("S1097-2765(18)30978-X") > -1 
)
    {
document.write('<input type="hidden" id="gpttc" value="nucleus-single-molecule-technologies" class="gpt-Target">')
}
</script>
        </div></div>
                </div>
                </div>

        

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none  widget-" id="754b1780-1117-4371-9285-9cd7020963d5">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body- "><div class="pb-ad">
            <script type="text/javascript">
URL_var = 'https://www.cell.com/cell/fulltext/S0092-8674(21)00753-4';

if(URL_var.indexOf("S1535-6108(19)30104-7") > -1
|| URL_var.indexOf("S0092-8674(18)31394-1") > -1
|| URL_var.indexOf("S1074-7613(19)30033-0") > -1
|| URL_var.indexOf("S1535-6108(18)30473-2") > -1
|| URL_var.indexOf("S0092-8674(19)30154-0") > -1
|| URL_var.indexOf("S1550-4131(18)30178-5") > -1
|| URL_var.indexOf("S1535-6108(19)30242-9") > -1
|| URL_var.indexOf("S1074-7613(18)30431-X") > -1
|| URL_var.indexOf("S0092-8674(19)30618-X") > -1
|| URL_var.indexOf("S1535-6108(18)30472-0") > -1
|| URL_var.indexOf("S1535-6108(18)30120-X") > -1
|| URL_var.indexOf("S0092-8674(18)31247-9") > -1
|| URL_var.indexOf("S1074-7613(18)30085-2") > -1
|| URL_var.indexOf("S1074-7613(18)30080-3") > -1
|| URL_var.indexOf("S1471-4906(18)30210-2") > -1
|| URL_var.indexOf("S1550-4131(19)30304-3") > -1
|| URL_var.indexOf("S1934-5909(18)30599-X") > -1
|| URL_var.indexOf("S2405-8033(19)30024-X") > -1
|| URL_var.indexOf("S1535-6108(18)30109-0") > -1
|| URL_var.indexOf("S0962-8924(18)30127-2") > -1
)
    {
document.write('<input type="hidden" id="gpttc" value="nucleus-tumor-immune-microenvironment" class="gpt-Target">')
}
</script>
        </div></div>
                </div>
                </div>

        

    

        


    

            

            

            <div data-widget-id="9c9ff7a1-a035-49f6-a336-600c9419c518" data-widget-name="UX3HTMLWidget"><!-- Start of HubSpot Embed Code --> 

<script type="text/plain" class="optanon-category-4" id="hs-script-loader" async="" defer="" src="//js.hs-scripts.com/519224.js"></script> 

<!-- End of HubSpot Embed Code --></div>

            

        

    

        


    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget sessionValidationWidget none  widget-none  widget-compact-all" id="0257aef7-6074-4898-a989-1c3f693225a2">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none  body-compact-all"><script>
    $.ajax({
        url: "https://www.cell.com/ssoSessionCheck",
        type: "GET",
        dataType: "jsonp",
        success: function (data) {
            if(data.RefreshSession) {
                location.reload();
            }
        },
        error : function (){
            console.warn("can't validate CSAS session.");
        }
    });
</script></div>
                </div>
                </div>

        

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget csasSsoUrls none  widget-none  widget-compact-all" id="948bc5c4-8b0d-47be-b5ad-6570559802d1">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none  body-compact-all"><!-- clear SSO URLs after all iFrames are rendered or after passing a time out-->
<script>
    setTimeout(clearSsoUrlsFun,10000);
    var iFramesList = document.getElementById("sso-urls-div").getElementsByTagName("iframe");
    var renderedIFramesCount = 0;
    var numberOfIFrames = iFramesList.length;
    var skipRefresh = false;
    for (var i = 0; i < iFramesList.length; i++) {
        var iFrame = iFramesList[i];
        iFrame.addEventListener("load", function() {
            renderedIFramesCount = renderedIFramesCount + 1;
            if (renderedIFramesCount >= numberOfIFrames)
            {
                clearSsoUrlsFun();
            }
        });
    }

    function clearSsoUrlsFun() {
        if (skipRefresh == false && numberOfIFrames > 0)
        {
            skipRefresh = true;
            $.ajax({
                url: "https://www.cell.com/action/clearSsoUrls",
                type: "GET",
                dataType: "jsonp",
                error : errorCallingAjx
            });
        }
    }

    function errorCallingAjx(){
        console.log("can't delete sso urls .")
    }
</script></div>
                </div>
                </div>

        










        <div data-widget-def="ux3-layout-widget" data-widget-id="d95b4a46-dc4d-4ca8-83f9-8b6724c649ec" class="ux3 ">
        
    

        


    

            

            

            <div data-widget-id="d95b4a46-dc4d-4ca8-83f9-8b6724c649ec" data-widget-name="ux3-layout-widget" class="fonts-loaded"><div class="footer clearfix">
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none adplaceholder adplaceholder--header widget-horizontalAd" id="a07ee92b-3942-49f8-9186-a2a08fc3fc56">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-horizontalAd "><div class="horizontalAdContainer">
    <div class="horizontalAdLabelContainer">
        <div class="horizontalAdLabel">
                <img src="/templates/jsp/_style2/_marlin/images/bg_adHoriz.gif" alt="Advertisement">
        </div>
    </div>
    <div class="horizontalAd">
        





        <div class="pb-ad">
            <!-- Beginning Async AdSlot 4 for Ad unit els.cellpress/cell  ### size: [[728,90]] -->
<div class="gpt-ad" data-adsize="cell-leaderboard" data-page="" data-site="cell" data-pos="4"></div>
<!-- End AdSlot 4 -->
        </div>
    

    </div>
</div></div>
                </div>
                </div>

        

    

        


    

            

            

            <div data-widget-id="23703bbc-af46-4f3f-a5bb-dd52891ff0ff" data-widget-name="UX3HTMLWidget"><div class="footer__top">
    <div class="footer__logo centered"><a href="/home" title="Cell Press"><!-- Generator: Adobe Illustrator 25.2.1, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 300 72" style="enable-background:new 0 0 300 72;" xml:space="preserve">
<style type="text/css">
	.st0{fill:#007DBC;}
</style>
<g>
	<path class="st0" d="M39.4,7.5c-2.3-0.3-4.4,1.1-4.9,3.3l-1.8,7.4c-0.4,1.8-1.9,3.1-3.8,3.3l-10.5,1.2C7.1,24-1,34.3,0.7,45.6
		c1.3,9,8.6,15.7,17.2,17c2.3,0.3,4.4-1.1,4.9-3.3l1.8-7.4c0.4-1.8,1.9-3.1,3.8-3.3l10.5-1.2c11.3-1.2,19.4-11.6,17.8-22.9
		C55.3,15.4,48.1,8.8,39.4,7.5z M37.1,34l-6.6,0.8c-1.1,0.1-2,0.9-2.3,2l-1.9,8.1c-0.6,2.4-2.5,4.2-5,4.5c-3.8,0.4-7.1-2.4-7.4-6.2
		c-0.3-3.7,2.6-6.8,6.3-7.2l6.6-0.8c1.1-0.1,2-0.9,2.3-2l1.9-8.1c0.6-2.4,2.5-4.2,5-4.5c3.8-0.4,7.1,2.4,7.4,6.2
		C43.7,30.5,40.8,33.6,37.1,34z"></path>
	<rect x="153.1" y="13.9" class="st0" width="8.2" height="42.3"></rect>
	<path class="st0" d="M90.2,20.4c7.3,0,8.6,4.8,9.4,7.3h8.8c-0.5-6.2-5.6-14.9-18.2-14.9c-10.9,0-19.4,7.5-19.4,22.3
		c0,14.6,8.1,22.2,19.3,22.2c10.6,0,16.6-6.4,18.4-15.6h-8.8c-1.1,5.2-4.5,7.9-9.4,7.9c-7.5,0-10.5-6.8-10.5-14.4
		C79.7,23.3,85.9,20.4,90.2,20.4z"></path>
	<path class="st0" d="M125.2,24c-13.7,0-15,12.1-15,15.8c0,12.9,6.8,17.1,15.6,17.1c5.4,0,11.6-2.5,13.8-10H131
		c-0.7,2.2-3,3.4-5.3,3.4c-6.6,0-7.1-5.3-7.3-7.7h21.5v-1.5C139.9,27,131.3,24,125.2,24z M118.6,37.4c0.5-4.1,2.5-6.6,6.4-6.6
		c2.6,0,5.9,1.5,6.4,6.6H118.6z"></path>
	<rect x="142" y="13.9" class="st0" width="8.2" height="42.3"></rect>
	<path class="st0" d="M174,13.8h-9.8V21h7.2c4.3,0,8.2,0.9,8.2,6.4c0,5.4-3.9,6.4-8.2,6.4h-7.2V41h9.8c10.6,0,14.7-6.7,14.7-13.6
		C188.7,20.5,184.6,13.8,174,13.8z"></path>
	<path class="st0" d="M210.5,24.9c-0.5-0.2-1.2-0.3-1.8-0.3c-4.1,0-8.3,2.7-9.8,6.5h-0.1v-5.7h-8v30.7h8.4V42.3
		c0-5.4,2.1-9.9,8.3-9.9c1,0,2.3,0.1,3,0.3V24.9z"></path>
	<path class="st0" d="M225.6,24.6c-9.3,0-15.7,7-15.7,16.2c0,9.5,6,16.1,15.7,16.1c6.9,0,12-3.1,14.4-10.3h-7.4
		c-0.5,1.9-3.3,4-6.6,4c-4.7,0-7.3-2.4-7.5-7.8h22.1C241.1,33.4,236,24.6,225.6,24.6z M218.4,37.5c0.1-2.4,1.7-6.5,7-6.5
		c4.1,0,5.9,2.3,6.7,6.5H218.4z"></path>
	<path class="st0" d="M260.3,37.7c-4.8-1.1-9.6-1.5-9.6-4.6c0-2.5,2.8-2.9,4.7-2.9c3,0,5.5,0.9,5.8,4.2h8c-0.7-7.5-6.8-9.8-13.5-9.8
		c-6.6,0-13.4,2-13.4,9.9c0,5.4,4.9,7.1,9.7,8.2c6.1,1.3,9.6,2.2,9.6,4.8c0,3-3.1,3.9-5.5,3.9c-3.3,0-6.4-1.5-6.5-5.2h-8
		C242,54.3,248.9,57,256,57c7,0,14-2.6,14-10.7C270.1,40.6,265.2,38.8,260.3,37.7z"></path>
	<path class="st0" d="M289.8,37.7c-4.8-1.1-9.6-1.5-9.6-4.6c0-2.5,2.8-2.9,4.7-2.9c3,0,5.5,0.9,5.8,4.2h8c-0.7-7.5-6.8-9.8-13.5-9.8
		c-6.6,0-13.4,2-13.4,9.9c0,5.4,4.9,7.1,9.7,8.2c6.1,1.3,9.6,2.2,9.6,4.8c0,3-3.1,3.9-5.5,3.9c-3.3,0-6.4-1.5-6.5-5.2h-8
		c0.4,8.1,7.3,10.8,14.4,10.8c7,0,14-2.6,14-10.7C299.6,40.6,294.7,38.8,289.8,37.7z"></path>
	<path class="st0" d="M34.1,57.6c0.3-0.2,0.4-0.6,0.4-1c0-0.5-0.2-0.9-0.5-1.1c-0.3-0.2-0.8-0.3-1.4-0.3h-1.9v5.1h0.8V58h0.8
		l1.3,2.2h0.9l-1.4-2.2C33.6,57.9,33.9,57.8,34.1,57.6z M32.5,57.3h-0.9v-1.5h1c0.1,0,0.3,0,0.4,0c0.1,0,0.3,0,0.4,0.1
		c0.1,0,0.2,0.1,0.3,0.2c0.1,0.1,0.1,0.2,0.1,0.4c0,0.2,0,0.3-0.1,0.4c-0.1,0.1-0.2,0.2-0.3,0.2c-0.1,0.1-0.2,0.1-0.4,0.1
		C32.8,57.3,32.6,57.3,32.5,57.3z"></path>
	<path class="st0" d="M35.6,54.2c-0.4-0.4-0.9-0.7-1.4-0.9c-0.5-0.2-1.1-0.3-1.7-0.3c-0.6,0-1.2,0.1-1.7,0.3c-0.6,0.2-1,0.5-1.4,0.9
		c-0.4,0.4-0.7,0.9-1,1.5c-0.2,0.6-0.4,1.2-0.4,1.9c0,0.7,0.1,1.3,0.3,1.8c0.2,0.6,0.5,1,0.9,1.5c0.4,0.4,0.9,0.7,1.4,1
		c0.6,0.2,1.2,0.4,1.8,0.4c0.6,0,1.2-0.1,1.7-0.3c0.5-0.2,1-0.5,1.4-0.9c0.4-0.4,0.7-0.9,1-1.5c0.2-0.6,0.4-1.2,0.4-1.9
		c0-0.7-0.1-1.3-0.4-1.9C36.4,55.1,36,54.6,35.6,54.2z M35.7,59.1c-0.2,0.5-0.4,0.9-0.8,1.2c-0.3,0.3-0.7,0.6-1.1,0.8
		c-0.4,0.2-0.9,0.3-1.4,0.3c-0.6,0-1-0.1-1.5-0.3c-0.4-0.2-0.8-0.5-1.1-0.8c-0.3-0.3-0.5-0.7-0.7-1.2c-0.2-0.5-0.2-1-0.2-1.5
		c0-0.6,0.1-1.1,0.3-1.5c0.2-0.5,0.4-0.9,0.8-1.2c0.3-0.3,0.7-0.6,1.1-0.8c0.4-0.2,0.9-0.3,1.4-0.3c0.5,0,1,0.1,1.4,0.3
		c0.4,0.2,0.8,0.4,1.1,0.8c0.3,0.3,0.6,0.7,0.8,1.2c0.2,0.5,0.3,1,0.3,1.5C36,58.1,35.9,58.6,35.7,59.1z"></path>
</g>
</svg></a></div>
    <div class="footer__social-links">
        <ul class="rlist rlist--inline centered">
            <li><a href="http://on.fb.me/1tgHakh" title="Facebook" target="_blank"><i aria-hidden="true" class="icon-Icon_Facebook"></i></a></li>

            <li><a href="https://www.instagram.com/cellpress/" title="Instagram" target="_blank"><svg width="18px" height="18px" viewBox="0 0 18 18" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-147.000000, -16.000000)" fill-rule="nonzero" fill="#969696"> <g id="Instagram-Logo" transform="translate(147.000000, 16.000000)"> <path d="M9,1.62580645 C11.4096774,1.62580645 11.7,1.62580645 12.6290323,1.68387097 C13.5,1.71290323 13.9935484,1.85806452 14.3129032,2.00322581 C14.7193548,2.17741935 15.0387097,2.3516129 15.3580645,2.67096774 C15.6774194,2.99032258 15.8806452,3.28064516 16.0258065,3.71612903 C16.1419355,4.03548387 16.2870968,4.5 16.3451613,5.4 C16.3741935,6.35806452 16.4032258,6.61935484 16.4032258,9.02903226 C16.4032258,11.4387097 16.4032258,11.7290323 16.3451613,12.6580645 C16.316129,13.5290323 16.1709677,14.0225806 16.0258065,14.3419355 C15.8516129,14.7483871 15.6774194,15.0677419 15.3580645,15.3870968 C15.0387097,15.7064516 14.7483871,15.9096774 14.3129032,16.0548387 C13.9935484,16.1709677 13.5290323,16.316129 12.6290323,16.3741935 C11.6709677,16.4032258 11.4096774,16.4322581 9,16.4322581 C6.59032258,16.4322581 6.3,16.4322581 5.37096774,16.3741935 C4.5,16.3451613 4.00645161,16.2 3.68709677,16.0548387 C3.28064516,15.8806452 2.96129032,15.7064516 2.64193548,15.3870968 C2.32258065,15.0677419 2.11935484,14.7774194 1.97419355,14.3419355 C1.85806452,14.0225806 1.71290323,13.5580645 1.65483871,12.6580645 C1.62580645,11.7 1.62580645,11.4096774 1.62580645,9 C1.62580645,6.59032258 1.62580645,6.3 1.68387097,5.37096774 C1.71290323,4.5 1.85806452,4.00645161 2.00322581,3.68709677 C2.17741935,3.28064516 2.3516129,2.96129032 2.67096774,2.64193548 C2.99032258,2.32258065 3.28064516,2.11935484 3.71612903,1.97419355 C4.03548387,1.85806452 4.5,1.71290323 5.4,1.65483871 C6.3,1.62580645 6.59032258,1.62580645 9,1.62580645 L9,1.62580645 Z M9,0 C6.56129032,0 6.24193548,0 5.28387097,0.0580645161 C4.32580645,0.116129032 3.65806452,0.261290323 3.10645161,0.464516129 C2.52580645,0.696774194 2.00322581,1.01612903 1.50967742,1.50967742 C1.01612903,2.00322581 0.696774194,2.52580645 0.464516129,3.10645161 C0.232258065,3.68709677 0.0870967742,4.32580645 0.0580645161,5.28387097 C0,6.24193548 0,6.56129032 0,9 C0,11.4387097 0,11.7580645 0.0580645161,12.716129 C0.116129032,13.6741935 0.261290323,14.3419355 0.464516129,14.8935484 C0.696774194,15.4741935 1.01612903,15.9967742 1.50967742,16.4903226 C2.00322581,16.983871 2.52580645,17.3032258 3.10645161,17.5354839 C3.68709677,17.7677419 4.32580645,17.9129032 5.28387097,17.9419355 C6.24193548,18 6.56129032,18 9,18 C11.4387097,18 11.7580645,18 12.716129,17.9419355 C13.6741935,17.883871 14.3419355,17.7387097 14.8935484,17.5354839 C15.4741935,17.3032258 15.9967742,16.983871 16.4903226,16.4903226 C16.983871,15.9967742 17.3032258,15.4741935 17.5354839,14.8935484 C17.7677419,14.3129032 17.9129032,13.6741935 17.9419355,12.716129 C18,11.7580645 18,11.4387097 18,9 C18,6.56129032 18,6.24193548 17.9419355,5.28387097 C17.883871,4.32580645 17.7387097,3.65806452 17.5354839,3.10645161 C17.3032258,2.52580645 16.983871,2.00322581 16.4903226,1.50967742 C15.9967742,1.01612903 15.4741935,0.696774194 14.8935484,0.464516129 C14.3129032,0.232258065 13.6741935,0.0870967742 12.716129,0.0580645161 C11.7580645,0 11.4387097,0 9,0 L9,0 Z"></path> <path d="M9,4.38387097 C6.44516129,4.38387097 4.38387097,6.44516129 4.38387097,9 C4.38387097,11.5548387 6.44516129,13.616129 9,13.616129 C11.5548387,13.616129 13.616129,11.5548387 13.616129,9 C13.616129,6.44516129 11.5548387,4.38387097 9,4.38387097 Z M9,11.9903226 C7.34516129,11.9903226 6.00967742,10.6548387 6.00967742,9 C6.00967742,7.34516129 7.34516129,6.00967742 9,6.00967742 C10.6548387,6.00967742 11.9903226,7.34516129 11.9903226,9 C11.9903226,10.6548387 10.6548387,11.9903226 9,11.9903226 Z"></path> <circle id="Oval" cx="13.7903226" cy="4.20967742" r="1.07419355"></circle> </g> </g> </g></svg></a></li>
            
            <li><a href="https://www.linkedin.com/company/cell-press/" title="LinkedIn" target="_blank"><i aria-hidden="true" class="icon-linkedin"></i></a></li>
            
                        <li><a href="http://bit.ly/1zq7Q5T" title="Twitter" target="_blank">
                <!--<i aria-hidden="true" class="icon-Icon_Twitter"></i></a>-->
                
                <svg id="xicon" viewBox="0 0 1200 1227" xmlns="http://www.w3.org/2000/svg" aria-hidden="true" role="none"><path fill="#969696" d="M714.163 519.284L1160.89 0H1055.03L667.137 450.887L357.328 0H0L468.492 681.821L0 1226.37H105.866L515.491 750.218L842.672 1226.37H1200L714.137 519.284H714.163ZM569.165 687.828L521.697 619.934L144.011 79.6944H306.615L611.412 515.685L658.88 583.579L1055.08 1150.3H892.476L569.165 687.854V687.828Z"></path></svg> 

                    <style>
                    .footer svg#xicon {
                      height: 16px;
                    }
                    </style>
            </a></li>

            
            <li><a href="/pb-assets/marketing/WeChat/WeChat-1557753222727.jpg" title="WeChat" target="_blank"><svg width="18px" height="15px" viewBox="0 0 18 15" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-122.000000, -17.000000)" fill-rule="nonzero" fill="#969696"> <g id="wechat" transform="translate(122.000000, 17.000000)"> <path d="M12.375,4.5 L12.375,4.5 L12.4875,4.5 L12.4875,4.5 C12.6898579,4.48604473 12.8520408,4.32704187 12.87,4.125 C12.8705905,4.04725572 12.8470025,3.9712499 12.8025,3.9075 C11.94,1.5675 9.4425,0 6.5625,0 C2.9475,0 0,2.52 0,5.625 C0.0313772094,7.34410122 0.895513317,8.94093526 2.3175,9.9075 L1.5675,11.46 C1.49009368,11.6087816 1.52088865,11.7908702 1.64291494,11.9059236 C1.76494122,12.0209769 1.94852501,12.0410168 2.0925,11.955 L4.08,10.8225 C4.61861992,11.0129999 5.17724298,11.1413322 5.745,11.205 C5.86564124,11.2178337 5.98503738,11.1715098 6.06545426,11.0806685 C6.14587113,10.9898272 6.17737153,10.865694 6.15,10.7475 C6.06148727,10.4219419 6.01114891,10.0871918 6,9.75 C6,6.855 8.8575,4.5 12.375,4.5 Z M8.25,3 C8.66421356,3 9,3.33578644 9,3.75 C9,4.16421356 8.66421356,4.5 8.25,4.5 C7.83578644,4.5 7.5,4.16421356 7.5,3.75 C7.5,3.33578644 7.83578644,3 8.25,3 Z M4.5,4.5 C4.08578644,4.5 3.75,4.16421356 3.75,3.75 C3.75,3.33578644 4.08578644,3 4.5,3 C4.91421356,3 5.25,3.33578644 5.25,3.75 C5.25,4.16421356 4.91421356,4.5 4.5,4.5 Z"></path> <path d="M18,9.75 C18,7.2675 15.48,5.25 12.375,5.25 C9.27,5.25 6.75,7.2675 6.75,9.75 C6.75,12.2325 9.27,14.25 12.375,14.25 C12.9568557,14.2502837 13.5364258,14.1772075 14.1,14.0325 L15.96,14.9625 C16.0114248,14.9874074 16.0678613,15.0002339 16.125,15 C16.2497193,14.9993787 16.3659682,14.9367912 16.4351508,14.8330172 C16.5043335,14.7292432 16.5173955,14.5978644 16.47,14.4825 L15.96,13.2075 C17.1784092,12.4716325 17.9450305,11.1723221 18,9.75 L18,9.75 Z M10.5,9 C10.0857864,9 9.75,8.66421356 9.75,8.25 C9.75,7.83578644 10.0857864,7.5 10.5,7.5 C10.9142136,7.5 11.25,7.83578644 11.25,8.25 C11.25,8.66421356 10.9142136,9 10.5,9 Z M14.25,9 C13.8357864,9 13.5,8.66421356 13.5,8.25 C13.5,7.83578644 13.8357864,7.5 14.25,7.5 C14.6642136,7.5 15,7.83578644 15,8.25 C15,8.66421356 14.6642136,9 14.25,9 Z" opacity="0.959999979"></path> </g> </g> </g></svg></a></li>
            
            <li><a href="https://www.weibo.com/CellPress" title="Weibo" target="_blank"><svg width="18px" height="15px" viewBox="0 0 18 15" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-99.000000, -17.000000)" fill-rule="nonzero" fill="#969696"> <g id="weibo" transform="translate(99.000000, 14.000000)"> <g transform="translate(0.000000, 3.000000)"> <path d="M13.2402211,7.47054172 C12.9918392,7.39866484 12.821608,7.34902326 12.9514975,7.03290287 C13.232804,6.34602189 13.2622915,5.75359781 12.9571055,5.33095486 C12.3847236,4.5380684 10.8191759,4.58064295 9.02514573,5.30975376 C9.02514573,5.30889193 8.46144724,5.54882627 8.60544724,5.11515185 C8.88150754,4.25435294 8.83953769,3.53386047 8.41043216,3.11759508 C7.43716583,2.17285363 4.84824121,3.15344733 2.62745729,5.3059617 C0.964944724,6.9186238 0,8.62798358 0,10.105855 C0,12.9324952 3.73748744,14.8235294 7.39429146,14.8235294 C12.1877186,14.8235294 15.3768844,11.9501778 15.3768844,9.80576471 C15.3768844,8.50974008 14.2514774,7.77442408 13.2402211,7.47054172 Z M7.40424121,13.5561176 C4.48625126,13.8315595 1.9678794,12.5686293 1.77811055,10.7327524 C1.58870352,8.89773735 3.80098492,7.18579207 6.71861307,6.90948837 C9.63678392,6.63266758 12.1556985,7.89663201 12.3447437,9.73164706 C12.533608,11.5675239 10.3227739,13.2796416 7.40424121,13.5561176 Z M18,5.16875787 C17.9985528,2.31402189 15.5695477,0 12.5728643,0 C12.222995,0 11.9396985,0.270098495 11.9396985,0.603283174 C11.9396985,0.936640219 12.222995,1.20656635 12.5728643,1.20656635 C14.8708945,1.20656635 16.7336683,2.98159781 16.7336683,5.17099863 C16.7336683,5.50435568 17.0173266,5.77428181 17.3668342,5.77428181 C17.7167035,5.77428181 18,5.50435568 18,5.17099863 C18,5.17099863 18,5.1696197 18,5.16875787 Z M15.4841608,5.08016142 C15.1948945,3.73415048 14.0812462,2.67323393 12.6683819,2.39744733 C12.3262915,2.33039672 11.9914372,2.54051163 11.9212462,2.86680164 C11.8506935,3.19309166 12.0712161,3.51179754 12.4133065,3.57919289 C13.3319397,3.75845417 14.0559196,4.44774829 14.2438794,5.32302599 C14.3146131,5.64931601 14.6491055,5.85925855 14.9915578,5.7923803 C15.3338291,5.72498495 15.5541709,5.40627907 15.4841608,5.08016142 Z M5.90761809,8.7069275 C4.7321005,8.9320383 3.93901508,9.87126402 4.1358392,10.8044569 C4.33248241,11.7378222 5.44504523,12.311803 6.62056281,12.0865198 C7.79571859,11.861409 8.58916583,10.922528 8.3921608,9.98899042 C8.19569849,9.0559699 7.08277387,8.48181669 5.90761809,8.7069275 Z"></path> </g> </g> </g> </g></svg></a></li>
            
            <li><a href="https://www.youtube.com/user/cellvideoabstracts" title="YouTube" target="_blank"><svg width="18px" height="13px" viewBox="0 0 18 13" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-76.000000, -19.000000)" fill-rule="nonzero" fill="#969696"> <g id="youtube" transform="translate(76.000000, 14.000000)"> <path d="M17.2984219,6.41974219 C16.6486641,5.64735937 15.4490625,5.33232422 13.1580703,5.33232422 L4.84175391,5.33232422 C2.49834375,5.33232422 1.27838672,5.66771484 0.631089844,6.49001953 C0,7.29172266 0,8.47304297 0,10.1080547 L0,13.2243047 C0,16.3917773 0.748792969,18 4.84175391,18 L13.1581055,18 C15.1448555,18 16.2457383,17.7219844 16.9579688,17.040375 C17.688375,16.3413984 18,15.2001211 18,13.2243047 L18,10.1080547 C18,8.38381641 17.951168,7.19553516 17.2984219,6.41974219 Z M11.5560703,12.0963867 L7.77965625,14.0700586 C7.69524609,14.1141797 7.60289062,14.136082 7.51071094,14.136082 C7.40629687,14.136082 7.30212891,14.1079922 7.20987891,14.0521289 C7.03613672,13.9468711 6.93007031,13.7585391 6.93007031,13.5554414 L6.93007031,9.62075391 C6.93007031,9.41800781 7.03585547,9.22985156 7.20924609,9.12452344 C7.38267187,9.01919531 7.59839062,9.01198828 7.77832031,9.10546875 L11.5547344,11.0664141 C11.7468633,11.1661523 11.8675195,11.3645039 11.8678008,11.5809258 C11.8680469,11.7975586 11.747918,11.9961914 11.5560703,12.0963867 Z"></path> </g> </g> </g></svg></a></li>
            
        </ul>
    </div>
</div>

<style>

.ux3 .footer__logo svg {
    max-width: 160px !important;
}

.ux3 .footer__social-links a svg {
    max-width: 26px;
    height: 1.15rem;
    width: auto;
}

.ux3 .footer__social-links a:hover svg path {
    fill: #353535;
}

</style></div>

            

        

    

        


    

            

            
                <div class="footer__bottom text-onDark">
            

            <div data-widget-id="06227714-c621-4651-9879-b0ce5176cccc" data-widget-name="ux3-layout-widget"><div class="col-sm-4 col-md-5 clearfix">
    

        


    

            

            

            <div data-widget-id="07e1d276-b4dd-4870-b858-ed74e70d64ef" data-widget-name="UX3HTMLWidget"><div class="groups">
	<div class="group">
		<h2 class="group__title">Life &amp; medical sciences journals</h2>
		<ul class="rlist clearfix sided">
		<li>
			<a alt="Cell" href="/cell/home">Cell</a>
		</li>
		<li>
			<a alt="Cancer Cell" href="/cancer-cell/home">Cancer Cell</a>
		</li>
		<li>
			<a alt="Cell Chemical Biology" href="/cell-chemical-biology">Cell Chemical Biology</a>
		</li>
		<li>
			<a alt="Cell Genomics" href="/cell-genomics/home">Cell Genomics</a>
		</li>
		<li>
			<a alt="Cell Host and Microbe" href="/cell-host-microbe/home">Cell Host &amp; Microbe</a>
		</li>
		<li>
			<a alt="Cell Metabolism" href="/cell-metabolism/home">Cell Metabolism</a>
		</li>
		<li>
			<a alt="Cell Reports" href="/cell-reports/home">Cell Reports</a>
		</li>
		<li>
			<a alt="Cell Reports Medicine" href="/cell-reports-medicine/home">Cell Reports Medicine</a>
		</li>
		<li>
			<a alt="Cell Stem Cell" href="/cell-stem-cell/home">Cell Stem Cell</a>
		</li>
		<li>
			<a alt="Cell Systems" href="/cell-systems/home">Cell Systems</a>
		</li>
		<li>
			<a alt="Current Biology" href="/current-biology/home">Current Biology</a>
		</li>
		<li>
			<a alt="Developmental Cell" href="/developmental-cell/home">Developmental Cell</a>
		</li>
		<li>
			<a alt="Immunity" href="/immunity/home">Immunity</a>
		</li>
		<li>
			<a alt="Med" href="/med/home">Med</a>
		</li>
		<li>
			<a alt="Molecular Cell" href="/molecular-cell/home">Molecular Cell</a>
		</li>
		<li>
			<a alt="Neuron" href="/neuron/home">Neuron</a>
		</li>
		<li>
			<a alt="Structure" href="/structure/home">Structure</a>
		</li>
		<li>
			<a alt="American Journal of Human Genetics" href="/ajhg/home">American Journal of Human Genetics (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Biophysical Journal" href="/biophysj/home">Biophysical Journal (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Biophysical Reports" href="/biophysreports/home">Biophysical Reports (<i>partner</i>)</a>
		</li>	
		<li>
			<a alt="HGG Advances" href="/hgg-advances/home">Human Genetics and Genomics Advances (<i>partner</i>)</a></li>
		<li>
			<a alt="Molecular Plant" href="/molecular-plant/home">Molecular Plant (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy" href="/molecular-therapy-family/molecular-therapy/home">Molecular Therapy (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy Methods and Clinical Development" href="/molecular-therapy-family/methods/home">Molecular Therapy Methods &amp; Clinical Development (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy Nucleic Acids" href="/molecular-therapy-family/nucleic-acids/home">Molecular Therapy Nucleic Acids (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy Oncology" href="/molecular-therapy-family/oncology/home">Molecular Therapy Oncology (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Plant Communications" href="/plant-communications/home">Plant Communications (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Stem Cell Reports" href="/stem-cell-reports/home">Stem Cell Reports (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Trends in Biochemical Sciences" href="/trends/biochemical-sciences/home">Trends in Biochemical Sciences</a>
		</li>
		<li>
		</li><li>
			<a alt="Trends in Cancer" href="/trends/cancer/home">Trends in Cancer</a>
		</li>
		<li>
			<a alt="Trends in Cell Biology" href="/trends/cell-biology/home">Trends in Cell Biology</a>
		</li>
		<li>
			<a alt="Trends in Ecology and Evolution" href="/trends/ecology-evolution/home">Trends in Ecology &amp; Evolution</a>
		</li>
		<li>
			<a alt="Trends in Endocrinology and Metabolism" href="/trends/endocrinology-metabolism/home">Trends in Endocrinology &amp; Metabolism</a>
		</li>
		<li>
			<a alt="Trends in Genetics" href="/trends/genetics/home">Trends in Genetics</a>
		</li>
		<li>
			<a alt="Trends in Immunology" href="/trends/immunology/home">Trends in Immunology</a>
		</li>
		<li>
			<a alt="Trends in Microbiology" href="/trends/microbiology/home">Trends in Microbiology</a>
		</li>
		<li>
			<a alt="Trends in Molecular Medicine" href="/trends/molecular-medicine/home">Trends in Molecular Medicine</a>
		</li>
		<li>
			<a alt="Trends in Neurosciences" href="/trends/neurosciences/home">Trends in Neurosciences</a>
		</li>
		<li>
			<a alt="Trends in Parasitology" href="/trends/parasitology/home">Trends in Parasitology</a>
		</li>
		<li>
			<a alt="Trends in Pharmacological Sciences" href="/trends/pharmacological-sciences/home">Trends in Pharmacological Sciences</a>
		</li>
		<li>
			<a alt="Trends in Plant Science" href="/trends/plant-science/home">Trends in Plant Science</a>
		</li>
		</ul>
	</div>

	<div class="group">
		<h2 class="group__title">Physical sciences &amp; engineering journals</h2>
		<ul class="rlist clearfix sided">
		  <li><a alt="Cell Biomaterials" href="/cell-biomaterials/home">Cell Biomaterials</a></li>
		  <li>
			<a alt="Cell Reports Physical Science" href="/cell-reports-physical-science/home">Cell Reports Physical Science</a>
		</li>
		<li>
			<a alt="Chem" href="/chem/home">Chem</a>
		</li>
		<li>
			<a alt="Chem Catalysis" href="/chem-catalysis/home">Chem Catalysis</a>
		</li>
		<li>
			<a alt="Device" href="/device/home">Device</a>
		</li>
		<li>
			<a alt="Joule" href="/joule/home">Joule</a>
		</li>
		<li>
			<a alt="Matter" href="/matter/home">Matter</a>
		</li>	
		<li><a alt="Matter" href="/newton/home">Newton</a></li>
		<li>
			<a alt="Trends in Chemistry" href="/trends/chemistry/home">Trends in Chemistry</a>
		</li>			
		</ul>
	</div>

<div class="group">
	<h2 class="group__title">Multidisciplinary journals</h2>
	<ul class="rlist clearfix sided">
	  		<li>
			<a alt="Cell Reports Methods" href="/cell-reports-methods/home">Cell Reports Methods</a>
		</li>
		<li>
			<a alt="Cell Reports Sustainability" href="/cell-reports-sustainability/home">Cell Reports Sustainability</a>
		</li>
		<li>
			<a alt="Heliyon" href="/heliyon/home">Heliyon</a>
		</li>	
		<li>
			<a alt="iScience" href="/iscience/home">iScience</a>
		</li>
		<li>
			<a alt="One Earth" href="/one-earth/home">One Earth</a>
		</li>
		<li>
			<a alt="Patterns" href="/patterns/home">Patterns</a>
		</li>
		<li>
			<a alt="STAR Protocols" href="/star-protocols/home">STAR Protocols</a>
		</li>
		<li>
			<a alt="Nexus" href="/nexus/home">Nexus (<i>partner</i>)</a>
		</li>			
        <li>
			<a alt="The Innovation" href="/the-innovation/home">The Innovation (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Trends in Biotechnology" href="/trends/biotechnology/home">Trends in Biotechnology</a>
		</li>
		<li>
			<a alt="Trends in Cognitive Sciences" href="/trends/cognitive-sciences/home">Trends in Cognitive Sciences</a>
		</li>
	</ul>
</div>

</div>
</div>

            

        
</div><div class="col-sm-4 col-md-5 clearfix">
    

        


    

            

            

            <div data-widget-id="850476de-00cb-4c0d-943c-bf1881da24f9" data-widget-name="UX3HTMLWidget"><div class="groups">
	<div class="clearfix">
		<div class="group sided">
			<h2 class="group__title">Authors</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Submit article" href="/submit-to-cell-press">Submit article</a>
				</li>
				<li>
					<a alt="Multi-Journal Submission" href="/multi-journal-submission">Multi-Journal Submission</a>
				</li>
				<li>
					<a alt="STAR Methods" href="/star-authors-guide">STAR Methods</a>
				</li>
				<li>
					<a alt="Sneak Peek – Preprints" href="/sneakpeek">Sneak Peek – Preprints</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
		<h2 class="group__title">Reviewers</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Reviewers" href="/reviewers">Information for reviewers</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
		<h2 class="group__title">News &amp; events</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Newsroom" href="/newsroom">Newsroom</a>
				</li>
				<li>
					<a alt="Events" href="/events">Events</a>
				</li>
				<li>
					<a alt="Cell Symposia" href="/symposia">Cell Symposia</a>
				</li>
				<li>
					<a alt="Consortia Hub" href="/consortia">Consortia Hub</a>
				</li>
				<li>
					<a alt="Webinars" href="/webinars">Webinars</a>
				</li>
								<li>
					&nbsp;
				</li>
			</ul>
		<h2 class="group__title">Multimedia</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Cell Press Podcast" href="/podcast">Cell Press Podcast</a>
				</li>
				<li>
					<a alt="Cell Press Videos" href="/video">Cell Press Videos</a>
				</li>
				<li>
					<a alt="Color and Comics" href="/comics">Coloring and Comics</a>
				</li>
				<li>
					<a alt="Figure360" href="/video-guidelines">Figure360</a>
				</li>
				<li>
					<a alt="Cell Picture Show" href="/pictureshow">Cell Picture Show</a>
				</li>
				<li>
					<a alt="Research Arc" href="/research-arc">Research Arc</a>
				</li>
			</ul>
		</div>

		<div class="group sided">
			<h2 class="group__title">About</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="About Cell Press" href="/about">About Cell Press</a>
				</li>

				<li>
					<a alt="Open access" href="/open-access">Open access</a>
				</li>
				<li>
					<a alt="COVID Hub" href="/covid-19">COVID Hub</a>
				</li>
				<li>
					<a alt="Sustainability" href="/sustainability">Sustainability hub</a>
				</li>
				<li>
					<a alt="Inclusion and diversity" href="/diversity">Inclusion and diversity</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
			<h2 class="group__title">Contact</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Contact us" href="/contact">Contact us</a>
				</li>
				<li>
					<a alt="Help and Support" href="https://service.elsevier.com/app/overview/cell-press/" target="_blank">Help &amp; Support</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
			<h2 class="group__title">Careers</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Cell Press Careers" href="https://relx.wd3.myworkdayjobs.com/en-US/ElsevierJobs" target="_blank">Cell Press Careers</a>
				</li>
			</ul>
		</div>
	</div>
</div></div>

            

        
</div><div class="col-sm-4 col-md-2 clearfix">
    

        


    

            

            

            <div data-widget-id="beafac41-1cfc-440b-9b55-74aba4a5612c" data-widget-name="UX3HTMLWidget"><div class="groups">
	<div class="group">
		<h2 class="group__title">Access</h2>
		<ul class="rlist clearfix">
			<li>
				<a alt="Subscribe" href="/subscribe">Subscribe</a>
			</li>
			<li>
				<a alt="Claim" href="/activate">Claim</a>
			</li>
			<li>
				<a alt="Read It Now" href="/read-it-now">Read-It-Now</a>
			</li>
			<li>
				<a alt="Recommend to Librarian" href="http://info.cell.com/request-access-to-cell-press-content" target="_blank">Recommend to Librarian</a>
			</li>
			<li>
				<a alt="Publication Alerts" href="/action/showPreferences?menuTab=Alerts">Publication Alerts</a>
			</li>
		</ul>
	</div>
	<div class="group">
		<h2 class="group__title">Collections</h2>
		<ul class="rlist clearfix">
			<li>
				<a alt="Best of Cell Press" href="/bestof">Best of Cell Press</a>
			</li>
			<li>
				<a alt="Cell Press Reviews" href="/reviews">Cell Press Reviews</a>
			</li>
			<li>
				<a alt="Cell Selections" href="/selections">Cell Press Selections</a>
			</li>
			<li>
				<a alt="Nucleus Collections" href="/nucleus">Nucleus Collections</a>
			</li>
			<li>
				<a alt="SnapShots" href="/snapshots">SnapShot Archive</a>
			</li>
		</ul>
	</div>
	<div class="group">
		<h2 class="group__title">Information</h2>
		<ul class="rlist clearfix">
			<li>
				<a alt="For Advertisters" href="/media-kit">For Advertisers</a>
			</li>
			<li>
				<a alt="For Recruiters" target="_blank" href="/pb-assets/Cell/common/CP-RecruitmentKit-US-2018-LowRes-1712070648.pdf">For Recruiters</a>
			</li>
			<li>
				<a alt="For Librarians" target="_blank" href="https://www.elsevier.com/products/sciencedirect/journals/direct-subscription-journals">For Librarians</a>
			</li>
			<li>
				<a alt="Privacy Policy" target="_blank" href="https://www.elsevier.com/legal/privacy-policy">Privacy Policy</a>
			</li>
			<li>
				<a alt="Terms and Conditions" target="_blank" href="https://www.elsevier.com/legal/elsevier-website-terms-and-conditions">Terms and Conditions</a>
			</li>
			<li>
				<a alt="Accessibility" href="/accessibility">Accessibility</a>
			</li>
		</ul>
	</div>
</div></div>

            

        
</div></div>

            
                </div>
            

        

    

        


    

            

            

            <div data-widget-id="779ad11f-7bcd-4a2a-a2cd-8c67b649cbb2" data-widget-name="UX3HTMLWidget"><style>
 
button#cookieBtn {
   text-decoration: none;
   background-color: transparent;
   color: #005789;
   font-weight: bold;
   padding: 0;
   margin: 0;
   font-size: 14px;
}
 
button#cookieBtn:hover {
   text-decoration: underline;
}

</style>
<script>
// HIDE ONETRUST BACKGROUND AFTER USER PRESSES ESCAPE KEY
setTimeout(function(){ 
   $("#onetrust-pc-sdk").on("keydown", function(e) { 
      if (e.keyCode == 27) {  // ESC key
         setTimeout(function(){ 
            $(".onetrust-pc-dark-filter").css("display","none");
         }, 100); 
      }
   });
}, 5000);
</script>

<span class="align-center copy-right">
<hr>
<p align="center"><b>The content on this site is intended for healthcare professionals and researchers across all fields of science.</b></p>
<hr>
<div>
<p>We use cookies to help provide and enhance our service and tailor content. To update your cookie settings, please visit the 
      <b><button id="cookieBtn" class="ot-sdk-show-settings" type="button" aria-label="Manage cookie consent preferences">Cookie Settings</button></b> for this site.<br>

All content on this site: Copyright © 2025 Elsevier Inc., its licensors, and contributors. <br>All rights are reserved, including those for text and data mining, AI training, and similar technologies. <br>For all open access content, the relevant licensing terms apply.
</p>
<ul class="rlist--inline">
			<li>
				<a alt="Privacy Policy" target="_blank" href="https://www.elsevier.com/legal/privacy-policy">Privacy Policy</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Terms and Conditions" target="_blank" href="https://www.elsevier.com/legal/elsevier-website-terms-and-conditions">Terms &amp; Conditions</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Accessibility" href="/accessibility">Accessibility</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Help and Support" target="_blank" href="https://service.elsevier.com/app/overview/cell-press/">Help &amp; Support</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Contact" href="/contact">Contact</a>
			</li>
			</ul>
<p>
<a href="https://www.relx.com"><img src="/pb/assets/raw/Health%20Advance/images/relx-1554393210983.png" alt="RELX" rel="nofollow" target="_new"></a></p>
</div>

</span></div>

            

        

    

        


    

            

            

            <div data-widget-id="c5e179c1-156e-41e3-af7f-5e307efa8acf" data-widget-name="UX3HTMLWidget"><style>#brandingRow{
    background-color:#fff;
    padding:7px 26px 7px 30px;
    color:#007DBC;
    font-size:14px;
    clear:both;
    width:100%
}
#brandingRow a{
    text-decoration:none;
    color:#007DBC
}
#brandingRow a:hover{
    text-decoration:underline
}
#brandingRowRight{
    display:inline;
    float:right;
    width:auto;
    padding:0;
    margin:0
}
svg#cp_logo {
  float: none;
  height: 36px;
  width: 36px;
  margin-left: 0px;
  margin-top: 0px;
}

footer hr {
  display: block;
}

@media screen and (max-width: 600px){
    #brandingRow{
        font-size:12px
    }
}
@media screen and (max-width: 750px){
    #brandingRow{
        padding:7px 18px 7px 16px
    }
    svg#cp_logo{
        margin-right:10px
    }
}
@media screen and (max-width: 800px) {
   a#cpIconLnk {
      display: none;
   }
}

.header__wrapper.clearfix .col-xs-6:first-of-type{
    width:55%
}
.header__wrapper.clearfix .col-xs-6:nth-of-type(2){
    width:45%
}
@media screen and (min-width: 800px) and (max-width: 1000px){
    .header__wrapper.clearfix .col-xs-6:first-of-type{
        width:75%
    }
    .header__wrapper.clearfix .col-xs-6:nth-of-type(2){
        width:25%
    }
}

</style>

<script>
var partnerx = false;
var overrideEndorsement = "";
var tmpj = $("input[name='journalCode']").attr("value");
	
if (window.location.pathname == "/molecular-therapy-family/home") {
	tmpj = "ymthe";
}
	
var ribbonx = "";
if (tmpj == "ajhg" || tmpj == "bpj" || tmpj == "molp" || tmpj == "xplc" || tmpj == "stemcr" || tmpj == "ymthe" || tmpj == "xhgg" || tmpj == "omtn" || tmpj == "omto" || tmpj == "omtm" || tmpj == "biophysreports" ||  tmpj == "bpr" || tmpj == "xinn" || tmpj == "omton") {
	partnerx = true;
	ribbonx = "<a href='https://www.cell.com' data-aa-name='cell press endorsement' data-aa-region='cp global header' title='Visit Cell.com'>A Cell Press partner journal</a>";
} else if (tmpj != "" && tmpj != "undefined" && tmpj != undefined) {
	ribbonx = "<a href='/' data-aa-name='cell press endorsement' data-aa-region='cp global header' title='Visit Cell.com'>A Cell Press journal</a>";
} else {
	ribbonx = "<a href='/' data-aa-name='cell press endorsement' data-aa-region='cp global header' title='Visit Cell.com'>A Cell Press resource</a>";
}
	
if (window.location.pathname == "/") {
	ribbonx = "";
}
if (overrideEndorsement != "") {
	ribbonx = overrideEndorsement;
}

$(".header__wrapper.clearfix").prepend("<div id='brandingRow'><a href='/' data-aa-name='science that inspires' data-aa-region='cp global header' title='Visit Cell.com'>Science that inspires</a><p id='brandingRowRight'>" + ribbonx + "</p></div>");
	
// handle scrolling effect on article pages
if ($("article").length > 0) {
	var scrollx;
	$(window).scroll(function() {
		scrollx = $(window).scrollTop();
		if (scrollx > 0) { 
			$("#brandingRow").css("display","none"); 
		} else {
			$("#brandingRow").css("display","block");
		}
	});
}

if (window.location.pathname == "/" || partnerx == false) {
               $("<a href='/home' id='cpIconLnk' data-aa-name='cell press logo' data-aa-region='cp global header' title='Cell Press' target='_blank'><svg id='cp_logo' version='1.1' xmlns='http://www.w3.org/2000/svg' xmlns:xlink='http://www.w3.org/1999/xlink' x='0px' y='0px' viewBox='0 0 100 98.1' style='enable-background:new 0 0 100 98.1;' xml:space='preserve' height='100' width='98.1'><path style='fill:#FFFFFF;' d='M69.09,0.09c-4.01-0.6-7.83,1.94-8.78,5.89l-3.14,13.11c-0.76,3.19-3.44,5.56-6.7,5.93l-18.6,2.13          C11.83,29.31-2.53,47.66,0.37,67.78c2.3,15.96,15.18,27.77,30.54,30.08c4.01,0.6,7.83-1.94,8.78-5.89l3.14-13.11 c0.76-3.19,3.44-5.56,6.69-5.93l18.6-2.13c20.04-2.17,34.41-20.52,31.5-40.65C97.32,14.21,84.45,2.39,69.09,0.09z M65.06,47.2  l-11.75,1.4c-1.98,0.21-3.62,1.65-4.08,3.59l-3.43,14.3c-1.01,4.22-4.5,7.39-8.8,8c-6.66,0.72-12.63-4.2-13.15-10.92                c-0.51-6.5,4.6-12.11,11.08-12.81l11.75-1.39c1.98-0.22,3.62-1.65,4.08-3.59l3.43-14.3c1.01-4.22,4.5-7.39,8.8-8          c6.66-0.72,12.63,4.19,13.15,10.92C76.66,40.89,71.55,46.5,65.06,47.2z'/></svg></a>").insertBefore(".header__wrapper.clearfix .journal-logos:eq(0)");
}


</script></div>

            

        

    

        


    

            

            

            <div data-widget-id="1af759b7-9bb6-4413-ada9-51a8333e8fb8" data-widget-name="UX3HTMLWidget"><style>
    /*This was added temporarily until the PB changelog tab issue got fixed. [LIT-444969 - c#4]*/
    [data-tab-id="changelogTab"] {
        display: none !important;
    }
    
    /*This was added temporarily until the PB changelog tab issue got fixed. [LIT-518839]*/
    @media (min-width: 769px) {
        .ux3 .quick-search .dropBlock [data-db-target-of],
        .quick-search .dropBlock [data-db-target-of] {
            width: 100vw !important;
        }
    }
</style></div>

            

        
</div></div>

            

        

        </div>
    

    

        


    

            

            

            <div data-widget-id="d7e0813b-a300-497a-ab37-9c3ddaf03e71" data-widget-name="UX3HTMLWidget"><!--
<div class="session-reminder hidden">
    <div class="session-reminder__overlay">
        <div class="session-reminder__dialog">
            <div class="session-reminder__dialog-inner">
                <div class="dialog-header" role="dialog" aria-labelledby="dialog-title" aria-describedby="dialog-desc">
                    <h2 class="dialog-header-title" id="dialog-title">
                        <span>Session Timeout</span>
                        <div class="dialog-timer">(2:00)</div>
                    </h2>
                    <button class="dialog-close" type="button" aria-label="close dialog">
                        <i class="icon-tools_close" aria-hidden="true"></i>
                    </button>
                </div>
                <div class="dialog-body">
                    <p class="dialog-desc" id="dialog-desc">Your session will expire shortly. If you are still working, click the ‘Keep Me Logged In’ button below. If you do not respond within the next minute, you will be automatically logged out.</p>
                    <p class="dialog-expired sr-only" aria-live="polite"></p>
                </div>
                <div class="dialog-footer">
                    <button class="session-reset" type="button">Keep Me Logged In</button>
                </div>
            </div>
        </div>
    </div>
</div>
<a class="sr-only" href="javascript:void(0)" aria-hidden="true"></a>
--></div>

            

        
        </div>
    </div>


































    



















        
        
            
                <script src="/products/marlin/cell/releasedAssets/js/main.bundle-90475670808d5d3eee41.js"></script>
            
        
        
        

    




<script type="text/javascript">
    var pageData =  {"journal":{"specialty":"Cell Press","issn":"0092-8674","name":"Cell"},"page":{"loadTimestamp":"1749590959742","server":"cell","environment":"prod","businessUnit":"ELS:RAP:JBS","noTracking":"","name":"fulltext:html","section":"Cell","language":"en","type":"CP-CA","productName":"JB"},"visitor":{"departmentName":"|ae:ScienceDirect Guests","accessType":"ae:ANON_GUEST|ae:ANON_GUEST","accountId":"ae:838949|ae:228598","accountName":"ae:Cell Guest Account|ae:ScienceDirect Guests","departmentId":"ae:977212|ae:291352","ipAddress":"189.120.74.179","loginStatus":"anonymous","userId":"ae:246228266|ae:12975512"},"content":[{"accessType":"article:free:openarchive","volumeNumber":"184","issueNumber":"15","viewState":"fulltext","format":"MIME-HTML","id":"pii:S0092867421007534","coverDate":"202107","type":"XOCS-JOURNAL:SCOPE-FULL","title":"GABAreceptive microglia selectively sculpt developing inhibitory circuits","status":"S300"}]};
</script>

<script type="text/javascript">
    if(typeof pageDataTracker !== "undefined") {
        pageDataTracker.trackPageLoad();
    }
</script>
<script>(function(){function c(){var b=a.contentDocument||a.contentWindow.document;if(b){var d=b.createElement('script');d.innerHTML="window.__CF$cv$params={r:'94dbf6a2282352eb',t:'MTc0OTU5MDk1OC4wMDAwMDA='};var a=document.createElement('script');a.nonce='';a.src='/cdn-cgi/challenge-platform/scripts/jsd/main.js';document.getElementsByTagName('head')[0].appendChild(a);";b.getElementsByTagName('head')[0].appendChild(d)}}if(document.body){var a=document.createElement('iframe');a.height=1;a.width=1;a.style.position='absolute';a.style.top=0;a.style.left=0;a.style.border='none';a.style.visibility='hidden';document.body.appendChild(a);if('loading'!==document.readyState)c();else if(window.addEventListener)document.addEventListener('DOMContentLoaded',c);else{var e=document.onreadystatechange||function(){};document.onreadystatechange=function(b){e(b);'loading'!==document.readyState&&(document.onreadystatechange=e,c())}}}})();</script><iframe height="1" width="1" style="position: absolute; top: 0px; left: 0px; border: none; visibility: hidden;"></iframe><script defer="" src="https://static.cloudflareinsights.com/beacon.min.js/vcd15cbe7772f49c399c6a5babf22c1241717689176015" integrity="sha512-ZpsOmlRQV6y907TI0dKBHq9Md29nnaEIPlkf84rnaERnq6zvWvPUqr2ft8M1aS28oN72PdrCzSjY4U6VaAw1EQ==" data-cf-beacon="{&quot;rayId&quot;:&quot;94dbf6a2282352eb&quot;,&quot;serverTiming&quot;:{&quot;name&quot;:{&quot;cfExtPri&quot;:true,&quot;cfEdge&quot;:true,&quot;cfOrigin&quot;:true,&quot;cfL4&quot;:true,&quot;cfSpeedBrain&quot;:true,&quot;cfCacheStatus&quot;:true}},&quot;version&quot;:&quot;2025.5.0&quot;,&quot;token&quot;:&quot;57a02315956f4b2e82817b3dfa99b1a2&quot;}" crossorigin="anonymous"></script>


<div id="addtoany" style="position: static;"><div class="a2a_overlay" id="a2a_overlay"></div><div id="a2a_modal" class="a2a_modal a2a_hide" role="dialog" tabindex="-1" aria-label=""><div class="a2a_modal_body a2a_menu a2a_hide" id="a2a_copy_link"><label for="a2a_copy_link_text" id="a2a_copy_link_icon" class="a2a_svg a2a_s_link a2a_logo_color"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"><title>Copy link</title><path fill="#FFF" d="M7.591 21.177c0-.36.126-.665.377-.917l2.804-2.804a1.235 1.235 0 0 1 .913-.378c.377 0 .7.144.97.43-.026.028-.11.11-.255.25-.144.14-.24.236-.29.29a2.82 2.82 0 0 0-.2.256 1.056 1.056 0 0 0-.177.344 1.43 1.43 0 0 0-.046.37c0 .36.126.666.377.918a1.25 1.25 0 0 0 .918.377c.126.001.251-.015.373-.047.125-.037.242-.096.345-.175.09-.06.176-.127.256-.2.1-.094.196-.19.29-.29.14-.142.223-.23.25-.254.297.28.445.607.445.984 0 .36-.126.664-.377.916l-2.778 2.79a1.242 1.242 0 0 1-.917.364c-.36 0-.665-.118-.917-.35l-1.982-1.97a1.223 1.223 0 0 1-.378-.9l-.001-.004Zm9.477-9.504c0-.36.126-.665.377-.917l2.777-2.79a1.235 1.235 0 0 1 .913-.378c.35 0 .656.12.917.364l1.984 1.968c.254.252.38.553.38.903 0 .36-.126.665-.38.917l-2.802 2.804a1.238 1.238 0 0 1-.916.364c-.377 0-.7-.14-.97-.418.026-.027.11-.11.255-.25a7.5 7.5 0 0 0 .29-.29c.072-.08.139-.166.2-.255.08-.103.14-.22.176-.344.032-.12.048-.245.047-.37 0-.36-.126-.662-.377-.914a1.247 1.247 0 0 0-.917-.377c-.136 0-.26.015-.37.046-.114.03-.23.09-.346.175a3.868 3.868 0 0 0-.256.2c-.054.05-.15.148-.29.29-.14.146-.222.23-.25.258-.294-.278-.442-.606-.442-.983v-.003ZM5.003 21.177c0 1.078.382 1.99 1.146 2.736l1.982 1.968c.745.75 1.658 1.12 2.736 1.12 1.087 0 2.004-.38 2.75-1.143l2.777-2.79c.75-.747 1.12-1.66 1.12-2.737 0-1.106-.392-2.046-1.183-2.818l1.186-1.185c.774.79 1.708 1.186 2.805 1.186 1.078 0 1.995-.376 2.75-1.13l2.803-2.81c.751-.754 1.128-1.671 1.128-2.748 0-1.08-.382-1.993-1.146-2.738L23.875 6.12C23.13 5.372 22.218 5 21.139 5c-1.087 0-2.004.382-2.75 1.146l-2.777 2.79c-.75.747-1.12 1.66-1.12 2.737 0 1.105.392 2.045 1.183 2.817l-1.186 1.186c-.774-.79-1.708-1.186-2.805-1.186-1.078 0-1.995.377-2.75 1.132L6.13 18.426c-.754.755-1.13 1.672-1.13 2.75l.003.001Z"></path></svg></label><input id="a2a_copy_link_text" type="text" title="Copy link" readonly=""><div id="a2a_copy_link_copied">✓</div></div><div class="a2a_modal_body a2a_menu a2a_thanks a2a_hide" id="a2a_thanks"><div class="a2a_localize" data-a2a-localize="inner,ThanksForSharing">Thanks for sharing!</div></div></div><div class="a2a_menu a2a_full a2a_localize" id="a2apage_full" role="dialog" tabindex="-1" aria-label="Share" data-a2a-localize="title,Share"><div class="a2a_full_header"><div id="a2apage_find_container" class="a2a_menu_find_container"><label for="a2apage_find" id="a2apage_find_icon" class="a2a_svg a2a_s_find"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" aria-hidden="true"><title>Find any service</title><path fill="#CCC" d="M19.7 18.2l-4.5-4.5c.7-1.1 1.2-2.3 1.2-3.6 0-3.5-2.8-6.3-6.3-6.3s-6.3 2.8-6.3 6.3 2.8 6.3 6.3 6.3c1.4 0 2.6-.4 3.6-1.2l4.5 4.5c.6.6 1.3.7 1.7.2.5-.4.4-1.1-.2-1.7zm-9.6-3.6c-2.5 0-4.5-2.1-4.5-4.5 0-2.5 2.1-4.5 4.5-4.5 2.5 0 4.5 2.1 4.5 4.5s-2 4.5-4.5 4.5z"></path></svg></label><input id="a2apage_find" class="a2a_menu_find a2a_localize" type="text" autocomplete="off" title="Find any service" data-a2a-localize="title,FindAnyServiceToAddTo"></div></div><div class="a2a_full_services" id="a2apage_full_services" role="presentation"></div><div class="a2a_full_footer"><a href="https://www.addtoany.com" title="Share Buttons" rel="noopener" target="_blank"><span class="a2a_svg a2a_s__default a2a_s_a2a a2a_logo_color"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"><g fill="#FFF"><path d="M14 7h4v18h-4z"></path><path d="M7 14h18v4H7z"></path></g></svg></span>AddToAny</a></div></div><div id="a2apage_dropdown" class="a2a_menu a2a_mini a2a_localize a2a_hide" tabindex="-1" aria-label="Share" data-a2a-localize="label,Share"><div class="a2a_mini_services" id="a2apage_mini_services"></div><div id="a2apage_cols_container" class="a2a_cols_container"><div class="a2a_col1" id="a2apage_col1"></div><div id="a2apage_2_col1" class="a2a_hide"></div><div class="a2a_clear"></div></div><div class="a2apage_wide a2a_wide"><a href="#addtoany" id="a2apage_show_more_less" class="a2a_more a2a_localize" title="Show all" data-a2a-localize="title,ShowAll"><span class="a2a_svg a2a_s__default a2a_s_a2a a2a_logo_color"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"><g fill="#FFF"><path d="M14 7h4v18h-4z"></path><path d="M7 14h18v4H7z"></path></g></svg></span><span class="a2a_localize" data-a2a-localize="inner,More">More…</span></a></div></div><div style="height: 1px; width: 1px; position: absolute; z-index: 100000; top: 0px; visibility: hidden;"><iframe id="a2a_sm_ifr" title="AddToAny Utility Frame" aria-hidden="true" src="https://static.addtoany.com/menu/sm.25.html#type=core&amp;event=load" style="height: 1px; width: 1px; border: 0px; left: 0px; top: 0px; position: absolute; z-index: 100000; display: none;"></iframe></div></div><div role="status" aria-live="assertive" aria-relevant="additions" class="ui-helper-hidden-accessible"></div><div id="crossmark-widget" style="display: none;"><div class="crossmark-reset crossmark-overlay"></div><div class="crossmark-reset crossmark-popup"><div class="crossmark-reset crossmark-popup__offset"><div class="crossmark-reset crossmark-popup__inner"><div class="crossmark-reset crossmark-popup__header"><button class="crossmark-reset crossmark-popup__btn-close"><span class="sr-only">__("articleCrossmark.closePopup")</span></button></div><div class="crossmark-reset crossmark-popup__content-wrapper"><iframe class="crossmark-reset crossmark-popup__content" title="Crossmark service - Intentionally blank" aria-hidden="false" tabindex="0" src="https://crossmark.crossref.org/dialog?doi=10.1016%2Fj.cell.2021.06.018&amp;domain=www.cell.com&amp;uri_scheme=https%3A&amp;cm_version=v2.0"></iframe></div></div></div></div></div><div id="onetrust-consent-sdk" data-nosnippet="true"><div class="onetrust-pc-dark-filter ot-fade-in" style="z-index:2147483645;"></div><div id="onetrust-banner-sdk" class="otFlat bottom ot-wo-title ot-buttons-fw" tabindex="0" role="region" aria-label="Cookie banner"><div role="dialog" aria-modal="true" aria-label="Privacy"><div class="ot-sdk-container"><div class="ot-sdk-row"><div id="onetrust-group-container" class="ot-sdk-eight ot-sdk-columns"><div class="banner_logo"></div><div id="onetrust-policy"><div id="onetrust-policy-text">We use cookies that are necessary to make our site work. We may also use additional cookies to analyze, improve, and personalize our content and your digital experience. For more information, see our<a class="ot-cookie-policy-link" href="https://www.elsevier.com/legal/cookienotice" target="_blank" aria-label=", opens in a new tab" rel="noopener">Cookie Policy</a></div></div></div><div id="onetrust-button-group-parent" class="ot-sdk-three ot-sdk-columns"><div id="onetrust-button-group"><button id="onetrust-pc-btn-handler" class="cookie-setting-link">Cookie Settings</button>  <button id="onetrust-accept-btn-handler">Accept all cookies</button></div></div></div></div><!-- Close Button --><div id="onetrust-close-btn-container"></div><!-- Close Button END--></div></div><div id="onetrust-pc-sdk" class="otPcCenter ot-hide ot-fade-in" lang="en" aria-label="Preference center" role="region"><div role="dialog" aria-modal="true" style="height: 100%;" aria-label="Cookie Preference Center"><!-- Close Button --><div class="ot-pc-header"><!-- Logo Tag --><div class="ot-pc-logo" role="img" aria-label="Company Logo"><img alt="Company Logo" src="https://cdn.cookielaw.org/logos/static/ot_company_logo.png"></div></div><!-- Close Button --><div id="ot-pc-content" class="ot-pc-scrollbar"><div class="ot-optout-signal ot-hide"><div class="ot-optout-icon"><svg xmlns="http://www.w3.org/2000/svg"><path class="ot-floating-button__svg-fill" d="M14.588 0l.445.328c1.807 1.303 3.961 2.533 6.461 3.688 2.015.93 4.576 1.746 7.682 2.446 0 14.178-4.73 24.133-14.19 29.864l-.398.236C4.863 30.87 0 20.837 0 6.462c3.107-.7 5.668-1.516 7.682-2.446 2.709-1.251 5.01-2.59 6.906-4.016zm5.87 13.88a.75.75 0 00-.974.159l-5.475 6.625-3.005-2.997-.077-.067a.75.75 0 00-.983 1.13l4.172 4.16 6.525-7.895.06-.083a.75.75 0 00-.16-.973z" fill="#FFF" fill-rule="evenodd"></path></svg></div><span></span></div><h2 id="ot-pc-title">Cookie Preference Center</h2><div id="ot-pc-desc">We use cookies which are necessary to make our site work. We may also use additional cookies to analyse, improve and personalise our content and your digital experience. For more information, see our <a href="https://www.elsevier.com/legal/cookienotice/_nocache" target="_blank">Cookie Policy</a> and the list of <a href="https://support.google.com/admanager/answer/9012903" target="_blank">Google Ad-Tech Vendors</a>.
<br>
<br>
You may choose not to allow some types of cookies. However, blocking some types may impact your experience of our site and the services we are able to offer. See the different category headings below to find out more or change your settings.
<br>
</div><button id="accept-recommended-btn-handler">Allow all</button><section class="ot-sdk-row ot-cat-grp"><h3 id="ot-category-title"> Manage Consent Preferences</h3><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="1"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-1" aria-labelledby="ot-header-id-1 ot-status-id-1"></button><!-- Accordion header --><div class="ot-acc-hdr ot-always-active-group"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-1">Strictly Necessary Cookies</h4><div id="ot-status-id-1" class="ot-always-active">Always active</div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-1">These cookies are necessary for the website to function and cannot be switched off in our systems. They are usually only set in response to actions made by you which amount to a request for services, such as setting your privacy preferences, logging in or filling in forms. You can set your browser to block or alert you about these cookies, but some parts of the site will not then work. These cookies do not store any personally identifiable information.
<br><br></p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="1">Cookie Details List‎</button></div></div></div><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="3"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-3" aria-labelledby="ot-header-id-3"></button><!-- Accordion header --><div class="ot-acc-hdr"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-3">Functional Cookies</h4><div class="ot-tgl"><input type="checkbox" name="ot-group-id-3" id="ot-group-id-3" role="switch" class="category-switch-handler" data-optanongroupid="3" aria-labelledby="ot-header-id-3"> <label class="ot-switch" for="ot-group-id-3"><span class="ot-switch-nob"></span> <span class="ot-label-txt">Functional Cookies</span></label> </div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-3">These cookies enable the website to provide enhanced functionality and personalisation. They may be set by us or by third party providers whose services we have added to our pages. If you do not allow these cookies then some or all of these services may not function properly.</p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="3">Cookie Details List‎</button></div></div></div><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="2"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-2" aria-labelledby="ot-header-id-2"></button><!-- Accordion header --><div class="ot-acc-hdr"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-2">Performance Cookies</h4><div class="ot-tgl"><input type="checkbox" name="ot-group-id-2" id="ot-group-id-2" role="switch" class="category-switch-handler" data-optanongroupid="2" aria-labelledby="ot-header-id-2"> <label class="ot-switch" for="ot-group-id-2"><span class="ot-switch-nob"></span> <span class="ot-label-txt">Performance Cookies</span></label> </div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-2">These cookies allow us to count visits and traffic sources so we can measure and improve the performance of our site. They help us to know which pages are the most and least popular and see how visitors move around the site.</p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="2">Cookie Details List‎</button></div></div></div><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="4"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-4" aria-labelledby="ot-header-id-4"></button><!-- Accordion header --><div class="ot-acc-hdr"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-4">Targeting Cookies</h4><div class="ot-tgl"><input type="checkbox" name="ot-group-id-4" id="ot-group-id-4" role="switch" class="category-switch-handler" data-optanongroupid="4" aria-labelledby="ot-header-id-4"> <label class="ot-switch" for="ot-group-id-4"><span class="ot-switch-nob"></span> <span class="ot-label-txt">Targeting Cookies</span></label> </div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-4">These cookies may be set through our site by our advertising partners. They may be used by those companies to build a profile of your interests and show you relevant adverts on other sites. If you do not allow these cookies, you will experience less targeted advertising.</p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="4">Cookie Details List‎</button></div></div></div><!-- Groups sections starts --><!-- Group section ends --><!-- Accordion Group section starts --><!-- Accordion Group section ends --></section></div><section id="ot-pc-lst" class="ot-hide ot-hosts-ui ot-pc-scrollbar"><div id="ot-pc-hdr"><div id="ot-lst-title"><button class="ot-link-btn back-btn-handler" aria-label="Back"><svg id="ot-back-arw" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 444.531 444.531" xml:space="preserve"><title>Back Button</title><g><path fill="#656565" d="M213.13,222.409L351.88,83.653c7.05-7.043,10.567-15.657,10.567-25.841c0-10.183-3.518-18.793-10.567-25.835
                    l-21.409-21.416C323.432,3.521,314.817,0,304.637,0s-18.791,3.521-25.841,10.561L92.649,196.425
                    c-7.044,7.043-10.566,15.656-10.566,25.841s3.521,18.791,10.566,25.837l186.146,185.864c7.05,7.043,15.66,10.564,25.841,10.564
                    s18.795-3.521,25.834-10.564l21.409-21.412c7.05-7.039,10.567-15.604,10.567-25.697c0-10.085-3.518-18.746-10.567-25.978
                    L213.13,222.409z"></path></g></svg></button><h3>Cookie List</h3></div><div class="ot-lst-subhdr"><div class="ot-search-cntr"><p role="status" class="ot-scrn-rdr"></p><input id="vendor-search-handler" type="text" name="vendor-search-handler" placeholder="Search…" aria-label="Cookie list search"> <svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 -30 110 110" aria-hidden="true"><title>Search Icon</title><path fill="#2e3644" d="M55.146,51.887L41.588,37.786c3.486-4.144,5.396-9.358,5.396-14.786c0-12.682-10.318-23-23-23s-23,10.318-23,23
            s10.318,23,23,23c4.761,0,9.298-1.436,13.177-4.162l13.661,14.208c0.571,0.593,1.339,0.92,2.162,0.92
            c0.779,0,1.518-0.297,2.079-0.837C56.255,54.982,56.293,53.08,55.146,51.887z M23.984,6c9.374,0,17,7.626,17,17s-7.626,17-17,17
            s-17-7.626-17-17S14.61,6,23.984,6z"></path></svg></div><div class="ot-fltr-cntr"><button id="filter-btn-handler" aria-label="Filter" aria-haspopup="true"><svg role="presentation" aria-hidden="true" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 402.577 402.577" xml:space="preserve"><title>Filter Icon</title><g><path fill="#fff" d="M400.858,11.427c-3.241-7.421-8.85-11.132-16.854-11.136H18.564c-7.993,0-13.61,3.715-16.846,11.136
      c-3.234,7.801-1.903,14.467,3.999,19.985l140.757,140.753v138.755c0,4.955,1.809,9.232,5.424,12.854l73.085,73.083
      c3.429,3.614,7.71,5.428,12.851,5.428c2.282,0,4.66-0.479,7.135-1.43c7.426-3.238,11.14-8.851,11.14-16.845V172.166L396.861,31.413
      C402.765,25.895,404.093,19.231,400.858,11.427z"></path></g></svg></button></div><div id="ot-anchor"></div><section id="ot-fltr-modal"><div id="ot-fltr-cnt"><button id="clear-filters-handler">Clear</button><div class="ot-fltr-scrlcnt ot-pc-scrollbar"><div class="ot-fltr-opts" role="list"><div class="ot-fltr-opt" role="listitem"><div class="ot-chkbox"><input id="chkbox-id" type="checkbox" class="category-filter-handler"> <label for="chkbox-id"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div></div></div><div class="ot-fltr-btns"><button id="filter-apply-handler">Apply</button> <button id="filter-cancel-handler">Cancel</button></div></div></div></section></div></div><section id="ot-lst-cnt" class="ot-host-cnt ot-pc-scrollbar"><div id="ot-sel-blk"><div class="ot-sel-all"><div class="ot-sel-all-hdr"><span class="ot-consent-hdr">Consent</span> <span class="ot-li-hdr">Leg.Interest</span></div><div class="ot-sel-all-chkbox"><div class="ot-chkbox" id="ot-selall-hostcntr"><input id="select-all-hosts-groups-handler" type="checkbox"> <label for="select-all-hosts-groups-handler"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div><div class="ot-chkbox" id="ot-selall-vencntr"><input id="select-all-vendor-groups-handler" type="checkbox"> <label for="select-all-vendor-groups-handler"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div><div class="ot-chkbox" id="ot-selall-licntr"><input id="select-all-vendor-leg-handler" type="checkbox"> <label for="select-all-vendor-leg-handler"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div></div></div></div><div class="ot-sdk-row"><div class="ot-sdk-column"><ul id="ot-host-lst"></ul></div></div></section></section><div class="ot-pc-footer ot-pc-scrollbar"><div class="ot-btn-container"> <button class="save-preference-btn-handler onetrust-close-btn-handler">Confirm my choices</button></div><!-- Footer logo --><div class="ot-pc-footer-logo"><a href="https://www.onetrust.com/products/cookie-consent/" target="_blank" rel="noopener noreferrer" aria-label="Powered by OneTrust Opens in a new Tab"><img alt="Powered by Onetrust" src="https://cdn.cookielaw.org/logos/static/powered_by_logo.svg" title="Powered by OneTrust Opens in a new Tab"></a></div></div><!-- Cookie subgroup container --><!-- Vendor list link --><!-- Cookie lost link --><!-- Toggle HTML element --><!-- Checkbox HTML --><!-- plus minus--><!-- Arrow SVG element --><!-- Accordion basic element --><span class="ot-scrn-rdr" aria-atomic="true" aria-live="polite"></span><!-- Vendor Service container and item template --></div><iframe class="ot-text-resize" sandbox="allow-same-origin" title="onetrust-text-resize" style="position: absolute; top: -50000px; width: 100em;" aria-hidden="true"></iframe></div></div></body></html>