<html lang="en" class="pb-page fonts-loaded fonts-loaded" data-request-id="ecb4e572-856a-4f33-94d9-43755298633b" style="--article-sticky-top: 82.00019999999999px; --scroll-padding: 163.594px auto auto;"><head data-pb-dropzone="head"><meta name="pbContext" content=";product:product:elsevier\:product\:cell;website:website:cell-site;ctype:string:Journal Content;journal:journal:cell;pageGroup:string:Article View;article:article:pii\:S0092867421013325;page:string:Show Full Text;requestedJournal:journal:cell;issue:issue:pii\:S0092867420X00262;wgroup:string:Default Website Group;subPage:string:Full Text">




<!-- if there is any customization for Responsive Project widget  -->


























    
    







    
            
        



        
                <meta name="robots" content="noarchive,nocache">
            
    






    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=1, user-scalable=1">











        <meta property="og:title" content="Microenvironment drives cell state, plasticity, and drug response in pancreatic cancer">
    


        <meta property="og:type" content="Article">
    


        <meta property="og:url" content="https://www.cell.com/cell/abstract/S0092-8674(21)01332-5">
    



        <meta property="og:site_name" content="Cell">
    


        <meta property="og:image" content="https://www.cell.com/cms/asset/8197d954-74a1-42d8-bf84-c79b6d35b1c5/fx1.jpg">
    


        <meta property="og:description" content="Systematic profiling of metastatic pancreatic cancer biopsies and matched organoid
models provides a view of cellular states, their regulation by the tumor microenvironment,
and the ability to modulate these states to impact drug responses.">
    











        <meta name="twitter:title" content="Microenvironment drives cell state, plasticity, and drug response in pancreatic cancer">
    




        <meta name="twitter:image" content="https://www.cell.com/cms/asset/8197d954-74a1-42d8-bf84-c79b6d35b1c5/fx1.jpg">
    


        <meta name="twitter:description" content="Systematic profiling of metastatic pancreatic cancer biopsies and matched organoid
models provides a view of cellular states, their regulation by the tumor microenvironment,
and the ability to modulate these states to impact drug responses.">
    

        <meta name="twitter:card" content="summary_large_image">
    











    
    
            
            
        <link rel="canonical" href="https://www.cell.com/cell/fulltext/S0092-8674(21)01332-5">
    


<meta name="tdm-reservation" content="1">
<meta name="tdm-policy" content="https://www.elsevier.com/tdm/tdmrep-policy.json">

<title>Microenvironment drives cell state, plasticity, and drug response in pancreatic cancer: Cell</title><meta name="citation_journal_title" content="Cell"> <meta name="citation_journal_abbrev" content="Cell"> <meta name="citation_publisher" content="Elsevier"> <meta name="citation_title" content="Microenvironment drives cell state, plasticity, and drug response in pancreatic cancer"> <meta name="citation_date" content="2021/12/09"> <meta name="citation_volume" content="184"> <meta name="citation_issue" content="25"> <meta name="citation_firstpage" content="6119"> <meta name="citation_lastpage" content="6137.e26"> <meta name="citation_pmid" content="34890551"> <meta name="citation_abstract_html_url" content="https://www.cell.com/action/showAbstract?pii=S0092867421013325"> <meta name="citation_fulltext_html_url" content="https://www.cell.com/action/showFullText?pii=S0092867421013325"> <meta name="citation_pdf_url" content="https://www.cell.com/action/showPdf?pii=S0092867421013325"> <meta name="citation_issn" content="0092-8674"> <meta name="citation_issn" content="1097-4172"> <meta name="citation_language" content="English"> <meta name="citation_doi" content="10.1016/j.cell.2021.11.017"> <meta name="citation_pmid" content="34890551"> <meta name="citation_author" content="Srivatsan Raghavan"><meta name="citation_author_institution" content="Department of Medical Oncology, Dana-Farber Cancer Institute, Boston, MA 02215, USA"><meta name="citation_author_institution" content="Broad Institute of MIT and Harvard, Cambridge, MA 02142, USA"><meta name="citation_author_institution" content="Harvard Medical School, Boston, MA 02115, USA"><meta name="citation_author_institution" content="Department of Medicine, Brigham and Women’s Hospital, Boston, MA 02115, USA"><meta name="citation_author" content="Peter S. Winter"><meta name="citation_author_email" content="pswinter@mit.edu"><meta name="citation_author_institution" content="Department of Medical Oncology, Dana-Farber Cancer Institute, Boston, MA 02215, USA"><meta name="citation_author_institution" content="Broad Institute of MIT and Harvard, Cambridge, MA 02142, USA"><meta name="citation_author_institution" content="Institute for Medical Engineering &amp; Science, Massachusetts Institute of Technology, Cambridge, MA 02139, USA"><meta name="citation_author_institution" content="Koch Institute for Integrative Cancer Research, Massachusetts Institute of Technology, Cambridge, MA 02139, USA"><meta name="citation_author_institution" content="Department of Chemistry, Massachusetts Institute of Technology, Cambridge, MA 02139, USA"><meta name="citation_author" content="Andrew W. Navia"><meta name="citation_author_institution" content="Department of Medical Oncology, Dana-Farber Cancer Institute, Boston, MA 02215, USA"><meta name="citation_author_institution" content="Broad Institute of MIT and Harvard, Cambridge, MA 02142, USA"><meta name="citation_author_institution" content="Institute for Medical Engineering &amp; Science, Massachusetts Institute of Technology, Cambridge, MA 02139, USA"><meta name="citation_author_institution" content="Koch Institute for Integrative Cancer Research, Massachusetts Institute of Technology, Cambridge, MA 02139, USA"><meta name="citation_author_institution" content="Department of Chemistry, Massachusetts Institute of Technology, Cambridge, MA 02139, USA"><meta name="citation_author" content="Hannah L. Williams"><meta name="citation_author_orcid" content="0000-0003-0434-3900"><meta name="citation_author_institution" content="Department of Medical Oncology, Dana-Farber Cancer Institute, Boston, MA 02215, USA"><meta name="citation_author_institution" content="Harvard Medical School, Boston, MA 02115, USA"><meta name="citation_author" content="Alan DenAdel"><meta name="citation_author_institution" content="Center for Computational Molecular Biology, Brown University, Providence, RI 02912, USA"><meta name="citation_author_institution" content="Division of Applied Mathematics, Brown University, Providence, RI 02912, USA"><meta name="citation_author" content="Kristen E. Lowder"><meta name="citation_author_orcid" content="0000-0003-2615-4043"><meta name="citation_author_institution" content="Department of Medical Oncology, Dana-Farber Cancer Institute, Boston, MA 02215, USA"><meta name="citation_author_institution" content="Broad Institute of MIT and Harvard, Cambridge, MA 02142, USA"><meta name="citation_author" content="Jennyfer Galvez-Reyes"><meta name="citation_author_institution" content="Broad Institute of MIT and Harvard, Cambridge, MA 02142, USA"><meta name="citation_author_institution" content="Institute for Medical Engineering &amp; Science, Massachusetts Institute of Technology, Cambridge, MA 02139, USA"><meta name="citation_author_institution" content="Koch Institute for Integrative Cancer Research, Massachusetts Institute of Technology, Cambridge, MA 02139, USA"><meta name="citation_author" content="Radha L. Kalekar"><meta name="citation_author_institution" content="Department of Medical Oncology, Dana-Farber Cancer Institute, Boston, MA 02215, USA"><meta name="citation_author_institution" content="Broad Institute of MIT and Harvard, Cambridge, MA 02142, USA"><meta name="citation_author" content="Nolawit Mulugeta"><meta name="citation_author_institution" content="Broad Institute of MIT and Harvard, Cambridge, MA 02142, USA"><meta name="citation_author_institution" content="Institute for Medical Engineering &amp; Science, Massachusetts Institute of Technology, Cambridge, MA 02139, USA"><meta name="citation_author_institution" content="Koch Institute for Integrative Cancer Research, Massachusetts Institute of Technology, Cambridge, MA 02139, USA"><meta name="citation_author" content="Kevin S. Kapner"><meta name="citation_author_orcid" content="0000-0002-2306-2373"><meta name="citation_author_institution" content="Department of Medical Oncology, Dana-Farber Cancer Institute, Boston, MA 02215, USA"><meta name="citation_author_institution" content="Broad Institute of MIT and Harvard, Cambridge, MA 02142, USA"><meta name="citation_author" content="Manisha S. Raghavan"><meta name="citation_author_institution" content="Department of Medical Oncology, Dana-Farber Cancer Institute, Boston, MA 02215, USA"><meta name="citation_author_institution" content="Broad Institute of MIT and Harvard, Cambridge, MA 02142, USA"><meta name="citation_author" content="Ashir A. Borah"><meta name="citation_author_orcid" content="0000-0003-1193-4987"><meta name="citation_author_institution" content="Broad Institute of MIT and Harvard, Cambridge, MA 02142, USA"><meta name="citation_author" content="Nuo Liu"><meta name="citation_author_institution" content="Broad Institute of MIT and Harvard, Cambridge, MA 02142, USA"><meta name="citation_author_institution" content="Institute for Medical Engineering &amp; Science, Massachusetts Institute of Technology, Cambridge, MA 02139, USA"><meta name="citation_author_institution" content="Koch Institute for Integrative Cancer Research, Massachusetts Institute of Technology, Cambridge, MA 02139, USA"><meta name="citation_author" content="Sara A. Väyrynen"><meta name="citation_author_institution" content="Department of Medical Oncology, Dana-Farber Cancer Institute, Boston, MA 02215, USA"><meta name="citation_author_institution" content="Harvard Medical School, Boston, MA 02115, USA"><meta name="citation_author" content="Andressa Dias Costa"><meta name="citation_author_institution" content="Department of Medical Oncology, Dana-Farber Cancer Institute, Boston, MA 02215, USA"><meta name="citation_author_institution" content="Harvard Medical School, Boston, MA 02115, USA"><meta name="citation_author" content="Raymond W.S. Ng"><meta name="citation_author_orcid" content="0000-0003-2573-8207"><meta name="citation_author_institution" content="Department of Medical Oncology, Dana-Farber Cancer Institute, Boston, MA 02215, USA"><meta name="citation_author_institution" content="Broad Institute of MIT and Harvard, Cambridge, MA 02142, USA"><meta name="citation_author" content="Junning Wang"><meta name="citation_author_institution" content="Department of Medical Oncology, Dana-Farber Cancer Institute, Boston, MA 02215, USA"><meta name="citation_author" content="Emma K. Hill"><meta name="citation_author_institution" content="Department of Medical Oncology, Dana-Farber Cancer Institute, Boston, MA 02215, USA"><meta name="citation_author" content="Dorisanne Y. Ragon"><meta name="citation_author_institution" content="Department of Medical Oncology, Dana-Farber Cancer Institute, Boston, MA 02215, USA"><meta name="citation_author" content="Lauren K. Brais"><meta name="citation_author_institution" content="Department of Medical Oncology, Dana-Farber Cancer Institute, Boston, MA 02215, USA"><meta name="citation_author" content="Alex M. Jaeger"><meta name="citation_author_institution" content="Koch Institute for Integrative Cancer Research, Massachusetts Institute of Technology, Cambridge, MA 02139, USA"><meta name="citation_author" content="Liam F. Spurr"><meta name="citation_author_institution" content="Department of Medical Oncology, Dana-Farber Cancer Institute, Boston, MA 02215, USA"><meta name="citation_author_institution" content="Broad Institute of MIT and Harvard, Cambridge, MA 02142, USA"><meta name="citation_author" content="Yvonne Y. Li"><meta name="citation_author_institution" content="Department of Medical Oncology, Dana-Farber Cancer Institute, Boston, MA 02215, USA"><meta name="citation_author_institution" content="Broad Institute of MIT and Harvard, Cambridge, MA 02142, USA"><meta name="citation_author" content="Andrew D. Cherniack"><meta name="citation_author_institution" content="Department of Medical Oncology, Dana-Farber Cancer Institute, Boston, MA 02215, USA"><meta name="citation_author_institution" content="Broad Institute of MIT and Harvard, Cambridge, MA 02142, USA"><meta name="citation_author_institution" content="Harvard Medical School, Boston, MA 02115, USA"><meta name="citation_author" content="Matthew A. Booker"><meta name="citation_author_orcid" content="0000-0002-6902-1032"><meta name="citation_author_institution" content="Department of Informatics and Analytics, Dana-Farber Cancer Institute, Boston, MA 02215, USA"><meta name="citation_author" content="Elizabeth F. Cohen"><meta name="citation_author_institution" content="Department of Informatics and Analytics, Dana-Farber Cancer Institute, Boston, MA 02215, USA"><meta name="citation_author" content="Michael Y. Tolstorukov"><meta name="citation_author_institution" content="Department of Informatics and Analytics, Dana-Farber Cancer Institute, Boston, MA 02215, USA"><meta name="citation_author" content="Isaac Wakiro"><meta name="citation_author_orcid" content="0000-0002-5248-4823"><meta name="citation_author_institution" content="Department of Medical Oncology, Dana-Farber Cancer Institute, Boston, MA 02215, USA"><meta name="citation_author" content="Asaf Rotem"><meta name="citation_author_orcid" content="0000-0002-6859-7435"><meta name="citation_author_institution" content="Department of Medical Oncology, Dana-Farber Cancer Institute, Boston, MA 02215, USA"><meta name="citation_author_institution" content="Broad Institute of MIT and Harvard, Cambridge, MA 02142, USA"><meta name="citation_author_institution" content="Center for Cancer Genomics, Dana-Farber Cancer Institute, Boston, MA 02215, USA"><meta name="citation_author" content="Bruce E. Johnson"><meta name="citation_author_institution" content="Department of Medical Oncology, Dana-Farber Cancer Institute, Boston, MA 02215, USA"><meta name="citation_author_institution" content="Harvard Medical School, Boston, MA 02115, USA"><meta name="citation_author_institution" content="Department of Medicine, Brigham and Women’s Hospital, Boston, MA 02115, USA"><meta name="citation_author_institution" content="Center for Cancer Genomics, Dana-Farber Cancer Institute, Boston, MA 02215, USA"><meta name="citation_author" content="James M. McFarland"><meta name="citation_author_institution" content="Broad Institute of MIT and Harvard, Cambridge, MA 02142, USA"><meta name="citation_author" content="Ewa T. Sicinska"><meta name="citation_author_institution" content="Harvard Medical School, Boston, MA 02115, USA"><meta name="citation_author_institution" content="Department of Oncologic Pathology, Dana-Farber Cancer Institute, Boston, MA 02215, USA"><meta name="citation_author" content="Tyler E. Jacks"><meta name="citation_author_institution" content="Koch Institute for Integrative Cancer Research, Massachusetts Institute of Technology, Cambridge, MA 02139, USA"><meta name="citation_author" content="Ryan J. Sullivan"><meta name="citation_author_institution" content="Harvard Medical School, Boston, MA 02115, USA"><meta name="citation_author_institution" content="Massachusetts General Hospital Cancer Center, Boston, MA 02114, USA"><meta name="citation_author" content="Geoffrey I. Shapiro"><meta name="citation_author_orcid" content="0000-0002-3331-4095"><meta name="citation_author_institution" content="Department of Medical Oncology, Dana-Farber Cancer Institute, Boston, MA 02215, USA"><meta name="citation_author_institution" content="Harvard Medical School, Boston, MA 02115, USA"><meta name="citation_author_institution" content="Department of Medicine, Brigham and Women’s Hospital, Boston, MA 02115, USA"><meta name="citation_author" content="Thomas E. Clancy"><meta name="citation_author_institution" content="Department of Medical Oncology, Dana-Farber Cancer Institute, Boston, MA 02215, USA"><meta name="citation_author_institution" content="Harvard Medical School, Boston, MA 02115, USA"><meta name="citation_author_institution" content="Department of Surgery, Brigham and Women’s Hospital, Boston, MA 02115, USA"><meta name="citation_author" content="Kimberly Perez"><meta name="citation_author_orcid" content="0000-0002-5745-2994"><meta name="citation_author_institution" content="Department of Medical Oncology, Dana-Farber Cancer Institute, Boston, MA 02215, USA"><meta name="citation_author_institution" content="Harvard Medical School, Boston, MA 02115, USA"><meta name="citation_author_institution" content="Department of Medicine, Brigham and Women’s Hospital, Boston, MA 02115, USA"><meta name="citation_author" content="Douglas A. Rubinson"><meta name="citation_author_orcid" content="0000-0002-5337-5231"><meta name="citation_author_institution" content="Department of Medical Oncology, Dana-Farber Cancer Institute, Boston, MA 02215, USA"><meta name="citation_author_institution" content="Harvard Medical School, Boston, MA 02115, USA"><meta name="citation_author_institution" content="Department of Medicine, Brigham and Women’s Hospital, Boston, MA 02115, USA"><meta name="citation_author" content="Kimmie Ng"><meta name="citation_author_orcid" content="0000-0003-0631-1494"><meta name="citation_author_institution" content="Department of Medical Oncology, Dana-Farber Cancer Institute, Boston, MA 02215, USA"><meta name="citation_author_institution" content="Harvard Medical School, Boston, MA 02115, USA"><meta name="citation_author_institution" content="Department of Medicine, Brigham and Women’s Hospital, Boston, MA 02115, USA"><meta name="citation_author" content="James M. Cleary"><meta name="citation_author_institution" content="Department of Medical Oncology, Dana-Farber Cancer Institute, Boston, MA 02215, USA"><meta name="citation_author_institution" content="Harvard Medical School, Boston, MA 02115, USA"><meta name="citation_author_institution" content="Department of Medicine, Brigham and Women’s Hospital, Boston, MA 02115, USA"><meta name="citation_author" content="Lorin Crawford"><meta name="citation_author_orcid" content="0000-0003-0178-8242"><meta name="citation_author_institution" content="Center for Computational Molecular Biology, Brown University, Providence, RI 02912, USA"><meta name="citation_author_institution" content="Department of Biostatistics, Brown University, Providence, RI 02912, USA"><meta name="citation_author_institution" content="Microsoft Research New England, Cambridge, MA 02142, USA"><meta name="citation_author" content="Scott R. Manalis"><meta name="citation_author_orcid" content="0000-0001-5223-9433"><meta name="citation_author_institution" content="Koch Institute for Integrative Cancer Research, Massachusetts Institute of Technology, Cambridge, MA 02139, USA"><meta name="citation_author_institution" content="Department of Biological Engineering, Massachusetts Institute of Technology, Cambridge, MA 02142, USA"><meta name="citation_author" content="Jonathan A. Nowak"><meta name="citation_author_institution" content="Harvard Medical School, Boston, MA 02115, USA"><meta name="citation_author_institution" content="Center for Cancer Genomics, Dana-Farber Cancer Institute, Boston, MA 02215, USA"><meta name="citation_author_institution" content="Department of Pathology, Brigham and Women’s Hospital, Boston, MA 02115, USA"><meta name="citation_author" content="Brian M. Wolpin"><meta name="citation_author_institution" content="Department of Medical Oncology, Dana-Farber Cancer Institute, Boston, MA 02215, USA"><meta name="citation_author_institution" content="Harvard Medical School, Boston, MA 02115, USA"><meta name="citation_author_institution" content="Department of Medicine, Brigham and Women’s Hospital, Boston, MA 02115, USA"><meta name="citation_author" content="William C. Hahn"><meta name="citation_author_orcid" content="0000-0003-2840-9791"><meta name="citation_author_institution" content="Department of Medical Oncology, Dana-Farber Cancer Institute, Boston, MA 02215, USA"><meta name="citation_author_institution" content="Broad Institute of MIT and Harvard, Cambridge, MA 02142, USA"><meta name="citation_author_institution" content="Harvard Medical School, Boston, MA 02115, USA"><meta name="citation_author_institution" content="Department of Medicine, Brigham and Women’s Hospital, Boston, MA 02115, USA"><meta name="citation_author" content="Andrew J. Aguirre"><meta name="citation_author_email" content="andrew_aguirre@dfci.harvard.edu"><meta name="citation_author_institution" content="Department of Medical Oncology, Dana-Farber Cancer Institute, Boston, MA 02215, USA"><meta name="citation_author_institution" content="Broad Institute of MIT and Harvard, Cambridge, MA 02142, USA"><meta name="citation_author_institution" content="Harvard Medical School, Boston, MA 02115, USA"><meta name="citation_author_institution" content="Department of Medicine, Brigham and Women’s Hospital, Boston, MA 02115, USA"><meta name="citation_author" content="Alex K. Shalek"><meta name="citation_author_email" content="shalek@mit.edu"><meta name="citation_author_orcid" content="0000-0001-5670-8778"><meta name="citation_author_institution" content="Broad Institute of MIT and Harvard, Cambridge, MA 02142, USA"><meta name="citation_author_institution" content="Harvard Medical School, Boston, MA 02115, USA"><meta name="citation_author_institution" content="Institute for Medical Engineering &amp; Science, Massachusetts Institute of Technology, Cambridge, MA 02139, USA"><meta name="citation_author_institution" content="Koch Institute for Integrative Cancer Research, Massachusetts Institute of Technology, Cambridge, MA 02139, USA"><meta name="citation_author_institution" content="Department of Chemistry, Massachusetts Institute of Technology, Cambridge, MA 02139, USA"><meta name="citation_author_institution" content="Ragon Institute of MGH, MIT, and Harvard, Cambridge, MA 02139, USA"><meta name="citation_keywords" content="pancreatic cancer; single-cell RNA-sequencing; transcriptional states; plasticity; patient-derived organoid models; tumor microenvironment; liver metastases; tumor heterogeneity"><meta name="citation_reference" content="citation_title=BreaKmer: detection of structural variation in targeted massively parallel sequencing data using kmers; citation_author=R.P. Abo; citation_author=M. Ducar; citation_author=E.P. Garcia; citation_author=A.R. Thorner; citation_author=V. Rojas-Rudilla; citation_author=L. Lin; citation_author=L.M. Sholl; citation_author=W.C. Hahn; citation_author=M. Meyerson; citation_author=N.I. Lindeman; citation_author=et al.; citation_journal_title=Nucleic Acids Res.; citation_volume=43; citation_firstpage=e19; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Real-time Genomic Characterization of Advanced Pancreatic Cancer to Enable Precision Medicine; citation_author=A.J. Aguirre; citation_author=J.A. Nowak; citation_author=N.D. Camarda; citation_author=R.A. Moffitt; citation_author=A.A. Ghazani; citation_author=M. Hazar-Rethinam; citation_author=S. Raghavan; citation_author=J. Kim; citation_author=L.K. Brais; citation_author=D. Ragon; citation_author=et al.; citation_journal_title=Cancer Discov.; citation_volume=8; citation_firstpage=1096; citation_lastpage=1111; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=A gene-environment-induced epigenetic program initiates tumorigenesis; citation_author=D. Alonso-Curbelo; citation_author=Y.J. Ho; citation_author=C. Burdziak; citation_author=J.L.V. Maag; citation_author=J.P. Morris; citation_author=R. Chandwani; citation_author=H.A. Chen; citation_author=K.M. Tsanov; citation_author=F.M. Barriga; citation_author=W. Luan; citation_author=et al.; citation_journal_title=Nature; citation_volume=590; citation_firstpage=642; citation_lastpage=648; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=Genomics-Driven Precision Medicine for Advanced Pancreatic Cancer: Early Results from the COMPASS Trial; citation_author=K.L. Aung; citation_author=S.E. Fischer; citation_author=R.E. Denroche; citation_author=G.H. Jang; citation_author=A. Dodd; citation_author=S. Creighton; citation_author=B. Southwood; citation_author=S.B. Liang; citation_author=D. Chadwick; citation_author=A. Zhang; citation_author=et al.; citation_journal_title=Clin. Cancer Res.; citation_volume=24; citation_firstpage=1344; citation_lastpage=1354; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Genomic analyses identify molecular subtypes of pancreatic cancer; citation_author=P. Bailey; citation_author=D.K. Chang; citation_author=K. Nones; citation_author=A.L. Johns; citation_author=A.M. Patch; citation_author=M.C. Gingras; citation_author=D.K. Miller; citation_author=A.N. Christ; citation_author=T.J. Bruxner; citation_author=M.C. Quinn; citation_author=Australian Pancreatic Cancer Genome Initiative; citation_author=et al.; citation_journal_title=Nature; citation_volume=531; citation_firstpage=47; citation_lastpage=52; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Broadening the Impact of Immunotherapy to Pancreatic Cancer: Challenges and Opportunities; citation_author=V.P. Balachandran; citation_author=G.L. Beatty; citation_author=S.K. Dougan; citation_journal_title=Gastroenterology; citation_volume=156; citation_firstpage=2056; citation_lastpage=2072; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Spatially and functionally distinct subclasses of breast cancer-associated fibroblasts revealed by single cell RNA sequencing; citation_author=M. Bartoschek; citation_author=N. Oskolkov; citation_author=M. Bocci; citation_author=J. Lövrot; citation_author=C. Larsson; citation_author=M. Sommarin; citation_author=C.D. Madsen; citation_author=D. Lindgren; citation_author=G. Pekar; citation_author=G. Karlsson; citation_author=et al.; citation_journal_title=Nat. Commun.; citation_volume=9; citation_firstpage=5150; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Genetic and transcriptional evolution alters cancer cell line drug response; citation_author=U. Ben-David; citation_author=B. Siranosian; citation_author=G. Ha; citation_author=H. Tang; citation_author=Y. Oren; citation_author=K. Hinohara; citation_author=C.A. Strathdee; citation_author=J. Dempster; citation_author=N.J. Lyons; citation_author=R. Burns; citation_author=et al.; citation_journal_title=Nature; citation_volume=560; citation_firstpage=325; citation_lastpage=330; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Genomic evolution of cancer models: perils and opportunities; citation_author=U. Ben-David; citation_author=R. Beroukhim; citation_author=T.R. Golub; citation_journal_title=Nat. Rev. Cancer; citation_volume=19; citation_firstpage=97; citation_lastpage=109; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Single-Cell Transcriptomics of Pancreatic Cancer Precursors Demonstrates Epithelial and Microenvironmental Heterogeneity as an Early Event in Neoplastic Progression; citation_author=V. Bernard; citation_author=A. Semaan; citation_author=J. Huang; citation_author=F.A. San Lucas; citation_author=F.C. Mulu; citation_author=B.M. Stephens; citation_author=P.A. Guerrero; citation_author=Y. Huang; citation_author=J. Zhao; citation_author=N. Kamyabi; citation_author=et al.; citation_journal_title=Clin. Cancer Res.; citation_volume=25; citation_firstpage=2194; citation_lastpage=2205; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Clinical Identification of Oncogenic Drivers and Copy-Number Alterations in Pituitary Tumors; citation_author=W.L. Bi; citation_author=N.F. Greenwald; citation_author=S.H. Ramkissoon; citation_author=M. Abedalthagafi; citation_author=S.M. Coy; citation_author=K.L. Ligon; citation_author=Y. Mei; citation_author=L. MacConaill; citation_author=M. Ducar; citation_author=L. Min; citation_author=et al.; citation_journal_title=Endocrinology; citation_volume=158; citation_firstpage=2284; citation_lastpage=2291; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Organoid models of human and mouse ductal pancreatic cancer; citation_author=S.F. Boj; citation_author=C.I. Hwang; citation_author=L.A. Baker; citation_author=I.I. Chio; citation_author=D.D. Engle; citation_author=V. Corbo; citation_author=M. Jager; citation_author=M. Ponz-Sarvise; citation_author=H. Tiriac; citation_author=M.S. Spector; citation_author=et al.; citation_journal_title=Cell; citation_volume=160; citation_firstpage=324; citation_lastpage=338; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=HNF4A and GATA6 Loss Reveals Therapeutically Actionable Subtypes in Pancreatic Cancer; citation_author=H. Brunton; citation_author=G. Caligiuri; citation_author=R. Cunningham; citation_author=R. Upstill-Goddard; citation_author=U.M. Bailey; citation_author=I.M. Garner; citation_author=C. Nourse; citation_author=S. Dreyer; citation_author=M. Jones; citation_author=K. Moran-Jones; citation_author=Glasgow Precision Oncology Laboratory; citation_author=Australian Pancreatic Cancer Genome Initiative; citation_author=et al.; citation_journal_title=Cell Rep.; citation_volume=31; citation_firstpage=107625; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Interrogating open issues in cancer medicine with patient-derived xenografts; citation_author=A.T. Byrne; citation_author=D.G. Alférez; citation_author=F. Amant; citation_author=D. Annibali; citation_author=J. Arribas; citation_author=A.V. Biankin; citation_author=A. Bruna; citation_author=E. Budinská; citation_author=C. Caldas; citation_author=D.K. Chang; citation_author=et al.; citation_journal_title=Nat. Rev. Cancer; citation_volume=17; citation_firstpage=632; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Integrated Genomic Characterization of Pancreatic Ductal Adenocarcinoma; citation_author=Cancer Genome Atlas Research Network; citation_journal_title=Cancer Cell; citation_volume=32; citation_firstpage=185; citation_lastpage=203.e13; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Transcription phenotypes of pancreatic cancer are driven by genomic events during tumor evolution; citation_author=M. Chan-Seng-Yue; citation_author=J.C. Kim; citation_author=G.W. Wilson; citation_author=K. Ng; citation_author=E.F. Figueroa; citation_author=G.M. O’Kane; citation_author=A.A. Connor; citation_author=R.E. Denroche; citation_author=R.C. Grant; citation_author=J. McLeod; citation_author=et al.; citation_journal_title=Nat. Genet.; citation_volume=52; citation_firstpage=231; citation_lastpage=240; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Subtypes of pancreatic ductal adenocarcinoma and their differing responses to therapy; citation_author=E.A. Collisson; citation_author=A. Sadanandam; citation_author=P. Olson; citation_author=W.J. Gibb; citation_author=M. Truitt; citation_author=S. Gu; citation_author=J. Cooc; citation_author=J. Weinkle; citation_author=G.E. Kim; citation_author=L. Jakkula; citation_author=et al.; citation_journal_title=Nat. Med.; citation_volume=17; citation_firstpage=500; citation_lastpage=503; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=Molecular subtypes of pancreatic cancer; citation_author=E.A. Collisson; citation_author=P. Bailey; citation_author=D.K. Chang; citation_author=A.V. Biankin; citation_journal_title=Nat. Rev. Gastroenterol. Hepatol.; citation_volume=16; citation_firstpage=207; citation_lastpage=220; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Integration of Genomic and Transcriptional Features in Pancreatic Cancer Reveals Increased Cell Cycle Progression in Metastases; citation_author=A.A. Connor; citation_author=R.E. Denroche; citation_author=G.H. Jang; citation_author=M. Lemire; citation_author=A. Zhang; citation_author=M. Chan-Seng-Yue; citation_author=G. Wilson; citation_author=R.C. Grant; citation_author=D. Merico; citation_author=I. Lungu; citation_author=et al.; citation_journal_title=Cancer Cell; citation_volume=35; citation_firstpage=267; citation_lastpage=282 e267; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=The perivascular origin of pathological fibroblasts; citation_author=S.E. Di Carlo; citation_author=L. Peduto; citation_journal_title=J.&nbsp;Clin. Invest.; citation_volume=128; citation_firstpage=54; citation_lastpage=63; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Organoids in cancer research; citation_author=J. Drost; citation_author=H. Clevers; citation_journal_title=Nat. Rev. Cancer; citation_volume=18; citation_firstpage=407; citation_lastpage=418; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Cross-Species Single-Cell Analysis of Pancreatic Ductal Adenocarcinoma Reveals Antigen-Presenting Cancer-Associated Fibroblasts; citation_author=E. Elyada; citation_author=M. Bolisetty; citation_author=P. Laise; citation_author=W.F. Flynn; citation_author=E.T. Courtois; citation_author=R.A. Burkhart; citation_author=J.A. Teinor; citation_author=P. Belleau; citation_author=G. Biffi; citation_author=M.S. Lucito; citation_author=et al.; citation_journal_title=Cancer Discov.; citation_volume=9; citation_firstpage=1102; citation_lastpage=1123; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Linking transcriptional and genetic tumor heterogeneity through allele analysis of single-cell RNA-seq data; citation_author=J. Fan; citation_author=H.O. Lee; citation_author=S. Lee; citation_author=D.E. Ryu; citation_author=S. Lee; citation_author=C. Xue; citation_author=S.J. Kim; citation_author=K. Kim; citation_author=N. Barkas; citation_author=P.J. Park; citation_author=et al.; citation_journal_title=Genome Res.; citation_volume=28; citation_firstpage=1217; citation_lastpage=1227; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Developmental and oncogenic programs in H3K27M gliomas dissected by single-cell RNA-seq; citation_author=M.G. Filbin; citation_author=I. Tirosh; citation_author=V. Hovestadt; citation_author=M.L. Shaw; citation_author=L.E. Escalante; citation_author=N.D. Mathewson; citation_author=C. Neftel; citation_author=N. Frank; citation_author=K. Pelton; citation_author=C.M. Hebert; citation_author=et al.; citation_journal_title=Science; citation_volume=360; citation_firstpage=331; citation_lastpage=335; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Human Intestinal Organoids Maintain Self-Renewal Capacity and Cellular Diversity in Niche-Inspired Culture Condition; citation_author=M. Fujii; citation_author=M. Matano; citation_author=K. Toshimitsu; citation_author=A. Takano; citation_author=Y. Mikami; citation_author=S. Nishikori; citation_author=S. Sugimoto; citation_author=T. Sato; citation_journal_title=Cell Stem Cell; citation_volume=23; citation_firstpage=787; citation_lastpage=793 e786; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Validation of OncoPanel: A Targeted Next-Generation Sequencing Assay for the Detection of Somatic Variants in Cancer; citation_author=E.P. Garcia; citation_author=A. Minkovsky; citation_author=Y. Jia; citation_author=M.D. Ducar; citation_author=P. Shivdasani; citation_author=X. Gong; citation_author=A.H. Ligon; citation_author=L.M. Sholl; citation_author=F.C. Kuo; citation_author=L.E. MacConaill; citation_author=et al.; citation_journal_title=Arch. Pathol. Lab. Med.; citation_volume=141; citation_firstpage=751; citation_lastpage=758; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Lessons from the cancer genome; citation_author=L.A. Garraway; citation_author=E.S. Lander; citation_journal_title=Cell; citation_volume=153; citation_firstpage=17; citation_lastpage=37; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=Next-generation characterization of the Cancer Cell Line Encyclopedia; citation_author=M. Ghandi; citation_author=F.W. Huang; citation_author=J. Jané-Valbuena; citation_author=G.V. Kryukov; citation_author=C.C. Lo; citation_author=E.R. McDonald; citation_author=J. Barretina; citation_author=E.T. Gelfand; citation_author=C.M. Bielski; citation_author=H. Li; citation_author=et al.; citation_journal_title=Nature; citation_volume=569; citation_firstpage=503; citation_lastpage=508; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Seq-Well: portable, low-cost RNA sequencing of single cells at high throughput; citation_author=T.M. Gierahn; citation_author=M.H. Wadsworth; citation_author=T.K. Hughes; citation_author=B.D. Bryson; citation_author=A. Butler; citation_author=R. Satija; citation_author=S. Fortune; citation_author=J.C. Love; citation_author=A.K. Shalek; citation_journal_title=Nat. Methods; citation_volume=14; citation_firstpage=395; citation_lastpage=398; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=The clinical relevance of cancer cell lines; citation_author=J.P. Gillet; citation_author=S. Varma; citation_author=M.M. Gottesman; citation_journal_title=J.&nbsp;Natl. Cancer Inst.; citation_volume=105; citation_firstpage=452; citation_lastpage=458; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=Meta-analysis of gene expression signatures defining the epithelial to mesenchymal transition during cancer progression; citation_author=C.J. Gröger; citation_author=M. Grubinger; citation_author=T. Waldhör; citation_author=K. Vierlinger; citation_author=W. Mikulits; citation_journal_title=PLoS ONE; citation_volume=7; citation_firstpage=e51136; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=Spatially confined sub-tumor microenvironments in pancreatic cancer; citation_author=B.T. Grünwald; citation_author=A. Devisme; citation_author=G. Andrieux; citation_author=F. Vyas; citation_author=K. Aliar; citation_author=C.W. McCloskey; citation_author=A. Macklin; citation_author=G.H. Jang; citation_author=R. Denroche; citation_author=J.M. Romero; citation_author=et al.; citation_journal_title=Cell; citation_volume=184; citation_firstpage=5577; citation_lastpage=5592.e18; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=Growth rate inhibition metrics correct for confounders in measuring sensitivity to cancer drugs; citation_author=M. Hafner; citation_author=M. Niepel; citation_author=M. Chung; citation_author=P.K. Sorger; citation_journal_title=Nat. Methods; citation_volume=13; citation_firstpage=521; citation_lastpage=527; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=An expanded universe of cancer targets; citation_author=W.C. Hahn; citation_author=J.S. Bader; citation_author=T.P. Braun; citation_author=A. Califano; citation_author=P.A. Clemons; citation_author=B.J. Druker; citation_author=A.J. Ewald; citation_author=H. Fu; citation_author=S. Jagu; citation_author=C.J. Kemp; citation_author=Cancer Target Discovery and Development Network; citation_author=et al.; citation_journal_title=Cell; citation_volume=184; citation_firstpage=1142; citation_lastpage=1155; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=Interactions between cancer cells and immune cells drive transitions to mesenchymal-like states in glioblastoma; citation_author=T. Hara; citation_author=R. Chanoch-Myers; citation_author=N.D. Mathewson; citation_author=C. Myskiw; citation_author=L. Atta; citation_author=L. Bussema; citation_author=S.W. Eichhorn; citation_author=A.C. Greenwald; citation_author=G.S. Kinker; citation_author=C. Rodman; citation_author=et al.; citation_journal_title=Cancer Cell; citation_volume=39; citation_firstpage=779; citation_lastpage=792 e711; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=A unifying paradigm for transcriptional heterogeneity and squamous features in pancreatic ductal adenocarcinoma; citation_author=A. Hayashi; citation_author=J. Fan; citation_author=R. Chen; citation_author=Y.-j. Ho; citation_author=A.P. Makohon-Moore; citation_author=N. Lecomte; citation_author=Y. Zhong; citation_author=J. Hong; citation_author=J. Huang; citation_author=H. Sakamoto; citation_author=et al.; citation_journal_title=Nature Cancer; citation_volume=1; citation_firstpage=59; citation_lastpage=74; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=The tumour microenvironment in pancreatic cancer - clinical challenges and opportunities; citation_author=W.J. Ho; citation_author=E.M. Jaffee; citation_author=L. Zheng; citation_journal_title=Nat. Rev. Clin. Oncol.; citation_volume=17; citation_firstpage=527; citation_lastpage=540; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Pericyte-fibroblast transition promotes tumor growth and metastasis; citation_author=K. Hosaka; citation_author=Y. Yang; citation_author=T. Seki; citation_author=C. Fischer; citation_author=O. Dubey; citation_author=E. Fredlund; citation_author=J. Hartman; citation_author=P. Religa; citation_author=H. Morikawa; citation_author=Y. Ishii; citation_author=et al.; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=113; citation_firstpage=E5618; citation_lastpage=E5627; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Resolving medulloblastoma cellular architecture by single-cell genomics; citation_author=V. Hovestadt; citation_author=K.S. Smith; citation_author=L. Bihannic; citation_author=M.G. Filbin; citation_author=M.L. Shaw; citation_author=A. Baumgartner; citation_author=J.C. DeWitt; citation_author=A. Groves; citation_author=L. Mayr; citation_author=H.R. Weisman; citation_author=et al.; citation_journal_title=Nature; citation_volume=572; citation_firstpage=74; citation_lastpage=79; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Second-Strand Synthesis-Based Massively Parallel scRNA-Seq Reveals Cellular States and Molecular Features of Human Inflammatory Skin Pathologies; citation_author=T.K. Hughes; citation_author=M.H. Wadsworth; citation_author=T.M. Gierahn; citation_author=T. Do; citation_author=D. Weiss; citation_author=P.R. Andrade; citation_author=F. Ma; citation_author=B.J. de Andrade Silva; citation_author=S. Shao; citation_author=L.C. Tsoi; citation_author=et al.; citation_journal_title=Immunity; citation_volume=53; citation_firstpage=878; citation_lastpage=894 e877; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Implementing Genome-Driven Oncology; citation_author=D.M. Hyman; citation_author=B.S. Taylor; citation_author=J. Baselga; citation_journal_title=Cell; citation_volume=168; citation_firstpage=584; citation_lastpage=599; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Roles of Wnt Target Genes in the Journey of Cancer Stem Cells; citation_author=J.H. Kim; citation_author=S.Y. Park; citation_author=Y. Jun; citation_author=J.Y. Kim; citation_author=J.S. Nam; citation_journal_title=Int. J. Mol. Sci.; citation_volume=18; citation_firstpage=1604; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Chemoresistance Evolution in Triple-Negative Breast Cancer Delineated by Single-Cell Sequencing; citation_author=C. Kim; citation_author=R. Gao; citation_author=E. Sei; citation_author=R. Brandt; citation_author=J. Hartman; citation_author=T. Hatschek; citation_author=N. Crosetto; citation_author=T. Foukakis; citation_author=N.E. Navin; citation_journal_title=Cell; citation_volume=173; citation_firstpage=879; citation_lastpage=893 e813; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Cumulus provides cloud-based data analysis for large-scale single-cell and single-nucleus RNA-seq; citation_author=B. Li; citation_author=J. Gould; citation_author=Y. Yang; citation_author=S. Sarkizova; citation_author=M. Tabaka; citation_author=O. Ashenberg; citation_author=Y. Rosen; citation_author=M. Slyper; citation_author=M.S. Kowalczyk; citation_author=A.C. Villani; citation_author=et al.; citation_journal_title=Nat. Methods; citation_volume=17; citation_firstpage=793; citation_lastpage=798; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Mutant Kras co-opts a proto-oncogenic enhancer network in inflammation-induced metaplastic progenitor cells to initiate pancreatic cancer; citation_author=Y. Li; citation_author=Y. He; citation_author=J. Peng; citation_author=Z. Su; citation_author=Z. Li; citation_author=B. Zhang; citation_author=J. Ma; citation_author=M. Zhuo; citation_author=D. Zou; citation_author=X. Liu; citation_author=et al.; citation_journal_title=Nature Cancer; citation_volume=2; citation_firstpage=49; citation_lastpage=65; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=Stromal Microenvironment Shapes the Intratumoral Architecture of Pancreatic Cancer.; citation_author=M. Ligorio; citation_author=S. Sil; citation_author=J. Malagon-Lopez; citation_author=L.T. Nieman; citation_author=S. Misale; citation_author=M. Di Pilato; citation_author=R.Y. Ebright; citation_author=M.N. Karabacak; citation_author=A.S. Kulkarni; citation_author=A. Liu; citation_author=et al.; citation_journal_title=Cell; citation_volume=178; citation_firstpage=160; citation_lastpage=175 e127; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Integrating single-cell and spatial transcriptomics to elucidate intercellular tissue dynamics; citation_author=S.K. Longo; citation_author=M.G. Guo; citation_author=A.L. Ji; citation_author=P.A. Khavari; citation_journal_title=Nat. Rev. Genet.; citation_volume=22; citation_firstpage=627; citation_lastpage=644; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=Highly Parallel Genome-wide Expression Profiling of Individual Cells Using Nanoliter Droplets; citation_author=E.Z. Macosko; citation_author=A. Basu; citation_author=R. Satija; citation_author=J. Nemesh; citation_author=K. Shekhar; citation_author=M. Goldman; citation_author=I. Tirosh; citation_author=A.R. Bialas; citation_author=N. Kamitaki; citation_author=E.M. Martersteck; citation_author=et al.; citation_journal_title=Cell; citation_volume=161; citation_firstpage=1202; citation_lastpage=1214; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Intraductal Transplantation Models of Human Pancreatic Ductal Adenocarcinoma Reveal Progressive Transition of Molecular Subtypes; citation_author=K. Miyabayashi; citation_author=L.A. Baker; citation_author=A. Deschênes; citation_author=B. Traub; citation_author=G. Caligiuri; citation_author=D. Plenker; citation_author=B. Alagesan; citation_author=P. Belleau; citation_author=S. Li; citation_author=J. Kendall; citation_author=et al.; citation_journal_title=Cancer Discov.; citation_volume=10; citation_firstpage=1566; citation_lastpage=1589; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Virtual microdissection identifies distinct tumor- and stroma-specific subtypes of pancreatic ductal adenocarcinoma; citation_author=R.A. Moffitt; citation_author=R. Marayati; citation_author=E.L. Flate; citation_author=K.E. Volmar; citation_author=S.G. Loeza; citation_author=K.A. Hoadley; citation_author=N.U. Rashid; citation_author=L.A. Williams; citation_author=S.C. Eaton; citation_author=A.H. Chung; citation_author=et al.; citation_journal_title=Nat. Genet.; citation_volume=47; citation_firstpage=1168; citation_lastpage=1178; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Integrating genetic and non-genetic determinants of cancer evolution by single-cell multi-omics; citation_author=A.S. Nam; citation_author=R. Chaligne; citation_author=D.A. Landau; citation_journal_title=Nat. Rev. Genet.; citation_volume=22; citation_firstpage=3; citation_lastpage=18; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=An Integrative Model of Cellular States, Plasticity, and Genetics for Glioblastoma; citation_author=C. Neftel; citation_author=J. Laffy; citation_author=M.G. Filbin; citation_author=T. Hara; citation_author=M.E. Shore; citation_author=G.J. Rahme; citation_author=A.R. Richman; citation_author=D. Silverbush; citation_author=M.L. Shaw; citation_author=C.M. Hebert; citation_author=et al.; citation_journal_title=Cell; citation_volume=178; citation_firstpage=835; citation_lastpage=849 e821; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=GATA6 Expression Distinguishes Classical and Basal-like Subtypes in Advanced Pancreatic Cancer; citation_author=G.M. O’Kane; citation_author=B.T. Grunwald; citation_author=G.H. Jang; citation_author=M. Masoomian; citation_author=S. Picardo; citation_author=R.C. Grant; citation_author=R.E. Denroche; citation_author=A. Zhang; citation_author=Y. Wang; citation_author=B. Lam; citation_author=et al.; citation_journal_title=Clin Cancer Res.; citation_volume=26; citation_firstpage=4901; citation_lastpage=4910; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Distinct populations of inflammatory fibroblasts and myofibroblasts in pancreatic cancer; citation_author=D. Öhlund; citation_author=A. Handly-Santana; citation_author=G. Biffi; citation_author=E. Elyada; citation_author=A.S. Almeida; citation_author=M. Ponz-Sarvise; citation_author=V. Corbo; citation_author=T.E. Oni; citation_author=S.A. Hearn; citation_author=E.J. Lee; citation_author=et al.; citation_journal_title=J.&nbsp;Exp. Med.; citation_volume=214; citation_firstpage=579; citation_lastpage=596; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Combination Cancer Therapy Can Confer Benefit via Patient-to-Patient Variability without Drug Additivity or Synergy; citation_author=A.C. Palmer; citation_author=P.K. Sorger; citation_journal_title=Cell; citation_volume=171; citation_firstpage=1678; citation_lastpage=1691 e1613; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=A curative combination cancer therapy achieves high fractional cell killing through low cross-resistance and drug additivity; citation_author=A.C. Palmer; citation_author=C. Chidley; citation_author=P.K. Sorger; citation_journal_title=eLife; citation_volume=8; citation_firstpage=e50036; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Single-cell RNA-seq highlights intratumoral heterogeneity in primary glioblastoma; citation_author=A.P. Patel; citation_author=I. Tirosh; citation_author=J.J. Trombetta; citation_author=A.K. Shalek; citation_author=S.M. Gillespie; citation_author=H. Wakimoto; citation_author=D.P. Cahill; citation_author=B.V. Nahed; citation_author=W.T. Curry; citation_author=R.L. Martuza; citation_author=et al.; citation_journal_title=Science; citation_volume=344; citation_firstpage=1396; citation_lastpage=1401; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Cancer-associated fibroblast heterogeneity in axillary lymph nodes drives metastases in breast cancer through complementary mechanisms; citation_author=F. Pelon; citation_author=B. Bourachot; citation_author=Y. Kieffer; citation_author=I. Magagna; citation_author=F. Mermet-Meillon; citation_author=I. Bonnet; citation_author=A. Costa; citation_author=A.M. Givel; citation_author=Y. Attieh; citation_author=J. Barbazan; citation_author=et al.; citation_journal_title=Nat. Commun.; citation_volume=11; citation_firstpage=404; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Epithelial to mesenchymal plasticity and differential response to therapies in pancreatic ductal adenocarcinoma; citation_author=R.L. Porter; citation_author=N.K.C. Magnus; citation_author=V. Thapar; citation_author=R. Morris; citation_author=A. Szabolcs; citation_author=A. Neyaz; citation_author=A.S. Kulkarni; citation_author=E. Tai; citation_author=A. Chougule; citation_author=A. Hillis; citation_author=et al.; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=116; citation_firstpage=26835; citation_lastpage=26845; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Single-Cell Transcriptomic Analysis of Primary and Metastatic Tumor Ecosystems in Head and Neck Cancer; citation_author=S.V. Puram; citation_author=I. Tirosh; citation_author=A.S. Parikh; citation_author=A.P. Patel; citation_author=K. Yizhak; citation_author=S. Gillespie; citation_author=C. Rodman; citation_author=C.L. Luo; citation_author=E.A. Mroz; citation_author=K.S. Emerick; citation_author=et al.; citation_journal_title=Cell; citation_volume=171; citation_firstpage=1611; citation_lastpage=1624 e1624; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Single-cell resolution analysis of the human pancreatic ductal progenitor cell niche; citation_author=M.M.F. Qadir; citation_author=S. Álvarez-Cubela; citation_author=D. Klein; citation_author=J. van Dijk; citation_author=R. Muñiz-Anquela; citation_author=Y.B. Moreno-Hernández; citation_author=G. Lanzoni; citation_author=S. Sadiq; citation_author=B. Navarro-Rubio; citation_author=M.T. García; citation_author=et al.; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=117; citation_firstpage=10876; citation_lastpage=10887; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Defining T Cell States Associated with Response to Checkpoint Immunotherapy in Melanoma; citation_author=M. Sade-Feldman; citation_author=K. Yizhak; citation_author=S.L. Bjorgaard; citation_author=J.P. Ray; citation_author=C.G. de Boer; citation_author=R.W. Jenkins; citation_author=D.J. Lieb; citation_author=J.H. Chen; citation_author=D.T. Frederick; citation_author=M. Barzily-Rokni; citation_author=et al.; citation_journal_title=Cell; citation_volume=176; citation_firstpage=404; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=A framework for advancing our understanding of cancer-associated fibroblasts; citation_author=E. Sahai; citation_author=I. Astsaturov; citation_author=E. Cukierman; citation_author=D.G. DeNardo; citation_author=M. Egeblad; citation_author=R.M. Evans; citation_author=D. Fearon; citation_author=F.R. Greten; citation_author=S.R. Hingorani; citation_author=T. Hunter; citation_author=et al.; citation_journal_title=Nat. Rev. Cancer; citation_volume=20; citation_firstpage=174; citation_lastpage=186; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Institutional implementation of clinical tumor profiling on an unselected cancer population; citation_author=L.M. Sholl; citation_author=K. Do; citation_author=P. Shivdasani; citation_author=E. Cerami; citation_author=A.M. Dubuc; citation_author=F.C. Kuo; citation_author=E.P. Garcia; citation_author=Y. Jia; citation_author=P. Davineni; citation_author=R.P. Abo; citation_author=et al.; citation_journal_title=JCI Insight; citation_volume=1; citation_firstpage=e87062; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=TP63-Mediated Enhancer Reprogramming Drives the Squamous Subtype of Pancreatic Ductal Adenocarcinoma; citation_author=T.D.D. Somerville; citation_author=Y. Xu; citation_author=K. Miyabayashi; citation_author=H. Tiriac; citation_author=C.R. Cleary; citation_author=D. Maia-Silva; citation_author=J.P. Milazzo; citation_author=D.A. Tuveson; citation_author=C.R. Vakoc; citation_journal_title=Cell Rep.; citation_volume=25; citation_firstpage=1741; citation_lastpage=1755 e1747; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Quantification of aneuploidy in targeted sequencing data using ASCETS; citation_author=L.F. Spurr; citation_author=M. Touat; citation_author=A.M. Taylor; citation_author=A.M. Dubuc; citation_author=J. Shih; citation_author=D.M. Meredith; citation_author=W.V. Pisano; citation_author=M.L. Meyerson; citation_author=K.L. Ligon; citation_author=A.D. Cherniack; citation_author=et al.; citation_journal_title=Bioinformatics; citation_volume=37; citation_firstpage=2461; citation_lastpage=2463; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=Single-Cell RNA Sequencing in Cancer: Lessons Learned and Emerging Challenges; citation_author=M.L. Suvà; citation_author=I. Tirosh; citation_journal_title=Mol. Cell; citation_volume=75; citation_firstpage=7; citation_lastpage=12; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Organoid Profiling Identifies Common Responders to Chemotherapy in Pancreatic Cancer; citation_author=H. Tiriac; citation_author=P. Belleau; citation_author=D.D. Engle; citation_author=D. Plenker; citation_author=A. Deschênes; citation_author=T.D.D. Somerville; citation_author=F.E.M. Froeling; citation_author=R.A. Burkhart; citation_author=R.E. Denroche; citation_author=G.H. Jang; citation_author=et al.; citation_journal_title=Cancer Discov.; citation_volume=8; citation_firstpage=1112; citation_lastpage=1129; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Dissecting the multicellular ecosystem of metastatic melanoma by single-cell RNA-seq; citation_author=I. Tirosh; citation_author=B. Izar; citation_author=S.M. Prakadan; citation_author=M.H. Wadsworth; citation_author=D. Treacy; citation_author=J.J. Trombetta; citation_author=A. Rotem; citation_author=C. Rodman; citation_author=C. Lian; citation_author=G. Murphy; citation_author=et al.; citation_journal_title=Science; citation_volume=352; citation_firstpage=189; citation_lastpage=196; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Single-cell RNA-seq supports a developmental hierarchy in human oligodendroglioma; citation_author=I. Tirosh; citation_author=A.S. Venteicher; citation_author=C. Hebert; citation_author=L.E. Escalante; citation_author=A.P. Patel; citation_author=K. Yizhak; citation_author=J.M. Fisher; citation_author=C. Rodman; citation_author=C. Mount; citation_author=M.G. Filbin; citation_author=et al.; citation_journal_title=Nature; citation_volume=539; citation_firstpage=309; citation_lastpage=313; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Limited evolution of the actionable metastatic cancer genome under therapeutic pressure; citation_author=J. van de Haar; citation_author=L.R. Hoes; citation_author=P. Roepman; citation_author=M.P. Lolkema; citation_author=H.M.W. Verheul; citation_author=H. Gelderblom; citation_author=A.J. de Langen; citation_author=E.F. Smit; citation_author=E. Cuppen; citation_author=L.F.A. Wessels; citation_author=E.E. Voest; citation_journal_title=Nat. Med.; citation_volume=27; citation_firstpage=1553; citation_lastpage=1563; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=Single-Cell RNA-Seq Reveals AML Hierarchies Relevant to Disease Progression and Immunity; citation_author=P. van Galen; citation_author=V. Hovestadt; citation_author=M.H. Wadsworth Ii; citation_author=T.K. Hughes; citation_author=G.K. Griffin; citation_author=S. Battaglia; citation_author=J.A. Verga; citation_author=J. Stephansky; citation_author=T.J. Pastika; citation_author=J. Lombardi Story; citation_author=et al.; citation_journal_title=Cell; citation_volume=176; citation_firstpage=1265; citation_lastpage=1281 e1224; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Decoupling genetics, lineages, and microenvironment in IDH-mutant gliomas by single-cell RNA-seq; citation_author=A.S. Venteicher; citation_author=I. Tirosh; citation_author=C. Hebert; citation_author=K. Yizhak; citation_author=C. Neftel; citation_author=M.G. Filbin; citation_author=V. Hovestadt; citation_author=L.E. Escalante; citation_author=M.L. Shaw; citation_author=C. Rodman; citation_author=et al.; citation_journal_title=Science; citation_volume=355; citation_firstpage=eaai8478; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Toil enables reproducible, open source, big biomedical data analyses; citation_author=J. Vivian; citation_author=A.A. Rao; citation_author=F.A. Nothaft; citation_author=C. Ketchum; citation_author=J. Armstrong; citation_author=A. Novak; citation_author=J. Pfeil; citation_author=J. Narkizian; citation_author=A.D. Deran; citation_author=A. Musselman-Brown; citation_author=et al.; citation_journal_title=Nat. Biotechnol.; citation_volume=35; citation_firstpage=314; citation_lastpage=316; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Lineage tracing meets single-cell omics: opportunities and challenges; citation_author=D.E. Wagner; citation_author=A.M. Klein; citation_journal_title=Nat. Rev. Genet.; citation_volume=21; citation_firstpage=410; citation_lastpage=427; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=The Cancer Genome Atlas Pan-Cancer analysis project; citation_author=J.N. Weinstein; citation_author=E.A. Collisson; citation_author=G.B. Mills; citation_author=K.R. Shaw; citation_author=B.A. Ozenberger; citation_author=K. Ellrott; citation_author=I. Shmulevich; citation_author=C. Sander; citation_author=J.M. Stuart; citation_author=Cancer Genome Atlas Research Network; citation_journal_title=Nat. Genet.; citation_volume=45; citation_firstpage=1113; citation_lastpage=1120; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=Claudin 18.2 is a target for IMAB362 antibody in pancreatic neoplasms; citation_author=S. Wöll; citation_author=A.M. Schlitter; citation_author=K. Dhaene; citation_author=M. Roller; citation_author=I. Esposito; citation_author=U. Sahin; citation_author=Ö. Türeci; citation_journal_title=Int. J. Cancer; citation_volume=134; citation_firstpage=731; citation_lastpage=739; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Single-Cell Analyses Inform Mechanisms of Myeloid-Targeted Therapies in Colon Cancer; citation_author=L. Zhang; citation_author=Z. Li; citation_author=K.M. Skrzypczynska; citation_author=Q. Fang; citation_author=W. Zhang; citation_author=S.A. O’Brien; citation_author=Y. He; citation_author=L. Wang; citation_author=Q. Zhang; citation_author=A. Kim; citation_author=et al.; citation_journal_title=Cell; citation_volume=181; citation_firstpage=442; citation_lastpage=459 e429; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Single-Cell Transcriptomics of Human and Mouse Lung Cancers Reveals Conserved Myeloid Populations across Individuals and Species; citation_author=R. Zilionis; citation_author=C. Engblom; citation_author=C. Pfirschke; citation_author=V. Savova; citation_author=D. Zemmour; citation_author=H.D. Saatcioglu; citation_author=I. Krishnan; citation_author=G. Maroni; citation_author=C.V. Meyerovitz; citation_author=C.M. Kerwin; citation_author=et al.; citation_journal_title=Immunity; citation_volume=50; citation_firstpage=1317; citation_lastpage=1334 e1310; citation_publication_date=2019; "><meta name="citation_abstract" content="<h2>Summary</h2><p>Prognostically relevant RNA expression states exist in pancreatic ductal adenocarcinoma (PDAC), but our understanding of their drivers, stability, and relationship to therapeutic response is limited. To examine these attributes systematically, we profiled metastatic biopsies and matched organoid models at single-cell resolution. <i>In&nbsp;vivo</i>, we identify a new intermediate PDAC transcriptional cell state and uncover distinct site- and state-specific tumor microenvironments (TMEs). Benchmarking models against this reference map, we reveal strong culture-specific biases in cancer cell transcriptional state representation driven by altered TME signals. We restore expression state heterogeneity by adding back <i>in&nbsp;vivo</i>-relevant factors and show plasticity in culture models. Further, we prove that non-genetic modulation of cell state can strongly influence drug responses, uncovering state-specific vulnerabilities. This work provides a broadly applicable framework for aligning cell states across <i>in&nbsp;vivo</i> and <i>ex&nbsp;vivo</i> settings, identifying drivers of transcriptional plasticity and manipulating cell state to target associated vulnerabilities.</p>"><meta charset="UTF-8"><!--

    Pagebuild admin UI
-->



<script type="text/javascript" src="https://bam.nr-data.net/1/16e2192192?a=916957862&amp;sa=1&amp;v=1177.96a4d39&amp;t=Unnamed%20Transaction&amp;rst=7829&amp;ck=1&amp;ref=https://www.cell.com/cell/fulltext/S0092-8674(21)01332-5&amp;be=2734&amp;fe=7789&amp;dc=6312&amp;af=err,xhr,stn,ins,spa&amp;perf=%7B%22timing%22:%7B%22of%22:1749592087859,%22n%22:0,%22f%22:2206,%22dn%22:2207,%22dne%22:2207,%22c%22:2207,%22s%22:2207,%22ce%22:2207,%22rq%22:2207,%22rp%22:2718,%22rpe%22:3733,%22dl%22:2723,%22di%22:3964,%22ds%22:6312,%22de%22:6323,%22dc%22:7788,%22l%22:7789,%22le%22:7819%7D,%22navigation%22:%7B%7D%7D&amp;jsonp=NREUM.setToken"></script><script src="https://js-agent.newrelic.com/nr-spa-1177.min.js"></script><script type="text/javascript">
    ;window.NREUM||(NREUM={});NREUM.init={privacy:{cookies_enabled:true}};
    window.NREUM||(NREUM={}),__nr_require=function(t,e,n){function r(n){if(!e[n]){var o=e[n]={exports:{}};t[n][0].call(o.exports,function(e){var o=t[n][1][e];return r(o||e)},o,o.exports)}return e[n].exports}if("function"==typeof __nr_require)return __nr_require;for(var o=0;o<n.length;o++)r(n[o]);return r}({1:[function(t,e,n){function r(t){try{c.console&&console.log(t)}catch(e){}}var o,i=t("ee"),a=t(26),c={};try{o=localStorage.getItem("__nr_flags").split(","),console&&"function"==typeof console.log&&(c.console=!0,o.indexOf("dev")!==-1&&(c.dev=!0),o.indexOf("nr_dev")!==-1&&(c.nrDev=!0))}catch(s){}c.nrDev&&i.on("internal-error",function(t){r(t.stack)}),c.dev&&i.on("fn-err",function(t,e,n){r(n.stack)}),c.dev&&(r("NR AGENT IN DEVELOPMENT MODE"),r("flags: "+a(c,function(t,e){return t}).join(", ")))},{}],2:[function(t,e,n){function r(t,e,n,r,c){try{l?l-=1:o(c||new UncaughtException(t,e,n),!0)}catch(f){try{i("ierr",[f,s.now(),!0])}catch(d){}}return"function"==typeof u&&u.apply(this,a(arguments))}function UncaughtException(t,e,n){this.message=t||"Uncaught error with no additional information",this.sourceURL=e,this.line=n}function o(t,e){var n=e?null:s.now();i("err",[t,n])}var i=t("handle"),a=t(27),c=t("ee"),s=t("loader"),f=t("gos"),u=window.onerror,d=!1,p="nr@seenError",l=0;s.features.err=!0,t(1),window.onerror=r;try{throw new Error}catch(h){"stack"in h&&(t(13),t(12),"addEventListener"in window&&t(6),s.xhrWrappable&&t(14),d=!0)}c.on("fn-start",function(t,e,n){d&&(l+=1)}),c.on("fn-err",function(t,e,n){d&&!n[p]&&(f(n,p,function(){return!0}),this.thrown=!0,o(n))}),c.on("fn-end",function(){d&&!this.thrown&&l>0&&(l-=1)}),c.on("internal-error",function(t){i("ierr",[t,s.now(),!0])})},{}],3:[function(t,e,n){t("loader").features.ins=!0},{}],4:[function(t,e,n){function r(){_++,T=g.hash,this[u]=y.now()}function o(){_--,g.hash!==T&&i(0,!0);var t=y.now();this[h]=~~this[h]+t-this[u],this[d]=t}function i(t,e){E.emit("newURL",[""+g,e])}function a(t,e){t.on(e,function(){this[e]=y.now()})}var c="-start",s="-end",f="-body",u="fn"+c,d="fn"+s,p="cb"+c,l="cb"+s,h="jsTime",m="fetch",v="addEventListener",w=window,g=w.location,y=t("loader");if(w[v]&&y.xhrWrappable){var x=t(10),b=t(11),E=t(8),R=t(6),O=t(13),N=t(7),M=t(14),P=t(9),S=t("ee"),C=S.get("tracer");t(16),y.features.spa=!0;var T,_=0;S.on(u,r),S.on(p,r),S.on(d,o),S.on(l,o),S.buffer([u,d,"xhr-done","xhr-resolved"]),R.buffer([u]),O.buffer(["setTimeout"+s,"clearTimeout"+c,u]),M.buffer([u,"new-xhr","send-xhr"+c]),N.buffer([m+c,m+"-done",m+f+c,m+f+s]),E.buffer(["newURL"]),x.buffer([u]),b.buffer(["propagate",p,l,"executor-err","resolve"+c]),C.buffer([u,"no-"+u]),P.buffer(["new-jsonp","cb-start","jsonp-error","jsonp-end"]),a(M,"send-xhr"+c),a(S,"xhr-resolved"),a(S,"xhr-done"),a(N,m+c),a(N,m+"-done"),a(P,"new-jsonp"),a(P,"jsonp-end"),a(P,"cb-start"),E.on("pushState-end",i),E.on("replaceState-end",i),w[v]("hashchange",i,!0),w[v]("load",i,!0),w[v]("popstate",function(){i(0,_>1)},!0)}},{}],5:[function(t,e,n){function r(t){}if(window.performance&&window.performance.timing&&window.performance.getEntriesByType){var o=t("ee"),i=t("handle"),a=t(13),c=t(12),s="learResourceTimings",f="addEventListener",u="resourcetimingbufferfull",d="bstResource",p="resource",l="-start",h="-end",m="fn"+l,v="fn"+h,w="bstTimer",g="pushState",y=t("loader");y.features.stn=!0,t(8),"addEventListener"in window&&t(6);var x=NREUM.o.EV;o.on(m,function(t,e){var n=t[0];n instanceof x&&(this.bstStart=y.now())}),o.on(v,function(t,e){var n=t[0];n instanceof x&&i("bst",[n,e,this.bstStart,y.now()])}),a.on(m,function(t,e,n){this.bstStart=y.now(),this.bstType=n}),a.on(v,function(t,e){i(w,[e,this.bstStart,y.now(),this.bstType])}),c.on(m,function(){this.bstStart=y.now()}),c.on(v,function(t,e){i(w,[e,this.bstStart,y.now(),"requestAnimationFrame"])}),o.on(g+l,function(t){this.time=y.now(),this.startPath=location.pathname+location.hash}),o.on(g+h,function(t){i("bstHist",[location.pathname+location.hash,this.startPath,this.time])}),f in window.performance&&(window.performance["c"+s]?window.performance[f](u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["c"+s]()},!1):window.performance[f]("webkit"+u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["webkitC"+s]()},!1)),document[f]("scroll",r,{passive:!0}),document[f]("keypress",r,!1),document[f]("click",r,!1)}},{}],6:[function(t,e,n){function r(t){for(var e=t;e&&!e.hasOwnProperty(u);)e=Object.getPrototypeOf(e);e&&o(e)}function o(t){c.inPlace(t,[u,d],"-",i)}function i(t,e){return t[1]}var a=t("ee").get("events"),c=t("wrap-function")(a,!0),s=t("gos"),f=XMLHttpRequest,u="addEventListener",d="removeEventListener";e.exports=a,"getPrototypeOf"in Object?(r(document),r(window),r(f.prototype)):f.prototype.hasOwnProperty(u)&&(o(window),o(f.prototype)),a.on(u+"-start",function(t,e){var n=t[1],r=s(n,"nr@wrapped",function(){function t(){if("function"==typeof n.handleEvent)return n.handleEvent.apply(n,arguments)}var e={object:t,"function":n}[typeof n];return e?c(e,"fn-",null,e.name||"anonymous"):n});this.wrapped=t[1]=r}),a.on(d+"-start",function(t){t[1]=this.wrapped||t[1]})},{}],7:[function(t,e,n){function r(t,e,n){var r=t[e];"function"==typeof r&&(t[e]=function(){var t=i(arguments),e={};o.emit(n+"before-start",[t],e);var a;e[m]&&e[m].dt&&(a=e[m].dt);var c=r.apply(this,t);return o.emit(n+"start",[t,a],c),c.then(function(t){return o.emit(n+"end",[null,t],c),t},function(t){throw o.emit(n+"end",[t],c),t})})}var o=t("ee").get("fetch"),i=t(27),a=t(26);e.exports=o;var c=window,s="fetch-",f=s+"body-",u=["arrayBuffer","blob","json","text","formData"],d=c.Request,p=c.Response,l=c.fetch,h="prototype",m="nr@context";d&&p&&l&&(a(u,function(t,e){r(d[h],e,f),r(p[h],e,f)}),r(c,"fetch",s),o.on(s+"end",function(t,e){var n=this;if(e){var r=e.headers.get("content-length");null!==r&&(n.rxSize=r),o.emit(s+"done",[null,e],n)}else o.emit(s+"done",[t],n)}))},{}],8:[function(t,e,n){var r=t("ee").get("history"),o=t("wrap-function")(r);e.exports=r;var i=window.history&&window.history.constructor&&window.history.constructor.prototype,a=window.history;i&&i.pushState&&i.replaceState&&(a=i),o.inPlace(a,["pushState","replaceState"],"-")},{}],9:[function(t,e,n){function r(t){function e(){s.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}function n(){s.emit("jsonp-error",[],p),s.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}var r=t&&"string"==typeof t.nodeName&&"script"===t.nodeName.toLowerCase();if(r){var o="function"==typeof t.addEventListener;if(o){var a=i(t.src);if(a){var u=c(a),d="function"==typeof u.parent[u.key];if(d){var p={};f.inPlace(u.parent,[u.key],"cb-",p),t.addEventListener("load",e,!1),t.addEventListener("error",n,!1),s.emit("new-jsonp",[t.src],p)}}}}}function o(){return"addEventListener"in window}function i(t){var e=t.match(u);return e?e[1]:null}function a(t,e){var n=t.match(p),r=n[1],o=n[3];return o?a(o,e[r]):e[r]}function c(t){var e=t.match(d);return e&&e.length>=3?{key:e[2],parent:a(e[1],window)}:{key:t,parent:window}}var s=t("ee").get("jsonp"),f=t("wrap-function")(s);if(e.exports=s,o()){var u=/[?&](?:callback|cb)=([^&#]+)/,d=/(.*)\.([^.]+)/,p=/^(\w+)(\.|$)(.*)$/,l=["appendChild","insertBefore","replaceChild"];Node&&Node.prototype&&Node.prototype.appendChild?f.inPlace(Node.prototype,l,"dom-"):(f.inPlace(HTMLElement.prototype,l,"dom-"),f.inPlace(HTMLHeadElement.prototype,l,"dom-"),f.inPlace(HTMLBodyElement.prototype,l,"dom-")),s.on("dom-start",function(t){r(t[0])})}},{}],10:[function(t,e,n){var r=t("ee").get("mutation"),o=t("wrap-function")(r),i=NREUM.o.MO;e.exports=r,i&&(window.MutationObserver=function(t){return this instanceof i?new i(o(t,"fn-")):i.apply(this,arguments)},MutationObserver.prototype=i.prototype)},{}],11:[function(t,e,n){function r(t){var e=a.context(),n=c(t,"executor-",e),r=new f(n);return a.context(r).getCtx=function(){return e},a.emit("new-promise",[r,e],e),r}function o(t,e){return e}var i=t("wrap-function"),a=t("ee").get("promise"),c=i(a),s=t(26),f=NREUM.o.PR;e.exports=a,f&&(window.Promise=r,["all","race"].forEach(function(t){var e=f[t];f[t]=function(n){function r(t){return function(){a.emit("propagate",[null,!o],i),o=o||!t}}var o=!1;s(n,function(e,n){Promise.resolve(n).then(r("all"===t),r(!1))});var i=e.apply(f,arguments),c=f.resolve(i);return c}}),["resolve","reject"].forEach(function(t){var e=f[t];f[t]=function(t){var n=e.apply(f,arguments);return t!==n&&a.emit("propagate",[t,!0],n),n}}),f.prototype["catch"]=function(t){return this.then(null,t)},f.prototype=Object.create(f.prototype,{constructor:{value:r}}),s(Object.getOwnPropertyNames(f),function(t,e){try{r[e]=f[e]}catch(n){}}),a.on("executor-start",function(t){t[0]=c(t[0],"resolve-",this),t[1]=c(t[1],"resolve-",this)}),a.on("executor-err",function(t,e,n){t[1](n)}),c.inPlace(f.prototype,["then"],"then-",o),a.on("then-start",function(t,e){this.promise=e,t[0]=c(t[0],"cb-",this),t[1]=c(t[1],"cb-",this)}),a.on("then-end",function(t,e,n){this.nextPromise=n;var r=this.promise;a.emit("propagate",[r,!0],n)}),a.on("cb-end",function(t,e,n){a.emit("propagate",[n,!0],this.nextPromise)}),a.on("propagate",function(t,e,n){this.getCtx&&!e||(this.getCtx=function(){if(t instanceof Promise)var e=a.context(t);return e&&e.getCtx?e.getCtx():this})}),r.toString=function(){return""+f})},{}],12:[function(t,e,n){var r=t("ee").get("raf"),o=t("wrap-function")(r),i="equestAnimationFrame";e.exports=r,o.inPlace(window,["r"+i,"mozR"+i,"webkitR"+i,"msR"+i],"raf-"),r.on("raf-start",function(t){t[0]=o(t[0],"fn-")})},{}],13:[function(t,e,n){function r(t,e,n){t[0]=a(t[0],"fn-",null,n)}function o(t,e,n){this.method=n,this.timerDuration=isNaN(t[1])?0:+t[1],t[0]=a(t[0],"fn-",this,n)}var i=t("ee").get("timer"),a=t("wrap-function")(i),c="setTimeout",s="setInterval",f="clearTimeout",u="-start",d="-";e.exports=i,a.inPlace(window,[c,"setImmediate"],c+d),a.inPlace(window,[s],s+d),a.inPlace(window,[f,"clearImmediate"],f+d),i.on(s+u,r),i.on(c+u,o)},{}],14:[function(t,e,n){function r(t,e){d.inPlace(e,["onreadystatechange"],"fn-",c)}function o(){var t=this,e=u.context(t);t.readyState>3&&!e.resolved&&(e.resolved=!0,u.emit("xhr-resolved",[],t)),d.inPlace(t,g,"fn-",c)}function i(t){y.push(t),h&&(b?b.then(a):v?v(a):(E=-E,R.data=E))}function a(){for(var t=0;t<y.length;t++)r([],y[t]);y.length&&(y=[])}function c(t,e){return e}function s(t,e){for(var n in t)e[n]=t[n];return e}t(6);var f=t("ee"),u=f.get("xhr"),d=t("wrap-function")(u),p=NREUM.o,l=p.XHR,h=p.MO,m=p.PR,v=p.SI,w="readystatechange",g=["onload","onerror","onabort","onloadstart","onloadend","onprogress","ontimeout"],y=[];e.exports=u;var x=window.XMLHttpRequest=function(t){var e=new l(t);try{u.emit("new-xhr",[e],e),e.addEventListener(w,o,!1)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}return e};if(s(l,x),x.prototype=l.prototype,d.inPlace(x.prototype,["open","send"],"-xhr-",c),u.on("send-xhr-start",function(t,e){r(t,e),i(e)}),u.on("open-xhr-start",r),h){var b=m&&m.resolve();if(!v&&!m){var E=1,R=document.createTextNode(E);new h(a).observe(R,{characterData:!0})}}else f.on("fn-end",function(t){t[0]&&t[0].type===w||a()})},{}],15:[function(t,e,n){function r(t){if(!c(t))return null;var e=window.NREUM;if(!e.loader_config)return null;var n=(e.loader_config.accountID||"").toString()||null,r=(e.loader_config.agentID||"").toString()||null,f=(e.loader_config.trustKey||"").toString()||null;if(!n||!r)return null;var h=l.generateSpanId(),m=l.generateTraceId(),v=Date.now(),w={spanId:h,traceId:m,timestamp:v};return(t.sameOrigin||s(t)&&p())&&(w.traceContextParentHeader=o(h,m),w.traceContextStateHeader=i(h,v,n,r,f)),(t.sameOrigin&&!u()||!t.sameOrigin&&s(t)&&d())&&(w.newrelicHeader=a(h,m,v,n,r,f)),w}function o(t,e){return"00-"+e+"-"+t+"-01"}function i(t,e,n,r,o){var i=0,a="",c=1,s="",f="";return o+"@nr="+i+"-"+c+"-"+n+"-"+r+"-"+t+"-"+a+"-"+s+"-"+f+"-"+e}function a(t,e,n,r,o,i){var a="btoa"in window&&"function"==typeof window.btoa;if(!a)return null;var c={v:[0,1],d:{ty:"Browser",ac:r,ap:o,id:t,tr:e,ti:n}};return i&&r!==i&&(c.d.tk=i),btoa(JSON.stringify(c))}function c(t){return f()&&s(t)}function s(t){var e=!1,n={};if("init"in NREUM&&"distributed_tracing"in NREUM.init&&(n=NREUM.init.distributed_tracing),t.sameOrigin)e=!0;else if(n.allowed_origins instanceof Array)for(var r=0;r<n.allowed_origins.length;r++){var o=h(n.allowed_origins[r]);if(t.hostname===o.hostname&&t.protocol===o.protocol&&t.port===o.port){e=!0;break}}return e}function f(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.enabled}function u(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.exclude_newrelic_header}function d(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&NREUM.init.distributed_tracing.cors_use_newrelic_header!==!1}function p(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.cors_use_tracecontext_headers}var l=t(23),h=t(17);e.exports={generateTracePayload:r,shouldGenerateTrace:c}},{}],16:[function(t,e,n){function r(t){var e=this.params,n=this.metrics;if(!this.ended){this.ended=!0;for(var r=0;r<p;r++)t.removeEventListener(d[r],this.listener,!1);e.aborted||(n.duration=a.now()-this.startTime,this.loadCaptureCalled||4!==t.readyState?null==e.status&&(e.status=0):i(this,t),n.cbTime=this.cbTime,u.emit("xhr-done",[t],t),c("xhr",[e,n,this.startTime]))}}function o(t,e){var n=s(e),r=t.params;r.host=n.hostname+":"+n.port,r.pathname=n.pathname,t.parsedOrigin=s(e),t.sameOrigin=t.parsedOrigin.sameOrigin}function i(t,e){t.params.status=e.status;var n=v(e,t.lastSize);if(n&&(t.metrics.rxSize=n),t.sameOrigin){var r=e.getResponseHeader("X-NewRelic-App-Data");r&&(t.params.cat=r.split(", ").pop())}t.loadCaptureCalled=!0}var a=t("loader");if(a.xhrWrappable){var c=t("handle"),s=t(17),f=t(15).generateTracePayload,u=t("ee"),d=["load","error","abort","timeout"],p=d.length,l=t("id"),h=t(21),m=t(20),v=t(18),w=window.XMLHttpRequest;a.features.xhr=!0,t(14),t(7),u.on("new-xhr",function(t){var e=this;e.totalCbs=0,e.called=0,e.cbTime=0,e.end=r,e.ended=!1,e.xhrGuids={},e.lastSize=null,e.loadCaptureCalled=!1,t.addEventListener("load",function(n){i(e,t)},!1),h&&(h>34||h<10)||window.opera||t.addEventListener("progress",function(t){e.lastSize=t.loaded},!1)}),u.on("open-xhr-start",function(t){this.params={method:t[0]},o(this,t[1]),this.metrics={}}),u.on("open-xhr-end",function(t,e){"loader_config"in NREUM&&"xpid"in NREUM.loader_config&&this.sameOrigin&&e.setRequestHeader("X-NewRelic-ID",NREUM.loader_config.xpid);var n=f(this.parsedOrigin);if(n){var r=!1;n.newrelicHeader&&(e.setRequestHeader("newrelic",n.newrelicHeader),r=!0),n.traceContextParentHeader&&(e.setRequestHeader("traceparent",n.traceContextParentHeader),n.traceContextStateHeader&&e.setRequestHeader("tracestate",n.traceContextStateHeader),r=!0),r&&(this.dt=n)}}),u.on("send-xhr-start",function(t,e){var n=this.metrics,r=t[0],o=this;if(n&&r){var i=m(r);i&&(n.txSize=i)}this.startTime=a.now(),this.listener=function(t){try{"abort"!==t.type||o.loadCaptureCalled||(o.params.aborted=!0),("load"!==t.type||o.called===o.totalCbs&&(o.onloadCalled||"function"!=typeof e.onload))&&o.end(e)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}};for(var c=0;c<p;c++)e.addEventListener(d[c],this.listener,!1)}),u.on("xhr-cb-time",function(t,e,n){this.cbTime+=t,e?this.onloadCalled=!0:this.called+=1,this.called!==this.totalCbs||!this.onloadCalled&&"function"==typeof n.onload||this.end(n)}),u.on("xhr-load-added",function(t,e){var n=""+l(t)+!!e;this.xhrGuids&&!this.xhrGuids[n]&&(this.xhrGuids[n]=!0,this.totalCbs+=1)}),u.on("xhr-load-removed",function(t,e){var n=""+l(t)+!!e;this.xhrGuids&&this.xhrGuids[n]&&(delete this.xhrGuids[n],this.totalCbs-=1)}),u.on("addEventListener-end",function(t,e){e instanceof w&&"load"===t[0]&&u.emit("xhr-load-added",[t[1],t[2]],e)}),u.on("removeEventListener-end",function(t,e){e instanceof w&&"load"===t[0]&&u.emit("xhr-load-removed",[t[1],t[2]],e)}),u.on("fn-start",function(t,e,n){e instanceof w&&("onload"===n&&(this.onload=!0),("load"===(t[0]&&t[0].type)||this.onload)&&(this.xhrCbStart=a.now()))}),u.on("fn-end",function(t,e){this.xhrCbStart&&u.emit("xhr-cb-time",[a.now()-this.xhrCbStart,this.onload,e],e)}),u.on("fetch-before-start",function(t){function e(t,e){var n=!1;return e.newrelicHeader&&(t.set("newrelic",e.newrelicHeader),n=!0),e.traceContextParentHeader&&(t.set("traceparent",e.traceContextParentHeader),e.traceContextStateHeader&&t.set("tracestate",e.traceContextStateHeader),n=!0),n}var n,r=t[1]||{};"string"==typeof t[0]?n=t[0]:t[0]&&t[0].url&&(n=t[0].url),n&&(this.parsedOrigin=s(n),this.sameOrigin=this.parsedOrigin.sameOrigin);var o=f(this.parsedOrigin);if(o&&(o.newrelicHeader||o.traceContextParentHeader))if("string"==typeof t[0]){var i={};for(var a in r)i[a]=r[a];i.headers=new Headers(r.headers||{}),e(i.headers,o)&&(this.dt=o),t.length>1?t[1]=i:t.push(i)}else t[0]&&t[0].headers&&e(t[0].headers,o)&&(this.dt=o)})}},{}],17:[function(t,e,n){var r={};e.exports=function(t){if(t in r)return r[t];var e=document.createElement("a"),n=window.location,o={};e.href=t,o.port=e.port;var i=e.href.split("://");!o.port&&i[1]&&(o.port=i[1].split("/")[0].split("@").pop().split(":")[1]),o.port&&"0"!==o.port||(o.port="https"===i[0]?"443":"80"),o.hostname=e.hostname||n.hostname,o.pathname=e.pathname,o.protocol=i[0],"/"!==o.pathname.charAt(0)&&(o.pathname="/"+o.pathname);var a=!e.protocol||":"===e.protocol||e.protocol===n.protocol,c=e.hostname===document.domain&&e.port===n.port;return o.sameOrigin=a&&(!e.hostname||c),"/"===o.pathname&&(r[t]=o),o}},{}],18:[function(t,e,n){function r(t,e){var n=t.responseType;return"json"===n&&null!==e?e:"arraybuffer"===n||"blob"===n||"json"===n?o(t.response):"text"===n||""===n||void 0===n?o(t.responseText):void 0}var o=t(20);e.exports=r},{}],19:[function(t,e,n){function r(){}function o(t,e,n){return function(){return i(t,[f.now()].concat(c(arguments)),e?null:this,n),e?void 0:this}}var i=t("handle"),a=t(26),c=t(27),s=t("ee").get("tracer"),f=t("loader"),u=NREUM;"undefined"==typeof window.newrelic&&(newrelic=u);var d=["setPageViewName","setCustomAttribute","setErrorHandler","finished","addToTrace","inlineHit","addRelease"],p="api-",l=p+"ixn-";a(d,function(t,e){u[e]=o(p+e,!0,"api")}),u.addPageAction=o(p+"addPageAction",!0),u.setCurrentRouteName=o(p+"routeName",!0),e.exports=newrelic,u.interaction=function(){return(new r).get()};var h=r.prototype={createTracer:function(t,e){var n={},r=this,o="function"==typeof e;return i(l+"tracer",[f.now(),t,n],r),function(){if(s.emit((o?"":"no-")+"fn-start",[f.now(),r,o],n),o)try{return e.apply(this,arguments)}catch(t){throw s.emit("fn-err",[arguments,this,t],n),t}finally{s.emit("fn-end",[f.now()],n)}}}};a("actionText,setName,setAttribute,save,ignore,onEnd,getContext,end,get".split(","),function(t,e){h[e]=o(l+e)}),newrelic.noticeError=function(t,e){"string"==typeof t&&(t=new Error(t)),i("err",[t,f.now(),!1,e])}},{}],20:[function(t,e,n){e.exports=function(t){if("string"==typeof t&&t.length)return t.length;if("object"==typeof t){if("undefined"!=typeof ArrayBuffer&&t instanceof ArrayBuffer&&t.byteLength)return t.byteLength;if("undefined"!=typeof Blob&&t instanceof Blob&&t.size)return t.size;if(!("undefined"!=typeof FormData&&t instanceof FormData))try{return JSON.stringify(t).length}catch(e){return}}}},{}],21:[function(t,e,n){var r=0,o=navigator.userAgent.match(/Firefox[\/\s](\d+\.\d+)/);o&&(r=+o[1]),e.exports=r},{}],22:[function(t,e,n){function r(t,e){var n=t.getEntries();n.forEach(function(t){"first-paint"===t.name?d("timing",["fp",Math.floor(t.startTime)]):"first-contentful-paint"===t.name&&d("timing",["fcp",Math.floor(t.startTime)])})}function o(t,e){var n=t.getEntries();n.length>0&&d("lcp",[n[n.length-1]])}function i(t){t.getEntries().forEach(function(t){t.hadRecentInput||d("cls",[t])})}function a(t){if(t instanceof h&&!v){var e=Math.round(t.timeStamp),n={type:t.type};e<=p.now()?n.fid=p.now()-e:e>p.offset&&e<=Date.now()?(e-=p.offset,n.fid=p.now()-e):e=p.now(),v=!0,d("timing",["fi",e,n])}}function c(t){d("pageHide",[p.now(),t])}if(!("init"in NREUM&&"page_view_timing"in NREUM.init&&"enabled"in NREUM.init.page_view_timing&&NREUM.init.page_view_timing.enabled===!1)){var s,f,u,d=t("handle"),p=t("loader"),l=t(25),h=NREUM.o.EV;if("PerformanceObserver"in window&&"function"==typeof window.PerformanceObserver){s=new PerformanceObserver(r);try{s.observe({entryTypes:["paint"]})}catch(m){}f=new PerformanceObserver(o);try{f.observe({entryTypes:["largest-contentful-paint"]})}catch(m){}u=new PerformanceObserver(i);try{u.observe({type:"layout-shift",buffered:!0})}catch(m){}}if("addEventListener"in document){var v=!1,w=["click","keydown","mousedown","pointerdown","touchstart"];w.forEach(function(t){document.addEventListener(t,a,!1)})}l(c)}},{}],23:[function(t,e,n){function r(){function t(){return e?15&e[n++]:16*Math.random()|0}var e=null,n=0,r=window.crypto||window.msCrypto;r&&r.getRandomValues&&(e=r.getRandomValues(new Uint8Array(31)));for(var o,i="xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx",a="",c=0;c<i.length;c++)o=i[c],"x"===o?a+=t().toString(16):"y"===o?(o=3&t()|8,a+=o.toString(16)):a+=o;return a}function o(){return a(16)}function i(){return a(32)}function a(t){function e(){return n?15&n[r++]:16*Math.random()|0}var n=null,r=0,o=window.crypto||window.msCrypto;o&&o.getRandomValues&&Uint8Array&&(n=o.getRandomValues(new Uint8Array(31)));for(var i=[],a=0;a<t;a++)i.push(e().toString(16));return i.join("")}e.exports={generateUuid:r,generateSpanId:o,generateTraceId:i}},{}],24:[function(t,e,n){function r(t,e){if(!o)return!1;if(t!==o)return!1;if(!e)return!0;if(!i)return!1;for(var n=i.split("."),r=e.split("."),a=0;a<r.length;a++)if(r[a]!==n[a])return!1;return!0}var o=null,i=null,a=/Version\/(\S+)\s+Safari/;if(navigator.userAgent){var c=navigator.userAgent,s=c.match(a);s&&c.indexOf("Chrome")===-1&&c.indexOf("Chromium")===-1&&(o="Safari",i=s[1])}e.exports={agent:o,version:i,match:r}},{}],25:[function(t,e,n){function r(t){function e(){t(a&&document[a]?document[a]:document[o]?"hidden":"visible")}"addEventListener"in document&&i&&document.addEventListener(i,e,!1)}e.exports=r;var o,i,a;"undefined"!=typeof document.hidden?(o="hidden",i="visibilitychange",a="visibilityState"):"undefined"!=typeof document.msHidden?(o="msHidden",i="msvisibilitychange"):"undefined"!=typeof document.webkitHidden&&(o="webkitHidden",i="webkitvisibilitychange",a="webkitVisibilityState")},{}],26:[function(t,e,n){function r(t,e){var n=[],r="",i=0;for(r in t)o.call(t,r)&&(n[i]=e(r,t[r]),i+=1);return n}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],27:[function(t,e,n){function r(t,e,n){e||(e=0),"undefined"==typeof n&&(n=t?t.length:0);for(var r=-1,o=n-e||0,i=Array(o<0?0:o);++r<o;)i[r]=t[e+r];return i}e.exports=r},{}],28:[function(t,e,n){e.exports={exists:"undefined"!=typeof window.performance&&window.performance.timing&&"undefined"!=typeof window.performance.timing.navigationStart}},{}],ee:[function(t,e,n){function r(){}function o(t){function e(t){return t&&t instanceof r?t:t?s(t,c,i):i()}function n(n,r,o,i){if(!p.aborted||i){t&&t(n,r,o);for(var a=e(o),c=m(n),s=c.length,f=0;f<s;f++)c[f].apply(a,r);var d=u[y[n]];return d&&d.push([x,n,r,a]),a}}function l(t,e){g[t]=m(t).concat(e)}function h(t,e){var n=g[t];if(n)for(var r=0;r<n.length;r++)n[r]===e&&n.splice(r,1)}function m(t){return g[t]||[]}function v(t){return d[t]=d[t]||o(n)}function w(t,e){f(t,function(t,n){e=e||"feature",y[n]=e,e in u||(u[e]=[])})}var g={},y={},x={on:l,addEventListener:l,removeEventListener:h,emit:n,get:v,listeners:m,context:e,buffer:w,abort:a,aborted:!1};return x}function i(){return new r}function a(){(u.api||u.feature)&&(p.aborted=!0,u=p.backlog={})}var c="nr@context",s=t("gos"),f=t(26),u={},d={},p=e.exports=o();p.backlog=u},{}],gos:[function(t,e,n){function r(t,e,n){if(o.call(t,e))return t[e];var r=n();if(Object.defineProperty&&Object.keys)try{return Object.defineProperty(t,e,{value:r,writable:!0,enumerable:!1}),r}catch(i){}return t[e]=r,r}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],handle:[function(t,e,n){function r(t,e,n,r){o.buffer([t],r),o.emit(t,e,n)}var o=t("ee").get("handle");e.exports=r,r.ee=o},{}],id:[function(t,e,n){function r(t){var e=typeof t;return!t||"object"!==e&&"function"!==e?-1:t===window?0:a(t,i,function(){return o++})}var o=1,i="nr@id",a=t("gos");e.exports=r},{}],loader:[function(t,e,n){function r(){if(!E++){var t=b.info=NREUM.info,e=l.getElementsByTagName("script")[0];if(setTimeout(u.abort,3e4),!(t&&t.licenseKey&&t.applicationID&&e))return u.abort();f(y,function(e,n){t[e]||(t[e]=n)});var n=a();s("mark",["onload",n+b.offset],null,"api"),s("timing",["load",n]);var r=l.createElement("script");r.src="https://"+t.agent,e.parentNode.insertBefore(r,e)}}function o(){"complete"===l.readyState&&i()}function i(){s("mark",["domContent",a()+b.offset],null,"api")}function a(){return R.exists&&performance.now?Math.round(performance.now()):(c=Math.max((new Date).getTime(),c))-b.offset}var c=(new Date).getTime(),s=t("handle"),f=t(26),u=t("ee"),d=t(24),p=window,l=p.document,h="addEventListener",m="attachEvent",v=p.XMLHttpRequest,w=v&&v.prototype;NREUM.o={ST:setTimeout,SI:p.setImmediate,CT:clearTimeout,XHR:v,REQ:p.Request,EV:p.Event,PR:p.Promise,MO:p.MutationObserver};var g=""+location,y={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",agent:"js-agent.newrelic.com/nr-spa-1177.min.js"},x=v&&w&&w[h]&&!/CriOS/.test(navigator.userAgent),b=e.exports={offset:c,now:a,origin:g,features:{},xhrWrappable:x,userAgent:d};t(19),t(22),l[h]?(l[h]("DOMContentLoaded",i,!1),p[h]("load",r,!1)):(l[m]("onreadystatechange",o),p[m]("onload",r)),s("mark",["firstbyte",c],null,"api");var E=0,R=t(28)},{}],"wrap-function":[function(t,e,n){function r(t){return!(t&&t instanceof Function&&t.apply&&!t[a])}var o=t("ee"),i=t(27),a="nr@original",c=Object.prototype.hasOwnProperty,s=!1;e.exports=function(t,e){function n(t,e,n,o){function nrWrapper(){var r,a,c,s;try{a=this,r=i(arguments),c="function"==typeof n?n(r,a):n||{}}catch(f){p([f,"",[r,a,o],c])}u(e+"start",[r,a,o],c);try{return s=t.apply(a,r)}catch(d){throw u(e+"err",[r,a,d],c),d}finally{u(e+"end",[r,a,s],c)}}return r(t)?t:(e||(e=""),nrWrapper[a]=t,d(t,nrWrapper),nrWrapper)}function f(t,e,o,i){o||(o="");var a,c,s,f="-"===o.charAt(0);for(s=0;s<e.length;s++)c=e[s],a=t[c],r(a)||(t[c]=n(a,f?c+o:o,i,c))}function u(n,r,o){if(!s||e){var i=s;s=!0;try{t.emit(n,r,o,e)}catch(a){p([a,n,r,o])}s=i}}function d(t,e){if(Object.defineProperty&&Object.keys)try{var n=Object.keys(t);return n.forEach(function(n){Object.defineProperty(e,n,{get:function(){return t[n]},set:function(e){return t[n]=e,e}})}),e}catch(r){p([r])}for(var o in t)c.call(t,o)&&(e[o]=t[o]);return e}function p(e){try{t.emit("internal-error",e)}catch(n){}}return t||(t=o),n.inPlace=f,n.flag=a,n}},{}]},{},["loader",2,16,5,3,4]);
    ;NREUM.loader_config={accountID:"2095297",trustKey:"2038175",agentID:"916957862",licenseKey:"16e2192192",applicationID:"916957862"}
    ;NREUM.info={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",licenseKey:"16e2192192",applicationID:"916957862",sa:1}
</script>
    <script src="https://cdn.cookielaw.org/scripttemplates/otSDKStub.js" type="text/javascript" charset="UTF-8" data-domain-script="dada9834-04e2-4bcd-b5ba-cae44d498dfa"></script>
    <script type="text/javascript">
        function getCookie(name) {
            var value = "; " + document.cookie;
            var parts = value.split("; " + name + "=");
            if (parts.length == 2) {
                return true;
            }
        }
            function OptanonWrapper() {
                var optCookie = getCookie("OptanonAlertBoxClosed");
                var isActiveCategory = "false";
                var isFoundCookieBox = "false";
                if(isActiveCategory == 'false' && isFoundCookieBox == 'false' && optCookie) {
                    OneTrust.InsertScript('/pb-assets/adobe/prod/adobedtm-1616606406233.js', 'head', null, null, '2');
                }
            }
    </script>


<noscript>
    <style>.requiresJS {
        display: none
    }

    .dartAd {
        display: block !important
    }</style>
</noscript>
        <script type="text/javascript" src="/wro/o4ny~selected-legacy.js"></script>
        <link rel="stylesheet" type="text/css" href="/wro/o4ny~selected-legacy.css">
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Open+Sans">
        <script>window.MSInputMethodContext && document.documentMode && document.write('<script src="https://cdn.jsdelivr.net/gh/nuxodin/ie11CustomProperties@4.1.0/ie11CustomProperties.min.js"><\x2fscript>');</script><link rel="stylesheet" href="/products/marlin/cell/releasedAssets/css/build-b26f3342b722959c7931.css"><link rel="stylesheet" href="/products/marlin/cell/releasedAssets/css/print-b26f3342b722959c7931.css" media="print"><link rel="stylesheet" href="/products/marlin/cell/releasedAssets/css/themes/theme-cell.css">







        
        
        <script class="optanon-category-4" type="text/plain" async="" src="https://www.youtube.com/iframe_api"></script>
        
    





<!---->
<meta http-equiv="X-UA-Compatible" content="IE=edge">












<script src="https://cdn.cookielaw.org/scripttemplates/202411.2.0/otBannerSdk.js" async="" type="text/javascript"></script><style id="onetrust-style">#onetrust-banner-sdk .onetrust-vendors-list-handler{cursor:pointer;color:#1f96db;font-size:inherit;font-weight:bold;text-decoration:none;margin-left:5px}#onetrust-banner-sdk .onetrust-vendors-list-handler:hover{color:#1f96db}#onetrust-banner-sdk:focus{outline:2px solid #000;outline-offset:-2px}#onetrust-banner-sdk a:focus{outline:2px solid #000}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{outline-offset:1px}#onetrust-banner-sdk.ot-bnr-w-logo .ot-bnr-logo{height:64px;width:64px}#onetrust-banner-sdk .ot-tcf2-vendor-count.ot-text-bold{font-weight:bold}#onetrust-banner-sdk .ot-close-icon,#onetrust-pc-sdk .ot-close-icon,#ot-sync-ntfy .ot-close-icon{background-size:contain;background-repeat:no-repeat;background-position:center;height:12px;width:12px}#onetrust-banner-sdk .powered-by-logo,#onetrust-banner-sdk .ot-pc-footer-logo a,#onetrust-pc-sdk .powered-by-logo,#onetrust-pc-sdk .ot-pc-footer-logo a,#ot-sync-ntfy .powered-by-logo,#ot-sync-ntfy .ot-pc-footer-logo a{background-size:contain;background-repeat:no-repeat;background-position:center;height:25px;width:152px;display:block;text-decoration:none;font-size:.75em}#onetrust-banner-sdk .powered-by-logo:hover,#onetrust-banner-sdk .ot-pc-footer-logo a:hover,#onetrust-pc-sdk .powered-by-logo:hover,#onetrust-pc-sdk .ot-pc-footer-logo a:hover,#ot-sync-ntfy .powered-by-logo:hover,#ot-sync-ntfy .ot-pc-footer-logo a:hover{color:#565656}#onetrust-banner-sdk h3 *,#onetrust-banner-sdk h4 *,#onetrust-banner-sdk h6 *,#onetrust-banner-sdk button *,#onetrust-banner-sdk a[data-parent-id] *,#onetrust-pc-sdk h3 *,#onetrust-pc-sdk h4 *,#onetrust-pc-sdk h6 *,#onetrust-pc-sdk button *,#onetrust-pc-sdk a[data-parent-id] *,#ot-sync-ntfy h3 *,#ot-sync-ntfy h4 *,#ot-sync-ntfy h6 *,#ot-sync-ntfy button *,#ot-sync-ntfy a[data-parent-id] *{font-size:inherit;font-weight:inherit;color:inherit}#onetrust-banner-sdk .ot-hide,#onetrust-pc-sdk .ot-hide,#ot-sync-ntfy .ot-hide{display:none !important}#onetrust-banner-sdk button.ot-link-btn:hover,#onetrust-pc-sdk button.ot-link-btn:hover,#ot-sync-ntfy button.ot-link-btn:hover{text-decoration:underline;opacity:1}#onetrust-pc-sdk .ot-sdk-row .ot-sdk-column{padding:0}#onetrust-pc-sdk .ot-sdk-container{padding-right:0}#onetrust-pc-sdk .ot-sdk-row{flex-direction:initial;width:100%}#onetrust-pc-sdk [type=checkbox]:checked,#onetrust-pc-sdk [type=checkbox]:not(:checked){pointer-events:initial}#onetrust-pc-sdk [type=checkbox]:disabled+label::before,#onetrust-pc-sdk [type=checkbox]:disabled+label:after,#onetrust-pc-sdk [type=checkbox]:disabled+label{pointer-events:none;opacity:.8}#onetrust-pc-sdk #vendor-list-content{transform:translate3d(0, 0, 0)}#onetrust-pc-sdk li input[type=checkbox]{z-index:1}#onetrust-pc-sdk li .ot-checkbox label{z-index:2}#onetrust-pc-sdk li .ot-checkbox input[type=checkbox]{height:auto;width:auto}#onetrust-pc-sdk li .host-title a,#onetrust-pc-sdk li .ot-host-name a,#onetrust-pc-sdk li .accordion-text,#onetrust-pc-sdk li .ot-acc-txt{z-index:2;position:relative}#onetrust-pc-sdk input{margin:3px .1ex}#onetrust-pc-sdk .pc-logo,#onetrust-pc-sdk .ot-pc-logo{height:60px;width:180px;background-position:center;background-size:contain;background-repeat:no-repeat;display:inline-flex;justify-content:center;align-items:center}#onetrust-pc-sdk .pc-logo img,#onetrust-pc-sdk .ot-pc-logo img{max-height:100%;max-width:100%}#onetrust-pc-sdk .screen-reader-only,#onetrust-pc-sdk .ot-scrn-rdr,.ot-sdk-cookie-policy .screen-reader-only,.ot-sdk-cookie-policy .ot-scrn-rdr{border:0;clip:rect(0 0 0 0);height:1px;margin:-1px;overflow:hidden;padding:0;position:absolute;width:1px}#onetrust-pc-sdk.ot-fade-in,.onetrust-pc-dark-filter.ot-fade-in,#onetrust-banner-sdk.ot-fade-in{animation-name:onetrust-fade-in;animation-duration:400ms;animation-timing-function:ease-in-out}#onetrust-pc-sdk.ot-hide{display:none !important}.onetrust-pc-dark-filter.ot-hide{display:none !important}#ot-sdk-btn.ot-sdk-show-settings,#ot-sdk-btn.optanon-show-settings{color:#fff;background-color:#468254;height:auto;white-space:normal;word-wrap:break-word;padding:.8em 2em;font-size:.8em;line-height:1.2;cursor:pointer;-moz-transition:.1s ease;-o-transition:.1s ease;-webkit-transition:1s ease;transition:.1s ease}#ot-sdk-btn.ot-sdk-show-settings:hover,#ot-sdk-btn.optanon-show-settings:hover{color:#fff;background-color:#2c6415}#ot-sdk-btn.ot-sdk-show-settings:active,#ot-sdk-btn.optanon-show-settings:active{color:#fff;background-color:#2c6415;border:1px solid rgba(162,192,169,.5)}.onetrust-pc-dark-filter{background:rgba(0,0,0,.5);z-index:2147483646;width:100%;height:100%;overflow:hidden;position:fixed;top:0;bottom:0;left:0}@keyframes onetrust-fade-in{0%{opacity:0}100%{opacity:1}}.ot-cookie-label{text-decoration:underline}@media only screen and (min-width: 426px)and (max-width: 896px)and (orientation: landscape){#onetrust-pc-sdk p{font-size:.75em}}#onetrust-banner-sdk .banner-option-input:focus+label{outline:1px solid #000;outline-style:auto}.category-vendors-list-handler+a:focus,.category-vendors-list-handler+a:focus-visible{outline:2px solid #000}#onetrust-pc-sdk .ot-userid-title{margin-top:10px}#onetrust-pc-sdk .ot-userid-title>span,#onetrust-pc-sdk .ot-userid-timestamp>span{font-weight:700}#onetrust-pc-sdk .ot-userid-desc{font-style:italic}#onetrust-pc-sdk .ot-host-desc a{pointer-events:initial}#onetrust-pc-sdk .ot-ven-hdr>p a{position:relative;z-index:2;pointer-events:initial}#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info a,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info a{margin-right:auto}#onetrust-pc-sdk .ot-pc-footer-logo img{width:136px;height:16px}#onetrust-pc-sdk .ot-pur-vdr-count{font-weight:400;font-size:.7rem;padding-top:3px;display:block}#onetrust-banner-sdk .ot-optout-signal,#onetrust-pc-sdk .ot-optout-signal{border:1px solid #32ae88;border-radius:3px;padding:5px;margin-bottom:10px;background-color:#f9fffa;font-size:.85rem;line-height:2}#onetrust-banner-sdk .ot-optout-signal .ot-optout-icon,#onetrust-pc-sdk .ot-optout-signal .ot-optout-icon{display:inline;margin-right:5px}#onetrust-banner-sdk .ot-optout-signal svg,#onetrust-pc-sdk .ot-optout-signal svg{height:20px;width:30px;transform:scale(0.5)}#onetrust-banner-sdk .ot-optout-signal svg path,#onetrust-pc-sdk .ot-optout-signal svg path{fill:#32ae88}#onetrust-consent-sdk .ot-general-modal{overflow:hidden;position:fixed;margin:0 auto;top:50%;left:50%;width:40%;padding:1.5rem;max-width:575px;min-width:575px;z-index:2147483647;border-radius:2.5px;transform:translate(-50%, -50%)}#onetrust-consent-sdk .ot-signature-health-group{margin-top:1rem;padding-left:1.25rem;padding-right:1.25rem;margin-bottom:.625rem;width:calc(100% - 2.5rem)}#onetrust-consent-sdk .ot-signature-health-group .ot-signature-health-form{gap:.5rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-health-form{width:70%;gap:.35rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-input{height:38px;padding:6px 10px;background-color:#fff;border:1px solid #d1d1d1;border-radius:4px;box-shadow:none;box-sizing:border-box}#onetrust-consent-sdk .ot-signature-health .ot-signature-subtitle{font-size:1.125rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-group-title{font-size:1.25rem;font-weight:bold}#onetrust-consent-sdk .ot-signature-health,#onetrust-consent-sdk .ot-signature-health-group{display:flex;flex-direction:column;gap:1rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-cont,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-cont{display:flex;flex-direction:column;gap:.25rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-paragraph,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-paragraph{margin:0;line-height:20px;font-size:max(14px,.875rem)}#onetrust-consent-sdk .ot-signature-health .ot-health-signature-error,#onetrust-consent-sdk .ot-signature-health-group .ot-health-signature-error{color:#4d4d4d;font-size:min(12px,.75rem)}#onetrust-consent-sdk .ot-signature-health .ot-signature-buttons-cont,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-buttons-cont{margin-top:max(.75rem,2%);gap:1rem;display:flex;justify-content:flex-end}#onetrust-consent-sdk .ot-signature-health .ot-signature-button,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-button{flex:1;height:auto;color:#fff;cursor:pointer;line-height:1.2;min-width:125px;font-weight:600;font-size:.813em;border-radius:2px;padding:12px 10px;white-space:normal;word-wrap:break-word;word-break:break-word;background-color:#68b631;border:2px solid #68b631}#onetrust-consent-sdk .ot-signature-health .ot-signature-button.reject,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-button.reject{background-color:#fff}#onetrust-consent-sdk .ot-input-field-cont{display:flex;flex-direction:column;gap:.5rem}#onetrust-consent-sdk .ot-input-field-cont .ot-signature-input{width:65%}#onetrust-consent-sdk .ot-signature-health-form{display:flex;flex-direction:column}#onetrust-consent-sdk .ot-signature-health-form .ot-signature-label{margin-bottom:0;line-height:20px;font-size:max(14px,.875rem)}@media only screen and (max-width: 600px){#onetrust-consent-sdk .ot-general-modal{min-width:100%}#onetrust-consent-sdk .ot-signature-health .ot-signature-health-form{width:100%}#onetrust-consent-sdk .ot-input-field-cont .ot-signature-input{width:100%}}#onetrust-banner-sdk,#onetrust-pc-sdk,#ot-sdk-cookie-policy,#ot-sync-ntfy{font-size:16px}#onetrust-banner-sdk *,#onetrust-banner-sdk ::after,#onetrust-banner-sdk ::before,#onetrust-pc-sdk *,#onetrust-pc-sdk ::after,#onetrust-pc-sdk ::before,#ot-sdk-cookie-policy *,#ot-sdk-cookie-policy ::after,#ot-sdk-cookie-policy ::before,#ot-sync-ntfy *,#ot-sync-ntfy ::after,#ot-sync-ntfy ::before{-webkit-box-sizing:content-box;-moz-box-sizing:content-box;box-sizing:content-box}#onetrust-banner-sdk div,#onetrust-banner-sdk span,#onetrust-banner-sdk h1,#onetrust-banner-sdk h2,#onetrust-banner-sdk h3,#onetrust-banner-sdk h4,#onetrust-banner-sdk h5,#onetrust-banner-sdk h6,#onetrust-banner-sdk p,#onetrust-banner-sdk img,#onetrust-banner-sdk svg,#onetrust-banner-sdk button,#onetrust-banner-sdk section,#onetrust-banner-sdk a,#onetrust-banner-sdk label,#onetrust-banner-sdk input,#onetrust-banner-sdk ul,#onetrust-banner-sdk li,#onetrust-banner-sdk nav,#onetrust-banner-sdk table,#onetrust-banner-sdk thead,#onetrust-banner-sdk tr,#onetrust-banner-sdk td,#onetrust-banner-sdk tbody,#onetrust-banner-sdk .ot-main-content,#onetrust-banner-sdk .ot-toggle,#onetrust-banner-sdk #ot-content,#onetrust-banner-sdk #ot-pc-content,#onetrust-banner-sdk .checkbox,#onetrust-pc-sdk div,#onetrust-pc-sdk span,#onetrust-pc-sdk h1,#onetrust-pc-sdk h2,#onetrust-pc-sdk h3,#onetrust-pc-sdk h4,#onetrust-pc-sdk h5,#onetrust-pc-sdk h6,#onetrust-pc-sdk p,#onetrust-pc-sdk img,#onetrust-pc-sdk svg,#onetrust-pc-sdk button,#onetrust-pc-sdk section,#onetrust-pc-sdk a,#onetrust-pc-sdk label,#onetrust-pc-sdk input,#onetrust-pc-sdk ul,#onetrust-pc-sdk li,#onetrust-pc-sdk nav,#onetrust-pc-sdk table,#onetrust-pc-sdk thead,#onetrust-pc-sdk tr,#onetrust-pc-sdk td,#onetrust-pc-sdk tbody,#onetrust-pc-sdk .ot-main-content,#onetrust-pc-sdk .ot-toggle,#onetrust-pc-sdk #ot-content,#onetrust-pc-sdk #ot-pc-content,#onetrust-pc-sdk .checkbox,#ot-sdk-cookie-policy div,#ot-sdk-cookie-policy span,#ot-sdk-cookie-policy h1,#ot-sdk-cookie-policy h2,#ot-sdk-cookie-policy h3,#ot-sdk-cookie-policy h4,#ot-sdk-cookie-policy h5,#ot-sdk-cookie-policy h6,#ot-sdk-cookie-policy p,#ot-sdk-cookie-policy img,#ot-sdk-cookie-policy svg,#ot-sdk-cookie-policy button,#ot-sdk-cookie-policy section,#ot-sdk-cookie-policy a,#ot-sdk-cookie-policy label,#ot-sdk-cookie-policy input,#ot-sdk-cookie-policy ul,#ot-sdk-cookie-policy li,#ot-sdk-cookie-policy nav,#ot-sdk-cookie-policy table,#ot-sdk-cookie-policy thead,#ot-sdk-cookie-policy tr,#ot-sdk-cookie-policy td,#ot-sdk-cookie-policy tbody,#ot-sdk-cookie-policy .ot-main-content,#ot-sdk-cookie-policy .ot-toggle,#ot-sdk-cookie-policy #ot-content,#ot-sdk-cookie-policy #ot-pc-content,#ot-sdk-cookie-policy .checkbox,#ot-sync-ntfy div,#ot-sync-ntfy span,#ot-sync-ntfy h1,#ot-sync-ntfy h2,#ot-sync-ntfy h3,#ot-sync-ntfy h4,#ot-sync-ntfy h5,#ot-sync-ntfy h6,#ot-sync-ntfy p,#ot-sync-ntfy img,#ot-sync-ntfy svg,#ot-sync-ntfy button,#ot-sync-ntfy section,#ot-sync-ntfy a,#ot-sync-ntfy label,#ot-sync-ntfy input,#ot-sync-ntfy ul,#ot-sync-ntfy li,#ot-sync-ntfy nav,#ot-sync-ntfy table,#ot-sync-ntfy thead,#ot-sync-ntfy tr,#ot-sync-ntfy td,#ot-sync-ntfy tbody,#ot-sync-ntfy .ot-main-content,#ot-sync-ntfy .ot-toggle,#ot-sync-ntfy #ot-content,#ot-sync-ntfy #ot-pc-content,#ot-sync-ntfy .checkbox{font-family:inherit;font-weight:normal;-webkit-font-smoothing:auto;letter-spacing:normal;line-height:normal;padding:0;margin:0;height:auto;min-height:0;max-height:none;width:auto;min-width:0;max-width:none;border-radius:0;border:none;clear:none;float:none;position:static;bottom:auto;left:auto;right:auto;top:auto;text-align:left;text-decoration:none;text-indent:0;text-shadow:none;text-transform:none;white-space:normal;background:none;overflow:visible;vertical-align:baseline;visibility:visible;z-index:auto;box-shadow:none}#onetrust-banner-sdk label:before,#onetrust-banner-sdk label:after,#onetrust-banner-sdk .checkbox:after,#onetrust-banner-sdk .checkbox:before,#onetrust-pc-sdk label:before,#onetrust-pc-sdk label:after,#onetrust-pc-sdk .checkbox:after,#onetrust-pc-sdk .checkbox:before,#ot-sdk-cookie-policy label:before,#ot-sdk-cookie-policy label:after,#ot-sdk-cookie-policy .checkbox:after,#ot-sdk-cookie-policy .checkbox:before,#ot-sync-ntfy label:before,#ot-sync-ntfy label:after,#ot-sync-ntfy .checkbox:after,#ot-sync-ntfy .checkbox:before{content:"";content:none}#onetrust-banner-sdk .ot-sdk-container,#onetrust-pc-sdk .ot-sdk-container,#ot-sdk-cookie-policy .ot-sdk-container{position:relative;width:100%;max-width:100%;margin:0 auto;padding:0 20px;box-sizing:border-box}#onetrust-banner-sdk .ot-sdk-column,#onetrust-banner-sdk .ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-column,#onetrust-pc-sdk .ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-column,#ot-sdk-cookie-policy .ot-sdk-columns{width:100%;float:left;box-sizing:border-box;padding:0;display:initial}@media(min-width: 400px){#onetrust-banner-sdk .ot-sdk-container,#onetrust-pc-sdk .ot-sdk-container,#ot-sdk-cookie-policy .ot-sdk-container{width:90%;padding:0}}@media(min-width: 550px){#onetrust-banner-sdk .ot-sdk-container,#onetrust-pc-sdk .ot-sdk-container,#ot-sdk-cookie-policy .ot-sdk-container{width:100%}#onetrust-banner-sdk .ot-sdk-column,#onetrust-banner-sdk .ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-column,#onetrust-pc-sdk .ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-column,#ot-sdk-cookie-policy .ot-sdk-columns{margin-left:4%}#onetrust-banner-sdk .ot-sdk-column:first-child,#onetrust-banner-sdk .ot-sdk-columns:first-child,#onetrust-pc-sdk .ot-sdk-column:first-child,#onetrust-pc-sdk .ot-sdk-columns:first-child,#ot-sdk-cookie-policy .ot-sdk-column:first-child,#ot-sdk-cookie-policy .ot-sdk-columns:first-child{margin-left:0}#onetrust-banner-sdk .ot-sdk-two.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-two.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-two.ot-sdk-columns{width:13.3333333333%}#onetrust-banner-sdk .ot-sdk-three.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-three.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-three.ot-sdk-columns{width:22%}#onetrust-banner-sdk .ot-sdk-four.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-four.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-four.ot-sdk-columns{width:30.6666666667%}#onetrust-banner-sdk .ot-sdk-eight.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-eight.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-eight.ot-sdk-columns{width:65.3333333333%}#onetrust-banner-sdk .ot-sdk-nine.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-nine.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-nine.ot-sdk-columns{width:74%}#onetrust-banner-sdk .ot-sdk-ten.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-ten.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-ten.ot-sdk-columns{width:82.6666666667%}#onetrust-banner-sdk .ot-sdk-eleven.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-eleven.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-eleven.ot-sdk-columns{width:91.3333333333%}#onetrust-banner-sdk .ot-sdk-twelve.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-twelve.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-twelve.ot-sdk-columns{width:100%;margin-left:0}}#onetrust-banner-sdk h1,#onetrust-banner-sdk h2,#onetrust-banner-sdk h3,#onetrust-banner-sdk h4,#onetrust-banner-sdk h5,#onetrust-banner-sdk h6,#onetrust-pc-sdk h1,#onetrust-pc-sdk h2,#onetrust-pc-sdk h3,#onetrust-pc-sdk h4,#onetrust-pc-sdk h5,#onetrust-pc-sdk h6,#ot-sdk-cookie-policy h1,#ot-sdk-cookie-policy h2,#ot-sdk-cookie-policy h3,#ot-sdk-cookie-policy h4,#ot-sdk-cookie-policy h5,#ot-sdk-cookie-policy h6{margin-top:0;font-weight:600;font-family:inherit}#onetrust-banner-sdk h1,#onetrust-pc-sdk h1,#ot-sdk-cookie-policy h1{font-size:1.5rem;line-height:1.2}#onetrust-banner-sdk h2,#onetrust-pc-sdk h2,#ot-sdk-cookie-policy h2{font-size:1.5rem;line-height:1.25}#onetrust-banner-sdk h3,#onetrust-pc-sdk h3,#ot-sdk-cookie-policy h3{font-size:1.5rem;line-height:1.3}#onetrust-banner-sdk h4,#onetrust-pc-sdk h4,#ot-sdk-cookie-policy h4{font-size:1.5rem;line-height:1.35}#onetrust-banner-sdk h5,#onetrust-pc-sdk h5,#ot-sdk-cookie-policy h5{font-size:1.5rem;line-height:1.5}#onetrust-banner-sdk h6,#onetrust-pc-sdk h6,#ot-sdk-cookie-policy h6{font-size:1.5rem;line-height:1.6}@media(min-width: 550px){#onetrust-banner-sdk h1,#onetrust-pc-sdk h1,#ot-sdk-cookie-policy h1{font-size:1.5rem}#onetrust-banner-sdk h2,#onetrust-pc-sdk h2,#ot-sdk-cookie-policy h2{font-size:1.5rem}#onetrust-banner-sdk h3,#onetrust-pc-sdk h3,#ot-sdk-cookie-policy h3{font-size:1.5rem}#onetrust-banner-sdk h4,#onetrust-pc-sdk h4,#ot-sdk-cookie-policy h4{font-size:1.5rem}#onetrust-banner-sdk h5,#onetrust-pc-sdk h5,#ot-sdk-cookie-policy h5{font-size:1.5rem}#onetrust-banner-sdk h6,#onetrust-pc-sdk h6,#ot-sdk-cookie-policy h6{font-size:1.5rem}}#onetrust-banner-sdk p,#onetrust-pc-sdk p,#ot-sdk-cookie-policy p{margin:0 0 1em 0;font-family:inherit;line-height:normal}#onetrust-banner-sdk a,#onetrust-pc-sdk a,#ot-sdk-cookie-policy a{color:#565656;text-decoration:underline}#onetrust-banner-sdk a:hover,#onetrust-pc-sdk a:hover,#ot-sdk-cookie-policy a:hover{color:#565656;text-decoration:none}#onetrust-banner-sdk .ot-sdk-button,#onetrust-banner-sdk button,#onetrust-pc-sdk .ot-sdk-button,#onetrust-pc-sdk button,#ot-sdk-cookie-policy .ot-sdk-button,#ot-sdk-cookie-policy button{margin-bottom:1rem;font-family:inherit}#onetrust-banner-sdk .ot-sdk-button,#onetrust-banner-sdk button,#onetrust-pc-sdk .ot-sdk-button,#onetrust-pc-sdk button,#ot-sdk-cookie-policy .ot-sdk-button,#ot-sdk-cookie-policy button{display:inline-block;height:38px;padding:0 30px;color:#555;text-align:center;font-size:.9em;font-weight:400;line-height:38px;letter-spacing:.01em;text-decoration:none;white-space:nowrap;background-color:rgba(0,0,0,0);border-radius:2px;border:1px solid #bbb;cursor:pointer;box-sizing:border-box}#onetrust-banner-sdk .ot-sdk-button:hover,#onetrust-banner-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):hover,#onetrust-banner-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):focus,#onetrust-pc-sdk .ot-sdk-button:hover,#onetrust-pc-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):hover,#onetrust-pc-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):focus,#ot-sdk-cookie-policy .ot-sdk-button:hover,#ot-sdk-cookie-policy :not(.ot-leg-btn-container)>button:not(.ot-link-btn):hover,#ot-sdk-cookie-policy :not(.ot-leg-btn-container)>button:not(.ot-link-btn):focus{color:#333;border-color:#888;opacity:.9}#onetrust-banner-sdk .ot-sdk-button:focus,#onetrust-banner-sdk :not(.ot-leg-btn-container)>button:focus,#onetrust-pc-sdk .ot-sdk-button:focus,#onetrust-pc-sdk :not(.ot-leg-btn-container)>button:focus,#ot-sdk-cookie-policy .ot-sdk-button:focus,#ot-sdk-cookie-policy :not(.ot-leg-btn-container)>button:focus{outline:2px solid #000}#onetrust-banner-sdk .ot-sdk-button.ot-sdk-button-primary,#onetrust-banner-sdk button.ot-sdk-button-primary,#onetrust-banner-sdk input[type=submit].ot-sdk-button-primary,#onetrust-banner-sdk input[type=reset].ot-sdk-button-primary,#onetrust-banner-sdk input[type=button].ot-sdk-button-primary,#onetrust-pc-sdk .ot-sdk-button.ot-sdk-button-primary,#onetrust-pc-sdk button.ot-sdk-button-primary,#onetrust-pc-sdk input[type=submit].ot-sdk-button-primary,#onetrust-pc-sdk input[type=reset].ot-sdk-button-primary,#onetrust-pc-sdk input[type=button].ot-sdk-button-primary,#ot-sdk-cookie-policy .ot-sdk-button.ot-sdk-button-primary,#ot-sdk-cookie-policy button.ot-sdk-button-primary,#ot-sdk-cookie-policy input[type=submit].ot-sdk-button-primary,#ot-sdk-cookie-policy input[type=reset].ot-sdk-button-primary,#ot-sdk-cookie-policy input[type=button].ot-sdk-button-primary{color:#fff;background-color:#33c3f0;border-color:#33c3f0}#onetrust-banner-sdk .ot-sdk-button.ot-sdk-button-primary:hover,#onetrust-banner-sdk button.ot-sdk-button-primary:hover,#onetrust-banner-sdk input[type=submit].ot-sdk-button-primary:hover,#onetrust-banner-sdk input[type=reset].ot-sdk-button-primary:hover,#onetrust-banner-sdk input[type=button].ot-sdk-button-primary:hover,#onetrust-banner-sdk .ot-sdk-button.ot-sdk-button-primary:focus,#onetrust-banner-sdk button.ot-sdk-button-primary:focus,#onetrust-banner-sdk input[type=submit].ot-sdk-button-primary:focus,#onetrust-banner-sdk input[type=reset].ot-sdk-button-primary:focus,#onetrust-banner-sdk input[type=button].ot-sdk-button-primary:focus,#onetrust-pc-sdk .ot-sdk-button.ot-sdk-button-primary:hover,#onetrust-pc-sdk button.ot-sdk-button-primary:hover,#onetrust-pc-sdk input[type=submit].ot-sdk-button-primary:hover,#onetrust-pc-sdk input[type=reset].ot-sdk-button-primary:hover,#onetrust-pc-sdk input[type=button].ot-sdk-button-primary:hover,#onetrust-pc-sdk .ot-sdk-button.ot-sdk-button-primary:focus,#onetrust-pc-sdk button.ot-sdk-button-primary:focus,#onetrust-pc-sdk input[type=submit].ot-sdk-button-primary:focus,#onetrust-pc-sdk input[type=reset].ot-sdk-button-primary:focus,#onetrust-pc-sdk input[type=button].ot-sdk-button-primary:focus,#ot-sdk-cookie-policy .ot-sdk-button.ot-sdk-button-primary:hover,#ot-sdk-cookie-policy button.ot-sdk-button-primary:hover,#ot-sdk-cookie-policy input[type=submit].ot-sdk-button-primary:hover,#ot-sdk-cookie-policy input[type=reset].ot-sdk-button-primary:hover,#ot-sdk-cookie-policy input[type=button].ot-sdk-button-primary:hover,#ot-sdk-cookie-policy .ot-sdk-button.ot-sdk-button-primary:focus,#ot-sdk-cookie-policy button.ot-sdk-button-primary:focus,#ot-sdk-cookie-policy input[type=submit].ot-sdk-button-primary:focus,#ot-sdk-cookie-policy input[type=reset].ot-sdk-button-primary:focus,#ot-sdk-cookie-policy input[type=button].ot-sdk-button-primary:focus{color:#fff;background-color:#1eaedb;border-color:#1eaedb}#onetrust-banner-sdk input[type=text],#onetrust-pc-sdk input[type=text],#ot-sdk-cookie-policy input[type=text]{height:38px;padding:6px 10px;background-color:#fff;border:1px solid #d1d1d1;border-radius:4px;box-shadow:none;box-sizing:border-box}#onetrust-banner-sdk input[type=text],#onetrust-pc-sdk input[type=text],#ot-sdk-cookie-policy input[type=text]{-webkit-appearance:none;-moz-appearance:none;appearance:none}#onetrust-banner-sdk input[type=text]:focus,#onetrust-pc-sdk input[type=text]:focus,#ot-sdk-cookie-policy input[type=text]:focus{border:1px solid #000;outline:0}#onetrust-banner-sdk label,#onetrust-pc-sdk label,#ot-sdk-cookie-policy label{display:block;margin-bottom:.5rem;font-weight:600}#onetrust-banner-sdk input[type=checkbox],#onetrust-pc-sdk input[type=checkbox],#ot-sdk-cookie-policy input[type=checkbox]{display:inline}#onetrust-banner-sdk ul,#onetrust-pc-sdk ul,#ot-sdk-cookie-policy ul{list-style:circle inside}#onetrust-banner-sdk ul,#onetrust-pc-sdk ul,#ot-sdk-cookie-policy ul{padding-left:0;margin-top:0}#onetrust-banner-sdk ul ul,#onetrust-pc-sdk ul ul,#ot-sdk-cookie-policy ul ul{margin:1.5rem 0 1.5rem 3rem;font-size:90%}#onetrust-banner-sdk li,#onetrust-pc-sdk li,#ot-sdk-cookie-policy li{margin-bottom:1rem}#onetrust-banner-sdk th,#onetrust-banner-sdk td,#onetrust-pc-sdk th,#onetrust-pc-sdk td,#ot-sdk-cookie-policy th,#ot-sdk-cookie-policy td{padding:12px 15px;text-align:left;border-bottom:1px solid #e1e1e1}#onetrust-banner-sdk button,#onetrust-pc-sdk button,#ot-sdk-cookie-policy button{margin-bottom:1rem;font-family:inherit}#onetrust-banner-sdk .ot-sdk-container:after,#onetrust-banner-sdk .ot-sdk-row:after,#onetrust-pc-sdk .ot-sdk-container:after,#onetrust-pc-sdk .ot-sdk-row:after,#ot-sdk-cookie-policy .ot-sdk-container:after,#ot-sdk-cookie-policy .ot-sdk-row:after{content:"";display:table;clear:both}#onetrust-banner-sdk .ot-sdk-row,#onetrust-pc-sdk .ot-sdk-row,#ot-sdk-cookie-policy .ot-sdk-row{margin:0;max-width:none;display:block}#onetrust-banner-sdk{box-shadow:0 0 18px rgba(0,0,0,.2)}#onetrust-banner-sdk.otFlat{position:fixed;z-index:2147483645;bottom:0;right:0;left:0;background-color:#fff;max-height:90%;overflow-x:hidden;overflow-y:auto}#onetrust-banner-sdk.otFlat.top{top:0px;bottom:auto}#onetrust-banner-sdk.otRelFont{font-size:1rem}#onetrust-banner-sdk>.ot-sdk-container{overflow:hidden}#onetrust-banner-sdk::-webkit-scrollbar{width:11px}#onetrust-banner-sdk::-webkit-scrollbar-thumb{border-radius:10px;background:#c1c1c1}#onetrust-banner-sdk{scrollbar-arrow-color:#c1c1c1;scrollbar-darkshadow-color:#c1c1c1;scrollbar-face-color:#c1c1c1;scrollbar-shadow-color:#c1c1c1}#onetrust-banner-sdk #onetrust-policy{margin:1.25em 0 .625em 2em;overflow:hidden}#onetrust-banner-sdk #onetrust-policy .ot-gv-list-handler{float:left;font-size:.82em;padding:0;margin-bottom:0;border:0;line-height:normal;height:auto;width:auto}#onetrust-banner-sdk #onetrust-policy-title{font-size:1.2em;line-height:1.3;margin-bottom:10px}#onetrust-banner-sdk #onetrust-policy-text{clear:both;text-align:left;font-size:.88em;line-height:1.4}#onetrust-banner-sdk #onetrust-policy-text *{font-size:inherit;line-height:inherit}#onetrust-banner-sdk #onetrust-policy-text a{font-weight:bold;margin-left:5px}#onetrust-banner-sdk #onetrust-policy-title,#onetrust-banner-sdk #onetrust-policy-text{color:dimgray;float:left}#onetrust-banner-sdk #onetrust-button-group-parent{min-height:1px;text-align:center}#onetrust-banner-sdk #onetrust-button-group{display:inline-block}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{background-color:#68b631;color:#fff;border-color:#68b631;margin-right:1em;min-width:125px;height:auto;white-space:normal;word-break:break-word;word-wrap:break-word;padding:12px 10px;line-height:1.2;font-size:.813em;font-weight:600}#onetrust-banner-sdk #onetrust-pc-btn-handler.cookie-setting-link{background-color:#fff;border:none;color:#68b631;text-decoration:underline;padding-left:0;padding-right:0}#onetrust-banner-sdk .onetrust-close-btn-ui{width:44px;height:44px;background-size:12px;border:none;position:relative;margin:auto;padding:0}#onetrust-banner-sdk .banner_logo{display:none}#onetrust-banner-sdk.ot-bnr-w-logo .ot-bnr-logo{position:absolute;top:50%;transform:translateY(-50%);left:0px}#onetrust-banner-sdk.ot-bnr-w-logo #onetrust-policy{margin-left:65px}#onetrust-banner-sdk .ot-b-addl-desc{clear:both;float:left;display:block}#onetrust-banner-sdk #banner-options{float:left;display:table;margin-right:0;margin-left:1em;width:calc(100% - 1em)}#onetrust-banner-sdk .banner-option-input{cursor:pointer;width:auto;height:auto;border:none;padding:0;padding-right:3px;margin:0 0 10px;font-size:.82em;line-height:1.4}#onetrust-banner-sdk .banner-option-input *{pointer-events:none;font-size:inherit;line-height:inherit}#onetrust-banner-sdk .banner-option-input[aria-expanded=true]~.banner-option-details{display:block;height:auto}#onetrust-banner-sdk .banner-option-input[aria-expanded=true] .ot-arrow-container{transform:rotate(90deg)}#onetrust-banner-sdk .banner-option{margin-bottom:12px;margin-left:0;border:none;float:left;padding:0}#onetrust-banner-sdk .banner-option:first-child{padding-left:2px}#onetrust-banner-sdk .banner-option:not(:first-child){padding:0;border:none}#onetrust-banner-sdk .banner-option-header{cursor:pointer;display:inline-block}#onetrust-banner-sdk .banner-option-header :first-child{color:dimgray;font-weight:bold;float:left}#onetrust-banner-sdk .banner-option-header .ot-arrow-container{display:inline-block;border-top:6px solid rgba(0,0,0,0);border-bottom:6px solid rgba(0,0,0,0);border-left:6px solid dimgray;margin-left:10px;vertical-align:middle}#onetrust-banner-sdk .banner-option-details{display:none;font-size:.83em;line-height:1.5;padding:10px 0px 5px 10px;margin-right:10px;height:0px}#onetrust-banner-sdk .banner-option-details *{font-size:inherit;line-height:inherit;color:dimgray}#onetrust-banner-sdk .ot-arrow-container,#onetrust-banner-sdk .banner-option-details{transition:all 300ms ease-in 0s;-webkit-transition:all 300ms ease-in 0s;-moz-transition:all 300ms ease-in 0s;-o-transition:all 300ms ease-in 0s}#onetrust-banner-sdk .ot-dpd-container{float:left}#onetrust-banner-sdk .ot-dpd-title{margin-bottom:10px}#onetrust-banner-sdk .ot-dpd-title,#onetrust-banner-sdk .ot-dpd-desc{font-size:.88em;line-height:1.4;color:dimgray}#onetrust-banner-sdk .ot-dpd-title *,#onetrust-banner-sdk .ot-dpd-desc *{font-size:inherit;line-height:inherit}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-text *{margin-bottom:0}#onetrust-banner-sdk.ot-iab-2 .onetrust-vendors-list-handler{display:block;margin-left:0;margin-top:5px;clear:both;margin-bottom:0;padding:0;border:0;height:auto;width:auto}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group button{display:block}#onetrust-banner-sdk.ot-close-btn-link{padding-top:25px}#onetrust-banner-sdk.ot-close-btn-link #onetrust-close-btn-container{top:15px;transform:none;right:15px}#onetrust-banner-sdk.ot-close-btn-link #onetrust-close-btn-container button{padding:0;white-space:pre-wrap;border:none;height:auto;line-height:1.5;text-decoration:underline;font-size:.69em}#onetrust-banner-sdk #onetrust-policy-text,#onetrust-banner-sdk .ot-dpd-desc,#onetrust-banner-sdk .ot-b-addl-desc{font-size:.813em;line-height:1.5}#onetrust-banner-sdk .ot-dpd-desc{margin-bottom:10px}#onetrust-banner-sdk .ot-dpd-desc>.ot-b-addl-desc{margin-top:10px;margin-bottom:10px;font-size:1em}@media only screen and (max-width: 425px){#onetrust-banner-sdk #onetrust-close-btn-container{position:absolute;top:6px;right:2px}#onetrust-banner-sdk #onetrust-policy{margin-left:0;margin-top:3em}#onetrust-banner-sdk #onetrust-button-group{display:block}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{width:100%}#onetrust-banner-sdk .onetrust-close-btn-ui{top:auto;transform:none}#onetrust-banner-sdk #onetrust-policy-title{display:inline;float:none}#onetrust-banner-sdk #banner-options{margin:0;padding:0;width:100%}}@media only screen and (min-width: 426px)and (max-width: 896px){#onetrust-banner-sdk #onetrust-close-btn-container{position:absolute;top:0;right:0}#onetrust-banner-sdk #onetrust-policy{margin-left:1em;margin-right:1em}#onetrust-banner-sdk .onetrust-close-btn-ui{top:10px;right:10px}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{width:95%}#onetrust-banner-sdk.ot-iab-2 #onetrust-group-container{width:100%}#onetrust-banner-sdk.ot-bnr-w-logo #onetrust-button-group-parent{padding-left:50px}#onetrust-banner-sdk #onetrust-button-group-parent{width:100%;position:relative;margin-left:0}#onetrust-banner-sdk #onetrust-button-group button{display:inline-block}#onetrust-banner-sdk #onetrust-button-group{margin-right:0;text-align:center}#onetrust-banner-sdk .has-reject-all-button #onetrust-pc-btn-handler{float:left}#onetrust-banner-sdk .has-reject-all-button #onetrust-reject-all-handler,#onetrust-banner-sdk .has-reject-all-button #onetrust-accept-btn-handler{float:right}#onetrust-banner-sdk .has-reject-all-button #onetrust-button-group{width:calc(100% - 2em);margin-right:0}#onetrust-banner-sdk .has-reject-all-button #onetrust-pc-btn-handler.cookie-setting-link{padding-left:0px;text-align:left}#onetrust-banner-sdk.ot-buttons-fw .ot-sdk-three button{width:100%;text-align:center}#onetrust-banner-sdk.ot-buttons-fw #onetrust-button-group-parent button{float:none}#onetrust-banner-sdk.ot-buttons-fw #onetrust-pc-btn-handler.cookie-setting-link{text-align:center}}@media only screen and (min-width: 550px){#onetrust-banner-sdk .banner-option:not(:first-child){border-left:1px solid #d8d8d8;padding-left:25px}}@media only screen and (min-width: 425px)and (max-width: 550px){#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group,#onetrust-banner-sdk.ot-iab-2 #onetrust-policy,#onetrust-banner-sdk.ot-iab-2 .banner-option{width:100%}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-accept-btn-handler,#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-reject-all-handler,#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-pc-btn-handler{width:100%}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-accept-btn-handler,#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-reject-all-handler{float:left}}@media only screen and (min-width: 769px){#onetrust-banner-sdk #onetrust-button-group{margin-right:30%}#onetrust-banner-sdk #banner-options{margin-left:2em;margin-right:5em;margin-bottom:1.25em;width:calc(100% - 7em)}}@media only screen and (min-width: 897px)and (max-width: 1023px){#onetrust-banner-sdk.vertical-align-content #onetrust-button-group-parent{position:absolute;top:50%;left:75%;transform:translateY(-50%)}#onetrust-banner-sdk #onetrust-close-btn-container{top:50%;margin:auto;transform:translate(-50%, -50%);position:absolute;padding:0;right:0}#onetrust-banner-sdk #onetrust-close-btn-container button{position:relative;margin:0;right:-22px;top:2px}}@media only screen and (min-width: 1024px){#onetrust-banner-sdk #onetrust-close-btn-container{top:50%;margin:auto;transform:translate(-50%, -50%);position:absolute;right:0}#onetrust-banner-sdk #onetrust-close-btn-container button{right:-12px}#onetrust-banner-sdk #onetrust-policy{margin-left:2em}#onetrust-banner-sdk.vertical-align-content #onetrust-button-group-parent{position:absolute;top:50%;left:60%;transform:translateY(-50%)}#onetrust-banner-sdk .ot-optout-signal{width:50%}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-title{width:50%}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-text,#onetrust-banner-sdk.ot-iab-2 :not(.ot-dpd-desc)>.ot-b-addl-desc{margin-bottom:1em;width:50%;border-right:1px solid #d8d8d8;padding-right:1rem}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-text{margin-bottom:0;padding-bottom:1em}#onetrust-banner-sdk.ot-iab-2 :not(.ot-dpd-desc)>.ot-b-addl-desc{margin-bottom:0;padding-bottom:1em}#onetrust-banner-sdk.ot-iab-2 .ot-dpd-container{width:45%;padding-left:1rem;display:inline-block;float:none}#onetrust-banner-sdk.ot-iab-2 .ot-dpd-title{line-height:1.7}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group-parent{left:auto;right:4%;margin-left:0}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group button{display:block}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{margin:auto;width:30%}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{width:60%}#onetrust-banner-sdk #onetrust-button-group{margin-right:auto}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{margin-top:1em}}@media only screen and (min-width: 890px){#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group-parent{padding-left:3%;padding-right:4%;margin-left:0}#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group{margin-right:0;margin-top:1.25em;width:100%}#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button{width:100%;margin-bottom:5px;margin-top:5px}#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button:last-of-type{margin-bottom:20px}}@media only screen and (min-width: 1280px){#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{width:55%}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{width:44%;padding-left:2%;padding-right:2%}#onetrust-banner-sdk:not(.ot-iab-2).vertical-align-content #onetrust-button-group-parent{position:absolute;left:55%}}
        #onetrust-consent-sdk #onetrust-banner-sdk {background-color: #FFF;}
            #onetrust-consent-sdk #onetrust-policy-title,
                    #onetrust-consent-sdk #onetrust-policy-text,
                    #onetrust-consent-sdk .ot-b-addl-desc,
                    #onetrust-consent-sdk .ot-dpd-desc,
                    #onetrust-consent-sdk .ot-dpd-title,
                    #onetrust-consent-sdk #onetrust-policy-text *:not(.onetrust-vendors-list-handler),
                    #onetrust-consent-sdk .ot-dpd-desc *:not(.onetrust-vendors-list-handler),
                    #onetrust-consent-sdk #onetrust-banner-sdk #banner-options *,
                    #onetrust-banner-sdk .ot-cat-header,
                    #onetrust-banner-sdk .ot-optout-signal
                    {
                        color: #2E2E2E;
                    }
            #onetrust-consent-sdk #onetrust-banner-sdk .banner-option-details {
                    background-color: #E9E9E9;}
             #onetrust-consent-sdk #onetrust-banner-sdk a[href],
                    #onetrust-consent-sdk #onetrust-banner-sdk a[href] font,
                    #onetrust-consent-sdk #onetrust-banner-sdk .ot-link-btn
                        {
                            color: #007398;
                        }#onetrust-consent-sdk #onetrust-accept-btn-handler,
                         #onetrust-banner-sdk #onetrust-reject-all-handler {
                            background-color: #007398;border-color: #007398;
                color: #FFF;
            }
            #onetrust-consent-sdk #onetrust-banner-sdk *:focus,
            #onetrust-consent-sdk #onetrust-banner-sdk:focus {
               outline-color: #000000;
               outline-width: 1px;
            }
            #onetrust-consent-sdk #onetrust-pc-btn-handler,
            #onetrust-consent-sdk #onetrust-pc-btn-handler.cookie-setting-link {
                color: #6CC04A; border-color: #6CC04A;
                background-color:
                #FFF;
            }/*! v6.10.0 2021-01-18 */
div#onetrust-consent-sdk #onetrust-banner-sdk{border-top:2px solid #d0d3d4!important;box-shadow:none;padding:24px}div#onetrust-consent-sdk button{border-radius:0!important;box-shadow:none!important;box-sizing:border-box!important;font-size:20px!important;font-weight:400!important;letter-spacing:0!important;max-width:none!important;white-space:nowrap!important}div#onetrust-consent-sdk button:not(.ot-link-btn){background-color:#007dbc!important;border:2px solid #007dbc!important;color:#fff!important;height:48px!important;padding:0 1em!important;width:auto!important}div#onetrust-consent-sdk button:hover{background-color:#fff!important;border-color:#949494!important;color:#2e2e2e!important}div#onetrust-consent-sdk button.ot-link-btn{color:#007dbc!important;font-size:16px!important;text-decoration:underline}div#onetrust-consent-sdk button.ot-link-btn:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-consent-sdk a,div#onetrust-pc-sdk a{color:#007dbc!important;text-decoration:underline!important}div#onetrust-consent-sdk a,div#onetrust-consent-sdk button,div#onetrust-consent-sdk p:hover{opacity:1!important}div#onetrust-consent-sdk a:focus,div#onetrust-consent-sdk button:focus,div#onetrust-consent-sdk input:focus{outline:2px solid #949494!important;outline-offset:1px!important}div#onetrust-banner-sdk .ot-sdk-container{padding:0;width:auto}div#onetrust-banner-sdk .ot-sdk-row{align-items:flex-start;box-sizing:border-box;display:flex;flex-direction:column;justify-content:space-between;margin:auto;max-width:1152px}div#onetrust-banner-sdk .ot-sdk-row:after{display:none}div#onetrust-banner-sdk #onetrust-group-container,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-group-container,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{flex-grow:1;width:auto}div#onetrust-banner-sdk #onetrust-policy,div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy{margin:0;overflow:visible}div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy-text,div#onetrust-consent-sdk #onetrust-policy-text{font-size:16px;line-height:24px;max-width:44em;margin:0}div#onetrust-consent-sdk #onetrust-policy-text a[href]{font-weight:400;margin-left:8px}div#onetrust-banner-sdk #onetrust-button-group-parent{flex:0 0 auto;margin:32px 0 0;width:100%}div#onetrust-banner-sdk #onetrust-button-group{display:flex;flex-direction:row;flex-wrap:wrap;justify-content:flex-end;margin:-8px}div#onetrust-banner-sdk .banner-actions-container{display:flex;flex:1 0 auto}div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button:last-of-type,div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto;margin:8px;width:auto}div#onetrust-consent-sdk #onetrust-pc-btn-handler{background-color:#fff!important;color:inherit!important}div#onetrust-banner-sdk #onetrust-close-btn-container{display:none}@media only screen and (min-width:556px){div#onetrust-consent-sdk #onetrust-banner-sdk{padding:40px}div#onetrust-banner-sdk #onetrust-policy{margin:0 40px 0 0}div#onetrust-banner-sdk .ot-sdk-row{align-items:center;flex-direction:row}div#onetrust-banner-sdk #onetrust-button-group-parent,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-button-group-parent,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{margin:0;padding:0;width:auto}div#onetrust-banner-sdk #onetrust-button-group,div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group{align-items:stretch;flex-direction:column-reverse;margin:0}div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto}}@media only screen and (min-width:768px){div#onetrust-banner-sdk #onetrust-policy{margin:0 48px 0 0}div#onetrust-consent-sdk #onetrust-banner-sdk{padding:48px}}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-size:16px;line-height:24px}div#onetrust-consent-sdk #onetrust-pc-sdk p,div#onetrust-pc-sdk #ot-pc-desc,div#onetrust-pc-sdk .category-host-list-handler,div#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{font-size:16px;font-weight:400;line-height:24px}div#onetrust-consent-sdk a:hover,div#onetrust-pc-sdk a:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-pc-sdk{border-radius:0;bottom:0;height:auto;left:0;margin:auto;max-width:100%;overflow:hidden;right:0;top:0;width:512px;max-height:800px}div#onetrust-pc-sdk .ot-pc-header{display:none}div#onetrust-pc-sdk #ot-pc-content{overscroll-behavior:contain;padding:0 12px 0 24px;margin:16px 4px 0 0;top:0;right:16px;left:0;width:auto}div#onetrust-pc-sdk #ot-category-title,div#onetrust-pc-sdk #ot-pc-title{font-size:24px;font-weight:400;line-height:32px;margin:0 0 16px}div#onetrust-pc-sdk #ot-pc-desc{padding:0}div#onetrust-pc-sdk #ot-pc-desc a{display:inline}div#onetrust-pc-sdk #accept-recommended-btn-handler{display:none!important}div#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:2px solid #007dbc;outline-offset:-1px;transition:none}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item{border-width:0 0 2px}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item:first-of-type{border-width:2px 0}div#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{padding:8px 0;width:100%}div#onetrust-pc-sdk .ot-plus-minus{transform:translateY(2px)}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item>button{background:0 0!important;border:0!important;height:44px!important;max-width:none!important;width:calc(100% - 48px)!important}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-weight:700}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr{padding:0}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item .ot-acc-grpdesc{padding:0;width:100%}div#onetrust-pc-sdk .ot-acc-grpcntr .ot-subgrp-cntr{border:0;padding:0}div#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp{margin:0}div#onetrust-pc-sdk .ot-always-active-group .ot-cat-header{width:calc(100% - 160px)}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{width:calc(100% - 88px)}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active{color:inherit;font-size:12px;font-weight:400;line-height:1.5;padding-right:48px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:before{border-radius:12px;position:absolute;right:0;top:0;content:'';background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:after{border-radius:50%;position:absolute;right:5px;top:4px;content:'';background-color:#007dbc;height:12px;width:12px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active,div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl{right:2px}div#onetrust-pc-sdk .ot-switch{display:block;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob,div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob{background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob:before{background-color:#737373;height:8px;left:4px;top:4px;width:8px}div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob:before{background-color:#007dbc;height:12px;left:0;top:2px;width:12px}div#onetrust-pc-sdk .ot-tgl input:focus+.ot-switch .ot-switch-nob{box-shadow:0 0;outline:2px solid #007dbc!important;outline-offset:1px;transition:none}div#onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-grpcntr.ot-acc-txt{background-color:transparent;padding-left:3px}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr,div#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr{overflow:visible;width:100%}div#onetrust-pc-sdk .ot-pc-footer{border-top:0 solid}div#onetrust-pc-sdk .ot-btn-container{padding-top:24px;text-align:center}div#onetrust-pc-sdk .ot-pc-footer button{margin:8px 0;background-color:#fff}div#onetrust-pc-sdk .ot-pc-footer-logo{background-color:#fff}div#onetrust-pc-sdk #ot-lst-title span{font-size:24px;font-weight:400;line-height:32px}div#onetrust-pc-sdk #ot-host-lst .ot-host-desc,div#onetrust-pc-sdk #ot-host-lst .ot-host-expand,div#onetrust-pc-sdk #ot-host-lst .ot-host-name,div#onetrust-pc-sdk #ot-host-lst .ot-host-name a,div#onetrust-pc-sdk .back-btn-handler,div#onetrust-pc-sdk .ot-host-opt li>div div{font-size:16px;font-weight:400;line-height:24px}div#onetrust-pc-sdk #ot-pc-lst{top:0}div#onetrust-pc-sdk .back-btn-handler{text-decoration:none!important}div#onetrust-pc-sdk #filter-btn-handler:hover svg{filter:invert(1)}div#onetrust-pc-sdk .back-btn-handler svg{width:16px;height:16px}div#onetrust-pc-sdk .ot-host-item>button{background:0 0!important;border:0!important;height:66px!important;max-width:none!important;width:calc(100% - 5px)!important;transform:translate(2px,2px)}div#onetrust-pc-sdk .ot-host-item{border-bottom:2px solid #b9b9b9;padding:0}div#onetrust-pc-sdk .ot-host-item .ot-acc-hdr{margin:0 0 -6px;padding:8px 0}div#onetrust-pc-sdk ul li:first-child{border-top:2px solid #b9b9b9}div#onetrust-pc-sdk .ot-host-item .ot-plus-minus{margin:0 8px 0 0}div#onetrust-pc-sdk .ot-search-cntr{width:calc(100% - 48px)}div#onetrust-pc-sdk .ot-host-opt .ot-host-info{background-color:transparent}div#onetrust-pc-sdk .ot-host-opt li>div div{padding:0}div#onetrust-pc-sdk #vendor-search-handler{border-radius:0;border-color:#939393;border-style:solid;border-width:2px 0 2px 2px;font-size:20px;height:48px;margin:0}div#onetrust-pc-sdk .ot-lst-subhdr svg{right:0;top:8px}div#onetrust-pc-sdk .ot-fltr-cntr{box-sizing:border-box;right:0;width:48px}div#onetrust-pc-sdk #filter-btn-handler{width:48px!important;padding:8px!important}div#onetrust-pc-sdk #ot-fltr-modal{max-height:400px}
#onetrust-pc-sdk.otPcCenter{overflow:hidden;position:fixed;margin:0 auto;top:5%;right:0;left:0;width:40%;max-width:575px;min-width:575px;border-radius:2.5px;z-index:2147483647;background-color:#fff;-webkit-box-shadow:0px 2px 10px -3px #999;-moz-box-shadow:0px 2px 10px -3px #999;box-shadow:0px 2px 10px -3px #999}#onetrust-pc-sdk.otPcCenter[dir=rtl]{right:0;left:0}#onetrust-pc-sdk.otRelFont{font-size:1rem}#onetrust-pc-sdk .ot-optout-signal{margin-top:.625rem}#onetrust-pc-sdk #ot-addtl-venlst .ot-arw-cntr,#onetrust-pc-sdk #ot-addtl-venlst .ot-plus-minus,#onetrust-pc-sdk .ot-hide-tgl{visibility:hidden}#onetrust-pc-sdk #ot-addtl-venlst .ot-arw-cntr *,#onetrust-pc-sdk #ot-addtl-venlst .ot-plus-minus *,#onetrust-pc-sdk .ot-hide-tgl *{visibility:hidden}#onetrust-pc-sdk #ot-gn-venlst .ot-ven-item .ot-acc-hdr{min-height:40px}#onetrust-pc-sdk .ot-pc-header{height:39px;padding:10px 0 10px 30px;border-bottom:1px solid #e9e9e9}#onetrust-pc-sdk #ot-pc-title,#onetrust-pc-sdk #ot-category-title,#onetrust-pc-sdk .ot-cat-header,#onetrust-pc-sdk #ot-lst-title,#onetrust-pc-sdk .ot-ven-hdr .ot-ven-name,#onetrust-pc-sdk .ot-always-active{font-weight:bold;color:dimgray}#onetrust-pc-sdk .ot-always-active-group .ot-cat-header{width:55%;font-weight:700}#onetrust-pc-sdk .ot-cat-item p{clear:both;float:left;margin-top:10px;margin-bottom:5px;line-height:1.5;font-size:.812em;color:dimgray}#onetrust-pc-sdk .ot-close-icon{height:44px;width:44px;background-size:10px}#onetrust-pc-sdk #ot-pc-title{float:left;font-size:1em;line-height:1.5;margin-bottom:10px;margin-top:10px;width:100%}#onetrust-pc-sdk #accept-recommended-btn-handler{margin-right:10px;margin-bottom:25px;outline-offset:-1px}#onetrust-pc-sdk #ot-pc-desc{clear:both;width:100%;font-size:.812em;line-height:1.5;margin-bottom:25px}#onetrust-pc-sdk #ot-pc-desc *{font-size:inherit;line-height:inherit}#onetrust-pc-sdk #ot-pc-desc ul li{padding:10px 0px}#onetrust-pc-sdk a{color:#656565;cursor:pointer}#onetrust-pc-sdk a:hover{color:#3860be}#onetrust-pc-sdk label{margin-bottom:0}#onetrust-pc-sdk #vdr-lst-dsc{font-size:.812em;line-height:1.5;padding:10px 15px 5px 15px}#onetrust-pc-sdk button{max-width:394px;padding:12px 30px;line-height:1;word-break:break-word;word-wrap:break-word;white-space:normal;font-weight:bold;height:auto}#onetrust-pc-sdk .ot-link-btn{padding:0;margin-bottom:0;border:0;font-weight:normal;line-height:normal;width:auto;height:auto}#onetrust-pc-sdk #ot-pc-content{position:absolute;overflow-y:scroll;padding-left:2px;padding-right:30px;top:60px;bottom:110px;margin:1px 3px 0 30px;width:calc(100% - 63px)}#onetrust-pc-sdk .ot-vs-list .ot-always-active,#onetrust-pc-sdk .ot-cat-grp .ot-always-active{float:right;clear:none;color:#3860be;margin:0;font-size:.813em;line-height:1.3}#onetrust-pc-sdk .ot-pc-scrollbar::-webkit-scrollbar-track{margin-right:20px}#onetrust-pc-sdk .ot-pc-scrollbar::-webkit-scrollbar{width:11px}#onetrust-pc-sdk .ot-pc-scrollbar::-webkit-scrollbar-thumb{border-radius:10px;background:#d8d8d8}#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:#000 1px solid}#onetrust-pc-sdk .ot-pc-scrollbar{scrollbar-arrow-color:#d8d8d8;scrollbar-darkshadow-color:#d8d8d8;scrollbar-face-color:#d8d8d8;scrollbar-shadow-color:#d8d8d8}#onetrust-pc-sdk .save-preference-btn-handler{margin-right:20px}#onetrust-pc-sdk .ot-pc-refuse-all-handler{margin-right:10px}#onetrust-pc-sdk #ot-pc-desc .privacy-notice-link{margin-left:0;margin-right:8px}#onetrust-pc-sdk #ot-pc-desc .ot-imprint-handler{margin-left:0;margin-right:8px}#onetrust-pc-sdk .ot-subgrp-cntr{display:inline-block;clear:both;width:100%;padding-top:15px}#onetrust-pc-sdk .ot-switch+.ot-subgrp-cntr{padding-top:10px}#onetrust-pc-sdk ul.ot-subgrps{margin:0;font-size:initial}#onetrust-pc-sdk ul.ot-subgrps li p,#onetrust-pc-sdk ul.ot-subgrps li h5{font-size:.813em;line-height:1.4;color:dimgray}#onetrust-pc-sdk ul.ot-subgrps .ot-switch{min-height:auto}#onetrust-pc-sdk ul.ot-subgrps .ot-switch-nob{top:0}#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr{display:inline-block;width:100%}#onetrust-pc-sdk ul.ot-subgrps .ot-acc-txt{margin:0}#onetrust-pc-sdk ul.ot-subgrps li{padding:0;border:none}#onetrust-pc-sdk ul.ot-subgrps li h5{position:relative;top:5px;font-weight:bold;margin-bottom:0;float:left}#onetrust-pc-sdk li.ot-subgrp{margin-left:20px;overflow:auto}#onetrust-pc-sdk li.ot-subgrp>h5{width:calc(100% - 100px)}#onetrust-pc-sdk .ot-cat-item p>ul,#onetrust-pc-sdk li.ot-subgrp p>ul{margin:0px;list-style:disc;margin-left:15px;font-size:inherit}#onetrust-pc-sdk .ot-cat-item p>ul li,#onetrust-pc-sdk li.ot-subgrp p>ul li{font-size:inherit;padding-top:10px;padding-left:0px;padding-right:0px;border:none}#onetrust-pc-sdk .ot-cat-item p>ul li:last-child,#onetrust-pc-sdk li.ot-subgrp p>ul li:last-child{padding-bottom:10px}#onetrust-pc-sdk .ot-pc-logo{height:40px;width:120px}#onetrust-pc-sdk .ot-pc-footer{position:absolute;bottom:0px;width:100%;max-height:160px;border-top:1px solid #d8d8d8}#onetrust-pc-sdk.ot-ftr-stacked .ot-pc-refuse-all-handler{margin-bottom:0px}#onetrust-pc-sdk.ot-ftr-stacked #ot-pc-content{bottom:160px}#onetrust-pc-sdk.ot-ftr-stacked .ot-pc-footer button{width:100%;max-width:none}#onetrust-pc-sdk.ot-ftr-stacked .ot-btn-container{margin:0 30px;width:calc(100% - 60px);padding-right:0}#onetrust-pc-sdk .ot-pc-footer-logo{height:30px;width:100%;text-align:right;background:#f4f4f4}#onetrust-pc-sdk .ot-pc-footer-logo a{display:inline-block;margin-top:5px;margin-right:10px}#onetrust-pc-sdk[dir=rtl] .ot-pc-footer-logo{direction:rtl}#onetrust-pc-sdk[dir=rtl] .ot-pc-footer-logo a{margin-right:25px}#onetrust-pc-sdk .ot-tgl{float:right;position:relative;z-index:1}#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob{background-color:#468254;border:1px solid #fff}#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob:before{-webkit-transform:translateX(20px);-ms-transform:translateX(20px);transform:translateX(20px);background-color:#fff;border-color:#fff}#onetrust-pc-sdk .ot-tgl input:focus+.ot-switch{outline:#000 solid 1px}#onetrust-pc-sdk .ot-switch{position:relative;display:inline-block;width:45px;height:25px}#onetrust-pc-sdk .ot-switch-nob{position:absolute;cursor:pointer;top:0;left:0;right:0;bottom:0;background-color:#767676;border:1px solid #ddd;transition:all .2s ease-in 0s;-moz-transition:all .2s ease-in 0s;-o-transition:all .2s ease-in 0s;-webkit-transition:all .2s ease-in 0s;border-radius:20px}#onetrust-pc-sdk .ot-switch-nob:before{position:absolute;content:"";height:18px;width:18px;bottom:3px;left:3px;background-color:#fff;-webkit-transition:.4s;transition:.4s;border-radius:20px}#onetrust-pc-sdk .ot-chkbox input:checked~label::before{background-color:#3860be}#onetrust-pc-sdk .ot-chkbox input+label::after{content:none;color:#fff}#onetrust-pc-sdk .ot-chkbox input:checked+label::after{content:""}#onetrust-pc-sdk .ot-chkbox input:focus+label::before{outline-style:solid;outline-width:2px;outline-style:auto}#onetrust-pc-sdk .ot-chkbox input[aria-checked=mixed]~label::before{background-color:#3860be}#onetrust-pc-sdk .ot-chkbox input[aria-checked=mixed]+label::after{content:""}#onetrust-pc-sdk .ot-chkbox label{position:relative;display:inline-block;padding-left:30px;cursor:pointer;font-weight:500}#onetrust-pc-sdk .ot-chkbox label::before,#onetrust-pc-sdk .ot-chkbox label::after{position:absolute;content:"";display:inline-block;border-radius:3px}#onetrust-pc-sdk .ot-chkbox label::before{height:18px;width:18px;border:1px solid #3860be;left:0px;top:auto}#onetrust-pc-sdk .ot-chkbox label::after{height:5px;width:9px;border-left:3px solid;border-bottom:3px solid;transform:rotate(-45deg);-o-transform:rotate(-45deg);-ms-transform:rotate(-45deg);-webkit-transform:rotate(-45deg);left:4px;top:5px}#onetrust-pc-sdk .ot-label-txt{display:none}#onetrust-pc-sdk .ot-chkbox input,#onetrust-pc-sdk .ot-tgl input{position:absolute;opacity:0;width:0;height:0}#onetrust-pc-sdk .ot-arw-cntr{float:right;position:relative;pointer-events:none}#onetrust-pc-sdk .ot-arw-cntr .ot-arw{width:16px;height:16px;margin-left:5px;color:dimgray;display:inline-block;vertical-align:middle;-webkit-transition:all 150ms ease-in 0s;-moz-transition:all 150ms ease-in 0s;-o-transition:all 150ms ease-in 0s;transition:all 150ms ease-in 0s}#onetrust-pc-sdk input:checked~.ot-acc-hdr .ot-arw,#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-arw-cntr svg{transform:rotate(90deg);-o-transform:rotate(90deg);-ms-transform:rotate(90deg);-webkit-transform:rotate(90deg)}#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:#000 1px solid}#onetrust-pc-sdk .ot-tgl-cntr,#onetrust-pc-sdk .ot-arw-cntr{display:inline-block}#onetrust-pc-sdk .ot-tgl-cntr{width:45px;float:right;margin-top:2px}#onetrust-pc-sdk #ot-lst-cnt .ot-tgl-cntr{margin-top:10px}#onetrust-pc-sdk .ot-always-active-subgroup{width:auto;padding-left:0px !important;top:3px;position:relative}#onetrust-pc-sdk .ot-label-status{font-size:.75em;display:none;font-size:.75em;position:relative;top:2px;padding-right:5px;float:left}#onetrust-pc-sdk .ot-arw-cntr{margin-top:-1px}#onetrust-pc-sdk .ot-arw-cntr svg{-webkit-transition:all 300ms ease-in 0s;-moz-transition:all 300ms ease-in 0s;-o-transition:all 300ms ease-in 0s;transition:all 300ms ease-in 0s;height:10px;width:10px}#onetrust-pc-sdk input:checked~.ot-acc-hdr .ot-arw{transform:rotate(90deg);-o-transform:rotate(90deg);-ms-transform:rotate(90deg);-webkit-transform:rotate(90deg)}#onetrust-pc-sdk .ot-arw{width:10px;margin-left:15px;transition:all 300ms ease-in 0s;-webkit-transition:all 300ms ease-in 0s;-moz-transition:all 300ms ease-in 0s;-o-transition:all 300ms ease-in 0s}#onetrust-pc-sdk .ot-vlst-cntr{margin-bottom:0}#onetrust-pc-sdk .ot-hlst-cntr{margin-top:5px;display:inline-block;width:100%}#onetrust-pc-sdk .category-vendors-list-handler,#onetrust-pc-sdk .category-vendors-list-handler+a,#onetrust-pc-sdk .category-host-list-handler{clear:both;color:#3860be;margin-left:0;font-size:.813em;text-decoration:none;float:left;overflow:hidden}#onetrust-pc-sdk .category-vendors-list-handler:hover,#onetrust-pc-sdk .category-vendors-list-handler+a:hover,#onetrust-pc-sdk .category-host-list-handler:hover{text-decoration-line:underline}#onetrust-pc-sdk .category-vendors-list-handler+a{clear:none}#onetrust-pc-sdk .ot-vlst-cntr .ot-ext-lnk,#onetrust-pc-sdk .ot-ven-hdr .ot-ext-lnk{display:inline-block;height:13px;width:13px;background-repeat:no-repeat;margin-left:1px;margin-top:6px;cursor:pointer}#onetrust-pc-sdk .ot-ven-hdr .ot-ext-lnk{margin-bottom:-1px}#onetrust-pc-sdk .back-btn-handler{font-size:1em;text-decoration:none}#onetrust-pc-sdk .back-btn-handler:hover{opacity:.6}#onetrust-pc-sdk #ot-lst-title h3{display:inline-block;word-break:break-word;word-wrap:break-word;margin-bottom:0;color:#656565;font-size:1em;font-weight:bold;margin-left:15px}#onetrust-pc-sdk #ot-lst-title{margin:10px 0 10px 0px;font-size:1em;text-align:left}#onetrust-pc-sdk #ot-pc-hdr{margin:0 0 0 30px;height:auto;width:auto}#onetrust-pc-sdk #ot-pc-hdr input::placeholder{color:#707070;font-style:italic}#onetrust-pc-sdk #vendor-search-handler{height:31px;width:100%;border-radius:50px;font-size:.8em;padding-right:35px;padding-left:15px;float:left;margin-left:15px}#onetrust-pc-sdk .ot-ven-name{display:block;width:auto;padding-right:5px}#onetrust-pc-sdk #ot-lst-cnt{overflow-y:auto;margin-left:20px;margin-right:7px;width:calc(100% - 27px);max-height:calc(100% - 80px);height:100%;transform:translate3d(0, 0, 0)}#onetrust-pc-sdk #ot-pc-lst{width:100%;bottom:100px;position:absolute;top:60px}#onetrust-pc-sdk #ot-pc-lst:not(.ot-enbl-chr) .ot-tgl-cntr .ot-arw-cntr,#onetrust-pc-sdk #ot-pc-lst:not(.ot-enbl-chr) .ot-tgl-cntr .ot-arw-cntr *{visibility:hidden}#onetrust-pc-sdk #ot-pc-lst .ot-tgl-cntr{right:12px;position:absolute}#onetrust-pc-sdk #ot-pc-lst .ot-arw-cntr{float:right;position:relative}#onetrust-pc-sdk #ot-pc-lst .ot-arw{margin-left:10px}#onetrust-pc-sdk #ot-pc-lst .ot-acc-hdr{overflow:hidden;cursor:pointer}#onetrust-pc-sdk .ot-vlst-cntr{overflow:hidden}#onetrust-pc-sdk #ot-sel-blk{overflow:hidden;width:100%;position:sticky;position:-webkit-sticky;top:0;z-index:3}#onetrust-pc-sdk #ot-back-arw{height:12px;width:12px}#onetrust-pc-sdk .ot-lst-subhdr{width:100%;display:inline-block}#onetrust-pc-sdk .ot-search-cntr{float:left;width:78%;position:relative}#onetrust-pc-sdk .ot-search-cntr>svg{width:30px;height:30px;position:absolute;float:left;right:-15px}#onetrust-pc-sdk .ot-fltr-cntr{float:right;right:50px;position:relative}#onetrust-pc-sdk #filter-btn-handler{background-color:#3860be;border-radius:17px;display:inline-block;position:relative;width:32px;height:32px;-moz-transition:.1s ease;-o-transition:.1s ease;-webkit-transition:1s ease;transition:.1s ease;padding:0;margin:0}#onetrust-pc-sdk #filter-btn-handler:hover{background-color:#3860be}#onetrust-pc-sdk #filter-btn-handler svg{width:12px;height:12px;margin:3px 10px 0 10px;display:block;position:static;right:auto;top:auto}#onetrust-pc-sdk .ot-ven-link,#onetrust-pc-sdk .ot-ven-legclaim-link{color:#3860be;text-decoration:none;font-weight:100;display:inline-block;padding-top:10px;transform:translate(0, 1%);-o-transform:translate(0, 1%);-ms-transform:translate(0, 1%);-webkit-transform:translate(0, 1%);position:relative;z-index:2}#onetrust-pc-sdk .ot-ven-link *,#onetrust-pc-sdk .ot-ven-legclaim-link *{font-size:inherit}#onetrust-pc-sdk .ot-ven-link:hover,#onetrust-pc-sdk .ot-ven-legclaim-link:hover{text-decoration:underline}#onetrust-pc-sdk .ot-ven-hdr{width:calc(100% - 160px);height:auto;float:left;word-break:break-word;word-wrap:break-word;vertical-align:middle;padding-bottom:3px}#onetrust-pc-sdk .ot-ven-link,#onetrust-pc-sdk .ot-ven-legclaim-link{letter-spacing:.03em;font-size:.75em;font-weight:400}#onetrust-pc-sdk .ot-ven-dets{border-radius:2px;background-color:#f8f8f8}#onetrust-pc-sdk .ot-ven-dets li:first-child p:first-child{border-top:none}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:not(:first-child){border-top:1px solid #ddd !important}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:nth-child(n+3) p{display:inline-block}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:nth-child(n+3) p:nth-of-type(odd){width:30%}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:nth-child(n+3) p:nth-of-type(even){width:50%;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc p,#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc h5{padding-top:5px;padding-bottom:5px;display:block}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc h5{display:inline-block}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc p:nth-last-child(-n+1){padding-bottom:10px}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc p:nth-child(-n+2):not(.disc-pur){padding-top:10px}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc .disc-pur-cont{display:inline}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc .disc-pur{position:relative;width:50% !important;word-break:break-word;word-wrap:break-word;left:calc(30% + 17px)}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc .disc-pur:nth-child(-n+1){position:static}#onetrust-pc-sdk .ot-ven-dets p,#onetrust-pc-sdk .ot-ven-dets h5,#onetrust-pc-sdk .ot-ven-dets span{font-size:.69em;text-align:left;vertical-align:middle;word-break:break-word;word-wrap:break-word;margin:0;padding-bottom:10px;padding-left:15px;color:#2e3644}#onetrust-pc-sdk .ot-ven-dets h5{padding-top:5px}#onetrust-pc-sdk .ot-ven-dets span{color:dimgray;padding:0;vertical-align:baseline}#onetrust-pc-sdk .ot-ven-dets .ot-ven-pur h5{border-top:1px solid #e9e9e9;border-bottom:1px solid #e9e9e9;padding-bottom:5px;margin-bottom:5px;font-weight:bold}#onetrust-pc-sdk #ot-host-lst .ot-sel-all{float:right;position:relative;margin-right:42px;top:10px}#onetrust-pc-sdk #ot-host-lst .ot-sel-all input[type=checkbox]{width:auto;height:auto}#onetrust-pc-sdk #ot-host-lst .ot-sel-all label{height:20px;width:20px;padding-left:0px}#onetrust-pc-sdk #ot-host-lst .ot-acc-txt{overflow:hidden;width:95%}#onetrust-pc-sdk .ot-host-hdr{position:relative;z-index:1;pointer-events:none;width:calc(100% - 125px);float:left}#onetrust-pc-sdk .ot-host-name,#onetrust-pc-sdk .ot-host-desc{display:inline-block;width:90%}#onetrust-pc-sdk .ot-host-name{pointer-events:none}#onetrust-pc-sdk .ot-host-hdr>a{text-decoration:underline;font-size:.82em;position:relative;z-index:2;float:left;margin-bottom:5px;pointer-events:initial}#onetrust-pc-sdk .ot-host-name+a{margin-top:5px}#onetrust-pc-sdk .ot-host-name,#onetrust-pc-sdk .ot-host-name a,#onetrust-pc-sdk .ot-host-desc,#onetrust-pc-sdk .ot-host-info{color:dimgray;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-host-name,#onetrust-pc-sdk .ot-host-name a{font-weight:bold;font-size:.82em;line-height:1.3}#onetrust-pc-sdk .ot-host-name a{font-size:1em}#onetrust-pc-sdk .ot-host-expand{margin-top:3px;margin-bottom:3px;clear:both;display:block;color:#3860be;font-size:.72em;font-weight:normal}#onetrust-pc-sdk .ot-host-expand *{font-size:inherit}#onetrust-pc-sdk .ot-host-desc,#onetrust-pc-sdk .ot-host-info{font-size:.688em;line-height:1.4;font-weight:normal}#onetrust-pc-sdk .ot-host-desc{margin-top:10px}#onetrust-pc-sdk .ot-host-opt{margin:0;font-size:inherit;display:inline-block;width:100%}#onetrust-pc-sdk .ot-host-opt li>div div{font-size:.8em;padding:5px 0}#onetrust-pc-sdk .ot-host-opt li>div div:nth-child(1){width:30%;float:left}#onetrust-pc-sdk .ot-host-opt li>div div:nth-child(2){width:70%;float:left;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-host-info{border:none;display:inline-block;width:calc(100% - 10px);padding:10px;margin-bottom:10px;background-color:#f8f8f8}#onetrust-pc-sdk .ot-host-info>div{overflow:auto}#onetrust-pc-sdk #no-results{text-align:center;margin-top:30px}#onetrust-pc-sdk #no-results p{font-size:1em;color:#2e3644;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk #no-results p span{font-weight:bold}#onetrust-pc-sdk #ot-fltr-modal{width:100%;height:auto;display:none;-moz-transition:.2s ease;-o-transition:.2s ease;-webkit-transition:2s ease;transition:.2s ease;overflow:hidden;opacity:1;right:0}#onetrust-pc-sdk #ot-fltr-modal .ot-label-txt{display:inline-block;font-size:.85em;color:dimgray}#onetrust-pc-sdk #ot-fltr-cnt{z-index:2147483646;background-color:#fff;position:absolute;height:90%;max-height:300px;width:325px;left:210px;margin-top:10px;margin-bottom:20px;padding-right:10px;border-radius:3px;-webkit-box-shadow:0px 0px 12px 2px #c7c5c7;-moz-box-shadow:0px 0px 12px 2px #c7c5c7;box-shadow:0px 0px 12px 2px #c7c5c7}#onetrust-pc-sdk .ot-fltr-scrlcnt{overflow-y:auto;overflow-x:hidden;clear:both;max-height:calc(100% - 60px)}#onetrust-pc-sdk #ot-anchor{border:12px solid rgba(0,0,0,0);display:none;position:absolute;z-index:2147483647;right:55px;top:75px;transform:rotate(45deg);-o-transform:rotate(45deg);-ms-transform:rotate(45deg);-webkit-transform:rotate(45deg);background-color:#fff;-webkit-box-shadow:-3px -3px 5px -2px #c7c5c7;-moz-box-shadow:-3px -3px 5px -2px #c7c5c7;box-shadow:-3px -3px 5px -2px #c7c5c7}#onetrust-pc-sdk .ot-fltr-btns{margin-left:15px}#onetrust-pc-sdk #filter-apply-handler{margin-right:15px}#onetrust-pc-sdk .ot-fltr-opt{margin-bottom:5px;margin-left:15px;min-height:20px;width:75%;position:relative}#onetrust-pc-sdk .ot-fltr-opt p{display:inline-block;margin:0;font-size:.9em;color:#2e3644}#onetrust-pc-sdk .ot-chkbox label span{font-size:.85em;color:dimgray}#onetrust-pc-sdk .ot-chkbox input[type=checkbox]+label::after{content:none;color:#fff}#onetrust-pc-sdk .ot-chkbox input[type=checkbox]:checked+label::after{content:""}#onetrust-pc-sdk .ot-chkbox input[type=checkbox][aria-checked=mixed]+label::after{content:""}#onetrust-pc-sdk .ot-chkbox input[type=checkbox]:focus+label::before{outline-style:solid;outline-width:2px;outline-style:auto}#onetrust-pc-sdk #ot-selall-vencntr,#onetrust-pc-sdk #ot-selall-adtlvencntr,#onetrust-pc-sdk #ot-selall-hostcntr,#onetrust-pc-sdk #ot-selall-licntr,#onetrust-pc-sdk #ot-selall-gnvencntr{right:15px;position:relative;width:20px;height:20px;float:right}#onetrust-pc-sdk #ot-selall-vencntr label,#onetrust-pc-sdk #ot-selall-adtlvencntr label,#onetrust-pc-sdk #ot-selall-hostcntr label,#onetrust-pc-sdk #ot-selall-licntr label,#onetrust-pc-sdk #ot-selall-gnvencntr label{float:left;padding-left:0}#onetrust-pc-sdk #ot-ven-lst:first-child{border-top:1px solid #e2e2e2}#onetrust-pc-sdk ul{list-style:none;padding:0}#onetrust-pc-sdk ul li{position:relative;margin:0;padding:15px 15px 15px 10px;border-bottom:1px solid #e2e2e2}#onetrust-pc-sdk ul li h3,#onetrust-pc-sdk ul li h4{font-size:.75em;color:#656565;margin:0;display:inline-block;width:70%;height:auto;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk ul li p{margin:0;font-size:.7em}#onetrust-pc-sdk ul li input[type=checkbox]{position:absolute;cursor:pointer;width:100%;height:100%;opacity:0;margin:0;top:0;left:0}#onetrust-pc-sdk .ot-cat-item>button:focus,#onetrust-pc-sdk .ot-acc-cntr>button:focus,#onetrust-pc-sdk li>button:focus{outline:#000 solid 2px}#onetrust-pc-sdk .ot-cat-item>button,#onetrust-pc-sdk .ot-acc-cntr>button,#onetrust-pc-sdk li>button{position:absolute;cursor:pointer;width:100%;height:100%;margin:0;top:0;left:0;z-index:1;max-width:none;border:none}#onetrust-pc-sdk .ot-cat-item>button[aria-expanded=false]~.ot-acc-txt,#onetrust-pc-sdk .ot-acc-cntr>button[aria-expanded=false]~.ot-acc-txt,#onetrust-pc-sdk li>button[aria-expanded=false]~.ot-acc-txt{margin-top:0;max-height:0;opacity:0;overflow:hidden;width:100%;transition:.25s ease-out;display:none}#onetrust-pc-sdk .ot-cat-item>button[aria-expanded=true]~.ot-acc-txt,#onetrust-pc-sdk .ot-acc-cntr>button[aria-expanded=true]~.ot-acc-txt,#onetrust-pc-sdk li>button[aria-expanded=true]~.ot-acc-txt{transition:.1s ease-in;margin-top:10px;width:100%;overflow:auto;display:block}#onetrust-pc-sdk .ot-cat-item>button[aria-expanded=true]~.ot-acc-grpcntr,#onetrust-pc-sdk .ot-acc-cntr>button[aria-expanded=true]~.ot-acc-grpcntr,#onetrust-pc-sdk li>button[aria-expanded=true]~.ot-acc-grpcntr{width:auto;margin-top:0px;padding-bottom:10px}#onetrust-pc-sdk .ot-host-item>button:focus,#onetrust-pc-sdk .ot-ven-item>button:focus{outline:0;border:2px solid #000}#onetrust-pc-sdk .ot-hide-acc>button{pointer-events:none}#onetrust-pc-sdk .ot-hide-acc .ot-plus-minus>*,#onetrust-pc-sdk .ot-hide-acc .ot-arw-cntr>*{visibility:hidden}#onetrust-pc-sdk .ot-hide-acc .ot-acc-hdr{min-height:30px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt){padding-right:10px;width:calc(100% - 37px);margin-top:10px;max-height:calc(100% - 90px)}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk{background-color:#f9f9fc;border:1px solid #e2e2e2;width:calc(100% - 2px);padding-bottom:5px;padding-top:5px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt{border:unset;background-color:unset}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt .ot-sel-all-hdr{display:none}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt .ot-sel-all{padding-right:.5rem}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt .ot-sel-all .ot-chkbox{right:0}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) .ot-sel-all{padding-right:34px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) .ot-sel-all-chkbox{width:auto}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) ul li{border:1px solid #e2e2e2;margin-bottom:10px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) .ot-acc-cntr>.ot-acc-hdr{padding:10px 0 10px 15px}#onetrust-pc-sdk.ot-addtl-vendors .ot-sel-all-chkbox{float:right}#onetrust-pc-sdk.ot-addtl-vendors .ot-plus-minus~.ot-sel-all-chkbox{right:34px}#onetrust-pc-sdk.ot-addtl-vendors #ot-ven-lst:first-child{border-top:none}#onetrust-pc-sdk .ot-acc-cntr{position:relative;border-left:1px solid #e2e2e2;border-right:1px solid #e2e2e2;border-bottom:1px solid #e2e2e2}#onetrust-pc-sdk .ot-acc-cntr input{z-index:1}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr{background-color:#f9f9fc;padding:5px 0 5px 15px;width:auto}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr .ot-plus-minus{vertical-align:middle;top:auto}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr .ot-arw-cntr{right:10px}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr input{z-index:2}#onetrust-pc-sdk .ot-acc-cntr.ot-add-tech .ot-acc-hdr{padding:10px 0 10px 15px}#onetrust-pc-sdk .ot-acc-cntr>input[type=checkbox]:checked~.ot-acc-hdr{border-bottom:1px solid #e2e2e2}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-txt{padding-left:10px;padding-right:10px}#onetrust-pc-sdk .ot-acc-cntr button[aria-expanded=true]~.ot-acc-txt{width:auto}#onetrust-pc-sdk .ot-acc-cntr .ot-addtl-venbox{display:none}#onetrust-pc-sdk .ot-vlst-cntr{margin-bottom:0;width:100%}#onetrust-pc-sdk .ot-vensec-title{font-size:.813em;vertical-align:middle;display:inline-block}#onetrust-pc-sdk .category-vendors-list-handler,#onetrust-pc-sdk .category-vendors-list-handler+a{margin-left:0;margin-top:10px}#onetrust-pc-sdk #ot-selall-vencntr.line-through label::after,#onetrust-pc-sdk #ot-selall-adtlvencntr.line-through label::after,#onetrust-pc-sdk #ot-selall-licntr.line-through label::after,#onetrust-pc-sdk #ot-selall-hostcntr.line-through label::after,#onetrust-pc-sdk #ot-selall-gnvencntr.line-through label::after{height:auto;border-left:0;transform:none;-o-transform:none;-ms-transform:none;-webkit-transform:none;left:5px;top:9px}#onetrust-pc-sdk #ot-category-title{float:left;padding-bottom:10px;font-size:1em;width:100%}#onetrust-pc-sdk .ot-cat-grp{margin-top:10px}#onetrust-pc-sdk .ot-cat-item{line-height:1.1;margin-top:10px;display:inline-block;width:100%}#onetrust-pc-sdk .ot-btn-container{text-align:right}#onetrust-pc-sdk .ot-btn-container button{display:inline-block;font-size:.75em;letter-spacing:.08em;margin-top:19px}#onetrust-pc-sdk #close-pc-btn-handler.ot-close-icon{position:absolute;top:10px;right:0;z-index:1;padding:0;background-color:rgba(0,0,0,0);border:none}#onetrust-pc-sdk #close-pc-btn-handler.ot-close-icon svg{display:block;height:10px;width:10px}#onetrust-pc-sdk #clear-filters-handler{margin-top:20px;margin-bottom:10px;float:right;max-width:200px;text-decoration:none;color:#3860be;font-size:.9em;font-weight:bold;background-color:rgba(0,0,0,0);border-color:rgba(0,0,0,0);padding:1px}#onetrust-pc-sdk #clear-filters-handler:hover{color:#2285f7}#onetrust-pc-sdk #clear-filters-handler:focus{outline:#000 solid 1px}#onetrust-pc-sdk .ot-enbl-chr h4~.ot-tgl,#onetrust-pc-sdk .ot-enbl-chr h4~.ot-always-active{right:45px}#onetrust-pc-sdk .ot-enbl-chr h4~.ot-tgl+.ot-tgl{right:120px}#onetrust-pc-sdk .ot-enbl-chr .ot-pli-hdr.ot-leg-border-color span:first-child{width:90px}#onetrust-pc-sdk .ot-enbl-chr li.ot-subgrp>h5+.ot-tgl-cntr{padding-right:25px}#onetrust-pc-sdk .ot-plus-minus{width:20px;height:20px;font-size:1.5em;position:relative;display:inline-block;margin-right:5px;top:3px}#onetrust-pc-sdk .ot-plus-minus span{position:absolute;background:#27455c;border-radius:1px}#onetrust-pc-sdk .ot-plus-minus span:first-of-type{top:25%;bottom:25%;width:10%;left:45%}#onetrust-pc-sdk .ot-plus-minus span:last-of-type{left:25%;right:25%;height:10%;top:45%}#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-arw,#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-plus-minus span:first-of-type,#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-plus-minus span:last-of-type{transform:rotate(90deg)}#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-plus-minus span:last-of-type{left:50%;right:50%}#onetrust-pc-sdk #ot-selall-vencntr label,#onetrust-pc-sdk #ot-selall-adtlvencntr label,#onetrust-pc-sdk #ot-selall-hostcntr label,#onetrust-pc-sdk #ot-selall-licntr label{position:relative;display:inline-block;width:20px;height:20px}#onetrust-pc-sdk .ot-host-item .ot-plus-minus,#onetrust-pc-sdk .ot-ven-item .ot-plus-minus{float:left;margin-right:8px;top:10px}#onetrust-pc-sdk .ot-ven-item ul{list-style:none inside;font-size:100%;margin:0}#onetrust-pc-sdk .ot-ven-item ul li{margin:0 !important;padding:0;border:none !important}#onetrust-pc-sdk .ot-pli-hdr{color:#77808e;overflow:hidden;padding-top:7.5px;padding-bottom:7.5px;width:calc(100% - 2px);border-top-left-radius:3px;border-top-right-radius:3px}#onetrust-pc-sdk .ot-pli-hdr span:first-child{top:50%;transform:translateY(50%);max-width:90px}#onetrust-pc-sdk .ot-pli-hdr span:last-child{padding-right:10px;max-width:95px;text-align:center}#onetrust-pc-sdk .ot-li-title{float:right;font-size:.813em}#onetrust-pc-sdk .ot-pli-hdr.ot-leg-border-color{background-color:#f4f4f4;border:1px solid #d8d8d8}#onetrust-pc-sdk .ot-pli-hdr.ot-leg-border-color span:first-child{text-align:left;width:70px}#onetrust-pc-sdk li.ot-subgrp>h5,#onetrust-pc-sdk .ot-cat-header{width:calc(100% - 130px)}#onetrust-pc-sdk li.ot-subgrp>h5+.ot-tgl-cntr{padding-left:13px}#onetrust-pc-sdk .ot-acc-grpcntr .ot-acc-grpdesc{margin-bottom:5px}#onetrust-pc-sdk .ot-acc-grpcntr .ot-subgrp-cntr{border-top:1px solid #d8d8d8}#onetrust-pc-sdk .ot-acc-grpcntr .ot-vlst-cntr+.ot-subgrp-cntr{border-top:none}#onetrust-pc-sdk .ot-acc-hdr .ot-arw-cntr+.ot-tgl-cntr,#onetrust-pc-sdk .ot-acc-txt h4+.ot-tgl-cntr{padding-left:13px}#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item .ot-subgrp>h5,#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item .ot-cat-header{width:calc(100% - 145px)}#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item h5+.ot-tgl-cntr,#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item .ot-cat-header+.ot-tgl{padding-left:28px}#onetrust-pc-sdk .ot-sel-all-hdr,#onetrust-pc-sdk .ot-sel-all-chkbox{display:inline-block;width:100%;position:relative}#onetrust-pc-sdk .ot-sel-all-chkbox{z-index:1}#onetrust-pc-sdk .ot-sel-all{margin:0;position:relative;padding-right:23px;float:right}#onetrust-pc-sdk .ot-consent-hdr,#onetrust-pc-sdk .ot-li-hdr{float:right;font-size:.812em;line-height:normal;text-align:center;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-li-hdr{max-width:100px;padding-right:10px}#onetrust-pc-sdk .ot-consent-hdr{max-width:55px}#onetrust-pc-sdk #ot-selall-licntr{display:block;width:21px;height:auto;float:right;position:relative;right:80px}#onetrust-pc-sdk #ot-selall-licntr label{position:absolute}#onetrust-pc-sdk .ot-ven-ctgl{margin-left:66px}#onetrust-pc-sdk .ot-ven-litgl+.ot-arw-cntr{margin-left:81px}#onetrust-pc-sdk .ot-enbl-chr .ot-host-cnt .ot-tgl-cntr{width:auto}#onetrust-pc-sdk #ot-lst-cnt:not(.ot-host-cnt) .ot-tgl-cntr{width:auto;top:auto;height:20px}#onetrust-pc-sdk #ot-lst-cnt .ot-chkbox{position:relative;display:inline-block;width:20px;height:20px}#onetrust-pc-sdk #ot-lst-cnt .ot-chkbox label{position:absolute;padding:0;width:20px;height:20px}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info-cntr{border:1px solid #d8d8d8;padding:.75rem 2rem;padding-bottom:0;width:auto;margin-top:.5rem}#onetrust-pc-sdk .ot-acc-grpdesc+.ot-leg-btn-container{padding-left:20px;padding-right:20px;width:calc(100% - 40px);margin-bottom:5px}#onetrust-pc-sdk .ot-subgrp .ot-leg-btn-container{margin-bottom:5px}#onetrust-pc-sdk #ot-ven-lst .ot-leg-btn-container{margin-top:10px}#onetrust-pc-sdk .ot-leg-btn-container{display:inline-block;width:100%;margin-bottom:10px}#onetrust-pc-sdk .ot-leg-btn-container button{height:auto;padding:6.5px 8px;margin-bottom:0;letter-spacing:0;font-size:.75em;line-height:normal}#onetrust-pc-sdk .ot-leg-btn-container svg{display:none;height:14px;width:14px;padding-right:5px;vertical-align:sub}#onetrust-pc-sdk .ot-active-leg-btn{cursor:default;pointer-events:none}#onetrust-pc-sdk .ot-active-leg-btn svg{display:inline-block}#onetrust-pc-sdk .ot-remove-objection-handler{text-decoration:underline;padding:0;font-size:.75em;font-weight:600;line-height:1;padding-left:10px}#onetrust-pc-sdk .ot-obj-leg-btn-handler span{font-weight:bold;text-align:center;font-size:inherit;line-height:1.5}#onetrust-pc-sdk.ot-close-btn-link #close-pc-btn-handler{border:none;height:auto;line-height:1.5;text-decoration:underline;font-size:.69em;background:none;right:15px;top:15px;width:auto;font-weight:normal}#onetrust-pc-sdk .ot-pgph-link{font-size:.813em !important;margin-top:5px;position:relative}#onetrust-pc-sdk .ot-pgph-link.ot-pgph-link-subgroup{margin-bottom:1rem}#onetrust-pc-sdk .ot-pgph-contr{margin:0 2.5rem}#onetrust-pc-sdk .ot-pgph-title{font-size:1.18rem;margin-bottom:2rem}#onetrust-pc-sdk .ot-pgph-desc{font-size:1rem;font-weight:400;margin-bottom:2rem;line-height:1.5rem}#onetrust-pc-sdk .ot-pgph-desc:not(:last-child):after{content:"";width:96%;display:block;margin:0 auto;padding-bottom:2rem;border-bottom:1px solid #e9e9e9}#onetrust-pc-sdk .ot-cat-header{float:left;font-weight:600;font-size:.875em;line-height:1.5;max-width:90%;vertical-align:middle}#onetrust-pc-sdk .ot-vnd-item>button:focus{outline:#000 solid 2px}#onetrust-pc-sdk .ot-vnd-item>button{position:absolute;cursor:pointer;width:100%;height:100%;margin:0;top:0;left:0;z-index:1;max-width:none;border:none}#onetrust-pc-sdk .ot-vnd-item>button[aria-expanded=false]~.ot-acc-txt{margin-top:0;max-height:0;opacity:0;overflow:hidden;width:100%;transition:.25s ease-out;display:none}#onetrust-pc-sdk .ot-vnd-item>button[aria-expanded=true]~.ot-acc-txt{transition:.1s ease-in;margin-top:10px;width:100%;overflow:auto;display:block}#onetrust-pc-sdk .ot-vnd-item>button[aria-expanded=true]~.ot-acc-grpcntr{width:auto;margin-top:0px;padding-bottom:10px}#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item{position:relative;border-radius:2px;margin:0;padding:0;border:1px solid #d8d8d8;border-top:none;width:calc(100% - 2px);float:left}#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item:first-of-type{margin-top:10px;border-top:1px solid #d8d8d8}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpdesc{padding-left:20px;padding-right:20px;width:calc(100% - 40px);font-size:.812em;margin-bottom:10px;margin-top:15px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpdesc>ul{padding-top:10px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpdesc>ul li{padding-top:0;line-height:1.5;padding-bottom:10px}#onetrust-pc-sdk .ot-accordion-layout div+.ot-acc-grpdesc{margin-top:5px}#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr:first-child{margin-top:10px}#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr:last-child,#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr:last-child{margin-bottom:5px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{padding-top:11.5px;padding-bottom:11.5px;padding-left:20px;padding-right:20px;width:calc(100% - 40px);display:inline-block}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-txt{width:100%;padding:0}#onetrust-pc-sdk .ot-accordion-layout .ot-subgrp-cntr{padding-left:20px;padding-right:15px;padding-bottom:0;width:calc(100% - 35px)}#onetrust-pc-sdk .ot-accordion-layout .ot-subgrp{padding-right:5px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpcntr{z-index:1;position:relative}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header+.ot-arw-cntr{position:absolute;top:50%;transform:translateY(-50%);right:20px;margin-top:-2px}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header+.ot-arw-cntr .ot-arw{width:15px;height:20px;margin-left:5px;color:dimgray}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{float:none;color:#2e3644;margin:0;display:inline-block;height:auto;word-wrap:break-word;min-height:inherit}#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr,#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr{padding-left:20px;width:calc(100% - 20px);display:inline-block;margin-top:0;padding-bottom:2px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{position:relative;min-height:25px}#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl,#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active{position:absolute;top:50%;transform:translateY(-50%);right:20px}#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl+.ot-tgl{right:95px}#onetrust-pc-sdk .ot-accordion-layout .category-vendors-list-handler,#onetrust-pc-sdk .ot-accordion-layout .category-vendors-list-handler+a{margin-top:5px}#onetrust-pc-sdk #ot-lst-cnt{margin-top:1rem;max-height:calc(100% - 96px)}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info-cntr{border:1px solid #d8d8d8;padding:.75rem 2rem;padding-bottom:0;width:auto;margin-top:.5rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info{margin-bottom:1rem;padding-left:.75rem;padding-right:.75rem;display:flex;flex-direction:column}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info[data-vnd-info-key*=DPOEmail]{border-top:1px solid #d8d8d8;padding-top:1rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info[data-vnd-info-key*=DPOLink]{border-bottom:1px solid #d8d8d8;padding-bottom:1rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info .ot-vnd-lbl{font-weight:bold;font-size:.85em;margin-bottom:.5rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info .ot-vnd-cnt{margin-left:.5rem;font-weight:500;font-size:.85rem}#onetrust-pc-sdk .ot-vs-list,#onetrust-pc-sdk .ot-vnd-serv{width:auto;padding:1rem 1.25rem;padding-bottom:0}#onetrust-pc-sdk .ot-vs-list .ot-vnd-serv-hdr-cntr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-serv-hdr-cntr{padding-bottom:.75rem;border-bottom:1px solid #d8d8d8}#onetrust-pc-sdk .ot-vs-list .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr{font-weight:600;font-size:.95em;line-height:2;margin-left:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item{border:none;margin:0;padding:0}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item button,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item button{outline:none;border-bottom:1px solid #d8d8d8}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item button[aria-expanded=true],#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item button[aria-expanded=true]{border-bottom:none}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item:first-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item:first-child{margin-top:.25rem;border-top:unset}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item:last-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item:last-child{margin-bottom:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item:last-child button,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item:last-child button{border-bottom:none}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info-cntr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info-cntr{border:1px solid #d8d8d8;padding:.75rem 1.75rem;padding-bottom:0;width:auto;margin-top:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info{margin-bottom:1rem;padding-left:.75rem;padding-right:.75rem;display:flex;flex-direction:column}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOEmail],#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOEmail]{border-top:1px solid #d8d8d8;padding-top:1rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOLink],#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOLink]{border-bottom:1px solid #d8d8d8;padding-bottom:1rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info .ot-vnd-lbl,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info .ot-vnd-lbl{font-weight:bold;font-size:.85em;margin-bottom:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info .ot-vnd-cnt,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info .ot-vnd-cnt{margin-left:.5rem;font-weight:500;font-size:.85rem}#onetrust-pc-sdk .ot-vs-list.ot-vnd-subgrp-cnt,#onetrust-pc-sdk .ot-vnd-serv.ot-vnd-subgrp-cnt{padding-left:40px}#onetrust-pc-sdk .ot-vs-list.ot-vnd-subgrp-cnt .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr,#onetrust-pc-sdk .ot-vnd-serv.ot-vnd-subgrp-cnt .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr{font-size:.8em}#onetrust-pc-sdk .ot-vs-list.ot-vnd-subgrp-cnt .ot-cat-header,#onetrust-pc-sdk .ot-vnd-serv.ot-vnd-subgrp-cnt .ot-cat-header{font-size:.8em}#onetrust-pc-sdk .ot-subgrp-cntr .ot-vnd-serv{margin-bottom:1rem;padding:1rem .95rem}#onetrust-pc-sdk .ot-subgrp-cntr .ot-vnd-serv .ot-vnd-serv-hdr-cntr{padding-bottom:.75rem;border-bottom:1px solid #d8d8d8}#onetrust-pc-sdk .ot-subgrp-cntr .ot-vnd-serv .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr{font-weight:700;font-size:.8em;line-height:20px;margin-left:.82rem}#onetrust-pc-sdk .ot-subgrp-cntr .ot-cat-header{font-weight:700;font-size:.8em;line-height:20px}#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-vnd-serv .ot-vnd-lst-cont .ot-accordion-layout .ot-acc-hdr div.ot-chkbox{margin-left:.82rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr{padding:.7rem 0;margin:0;display:flex;width:100%;align-items:center;justify-content:space-between}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr div:first-child,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr div:first-child,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr div:first-child{margin-left:.5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr div:last-child,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr div:last-child,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr div:last-child{margin-right:.5rem;margin-left:.5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-always-active{position:relative;right:unset;top:unset;transform:unset}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-plus-minus{top:0}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-arw-cntr{float:none;top:unset;right:unset;transform:unset;margin-top:-2px;position:relative}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-cat-header{flex:1;margin:0 .5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-tgl{position:relative;transform:none;right:0;top:0;float:none}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox{position:relative;margin:0 .5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox label{padding:0}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox label::before{position:relative}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox input{position:absolute;cursor:pointer;width:100%;height:100%;opacity:0;margin:0;top:0;left:0;z-index:1}#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp .ot-acc-hdr h5.ot-cat-header,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp .ot-acc-hdr h4.ot-cat-header{margin:0}#onetrust-pc-sdk .ot-vs-config .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp h5{top:0;line-height:20px}#onetrust-pc-sdk .ot-vs-list{display:flex;flex-direction:column;padding:0;margin:.5rem 4px}#onetrust-pc-sdk .ot-vs-selc-all{display:flex;padding:0;float:unset;align-items:center;justify-content:flex-start}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf{justify-content:flex-end}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf.ot-caret-conf .ot-sel-all-chkbox{margin-right:48px}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf .ot-sel-all-chkbox{margin:0;padding:0;margin-right:14px;justify-content:flex-end}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf #ot-selall-vencntr.ot-chkbox,#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf #ot-selall-vencntr.ot-tgl{display:inline-block;right:unset;width:auto;height:auto;float:none}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf #ot-selall-vencntr label{width:45px;height:25px}#onetrust-pc-sdk .ot-vs-selc-all .ot-sel-all-chkbox{margin-right:11px;margin-left:.75rem;display:flex;align-items:center}#onetrust-pc-sdk .ot-vs-selc-all .sel-all-hdr{margin:0 1.25rem;font-size:.812em;line-height:normal;text-align:center;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-vnd-list-cnt #ot-selall-vencntr.ot-chkbox{float:unset;right:0}#onetrust-pc-sdk[dir=rtl] #ot-back-arw,#onetrust-pc-sdk[dir=rtl] input~.ot-acc-hdr .ot-arw{transform:rotate(180deg);-o-transform:rotate(180deg);-ms-transform:rotate(180deg);-webkit-transform:rotate(180deg)}#onetrust-pc-sdk[dir=rtl] input:checked~.ot-acc-hdr .ot-arw{transform:rotate(270deg);-o-transform:rotate(270deg);-ms-transform:rotate(270deg);-webkit-transform:rotate(270deg)}#onetrust-pc-sdk[dir=rtl] .ot-chkbox label::after{transform:rotate(45deg);-webkit-transform:rotate(45deg);-o-transform:rotate(45deg);-ms-transform:rotate(45deg);border-left:0;border-right:3px solid}#onetrust-pc-sdk[dir=rtl] .ot-search-cntr>svg{right:0}@media only screen and (max-width: 600px){#onetrust-pc-sdk.otPcCenter{left:0;min-width:100%;height:100%;top:0;border-radius:0}#onetrust-pc-sdk #ot-pc-content,#onetrust-pc-sdk.ot-ftr-stacked .ot-btn-container{margin:1px 3px 0 10px;padding-right:10px;width:calc(100% - 23px)}#onetrust-pc-sdk .ot-btn-container button{max-width:none;letter-spacing:.01em}#onetrust-pc-sdk #close-pc-btn-handler{top:10px;right:17px}#onetrust-pc-sdk p{font-size:.7em}#onetrust-pc-sdk #ot-pc-hdr{margin:10px 10px 0 5px;width:calc(100% - 15px)}#onetrust-pc-sdk .vendor-search-handler{font-size:1em}#onetrust-pc-sdk #ot-back-arw{margin-left:12px}#onetrust-pc-sdk #ot-lst-cnt{margin:0;padding:0 5px 0 10px;min-width:95%}#onetrust-pc-sdk .switch+p{max-width:80%}#onetrust-pc-sdk .ot-ftr-stacked button{width:100%}#onetrust-pc-sdk #ot-fltr-cnt{max-width:320px;width:90%;border-top-right-radius:0;border-bottom-right-radius:0;margin:0;margin-left:15px;left:auto;right:40px;top:85px}#onetrust-pc-sdk .ot-fltr-opt{margin-left:25px;margin-bottom:10px}#onetrust-pc-sdk .ot-pc-refuse-all-handler{margin-bottom:0}#onetrust-pc-sdk #ot-fltr-cnt{right:40px}}@media only screen and (max-width: 476px){#onetrust-pc-sdk .ot-fltr-cntr,#onetrust-pc-sdk #ot-fltr-cnt{right:10px}#onetrust-pc-sdk #ot-anchor{right:25px}#onetrust-pc-sdk button{width:100%}#onetrust-pc-sdk:not(.ot-addtl-vendors) #ot-pc-lst:not(.ot-enbl-chr) .ot-sel-all{padding-right:9px}#onetrust-pc-sdk:not(.ot-addtl-vendors) #ot-pc-lst:not(.ot-enbl-chr) .ot-tgl-cntr{right:0}}@media(min-width: 768px){#onetrust-pc-sdk.ot-tgl-with-label .ot-label-status{display:inline}#onetrust-pc-sdk.ot-tgl-with-label #ot-pc-lst .ot-label-status{display:none}}@media only screen and (max-width: 896px)and (max-height: 425px)and (orientation: landscape){#onetrust-pc-sdk.otPcCenter{left:0;top:0;min-width:100%;height:100%;border-radius:0}#onetrust-pc-sdk .ot-pc-header{height:auto;min-height:20px}#onetrust-pc-sdk .ot-pc-header .ot-pc-logo{max-height:30px}#onetrust-pc-sdk .ot-pc-footer{max-height:60px;overflow-y:auto}#onetrust-pc-sdk #ot-pc-content,#onetrust-pc-sdk #ot-pc-lst{bottom:70px}#onetrust-pc-sdk.ot-ftr-stacked #ot-pc-content{bottom:70px}#onetrust-pc-sdk #ot-anchor{left:initial;right:50px}#onetrust-pc-sdk #ot-lst-title{margin-top:12px}#onetrust-pc-sdk #ot-lst-title *{font-size:inherit}#onetrust-pc-sdk #ot-pc-hdr input{margin-right:0;padding-right:45px}#onetrust-pc-sdk .switch+p{max-width:85%}#onetrust-pc-sdk #ot-sel-blk{position:static}#onetrust-pc-sdk #ot-pc-lst{overflow:auto}#onetrust-pc-sdk #ot-lst-cnt{max-height:none;overflow:initial}#onetrust-pc-sdk #ot-lst-cnt.no-results{height:auto}#onetrust-pc-sdk input{font-size:1em !important}#onetrust-pc-sdk p{font-size:.6em}#onetrust-pc-sdk #ot-fltr-modal{width:100%;top:0}#onetrust-pc-sdk ul li p,#onetrust-pc-sdk .category-vendors-list-handler,#onetrust-pc-sdk .category-vendors-list-handler+a,#onetrust-pc-sdk .category-host-list-handler{font-size:.6em}#onetrust-pc-sdk.ot-shw-fltr #ot-anchor{display:none !important}#onetrust-pc-sdk.ot-shw-fltr #ot-pc-lst{height:100% !important;overflow:hidden;top:0px}#onetrust-pc-sdk.ot-shw-fltr #ot-fltr-cnt{margin:0;height:100%;max-height:none;padding:10px;top:0;width:calc(100% - 20px);position:absolute;right:0;left:0;max-width:none}#onetrust-pc-sdk.ot-shw-fltr .ot-fltr-scrlcnt{max-height:calc(100% - 65px)}}
            #onetrust-consent-sdk #onetrust-pc-sdk,
                #onetrust-consent-sdk #ot-search-cntr,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-switch.ot-toggle,
                #onetrust-consent-sdk #onetrust-pc-sdk ot-grp-hdr1 .checkbox,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-title:after
                ,#onetrust-consent-sdk #onetrust-pc-sdk #ot-sel-blk,
                        #onetrust-consent-sdk #onetrust-pc-sdk #ot-fltr-cnt,
                        #onetrust-consent-sdk #onetrust-pc-sdk #ot-anchor {
                    background-color: #FFF;
                }
               
            #onetrust-consent-sdk #onetrust-pc-sdk h3,
                #onetrust-consent-sdk #onetrust-pc-sdk h4,
                #onetrust-consent-sdk #onetrust-pc-sdk h5,
                #onetrust-consent-sdk #onetrust-pc-sdk h6,
                #onetrust-consent-sdk #onetrust-pc-sdk p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-ven-lst .ot-ven-opts p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-desc,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-title,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-li-title,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-sel-all-hdr span,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-info,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-fltr-modal #modal-header,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-checkbox label span,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst #ot-sel-blk p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst #ot-lst-title h3,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst .back-btn-handler p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst .ot-ven-name,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst #ot-ven-lst .consent-category,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-inactive-leg-btn,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-label-status,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-chkbox label span,
                #onetrust-consent-sdk #onetrust-pc-sdk #clear-filters-handler,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-optout-signal
                {
                    color: #2E2E2E;
                }
             #onetrust-consent-sdk #onetrust-pc-sdk .privacy-notice-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-pgph-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk .category-vendors-list-handler,
                    #onetrust-consent-sdk #onetrust-pc-sdk .category-vendors-list-handler + a,
                    #onetrust-consent-sdk #onetrust-pc-sdk .category-host-list-handler,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-ven-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-ven-legclaim-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-name a,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-acc-hdr .ot-host-expand,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-info a,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-content #ot-pc-desc .ot-link-btn,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info a,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info a
                    {
                        color: #007398;
                    }
            #onetrust-consent-sdk #onetrust-pc-sdk .category-vendors-list-handler:hover { text-decoration: underline;}
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-grpcntr.ot-acc-txt,
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-txt .ot-subgrp-tgl .ot-switch.ot-toggle
             {
                background-color: #F8F8F8;
            }
             #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-info,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-txt .ot-ven-dets
                            {
                                background-color: #F8F8F8;
                            }
        #onetrust-consent-sdk #onetrust-pc-sdk
            button:not(#clear-filters-handler):not(.ot-close-icon):not(#filter-btn-handler):not(.ot-remove-objection-handler):not(.ot-obj-leg-btn-handler):not([aria-expanded]):not(.ot-link-btn),
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-active-leg-btn {
                background-color: #007398;border-color: #007398;
                color: #FFF;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-active-menu {
                border-color: #007398;
            }
            
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-remove-objection-handler{
                background-color: transparent;
                border: 1px solid transparent;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-inactive-leg-btn {
                background-color: #FFFFFF;
                color: #4D4D4D; border-color: #4D4D4D;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-tgl input:focus + .ot-switch, .ot-switch .ot-switch-nob, .ot-switch .ot-switch-nob:before,
            #onetrust-pc-sdk .ot-checkbox input[type="checkbox"]:focus + label::before,
            #onetrust-pc-sdk .ot-chkbox input[type="checkbox"]:focus + label::before {
                outline-color: #000000;
                outline-width: 1px;
                outline-offset: 1px;
            }
            #onetrust-pc-sdk .ot-host-item > button:focus, #onetrust-pc-sdk .ot-ven-item > button:focus {
                border: 1px solid #000000;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk *:focus,
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-vlst-cntr > a:focus {
               outline: 1px solid #000000;
               outline-offset: 1px;
            }#onetrust-pc-sdk .ot-vlst-cntr .ot-ext-lnk,  #onetrust-pc-sdk .ot-ven-hdr .ot-ext-lnk{
                    background-image: url('https://cdn.cookielaw.org/logos/static/ot_external_link.svg');
                }
            /*! v6.10.0 2021-01-18 */
div#onetrust-consent-sdk #onetrust-banner-sdk{border-top:2px solid #d0d3d4!important;box-shadow:none;padding:24px}div#onetrust-consent-sdk button{border-radius:0!important;box-shadow:none!important;box-sizing:border-box!important;font-size:20px!important;font-weight:400!important;letter-spacing:0!important;max-width:none!important;white-space:nowrap!important}div#onetrust-consent-sdk button:not(.ot-link-btn){background-color:#007dbc!important;border:2px solid #007dbc!important;color:#fff!important;height:48px!important;padding:0 1em!important;width:auto!important}div#onetrust-consent-sdk button:hover{background-color:#fff!important;border-color:#949494!important;color:#2e2e2e!important}div#onetrust-consent-sdk button.ot-link-btn{color:#007dbc!important;font-size:16px!important;text-decoration:underline}div#onetrust-consent-sdk button.ot-link-btn:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-consent-sdk a,div#onetrust-pc-sdk a{color:#007dbc!important;text-decoration:underline!important}div#onetrust-consent-sdk a,div#onetrust-consent-sdk button,div#onetrust-consent-sdk p:hover{opacity:1!important}div#onetrust-consent-sdk a:focus,div#onetrust-consent-sdk button:focus,div#onetrust-consent-sdk input:focus{outline:2px solid #949494!important;outline-offset:1px!important}div#onetrust-banner-sdk .ot-sdk-container{padding:0;width:auto}div#onetrust-banner-sdk .ot-sdk-row{align-items:flex-start;box-sizing:border-box;display:flex;flex-direction:column;justify-content:space-between;margin:auto;max-width:1152px}div#onetrust-banner-sdk .ot-sdk-row:after{display:none}div#onetrust-banner-sdk #onetrust-group-container,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-group-container,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{flex-grow:1;width:auto}div#onetrust-banner-sdk #onetrust-policy,div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy{margin:0;overflow:visible}div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy-text,div#onetrust-consent-sdk #onetrust-policy-text{font-size:16px;line-height:24px;max-width:44em;margin:0}div#onetrust-consent-sdk #onetrust-policy-text a[href]{font-weight:400;margin-left:8px}div#onetrust-banner-sdk #onetrust-button-group-parent{flex:0 0 auto;margin:32px 0 0;width:100%}div#onetrust-banner-sdk #onetrust-button-group{display:flex;flex-direction:row;flex-wrap:wrap;justify-content:flex-end;margin:-8px}div#onetrust-banner-sdk .banner-actions-container{display:flex;flex:1 0 auto}div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button:last-of-type,div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto;margin:8px;width:auto}div#onetrust-consent-sdk #onetrust-pc-btn-handler{background-color:#fff!important;color:inherit!important}div#onetrust-banner-sdk #onetrust-close-btn-container{display:none}@media only screen and (min-width:556px){div#onetrust-consent-sdk #onetrust-banner-sdk{padding:40px}div#onetrust-banner-sdk #onetrust-policy{margin:0 40px 0 0}div#onetrust-banner-sdk .ot-sdk-row{align-items:center;flex-direction:row}div#onetrust-banner-sdk #onetrust-button-group-parent,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-button-group-parent,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{margin:0;padding:0;width:auto}div#onetrust-banner-sdk #onetrust-button-group,div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group{align-items:stretch;flex-direction:column-reverse;margin:0}div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto}}@media only screen and (min-width:768px){div#onetrust-banner-sdk #onetrust-policy{margin:0 48px 0 0}div#onetrust-consent-sdk #onetrust-banner-sdk{padding:48px}}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-size:16px;line-height:24px}div#onetrust-consent-sdk #onetrust-pc-sdk p,div#onetrust-pc-sdk #ot-pc-desc,div#onetrust-pc-sdk .category-host-list-handler,div#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{font-size:16px;font-weight:400;line-height:24px}div#onetrust-consent-sdk a:hover,div#onetrust-pc-sdk a:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-pc-sdk{border-radius:0;bottom:0;height:auto;left:0;margin:auto;max-width:100%;overflow:hidden;right:0;top:0;width:512px;max-height:800px}div#onetrust-pc-sdk .ot-pc-header{display:none}div#onetrust-pc-sdk #ot-pc-content{overscroll-behavior:contain;padding:0 12px 0 24px;margin:16px 4px 0 0;top:0;right:16px;left:0;width:auto}div#onetrust-pc-sdk #ot-category-title,div#onetrust-pc-sdk #ot-pc-title{font-size:24px;font-weight:400;line-height:32px;margin:0 0 16px}div#onetrust-pc-sdk #ot-pc-desc{padding:0}div#onetrust-pc-sdk #ot-pc-desc a{display:inline}div#onetrust-pc-sdk #accept-recommended-btn-handler{display:none!important}div#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:2px solid #007dbc;outline-offset:-1px;transition:none}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item{border-width:0 0 2px}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item:first-of-type{border-width:2px 0}div#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{padding:8px 0;width:100%}div#onetrust-pc-sdk .ot-plus-minus{transform:translateY(2px)}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item>button{background:0 0!important;border:0!important;height:44px!important;max-width:none!important;width:calc(100% - 48px)!important}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-weight:700}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr{padding:0}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item .ot-acc-grpdesc{padding:0;width:100%}div#onetrust-pc-sdk .ot-acc-grpcntr .ot-subgrp-cntr{border:0;padding:0}div#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp{margin:0}div#onetrust-pc-sdk .ot-always-active-group .ot-cat-header{width:calc(100% - 160px)}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{width:calc(100% - 88px)}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active{color:inherit;font-size:12px;font-weight:400;line-height:1.5;padding-right:48px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:before{border-radius:12px;position:absolute;right:0;top:0;content:'';background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:after{border-radius:50%;position:absolute;right:5px;top:4px;content:'';background-color:#007dbc;height:12px;width:12px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active,div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl{right:2px}div#onetrust-pc-sdk .ot-switch{display:block;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob,div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob{background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob:before{background-color:#737373;height:8px;left:4px;top:4px;width:8px}div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob:before{background-color:#007dbc;height:12px;left:0;top:2px;width:12px}div#onetrust-pc-sdk .ot-tgl input:focus+.ot-switch .ot-switch-nob{box-shadow:0 0;outline:2px solid #007dbc!important;outline-offset:1px;transition:none}div#onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-grpcntr.ot-acc-txt{background-color:transparent;padding-left:3px}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr,div#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr{overflow:visible;width:100%}div#onetrust-pc-sdk .ot-pc-footer{border-top:0 solid}div#onetrust-pc-sdk .ot-btn-container{padding-top:24px;text-align:center}div#onetrust-pc-sdk .ot-pc-footer button{margin:8px 0;background-color:#fff}div#onetrust-pc-sdk .ot-pc-footer-logo{background-color:#fff}div#onetrust-pc-sdk #ot-lst-title span{font-size:24px;font-weight:400;line-height:32px}div#onetrust-pc-sdk #ot-host-lst .ot-host-desc,div#onetrust-pc-sdk #ot-host-lst .ot-host-expand,div#onetrust-pc-sdk #ot-host-lst .ot-host-name,div#onetrust-pc-sdk #ot-host-lst .ot-host-name a,div#onetrust-pc-sdk .back-btn-handler,div#onetrust-pc-sdk .ot-host-opt li>div div{font-size:16px;font-weight:400;line-height:24px}div#onetrust-pc-sdk #ot-pc-lst{top:0}div#onetrust-pc-sdk .back-btn-handler{text-decoration:none!important}div#onetrust-pc-sdk #filter-btn-handler:hover svg{filter:invert(1)}div#onetrust-pc-sdk .back-btn-handler svg{width:16px;height:16px}div#onetrust-pc-sdk .ot-host-item>button{background:0 0!important;border:0!important;height:66px!important;max-width:none!important;width:calc(100% - 5px)!important;transform:translate(2px,2px)}div#onetrust-pc-sdk .ot-host-item{border-bottom:2px solid #b9b9b9;padding:0}div#onetrust-pc-sdk .ot-host-item .ot-acc-hdr{margin:0 0 -6px;padding:8px 0}div#onetrust-pc-sdk ul li:first-child{border-top:2px solid #b9b9b9}div#onetrust-pc-sdk .ot-host-item .ot-plus-minus{margin:0 8px 0 0}div#onetrust-pc-sdk .ot-search-cntr{width:calc(100% - 48px)}div#onetrust-pc-sdk .ot-host-opt .ot-host-info{background-color:transparent}div#onetrust-pc-sdk .ot-host-opt li>div div{padding:0}div#onetrust-pc-sdk #vendor-search-handler{border-radius:0;border-color:#939393;border-style:solid;border-width:2px 0 2px 2px;font-size:20px;height:48px;margin:0}div#onetrust-pc-sdk .ot-lst-subhdr svg{right:0;top:8px}div#onetrust-pc-sdk .ot-fltr-cntr{box-sizing:border-box;right:0;width:48px}div#onetrust-pc-sdk #filter-btn-handler{width:48px!important;padding:8px!important}div#onetrust-pc-sdk #ot-fltr-modal{max-height:400px}
.ot-sdk-cookie-policy{font-family:inherit;font-size:16px}.ot-sdk-cookie-policy.otRelFont{font-size:1rem}.ot-sdk-cookie-policy h3,.ot-sdk-cookie-policy h4,.ot-sdk-cookie-policy h6,.ot-sdk-cookie-policy p,.ot-sdk-cookie-policy li,.ot-sdk-cookie-policy a,.ot-sdk-cookie-policy th,.ot-sdk-cookie-policy #cookie-policy-description,.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group,.ot-sdk-cookie-policy #cookie-policy-title{color:dimgray}.ot-sdk-cookie-policy #cookie-policy-description{margin-bottom:1em}.ot-sdk-cookie-policy h4{font-size:1.2em}.ot-sdk-cookie-policy h6{font-size:1em;margin-top:2em}.ot-sdk-cookie-policy th{min-width:75px}.ot-sdk-cookie-policy a,.ot-sdk-cookie-policy a:hover{background:#fff}.ot-sdk-cookie-policy thead{background-color:#f6f6f4;font-weight:bold}.ot-sdk-cookie-policy .ot-mobile-border{display:none}.ot-sdk-cookie-policy section{margin-bottom:2em}.ot-sdk-cookie-policy table{border-collapse:inherit}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy{font-family:inherit;font-size:1rem}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h3,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h4,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h6,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy p,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy li,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-title{color:dimgray}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description{margin-bottom:1em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-subgroup{margin-left:1.5em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group-desc,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-table-header,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy span,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td{font-size:.9em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td span,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td a{font-size:inherit}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group{font-size:1em;margin-bottom:.6em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-title{margin-bottom:1.2em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy>section{margin-bottom:1em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th{min-width:75px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a:hover{background:#fff}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy thead{background-color:#f6f6f4;font-weight:bold}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-mobile-border{display:none}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy section{margin-bottom:2em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-subgroup ul li{list-style:disc;margin-left:1.5em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-subgroup ul li h4{display:inline-block}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table{border-collapse:inherit;margin:auto;border:1px solid #d7d7d7;border-radius:5px;border-spacing:initial;width:100%;overflow:hidden}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table th,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table td{border-bottom:1px solid #d7d7d7;border-right:1px solid #d7d7d7}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr:last-child td{border-bottom:0px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr th:last-child,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr td:last-child{border-right:0px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-host,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-cookies-type{width:25%}.ot-sdk-cookie-policy[dir=rtl]{text-align:left}#ot-sdk-cookie-policy h3{font-size:1.5em}@media only screen and (max-width: 530px){.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) table,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) thead,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tbody,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) th,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr{display:block}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) thead tr{position:absolute;top:-9999px;left:-9999px}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr{margin:0 0 1em 0}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr:nth-child(odd),.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr:nth-child(odd) a{background:#f6f6f4}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td{border:none;border-bottom:1px solid #eee;position:relative;padding-left:50%}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td:before{position:absolute;height:100%;left:6px;width:40%;padding-right:10px}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) .ot-mobile-border{display:inline-block;background-color:#e4e4e4;position:absolute;height:100%;top:0;left:45%;width:2px}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td:before{content:attr(data-label);font-weight:bold}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) li{word-break:break-word;word-wrap:break-word}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table{overflow:hidden}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table td{border:none;border-bottom:1px solid #d7d7d7}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy thead,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy tbody,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy tr{display:block}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-host,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-cookies-type{width:auto}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy tr{margin:0 0 1em 0}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td:before{height:100%;width:40%;padding-right:10px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td:before{content:attr(data-label);font-weight:bold}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy li{word-break:break-word;word-wrap:break-word}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy thead tr{position:absolute;top:-9999px;left:-9999px;z-index:-9999}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr:last-child td{border-bottom:1px solid #d7d7d7;border-right:0px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr:last-child td:last-child{border-bottom:0px}}
                
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h5,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h6,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy li,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy p,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy span,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description {
                        color: #696969;
                    }
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th {
                        color: #696969;
                    }
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group {
                        color: #696969;
                    }
                    
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-title {
                            color: #696969;
                        }
                    
            
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table th {
                            background-color: #F8F8F8;
                        }
                    
            .ot-floating-button__front{background-image:url('https://cdn.cookielaw.org/logos/static/ot_persistent_cookie_icon.png')}</style><script src="https://static.addtoany.com/menu/modules/core.pt5ow5lr.js" type="module"></script><style type="text/css">.a2a_hide{display:none}.a2a_logo_color{background-color:#0166ff}.a2a_menu,.a2a_menu *{-moz-box-sizing:content-box;-webkit-box-sizing:content-box;box-sizing:content-box;float:none;margin:0;padding:0;position:static;height:auto;width:auto}.a2a_menu{border-radius:6px;display:none;direction:ltr;background:#FFF;font:16px sans-serif-light,HelveticaNeue-Light,"Helvetica Neue Light","Helvetica Neue",Arial,Helvetica,"Liberation Sans",sans-serif;color:#000;line-height:12px;border:1px solid #CCC;vertical-align:baseline;overflow:hidden}.a2a_mini{min-width:200px;position:absolute;width:300px;z-index:9999997}.a2a_overlay{display:none;background:#616c7deb;backdrop-filter:blur(4px);-webkit-backdrop-filter:blur(4px);position:fixed;top:0;right:0;left:0;bottom:0;z-index:9999998;-webkit-tap-highlight-color:transparent;transition:opacity .14s,backdrop-filter .14s}.a2a_full{background:#FFF;border:1px solid #FFF;box-shadow:#2a2a2a1a 0 0 20px 10px;height:auto;height:calc(320px);top:15%;left:50%;margin-left:-320px;position:fixed;text-align:center;width:640px;z-index:9999999;transition:transform .14s,opacity .14s}.a2a_full_footer,.a2a_full_header,.a2a_full_services{border:0;margin:0;padding:12px;box-sizing:border-box}.a2a_full_header{padding-bottom:8px}.a2a_full_services{height:280px;overflow-y:scroll;padding:0 12px;-webkit-overflow-scrolling:touch}.a2a_full_services .a2a_i{display:inline-block;float:none;width:181px;width:calc(33.334% - 18px)}div.a2a_full_footer{font-size:12px;text-align:center;padding:8px 14px}div.a2a_full_footer a,div.a2a_full_footer a:visited{display:inline;font-size:12px;line-height:14px;padding:8px 14px}div.a2a_full_footer a:focus,div.a2a_full_footer a:hover{background:0 0;border:0;color:#0166FF}div.a2a_full_footer a span.a2a_s_a2a,div.a2a_full_footer a span.a2a_w_a2a{background-size:14px;border-radius:3px;display:inline-block;height:14px;line-height:14px;margin:0 3px 0 0;vertical-align:top;width:14px}.a2a_modal{height:0;left:50%;margin-left:-320px;position:fixed;text-align:center;top:15%;width:640px;z-index:9999999;transition:transform .14s,opacity .14s;-webkit-tap-highlight-color:transparent}.a2a_modal_body{background:0 0;border:0;font:24px sans-serif-light,HelveticaNeue-Light,"Helvetica Neue Light","Helvetica Neue",Arial,Helvetica,"Liberation Sans",sans-serif;position:relative;height:auto;width:auto}.a2a_thanks{color:#fff;height:auto;margin-top:20px;width:auto}.a2a_thanks>div:first-child{margin:0 0 40px 0}.a2a_thanks div *{height:inherit}#a2a_copy_link{background:#FFF;border:1px solid #FFF;cursor:pointer;margin-top:15%}label.a2a_s_link#a2a_copy_link_icon,label.a2a_w_link#a2a_copy_link_icon{background-size:48px;border-radius:0;display:inline-block;height:48px;left:0;line-height:48px;margin:0 3px 0 0;position:absolute;vertical-align:top;width:48px}#a2a_modal input#a2a_copy_link_text{background-color:transparent;border:0;color:#2A2A2A;cursor:pointer;font:inherit;height:48px;left:62px;max-width:initial;min-height:auto;padding:0;position:relative;width:564px;width:calc(100% - 76px)}#a2a_copy_link_copied{background-color:#0166ff;color:#fff;display:none;font:inherit;font-size:16px;margin-top:1px;padding:3px 8px}@media (forced-colors:active){.a2a_color_buttons a,.a2a_svg{forced-color-adjust:none}}@media (prefers-color-scheme:dark){.a2a_menu a,.a2a_menu a.a2a_i,.a2a_menu a.a2a_i:visited,.a2a_menu a.a2a_more,i.a2a_i{border-color:#2a2a2a!important;color:#fff!important}.a2a_menu a.a2a_i:active,.a2a_menu a.a2a_i:focus,.a2a_menu a.a2a_i:hover,.a2a_menu a.a2a_more:active,.a2a_menu a.a2a_more:focus,.a2a_menu a.a2a_more:hover,.a2a_menu_find_container{border-color:#444!important;background-color:#444!important}.a2a_menu:not(.a2a_thanks){background-color:#2a2a2a;border-color:#2a2a2a}.a2a_menu_find{color:#fff!important}.a2a_menu label.a2a_s_find svg{background-color:transparent!important}.a2a_menu label.a2a_s_find svg path{fill:#fff!important}.a2a_full{box-shadow:#00000066 0 0 20px 10px}.a2a_overlay{background-color:#373737eb}}@media print{.a2a_floating_style,.a2a_menu,.a2a_overlay{visibility:hidden}}@keyframes a2aFadeIn{from{opacity:0}to{opacity:1}}.a2a_starting{opacity:0}.a2a_starting.a2a_full,.a2a_starting.a2a_modal{transform:scale(.8)}@media (max-width:639px){.a2a_full{border-radius:0;top:15%;left:0;margin-left:auto;width:100%}.a2a_modal{left:0;margin-left:10px;width:calc(100% - 20px)}}@media (min-width:318px) and (max-width:437px){.a2a_full .a2a_full_services .a2a_i{width:calc(50% - 18px)}}@media (max-width:317px){.a2a_full .a2a_full_services .a2a_i{width:calc(100% - 18px)}}@media (max-height:436px){.a2a_full{bottom:40px;height:auto;top:40px}}@media (max-height:550px){.a2a_modal{top:30px}}@media (max-height:360px){.a2a_modal{top:20px}.a2a_thanks>div:first-child{margin-bottom:20px}}.a2a_menu a{color:#0166FF;text-decoration:none;font:16px sans-serif-light,HelveticaNeue-Light,"Helvetica Neue Light","Helvetica Neue",Arial,Helvetica,"Liberation Sans",sans-serif;line-height:14px;height:auto;width:auto;outline:0}.a2a_menu a.a2a_i:visited,.a2a_menu a.a2a_more{color:#0166FF}.a2a_menu a.a2a_i:active,.a2a_menu a.a2a_i:focus,.a2a_menu a.a2a_i:hover,.a2a_menu a.a2a_more:active,.a2a_menu a.a2a_more:focus,.a2a_menu a.a2a_more:hover{color:#2A2A2A;border-color:#EEE;border-style:solid;background-color:#EEE;text-decoration:none}.a2a_menu label.a2a_s_find{background-size:24px;height:24px;left:8px;pointer-events:auto;position:absolute;top:7px;width:24px}.a2a_menu label.a2a_s_find svg{background-color:#FFF}.a2a_menu label.a2a_s_find svg path{fill:#CCC}#a2a_menu_container{display:inline-block}.a2a_menu_find_container{border:1px solid #CCC;border-radius:6px;padding:2px 24px 2px 0;position:relative;text-align:left}.a2a_cols_container .a2a_col1{overflow-x:hidden;overflow-y:auto;-webkit-overflow-scrolling:touch}#a2a_modal input,#a2a_modal input[type=text],.a2a_menu input,.a2a_menu input[type=text]{display:block;background-image:none;box-shadow:none;line-height:100%;margin:0;outline:0;overflow:hidden;padding:0;-moz-box-shadow:none;-webkit-box-shadow:none;-webkit-appearance:none}#a2afeed_find_container input,#a2afeed_find_container input[type=text],#a2apage_find_container input,#a2apage_find_container input[type=text]{background-color:transparent;border:0;box-sizing:content-box;color:#2A2A2A;float:none;font:inherit;font-size:16px;height:28px;line-height:20px;left:38px;outline:0;margin:0;max-width:initial;min-height:initial;padding:2px 0;position:relative;width:99%}.a2a_clear{clear:both}.a2a_svg{background-repeat:no-repeat;display:block;overflow:hidden;height:32px;line-height:32px;padding:0;pointer-events:none;width:32px}.a2a_svg svg{background-repeat:no-repeat;background-position:50% 50%;border:none;display:block;left:0;margin:0 auto;overflow:hidden;padding:0;position:relative;top:0;width:auto;height:auto}a.a2a_i,i.a2a_i{display:block;float:left;border:1px solid #FFF;line-height:24px;padding:6px 8px;text-align:left;white-space:nowrap;overflow:hidden;text-overflow:ellipsis;width:132px}a.a2a_i span,a.a2a_more span{display:inline-block;overflow:hidden;vertical-align:top}a.a2a_i .a2a_svg{margin:0 6px 0 0}a.a2a_i .a2a_svg,a.a2a_more .a2a_svg{background-size:24px;height:24px;line-height:24px;width:24px}a.a2a_sss:hover{border-left:1px solid #CCC}a.a2a_more{border-bottom:1px solid #FFF;border-left:0;border-right:0;line-height:24px;margin:6px 0 0;padding:6px;-webkit-touch-callout:none}a.a2a_more span{height:24px;margin:0 6px 0 0}.a2a_kit .a2a_svg{background-repeat:repeat}.a2a_default_style a:empty,.a2a_flex_style a:empty,.a2a_floating_style a:empty,.a2a_overlay_style a:empty{display:none}.a2a_color_buttons a,.a2a_floating_style a{text-decoration:none}.a2a_default_style:not(.a2a_flex_style) a{float:left;line-height:16px;padding:0 2px}.a2a_default_style a:hover .a2a_svg,.a2a_floating_style a:hover .a2a_svg,.a2a_overlay_style a:hover .a2a_svg svg{opacity:.7}.a2a_overlay_style.a2a_default_style a:hover .a2a_svg{opacity:1}.a2a_default_style .a2a_count,.a2a_default_style .a2a_svg,.a2a_floating_style .a2a_svg,.a2a_menu .a2a_svg,.a2a_vertical_style .a2a_count,.a2a_vertical_style .a2a_svg{border-radius:4px}.a2a_default_style .a2a_counter img,.a2a_default_style .a2a_dd,.a2a_default_style .a2a_svg{float:left}.a2a_default_style .a2a_img_text{margin-right:4px}.a2a_default_style .a2a_divider{border-left:1px solid #000;display:inline;float:left;height:16px;line-height:16px;margin:0 5px}.a2a_kit a{cursor:pointer;transition:none}.a2a_floating_style{background-color:#fff;border-radius:6px;position:fixed;z-index:9999995}.a2a_overlay_style{z-index:2147483647}.a2a_floating_style,.a2a_overlay_style{animation:a2aFadeIn .2s ease-in;padding:4px}.a2a_vertical_style:not(.a2a_flex_style) a{clear:left;display:block;overflow:hidden;padding:4px}.a2a_floating_style.a2a_default_style{bottom:0}.a2a_floating_style.a2a_default_style a,.a2a_overlay_style.a2a_default_style a{padding:4px}.a2a_count{background-color:#fff;border:1px solid #ccc;box-sizing:border-box;color:#2a2a2a;display:block;float:left;font:12px Arial,Helvetica,sans-serif;height:16px;margin-left:4px;position:relative;text-align:center;width:50px}.a2a_count:after,.a2a_count:before{border:solid transparent;border-width:4px 4px 4px 0;content:"";height:0;left:0;line-height:0;margin:-4px 0 0 -4px;position:absolute;top:50%;width:0}.a2a_count:before{border-right-color:#ccc}.a2a_count:after{border-right-color:#fff;margin-left:-3px}.a2a_count span{animation:a2aFadeIn .14s ease-in}.a2a_vertical_style .a2a_counter img{display:block}.a2a_vertical_style .a2a_count{float:none;margin-left:0;margin-top:6px}.a2a_vertical_style .a2a_count:after,.a2a_vertical_style .a2a_count:before{border:solid transparent;border-width:0 4px 4px 4px;content:"";height:0;left:50%;line-height:0;margin:-4px 0 0 -4px;position:absolute;top:0;width:0}.a2a_vertical_style .a2a_count:before{border-bottom-color:#ccc}.a2a_vertical_style .a2a_count:after{border-bottom-color:#fff;margin-top:-3px}.a2a_color_buttons .a2a_count,.a2a_color_buttons .a2a_count:after,.a2a_color_buttons .a2a_count:before,.a2a_color_buttons.a2a_vertical_style .a2a_count:after,.a2a_color_buttons.a2a_vertical_style .a2a_count:before{background-color:transparent;border:none;color:#fff;float:none;width:auto}.a2a_color_buttons.a2a_vertical_style .a2a_count{margin-top:0}.a2a_flex_style{display:flex;align-items:flex-start;gap:0}.a2a_default_style.a2a_flex_style{left:0;right:0;width:100%}.a2a_vertical_style.a2a_flex_style{flex-direction:column;top:0;bottom:0}.a2a_flex_style a{display:flex;justify-content:center;flex:1;padding:4px}.a2a_flex_style.a2a_vertical_style a{flex-direction:column}.a2a_floating_style.a2a_color_buttons,.a2a_floating_style.a2a_flex_style{border-radius:0;padding:0}.a2a_floating_style.a2a_default_style.a2a_flex_style{bottom:0}.a2a_kit.a2a_flex_style .a2a_counter img,.a2a_kit.a2a_flex_style .a2a_dd,.a2a_kit.a2a_flex_style .a2a_svg{float:none}.a2a_nowrap{white-space:nowrap}.a2a_note{margin:0 auto;padding:9px;font-size:12px;text-align:center}.a2a_note .a2a_note_note{margin:0;color:#2A2A2A}.a2a_wide a{display:block;margin-top:3px;border-top:1px solid #EEE;text-align:center}.a2a_label{position:absolute!important;clip-path:polygon(0px 0px,0px 0px,0px 0px);-webkit-clip-path:polygon(0px 0px,0px 0px,0px 0px);overflow:hidden;height:1px;width:1px}.a2a_kit,.a2a_menu,.a2a_modal,.a2a_overlay{-ms-touch-action:manipulation;touch-action:manipulation;outline:0}.a2a_dd{-webkit-user-drag:none}.a2a_dd img{border:0}.a2a_button_facebook_like iframe{max-width:none}</style><style></style><link href="https://crossmark-cdn.crossref.org/widget/v2.0/style.css" type="text/css" rel="stylesheet"></head>
<body class="pb-ui">
    <div class="skip">
        <a href="#page-body-id" id="skipNavigationLink" title="Skip to Main Content" tabindex="0" class="skipToContent">Skip to Main Content</a>
        <a href="#mega-menu-nav" id="skipToMainMenu" title="Skip to Main Menu" tabindex="0" class="skipToContent">Skip to Main Menu</a>
    </div>



    <div id="pb-page-content" data-ng-non-bindable="">
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="ux3-layout-widget" data-widget-id="832ad3f3-97ac-45ab-9af1-90c6f7bff54a" class="ux3">
        
    

        


    

            

            
                <div class="cell fonts-loaded">
            

            <div data-widget-id="832ad3f3-97ac-45ab-9af1-90c6f7bff54a" data-widget-name="ux3-layout-widget"><div>
    

        


    

            

            

            <div id="a1f1ba25-79af-4194-aaa9-66206104a800" class="widget loginPopup none widget-none widget-compact-all"><div class="wrapped"><div class="widget-body body body-none body-compact-all"><div role="dialog" aria-modal="true" aria-label="Login" class="popup login-popup hidden"><div class="content"><a href="#" aria-label="Close login popup" title="Close login popup" role="button" class="close"><i class="icon-close_thin"></i></a><h2 class="login-title">Login to your account</h2><div class="login-form"><p tabindex="-1" class="message error"></p><form action="/action/doLogin?redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2901332-5" method="post"><input type="hidden" name="redirectUri" value="https://www.cell.com/cell/fulltext/S0092-8674(21)01332-5"><input type="hidden" name="referer" value="/cell/fulltext/S0092-8674(21)01332-5"><input type="hidden" name="ecommRedirect"><input type="hidden" name="code" value="cell-site"><input type="hidden" name="UX3LoginWidget" value="true"><input type="hidden" name="isPopup" value="true"><div class="input-group login-input--wrapper"><div class="label"><label for="login">Email/Username</label></div><input id="login" type="text" name="login" value="" size="15" class="login"><div role="alert" class="error-hint js--hidden"><div class="error-hint__text"><div class="error-text">Your email address is a required field.</div><div class="email-example">E.g., j.smith@mail.com</div></div></div></div><div class="input-group login-input--wrapper"><div class="label"><label for="password">Password</label></div><input id="password" type="password" name="password" value="" autocomplete="off" class="password"><a href="#" class="login__field__action js-show-password">Show</a><div role="alert" class="error-hint js--hidden"><span class="error-hint__text">Your password is a required field.</span></div><div class="actions"><a href="https://www.cell.com/action/requestResetPassword" class="link show-request-reset-password">Forgot password?</a></div></div><div class="remember"><input type="checkbox" name="savePassword" id="a1f1ba25-79af-4194-aaa9-66206104a800-remember" value="1" checked="checked"><label for="a1f1ba25-79af-4194-aaa9-66206104a800-remember">Remember me</label></div><div class="submit"><input type="submit" name="submit" value="Log in" class="button submit primary"></div></form></div><div class="create-account"><div>Don’t have an account?</div><a href="/action/registration?redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2901332-5" class="show-registration">Create a Free Account</a></div></div></div></div></div></div>

            

        

    

        


    

            

            

            <div role="dialog" aria-modal="true" aria-label="Reset password" class="popup top-drawer request-reset-password-drawer hidden"><div class="content"><a href="#" aria-label="Close reset password popup" title="Close reset password popup" role="button" class="close"><i class="icon-close_thin"></i></a><div class="request-reset"><div class="form"><p class="sub">If you don't remember your password, you can reset it by entering your email address and clicking the Reset Password button. You will then receive an email that contains a secure link for resetting your password</p><div tabindex="-1" class="message error"></div><form action="/action/requestResetPassword" method="post" class="request-reset-password-form"><input type="hidden" name="platform"><input type="hidden" name="requestResetPassword" value="true"><input type="hidden" name="codeSite" value="cell-site"><div class="input-group"><div class="label"><label for="reset-password-email">Email*</label></div><input id="reset-password-email" type="text" name="email" value="" size="15" class="email"></div><div class="submit" disabled="disabled"><input type="submit" name="submit" value="Submit" disabled="disabled" class="button primary submit"></div></form></div><div class="success-template hidden"><p>If the address matches a valid account an email will be sent to __email__ with instructions for resetting your password</p></div></div><div class="request-reset-cancel"><a href="#" aria-label="Close reset password popup" title="Close reset password popup" class="cancel">Cancel</a></div></div></div>

            

        

    

        


    

            

            
                <div class="ux3-ads">
            

            <div class="pb-ad">
            <!-- Do Not Modify without contacting ELS-HSOnlineCampaign@elsevier.com -->
<!-- To be kept in sync with Global Commercial Publishing Operations-->
<!-- Start: GPT Async -->
<div id="gpt-Variables" data-adfile="CELLJOURNAL">
	<input type="hidden" id="gptSite" value="cell">
	<input type="hidden" id="gptLoggedIn" value="no">
	<input type="hidden" id="gptAccessType" value="ae:ANON_GUEST|ae:ANON_GUEST">
	<input type="hidden" id="gptDefaultSite" value="cell">
</div>
<div id="gpt-Interscroller" class="gpt-InScrollBox" style="display:none;">
	<div class="gpt-InScrollLabel">ADVERTISEMENT</div>
	<div class="gpt-InScrollContainer">
		<div class="gpt-InScrollWrapper">
			<div class="gpt-InScrollInner">
				<div class="gpt-adinterscroller" data-adsize="interscroller"></div>
			</div>
		</div>
	</div>
	<div class="gpt-InScrollLabel">SCROLL TO CONTINUE WITH CONTENT</div>
</div>
<div id="gpt-DebugDiv" class="gpt-DebugDiv" style="display:none;">
	<div style="width: 100%; display: table;">
		<div style="display: table-row">
			<div style="width: 200px; display: table-cell;">
				<button class="gpt-DebugBtn" onclick="gpt_ShowConsole()">Open GPT Console</button><br>
				<button id="gpt_GrapeShotDebugBtn" class="gpt-DebugBtn" onclick="gpt_GrapeShotDebug();">Open Oracle Keywords</button><br>
				<br>
				<button class="gpt-DebugBtn" onclick="gpt_LoadDebugValues()">Refresh Values</button>
			</div>
			<div style="display: table-cell;">
				<table class="gpt-DebugTable">
					<thead>
						<tr>
							<th>Property</th>
							<th>Value</th>
						</tr>
					</thead>
					<tbody>
						<tr><td class="gpt-DebugTitle">Status</td><td><div class="gptStatus_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Version</td><td><div class="gptVersion_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Ad File</td><td><div class="gptAdFile_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Disable Ads Flag</td><td><div class="gptDisableAds_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Environment</td><td><div class="gptTestENV_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Moat Init</td><td><div class="gptMoatInit_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Moat Ready</td><td><div class="gptMoatReady_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual Ready</td><td><div class="gptContextualReady_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual URL</td><td><div class="gptContextualURL_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual Initial Segments</td><td><div class="gptContextualInit_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual Used Segments</td><td><div class="gptContextualResult_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">AdUnit</td><td><div class="gptSite_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">SubAdUnit</td><td><div class="gptPage_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Custom Targeting</td><td><div class="gptCustomTargeting_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Ad Events</td><td><div class="gptEvent_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Invalid Ad Sizes</td><td><div class="gptInvalidAdSize_Debug"></div></td></tr>
					</tbody>
				</table>
			</div>
		</div>
	</div>
</div>
<style type="text/css">
	.gpt-DebugBtn {
		display: inline-block;
		text-align: center;
		text-decoration: none;
		margin: 2px 0;
		border: solid 1px #122b40;
		border-radius: 0.4em;
		padding: 0.5em 1em;
		color: #ffffff;
		background-color: #204d74;
		font-weight: normal;
		font-size: 12px;
	}

	.gpt-DebugDiv {
		margin: 5px;
	}

	.gpt-DebugTitle {
		font-weight: bold;
	}

	.gpt-DebugTable {
		border: 1px solid #dddddd;
		border-collapse: collapse;
		margin: 5px;
		font-size: 0.9em;
		font-family: sans-serif;
		min-width: 400px;
		margin-bottom: 25px;
	}

		.gpt-DebugTable td:first-child {
			padding: 5px;
		}

		.gpt-DebugTable thead tr th {
			background-color: #d9edf7;
			color: #ffffff;
			text-align: left;
			padding: 5px;
			border-bottom: 2px solid #204d74;
		}

		.gpt-DebugTable th,
		.gpt-DebugTable td {
			padding: 5px;
		}

		.gpt-DebugTable tbody tr {
			border-bottom: 1px solid #dddddd;
		}

			.gpt-DebugTable tbody tr:nth-of-type(even) {
				background-color: #f3f3f3;
			}

			.gpt-DebugTable tbody tr:last-of-type {
				border-bottom: 2px solid #204d74;
			}

	.gpt-InScrollLabel {
		background-color: rgb(0, 0, 0);
		color: rgb(255, 255, 255);
		position: relative;
		max-width: 100vw;
		left: 50%;
		right: 50%;
		margin-left: -50vw;
		margin-right: -50vw;
		padding: 0;
		text-align: center;
		width: 100vw !important;
		font-size: 10px;
	}

	.gpt-InScrollBox {
		height: 100vh !important;
		width: 100vw;
		cursor: pointer;
		max-width: 100vw;
		position: relative;
		left: 50%;
		right: 50%;
		margin-left: -50vw;
		margin-right: -50vw;
		margin-bottom: 50px;
		padding-top: 5px;
		padding-bottom: 5px;
		z-index: 10001;
		background-color: white;
	}

		.gpt-InScrollBox .gpt-InScrollContainer {
			height: 95vh !important;
			position: relative;
		}

		.gpt-InScrollBox .gpt-InScrollWrapper {
			position: absolute !important;
			top: 0 !important;
			left: 0;
			width: 100%;
			height: 95vh !important;
			border: 0 !important;
			margin: 0 !important;
			padding: 0 !important;
			clip: rect(0, auto, auto, 0) !important;
			-webkit-clip-path: polygon(0px 0px, 100% 0px, 100% 100%, 0px 100%) !important;
			clip-path: polygon(0px 0px, 100% 0px, 100% 100%, 0px 100%) !important;
		}

		.gpt-InScrollBox .gpt-InScrollInner {
			position: fixed !important;
			top: 0 !important;
			left: 0 !important;
			bottom: 0;
			margin: auto;
			width: 100vw;
			height: 95vh;
			-webkit-transform: translateZ(0) !important;
			display: -webkit-box;
			display: -ms-flexbox;
			display: flex;
			-webkit-box-orient: vertical;
			-webkit-box-direction: normal;
			-ms-flex-direction: column;
			flex-direction: column;
			-webkit-box-align: center;
			-ms-flex-align: center;
			align-items: center;
			-webkit-box-pack: center;
			-ms-flex-pack: center;
			justify-content: center;
			text-align: center;
		}
</style>
<script class="optanon-category-4" type="text/plain">
	//--------------------------------------------------------------------------
	var gpt_Version = '04072023';
	var gpt_AdFile = $('#gpt-Variables').data('adfile');
	var gpt_MaxMOATInitAttempts = 11;
	var gpt_MaxMOATReadyAttempts = 11;
	var gpt_MaxContextualReadyAttempts = 21;
	var gpt_MOATInitAttempts = 0;
	var gpt_MOATReadyAttempts = 0;
	var gpt_ContextualReadyAttempts = 0;
	var gpt_IsProd = true;
	var gpt_StartTime;
	var gs_channels = 'DEFAULT';
	var gpt_ContextualReady = false;
	var gpt_ContextualURL = 'Not enabled outside of Production';
	var gpt_DisableAdsFlagFound = false;

	var gptEvent_Debug = '';
	var gptCustomTargeting_Debug = '';
	var gptSite_Debug = "";
	var gptPage_Debug = "";
	var gptContextualEnabled = false;
	var gptContextualResult_Debug = '';
	var gptContextualInit_Debug = '';
	var gptContextualURL_Debug = '';
	var gptContextualReady_Debug = '';
	var gptMoatReady_Debug = '';
	var gptMoatInit_Debug = '';
	var gptTestENV_Debug = '';
	var gptDisableAds_Debug = '';
	var gptStatus_Debug = 'GPT Still Loading - Refresh Values';
	var gptInvalidAdSize_Debug = '';
	//--------------------------------------------------------------------------
	$(document).ready(function () {
		//Check to see if we are on a development env
		gpt_CheckEnvironment();

		if (!gpt_DisableAdsFlagFound) {
			gpt_StartTime = new Date();
			if (gpt_AdFile !== "PU") {
				//Once the dom has loaded, begin the process to init the ads.
				gpt_InitScripts();
				gpt_CheckForMOATInit();
			}
		}
	});
</script>
<script type="text/javascript">
	function gpt_CheckEnvironment() {
		if ($('#gptDisableAds').length > 0) {
			gptDisableAds_Debug = 'Flag Detected - Ads Disabled';
			gpt_DisableAdsFlagFound = true;
		}
		else {
			gptDisableAds_Debug = 'Ads Enabled';
			gpt_DisableAdsFlagFound = false;
		}
		switch (gpt_AdFile) {
			case 'PU':
				gptContextualEnabled = true;
				if ($("#Environment").val() !== "Prod") {
					gpt_IsProd = false;
					gptTestENV_Debug = 'Test Environment Detected';
				}
				else {
					gptTestENV_Debug = 'Production Environment Detected';
				}
				break;
			case 'JBS':
			case 'LANCET':
				gptContextualEnabled = true;
				if ($(".gpt-ad").length > 0) {
					//Other ads exist, allow the Prestitial
					$(document.body).append("<div class='gpt-ad' data-adsize='prestitial'></div>");
				}
				if (window.location.hostname.toLowerCase().indexOf('.literatumonline.') >= 0) {
					gpt_IsProd = false;
					gptTestENV_Debug = 'Test Environment Detected';
				}
				else {
					gptTestENV_Debug = 'Production Environment Detected';
				}
				break;
			default:
				//Cell
				gptContextualEnabled = false;
				if (window.location.hostname.toLowerCase().indexOf('.literatumonline.') >= 0) {
					gpt_IsProd = false;
					gptTestENV_Debug = 'Test Environment Detected';
				}
				else {
					gptTestENV_Debug = 'Production Environment Detected';
				}
				break;
		}
	}
	//--------------------------------------------------------------------------
    function gpt_GetNonCMEArticle() {
        var url;
        $('.articleCitation').each(function (i, obj) {
            if ($(obj).find('.toc__item__detials').length > 0) {
                var itemDetails = $(obj).first().find('.toc__item__detials');
                var cmeFound = 0;
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='CME']").length; }
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='(CME)']").length; }
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='Education']").length; }
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='education']").length; }
                if (cmeFound == 0) {
                    var itemLink = $(itemDetails).first().find('.toc__item__title');
                    var linkCount = $(itemLink).first().find("a:contains('CME')").length;
                    if (linkCount == 0) {
                        url = $(itemLink).first().find('a').attr('href');
                        return false;   //Found a URL, break the loop
                    }
                }
            }
        });
        return url;
    }
	//--------------------------------------------------------------------------
	function gpt_GetElapsedTime() {
		var endTime = new Date();
		var timeDiff = endTime - gpt_StartTime; //in ms
		// strip the ms
		timeDiff /= 1000;

		// get seconds
		return seconds = Math.round(timeDiff) + " seconds";
	}
	//--------------------------------------------------------------------------
	function gpt_CheckSSC() {
		//Check for ScreenShot Client
		var gpt_SSC = gpt_GetQuerystring('gpt_ssc', '');
		if (gpt_SSC == '1') {
			//it's the ScreenShotClient, disable MOAT IVT Detection
			$('#gpt-Variables').append('<input type="hidden" id="gptm_data">');
			$('#gptm_data').val(0);
			gpt_Target("m_data");
		}
	}
	//--------------------------------------------------------------------------
	function gpt_LocalSS(adTypeId) {
		window.scrollTo(0, 0);
		gpt_Sleep(1000);
		gpt_ScrollToAd(adTypeId);
		gpt_PreTearsheet(adTypeId);
	}
	//--------------------------------------------------------------------------
	function gpt_InitScripts() {
		if (gpt_AdFile == "PU") {
			if (gpt_IsProd) {
				gpt_ContextualURL = gpt_CheckForNULL($('#canonicalurl').val());
				if (gpt_ContextualURL === null) {
					gpt_ContextualURL = window.location.href.split('?')[0].split('#')[0];
				}
				else {
					gpt_ContextualURL = window.location.origin + gpt_ContextualURL;
				}
				var useSSL = 'https:' == document.location.protocol;
				var gs = document.createElement('script');
				gs.async = true;
				gs.type = 'text/javascript';
				gs.src = (useSSL ? 'https:' : 'http:') + '//elsevier.gscontxt.net/multizone/channels.cgi?url=' + encodeURIComponent(gpt_ContextualURL);
				gs.onload = function () {
					gpt_ContextualReady = true;
				};
				document.head.appendChild(gs);
			}
		}
		else {
			//JBS-Cell-Lancet
			var useSSL = 'https:' == document.location.protocol;

			var gads = document.createElement('script');
			gads.async = true;
			gads.type = 'text/javascript';
			gads.src = (useSSL ? 'https:' : 'http:') + '//securepubads.g.doubleclick.net/tag/js/gpt.js';
			document.head.appendChild(gads);

			if (gpt_IsProd && gptContextualEnabled) {
				gpt_ContextualURL = window.location.href.split('?')[0].split('#')[0];
				var gs = document.createElement('script');
				gs.async = true;
				gs.type = 'text/javascript';
				gs.src = (useSSL ? 'https:' : 'http:') + '//elsevier.gscontxt.net/multizone/channels.cgi?url=' + encodeURIComponent(gpt_ContextualURL);
				gs.onload = function () {
					gpt_ContextualReady = true;
				};
				document.head.appendChild(gs);
			}

			var elsevierMoatHeader = document.createElement('link');
			elsevierMoatHeader.rel = 'preload';
			elsevierMoatHeader.as = 'script';
			elsevierMoatHeader.src = (useSSL ? 'https:' : 'http:') + '//z.moatads.com/elsevierheader150204004183/moatheader.js';
			document.head.appendChild(elsevierMoatHeader);

			var elsevierMoatAds = document.createElement('link');
			elsevierMoatAds.rel = 'preconnect';
			elsevierMoatAds.src = (useSSL ? 'https:' : 'http:') + '//mb.moatads.com';
			document.head.appendChild(elsevierMoatAds);

			var moatHeader = document.createElement('script');
			moatHeader.async = true; gads.type = 'text/javascript';
			moatHeader.src = (useSSL ? 'https:' : 'http:') + '//z.moatads.com/elsevierheader150204004183/moatheader.js';
			document.head.appendChild(moatHeader);
		}
	}
	//--------------------------------------------------------------------------
	async function gpt_CheckForMOATInit() {
		//Check every 1/10 of a second until MOAT is Loaded or 1 second timeout
		if (gpt_MOATInitAttempts < gpt_MaxMOATInitAttempts) {
			gpt_MOATInitAttempts += 1;
			if (typeof window.moatPrebidApi === "object") {
				//MOAT LOADED
				gptMoatInit_Debug = '0.0' + gpt_MOATInitAttempts + ' seconds';
				gpt_CheckForMOATReady();
			}
			else {
				setTimeout(() => { gpt_CheckForMOATInit(); }, 100);
			}
		}
		else {
			//TIMEOUT: MOAT Not LOADED
			gptMoatInit_Debug = 'Timeout Expired';
			var googletag = googletag || {};
			googletag.cmd = googletag.cmd || [];
			gpt_CheckForContextualReady();
		}
	}
	//--------------------------------------------------------------------------
	function gpt_CheckForMOATReady() {
		//Check every 1/10 of a second until MOAT is Ready or 1 second timeout
		if (gpt_MOATReadyAttempts < gpt_MaxMOATReadyAttempts) {
			gpt_MOATReadyAttempts += 1;
			if (window.moatPrebidApi.safetyDataAvailable()) {
				//MOAT Ready
				gptMoatReady_Debug = '0.0' + gpt_MOATReadyAttempts + " seconds";
				gpt_CheckForContextualReady();
			}
			else {
				setTimeout(() => { gpt_CheckForMOATReady(); }, 100);
			}
		}
		else {
			//TIMEOUT: MOAT Not Ready
			gptMoatReady_Debug = 'Timeout Expired';
			gpt_CheckForContextualReady();
		}
	}
	//--------------------------------------------------------------------------
	function gpt_CheckForContextualReady() {
		//Check every 1/10 of a second until Contextual is Ready or 2 second timeout
		if (gpt_IsProd && gptContextualEnabled) {
			//Enabled
			if (gpt_ContextualReadyAttempts < gpt_MaxContextualReadyAttempts) {
				gpt_ContextualReadyAttempts += 1;
				if (gpt_ContextualReady) {
					//Contextual Ready
					gptContextualURL_Debug = gpt_ContextualURL;
					gptContextualReady_Debug = '0.0' + gpt_ContextualReadyAttempts + " seconds";
					gptContextualInit_Debug = gs_channels;
					gpt_InitAds();
				}
				else {
					setTimeout(() => { gpt_CheckForContextualReady(); }, 100);
				}
			}
			else {
				//LOG TIMEOUT error
				//REMOVE FOR NOW (SHOULD LOG TO A DIFF LOCATION)
				//TIMEOUT: Contextual Not Ready
				$('#gpt_GrapeShotDebugBtn').remove();
				gptContextualURL_Debug = gpt_ContextualURL;
				gptContextualReady_Debug = 'Timeout Expired';
				gptContextualInit_Debug = "";
				gpt_InitAds();
			}
		}
		else {
			//Not Enabled
			$('#gpt_GrapeShotDebugBtn').remove();
			switch (gpt_AdFile) {
				case 'PU':
				case 'LANCET':
				case 'JBS':
					gptContextualURL_Debug = 'Not enabled outside of Production';
					gptContextualReady_Debug = 'Not enabled outside of Production';
					gptContextualInit_Debug = 'Not enabled outside of Production';
					break;
				case "CELLJOURNAL":
				case "CELLBUCKET":
					gptContextualReady_Debug = 'Not enabled on Cell';
					gptContextualInit_Debug = 'Not enabled on Cell';
					gptContextualURL_Debug = 'Not enabled on Cell';
					break;
			}
			gpt_InitAds();
		}
	}
	//--------------------------------------------------------------------------
	function GPT_GetPageByURL() {
		var host = window.location.host.toLocaleLowerCase()
		var localpath = window.location.pathname.toLocaleLowerCase();

		///Check for Cell.com
		if (host.indexOf('cell.com') >= 0) {
			return null;
		}

		///Check for Cell.com (marlin-stag)
		if (host.indexOf('www-cell-com') >= 0) {
			return null;
		}

		///Check for Lancet
		if (host.indexOf('thelancet') >= 0) {
			return null;
		}

		if (localpath.indexOf('/article/') >= 0) {
			return 'article.fulltext';
		}

		if (localpath.indexOf('/action/dosearch') >= 0) {
			return 'search.searchResults';
		}

		if (localpath.indexOf('/content/') >= 0) {
			return 'periodical.editorial';
		}

		return 'periodical.home';
	}
	//--------------------------------------------------------------------------
	function gpt_CheckForNULL(x, valueifnull) {
		if (typeof valueifnull == "undefined") {
			valueifnull = null;
		}

		if (typeof x == 'undefined') {
			return valueifnull;
		}
		if (x == null) {
			return valueifnull;
		}
		if (x == "null") {
			return valueifnull;
		}
		if ($.trim(x) == '') {
			return valueifnull;
		}

		return x;
	}
	//--------------------------------------------------------------------------
	function gpt_GetAdditionalTargeting() {
		//adds page level targeting of custom variables
		//the id (minus gpt) and the value will be used as a key-value pair
		//eg: <input type="hidden" id="gptissuepii" value="x" class="gpt-Target">
		//eg: target issuepii=x
		$(".gpt-Target").each(function (index) {
			var gptTargetId = $(this).attr('id');
			gptTargetId = gptTargetId.replace('gpt', '').toLowerCase();
			if (gptTargetId !== '') {
				gpt_Target(gptTargetId);
			}
		})
	}
	//--------------------------------------------------------------------------
	function gpt_GetQuerystring(key, default_) {
		//function to read querystring variables
		if (default_ === null) default_ = '';
		key = key.replace(/[\[]/, "\\\[").replace(/[\]]/, "\\\]");
		var regex = new RegExp("[\\?&]" + key + "=([^&#]*)");
		var qs = regex.exec(window.location.href);
		if (qs === null)
			return default_;
		else
			return qs[1];
	}
	//--------------------------------------------------------------------------
	function gpt_GetDFPTestId() {
		//Get any request for a test id from querystring and set custom targeting parameter
		var gptDFPTestId = gpt_GetQuerystring('dfptestid', '');
		if (gptDFPTestId !== '') {
			$('#gpt-Variables').append('<input type="hidden" id="gptDFPTestId">');
			$('#gptDFPTestId').val(gptDFPTestId);
			gpt_Target("DFPTestId");
		}
	}
	//--------------------------------------------------------------------------
	function gpt_Target(targetName) {
		//Set page level target if value is found
		if ($('#gpt' + targetName).length > 0) {
			var targetValue = $('#gpt' + targetName).val();

			if (targetValue !== '') {
				targetValue = targetValue.replace(/[^-a-z0-9]/ig, '');	//Only Allow a-z, 0-9
				googletag.pubads().setTargeting(targetName.toLowerCase(), targetValue.toLowerCase());
				gptCustomTargeting_Debug = gptCustomTargeting_Debug + targetName.toLowerCase() + ' = ' + targetValue.toLowerCase() + '<br>';
			}
		}
	}
	//--------------------------------------------------------------------------
	function gpt_TargetAccessType() {
		var currentAccessType = $('#gptAccessType').val();
		if (typeof currentAccessType == "undefined") {
			currentAccessType = "";
		}
		if (currentAccessType.indexOf('ANON_IP') >= 0) {
			googletag.pubads().setTargeting("loggedin", "yes");
		}
		else {
			gpt_Target("LoggedIn");
		}
	}
	//--------------------------------------------------------------------------
	async function gpt_PreTearsheet(adTypeId) {
		if (typeof adTypeId === "undefined") {
			adTypeId = 0;
		}
		//Remove Elsevier Cookie Policy Notice
		if ($("#onetrust-consent-sdk").length > 0) {
			$("#onetrust-consent-sdk").remove();
		}

        //Mobile Nav
        $('#skip').remove();
        $('skipNavigationLink').remove();

		//JBS Modals
		if ($('.leadinModal').length > 0) {
			$('.leadinModal').remove();
		}
		if ($('.leadinModal').length > 0) {
			$('.leadinModal').remove();
		}
		if ($('.usabilla__overlay').length > 0) {
			$('.usabilla__overlay').remove();
		}

		//Pendo
		if ($('._pendo-step-container').length > 0) {
			$('._pendo-step-container').remove();
		}
		if ($('._pendo-guide-tt_').length > 0) {
			$('._pendo-guide-tt_').remove();
		}
		if ($('._pendo-image').length > 0) {
			$('._pendo-image').remove();
		}
		if ($('._pendo-badge-image').length > 0) {
			$('._pendo-badge-image').remove();
		}

		switch (gpt_AdFile) {
			case "CELLJOURNAL":
			case "CELLBUCKET":
				//Mobile Bug Fix on Cell
				if (parseInt(adTypeId) == 5) {
					$('.widget-horizontalAd').css('position', 'static');
					$('.body-horizontalAd').css('display', 'block');
				}
				break;
			case "JBS":
			case "LANCET":
				objLeaderboard = ".container-leaderboard";
				offset = 53;
				switch (parseInt(adTypeId)) {
					case 3: //MPU
					case 4: //SkyScraper
						//BUGFIX: Show Sidebar ad for JBS
						//skyscraper & mpu
						if ($('.widget-verticalAd').length > 0) {
							$('.widget-verticalAd').css('display', 'block');
						}
						$('#backgroundLayer').css('display', 'none');   //Hide any Prestitial
						break;
					case 6:     //Prestitial
						$('#backgroundLayer').css('display', 'block');
						break;
					default:
						$('#backgroundLayer').css('display', 'none');   //Hide any Prestitial
						break;
				}
				break;
		}
	}
	//--------------------------------------------------------------------------
	function gpt_Sleep(ms) {
		return new Promise(resolve => setTimeout(resolve, ms));
	}
	//--------------------------------------------------------------------------
	async function gpt_ScrollToAd(adTypeId) {
		if (typeof adTypeId === "undefined") {
			adTypeId = 0;
		}
		var adId = "";
		var offset = 0;
		var objLeaderboard = "";
		var gpt_ScrollToAd = true;

		switch (gpt_AdFile) {
			case "PU": //PracticeUpdate
				var hideOtherAds = true;
				$('#modalPrestitial').modal('hide');
				$('.jumbotron-welcome').hide();
				objLeaderboard = ".container-leaderboard";
				offset = 53;
				switch (parseInt(adTypeId)) {
					case 2:
						//LeaderBoard, no Scroll Needed
						gpt_ScrollToAd = false;
						adId = "adSlot1";
						break;
					case 3:
						//MPU
						adId = "adSlot2";
						break;
					case 4:
						//SkyScraper
						adId = "adSlot3";
						$('.most-read').hide();
						break;
					case 5:
						//Mobile, no Scroll Needed
						gpt_ScrollToAd = false;
						adId = "adSlot1";
						break;
					case 6:
						//Interstitial
						gpt_ScrollToAd = false;
						adId = "adSlot15";
						await gpt_Sleep(1000);
						$(objLeaderboard).hide();
						$('#modalPrestitial').unbind();
						$('#modalPrestitial').modal();
						break;
					case 7:
						//Native Ad
						break;
					case 10:
						//Master Companion
						hideOtherAds = false;
						gpt_ScrollToAd = false;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						adId = "adSlot1";
						break;
				}
				if (hideOtherAds) {
					//Hide all other ads
					$('.ad-container').each(function (i, obj) {
						if (this.id !== adId) {
							var tempParent = this.id;
							tempParent = tempParent.replace("adSlot", "adParent");
							$("#" + tempParent).addClass('hide');
						}
					});
				}
				break;
			case "JBS":
				objLeaderboard = ".js__adToHide";
				offset = 80;
				switch (parseInt(adTypeId)) {
					case 2:
						//Leaderboard
						gpt_ScrollToAd = false;
						$(".widget-verticalAd").hide();
						break;
					case 5:
						//Mobile
						//Scroll to Top
						gpt_ScrollToAd = false;
						break;
					case 3: //MPU
					case 4: //SkyScraper
						$(objLeaderboard).hide();
						adId = $('.gpt-ad-defined[data-adsize="sidebar"').attr('id');
						break;
					case 10:
						//MasterCompanion
						gpt_ScrollToAd = false;
						break;
					case 13:	    //Phone - Portrait
					case 14:        //Phone - Landscape
						$(objLeaderboard).hide();
						offset = 60;
						adId = 'gpt-Interscroller';
						gpt_ScrollToAd = true;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						break;
				}
				break;
			case "CELLJOURNAL":
			case "CELLBUCKET":
				objLeaderboard = ".js__adToHide";
				offset = 260;
				switch (parseInt(adTypeId)) {
					case 2:
						//Leaderboard
						gpt_ScrollToAd = false;
						$('.body-verticalAd').hide();
						break;
					case 5:
						//Mobile
						//Scroll to Top
						gpt_ScrollToAd = false;
						break;
					case 3: //MPU
					case 4: //SkyScraper
						$(objLeaderboard).hide();
						adId = $('.gpt-ad-defined[data-adsize="boombox"').attr('id');
						break;
					case 10:
						//MasterCompanion
						gpt_ScrollToAd = false;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						break;
				}
				//Hide the second ad if found
				var tCount = 0;
				$(".widget.literatumAd.none.GPT-ad-placeholder.widget-horizontalAd.widget-compact-all").each(function (index) {
					tCount += 1;
					if (tCount !== 1) {
						$(this).hide();
					}
				});
				break;
			case "LANCET":
				objLeaderboard = ".js__adToHide";
				offset = 80;
				switch (parseInt(adTypeId)) {
					case 2:
						//Leaderboard
						gpt_ScrollToAd = false;
						$('.body-verticalAd').hide();
						break;
					case 5:
						//Mobile
						//Scroll to Top
						gpt_ScrollToAd = false;
						break;
					case 3: //MPU
					case 4: //SkyScraper
						$(objLeaderboard).hide();
						adId = $('.gpt-ad-defined[data-adsize="skyscraper"').attr('id');
						break;
					case 10:
						//MasterCompanion
						gpt_ScrollToAd = false;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						break;
				}
				break;
		}

		//Scroll to Top
		var gpt_container = $("html,body");
		gpt_container.animate({ scrollTop: 0, scrollLeft: 0 });

		if (gpt_ScrollToAd) {
			//If Adexists, scroll to it
			var gpt_scrollTo = $('#' + adId);
			if (gpt_scrollTo.length > 0) {
				//Scroll to the Ad
				var gpt_topPosition = gpt_scrollTo.offset().top - gpt_container.offset().top + gpt_container.scrollTop();
				if (gpt_topPosition > 0) {
					gpt_container.animate({ scrollTop: gpt_topPosition, scrollLeft: 0 });
					if (gpt_topPosition - offset > 0 && offset > 0) {
						gpt_container.animate({ scrollTop: gpt_topPosition - offset, scrollLeft: 0 });
					}
				}
			}
		}
	}
	//--------------------------------------------------------------------------
	async function gpt_CheckForCookiePolicy() {
		var returnVal = false;
		if ($('#onetrust-accept-btn-handler').length > 0) {
			$('#onetrust-accept-btn-handler').click();
			returnVal = true;
		}
		return returnVal;
	}
	//--------------------------------------------------------------------------
	function gpt_InitAds() {
		var gpt_ResponsiveHomepage = false;

		switch (gpt_AdFile) {
			case "JBS":
			case "CELLJOURNAL":
			case "CELLBUCKET":
				//Responsive Check
				if ($('#hdrHomeBrand').length > 0) {
					gpt_ResponsiveHomepage = true;
				}
				break;
		}

		//InterScroller Check
		if (gpt_AdFile === "JBS") {
			var gpt_InterScrollerEnabled = false;
			var gpt_InterScrollerInsertObj;

			if ($("#articleHeader").length > 0) {
				//It's an article Page
				if ($('.article__sections > section').length > 0) {
					gpt_InterScrollerEnabled = true;
					gpt_InterScrollerInsertObj = $(".article__sections > section");
				}
			}
			else {
				if ($('.toc__section').length > 0) {
					//It's a TOC Page
					gpt_InterScrollerEnabled = true;
					gpt_InterScrollerInsertObj = $(".toc__section").first();
				}
			}

			//Add the Interscroller
			if (gpt_InterScrollerEnabled) {
				gpt_InterScrollerInsertObj.first().append($("#gpt-Interscroller").clone());
				$("#gpt-Interscroller").remove();
				$(".gpt-adinterscroller").addClass("gpt-ad");
			}
		}

		googletag.cmd.push(function () {
			var gptBoomBoxSizeMapping;
			var gptFluidMapping;
			var gptHomePageSidebarMapping;
			var gptCellHomePageSidebarMapping;
			var gptInterscrollerMapping;
			var gptLeaderboardSizeMapping;
			var gptMobileSizeMapping;
			var gptPrestitialMapping;
			var gptRectangleShortSizeMapping;
			var gptRectangleSizeMapping;
			var gptSideBarSizeMapping;
			var gptTallSizeMapping;
			var gptWideSizeMapping;
			var gptWideRectangleSizeMapping;

			//***********************************************
			//Start Define Size mapping based on client Viewport
			//***********************************************
			if (gpt_AdFile === 'PU') {
				gptTallSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[300, 600], [160, 600], [120, 600]]).
					addSize([0, 0], []).
					build();

				gptRectangleSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[300, 250]]).
					addSize([0, 0], []).
					build();

				gptRectangleShortSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[300, 250], [320, 50], [300, 50]]).
					addSize([0, 0], [[300, 250], [320, 50], [300, 50]]).
					build();

				gptWideSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[728, 90]]).
					addSize([760, 0], [[728, 90], [320, 50], [300, 50]]).
					addSize([0, 0], [[320, 50], [300, 50]]).
					build();

				gptWideRectangleSizeMapping = googletag.sizeMapping().
					addSize([992, 0], []).
					addSize([760, 0], [[320, 50], [300, 50], [300, 250]]).
					addSize([0, 0], [[320, 50], [300, 50], [300, 250]]).
					build();

				gptMobileSizeMapping = googletag.sizeMapping().
					addSize([0, 0], [[320, 50], [300, 50]]).
					build();

				gptFluidMapping = googletag.sizeMapping().
					addSize([0, 0], ['fluid']).
					build();

				gptPrestitialMapping = googletag.sizeMapping().
					addSize([760, 0], [[1, 1]]).
					build();
			}
			else {
				//JBS-CELL-LANCET
				gptLeaderboardSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[728, 90]]).
					addSize([768, 0], [[728, 90]]).
					addSize([0, 0], [[320, 50], [300, 50]]).
					build();

				gptHomePageSidebarMapping = googletag.sizeMapping().
					addSize([2090, 795], [[300, 600], [300, 250], [160, 600], [120, 600]]).
					addSize([1910, 795], [[160, 600], [120, 600]]).
                    addSize([1740, 795], [[120, 600]]).
					addSize([0, 0], []).
					build();

				gptCellHomePageSidebarMapping = googletag.sizeMapping().
                    addSize([2180, 795], [[300, 600], [160, 600], [120, 600], [300, 250], [336, 280]]).
					addSize([2180, 445], [[336, 280]]).
                    addSize([2090, 795], [[300, 600], [160, 600], [120, 600], [300, 250]]).
                    addSize([2090, 445], [[300, 250]]).
					addSize([1830, 795], [[160, 600], [120, 600]]).
                    addSize([1740, 795], [[120, 600]]).
					addSize([0, 0], []).
					build();

				gptSideBarSizeMapping = googletag.sizeMapping().
					addSize([1140, 795], [[300, 600], [160, 600], [120, 600], [300, 250]]).
					addSize([1140, 445], [[300, 250]]).
					addSize([0, 0], []). 				
					build();

				gptBoomBoxSizeMapping = googletag.sizeMapping().
					addSize([768, 0], [[300, 250], [336, 280]]).
					addSize([0, 0], []). 						
					build();

				gptFluidMapping = googletag.sizeMapping().
					addSize([0, 0], ['fluid']).
					build();

				gptPrestitialMapping = googletag.sizeMapping().
					addSize([600, 700], [[1, 1]]).
					addSize([0, 0], []).
					build();

				gptInterscrollerMapping = googletag.sizeMapping().
					addSize([1140, 445], []).
					addSize([1024, 768], [[1024, 768]]).
					addSize([768, 1024], [[768, 1024]]).
					addSize([480, 320], [[480, 320]]).
					addSize([320, 480], [[320, 480]]).
					build();
			}
			//***********************************************
			//End Define Size mapping based on client Viewport
			//***********************************************

			//***********************************************
			//Start detection for ad slots on page
			//***********************************************
			var gptMasterPage = gpt_CheckForNULL($('#gptPage').val());
			var gptMasterSite;

			if (gpt_AdFile === 'PU') {
				gptMasterSite = $('#gptSite').val();
			}
			else {
				//JBS-CELL-LANCET
				gptMasterSite = "dev";

				if (gpt_IsProd) {
					gptMasterSite = gpt_CheckForNULL($('#gptSite').val());

					//If there is no master site, check for a defaultsite
					if (gptMasterSite == null) {
						gptMasterSite = gpt_CheckForNULL($('#gptDefaultSite').val());
					}
				}
				else {
					//Dev ENV
					//If the page has no gptSite Value, assume no value which will disable ads
					if (gpt_CheckForNULL($('#gptSite').val()) === null) {
						gptMasterSite = null;
					}
				}

				//If there is no master Page, check for a Page using the URL
				if (gptMasterPage === null) {
					gptMasterPage = GPT_GetPageByURL();
				}
			}

			$(".gpt-ad").each(function (index) {
				var gptPage = '';
				var adSizeMapping = '';
				var gptPOS;
				var gptTC;
				var gptNative;
				var adId;
				var gptSite;
				var tempSlot = null;


                if (gpt_IsProd) {
					if (gpt_AdFile !== "CELLJOURNAL") {
						//Check if the ad slot has it's own site defined
						if (typeof $(this).data('site') === 'undefined') {
							gptSite = gptMasterSite;
						}
						else {
							gptSite = gpt_CheckForNULL($(this).data('site'));
						}
					}
					else {
						gptSite = gptMasterSite;
					}
                }
                else {
					gptSite = gptMasterSite;
                }



				if (gpt_AdFile === 'PU') {
					adId = $(this).attr('id');
				}
				else {
					//JBS-CELL-LANCET
					adId = 'gpt-ad-' + index + 1;
				}

				if (gpt_CheckForNULL(gptSite) !== null) {
					gptSite_Debug = gptSite;

					gptPOS = parseInt($(this).data('pos')) || -1;
					if (gptPOS === -1) {
						gptPOS = index + 1;
					}

					//Check if the ad slot has it's own page defined
					gptPage = $(this).data('page');
					if (gpt_CheckForNULL(gptPage) == null) {
						//Use the Master Page
						gptPage = gptMasterPage;
					}

					//If the page is not null, preceed it with a slash
					if (gptPage !== null) {
						gptPage = '/' + gptPage;
					}
					else {
						gptPage = '';
					}
					gptPage_Debug = gptPage;


					gptTC = $(this).data('tc');
					if (typeof gptTC === "undefined") {
						gptTC = '';
					}

					gptNative = $(this).data('native');
					if (typeof gptNative === "undefined") {
						gptNative = '';
					}

					//Determine sizemapping based on data-adsize
					if (gpt_AdFile === 'PU') {
						switch ($(this).data('adsize')) {
							case 'tall':
								adSizeMapping = gptTallSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600]], adId);
								break;
							case 'rectangle':
								adSizeMapping = gptRectangleSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250]], adId);
								break;
							case 'rectangleshort':
								adSizeMapping = gptRectangleShortSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250], [320, 50], [300, 50]], adId);
								break;
							case 'wide':
								adSizeMapping = gptWideSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[728, 90], [320, 50], [300, 50]], adId);
								break;
							case 'widerectangle':
								adSizeMapping = gptWideRectangleSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[320, 50], [300, 50], [300, 250]], adId);
								break;
							case 'mobile':
								adSizeMapping = gptMobileSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[320, 50], [300, 50]], adId);
								break;
							case 'fluid':
								adSizeMapping = gptFluidMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, 'fluid', adId);
								break;
							case 'prestitial':
								adSizeMapping = gptPrestitialMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[1, 1]], adId);
								break;
						}
					}
					else {
						//JBS-CELL-LANCET
						switch ($(this).data('adsize')) {
							case 'cell-leaderboard':
							case 'leaderboard':
								adSizeMapping = gptLeaderboardSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[728, 90], [320, 50], [300, 50]], adId);
								break;
							case 'bottom':
								if (gpt_ResponsiveHomepage) {
									adSizeMapping = gptHomePageSidebarMapping;
									tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600], [300, 250]], adId);
								}
								else {
									adSizeMapping = gptBoomBoxSizeMapping;
									tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250]], adId);
								}
								break;
							case 'boombox':
								if (gpt_AdFile === "CELLJOURNAL" || gpt_AdFile == "CELLBUCKET") {
									if (gpt_ResponsiveHomepage) {
										adSizeMapping = gptCellHomePageSidebarMapping;
										tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600], [300, 250], [336, 280]], adId);
									}
									else {
										adSizeMapping = gptBoomBoxSizeMapping;
										tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250], [336, 280]], adId);
									}
								}
								else {
									adSizeMapping = gptBoomBoxSizeMapping;
									tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250]], adId);
								}
								break;
							case 'interscroller':
								adSizeMapping = gptInterscrollerMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[320, 480], [480, 320], [768, 1024], [1024, 768]], adId);
								break;
							case 'sidebar':
							case 'skyscraper':
								adSizeMapping = gptSideBarSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600], [300, 250]], adId);
								break;
							case 'native':
							case 'fluid':
								adSizeMapping = gptFluidMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, 'fluid', adId);
								break;
							case 'prestitial':
								adSizeMapping = gptPrestitialMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, 'fluid', adId);
								break;
						}
					}
				}

				//Ignore any invalid adsizes that could be on the page
				if (adSizeMapping !== '') {
					$(this).addClass('gpt-ad-defined');
					if (gpt_AdFile !== 'PU') {
						$(this).attr('id', adId);
					}
					tempSlot.defineSizeMapping(adSizeMapping).addService(googletag.pubads()).setTargeting("pos", gptPOS);

					if (gptTC !== '') {
						tempSlot.setTargeting("tc", gptTC);
					}
					if (gptNative !== '') {
						tempSlot.setTargeting("native", gptNative);
					}
					gptEvent_Debug = gptEvent_Debug + $(this).data('adsize') + ' Requested<br>';
				}
				else {
					gptInvalidAdSize_Debug = gptInvalidAdSize_Debug + "Invalid Size Found, Id: " + $(this).prop('id') + ', adSize: ' + $(this).data('adsize') + '<br>';
					$(this).remove();
				}
			});
			if (gpt_CheckForNULL(gptInvalidAdSize_Debug) === null) {
				gptInvalidAdSize_Debug = "None Detected";
			}
			//***********************************************
			//End detection for ad slots on page
			//***********************************************

			//***********************************************
			//Start page level targeting
			//***********************************************
			if (gpt_AdFile !== 'PU') {
				//Logged in or IP Access
				gpt_TargetAccessType();

				//Check for Screenshot Client
				gpt_CheckSSC();
			}

			//Check for DFP Test
			gpt_GetDFPTestId();

			//Check for Additional Targeting
			gpt_GetAdditionalTargeting();

			if (gpt_IsProd && gptContextualEnabled) {
				var gpt_ContextualRetry = false;
				//Check for Contextual Targeting
				if (gs_channels !== "DEFAULT") {
					var gpt_ContextualCategories = Array.from(gs_channels);
					for (var i = 0; i < gpt_ContextualCategories.length; i++) {
						if (gpt_ContextualCategories[i] == 'gx_retry') {
							gpt_ContextualRetry = true;
						}
						if (!gpt_ContextualCategories[i].startsWith("custom_")) {
							//Remove any tag that is not custom
							gpt_ContextualCategories.splice(i, 1);
							i--;
						}
						else {
							//Strip the 'custom_' from the tag
							gpt_ContextualCategories[i] = gpt_ContextualCategories[i].replace("custom_", "")
						}
					}
					if (gpt_ContextualCategories.length > 0) {
						googletag.pubads().setTargeting("gs_cat", gpt_ContextualCategories);
					}
					else {
						gpt_ContextualCategories = 'none';
					}
					if (gpt_ContextualRetry) {
						gpt_ContextualCategories = "*Indexing*";
					}
					gptContextualResult_Debug = gpt_ContextualCategories;
				}
				else {
					gptContextualResult_Debug = 'No segments found';
				}
			}
			else {
				//Not Prod || Contextual not enabled
				gptContextualResult_Debug = 'Not enabled';
				
				if (!gpt_IsProd) {
                    var gpt_DevSite = gpt_CheckForNULL($('#gptSite').val(), "dev");
                    $('#gpt-Variables').append('<input type="hidden" id="gptEnvironment" value="' + gpt_DevSite + '">');
                    gpt_Target("Environment");
                }

			}
			//***********************************************
			//End page level targeting
			//***********************************************

			//Enable google dfp services
			if (parseInt($('#gptLazyLoading').val()) === 1) {
				googletag.pubads().enableLazyLoad({
					fetchMarginPercent: 200,
					renderMarginPercent: 0,
					mobileScaling: 2.0
				});
			}
			else {
				googletag.pubads().enableSingleRequest();
			}

			//Add Listener for RenderEnded
			googletag.pubads().addEventListener('slotRenderEnded', function (event) {
				if (!event.isEmpty) {
					if (typeof LocalAdLoaded === "function") {
						LocalAdLoaded(event);
					}
					if (gpt_ResponsiveHomepage) {
						if (typeof positionAd === "function") {
							positionAd();
						}
					}
					if (event.size[0] == "480" || event.size[1] == "480") {
						$('.gpt-InScrollBox').css('display', 'block');
						gptEvent_Debug = gptEvent_Debug + 'Interscroller ' + event.size + ' rendered: ' + gpt_GetElapsedTime() + '<br>';
					}
					else {
						gptEvent_Debug = gptEvent_Debug + event.size + ' rendered: ' + gpt_GetElapsedTime() + '<br>';
					}

				}
			});

			if (gpt_AdFile !== 'PU') {
				googletag.pubads().collapseEmptyDivs();
			}
			googletag.enableServices();

			//Populate Defined Ad Slots
			$(".gpt-ad-defined").each(function (index) {
				var adId = $(this).attr('id');
				googletag.cmd.push(function () { googletag.display(adId); });
			});

			gptStatus_Debug = 'GPT Complete';
		});
	}
	//--------------------------------------------------------------------------
	function gpt_DisplayDebug() {
		//Clone the template and append to Body
		$(document.body).append($('#gpt-DebugDiv').clone());
		$('#gpt-DebugDiv').remove();

		//Populate the Values
		gpt_LoadDebugValues();

		//Display and scroll to debug section
		$('.gpt-DebugDiv').css('display', 'block');
		window.scrollTo(0, document.body.scrollHeight);
	}
	//--------------------------------------------------------------------------
	function gpt_LoadDebugValues() {
		$('.gpt-DebugBtn').blur();
		$('.gptVersion_Debug').text(gpt_Version);
		$('.gptAdFile_Debug').html(gpt_AdFile);
		$('.gptEvent_Debug').html(gptEvent_Debug);
		$('.gptCustomTargeting_Debug').html(gptCustomTargeting_Debug);
		$('.gptSite_Debug').text(gptSite_Debug);
		$('.gptContextualResult_Debug').text(gptContextualResult_Debug);
		$('.gptContextualInit_Debug').text(gptContextualInit_Debug);
		$('.gptContextualURL_Debug').text(gptContextualURL_Debug);
		$('.gptContextualReady_Debug').text(gptContextualReady_Debug);
		$('.gptMoatReady_Debug').text(gptMoatReady_Debug);
		$('.gptMoatInit_Debug').text(gptMoatInit_Debug);
		$('.gptTestENV_Debug').text(gptTestENV_Debug);
		$('.gptDisableAds_Debug').text(gptDisableAds_Debug);
		$('.gptStatus_Debug').text(gptStatus_Debug);
		$('.gptPage_Debug').text(gptPage_Debug);
		$('.gptInvalidAdSize_Debug').html(gptInvalidAdSize_Debug);
	}
	//--------------------------------------------------------------------------
	function gpt_GrapeShotDebug() {
		$('.gpt-DebugBtn').blur();
		window.open("http://admin-elsevier.grapeshot.co.uk/custom/bookmarklet.cgi?blocking=1&vn=2&url=" + encodeURIComponent(document.location), "GRAPESHOT_POPUP", "width=650,height=600,scrollbars=yes,screenX=40,screenY=50,menubar=no,toolbar=no,resizable=yes", "Custom").focus();
	}
	//--------------------------------------------------------------------------
	function gpt_ShowConsole() {
		$('.gpt-DebugBtn').blur();
		googletag.openConsole();
	}
	//--------------------------------------------------------------------------
</script>
        </div>

            
                </div>
            

        

    

        


    

            

            

            <div data-widget-id="912135d6-7135-4d45-b457-b503cacc3c8b" data-widget-name="ux3-layout-widget"><header class="header base fixed">
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none adplaceholder adplaceholder--header js__adToHide widget-horizontalAd" id="9d6062d7-fad6-4e64-81cc-7dd25d99b23d">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-horizontalAd "><div class="horizontalAdContainer">
    <div class="horizontalAdLabelContainer">
        <div class="horizontalAdLabel">
                <img src="/templates/jsp/_style2/_marlin/images/bg_adHoriz.gif" alt="Advertisement">
        </div>
    </div>
    <div class="horizontalAd">
        





        <div class="pb-ad">
            <!-- Beginning Async AdSlot 1 for Ad unit els.cellpress/cell  ### size: [[728,90]] -->
<div class="gpt-ad" data-adsize="cell-leaderboard" data-page="" data-site="cell" data-pos="1"></div>
<!-- End AdSlot 1 -->
        </div>
    

    </div>
</div></div>
                </div>
                </div>

        










        <div data-widget-def="ux3-layout-widget" data-widget-id="3a83e2a6-2e7e-4931-892c-4ef43b6e3312" class="header__wrapper clearfix" data-db-parent-of="quickSearch"><div id="brandingRow"><a href="/" data-aa-name="science that inspires" data-aa-region="cp global header" title="Visit Cell.com">Science that inspires</a><p id="brandingRowRight"><a href="/" data-aa-name="cell press endorsement" data-aa-region="cp global header" title="Visit Cell.com">A Cell Press journal</a></p></div>
        
    

        


    

            

            

            <div data-widget-id="3a83e2a6-2e7e-4931-892c-4ef43b6e3312" data-widget-name="ux3-layout-widget"><div class="col-xs-6 header__left-side">
    

        


    

            

            
                <a href="/home" id="cpIconLnk" data-aa-name="cell press logo" data-aa-region="cp global header" title="Cell Press" target="_blank"><svg id="cp_logo" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 100 98.1" style="enable-background:new 0 0 100 98.1;" xml:space="preserve" height="100" width="98.1"><path style="fill:#FFFFFF;" d="M69.09,0.09c-4.01-0.6-7.83,1.94-8.78,5.89l-3.14,13.11c-0.76,3.19-3.44,5.56-6.7,5.93l-18.6,2.13          C11.83,29.31-2.53,47.66,0.37,67.78c2.3,15.96,15.18,27.77,30.54,30.08c4.01,0.6,7.83-1.94,8.78-5.89l3.14-13.11 c0.76-3.19,3.44-5.56,6.69-5.93l18.6-2.13c20.04-2.17,34.41-20.52,31.5-40.65C97.32,14.21,84.45,2.39,69.09,0.09z M65.06,47.2  l-11.75,1.4c-1.98,0.21-3.62,1.65-4.08,3.59l-3.43,14.3c-1.01,4.22-4.5,7.39-8.8,8c-6.66,0.72-12.63-4.2-13.15-10.92                c-0.51-6.5,4.6-12.11,11.08-12.81l11.75-1.39c1.98-0.22,3.62-1.65,4.08-3.59l3.43-14.3c1.01-4.22,4.5-7.39,8.8-8          c6.66-0.72,12.63,4.19,13.15,10.92C76.66,40.89,71.55,46.5,65.06,47.2z"></path></svg></a><div class="journal-logos">
            

            <span data-widget-id="35d8d734-3aa0-4ec8-9e2d-14babe803594" data-widget-name="ux3-general-image"><a href="/cell/home" title="Cell"><img alt="Cell" src="/pb-assets/ux3/logos/cell/Cell_white-1532516611593.svg"></a></span>

            
                </div>
            

        

    

        


    

            

            
                <div class="journal-logos">
            

            <span data-widget-id="3ffcbf1f-79fd-4ba3-85b5-bef10a63475f" data-widget-name="ux3-general-image"><a href="https://www.cell.com/open-access?utm_source=oa-label&amp;__hstc=109691226.e4e812b53ff324eccbdbdb2e4fb9a6ab.1644503737405.1644524234622.1644587109507.4&amp;__hssc=109691226.7.1644587109507&amp;__hsfp=3253153019" title="This journal offers authors two options (open access or subscription) to publish research"><img alt="This journal offers authors two options (open access or subscription) to publish research" src="/pb-assets/ux3/logos/cell/Supports_open_access_border-1645554937897.svg"></a></span>

            
                </div>
            

        
</div><div class="col-xs-6">
    

        


    

            

            

            <div data-widget-id="2555f6db-b333-4edb-8b15-ef9213f1f0a5" data-widget-name="ux3-layout-widget"><div class="header__nav row">
    

        


    

            

            

            <ul data-widget-id="52432d24-0331-4c9d-85f3-2033edc7208b" data-widget-name="UX3ProfileLinksWidget" class="rlist rlist--inline profile-links"><li class="profile-links__submit"><a data-adobe-name="Publish" class="profile-links__submit__btn" title="Submit" href="https://www.editorialmanager.com/cell" target="_self" data-aa-conversiondriver="jbs:global-header:submitpage" data-profile-link="true">Submit</a></li><li class="profile-links__login-dropdown"><div class="profile-links__user-list__wrapper"><button data-db-target-for="loginmenu-300441" aria-expanded="false" aria-controls="login-links-300441" title="Log in links dropdown" class="profile-links__username"><span>Log in</span></button><div data-db-target-of="loginmenu-300441" id="login-links--300441" class="profile-links__user-list__holder dropBlock__holder"><ul class="rlist profile-links__list"><li class="profile-links__login"><a data-adobe-name="Log In" class="profile-links__login__btn" title="Log in to Cell.com" href="https://www.cell.com/action/idLogin?type=login&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2901332-5" target="_self" data-profile-link="true">Log in</a></li><li class="profile-links__institution"><a data-adobe-name="Get institutional access" class="profile-links__institution__btn" title="Institutional access" href="https://www.cell.com/action/idLogin?type=institution&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2901332-5" target="_self" data-profile-link="true">Get Institutional Access</a></li></ul></div></div></li><li class="profile-links__register"><a data-adobe-name="Register" class="profile-links__register__btn" title="Register a username and password for free" href="https://www.cell.com/action/idLogin?type=registration&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2901332-5" target="_self" data-profile-link="true">Register</a></li><li class="hidden-lg"> <div class="profile-links__controls"> <button aria-expanded="false" aria-controls="links-dialog-300441" data-opens="#links-dialog-300441" class="profile-links__btn--open"><span class="sr-only">open links dialog</span><i aria-hidden="true" class="icon-gizmo-person"></i></button></div><dialog id="links-dialog-300441" aria-modal="true"><button data-closes="#links-dialog-300441" class="profile-links__btn--close"><span class="sr-only">close links dialog</span><i aria-hidden="true" class="icon-close-solid"></i></button><ul>   <li class="profile-links__submit"><a data-adobe-name="Publish" class="profile-links__submit__btn" title="Submit" href="https://www.editorialmanager.com/cell" target="_self" data-aa-conversiondriver="jbs:global-header:submitpage" data-profile-link="true">Submit</a></li><li class="profile-links__login"><a data-adobe-name="Log In" class="profile-links__login__btn" title="Log in to Cell.com" href="https://www.cell.com/action/idLogin?type=login&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2901332-5" target="_self" data-profile-link="true">Log in</a></li><li class="profile-links__institution"><a data-adobe-name="Get institutional access" class="profile-links__institution__btn" title="Institutional access" href="https://www.cell.com/action/idLogin?type=institution&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2901332-5" target="_self" data-profile-link="true">Get Institutional Access</a></li><li class="profile-links__register"><a data-adobe-name="Register" class="profile-links__register__btn" title="Register a username and password for free" href="https://www.cell.com/action/idLogin?type=registration&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2901332-5" target="_self" data-profile-link="true">Register</a></li></ul></dialog></li></ul>

            

        
</div></div>

            

        
</div></div>

            

        

        </div>
    

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget csasInstitutionBanner none  widget-none" id="51b7fa33-2137-4f58-9e98-9746c4400513" style="display: none;">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none "><div class="ui-state-highlight ui-corner-all csas-institution-banner hidden" data-doi="10.1016/j.cell.2021.11.017">
    <span class="ui-icon ui-icon-close banner-close"></span>
    <p>
        Access provided by <span class="accountName"></span>
    </p>
</div>

<style>
    .csas-institution-banner {
        padding: 10px 0 0 10px;
        margin: 5px;
    }

    .csas-institution-banner.ui-state-highlight .ui-icon {
        background-image: url("/templates/jsp/css/jquery-ui-1.10.2/base/images/ui-icons_454545_256x240.png");
        margin-right: 5px;
        margin-top: -4px;
    }

    .csas-institution-banner .banner-close {
        float: right;
    }
</style>

<script>

    function resetBanner(doi) {
        $.get('/action/getAccount?doi=' + doi, function(data) {
            if (data) {
                $(".csas-institution-banner p .accountName").html(data);
                $(".csasInstitutionBanner").fadeIn();
                $(".csas-institution-banner").fadeIn();
            }
        });
    }

    var flag = 0;
    $(document).ready(function() {
        if(!$(".csas-institution-banner").is(":visible")){
            $(".csasInstitutionBanner").hide();
        }

        $(".banner-close").click(function() {
            $.get("/action/dismissBanner", function(data) {
                $(".csasInstitutionBanner").fadeOut(function () {
                    if(typeof UX !== 'undefined') {
                        UX.pageBody.init();
                    }
                });
            });
        });

        $(".articleContent .tabs li[role='tab'].showFullText").click(function() {
           flag = 1;
        });

        $(document).on("reloadTabsContent", function() {
            var isSelected = $(".articleContent .tabs li[role='tab'].showFullText").hasClass("active");
            if (isSelected) {
                var isHidden =  $(".csas-institution-banner").is(".hidden");
                if (!isHidden) {
                    resetBanner($(".csas-institution-banner").data("doi"));
                    console.log("fadeIn");
                }
            } else {
                $(".csas-institution-banner").fadeOut();
                console.log("fadeOut");
            }
            flag = 0;
        });

        $(document).on("click", ".articleContent .tabs li[role='tab']", function() {

        });
    });
</script></div>
                </div>
                </div>

        
</header></div>

            

        










        <div data-widget-def="graphQueryWidget" data-widget-id="2a125dae-6dbc-4e76-a75e-3e563f1d3a1d" class="ux3">
        
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget graphQueryWidget none top-padding-handler noMargin axel-mega-menu widget-none widget-compact-all fonts-loaded" id="2a125dae-6dbc-4e76-a75e-3e563f1d3a1d" style="padding-top: 115.594px;">
                
                
                    
                

                <div class="wrapped " id="CP TOP LEVEL NAV MENU">
                    
                    <div class="widget-body body body-none  body-compact-all"><nav id="mega-menu-nav" role="navigation" class="mega-menu-nav">
  <!--.skip//a(href='#searchText' aria-label='Skip menu') Skip menu
  -->
  <div class="mega-menu-nav__wrapper">
    <button class="mega-menu__toggle"><span class="sr-only">Main menu</span><i aria-hidden="true" class="icon-gizmo-hamburger"></i></button>
    <ul role="menubar" class="mega-menu rlist">
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-1" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-1" data-item-link="main-menu-link-1" data-index="0" class="mega-menu__item-link menu-link"><span>This journal</span></a>
        <div id="main-menu-panel-1" aria-labelledby="main-menu-link-1" data-item-wrapper="main-menu-link-1" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <div aria-label="Menu dropzone section" tabindex="-1" class="listDropZone">
    

        


    

            

            

            <div class="cover-image"><div class="cover-image__thumb"><a href="/issue/S0092-8674(24)X0012-4" class="menu-link"><img src="/cms/asset/atypon:cms:attachment:img:d5e6:rev:1748270185670-389:pii:S0092867424X00124/cover.tif.jpg" alt="Cover Image - Cell, Volume 188, Issue 11"></a></div><div class="cover-image__meta"></div></div>

            

        
</div>
            </li>
            <li style="grid-column: 2/3" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-1-2" class="sub-menu__heading">Read
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/newarticles" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Online now</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/current" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Current issue</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/archive" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Archive</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/action/showPreferences?menuTab=Alerts" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Publication alerts</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/sneakpeek" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Sneak Peek - preprints</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 3/5" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-1-3" class="sub-menu__heading">About
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-2">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/advisory-board" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Advisory board</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/aims" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Aims and scope</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/contact" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Contact</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/editors-and-staff" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Editors and staff</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/editorial-ethics" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Ethics</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/information-for-authors/permissions" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Permissions</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/reviewers" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>Reviewer instructions</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-2" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-2" data-item-link="main-menu-link-2" data-index="1" class="mega-menu__item-link menu-link"><span>Journals</span></a>
        <div id="main-menu-panel-2" aria-labelledby="main-menu-link-2" data-item-wrapper="main-menu-link-2" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/5" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-2-1" class="sub-menu__heading">Life &amp; medical sciences
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-4">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/home" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cancer-cell/home" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Cancer Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-chemical-biology" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Chemical Biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-genomics/home" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Genomics</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-host-microbe/home" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Host &amp; Microbe</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-metabolism/home" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Metabolism</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports/home" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-medicine/home" data-index="7" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Medicine</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-stem-cell/home" data-index="8" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Stem Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-systems/home" data-index="9" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Systems</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/current-biology/home" data-index="10" role="menuitem" class="sub-menu__item-link menu-link"><span>Current Biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/developmental-cell/home" data-index="11" role="menuitem" class="sub-menu__item-link menu-link"><span>Developmental Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/immunity/home" data-index="12" role="menuitem" class="sub-menu__item-link menu-link"><span>Immunity</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/med/home" data-index="13" role="menuitem" class="sub-menu__item-link menu-link"><span>Med</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-cell/home" data-index="14" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/neuron/home" data-index="15" role="menuitem" class="sub-menu__item-link menu-link"><span>Neuron</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/structure/home" data-index="16" role="menuitem" class="sub-menu__item-link menu-link"><span>Structure</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/ajhg/home" data-index="17" role="menuitem" class="sub-menu__item-link menu-link"><span>American Journal of Human Genetics (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/biophysj/home" data-index="18" role="menuitem" class="sub-menu__item-link menu-link"><span>Biophysical Journal (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/biophysreports/home" data-index="19" role="menuitem" class="sub-menu__item-link menu-link"><span>Biophysical Reports (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/hgg-advances/home" data-index="20" role="menuitem" class="sub-menu__item-link menu-link"><span>Human Genetics and Genomics Advances (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-plant/home" data-index="21" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Plant (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/molecular-therapy/home" data-index="22" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/methods/home" data-index="23" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy Methods &amp; Clinical Development (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/nucleic-acids/home" data-index="24" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy Nucleic Acids (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/oncology/home" data-index="25" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy Oncology (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/plant-communications/home" data-index="26" role="menuitem" class="sub-menu__item-link menu-link"><span>Plant Communications (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/stem-cell-reports/home" data-index="27" role="menuitem" class="sub-menu__item-link menu-link"><span>Stem Cell Reports (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/biochemical-sciences/home" data-index="28" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Biochemical Sciences</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/cancer/home" data-index="29" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Cancer</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/cell-biology/home" data-index="30" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Cell Biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/ecology-evolution/home" data-index="31" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Ecology &amp; Evolution</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/endocrinology-metabolism/home" data-index="32" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Endocrinology &amp; Metabolism</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/genetics/home" data-index="33" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Genetics</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/immunology/home" data-index="34" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Immunology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/microbiology/home" data-index="35" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Microbiology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/molecular-medicine/home" data-index="36" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Molecular Medicine</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/neurosciences/home" data-index="37" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Neurosciences</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/parasitology/home" data-index="38" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Parasitology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/pharmacological-sciences/home" data-index="39" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Pharmacological Sciences</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/plant-science/home" data-index="40" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Plant Science</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 5/6" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-2-2" class="sub-menu__heading">Physical sciences &amp; engineering
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-biomaterials/home" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Biomaterials</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-physical-science/home" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Physical Science</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/chem/home" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Chem</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/chem-catalysis/home" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Chem Catalysis</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/device/home" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Device</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/joule/home" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Joule</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/matter/home" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>Matter</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/newton/home" data-index="7" role="menuitem" class="sub-menu__item-link menu-link"><span>Newton</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/chemistry/home" data-index="8" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Chemistry</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 6/7" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-2-3" class="sub-menu__heading">Multidisciplinary
              </h4>
              <ul role="menubar" data-index="2" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-methods/home" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Methods</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-sustainability/home" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Sustainability</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/heliyon/home" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Heliyon</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/iscience/home" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>iScience</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/one-earth/home" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>One Earth</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/patterns/home" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Patterns</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/star-protocols/home" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>STAR Protocols</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/nexus/home" data-index="7" role="menuitem" class="sub-menu__item-link menu-link"><span>Nexus (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/the-innovation/home" data-index="8" role="menuitem" class="sub-menu__item-link menu-link"><span>The Innovation (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/biotechnology/home" data-index="9" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Biotechnology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/cognitive-sciences/home" data-index="10" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Cognitive Sciences</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-3" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-3" data-item-link="main-menu-link-3" data-index="2" class="mega-menu__item-link menu-link"><span>Publish</span></a>
        <div id="main-menu-panel-3" aria-labelledby="main-menu-link-3" data-item-wrapper="main-menu-link-3" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-3-1" class="sub-menu__heading">Authors
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="https://www.editorialmanager.com/cell?__hstc=25856994.7201670a4397fa03148684920d4a8d38.1672762013831.1694716659709.1694718909620.146&amp;__hssc=25856994.15.1694718909620&amp;__hsfp=1366844671" data-index="0" target="_blank" role="menuitem" class="sub-menu__item-link menu-link"><span>Submit article
                      <div class="external-icon"></div><span class="sr-only">External Link</span></span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/information-for-authors" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Information for authors</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/open-access" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Open access</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/multi-journal-submission" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Multi-Journal Submission</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/information-for-authors/star-authors-guide" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>STAR Methods</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 2/3" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-3-2" class="sub-menu__heading">Reviewers
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/reviewers" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Information for reviewers</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-4" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-4" data-item-link="main-menu-link-4" data-index="3" class="mega-menu__item-link menu-link"><span>News &amp; events</span></a>
        <div id="main-menu-panel-4" aria-labelledby="main-menu-link-4" data-item-wrapper="main-menu-link-4" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/newsroom" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Newsroom</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/events" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Events</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/symposia" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Symposia</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/webinars" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Webinars</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/consortia" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Consortia Hub</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-5" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-5" data-item-link="main-menu-link-5" data-index="4" class="mega-menu__item-link menu-link"><span>About Cell Press</span></a>
        <div id="main-menu-panel-5" aria-labelledby="main-menu-link-5" data-item-wrapper="main-menu-link-5" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-5-1" class="sub-menu__heading">Cell Press
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/about" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>About Cell Press</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/contact" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Contact us</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="https://service.elsevier.com/app/overview/cell-press/" data-index="2" target="_blank" role="menuitem" class="sub-menu__item-link menu-link"><span>Help &amp; support
                      <div class="external-icon"></div><span class="sr-only">External Link</span></span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/open-access" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Open access at Cell Press</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/diversity" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Inclusion and diversity</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-press-50" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>50th anniversary</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 2/3" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-5-2" class="sub-menu__heading">Hubs
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/biotechnology-bioengineering" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Biotechnology and bioengineering</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-press-chemistry" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Chemistry</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/clinical-translational" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Clinical and translational</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/covid-19" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>COVID-19</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/fundamental-biology" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Fundamental biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/sustainability" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Sustainability</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 3/4" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-5-3" class="sub-menu__heading">Careers
              </h4>
              <ul role="menubar" data-index="2" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="https://relx.wd3.myworkdayjobs.com/en-US/ElsevierJobs" data-index="0" target="_blank" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Press careers
                      <div class="external-icon"></div><span class="sr-only">External Link</span></span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
    </ul>
    <div class="mega-menu-nav__quick-search">
    

        


    

            

            

            <div data-widget-id="a1f05748-166f-41e0-b01a-ca8ac1cc95d4" data-widget-name="UX3QuickSearchWidget" class="quick-search"><div class="dropBlock"><form action="https://www.cell.com/action/doSearch?journalCode=cell" name="quickSearchBar" method="get" title="Quick Search"><input id="type" type="hidden" value="quicksearch" name="type"><div class="quick-search__toggle autoCompleteWrapper"><input id="searchText" type="text" name="text1" placeholder="Search for..." enterkeyhint="search" autocomplete="off" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" data-group-titles-conf="3" data-publication-items-conf="3" aria-label="Enter a search term" class="searchText searchText-autocomplete ui-autocomplete-input"><button type="submit" title="Search" aria-label="Submit the search form" class="btn quick-search__button icon-search"><span>Go search</span></button><a href="https://www.cell.com/search/advanced?SeriesKey=cell&amp;ISSN=0092-8674&amp;journalCode=cell" title="Advanced search" class="quick-search__advanced-link">Advanced search</a><ul id="ui-id-1" tabindex="0" class="ui-menu ui-widget ui-widget-content ui-autocomplete ui-front" unselectable="on" style="display: none;"></ul></div><div class="quick-search__box"><div class="quick-search__box-inner"><div class="quick-search__box-wrapper"><select name="field1" aria-label="Select a search area"><option value="AllField">All content</option><option value="Title">Article title</option><option value="Contrib">Authors</option><option value="Keyword">Keywords</option><option value="Abstract">Abstract</option><option value="AbstractTitleKeywordFilterField">Article title, abstract, keywords</option></select><input type="hidden" name="journalCode" value="cell"><div class="quick-search__scope-options"><span>Search within:</span><label title="This journal" class="radio--primary"><input type="radio" name="SeriesKey" id="searchScopeSeries-a1f05748-166f-41e0-b01a-ca8ac1cc95d4" title="This journal" value="cell" checked="checked" aria-label="Search within current journal"><span>This journal</span></label><label title="Full site" class="radio--primary"><input type="radio" name="SeriesKey" id="searchScopeFullSite-a1f05748-166f-41e0-b01a-ca8ac1cc95d4" value="" title="Full site" aria-label="Search within full site"><span>Full site</span></label></div><a href="https://www.cell.com/search/advanced?SeriesKey=cell&amp;ISSN=0092-8674&amp;journalCode=cell" title="Advanced search" class="quick-search__advanced-link">Advanced search</a></div><a href="javascript:void(0)" aria-label="Close search options" title="Close search options" class="quick-search__box-close"><i aria-hidden="true" class="icon-tools_close"></i></a></div></div><div class="quick-search__alert hidden"><p>Please enter a term before submitting your search.</p><a href="#" class="quick-search__alert--close"> Ok</a></div></form></div></div>

            

        
</div>
  </div>
</nav>
<div id="page-body-id" class="page-body-flag"></div></div>
                </div>
                </div>

        

        </div>
    

    

        


    

            

            

            <div data-widget-id="f8c6daf9-9df1-4fd9-ba3a-7f5d7b3d4466" data-widget-name="UX3HTMLWidget"><style>
.journal-logos:nth-of-type(2) img {
max-height: 30px !important;
}
.journal-logos:nth-of-type(3) img {
max-height: 30px !important;
}
.ux3 .journal-logos + .journal-logos {
margin-left: 30px;
}
.header__wrapper.clearfix .col-xs-6:first-of-type {
width: 55%;
}
.header__wrapper.clearfix .col-xs-6:nth-of-type(2) {
width: 45%;
}
@media screen and (max-width: 1000px) {
.ux3 .journal-logos + .journal-logos {
display: none;
}
.header__wrapper.clearfix .col-xs-6:first-of-type {
width: 50%;
}
.header__wrapper.clearfix .col-xs-6:nth-of-type(2) {
width: 50%;
}
}

.ux3 .journal-logos:first-of-type {
   max-width: 500px;
}

.profile-links {
   white-space: nowrap;
}

@media only screen and (min-width: 740px) and (max-width: 1600px) {
   .journal-logos:nth-of-type(2) img {
      max-width: 250px !important;
   }
}

@media only screen and (min-width: 1024px) and (max-width: 1600px) {
   .header__wrapper.clearfix .col-xs-6:first-of-type {
      width: 60% !important;
   }
   .header__wrapper.clearfix .col-xs-6:nth-of-type(2) {
      width: 40% !important;
   }
}

@media only screen and (max-width: 1240px) {
   .ux3 .journal-logos + .journal-logos {
      display: none !important;
   }
}

</style></div>

            

        
</div></div>

            
                </div>
            

        

        </div>
    

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget axelPublicationContent none  widget-none" id="cadc9b41-a542-448a-9de4-8d0803320174">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none "><link id="build-style-article" rel="stylesheet" type="text/css" href="/products/marlin/cell/releasedAssets/css/build-article-b26f3342b722959c7931.css"><article xmlns="http://www.w3.org/1999/xhtml" data-design="pill" data-has="right-rail" vocab="http://schema.org/" typeof="ScholarlyArticle" data-pii="S0092-8674(21)01332-5" lang="en" dir="ltr"><div data-core-wrapper="header"><header data-extent="frontmatter"><div class="core-container"><div data-article-access="full" data-article-access-type="open" class="meta-panel"><span class="meta-panel__left-content"><span class="meta-panel__type"><span>Article</span></span></span><span class="meta-panel__middle-content"><span class="meta-panel__volumeIssue"><a href="/cell/issue?pii=S0092-8674(20)X0026-2"><span>Volume 184</span>, <span>Issue 25</span></a></span><span class="meta-panel__pages"><span>p6119-6137.e26</span></span><span class="meta-panel__onlineDate">December 09, 2021</span></span><span class="meta-panel__right-content"><span class="meta-panel__access meta-panel__access--open"><span>Open access</span></span></span><div class="article-header__download-full-issue"><button title="Download full issue" type="submit" formaction="/action/downloadMultiplePdf?action=downloadIssuePdf&amp;journal=cell&amp;volume=184&amp;issue=25&amp;pii=S0092-8674%2820%29X0026-2" class="btn--download-issue btn--cancel download-full-issue"><i class="icon-download-new"></i><span>Download Full Issue</span></button><div class="table-of-content__alert table-of-content__alert--download"><p>Download started</p><a href="#" class="table-of-content__alert--close">Ok</a></div></div></div><h1 property="name">Microenvironment drives cell state, plasticity, and drug response in pancreatic cancer</h1><div class="contributors"><span class="authors"><span role="list"><span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au1" aria-controls="axel_author_au1"><span property="givenName">Srivatsan</span> <span property="familyName">Raghavan</span></a><div id="au1" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Srivatsan</span> <span property="familyName">Raghavan</span></div><div class="content"><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">20</div><div id="fn1"><div id="ntpara0010" role="paragraph">These authors contributed equally</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Medical Oncology, Dana-Farber Cancer Institute, Boston, MA 02215, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Broad Institute of MIT and Harvard, Cambridge, MA 02142, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Harvard Medical School, Boston, MA 02115, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Medicine, Brigham and Women’s Hospital, Boston, MA 02115, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Raghavan/Srivatsan" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">2</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">3</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">4</a>,<a role="doc-noteref">20</a></sup></span> ∙ <span property="author" typeof="Person" role="listitem" class="corresponding-author"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au2" aria-controls="axel_author_au2"><span property="givenName">Peter S.</span> <span property="familyName">Winter</span></a><div id="au2" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Peter S.</span> <span property="familyName">Winter</span></div><div class="content"><div class="core-author-emails"><div class="label">Correspondence</div><div>Corresponding author</div><div><a id="eadd0010" href="mailto:pswinter@mit.edu" property="email" title="Send email to Peter S. Winter">pswinter@mit.edu</a></div></div><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">20</div><div id="fn1"><div id="ntpara0010" role="paragraph">These authors contributed equally</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Medical Oncology, Dana-Farber Cancer Institute, Boston, MA 02215, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Broad Institute of MIT and Harvard, Cambridge, MA 02142, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Institute for Medical Engineering &amp; Science, Massachusetts Institute of Technology, Cambridge, MA 02139, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Koch Institute for Integrative Cancer Research, Massachusetts Institute of Technology, Cambridge, MA 02139, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Chemistry, Massachusetts Institute of Technology, Cambridge, MA 02139, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Winter/Peter+S" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">2</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">5</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">6</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">7</a>,<a role="doc-noteref">20</a></sup> <a id="eadd0010" href="mailto:pswinter@mit.edu" property="email" title="Send email to Peter S. Winter">pswinter@mit.edu</a></span> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au3" aria-controls="axel_author_au3"><span property="givenName">Andrew W.</span> <span property="familyName">Navia</span></a><div id="au3" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Andrew W.</span> <span property="familyName">Navia</span></div><div class="content"><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">20</div><div id="fn1"><div id="ntpara0010" role="paragraph">These authors contributed equally</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Medical Oncology, Dana-Farber Cancer Institute, Boston, MA 02215, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Broad Institute of MIT and Harvard, Cambridge, MA 02142, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Institute for Medical Engineering &amp; Science, Massachusetts Institute of Technology, Cambridge, MA 02139, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Koch Institute for Integrative Cancer Research, Massachusetts Institute of Technology, Cambridge, MA 02139, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Chemistry, Massachusetts Institute of Technology, Cambridge, MA 02139, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Navia/Andrew+W" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">2</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">5</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">6</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">7</a>,<a role="doc-noteref">20</a></sup></span><span data-displayed-on="all"> ∙ … </span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au4" aria-controls="axel_author_au4"><span property="givenName">Hannah L.</span> <span property="familyName">Williams</span></a><div id="au4" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Hannah L.</span> <span property="familyName">Williams</span> <a class="orcid-id" href="https://orcid.org/0000-0003-0434-3900" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0003-0434-3900</a></div><div class="content"><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">20</div><div id="fn1"><div id="ntpara0010" role="paragraph">These authors contributed equally</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Medical Oncology, Dana-Farber Cancer Institute, Boston, MA 02215, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Harvard Medical School, Boston, MA 02115, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Williams/Hannah+L" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">3</a>,<a role="doc-noteref">20</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au5" aria-controls="axel_author_au5"><span property="givenName">Alan</span> <span property="familyName">DenAdel</span></a><div id="au5" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Alan</span> <span property="familyName">DenAdel</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Center for Computational Molecular Biology, Brown University, Providence, RI 02912, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Division of Applied Mathematics, Brown University, Providence, RI 02912, USA</span></div></div><div class="core-author-link"><a href="/authored-by/DenAdel/Alan" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">8</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">9</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au6" aria-controls="axel_author_au6"><span property="givenName">Kristen E.</span> <span property="familyName">Lowder</span></a><div id="au6" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Kristen E.</span> <span property="familyName">Lowder</span> <a class="orcid-id" href="https://orcid.org/0000-0003-2615-4043" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0003-2615-4043</a></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Medical Oncology, Dana-Farber Cancer Institute, Boston, MA 02215, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Broad Institute of MIT and Harvard, Cambridge, MA 02142, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Lowder/Kristen+E" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">2</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au7" aria-controls="axel_author_au7"><span property="givenName">Jennyfer</span> <span property="familyName">Galvez-Reyes</span></a><div id="au7" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au7"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Jennyfer</span> <span property="familyName">Galvez-Reyes</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Broad Institute of MIT and Harvard, Cambridge, MA 02142, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Institute for Medical Engineering &amp; Science, Massachusetts Institute of Technology, Cambridge, MA 02139, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Koch Institute for Integrative Cancer Research, Massachusetts Institute of Technology, Cambridge, MA 02139, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Galvez-Reyes/Jennyfer" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">2</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">5</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">6</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au8" aria-controls="axel_author_au8"><span property="givenName">Radha L.</span> <span property="familyName">Kalekar</span></a><div id="au8" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au8"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Radha L.</span> <span property="familyName">Kalekar</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Medical Oncology, Dana-Farber Cancer Institute, Boston, MA 02215, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Broad Institute of MIT and Harvard, Cambridge, MA 02142, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Kalekar/Radha+L" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">2</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au9" aria-controls="axel_author_au9"><span property="givenName">Nolawit</span> <span property="familyName">Mulugeta</span></a><div id="au9" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au9"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Nolawit</span> <span property="familyName">Mulugeta</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Broad Institute of MIT and Harvard, Cambridge, MA 02142, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Institute for Medical Engineering &amp; Science, Massachusetts Institute of Technology, Cambridge, MA 02139, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Koch Institute for Integrative Cancer Research, Massachusetts Institute of Technology, Cambridge, MA 02139, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Mulugeta/Nolawit" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">2</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">5</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">6</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au10" aria-controls="axel_author_au10"><span property="givenName">Kevin S.</span> <span property="familyName">Kapner</span></a><div id="au10" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au10"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Kevin S.</span> <span property="familyName">Kapner</span> <a class="orcid-id" href="https://orcid.org/0000-0002-2306-2373" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0002-2306-2373</a></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Medical Oncology, Dana-Farber Cancer Institute, Boston, MA 02215, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Broad Institute of MIT and Harvard, Cambridge, MA 02142, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Kapner/Kevin+S" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">2</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au11" aria-controls="axel_author_au11"><span property="givenName">Manisha S.</span> <span property="familyName">Raghavan</span></a><div id="au11" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au11"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Manisha S.</span> <span property="familyName">Raghavan</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Medical Oncology, Dana-Farber Cancer Institute, Boston, MA 02215, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Broad Institute of MIT and Harvard, Cambridge, MA 02142, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Raghavan/Manisha+S" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">2</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au12" aria-controls="axel_author_au12"><span property="givenName">Ashir A.</span> <span property="familyName">Borah</span></a><div id="au12" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au12"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Ashir A.</span> <span property="familyName">Borah</span> <a class="orcid-id" href="https://orcid.org/0000-0003-1193-4987" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0003-1193-4987</a></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Broad Institute of MIT and Harvard, Cambridge, MA 02142, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Borah/Ashir+A" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">2</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au13" aria-controls="axel_author_au13"><span property="givenName">Nuo</span> <span property="familyName">Liu</span></a><div id="au13" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au13"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Nuo</span> <span property="familyName">Liu</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Broad Institute of MIT and Harvard, Cambridge, MA 02142, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Institute for Medical Engineering &amp; Science, Massachusetts Institute of Technology, Cambridge, MA 02139, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Koch Institute for Integrative Cancer Research, Massachusetts Institute of Technology, Cambridge, MA 02139, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Liu/Nuo" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">2</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">5</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">6</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au14" aria-controls="axel_author_au14"><span property="givenName">Sara A.</span> <span property="familyName">Väyrynen</span></a><div id="au14" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au14"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Sara A.</span> <span property="familyName">Väyrynen</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Medical Oncology, Dana-Farber Cancer Institute, Boston, MA 02215, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Harvard Medical School, Boston, MA 02115, USA</span></div></div><div class="core-author-link"><a href="/authored-by/V%C3%A4yrynen/Sara+A" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">3</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au15" aria-controls="axel_author_au15"><span property="givenName">Andressa Dias</span> <span property="familyName">Costa</span></a><div id="au15" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au15"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Andressa Dias</span> <span property="familyName">Costa</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Medical Oncology, Dana-Farber Cancer Institute, Boston, MA 02215, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Harvard Medical School, Boston, MA 02115, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Costa/Andressa+Dias" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">3</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au16" aria-controls="axel_author_au16"><span property="givenName">Raymond W.S.</span> <span property="familyName">Ng</span></a><div id="au16" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au16"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Raymond W.S.</span> <span property="familyName">Ng</span> <a class="orcid-id" href="https://orcid.org/0000-0003-2573-8207" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0003-2573-8207</a></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Medical Oncology, Dana-Farber Cancer Institute, Boston, MA 02215, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Broad Institute of MIT and Harvard, Cambridge, MA 02142, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Ng/Raymond+W+S" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">2</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au17" aria-controls="axel_author_au17"><span property="givenName">Junning</span> <span property="familyName">Wang</span></a><div id="au17" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au17"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Junning</span> <span property="familyName">Wang</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Medical Oncology, Dana-Farber Cancer Institute, Boston, MA 02215, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Wang/Junning" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au18" aria-controls="axel_author_au18"><span property="givenName">Emma K.</span> <span property="familyName">Hill</span></a><div id="au18" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au18"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Emma K.</span> <span property="familyName">Hill</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Medical Oncology, Dana-Farber Cancer Institute, Boston, MA 02215, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Hill/Emma+K" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au19" aria-controls="axel_author_au19"><span property="givenName">Dorisanne Y.</span> <span property="familyName">Ragon</span></a><div id="au19" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au19"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Dorisanne Y.</span> <span property="familyName">Ragon</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Medical Oncology, Dana-Farber Cancer Institute, Boston, MA 02215, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Ragon/Dorisanne+Y" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au20" aria-controls="axel_author_au20"><span property="givenName">Lauren K.</span> <span property="familyName">Brais</span></a><div id="au20" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au20"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Lauren K.</span> <span property="familyName">Brais</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Medical Oncology, Dana-Farber Cancer Institute, Boston, MA 02215, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Brais/Lauren+K" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au21" aria-controls="axel_author_au21"><span property="givenName">Alex M.</span> <span property="familyName">Jaeger</span></a><div id="au21" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au21"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Alex M.</span> <span property="familyName">Jaeger</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Koch Institute for Integrative Cancer Research, Massachusetts Institute of Technology, Cambridge, MA 02139, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Jaeger/Alex+M" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">6</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au22" aria-controls="axel_author_au22"><span property="givenName">Liam F.</span> <span property="familyName">Spurr</span></a><div id="au22" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au22"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Liam F.</span> <span property="familyName">Spurr</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Medical Oncology, Dana-Farber Cancer Institute, Boston, MA 02215, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Broad Institute of MIT and Harvard, Cambridge, MA 02142, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Spurr/Liam+F" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">2</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au23" aria-controls="axel_author_au23"><span property="givenName">Yvonne Y.</span> <span property="familyName">Li</span></a><div id="au23" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au23"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Yvonne Y.</span> <span property="familyName">Li</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Medical Oncology, Dana-Farber Cancer Institute, Boston, MA 02215, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Broad Institute of MIT and Harvard, Cambridge, MA 02142, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Li/Yvonne+Y" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">2</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au24" aria-controls="axel_author_au24"><span property="givenName">Andrew D.</span> <span property="familyName">Cherniack</span></a><div id="au24" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au24"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Andrew D.</span> <span property="familyName">Cherniack</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Medical Oncology, Dana-Farber Cancer Institute, Boston, MA 02215, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Broad Institute of MIT and Harvard, Cambridge, MA 02142, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Harvard Medical School, Boston, MA 02115, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Cherniack/Andrew+D" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">2</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">3</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au25" aria-controls="axel_author_au25"><span property="givenName">Matthew A.</span> <span property="familyName">Booker</span></a><div id="au25" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au25"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Matthew A.</span> <span property="familyName">Booker</span> <a class="orcid-id" href="https://orcid.org/0000-0002-6902-1032" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0002-6902-1032</a></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Informatics and Analytics, Dana-Farber Cancer Institute, Boston, MA 02215, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Booker/Matthew+A" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">10</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au26" aria-controls="axel_author_au26"><span property="givenName">Elizabeth F.</span> <span property="familyName">Cohen</span></a><div id="au26" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au26"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Elizabeth F.</span> <span property="familyName">Cohen</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Informatics and Analytics, Dana-Farber Cancer Institute, Boston, MA 02215, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Cohen/Elizabeth+F" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">10</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au27" aria-controls="axel_author_au27"><span property="givenName">Michael Y.</span> <span property="familyName">Tolstorukov</span></a><div id="au27" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au27"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Michael Y.</span> <span property="familyName">Tolstorukov</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Informatics and Analytics, Dana-Farber Cancer Institute, Boston, MA 02215, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Tolstorukov/Michael+Y" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">10</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au28" aria-controls="axel_author_au28"><span property="givenName">Isaac</span> <span property="familyName">Wakiro</span></a><div id="au28" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au28"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Isaac</span> <span property="familyName">Wakiro</span> <a class="orcid-id" href="https://orcid.org/0000-0002-5248-4823" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0002-5248-4823</a></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Medical Oncology, Dana-Farber Cancer Institute, Boston, MA 02215, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Wakiro/Isaac" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au29" aria-controls="axel_author_au29"><span property="givenName">Asaf</span> <span property="familyName">Rotem</span></a><div id="au29" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au29"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Asaf</span> <span property="familyName">Rotem</span> <a class="orcid-id" href="https://orcid.org/0000-0002-6859-7435" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0002-6859-7435</a></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Medical Oncology, Dana-Farber Cancer Institute, Boston, MA 02215, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Broad Institute of MIT and Harvard, Cambridge, MA 02142, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Center for Cancer Genomics, Dana-Farber Cancer Institute, Boston, MA 02215, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Rotem/Asaf" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">2</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">11</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au30" aria-controls="axel_author_au30"><span property="givenName">Bruce E.</span> <span property="familyName">Johnson</span></a><div id="au30" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au30"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Bruce E.</span> <span property="familyName">Johnson</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Medical Oncology, Dana-Farber Cancer Institute, Boston, MA 02215, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Harvard Medical School, Boston, MA 02115, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Medicine, Brigham and Women’s Hospital, Boston, MA 02115, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Center for Cancer Genomics, Dana-Farber Cancer Institute, Boston, MA 02215, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Johnson/Bruce+E" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">3</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">4</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">11</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au31" aria-controls="axel_author_au31"><span property="givenName">James M.</span> <span property="familyName">McFarland</span></a><div id="au31" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au31"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">James M.</span> <span property="familyName">McFarland</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Broad Institute of MIT and Harvard, Cambridge, MA 02142, USA</span></div></div><div class="core-author-link"><a href="/authored-by/McFarland/James+M" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">2</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au32" aria-controls="axel_author_au32"><span property="givenName">Ewa T.</span> <span property="familyName">Sicinska</span></a><div id="au32" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au32"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Ewa T.</span> <span property="familyName">Sicinska</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Harvard Medical School, Boston, MA 02115, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Oncologic Pathology, Dana-Farber Cancer Institute, Boston, MA 02215, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Sicinska/Ewa+T" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">3</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">12</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au33" aria-controls="axel_author_au33"><span property="givenName">Tyler E.</span> <span property="familyName">Jacks</span></a><div id="au33" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au33"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Tyler E.</span> <span property="familyName">Jacks</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Koch Institute for Integrative Cancer Research, Massachusetts Institute of Technology, Cambridge, MA 02139, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Jacks/Tyler+E" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">6</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au34" aria-controls="axel_author_au34"><span property="givenName">Ryan J.</span> <span property="familyName">Sullivan</span></a><div id="au34" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au34"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Ryan J.</span> <span property="familyName">Sullivan</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Harvard Medical School, Boston, MA 02115, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Massachusetts General Hospital Cancer Center, Boston, MA 02114, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Sullivan/Ryan+J" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">3</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">13</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au35" aria-controls="axel_author_au35"><span property="givenName">Geoffrey I.</span> <span property="familyName">Shapiro</span></a><div id="au35" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au35"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Geoffrey I.</span> <span property="familyName">Shapiro</span> <a class="orcid-id" href="https://orcid.org/0000-0002-3331-4095" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0002-3331-4095</a></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Medical Oncology, Dana-Farber Cancer Institute, Boston, MA 02215, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Harvard Medical School, Boston, MA 02115, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Medicine, Brigham and Women’s Hospital, Boston, MA 02115, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Shapiro/Geoffrey+I" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">3</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">4</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au36" aria-controls="axel_author_au36"><span property="givenName">Thomas E.</span> <span property="familyName">Clancy</span></a><div id="au36" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au36"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Thomas E.</span> <span property="familyName">Clancy</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Medical Oncology, Dana-Farber Cancer Institute, Boston, MA 02215, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Harvard Medical School, Boston, MA 02115, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Surgery, Brigham and Women’s Hospital, Boston, MA 02115, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Clancy/Thomas+E" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">3</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">14</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au37" aria-controls="axel_author_au37"><span property="givenName">Kimberly</span> <span property="familyName">Perez</span></a><div id="au37" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au37"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Kimberly</span> <span property="familyName">Perez</span> <a class="orcid-id" href="https://orcid.org/0000-0002-5745-2994" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0002-5745-2994</a></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Medical Oncology, Dana-Farber Cancer Institute, Boston, MA 02215, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Harvard Medical School, Boston, MA 02115, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Medicine, Brigham and Women’s Hospital, Boston, MA 02115, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Perez/Kimberly" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">3</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">4</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au38" aria-controls="axel_author_au38"><span property="givenName">Douglas A.</span> <span property="familyName">Rubinson</span></a><div id="au38" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au38"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Douglas A.</span> <span property="familyName">Rubinson</span> <a class="orcid-id" href="https://orcid.org/0000-0002-5337-5231" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0002-5337-5231</a></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Medical Oncology, Dana-Farber Cancer Institute, Boston, MA 02215, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Harvard Medical School, Boston, MA 02115, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Medicine, Brigham and Women’s Hospital, Boston, MA 02115, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Rubinson/Douglas+A" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">3</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">4</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au39" aria-controls="axel_author_au39"><span property="givenName">Kimmie</span> <span property="familyName">Ng</span></a><div id="au39" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au39"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Kimmie</span> <span property="familyName">Ng</span> <a class="orcid-id" href="https://orcid.org/0000-0003-0631-1494" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0003-0631-1494</a></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Medical Oncology, Dana-Farber Cancer Institute, Boston, MA 02215, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Harvard Medical School, Boston, MA 02115, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Medicine, Brigham and Women’s Hospital, Boston, MA 02115, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Ng/Kimmie" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">3</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">4</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au40" aria-controls="axel_author_au40"><span property="givenName">James M.</span> <span property="familyName">Cleary</span></a><div id="au40" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au40"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">James M.</span> <span property="familyName">Cleary</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Medical Oncology, Dana-Farber Cancer Institute, Boston, MA 02215, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Harvard Medical School, Boston, MA 02115, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Medicine, Brigham and Women’s Hospital, Boston, MA 02115, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Cleary/James+M" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">3</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">4</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au41" aria-controls="axel_author_au41"><span property="givenName">Lorin</span> <span property="familyName">Crawford</span></a><div id="au41" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au41"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Lorin</span> <span property="familyName">Crawford</span> <a class="orcid-id" href="https://orcid.org/0000-0003-0178-8242" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0003-0178-8242</a></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Center for Computational Molecular Biology, Brown University, Providence, RI 02912, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Biostatistics, Brown University, Providence, RI 02912, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Microsoft Research New England, Cambridge, MA 02142, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Crawford/Lorin" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">8</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">15</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">16</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au42" aria-controls="axel_author_au42"><span property="givenName">Scott R.</span> <span property="familyName">Manalis</span></a><div id="au42" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au42"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Scott R.</span> <span property="familyName">Manalis</span> <a class="orcid-id" href="https://orcid.org/0000-0001-5223-9433" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0001-5223-9433</a></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Koch Institute for Integrative Cancer Research, Massachusetts Institute of Technology, Cambridge, MA 02139, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Biological Engineering, Massachusetts Institute of Technology, Cambridge, MA 02142, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Manalis/Scott+R" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">6</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">17</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au43" aria-controls="axel_author_au43"><span property="givenName">Jonathan A.</span> <span property="familyName">Nowak</span></a><div id="au43" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au43"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Jonathan A.</span> <span property="familyName">Nowak</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Harvard Medical School, Boston, MA 02115, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Center for Cancer Genomics, Dana-Farber Cancer Institute, Boston, MA 02215, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Pathology, Brigham and Women’s Hospital, Boston, MA 02115, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Nowak/Jonathan+A" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">3</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">11</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">18</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au44" aria-controls="axel_author_au44"><span property="givenName">Brian M.</span> <span property="familyName">Wolpin</span></a><div id="au44" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au44"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Brian M.</span> <span property="familyName">Wolpin</span></div><div class="content"><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">21</div><div id="fn2"><div id="ntpara0015" role="paragraph">These authors contributed equally</div></div></div><div role="doc-footnote" data-has="label"><div class="label">22</div><div id="fn3"><div id="ntpara0020" role="paragraph">Senior author</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Medical Oncology, Dana-Farber Cancer Institute, Boston, MA 02215, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Harvard Medical School, Boston, MA 02115, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Medicine, Brigham and Women’s Hospital, Boston, MA 02115, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Wolpin/Brian+M" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">3</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">4</a>,<a role="doc-noteref">21</a>,<a role="doc-noteref">22</a></sup></span></span> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au45" aria-controls="axel_author_au45"><span property="givenName">William C.</span> <span property="familyName">Hahn</span></a><div id="au45" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au45"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">William C.</span> <span property="familyName">Hahn</span> <a class="orcid-id" href="https://orcid.org/0000-0003-2840-9791" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0003-2840-9791</a></div><div class="content"><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">21</div><div id="fn2"><div id="ntpara0015" role="paragraph">These authors contributed equally</div></div></div><div role="doc-footnote" data-has="label"><div class="label">22</div><div id="fn3"><div id="ntpara0020" role="paragraph">Senior author</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Medical Oncology, Dana-Farber Cancer Institute, Boston, MA 02215, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Broad Institute of MIT and Harvard, Cambridge, MA 02142, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Harvard Medical School, Boston, MA 02115, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Medicine, Brigham and Women’s Hospital, Boston, MA 02115, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Hahn/William+C" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">2</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">3</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">4</a>,<a role="doc-noteref">21</a>,<a role="doc-noteref">22</a></sup></span> ∙ <span property="author" typeof="Person" role="listitem" class="corresponding-author"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au46" aria-controls="axel_author_au46"><span property="givenName">Andrew J.</span> <span property="familyName">Aguirre</span></a><div id="au46" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au46"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Andrew J.</span> <span property="familyName">Aguirre</span></div><div class="content"><div class="core-author-emails"><div class="label">Correspondence</div><div>Corresponding author</div><div><a id="eadd0015" href="mailto:andrew_aguirre@dfci.harvard.edu" property="email" title="Send email to Andrew J. Aguirre">andrew_aguirre@dfci.harvard.edu</a></div></div><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">21</div><div id="fn2"><div id="ntpara0015" role="paragraph">These authors contributed equally</div></div></div><div role="doc-footnote" data-has="label"><div class="label">22</div><div id="fn3"><div id="ntpara0020" role="paragraph">Senior author</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Medical Oncology, Dana-Farber Cancer Institute, Boston, MA 02215, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Broad Institute of MIT and Harvard, Cambridge, MA 02142, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Harvard Medical School, Boston, MA 02115, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Medicine, Brigham and Women’s Hospital, Boston, MA 02115, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Aguirre/Andrew+J" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">2</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">3</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">4</a>,<a role="doc-noteref">21</a>,<a role="doc-noteref">22</a></sup> <a id="eadd0015" href="mailto:andrew_aguirre@dfci.harvard.edu" property="email" title="Send email to Andrew J. Aguirre">andrew_aguirre@dfci.harvard.edu</a></span> ∙ <span property="author" typeof="Person" role="listitem" class="corresponding-author"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au47" aria-controls="axel_author_au47"><span property="givenName">Alex K.</span> <span property="familyName">Shalek</span></a><div id="au47" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au47"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Alex K.</span> <span property="familyName">Shalek</span> <a class="orcid-id" href="https://orcid.org/0000-0001-5670-8778" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0001-5670-8778</a></div><div class="content"><div class="core-author-emails"><div class="label">Correspondence</div><div>Corresponding author</div><div><a id="eadd0020" href="mailto:shalek@mit.edu" property="email" title="Send email to Alex K. Shalek">shalek@mit.edu</a></div></div><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">21</div><div id="fn2"><div id="ntpara0015" role="paragraph">These authors contributed equally</div></div></div><div role="doc-footnote" data-has="label"><div class="label">22</div><div id="fn3"><div id="ntpara0020" role="paragraph">Senior author</div></div></div><div role="doc-footnote" data-has="label"><div class="label">23</div><div id="fn4"><div id="ntpara0025" role="paragraph">Lead contact</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Broad Institute of MIT and Harvard, Cambridge, MA 02142, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Harvard Medical School, Boston, MA 02115, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Institute for Medical Engineering &amp; Science, Massachusetts Institute of Technology, Cambridge, MA 02139, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Koch Institute for Integrative Cancer Research, Massachusetts Institute of Technology, Cambridge, MA 02139, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Chemistry, Massachusetts Institute of Technology, Cambridge, MA 02139, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Ragon Institute of MGH, MIT, and Harvard, Cambridge, MA 02139, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Shalek/Alex+K" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">2</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">3</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">5</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">6</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">7</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">19</a>,<a role="doc-noteref">21</a>,<a role="doc-noteref">22</a>,<a role="doc-noteref">23</a></sup> <a id="eadd0020" href="mailto:shalek@mit.edu" property="email" title="Send email to Alex K. Shalek">shalek@mit.edu</a></span><span data-displayed-on="all"> … <span><span data-action="reveal" tabindex="0" role="listitem">Show more</span></span> </span><span data-displayed-on="none" data-on-display="all" aria-hidden="true"> <span data-action="hide" tabindex="0" role="listitem">Show less</span></span></span></span></div><div class="core-self-citation"><div class="expandable-triggers"><button aria-controls="core-affiliations-notes" aria-expanded="false" tabindex="0" role="button"><span>Affiliations &amp; Notes</span><i class="icon-arrow_d_n" aria-hidden="true"></i></button><button aria-controls="core-content-info" aria-expanded="false" tabindex="0" role="button"><span>Article Info</span><i class="icon-arrow_d_n" aria-hidden="true"></i></button></div><div id="core-linked-content">
    

        

</div><div class="expandable-content"><div id="core-affiliations-notes" role="region" data-expandable="item" style="display: none;"><div class="affiliations"><div property="affiliation" typeof="Organization"><span class="label">1</span><span id="aff1" property="name">Department of Medical Oncology, Dana-Farber Cancer Institute, Boston, MA 02215, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">2</span><span id="aff2" property="name">Broad Institute of MIT and Harvard, Cambridge, MA 02142, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">3</span><span id="aff3" property="name">Harvard Medical School, Boston, MA 02115, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">4</span><span id="aff4" property="name">Department of Medicine, Brigham and Women’s Hospital, Boston, MA 02115, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">5</span><span id="aff5" property="name">Institute for Medical Engineering &amp; Science, Massachusetts Institute of Technology, Cambridge, MA 02139, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">6</span><span id="aff6" property="name">Koch Institute for Integrative Cancer Research, Massachusetts Institute of Technology, Cambridge, MA 02139, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">7</span><span id="aff7" property="name">Department of Chemistry, Massachusetts Institute of Technology, Cambridge, MA 02139, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">8</span><span id="aff8" property="name">Center for Computational Molecular Biology, Brown University, Providence, RI 02912, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">9</span><span id="aff9" property="name">Division of Applied Mathematics, Brown University, Providence, RI 02912, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">10</span><span id="aff10" property="name">Department of Informatics and Analytics, Dana-Farber Cancer Institute, Boston, MA 02215, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">11</span><span id="aff11" property="name">Center for Cancer Genomics, Dana-Farber Cancer Institute, Boston, MA 02215, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">12</span><span id="aff12" property="name">Department of Oncologic Pathology, Dana-Farber Cancer Institute, Boston, MA 02215, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">13</span><span id="aff13" property="name">Massachusetts General Hospital Cancer Center, Boston, MA 02114, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">14</span><span id="aff14" property="name">Department of Surgery, Brigham and Women’s Hospital, Boston, MA 02115, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">15</span><span id="aff15" property="name">Department of Biostatistics, Brown University, Providence, RI 02912, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">16</span><span id="aff16" property="name">Microsoft Research New England, Cambridge, MA 02142, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">17</span><span id="aff17" property="name">Department of Biological Engineering, Massachusetts Institute of Technology, Cambridge, MA 02142, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">18</span><span id="aff18" property="name">Department of Pathology, Brigham and Women’s Hospital, Boston, MA 02115, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">19</span><span id="aff19" property="name">Ragon Institute of MGH, MIT, and Harvard, Cambridge, MA 02139, USA</span></div></div><div class="core-authors-notes"><div role="doc-footnote" data-has="label"><div class="label">20</div><div id="fn1"><div id="ntpara0010" role="paragraph">These authors contributed equally</div></div></div><div role="doc-footnote" data-has="label"><div class="label">21</div><div id="fn2"><div id="ntpara0015" role="paragraph">These authors contributed equally</div></div></div><div role="doc-footnote" data-has="label"><div class="label">22</div><div id="fn3"><div id="ntpara0020" role="paragraph">Senior author</div></div></div><div role="doc-footnote" data-has="label"><div class="label">23</div><div id="fn4"><div id="ntpara0025" role="paragraph">Lead contact</div></div></div></div></div><div id="core-content-info" role="region" data-expandable="item" style="display: none;"><div class="history"><div>Publication History:</div><div><span>Received March 12, 2021</span>; <span>Revised September 24, 2021</span>; <span>Accepted November 11, 2021</span></div></div><div class="core-links"><span class="doi">DOI: <a href="https://doi.org/10.1016/j.cell.2021.11.017" property="sameAs" target="_blank">10.1016/j.cell.2021.11.017 <span class="sr-only">External Link</span></a></span><span class="science-direct">Also available on <a href="https://www.sciencedirect.com/science/article/pii/S0092867421013325" target="_blank">ScienceDirect <span class="sr-only">External Link</span></a></span></div><div role="paragraph">Copyright: © 2021 The Author(s). Published by Elsevier Inc.</div><div class="core-links"><span>User License: <a href="http://creativecommons.org/licenses/by-nc-nd/4.0/" target="_blank">Creative Commons Attribution – NonCommercial – NoDerivs (CC BY-NC-ND 4.0)</a> | <a href="https://www.elsevier.com/open-access" target="_blank">Elsevier's open access license policy</a></span></div><div class="core-crossmark-badge"><a data-target="crossmark" href="#" title="Check for updates on crossmark" data-doi="10.1016/j.cell.2021.11.017" data-id="article-info-crossmark" class="crossmark__link" style="cursor: pointer;"><svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 290 58" height="22px" role="presentation"><path fill="url(#horizontal_crossmark)" stroke="#948F8F" stroke-miterlimit="10" stroke-width=".75" d="M287 1H3a2 2 0 0 0-2 2v52a2 2 0 0 0 2 2h284a2 2 0 0 0 2-2V3a2 2 0 0 0-2-2Z"></path><path fill="#535353" d="M80.3 23.58a6.395 6.395 0 0 0-.55-1.43 3.92 3.92 0 0 0-2.19-1.86 5.099 5.099 0 0 0-1.71-.29 5 5 0 0 0-2.46.58 4.831 4.831 0 0 0-1.69 1.54 6.921 6.921 0 0 0-1 2.19 10.16 10.16 0 0 0 0 5 6.921 6.921 0 0 0 1 2.19 4.83 4.83 0 0 0 1.69 1.5 5 5 0 0 0 2.46.58 4.44 4.44 0 0 0 1.85-.36 4.12 4.12 0 0 0 1.38-1c.39-.436.695-.942.9-1.49A7.094 7.094 0 0 0 80.4 29h3a8.46 8.46 0 0 1-.69 2.92 7.179 7.179 0 0 1-1.59 2.29 7 7 0 0 1-2.35 1.49 8.14 8.14 0 0 1-3 .52 8.43 8.43 0 0 1-3.59-.74 7.88 7.88 0 0 1-2.69-2 9.001 9.001 0 0 1-1.69-3 11.41 11.41 0 0 1 0-7.23 9.001 9.001 0 0 1 1.69-3 8.001 8.001 0 0 1 2.69-2 8.3 8.3 0 0 1 3.59-.75 9.44 9.44 0 0 1 2.83.41c.866.242 1.68.642 2.4 1.18A6.229 6.229 0 0 1 82.65 21a6.66 6.66 0 0 1 .78 2.58H80.3ZM85.9 17.9h2.85v6.63h.05a4.15 4.15 0 0 1 1.56-1.46 4.61 4.61 0 0 1 2.31-.59 4.83 4.83 0 0 1 3.33 1.1 4.21 4.21 0 0 1 1.22 3.3v8.88H94.4v-8.13a3.51 3.51 0 0 0-.65-2.21 2.36 2.36 0 0 0-1.87-.69 3.1 3.1 0 0 0-1.3.26 2.821 2.821 0 0 0-1 .73A3.41 3.41 0 0 0 89 26.8a3.88 3.88 0 0 0-.23 1.33v7.63H85.9V17.9ZM102.47 30c-.002.491.069.98.21 1.45.131.448.348.866.64 1.23a3.08 3.08 0 0 0 1.08.84 3.543 3.543 0 0 0 1.55.31 3.431 3.431 0 0 0 2-.54c.541-.402.94-.966 1.14-1.61h2.7A5.349 5.349 0 0 1 109.7 35a5.715 5.715 0 0 1-1.74.85 7 7 0 0 1-2 .29 6.848 6.848 0 0 1-2.7-.5 5.577 5.577 0 0 1-2-1.4A5.953 5.953 0 0 1 100 32a8.748 8.748 0 0 1-.41-2.75 7.768 7.768 0 0 1 .44-2.61 6.67 6.67 0 0 1 1.25-2.17 5.863 5.863 0 0 1 4.56-2 5.79 5.79 0 0 1 2.74.64c.783.41 1.466.986 2 1.69a6.824 6.824 0 0 1 1.16 2.41c.264.905.352 1.852.26 2.79h-9.53Zm6.65-1.87a4.605 4.605 0 0 0-.29-1.3c-.148-.4-.368-.77-.65-1.09a3.22 3.22 0 0 0-1-.75 3 3 0 0 0-1.31-.29 3.431 3.431 0 0 0-1.36.26 3.108 3.108 0 0 0-1 .73 3.594 3.594 0 0 0-.7 1.09 3.798 3.798 0 0 0-.29 1.35h6.65-.05ZM122.88 27.15a2.666 2.666 0 0 0-1-1.81 3.14 3.14 0 0 0-2-.61c-.4.005-.798.07-1.18.19-.435.136-.83.376-1.15.7a3.92 3.92 0 0 0-.87 1.43 6.783 6.783 0 0 0-.35 2.39c.002.539.065 1.076.19 1.6.113.509.316.993.6 1.43.273.41.635.752 1.06 1a3 3 0 0 0 1.58.39 2.872 2.872 0 0 0 2.06-.77 3.596 3.596 0 0 0 1-2.18h2.85a6.26 6.26 0 0 1-1.91 3.86 5.83 5.83 0 0 1-4 1.34 6.61 6.61 0 0 1-2.69-.51 5.695 5.695 0 0 1-2-1.4 6.001 6.001 0 0 1-1.21-2.11 8.279 8.279 0 0 1-.41-2.65 8.987 8.987 0 0 1 .4-2.72 6.237 6.237 0 0 1 1.2-2.21 5.614 5.614 0 0 1 2-1.47 6.7 6.7 0 0 1 2.79-.54 7.801 7.801 0 0 1 2.14.29 5.634 5.634 0 0 1 1.8.86c.529.386.969.88 1.29 1.45a5 5 0 0 1 .6 2.07h-2.85l.06-.02ZM128.13 17.9H131V28l5.15-5.22h3.5l-5 4.75 5.42 8.17h-3.47l-4-6.27-1.6 1.7v4.63h-2.85V17.9h-.02ZM147.15 22.83h2.13v-1.08a5.505 5.505 0 0 1 .3-2c.161-.456.436-.862.8-1.18.33-.273.721-.462 1.14-.55.44-.094.89-.141 1.34-.14a8.158 8.158 0 0 1 2 .18v2.24a4.113 4.113 0 0 0-.59-.11 6.343 6.343 0 0 0-.74 0 1.74 1.74 0 0 0-1 .27 1.204 1.204 0 0 0-.41 1.08v1.33h2.42V25h-2.42v10.8h-2.85V25h-2.12v-2.17ZM162.23 36.1a6.999 6.999 0 0 1-2.76-.51 5.877 5.877 0 0 1-2-1.41 5.998 5.998 0 0 1-1.33-2.18 8.775 8.775 0 0 1 0-5.48 6.008 6.008 0 0 1 1.27-2.15 5.894 5.894 0 0 1 2-1.41A7.71 7.71 0 0 1 165 23a5.894 5.894 0 0 1 2 1.41 6.003 6.003 0 0 1 1.28 2.15c.585 1.78.585 3.7 0 5.48a5.99 5.99 0 0 1-3.28 3.54 7.002 7.002 0 0 1-2.77.52Zm0-2.25a3.268 3.268 0 0 0 1.65-.4c.447-.262.83-.621 1.12-1.05.304-.444.528-.938.66-1.46a6.581 6.581 0 0 0 0-3.31 4.416 4.416 0 0 0-.66-1.46 3.526 3.526 0 0 0-1.15-1 3.6 3.6 0 0 0-3.3 0 3.526 3.526 0 0 0-1.15 1 4.414 4.414 0 0 0-.66 1.46 6.558 6.558 0 0 0 0 3.31c.132.522.356 1.016.66 1.46a3.47 3.47 0 0 0 1.15 1.05 3.26 3.26 0 0 0 1.68.4ZM171.13 22.83h2.67v2.5c.097-.362.264-.701.49-1 .251-.341.547-.647.88-.91a4.404 4.404 0 0 1 1.14-.66 3.41 3.41 0 0 1 1.28-.25c.23-.013.46-.013.69 0h.39v2.79l-.61-.09a5.173 5.173 0 0 0-.61 0 3.278 3.278 0 0 0-1.36.29 3.131 3.131 0 0 0-1.11.85 4.171 4.171 0 0 0-.75 1.39 6 6 0 0 0-.28 1.9v6.15h-2.85V22.83h.03ZM198.22 35.75h-2.8V34a3.907 3.907 0 0 1-1.56 1.56 4.25 4.25 0 0 1-2.11.59 4.742 4.742 0 0 1-3.75-1.31 5.537 5.537 0 0 1-1.12-3.84v-8.2h2.85v7.92a3.503 3.503 0 0 0 .65 2.4 2.38 2.38 0 0 0 1.83.7 3.599 3.599 0 0 0 1.5-.27 2.66 2.66 0 0 0 1-.74c.259-.326.444-.705.54-1.11a5.79 5.79 0 0 0 .16-1.4v-7.5h2.85v12.95h-.04ZM201.3 22.83h2.7v1.75a3.523 3.523 0 0 1 1.72-1.58 5.568 5.568 0 0 1 2.33-.49 6.17 6.17 0 0 1 2.66.54 5.246 5.246 0 0 1 1.89 1.47c.515.65.899 1.394 1.13 2.19.257.87.385 1.773.38 2.68.001.851-.114 1.7-.34 2.52a6.568 6.568 0 0 1-1 2.16 5.002 5.002 0 0 1-4.21 2.06c-.419 0-.837-.036-1.25-.11a5.39 5.39 0 0 1-1.2-.36A4.737 4.737 0 0 1 205 35a3.567 3.567 0 0 1-.81-.91v6.45h-2.89V22.83Zm10 6.47c0-.574-.078-1.146-.23-1.7a4.606 4.606 0 0 0-.67-1.46 3.475 3.475 0 0 0-1.12-1 3.06 3.06 0 0 0-1.55-.39A3.141 3.141 0 0 0 205 26a5.518 5.518 0 0 0-1 3.3c-.004.611.077 1.22.24 1.81.138.523.38 1.012.71 1.44a3.55 3.55 0 0 0 1.14 1 3.242 3.242 0 0 0 1.54.35 3.18 3.18 0 0 0 1.65-.4c.438-.25.817-.59 1.11-1 .299-.44.513-.931.63-1.45a7.38 7.38 0 0 0 .23-1.75h.05ZM228.67 35.75H226V34a3.38 3.38 0 0 1-1.67 1.61 5.668 5.668 0 0 1-2.33.49 6.17 6.17 0 0 1-2.66-.54 5.307 5.307 0 0 1-1.89-1.46 6.254 6.254 0 0 1-1.12-2.19 9.48 9.48 0 0 1-.37-2.71 8.67 8.67 0 0 1 .47-3 6.058 6.058 0 0 1 1.26-2.1 4.89 4.89 0 0 1 1.8-1.21 5.738 5.738 0 0 1 2.06-.39c.412 0 .824.037 1.23.11.413.073.815.194 1.2.36.38.166.736.381 1.06.64.32.255.594.562.81.91V17.9h2.85v17.85h-.03Zm-10-6.33c-.001.55.069 1.098.21 1.63.13.511.35.996.65 1.43.288.414.668.757 1.11 1a3.241 3.241 0 0 0 1.6.38 3.199 3.199 0 0 0 1.64-.4 3.398 3.398 0 0 0 1.13-1.05c.297-.447.517-.94.65-1.46.14-.542.211-1.1.21-1.66a5.227 5.227 0 0 0-1-3.35 3.175 3.175 0 0 0-2.61-1.2 3.23 3.23 0 0 0-1.69.41 3.478 3.478 0 0 0-1.12 1.07 4.494 4.494 0 0 0-.62 1.5 7.871 7.871 0 0 0-.11 1.7h-.05ZM242.6 32.88c-.018.258.03.516.14.75a.577.577 0 0 0 .54.23h.3c.135-.002.269-.019.4-.05v2l-.39.11-.49.08-.5.08h-.42a2.75 2.75 0 0 1-1.45-.35 1.731 1.731 0 0 1-.75-1.23 4.998 4.998 0 0 1-2.09 1.2 8.224 8.224 0 0 1-2.39.38 5.861 5.861 0 0 1-1.67-.24 4.394 4.394 0 0 1-1.41-.7 3.35 3.35 0 0 1-1-1.18 3.624 3.624 0 0 1-.36-1.66 3.84 3.84 0 0 1 .44-2 3.246 3.246 0 0 1 1.15-1.17 5.195 5.195 0 0 1 1.6-.61c.593-.127 1.19-.223 1.79-.29a14.77 14.77 0 0 1 1.47-.21 6.896 6.896 0 0 0 1.24-.21c.318-.077.611-.236.85-.46a1.286 1.286 0 0 0 .31-.94 1.458 1.458 0 0 0-.26-.9A1.759 1.759 0 0 0 239 25a2.807 2.807 0 0 0-.86-.25 6.941 6.941 0 0 0-.9-.06 3.58 3.58 0 0 0-2 .5 2 2 0 0 0-.87 1.55h-2.85c.021-.73.227-1.442.6-2.07.338-.54.797-.996 1.34-1.33a5.475 5.475 0 0 1 1.84-.7c.692-.134 1.395-.2 2.1-.2.632 0 1.262.067 1.88.2a5.337 5.337 0 0 1 1.66.65 3.6 3.6 0 0 1 1.19 1.16c.314.525.47 1.129.45 1.74v6.65l.02.04Zm-2.85-3.6a3.45 3.45 0 0 1-1.6.51c-.667.06-1.3.147-1.9.26a6.137 6.137 0 0 0-.87.21 2.64 2.64 0 0 0-.75.38c-.22.164-.395.38-.51.63a2.129 2.129 0 0 0-.19.94c-.011.29.085.575.27.8.182.215.407.388.66.51.269.128.555.216.85.26.277.05.558.076.84.08a4.63 4.63 0 0 0 1-.12 3.418 3.418 0 0 0 1-.43 2.73 2.73 0 0 0 .81-.76 1.93 1.93 0 0 0 .32-1.14v-2.14l.07.01ZM244.37 22.83h2.15V19h2.85v3.88h2.58V25h-2.58v6.9a7.064 7.064 0 0 0 0 .77c.016.194.074.381.17.55a.84.84 0 0 0 .41.34 2 2 0 0 0 .75.11h.6c.203-.01.404-.04.6-.09v2.2l-.93.1a8.403 8.403 0 0 1-.92 0 6.33 6.33 0 0 1-1.81-.21 2.421 2.421 0 0 1-1.08-.62 2.15 2.15 0 0 1-.52-1 7.791 7.791 0 0 1-.16-1.42V25h-2.15v-2.17h.04ZM256.22 30c-.002.491.069.98.21 1.45.131.448.348.866.64 1.23a3.08 3.08 0 0 0 1.08.84 3.543 3.543 0 0 0 1.55.31 3.431 3.431 0 0 0 2-.54c.541-.402.94-.966 1.14-1.61h2.7a5.352 5.352 0 0 1-2.1 3.26 5.715 5.715 0 0 1-1.74.85 7 7 0 0 1-2 .29 6.848 6.848 0 0 1-2.7-.5 5.577 5.577 0 0 1-2-1.4 5.947 5.947 0 0 1-1.21-2.18 8.754 8.754 0 0 1-.41-2.75 7.758 7.758 0 0 1 .44-2.61 6.67 6.67 0 0 1 1.25-2.17 5.863 5.863 0 0 1 4.56-2 5.79 5.79 0 0 1 2.74.64c.783.41 1.466.986 2 1.69a6.824 6.824 0 0 1 1.16 2.41c.243.916.308 1.87.19 2.81h-9.5V30Zm6.65-1.87a4.605 4.605 0 0 0-.29-1.3c-.148-.4-.368-.77-.65-1.09a3.22 3.22 0 0 0-1-.75 3 3 0 0 0-1.31-.29 3.431 3.431 0 0 0-1.36.26 3.108 3.108 0 0 0-1 .73 3.575 3.575 0 0 0-.7 1.09 3.798 3.798 0 0 0-.29 1.35h6.65-.05ZM270 31.6a2.154 2.154 0 0 0 .95 1.75c.607.35 1.3.524 2 .5.304-.002.608-.022.91-.06a3.943 3.943 0 0 0 1-.24c.286-.105.54-.28.74-.51a1.23 1.23 0 0 0 .26-.89 1.261 1.261 0 0 0-.4-.9 2.779 2.779 0 0 0-1-.56 9.315 9.315 0 0 0-1.34-.36l-1.5-.33a14.219 14.219 0 0 1-1.54-.42 4.937 4.937 0 0 1-1.33-.68 3.12 3.12 0 0 1-.94-1.09 3.472 3.472 0 0 1-.35-1.64 2.94 2.94 0 0 1 .51-1.76 3.913 3.913 0 0 1 1.3-1.15 5.66 5.66 0 0 1 1.75-.61 10.316 10.316 0 0 1 1.84-.17 8.418 8.418 0 0 1 1.91.21 5.19 5.19 0 0 1 1.65.69c.496.318.916.741 1.23 1.24.341.559.55 1.188.61 1.84h-3a1.783 1.783 0 0 0-.94-1.37 3.892 3.892 0 0 0-1.69-.35 6.093 6.093 0 0 0-.71 0 3.327 3.327 0 0 0-.78.19 1.642 1.642 0 0 0-.61.4 1 1 0 0 0-.25.69 1.093 1.093 0 0 0 .36.85c.294.245.634.428 1 .54.437.153.885.274 1.34.36l1.55.33a20.9 20.9 0 0 1 1.53.42c.479.157.931.386 1.34.68.41.277.751.643 1 1.07.255.494.379 1.045.36 1.6a3.48 3.48 0 0 1-.52 2 4.15 4.15 0 0 1-1.36 1.3 6.19 6.19 0 0 1-1.86.73 9.435 9.435 0 0 1-2 .23 8.823 8.823 0 0 1-2.26-.27 5.265 5.265 0 0 1-1.8-.84 4 4 0 0 1-1.2-1.4 4.445 4.445 0 0 1-.46-2h2.7v-.02Z"></path><path fill="#C72914" d="m21.94 38.74 18.12-12.08v-14.5H21.94v26.58Z"></path><path fill="#EF3340" d="M40.06 38.74 21.94 26.66v-14.5h18.12v26.58Z"></path><path fill="#3EB1C8" d="M31 9a20 20 0 1 0 0 40 20 20 0 0 0 0-40Zm0 35.1a15.1 15.1 0 1 1 0-30.2 15.1 15.1 0 0 1 0 30.2Z"></path><path fill="#FFC72C" d="M42.09 39.23A15.09 15.09 0 0 1 17 23.38l-4.2-2.64a20 20 0 0 0 33.49 21.14l-4.2-2.65Z"></path><defs><linearGradient id="horizontal_crossmark" x1="145" x2="145" y1="7.33" y2="56.82" gradientUnits="userSpaceOnUse"><stop stop-color="#fff"></stop><stop offset="1" stop-color="#c4c4c4"></stop></linearGradient></defs></svg></a></div><div class="core-content-info"></div></div></div></div></div></header><div class="core-header-aside">
    

        


    

            

            

            <div class="cover-image cover-image--axel"><div class="cover-image__thumb"><a href="https://www.cell.com/cell/issue?pii=S0092-8674(20)X0026-2"><img src="/cms/asset/0540b9ed-f4ad-4863-9830-19ca05143514/cov200h.gif" alt="Cover Image - Cell, Volume 184, Issue 25"></a></div></div>

            

        
</div></div><div data-core-wrapper="content"><div id="dummy-sticky-observer"></div><div data-core-nav="header" data-extent="frontmatter"><div class="core-nav-wrapper core-container" aria-label="Article navigation"><div class="navigation-menu-pdf">
    

        


    

            

            

            <div data-widget-id="ecff07e8-0bc4-4533-b1f1-49365e2ead20" data-widget-name="UX3ArticleToolsWidget" class="article-tools"><a href="/action/showPdf?pii=S0092-8674%2821%2901332-5" title="Standard PDF (opens in a new tab)" target="_blank" aria-hidden="false" role="menuitem" class="article-tools__item__pdf"><svg focusable="false" viewBox="0 0 32 32" width="20" height="20" title="PDF" class="pdf-icon"><path id="pdfwhite" d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path id="pdfda0000" d="M.167 2.592H22.39V9.72H.166z" fill="#da0000"></path><path id="pdffff9f9" fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path id="pdff91d0a" d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="basic">Download PDF</span><span class="alternative">Download PDF</span></a></div>

            

        
</div><div class="core-sections-menu"><button data-id="article-toolbar-showhide" aria-controls="article_sections_menu" aria-expanded="false" aria-haspopup="true" aria-label="Toggle section navigation menu" style="margin-top: 0px;"><i class="icon-list" aria-hidden="true"></i><span>Outline</span></button><nav id="article_sections_menu" aria-label="Contents" data-core-nav="article" style="margin-top: 0px;"><div class="heading">Outline<button aria-haspopup="true" aria-controls="article_sections_menu" title="Close outline menu"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div><ul><li><a href="#author-highlights-abstract">Highlights</a></li><li><a href="#author-abstract">Summary</a></li><li><a href="#graphical-abstract">Graphical abstract</a></li><li><a href="#keywords">Keywords</a></li><li><a href="#sec-1">Introduction</a></li><li><a href="#sec-2">Results</a></li><li><a href="#sec-3">Discussion</a></li><li><a href="#sec-4">STAR★Methods</a></li><li><a href="#acknowledgments">Acknowledgments</a></li><li><a href="#supplementary-material">Supplemental information (7)</a></li><li><a href="#data-availability">Data and code availability</a></li><li><a href="#references">References</a></li><li role="none" class="mobile-only"><a role="menuitem" href="#core-collateral-metrics">Article metrics</a></li><li role="none" class="mobile-only"><a role="menuitem" href="#core-collateral-relatedArticles">Related Articles</a></li></ul></nav></div><div class="navigation-menu-share">
    

        


    

            

            

            <!-- Go to https://www.addtoany.com/buttons/customize/ to customize your tools --><script type="text/javascript" defer="defer" src="https://static.addtoany.com/menu/page.js"></script><script type="text/javascript">var a2a_config = a2a_config || {};
a2a_config.thanks = {
    postShare: false,
    ad: false,
};</script><script>var a2a_config = a2a_config || {};
let res = window.location.href.replace(/\(/g, "%28").replace(/\)/g, "%29");
a2a_config.templates = a2a_config.templates || {};

a2a_config.templates.email = {
    body: res
};
</script><div data-widget-id="5f978446-c80a-450c-a185-94d633df2db2" data-widget-name="UX3share" aria-hidden="false" role="group" class="article-tools__dropblock article-tools__share"><button aria-label="Toggle share menu" aria-expanded="false" aria-haspopup="true" role="menuitem" aria-controls="navigation_menu_share_block" aria-labelledby="share-label" class="article-tools__item__share"><i aria-hidden="true" class="icon-article-share"></i><span id="share-label">Share</span></button><div id="navigation_menu_share_block" role="menu" class="share__block axel-share article-tools__dropblock__holder"><h4 class="axel-share__title heading">Share<button title="Close share menu" aria-expanded="false" aria-haspopup="true" aria-controls="navigation_menu_share_block"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></h4><div class="axel-share__inner"><span class="sr-only">Share on</span><ul class="a2a__share-list rlist w-slide--list a2a a2a_kit a2a_kit_size_32 a2a_default_style" data-a2a-url="https://www.cell.com/cell/fulltext/S0092-8674%2821%2901332-5" style="line-height: 32px;"><li class="a2a_listitem_custom"><a role="link" title="Email" class="a2a_button_email" target="_blank" rel="nofollow noopener" href="/#email"><i aria-hidden="true" class="at-icon-wrapper icon-gizmo-email-plus"></i><span>Email</span></a></li><li class="a2a_listitem_custom"><a role="link" title="formerly known as Twitter" class="a2a_button_twitter" target="_blank" rel="nofollow noopener" href="/#twitter"><i aria-hidden="true" class="at-icon-wrapper icon-x-default"></i><span>X</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Facebook" class="a2a_button_facebook" target="_blank" rel="nofollow noopener" href="/#facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook-square-2"></i><span>Facebook</span></a></li><li class="a2a_listitem_custom"><a role="link" title="LinkedIn" class="a2a_button_linkedin" target="_blank" rel="nofollow noopener" href="/#linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin-square"></i><span>LinkedIn</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Sina Weibo" class="a2a_button_sina_weibo" target="_blank" rel="nofollow noopener" href="/#sina_weibo"><i aria-hidden="true" class="at-icon-wrapper icon-weibo"></i><span>Sina Weibo</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Mendeley" class="a2a_button_mendeley" target="_blank" rel="nofollow noopener" href="/#mendeley"><i aria-hidden="true" class="at-icon-wrapper icon-mendeley-2"></i><span>Mendeley</span></a></li><li class="a2a_listitem_custom"><a role="link" title="bluesky" class="a2a_button_bluesky" target="_blank" rel="nofollow noopener" href="/#bluesky"><i aria-hidden="true" class="at-icon-wrapper icon-bluesky-logo"></i><span>bluesky</span></a></li><div style="clear: both;"></div></ul><div class="extra-links-drop-zone">
    

        


    

            

            

            <div data-widget-id="f1a208d3-ac5e-4db2-9c3a-40ef97ddfa10" data-widget-name="UX3HTMLWidget"><a class="axel-share__extra" target="_blank" href="/personalize/addFavoritePublication?doi=10.1016/j.cell.2021.11.017">
	<i class="icon-reading-list" aria-hidden="true"></i>
	<span>Add to my reading list</span>
</a></div>

            

        
</div></div></div></div>

            

        
</div><div><button data-id="article-nav-menubar-more" aria-controls="article_more_menu" aria-expanded="false" aria-haspopup="true" aria-label="Toggle more menu"><i aria-hidden="true" class="icon-more-mobile"></i><span>More</span></button><div class="info-panel"><div class="info-panel__left-content"><div class="info-panel__article_tools info-panel__item"><div id="article_more_menu" aria-label="More menu" data-core-nav="more"><div class="heading">More<button title="Close more menu" aria-controls="article_more_menu" aria-haspopup="true"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div>
    

        


    

            

            

            <ul data-widget-id="183a17a2-87b5-4e6f-bb85-6b5cda068c89" data-widget-name="UX3ArticleToolsWidget" class="article-tools"><li class="article-tools__item article-tools__pdf"><a href="/action/showPdf?pii=S0092-8674%2821%2901332-5" title="Standard PDF (opens in a new tab)" target="_blank" aria-hidden="false" class="article-tools__ctrl article-tools__item__pdf"><svg focusable="false" viewBox="0 0 32 32" width="20" height="20" title="PDF" class="pdf-icon"><path id="pdfwhite" d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path id="pdfda0000" d="M.167 2.592H22.39V9.72H.166z" fill="#da0000"></path><path id="pdffff9f9" fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path id="pdff91d0a" d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="basic">Download PDF</span><span class="alternative">Download PDF</span></a></li><li><a href="https://www.cell.com/action/showCitFormats?doi=10.1016%2Fj.cell.2021.11.017&amp;pii=S0092-8674%2821%2901332-5" title="Cite" target="_blank" aria-label="Open Cite in new tab" class="article-tools__item__displayExportCitation"><i aria-hidden="true" class="icon-export-Citation"></i><span>Cite</span></a></li><li id="article_tools_share">
    

        


    

            

            

            <!-- Go to https://www.addtoany.com/buttons/customize/ to customize your tools --><script type="text/javascript" defer="defer" src="https://static.addtoany.com/menu/page.js"></script><script type="text/javascript">var a2a_config = a2a_config || {};
a2a_config.thanks = {
    postShare: false,
    ad: false,
};</script><script>var a2a_config = a2a_config || {};
let res = window.location.href.replace(/\(/g, "%28").replace(/\)/g, "%29");
a2a_config.templates = a2a_config.templates || {};

a2a_config.templates.email = {
    body: res
};
</script><div data-widget-id="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da" data-widget-name="UX3share" class="dropBlock article-tools__dropblock article-tools__share"><button data-db-target-for="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da" aria-expanded="false" title="open Share menu" class="article-tools__ctrl share__ctrl"><i aria-hidden="true" class="icon-article-share"></i><span>Share</span></button><div data-db-target-of="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da" id="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da_Pop" class="share__block axel-share dropBlock__holder article-tools__dropblock__holder"><h4 class="axel-share__title">Share</h4><div class="axel-share__inner"><span class="sr-only">Share on</span><ul class="a2a__share-list rlist w-slide--list a2a a2a_kit a2a_kit_size_32 a2a_default_style" data-a2a-url="https://www.cell.com/cell/fulltext/S0092-8674%2821%2901332-5" style="line-height: 32px;"><li class="a2a_listitem_custom"><a role="link" title="Email" class="a2a_button_email" target="_blank" rel="nofollow noopener" href="/#email"><i aria-hidden="true" class="at-icon-wrapper icon-gizmo-email-plus"></i><span>Email</span></a></li><li class="a2a_listitem_custom"><a role="link" title="formerly known as Twitter" class="a2a_button_twitter" target="_blank" rel="nofollow noopener" href="/#twitter"><i aria-hidden="true" class="at-icon-wrapper icon-x-default"></i><span>X</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Facebook" class="a2a_button_facebook" target="_blank" rel="nofollow noopener" href="/#facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook-square-2"></i><span>Facebook</span></a></li><li class="a2a_listitem_custom"><a role="link" title="LinkedIn" class="a2a_button_linkedin" target="_blank" rel="nofollow noopener" href="/#linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin-square"></i><span>LinkedIn</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Sina Weibo" class="a2a_button_sina_weibo" target="_blank" rel="nofollow noopener" href="/#sina_weibo"><i aria-hidden="true" class="at-icon-wrapper icon-weibo"></i><span>Sina Weibo</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Mendeley" class="a2a_button_mendeley" target="_blank" rel="nofollow noopener" href="/#mendeley"><i aria-hidden="true" class="at-icon-wrapper icon-mendeley-2"></i><span>Mendeley</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Bluesky" class="a2a_button_bluesky" target="_blank" rel="nofollow noopener" href="/#bluesky"><i aria-hidden="true" class="at-icon-wrapper icon-bluesky-logo"></i><span>Bluesky</span></a></li><div style="clear: both;"></div></ul><div class="extra-links-drop-zone">
    

        


    

            

            

            <div data-widget-id="ae9bf5c5-a2ee-4f5c-8e41-9b21638b1a9f" data-widget-name="UX3HTMLWidget"><a class="axel-share__extra" target="_blank" href="/personalize/addFavoritePublication?doi=10.1016/j.cell.2021.11.017">
	<i class="icon-reading-list" aria-hidden="true"></i>
	<span>Add to my reading list</span>
</a></div>

            

        
</div></div></div></div>

            

        
</li><li><a href="https://www.cell.com/action/addCitationAlert?doi=10.1016%2Fj.cell.2021.11.017" title="Set Alert" target="_blank" aria-label="Open Set Alert in new tab" class="article-tools__item__displayCreateCitationAlert"><i aria-hidden="true" class="icon-set-alert"></i><span>Set Alert</span></a></li><li><a href="https://s100.copyright.com/AppDispatchServlet?publisherName=ELS&amp;contentID=S0092867421013325&amp;orderBeanReset=true&amp;orderSource=Phoenix" title="Get Rights" target="_blank" aria-label="Open Get Rights in new tab" class="article-tools__item__request"><i aria-hidden="true" class="icon-get-rights"></i><span>Get Rights</span></a></li><li><a href="https://www.contentednet.com/salesforce-form/?pii=S0092867421013325&amp;issn=00928674&amp;source=jbs" title="Reprints" target="_blank" aria-label="Open Reprints in new tab" class="article-tools__item__reprints"><i aria-hidden="true" class="icon-article-reprints"></i><span>Reprints</span></a></li></ul>

            

        
</div></div><div class="info-panel__download-issue info-panel__item"><div class="article-header__download-full-issue"><button title="Download full issue" type="submit" formaction="/action/downloadMultiplePdf?action=downloadIssuePdf&amp;journal=cell&amp;volume=184&amp;issue=25&amp;pii=S0092-8674%2820%29X0026-2" class="btn--download-issue btn--cancel download-full-issue"><i class="icon-download-new"></i><span>Download Full Issue</span></button><div class="table-of-content__alert table-of-content__alert--download"><p>Download started</p><a href="#" class="table-of-content__alert--close">Ok</a></div></div></div></div><div class="info-panel__right-content"><div class="info-panel__navigation info-panel__item"><div class="content-navigation"><a href="/cell/fulltext/S0092-8674(21)01322-2" aria-label="Previous article" aria-disabled="false" class="content-navigation__prev"><i aria-hidden="true" class="icon-arrow_l"></i><span>Previous article</span></a><a href="/cell/fulltext/S0092-8674(21)01329-5" aria-label="Next article" aria-disabled="false" class="content-navigation__next"><span>Next article</span><i aria-hidden="true" class="icon-arrow_r"></i></a></div></div></div></div></div></div></div><div class="core-sections-menu-outer"><div class="core-sections-menu"><button aria-controls="article_sections_menu-desktop" aria-haspopup="true" aria-expanded="false" aria-label="Toggle sections menu" data-id="article-toolbar-showhide"><i class="icon-show-outline" aria-hidden="true"></i><span>Show Outline</span></button><nav id="article_sections_menu-desktop" aria-label="Contents" data-core-nav="article" style="margin-top: 0px;"><button aria-controls="article_sections_menu-desktop" aria-haspopup="true" aria-expanded="false" aria-label="Toggle sections menu" data-id="article-toolbar-showhide"><i class="icon-hide-outline" aria-hidden="true"></i><span>Hide Outline</span></button><ul><li><a href="#author-highlights-abstract">Highlights</a></li><li><a href="#author-abstract">Summary</a></li><li><a href="#graphical-abstract">Graphical abstract</a></li><li><a href="#keywords">Keywords</a></li><li><a href="#sec-1">Introduction</a></li><li><a href="#sec-2">Results</a></li><li><a href="#sec-3">Discussion</a></li><li><a href="#sec-4">STAR★Methods</a></li><li><a href="#acknowledgments">Acknowledgments</a></li><li><a href="#supplementary-material">Supplemental information (7)</a></li><li><a href="#data-availability">Data and code availability</a></li><li><a href="#references">References</a></li><li role="none" class="mobile-only"><a role="menuitem" href="#core-collateral-metrics">Article metrics</a></li><li role="none" class="mobile-only"><a role="menuitem" href="#core-collateral-relatedArticles">Related Articles</a></li></ul></nav></div></div><div><div class="core-content-inner"><div class="core-linked-content"></div><div id="abstracts" data-extent="frontmatter"><div class="core-container"><section id="author-highlights-abstract" property="abstract" typeof="Text" role="doc-abstract"><h2 property="name">Highlights</h2><div id="abspara0020" role="paragraph"><div id="ulist0010" role="list"><div id="u0010" role="listitem"><div class="label">•</div><div class="content"><div id="p0010" role="paragraph">scRNA-seq of metastatic pancreatic cancer and matched organoid models</div></div></div><div id="u0015" role="listitem"><div class="label">•</div><div class="content"><div id="p0015" role="paragraph"><i>Ex&nbsp;vivo</i> to <i>in&nbsp;vivo</i> comparisons reveal loss of cell state heterogeneity in models</div></div></div><div id="u0020" role="listitem"><div class="label">•</div><div class="content"><div id="p0020" role="paragraph">Cell state is shaped by the microenvironment <i>in&nbsp;vivo</i> and can be controlled <i>ex&nbsp;vivo</i></div></div></div><div id="u0025" role="listitem"><div class="label">•</div><div class="content"><div id="p0025" role="paragraph">Cell state drives drug response</div></div></div></div></div></section><section id="author-abstract" property="abstract" typeof="Text" role="doc-abstract"><h2 property="name">Summary</h2><div id="abspara0010" role="paragraph">Prognostically relevant RNA expression states exist in pancreatic ductal adenocarcinoma (PDAC), but our understanding of their drivers, stability, and relationship to therapeutic response is limited. To examine these attributes systematically, we profiled metastatic biopsies and matched organoid models at single-cell resolution. <i>In&nbsp;vivo</i>, we identify a new intermediate PDAC transcriptional cell state and uncover distinct site- and state-specific tumor microenvironments (TMEs). Benchmarking models against this reference map, we reveal strong culture-specific biases in cancer cell transcriptional state representation driven by altered TME signals. We restore expression state heterogeneity by adding back <i>in&nbsp;vivo</i>-relevant factors and show plasticity in culture models. Further, we prove that non-genetic modulation of cell state can strongly influence drug responses, uncovering state-specific vulnerabilities. This work provides a broadly applicable framework for aligning cell states across <i>in&nbsp;vivo</i> and <i>ex&nbsp;vivo</i> settings, identifying drivers of transcriptional plasticity and manipulating cell state to target associated vulnerabilities.</div></section><section id="graphical-abstract" property="abstract" typeof="Text" role="doc-abstract"><h2 property="name">Graphical abstract</h2><div id="abspara0015" role="paragraph"><div class="figure-wrap"><figure id="undfig1" class="graphic" data-figureviewer-ignore="true"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.11.017/asset/47885843-3d06-4f60-9675-c38cae0a2239/main.assets/fx1_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="fx1" alt="Graphical abstract undfig1" src="/cms/10.1016/j.cell.2021.11.017/asset/8197d954-74a1-42d8-bf84-c79b6d35b1c5/main.assets/fx1.jpg" height="375" width="375" data-viewer-src="/cms/10.1016/j.cell.2021.11.017/asset/47885843-3d06-4f60-9675-c38cae0a2239/main.assets/fx1_lrg.jpg" loading="lazy"></a></figure></div></div></section></div></div><div class="core-linked-content"></div><section id="keywords" property="keywords"><h2>Keywords</h2><ol><li><a href="/action/doSearch?AllField=&quot;pancreatic+cancer&quot;" alt="Search on this keyword" rel="nofollow">pancreatic cancer</a></li><li><a href="/action/doSearch?AllField=&quot;single-cell+RNA-sequencing&quot;" alt="Search on this keyword" rel="nofollow">single-cell RNA-sequencing</a></li><li><a href="/action/doSearch?AllField=&quot;transcriptional+states&quot;" alt="Search on this keyword" rel="nofollow">transcriptional states</a></li><li><a href="/action/doSearch?AllField=&quot;plasticity&quot;" alt="Search on this keyword" rel="nofollow">plasticity</a></li><li><a href="/action/doSearch?AllField=&quot;patient-derived+organoid+models&quot;" alt="Search on this keyword" rel="nofollow">patient-derived organoid models</a></li><li><a href="/action/doSearch?AllField=&quot;tumor+microenvironment&quot;" alt="Search on this keyword" rel="nofollow">tumor microenvironment</a></li><li><a href="/action/doSearch?AllField=&quot;liver+metastases&quot;" alt="Search on this keyword" rel="nofollow">liver metastases</a></li><li><a href="/action/doSearch?AllField=&quot;tumor+heterogeneity&quot;" alt="Search on this keyword" rel="nofollow">tumor heterogeneity</a></li></ol></section><div class="core-linked-content"></div><div class="core-linked-content"></div><section id="bodymatter" data-extent="bodymatter" property="articleBody" typeof="Text"><div class="core-container"><section id="sec-1"><h2>Introduction</h2><div id="p0045" role="paragraph">Recent advances in high-throughput genomic sequencing have led to a detailed understanding of the genetic alterations that underlie human tumors (<span class="dropBlock reference-citations"><a id="body-ref-sref27-1" href="#" role="doc-biblioref" data-xml-rid="bib27" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref27-1" aria-controls="sref27-1">Garraway and Lander, 2013</a><div class="dropBlock__holder" data-db-target-of="sref27-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">27.</div><div>Garraway, L.A. ∙ Lander, E.S.</div><div><strong>Lessons from the cancer genome</strong></div><div><em>Cell.</em> 2013; <strong>153</strong>:17-37</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=L.A.GarrawayE.S.LanderLessons+from+the+cancer+genomeCell15320131737" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). These reference maps have driven a “mutation-centric” view of cancer that informs our current approach to precision medicine. In this framework, DNA alterations are used as biomarkers to guide therapy selection (<span class="dropBlock reference-citations"><a id="body-ref-sref41-1" href="#" role="doc-biblioref" data-xml-rid="bib41" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref41-1" aria-controls="sref41-1">Hyman et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref41-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">41.</div><div>Hyman, D.M. ∙ Taylor, B.S. ∙ Baselga, J.</div><div><strong>Implementing Genome-Driven Oncology</strong></div><div><em>Cell.</em> 2017; <strong>168</strong>:584-599</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=D.M.HymanB.S.TaylorJ.BaselgaImplementing+Genome-Driven+OncologyCell1682017584599" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), and <i>ex&nbsp;vivo</i> models are used to validate mutational associations and power therapeutic discovery efforts. To maintain translational relevance, the fidelity of <i>ex&nbsp;vivo</i> models to <i>in&nbsp;vivo</i> attributes is paramount.</div><div id="p0050" role="paragraph">Driven by the mutation-centric view of cancer, model fidelity is typically assessed via genomic similarity (<span class="dropBlock reference-citations"><a id="body-ref-sref9-1" href="#" role="doc-biblioref" data-xml-rid="bib9" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref9-1" aria-controls="sref9-1">Ben-David et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref9-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">9.</div><div>Ben-David, U. ∙ Beroukhim, R. ∙ Golub, T.R.</div><div><strong>Genomic evolution of cancer models: perils and opportunities</strong></div><div><em>Nat. Rev. Cancer.</em> 2019; <strong>19</strong>:97-109</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=U.Ben-DavidR.BeroukhimT.R.GolubGenomic+evolution+of+cancer+models%3A+perils+and+opportunitiesNat.+Rev.+Cancer19201997109" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref14" href="#" role="doc-biblioref" data-xml-rid="bib14" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref14" aria-controls="sref14">Byrne et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref14"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">14.</div><div>Byrne, A.T. ∙ Alférez, D.G. ∙ Amant, F. ...</div><div><strong>Interrogating open issues in cancer medicine with patient-derived xenografts</strong></div><div><em>Nat. Rev. Cancer.</em> 2017; <strong>17</strong>:632</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.T.ByrneD.G.Alf%C3%A9rezF.AmantD.AnnibaliJ.ArribasA.V.BiankinA.BrunaE.Budinsk%C3%A1C.CaldasD.K.ChangInterrogating+open+issues+in+cancer+medicine+with+patient-derived+xenograftsNat.+Rev.+Cancer172017632" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref21" href="#" role="doc-biblioref" data-xml-rid="bib21" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref21" aria-controls="sref21">Drost and Clevers, 2018</a><div class="dropBlock__holder" data-db-target-of="sref21"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">21.</div><div>Drost, J. ∙ Clevers, H.</div><div><strong>Organoids in cancer research</strong></div><div><em>Nat. Rev. Cancer.</em> 2018; <strong>18</strong>:407-418</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.DrostH.CleversOrganoids+in+cancer+researchNat.+Rev.+Cancer182018407418" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref30" href="#" role="doc-biblioref" data-xml-rid="bib30" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref30" aria-controls="sref30">Gillet et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref30"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">30.</div><div>Gillet, J.P. ∙ Varma, S. ∙ Gottesman, M.M.</div><div><strong>The clinical relevance of cancer cell lines</strong></div><div><em>J.&nbsp;Natl. Cancer Inst.</em> 2013; <strong>105</strong>:452-458</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.P.GilletS.VarmaM.M.GottesmanThe+clinical+relevance+of+cancer+cell+linesJ.%C2%A0Natl.+Cancer+Inst.1052013452458" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Assessment of mutational fidelity is feasible and useful for two reasons: first, bulk measurements of DNA alterations have favorable signal-to-noise profiles (i.e., somatic mutations are present or absent); and, second, some cancers respond to therapies in a genotype-directed manner (<span class="dropBlock reference-citations"><a id="body-ref-sref27-2" href="#" role="doc-biblioref" data-xml-rid="bib27" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref27-2" aria-controls="sref27-2">Garraway and Lander, 2013</a><div class="dropBlock__holder" data-db-target-of="sref27-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">27.</div><div>Garraway, L.A. ∙ Lander, E.S.</div><div><strong>Lessons from the cancer genome</strong></div><div><em>Cell.</em> 2013; <strong>153</strong>:17-37</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=L.A.GarrawayE.S.LanderLessons+from+the+cancer+genomeCell15320131737" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref41-2" href="#" role="doc-biblioref" data-xml-rid="bib41" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref41-2" aria-controls="sref41-2">Hyman et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref41-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">41.</div><div>Hyman, D.M. ∙ Taylor, B.S. ∙ Baselga, J.</div><div><strong>Implementing Genome-Driven Oncology</strong></div><div><em>Cell.</em> 2017; <strong>168</strong>:584-599</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=D.M.HymanB.S.TaylorJ.BaselgaImplementing+Genome-Driven+OncologyCell1682017584599" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). However, a growing body of evidence indicates that using mutations alone to assign therapies has limitations (<span class="dropBlock reference-citations"><a id="body-ref-sref51" href="#" role="doc-biblioref" data-xml-rid="bib51" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref51" aria-controls="sref51">Nam et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref51"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">51.</div><div>Nam, A.S. ∙ Chaligne, R. ∙ Landau, D.A.</div><div><strong>Integrating genetic and non-genetic determinants of cancer evolution by single-cell multi-omics</strong></div><div><em>Nat. Rev. Genet.</em> 2021; <strong>22</strong>:3-18</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.S.NamR.ChaligneD.A.LandauIntegrating+genetic+and+non-genetic+determinants+of+cancer+evolution+by+single-cell+multi-omicsNat.+Rev.+Genet.222021318" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref71" href="#" role="doc-biblioref" data-xml-rid="bib71" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref71" aria-controls="sref71">van de Haar et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref71"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">71.</div><div>van de Haar, J. ∙ Hoes, L.R. ∙ Roepman, P. ...</div><div><strong>Limited evolution of the actionable metastatic cancer genome under therapeutic pressure</strong></div><div><em>Nat. Med.</em> 2021; <strong>27</strong>:1553-1563</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.van+de+HaarL.R.HoesP.RoepmanM.P.LolkemaH.M.W.VerheulH.GelderblomA.J.de+LangenE.F.SmitE.CuppenL.F.A.WesselsE.E.VoestLimited+evolution+of+the+actionable+metastatic+cancer+genome+under+therapeutic+pressureNat.+Med.27202115531563" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The advent of single-cell genomic technologies has confirmed extensive mutational heterogeneity in human tumors but also revealed that the complexity of cancer extends to variation in cell transcriptional state. The relationship between cell state and therapeutic sensitivity represents a new but poorly understood opportunity for cancer therapeutic development (<span class="dropBlock reference-citations"><a id="body-ref-sref34-1" href="#" role="doc-biblioref" data-xml-rid="bib34" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref34-1" aria-controls="sref34-1">Hahn et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref34-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">34.</div><div>Hahn, W.C. ∙ Bader, J.S. ∙ Braun, T.P. ..., Cancer Target Discovery and Development Network</div><div><strong>An expanded universe of cancer targets</strong></div><div><em>Cell.</em> 2021; <strong>184</strong>:1142-1155</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=W.C.HahnJ.S.BaderT.P.BraunA.CalifanoP.A.ClemonsB.J.DrukerA.J.EwaldH.FuS.JaguC.J.KempCancer+Target+Discovery+and+Development+NetworkAn+expanded+universe+of+cancer+targetsCell184202111421155" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="p0055" role="paragraph">Cell state, as measured by RNA expression, is a complex representation of tumor phenotype because it integrates inputs from cell-intrinsic (e.g., mutational background, epigenetic state) and cell-extrinsic (e.g., cell-to-cell interactions, tissue architecture) sources. Although the field has generated high-resolution single-cell RNA sequencing (scRNA-seq) maps of cancer cell transcriptional states across diverse contexts (<span class="dropBlock reference-citations"><a id="body-ref-sref24" href="#" role="doc-biblioref" data-xml-rid="bib24" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref24" aria-controls="sref24">Filbin et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref24"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">24.</div><div>Filbin, M.G. ∙ Tirosh, I. ∙ Hovestadt, V. ...</div><div><strong>Developmental and oncogenic programs in H3K27M gliomas dissected by single-cell RNA-seq</strong></div><div><em>Science.</em> 2018; <strong>360</strong>:331-335</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.G.FilbinI.TiroshV.HovestadtM.L.ShawL.E.EscalanteN.D.MathewsonC.NeftelN.FrankK.PeltonC.M.HebertDevelopmental+and+oncogenic+programs+in+H3K27M+gliomas+dissected+by+single-cell+RNA-seqScience3602018331335" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref39" href="#" role="doc-biblioref" data-xml-rid="bib39" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref39" aria-controls="sref39">Hovestadt et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref39"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">39.</div><div>Hovestadt, V. ∙ Smith, K.S. ∙ Bihannic, L. ...</div><div><strong>Resolving medulloblastoma cellular architecture by single-cell genomics</strong></div><div><em>Nature.</em> 2019; <strong>572</strong>:74-79</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=V.HovestadtK.S.SmithL.BihannicM.G.FilbinM.L.ShawA.BaumgartnerJ.C.DeWittA.GrovesL.MayrH.R.WeismanResolving+medulloblastoma+cellular+architecture+by+single-cell+genomicsNature57220197479" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref43-1" href="#" role="doc-biblioref" data-xml-rid="bib43" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref43-1" aria-controls="sref43-1">Kim et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref43-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">43.</div><div>Kim, C. ∙ Gao, R. ∙ Sei, E. ...</div><div><strong>Chemoresistance Evolution in Triple-Negative Breast Cancer Delineated by Single-Cell Sequencing</strong></div><div><em>Cell.</em> 2018; <strong>173</strong>:879-893 e813</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=C.KimR.GaoE.SeiR.BrandtJ.HartmanT.HatschekN.CrosettoT.FoukakisN.E.NavinChemoresistance+Evolution+in+Triple-Negative+Breast+Cancer+Delineated+by+Single-Cell+SequencingCell1732018879893+e813" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref52" href="#" role="doc-biblioref" data-xml-rid="bib52" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref52" aria-controls="sref52">Neftel et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref52"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">52.</div><div>Neftel, C. ∙ Laffy, J. ∙ Filbin, M.G. ...</div><div><strong>An Integrative Model of Cellular States, Plasticity, and Genetics for Glioblastoma</strong></div><div><em>Cell.</em> 2019; <strong>178</strong>:835-849 e821</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=C.NeftelJ.LaffyM.G.FilbinT.HaraM.E.ShoreG.J.RahmeA.R.RichmanD.SilverbushM.L.ShawC.M.HebertAn+Integrative+Model+of+Cellular+States%2C+Plasticity%2C+and+Genetics+for+GlioblastomaCell1782019835849+e821" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref57-1" href="#" role="doc-biblioref" data-xml-rid="bib57" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref57-1" aria-controls="sref57-1">Patel et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref57-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">57.</div><div>Patel, A.P. ∙ Tirosh, I. ∙ Trombetta, J.J. ...</div><div><strong>Single-cell RNA-seq highlights intratumoral heterogeneity in primary glioblastoma</strong></div><div><em>Science.</em> 2014; <strong>344</strong>:1396-1401</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.P.PatelI.TiroshJ.J.TrombettaA.K.ShalekS.M.GillespieH.WakimotoD.P.CahillB.V.NahedW.T.CurryR.L.MartuzaSingle-cell+RNA-seq+highlights+intratumoral+heterogeneity+in+primary+glioblastomaScience344201413961401" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref60-1" href="#" role="doc-biblioref" data-xml-rid="bib60" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref60-1" aria-controls="sref60-1">Puram et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref60-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">60.</div><div>Puram, S.V. ∙ Tirosh, I. ∙ Parikh, A.S. ...</div><div><strong>Single-Cell Transcriptomic Analysis of Primary and Metastatic Tumor Ecosystems in Head and Neck Cancer</strong></div><div><em>Cell.</em> 2017; <strong>171</strong>:1611-1624 e1624</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=S.V.PuramI.TiroshA.S.ParikhA.P.PatelK.YizhakS.GillespieC.RodmanC.L.LuoE.A.MrozK.S.EmerickSingle-Cell+Transcriptomic+Analysis+of+Primary+and+Metastatic+Tumor+Ecosystems+in+Head+and+Neck+CancerCell171201716111624+e1624" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref62-1" href="#" role="doc-biblioref" data-xml-rid="bib62" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref62-1" aria-controls="sref62-1">Sade-Feldman et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref62-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">62.</div><div>Sade-Feldman, M. ∙ Yizhak, K. ∙ Bjorgaard, S.L. ...</div><div><strong>Defining T Cell States Associated with Response to Checkpoint Immunotherapy in Melanoma</strong></div><div><em>Cell.</em> 2019; <strong>176</strong>:404</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.Sade-FeldmanK.YizhakS.L.BjorgaardJ.P.RayC.G.de+BoerR.W.JenkinsD.J.LiebJ.H.ChenD.T.FrederickM.Barzily-RokniDefining+T+Cell+States+Associated+with+Response+to+Checkpoint+Immunotherapy+in+MelanomaCell1762019404" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref67" href="#" role="doc-biblioref" data-xml-rid="bib67" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref67" aria-controls="sref67">Suvà and Tirosh, 2019</a><div class="dropBlock__holder" data-db-target-of="sref67"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">67.</div><div>Suvà, M.L. ∙ Tirosh, I.</div><div><strong>Single-Cell RNA Sequencing in Cancer: Lessons Learned and Emerging Challenges</strong></div><div><em>Mol. Cell.</em> 2019; <strong>75</strong>:7-12</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.L.Suv%C3%A0I.TiroshSingle-Cell+RNA+Sequencing+in+Cancer%3A+Lessons+Learned+and+Emerging+ChallengesMol.+Cell752019712" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref69-1" href="#" role="doc-biblioref" data-xml-rid="bib69" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref69-1" aria-controls="sref69-1">Tirosh et&nbsp;al., 2016a</a><div class="dropBlock__holder" data-db-target-of="sref69-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">69.</div><div>Tirosh, I. ∙ Izar, B. ∙ Prakadan, S.M. ...</div><div><strong>Dissecting the multicellular ecosystem of metastatic melanoma by single-cell RNA-seq</strong></div><div><em>Science.</em> 2016; <strong>352</strong>:189-196</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=I.TiroshB.IzarS.M.PrakadanM.H.Wadsworth2ndD.TreacyJ.J.TrombettaA.RotemC.RodmanC.LianG.MurphyDissecting+the+multicellular+ecosystem+of+metastatic+melanoma+by+single-cell+RNA-seqScience3522016189196" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref70-1" href="#" role="doc-biblioref" data-xml-rid="bib70" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref70-1" aria-controls="sref70-1">Tirosh et&nbsp;al., 2016b</a><div class="dropBlock__holder" data-db-target-of="sref70-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">70.</div><div>Tirosh, I. ∙ Venteicher, A.S. ∙ Hebert, C. ...</div><div><strong>Single-cell RNA-seq supports a developmental hierarchy in human oligodendroglioma</strong></div><div><em>Nature.</em> 2016; <strong>539</strong>:309-313</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=I.TiroshA.S.VenteicherC.HebertL.E.EscalanteA.P.PatelK.YizhakJ.M.FisherC.RodmanC.MountM.G.FilbinSingle-cell+RNA-seq+supports+a+developmental+hierarchy+in+human+oligodendrogliomaNature5392016309313" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref72-1" href="#" role="doc-biblioref" data-xml-rid="bib72" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref72-1" aria-controls="sref72-1">van Galen et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref72-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">72.</div><div>van Galen, P. ∙ Hovestadt, V. ∙ Wadsworth Ii, M.H. ...</div><div><strong>Single-Cell RNA-Seq Reveals AML Hierarchies Relevant to Disease Progression and Immunity</strong></div><div><em>Cell.</em> 2019; <strong>176</strong>:1265-1281 e1224</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=P.van+GalenV.HovestadtM.H.Wadsworth+IiT.K.HughesG.K.GriffinS.BattagliaJ.A.VergaJ.StephanskyT.J.PastikaJ.Lombardi+StorySingle-Cell+RNA-Seq+Reveals+AML+Hierarchies+Relevant+to+Disease+Progression+and+ImmunityCell176201912651281+e1224" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref73" href="#" role="doc-biblioref" data-xml-rid="bib73" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref73" aria-controls="sref73">Venteicher et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref73"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">73.</div><div>Venteicher, A.S. ∙ Tirosh, I. ∙ Hebert, C. ...</div><div><strong>Decoupling genetics, lineages, and microenvironment in IDH-mutant gliomas by single-cell RNA-seq</strong></div><div><em>Science.</em> 2017; <strong>355</strong>:eaai8478</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.S.VenteicherI.TiroshC.HebertK.YizhakC.NeftelM.G.FilbinV.HovestadtL.E.EscalanteM.L.ShawC.RodmanDecoupling+genetics%2C+lineages%2C+and+microenvironment+in+IDH-mutant+gliomas+by+single-cell+RNA-seqScience3552017eaai8478" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), we have not mapped their stability or the relative influences of cell-intrinsic and cell-extrinsic factors in specifying them. Moreover, we have a limited understanding of the degree to which models accurately recapitulate the distribution of cancer cell states seen in patients.</div><div id="p0060" role="paragraph">Despite extensive genomic characterization of pancreatic ductal adenocarcinoma (PDAC), most cancers do not harbor therapeutically tractable alterations (<span class="dropBlock reference-citations"><a id="body-ref-sref2-1" href="#" role="doc-biblioref" data-xml-rid="bib2" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref2-1" aria-controls="sref2-1">Aguirre et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref2-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">2.</div><div>Aguirre, A.J. ∙ Nowak, J.A. ∙ Camarda, N.D. ...</div><div><strong>Real-time Genomic Characterization of Advanced Pancreatic Cancer to Enable Precision Medicine</strong></div><div><em>Cancer Discov.</em> 2018; <strong>8</strong>:1096-1111</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.J.AguirreJ.A.NowakN.D.CamardaR.A.MoffittA.A.GhazaniM.Hazar-RethinamS.RaghavanJ.KimL.K.BraisD.RagonReal-time+Genomic+Characterization+of+Advanced+Pancreatic+Cancer+to+Enable+Precision+MedicineCancer+Discov.8201810961111" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref5-1" href="#" role="doc-biblioref" data-xml-rid="bib5" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref5-1" aria-controls="sref5-1">Bailey et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref5-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">5.</div><div>Bailey, P. ∙ Chang, D.K. ∙ Nones, K. ..., Australian Pancreatic Cancer Genome Initiative</div><div><strong>Genomic analyses identify molecular subtypes of pancreatic cancer</strong></div><div><em>Nature.</em> 2016; <strong>531</strong>:47-52</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=P.BaileyD.K.ChangK.NonesA.L.JohnsA.M.PatchM.C.GingrasD.K.MillerA.N.ChristT.J.BruxnerM.C.QuinnAustralian+Pancreatic+Cancer+Genome+InitiativeGenomic+analyses+identify+molecular+subtypes+of+pancreatic+cancerNature53120164752" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref15-1" href="#" role="doc-biblioref" data-xml-rid="bib15" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref15-1" aria-controls="sref15-1">Cancer Genome Atlas Research Network, 2017</a><div class="dropBlock__holder" data-db-target-of="sref15-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">15.</div><div>Cancer Genome Atlas Research Network</div><div><strong>Integrated Genomic Characterization of Pancreatic Ductal Adenocarcinoma</strong></div><div><em>Cancer Cell.</em> 2017; <strong>32</strong>:185-203.e13</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=Cancer+Genome+Atlas+Research+NetworkIntegrated+Genomic+Characterization+of+Pancreatic+Ductal+AdenocarcinomaCancer+Cell322017185203.e13" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). However, RNA subtypes (states) derived from bulk measurements have emerged as an important clinical biomarker (<span class="dropBlock reference-citations"><a id="body-ref-sref2-2" href="#" role="doc-biblioref" data-xml-rid="bib2" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref2-2" aria-controls="sref2-2">Aguirre et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref2-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">2.</div><div>Aguirre, A.J. ∙ Nowak, J.A. ∙ Camarda, N.D. ...</div><div><strong>Real-time Genomic Characterization of Advanced Pancreatic Cancer to Enable Precision Medicine</strong></div><div><em>Cancer Discov.</em> 2018; <strong>8</strong>:1096-1111</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.J.AguirreJ.A.NowakN.D.CamardaR.A.MoffittA.A.GhazaniM.Hazar-RethinamS.RaghavanJ.KimL.K.BraisD.RagonReal-time+Genomic+Characterization+of+Advanced+Pancreatic+Cancer+to+Enable+Precision+MedicineCancer+Discov.8201810961111" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref4-1" href="#" role="doc-biblioref" data-xml-rid="bib4" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref4-1" aria-controls="sref4-1">Aung et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref4-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">4.</div><div>Aung, K.L. ∙ Fischer, S.E. ∙ Denroche, R.E. ...</div><div><strong>Genomics-Driven Precision Medicine for Advanced Pancreatic Cancer: Early Results from the COMPASS Trial</strong></div><div><em>Clin. Cancer Res.</em> 2018; <strong>24</strong>:1344-1354</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=K.L.AungS.E.FischerR.E.DenrocheG.H.JangA.DoddS.CreightonB.SouthwoodS.B.LiangD.ChadwickA.ZhangGenomics-Driven+Precision+Medicine+for+Advanced+Pancreatic+Cancer%3A+Early+Results+from+the+COMPASS+TrialClin.+Cancer+Res.24201813441354" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref5-2" href="#" role="doc-biblioref" data-xml-rid="bib5" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref5-2" aria-controls="sref5-2">Bailey et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref5-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">5.</div><div>Bailey, P. ∙ Chang, D.K. ∙ Nones, K. ..., Australian Pancreatic Cancer Genome Initiative</div><div><strong>Genomic analyses identify molecular subtypes of pancreatic cancer</strong></div><div><em>Nature.</em> 2016; <strong>531</strong>:47-52</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=P.BaileyD.K.ChangK.NonesA.L.JohnsA.M.PatchM.C.GingrasD.K.MillerA.N.ChristT.J.BruxnerM.C.QuinnAustralian+Pancreatic+Cancer+Genome+InitiativeGenomic+analyses+identify+molecular+subtypes+of+pancreatic+cancerNature53120164752" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref15-2" href="#" role="doc-biblioref" data-xml-rid="bib15" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref15-2" aria-controls="sref15-2">Cancer Genome Atlas Research Network, 2017</a><div class="dropBlock__holder" data-db-target-of="sref15-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">15.</div><div>Cancer Genome Atlas Research Network</div><div><strong>Integrated Genomic Characterization of Pancreatic Ductal Adenocarcinoma</strong></div><div><em>Cancer Cell.</em> 2017; <strong>32</strong>:185-203.e13</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=Cancer+Genome+Atlas+Research+NetworkIntegrated+Genomic+Characterization+of+Pancreatic+Ductal+AdenocarcinomaCancer+Cell322017185203.e13" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref16-1" href="#" role="doc-biblioref" data-xml-rid="bib16" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref16-1" aria-controls="sref16-1">Chan-Seng-Yue et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref16-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">16.</div><div>Chan-Seng-Yue, M. ∙ Kim, J.C. ∙ Wilson, G.W. ...</div><div><strong>Transcription phenotypes of pancreatic cancer are driven by genomic events during tumor evolution</strong></div><div><em>Nat. Genet.</em> 2020; <strong>52</strong>:231-240</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.Chan-Seng-YueJ.C.KimG.W.WilsonK.NgE.F.FigueroaG.M.O%E2%80%99KaneA.A.ConnorR.E.DenrocheR.C.GrantJ.McLeodTranscription+phenotypes+of+pancreatic+cancer+are+driven+by+genomic+events+during+tumor+evolutionNat.+Genet.522020231240" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref18-1" href="#" role="doc-biblioref" data-xml-rid="bib18" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref18-1" aria-controls="sref18-1">Collisson et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref18-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">18.</div><div>Collisson, E.A. ∙ Bailey, P. ∙ Chang, D.K. ...</div><div><strong>Molecular subtypes of pancreatic cancer</strong></div><div><em>Nat. Rev. Gastroenterol. Hepatol.</em> 2019; <strong>16</strong>:207-220</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=E.A.CollissonP.BaileyD.K.ChangA.V.BiankinMolecular+subtypes+of+pancreatic+cancerNat.+Rev.+Gastroenterol.+Hepatol.162019207220" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref17-1" href="#" role="doc-biblioref" data-xml-rid="bib17" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref17-1" aria-controls="sref17-1">Collisson et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref17-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">17.</div><div>Collisson, E.A. ∙ Sadanandam, A. ∙ Olson, P. ...</div><div><strong>Subtypes of pancreatic ductal adenocarcinoma and their differing responses to therapy</strong></div><div><em>Nat. Med.</em> 2011; <strong>17</strong>:500-503</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=E.A.CollissonA.SadanandamP.OlsonW.J.GibbM.TruittS.GuJ.CoocJ.WeinkleG.E.KimL.JakkulaSubtypes+of+pancreatic+ductal+adenocarcinoma+and+their+differing+responses+to+therapyNat.+Med.172011500503" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref19-1" href="#" role="doc-biblioref" data-xml-rid="bib19" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref19-1" aria-controls="sref19-1">Connor et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref19-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">19.</div><div>Connor, A.A. ∙ Denroche, R.E. ∙ Jang, G.H. ...</div><div><strong>Integration of Genomic and Transcriptional Features in Pancreatic Cancer Reveals Increased Cell Cycle Progression in Metastases</strong></div><div><em>Cancer Cell.</em> 2019; <strong>35</strong>:267-282 e267</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.A.ConnorR.E.DenrocheG.H.JangM.LemireA.ZhangM.Chan-Seng-YueG.WilsonR.C.GrantD.MericoI.LunguIntegration+of+Genomic+and+Transcriptional+Features+in+Pancreatic+Cancer+Reveals+Increased+Cell+Cycle+Progression+in+MetastasesCancer+Cell352019267282+e267" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref36" href="#" role="doc-biblioref" data-xml-rid="bib36" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref36" aria-controls="sref36">Hayashi et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref36"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">36.</div><div>Hayashi, A. ∙ Fan, J. ∙ Chen, R. ...</div><div><strong>A unifying paradigm for transcriptional heterogeneity and squamous features in pancreatic ductal adenocarcinoma</strong></div><div><em>Nature Cancer.</em> 2020; <strong>1</strong>:59-74</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.HayashiJ.FanR.ChenY.-j.HoA.P.Makohon-MooreN.LecomteY.ZhongJ.HongJ.HuangH.SakamotoA+unifying+paradigm+for+transcriptional+heterogeneity+and+squamous+features+in+pancreatic+ductal+adenocarcinomaNature+Cancer120205974" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref50-1" href="#" role="doc-biblioref" data-xml-rid="bib50" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref50-1" aria-controls="sref50-1">Moffitt et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref50-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">50.</div><div>Moffitt, R.A. ∙ Marayati, R. ∙ Flate, E.L. ...</div><div><strong>Virtual microdissection identifies distinct tumor- and stroma-specific subtypes of pancreatic ductal adenocarcinoma</strong></div><div><em>Nat. Genet.</em> 2015; <strong>47</strong>:1168-1178</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=R.A.MoffittR.MarayatiE.L.FlateK.E.VolmarS.G.LoezaK.A.HoadleyN.U.RashidL.A.WilliamsS.C.EatonA.H.ChungVirtual+microdissection+identifies+distinct+tumor-+and+stroma-specific+subtypes+of+pancreatic+ductal+adenocarcinomaNat.+Genet.47201511681178" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref53-1" href="#" role="doc-biblioref" data-xml-rid="bib53" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref53-1" aria-controls="sref53-1">O’Kane et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref53-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">53.</div><div>O’Kane, G.M. ∙ Grunwald, B.T. ∙ Jang, G.H. ...</div><div><strong>GATA6 Expression Distinguishes Classical and Basal-like Subtypes in Advanced Pancreatic Cancer</strong></div><div><em>Clin Cancer Res.</em> 2020; <strong>26</strong>:4901-4910</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=G.M.O%E2%80%99KaneB.T.GrunwaldG.H.JangM.MasoomianS.PicardoR.C.GrantR.E.DenrocheA.ZhangY.WangB.LamGATA6+Expression+Distinguishes+Classical+and+Basal-like+Subtypes+in+Advanced+Pancreatic+CancerClin+Cancer+Res.26202049014910" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref59" href="#" role="doc-biblioref" data-xml-rid="bib59" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref59" aria-controls="sref59">Porter et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref59"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">59.</div><div>Porter, R.L. ∙ Magnus, N.K.C. ∙ Thapar, V. ...</div><div><strong>Epithelial to mesenchymal plasticity and differential response to therapies in pancreatic ductal adenocarcinoma</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2019; <strong>116</strong>:26835-26845</div></div><div class="external-links"><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31843922/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?pmid=31843922" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref68-1" href="#" role="doc-biblioref" data-xml-rid="bib68" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref68-1" aria-controls="sref68-1">Tiriac et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref68-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">68.</div><div>Tiriac, H. ∙ Belleau, P. ∙ Engle, D.D. ...</div><div><strong>Organoid Profiling Identifies Common Responders to Chemotherapy in Pancreatic Cancer</strong></div><div><em>Cancer Discov.</em> 2018; <strong>8</strong>:1112-1129</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=H.TiriacP.BelleauD.D.EngleD.PlenkerA.Desch%C3%AAnesT.D.D.SomervilleF.E.M.FroelingR.A.BurkhartR.E.DenrocheG.H.JangOrganoid+Profiling+Identifies+Common+Responders+to+Chemotherapy+in+Pancreatic+CancerCancer+Discov.8201811121129" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). While PDAC subtyping studies have largely recovered common expression features, it remains unclear whether these bulk RNA-seq state measurements mask heterogeneity at the single-cell level, which features derive from malignant versus non-malignant cells, and how well they are recapitulated in laboratory models. Moreover, the PDAC tumor microenvironment (TME) includes numerous non-malignant immune and stromal cell types (<span class="dropBlock reference-citations"><a id="body-ref-sref6" href="#" role="doc-biblioref" data-xml-rid="bib6" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref6" aria-controls="sref6">Balachandran et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">6.</div><div>Balachandran, V.P. ∙ Beatty, G.L. ∙ Dougan, S.K.</div><div><strong>Broadening the Impact of Immunotherapy to Pancreatic Cancer: Challenges and Opportunities</strong></div><div><em>Gastroenterology.</em> 2019; <strong>156</strong>:2056-2072</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=V.P.BalachandranG.L.BeattyS.K.DouganBroadening+the+Impact+of+Immunotherapy+to+Pancreatic+Cancer%3A+Challenges+and+OpportunitiesGastroenterology156201920562072" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref10" href="#" role="doc-biblioref" data-xml-rid="bib10" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref10" aria-controls="sref10">Bernard et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref10"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">10.</div><div>Bernard, V. ∙ Semaan, A. ∙ Huang, J. ...</div><div><strong>Single-Cell Transcriptomics of Pancreatic Cancer Precursors Demonstrates Epithelial and Microenvironmental Heterogeneity as an Early Event in Neoplastic Progression</strong></div><div><em>Clin. Cancer Res.</em> 2019; <strong>25</strong>:2194-2205</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=V.BernardA.SemaanJ.HuangF.A.San+LucasF.C.MuluB.M.StephensP.A.GuerreroY.HuangJ.ZhaoN.KamyabiSingle-Cell+Transcriptomics+of+Pancreatic+Cancer+Precursors+Demonstrates+Epithelial+and+Microenvironmental+Heterogeneity+as+an+Early+Event+in+Neoplastic+ProgressionClin.+Cancer+Res.25201921942205" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref22-1" href="#" role="doc-biblioref" data-xml-rid="bib22" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref22-1" aria-controls="sref22-1">Elyada et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref22-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">22.</div><div>Elyada, E. ∙ Bolisetty, M. ∙ Laise, P. ...</div><div><strong>Cross-Species Single-Cell Analysis of Pancreatic Ductal Adenocarcinoma Reveals Antigen-Presenting Cancer-Associated Fibroblasts</strong></div><div><em>Cancer Discov.</em> 2019; <strong>9</strong>:1102-1123</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=E.ElyadaM.BolisettyP.LaiseW.F.FlynnE.T.CourtoisR.A.BurkhartJ.A.TeinorP.BelleauG.BiffiM.S.LucitoCross-Species+Single-Cell+Analysis+of+Pancreatic+Ductal+Adenocarcinoma+Reveals+Antigen-Presenting+Cancer-Associated+FibroblastsCancer+Discov.9201911021123" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref32-1" href="#" role="doc-biblioref" data-xml-rid="bib32" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref32-1" aria-controls="sref32-1">Grünwald et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref32-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">32.</div><div>Grünwald, B.T. ∙ Devisme, A. ∙ Andrieux, G. ...</div><div><strong>Spatially confined sub-tumor microenvironments in pancreatic cancer</strong></div><div><em>Cell.</em> 2021; <strong>184</strong>:5577-5592.e18</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=B.T.Gr%C3%BCnwaldA.DevismeG.AndrieuxF.VyasK.AliarC.W.McCloskeyA.MacklinG.H.JangR.DenrocheJ.M.RomeroSpatially+confined+sub-tumor+microenvironments+in+pancreatic+cancerCell184202155775592.e18" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref46" href="#" role="doc-biblioref" data-xml-rid="bib46" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref46" aria-controls="sref46">Ligorio et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref46"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">46.</div><div>Ligorio, M. ∙ Sil, S. ∙ Malagon-Lopez, J. ...</div><div><strong>Stromal Microenvironment Shapes the Intratumoral Architecture of Pancreatic Cancer.</strong></div><div><em>Cell.</em> 2019; <strong>178</strong>:160-175 e127</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.LigorioS.SilJ.Malagon-LopezL.T.NiemanS.MisaleM.Di+PilatoR.Y.EbrightM.N.KarabacakA.S.KulkarniA.LiuStromal+Microenvironment+Shapes+the+Intratumoral+Architecture+of+Pancreatic+Cancer.Cell1782019160175+e127" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), but their variation across different sites of disease and their effects on malignant cell state and therapeutic response is not well characterized. Given the lack of mutational biomarkers for PDAC, understanding how cell state is shaped by the local TME and whether cell state can be used as a tractable biomarker for therapy selection remains of critical importance.</div><div id="p0065" role="paragraph">To parse the instructive roles of cell-intrinsic and cell-extrinsic inputs to cancer cell state, we developed and employed an optimized translational workflow to perform both high-resolution profiling of metastatic PDAC patient tissue using scRNA-seq (<span class="dropBlock reference-citations"><a id="body-ref-sref29-1" href="#" role="doc-biblioref" data-xml-rid="bib29" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref29-1" aria-controls="sref29-1">Gierahn et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref29-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">29.</div><div>Gierahn, T.M. ∙ Wadsworth, 2nd, M.H. ∙ Hughes, T.K. ...</div><div><strong>Seq-Well: portable, low-cost RNA sequencing of single cells at high throughput</strong></div><div><em>Nat. Methods.</em> 2017; <strong>14</strong>:395-398</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=T.M.GierahnM.H.Wadsworth2ndT.K.HughesB.D.BrysonA.ButlerR.SatijaS.FortuneJ.C.LoveA.K.ShalekSeq-Well%3A+portable%2C+low-cost+RNA+sequencing+of+single+cells+at+high+throughputNat.+Methods142017395398" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref40-1" href="#" role="doc-biblioref" data-xml-rid="bib40" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref40-1" aria-controls="sref40-1">Hughes et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref40-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">40.</div><div>Hughes, T.K. ∙ Wadsworth, 2nd, M.H. ∙ Gierahn, T.M. ...</div><div><strong>Second-Strand Synthesis-Based Massively Parallel scRNA-Seq Reveals Cellular States and Molecular Features of Human Inflammatory Skin Pathologies</strong></div><div><em>Immunity.</em> 2020; <strong>53</strong>:878-894 e877</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=T.K.HughesM.H.Wadsworth2ndT.M.GierahnT.DoD.WeissP.R.AndradeF.MaB.J.de+Andrade+SilvaS.ShaoL.C.TsoiSecond-Strand+Synthesis-Based+Massively+Parallel+scRNA-Seq+Reveals+Cellular+States+and+Molecular+Features+of+Human+Inflammatory+Skin+PathologiesImmunity532020878894+e877" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and derivation of matched organoid models (<span class="dropBlock reference-citations"><a id="body-ref-sref12-1" href="#" role="doc-biblioref" data-xml-rid="bib12" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref12-1" aria-controls="sref12-1">Boj et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref12-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">12.</div><div>Boj, S.F. ∙ Hwang, C.I. ∙ Baker, L.A. ...</div><div><strong>Organoid models of human and mouse ductal pancreatic cancer</strong></div><div><em>Cell.</em> 2015; <strong>160</strong>:324-338</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=S.F.BojC.I.HwangL.A.BakerI.I.ChioD.D.EngleV.CorboM.JagerM.Ponz-SarviseH.TiriacM.S.SpectorOrganoid+models+of+human+and+mouse+ductal+pancreatic+cancerCell1602015324338" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref68-2" href="#" role="doc-biblioref" data-xml-rid="bib68" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref68-2" aria-controls="sref68-2">Tiriac et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref68-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">68.</div><div>Tiriac, H. ∙ Belleau, P. ∙ Engle, D.D. ...</div><div><strong>Organoid Profiling Identifies Common Responders to Chemotherapy in Pancreatic Cancer</strong></div><div><em>Cancer Discov.</em> 2018; <strong>8</strong>:1112-1129</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=H.TiriacP.BelleauD.D.EngleD.PlenkerA.Desch%C3%AAnesT.D.D.SomervilleF.E.M.FroelingR.A.BurkhartR.E.DenrocheG.H.JangOrganoid+Profiling+Identifies+Common+Responders+to+Chemotherapy+in+Pancreatic+CancerCancer+Discov.8201811121129" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) from the same core needle biopsy. Using this approach, we generated a single-cell map of metastatic PDAC and used it as a reference to benchmark cell states in matched organoid models. We identify a new intermediate co-expressor (IC) PDAC cell state, uncover distinct site- and subtype-specific TMEs, and demonstrate that TME signals are critical regulators of cancer cell state, plasticity, and response to therapy.</div></section><section id="sec-2"><h2>Results</h2><section id="sec-2-1"><h3>Model systems retain genetic fidelity but lose expression state heterogeneity</h3><div id="p0070" role="paragraph">Current precision medicine pipelines focus on preserving mutational fidelity in cancer models; however, it is unclear how well these same models represent prognostic RNA states (<a id="crosref0900" href="#fig1">Figure&nbsp;1</a>A). We compared bulk DNA and RNA-sequencing profiles of primary and metastatic patient tumors with established cell lines and a cohort of newly generated patient-derived organoids to understand how each model system represents the distribution of mutational and RNA phenotypes seen in patients. We observed no significant difference in driver oncogene alteration frequencies among the groups, suggesting that model systems are relatively faithful genetic representations of patient cohorts (Fisher’s exact test; <a id="crosref0905" href="#fig1">Figure&nbsp;1</a>B<b>,</b> far right). Next, we assessed PDAC subtypes derived from several recent bulk RNA-sequencing studies (<a id="crosref0910" href="#fig1">Figure&nbsp;1</a>C) (<span class="dropBlock reference-citations"><a id="body-ref-sref5-3" href="#" role="doc-biblioref" data-xml-rid="bib5" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref5-3" aria-controls="sref5-3">Bailey et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref5-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">5.</div><div>Bailey, P. ∙ Chang, D.K. ∙ Nones, K. ..., Australian Pancreatic Cancer Genome Initiative</div><div><strong>Genomic analyses identify molecular subtypes of pancreatic cancer</strong></div><div><em>Nature.</em> 2016; <strong>531</strong>:47-52</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=P.BaileyD.K.ChangK.NonesA.L.JohnsA.M.PatchM.C.GingrasD.K.MillerA.N.ChristT.J.BruxnerM.C.QuinnAustralian+Pancreatic+Cancer+Genome+InitiativeGenomic+analyses+identify+molecular+subtypes+of+pancreatic+cancerNature53120164752" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref16-2" href="#" role="doc-biblioref" data-xml-rid="bib16" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref16-2" aria-controls="sref16-2">Chan-Seng-Yue et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref16-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">16.</div><div>Chan-Seng-Yue, M. ∙ Kim, J.C. ∙ Wilson, G.W. ...</div><div><strong>Transcription phenotypes of pancreatic cancer are driven by genomic events during tumor evolution</strong></div><div><em>Nat. Genet.</em> 2020; <strong>52</strong>:231-240</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.Chan-Seng-YueJ.C.KimG.W.WilsonK.NgE.F.FigueroaG.M.O%E2%80%99KaneA.A.ConnorR.E.DenrocheR.C.GrantJ.McLeodTranscription+phenotypes+of+pancreatic+cancer+are+driven+by+genomic+events+during+tumor+evolutionNat.+Genet.522020231240" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref17-2" href="#" role="doc-biblioref" data-xml-rid="bib17" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref17-2" aria-controls="sref17-2">Collisson et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref17-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">17.</div><div>Collisson, E.A. ∙ Sadanandam, A. ∙ Olson, P. ...</div><div><strong>Subtypes of pancreatic ductal adenocarcinoma and their differing responses to therapy</strong></div><div><em>Nat. Med.</em> 2011; <strong>17</strong>:500-503</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=E.A.CollissonA.SadanandamP.OlsonW.J.GibbM.TruittS.GuJ.CoocJ.WeinkleG.E.KimL.JakkulaSubtypes+of+pancreatic+ductal+adenocarcinoma+and+their+differing+responses+to+therapyNat.+Med.172011500503" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref50-2" href="#" role="doc-biblioref" data-xml-rid="bib50" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref50-2" aria-controls="sref50-2">Moffitt et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref50-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">50.</div><div>Moffitt, R.A. ∙ Marayati, R. ∙ Flate, E.L. ...</div><div><strong>Virtual microdissection identifies distinct tumor- and stroma-specific subtypes of pancreatic ductal adenocarcinoma</strong></div><div><em>Nat. Genet.</em> 2015; <strong>47</strong>:1168-1178</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=R.A.MoffittR.MarayatiE.L.FlateK.E.VolmarS.G.LoezaK.A.HoadleyN.U.RashidL.A.WilliamsS.C.EatonA.H.ChungVirtual+microdissection+identifies+distinct+tumor-+and+stroma-specific+subtypes+of+pancreatic+ductal+adenocarcinomaNat.+Genet.47201511681178" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Subsets of signatures from each study were highly concordant and separated primarily into classical-like (clade 1) and basal-like (clade 2) groups. Clade 3 signatures were generally lower in expression but tended to associate with clade 1, while clade 4 signatures represented exocrine pancreas expression patterns and tended to be expressed only in primary disease, suggesting that they may be due to contributions from the primary PDAC TME. Relatedly, the documented low malignant cellularity in PDAC may obstruct malignant cell-specific signature identification in bulk RNA-sequencing datasets (<span class="dropBlock reference-citations"><a id="body-ref-sref18-2" href="#" role="doc-biblioref" data-xml-rid="bib18" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref18-2" aria-controls="sref18-2">Collisson et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref18-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">18.</div><div>Collisson, E.A. ∙ Bailey, P. ∙ Chang, D.K. ...</div><div><strong>Molecular subtypes of pancreatic cancer</strong></div><div><em>Nat. Rev. Gastroenterol. Hepatol.</em> 2019; <strong>16</strong>:207-220</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=E.A.CollissonP.BaileyD.K.ChangA.V.BiankinMolecular+subtypes+of+pancreatic+cancerNat.+Rev.+Gastroenterol.+Hepatol.162019207220" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div class="figure-wrap"><figure id="fig1" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.11.017/asset/e7f58a9a-6cf2-4205-80e5-ed5bd5636ad4/main.assets/gr1_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr1" src="/cms/10.1016/j.cell.2021.11.017/asset/92bcff86-ebe0-4bad-8a05-84b74de20e9f/main.assets/gr1.jpg" height="666" width="657" aria-labelledby="fig1-title fig1-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.017/asset/e7f58a9a-6cf2-4205-80e5-ed5bd5636ad4/main.assets/gr1_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig1-title" class="figure__title"><span class="label figure__label">Figure&nbsp;1</span> <span class="figure__title__text">Assessing transcriptional states in patient tumors and cancer models</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig1-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0015" role="paragraph">(A) Precision medicine pipelines assess model fidelity for genetics but typically do not evaluate RNA states.</div><div id="fspara0020" role="paragraph">(B) Alterations in PDAC driver genes across primary resections (TCGA), metastatic biopsies (Panc-Seq), and organoid and cell line (CCLE) models. Grey indicates where genomic data were not available. <i>P</i>-values by Fisher’s exact test.</div><div id="fspara0025" role="paragraph">(C) Comparison of PDAC expression signatures from bulk RNA-sequencing in primary and metastatic tumors, cell lines, and organoid models in (B). Rows are clustered, columns are sorted by average basal-classical score difference. <i>P</i>-values indicate differences between patient tumors, cell lines, and organoids by ANOVA.</div><div id="fspara0030" role="paragraph">(D) Schematic of contributors to RNA state that may lead to differences between <i>in&nbsp;vivo</i> and <i>ex&nbsp;vivo</i> expression patterns.</div><div id="fspara0035" role="paragraph">(E) Metastatic patient samples were collected via core needle biopsies and dissociated. Biopsy cells were allocated for scRNA-seq, and patient-matched organoids were developed with downstream serial scRNA-seq sampling.</div><div id="fspara0040" role="paragraph">(F and G) <i>t</i>-distributed stochastic neighbor embedding (<i>t</i>-SNE) for biopsy (F) and matched patient-derived organoid cells (G).</div><div id="fspara0045" role="paragraph">See also <a id="crosref2340" href="#figs1">Figure&nbsp;S1</a>; <a id="crosref2345" href="#mmc1">Tables S1</a> and <a id="crosref2350" href="#mmc2">S2</a>.</div></div></div></div></figcaption></figure></div><div id="p0075" role="paragraph">In contrast to the mutational data, we observed significant differences for all RNA signatures when comparing tumors to model types (<a id="crosref0940" href="#fig1">Figure&nbsp;1</a>C). Whereas primary and metastatic samples include tumors with both classical-like (clade 1) and basal-like (clade 2) signatures, cell lines exhibited predominantly basal-like (clade 2) subtypes while organoids were nearly entirely classical-like (clade 1) with partially overlapping expression of some clade 3 signatures (<a id="crosref0945" href="#fig1">Figures 1</a>C and <a id="crosref0950" href="#figs1">S1</a>A). These observations demonstrate that despite preservation of genomic alterations, neither PDAC cell lines nor organoids represent the full repertoire of expression subtypes seen in patient cohorts.</div><div class="figure-wrap"><figure id="figs1" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.11.017/asset/575614e1-82a5-4936-b0bb-cfd858f0810e/main.assets/figs1_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs1" src="/cms/10.1016/j.cell.2021.11.017/asset/daaa3cd2-7403-4552-903f-3868a004318c/main.assets/figs1.jpg" height="921" width="728" aria-labelledby="figs1-title figs1-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.017/asset/575614e1-82a5-4936-b0bb-cfd858f0810e/main.assets/figs1_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs1-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S1</span> <span class="figure__title__text">Clinical and genomic features, quality metrics, unsupervised cell type identification, and malignant cell confirmation across the biopsy cohort, related to <a id="crosref2435" href="#fig1">Figure&nbsp;1</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs1-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0320" role="paragraph">(A) Distribution of PDAC patient tumor and cell model phenotypes along the basal-classical spectrum for bulk RNA-seq samples in <a id="crosref2440" href="#fig1">Figure&nbsp;1</a>C. <i>P</i>-value for group differences was calculated by ANOVA.</div><div id="fspara0325" role="paragraph">(B) Clinical and molecular features for all patients included in the dataset (R<sub>x</sub>&nbsp;= Therapy; Other&nbsp;= Adrenal (PANFR0637), Omentum (PANFR0635, PANFR0598), Peritoneum (PANFR0588); Org. at P2&nbsp;= Organoid measured at passage 2). Mutations were determined by bulk targeted DNA-seq (Red, Altered; White, wild type; Grey, Data not available). Number of single cells captured per biopsy and their malignant and non-malignant fraction is visualized at the right. The scRNA-seq biopsy for sample PANFR0604 did not contain any malignant cells.</div><div id="fspara0330" role="paragraph">(C) Distribution of unique molecules and genes captured in quality cells per biopsy, median values are indicated for each metric (dotted line) and violin plots are colored by patient (top, Log<sub>10</sub>(UMIs); bottom, number of genes).</div><div id="fspara0335" role="paragraph">(D) Distribution of fraction mitochondrial reads across the entire trimmed biopsy dataset (n&nbsp;= 23,042 cells). Red dotted line denotes the median.</div><div id="fspara0340" role="paragraph">(E) <i>t-</i>SNE visualization of the entire single-cell biopsy dataset colored by the identified SNN clusters (inset numbers).</div><div id="fspara0345" role="paragraph">(F) Distribution of single cells captured per biopsy across the identified and putative malignant and non-malignant SNN clusters.</div><div id="fspara0350" role="paragraph">(G) Heatmaps represent select scRNA-seq-derived copy number profiles where expression across the transcriptome is organized by chromosome (columns) for each single putative malignant cell (rows) from a given biopsy. Top bar indicates reference bulk targeted DNA-seq for the same patient.</div><div id="fspara0355" role="paragraph">(H) CNV correlation (averaged top 5% of altered cells per biopsy) versus CNV score (mean square of modified expression) for each single putative malignant (colored points) and reference normal (empty black circles) cell within a given biopsy. One sample, PANFR0604, did not contain any malignant cells.</div><div id="fspara0360" role="paragraph">(I) Overview of cell-typing for all cells in the biopsy dataset. Cells are ordered by SNN cluster and separated by cell types. Top heatmap represents expression levels for a subset of select markers (n&nbsp;= 73 genes) used to identify cell types. Color bar indicates cell types and binarized cell cycle phenotypes are labeled (black, cycling; white, not). CNV scores (mean square of alterations per cell) used to parse malignant from non-malignant are shown using T/NK, endothelial, fibroblasts, and hepatocytes as reference; gray boxes denote normal cell types where we did not compute reference CNV scores. Bottom panel shows biopsy of origin for each cell. The data are split by non-malignant (n&nbsp;= 15,302) and malignant (7,740) identity.</div><div id="fspara0365" role="paragraph">(J) <i>t-</i>SNE visualization as in <b>S1E</b> but colored by cell types identified. Endo, Endothelial; Mes, Mesenchymal; B, B cell; Hep, Hepatocyte; DC, Dendritic cell; pDC, Plasmacytoid dendritic cell; Mac, Macrophage; T, T&nbsp;cell; NK, Natural killer cell.</div><div id="fspara0370" role="paragraph">(K) Fraction of each cell type contributed by each biopsy sample (color fill, patient ID as in S1B), cell type totals are noted at the top of each bar.</div></div></div></div></figcaption></figure></div></section><section id="sec-2-2"><h3>Single-cell profiling of metastatic PDAC and matched organoid models</h3><div id="p0080" role="paragraph">These findings highlight a critical need for new approaches to identify the determinants of cancer cell state both <i>in&nbsp;vivo</i> and in model systems. Cell state is a complex attribute since it integrates cell-intrinsic as well as TME-dependent features; therefore, multiple mechanisms (e.g., clonal selection and/or plasticity) may contribute to the divergence between <i>in&nbsp;vivo</i> and <i>ex&nbsp;vivo</i> expression patterns (<a id="crosref0955" href="#fig1">Figure&nbsp;1</a>D). We hypothesized that a dataset at single-cell resolution allowing for matched comparisons of <i>in&nbsp;vivo</i> and <i>ex&nbsp;vivo</i> attributes would enable a better understanding of cell state drivers, stability, and functional relevance. To this end, we established a pipeline to generate matched scRNA-seq profiles and organoid models using core needle biopsies from patients with metastatic PDAC (n&nbsp;= 23) (<a id="crosref0960" href="#fig1">Figures 1</a>E and <a id="crosref0965" href="#figs1">S1</a>B; <a id="crosref0970" href="#mmc1">Table S1</a>). Most samples were obtained from metastatic lesions residing in the liver (19/23), and the majority (21/23) were analyzed by targeted DNA-sequencing, yielding the expected mutational patterns (<a id="crosref0975" href="#figs1">Figure&nbsp;S1</a>B).</div><div id="p0085" role="paragraph">Our pipeline generated approximately 1,000 high-quality single cells per biopsy and successful early-passage organoid cultures from 70% of patient samples (16/23 samples reaching at least passage 2) (<a id="crosref0980" href="#fig1">Figures 1</a>E, 1F, and <a id="crosref0985" href="#figs1">S1</a>B–S1D). Consistent with other studies, we observed patient-specific and admixed clusters of single cells suggesting the presence of both malignant and non-malignant cells in each biopsy (<a id="crosref0990" href="#figs1">Figures S1</a>E and S1F; <a id="crosref0995" href="#sec-4">STAR Methods</a>) (<span class="dropBlock reference-citations"><a id="body-ref-sref43-2" href="#" role="doc-biblioref" data-xml-rid="bib43" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref43-2" aria-controls="sref43-2">Kim et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref43-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">43.</div><div>Kim, C. ∙ Gao, R. ∙ Sei, E. ...</div><div><strong>Chemoresistance Evolution in Triple-Negative Breast Cancer Delineated by Single-Cell Sequencing</strong></div><div><em>Cell.</em> 2018; <strong>173</strong>:879-893 e813</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=C.KimR.GaoE.SeiR.BrandtJ.HartmanT.HatschekN.CrosettoT.FoukakisN.E.NavinChemoresistance+Evolution+in+Triple-Negative+Breast+Cancer+Delineated+by+Single-Cell+SequencingCell1732018879893+e813" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref60-2" href="#" role="doc-biblioref" data-xml-rid="bib60" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref60-2" aria-controls="sref60-2">Puram et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref60-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">60.</div><div>Puram, S.V. ∙ Tirosh, I. ∙ Parikh, A.S. ...</div><div><strong>Single-Cell Transcriptomic Analysis of Primary and Metastatic Tumor Ecosystems in Head and Neck Cancer</strong></div><div><em>Cell.</em> 2017; <strong>171</strong>:1611-1624 e1624</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=S.V.PuramI.TiroshA.S.ParikhA.P.PatelK.YizhakS.GillespieC.RodmanC.L.LuoE.A.MrozK.S.EmerickSingle-Cell+Transcriptomic+Analysis+of+Primary+and+Metastatic+Tumor+Ecosystems+in+Head+and+Neck+CancerCell171201716111624+e1624" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref62-2" href="#" role="doc-biblioref" data-xml-rid="bib62" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref62-2" aria-controls="sref62-2">Sade-Feldman et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref62-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">62.</div><div>Sade-Feldman, M. ∙ Yizhak, K. ∙ Bjorgaard, S.L. ...</div><div><strong>Defining T Cell States Associated with Response to Checkpoint Immunotherapy in Melanoma</strong></div><div><em>Cell.</em> 2019; <strong>176</strong>:404</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.Sade-FeldmanK.YizhakS.L.BjorgaardJ.P.RayC.G.de+BoerR.W.JenkinsD.J.LiebJ.H.ChenD.T.FrederickM.Barzily-RokniDefining+T+Cell+States+Associated+with+Response+to+Checkpoint+Immunotherapy+in+MelanomaCell1762019404" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref69-2" href="#" role="doc-biblioref" data-xml-rid="bib69" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref69-2" aria-controls="sref69-2">Tirosh et&nbsp;al., 2016a</a><div class="dropBlock__holder" data-db-target-of="sref69-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">69.</div><div>Tirosh, I. ∙ Izar, B. ∙ Prakadan, S.M. ...</div><div><strong>Dissecting the multicellular ecosystem of metastatic melanoma by single-cell RNA-seq</strong></div><div><em>Science.</em> 2016; <strong>352</strong>:189-196</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=I.TiroshB.IzarS.M.PrakadanM.H.Wadsworth2ndD.TreacyJ.J.TrombettaA.RotemC.RodmanC.LianG.MurphyDissecting+the+multicellular+ecosystem+of+metastatic+melanoma+by+single-cell+RNA-seqScience3522016189196" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Inferred copy number variation (CNV) alteration scores separated putative cancerous and non-cancerous cells in each biopsy and demonstrated high concordance with reference targeted DNA-seq (<a id="crosref1020" href="#figs1">Figures S1</a>G and S1H) (<span class="dropBlock reference-citations"><a id="body-ref-sref57-2" href="#" role="doc-biblioref" data-xml-rid="bib57" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref57-2" aria-controls="sref57-2">Patel et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref57-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">57.</div><div>Patel, A.P. ∙ Tirosh, I. ∙ Trombetta, J.J. ...</div><div><strong>Single-cell RNA-seq highlights intratumoral heterogeneity in primary glioblastoma</strong></div><div><em>Science.</em> 2014; <strong>344</strong>:1396-1401</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.P.PatelI.TiroshJ.J.TrombettaA.K.ShalekS.M.GillespieH.WakimotoD.P.CahillB.V.NahedW.T.CurryR.L.MartuzaSingle-cell+RNA-seq+highlights+intratumoral+heterogeneity+in+primary+glioblastomaScience344201413961401" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref70-2" href="#" role="doc-biblioref" data-xml-rid="bib70" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref70-2" aria-controls="sref70-2">Tirosh et&nbsp;al., 2016b</a><div class="dropBlock__holder" data-db-target-of="sref70-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">70.</div><div>Tirosh, I. ∙ Venteicher, A.S. ∙ Hebert, C. ...</div><div><strong>Single-cell RNA-seq supports a developmental hierarchy in human oligodendroglioma</strong></div><div><em>Nature.</em> 2016; <strong>539</strong>:309-313</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=I.TiroshA.S.VenteicherC.HebertL.E.EscalanteA.P.PatelK.YizhakJ.M.FisherC.RodmanC.MountM.G.FilbinSingle-cell+RNA-seq+supports+a+developmental+hierarchy+in+human+oligodendrogliomaNature5392016309313" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). CNV analysis and manual inspection of expression patterns for known markers across single cells supported the identification of malignant cells as well as 11 unique non-malignant cell types (<a id="crosref1035" href="#figs1">Figures S1</a>I–S1K; <a id="crosref1040" href="#mmc2">Table S2</a>). Thus, despite the documented low malignant cellularity in PDAC (<span class="dropBlock reference-citations"><a id="body-ref-sref2-3" href="#" role="doc-biblioref" data-xml-rid="bib2" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref2-3" aria-controls="sref2-3">Aguirre et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref2-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">2.</div><div>Aguirre, A.J. ∙ Nowak, J.A. ∙ Camarda, N.D. ...</div><div><strong>Real-time Genomic Characterization of Advanced Pancreatic Cancer to Enable Precision Medicine</strong></div><div><em>Cancer Discov.</em> 2018; <strong>8</strong>:1096-1111</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.J.AguirreJ.A.NowakN.D.CamardaR.A.MoffittA.A.GhazaniM.Hazar-RethinamS.RaghavanJ.KimL.K.BraisD.RagonReal-time+Genomic+Characterization+of+Advanced+Pancreatic+Cancer+to+Enable+Precision+MedicineCancer+Discov.8201810961111" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref15-3" href="#" role="doc-biblioref" data-xml-rid="bib15" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref15-3" aria-controls="sref15-3">Cancer Genome Atlas Research Network, 2017</a><div class="dropBlock__holder" data-db-target-of="sref15-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">15.</div><div>Cancer Genome Atlas Research Network</div><div><strong>Integrated Genomic Characterization of Pancreatic Ductal Adenocarcinoma</strong></div><div><em>Cancer Cell.</em> 2017; <strong>32</strong>:185-203.e13</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=Cancer+Genome+Atlas+Research+NetworkIntegrated+Genomic+Characterization+of+Pancreatic+Ductal+AdenocarcinomaCancer+Cell322017185203.e13" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref16-3" href="#" role="doc-biblioref" data-xml-rid="bib16" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref16-3" aria-controls="sref16-3">Chan-Seng-Yue et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref16-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">16.</div><div>Chan-Seng-Yue, M. ∙ Kim, J.C. ∙ Wilson, G.W. ...</div><div><strong>Transcription phenotypes of pancreatic cancer are driven by genomic events during tumor evolution</strong></div><div><em>Nat. Genet.</em> 2020; <strong>52</strong>:231-240</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.Chan-Seng-YueJ.C.KimG.W.WilsonK.NgE.F.FigueroaG.M.O%E2%80%99KaneA.A.ConnorR.E.DenrocheR.C.GrantJ.McLeodTranscription+phenotypes+of+pancreatic+cancer+are+driven+by+genomic+events+during+tumor+evolutionNat.+Genet.522020231240" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), we established a robust pipeline that retrieved high-quality malignant (n&nbsp;= 7,740) and non-malignant (n&nbsp;= 15,302) single-cell transcriptomes from metastatic PDAC needle biopsies, as well as those of matched organoids (n&nbsp;= 24,995) (<a id="crosref1060" href="#fig1">Figures 1</a>E–1G).</div></section><section id="sec-2-3"><h3>Single-cell resolution identifies an IC cancer cell state in metastatic PDAC</h3><div id="p0090" role="paragraph">After separating the CNV-confirmed PDAC cells from non-malignant cells (excluding one neuroendocrine sample; <a id="crosref1065" href="#figs2">Figure&nbsp;S2</a>A; see <a id="crosref1070" href="#sec-4">STAR Methods</a>), we first interrogated whether previously described RNA subtypes (<a id="crosref1075" href="#fig1">Figure&nbsp;1</a>C) are represented in single metastatic PDAC cells. Subsets of metastatic cells scored highly for either classical-like or basal-like signatures derived from independent bulk studies indicating the general relevance of these expression programs to <i>in&nbsp;vivo</i> PDAC biology (<a id="crosref1080" href="#fig2">Figure&nbsp;2</a>A). The remaining literature-derived signatures had low expression in these metastatic cells, suggesting either specificity for primary disease or influence from non-malignant expression in bulk RNA profiles. Interestingly, a large subset of cells showed low expression for all previously proposed signatures, highlighting a gap in our understanding of PDAC expression states (<a id="crosref1085" href="#fig2">Figure&nbsp;2</a>A; “Low for bulk subtypes”).</div><div class="figure-wrap"><figure id="fig2" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.11.017/asset/6d000041-a3a0-48c1-ba5f-0f8cc0c50bb4/main.assets/gr2_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr2" src="/cms/10.1016/j.cell.2021.11.017/asset/9f44ae90-e7b3-4fe9-a3c5-2a09a5f21fcc/main.assets/gr2.jpg" height="573" width="657" aria-labelledby="fig2-title fig2-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.017/asset/6d000041-a3a0-48c1-ba5f-0f8cc0c50bb4/main.assets/gr2_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig2-title" class="figure__title"><span class="label figure__label">Figure&nbsp;2</span> <span class="figure__title__text">An intermediate co-expressor state bridges basal and classical phenotypes</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig2-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0055" role="paragraph">(A) Signature scores (rows) for bulk derived expression subtypes in malignant cells (columns).</div><div id="fspara0060" role="paragraph">(B) Heatmaps depict the expression of scBasal and scClassical expression programs and highlight a co-expressing cell population.</div><div id="fspara0065" role="paragraph">(C) Variation in EMT and IFN response signature expression within malignant cells that have scBasal expression.</div><div id="fspara0070" role="paragraph">(D) The intermediate co-expressor (“IC”) expression program is enriched in co-expressing cells. Enrichment adjusted <i>P</i>-values (hypergeometric test) for EMT, <i>KRAS</i>, and <i>AKT</i> gene sets are indicated at right in (B) and (D).</div><div id="fspara0075" role="paragraph">(E) Gene set enrichment analysis for the 115 genes specific to the intermediate co-expressor program.</div><div id="fspara0080" role="paragraph">(F) Malignant cell state diagram for PDAC. ScBasal-scClassical commitment score (x axis) and IC score (y axis).</div><div id="fspara0085" role="paragraph">(G) Frequency of co-expressing cells is related to increased mixing of scBasal and scClassical cell populations. Log ratio of % scBasal and scClassical cells in each sample (x axis; dotted line at 0 indicates equal percentages of scBasal and scClassical cells) versus their % IC cells (y axis).</div><div id="fspara0090" role="paragraph">(H) Multiplex immunofluorescence analysis identifies co-expressing cells in matched metastatic samples. White box indicates region for co-expression insets at bottom. Scale bar represents 10&nbsp;μm.</div><div id="fspara0095" role="paragraph">See also <a id="crosref2540" href="#figs2">Figures S2</a>, <a id="crosref2545" href="#figs3">S3</a>, and <a id="crosref2550" href="#figs4">S4</a>; <a id="crosref2355" href="#mmc3">Table S3</a>.</div></div></div></div></figcaption></figure></div><div class="figure-wrap"><figure id="figs2" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.11.017/asset/03b1f209-d1c0-4b65-b75c-8382c1a39984/main.assets/figs2_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs2" src="/cms/10.1016/j.cell.2021.11.017/asset/c5bc289b-af8c-4c9f-8ddc-63d45cd4a98c/main.assets/figs2.jpg" height="799" width="657" aria-labelledby="figs2-title figs2-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.017/asset/03b1f209-d1c0-4b65-b75c-8382c1a39984/main.assets/figs2_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs2-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S2</span> <span class="figure__title__text">Identifying and contextualizing single-cell scBasal and scClassical associated biology, related to <a id="crosref2445" href="#fig2">Figure&nbsp;2</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs2-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0380" role="paragraph">(A) Principal component analysis (PCA) and scatterplot for PC1 and PC2 across all malignant cells (n&nbsp;= 7,740, colored by patient ID as in Figure&nbsp;S1B) separates PANFR0580’s malignant cells (n&nbsp;= 662) from the rest. Heatmap for genes with the strongest negative loading on PC1 (n&nbsp;= 30) denotes a neuroendocrine identity (<i>TTR, CHGB</i>). This tumor was later classified by histology as a pancreatic neuroendocrine tumor (PanNET).</div><div id="fspara0385" role="paragraph">(B) Principal component analysis (PCA) and scatterplot for PC1 and PC2 across all PDAC malignant cells (n&nbsp;= 7,078, colored by patient ID as in Figure&nbsp;S1B), excluding PANFR0580’s PanNET malignant cells. Heatmap for genes with the strongest positive and negative loadings on PC2 (n&nbsp;= 30 each) shows overlap with genes from the Moffitt basal and classical signatures.</div><div id="fspara0390" role="paragraph">(C) Principal component (PC) elbow plot (top) showing the standard deviation for the first 20 components calculated over the verified PDAC malignant cell variable genes. Line is drawn at the putative “elbow” (black versus gray points) as inclusion of additional PCs described overlapping information or quality metrics. Cross-correlational analysis (bottom) for each single-cell’s embeddings across first 9 PCs (black points) and scores for literature curated gene sets describing EMT, classical and basal, and cell cycle phenotypes.</div><div id="fspara0395" role="paragraph">(D) Pairwise correlation of genes significantly enriched in scBasal or scClassical expression states within single-cell cohort. Left bar indicates the subtype association of each gene (orange, scBasal-enriched; blue, scClassical-enriched).</div><div id="fspara0400" role="paragraph">(E) Tied dot plots depict the correlation coefficient for each gene (points) from select literature-derived gene sets, indicated at the top of each plot, to either scBasal or scClassical cell states. Dotted lines represent significance threshold (3 SD above the mean of shuffled data), points and lines are colored if that gene passes the threshold.</div><div id="fspara0405" role="paragraph">(F) GSEA pathway enrichments for the top 100 genes correlated to either scBasal or scClassical expression scores.</div></div></div></div></figcaption></figure></div><div id="p0095" role="paragraph">Given this large subset of cells that score weakly for previously identified signatures, we sought to understand the spectrum of <i>in&nbsp;vivo</i> expression states through an unbiased analysis of our single-cell dataset. Principal component analysis (PCA) on all malignant cells revealed that genes enriched for signatures of epithelial/mesenchymal transition (EMT) (PC1) (<span class="dropBlock reference-citations"><a id="body-ref-sref31-1" href="#" role="doc-biblioref" data-xml-rid="bib31" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref31-1" aria-controls="sref31-1">Gröger et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref31-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">31.</div><div>Gröger, C.J. ∙ Grubinger, M. ∙ Waldhör, T. ...</div><div><strong>Meta-analysis of gene expression signatures defining the epithelial to mesenchymal transition during cancer progression</strong></div><div><em>PLoS ONE.</em> 2012; <strong>7</strong>:e51136</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=C.J.Gr%C3%B6gerM.GrubingerT.Waldh%C3%B6rK.VierlingerW.MikulitsMeta-analysis+of+gene+expression+signatures+defining+the+epithelial+to+mesenchymal+transition+during+cancer+progressionPLoS+ONE72012e51136" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), basal-like and classical PDAC states (PC1 and PC2), and cell cycle (PC3 and PC8) (<span class="dropBlock reference-citations"><a id="body-ref-sref69-3" href="#" role="doc-biblioref" data-xml-rid="bib69" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref69-3" aria-controls="sref69-3">Tirosh et&nbsp;al., 2016a</a><div class="dropBlock__holder" data-db-target-of="sref69-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">69.</div><div>Tirosh, I. ∙ Izar, B. ∙ Prakadan, S.M. ...</div><div><strong>Dissecting the multicellular ecosystem of metastatic melanoma by single-cell RNA-seq</strong></div><div><em>Science.</em> 2016; <strong>352</strong>:189-196</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=I.TiroshB.IzarS.M.PrakadanM.H.Wadsworth2ndD.TreacyJ.J.TrombettaA.RotemC.RodmanC.LianG.MurphyDissecting+the+multicellular+ecosystem+of+metastatic+melanoma+by+single-cell+RNA-seqScience3522016189196" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) drove the major axes of variation in the dataset (<a id="crosref1100" href="#figs2">Figures S2</a>B and S2C). The PC1-PC2 score difference was correlated with the basal-classical signature axis previously identified in bulk studies (r&nbsp;= 0.8). We identified 1,909 genes significantly correlated with either end of the basal or classical-enriched continuum within our single-cell cohort (<a id="crosref1105" href="#figs2">Figure&nbsp;S2</a>D; <a id="crosref1110" href="#mmc3">Table S3</a>; <a id="crosref1115" href="#sec-4">STAR Methods</a>). Inspection of these genes revealed that more basal-like cells are associated with transforming growth factor beta (TGF-β) signaling, WNT signaling, EMT, and cell cycle progression. Epithelial and pancreatic lineage programs are enriched in the cells with more classical-like attributes (<a id="crosref1120" href="#figs2">Figures S2</a>E and S2F). We term the signatures derived from our single-cell cohort as single-cell basal (scBasal) and single-cell classical (scClassical) (<a id="crosref1125" href="#fig2">Figure&nbsp;2</a>B).</div><div id="p0100" role="paragraph">Single-cell analysis enabled a dissection of expression states that are confounded by contaminating non-malignant cells in bulk measurements. For example, in bulk RNA-seq studies (<span class="dropBlock reference-citations"><a id="body-ref-sref2-4" href="#" role="doc-biblioref" data-xml-rid="bib2" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref2-4" aria-controls="sref2-4">Aguirre et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref2-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">2.</div><div>Aguirre, A.J. ∙ Nowak, J.A. ∙ Camarda, N.D. ...</div><div><strong>Real-time Genomic Characterization of Advanced Pancreatic Cancer to Enable Precision Medicine</strong></div><div><em>Cancer Discov.</em> 2018; <strong>8</strong>:1096-1111</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.J.AguirreJ.A.NowakN.D.CamardaR.A.MoffittA.A.GhazaniM.Hazar-RethinamS.RaghavanJ.KimL.K.BraisD.RagonReal-time+Genomic+Characterization+of+Advanced+Pancreatic+Cancer+to+Enable+Precision+MedicineCancer+Discov.8201810961111" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref15-4" href="#" role="doc-biblioref" data-xml-rid="bib15" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref15-4" aria-controls="sref15-4">Cancer Genome Atlas Research Network, 2017</a><div class="dropBlock__holder" data-db-target-of="sref15-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">15.</div><div>Cancer Genome Atlas Research Network</div><div><strong>Integrated Genomic Characterization of Pancreatic Ductal Adenocarcinoma</strong></div><div><em>Cancer Cell.</em> 2017; <strong>32</strong>:185-203.e13</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=Cancer+Genome+Atlas+Research+NetworkIntegrated+Genomic+Characterization+of+Pancreatic+Ductal+AdenocarcinomaCancer+Cell322017185203.e13" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), EMT programs are strongly correlated with markers of fibroblasts (r&nbsp;= 0.9), inversely related to purity metrics (r&nbsp;= −0.5), and poorly correlated with basal-like expression (r&nbsp;= −0.08). At single-cell resolution, we observed a subset of cells that express both scBasal and EMT programs, while other scBasal cells had low expression of EMT programs (<a id="crosref1140" href="#fig2">Figure&nbsp;2</a>C, left). Similarly, the origin of cytokine response signatures can be difficult to interpret from bulk studies, as interferon (IFN) response gene signatures are positively correlated with markers of several cell types including macrophages (r&nbsp;= 0.6) and T&nbsp;cells (r&nbsp;= 0.4), negatively associated with purity metrics (r&nbsp;= −0.4), and not associated with either basal-like or classical scores (r&nbsp;= 0.01 and −0.04, respectively). In single cells, we observe clear patterns of association between IFN response and a subset of cells with intermediate scBasal expression (<a id="crosref1145" href="#fig2">Figure&nbsp;2</a>C, right). These findings suggest not all basal-like cells have the same underlying attributes and highlight the importance of utilizing scRNA-seq to understand how malignant cells sense and respond to their local TME.</div><div id="p0105" role="paragraph">Although some studies suggest that basal and classical programs exist only in discrete cell populations (<span class="dropBlock reference-citations"><a id="body-ref-sref16-4" href="#" role="doc-biblioref" data-xml-rid="bib16" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref16-4" aria-controls="sref16-4">Chan-Seng-Yue et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref16-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">16.</div><div>Chan-Seng-Yue, M. ∙ Kim, J.C. ∙ Wilson, G.W. ...</div><div><strong>Transcription phenotypes of pancreatic cancer are driven by genomic events during tumor evolution</strong></div><div><em>Nat. Genet.</em> 2020; <strong>52</strong>:231-240</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.Chan-Seng-YueJ.C.KimG.W.WilsonK.NgE.F.FigueroaG.M.O%E2%80%99KaneA.A.ConnorR.E.DenrocheR.C.GrantJ.McLeodTranscription+phenotypes+of+pancreatic+cancer+are+driven+by+genomic+events+during+tumor+evolutionNat.+Genet.522020231240" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), we observed that the scBasal and scClassical programs were not mutually exclusive. Rather, we identified cells that are intermediate for scBasal and scClassical gene expression signatures, co-expressed features of both programs to varying degrees, and were poorly described by previously identified bulk RNA subtypes (<a id="crosref1155" href="#fig2">Figures 2</a>B [“Co-expressing cells”], <a id="crosref1160" href="#figs3">S3</a>A, and S3B). We identified 115 genes whose expression was correlated with scBasal-scClassical co-expression and termed this gene set the IC state (<a id="crosref1165" href="#fig2">Figures 2</a>D, <a id="crosref1170" href="#figs3">S3</a>C, and S3D; <a id="crosref1175" href="#mmc3">Table S3</a>; <a id="crosref1180" href="#sec-4">STAR Methods</a>).</div><div class="figure-wrap"><figure id="figs3" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.11.017/asset/c86b72ae-26c7-4e59-8bb0-5f1375419d3e/main.assets/figs3_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs3" src="/cms/10.1016/j.cell.2021.11.017/asset/21b9c88f-5212-44bd-afc1-85eabf7e0270/main.assets/figs3.jpg" height="839" width="657" aria-labelledby="figs3-title figs3-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.017/asset/c86b72ae-26c7-4e59-8bb0-5f1375419d3e/main.assets/figs3_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs3-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S3</span> <span class="figure__title__text">Cells with co-expression of scBasal and scClassical states express a distinct intermediate co-expressor gene program, related to <a id="crosref2450" href="#fig2">Figure&nbsp;2</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs3-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0415" role="paragraph">(A) Expression of scBasal and scClassical gene programs, with cells ordered by their scBasal-scClassical score difference. Quality metrics, EMT scores and the binarized cell cycle program are shown for each single cell below the heatmap. <i>P</i>-value for binarized cycling group differences was calculated using Fisher’s Exact test. <i>P</i>-value for EMT score was calculated by Kruskal-Wallis test with multiple hypothesis correction.</div><div id="fspara0420" role="paragraph">(B) Difference between PC1 and PC2 (top) and scClassical–scBasal score difference (bottom) are shown. Cells with equal scBasal and scClassical expression are associated with intermediate PC scores and cells are ordered as in S3A. <i>P</i>-values for group differences were calculated by Kruskal-Wallis test with multiple hypothesis correction.</div><div id="fspara0425" role="paragraph">(C) Euclidean distance for each cell to co-expression (y&nbsp;= x) of scBasal (x) and scClassical (y) expression scores. Bottom track indicates the score derived from the genes specific to the intermediate co-expressor state shown in S3D. <i>P</i>-values for group differences were calculated by Kruskal-Wallis test with multiple hypothesis correction.</div><div id="fspara0430" role="paragraph">(D) Gene correlation to either scBasal or scClassical score (x axis) or the corrected intermediate co-expressor correlation (Euclidean distance in S3C). Green highlighted genes have corrected intermediate co-expressor correlation &gt; 0.1 (p&nbsp;&lt; <i>0.00001</i> above shuffled).</div><div id="fspara0435" role="paragraph">(E) Averaged expression of all three malignant programs in normal pancreatic progenitor niche subsets (Acinar (Pro Ac.), Ductal (Pro Duct.), or Undifferentiated (Undiff.) and mesenchymal (Mes.) cells defined in Qadir et&nbsp;al., 2020. <i>P</i>-values for each set of genes are computed by Kruskal-Wallis test with multiple hypothesis correction.</div><div id="fspara0440" role="paragraph">(F) Cross-correlation between new and previously proposed expression signatures in our PDAC single cells. Average expression for each signature (rows) is shown at the right for cells in the malignant subtypes from our cohort and the normal pancreatic progenitor cells from Qadir et&nbsp;al., 2020. White dot indicates the subset with the highest average significant expression for each signature (Kruskal-Wallis test); no white dot indicates no significant expression.</div><div id="fspara0445" role="paragraph">(G) Pairwise correlation for biopsies with malignant cells (n&nbsp;= 22). Data are correlation coefficients for the average expression of all signature genes in the malignant cells from a given biopsy. Clade identities are at left with the one PanNET tumor (PANFR0580) included for comparison.</div><div id="fspara0450" role="paragraph">(H) Average expression for the 184 genes used for clustering in S3G. Clade identity colors match text color in S3G, and individual samples (columns) are ordered as in S3G with sample ID numbers provided below.</div><div id="fspara0455" role="paragraph">(I) Scores for the expression of genes in S3H (gray scale heat) across the 4 main cell types found in the pancreatic progenitor niche (<span class="dropBlock reference-citations"><a id="body-ref-sref61-1" href="#" role="doc-biblioref" data-xml-rid="bib61" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref61-1" aria-controls="sref61-1">Qadir et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref61-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">61.</div><div>Qadir, M.M.F. ∙ Álvarez-Cubela, S. ∙ Klein, D. ...</div><div><strong>Single-cell resolution analysis of the human pancreatic ductal progenitor cell niche</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2020; <strong>117</strong>:10876-10887</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.M.F.QadirS.%C3%81lvarez-CubelaD.KleinJ.van+DijkR.Mu%C3%B1iz-AnquelaY.B.Moreno-Hern%C3%A1ndezG.LanzoniS.SadiqB.Navarro-RubioM.T.Garc%C3%ADaSingle-cell+resolution+analysis+of+the+human+pancreatic+ductal+progenitor+cell+nicheProc.+Natl.+Acad.+Sci.+USA11720201087610887" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). White dot indicates the normal subset with the highest average expression for each malignant program (Kruskal-Wallis test), none of the normal subsets significantly express the neuroendocrine gene signature.</div><div id="fspara0460" role="paragraph">(J) Frequencies of scBasal, scClassical, and IC states in individual tumors demonstrates intratumoral transcriptional heterogeneity.</div></div></div></div></figcaption></figure></div><div id="p0110" role="paragraph">The IC state is enriched for developmental, RAS signaling, and inflammation/stress response gene sets (<a id="crosref1185" href="#fig2">Figure&nbsp;2</a>E). Signatures of RAS signaling are enriched in the IC state even compared with scBasal and scClassical programs, and, by contrast, scClassical states are enriched for AKT-associated gene sets (<a id="crosref1190" href="#fig2">Figures 2</a>B, 2D, 2E, and <a id="crosref1195" href="#figs2">S2</a>F). We also assessed whether the IC state overlapped with any phenotypes recently reported in a study of normal pancreas progenitors (<span class="dropBlock reference-citations"><a id="body-ref-sref61-2" href="#" role="doc-biblioref" data-xml-rid="bib61" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref61-2" aria-controls="sref61-2">Qadir et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref61-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">61.</div><div>Qadir, M.M.F. ∙ Álvarez-Cubela, S. ∙ Klein, D. ...</div><div><strong>Single-cell resolution analysis of the human pancreatic ductal progenitor cell niche</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2020; <strong>117</strong>:10876-10887</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.M.F.QadirS.%C3%81lvarez-CubelaD.KleinJ.van+DijkR.Mu%C3%B1iz-AnquelaY.B.Moreno-Hern%C3%A1ndezG.LanzoniS.SadiqB.Navarro-RubioM.T.Garc%C3%ADaSingle-cell+resolution+analysis+of+the+human+pancreatic+ductal+progenitor+cell+nicheProc.+Natl.+Acad.+Sci.+USA11720201087610887" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). We found that both scBasal and scClassical gene expression signatures are expressed by pro-ductal progenitor cells, while the IC gene expression program is enriched in an undifferentiated, stress-responsive progenitor population (<a id="crosref1205" href="#figs3">Figure&nbsp;S3</a>E) (<span class="dropBlock reference-citations"><a id="body-ref-sref61-3" href="#" role="doc-biblioref" data-xml-rid="bib61" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref61-3" aria-controls="sref61-3">Qadir et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref61-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">61.</div><div>Qadir, M.M.F. ∙ Álvarez-Cubela, S. ∙ Klein, D. ...</div><div><strong>Single-cell resolution analysis of the human pancreatic ductal progenitor cell niche</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2020; <strong>117</strong>:10876-10887</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.M.F.QadirS.%C3%81lvarez-CubelaD.KleinJ.van+DijkR.Mu%C3%B1iz-AnquelaY.B.Moreno-Hern%C3%A1ndezG.LanzoniS.SadiqB.Navarro-RubioM.T.Garc%C3%ADaSingle-cell+resolution+analysis+of+the+human+pancreatic+ductal+progenitor+cell+nicheProc.+Natl.+Acad.+Sci.+USA11720201087610887" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Although the IC state may not represent a distinct step along a developmental trajectory, it may represent a similar stress-induced transition state in a cancer context.</div><div id="p0115" role="paragraph">We next assessed how this new IC state relates to previously proposed bulk signatures to clarify potential inter-relationships. Pairwise correlation of our PDAC single-cell and established bulk RNA-seq signatures in malignant cells revealed that the IC state is unique and not well described by prior signatures (<a id="crosref1215" href="#figs3">Figure&nbsp;S3</a>F). Our findings suggest that malignant PDAC cells organize in a tripartite cell state framework that spans committed basal and classical phenotypes, with considerable signature co-expression in single cells (<a id="crosref1220" href="#fig2">Figure&nbsp;2</a>F). Similar to the variation in EMT scores observed in scBasal-expressing cancer cells (<a id="crosref1225" href="#fig2">Figure&nbsp;2</a>C), we noted heterogeneity among co-expressing cells for the IC program (<a id="crosref1230" href="#fig2">Figure&nbsp;2</a>F).</div><div id="p0120" role="paragraph">Classification of tumors by their malignant pseudo-bulk signature expression stratified the cohort into those that expressed predominantly scBasal, scClassical, or IC signatures (<a id="crosref1235" href="#figs3">Figure&nbsp;S3</a>G–S3I). Individual patient specimens still exhibited significant heterogeneity at the cellular level, containing at least two and sometimes all three malignant cell states (<a id="crosref1240" href="#figs3">Figure&nbsp;S3</a>J). Samples with greater malignant cell state diversity (i.e., higher proportions of both scBasal and scClassical cells) also exhibited a higher proportion of cells expressing the IC state, suggesting the IC state may serve as a transition between scBasal and scClassical poles (<a id="crosref1245" href="#fig2">Figure&nbsp;2</a>G).</div></section><section id="sec-2-4"><h3>Multiplex immunofluorescence confirms a tripartite cell state framework in metastatic and primary PDAC</h3><div id="p0125" role="paragraph">To validate this extensive heterogeneity and the presence of co-expressing cells in our metastatic cohort, we used a subtype-specific multiplex immunofluorescence (mIF) panel to categorize single malignant cells by their patterns of marker detection in 10 matched cases from our single cell study (<a id="crosref1250" href="#figs4">Figure&nbsp;S4</a>A; <a id="crosref1255" href="#mmc3">Table S3</a>; <a id="crosref1260" href="#sec-4">STAR Methods</a>). We observed overlap of basal and classical markers within single cells at the protein level, corroborating the existence of co-expressing cells (<a id="crosref1265" href="#fig2">Figures 2</a>H and <a id="crosref1270" href="#figs4">S4</a>B). Moreover, we observed a significant correlation between malignant phenotypes assessed by mIF protein detection for samples from the same RNA subtype (average <i>r&nbsp;=</i> 0.52) compared to those of different subtypes (average <i>r&nbsp;=</i> 0.06, p&nbsp;&lt; 10<sup>−7</sup>, Student’s t test) (<a id="crosref1275" href="#figs4">Figure&nbsp;S4</a>C, white dots). Deep sampling of each matched biopsy using mIF (mean&nbsp;= 13,078 cells per sample) also confirmed the increased fraction of co-expressing cells in tumors with equal basal and classical cell state mixing (<a id="crosref1280" href="#figs4">Figure&nbsp;S4</a>D). Finally, we used mIF to identify co-expressing cells in primary tumor samples, suggesting that the IC state may be a general feature of PDAC (<a id="crosref1285" href="#figs4">Figure&nbsp;S4</a>E).</div><div class="figure-wrap"><figure id="figs4" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.11.017/asset/05e8c667-68ad-421c-8e24-0cfcd56344c2/main.assets/figs4_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs4" src="/cms/10.1016/j.cell.2021.11.017/asset/d35ebd64-74e1-423a-a7a2-7058ea8e9b8d/main.assets/figs4.jpg" height="831" width="657" aria-labelledby="figs4-title figs4-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.017/asset/05e8c667-68ad-421c-8e24-0cfcd56344c2/main.assets/figs4_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs4-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S4</span> <span class="figure__title__text">Multiplex immunofluorescence is concordant with scRNA-seq and demonstrates intratumoral heterogeneity with the presence of intermediate co-expressor cells, related to <a id="crosref2460" href="#fig2">Figure&nbsp;2</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs4-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0470" role="paragraph">(A) Schematic for comparison of the matched datasets by combinatorial marker phenotypes.</div><div id="fspara0475" role="paragraph">(B) Marker detection in each single cell from the 10 samples in the mIF (top, 130,784 cells) and matched scRNA-seq datasets (bottom, 3,062 cells). Cells are sorted by their combinatorial phenotype outlined in S4A.</div><div id="fspara0480" role="paragraph">(C) Comparison within and between modalities on matched samples. Samples are sorted by their pseudo-bulk RNA subtype identity (color bar, top; dendrogram from Figure&nbsp;S3G). Pearson correlations (red to blue heat) were performed over the fractional representation of each marker pattern in S4A for each biopsy using either modality (scRNA-seq or mIF). The upper left and lower right quadrants cross-compare cell state composition from individual biopsies assessed by mIF and scRNA-seq, respectively, with both showing agreement with averaged RNA subtype (top color bar). The upper right and lower left quadrants cross-compare modalities (scRNA-seq versus mIF) for each sample (white dots), demonstrating that individual marker patterns for each sample are similar despite being measured by different methods in different cores from the same patient.</div><div id="fspara0485" role="paragraph">(D) Frequency of co-expressing cells is related to increased mixing of basal and classical cell populations within patient samples assessed by mIF. Log ratio of % basal and classical cells in each sample (x axis; dotted line at 0 indicates equal percentages of basal and classical cells) versus their % co-expressing cells (y axis).</div><div id="fspara0490" role="paragraph">(E) mIF images and marker detection in single cells from two primary PDAC samples identifies IC cells in primary specimens also. Scale bar represents 10&nbsp;μm.</div></div></div></div></figcaption></figure></div></section><section id="sec-2-5"><h3>Benchmarking model expression state using matched <i>in&nbsp;vivo</i> reference maps</h3><div id="p0130" role="paragraph">With this high-resolution reference map of <i>in&nbsp;vivo</i> malignant cell states, we next asked whether matched <i>ex&nbsp;vivo</i> models retained the cell state distribution of the tissues from which they were derived. An unbiased comparison of all malignant cells (biopsy and organoid; 32,073 cells) revealed separation of biopsy and organoid profiles, while organoid samples from iterative passages <i>ex&nbsp;vivo</i> clustered together (<a id="crosref1290" href="#figs5">Figures S5</a>A and S5B). After removing low-frequency non-malignant cells (<a id="crosref1295" href="#figs5">Figures S5</a>B–S5D; <a id="crosref1300" href="#sec-4">STAR Methods</a>), we observed that models attempted from biopsies with high malignant-cell-averaged scBasal or IC states exhibited lower rates of long-term propagation than models derived from scClassical tumors (<a id="crosref1305" href="#fig3">Figure&nbsp;3</a>A). When comparing early passage CNV-confirmed organoid cancer cells to their cognates from patient tissues, culture in an <i>ex&nbsp;vivo</i> microenvironment caused greater deviation in cell state than CNV-defined genotype (<a id="crosref1310" href="#fig3">Figure&nbsp;3</a>A; <a id="crosref1315" href="#sec-4">STAR Methods</a>).</div><div class="figure-wrap"><figure id="fig3" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.11.017/asset/9ed7679c-2c8c-4d73-a28b-d3549ebcb057/main.assets/gr3_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr3" src="/cms/10.1016/j.cell.2021.11.017/asset/19bb9976-a1d2-402e-9a39-6796596f35b0/main.assets/gr3.jpg" height="730" width="657" aria-labelledby="fig3-title fig3-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.017/asset/9ed7679c-2c8c-4d73-a28b-d3549ebcb057/main.assets/gr3_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig3-title" class="figure__title"><span class="label figure__label">Figure&nbsp;3</span> <span class="figure__title__text">Organoid culture selects against the scBasal state with transcriptional evolution over time</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig3-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0105" role="paragraph">(A) Outgrowth and similarity between organoids and matched biopsy samples. Red fill indicates successful early (Early) and long-term (Estab.) culture. Right gray scale indicates similarity between each biopsy-early organoid pair for inferred CNVs (genotype, Geno.) or cell state (State). <i>P</i>-value for Geno. versus State differences determined by Student’s t test.</div><div id="fspara0110" role="paragraph">(B) Schematic for matched <i>in&nbsp;vivo</i> malignant cell and organoid comparison.</div><div id="fspara0115" role="paragraph">(C) Single-cell and average expression of malignant programs (top) and organoid-specific genes (bottom) in biopsy cells and matched, early passage organoid cells (n&nbsp;= 13 models). <i>P</i>-values determined by Student’s t test. Parenthetical <i>P</i>-values (left) indicate hypergeometric test for pathway enrichments.</div><div id="fspara0120" role="paragraph">(D) Swimmer’s plot depicting organoid state evolution in culture. Pie charts indicate the fraction of cell states at each time point.</div><div id="fspara0125" role="paragraph">(E) Clonal fractions from <i>KRAS</i>-amplified PANFR0575 biopsy (gray) and organoid (red) cells. Clone A (green) is present in both. Heatmap shows expression of scBasal and scClassical states in clone A from both contexts.</div><div id="fspara0130" role="paragraph">See also <a id="crosref2360" href="#figs5">Figure&nbsp;S5</a>.</div></div></div></div></figcaption></figure></div><div class="figure-wrap"><figure id="figs5" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.11.017/asset/ceb2d29e-0a03-491e-a4b5-0ef64c698d55/main.assets/figs5_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs5" src="/cms/10.1016/j.cell.2021.11.017/asset/10e8c5f7-7b3d-4df7-8a1b-cddd166c3699/main.assets/figs5.jpg" height="929" width="753" aria-labelledby="figs5-title figs5-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.017/asset/ceb2d29e-0a03-491e-a4b5-0ef64c698d55/main.assets/figs5_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs5-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S5</span> <span class="figure__title__text">Quality metrics, cell type identification, serial sampling, and media influences on transcriptional state in patient-matched organoids, related to <a id="crosref2465" href="#fig3">Figures 3</a> and <a id="crosref2470" href="#fig4">4</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs5-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0500" role="paragraph">(A) Distribution of unique molecules and genes captured in quality cells per organoid sample, median values are indicated for each metric (dotted line) and violin plots are colored by patient ID (top, Log<sub>10</sub>(UMIs); bottom, number of genes).</div><div id="fspara0505" role="paragraph">(B) <i>t</i>-SNE visualization of all biopsy and matched organoid cells from iterative passages, colored by patient ID (left). Dotted circles indicate the only two SNN clusters (4 and 32) with appreciably admixed clusters and low CNV scores, the rest were patient-specific. Bar chart shows number of organoid cells recovered per matched sample (center). On the right <i>t-</i>SNE visualization, cells are organized in the same manner but organoid cells are colored by passage number.</div><div id="fspara0510" role="paragraph">(C) Relative expression for genes defining cluster 4 (top) and cluster 32 (bottom). Cluster 4 had an ambiguous epithelial identity while cluster 32 cells were defined by canonical fibroblast genes and low to absent detection of CNVs.</div><div id="fspara0515" role="paragraph">(D) Fraction of cluster 4 cells at each passage, demonstrating that these cells did not survive iterative passaging.</div><div id="fspara0520" role="paragraph">(E) Single-cell inferred copy number alterations for each sample in the biopsy cohort. Tumors are grouped by expression of their dominant subtype based on the clustering in Figure&nbsp;S3G, <i>P</i>-values comparing presence of each alteration among the groups (scBasal, scClassical, IC) are determined by Fisher’s exact test.</div><div id="fspara0525" role="paragraph">(F) Cell state diagram with marginal density plots for all cells with inferred <i>KRAS</i> amplifications (4 biopsy-model pairs) in biopsy (black) and organoid (red) microenvironments. <i>P</i>-value compares biopsy versus early passage organoid score distributions (top density) and was determined by Student’s t test.</div><div id="fspara0530" role="paragraph">(G-I) Heatmaps show inferred CNV copy number status for every cell in each of three biopsy/early passage organoid pairs. Cells (in rows) are ordered by hierarchical clustering of their CNV profiles and letters on the far left indicate subclones that have significant statistical evidence for tree-splitting (STAR Methods). Each cell’s origin is also noted (“Source” column; biopsy tissue, gray; early passage organoid, red). Right metadata bars indicate if that cell came from an admixed, likely non-malignant SNN cluster (4 or 32 in S5B).</div><div id="fspara0535" role="paragraph">(J-K) Matched cell state and genotype evolution at each passage in PANFR0489R (S5J) and PANFR0575 (S5K). Frequencies of individual CNV clones at each time point (y axis) are tied by colored lines. Fill represents the transcriptional cell state fraction for each CNV clone. In sample PANFR0575 (S5K), clones D and E had inferred <i>TP63</i> amplifications which expanded over time.</div><div id="fspara0540" role="paragraph">(L) Bulk RNA-seq expression of basal and classical states in 4 organoid models cultured in 3D or adapted to 2D in complete organoid media.</div><div id="fspara0545" role="paragraph">(M) Inferred CNVs for each cell from the PANFR489R samples cultured in either Minimal (gray) or Complete (red) organoid media conditions in <a id="crosref2475" href="#fig4">Fig.&nbsp;4</a>B.</div><div id="fspara0550" role="paragraph">(N) Brightfield images of organoids grown in standard organoid media (“Complete”) or in media without any growth factors (“Minimal”) at days 1 and 11 after seeding. Scale bar represents 200&nbsp;μm.</div><div id="fspara0555" role="paragraph">(O) Expression of scBasal, IC, and scClassical genes (rows) in organoid cells (columns) from model PANFR0562 cultured for 6&nbsp;days in Complete medium, Minimal medium, or OWRNA medium. <i>P</i>-values for group differences were calculated by ANOVA followed by Tukey’s HSD.</div><div id="fspara0560" role="paragraph">(P) Expression of scBasal, IC, and scClassical genes (rows) in organoid cells (columns) from model PANFR0562 cultured for 6&nbsp;days in Complete organoid medium or in “Cell line” medium. <i>P</i>-values for group differences were calculated by ANOVA followed by Tukey’s HSD.</div><div id="fspara0565" role="paragraph">(Q) Expression of scBasal, IC, and scClassical genes (rows) in single cells from the PDAC cell line CFPAC1 (columns) cultured for 6&nbsp;days in standard “Cell line” medium or in Complete organoid medium. <i>P</i>-values for group differences were calculated by ANOVA followed by Tukey’s HSD.</div></div></div></div></figcaption></figure></div><div id="p0135" role="paragraph">We next examined how <i>ex&nbsp;vivo</i> transcriptional states in our organoid cohort differed from their matched patient samples (<a id="crosref1320" href="#fig3">Figure&nbsp;3</a>B). We observed a striking loss of scBasal gene expression and to a lesser extent, the IC program. By contrast, aggregate scClassical gene expression remained largely unchanged in organoid conditions. This comparative analysis also nominated a set of upregulated organoid-specific genes that were not present <i>in&nbsp;vivo</i>, including markers of epithelial identity, oxidative stress response pathways (e.g., NRF2 target genes), and amino acid metabolism (hereafter collectively referred to as “organoid-specific” gene expression) (<a id="crosref1325" href="#fig3">Figure&nbsp;3</a>C, bottom; <a id="crosref1330" href="#mmc4">Table S4</a>). These findings suggest that changes to microenvironmental growth conditions significantly alter cellular transcriptional states and induce culture-specific expression programs, highlighting the importance of benchmarking <i>ex&nbsp;vivo</i> models using matched <i>in&nbsp;vivo</i> states as a reference.</div></section><section id="sec-2-6"><h3>Transcriptional state heterogeneity is shaped by the <i>ex&nbsp;vivo</i> microenvironment</h3><div id="p0140" role="paragraph">Regardless of the cell state distribution in the original biopsy, individual models assumed more scClassical or organoid-specific cell states over time in culture (<a id="crosref1335" href="#fig3">Figure&nbsp;3</a>D), mirroring the results in our larger bulk RNA-seq cohort (<a id="crosref1340" href="#fig1">Figures 1</a>B and <a id="crosref1345" href="#fig3">3</a>C [top]). To better understand the drivers of this cell state bias, we first investigated genetic alterations associated with either the basal or classical subtypes. Prior work has suggested that <i>KRAS</i> amplifications associate with basal features (<span class="dropBlock reference-citations"><a id="body-ref-sref16-5" href="#" role="doc-biblioref" data-xml-rid="bib16" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref16-5" aria-controls="sref16-5">Chan-Seng-Yue et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref16-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">16.</div><div>Chan-Seng-Yue, M. ∙ Kim, J.C. ∙ Wilson, G.W. ...</div><div><strong>Transcription phenotypes of pancreatic cancer are driven by genomic events during tumor evolution</strong></div><div><em>Nat. Genet.</em> 2020; <strong>52</strong>:231-240</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.Chan-Seng-YueJ.C.KimG.W.WilsonK.NgE.F.FigueroaG.M.O%E2%80%99KaneA.A.ConnorR.E.DenrocheR.C.GrantJ.McLeodTranscription+phenotypes+of+pancreatic+cancer+are+driven+by+genomic+events+during+tumor+evolutionNat.+Genet.522020231240" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref49-1" href="#" role="doc-biblioref" data-xml-rid="bib49" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref49-1" aria-controls="sref49-1">Miyabayashi et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref49-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">49.</div><div>Miyabayashi, K. ∙ Baker, L.A. ∙ Deschênes, A. ...</div><div><strong>Intraductal Transplantation Models of Human Pancreatic Ductal Adenocarcinoma Reveal Progressive Transition of Molecular Subtypes</strong></div><div><em>Cancer Discov.</em> 2020; <strong>10</strong>:1566-1589</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=K.MiyabayashiL.A.BakerA.Desch%C3%AAnesB.TraubG.CaligiuriD.PlenkerB.AlagesanP.BelleauS.LiJ.KendallIntraductal+Transplantation+Models+of+Human+Pancreatic+Ductal+Adenocarcinoma+Reveal+Progressive+Transition+of+Molecular+SubtypesCancer+Discov.10202015661589" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), while amplifications of lineage transcription factors like <i>GATA6</i> associate with classical subtypes (<span class="dropBlock reference-citations"><a id="body-ref-sref16-6" href="#" role="doc-biblioref" data-xml-rid="bib16" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref16-6" aria-controls="sref16-6">Chan-Seng-Yue et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref16-6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">16.</div><div>Chan-Seng-Yue, M. ∙ Kim, J.C. ∙ Wilson, G.W. ...</div><div><strong>Transcription phenotypes of pancreatic cancer are driven by genomic events during tumor evolution</strong></div><div><em>Nat. Genet.</em> 2020; <strong>52</strong>:231-240</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.Chan-Seng-YueJ.C.KimG.W.WilsonK.NgE.F.FigueroaG.M.O%E2%80%99KaneA.A.ConnorR.E.DenrocheR.C.GrantJ.McLeodTranscription+phenotypes+of+pancreatic+cancer+are+driven+by+genomic+events+during+tumor+evolutionNat.+Genet.522020231240" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). In agreement with prior bulk studies, we observed a significant association between single-cell inferred <i>KRAS</i> copy number gain and the scBasal state in metastatic cells (p&nbsp;&lt; <i>0.03</i> Fisher’s exact test) (<a id="crosref1365" href="#figs5">Figure&nbsp;S5</a>E). Four <i>KRAS</i>-amplified samples proliferated <i>ex&nbsp;vivo</i> and maintained this alteration, but their malignant cell state shifted from scBasal <i>in&nbsp;vivo</i> to scClassical in organoid culture (<a id="crosref1370" href="#fig3">Figures 3</a>D [black dots] and <a id="crosref1375" href="#figs5">S5</a>F), including one sample where the same <i>KRAS</i>-amplified clone exhibited a similar scBasal (biopsy) to scClassical (organoid) state shift (<a id="crosref1380" href="#fig3">Figure&nbsp;3</a>E). These findings demonstrate that <i>KRAS</i> amplification alone is not sufficient to lock the scBasal expression state, PDAC cells are plastic, and the microenvironment can influence cell state independent of genotype in this context.</div><div id="p0145" role="paragraph">We next compared genetic heterogeneity (inferred CNV sub-clones) and transcriptional states from matched biopsy tissue and organoid samples from iterative passages. ScClassical tumors tended to maintain their genotype and transcriptional state both early in culture and at later passages (e.g., PANFR0631) (<a id="crosref1385" href="#fig3">Figures 3</a>D and <a id="crosref1390" href="#figs5">S5</a>G [clone A]). In contrast, most models derived from scBasal or IC tumors exhibited early cell-state deviation and cessation of growth within 100&nbsp;days of initiation (e.g., PANFR0552) (<a id="crosref1395" href="#fig3">Figures 3</a>D and <a id="crosref1400" href="#figs5">S5</a>H). In some cases, we observed outgrowth of an scClassical sub-clone in culture. For example, the <i>in&nbsp;vivo</i> scBasal clones from PANFR0489R rapidly decreased in abundance while clones with scClassical or organoid-specific states emerged at later passages (<a id="crosref1405" href="#figs5">Figures S5</a>I and S5J). By contrast, <i>in&nbsp;vivo</i> scBasal clones from PANFR0575 were maintained initially in culture, but their cell state was highly plastic and changed to scClassical when measured as organoids at passages 2 and 3 (<a id="crosref1410" href="#fig3">Figures 3</a>E and <a id="crosref1415" href="#figs5">S5</a>K). After &gt; 100 additional days in <i>ex&nbsp;vivo</i> culture, PANFR0575 regained scBasal expression and clones with inferred <i>TP63</i> amplifications, a squamous-specifying transcription factor (<span class="dropBlock reference-citations"><a id="body-ref-sref65-1" href="#" role="doc-biblioref" data-xml-rid="bib65" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref65-1" aria-controls="sref65-1">Somerville et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref65-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">65.</div><div>Somerville, T.D.D. ∙ Xu, Y. ∙ Miyabayashi, K. ...</div><div><strong>TP63-Mediated Enhancer Reprogramming Drives the Squamous Subtype of Pancreatic Ductal Adenocarcinoma</strong></div><div><em>Cell Rep.</em> 2018; <strong>25</strong>:1741-1755 e1747</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=T.D.D.SomervilleY.XuK.MiyabayashiH.TiriacC.R.ClearyD.Maia-SilvaJ.P.MilazzoD.A.TuvesonC.R.VakocTP63-Mediated+Enhancer+Reprogramming+Drives+the+Squamous+Subtype+of+Pancreatic+Ductal+AdenocarcinomaCell+Rep.25201817411755+e1747" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), dominated the culture (<a id="crosref1425" href="#figs5">Figure&nbsp;S5</a>K [clones D and E]). Our detailed analysis reveals that both plasticity and selection occur in the same model, where certain clones demonstrate plasticity in response to microenvironmental signals, and certain genotypes, though rare, may still exert a strong effect despite opposing cues from the microenvironment. In sum, these findings underscore the need to consider both mutational and transcriptional state to ensure faithful representation of <i>in&nbsp;vivo</i> cancer cell phenotypes.</div></section><section id="sec-2-7"><h3>Media formulation influences PDAC transcriptional states <i>ex&nbsp;vivo</i></h3><div id="p0150" role="paragraph">Given the evidence for selection and plasticity in standard organoid conditions, we tested whether specific aspects of the <i>ex&nbsp;vivo</i> culture environment governed cell state determination. We first evaluated the effects of extracellular matrix dimensionality by culturing established 3-dimensional (3D) organoid models as 2-dimensional (2D) cell lines in the same organoid medium. This did not affect transcriptional subtype across the 4 models tested (<a id="crosref1430" href="#figs5">Figure&nbsp;S5</a>L). Next, we asked whether culturing established organoid models in altered media conditions could rescue expression heterogeneity (<a id="crosref1435" href="#fig4">Figure&nbsp;4</a>A). We cultured 4 organoid models in media without any additives for 6&nbsp;days (“Minimal” medium) (<a id="crosref1440" href="#mmc5">Table S5</a>; <a id="crosref1445" href="#sec-4">STAR Methods</a>) and observed a robust increase in scBasal expression, a decrease in organoid-specific gene expression, and stable CNV profiles (<a id="crosref1450" href="#fig4">Figures 4</a>B, <a id="crosref1455" href="#figs5">S5</a>M, and S5N). With greater time in minimal medium, the distribution of cell states shifted even further toward IC and scBasal (<a id="crosref1460" href="#fig4">Figure&nbsp;4</a>C). Since minimal medium lacks serum and mitogens to maintain prolonged cell growth, we tested a reduced organoid media formulation (“OWRNA”) (<a id="crosref1465" href="#mmc5">Table S5</a>; <a id="crosref1470" href="#sec-4">STAR Methods</a>) and found that OWRNA supported proliferation, strengthened scClassical, and allowed for scBasal expression (<a id="crosref1475" href="#figs5">Figure&nbsp;S5</a>O).</div><div class="figure-wrap"><figure id="fig4" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.11.017/asset/0fbcc286-7536-4d99-b3cd-16edafb21927/main.assets/gr4_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr4" src="/cms/10.1016/j.cell.2021.11.017/asset/9e730ef2-f7ed-4457-8b56-166d5c077dc7/main.assets/gr4.jpg" height="721" width="657" aria-labelledby="fig4-title fig4-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.017/asset/0fbcc286-7536-4d99-b3cd-16edafb21927/main.assets/gr4_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig4-title" class="figure__title"><span class="label figure__label">Figure&nbsp;4</span> <span class="figure__title__text">Modulation of the media microenvironment allows recovery of scBasal states</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig4-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0140" role="paragraph">(A) Strategies to recover scBasal expression in different media conditions.</div><div id="fspara0145" role="paragraph">(B) Tied dot plot for sample average scBasal score (left) and organoid-specific score (right) in the indicated conditions. Color outline indicates sample identity. <i>P</i>-value compares single cell distributions within models and was calculated by Student’s t test.</div><div id="fspara0150" role="paragraph">(C) Cell state diagrams for organoid cells cultured in minimal media. <i>P</i>-values are for that time point versus the complete media condition and compare B/C commitment (top) and IC scores (right) by ANOVA followed by Tukey’s HSD.</div><div id="fspara0155" role="paragraph">(D and E) Violin plots depict scBasal and organoid-specific expression scores in PANFR0562 organoid cells (D) or CFPAC1 cell line (E) after 6&nbsp;days in Complete organoid medium or in “Cell line” medium. <i>P</i>-values for differences were calculated by Student’s t test.</div><div id="fspara0160" role="paragraph">(F) CFPAC1 cell line growth rate-adjusted dose response curves to SN-38 and paclitaxel after culture in standard “Cell line” medium or in Complete organoid medium. Points are the mean ± SD of 3 technical replicates. Curves are representative of 2 independent experiments.</div><div id="fspara0165" role="paragraph">(G) Significant pathway enrichments (<i>P</i>-value&nbsp;&lt; 10<sup>−12</sup>) for top <i>in&nbsp;vivo</i> differentially expressed genes (143 genes).</div><div id="fspara0170" role="paragraph">(H) Average expression in biopsy (left) and organoid cells (right) for the 143 top <i>in&nbsp;vivo</i> differentially expressed genes (rows), organized by originating tumor’s overall transcriptional subtype (colored dots). Parenthetical <i>P</i>-values (left) for enrichment of indicated pathways are by hypergeometric test. Overall biopsy versus organoid expression difference is determined by Student’s t test (bottom). <i>P</i>-values computed by one-way ANOVA followed by Tukey’s HSD (center) are for differences in average expression between biopsy transcriptional subtypes (<sup>∗</sup><i>P</i>-value&nbsp;&lt; 10<sup>−8</sup>; <sup>∗∗</sup><i>P</i>-value&nbsp;&lt; 10<sup>−16</sup>).</div><div id="fspara0175" role="paragraph">See also <a id="crosref2365" href="#figs5">Figure&nbsp;S5</a>; <a id="crosref2370" href="#mmc4">Tables S4</a> and <a id="crosref2375" href="#mmc5">S5</a>.</div></div></div></div></figcaption></figure></div><div id="p0155" role="paragraph">Given the divergence in cell state for cell line versus organoid models of PDAC (<a id="crosref1480" href="#fig1">Figures 1</a>A and 1B), we took an established cell line and an organoid model and cultured them in the reciprocal media condition. Organoid cells grown in standard cancer cell line medium (“Cell line media,” RPMI-1640 with 10% fetal bovine serum) (<a id="crosref1485" href="#sec-4">STAR Methods</a>) gained scBasal expression, while CFPAC1, an established pancreatic cancer cell line, lost scBasal features when grown in complete organoid media (<a id="crosref1490" href="#fig4">Figures 4</a>D, 4E, <a id="crosref1495" href="#figs5">S5</a>P, and S5Q). To understand whether these state changes are functionally significant, we exposed CFPAC1 cells grown in basal media conditions (“cell line media”) or classical conditions (complete organoid media) to SN-38 and paclitaxel. Strikingly, in both instances, we observed greater sensitivity among cells grown in organoid media, suggesting that environmental factors can shape therapeutic response through state changes (<a id="crosref1500" href="#fig4">Figure&nbsp;4</a>F). These findings demonstrate that even established models are sensitive to changes in culture conditions and that a combination of up-front selective pressures and culture medium-imprinting shapes the cell-state biases in extant model systems, influencing their functional attributes.</div><div id="p0160" role="paragraph">Although maintaining organoids in minimal or cell line media conditions resulted in partial recovery of scBasal and IC expression, we failed to observe fully polarized models (Figures S5P and S5Q), suggesting that these conditions lack critical TME components to fully specify cell state. We used differential expression across each biopsy-organoid pair to nominate genes that were expressed <i>in&nbsp;vivo</i> but missing <i>ex&nbsp;vivo</i> (<a id="crosref1505" href="#fig3">Figure&nbsp;3</a>B; <a id="crosref1510" href="#sec-4">STAR Methods</a>). Genes differentially expressed by malignant cells <i>in&nbsp;vivo</i> were related to soluble cytokine signaling, cell-cell communication, and microenvironmental interactions (<a id="crosref1515" href="#fig4">Figure&nbsp;4</a>G). Hierarchical clustering revealed state-dependent expression patterns for <i>in&nbsp;vivo</i>-specific genes (<a id="crosref1520" href="#fig4">Figure&nbsp;4</a>H; <a id="crosref1525" href="#mmc4">Table S4</a>). For example, IFN response and EMT genes were significantly upregulated in scBasal and IC malignant cells <i>in&nbsp;vivo</i> (clusters 1 and 2), while genes associated with cell-cell interactions and surface glycoproteins were more strongly expressed in IC and scClassical cells (cluster 3). Genes related to biological adhesion were more uniform in their expression across the subtypes (cluster 4) (<a id="crosref1530" href="#fig4">Figure&nbsp;4</a>H). The relative absence of these genes in organoid culture and their differences in expression across transcriptional subtypes <i>in&nbsp;vivo</i> suggests that TME signals might play a role in fully specifying cancer cell transcriptional state and potentially therapeutic responses.</div></section><section id="sec-2-8"><h3>Composition of the metastatic TME and site-specific differences in mesenchymal populations</h3><div id="p0165" role="paragraph">The presence of TME-associated expression patterns in cancer cells <i>in&nbsp;vivo</i> suggested there may be subtype-dependent structure to, and instructive signaling from, the metastatic TME; however, relatively little is known about the composition of the metastatic TME in PDAC. We analyzed the non-malignant cells (n&nbsp;= 14,811) in the metastatic niche to further subclassify cell types and provide a more complete picture of the immune/stromal composition of metastatic disease (<a id="crosref1535" href="#fig5">Figure&nbsp;5</a>A). Sub-clustering of T&nbsp;cells and natural killer (NK) cells revealed 4 cell types (<i>CD4</i>+ T, <i>CD8</i>+ T, NK, and <i>CD16+</i> [<i>FCGR3A+</i>] NK cells) (<a id="crosref1540" href="#figs6">Figures S6</a>A and S6B; <a id="crosref1545" href="#sec-4">STAR Methods</a>) Similarly, an unbiased analysis within the monocyte/macrophage compartment revealed 3 subsets of tumor associated macrophages (TAMs) (<i>FCN1</i>+ “monocyte-like” TAMs, <i>C1QC</i>+ TAMs, and <i>SPP1+</i> TAMs) (<a id="crosref1550" href="#figs6">Figures S6</a>C and S6D; <a id="crosref1555" href="#mmc2">Table S2</a>) (<span class="dropBlock reference-citations"><a id="body-ref-sref78-1" href="#" role="doc-biblioref" data-xml-rid="bib78" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref78-1" aria-controls="sref78-1">Zhang et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref78-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">78.</div><div>Zhang, L. ∙ Li, Z. ∙ Skrzypczynska, K.M. ...</div><div><strong>Single-Cell Analyses Inform Mechanisms of Myeloid-Targeted Therapies in Colon Cancer</strong></div><div><em>Cell.</em> 2020; <strong>181</strong>:442-459 e429</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=L.ZhangZ.LiK.M.SkrzypczynskaQ.FangW.ZhangS.A.O%E2%80%99BrienY.HeL.WangQ.ZhangA.KimSingle-Cell+Analyses+Inform+Mechanisms+of+Myeloid-Targeted+Therapies+in+Colon+CancerCell1812020442459+e429" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref79-1" href="#" role="doc-biblioref" data-xml-rid="bib79" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref79-1" aria-controls="sref79-1">Zilionis et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref79-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">79.</div><div>Zilionis, R. ∙ Engblom, C. ∙ Pfirschke, C. ...</div><div><strong>Single-Cell Transcriptomics of Human and Mouse Lung Cancers Reveals Conserved Myeloid Populations across Individuals and Species</strong></div><div><em>Immunity.</em> 2019; <strong>50</strong>:1317-1334 e1310</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=R.ZilionisC.EngblomC.PfirschkeV.SavovaD.ZemmourH.D.SaatciogluI.KrishnanG.MaroniC.V.MeyerovitzC.M.KerwinSingle-Cell+Transcriptomics+of+Human+and+Mouse+Lung+Cancers+Reveals+Conserved+Myeloid+Populations+across+Individuals+and+SpeciesImmunity50201913171334+e1310" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Marker expression across all previously described non-malignant cells is summarized in <a id="crosref1570" href="#figs6">Figure&nbsp;S6</a>E.</div><div class="figure-wrap"><figure id="fig5" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.11.017/asset/97e19c23-fef4-44b3-a993-fbd7e8659517/main.assets/gr5_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr5" src="/cms/10.1016/j.cell.2021.11.017/asset/6324ba1e-d8e9-4503-981c-b925cece9537/main.assets/gr5.jpg" height="851" width="657" aria-labelledby="fig5-title fig5-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.017/asset/97e19c23-fef4-44b3-a993-fbd7e8659517/main.assets/gr5_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig5-title" class="figure__title"><span class="label figure__label">Figure&nbsp;5</span> <span class="figure__title__text">Distinct mesenchymal phenotypes and transcriptional state-specific immune heterogeneity exist in the liver metastatic microenvironment</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig5-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0185" role="paragraph">(A) <i>t-</i>SNE visualization of non-malignant cells identified in the metastatic microenvironment, abbreviations as in <a id="crosref2380" href="#figs1">Figure&nbsp;S1</a>J (TAM, tumor associated macrophage).</div><div id="fspara0190" role="paragraph">(B) Expression of Fibroblast-like (PC2 low) and Pericyte-like (PC2 high) mesenchymal (Mes.) cell programs across different metastatic sites (top).</div><div id="fspara0195" role="paragraph">(C) Density plots for mesenchymal cell phenotype score in single cells from our metastatic cohort (top) or previously published PDAC bulk RNA-seq profiles (bottom), fill indicates anatomic site. <i>P</i>-value determined by Student’s t test (top) or by ANOVA followed by Tukey’s HSD (bottom).</div><div id="fspara0200" role="paragraph">(D) Summary of mesenchymal phenotypes in primary versus liver metastatic PDAC.</div><div id="fspara0205" role="paragraph">(E) Correlation between microenvironment diversity (Simpson’s index, x axis) and the average malignant scBasal-scClassical commitment score (y axis) for each scRNA-seq sample.</div><div id="fspara0210" role="paragraph">(F) Correlation between TME diversity as inferred by immunogenic signature score (x axis) versus average tumor scBasal-scClassical commitment score (y axis) in primary and metastatic bulk RNA-seq samples.</div><div id="fspara0215" role="paragraph">(G and H) Sample-level (columns) variation in Simpson’s index (G, dot plot), average malignant scBasal-scClassical (G, top heat bar) and IC (G, bottom heat bar) expression, and fraction of non-malignant cell types (H). Samples were clustered and ordered within metastatic site (liver versus other) by their fractional representation of cell types. Dots indicate top statistically significant cell type frequency differences calculated using a Kruskal-Wallis test with multiple hypothesis correction.</div><div id="fspara0220" role="paragraph">(I) Boxplots compare cell type fractions between the scBasal predominant tumors with low diversity (PANFR0593, 575, 545) and all others. <i>P</i>-value determined by Student’s t test.</div><div id="fspara0225" role="paragraph">(J) Summary of associations between microenvironmental diversity, non-malignant cell types, and malignant cell state.</div><div id="fspara0230" role="paragraph">See also <a id="crosref2385" href="#figs6">Figure&nbsp;S6</a>; <a id="crosref2390" href="#mmc2">Table S2</a>.</div></div></div></div></figcaption></figure></div><div class="figure-wrap"><figure id="figs6" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.11.017/asset/834d14a7-0517-4d2d-b310-6c8227eb0dd0/main.assets/figs6_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs6" src="/cms/10.1016/j.cell.2021.11.017/asset/405a5c2d-3487-4b04-8223-c5b2b7b9f98c/main.assets/figs6.jpg" height="941" width="729" aria-labelledby="figs6-title figs6-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.017/asset/834d14a7-0517-4d2d-b310-6c8227eb0dd0/main.assets/figs6_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs6-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S6</span> <span class="figure__title__text">Identification of T cell, natural killer cell, macrophage, and mesenchymal heterogeneity and transcriptional state associations in the metastatic microenvironment, related to <a id="crosref2480" href="#fig5">Figure&nbsp;5</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs6-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0575" role="paragraph">(A) <i>t-</i>SNE visualization of sub-clustering (SNN) performed on T/NK cells in the metastatic cohort. Cells are colored by their type identity based on shared SNN cluster membership.</div><div id="fspara0580" role="paragraph">(B) Select cell type marker expression overlaid on the <i>t-</i>SNE visualization from S6A<b>.</b></div><div id="fspara0585" role="paragraph">(C) PCA identifies 3 major subsets of TAMs in the metastatic niche. PC1 largely separates <i>FCN1</i>+ monocyte-like TAMs from more committed macrophage phenotypes. PC2 separates <i>SPP1</i>+ from <i>C1QC</i>+ macrophage phenotypes.</div><div id="fspara0590" role="paragraph">(D) Heatmap visualization of the gene expression programs specific to each TAM subset identified by the PCA and SNN clusters in S6C.</div><div id="fspara0595" role="paragraph">(E) Relative expression for select cell type markers and binarized cell cycle program (top bar; black, cycling) with cell type colors (bottom color bar) as in <a id="crosref2485" href="#fig5">Figure&nbsp;5</a>A.</div><div id="fspara0600" role="paragraph">(F) <i>t-</i>SNE visualization of sub-clustering (SNN) performed on mesenchymal cells colored by their anatomical site. Cell subsets (1-4) determined by SNN clustering.</div><div id="fspara0605" role="paragraph">(G) Average expression of the indicated mesenchymal/fibroblast and adrenal endocrine marker genes in each of the cell subsets (1-4) identified in S6F. Dot size indicates fraction of cells expressing a given gene.</div><div id="fspara0610" role="paragraph">(H) PCA over mesenchymal cells in the cohort (excluding Adrenal endocrine cells; subset 4, <b>S6F</b>). Scatterplot of PC2 versus PC3 defines 3 states for mesenchymal cells in our cohort.</div><div id="fspara0615" role="paragraph">(I) Same visualization as in S6H, but cells are colored by previously identified myCAF or iCAF signature scores. myCAF is evenly distributed across PC2 and iCAF associates with higher PC3 scores.</div><div id="fspara0620" role="paragraph">(J) Expression for select markers overlaid on the PCA from S6H.</div><div id="fspara0625" role="paragraph">(K) Cross-correlation of mesenchymal signatures in single-cells. Fibroblast-like versus pericyte-like signatures provide non-overlapping information. PC3 inflammatory phenotypes are similar to the previously reported iCaf phenotype and our PC3-derived inflammatory mesenchymal signature.</div><div id="fspara0630" role="paragraph">(L) Frequency of mesenchymal cells (y axis, cell count) across PC2 scores, colored by site of biopsy tissue. <i>P</i>-value determined by Student’s t test.</div><div id="fspara0635" role="paragraph">(M) Comparison of the distribution of mesenchymal phenotypes across different sites as groups (top) or individual tumors (bottom). Heat indicates the fraction of mesenchymal cells in that score bin.</div><div id="fspara0640" role="paragraph">(N) Non-malignant cell types (color fill) and counts per biopsy, included samples organized as in <a id="crosref2490" href="#fig5">Figure&nbsp;5</a>G. Five biopsies were excluded from the analysis in <a id="crosref2495" href="#fig5">Figure&nbsp;5</a>E,5G-I because they either had low cell capture or were from a tumor with indeterminant malignant transcriptional subtype due to negligible malignant cell capture.</div><div id="fspara0645" role="paragraph">(O) Correlation between T&nbsp;cell fraction and malignant IC score.</div><div id="fspara0650" role="paragraph">(P) Cross-TCGA analysis for basal and immune cell type markers in epithelial tumors with known basal subtypes. Clusters were determined by dendrogram splitting, and disease type for each sample is indicated at bottom.</div></div></div></div></figcaption></figure></div><div id="p0170" role="paragraph">Whether the TME differs between primary PDAC and different metastatic sites is not well understood. Although we found equal distribution of immune cells across different metastatic sites, mesenchymal cells clustered by the site of disease (<a id="crosref1575" href="#figs6">Figures S6</a>F and S6G). Despite uniform expression of a previously described myofibroblast signature (<span class="dropBlock reference-citations"><a id="body-ref-sref22-2" href="#" role="doc-biblioref" data-xml-rid="bib22" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref22-2" aria-controls="sref22-2">Elyada et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref22-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">22.</div><div>Elyada, E. ∙ Bolisetty, M. ∙ Laise, P. ...</div><div><strong>Cross-Species Single-Cell Analysis of Pancreatic Ductal Adenocarcinoma Reveals Antigen-Presenting Cancer-Associated Fibroblasts</strong></div><div><em>Cancer Discov.</em> 2019; <strong>9</strong>:1102-1123</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=E.ElyadaM.BolisettyP.LaiseW.F.FlynnE.T.CourtoisR.A.BurkhartJ.A.TeinorP.BelleauG.BiffiM.S.LucitoCross-Species+Single-Cell+Analysis+of+Pancreatic+Ductal+Adenocarcinoma+Reveals+Antigen-Presenting+Cancer-Associated+FibroblastsCancer+Discov.9201911021123" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref54" href="#" role="doc-biblioref" data-xml-rid="bib54" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref54" aria-controls="sref54">Ӧhlund et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref54"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">54.</div><div>Öhlund, D. ∙ Handly-Santana, A. ∙ Biffi, G. ...</div><div><strong>Distinct populations of inflammatory fibroblasts and myofibroblasts in pancreatic cancer</strong></div><div><em>J.&nbsp;Exp. Med.</em> 2017; <strong>214</strong>:579-596</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=D.%C3%96hlundA.Handly-SantanaG.BiffiE.ElyadaA.S.AlmeidaM.Ponz-SarviseV.CorboT.E.OniS.A.HearnE.J.LeeDistinct+populations+of+inflammatory+fibroblasts+and+myofibroblasts+in+pancreatic+cancerJ.%C2%A0Exp.+Med.2142017579596" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), an unbiased analysis revealed divergent mesenchymal states favoring expression of either fibroblast-like or pericyte-like genes (<a id="crosref1590" href="#figs6">Figures S6</a>H–S6K; <a id="crosref1595" href="#mmc2">Table S2</a>) (<span class="dropBlock reference-citations"><a id="body-ref-sref7" href="#" role="doc-biblioref" data-xml-rid="bib7" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref7" aria-controls="sref7">Bartoschek et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref7"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">7.</div><div>Bartoschek, M. ∙ Oskolkov, N. ∙ Bocci, M. ...</div><div><strong>Spatially and functionally distinct subclasses of breast cancer-associated fibroblasts revealed by single cell RNA sequencing</strong></div><div><em>Nat. Commun.</em> 2018; <strong>9</strong>:5150</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.BartoschekN.OskolkovM.BocciJ.L%C3%B6vrotC.LarssonM.SommarinC.D.MadsenD.LindgrenG.PekarG.KarlssonSpatially+and+functionally+distinct+subclasses+of+breast+cancer-associated+fibroblasts+revealed+by+single+cell+RNA+sequencingNat.+Commun.920185150" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref20" href="#" role="doc-biblioref" data-xml-rid="bib20" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref20" aria-controls="sref20">Di Carlo and Peduto, 2018</a><div class="dropBlock__holder" data-db-target-of="sref20"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">20.</div><div>Di Carlo, S.E. ∙ Peduto, L.</div><div><strong>The perivascular origin of pathological fibroblasts</strong></div><div><em>J.&nbsp;Clin. Invest.</em> 2018; <strong>128</strong>:54-63</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=S.E.Di+CarloL.PedutoThe+perivascular+origin+of+pathological+fibroblastsJ.%C2%A0Clin.+Invest.12820185463" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref38" href="#" role="doc-biblioref" data-xml-rid="bib38" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref38" aria-controls="sref38">Hosaka et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref38"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">38.</div><div>Hosaka, K. ∙ Yang, Y. ∙ Seki, T. ...</div><div><strong>Pericyte-fibroblast transition promotes tumor growth and metastasis</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2016; <strong>113</strong>:E5618-E5627</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=K.HosakaY.YangT.SekiC.FischerO.DubeyE.FredlundJ.HartmanP.ReligaH.MorikawaY.IshiiPericyte-fibroblast+transition+promotes+tumor+growth+and+metastasisProc.+Natl.+Acad.+Sci.+USA1132016E5618E5627" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref58" href="#" role="doc-biblioref" data-xml-rid="bib58" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref58" aria-controls="sref58">Pelon et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref58"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">58.</div><div>Pelon, F. ∙ Bourachot, B. ∙ Kieffer, Y. ...</div><div><strong>Cancer-associated fibroblast heterogeneity in axillary lymph nodes drives metastases in breast cancer through complementary mechanisms</strong></div><div><em>Nat. Commun.</em> 2020; <strong>11</strong>:404</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=F.PelonB.BourachotY.KiefferI.MagagnaF.Mermet-MeillonI.BonnetA.CostaA.M.GivelY.AttiehJ.BarbazanCancer-associated+fibroblast+heterogeneity+in+axillary+lymph+nodes+drives+metastases+in+breast+cancer+through+complementary+mechanismsNat.+Commun.112020404" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). While biopsies from each location contained both subsets, we detected a strong association between liver biopsies and the pericyte-like mesenchymal state (<a id="crosref1620" href="#fig5">Figures 5</a>B, <a id="crosref1625" href="#figs6">S6</a>L, and S6M). We validated this association in larger cohorts and observed a similar relationship between pericyte-like expression and liver metastases, while in contrast, tumors in the pancreas favored fibroblast-like mesenchymal expression (<a id="crosref1630" href="#fig5">Figure&nbsp;5</a>C). Thus, we observed diverse immune and stromal cell types in the metastatic TME and identified mesenchymal features unique to the liver niche compared with primary disease (<a id="crosref1635" href="#fig5">Figure&nbsp;5</a>D).</div></section><section id="sec-2-9"><h3>Transcriptional subtypes associate with distinct immune microenvironments</h3><div id="p0175" role="paragraph">We next searched for associations between malignant cell state and the composition of the TME. Five samples were excluded from this analysis on the basis of low cell counts (&lt; 200 cells) or indeterminant transcriptional subtype (<a id="crosref1640" href="#figs6">Figure&nbsp;S6</a>N). We applied Simpson’s diversity index to define each tumor’s overall microenvironmental composition (<a id="crosref1645" href="#sec-4">STAR Methods</a>). Tumors with higher average malignant scClassical or IC expression harbored greater TME diversity, while strongly scBasal tumors exhibited more homogeneous TMEs (<a id="crosref1650" href="#fig5">Figure&nbsp;5</a>E). We observed a similar pattern in bulk samples where we inferred diversity by using the “immunogenic” signature (<span class="dropBlock reference-citations"><a id="body-ref-sref5-4" href="#" role="doc-biblioref" data-xml-rid="bib5" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref5-4" aria-controls="sref5-4">Bailey et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref5-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">5.</div><div>Bailey, P. ∙ Chang, D.K. ∙ Nones, K. ..., Australian Pancreatic Cancer Genome Initiative</div><div><strong>Genomic analyses identify molecular subtypes of pancreatic cancer</strong></div><div><em>Nature.</em> 2016; <strong>531</strong>:47-52</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=P.BaileyD.K.ChangK.NonesA.L.JohnsA.M.PatchM.C.GingrasD.K.MillerA.N.ChristT.J.BruxnerM.C.QuinnAustralian+Pancreatic+Cancer+Genome+InitiativeGenomic+analyses+identify+molecular+subtypes+of+pancreatic+cancerNature53120164752" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) to indicate greater immune cell infiltration (<a id="crosref1660" href="#fig5">Figure&nbsp;5</a>F). Clustering over the cell type fractions in each biopsy revealed the non-malignant cell types driving overall diversity differences (<a id="crosref1665" href="#fig5">Figures 5</a>G and 5H). C1QC+ TAMs dominated the TME of strongly scBasal tumors, and both CD8+ and CD4+ T&nbsp;cells were significantly depleted in scBasal contexts compared to the rest of the samples in the cohort (<a id="crosref1670" href="#fig5">Figures 5</a>H and 5I). By contrast, T&nbsp;cells were most often isolated from, and their abundance positively correlated with, higher IC malignant fractions in our cohort (<a id="crosref1675" href="#fig5">Figures 5</a>I and <a id="crosref1680" href="#figs6">S6</a>O). We observed similar patterns in bulk RNA-seq data from The Cancer Genome Atlas (TCGA) (<span class="dropBlock reference-citations"><a id="body-ref-sref76-1" href="#" role="doc-biblioref" data-xml-rid="bib76" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref76-1" aria-controls="sref76-1">Weinstein et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref76-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">76.</div><div>Weinstein, J.N. ∙ Collisson, E.A. ∙ Mills, G.B. ..., Cancer Genome Atlas Research Network</div><div><strong>The Cancer Genome Atlas Pan-Cancer analysis project</strong></div><div><em>Nat. Genet.</em> 2013; <strong>45</strong>:1113-1120</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.N.WeinsteinE.A.CollissonG.B.MillsK.R.ShawB.A.OzenbergerK.EllrottI.ShmulevichC.SanderJ.M.StuartCancer+Genome+Atlas+Research+NetworkThe+Cancer+Genome+Atlas+Pan-Cancer+analysis+projectNat.+Genet.45201311131120" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref74-1" href="#" role="doc-biblioref" data-xml-rid="bib74" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref74-1" aria-controls="sref74-1">Vivian et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref74-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">74.</div><div>Vivian, J. ∙ Rao, A.A. ∙ Nothaft, F.A. ...</div><div><strong>Toil enables reproducible, open source, big biomedical data analyses</strong></div><div><em>Nat. Biotechnol.</em> 2017; <strong>35</strong>:314-316</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.VivianA.A.RaoF.A.NothaftC.KetchumJ.ArmstrongA.NovakJ.PfeilJ.NarkizianA.D.DeranA.Musselman-BrownToil+enables+reproducible%2C+open+source%2C+big+biomedical+data+analysesNat.+Biotechnol.352017314316" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), noting reduced levels of immune-related gene expression in other epithelial tumors with high basal/squamous gene expression (<a id="crosref1695" href="#figs6">Figure&nbsp;S6</a>P [cluster 4]). Together, these findings suggest coordination between cancer cell states and the local TME, with decreased immune cell diversity in basal contexts (<a id="crosref1700" href="#fig5">Figure&nbsp;5</a>J).</div></section><section id="sec-2-10"><h3>State-specific TME signals drive transcriptional heterogeneity and drug response</h3><div id="p0180" role="paragraph">Based on these observations, we hypothesized that soluble factors specific to the TME of each transcriptional subtype may drive cancer cell states and potentially influence their therapeutic responses (<a id="crosref1705" href="#fig6">Figure&nbsp;6</a>A). <i>In&nbsp;vivo</i>, the secreted factor milieu surrounding cancer cells originates from at least two sources: malignant cells themselves (“autocrine” factors) and non-malignant cells (“paracrine” factors) (<a id="crosref1710" href="#fig6">Figure&nbsp;6</a>A). We first identified secreted factors differentially expressed by cancer cells in each transcriptional state (“autocrine” signals) and applied these to rescue IC and scBasal expression in <i>ex&nbsp;vivo</i> models (<a id="crosref1715" href="#fig6">Figure&nbsp;6</a>B; <a id="crosref1720" href="#mmc6">Table S6</a>; <a id="crosref1725" href="#sec-4">STAR Methods</a>). <i>TGFB2</i> was the top differentially expressed factor shared by malignant cells in both scBasal and IC TMEs (<a id="crosref1730" href="#fig6">Figure&nbsp;6</a>B), and organoids cultured with TGF-β ligands exhibited a pronounced shift toward IC and scBasal states (<a id="crosref1735" href="#fig6">Figure&nbsp;6</a>C). The reemergence of scBasal transcriptional heterogeneity in both minimal media (<a id="crosref1740" href="#fig5">Figure&nbsp;5</a>C) and TGF-β conditions (<a id="crosref1745" href="#fig6">Figure&nbsp;6</a>C) suggested that different types of microenvironmental pressure can lead to the basal-like cell state. These experiments also indicate that culture conditions can be tuned to achieve compositional differences spanning scClassical, heterogenous, and scBasal expression, akin to those seen <i>in&nbsp;vivo</i>.</div><div class="figure-wrap"><figure id="fig6" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.11.017/asset/ca83b174-1ae6-4c2e-8950-72044d3154b8/main.assets/gr6_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr6" src="/cms/10.1016/j.cell.2021.11.017/asset/71cb8572-f96b-47b2-a4c8-7757280fbbf4/main.assets/gr6.jpg" height="835" width="657" aria-labelledby="fig6-title fig6-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.017/asset/ca83b174-1ae6-4c2e-8950-72044d3154b8/main.assets/gr6_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig6-title" class="figure__title"><span class="label figure__label">Figure&nbsp;6</span> <span class="figure__title__text">Tumor state-specific factors rescue malignant transcriptional heterogeneity and reveal state-specific drug sensitivities</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig6-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0240" role="paragraph">(A) Schematic describing microenvironmental inputs to tumor cell state <i>in&nbsp;vivo</i> (left, “Metastatic environment”) versus <i>ex&nbsp;vivo</i> (center, “Organoid environment”) and a strategy to recover malignant transcriptional heterogeneity <i>ex&nbsp;vivo</i> (right, “State-supportive environment”).</div><div id="fspara0245" role="paragraph">(B) Differential expression (Wilcoxon rank sum test) of secreted factors between <i>in&nbsp;vivo</i> tumor cells scored as scBasal versus scClassical (x axis) and IC malignant cells versus the rest (y axis). State-specific genes that pass significance after multiple hypothesis correction (p&nbsp;&lt; <i>0.05</i>) are colored by their group association.</div><div id="fspara0250" role="paragraph">(C and D) Cell state diagrams with marginal density plots (C) and growth rate-adjusted dose response curves to gemcitabine and SN-38 (D) for organoid model PANFR0562 cultured for 3&nbsp;weeks in control medium (OWRNA) or in control medium with TGF-β. <i>P</i>-values in (C) for group differences between B/C commitment (top) and IC scores (right) were calculated by ANOVA followed by Tukey’s HSD. In (D), points are the mean ± SD of 3 technical replicates. Curves are representative of 2 independent experiments.</div><div id="fspara0255" role="paragraph">(E) Cell state diagram time series for PANFR0562 organoids cultured with TGF-β or after TGF-β removal.</div><div id="fspara0260" role="paragraph">(F) Growth rate-adjusted dose response curves to gemcitabine and paclitaxel for models in (E). Points are the mean ± SD of 3 technical replicates.</div><div id="fspara0265" role="paragraph">(G) State-specific drug sensitivities in isogenic organoid model pairs skewed toward scBasal or scClassical states by altering media formulation. Points are the mean ± SEM of 2–6 biologic replicates for the difference in growth rate-corrected Area Over the Curve (AOC) between each scBasal-scClassical model pair.</div><div id="fspara0270" role="paragraph">See also <a id="crosref2395" href="#figs7">Figure&nbsp;S7</a>; <a id="crosref2400" href="#mmc5">Tables S5</a>, <a id="crosref2405" href="#mmc6">S6</a>, and <a id="crosref2410" href="#mmc7">S7</a>.</div></div></div></div></figcaption></figure></div><div id="p0185" role="paragraph">We next assessed whether TME signals, like media formulation (<a id="crosref1750" href="#fig4">Figure&nbsp;4</a>F), could influence drug sensitivity through altering transcriptional state. In an isogenic organoid system (<a id="crosref1755" href="#sec-4">STAR Methods</a>), models induced to adopt scBasal expression through exposure to TGF-β for 3&nbsp;weeks were less sensitive to several standard-of-care chemotherapeutic agents including gemcitabine, paclitaxel, and SN-38, the active analog of irinotecan (<a id="crosref1760" href="#fig6">Figures 6</a>D, <a id="crosref1765" href="#figs7">S7</a>A, and S7B; <a id="crosref1770" href="#mmc7">Table S7</a>). The duration of exposure to TGF-β corresponded with the degree of state shift, and these states were highly plastic, as withdrawal of TGF-β resulted in a return to scClassical expression (<a id="crosref1775" href="#fig6">Figure&nbsp;6</a>E). Drug sensitivity tracked with cancer cell state, as models were re-sensitized to chemotherapeutic agents upon shifting back to the scClassical state (<a id="crosref1780" href="#fig6">Figures 6</a>E and 6F). These observations agree with recent clinical trial data showing that patients with basal phenotypes tend to have poorer outcomes with combination chemotherapy (<span class="dropBlock reference-citations"><a id="body-ref-sref4-2" href="#" role="doc-biblioref" data-xml-rid="bib4" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref4-2" aria-controls="sref4-2">Aung et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref4-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">4.</div><div>Aung, K.L. ∙ Fischer, S.E. ∙ Denroche, R.E. ...</div><div><strong>Genomics-Driven Precision Medicine for Advanced Pancreatic Cancer: Early Results from the COMPASS Trial</strong></div><div><em>Clin. Cancer Res.</em> 2018; <strong>24</strong>:1344-1354</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=K.L.AungS.E.FischerR.E.DenrocheG.H.JangA.DoddS.CreightonB.SouthwoodS.B.LiangD.ChadwickA.ZhangGenomics-Driven+Precision+Medicine+for+Advanced+Pancreatic+Cancer%3A+Early+Results+from+the+COMPASS+TrialClin.+Cancer+Res.24201813441354" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref5-5" href="#" role="doc-biblioref" data-xml-rid="bib5" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref5-5" aria-controls="sref5-5">Bailey et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref5-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">5.</div><div>Bailey, P. ∙ Chang, D.K. ∙ Nones, K. ..., Australian Pancreatic Cancer Genome Initiative</div><div><strong>Genomic analyses identify molecular subtypes of pancreatic cancer</strong></div><div><em>Nature.</em> 2016; <strong>531</strong>:47-52</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=P.BaileyD.K.ChangK.NonesA.L.JohnsA.M.PatchM.C.GingrasD.K.MillerA.N.ChristT.J.BruxnerM.C.QuinnAustralian+Pancreatic+Cancer+Genome+InitiativeGenomic+analyses+identify+molecular+subtypes+of+pancreatic+cancerNature53120164752" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref17-3" href="#" role="doc-biblioref" data-xml-rid="bib17" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref17-3" aria-controls="sref17-3">Collisson et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref17-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">17.</div><div>Collisson, E.A. ∙ Sadanandam, A. ∙ Olson, P. ...</div><div><strong>Subtypes of pancreatic ductal adenocarcinoma and their differing responses to therapy</strong></div><div><em>Nat. Med.</em> 2011; <strong>17</strong>:500-503</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=E.A.CollissonA.SadanandamP.OlsonW.J.GibbM.TruittS.GuJ.CoocJ.WeinkleG.E.KimL.JakkulaSubtypes+of+pancreatic+ductal+adenocarcinoma+and+their+differing+responses+to+therapyNat.+Med.172011500503" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref19-2" href="#" role="doc-biblioref" data-xml-rid="bib19" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref19-2" aria-controls="sref19-2">Connor et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref19-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">19.</div><div>Connor, A.A. ∙ Denroche, R.E. ∙ Jang, G.H. ...</div><div><strong>Integration of Genomic and Transcriptional Features in Pancreatic Cancer Reveals Increased Cell Cycle Progression in Metastases</strong></div><div><em>Cancer Cell.</em> 2019; <strong>35</strong>:267-282 e267</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.A.ConnorR.E.DenrocheG.H.JangM.LemireA.ZhangM.Chan-Seng-YueG.WilsonR.C.GrantD.MericoI.LunguIntegration+of+Genomic+and+Transcriptional+Features+in+Pancreatic+Cancer+Reveals+Increased+Cell+Cycle+Progression+in+MetastasesCancer+Cell352019267282+e267" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref50-3" href="#" role="doc-biblioref" data-xml-rid="bib50" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref50-3" aria-controls="sref50-3">Moffitt et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref50-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">50.</div><div>Moffitt, R.A. ∙ Marayati, R. ∙ Flate, E.L. ...</div><div><strong>Virtual microdissection identifies distinct tumor- and stroma-specific subtypes of pancreatic ductal adenocarcinoma</strong></div><div><em>Nat. Genet.</em> 2015; <strong>47</strong>:1168-1178</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=R.A.MoffittR.MarayatiE.L.FlateK.E.VolmarS.G.LoezaK.A.HoadleyN.U.RashidL.A.WilliamsS.C.EatonA.H.ChungVirtual+microdissection+identifies+distinct+tumor-+and+stroma-specific+subtypes+of+pancreatic+ductal+adenocarcinomaNat.+Genet.47201511681178" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Together, these findings highlight the remarkable phenotypic and functional plasticity inherent in tumor models.</div><div class="figure-wrap"><figure id="figs7" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.11.017/asset/655e8721-3c4e-4060-8191-e6bc1c5c2962/main.assets/figs7_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs7" src="/cms/10.1016/j.cell.2021.11.017/asset/269bb558-7e08-4019-b6fd-8dc4c8727d05/main.assets/figs7.jpg" height="941" width="720" aria-labelledby="figs7-title figs7-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.017/asset/655e8721-3c4e-4060-8191-e6bc1c5c2962/main.assets/figs7_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs7-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S7</span> <span class="figure__title__text">State-specific drug responses and TME-associated state shifts in organoid models and patient samples, related to <a id="crosref2500" href="#fig6">Figures 6</a> and <a id="crosref2505" href="#fig7">7</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs7-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0660" role="paragraph">(A) Independent experiments demonstrating state-specific drug sensitivities in isogenic PANFR0562 organoid model pairs skewed toward scBasal or scClassical states by altering media formulation (scBasal, TGF-β or Minimal media; scClassical, OWRNA media) for different time periods. Bars represent the difference in growth rate-corrected Area Over the Curve (AOC) between each scBasal-scClassical model pair for each independent experiment depicted in <a id="crosref2510" href="#fig6">Figure&nbsp;6</a>G.</div><div id="fspara0665" role="paragraph">(B) Growth rate-adjusted dose response curves for isogenic PANFR0562 organoid model pairs in different media conditions and at different time points. Points are mean ± SD of 3 technical replicates. Orange curves represent responses in more scBasal conditions and blue curves represent responses in more scClassical conditions.</div><div id="fspara0670" role="paragraph">(C) Cell state diagrams with marginal density plots for the organoid model PANFR0489 depicted in <a id="crosref2515" href="#fig7">Figure&nbsp;7</a>C in control medium (OWRNA, reduced organoid medium) or control medium with IFNγ. <i>P</i>-values for group differences between B/C commitment (top) and IC scores (right) were calculated by ANOVA followed by Tukey’s HSD.</div><div id="fspara0675" role="paragraph">(D) Cell state diagrams with marginal density plots for organoid model PANFR0562 at 2 time points with exposure to IFNγ. <i>P</i>-values for group differences between B/C commitment (top) and IC scores (right) were calculated by ANOVA followed by Tukey’s HSD.</div><div id="fspara0680" role="paragraph">(E) <i>t</i>-SNE visualization of single cells from metastatic biopsies collected from the same patient (PANFR0473, <a id="crosref2520" href="#fig7">Figure&nbsp;7</a>D) but at two different sites (liver, left; lung, right). All cells from both samples are shown in both <i>t</i>-SNE plots, but cells for each site are labeled red in the corresponding plot.</div><div id="fspara0685" role="paragraph">(F) Cell state diagrams with marginal density plots (top) and expression heatmaps (bottom) depict cell state differences between malignant cells from two distinct metastatic sites for the samples (PANFR0473) in <a id="crosref2525" href="#fig7">Figure&nbsp;7</a>D and S7E. <i>P</i>-values for group differences between B/C commitment (top) and IC scores (right) were calculated by ANOVA followed by Tukey’s HSD.</div><div id="fspara0690" role="paragraph">(G) <i>t</i>-SNE visualization of single cells from metastatic biopsies collected from the same site in one patient (PANFR0489, <a id="crosref2530" href="#fig7">Figure&nbsp;7</a>E) pre- and post-immunotherapy. All cells from both samples are shown in both <i>t</i>-SNE plots, but cells for each time point are labeled red in the corresponding plot.</div><div id="fspara0695" role="paragraph">(H) Cell state diagrams with marginal density plots from metastatic samples collected pre- and post-immunotherapy (PANFR0489, <a id="crosref2535" href="#fig7">Figure&nbsp;7</a>E,F and S7G). <i>P</i>-values for group differences between B/C commitment (top) and IC scores (right) were calculated by ANOVA followed by Tukey’s HSD.</div></div></div></div></figcaption></figure></div><div id="p0190" role="paragraph">To assess for state-specific vulnerabilities more broadly, we tested a 24-drug panel in isogenic organoid systems with the scBasal state induced via two distinct routes, either by culturing in minimal or TGF-β-containing medium (<a id="crosref1810" href="#fig6">Figure&nbsp;6</a>G; <a id="crosref1815" href="#mmc7">Table S7</a>). Within the same model, scClassical states were on average more sensitive to both chemotherapy and agents targeting DNA-damage repair pathways while scBasal states were more sensitive to mitogen-activated protein kinase (MAPK) pathway inhibitors (MEK and ERK inhibitors) (<a id="crosref1820" href="#fig6">Figures 6</a>G, <a id="crosref1825" href="#figs7">S7</a>A, and S7B). These findings provide direct evidence that transcriptional state can be a major determinant of drug response and that differential targeting of cell states represents an actionable therapeutic paradigm.</div></section><section id="sec-2-11"><h3>Paracrine signals from the local TME direct cancer cell transcriptional phenotypes</h3><div id="p0195" role="paragraph">We next searched for paracrine factors differentially expressed by the non-malignant cells in each subtype. We noted an increasing number of differentially expressed factors in the IC and scBasal contexts and mapped each paracrine factor to its cognate cell type to summarize the overall secreted factor combinations that shape subtype-specific TMEs in metastatic PDAC (<a id="crosref1830" href="#fig7">Figures 7</a>A and 7B; <a id="crosref1835" href="#mmc6">Table S6</a>; <a id="crosref1840" href="#sec-4">STAR Methods</a>). Interestingly, <i>IFNG</i> from <i>CD8+</i> T&nbsp;cells was most highly expressed in the IC TME, consistent with a higher T&nbsp;cell fraction in IC tumors (<a id="crosref1845" href="#fig5">Figures 5</a>G and <a id="crosref1850" href="#figs6">S6</a>O) and the relative increase in IFN-responsive gene expression in IC and scBasal malignant cells (<a id="crosref1855" href="#fig4">Figures 4</a>G and 4H). Correspondingly, cells from organoid models exposed to interferon γ (IFNγ) showed increased IFN response gene expression (IFN response score) and a concomitant shift toward the IC state (<a id="crosref1860" href="#fig7">Figures 7</a>C, <a id="crosref1865" href="#figs7">S7</a>C, and S7D).</div><div class="figure-wrap"><figure id="fig7" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.11.017/asset/2742d5fd-de46-4740-9ef0-97a2f9650535/main.assets/gr7_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr7" src="/cms/10.1016/j.cell.2021.11.017/asset/6403b143-9879-49e3-ad61-7a7601d3a21e/main.assets/gr7.jpg" height="917" width="657" aria-labelledby="fig7-title fig7-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.017/asset/2742d5fd-de46-4740-9ef0-97a2f9650535/main.assets/gr7_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig7-title" class="figure__title"><span class="label figure__label">Figure&nbsp;7</span> <span class="figure__title__text">Malignant transcriptional states respond to TME alterations in organoid models and <i>in&nbsp;vivo</i></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig7-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0280" role="paragraph">(A) Differential expression (Wilcoxon rank sum test) of secreted factors by non-malignant cells (paracrine) grouped by their sample-averaged malignant cell expression state in scBasal and scClassical (x axis) tumors and IC biopsies and the rest (y axis). State-specific genes that pass significance after multiple hypothesis correction (p&nbsp;&lt; <i>0.05</i>) are colored by their group association.</div><div id="fspara0285" role="paragraph">(B) Dot plot for state-specific significant differentially expressed paracrine factors (rows) by subtype-specific non-malignant cell types (columns). Dot size represents that cell type’s fraction within tumors of each subtype, and fill color indicates average expression. Only cell types with a fractional representation &gt; 5% from each subtype are visualized.</div><div id="fspara0290" role="paragraph">(C) Density plots of IFN response score (top) and IC score (bottom) in control organoid cells and after addition of IFNγ for 6&nbsp;days. <i>P</i>-values were calculated by Student’s t test.</div><div id="fspara0295" role="paragraph">(D) Biopsy samples from distinct metastatic sites (liver, dark gray; lung, light gray) in the same patient (PANFR0473) demonstrate co-variation in T&nbsp;cell IFNG expression (top), malignant cell IFN response score (middle), and malignant IC score (bottom). <i>P</i>-values for density plot differences were calculated by Student’s t test.</div><div id="fspara0300" role="paragraph">(E) Biopsy samples from the same lesion pre- and post-immunotherapy (checkpoint inhibitor plus a macrophage-targeting agent; pre-, PANFR0489, pink; post-, PANFR0489R, blue) demonstrate coordinated changes with treatment in T&nbsp;cell IFNG expression (top), malignant cell IFN response score (middle), and malignant IC score (bottom). <i>P</i>-values for density plot differences were calculated by Student’s t test.</div><div id="fspara0305" role="paragraph">(F) Heatmap for malignant cell state shifts from samples in (E).</div><div id="fspara0310" role="paragraph">See also <a id="crosref2415" href="#figs7">Figure&nbsp;S7</a>; <a id="crosref2420" href="#mmc5">Tables S5</a>, <a id="crosref2425" href="#mmc6">S6</a>, and <a id="crosref2430" href="#mmc7">S7</a>.</div></div></div></div></figcaption></figure></div><div id="p0200" role="paragraph">We next examined two <i>in&nbsp;vivo</i> scenarios for evidence that the TME can drive cell state variation by analyzing samples from (1) distinct metastatic sites within the same patient and (2) the same metastatic site before and after immunotherapy. In the first case (PANRF0473), a larger fraction of T&nbsp;cells within the liver metastatic niche expressed high levels of <i>IFNG</i> (IFNG expression score) relative to the lung metastatic niche. Correspondingly, malignant cells in the liver showed evidence of IFN response and higher IC state expression (<a id="crosref1870" href="#fig7">Figure&nbsp;7</a>D, S7E,F). Similarly, in serial samples from the same liver lesion in patient PANFR0489, the post-progression biopsy harbored a higher fraction of T&nbsp;cells with high IFNG expression scores, consistent with changes to the TME stemming from immunotherapy. Post-progression malignant cells again expressed higher IFN response scores, and their transcriptional phenotypes were shifted toward the IC and scBasal states relative to the pre-treatment malignant cells (<a id="crosref1875" href="#fig7">Figures 7</a>E, 7F, <a id="crosref1880" href="#figs7">S7</a>G, and S7H). These paired biopsies allow a window into the complex interplay between the TME and cancer cells within the same patient and, consistent with our organoid studies, illustrate the critical role that TME-supplied cytokines and immune activation may play in directing cancer cell state.</div></section></section><section id="sec-3"><h2>Discussion</h2><div id="p0205" role="paragraph">Single-cell atlases of cancer have revolutionized our understanding of human malignancies and revealed that mutational and transcriptional heterogeneity are common. Critical next steps include understanding what drives cancer cell state and whether it can be targeted therapeutically. Here, we provide a systematic framework for assessing cancer cell states, identifying drivers of transcriptional plasticity, and evaluating their functional significance in model systems. The use of matched <i>in&nbsp;vivo</i> tissue as a reference for culture model state fidelity, an approach that has been applied in normal organoid systems (<span class="dropBlock reference-citations"><a id="body-ref-sref25" href="#" role="doc-biblioref" data-xml-rid="bib25" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref25" aria-controls="sref25">Fujii et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref25"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">25.</div><div>Fujii, M. ∙ Matano, M. ∙ Toshimitsu, K. ...</div><div><strong>Human Intestinal Organoids Maintain Self-Renewal Capacity and Cellular Diversity in Niche-Inspired Culture Condition</strong></div><div><em>Cell Stem Cell.</em> 2018; <strong>23</strong>:787-793 e786</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.FujiiM.MatanoK.ToshimitsuA.TakanoY.MikamiS.NishikoriS.SugimotoT.SatoHuman+Intestinal+Organoids+Maintain+Self-Renewal+Capacity+and+Cellular+Diversity+in+Niche-Inspired+Culture+ConditionCell+Stem+Cell232018787793+e786" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) but not in cancer, enabled a structured dissection of cell-intrinsic versus TME-induced contributions to malignant cell state. As the community continues to catalog these transcriptional states in clinical samples at single-cell resolution, we anticipate this framework will be broadly applicable for understanding their functional significance across a variety of cancers.</div><div id="p0210" role="paragraph">Single-cell resolution enabled us to appreciate the layering of malignant states whereby single cells can be unified in their similarity for an expression subtype (e.g., scBasal or IC) but differ in their expression of TME-influenced programs (e.g., IFN response). Future studies across more patients and different anatomic disease sites at single-cell resolution will be needed to fully parse which invariant “core” genes mark archetypal cell states in PDAC and which expression features are superimposed by the TME. Clinically annotated datasets will also aid in the assessment of these expression states for prognostic value and their utility in nominating cell state-specific therapeutic liabilities.</div><div id="p0215" role="paragraph">We also uncovered formerly unappreciated relationships between cancer cell transcriptional states and the local TME. Similar to the relationship between inflammation and tumorigenesis (<span class="dropBlock reference-citations"><a id="body-ref-sref3" href="#" role="doc-biblioref" data-xml-rid="bib3" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref3" aria-controls="sref3">Alonso-Curbelo et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">3.</div><div>Alonso-Curbelo, D. ∙ Ho, Y.J. ∙ Burdziak, C. ...</div><div><strong>A gene-environment-induced epigenetic program initiates tumorigenesis</strong></div><div><em>Nature.</em> 2021; <strong>590</strong>:642-648</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=D.Alonso-CurbeloY.J.HoC.BurdziakJ.L.V.MaagJ.P.Morris4thR.ChandwaniH.A.ChenK.M.TsanovF.M.BarrigaW.LuanA+gene-environment-induced+epigenetic+program+initiates+tumorigenesisNature5902021642648" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref45" href="#" role="doc-biblioref" data-xml-rid="bib45" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref45" aria-controls="sref45">Li et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref45"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">45.</div><div>Li, Y. ∙ He, Y. ∙ Peng, J. ...</div><div><strong>Mutant Kras co-opts a proto-oncogenic enhancer network in inflammation-induced metaplastic progenitor cells to initiate pancreatic cancer</strong></div><div><em>Nature Cancer.</em> 2021; <strong>2</strong>:49-65</div></div><div class="external-links"><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/35121887/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?pmid=35121887" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), our data support a model wherein as tumors become inflamed and immune-activated, malignant cells display enhanced state plasticity. These relationships may have implications for PDAC immunotherapy strategies given that a productive CD8 T cell response may promote more aggressive basal-like states, as suggested by our paired pre- and post-immunotherapy samples. Coordination between basal/mesenchymal malignant cell states and immune responses may be a broadly relevant phenomenon given our observations in other basal-like tumors and recent work in glioblastoma (<span class="dropBlock reference-citations"><a id="body-ref-sref35" href="#" role="doc-biblioref" data-xml-rid="bib35" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref35" aria-controls="sref35">Hara et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref35"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">35.</div><div>Hara, T. ∙ Chanoch-Myers, R. ∙ Mathewson, N.D. ...</div><div><strong>Interactions between cancer cells and immune cells drive transitions to mesenchymal-like states in glioblastoma</strong></div><div><em>Cancer Cell.</em> 2021; <strong>39</strong>:779-792 e711</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=T.HaraR.Chanoch-MyersN.D.MathewsonC.MyskiwL.AttaL.BussemaS.W.EichhornA.C.GreenwaldG.S.KinkerC.RodmanInteractions+between+cancer+cells+and+immune+cells+drive+transitions+to+mesenchymal-like+states+in+glioblastomaCancer+Cell392021779792+e711" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). A recent publication further supports the idea that local TME variation can influence clinical outcomes in PDAC (<span class="dropBlock reference-citations"><a id="body-ref-sref32-2" href="#" role="doc-biblioref" data-xml-rid="bib32" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref32-2" aria-controls="sref32-2">Grünwald et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref32-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">32.</div><div>Grünwald, B.T. ∙ Devisme, A. ∙ Andrieux, G. ...</div><div><strong>Spatially confined sub-tumor microenvironments in pancreatic cancer</strong></div><div><em>Cell.</em> 2021; <strong>184</strong>:5577-5592.e18</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=B.T.Gr%C3%BCnwaldA.DevismeG.AndrieuxF.VyasK.AliarC.W.McCloskeyA.MacklinG.H.JangR.DenrocheJ.M.RomeroSpatially+confined+sub-tumor+microenvironments+in+pancreatic+cancerCell184202155775592.e18" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), while our work provides direct evidence that TME signals are critical drivers of malignant cell state and that cell state dictates drug sensitivity. Larger cohorts of longitudinal single-cell measurements, both spatially resolved and transcriptome-wide, from individual patients undergoing therapy are needed to fully assess variation in the TME, the kinetics of cell state plasticity, and their consequences for therapeutic response.</div><div id="p0220" role="paragraph">While the genetic evolution of <i>ex&nbsp;vivo</i> models is an established phenomenon (<span class="dropBlock reference-citations"><a id="body-ref-sref9-2" href="#" role="doc-biblioref" data-xml-rid="bib9" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref9-2" aria-controls="sref9-2">Ben-David et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref9-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">9.</div><div>Ben-David, U. ∙ Beroukhim, R. ∙ Golub, T.R.</div><div><strong>Genomic evolution of cancer models: perils and opportunities</strong></div><div><em>Nat. Rev. Cancer.</em> 2019; <strong>19</strong>:97-109</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=U.Ben-DavidR.BeroukhimT.R.GolubGenomic+evolution+of+cancer+models%3A+perils+and+opportunitiesNat.+Rev.+Cancer19201997109" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref8" href="#" role="doc-biblioref" data-xml-rid="bib8" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref8" aria-controls="sref8">Ben-David et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref8"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">8.</div><div>Ben-David, U. ∙ Siranosian, B. ∙ Ha, G. ...</div><div><strong>Genetic and transcriptional evolution alters cancer cell line drug response</strong></div><div><em>Nature.</em> 2018; <strong>560</strong>:325-330</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=U.Ben-DavidB.SiranosianG.HaH.TangY.OrenK.HinoharaC.A.StrathdeeJ.DempsterN.J.LyonsR.BurnsGenetic+and+transcriptional+evolution+alters+cancer+cell+line+drug+responseNature5602018325330" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), our study shows similar <i>ex&nbsp;vivo</i> evolution along a transcriptional axis that causes discordance between a patient sample and its matched avatar. For example, biopsies taken from PANFR0489R and PANFR0576 establish long-term cultures <i>ex&nbsp;vivo</i>, but they do not represent the dominant cell state of the corresponding patient tissue <i>in&nbsp;vivo</i> (<a id="crosref1920" href="#fig3">Figure&nbsp;3</a>D). Given our demonstrated link between cell state and therapeutic response, these findings establish the necessity for preserving transcriptional fidelity in personalized medicine pipelines (<span class="dropBlock reference-citations"><a id="body-ref-sref34-2" href="#" role="doc-biblioref" data-xml-rid="bib34" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref34-2" aria-controls="sref34-2">Hahn et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref34-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">34.</div><div>Hahn, W.C. ∙ Bader, J.S. ∙ Braun, T.P. ..., Cancer Target Discovery and Development Network</div><div><strong>An expanded universe of cancer targets</strong></div><div><em>Cell.</em> 2021; <strong>184</strong>:1142-1155</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=W.C.HahnJ.S.BaderT.P.BraunA.CalifanoP.A.ClemonsB.J.DrukerA.J.EwaldH.FuS.JaguC.J.KempCancer+Target+Discovery+and+Development+NetworkAn+expanded+universe+of+cancer+targetsCell184202111421155" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Since models from different patients may also harbor differential plasticity and adaptability, future efforts will need to evaluate models in multiple conditions to account for this latent property, define its drivers (e.g., genetics, epigenetics), and accurately map vulnerabilities. Importantly, manipulating the soluble microenvironment may offer a more tractable approach for state-specific high-throughput screening compared to more complex heterotypic co-cultures or patient-derived xenograft systems.</div><div id="p0225" role="paragraph">Within the context of PDAC, clinical studies are ongoing to evaluate the efficacy of gemcitabine/nab-paclitaxel and FOLFIRINOX in patients with basal- versus classical-predominant metastatic disease (e.g., PASS01 – NCT04469556). Our observations suggest that basal tumors may exhibit broadly decreased sensitivity to chemotherapy and highlight the need for new strategies to target this transcriptional subtype of PDAC. Importantly, we show that the transition to an scBasal state may render cells sensitive to other classes of inhibitors. Since most PDAC tumors are heterogenous, combination strategies that suppress distinct cell states may be necessary for maximal synergistic effect (<span class="dropBlock reference-citations"><a id="body-ref-sref56" href="#" role="doc-biblioref" data-xml-rid="bib56" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref56" aria-controls="sref56">Palmer et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref56"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">56.</div><div>Palmer, A.C. ∙ Chidley, C. ∙ Sorger, P.K.</div><div><strong>A curative combination cancer therapy achieves high fractional cell killing through low cross-resistance and drug additivity</strong></div><div><em>eLife.</em> 2019; <strong>8</strong>:e50036</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.C.PalmerC.ChidleyP.K.SorgerA+curative+combination+cancer+therapy+achieves+high+fractional+cell+killing+through+low+cross-resistance+and+drug+additivityeLife82019e50036" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref55" href="#" role="doc-biblioref" data-xml-rid="bib55" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref55" aria-controls="sref55">Palmer and Sorger, 2017</a><div class="dropBlock__holder" data-db-target-of="sref55"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">55.</div><div>Palmer, A.C. ∙ Sorger, P.K.</div><div><strong>Combination Cancer Therapy Can Confer Benefit via Patient-to-Patient Variability without Drug Additivity or Synergy</strong></div><div><em>Cell.</em> 2017; <strong>171</strong>:1678-1691 e1613</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.C.PalmerP.K.SorgerCombination+Cancer+Therapy+Can+Confer+Benefit+via+Patient-to-Patient+Variability+without+Drug+Additivity+or+SynergyCell171201716781691+e1613" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Furthermore, the ability of the TME to drive malignant cell state transitions suggests that next generation therapeutic strategies may also need to target site-specific supportive cells in the TME to control cell-state evolution during therapy.</div><div id="p0230" role="paragraph">In sum, we provide a widely applicable framework to benchmark cell states in patient-derived model systems, identify the drivers of malignant transcriptional heterogeneity, and examine the functional significance of cell state. As efforts to characterize cell states across malignancies provide new and increasingly higher resolution maps of patient tissues, an important next step will be to understand better how to control and target cell state. We anticipate that our approach will provide a path toward the systematic evaluation of cell state as a targetable feature in cancer.</div><section id="sec-3-1"><h3>Limitations of the study</h3><div id="p0235" role="paragraph">We focused on understanding how the TME supports cell state, and thus we cannot comment directly on the epigenetic mechanisms through which these cell state transitions occur. Additional studies into the regulatory mechanisms underlying PDAC state transitions will be a critical next step in further delineating the relationships between cell-intrinsic and extrinsic factors and state. In addition, we tracked CNV alterations to define genetic clones, but future studies using barcode-based lineage tracing approaches are needed to confirm whether individual cells transition between scClassical and scBasal states via the IC state as our data suggest (<span class="dropBlock reference-citations"><a id="body-ref-sref75" href="#" role="doc-biblioref" data-xml-rid="bib75" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref75" aria-controls="sref75">Wagner and Klein, 2020</a><div class="dropBlock__holder" data-db-target-of="sref75"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">75.</div><div>Wagner, D.E. ∙ Klein, A.M.</div><div><strong>Lineage tracing meets single-cell omics: opportunities and challenges</strong></div><div><em>Nat. Rev. Genet.</em> 2020; <strong>21</strong>:410-427</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=D.E.WagnerA.M.KleinLineage+tracing+meets+single-cell+omics%3A+opportunities+and+challengesNat.+Rev.+Genet.212020410427" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). While we focused predominantly on metastatic biopsies from the liver, it will be important in future studies to more broadly analyze how site-specific cues drive state plasticity. Even with our limited number of samples from outside the liver, we uncovered site-specific differences in mesenchymal populations which revealed important differences between metastatic and primary disease. Given the pivotal role that has been suggested for the fibrotic TME in primary disease (<span class="dropBlock reference-citations"><a id="body-ref-sref37" href="#" role="doc-biblioref" data-xml-rid="bib37" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref37" aria-controls="sref37">Ho et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref37"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">37.</div><div>Ho, W.J. ∙ Jaffee, E.M. ∙ Zheng, L.</div><div><strong>The tumour microenvironment in pancreatic cancer - clinical challenges and opportunities</strong></div><div><em>Nat. Rev. Clin. Oncol.</em> 2020; <strong>17</strong>:527-540</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=W.J.HoE.M.JaffeeL.ZhengThe+tumour+microenvironment+in+pancreatic+cancer+-+clinical+challenges+and+opportunitiesNat.+Rev.+Clin.+Oncol.172020527540" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref63" href="#" role="doc-biblioref" data-xml-rid="bib63" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref63" aria-controls="sref63">Sahai et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref63"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">63.</div><div>Sahai, E. ∙ Astsaturov, I. ∙ Cukierman, E. ...</div><div><strong>A framework for advancing our understanding of cancer-associated fibroblasts</strong></div><div><em>Nat. Rev. Cancer.</em> 2020; <strong>20</strong>:174-186</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=E.SahaiI.AstsaturovE.CukiermanD.G.DeNardoM.EgebladR.M.EvansD.FearonF.R.GretenS.R.HingoraniT.HunterA+framework+for+advancing+our+understanding+of+cancer-associated+fibroblastsNat.+Rev.+Cancer202020174186" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), these findings carry important implications for targeting the stromal compartment in primary versus metastatic PDAC. Future studies with larger sample and cell numbers will be needed to make comparisons across cell types in primary and metastatic disease to fully understand how the TME regulates transcriptional phenotypes in these distinct niches. Here, advances in spatial transcriptomics will likely empower a deeper understanding of these relationships across compartments (<span class="dropBlock reference-citations"><a id="body-ref-sref47" href="#" role="doc-biblioref" data-xml-rid="bib47" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref47" aria-controls="sref47">Longo et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref47"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">47.</div><div>Longo, S.K. ∙ Guo, M.G. ∙ Ji, A.L. ...</div><div><strong>Integrating single-cell and spatial transcriptomics to elucidate intercellular tissue dynamics</strong></div><div><em>Nat. Rev. Genet.</em> 2021; <strong>22</strong>:627-644</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=S.K.LongoM.G.GuoA.L.JiP.A.KhavariIntegrating+single-cell+and+spatial+transcriptomics+to+elucidate+intercellular+tissue+dynamicsNat.+Rev.+Genet.222021627644" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Finally, future studies that broadly map the landscape of RNA state-dependent drug response across various cancer models will be needed to fully define the links between malignant cell transcriptional plasticity and therapeutic response.</div></section></section><section id="sec-4"><h2>STAR★Methods</h2><section id="sec-4-1"><h3>Key resources table</h3><div id="p0255" role="paragraph"><div class="figure-wrap"><figure id="undtbl1" class="table" data-figureviewer-ignore="true"><div class="table-wrap"><table data-figureviewer-ignore="true" tabindex="0"><thead><tr class="rowsep"><th>REAGENT or RESOURCE</th><th>SOURCE</th><th>IDENTIFIER</th></tr></thead><tbody><tr class="rowsep"><td colspan="3"><b>Antibodies</b></td></tr><tr><td>Mouse monoclonal anti-cytokeratin 17, clone E3</td><td>Thermo Fisher</td><td>Cat# MA5-13539; RRID:<a id="intref0015" href="https://antibodyregistry.org/AB_10980102" target="_blank">AB_10980102</a></td></tr><tr><td>Rabbit monoclonal anti-S100A2, clone EPR5392</td><td>Abcam</td><td>Cat# ab109494; RRID:<a id="intref0020" href="https://antibodyregistry.org/AB_10859000" target="_blank">AB_10859000</a></td></tr><tr><td>Rabbit monoclonal anti-Claudin18.2, clone EPR19202-244</td><td>Abcam</td><td>Cat# ab241330</td></tr><tr><td>Rabbit monoclonal anti-GATA-6 XP, clone D61E4</td><td>Cell Signaling Technology</td><td>Cat# 5851; RRID:AB_10705521</td></tr><tr><td>Rabbit monoclonal anti-TFF1 (estrogen inducible protein pS2), clone EPR3972</td><td>Abcam</td><td>Cat# ab92377; RRID:AB_10562112</td></tr><tr><td>Mouse monoclonal anti-cytokeratin, clone AE1/AE3</td><td>Agilent/Dako</td><td>Cat# M3515; RRID:AB_2132885</td></tr><tr><td>Mouse monoclonal anti-pan-keratin, clone C11</td><td>Cell Signaling Technology</td><td>Cat# 4545; RRID:<a id="intref0025" href="https://antibodyregistry.org/AB_490860" target="_blank">AB_490860</a></td></tr><tr class="rowsep"><td>Opal polymer HRP anti-mouse and anti-rabbit secondary antibody</td><td>Akoya Biosciences</td><td>Cat# ARH1001EA; RRID:AB_2890927</td></tr><tr class="rowsep"><td colspan="3"><b>Biological samples</b></td></tr><tr class="rowsep"><td>Human PDAC samples</td><td>This study</td><td>N/A</td></tr><tr class="rowsep"><td colspan="3"><b>Chemicals, peptides, and recombinant proteins</b></td></tr><tr><td>Advanced DMEM/F12</td><td>Thermo Fisher</td><td>Cat# 12634028</td></tr><tr><td>RPMI 1640</td><td>Corning</td><td>Cat# 10-040-CV</td></tr><tr><td>Fetal bovine serum</td><td>Sigma</td><td>Cat# F4135</td></tr><tr><td>Penicillin/streptomycin</td><td>Thermo Fisher</td><td>Cat# 15140122</td></tr><tr><td>Primocin</td><td>Invivogen</td><td>Cat# ant-pm-1</td></tr><tr><td>HEPES</td><td>Thermo Fisher</td><td>Cat# 15630080</td></tr><tr><td>GlutaMAX</td><td>Thermo Fisher</td><td>Cat# 35050061</td></tr><tr><td>A83-01</td><td>Tocris</td><td>Cat# 2939</td></tr><tr><td>Recombinant mouse Noggin</td><td>Peprotech</td><td>Cat# 250-38E</td></tr><tr><td>Recombinant mouse EGF</td><td>Peprotech</td><td>Cat# 315-09</td></tr><tr><td>Recombinant human FGF10</td><td>Peprotech</td><td>Cat# 100-26</td></tr><tr><td>Human [Leu15]-Gastrin I</td><td>Sigma</td><td>Cat# G9145</td></tr><tr><td>N-acetylcysteine</td><td>Sigma</td><td>Cat# A9165</td></tr><tr><td>Nicotinamide</td><td>Sigma</td><td>Cat# N0636</td></tr><tr><td>B-27 supplement</td><td>Thermo Fisher</td><td>Cat# 17504044</td></tr><tr><td>Recombinant human TGF-β1</td><td>Peprotech</td><td>Cat# 100-21</td></tr><tr><td>Recombinant human IFNγ</td><td>Peprotech</td><td>Cat# 300-02</td></tr><tr><td>Growth factor reduced Matrigel</td><td>Corning</td><td>Cat# 356231</td></tr><tr><td>TrypLE Express</td><td>Thermo Fisher</td><td>Cat# 12604054</td></tr><tr><td>Trypsin-EDTA (0.25%)</td><td>Thermo Fisher</td><td>Cat# 25200056</td></tr><tr><td>CellTiterGlo 3D</td><td>Promega</td><td>Cat# G9683</td></tr><tr><td>Collagenase XI</td><td>Sigma</td><td>Cat# C7657</td></tr><tr><td>Dnase</td><td>StemCell Technologies</td><td>Cat# 07900</td></tr><tr><td>Y-27632</td><td>Selleck Chemicals</td><td>Cat# S1049</td></tr><tr><td>ACK lysing biffer</td><td>Thermo Fisher</td><td>Cat# A1049201</td></tr><tr><td>Trypan blue solution, 0.4%</td><td>Thermo Fisher</td><td>Cat# 15250061</td></tr><tr><td>Spectral DAPI</td><td>Akoya Biosciences</td><td>Cat# FP1490</td></tr><tr><td>Opal 520 reagent pack</td><td>Akoya Biosciences</td><td>Cat# FP1487001KT</td></tr><tr><td>Opal 540 reagent pack</td><td>Akoya Biosciences</td><td>Cat# FP1494001KT</td></tr><tr><td>Opal 570 reagent pack</td><td>Akoya Biosciences</td><td>Cat# FP1488001KT</td></tr><tr><td>Opal 620 reagent pack</td><td>Akoya Biosciences</td><td>Cat# FP1495001KT</td></tr><tr><td>Opal 650 reagent pack</td><td>Akoya Biosciences</td><td>Cat# FP1496001KT</td></tr><tr><td>Opal 690 reagent pack</td><td>Akoya Biosciences</td><td>Cat# FP1497001KT</td></tr><tr><td>Xylenes (histological)</td><td>Fisher Scientific, X3P1GAL</td><td>Cat# X3P-1GAL</td></tr><tr><td>BOND Epitope Retrieval Solution 1</td><td>Leica Biosystems</td><td>Cat# AR9961</td></tr><tr><td>BOND Epitope Retrieval Solution 2</td><td>Leica Biosystems</td><td>Cat# AR9640</td></tr><tr><td>Antibody diluent/block</td><td>Akoya Biosciences</td><td>Cat# ARD1001EA</td></tr><tr><td>1x Plus Automation Amplification Diluent</td><td>Akoya Biosciences</td><td>Cat# FP1609</td></tr><tr><td>ProLong Gold Antifade Mountant</td><td>Fisher Scientific/Molecular Probes</td><td>Cat# P36930</td></tr><tr><td>Gemcitabine</td><td>Selleck Chemicals</td><td>Cat# S1149</td></tr><tr><td>Paclitaxel</td><td>Selleck Chemicals</td><td>Cat# S1150</td></tr><tr><td>SN-38</td><td>Selleck Chemicals</td><td>Cat# S4908</td></tr><tr><td>Trametinib</td><td>Selleck Chemicals</td><td>Cat# S2673</td></tr><tr><td>5-FU</td><td>Selleck Chemicals</td><td>Cat# S1209</td></tr><tr><td>Afatinib</td><td>Selleck Chemicals</td><td>Cat# S1011</td></tr><tr><td>AZD6738 (Ceralasertib)</td><td>Selleck Chemicals</td><td>Cat# S7693</td></tr><tr><td>Binimetinib</td><td>Selleck Chemicals</td><td>Cat# S7007</td></tr><tr><td>BVD-523 (Ulixertinib)</td><td>Selleck Chemicals</td><td>Cat# S7854</td></tr><tr><td>Dinaciclib</td><td>Selleck Chemicals</td><td>Cat# S2768</td></tr><tr><td>Everolimus</td><td>Selleck Chemicals</td><td>Cat# S1120</td></tr><tr><td>Gedatolisib</td><td>Selleck Chemicals</td><td>Cat# S2628</td></tr><tr><td>GSK126</td><td>Selleck Chemicals</td><td>Cat# S7061</td></tr><tr><td>(+)-JQ1</td><td>Selleck Chemicals</td><td>Cat# S7110</td></tr><tr><td>LY3023414 (Samotolisib)</td><td>Selleck Chemicals</td><td>Cat# S8322</td></tr><tr><td>MK-1775 (Adavosertib)</td><td>Selleck Chemicals</td><td>Cat# S1525</td></tr><tr><td>Navitoclax</td><td>Selleck Chemicals</td><td>Cat# S1001</td></tr><tr><td>Olaparib</td><td>Selleck Chemicals</td><td>Cat# S1060</td></tr><tr><td>Oxaliplatin</td><td>Selleck Chemicals</td><td>Cat# S1224</td></tr><tr><td>Palbociclib</td><td>Selleck Chemicals</td><td>Cat# S4482</td></tr><tr><td>Prexasertib</td><td>Selleck Chemicals</td><td>Cat# S6385</td></tr><tr><td>SHP099</td><td>Selleck Chemicals</td><td>Cat# S8278</td></tr><tr><td>Tazemetostat</td><td>Selleck Chemicals</td><td>Cat# S7128</td></tr><tr><td>YKL-5-124</td><td>Selleck Chemicals</td><td>Cat# S8863</td></tr><tr><td>2-mercaptoethanol</td><td>Sigma</td><td>Cat# M3148</td></tr><tr><td>Buffer RLT</td><td>QIAGEN</td><td>Cat# 79216</td></tr><tr><td>Buffer RLT Plus</td><td>QIAGEN</td><td>Cat# 1053393</td></tr><tr><td>Deoxynucleotide (dNTP) solution mix</td><td>NewEngland BioLabs</td><td>Cat# N0447L</td></tr><tr><td>Superase.In RNase Inhibitor</td><td>Thermo Fisher</td><td>Cat# AM2696</td></tr><tr><td>Maxima H minus reverse transcriptase</td><td>Fisher Scientific</td><td>Cat# EP0753</td></tr><tr><td>AMPure XP beads</td><td>Beckman Coulter</td><td>Cat# A63881</td></tr><tr><td>Guanidinium thiocyanate</td><td>Thermo Fisher</td><td>Cat# AM9422</td></tr><tr><td>N-Lauroylsarcosine sodium salt solution (Sarkosyl NL)</td><td>Sigma</td><td>Cat# L7414</td></tr><tr><td>Exonuclease l</td><td>New England BioLabs</td><td>Cat# M0293S</td></tr><tr><td>Klenow Fragment</td><td>New England BioLabs</td><td>Cat# M0212L</td></tr><tr><td>Polycarbonate membrane filters 62x22</td><td>Fisher Scientific/Sterlitech Corporation</td><td>Cat# NC1421644</td></tr><tr><td>MACOSKO-2011-10 mRNA Capture Beads</td><td>Fisher Scientific/ChemGenes</td><td>Cat# NC0927472</td></tr><tr class="rowsep"><td>ERCC RNA spike-in mix</td><td>Thermo Fisher</td><td>Cat# 4456740</td></tr><tr class="rowsep"><td colspan="3"><b>Critical commercial assays</b></td></tr><tr><td>AllPrep DNA/RNA/miRNA Universal kit</td><td>QIAGEN</td><td>Cat# 80224</td></tr><tr><td>Nextera XT DNA Library Preparation Kit</td><td>Illumina</td><td>Cat# FC-131-1096</td></tr><tr><td>Nextseq 500/550 High output v2.5 kit (75 cycles)</td><td>Illumina</td><td>Cat# 20024906</td></tr><tr><td>NovaSeq 6000 S2 kit (100 cycles)</td><td>Illumina</td><td>Cat# 20012862</td></tr><tr><td>TruSeq Stranded mRNA Library Prep kit</td><td>Illumina</td><td>Cat# 20020595</td></tr><tr><td>Kapa HiFi HotStart ReadyMix</td><td>Kapa Biosystems</td><td>Cat# KK2602</td></tr><tr><td>KAPA HyperPrep kit (PCR-free)</td><td>Kapa Biosystems</td><td>Cat# KK8505</td></tr><tr><td>High Sensitivity D5000 ScreenTape</td><td>Agilent</td><td>Cat# 5067-5592</td></tr><tr><td>Qubit dsDNA High-Sensitivity kit</td><td>Thermo Fisher</td><td>Cat# Q32854</td></tr><tr><td>Quant-iT Ribogreen RNA Assay kit</td><td>Thermo Fisher</td><td>Cat# R11490</td></tr><tr class="rowsep"><td>Quant-iT PicoGreen dsDNA Assay kit</td><td>Thermo Fisher</td><td>Cat# P11496</td></tr><tr class="rowsep"><td colspan="3"><b>Deposited data</b></td></tr><tr><td>Bulk and single-cell transcriptomic data from PDAC patient samples and organoid models</td><td>This study</td><td><a id="intref5610" href="https://singlecell.broadinstitute.org/single_cell/study/SCP1644" target="_blank">https://singlecell.broadinstitute.org/single_cell/study/SCP1644</a> dbGaP: phs002712.v1.p1</td></tr><tr><td>Primary PDAC genomic and transcriptomic data (TCGA, Pancreatic Ductal Adenocarcinoma)</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref15-5" href="#" role="doc-biblioref" data-xml-rid="bib15" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref15-5" aria-controls="sref15-5">Cancer Genome Atlas Research Network, 2017</a><div class="dropBlock__holder" data-db-target-of="sref15-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">15.</div><div>Cancer Genome Atlas Research Network</div><div><strong>Integrated Genomic Characterization of Pancreatic Ductal Adenocarcinoma</strong></div><div><em>Cancer Cell.</em> 2017; <strong>32</strong>:185-203.e13</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=Cancer+Genome+Atlas+Research+NetworkIntegrated+Genomic+Characterization+of+Pancreatic+Ductal+AdenocarcinomaCancer+Cell322017185203.e13" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref7810" href="https://portal.gdc.cancer.gov/projects/TCGA-PAAD" target="_blank">https://portal.gdc.cancer.gov/projects/TCGA-PAAD</a></td></tr><tr><td>Metastatic PDAC genomic and transcriptomic data (Panc-Seq)</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref2-5" href="#" role="doc-biblioref" data-xml-rid="bib2" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref2-5" aria-controls="sref2-5">Aguirre et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref2-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">2.</div><div>Aguirre, A.J. ∙ Nowak, J.A. ∙ Camarda, N.D. ...</div><div><strong>Real-time Genomic Characterization of Advanced Pancreatic Cancer to Enable Precision Medicine</strong></div><div><em>Cancer Discov.</em> 2018; <strong>8</strong>:1096-1111</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.J.AguirreJ.A.NowakN.D.CamardaR.A.MoffittA.A.GhazaniM.Hazar-RethinamS.RaghavanJ.KimL.K.BraisD.RagonReal-time+Genomic+Characterization+of+Advanced+Pancreatic+Cancer+to+Enable+Precision+MedicineCancer+Discov.8201810961111" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>dbGaP: phs001652.v1.p1</td></tr><tr><td>CCLE transcriptomic data</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref28-1" href="#" role="doc-biblioref" data-xml-rid="bib28" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref28-1" aria-controls="sref28-1">Ghandi et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref28-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">28.</div><div>Ghandi, M. ∙ Huang, F.W. ∙ Jané-Valbuena, J. ...</div><div><strong>Next-generation characterization of the Cancer Cell Line Encyclopedia</strong></div><div><em>Nature.</em> 2019; <strong>569</strong>:503-508</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.GhandiF.W.HuangJ.Jan%C3%A9-ValbuenaG.V.KryukovC.C.LoE.R.McDonald3rdJ.BarretinaE.T.GelfandC.M.BielskiH.LiNext-generation+characterization+of+the+Cancer+Cell+Line+EncyclopediaNature5692019503508" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref8910" href="https://portals.broadinstitute.org/ccle" target="_blank">https://portals.broadinstitute.org/ccle</a></td></tr><tr class="rowsep"><td>TCGA transcriptomic data (other malignancies)</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref74-2" href="#" role="doc-biblioref" data-xml-rid="bib74" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref74-2" aria-controls="sref74-2">Vivian et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref74-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">74.</div><div>Vivian, J. ∙ Rao, A.A. ∙ Nothaft, F.A. ...</div><div><strong>Toil enables reproducible, open source, big biomedical data analyses</strong></div><div><em>Nat. Biotechnol.</em> 2017; <strong>35</strong>:314-316</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.VivianA.A.RaoF.A.NothaftC.KetchumJ.ArmstrongA.NovakJ.PfeilJ.NarkizianA.D.DeranA.Musselman-BrownToil+enables+reproducible%2C+open+source%2C+big+biomedical+data+analysesNat.+Biotechnol.352017314316" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref4710" href="https://toil.xenahubs.net" target="_blank">https://toil.xenahubs.net</a></td></tr><tr class="rowsep"><td colspan="3"><b>Experimental models: Cell lines</b></td></tr><tr><td>Human PDAC organoids</td><td>This study</td><td>N/A</td></tr><tr><td>CFPAC-1</td><td>ATCC</td><td>Cat# CRL-1918; RRID:CVCL_1119</td></tr><tr><td>L Wnt-3A cells for Wnt-3A conditioned medium</td><td>ATCC</td><td>Cat# CRL-2647; RRID:CVCL_0635</td></tr><tr class="rowsep"><td>Cultrex 293T cells for R-spondin1 conditioned medium</td><td>Trevigen</td><td>Cat# 3710-001-K; RRID:CVCL_RU08</td></tr><tr class="rowsep"><td colspan="3"><b>Oligonucleotides</b></td></tr><tr><td>Seq-Well ISPCR: AAG CAG TGG TAT CAA CGC AGA GT</td><td>Integrated DNA Technologies</td><td>N/A</td></tr><tr><td>Custom Read 1 Primer: GCC TGT CCG CGG AAG CAG TGG TAT CAA CGC AGA GTA C</td><td>Integrated DNA Technologies</td><td>N/A</td></tr><tr><td>Seq-Well 5¢ TSO: AAG CAG TGG TAT CAA CGC AGA GTG AAT rGrGrG</td><td>Integrated DNA Technologies</td><td>N/A</td></tr><tr><td>Seq-Well Custom P5-SMART PCR hybrid oligo: AAT GAT ACG GCG ACC ACC GAG ATC TAC ACG CCT GTC CGC GGA AGC AGT GGT ATC AAC GCA GAG TAC</td><td>Integrated DNA Technologies</td><td>N/A</td></tr><tr class="rowsep"><td>Seq-Well dN-SMRT oligo: AAG CAG TGG TAT CAA CGC AGA GTG ANN NGG NNN B</td><td>Integrated DNA Technologies</td><td>N/A</td></tr><tr class="rowsep"><td colspan="3"><b>Software and algorithms</b></td></tr><tr><td>R project for statistical computing v3.5.1</td><td>R Core Team</td><td><a id="intref0030" href="https://www.r-project.org" target="_blank">https://www.r-project.org</a></td></tr><tr><td>R package – Seurat v2.3.4</td><td>GitHub</td><td><a id="intref0035" href="https://github.com/satijalab/seurat" target="_blank">https://github.com/satijalab/seurat</a></td></tr><tr><td>R package – Circlize v0.4.8</td><td>CRAN</td><td><a id="intref0040" href="https://CRAN.R-project.org/%20package%20=%20circlize" target="_blank">https://CRAN.R-project.org/ package&nbsp;= circlize</a></td></tr><tr><td>R package – infercnv v0.99.4</td><td>GitHub</td><td><a id="intref0045" href="https://github.com/broadinstitute/inferCNV" target="_blank">https://github.com/broadinstitute/inferCNV</a></td></tr><tr><td>R package – data.table v1.12.0</td><td>GitHub</td><td><a id="intref0050" href="https://github.com/Rdatatable/data.table" target="_blank">https://github.com/Rdatatable/data.table</a></td></tr><tr><td>R package – ggplot2 v3.2.1</td><td>CRAN</td><td><a id="intref0055" href="https://CRAN.R-project.org/%20package%20=%20ggplot2" target="_blank">https://CRAN.R-project.org/ package&nbsp;= ggplot2</a></td></tr><tr><td>R package – ComplexHeatmap v2.7.3</td><td>Bioconductor</td><td><a id="intref0060" href="https://bioconductor.org/packages/%20ComplexHeatmap/" target="_blank">https://bioconductor.org/packages/ ComplexHeatmap/</a></td></tr><tr><td>R package – dplyr v1.0.7</td><td>CRAN</td><td><a id="intref0065" href="https://cran.r-project.org/web/packages/dplyr/" target="_blank">https://cran.r-project.org/web/packages/dplyr/</a></td></tr><tr><td>STAR</td><td>GitHub</td><td><a id="intref0070" href="https://github.com/alexdobin/STAR" target="_blank">https://github.com/alexdobin/STAR</a></td></tr><tr><td>Cumulus</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref44-1" href="#" role="doc-biblioref" data-xml-rid="bib44" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref44-1" aria-controls="sref44-1">Li et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref44-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">44.</div><div>Li, B. ∙ Gould, J. ∙ Yang, Y. ...</div><div><strong>Cumulus provides cloud-based data analysis for large-scale single-cell and single-nucleus RNA-seq</strong></div><div><em>Nat. Methods.</em> 2020; <strong>17</strong>:793-798</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=B.LiJ.GouldY.YangS.SarkizovaM.TabakaO.AshenbergY.RosenM.SlyperM.S.KowalczykA.C.VillaniCumulus+provides+cloud-based+data+analysis+for+large-scale+single-cell+and+single-nucleus+RNA-seqNat.+Methods172020793798" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0075" href="https://cumulus.readthedocs.io/" target="_blank">https://cumulus.readthedocs.io/</a></td></tr><tr><td>Broad Picard pipeline v1.90</td><td>GitHub</td><td><a id="intref0080" href="https://broadinstitute.github.io/picard/" target="_blank">https://broadinstitute.github.io/picard/</a></td></tr><tr><td>Genome Analysis Toolkit (GATK) v1.6-5-g557da77 and v.4.1.6.0</td><td>Broad Institute</td><td><a id="intref0085" href="https://gatk.broadinstitute.org/hc/en-us" target="_blank">https://gatk.broadinstitute.org/hc/en-us</a></td></tr><tr class="rowsep"><td>Python Programming Language v3.7.4</td><td>Python</td><td><a id="intref0090" href="https://www.python.org" target="_blank">https://www.python.org</a></td></tr><tr class="rowsep"><td colspan="3"><b>Other</b></td></tr><tr><td>Leica BOND RX Research Stainer</td><td>Leica Biosystems</td><td><a id="intref0095" href="https://www.leicabiosystems.com/ihc-ish-fish/fully-automated-ihc-ish-instruments/bond-rx/" target="_blank">https://www.leicabiosystems.com/ihc-ish-fish/fully-automated-ihc-ish-instruments/bond-rx/</a></td></tr><tr><td>Vectra 3.0 Automated Quantitative Imaging System</td><td>PerkinElmer/Akoya Biosciences</td><td><a id="intref0100" href="https://www.akoyabio.com/phenoptics/mantra-vectra-instruments/vectra-3-0/" target="_blank">https://www.akoyabio.com/phenoptics/mantra-vectra-instruments/vectra-3-0/</a></td></tr></tbody></table></div><figcaption><div class="core-table-tools"><ul><li><a class="core-table-show-action" href="/action/showFullTableHTML?isHtml=true&amp;tableId=undtbl1&amp;pii=S0092-8674%2821%2901332-5" target="_blank">Open table in a new tab</a></li></ul></div></figcaption></figure></div></div></section><section id="sec-4-2"><h3>Resource availability</h3><section id="sec-4-2-1"><h4>Lead contact</h4><div id="p0260" role="paragraph">Further information and requests for resources and reagents should be sent to and will be fulfilled by Dr. Alex K. Shalek (<a id="intref0105" href="mailto:shalek@mit.edu">shalek@mit.edu</a>).</div></section><section id="sec-4-2-2"><h4>Materials availability</h4><div id="p0265" role="paragraph">Organoid models generated in this study are available upon request with a materials transfer agreement.</div></section></section><section id="sec-4-3"><h3>Experimental model and subject details</h3><section id="sec-4-3-1"><h4>Human specimens</h4><div id="p0270" role="paragraph">Eligible participants were recruited from outpatient clinics and inpatient units at the Dana-Farber Cancer Institute and the Brigham and Women’s Hospital. Investigators obtained written, informed consent from patients at least 18 years old with pancreatic cancer for Dana-Farber/Harvard Cancer Center Institutional Review Board (IRB)-approved protocols 11-104, 17-000, 03-189, and/or 14-408 for tissue collection, molecular analysis, and organoid generation. ScRNA-seq samples were collected from 23 patients between October 2018 and December 2020, both male (n&nbsp;= 13) and female (n&nbsp;= 10). Organoid samples for bulk genomic and transcriptomic analyses were collected between May 2015 and January 2018. Core needle biopsy specimens were collected and the first core was sent for pathologic analysis. One or more additional cores were then allocated for scRNA-seq and/or organoid generation. Clinical features of our patient cohort are included in <b>Table S1</b>.</div></section><section id="sec-4-3-2"><h4>Human PDAC patient-derived organoid models</h4><div id="p0275" role="paragraph">Tissue samples were minced into small portions using a scalpel and then digested at 37°C for 15&nbsp;min using digest medium that consisted of human complete organoid medium (see below), 1&nbsp;mg/mL collagenase XI (Sigma Aldrich), 10&nbsp;μg/mL DNase (Stem Cell Technologies), and 10&nbsp;μM Y27632 (Selleck) (<span class="dropBlock reference-citations"><a id="body-ref-sref68-3" href="#" role="doc-biblioref" data-xml-rid="bib68" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref68-3" aria-controls="sref68-3">Tiriac et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref68-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">68.</div><div>Tiriac, H. ∙ Belleau, P. ∙ Engle, D.D. ...</div><div><strong>Organoid Profiling Identifies Common Responders to Chemotherapy in Pancreatic Cancer</strong></div><div><em>Cancer Discov.</em> 2018; <strong>8</strong>:1112-1129</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=H.TiriacP.BelleauD.D.EngleD.PlenkerA.Desch%C3%AAnesT.D.D.SomervilleF.E.M.FroelingR.A.BurkhartR.E.DenrocheG.H.JangOrganoid+Profiling+Identifies+Common+Responders+to+Chemotherapy+in+Pancreatic+CancerCancer+Discov.8201811121129" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). After dissociation, a portion of the cells from the fresh tumor specimen were allocated for scRNA-seq, and the remainder were initiated and maintained as patient-derived organoid cultures as previously described (<span class="dropBlock reference-citations"><a id="body-ref-sref12-2" href="#" role="doc-biblioref" data-xml-rid="bib12" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref12-2" aria-controls="sref12-2">Boj et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref12-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">12.</div><div>Boj, S.F. ∙ Hwang, C.I. ∙ Baker, L.A. ...</div><div><strong>Organoid models of human and mouse ductal pancreatic cancer</strong></div><div><em>Cell.</em> 2015; <strong>160</strong>:324-338</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=S.F.BojC.I.HwangL.A.BakerI.I.ChioD.D.EngleV.CorboM.JagerM.Ponz-SarviseH.TiriacM.S.SpectorOrganoid+models+of+human+and+mouse+ductal+pancreatic+cancerCell1602015324338" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref68-4" href="#" role="doc-biblioref" data-xml-rid="bib68" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref68-4" aria-controls="sref68-4">Tiriac et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref68-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">68.</div><div>Tiriac, H. ∙ Belleau, P. ∙ Engle, D.D. ...</div><div><strong>Organoid Profiling Identifies Common Responders to Chemotherapy in Pancreatic Cancer</strong></div><div><em>Cancer Discov.</em> 2018; <strong>8</strong>:1112-1129</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=H.TiriacP.BelleauD.D.EngleD.PlenkerA.Desch%C3%AAnesT.D.D.SomervilleF.E.M.FroelingR.A.BurkhartR.E.DenrocheG.H.JangOrganoid+Profiling+Identifies+Common+Responders+to+Chemotherapy+in+Pancreatic+CancerCancer+Discov.8201811121129" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). In brief, digested cells were seeded in 3-dimensional (3D) Growth-factor Reduced Matrigel (Corning), fed with human complete organoid medium containing Advanced DMEM/F12 (GIBCO), 10&nbsp;mM HEPES (GIBCO), 1x GlutaMAX (GIBCO), 500&nbsp;nM A83-01 (Tocris), 50&nbsp;ng/mL mEGF (Peprotech), 100&nbsp;ng/mL mNoggin (Peprotech), 100&nbsp;ng/mL hFGF10 (Peprotech), 10&nbsp;nM hGastrin I (Sigma), 1.25&nbsp;mM N-acetylcysteine (Sigma), 10&nbsp;mM Nicotinamide (Sigma), 1x B27 supplement (GIBCO), RSPONDIN-1 conditioned media 10% final, WNT3A conditioned media 50% final, 100&nbsp;U/mL penicillin/streptomycin (GIBCO), and 1x Primocin (Invivogen) (Table S5), and maintained at 37°C in 5% CO<sub>2</sub>. 10&nbsp;μM Y27632 (Selleck) was included in the culture medium of newly initiated samples until the first media exchange. For propagation, organoids were dissociated with TrypLE Express (GIBCO) before re-seeding into fresh Matrigel and culture medium.</div><div id="p0280" role="paragraph">After initial processing of fresh tissue specimens, we monitored samples closely for organoid growth. We did not passage organoids at set time intervals, as there was significant variability in the time needed to establish relatively robust growth of organoids (<a id="crosref2000" href="#fig3">Figure&nbsp;3</a>D). Instead, we maintained early passage organoids until they reached relative confluence, and then passaged them at low split ratios (1:1, 1:1.5, or 1:2 dilutions) in complete organoid medium to promote continued growth. In one case, PANFR0489R, cells persisted as individuals and small organoids after initiation in complete organoid medium, but did not grow and expand cell numbers significantly. Approximately 15&nbsp;weeks after initiation, we switched a portion of the surviving cells to organoid medium without A83-01 or mNoggin, and observed renewed growth of organoids under these media conditions but not of those that remained in complete organoid medium. Consequently, we expanded this sample in media without A83-01 or mNoggin, including performing early passage scRNA-seq. After several additional passages, once the organoids were robustly growing, we were able to transition this model back to complete organoid medium with no apparent change in growth rate, morphology, or transcriptional state. All other serially sampled organoids were maintained and assessed in complete medium except as indicated when specific media alterations or experimental perturbations were performed. The identify of organoid models was authenticated by comparison of their inferred CNV profiles with targeted genomic sequencing and CNV profiles of matched patient tissue and with inferred CNV profiles from patient tissue and earlier passage models in the case of samples serially assessed with scRNA-seq. The identify of cell line models was authenticated by short tandem repeat (STR) analysis. Cell line and organoid cultures were routinely tested for mycoplasma contamination.</div></section></section><section id="sec-4-4"><h3>Method details</h3><section id="sec-4-4-1"><h4>Sample preparation for single-cell RNA-sequencing of clinical samples, organoids, and cell lines</h4><div id="p0285" role="paragraph">Patient specimens were dissociated for paired scRNA-seq and organoid generation as described above. In our initial process optimization for fresh patient specimens, we found that dissociation times below 30&nbsp;min, while not always completely digesting all biopsy material and potentially affecting the representation of difficult to dissociate cell types (e.g., fibroblasts), resulted in greater cell viability and improved RNA quality downstream. After tissue dissociation, cells were washed, treated with ACK lysing buffer (Thermo Fisher) to lyse red blood cells, washed again, and counted using a hemocytometer with 0.4% Trypan blue (Thermo Fisher) added at 1:1 dilution for viability assessment. We allowed residual tissue chunks to settle before selecting a predominance of single cells for counting and Seq-Well processing. We allocated between 10,000 and 15,000 viable cells per Seq-Well array, and where possible we prepared two arrays per sample. Most samples were processed and loaded onto Seq-Well arrays within 2-3&nbsp;h of biopsy acquisition. Remaining cells and tissue chunks were allocated for patient-matched organoid generation.</div><div id="p0290" role="paragraph">For scRNA-seq of organoid samples and cell lines (CFPAC1), we passaged models and allowed them to grow for 6&nbsp;days before dissociating to single cells (organoids – TrypLE Express, Thermo Fisher; cell lines – 0.25% Trypsin-EDTA, Thermo Fisher), counting, and allocating 15,000 viable cells for Seq-Well. By standardizing the collection of organoid scRNA-seq samples at 6&nbsp;days after passaging, we tried to minimize bias arising from cell cycle differences in samples at different degrees of confluence.</div></section><section id="sec-4-4-2"><h4>Assessing organoid and cell line transcriptional states under different matrix and media conditions</h4><div id="p0295" role="paragraph">For adaptation of patient-derived organoids onto 2-dimensional (2D) culture surfaces as patient-derived cell lines, tissue culture plates were pre-coated with 100&nbsp;μg/mL Matrigel dissolved in basal media for 2&nbsp;h at 37°C before washing with PBS. Established organoid models were dissociated and seeded onto these Matrigel-coated culture wells in complete organoid media. In parallel, a portion of these passage-matched organoid cells were re-seeded into Matrigel droplets as above. Cells were cultured in both matrix conditions in complete organoid media until they were confluent, approximately 2-3&nbsp;weeks. Cells were collected and lysed using Trizol before snap freezing. RNA was isolated and purified as described below (“Bulk RNA-sequencing of organoids” section) using chloroform extraction, aqueous phase isolation, and processing using the QIAGEN AllPrep DNA/RNA/miRNA Universal kit before being submitted for sequencing.</div><div id="p0300" role="paragraph">For scRNA-seq assessment of organoid cell states when cultured under different media conditions, established organoid models were passaged as above by dissociating and reseeding into Matrigel droplets. A portion of the cells were cultured with complete organoid media (“Complete media”), while a distinct portion of passage-matched cells were cultured in “Minimal” media, which consisted of Advanced DMEM/F12 (Thermo Fisher), 10&nbsp;mM HEPES (Thermo Fisher), 1x GlutaMAX (Thermo Fisher), 100&nbsp;U/mL penicillin/streptomycin (Thermo Fisher), and 1x Primocin (Invivogen) (Table S5). Cells were cultured for 6&nbsp;days before being collected, dissociated, and aliquoted for scRNA-seq. Images were taken with an Olympus XM10 camera mounted to an Olympus CKX41 microscope 1&nbsp;day after seeding and again after 11&nbsp;days in culture to assess organoid growth in both conditions. The portion of cells cultured in minimal media were maintained in the same conditions for a longer duration and harvested again for scRNA-seq at 27&nbsp;days and 59&nbsp;days after the initial introduction of minimal media. To mirror the standard scRNA-seq workflow, the cells harvested at the 27- and 59-day time points were collected 6&nbsp;days after passaging.</div><div id="p0305" role="paragraph">In addition to the minimal media experiment, organoid cells were also cultured in standard cell line media (“RP10”), which contains RPMI-1640 (Thermo Fisher) and 100&nbsp;U/mL penicillin/streptomycin (Thermo Fisher) with 10% fetal bovine serum (Sigma), or in reduced organoid media “OWRNA,” which consists of Advanced DMEM/F12 (Thermo Fisher), 10&nbsp;mM HEPES (Thermo Fisher), 1x GlutaMAX (Thermo Fisher), 50&nbsp;ng/mL mEGF (Peprotech), 100&nbsp;ng/mL hFGF10 (Peprotech), 10&nbsp;nM hGastrin I (Sigma), 1.25&nbsp;mM N-acetylcysteine (Sigma), 10&nbsp;mM Nicotinamide (Sigma), 1x B27 supplement (Thermo Fisher), 100&nbsp;U/mL penicillin/streptomycin (Thermo Fisher), and 1x Primocin (Invivogen) (i.e., complete organoid medium with removal of WNT3A, RSPONDIN-1, NOGGIN, and A-8301; Table S5). Furthermore, OWRNA reduced organoid medium served as the baseline control medium when assessing the effect of specific factors (IFNGγ and TGF-β1) from the TME on malignant cell states. Cells were cultured for 6&nbsp;days before being collected, dissociated, and aliquoted for scRNA-seq in each of the following conditions: RP10, OWRNA, OWRNA with 50&nbsp;ng/mL IFNGγ (Peprotech), and OWRNA with 5&nbsp;ng/mL TGF-β1 (Peprotech) (Table S5). The cells cultured under the IFNGγ and TGF-β1 conditions were maintained in culture and harvested again for scRNA-seq 38&nbsp;days after being introduced to these new media conditions. For these longer duration time points, cells were again passaged 6&nbsp;days before collecting for scRNA-seq.</div><div id="p0310" role="paragraph">For scRNA-seq assessment of transcriptional states of the established pancreatic cancer cell line CFPAC1 under different media conditions, CFPAC1 cells were cultured in parallel in either standard cell line medium RP10 or complete organoid medium for 6&nbsp;days before being collected, dissociated, and aliquoted for scRNA-seq.</div></section><section id="sec-4-4-3"><h4>Compound sensitivity testing in cell lines and organoid models</h4><div id="p0315" role="paragraph">Organoids were propagated in their respective media formulations for the indicated times (e.g., 3&nbsp;weeks, 6&nbsp;weeks) before performing drug sensitivity testing. After dissociation, cells were seeded into ultra-low attachment 384-well plates (Corning) at 1000 viable cells per well with 20&nbsp;μL of their respective medium containing 10% Matrigel by volume. After 24 h, compounds were added to each well over 12-point dose curves along with DMSO controls using a Tecan D300e digital dispenser. A parallel untreated plate was harvested at this 24&nbsp;h time point for growth rate correction. Drug- or DMSO-treated cells were cultured for 5&nbsp;days before assessing for viability. Cell viability was measured by adding 20 uL of CellTiter-Glo 3D (Promega) to each well, incubating for 1&nbsp;h at room temperature on a shaker, and measuring luminescence using an EnVision plate reader (PerkinElmer). Given plate well randomization, raw luminescence data were deconvoluted with an in-house Python script (Python v3.7.4). Each condition was tested in triplicate, and each dose point was normalized to DMSO controls to estimate relative viability. Growth rate correction was performed as previously described, with growth-rate adjusted dose response curves fit to a 3-parameter sigmoidal curve (<span class="dropBlock reference-citations"><a id="body-ref-sref33" href="#" role="doc-biblioref" data-xml-rid="bib33" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref33" aria-controls="sref33">Hafner et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref33"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">33.</div><div>Hafner, M. ∙ Niepel, M. ∙ Chung, M. ...</div><div><strong>Growth rate inhibition metrics correct for confounders in measuring sensitivity to cancer drugs</strong></div><div><em>Nat. Methods.</em> 2016; <strong>13</strong>:521-527</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.HafnerM.NiepelM.ChungP.K.SorgerGrowth+rate+inhibition+metrics+correct+for+confounders+in+measuring+sensitivity+to+cancer+drugsNat.+Methods132016521527" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Each experiment was performed using cells cultured in paired basal (OWRNA+TGF-β1 or Minimal) and classical (OWRNA) media conditions, and independent experiments were performed as summarized in Figure&nbsp;S7A and Table S7. To quantify the relative sensitivity of basal versus classical models to a given compound, the difference in growth-rate adjusted areas over the curve (AOC) for organoids cultured in paired media conditions was calculated for each compound within each experiment, as illustrated in <a id="crosref2010" href="#fig6">Figure&nbsp;6</a>G and Figure&nbsp;S7A.</div><div id="p0320" role="paragraph">CFPAC1 cells were cultured in standard cell line medium RP10 or in complete organoid medium for 2 or 5&nbsp;weeks before seeding for drug testing. Cells were dissociated and seeded into 384-well tissue culture plates (Greiner Bio-One) at 400 viable cells per well with 20&nbsp;μL of their respective medium. Addition of compounds and sample harvesting were as above, except cell viability was measured by adding 20 uL of CellTiter-Glo (Promega) to each well and incubating for 15&nbsp;min at room temperature on a shaker before measuring luminescence. Data analysis and dose response curve fitting were performed as described above.</div></section><section id="sec-4-4-4"><h4>Single-cell RNA-seq (scRNA-seq) data library generation, sequencing, and alignment</h4><div id="p0325" role="paragraph">ScRNA-seq processing followed the Seq-Well protocol, uniquely compatible with low-input samples (<span class="dropBlock reference-citations"><a id="body-ref-sref29-2" href="#" role="doc-biblioref" data-xml-rid="bib29" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref29-2" aria-controls="sref29-2">Gierahn et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref29-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">29.</div><div>Gierahn, T.M. ∙ Wadsworth, 2nd, M.H. ∙ Hughes, T.K. ...</div><div><strong>Seq-Well: portable, low-cost RNA sequencing of single cells at high throughput</strong></div><div><em>Nat. Methods.</em> 2017; <strong>14</strong>:395-398</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=T.M.GierahnM.H.Wadsworth2ndT.K.HughesB.D.BrysonA.ButlerR.SatijaS.FortuneJ.C.LoveA.K.ShalekSeq-Well%3A+portable%2C+low-cost+RNA+sequencing+of+single+cells+at+high+throughputNat.+Methods142017395398" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref40-2" href="#" role="doc-biblioref" data-xml-rid="bib40" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref40-2" aria-controls="sref40-2">Hughes et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref40-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">40.</div><div>Hughes, T.K. ∙ Wadsworth, 2nd, M.H. ∙ Gierahn, T.M. ...</div><div><strong>Second-Strand Synthesis-Based Massively Parallel scRNA-Seq Reveals Cellular States and Molecular Features of Human Inflammatory Skin Pathologies</strong></div><div><em>Immunity.</em> 2020; <strong>53</strong>:878-894 e877</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=T.K.HughesM.H.Wadsworth2ndT.M.GierahnT.DoD.WeissP.R.AndradeF.MaB.J.de+Andrade+SilvaS.ShaoL.C.TsoiSecond-Strand+Synthesis-Based+Massively+Parallel+scRNA-Seq+Reveals+Cellular+States+and+Molecular+Features+of+Human+Inflammatory+Skin+PathologiesImmunity532020878894+e877" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Briefly, arrays were preloaded with RNA capture beads (Fisher Scientific/ChemGenes) and stored in quenching buffer until used. Prior to cell loading, arrays were resuspended in 5&nbsp;mL RPMI-1640 medium (Thermo Fisher) with 10% fetal bovine serum (Sigma), hereafter referred to as RP10. After dissociation, single-cell suspensions were manually counted and diluted to 15,000 cells per 200&nbsp;μL of RP10 when cell numbers allowed. Excess RP10 was aspirated from the array and cells were loaded onto arrays. Excess cells were washed off with PBS (4x5&nbsp;mL), briefly left in RPMI (5&nbsp;mL) and cell+bead pairs were sealed for 40&nbsp;min at 37°C using a polycarbonate membrane (Fisher Scientific). Arrays were rocked in lysis buffer for 20&nbsp;min and RNA was hybridized onto the beads for 40&nbsp;min. Beads were removed and reverse transcription was performed overnight using Maxima H Minus Reverse Transcriptase (Fisher Scientific). Prior to sequencing, the beads underwent an exonuclease treatment (New England BioLabs) and second strand synthesis <i>en masse</i> followed by whole transcriptome amplification (WTA, Kapa Biosystems) in 1,500 bead reactions (50&nbsp;μL). cDNA was isolated using Agencourt AMPure XP beads (Beckman Coulter) at 0.6X SPRI (solid-phase reversible immobilization) followed by a 1X SPRI and quantified using a Qubit dsDNA High Sensitivity assay kit (Thermo Fisher). Library preparation was performed using Nextera XT DNA tagmentation (Illumina FC-131-1096) and N700 and N500 indices specific to a given sample. Tagmented and amplified sequences were purified with a 0.6X SPRI, and cDNA was quantified (Qubit dsDNA High sensitivity assay kit, Thermo Fisher) and the base pair distribution measured (High sensitivity D5000 screen tape, Agilent). cDNA was loaded onto either an Illumina Nextseq (75 Cycle NextSeq 500/550 v2.5 kit) or Novaseq (100 Cycle NovaSeq 6000 S2 kit) at 2.4 pM. Regardless of platform, the paired end read structure was 21 bases (cell barcode and UMI) by 50 bases (transcriptomic information) with an 8 base pair (bp) custom read one primer. The demultiplex and alignment protocol was followed as previously described (<span class="dropBlock reference-citations"><a id="body-ref-sref48" href="#" role="doc-biblioref" data-xml-rid="bib48" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref48" aria-controls="sref48">Macosko et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref48"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">48.</div><div>Macosko, E.Z. ∙ Basu, A. ∙ Satija, R. ...</div><div><strong>Highly Parallel Genome-wide Expression Profiling of Individual Cells Using Nanoliter Droplets</strong></div><div><em>Cell.</em> 2015; <strong>161</strong>:1202-1214</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=E.Z.MacoskoA.BasuR.SatijaJ.NemeshK.ShekharM.GoldmanI.TiroshA.R.BialasN.KamitakiE.M.MartersteckHighly+Parallel+Genome-wide+Expression+Profiling+of+Individual+Cells+Using+Nanoliter+DropletsCell161201512021214" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). While Novaseq data were directly output as FASTQs, Nextseq BCL files were converted to FASTQs using bcl2fastq2. The resultant Nextseq and Novaseq FASTQs were demultiplexed by sample based on Nextera N700 and N500 indices. Reads were then aligned to the hg19 transcriptome using the cumulus/dropseq_tools pipeline on Terra maintained by the Broad Institute using standard settings (<span class="dropBlock reference-citations"><a id="body-ref-sref44-2" href="#" role="doc-biblioref" data-xml-rid="bib44" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref44-2" aria-controls="sref44-2">Li et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref44-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">44.</div><div>Li, B. ∙ Gould, J. ∙ Yang, Y. ...</div><div><strong>Cumulus provides cloud-based data analysis for large-scale single-cell and single-nucleus RNA-seq</strong></div><div><em>Nat. Methods.</em> 2020; <strong>17</strong>:793-798</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=B.LiJ.GouldY.YangS.SarkizovaM.TabakaO.AshenbergY.RosenM.SlyperM.S.KowalczykA.C.VillaniCumulus+provides+cloud-based+data+analysis+for+large-scale+single-cell+and+single-nucleus+RNA-seqNat.+Methods172020793798" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div></section><section id="sec-4-4-5"><h4>Bulk RNA- and DNA-sequencing of organoids</h4><div id="p0330" role="paragraph">RNA was obtained for bulk RNA-sequencing from established organoids using one of two approaches. Dissociated organoids were resuspended into cold Matrigel, added as droplets to tissue culture plates (Greiner BioOne), and allowed to polymerize for 30&nbsp;min before addition of media. Organoids were grown for 14-21&nbsp;days (until confluent) under these conditions with regular media changes. At the time of harvest, cells were washed with cold phosphate buffered saline (PBS) at 4°C, then lysed with Trizol (Invitrogen) before snap-freezing. To isolate RNA, we performed chloroform extraction with isolation of the aqueous phase before processing RNA as per protocols outlined in the QIAGEN AllPrep DNA/RNA/miRNA Universal kit.</div><div id="p0335" role="paragraph">In the second approach, used to obtain both RNA and DNA, dissociated organoids were resuspended in a solution of 10% Matrigel in complete organoid media (volume/volume) and cultured in ultra-low-attachment culture flasks (Corning). Organoids were grown for 14-21&nbsp;days (until confluent) before pelleting, washing with cold PBS at 4°C until most Matrigel was dissipated, and then snap frozen. Cell pellets were homogenized using buffer RLT Plus (QIAGEN) and a Precellys homogenizer. Samples were then processed for both DNA extraction and RNA isolation as per the QIAGEN AllPrep DNA/RNA/miRNA Universal kit. Purified RNA and DNA were then submitted for sequencing by the Broad Institute Genomics Platform.</div><div id="p0340" role="paragraph">For bulk RNA-sequencing, total RNA was quantified using the Quant-iT RiboGreen RNA Assay Kit (Thermo Fisher) and normalized to 5&nbsp;ng/μL. Following plating, 2&nbsp;μL of a 1:1000 dilution of ERCC RNA controls (Thermo Fisher) were spiked into each sample. An aliquot of 200&nbsp;ng for each sample was transferred into library preparation which uses an automated variant of the Illumina TruSeq Stranded mRNA Sample Preparation Kit. This method preserves strand orientation of the RNA transcript, and uses oligo dT beads to select mRNA from the total RNA sample followed by heat fragmentation and cDNA synthesis from the RNA template. The resultant 400&nbsp;bp cDNA then goes through dual-indexed library preparation: ‘A’ base addition, adaptor ligation using P7 adapters, and PCR enrichment using P5 adapters. After enrichment, the libraries were quantified using Quant-iT PicoGreen (1:200 dilution; Thermo Fisher). After normalizing samples to 5&nbsp;ng/μL, the set was pooled and quantified using the KAPA Library Quantification Kit for Illumina Sequencing Platforms. The entire process was performed in 96-well format and all pipetting was done by either Agilent Bravo or Hamilton Starlet.</div><div id="p0345" role="paragraph">Pooled libraries were normalized to 2&nbsp;nM and denatured using 0.1&nbsp;N NaOH prior to sequencing. Flowcell cluster amplification and sequencing were performed according to the manufacturer’s protocols using the NovaSeq 6000. Each run was a 101&nbsp;bp paired-end with an eight-base index barcode read. Data were analyzed using the Broad Picard pipeline which includes de-multiplexing and data aggregation (<a id="intref0110" href="https://broadinstitute.github.io/picard/" target="_blank">https://broadinstitute.github.io/picard/</a>). FASTQ files were then processed as described below (see Bulk RNA-sequencing analysis).</div><div id="p0350" role="paragraph">For whole genome sequencing, 350&nbsp;ng of genomic DNA was fragmented using a Covaris focused-ultrasonicator targeting 385bp fragments followed by size selection using SPRI cleanup. Library preparation was performed using a KAPA HyperPrep without amplification kit (KAPA Biosystems) with palindromic forked adapters with unique 8-base index sequences embedded within the adaptor (Roche). Libraries were then quantified using quantitative PCR (kit purchased from KAPA Biosystems) with probes specific to the adaptor ends on an Agilent Bravo liquid handling platform. Libraries were normalized to 2.2&nbsp;nM, pooled into 24-plexes, combined with NovaSeq Cluster Amp Reagents DPX1, DPX2, and DPX3, and loaded into single lanes of a NovaSeq 6000 S4 flowcell using a Hamilton Starlet Liquid Handling system. Cluster amplification and sequencing occurred utilizing sequencing-by-synthesis kits to produce 151bp paired-end reads. Output from Illumina software was processed by the Broad Picard pipeline (<a id="intref0115" href="https://broadinstitute.github.io/picard/" target="_blank">https://broadinstitute.github.io/picard/</a>) to yield BAM files containing demultiplexed, aggregated aligned reads. BAM files were then processed as described below (see Mutation and CNV identification from bulk DNA-sequencing).</div></section><section id="sec-4-4-6"><h4>Multiplex immunofluorescence imaging</h4><div id="p0355" role="paragraph">A multi-marker panel was developed to characterize malignant cell subtype in formalin-fixed paraffin-embedded (FFPE) 4&nbsp;μm tissue sections using multiplex immunofluorescence (mIF). The panel comprises markers associated with either a basal (Keratin-17: Thermo Fisher MA513539 and S100A2: Abcam 109494) or classical (CLDN18.2: Abcam 241330, GATA6: Cell Signaling Technology 5851 and TFF1: Abcam 92377) subtype. Additionally, DAPI (Akoya Biosciences FP1490) was included for identification of nuclei and pan-cytokeratin (AE1/AE3: Dako M3515; C11: Cell Signaling Technology 4545) for identification of epithelial cells. Secondary Opal Polymer HRP anti-mouse and anti-rabbit antibody (Dako ARH1001EA), Tyramide signal amplification, and Opal fluorophores (Akoya Biosciences) were used to detect primary antibodies (Keratin-17, Opal 520; S100A2, Opal 650; GATA6, Opal 540; CLDN18.2, Opal 570; TFF1, Opal 690; panCK, Opal 620).</div><div id="p0360" role="paragraph">These specific mIF markers were chosen for several reasons. KRT17, S100A2, and TFF1 are included in the original basal and classical RNA gene signatures (<span class="dropBlock reference-citations"><a id="body-ref-sref50-4" href="#" role="doc-biblioref" data-xml-rid="bib50" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref50-4" aria-controls="sref50-4">Moffitt et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref50-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">50.</div><div>Moffitt, R.A. ∙ Marayati, R. ∙ Flate, E.L. ...</div><div><strong>Virtual microdissection identifies distinct tumor- and stroma-specific subtypes of pancreatic ductal adenocarcinoma</strong></div><div><em>Nat. Genet.</em> 2015; <strong>47</strong>:1168-1178</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=R.A.MoffittR.MarayatiE.L.FlateK.E.VolmarS.G.LoezaK.A.HoadleyN.U.RashidL.A.WilliamsS.C.EatonA.H.ChungVirtual+microdissection+identifies+distinct+tumor-+and+stroma-specific+subtypes+of+pancreatic+ductal+adenocarcinomaNat.+Genet.47201511681178" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). GATA6 is also a classical marker in an extended RNA gene panel and has been reported to be a potential driver of the classical phenotype (<span class="dropBlock reference-citations"><a id="body-ref-sref13" href="#" role="doc-biblioref" data-xml-rid="bib13" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref13" aria-controls="sref13">Brunton et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref13"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">13.</div><div>Brunton, H. ∙ Caligiuri, G. ∙ Cunningham, R. ..., Glasgow Precision Oncology Laboratory, Australian Pancreatic Cancer Genome Initiative</div><div><strong>HNF4A and GATA6 Loss Reveals Therapeutically Actionable Subtypes in Pancreatic Cancer</strong></div><div><em>Cell Rep.</em> 2020; <strong>31</strong>:107625</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=H.BruntonG.CaligiuriR.CunninghamR.Upstill-GoddardU.M.BaileyI.M.GarnerC.NourseS.DreyerM.JonesK.Moran-JonesGlasgow+Precision+Oncology+LaboratoryAustralian+Pancreatic+Cancer+Genome+InitiativeHNF4A+and+GATA6+Loss+Reveals+Therapeutically+Actionable+Subtypes+in+Pancreatic+CancerCell+Rep.312020107625" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref50-5" href="#" role="doc-biblioref" data-xml-rid="bib50" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref50-5" aria-controls="sref50-5">Moffitt et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref50-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">50.</div><div>Moffitt, R.A. ∙ Marayati, R. ∙ Flate, E.L. ...</div><div><strong>Virtual microdissection identifies distinct tumor- and stroma-specific subtypes of pancreatic ductal adenocarcinoma</strong></div><div><em>Nat. Genet.</em> 2015; <strong>47</strong>:1168-1178</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=R.A.MoffittR.MarayatiE.L.FlateK.E.VolmarS.G.LoezaK.A.HoadleyN.U.RashidL.A.WilliamsS.C.EatonA.H.ChungVirtual+microdissection+identifies+distinct+tumor-+and+stroma-specific+subtypes+of+pancreatic+ductal+adenocarcinomaNat.+Genet.47201511681178" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref53-2" href="#" role="doc-biblioref" data-xml-rid="bib53" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref53-2" aria-controls="sref53-2">O’Kane et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref53-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">53.</div><div>O’Kane, G.M. ∙ Grunwald, B.T. ∙ Jang, G.H. ...</div><div><strong>GATA6 Expression Distinguishes Classical and Basal-like Subtypes in Advanced Pancreatic Cancer</strong></div><div><em>Clin Cancer Res.</em> 2020; <strong>26</strong>:4901-4910</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=G.M.O%E2%80%99KaneB.T.GrunwaldG.H.JangM.MasoomianS.PicardoR.C.GrantR.E.DenrocheA.ZhangY.WangB.LamGATA6+Expression+Distinguishes+Classical+and+Basal-like+Subtypes+in+Advanced+Pancreatic+CancerClin+Cancer+Res.26202049014910" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Single markers of S100A2 and GATA6 have also been used extensively in imaging experiments in past literature to mark cells in the basal or classical state, respectively (<span class="dropBlock reference-citations"><a id="body-ref-sref4-3" href="#" role="doc-biblioref" data-xml-rid="bib4" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref4-3" aria-controls="sref4-3">Aung et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref4-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">4.</div><div>Aung, K.L. ∙ Fischer, S.E. ∙ Denroche, R.E. ...</div><div><strong>Genomics-Driven Precision Medicine for Advanced Pancreatic Cancer: Early Results from the COMPASS Trial</strong></div><div><em>Clin. Cancer Res.</em> 2018; <strong>24</strong>:1344-1354</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=K.L.AungS.E.FischerR.E.DenrocheG.H.JangA.DoddS.CreightonB.SouthwoodS.B.LiangD.ChadwickA.ZhangGenomics-Driven+Precision+Medicine+for+Advanced+Pancreatic+Cancer%3A+Early+Results+from+the+COMPASS+TrialClin.+Cancer+Res.24201813441354" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref16-7" href="#" role="doc-biblioref" data-xml-rid="bib16" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref16-7" aria-controls="sref16-7">Chan-Seng-Yue et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref16-7"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">16.</div><div>Chan-Seng-Yue, M. ∙ Kim, J.C. ∙ Wilson, G.W. ...</div><div><strong>Transcription phenotypes of pancreatic cancer are driven by genomic events during tumor evolution</strong></div><div><em>Nat. Genet.</em> 2020; <strong>52</strong>:231-240</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.Chan-Seng-YueJ.C.KimG.W.WilsonK.NgE.F.FigueroaG.M.O%E2%80%99KaneA.A.ConnorR.E.DenrocheR.C.GrantJ.McLeodTranscription+phenotypes+of+pancreatic+cancer+are+driven+by+genomic+events+during+tumor+evolutionNat.+Genet.522020231240" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref49-2" href="#" role="doc-biblioref" data-xml-rid="bib49" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref49-2" aria-controls="sref49-2">Miyabayashi et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref49-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">49.</div><div>Miyabayashi, K. ∙ Baker, L.A. ∙ Deschênes, A. ...</div><div><strong>Intraductal Transplantation Models of Human Pancreatic Ductal Adenocarcinoma Reveal Progressive Transition of Molecular Subtypes</strong></div><div><em>Cancer Discov.</em> 2020; <strong>10</strong>:1566-1589</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=K.MiyabayashiL.A.BakerA.Desch%C3%AAnesB.TraubG.CaligiuriD.PlenkerB.AlagesanP.BelleauS.LiJ.KendallIntraductal+Transplantation+Models+of+Human+Pancreatic+Ductal+Adenocarcinoma+Reveal+Progressive+Transition+of+Molecular+SubtypesCancer+Discov.10202015661589" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref53-3" href="#" role="doc-biblioref" data-xml-rid="bib53" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref53-3" aria-controls="sref53-3">O’Kane et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref53-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">53.</div><div>O’Kane, G.M. ∙ Grunwald, B.T. ∙ Jang, G.H. ...</div><div><strong>GATA6 Expression Distinguishes Classical and Basal-like Subtypes in Advanced Pancreatic Cancer</strong></div><div><em>Clin Cancer Res.</em> 2020; <strong>26</strong>:4901-4910</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=G.M.O%E2%80%99KaneB.T.GrunwaldG.H.JangM.MasoomianS.PicardoR.C.GrantR.E.DenrocheA.ZhangY.WangB.LamGATA6+Expression+Distinguishes+Classical+and+Basal-like+Subtypes+in+Advanced+Pancreatic+CancerClin+Cancer+Res.26202049014910" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref65-2" href="#" role="doc-biblioref" data-xml-rid="bib65" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref65-2" aria-controls="sref65-2">Somerville et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref65-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">65.</div><div>Somerville, T.D.D. ∙ Xu, Y. ∙ Miyabayashi, K. ...</div><div><strong>TP63-Mediated Enhancer Reprogramming Drives the Squamous Subtype of Pancreatic Ductal Adenocarcinoma</strong></div><div><em>Cell Rep.</em> 2018; <strong>25</strong>:1741-1755 e1747</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=T.D.D.SomervilleY.XuK.MiyabayashiH.TiriacC.R.ClearyD.Maia-SilvaJ.P.MilazzoD.A.TuvesonC.R.VakocTP63-Mediated+Enhancer+Reprogramming+Drives+the+Squamous+Subtype+of+Pancreatic+Ductal+AdenocarcinomaCell+Rep.25201817411755+e1747" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). CLDN18.2 has also been associated with classical phenotypes, and antibody-based therapies targeting CLDN18.2 have been developed and tested in PDAC (<span class="dropBlock reference-citations"><a id="body-ref-sref77" href="#" role="doc-biblioref" data-xml-rid="bib77" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref77" aria-controls="sref77">Wöll et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref77"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">77.</div><div>Wöll, S. ∙ Schlitter, A.M. ∙ Dhaene, K. ...</div><div><strong>Claudin 18.2 is a target for IMAB362 antibody in pancreatic neoplasms</strong></div><div><em>Int. J. Cancer.</em> 2014; <strong>134</strong>:731-739</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=S.W%C3%B6llA.M.SchlitterK.DhaeneM.RollerI.EspositoU.Sahin%C3%96.T%C3%BCreciClaudin+18.2+is+a+target+for+IMAB362+antibody+in+pancreatic+neoplasmsInt.+J.+Cancer1342014731739" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Furthermore, the markers chosen for the mIF subtyping panel agreed with those selected as optimal markers to differentiate basal-like versus classical by an international panel of experts at a workshop on pancreatic cancer subtyping held at Memorial Sloan Kettering in 2019. Here, we used multiple markers for each state to provide greater confidence in cell state identification and to assess marker heterogeneity across our mIF cohort.</div><div id="p0365" role="paragraph">Prior to use in multiplex staining, primary antibodies were first optimized via immunohistochemistry on control tissue to confirm contextual specificity. Monoplex immunofluorescence and iterative multiplex fluorescent staining were then used to optimize staining order, antibody-fluorophore assignments and fluorophore concentrations. Multiplex staining was performed using a Leica BOND RX Research Stainer (Leica Biosystems, Buffalo, IL) with sequential cycles of antigen retrieval, protein blocking, primary antibody incubation, secondary antibody incubation, and fluorescent labeling. Overview images of stained slides were acquired at 10X magnification using a Vectra 3.0 Automated Quantitative Imaging System (Perkin Elmer, Waltham, MA) and regions of interest (ROIs) were selected for multispectral image acquisition at 20X. After unmixing using a spectral library of single-color references, each image was inspected to ensure uniform staining quality and adequate tumor representation.</div></section></section><section id="sec-4-5"><h3>Quantification and statistical analysis</h3><section id="sec-4-5-1"><h4>Mutation and CNV identification from bulk DNA-sequencing</h4><div id="p0370" role="paragraph">For targeted DNA-sequencing of clinical samples, next-generation sequencing using a custom-designed hybrid capture library preparation was performed on an Illumina HiSeq 2500 with 2x100 paired-end reads, as previously described (<span class="dropBlock reference-citations"><a id="body-ref-sref26" href="#" role="doc-biblioref" data-xml-rid="bib26" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref26" aria-controls="sref26">Garcia et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref26"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">26.</div><div>Garcia, E.P. ∙ Minkovsky, A. ∙ Jia, Y. ...</div><div><strong>Validation of OncoPanel: A Targeted Next-Generation Sequencing Assay for the Detection of Somatic Variants in Cancer</strong></div><div><em>Arch. Pathol. Lab. Med.</em> 2017; <strong>141</strong>:751-758</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=E.P.GarciaA.MinkovskyY.JiaM.D.DucarP.ShivdasaniX.GongA.H.LigonL.M.ShollF.C.KuoL.E.MacConaillValidation+of+OncoPanel%3A+A+Targeted+Next-Generation+Sequencing+Assay+for+the+Detection+of+Somatic+Variants+in+CancerArch.+Pathol.+Lab.+Med.1412017751758" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref64" href="#" role="doc-biblioref" data-xml-rid="bib64" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref64" aria-controls="sref64">Sholl et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref64"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">64.</div><div>Sholl, L.M. ∙ Do, K. ∙ Shivdasani, P. ...</div><div><strong>Institutional implementation of clinical tumor profiling on an unselected cancer population</strong></div><div><em>JCI Insight.</em> 2016; <strong>1</strong>:e87062</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=L.M.ShollK.DoP.ShivdasaniE.CeramiA.M.DubucF.C.KuoE.P.GarciaY.JiaP.DavineniR.P.AboInstitutional+implementation+of+clinical+tumor+profiling+on+an+unselected+cancer+populationJCI+Insight12016e87062" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Sequence reads were aligned to reference sequence b37 edition from the Human Genome Reference Consortium using bwa, and further processed using Picard (version 1.90, <a id="intref0120" href="http://broadinstitute.github.io/picard/" target="_blank">http://broadinstitute.github.io/picard/</a>) to remove duplicates and Genome Analysis Toolkit (GATK, version 1.6-5-g557da77) to perform localized realignment around indel sites. Single nucleotide variants were called using MuTect v1.1.45, insertions and deletions were called using GATK Indelocator. Copy number variants (CNV) and structural variants were called using the internally-developed algorithms RobustCNV and BreaKmer followed by manual review (<span class="dropBlock reference-citations"><a id="body-ref-sref1" href="#" role="doc-biblioref" data-xml-rid="bib1" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref1" aria-controls="sref1">Abo et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">1.</div><div>Abo, R.P. ∙ Ducar, M. ∙ Garcia, E.P. ...</div><div><strong>BreaKmer: detection of structural variation in targeted massively parallel sequencing data using kmers</strong></div><div><em>Nucleic Acids Res.</em> 2015; <strong>43</strong>:e19</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=R.P.AboM.DucarE.P.GarciaA.R.ThornerV.Rojas-RudillaL.LinL.M.ShollW.C.HahnM.MeyersonN.I.LindemanBreaKmer%3A+detection+of+structural+variation+in+targeted+massively+parallel+sequencing+data+using+kmersNucleic+Acids+Res.432015e19" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). RobustCNV calculates copy ratios by performing a robust linear regression against a panel of normal samples. The data were segmented using circular binary segmentation, and event identification was performed based on the observed variance of the data points (<span class="dropBlock reference-citations"><a id="body-ref-sref11" href="#" role="doc-biblioref" data-xml-rid="bib11" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref11" aria-controls="sref11">Bi et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref11"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">11.</div><div>Bi, W.L. ∙ Greenwald, N.F. ∙ Ramkissoon, S.H. ...</div><div><strong>Clinical Identification of Oncogenic Drivers and Copy-Number Alterations in Pituitary Tumors</strong></div><div><em>Endocrinology.</em> 2017; <strong>158</strong>:2284-2291</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=W.L.BiN.F.GreenwaldS.H.RamkissoonM.AbedalthagafiS.M.CoyK.L.LigonY.MeiL.MacConaillM.DucarL.MinClinical+Identification+of+Oncogenic+Drivers+and+Copy-Number+Alterations+in+Pituitary+TumorsEndocrinology158201722842291" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="p0375" role="paragraph">We computed the cytoband-level copy number calls and weighted (by length) average segment means across the covered regions of each cytoband using ASCETS (<span class="dropBlock reference-citations"><a id="body-ref-sref66" href="#" role="doc-biblioref" data-xml-rid="bib66" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref66" aria-controls="sref66">Spurr et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref66"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">66.</div><div>Spurr, L.F. ∙ Touat, M. ∙ Taylor, A.M. ...</div><div><strong>Quantification of aneuploidy in targeted sequencing data using ASCETS</strong></div><div><em>Bioinformatics.</em> 2021; <strong>37</strong>:2461-2463</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=L.F.SpurrM.TouatA.M.TaylorA.M.DubucJ.ShihD.M.MeredithW.V.PisanoM.L.MeyersonK.L.LigonA.D.CherniackQuantification+of+aneuploidy+in+targeted+sequencing+data+using+ASCETSBioinformatics37202124612463" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Briefly, cytobands were considered amplified/deleted if more than 70% of the covered regions had a log2 copy ratio of greater than 0.2/less than −0.2, and were considered neutral if more than 70% of the covered regions had a log2 copy ratio between −0.2 and 0.2.</div><div id="p0380" role="paragraph">For comparisons of driver mutation frequencies across patient tumors, cell lines, and organoid models, mutation and CNV calls for <i>KRAS</i>, <i>TP53</i>, <i>CDKN2A</i>, and <i>SMAD4</i> were either compiled from prior publications for patient samples and CCLE cell lines (<span class="dropBlock reference-citations"><a id="body-ref-sref2-6" href="#" role="doc-biblioref" data-xml-rid="bib2" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref2-6" aria-controls="sref2-6">Aguirre et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref2-6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">2.</div><div>Aguirre, A.J. ∙ Nowak, J.A. ∙ Camarda, N.D. ...</div><div><strong>Real-time Genomic Characterization of Advanced Pancreatic Cancer to Enable Precision Medicine</strong></div><div><em>Cancer Discov.</em> 2018; <strong>8</strong>:1096-1111</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.J.AguirreJ.A.NowakN.D.CamardaR.A.MoffittA.A.GhazaniM.Hazar-RethinamS.RaghavanJ.KimL.K.BraisD.RagonReal-time+Genomic+Characterization+of+Advanced+Pancreatic+Cancer+to+Enable+Precision+MedicineCancer+Discov.8201810961111" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref15-6" href="#" role="doc-biblioref" data-xml-rid="bib15" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref15-6" aria-controls="sref15-6">Cancer Genome Atlas Research Network, 2017</a><div class="dropBlock__holder" data-db-target-of="sref15-6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">15.</div><div>Cancer Genome Atlas Research Network</div><div><strong>Integrated Genomic Characterization of Pancreatic Ductal Adenocarcinoma</strong></div><div><em>Cancer Cell.</em> 2017; <strong>32</strong>:185-203.e13</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=Cancer+Genome+Atlas+Research+NetworkIntegrated+Genomic+Characterization+of+Pancreatic+Ductal+AdenocarcinomaCancer+Cell322017185203.e13" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref28-2" href="#" role="doc-biblioref" data-xml-rid="bib28" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref28-2" aria-controls="sref28-2">Ghandi et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref28-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">28.</div><div>Ghandi, M. ∙ Huang, F.W. ∙ Jané-Valbuena, J. ...</div><div><strong>Next-generation characterization of the Cancer Cell Line Encyclopedia</strong></div><div><em>Nature.</em> 2019; <strong>569</strong>:503-508</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.GhandiF.W.HuangJ.Jan%C3%A9-ValbuenaG.V.KryukovC.C.LoE.R.McDonald3rdJ.BarretinaE.T.GelfandC.M.BielskiH.LiNext-generation+characterization+of+the+Cancer+Cell+Line+EncyclopediaNature5692019503508" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) or generated from whole genome sequencing of organoid models. Organoid cohort variants were called from tumor/germline pairs using GATK (v.4.1.6.0, Paired tumor-control mode). Potential germline variants were additionally filtered using gnomAD (v2.1). Significance of short-nucleotide variants (SNVs) was determined using MuSiC2 (v0.2, q-value&nbsp;&lt; 0.1). CNVs were initially called using GATK (v.4.1.6.0) followed by analysis with GISTIC (v2.0.23). Genomic alterations in <i>KRAS</i>, <i>TP53</i>, <i>CDKN2A</i>, and <i>SMAD4</i> were binarized by counting a gene in the given sample as altered if it contained at least one of the following alterations: missense mutation, nonsense mutation, splice site alteration, frameshift insertion or deletion, in-frame insertion or deletion, a high amplification, or a homozygous deletion. Statistical significance of alteration occurrence per gene across sample cohorts was determined using a Fisher’s exact test with multiple test correction using the Benjamini-Hochberg procedure (R v4.0.3).</div></section><section id="sec-4-5-2"><h4>Bulk RNA-sequencing analysis</h4><div id="p0385" role="paragraph">FASTQs for bulk RNA expression profiles were downloaded from the relevant repository (TCGA, https://toil.xenahubs.net; PDAC Cell lines, <a id="intref0125" href="https://portals.broadinstitute.org/ccle" target="_blank">https://portals.broadinstitute.org/ccle</a>), available in-house (Panc-Seq, metastatic PDAC), or generated for this study (organoid cohort) (<span class="dropBlock reference-citations"><a id="body-ref-sref2-7" href="#" role="doc-biblioref" data-xml-rid="bib2" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref2-7" aria-controls="sref2-7">Aguirre et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref2-7"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">2.</div><div>Aguirre, A.J. ∙ Nowak, J.A. ∙ Camarda, N.D. ...</div><div><strong>Real-time Genomic Characterization of Advanced Pancreatic Cancer to Enable Precision Medicine</strong></div><div><em>Cancer Discov.</em> 2018; <strong>8</strong>:1096-1111</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.J.AguirreJ.A.NowakN.D.CamardaR.A.MoffittA.A.GhazaniM.Hazar-RethinamS.RaghavanJ.KimL.K.BraisD.RagonReal-time+Genomic+Characterization+of+Advanced+Pancreatic+Cancer+to+Enable+Precision+MedicineCancer+Discov.8201810961111" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; (<span class="dropBlock reference-citations"><a id="body-ref-sref76-2" href="#" role="doc-biblioref" data-xml-rid="bib76" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref76-2" aria-controls="sref76-2">Weinstein et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref76-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">76.</div><div>Weinstein, J.N. ∙ Collisson, E.A. ∙ Mills, G.B. ..., Cancer Genome Atlas Research Network</div><div><strong>The Cancer Genome Atlas Pan-Cancer analysis project</strong></div><div><em>Nat. Genet.</em> 2013; <strong>45</strong>:1113-1120</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.N.WeinsteinE.A.CollissonG.B.MillsK.R.ShawB.A.OzenbergerK.EllrottI.ShmulevichC.SanderJ.M.StuartCancer+Genome+Atlas+Research+NetworkThe+Cancer+Genome+Atlas+Pan-Cancer+analysis+projectNat.+Genet.45201311131120" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) <span class="dropBlock reference-citations"><a id="body-ref-sref15-7" href="#" role="doc-biblioref" data-xml-rid="bib15" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref15-7" aria-controls="sref15-7">Cancer Genome Atlas Research Network, 2017</a><div class="dropBlock__holder" data-db-target-of="sref15-7"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">15.</div><div>Cancer Genome Atlas Research Network</div><div><strong>Integrated Genomic Characterization of Pancreatic Ductal Adenocarcinoma</strong></div><div><em>Cancer Cell.</em> 2017; <strong>32</strong>:185-203.e13</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=Cancer+Genome+Atlas+Research+NetworkIntegrated+Genomic+Characterization+of+Pancreatic+Ductal+AdenocarcinomaCancer+Cell322017185203.e13" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref28-3" href="#" role="doc-biblioref" data-xml-rid="bib28" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref28-3" aria-controls="sref28-3">Ghandi et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref28-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">28.</div><div>Ghandi, M. ∙ Huang, F.W. ∙ Jané-Valbuena, J. ...</div><div><strong>Next-generation characterization of the Cancer Cell Line Encyclopedia</strong></div><div><em>Nature.</em> 2019; <strong>569</strong>:503-508</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.GhandiF.W.HuangJ.Jan%C3%A9-ValbuenaG.V.KryukovC.C.LoE.R.McDonald3rdJ.BarretinaE.T.GelfandC.M.BielskiH.LiNext-generation+characterization+of+the+Cancer+Cell+Line+EncyclopediaNature5692019503508" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref74-3" href="#" role="doc-biblioref" data-xml-rid="bib74" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref74-3" aria-controls="sref74-3">Vivian et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref74-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">74.</div><div>Vivian, J. ∙ Rao, A.A. ∙ Nothaft, F.A. ...</div><div><strong>Toil enables reproducible, open source, big biomedical data analyses</strong></div><div><em>Nat. Biotechnol.</em> 2017; <strong>35</strong>:314-316</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.VivianA.A.RaoF.A.NothaftC.KetchumJ.ArmstrongA.NovakJ.PfeilJ.NarkizianA.D.DeranA.Musselman-BrownToil+enables+reproducible%2C+open+source%2C+big+biomedical+data+analysesNat.+Biotechnol.352017314316" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). All were processed using the same pipeline. Briefly, each sample’s sequences were marked for duplicates and then mapped to hg38 using STAR. After running QC checks using RNaseqQC, gene-level count matrices were generated using RSEM. Instructions to run the pipeline are given in the Broad CCLE github repository <a id="intref0130" href="https://github.com/broadinstitute/ccle_processing" target="_blank">https://github.com/broadinstitute/ccle_processing</a>. Length-normalized values (TPM) were then transformed according to log2(TPM+1) for downstream analysis. The entire dataset was scaled and centered to allow relative comparisons across sample types (e.g., tumors, organoids, and cell lines). Signature scores were computed as below (e.g., basal and classical; see Generation of expression signatures/scores below) (<span class="dropBlock reference-citations"><a id="body-ref-sref60-3" href="#" role="doc-biblioref" data-xml-rid="bib60" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref60-3" aria-controls="sref60-3">Puram et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref60-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">60.</div><div>Puram, S.V. ∙ Tirosh, I. ∙ Parikh, A.S. ...</div><div><strong>Single-Cell Transcriptomic Analysis of Primary and Metastatic Tumor Ecosystems in Head and Neck Cancer</strong></div><div><em>Cell.</em> 2017; <strong>171</strong>:1611-1624 e1624</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=S.V.PuramI.TiroshA.S.ParikhA.P.PatelK.YizhakS.GillespieC.RodmanC.L.LuoE.A.MrozK.S.EmerickSingle-Cell+Transcriptomic+Analysis+of+Primary+and+Metastatic+Tumor+Ecosystems+in+Head+and+Neck+CancerCell171201716111624+e1624" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div></section><section id="sec-4-5-3"><h4>Single-cell data quality pre-processing and initial cell type discovery</h4><div id="p0390" role="paragraph">All single-cell data analysis was performed using the R language for Statistical Computing (v3.5.1). Each biopsy sample’s digital gene expression (DGE) matrix (cells x genes) was trimmed to exclude low quality cells (&lt; 400 genes detected;&nbsp;&lt; 1,000&nbsp;UMIs; &gt; 50% mitochondrial reads) before being merged together (preserving all unique genes) to create the larger biopsy dataset. The merged dataset was further trimmed to remove cells with &gt; 8,000 genes which represent outliers and likely doublet cells. We also removed genes that were not detected in at least 50 cells. The same metrics were applied to the organoid single-cell cohort (see below). On a per cell basis, UMI count data was divided by total transcripts captured and multiplied by a scaling factor of 10,000. These normalized values were then natural log transformed for downstream analysis (i.e., log-normalized cell x gene matrix). Initial exploration of the data was performed using the R package Seurat (v2.3.4) and followed two steps: 1) SNN-guided quality assessment and 2) cell type composition determination. In step 1, we intentionally left cells in the DGE matrix of dubious quality (e.g., % mitochondrial reads &gt; 25% but&nbsp;&lt; 50%), performed PCA over the variable genes (n&nbsp;= 1,070 genes), and input the first 50 PCs (determined by Jackstraw analysis implemented through Seurat) to build an SNN graph and cluster the cells (res&nbsp;= 1; k.param&nbsp;= 40). The inclusion of poor-quality cells essentially acts as a variance “sink” for other poor-quality cells and they cluster together based on their shared patterns in quality-associated gene expression. This method helped to nominate additional low quality (e.g., defined exclusively by mitochondrial genes) or likely doublet cells (e.g., clusters defined by co-expression of divergent lineage markers) which were removed from the dataset (n&nbsp;= 1,678 cells). This led to an overall high-quality dataset of single-cells with a low overall fraction of mitochondrial reads (median&nbsp;= 0.09) for downstream analysis (Figure&nbsp;S1D).</div><div id="p0395" role="paragraph">Using the trimmed dataset, we proceeded to step 2 using a very similar workflow as above but with slightly altered input conditions for defining clusters. Here we used PCs 1-45 and their associated statistically significant genes for building the SNN graph and determining cluster membership (resolution&nbsp;= 1.2; k.param&nbsp;= 40). This identified the 36 clusters shown (visualized using <i>t</i>-SNE; perplexity, 40; iterations, 2,500) in Figure&nbsp;S1E. The expression of known markers was used to collapse clusters containing shared lineage information. For example, clusters 1, 2, and 4 all express high levels of macrophage markers—<i>CD14</i>, <i>FCGR3A (CD16)</i>, <i>CD68</i>—and were accordingly collapsed for this first pass analysis (Figure&nbsp;S1E,I). To aid our cell type identification, we performed an ROC test implemented in Seurat to confirm the specificity (power &gt; 0.6) of the top marker genes used to discern the cell types. Combined with inferred CNV information (see below), this analysis confirmed the presence of 11 broad non-malignant cell types in our biopsy dataset (Table S2). Variation in the SNN graph parameters above did not strongly affect cell type identification.</div></section><section id="sec-4-5-4"><h4>Single-cell CNV identification</h4><div id="p0400" role="paragraph">To confirm the identity of the putative malignant clusters identified in Figure&nbsp;S1F, we estimated single-cell CNVs as previously described by computing the average expression in a sliding window of 100 genes within each chromosome after sorting the detected genes by their chromosomal coordinates (<span class="dropBlock reference-citations"><a id="body-ref-sref57-3" href="#" role="doc-biblioref" data-xml-rid="bib57" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref57-3" aria-controls="sref57-3">Patel et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref57-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">57.</div><div>Patel, A.P. ∙ Tirosh, I. ∙ Trombetta, J.J. ...</div><div><strong>Single-cell RNA-seq highlights intratumoral heterogeneity in primary glioblastoma</strong></div><div><em>Science.</em> 2014; <strong>344</strong>:1396-1401</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.P.PatelI.TiroshJ.J.TrombettaA.K.ShalekS.M.GillespieH.WakimotoD.P.CahillB.V.NahedW.T.CurryR.L.MartuzaSingle-cell+RNA-seq+highlights+intratumoral+heterogeneity+in+primary+glioblastomaScience344201413961401" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref70-3" href="#" role="doc-biblioref" data-xml-rid="bib70" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref70-3" aria-controls="sref70-3">Tirosh et&nbsp;al., 2016b</a><div class="dropBlock__holder" data-db-target-of="sref70-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">70.</div><div>Tirosh, I. ∙ Venteicher, A.S. ∙ Hebert, C. ...</div><div><strong>Single-cell RNA-seq supports a developmental hierarchy in human oligodendroglioma</strong></div><div><em>Nature.</em> 2016; <strong>539</strong>:309-313</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=I.TiroshA.S.VenteicherC.HebertL.E.EscalanteA.P.PatelK.YizhakJ.M.FisherC.RodmanC.MountM.G.FilbinSingle-cell+RNA-seq+supports+a+developmental+hierarchy+in+human+oligodendrogliomaNature5392016309313" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). We used all T/NK, Fib, Hep, and Endo cells identified above as reference normal populations for this analysis. Complete information on the inferCNV workflow used for this analysis can be found here <a id="intref0135" href="https://github.com/broadinstitute/inferCNV/wiki" target="_blank">https://github.com/broadinstitute/inferCNV/wiki</a>. To compare with bulk targeted DNA-sequencing, we collapsed individual probes to cytoband-level information (weighted average of log2 ratios across each cytoband, see above) within each sample. ScRNA-seq-inferred CNVs showed high concordance across samples with the bulk measurements and suggests that, at least by this metric, we are likely sampling the same dominant clones within sequential but distinct cores from each needle biopsy procedure (Figure&nbsp;S1G). For plotting CNV profiles in putative malignant versus normal cells (Figure&nbsp;S1H), we computed the average CNV signal for the top 5% of altered cells in each biopsy and correlated all cells in that biopsy to the averaged profile as has been previously described (<span class="dropBlock reference-citations"><a id="body-ref-sref69-4" href="#" role="doc-biblioref" data-xml-rid="bib69" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref69-4" aria-controls="sref69-4">Tirosh et&nbsp;al., 2016a</a><div class="dropBlock__holder" data-db-target-of="sref69-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">69.</div><div>Tirosh, I. ∙ Izar, B. ∙ Prakadan, S.M. ...</div><div><strong>Dissecting the multicellular ecosystem of metastatic melanoma by single-cell RNA-seq</strong></div><div><em>Science.</em> 2016; <strong>352</strong>:189-196</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=I.TiroshB.IzarS.M.PrakadanM.H.Wadsworth2ndD.TreacyJ.J.TrombettaA.RotemC.RodmanC.LianG.MurphyDissecting+the+multicellular+ecosystem+of+metastatic+melanoma+by+single-cell+RNA-seqScience3522016189196" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Relation of this correlation coefficient to the CNV score (mean square deviation from diploidy) in the single cells from each biopsy shows consistent separation of malignant from non-malignant cells, and, combined with membership in patient-specific SNN clusters, substantiates the identification of malignant cells in our dataset. One patient sample, PANFR0604, did not contain any malignant cells within the core biopsy used for scRNA-seq analysis.</div></section><section id="sec-4-5-5"><h4>Subclonal analysis with single-cell inferred CNVs</h4><div id="p0405" role="paragraph">The inferCNV workflow can be used to call subclonal genomic variation with high sensitivity and is described at <a id="intref0140" href="https://github.com/broadinstitute/inferCNV/wiki" target="_blank">https://github.com/broadinstitute/inferCNV/wiki</a> (<span class="dropBlock reference-citations"><a id="body-ref-sref23" href="#" role="doc-biblioref" data-xml-rid="bib23" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref23" aria-controls="sref23">Fan et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref23"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">23.</div><div>Fan, J. ∙ Lee, H.O. ∙ Lee, S. ...</div><div><strong>Linking transcriptional and genetic tumor heterogeneity through allele analysis of single-cell RNA-seq data</strong></div><div><em>Genome Res.</em> 2018; <strong>28</strong>:1217-1227</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.FanH.O.LeeS.LeeD.E.RyuS.LeeC.XueS.J.KimK.KimN.BarkasP.J.ParkLinking+transcriptional+and+genetic+tumor+heterogeneity+through+allele+analysis+of+single-cell+RNA-seq+dataGenome+Res.28201812171227" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref57-4" href="#" role="doc-biblioref" data-xml-rid="bib57" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref57-4" aria-controls="sref57-4">Patel et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref57-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">57.</div><div>Patel, A.P. ∙ Tirosh, I. ∙ Trombetta, J.J. ...</div><div><strong>Single-cell RNA-seq highlights intratumoral heterogeneity in primary glioblastoma</strong></div><div><em>Science.</em> 2014; <strong>344</strong>:1396-1401</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.P.PatelI.TiroshJ.J.TrombettaA.K.ShalekS.M.GillespieH.WakimotoD.P.CahillB.V.NahedW.T.CurryR.L.MartuzaSingle-cell+RNA-seq+highlights+intratumoral+heterogeneity+in+primary+glioblastomaScience344201413961401" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref70-4" href="#" role="doc-biblioref" data-xml-rid="bib70" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref70-4" aria-controls="sref70-4">Tirosh et&nbsp;al., 2016b</a><div class="dropBlock__holder" data-db-target-of="sref70-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">70.</div><div>Tirosh, I. ∙ Venteicher, A.S. ∙ Hebert, C. ...</div><div><strong>Single-cell RNA-seq supports a developmental hierarchy in human oligodendroglioma</strong></div><div><em>Nature.</em> 2016; <strong>539</strong>:309-313</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=I.TiroshA.S.VenteicherC.HebertL.E.EscalanteA.P.PatelK.YizhakJ.M.FisherC.RodmanC.MountM.G.FilbinSingle-cell+RNA-seq+supports+a+developmental+hierarchy+in+human+oligodendrogliomaNature5392016309313" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Briefly, we used a six-state Hidden Markov Model (i6-HMM) to predict relative copy number status (complete loss to &gt; 3x gain) across putative altered regions in each cell. A Bayesian latent mixture model then evaluated the posterior probability that a given copy number alteration is a true positive. We set a relatively stringent cutoff for this step (BayesMaxPNormal&nbsp;= 0.2) to only include high probability alterations for downstream clustering. The results of this filtered i6-HMM output were then used to cluster the single cells using Ward’s method. We used inferCNV’s “random trees” method to test for statistical significance (p&nbsp;&lt; 0.05, 100 random permutations for each split) at each tree bifurcation and only retained subclusters that had statistical evidence underlying the presumed heterogeneity. To track subclonal heterogeneity between biopsy and matched organoid cells in <a id="crosref2180" href="#fig3">Figure&nbsp;3</a>E and Figure&nbsp;S5E-K, the above workflow was implemented within each biopsy and the relevant matched organoid samples, essentially treating all cells as the same “tumor” and allowing the CNVs to determine cell sorting agnostic to sample-of-origin. The results of the HMM output can be used to infer gene-level information based on which genes are in the affected window. This (like the rest of the HMM workflow) is computed over groups of cells (e.g., samples or sub-clones) and used to map <i>KRAS</i> and other alterations to samples or sub-clones (<a id="crosref2185" href="#fig3">Figure&nbsp;3</a>E, Figure&nbsp;S5E-K).</div></section><section id="sec-4-5-6"><h4>Subclustering of malignant and non-malignant cells</h4><div id="p0410" role="paragraph">Detailed phenotyping required splitting the dataset into malignant and non-malignant fractions. After subsetting to only the malignant cells, we re-scaled the data and ran PCA including the first 35 PCs for SNN clustering and <i>t-</i>SNE visualization. This PCA was used to determine the PanNET identity for PANFR0580 (Figure&nbsp;S2A). After removing PANFR0580, we repeated the steps above and used this new PCA for the remainder of PDAC malignant cell analysis. Subsequent phenotyping for malignant cells is discussed below (Generation of expression signatures/scores). A similar approach was used for calling the non-malignant subsets in <a id="crosref2190" href="#fig5">Figure&nbsp;5</a>A. To determine the specific phenotypes within T/NK, macrophage, and mesenchymal populations, we separately subclustered these groups using PCs 1-20 and a resolution of 0.6 in each case. Of note, subclustering within the macrophages revealed a distinct cluster of cells co-expressing markers of both T/NK cells and macrophages (n&nbsp;= 491 cells). We discarded these cells as likely doublets, as have others, and re-ran the macrophage PCA and clustering (<span class="dropBlock reference-citations"><a id="body-ref-sref78-2" href="#" role="doc-biblioref" data-xml-rid="bib78" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref78-2" aria-controls="sref78-2">Zhang et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref78-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">78.</div><div>Zhang, L. ∙ Li, Z. ∙ Skrzypczynska, K.M. ...</div><div><strong>Single-Cell Analyses Inform Mechanisms of Myeloid-Targeted Therapies in Colon Cancer</strong></div><div><em>Cell.</em> 2020; <strong>181</strong>:442-459 e429</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=L.ZhangZ.LiK.M.SkrzypczynskaQ.FangW.ZhangS.A.O%E2%80%99BrienY.HeL.WangQ.ZhangA.KimSingle-Cell+Analyses+Inform+Mechanisms+of+Myeloid-Targeted+Therapies+in+Colon+CancerCell1812020442459+e429" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref79-2" href="#" role="doc-biblioref" data-xml-rid="bib79" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref79-2" aria-controls="sref79-2">Zilionis et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref79-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">79.</div><div>Zilionis, R. ∙ Engblom, C. ∙ Pfirschke, C. ...</div><div><strong>Single-Cell Transcriptomics of Human and Mouse Lung Cancers Reveals Conserved Myeloid Populations across Individuals and Species</strong></div><div><em>Immunity.</em> 2019; <strong>50</strong>:1317-1334 e1310</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=R.ZilionisC.EngblomC.PfirschkeV.SavovaD.ZemmourH.D.SaatciogluI.KrishnanG.MaroniC.V.MeyerovitzC.M.KerwinSingle-Cell+Transcriptomics+of+Human+and+Mouse+Lung+Cancers+Reveals+Conserved+Myeloid+Populations+across+Individuals+and+SpeciesImmunity50201913171334+e1310" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). These cells are included in the full dataset in case they are of interest to others. Each unbiased analysis helped to define the non-malignant phenotypes summarized in <a id="crosref2205" href="#fig5">Figure&nbsp;5</a> and Figure&nbsp;S6.</div></section><section id="sec-4-5-7"><h4>Generation of expression signatures/scores</h4><div id="p0415" role="paragraph">All expression scores were computed as previously described by taking a given input set of genes and comparing their average relative expression to that of a control set (n&nbsp;= 100 genes) randomly sampled to mirror the expression distribution of the genes used for the input (<span class="dropBlock reference-citations"><a id="body-ref-sref70-5" href="#" role="doc-biblioref" data-xml-rid="bib70" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref70-5" aria-controls="sref70-5">Tirosh et&nbsp;al., 2016b</a><div class="dropBlock__holder" data-db-target-of="sref70-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">70.</div><div>Tirosh, I. ∙ Venteicher, A.S. ∙ Hebert, C. ...</div><div><strong>Single-cell RNA-seq supports a developmental hierarchy in human oligodendroglioma</strong></div><div><em>Nature.</em> 2016; <strong>539</strong>:309-313</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=I.TiroshA.S.VenteicherC.HebertL.E.EscalanteA.P.PatelK.YizhakJ.M.FisherC.RodmanC.MountM.G.FilbinSingle-cell+RNA-seq+supports+a+developmental+hierarchy+in+human+oligodendrogliomaNature5392016309313" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). While all scores were computed in the same way, choosing the genes for input varied. We have outlined the relevant approaches below. Where correlations (Pearson’s <i>r)</i> are performed over genes, we used the log-transformed UMI count data for each case. Unless otherwise noted, we selected the top 30 statistically significant genes for each signature (&gt; 3 SD above the mean for shuffled data) for visualization and scoring.</div><section id="sec-4-5-7-1"><h5>Cell cycle</h5><div id="p0420" role="paragraph">We utilized previously established signatures for G1/S (n&nbsp;= 43 genes) and G2/M (n&nbsp;= 55 genes) to place each cell along this dynamic process (<span class="dropBlock reference-citations"><a id="body-ref-sref69-5" href="#" role="doc-biblioref" data-xml-rid="bib69" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref69-5" aria-controls="sref69-5">Tirosh et&nbsp;al., 2016a</a><div class="dropBlock__holder" data-db-target-of="sref69-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">69.</div><div>Tirosh, I. ∙ Izar, B. ∙ Prakadan, S.M. ...</div><div><strong>Dissecting the multicellular ecosystem of metastatic melanoma by single-cell RNA-seq</strong></div><div><em>Science.</em> 2016; <strong>352</strong>:189-196</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=I.TiroshB.IzarS.M.PrakadanM.H.Wadsworth2ndD.TreacyJ.J.TrombettaA.RotemC.RodmanC.LianG.MurphyDissecting+the+multicellular+ecosystem+of+metastatic+melanoma+by+single-cell+RNA-seqScience3522016189196" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). After inspecting the distribution of scores in the complete dataset, we considered any cell &gt; 1.5 SD above the mean for either the G1/S or the G2/M scores to be cycling (<span class="dropBlock reference-citations"><a id="body-ref-sref72-2" href="#" role="doc-biblioref" data-xml-rid="bib72" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref72-2" aria-controls="sref72-2">van Galen et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref72-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">72.</div><div>van Galen, P. ∙ Hovestadt, V. ∙ Wadsworth Ii, M.H. ...</div><div><strong>Single-Cell RNA-Seq Reveals AML Hierarchies Relevant to Disease Progression and Immunity</strong></div><div><em>Cell.</em> 2019; <strong>176</strong>:1265-1281 e1224</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=P.van+GalenV.HovestadtM.H.Wadsworth+IiT.K.HughesG.K.GriffinS.BattagliaJ.A.VergaJ.StephanskyT.J.PastikaJ.Lombardi+StorySingle-Cell+RNA-Seq+Reveals+AML+Hierarchies+Relevant+to+Disease+Progression+and+ImmunityCell176201912651281+e1224" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div></section><section id="sec-4-5-7-2"><h5>PDAC bulk subtype signatures</h5><div id="p0425" role="paragraph">We scored malignant cells within our single-cell cohort for expression of previously published signatures derived from bulk RNA-sequencing of primary and metastatic tumors.</div></section><section id="sec-4-5-7-3"><h5>scBasal and scClassical programs</h5><div id="p0430" role="paragraph">We first scored each malignant single cell for the basal-like and classical genes identified by Moffitt et&nbsp;al., 2015 as these were well described by unbiased analysis in our data (PCA, Figure&nbsp;S2B,C, S3B). To derive refined single-cell basal (scBasal) and single-cell classical (scClassical) signatures using our malignant cohort and determine programs associated with these cell states, we correlated the aforementioned basal and classical scores to the entire gene expression matrix containing malignant cells and identified the 1,909 genes significantly associated with either subtype (<i>r</i> &gt; 0.1; &gt; 3 SD above the mean for shuffled data, full data in Table S3). Biological pathway correlates for scBasal and scClassical are summarized in Figure&nbsp;S2E,F [WNT signaling (<span class="dropBlock reference-citations"><a id="body-ref-sref42" href="#" role="doc-biblioref" data-xml-rid="bib42" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref42" aria-controls="sref42">Kim et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref42"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">42.</div><div>Kim, J.H. ∙ Park, S.Y. ∙ Jun, Y. ...</div><div><strong>Roles of Wnt Target Genes in the Journey of Cancer Stem Cells</strong></div><div><em>Int. J. Mol. Sci.</em> 2017; <strong>18</strong>:1604</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.H.KimS.Y.ParkY.JunJ.Y.KimJ.S.NamRoles+of+Wnt+Target+Genes+in+the+Journey+of+Cancer+Stem+CellsInt.+J.+Mol.+Sci.1820171604" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>); EMT (<span class="dropBlock reference-citations"><a id="body-ref-sref31-2" href="#" role="doc-biblioref" data-xml-rid="bib31" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref31-2" aria-controls="sref31-2">Gröger et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref31-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">31.</div><div>Gröger, C.J. ∙ Grubinger, M. ∙ Waldhör, T. ...</div><div><strong>Meta-analysis of gene expression signatures defining the epithelial to mesenchymal transition during cancer progression</strong></div><div><em>PLoS ONE.</em> 2012; <strong>7</strong>:e51136</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=C.J.Gr%C3%B6gerM.GrubingerT.Waldh%C3%B6rK.VierlingerW.MikulitsMeta-analysis+of+gene+expression+signatures+defining+the+epithelial+to+mesenchymal+transition+during+cancer+progressionPLoS+ONE72012e51136" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>); cell cycle progression (<span class="dropBlock reference-citations"><a id="body-ref-sref69-6" href="#" role="doc-biblioref" data-xml-rid="bib69" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref69-6" aria-controls="sref69-6">Tirosh et&nbsp;al., 2016a</a><div class="dropBlock__holder" data-db-target-of="sref69-6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">69.</div><div>Tirosh, I. ∙ Izar, B. ∙ Prakadan, S.M. ...</div><div><strong>Dissecting the multicellular ecosystem of metastatic melanoma by single-cell RNA-seq</strong></div><div><em>Science.</em> 2016; <strong>352</strong>:189-196</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=I.TiroshB.IzarS.M.PrakadanM.H.Wadsworth2ndD.TreacyJ.J.TrombettaA.RotemC.RodmanC.LianG.MurphyDissecting+the+multicellular+ecosystem+of+metastatic+melanoma+by+single-cell+RNA-seqScience3522016189196" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)]. For visualization, we use the scBasal and scClassical genes (top 30 correlated genes for each). In <a id="crosref2240" href="#fig2">Figure&nbsp;2</a>C we score single cells for EMT (<span class="dropBlock reference-citations"><a id="body-ref-sref31-3" href="#" role="doc-biblioref" data-xml-rid="bib31" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref31-3" aria-controls="sref31-3">Gröger et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref31-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">31.</div><div>Gröger, C.J. ∙ Grubinger, M. ∙ Waldhör, T. ...</div><div><strong>Meta-analysis of gene expression signatures defining the epithelial to mesenchymal transition during cancer progression</strong></div><div><em>PLoS ONE.</em> 2012; <strong>7</strong>:e51136</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=C.J.Gr%C3%B6gerM.GrubingerT.Waldh%C3%B6rK.VierlingerW.MikulitsMeta-analysis+of+gene+expression+signatures+defining+the+epithelial+to+mesenchymal+transition+during+cancer+progressionPLoS+ONE72012e51136" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and the union of Hallmark and Reactome interferon response gene sets to show their divergence within cells expressing the scBasal state.</div></section><section id="sec-4-5-7-4"><h5>Intermediate co-expressor (IC) program</h5><div id="p0435" role="paragraph">Ordering the cells by their polarization or “score difference,” simply the difference of the two scores, using these basal and classical scores related to PC1 and PC2 revealed a significant fraction of cells co-expressing intermediate levels of both cell states (<a id="crosref2250" href="#fig2">Figure&nbsp;2</a>B, Figure&nbsp;S3A,B). Co-expressing cells showed associations with features across several additional PCs, but lacked a single dominant axis. To define a consensus set of genes that are preferentially expressed by coexpressing cells, we computed the Euclidean distance to the line representing equal basal and classical co-expression for each cell. To limit the influence of cell quality on this analysis and to specifically identify genes related to co-expression, we used cells from each group (basal, intermediate, and classical) with fractionally low mitochondrial genes (&lt; 0.2) and non-zero basal or classical expression (basal or classical score &gt; 0) and correlated their Euclidean distance (Figure&nbsp;S3C) to the entire gene expression matrix of malignant cells. Next, for each gene positively associated with this co-expressor state (Pearson’s <i>r</i> &gt; 0), we subtracted the second highest correlation coefficient for each subtype-associated gene (basal and classical), and then re-ranked the matrix by this corrected value. This enriched for genes more specific to the co-expressor state by excluding those that were also associated with basal or classical programs. We then selected the 115 genes with a corrected correlation value &gt; 0.1 (p&nbsp;&lt; <i>0.00001</i>, shuffled data) as our intermediate co-expressor (IC) signature (Figure&nbsp;S3D, Table S3). Single cells were classified based on Euclidian distance to co-expression, where cells with Euclidian distance&nbsp;&lt; 0.2 are defined as intermediate co-expressor and the remainder (Euclidian distance &gt; 0.2) by their maximal of either scBasal or scClassical scores. We binned each organoid cell (<a id="crosref2255" href="#fig4">Figure&nbsp;4</a>C,D) by its maximal expression for one of the 3 <i>in&nbsp;vivo</i> scores (scBasal, scClassical, or IC). Here&nbsp;a cell must be within 1 SD of the mean expression for a given subtype <i>in&nbsp;vivo</i>, else it was considered “organoid-specific” as this program was superimposed on all organoid cells, regardless of their subtype identity (<a id="crosref2260" href="#fig4">Figure&nbsp;4</a>C). We used these classifications to summarize overall sample malignant cell composition and visualize the groups. Tumor heterogeneity measures were not significantly affected by changing these cutoffs.</div></section><section id="sec-4-5-7-5"><h5>Non-Malignant programs</h5><div id="p0440" role="paragraph">TAM signatures were determined similar to above and previous work (<span class="dropBlock reference-citations"><a id="body-ref-sref72-3" href="#" role="doc-biblioref" data-xml-rid="bib72" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref72-3" aria-controls="sref72-3">van Galen et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref72-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">72.</div><div>van Galen, P. ∙ Hovestadt, V. ∙ Wadsworth Ii, M.H. ...</div><div><strong>Single-Cell RNA-Seq Reveals AML Hierarchies Relevant to Disease Progression and Immunity</strong></div><div><em>Cell.</em> 2019; <strong>176</strong>:1265-1281 e1224</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=P.van+GalenV.HovestadtM.H.Wadsworth+IiT.K.HughesG.K.GriffinS.BattagliaJ.A.VergaJ.StephanskyT.J.PastikaJ.Lombardi+StorySingle-Cell+RNA-Seq+Reveals+AML+Hierarchies+Relevant+to+Disease+Progression+and+ImmunityCell176201912651281+e1224" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref78-3" href="#" role="doc-biblioref" data-xml-rid="bib78" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref78-3" aria-controls="sref78-3">Zhang et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref78-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">78.</div><div>Zhang, L. ∙ Li, Z. ∙ Skrzypczynska, K.M. ...</div><div><strong>Single-Cell Analyses Inform Mechanisms of Myeloid-Targeted Therapies in Colon Cancer</strong></div><div><em>Cell.</em> 2020; <strong>181</strong>:442-459 e429</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=L.ZhangZ.LiK.M.SkrzypczynskaQ.FangW.ZhangS.A.O%E2%80%99BrienY.HeL.WangQ.ZhangA.KimSingle-Cell+Analyses+Inform+Mechanisms+of+Myeloid-Targeted+Therapies+in+Colon+CancerCell1812020442459+e429" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref79-3" href="#" role="doc-biblioref" data-xml-rid="bib79" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref79-3" aria-controls="sref79-3">Zilionis et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref79-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">79.</div><div>Zilionis, R. ∙ Engblom, C. ∙ Pfirschke, C. ...</div><div><strong>Single-Cell Transcriptomics of Human and Mouse Lung Cancers Reveals Conserved Myeloid Populations across Individuals and Species</strong></div><div><em>Immunity.</em> 2019; <strong>50</strong>:1317-1334 e1310</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=R.ZilionisC.EngblomC.PfirschkeV.SavovaD.ZemmourH.D.SaatciogluI.KrishnanG.MaroniC.V.MeyerovitzC.M.KerwinSingle-Cell+Transcriptomics+of+Human+and+Mouse+Lung+Cancers+Reveals+Conserved+Myeloid+Populations+across+Individuals+and+SpeciesImmunity50201913171334+e1310" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Using PCA as an anchor (Figure&nbsp;S6C), we correlated expression within the TAM compartment to either <i>FCN1</i>, <i>SPP1</i>, or <i>C1QC</i> (top loaded genes on each relevant PC) and merged the resultant correlation coefficients for every detected gene to the 3 subtypes into one matrix (i.e., a 16,920&nbsp;× 3 matrix). For each TAM type (i.e., vector of correlation coeffects to each marker), we first ranked the matrix by decreasing correlation coefficient, selected only the most significantly associated genes to that type (<i>r</i> &gt; 0.1; &gt; 3 SD above the mean for shuffled data), subtracted the second highest correlation coefficient for each subtype-associated gene, and then re-ranked the matrix by this corrected value. We repeated this procedure for each TAM subtype independently. This ensures that the genes selected are specific to a given TAM subset and do not describe general TAM features. The top 30 genes for each were used for scoring and visualization (Table S2; Figure&nbsp;S6D).</div><div id="p0445" role="paragraph">Mesenchymal phenotypes were determined using a similar workflow. To examine mesenchymal heterogeneity, we removed a subset of adrenal endocrine cells (cluster 4, 40 cells; <a id="crosref2280" href="#fig5">Figure&nbsp;5</a>C) and then performed PCA on the remaining mesenchymal cells. PC1 was driven by spillover genes (likely contributed from ambient RNA) and lacked any coherent biological program and was not considered further. PCs 2 and 3 by contrast were consistent with variable mesenchymal (PC2) and inflammatory CAF (PC3) phenotypes. All these cells scored highly for previous myCAF gene expression programs so this phenotype did not fully explain the heterogeneity in mesenchymal cells. Again, using correlation, we determined the genes driving low PC2 scores (Fibroblast-like), and high PC2 scores (Pericyte-like), as well as those associated with the high PC3 scores (Inflammatory). As before, we used the top 30 genes within each subset for scoring and visualization. These same genes (Fibroblast-like and Pericyte-like) were used to examine bulk RNA-seq profiles and their difference in each sample quantifies which phenotype is favored in the bulk averages (<a id="crosref2285" href="#fig5">Figure&nbsp;5</a>C).</div></section></section><section id="sec-4-5-8"><h4>Analysis of normal pancreas progenitor data</h4><div id="p0450" role="paragraph">We obtained the genes by cells matrix of normal pancreas progenitors (<span class="dropBlock reference-citations"><a id="body-ref-sref61-4" href="#" role="doc-biblioref" data-xml-rid="bib61" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref61-4" aria-controls="sref61-4">Qadir et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref61-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">61.</div><div>Qadir, M.M.F. ∙ Álvarez-Cubela, S. ∙ Klein, D. ...</div><div><strong>Single-cell resolution analysis of the human pancreatic ductal progenitor cell niche</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2020; <strong>117</strong>:10876-10887</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.M.F.QadirS.%C3%81lvarez-CubelaD.KleinJ.van+DijkR.Mu%C3%B1iz-AnquelaY.B.Moreno-Hern%C3%A1ndezG.LanzoniS.SadiqB.Navarro-RubioM.T.Garc%C3%ADaSingle-cell+resolution+analysis+of+the+human+pancreatic+ductal+progenitor+cell+nicheProc.+Natl.+Acad.+Sci.+USA11720201087610887" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) at <a id="intref0145" href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE131886" target="_blank">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE131886</a>. We clustered the original data and excluded a small subset of immune cells (CD45+). We then collapsed cell types from the original paper into broad categories (Pro-Acinar, Pro-ductal, Undifferentiated, and Mesenchymal) based on lineage marker expression. For analysis in Figure&nbsp;S3E we averaged the expression for scBasal, scClassical and IC genes in each group. For use in Figure&nbsp;S3F we generated signatures for each population using differential expression (FindAllMarkers function in Seurat using the “wilcox” option) and scored our single cells for these normal-derived signatures as above.</div></section><section id="sec-4-5-9"><h4>Matched organoid clustering and cell-typing</h4><div id="p0455" role="paragraph">After applying similar quality metrics as above, we performed PCA, SNN clustering, and <i>t</i>-SNE embedding for 32,073 cells including organoid cells and all malignant cells from primary PDAC biopsies (PCs 1-50; resolution&nbsp;= 1.2; k.param&nbsp;= 45; perplexity&nbsp;= 45; max_iter&nbsp;= 2,500), and identified 39 total clusters. Organoids clustered separately from their matched biopsies, suggesting expression and/or CNV related drift in culture. Only two SNN clusters—clusters 4 and 32—were admixed by sample. We determined the specific gene expression programs in these two clusters via differential expression testing by Wilcoxon rank sum test (p&nbsp;&lt; 0.05, Bonferroni correction; log(fold change) &gt; 0.5). These comparisons were done in a “1 versus rest” fashion, testing for genes defining each cluster (4 or 32) compared to the entire dataset. Their expression profiles were consistent with non-malignant cell types, likely fibroblasts (cluster 32) and epithelial cells (cluster 4; Figure&nbsp;S5B,C).</div></section><section id="sec-4-5-10"><h4>Correlation distances for genotype and transcriptional cell state</h4><div id="p0460" role="paragraph">To generate correlation distances for genotype and transcriptional cell state, each single cell in a biopsy-organoid pair was represented by two vectors of information: (i) a cell state vector containing expression values for scBasal and scClassical genes (n&nbsp;= 60 genes) and (ii) a genotype vector containing the average CNV score for each cytoband. The cell state and genotype distances between every single cell within a biopsy/early organoid pair was computed from these vectors using a correlation-based (Pearson’s <i>r</i>) distance metric of the form <i>d</i>&nbsp;= (1-<i>r</i>)/2. This resulted in two distance matrices of n x n dimension where n is the total number of cells from each biopsy/early organoid sample pair. Values in <a id="crosref2295" href="#fig3">Figure&nbsp;3</a>A are computed by averaging the values for <i>d</i> between only early organoid and matched biopsy cells.</div></section><section id="sec-4-5-11"><h4>Matched biopsy versus organoid malignant cell comparison</h4><div id="p0465" role="paragraph">For CNV-confirmed malignant cells from each biopsy and its matched organoid (earliest passage), we used differential expression (Wilcoxon rank sum test; p&nbsp;&lt; 0.05, Bonferroni correction; log(fold change) &gt; 0.3) to understand the features lost from malignant cells in the <i>in&nbsp;vivo</i> setting and gained when transitioning into growth in organoid culture. We required any gene to be significantly differentially expressed in at least 3 model-biopsy comparisons to summarize the consistent changes. We repeated this same workflow for both organoid- and biopsy-specific genes (Table S4) outlined in <a id="crosref2300" href="#fig3">Figure&nbsp;3</a>C and <a id="crosref2305" href="#fig4">Figure&nbsp;4</a>G,H, respectively.</div></section><section id="sec-4-5-12"><h4>TME associations</h4><div id="p0470" role="paragraph">We determined the transcriptional-subtype-dependent composition of the TME (<a id="crosref2310" href="#fig5">Figure&nbsp;5</a>E-I) following two steps. First, we computed the Simpson’s Index (measure of ecological diversity) using the count of each non-malignant cell type captured from each sample as input (<a id="crosref2315" href="#fig5">Figure&nbsp;5</a>E,G) and correlated each biopsy’s diversity score to its scBasal versus scClassical commitment score. Importantly, the number of non-malignant cells captured from each biopsy was not associated with basal versus classical commitment score (r&nbsp;= 0.09). Next, to understand which cell types drive these differences, we computed the fractional representation for every non-malignant cell type in each core needle biopsy and determined their pairwise correlation distance (Pearson’s <i>r</i>) followed by hierarchical clustering using Ward’s method (dendrogram in <a id="crosref2320" href="#fig5">Figure&nbsp;5</a>G). For both analyses we only used samples with &gt; 200 non-malignant cells captured (Figure&nbsp;S6N).</div></section><section id="sec-4-5-13"><h4>Biopsy paracrine and autocrine subtype-specific factor analysis</h4><div id="p0475" role="paragraph">Factors present in the TME but absent from organoid culture could originate from at least two sources, the malignant cells themselves (autocrine) or non-malignant cells in the local microenvironment (paracrine). We examined any gene with gene ontology annotations related to “cytokines,” “chemokines,” or “growth factors” and took the union of these lists, yielding 321 genes, 218 of which were detected in our dataset. For “autocrine” factors we performed differential expression between malignant cells binned as scBasal and scClassical, and then IC versus rest. A gene was considered differentially expressed if it passed a p&nbsp;&lt; 0.05 with Bonferroni correction and a log(fold change) &gt; 0.2 in one of these comparisons. Genes were then assigned to subtypes based on the log fold change direction (<a id="crosref2325" href="#fig6">Figure&nbsp;6</a>B, Table S6). “Paracrine” factors were determined in a similar manner with slight modifications. We grouped non-malignant cells into basal, classical or IC based on the average expression and clustering for malignant programs from their respective tumor samples (Figure&nbsp;S3G). We then assessed for differential expression between all cells from a given group and the rest using the same cutoffs as above and sorted factors into subtypes based on their log fold change directionality (<a id="crosref2330" href="#fig7">Figure&nbsp;7</a>A, Table S6). We visualized which cell type contributed the highest average expression for each factor among the cell types from each of the respective cell state-specific TMEs (<a id="crosref2335" href="#fig7">Figure&nbsp;7</a>B). We note that our use of “paracrine” and “autocrine” here is somewhat inexact as these secreted factors could act in either manner depending on the context. We merely use this nomenclature to reflect a “cancer cell centric” view, i.e., factors secreted by malignant cells are autocrine and those deriving from the TME are paracrine.</div></section><section id="sec-4-5-14"><h4>Tumor phenotyping from mIF data</h4><div id="p0480" role="paragraph">Supervised machine learning algorithms were applied for tissue and cell segmentation (inForm 2.4.1, Akoya Biosciences). Single-cell-level imaging data were exported and further processed and analyzed using R (v3.6.2). To assign phenotypes to individual malignant epithelial cells, mean expression intensity in the relevant subcellular compartment was first used to classify cells as positive or negative for each of the 5 markers. Combinatorial expression patterns for the five markers were then used to phenotypically classify cells as basal, classical, co-expressing / IC or marker negative (3 combinations of 2 basal markers, 7 combinations of 3 classical markers, 1 pan-marker negative, 21 combinations of co-expression of basal and classical markers, Figure&nbsp;S4A, Table S3). Tumor subtype composition was assessed by calculating the fraction of total malignant cells positive for each cell phenotype (Figure&nbsp;S4B, excluding pan-marker negative cells).</div></section></section></section><section id="acknowledgments" role="doc-acknowledgments"><h2>Acknowledgments</h2><div id="p0485" role="paragraph">This work was funded by the Lustgarten Foundation Dedicated Laboratory (B.M.W. and A.J.A.), Dana-Farber Cancer Institute Hale Center for Pancreatic Cancer Research (A.J.A., B.M.W., and W.C.H.), the Doris Duke Charitable Foundation (A.J.A.), Pancreatic Cancer Action Network (A.J.A. and B.M.W.), NIH-NCI K08 CA218420-02 (A.J.A.), P50 CA127003 (A.J.A., B.M.W., and G.I.S.), U01 CA176058 (W.C.H.), U01 CA224146 (W.C.H. and A.J.A.), U01 CA210171 (B.M.W.), U01 28020510 (W.C.H. and A.K.S.), 1U2CCA23319501 (A.K.S.), U54 CA217377 (W.C.H., S.R.M., and A.K.S.), U01 CA250549 (W.C.H., S.R., and A.J.A.), Stand Up To Cancer (B.M.W.), Noble Effort Fund (B.M.W.), Wexler Family Fund (B.M.W.), Promises for Purple (B.M.W.), Hope Funds for Cancer Research Postdoctoral Fellowship (S.R.), Harvard Catalyst/The Harvard Clinical and Translational Science Center UL1 TR002541 (S.R.), Koch Institute for Integrative Cancer Research at MIT and the Dana-Farber/Harvard Cancer Center Bridge Project (W.C.H., S.R.M., and S.R.), Dana-Farber Cancer Institute Gloria Spivak Faculty Advancement Fund (S.R.), Hopper-Belmont Foundation Inspiration Award (S.R.), Marcotte Center for Cancer Research (B.E.J.), Finnish Cultural Foundation (S.A.V.), Orion Research Foundation sr (S.A.V.), Damon Runyon Cancer Research Foundation (A.M.J.), NIH K99 Award CA241072-01 (A.M.J.), Koch Institute Cancer Core Grant NIH P30 CA14051 (T.E.J. and A.M.J.), Broman Family Fund for Pancreatic Cancer Research (K.N.), Ludwig Center for Molecular Oncology at MIT (A.W.N.), Searle Scholars Program (A.K.S.), Beckman Young Investigator Program (A.K.S.), Sloan Fellowship in Chemistry (A.K.S.), and the Pew-Stewart Scholars Program for Cancer Research (A.K.S.). This work was supported by the National Cancer Institute’s Office of Cancer Genomics Cancer Target Discovery and Development (CTD<sup>∧</sup>2) initiative. We also thank SciStories for assistance with schematics and Bryan Bryson for helpful discussions of macrophage phenotypes.</div><section id="sec-5"><h3>Author contributions</h3><div id="p0490" role="paragraph">Conceptualization, S.R., P.S.W., B.M.W., W.C.H., A.J.A., A.K.S.; Methodology, S.R., P.S.W., A.W.N., H.L.W., L.C., J.A.N., B.M.W., W.C.H., A.J.A., A.K.S.; Validation, S.R., P.S.W., A.W.N., H.L.W., K.E.L., J.A.N., B.M.W., W.C.H., A.J.A., A.K.S.; Formal analysis, S.R., P.S.W., A.W.N., H.L.W., A.D., K.S.K., A.A.B., N.L., L.F.S., Y.Y.L., A.D. Cherniack, M.A.B., E.F.C., M.Y.T., J.M.M., L.C.; Investigation, S.R., P.S.W., A.W.N., H.L.W., K.E.L., J.G.-R., R.L.K., N.M., M.S.R., R.W.S.N., J.W., I.W.; Resources, S.A.V., A.D. Costa, E.K.H., D.Y.R., L.K.B., A.R., B.E.J., E.T.S., R.J.S., G.I.S, T.E.C., K.P., D.A.R., K.N., J.M.C., J.A.N., B.M.W.; Data Curation, P.S.W., S.R., A.W.N., H.L.W., K.S.K.; Writing – original draft, S.R., P.S.W., A.W.N., H.L.W.; Writing – review &amp; editing, S.R., P.S.W., A.W.N., H.L.W., A.M.J., L.C., J.A.N., S.R.M., B.M.W., W.C.H., A.J.A., A.K.S.; Visualization, S.R., P.S.W., A.W.N., H.L.W.; Supervision, B.E.J., T.E.J., L.C., J.A.N., S.R.M., B.M.W., W.C.H., A.J.A., A.K.S.; Project administration, S.R., P.S.W., B.M.W., W.C.H., A.J.A., A.K.S.; Funding acquisition, S.R., P.S.W., S.R.M., B.M.W., W.C.H., A.J.A., A.K.S.</div></section><section id="sec-6"><h3>Declaration of interests</h3><div id="p0495" role="paragraph">B.M.W. reports research support from Celgene and Eli Lilly and consulting for BioLineRx, Celgene, G1 Therapeutics, and GRAIL. W.C.H. is a consultant for Thermo Fisher, Solasta Ventures, MPM Capital, Tyra Biosciences, iTeos, Frontier Medicines, Function Oncology, KSQ Therapeutics, Jubilant Therapeutics, RAPPTA Therapeutics, and Paraxel. A.K.S. reports compensation for consulting and/or SAB membership from Merck, Honeycomb Biotechnologies, Cellarity, Repertoire Immune Medicines, Ochre Bio, Third Rock Ventures, Hovione, Relation Therapeutics, FL82, and Dahlia Biosciences. A.J.A. has consulted for Oncorus, Arrakis Therapeutics, and Merck &amp; Co. and has research funding from Mirati Therapeutics, Syros, Deerfield, and Novo Ventures that are unrelated to this project. S.R.M. is a founder of Travera. S.R. has equity in Amgen. J.M.C. has received research funding from Merck, Tesaro, AstraZeneca, Bayer, and Esperas Pharma; has served as a consultant to Bristol Myers Squibb; and has received travel funding from Bristol Myers Squibb. A.R. is an employee of AstraZeneca and an equity holder in Celsius Therapeutics and NucleAI. Y.Y.L. reports equity from g.Root Biomedical Services. A.D. Cherniack reports research support from Bayer. R.J.S reports research support from Merck and consulting and/or SAB membership for Bristol Myers Squibb, Merck, Novartis, and Pfizer. G.I.S. reports sponsored research support from Eli Lilly, Merck KGaA/EMD-Serono, Merck, and Sierra Oncology and has served on advisory boards for Pfizer, Eli Lilly, G1 Therapeutics, Roche, Merck KGaA/EMD-Serono, Sierra Oncology, Bicycle Therapeutics, Fusion Pharmaceuticals, Cybrexa Therapeutics, Astex, Almac, Ipsen, Bayer, Angiex, Daiichi Sankyo, Seattle Genetics, Boehringer Ingelheim, ImmunoMet, Asana, Artios, Atrin, Concarlo Holdings, Syros, Zentalis, CytomX Therapeutics, Blueprint Medicines, and Kymera Therapeutics. T.J. is a member of the Board of Directors of Amgen and Thermo Fisher Scientific; a co-Founder of Dragonfly Therapeutics and T2 Biosystems; serves on the Scientific Advisory Board of Dragonfly Therapeutics, SQZ Biotech, and Skyhawk Therapeutics; and is the President of Break Through Cancer. None of these affiliations represent a conflict of interest with respect to the design or execution of this study or interpretation of data presented in this manuscript. T.J.’s laboratory currently also receives funding from the Johnson &amp; Johnson Lung Cancer Initiative and The Lustgarten Foundation for Pancreatic Cancer Research, but this funding did not support the research described in this manuscript. K.N. reports research support from Revolution Medicines, Evergrande Group, Pharmavite, and Janssen; is a member of SABs for SeaGen, BiomX, and Bicara Therapeutics; and has consulted for X-Biotix Therapeutics and Redesign Health. A.K.S., P.S.W., A.W.N., S.R., W.C.H., A.J.A., B.W.M., and J.G.R. have filed a patent (Publication number WO/2021/081491; International application number PCT/US2020/057342) related to this work.</div></section><section id="sec-7"><h3>Inclusion and diversity</h3><div id="p0500" role="paragraph">We worked to ensure gender balance in the recruitment of human subjects. We worked to ensure ethnic or other types of diversity in the recruitment of human subjects. We worked to ensure diversity in experimental samples through the selection of the cell lines. We worked to ensure diversity in experimental samples through the selection of the genomic datasets. One or more of the authors of this paper self-identifies as an underrepresented ethnic minority in science. One or more of the authors of this paper self-identifies as a member of the LGBTQ+ community. The author list of this paper includes contributors from the location where the research was conducted who participated in the data collection, design, analysis, and/or interpretation of the work.</div></section></section><section id="supplementary-material" data-type="supplementary-materials"><h2>Supplemental information (7)</h2><div id="p0510" role="paragraph"><div class="core-supplementary-material"><div class="core-link"><button class="downloadAllSupplementary"><span>Download all</span></button></div></div><div id="mmc1" class="core-supplementary-material" data-category="pdf"><div class="core-link"><a href="/cms/10.1016/j.cell.2021.11.017/attachment/1ca3e3ad-032c-4019-a9e3-69cc3d390193/mmc1.pdf" download="mmc1.pdf" aria-labelledby="mmc1-heading mmc1-link"><span id="mmc1-link">PDF (165.61 KB)</span></a></div><div class="core-description"><div id="mmc1-heading" class="heading">Table S1. Cohort patient characteristics, related to Figure&nbsp;1</div></div></div><div id="mmc2" class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="/cms/10.1016/j.cell.2021.11.017/attachment/c92d07b6-50d2-4ff3-b27e-2d56885c8814/mmc2.xlsx" download="mmc2.xlsx" aria-labelledby="mmc2-heading mmc2-link"><span id="mmc2-link">Spreadsheet (1.85 MB)</span></a></div><div class="core-description"><div id="mmc2-heading" class="heading">Table S2. Normal cell type markers, related to Figures 1, 5, and 7</div></div></div><div id="mmc3" class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="/cms/10.1016/j.cell.2021.11.017/attachment/bbbc8bb8-00aa-4bc6-9d8f-b27b3d3850ee/mmc3.xlsx" download="mmc3.xlsx" aria-labelledby="mmc3-heading mmc3-link"><span id="mmc3-link">Spreadsheet (1.70 MB)</span></a></div><div class="core-description"><div id="mmc3-heading" class="heading">Table S3. Malignant cell state single-cell gene correlates, signatures, mIF marker combinations, and cell counts, related to Figure&nbsp;2</div></div></div><div id="mmc4" class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="/cms/10.1016/j.cell.2021.11.017/attachment/b793ae79-62bc-40e5-afad-31cc46627715/mmc4.xlsx" download="mmc4.xlsx" aria-labelledby="mmc4-heading mmc4-link"><span id="mmc4-link">Spreadsheet (508.15 KB)</span></a></div><div class="core-description"><div id="mmc4-heading" class="heading">Table S4. Organoid- and <i>in&nbsp;vivo</i> malignant-specific gene expression features, related to Figures 3 and 4</div></div></div><div id="mmc5" class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="/cms/10.1016/j.cell.2021.11.017/attachment/31a1fefb-c62c-435d-9a8a-a77e749a910e/mmc5.xlsx" download="mmc5.xlsx" aria-labelledby="mmc5-heading mmc5-link"><span id="mmc5-link">Spreadsheet (13.98 KB)</span></a></div><div class="core-description"><div id="mmc5-heading" class="heading">Table S5. Organoid and cell line models and media formulations for perturbation experiments, related to Figures 3, 4, 6, and 7</div></div></div><div id="mmc6" class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="/cms/10.1016/j.cell.2021.11.017/attachment/366a4706-6157-4775-9768-e6e0510c48f6/mmc6.xlsx" download="mmc6.xlsx" aria-labelledby="mmc6-heading mmc6-link"><span id="mmc6-link">Spreadsheet (522.80 KB)</span></a></div><div class="core-description"><div id="mmc6-heading" class="heading">Table S6. Cell state-specific autocrine and paracrine secreted factors, related to Figures 6 &amp; 7</div></div></div><div id="mmc7" class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="/cms/10.1016/j.cell.2021.11.017/attachment/20f729e7-71ec-4f32-8afd-bf1300d63415/mmc7.xlsx" download="mmc7.xlsx" aria-labelledby="mmc7-heading mmc7-link"><span id="mmc7-link">Spreadsheet (39.12 KB)</span></a></div><div class="core-description"><div id="mmc7-heading" class="heading">Table S7. Drug response parameters for isogenic organoid models induced to adopt different cell states by altering media conditions, related to Figure&nbsp;6</div></div></div></div></section></div></section><div class="core-linked-content">
    

        

</div><section id="backmatter" data-extent="backmatter"><div class="core-container"><section id="data-availability" class="core-data-availability"><h2>Data and code availability</h2><div id="p0030" role="paragraph">De-identified single-cell RNA-seq data are publicly available for download and visualization via the Single Cell Portal: <a id="intref0010" href="https://singlecell.broadinstitute.org/single_cell/study/SCP1644" target="_blank">https://singlecell.broadinstitute.org/single_cell/study/SCP1644</a>. Genomic and transcriptomic data will be available at the NCBI Database of Genotypes and Phenotypes (dbGaP). This paper analyzes existing, publicly available data. The links and accession numbers for these datasets are listed in the <a id="crosref2555" href="#sec-4-1">key resources table</a>.</div><div id="p0035" role="paragraph">Code is available from the lead contact upon request.</div><div id="p0040" role="paragraph">Any additional information required to reanalyze the data reported in this paper is available from the lead contact upon request.</div></section><section id="references"><h2>References</h2><div id="bibliography" role="doc-bibliography"><div role="list"><div role="listitem" data-has="label"><div id="bib1" class="citations"><div id="sref1" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref1" title="View in article">1.</a></div><div>Abo, R.P. ∙ Ducar, M. ∙ Garcia, E.P. ...</div><div><strong>BreaKmer: detection of structural variation in targeted massively parallel sequencing data using kmers</strong></div><div><em>Nucleic Acids Res.</em> 2015; <strong>43</strong>:e19</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=R.P.AboM.DucarE.P.GarciaA.R.ThornerV.Rojas-RudillaL.LinL.M.ShollW.C.HahnM.MeyersonN.I.LindemanBreaKmer%3A+detection+of+structural+variation+in+targeted+massively+parallel+sequencing+data+using+kmersNucleic+Acids+Res.432015e19" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib2" class="citations"><div id="sref2" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref2-1" title="View in article">2.</a></div><div>Aguirre, A.J. ∙ Nowak, J.A. ∙ Camarda, N.D. ...</div><div><strong>Real-time Genomic Characterization of Advanced Pancreatic Cancer to Enable Precision Medicine</strong></div><div><em>Cancer Discov.</em> 2018; <strong>8</strong>:1096-1111</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.J.AguirreJ.A.NowakN.D.CamardaR.A.MoffittA.A.GhazaniM.Hazar-RethinamS.RaghavanJ.KimL.K.BraisD.RagonReal-time+Genomic+Characterization+of+Advanced+Pancreatic+Cancer+to+Enable+Precision+MedicineCancer+Discov.8201810961111" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib3" class="citations"><div id="sref3" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref3" title="View in article">3.</a></div><div>Alonso-Curbelo, D. ∙ Ho, Y.J. ∙ Burdziak, C. ...</div><div><strong>A gene-environment-induced epigenetic program initiates tumorigenesis</strong></div><div><em>Nature.</em> 2021; <strong>590</strong>:642-648</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=D.Alonso-CurbeloY.J.HoC.BurdziakJ.L.V.MaagJ.P.Morris4thR.ChandwaniH.A.ChenK.M.TsanovF.M.BarrigaW.LuanA+gene-environment-induced+epigenetic+program+initiates+tumorigenesisNature5902021642648" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib4" class="citations"><div id="sref4" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref4-1" title="View in article">4.</a></div><div>Aung, K.L. ∙ Fischer, S.E. ∙ Denroche, R.E. ...</div><div><strong>Genomics-Driven Precision Medicine for Advanced Pancreatic Cancer: Early Results from the COMPASS Trial</strong></div><div><em>Clin. Cancer Res.</em> 2018; <strong>24</strong>:1344-1354</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=K.L.AungS.E.FischerR.E.DenrocheG.H.JangA.DoddS.CreightonB.SouthwoodS.B.LiangD.ChadwickA.ZhangGenomics-Driven+Precision+Medicine+for+Advanced+Pancreatic+Cancer%3A+Early+Results+from+the+COMPASS+TrialClin.+Cancer+Res.24201813441354" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib5" class="citations"><div id="sref5" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref5-1" title="View in article">5.</a></div><div>Bailey, P. ∙ Chang, D.K. ∙ Nones, K. ..., Australian Pancreatic Cancer Genome Initiative</div><div><strong>Genomic analyses identify molecular subtypes of pancreatic cancer</strong></div><div><em>Nature.</em> 2016; <strong>531</strong>:47-52</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=P.BaileyD.K.ChangK.NonesA.L.JohnsA.M.PatchM.C.GingrasD.K.MillerA.N.ChristT.J.BruxnerM.C.QuinnAustralian+Pancreatic+Cancer+Genome+InitiativeGenomic+analyses+identify+molecular+subtypes+of+pancreatic+cancerNature53120164752" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib6" class="citations"><div id="sref6" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref6" title="View in article">6.</a></div><div>Balachandran, V.P. ∙ Beatty, G.L. ∙ Dougan, S.K.</div><div><strong>Broadening the Impact of Immunotherapy to Pancreatic Cancer: Challenges and Opportunities</strong></div><div><em>Gastroenterology.</em> 2019; <strong>156</strong>:2056-2072</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=V.P.BalachandranG.L.BeattyS.K.DouganBroadening+the+Impact+of+Immunotherapy+to+Pancreatic+Cancer%3A+Challenges+and+OpportunitiesGastroenterology156201920562072" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib7" class="citations"><div id="sref7" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref7" title="View in article">7.</a></div><div>Bartoschek, M. ∙ Oskolkov, N. ∙ Bocci, M. ...</div><div><strong>Spatially and functionally distinct subclasses of breast cancer-associated fibroblasts revealed by single cell RNA sequencing</strong></div><div><em>Nat. Commun.</em> 2018; <strong>9</strong>:5150</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.BartoschekN.OskolkovM.BocciJ.L%C3%B6vrotC.LarssonM.SommarinC.D.MadsenD.LindgrenG.PekarG.KarlssonSpatially+and+functionally+distinct+subclasses+of+breast+cancer-associated+fibroblasts+revealed+by+single+cell+RNA+sequencingNat.+Commun.920185150" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib8" class="citations"><div id="sref8" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref8" title="View in article">8.</a></div><div>Ben-David, U. ∙ Siranosian, B. ∙ Ha, G. ...</div><div><strong>Genetic and transcriptional evolution alters cancer cell line drug response</strong></div><div><em>Nature.</em> 2018; <strong>560</strong>:325-330</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=U.Ben-DavidB.SiranosianG.HaH.TangY.OrenK.HinoharaC.A.StrathdeeJ.DempsterN.J.LyonsR.BurnsGenetic+and+transcriptional+evolution+alters+cancer+cell+line+drug+responseNature5602018325330" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib9" class="citations"><div id="sref9" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref9-1" title="View in article">9.</a></div><div>Ben-David, U. ∙ Beroukhim, R. ∙ Golub, T.R.</div><div><strong>Genomic evolution of cancer models: perils and opportunities</strong></div><div><em>Nat. Rev. Cancer.</em> 2019; <strong>19</strong>:97-109</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=U.Ben-DavidR.BeroukhimT.R.GolubGenomic+evolution+of+cancer+models%3A+perils+and+opportunitiesNat.+Rev.+Cancer19201997109" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib10" class="citations"><div id="sref10" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref10" title="View in article">10.</a></div><div>Bernard, V. ∙ Semaan, A. ∙ Huang, J. ...</div><div><strong>Single-Cell Transcriptomics of Pancreatic Cancer Precursors Demonstrates Epithelial and Microenvironmental Heterogeneity as an Early Event in Neoplastic Progression</strong></div><div><em>Clin. Cancer Res.</em> 2019; <strong>25</strong>:2194-2205</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=V.BernardA.SemaanJ.HuangF.A.San+LucasF.C.MuluB.M.StephensP.A.GuerreroY.HuangJ.ZhaoN.KamyabiSingle-Cell+Transcriptomics+of+Pancreatic+Cancer+Precursors+Demonstrates+Epithelial+and+Microenvironmental+Heterogeneity+as+an+Early+Event+in+Neoplastic+ProgressionClin.+Cancer+Res.25201921942205" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib11" class="citations"><div id="sref11" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref11" title="View in article">11.</a></div><div>Bi, W.L. ∙ Greenwald, N.F. ∙ Ramkissoon, S.H. ...</div><div><strong>Clinical Identification of Oncogenic Drivers and Copy-Number Alterations in Pituitary Tumors</strong></div><div><em>Endocrinology.</em> 2017; <strong>158</strong>:2284-2291</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=W.L.BiN.F.GreenwaldS.H.RamkissoonM.AbedalthagafiS.M.CoyK.L.LigonY.MeiL.MacConaillM.DucarL.MinClinical+Identification+of+Oncogenic+Drivers+and+Copy-Number+Alterations+in+Pituitary+TumorsEndocrinology158201722842291" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib12" class="citations"><div id="sref12" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref12-1" title="View in article">12.</a></div><div>Boj, S.F. ∙ Hwang, C.I. ∙ Baker, L.A. ...</div><div><strong>Organoid models of human and mouse ductal pancreatic cancer</strong></div><div><em>Cell.</em> 2015; <strong>160</strong>:324-338</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=S.F.BojC.I.HwangL.A.BakerI.I.ChioD.D.EngleV.CorboM.JagerM.Ponz-SarviseH.TiriacM.S.SpectorOrganoid+models+of+human+and+mouse+ductal+pancreatic+cancerCell1602015324338" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib13" class="citations"><div id="sref13" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref13" title="View in article">13.</a></div><div>Brunton, H. ∙ Caligiuri, G. ∙ Cunningham, R. ..., Glasgow Precision Oncology Laboratory, Australian Pancreatic Cancer Genome Initiative</div><div><strong>HNF4A and GATA6 Loss Reveals Therapeutically Actionable Subtypes in Pancreatic Cancer</strong></div><div><em>Cell Rep.</em> 2020; <strong>31</strong>:107625</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=H.BruntonG.CaligiuriR.CunninghamR.Upstill-GoddardU.M.BaileyI.M.GarnerC.NourseS.DreyerM.JonesK.Moran-JonesGlasgow+Precision+Oncology+LaboratoryAustralian+Pancreatic+Cancer+Genome+InitiativeHNF4A+and+GATA6+Loss+Reveals+Therapeutically+Actionable+Subtypes+in+Pancreatic+CancerCell+Rep.312020107625" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib14" class="citations"><div id="sref14" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref14" title="View in article">14.</a></div><div>Byrne, A.T. ∙ Alférez, D.G. ∙ Amant, F. ...</div><div><strong>Interrogating open issues in cancer medicine with patient-derived xenografts</strong></div><div><em>Nat. Rev. Cancer.</em> 2017; <strong>17</strong>:632</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.T.ByrneD.G.Alf%C3%A9rezF.AmantD.AnnibaliJ.ArribasA.V.BiankinA.BrunaE.Budinsk%C3%A1C.CaldasD.K.ChangInterrogating+open+issues+in+cancer+medicine+with+patient-derived+xenograftsNat.+Rev.+Cancer172017632" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib15" class="citations"><div id="sref15" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref15-1" title="View in article">15.</a></div><div>Cancer Genome Atlas Research Network</div><div><strong>Integrated Genomic Characterization of Pancreatic Ductal Adenocarcinoma</strong></div><div><em>Cancer Cell.</em> 2017; <strong>32</strong>:185-203.e13</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=Cancer+Genome+Atlas+Research+NetworkIntegrated+Genomic+Characterization+of+Pancreatic+Ductal+AdenocarcinomaCancer+Cell322017185203.e13" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib16" class="citations"><div id="sref16" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref16-1" title="View in article">16.</a></div><div>Chan-Seng-Yue, M. ∙ Kim, J.C. ∙ Wilson, G.W. ...</div><div><strong>Transcription phenotypes of pancreatic cancer are driven by genomic events during tumor evolution</strong></div><div><em>Nat. Genet.</em> 2020; <strong>52</strong>:231-240</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.Chan-Seng-YueJ.C.KimG.W.WilsonK.NgE.F.FigueroaG.M.O%E2%80%99KaneA.A.ConnorR.E.DenrocheR.C.GrantJ.McLeodTranscription+phenotypes+of+pancreatic+cancer+are+driven+by+genomic+events+during+tumor+evolutionNat.+Genet.522020231240" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib17" class="citations"><div id="sref17" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref17-1" title="View in article">17.</a></div><div>Collisson, E.A. ∙ Sadanandam, A. ∙ Olson, P. ...</div><div><strong>Subtypes of pancreatic ductal adenocarcinoma and their differing responses to therapy</strong></div><div><em>Nat. Med.</em> 2011; <strong>17</strong>:500-503</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=E.A.CollissonA.SadanandamP.OlsonW.J.GibbM.TruittS.GuJ.CoocJ.WeinkleG.E.KimL.JakkulaSubtypes+of+pancreatic+ductal+adenocarcinoma+and+their+differing+responses+to+therapyNat.+Med.172011500503" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib18" class="citations"><div id="sref18" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref18-1" title="View in article">18.</a></div><div>Collisson, E.A. ∙ Bailey, P. ∙ Chang, D.K. ...</div><div><strong>Molecular subtypes of pancreatic cancer</strong></div><div><em>Nat. Rev. Gastroenterol. Hepatol.</em> 2019; <strong>16</strong>:207-220</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=E.A.CollissonP.BaileyD.K.ChangA.V.BiankinMolecular+subtypes+of+pancreatic+cancerNat.+Rev.+Gastroenterol.+Hepatol.162019207220" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib19" class="citations"><div id="sref19" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref19-1" title="View in article">19.</a></div><div>Connor, A.A. ∙ Denroche, R.E. ∙ Jang, G.H. ...</div><div><strong>Integration of Genomic and Transcriptional Features in Pancreatic Cancer Reveals Increased Cell Cycle Progression in Metastases</strong></div><div><em>Cancer Cell.</em> 2019; <strong>35</strong>:267-282 e267</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.A.ConnorR.E.DenrocheG.H.JangM.LemireA.ZhangM.Chan-Seng-YueG.WilsonR.C.GrantD.MericoI.LunguIntegration+of+Genomic+and+Transcriptional+Features+in+Pancreatic+Cancer+Reveals+Increased+Cell+Cycle+Progression+in+MetastasesCancer+Cell352019267282+e267" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib20" class="citations"><div id="sref20" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref20" title="View in article">20.</a></div><div>Di Carlo, S.E. ∙ Peduto, L.</div><div><strong>The perivascular origin of pathological fibroblasts</strong></div><div><em>J.&nbsp;Clin. Invest.</em> 2018; <strong>128</strong>:54-63</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=S.E.Di+CarloL.PedutoThe+perivascular+origin+of+pathological+fibroblastsJ.%C2%A0Clin.+Invest.12820185463" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib21" class="citations"><div id="sref21" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref21" title="View in article">21.</a></div><div>Drost, J. ∙ Clevers, H.</div><div><strong>Organoids in cancer research</strong></div><div><em>Nat. Rev. Cancer.</em> 2018; <strong>18</strong>:407-418</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.DrostH.CleversOrganoids+in+cancer+researchNat.+Rev.+Cancer182018407418" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib22" class="citations"><div id="sref22" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref22-1" title="View in article">22.</a></div><div>Elyada, E. ∙ Bolisetty, M. ∙ Laise, P. ...</div><div><strong>Cross-Species Single-Cell Analysis of Pancreatic Ductal Adenocarcinoma Reveals Antigen-Presenting Cancer-Associated Fibroblasts</strong></div><div><em>Cancer Discov.</em> 2019; <strong>9</strong>:1102-1123</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=E.ElyadaM.BolisettyP.LaiseW.F.FlynnE.T.CourtoisR.A.BurkhartJ.A.TeinorP.BelleauG.BiffiM.S.LucitoCross-Species+Single-Cell+Analysis+of+Pancreatic+Ductal+Adenocarcinoma+Reveals+Antigen-Presenting+Cancer-Associated+FibroblastsCancer+Discov.9201911021123" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib23" class="citations"><div id="sref23" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref23" title="View in article">23.</a></div><div>Fan, J. ∙ Lee, H.O. ∙ Lee, S. ...</div><div><strong>Linking transcriptional and genetic tumor heterogeneity through allele analysis of single-cell RNA-seq data</strong></div><div><em>Genome Res.</em> 2018; <strong>28</strong>:1217-1227</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.FanH.O.LeeS.LeeD.E.RyuS.LeeC.XueS.J.KimK.KimN.BarkasP.J.ParkLinking+transcriptional+and+genetic+tumor+heterogeneity+through+allele+analysis+of+single-cell+RNA-seq+dataGenome+Res.28201812171227" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib24" class="citations"><div id="sref24" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref24" title="View in article">24.</a></div><div>Filbin, M.G. ∙ Tirosh, I. ∙ Hovestadt, V. ...</div><div><strong>Developmental and oncogenic programs in H3K27M gliomas dissected by single-cell RNA-seq</strong></div><div><em>Science.</em> 2018; <strong>360</strong>:331-335</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.G.FilbinI.TiroshV.HovestadtM.L.ShawL.E.EscalanteN.D.MathewsonC.NeftelN.FrankK.PeltonC.M.HebertDevelopmental+and+oncogenic+programs+in+H3K27M+gliomas+dissected+by+single-cell+RNA-seqScience3602018331335" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib25" class="citations"><div id="sref25" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref25" title="View in article">25.</a></div><div>Fujii, M. ∙ Matano, M. ∙ Toshimitsu, K. ...</div><div><strong>Human Intestinal Organoids Maintain Self-Renewal Capacity and Cellular Diversity in Niche-Inspired Culture Condition</strong></div><div><em>Cell Stem Cell.</em> 2018; <strong>23</strong>:787-793 e786</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.FujiiM.MatanoK.ToshimitsuA.TakanoY.MikamiS.NishikoriS.SugimotoT.SatoHuman+Intestinal+Organoids+Maintain+Self-Renewal+Capacity+and+Cellular+Diversity+in+Niche-Inspired+Culture+ConditionCell+Stem+Cell232018787793+e786" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib26" class="citations"><div id="sref26" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref26" title="View in article">26.</a></div><div>Garcia, E.P. ∙ Minkovsky, A. ∙ Jia, Y. ...</div><div><strong>Validation of OncoPanel: A Targeted Next-Generation Sequencing Assay for the Detection of Somatic Variants in Cancer</strong></div><div><em>Arch. Pathol. Lab. Med.</em> 2017; <strong>141</strong>:751-758</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=E.P.GarciaA.MinkovskyY.JiaM.D.DucarP.ShivdasaniX.GongA.H.LigonL.M.ShollF.C.KuoL.E.MacConaillValidation+of+OncoPanel%3A+A+Targeted+Next-Generation+Sequencing+Assay+for+the+Detection+of+Somatic+Variants+in+CancerArch.+Pathol.+Lab.+Med.1412017751758" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib27" class="citations"><div id="sref27" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref27-1" title="View in article">27.</a></div><div>Garraway, L.A. ∙ Lander, E.S.</div><div><strong>Lessons from the cancer genome</strong></div><div><em>Cell.</em> 2013; <strong>153</strong>:17-37</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=L.A.GarrawayE.S.LanderLessons+from+the+cancer+genomeCell15320131737" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib28" class="citations"><div id="sref28" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref28-1" title="View in article">28.</a></div><div>Ghandi, M. ∙ Huang, F.W. ∙ Jané-Valbuena, J. ...</div><div><strong>Next-generation characterization of the Cancer Cell Line Encyclopedia</strong></div><div><em>Nature.</em> 2019; <strong>569</strong>:503-508</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.GhandiF.W.HuangJ.Jan%C3%A9-ValbuenaG.V.KryukovC.C.LoE.R.McDonald3rdJ.BarretinaE.T.GelfandC.M.BielskiH.LiNext-generation+characterization+of+the+Cancer+Cell+Line+EncyclopediaNature5692019503508" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib29" class="citations"><div id="sref29" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref29-1" title="View in article">29.</a></div><div>Gierahn, T.M. ∙ Wadsworth, 2nd, M.H. ∙ Hughes, T.K. ...</div><div><strong>Seq-Well: portable, low-cost RNA sequencing of single cells at high throughput</strong></div><div><em>Nat. Methods.</em> 2017; <strong>14</strong>:395-398</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=T.M.GierahnM.H.Wadsworth2ndT.K.HughesB.D.BrysonA.ButlerR.SatijaS.FortuneJ.C.LoveA.K.ShalekSeq-Well%3A+portable%2C+low-cost+RNA+sequencing+of+single+cells+at+high+throughputNat.+Methods142017395398" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib30" class="citations"><div id="sref30" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref30" title="View in article">30.</a></div><div>Gillet, J.P. ∙ Varma, S. ∙ Gottesman, M.M.</div><div><strong>The clinical relevance of cancer cell lines</strong></div><div><em>J.&nbsp;Natl. Cancer Inst.</em> 2013; <strong>105</strong>:452-458</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.P.GilletS.VarmaM.M.GottesmanThe+clinical+relevance+of+cancer+cell+linesJ.%C2%A0Natl.+Cancer+Inst.1052013452458" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib31" class="citations"><div id="sref31" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref31-1" title="View in article">31.</a></div><div>Gröger, C.J. ∙ Grubinger, M. ∙ Waldhör, T. ...</div><div><strong>Meta-analysis of gene expression signatures defining the epithelial to mesenchymal transition during cancer progression</strong></div><div><em>PLoS ONE.</em> 2012; <strong>7</strong>:e51136</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=C.J.Gr%C3%B6gerM.GrubingerT.Waldh%C3%B6rK.VierlingerW.MikulitsMeta-analysis+of+gene+expression+signatures+defining+the+epithelial+to+mesenchymal+transition+during+cancer+progressionPLoS+ONE72012e51136" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib32" class="citations"><div id="sref32" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref32-1" title="View in article">32.</a></div><div>Grünwald, B.T. ∙ Devisme, A. ∙ Andrieux, G. ...</div><div><strong>Spatially confined sub-tumor microenvironments in pancreatic cancer</strong></div><div><em>Cell.</em> 2021; <strong>184</strong>:5577-5592.e18</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=B.T.Gr%C3%BCnwaldA.DevismeG.AndrieuxF.VyasK.AliarC.W.McCloskeyA.MacklinG.H.JangR.DenrocheJ.M.RomeroSpatially+confined+sub-tumor+microenvironments+in+pancreatic+cancerCell184202155775592.e18" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib33" class="citations"><div id="sref33" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref33" title="View in article">33.</a></div><div>Hafner, M. ∙ Niepel, M. ∙ Chung, M. ...</div><div><strong>Growth rate inhibition metrics correct for confounders in measuring sensitivity to cancer drugs</strong></div><div><em>Nat. Methods.</em> 2016; <strong>13</strong>:521-527</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.HafnerM.NiepelM.ChungP.K.SorgerGrowth+rate+inhibition+metrics+correct+for+confounders+in+measuring+sensitivity+to+cancer+drugsNat.+Methods132016521527" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib34" class="citations"><div id="sref34" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref34-1" title="View in article">34.</a></div><div>Hahn, W.C. ∙ Bader, J.S. ∙ Braun, T.P. ..., Cancer Target Discovery and Development Network</div><div><strong>An expanded universe of cancer targets</strong></div><div><em>Cell.</em> 2021; <strong>184</strong>:1142-1155</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=W.C.HahnJ.S.BaderT.P.BraunA.CalifanoP.A.ClemonsB.J.DrukerA.J.EwaldH.FuS.JaguC.J.KempCancer+Target+Discovery+and+Development+NetworkAn+expanded+universe+of+cancer+targetsCell184202111421155" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib35" class="citations"><div id="sref35" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref35" title="View in article">35.</a></div><div>Hara, T. ∙ Chanoch-Myers, R. ∙ Mathewson, N.D. ...</div><div><strong>Interactions between cancer cells and immune cells drive transitions to mesenchymal-like states in glioblastoma</strong></div><div><em>Cancer Cell.</em> 2021; <strong>39</strong>:779-792 e711</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=T.HaraR.Chanoch-MyersN.D.MathewsonC.MyskiwL.AttaL.BussemaS.W.EichhornA.C.GreenwaldG.S.KinkerC.RodmanInteractions+between+cancer+cells+and+immune+cells+drive+transitions+to+mesenchymal-like+states+in+glioblastomaCancer+Cell392021779792+e711" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib36" class="citations"><div id="sref36" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref36" title="View in article">36.</a></div><div>Hayashi, A. ∙ Fan, J. ∙ Chen, R. ...</div><div><strong>A unifying paradigm for transcriptional heterogeneity and squamous features in pancreatic ductal adenocarcinoma</strong></div><div><em>Nature Cancer.</em> 2020; <strong>1</strong>:59-74</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.HayashiJ.FanR.ChenY.-j.HoA.P.Makohon-MooreN.LecomteY.ZhongJ.HongJ.HuangH.SakamotoA+unifying+paradigm+for+transcriptional+heterogeneity+and+squamous+features+in+pancreatic+ductal+adenocarcinomaNature+Cancer120205974" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib37" class="citations"><div id="sref37" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref37" title="View in article">37.</a></div><div>Ho, W.J. ∙ Jaffee, E.M. ∙ Zheng, L.</div><div><strong>The tumour microenvironment in pancreatic cancer - clinical challenges and opportunities</strong></div><div><em>Nat. Rev. Clin. Oncol.</em> 2020; <strong>17</strong>:527-540</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=W.J.HoE.M.JaffeeL.ZhengThe+tumour+microenvironment+in+pancreatic+cancer+-+clinical+challenges+and+opportunitiesNat.+Rev.+Clin.+Oncol.172020527540" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib38" class="citations"><div id="sref38" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref38" title="View in article">38.</a></div><div>Hosaka, K. ∙ Yang, Y. ∙ Seki, T. ...</div><div><strong>Pericyte-fibroblast transition promotes tumor growth and metastasis</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2016; <strong>113</strong>:E5618-E5627</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=K.HosakaY.YangT.SekiC.FischerO.DubeyE.FredlundJ.HartmanP.ReligaH.MorikawaY.IshiiPericyte-fibroblast+transition+promotes+tumor+growth+and+metastasisProc.+Natl.+Acad.+Sci.+USA1132016E5618E5627" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib39" class="citations"><div id="sref39" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref39" title="View in article">39.</a></div><div>Hovestadt, V. ∙ Smith, K.S. ∙ Bihannic, L. ...</div><div><strong>Resolving medulloblastoma cellular architecture by single-cell genomics</strong></div><div><em>Nature.</em> 2019; <strong>572</strong>:74-79</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=V.HovestadtK.S.SmithL.BihannicM.G.FilbinM.L.ShawA.BaumgartnerJ.C.DeWittA.GrovesL.MayrH.R.WeismanResolving+medulloblastoma+cellular+architecture+by+single-cell+genomicsNature57220197479" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib40" class="citations"><div id="sref40" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref40-1" title="View in article">40.</a></div><div>Hughes, T.K. ∙ Wadsworth, 2nd, M.H. ∙ Gierahn, T.M. ...</div><div><strong>Second-Strand Synthesis-Based Massively Parallel scRNA-Seq Reveals Cellular States and Molecular Features of Human Inflammatory Skin Pathologies</strong></div><div><em>Immunity.</em> 2020; <strong>53</strong>:878-894 e877</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=T.K.HughesM.H.Wadsworth2ndT.M.GierahnT.DoD.WeissP.R.AndradeF.MaB.J.de+Andrade+SilvaS.ShaoL.C.TsoiSecond-Strand+Synthesis-Based+Massively+Parallel+scRNA-Seq+Reveals+Cellular+States+and+Molecular+Features+of+Human+Inflammatory+Skin+PathologiesImmunity532020878894+e877" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib41" class="citations"><div id="sref41" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref41-1" title="View in article">41.</a></div><div>Hyman, D.M. ∙ Taylor, B.S. ∙ Baselga, J.</div><div><strong>Implementing Genome-Driven Oncology</strong></div><div><em>Cell.</em> 2017; <strong>168</strong>:584-599</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=D.M.HymanB.S.TaylorJ.BaselgaImplementing+Genome-Driven+OncologyCell1682017584599" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib42" class="citations"><div id="sref42" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref42" title="View in article">42.</a></div><div>Kim, J.H. ∙ Park, S.Y. ∙ Jun, Y. ...</div><div><strong>Roles of Wnt Target Genes in the Journey of Cancer Stem Cells</strong></div><div><em>Int. J. Mol. Sci.</em> 2017; <strong>18</strong>:1604</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.H.KimS.Y.ParkY.JunJ.Y.KimJ.S.NamRoles+of+Wnt+Target+Genes+in+the+Journey+of+Cancer+Stem+CellsInt.+J.+Mol.+Sci.1820171604" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib43" class="citations"><div id="sref43" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref43-1" title="View in article">43.</a></div><div>Kim, C. ∙ Gao, R. ∙ Sei, E. ...</div><div><strong>Chemoresistance Evolution in Triple-Negative Breast Cancer Delineated by Single-Cell Sequencing</strong></div><div><em>Cell.</em> 2018; <strong>173</strong>:879-893 e813</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=C.KimR.GaoE.SeiR.BrandtJ.HartmanT.HatschekN.CrosettoT.FoukakisN.E.NavinChemoresistance+Evolution+in+Triple-Negative+Breast+Cancer+Delineated+by+Single-Cell+SequencingCell1732018879893+e813" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib44" class="citations"><div id="sref44" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref44-1" title="View in article">44.</a></div><div>Li, B. ∙ Gould, J. ∙ Yang, Y. ...</div><div><strong>Cumulus provides cloud-based data analysis for large-scale single-cell and single-nucleus RNA-seq</strong></div><div><em>Nat. Methods.</em> 2020; <strong>17</strong>:793-798</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=B.LiJ.GouldY.YangS.SarkizovaM.TabakaO.AshenbergY.RosenM.SlyperM.S.KowalczykA.C.VillaniCumulus+provides+cloud-based+data+analysis+for+large-scale+single-cell+and+single-nucleus+RNA-seqNat.+Methods172020793798" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib45" class="citations"><div id="sref45" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref45" title="View in article">45.</a></div><div>Li, Y. ∙ He, Y. ∙ Peng, J. ...</div><div><strong>Mutant Kras co-opts a proto-oncogenic enhancer network in inflammation-induced metaplastic progenitor cells to initiate pancreatic cancer</strong></div><div><em>Nature Cancer.</em> 2021; <strong>2</strong>:49-65</div></div><div class="external-links"><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/35121887/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?pmid=35121887" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib46" class="citations"><div id="sref46" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref46" title="View in article">46.</a></div><div>Ligorio, M. ∙ Sil, S. ∙ Malagon-Lopez, J. ...</div><div><strong>Stromal Microenvironment Shapes the Intratumoral Architecture of Pancreatic Cancer.</strong></div><div><em>Cell.</em> 2019; <strong>178</strong>:160-175 e127</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.LigorioS.SilJ.Malagon-LopezL.T.NiemanS.MisaleM.Di+PilatoR.Y.EbrightM.N.KarabacakA.S.KulkarniA.LiuStromal+Microenvironment+Shapes+the+Intratumoral+Architecture+of+Pancreatic+Cancer.Cell1782019160175+e127" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib47" class="citations"><div id="sref47" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref47" title="View in article">47.</a></div><div>Longo, S.K. ∙ Guo, M.G. ∙ Ji, A.L. ...</div><div><strong>Integrating single-cell and spatial transcriptomics to elucidate intercellular tissue dynamics</strong></div><div><em>Nat. Rev. Genet.</em> 2021; <strong>22</strong>:627-644</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=S.K.LongoM.G.GuoA.L.JiP.A.KhavariIntegrating+single-cell+and+spatial+transcriptomics+to+elucidate+intercellular+tissue+dynamicsNat.+Rev.+Genet.222021627644" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib48" class="citations"><div id="sref48" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref48" title="View in article">48.</a></div><div>Macosko, E.Z. ∙ Basu, A. ∙ Satija, R. ...</div><div><strong>Highly Parallel Genome-wide Expression Profiling of Individual Cells Using Nanoliter Droplets</strong></div><div><em>Cell.</em> 2015; <strong>161</strong>:1202-1214</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=E.Z.MacoskoA.BasuR.SatijaJ.NemeshK.ShekharM.GoldmanI.TiroshA.R.BialasN.KamitakiE.M.MartersteckHighly+Parallel+Genome-wide+Expression+Profiling+of+Individual+Cells+Using+Nanoliter+DropletsCell161201512021214" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib49" class="citations"><div id="sref49" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref49-1" title="View in article">49.</a></div><div>Miyabayashi, K. ∙ Baker, L.A. ∙ Deschênes, A. ...</div><div><strong>Intraductal Transplantation Models of Human Pancreatic Ductal Adenocarcinoma Reveal Progressive Transition of Molecular Subtypes</strong></div><div><em>Cancer Discov.</em> 2020; <strong>10</strong>:1566-1589</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=K.MiyabayashiL.A.BakerA.Desch%C3%AAnesB.TraubG.CaligiuriD.PlenkerB.AlagesanP.BelleauS.LiJ.KendallIntraductal+Transplantation+Models+of+Human+Pancreatic+Ductal+Adenocarcinoma+Reveal+Progressive+Transition+of+Molecular+SubtypesCancer+Discov.10202015661589" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib50" class="citations"><div id="sref50" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref50-1" title="View in article">50.</a></div><div>Moffitt, R.A. ∙ Marayati, R. ∙ Flate, E.L. ...</div><div><strong>Virtual microdissection identifies distinct tumor- and stroma-specific subtypes of pancreatic ductal adenocarcinoma</strong></div><div><em>Nat. Genet.</em> 2015; <strong>47</strong>:1168-1178</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=R.A.MoffittR.MarayatiE.L.FlateK.E.VolmarS.G.LoezaK.A.HoadleyN.U.RashidL.A.WilliamsS.C.EatonA.H.ChungVirtual+microdissection+identifies+distinct+tumor-+and+stroma-specific+subtypes+of+pancreatic+ductal+adenocarcinomaNat.+Genet.47201511681178" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib51" class="citations"><div id="sref51" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref51" title="View in article">51.</a></div><div>Nam, A.S. ∙ Chaligne, R. ∙ Landau, D.A.</div><div><strong>Integrating genetic and non-genetic determinants of cancer evolution by single-cell multi-omics</strong></div><div><em>Nat. Rev. Genet.</em> 2021; <strong>22</strong>:3-18</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.S.NamR.ChaligneD.A.LandauIntegrating+genetic+and+non-genetic+determinants+of+cancer+evolution+by+single-cell+multi-omicsNat.+Rev.+Genet.222021318" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib52" class="citations"><div id="sref52" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref52" title="View in article">52.</a></div><div>Neftel, C. ∙ Laffy, J. ∙ Filbin, M.G. ...</div><div><strong>An Integrative Model of Cellular States, Plasticity, and Genetics for Glioblastoma</strong></div><div><em>Cell.</em> 2019; <strong>178</strong>:835-849 e821</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=C.NeftelJ.LaffyM.G.FilbinT.HaraM.E.ShoreG.J.RahmeA.R.RichmanD.SilverbushM.L.ShawC.M.HebertAn+Integrative+Model+of+Cellular+States%2C+Plasticity%2C+and+Genetics+for+GlioblastomaCell1782019835849+e821" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib53" class="citations"><div id="sref53" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref53-1" title="View in article">53.</a></div><div>O’Kane, G.M. ∙ Grunwald, B.T. ∙ Jang, G.H. ...</div><div><strong>GATA6 Expression Distinguishes Classical and Basal-like Subtypes in Advanced Pancreatic Cancer</strong></div><div><em>Clin Cancer Res.</em> 2020; <strong>26</strong>:4901-4910</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=G.M.O%E2%80%99KaneB.T.GrunwaldG.H.JangM.MasoomianS.PicardoR.C.GrantR.E.DenrocheA.ZhangY.WangB.LamGATA6+Expression+Distinguishes+Classical+and+Basal-like+Subtypes+in+Advanced+Pancreatic+CancerClin+Cancer+Res.26202049014910" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib54" class="citations"><div id="sref54" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref54" title="View in article">54.</a></div><div>Öhlund, D. ∙ Handly-Santana, A. ∙ Biffi, G. ...</div><div><strong>Distinct populations of inflammatory fibroblasts and myofibroblasts in pancreatic cancer</strong></div><div><em>J.&nbsp;Exp. Med.</em> 2017; <strong>214</strong>:579-596</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=D.%C3%96hlundA.Handly-SantanaG.BiffiE.ElyadaA.S.AlmeidaM.Ponz-SarviseV.CorboT.E.OniS.A.HearnE.J.LeeDistinct+populations+of+inflammatory+fibroblasts+and+myofibroblasts+in+pancreatic+cancerJ.%C2%A0Exp.+Med.2142017579596" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib55" class="citations"><div id="sref55" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref55" title="View in article">55.</a></div><div>Palmer, A.C. ∙ Sorger, P.K.</div><div><strong>Combination Cancer Therapy Can Confer Benefit via Patient-to-Patient Variability without Drug Additivity or Synergy</strong></div><div><em>Cell.</em> 2017; <strong>171</strong>:1678-1691 e1613</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.C.PalmerP.K.SorgerCombination+Cancer+Therapy+Can+Confer+Benefit+via+Patient-to-Patient+Variability+without+Drug+Additivity+or+SynergyCell171201716781691+e1613" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib56" class="citations"><div id="sref56" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref56" title="View in article">56.</a></div><div>Palmer, A.C. ∙ Chidley, C. ∙ Sorger, P.K.</div><div><strong>A curative combination cancer therapy achieves high fractional cell killing through low cross-resistance and drug additivity</strong></div><div><em>eLife.</em> 2019; <strong>8</strong>:e50036</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.C.PalmerC.ChidleyP.K.SorgerA+curative+combination+cancer+therapy+achieves+high+fractional+cell+killing+through+low+cross-resistance+and+drug+additivityeLife82019e50036" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib57" class="citations"><div id="sref57" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref57-1" title="View in article">57.</a></div><div>Patel, A.P. ∙ Tirosh, I. ∙ Trombetta, J.J. ...</div><div><strong>Single-cell RNA-seq highlights intratumoral heterogeneity in primary glioblastoma</strong></div><div><em>Science.</em> 2014; <strong>344</strong>:1396-1401</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.P.PatelI.TiroshJ.J.TrombettaA.K.ShalekS.M.GillespieH.WakimotoD.P.CahillB.V.NahedW.T.CurryR.L.MartuzaSingle-cell+RNA-seq+highlights+intratumoral+heterogeneity+in+primary+glioblastomaScience344201413961401" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib58" class="citations"><div id="sref58" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref58" title="View in article">58.</a></div><div>Pelon, F. ∙ Bourachot, B. ∙ Kieffer, Y. ...</div><div><strong>Cancer-associated fibroblast heterogeneity in axillary lymph nodes drives metastases in breast cancer through complementary mechanisms</strong></div><div><em>Nat. Commun.</em> 2020; <strong>11</strong>:404</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=F.PelonB.BourachotY.KiefferI.MagagnaF.Mermet-MeillonI.BonnetA.CostaA.M.GivelY.AttiehJ.BarbazanCancer-associated+fibroblast+heterogeneity+in+axillary+lymph+nodes+drives+metastases+in+breast+cancer+through+complementary+mechanismsNat.+Commun.112020404" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib59" class="citations"><div id="sref59" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref59" title="View in article">59.</a></div><div>Porter, R.L. ∙ Magnus, N.K.C. ∙ Thapar, V. ...</div><div><strong>Epithelial to mesenchymal plasticity and differential response to therapies in pancreatic ductal adenocarcinoma</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2019; <strong>116</strong>:26835-26845</div></div><div class="external-links"><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31843922/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?pmid=31843922" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib60" class="citations"><div id="sref60" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref60-1" title="View in article">60.</a></div><div>Puram, S.V. ∙ Tirosh, I. ∙ Parikh, A.S. ...</div><div><strong>Single-Cell Transcriptomic Analysis of Primary and Metastatic Tumor Ecosystems in Head and Neck Cancer</strong></div><div><em>Cell.</em> 2017; <strong>171</strong>:1611-1624 e1624</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=S.V.PuramI.TiroshA.S.ParikhA.P.PatelK.YizhakS.GillespieC.RodmanC.L.LuoE.A.MrozK.S.EmerickSingle-Cell+Transcriptomic+Analysis+of+Primary+and+Metastatic+Tumor+Ecosystems+in+Head+and+Neck+CancerCell171201716111624+e1624" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib61" class="citations"><div id="sref61" class="citation"><div class="citation-content"><div class="label"><a href="#figs3" title="View in article">61.</a></div><div>Qadir, M.M.F. ∙ Álvarez-Cubela, S. ∙ Klein, D. ...</div><div><strong>Single-cell resolution analysis of the human pancreatic ductal progenitor cell niche</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2020; <strong>117</strong>:10876-10887</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.M.F.QadirS.%C3%81lvarez-CubelaD.KleinJ.van+DijkR.Mu%C3%B1iz-AnquelaY.B.Moreno-Hern%C3%A1ndezG.LanzoniS.SadiqB.Navarro-RubioM.T.Garc%C3%ADaSingle-cell+resolution+analysis+of+the+human+pancreatic+ductal+progenitor+cell+nicheProc.+Natl.+Acad.+Sci.+USA11720201087610887" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib62" class="citations"><div id="sref62" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref62-1" title="View in article">62.</a></div><div>Sade-Feldman, M. ∙ Yizhak, K. ∙ Bjorgaard, S.L. ...</div><div><strong>Defining T Cell States Associated with Response to Checkpoint Immunotherapy in Melanoma</strong></div><div><em>Cell.</em> 2019; <strong>176</strong>:404</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.Sade-FeldmanK.YizhakS.L.BjorgaardJ.P.RayC.G.de+BoerR.W.JenkinsD.J.LiebJ.H.ChenD.T.FrederickM.Barzily-RokniDefining+T+Cell+States+Associated+with+Response+to+Checkpoint+Immunotherapy+in+MelanomaCell1762019404" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib63" class="citations"><div id="sref63" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref63" title="View in article">63.</a></div><div>Sahai, E. ∙ Astsaturov, I. ∙ Cukierman, E. ...</div><div><strong>A framework for advancing our understanding of cancer-associated fibroblasts</strong></div><div><em>Nat. Rev. Cancer.</em> 2020; <strong>20</strong>:174-186</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=E.SahaiI.AstsaturovE.CukiermanD.G.DeNardoM.EgebladR.M.EvansD.FearonF.R.GretenS.R.HingoraniT.HunterA+framework+for+advancing+our+understanding+of+cancer-associated+fibroblastsNat.+Rev.+Cancer202020174186" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib64" class="citations"><div id="sref64" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref64" title="View in article">64.</a></div><div>Sholl, L.M. ∙ Do, K. ∙ Shivdasani, P. ...</div><div><strong>Institutional implementation of clinical tumor profiling on an unselected cancer population</strong></div><div><em>JCI Insight.</em> 2016; <strong>1</strong>:e87062</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=L.M.ShollK.DoP.ShivdasaniE.CeramiA.M.DubucF.C.KuoE.P.GarciaY.JiaP.DavineniR.P.AboInstitutional+implementation+of+clinical+tumor+profiling+on+an+unselected+cancer+populationJCI+Insight12016e87062" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib65" class="citations"><div id="sref65" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref65-1" title="View in article">65.</a></div><div>Somerville, T.D.D. ∙ Xu, Y. ∙ Miyabayashi, K. ...</div><div><strong>TP63-Mediated Enhancer Reprogramming Drives the Squamous Subtype of Pancreatic Ductal Adenocarcinoma</strong></div><div><em>Cell Rep.</em> 2018; <strong>25</strong>:1741-1755 e1747</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=T.D.D.SomervilleY.XuK.MiyabayashiH.TiriacC.R.ClearyD.Maia-SilvaJ.P.MilazzoD.A.TuvesonC.R.VakocTP63-Mediated+Enhancer+Reprogramming+Drives+the+Squamous+Subtype+of+Pancreatic+Ductal+AdenocarcinomaCell+Rep.25201817411755+e1747" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib66" class="citations"><div id="sref66" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref66" title="View in article">66.</a></div><div>Spurr, L.F. ∙ Touat, M. ∙ Taylor, A.M. ...</div><div><strong>Quantification of aneuploidy in targeted sequencing data using ASCETS</strong></div><div><em>Bioinformatics.</em> 2021; <strong>37</strong>:2461-2463</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=L.F.SpurrM.TouatA.M.TaylorA.M.DubucJ.ShihD.M.MeredithW.V.PisanoM.L.MeyersonK.L.LigonA.D.CherniackQuantification+of+aneuploidy+in+targeted+sequencing+data+using+ASCETSBioinformatics37202124612463" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib67" class="citations"><div id="sref67" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref67" title="View in article">67.</a></div><div>Suvà, M.L. ∙ Tirosh, I.</div><div><strong>Single-Cell RNA Sequencing in Cancer: Lessons Learned and Emerging Challenges</strong></div><div><em>Mol. Cell.</em> 2019; <strong>75</strong>:7-12</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.L.Suv%C3%A0I.TiroshSingle-Cell+RNA+Sequencing+in+Cancer%3A+Lessons+Learned+and+Emerging+ChallengesMol.+Cell752019712" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib68" class="citations"><div id="sref68" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref68-1" title="View in article">68.</a></div><div>Tiriac, H. ∙ Belleau, P. ∙ Engle, D.D. ...</div><div><strong>Organoid Profiling Identifies Common Responders to Chemotherapy in Pancreatic Cancer</strong></div><div><em>Cancer Discov.</em> 2018; <strong>8</strong>:1112-1129</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=H.TiriacP.BelleauD.D.EngleD.PlenkerA.Desch%C3%AAnesT.D.D.SomervilleF.E.M.FroelingR.A.BurkhartR.E.DenrocheG.H.JangOrganoid+Profiling+Identifies+Common+Responders+to+Chemotherapy+in+Pancreatic+CancerCancer+Discov.8201811121129" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib69" class="citations"><div id="sref69" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref69-1" title="View in article">69.</a></div><div>Tirosh, I. ∙ Izar, B. ∙ Prakadan, S.M. ...</div><div><strong>Dissecting the multicellular ecosystem of metastatic melanoma by single-cell RNA-seq</strong></div><div><em>Science.</em> 2016; <strong>352</strong>:189-196</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=I.TiroshB.IzarS.M.PrakadanM.H.Wadsworth2ndD.TreacyJ.J.TrombettaA.RotemC.RodmanC.LianG.MurphyDissecting+the+multicellular+ecosystem+of+metastatic+melanoma+by+single-cell+RNA-seqScience3522016189196" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib70" class="citations"><div id="sref70" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref70-1" title="View in article">70.</a></div><div>Tirosh, I. ∙ Venteicher, A.S. ∙ Hebert, C. ...</div><div><strong>Single-cell RNA-seq supports a developmental hierarchy in human oligodendroglioma</strong></div><div><em>Nature.</em> 2016; <strong>539</strong>:309-313</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=I.TiroshA.S.VenteicherC.HebertL.E.EscalanteA.P.PatelK.YizhakJ.M.FisherC.RodmanC.MountM.G.FilbinSingle-cell+RNA-seq+supports+a+developmental+hierarchy+in+human+oligodendrogliomaNature5392016309313" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib71" class="citations"><div id="sref71" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref71" title="View in article">71.</a></div><div>van de Haar, J. ∙ Hoes, L.R. ∙ Roepman, P. ...</div><div><strong>Limited evolution of the actionable metastatic cancer genome under therapeutic pressure</strong></div><div><em>Nat. Med.</em> 2021; <strong>27</strong>:1553-1563</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.van+de+HaarL.R.HoesP.RoepmanM.P.LolkemaH.M.W.VerheulH.GelderblomA.J.de+LangenE.F.SmitE.CuppenL.F.A.WesselsE.E.VoestLimited+evolution+of+the+actionable+metastatic+cancer+genome+under+therapeutic+pressureNat.+Med.27202115531563" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib72" class="citations"><div id="sref72" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref72-1" title="View in article">72.</a></div><div>van Galen, P. ∙ Hovestadt, V. ∙ Wadsworth Ii, M.H. ...</div><div><strong>Single-Cell RNA-Seq Reveals AML Hierarchies Relevant to Disease Progression and Immunity</strong></div><div><em>Cell.</em> 2019; <strong>176</strong>:1265-1281 e1224</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=P.van+GalenV.HovestadtM.H.Wadsworth+IiT.K.HughesG.K.GriffinS.BattagliaJ.A.VergaJ.StephanskyT.J.PastikaJ.Lombardi+StorySingle-Cell+RNA-Seq+Reveals+AML+Hierarchies+Relevant+to+Disease+Progression+and+ImmunityCell176201912651281+e1224" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib73" class="citations"><div id="sref73" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref73" title="View in article">73.</a></div><div>Venteicher, A.S. ∙ Tirosh, I. ∙ Hebert, C. ...</div><div><strong>Decoupling genetics, lineages, and microenvironment in IDH-mutant gliomas by single-cell RNA-seq</strong></div><div><em>Science.</em> 2017; <strong>355</strong>:eaai8478</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.S.VenteicherI.TiroshC.HebertK.YizhakC.NeftelM.G.FilbinV.HovestadtL.E.EscalanteM.L.ShawC.RodmanDecoupling+genetics%2C+lineages%2C+and+microenvironment+in+IDH-mutant+gliomas+by+single-cell+RNA-seqScience3552017eaai8478" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib74" class="citations"><div id="sref74" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref74-1" title="View in article">74.</a></div><div>Vivian, J. ∙ Rao, A.A. ∙ Nothaft, F.A. ...</div><div><strong>Toil enables reproducible, open source, big biomedical data analyses</strong></div><div><em>Nat. Biotechnol.</em> 2017; <strong>35</strong>:314-316</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.VivianA.A.RaoF.A.NothaftC.KetchumJ.ArmstrongA.NovakJ.PfeilJ.NarkizianA.D.DeranA.Musselman-BrownToil+enables+reproducible%2C+open+source%2C+big+biomedical+data+analysesNat.+Biotechnol.352017314316" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib75" class="citations"><div id="sref75" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref75" title="View in article">75.</a></div><div>Wagner, D.E. ∙ Klein, A.M.</div><div><strong>Lineage tracing meets single-cell omics: opportunities and challenges</strong></div><div><em>Nat. Rev. Genet.</em> 2020; <strong>21</strong>:410-427</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=D.E.WagnerA.M.KleinLineage+tracing+meets+single-cell+omics%3A+opportunities+and+challengesNat.+Rev.+Genet.212020410427" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib76" class="citations"><div id="sref76" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref76-1" title="View in article">76.</a></div><div>Weinstein, J.N. ∙ Collisson, E.A. ∙ Mills, G.B. ..., Cancer Genome Atlas Research Network</div><div><strong>The Cancer Genome Atlas Pan-Cancer analysis project</strong></div><div><em>Nat. Genet.</em> 2013; <strong>45</strong>:1113-1120</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.N.WeinsteinE.A.CollissonG.B.MillsK.R.ShawB.A.OzenbergerK.EllrottI.ShmulevichC.SanderJ.M.StuartCancer+Genome+Atlas+Research+NetworkThe+Cancer+Genome+Atlas+Pan-Cancer+analysis+projectNat.+Genet.45201311131120" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib77" class="citations"><div id="sref77" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref77" title="View in article">77.</a></div><div>Wöll, S. ∙ Schlitter, A.M. ∙ Dhaene, K. ...</div><div><strong>Claudin 18.2 is a target for IMAB362 antibody in pancreatic neoplasms</strong></div><div><em>Int. J. Cancer.</em> 2014; <strong>134</strong>:731-739</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=S.W%C3%B6llA.M.SchlitterK.DhaeneM.RollerI.EspositoU.Sahin%C3%96.T%C3%BCreciClaudin+18.2+is+a+target+for+IMAB362+antibody+in+pancreatic+neoplasmsInt.+J.+Cancer1342014731739" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib78" class="citations"><div id="sref78" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref78-1" title="View in article">78.</a></div><div>Zhang, L. ∙ Li, Z. ∙ Skrzypczynska, K.M. ...</div><div><strong>Single-Cell Analyses Inform Mechanisms of Myeloid-Targeted Therapies in Colon Cancer</strong></div><div><em>Cell.</em> 2020; <strong>181</strong>:442-459 e429</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=L.ZhangZ.LiK.M.SkrzypczynskaQ.FangW.ZhangS.A.O%E2%80%99BrienY.HeL.WangQ.ZhangA.KimSingle-Cell+Analyses+Inform+Mechanisms+of+Myeloid-Targeted+Therapies+in+Colon+CancerCell1812020442459+e429" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib79" class="citations"><div id="sref79" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref79-1" title="View in article">79.</a></div><div>Zilionis, R. ∙ Engblom, C. ∙ Pfirschke, C. ...</div><div><strong>Single-Cell Transcriptomics of Human and Mouse Lung Cancers Reveals Conserved Myeloid Populations across Individuals and Species</strong></div><div><em>Immunity.</em> 2019; <strong>50</strong>:1317-1334 e1310</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=R.ZilionisC.EngblomC.PfirschkeV.SavovaD.ZemmourH.D.SaatciogluI.KrishnanG.MaroniC.V.MeyerovitzC.M.KerwinSingle-Cell+Transcriptomics+of+Human+and+Mouse+Lung+Cancers+Reveals+Conserved+Myeloid+Populations+across+Individuals+and+SpeciesImmunity50201913171334+e1310" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div></div></div></section></div></section><div class="core-linked-content">
    

        


    

            

            

            <div data-widget-id="8633d695-b258-4908-b86f-e2c268ec3c6b" data-widget-name="UX3HTMLWidget"><script type="text/javascript">
if($('.content-navigation__body__title').length){
$('.content-navigation__body__title').each(function(){
var html=this.innerHTML;
var div = document.createElement("div");
div.innerHTML = html;
var text = div.textContent || div.innerText || "";
this.innerHTML=text;
});
}
</script></div>

            

        

    

        


    

            

            

            <div data-widget-id="393effff-5b44-4681-a9b4-4f029b3540b7" data-widget-name="UX3HTMLWidget"><style>
.linkback-ad .pb-ad {
    display: block;
   height: auto;
    margin-left: 0px;
    margin-right: auto;
}
</style></div>

            

        

    

        

</div></div></div><div class="core-collateral-aside"><section id="core-collateral-figures"><h2>Figures (14)<button class="js--openFigureViewer"><span>Figure Viewer</span></button></h2><div class="core-collateral-aside-wrapper"><div class="grid"><a class="figure__open__ctrl" href="#fig1" aria-label="Go to Figure&amp;nbsp;1"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.11.017/asset/92bcff86-ebe0-4bad-8a05-84b74de20e9f/main.assets/gr1.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs1" aria-label="Go to Figure&amp;nbsp;S1"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.11.017/asset/daaa3cd2-7403-4552-903f-3868a004318c/main.assets/figs1.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig2" aria-label="Go to Figure&amp;nbsp;2"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.11.017/asset/9f44ae90-e7b3-4fe9-a3c5-2a09a5f21fcc/main.assets/gr2.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs2" aria-label="Go to Figure&amp;nbsp;S2"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.11.017/asset/c5bc289b-af8c-4c9f-8ddc-63d45cd4a98c/main.assets/figs2.jpg" aria-hidden="true"></a></div><div id="collapsible-figures" data-expandable="item"><div class="content grid" style="display:none;" role="region" id="collapsible-figures_content"><a class="figure__open__ctrl" href="#figs3" aria-label="Go to Figure&amp;nbsp;S3"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.11.017/asset/21b9c88f-5212-44bd-afc1-85eabf7e0270/main.assets/figs3.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs4" aria-label="Go to Figure&amp;nbsp;S4"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.11.017/asset/d35ebd64-74e1-423a-a7a2-7058ea8e9b8d/main.assets/figs4.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig3" aria-label="Go to Figure&amp;nbsp;3"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.11.017/asset/19bb9976-a1d2-402e-9a39-6796596f35b0/main.assets/gr3.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs5" aria-label="Go to Figure&amp;nbsp;S5"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.11.017/asset/10e8c5f7-7b3d-4df7-8a1b-cddd166c3699/main.assets/figs5.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig4" aria-label="Go to Figure&amp;nbsp;4"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.11.017/asset/9e730ef2-f7ed-4457-8b56-166d5c077dc7/main.assets/gr4.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig5" aria-label="Go to Figure&amp;nbsp;5"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.11.017/asset/6324ba1e-d8e9-4503-981c-b925cece9537/main.assets/gr5.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs6" aria-label="Go to Figure&amp;nbsp;S6"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.11.017/asset/405a5c2d-3487-4b04-8223-c5b2b7b9f98c/main.assets/figs6.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig6" aria-label="Go to Figure&amp;nbsp;6"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.11.017/asset/71cb8572-f96b-47b2-a4c8-7757280fbbf4/main.assets/gr6.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs7" aria-label="Go to Figure&amp;nbsp;S7"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.11.017/asset/269bb558-7e08-4019-b6fd-8dc4c8727d05/main.assets/figs7.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig7" aria-label="Go to Figure&amp;nbsp;7"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.11.017/asset/6403b143-9879-49e3-ad61-7a7601d3a21e/main.assets/gr7.jpg" aria-hidden="true"></a></div><button class="trigger" aria-controls="collapsible-figures_content" aria-expanded="false" tabindex="0" role="button"><span class="expand">Show all figures</span><span class="collapse">Hide figures</span><i class="icon-section_arrow_d" aria-hidden="true"></i></button></div></div></section>
    

        


    

            

            

            <section id="core-collateral-metrics" class="publication-list__axel-section"><h2>Article metrics</h2><div><ul class="publication-list rlist publication-list--list plum-x--lazy-load publication-list--axel" data-plum-x-url="/pb/widgets/ux3/publicationList/plumx?widgetId=66dd7e11-7768-4006-93d8-a72b60e2a264&amp;pbContext=%3Bproduct%3Aproduct%3Aelsevier%5C%3Aproduct%5C%3Acell%3Bwebsite%3Awebsite%3Acell-site%3Bctype%3Astring%3AJournal+Content%3Bjournal%3Ajournal%3Acell%3BpageGroup%3Astring%3AArticle+View%3Barticle%3Aarticle%3Apii%5C%3AS0092867421013325%3Bpage%3Astring%3AShow+Full+Text%3BrequestedJournal%3Ajournal%3Acell%3Bissue%3Aissue%3Apii%5C%3AS0092867420X00262%3Bwgroup%3Astring%3ADefault+Website+Group%3BsubPage%3Astring%3AFull+Text" data-pb-mode-enabled="false"><li aria-label="loading articles" role="alert" class="js--loading plum-x--spinner hidden"></li></ul></div></section>

            

        
<section id="core-collateral-supplementary"><h2>Supplemental information (7)</h2><div class="core-collateral-aside-wrapper"><div class="core-supplementary-material"><div class="core-link"><button class="downloadAllSupplementary"><span>Download all</span></button></div></div><div class="core-supplementary-material" data-category="pdf"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2021.11.017/attachment/1ca3e3ad-032c-4019-a9e3-69cc3d390193/mmc1.pdf" download="mmc1.pdf"><span id="mmc1-link">PDF (165.61 KB)</span></a><div class="core-description">Table S1. Cohort patient characteristics, related to Figure&nbsp;1</div></div></div><div class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2021.11.017/attachment/c92d07b6-50d2-4ff3-b27e-2d56885c8814/mmc2.xlsx" download="mmc2.xlsx"><span id="mmc2-link">Spreadsheet (1.85 MB)</span></a><div class="core-description">Table S2. Normal cell type markers, related to Figures 1, 5, and 7</div></div></div><div class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2021.11.017/attachment/bbbc8bb8-00aa-4bc6-9d8f-b27b3d3850ee/mmc3.xlsx" download="mmc3.xlsx"><span id="mmc3-link">Spreadsheet (1.70 MB)</span></a><div class="core-description">Table S3. Malignant cell state single-cell gene correlates, signatures, mIF marker combinations, and cell counts, related to Figure&nbsp;2</div></div></div><div class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2021.11.017/attachment/b793ae79-62bc-40e5-afad-31cc46627715/mmc4.xlsx" download="mmc4.xlsx"><span id="mmc4-link">Spreadsheet (508.15 KB)</span></a><div class="core-description">Table S4. Organoid- and <i>in&nbsp;vivo</i> malignant-specific gene expression features, related to Figures 3 and 4</div></div></div><div class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2021.11.017/attachment/31a1fefb-c62c-435d-9a8a-a77e749a910e/mmc5.xlsx" download="mmc5.xlsx"><span id="mmc5-link">Spreadsheet (13.98 KB)</span></a><div class="core-description">Table S5. Organoid and cell line models and media formulations for perturbation experiments, related to Figures 3, 4, 6, and 7</div></div></div><div id="collapsible-supplementary" data-expandable="item"><div class="content" style="display:none;" role="region" id="collapsible-supplementary_content"><div class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2021.11.017/attachment/366a4706-6157-4775-9768-e6e0510c48f6/mmc6.xlsx" download="mmc6.xlsx"><span id="mmc6-link">Spreadsheet (522.80 KB)</span></a><div class="core-description">Table S6. Cell state-specific autocrine and paracrine secreted factors, related to Figures 6 &amp; 7</div></div></div><div class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2021.11.017/attachment/20f729e7-71ec-4f32-8afd-bf1300d63415/mmc7.xlsx" download="mmc7.xlsx"><span id="mmc7-link">Spreadsheet (39.12 KB)</span></a><div class="core-description">Table S7. Drug response parameters for isogenic organoid models induced to adopt different cell states by altering media conditions, related to Figure&nbsp;6</div></div></div></div><button class="trigger" aria-controls="collapsible-supplementary_content" aria-expanded="false" tabindex="0" role="button"><span class="expand">Show more</span><span class="collapse">Show less</span><i class="icon-section_arrow_d" aria-hidden="true"></i></button></div></div></section>
    

        


    

            

            

            <section id="core-collateral-relatedArticles" class="axel-related-articles"><h2 id="relatedArticles" class="top axel-related-articles__title" data-title="Related articles">Related Articles</h2><div data-pb-widget-type="ux3-relatedArticles" data-widget-id="8d3844d1-450e-497c-ab75-a63fc41750ac" data-widget-name="AtyponTempUX3RelatedArticlesWidget" data-article-pii="S0092867421013325" class="ux3DeferredOnScrollWidget axel-related-articles__body" data-show-more="true"></div></section>

            

        
</div></div><aside data-core-aside="right-rail"><div class="inner-wrapper">

    

        


    

            

            

            <div data-widget-id="dca7470d-9085-4d21-a6f4-a66397718146" data-widget-name="UX3HTMLWidget"><input type="hidden" id="gptPage" value="cell-videos"></div>

            

        











        <div data-widget-def="literatumAd" data-widget-id="481d10c2-bbe0-470e-a5ba-207a81ecc8dc" class="adplaceholders" style="right: 0px; position: static;">
        
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd alignCenter  widget-verticalAd" id="481d10c2-bbe0-470e-a5ba-207a81ecc8dc">
                
                
                    
                

                <div class="wrapped " id="Marketplace-Recs">
                    
                    <div class="widget-body body body-verticalAd "><div class="verticalAdContainer">
    <div class="verticalAdLabelContainer">
        <div class="verticalAdLabel">
                <img src="/templates/jsp/_style2/_marlin/images/bg_adVert.gif" alt="Advertisement">
        </div>
    </div>
    <div class="verticalAd">
        





        <div class="pb-ad">
            <script src="//cdn.pbgrd.com/core-cell.js" async=""></script>
<div id="pbgrd-skyr-c"><div id="pbgrd-skyr" style="width: inherit; position: relative; background: transparent;"></div></div>
        </div>
    

    </div>
</div></div>
                </div>
                </div>

        

        </div>
    

</div></aside><div class="core-authors-details" style="display:none"><section class="core-authors"></section></div></article><div data-extent="article-wrapper"><div class="core-container"></div></div><script id="axel-publication-metadata" type="application/json">{"doi":"10.1016/j.cell.2021.11.017","data-is-retraction":false,"data-is-retired":false}
</script><script type="text/javascript" data-ot-ignore="data-ot-ignore" defer="defer" src="https://static.addtoany.com/menu/page.js" class="optanon-category-C0004 ot-vscat-C0004"></script><template id="figure_nav_template"><a href="#" class="open-in-viewer btn btn-dark figure-pop-btn"><i aria-hidden="true" class="icon-export_citation"></i><span>Open in viewer</span></a></template><div id="figure-viewer" role="dialog" aria-modal="true" aria-label="Figure Viewer" class="showCaption showThumbstrip figure-viewer"><nav class="nav-bar figure-viewer__top"><a href="#" title="Close Figure Viewer" role="button" class="figure-viewer__close"><i aria-hidden="true" class="icon-tools_close"></i></a><div class="figure-viewer__title">Microenvironment drives cell state, plasticity, and drug response in pancreatic cancer</div></nav><div class="frames"><ul class="frames__list"><li><div class="thumbnail__image"><img id="gr1" src="/cms/10.1016/j.cell.2021.11.017/asset/92bcff86-ebe0-4bad-8a05-84b74de20e9f/main.assets/gr1.jpg" height="666" width="657" aria-labelledby="fig1-title fig1-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.017/asset/e7f58a9a-6cf2-4205-80e5-ed5bd5636ad4/main.assets/gr1_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs1" src="/cms/10.1016/j.cell.2021.11.017/asset/daaa3cd2-7403-4552-903f-3868a004318c/main.assets/figs1.jpg" height="921" width="728" aria-labelledby="figs1-title figs1-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.017/asset/575614e1-82a5-4936-b0bb-cfd858f0810e/main.assets/figs1_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr2" src="/cms/10.1016/j.cell.2021.11.017/asset/9f44ae90-e7b3-4fe9-a3c5-2a09a5f21fcc/main.assets/gr2.jpg" height="573" width="657" aria-labelledby="fig2-title fig2-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.017/asset/6d000041-a3a0-48c1-ba5f-0f8cc0c50bb4/main.assets/gr2_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs2" src="/cms/10.1016/j.cell.2021.11.017/asset/c5bc289b-af8c-4c9f-8ddc-63d45cd4a98c/main.assets/figs2.jpg" height="799" width="657" aria-labelledby="figs2-title figs2-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.017/asset/03b1f209-d1c0-4b65-b75c-8382c1a39984/main.assets/figs2_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs3" src="/cms/10.1016/j.cell.2021.11.017/asset/21b9c88f-5212-44bd-afc1-85eabf7e0270/main.assets/figs3.jpg" height="839" width="657" aria-labelledby="figs3-title figs3-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.017/asset/c86b72ae-26c7-4e59-8bb0-5f1375419d3e/main.assets/figs3_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs4" src="/cms/10.1016/j.cell.2021.11.017/asset/d35ebd64-74e1-423a-a7a2-7058ea8e9b8d/main.assets/figs4.jpg" height="831" width="657" aria-labelledby="figs4-title figs4-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.017/asset/05e8c667-68ad-421c-8e24-0cfcd56344c2/main.assets/figs4_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr3" src="/cms/10.1016/j.cell.2021.11.017/asset/19bb9976-a1d2-402e-9a39-6796596f35b0/main.assets/gr3.jpg" height="730" width="657" aria-labelledby="fig3-title fig3-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.017/asset/9ed7679c-2c8c-4d73-a28b-d3549ebcb057/main.assets/gr3_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs5" src="/cms/10.1016/j.cell.2021.11.017/asset/10e8c5f7-7b3d-4df7-8a1b-cddd166c3699/main.assets/figs5.jpg" height="929" width="753" aria-labelledby="figs5-title figs5-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.017/asset/ceb2d29e-0a03-491e-a4b5-0ef64c698d55/main.assets/figs5_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr4" src="/cms/10.1016/j.cell.2021.11.017/asset/9e730ef2-f7ed-4457-8b56-166d5c077dc7/main.assets/gr4.jpg" height="721" width="657" aria-labelledby="fig4-title fig4-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.017/asset/0fbcc286-7536-4d99-b3cd-16edafb21927/main.assets/gr4_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr5" src="/cms/10.1016/j.cell.2021.11.017/asset/6324ba1e-d8e9-4503-981c-b925cece9537/main.assets/gr5.jpg" height="851" width="657" aria-labelledby="fig5-title fig5-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.017/asset/97e19c23-fef4-44b3-a993-fbd7e8659517/main.assets/gr5_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs6" src="/cms/10.1016/j.cell.2021.11.017/asset/405a5c2d-3487-4b04-8223-c5b2b7b9f98c/main.assets/figs6.jpg" height="941" width="729" aria-labelledby="figs6-title figs6-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.017/asset/834d14a7-0517-4d2d-b310-6c8227eb0dd0/main.assets/figs6_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr6" src="/cms/10.1016/j.cell.2021.11.017/asset/71cb8572-f96b-47b2-a4c8-7757280fbbf4/main.assets/gr6.jpg" height="835" width="657" aria-labelledby="fig6-title fig6-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.017/asset/ca83b174-1ae6-4c2e-8950-72044d3154b8/main.assets/gr6_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs7" src="/cms/10.1016/j.cell.2021.11.017/asset/269bb558-7e08-4019-b6fd-8dc4c8727d05/main.assets/figs7.jpg" height="941" width="720" aria-labelledby="figs7-title figs7-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.017/asset/655e8721-3c4e-4060-8191-e6bc1c5c2962/main.assets/figs7_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr7" src="/cms/10.1016/j.cell.2021.11.017/asset/6403b143-9879-49e3-ad61-7a7601d3a21e/main.assets/gr7.jpg" height="917" width="657" aria-labelledby="fig7-title fig7-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.017/asset/2742d5fd-de46-4740-9ef0-97a2f9650535/main.assets/gr7_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li></ul><a href="#" title="Previous Frame" class="frameButton previousFrameButton frames__prev"><!--span Previous Frame--><i class="icon-arrow_l"></i></a><a href="#" title="Next Frame" class="frameButton nextFrameButton frames__next"><!--span Next Frame--><i class="icon-arrow_r"></i></a></div><div class="caption"><div class="figure-viewer__caption-body"></div></div><div class="toolbar figure-viewer__toolbar"><span class="figure-viewer__counter"></span><a href="#" class="button toggle-caption-button figure-viewer__caption__ctrl"><i aria-hidden="true" class="icon icon-info"></i><span class="label">Hide Caption</span></a><a href="#" class="button download-options-button"><i aria-hidden="true" class="icon icon-gizmo-folder-in download-size-icon"></i><span class="label">Download</span></a><a href="#figures" class="button see-figures figure-viewer__see-figure__ctrl"><i aria-hidden="true" class="icon icon-gizmo-image_figure viewer-size-icon"></i><span class="label">See figure in Article</span></a></div><div class="thumbstrip"><a href="#" class="toggleButton thumbstrip__ctrl"><span>Toggle Thumbstrip</span></a><a href="#" title="Previous Thumbnails" class="thumbstripButton previousThumbstripButton thumbstrip__prev"><i aria-hidden="true" class="icon-arrow_l"></i></a><a href="#" title="Next Thumbnails" class="thumbstripButton nextThumbstripButton thumbstrip__next"><i aria-hidden="true" class="icon-arrow_r"></i></a><div class="thumbstrip__wrapper"><ul class="rlist figure-viewer__thumbslist"><li><div class="thumbnail__label">Figure&nbsp;1</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr1" src="/cms/10.1016/j.cell.2021.11.017/asset/92bcff86-ebe0-4bad-8a05-84b74de20e9f/main.assets/gr1.jpg" height="666" width="657" aria-labelledby="fig1-title fig1-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.017/asset/e7f58a9a-6cf2-4205-80e5-ed5bd5636ad4/main.assets/gr1_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S1</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs1" src="/cms/10.1016/j.cell.2021.11.017/asset/daaa3cd2-7403-4552-903f-3868a004318c/main.assets/figs1.jpg" height="921" width="728" aria-labelledby="figs1-title figs1-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.017/asset/575614e1-82a5-4936-b0bb-cfd858f0810e/main.assets/figs1_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;2</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr2" src="/cms/10.1016/j.cell.2021.11.017/asset/9f44ae90-e7b3-4fe9-a3c5-2a09a5f21fcc/main.assets/gr2.jpg" height="573" width="657" aria-labelledby="fig2-title fig2-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.017/asset/6d000041-a3a0-48c1-ba5f-0f8cc0c50bb4/main.assets/gr2_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S2</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs2" src="/cms/10.1016/j.cell.2021.11.017/asset/c5bc289b-af8c-4c9f-8ddc-63d45cd4a98c/main.assets/figs2.jpg" height="799" width="657" aria-labelledby="figs2-title figs2-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.017/asset/03b1f209-d1c0-4b65-b75c-8382c1a39984/main.assets/figs2_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S3</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs3" src="/cms/10.1016/j.cell.2021.11.017/asset/21b9c88f-5212-44bd-afc1-85eabf7e0270/main.assets/figs3.jpg" height="839" width="657" aria-labelledby="figs3-title figs3-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.017/asset/c86b72ae-26c7-4e59-8bb0-5f1375419d3e/main.assets/figs3_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S4</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs4" src="/cms/10.1016/j.cell.2021.11.017/asset/d35ebd64-74e1-423a-a7a2-7058ea8e9b8d/main.assets/figs4.jpg" height="831" width="657" aria-labelledby="figs4-title figs4-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.017/asset/05e8c667-68ad-421c-8e24-0cfcd56344c2/main.assets/figs4_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;3</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr3" src="/cms/10.1016/j.cell.2021.11.017/asset/19bb9976-a1d2-402e-9a39-6796596f35b0/main.assets/gr3.jpg" height="730" width="657" aria-labelledby="fig3-title fig3-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.017/asset/9ed7679c-2c8c-4d73-a28b-d3549ebcb057/main.assets/gr3_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S5</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs5" src="/cms/10.1016/j.cell.2021.11.017/asset/10e8c5f7-7b3d-4df7-8a1b-cddd166c3699/main.assets/figs5.jpg" height="929" width="753" aria-labelledby="figs5-title figs5-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.017/asset/ceb2d29e-0a03-491e-a4b5-0ef64c698d55/main.assets/figs5_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;4</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr4" src="/cms/10.1016/j.cell.2021.11.017/asset/9e730ef2-f7ed-4457-8b56-166d5c077dc7/main.assets/gr4.jpg" height="721" width="657" aria-labelledby="fig4-title fig4-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.017/asset/0fbcc286-7536-4d99-b3cd-16edafb21927/main.assets/gr4_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;5</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr5" src="/cms/10.1016/j.cell.2021.11.017/asset/6324ba1e-d8e9-4503-981c-b925cece9537/main.assets/gr5.jpg" height="851" width="657" aria-labelledby="fig5-title fig5-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.017/asset/97e19c23-fef4-44b3-a993-fbd7e8659517/main.assets/gr5_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S6</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs6" src="/cms/10.1016/j.cell.2021.11.017/asset/405a5c2d-3487-4b04-8223-c5b2b7b9f98c/main.assets/figs6.jpg" height="941" width="729" aria-labelledby="figs6-title figs6-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.017/asset/834d14a7-0517-4d2d-b310-6c8227eb0dd0/main.assets/figs6_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;6</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr6" src="/cms/10.1016/j.cell.2021.11.017/asset/71cb8572-f96b-47b2-a4c8-7757280fbbf4/main.assets/gr6.jpg" height="835" width="657" aria-labelledby="fig6-title fig6-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.017/asset/ca83b174-1ae6-4c2e-8950-72044d3154b8/main.assets/gr6_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S7</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs7" src="/cms/10.1016/j.cell.2021.11.017/asset/269bb558-7e08-4019-b6fd-8dc4c8727d05/main.assets/figs7.jpg" height="941" width="720" aria-labelledby="figs7-title figs7-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.017/asset/655e8721-3c4e-4060-8191-e6bc1c5c2962/main.assets/figs7_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;7</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr7" src="/cms/10.1016/j.cell.2021.11.017/asset/6403b143-9879-49e3-ad61-7a7601d3a21e/main.assets/gr7.jpg" height="917" width="657" aria-labelledby="fig7-title fig7-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.017/asset/2742d5fd-de46-4740-9ef0-97a2f9650535/main.assets/gr7_lrg.jpg" loading="lazy" tabindex="0"></div></div></li></ul></div></div><div class="download figure-viewer__download"><div class="pointer"></div><ul class="rlist downlad__wrapper"><li class="hidden-xs hidden-sm"><a href="/action/downloadFigures?pii=e00489&amp;id=" class="downloadAllButton"><i class="icon icon-gizmo-ppt figure-viewer__download__viewer-size-icon figure-viewer__download__viewer-size-icon--ppt"></i><span class="figure-viewer__download__text"> Download .PPT</span></a></li></ul></div></div><template id="toCitationLink" data-citation="Citation" data-footnote="Footnote"><div class="to-citation__wrapper"><a class="to-citation"><i aria-hidden="true" class="icon-subdirectory_arrow_left"></i><span>Go to</span></a></div></template><template id="toCitationButton" data-citation="Citation" data-footnote="Footnote"><div class="to-citation__wrapper"><button class="to-citation"><i aria-hidden="true" class="icon-subdirectory_arrow_left"></i><span>Go to</span></button></div></template><template id="toCitationAccordion" data-collapse-citation="Hide Citations" data-expand-citation="Show Citations" data-collapse-footnote="Hide Footnotes" data-expand-footnote="Show Footnotes"><div class="to-citation__wrapper"><button aria-controls="aria-controls" aria-expanded="false" aria-label="Toggle citations menu" class="to-citation__toggle"><i aria-hidden="true" class="icon-plus"></i><span data-expand-title="data-expand-title" data-collapsed-title="data-collapsed-title" class="accordion__toggle__title"></span></button><div role="menu" aria-label="links" class="to-citation__accordion no-separator"><ul></ul></div></div></template><template id="toCitationAccordionItem"><li><a class="to-citation"><i aria-hidden="true" class="icon-subdirectory_arrow_left"></i><span></span></a></li></template><template id="citations_truncate_template"><div class="citations-truncation"><button data-label-expand="Show all references" data-label-collapse="Show fewer" data-label-remaining="references remaining" class="btn btn--inverse"><span>Show all references</span></button></div></template><template id="collateral_texts_template" data-do="Digital Object" data-figures="Figures" data-metrics="Article Metrics" data-related="Related articles" data-supplementary="Supplemental information"></template><template id="collapsible_authors_template"><button aria-expanded="false" data-expandable="all" data-label-expand="Expand All" data-label-collapse="Collapse All" class="collateral-contributors-control"><span>Expand All</span></button></template>
<template id="collapsible_tables_collapse_template"><div class="collapsible-figure-btn__wrapper expanded"><button aria-expanded="true" class="btn collapsible-figure-btn btn--inverse"><i aria-hidden="true" class="icon-arrow-up"></i><span class="text-uppercase">Collapse</span></button></div></template>
<template id="collapsible_tables_expand_template"><div class="collapsible-figure-btn__wrapper collapsed"><button aria-expanded="false" class="btn collapsible-figure-btn btn--inverse"><i aria-hidden="true" class="icon-arrow-down"></i><span class="text-uppercase">Expand Table</span></button></div></template><template id="authorsAffiliationsLink"><a href="#tab-contributors" class="to-authors-affiliations">Authors Info &amp; Affiliations</a></template></div>
                </div>
                </div>

        

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none  widget-  widget-compact-all" id="756cf18d-e910-4468-936c-0225ae7c669f">
                
                
                    
                

                <div class="wrapped " id="ad for nucleus-single-molecule-technologies">
                    
                    <div class="widget-body body body-  body-compact-all"><div class="pb-ad">
            <script type="text/javascript">
URL_var = 'https://www.cell.com/cell/fulltext/S0092-8674(21)01332-5';

if(URL_var.indexOf("S0165-6147(17)30205-5") > -1
|| URL_var.indexOf("S0167-7799(18)30204-X") > -1 
|| URL_var.indexOf("S0092-8674(18)31649-0") > -1 
|| URL_var.indexOf("S0092-8674(19)30404-0") > -1 
|| URL_var.indexOf("S1097-2765(18)30756-1") > -1 
|| URL_var.indexOf("S0092-8674(18)31452-1") > -1 
|| URL_var.indexOf("S1097-2765(19)30175-3") > -1 
|| URL_var.indexOf("S0092-8674(18)30852-3") > -1 
|| URL_var.indexOf("S0006-3495(18)30295-9") > -1 
|| URL_var.indexOf("S1097-2765(19)30063-2") > -1 
|| URL_var.indexOf("S0006-3495(17)33208-3") > -1 
|| URL_var.indexOf("S2211-1247(18)31639-5") > -1 
|| URL_var.indexOf("S1097-2765(18)30978-X") > -1 
)
    {
document.write('<input type="hidden" id="gpttc" value="nucleus-single-molecule-technologies" class="gpt-Target">')
}
</script>
        </div></div>
                </div>
                </div>

        

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none  widget-" id="754b1780-1117-4371-9285-9cd7020963d5">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body- "><div class="pb-ad">
            <script type="text/javascript">
URL_var = 'https://www.cell.com/cell/fulltext/S0092-8674(21)01332-5';

if(URL_var.indexOf("S1535-6108(19)30104-7") > -1
|| URL_var.indexOf("S0092-8674(18)31394-1") > -1
|| URL_var.indexOf("S1074-7613(19)30033-0") > -1
|| URL_var.indexOf("S1535-6108(18)30473-2") > -1
|| URL_var.indexOf("S0092-8674(19)30154-0") > -1
|| URL_var.indexOf("S1550-4131(18)30178-5") > -1
|| URL_var.indexOf("S1535-6108(19)30242-9") > -1
|| URL_var.indexOf("S1074-7613(18)30431-X") > -1
|| URL_var.indexOf("S0092-8674(19)30618-X") > -1
|| URL_var.indexOf("S1535-6108(18)30472-0") > -1
|| URL_var.indexOf("S1535-6108(18)30120-X") > -1
|| URL_var.indexOf("S0092-8674(18)31247-9") > -1
|| URL_var.indexOf("S1074-7613(18)30085-2") > -1
|| URL_var.indexOf("S1074-7613(18)30080-3") > -1
|| URL_var.indexOf("S1471-4906(18)30210-2") > -1
|| URL_var.indexOf("S1550-4131(19)30304-3") > -1
|| URL_var.indexOf("S1934-5909(18)30599-X") > -1
|| URL_var.indexOf("S2405-8033(19)30024-X") > -1
|| URL_var.indexOf("S1535-6108(18)30109-0") > -1
|| URL_var.indexOf("S0962-8924(18)30127-2") > -1
)
    {
document.write('<input type="hidden" id="gpttc" value="nucleus-tumor-immune-microenvironment" class="gpt-Target">')
}
</script>
        </div></div>
                </div>
                </div>

        

    

        


    

            

            

            <div data-widget-id="9c9ff7a1-a035-49f6-a336-600c9419c518" data-widget-name="UX3HTMLWidget"><!-- Start of HubSpot Embed Code --> 

<script type="text/plain" class="optanon-category-4" id="hs-script-loader" async="" defer="" src="//js.hs-scripts.com/519224.js"></script> 

<!-- End of HubSpot Embed Code --></div>

            

        

    

        


    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget sessionValidationWidget none  widget-none  widget-compact-all" id="0257aef7-6074-4898-a989-1c3f693225a2">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none  body-compact-all"><script>
    $.ajax({
        url: "https://www.cell.com/ssoSessionCheck",
        type: "GET",
        dataType: "jsonp",
        success: function (data) {
            if(data.RefreshSession) {
                location.reload();
            }
        },
        error : function (){
            console.warn("can't validate CSAS session.");
        }
    });
</script></div>
                </div>
                </div>

        

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget csasSsoUrls none  widget-none  widget-compact-all" id="948bc5c4-8b0d-47be-b5ad-6570559802d1">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none  body-compact-all"><!-- clear SSO URLs after all iFrames are rendered or after passing a time out-->
<script>
    setTimeout(clearSsoUrlsFun,10000);
    var iFramesList = document.getElementById("sso-urls-div").getElementsByTagName("iframe");
    var renderedIFramesCount = 0;
    var numberOfIFrames = iFramesList.length;
    var skipRefresh = false;
    for (var i = 0; i < iFramesList.length; i++) {
        var iFrame = iFramesList[i];
        iFrame.addEventListener("load", function() {
            renderedIFramesCount = renderedIFramesCount + 1;
            if (renderedIFramesCount >= numberOfIFrames)
            {
                clearSsoUrlsFun();
            }
        });
    }

    function clearSsoUrlsFun() {
        if (skipRefresh == false && numberOfIFrames > 0)
        {
            skipRefresh = true;
            $.ajax({
                url: "https://www.cell.com/action/clearSsoUrls",
                type: "GET",
                dataType: "jsonp",
                error : errorCallingAjx
            });
        }
    }

    function errorCallingAjx(){
        console.log("can't delete sso urls .")
    }
</script></div>
                </div>
                </div>

        










        <div data-widget-def="ux3-layout-widget" data-widget-id="d95b4a46-dc4d-4ca8-83f9-8b6724c649ec" class="ux3 ">
        
    

        


    

            

            

            <div data-widget-id="d95b4a46-dc4d-4ca8-83f9-8b6724c649ec" data-widget-name="ux3-layout-widget" class="fonts-loaded"><div class="footer clearfix">
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none adplaceholder adplaceholder--header widget-horizontalAd" id="a07ee92b-3942-49f8-9186-a2a08fc3fc56">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-horizontalAd "><div class="horizontalAdContainer">
    <div class="horizontalAdLabelContainer">
        <div class="horizontalAdLabel">
                <img src="/templates/jsp/_style2/_marlin/images/bg_adHoriz.gif" alt="Advertisement">
        </div>
    </div>
    <div class="horizontalAd">
        





        <div class="pb-ad">
            <!-- Beginning Async AdSlot 4 for Ad unit els.cellpress/cell  ### size: [[728,90]] -->
<div class="gpt-ad" data-adsize="cell-leaderboard" data-page="" data-site="cell" data-pos="4"></div>
<!-- End AdSlot 4 -->
        </div>
    

    </div>
</div></div>
                </div>
                </div>

        

    

        


    

            

            

            <div data-widget-id="23703bbc-af46-4f3f-a5bb-dd52891ff0ff" data-widget-name="UX3HTMLWidget"><div class="footer__top">
    <div class="footer__logo centered"><a href="/home" title="Cell Press"><!-- Generator: Adobe Illustrator 25.2.1, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 300 72" style="enable-background:new 0 0 300 72;" xml:space="preserve">
<style type="text/css">
	.st0{fill:#007DBC;}
</style>
<g>
	<path class="st0" d="M39.4,7.5c-2.3-0.3-4.4,1.1-4.9,3.3l-1.8,7.4c-0.4,1.8-1.9,3.1-3.8,3.3l-10.5,1.2C7.1,24-1,34.3,0.7,45.6
		c1.3,9,8.6,15.7,17.2,17c2.3,0.3,4.4-1.1,4.9-3.3l1.8-7.4c0.4-1.8,1.9-3.1,3.8-3.3l10.5-1.2c11.3-1.2,19.4-11.6,17.8-22.9
		C55.3,15.4,48.1,8.8,39.4,7.5z M37.1,34l-6.6,0.8c-1.1,0.1-2,0.9-2.3,2l-1.9,8.1c-0.6,2.4-2.5,4.2-5,4.5c-3.8,0.4-7.1-2.4-7.4-6.2
		c-0.3-3.7,2.6-6.8,6.3-7.2l6.6-0.8c1.1-0.1,2-0.9,2.3-2l1.9-8.1c0.6-2.4,2.5-4.2,5-4.5c3.8-0.4,7.1,2.4,7.4,6.2
		C43.7,30.5,40.8,33.6,37.1,34z"></path>
	<rect x="153.1" y="13.9" class="st0" width="8.2" height="42.3"></rect>
	<path class="st0" d="M90.2,20.4c7.3,0,8.6,4.8,9.4,7.3h8.8c-0.5-6.2-5.6-14.9-18.2-14.9c-10.9,0-19.4,7.5-19.4,22.3
		c0,14.6,8.1,22.2,19.3,22.2c10.6,0,16.6-6.4,18.4-15.6h-8.8c-1.1,5.2-4.5,7.9-9.4,7.9c-7.5,0-10.5-6.8-10.5-14.4
		C79.7,23.3,85.9,20.4,90.2,20.4z"></path>
	<path class="st0" d="M125.2,24c-13.7,0-15,12.1-15,15.8c0,12.9,6.8,17.1,15.6,17.1c5.4,0,11.6-2.5,13.8-10H131
		c-0.7,2.2-3,3.4-5.3,3.4c-6.6,0-7.1-5.3-7.3-7.7h21.5v-1.5C139.9,27,131.3,24,125.2,24z M118.6,37.4c0.5-4.1,2.5-6.6,6.4-6.6
		c2.6,0,5.9,1.5,6.4,6.6H118.6z"></path>
	<rect x="142" y="13.9" class="st0" width="8.2" height="42.3"></rect>
	<path class="st0" d="M174,13.8h-9.8V21h7.2c4.3,0,8.2,0.9,8.2,6.4c0,5.4-3.9,6.4-8.2,6.4h-7.2V41h9.8c10.6,0,14.7-6.7,14.7-13.6
		C188.7,20.5,184.6,13.8,174,13.8z"></path>
	<path class="st0" d="M210.5,24.9c-0.5-0.2-1.2-0.3-1.8-0.3c-4.1,0-8.3,2.7-9.8,6.5h-0.1v-5.7h-8v30.7h8.4V42.3
		c0-5.4,2.1-9.9,8.3-9.9c1,0,2.3,0.1,3,0.3V24.9z"></path>
	<path class="st0" d="M225.6,24.6c-9.3,0-15.7,7-15.7,16.2c0,9.5,6,16.1,15.7,16.1c6.9,0,12-3.1,14.4-10.3h-7.4
		c-0.5,1.9-3.3,4-6.6,4c-4.7,0-7.3-2.4-7.5-7.8h22.1C241.1,33.4,236,24.6,225.6,24.6z M218.4,37.5c0.1-2.4,1.7-6.5,7-6.5
		c4.1,0,5.9,2.3,6.7,6.5H218.4z"></path>
	<path class="st0" d="M260.3,37.7c-4.8-1.1-9.6-1.5-9.6-4.6c0-2.5,2.8-2.9,4.7-2.9c3,0,5.5,0.9,5.8,4.2h8c-0.7-7.5-6.8-9.8-13.5-9.8
		c-6.6,0-13.4,2-13.4,9.9c0,5.4,4.9,7.1,9.7,8.2c6.1,1.3,9.6,2.2,9.6,4.8c0,3-3.1,3.9-5.5,3.9c-3.3,0-6.4-1.5-6.5-5.2h-8
		C242,54.3,248.9,57,256,57c7,0,14-2.6,14-10.7C270.1,40.6,265.2,38.8,260.3,37.7z"></path>
	<path class="st0" d="M289.8,37.7c-4.8-1.1-9.6-1.5-9.6-4.6c0-2.5,2.8-2.9,4.7-2.9c3,0,5.5,0.9,5.8,4.2h8c-0.7-7.5-6.8-9.8-13.5-9.8
		c-6.6,0-13.4,2-13.4,9.9c0,5.4,4.9,7.1,9.7,8.2c6.1,1.3,9.6,2.2,9.6,4.8c0,3-3.1,3.9-5.5,3.9c-3.3,0-6.4-1.5-6.5-5.2h-8
		c0.4,8.1,7.3,10.8,14.4,10.8c7,0,14-2.6,14-10.7C299.6,40.6,294.7,38.8,289.8,37.7z"></path>
	<path class="st0" d="M34.1,57.6c0.3-0.2,0.4-0.6,0.4-1c0-0.5-0.2-0.9-0.5-1.1c-0.3-0.2-0.8-0.3-1.4-0.3h-1.9v5.1h0.8V58h0.8
		l1.3,2.2h0.9l-1.4-2.2C33.6,57.9,33.9,57.8,34.1,57.6z M32.5,57.3h-0.9v-1.5h1c0.1,0,0.3,0,0.4,0c0.1,0,0.3,0,0.4,0.1
		c0.1,0,0.2,0.1,0.3,0.2c0.1,0.1,0.1,0.2,0.1,0.4c0,0.2,0,0.3-0.1,0.4c-0.1,0.1-0.2,0.2-0.3,0.2c-0.1,0.1-0.2,0.1-0.4,0.1
		C32.8,57.3,32.6,57.3,32.5,57.3z"></path>
	<path class="st0" d="M35.6,54.2c-0.4-0.4-0.9-0.7-1.4-0.9c-0.5-0.2-1.1-0.3-1.7-0.3c-0.6,0-1.2,0.1-1.7,0.3c-0.6,0.2-1,0.5-1.4,0.9
		c-0.4,0.4-0.7,0.9-1,1.5c-0.2,0.6-0.4,1.2-0.4,1.9c0,0.7,0.1,1.3,0.3,1.8c0.2,0.6,0.5,1,0.9,1.5c0.4,0.4,0.9,0.7,1.4,1
		c0.6,0.2,1.2,0.4,1.8,0.4c0.6,0,1.2-0.1,1.7-0.3c0.5-0.2,1-0.5,1.4-0.9c0.4-0.4,0.7-0.9,1-1.5c0.2-0.6,0.4-1.2,0.4-1.9
		c0-0.7-0.1-1.3-0.4-1.9C36.4,55.1,36,54.6,35.6,54.2z M35.7,59.1c-0.2,0.5-0.4,0.9-0.8,1.2c-0.3,0.3-0.7,0.6-1.1,0.8
		c-0.4,0.2-0.9,0.3-1.4,0.3c-0.6,0-1-0.1-1.5-0.3c-0.4-0.2-0.8-0.5-1.1-0.8c-0.3-0.3-0.5-0.7-0.7-1.2c-0.2-0.5-0.2-1-0.2-1.5
		c0-0.6,0.1-1.1,0.3-1.5c0.2-0.5,0.4-0.9,0.8-1.2c0.3-0.3,0.7-0.6,1.1-0.8c0.4-0.2,0.9-0.3,1.4-0.3c0.5,0,1,0.1,1.4,0.3
		c0.4,0.2,0.8,0.4,1.1,0.8c0.3,0.3,0.6,0.7,0.8,1.2c0.2,0.5,0.3,1,0.3,1.5C36,58.1,35.9,58.6,35.7,59.1z"></path>
</g>
</svg></a></div>
    <div class="footer__social-links">
        <ul class="rlist rlist--inline centered">
            <li><a href="http://on.fb.me/1tgHakh" title="Facebook" target="_blank"><i aria-hidden="true" class="icon-Icon_Facebook"></i></a></li>

            <li><a href="https://www.instagram.com/cellpress/" title="Instagram" target="_blank"><svg width="18px" height="18px" viewBox="0 0 18 18" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-147.000000, -16.000000)" fill-rule="nonzero" fill="#969696"> <g id="Instagram-Logo" transform="translate(147.000000, 16.000000)"> <path d="M9,1.62580645 C11.4096774,1.62580645 11.7,1.62580645 12.6290323,1.68387097 C13.5,1.71290323 13.9935484,1.85806452 14.3129032,2.00322581 C14.7193548,2.17741935 15.0387097,2.3516129 15.3580645,2.67096774 C15.6774194,2.99032258 15.8806452,3.28064516 16.0258065,3.71612903 C16.1419355,4.03548387 16.2870968,4.5 16.3451613,5.4 C16.3741935,6.35806452 16.4032258,6.61935484 16.4032258,9.02903226 C16.4032258,11.4387097 16.4032258,11.7290323 16.3451613,12.6580645 C16.316129,13.5290323 16.1709677,14.0225806 16.0258065,14.3419355 C15.8516129,14.7483871 15.6774194,15.0677419 15.3580645,15.3870968 C15.0387097,15.7064516 14.7483871,15.9096774 14.3129032,16.0548387 C13.9935484,16.1709677 13.5290323,16.316129 12.6290323,16.3741935 C11.6709677,16.4032258 11.4096774,16.4322581 9,16.4322581 C6.59032258,16.4322581 6.3,16.4322581 5.37096774,16.3741935 C4.5,16.3451613 4.00645161,16.2 3.68709677,16.0548387 C3.28064516,15.8806452 2.96129032,15.7064516 2.64193548,15.3870968 C2.32258065,15.0677419 2.11935484,14.7774194 1.97419355,14.3419355 C1.85806452,14.0225806 1.71290323,13.5580645 1.65483871,12.6580645 C1.62580645,11.7 1.62580645,11.4096774 1.62580645,9 C1.62580645,6.59032258 1.62580645,6.3 1.68387097,5.37096774 C1.71290323,4.5 1.85806452,4.00645161 2.00322581,3.68709677 C2.17741935,3.28064516 2.3516129,2.96129032 2.67096774,2.64193548 C2.99032258,2.32258065 3.28064516,2.11935484 3.71612903,1.97419355 C4.03548387,1.85806452 4.5,1.71290323 5.4,1.65483871 C6.3,1.62580645 6.59032258,1.62580645 9,1.62580645 L9,1.62580645 Z M9,0 C6.56129032,0 6.24193548,0 5.28387097,0.0580645161 C4.32580645,0.116129032 3.65806452,0.261290323 3.10645161,0.464516129 C2.52580645,0.696774194 2.00322581,1.01612903 1.50967742,1.50967742 C1.01612903,2.00322581 0.696774194,2.52580645 0.464516129,3.10645161 C0.232258065,3.68709677 0.0870967742,4.32580645 0.0580645161,5.28387097 C0,6.24193548 0,6.56129032 0,9 C0,11.4387097 0,11.7580645 0.0580645161,12.716129 C0.116129032,13.6741935 0.261290323,14.3419355 0.464516129,14.8935484 C0.696774194,15.4741935 1.01612903,15.9967742 1.50967742,16.4903226 C2.00322581,16.983871 2.52580645,17.3032258 3.10645161,17.5354839 C3.68709677,17.7677419 4.32580645,17.9129032 5.28387097,17.9419355 C6.24193548,18 6.56129032,18 9,18 C11.4387097,18 11.7580645,18 12.716129,17.9419355 C13.6741935,17.883871 14.3419355,17.7387097 14.8935484,17.5354839 C15.4741935,17.3032258 15.9967742,16.983871 16.4903226,16.4903226 C16.983871,15.9967742 17.3032258,15.4741935 17.5354839,14.8935484 C17.7677419,14.3129032 17.9129032,13.6741935 17.9419355,12.716129 C18,11.7580645 18,11.4387097 18,9 C18,6.56129032 18,6.24193548 17.9419355,5.28387097 C17.883871,4.32580645 17.7387097,3.65806452 17.5354839,3.10645161 C17.3032258,2.52580645 16.983871,2.00322581 16.4903226,1.50967742 C15.9967742,1.01612903 15.4741935,0.696774194 14.8935484,0.464516129 C14.3129032,0.232258065 13.6741935,0.0870967742 12.716129,0.0580645161 C11.7580645,0 11.4387097,0 9,0 L9,0 Z"></path> <path d="M9,4.38387097 C6.44516129,4.38387097 4.38387097,6.44516129 4.38387097,9 C4.38387097,11.5548387 6.44516129,13.616129 9,13.616129 C11.5548387,13.616129 13.616129,11.5548387 13.616129,9 C13.616129,6.44516129 11.5548387,4.38387097 9,4.38387097 Z M9,11.9903226 C7.34516129,11.9903226 6.00967742,10.6548387 6.00967742,9 C6.00967742,7.34516129 7.34516129,6.00967742 9,6.00967742 C10.6548387,6.00967742 11.9903226,7.34516129 11.9903226,9 C11.9903226,10.6548387 10.6548387,11.9903226 9,11.9903226 Z"></path> <circle id="Oval" cx="13.7903226" cy="4.20967742" r="1.07419355"></circle> </g> </g> </g></svg></a></li>
            
            <li><a href="https://www.linkedin.com/company/cell-press/" title="LinkedIn" target="_blank"><i aria-hidden="true" class="icon-linkedin"></i></a></li>
            
                        <li><a href="http://bit.ly/1zq7Q5T" title="Twitter" target="_blank">
                <!--<i aria-hidden="true" class="icon-Icon_Twitter"></i></a>-->
                
                <svg id="xicon" viewBox="0 0 1200 1227" xmlns="http://www.w3.org/2000/svg" aria-hidden="true" role="none"><path fill="#969696" d="M714.163 519.284L1160.89 0H1055.03L667.137 450.887L357.328 0H0L468.492 681.821L0 1226.37H105.866L515.491 750.218L842.672 1226.37H1200L714.137 519.284H714.163ZM569.165 687.828L521.697 619.934L144.011 79.6944H306.615L611.412 515.685L658.88 583.579L1055.08 1150.3H892.476L569.165 687.854V687.828Z"></path></svg> 

                    <style>
                    .footer svg#xicon {
                      height: 16px;
                    }
                    </style>
            </a></li>

            
            <li><a href="/pb-assets/marketing/WeChat/WeChat-1557753222727.jpg" title="WeChat" target="_blank"><svg width="18px" height="15px" viewBox="0 0 18 15" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-122.000000, -17.000000)" fill-rule="nonzero" fill="#969696"> <g id="wechat" transform="translate(122.000000, 17.000000)"> <path d="M12.375,4.5 L12.375,4.5 L12.4875,4.5 L12.4875,4.5 C12.6898579,4.48604473 12.8520408,4.32704187 12.87,4.125 C12.8705905,4.04725572 12.8470025,3.9712499 12.8025,3.9075 C11.94,1.5675 9.4425,0 6.5625,0 C2.9475,0 0,2.52 0,5.625 C0.0313772094,7.34410122 0.895513317,8.94093526 2.3175,9.9075 L1.5675,11.46 C1.49009368,11.6087816 1.52088865,11.7908702 1.64291494,11.9059236 C1.76494122,12.0209769 1.94852501,12.0410168 2.0925,11.955 L4.08,10.8225 C4.61861992,11.0129999 5.17724298,11.1413322 5.745,11.205 C5.86564124,11.2178337 5.98503738,11.1715098 6.06545426,11.0806685 C6.14587113,10.9898272 6.17737153,10.865694 6.15,10.7475 C6.06148727,10.4219419 6.01114891,10.0871918 6,9.75 C6,6.855 8.8575,4.5 12.375,4.5 Z M8.25,3 C8.66421356,3 9,3.33578644 9,3.75 C9,4.16421356 8.66421356,4.5 8.25,4.5 C7.83578644,4.5 7.5,4.16421356 7.5,3.75 C7.5,3.33578644 7.83578644,3 8.25,3 Z M4.5,4.5 C4.08578644,4.5 3.75,4.16421356 3.75,3.75 C3.75,3.33578644 4.08578644,3 4.5,3 C4.91421356,3 5.25,3.33578644 5.25,3.75 C5.25,4.16421356 4.91421356,4.5 4.5,4.5 Z"></path> <path d="M18,9.75 C18,7.2675 15.48,5.25 12.375,5.25 C9.27,5.25 6.75,7.2675 6.75,9.75 C6.75,12.2325 9.27,14.25 12.375,14.25 C12.9568557,14.2502837 13.5364258,14.1772075 14.1,14.0325 L15.96,14.9625 C16.0114248,14.9874074 16.0678613,15.0002339 16.125,15 C16.2497193,14.9993787 16.3659682,14.9367912 16.4351508,14.8330172 C16.5043335,14.7292432 16.5173955,14.5978644 16.47,14.4825 L15.96,13.2075 C17.1784092,12.4716325 17.9450305,11.1723221 18,9.75 L18,9.75 Z M10.5,9 C10.0857864,9 9.75,8.66421356 9.75,8.25 C9.75,7.83578644 10.0857864,7.5 10.5,7.5 C10.9142136,7.5 11.25,7.83578644 11.25,8.25 C11.25,8.66421356 10.9142136,9 10.5,9 Z M14.25,9 C13.8357864,9 13.5,8.66421356 13.5,8.25 C13.5,7.83578644 13.8357864,7.5 14.25,7.5 C14.6642136,7.5 15,7.83578644 15,8.25 C15,8.66421356 14.6642136,9 14.25,9 Z" opacity="0.959999979"></path> </g> </g> </g></svg></a></li>
            
            <li><a href="https://www.weibo.com/CellPress" title="Weibo" target="_blank"><svg width="18px" height="15px" viewBox="0 0 18 15" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-99.000000, -17.000000)" fill-rule="nonzero" fill="#969696"> <g id="weibo" transform="translate(99.000000, 14.000000)"> <g transform="translate(0.000000, 3.000000)"> <path d="M13.2402211,7.47054172 C12.9918392,7.39866484 12.821608,7.34902326 12.9514975,7.03290287 C13.232804,6.34602189 13.2622915,5.75359781 12.9571055,5.33095486 C12.3847236,4.5380684 10.8191759,4.58064295 9.02514573,5.30975376 C9.02514573,5.30889193 8.46144724,5.54882627 8.60544724,5.11515185 C8.88150754,4.25435294 8.83953769,3.53386047 8.41043216,3.11759508 C7.43716583,2.17285363 4.84824121,3.15344733 2.62745729,5.3059617 C0.964944724,6.9186238 0,8.62798358 0,10.105855 C0,12.9324952 3.73748744,14.8235294 7.39429146,14.8235294 C12.1877186,14.8235294 15.3768844,11.9501778 15.3768844,9.80576471 C15.3768844,8.50974008 14.2514774,7.77442408 13.2402211,7.47054172 Z M7.40424121,13.5561176 C4.48625126,13.8315595 1.9678794,12.5686293 1.77811055,10.7327524 C1.58870352,8.89773735 3.80098492,7.18579207 6.71861307,6.90948837 C9.63678392,6.63266758 12.1556985,7.89663201 12.3447437,9.73164706 C12.533608,11.5675239 10.3227739,13.2796416 7.40424121,13.5561176 Z M18,5.16875787 C17.9985528,2.31402189 15.5695477,0 12.5728643,0 C12.222995,0 11.9396985,0.270098495 11.9396985,0.603283174 C11.9396985,0.936640219 12.222995,1.20656635 12.5728643,1.20656635 C14.8708945,1.20656635 16.7336683,2.98159781 16.7336683,5.17099863 C16.7336683,5.50435568 17.0173266,5.77428181 17.3668342,5.77428181 C17.7167035,5.77428181 18,5.50435568 18,5.17099863 C18,5.17099863 18,5.1696197 18,5.16875787 Z M15.4841608,5.08016142 C15.1948945,3.73415048 14.0812462,2.67323393 12.6683819,2.39744733 C12.3262915,2.33039672 11.9914372,2.54051163 11.9212462,2.86680164 C11.8506935,3.19309166 12.0712161,3.51179754 12.4133065,3.57919289 C13.3319397,3.75845417 14.0559196,4.44774829 14.2438794,5.32302599 C14.3146131,5.64931601 14.6491055,5.85925855 14.9915578,5.7923803 C15.3338291,5.72498495 15.5541709,5.40627907 15.4841608,5.08016142 Z M5.90761809,8.7069275 C4.7321005,8.9320383 3.93901508,9.87126402 4.1358392,10.8044569 C4.33248241,11.7378222 5.44504523,12.311803 6.62056281,12.0865198 C7.79571859,11.861409 8.58916583,10.922528 8.3921608,9.98899042 C8.19569849,9.0559699 7.08277387,8.48181669 5.90761809,8.7069275 Z"></path> </g> </g> </g> </g></svg></a></li>
            
            <li><a href="https://www.youtube.com/user/cellvideoabstracts" title="YouTube" target="_blank"><svg width="18px" height="13px" viewBox="0 0 18 13" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-76.000000, -19.000000)" fill-rule="nonzero" fill="#969696"> <g id="youtube" transform="translate(76.000000, 14.000000)"> <path d="M17.2984219,6.41974219 C16.6486641,5.64735937 15.4490625,5.33232422 13.1580703,5.33232422 L4.84175391,5.33232422 C2.49834375,5.33232422 1.27838672,5.66771484 0.631089844,6.49001953 C0,7.29172266 0,8.47304297 0,10.1080547 L0,13.2243047 C0,16.3917773 0.748792969,18 4.84175391,18 L13.1581055,18 C15.1448555,18 16.2457383,17.7219844 16.9579688,17.040375 C17.688375,16.3413984 18,15.2001211 18,13.2243047 L18,10.1080547 C18,8.38381641 17.951168,7.19553516 17.2984219,6.41974219 Z M11.5560703,12.0963867 L7.77965625,14.0700586 C7.69524609,14.1141797 7.60289062,14.136082 7.51071094,14.136082 C7.40629687,14.136082 7.30212891,14.1079922 7.20987891,14.0521289 C7.03613672,13.9468711 6.93007031,13.7585391 6.93007031,13.5554414 L6.93007031,9.62075391 C6.93007031,9.41800781 7.03585547,9.22985156 7.20924609,9.12452344 C7.38267187,9.01919531 7.59839062,9.01198828 7.77832031,9.10546875 L11.5547344,11.0664141 C11.7468633,11.1661523 11.8675195,11.3645039 11.8678008,11.5809258 C11.8680469,11.7975586 11.747918,11.9961914 11.5560703,12.0963867 Z"></path> </g> </g> </g></svg></a></li>
            
        </ul>
    </div>
</div>

<style>

.ux3 .footer__logo svg {
    max-width: 160px !important;
}

.ux3 .footer__social-links a svg {
    max-width: 26px;
    height: 1.15rem;
    width: auto;
}

.ux3 .footer__social-links a:hover svg path {
    fill: #353535;
}

</style></div>

            

        

    

        


    

            

            
                <div class="footer__bottom text-onDark">
            

            <div data-widget-id="06227714-c621-4651-9879-b0ce5176cccc" data-widget-name="ux3-layout-widget"><div class="col-sm-4 col-md-5 clearfix">
    

        


    

            

            

            <div data-widget-id="07e1d276-b4dd-4870-b858-ed74e70d64ef" data-widget-name="UX3HTMLWidget"><div class="groups">
	<div class="group">
		<h2 class="group__title">Life &amp; medical sciences journals</h2>
		<ul class="rlist clearfix sided">
		<li>
			<a alt="Cell" href="/cell/home">Cell</a>
		</li>
		<li>
			<a alt="Cancer Cell" href="/cancer-cell/home">Cancer Cell</a>
		</li>
		<li>
			<a alt="Cell Chemical Biology" href="/cell-chemical-biology">Cell Chemical Biology</a>
		</li>
		<li>
			<a alt="Cell Genomics" href="/cell-genomics/home">Cell Genomics</a>
		</li>
		<li>
			<a alt="Cell Host and Microbe" href="/cell-host-microbe/home">Cell Host &amp; Microbe</a>
		</li>
		<li>
			<a alt="Cell Metabolism" href="/cell-metabolism/home">Cell Metabolism</a>
		</li>
		<li>
			<a alt="Cell Reports" href="/cell-reports/home">Cell Reports</a>
		</li>
		<li>
			<a alt="Cell Reports Medicine" href="/cell-reports-medicine/home">Cell Reports Medicine</a>
		</li>
		<li>
			<a alt="Cell Stem Cell" href="/cell-stem-cell/home">Cell Stem Cell</a>
		</li>
		<li>
			<a alt="Cell Systems" href="/cell-systems/home">Cell Systems</a>
		</li>
		<li>
			<a alt="Current Biology" href="/current-biology/home">Current Biology</a>
		</li>
		<li>
			<a alt="Developmental Cell" href="/developmental-cell/home">Developmental Cell</a>
		</li>
		<li>
			<a alt="Immunity" href="/immunity/home">Immunity</a>
		</li>
		<li>
			<a alt="Med" href="/med/home">Med</a>
		</li>
		<li>
			<a alt="Molecular Cell" href="/molecular-cell/home">Molecular Cell</a>
		</li>
		<li>
			<a alt="Neuron" href="/neuron/home">Neuron</a>
		</li>
		<li>
			<a alt="Structure" href="/structure/home">Structure</a>
		</li>
		<li>
			<a alt="American Journal of Human Genetics" href="/ajhg/home">American Journal of Human Genetics (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Biophysical Journal" href="/biophysj/home">Biophysical Journal (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Biophysical Reports" href="/biophysreports/home">Biophysical Reports (<i>partner</i>)</a>
		</li>	
		<li>
			<a alt="HGG Advances" href="/hgg-advances/home">Human Genetics and Genomics Advances (<i>partner</i>)</a></li>
		<li>
			<a alt="Molecular Plant" href="/molecular-plant/home">Molecular Plant (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy" href="/molecular-therapy-family/molecular-therapy/home">Molecular Therapy (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy Methods and Clinical Development" href="/molecular-therapy-family/methods/home">Molecular Therapy Methods &amp; Clinical Development (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy Nucleic Acids" href="/molecular-therapy-family/nucleic-acids/home">Molecular Therapy Nucleic Acids (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy Oncology" href="/molecular-therapy-family/oncology/home">Molecular Therapy Oncology (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Plant Communications" href="/plant-communications/home">Plant Communications (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Stem Cell Reports" href="/stem-cell-reports/home">Stem Cell Reports (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Trends in Biochemical Sciences" href="/trends/biochemical-sciences/home">Trends in Biochemical Sciences</a>
		</li>
		<li>
		</li><li>
			<a alt="Trends in Cancer" href="/trends/cancer/home">Trends in Cancer</a>
		</li>
		<li>
			<a alt="Trends in Cell Biology" href="/trends/cell-biology/home">Trends in Cell Biology</a>
		</li>
		<li>
			<a alt="Trends in Ecology and Evolution" href="/trends/ecology-evolution/home">Trends in Ecology &amp; Evolution</a>
		</li>
		<li>
			<a alt="Trends in Endocrinology and Metabolism" href="/trends/endocrinology-metabolism/home">Trends in Endocrinology &amp; Metabolism</a>
		</li>
		<li>
			<a alt="Trends in Genetics" href="/trends/genetics/home">Trends in Genetics</a>
		</li>
		<li>
			<a alt="Trends in Immunology" href="/trends/immunology/home">Trends in Immunology</a>
		</li>
		<li>
			<a alt="Trends in Microbiology" href="/trends/microbiology/home">Trends in Microbiology</a>
		</li>
		<li>
			<a alt="Trends in Molecular Medicine" href="/trends/molecular-medicine/home">Trends in Molecular Medicine</a>
		</li>
		<li>
			<a alt="Trends in Neurosciences" href="/trends/neurosciences/home">Trends in Neurosciences</a>
		</li>
		<li>
			<a alt="Trends in Parasitology" href="/trends/parasitology/home">Trends in Parasitology</a>
		</li>
		<li>
			<a alt="Trends in Pharmacological Sciences" href="/trends/pharmacological-sciences/home">Trends in Pharmacological Sciences</a>
		</li>
		<li>
			<a alt="Trends in Plant Science" href="/trends/plant-science/home">Trends in Plant Science</a>
		</li>
		</ul>
	</div>

	<div class="group">
		<h2 class="group__title">Physical sciences &amp; engineering journals</h2>
		<ul class="rlist clearfix sided">
		  <li><a alt="Cell Biomaterials" href="/cell-biomaterials/home">Cell Biomaterials</a></li>
		  <li>
			<a alt="Cell Reports Physical Science" href="/cell-reports-physical-science/home">Cell Reports Physical Science</a>
		</li>
		<li>
			<a alt="Chem" href="/chem/home">Chem</a>
		</li>
		<li>
			<a alt="Chem Catalysis" href="/chem-catalysis/home">Chem Catalysis</a>
		</li>
		<li>
			<a alt="Device" href="/device/home">Device</a>
		</li>
		<li>
			<a alt="Joule" href="/joule/home">Joule</a>
		</li>
		<li>
			<a alt="Matter" href="/matter/home">Matter</a>
		</li>	
		<li><a alt="Matter" href="/newton/home">Newton</a></li>
		<li>
			<a alt="Trends in Chemistry" href="/trends/chemistry/home">Trends in Chemistry</a>
		</li>			
		</ul>
	</div>

<div class="group">
	<h2 class="group__title">Multidisciplinary journals</h2>
	<ul class="rlist clearfix sided">
	  		<li>
			<a alt="Cell Reports Methods" href="/cell-reports-methods/home">Cell Reports Methods</a>
		</li>
		<li>
			<a alt="Cell Reports Sustainability" href="/cell-reports-sustainability/home">Cell Reports Sustainability</a>
		</li>
		<li>
			<a alt="Heliyon" href="/heliyon/home">Heliyon</a>
		</li>	
		<li>
			<a alt="iScience" href="/iscience/home">iScience</a>
		</li>
		<li>
			<a alt="One Earth" href="/one-earth/home">One Earth</a>
		</li>
		<li>
			<a alt="Patterns" href="/patterns/home">Patterns</a>
		</li>
		<li>
			<a alt="STAR Protocols" href="/star-protocols/home">STAR Protocols</a>
		</li>
		<li>
			<a alt="Nexus" href="/nexus/home">Nexus (<i>partner</i>)</a>
		</li>			
        <li>
			<a alt="The Innovation" href="/the-innovation/home">The Innovation (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Trends in Biotechnology" href="/trends/biotechnology/home">Trends in Biotechnology</a>
		</li>
		<li>
			<a alt="Trends in Cognitive Sciences" href="/trends/cognitive-sciences/home">Trends in Cognitive Sciences</a>
		</li>
	</ul>
</div>

</div>
</div>

            

        
</div><div class="col-sm-4 col-md-5 clearfix">
    

        


    

            

            

            <div data-widget-id="850476de-00cb-4c0d-943c-bf1881da24f9" data-widget-name="UX3HTMLWidget"><div class="groups">
	<div class="clearfix">
		<div class="group sided">
			<h2 class="group__title">Authors</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Submit article" href="/submit-to-cell-press">Submit article</a>
				</li>
				<li>
					<a alt="Multi-Journal Submission" href="/multi-journal-submission">Multi-Journal Submission</a>
				</li>
				<li>
					<a alt="STAR Methods" href="/star-authors-guide">STAR Methods</a>
				</li>
				<li>
					<a alt="Sneak Peek – Preprints" href="/sneakpeek">Sneak Peek – Preprints</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
		<h2 class="group__title">Reviewers</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Reviewers" href="/reviewers">Information for reviewers</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
		<h2 class="group__title">News &amp; events</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Newsroom" href="/newsroom">Newsroom</a>
				</li>
				<li>
					<a alt="Events" href="/events">Events</a>
				</li>
				<li>
					<a alt="Cell Symposia" href="/symposia">Cell Symposia</a>
				</li>
				<li>
					<a alt="Consortia Hub" href="/consortia">Consortia Hub</a>
				</li>
				<li>
					<a alt="Webinars" href="/webinars">Webinars</a>
				</li>
								<li>
					&nbsp;
				</li>
			</ul>
		<h2 class="group__title">Multimedia</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Cell Press Podcast" href="/podcast">Cell Press Podcast</a>
				</li>
				<li>
					<a alt="Cell Press Videos" href="/video">Cell Press Videos</a>
				</li>
				<li>
					<a alt="Color and Comics" href="/comics">Coloring and Comics</a>
				</li>
				<li>
					<a alt="Figure360" href="/video-guidelines">Figure360</a>
				</li>
				<li>
					<a alt="Cell Picture Show" href="/pictureshow">Cell Picture Show</a>
				</li>
				<li>
					<a alt="Research Arc" href="/research-arc">Research Arc</a>
				</li>
			</ul>
		</div>

		<div class="group sided">
			<h2 class="group__title">About</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="About Cell Press" href="/about">About Cell Press</a>
				</li>

				<li>
					<a alt="Open access" href="/open-access">Open access</a>
				</li>
				<li>
					<a alt="COVID Hub" href="/covid-19">COVID Hub</a>
				</li>
				<li>
					<a alt="Sustainability" href="/sustainability">Sustainability hub</a>
				</li>
				<li>
					<a alt="Inclusion and diversity" href="/diversity">Inclusion and diversity</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
			<h2 class="group__title">Contact</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Contact us" href="/contact">Contact us</a>
				</li>
				<li>
					<a alt="Help and Support" href="https://service.elsevier.com/app/overview/cell-press/" target="_blank">Help &amp; Support</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
			<h2 class="group__title">Careers</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Cell Press Careers" href="https://relx.wd3.myworkdayjobs.com/en-US/ElsevierJobs" target="_blank">Cell Press Careers</a>
				</li>
			</ul>
		</div>
	</div>
</div></div>

            

        
</div><div class="col-sm-4 col-md-2 clearfix">
    

        


    

            

            

            <div data-widget-id="beafac41-1cfc-440b-9b55-74aba4a5612c" data-widget-name="UX3HTMLWidget"><div class="groups">
	<div class="group">
		<h2 class="group__title">Access</h2>
		<ul class="rlist clearfix">
			<li>
				<a alt="Subscribe" href="/subscribe">Subscribe</a>
			</li>
			<li>
				<a alt="Claim" href="/activate">Claim</a>
			</li>
			<li>
				<a alt="Read It Now" href="/read-it-now">Read-It-Now</a>
			</li>
			<li>
				<a alt="Recommend to Librarian" href="http://info.cell.com/request-access-to-cell-press-content" target="_blank">Recommend to Librarian</a>
			</li>
			<li>
				<a alt="Publication Alerts" href="/action/showPreferences?menuTab=Alerts">Publication Alerts</a>
			</li>
		</ul>
	</div>
	<div class="group">
		<h2 class="group__title">Collections</h2>
		<ul class="rlist clearfix">
			<li>
				<a alt="Best of Cell Press" href="/bestof">Best of Cell Press</a>
			</li>
			<li>
				<a alt="Cell Press Reviews" href="/reviews">Cell Press Reviews</a>
			</li>
			<li>
				<a alt="Cell Selections" href="/selections">Cell Press Selections</a>
			</li>
			<li>
				<a alt="Nucleus Collections" href="/nucleus">Nucleus Collections</a>
			</li>
			<li>
				<a alt="SnapShots" href="/snapshots">SnapShot Archive</a>
			</li>
		</ul>
	</div>
	<div class="group">
		<h2 class="group__title">Information</h2>
		<ul class="rlist clearfix">
			<li>
				<a alt="For Advertisters" href="/media-kit">For Advertisers</a>
			</li>
			<li>
				<a alt="For Recruiters" target="_blank" href="/pb-assets/Cell/common/CP-RecruitmentKit-US-2018-LowRes-1712070648.pdf">For Recruiters</a>
			</li>
			<li>
				<a alt="For Librarians" target="_blank" href="https://www.elsevier.com/products/sciencedirect/journals/direct-subscription-journals">For Librarians</a>
			</li>
			<li>
				<a alt="Privacy Policy" target="_blank" href="https://www.elsevier.com/legal/privacy-policy">Privacy Policy</a>
			</li>
			<li>
				<a alt="Terms and Conditions" target="_blank" href="https://www.elsevier.com/legal/elsevier-website-terms-and-conditions">Terms and Conditions</a>
			</li>
			<li>
				<a alt="Accessibility" href="/accessibility">Accessibility</a>
			</li>
		</ul>
	</div>
</div></div>

            

        
</div></div>

            
                </div>
            

        

    

        


    

            

            

            <div data-widget-id="779ad11f-7bcd-4a2a-a2cd-8c67b649cbb2" data-widget-name="UX3HTMLWidget"><style>
 
button#cookieBtn {
   text-decoration: none;
   background-color: transparent;
   color: #005789;
   font-weight: bold;
   padding: 0;
   margin: 0;
   font-size: 14px;
}
 
button#cookieBtn:hover {
   text-decoration: underline;
}

</style>
<script>
// HIDE ONETRUST BACKGROUND AFTER USER PRESSES ESCAPE KEY
setTimeout(function(){ 
   $("#onetrust-pc-sdk").on("keydown", function(e) { 
      if (e.keyCode == 27) {  // ESC key
         setTimeout(function(){ 
            $(".onetrust-pc-dark-filter").css("display","none");
         }, 100); 
      }
   });
}, 5000);
</script>

<span class="align-center copy-right">
<hr>
<p align="center"><b>The content on this site is intended for healthcare professionals and researchers across all fields of science.</b></p>
<hr>
<div>
<p>We use cookies to help provide and enhance our service and tailor content. To update your cookie settings, please visit the 
      <b><button id="cookieBtn" class="ot-sdk-show-settings" type="button" aria-label="Manage cookie consent preferences">Cookie Settings</button></b> for this site.<br>

All content on this site: Copyright © 2025 Elsevier Inc., its licensors, and contributors. <br>All rights are reserved, including those for text and data mining, AI training, and similar technologies. <br>For all open access content, the relevant licensing terms apply.
</p>
<ul class="rlist--inline">
			<li>
				<a alt="Privacy Policy" target="_blank" href="https://www.elsevier.com/legal/privacy-policy">Privacy Policy</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Terms and Conditions" target="_blank" href="https://www.elsevier.com/legal/elsevier-website-terms-and-conditions">Terms &amp; Conditions</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Accessibility" href="/accessibility">Accessibility</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Help and Support" target="_blank" href="https://service.elsevier.com/app/overview/cell-press/">Help &amp; Support</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Contact" href="/contact">Contact</a>
			</li>
			</ul>
<p>
<a href="https://www.relx.com"><img src="/pb/assets/raw/Health%20Advance/images/relx-1554393210983.png" alt="RELX" rel="nofollow" target="_new"></a></p>
</div>

</span></div>

            

        

    

        


    

            

            

            <div data-widget-id="c5e179c1-156e-41e3-af7f-5e307efa8acf" data-widget-name="UX3HTMLWidget"><style>#brandingRow{
    background-color:#fff;
    padding:7px 26px 7px 30px;
    color:#007DBC;
    font-size:14px;
    clear:both;
    width:100%
}
#brandingRow a{
    text-decoration:none;
    color:#007DBC
}
#brandingRow a:hover{
    text-decoration:underline
}
#brandingRowRight{
    display:inline;
    float:right;
    width:auto;
    padding:0;
    margin:0
}
svg#cp_logo {
  float: none;
  height: 36px;
  width: 36px;
  margin-left: 0px;
  margin-top: 0px;
}

footer hr {
  display: block;
}

@media screen and (max-width: 600px){
    #brandingRow{
        font-size:12px
    }
}
@media screen and (max-width: 750px){
    #brandingRow{
        padding:7px 18px 7px 16px
    }
    svg#cp_logo{
        margin-right:10px
    }
}
@media screen and (max-width: 800px) {
   a#cpIconLnk {
      display: none;
   }
}

.header__wrapper.clearfix .col-xs-6:first-of-type{
    width:55%
}
.header__wrapper.clearfix .col-xs-6:nth-of-type(2){
    width:45%
}
@media screen and (min-width: 800px) and (max-width: 1000px){
    .header__wrapper.clearfix .col-xs-6:first-of-type{
        width:75%
    }
    .header__wrapper.clearfix .col-xs-6:nth-of-type(2){
        width:25%
    }
}

</style>

<script>
var partnerx = false;
var overrideEndorsement = "";
var tmpj = $("input[name='journalCode']").attr("value");
	
if (window.location.pathname == "/molecular-therapy-family/home") {
	tmpj = "ymthe";
}
	
var ribbonx = "";
if (tmpj == "ajhg" || tmpj == "bpj" || tmpj == "molp" || tmpj == "xplc" || tmpj == "stemcr" || tmpj == "ymthe" || tmpj == "xhgg" || tmpj == "omtn" || tmpj == "omto" || tmpj == "omtm" || tmpj == "biophysreports" ||  tmpj == "bpr" || tmpj == "xinn" || tmpj == "omton") {
	partnerx = true;
	ribbonx = "<a href='https://www.cell.com' data-aa-name='cell press endorsement' data-aa-region='cp global header' title='Visit Cell.com'>A Cell Press partner journal</a>";
} else if (tmpj != "" && tmpj != "undefined" && tmpj != undefined) {
	ribbonx = "<a href='/' data-aa-name='cell press endorsement' data-aa-region='cp global header' title='Visit Cell.com'>A Cell Press journal</a>";
} else {
	ribbonx = "<a href='/' data-aa-name='cell press endorsement' data-aa-region='cp global header' title='Visit Cell.com'>A Cell Press resource</a>";
}
	
if (window.location.pathname == "/") {
	ribbonx = "";
}
if (overrideEndorsement != "") {
	ribbonx = overrideEndorsement;
}

$(".header__wrapper.clearfix").prepend("<div id='brandingRow'><a href='/' data-aa-name='science that inspires' data-aa-region='cp global header' title='Visit Cell.com'>Science that inspires</a><p id='brandingRowRight'>" + ribbonx + "</p></div>");
	
// handle scrolling effect on article pages
if ($("article").length > 0) {
	var scrollx;
	$(window).scroll(function() {
		scrollx = $(window).scrollTop();
		if (scrollx > 0) { 
			$("#brandingRow").css("display","none"); 
		} else {
			$("#brandingRow").css("display","block");
		}
	});
}

if (window.location.pathname == "/" || partnerx == false) {
               $("<a href='/home' id='cpIconLnk' data-aa-name='cell press logo' data-aa-region='cp global header' title='Cell Press' target='_blank'><svg id='cp_logo' version='1.1' xmlns='http://www.w3.org/2000/svg' xmlns:xlink='http://www.w3.org/1999/xlink' x='0px' y='0px' viewBox='0 0 100 98.1' style='enable-background:new 0 0 100 98.1;' xml:space='preserve' height='100' width='98.1'><path style='fill:#FFFFFF;' d='M69.09,0.09c-4.01-0.6-7.83,1.94-8.78,5.89l-3.14,13.11c-0.76,3.19-3.44,5.56-6.7,5.93l-18.6,2.13          C11.83,29.31-2.53,47.66,0.37,67.78c2.3,15.96,15.18,27.77,30.54,30.08c4.01,0.6,7.83-1.94,8.78-5.89l3.14-13.11 c0.76-3.19,3.44-5.56,6.69-5.93l18.6-2.13c20.04-2.17,34.41-20.52,31.5-40.65C97.32,14.21,84.45,2.39,69.09,0.09z M65.06,47.2  l-11.75,1.4c-1.98,0.21-3.62,1.65-4.08,3.59l-3.43,14.3c-1.01,4.22-4.5,7.39-8.8,8c-6.66,0.72-12.63-4.2-13.15-10.92                c-0.51-6.5,4.6-12.11,11.08-12.81l11.75-1.39c1.98-0.22,3.62-1.65,4.08-3.59l3.43-14.3c1.01-4.22,4.5-7.39,8.8-8          c6.66-0.72,12.63,4.19,13.15,10.92C76.66,40.89,71.55,46.5,65.06,47.2z'/></svg></a>").insertBefore(".header__wrapper.clearfix .journal-logos:eq(0)");
}


</script></div>

            

        

    

        


    

            

            

            <div data-widget-id="1af759b7-9bb6-4413-ada9-51a8333e8fb8" data-widget-name="UX3HTMLWidget"><style>
    /*This was added temporarily until the PB changelog tab issue got fixed. [LIT-444969 - c#4]*/
    [data-tab-id="changelogTab"] {
        display: none !important;
    }
    
    /*This was added temporarily until the PB changelog tab issue got fixed. [LIT-518839]*/
    @media (min-width: 769px) {
        .ux3 .quick-search .dropBlock [data-db-target-of],
        .quick-search .dropBlock [data-db-target-of] {
            width: 100vw !important;
        }
    }
</style></div>

            

        
</div></div>

            

        

        </div>
    

    

        


    

            

            

            <div data-widget-id="d7e0813b-a300-497a-ab37-9c3ddaf03e71" data-widget-name="UX3HTMLWidget"><!--
<div class="session-reminder hidden">
    <div class="session-reminder__overlay">
        <div class="session-reminder__dialog">
            <div class="session-reminder__dialog-inner">
                <div class="dialog-header" role="dialog" aria-labelledby="dialog-title" aria-describedby="dialog-desc">
                    <h2 class="dialog-header-title" id="dialog-title">
                        <span>Session Timeout</span>
                        <div class="dialog-timer">(2:00)</div>
                    </h2>
                    <button class="dialog-close" type="button" aria-label="close dialog">
                        <i class="icon-tools_close" aria-hidden="true"></i>
                    </button>
                </div>
                <div class="dialog-body">
                    <p class="dialog-desc" id="dialog-desc">Your session will expire shortly. If you are still working, click the ‘Keep Me Logged In’ button below. If you do not respond within the next minute, you will be automatically logged out.</p>
                    <p class="dialog-expired sr-only" aria-live="polite"></p>
                </div>
                <div class="dialog-footer">
                    <button class="session-reset" type="button">Keep Me Logged In</button>
                </div>
            </div>
        </div>
    </div>
</div>
<a class="sr-only" href="javascript:void(0)" aria-hidden="true"></a>
--></div>

            

        
        </div>
    </div>


































    



















        
        
            
                <script src="/products/marlin/cell/releasedAssets/js/main.bundle-90475670808d5d3eee41.js"></script>
            
        
        
        

    




<script type="text/javascript">
    var pageData =  {"journal":{"specialty":"Cell Press","issn":"0092-8674","name":"Cell"},"page":{"loadTimestamp":"1749592091384","server":"cell","environment":"prod","businessUnit":"ELS:RAP:JBS","noTracking":"","name":"fulltext:html","section":"Cell","language":"en","type":"CP-CA","productName":"JB"},"visitor":{"departmentName":"|ae:ScienceDirect Guests","accessType":"ae:ANON_GUEST|ae:ANON_GUEST","accountId":"ae:838949|ae:228598","accountName":"ae:Cell Guest Account|ae:ScienceDirect Guests","departmentId":"ae:977212|ae:291352","ipAddress":"189.120.74.179","loginStatus":"anonymous","userId":"ae:246228266|ae:12975512"},"content":[{"accessType":"article:free:openaccess","volumeNumber":"184","issueNumber":"25","viewState":"fulltext","format":"MIME-HTML","id":"pii:S0092867421013325","coverDate":"202112","type":"XOCS-JOURNAL:SCOPE-FULL","title":"Microenvironment drives cell state plasticity and drug response in pancreatic cancer","status":"S300"}]};
</script>

<script type="text/javascript">
    if(typeof pageDataTracker !== "undefined") {
        pageDataTracker.trackPageLoad();
    }
</script>
<script>(function(){function c(){var b=a.contentDocument||a.contentWindow.document;if(b){var d=b.createElement('script');d.innerHTML="window.__CF$cv$params={r:'94dc124409e24ee9',t:'MTc0OTU5MjA5MC4wMDAwMDA='};var a=document.createElement('script');a.nonce='';a.src='/cdn-cgi/challenge-platform/scripts/jsd/main.js';document.getElementsByTagName('head')[0].appendChild(a);";b.getElementsByTagName('head')[0].appendChild(d)}}if(document.body){var a=document.createElement('iframe');a.height=1;a.width=1;a.style.position='absolute';a.style.top=0;a.style.left=0;a.style.border='none';a.style.visibility='hidden';document.body.appendChild(a);if('loading'!==document.readyState)c();else if(window.addEventListener)document.addEventListener('DOMContentLoaded',c);else{var e=document.onreadystatechange||function(){};document.onreadystatechange=function(b){e(b);'loading'!==document.readyState&&(document.onreadystatechange=e,c())}}}})();</script><iframe height="1" width="1" style="position: absolute; top: 0px; left: 0px; border: none; visibility: hidden;"></iframe><script defer="" src="https://static.cloudflareinsights.com/beacon.min.js/vcd15cbe7772f49c399c6a5babf22c1241717689176015" integrity="sha512-ZpsOmlRQV6y907TI0dKBHq9Md29nnaEIPlkf84rnaERnq6zvWvPUqr2ft8M1aS28oN72PdrCzSjY4U6VaAw1EQ==" data-cf-beacon="{&quot;rayId&quot;:&quot;94dc124409e24ee9&quot;,&quot;serverTiming&quot;:{&quot;name&quot;:{&quot;cfExtPri&quot;:true,&quot;cfEdge&quot;:true,&quot;cfOrigin&quot;:true,&quot;cfL4&quot;:true,&quot;cfSpeedBrain&quot;:true,&quot;cfCacheStatus&quot;:true}},&quot;version&quot;:&quot;2025.5.0&quot;,&quot;token&quot;:&quot;57a02315956f4b2e82817b3dfa99b1a2&quot;}" crossorigin="anonymous"></script>


<div id="addtoany" style="position: static;"><div class="a2a_overlay" id="a2a_overlay"></div><div id="a2a_modal" class="a2a_modal a2a_hide" role="dialog" tabindex="-1" aria-label=""><div class="a2a_modal_body a2a_menu a2a_hide" id="a2a_copy_link"><label for="a2a_copy_link_text" id="a2a_copy_link_icon" class="a2a_svg a2a_s_link a2a_logo_color"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"><title>Copy link</title><path fill="#FFF" d="M7.591 21.177c0-.36.126-.665.377-.917l2.804-2.804a1.235 1.235 0 0 1 .913-.378c.377 0 .7.144.97.43-.026.028-.11.11-.255.25-.144.14-.24.236-.29.29a2.82 2.82 0 0 0-.2.256 1.056 1.056 0 0 0-.177.344 1.43 1.43 0 0 0-.046.37c0 .36.126.666.377.918a1.25 1.25 0 0 0 .918.377c.126.001.251-.015.373-.047.125-.037.242-.096.345-.175.09-.06.176-.127.256-.2.1-.094.196-.19.29-.29.14-.142.223-.23.25-.254.297.28.445.607.445.984 0 .36-.126.664-.377.916l-2.778 2.79a1.242 1.242 0 0 1-.917.364c-.36 0-.665-.118-.917-.35l-1.982-1.97a1.223 1.223 0 0 1-.378-.9l-.001-.004Zm9.477-9.504c0-.36.126-.665.377-.917l2.777-2.79a1.235 1.235 0 0 1 .913-.378c.35 0 .656.12.917.364l1.984 1.968c.254.252.38.553.38.903 0 .36-.126.665-.38.917l-2.802 2.804a1.238 1.238 0 0 1-.916.364c-.377 0-.7-.14-.97-.418.026-.027.11-.11.255-.25a7.5 7.5 0 0 0 .29-.29c.072-.08.139-.166.2-.255.08-.103.14-.22.176-.344.032-.12.048-.245.047-.37 0-.36-.126-.662-.377-.914a1.247 1.247 0 0 0-.917-.377c-.136 0-.26.015-.37.046-.114.03-.23.09-.346.175a3.868 3.868 0 0 0-.256.2c-.054.05-.15.148-.29.29-.14.146-.222.23-.25.258-.294-.278-.442-.606-.442-.983v-.003ZM5.003 21.177c0 1.078.382 1.99 1.146 2.736l1.982 1.968c.745.75 1.658 1.12 2.736 1.12 1.087 0 2.004-.38 2.75-1.143l2.777-2.79c.75-.747 1.12-1.66 1.12-2.737 0-1.106-.392-2.046-1.183-2.818l1.186-1.185c.774.79 1.708 1.186 2.805 1.186 1.078 0 1.995-.376 2.75-1.13l2.803-2.81c.751-.754 1.128-1.671 1.128-2.748 0-1.08-.382-1.993-1.146-2.738L23.875 6.12C23.13 5.372 22.218 5 21.139 5c-1.087 0-2.004.382-2.75 1.146l-2.777 2.79c-.75.747-1.12 1.66-1.12 2.737 0 1.105.392 2.045 1.183 2.817l-1.186 1.186c-.774-.79-1.708-1.186-2.805-1.186-1.078 0-1.995.377-2.75 1.132L6.13 18.426c-.754.755-1.13 1.672-1.13 2.75l.003.001Z"></path></svg></label><input id="a2a_copy_link_text" type="text" title="Copy link" readonly=""><div id="a2a_copy_link_copied">✓</div></div><div class="a2a_modal_body a2a_menu a2a_thanks a2a_hide" id="a2a_thanks"><div class="a2a_localize" data-a2a-localize="inner,ThanksForSharing">Thanks for sharing!</div></div></div><div class="a2a_menu a2a_full a2a_localize" id="a2apage_full" role="dialog" tabindex="-1" aria-label="Share" data-a2a-localize="title,Share"><div class="a2a_full_header"><div id="a2apage_find_container" class="a2a_menu_find_container"><label for="a2apage_find" id="a2apage_find_icon" class="a2a_svg a2a_s_find"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" aria-hidden="true"><title>Find any service</title><path fill="#CCC" d="M19.7 18.2l-4.5-4.5c.7-1.1 1.2-2.3 1.2-3.6 0-3.5-2.8-6.3-6.3-6.3s-6.3 2.8-6.3 6.3 2.8 6.3 6.3 6.3c1.4 0 2.6-.4 3.6-1.2l4.5 4.5c.6.6 1.3.7 1.7.2.5-.4.4-1.1-.2-1.7zm-9.6-3.6c-2.5 0-4.5-2.1-4.5-4.5 0-2.5 2.1-4.5 4.5-4.5 2.5 0 4.5 2.1 4.5 4.5s-2 4.5-4.5 4.5z"></path></svg></label><input id="a2apage_find" class="a2a_menu_find a2a_localize" type="text" autocomplete="off" title="Find any service" data-a2a-localize="title,FindAnyServiceToAddTo"></div></div><div class="a2a_full_services" id="a2apage_full_services" role="presentation"></div><div class="a2a_full_footer"><a href="https://www.addtoany.com" title="Share Buttons" rel="noopener" target="_blank"><span class="a2a_svg a2a_s__default a2a_s_a2a a2a_logo_color"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"><g fill="#FFF"><path d="M14 7h4v18h-4z"></path><path d="M7 14h18v4H7z"></path></g></svg></span>AddToAny</a></div></div><div id="a2apage_dropdown" class="a2a_menu a2a_mini a2a_localize a2a_hide" tabindex="-1" aria-label="Share" data-a2a-localize="label,Share"><div class="a2a_mini_services" id="a2apage_mini_services"></div><div id="a2apage_cols_container" class="a2a_cols_container"><div class="a2a_col1" id="a2apage_col1"></div><div id="a2apage_2_col1" class="a2a_hide"></div><div class="a2a_clear"></div></div><div class="a2apage_wide a2a_wide"><a href="#addtoany" id="a2apage_show_more_less" class="a2a_more a2a_localize" title="Show all" data-a2a-localize="title,ShowAll"><span class="a2a_svg a2a_s__default a2a_s_a2a a2a_logo_color"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"><g fill="#FFF"><path d="M14 7h4v18h-4z"></path><path d="M7 14h18v4H7z"></path></g></svg></span><span class="a2a_localize" data-a2a-localize="inner,More">More…</span></a></div></div><div style="height: 1px; width: 1px; position: absolute; z-index: 100000; top: 0px; visibility: hidden;"><iframe id="a2a_sm_ifr" title="AddToAny Utility Frame" aria-hidden="true" src="https://static.addtoany.com/menu/sm.25.html#type=core&amp;event=load" style="height: 1px; width: 1px; border: 0px; left: 0px; top: 0px; position: absolute; z-index: 100000; display: none;"></iframe></div></div><div role="status" aria-live="assertive" aria-relevant="additions" class="ui-helper-hidden-accessible"></div><div id="crossmark-widget" style="display: none;"><div class="crossmark-reset crossmark-overlay"></div><div class="crossmark-reset crossmark-popup"><div class="crossmark-reset crossmark-popup__offset"><div class="crossmark-reset crossmark-popup__inner"><div class="crossmark-reset crossmark-popup__header"><button class="crossmark-reset crossmark-popup__btn-close"><span class="sr-only">__("articleCrossmark.closePopup")</span></button></div><div class="crossmark-reset crossmark-popup__content-wrapper"><iframe class="crossmark-reset crossmark-popup__content" title="Crossmark service - Intentionally blank" aria-hidden="false" tabindex="0" src="https://crossmark.crossref.org/dialog?doi=10.1016%2Fj.cell.2021.11.017&amp;domain=www.cell.com&amp;uri_scheme=https%3A&amp;cm_version=v2.0"></iframe></div></div></div></div></div><div id="onetrust-consent-sdk" data-nosnippet="true"><div class="onetrust-pc-dark-filter ot-fade-in" style="z-index:2147483645;"></div><div id="onetrust-banner-sdk" class="otFlat bottom ot-wo-title ot-buttons-fw" tabindex="0" role="region" aria-label="Cookie banner"><div role="dialog" aria-modal="true" aria-label="Privacy"><div class="ot-sdk-container"><div class="ot-sdk-row"><div id="onetrust-group-container" class="ot-sdk-eight ot-sdk-columns"><div class="banner_logo"></div><div id="onetrust-policy"><div id="onetrust-policy-text">We use cookies that are necessary to make our site work. We may also use additional cookies to analyze, improve, and personalize our content and your digital experience. For more information, see our<a class="ot-cookie-policy-link" href="https://www.elsevier.com/legal/cookienotice" target="_blank" aria-label=", opens in a new tab" rel="noopener">Cookie Policy</a></div></div></div><div id="onetrust-button-group-parent" class="ot-sdk-three ot-sdk-columns"><div id="onetrust-button-group"><button id="onetrust-pc-btn-handler" class="cookie-setting-link">Cookie Settings</button>  <button id="onetrust-accept-btn-handler">Accept all cookies</button></div></div></div></div><!-- Close Button --><div id="onetrust-close-btn-container"></div><!-- Close Button END--></div></div><div id="onetrust-pc-sdk" class="otPcCenter ot-hide ot-fade-in" lang="en" aria-label="Preference center" role="region"><div role="dialog" aria-modal="true" style="height: 100%;" aria-label="Cookie Preference Center"><!-- Close Button --><div class="ot-pc-header"><!-- Logo Tag --><div class="ot-pc-logo" role="img" aria-label="Company Logo"><img alt="Company Logo" src="https://cdn.cookielaw.org/logos/static/ot_company_logo.png"></div></div><!-- Close Button --><div id="ot-pc-content" class="ot-pc-scrollbar"><div class="ot-optout-signal ot-hide"><div class="ot-optout-icon"><svg xmlns="http://www.w3.org/2000/svg"><path class="ot-floating-button__svg-fill" d="M14.588 0l.445.328c1.807 1.303 3.961 2.533 6.461 3.688 2.015.93 4.576 1.746 7.682 2.446 0 14.178-4.73 24.133-14.19 29.864l-.398.236C4.863 30.87 0 20.837 0 6.462c3.107-.7 5.668-1.516 7.682-2.446 2.709-1.251 5.01-2.59 6.906-4.016zm5.87 13.88a.75.75 0 00-.974.159l-5.475 6.625-3.005-2.997-.077-.067a.75.75 0 00-.983 1.13l4.172 4.16 6.525-7.895.06-.083a.75.75 0 00-.16-.973z" fill="#FFF" fill-rule="evenodd"></path></svg></div><span></span></div><h2 id="ot-pc-title">Cookie Preference Center</h2><div id="ot-pc-desc">We use cookies which are necessary to make our site work. We may also use additional cookies to analyse, improve and personalise our content and your digital experience. For more information, see our <a href="https://www.elsevier.com/legal/cookienotice/_nocache" target="_blank">Cookie Policy</a> and the list of <a href="https://support.google.com/admanager/answer/9012903" target="_blank">Google Ad-Tech Vendors</a>.
<br>
<br>
You may choose not to allow some types of cookies. However, blocking some types may impact your experience of our site and the services we are able to offer. See the different category headings below to find out more or change your settings.
<br>
</div><button id="accept-recommended-btn-handler">Allow all</button><section class="ot-sdk-row ot-cat-grp"><h3 id="ot-category-title"> Manage Consent Preferences</h3><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="1"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-1" aria-labelledby="ot-header-id-1 ot-status-id-1"></button><!-- Accordion header --><div class="ot-acc-hdr ot-always-active-group"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-1">Strictly Necessary Cookies</h4><div id="ot-status-id-1" class="ot-always-active">Always active</div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-1">These cookies are necessary for the website to function and cannot be switched off in our systems. They are usually only set in response to actions made by you which amount to a request for services, such as setting your privacy preferences, logging in or filling in forms. You can set your browser to block or alert you about these cookies, but some parts of the site will not then work. These cookies do not store any personally identifiable information.
<br><br></p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="1">Cookie Details List‎</button></div></div></div><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="3"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-3" aria-labelledby="ot-header-id-3"></button><!-- Accordion header --><div class="ot-acc-hdr"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-3">Functional Cookies</h4><div class="ot-tgl"><input type="checkbox" name="ot-group-id-3" id="ot-group-id-3" role="switch" class="category-switch-handler" data-optanongroupid="3" aria-labelledby="ot-header-id-3"> <label class="ot-switch" for="ot-group-id-3"><span class="ot-switch-nob"></span> <span class="ot-label-txt">Functional Cookies</span></label> </div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-3">These cookies enable the website to provide enhanced functionality and personalisation. They may be set by us or by third party providers whose services we have added to our pages. If you do not allow these cookies then some or all of these services may not function properly.</p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="3">Cookie Details List‎</button></div></div></div><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="2"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-2" aria-labelledby="ot-header-id-2"></button><!-- Accordion header --><div class="ot-acc-hdr"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-2">Performance Cookies</h4><div class="ot-tgl"><input type="checkbox" name="ot-group-id-2" id="ot-group-id-2" role="switch" class="category-switch-handler" data-optanongroupid="2" aria-labelledby="ot-header-id-2"> <label class="ot-switch" for="ot-group-id-2"><span class="ot-switch-nob"></span> <span class="ot-label-txt">Performance Cookies</span></label> </div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-2">These cookies allow us to count visits and traffic sources so we can measure and improve the performance of our site. They help us to know which pages are the most and least popular and see how visitors move around the site.</p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="2">Cookie Details List‎</button></div></div></div><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="4"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-4" aria-labelledby="ot-header-id-4"></button><!-- Accordion header --><div class="ot-acc-hdr"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-4">Targeting Cookies</h4><div class="ot-tgl"><input type="checkbox" name="ot-group-id-4" id="ot-group-id-4" role="switch" class="category-switch-handler" data-optanongroupid="4" aria-labelledby="ot-header-id-4"> <label class="ot-switch" for="ot-group-id-4"><span class="ot-switch-nob"></span> <span class="ot-label-txt">Targeting Cookies</span></label> </div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-4">These cookies may be set through our site by our advertising partners. They may be used by those companies to build a profile of your interests and show you relevant adverts on other sites. If you do not allow these cookies, you will experience less targeted advertising.</p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="4">Cookie Details List‎</button></div></div></div><!-- Groups sections starts --><!-- Group section ends --><!-- Accordion Group section starts --><!-- Accordion Group section ends --></section></div><section id="ot-pc-lst" class="ot-hide ot-hosts-ui ot-pc-scrollbar"><div id="ot-pc-hdr"><div id="ot-lst-title"><button class="ot-link-btn back-btn-handler" aria-label="Back"><svg id="ot-back-arw" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 444.531 444.531" xml:space="preserve"><title>Back Button</title><g><path fill="#656565" d="M213.13,222.409L351.88,83.653c7.05-7.043,10.567-15.657,10.567-25.841c0-10.183-3.518-18.793-10.567-25.835
                    l-21.409-21.416C323.432,3.521,314.817,0,304.637,0s-18.791,3.521-25.841,10.561L92.649,196.425
                    c-7.044,7.043-10.566,15.656-10.566,25.841s3.521,18.791,10.566,25.837l186.146,185.864c7.05,7.043,15.66,10.564,25.841,10.564
                    s18.795-3.521,25.834-10.564l21.409-21.412c7.05-7.039,10.567-15.604,10.567-25.697c0-10.085-3.518-18.746-10.567-25.978
                    L213.13,222.409z"></path></g></svg></button><h3>Cookie List</h3></div><div class="ot-lst-subhdr"><div class="ot-search-cntr"><p role="status" class="ot-scrn-rdr"></p><input id="vendor-search-handler" type="text" name="vendor-search-handler" placeholder="Search…" aria-label="Cookie list search"> <svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 -30 110 110" aria-hidden="true"><title>Search Icon</title><path fill="#2e3644" d="M55.146,51.887L41.588,37.786c3.486-4.144,5.396-9.358,5.396-14.786c0-12.682-10.318-23-23-23s-23,10.318-23,23
            s10.318,23,23,23c4.761,0,9.298-1.436,13.177-4.162l13.661,14.208c0.571,0.593,1.339,0.92,2.162,0.92
            c0.779,0,1.518-0.297,2.079-0.837C56.255,54.982,56.293,53.08,55.146,51.887z M23.984,6c9.374,0,17,7.626,17,17s-7.626,17-17,17
            s-17-7.626-17-17S14.61,6,23.984,6z"></path></svg></div><div class="ot-fltr-cntr"><button id="filter-btn-handler" aria-label="Filter" aria-haspopup="true"><svg role="presentation" aria-hidden="true" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 402.577 402.577" xml:space="preserve"><title>Filter Icon</title><g><path fill="#fff" d="M400.858,11.427c-3.241-7.421-8.85-11.132-16.854-11.136H18.564c-7.993,0-13.61,3.715-16.846,11.136
      c-3.234,7.801-1.903,14.467,3.999,19.985l140.757,140.753v138.755c0,4.955,1.809,9.232,5.424,12.854l73.085,73.083
      c3.429,3.614,7.71,5.428,12.851,5.428c2.282,0,4.66-0.479,7.135-1.43c7.426-3.238,11.14-8.851,11.14-16.845V172.166L396.861,31.413
      C402.765,25.895,404.093,19.231,400.858,11.427z"></path></g></svg></button></div><div id="ot-anchor"></div><section id="ot-fltr-modal"><div id="ot-fltr-cnt"><button id="clear-filters-handler">Clear</button><div class="ot-fltr-scrlcnt ot-pc-scrollbar"><div class="ot-fltr-opts" role="list"><div class="ot-fltr-opt" role="listitem"><div class="ot-chkbox"><input id="chkbox-id" type="checkbox" class="category-filter-handler"> <label for="chkbox-id"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div></div></div><div class="ot-fltr-btns"><button id="filter-apply-handler">Apply</button> <button id="filter-cancel-handler">Cancel</button></div></div></div></section></div></div><section id="ot-lst-cnt" class="ot-host-cnt ot-pc-scrollbar"><div id="ot-sel-blk"><div class="ot-sel-all"><div class="ot-sel-all-hdr"><span class="ot-consent-hdr">Consent</span> <span class="ot-li-hdr">Leg.Interest</span></div><div class="ot-sel-all-chkbox"><div class="ot-chkbox" id="ot-selall-hostcntr"><input id="select-all-hosts-groups-handler" type="checkbox"> <label for="select-all-hosts-groups-handler"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div><div class="ot-chkbox" id="ot-selall-vencntr"><input id="select-all-vendor-groups-handler" type="checkbox"> <label for="select-all-vendor-groups-handler"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div><div class="ot-chkbox" id="ot-selall-licntr"><input id="select-all-vendor-leg-handler" type="checkbox"> <label for="select-all-vendor-leg-handler"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div></div></div></div><div class="ot-sdk-row"><div class="ot-sdk-column"><ul id="ot-host-lst"></ul></div></div></section></section><div class="ot-pc-footer ot-pc-scrollbar"><div class="ot-btn-container"> <button class="save-preference-btn-handler onetrust-close-btn-handler">Confirm my choices</button></div><!-- Footer logo --><div class="ot-pc-footer-logo"><a href="https://www.onetrust.com/products/cookie-consent/" target="_blank" rel="noopener noreferrer" aria-label="Powered by OneTrust Opens in a new Tab"><img alt="Powered by Onetrust" src="https://cdn.cookielaw.org/logos/static/powered_by_logo.svg" title="Powered by OneTrust Opens in a new Tab"></a></div></div><!-- Cookie subgroup container --><!-- Vendor list link --><!-- Cookie lost link --><!-- Toggle HTML element --><!-- Checkbox HTML --><!-- plus minus--><!-- Arrow SVG element --><!-- Accordion basic element --><span class="ot-scrn-rdr" aria-atomic="true" aria-live="polite"></span><!-- Vendor Service container and item template --></div><iframe class="ot-text-resize" sandbox="allow-same-origin" title="onetrust-text-resize" style="position: absolute; top: -50000px; width: 100em;" aria-hidden="true"></iframe></div></div></body></html>