<html lang="en" class="pb-page fonts-loaded fonts-loaded" data-request-id="c8a8e42a-3b60-4afb-89ca-e0f1d1f02bd9" style="--article-sticky-top: 82.00019999999999px; --scroll-padding: 163.594px auto auto;"><head data-pb-dropzone="head"><meta name="pbContext" content=";product:product:elsevier\:product\:cell;website:website:cell-site;ctype:string:Journal Content;journal:journal:cell;pageGroup:string:Article View;article:article:pii\:S0092867420306826;page:string:Show Full Text;requestedJournal:journal:cell;issue:issue:pii\:S0092867419X00161;wgroup:string:Default Website Group;subPage:string:Full Text">




<!-- if there is any customization for Responsive Project widget  -->


























    
    







    
            
        



        
                <meta name="robots" content="noarchive,nocache">
            
    






    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=1, user-scalable=1">











        <meta property="og:title" content="TBL1XR1 Mutations Drive Extranodal Lymphoma by Inducing a Pro-tumorigenic Memory Fate">
    


        <meta property="og:type" content="Article">
    


        <meta property="og:url" content="https://www.cell.com/cell/abstract/S0092-8674(20)30682-6">
    



        <meta property="og:site_name" content="Cell">
    


        <meta property="og:image" content="https://www.cell.com/cms/asset/7ed32941-8046-46d8-95ad-71914133ef00/fx1.jpg">
    


        <meta property="og:description" content="A subset of B cell lymphomas is driven by mutations that impair plasma cell differentiation
and instead bias cell fate toward immature memory B cells, which preferentially re-enter
germinal center reactions to drive lymphomagenesis.">
    











        <meta name="twitter:title" content="TBL1XR1 Mutations Drive Extranodal Lymphoma by Inducing a Pro-tumorigenic Memory Fate">
    




        <meta name="twitter:image" content="https://www.cell.com/cms/asset/7ed32941-8046-46d8-95ad-71914133ef00/fx1.jpg">
    


        <meta name="twitter:description" content="A subset of B cell lymphomas is driven by mutations that impair plasma cell differentiation
and instead bias cell fate toward immature memory B cells, which preferentially re-enter
germinal center reactions to drive lymphomagenesis.">
    

        <meta name="twitter:card" content="summary_large_image">
    











    
    
            
            
        <link rel="canonical" href="https://www.cell.com/cell/fulltext/S0092-8674(20)30682-6">
    


<meta name="tdm-reservation" content="1">
<meta name="tdm-policy" content="https://www.elsevier.com/tdm/tdmrep-policy.json">

<title>TBL1XR1 Mutations Drive Extranodal Lymphoma by Inducing a Pro-tumorigenic Memory Fate: Cell</title><meta name="citation_journal_title" content="Cell"> <meta name="citation_journal_abbrev" content="Cell"> <meta name="citation_publisher" content="Elsevier"> <meta name="citation_title" content="TBL1XR1 Mutations Drive Extranodal Lymphoma by Inducing a Pro-tumorigenic Memory Fate"> <meta name="citation_date" content="2020/07/23"> <meta name="citation_online_date" content="2020/07/02"> <meta name="citation_volume" content="182"> <meta name="citation_issue" content="2"> <meta name="citation_firstpage" content="297"> <meta name="citation_lastpage" content="316.e27"> <meta name="citation_pmid" content="32619424"> <meta name="citation_abstract_html_url" content="https://www.cell.com/action/showAbstract?pii=S0092867420306826"> <meta name="citation_fulltext_html_url" content="https://www.cell.com/action/showFullText?pii=S0092867420306826"> <meta name="citation_pdf_url" content="https://www.cell.com/action/showPdf?pii=S0092867420306826"> <meta name="citation_issn" content="0092-8674"> <meta name="citation_issn" content="1097-4172"> <meta name="citation_language" content="English"> <meta name="citation_doi" content="10.1016/j.cell.2020.05.049"> <meta name="citation_pmid" content="32619424"> <meta name="citation_author" content="Leandro Venturutti"><meta name="citation_author_institution" content="Division of Hematology/Oncology, Department of Medicine, Weill Cornell Medicine, Cornell University, New York, NY 10021, USA"><meta name="citation_author" content="Matt Teater"><meta name="citation_author_institution" content="Division of Hematology/Oncology, Department of Medicine, Weill Cornell Medicine, Cornell University, New York, NY 10021, USA"><meta name="citation_author" content="Andrew Zhai"><meta name="citation_author_institution" content="Department of Biochemistry, University of Toronto, Toronto, ON M5S 1A8, Canada"><meta name="citation_author" content="Amy Chadburn"><meta name="citation_author_institution" content="Department of Pathology and Laboratory Medicine, Weill Cornell Medicine, New York, NY 10065, USA"><meta name="citation_author" content="Leena Babiker"><meta name="citation_author_institution" content="Division of Hematology/Oncology, Department of Medicine, Weill Cornell Medicine, Cornell University, New York, NY 10021, USA"><meta name="citation_author" content="Daleum Kim"><meta name="citation_author_institution" content="Division of Hematology/Oncology, Department of Medicine, Weill Cornell Medicine, Cornell University, New York, NY 10021, USA"><meta name="citation_author" content="Wendy Béguelin"><meta name="citation_author_institution" content="Division of Hematology/Oncology, Department of Medicine, Weill Cornell Medicine, Cornell University, New York, NY 10021, USA"><meta name="citation_author" content="Tak C. Lee"><meta name="citation_author_institution" content="Division of Hematology/Oncology, Department of Medicine, Weill Cornell Medicine, Cornell University, New York, NY 10021, USA"><meta name="citation_author" content="Youngjun Kim"><meta name="citation_author_institution" content="Immunology and Microbial Pathogenesis Program, Weill Cornell Graduate School of Medical Sciences, New York, NY 10065, USA"><meta name="citation_author" content="Christopher R. Chin"><meta name="citation_author_institution" content="Division of Hematology/Oncology, Department of Medicine, Weill Cornell Medicine, Cornell University, New York, NY 10021, USA"><meta name="citation_author_institution" content="Tri-Institutional Program in Computational Biology and Medicine, New York, NY 10065, USA"><meta name="citation_author" content="William T. Yewdell"><meta name="citation_author_institution" content="Immunology Program, Memorial Sloan Kettering Cancer Center, New York, NY 10065, USA"><meta name="citation_author" content="Brian Raught"><meta name="citation_author_institution" content="Princess Margaret Cancer Centre, University of Toronto, Toronto, ON M5G 1L7, Canada"><meta name="citation_author" content="Jude M. Phillip"><meta name="citation_author_institution" content="Division of Hematology/Oncology, Department of Medicine, Weill Cornell Medicine, Cornell University, New York, NY 10021, USA"><meta name="citation_author" content="Yanwen Jiang"><meta name="citation_author_institution" content="Division of Hematology/Oncology, Department of Medicine, Weill Cornell Medicine, Cornell University, New York, NY 10021, USA"><meta name="citation_author" content="Louis M. Staudt"><meta name="citation_author_institution" content="Center for Cancer Genomics, National Cancer Institute, Bethesda, MD 20892, USA"><meta name="citation_author" content="Michael R. Green"><meta name="citation_author_institution" content="Department of Lymphoma and Myeloma, University of Texas MD Anderson Cancer Center, Houston, TX 77030, USA"><meta name="citation_author_institution" content="Department of Genomic Medicine, University of Texas MD Anderson Cancer Center, Houston, TX 77030, USA"><meta name="citation_author" content="Jayanta Chaudhuri"><meta name="citation_author_institution" content="Immunology and Microbial Pathogenesis Program, Weill Cornell Graduate School of Medical Sciences, New York, NY 10065, USA"><meta name="citation_author_institution" content="Immunology Program, Memorial Sloan Kettering Cancer Center, New York, NY 10065, USA"><meta name="citation_author_institution" content="Gerstner Sloan Kettering Graduate School of Biomedical Sciences, New York, NY 10065, USA"><meta name="citation_author" content="Olivier Elemento"><meta name="citation_author_institution" content="Caryl and Israel Englander Institute for Precision Medicine, Weill Cornell Medicine, New York, NY 10021, USA"><meta name="citation_author" content="Pedro Farinha"><meta name="citation_author_institution" content="Centre for Lymphoid Cancer, BC Cancer Agency, Vancouver, BC V5Z1L3, Canada"><meta name="citation_author" content="Andrew P. Weng"><meta name="citation_author_institution" content="Terry Fox Laboratory, BC Cancer Agency, Vancouver, BC V5Z1L3, Canada"><meta name="citation_author_institution" content="Department of Pathology and Lab Medicine, BC Cancer Agency, Vancouver, BC V5Z1L3, Canada"><meta name="citation_author" content="Michael D. Nissen"><meta name="citation_author_institution" content="Terry Fox Laboratory, BC Cancer Agency, Vancouver, BC V5Z1L3, Canada"><meta name="citation_author" content="Christian Steidl"><meta name="citation_author_institution" content="Centre for Lymphoid Cancer, BC Cancer Agency, Vancouver, BC V5Z1L3, Canada"><meta name="citation_author" content="Ryan D. Morin"><meta name="citation_author_institution" content="Centre for Lymphoid Cancer, BC Cancer Agency, Vancouver, BC V5Z1L3, Canada"><meta name="citation_author" content="David W. Scott"><meta name="citation_author_institution" content="Centre for Lymphoid Cancer, BC Cancer Agency, Vancouver, BC V5Z1L3, Canada"><meta name="citation_author" content="Gilbert G. Privé"><meta name="citation_author_institution" content="Princess Margaret Cancer Centre, University of Toronto, Toronto, ON M5G 1L7, Canada"><meta name="citation_author_institution" content="Department of Medical Biophysics, University of Toronto, and Princess Margaret Cancer Centre, Toronto, ON M5S 1A8, Canada"><meta name="citation_author" content="Ari M. Melnick"><meta name="citation_author_email" content="amm2014@med.cornell.edu"><meta name="citation_author_orcid" content="0000-0002-8074-2287"><meta name="citation_author_institution" content="Division of Hematology/Oncology, Department of Medicine, Weill Cornell Medicine, Cornell University, New York, NY 10021, USA"><meta name="citation_keywords" content="memory B cells; germinal center; ABC-DLBCL; extranodal lymphoma; cell fate; TBL1XR1; BCL6; BACH2; cell of origin"><meta name="citation_reference" content="citation_title=CD27: a memory B-cell marker; citation_author=K. Agematsu; citation_author=S. Hokibara; citation_author=H. Nagumo; citation_author=A. Komiyama; citation_journal_title=Immunol. Today; citation_volume=21; citation_firstpage=204; citation_lastpage=206; citation_publication_date=2000; "><meta name="citation_reference" content="citation_title=New markers for murine memory B cells that define mutated and unmutated subsets; citation_author=S.M. Anderson; citation_author=M.M. Tomayko; citation_author=A. Ahuja; citation_author=A.M. Haberman; citation_author=M.J. Shlomchik; citation_journal_title=J.&nbsp;Exp. Med.; citation_volume=204; citation_firstpage=2103; citation_lastpage=2114; citation_publication_date=2007; "><meta name="citation_reference" content="citation_title=Visualization of splenic marginal zone B-cell shuttling and follicular B-cell egress; citation_author=T.I. Arnon; citation_author=R.M. Horton; citation_author=I.L. Grigorova; citation_author=J.G. Cyster; citation_journal_title=Nature; citation_volume=493; citation_firstpage=684; citation_lastpage=688; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=Generation of memory B cells and plasma cells in&nbsp;vitro; citation_author=C. Arpin; citation_author=J. Déchanet; citation_author=C. Van Kooten; citation_author=P. Merville; citation_author=G. Grouard; citation_author=F. Brière; citation_author=J. Banchereau; citation_author=Y.J. Liu; citation_journal_title=Science; citation_volume=268; citation_firstpage=720; citation_lastpage=722; citation_publication_date=1995; "><meta name="citation_reference" content="citation_title=Genome-wide discovery of somatic regulatory variants in diffuse large B-cell lymphoma; citation_author=S.E. Arthur; citation_author=A. Jiang; citation_author=B.M. Grande; citation_author=M. Alcaide; citation_author=R. Cojocaru; citation_author=C.K. Rushton; citation_author=A. Mottok; citation_author=L.K. Hilton; citation_author=P.K. Lat; citation_author=E.Y. Zhao; citation_author=et al.; citation_journal_title=Nat. Commun.; citation_volume=9; citation_firstpage=4001; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Defective TCR expression in transgenic mice constructed using cDNA-based alpha- and beta-chain genes under the control of heterologous regulatory elements; citation_author=M.J. Barnden; citation_author=J. Allison; citation_author=W.R. Heath; citation_author=F.R. Carbone; citation_journal_title=Immunol. Cell Biol.; citation_volume=76; citation_firstpage=34; citation_lastpage=40; citation_publication_date=1998; "><meta name="citation_reference" content="citation_title=Persistent antigen and germinal center B cells sustain T follicular helper cell responses and phenotype; citation_author=D. Baumjohann; citation_author=S. Preite; citation_author=A. Reboldi; citation_author=F. Ronchi; citation_author=K.M. Ansel; citation_author=A. Lanzavecchia; citation_author=F. Sallusto; citation_journal_title=Immunity; citation_volume=38; citation_firstpage=596; citation_lastpage=605; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=EZH2 is required for germinal center formation and somatic EZH2 mutations promote lymphoid transformation; citation_author=W. Béguelin; citation_author=R. Popovic; citation_author=M. Teater; citation_author=Y. Jiang; citation_author=K.L. Bunting; citation_author=M. Rosen; citation_author=H. Shen; citation_author=S.N. Yang; citation_author=L. Wang; citation_author=T. Ezponda; citation_author=et al.; citation_journal_title=Cancer Cell; citation_volume=23; citation_firstpage=677; citation_lastpage=692; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=EZH2 enables germinal centre formation through epigenetic silencing of CDKN1A and an Rb-E2F1 feedback loop; citation_author=W. Béguelin; citation_author=M.A. Rivas; citation_author=M.T. Calvo Fernández; citation_author=M. Teater; citation_author=A. Purwada; citation_author=D. Redmond; citation_author=H. Shen; citation_author=M.F. Challman; citation_author=O. Elemento; citation_author=A. Singh; citation_author=A.M. Melnick; citation_journal_title=Nat. Commun.; citation_volume=8; citation_firstpage=877; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Aggressive lymphoma 2016: revision of the WHO classification; citation_author=C. Beham-Schmid; citation_journal_title=Memo; citation_volume=10; citation_firstpage=248; citation_lastpage=254; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Rationally designed BCL6 inhibitors target activated B cell diffuse large B cell lymphoma; citation_author=M.G. Cardenas; citation_author=W. Yu; citation_author=W. Beguelin; citation_author=M.R. Teater; citation_author=H. Geng; citation_author=R.L. Goldstein; citation_author=E. Oswald; citation_author=K. Hatzi; citation_author=S.N. Yang; citation_author=J. Cohen; citation_author=et al.; citation_journal_title=J.&nbsp;Clin. Invest.; citation_volume=126; citation_firstpage=3351; citation_lastpage=3362; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Tracking germinal center B cells expressing germ-line immunoglobulin gamma1 transcripts by conditional gene targeting; citation_author=S. Casola; citation_author=G. Cattoretti; citation_author=N. Uyttersprot; citation_author=S.B. Koralov; citation_author=J. Seagal; citation_author=Z. Hao; citation_author=A. Waisman; citation_author=A. Egert; citation_author=D. Ghitza; citation_author=K. Rajewsky; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=103; citation_firstpage=7396; citation_lastpage=7401; citation_publication_date=2006; "><meta name="citation_reference" content="citation_title=Deregulated BCL6 expression recapitulates the pathogenesis of human diffuse large B cell lymphomas in mice; citation_author=G. Cattoretti; citation_author=L. Pasqualucci; citation_author=G. Ballon; citation_author=W. Tam; citation_author=S.V. Nandula; citation_author=Q. Shen; citation_author=T. Mo; citation_author=V.V. Murty; citation_author=R. Dalla-Favera; citation_journal_title=Cancer Cell; citation_volume=7; citation_firstpage=445; citation_lastpage=455; citation_publication_date=2005; "><meta name="citation_reference" content="citation_title=A small-molecule inhibitor of BCL6 kills DLBCL cells in&nbsp;vitro and in&nbsp;vivo; citation_author=L.C. Cerchietti; citation_author=A.F. Ghetu; citation_author=X. Zhu; citation_author=G.F. Da Silva; citation_author=S. Zhong; citation_author=M. Matthews; citation_author=K.L. Bunting; citation_author=J.M. Polo; citation_author=C. Farès; citation_author=C.H. Arrowsmith; citation_author=et al.; citation_journal_title=Cancer Cell; citation_volume=17; citation_firstpage=400; citation_lastpage=411; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=Targetable genetic features of primary testicular and primary central nervous system lymphomas; citation_author=B. Chapuy; citation_author=M.G. Roemer; citation_author=C. Stewart; citation_author=Y. Tan; citation_author=R.P. Abo; citation_author=L. Zhang; citation_author=A.J. Dunford; citation_author=D.M. Meredith; citation_author=A.R. Thorner; citation_author=E.S. Jordanova; citation_author=et al.; citation_journal_title=Blood; citation_volume=127; citation_firstpage=869; citation_lastpage=881; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Molecular subtypes of diffuse large B cell lymphoma are associated with distinct pathogenic mechanisms and outcomes; citation_author=B. Chapuy; citation_author=C. Stewart; citation_author=A.J. Dunford; citation_author=J. Kim; citation_author=A. Kamburov; citation_author=R.A. Redd; citation_author=M.S. Lawrence; citation_author=M.G.M. Roemer; citation_author=A.J. Li; citation_author=M. Ziepert; citation_author=et al.; citation_journal_title=Nat. Med.; citation_volume=24; citation_firstpage=679; citation_lastpage=690; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=SAINT: probabilistic scoring of affinity purification-mass spectrometry data; citation_author=H. Choi; citation_author=B. Larsen; citation_author=Z.Y. Lin; citation_author=A. Breitkreutz; citation_author=D. Mellacheruvu; citation_author=D. Fermin; citation_author=Z.S. Qin; citation_author=M. Tyers; citation_author=A.C. Gingras; citation_author=A.I. Nesvizhskii; citation_journal_title=Nat. Methods; citation_volume=8; citation_firstpage=70; citation_lastpage=73; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=Sensitive detection of somatic point mutations in impure and heterogeneous cancer samples; citation_author=K. Cibulskis; citation_author=M.S. Lawrence; citation_author=S.L. Carter; citation_author=A. Sivachenko; citation_author=D. Jaffe; citation_author=C. Sougnez; citation_author=S. Gabriel; citation_author=M. Meyerson; citation_author=E.S. Lander; citation_author=G. Getz; citation_journal_title=Nat. Biotechnol.; citation_volume=31; citation_firstpage=213; citation_lastpage=219; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=A program for annotating and predicting the effects of single nucleotide polymorphisms, SnpEff: SNPs in the genome of Drosophila melanogaster strain w1118; iso-2; iso-3; citation_author=P. Cingolani; citation_author=A. Platts; citation_author=L. Wang; citation_author=M. Coon; citation_author=T. Nguyen; citation_author=L. Wang; citation_author=S.J. Land; citation_author=X. Lu; citation_author=D.M. Ruden; citation_journal_title=Fly (Austin); citation_volume=6; citation_firstpage=80; citation_lastpage=92; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=Biopython: freely available Python tools for computational molecular biology and bioinformatics; citation_author=P.J. Cock; citation_author=T. Antao; citation_author=J.T. Chang; citation_author=B.A. Chapman; citation_author=C.J. Cox; citation_author=A. Dalke; citation_author=I. Friedberg; citation_author=T. Hamelryck; citation_author=F. Kauff; citation_author=B. Wilczynski; citation_author=M.J. de Hoon; citation_journal_title=Bioinformatics; citation_volume=25; citation_firstpage=1422; citation_lastpage=1423; citation_publication_date=2009; "><meta name="citation_reference" content="citation_title=TANDEM: matching proteins with tandem mass spectra; citation_author=R. Craig; citation_author=R.C. Beavis; citation_journal_title=Bioinformatics; citation_volume=20; citation_firstpage=1466; citation_lastpage=1467; citation_publication_date=2004; "><meta name="citation_reference" content="citation_title=B Cell Responses: Cell Interaction Dynamics and Decisions; citation_author=J.G. Cyster; citation_author=C.D.C. Allen; citation_journal_title=Cell; citation_volume=177; citation_firstpage=524; citation_lastpage=540; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=STAT3-mediated up-regulation of BLIMP1 Is coordinated with BCL6 down-regulation to control human plasma cell differentiation; citation_author=S.A. Diehl; citation_author=H. Schmidlin; citation_author=M. Nagasawa; citation_author=S.D. van Haren; citation_author=M.J. Kwakkenbos; citation_author=E. Yasuda; citation_author=T. Beaumont; citation_author=F.A. Scheeren; citation_author=H. Spits; citation_journal_title=J.&nbsp;Immunol.; citation_volume=180; citation_firstpage=4805; citation_lastpage=4815; citation_publication_date=2008; "><meta name="citation_reference" content="citation_title=STAR: ultrafast universal RNA-seq aligner; citation_author=A. Dobin; citation_author=C.A. Davis; citation_author=F. Schlesinger; citation_author=J. Drenkow; citation_author=C. Zaleski; citation_author=S. Jha; citation_author=P. Batut; citation_author=M. Chaisson; citation_author=T.R. Gingeras; citation_journal_title=Bioinformatics; citation_volume=29; citation_firstpage=15; citation_lastpage=21; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=Multiple layers of B cell memory with different effector functions; citation_author=I. Dogan; citation_author=B. Bertocci; citation_author=V. Vilmont; citation_author=F. Delbos; citation_author=J. Mégret; citation_author=S. Storck; citation_author=C.A. Reynaud; citation_author=J.C. Weill; citation_journal_title=Nat. Immunol.; citation_volume=10; citation_firstpage=1292; citation_lastpage=1299; citation_publication_date=2009; "><meta name="citation_reference" content="citation_title=TET2 Deficiency Causes Germinal Center Hyperplasia, Impairs Plasma Cell Differentiation, and Promotes B-cell Lymphomagenesis; citation_author=P.M. Dominguez; citation_author=H. Ghamlouch; citation_author=W. Rosikiewicz; citation_author=P. Kumar; citation_author=W. Béguelin; citation_author=L. Fontán; citation_author=M.A. Rivas; citation_author=P. Pawlikowska; citation_author=M. Armand; citation_author=E. Mouly; citation_author=et al.; citation_journal_title=Cancer Discov.; citation_volume=8; citation_firstpage=1632; citation_lastpage=1653; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=VavP-Bcl2 transgenic mice develop follicular lymphoma preceded by germinal center hyperplasia; citation_author=A. Egle; citation_author=A.W. Harris; citation_author=M.L. Bath; citation_author=L. O’Reilly; citation_author=S. Cory; citation_journal_title=Blood; citation_volume=103; citation_firstpage=2276; citation_lastpage=2283; citation_publication_date=2004; "><meta name="citation_reference" content="citation_title=Genetic profiling of MYC and BCL2 in diffuse large B-cell lymphoma determines cell-of-origin-specific clinical impact; citation_author=D. Ennishi; citation_author=A. Mottok; citation_author=S. Ben-Neriah; citation_author=H.P. Shulha; citation_author=P. Farinha; citation_author=F.C. Chan; citation_author=B. Meissner; citation_author=M. Boyle; citation_author=C. Hother; citation_author=R. Kridel; citation_author=et al.; citation_journal_title=Blood; citation_volume=129; citation_firstpage=2760; citation_lastpage=2770; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Identification of programmed cell death in&nbsp;situ via specific labeling of nuclear DNA fragmentation; citation_author=Y. Gavrieli; citation_author=Y. Sherman; citation_author=S.A. Ben-Sasson; citation_journal_title=J.&nbsp;Cell Biol.; citation_volume=119; citation_firstpage=493; citation_lastpage=501; citation_publication_date=1992; "><meta name="citation_reference" content="citation_title=Bach2 Negatively Regulates T Follicular Helper Cell Differentiation and Is Critical for CD4+ T Cell Memory; citation_author=J. Geng; citation_author=H. Wei; citation_author=B. Shi; citation_author=Y.H. Wang; citation_author=B.D. Greer; citation_author=M. Pittman; citation_author=E. Smith; citation_author=P.G. Thomas; citation_author=O. Kutsch; citation_author=H. Hu; citation_journal_title=J.&nbsp;Immunol.; citation_volume=202; citation_firstpage=2991; citation_lastpage=2998; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Recurrent mutations of MYD88 and TBL1XR1 in primary central nervous system lymphomas; citation_author=A. Gonzalez-Aguilar; citation_author=A. Idbaih; citation_author=B. Boisselier; citation_author=N. Habbita; citation_author=M. Rossetto; citation_author=A. Laurenge; citation_author=A. Bruno; citation_author=A. Jouvet; citation_author=M. Polivka; citation_author=C. Adam; citation_author=et al.; citation_journal_title=Clin. Cancer Res.; citation_volume=18; citation_firstpage=5203; citation_lastpage=5211; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=Revealing global regulatory perturbations across human cancers; citation_author=H. Goodarzi; citation_author=O. Elemento; citation_author=S. Tavazoie; citation_journal_title=Mol. Cell; citation_volume=36; citation_firstpage=900; citation_lastpage=911; citation_publication_date=2009; "><meta name="citation_reference" content="citation_title=Inactivation of CREBBP expands the germinal center B cell compartment, down-regulates MHCII expression and promotes DLBCL growth; citation_author=H. Hashwah; citation_author=C.A. Schmid; citation_author=S. Kasser; citation_author=K. Bertram; citation_author=A. Stelling; citation_author=M.G. Manz; citation_author=A. Müller; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=114; citation_firstpage=9701; citation_lastpage=9706; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Breaking bad in the germinal center: how deregulation of BCL6 contributes to lymphomagenesis; citation_author=K. Hatzi; citation_author=A. Melnick; citation_journal_title=Trends Mol. Med.; citation_volume=20; citation_firstpage=343; citation_lastpage=352; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=A hybrid mechanism of action for BCL6 in B cells defined by formation of functionally distinct complexes at enhancers and promoters; citation_author=K. Hatzi; citation_author=Y. Jiang; citation_author=C. Huang; citation_author=F. Garrett-Bakelman; citation_author=M.D. Gearhart; citation_author=E.G. Giannopoulou; citation_author=P. Zumbo; citation_author=K. Kirouac; citation_author=S. Bhaskara; citation_author=J.M. Polo; citation_author=et al.; citation_journal_title=Cell Rep.; citation_volume=4; citation_firstpage=578; citation_lastpage=588; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=Histone demethylase LSD1 is required for germinal center formation and BCL6-driven lymphomagenesis; citation_author=K. Hatzi; citation_author=H. Geng; citation_author=A.S. Doane; citation_author=C. Meydan; citation_author=R. LaRiviere; citation_author=M. Cardenas; citation_author=C. Duy; citation_author=H. Shen; citation_author=M.N.C. Vidal; citation_author=T. Baslan; citation_author=et al.; citation_journal_title=Nat. Immunol.; citation_volume=20; citation_firstpage=86; citation_lastpage=96; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=A specific mutation in TBL1XR1 causes Pierpont syndrome; citation_author=C.A. Heinen; citation_author=A. Jongejan; citation_author=P.J. Watson; citation_author=B. Redeker; citation_author=A. Boelen; citation_author=O. Boudzovitch-Surovtseva; citation_author=F. Forzano; citation_author=R. Hordijk; citation_author=R. Kelley; citation_author=A.H. Olney; citation_author=et al.; citation_journal_title=J.&nbsp;Med. Genet.; citation_volume=53; citation_firstpage=330; citation_lastpage=337; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Poly peak parser: Method and software for identification of unknown indels using sanger sequencing of polymerase chain reaction products; citation_author=J.T. Hill; citation_author=B.L. Demarest; citation_author=B.W. Bisgrove; citation_author=Y.C. Su; citation_author=M. Smith; citation_author=H.J. Yost; citation_journal_title=Dev. Dyn.; citation_volume=243; citation_firstpage=1632; citation_lastpage=1636; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Transcriptional integration of TLR2 and TLR4 signaling at the NCoR derepression checkpoint; citation_author=W. Huang; citation_author=S. Ghisletti; citation_author=V. Perissi; citation_author=M.G. Rosenfeld; citation_author=C.K. Glass; citation_journal_title=Mol. Cell; citation_volume=35; citation_firstpage=48; citation_lastpage=57; citation_publication_date=2009; "><meta name="citation_reference" content="citation_title=Cooperative transcriptional repression by BCL6 and BACH2 in germinal center B-cell differentiation; citation_author=C. Huang; citation_author=H. Geng; citation_author=I. Boss; citation_author=L. Wang; citation_author=A. Melnick; citation_journal_title=Blood; citation_volume=123; citation_firstpage=1012; citation_lastpage=1020; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=T Follicular Helper Cell-Germinal Center B Cell Interaction Strength Regulates Entry into Plasma Cell or Recycling Germinal Center Cell Fate; citation_author=W. Ise; citation_author=K. Fujii; citation_author=K. Shiroguchi; citation_author=A. Ito; citation_author=K. Kometani; citation_author=K. Takeda; citation_author=E. Kawakami; citation_author=K. Yamashita; citation_author=K. Suzuki; citation_author=T. Okada; citation_author=et al.; citation_journal_title=Immunity; citation_volume=48; citation_firstpage=702; citation_lastpage=715; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=WD40 Repeat Proteins: Signalling Scaffold with Diverse Functions; citation_author=B.P. Jain; citation_author=S. Pandey; citation_journal_title=Protein J.; citation_volume=37; citation_firstpage=391; citation_lastpage=406; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=CREBBP Inactivation Promotes the Development of HDAC3-Dependent Lymphomas; citation_author=Y. Jiang; citation_author=A. Ortega-Molina; citation_author=H. Geng; citation_author=H.Y. Ying; citation_author=K. Hatzi; citation_author=S. Parsa; citation_author=D. McNally; citation_author=L. Wang; citation_author=A.S. Doane; citation_author=X. Agirre; citation_author=et al.; citation_journal_title=Cancer Discov.; citation_volume=7; citation_firstpage=38; citation_lastpage=53; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Rapid methods for the analysis of immunoglobulin gene hypermutation: application to transgenic and gene targeted mice; citation_author=C.J. Jolly; citation_author=N. Klix; citation_author=M.S. Neuberger; citation_journal_title=Nucleic Acids Res.; citation_volume=25; citation_firstpage=1913; citation_lastpage=1919; citation_publication_date=1997; "><meta name="citation_reference" content="citation_title=Extranodal manifestations of lymphoma on [18F]FDG-PET/CT: a pictorial essay; citation_author=R. Kashyap; citation_author=B. Rai Mittal; citation_author=K. Manohar; citation_author=C.N. Balasubramanian Harisankar; citation_author=A. Bhattacharya; citation_author=B. Singh; citation_author=P. Malhotra; citation_author=S. Varma; citation_journal_title=Cancer Imaging; citation_volume=11; citation_firstpage=166; citation_lastpage=174; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=ProteoWizard: open source software for rapid proteomics tools development; citation_author=D. Kessner; citation_author=M. Chambers; citation_author=R. Burke; citation_author=D. Agus; citation_author=P. Mallick; citation_journal_title=Bioinformatics; citation_volume=24; citation_firstpage=2534; citation_lastpage=2536; citation_publication_date=2008; "><meta name="citation_reference" content="citation_title=A B cell-deficient mouse by targeted disruption of the membrane exon of the immunoglobulin mu chain gene; citation_author=D. Kitamura; citation_author=J. Roes; citation_author=R. Kühn; citation_author=K. Rajewsky; citation_journal_title=Nature; citation_volume=350; citation_firstpage=423; citation_lastpage=426; citation_publication_date=1991; "><meta name="citation_reference" content="citation_title=VarScan 2: somatic mutation and copy number alteration discovery in cancer by exome sequencing; citation_author=D.C. Koboldt; citation_author=Q. Zhang; citation_author=D.E. Larson; citation_author=D. Shen; citation_author=M.D. McLellan; citation_author=L. Lin; citation_author=C.A. Miller; citation_author=E.R. Mardis; citation_author=L. Ding; citation_author=R.K. Wilson; citation_journal_title=Genome Res.; citation_volume=22; citation_firstpage=568; citation_lastpage=576; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=Repression of the transcription factor Bach2 contributes to predisposition of IgG1 memory B cells toward plasma cell differentiation; citation_author=K. Kometani; citation_author=R. Nakagawa; citation_author=R. Shinnakasu; citation_author=T. Kaji; citation_author=A. Rybouchkin; citation_author=S. Moriyama; citation_author=K. Furukawa; citation_author=H. Koseki; citation_author=T. Takemori; citation_author=T. Kurosaki; citation_journal_title=Immunity; citation_volume=39; citation_firstpage=136; citation_lastpage=147; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=Optimized Sleeping Beauty transposons rapidly generate stable transgenic cell lines; citation_author=E. Kowarz; citation_author=D. Löscher; citation_author=R. Marschalek; citation_journal_title=Biotechnol. J.; citation_volume=10; citation_firstpage=647; citation_lastpage=653; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Recurrent somatic mutations affecting B-cell receptor signaling pathway genes in follicular lymphoma; citation_author=K. Krysiak; citation_author=F. Gomez; citation_author=B.S. White; citation_author=M. Matlock; citation_author=C.A. Miller; citation_author=L. Trani; citation_author=C.C. Fronick; citation_author=R.S. Fulton; citation_author=F. Kreisel; citation_author=A.F. Cashen; citation_author=et al.; citation_journal_title=Blood; citation_volume=129; citation_firstpage=473; citation_lastpage=483; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Repression of BCL-6 is required for the formation of human memory B cells in&nbsp;vitro; citation_author=T.C. Kuo; citation_author=A.L. Shaffer; citation_author=J. Haddad; citation_author=Y.S. Choi; citation_author=L.M. Staudt; citation_author=K. Calame; citation_journal_title=J.&nbsp;Exp. Med.; citation_volume=204; citation_firstpage=819; citation_lastpage=830; citation_publication_date=2007; "><meta name="citation_reference" content="citation_title=The Eph-related tyrosine kinase ligand Ephrin-B1 marks germinal center and memory precursor B cells; citation_author=B.J. Laidlaw; citation_author=T.H. Schmidt; citation_author=J.A. Green; citation_author=C.D. Allen; citation_author=T. Okada; citation_author=J.G. Cyster; citation_journal_title=J.&nbsp;Exp. Med.; citation_volume=214; citation_firstpage=639; citation_lastpage=649; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Integrating population variation and protein structural analysis to improve clinical interpretation of missense variation: application to the WD40 domain; citation_author=R.A. Laskowski; citation_author=N. Tyagi; citation_author=D. Johnson; citation_author=S. Joss; citation_author=E. Kinning; citation_author=C. McWilliam; citation_author=M. Splitt; citation_author=J.M. Thornton; citation_author=H.V. Firth; citation_author=C.F. Wright; citation_author=DDD Study; citation_journal_title=Hum. Mol. Genet.; citation_volume=25; citation_firstpage=927; citation_lastpage=935; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Aberrant immunoglobulin class switch recombination and switch translocations in activated B cell-like diffuse large B cell lymphoma; citation_author=G. Lenz; citation_author=I. Nagel; citation_author=R. Siebert; citation_author=A.V. Roschke; citation_author=W. Sanger; citation_author=G.W. Wright; citation_author=S.S. Dave; citation_author=B. Tan; citation_author=H. Zhao; citation_author=A. Rosenwald; citation_author=et al.; citation_journal_title=J.&nbsp;Exp. Med.; citation_volume=204; citation_firstpage=633; citation_lastpage=643; citation_publication_date=2007; "><meta name="citation_reference" content="citation_title=TBL1-TBLR1 and beta-catenin recruit each other to Wnt target-gene promoter for transcription activation and oncogenesis; citation_author=J. Li; citation_author=C.Y. Wang; citation_journal_title=Nat. Cell Biol.; citation_volume=10; citation_firstpage=160; citation_lastpage=169; citation_publication_date=2008; "><meta name="citation_reference" content="citation_title=featureCounts: an efficient general purpose program for assigning sequence reads to genomic features; citation_author=Y. Liao; citation_author=G.K. Smyth; citation_author=W. Shi; citation_journal_title=Bioinformatics; citation_volume=30; citation_firstpage=923; citation_lastpage=930; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Two levels of protection for the B cell genome during somatic hypermutation; citation_author=M. Liu; citation_author=J.L. Duke; citation_author=D.J. Richter; citation_author=C.G. Vinuesa; citation_author=C.C. Goodnow; citation_author=S.H. Kleinstein; citation_author=D.G. Schatz; citation_journal_title=Nature; citation_volume=451; citation_firstpage=841; citation_lastpage=845; citation_publication_date=2008; "><meta name="citation_reference" content="citation_title=ProHits: integrated software for mass spectrometry-based interaction proteomics; citation_author=G. Liu; citation_author=J. Zhang; citation_author=B. Larsen; citation_author=C. Stark; citation_author=A. Breitkreutz; citation_author=Z.Y. Lin; citation_author=B.J. Breitkreutz; citation_author=Y. Ding; citation_author=K. Colwill; citation_author=A. Pasculescu; citation_author=et al.; citation_journal_title=Nat. Biotechnol.; citation_volume=28; citation_firstpage=1015; citation_lastpage=1017; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=Memory T and memory B cells share a transcriptional program of self-renewal with long-term hematopoietic stem cells; citation_author=C.J. Luckey; citation_author=D. Bhattacharya; citation_author=A.W. Goldrath; citation_author=I.L. Weissman; citation_author=C. Benoist; citation_author=D. Mathis; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=103; citation_firstpage=3304; citation_lastpage=3309; citation_publication_date=2006; "><meta name="citation_reference" content="citation_title=Pathognomonic and epistatic genetic alterations in B-cell non-Hodgkin lymphoma; citation_author=M.C.J. Ma; citation_author=S. Tadros; citation_author=A. Bouska; citation_author=T.B. Heavican; citation_author=H. Yang; citation_author=Q. Deng; citation_author=D. Moore; citation_author=A. Akhter; citation_author=K. Hartert; citation_author=N. Jain; citation_author=et al.; citation_journal_title=bioRxiv; citation_doi=10.1101/674259; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Molecular evolution of a novel hyperactive Sleeping Beauty transposase enables robust stable gene transfer in vertebrates; citation_author=L. Mátés; citation_author=M.K. Chuah; citation_author=E. Belay; citation_author=B. Jerchow; citation_author=N. Manoj; citation_author=A. Acosta-Sanchez; citation_author=D.P. Grzela; citation_author=A. Schmitt; citation_author=K. Becker; citation_author=J. Matrai; citation_author=et al.; citation_journal_title=Nat. Genet.; citation_volume=41; citation_firstpage=753; citation_lastpage=761; citation_publication_date=2009; "><meta name="citation_reference" content="citation_title=The microanatomic segregation of selection by apoptosis in the germinal center; citation_author=C.T. Mayer; citation_author=A. Gazumyan; citation_author=E.E. Kara; citation_author=A.D. Gitlin; citation_author=J. Golijanin; citation_author=C. Viant; citation_author=J. Pai; citation_author=T.Y. Oliveira; citation_author=Q. Wang; citation_author=A. Escolano; citation_author=et al.; citation_journal_title=Science; citation_volume=358; citation_firstpage=eaao2602; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=The CRAPome: a contaminant repository for affinity purification-mass spectrometry data; citation_author=D. Mellacheruvu; citation_author=Z. Wright; citation_author=A.L. Couzens; citation_author=J.P. Lambert; citation_author=N.A. St-Denis; citation_author=T. Li; citation_author=Y.V. Miteva; citation_author=S. Hauri; citation_author=M.E. Sardiu; citation_author=T.Y. Low; citation_author=et al.; citation_journal_title=Nat. Methods; citation_volume=10; citation_firstpage=730; citation_lastpage=736; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=Analysis of BCL6-interacting proteins by tandem mass spectrometry; citation_author=R.R. Miles; citation_author=D.K. Crockett; citation_author=M.S. Lim; citation_author=K.S. Elenitoba-Johnson; citation_journal_title=Mol. Cell. Proteomics; citation_volume=4; citation_firstpage=1898; citation_lastpage=1909; citation_publication_date=2005; "><meta name="citation_reference" content="citation_title=Multifunctional role of the transcription factor Blimp-1 in coordinating plasma cell differentiation; citation_author=M. Minnich; citation_author=H. Tagoh; citation_author=P. Bönelt; citation_author=E. Axelsson; citation_author=M. Fischer; citation_author=B. Cebolla; citation_author=A. Tarakhovsky; citation_author=S.L. Nutt; citation_author=M. Jaritz; citation_author=M. Busslinger; citation_journal_title=Nat. Immunol.; citation_volume=17; citation_firstpage=331; citation_lastpage=343; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Rapid immunoprecipitation mass spectrometry of endogenous proteins (RIME) for analysis of chromatin complexes; citation_author=H. Mohammed; citation_author=C. Taylor; citation_author=G.D. Brown; citation_author=E.K. Papachristou; citation_author=J.S. Carroll; citation_author=C.S. D’Santos; citation_journal_title=Nat. Protoc.; citation_volume=11; citation_firstpage=316; citation_lastpage=326; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Selective Inhibition of HDAC3 Targets Synthetic Vulnerabilities and Activates Immune Surveillance in Lymphoma; citation_author=P. Mondello; citation_author=S. Tadros; citation_author=M. Teater; citation_author=L. Fontan; citation_author=A.Y. Chang; citation_author=N. Jain; citation_author=H. Yang; citation_author=S. Singh; citation_author=H.Y. Ying; citation_author=C.S. Chu; citation_author=et al.; citation_journal_title=Cancer Discov.; citation_volume=10; citation_firstpage=440; citation_lastpage=459; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Fucci2a: a bicistronic cell cycle reporter that allows Cre mediated tissue specific expression in mice; citation_author=R.L. Mort; citation_author=M.J. Ford; citation_author=A. Sakaue-Sawano; citation_author=N.O. Lindstrom; citation_author=A. Casadio; citation_author=A.T. Douglas; citation_author=M.A. Keighren; citation_author=P. Hohenstein; citation_author=A. Miyawaki; citation_author=I.J. Jackson; citation_journal_title=Cell Cycle; citation_volume=13; citation_firstpage=2681; citation_lastpage=2696; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Class switch recombination and hypermutation require activation-induced cytidine deaminase (AID), a potential RNA editing enzyme; citation_author=M. Muramatsu; citation_author=K. Kinoshita; citation_author=S. Fagarasan; citation_author=S. Yamada; citation_author=Y. Shinkai; citation_author=T. Honjo; citation_journal_title=Cell; citation_volume=102; citation_firstpage=553; citation_lastpage=563; citation_publication_date=2000; "><meta name="citation_reference" content="citation_title=Guide for the Care and Use of Laboratory Animals; citation_author=National Research Council (US) Institute for Laboratory Animal Research; citation_publication_date=1996; citation_publisher=National Academies Press (US), Washington, DC; "><meta name="citation_reference" content="citation_title=Single Cell Phenotypic Profiling of 27 DLBCL Cases Reveals Marked Intertumoral and Intratumoral Heterogeneity; citation_author=M.D. Nissen; citation_author=M. Kusakabe; citation_author=X. Wang; citation_author=G. Simkin; citation_author=D. Gracias; citation_author=K. Tyshchenko; citation_author=A. Hill; citation_author=J. Meskas; citation_author=S. Hung; citation_author=E.A. Chavez; citation_author=et al.; citation_journal_title=Cytometry A; citation_doi=10.1002/cyto.a.23919; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=In-vitro derived germinal centre B cells differentially generate memory B or plasma cells in&nbsp;vivo; citation_author=T. Nojima; citation_author=K. Haniuda; citation_author=T. Moutai; citation_author=M. Matsudaira; citation_author=S. Mizokawa; citation_author=I. Shiratori; citation_author=T. Azuma; citation_author=D. Kitamura; citation_journal_title=Nat. Commun.; citation_volume=2; citation_firstpage=465; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=Structural basis for the assembly of the SMRT/NCoR core transcriptional repression machinery; citation_author=J. Oberoi; citation_author=L. Fairall; citation_author=P.J. Watson; citation_author=J.C. Yang; citation_author=Z. Czimmerer; citation_author=T. Kampmann; citation_author=B.T. Goult; citation_author=J.A. Greenwood; citation_author=J.T. Gooch; citation_author=B.C. Kallenberger; citation_author=et al.; citation_journal_title=Nat. Struct. Mol. Biol.; citation_volume=18; citation_firstpage=177; citation_lastpage=184; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=Plasmacytic transcription factor Blimp-1 is repressed by Bach2 in B cells; citation_author=K. Ochiai; citation_author=Y. Katoh; citation_author=T. Ikura; citation_author=Y. Hoshikawa; citation_author=T. Noda; citation_author=H. Karasuyama; citation_author=S. Tashiro; citation_author=A. Muto; citation_author=K. Igarashi; citation_journal_title=J.&nbsp;Biol. Chem.; citation_volume=281; citation_firstpage=38226; citation_lastpage=38234; citation_publication_date=2006; "><meta name="citation_reference" content="citation_title=Constitutive Bcl-2 expression throughout the hematopoietic compartment affects multiple lineages and enhances progenitor cell survival; citation_author=S. Ogilvy; citation_author=D. Metcalf; citation_author=C.G. Print; citation_author=M.L. Bath; citation_author=A.W. Harris; citation_author=J.M. Adams; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=96; citation_firstpage=14943; citation_lastpage=14948; citation_publication_date=1999; "><meta name="citation_reference" content="citation_title=Mouse CD38 is down-regulated on germinal center B cells and mature plasma cells; citation_author=A.M. Oliver; citation_author=F. Martin; citation_author=J.F. Kearney; citation_journal_title=J.&nbsp;Immunol.; citation_volume=158; citation_firstpage=1108; citation_lastpage=1115; citation_publication_date=1997; "><meta name="citation_reference" content="citation_title=The histone lysine methyltransferase KMT2D sustains a gene expression program that represses B cell lymphoma development; citation_author=A. Ortega-Molina; citation_author=I.W. Boss; citation_author=A. Canela; citation_author=H. Pan; citation_author=Y. Jiang; citation_author=C. Zhao; citation_author=M. Jiang; citation_author=D. Hu; citation_author=X. Agirre; citation_author=I. Niesvizky; citation_author=et al.; citation_journal_title=Nat. Med.; citation_volume=21; citation_firstpage=1199; citation_lastpage=1208; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Biostrings: Efficient manipulation of biological strings. R package version 2.54.0; citation_author=H. Pagès; citation_author=P. Aboyoun; citation_author=R. Gentleman; citation_author=S. DebRoy; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=EBI2 mediates B cell segregation between the outer and centre follicle; citation_author=J.P. Pereira; citation_author=L.M. Kelly; citation_author=Y. Xu; citation_author=J.G. Cyster; citation_journal_title=Nature; citation_volume=460; citation_firstpage=1122; citation_lastpage=1126; citation_publication_date=2009; "><meta name="citation_reference" content="citation_title=A corepressor/coactivator exchange complex required for transcriptional activation by nuclear receptors and other regulated transcription factors; citation_author=V. Perissi; citation_author=A. Aggarwal; citation_author=C.K. Glass; citation_author=D.W. Rose; citation_author=M.G. Rosenfeld; citation_journal_title=Cell; citation_volume=116; citation_firstpage=511; citation_lastpage=526; citation_publication_date=2004; "><meta name="citation_reference" content="citation_title=A multiprotein supercomplex controlling oncogenic signalling in lymphoma; citation_author=J.D. Phelan; citation_author=R.M. Young; citation_author=D.E. Webster; citation_author=S. Roulland; citation_author=G.W. Wright; citation_author=M. Kasbekar; citation_author=A.L. Shaffer; citation_author=M. Ceribelli; citation_author=J.Q. Wang; citation_author=R. Schmitz; citation_author=et al.; citation_journal_title=Nature; citation_volume=560; citation_firstpage=387; citation_lastpage=391; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Reversible disruption of BCL6 repression complexes by CD40 signaling in normal and malignant B cells; citation_author=J.M. Polo; citation_author=W. Ci; citation_author=J.D. Licht; citation_author=A. Melnick; citation_journal_title=Blood; citation_volume=112; citation_firstpage=644; citation_lastpage=651; citation_publication_date=2008; "><meta name="citation_reference" content="citation_title=Genetic and Functional Drivers of Diffuse Large B Cell Lymphoma; citation_author=A. Reddy; citation_author=J. Zhang; citation_author=N.S. Davis; citation_author=A.B. Moffitt; citation_author=C.L. Love; citation_author=A. Waldrop; citation_author=S. Leppa; citation_author=A. Pasanen; citation_author=L. Meriranta; citation_author=M.L. Karjalainen-Lindsberg; citation_author=et al.; citation_journal_title=Cell; citation_volume=171; citation_firstpage=481; citation_lastpage=494; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Kinetics of establishing the memory B cell population as revealed by CD38 expression; citation_author=A. Ridderstad; citation_author=D.M. Tarlinton; citation_journal_title=J.&nbsp;Immunol.; citation_volume=160; citation_firstpage=4688; citation_lastpage=4695; citation_publication_date=1998; "><meta name="citation_reference" content="citation_title=edgeR: a Bioconductor package for differential expression analysis of digital gene expression data; citation_author=M.D. Robinson; citation_author=D.J. McCarthy; citation_author=G.K. Smyth; citation_journal_title=Bioinformatics; citation_volume=26; citation_firstpage=139; citation_lastpage=140; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=A promiscuous biotin ligase fusion protein identifies proximal and interacting proteins in mammalian cells; citation_author=K.J. Roux; citation_author=D.I. Kim; citation_author=M. Raida; citation_author=B. Burke; citation_journal_title=J.&nbsp;Cell Biol.; citation_volume=196; citation_firstpage=801; citation_lastpage=810; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=Strelka: accurate somatic small-variant calling from sequenced tumor-normal sample pairs; citation_author=C.T. Saunders; citation_author=W.S. Wong; citation_author=S. Swamy; citation_author=J. Becq; citation_author=L.J. Murray; citation_author=R.K. Cheetham; citation_journal_title=Bioinformatics; citation_volume=28; citation_firstpage=1811; citation_lastpage=1817; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=Fiji: an open-source platform for biological-image analysis; citation_author=J. Schindelin; citation_author=I. Arganda-Carreras; citation_author=E. Frise; citation_author=V. Kaynig; citation_author=M. Longair; citation_author=T. Pietzsch; citation_author=S. Preibisch; citation_author=C. Rueden; citation_author=S. Saalfeld; citation_author=B. Schmid; citation_author=et al.; citation_journal_title=Nat. Methods; citation_volume=9; citation_firstpage=676; citation_lastpage=682; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=Genetics and Pathogenesis of Diffuse Large B-Cell Lymphoma; citation_author=R. Schmitz; citation_author=G.W. Wright; citation_author=D.W. Huang; citation_author=C.A. Johnson; citation_author=J.D. Phelan; citation_author=J.Q. Wang; citation_author=S. Roulland; citation_author=M. Kasbekar; citation_author=R.M. Young; citation_author=A.L. Shaffer; citation_author=et al.; citation_journal_title=N.&nbsp;Engl. J. Med.; citation_volume=378; citation_firstpage=1396; citation_lastpage=1407; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=NIH Image to ImageJ: 25 years of image analysis; citation_author=C.A. Schneider; citation_author=W.S. Rasband; citation_author=K.W. Eliceiri; citation_journal_title=Nat. Methods; citation_volume=9; citation_firstpage=671; citation_lastpage=675; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=Prognostic Significance of Diffuse Large B-Cell Lymphoma Cell of Origin Determined by Digital Gene Expression in Formalin-Fixed Paraffin-Embedded Tissue Biopsies; citation_author=D.W. Scott; citation_author=A. Mottok; citation_author=D. Ennishi; citation_author=G.W. Wright; citation_author=P. Farinha; citation_author=S. Ben-Neriah; citation_author=R. Kridel; citation_author=G.S. Barry; citation_author=C. Hother; citation_author=P. Abrisqueta; citation_author=et al.; citation_journal_title=J.&nbsp;Clin. Oncol.; citation_volume=33; citation_firstpage=2848; citation_lastpage=2856; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=dbSNP: the NCBI database of genetic variation; citation_author=S.T. Sherry; citation_author=M.H. Ward; citation_author=M. Kholodov; citation_author=J. Baker; citation_author=L. Phan; citation_author=E.M. Smigielski; citation_author=K. Sirotkin; citation_journal_title=Nucleic Acids Res.; citation_volume=29; citation_firstpage=308; citation_lastpage=311; citation_publication_date=2001; "><meta name="citation_reference" content="citation_title=Regulated selection of germinal-center cells into the memory B cell compartment; citation_author=R. Shinnakasu; citation_author=T. Inoue; citation_author=K. Kometani; citation_author=S. Moriyama; citation_author=Y. Adachi; citation_author=M. Nakayama; citation_author=Y. Takahashi; citation_author=H. Fukuyama; citation_author=T. Okada; citation_author=T. Kurosaki; citation_journal_title=Nat. Immunol.; citation_volume=17; citation_firstpage=861; citation_lastpage=869; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Lymphoma Driver Mutations in the Pathogenic Evolution of an Iconic Human Autoantibody; citation_author=M. Singh; citation_author=K.J.L. Jackson; citation_author=J.J. Wang; citation_author=P. Schofield; citation_author=M.A. Field; citation_author=D. Koppstein; citation_author=T.J. Peters; citation_author=D.L. Burnett; citation_author=S. Rizzetto; citation_author=D. Nevoltris; citation_author=et al.; citation_journal_title=Cell; citation_volume=180; citation_firstpage=878; citation_lastpage=894; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Histone deacetylase 3 controls a transcriptional network required for B cell maturation; citation_author=K.R. Stengel; citation_author=S. Bhaskara; citation_author=J. Wang; citation_author=Q. Liu; citation_author=J.D. Ellis; citation_author=S. Sampathi; citation_author=S.W. Hiebert; citation_journal_title=Nucleic Acids Res.; citation_volume=47; citation_firstpage=10612; citation_lastpage=10627; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Lentivirus-delivered stable gene silencing by RNAi in primary cells; citation_author=S.A. Stewart; citation_author=D.M. Dykxhoorn; citation_author=D. Palliser; citation_author=H. Mizuno; citation_author=E.Y. Yu; citation_author=D.S. An; citation_author=D.M. Sabatini; citation_author=I.S. Chen; citation_author=W.C. Hahn; citation_author=P.A. Sharp; citation_author=et al.; citation_journal_title=RNA; citation_volume=9; citation_firstpage=493; citation_lastpage=501; citation_publication_date=2003; "><meta name="citation_reference" content="citation_title=Germinal Center B Cells Replace Their Antigen Receptors in Dark Zones and Fail Light Zone Entry when Immunoglobulin Gene Mutations are Damaging; citation_author=I. Stewart; citation_author=D. Radtke; citation_author=B. Phillips; citation_author=S.J. McGowan; citation_author=O. Bannard; citation_journal_title=Immunity; citation_volume=49; citation_firstpage=477; citation_lastpage=489; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=CCR6 Defines Memory B Cell Precursors in Mouse and Human Germinal Centers, Revealing Light-Zone Location and Predominant Low Antigen Affinity; citation_author=D. Suan; citation_author=N.J. Krautler; citation_author=J.L.V. Maag; citation_author=D. Butt; citation_author=K. Bourne; citation_author=J.R. Hermes; citation_author=D.T. Avery; citation_author=C. Young; citation_author=A. Statham; citation_author=M. Elliott; citation_author=et al.; citation_journal_title=Immunity; citation_volume=47; citation_firstpage=1142; citation_lastpage=1153; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Gene set enrichment analysis: a knowledge-based approach for interpreting genome-wide expression profiles; citation_author=A. Subramanian; citation_author=P. Tamayo; citation_author=V.K. Mootha; citation_author=S. Mukherjee; citation_author=B.L. Ebert; citation_author=M.A. Gillette; citation_author=A. Paulovich; citation_author=S.L. Pomeroy; citation_author=T.R. Golub; citation_author=E.S. Lander; citation_author=J.P. Mesirov; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=102; citation_firstpage=15545; citation_lastpage=15550; citation_publication_date=2005; "><meta name="citation_reference" content="citation_title=Germinal center reentries of BCL2-overexpressing B cells drive follicular lymphoma progression; citation_author=S. Sungalee; citation_author=E. Mamessier; citation_author=E. Morgado; citation_author=E. Grégoire; citation_author=P.Z. Brohawn; citation_author=C.A. Morehouse; citation_author=N. Jouve; citation_author=C. Monvoisin; citation_author=C. Menard; citation_author=G. Debroas; citation_author=et al.; citation_journal_title=J.&nbsp;Clin. Invest.; citation_volume=124; citation_firstpage=5337; citation_lastpage=5351; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=COSMIC: the Catalogue Of Somatic Mutations In Cancer; citation_author=J.G. Tate; citation_author=S. Bamford; citation_author=H.C. Jubb; citation_author=Z. Sondka; citation_author=D.M. Beare; citation_author=N. Bindal; citation_author=H. Boutselakis; citation_author=C.G. Cole; citation_author=C. Creatore; citation_author=E. Dawson; citation_author=et al.; citation_journal_title=Nucleic Acids Res.; citation_volume=47; citation_number=D1; citation_firstpage=D941; citation_lastpage=D947; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Recruitment of N-CoR/SMRT-TBLR1 corepressor complex by unliganded thyroid hormone receptor for gene repression during frog development; citation_author=A. Tomita; citation_author=D.R. Buchholz; citation_author=Y.B. Shi; citation_journal_title=Mol. Cell. Biol.; citation_volume=24; citation_firstpage=3337; citation_lastpage=3346; citation_publication_date=2004; "><meta name="citation_reference" content="citation_title=Direct repression of prdm1 by Bcl-6 inhibits plasmacytic differentiation; citation_author=C. Tunyaplin; citation_author=A.L. Shaffer; citation_author=C.D. Angelin-Duclos; citation_author=X. Yu; citation_author=L.M. Staudt; citation_author=K.L. Calame; citation_journal_title=J.&nbsp;Immunol.; citation_volume=173; citation_firstpage=1158; citation_lastpage=1165; citation_publication_date=2004; "><meta name="citation_reference" content="citation_title=WDSPdb: a database for WD40-repeat proteins; citation_author=Y. Wang; citation_author=X.J. Hu; citation_author=X.D. Zou; citation_author=X.H. Wu; citation_author=Z.Q. Ye; citation_author=Y.D. Wu; citation_journal_title=Nucleic Acids Res.; citation_volume=43; citation_firstpage=D339; citation_lastpage=D344; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Synergistic cooperation and crosstalk between MYD88L265P and mutations that dysregulate CD79B and surface IgM; citation_author=J.Q. Wang; citation_author=Y.S. Jeelall; citation_author=P. Humburg; citation_author=E.L. Batchelor; citation_author=S.M. Kaya; citation_author=H.M. Yoo; citation_author=C.C. Goodnow; citation_author=K. Horikawa; citation_journal_title=J.&nbsp;Exp. Med.; citation_volume=214; citation_firstpage=2759; citation_lastpage=2776; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Germinal-center development of memory B cells driven by IL-9 from follicular helper T&nbsp;cells; citation_author=Y. Wang; citation_author=J. Shi; citation_author=J. Yan; citation_author=Z. Xiao; citation_author=X. Hou; citation_author=P. Lu; citation_author=S. Hou; citation_author=T. Mao; citation_author=W. Liu; citation_author=Y. Ma; citation_author=et al.; citation_journal_title=Nat. Immunol.; citation_volume=18; citation_firstpage=921; citation_lastpage=930; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=A Temporal Switch in the Germinal Center Determines Differential Output of Memory B and Plasma Cells; citation_author=F.J. Weisel; citation_author=G.V. Zuccarino-Catania; citation_author=M. Chikina; citation_author=M.J. Shlomchik; citation_journal_title=Immunity; citation_volume=44; citation_firstpage=116; citation_lastpage=130; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=A gene expression-based method to diagnose clinically distinct subgroups of diffuse large B cell lymphoma; citation_author=G. Wright; citation_author=B. Tan; citation_author=A. Rosenwald; citation_author=E.H. Hurt; citation_author=A. Wiestner; citation_author=L.M. Staudt; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=100; citation_firstpage=9991; citation_lastpage=9996; citation_publication_date=2003; "><meta name="citation_reference" content="citation_title=A Probabilistic Classification Tool for Genetic Subtypes of Diffuse Large B Cell Lymphoma with Therapeutic Implications; citation_author=G.W. Wright; citation_author=D.W. Huang; citation_author=J.D. Phelan; citation_author=Z.A. Coulibaly; citation_author=S. Roulland; citation_author=R.M. Young; citation_author=J.Q. Wang; citation_author=R. Schmitz; citation_author=R.D. Morin; citation_author=J. Tang; citation_author=et al.; citation_journal_title=Cancer Cell; citation_volume=37; citation_firstpage=551; citation_lastpage=568.e14; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Purification and functional characterization of the human N-CoR complex: the roles of HDAC3, TBL1 and TBLR1; citation_author=H.G. Yoon; citation_author=D.W. Chan; citation_author=Z.Q. Huang; citation_author=J. Li; citation_author=J.D. Fondell; citation_author=J. Qin; citation_author=J. Wong; citation_journal_title=EMBO J.; citation_volume=22; citation_firstpage=1336; citation_lastpage=1346; citation_publication_date=2003; "><meta name="citation_reference" content="citation_title=Reading and function of a histone code involved in targeting corepressor complexes for repression; citation_author=H.G. Yoon; citation_author=Y. Choi; citation_author=P.A. Cole; citation_author=J. Wong; citation_journal_title=Mol. Cell. Biol.; citation_volume=25; citation_firstpage=324; citation_lastpage=335; citation_publication_date=2005; "><meta name="citation_reference" content="citation_title=The N-CoR-HDAC3 nuclear receptor corepressor complex inhibits the JNK pathway through the integral subunit GPS2; citation_author=J. Zhang; citation_author=M. Kalkum; citation_author=B.T. Chait; citation_author=R.G. Roeder; citation_journal_title=Mol. Cell; citation_volume=9; citation_firstpage=611; citation_lastpage=623; citation_publication_date=2002; "><meta name="citation_reference" content="citation_title=TBLR1 regulates the expression of nuclear hormone receptor co-repressors; citation_author=X.M. Zhang; citation_author=Q. Chang; citation_author=L. Zeng; citation_author=J. Gu; citation_author=S. Brown; citation_author=R.S. Basch; citation_journal_title=BMC Cell Biol.; citation_volume=7; citation_firstpage=31; citation_publication_date=2006; "><meta name="citation_reference" content="citation_title=CD80 and PD-L2 define functionally distinct memory B cell subsets that are independent of antibody isotype; citation_author=G.V. Zuccarino-Catania; citation_author=S. Sadanand; citation_author=F.J. Weisel; citation_author=M.M. Tomayko; citation_author=H. Meng; citation_author=S.H. Kleinstein; citation_author=K.L. Good-Jacobson; citation_author=M.J. Shlomchik; citation_journal_title=Nat. Immunol.; citation_volume=15; citation_firstpage=631; citation_lastpage=637; citation_publication_date=2014; "><meta name="citation_abstract" content="<h2>Summary</h2><p>The most aggressive B cell lymphomas frequently manifest extranodal distribution and carry somatic mutations in the poorly characterized gene <i>TBL1XR1</i>. Here, we show that <i>TBL1XR1</i> mutations skew the humoral immune response toward generating abnormal immature memory B cells (MB), while impairing plasma cell differentiation. At the molecular level, TBL1XR1 mutants co-opt SMRT/HDAC3 repressor complexes toward binding the MB cell transcription factor (TF) BACH2 at the expense of the germinal center (GC) TF BCL6, leading to pre-memory transcriptional reprogramming and cell-fate bias. Upon antigen recall, TBL1XR1 mutant MB cells fail to differentiate into plasma cells and instead preferentially reenter new GC reactions, providing evidence for a cyclic reentry lymphomagenesis mechanism. Ultimately, <i>TBL1XR1</i> alterations lead to a striking extranodal immunoblastic lymphoma phenotype that mimics the human disease. Both human and murine lymphomas feature expanded MB-like cell populations, consistent with a MB-cell origin and delineating an unforeseen pathway for malignant transformation of the immune system.</p>"><meta charset="UTF-8"><!--

    Pagebuild admin UI
-->



<script type="text/javascript" src="https://bam.nr-data.net/1/16e2192192?a=916957862&amp;sa=1&amp;v=1177.96a4d39&amp;t=Unnamed%20Transaction&amp;rst=6341&amp;ck=1&amp;ref=https://www.cell.com/cell/fulltext/S0092-8674(20)30682-6&amp;be=4631&amp;fe=6168&amp;dc=6142&amp;af=err,xhr,stn,ins,spa&amp;perf=%7B%22timing%22:%7B%22of%22:1749586966413,%22n%22:0,%22f%22:2908,%22dn%22:2909,%22dne%22:2909,%22c%22:2909,%22s%22:2909,%22ce%22:2909,%22rq%22:2909,%22rp%22:4138,%22rpe%22:5958,%22dl%22:4142,%22di%22:6076,%22ds%22:6142,%22de%22:6161,%22dc%22:6168,%22l%22:6168,%22le%22:6199%7D,%22navigation%22:%7B%7D%7D&amp;fp=5108&amp;fcp=5108&amp;jsonp=NREUM.setToken"></script><script src="https://js-agent.newrelic.com/nr-spa-1177.min.js"></script><script type="text/javascript">
    ;window.NREUM||(NREUM={});NREUM.init={privacy:{cookies_enabled:true}};
    window.NREUM||(NREUM={}),__nr_require=function(t,e,n){function r(n){if(!e[n]){var o=e[n]={exports:{}};t[n][0].call(o.exports,function(e){var o=t[n][1][e];return r(o||e)},o,o.exports)}return e[n].exports}if("function"==typeof __nr_require)return __nr_require;for(var o=0;o<n.length;o++)r(n[o]);return r}({1:[function(t,e,n){function r(t){try{c.console&&console.log(t)}catch(e){}}var o,i=t("ee"),a=t(26),c={};try{o=localStorage.getItem("__nr_flags").split(","),console&&"function"==typeof console.log&&(c.console=!0,o.indexOf("dev")!==-1&&(c.dev=!0),o.indexOf("nr_dev")!==-1&&(c.nrDev=!0))}catch(s){}c.nrDev&&i.on("internal-error",function(t){r(t.stack)}),c.dev&&i.on("fn-err",function(t,e,n){r(n.stack)}),c.dev&&(r("NR AGENT IN DEVELOPMENT MODE"),r("flags: "+a(c,function(t,e){return t}).join(", ")))},{}],2:[function(t,e,n){function r(t,e,n,r,c){try{l?l-=1:o(c||new UncaughtException(t,e,n),!0)}catch(f){try{i("ierr",[f,s.now(),!0])}catch(d){}}return"function"==typeof u&&u.apply(this,a(arguments))}function UncaughtException(t,e,n){this.message=t||"Uncaught error with no additional information",this.sourceURL=e,this.line=n}function o(t,e){var n=e?null:s.now();i("err",[t,n])}var i=t("handle"),a=t(27),c=t("ee"),s=t("loader"),f=t("gos"),u=window.onerror,d=!1,p="nr@seenError",l=0;s.features.err=!0,t(1),window.onerror=r;try{throw new Error}catch(h){"stack"in h&&(t(13),t(12),"addEventListener"in window&&t(6),s.xhrWrappable&&t(14),d=!0)}c.on("fn-start",function(t,e,n){d&&(l+=1)}),c.on("fn-err",function(t,e,n){d&&!n[p]&&(f(n,p,function(){return!0}),this.thrown=!0,o(n))}),c.on("fn-end",function(){d&&!this.thrown&&l>0&&(l-=1)}),c.on("internal-error",function(t){i("ierr",[t,s.now(),!0])})},{}],3:[function(t,e,n){t("loader").features.ins=!0},{}],4:[function(t,e,n){function r(){_++,T=g.hash,this[u]=y.now()}function o(){_--,g.hash!==T&&i(0,!0);var t=y.now();this[h]=~~this[h]+t-this[u],this[d]=t}function i(t,e){E.emit("newURL",[""+g,e])}function a(t,e){t.on(e,function(){this[e]=y.now()})}var c="-start",s="-end",f="-body",u="fn"+c,d="fn"+s,p="cb"+c,l="cb"+s,h="jsTime",m="fetch",v="addEventListener",w=window,g=w.location,y=t("loader");if(w[v]&&y.xhrWrappable){var x=t(10),b=t(11),E=t(8),R=t(6),O=t(13),N=t(7),M=t(14),P=t(9),S=t("ee"),C=S.get("tracer");t(16),y.features.spa=!0;var T,_=0;S.on(u,r),S.on(p,r),S.on(d,o),S.on(l,o),S.buffer([u,d,"xhr-done","xhr-resolved"]),R.buffer([u]),O.buffer(["setTimeout"+s,"clearTimeout"+c,u]),M.buffer([u,"new-xhr","send-xhr"+c]),N.buffer([m+c,m+"-done",m+f+c,m+f+s]),E.buffer(["newURL"]),x.buffer([u]),b.buffer(["propagate",p,l,"executor-err","resolve"+c]),C.buffer([u,"no-"+u]),P.buffer(["new-jsonp","cb-start","jsonp-error","jsonp-end"]),a(M,"send-xhr"+c),a(S,"xhr-resolved"),a(S,"xhr-done"),a(N,m+c),a(N,m+"-done"),a(P,"new-jsonp"),a(P,"jsonp-end"),a(P,"cb-start"),E.on("pushState-end",i),E.on("replaceState-end",i),w[v]("hashchange",i,!0),w[v]("load",i,!0),w[v]("popstate",function(){i(0,_>1)},!0)}},{}],5:[function(t,e,n){function r(t){}if(window.performance&&window.performance.timing&&window.performance.getEntriesByType){var o=t("ee"),i=t("handle"),a=t(13),c=t(12),s="learResourceTimings",f="addEventListener",u="resourcetimingbufferfull",d="bstResource",p="resource",l="-start",h="-end",m="fn"+l,v="fn"+h,w="bstTimer",g="pushState",y=t("loader");y.features.stn=!0,t(8),"addEventListener"in window&&t(6);var x=NREUM.o.EV;o.on(m,function(t,e){var n=t[0];n instanceof x&&(this.bstStart=y.now())}),o.on(v,function(t,e){var n=t[0];n instanceof x&&i("bst",[n,e,this.bstStart,y.now()])}),a.on(m,function(t,e,n){this.bstStart=y.now(),this.bstType=n}),a.on(v,function(t,e){i(w,[e,this.bstStart,y.now(),this.bstType])}),c.on(m,function(){this.bstStart=y.now()}),c.on(v,function(t,e){i(w,[e,this.bstStart,y.now(),"requestAnimationFrame"])}),o.on(g+l,function(t){this.time=y.now(),this.startPath=location.pathname+location.hash}),o.on(g+h,function(t){i("bstHist",[location.pathname+location.hash,this.startPath,this.time])}),f in window.performance&&(window.performance["c"+s]?window.performance[f](u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["c"+s]()},!1):window.performance[f]("webkit"+u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["webkitC"+s]()},!1)),document[f]("scroll",r,{passive:!0}),document[f]("keypress",r,!1),document[f]("click",r,!1)}},{}],6:[function(t,e,n){function r(t){for(var e=t;e&&!e.hasOwnProperty(u);)e=Object.getPrototypeOf(e);e&&o(e)}function o(t){c.inPlace(t,[u,d],"-",i)}function i(t,e){return t[1]}var a=t("ee").get("events"),c=t("wrap-function")(a,!0),s=t("gos"),f=XMLHttpRequest,u="addEventListener",d="removeEventListener";e.exports=a,"getPrototypeOf"in Object?(r(document),r(window),r(f.prototype)):f.prototype.hasOwnProperty(u)&&(o(window),o(f.prototype)),a.on(u+"-start",function(t,e){var n=t[1],r=s(n,"nr@wrapped",function(){function t(){if("function"==typeof n.handleEvent)return n.handleEvent.apply(n,arguments)}var e={object:t,"function":n}[typeof n];return e?c(e,"fn-",null,e.name||"anonymous"):n});this.wrapped=t[1]=r}),a.on(d+"-start",function(t){t[1]=this.wrapped||t[1]})},{}],7:[function(t,e,n){function r(t,e,n){var r=t[e];"function"==typeof r&&(t[e]=function(){var t=i(arguments),e={};o.emit(n+"before-start",[t],e);var a;e[m]&&e[m].dt&&(a=e[m].dt);var c=r.apply(this,t);return o.emit(n+"start",[t,a],c),c.then(function(t){return o.emit(n+"end",[null,t],c),t},function(t){throw o.emit(n+"end",[t],c),t})})}var o=t("ee").get("fetch"),i=t(27),a=t(26);e.exports=o;var c=window,s="fetch-",f=s+"body-",u=["arrayBuffer","blob","json","text","formData"],d=c.Request,p=c.Response,l=c.fetch,h="prototype",m="nr@context";d&&p&&l&&(a(u,function(t,e){r(d[h],e,f),r(p[h],e,f)}),r(c,"fetch",s),o.on(s+"end",function(t,e){var n=this;if(e){var r=e.headers.get("content-length");null!==r&&(n.rxSize=r),o.emit(s+"done",[null,e],n)}else o.emit(s+"done",[t],n)}))},{}],8:[function(t,e,n){var r=t("ee").get("history"),o=t("wrap-function")(r);e.exports=r;var i=window.history&&window.history.constructor&&window.history.constructor.prototype,a=window.history;i&&i.pushState&&i.replaceState&&(a=i),o.inPlace(a,["pushState","replaceState"],"-")},{}],9:[function(t,e,n){function r(t){function e(){s.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}function n(){s.emit("jsonp-error",[],p),s.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}var r=t&&"string"==typeof t.nodeName&&"script"===t.nodeName.toLowerCase();if(r){var o="function"==typeof t.addEventListener;if(o){var a=i(t.src);if(a){var u=c(a),d="function"==typeof u.parent[u.key];if(d){var p={};f.inPlace(u.parent,[u.key],"cb-",p),t.addEventListener("load",e,!1),t.addEventListener("error",n,!1),s.emit("new-jsonp",[t.src],p)}}}}}function o(){return"addEventListener"in window}function i(t){var e=t.match(u);return e?e[1]:null}function a(t,e){var n=t.match(p),r=n[1],o=n[3];return o?a(o,e[r]):e[r]}function c(t){var e=t.match(d);return e&&e.length>=3?{key:e[2],parent:a(e[1],window)}:{key:t,parent:window}}var s=t("ee").get("jsonp"),f=t("wrap-function")(s);if(e.exports=s,o()){var u=/[?&](?:callback|cb)=([^&#]+)/,d=/(.*)\.([^.]+)/,p=/^(\w+)(\.|$)(.*)$/,l=["appendChild","insertBefore","replaceChild"];Node&&Node.prototype&&Node.prototype.appendChild?f.inPlace(Node.prototype,l,"dom-"):(f.inPlace(HTMLElement.prototype,l,"dom-"),f.inPlace(HTMLHeadElement.prototype,l,"dom-"),f.inPlace(HTMLBodyElement.prototype,l,"dom-")),s.on("dom-start",function(t){r(t[0])})}},{}],10:[function(t,e,n){var r=t("ee").get("mutation"),o=t("wrap-function")(r),i=NREUM.o.MO;e.exports=r,i&&(window.MutationObserver=function(t){return this instanceof i?new i(o(t,"fn-")):i.apply(this,arguments)},MutationObserver.prototype=i.prototype)},{}],11:[function(t,e,n){function r(t){var e=a.context(),n=c(t,"executor-",e),r=new f(n);return a.context(r).getCtx=function(){return e},a.emit("new-promise",[r,e],e),r}function o(t,e){return e}var i=t("wrap-function"),a=t("ee").get("promise"),c=i(a),s=t(26),f=NREUM.o.PR;e.exports=a,f&&(window.Promise=r,["all","race"].forEach(function(t){var e=f[t];f[t]=function(n){function r(t){return function(){a.emit("propagate",[null,!o],i),o=o||!t}}var o=!1;s(n,function(e,n){Promise.resolve(n).then(r("all"===t),r(!1))});var i=e.apply(f,arguments),c=f.resolve(i);return c}}),["resolve","reject"].forEach(function(t){var e=f[t];f[t]=function(t){var n=e.apply(f,arguments);return t!==n&&a.emit("propagate",[t,!0],n),n}}),f.prototype["catch"]=function(t){return this.then(null,t)},f.prototype=Object.create(f.prototype,{constructor:{value:r}}),s(Object.getOwnPropertyNames(f),function(t,e){try{r[e]=f[e]}catch(n){}}),a.on("executor-start",function(t){t[0]=c(t[0],"resolve-",this),t[1]=c(t[1],"resolve-",this)}),a.on("executor-err",function(t,e,n){t[1](n)}),c.inPlace(f.prototype,["then"],"then-",o),a.on("then-start",function(t,e){this.promise=e,t[0]=c(t[0],"cb-",this),t[1]=c(t[1],"cb-",this)}),a.on("then-end",function(t,e,n){this.nextPromise=n;var r=this.promise;a.emit("propagate",[r,!0],n)}),a.on("cb-end",function(t,e,n){a.emit("propagate",[n,!0],this.nextPromise)}),a.on("propagate",function(t,e,n){this.getCtx&&!e||(this.getCtx=function(){if(t instanceof Promise)var e=a.context(t);return e&&e.getCtx?e.getCtx():this})}),r.toString=function(){return""+f})},{}],12:[function(t,e,n){var r=t("ee").get("raf"),o=t("wrap-function")(r),i="equestAnimationFrame";e.exports=r,o.inPlace(window,["r"+i,"mozR"+i,"webkitR"+i,"msR"+i],"raf-"),r.on("raf-start",function(t){t[0]=o(t[0],"fn-")})},{}],13:[function(t,e,n){function r(t,e,n){t[0]=a(t[0],"fn-",null,n)}function o(t,e,n){this.method=n,this.timerDuration=isNaN(t[1])?0:+t[1],t[0]=a(t[0],"fn-",this,n)}var i=t("ee").get("timer"),a=t("wrap-function")(i),c="setTimeout",s="setInterval",f="clearTimeout",u="-start",d="-";e.exports=i,a.inPlace(window,[c,"setImmediate"],c+d),a.inPlace(window,[s],s+d),a.inPlace(window,[f,"clearImmediate"],f+d),i.on(s+u,r),i.on(c+u,o)},{}],14:[function(t,e,n){function r(t,e){d.inPlace(e,["onreadystatechange"],"fn-",c)}function o(){var t=this,e=u.context(t);t.readyState>3&&!e.resolved&&(e.resolved=!0,u.emit("xhr-resolved",[],t)),d.inPlace(t,g,"fn-",c)}function i(t){y.push(t),h&&(b?b.then(a):v?v(a):(E=-E,R.data=E))}function a(){for(var t=0;t<y.length;t++)r([],y[t]);y.length&&(y=[])}function c(t,e){return e}function s(t,e){for(var n in t)e[n]=t[n];return e}t(6);var f=t("ee"),u=f.get("xhr"),d=t("wrap-function")(u),p=NREUM.o,l=p.XHR,h=p.MO,m=p.PR,v=p.SI,w="readystatechange",g=["onload","onerror","onabort","onloadstart","onloadend","onprogress","ontimeout"],y=[];e.exports=u;var x=window.XMLHttpRequest=function(t){var e=new l(t);try{u.emit("new-xhr",[e],e),e.addEventListener(w,o,!1)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}return e};if(s(l,x),x.prototype=l.prototype,d.inPlace(x.prototype,["open","send"],"-xhr-",c),u.on("send-xhr-start",function(t,e){r(t,e),i(e)}),u.on("open-xhr-start",r),h){var b=m&&m.resolve();if(!v&&!m){var E=1,R=document.createTextNode(E);new h(a).observe(R,{characterData:!0})}}else f.on("fn-end",function(t){t[0]&&t[0].type===w||a()})},{}],15:[function(t,e,n){function r(t){if(!c(t))return null;var e=window.NREUM;if(!e.loader_config)return null;var n=(e.loader_config.accountID||"").toString()||null,r=(e.loader_config.agentID||"").toString()||null,f=(e.loader_config.trustKey||"").toString()||null;if(!n||!r)return null;var h=l.generateSpanId(),m=l.generateTraceId(),v=Date.now(),w={spanId:h,traceId:m,timestamp:v};return(t.sameOrigin||s(t)&&p())&&(w.traceContextParentHeader=o(h,m),w.traceContextStateHeader=i(h,v,n,r,f)),(t.sameOrigin&&!u()||!t.sameOrigin&&s(t)&&d())&&(w.newrelicHeader=a(h,m,v,n,r,f)),w}function o(t,e){return"00-"+e+"-"+t+"-01"}function i(t,e,n,r,o){var i=0,a="",c=1,s="",f="";return o+"@nr="+i+"-"+c+"-"+n+"-"+r+"-"+t+"-"+a+"-"+s+"-"+f+"-"+e}function a(t,e,n,r,o,i){var a="btoa"in window&&"function"==typeof window.btoa;if(!a)return null;var c={v:[0,1],d:{ty:"Browser",ac:r,ap:o,id:t,tr:e,ti:n}};return i&&r!==i&&(c.d.tk=i),btoa(JSON.stringify(c))}function c(t){return f()&&s(t)}function s(t){var e=!1,n={};if("init"in NREUM&&"distributed_tracing"in NREUM.init&&(n=NREUM.init.distributed_tracing),t.sameOrigin)e=!0;else if(n.allowed_origins instanceof Array)for(var r=0;r<n.allowed_origins.length;r++){var o=h(n.allowed_origins[r]);if(t.hostname===o.hostname&&t.protocol===o.protocol&&t.port===o.port){e=!0;break}}return e}function f(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.enabled}function u(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.exclude_newrelic_header}function d(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&NREUM.init.distributed_tracing.cors_use_newrelic_header!==!1}function p(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.cors_use_tracecontext_headers}var l=t(23),h=t(17);e.exports={generateTracePayload:r,shouldGenerateTrace:c}},{}],16:[function(t,e,n){function r(t){var e=this.params,n=this.metrics;if(!this.ended){this.ended=!0;for(var r=0;r<p;r++)t.removeEventListener(d[r],this.listener,!1);e.aborted||(n.duration=a.now()-this.startTime,this.loadCaptureCalled||4!==t.readyState?null==e.status&&(e.status=0):i(this,t),n.cbTime=this.cbTime,u.emit("xhr-done",[t],t),c("xhr",[e,n,this.startTime]))}}function o(t,e){var n=s(e),r=t.params;r.host=n.hostname+":"+n.port,r.pathname=n.pathname,t.parsedOrigin=s(e),t.sameOrigin=t.parsedOrigin.sameOrigin}function i(t,e){t.params.status=e.status;var n=v(e,t.lastSize);if(n&&(t.metrics.rxSize=n),t.sameOrigin){var r=e.getResponseHeader("X-NewRelic-App-Data");r&&(t.params.cat=r.split(", ").pop())}t.loadCaptureCalled=!0}var a=t("loader");if(a.xhrWrappable){var c=t("handle"),s=t(17),f=t(15).generateTracePayload,u=t("ee"),d=["load","error","abort","timeout"],p=d.length,l=t("id"),h=t(21),m=t(20),v=t(18),w=window.XMLHttpRequest;a.features.xhr=!0,t(14),t(7),u.on("new-xhr",function(t){var e=this;e.totalCbs=0,e.called=0,e.cbTime=0,e.end=r,e.ended=!1,e.xhrGuids={},e.lastSize=null,e.loadCaptureCalled=!1,t.addEventListener("load",function(n){i(e,t)},!1),h&&(h>34||h<10)||window.opera||t.addEventListener("progress",function(t){e.lastSize=t.loaded},!1)}),u.on("open-xhr-start",function(t){this.params={method:t[0]},o(this,t[1]),this.metrics={}}),u.on("open-xhr-end",function(t,e){"loader_config"in NREUM&&"xpid"in NREUM.loader_config&&this.sameOrigin&&e.setRequestHeader("X-NewRelic-ID",NREUM.loader_config.xpid);var n=f(this.parsedOrigin);if(n){var r=!1;n.newrelicHeader&&(e.setRequestHeader("newrelic",n.newrelicHeader),r=!0),n.traceContextParentHeader&&(e.setRequestHeader("traceparent",n.traceContextParentHeader),n.traceContextStateHeader&&e.setRequestHeader("tracestate",n.traceContextStateHeader),r=!0),r&&(this.dt=n)}}),u.on("send-xhr-start",function(t,e){var n=this.metrics,r=t[0],o=this;if(n&&r){var i=m(r);i&&(n.txSize=i)}this.startTime=a.now(),this.listener=function(t){try{"abort"!==t.type||o.loadCaptureCalled||(o.params.aborted=!0),("load"!==t.type||o.called===o.totalCbs&&(o.onloadCalled||"function"!=typeof e.onload))&&o.end(e)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}};for(var c=0;c<p;c++)e.addEventListener(d[c],this.listener,!1)}),u.on("xhr-cb-time",function(t,e,n){this.cbTime+=t,e?this.onloadCalled=!0:this.called+=1,this.called!==this.totalCbs||!this.onloadCalled&&"function"==typeof n.onload||this.end(n)}),u.on("xhr-load-added",function(t,e){var n=""+l(t)+!!e;this.xhrGuids&&!this.xhrGuids[n]&&(this.xhrGuids[n]=!0,this.totalCbs+=1)}),u.on("xhr-load-removed",function(t,e){var n=""+l(t)+!!e;this.xhrGuids&&this.xhrGuids[n]&&(delete this.xhrGuids[n],this.totalCbs-=1)}),u.on("addEventListener-end",function(t,e){e instanceof w&&"load"===t[0]&&u.emit("xhr-load-added",[t[1],t[2]],e)}),u.on("removeEventListener-end",function(t,e){e instanceof w&&"load"===t[0]&&u.emit("xhr-load-removed",[t[1],t[2]],e)}),u.on("fn-start",function(t,e,n){e instanceof w&&("onload"===n&&(this.onload=!0),("load"===(t[0]&&t[0].type)||this.onload)&&(this.xhrCbStart=a.now()))}),u.on("fn-end",function(t,e){this.xhrCbStart&&u.emit("xhr-cb-time",[a.now()-this.xhrCbStart,this.onload,e],e)}),u.on("fetch-before-start",function(t){function e(t,e){var n=!1;return e.newrelicHeader&&(t.set("newrelic",e.newrelicHeader),n=!0),e.traceContextParentHeader&&(t.set("traceparent",e.traceContextParentHeader),e.traceContextStateHeader&&t.set("tracestate",e.traceContextStateHeader),n=!0),n}var n,r=t[1]||{};"string"==typeof t[0]?n=t[0]:t[0]&&t[0].url&&(n=t[0].url),n&&(this.parsedOrigin=s(n),this.sameOrigin=this.parsedOrigin.sameOrigin);var o=f(this.parsedOrigin);if(o&&(o.newrelicHeader||o.traceContextParentHeader))if("string"==typeof t[0]){var i={};for(var a in r)i[a]=r[a];i.headers=new Headers(r.headers||{}),e(i.headers,o)&&(this.dt=o),t.length>1?t[1]=i:t.push(i)}else t[0]&&t[0].headers&&e(t[0].headers,o)&&(this.dt=o)})}},{}],17:[function(t,e,n){var r={};e.exports=function(t){if(t in r)return r[t];var e=document.createElement("a"),n=window.location,o={};e.href=t,o.port=e.port;var i=e.href.split("://");!o.port&&i[1]&&(o.port=i[1].split("/")[0].split("@").pop().split(":")[1]),o.port&&"0"!==o.port||(o.port="https"===i[0]?"443":"80"),o.hostname=e.hostname||n.hostname,o.pathname=e.pathname,o.protocol=i[0],"/"!==o.pathname.charAt(0)&&(o.pathname="/"+o.pathname);var a=!e.protocol||":"===e.protocol||e.protocol===n.protocol,c=e.hostname===document.domain&&e.port===n.port;return o.sameOrigin=a&&(!e.hostname||c),"/"===o.pathname&&(r[t]=o),o}},{}],18:[function(t,e,n){function r(t,e){var n=t.responseType;return"json"===n&&null!==e?e:"arraybuffer"===n||"blob"===n||"json"===n?o(t.response):"text"===n||""===n||void 0===n?o(t.responseText):void 0}var o=t(20);e.exports=r},{}],19:[function(t,e,n){function r(){}function o(t,e,n){return function(){return i(t,[f.now()].concat(c(arguments)),e?null:this,n),e?void 0:this}}var i=t("handle"),a=t(26),c=t(27),s=t("ee").get("tracer"),f=t("loader"),u=NREUM;"undefined"==typeof window.newrelic&&(newrelic=u);var d=["setPageViewName","setCustomAttribute","setErrorHandler","finished","addToTrace","inlineHit","addRelease"],p="api-",l=p+"ixn-";a(d,function(t,e){u[e]=o(p+e,!0,"api")}),u.addPageAction=o(p+"addPageAction",!0),u.setCurrentRouteName=o(p+"routeName",!0),e.exports=newrelic,u.interaction=function(){return(new r).get()};var h=r.prototype={createTracer:function(t,e){var n={},r=this,o="function"==typeof e;return i(l+"tracer",[f.now(),t,n],r),function(){if(s.emit((o?"":"no-")+"fn-start",[f.now(),r,o],n),o)try{return e.apply(this,arguments)}catch(t){throw s.emit("fn-err",[arguments,this,t],n),t}finally{s.emit("fn-end",[f.now()],n)}}}};a("actionText,setName,setAttribute,save,ignore,onEnd,getContext,end,get".split(","),function(t,e){h[e]=o(l+e)}),newrelic.noticeError=function(t,e){"string"==typeof t&&(t=new Error(t)),i("err",[t,f.now(),!1,e])}},{}],20:[function(t,e,n){e.exports=function(t){if("string"==typeof t&&t.length)return t.length;if("object"==typeof t){if("undefined"!=typeof ArrayBuffer&&t instanceof ArrayBuffer&&t.byteLength)return t.byteLength;if("undefined"!=typeof Blob&&t instanceof Blob&&t.size)return t.size;if(!("undefined"!=typeof FormData&&t instanceof FormData))try{return JSON.stringify(t).length}catch(e){return}}}},{}],21:[function(t,e,n){var r=0,o=navigator.userAgent.match(/Firefox[\/\s](\d+\.\d+)/);o&&(r=+o[1]),e.exports=r},{}],22:[function(t,e,n){function r(t,e){var n=t.getEntries();n.forEach(function(t){"first-paint"===t.name?d("timing",["fp",Math.floor(t.startTime)]):"first-contentful-paint"===t.name&&d("timing",["fcp",Math.floor(t.startTime)])})}function o(t,e){var n=t.getEntries();n.length>0&&d("lcp",[n[n.length-1]])}function i(t){t.getEntries().forEach(function(t){t.hadRecentInput||d("cls",[t])})}function a(t){if(t instanceof h&&!v){var e=Math.round(t.timeStamp),n={type:t.type};e<=p.now()?n.fid=p.now()-e:e>p.offset&&e<=Date.now()?(e-=p.offset,n.fid=p.now()-e):e=p.now(),v=!0,d("timing",["fi",e,n])}}function c(t){d("pageHide",[p.now(),t])}if(!("init"in NREUM&&"page_view_timing"in NREUM.init&&"enabled"in NREUM.init.page_view_timing&&NREUM.init.page_view_timing.enabled===!1)){var s,f,u,d=t("handle"),p=t("loader"),l=t(25),h=NREUM.o.EV;if("PerformanceObserver"in window&&"function"==typeof window.PerformanceObserver){s=new PerformanceObserver(r);try{s.observe({entryTypes:["paint"]})}catch(m){}f=new PerformanceObserver(o);try{f.observe({entryTypes:["largest-contentful-paint"]})}catch(m){}u=new PerformanceObserver(i);try{u.observe({type:"layout-shift",buffered:!0})}catch(m){}}if("addEventListener"in document){var v=!1,w=["click","keydown","mousedown","pointerdown","touchstart"];w.forEach(function(t){document.addEventListener(t,a,!1)})}l(c)}},{}],23:[function(t,e,n){function r(){function t(){return e?15&e[n++]:16*Math.random()|0}var e=null,n=0,r=window.crypto||window.msCrypto;r&&r.getRandomValues&&(e=r.getRandomValues(new Uint8Array(31)));for(var o,i="xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx",a="",c=0;c<i.length;c++)o=i[c],"x"===o?a+=t().toString(16):"y"===o?(o=3&t()|8,a+=o.toString(16)):a+=o;return a}function o(){return a(16)}function i(){return a(32)}function a(t){function e(){return n?15&n[r++]:16*Math.random()|0}var n=null,r=0,o=window.crypto||window.msCrypto;o&&o.getRandomValues&&Uint8Array&&(n=o.getRandomValues(new Uint8Array(31)));for(var i=[],a=0;a<t;a++)i.push(e().toString(16));return i.join("")}e.exports={generateUuid:r,generateSpanId:o,generateTraceId:i}},{}],24:[function(t,e,n){function r(t,e){if(!o)return!1;if(t!==o)return!1;if(!e)return!0;if(!i)return!1;for(var n=i.split("."),r=e.split("."),a=0;a<r.length;a++)if(r[a]!==n[a])return!1;return!0}var o=null,i=null,a=/Version\/(\S+)\s+Safari/;if(navigator.userAgent){var c=navigator.userAgent,s=c.match(a);s&&c.indexOf("Chrome")===-1&&c.indexOf("Chromium")===-1&&(o="Safari",i=s[1])}e.exports={agent:o,version:i,match:r}},{}],25:[function(t,e,n){function r(t){function e(){t(a&&document[a]?document[a]:document[o]?"hidden":"visible")}"addEventListener"in document&&i&&document.addEventListener(i,e,!1)}e.exports=r;var o,i,a;"undefined"!=typeof document.hidden?(o="hidden",i="visibilitychange",a="visibilityState"):"undefined"!=typeof document.msHidden?(o="msHidden",i="msvisibilitychange"):"undefined"!=typeof document.webkitHidden&&(o="webkitHidden",i="webkitvisibilitychange",a="webkitVisibilityState")},{}],26:[function(t,e,n){function r(t,e){var n=[],r="",i=0;for(r in t)o.call(t,r)&&(n[i]=e(r,t[r]),i+=1);return n}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],27:[function(t,e,n){function r(t,e,n){e||(e=0),"undefined"==typeof n&&(n=t?t.length:0);for(var r=-1,o=n-e||0,i=Array(o<0?0:o);++r<o;)i[r]=t[e+r];return i}e.exports=r},{}],28:[function(t,e,n){e.exports={exists:"undefined"!=typeof window.performance&&window.performance.timing&&"undefined"!=typeof window.performance.timing.navigationStart}},{}],ee:[function(t,e,n){function r(){}function o(t){function e(t){return t&&t instanceof r?t:t?s(t,c,i):i()}function n(n,r,o,i){if(!p.aborted||i){t&&t(n,r,o);for(var a=e(o),c=m(n),s=c.length,f=0;f<s;f++)c[f].apply(a,r);var d=u[y[n]];return d&&d.push([x,n,r,a]),a}}function l(t,e){g[t]=m(t).concat(e)}function h(t,e){var n=g[t];if(n)for(var r=0;r<n.length;r++)n[r]===e&&n.splice(r,1)}function m(t){return g[t]||[]}function v(t){return d[t]=d[t]||o(n)}function w(t,e){f(t,function(t,n){e=e||"feature",y[n]=e,e in u||(u[e]=[])})}var g={},y={},x={on:l,addEventListener:l,removeEventListener:h,emit:n,get:v,listeners:m,context:e,buffer:w,abort:a,aborted:!1};return x}function i(){return new r}function a(){(u.api||u.feature)&&(p.aborted=!0,u=p.backlog={})}var c="nr@context",s=t("gos"),f=t(26),u={},d={},p=e.exports=o();p.backlog=u},{}],gos:[function(t,e,n){function r(t,e,n){if(o.call(t,e))return t[e];var r=n();if(Object.defineProperty&&Object.keys)try{return Object.defineProperty(t,e,{value:r,writable:!0,enumerable:!1}),r}catch(i){}return t[e]=r,r}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],handle:[function(t,e,n){function r(t,e,n,r){o.buffer([t],r),o.emit(t,e,n)}var o=t("ee").get("handle");e.exports=r,r.ee=o},{}],id:[function(t,e,n){function r(t){var e=typeof t;return!t||"object"!==e&&"function"!==e?-1:t===window?0:a(t,i,function(){return o++})}var o=1,i="nr@id",a=t("gos");e.exports=r},{}],loader:[function(t,e,n){function r(){if(!E++){var t=b.info=NREUM.info,e=l.getElementsByTagName("script")[0];if(setTimeout(u.abort,3e4),!(t&&t.licenseKey&&t.applicationID&&e))return u.abort();f(y,function(e,n){t[e]||(t[e]=n)});var n=a();s("mark",["onload",n+b.offset],null,"api"),s("timing",["load",n]);var r=l.createElement("script");r.src="https://"+t.agent,e.parentNode.insertBefore(r,e)}}function o(){"complete"===l.readyState&&i()}function i(){s("mark",["domContent",a()+b.offset],null,"api")}function a(){return R.exists&&performance.now?Math.round(performance.now()):(c=Math.max((new Date).getTime(),c))-b.offset}var c=(new Date).getTime(),s=t("handle"),f=t(26),u=t("ee"),d=t(24),p=window,l=p.document,h="addEventListener",m="attachEvent",v=p.XMLHttpRequest,w=v&&v.prototype;NREUM.o={ST:setTimeout,SI:p.setImmediate,CT:clearTimeout,XHR:v,REQ:p.Request,EV:p.Event,PR:p.Promise,MO:p.MutationObserver};var g=""+location,y={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",agent:"js-agent.newrelic.com/nr-spa-1177.min.js"},x=v&&w&&w[h]&&!/CriOS/.test(navigator.userAgent),b=e.exports={offset:c,now:a,origin:g,features:{},xhrWrappable:x,userAgent:d};t(19),t(22),l[h]?(l[h]("DOMContentLoaded",i,!1),p[h]("load",r,!1)):(l[m]("onreadystatechange",o),p[m]("onload",r)),s("mark",["firstbyte",c],null,"api");var E=0,R=t(28)},{}],"wrap-function":[function(t,e,n){function r(t){return!(t&&t instanceof Function&&t.apply&&!t[a])}var o=t("ee"),i=t(27),a="nr@original",c=Object.prototype.hasOwnProperty,s=!1;e.exports=function(t,e){function n(t,e,n,o){function nrWrapper(){var r,a,c,s;try{a=this,r=i(arguments),c="function"==typeof n?n(r,a):n||{}}catch(f){p([f,"",[r,a,o],c])}u(e+"start",[r,a,o],c);try{return s=t.apply(a,r)}catch(d){throw u(e+"err",[r,a,d],c),d}finally{u(e+"end",[r,a,s],c)}}return r(t)?t:(e||(e=""),nrWrapper[a]=t,d(t,nrWrapper),nrWrapper)}function f(t,e,o,i){o||(o="");var a,c,s,f="-"===o.charAt(0);for(s=0;s<e.length;s++)c=e[s],a=t[c],r(a)||(t[c]=n(a,f?c+o:o,i,c))}function u(n,r,o){if(!s||e){var i=s;s=!0;try{t.emit(n,r,o,e)}catch(a){p([a,n,r,o])}s=i}}function d(t,e){if(Object.defineProperty&&Object.keys)try{var n=Object.keys(t);return n.forEach(function(n){Object.defineProperty(e,n,{get:function(){return t[n]},set:function(e){return t[n]=e,e}})}),e}catch(r){p([r])}for(var o in t)c.call(t,o)&&(e[o]=t[o]);return e}function p(e){try{t.emit("internal-error",e)}catch(n){}}return t||(t=o),n.inPlace=f,n.flag=a,n}},{}]},{},["loader",2,16,5,3,4]);
    ;NREUM.loader_config={accountID:"2095297",trustKey:"2038175",agentID:"916957862",licenseKey:"16e2192192",applicationID:"916957862"}
    ;NREUM.info={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",licenseKey:"16e2192192",applicationID:"916957862",sa:1}
</script>
    <script src="https://cdn.cookielaw.org/scripttemplates/otSDKStub.js" type="text/javascript" charset="UTF-8" data-domain-script="dada9834-04e2-4bcd-b5ba-cae44d498dfa"></script>
    <script type="text/javascript">
        function getCookie(name) {
            var value = "; " + document.cookie;
            var parts = value.split("; " + name + "=");
            if (parts.length == 2) {
                return true;
            }
        }
            function OptanonWrapper() {
                var optCookie = getCookie("OptanonAlertBoxClosed");
                var isActiveCategory = "false";
                var isFoundCookieBox = "false";
                if(isActiveCategory == 'false' && isFoundCookieBox == 'false' && optCookie) {
                    OneTrust.InsertScript('/pb-assets/adobe/prod/adobedtm-1616606406233.js', 'head', null, null, '2');
                }
            }
    </script>


<noscript>
    <style>.requiresJS {
        display: none
    }

    .dartAd {
        display: block !important
    }</style>
</noscript>
        <script type="text/javascript" src="/wro/o4ny~selected-legacy.js"></script>
        <link rel="stylesheet" type="text/css" href="/wro/o4ny~selected-legacy.css">
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Open+Sans">
        <script>window.MSInputMethodContext && document.documentMode && document.write('<script src="https://cdn.jsdelivr.net/gh/nuxodin/ie11CustomProperties@4.1.0/ie11CustomProperties.min.js"><\x2fscript>');</script><link rel="stylesheet" href="/products/marlin/cell/releasedAssets/css/build-b26f3342b722959c7931.css"><link rel="stylesheet" href="/products/marlin/cell/releasedAssets/css/print-b26f3342b722959c7931.css" media="print"><link rel="stylesheet" href="/products/marlin/cell/releasedAssets/css/themes/theme-cell.css">







        
        
        <script class="optanon-category-4" type="text/plain" async="" src="https://www.youtube.com/iframe_api"></script>
        
    





<!---->
<meta http-equiv="X-UA-Compatible" content="IE=edge">












<script src="https://cdn.cookielaw.org/scripttemplates/202411.2.0/otBannerSdk.js" async="" type="text/javascript"></script><style id="onetrust-style">#onetrust-banner-sdk .onetrust-vendors-list-handler{cursor:pointer;color:#1f96db;font-size:inherit;font-weight:bold;text-decoration:none;margin-left:5px}#onetrust-banner-sdk .onetrust-vendors-list-handler:hover{color:#1f96db}#onetrust-banner-sdk:focus{outline:2px solid #000;outline-offset:-2px}#onetrust-banner-sdk a:focus{outline:2px solid #000}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{outline-offset:1px}#onetrust-banner-sdk.ot-bnr-w-logo .ot-bnr-logo{height:64px;width:64px}#onetrust-banner-sdk .ot-tcf2-vendor-count.ot-text-bold{font-weight:bold}#onetrust-banner-sdk .ot-close-icon,#onetrust-pc-sdk .ot-close-icon,#ot-sync-ntfy .ot-close-icon{background-size:contain;background-repeat:no-repeat;background-position:center;height:12px;width:12px}#onetrust-banner-sdk .powered-by-logo,#onetrust-banner-sdk .ot-pc-footer-logo a,#onetrust-pc-sdk .powered-by-logo,#onetrust-pc-sdk .ot-pc-footer-logo a,#ot-sync-ntfy .powered-by-logo,#ot-sync-ntfy .ot-pc-footer-logo a{background-size:contain;background-repeat:no-repeat;background-position:center;height:25px;width:152px;display:block;text-decoration:none;font-size:.75em}#onetrust-banner-sdk .powered-by-logo:hover,#onetrust-banner-sdk .ot-pc-footer-logo a:hover,#onetrust-pc-sdk .powered-by-logo:hover,#onetrust-pc-sdk .ot-pc-footer-logo a:hover,#ot-sync-ntfy .powered-by-logo:hover,#ot-sync-ntfy .ot-pc-footer-logo a:hover{color:#565656}#onetrust-banner-sdk h3 *,#onetrust-banner-sdk h4 *,#onetrust-banner-sdk h6 *,#onetrust-banner-sdk button *,#onetrust-banner-sdk a[data-parent-id] *,#onetrust-pc-sdk h3 *,#onetrust-pc-sdk h4 *,#onetrust-pc-sdk h6 *,#onetrust-pc-sdk button *,#onetrust-pc-sdk a[data-parent-id] *,#ot-sync-ntfy h3 *,#ot-sync-ntfy h4 *,#ot-sync-ntfy h6 *,#ot-sync-ntfy button *,#ot-sync-ntfy a[data-parent-id] *{font-size:inherit;font-weight:inherit;color:inherit}#onetrust-banner-sdk .ot-hide,#onetrust-pc-sdk .ot-hide,#ot-sync-ntfy .ot-hide{display:none !important}#onetrust-banner-sdk button.ot-link-btn:hover,#onetrust-pc-sdk button.ot-link-btn:hover,#ot-sync-ntfy button.ot-link-btn:hover{text-decoration:underline;opacity:1}#onetrust-pc-sdk .ot-sdk-row .ot-sdk-column{padding:0}#onetrust-pc-sdk .ot-sdk-container{padding-right:0}#onetrust-pc-sdk .ot-sdk-row{flex-direction:initial;width:100%}#onetrust-pc-sdk [type=checkbox]:checked,#onetrust-pc-sdk [type=checkbox]:not(:checked){pointer-events:initial}#onetrust-pc-sdk [type=checkbox]:disabled+label::before,#onetrust-pc-sdk [type=checkbox]:disabled+label:after,#onetrust-pc-sdk [type=checkbox]:disabled+label{pointer-events:none;opacity:.8}#onetrust-pc-sdk #vendor-list-content{transform:translate3d(0, 0, 0)}#onetrust-pc-sdk li input[type=checkbox]{z-index:1}#onetrust-pc-sdk li .ot-checkbox label{z-index:2}#onetrust-pc-sdk li .ot-checkbox input[type=checkbox]{height:auto;width:auto}#onetrust-pc-sdk li .host-title a,#onetrust-pc-sdk li .ot-host-name a,#onetrust-pc-sdk li .accordion-text,#onetrust-pc-sdk li .ot-acc-txt{z-index:2;position:relative}#onetrust-pc-sdk input{margin:3px .1ex}#onetrust-pc-sdk .pc-logo,#onetrust-pc-sdk .ot-pc-logo{height:60px;width:180px;background-position:center;background-size:contain;background-repeat:no-repeat;display:inline-flex;justify-content:center;align-items:center}#onetrust-pc-sdk .pc-logo img,#onetrust-pc-sdk .ot-pc-logo img{max-height:100%;max-width:100%}#onetrust-pc-sdk .screen-reader-only,#onetrust-pc-sdk .ot-scrn-rdr,.ot-sdk-cookie-policy .screen-reader-only,.ot-sdk-cookie-policy .ot-scrn-rdr{border:0;clip:rect(0 0 0 0);height:1px;margin:-1px;overflow:hidden;padding:0;position:absolute;width:1px}#onetrust-pc-sdk.ot-fade-in,.onetrust-pc-dark-filter.ot-fade-in,#onetrust-banner-sdk.ot-fade-in{animation-name:onetrust-fade-in;animation-duration:400ms;animation-timing-function:ease-in-out}#onetrust-pc-sdk.ot-hide{display:none !important}.onetrust-pc-dark-filter.ot-hide{display:none !important}#ot-sdk-btn.ot-sdk-show-settings,#ot-sdk-btn.optanon-show-settings{color:#fff;background-color:#468254;height:auto;white-space:normal;word-wrap:break-word;padding:.8em 2em;font-size:.8em;line-height:1.2;cursor:pointer;-moz-transition:.1s ease;-o-transition:.1s ease;-webkit-transition:1s ease;transition:.1s ease}#ot-sdk-btn.ot-sdk-show-settings:hover,#ot-sdk-btn.optanon-show-settings:hover{color:#fff;background-color:#2c6415}#ot-sdk-btn.ot-sdk-show-settings:active,#ot-sdk-btn.optanon-show-settings:active{color:#fff;background-color:#2c6415;border:1px solid rgba(162,192,169,.5)}.onetrust-pc-dark-filter{background:rgba(0,0,0,.5);z-index:2147483646;width:100%;height:100%;overflow:hidden;position:fixed;top:0;bottom:0;left:0}@keyframes onetrust-fade-in{0%{opacity:0}100%{opacity:1}}.ot-cookie-label{text-decoration:underline}@media only screen and (min-width: 426px)and (max-width: 896px)and (orientation: landscape){#onetrust-pc-sdk p{font-size:.75em}}#onetrust-banner-sdk .banner-option-input:focus+label{outline:1px solid #000;outline-style:auto}.category-vendors-list-handler+a:focus,.category-vendors-list-handler+a:focus-visible{outline:2px solid #000}#onetrust-pc-sdk .ot-userid-title{margin-top:10px}#onetrust-pc-sdk .ot-userid-title>span,#onetrust-pc-sdk .ot-userid-timestamp>span{font-weight:700}#onetrust-pc-sdk .ot-userid-desc{font-style:italic}#onetrust-pc-sdk .ot-host-desc a{pointer-events:initial}#onetrust-pc-sdk .ot-ven-hdr>p a{position:relative;z-index:2;pointer-events:initial}#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info a,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info a{margin-right:auto}#onetrust-pc-sdk .ot-pc-footer-logo img{width:136px;height:16px}#onetrust-pc-sdk .ot-pur-vdr-count{font-weight:400;font-size:.7rem;padding-top:3px;display:block}#onetrust-banner-sdk .ot-optout-signal,#onetrust-pc-sdk .ot-optout-signal{border:1px solid #32ae88;border-radius:3px;padding:5px;margin-bottom:10px;background-color:#f9fffa;font-size:.85rem;line-height:2}#onetrust-banner-sdk .ot-optout-signal .ot-optout-icon,#onetrust-pc-sdk .ot-optout-signal .ot-optout-icon{display:inline;margin-right:5px}#onetrust-banner-sdk .ot-optout-signal svg,#onetrust-pc-sdk .ot-optout-signal svg{height:20px;width:30px;transform:scale(0.5)}#onetrust-banner-sdk .ot-optout-signal svg path,#onetrust-pc-sdk .ot-optout-signal svg path{fill:#32ae88}#onetrust-consent-sdk .ot-general-modal{overflow:hidden;position:fixed;margin:0 auto;top:50%;left:50%;width:40%;padding:1.5rem;max-width:575px;min-width:575px;z-index:2147483647;border-radius:2.5px;transform:translate(-50%, -50%)}#onetrust-consent-sdk .ot-signature-health-group{margin-top:1rem;padding-left:1.25rem;padding-right:1.25rem;margin-bottom:.625rem;width:calc(100% - 2.5rem)}#onetrust-consent-sdk .ot-signature-health-group .ot-signature-health-form{gap:.5rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-health-form{width:70%;gap:.35rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-input{height:38px;padding:6px 10px;background-color:#fff;border:1px solid #d1d1d1;border-radius:4px;box-shadow:none;box-sizing:border-box}#onetrust-consent-sdk .ot-signature-health .ot-signature-subtitle{font-size:1.125rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-group-title{font-size:1.25rem;font-weight:bold}#onetrust-consent-sdk .ot-signature-health,#onetrust-consent-sdk .ot-signature-health-group{display:flex;flex-direction:column;gap:1rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-cont,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-cont{display:flex;flex-direction:column;gap:.25rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-paragraph,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-paragraph{margin:0;line-height:20px;font-size:max(14px,.875rem)}#onetrust-consent-sdk .ot-signature-health .ot-health-signature-error,#onetrust-consent-sdk .ot-signature-health-group .ot-health-signature-error{color:#4d4d4d;font-size:min(12px,.75rem)}#onetrust-consent-sdk .ot-signature-health .ot-signature-buttons-cont,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-buttons-cont{margin-top:max(.75rem,2%);gap:1rem;display:flex;justify-content:flex-end}#onetrust-consent-sdk .ot-signature-health .ot-signature-button,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-button{flex:1;height:auto;color:#fff;cursor:pointer;line-height:1.2;min-width:125px;font-weight:600;font-size:.813em;border-radius:2px;padding:12px 10px;white-space:normal;word-wrap:break-word;word-break:break-word;background-color:#68b631;border:2px solid #68b631}#onetrust-consent-sdk .ot-signature-health .ot-signature-button.reject,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-button.reject{background-color:#fff}#onetrust-consent-sdk .ot-input-field-cont{display:flex;flex-direction:column;gap:.5rem}#onetrust-consent-sdk .ot-input-field-cont .ot-signature-input{width:65%}#onetrust-consent-sdk .ot-signature-health-form{display:flex;flex-direction:column}#onetrust-consent-sdk .ot-signature-health-form .ot-signature-label{margin-bottom:0;line-height:20px;font-size:max(14px,.875rem)}@media only screen and (max-width: 600px){#onetrust-consent-sdk .ot-general-modal{min-width:100%}#onetrust-consent-sdk .ot-signature-health .ot-signature-health-form{width:100%}#onetrust-consent-sdk .ot-input-field-cont .ot-signature-input{width:100%}}#onetrust-banner-sdk,#onetrust-pc-sdk,#ot-sdk-cookie-policy,#ot-sync-ntfy{font-size:16px}#onetrust-banner-sdk *,#onetrust-banner-sdk ::after,#onetrust-banner-sdk ::before,#onetrust-pc-sdk *,#onetrust-pc-sdk ::after,#onetrust-pc-sdk ::before,#ot-sdk-cookie-policy *,#ot-sdk-cookie-policy ::after,#ot-sdk-cookie-policy ::before,#ot-sync-ntfy *,#ot-sync-ntfy ::after,#ot-sync-ntfy ::before{-webkit-box-sizing:content-box;-moz-box-sizing:content-box;box-sizing:content-box}#onetrust-banner-sdk div,#onetrust-banner-sdk span,#onetrust-banner-sdk h1,#onetrust-banner-sdk h2,#onetrust-banner-sdk h3,#onetrust-banner-sdk h4,#onetrust-banner-sdk h5,#onetrust-banner-sdk h6,#onetrust-banner-sdk p,#onetrust-banner-sdk img,#onetrust-banner-sdk svg,#onetrust-banner-sdk button,#onetrust-banner-sdk section,#onetrust-banner-sdk a,#onetrust-banner-sdk label,#onetrust-banner-sdk input,#onetrust-banner-sdk ul,#onetrust-banner-sdk li,#onetrust-banner-sdk nav,#onetrust-banner-sdk table,#onetrust-banner-sdk thead,#onetrust-banner-sdk tr,#onetrust-banner-sdk td,#onetrust-banner-sdk tbody,#onetrust-banner-sdk .ot-main-content,#onetrust-banner-sdk .ot-toggle,#onetrust-banner-sdk #ot-content,#onetrust-banner-sdk #ot-pc-content,#onetrust-banner-sdk .checkbox,#onetrust-pc-sdk div,#onetrust-pc-sdk span,#onetrust-pc-sdk h1,#onetrust-pc-sdk h2,#onetrust-pc-sdk h3,#onetrust-pc-sdk h4,#onetrust-pc-sdk h5,#onetrust-pc-sdk h6,#onetrust-pc-sdk p,#onetrust-pc-sdk img,#onetrust-pc-sdk svg,#onetrust-pc-sdk button,#onetrust-pc-sdk section,#onetrust-pc-sdk a,#onetrust-pc-sdk label,#onetrust-pc-sdk input,#onetrust-pc-sdk ul,#onetrust-pc-sdk li,#onetrust-pc-sdk nav,#onetrust-pc-sdk table,#onetrust-pc-sdk thead,#onetrust-pc-sdk tr,#onetrust-pc-sdk td,#onetrust-pc-sdk tbody,#onetrust-pc-sdk .ot-main-content,#onetrust-pc-sdk .ot-toggle,#onetrust-pc-sdk #ot-content,#onetrust-pc-sdk #ot-pc-content,#onetrust-pc-sdk .checkbox,#ot-sdk-cookie-policy div,#ot-sdk-cookie-policy span,#ot-sdk-cookie-policy h1,#ot-sdk-cookie-policy h2,#ot-sdk-cookie-policy h3,#ot-sdk-cookie-policy h4,#ot-sdk-cookie-policy h5,#ot-sdk-cookie-policy h6,#ot-sdk-cookie-policy p,#ot-sdk-cookie-policy img,#ot-sdk-cookie-policy svg,#ot-sdk-cookie-policy button,#ot-sdk-cookie-policy section,#ot-sdk-cookie-policy a,#ot-sdk-cookie-policy label,#ot-sdk-cookie-policy input,#ot-sdk-cookie-policy ul,#ot-sdk-cookie-policy li,#ot-sdk-cookie-policy nav,#ot-sdk-cookie-policy table,#ot-sdk-cookie-policy thead,#ot-sdk-cookie-policy tr,#ot-sdk-cookie-policy td,#ot-sdk-cookie-policy tbody,#ot-sdk-cookie-policy .ot-main-content,#ot-sdk-cookie-policy .ot-toggle,#ot-sdk-cookie-policy #ot-content,#ot-sdk-cookie-policy #ot-pc-content,#ot-sdk-cookie-policy .checkbox,#ot-sync-ntfy div,#ot-sync-ntfy span,#ot-sync-ntfy h1,#ot-sync-ntfy h2,#ot-sync-ntfy h3,#ot-sync-ntfy h4,#ot-sync-ntfy h5,#ot-sync-ntfy h6,#ot-sync-ntfy p,#ot-sync-ntfy img,#ot-sync-ntfy svg,#ot-sync-ntfy button,#ot-sync-ntfy section,#ot-sync-ntfy a,#ot-sync-ntfy label,#ot-sync-ntfy input,#ot-sync-ntfy ul,#ot-sync-ntfy li,#ot-sync-ntfy nav,#ot-sync-ntfy table,#ot-sync-ntfy thead,#ot-sync-ntfy tr,#ot-sync-ntfy td,#ot-sync-ntfy tbody,#ot-sync-ntfy .ot-main-content,#ot-sync-ntfy .ot-toggle,#ot-sync-ntfy #ot-content,#ot-sync-ntfy #ot-pc-content,#ot-sync-ntfy .checkbox{font-family:inherit;font-weight:normal;-webkit-font-smoothing:auto;letter-spacing:normal;line-height:normal;padding:0;margin:0;height:auto;min-height:0;max-height:none;width:auto;min-width:0;max-width:none;border-radius:0;border:none;clear:none;float:none;position:static;bottom:auto;left:auto;right:auto;top:auto;text-align:left;text-decoration:none;text-indent:0;text-shadow:none;text-transform:none;white-space:normal;background:none;overflow:visible;vertical-align:baseline;visibility:visible;z-index:auto;box-shadow:none}#onetrust-banner-sdk label:before,#onetrust-banner-sdk label:after,#onetrust-banner-sdk .checkbox:after,#onetrust-banner-sdk .checkbox:before,#onetrust-pc-sdk label:before,#onetrust-pc-sdk label:after,#onetrust-pc-sdk .checkbox:after,#onetrust-pc-sdk .checkbox:before,#ot-sdk-cookie-policy label:before,#ot-sdk-cookie-policy label:after,#ot-sdk-cookie-policy .checkbox:after,#ot-sdk-cookie-policy .checkbox:before,#ot-sync-ntfy label:before,#ot-sync-ntfy label:after,#ot-sync-ntfy .checkbox:after,#ot-sync-ntfy .checkbox:before{content:"";content:none}#onetrust-banner-sdk .ot-sdk-container,#onetrust-pc-sdk .ot-sdk-container,#ot-sdk-cookie-policy .ot-sdk-container{position:relative;width:100%;max-width:100%;margin:0 auto;padding:0 20px;box-sizing:border-box}#onetrust-banner-sdk .ot-sdk-column,#onetrust-banner-sdk .ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-column,#onetrust-pc-sdk .ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-column,#ot-sdk-cookie-policy .ot-sdk-columns{width:100%;float:left;box-sizing:border-box;padding:0;display:initial}@media(min-width: 400px){#onetrust-banner-sdk .ot-sdk-container,#onetrust-pc-sdk .ot-sdk-container,#ot-sdk-cookie-policy .ot-sdk-container{width:90%;padding:0}}@media(min-width: 550px){#onetrust-banner-sdk .ot-sdk-container,#onetrust-pc-sdk .ot-sdk-container,#ot-sdk-cookie-policy .ot-sdk-container{width:100%}#onetrust-banner-sdk .ot-sdk-column,#onetrust-banner-sdk .ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-column,#onetrust-pc-sdk .ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-column,#ot-sdk-cookie-policy .ot-sdk-columns{margin-left:4%}#onetrust-banner-sdk .ot-sdk-column:first-child,#onetrust-banner-sdk .ot-sdk-columns:first-child,#onetrust-pc-sdk .ot-sdk-column:first-child,#onetrust-pc-sdk .ot-sdk-columns:first-child,#ot-sdk-cookie-policy .ot-sdk-column:first-child,#ot-sdk-cookie-policy .ot-sdk-columns:first-child{margin-left:0}#onetrust-banner-sdk .ot-sdk-two.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-two.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-two.ot-sdk-columns{width:13.3333333333%}#onetrust-banner-sdk .ot-sdk-three.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-three.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-three.ot-sdk-columns{width:22%}#onetrust-banner-sdk .ot-sdk-four.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-four.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-four.ot-sdk-columns{width:30.6666666667%}#onetrust-banner-sdk .ot-sdk-eight.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-eight.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-eight.ot-sdk-columns{width:65.3333333333%}#onetrust-banner-sdk .ot-sdk-nine.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-nine.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-nine.ot-sdk-columns{width:74%}#onetrust-banner-sdk .ot-sdk-ten.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-ten.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-ten.ot-sdk-columns{width:82.6666666667%}#onetrust-banner-sdk .ot-sdk-eleven.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-eleven.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-eleven.ot-sdk-columns{width:91.3333333333%}#onetrust-banner-sdk .ot-sdk-twelve.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-twelve.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-twelve.ot-sdk-columns{width:100%;margin-left:0}}#onetrust-banner-sdk h1,#onetrust-banner-sdk h2,#onetrust-banner-sdk h3,#onetrust-banner-sdk h4,#onetrust-banner-sdk h5,#onetrust-banner-sdk h6,#onetrust-pc-sdk h1,#onetrust-pc-sdk h2,#onetrust-pc-sdk h3,#onetrust-pc-sdk h4,#onetrust-pc-sdk h5,#onetrust-pc-sdk h6,#ot-sdk-cookie-policy h1,#ot-sdk-cookie-policy h2,#ot-sdk-cookie-policy h3,#ot-sdk-cookie-policy h4,#ot-sdk-cookie-policy h5,#ot-sdk-cookie-policy h6{margin-top:0;font-weight:600;font-family:inherit}#onetrust-banner-sdk h1,#onetrust-pc-sdk h1,#ot-sdk-cookie-policy h1{font-size:1.5rem;line-height:1.2}#onetrust-banner-sdk h2,#onetrust-pc-sdk h2,#ot-sdk-cookie-policy h2{font-size:1.5rem;line-height:1.25}#onetrust-banner-sdk h3,#onetrust-pc-sdk h3,#ot-sdk-cookie-policy h3{font-size:1.5rem;line-height:1.3}#onetrust-banner-sdk h4,#onetrust-pc-sdk h4,#ot-sdk-cookie-policy h4{font-size:1.5rem;line-height:1.35}#onetrust-banner-sdk h5,#onetrust-pc-sdk h5,#ot-sdk-cookie-policy h5{font-size:1.5rem;line-height:1.5}#onetrust-banner-sdk h6,#onetrust-pc-sdk h6,#ot-sdk-cookie-policy h6{font-size:1.5rem;line-height:1.6}@media(min-width: 550px){#onetrust-banner-sdk h1,#onetrust-pc-sdk h1,#ot-sdk-cookie-policy h1{font-size:1.5rem}#onetrust-banner-sdk h2,#onetrust-pc-sdk h2,#ot-sdk-cookie-policy h2{font-size:1.5rem}#onetrust-banner-sdk h3,#onetrust-pc-sdk h3,#ot-sdk-cookie-policy h3{font-size:1.5rem}#onetrust-banner-sdk h4,#onetrust-pc-sdk h4,#ot-sdk-cookie-policy h4{font-size:1.5rem}#onetrust-banner-sdk h5,#onetrust-pc-sdk h5,#ot-sdk-cookie-policy h5{font-size:1.5rem}#onetrust-banner-sdk h6,#onetrust-pc-sdk h6,#ot-sdk-cookie-policy h6{font-size:1.5rem}}#onetrust-banner-sdk p,#onetrust-pc-sdk p,#ot-sdk-cookie-policy p{margin:0 0 1em 0;font-family:inherit;line-height:normal}#onetrust-banner-sdk a,#onetrust-pc-sdk a,#ot-sdk-cookie-policy a{color:#565656;text-decoration:underline}#onetrust-banner-sdk a:hover,#onetrust-pc-sdk a:hover,#ot-sdk-cookie-policy a:hover{color:#565656;text-decoration:none}#onetrust-banner-sdk .ot-sdk-button,#onetrust-banner-sdk button,#onetrust-pc-sdk .ot-sdk-button,#onetrust-pc-sdk button,#ot-sdk-cookie-policy .ot-sdk-button,#ot-sdk-cookie-policy button{margin-bottom:1rem;font-family:inherit}#onetrust-banner-sdk .ot-sdk-button,#onetrust-banner-sdk button,#onetrust-pc-sdk .ot-sdk-button,#onetrust-pc-sdk button,#ot-sdk-cookie-policy .ot-sdk-button,#ot-sdk-cookie-policy button{display:inline-block;height:38px;padding:0 30px;color:#555;text-align:center;font-size:.9em;font-weight:400;line-height:38px;letter-spacing:.01em;text-decoration:none;white-space:nowrap;background-color:rgba(0,0,0,0);border-radius:2px;border:1px solid #bbb;cursor:pointer;box-sizing:border-box}#onetrust-banner-sdk .ot-sdk-button:hover,#onetrust-banner-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):hover,#onetrust-banner-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):focus,#onetrust-pc-sdk .ot-sdk-button:hover,#onetrust-pc-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):hover,#onetrust-pc-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):focus,#ot-sdk-cookie-policy .ot-sdk-button:hover,#ot-sdk-cookie-policy :not(.ot-leg-btn-container)>button:not(.ot-link-btn):hover,#ot-sdk-cookie-policy :not(.ot-leg-btn-container)>button:not(.ot-link-btn):focus{color:#333;border-color:#888;opacity:.9}#onetrust-banner-sdk .ot-sdk-button:focus,#onetrust-banner-sdk :not(.ot-leg-btn-container)>button:focus,#onetrust-pc-sdk .ot-sdk-button:focus,#onetrust-pc-sdk :not(.ot-leg-btn-container)>button:focus,#ot-sdk-cookie-policy .ot-sdk-button:focus,#ot-sdk-cookie-policy :not(.ot-leg-btn-container)>button:focus{outline:2px solid #000}#onetrust-banner-sdk .ot-sdk-button.ot-sdk-button-primary,#onetrust-banner-sdk button.ot-sdk-button-primary,#onetrust-banner-sdk input[type=submit].ot-sdk-button-primary,#onetrust-banner-sdk input[type=reset].ot-sdk-button-primary,#onetrust-banner-sdk input[type=button].ot-sdk-button-primary,#onetrust-pc-sdk .ot-sdk-button.ot-sdk-button-primary,#onetrust-pc-sdk button.ot-sdk-button-primary,#onetrust-pc-sdk input[type=submit].ot-sdk-button-primary,#onetrust-pc-sdk input[type=reset].ot-sdk-button-primary,#onetrust-pc-sdk input[type=button].ot-sdk-button-primary,#ot-sdk-cookie-policy .ot-sdk-button.ot-sdk-button-primary,#ot-sdk-cookie-policy button.ot-sdk-button-primary,#ot-sdk-cookie-policy input[type=submit].ot-sdk-button-primary,#ot-sdk-cookie-policy input[type=reset].ot-sdk-button-primary,#ot-sdk-cookie-policy input[type=button].ot-sdk-button-primary{color:#fff;background-color:#33c3f0;border-color:#33c3f0}#onetrust-banner-sdk .ot-sdk-button.ot-sdk-button-primary:hover,#onetrust-banner-sdk button.ot-sdk-button-primary:hover,#onetrust-banner-sdk input[type=submit].ot-sdk-button-primary:hover,#onetrust-banner-sdk input[type=reset].ot-sdk-button-primary:hover,#onetrust-banner-sdk input[type=button].ot-sdk-button-primary:hover,#onetrust-banner-sdk .ot-sdk-button.ot-sdk-button-primary:focus,#onetrust-banner-sdk button.ot-sdk-button-primary:focus,#onetrust-banner-sdk input[type=submit].ot-sdk-button-primary:focus,#onetrust-banner-sdk input[type=reset].ot-sdk-button-primary:focus,#onetrust-banner-sdk input[type=button].ot-sdk-button-primary:focus,#onetrust-pc-sdk .ot-sdk-button.ot-sdk-button-primary:hover,#onetrust-pc-sdk button.ot-sdk-button-primary:hover,#onetrust-pc-sdk input[type=submit].ot-sdk-button-primary:hover,#onetrust-pc-sdk input[type=reset].ot-sdk-button-primary:hover,#onetrust-pc-sdk input[type=button].ot-sdk-button-primary:hover,#onetrust-pc-sdk .ot-sdk-button.ot-sdk-button-primary:focus,#onetrust-pc-sdk button.ot-sdk-button-primary:focus,#onetrust-pc-sdk input[type=submit].ot-sdk-button-primary:focus,#onetrust-pc-sdk input[type=reset].ot-sdk-button-primary:focus,#onetrust-pc-sdk input[type=button].ot-sdk-button-primary:focus,#ot-sdk-cookie-policy .ot-sdk-button.ot-sdk-button-primary:hover,#ot-sdk-cookie-policy button.ot-sdk-button-primary:hover,#ot-sdk-cookie-policy input[type=submit].ot-sdk-button-primary:hover,#ot-sdk-cookie-policy input[type=reset].ot-sdk-button-primary:hover,#ot-sdk-cookie-policy input[type=button].ot-sdk-button-primary:hover,#ot-sdk-cookie-policy .ot-sdk-button.ot-sdk-button-primary:focus,#ot-sdk-cookie-policy button.ot-sdk-button-primary:focus,#ot-sdk-cookie-policy input[type=submit].ot-sdk-button-primary:focus,#ot-sdk-cookie-policy input[type=reset].ot-sdk-button-primary:focus,#ot-sdk-cookie-policy input[type=button].ot-sdk-button-primary:focus{color:#fff;background-color:#1eaedb;border-color:#1eaedb}#onetrust-banner-sdk input[type=text],#onetrust-pc-sdk input[type=text],#ot-sdk-cookie-policy input[type=text]{height:38px;padding:6px 10px;background-color:#fff;border:1px solid #d1d1d1;border-radius:4px;box-shadow:none;box-sizing:border-box}#onetrust-banner-sdk input[type=text],#onetrust-pc-sdk input[type=text],#ot-sdk-cookie-policy input[type=text]{-webkit-appearance:none;-moz-appearance:none;appearance:none}#onetrust-banner-sdk input[type=text]:focus,#onetrust-pc-sdk input[type=text]:focus,#ot-sdk-cookie-policy input[type=text]:focus{border:1px solid #000;outline:0}#onetrust-banner-sdk label,#onetrust-pc-sdk label,#ot-sdk-cookie-policy label{display:block;margin-bottom:.5rem;font-weight:600}#onetrust-banner-sdk input[type=checkbox],#onetrust-pc-sdk input[type=checkbox],#ot-sdk-cookie-policy input[type=checkbox]{display:inline}#onetrust-banner-sdk ul,#onetrust-pc-sdk ul,#ot-sdk-cookie-policy ul{list-style:circle inside}#onetrust-banner-sdk ul,#onetrust-pc-sdk ul,#ot-sdk-cookie-policy ul{padding-left:0;margin-top:0}#onetrust-banner-sdk ul ul,#onetrust-pc-sdk ul ul,#ot-sdk-cookie-policy ul ul{margin:1.5rem 0 1.5rem 3rem;font-size:90%}#onetrust-banner-sdk li,#onetrust-pc-sdk li,#ot-sdk-cookie-policy li{margin-bottom:1rem}#onetrust-banner-sdk th,#onetrust-banner-sdk td,#onetrust-pc-sdk th,#onetrust-pc-sdk td,#ot-sdk-cookie-policy th,#ot-sdk-cookie-policy td{padding:12px 15px;text-align:left;border-bottom:1px solid #e1e1e1}#onetrust-banner-sdk button,#onetrust-pc-sdk button,#ot-sdk-cookie-policy button{margin-bottom:1rem;font-family:inherit}#onetrust-banner-sdk .ot-sdk-container:after,#onetrust-banner-sdk .ot-sdk-row:after,#onetrust-pc-sdk .ot-sdk-container:after,#onetrust-pc-sdk .ot-sdk-row:after,#ot-sdk-cookie-policy .ot-sdk-container:after,#ot-sdk-cookie-policy .ot-sdk-row:after{content:"";display:table;clear:both}#onetrust-banner-sdk .ot-sdk-row,#onetrust-pc-sdk .ot-sdk-row,#ot-sdk-cookie-policy .ot-sdk-row{margin:0;max-width:none;display:block}#onetrust-banner-sdk{box-shadow:0 0 18px rgba(0,0,0,.2)}#onetrust-banner-sdk.otFlat{position:fixed;z-index:2147483645;bottom:0;right:0;left:0;background-color:#fff;max-height:90%;overflow-x:hidden;overflow-y:auto}#onetrust-banner-sdk.otFlat.top{top:0px;bottom:auto}#onetrust-banner-sdk.otRelFont{font-size:1rem}#onetrust-banner-sdk>.ot-sdk-container{overflow:hidden}#onetrust-banner-sdk::-webkit-scrollbar{width:11px}#onetrust-banner-sdk::-webkit-scrollbar-thumb{border-radius:10px;background:#c1c1c1}#onetrust-banner-sdk{scrollbar-arrow-color:#c1c1c1;scrollbar-darkshadow-color:#c1c1c1;scrollbar-face-color:#c1c1c1;scrollbar-shadow-color:#c1c1c1}#onetrust-banner-sdk #onetrust-policy{margin:1.25em 0 .625em 2em;overflow:hidden}#onetrust-banner-sdk #onetrust-policy .ot-gv-list-handler{float:left;font-size:.82em;padding:0;margin-bottom:0;border:0;line-height:normal;height:auto;width:auto}#onetrust-banner-sdk #onetrust-policy-title{font-size:1.2em;line-height:1.3;margin-bottom:10px}#onetrust-banner-sdk #onetrust-policy-text{clear:both;text-align:left;font-size:.88em;line-height:1.4}#onetrust-banner-sdk #onetrust-policy-text *{font-size:inherit;line-height:inherit}#onetrust-banner-sdk #onetrust-policy-text a{font-weight:bold;margin-left:5px}#onetrust-banner-sdk #onetrust-policy-title,#onetrust-banner-sdk #onetrust-policy-text{color:dimgray;float:left}#onetrust-banner-sdk #onetrust-button-group-parent{min-height:1px;text-align:center}#onetrust-banner-sdk #onetrust-button-group{display:inline-block}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{background-color:#68b631;color:#fff;border-color:#68b631;margin-right:1em;min-width:125px;height:auto;white-space:normal;word-break:break-word;word-wrap:break-word;padding:12px 10px;line-height:1.2;font-size:.813em;font-weight:600}#onetrust-banner-sdk #onetrust-pc-btn-handler.cookie-setting-link{background-color:#fff;border:none;color:#68b631;text-decoration:underline;padding-left:0;padding-right:0}#onetrust-banner-sdk .onetrust-close-btn-ui{width:44px;height:44px;background-size:12px;border:none;position:relative;margin:auto;padding:0}#onetrust-banner-sdk .banner_logo{display:none}#onetrust-banner-sdk.ot-bnr-w-logo .ot-bnr-logo{position:absolute;top:50%;transform:translateY(-50%);left:0px}#onetrust-banner-sdk.ot-bnr-w-logo #onetrust-policy{margin-left:65px}#onetrust-banner-sdk .ot-b-addl-desc{clear:both;float:left;display:block}#onetrust-banner-sdk #banner-options{float:left;display:table;margin-right:0;margin-left:1em;width:calc(100% - 1em)}#onetrust-banner-sdk .banner-option-input{cursor:pointer;width:auto;height:auto;border:none;padding:0;padding-right:3px;margin:0 0 10px;font-size:.82em;line-height:1.4}#onetrust-banner-sdk .banner-option-input *{pointer-events:none;font-size:inherit;line-height:inherit}#onetrust-banner-sdk .banner-option-input[aria-expanded=true]~.banner-option-details{display:block;height:auto}#onetrust-banner-sdk .banner-option-input[aria-expanded=true] .ot-arrow-container{transform:rotate(90deg)}#onetrust-banner-sdk .banner-option{margin-bottom:12px;margin-left:0;border:none;float:left;padding:0}#onetrust-banner-sdk .banner-option:first-child{padding-left:2px}#onetrust-banner-sdk .banner-option:not(:first-child){padding:0;border:none}#onetrust-banner-sdk .banner-option-header{cursor:pointer;display:inline-block}#onetrust-banner-sdk .banner-option-header :first-child{color:dimgray;font-weight:bold;float:left}#onetrust-banner-sdk .banner-option-header .ot-arrow-container{display:inline-block;border-top:6px solid rgba(0,0,0,0);border-bottom:6px solid rgba(0,0,0,0);border-left:6px solid dimgray;margin-left:10px;vertical-align:middle}#onetrust-banner-sdk .banner-option-details{display:none;font-size:.83em;line-height:1.5;padding:10px 0px 5px 10px;margin-right:10px;height:0px}#onetrust-banner-sdk .banner-option-details *{font-size:inherit;line-height:inherit;color:dimgray}#onetrust-banner-sdk .ot-arrow-container,#onetrust-banner-sdk .banner-option-details{transition:all 300ms ease-in 0s;-webkit-transition:all 300ms ease-in 0s;-moz-transition:all 300ms ease-in 0s;-o-transition:all 300ms ease-in 0s}#onetrust-banner-sdk .ot-dpd-container{float:left}#onetrust-banner-sdk .ot-dpd-title{margin-bottom:10px}#onetrust-banner-sdk .ot-dpd-title,#onetrust-banner-sdk .ot-dpd-desc{font-size:.88em;line-height:1.4;color:dimgray}#onetrust-banner-sdk .ot-dpd-title *,#onetrust-banner-sdk .ot-dpd-desc *{font-size:inherit;line-height:inherit}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-text *{margin-bottom:0}#onetrust-banner-sdk.ot-iab-2 .onetrust-vendors-list-handler{display:block;margin-left:0;margin-top:5px;clear:both;margin-bottom:0;padding:0;border:0;height:auto;width:auto}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group button{display:block}#onetrust-banner-sdk.ot-close-btn-link{padding-top:25px}#onetrust-banner-sdk.ot-close-btn-link #onetrust-close-btn-container{top:15px;transform:none;right:15px}#onetrust-banner-sdk.ot-close-btn-link #onetrust-close-btn-container button{padding:0;white-space:pre-wrap;border:none;height:auto;line-height:1.5;text-decoration:underline;font-size:.69em}#onetrust-banner-sdk #onetrust-policy-text,#onetrust-banner-sdk .ot-dpd-desc,#onetrust-banner-sdk .ot-b-addl-desc{font-size:.813em;line-height:1.5}#onetrust-banner-sdk .ot-dpd-desc{margin-bottom:10px}#onetrust-banner-sdk .ot-dpd-desc>.ot-b-addl-desc{margin-top:10px;margin-bottom:10px;font-size:1em}@media only screen and (max-width: 425px){#onetrust-banner-sdk #onetrust-close-btn-container{position:absolute;top:6px;right:2px}#onetrust-banner-sdk #onetrust-policy{margin-left:0;margin-top:3em}#onetrust-banner-sdk #onetrust-button-group{display:block}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{width:100%}#onetrust-banner-sdk .onetrust-close-btn-ui{top:auto;transform:none}#onetrust-banner-sdk #onetrust-policy-title{display:inline;float:none}#onetrust-banner-sdk #banner-options{margin:0;padding:0;width:100%}}@media only screen and (min-width: 426px)and (max-width: 896px){#onetrust-banner-sdk #onetrust-close-btn-container{position:absolute;top:0;right:0}#onetrust-banner-sdk #onetrust-policy{margin-left:1em;margin-right:1em}#onetrust-banner-sdk .onetrust-close-btn-ui{top:10px;right:10px}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{width:95%}#onetrust-banner-sdk.ot-iab-2 #onetrust-group-container{width:100%}#onetrust-banner-sdk.ot-bnr-w-logo #onetrust-button-group-parent{padding-left:50px}#onetrust-banner-sdk #onetrust-button-group-parent{width:100%;position:relative;margin-left:0}#onetrust-banner-sdk #onetrust-button-group button{display:inline-block}#onetrust-banner-sdk #onetrust-button-group{margin-right:0;text-align:center}#onetrust-banner-sdk .has-reject-all-button #onetrust-pc-btn-handler{float:left}#onetrust-banner-sdk .has-reject-all-button #onetrust-reject-all-handler,#onetrust-banner-sdk .has-reject-all-button #onetrust-accept-btn-handler{float:right}#onetrust-banner-sdk .has-reject-all-button #onetrust-button-group{width:calc(100% - 2em);margin-right:0}#onetrust-banner-sdk .has-reject-all-button #onetrust-pc-btn-handler.cookie-setting-link{padding-left:0px;text-align:left}#onetrust-banner-sdk.ot-buttons-fw .ot-sdk-three button{width:100%;text-align:center}#onetrust-banner-sdk.ot-buttons-fw #onetrust-button-group-parent button{float:none}#onetrust-banner-sdk.ot-buttons-fw #onetrust-pc-btn-handler.cookie-setting-link{text-align:center}}@media only screen and (min-width: 550px){#onetrust-banner-sdk .banner-option:not(:first-child){border-left:1px solid #d8d8d8;padding-left:25px}}@media only screen and (min-width: 425px)and (max-width: 550px){#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group,#onetrust-banner-sdk.ot-iab-2 #onetrust-policy,#onetrust-banner-sdk.ot-iab-2 .banner-option{width:100%}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-accept-btn-handler,#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-reject-all-handler,#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-pc-btn-handler{width:100%}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-accept-btn-handler,#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-reject-all-handler{float:left}}@media only screen and (min-width: 769px){#onetrust-banner-sdk #onetrust-button-group{margin-right:30%}#onetrust-banner-sdk #banner-options{margin-left:2em;margin-right:5em;margin-bottom:1.25em;width:calc(100% - 7em)}}@media only screen and (min-width: 897px)and (max-width: 1023px){#onetrust-banner-sdk.vertical-align-content #onetrust-button-group-parent{position:absolute;top:50%;left:75%;transform:translateY(-50%)}#onetrust-banner-sdk #onetrust-close-btn-container{top:50%;margin:auto;transform:translate(-50%, -50%);position:absolute;padding:0;right:0}#onetrust-banner-sdk #onetrust-close-btn-container button{position:relative;margin:0;right:-22px;top:2px}}@media only screen and (min-width: 1024px){#onetrust-banner-sdk #onetrust-close-btn-container{top:50%;margin:auto;transform:translate(-50%, -50%);position:absolute;right:0}#onetrust-banner-sdk #onetrust-close-btn-container button{right:-12px}#onetrust-banner-sdk #onetrust-policy{margin-left:2em}#onetrust-banner-sdk.vertical-align-content #onetrust-button-group-parent{position:absolute;top:50%;left:60%;transform:translateY(-50%)}#onetrust-banner-sdk .ot-optout-signal{width:50%}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-title{width:50%}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-text,#onetrust-banner-sdk.ot-iab-2 :not(.ot-dpd-desc)>.ot-b-addl-desc{margin-bottom:1em;width:50%;border-right:1px solid #d8d8d8;padding-right:1rem}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-text{margin-bottom:0;padding-bottom:1em}#onetrust-banner-sdk.ot-iab-2 :not(.ot-dpd-desc)>.ot-b-addl-desc{margin-bottom:0;padding-bottom:1em}#onetrust-banner-sdk.ot-iab-2 .ot-dpd-container{width:45%;padding-left:1rem;display:inline-block;float:none}#onetrust-banner-sdk.ot-iab-2 .ot-dpd-title{line-height:1.7}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group-parent{left:auto;right:4%;margin-left:0}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group button{display:block}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{margin:auto;width:30%}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{width:60%}#onetrust-banner-sdk #onetrust-button-group{margin-right:auto}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{margin-top:1em}}@media only screen and (min-width: 890px){#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group-parent{padding-left:3%;padding-right:4%;margin-left:0}#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group{margin-right:0;margin-top:1.25em;width:100%}#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button{width:100%;margin-bottom:5px;margin-top:5px}#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button:last-of-type{margin-bottom:20px}}@media only screen and (min-width: 1280px){#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{width:55%}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{width:44%;padding-left:2%;padding-right:2%}#onetrust-banner-sdk:not(.ot-iab-2).vertical-align-content #onetrust-button-group-parent{position:absolute;left:55%}}
        #onetrust-consent-sdk #onetrust-banner-sdk {background-color: #FFF;}
            #onetrust-consent-sdk #onetrust-policy-title,
                    #onetrust-consent-sdk #onetrust-policy-text,
                    #onetrust-consent-sdk .ot-b-addl-desc,
                    #onetrust-consent-sdk .ot-dpd-desc,
                    #onetrust-consent-sdk .ot-dpd-title,
                    #onetrust-consent-sdk #onetrust-policy-text *:not(.onetrust-vendors-list-handler),
                    #onetrust-consent-sdk .ot-dpd-desc *:not(.onetrust-vendors-list-handler),
                    #onetrust-consent-sdk #onetrust-banner-sdk #banner-options *,
                    #onetrust-banner-sdk .ot-cat-header,
                    #onetrust-banner-sdk .ot-optout-signal
                    {
                        color: #2E2E2E;
                    }
            #onetrust-consent-sdk #onetrust-banner-sdk .banner-option-details {
                    background-color: #E9E9E9;}
             #onetrust-consent-sdk #onetrust-banner-sdk a[href],
                    #onetrust-consent-sdk #onetrust-banner-sdk a[href] font,
                    #onetrust-consent-sdk #onetrust-banner-sdk .ot-link-btn
                        {
                            color: #007398;
                        }#onetrust-consent-sdk #onetrust-accept-btn-handler,
                         #onetrust-banner-sdk #onetrust-reject-all-handler {
                            background-color: #007398;border-color: #007398;
                color: #FFF;
            }
            #onetrust-consent-sdk #onetrust-banner-sdk *:focus,
            #onetrust-consent-sdk #onetrust-banner-sdk:focus {
               outline-color: #000000;
               outline-width: 1px;
            }
            #onetrust-consent-sdk #onetrust-pc-btn-handler,
            #onetrust-consent-sdk #onetrust-pc-btn-handler.cookie-setting-link {
                color: #6CC04A; border-color: #6CC04A;
                background-color:
                #FFF;
            }/*! v6.10.0 2021-01-18 */
div#onetrust-consent-sdk #onetrust-banner-sdk{border-top:2px solid #d0d3d4!important;box-shadow:none;padding:24px}div#onetrust-consent-sdk button{border-radius:0!important;box-shadow:none!important;box-sizing:border-box!important;font-size:20px!important;font-weight:400!important;letter-spacing:0!important;max-width:none!important;white-space:nowrap!important}div#onetrust-consent-sdk button:not(.ot-link-btn){background-color:#007dbc!important;border:2px solid #007dbc!important;color:#fff!important;height:48px!important;padding:0 1em!important;width:auto!important}div#onetrust-consent-sdk button:hover{background-color:#fff!important;border-color:#949494!important;color:#2e2e2e!important}div#onetrust-consent-sdk button.ot-link-btn{color:#007dbc!important;font-size:16px!important;text-decoration:underline}div#onetrust-consent-sdk button.ot-link-btn:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-consent-sdk a,div#onetrust-pc-sdk a{color:#007dbc!important;text-decoration:underline!important}div#onetrust-consent-sdk a,div#onetrust-consent-sdk button,div#onetrust-consent-sdk p:hover{opacity:1!important}div#onetrust-consent-sdk a:focus,div#onetrust-consent-sdk button:focus,div#onetrust-consent-sdk input:focus{outline:2px solid #949494!important;outline-offset:1px!important}div#onetrust-banner-sdk .ot-sdk-container{padding:0;width:auto}div#onetrust-banner-sdk .ot-sdk-row{align-items:flex-start;box-sizing:border-box;display:flex;flex-direction:column;justify-content:space-between;margin:auto;max-width:1152px}div#onetrust-banner-sdk .ot-sdk-row:after{display:none}div#onetrust-banner-sdk #onetrust-group-container,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-group-container,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{flex-grow:1;width:auto}div#onetrust-banner-sdk #onetrust-policy,div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy{margin:0;overflow:visible}div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy-text,div#onetrust-consent-sdk #onetrust-policy-text{font-size:16px;line-height:24px;max-width:44em;margin:0}div#onetrust-consent-sdk #onetrust-policy-text a[href]{font-weight:400;margin-left:8px}div#onetrust-banner-sdk #onetrust-button-group-parent{flex:0 0 auto;margin:32px 0 0;width:100%}div#onetrust-banner-sdk #onetrust-button-group{display:flex;flex-direction:row;flex-wrap:wrap;justify-content:flex-end;margin:-8px}div#onetrust-banner-sdk .banner-actions-container{display:flex;flex:1 0 auto}div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button:last-of-type,div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto;margin:8px;width:auto}div#onetrust-consent-sdk #onetrust-pc-btn-handler{background-color:#fff!important;color:inherit!important}div#onetrust-banner-sdk #onetrust-close-btn-container{display:none}@media only screen and (min-width:556px){div#onetrust-consent-sdk #onetrust-banner-sdk{padding:40px}div#onetrust-banner-sdk #onetrust-policy{margin:0 40px 0 0}div#onetrust-banner-sdk .ot-sdk-row{align-items:center;flex-direction:row}div#onetrust-banner-sdk #onetrust-button-group-parent,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-button-group-parent,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{margin:0;padding:0;width:auto}div#onetrust-banner-sdk #onetrust-button-group,div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group{align-items:stretch;flex-direction:column-reverse;margin:0}div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto}}@media only screen and (min-width:768px){div#onetrust-banner-sdk #onetrust-policy{margin:0 48px 0 0}div#onetrust-consent-sdk #onetrust-banner-sdk{padding:48px}}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-size:16px;line-height:24px}div#onetrust-consent-sdk #onetrust-pc-sdk p,div#onetrust-pc-sdk #ot-pc-desc,div#onetrust-pc-sdk .category-host-list-handler,div#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{font-size:16px;font-weight:400;line-height:24px}div#onetrust-consent-sdk a:hover,div#onetrust-pc-sdk a:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-pc-sdk{border-radius:0;bottom:0;height:auto;left:0;margin:auto;max-width:100%;overflow:hidden;right:0;top:0;width:512px;max-height:800px}div#onetrust-pc-sdk .ot-pc-header{display:none}div#onetrust-pc-sdk #ot-pc-content{overscroll-behavior:contain;padding:0 12px 0 24px;margin:16px 4px 0 0;top:0;right:16px;left:0;width:auto}div#onetrust-pc-sdk #ot-category-title,div#onetrust-pc-sdk #ot-pc-title{font-size:24px;font-weight:400;line-height:32px;margin:0 0 16px}div#onetrust-pc-sdk #ot-pc-desc{padding:0}div#onetrust-pc-sdk #ot-pc-desc a{display:inline}div#onetrust-pc-sdk #accept-recommended-btn-handler{display:none!important}div#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:2px solid #007dbc;outline-offset:-1px;transition:none}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item{border-width:0 0 2px}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item:first-of-type{border-width:2px 0}div#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{padding:8px 0;width:100%}div#onetrust-pc-sdk .ot-plus-minus{transform:translateY(2px)}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item>button{background:0 0!important;border:0!important;height:44px!important;max-width:none!important;width:calc(100% - 48px)!important}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-weight:700}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr{padding:0}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item .ot-acc-grpdesc{padding:0;width:100%}div#onetrust-pc-sdk .ot-acc-grpcntr .ot-subgrp-cntr{border:0;padding:0}div#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp{margin:0}div#onetrust-pc-sdk .ot-always-active-group .ot-cat-header{width:calc(100% - 160px)}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{width:calc(100% - 88px)}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active{color:inherit;font-size:12px;font-weight:400;line-height:1.5;padding-right:48px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:before{border-radius:12px;position:absolute;right:0;top:0;content:'';background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:after{border-radius:50%;position:absolute;right:5px;top:4px;content:'';background-color:#007dbc;height:12px;width:12px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active,div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl{right:2px}div#onetrust-pc-sdk .ot-switch{display:block;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob,div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob{background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob:before{background-color:#737373;height:8px;left:4px;top:4px;width:8px}div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob:before{background-color:#007dbc;height:12px;left:0;top:2px;width:12px}div#onetrust-pc-sdk .ot-tgl input:focus+.ot-switch .ot-switch-nob{box-shadow:0 0;outline:2px solid #007dbc!important;outline-offset:1px;transition:none}div#onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-grpcntr.ot-acc-txt{background-color:transparent;padding-left:3px}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr,div#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr{overflow:visible;width:100%}div#onetrust-pc-sdk .ot-pc-footer{border-top:0 solid}div#onetrust-pc-sdk .ot-btn-container{padding-top:24px;text-align:center}div#onetrust-pc-sdk .ot-pc-footer button{margin:8px 0;background-color:#fff}div#onetrust-pc-sdk .ot-pc-footer-logo{background-color:#fff}div#onetrust-pc-sdk #ot-lst-title span{font-size:24px;font-weight:400;line-height:32px}div#onetrust-pc-sdk #ot-host-lst .ot-host-desc,div#onetrust-pc-sdk #ot-host-lst .ot-host-expand,div#onetrust-pc-sdk #ot-host-lst .ot-host-name,div#onetrust-pc-sdk #ot-host-lst .ot-host-name a,div#onetrust-pc-sdk .back-btn-handler,div#onetrust-pc-sdk .ot-host-opt li>div div{font-size:16px;font-weight:400;line-height:24px}div#onetrust-pc-sdk #ot-pc-lst{top:0}div#onetrust-pc-sdk .back-btn-handler{text-decoration:none!important}div#onetrust-pc-sdk #filter-btn-handler:hover svg{filter:invert(1)}div#onetrust-pc-sdk .back-btn-handler svg{width:16px;height:16px}div#onetrust-pc-sdk .ot-host-item>button{background:0 0!important;border:0!important;height:66px!important;max-width:none!important;width:calc(100% - 5px)!important;transform:translate(2px,2px)}div#onetrust-pc-sdk .ot-host-item{border-bottom:2px solid #b9b9b9;padding:0}div#onetrust-pc-sdk .ot-host-item .ot-acc-hdr{margin:0 0 -6px;padding:8px 0}div#onetrust-pc-sdk ul li:first-child{border-top:2px solid #b9b9b9}div#onetrust-pc-sdk .ot-host-item .ot-plus-minus{margin:0 8px 0 0}div#onetrust-pc-sdk .ot-search-cntr{width:calc(100% - 48px)}div#onetrust-pc-sdk .ot-host-opt .ot-host-info{background-color:transparent}div#onetrust-pc-sdk .ot-host-opt li>div div{padding:0}div#onetrust-pc-sdk #vendor-search-handler{border-radius:0;border-color:#939393;border-style:solid;border-width:2px 0 2px 2px;font-size:20px;height:48px;margin:0}div#onetrust-pc-sdk .ot-lst-subhdr svg{right:0;top:8px}div#onetrust-pc-sdk .ot-fltr-cntr{box-sizing:border-box;right:0;width:48px}div#onetrust-pc-sdk #filter-btn-handler{width:48px!important;padding:8px!important}div#onetrust-pc-sdk #ot-fltr-modal{max-height:400px}
#onetrust-pc-sdk.otPcCenter{overflow:hidden;position:fixed;margin:0 auto;top:5%;right:0;left:0;width:40%;max-width:575px;min-width:575px;border-radius:2.5px;z-index:2147483647;background-color:#fff;-webkit-box-shadow:0px 2px 10px -3px #999;-moz-box-shadow:0px 2px 10px -3px #999;box-shadow:0px 2px 10px -3px #999}#onetrust-pc-sdk.otPcCenter[dir=rtl]{right:0;left:0}#onetrust-pc-sdk.otRelFont{font-size:1rem}#onetrust-pc-sdk .ot-optout-signal{margin-top:.625rem}#onetrust-pc-sdk #ot-addtl-venlst .ot-arw-cntr,#onetrust-pc-sdk #ot-addtl-venlst .ot-plus-minus,#onetrust-pc-sdk .ot-hide-tgl{visibility:hidden}#onetrust-pc-sdk #ot-addtl-venlst .ot-arw-cntr *,#onetrust-pc-sdk #ot-addtl-venlst .ot-plus-minus *,#onetrust-pc-sdk .ot-hide-tgl *{visibility:hidden}#onetrust-pc-sdk #ot-gn-venlst .ot-ven-item .ot-acc-hdr{min-height:40px}#onetrust-pc-sdk .ot-pc-header{height:39px;padding:10px 0 10px 30px;border-bottom:1px solid #e9e9e9}#onetrust-pc-sdk #ot-pc-title,#onetrust-pc-sdk #ot-category-title,#onetrust-pc-sdk .ot-cat-header,#onetrust-pc-sdk #ot-lst-title,#onetrust-pc-sdk .ot-ven-hdr .ot-ven-name,#onetrust-pc-sdk .ot-always-active{font-weight:bold;color:dimgray}#onetrust-pc-sdk .ot-always-active-group .ot-cat-header{width:55%;font-weight:700}#onetrust-pc-sdk .ot-cat-item p{clear:both;float:left;margin-top:10px;margin-bottom:5px;line-height:1.5;font-size:.812em;color:dimgray}#onetrust-pc-sdk .ot-close-icon{height:44px;width:44px;background-size:10px}#onetrust-pc-sdk #ot-pc-title{float:left;font-size:1em;line-height:1.5;margin-bottom:10px;margin-top:10px;width:100%}#onetrust-pc-sdk #accept-recommended-btn-handler{margin-right:10px;margin-bottom:25px;outline-offset:-1px}#onetrust-pc-sdk #ot-pc-desc{clear:both;width:100%;font-size:.812em;line-height:1.5;margin-bottom:25px}#onetrust-pc-sdk #ot-pc-desc *{font-size:inherit;line-height:inherit}#onetrust-pc-sdk #ot-pc-desc ul li{padding:10px 0px}#onetrust-pc-sdk a{color:#656565;cursor:pointer}#onetrust-pc-sdk a:hover{color:#3860be}#onetrust-pc-sdk label{margin-bottom:0}#onetrust-pc-sdk #vdr-lst-dsc{font-size:.812em;line-height:1.5;padding:10px 15px 5px 15px}#onetrust-pc-sdk button{max-width:394px;padding:12px 30px;line-height:1;word-break:break-word;word-wrap:break-word;white-space:normal;font-weight:bold;height:auto}#onetrust-pc-sdk .ot-link-btn{padding:0;margin-bottom:0;border:0;font-weight:normal;line-height:normal;width:auto;height:auto}#onetrust-pc-sdk #ot-pc-content{position:absolute;overflow-y:scroll;padding-left:2px;padding-right:30px;top:60px;bottom:110px;margin:1px 3px 0 30px;width:calc(100% - 63px)}#onetrust-pc-sdk .ot-vs-list .ot-always-active,#onetrust-pc-sdk .ot-cat-grp .ot-always-active{float:right;clear:none;color:#3860be;margin:0;font-size:.813em;line-height:1.3}#onetrust-pc-sdk .ot-pc-scrollbar::-webkit-scrollbar-track{margin-right:20px}#onetrust-pc-sdk .ot-pc-scrollbar::-webkit-scrollbar{width:11px}#onetrust-pc-sdk .ot-pc-scrollbar::-webkit-scrollbar-thumb{border-radius:10px;background:#d8d8d8}#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:#000 1px solid}#onetrust-pc-sdk .ot-pc-scrollbar{scrollbar-arrow-color:#d8d8d8;scrollbar-darkshadow-color:#d8d8d8;scrollbar-face-color:#d8d8d8;scrollbar-shadow-color:#d8d8d8}#onetrust-pc-sdk .save-preference-btn-handler{margin-right:20px}#onetrust-pc-sdk .ot-pc-refuse-all-handler{margin-right:10px}#onetrust-pc-sdk #ot-pc-desc .privacy-notice-link{margin-left:0;margin-right:8px}#onetrust-pc-sdk #ot-pc-desc .ot-imprint-handler{margin-left:0;margin-right:8px}#onetrust-pc-sdk .ot-subgrp-cntr{display:inline-block;clear:both;width:100%;padding-top:15px}#onetrust-pc-sdk .ot-switch+.ot-subgrp-cntr{padding-top:10px}#onetrust-pc-sdk ul.ot-subgrps{margin:0;font-size:initial}#onetrust-pc-sdk ul.ot-subgrps li p,#onetrust-pc-sdk ul.ot-subgrps li h5{font-size:.813em;line-height:1.4;color:dimgray}#onetrust-pc-sdk ul.ot-subgrps .ot-switch{min-height:auto}#onetrust-pc-sdk ul.ot-subgrps .ot-switch-nob{top:0}#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr{display:inline-block;width:100%}#onetrust-pc-sdk ul.ot-subgrps .ot-acc-txt{margin:0}#onetrust-pc-sdk ul.ot-subgrps li{padding:0;border:none}#onetrust-pc-sdk ul.ot-subgrps li h5{position:relative;top:5px;font-weight:bold;margin-bottom:0;float:left}#onetrust-pc-sdk li.ot-subgrp{margin-left:20px;overflow:auto}#onetrust-pc-sdk li.ot-subgrp>h5{width:calc(100% - 100px)}#onetrust-pc-sdk .ot-cat-item p>ul,#onetrust-pc-sdk li.ot-subgrp p>ul{margin:0px;list-style:disc;margin-left:15px;font-size:inherit}#onetrust-pc-sdk .ot-cat-item p>ul li,#onetrust-pc-sdk li.ot-subgrp p>ul li{font-size:inherit;padding-top:10px;padding-left:0px;padding-right:0px;border:none}#onetrust-pc-sdk .ot-cat-item p>ul li:last-child,#onetrust-pc-sdk li.ot-subgrp p>ul li:last-child{padding-bottom:10px}#onetrust-pc-sdk .ot-pc-logo{height:40px;width:120px}#onetrust-pc-sdk .ot-pc-footer{position:absolute;bottom:0px;width:100%;max-height:160px;border-top:1px solid #d8d8d8}#onetrust-pc-sdk.ot-ftr-stacked .ot-pc-refuse-all-handler{margin-bottom:0px}#onetrust-pc-sdk.ot-ftr-stacked #ot-pc-content{bottom:160px}#onetrust-pc-sdk.ot-ftr-stacked .ot-pc-footer button{width:100%;max-width:none}#onetrust-pc-sdk.ot-ftr-stacked .ot-btn-container{margin:0 30px;width:calc(100% - 60px);padding-right:0}#onetrust-pc-sdk .ot-pc-footer-logo{height:30px;width:100%;text-align:right;background:#f4f4f4}#onetrust-pc-sdk .ot-pc-footer-logo a{display:inline-block;margin-top:5px;margin-right:10px}#onetrust-pc-sdk[dir=rtl] .ot-pc-footer-logo{direction:rtl}#onetrust-pc-sdk[dir=rtl] .ot-pc-footer-logo a{margin-right:25px}#onetrust-pc-sdk .ot-tgl{float:right;position:relative;z-index:1}#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob{background-color:#468254;border:1px solid #fff}#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob:before{-webkit-transform:translateX(20px);-ms-transform:translateX(20px);transform:translateX(20px);background-color:#fff;border-color:#fff}#onetrust-pc-sdk .ot-tgl input:focus+.ot-switch{outline:#000 solid 1px}#onetrust-pc-sdk .ot-switch{position:relative;display:inline-block;width:45px;height:25px}#onetrust-pc-sdk .ot-switch-nob{position:absolute;cursor:pointer;top:0;left:0;right:0;bottom:0;background-color:#767676;border:1px solid #ddd;transition:all .2s ease-in 0s;-moz-transition:all .2s ease-in 0s;-o-transition:all .2s ease-in 0s;-webkit-transition:all .2s ease-in 0s;border-radius:20px}#onetrust-pc-sdk .ot-switch-nob:before{position:absolute;content:"";height:18px;width:18px;bottom:3px;left:3px;background-color:#fff;-webkit-transition:.4s;transition:.4s;border-radius:20px}#onetrust-pc-sdk .ot-chkbox input:checked~label::before{background-color:#3860be}#onetrust-pc-sdk .ot-chkbox input+label::after{content:none;color:#fff}#onetrust-pc-sdk .ot-chkbox input:checked+label::after{content:""}#onetrust-pc-sdk .ot-chkbox input:focus+label::before{outline-style:solid;outline-width:2px;outline-style:auto}#onetrust-pc-sdk .ot-chkbox input[aria-checked=mixed]~label::before{background-color:#3860be}#onetrust-pc-sdk .ot-chkbox input[aria-checked=mixed]+label::after{content:""}#onetrust-pc-sdk .ot-chkbox label{position:relative;display:inline-block;padding-left:30px;cursor:pointer;font-weight:500}#onetrust-pc-sdk .ot-chkbox label::before,#onetrust-pc-sdk .ot-chkbox label::after{position:absolute;content:"";display:inline-block;border-radius:3px}#onetrust-pc-sdk .ot-chkbox label::before{height:18px;width:18px;border:1px solid #3860be;left:0px;top:auto}#onetrust-pc-sdk .ot-chkbox label::after{height:5px;width:9px;border-left:3px solid;border-bottom:3px solid;transform:rotate(-45deg);-o-transform:rotate(-45deg);-ms-transform:rotate(-45deg);-webkit-transform:rotate(-45deg);left:4px;top:5px}#onetrust-pc-sdk .ot-label-txt{display:none}#onetrust-pc-sdk .ot-chkbox input,#onetrust-pc-sdk .ot-tgl input{position:absolute;opacity:0;width:0;height:0}#onetrust-pc-sdk .ot-arw-cntr{float:right;position:relative;pointer-events:none}#onetrust-pc-sdk .ot-arw-cntr .ot-arw{width:16px;height:16px;margin-left:5px;color:dimgray;display:inline-block;vertical-align:middle;-webkit-transition:all 150ms ease-in 0s;-moz-transition:all 150ms ease-in 0s;-o-transition:all 150ms ease-in 0s;transition:all 150ms ease-in 0s}#onetrust-pc-sdk input:checked~.ot-acc-hdr .ot-arw,#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-arw-cntr svg{transform:rotate(90deg);-o-transform:rotate(90deg);-ms-transform:rotate(90deg);-webkit-transform:rotate(90deg)}#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:#000 1px solid}#onetrust-pc-sdk .ot-tgl-cntr,#onetrust-pc-sdk .ot-arw-cntr{display:inline-block}#onetrust-pc-sdk .ot-tgl-cntr{width:45px;float:right;margin-top:2px}#onetrust-pc-sdk #ot-lst-cnt .ot-tgl-cntr{margin-top:10px}#onetrust-pc-sdk .ot-always-active-subgroup{width:auto;padding-left:0px !important;top:3px;position:relative}#onetrust-pc-sdk .ot-label-status{font-size:.75em;display:none;font-size:.75em;position:relative;top:2px;padding-right:5px;float:left}#onetrust-pc-sdk .ot-arw-cntr{margin-top:-1px}#onetrust-pc-sdk .ot-arw-cntr svg{-webkit-transition:all 300ms ease-in 0s;-moz-transition:all 300ms ease-in 0s;-o-transition:all 300ms ease-in 0s;transition:all 300ms ease-in 0s;height:10px;width:10px}#onetrust-pc-sdk input:checked~.ot-acc-hdr .ot-arw{transform:rotate(90deg);-o-transform:rotate(90deg);-ms-transform:rotate(90deg);-webkit-transform:rotate(90deg)}#onetrust-pc-sdk .ot-arw{width:10px;margin-left:15px;transition:all 300ms ease-in 0s;-webkit-transition:all 300ms ease-in 0s;-moz-transition:all 300ms ease-in 0s;-o-transition:all 300ms ease-in 0s}#onetrust-pc-sdk .ot-vlst-cntr{margin-bottom:0}#onetrust-pc-sdk .ot-hlst-cntr{margin-top:5px;display:inline-block;width:100%}#onetrust-pc-sdk .category-vendors-list-handler,#onetrust-pc-sdk .category-vendors-list-handler+a,#onetrust-pc-sdk .category-host-list-handler{clear:both;color:#3860be;margin-left:0;font-size:.813em;text-decoration:none;float:left;overflow:hidden}#onetrust-pc-sdk .category-vendors-list-handler:hover,#onetrust-pc-sdk .category-vendors-list-handler+a:hover,#onetrust-pc-sdk .category-host-list-handler:hover{text-decoration-line:underline}#onetrust-pc-sdk .category-vendors-list-handler+a{clear:none}#onetrust-pc-sdk .ot-vlst-cntr .ot-ext-lnk,#onetrust-pc-sdk .ot-ven-hdr .ot-ext-lnk{display:inline-block;height:13px;width:13px;background-repeat:no-repeat;margin-left:1px;margin-top:6px;cursor:pointer}#onetrust-pc-sdk .ot-ven-hdr .ot-ext-lnk{margin-bottom:-1px}#onetrust-pc-sdk .back-btn-handler{font-size:1em;text-decoration:none}#onetrust-pc-sdk .back-btn-handler:hover{opacity:.6}#onetrust-pc-sdk #ot-lst-title h3{display:inline-block;word-break:break-word;word-wrap:break-word;margin-bottom:0;color:#656565;font-size:1em;font-weight:bold;margin-left:15px}#onetrust-pc-sdk #ot-lst-title{margin:10px 0 10px 0px;font-size:1em;text-align:left}#onetrust-pc-sdk #ot-pc-hdr{margin:0 0 0 30px;height:auto;width:auto}#onetrust-pc-sdk #ot-pc-hdr input::placeholder{color:#707070;font-style:italic}#onetrust-pc-sdk #vendor-search-handler{height:31px;width:100%;border-radius:50px;font-size:.8em;padding-right:35px;padding-left:15px;float:left;margin-left:15px}#onetrust-pc-sdk .ot-ven-name{display:block;width:auto;padding-right:5px}#onetrust-pc-sdk #ot-lst-cnt{overflow-y:auto;margin-left:20px;margin-right:7px;width:calc(100% - 27px);max-height:calc(100% - 80px);height:100%;transform:translate3d(0, 0, 0)}#onetrust-pc-sdk #ot-pc-lst{width:100%;bottom:100px;position:absolute;top:60px}#onetrust-pc-sdk #ot-pc-lst:not(.ot-enbl-chr) .ot-tgl-cntr .ot-arw-cntr,#onetrust-pc-sdk #ot-pc-lst:not(.ot-enbl-chr) .ot-tgl-cntr .ot-arw-cntr *{visibility:hidden}#onetrust-pc-sdk #ot-pc-lst .ot-tgl-cntr{right:12px;position:absolute}#onetrust-pc-sdk #ot-pc-lst .ot-arw-cntr{float:right;position:relative}#onetrust-pc-sdk #ot-pc-lst .ot-arw{margin-left:10px}#onetrust-pc-sdk #ot-pc-lst .ot-acc-hdr{overflow:hidden;cursor:pointer}#onetrust-pc-sdk .ot-vlst-cntr{overflow:hidden}#onetrust-pc-sdk #ot-sel-blk{overflow:hidden;width:100%;position:sticky;position:-webkit-sticky;top:0;z-index:3}#onetrust-pc-sdk #ot-back-arw{height:12px;width:12px}#onetrust-pc-sdk .ot-lst-subhdr{width:100%;display:inline-block}#onetrust-pc-sdk .ot-search-cntr{float:left;width:78%;position:relative}#onetrust-pc-sdk .ot-search-cntr>svg{width:30px;height:30px;position:absolute;float:left;right:-15px}#onetrust-pc-sdk .ot-fltr-cntr{float:right;right:50px;position:relative}#onetrust-pc-sdk #filter-btn-handler{background-color:#3860be;border-radius:17px;display:inline-block;position:relative;width:32px;height:32px;-moz-transition:.1s ease;-o-transition:.1s ease;-webkit-transition:1s ease;transition:.1s ease;padding:0;margin:0}#onetrust-pc-sdk #filter-btn-handler:hover{background-color:#3860be}#onetrust-pc-sdk #filter-btn-handler svg{width:12px;height:12px;margin:3px 10px 0 10px;display:block;position:static;right:auto;top:auto}#onetrust-pc-sdk .ot-ven-link,#onetrust-pc-sdk .ot-ven-legclaim-link{color:#3860be;text-decoration:none;font-weight:100;display:inline-block;padding-top:10px;transform:translate(0, 1%);-o-transform:translate(0, 1%);-ms-transform:translate(0, 1%);-webkit-transform:translate(0, 1%);position:relative;z-index:2}#onetrust-pc-sdk .ot-ven-link *,#onetrust-pc-sdk .ot-ven-legclaim-link *{font-size:inherit}#onetrust-pc-sdk .ot-ven-link:hover,#onetrust-pc-sdk .ot-ven-legclaim-link:hover{text-decoration:underline}#onetrust-pc-sdk .ot-ven-hdr{width:calc(100% - 160px);height:auto;float:left;word-break:break-word;word-wrap:break-word;vertical-align:middle;padding-bottom:3px}#onetrust-pc-sdk .ot-ven-link,#onetrust-pc-sdk .ot-ven-legclaim-link{letter-spacing:.03em;font-size:.75em;font-weight:400}#onetrust-pc-sdk .ot-ven-dets{border-radius:2px;background-color:#f8f8f8}#onetrust-pc-sdk .ot-ven-dets li:first-child p:first-child{border-top:none}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:not(:first-child){border-top:1px solid #ddd !important}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:nth-child(n+3) p{display:inline-block}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:nth-child(n+3) p:nth-of-type(odd){width:30%}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:nth-child(n+3) p:nth-of-type(even){width:50%;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc p,#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc h5{padding-top:5px;padding-bottom:5px;display:block}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc h5{display:inline-block}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc p:nth-last-child(-n+1){padding-bottom:10px}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc p:nth-child(-n+2):not(.disc-pur){padding-top:10px}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc .disc-pur-cont{display:inline}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc .disc-pur{position:relative;width:50% !important;word-break:break-word;word-wrap:break-word;left:calc(30% + 17px)}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc .disc-pur:nth-child(-n+1){position:static}#onetrust-pc-sdk .ot-ven-dets p,#onetrust-pc-sdk .ot-ven-dets h5,#onetrust-pc-sdk .ot-ven-dets span{font-size:.69em;text-align:left;vertical-align:middle;word-break:break-word;word-wrap:break-word;margin:0;padding-bottom:10px;padding-left:15px;color:#2e3644}#onetrust-pc-sdk .ot-ven-dets h5{padding-top:5px}#onetrust-pc-sdk .ot-ven-dets span{color:dimgray;padding:0;vertical-align:baseline}#onetrust-pc-sdk .ot-ven-dets .ot-ven-pur h5{border-top:1px solid #e9e9e9;border-bottom:1px solid #e9e9e9;padding-bottom:5px;margin-bottom:5px;font-weight:bold}#onetrust-pc-sdk #ot-host-lst .ot-sel-all{float:right;position:relative;margin-right:42px;top:10px}#onetrust-pc-sdk #ot-host-lst .ot-sel-all input[type=checkbox]{width:auto;height:auto}#onetrust-pc-sdk #ot-host-lst .ot-sel-all label{height:20px;width:20px;padding-left:0px}#onetrust-pc-sdk #ot-host-lst .ot-acc-txt{overflow:hidden;width:95%}#onetrust-pc-sdk .ot-host-hdr{position:relative;z-index:1;pointer-events:none;width:calc(100% - 125px);float:left}#onetrust-pc-sdk .ot-host-name,#onetrust-pc-sdk .ot-host-desc{display:inline-block;width:90%}#onetrust-pc-sdk .ot-host-name{pointer-events:none}#onetrust-pc-sdk .ot-host-hdr>a{text-decoration:underline;font-size:.82em;position:relative;z-index:2;float:left;margin-bottom:5px;pointer-events:initial}#onetrust-pc-sdk .ot-host-name+a{margin-top:5px}#onetrust-pc-sdk .ot-host-name,#onetrust-pc-sdk .ot-host-name a,#onetrust-pc-sdk .ot-host-desc,#onetrust-pc-sdk .ot-host-info{color:dimgray;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-host-name,#onetrust-pc-sdk .ot-host-name a{font-weight:bold;font-size:.82em;line-height:1.3}#onetrust-pc-sdk .ot-host-name a{font-size:1em}#onetrust-pc-sdk .ot-host-expand{margin-top:3px;margin-bottom:3px;clear:both;display:block;color:#3860be;font-size:.72em;font-weight:normal}#onetrust-pc-sdk .ot-host-expand *{font-size:inherit}#onetrust-pc-sdk .ot-host-desc,#onetrust-pc-sdk .ot-host-info{font-size:.688em;line-height:1.4;font-weight:normal}#onetrust-pc-sdk .ot-host-desc{margin-top:10px}#onetrust-pc-sdk .ot-host-opt{margin:0;font-size:inherit;display:inline-block;width:100%}#onetrust-pc-sdk .ot-host-opt li>div div{font-size:.8em;padding:5px 0}#onetrust-pc-sdk .ot-host-opt li>div div:nth-child(1){width:30%;float:left}#onetrust-pc-sdk .ot-host-opt li>div div:nth-child(2){width:70%;float:left;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-host-info{border:none;display:inline-block;width:calc(100% - 10px);padding:10px;margin-bottom:10px;background-color:#f8f8f8}#onetrust-pc-sdk .ot-host-info>div{overflow:auto}#onetrust-pc-sdk #no-results{text-align:center;margin-top:30px}#onetrust-pc-sdk #no-results p{font-size:1em;color:#2e3644;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk #no-results p span{font-weight:bold}#onetrust-pc-sdk #ot-fltr-modal{width:100%;height:auto;display:none;-moz-transition:.2s ease;-o-transition:.2s ease;-webkit-transition:2s ease;transition:.2s ease;overflow:hidden;opacity:1;right:0}#onetrust-pc-sdk #ot-fltr-modal .ot-label-txt{display:inline-block;font-size:.85em;color:dimgray}#onetrust-pc-sdk #ot-fltr-cnt{z-index:2147483646;background-color:#fff;position:absolute;height:90%;max-height:300px;width:325px;left:210px;margin-top:10px;margin-bottom:20px;padding-right:10px;border-radius:3px;-webkit-box-shadow:0px 0px 12px 2px #c7c5c7;-moz-box-shadow:0px 0px 12px 2px #c7c5c7;box-shadow:0px 0px 12px 2px #c7c5c7}#onetrust-pc-sdk .ot-fltr-scrlcnt{overflow-y:auto;overflow-x:hidden;clear:both;max-height:calc(100% - 60px)}#onetrust-pc-sdk #ot-anchor{border:12px solid rgba(0,0,0,0);display:none;position:absolute;z-index:2147483647;right:55px;top:75px;transform:rotate(45deg);-o-transform:rotate(45deg);-ms-transform:rotate(45deg);-webkit-transform:rotate(45deg);background-color:#fff;-webkit-box-shadow:-3px -3px 5px -2px #c7c5c7;-moz-box-shadow:-3px -3px 5px -2px #c7c5c7;box-shadow:-3px -3px 5px -2px #c7c5c7}#onetrust-pc-sdk .ot-fltr-btns{margin-left:15px}#onetrust-pc-sdk #filter-apply-handler{margin-right:15px}#onetrust-pc-sdk .ot-fltr-opt{margin-bottom:5px;margin-left:15px;min-height:20px;width:75%;position:relative}#onetrust-pc-sdk .ot-fltr-opt p{display:inline-block;margin:0;font-size:.9em;color:#2e3644}#onetrust-pc-sdk .ot-chkbox label span{font-size:.85em;color:dimgray}#onetrust-pc-sdk .ot-chkbox input[type=checkbox]+label::after{content:none;color:#fff}#onetrust-pc-sdk .ot-chkbox input[type=checkbox]:checked+label::after{content:""}#onetrust-pc-sdk .ot-chkbox input[type=checkbox][aria-checked=mixed]+label::after{content:""}#onetrust-pc-sdk .ot-chkbox input[type=checkbox]:focus+label::before{outline-style:solid;outline-width:2px;outline-style:auto}#onetrust-pc-sdk #ot-selall-vencntr,#onetrust-pc-sdk #ot-selall-adtlvencntr,#onetrust-pc-sdk #ot-selall-hostcntr,#onetrust-pc-sdk #ot-selall-licntr,#onetrust-pc-sdk #ot-selall-gnvencntr{right:15px;position:relative;width:20px;height:20px;float:right}#onetrust-pc-sdk #ot-selall-vencntr label,#onetrust-pc-sdk #ot-selall-adtlvencntr label,#onetrust-pc-sdk #ot-selall-hostcntr label,#onetrust-pc-sdk #ot-selall-licntr label,#onetrust-pc-sdk #ot-selall-gnvencntr label{float:left;padding-left:0}#onetrust-pc-sdk #ot-ven-lst:first-child{border-top:1px solid #e2e2e2}#onetrust-pc-sdk ul{list-style:none;padding:0}#onetrust-pc-sdk ul li{position:relative;margin:0;padding:15px 15px 15px 10px;border-bottom:1px solid #e2e2e2}#onetrust-pc-sdk ul li h3,#onetrust-pc-sdk ul li h4{font-size:.75em;color:#656565;margin:0;display:inline-block;width:70%;height:auto;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk ul li p{margin:0;font-size:.7em}#onetrust-pc-sdk ul li input[type=checkbox]{position:absolute;cursor:pointer;width:100%;height:100%;opacity:0;margin:0;top:0;left:0}#onetrust-pc-sdk .ot-cat-item>button:focus,#onetrust-pc-sdk .ot-acc-cntr>button:focus,#onetrust-pc-sdk li>button:focus{outline:#000 solid 2px}#onetrust-pc-sdk .ot-cat-item>button,#onetrust-pc-sdk .ot-acc-cntr>button,#onetrust-pc-sdk li>button{position:absolute;cursor:pointer;width:100%;height:100%;margin:0;top:0;left:0;z-index:1;max-width:none;border:none}#onetrust-pc-sdk .ot-cat-item>button[aria-expanded=false]~.ot-acc-txt,#onetrust-pc-sdk .ot-acc-cntr>button[aria-expanded=false]~.ot-acc-txt,#onetrust-pc-sdk li>button[aria-expanded=false]~.ot-acc-txt{margin-top:0;max-height:0;opacity:0;overflow:hidden;width:100%;transition:.25s ease-out;display:none}#onetrust-pc-sdk .ot-cat-item>button[aria-expanded=true]~.ot-acc-txt,#onetrust-pc-sdk .ot-acc-cntr>button[aria-expanded=true]~.ot-acc-txt,#onetrust-pc-sdk li>button[aria-expanded=true]~.ot-acc-txt{transition:.1s ease-in;margin-top:10px;width:100%;overflow:auto;display:block}#onetrust-pc-sdk .ot-cat-item>button[aria-expanded=true]~.ot-acc-grpcntr,#onetrust-pc-sdk .ot-acc-cntr>button[aria-expanded=true]~.ot-acc-grpcntr,#onetrust-pc-sdk li>button[aria-expanded=true]~.ot-acc-grpcntr{width:auto;margin-top:0px;padding-bottom:10px}#onetrust-pc-sdk .ot-host-item>button:focus,#onetrust-pc-sdk .ot-ven-item>button:focus{outline:0;border:2px solid #000}#onetrust-pc-sdk .ot-hide-acc>button{pointer-events:none}#onetrust-pc-sdk .ot-hide-acc .ot-plus-minus>*,#onetrust-pc-sdk .ot-hide-acc .ot-arw-cntr>*{visibility:hidden}#onetrust-pc-sdk .ot-hide-acc .ot-acc-hdr{min-height:30px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt){padding-right:10px;width:calc(100% - 37px);margin-top:10px;max-height:calc(100% - 90px)}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk{background-color:#f9f9fc;border:1px solid #e2e2e2;width:calc(100% - 2px);padding-bottom:5px;padding-top:5px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt{border:unset;background-color:unset}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt .ot-sel-all-hdr{display:none}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt .ot-sel-all{padding-right:.5rem}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt .ot-sel-all .ot-chkbox{right:0}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) .ot-sel-all{padding-right:34px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) .ot-sel-all-chkbox{width:auto}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) ul li{border:1px solid #e2e2e2;margin-bottom:10px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) .ot-acc-cntr>.ot-acc-hdr{padding:10px 0 10px 15px}#onetrust-pc-sdk.ot-addtl-vendors .ot-sel-all-chkbox{float:right}#onetrust-pc-sdk.ot-addtl-vendors .ot-plus-minus~.ot-sel-all-chkbox{right:34px}#onetrust-pc-sdk.ot-addtl-vendors #ot-ven-lst:first-child{border-top:none}#onetrust-pc-sdk .ot-acc-cntr{position:relative;border-left:1px solid #e2e2e2;border-right:1px solid #e2e2e2;border-bottom:1px solid #e2e2e2}#onetrust-pc-sdk .ot-acc-cntr input{z-index:1}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr{background-color:#f9f9fc;padding:5px 0 5px 15px;width:auto}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr .ot-plus-minus{vertical-align:middle;top:auto}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr .ot-arw-cntr{right:10px}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr input{z-index:2}#onetrust-pc-sdk .ot-acc-cntr.ot-add-tech .ot-acc-hdr{padding:10px 0 10px 15px}#onetrust-pc-sdk .ot-acc-cntr>input[type=checkbox]:checked~.ot-acc-hdr{border-bottom:1px solid #e2e2e2}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-txt{padding-left:10px;padding-right:10px}#onetrust-pc-sdk .ot-acc-cntr button[aria-expanded=true]~.ot-acc-txt{width:auto}#onetrust-pc-sdk .ot-acc-cntr .ot-addtl-venbox{display:none}#onetrust-pc-sdk .ot-vlst-cntr{margin-bottom:0;width:100%}#onetrust-pc-sdk .ot-vensec-title{font-size:.813em;vertical-align:middle;display:inline-block}#onetrust-pc-sdk .category-vendors-list-handler,#onetrust-pc-sdk .category-vendors-list-handler+a{margin-left:0;margin-top:10px}#onetrust-pc-sdk #ot-selall-vencntr.line-through label::after,#onetrust-pc-sdk #ot-selall-adtlvencntr.line-through label::after,#onetrust-pc-sdk #ot-selall-licntr.line-through label::after,#onetrust-pc-sdk #ot-selall-hostcntr.line-through label::after,#onetrust-pc-sdk #ot-selall-gnvencntr.line-through label::after{height:auto;border-left:0;transform:none;-o-transform:none;-ms-transform:none;-webkit-transform:none;left:5px;top:9px}#onetrust-pc-sdk #ot-category-title{float:left;padding-bottom:10px;font-size:1em;width:100%}#onetrust-pc-sdk .ot-cat-grp{margin-top:10px}#onetrust-pc-sdk .ot-cat-item{line-height:1.1;margin-top:10px;display:inline-block;width:100%}#onetrust-pc-sdk .ot-btn-container{text-align:right}#onetrust-pc-sdk .ot-btn-container button{display:inline-block;font-size:.75em;letter-spacing:.08em;margin-top:19px}#onetrust-pc-sdk #close-pc-btn-handler.ot-close-icon{position:absolute;top:10px;right:0;z-index:1;padding:0;background-color:rgba(0,0,0,0);border:none}#onetrust-pc-sdk #close-pc-btn-handler.ot-close-icon svg{display:block;height:10px;width:10px}#onetrust-pc-sdk #clear-filters-handler{margin-top:20px;margin-bottom:10px;float:right;max-width:200px;text-decoration:none;color:#3860be;font-size:.9em;font-weight:bold;background-color:rgba(0,0,0,0);border-color:rgba(0,0,0,0);padding:1px}#onetrust-pc-sdk #clear-filters-handler:hover{color:#2285f7}#onetrust-pc-sdk #clear-filters-handler:focus{outline:#000 solid 1px}#onetrust-pc-sdk .ot-enbl-chr h4~.ot-tgl,#onetrust-pc-sdk .ot-enbl-chr h4~.ot-always-active{right:45px}#onetrust-pc-sdk .ot-enbl-chr h4~.ot-tgl+.ot-tgl{right:120px}#onetrust-pc-sdk .ot-enbl-chr .ot-pli-hdr.ot-leg-border-color span:first-child{width:90px}#onetrust-pc-sdk .ot-enbl-chr li.ot-subgrp>h5+.ot-tgl-cntr{padding-right:25px}#onetrust-pc-sdk .ot-plus-minus{width:20px;height:20px;font-size:1.5em;position:relative;display:inline-block;margin-right:5px;top:3px}#onetrust-pc-sdk .ot-plus-minus span{position:absolute;background:#27455c;border-radius:1px}#onetrust-pc-sdk .ot-plus-minus span:first-of-type{top:25%;bottom:25%;width:10%;left:45%}#onetrust-pc-sdk .ot-plus-minus span:last-of-type{left:25%;right:25%;height:10%;top:45%}#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-arw,#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-plus-minus span:first-of-type,#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-plus-minus span:last-of-type{transform:rotate(90deg)}#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-plus-minus span:last-of-type{left:50%;right:50%}#onetrust-pc-sdk #ot-selall-vencntr label,#onetrust-pc-sdk #ot-selall-adtlvencntr label,#onetrust-pc-sdk #ot-selall-hostcntr label,#onetrust-pc-sdk #ot-selall-licntr label{position:relative;display:inline-block;width:20px;height:20px}#onetrust-pc-sdk .ot-host-item .ot-plus-minus,#onetrust-pc-sdk .ot-ven-item .ot-plus-minus{float:left;margin-right:8px;top:10px}#onetrust-pc-sdk .ot-ven-item ul{list-style:none inside;font-size:100%;margin:0}#onetrust-pc-sdk .ot-ven-item ul li{margin:0 !important;padding:0;border:none !important}#onetrust-pc-sdk .ot-pli-hdr{color:#77808e;overflow:hidden;padding-top:7.5px;padding-bottom:7.5px;width:calc(100% - 2px);border-top-left-radius:3px;border-top-right-radius:3px}#onetrust-pc-sdk .ot-pli-hdr span:first-child{top:50%;transform:translateY(50%);max-width:90px}#onetrust-pc-sdk .ot-pli-hdr span:last-child{padding-right:10px;max-width:95px;text-align:center}#onetrust-pc-sdk .ot-li-title{float:right;font-size:.813em}#onetrust-pc-sdk .ot-pli-hdr.ot-leg-border-color{background-color:#f4f4f4;border:1px solid #d8d8d8}#onetrust-pc-sdk .ot-pli-hdr.ot-leg-border-color span:first-child{text-align:left;width:70px}#onetrust-pc-sdk li.ot-subgrp>h5,#onetrust-pc-sdk .ot-cat-header{width:calc(100% - 130px)}#onetrust-pc-sdk li.ot-subgrp>h5+.ot-tgl-cntr{padding-left:13px}#onetrust-pc-sdk .ot-acc-grpcntr .ot-acc-grpdesc{margin-bottom:5px}#onetrust-pc-sdk .ot-acc-grpcntr .ot-subgrp-cntr{border-top:1px solid #d8d8d8}#onetrust-pc-sdk .ot-acc-grpcntr .ot-vlst-cntr+.ot-subgrp-cntr{border-top:none}#onetrust-pc-sdk .ot-acc-hdr .ot-arw-cntr+.ot-tgl-cntr,#onetrust-pc-sdk .ot-acc-txt h4+.ot-tgl-cntr{padding-left:13px}#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item .ot-subgrp>h5,#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item .ot-cat-header{width:calc(100% - 145px)}#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item h5+.ot-tgl-cntr,#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item .ot-cat-header+.ot-tgl{padding-left:28px}#onetrust-pc-sdk .ot-sel-all-hdr,#onetrust-pc-sdk .ot-sel-all-chkbox{display:inline-block;width:100%;position:relative}#onetrust-pc-sdk .ot-sel-all-chkbox{z-index:1}#onetrust-pc-sdk .ot-sel-all{margin:0;position:relative;padding-right:23px;float:right}#onetrust-pc-sdk .ot-consent-hdr,#onetrust-pc-sdk .ot-li-hdr{float:right;font-size:.812em;line-height:normal;text-align:center;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-li-hdr{max-width:100px;padding-right:10px}#onetrust-pc-sdk .ot-consent-hdr{max-width:55px}#onetrust-pc-sdk #ot-selall-licntr{display:block;width:21px;height:auto;float:right;position:relative;right:80px}#onetrust-pc-sdk #ot-selall-licntr label{position:absolute}#onetrust-pc-sdk .ot-ven-ctgl{margin-left:66px}#onetrust-pc-sdk .ot-ven-litgl+.ot-arw-cntr{margin-left:81px}#onetrust-pc-sdk .ot-enbl-chr .ot-host-cnt .ot-tgl-cntr{width:auto}#onetrust-pc-sdk #ot-lst-cnt:not(.ot-host-cnt) .ot-tgl-cntr{width:auto;top:auto;height:20px}#onetrust-pc-sdk #ot-lst-cnt .ot-chkbox{position:relative;display:inline-block;width:20px;height:20px}#onetrust-pc-sdk #ot-lst-cnt .ot-chkbox label{position:absolute;padding:0;width:20px;height:20px}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info-cntr{border:1px solid #d8d8d8;padding:.75rem 2rem;padding-bottom:0;width:auto;margin-top:.5rem}#onetrust-pc-sdk .ot-acc-grpdesc+.ot-leg-btn-container{padding-left:20px;padding-right:20px;width:calc(100% - 40px);margin-bottom:5px}#onetrust-pc-sdk .ot-subgrp .ot-leg-btn-container{margin-bottom:5px}#onetrust-pc-sdk #ot-ven-lst .ot-leg-btn-container{margin-top:10px}#onetrust-pc-sdk .ot-leg-btn-container{display:inline-block;width:100%;margin-bottom:10px}#onetrust-pc-sdk .ot-leg-btn-container button{height:auto;padding:6.5px 8px;margin-bottom:0;letter-spacing:0;font-size:.75em;line-height:normal}#onetrust-pc-sdk .ot-leg-btn-container svg{display:none;height:14px;width:14px;padding-right:5px;vertical-align:sub}#onetrust-pc-sdk .ot-active-leg-btn{cursor:default;pointer-events:none}#onetrust-pc-sdk .ot-active-leg-btn svg{display:inline-block}#onetrust-pc-sdk .ot-remove-objection-handler{text-decoration:underline;padding:0;font-size:.75em;font-weight:600;line-height:1;padding-left:10px}#onetrust-pc-sdk .ot-obj-leg-btn-handler span{font-weight:bold;text-align:center;font-size:inherit;line-height:1.5}#onetrust-pc-sdk.ot-close-btn-link #close-pc-btn-handler{border:none;height:auto;line-height:1.5;text-decoration:underline;font-size:.69em;background:none;right:15px;top:15px;width:auto;font-weight:normal}#onetrust-pc-sdk .ot-pgph-link{font-size:.813em !important;margin-top:5px;position:relative}#onetrust-pc-sdk .ot-pgph-link.ot-pgph-link-subgroup{margin-bottom:1rem}#onetrust-pc-sdk .ot-pgph-contr{margin:0 2.5rem}#onetrust-pc-sdk .ot-pgph-title{font-size:1.18rem;margin-bottom:2rem}#onetrust-pc-sdk .ot-pgph-desc{font-size:1rem;font-weight:400;margin-bottom:2rem;line-height:1.5rem}#onetrust-pc-sdk .ot-pgph-desc:not(:last-child):after{content:"";width:96%;display:block;margin:0 auto;padding-bottom:2rem;border-bottom:1px solid #e9e9e9}#onetrust-pc-sdk .ot-cat-header{float:left;font-weight:600;font-size:.875em;line-height:1.5;max-width:90%;vertical-align:middle}#onetrust-pc-sdk .ot-vnd-item>button:focus{outline:#000 solid 2px}#onetrust-pc-sdk .ot-vnd-item>button{position:absolute;cursor:pointer;width:100%;height:100%;margin:0;top:0;left:0;z-index:1;max-width:none;border:none}#onetrust-pc-sdk .ot-vnd-item>button[aria-expanded=false]~.ot-acc-txt{margin-top:0;max-height:0;opacity:0;overflow:hidden;width:100%;transition:.25s ease-out;display:none}#onetrust-pc-sdk .ot-vnd-item>button[aria-expanded=true]~.ot-acc-txt{transition:.1s ease-in;margin-top:10px;width:100%;overflow:auto;display:block}#onetrust-pc-sdk .ot-vnd-item>button[aria-expanded=true]~.ot-acc-grpcntr{width:auto;margin-top:0px;padding-bottom:10px}#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item{position:relative;border-radius:2px;margin:0;padding:0;border:1px solid #d8d8d8;border-top:none;width:calc(100% - 2px);float:left}#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item:first-of-type{margin-top:10px;border-top:1px solid #d8d8d8}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpdesc{padding-left:20px;padding-right:20px;width:calc(100% - 40px);font-size:.812em;margin-bottom:10px;margin-top:15px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpdesc>ul{padding-top:10px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpdesc>ul li{padding-top:0;line-height:1.5;padding-bottom:10px}#onetrust-pc-sdk .ot-accordion-layout div+.ot-acc-grpdesc{margin-top:5px}#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr:first-child{margin-top:10px}#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr:last-child,#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr:last-child{margin-bottom:5px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{padding-top:11.5px;padding-bottom:11.5px;padding-left:20px;padding-right:20px;width:calc(100% - 40px);display:inline-block}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-txt{width:100%;padding:0}#onetrust-pc-sdk .ot-accordion-layout .ot-subgrp-cntr{padding-left:20px;padding-right:15px;padding-bottom:0;width:calc(100% - 35px)}#onetrust-pc-sdk .ot-accordion-layout .ot-subgrp{padding-right:5px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpcntr{z-index:1;position:relative}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header+.ot-arw-cntr{position:absolute;top:50%;transform:translateY(-50%);right:20px;margin-top:-2px}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header+.ot-arw-cntr .ot-arw{width:15px;height:20px;margin-left:5px;color:dimgray}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{float:none;color:#2e3644;margin:0;display:inline-block;height:auto;word-wrap:break-word;min-height:inherit}#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr,#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr{padding-left:20px;width:calc(100% - 20px);display:inline-block;margin-top:0;padding-bottom:2px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{position:relative;min-height:25px}#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl,#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active{position:absolute;top:50%;transform:translateY(-50%);right:20px}#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl+.ot-tgl{right:95px}#onetrust-pc-sdk .ot-accordion-layout .category-vendors-list-handler,#onetrust-pc-sdk .ot-accordion-layout .category-vendors-list-handler+a{margin-top:5px}#onetrust-pc-sdk #ot-lst-cnt{margin-top:1rem;max-height:calc(100% - 96px)}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info-cntr{border:1px solid #d8d8d8;padding:.75rem 2rem;padding-bottom:0;width:auto;margin-top:.5rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info{margin-bottom:1rem;padding-left:.75rem;padding-right:.75rem;display:flex;flex-direction:column}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info[data-vnd-info-key*=DPOEmail]{border-top:1px solid #d8d8d8;padding-top:1rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info[data-vnd-info-key*=DPOLink]{border-bottom:1px solid #d8d8d8;padding-bottom:1rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info .ot-vnd-lbl{font-weight:bold;font-size:.85em;margin-bottom:.5rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info .ot-vnd-cnt{margin-left:.5rem;font-weight:500;font-size:.85rem}#onetrust-pc-sdk .ot-vs-list,#onetrust-pc-sdk .ot-vnd-serv{width:auto;padding:1rem 1.25rem;padding-bottom:0}#onetrust-pc-sdk .ot-vs-list .ot-vnd-serv-hdr-cntr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-serv-hdr-cntr{padding-bottom:.75rem;border-bottom:1px solid #d8d8d8}#onetrust-pc-sdk .ot-vs-list .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr{font-weight:600;font-size:.95em;line-height:2;margin-left:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item{border:none;margin:0;padding:0}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item button,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item button{outline:none;border-bottom:1px solid #d8d8d8}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item button[aria-expanded=true],#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item button[aria-expanded=true]{border-bottom:none}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item:first-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item:first-child{margin-top:.25rem;border-top:unset}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item:last-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item:last-child{margin-bottom:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item:last-child button,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item:last-child button{border-bottom:none}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info-cntr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info-cntr{border:1px solid #d8d8d8;padding:.75rem 1.75rem;padding-bottom:0;width:auto;margin-top:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info{margin-bottom:1rem;padding-left:.75rem;padding-right:.75rem;display:flex;flex-direction:column}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOEmail],#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOEmail]{border-top:1px solid #d8d8d8;padding-top:1rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOLink],#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOLink]{border-bottom:1px solid #d8d8d8;padding-bottom:1rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info .ot-vnd-lbl,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info .ot-vnd-lbl{font-weight:bold;font-size:.85em;margin-bottom:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info .ot-vnd-cnt,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info .ot-vnd-cnt{margin-left:.5rem;font-weight:500;font-size:.85rem}#onetrust-pc-sdk .ot-vs-list.ot-vnd-subgrp-cnt,#onetrust-pc-sdk .ot-vnd-serv.ot-vnd-subgrp-cnt{padding-left:40px}#onetrust-pc-sdk .ot-vs-list.ot-vnd-subgrp-cnt .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr,#onetrust-pc-sdk .ot-vnd-serv.ot-vnd-subgrp-cnt .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr{font-size:.8em}#onetrust-pc-sdk .ot-vs-list.ot-vnd-subgrp-cnt .ot-cat-header,#onetrust-pc-sdk .ot-vnd-serv.ot-vnd-subgrp-cnt .ot-cat-header{font-size:.8em}#onetrust-pc-sdk .ot-subgrp-cntr .ot-vnd-serv{margin-bottom:1rem;padding:1rem .95rem}#onetrust-pc-sdk .ot-subgrp-cntr .ot-vnd-serv .ot-vnd-serv-hdr-cntr{padding-bottom:.75rem;border-bottom:1px solid #d8d8d8}#onetrust-pc-sdk .ot-subgrp-cntr .ot-vnd-serv .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr{font-weight:700;font-size:.8em;line-height:20px;margin-left:.82rem}#onetrust-pc-sdk .ot-subgrp-cntr .ot-cat-header{font-weight:700;font-size:.8em;line-height:20px}#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-vnd-serv .ot-vnd-lst-cont .ot-accordion-layout .ot-acc-hdr div.ot-chkbox{margin-left:.82rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr{padding:.7rem 0;margin:0;display:flex;width:100%;align-items:center;justify-content:space-between}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr div:first-child,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr div:first-child,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr div:first-child{margin-left:.5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr div:last-child,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr div:last-child,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr div:last-child{margin-right:.5rem;margin-left:.5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-always-active{position:relative;right:unset;top:unset;transform:unset}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-plus-minus{top:0}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-arw-cntr{float:none;top:unset;right:unset;transform:unset;margin-top:-2px;position:relative}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-cat-header{flex:1;margin:0 .5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-tgl{position:relative;transform:none;right:0;top:0;float:none}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox{position:relative;margin:0 .5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox label{padding:0}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox label::before{position:relative}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox input{position:absolute;cursor:pointer;width:100%;height:100%;opacity:0;margin:0;top:0;left:0;z-index:1}#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp .ot-acc-hdr h5.ot-cat-header,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp .ot-acc-hdr h4.ot-cat-header{margin:0}#onetrust-pc-sdk .ot-vs-config .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp h5{top:0;line-height:20px}#onetrust-pc-sdk .ot-vs-list{display:flex;flex-direction:column;padding:0;margin:.5rem 4px}#onetrust-pc-sdk .ot-vs-selc-all{display:flex;padding:0;float:unset;align-items:center;justify-content:flex-start}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf{justify-content:flex-end}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf.ot-caret-conf .ot-sel-all-chkbox{margin-right:48px}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf .ot-sel-all-chkbox{margin:0;padding:0;margin-right:14px;justify-content:flex-end}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf #ot-selall-vencntr.ot-chkbox,#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf #ot-selall-vencntr.ot-tgl{display:inline-block;right:unset;width:auto;height:auto;float:none}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf #ot-selall-vencntr label{width:45px;height:25px}#onetrust-pc-sdk .ot-vs-selc-all .ot-sel-all-chkbox{margin-right:11px;margin-left:.75rem;display:flex;align-items:center}#onetrust-pc-sdk .ot-vs-selc-all .sel-all-hdr{margin:0 1.25rem;font-size:.812em;line-height:normal;text-align:center;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-vnd-list-cnt #ot-selall-vencntr.ot-chkbox{float:unset;right:0}#onetrust-pc-sdk[dir=rtl] #ot-back-arw,#onetrust-pc-sdk[dir=rtl] input~.ot-acc-hdr .ot-arw{transform:rotate(180deg);-o-transform:rotate(180deg);-ms-transform:rotate(180deg);-webkit-transform:rotate(180deg)}#onetrust-pc-sdk[dir=rtl] input:checked~.ot-acc-hdr .ot-arw{transform:rotate(270deg);-o-transform:rotate(270deg);-ms-transform:rotate(270deg);-webkit-transform:rotate(270deg)}#onetrust-pc-sdk[dir=rtl] .ot-chkbox label::after{transform:rotate(45deg);-webkit-transform:rotate(45deg);-o-transform:rotate(45deg);-ms-transform:rotate(45deg);border-left:0;border-right:3px solid}#onetrust-pc-sdk[dir=rtl] .ot-search-cntr>svg{right:0}@media only screen and (max-width: 600px){#onetrust-pc-sdk.otPcCenter{left:0;min-width:100%;height:100%;top:0;border-radius:0}#onetrust-pc-sdk #ot-pc-content,#onetrust-pc-sdk.ot-ftr-stacked .ot-btn-container{margin:1px 3px 0 10px;padding-right:10px;width:calc(100% - 23px)}#onetrust-pc-sdk .ot-btn-container button{max-width:none;letter-spacing:.01em}#onetrust-pc-sdk #close-pc-btn-handler{top:10px;right:17px}#onetrust-pc-sdk p{font-size:.7em}#onetrust-pc-sdk #ot-pc-hdr{margin:10px 10px 0 5px;width:calc(100% - 15px)}#onetrust-pc-sdk .vendor-search-handler{font-size:1em}#onetrust-pc-sdk #ot-back-arw{margin-left:12px}#onetrust-pc-sdk #ot-lst-cnt{margin:0;padding:0 5px 0 10px;min-width:95%}#onetrust-pc-sdk .switch+p{max-width:80%}#onetrust-pc-sdk .ot-ftr-stacked button{width:100%}#onetrust-pc-sdk #ot-fltr-cnt{max-width:320px;width:90%;border-top-right-radius:0;border-bottom-right-radius:0;margin:0;margin-left:15px;left:auto;right:40px;top:85px}#onetrust-pc-sdk .ot-fltr-opt{margin-left:25px;margin-bottom:10px}#onetrust-pc-sdk .ot-pc-refuse-all-handler{margin-bottom:0}#onetrust-pc-sdk #ot-fltr-cnt{right:40px}}@media only screen and (max-width: 476px){#onetrust-pc-sdk .ot-fltr-cntr,#onetrust-pc-sdk #ot-fltr-cnt{right:10px}#onetrust-pc-sdk #ot-anchor{right:25px}#onetrust-pc-sdk button{width:100%}#onetrust-pc-sdk:not(.ot-addtl-vendors) #ot-pc-lst:not(.ot-enbl-chr) .ot-sel-all{padding-right:9px}#onetrust-pc-sdk:not(.ot-addtl-vendors) #ot-pc-lst:not(.ot-enbl-chr) .ot-tgl-cntr{right:0}}@media(min-width: 768px){#onetrust-pc-sdk.ot-tgl-with-label .ot-label-status{display:inline}#onetrust-pc-sdk.ot-tgl-with-label #ot-pc-lst .ot-label-status{display:none}}@media only screen and (max-width: 896px)and (max-height: 425px)and (orientation: landscape){#onetrust-pc-sdk.otPcCenter{left:0;top:0;min-width:100%;height:100%;border-radius:0}#onetrust-pc-sdk .ot-pc-header{height:auto;min-height:20px}#onetrust-pc-sdk .ot-pc-header .ot-pc-logo{max-height:30px}#onetrust-pc-sdk .ot-pc-footer{max-height:60px;overflow-y:auto}#onetrust-pc-sdk #ot-pc-content,#onetrust-pc-sdk #ot-pc-lst{bottom:70px}#onetrust-pc-sdk.ot-ftr-stacked #ot-pc-content{bottom:70px}#onetrust-pc-sdk #ot-anchor{left:initial;right:50px}#onetrust-pc-sdk #ot-lst-title{margin-top:12px}#onetrust-pc-sdk #ot-lst-title *{font-size:inherit}#onetrust-pc-sdk #ot-pc-hdr input{margin-right:0;padding-right:45px}#onetrust-pc-sdk .switch+p{max-width:85%}#onetrust-pc-sdk #ot-sel-blk{position:static}#onetrust-pc-sdk #ot-pc-lst{overflow:auto}#onetrust-pc-sdk #ot-lst-cnt{max-height:none;overflow:initial}#onetrust-pc-sdk #ot-lst-cnt.no-results{height:auto}#onetrust-pc-sdk input{font-size:1em !important}#onetrust-pc-sdk p{font-size:.6em}#onetrust-pc-sdk #ot-fltr-modal{width:100%;top:0}#onetrust-pc-sdk ul li p,#onetrust-pc-sdk .category-vendors-list-handler,#onetrust-pc-sdk .category-vendors-list-handler+a,#onetrust-pc-sdk .category-host-list-handler{font-size:.6em}#onetrust-pc-sdk.ot-shw-fltr #ot-anchor{display:none !important}#onetrust-pc-sdk.ot-shw-fltr #ot-pc-lst{height:100% !important;overflow:hidden;top:0px}#onetrust-pc-sdk.ot-shw-fltr #ot-fltr-cnt{margin:0;height:100%;max-height:none;padding:10px;top:0;width:calc(100% - 20px);position:absolute;right:0;left:0;max-width:none}#onetrust-pc-sdk.ot-shw-fltr .ot-fltr-scrlcnt{max-height:calc(100% - 65px)}}
            #onetrust-consent-sdk #onetrust-pc-sdk,
                #onetrust-consent-sdk #ot-search-cntr,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-switch.ot-toggle,
                #onetrust-consent-sdk #onetrust-pc-sdk ot-grp-hdr1 .checkbox,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-title:after
                ,#onetrust-consent-sdk #onetrust-pc-sdk #ot-sel-blk,
                        #onetrust-consent-sdk #onetrust-pc-sdk #ot-fltr-cnt,
                        #onetrust-consent-sdk #onetrust-pc-sdk #ot-anchor {
                    background-color: #FFF;
                }
               
            #onetrust-consent-sdk #onetrust-pc-sdk h3,
                #onetrust-consent-sdk #onetrust-pc-sdk h4,
                #onetrust-consent-sdk #onetrust-pc-sdk h5,
                #onetrust-consent-sdk #onetrust-pc-sdk h6,
                #onetrust-consent-sdk #onetrust-pc-sdk p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-ven-lst .ot-ven-opts p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-desc,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-title,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-li-title,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-sel-all-hdr span,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-info,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-fltr-modal #modal-header,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-checkbox label span,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst #ot-sel-blk p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst #ot-lst-title h3,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst .back-btn-handler p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst .ot-ven-name,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst #ot-ven-lst .consent-category,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-inactive-leg-btn,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-label-status,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-chkbox label span,
                #onetrust-consent-sdk #onetrust-pc-sdk #clear-filters-handler,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-optout-signal
                {
                    color: #2E2E2E;
                }
             #onetrust-consent-sdk #onetrust-pc-sdk .privacy-notice-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-pgph-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk .category-vendors-list-handler,
                    #onetrust-consent-sdk #onetrust-pc-sdk .category-vendors-list-handler + a,
                    #onetrust-consent-sdk #onetrust-pc-sdk .category-host-list-handler,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-ven-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-ven-legclaim-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-name a,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-acc-hdr .ot-host-expand,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-info a,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-content #ot-pc-desc .ot-link-btn,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info a,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info a
                    {
                        color: #007398;
                    }
            #onetrust-consent-sdk #onetrust-pc-sdk .category-vendors-list-handler:hover { text-decoration: underline;}
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-grpcntr.ot-acc-txt,
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-txt .ot-subgrp-tgl .ot-switch.ot-toggle
             {
                background-color: #F8F8F8;
            }
             #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-info,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-txt .ot-ven-dets
                            {
                                background-color: #F8F8F8;
                            }
        #onetrust-consent-sdk #onetrust-pc-sdk
            button:not(#clear-filters-handler):not(.ot-close-icon):not(#filter-btn-handler):not(.ot-remove-objection-handler):not(.ot-obj-leg-btn-handler):not([aria-expanded]):not(.ot-link-btn),
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-active-leg-btn {
                background-color: #007398;border-color: #007398;
                color: #FFF;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-active-menu {
                border-color: #007398;
            }
            
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-remove-objection-handler{
                background-color: transparent;
                border: 1px solid transparent;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-inactive-leg-btn {
                background-color: #FFFFFF;
                color: #4D4D4D; border-color: #4D4D4D;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-tgl input:focus + .ot-switch, .ot-switch .ot-switch-nob, .ot-switch .ot-switch-nob:before,
            #onetrust-pc-sdk .ot-checkbox input[type="checkbox"]:focus + label::before,
            #onetrust-pc-sdk .ot-chkbox input[type="checkbox"]:focus + label::before {
                outline-color: #000000;
                outline-width: 1px;
                outline-offset: 1px;
            }
            #onetrust-pc-sdk .ot-host-item > button:focus, #onetrust-pc-sdk .ot-ven-item > button:focus {
                border: 1px solid #000000;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk *:focus,
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-vlst-cntr > a:focus {
               outline: 1px solid #000000;
               outline-offset: 1px;
            }#onetrust-pc-sdk .ot-vlst-cntr .ot-ext-lnk,  #onetrust-pc-sdk .ot-ven-hdr .ot-ext-lnk{
                    background-image: url('https://cdn.cookielaw.org/logos/static/ot_external_link.svg');
                }
            /*! v6.10.0 2021-01-18 */
div#onetrust-consent-sdk #onetrust-banner-sdk{border-top:2px solid #d0d3d4!important;box-shadow:none;padding:24px}div#onetrust-consent-sdk button{border-radius:0!important;box-shadow:none!important;box-sizing:border-box!important;font-size:20px!important;font-weight:400!important;letter-spacing:0!important;max-width:none!important;white-space:nowrap!important}div#onetrust-consent-sdk button:not(.ot-link-btn){background-color:#007dbc!important;border:2px solid #007dbc!important;color:#fff!important;height:48px!important;padding:0 1em!important;width:auto!important}div#onetrust-consent-sdk button:hover{background-color:#fff!important;border-color:#949494!important;color:#2e2e2e!important}div#onetrust-consent-sdk button.ot-link-btn{color:#007dbc!important;font-size:16px!important;text-decoration:underline}div#onetrust-consent-sdk button.ot-link-btn:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-consent-sdk a,div#onetrust-pc-sdk a{color:#007dbc!important;text-decoration:underline!important}div#onetrust-consent-sdk a,div#onetrust-consent-sdk button,div#onetrust-consent-sdk p:hover{opacity:1!important}div#onetrust-consent-sdk a:focus,div#onetrust-consent-sdk button:focus,div#onetrust-consent-sdk input:focus{outline:2px solid #949494!important;outline-offset:1px!important}div#onetrust-banner-sdk .ot-sdk-container{padding:0;width:auto}div#onetrust-banner-sdk .ot-sdk-row{align-items:flex-start;box-sizing:border-box;display:flex;flex-direction:column;justify-content:space-between;margin:auto;max-width:1152px}div#onetrust-banner-sdk .ot-sdk-row:after{display:none}div#onetrust-banner-sdk #onetrust-group-container,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-group-container,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{flex-grow:1;width:auto}div#onetrust-banner-sdk #onetrust-policy,div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy{margin:0;overflow:visible}div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy-text,div#onetrust-consent-sdk #onetrust-policy-text{font-size:16px;line-height:24px;max-width:44em;margin:0}div#onetrust-consent-sdk #onetrust-policy-text a[href]{font-weight:400;margin-left:8px}div#onetrust-banner-sdk #onetrust-button-group-parent{flex:0 0 auto;margin:32px 0 0;width:100%}div#onetrust-banner-sdk #onetrust-button-group{display:flex;flex-direction:row;flex-wrap:wrap;justify-content:flex-end;margin:-8px}div#onetrust-banner-sdk .banner-actions-container{display:flex;flex:1 0 auto}div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button:last-of-type,div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto;margin:8px;width:auto}div#onetrust-consent-sdk #onetrust-pc-btn-handler{background-color:#fff!important;color:inherit!important}div#onetrust-banner-sdk #onetrust-close-btn-container{display:none}@media only screen and (min-width:556px){div#onetrust-consent-sdk #onetrust-banner-sdk{padding:40px}div#onetrust-banner-sdk #onetrust-policy{margin:0 40px 0 0}div#onetrust-banner-sdk .ot-sdk-row{align-items:center;flex-direction:row}div#onetrust-banner-sdk #onetrust-button-group-parent,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-button-group-parent,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{margin:0;padding:0;width:auto}div#onetrust-banner-sdk #onetrust-button-group,div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group{align-items:stretch;flex-direction:column-reverse;margin:0}div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto}}@media only screen and (min-width:768px){div#onetrust-banner-sdk #onetrust-policy{margin:0 48px 0 0}div#onetrust-consent-sdk #onetrust-banner-sdk{padding:48px}}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-size:16px;line-height:24px}div#onetrust-consent-sdk #onetrust-pc-sdk p,div#onetrust-pc-sdk #ot-pc-desc,div#onetrust-pc-sdk .category-host-list-handler,div#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{font-size:16px;font-weight:400;line-height:24px}div#onetrust-consent-sdk a:hover,div#onetrust-pc-sdk a:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-pc-sdk{border-radius:0;bottom:0;height:auto;left:0;margin:auto;max-width:100%;overflow:hidden;right:0;top:0;width:512px;max-height:800px}div#onetrust-pc-sdk .ot-pc-header{display:none}div#onetrust-pc-sdk #ot-pc-content{overscroll-behavior:contain;padding:0 12px 0 24px;margin:16px 4px 0 0;top:0;right:16px;left:0;width:auto}div#onetrust-pc-sdk #ot-category-title,div#onetrust-pc-sdk #ot-pc-title{font-size:24px;font-weight:400;line-height:32px;margin:0 0 16px}div#onetrust-pc-sdk #ot-pc-desc{padding:0}div#onetrust-pc-sdk #ot-pc-desc a{display:inline}div#onetrust-pc-sdk #accept-recommended-btn-handler{display:none!important}div#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:2px solid #007dbc;outline-offset:-1px;transition:none}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item{border-width:0 0 2px}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item:first-of-type{border-width:2px 0}div#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{padding:8px 0;width:100%}div#onetrust-pc-sdk .ot-plus-minus{transform:translateY(2px)}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item>button{background:0 0!important;border:0!important;height:44px!important;max-width:none!important;width:calc(100% - 48px)!important}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-weight:700}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr{padding:0}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item .ot-acc-grpdesc{padding:0;width:100%}div#onetrust-pc-sdk .ot-acc-grpcntr .ot-subgrp-cntr{border:0;padding:0}div#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp{margin:0}div#onetrust-pc-sdk .ot-always-active-group .ot-cat-header{width:calc(100% - 160px)}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{width:calc(100% - 88px)}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active{color:inherit;font-size:12px;font-weight:400;line-height:1.5;padding-right:48px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:before{border-radius:12px;position:absolute;right:0;top:0;content:'';background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:after{border-radius:50%;position:absolute;right:5px;top:4px;content:'';background-color:#007dbc;height:12px;width:12px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active,div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl{right:2px}div#onetrust-pc-sdk .ot-switch{display:block;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob,div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob{background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob:before{background-color:#737373;height:8px;left:4px;top:4px;width:8px}div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob:before{background-color:#007dbc;height:12px;left:0;top:2px;width:12px}div#onetrust-pc-sdk .ot-tgl input:focus+.ot-switch .ot-switch-nob{box-shadow:0 0;outline:2px solid #007dbc!important;outline-offset:1px;transition:none}div#onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-grpcntr.ot-acc-txt{background-color:transparent;padding-left:3px}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr,div#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr{overflow:visible;width:100%}div#onetrust-pc-sdk .ot-pc-footer{border-top:0 solid}div#onetrust-pc-sdk .ot-btn-container{padding-top:24px;text-align:center}div#onetrust-pc-sdk .ot-pc-footer button{margin:8px 0;background-color:#fff}div#onetrust-pc-sdk .ot-pc-footer-logo{background-color:#fff}div#onetrust-pc-sdk #ot-lst-title span{font-size:24px;font-weight:400;line-height:32px}div#onetrust-pc-sdk #ot-host-lst .ot-host-desc,div#onetrust-pc-sdk #ot-host-lst .ot-host-expand,div#onetrust-pc-sdk #ot-host-lst .ot-host-name,div#onetrust-pc-sdk #ot-host-lst .ot-host-name a,div#onetrust-pc-sdk .back-btn-handler,div#onetrust-pc-sdk .ot-host-opt li>div div{font-size:16px;font-weight:400;line-height:24px}div#onetrust-pc-sdk #ot-pc-lst{top:0}div#onetrust-pc-sdk .back-btn-handler{text-decoration:none!important}div#onetrust-pc-sdk #filter-btn-handler:hover svg{filter:invert(1)}div#onetrust-pc-sdk .back-btn-handler svg{width:16px;height:16px}div#onetrust-pc-sdk .ot-host-item>button{background:0 0!important;border:0!important;height:66px!important;max-width:none!important;width:calc(100% - 5px)!important;transform:translate(2px,2px)}div#onetrust-pc-sdk .ot-host-item{border-bottom:2px solid #b9b9b9;padding:0}div#onetrust-pc-sdk .ot-host-item .ot-acc-hdr{margin:0 0 -6px;padding:8px 0}div#onetrust-pc-sdk ul li:first-child{border-top:2px solid #b9b9b9}div#onetrust-pc-sdk .ot-host-item .ot-plus-minus{margin:0 8px 0 0}div#onetrust-pc-sdk .ot-search-cntr{width:calc(100% - 48px)}div#onetrust-pc-sdk .ot-host-opt .ot-host-info{background-color:transparent}div#onetrust-pc-sdk .ot-host-opt li>div div{padding:0}div#onetrust-pc-sdk #vendor-search-handler{border-radius:0;border-color:#939393;border-style:solid;border-width:2px 0 2px 2px;font-size:20px;height:48px;margin:0}div#onetrust-pc-sdk .ot-lst-subhdr svg{right:0;top:8px}div#onetrust-pc-sdk .ot-fltr-cntr{box-sizing:border-box;right:0;width:48px}div#onetrust-pc-sdk #filter-btn-handler{width:48px!important;padding:8px!important}div#onetrust-pc-sdk #ot-fltr-modal{max-height:400px}
.ot-sdk-cookie-policy{font-family:inherit;font-size:16px}.ot-sdk-cookie-policy.otRelFont{font-size:1rem}.ot-sdk-cookie-policy h3,.ot-sdk-cookie-policy h4,.ot-sdk-cookie-policy h6,.ot-sdk-cookie-policy p,.ot-sdk-cookie-policy li,.ot-sdk-cookie-policy a,.ot-sdk-cookie-policy th,.ot-sdk-cookie-policy #cookie-policy-description,.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group,.ot-sdk-cookie-policy #cookie-policy-title{color:dimgray}.ot-sdk-cookie-policy #cookie-policy-description{margin-bottom:1em}.ot-sdk-cookie-policy h4{font-size:1.2em}.ot-sdk-cookie-policy h6{font-size:1em;margin-top:2em}.ot-sdk-cookie-policy th{min-width:75px}.ot-sdk-cookie-policy a,.ot-sdk-cookie-policy a:hover{background:#fff}.ot-sdk-cookie-policy thead{background-color:#f6f6f4;font-weight:bold}.ot-sdk-cookie-policy .ot-mobile-border{display:none}.ot-sdk-cookie-policy section{margin-bottom:2em}.ot-sdk-cookie-policy table{border-collapse:inherit}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy{font-family:inherit;font-size:1rem}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h3,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h4,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h6,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy p,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy li,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-title{color:dimgray}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description{margin-bottom:1em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-subgroup{margin-left:1.5em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group-desc,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-table-header,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy span,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td{font-size:.9em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td span,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td a{font-size:inherit}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group{font-size:1em;margin-bottom:.6em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-title{margin-bottom:1.2em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy>section{margin-bottom:1em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th{min-width:75px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a:hover{background:#fff}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy thead{background-color:#f6f6f4;font-weight:bold}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-mobile-border{display:none}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy section{margin-bottom:2em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-subgroup ul li{list-style:disc;margin-left:1.5em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-subgroup ul li h4{display:inline-block}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table{border-collapse:inherit;margin:auto;border:1px solid #d7d7d7;border-radius:5px;border-spacing:initial;width:100%;overflow:hidden}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table th,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table td{border-bottom:1px solid #d7d7d7;border-right:1px solid #d7d7d7}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr:last-child td{border-bottom:0px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr th:last-child,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr td:last-child{border-right:0px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-host,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-cookies-type{width:25%}.ot-sdk-cookie-policy[dir=rtl]{text-align:left}#ot-sdk-cookie-policy h3{font-size:1.5em}@media only screen and (max-width: 530px){.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) table,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) thead,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tbody,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) th,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr{display:block}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) thead tr{position:absolute;top:-9999px;left:-9999px}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr{margin:0 0 1em 0}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr:nth-child(odd),.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr:nth-child(odd) a{background:#f6f6f4}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td{border:none;border-bottom:1px solid #eee;position:relative;padding-left:50%}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td:before{position:absolute;height:100%;left:6px;width:40%;padding-right:10px}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) .ot-mobile-border{display:inline-block;background-color:#e4e4e4;position:absolute;height:100%;top:0;left:45%;width:2px}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td:before{content:attr(data-label);font-weight:bold}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) li{word-break:break-word;word-wrap:break-word}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table{overflow:hidden}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table td{border:none;border-bottom:1px solid #d7d7d7}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy thead,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy tbody,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy tr{display:block}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-host,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-cookies-type{width:auto}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy tr{margin:0 0 1em 0}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td:before{height:100%;width:40%;padding-right:10px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td:before{content:attr(data-label);font-weight:bold}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy li{word-break:break-word;word-wrap:break-word}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy thead tr{position:absolute;top:-9999px;left:-9999px;z-index:-9999}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr:last-child td{border-bottom:1px solid #d7d7d7;border-right:0px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr:last-child td:last-child{border-bottom:0px}}
                
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h5,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h6,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy li,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy p,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy span,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description {
                        color: #696969;
                    }
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th {
                        color: #696969;
                    }
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group {
                        color: #696969;
                    }
                    
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-title {
                            color: #696969;
                        }
                    
            
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table th {
                            background-color: #F8F8F8;
                        }
                    
            .ot-floating-button__front{background-image:url('https://cdn.cookielaw.org/logos/static/ot_persistent_cookie_icon.png')}</style><script src="https://static.addtoany.com/menu/modules/core.pt5ow5lr.js" type="module"></script><style type="text/css">.a2a_hide{display:none}.a2a_logo_color{background-color:#0166ff}.a2a_menu,.a2a_menu *{-moz-box-sizing:content-box;-webkit-box-sizing:content-box;box-sizing:content-box;float:none;margin:0;padding:0;position:static;height:auto;width:auto}.a2a_menu{border-radius:6px;display:none;direction:ltr;background:#FFF;font:16px sans-serif-light,HelveticaNeue-Light,"Helvetica Neue Light","Helvetica Neue",Arial,Helvetica,"Liberation Sans",sans-serif;color:#000;line-height:12px;border:1px solid #CCC;vertical-align:baseline;overflow:hidden}.a2a_mini{min-width:200px;position:absolute;width:300px;z-index:9999997}.a2a_overlay{display:none;background:#616c7deb;backdrop-filter:blur(4px);-webkit-backdrop-filter:blur(4px);position:fixed;top:0;right:0;left:0;bottom:0;z-index:9999998;-webkit-tap-highlight-color:transparent;transition:opacity .14s,backdrop-filter .14s}.a2a_full{background:#FFF;border:1px solid #FFF;box-shadow:#2a2a2a1a 0 0 20px 10px;height:auto;height:calc(320px);top:15%;left:50%;margin-left:-320px;position:fixed;text-align:center;width:640px;z-index:9999999;transition:transform .14s,opacity .14s}.a2a_full_footer,.a2a_full_header,.a2a_full_services{border:0;margin:0;padding:12px;box-sizing:border-box}.a2a_full_header{padding-bottom:8px}.a2a_full_services{height:280px;overflow-y:scroll;padding:0 12px;-webkit-overflow-scrolling:touch}.a2a_full_services .a2a_i{display:inline-block;float:none;width:181px;width:calc(33.334% - 18px)}div.a2a_full_footer{font-size:12px;text-align:center;padding:8px 14px}div.a2a_full_footer a,div.a2a_full_footer a:visited{display:inline;font-size:12px;line-height:14px;padding:8px 14px}div.a2a_full_footer a:focus,div.a2a_full_footer a:hover{background:0 0;border:0;color:#0166FF}div.a2a_full_footer a span.a2a_s_a2a,div.a2a_full_footer a span.a2a_w_a2a{background-size:14px;border-radius:3px;display:inline-block;height:14px;line-height:14px;margin:0 3px 0 0;vertical-align:top;width:14px}.a2a_modal{height:0;left:50%;margin-left:-320px;position:fixed;text-align:center;top:15%;width:640px;z-index:9999999;transition:transform .14s,opacity .14s;-webkit-tap-highlight-color:transparent}.a2a_modal_body{background:0 0;border:0;font:24px sans-serif-light,HelveticaNeue-Light,"Helvetica Neue Light","Helvetica Neue",Arial,Helvetica,"Liberation Sans",sans-serif;position:relative;height:auto;width:auto}.a2a_thanks{color:#fff;height:auto;margin-top:20px;width:auto}.a2a_thanks>div:first-child{margin:0 0 40px 0}.a2a_thanks div *{height:inherit}#a2a_copy_link{background:#FFF;border:1px solid #FFF;cursor:pointer;margin-top:15%}label.a2a_s_link#a2a_copy_link_icon,label.a2a_w_link#a2a_copy_link_icon{background-size:48px;border-radius:0;display:inline-block;height:48px;left:0;line-height:48px;margin:0 3px 0 0;position:absolute;vertical-align:top;width:48px}#a2a_modal input#a2a_copy_link_text{background-color:transparent;border:0;color:#2A2A2A;cursor:pointer;font:inherit;height:48px;left:62px;max-width:initial;min-height:auto;padding:0;position:relative;width:564px;width:calc(100% - 76px)}#a2a_copy_link_copied{background-color:#0166ff;color:#fff;display:none;font:inherit;font-size:16px;margin-top:1px;padding:3px 8px}@media (forced-colors:active){.a2a_color_buttons a,.a2a_svg{forced-color-adjust:none}}@media (prefers-color-scheme:dark){.a2a_menu a,.a2a_menu a.a2a_i,.a2a_menu a.a2a_i:visited,.a2a_menu a.a2a_more,i.a2a_i{border-color:#2a2a2a!important;color:#fff!important}.a2a_menu a.a2a_i:active,.a2a_menu a.a2a_i:focus,.a2a_menu a.a2a_i:hover,.a2a_menu a.a2a_more:active,.a2a_menu a.a2a_more:focus,.a2a_menu a.a2a_more:hover,.a2a_menu_find_container{border-color:#444!important;background-color:#444!important}.a2a_menu:not(.a2a_thanks){background-color:#2a2a2a;border-color:#2a2a2a}.a2a_menu_find{color:#fff!important}.a2a_menu label.a2a_s_find svg{background-color:transparent!important}.a2a_menu label.a2a_s_find svg path{fill:#fff!important}.a2a_full{box-shadow:#00000066 0 0 20px 10px}.a2a_overlay{background-color:#373737eb}}@media print{.a2a_floating_style,.a2a_menu,.a2a_overlay{visibility:hidden}}@keyframes a2aFadeIn{from{opacity:0}to{opacity:1}}.a2a_starting{opacity:0}.a2a_starting.a2a_full,.a2a_starting.a2a_modal{transform:scale(.8)}@media (max-width:639px){.a2a_full{border-radius:0;top:15%;left:0;margin-left:auto;width:100%}.a2a_modal{left:0;margin-left:10px;width:calc(100% - 20px)}}@media (min-width:318px) and (max-width:437px){.a2a_full .a2a_full_services .a2a_i{width:calc(50% - 18px)}}@media (max-width:317px){.a2a_full .a2a_full_services .a2a_i{width:calc(100% - 18px)}}@media (max-height:436px){.a2a_full{bottom:40px;height:auto;top:40px}}@media (max-height:550px){.a2a_modal{top:30px}}@media (max-height:360px){.a2a_modal{top:20px}.a2a_thanks>div:first-child{margin-bottom:20px}}.a2a_menu a{color:#0166FF;text-decoration:none;font:16px sans-serif-light,HelveticaNeue-Light,"Helvetica Neue Light","Helvetica Neue",Arial,Helvetica,"Liberation Sans",sans-serif;line-height:14px;height:auto;width:auto;outline:0}.a2a_menu a.a2a_i:visited,.a2a_menu a.a2a_more{color:#0166FF}.a2a_menu a.a2a_i:active,.a2a_menu a.a2a_i:focus,.a2a_menu a.a2a_i:hover,.a2a_menu a.a2a_more:active,.a2a_menu a.a2a_more:focus,.a2a_menu a.a2a_more:hover{color:#2A2A2A;border-color:#EEE;border-style:solid;background-color:#EEE;text-decoration:none}.a2a_menu label.a2a_s_find{background-size:24px;height:24px;left:8px;pointer-events:auto;position:absolute;top:7px;width:24px}.a2a_menu label.a2a_s_find svg{background-color:#FFF}.a2a_menu label.a2a_s_find svg path{fill:#CCC}#a2a_menu_container{display:inline-block}.a2a_menu_find_container{border:1px solid #CCC;border-radius:6px;padding:2px 24px 2px 0;position:relative;text-align:left}.a2a_cols_container .a2a_col1{overflow-x:hidden;overflow-y:auto;-webkit-overflow-scrolling:touch}#a2a_modal input,#a2a_modal input[type=text],.a2a_menu input,.a2a_menu input[type=text]{display:block;background-image:none;box-shadow:none;line-height:100%;margin:0;outline:0;overflow:hidden;padding:0;-moz-box-shadow:none;-webkit-box-shadow:none;-webkit-appearance:none}#a2afeed_find_container input,#a2afeed_find_container input[type=text],#a2apage_find_container input,#a2apage_find_container input[type=text]{background-color:transparent;border:0;box-sizing:content-box;color:#2A2A2A;float:none;font:inherit;font-size:16px;height:28px;line-height:20px;left:38px;outline:0;margin:0;max-width:initial;min-height:initial;padding:2px 0;position:relative;width:99%}.a2a_clear{clear:both}.a2a_svg{background-repeat:no-repeat;display:block;overflow:hidden;height:32px;line-height:32px;padding:0;pointer-events:none;width:32px}.a2a_svg svg{background-repeat:no-repeat;background-position:50% 50%;border:none;display:block;left:0;margin:0 auto;overflow:hidden;padding:0;position:relative;top:0;width:auto;height:auto}a.a2a_i,i.a2a_i{display:block;float:left;border:1px solid #FFF;line-height:24px;padding:6px 8px;text-align:left;white-space:nowrap;overflow:hidden;text-overflow:ellipsis;width:132px}a.a2a_i span,a.a2a_more span{display:inline-block;overflow:hidden;vertical-align:top}a.a2a_i .a2a_svg{margin:0 6px 0 0}a.a2a_i .a2a_svg,a.a2a_more .a2a_svg{background-size:24px;height:24px;line-height:24px;width:24px}a.a2a_sss:hover{border-left:1px solid #CCC}a.a2a_more{border-bottom:1px solid #FFF;border-left:0;border-right:0;line-height:24px;margin:6px 0 0;padding:6px;-webkit-touch-callout:none}a.a2a_more span{height:24px;margin:0 6px 0 0}.a2a_kit .a2a_svg{background-repeat:repeat}.a2a_default_style a:empty,.a2a_flex_style a:empty,.a2a_floating_style a:empty,.a2a_overlay_style a:empty{display:none}.a2a_color_buttons a,.a2a_floating_style a{text-decoration:none}.a2a_default_style:not(.a2a_flex_style) a{float:left;line-height:16px;padding:0 2px}.a2a_default_style a:hover .a2a_svg,.a2a_floating_style a:hover .a2a_svg,.a2a_overlay_style a:hover .a2a_svg svg{opacity:.7}.a2a_overlay_style.a2a_default_style a:hover .a2a_svg{opacity:1}.a2a_default_style .a2a_count,.a2a_default_style .a2a_svg,.a2a_floating_style .a2a_svg,.a2a_menu .a2a_svg,.a2a_vertical_style .a2a_count,.a2a_vertical_style .a2a_svg{border-radius:4px}.a2a_default_style .a2a_counter img,.a2a_default_style .a2a_dd,.a2a_default_style .a2a_svg{float:left}.a2a_default_style .a2a_img_text{margin-right:4px}.a2a_default_style .a2a_divider{border-left:1px solid #000;display:inline;float:left;height:16px;line-height:16px;margin:0 5px}.a2a_kit a{cursor:pointer;transition:none}.a2a_floating_style{background-color:#fff;border-radius:6px;position:fixed;z-index:9999995}.a2a_overlay_style{z-index:2147483647}.a2a_floating_style,.a2a_overlay_style{animation:a2aFadeIn .2s ease-in;padding:4px}.a2a_vertical_style:not(.a2a_flex_style) a{clear:left;display:block;overflow:hidden;padding:4px}.a2a_floating_style.a2a_default_style{bottom:0}.a2a_floating_style.a2a_default_style a,.a2a_overlay_style.a2a_default_style a{padding:4px}.a2a_count{background-color:#fff;border:1px solid #ccc;box-sizing:border-box;color:#2a2a2a;display:block;float:left;font:12px Arial,Helvetica,sans-serif;height:16px;margin-left:4px;position:relative;text-align:center;width:50px}.a2a_count:after,.a2a_count:before{border:solid transparent;border-width:4px 4px 4px 0;content:"";height:0;left:0;line-height:0;margin:-4px 0 0 -4px;position:absolute;top:50%;width:0}.a2a_count:before{border-right-color:#ccc}.a2a_count:after{border-right-color:#fff;margin-left:-3px}.a2a_count span{animation:a2aFadeIn .14s ease-in}.a2a_vertical_style .a2a_counter img{display:block}.a2a_vertical_style .a2a_count{float:none;margin-left:0;margin-top:6px}.a2a_vertical_style .a2a_count:after,.a2a_vertical_style .a2a_count:before{border:solid transparent;border-width:0 4px 4px 4px;content:"";height:0;left:50%;line-height:0;margin:-4px 0 0 -4px;position:absolute;top:0;width:0}.a2a_vertical_style .a2a_count:before{border-bottom-color:#ccc}.a2a_vertical_style .a2a_count:after{border-bottom-color:#fff;margin-top:-3px}.a2a_color_buttons .a2a_count,.a2a_color_buttons .a2a_count:after,.a2a_color_buttons .a2a_count:before,.a2a_color_buttons.a2a_vertical_style .a2a_count:after,.a2a_color_buttons.a2a_vertical_style .a2a_count:before{background-color:transparent;border:none;color:#fff;float:none;width:auto}.a2a_color_buttons.a2a_vertical_style .a2a_count{margin-top:0}.a2a_flex_style{display:flex;align-items:flex-start;gap:0}.a2a_default_style.a2a_flex_style{left:0;right:0;width:100%}.a2a_vertical_style.a2a_flex_style{flex-direction:column;top:0;bottom:0}.a2a_flex_style a{display:flex;justify-content:center;flex:1;padding:4px}.a2a_flex_style.a2a_vertical_style a{flex-direction:column}.a2a_floating_style.a2a_color_buttons,.a2a_floating_style.a2a_flex_style{border-radius:0;padding:0}.a2a_floating_style.a2a_default_style.a2a_flex_style{bottom:0}.a2a_kit.a2a_flex_style .a2a_counter img,.a2a_kit.a2a_flex_style .a2a_dd,.a2a_kit.a2a_flex_style .a2a_svg{float:none}.a2a_nowrap{white-space:nowrap}.a2a_note{margin:0 auto;padding:9px;font-size:12px;text-align:center}.a2a_note .a2a_note_note{margin:0;color:#2A2A2A}.a2a_wide a{display:block;margin-top:3px;border-top:1px solid #EEE;text-align:center}.a2a_label{position:absolute!important;clip-path:polygon(0px 0px,0px 0px,0px 0px);-webkit-clip-path:polygon(0px 0px,0px 0px,0px 0px);overflow:hidden;height:1px;width:1px}.a2a_kit,.a2a_menu,.a2a_modal,.a2a_overlay{-ms-touch-action:manipulation;touch-action:manipulation;outline:0}.a2a_dd{-webkit-user-drag:none}.a2a_dd img{border:0}.a2a_button_facebook_like iframe{max-width:none}</style><style></style><link href="https://crossmark-cdn.crossref.org/widget/v2.0/style.css" type="text/css" rel="stylesheet"></head>
<body class="pb-ui">
    <div class="skip">
        <a href="#page-body-id" id="skipNavigationLink" title="Skip to Main Content" tabindex="0" class="skipToContent">Skip to Main Content</a>
        <a href="#mega-menu-nav" id="skipToMainMenu" title="Skip to Main Menu" tabindex="0" class="skipToContent">Skip to Main Menu</a>
    </div>



    <div id="pb-page-content" data-ng-non-bindable="">
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="ux3-layout-widget" data-widget-id="832ad3f3-97ac-45ab-9af1-90c6f7bff54a" class="ux3">
        
    

        


    

            

            
                <div class="cell fonts-loaded">
            

            <div data-widget-id="832ad3f3-97ac-45ab-9af1-90c6f7bff54a" data-widget-name="ux3-layout-widget"><div>
    

        


    

            

            

            <div id="a1f1ba25-79af-4194-aaa9-66206104a800" class="widget loginPopup none widget-none widget-compact-all"><div class="wrapped"><div class="widget-body body body-none body-compact-all"><div role="dialog" aria-modal="true" aria-label="Login" class="popup login-popup hidden"><div class="content"><a href="#" aria-label="Close login popup" title="Close login popup" role="button" class="close"><i class="icon-close_thin"></i></a><h2 class="login-title">Login to your account</h2><div class="login-form"><p tabindex="-1" class="message error"></p><form action="/action/doLogin?redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2820%2930682-6" method="post"><input type="hidden" name="redirectUri" value="https://www.cell.com/cell/fulltext/S0092-8674(20)30682-6"><input type="hidden" name="referer" value="/cell/fulltext/S0092-8674(20)30682-6"><input type="hidden" name="ecommRedirect"><input type="hidden" name="code" value="cell-site"><input type="hidden" name="UX3LoginWidget" value="true"><input type="hidden" name="isPopup" value="true"><div class="input-group login-input--wrapper"><div class="label"><label for="login">Email/Username</label></div><input id="login" type="text" name="login" value="" size="15" class="login"><div role="alert" class="error-hint js--hidden"><div class="error-hint__text"><div class="error-text">Your email address is a required field.</div><div class="email-example">E.g., j.smith@mail.com</div></div></div></div><div class="input-group login-input--wrapper"><div class="label"><label for="password">Password</label></div><input id="password" type="password" name="password" value="" autocomplete="off" class="password"><a href="#" class="login__field__action js-show-password">Show</a><div role="alert" class="error-hint js--hidden"><span class="error-hint__text">Your password is a required field.</span></div><div class="actions"><a href="https://www.cell.com/action/requestResetPassword" class="link show-request-reset-password">Forgot password?</a></div></div><div class="remember"><input type="checkbox" name="savePassword" id="a1f1ba25-79af-4194-aaa9-66206104a800-remember" value="1" checked="checked"><label for="a1f1ba25-79af-4194-aaa9-66206104a800-remember">Remember me</label></div><div class="submit"><input type="submit" name="submit" value="Log in" class="button submit primary"></div></form></div><div class="create-account"><div>Don’t have an account?</div><a href="/action/registration?redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2820%2930682-6" class="show-registration">Create a Free Account</a></div></div></div></div></div></div>

            

        

    

        


    

            

            

            <div role="dialog" aria-modal="true" aria-label="Reset password" class="popup top-drawer request-reset-password-drawer hidden"><div class="content"><a href="#" aria-label="Close reset password popup" title="Close reset password popup" role="button" class="close"><i class="icon-close_thin"></i></a><div class="request-reset"><div class="form"><p class="sub">If you don't remember your password, you can reset it by entering your email address and clicking the Reset Password button. You will then receive an email that contains a secure link for resetting your password</p><div tabindex="-1" class="message error"></div><form action="/action/requestResetPassword" method="post" class="request-reset-password-form"><input type="hidden" name="platform"><input type="hidden" name="requestResetPassword" value="true"><input type="hidden" name="codeSite" value="cell-site"><div class="input-group"><div class="label"><label for="reset-password-email">Email*</label></div><input id="reset-password-email" type="text" name="email" value="" size="15" class="email"></div><div class="submit" disabled="disabled"><input type="submit" name="submit" value="Submit" disabled="disabled" class="button primary submit"></div></form></div><div class="success-template hidden"><p>If the address matches a valid account an email will be sent to __email__ with instructions for resetting your password</p></div></div><div class="request-reset-cancel"><a href="#" aria-label="Close reset password popup" title="Close reset password popup" class="cancel">Cancel</a></div></div></div>

            

        

    

        


    

            

            
                <div class="ux3-ads">
            

            <div class="pb-ad">
            <!-- Do Not Modify without contacting ELS-HSOnlineCampaign@elsevier.com -->
<!-- To be kept in sync with Global Commercial Publishing Operations-->
<!-- Start: GPT Async -->
<div id="gpt-Variables" data-adfile="CELLJOURNAL">
	<input type="hidden" id="gptSite" value="cell">
	<input type="hidden" id="gptLoggedIn" value="no">
	<input type="hidden" id="gptAccessType" value="ae:ANON_GUEST|ae:ANON_GUEST">
	<input type="hidden" id="gptDefaultSite" value="cell">
</div>
<div id="gpt-Interscroller" class="gpt-InScrollBox" style="display:none;">
	<div class="gpt-InScrollLabel">ADVERTISEMENT</div>
	<div class="gpt-InScrollContainer">
		<div class="gpt-InScrollWrapper">
			<div class="gpt-InScrollInner">
				<div class="gpt-adinterscroller" data-adsize="interscroller"></div>
			</div>
		</div>
	</div>
	<div class="gpt-InScrollLabel">SCROLL TO CONTINUE WITH CONTENT</div>
</div>
<div id="gpt-DebugDiv" class="gpt-DebugDiv" style="display:none;">
	<div style="width: 100%; display: table;">
		<div style="display: table-row">
			<div style="width: 200px; display: table-cell;">
				<button class="gpt-DebugBtn" onclick="gpt_ShowConsole()">Open GPT Console</button><br>
				<button id="gpt_GrapeShotDebugBtn" class="gpt-DebugBtn" onclick="gpt_GrapeShotDebug();">Open Oracle Keywords</button><br>
				<br>
				<button class="gpt-DebugBtn" onclick="gpt_LoadDebugValues()">Refresh Values</button>
			</div>
			<div style="display: table-cell;">
				<table class="gpt-DebugTable">
					<thead>
						<tr>
							<th>Property</th>
							<th>Value</th>
						</tr>
					</thead>
					<tbody>
						<tr><td class="gpt-DebugTitle">Status</td><td><div class="gptStatus_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Version</td><td><div class="gptVersion_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Ad File</td><td><div class="gptAdFile_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Disable Ads Flag</td><td><div class="gptDisableAds_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Environment</td><td><div class="gptTestENV_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Moat Init</td><td><div class="gptMoatInit_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Moat Ready</td><td><div class="gptMoatReady_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual Ready</td><td><div class="gptContextualReady_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual URL</td><td><div class="gptContextualURL_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual Initial Segments</td><td><div class="gptContextualInit_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual Used Segments</td><td><div class="gptContextualResult_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">AdUnit</td><td><div class="gptSite_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">SubAdUnit</td><td><div class="gptPage_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Custom Targeting</td><td><div class="gptCustomTargeting_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Ad Events</td><td><div class="gptEvent_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Invalid Ad Sizes</td><td><div class="gptInvalidAdSize_Debug"></div></td></tr>
					</tbody>
				</table>
			</div>
		</div>
	</div>
</div>
<style type="text/css">
	.gpt-DebugBtn {
		display: inline-block;
		text-align: center;
		text-decoration: none;
		margin: 2px 0;
		border: solid 1px #122b40;
		border-radius: 0.4em;
		padding: 0.5em 1em;
		color: #ffffff;
		background-color: #204d74;
		font-weight: normal;
		font-size: 12px;
	}

	.gpt-DebugDiv {
		margin: 5px;
	}

	.gpt-DebugTitle {
		font-weight: bold;
	}

	.gpt-DebugTable {
		border: 1px solid #dddddd;
		border-collapse: collapse;
		margin: 5px;
		font-size: 0.9em;
		font-family: sans-serif;
		min-width: 400px;
		margin-bottom: 25px;
	}

		.gpt-DebugTable td:first-child {
			padding: 5px;
		}

		.gpt-DebugTable thead tr th {
			background-color: #d9edf7;
			color: #ffffff;
			text-align: left;
			padding: 5px;
			border-bottom: 2px solid #204d74;
		}

		.gpt-DebugTable th,
		.gpt-DebugTable td {
			padding: 5px;
		}

		.gpt-DebugTable tbody tr {
			border-bottom: 1px solid #dddddd;
		}

			.gpt-DebugTable tbody tr:nth-of-type(even) {
				background-color: #f3f3f3;
			}

			.gpt-DebugTable tbody tr:last-of-type {
				border-bottom: 2px solid #204d74;
			}

	.gpt-InScrollLabel {
		background-color: rgb(0, 0, 0);
		color: rgb(255, 255, 255);
		position: relative;
		max-width: 100vw;
		left: 50%;
		right: 50%;
		margin-left: -50vw;
		margin-right: -50vw;
		padding: 0;
		text-align: center;
		width: 100vw !important;
		font-size: 10px;
	}

	.gpt-InScrollBox {
		height: 100vh !important;
		width: 100vw;
		cursor: pointer;
		max-width: 100vw;
		position: relative;
		left: 50%;
		right: 50%;
		margin-left: -50vw;
		margin-right: -50vw;
		margin-bottom: 50px;
		padding-top: 5px;
		padding-bottom: 5px;
		z-index: 10001;
		background-color: white;
	}

		.gpt-InScrollBox .gpt-InScrollContainer {
			height: 95vh !important;
			position: relative;
		}

		.gpt-InScrollBox .gpt-InScrollWrapper {
			position: absolute !important;
			top: 0 !important;
			left: 0;
			width: 100%;
			height: 95vh !important;
			border: 0 !important;
			margin: 0 !important;
			padding: 0 !important;
			clip: rect(0, auto, auto, 0) !important;
			-webkit-clip-path: polygon(0px 0px, 100% 0px, 100% 100%, 0px 100%) !important;
			clip-path: polygon(0px 0px, 100% 0px, 100% 100%, 0px 100%) !important;
		}

		.gpt-InScrollBox .gpt-InScrollInner {
			position: fixed !important;
			top: 0 !important;
			left: 0 !important;
			bottom: 0;
			margin: auto;
			width: 100vw;
			height: 95vh;
			-webkit-transform: translateZ(0) !important;
			display: -webkit-box;
			display: -ms-flexbox;
			display: flex;
			-webkit-box-orient: vertical;
			-webkit-box-direction: normal;
			-ms-flex-direction: column;
			flex-direction: column;
			-webkit-box-align: center;
			-ms-flex-align: center;
			align-items: center;
			-webkit-box-pack: center;
			-ms-flex-pack: center;
			justify-content: center;
			text-align: center;
		}
</style>
<script class="optanon-category-4" type="text/plain">
	//--------------------------------------------------------------------------
	var gpt_Version = '04072023';
	var gpt_AdFile = $('#gpt-Variables').data('adfile');
	var gpt_MaxMOATInitAttempts = 11;
	var gpt_MaxMOATReadyAttempts = 11;
	var gpt_MaxContextualReadyAttempts = 21;
	var gpt_MOATInitAttempts = 0;
	var gpt_MOATReadyAttempts = 0;
	var gpt_ContextualReadyAttempts = 0;
	var gpt_IsProd = true;
	var gpt_StartTime;
	var gs_channels = 'DEFAULT';
	var gpt_ContextualReady = false;
	var gpt_ContextualURL = 'Not enabled outside of Production';
	var gpt_DisableAdsFlagFound = false;

	var gptEvent_Debug = '';
	var gptCustomTargeting_Debug = '';
	var gptSite_Debug = "";
	var gptPage_Debug = "";
	var gptContextualEnabled = false;
	var gptContextualResult_Debug = '';
	var gptContextualInit_Debug = '';
	var gptContextualURL_Debug = '';
	var gptContextualReady_Debug = '';
	var gptMoatReady_Debug = '';
	var gptMoatInit_Debug = '';
	var gptTestENV_Debug = '';
	var gptDisableAds_Debug = '';
	var gptStatus_Debug = 'GPT Still Loading - Refresh Values';
	var gptInvalidAdSize_Debug = '';
	//--------------------------------------------------------------------------
	$(document).ready(function () {
		//Check to see if we are on a development env
		gpt_CheckEnvironment();

		if (!gpt_DisableAdsFlagFound) {
			gpt_StartTime = new Date();
			if (gpt_AdFile !== "PU") {
				//Once the dom has loaded, begin the process to init the ads.
				gpt_InitScripts();
				gpt_CheckForMOATInit();
			}
		}
	});
</script>
<script type="text/javascript">
	function gpt_CheckEnvironment() {
		if ($('#gptDisableAds').length > 0) {
			gptDisableAds_Debug = 'Flag Detected - Ads Disabled';
			gpt_DisableAdsFlagFound = true;
		}
		else {
			gptDisableAds_Debug = 'Ads Enabled';
			gpt_DisableAdsFlagFound = false;
		}
		switch (gpt_AdFile) {
			case 'PU':
				gptContextualEnabled = true;
				if ($("#Environment").val() !== "Prod") {
					gpt_IsProd = false;
					gptTestENV_Debug = 'Test Environment Detected';
				}
				else {
					gptTestENV_Debug = 'Production Environment Detected';
				}
				break;
			case 'JBS':
			case 'LANCET':
				gptContextualEnabled = true;
				if ($(".gpt-ad").length > 0) {
					//Other ads exist, allow the Prestitial
					$(document.body).append("<div class='gpt-ad' data-adsize='prestitial'></div>");
				}
				if (window.location.hostname.toLowerCase().indexOf('.literatumonline.') >= 0) {
					gpt_IsProd = false;
					gptTestENV_Debug = 'Test Environment Detected';
				}
				else {
					gptTestENV_Debug = 'Production Environment Detected';
				}
				break;
			default:
				//Cell
				gptContextualEnabled = false;
				if (window.location.hostname.toLowerCase().indexOf('.literatumonline.') >= 0) {
					gpt_IsProd = false;
					gptTestENV_Debug = 'Test Environment Detected';
				}
				else {
					gptTestENV_Debug = 'Production Environment Detected';
				}
				break;
		}
	}
	//--------------------------------------------------------------------------
    function gpt_GetNonCMEArticle() {
        var url;
        $('.articleCitation').each(function (i, obj) {
            if ($(obj).find('.toc__item__detials').length > 0) {
                var itemDetails = $(obj).first().find('.toc__item__detials');
                var cmeFound = 0;
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='CME']").length; }
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='(CME)']").length; }
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='Education']").length; }
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='education']").length; }
                if (cmeFound == 0) {
                    var itemLink = $(itemDetails).first().find('.toc__item__title');
                    var linkCount = $(itemLink).first().find("a:contains('CME')").length;
                    if (linkCount == 0) {
                        url = $(itemLink).first().find('a').attr('href');
                        return false;   //Found a URL, break the loop
                    }
                }
            }
        });
        return url;
    }
	//--------------------------------------------------------------------------
	function gpt_GetElapsedTime() {
		var endTime = new Date();
		var timeDiff = endTime - gpt_StartTime; //in ms
		// strip the ms
		timeDiff /= 1000;

		// get seconds
		return seconds = Math.round(timeDiff) + " seconds";
	}
	//--------------------------------------------------------------------------
	function gpt_CheckSSC() {
		//Check for ScreenShot Client
		var gpt_SSC = gpt_GetQuerystring('gpt_ssc', '');
		if (gpt_SSC == '1') {
			//it's the ScreenShotClient, disable MOAT IVT Detection
			$('#gpt-Variables').append('<input type="hidden" id="gptm_data">');
			$('#gptm_data').val(0);
			gpt_Target("m_data");
		}
	}
	//--------------------------------------------------------------------------
	function gpt_LocalSS(adTypeId) {
		window.scrollTo(0, 0);
		gpt_Sleep(1000);
		gpt_ScrollToAd(adTypeId);
		gpt_PreTearsheet(adTypeId);
	}
	//--------------------------------------------------------------------------
	function gpt_InitScripts() {
		if (gpt_AdFile == "PU") {
			if (gpt_IsProd) {
				gpt_ContextualURL = gpt_CheckForNULL($('#canonicalurl').val());
				if (gpt_ContextualURL === null) {
					gpt_ContextualURL = window.location.href.split('?')[0].split('#')[0];
				}
				else {
					gpt_ContextualURL = window.location.origin + gpt_ContextualURL;
				}
				var useSSL = 'https:' == document.location.protocol;
				var gs = document.createElement('script');
				gs.async = true;
				gs.type = 'text/javascript';
				gs.src = (useSSL ? 'https:' : 'http:') + '//elsevier.gscontxt.net/multizone/channels.cgi?url=' + encodeURIComponent(gpt_ContextualURL);
				gs.onload = function () {
					gpt_ContextualReady = true;
				};
				document.head.appendChild(gs);
			}
		}
		else {
			//JBS-Cell-Lancet
			var useSSL = 'https:' == document.location.protocol;

			var gads = document.createElement('script');
			gads.async = true;
			gads.type = 'text/javascript';
			gads.src = (useSSL ? 'https:' : 'http:') + '//securepubads.g.doubleclick.net/tag/js/gpt.js';
			document.head.appendChild(gads);

			if (gpt_IsProd && gptContextualEnabled) {
				gpt_ContextualURL = window.location.href.split('?')[0].split('#')[0];
				var gs = document.createElement('script');
				gs.async = true;
				gs.type = 'text/javascript';
				gs.src = (useSSL ? 'https:' : 'http:') + '//elsevier.gscontxt.net/multizone/channels.cgi?url=' + encodeURIComponent(gpt_ContextualURL);
				gs.onload = function () {
					gpt_ContextualReady = true;
				};
				document.head.appendChild(gs);
			}

			var elsevierMoatHeader = document.createElement('link');
			elsevierMoatHeader.rel = 'preload';
			elsevierMoatHeader.as = 'script';
			elsevierMoatHeader.src = (useSSL ? 'https:' : 'http:') + '//z.moatads.com/elsevierheader150204004183/moatheader.js';
			document.head.appendChild(elsevierMoatHeader);

			var elsevierMoatAds = document.createElement('link');
			elsevierMoatAds.rel = 'preconnect';
			elsevierMoatAds.src = (useSSL ? 'https:' : 'http:') + '//mb.moatads.com';
			document.head.appendChild(elsevierMoatAds);

			var moatHeader = document.createElement('script');
			moatHeader.async = true; gads.type = 'text/javascript';
			moatHeader.src = (useSSL ? 'https:' : 'http:') + '//z.moatads.com/elsevierheader150204004183/moatheader.js';
			document.head.appendChild(moatHeader);
		}
	}
	//--------------------------------------------------------------------------
	async function gpt_CheckForMOATInit() {
		//Check every 1/10 of a second until MOAT is Loaded or 1 second timeout
		if (gpt_MOATInitAttempts < gpt_MaxMOATInitAttempts) {
			gpt_MOATInitAttempts += 1;
			if (typeof window.moatPrebidApi === "object") {
				//MOAT LOADED
				gptMoatInit_Debug = '0.0' + gpt_MOATInitAttempts + ' seconds';
				gpt_CheckForMOATReady();
			}
			else {
				setTimeout(() => { gpt_CheckForMOATInit(); }, 100);
			}
		}
		else {
			//TIMEOUT: MOAT Not LOADED
			gptMoatInit_Debug = 'Timeout Expired';
			var googletag = googletag || {};
			googletag.cmd = googletag.cmd || [];
			gpt_CheckForContextualReady();
		}
	}
	//--------------------------------------------------------------------------
	function gpt_CheckForMOATReady() {
		//Check every 1/10 of a second until MOAT is Ready or 1 second timeout
		if (gpt_MOATReadyAttempts < gpt_MaxMOATReadyAttempts) {
			gpt_MOATReadyAttempts += 1;
			if (window.moatPrebidApi.safetyDataAvailable()) {
				//MOAT Ready
				gptMoatReady_Debug = '0.0' + gpt_MOATReadyAttempts + " seconds";
				gpt_CheckForContextualReady();
			}
			else {
				setTimeout(() => { gpt_CheckForMOATReady(); }, 100);
			}
		}
		else {
			//TIMEOUT: MOAT Not Ready
			gptMoatReady_Debug = 'Timeout Expired';
			gpt_CheckForContextualReady();
		}
	}
	//--------------------------------------------------------------------------
	function gpt_CheckForContextualReady() {
		//Check every 1/10 of a second until Contextual is Ready or 2 second timeout
		if (gpt_IsProd && gptContextualEnabled) {
			//Enabled
			if (gpt_ContextualReadyAttempts < gpt_MaxContextualReadyAttempts) {
				gpt_ContextualReadyAttempts += 1;
				if (gpt_ContextualReady) {
					//Contextual Ready
					gptContextualURL_Debug = gpt_ContextualURL;
					gptContextualReady_Debug = '0.0' + gpt_ContextualReadyAttempts + " seconds";
					gptContextualInit_Debug = gs_channels;
					gpt_InitAds();
				}
				else {
					setTimeout(() => { gpt_CheckForContextualReady(); }, 100);
				}
			}
			else {
				//LOG TIMEOUT error
				//REMOVE FOR NOW (SHOULD LOG TO A DIFF LOCATION)
				//TIMEOUT: Contextual Not Ready
				$('#gpt_GrapeShotDebugBtn').remove();
				gptContextualURL_Debug = gpt_ContextualURL;
				gptContextualReady_Debug = 'Timeout Expired';
				gptContextualInit_Debug = "";
				gpt_InitAds();
			}
		}
		else {
			//Not Enabled
			$('#gpt_GrapeShotDebugBtn').remove();
			switch (gpt_AdFile) {
				case 'PU':
				case 'LANCET':
				case 'JBS':
					gptContextualURL_Debug = 'Not enabled outside of Production';
					gptContextualReady_Debug = 'Not enabled outside of Production';
					gptContextualInit_Debug = 'Not enabled outside of Production';
					break;
				case "CELLJOURNAL":
				case "CELLBUCKET":
					gptContextualReady_Debug = 'Not enabled on Cell';
					gptContextualInit_Debug = 'Not enabled on Cell';
					gptContextualURL_Debug = 'Not enabled on Cell';
					break;
			}
			gpt_InitAds();
		}
	}
	//--------------------------------------------------------------------------
	function GPT_GetPageByURL() {
		var host = window.location.host.toLocaleLowerCase()
		var localpath = window.location.pathname.toLocaleLowerCase();

		///Check for Cell.com
		if (host.indexOf('cell.com') >= 0) {
			return null;
		}

		///Check for Cell.com (marlin-stag)
		if (host.indexOf('www-cell-com') >= 0) {
			return null;
		}

		///Check for Lancet
		if (host.indexOf('thelancet') >= 0) {
			return null;
		}

		if (localpath.indexOf('/article/') >= 0) {
			return 'article.fulltext';
		}

		if (localpath.indexOf('/action/dosearch') >= 0) {
			return 'search.searchResults';
		}

		if (localpath.indexOf('/content/') >= 0) {
			return 'periodical.editorial';
		}

		return 'periodical.home';
	}
	//--------------------------------------------------------------------------
	function gpt_CheckForNULL(x, valueifnull) {
		if (typeof valueifnull == "undefined") {
			valueifnull = null;
		}

		if (typeof x == 'undefined') {
			return valueifnull;
		}
		if (x == null) {
			return valueifnull;
		}
		if (x == "null") {
			return valueifnull;
		}
		if ($.trim(x) == '') {
			return valueifnull;
		}

		return x;
	}
	//--------------------------------------------------------------------------
	function gpt_GetAdditionalTargeting() {
		//adds page level targeting of custom variables
		//the id (minus gpt) and the value will be used as a key-value pair
		//eg: <input type="hidden" id="gptissuepii" value="x" class="gpt-Target">
		//eg: target issuepii=x
		$(".gpt-Target").each(function (index) {
			var gptTargetId = $(this).attr('id');
			gptTargetId = gptTargetId.replace('gpt', '').toLowerCase();
			if (gptTargetId !== '') {
				gpt_Target(gptTargetId);
			}
		})
	}
	//--------------------------------------------------------------------------
	function gpt_GetQuerystring(key, default_) {
		//function to read querystring variables
		if (default_ === null) default_ = '';
		key = key.replace(/[\[]/, "\\\[").replace(/[\]]/, "\\\]");
		var regex = new RegExp("[\\?&]" + key + "=([^&#]*)");
		var qs = regex.exec(window.location.href);
		if (qs === null)
			return default_;
		else
			return qs[1];
	}
	//--------------------------------------------------------------------------
	function gpt_GetDFPTestId() {
		//Get any request for a test id from querystring and set custom targeting parameter
		var gptDFPTestId = gpt_GetQuerystring('dfptestid', '');
		if (gptDFPTestId !== '') {
			$('#gpt-Variables').append('<input type="hidden" id="gptDFPTestId">');
			$('#gptDFPTestId').val(gptDFPTestId);
			gpt_Target("DFPTestId");
		}
	}
	//--------------------------------------------------------------------------
	function gpt_Target(targetName) {
		//Set page level target if value is found
		if ($('#gpt' + targetName).length > 0) {
			var targetValue = $('#gpt' + targetName).val();

			if (targetValue !== '') {
				targetValue = targetValue.replace(/[^-a-z0-9]/ig, '');	//Only Allow a-z, 0-9
				googletag.pubads().setTargeting(targetName.toLowerCase(), targetValue.toLowerCase());
				gptCustomTargeting_Debug = gptCustomTargeting_Debug + targetName.toLowerCase() + ' = ' + targetValue.toLowerCase() + '<br>';
			}
		}
	}
	//--------------------------------------------------------------------------
	function gpt_TargetAccessType() {
		var currentAccessType = $('#gptAccessType').val();
		if (typeof currentAccessType == "undefined") {
			currentAccessType = "";
		}
		if (currentAccessType.indexOf('ANON_IP') >= 0) {
			googletag.pubads().setTargeting("loggedin", "yes");
		}
		else {
			gpt_Target("LoggedIn");
		}
	}
	//--------------------------------------------------------------------------
	async function gpt_PreTearsheet(adTypeId) {
		if (typeof adTypeId === "undefined") {
			adTypeId = 0;
		}
		//Remove Elsevier Cookie Policy Notice
		if ($("#onetrust-consent-sdk").length > 0) {
			$("#onetrust-consent-sdk").remove();
		}

        //Mobile Nav
        $('#skip').remove();
        $('skipNavigationLink').remove();

		//JBS Modals
		if ($('.leadinModal').length > 0) {
			$('.leadinModal').remove();
		}
		if ($('.leadinModal').length > 0) {
			$('.leadinModal').remove();
		}
		if ($('.usabilla__overlay').length > 0) {
			$('.usabilla__overlay').remove();
		}

		//Pendo
		if ($('._pendo-step-container').length > 0) {
			$('._pendo-step-container').remove();
		}
		if ($('._pendo-guide-tt_').length > 0) {
			$('._pendo-guide-tt_').remove();
		}
		if ($('._pendo-image').length > 0) {
			$('._pendo-image').remove();
		}
		if ($('._pendo-badge-image').length > 0) {
			$('._pendo-badge-image').remove();
		}

		switch (gpt_AdFile) {
			case "CELLJOURNAL":
			case "CELLBUCKET":
				//Mobile Bug Fix on Cell
				if (parseInt(adTypeId) == 5) {
					$('.widget-horizontalAd').css('position', 'static');
					$('.body-horizontalAd').css('display', 'block');
				}
				break;
			case "JBS":
			case "LANCET":
				objLeaderboard = ".container-leaderboard";
				offset = 53;
				switch (parseInt(adTypeId)) {
					case 3: //MPU
					case 4: //SkyScraper
						//BUGFIX: Show Sidebar ad for JBS
						//skyscraper & mpu
						if ($('.widget-verticalAd').length > 0) {
							$('.widget-verticalAd').css('display', 'block');
						}
						$('#backgroundLayer').css('display', 'none');   //Hide any Prestitial
						break;
					case 6:     //Prestitial
						$('#backgroundLayer').css('display', 'block');
						break;
					default:
						$('#backgroundLayer').css('display', 'none');   //Hide any Prestitial
						break;
				}
				break;
		}
	}
	//--------------------------------------------------------------------------
	function gpt_Sleep(ms) {
		return new Promise(resolve => setTimeout(resolve, ms));
	}
	//--------------------------------------------------------------------------
	async function gpt_ScrollToAd(adTypeId) {
		if (typeof adTypeId === "undefined") {
			adTypeId = 0;
		}
		var adId = "";
		var offset = 0;
		var objLeaderboard = "";
		var gpt_ScrollToAd = true;

		switch (gpt_AdFile) {
			case "PU": //PracticeUpdate
				var hideOtherAds = true;
				$('#modalPrestitial').modal('hide');
				$('.jumbotron-welcome').hide();
				objLeaderboard = ".container-leaderboard";
				offset = 53;
				switch (parseInt(adTypeId)) {
					case 2:
						//LeaderBoard, no Scroll Needed
						gpt_ScrollToAd = false;
						adId = "adSlot1";
						break;
					case 3:
						//MPU
						adId = "adSlot2";
						break;
					case 4:
						//SkyScraper
						adId = "adSlot3";
						$('.most-read').hide();
						break;
					case 5:
						//Mobile, no Scroll Needed
						gpt_ScrollToAd = false;
						adId = "adSlot1";
						break;
					case 6:
						//Interstitial
						gpt_ScrollToAd = false;
						adId = "adSlot15";
						await gpt_Sleep(1000);
						$(objLeaderboard).hide();
						$('#modalPrestitial').unbind();
						$('#modalPrestitial').modal();
						break;
					case 7:
						//Native Ad
						break;
					case 10:
						//Master Companion
						hideOtherAds = false;
						gpt_ScrollToAd = false;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						adId = "adSlot1";
						break;
				}
				if (hideOtherAds) {
					//Hide all other ads
					$('.ad-container').each(function (i, obj) {
						if (this.id !== adId) {
							var tempParent = this.id;
							tempParent = tempParent.replace("adSlot", "adParent");
							$("#" + tempParent).addClass('hide');
						}
					});
				}
				break;
			case "JBS":
				objLeaderboard = ".js__adToHide";
				offset = 80;
				switch (parseInt(adTypeId)) {
					case 2:
						//Leaderboard
						gpt_ScrollToAd = false;
						$(".widget-verticalAd").hide();
						break;
					case 5:
						//Mobile
						//Scroll to Top
						gpt_ScrollToAd = false;
						break;
					case 3: //MPU
					case 4: //SkyScraper
						$(objLeaderboard).hide();
						adId = $('.gpt-ad-defined[data-adsize="sidebar"').attr('id');
						break;
					case 10:
						//MasterCompanion
						gpt_ScrollToAd = false;
						break;
					case 13:	    //Phone - Portrait
					case 14:        //Phone - Landscape
						$(objLeaderboard).hide();
						offset = 60;
						adId = 'gpt-Interscroller';
						gpt_ScrollToAd = true;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						break;
				}
				break;
			case "CELLJOURNAL":
			case "CELLBUCKET":
				objLeaderboard = ".js__adToHide";
				offset = 260;
				switch (parseInt(adTypeId)) {
					case 2:
						//Leaderboard
						gpt_ScrollToAd = false;
						$('.body-verticalAd').hide();
						break;
					case 5:
						//Mobile
						//Scroll to Top
						gpt_ScrollToAd = false;
						break;
					case 3: //MPU
					case 4: //SkyScraper
						$(objLeaderboard).hide();
						adId = $('.gpt-ad-defined[data-adsize="boombox"').attr('id');
						break;
					case 10:
						//MasterCompanion
						gpt_ScrollToAd = false;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						break;
				}
				//Hide the second ad if found
				var tCount = 0;
				$(".widget.literatumAd.none.GPT-ad-placeholder.widget-horizontalAd.widget-compact-all").each(function (index) {
					tCount += 1;
					if (tCount !== 1) {
						$(this).hide();
					}
				});
				break;
			case "LANCET":
				objLeaderboard = ".js__adToHide";
				offset = 80;
				switch (parseInt(adTypeId)) {
					case 2:
						//Leaderboard
						gpt_ScrollToAd = false;
						$('.body-verticalAd').hide();
						break;
					case 5:
						//Mobile
						//Scroll to Top
						gpt_ScrollToAd = false;
						break;
					case 3: //MPU
					case 4: //SkyScraper
						$(objLeaderboard).hide();
						adId = $('.gpt-ad-defined[data-adsize="skyscraper"').attr('id');
						break;
					case 10:
						//MasterCompanion
						gpt_ScrollToAd = false;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						break;
				}
				break;
		}

		//Scroll to Top
		var gpt_container = $("html,body");
		gpt_container.animate({ scrollTop: 0, scrollLeft: 0 });

		if (gpt_ScrollToAd) {
			//If Adexists, scroll to it
			var gpt_scrollTo = $('#' + adId);
			if (gpt_scrollTo.length > 0) {
				//Scroll to the Ad
				var gpt_topPosition = gpt_scrollTo.offset().top - gpt_container.offset().top + gpt_container.scrollTop();
				if (gpt_topPosition > 0) {
					gpt_container.animate({ scrollTop: gpt_topPosition, scrollLeft: 0 });
					if (gpt_topPosition - offset > 0 && offset > 0) {
						gpt_container.animate({ scrollTop: gpt_topPosition - offset, scrollLeft: 0 });
					}
				}
			}
		}
	}
	//--------------------------------------------------------------------------
	async function gpt_CheckForCookiePolicy() {
		var returnVal = false;
		if ($('#onetrust-accept-btn-handler').length > 0) {
			$('#onetrust-accept-btn-handler').click();
			returnVal = true;
		}
		return returnVal;
	}
	//--------------------------------------------------------------------------
	function gpt_InitAds() {
		var gpt_ResponsiveHomepage = false;

		switch (gpt_AdFile) {
			case "JBS":
			case "CELLJOURNAL":
			case "CELLBUCKET":
				//Responsive Check
				if ($('#hdrHomeBrand').length > 0) {
					gpt_ResponsiveHomepage = true;
				}
				break;
		}

		//InterScroller Check
		if (gpt_AdFile === "JBS") {
			var gpt_InterScrollerEnabled = false;
			var gpt_InterScrollerInsertObj;

			if ($("#articleHeader").length > 0) {
				//It's an article Page
				if ($('.article__sections > section').length > 0) {
					gpt_InterScrollerEnabled = true;
					gpt_InterScrollerInsertObj = $(".article__sections > section");
				}
			}
			else {
				if ($('.toc__section').length > 0) {
					//It's a TOC Page
					gpt_InterScrollerEnabled = true;
					gpt_InterScrollerInsertObj = $(".toc__section").first();
				}
			}

			//Add the Interscroller
			if (gpt_InterScrollerEnabled) {
				gpt_InterScrollerInsertObj.first().append($("#gpt-Interscroller").clone());
				$("#gpt-Interscroller").remove();
				$(".gpt-adinterscroller").addClass("gpt-ad");
			}
		}

		googletag.cmd.push(function () {
			var gptBoomBoxSizeMapping;
			var gptFluidMapping;
			var gptHomePageSidebarMapping;
			var gptCellHomePageSidebarMapping;
			var gptInterscrollerMapping;
			var gptLeaderboardSizeMapping;
			var gptMobileSizeMapping;
			var gptPrestitialMapping;
			var gptRectangleShortSizeMapping;
			var gptRectangleSizeMapping;
			var gptSideBarSizeMapping;
			var gptTallSizeMapping;
			var gptWideSizeMapping;
			var gptWideRectangleSizeMapping;

			//***********************************************
			//Start Define Size mapping based on client Viewport
			//***********************************************
			if (gpt_AdFile === 'PU') {
				gptTallSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[300, 600], [160, 600], [120, 600]]).
					addSize([0, 0], []).
					build();

				gptRectangleSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[300, 250]]).
					addSize([0, 0], []).
					build();

				gptRectangleShortSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[300, 250], [320, 50], [300, 50]]).
					addSize([0, 0], [[300, 250], [320, 50], [300, 50]]).
					build();

				gptWideSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[728, 90]]).
					addSize([760, 0], [[728, 90], [320, 50], [300, 50]]).
					addSize([0, 0], [[320, 50], [300, 50]]).
					build();

				gptWideRectangleSizeMapping = googletag.sizeMapping().
					addSize([992, 0], []).
					addSize([760, 0], [[320, 50], [300, 50], [300, 250]]).
					addSize([0, 0], [[320, 50], [300, 50], [300, 250]]).
					build();

				gptMobileSizeMapping = googletag.sizeMapping().
					addSize([0, 0], [[320, 50], [300, 50]]).
					build();

				gptFluidMapping = googletag.sizeMapping().
					addSize([0, 0], ['fluid']).
					build();

				gptPrestitialMapping = googletag.sizeMapping().
					addSize([760, 0], [[1, 1]]).
					build();
			}
			else {
				//JBS-CELL-LANCET
				gptLeaderboardSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[728, 90]]).
					addSize([768, 0], [[728, 90]]).
					addSize([0, 0], [[320, 50], [300, 50]]).
					build();

				gptHomePageSidebarMapping = googletag.sizeMapping().
					addSize([2090, 795], [[300, 600], [300, 250], [160, 600], [120, 600]]).
					addSize([1910, 795], [[160, 600], [120, 600]]).
                    addSize([1740, 795], [[120, 600]]).
					addSize([0, 0], []).
					build();

				gptCellHomePageSidebarMapping = googletag.sizeMapping().
                    addSize([2180, 795], [[300, 600], [160, 600], [120, 600], [300, 250], [336, 280]]).
					addSize([2180, 445], [[336, 280]]).
                    addSize([2090, 795], [[300, 600], [160, 600], [120, 600], [300, 250]]).
                    addSize([2090, 445], [[300, 250]]).
					addSize([1830, 795], [[160, 600], [120, 600]]).
                    addSize([1740, 795], [[120, 600]]).
					addSize([0, 0], []).
					build();

				gptSideBarSizeMapping = googletag.sizeMapping().
					addSize([1140, 795], [[300, 600], [160, 600], [120, 600], [300, 250]]).
					addSize([1140, 445], [[300, 250]]).
					addSize([0, 0], []). 				
					build();

				gptBoomBoxSizeMapping = googletag.sizeMapping().
					addSize([768, 0], [[300, 250], [336, 280]]).
					addSize([0, 0], []). 						
					build();

				gptFluidMapping = googletag.sizeMapping().
					addSize([0, 0], ['fluid']).
					build();

				gptPrestitialMapping = googletag.sizeMapping().
					addSize([600, 700], [[1, 1]]).
					addSize([0, 0], []).
					build();

				gptInterscrollerMapping = googletag.sizeMapping().
					addSize([1140, 445], []).
					addSize([1024, 768], [[1024, 768]]).
					addSize([768, 1024], [[768, 1024]]).
					addSize([480, 320], [[480, 320]]).
					addSize([320, 480], [[320, 480]]).
					build();
			}
			//***********************************************
			//End Define Size mapping based on client Viewport
			//***********************************************

			//***********************************************
			//Start detection for ad slots on page
			//***********************************************
			var gptMasterPage = gpt_CheckForNULL($('#gptPage').val());
			var gptMasterSite;

			if (gpt_AdFile === 'PU') {
				gptMasterSite = $('#gptSite').val();
			}
			else {
				//JBS-CELL-LANCET
				gptMasterSite = "dev";

				if (gpt_IsProd) {
					gptMasterSite = gpt_CheckForNULL($('#gptSite').val());

					//If there is no master site, check for a defaultsite
					if (gptMasterSite == null) {
						gptMasterSite = gpt_CheckForNULL($('#gptDefaultSite').val());
					}
				}
				else {
					//Dev ENV
					//If the page has no gptSite Value, assume no value which will disable ads
					if (gpt_CheckForNULL($('#gptSite').val()) === null) {
						gptMasterSite = null;
					}
				}

				//If there is no master Page, check for a Page using the URL
				if (gptMasterPage === null) {
					gptMasterPage = GPT_GetPageByURL();
				}
			}

			$(".gpt-ad").each(function (index) {
				var gptPage = '';
				var adSizeMapping = '';
				var gptPOS;
				var gptTC;
				var gptNative;
				var adId;
				var gptSite;
				var tempSlot = null;


                if (gpt_IsProd) {
					if (gpt_AdFile !== "CELLJOURNAL") {
						//Check if the ad slot has it's own site defined
						if (typeof $(this).data('site') === 'undefined') {
							gptSite = gptMasterSite;
						}
						else {
							gptSite = gpt_CheckForNULL($(this).data('site'));
						}
					}
					else {
						gptSite = gptMasterSite;
					}
                }
                else {
					gptSite = gptMasterSite;
                }



				if (gpt_AdFile === 'PU') {
					adId = $(this).attr('id');
				}
				else {
					//JBS-CELL-LANCET
					adId = 'gpt-ad-' + index + 1;
				}

				if (gpt_CheckForNULL(gptSite) !== null) {
					gptSite_Debug = gptSite;

					gptPOS = parseInt($(this).data('pos')) || -1;
					if (gptPOS === -1) {
						gptPOS = index + 1;
					}

					//Check if the ad slot has it's own page defined
					gptPage = $(this).data('page');
					if (gpt_CheckForNULL(gptPage) == null) {
						//Use the Master Page
						gptPage = gptMasterPage;
					}

					//If the page is not null, preceed it with a slash
					if (gptPage !== null) {
						gptPage = '/' + gptPage;
					}
					else {
						gptPage = '';
					}
					gptPage_Debug = gptPage;


					gptTC = $(this).data('tc');
					if (typeof gptTC === "undefined") {
						gptTC = '';
					}

					gptNative = $(this).data('native');
					if (typeof gptNative === "undefined") {
						gptNative = '';
					}

					//Determine sizemapping based on data-adsize
					if (gpt_AdFile === 'PU') {
						switch ($(this).data('adsize')) {
							case 'tall':
								adSizeMapping = gptTallSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600]], adId);
								break;
							case 'rectangle':
								adSizeMapping = gptRectangleSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250]], adId);
								break;
							case 'rectangleshort':
								adSizeMapping = gptRectangleShortSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250], [320, 50], [300, 50]], adId);
								break;
							case 'wide':
								adSizeMapping = gptWideSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[728, 90], [320, 50], [300, 50]], adId);
								break;
							case 'widerectangle':
								adSizeMapping = gptWideRectangleSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[320, 50], [300, 50], [300, 250]], adId);
								break;
							case 'mobile':
								adSizeMapping = gptMobileSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[320, 50], [300, 50]], adId);
								break;
							case 'fluid':
								adSizeMapping = gptFluidMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, 'fluid', adId);
								break;
							case 'prestitial':
								adSizeMapping = gptPrestitialMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[1, 1]], adId);
								break;
						}
					}
					else {
						//JBS-CELL-LANCET
						switch ($(this).data('adsize')) {
							case 'cell-leaderboard':
							case 'leaderboard':
								adSizeMapping = gptLeaderboardSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[728, 90], [320, 50], [300, 50]], adId);
								break;
							case 'bottom':
								if (gpt_ResponsiveHomepage) {
									adSizeMapping = gptHomePageSidebarMapping;
									tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600], [300, 250]], adId);
								}
								else {
									adSizeMapping = gptBoomBoxSizeMapping;
									tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250]], adId);
								}
								break;
							case 'boombox':
								if (gpt_AdFile === "CELLJOURNAL" || gpt_AdFile == "CELLBUCKET") {
									if (gpt_ResponsiveHomepage) {
										adSizeMapping = gptCellHomePageSidebarMapping;
										tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600], [300, 250], [336, 280]], adId);
									}
									else {
										adSizeMapping = gptBoomBoxSizeMapping;
										tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250], [336, 280]], adId);
									}
								}
								else {
									adSizeMapping = gptBoomBoxSizeMapping;
									tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250]], adId);
								}
								break;
							case 'interscroller':
								adSizeMapping = gptInterscrollerMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[320, 480], [480, 320], [768, 1024], [1024, 768]], adId);
								break;
							case 'sidebar':
							case 'skyscraper':
								adSizeMapping = gptSideBarSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600], [300, 250]], adId);
								break;
							case 'native':
							case 'fluid':
								adSizeMapping = gptFluidMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, 'fluid', adId);
								break;
							case 'prestitial':
								adSizeMapping = gptPrestitialMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, 'fluid', adId);
								break;
						}
					}
				}

				//Ignore any invalid adsizes that could be on the page
				if (adSizeMapping !== '') {
					$(this).addClass('gpt-ad-defined');
					if (gpt_AdFile !== 'PU') {
						$(this).attr('id', adId);
					}
					tempSlot.defineSizeMapping(adSizeMapping).addService(googletag.pubads()).setTargeting("pos", gptPOS);

					if (gptTC !== '') {
						tempSlot.setTargeting("tc", gptTC);
					}
					if (gptNative !== '') {
						tempSlot.setTargeting("native", gptNative);
					}
					gptEvent_Debug = gptEvent_Debug + $(this).data('adsize') + ' Requested<br>';
				}
				else {
					gptInvalidAdSize_Debug = gptInvalidAdSize_Debug + "Invalid Size Found, Id: " + $(this).prop('id') + ', adSize: ' + $(this).data('adsize') + '<br>';
					$(this).remove();
				}
			});
			if (gpt_CheckForNULL(gptInvalidAdSize_Debug) === null) {
				gptInvalidAdSize_Debug = "None Detected";
			}
			//***********************************************
			//End detection for ad slots on page
			//***********************************************

			//***********************************************
			//Start page level targeting
			//***********************************************
			if (gpt_AdFile !== 'PU') {
				//Logged in or IP Access
				gpt_TargetAccessType();

				//Check for Screenshot Client
				gpt_CheckSSC();
			}

			//Check for DFP Test
			gpt_GetDFPTestId();

			//Check for Additional Targeting
			gpt_GetAdditionalTargeting();

			if (gpt_IsProd && gptContextualEnabled) {
				var gpt_ContextualRetry = false;
				//Check for Contextual Targeting
				if (gs_channels !== "DEFAULT") {
					var gpt_ContextualCategories = Array.from(gs_channels);
					for (var i = 0; i < gpt_ContextualCategories.length; i++) {
						if (gpt_ContextualCategories[i] == 'gx_retry') {
							gpt_ContextualRetry = true;
						}
						if (!gpt_ContextualCategories[i].startsWith("custom_")) {
							//Remove any tag that is not custom
							gpt_ContextualCategories.splice(i, 1);
							i--;
						}
						else {
							//Strip the 'custom_' from the tag
							gpt_ContextualCategories[i] = gpt_ContextualCategories[i].replace("custom_", "")
						}
					}
					if (gpt_ContextualCategories.length > 0) {
						googletag.pubads().setTargeting("gs_cat", gpt_ContextualCategories);
					}
					else {
						gpt_ContextualCategories = 'none';
					}
					if (gpt_ContextualRetry) {
						gpt_ContextualCategories = "*Indexing*";
					}
					gptContextualResult_Debug = gpt_ContextualCategories;
				}
				else {
					gptContextualResult_Debug = 'No segments found';
				}
			}
			else {
				//Not Prod || Contextual not enabled
				gptContextualResult_Debug = 'Not enabled';
				
				if (!gpt_IsProd) {
                    var gpt_DevSite = gpt_CheckForNULL($('#gptSite').val(), "dev");
                    $('#gpt-Variables').append('<input type="hidden" id="gptEnvironment" value="' + gpt_DevSite + '">');
                    gpt_Target("Environment");
                }

			}
			//***********************************************
			//End page level targeting
			//***********************************************

			//Enable google dfp services
			if (parseInt($('#gptLazyLoading').val()) === 1) {
				googletag.pubads().enableLazyLoad({
					fetchMarginPercent: 200,
					renderMarginPercent: 0,
					mobileScaling: 2.0
				});
			}
			else {
				googletag.pubads().enableSingleRequest();
			}

			//Add Listener for RenderEnded
			googletag.pubads().addEventListener('slotRenderEnded', function (event) {
				if (!event.isEmpty) {
					if (typeof LocalAdLoaded === "function") {
						LocalAdLoaded(event);
					}
					if (gpt_ResponsiveHomepage) {
						if (typeof positionAd === "function") {
							positionAd();
						}
					}
					if (event.size[0] == "480" || event.size[1] == "480") {
						$('.gpt-InScrollBox').css('display', 'block');
						gptEvent_Debug = gptEvent_Debug + 'Interscroller ' + event.size + ' rendered: ' + gpt_GetElapsedTime() + '<br>';
					}
					else {
						gptEvent_Debug = gptEvent_Debug + event.size + ' rendered: ' + gpt_GetElapsedTime() + '<br>';
					}

				}
			});

			if (gpt_AdFile !== 'PU') {
				googletag.pubads().collapseEmptyDivs();
			}
			googletag.enableServices();

			//Populate Defined Ad Slots
			$(".gpt-ad-defined").each(function (index) {
				var adId = $(this).attr('id');
				googletag.cmd.push(function () { googletag.display(adId); });
			});

			gptStatus_Debug = 'GPT Complete';
		});
	}
	//--------------------------------------------------------------------------
	function gpt_DisplayDebug() {
		//Clone the template and append to Body
		$(document.body).append($('#gpt-DebugDiv').clone());
		$('#gpt-DebugDiv').remove();

		//Populate the Values
		gpt_LoadDebugValues();

		//Display and scroll to debug section
		$('.gpt-DebugDiv').css('display', 'block');
		window.scrollTo(0, document.body.scrollHeight);
	}
	//--------------------------------------------------------------------------
	function gpt_LoadDebugValues() {
		$('.gpt-DebugBtn').blur();
		$('.gptVersion_Debug').text(gpt_Version);
		$('.gptAdFile_Debug').html(gpt_AdFile);
		$('.gptEvent_Debug').html(gptEvent_Debug);
		$('.gptCustomTargeting_Debug').html(gptCustomTargeting_Debug);
		$('.gptSite_Debug').text(gptSite_Debug);
		$('.gptContextualResult_Debug').text(gptContextualResult_Debug);
		$('.gptContextualInit_Debug').text(gptContextualInit_Debug);
		$('.gptContextualURL_Debug').text(gptContextualURL_Debug);
		$('.gptContextualReady_Debug').text(gptContextualReady_Debug);
		$('.gptMoatReady_Debug').text(gptMoatReady_Debug);
		$('.gptMoatInit_Debug').text(gptMoatInit_Debug);
		$('.gptTestENV_Debug').text(gptTestENV_Debug);
		$('.gptDisableAds_Debug').text(gptDisableAds_Debug);
		$('.gptStatus_Debug').text(gptStatus_Debug);
		$('.gptPage_Debug').text(gptPage_Debug);
		$('.gptInvalidAdSize_Debug').html(gptInvalidAdSize_Debug);
	}
	//--------------------------------------------------------------------------
	function gpt_GrapeShotDebug() {
		$('.gpt-DebugBtn').blur();
		window.open("http://admin-elsevier.grapeshot.co.uk/custom/bookmarklet.cgi?blocking=1&vn=2&url=" + encodeURIComponent(document.location), "GRAPESHOT_POPUP", "width=650,height=600,scrollbars=yes,screenX=40,screenY=50,menubar=no,toolbar=no,resizable=yes", "Custom").focus();
	}
	//--------------------------------------------------------------------------
	function gpt_ShowConsole() {
		$('.gpt-DebugBtn').blur();
		googletag.openConsole();
	}
	//--------------------------------------------------------------------------
</script>
        </div>

            
                </div>
            

        

    

        


    

            

            

            <div data-widget-id="912135d6-7135-4d45-b457-b503cacc3c8b" data-widget-name="ux3-layout-widget"><header class="header base fixed">
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none adplaceholder adplaceholder--header js__adToHide widget-horizontalAd" id="9d6062d7-fad6-4e64-81cc-7dd25d99b23d">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-horizontalAd "><div class="horizontalAdContainer">
    <div class="horizontalAdLabelContainer">
        <div class="horizontalAdLabel">
                <img src="/templates/jsp/_style2/_marlin/images/bg_adHoriz.gif" alt="Advertisement">
        </div>
    </div>
    <div class="horizontalAd">
        





        <div class="pb-ad">
            <!-- Beginning Async AdSlot 1 for Ad unit els.cellpress/cell  ### size: [[728,90]] -->
<div class="gpt-ad" data-adsize="cell-leaderboard" data-page="" data-site="cell" data-pos="1"></div>
<!-- End AdSlot 1 -->
        </div>
    

    </div>
</div></div>
                </div>
                </div>

        










        <div data-widget-def="ux3-layout-widget" data-widget-id="3a83e2a6-2e7e-4931-892c-4ef43b6e3312" class="header__wrapper clearfix" data-db-parent-of="quickSearch"><div id="brandingRow"><a href="/" data-aa-name="science that inspires" data-aa-region="cp global header" title="Visit Cell.com">Science that inspires</a><p id="brandingRowRight"><a href="/" data-aa-name="cell press endorsement" data-aa-region="cp global header" title="Visit Cell.com">A Cell Press journal</a></p></div>
        
    

        


    

            

            

            <div data-widget-id="3a83e2a6-2e7e-4931-892c-4ef43b6e3312" data-widget-name="ux3-layout-widget"><div class="col-xs-6 header__left-side">
    

        


    

            

            
                <a href="/home" id="cpIconLnk" data-aa-name="cell press logo" data-aa-region="cp global header" title="Cell Press" target="_blank"><svg id="cp_logo" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 100 98.1" style="enable-background:new 0 0 100 98.1;" xml:space="preserve" height="100" width="98.1"><path style="fill:#FFFFFF;" d="M69.09,0.09c-4.01-0.6-7.83,1.94-8.78,5.89l-3.14,13.11c-0.76,3.19-3.44,5.56-6.7,5.93l-18.6,2.13          C11.83,29.31-2.53,47.66,0.37,67.78c2.3,15.96,15.18,27.77,30.54,30.08c4.01,0.6,7.83-1.94,8.78-5.89l3.14-13.11 c0.76-3.19,3.44-5.56,6.69-5.93l18.6-2.13c20.04-2.17,34.41-20.52,31.5-40.65C97.32,14.21,84.45,2.39,69.09,0.09z M65.06,47.2  l-11.75,1.4c-1.98,0.21-3.62,1.65-4.08,3.59l-3.43,14.3c-1.01,4.22-4.5,7.39-8.8,8c-6.66,0.72-12.63-4.2-13.15-10.92                c-0.51-6.5,4.6-12.11,11.08-12.81l11.75-1.39c1.98-0.22,3.62-1.65,4.08-3.59l3.43-14.3c1.01-4.22,4.5-7.39,8.8-8          c6.66-0.72,12.63,4.19,13.15,10.92C76.66,40.89,71.55,46.5,65.06,47.2z"></path></svg></a><div class="journal-logos">
            

            <span data-widget-id="35d8d734-3aa0-4ec8-9e2d-14babe803594" data-widget-name="ux3-general-image"><a href="/cell/home" title="Cell"><img alt="Cell" src="/pb-assets/ux3/logos/cell/Cell_white-1532516611593.svg"></a></span>

            
                </div>
            

        

    

        


    

            

            
                <div class="journal-logos">
            

            <span data-widget-id="3ffcbf1f-79fd-4ba3-85b5-bef10a63475f" data-widget-name="ux3-general-image"><a href="https://www.cell.com/open-access?utm_source=oa-label&amp;__hstc=109691226.e4e812b53ff324eccbdbdb2e4fb9a6ab.1644503737405.1644524234622.1644587109507.4&amp;__hssc=109691226.7.1644587109507&amp;__hsfp=3253153019" title="This journal offers authors two options (open access or subscription) to publish research"><img alt="This journal offers authors two options (open access or subscription) to publish research" src="/pb-assets/ux3/logos/cell/Supports_open_access_border-1645554937897.svg"></a></span>

            
                </div>
            

        
</div><div class="col-xs-6">
    

        


    

            

            

            <div data-widget-id="2555f6db-b333-4edb-8b15-ef9213f1f0a5" data-widget-name="ux3-layout-widget"><div class="header__nav row">
    

        


    

            

            

            <ul data-widget-id="52432d24-0331-4c9d-85f3-2033edc7208b" data-widget-name="UX3ProfileLinksWidget" class="rlist rlist--inline profile-links"><li class="profile-links__submit"><a data-adobe-name="Publish" class="profile-links__submit__btn" title="Submit" href="https://www.editorialmanager.com/cell" target="_self" data-aa-conversiondriver="jbs:global-header:submitpage" data-profile-link="true">Submit</a></li><li class="profile-links__login-dropdown"><div class="profile-links__user-list__wrapper"><button data-db-target-for="loginmenu-192970" aria-expanded="false" aria-controls="login-links-192970" title="Log in links dropdown" class="profile-links__username"><span>Log in</span></button><div data-db-target-of="loginmenu-192970" id="login-links--192970" class="profile-links__user-list__holder dropBlock__holder"><ul class="rlist profile-links__list"><li class="profile-links__login"><a data-adobe-name="Log In" class="profile-links__login__btn" title="Log in to Cell.com" href="https://www.cell.com/action/idLogin?type=login&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2820%2930682-6" target="_self" data-profile-link="true">Log in</a></li><li class="profile-links__institution"><a data-adobe-name="Get institutional access" class="profile-links__institution__btn" title="Institutional access" href="https://www.cell.com/action/idLogin?type=institution&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2820%2930682-6" target="_self" data-profile-link="true">Get Institutional Access</a></li></ul></div></div></li><li class="profile-links__register"><a data-adobe-name="Register" class="profile-links__register__btn" title="Register a username and password for free" href="https://www.cell.com/action/idLogin?type=registration&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2820%2930682-6" target="_self" data-profile-link="true">Register</a></li><li class="hidden-lg"> <div class="profile-links__controls"> <button aria-expanded="false" aria-controls="links-dialog-192970" data-opens="#links-dialog-192970" class="profile-links__btn--open"><span class="sr-only">open links dialog</span><i aria-hidden="true" class="icon-gizmo-person"></i></button></div><dialog id="links-dialog-192970" aria-modal="true"><button data-closes="#links-dialog-192970" class="profile-links__btn--close"><span class="sr-only">close links dialog</span><i aria-hidden="true" class="icon-close-solid"></i></button><ul>   <li class="profile-links__submit"><a data-adobe-name="Publish" class="profile-links__submit__btn" title="Submit" href="https://www.editorialmanager.com/cell" target="_self" data-aa-conversiondriver="jbs:global-header:submitpage" data-profile-link="true">Submit</a></li><li class="profile-links__login"><a data-adobe-name="Log In" class="profile-links__login__btn" title="Log in to Cell.com" href="https://www.cell.com/action/idLogin?type=login&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2820%2930682-6" target="_self" data-profile-link="true">Log in</a></li><li class="profile-links__institution"><a data-adobe-name="Get institutional access" class="profile-links__institution__btn" title="Institutional access" href="https://www.cell.com/action/idLogin?type=institution&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2820%2930682-6" target="_self" data-profile-link="true">Get Institutional Access</a></li><li class="profile-links__register"><a data-adobe-name="Register" class="profile-links__register__btn" title="Register a username and password for free" href="https://www.cell.com/action/idLogin?type=registration&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2820%2930682-6" target="_self" data-profile-link="true">Register</a></li></ul></dialog></li></ul>

            

        
</div></div>

            

        
</div></div>

            

        

        </div>
    

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget csasInstitutionBanner none  widget-none" id="51b7fa33-2137-4f58-9e98-9746c4400513" style="display: none;">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none "><div class="ui-state-highlight ui-corner-all csas-institution-banner hidden" data-doi="10.1016/j.cell.2020.05.049">
    <span class="ui-icon ui-icon-close banner-close"></span>
    <p>
        Access provided by <span class="accountName"></span>
    </p>
</div>

<style>
    .csas-institution-banner {
        padding: 10px 0 0 10px;
        margin: 5px;
    }

    .csas-institution-banner.ui-state-highlight .ui-icon {
        background-image: url("/templates/jsp/css/jquery-ui-1.10.2/base/images/ui-icons_454545_256x240.png");
        margin-right: 5px;
        margin-top: -4px;
    }

    .csas-institution-banner .banner-close {
        float: right;
    }
</style>

<script>

    function resetBanner(doi) {
        $.get('/action/getAccount?doi=' + doi, function(data) {
            if (data) {
                $(".csas-institution-banner p .accountName").html(data);
                $(".csasInstitutionBanner").fadeIn();
                $(".csas-institution-banner").fadeIn();
            }
        });
    }

    var flag = 0;
    $(document).ready(function() {
        if(!$(".csas-institution-banner").is(":visible")){
            $(".csasInstitutionBanner").hide();
        }

        $(".banner-close").click(function() {
            $.get("/action/dismissBanner", function(data) {
                $(".csasInstitutionBanner").fadeOut(function () {
                    if(typeof UX !== 'undefined') {
                        UX.pageBody.init();
                    }
                });
            });
        });

        $(".articleContent .tabs li[role='tab'].showFullText").click(function() {
           flag = 1;
        });

        $(document).on("reloadTabsContent", function() {
            var isSelected = $(".articleContent .tabs li[role='tab'].showFullText").hasClass("active");
            if (isSelected) {
                var isHidden =  $(".csas-institution-banner").is(".hidden");
                if (!isHidden) {
                    resetBanner($(".csas-institution-banner").data("doi"));
                    console.log("fadeIn");
                }
            } else {
                $(".csas-institution-banner").fadeOut();
                console.log("fadeOut");
            }
            flag = 0;
        });

        $(document).on("click", ".articleContent .tabs li[role='tab']", function() {

        });
    });
</script></div>
                </div>
                </div>

        
</header></div>

            

        










        <div data-widget-def="graphQueryWidget" data-widget-id="2a125dae-6dbc-4e76-a75e-3e563f1d3a1d" class="ux3">
        
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget graphQueryWidget none top-padding-handler noMargin axel-mega-menu widget-none widget-compact-all fonts-loaded" id="2a125dae-6dbc-4e76-a75e-3e563f1d3a1d" style="padding-top: 115.594px;">
                
                
                    
                

                <div class="wrapped " id="CP TOP LEVEL NAV MENU">
                    
                    <div class="widget-body body body-none  body-compact-all"><nav id="mega-menu-nav" role="navigation" class="mega-menu-nav">
  <!--.skip//a(href='#searchText' aria-label='Skip menu') Skip menu
  -->
  <div class="mega-menu-nav__wrapper">
    <button class="mega-menu__toggle"><span class="sr-only">Main menu</span><i aria-hidden="true" class="icon-gizmo-hamburger"></i></button>
    <ul role="menubar" class="mega-menu rlist">
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-1" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-1" data-item-link="main-menu-link-1" data-index="0" class="mega-menu__item-link menu-link"><span>This journal</span></a>
        <div id="main-menu-panel-1" aria-labelledby="main-menu-link-1" data-item-wrapper="main-menu-link-1" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <div aria-label="Menu dropzone section" tabindex="-1" class="listDropZone">
    

        


    

            

            

            <div class="cover-image"><div class="cover-image__thumb"><a href="/issue/S0092-8674(24)X0012-4" class="menu-link"><img src="/cms/asset/atypon:cms:attachment:img:d5e6:rev:1748270185670-389:pii:S0092867424X00124/cover.tif.jpg" alt="Cover Image - Cell, Volume 188, Issue 11"></a></div><div class="cover-image__meta"></div></div>

            

        
</div>
            </li>
            <li style="grid-column: 2/3" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-1-2" class="sub-menu__heading">Read
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/newarticles" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Online now</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/current" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Current issue</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/archive" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Archive</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/action/showPreferences?menuTab=Alerts" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Publication alerts</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/sneakpeek" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Sneak Peek - preprints</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 3/5" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-1-3" class="sub-menu__heading">About
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-2">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/advisory-board" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Advisory board</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/aims" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Aims and scope</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/contact" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Contact</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/editors-and-staff" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Editors and staff</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/editorial-ethics" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Ethics</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/information-for-authors/permissions" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Permissions</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/reviewers" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>Reviewer instructions</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-2" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-2" data-item-link="main-menu-link-2" data-index="1" class="mega-menu__item-link menu-link"><span>Journals</span></a>
        <div id="main-menu-panel-2" aria-labelledby="main-menu-link-2" data-item-wrapper="main-menu-link-2" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/5" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-2-1" class="sub-menu__heading">Life &amp; medical sciences
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-4">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/home" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cancer-cell/home" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Cancer Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-chemical-biology" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Chemical Biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-genomics/home" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Genomics</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-host-microbe/home" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Host &amp; Microbe</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-metabolism/home" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Metabolism</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports/home" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-medicine/home" data-index="7" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Medicine</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-stem-cell/home" data-index="8" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Stem Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-systems/home" data-index="9" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Systems</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/current-biology/home" data-index="10" role="menuitem" class="sub-menu__item-link menu-link"><span>Current Biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/developmental-cell/home" data-index="11" role="menuitem" class="sub-menu__item-link menu-link"><span>Developmental Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/immunity/home" data-index="12" role="menuitem" class="sub-menu__item-link menu-link"><span>Immunity</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/med/home" data-index="13" role="menuitem" class="sub-menu__item-link menu-link"><span>Med</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-cell/home" data-index="14" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/neuron/home" data-index="15" role="menuitem" class="sub-menu__item-link menu-link"><span>Neuron</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/structure/home" data-index="16" role="menuitem" class="sub-menu__item-link menu-link"><span>Structure</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/ajhg/home" data-index="17" role="menuitem" class="sub-menu__item-link menu-link"><span>American Journal of Human Genetics (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/biophysj/home" data-index="18" role="menuitem" class="sub-menu__item-link menu-link"><span>Biophysical Journal (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/biophysreports/home" data-index="19" role="menuitem" class="sub-menu__item-link menu-link"><span>Biophysical Reports (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/hgg-advances/home" data-index="20" role="menuitem" class="sub-menu__item-link menu-link"><span>Human Genetics and Genomics Advances (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-plant/home" data-index="21" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Plant (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/molecular-therapy/home" data-index="22" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/methods/home" data-index="23" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy Methods &amp; Clinical Development (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/nucleic-acids/home" data-index="24" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy Nucleic Acids (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/oncology/home" data-index="25" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy Oncology (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/plant-communications/home" data-index="26" role="menuitem" class="sub-menu__item-link menu-link"><span>Plant Communications (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/stem-cell-reports/home" data-index="27" role="menuitem" class="sub-menu__item-link menu-link"><span>Stem Cell Reports (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/biochemical-sciences/home" data-index="28" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Biochemical Sciences</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/cancer/home" data-index="29" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Cancer</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/cell-biology/home" data-index="30" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Cell Biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/ecology-evolution/home" data-index="31" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Ecology &amp; Evolution</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/endocrinology-metabolism/home" data-index="32" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Endocrinology &amp; Metabolism</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/genetics/home" data-index="33" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Genetics</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/immunology/home" data-index="34" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Immunology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/microbiology/home" data-index="35" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Microbiology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/molecular-medicine/home" data-index="36" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Molecular Medicine</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/neurosciences/home" data-index="37" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Neurosciences</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/parasitology/home" data-index="38" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Parasitology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/pharmacological-sciences/home" data-index="39" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Pharmacological Sciences</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/plant-science/home" data-index="40" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Plant Science</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 5/6" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-2-2" class="sub-menu__heading">Physical sciences &amp; engineering
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-biomaterials/home" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Biomaterials</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-physical-science/home" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Physical Science</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/chem/home" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Chem</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/chem-catalysis/home" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Chem Catalysis</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/device/home" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Device</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/joule/home" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Joule</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/matter/home" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>Matter</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/newton/home" data-index="7" role="menuitem" class="sub-menu__item-link menu-link"><span>Newton</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/chemistry/home" data-index="8" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Chemistry</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 6/7" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-2-3" class="sub-menu__heading">Multidisciplinary
              </h4>
              <ul role="menubar" data-index="2" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-methods/home" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Methods</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-sustainability/home" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Sustainability</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/heliyon/home" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Heliyon</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/iscience/home" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>iScience</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/one-earth/home" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>One Earth</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/patterns/home" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Patterns</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/star-protocols/home" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>STAR Protocols</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/nexus/home" data-index="7" role="menuitem" class="sub-menu__item-link menu-link"><span>Nexus (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/the-innovation/home" data-index="8" role="menuitem" class="sub-menu__item-link menu-link"><span>The Innovation (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/biotechnology/home" data-index="9" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Biotechnology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/cognitive-sciences/home" data-index="10" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Cognitive Sciences</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-3" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-3" data-item-link="main-menu-link-3" data-index="2" class="mega-menu__item-link menu-link"><span>Publish</span></a>
        <div id="main-menu-panel-3" aria-labelledby="main-menu-link-3" data-item-wrapper="main-menu-link-3" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-3-1" class="sub-menu__heading">Authors
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="https://www.editorialmanager.com/cell?__hstc=25856994.7201670a4397fa03148684920d4a8d38.1672762013831.1694716659709.1694718909620.146&amp;__hssc=25856994.15.1694718909620&amp;__hsfp=1366844671" data-index="0" target="_blank" role="menuitem" class="sub-menu__item-link menu-link"><span>Submit article
                      <div class="external-icon"></div><span class="sr-only">External Link</span></span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/information-for-authors" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Information for authors</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/open-access" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Open access</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/multi-journal-submission" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Multi-Journal Submission</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/information-for-authors/star-authors-guide" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>STAR Methods</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 2/3" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-3-2" class="sub-menu__heading">Reviewers
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/reviewers" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Information for reviewers</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-4" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-4" data-item-link="main-menu-link-4" data-index="3" class="mega-menu__item-link menu-link"><span>News &amp; events</span></a>
        <div id="main-menu-panel-4" aria-labelledby="main-menu-link-4" data-item-wrapper="main-menu-link-4" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/newsroom" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Newsroom</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/events" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Events</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/symposia" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Symposia</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/webinars" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Webinars</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/consortia" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Consortia Hub</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-5" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-5" data-item-link="main-menu-link-5" data-index="4" class="mega-menu__item-link menu-link"><span>About Cell Press</span></a>
        <div id="main-menu-panel-5" aria-labelledby="main-menu-link-5" data-item-wrapper="main-menu-link-5" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-5-1" class="sub-menu__heading">Cell Press
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/about" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>About Cell Press</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/contact" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Contact us</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="https://service.elsevier.com/app/overview/cell-press/" data-index="2" target="_blank" role="menuitem" class="sub-menu__item-link menu-link"><span>Help &amp; support
                      <div class="external-icon"></div><span class="sr-only">External Link</span></span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/open-access" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Open access at Cell Press</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/diversity" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Inclusion and diversity</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-press-50" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>50th anniversary</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 2/3" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-5-2" class="sub-menu__heading">Hubs
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/biotechnology-bioengineering" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Biotechnology and bioengineering</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-press-chemistry" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Chemistry</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/clinical-translational" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Clinical and translational</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/covid-19" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>COVID-19</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/fundamental-biology" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Fundamental biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/sustainability" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Sustainability</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 3/4" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-5-3" class="sub-menu__heading">Careers
              </h4>
              <ul role="menubar" data-index="2" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="https://relx.wd3.myworkdayjobs.com/en-US/ElsevierJobs" data-index="0" target="_blank" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Press careers
                      <div class="external-icon"></div><span class="sr-only">External Link</span></span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
    </ul>
    <div class="mega-menu-nav__quick-search">
    

        


    

            

            

            <div data-widget-id="a1f05748-166f-41e0-b01a-ca8ac1cc95d4" data-widget-name="UX3QuickSearchWidget" class="quick-search"><div class="dropBlock"><form action="https://www.cell.com/action/doSearch?journalCode=cell" name="quickSearchBar" method="get" title="Quick Search"><input id="type" type="hidden" value="quicksearch" name="type"><div class="quick-search__toggle autoCompleteWrapper"><input id="searchText" type="text" name="text1" placeholder="Search for..." enterkeyhint="search" autocomplete="off" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" data-group-titles-conf="3" data-publication-items-conf="3" aria-label="Enter a search term" class="searchText searchText-autocomplete ui-autocomplete-input"><button type="submit" title="Search" aria-label="Submit the search form" class="btn quick-search__button icon-search"><span>Go search</span></button><a href="https://www.cell.com/search/advanced?SeriesKey=cell&amp;ISSN=0092-8674&amp;journalCode=cell" title="Advanced search" class="quick-search__advanced-link">Advanced search</a><ul id="ui-id-1" tabindex="0" class="ui-menu ui-widget ui-widget-content ui-autocomplete ui-front" unselectable="on" style="display: none;"></ul></div><div class="quick-search__box"><div class="quick-search__box-inner"><div class="quick-search__box-wrapper"><select name="field1" aria-label="Select a search area"><option value="AllField">All content</option><option value="Title">Article title</option><option value="Contrib">Authors</option><option value="Keyword">Keywords</option><option value="Abstract">Abstract</option><option value="AbstractTitleKeywordFilterField">Article title, abstract, keywords</option></select><input type="hidden" name="journalCode" value="cell"><div class="quick-search__scope-options"><span>Search within:</span><label title="This journal" class="radio--primary"><input type="radio" name="SeriesKey" id="searchScopeSeries-a1f05748-166f-41e0-b01a-ca8ac1cc95d4" title="This journal" value="cell" checked="checked" aria-label="Search within current journal"><span>This journal</span></label><label title="Full site" class="radio--primary"><input type="radio" name="SeriesKey" id="searchScopeFullSite-a1f05748-166f-41e0-b01a-ca8ac1cc95d4" value="" title="Full site" aria-label="Search within full site"><span>Full site</span></label></div><a href="https://www.cell.com/search/advanced?SeriesKey=cell&amp;ISSN=0092-8674&amp;journalCode=cell" title="Advanced search" class="quick-search__advanced-link">Advanced search</a></div><a href="javascript:void(0)" aria-label="Close search options" title="Close search options" class="quick-search__box-close"><i aria-hidden="true" class="icon-tools_close"></i></a></div></div><div class="quick-search__alert hidden"><p>Please enter a term before submitting your search.</p><a href="#" class="quick-search__alert--close"> Ok</a></div></form></div></div>

            

        
</div>
  </div>
</nav>
<div id="page-body-id" class="page-body-flag"></div></div>
                </div>
                </div>

        

        </div>
    

    

        


    

            

            

            <div data-widget-id="f8c6daf9-9df1-4fd9-ba3a-7f5d7b3d4466" data-widget-name="UX3HTMLWidget"><style>
.journal-logos:nth-of-type(2) img {
max-height: 30px !important;
}
.journal-logos:nth-of-type(3) img {
max-height: 30px !important;
}
.ux3 .journal-logos + .journal-logos {
margin-left: 30px;
}
.header__wrapper.clearfix .col-xs-6:first-of-type {
width: 55%;
}
.header__wrapper.clearfix .col-xs-6:nth-of-type(2) {
width: 45%;
}
@media screen and (max-width: 1000px) {
.ux3 .journal-logos + .journal-logos {
display: none;
}
.header__wrapper.clearfix .col-xs-6:first-of-type {
width: 50%;
}
.header__wrapper.clearfix .col-xs-6:nth-of-type(2) {
width: 50%;
}
}

.ux3 .journal-logos:first-of-type {
   max-width: 500px;
}

.profile-links {
   white-space: nowrap;
}

@media only screen and (min-width: 740px) and (max-width: 1600px) {
   .journal-logos:nth-of-type(2) img {
      max-width: 250px !important;
   }
}

@media only screen and (min-width: 1024px) and (max-width: 1600px) {
   .header__wrapper.clearfix .col-xs-6:first-of-type {
      width: 60% !important;
   }
   .header__wrapper.clearfix .col-xs-6:nth-of-type(2) {
      width: 40% !important;
   }
}

@media only screen and (max-width: 1240px) {
   .ux3 .journal-logos + .journal-logos {
      display: none !important;
   }
}

</style></div>

            

        
</div></div>

            
                </div>
            

        

        </div>
    

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget axelPublicationContent none  widget-none" id="cadc9b41-a542-448a-9de4-8d0803320174">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none "><link id="build-style-article" rel="stylesheet" type="text/css" href="/products/marlin/cell/releasedAssets/css/build-article-b26f3342b722959c7931.css"><article xmlns="http://www.w3.org/1999/xhtml" data-design="pill" data-has="right-rail" vocab="http://schema.org/" typeof="ScholarlyArticle" data-pii="S0092-8674(20)30682-6" lang="en" dir="ltr"><div data-core-wrapper="header"><header data-extent="frontmatter"><div class="core-container"><div data-article-access="free" data-article-access-type="open" class="meta-panel"><span class="meta-panel__left-content"><span class="meta-panel__type"><span>Article</span></span></span><span class="meta-panel__middle-content"><span class="meta-panel__volumeIssue"><a href="/cell/issue?pii=S0092-8674(19)X0016-1"><span>Volume 182</span>, <span>Issue 2</span></a></span><span class="meta-panel__pages"><span>p297-316.e27</span></span><span class="meta-panel__onlineDate">July 23, 2020</span></span><span class="meta-panel__right-content"><span class="meta-panel__access meta-panel__access--archive"><span>Open Archive</span></span></span><div class="article-header__download-full-issue"><button title="Download full issue" type="submit" formaction="/action/downloadMultiplePdf?action=downloadIssuePdf&amp;journal=cell&amp;volume=182&amp;issue=2&amp;pii=S0092-8674%2819%29X0016-1" class="btn--download-issue btn--cancel download-full-issue"><i class="icon-download-new"></i><span>Download Full Issue</span></button><div class="table-of-content__alert table-of-content__alert--download"><p>Download started</p><a href="#" class="table-of-content__alert--close">Ok</a></div></div></div><h1 property="name">TBL1XR1 Mutations Drive Extranodal Lymphoma by Inducing a Pro-tumorigenic Memory Fate</h1><div class="contributors"><span class="authors"><span role="list"><span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au1" aria-controls="axel_author_au1"><span property="givenName">Leandro</span> <span property="familyName">Venturutti</span></a><div id="au1" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Leandro</span> <span property="familyName">Venturutti</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Division of Hematology/Oncology, Department of Medicine, Weill Cornell Medicine, Cornell University, New York, NY 10021, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Venturutti/Leandro" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a></sup></span> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au2" aria-controls="axel_author_au2"><span property="givenName">Matt</span> <span property="familyName">Teater</span></a><div id="au2" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Matt</span> <span property="familyName">Teater</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Division of Hematology/Oncology, Department of Medicine, Weill Cornell Medicine, Cornell University, New York, NY 10021, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Teater/Matt" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a></sup></span> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au3" aria-controls="axel_author_au3"><span property="givenName">Andrew</span> <span property="familyName">Zhai</span></a><div id="au3" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Andrew</span> <span property="familyName">Zhai</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Biochemistry, University of Toronto, Toronto, ON M5S 1A8, Canada</span></div></div><div class="core-author-link"><a href="/authored-by/Zhai/Andrew" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">2</a></sup></span><span data-displayed-on="all"> ∙ … </span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au4" aria-controls="axel_author_au4"><span property="givenName">Amy</span> <span property="familyName">Chadburn</span></a><div id="au4" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Amy</span> <span property="familyName">Chadburn</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Pathology and Laboratory Medicine, Weill Cornell Medicine, New York, NY 10065, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Chadburn/Amy" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">3</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au5" aria-controls="axel_author_au5"><span property="givenName">Leena</span> <span property="familyName">Babiker</span></a><div id="au5" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Leena</span> <span property="familyName">Babiker</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Division of Hematology/Oncology, Department of Medicine, Weill Cornell Medicine, Cornell University, New York, NY 10021, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Babiker/Leena" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au6" aria-controls="axel_author_au6"><span property="givenName">Daleum</span> <span property="familyName">Kim</span></a><div id="au6" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Daleum</span> <span property="familyName">Kim</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Division of Hematology/Oncology, Department of Medicine, Weill Cornell Medicine, Cornell University, New York, NY 10021, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Kim/Daleum" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au7" aria-controls="axel_author_au7"><span property="givenName">Wendy</span> <span property="familyName">Béguelin</span></a><div id="au7" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au7"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Wendy</span> <span property="familyName">Béguelin</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Division of Hematology/Oncology, Department of Medicine, Weill Cornell Medicine, Cornell University, New York, NY 10021, USA</span></div></div><div class="core-author-link"><a href="/authored-by/B%C3%A9guelin/Wendy" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au8" aria-controls="axel_author_au8"><span property="givenName">Tak C.</span> <span property="familyName">Lee</span></a><div id="au8" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au8"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Tak C.</span> <span property="familyName">Lee</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Division of Hematology/Oncology, Department of Medicine, Weill Cornell Medicine, Cornell University, New York, NY 10021, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Lee/Tak+C" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au9" aria-controls="axel_author_au9"><span property="givenName">Youngjun</span> <span property="familyName">Kim</span></a><div id="au9" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au9"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Youngjun</span> <span property="familyName">Kim</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Immunology and Microbial Pathogenesis Program, Weill Cornell Graduate School of Medical Sciences, New York, NY 10065, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Kim/Youngjun" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">4</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au10" aria-controls="axel_author_au10"><span property="givenName">Christopher R.</span> <span property="familyName">Chin</span></a><div id="au10" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au10"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Christopher R.</span> <span property="familyName">Chin</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Division of Hematology/Oncology, Department of Medicine, Weill Cornell Medicine, Cornell University, New York, NY 10021, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Tri-Institutional Program in Computational Biology and Medicine, New York, NY 10065, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Chin/Christopher+R" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">5</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au11" aria-controls="axel_author_au11"><span property="givenName">William T.</span> <span property="familyName">Yewdell</span></a><div id="au11" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au11"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">William T.</span> <span property="familyName">Yewdell</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Immunology Program, Memorial Sloan Kettering Cancer Center, New York, NY 10065, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Yewdell/William+T" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">6</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au12" aria-controls="axel_author_au12"><span property="givenName">Brian</span> <span property="familyName">Raught</span></a><div id="au12" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au12"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Brian</span> <span property="familyName">Raught</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Princess Margaret Cancer Centre, University of Toronto, Toronto, ON M5G 1L7, Canada</span></div></div><div class="core-author-link"><a href="/authored-by/Raught/Brian" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">7</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au13" aria-controls="axel_author_au13"><span property="givenName">Jude M.</span> <span property="familyName">Phillip</span></a><div id="au13" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au13"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Jude M.</span> <span property="familyName">Phillip</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Division of Hematology/Oncology, Department of Medicine, Weill Cornell Medicine, Cornell University, New York, NY 10021, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Phillip/Jude+M" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au14" aria-controls="axel_author_au14"><span property="givenName">Yanwen</span> <span property="familyName">Jiang</span></a><div id="au14" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au14"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Yanwen</span> <span property="familyName">Jiang</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Division of Hematology/Oncology, Department of Medicine, Weill Cornell Medicine, Cornell University, New York, NY 10021, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Jiang/Yanwen" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au15" aria-controls="axel_author_au15"><span property="givenName">Louis M.</span> <span property="familyName">Staudt</span></a><div id="au15" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au15"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Louis M.</span> <span property="familyName">Staudt</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Center for Cancer Genomics, National Cancer Institute, Bethesda, MD 20892, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Staudt/Louis+M" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">8</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au16" aria-controls="axel_author_au16"><span property="givenName">Michael R.</span> <span property="familyName">Green</span></a><div id="au16" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au16"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Michael R.</span> <span property="familyName">Green</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Lymphoma and Myeloma, University of Texas MD Anderson Cancer Center, Houston, TX 77030, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Genomic Medicine, University of Texas MD Anderson Cancer Center, Houston, TX 77030, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Green/Michael+R" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">9</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">10</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au17" aria-controls="axel_author_au17"><span property="givenName">Jayanta</span> <span property="familyName">Chaudhuri</span></a><div id="au17" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au17"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Jayanta</span> <span property="familyName">Chaudhuri</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Immunology and Microbial Pathogenesis Program, Weill Cornell Graduate School of Medical Sciences, New York, NY 10065, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Immunology Program, Memorial Sloan Kettering Cancer Center, New York, NY 10065, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Gerstner Sloan Kettering Graduate School of Biomedical Sciences, New York, NY 10065, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Chaudhuri/Jayanta" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">4</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">6</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">11</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au18" aria-controls="axel_author_au18"><span property="givenName">Olivier</span> <span property="familyName">Elemento</span></a><div id="au18" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au18"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Olivier</span> <span property="familyName">Elemento</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Caryl and Israel Englander Institute for Precision Medicine, Weill Cornell Medicine, New York, NY 10021, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Elemento/Olivier" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">12</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au19" aria-controls="axel_author_au19"><span property="givenName">Pedro</span> <span property="familyName">Farinha</span></a><div id="au19" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au19"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Pedro</span> <span property="familyName">Farinha</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Centre for Lymphoid Cancer, BC Cancer Agency, Vancouver, BC V5Z1L3, Canada</span></div></div><div class="core-author-link"><a href="/authored-by/Farinha/Pedro" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">13</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au20" aria-controls="axel_author_au20"><span property="givenName">Andrew P.</span> <span property="familyName">Weng</span></a><div id="au20" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au20"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Andrew P.</span> <span property="familyName">Weng</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Terry Fox Laboratory, BC Cancer Agency, Vancouver, BC V5Z1L3, Canada</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Pathology and Lab Medicine, BC Cancer Agency, Vancouver, BC V5Z1L3, Canada</span></div></div><div class="core-author-link"><a href="/authored-by/Weng/Andrew+P" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">14</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">15</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au21" aria-controls="axel_author_au21"><span property="givenName">Michael D.</span> <span property="familyName">Nissen</span></a><div id="au21" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au21"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Michael D.</span> <span property="familyName">Nissen</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Terry Fox Laboratory, BC Cancer Agency, Vancouver, BC V5Z1L3, Canada</span></div></div><div class="core-author-link"><a href="/authored-by/Nissen/Michael+D" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">14</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au22" aria-controls="axel_author_au22"><span property="givenName">Christian</span> <span property="familyName">Steidl</span></a><div id="au22" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au22"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Christian</span> <span property="familyName">Steidl</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Centre for Lymphoid Cancer, BC Cancer Agency, Vancouver, BC V5Z1L3, Canada</span></div></div><div class="core-author-link"><a href="/authored-by/Steidl/Christian" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">13</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au23" aria-controls="axel_author_au23"><span property="givenName">Ryan D.</span> <span property="familyName">Morin</span></a><div id="au23" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au23"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Ryan D.</span> <span property="familyName">Morin</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Centre for Lymphoid Cancer, BC Cancer Agency, Vancouver, BC V5Z1L3, Canada</span></div></div><div class="core-author-link"><a href="/authored-by/Morin/Ryan+D" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">13</a></sup></span></span> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au24" aria-controls="axel_author_au24"><span property="givenName">David W.</span> <span property="familyName">Scott</span></a><div id="au24" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au24"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">David W.</span> <span property="familyName">Scott</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Centre for Lymphoid Cancer, BC Cancer Agency, Vancouver, BC V5Z1L3, Canada</span></div></div><div class="core-author-link"><a href="/authored-by/Scott/David+W" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">13</a></sup></span> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au25" aria-controls="axel_author_au25"><span property="givenName">Gilbert G.</span> <span property="familyName">Privé</span></a><div id="au25" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au25"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Gilbert G.</span> <span property="familyName">Privé</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Princess Margaret Cancer Centre, University of Toronto, Toronto, ON M5G 1L7, Canada</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Medical Biophysics, University of Toronto, and Princess Margaret Cancer Centre, Toronto, ON M5S 1A8, Canada</span></div></div><div class="core-author-link"><a href="/authored-by/Priv%C3%A9/Gilbert+G" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">7</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">16</a></sup></span> ∙ <span property="author" typeof="Person" role="listitem" class="corresponding-author"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au26" aria-controls="axel_author_au26"><span property="givenName">Ari M.</span> <span property="familyName">Melnick</span></a><div id="au26" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au26"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Ari M.</span> <span property="familyName">Melnick</span> <a class="orcid-id" href="https://orcid.org/0000-0002-8074-2287" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0002-8074-2287</a></div><div class="content"><div class="core-author-emails"><div class="label">Correspondence</div><div>Corresponding author</div><div><a id="eadd0010" href="mailto:amm2014@med.cornell.edu" property="email" title="Send email to Ari M. Melnick">amm2014@med.cornell.edu</a></div></div><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">17</div><div id="fn1"><div id="ntpara0010" role="paragraph">Lead Contact</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Division of Hematology/Oncology, Department of Medicine, Weill Cornell Medicine, Cornell University, New York, NY 10021, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Melnick/Ari+M" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a role="doc-noteref">17</a></sup> <a id="eadd0010" href="mailto:amm2014@med.cornell.edu" property="email" title="Send email to Ari M. Melnick">amm2014@med.cornell.edu</a></span><span data-displayed-on="all"> … <span><span data-action="reveal" tabindex="0" role="listitem">Show more</span></span> </span><span data-displayed-on="none" data-on-display="all" aria-hidden="true"> <span data-action="hide" tabindex="0" role="listitem">Show less</span></span></span></span></div><div class="core-self-citation"><div class="expandable-triggers"><button aria-controls="core-affiliations-notes" aria-expanded="false" tabindex="0" role="button"><span>Affiliations &amp; Notes</span><i class="icon-arrow_d_n" aria-hidden="true"></i></button><button aria-controls="core-content-info" aria-expanded="false" tabindex="0" role="button"><span>Article Info</span><i class="icon-arrow_d_n" aria-hidden="true"></i></button></div><div id="core-linked-content">
    

        

</div><div class="expandable-content"><div id="core-affiliations-notes" role="region" data-expandable="item" style="display: none;"><div class="affiliations"><div property="affiliation" typeof="Organization"><span class="label">1</span><span id="aff1" property="name">Division of Hematology/Oncology, Department of Medicine, Weill Cornell Medicine, Cornell University, New York, NY 10021, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">2</span><span id="aff2" property="name">Department of Biochemistry, University of Toronto, Toronto, ON M5S 1A8, Canada</span></div><div property="affiliation" typeof="Organization"><span class="label">3</span><span id="aff3" property="name">Department of Pathology and Laboratory Medicine, Weill Cornell Medicine, New York, NY 10065, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">4</span><span id="aff4" property="name">Immunology and Microbial Pathogenesis Program, Weill Cornell Graduate School of Medical Sciences, New York, NY 10065, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">5</span><span id="aff5" property="name">Tri-Institutional Program in Computational Biology and Medicine, New York, NY 10065, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">6</span><span id="aff6" property="name">Immunology Program, Memorial Sloan Kettering Cancer Center, New York, NY 10065, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">7</span><span id="aff7" property="name">Princess Margaret Cancer Centre, University of Toronto, Toronto, ON M5G 1L7, Canada</span></div><div property="affiliation" typeof="Organization"><span class="label">8</span><span id="aff8" property="name">Center for Cancer Genomics, National Cancer Institute, Bethesda, MD 20892, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">9</span><span id="aff9" property="name">Department of Lymphoma and Myeloma, University of Texas MD Anderson Cancer Center, Houston, TX 77030, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">10</span><span id="aff10" property="name">Department of Genomic Medicine, University of Texas MD Anderson Cancer Center, Houston, TX 77030, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">11</span><span id="aff11" property="name">Gerstner Sloan Kettering Graduate School of Biomedical Sciences, New York, NY 10065, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">12</span><span id="aff12" property="name">Caryl and Israel Englander Institute for Precision Medicine, Weill Cornell Medicine, New York, NY 10021, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">13</span><span id="aff13" property="name">Centre for Lymphoid Cancer, BC Cancer Agency, Vancouver, BC V5Z1L3, Canada</span></div><div property="affiliation" typeof="Organization"><span class="label">14</span><span id="aff14" property="name">Terry Fox Laboratory, BC Cancer Agency, Vancouver, BC V5Z1L3, Canada</span></div><div property="affiliation" typeof="Organization"><span class="label">15</span><span id="aff15" property="name">Department of Pathology and Lab Medicine, BC Cancer Agency, Vancouver, BC V5Z1L3, Canada</span></div><div property="affiliation" typeof="Organization"><span class="label">16</span><span id="aff16" property="name">Department of Medical Biophysics, University of Toronto, and Princess Margaret Cancer Centre, Toronto, ON M5S 1A8, Canada</span></div></div><div class="core-authors-notes"><div role="doc-footnote" data-has="label"><div class="label">17</div><div id="fn1"><div id="ntpara0010" role="paragraph">Lead Contact</div></div></div></div></div><div id="core-content-info" role="region" data-expandable="item" style="display: none;"><div class="history"><div>Publication History:</div><div><span>Received October 28, 2019</span>; <span>Revised March 24, 2020</span>; <span>Accepted May 27, 2020</span>; <span>Published online July 2, 2020</span></div></div><div class="core-links"><span class="doi">DOI: <a href="https://doi.org/10.1016/j.cell.2020.05.049" property="sameAs" target="_blank">10.1016/j.cell.2020.05.049 <span class="sr-only">External Link</span></a></span><span class="science-direct">Also available on <a href="https://www.sciencedirect.com/science/article/pii/S0092867420306826" target="_blank">ScienceDirect <span class="sr-only">External Link</span></a></span></div><div role="paragraph">Copyright: © 2020 Elsevier Inc.</div><div class="core-links"><span>User License: <a href="http://www.elsevier.com/open-access/userlicense/1.0/" target="_blank">Elsevier user license</a> | <a href="https://www.elsevier.com/open-access" target="_blank">Elsevier's open access license policy</a></span></div><div class="core-crossmark-badge"><a data-target="crossmark" href="#" title="Check for updates on crossmark" data-doi="10.1016/j.cell.2020.05.049" data-id="article-info-crossmark" class="crossmark__link" style="cursor: pointer;"><svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 290 58" height="22px" role="presentation"><path fill="url(#horizontal_crossmark)" stroke="#948F8F" stroke-miterlimit="10" stroke-width=".75" d="M287 1H3a2 2 0 0 0-2 2v52a2 2 0 0 0 2 2h284a2 2 0 0 0 2-2V3a2 2 0 0 0-2-2Z"></path><path fill="#535353" d="M80.3 23.58a6.395 6.395 0 0 0-.55-1.43 3.92 3.92 0 0 0-2.19-1.86 5.099 5.099 0 0 0-1.71-.29 5 5 0 0 0-2.46.58 4.831 4.831 0 0 0-1.69 1.54 6.921 6.921 0 0 0-1 2.19 10.16 10.16 0 0 0 0 5 6.921 6.921 0 0 0 1 2.19 4.83 4.83 0 0 0 1.69 1.5 5 5 0 0 0 2.46.58 4.44 4.44 0 0 0 1.85-.36 4.12 4.12 0 0 0 1.38-1c.39-.436.695-.942.9-1.49A7.094 7.094 0 0 0 80.4 29h3a8.46 8.46 0 0 1-.69 2.92 7.179 7.179 0 0 1-1.59 2.29 7 7 0 0 1-2.35 1.49 8.14 8.14 0 0 1-3 .52 8.43 8.43 0 0 1-3.59-.74 7.88 7.88 0 0 1-2.69-2 9.001 9.001 0 0 1-1.69-3 11.41 11.41 0 0 1 0-7.23 9.001 9.001 0 0 1 1.69-3 8.001 8.001 0 0 1 2.69-2 8.3 8.3 0 0 1 3.59-.75 9.44 9.44 0 0 1 2.83.41c.866.242 1.68.642 2.4 1.18A6.229 6.229 0 0 1 82.65 21a6.66 6.66 0 0 1 .78 2.58H80.3ZM85.9 17.9h2.85v6.63h.05a4.15 4.15 0 0 1 1.56-1.46 4.61 4.61 0 0 1 2.31-.59 4.83 4.83 0 0 1 3.33 1.1 4.21 4.21 0 0 1 1.22 3.3v8.88H94.4v-8.13a3.51 3.51 0 0 0-.65-2.21 2.36 2.36 0 0 0-1.87-.69 3.1 3.1 0 0 0-1.3.26 2.821 2.821 0 0 0-1 .73A3.41 3.41 0 0 0 89 26.8a3.88 3.88 0 0 0-.23 1.33v7.63H85.9V17.9ZM102.47 30c-.002.491.069.98.21 1.45.131.448.348.866.64 1.23a3.08 3.08 0 0 0 1.08.84 3.543 3.543 0 0 0 1.55.31 3.431 3.431 0 0 0 2-.54c.541-.402.94-.966 1.14-1.61h2.7A5.349 5.349 0 0 1 109.7 35a5.715 5.715 0 0 1-1.74.85 7 7 0 0 1-2 .29 6.848 6.848 0 0 1-2.7-.5 5.577 5.577 0 0 1-2-1.4A5.953 5.953 0 0 1 100 32a8.748 8.748 0 0 1-.41-2.75 7.768 7.768 0 0 1 .44-2.61 6.67 6.67 0 0 1 1.25-2.17 5.863 5.863 0 0 1 4.56-2 5.79 5.79 0 0 1 2.74.64c.783.41 1.466.986 2 1.69a6.824 6.824 0 0 1 1.16 2.41c.264.905.352 1.852.26 2.79h-9.53Zm6.65-1.87a4.605 4.605 0 0 0-.29-1.3c-.148-.4-.368-.77-.65-1.09a3.22 3.22 0 0 0-1-.75 3 3 0 0 0-1.31-.29 3.431 3.431 0 0 0-1.36.26 3.108 3.108 0 0 0-1 .73 3.594 3.594 0 0 0-.7 1.09 3.798 3.798 0 0 0-.29 1.35h6.65-.05ZM122.88 27.15a2.666 2.666 0 0 0-1-1.81 3.14 3.14 0 0 0-2-.61c-.4.005-.798.07-1.18.19-.435.136-.83.376-1.15.7a3.92 3.92 0 0 0-.87 1.43 6.783 6.783 0 0 0-.35 2.39c.002.539.065 1.076.19 1.6.113.509.316.993.6 1.43.273.41.635.752 1.06 1a3 3 0 0 0 1.58.39 2.872 2.872 0 0 0 2.06-.77 3.596 3.596 0 0 0 1-2.18h2.85a6.26 6.26 0 0 1-1.91 3.86 5.83 5.83 0 0 1-4 1.34 6.61 6.61 0 0 1-2.69-.51 5.695 5.695 0 0 1-2-1.4 6.001 6.001 0 0 1-1.21-2.11 8.279 8.279 0 0 1-.41-2.65 8.987 8.987 0 0 1 .4-2.72 6.237 6.237 0 0 1 1.2-2.21 5.614 5.614 0 0 1 2-1.47 6.7 6.7 0 0 1 2.79-.54 7.801 7.801 0 0 1 2.14.29 5.634 5.634 0 0 1 1.8.86c.529.386.969.88 1.29 1.45a5 5 0 0 1 .6 2.07h-2.85l.06-.02ZM128.13 17.9H131V28l5.15-5.22h3.5l-5 4.75 5.42 8.17h-3.47l-4-6.27-1.6 1.7v4.63h-2.85V17.9h-.02ZM147.15 22.83h2.13v-1.08a5.505 5.505 0 0 1 .3-2c.161-.456.436-.862.8-1.18.33-.273.721-.462 1.14-.55.44-.094.89-.141 1.34-.14a8.158 8.158 0 0 1 2 .18v2.24a4.113 4.113 0 0 0-.59-.11 6.343 6.343 0 0 0-.74 0 1.74 1.74 0 0 0-1 .27 1.204 1.204 0 0 0-.41 1.08v1.33h2.42V25h-2.42v10.8h-2.85V25h-2.12v-2.17ZM162.23 36.1a6.999 6.999 0 0 1-2.76-.51 5.877 5.877 0 0 1-2-1.41 5.998 5.998 0 0 1-1.33-2.18 8.775 8.775 0 0 1 0-5.48 6.008 6.008 0 0 1 1.27-2.15 5.894 5.894 0 0 1 2-1.41A7.71 7.71 0 0 1 165 23a5.894 5.894 0 0 1 2 1.41 6.003 6.003 0 0 1 1.28 2.15c.585 1.78.585 3.7 0 5.48a5.99 5.99 0 0 1-3.28 3.54 7.002 7.002 0 0 1-2.77.52Zm0-2.25a3.268 3.268 0 0 0 1.65-.4c.447-.262.83-.621 1.12-1.05.304-.444.528-.938.66-1.46a6.581 6.581 0 0 0 0-3.31 4.416 4.416 0 0 0-.66-1.46 3.526 3.526 0 0 0-1.15-1 3.6 3.6 0 0 0-3.3 0 3.526 3.526 0 0 0-1.15 1 4.414 4.414 0 0 0-.66 1.46 6.558 6.558 0 0 0 0 3.31c.132.522.356 1.016.66 1.46a3.47 3.47 0 0 0 1.15 1.05 3.26 3.26 0 0 0 1.68.4ZM171.13 22.83h2.67v2.5c.097-.362.264-.701.49-1 .251-.341.547-.647.88-.91a4.404 4.404 0 0 1 1.14-.66 3.41 3.41 0 0 1 1.28-.25c.23-.013.46-.013.69 0h.39v2.79l-.61-.09a5.173 5.173 0 0 0-.61 0 3.278 3.278 0 0 0-1.36.29 3.131 3.131 0 0 0-1.11.85 4.171 4.171 0 0 0-.75 1.39 6 6 0 0 0-.28 1.9v6.15h-2.85V22.83h.03ZM198.22 35.75h-2.8V34a3.907 3.907 0 0 1-1.56 1.56 4.25 4.25 0 0 1-2.11.59 4.742 4.742 0 0 1-3.75-1.31 5.537 5.537 0 0 1-1.12-3.84v-8.2h2.85v7.92a3.503 3.503 0 0 0 .65 2.4 2.38 2.38 0 0 0 1.83.7 3.599 3.599 0 0 0 1.5-.27 2.66 2.66 0 0 0 1-.74c.259-.326.444-.705.54-1.11a5.79 5.79 0 0 0 .16-1.4v-7.5h2.85v12.95h-.04ZM201.3 22.83h2.7v1.75a3.523 3.523 0 0 1 1.72-1.58 5.568 5.568 0 0 1 2.33-.49 6.17 6.17 0 0 1 2.66.54 5.246 5.246 0 0 1 1.89 1.47c.515.65.899 1.394 1.13 2.19.257.87.385 1.773.38 2.68.001.851-.114 1.7-.34 2.52a6.568 6.568 0 0 1-1 2.16 5.002 5.002 0 0 1-4.21 2.06c-.419 0-.837-.036-1.25-.11a5.39 5.39 0 0 1-1.2-.36A4.737 4.737 0 0 1 205 35a3.567 3.567 0 0 1-.81-.91v6.45h-2.89V22.83Zm10 6.47c0-.574-.078-1.146-.23-1.7a4.606 4.606 0 0 0-.67-1.46 3.475 3.475 0 0 0-1.12-1 3.06 3.06 0 0 0-1.55-.39A3.141 3.141 0 0 0 205 26a5.518 5.518 0 0 0-1 3.3c-.004.611.077 1.22.24 1.81.138.523.38 1.012.71 1.44a3.55 3.55 0 0 0 1.14 1 3.242 3.242 0 0 0 1.54.35 3.18 3.18 0 0 0 1.65-.4c.438-.25.817-.59 1.11-1 .299-.44.513-.931.63-1.45a7.38 7.38 0 0 0 .23-1.75h.05ZM228.67 35.75H226V34a3.38 3.38 0 0 1-1.67 1.61 5.668 5.668 0 0 1-2.33.49 6.17 6.17 0 0 1-2.66-.54 5.307 5.307 0 0 1-1.89-1.46 6.254 6.254 0 0 1-1.12-2.19 9.48 9.48 0 0 1-.37-2.71 8.67 8.67 0 0 1 .47-3 6.058 6.058 0 0 1 1.26-2.1 4.89 4.89 0 0 1 1.8-1.21 5.738 5.738 0 0 1 2.06-.39c.412 0 .824.037 1.23.11.413.073.815.194 1.2.36.38.166.736.381 1.06.64.32.255.594.562.81.91V17.9h2.85v17.85h-.03Zm-10-6.33c-.001.55.069 1.098.21 1.63.13.511.35.996.65 1.43.288.414.668.757 1.11 1a3.241 3.241 0 0 0 1.6.38 3.199 3.199 0 0 0 1.64-.4 3.398 3.398 0 0 0 1.13-1.05c.297-.447.517-.94.65-1.46.14-.542.211-1.1.21-1.66a5.227 5.227 0 0 0-1-3.35 3.175 3.175 0 0 0-2.61-1.2 3.23 3.23 0 0 0-1.69.41 3.478 3.478 0 0 0-1.12 1.07 4.494 4.494 0 0 0-.62 1.5 7.871 7.871 0 0 0-.11 1.7h-.05ZM242.6 32.88c-.018.258.03.516.14.75a.577.577 0 0 0 .54.23h.3c.135-.002.269-.019.4-.05v2l-.39.11-.49.08-.5.08h-.42a2.75 2.75 0 0 1-1.45-.35 1.731 1.731 0 0 1-.75-1.23 4.998 4.998 0 0 1-2.09 1.2 8.224 8.224 0 0 1-2.39.38 5.861 5.861 0 0 1-1.67-.24 4.394 4.394 0 0 1-1.41-.7 3.35 3.35 0 0 1-1-1.18 3.624 3.624 0 0 1-.36-1.66 3.84 3.84 0 0 1 .44-2 3.246 3.246 0 0 1 1.15-1.17 5.195 5.195 0 0 1 1.6-.61c.593-.127 1.19-.223 1.79-.29a14.77 14.77 0 0 1 1.47-.21 6.896 6.896 0 0 0 1.24-.21c.318-.077.611-.236.85-.46a1.286 1.286 0 0 0 .31-.94 1.458 1.458 0 0 0-.26-.9A1.759 1.759 0 0 0 239 25a2.807 2.807 0 0 0-.86-.25 6.941 6.941 0 0 0-.9-.06 3.58 3.58 0 0 0-2 .5 2 2 0 0 0-.87 1.55h-2.85c.021-.73.227-1.442.6-2.07.338-.54.797-.996 1.34-1.33a5.475 5.475 0 0 1 1.84-.7c.692-.134 1.395-.2 2.1-.2.632 0 1.262.067 1.88.2a5.337 5.337 0 0 1 1.66.65 3.6 3.6 0 0 1 1.19 1.16c.314.525.47 1.129.45 1.74v6.65l.02.04Zm-2.85-3.6a3.45 3.45 0 0 1-1.6.51c-.667.06-1.3.147-1.9.26a6.137 6.137 0 0 0-.87.21 2.64 2.64 0 0 0-.75.38c-.22.164-.395.38-.51.63a2.129 2.129 0 0 0-.19.94c-.011.29.085.575.27.8.182.215.407.388.66.51.269.128.555.216.85.26.277.05.558.076.84.08a4.63 4.63 0 0 0 1-.12 3.418 3.418 0 0 0 1-.43 2.73 2.73 0 0 0 .81-.76 1.93 1.93 0 0 0 .32-1.14v-2.14l.07.01ZM244.37 22.83h2.15V19h2.85v3.88h2.58V25h-2.58v6.9a7.064 7.064 0 0 0 0 .77c.016.194.074.381.17.55a.84.84 0 0 0 .41.34 2 2 0 0 0 .75.11h.6c.203-.01.404-.04.6-.09v2.2l-.93.1a8.403 8.403 0 0 1-.92 0 6.33 6.33 0 0 1-1.81-.21 2.421 2.421 0 0 1-1.08-.62 2.15 2.15 0 0 1-.52-1 7.791 7.791 0 0 1-.16-1.42V25h-2.15v-2.17h.04ZM256.22 30c-.002.491.069.98.21 1.45.131.448.348.866.64 1.23a3.08 3.08 0 0 0 1.08.84 3.543 3.543 0 0 0 1.55.31 3.431 3.431 0 0 0 2-.54c.541-.402.94-.966 1.14-1.61h2.7a5.352 5.352 0 0 1-2.1 3.26 5.715 5.715 0 0 1-1.74.85 7 7 0 0 1-2 .29 6.848 6.848 0 0 1-2.7-.5 5.577 5.577 0 0 1-2-1.4 5.947 5.947 0 0 1-1.21-2.18 8.754 8.754 0 0 1-.41-2.75 7.758 7.758 0 0 1 .44-2.61 6.67 6.67 0 0 1 1.25-2.17 5.863 5.863 0 0 1 4.56-2 5.79 5.79 0 0 1 2.74.64c.783.41 1.466.986 2 1.69a6.824 6.824 0 0 1 1.16 2.41c.243.916.308 1.87.19 2.81h-9.5V30Zm6.65-1.87a4.605 4.605 0 0 0-.29-1.3c-.148-.4-.368-.77-.65-1.09a3.22 3.22 0 0 0-1-.75 3 3 0 0 0-1.31-.29 3.431 3.431 0 0 0-1.36.26 3.108 3.108 0 0 0-1 .73 3.575 3.575 0 0 0-.7 1.09 3.798 3.798 0 0 0-.29 1.35h6.65-.05ZM270 31.6a2.154 2.154 0 0 0 .95 1.75c.607.35 1.3.524 2 .5.304-.002.608-.022.91-.06a3.943 3.943 0 0 0 1-.24c.286-.105.54-.28.74-.51a1.23 1.23 0 0 0 .26-.89 1.261 1.261 0 0 0-.4-.9 2.779 2.779 0 0 0-1-.56 9.315 9.315 0 0 0-1.34-.36l-1.5-.33a14.219 14.219 0 0 1-1.54-.42 4.937 4.937 0 0 1-1.33-.68 3.12 3.12 0 0 1-.94-1.09 3.472 3.472 0 0 1-.35-1.64 2.94 2.94 0 0 1 .51-1.76 3.913 3.913 0 0 1 1.3-1.15 5.66 5.66 0 0 1 1.75-.61 10.316 10.316 0 0 1 1.84-.17 8.418 8.418 0 0 1 1.91.21 5.19 5.19 0 0 1 1.65.69c.496.318.916.741 1.23 1.24.341.559.55 1.188.61 1.84h-3a1.783 1.783 0 0 0-.94-1.37 3.892 3.892 0 0 0-1.69-.35 6.093 6.093 0 0 0-.71 0 3.327 3.327 0 0 0-.78.19 1.642 1.642 0 0 0-.61.4 1 1 0 0 0-.25.69 1.093 1.093 0 0 0 .36.85c.294.245.634.428 1 .54.437.153.885.274 1.34.36l1.55.33a20.9 20.9 0 0 1 1.53.42c.479.157.931.386 1.34.68.41.277.751.643 1 1.07.255.494.379 1.045.36 1.6a3.48 3.48 0 0 1-.52 2 4.15 4.15 0 0 1-1.36 1.3 6.19 6.19 0 0 1-1.86.73 9.435 9.435 0 0 1-2 .23 8.823 8.823 0 0 1-2.26-.27 5.265 5.265 0 0 1-1.8-.84 4 4 0 0 1-1.2-1.4 4.445 4.445 0 0 1-.46-2h2.7v-.02Z"></path><path fill="#C72914" d="m21.94 38.74 18.12-12.08v-14.5H21.94v26.58Z"></path><path fill="#EF3340" d="M40.06 38.74 21.94 26.66v-14.5h18.12v26.58Z"></path><path fill="#3EB1C8" d="M31 9a20 20 0 1 0 0 40 20 20 0 0 0 0-40Zm0 35.1a15.1 15.1 0 1 1 0-30.2 15.1 15.1 0 0 1 0 30.2Z"></path><path fill="#FFC72C" d="M42.09 39.23A15.09 15.09 0 0 1 17 23.38l-4.2-2.64a20 20 0 0 0 33.49 21.14l-4.2-2.65Z"></path><defs><linearGradient id="horizontal_crossmark" x1="145" x2="145" y1="7.33" y2="56.82" gradientUnits="userSpaceOnUse"><stop stop-color="#fff"></stop><stop offset="1" stop-color="#c4c4c4"></stop></linearGradient></defs></svg></a></div><div class="core-content-info"></div></div></div></div></div></header><div class="core-header-aside">
    

        


    

            

            

            <div class="cover-image cover-image--axel"><div class="cover-image__thumb"><a href="https://www.cell.com/cell/issue?pii=S0092-8674(19)X0016-1"><img src="/cms/asset/b75635b3-5305-4e07-a55d-4bc3d41a161f/cov200h.gif" alt="Cover Image - Cell, Volume 182, Issue 2"></a></div></div>

            

        
</div></div><div data-core-wrapper="content"><div id="dummy-sticky-observer"></div><div data-core-nav="header" data-extent="frontmatter"><div class="core-nav-wrapper core-container" aria-label="Article navigation"><div class="navigation-menu-pdf">
    

        


    

            

            

            <div data-widget-id="ecff07e8-0bc4-4533-b1f1-49365e2ead20" data-widget-name="UX3ArticleToolsWidget" class="article-tools"><a href="/action/showPdf?pii=S0092-8674%2820%2930682-6" title="Standard PDF (opens in a new tab)" target="_blank" aria-hidden="false" role="menuitem" class="article-tools__item__pdf"><svg focusable="false" viewBox="0 0 32 32" width="20" height="20" title="PDF" class="pdf-icon"><path id="pdfwhite" d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path id="pdfda0000" d="M.167 2.592H22.39V9.72H.166z" fill="#da0000"></path><path id="pdffff9f9" fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path id="pdff91d0a" d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="basic">Download PDF</span><span class="alternative">Download PDF</span></a></div>

            

        
</div><div class="core-sections-menu"><button data-id="article-toolbar-showhide" aria-controls="article_sections_menu" aria-expanded="false" aria-haspopup="true" aria-label="Toggle section navigation menu" style="margin-top: 0px;"><i class="icon-list" aria-hidden="true"></i><span>Outline</span></button><nav id="article_sections_menu" aria-label="Contents" data-core-nav="article" style="margin-top: 0px;"><div class="heading">Outline<button aria-haspopup="true" aria-controls="article_sections_menu" title="Close outline menu"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div><ul><li><a href="#author-highlights-abstract">Highlights</a></li><li><a href="#author-abstract">Summary</a></li><li><a href="#graphical-abstract">Graphical Abstract</a></li><li><a href="#keywords">Keywords</a></li><li><a href="#sec-1">Introduction</a></li><li><a href="#sec-2">Results</a></li><li><a href="#sec-3">Discussion</a></li><li><a href="#sec-4">STAR★Methods</a></li><li><a href="#acknowledgments">Acknowledgments</a></li><li><a href="#supplementary-material">Supplemental Information (6)</a></li><li><a href="#references">References</a></li><li role="none" class="mobile-only"><a role="menuitem" href="#core-collateral-metrics">Article metrics</a></li><li role="none" class="mobile-only"><a role="menuitem" href="#core-collateral-relatedArticles">Related Articles</a></li></ul></nav></div><div class="navigation-menu-share">
    

        


    

            

            

            <!-- Go to https://www.addtoany.com/buttons/customize/ to customize your tools --><script type="text/javascript" defer="defer" src="https://static.addtoany.com/menu/page.js"></script><script type="text/javascript">var a2a_config = a2a_config || {};
a2a_config.thanks = {
    postShare: false,
    ad: false,
};</script><script>var a2a_config = a2a_config || {};
let res = window.location.href.replace(/\(/g, "%28").replace(/\)/g, "%29");
a2a_config.templates = a2a_config.templates || {};

a2a_config.templates.email = {
    body: res
};
</script><div data-widget-id="5f978446-c80a-450c-a185-94d633df2db2" data-widget-name="UX3share" aria-hidden="false" role="group" class="article-tools__dropblock article-tools__share"><button aria-label="Toggle share menu" aria-expanded="false" aria-haspopup="true" role="menuitem" aria-controls="navigation_menu_share_block" aria-labelledby="share-label" class="article-tools__item__share"><i aria-hidden="true" class="icon-article-share"></i><span id="share-label">Share</span></button><div id="navigation_menu_share_block" role="menu" class="share__block axel-share article-tools__dropblock__holder"><h4 class="axel-share__title heading">Share<button title="Close share menu" aria-expanded="false" aria-haspopup="true" aria-controls="navigation_menu_share_block"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></h4><div class="axel-share__inner"><span class="sr-only">Share on</span><ul class="a2a__share-list rlist w-slide--list a2a a2a_kit a2a_kit_size_32 a2a_default_style" data-a2a-url="https://www.cell.com/cell/fulltext/S0092-8674%2820%2930682-6" style="line-height: 32px;"><li class="a2a_listitem_custom"><a role="link" title="Email" class="a2a_button_email" target="_blank" rel="nofollow noopener" href="/#email"><i aria-hidden="true" class="at-icon-wrapper icon-gizmo-email-plus"></i><span>Email</span></a></li><li class="a2a_listitem_custom"><a role="link" title="formerly known as Twitter" class="a2a_button_twitter" target="_blank" rel="nofollow noopener" href="/#twitter"><i aria-hidden="true" class="at-icon-wrapper icon-x-default"></i><span>X</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Facebook" class="a2a_button_facebook" target="_blank" rel="nofollow noopener" href="/#facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook-square-2"></i><span>Facebook</span></a></li><li class="a2a_listitem_custom"><a role="link" title="LinkedIn" class="a2a_button_linkedin" target="_blank" rel="nofollow noopener" href="/#linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin-square"></i><span>LinkedIn</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Sina Weibo" class="a2a_button_sina_weibo" target="_blank" rel="nofollow noopener" href="/#sina_weibo"><i aria-hidden="true" class="at-icon-wrapper icon-weibo"></i><span>Sina Weibo</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Mendeley" class="a2a_button_mendeley" target="_blank" rel="nofollow noopener" href="/#mendeley"><i aria-hidden="true" class="at-icon-wrapper icon-mendeley-2"></i><span>Mendeley</span></a></li><li class="a2a_listitem_custom"><a role="link" title="bluesky" class="a2a_button_bluesky" target="_blank" rel="nofollow noopener" href="/#bluesky"><i aria-hidden="true" class="at-icon-wrapper icon-bluesky-logo"></i><span>bluesky</span></a></li><div style="clear: both;"></div></ul><div class="extra-links-drop-zone">
    

        


    

            

            

            <div data-widget-id="f1a208d3-ac5e-4db2-9c3a-40ef97ddfa10" data-widget-name="UX3HTMLWidget"><a class="axel-share__extra" target="_blank" href="/personalize/addFavoritePublication?doi=10.1016/j.cell.2020.05.049">
	<i class="icon-reading-list" aria-hidden="true"></i>
	<span>Add to my reading list</span>
</a></div>

            

        
</div></div></div></div>

            

        
</div><div><button data-id="article-nav-menubar-more" aria-controls="article_more_menu" aria-expanded="false" aria-haspopup="true" aria-label="Toggle more menu"><i aria-hidden="true" class="icon-more-mobile"></i><span>More</span></button><div class="info-panel"><div class="info-panel__left-content"><div class="info-panel__article_tools info-panel__item"><div id="article_more_menu" aria-label="More menu" data-core-nav="more"><div class="heading">More<button title="Close more menu" aria-controls="article_more_menu" aria-haspopup="true"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div>
    

        


    

            

            

            <ul data-widget-id="183a17a2-87b5-4e6f-bb85-6b5cda068c89" data-widget-name="UX3ArticleToolsWidget" class="article-tools"><li class="article-tools__item article-tools__pdf"><a href="/action/showPdf?pii=S0092-8674%2820%2930682-6" title="Standard PDF (opens in a new tab)" target="_blank" aria-hidden="false" class="article-tools__ctrl article-tools__item__pdf"><svg focusable="false" viewBox="0 0 32 32" width="20" height="20" title="PDF" class="pdf-icon"><path id="pdfwhite" d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path id="pdfda0000" d="M.167 2.592H22.39V9.72H.166z" fill="#da0000"></path><path id="pdffff9f9" fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path id="pdff91d0a" d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="basic">Download PDF</span><span class="alternative">Download PDF</span></a></li><li><a href="https://www.cell.com/action/showCitFormats?doi=10.1016%2Fj.cell.2020.05.049&amp;pii=S0092-8674%2820%2930682-6" title="Cite" target="_blank" aria-label="Open Cite in new tab" class="article-tools__item__displayExportCitation"><i aria-hidden="true" class="icon-export-Citation"></i><span>Cite</span></a></li><li id="article_tools_share">
    

        


    

            

            

            <!-- Go to https://www.addtoany.com/buttons/customize/ to customize your tools --><script type="text/javascript" defer="defer" src="https://static.addtoany.com/menu/page.js"></script><script type="text/javascript">var a2a_config = a2a_config || {};
a2a_config.thanks = {
    postShare: false,
    ad: false,
};</script><script>var a2a_config = a2a_config || {};
let res = window.location.href.replace(/\(/g, "%28").replace(/\)/g, "%29");
a2a_config.templates = a2a_config.templates || {};

a2a_config.templates.email = {
    body: res
};
</script><div data-widget-id="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da" data-widget-name="UX3share" class="dropBlock article-tools__dropblock article-tools__share"><button data-db-target-for="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da" aria-expanded="false" title="open Share menu" class="article-tools__ctrl share__ctrl"><i aria-hidden="true" class="icon-article-share"></i><span>Share</span></button><div data-db-target-of="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da" id="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da_Pop" class="share__block axel-share dropBlock__holder article-tools__dropblock__holder"><h4 class="axel-share__title">Share</h4><div class="axel-share__inner"><span class="sr-only">Share on</span><ul class="a2a__share-list rlist w-slide--list a2a a2a_kit a2a_kit_size_32 a2a_default_style" data-a2a-url="https://www.cell.com/cell/fulltext/S0092-8674%2820%2930682-6" style="line-height: 32px;"><li class="a2a_listitem_custom"><a role="link" title="Email" class="a2a_button_email" target="_blank" rel="nofollow noopener" href="/#email"><i aria-hidden="true" class="at-icon-wrapper icon-gizmo-email-plus"></i><span>Email</span></a></li><li class="a2a_listitem_custom"><a role="link" title="formerly known as Twitter" class="a2a_button_twitter" target="_blank" rel="nofollow noopener" href="/#twitter"><i aria-hidden="true" class="at-icon-wrapper icon-x-default"></i><span>X</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Facebook" class="a2a_button_facebook" target="_blank" rel="nofollow noopener" href="/#facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook-square-2"></i><span>Facebook</span></a></li><li class="a2a_listitem_custom"><a role="link" title="LinkedIn" class="a2a_button_linkedin" target="_blank" rel="nofollow noopener" href="/#linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin-square"></i><span>LinkedIn</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Sina Weibo" class="a2a_button_sina_weibo" target="_blank" rel="nofollow noopener" href="/#sina_weibo"><i aria-hidden="true" class="at-icon-wrapper icon-weibo"></i><span>Sina Weibo</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Mendeley" class="a2a_button_mendeley" target="_blank" rel="nofollow noopener" href="/#mendeley"><i aria-hidden="true" class="at-icon-wrapper icon-mendeley-2"></i><span>Mendeley</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Bluesky" class="a2a_button_bluesky" target="_blank" rel="nofollow noopener" href="/#bluesky"><i aria-hidden="true" class="at-icon-wrapper icon-bluesky-logo"></i><span>Bluesky</span></a></li><div style="clear: both;"></div></ul><div class="extra-links-drop-zone">
    

        


    

            

            

            <div data-widget-id="ae9bf5c5-a2ee-4f5c-8e41-9b21638b1a9f" data-widget-name="UX3HTMLWidget"><a class="axel-share__extra" target="_blank" href="/personalize/addFavoritePublication?doi=10.1016/j.cell.2020.05.049">
	<i class="icon-reading-list" aria-hidden="true"></i>
	<span>Add to my reading list</span>
</a></div>

            

        
</div></div></div></div>

            

        
</li><li><a href="https://www.cell.com/action/addCitationAlert?doi=10.1016%2Fj.cell.2020.05.049" title="Set Alert" target="_blank" aria-label="Open Set Alert in new tab" class="article-tools__item__displayCreateCitationAlert"><i aria-hidden="true" class="icon-set-alert"></i><span>Set Alert</span></a></li><li><a href="https://s100.copyright.com/AppDispatchServlet?publisherName=ELS&amp;contentID=S0092867420306826&amp;orderBeanReset=true&amp;orderSource=Phoenix" title="Get Rights" target="_blank" aria-label="Open Get Rights in new tab" class="article-tools__item__request"><i aria-hidden="true" class="icon-get-rights"></i><span>Get Rights</span></a></li><li><a href="https://www.contentednet.com/salesforce-form/?pii=S0092867420306826&amp;issn=00928674&amp;source=jbs" title="Reprints" target="_blank" aria-label="Open Reprints in new tab" class="article-tools__item__reprints"><i aria-hidden="true" class="icon-article-reprints"></i><span>Reprints</span></a></li></ul>

            

        
</div></div><div class="info-panel__download-issue info-panel__item"><div class="article-header__download-full-issue"><button title="Download full issue" type="submit" formaction="/action/downloadMultiplePdf?action=downloadIssuePdf&amp;journal=cell&amp;volume=182&amp;issue=2&amp;pii=S0092-8674%2819%29X0016-1" class="btn--download-issue btn--cancel download-full-issue"><i class="icon-download-new"></i><span>Download Full Issue</span></button><div class="table-of-content__alert table-of-content__alert--download"><p>Download started</p><a href="#" class="table-of-content__alert--close">Ok</a></div></div></div></div><div class="info-panel__right-content"><div class="info-panel__navigation info-panel__item"><div class="content-navigation"><a href="/cell/fulltext/S0092-8674(20)30816-3" aria-label="Previous article" aria-disabled="false" class="content-navigation__prev"><i aria-hidden="true" class="icon-arrow_l"></i><span>Previous article</span></a><a href="/cell/fulltext/S0092-8674(20)30671-1" aria-label="Next article" aria-disabled="false" class="content-navigation__next"><span>Next article</span><i aria-hidden="true" class="icon-arrow_r"></i></a></div></div></div></div></div></div></div><div class="core-sections-menu-outer"><div class="core-sections-menu"><button aria-controls="article_sections_menu-desktop" aria-haspopup="true" aria-expanded="false" aria-label="Toggle sections menu" data-id="article-toolbar-showhide"><i class="icon-show-outline" aria-hidden="true"></i><span>Show Outline</span></button><nav id="article_sections_menu-desktop" aria-label="Contents" data-core-nav="article" style="margin-top: 0px;"><button aria-controls="article_sections_menu-desktop" aria-haspopup="true" aria-expanded="false" aria-label="Toggle sections menu" data-id="article-toolbar-showhide"><i class="icon-hide-outline" aria-hidden="true"></i><span>Hide Outline</span></button><ul><li><a href="#author-highlights-abstract" aria-current="true">Highlights</a></li><li><a href="#author-abstract" aria-current="false">Summary</a></li><li><a href="#graphical-abstract" aria-current="false">Graphical Abstract</a></li><li><a href="#keywords" aria-current="false">Keywords</a></li><li><a href="#sec-1" aria-current="false">Introduction</a></li><li><a href="#sec-2" aria-current="false">Results</a></li><li><a href="#sec-3" aria-current="false">Discussion</a></li><li><a href="#sec-4" aria-current="false">STAR★Methods</a></li><li><a href="#acknowledgments" aria-current="false">Acknowledgments</a></li><li><a href="#supplementary-material" aria-current="false">Supplemental Information (6)</a></li><li><a href="#references" aria-current="false">References</a></li><li role="none" class="mobile-only"><a role="menuitem" href="#core-collateral-metrics">Article metrics</a></li><li role="none" class="mobile-only"><a role="menuitem" href="#core-collateral-relatedArticles">Related Articles</a></li></ul></nav></div></div><div><div class="core-content-inner"><div class="core-linked-content"></div><div id="abstracts" data-extent="frontmatter"><div class="core-container"><section id="author-highlights-abstract" property="abstract" typeof="Text" role="doc-abstract"><h2 property="name">Highlights</h2><div id="abspara0020" role="paragraph"><div id="ulist0010" role="list"><div id="u0010" role="listitem"><div class="label">•</div><div class="content"><div id="p0010" role="paragraph"><i>TBL1XR1</i> mutation skews the humoral immune response toward producing memory B cells</div></div></div><div id="u0015" role="listitem"><div class="label">•</div><div class="content"><div id="p0015" role="paragraph">TBL1XR1 mutant memory cells feature aberrant cyclic reentry to new germinal centers</div></div></div><div id="u0020" role="listitem"><div class="label">•</div><div class="content"><div id="p0020" role="paragraph">Mutant TBL1XR1 acts by triggering aberrant targeting of SMRT complex to BACH2</div></div></div><div id="u0025" role="listitem"><div class="label">•</div><div class="content"><div id="p0025" role="paragraph"><i>TBL1XR1</i> mutation gives rise to extranodal ABC-DLBCLs derived from memory B cells</div></div></div></div></div></section><section id="author-abstract" property="abstract" typeof="Text" role="doc-abstract"><h2 property="name">Summary</h2><div id="abspara0010" role="paragraph">The most aggressive B cell lymphomas frequently manifest extranodal distribution and carry somatic mutations in the poorly characterized gene <i>TBL1XR1</i>. Here, we show that <i>TBL1XR1</i> mutations skew the humoral immune response toward generating abnormal immature memory B cells (MB), while impairing plasma cell differentiation. At the molecular level, TBL1XR1 mutants co-opt SMRT/HDAC3 repressor complexes toward binding the MB cell transcription factor (TF) BACH2 at the expense of the germinal center (GC) TF BCL6, leading to pre-memory transcriptional reprogramming and cell-fate bias. Upon antigen recall, TBL1XR1 mutant MB cells fail to differentiate into plasma cells and instead preferentially reenter new GC reactions, providing evidence for a cyclic reentry lymphomagenesis mechanism. Ultimately, <i>TBL1XR1</i> alterations lead to a striking extranodal immunoblastic lymphoma phenotype that mimics the human disease. Both human and murine lymphomas feature expanded MB-like cell populations, consistent with a MB-cell origin and delineating an unforeseen pathway for malignant transformation of the immune system.</div></section><section id="graphical-abstract" property="abstract" typeof="Text" role="doc-abstract"><h2 property="name">Graphical Abstract</h2><div id="abspara0015" role="paragraph"><div class="figure-wrap"><figure id="undfig1" class="graphic" data-figureviewer-ignore="true"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2020.05.049/asset/607289b6-7090-47d5-9e5c-834d32ad9b93/main.assets/fx1_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="fx1" alt="Graphical abstract undfig1" src="/cms/10.1016/j.cell.2020.05.049/asset/7ed32941-8046-46d8-95ad-71914133ef00/main.assets/fx1.jpg" height="375" width="375" data-viewer-src="/cms/10.1016/j.cell.2020.05.049/asset/607289b6-7090-47d5-9e5c-834d32ad9b93/main.assets/fx1_lrg.jpg" loading="lazy"></a></figure></div></div></section></div></div><div class="core-linked-content"></div><section id="keywords" property="keywords"><h2>Keywords</h2><ol><li><a href="/action/doSearch?AllField=&quot;memory+B+cells&quot;" alt="Search on this keyword" rel="nofollow">memory B cells</a></li><li><a href="/action/doSearch?AllField=&quot;germinal+center&quot;" alt="Search on this keyword" rel="nofollow">germinal center</a></li><li><a href="/action/doSearch?AllField=&quot;ABC-DLBCL&quot;" alt="Search on this keyword" rel="nofollow">ABC-DLBCL</a></li><li><a href="/action/doSearch?AllField=&quot;extranodal+lymphoma&quot;" alt="Search on this keyword" rel="nofollow">extranodal lymphoma</a></li><li><a href="/action/doSearch?AllField=&quot;cell+fate&quot;" alt="Search on this keyword" rel="nofollow">cell fate</a></li><li><a href="/action/doSearch?AllField=&quot;TBL1XR1&quot;" alt="Search on this keyword" rel="nofollow">TBL1XR1</a></li><li><a href="/action/doSearch?AllField=&quot;BCL6&quot;" alt="Search on this keyword" rel="nofollow">BCL6</a></li><li><a href="/action/doSearch?AllField=&quot;BACH2&quot;" alt="Search on this keyword" rel="nofollow">BACH2</a></li><li><a href="/action/doSearch?AllField=&quot;cell+of+origin&quot;" alt="Search on this keyword" rel="nofollow">cell of origin</a></li></ol></section><div class="core-linked-content"></div><div class="core-linked-content"></div><section id="bodymatter" data-extent="bodymatter" property="articleBody" typeof="Text"><div class="core-container"><section id="sec-1"><h2>Introduction</h2><div id="p0030" role="paragraph">Diffuse large B cell lymphomas (DLBCLs) are the most common class of lymphomas (<span class="dropBlock reference-citations"><a id="body-ref-sref10-1" href="#" role="doc-biblioref" data-xml-rid="bib10" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref10-1" aria-controls="sref10-1">Beham-Schmid, 2017</a><div class="dropBlock__holder" data-db-target-of="sref10-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">10.</div><div>Beham-Schmid, C.</div><div><strong>Aggressive lymphoma 2016: revision of the WHO classification</strong></div><div><em>Memo.</em> 2017; <strong>10</strong>:248-254</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/s12254-017-0367-8" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-85035795436" target="_blank" aria-label="Open Scopus in new tab">Scopus (9)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29250206/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2Fs12254-017-0367-8&amp;pmid=29250206" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and likely represent a heterogeneous group of diseases arising from different immune system processes. Gene expression profiling studies classified DLBCLs into two major subtypes: germinal center B cell (GCB) and activated B cell (ABC)-DLBCLs. GCB-DLBCLs reflect the transcriptional signature of B cells that are transiting through the germinal center (GC) reaction (<span class="dropBlock reference-citations"><a id="body-ref-sref112-1" href="#" role="doc-biblioref" data-xml-rid="bib112" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref112-1" aria-controls="sref112-1">Wright et&nbsp;al., 2003</a><div class="dropBlock__holder" data-db-target-of="sref112-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">109.</div><div>Wright, G. ∙ Tan, B. ∙ Rosenwald, A. ...</div><div><strong>A gene expression-based method to diagnose clinically distinct subgroups of diffuse large B cell lymphoma</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2003; <strong>100</strong>:9991-9996</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1732008100" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_109_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-0043192901" target="_blank" aria-label="Open Scopus in new tab">Scopus (726)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/12900505/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1732008100&amp;pmid=12900505" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). GCs form transiently in response to T&nbsp;cell-dependent antigens and are composed of rapidly proliferating B cells that simultaneously undergo immunoglobulin somatic hypermutation (SHM). After transiting to a T&nbsp;cell-rich GC “light zone,” GCB expressing the highest affinity immunoglobulins undergo plasma cell (PC) differentiation, while others transition to forming memory B cells (reviewed in <span class="dropBlock reference-citations"><a id="body-ref-sref22" href="#" role="doc-biblioref" data-xml-rid="bib22" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref22" aria-controls="sref22">Cyster and Allen, 2019</a><div class="dropBlock__holder" data-db-target-of="sref22"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">22.</div><div>Cyster, J.G. ∙ Allen, C.D.C.</div><div><strong>B Cell Responses: Cell Interaction Dynamics and Decisions</strong></div><div><em>Cell.</em> 2019; <strong>177</strong>:524-540</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=10.1016%2Fj.cell.2019.03.016&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=10.1016%2Fj.cell.2019.03.016&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-85064249878" target="_blank" aria-label="Open Scopus in new tab">Scopus (43)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31002794/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2019.03.016&amp;pmid=31002794" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). ABC-DLCBLs are described as manifesting a “post-GC” transcriptional signature, and evidence of chronic active B cell receptor (BCR) signaling and nuclear factor κB (NF-κB) activation is linked to somatic mutations in the BCR and toll-like receptor (TLR) signaling pathways (<span class="dropBlock reference-citations"><a id="body-ref-sref82" href="#" role="doc-biblioref" data-xml-rid="bib82" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref82" aria-controls="sref82">Phelan et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref82"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">82.</div><div>Phelan, J.D. ∙ Young, R.M. ∙ Webster, D.E. ...</div><div><strong>A multiprotein supercomplex controlling oncogenic signalling in lymphoma</strong></div><div><em>Nature.</em> 2018; <strong>560</strong>:387-391</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-018-0290-0" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_82_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-85051208319" target="_blank" aria-label="Open Scopus in new tab">Scopus (79)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29925955/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-018-0290-0&amp;pmid=29925955" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref112-2" href="#" role="doc-biblioref" data-xml-rid="bib112" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref112-2" aria-controls="sref112-2">Wright et&nbsp;al., 2003</a><div class="dropBlock__holder" data-db-target-of="sref112-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">109.</div><div>Wright, G. ∙ Tan, B. ∙ Rosenwald, A. ...</div><div><strong>A gene expression-based method to diagnose clinically distinct subgroups of diffuse large B cell lymphoma</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2003; <strong>100</strong>:9991-9996</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1732008100" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_109_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-0043192901" target="_blank" aria-label="Open Scopus in new tab">Scopus (726)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/12900505/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1732008100&amp;pmid=12900505" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). However, the cell of origin (COO) and mechanisms of malignant transformation of ABC-DLBCLs remain poorly understood. From a clinical perspective, this is problematic, because these are among the most incurable and aggressive lymphomas (<span class="dropBlock reference-citations"><a id="body-ref-sref10-2" href="#" role="doc-biblioref" data-xml-rid="bib10" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref10-2" aria-controls="sref10-2">Beham-Schmid, 2017</a><div class="dropBlock__holder" data-db-target-of="sref10-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">10.</div><div>Beham-Schmid, C.</div><div><strong>Aggressive lymphoma 2016: revision of the WHO classification</strong></div><div><em>Memo.</em> 2017; <strong>10</strong>:248-254</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/s12254-017-0367-8" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-85035795436" target="_blank" aria-label="Open Scopus in new tab">Scopus (9)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29250206/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2Fs12254-017-0367-8&amp;pmid=29250206" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="p0035" role="paragraph">DLBCLs were more recently classified into putative distinct entities based on constellations of genetic lesions (<span class="dropBlock reference-citations"><a id="body-ref-sref16-1" href="#" role="doc-biblioref" data-xml-rid="bib16" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref16-1" aria-controls="sref16-1">Chapuy et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref16-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">16.</div><div>Chapuy, B. ∙ Stewart, C. ∙ Dunford, A.J. ...</div><div><strong>Molecular subtypes of diffuse large B cell lymphoma are associated with distinct pathogenic mechanisms and outcomes</strong></div><div><em>Nat. Med.</em> 2018; <strong>24</strong>:679-690</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41591-018-0016-8" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-85046087353" target="_blank" aria-label="Open Scopus in new tab">Scopus (257)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29713087/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41591-018-0016-8&amp;pmid=29713087" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref86-1" href="#" role="doc-biblioref" data-xml-rid="bib86" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref86-1" aria-controls="sref86-1">Reddy et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref86-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">84.</div><div>Reddy, A. ∙ Zhang, J. ∙ Davis, N.S. ...</div><div><strong>Genetic and Functional Drivers of Diffuse Large B Cell Lymphoma</strong></div><div><em>Cell.</em> 2017; <strong>171</strong>:481-494</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_84_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=10.1016%2Fj.cell.2017.09.027&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_84_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=10.1016%2Fj.cell.2017.09.027&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_84_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-85030529459" target="_blank" aria-label="Open Scopus in new tab">Scopus (251)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28985567/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2017.09.027&amp;pmid=28985567" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref92-1" href="#" role="doc-biblioref" data-xml-rid="bib92" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref92-1" aria-controls="sref92-1">Schmitz et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref92-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">90.</div><div>Schmitz, R. ∙ Wright, G.W. ∙ Huang, D.W. ...</div><div><strong>Genetics and Pathogenesis of Diffuse Large B-Cell Lymphoma</strong></div><div><em>N.&nbsp;Engl. J. Med.</em> 2018; <strong>378</strong>:1396-1407</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1056/NEJMoa1801445" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_90_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-85045505659" target="_blank" aria-label="Open Scopus in new tab">Scopus (314)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29641966/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1056%2FNEJMoa1801445&amp;pmid=29641966" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Two independent studies identified a novel genetically defined ABC-DLBCL subtype (“MCD” or “Cluster 5 [C5]” lymphomas) associated with unfavorable clinical outcomes, somatic activating mutations of <i>MYD88</i>, <i>CD79B</i>, and recurrent mutation of the poorly characterized gene <i>TBL1XR1</i> (<span class="dropBlock reference-citations"><a id="body-ref-sref16-2" href="#" role="doc-biblioref" data-xml-rid="bib16" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref16-2" aria-controls="sref16-2">Chapuy et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref16-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">16.</div><div>Chapuy, B. ∙ Stewart, C. ∙ Dunford, A.J. ...</div><div><strong>Molecular subtypes of diffuse large B cell lymphoma are associated with distinct pathogenic mechanisms and outcomes</strong></div><div><em>Nat. Med.</em> 2018; <strong>24</strong>:679-690</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41591-018-0016-8" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-85046087353" target="_blank" aria-label="Open Scopus in new tab">Scopus (257)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29713087/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41591-018-0016-8&amp;pmid=29713087" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref92-2" href="#" role="doc-biblioref" data-xml-rid="bib92" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref92-2" aria-controls="sref92-2">Schmitz et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref92-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">90.</div><div>Schmitz, R. ∙ Wright, G.W. ∙ Huang, D.W. ...</div><div><strong>Genetics and Pathogenesis of Diffuse Large B-Cell Lymphoma</strong></div><div><em>N.&nbsp;Engl. J. Med.</em> 2018; <strong>378</strong>:1396-1407</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1056/NEJMoa1801445" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_90_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-85045505659" target="_blank" aria-label="Open Scopus in new tab">Scopus (314)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29641966/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1056%2FNEJMoa1801445&amp;pmid=29641966" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). <i>TBL1XR</i>1 mutations are likely founder events, based on variant allele frequency (<span class="dropBlock reference-citations"><a id="body-ref-sref16-3" href="#" role="doc-biblioref" data-xml-rid="bib16" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref16-3" aria-controls="sref16-3">Chapuy et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref16-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">16.</div><div>Chapuy, B. ∙ Stewart, C. ∙ Dunford, A.J. ...</div><div><strong>Molecular subtypes of diffuse large B cell lymphoma are associated with distinct pathogenic mechanisms and outcomes</strong></div><div><em>Nat. Med.</em> 2018; <strong>24</strong>:679-690</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41591-018-0016-8" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-85046087353" target="_blank" aria-label="Open Scopus in new tab">Scopus (257)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29713087/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41591-018-0016-8&amp;pmid=29713087" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). From the clinical and biological perspective, a striking feature of these lymphomas is their unusual extranodal distribution (<span class="dropBlock reference-citations"><a id="body-ref-sref16-4" href="#" role="doc-biblioref" data-xml-rid="bib16" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref16-4" aria-controls="sref16-4">Chapuy et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref16-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">16.</div><div>Chapuy, B. ∙ Stewart, C. ∙ Dunford, A.J. ...</div><div><strong>Molecular subtypes of diffuse large B cell lymphoma are associated with distinct pathogenic mechanisms and outcomes</strong></div><div><em>Nat. Med.</em> 2018; <strong>24</strong>:679-690</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41591-018-0016-8" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-85046087353" target="_blank" aria-label="Open Scopus in new tab">Scopus (257)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29713087/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41591-018-0016-8&amp;pmid=29713087" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref92-3" href="#" role="doc-biblioref" data-xml-rid="bib92" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref92-3" aria-controls="sref92-3">Schmitz et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref92-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">90.</div><div>Schmitz, R. ∙ Wright, G.W. ∙ Huang, D.W. ...</div><div><strong>Genetics and Pathogenesis of Diffuse Large B-Cell Lymphoma</strong></div><div><em>N.&nbsp;Engl. J. Med.</em> 2018; <strong>378</strong>:1396-1407</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1056/NEJMoa1801445" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_90_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-85045505659" target="_blank" aria-label="Open Scopus in new tab">Scopus (314)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29641966/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1056%2FNEJMoa1801445&amp;pmid=29641966" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), invading sites like the CNS and testes (<span class="dropBlock reference-citations"><a id="body-ref-sref15" href="#" role="doc-biblioref" data-xml-rid="bib15" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref15" aria-controls="sref15">Chapuy et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref15"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">15.</div><div>Chapuy, B. ∙ Roemer, M.G. ∙ Stewart, C. ...</div><div><strong>Targetable genetic features of primary testicular and primary central nervous system lymphomas</strong></div><div><em>Blood.</em> 2016; <strong>127</strong>:869-881</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1182/blood-2015-10-673236" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-84959419747" target="_blank" aria-label="Open Scopus in new tab">Scopus (207)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26702065/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1182%2Fblood-2015-10-673236&amp;pmid=26702065" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref31" href="#" role="doc-biblioref" data-xml-rid="bib31" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref31" aria-controls="sref31">Gonzalez-Aguilar et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref31"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">31.</div><div>Gonzalez-Aguilar, A. ∙ Idbaih, A. ∙ Boisselier, B. ...</div><div><strong>Recurrent mutations of MYD88 and TBL1XR1 in primary central nervous system lymphomas</strong></div><div><em>Clin. Cancer Res.</em> 2012; <strong>18</strong>:5203-5211</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1158/1078-0432.CCR-12-0845" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-84866895485" target="_blank" aria-label="Open Scopus in new tab">Scopus (145)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22837180/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1158%2F1078-0432.CCR-12-0845&amp;pmid=22837180" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Neither GCB nor PC normally home to these tissues, raising questions about the pathogenesis and origin of these tumors within the complex milieu of the immune system.</div><div id="p0040" role="paragraph">Congenital missense mutations in <i>TBL1XR1</i> similar to those in lymphoma have also been linked to early-childhood developmental disorders, such as Pierpont syndrome (<span class="dropBlock reference-citations"><a id="body-ref-sref37-1" href="#" role="doc-biblioref" data-xml-rid="bib37" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref37-1" aria-controls="sref37-1">Heinen et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref37-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">37.</div><div>Heinen, C.A. ∙ Jongejan, A. ∙ Watson, P.J. ...</div><div><strong>A specific mutation in TBL1XR1 causes Pierpont syndrome</strong></div><div><em>J.&nbsp;Med. Genet.</em> 2016; <strong>53</strong>:330-337</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1136/jmedgenet-2015-103233" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-84957921471" target="_blank" aria-label="Open Scopus in new tab">Scopus (32)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26769062/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1136%2Fjmedgenet-2015-103233&amp;pmid=26769062" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref54-1" href="#" role="doc-biblioref" data-xml-rid="bib54" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref54-1" aria-controls="sref54-1">Laskowski et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref54-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">54.</div><div>Laskowski, R.A. ∙ Tyagi, N. ∙ Johnson, D. ..., DDD Study</div><div><strong>Integrating population variation and protein structural analysis to improve clinical interpretation of missense variation: application to the WD40 domain</strong></div><div><em>Hum. Mol. Genet.</em> 2016; <strong>25</strong>:927-935</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/hmg/ddv625" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-84962815246" target="_blank" aria-label="Open Scopus in new tab">Scopus (12)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26740553/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fhmg%2Fddv625&amp;pmid=26740553" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), suggestive of TBL1XR1’s broad biological relevance. From the biochemical standpoint, TBL1XR1 is a core component of the SMRT/NCOR1 transcriptional repressor complexes (<span class="dropBlock reference-citations"><a id="body-ref-sref114" href="#" role="doc-biblioref" data-xml-rid="bib114" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref114" aria-controls="sref114">Yoon et&nbsp;al., 2003</a><div class="dropBlock__holder" data-db-target-of="sref114"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">111.</div><div>Yoon, H.G. ∙ Chan, D.W. ∙ Huang, Z.Q. ...</div><div><strong>Purification and functional characterization of the human N-CoR complex: the roles of HDAC3, TBL1 and TBLR1</strong></div><div><em>EMBO J.</em> 2003; <strong>22</strong>:1336-1346</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/emboj/cdg120" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_111_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-0344405718" target="_blank" aria-label="Open Scopus in new tab">Scopus (326)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/12628926/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Femboj%2Fcdg120&amp;pmid=12628926" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). These complexes also contain HDAC3, which enables the complex to repress transcription (<span class="dropBlock reference-citations"><a id="body-ref-sref35-1" href="#" role="doc-biblioref" data-xml-rid="bib35" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref35-1" aria-controls="sref35-1">Hatzi et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref35-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">35.</div><div>Hatzi, K. ∙ Jiang, Y. ∙ Huang, C. ...</div><div><strong>A hybrid mechanism of action for BCL6 in B cells defined by formation of functionally distinct complexes at enhancers and promoters</strong></div><div><em>Cell Rep.</em> 2013; <strong>4</strong>:578-588</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=10.1016%2Fj.celrep.2013.06.016&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=10.1016%2Fj.celrep.2013.06.016&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-84881615483" target="_blank" aria-label="Open Scopus in new tab">Scopus (90)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23911289/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.celrep.2013.06.016&amp;pmid=23911289" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Little is known about the exact role of TBL1XR1, which has been alternatively described as (1) enhancing the functionality (<span class="dropBlock reference-citations"><a id="body-ref-sref106" href="#" role="doc-biblioref" data-xml-rid="bib106" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref106" aria-controls="sref106">Tomita et&nbsp;al., 2004</a><div class="dropBlock__holder" data-db-target-of="sref106"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">103.</div><div>Tomita, A. ∙ Buchholz, D.R. ∙ Shi, Y.B.</div><div><strong>Recruitment of N-CoR/SMRT-TBLR1 corepressor complex by unliganded thyroid hormone receptor for gene repression during frog development</strong></div><div><em>Mol. Cell. Biol.</em> 2004; <strong>24</strong>:3337-3346</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/MCB.24.8.3337-3346.2004" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_103_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-1842609856" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15060155/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FMCB.24.8.3337-3346.2004&amp;pmid=15060155" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), or (2) driving the disassembly (<span class="dropBlock reference-citations"><a id="body-ref-sref80-1" href="#" role="doc-biblioref" data-xml-rid="bib80" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref80-1" aria-controls="sref80-1">Perissi et&nbsp;al., 2004</a><div class="dropBlock__holder" data-db-target-of="sref80-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">81.</div><div>Perissi, V. ∙ Aggarwal, A. ∙ Glass, C.K. ...</div><div><strong>A corepressor/coactivator exchange complex required for transcriptional activation by nuclear receptors and other regulated transcription factors</strong></div><div><em>Cell.</em> 2004; <strong>116</strong>:511-526</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=10.1016%2FS0092-8674%2804%2900133-3&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=10.1016%2FS0092-8674%2804%2900133-3&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-1342264315" target="_blank" aria-label="Open Scopus in new tab">Scopus (412)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/14980219/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0092-8674%2804%2900133-3&amp;pmid=14980219" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) of these complexes. Whichever the case, it is notable that in GCB the majority of SMRT/NCOR1 complexes are bound and recruited by the transcriptional repressor BCL6 (<span class="dropBlock reference-citations"><a id="body-ref-sref35-2" href="#" role="doc-biblioref" data-xml-rid="bib35" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref35-2" aria-controls="sref35-2">Hatzi et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref35-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">35.</div><div>Hatzi, K. ∙ Jiang, Y. ∙ Huang, C. ...</div><div><strong>A hybrid mechanism of action for BCL6 in B cells defined by formation of functionally distinct complexes at enhancers and promoters</strong></div><div><em>Cell Rep.</em> 2013; <strong>4</strong>:578-588</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=10.1016%2Fj.celrep.2013.06.016&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=10.1016%2Fj.celrep.2013.06.016&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-84881615483" target="_blank" aria-label="Open Scopus in new tab">Scopus (90)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23911289/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.celrep.2013.06.016&amp;pmid=23911289" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), which is essential for GC formation. Indeed, most recurrent mutations in DLBCL directly or indirectly enhance BCL6 functionality (reviewed in <span class="dropBlock reference-citations"><a id="body-ref-sref34-1" href="#" role="doc-biblioref" data-xml-rid="bib34" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref34-1" aria-controls="sref34-1">Hatzi and Melnick, 2014</a><div class="dropBlock__holder" data-db-target-of="sref34-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">34.</div><div>Hatzi, K. ∙ Melnick, A.</div><div><strong>Breaking bad in the germinal center: how deregulation of BCL6 contributes to lymphomagenesis</strong></div><div><em>Trends Mol. Med.</em> 2014; <strong>20</strong>:343-352</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=10.1016%2Fj.molmed.2014.03.001&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=10.1016%2Fj.molmed.2014.03.001&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-84901585384" target="_blank" aria-label="Open Scopus in new tab">Scopus (80)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24698494/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.molmed.2014.03.001&amp;pmid=24698494" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), a driving force on which most of these tumors rely for their survival (<span class="dropBlock reference-citations"><a id="body-ref-sref11" href="#" role="doc-biblioref" data-xml-rid="bib11" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref11" aria-controls="sref11">Cardenas et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref11"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">11.</div><div>Cardenas, M.G. ∙ Yu, W. ∙ Beguelin, W. ...</div><div><strong>Rationally designed BCL6 inhibitors target activated B cell diffuse large B cell lymphoma</strong></div><div><em>J.&nbsp;Clin. Invest.</em> 2016; <strong>126</strong>:3351-3362</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1172/JCI85795" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-84987788606" target="_blank" aria-label="Open Scopus in new tab">Scopus (65)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27482887/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1172%2FJCI85795&amp;pmid=27482887" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref14" href="#" role="doc-biblioref" data-xml-rid="bib14" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref14" aria-controls="sref14">Cerchietti et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref14"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">14.</div><div>Cerchietti, L.C. ∙ Ghetu, A.F. ∙ Zhu, X. ...</div><div><strong>A small-molecule inhibitor of BCL6 kills DLBCL cells in&nbsp;vitro and in&nbsp;vivo</strong></div><div><em>Cancer Cell.</em> 2010; <strong>17</strong>:400-411</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=10.1016%2Fj.ccr.2009.12.050&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=10.1016%2Fj.ccr.2009.12.050&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-77950486830" target="_blank" aria-label="Open Scopus in new tab">Scopus (205)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20385364/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.ccr.2009.12.050&amp;pmid=20385364" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). <i>TBL1XR1</i> mutations are thus highly intriguing, given their potential to affect SMRT/NCOR1 complexes, as well as BCL6, and the complete lack of information on how this might play out from the molecular and biological perspectives. Herein, we explore the mechanism of action of <i>TBL1XR1</i> mutations, their impact on the humoral immune response, and whether they provide critical insight into the origin and immunological nature of aggressive extranodal MCD/C5 DLBCLs.</div></section><section id="sec-2"><h2>Results</h2><section id="sec-2-1"><h3><i>Tbl1xr1</i> Mutation Impairs Germinal Center Development</h3><div id="p0045" role="paragraph"><i>TBL1XR1</i> somatic mutations occur in ∼5%–10% of DLBCL and follicular lymphoma (FL) cases, mainly as heterozygous missense alleles (<a id="crosref0310" href="#fig1">Figure&nbsp;1</a>A). In line with recent studies (<span class="dropBlock reference-citations"><a id="body-ref-sref16-5" href="#" role="doc-biblioref" data-xml-rid="bib16" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref16-5" aria-controls="sref16-5">Chapuy et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref16-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">16.</div><div>Chapuy, B. ∙ Stewart, C. ∙ Dunford, A.J. ...</div><div><strong>Molecular subtypes of diffuse large B cell lymphoma are associated with distinct pathogenic mechanisms and outcomes</strong></div><div><em>Nat. Med.</em> 2018; <strong>24</strong>:679-690</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41591-018-0016-8" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-85046087353" target="_blank" aria-label="Open Scopus in new tab">Scopus (257)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29713087/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41591-018-0016-8&amp;pmid=29713087" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref92-4" href="#" role="doc-biblioref" data-xml-rid="bib92" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref92-4" aria-controls="sref92-4">Schmitz et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref92-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">90.</div><div>Schmitz, R. ∙ Wright, G.W. ∙ Huang, D.W. ...</div><div><strong>Genetics and Pathogenesis of Diffuse Large B-Cell Lymphoma</strong></div><div><em>N.&nbsp;Engl. J. Med.</em> 2018; <strong>378</strong>:1396-1407</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1056/NEJMoa1801445" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_90_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-85045505659" target="_blank" aria-label="Open Scopus in new tab">Scopus (314)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29641966/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1056%2FNEJMoa1801445&amp;pmid=29641966" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), our analysis of three independent DLBCL cohorts confirmed that mutations occur more frequently in ABC-DLBCL (<a id="crosref2525" href="#mmc1">Table S1</a>; <a id="crosref0330" href="#mmc6">Data S1</a>A), are highly enriched in the MCD subtype (<a id="crosref0335" href="#mmc5">Data S1</a>B), and often co-occur with <i>MYD88</i> mutations (<a id="crosref0340" href="#mmc6">Data S1</a>C). <i>TBL1XR1</i> mutations largely occur within its WD40 domain (<a id="crosref0345" href="#fig1">Figures 1</a>A and 1B) and affect aromatic residues exposed on the same surface of the barrel structure (<a id="crosref0350" href="#fig1">Figure&nbsp;1</a>B) that are predicted to mediate protein-protein interactions (PPI) (<span class="dropBlock reference-citations"><a id="body-ref-sref108" href="#" role="doc-biblioref" data-xml-rid="bib108" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref108" aria-controls="sref108">Wang et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref108"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">105.</div><div>Wang, Y. ∙ Hu, X.J. ∙ Zou, X.D. ...</div><div><strong>WDSPdb: a database for WD40-repeat proteins</strong></div><div><em>Nucleic Acids Res.</em> 2015; <strong>43</strong>:D339-D344</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gku1023" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_105_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-84941093734" target="_blank" aria-label="Open Scopus in new tab">Scopus (45)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25348404/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgku1023&amp;pmid=25348404" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div class="figure-wrap"><figure id="fig1" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2020.05.049/asset/da28a7b4-eb3c-4ef8-a301-20c9c6e93bf9/main.assets/gr1_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr1" src="/cms/10.1016/j.cell.2020.05.049/asset/ba82fc3d-99c6-4ccc-a053-85147c162594/main.assets/gr1.jpg" height="941" width="671" aria-labelledby="fig1-title fig1-content" data-viewer-src="/cms/10.1016/j.cell.2020.05.049/asset/da28a7b4-eb3c-4ef8-a301-20c9c6e93bf9/main.assets/gr1_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig1-title" class="figure__title"><span class="label figure__label">Figure&nbsp;1</span> <span class="figure__title__text"><i>Tbl1xr1</i> Mutation Impairs GC Development</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig1-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0015" role="paragraph">(A) <i>TBL1XR1</i> mutations in DLBCL (<span class="dropBlock reference-citations"><a id="body-ref-sref5-1" href="#" role="doc-biblioref" data-xml-rid="bib5" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref5-1" aria-controls="sref5-1">Arthur et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref5-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">5.</div><div>Arthur, S.E. ∙ Jiang, A. ∙ Grande, B.M. ...</div><div><strong>Genome-wide discovery of somatic regulatory variants in diffuse large B-cell lymphoma</strong></div><div><em>Nat. Commun.</em> 2018; <strong>9</strong>:4001</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41467-018-06354-3" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-85054070552" target="_blank" aria-label="Open Scopus in new tab">Scopus (32)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30275490/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41467-018-06354-3&amp;pmid=30275490" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref61-1" href="#" role="doc-biblioref" data-xml-rid="bib61" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref61-1" aria-controls="sref61-1">Ma et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref61-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">61.</div><div>Ma, M.C.J. ∙ Tadros, S. ∙ Bouska, A. ...</div><div><strong>Pathognomonic and epistatic genetic alterations in B-cell non-Hodgkin lymphoma</strong></div><div><em>bioRxiv.</em> 2019;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/674259" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2F674259" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref86-2" href="#" role="doc-biblioref" data-xml-rid="bib86" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref86-2" aria-controls="sref86-2">Reddy et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref86-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">84.</div><div>Reddy, A. ∙ Zhang, J. ∙ Davis, N.S. ...</div><div><strong>Genetic and Functional Drivers of Diffuse Large B Cell Lymphoma</strong></div><div><em>Cell.</em> 2017; <strong>171</strong>:481-494</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_84_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=10.1016%2Fj.cell.2017.09.027&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_84_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=10.1016%2Fj.cell.2017.09.027&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_84_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-85030529459" target="_blank" aria-label="Open Scopus in new tab">Scopus (251)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28985567/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2017.09.027&amp;pmid=28985567" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and FL (<span class="dropBlock reference-citations"><a id="body-ref-sref51-1" href="#" role="doc-biblioref" data-xml-rid="bib51" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref51-1" aria-controls="sref51-1">Krysiak et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref51-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">51.</div><div>Krysiak, K. ∙ Gomez, F. ∙ White, B.S. ...</div><div><strong>Recurrent somatic mutations affecting B-cell receptor signaling pathway genes in follicular lymphoma</strong></div><div><em>Blood.</em> 2017; <strong>129</strong>:473-483</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1182/blood-2016-07-729954" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_51_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-85014960882" target="_blank" aria-label="Open Scopus in new tab">Scopus (58)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28064239/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1182%2Fblood-2016-07-729954&amp;pmid=28064239" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref61-2" href="#" role="doc-biblioref" data-xml-rid="bib61" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref61-2" aria-controls="sref61-2">Ma et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref61-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">61.</div><div>Ma, M.C.J. ∙ Tadros, S. ∙ Bouska, A. ...</div><div><strong>Pathognomonic and epistatic genetic alterations in B-cell non-Hodgkin lymphoma</strong></div><div><em>bioRxiv.</em> 2019;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/674259" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2F674259" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref77-1" href="#" role="doc-biblioref" data-xml-rid="bib77" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref77-1" aria-controls="sref77-1">Ortega-Molina et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref77-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">78.</div><div>Ortega-Molina, A. ∙ Boss, I.W. ∙ Canela, A. ...</div><div><strong>The histone lysine methyltransferase KMT2D sustains a gene expression program that represses B cell lymphoma development</strong></div><div><em>Nat. Med.</em> 2015; <strong>21</strong>:1199-1208</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nm.3943" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_78_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-84943658052" target="_blank" aria-label="Open Scopus in new tab">Scopus (159)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26366710/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnm.3943&amp;pmid=26366710" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). TBL1XR1-SMRT interacting region (<span class="dropBlock reference-citations"><a id="body-ref-sref116-1" href="#" role="doc-biblioref" data-xml-rid="bib116" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref116-1" aria-controls="sref116-1">Zhang et&nbsp;al., 2002</a><div class="dropBlock__holder" data-db-target-of="sref116-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">113.</div><div>Zhang, J. ∙ Kalkum, M. ∙ Chait, B.T. ...</div><div><strong>The N-CoR-HDAC3 nuclear receptor corepressor complex inhibits the JNK pathway through the integral subunit GPS2</strong></div><div><em>Mol. Cell.</em> 2002; <strong>9</strong>:611-623</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_113_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=10.1016%2FS1097-2765%2802%2900468-9&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_113_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=10.1016%2FS1097-2765%2802%2900468-9&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_113_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-0036211850" target="_blank" aria-label="Open Scopus in new tab">Scopus (305)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11931768/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS1097-2765%2802%2900468-9&amp;pmid=11931768" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and PPI mutated positions are indicated. See also <a id="crosref2285" href="#mmc6">Data S1</a>A–S1C and <a id="crosref2530" href="#mmc1">Table S1</a>.</div><div id="fspara0020" role="paragraph">(B) Exposed (&gt;25% accessible surface) or buried residues affected by missense DLBCL mutations in TBL1XR1 WD40 domain (4LG9; <a id="intref0130" href="https://doi.org/10.2210/pdb4lg9/pdb">https://doi.org/10.2210/pdb4lg9/pdb</a>).</div><div id="fspara0025" role="paragraph">(C and D) FC analysis of splenic (C) total B cells or (D) GCB. See also <a id="crosref2295" href="#mmc6">Data S1</a>D–S1F.</div><div id="fspara0030" role="paragraph">(E) Spleen sections H&amp;E from animals treated as in (C). Insets show zoom of outlined areas. Scale, 500&nbsp;μm (top), 100&nbsp;μm (bottom).</div><div id="fspara0035" role="paragraph">(F) B220 or PNA IHC in consecutive spleen sections from (E). Scale, 100&nbsp;μm.</div><div id="fspara0040" role="paragraph">(G and H) Number of GC per spleen section (G) or GC size (H) as (left) number of cells or (right) area, based on PNA staining. Dots represent individual (G) animals or (H) GCs. Results for 5 animals per genotype.</div><div id="fspara0045" role="paragraph">(I) FC analysis of D370Y/WT and WT/WT relative contribution to total B cells and GCB, based on CD45 allelic frequencies. See also <a id="crosref2300" href="#mmc6">Data S1</a>H–S1J.</div><div id="fspara0050" role="paragraph">(J) Use of the <i>Rosa26YFPstop</i> reporter.</div><div id="fspara0055" role="paragraph">(K) FC analysis of splenic GCB. Left to right: n&nbsp;= 4, 5, 5, 4 per genotype. See also <a id="crosref2305" href="#mmc6">Data S1</a>K–S1L.</div><div id="fspara0060" role="paragraph">(L) FC analysis of splenic GCB.</div><div id="fspara0065" role="paragraph">Values represent mean ± SEM. Data reproducible with three repeats. NS, not significant; <sup>∗</sup>p&nbsp;&lt; 0.05; <sup>∗∗</sup>p&nbsp;&lt; 0.01; <sup>∗∗∗</sup>p&nbsp;&lt; 0.001, using unpaired (C, D, and K) or paired (I) two-tailed Student’s t test, or Mann-Whitney U-test (G and H) or one-way ANOVA with Tukey’s post-test (L).</div></div></div></div></figcaption></figure></div><div id="p0050" role="paragraph">Faced with the lack of a clear hotspot, we decided to model the D370Y allele in mice given it occurs both in lymphomas and congenital developmental disorders (<span class="dropBlock reference-citations"><a id="body-ref-sref54-2" href="#" role="doc-biblioref" data-xml-rid="bib54" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref54-2" aria-controls="sref54-2">Laskowski et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref54-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">54.</div><div>Laskowski, R.A. ∙ Tyagi, N. ∙ Johnson, D. ..., DDD Study</div><div><strong>Integrating population variation and protein structural analysis to improve clinical interpretation of missense variation: application to the WD40 domain</strong></div><div><em>Hum. Mol. Genet.</em> 2016; <strong>25</strong>:927-935</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/hmg/ddv625" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-84962815246" target="_blank" aria-label="Open Scopus in new tab">Scopus (12)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26740553/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fhmg%2Fddv625&amp;pmid=26740553" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), suggestive of its biological relevance. The <i>Tbl1xr1</i> locus was engineered for conditional expression of D370Y (<a id="crosref0365" href="#figs1">Figures S1</a>A and S1B). Because <i>TBL1XR1</i><sup>MUT</sup> lymphomas are derived from GC or post-GC B cells (<span class="dropBlock reference-citations"><a id="body-ref-sref16-6" href="#" role="doc-biblioref" data-xml-rid="bib16" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref16-6" aria-controls="sref16-6">Chapuy et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref16-6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">16.</div><div>Chapuy, B. ∙ Stewart, C. ∙ Dunford, A.J. ...</div><div><strong>Molecular subtypes of diffuse large B cell lymphoma are associated with distinct pathogenic mechanisms and outcomes</strong></div><div><em>Nat. Med.</em> 2018; <strong>24</strong>:679-690</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41591-018-0016-8" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-85046087353" target="_blank" aria-label="Open Scopus in new tab">Scopus (257)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29713087/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41591-018-0016-8&amp;pmid=29713087" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), we crossed <i>Tbl1xr1</i><sup><i>D370Y</i></sup> mice to the <i>C</i>γ<i>1Cre</i> strain (<span class="dropBlock reference-citations"><a id="body-ref-sref12" href="#" role="doc-biblioref" data-xml-rid="bib12" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref12" aria-controls="sref12">Casola et&nbsp;al., 2006</a><div class="dropBlock__holder" data-db-target-of="sref12"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">12.</div><div>Casola, S. ∙ Cattoretti, G. ∙ Uyttersprot, N. ...</div><div><strong>Tracking germinal center B cells expressing germ-line immunoglobulin gamma1 transcripts by conditional gene targeting</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2006; <strong>103</strong>:7396-7401</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.0602353103" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-33646555754" target="_blank" aria-label="Open Scopus in new tab">Scopus (114)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16651521/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.0602353103&amp;pmid=16651521" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) to restrict D370Y expression to (pre)GCB and GC-derived cells. Sorted GCB manifested the correct <i>Tbl1xr1</i> genotype (<a id="crosref0380" href="#figs1">Figure&nbsp;S1</a>C) and showed no difference in overall <i>Tbl1xr1</i> mRNA or protein abundance (<a id="crosref0385" href="#figs1">Figures S1</a>D and S1E).</div><div class="figure-wrap"><figure id="figs1" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2020.05.049/asset/f1df23bf-f28b-4de7-bda3-aee2919dd160/main.assets/figs1_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs1" src="/cms/10.1016/j.cell.2020.05.049/asset/887da720-0b0a-45b0-b2da-60e17e50e74e/main.assets/figs1.jpg" height="903" width="639" aria-labelledby="figs1-title figs1-content" data-viewer-src="/cms/10.1016/j.cell.2020.05.049/asset/f1df23bf-f28b-4de7-bda3-aee2919dd160/main.assets/figs1_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs1-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S1</span> <span class="figure__title__text"><i>Tbl1xr1</i>-D370Y and&nbsp;<i>Tbl1xr1</i>-KO Mouse Models, Related to <a id="crosref2425" href="#fig1">Figure&nbsp;1</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs1-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0380" role="paragraph">A, Schematic depiction of the targeted allele in the <i>Tbl1xr1</i>-D370Y mouse model, and the effects of Cre-induced recombination. “E”&nbsp;= Exon; “Neo”&nbsp;= Neomycin resistance cassette. Thin black arrows depict exon orientation.</div><div id="fspara0385" role="paragraph">B, (Top) Scheme depicting the PCR-based strategy used for genotyping the targeted allele in the <i>Tbl1xr1</i>-D370Y mouse model. Black arrows represent primers. (Bottom) Representative PCR-based genotyping results on genomic DNA extracted from tail tissue from <i>Tbl1xr1</i><sup><i>WT/WT</i></sup> or <i>Tbl1xr1</i><sup><i>D370Y/WT</i></sup> animals. Numbers designate different animals.</div><div id="fspara0390" role="paragraph">C, Sanger sequencing analysis of <i>Tbl1xr1</i> cDNA in sorted splenic GCB from <i>Tbl1xr1</i><sup><i>D370Y/WT</i></sup> or <i>Tbl1xr1</i><sup><i>WT/WT</i></sup> mice, 8&nbsp;days after SRBC immunization. Data shown for one animal per genotype, representative of a total of three.</div><div id="fspara0395" role="paragraph">D, Quantitative Real Time-PCR analysis of <i>Tbl1xr1</i> mRNA expression levels in <i>Tbl1xr1</i><sup><i>D370Y/WT</i></sup> and <i>Tbl1xr1</i><sup><i>WT/WT</i></sup> sorted GCB, 8&nbsp;days after SRBC immunization. Primers were designed to target <i>Tbl1xr1</i> exons 11-13. Results are expressed as fold change, relative to <i>Tbl1xr1</i><sup><i>WT/WT</i></sup>, and normalized to GAPDH expression levels.</div><div id="fspara0400" role="paragraph">E, WB analysis of TBL1XR1 expression levels in sorted GCB from three representative <i>Tbl1xr1</i><sup><i>D370Y/WT</i></sup> and <i>Tbl1xr1</i><sup><i>WT/WT</i></sup> animals, obtained as in (C). ACTIN protein expression levels were used as a loading control.</div><div id="fspara0405" role="paragraph">F, Schematic depiction of the targeted allele in the <i>Tbl1xr1</i>-KO mouse model, and the effects of Cre-induced recombination.</div><div id="fspara0410" role="paragraph">(G), (Top) Scheme depicting the PCR-based strategy used for verifying Cre-induced <i>Tbl1xr1</i> exon 5 excision in the <i>Tbl1xr1</i>-KO model. Thin black arrows represent primers. (Bottom) Representative PCR results on genomic DNA from sorted GCB, from <i>Tbl1xr1</i><sup><i>WT/WT</i></sup> or <i>Tbl1xr1</i><sup><i>KO/KO</i></sup> animals, 8&nbsp;days after SRBC immunization. Numbers designate different animals.</div><div id="fspara0415" role="paragraph">H, Quantitative Real Time-PCR analysis of <i>Tbl1xr1</i> mRNA expression levels in GCB obtained as in (G). Primers were designed to target <i>Tbl1xr1</i> exons 11-13. Results are expressed as fold change, relative to <i>Tbl1xr1</i><sup><i>WT/WT</i></sup> GCB, and normalized to GAPDH expression levels.</div><div id="fspara0420" role="paragraph">I, WB analysis of TBL1XR1 expression levels in sorted GCB from two representative <i>Tbl1xr1</i><sup><i>KO/KO</i></sup> and <i>Tbl1xr1</i><sup><i>WT/WT</i></sup> animals, obtained as in (G). ACTIN protein expression levels were used as a loading control.</div><div id="fspara0425" role="paragraph">J-K, Representative FC plots (left) and quantification (right) of (J) total splenic B cells or (K) GCB in Cγ1Cre-<i>Tbl1xr1</i><sup><i>KO/KO</i></sup> (<i>Tbl1xr1</i><sup><i>KO/KO</i></sup>) or <i>Tbl1xr1</i><sup><i>WT/WT</i></sup> mice. See also <a id="crosref2430" href="#mmc6">Data S1</a>N.</div><div id="fspara0430" role="paragraph">L, Absolute FC-based quantification of (left) total splenic B cells or (right) GCB from <i>Tbl1xr1</i><sup><i>KO/KO</i></sup> or <i>Tbl1xr1</i><sup><i>WT/WT</i></sup> mice, 9&nbsp;days after SRBC immunization.</div><div id="fspara0435" role="paragraph">M, Representative H&amp;E images of splenic sections taken from animals treated as in (J). Insets (bottom) show zoom of outlined areas. Scale bars&nbsp;= 500μm (top) or 100μm (bottom).</div><div id="fspara0440" role="paragraph">N, Representative images of (top) B220 or (bottom) PNA IHC staining in consecutive splenic sections from specimens in (M). Scale bars&nbsp;= 100μm.</div><div id="fspara0445" role="paragraph">O-P, Graphs show quantification of (O) the number of GC per spleen section, or (P) GC size as number of cells (left) or surface area (right), based on PNA staining. Dots in (A) represent different animals. Dots in (P) represent individual GC. Shown are pooled results for 4 animals per genotype.</div><div id="fspara0450" role="paragraph">(Q), Representative FC plots (left) and quantification (right) of the relative contribution of <i>Tbl1xr1</i><sup><i>KO/KO</i></sup> and <i>Tbl1xr1</i><sup><i>WT/WT</i></sup> cells to the total B cell and GCB compartments, based on CD45 allelic frequencies.</div><div id="fspara0455" role="paragraph">R, FC analysis of splenic GCB frequency. Left to right: n&nbsp;= 3, 4, 5, 5 (<i>Tbl1xr1</i><sup><i>WT/WT</i></sup>) or 3, 4, 5, 5 mice (<i>Tbl1xr1</i><sup><i>KO/KO</i></sup>).</div><div id="fspara0460" role="paragraph">Values represent mean ± SEM. All data were reproducible with at least two repeats. NS, not significant; <sup>∗</sup>p&nbsp;&lt; 0.05; <sup>∗∗</sup>p&nbsp;&lt; 0.01; <sup>∗∗∗</sup>p&nbsp;&lt; 0.001, using unpaired (D,H,J-L,R) or paired (Q) two-tailed Student’s t test, or Mann-Whitney U-test (O-P).</div></div></div></div></figcaption></figure></div><div id="p0055" role="paragraph">Given a previous report suggesting that TBL1XR1 might limit SMRT functionality (<span class="dropBlock reference-citations"><a id="body-ref-sref80-2" href="#" role="doc-biblioref" data-xml-rid="bib80" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref80-2" aria-controls="sref80-2">Perissi et&nbsp;al., 2004</a><div class="dropBlock__holder" data-db-target-of="sref80-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">81.</div><div>Perissi, V. ∙ Aggarwal, A. ∙ Glass, C.K. ...</div><div><strong>A corepressor/coactivator exchange complex required for transcriptional activation by nuclear receptors and other regulated transcription factors</strong></div><div><em>Cell.</em> 2004; <strong>116</strong>:511-526</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=10.1016%2FS0092-8674%2804%2900133-3&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=10.1016%2FS0092-8674%2804%2900133-3&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-1342264315" target="_blank" aria-label="Open Scopus in new tab">Scopus (412)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/14980219/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0092-8674%2804%2900133-3&amp;pmid=14980219" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), we hypothesized that TBL1XR1<sup>MUT</sup> would further enhance the activity of BCL6-SMRT complexes. We immunized <i>Tbl1xr1</i><sup><i>D370Y/WT</i></sup> (D370Y/WT) or <i>Tbl1xr1</i><sup><i>WT/WT</i></sup> (wild-type [WT]) mice with the T&nbsp;cell-dependent antigen sheep red blood cells (SRBC) and sacrificed them at the peak of the GC reaction. Unexpectedly, although B cell abundance remained unaltered in D370Y/WT mice (<a id="crosref0395" href="#fig1">Figure&nbsp;1</a>C; <a id="crosref0400" href="#mmc6">Data S1</a>D and S1E), these animals showed significant decrease in the absolute number (p value&nbsp;= 0.0008; <a id="crosref0405" href="#mmc6">Data S1</a>E) and proportion (p value&nbsp;=&nbsp;0.0006; <a id="crosref0410" href="#fig1">Figure&nbsp;1</a>D) of FAS<sup>+</sup>GL7<sup>+</sup> GCB. Similar results were observed gating GCB as FAS<sup>+</sup>CD38<sup>−</sup> (<a id="crosref0415" href="#mmc6">Data S1</a>F). Animals showed no disruption of splenic architecture (<a id="crosref0420" href="#fig1">Figure&nbsp;1</a>E). Immunohistochemistry (IHC) staining for B cells (B220) and GCB (peanut agglutinin, PNA) revealed normal follicular structures with smaller GCs in D370Y/WT mice, but conserved GC numbers per spleen (<a id="crosref0425" href="#fig1">Figures 1</a>F–1H). Dark zone (DZ) centroblasts (CB) and light zone (LZ) centrocytes (CC) ratios were similar to WT (<a id="crosref0430" href="#mmc6">Data S1</a>G).</div><div id="p0060" role="paragraph">Although D370Y/WT GC were smaller, it is possible that these GCB could be fitter when in direct competition. We performed mixed chimera experiments where equal numbers of <i>Cd45.2;Tbl1xr1</i><sup><i>D370Y/WT</i></sup> or <i>Cd45.1/2;Tbl1xr1</i><sup><i>WT/WT</i></sup> bone marrow (BM) cells were transplanted into lethally irradiated syngeneic recipients. Following immunization, the relative proportions of <i>Cd45.1</i> and <i>Cd45.1/2</i> B220<sup>+</sup> cells were roughly equal, but D370Y/WT GCB manifested significant disadvantage compared to WT (<a id="crosref0435" href="#fig1">Figure&nbsp;1</a>I), regardless of the initial ratio of transferred cells (<a id="crosref0440" href="#mmc6">Data S1</a>H). Immunization with hapten-protein conjugates yielded similar results (<a id="crosref0445" href="#mmc6">Data S1</a>I). Observations were also not dependent on the <i>Cd45</i> allelic variant, because identical results were obtained with <i>Cd45.1;Tbl1xr1</i><sup><i>D370Y/WT</i></sup> (<a id="crosref0450" href="#mmc6">Data S1</a>J).</div><div id="p0065" role="paragraph">To study the possibility of a premature/delayed GC burst as explanation for the reduced GC size, we crossed D370Y/WT mice to include a <i>Rosa26lox-stop-loxYFP</i> (<i>Rosa26YFP</i>) allele, which reports on Cre activity (<a id="crosref0455" href="#fig1">Figure&nbsp;1</a>J; <a id="crosref0460" href="#mmc6">Data S1</a>K). D370Y/WT mice presented reduced GCB abundance at all time points (<a id="crosref0465" href="#fig1">Figure&nbsp;1</a>K; <a id="crosref0470" href="#mmc6">Data S1</a>L), and GCs completely resolved by day 25. Collectively, these data indicate that <i>Tbl1xr1</i> mutation impairs, rather than enhances, the GC reaction (perhaps by hindering BCL6 function), while maintaining normal kinetics.</div></section><section id="sec-2-2"><h3><i>Tbl1xr1</i> Mutations Phenocopy TBL1XR1 Complete Loss</h3><div id="p0070" role="paragraph">The absence of a clear hotspot suggests that <i>TBL1XR1</i> mutations behave as loss-of-function (LOF). Along these lines, focal <i>TBL1XR1</i> deletions are also detected in DLBCL patients (<a id="crosref0475" href="#mmc6">Data S1</a>M) (<span class="dropBlock reference-citations"><a id="body-ref-sref92-5" href="#" role="doc-biblioref" data-xml-rid="bib92" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref92-5" aria-controls="sref92-5">Schmitz et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref92-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">90.</div><div>Schmitz, R. ∙ Wright, G.W. ∙ Huang, D.W. ...</div><div><strong>Genetics and Pathogenesis of Diffuse Large B-Cell Lymphoma</strong></div><div><em>N.&nbsp;Engl. J. Med.</em> 2018; <strong>378</strong>:1396-1407</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1056/NEJMoa1801445" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_90_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-85045505659" target="_blank" aria-label="Open Scopus in new tab">Scopus (314)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29641966/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1056%2FNEJMoa1801445&amp;pmid=29641966" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), albeit at lower frequency than missense mutations. We then generated a <i>C</i>γ<i>1Cre</i>-inducible <i>Tbl1xr1</i> knockout (KO) mouse (<a id="crosref0485" href="#figs1">Figures S1</a>F–S1I) and again observed that although total B cell abundance was unchanged (<a id="crosref0490" href="#figs1">Figures S1</a>J and S1L), there was significant GCB reduction in immunized <i>Tbl1xr1</i><sup><i>KO/KO</i></sup> (KO/KO) mice (<a id="crosref0495" href="#figs1">Figures S1</a>K and S1L; <a id="crosref0500" href="#mmc6">Data S1</a>N). There was no disruption of splenic architecture or lymphoid follicles (<a id="crosref0505" href="#figs1">Figure&nbsp;S1</a>M), although there was significant reduction in GC size, but not GC number (<a id="crosref0510" href="#figs1">Figures S1</a>N–S1P). Mixed chimera experiments confirmed that KO/KO GCB are at disadvantage against WT (<a id="crosref0515" href="#figs1">Figure&nbsp;S1</a>Q). The kinetics of the GC reaction in KO/KO mice were similar to WT, but were again consistently hypoplastic (<a id="crosref0520" href="#figs1">Figure&nbsp;S1</a>R). Finally, KO/KO GCs showed normal ratios of LZ-DZ GCB (<a id="crosref0525" href="#mmc6">Data S1</a>O).</div><div id="p0075" role="paragraph">The fact that KO/KO phenocopies D370Y/WT supports the idea that <i>Tbl1xr1</i> mutations confer LOF and suggests mutations may have a dominant negative (DN) character. Along these lines, GC formation appeared normal in heterozygous <i>Tbl1xr1</i><sup><i>KO/WT</i></sup> mice (<a id="crosref0530" href="#fig1">Figure&nbsp;1</a>L) and did not recapitulate the phenotype induced by D370Y/WT, which instead was virtually identical to KO/KO. This strongly suggests that <i>TBL1XR1</i> mutants function as DN LOF alleles.</div></section><section id="sec-2-3"><h3><i>Tbl1xr1</i> Mutation Impairs GC Proliferation</h3><div id="p0080" role="paragraph">Apoptosis is prevalent in the GC, with up to half GCBs dying every 6&nbsp;h (<span class="dropBlock reference-citations"><a id="body-ref-sref63" href="#" role="doc-biblioref" data-xml-rid="bib63" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref63" aria-controls="sref63">Mayer et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref63"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">63.</div><div>Mayer, C.T. ∙ Gazumyan, A. ∙ Kara, E.E. ...</div><div><strong>The microanatomic segregation of selection by apoptosis in the germinal center</strong></div><div><em>Science.</em> 2017; <strong>358</strong>:eaao2602</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aao2602" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-85029760325" target="_blank" aria-label="Open Scopus in new tab">Scopus (57)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28935768/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aao2602&amp;pmid=28935768" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). D370Y/WT manifested similar frequency of apoptotic/dead cells to WT GCB, by AnnexinV/DAPI FC (<a id="crosref0540" href="#fig2">Figure&nbsp;2</a>A; <a id="crosref0545" href="#mmc6">Data S1</a>P) or TUNEL staining (<a id="crosref0550" href="#fig2">Figure&nbsp;2</a>B). Similar findings were observed for KO/KO (<a id="crosref0555" href="#figs2">Figure&nbsp;S2</a>A). On the other hand, there was significant reduction in D370Y/WT GCB positive for the proliferation marker PCNA (<a id="crosref0560" href="#fig2">Figure&nbsp;2</a>C). Abundance of T follicular helper cells (T<sub>FH</sub>), which express CD40 ligand and induce GCB proliferation, was comparable in mutant and WT mice (<a id="crosref0565" href="#figs2">Figure&nbsp;S2</a>B). To assess for intrinsic defects in response to CD40 stimulation, we performed <i>ex&nbsp;vivo</i> activation of mixed D370Y/WT and WT naive B (NB) cells in organoid cultures (<span class="dropBlock reference-citations"><a id="body-ref-sref9-1" href="#" role="doc-biblioref" data-xml-rid="bib9" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref9-1" aria-controls="sref9-1">Béguelin et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref9-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">9.</div><div>Béguelin, W. ∙ Rivas, M.A. ∙ Calvo Fernández, M.T. ...</div><div><strong>EZH2 enables germinal centre formation through epigenetic silencing of CDKN1A and an Rb-E2F1 feedback loop</strong></div><div><em>Nat. Commun.</em> 2017; <strong>8</strong>:877</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41467-017-01029-x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-85031291005" target="_blank" aria-label="Open Scopus in new tab">Scopus (48)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29026085/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41467-017-01029-x&amp;pmid=29026085" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) (<a id="crosref0575" href="#figs2">Figure&nbsp;S2</a>C). D370Y/WT and WT B cells acquired GC surface markers to approximately the same extent (<a id="crosref0580" href="#figs2">Figure&nbsp;S2</a>D), without significant differences in proliferation rate (<a id="crosref0585" href="#figs2">Figure&nbsp;S2</a>E). Hence, proliferation of D370Y/WT B cells is not impaired when given unrestricted CD40 stimulation.</div><div class="figure-wrap"><figure id="fig2" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2020.05.049/asset/bcffcc65-cfc5-4d55-abcb-fa166d278d30/main.assets/gr2_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr2" src="/cms/10.1016/j.cell.2020.05.049/asset/e707e4a6-c241-4140-ba22-f22d6f9923e3/main.assets/gr2.jpg" height="743" width="765" aria-labelledby="fig2-title fig2-content" data-viewer-src="/cms/10.1016/j.cell.2020.05.049/asset/bcffcc65-cfc5-4d55-abcb-fa166d278d30/main.assets/gr2_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig2-title" class="figure__title"><span class="label figure__label">Figure&nbsp;2</span> <span class="figure__title__text"><i>Tbl1xr1</i> Mutation Impairs GCB Proliferation</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig2-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0075" role="paragraph">(A) FC analysis of AnnexinV/DAPI staining of splenic GCB. See also <a id="crosref2310" href="#mmc6">Data S1</a>P.</div><div id="fspara0080" role="paragraph">(B) IHC staining in consecutive spleen sections. Contour delineates GC area based on PNA stain. Insets show zoom of outlined areas. Arrows point representative TUNEL<sup>+</sup> GCB. Graph shows frequency of TUNEL<sup>+</sup> GCB per GC, where each dot is a GC. Results for 5 animals per genotype. Scale, 50&nbsp;μm (left and center), 10&nbsp;μm (right).</div><div id="fspara0085" role="paragraph">(C) FC analysis of splenic PCNA<sup>+</sup> GCB. NB from a WT mouse illustrates non-proliferating cells.</div><div id="fspara0090" role="paragraph">(D) FC analysis of EdU incorporation by splenic GCB. NB from a WT mouse illustrates non-proliferating cells.</div><div id="fspara0095" role="paragraph">(E) FC analysis of splenic GCB cell cycle in (D), based on EdU/DAPI staining. Data for n&nbsp;= 5 per genotype.</div><div id="fspara0100" role="paragraph">(F) Use of the <i>R26-Fucci2aR</i> reporter.</div><div id="fspara0105" role="paragraph">(G and H) FC analysis of cell-cycle distribution of (G) CC or (H) CB. See also <a id="crosref2315" href="#mmc6">Data S1</a>Q.</div><div id="fspara0110" role="paragraph">Values represent mean ± SEM. Data reproducible with two repeats. p values calculated using unpaired (A, C, D, G, and H) or paired (D and E) two-tailed Student’s t test, or Mann-Whitney U-test (B).</div></div></div></div></figcaption></figure></div><div class="figure-wrap"><figure id="figs2" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2020.05.049/asset/4ef57c5a-a5d7-4cdd-acce-01d51508b91f/main.assets/figs2_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs2" src="/cms/10.1016/j.cell.2020.05.049/asset/f1708978-4022-497e-8836-7c1de5107ea6/main.assets/figs2.jpg" height="950" width="662" aria-labelledby="figs2-title figs2-content" data-viewer-src="/cms/10.1016/j.cell.2020.05.049/asset/4ef57c5a-a5d7-4cdd-acce-01d51508b91f/main.assets/figs2_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs2-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S2</span> <span class="figure__title__text"><i>Tbl1xr1</i>-D370Y and&nbsp;<i>Tbl1xr1</i>-KO GCB Proliferation and Cell Death, Related to <a id="crosref2435" href="#fig2">Figure&nbsp;2</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs2-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0470" role="paragraph">A, Representative FC plots (left) and quantification (right) of AnnexinV/DAPI staining in splenic GCB.</div><div id="fspara0475" role="paragraph">B, Representative FC plots and quantification of (top) total CD4<sup>+</sup> or (bottom) GC T follicular helper (CD4<sup>+</sup>CXCR5<sup>hi</sup>PD-1<sup>hi</sup>) splenic populations.</div><div id="fspara0480" role="paragraph">C, Experimental scheme for experiment in D-E.</div><div id="fspara0485" role="paragraph">D, Representative FC plots of (left) GCB formation and (right) the relative contribution of <i>Tbl1xr1</i><sup><i>D370Y/W</i>T</sup> and <i>Tbl1xr1</i><sup><i>WT/WT</i></sup> cells to this population, at the indicated time-points.</div><div id="fspara0490" role="paragraph">E, Representative FC plots (left) and quantification (right) of GCB proliferation, based on progressive dilution of a cell proliferation dye. Results are expressed as the frequency of GCB that have undergone a certain number of cell divisions, for three biological replicates, with 3 technical replicates each. Representative plots of stain intensity at Day 0 are included, as a control for initial dye loading levels.</div><div id="fspara0495" role="paragraph">F, Representative FC plots (left) and quantification (right) of EdU incorporation by GCB.</div><div id="fspara0500" role="paragraph">G, FC analysis of cell cycle distribution of splenic GCB for animals in (F), based on EdU/DAPI staining.</div><div id="fspara0505" role="paragraph">H, RT-qPCR-based quantification of cell cycle checkpoint genes performed on DZ GCB sorted 9&nbsp;days after SRBC immunization. Results are expressed as fold change of gene-of-interest (G.O.I.), relative to the corresponding <i>Tbl1xr1</i><sup><i>WT/WT</i></sup> controls, and normalized to GAPDH expression levels</div><div id="fspara0510" role="paragraph">Values represent mean ± SEM. All the data were reproducible with at least two repeats. NS, not significant; <sup>∗</sup>p&nbsp;&lt; 0.05; <sup>∗∗</sup>p&nbsp;&lt; 0.01; <sup>∗∗∗</sup>p&nbsp;&lt; 0.001, using unpaired (A,B,H) or paired (F-G) two-tailed Student’s t test, or two-way ANOVA (E).</div></div></div></div></figcaption></figure></div><div id="p0085" role="paragraph">To assess cell-cycle kinetics, we tested ethynyldeoxyuridine (EdU) incorporation in immunized D370Y/WT or WT mice. In line with PCNA staining, EdU<sup>+</sup> GCB were significantly depleted in D370Y/WT (<a id="crosref0590" href="#fig2">Figure&nbsp;2</a>D), with a significant increase in GCB at the G0-1 stage, and reciprocal reduction in S and G2/M (<a id="crosref0595" href="#fig2">Figure&nbsp;2</a>E). Similar results were observed for KO/KO (<a id="crosref0600" href="#figs2">Figures S2</a>F and S2G). As an orthogonal approach, D370Y/WT animals were crossed with <i>R26-Fucci2aR</i> reporter mice (<a id="crosref0605" href="#fig2">Figure&nbsp;2</a>F; <a id="crosref0610" href="#mmc6">Data S1</a>Q). In line with previous studies (<span class="dropBlock reference-citations"><a id="body-ref-sref100" href="#" role="doc-biblioref" data-xml-rid="bib100" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref100" aria-controls="sref100">Stewart et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref100"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">98.</div><div>Stewart, I. ∙ Radtke, D. ∙ Phillips, B. ...</div><div><strong>Germinal Center B Cells Replace Their Antigen Receptors in Dark Zones and Fail Light Zone Entry when Immunoglobulin Gene Mutations are Damaging</strong></div><div><em>Immunity.</em> 2018; <strong>49</strong>:477-489</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_98_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=10.1016%2Fj.immuni.2018.08.025&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_98_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=10.1016%2Fj.immuni.2018.08.025&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_98_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-85053143886" target="_blank" aria-label="Open Scopus in new tab">Scopus (17)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30231983/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.immuni.2018.08.025&amp;pmid=30231983" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), most CC in WT mice were at G0-1, and D370Y/WT had no significant impact on cell-cycle distribution (<a id="crosref0620" href="#fig2">Figure&nbsp;2</a>G). As expected, WT CB showed high abundance of G2/M cells and fewer cells in G0-1 (<a id="crosref0625" href="#fig2">Figure&nbsp;2</a>H). In contrast, D370Y/WT CB manifested significant reduction in the proportion of G2/M and late M cells, as well as increase in G0-1 (<a id="crosref0630" href="#fig2">Figure&nbsp;2</a>H). BCL6 represses cell-cycle checkpoint genes in GCB (<span class="dropBlock reference-citations"><a id="body-ref-sref34-2" href="#" role="doc-biblioref" data-xml-rid="bib34" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref34-2" aria-controls="sref34-2">Hatzi and Melnick, 2014</a><div class="dropBlock__holder" data-db-target-of="sref34-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">34.</div><div>Hatzi, K. ∙ Melnick, A.</div><div><strong>Breaking bad in the germinal center: how deregulation of BCL6 contributes to lymphomagenesis</strong></div><div><em>Trends Mol. Med.</em> 2014; <strong>20</strong>:343-352</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=10.1016%2Fj.molmed.2014.03.001&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=10.1016%2Fj.molmed.2014.03.001&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-84901585384" target="_blank" aria-label="Open Scopus in new tab">Scopus (80)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24698494/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.molmed.2014.03.001&amp;pmid=24698494" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), but D370Y/WT or KO/KO CC showed no induction of these genes (<a id="crosref0640" href="#figs2">Figure&nbsp;S2</a>H), suggesting that proliferation effects are driven by a different mechanism. Impairment in proliferation largely explains the reduced abundance of GCB in <i>Tbl1xr1</i> mutant and KO/KO mice.</div></section><section id="sec-2-4"><h3><i>Tbl1xr1</i> Mutation Induces ABC-DLBCL-like Signatures Suggestive of Antagonism with BCL6</h3><div id="p0090" role="paragraph">Although most DLBCL mutations engineered in mice induce GC hyperplasia (<span class="dropBlock reference-citations"><a id="body-ref-sref8" href="#" role="doc-biblioref" data-xml-rid="bib8" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref8" aria-controls="sref8">Béguelin et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref8"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">8.</div><div>Béguelin, W. ∙ Popovic, R. ∙ Teater, M. ...</div><div><strong>EZH2 is required for germinal center formation and somatic EZH2 mutations promote lymphoid transformation</strong></div><div><em>Cancer Cell.</em> 2013; <strong>23</strong>:677-692</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=10.1016%2Fj.ccr.2013.04.011&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=10.1016%2Fj.ccr.2013.04.011&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-84877815031" target="_blank" aria-label="Open Scopus in new tab">Scopus (409)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23680150/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.ccr.2013.04.011&amp;pmid=23680150" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref26" href="#" role="doc-biblioref" data-xml-rid="bib26" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref26" aria-controls="sref26">Dominguez et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref26"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">26.</div><div>Dominguez, P.M. ∙ Ghamlouch, H. ∙ Rosikiewicz, W. ...</div><div><strong>TET2 Deficiency Causes Germinal Center Hyperplasia, Impairs Plasma Cell Differentiation, and Promotes B-cell Lymphomagenesis</strong></div><div><em>Cancer Discov.</em> 2018; <strong>8</strong>:1632-1653</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1158/2159-8290.CD-18-0657" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_26_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-85058063303" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30274972/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1158%2F2159-8290.CD-18-0657&amp;pmid=30274972" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref33" href="#" role="doc-biblioref" data-xml-rid="bib33" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref33" aria-controls="sref33">Hashwah et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref33"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">33.</div><div>Hashwah, H. ∙ Schmid, C.A. ∙ Kasser, S. ...</div><div><strong>Inactivation of CREBBP expands the germinal center B cell compartment, down-regulates MHCII expression and promotes DLBCL growth</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2017; <strong>114</strong>:9701-9706</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1619555114" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-85029224086" target="_blank" aria-label="Open Scopus in new tab">Scopus (32)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28831000/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1619555114&amp;pmid=28831000" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref36-1" href="#" role="doc-biblioref" data-xml-rid="bib36" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref36-1" aria-controls="sref36-1">Hatzi et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref36-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">36.</div><div>Hatzi, K. ∙ Geng, H. ∙ Doane, A.S. ...</div><div><strong>Histone demethylase LSD1 is required for germinal center formation and BCL6-driven lymphomagenesis</strong></div><div><em>Nat. Immunol.</em> 2019; <strong>20</strong>:86-96</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41590-018-0273-1" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-85058423458" target="_blank" aria-label="Open Scopus in new tab">Scopus (14)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30538335/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41590-018-0273-1&amp;pmid=30538335" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), the <i>Tbl1xr1</i> phenotype was opposite and counter-intuitive as a lymphoma lesion. To gain mechanistic insight, we performed RNA sequencing (RNA-seq) in sorted D370Y/WT GCB. Unsupervised analysis revealed that D370Y/WT induces a distinct transcriptome (<a id="crosref0665" href="#figs3">Figures S3</a>A and S3B). Supervised analysis identified a D370Y/WT specific signature (|log<sub>2</sub>FC| &gt; 1.5; false discovery rate [FDR] &lt;0.05), skewed toward gene de-repression (<a id="crosref0670" href="#fig3">Figure&nbsp;3</a>A; <a id="crosref0675" href="#mmc2">Table S2</a>). Pathway analysis revealed enrichment for ABC-DLBCL associated signatures (<a id="crosref0680" href="#fig3">Figure&nbsp;3</a>B; <a id="crosref0685" href="#mmc2">Table S2</a>), as well as pathways involved in chemotaxis, immune cytokine, and NF-κB signaling, all of which are repressed in GCB by BCL6-SMRT (<a id="crosref0690" href="#fig3">Figure&nbsp;3</a>B). Accordingly, we observed significant enrichment for genes repressed through BCL6-SMRT enhancer binding, genes induced by BCL6 small interfering RNA (siRNA) and CREBBP target genes (CREBBP normally reverses BCL6-SMRT effects in the LZ) (<span class="dropBlock reference-citations"><a id="body-ref-sref43-1" href="#" role="doc-biblioref" data-xml-rid="bib43" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref43-1" aria-controls="sref43-1">Jiang et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref43-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">43.</div><div>Jiang, Y. ∙ Ortega-Molina, A. ∙ Geng, H. ...</div><div><strong>CREBBP Inactivation Promotes the Development of HDAC3-Dependent Lymphomas</strong></div><div><em>Cancer Discov.</em> 2017; <strong>7</strong>:38-53</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1158/2159-8290.CD-16-0975" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-85009410568" target="_blank" aria-label="Open Scopus in new tab">Scopus (92)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27733359/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1158%2F2159-8290.CD-16-0975&amp;pmid=27733359" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) (<a id="crosref0700" href="#fig3">Figure&nbsp;3</a>B). This suggests that TBL1XR1 plays a critical role in supporting the function of the BCL6-SMRT complex. Repression of EBI2 (<i>GPR183</i>) and S1PR1 by BCL6 is largely responsible for GCB confinement to lymphoid follicles (<span class="dropBlock reference-citations"><a id="body-ref-sref3" href="#" role="doc-biblioref" data-xml-rid="bib3" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref3" aria-controls="sref3">Arnon et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">3.</div><div>Arnon, T.I. ∙ Horton, R.M. ∙ Grigorova, I.L. ...</div><div><strong>Visualization of splenic marginal zone B-cell shuttling and follicular B-cell egress</strong></div><div><em>Nature.</em> 2013; <strong>493</strong>:684-688</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature11738" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-84873088550" target="_blank" aria-label="Open Scopus in new tab">Scopus (126)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23263181/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature11738&amp;pmid=23263181" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref79" href="#" role="doc-biblioref" data-xml-rid="bib79" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref79" aria-controls="sref79">Pereira et&nbsp;al., 2009</a><div class="dropBlock__holder" data-db-target-of="sref79"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">80.</div><div>Pereira, J.P. ∙ Kelly, L.M. ∙ Xu, Y. ...</div><div><strong>EBI2 mediates B cell segregation between the outer and centre follicle</strong></div><div><em>Nature.</em> 2009; <strong>460</strong>:1122-1126</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature08226" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_80_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-69349087555" target="_blank" aria-label="Open Scopus in new tab">Scopus (203)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19597478/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature08226&amp;pmid=19597478" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), whereas they reactivate in cells exiting the GC. D370Y/WT GCB upregulated <i>Gpr183</i> and <i>S1pr1</i> (<a id="crosref0715" href="#fig3">Figures 3</a>A, <a id="crosref0720" href="#figs3">S3</a>C, and S3D), raising the possibility that TBL1XR1<sup>MUT</sup> drive GCB to manifest post-GC phenotypes.</div><div class="figure-wrap"><figure id="fig3" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2020.05.049/asset/b54dce69-f269-424f-b214-3d3a48f3c801/main.assets/gr3_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr3" src="/cms/10.1016/j.cell.2020.05.049/asset/30b177b1-f775-42e3-8c66-19ad2923efd4/main.assets/gr3.jpg" height="905" width="770" aria-labelledby="fig3-title fig3-content" data-viewer-src="/cms/10.1016/j.cell.2020.05.049/asset/b54dce69-f269-424f-b214-3d3a48f3c801/main.assets/gr3_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig3-title" class="figure__title"><span class="label figure__label">Figure&nbsp;3</span> <span class="figure__title__text"><i>Tbl1xr1</i> Mutation Induces PreMB Expansion</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig3-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0120" role="paragraph">(A) Differentially expressed genes in splenic D370Y/WT GCB. Genes of interest are highlighted. See also <a id="crosref2320" href="#figs3">Figures S3</a>A–S3C and <a id="crosref2325" href="#mmc2">Table S2</a>.</div><div id="fspara0125" role="paragraph">(B) Pathway enrichment for genes in (A). See also <a id="crosref2330" href="#mmc2">Table S2</a>.</div><div id="fspara0130" role="paragraph">(C and D) FC analysis of (C) CCR6<sup>+</sup> or (D) IL-9R<sup>+</sup> splenic GCB.</div><div id="fspara0135" role="paragraph">(E) GSEA of D370Y/WT GCB against preMB (EFNB1<sup>+</sup>S1PR2<sup>lo</sup>) GCB (GSE89897). See also <a id="crosref2335" href="#figs3">Figures S3</a>G and S3H.</div><div id="fspara0140" role="paragraph">(F and G) FC analysis of EFNB1<sup>+</sup> and preMB populations in splenic GCB in (F) D370Y/WT.or (G) KO/KO mice, as compared to WT/WT controls. See also <a id="crosref2340" href="#figs3">Figures S3</a>I–S3K and <a id="crosref2345" href="#mmc6">Data S1</a>R.</div><div id="fspara0145" role="paragraph">(H) Experimental scheme and timeline for (I)–(K).</div><div id="fspara0150" role="paragraph">(I) FC analysis of preMB in splenic GCB.</div><div id="fspara0155" role="paragraph">(J and K) FC analysis of splenic (J) total B cells or (K) GCB.</div><div id="fspara0160" role="paragraph">Values represent mean ± SEM. Data reproducible with two repeats. P values calculated using unpaired two-tailed Student’s t test (C, D, F, and G), or one-way ANOVA with Tukey’s post-test (I)–(K).</div></div></div></div></figcaption></figure></div><div class="figure-wrap"><figure id="figs3" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2020.05.049/asset/afac9c27-c17c-448e-a9f8-0eb0599dce89/main.assets/figs3_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs3" src="/cms/10.1016/j.cell.2020.05.049/asset/961f22d5-e0b9-497b-b934-e531fae97422/main.assets/figs3.jpg" height="950" width="679" aria-labelledby="figs3-title figs3-content" data-viewer-src="/cms/10.1016/j.cell.2020.05.049/asset/afac9c27-c17c-448e-a9f8-0eb0599dce89/main.assets/figs3_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs3-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S3</span> <span class="figure__title__text">PreMB Population in&nbsp;<i>Tbl1xr1</i>-D370Y Mice, Related to <a id="crosref2440" href="#fig3">Figure&nbsp;3</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs3-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0520" role="paragraph">A, Hierarchical clustering dendrogram, based on Euclidean distance, using log FPKM values of variable genes (top 5th percentile) and Ward’s method, for experiment in <a id="crosref2445" href="#fig3">Figure&nbsp;3</a>A. Squares represent individual animals</div><div id="fspara0525" role="paragraph">B, PCA plot for <i>Tbl1xr1</i><sup><i>D370Y/WT</i></sup> and <i>Tbl1xr1</i><sup><i>WT/WT</i></sup> GCB RNA-Seq samples from <a id="crosref2450" href="#fig3">Figure&nbsp;3</a>A.</div><div id="fspara0530" role="paragraph">C, Quantitative Real Time-PCR (RT-qPCR) validation of selected differentially expressed genes from <a id="crosref2455" href="#fig3">Figure&nbsp;3</a>A, performed on an independent set of GCB, sorted 8&nbsp;days after SRBC immunization. Results are expressed as fold change of gene-of-interest (G.O.I.), relative to <i>Tbl1xr1</i><sup><i>WT/WT</i></sup>, and normalized to GAPDH expression levels.</div><div id="fspara0535" role="paragraph">D, Representative FC plots (left) and quantification (right) of S1PR1<sup>+</sup> splenic GCB.</div><div id="fspara0540" role="paragraph">E, Representative FC plots (left) and quantification (right) of CCR6<sup>+</sup> splenic GCB.</div><div id="fspara0545" role="paragraph">F, RT-qPCR analysis of <i>Il9r</i> mRNA expression levels in <i>Tbl1xr1</i><sup><i>KO/KO</i></sup> and <i>Tbl1xr1</i><sup><i>WT/WT</i></sup> sorted GCB, 8&nbsp;days after SRBC immunization. Results are expressed as fold change, relative to <i>Tbl1xr1</i><sup><i>WT/WT</i></sup>, and normalized to GAPDH expression levels.</div><div id="fspara0550" role="paragraph">G-H, GSEA of gene signatures (left) upregulated or (right) downregulated in <i>Tbl1xr1</i><sup><i>D370Y/WT</i></sup> versus <i>Tbl1xr1</i><sup><i>WT/WT</i></sup> GCB, ranked against: (G) the preMB (CCR6<sup>+</sup>LZ) GCB signature from (<span class="dropBlock reference-citations"><a id="body-ref-sref101-1" href="#" role="doc-biblioref" data-xml-rid="bib101" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref101-1" aria-controls="sref101-1">Suan et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref101-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">99.</div><div>Suan, D. ∙ Krautler, N.J. ∙ Maag, J.L.V. ...</div><div><strong>CCR6 Defines Memory B Cell Precursors in Mouse and Human Germinal Centers, Revealing Light-Zone Location and Predominant Low Antigen Affinity</strong></div><div><em>Immunity.</em> 2017; <strong>47</strong>:1142-1153</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_99_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=10.1016%2Fj.immuni.2017.11.022&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_99_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=10.1016%2Fj.immuni.2017.11.022&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_99_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-85039037389" target="_blank" aria-label="Open Scopus in new tab">Scopus (50)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29262350/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.immuni.2017.11.022&amp;pmid=29262350" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>); (H) an alternative PreMB (mKO2<sup>hi</sup>) GCB signature from GSE85018.</div><div id="fspara0555" role="paragraph">I, FC analysis of (left) EFNB1<sup>+</sup> or (right) preMB populations in splenic GCB, at (top) 4&nbsp;days or (bottom) 15&nbsp;days post SRBC immunization.</div><div id="fspara0560" role="paragraph">J, FC analysis of (left) EFNB1<sup>+</sup> or (right) preMB populations in splenic GCB from mice immunized with an hapten conjugate.</div><div id="fspara0565" role="paragraph">Values represent mean ± SEM. All data were reproducible with at least two repeats. NS, not significant; <sup>∗</sup>p&nbsp;&lt; 0.05; <sup>∗∗</sup>p&nbsp;&lt; 0.01; <sup>∗∗∗</sup>p&nbsp;&lt; 0.001, using unpaired two-tailed Student’s t test (C)-(F),(I)-(K).</div></div></div></div></figcaption></figure></div></section><section id="sec-2-5"><h3><i>Tbl1xr1</i> Mutation Induces Expansion of the PreMB Compartment</h3><div id="p0095" role="paragraph">D370Y/WT GCB showed upregulation of <i>Cd38</i> (<a id="crosref0725" href="#fig3">Figures 3</a>A and <a id="crosref0730" href="#figs3">S3</a>C), a gene that in mice is upregulated in MB, but not in PC (<span class="dropBlock reference-citations"><a id="body-ref-sref76" href="#" role="doc-biblioref" data-xml-rid="bib76" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref76" aria-controls="sref76">Oliver et&nbsp;al., 1997</a><div class="dropBlock__holder" data-db-target-of="sref76"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">77.</div><div>Oliver, A.M. ∙ Martin, F. ∙ Kearney, J.F.</div><div><strong>Mouse CD38 is down-regulated on germinal center B cells and mature plasma cells</strong></div><div><em>J.&nbsp;Immunol.</em> 1997; <strong>158</strong>:1108-1115</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.4049/jimmunol.158.3.1108" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/9013949/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.4049%2Fjimmunol.158.3.1108&amp;pmid=9013949" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref87" href="#" role="doc-biblioref" data-xml-rid="bib87" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref87" aria-controls="sref87">Ridderstad and Tarlinton, 1998</a><div class="dropBlock__holder" data-db-target-of="sref87"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">85.</div><div>Ridderstad, A. ∙ Tarlinton, D.M.</div><div><strong>Kinetics of establishing the memory B cell population as revealed by CD38 expression</strong></div><div><em>J.&nbsp;Immunol.</em> 1998; <strong>160</strong>:4688-4695</div></div><div class="external-links"><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/9590214/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?pmid=9590214" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Upregulation of CD38 is also a feature of precursor MBs (preMB) in the GC (<span class="dropBlock reference-citations"><a id="body-ref-sref53-1" href="#" role="doc-biblioref" data-xml-rid="bib53" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref53-1" aria-controls="sref53-1">Laidlaw et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref53-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">53.</div><div>Laidlaw, B.J. ∙ Schmidt, T.H. ∙ Green, J.A. ...</div><div><strong>The Eph-related tyrosine kinase ligand Ephrin-B1 marks germinal center and memory precursor B cells</strong></div><div><em>J.&nbsp;Exp. Med.</em> 2017; <strong>214</strong>:639-649</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1084/jem.20161461" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-85016232208" target="_blank" aria-label="Open Scopus in new tab">Scopus (36)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28143955/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1084%2Fjem.20161461&amp;pmid=28143955" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). PreMB were alternatively defined as LZ GCB with augmented expression of CCR6 (<span class="dropBlock reference-citations"><a id="body-ref-sref101-2" href="#" role="doc-biblioref" data-xml-rid="bib101" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref101-2" aria-controls="sref101-2">Suan et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref101-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">99.</div><div>Suan, D. ∙ Krautler, N.J. ∙ Maag, J.L.V. ...</div><div><strong>CCR6 Defines Memory B Cell Precursors in Mouse and Human Germinal Centers, Revealing Light-Zone Location and Predominant Low Antigen Affinity</strong></div><div><em>Immunity.</em> 2017; <strong>47</strong>:1142-1153</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_99_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=10.1016%2Fj.immuni.2017.11.022&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_99_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=10.1016%2Fj.immuni.2017.11.022&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_99_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-85039037389" target="_blank" aria-label="Open Scopus in new tab">Scopus (50)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29262350/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.immuni.2017.11.022&amp;pmid=29262350" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), a receptor upregulated in D370Y/WT and KO/KO GCB (<a id="crosref0755" href="#fig3">Figures 3</a>A, 3C, <a id="crosref0760" href="#figs3">S3</a>C, and S3E). A third study defined preMB as cell-cycle arrested interleukin (IL)-9R<sup>+</sup> GCB (<span class="dropBlock reference-citations"><a id="body-ref-sref110-1" href="#" role="doc-biblioref" data-xml-rid="bib110" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref110-1" aria-controls="sref110-1">Wang et&nbsp;al., 2017b</a><div class="dropBlock__holder" data-db-target-of="sref110-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">107.</div><div>Wang, Y. ∙ Shi, J. ∙ Yan, J. ...</div><div><strong>Germinal-center development of memory B cells driven by IL-9 from follicular helper T&nbsp;cells</strong></div><div><em>Nat. Immunol.</em> 2017; <strong>18</strong>:921-930</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ni.3788" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_107_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-85025817827" target="_blank" aria-label="Open Scopus in new tab">Scopus (58)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28650481/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fni.3788&amp;pmid=28650481" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), traits present in D370Y/WT and KO/KO mice (<a id="crosref0770" href="#fig3">Figures 3</a>A, 3D, <a id="crosref0775" href="#figs2">S2</a>G, <a id="crosref0780" href="#figs3">S3</a>C, and S3F). Accordingly, we observed enrichment of the D370Y/WT signature among transcriptional profiles of preMB cells in these reports (<a id="crosref0785" href="#fig3">Figures 3</a>E, <a id="crosref0790" href="#figs3">S3</a>G, and S3H).</div><div id="p0100" role="paragraph">Immunophenotypic analysis of D370Y/WT GCs revealed marked and significant increase in preMB at all studied time points (p value&nbsp;= 0.0045) (<a id="crosref0795" href="#fig3">Figures 3</a>F and <a id="crosref0800" href="#figs3">S3</a>I), and also with independent antigens (<a id="crosref0805" href="#figs3">Figure&nbsp;S3</a>J). PreMB expansion was also seen for KO/KO (<a id="crosref0810" href="#fig3">Figure&nbsp;3</a>G; <a id="crosref0815" href="#mmc6">Data S1</a>R), but not for <i>Tbl1xr1</i><sup><i>KO/WT</i></sup> (<a id="crosref0820" href="#mmc6">Data S1</a>S), supporting that <i>TBL1XR1</i> mutations act as DN LOF. Given that D370Y/WT and KO/KO GCB showed significant <i>Il9r</i> upregulation, which plays a role in MB formation (<span class="dropBlock reference-citations"><a id="body-ref-sref110-2" href="#" role="doc-biblioref" data-xml-rid="bib110" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref110-2" aria-controls="sref110-2">Wang et&nbsp;al., 2017b</a><div class="dropBlock__holder" data-db-target-of="sref110-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">107.</div><div>Wang, Y. ∙ Shi, J. ∙ Yan, J. ...</div><div><strong>Germinal-center development of memory B cells driven by IL-9 from follicular helper T&nbsp;cells</strong></div><div><em>Nat. Immunol.</em> 2017; <strong>18</strong>:921-930</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ni.3788" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_107_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-85025817827" target="_blank" aria-label="Open Scopus in new tab">Scopus (58)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28650481/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fni.3788&amp;pmid=28650481" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), we tested whether IL-9 blockade would rescue the observed phenotype (<a id="crosref0830" href="#fig3">Figure&nbsp;3</a>H). Strikingly, blockade of IL-9R function reverted the KO/KO-driven preMB expansion (<a id="crosref0835" href="#fig3">Figure&nbsp;3</a>I). Although total B cells were not affected by IL-9 blockage (<a id="crosref0840" href="#fig3">Figure&nbsp;3</a>J), there was partial rescue of GC impairment in KO/KO mice (<a id="crosref0845" href="#fig3">Figure&nbsp;3</a>K), concomitant with increased GCB proliferation (<a id="crosref0850" href="#mmc6">Data S1</a>T). Hence, preMB expansion downstream of TBL1XR1 deficiency remains dependent on key MB signaling pathways.</div></section><section id="sec-2-6"><h3><i>Tbl1xr1</i> Deficiency Biases GC Fate toward MB Cells and away from the PC Lineage</h3><div id="p0105" role="paragraph">We next explored whether preMB expansion translated into bias toward MB. D370Y/WT GCB transcriptome showed strong enrichment for MB gene signatures, but no enrichment for PC genes (<a id="crosref0855" href="#fig4">Figures 4</a>A and 4B) (<span class="dropBlock reference-citations"><a id="body-ref-sref60" href="#" role="doc-biblioref" data-xml-rid="bib60" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref60" aria-controls="sref60">Luckey et&nbsp;al., 2006</a><div class="dropBlock__holder" data-db-target-of="sref60"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">60.</div><div>Luckey, C.J. ∙ Bhattacharya, D. ∙ Goldrath, A.W. ...</div><div><strong>Memory T and memory B cells share a transcriptional program of self-renewal with long-term hematopoietic stem cells</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2006; <strong>103</strong>:3304-3309</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.0511137103" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-33644746220" target="_blank" aria-label="Open Scopus in new tab">Scopus (172)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16492737/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.0511137103&amp;pmid=16492737" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). In agreement, GCB expression of the master PC TF BLIMP1 was not affected by <i>Tbl1x1</i> mutation or loss (<a id="crosref0865" href="#figs4">Figures S4</a>A and S4B). To confirm these findings, we used a mixed chimera approach with <i>Rosa26YFP;Tbl1xr1</i><sup><i>D370Y/WT</i></sup> versus <i>Rosa26YFP;Tbl1xr1</i><sup><i>WT/WT</i></sup> cells to track GC and post-GC cells (<a id="crosref0870" href="#fig4">Figure&nbsp;4</a>C; additional mice were administered CD40-blocking antibodies). D370Y/WT splenocytes revealed a lower percentage of YFP<sup>+</sup> cells (<a id="crosref0875" href="#fig4">Figure&nbsp;4</a>D). As expected, most YFP<sup>+</sup> cells corresponded to GCB (B220<sup>+</sup>CD138<sup>−</sup>GL7<sup>+</sup>FAS<sup>+</sup>), which were reduced in the D370Y/WT compartment (<a id="crosref0880" href="#fig4">Figures 4</a>E and <a id="crosref0885" href="#figs4">S4</a>C; <a id="crosref0890" href="#mmc6">Data S1</a>U). On the other&nbsp;hand, D370Y/WT cells showed significant increase in MB (B220<sup>+</sup>CD138<sup>−</sup>FAS<sup>-/lo</sup>GL7<sup>−</sup>CD38<sup>+</sup>IgD<sup>−</sup>), and a concomitant reduction&nbsp;in plasmablasts (PB) (B220<sup>+</sup>CD138<sup>+</sup>) and PC (B220<sup>−</sup>CD138<sup>+</sup>) (<a id="crosref0895" href="#fig4">Figures 4</a>E and <a id="crosref0900" href="#figs4">S4</a>C). Experiments with <i>Rosa26YFP;Tbl1xr1</i><sup><i>KO/KO</i></sup> mice yielded the same phenotype (<a id="crosref0905" href="#figs4">Figures S4</a>D and S4E).</div><div class="figure-wrap"><figure id="fig4" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2020.05.049/asset/c2dede1c-5a88-4682-b735-a6a38821260c/main.assets/gr4_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr4" src="/cms/10.1016/j.cell.2020.05.049/asset/4e2f7579-d2f0-4ae5-8b67-1ea3503f9926/main.assets/gr4.jpg" height="949" width="699" aria-labelledby="fig4-title fig4-content" data-viewer-src="/cms/10.1016/j.cell.2020.05.049/asset/c2dede1c-5a88-4682-b735-a6a38821260c/main.assets/gr4_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig4-title" class="figure__title"><span class="label figure__label">Figure&nbsp;4</span> <span class="figure__title__text"><i>Tbl1xr1</i> Mutations Bias Cell Fate toward MB Cells</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig4-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0170" role="paragraph">(A and B) GSEA of antigen-specific (A) MB or (B) PC, relative to GCB (GSE4142), against D370Y/WT GCB.</div><div id="fspara0175" role="paragraph">(C) Experimental scheme and timeline for (D)–(G).</div><div id="fspara0180" role="paragraph">(D and E) FC analysis of (D) total YFP<sup>+</sup> splenocytes in IgG(Ctrl Ab)-treated mice, and (E) the relative cell type composition of this population. Results for 4 animals. See also <a id="crosref2350" href="#figs4">Figure&nbsp;S4</a>C and <a id="crosref2355" href="#mmc6">Data S1</a>U.</div><div id="fspara0185" role="paragraph">(F and G) FC analysis of (F) total YFP<sup>+</sup> splenocytes in anti-CD40L-treated mice, and (G) the relative cell type composition of this population. Results for 4 animals.</div><div id="fspara0190" role="paragraph">(H) FC analysis of cell type composition of antigen-specific splenocytes. Results for 5 animals. See also <a id="crosref2360" href="#mmc6">Data S1</a>V.</div><div id="fspara0195" role="paragraph">(I) FC analysis of (top) total or (bottom) GC-derived BM LLPC.</div><div id="fspara0200" role="paragraph">Values represent mean ± SEM. Data reproducible with two repeats. p values calculated using unpaired (H and I) or paired (D)–(G) two-tailed Student’s t test.</div></div></div></div></figcaption></figure></div><div class="figure-wrap"><figure id="figs4" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2020.05.049/asset/f40e9970-f3ec-4a31-b4ed-0758978c148e/main.assets/figs4_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs4" src="/cms/10.1016/j.cell.2020.05.049/asset/71ecc933-5f53-4b1a-946c-ea0a5e0da314/main.assets/figs4.jpg" height="933" width="722" aria-labelledby="figs4-title figs4-content" data-viewer-src="/cms/10.1016/j.cell.2020.05.049/asset/f40e9970-f3ec-4a31-b4ed-0758978c148e/main.assets/figs4_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs4-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S4</span> <span class="figure__title__text">Cell Fate Bias in&nbsp;<i>Tbl1xr1</i>-D370Y and&nbsp;<i>Tbl1xr1</i>-KO Mice, Related to <a id="crosref2465" href="#fig4">Figure&nbsp;4</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs4-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0575" role="paragraph">A-B, Representative histograms (left) and quantification (right) of BLIMP1 expression levels, based on FC data, in (A) splenic GCB from <i>Tbl1xr1</i><sup><i>WT/WT</i></sup> or <i>Tbl1xr1</i><sup><i>D370Y/WT</i></sup> mice, 10&nbsp;days after SRBC immunization, or (B) splenic GCB from <i>Tbl1xr1</i><sup><i>WT/WT</i></sup> or <i>Tbl1xr1</i><sup><i>KO/KO</i></sup> mice, 9&nbsp;days after SRBC immunization. BLIMP1 expression in PC from a representative <i>Tbl1xr1</i><sup><i>WT/WT</i></sup> animal is included on each histogram plot, as a positive control.</div><div id="fspara0580" role="paragraph">C, Representative FC plots for YFP<sup>+</sup> cells in IgG1(Ctrl Ab)-treated animals in <a id="crosref2470" href="#fig4">Figure&nbsp;4</a>E.</div><div id="fspara0585" role="paragraph">D-E, Representative FC plots and quantification of (D) total YFP<sup>+</sup> splenocytes and (E) the relative cell type composition of this population. aB&nbsp;= Activated B cells.</div><div id="fspara0590" role="paragraph">Values represent mean ± SEM. All data were reproducible with at least two repeats. NS, not significant; <sup>∗</sup>p&nbsp;&lt; 0.05; <sup>∗∗</sup>p&nbsp;&lt; 0.01; <sup>∗∗∗</sup>p&nbsp;&lt; 0.001, using unpaired two-tailed Student’s t test.</div></div></div></div></figcaption></figure></div><div id="p0110" role="paragraph">GCB with intermediate affinity BCRs tend to become MB cells after T&nbsp;cell help, whereas high-affinity GCBs receive strong T&nbsp;cell help, causing GCBs to differentiate into PCs or recycle to the DZ (<span class="dropBlock reference-citations"><a id="body-ref-sref96-1" href="#" role="doc-biblioref" data-xml-rid="bib96" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref96-1" aria-controls="sref96-1">Shinnakasu et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref96-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">94.</div><div>Shinnakasu, R. ∙ Inoue, T. ∙ Kometani, K. ...</div><div><strong>Regulated selection of germinal-center cells into the memory B cell compartment</strong></div><div><em>Nat. Immunol.</em> 2016; <strong>17</strong>:861-869</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ni.3460" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_94_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-84966559006" target="_blank" aria-label="Open Scopus in new tab">Scopus (208)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27158841/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fni.3460&amp;pmid=27158841" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). CD40-blocking antibodies in mixed chimeras (<a id="crosref0915" href="#fig4">Figure&nbsp;4</a>C) led to profound reduction in GCBs, as expected (<span class="dropBlock reference-citations"><a id="body-ref-sref7" href="#" role="doc-biblioref" data-xml-rid="bib7" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref7" aria-controls="sref7">Baumjohann et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref7"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">7.</div><div>Baumjohann, D. ∙ Preite, S. ∙ Reboldi, A. ...</div><div><strong>Persistent antigen and germinal center B cells sustain T follicular helper cell responses and phenotype</strong></div><div><em>Immunity.</em> 2013; <strong>38</strong>:596-605</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=10.1016%2Fj.immuni.2012.11.020&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=10.1016%2Fj.immuni.2012.11.020&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-84875549975" target="_blank" aria-label="Open Scopus in new tab">Scopus (246)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23499493/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.immuni.2012.11.020&amp;pmid=23499493" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), to the point where differences in YFP<sup>+</sup> cell abundance among D370Y/WT and WT were lost (<a id="crosref0925" href="#fig4">Figures 4</a>F and 4G). Reduction of CD40 input drove the majority of the remaining D370Y/WT GC-derived B cells to the MB compartment. In contrast, the majority of residual WT cells were IgD<sup>+</sup> activated B cells (“aB”) that presumably had not yet entered the GC reaction. No PCs were identified in this setting from D370Y/WT, suggesting that the MB cell fate is hardwired into the mutant B cells (<a id="crosref0930" href="#fig4">Figure&nbsp;4</a>G).</div><div id="p0115" role="paragraph">Because PC formation is a rather late event in the GC (<span class="dropBlock reference-citations"><a id="body-ref-sref111" href="#" role="doc-biblioref" data-xml-rid="bib111" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref111" aria-controls="sref111">Weisel et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref111"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">108.</div><div>Weisel, F.J. ∙ Zuccarino-Catania, G.V. ∙ Chikina, M. ...</div><div><strong>A Temporal Switch in the Germinal Center Determines Differential Output of Memory B and Plasma Cells</strong></div><div><em>Immunity.</em> 2016; <strong>44</strong>:116-130</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_108_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=10.1016%2Fj.immuni.2015.12.004&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_108_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=10.1016%2Fj.immuni.2015.12.004&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_108_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-84958831765" target="_blank" aria-label="Open Scopus in new tab">Scopus (158)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26795247/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.immuni.2015.12.004&amp;pmid=26795247" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), we evaluated whether commitment bias was also detectable at later time points. To this end, used an adoptive transfer system to quantify antigen-specific MB and PC 24&nbsp;days after immunization (<a id="crosref0940" href="#fig4">Figure&nbsp;4</a>H; <a id="crosref0945" href="#mmc6">Data S1</a>V). Assessment of NP-specific populations again showed that D370Y/WT biased cell fate toward MB, at the expense of PC (<a id="crosref0950" href="#fig4">Figure&nbsp;4</a>H). Finally, assessment of long-lived PC (LLPC) in the BM, 70&nbsp;days after immunization of <i>Rosa26YFP;Tbl1xr1</i><sup><i>D370Y/WT</i></sup> mice, revealed that although the abundance of total LLPC was unaltered, D370Y/WT had less GC-derived LLPC than WT mice (<a id="crosref0955" href="#fig4">Figure&nbsp;4</a>I).</div></section><section id="sec-2-7"><h3><i>TBL1XR1</i> Mutations Cause a BCL6-to-BACH2 Molecular Switch</h3><div id="p0120" role="paragraph">TBL1XR1 is a core component of the SMRT/NCOR1 complex, which in GCB is recruited to chromatin by BCL6 (<span class="dropBlock reference-citations"><a id="body-ref-sref35-3" href="#" role="doc-biblioref" data-xml-rid="bib35" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref35-3" aria-controls="sref35-3">Hatzi et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref35-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">35.</div><div>Hatzi, K. ∙ Jiang, Y. ∙ Huang, C. ...</div><div><strong>A hybrid mechanism of action for BCL6 in B cells defined by formation of functionally distinct complexes at enhancers and promoters</strong></div><div><em>Cell Rep.</em> 2013; <strong>4</strong>:578-588</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=10.1016%2Fj.celrep.2013.06.016&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=10.1016%2Fj.celrep.2013.06.016&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-84881615483" target="_blank" aria-label="Open Scopus in new tab">Scopus (90)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23911289/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.celrep.2013.06.016&amp;pmid=23911289" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). We found that TBL1XR1 is also present in chromatin-bound BCL6 complexes, by RIME- liquid chromatography-tandem mass spectrometry (LC-MS/MS) (<a id="crosref0965" href="#figs5">Figure&nbsp;S5</a>A; <a id="crosref0970" href="#mmc3">Table S3</a>). Interaction of endogenous TBL1XR1 with the SMRT/HDAC3 complex and BCL6 was confirmed by coimmunoprecipitation (coIP) (<a id="crosref0975" href="#figs5">Figure&nbsp;S5</a>B). In other systems, TBL1XR1 can have SMRT-independent functions (<span class="dropBlock reference-citations"><a id="body-ref-sref56" href="#" role="doc-biblioref" data-xml-rid="bib56" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref56" aria-controls="sref56">Li and Wang, 2008</a><div class="dropBlock__holder" data-db-target-of="sref56"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">56.</div><div>Li, J. ∙ Wang, C.Y.</div><div><strong>TBL1-TBLR1 and beta-catenin recruit each other to Wnt target-gene promoter for transcription activation and oncogenesis</strong></div><div><em>Nat. Cell Biol.</em> 2008; <strong>10</strong>:160-169</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ncb1684" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_56_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-38849131230" target="_blank" aria-label="Open Scopus in new tab">Scopus (108)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/18193033/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fncb1684&amp;pmid=18193033" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). We therefore performed proximity ligation-mass spectrometry (BioID) for TBL1XR1<sup>WT</sup> interacting partners in GC-derived cell lines (<a id="crosref0985" href="#figs5">Figure&nbsp;S5</a>C). The majority of significantly interacting proteins for TBL1XR1<sup>WT</sup> were restricted to components of the SMRT complex (<a id="crosref0990" href="#fig5">Figure&nbsp;5</a>A; <a id="crosref0995" href="#mmc3">Table S3</a>). In addition to these, only BCL6 and another BTB-domain containing TF (BACH2), appeared as significant hits (<a id="crosref1000" href="#fig5">Figure&nbsp;5</a>A), largely ruling out alternative functions for TBL1XR1<sup>WT</sup> in GC-derived B cells.</div><div class="figure-wrap"><figure id="fig5" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2020.05.049/asset/9b288956-64dd-4f2b-92f2-af1de9e1cfbd/main.assets/gr5_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr5" src="/cms/10.1016/j.cell.2020.05.049/asset/9ecf4736-5a3c-4c66-a9d9-a9b7b384c4b6/main.assets/gr5.jpg" height="956" width="709" aria-labelledby="fig5-title fig5-content" data-viewer-src="/cms/10.1016/j.cell.2020.05.049/asset/9b288956-64dd-4f2b-92f2-af1de9e1cfbd/main.assets/gr5_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig5-title" class="figure__title"><span class="label figure__label">Figure&nbsp;5</span> <span class="figure__title__text"><i>TBL1XR1</i> Mutations Introduce a BCL6-to-BACH2 Switch</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig5-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0210" role="paragraph">(A) TBL1XR1<sup>WT</sup> interacting proteins. Shown are hits with SAINT &gt;0.75 and 2-fold spectral counts (averaged between biological duplicates) over eGFP-BirA. See also <a id="crosref2365" href="#figs5">Figure&nbsp;S5</a>C and <a id="crosref2370" href="#mmc3">Table S3</a>.</div><div id="fspara0215" role="paragraph">(B and C) Comparison between TBL1XR1 WT and Y446S BioID in (B) OCI-Ly1 or (C) U2932 cells. Results as average of biological duplicates for each condition.</div><div id="fspara0220" role="paragraph">(D) CoIP results for V5 pulldown in cells inducibly expressing WT or mutant TBL1XR1-V5 fusion proteins. See also <a id="crosref2375" href="#figs5">Figures S5</a>D and S5E.</div><div id="fspara0225" role="paragraph">(E) GSEA of D370Y/WT GCB against siBCL6-treated OCI-Ly1 (GSE29282).</div><div id="fspara0230" role="paragraph">(F) FC analysis of splenic GCB. See also <a id="crosref2380" href="#mmc6">Data S1</a>W.</div><div id="fspara0235" role="paragraph">(G) IHC staining of spleen sections from (F). Graph shows GC area based on PNA staining. Dots are individual GCs. Results for 4 animals per genotype. Scale, 100&nbsp;μm.</div><div id="fspara0240" role="paragraph">(H and I) GSEA of <i>IμBcl6</i>-<i>Tbl1xr1</i><sup><i>D370Y/WT</i></sup> GCB against (H) BACH2<sup>hi</sup> GCB (GSE77319) or (I) siBCL6-treated OCI-Ly1.</div><div id="fspara0245" role="paragraph">(J and K) GSEA of KO/KO GCB against (J) siBCL6-treated OCI-Ly1 or (K) BACH2<sup>hi</sup> GCB.</div><div id="fspara0250" role="paragraph">(L) FC analysis of splenic GCB.</div><div id="fspara0255" role="paragraph">Values represent mean ± SEM. Data reproducible with two repeats. p values calculated using one-way ANOVA with Tukey’s post-test (F and L), or Kruskal-Wallis H test with Dunn’s post-test (G).</div></div></div></div></figcaption></figure></div><div class="figure-wrap"><figure id="figs5" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2020.05.049/asset/5eebeb7e-b06f-465c-ad90-1333dabeffc7/main.assets/figs5_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs5" src="/cms/10.1016/j.cell.2020.05.049/asset/66389efa-498c-45ab-a71e-fc0a596b21f2/main.assets/figs5.jpg" height="951" width="693" aria-labelledby="figs5-title figs5-content" data-viewer-src="/cms/10.1016/j.cell.2020.05.049/asset/5eebeb7e-b06f-465c-ad90-1333dabeffc7/main.assets/figs5_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs5-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S5</span> <span class="figure__title__text">TF Switch Triggered by&nbsp;<i>TBL1XR1&nbsp;M</i>utation or Loss, Related to <a id="crosref2475" href="#fig5">Figure&nbsp;5</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs5-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0600" role="paragraph">A, RIME analysis of BCL6 interacting partners in OCI-Ly1 cells. A species-matched IgG antibody was used as negative control. Shown is the correlation for two biological replicates for BCL6 immunoprecipitations. Circle sizes are proportional to the fold induction over the IgG control.</div><div id="fspara0605" role="paragraph">B, Co-IP of endogenous TBL1XR1 with BCL6, HDAC3 or SMRT in nuclear extracts obtained from the stated cell lines. A species-matched IgG antibody was included as negative control.</div><div id="fspara0610" role="paragraph">C, WB results showing expression of TBL1XR1 and BirA, in cell lines used for BioID experiments, 48h after Doxycycline induction. ACTIN expression levels were used as a loading control.</div><div id="fspara0615" role="paragraph">D, Co-IP of endogenous BCL6 with TBL1XR1-V5 (WT, D370Y, Y395H, Y446S), in OCI-Ly1 engineered cells from <a id="crosref2480" href="#fig5">Figure&nbsp;5</a>D. BCL6 immunoprecipitation was conducted on nuclear extracts obtained 48h after Doxycycline induction.</div><div id="fspara0620" role="paragraph">E, WB results showing expression of the stated proteins in nuclear protein extracts from cells transduced to inducibly express WT or mutant (D370Y, Y395H, Y446S) TBL1XR1-V5 fusion proteins. Results shown correspond to extracts obtained 48h after Doxycycline induction. ACTIN expression levels were used as a loading control.</div><div id="fspara0625" role="paragraph">F, Representative histograms (left) and quantification (right) of BCL6 expression levels, based on FC data, in splenic GCB from <i>Tbl1xr1</i><sup><i>WT/WT</i></sup> or <i>Tbl1xr1</i><sup><i>D370Y/WT</i></sup> mice, 8&nbsp;days after SRBC immunization. BCL6 expression in NB from a representative Tbl1x1<sup><i>WT/WT</i></sup> animal is included on the histogram plot, as a negative control.</div><div id="fspara0630" role="paragraph">G-H, GSEA of gene signatures upregulated and downregulated in <i>Tbl1x1</i><sup><i>KO/KO</i></sup> versus <i>Tbl1xr1</i><sup><i>WT/WT</i></sup> GCB, ranked against: (G) the preMB (EFNB1<sup>+</sup>S1PR2<sup>lo</sup>) GCB signature from GSE89897 or (H) the <i>IμBcl6</i>-<i>Tbl1xr1</i><sup><i>D370Y/WT</i></sup> GCB signature from <a id="crosref2485" href="#fig5">Figure&nbsp;5</a>H.</div><div id="fspara0635" role="paragraph">I, Representative histograms (left) and quantification (right) of BCL6 expression levels, based on FC data, in splenic GCB from <i>Tbl1xr1</i><sup><i>WT/WT</i></sup> or <i>Tbl1xr1</i><sup><i>KO/KO</i></sup> mice, 8&nbsp;days after SRBC immunization. BCL6 expression in NB from a representative Tbl1x1<sup><i>WT/WT</i></sup> animal is included on the histogram plot, as a negative control.</div><div id="fspara0640" role="paragraph">J, WB analysis of TBL1XR1 protein expression in representative single-cell-derived CRISPR.KO or Control clones, obtained with 2 different anti-<i>TBL1XR1</i> guide RNAs (“1” or “2”).</div><div id="fspara0645" role="paragraph">K, Flow cytometry analysis of cell proliferation by progressive dilution of CellTrace Dye, in CRISPR.KO or Control clones in (J). Shown are representative histograms (left) and quantification (right). Bar plot presents data as proliferation dye MFI, relative to that on Day 0. Results for two clones per genotype were averaged, considering five technical replicates for each.</div><div id="fspara0650" role="paragraph">L, Co-IP of endogenous SMRT and HDAC3, by BACH2 or BCL6 immunoprecipitation, in nuclear extracts obtained from the stated cell lines. A species-matched IgG antibody was included as negative control.</div><div id="fspara0655" role="paragraph">Values represent mean ± SEM. All data were reproducible with at least two repeats. NS, not significant; <sup>∗</sup>p&nbsp;&lt; 0.05; <sup>∗∗</sup>p&nbsp;&lt; 0.01; <sup>∗∗∗</sup>p&nbsp;&lt; 0.001, using unpaired two-tailed Student’s t test (F,I); or two-way (K) ANOVA with Tukey’s post-test.</div></div></div></div></figcaption></figure></div><div id="p0125" role="paragraph">To assess whether <i>TBL1XR1</i> mutations alter its interactome, we conducted additional assays modeling one of the most recurrent DLBCL mutations (TBL1XR1<sup>Y446S</sup>) (<a id="crosref1005" href="#fig1">Figure&nbsp;1</a>A). Surprisingly, although the interaction with SMRT/HDAC3 complex was preserved, the only significant change in the TBL1XR1<sup>MUT</sup> interactome was ∼6-fold gain of interaction with BACH2, at the expense of BCL6 (<a id="crosref1010" href="#fig5">Figure&nbsp;5</a>B). Similar results were obtained in an ABC-DLBCL cell line (<a id="crosref1015" href="#fig5">Figures 5</a>C and <a id="crosref1020" href="#figs5">S5</a>C; <a id="crosref1025" href="#mmc3">Table S3</a>) and in HEK293 cells (not shown). Assessment of three independent TBL1XR1 mutants by coIP confirmed gain of interaction with BACH2 (<a id="crosref1030" href="#fig5">Figure&nbsp;5</a>D) and loss of interaction with BCL6 (<a id="crosref1035" href="#figs5">Figure&nbsp;S5</a>D) without alteration of total protein levels (<a id="crosref1040" href="#figs5">Figures S5</a>D and S5E).</div><div id="p0130" role="paragraph">In accordance with these data, genes de-repressed in D370Y/WT GCB <i>in&nbsp;vivo</i>, matched those repressed by BCL6 (<a id="crosref1045" href="#fig5">Figure&nbsp;5</a>E). Because <i>Tbl1xr1</i> mutation did not affect BCL6 expression levels (<a id="crosref1050" href="#figs5">Figure&nbsp;S5</a>F), this suggested interference at the functional level. To test this, we crossed D370Y/WT mice to the <i>IμBcl6</i> strain, which constitutively expresses BCL6 in GCB and manifests GC hyperplasia (<span class="dropBlock reference-citations"><a id="body-ref-sref13-1" href="#" role="doc-biblioref" data-xml-rid="bib13" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref13-1" aria-controls="sref13-1">Cattoretti et&nbsp;al., 2005</a><div class="dropBlock__holder" data-db-target-of="sref13-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">13.</div><div>Cattoretti, G. ∙ Pasqualucci, L. ∙ Ballon, G. ...</div><div><strong>Deregulated BCL6 expression recapitulates the pathogenesis of human diffuse large B cell lymphomas in mice</strong></div><div><em>Cancer Cell.</em> 2005; <strong>7</strong>:445-455</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_13_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=10.1016%2Fj.ccr.2005.03.037&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_13_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=10.1016%2Fj.ccr.2005.03.037&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_13_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-19344368745" target="_blank" aria-label="Open Scopus in new tab">Scopus (258)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15894265/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.ccr.2005.03.037&amp;pmid=15894265" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Indeed, <i>IμBcl6</i> mice manifested GC hyperplasia after immunization (<a id="crosref1060" href="#fig5">Figures 5</a>F and 5G; <a id="crosref1065" href="#mmc6">Data S1</a>W), which, in the presence of the <i>Tbl1xr1</i><sup>D370Y</sup>, was rescued back to the levels of the WT control (<a id="crosref1070" href="#fig5">Figures 5</a>F and 5G). As reported above in a WT background, <i>IμBcl6-Tbl1xr1</i><sup><i>D370Y/WT</i></sup> GCB manifested a preMB-like transcriptome (<a id="crosref1075" href="#mmc4">Table S4</a>; <a id="crosref1080" href="#mmc6">Data S1</a>X) and actual preMB expansion (<a id="crosref1085" href="#mmc6">Data S1</a>Y). In agreement with our biochemical data, we observed enrichment for a BACH2-driven signature in <i>IμBcl6-Tbl1xr1</i><sup><i>D370Y/WT</i></sup> GCB (<a id="crosref1090" href="#fig5">Figure&nbsp;5</a>H) and significant depletion of the BCL6 program (<a id="crosref1095" href="#fig5">Figure&nbsp;5</a>I). BACH2 is reported to be a key TF in MB formation (<span class="dropBlock reference-citations"><a id="body-ref-sref96-2" href="#" role="doc-biblioref" data-xml-rid="bib96" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref96-2" aria-controls="sref96-2">Shinnakasu et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref96-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">94.</div><div>Shinnakasu, R. ∙ Inoue, T. ∙ Kometani, K. ...</div><div><strong>Regulated selection of germinal-center cells into the memory B cell compartment</strong></div><div><em>Nat. Immunol.</em> 2016; <strong>17</strong>:861-869</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ni.3460" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_94_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-84966559006" target="_blank" aria-label="Open Scopus in new tab">Scopus (208)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27158841/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fni.3460&amp;pmid=27158841" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), and our data suggest that TBL1XR1 mutations mistarget the SMRT complex to BACH2, thus inducing preferential transcriptional rewiring toward MB, while preventing PC formation by maintaining BACH2 repression of BLIMP1 (<a id="crosref1105" href="#mmc5">Data S1</a>Z) (<span class="dropBlock reference-citations"><a id="body-ref-sref40" href="#" role="doc-biblioref" data-xml-rid="bib40" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref40" aria-controls="sref40">Huang et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref40"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">40.</div><div>Huang, C. ∙ Geng, H. ∙ Boss, I. ...</div><div><strong>Cooperative transcriptional repression by BCL6 and BACH2 in germinal center B-cell differentiation</strong></div><div><em>Blood.</em> 2014; <strong>123</strong>:1012-1020</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1182/blood-2013-07-518605" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_40_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-84897870661" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24277074/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1182%2Fblood-2013-07-518605&amp;pmid=24277074" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref74-1" href="#" role="doc-biblioref" data-xml-rid="bib74" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref74-1" aria-controls="sref74-1">Ochiai et&nbsp;al., 2006</a><div class="dropBlock__holder" data-db-target-of="sref74-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">75.</div><div>Ochiai, K. ∙ Katoh, Y. ∙ Ikura, T. ...</div><div><strong>Plasmacytic transcription factor Blimp-1 is repressed by Bach2 in B cells</strong></div><div><em>J.&nbsp;Biol. Chem.</em> 2006; <strong>281</strong>:38226-38234</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1074/jbc.M607592200" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_75_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-33846013955" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17046816/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1074%2Fjbc.M607592200&amp;pmid=17046816" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="p0135" role="paragraph">Transcriptional profiling of KO/KO GCB likewise enriched both preMB and <i>IμBcl6-Tbl1xr1</i><sup><i>D370Y/WT</i></sup> signatures (<a id="crosref1120" href="#figs5">Figures S5</a>G and S5H; <a id="crosref1125" href="#mmc4">Table S4</a>). More importantly, it recapitulated the upregulation of BCL6 targets (<a id="crosref1130" href="#fig5">Figure&nbsp;5</a>J), and engagement of BACH2-driven program (<a id="crosref1135" href="#fig5">Figure&nbsp;5</a>K). KO/KO did not affect BCL6 expression in GCB (<a id="crosref1140" href="#figs5">Figure&nbsp;S5</a>I) but was able to rescue BCL6-induced hyperplasia in <i>IμBcl6</i> GCs (<a id="crosref1145" href="#fig5">Figure&nbsp;5</a>L). To assess SMRT/HDAC3 shuttling at the biochemical level, we developed TBL1XR1-KO variants of a GC-derived lymphoma cell line using CRISPR.Cas9 (<a id="crosref1150" href="#figs5">Figure&nbsp;S5</a>J). Similar to primary GCB, these clones showed impaired proliferation (<a id="crosref1155" href="#figs5">Figure&nbsp;S5</a>K). Notably, TBLXR1-KO resulted in loss of BCL6-SMRT/HDAC3 interaction and gain in BACH2-SMRT/HDAC3 Co-IP (<a id="crosref1160" href="#figs5">Figure&nbsp;S5</a>L). TBL1XR1 is thus critical for the preferential association of SMRT with BCL6 in GCBs, and its LOF induces preferential interaction with BACH2 (<a id="crosref1165" href="#mmc6">Data S1</a>AA), thus explaining the GC impairment and MB cell fate bias.</div></section><section id="sec-2-8"><h3><i>Tbl1xr1</i> Mutant MB Cells Manifest Traits Associated with Preferential Reentry to the GC Reaction</h3><div id="p0140" role="paragraph">MB reentry into the GC has been proposed as a putative mechanism of GC-derived lymphomagenesis (<span class="dropBlock reference-citations"><a id="body-ref-sref103-1" href="#" role="doc-biblioref" data-xml-rid="bib103" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref103-1" aria-controls="sref103-1">Sungalee et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref103-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">101.</div><div>Sungalee, S. ∙ Mamessier, E. ∙ Morgado, E. ...</div><div><strong>Germinal center reentries of BCL2-overexpressing B cells drive follicular lymphoma progression</strong></div><div><em>J.&nbsp;Clin. Invest.</em> 2014; <strong>124</strong>:5337-5351</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1172/JCI72415" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_101_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-84915818499" target="_blank" aria-label="Open Scopus in new tab">Scopus (49)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25384217/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1172%2FJCI72415&amp;pmid=25384217" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Given our findings, we postulated that <i>Tbl1xr1</i> mutation might generate aberrant MBs with capacity for preferential reentry into new GCs. To explore whether D370Y/WT MBs persisted over time, we immunized mice with NP-CGG and sacrificed long after GC resolution, which revealed long-lasting expansion of NP-specific MB in spleen and BM (<a id="crosref1175" href="#fig6">Figures 6</a>A and 6B; <a id="crosref1180" href="#mmc6">Data S1</a>BB–S1CC). A subset of non-class switched MB cells in humans is proposed to re-seed GCs upon recall, while switched MB differentiate into Ig-secreting cells (<span class="dropBlock reference-citations"><a id="body-ref-sref25" href="#" role="doc-biblioref" data-xml-rid="bib25" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref25" aria-controls="sref25">Dogan et&nbsp;al., 2009</a><div class="dropBlock__holder" data-db-target-of="sref25"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">25.</div><div>Dogan, I. ∙ Bertocci, B. ∙ Vilmont, V. ...</div><div><strong>Multiple layers of B cell memory with different effector functions</strong></div><div><em>Nat. Immunol.</em> 2009; <strong>10</strong>:1292-1299</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ni.1814" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-70449722777" target="_blank" aria-label="Open Scopus in new tab">Scopus (355)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19855380/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fni.1814&amp;pmid=19855380" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). D370Y/WT and KO/KO GCB showed significant restriction in class-switch recombination (CSR), with a relative increase in IgM<sup>+</sup> and reduction in IgG1<sup>+</sup> cells (<a id="crosref1190" href="#fig6">Figures 6</a>C and <a id="crosref1195" href="#figs6">S6</a>A). This was not due to delayed kinetics, because later time points showed similar isotype representation (<a id="crosref1200" href="#figs6">Figure&nbsp;S6</a>B), nor was it due to an impairment in <i>Aicda</i> induction (<a id="crosref1205" href="#figs6">Figures S6</a>C–S6E). The CSR restriction was also evident in GC-derived MB, tracked with the <i>Rosa26YFP</i> reporter (<a id="crosref1210" href="#fig6">Figures 6</a>D and <a id="crosref1215" href="#figs6">S6</a>F; <a id="crosref1220" href="#mmc6">Data S1</a>EE). In accordance with surface Ig profiles, NP-OVA immunized D370Y/WT and KO/KO mice showed impaired IgG1 antibody responses (<a id="crosref1225" href="#figs6">Figures S6</a>G and S6H). Still, affinity maturation was not significantly affected (<a id="crosref1230" href="#figs6">Figures S6</a>I and S6J), nor were there differences in Ig SHM burden between D370Y/WT and WT GCB (<a id="crosref1235" href="#figs6">Figure&nbsp;S6</a>K), further suggesting that AID activity was not compromised.</div><div class="figure-wrap"><figure id="fig6" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2020.05.049/asset/666ac0b3-90e8-4b35-9ff0-7cfd418642ac/main.assets/gr6_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr6" src="/cms/10.1016/j.cell.2020.05.049/asset/875f9e06-0008-4d56-a798-7ec2240d02d6/main.assets/gr6.jpg" height="955" width="749" aria-labelledby="fig6-title fig6-content" data-viewer-src="/cms/10.1016/j.cell.2020.05.049/asset/666ac0b3-90e8-4b35-9ff0-7cfd418642ac/main.assets/gr6_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig6-title" class="figure__title"><span class="label figure__label">Figure&nbsp;6</span> <span class="figure__title__text"><i>Tbl1xr1</i> Mutant MB Cells Preferentially Become GCB upon Recall</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig6-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0265" role="paragraph">(A and B) FC analysis of total or antigen-specific MB population in (A) spleen or (B) BM. See also <a id="crosref2385" href="#mmc6">Data S1</a>BB–S1CC.</div><div id="fspara0270" role="paragraph">(C and D) FC analysis of IgG1<sup>+</sup> or IgM<sup>+</sup> splenic (C) GCB or (D) MB. See also <a id="crosref2390" href="#figs6">Figure&nbsp;S6</a>B and <a id="crosref2395" href="#mmc6">Data S1</a>DD.</div><div id="fspara0275" role="paragraph">(E) FC profiling of donor-derived antigen-specific splenic MB. See also <a id="crosref2400" href="#figs6">Figure&nbsp;S6</a>L.</div><div id="fspara0280" role="paragraph">(F) Experimental scheme and timeline for (G)–(J).</div><div id="fspara0285" role="paragraph">(G–I) FC analysis of (G) total B220<sup>+</sup>, (H) GCB, or (I) PC donor-derived antigen-specific cells, in the spleens of MB-recipient animals. See also <a id="crosref2405" href="#figs6">Figure&nbsp;S6</a>M.</div><div id="fspara0290" role="paragraph">(J) ELISA for NP-specific serum Ig in MB-recipients in (F). Graphs show difference between 0 and 3.5&nbsp;days post-immunization.</div><div id="fspara0295" role="paragraph">Values represent mean ± SEM. Data reproducible with two repeats. p values calculated using unpaired two-tailed Student’s t test.</div></div></div></div></figcaption></figure></div><div class="figure-wrap"><figure id="figs6" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2020.05.049/asset/3933a203-3c9d-4a65-82a2-278429f4e119/main.assets/figs6_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs6" src="/cms/10.1016/j.cell.2020.05.049/asset/19a3ccd7-1dab-455b-a8af-704fdc1262f2/main.assets/figs6.jpg" height="950" width="677" aria-labelledby="figs6-title figs6-content" data-viewer-src="/cms/10.1016/j.cell.2020.05.049/asset/3933a203-3c9d-4a65-82a2-278429f4e119/main.assets/figs6_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs6-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S6</span> <span class="figure__title__text">AID Function in&nbsp;<i>Tbl1xr1</i>-D370Y and&nbsp;<i>Tbl1xr1</i>-KO B-Cells, Related to <a id="crosref2490" href="#fig6">Figure&nbsp;6</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs6-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0665" role="paragraph">A, Representative FC plots (left) and quantification (right) of (top) IgG1<sup>+</sup> or (bottom) IgM<sup>+</sup> (bottom) splenic GCB.</div><div id="fspara0670" role="paragraph">B, Representative FC plots and quantification of IgG1<sup>+</sup> GCB at (top) 10&nbsp;days, (center) 15&nbsp;days, or (bottom) 25&nbsp;days post SRBC immunization.</div><div id="fspara0675" role="paragraph">C, RT-qPCR-based assessment of <i>Aicda</i> mRNA expression in DZ GCB, sorted 9&nbsp;days after SRBC immunization. Results are expressed as fold change, relative to <i>Tbl1xr1</i><sup><i>WT/WT</i></sup>, and normalized to GAPDH expression levels.</div><div id="fspara0680" role="paragraph">D-E, Representative histograms (left) and quantification (right) of AID expression levels, based on FC data, in (D) splenic GCB (B220<sup>+</sup>FAS<sup>+</sup>GL7<sup>+</sup>) from <i>Tbl1xr1</i><sup><i>WT/WT</i></sup> or <i>Tbl1xr1</i><sup><i>D370Y/WT</i></sup> mice, 8&nbsp;days after SRBC immunization, or (E) GCB from <i>Tbl1xr1</i><sup><i>WT/WT</i></sup> or <i>Tbl1xr1</i><sup><i>KO/KO</i></sup> mice, 10&nbsp;days after SRBC immunization. AID expression in NB (B220<sup>+</sup>FAS<sup>-</sup>GL7<sup>-</sup>IgD<sup>+</sup>) from a representative <i>Tbl1xr1</i><sup><i>WT/WT</i></sup> animal (gray), and isotype control staining on GCB (orange) are included on each histogram plot, as a negative controls.</div><div id="fspara0685" role="paragraph">F, Representative FC plots (left) and quantification (right) of IgG1<sup>+</sup> MB cells at (top) 15&nbsp;days or (bottom) 25&nbsp;days post SRBC immunization.</div><div id="fspara0690" role="paragraph">G, ELISA-based assessment of NP-specific serum antibodies in <i>Tbl1xr1</i><sup><i>D370Y/WT</i></sup> versus <i>Tbl1xr1</i><sup><i>WT/WT</i></sup> mice, represented as the difference in signal between days 0 and 24 post immunization.</div><div id="fspara0695" role="paragraph">H, ELISA-based assessment of NP-specific serum antibodies in <i>Tbl1xr1</i><sup><i>KO/KO</i></sup> versus <i>Tbl1xr1</i><sup><i>WT/WT</i></sup> mice, represented as the difference in signal between days 0 and 42 post immunization.</div><div id="fspara0700" role="paragraph">I, Relative levels of affinity maturation in <i>Tbl1xr1</i><sup><i>D370Y/WT</i></sup> versus <i>Tbl1xr1</i><sup><i>WT/WT</i></sup> mice, represented as the ratio between high-affinity and low-affinity NP-specific serum immunoglobulin signal, determined as in (G).</div><div id="fspara0705" role="paragraph">J, Relative levels of affinity maturation in <i>Tbl1xr1</i><sup><i>KO/KO</i></sup> versus <i>Tbl1xr1</i><sup><i>WT/WT</i></sup> mice, represented as the ratio between high-affinity and low-affinity NP-specific serum immunoglobulin signal, determined as in (H).</div><div id="fspara0710" role="paragraph">K, Bar plot showing SHM burden at the immunoglobulin intron J<sub>H</sub>4 loci (as in <span class="dropBlock reference-citations"><a id="body-ref-sref44-1" href="#" role="doc-biblioref" data-xml-rid="bib44" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref44-1" aria-controls="sref44-1">Jolly et&nbsp;al., 1997</a><div class="dropBlock__holder" data-db-target-of="sref44-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">44.</div><div>Jolly, C.J. ∙ Klix, N. ∙ Neuberger, M.S.</div><div><strong>Rapid methods for the analysis of immunoglobulin gene hypermutation: application to transgenic and gene targeted mice</strong></div><div><em>Nucleic Acids Res.</em> 1997; <strong>25</strong>:1913-1919</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/25.10.1913" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-0030845386" target="_blank" aria-label="Open Scopus in new tab">Scopus (106)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/9115357/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2F25.10.1913&amp;pmid=9115357" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) in GC B cells from <i>Rosa26YFP;Tbl1xr1</i><sup><i>D370Y/WT</i></sup> or <i>Rosa26YFP;Tbl1xr1</i><sup><i>WT/WT</i></sup> mice, sorted 9d after SRBC immunization. Genomic DNA extracted from sorted GC B cells from an <i>Aicda</i><sup><i>KO/KO</i></sup> mouse model (AID KO/KO) was used as negative control for the experiment. Shown are pooled results for 3 [D370Y/WT], 3 [WT/WT] and 1 [AID KO/KO] animals. Numbers in square brackets represent the total numbers of sequenced clones.</div><div id="fspara0715" role="paragraph">L, Representative FC plots (left) and quantification (right) of IgM<sup>+</sup> splenic antigen-specific donor-derived MB cells.</div><div id="fspara0720" role="paragraph">M, Gating strategy and quantification of total antigen-specific donor-derived cells detected in the spleens of recipient mice, 3.5d after immunization, from experiment in <a id="crosref2500" href="#fig6">Figure&nbsp;6</a>F.</div><div id="fspara0725" role="paragraph">Values represent mean ± SEM. All the data were reproducible with at least two repeats. NS, not significant; <sup>∗</sup>p&nbsp;&lt; 0.05; <sup>∗∗</sup>p&nbsp;&lt; 0.01; <sup>∗∗∗</sup>p&nbsp;&lt; 0.001, using unpaired two-tailed Student’s t test, or Wilcoxon rank sum test (K).</div></div></div></div></figcaption></figure></div><div id="p0145" role="paragraph">In mice, CD80<sup>−</sup>PDL2<sup>−</sup> MBs, which are mostly IgM<sup>+</sup>, show a higher capacity for GC reentry, while CD80<sup>+</sup>PDL2<sup>+</sup> MB are generally class-switched, and tend to differentiate into PC (<span class="dropBlock reference-citations"><a id="body-ref-sref118-1" href="#" role="doc-biblioref" data-xml-rid="bib118" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref118-1" aria-controls="sref118-1">Zuccarino-Catania et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref118-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">115.</div><div>Zuccarino-Catania, G.V. ∙ Sadanand, S. ∙ Weisel, F.J. ...</div><div><strong>CD80 and PD-L2 define functionally distinct memory B cell subsets that are independent of antibody isotype</strong></div><div><em>Nat. Immunol.</em> 2014; <strong>15</strong>:631-637</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ni.2914" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_115_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-84903646784" target="_blank" aria-label="Open Scopus in new tab">Scopus (154)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24880458/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fni.2914&amp;pmid=24880458" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Profiling of donor-derived antigen-specific MB cells (CD45.1<sup>+</sup>CD138<sup>−</sup>B220<sup>+</sup>IgD<sup>−</sup>NP<sup>+</sup>) from an adoptive transfer system revealed that D370Y/WT MB were significantly over-represented in the CD80<sup>−</sup>PDL2<sup>−</sup> population (<a id="crosref1245" href="#fig6">Figure&nbsp;6</a>E). Most D370Y/WT MB were also IgM<sup>+</sup>, further confirming a CSR restriction (<a id="crosref1250" href="#figs6">Figure&nbsp;S6</a>L).</div></section><section id="sec-2-9"><h3><i>Tbl1xr1</i> Mutant MB Cells Preferentially Become GCB-Cells upon Antigen Recall</h3><div id="p0150" role="paragraph">Results above suggest that TBL1XR1<sup>MUT</sup> might predispose MB for GC reentry upon recall. To test this, we conducted secondary adoptive transfer experiments (<a id="crosref1255" href="#fig6">Figure&nbsp;6</a>F), where sorted D370Y/WT or WT antigen-specific MB were transferred in equal numbers into μMT recipient mice, along with OT-II memory T&nbsp;cells to boost MB recall (<span class="dropBlock reference-citations"><a id="body-ref-sref118-2" href="#" role="doc-biblioref" data-xml-rid="bib118" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref118-2" aria-controls="sref118-2">Zuccarino-Catania et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref118-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">115.</div><div>Zuccarino-Catania, G.V. ∙ Sadanand, S. ∙ Weisel, F.J. ...</div><div><strong>CD80 and PD-L2 define functionally distinct memory B cell subsets that are independent of antibody isotype</strong></div><div><em>Nat. Immunol.</em> 2014; <strong>15</strong>:631-637</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ni.2914" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_115_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-84903646784" target="_blank" aria-label="Open Scopus in new tab">Scopus (154)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24880458/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fni.2914&amp;pmid=24880458" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). μMT mice lack mature B cells and antigen-specific antibodies (<span class="dropBlock reference-citations"><a id="body-ref-sref47" href="#" role="doc-biblioref" data-xml-rid="bib47" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref47" aria-controls="sref47">Kitamura et&nbsp;al., 1991</a><div class="dropBlock__holder" data-db-target-of="sref47"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">47.</div><div>Kitamura, D. ∙ Roes, J. ∙ Kühn, R. ...</div><div><strong>A B cell-deficient mouse by targeted disruption of the membrane exon of the immunoglobulin mu chain gene</strong></div><div><em>Nature.</em> 1991; <strong>350</strong>:423-426</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/350423a0" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-0025852445" target="_blank" aria-label="Open Scopus in new tab">Scopus (1459)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/1901381/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2F350423a0&amp;pmid=1901381" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), allowing specific detection of those coming from transferred cells. Recall was assessed 3.5&nbsp;days after immunization of recipients, to exclude confounding factors such as differences in GC expansion that might occur later on. D370Y/WT transferred cells (NP<sup>+</sup>CD45.1<sup>+</sup>) (<a id="crosref1270" href="#figs6">Figure&nbsp;S6</a>M) persisted largely as B220<sup>+</sup> (<a id="crosref1275" href="#fig6">Figure&nbsp;6</a>G) and over 70% of these were FAS<sup>+</sup>GL7<sup>+</sup> (<a id="crosref1280" href="#fig6">Figure&nbsp;6</a>H). The relative total abundance of GCB was almost 2-fold higher in animals transferred with mutant MB than those receiving WT MB (<a id="crosref1285" href="#fig6">Figure&nbsp;6</a>H), indicating that D370Y/WT favors MB reentry into new GC reactions. In contrast, ∼50% of the WT cells downregulated B220 (<a id="crosref1290" href="#fig6">Figure&nbsp;6</a>G), consistent with PC differentiation, as evidenced by CD138 upregulation in some of these (<a id="crosref1295" href="#fig6">Figure&nbsp;6</a>I). Terminal differentiation of D370Y/WT MB into PC was practically null at the assessed time point (<a id="crosref1300" href="#fig6">Figure&nbsp;6</a>I). In agreement, only WT MB led to a detectable secretory recall response in the form of high-affinity IgG1/Lambda Ig (<a id="crosref1305" href="#fig6">Figure&nbsp;6</a>J). These data suggest that TBL1XR1<sup>MUT</sup> causes aberrant expansion of persisting MB and provides the first evidence that a lymphoma mutation can drive preferential reentry of putative clonal precursor cells into subsequent immune reactions.</div></section><section id="sec-2-10"><h3><i>Tbl1xr1</i> Alterations Lead to Extranodal Immunoblastic-like Lymphomas</h3><div id="p0155" role="paragraph">Because TBL1XR1-driven transformation may require GC reentry over an extended period of time, incompatible with mice lifespan (<span class="dropBlock reference-citations"><a id="body-ref-sref103-2" href="#" role="doc-biblioref" data-xml-rid="bib103" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref103-2" aria-controls="sref103-2">Sungalee et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref103-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">101.</div><div>Sungalee, S. ∙ Mamessier, E. ∙ Morgado, E. ...</div><div><strong>Germinal center reentries of BCL2-overexpressing B cells drive follicular lymphoma progression</strong></div><div><em>J.&nbsp;Clin. Invest.</em> 2014; <strong>124</strong>:5337-5351</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1172/JCI72415" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_101_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-84915818499" target="_blank" aria-label="Open Scopus in new tab">Scopus (49)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25384217/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1172%2FJCI72415&amp;pmid=25384217" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), we used <i>CD19cre</i> to increase the frequency of mature B cells carrying <i>Tbl1xr1</i> alterations. This did not affect B cell development (<a id="crosref1315" href="#mmc6">Data S1</a>EE–S1GG) and recapitulated the <i>Cγ1Cre</i> phenotype (<a id="crosref1320" href="#figs7">Figures S7</a>A–S7D). Because C5/MCD carry near-uniform <i>18q</i> gain and high BCL2 levels (<span class="dropBlock reference-citations"><a id="body-ref-sref16-7" href="#" role="doc-biblioref" data-xml-rid="bib16" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref16-7" aria-controls="sref16-7">Chapuy et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref16-7"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">16.</div><div>Chapuy, B. ∙ Stewart, C. ∙ Dunford, A.J. ...</div><div><strong>Molecular subtypes of diffuse large B cell lymphoma are associated with distinct pathogenic mechanisms and outcomes</strong></div><div><em>Nat. Med.</em> 2018; <strong>24</strong>:679-690</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41591-018-0016-8" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-85046087353" target="_blank" aria-label="Open Scopus in new tab">Scopus (257)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29713087/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41591-018-0016-8&amp;pmid=29713087" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref113-1" href="#" role="doc-biblioref" data-xml-rid="bib113" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref113-1" aria-controls="sref113-1">Wright et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref113-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">110.</div><div>Wright, G.W. ∙ Huang, D.W. ∙ Phelan, J.D. ...</div><div><strong>A Probabilistic Classification Tool for Genetic Subtypes of Diffuse Large B Cell Lymphoma with Therapeutic Implications</strong></div><div><em>Cancer Cell.</em> 2020; <strong>37</strong>:551-568.e14</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_110_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=10.1016%2Fj.ccell.2020.03.015&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_110_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=10.1016%2Fj.ccell.2020.03.015&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_110_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-85082775478" target="_blank" aria-label="Open Scopus in new tab">Scopus (12)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32289277/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.ccell.2020.03.015&amp;pmid=32289277" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), we further&nbsp;crossed our animals with <i>VavP</i>-<i>Bcl2</i> mice (<span class="dropBlock reference-citations"><a id="body-ref-sref75-1" href="#" role="doc-biblioref" data-xml-rid="bib75" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref75-1" aria-controls="sref75-1">Ogilvy et&nbsp;al.,&nbsp;1999</a><div class="dropBlock__holder" data-db-target-of="sref75-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">76.</div><div>Ogilvy, S. ∙ Metcalf, D. ∙ Print, C.G. ...</div><div><strong>Constitutive Bcl-2 expression throughout the hematopoietic compartment affects multiple lineages and enhances progenitor cell survival</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 1999; <strong>96</strong>:14943-14948</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.96.26.14943" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_76_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-0033593038" target="_blank" aria-label="Open Scopus in new tab">Scopus (281)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/10611317/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.96.26.14943&amp;pmid=10611317" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) to phenocopy this effect. We generated cohorts&nbsp;of&nbsp;<i>VavP-Bcl2;CD19Cre;Tbl1xr1</i><sup><i>KO/KO</i></sup> and <i>VavP-Bcl2;CD19Cre;Tbl1xr1</i><sup><i>WT/WT</i></sup> mice and immunized them periodically to make sure they generated GCs (<a id="crosref1340" href="#fig7">Figure&nbsp;7</a>A). Mice were sacrificed at a time point when <i>VavP-Bcl2</i> animals show pre-tumoral lymphoproliferation (<span class="dropBlock reference-citations"><a id="body-ref-sref27" href="#" role="doc-biblioref" data-xml-rid="bib27" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref27" aria-controls="sref27">Egle et&nbsp;al., 2004</a><div class="dropBlock__holder" data-db-target-of="sref27"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">27.</div><div>Egle, A. ∙ Harris, A.W. ∙ Bath, M.L. ...</div><div><strong>VavP-Bcl2 transgenic mice develop follicular lymphoma preceded by germinal center hyperplasia</strong></div><div><em>Blood.</em> 2004; <strong>103</strong>:2276-2283</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1182/blood-2003-07-2469" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-1542283724" target="_blank" aria-label="Open Scopus in new tab">Scopus (137)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/14630790/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1182%2Fblood-2003-07-2469&amp;pmid=14630790" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Although all KO/KO animals had developed macroscopic tumors at necropsy, most with extranodal localization such as kidney, lung, liver, and intestines (<a id="crosref1350" href="#fig7">Figure&nbsp;7</a>B), no masses were found in controls. Splenic hyperplasia was evident in all animals but was exacerbated in KO/KO (<a id="crosref1355" href="#fig7">Figure&nbsp;7</a>C). Splenocytes in the control group had the expected representation of T&nbsp;cells (CD3<sup>+</sup>) and B cells (B220<sup>+</sup>) (<a id="crosref1360" href="#fig7">Figure&nbsp;7</a>D) but a massive over-representation of GCB (<a id="crosref1365" href="#fig7">Figure&nbsp;7</a>E). In contrast, KO/KO manifested an expansion of B220<sup>−</sup> cells (<a id="crosref1370" href="#fig7">Figure&nbsp;7</a>D). The few remaining B220<sup>+</sup> KO/KO cells lacked GCB markers for the most part but showed relative expansion of (pre)MB populations (<a id="crosref1375" href="#fig7">Figures 7</a>F and 7G; <a id="crosref1380" href="#mmc6">Data S1</a>HH–S1II). B220<sup>−</sup> KO/KO cells, in turn, expressed higher, albeit variable, levels of CD138 than the controls (<a id="crosref1385" href="#fig7">Figure&nbsp;7</a>H). Both groups showed comparable levels of mature B cells in their BM, suggesting the phenotype was not linked to an early-stage developmental impairment (<a id="crosref1390" href="#mmc6">Data S1</a>JJ–S1KK).</div><div class="figure-wrap"><figure id="fig7" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2020.05.049/asset/4e3c2e94-03c9-4549-9809-3be31d367987/main.assets/gr7_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr7" src="/cms/10.1016/j.cell.2020.05.049/asset/e047e0cd-51ea-4a91-a5b2-ecec5777e824/main.assets/gr7.jpg" height="954" width="676" aria-labelledby="fig7-title fig7-content" data-viewer-src="/cms/10.1016/j.cell.2020.05.049/asset/4e3c2e94-03c9-4549-9809-3be31d367987/main.assets/gr7_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig7-title" class="figure__title"><span class="label figure__label">Figure&nbsp;7</span> <span class="figure__title__text"><i>Tbl1xr1</i> Alterations Lead to Extranodal Immunoblastic-like Lymphomas</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig7-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0305" role="paragraph">(A) Experimental scheme and timeline for (B)–(K), (N), and (O).</div><div id="fspara0310" role="paragraph">(B) Incidence and representative images of macroscopic tumors found in <i>VavP-Bcl2-CD19Cre-Tbl1xr1</i><sup><i>KO/KO</i></sup> mice. Left to right: tumors associated to intestine, lungs, and para-aortic tissues. Numbers represent different animals.</div><div id="fspara0315" role="paragraph">(C) Spleens at time of necropsy. Scale, 500&nbsp;mm. Graph shows spleen to body weight ratios. Dashed line represents average for non-<i>VavP</i>-<i>Bcl2</i> age-matched controls (n&nbsp;= 10).</div><div id="fspara0320" role="paragraph">(D–H) FC analysis of splenic (D) B/T cell populations, (E) GCB, (F) preMB, or (G) MB (B220<sup>+</sup>CD138<sup>−</sup>IgD<sup>−</sup>FAS<sup>−/lo</sup>GL7<sup>−</sup>CD38<sup>+</sup>), or (H) CD138<sup>+</sup> cells. See also <a id="crosref2410" href="#mmc6">Data S1</a>HH–S1II.</div><div id="fspara0325" role="paragraph">(I) H&amp;E of spleen sections from animals in (A). Scale, 500&nbsp;μm (left) or 15&nbsp;μm (center and right). See also <a id="crosref2415" href="#figs7">Figure&nbsp;S7</a>M.</div><div id="fspara0330" role="paragraph">(J and K) H&amp;E of (J) liver or (K) kidney sections from animals in (A). Scale, 500&nbsp;μm (left), 100&nbsp;μm (center), and 15&nbsp;μm (right).</div><div id="fspara0335" role="paragraph">(L) Experimental scheme and timeline for (M).</div><div id="fspara0340" role="paragraph">(M) Survival curve for <i>VavP-Bcl2;CD19Cre-Tbl1xr1</i><sup><i>KO/KO</i></sup> and <i>VavP-Bcl2;CD19Cre-Tbl1xr1</i><sup><i>WT/WT</i></sup> mice.</div><div id="fspara0345" role="paragraph">(N) SHM burden at Ig intron J<sub>H</sub>4 in extranodal tumors from (A). Numbers in square brackets are total numbers of clones, pooled for 3 animals. <i>Aicda</i><sup>KO/KO</sup> GCB were used as control.</div><div id="fspara0350" role="paragraph">(O) Mutation burden at <i>Pim1</i> locus in tumors from (A). Numbers in square brackets represent total clones, pooled for n&nbsp;= 5 per genotype.</div><div id="fspara0355" role="paragraph">(P) Representative images and quantification of CD38 IHC for MCD/C5 and non-MCD/C5 ABC-DLBCL human specimens. Scale, 20&nbsp;μm.</div><div id="fspara0360" role="paragraph">(Q) Left: surface marker contribution to PCA loading. Right: PCA plots for MB and GCB from reactive LN, used as reference in CyTOF experiments.</div><div id="fspara0365" role="paragraph">(R and S) PCA for CyTOF profiling of human DLBCL specimens carrying canonical MCD/C5 lesions and (R) <i>TBL1XR1</i> mutations or (S) <i>TBL1XR1</i> WT alleles. See also <a id="crosref2420" href="#mmc5">Table S5</a>.</div><div id="fspara0370" role="paragraph">Values represent mean ± SEM. Data reproducible with two repeats. p values calculated using unpaired two-tailed Student’s t test (D)–(H), log-rank test (M), Wilcoxon rank-sum test with (O) or without (N) BH correction, or two-sided Chi-square test (P).</div></div></div></div></figcaption></figure></div><div class="figure-wrap"><figure id="figs7" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2020.05.049/asset/33e270e2-34ab-4c1b-b15e-a4a467df71b4/main.assets/figs7_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs7" src="/cms/10.1016/j.cell.2020.05.049/asset/0a5f9edf-ce98-42a9-b553-c6255571d74e/main.assets/figs7.jpg" height="952" width="636" aria-labelledby="figs7-title figs7-content" data-viewer-src="/cms/10.1016/j.cell.2020.05.049/asset/33e270e2-34ab-4c1b-b15e-a4a467df71b4/main.assets/figs7_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs7-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S7</span> <span class="figure__title__text">Lymphomagenic Effects of&nbsp;<i>Tbl1xr1&nbsp;G</i>enetic Lesions, Related to <a id="crosref2505" href="#fig7">Figure&nbsp;7</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs7-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0735" role="paragraph">A-B, Representative FC plots (left) and quantification (right) of (A) total splenic B cells or (B) GCB in <i>CD19Cre-Tbl1xr1</i><sup><i>KO/KO</i></sup> (KO/KO) or <i>CD19Cre-Tbl1xr1</i><sup><i>WT/WT</i></sup> (WT/WT) mice.</div><div id="fspara0740" role="paragraph">C, Representative FC plots and quantification of (top) EFNB1+ or (bottom) preMB populations in splenic GCB for experiment in (A).</div><div id="fspara0745" role="paragraph">D, Representative FC plots (left) and quantification (right) of splenic PC for experiment in (A).</div><div id="fspara0750" role="paragraph">(E), Representative H&amp;E images of ABC-DLBCL tumors from 4 independent patients, from Scott &amp; Morin cohort (<span class="dropBlock reference-citations"><a id="body-ref-sref5-2" href="#" role="doc-biblioref" data-xml-rid="bib5" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref5-2" aria-controls="sref5-2">Arthur et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref5-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">5.</div><div>Arthur, S.E. ∙ Jiang, A. ∙ Grande, B.M. ...</div><div><strong>Genome-wide discovery of somatic regulatory variants in diffuse large B-cell lymphoma</strong></div><div><em>Nat. Commun.</em> 2018; <strong>9</strong>:4001</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41467-018-06354-3" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-85054070552" target="_blank" aria-label="Open Scopus in new tab">Scopus (32)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30275490/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41467-018-06354-3&amp;pmid=30275490" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Scale bars&nbsp;= 20μm.</div><div id="fspara0755" role="paragraph">F-G, Representative images of B220, CD3, CD138 and KI67 IHC staining in (F) spleen or (G) kidney sections from animals in <a id="crosref2515" href="#fig7">Figure&nbsp;7</a>A. Scale bars&nbsp;= 500μm (left) or 15μm (insets).</div><div id="fspara0760" role="paragraph">H-I, <i>VavP-Bcl2;Cγ1Cre;Tbl1xr1</i><sup><i>KO/KO</i></sup> and <i>VavP-Bcl2;Cγ1Cre;Tbl1xr1</i><sup><i>WT/WT</i></sup> mice [n&nbsp;= 10 per genotype] received monthly SRBC immunizations, to induce GC formation, from 2&nbsp;m of age until time of sacrifice. Mice were euthanized at 11&nbsp;m of age, and soft organs were collected for pathology studies. (H) Representative H&amp;E images of spleen, lung and liver sections. (I) Representative images of B220 IHC staining in consecutive sections from specimens in (H). Insets (right) show zoom of outlined areas. Scale bars&nbsp;= 500μm (left) or 20μm (insets).</div><div id="fspara0765" role="paragraph">J, Representative images (left) and relative quantification (right) of CD38 IHC staining for <i>TBL1XR1</i> mutant or WT DLBCL human specimens, from the Scott &amp; Morin cohort (<span class="dropBlock reference-citations"><a id="body-ref-sref5-3" href="#" role="doc-biblioref" data-xml-rid="bib5" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref5-3" aria-controls="sref5-3">Arthur et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref5-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">5.</div><div>Arthur, S.E. ∙ Jiang, A. ∙ Grande, B.M. ...</div><div><strong>Genome-wide discovery of somatic regulatory variants in diffuse large B-cell lymphoma</strong></div><div><em>Nat. Commun.</em> 2018; <strong>9</strong>:4001</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41467-018-06354-3" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-85054070552" target="_blank" aria-label="Open Scopus in new tab">Scopus (32)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30275490/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41467-018-06354-3&amp;pmid=30275490" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Scale bars&nbsp;= 20μm.</div><div id="fspara0770" role="paragraph">Values represent mean ± SEM. All the data were reproducible with at least two repeats. NS, not significant; <sup>∗</sup>p&nbsp;&lt; 0.05; <sup>∗∗</sup>p&nbsp;&lt; 0.01; <sup>∗∗∗</sup>p&nbsp;&lt; 0.001, using unpaired two-tailed Student’s t test.</div></div></div></div></figcaption></figure></div><div id="p0160" role="paragraph">Histological analysis of lymphoid and other tissues in KO/KO mice revealed predominantly large and highly atypical immunoblasts, often with large and irregularly shaped nuclei and a moderate amount of cytoplasm, mimicking the appearance of human extranodal ABC-DLBCLs (<a id="crosref1395" href="#figs7">Figure&nbsp;S7</a>E). In the spleen, these cells were seen predominantly outside the follicles in the red pulp, disrupting the splenic architecture (<a id="crosref1400" href="#fig7">Figure&nbsp;7</a>I). These cells also extensively infiltrated and distorted many tissues including liver (<a id="crosref1405" href="#fig7">Figure&nbsp;7</a>J) and kidneys (<a id="crosref1410" href="#fig7">Figure&nbsp;7</a>K). Consistent with this extranodal scenario, lymph nodes (LN) were only focally involved. Immunoblastic cells in KO/KO specimens were virtually all Ki67<sup>+</sup> and many stained for CD138 (<a id="crosref1415" href="#figs7">Figures S7</a>F and S7G), consistent with the notion that TBL1XR1 loss results in a post-GC lymphoma phenotype. In comparison, the overall architecture of the liver, spleen, kidneys, and lung was relatively intact for <i>Tbl1xr1</i><sup><i>WT/WT</i></sup> (<a id="crosref1420" href="#fig7">Figures 7</a>I–7K), although LNs showed variable sinusoidal expansion by smaller infiltrating lymphocytes. Analysis of tissues yielded sporadic, small, focal lymphocytic infiltrates linked to blood vessels (<a id="crosref1425" href="#fig7">Figures 7</a>J and 7K) in the LN, spleen, and BM that rarely affected tissue architecture (<a id="crosref1430" href="#fig7">Figure&nbsp;7</a>I and not shown). IHC revealed infiltrates were composed of T and B cells, as well as PC (<a id="crosref1435" href="#figs7">Figures S7</a>F and S7G). Ki67 stain was low for small plasmacytoid cells/plasma cells and lymphoid cells and higher in the few larger cells (<a id="crosref1440" href="#figs7">Figures S7</a>F and S7G). To assess whether the disease translated into accelerated lethality, we conducted a survival study in an additional cohort of mice, immunized to induce GC formation (<a id="crosref1445" href="#fig7">Figure&nbsp;7</a>L). The study revealed that KO/KO mice died of their disease significantly sooner than WT (<a id="crosref1450" href="#fig7">Figure&nbsp;7</a>M).</div><div id="p0165" role="paragraph">Human C5/MCD DLBCLs show the highest AID footprint (<span class="dropBlock reference-citations"><a id="body-ref-sref16-8" href="#" role="doc-biblioref" data-xml-rid="bib16" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref16-8" aria-controls="sref16-8">Chapuy et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref16-8"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">16.</div><div>Chapuy, B. ∙ Stewart, C. ∙ Dunford, A.J. ...</div><div><strong>Molecular subtypes of diffuse large B cell lymphoma are associated with distinct pathogenic mechanisms and outcomes</strong></div><div><em>Nat. Med.</em> 2018; <strong>24</strong>:679-690</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41591-018-0016-8" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-85046087353" target="_blank" aria-label="Open Scopus in new tab">Scopus (257)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29713087/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41591-018-0016-8&amp;pmid=29713087" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), suggesting they originate from GC-transitioned cells. We examined Ig SHM in our KO/KO mouse lymphomas, to exclude that these derived from extrafollicular responses. Sequencing of intron J<sub>H</sub>4 revealed significant SHM burden in tumors that was absent in <i>Acida</i><sup><i>KO/KO</i></sup> GCB used as controls (<a id="crosref1460" href="#fig7">Figure&nbsp;7</a>N). We then performed targeted sequencing of a region in the <i>Pim1</i> locus frequently impacted by off-target AID activity (<span class="dropBlock reference-citations"><a id="body-ref-sref58-1" href="#" role="doc-biblioref" data-xml-rid="bib58" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref58-1" aria-controls="sref58-1">Liu et&nbsp;al., 2008</a><div class="dropBlock__holder" data-db-target-of="sref58-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">58.</div><div>Liu, M. ∙ Duke, J.L. ∙ Richter, D.J. ...</div><div><strong>Two levels of protection for the B cell genome during somatic hypermutation</strong></div><div><em>Nature.</em> 2008; <strong>451</strong>:841-845</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature06547" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-39149099085" target="_blank" aria-label="Open Scopus in new tab">Scopus (411)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/18273020/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature06547&amp;pmid=18273020" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), because <i>PIM1</i> mutations are a hallmark of C5/MCD tumors and frequently co-occur with <i>TBL1XR1</i> mutations (<a id="crosref1470" href="#mmc6">Data S1</a>C). Notably, KO/KO tumors accumulated more mutations than <i>VavP-Bcl2</i> controls (<a id="crosref1475" href="#fig7">Figure&nbsp;7</a>O). Strikingly, the highest rate of off-target mutations was detected in extranodal tumors from&nbsp;KO/KO mice (<a id="crosref1480" href="#fig7">Figure&nbsp;7</a>O). To further confirm that these lymphomas could arise from GCB, we generated <i>C</i>γ<i>1Cre;VavP-Bcl2;Tbl1xr1</i><sup><i>KO/KO</i></sup> mice. These mice again showed extensive extranodal infiltrates, composed of a heterogeneous mixture of lymphocytic and immunoblastic populations (<a id="crosref1485" href="#figs7">Figure&nbsp;S7</a>H), with a higher percentage of B220<sup>+</sup> cells than the <i>CD19Cre</i> variant (<a id="crosref1490" href="#figs7">Figure&nbsp;S7</a>I). Hence, the more homogeneous plasmacytic-like presentation in <i>CD19Cre</i> mice is not required for transformation or extranodal dissemination of malignant B cells but might instead represent a later or alternative stage in disease progression. These data are consistent with TBL1XR1 driving MCD/C5 pathogenesis by inducing MB cell reentry into successive GC reactions.</div></section><section id="sec-2-11"><h3>Human MCD/C5 Tumors Show Evidence of MB Origin</h3><div id="p0170" role="paragraph">If MB served as COO for C5/MCD lymphomas, we would expect to find phenotypic evidence in clinical specimens. In humans, CD38 is expressed in GCB and PC, but lost in MB (<span class="dropBlock reference-citations"><a id="body-ref-sref4" href="#" role="doc-biblioref" data-xml-rid="bib4" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref4" aria-controls="sref4">Arpin et&nbsp;al., 1995</a><div class="dropBlock__holder" data-db-target-of="sref4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">4.</div><div>Arpin, C. ∙ Déchanet, J. ∙ Van Kooten, C. ...</div><div><strong>Generation of memory B cells and plasma cells in&nbsp;vitro</strong></div><div><em>Science.</em> 1995; <strong>268</strong>:720-722</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.7537388" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/7537388/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.7537388&amp;pmid=7537388" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Performing IHC in a DLBCL patient's cohort (<span class="dropBlock reference-citations"><a id="body-ref-sref5-4" href="#" role="doc-biblioref" data-xml-rid="bib5" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref5-4" aria-controls="sref5-4">Arthur et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref5-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">5.</div><div>Arthur, S.E. ∙ Jiang, A. ∙ Grande, B.M. ...</div><div><strong>Genome-wide discovery of somatic regulatory variants in diffuse large B-cell lymphoma</strong></div><div><em>Nat. Commun.</em> 2018; <strong>9</strong>:4001</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41467-018-06354-3" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-85054070552" target="_blank" aria-label="Open Scopus in new tab">Scopus (32)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30275490/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41467-018-06354-3&amp;pmid=30275490" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), we observed that although ∼40% of all <i>TBL1XR1</i><sup>WT</sup> DLBCLs were CD38<sup>+</sup>, only ∼10% of <i>TBL1XR1</i><sup>MUT</sup> cases were CD38<sup>+</sup> (p&nbsp;= 0.03) (<a id="crosref1505" href="#figs7">Figure&nbsp;S7</a>J). We further subclassified (<span class="dropBlock reference-citations"><a id="body-ref-sref113-2" href="#" role="doc-biblioref" data-xml-rid="bib113" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref113-2" aria-controls="sref113-2">Wright et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref113-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">110.</div><div>Wright, G.W. ∙ Huang, D.W. ∙ Phelan, J.D. ...</div><div><strong>A Probabilistic Classification Tool for Genetic Subtypes of Diffuse Large B Cell Lymphoma with Therapeutic Implications</strong></div><div><em>Cancer Cell.</em> 2020; <strong>37</strong>:551-568.e14</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_110_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=10.1016%2Fj.ccell.2020.03.015&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_110_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=10.1016%2Fj.ccell.2020.03.015&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_110_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-85082775478" target="_blank" aria-label="Open Scopus in new tab">Scopus (12)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32289277/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.ccell.2020.03.015&amp;pmid=32289277" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) tumors in this cohort and observed significant and striking differences between MCD cases (0% CD38<sup>+</sup>) and non-MCD ABC-DLBCL cases (∼30% CD38<sup>+</sup>) (p&nbsp;= 0.01) (<a id="crosref1515" href="#fig7">Figure&nbsp;7</a>P).</div><div id="p0175" role="paragraph">To further support the notion that C5/MCD reflects a MB COO, we conducted CyTOF on patients from an independent DLBCL cohort (<a id="crosref2535" href="#mmc1">Table S6</a>) (<span class="dropBlock reference-citations"><a id="body-ref-sref71-1" href="#" role="doc-biblioref" data-xml-rid="bib71" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref71-1" aria-controls="sref71-1">Nissen et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref71-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">72.</div><div>Nissen, M.D. ∙ Kusakabe, M. ∙ Wang, X. ...</div><div><strong>Single Cell Phenotypic Profiling of 27 DLBCL Cases Reveals Marked Intertumoral and Intratumoral Heterogeneity</strong></div><div><em>Cytometry A.</em> 2019;</div><div>Published online October 22, 2019</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/cyto.a.23919" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-85075788395" target="_blank" aria-label="Open Scopus in new tab">Scopus (2)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31637838/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fcyto.a.23919&amp;pmid=31637838" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Cells from reactive LN specimens were used to define B cell subpopulations. In these samples, GCB were identified as CD10<sup>hi</sup>CD38<sup>hi</sup>, whereas MB showed lower levels of these markers and distinctive CD27 upregulation (<a id="crosref1530" href="#fig7">Figure&nbsp;7</a>Q) (<span class="dropBlock reference-citations"><a id="body-ref-sref1" href="#" role="doc-biblioref" data-xml-rid="bib1" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref1" aria-controls="sref1">Agematsu et&nbsp;al., 2000</a><div class="dropBlock__holder" data-db-target-of="sref1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">1.</div><div>Agematsu, K. ∙ Hokibara, S. ∙ Nagumo, H. ...</div><div><strong>CD27: a memory B-cell marker</strong></div><div><em>Immunol. Today.</em> 2000; <strong>21</strong>:204-206</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_1_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=10.1016%2FS0167-5699%2800%2901605-4&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_1_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=10.1016%2FS0167-5699%2800%2901605-4&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_1_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-0034102137" target="_blank" aria-label="Open Scopus in new tab">Scopus (288)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/10782048/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0167-5699%2800%2901605-4&amp;pmid=10782048" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). GCB-DLBCL cases, included as controls, mapped to GCB reference populations (not shown). In contrast, ABC-DLBCL tumors carrying <i>TBL1XR1</i> and MCD/C5 canonical mutations narrowly recapitulate the surface marker profile of MB cells (<a id="crosref1540" href="#fig7">Figure&nbsp;7</a>R). The MB profile was also observed in MCD DLBCLs without <i>TBL1XR1</i> mutations (<a id="crosref1545" href="#fig7">Figure&nbsp;7</a>S), consistent with the notion that MCD tumors arise from MB, albeit through distinct mutational trajectories. Collectively, we find that TBL1XR1 LOF drives expansion of aberrant MB cells with enhanced tendency to reenter GC reactions, ultimately resulting in canonical post-GC extranodal ABC-DLBCL with MB-like features and revealing the manner in which these tumors arise naturally from the humoral immune response.</div></section></section><section id="sec-3"><h2>Discussion</h2><div id="p0180" role="paragraph">Here, we show how <i>TBL1XR1</i> mutations result in transcriptional re-wiring of GCB cells to drive lymphomagenesis. A previous report noted TBL1XR1 interaction with BCL6 in a lymphoma cell line, using mass spectrometry (<span class="dropBlock reference-citations"><a id="body-ref-sref65" href="#" role="doc-biblioref" data-xml-rid="bib65" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref65" aria-controls="sref65">Miles et&nbsp;al., 2005</a><div class="dropBlock__holder" data-db-target-of="sref65"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">65.</div><div>Miles, R.R. ∙ Crockett, D.K. ∙ Lim, M.S. ...</div><div><strong>Analysis of BCL6-interacting proteins by tandem mass spectrometry</strong></div><div><em>Mol. Cell. Proteomics.</em> 2005; <strong>4</strong>:1898-1909</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1074/mcp.M500112-MCP200" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-29244465090" target="_blank" aria-label="Open Scopus in new tab">Scopus (28)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16147992/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1074%2Fmcp.M500112-MCP200&amp;pmid=16147992" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). We observed that TBL1XR1 forms complexes with BCL6 at the chromatin level, and its interactome is largely confined to the SMRT complex and the TFs BCL6 and BACH2. In GCB, BCL6-SMRT complexes transiently poise enhancers controlling terminal differentiation, immune synapse signaling, and cell-cycle checkpoint genes, thus enabling proliferative bursts (<span class="dropBlock reference-citations"><a id="body-ref-sref35-4" href="#" role="doc-biblioref" data-xml-rid="bib35" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref35-4" aria-controls="sref35-4">Hatzi et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref35-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">35.</div><div>Hatzi, K. ∙ Jiang, Y. ∙ Huang, C. ...</div><div><strong>A hybrid mechanism of action for BCL6 in B cells defined by formation of functionally distinct complexes at enhancers and promoters</strong></div><div><em>Cell Rep.</em> 2013; <strong>4</strong>:578-588</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=10.1016%2Fj.celrep.2013.06.016&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=10.1016%2Fj.celrep.2013.06.016&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-84881615483" target="_blank" aria-label="Open Scopus in new tab">Scopus (90)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23911289/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.celrep.2013.06.016&amp;pmid=23911289" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref43-2" href="#" role="doc-biblioref" data-xml-rid="bib43" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref43-2" aria-controls="sref43-2">Jiang et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref43-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">43.</div><div>Jiang, Y. ∙ Ortega-Molina, A. ∙ Geng, H. ...</div><div><strong>CREBBP Inactivation Promotes the Development of HDAC3-Dependent Lymphomas</strong></div><div><em>Cancer Discov.</em> 2017; <strong>7</strong>:38-53</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1158/2159-8290.CD-16-0975" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-85009410568" target="_blank" aria-label="Open Scopus in new tab">Scopus (92)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27733359/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1158%2F2159-8290.CD-16-0975&amp;pmid=27733359" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Accordingly, the GC reaction in <i>Tbl1xr1</i> LOF mice was significantly impaired, and GCB showed upregulation of BCL6 targets. Whereas <i>Bcl6</i>-KO completely abrogates GC formation (<span class="dropBlock reference-citations"><a id="body-ref-sref34-3" href="#" role="doc-biblioref" data-xml-rid="bib34" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref34-3" aria-controls="sref34-3">Hatzi and Melnick, 2014</a><div class="dropBlock__holder" data-db-target-of="sref34-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">34.</div><div>Hatzi, K. ∙ Melnick, A.</div><div><strong>Breaking bad in the germinal center: how deregulation of BCL6 contributes to lymphomagenesis</strong></div><div><em>Trends Mol. Med.</em> 2014; <strong>20</strong>:343-352</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=10.1016%2Fj.molmed.2014.03.001&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=10.1016%2Fj.molmed.2014.03.001&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-84901585384" target="_blank" aria-label="Open Scopus in new tab">Scopus (80)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24698494/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.molmed.2014.03.001&amp;pmid=24698494" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), <i>Tbl1xr1</i> LOF effects are reminiscent of the milder effects observed in mice carrying NCOR/SMRT LOF mutations that disrupt interaction with HDAC3 (<span class="dropBlock reference-citations"><a id="body-ref-sref43-3" href="#" role="doc-biblioref" data-xml-rid="bib43" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref43-3" aria-controls="sref43-3">Jiang et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref43-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">43.</div><div>Jiang, Y. ∙ Ortega-Molina, A. ∙ Geng, H. ...</div><div><strong>CREBBP Inactivation Promotes the Development of HDAC3-Dependent Lymphomas</strong></div><div><em>Cancer Discov.</em> 2017; <strong>7</strong>:38-53</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1158/2159-8290.CD-16-0975" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-85009410568" target="_blank" aria-label="Open Scopus in new tab">Scopus (92)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27733359/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1158%2F2159-8290.CD-16-0975&amp;pmid=27733359" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) or <i>Hdac3</i> conditional deletion (<span class="dropBlock reference-citations"><a id="body-ref-sref98" href="#" role="doc-biblioref" data-xml-rid="bib98" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref98" aria-controls="sref98">Stengel et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref98"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">96.</div><div>Stengel, K.R. ∙ Bhaskara, S. ∙ Wang, J. ...</div><div><strong>Histone deacetylase 3 controls a transcriptional network required for B cell maturation</strong></div><div><em>Nucleic Acids Res.</em> 2019; <strong>47</strong>:10612-10627</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gkz816" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_96_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-85074875427" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31586401/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgkz816&amp;pmid=31586401" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The difference in severity between BCL6 versus TBL1XR1/SMRT complex deficiency phenotype is likely due to SMRT/HDAC3-independent BCL6 functions such as interaction with BCOR and LSD1 (<span class="dropBlock reference-citations"><a id="body-ref-sref36-2" href="#" role="doc-biblioref" data-xml-rid="bib36" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref36-2" aria-controls="sref36-2">Hatzi et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref36-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">36.</div><div>Hatzi, K. ∙ Geng, H. ∙ Doane, A.S. ...</div><div><strong>Histone demethylase LSD1 is required for germinal center formation and BCL6-driven lymphomagenesis</strong></div><div><em>Nat. Immunol.</em> 2019; <strong>20</strong>:86-96</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41590-018-0273-1" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-85058423458" target="_blank" aria-label="Open Scopus in new tab">Scopus (14)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30538335/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41590-018-0273-1&amp;pmid=30538335" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="p0185" role="paragraph">TBL1XR1’s lack of enzymatic activity has largely hindered efforts to elucidate its function. Still, one study proposed that TBL1XR1 acts as a repressor/coactivator exchange factor for hormone receptors (<span class="dropBlock reference-citations"><a id="body-ref-sref80-3" href="#" role="doc-biblioref" data-xml-rid="bib80" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref80-3" aria-controls="sref80-3">Perissi et&nbsp;al., 2004</a><div class="dropBlock__holder" data-db-target-of="sref80-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">81.</div><div>Perissi, V. ∙ Aggarwal, A. ∙ Glass, C.K. ...</div><div><strong>A corepressor/coactivator exchange complex required for transcriptional activation by nuclear receptors and other regulated transcription factors</strong></div><div><em>Cell.</em> 2004; <strong>116</strong>:511-526</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=10.1016%2FS0092-8674%2804%2900133-3&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=10.1016%2FS0092-8674%2804%2900133-3&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-1342264315" target="_blank" aria-label="Open Scopus in new tab">Scopus (412)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/14980219/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0092-8674%2804%2900133-3&amp;pmid=14980219" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). In this model, TBL1XR1 recruited UbcH5 and the 19S proteasome, leading to SMRT degradation. This finding was not reproduced by later studies (<span class="dropBlock reference-citations"><a id="body-ref-sref39" href="#" role="doc-biblioref" data-xml-rid="bib39" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref39" aria-controls="sref39">Huang et&nbsp;al., 2009</a><div class="dropBlock__holder" data-db-target-of="sref39"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">39.</div><div>Huang, W. ∙ Ghisletti, S. ∙ Perissi, V. ...</div><div><strong>Transcriptional integration of TLR2 and TLR4 signaling at the NCoR derepression checkpoint</strong></div><div><em>Mol. Cell.</em> 2009; <strong>35</strong>:48-57</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=10.1016%2Fj.molcel.2009.05.023&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=10.1016%2Fj.molcel.2009.05.023&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-67649668789" target="_blank" aria-label="Open Scopus in new tab">Scopus (80)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19595715/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.molcel.2009.05.023&amp;pmid=19595715" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref115" href="#" role="doc-biblioref" data-xml-rid="bib115" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref115" aria-controls="sref115">Yoon et&nbsp;al., 2005</a><div class="dropBlock__holder" data-db-target-of="sref115"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">112.</div><div>Yoon, H.G. ∙ Choi, Y. ∙ Cole, P.A. ...</div><div><strong>Reading and function of a histone code involved in targeting corepressor complexes for repression</strong></div><div><em>Mol. Cell. Biol.</em> 2005; <strong>25</strong>:324-335</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/MCB.25.1.324-335.2005" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_112_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-11144333630" target="_blank" aria-label="Open Scopus in new tab">Scopus (83)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15601853/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FMCB.25.1.324-335.2005&amp;pmid=15601853" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref117" href="#" role="doc-biblioref" data-xml-rid="bib117" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref117" aria-controls="sref117">Zhang et&nbsp;al., 2006</a><div class="dropBlock__holder" data-db-target-of="sref117"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">114.</div><div>Zhang, X.M. ∙ Chang, Q. ∙ Zeng, L. ...</div><div><strong>TBLR1 regulates the expression of nuclear hormone receptor co-repressors</strong></div><div><em>BMC Cell Biol.</em> 2006; <strong>7</strong>:31</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/1471-2121-7-31" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_114_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-33748053323" target="_blank" aria-label="Open Scopus in new tab">Scopus (29)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16893456/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2F1471-2121-7-31&amp;pmid=16893456" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), and our own data show that in B cells, TBL1XR1<sup>MUT</sup> had no effect on SMRT protein levels. Furthermore, our analysis of TBL1XR1 interactome failed to identify ubiquitin or proteasome machinery. In agreement with this, acute CD40L signaling (a differentiation cue for GCB) evicts SMRT from chromatin, without affecting its stability (<span class="dropBlock reference-citations"><a id="body-ref-sref83" href="#" role="doc-biblioref" data-xml-rid="bib83" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref83" aria-controls="sref83">Polo et&nbsp;al., 2008</a><div class="dropBlock__holder" data-db-target-of="sref83"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">83.</div><div>Polo, J.M. ∙ Ci, W. ∙ Licht, J.D. ...</div><div><strong>Reversible disruption of BCL6 repression complexes by CD40 signaling in normal and malignant B cells</strong></div><div><em>Blood.</em> 2008; <strong>112</strong>:644-651</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1182/blood-2008-01-131813" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_83_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-50949097945" target="_blank" aria-label="Open Scopus in new tab">Scopus (24)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/18487509/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1182%2Fblood-2008-01-131813&amp;pmid=18487509" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Rather than altering SMRT turnover, <i>TBL1XR1</i> mutation or KO instead shift SMRT complex from BCL6 to BACH2, resulting in transcriptional reprogramming.</div><div id="p0190" role="paragraph">WD40 domains function as PPI or protein-DNA interaction motifs, mainly through aromatic residues on the surface of the β-propeller (reviewed in <span class="dropBlock reference-citations"><a id="body-ref-sref42" href="#" role="doc-biblioref" data-xml-rid="bib42" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref42" aria-controls="sref42">Jain and Pandey, 2018</a><div class="dropBlock__holder" data-db-target-of="sref42"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">42.</div><div>Jain, B.P. ∙ Pandey, S.</div><div><strong>WD40 Repeat Proteins: Signalling Scaffold with Diverse Functions</strong></div><div><em>Protein J.</em> 2018; <strong>37</strong>:391-406</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/s10930-018-9785-7" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-85051176958" target="_blank" aria-label="Open Scopus in new tab">Scopus (32)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30069656/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2Fs10930-018-9785-7&amp;pmid=30069656" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Notably, these TBL1XR1 residues are the most frequently mutated in DLBCL. Structural studies show that TBL1XR1 forms tetramers and binds SMRT through its N-terminal domain (<span class="dropBlock reference-citations"><a id="body-ref-sref73" href="#" role="doc-biblioref" data-xml-rid="bib73" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref73" aria-controls="sref73">Oberoi et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref73"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">74.</div><div>Oberoi, J. ∙ Fairall, L. ∙ Watson, P.J. ...</div><div><strong>Structural basis for the assembly of the SMRT/NCoR core transcriptional repression machinery</strong></div><div><em>Nat. Struct. Mol. Biol.</em> 2011; <strong>18</strong>:177-184</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nsmb.1983" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-79551626803" target="_blank" aria-label="Open Scopus in new tab">Scopus (89)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21240272/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnsmb.1983&amp;pmid=21240272" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref116-2" href="#" role="doc-biblioref" data-xml-rid="bib116" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref116-2" aria-controls="sref116-2">Zhang et&nbsp;al., 2002</a><div class="dropBlock__holder" data-db-target-of="sref116-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">113.</div><div>Zhang, J. ∙ Kalkum, M. ∙ Chait, B.T. ...</div><div><strong>The N-CoR-HDAC3 nuclear receptor corepressor complex inhibits the JNK pathway through the integral subunit GPS2</strong></div><div><em>Mol. Cell.</em> 2002; <strong>9</strong>:611-623</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_113_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=10.1016%2FS1097-2765%2802%2900468-9&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_113_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=10.1016%2FS1097-2765%2802%2900468-9&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_113_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-0036211850" target="_blank" aria-label="Open Scopus in new tab">Scopus (305)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11931768/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS1097-2765%2802%2900468-9&amp;pmid=11931768" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), which is largely spared by mutations. Hence, WD40 mutations would not be predicted to disrupt TBL1XR1-SMRT interaction. Accordingly, a study on Pierpont syndrome found that the interaction between TBL1XR1<sup>Y446C</sup> (a mutant also found in DLBCL) and SMRT/HDAC3, was maintained <i>in&nbsp;vitro</i> (<span class="dropBlock reference-citations"><a id="body-ref-sref37-2" href="#" role="doc-biblioref" data-xml-rid="bib37" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref37-2" aria-controls="sref37-2">Heinen et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref37-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">37.</div><div>Heinen, C.A. ∙ Jongejan, A. ∙ Watson, P.J. ...</div><div><strong>A specific mutation in TBL1XR1 causes Pierpont syndrome</strong></div><div><em>J.&nbsp;Med. Genet.</em> 2016; <strong>53</strong>:330-337</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1136/jmedgenet-2015-103233" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-84957921471" target="_blank" aria-label="Open Scopus in new tab">Scopus (32)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26769062/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1136%2Fjmedgenet-2015-103233&amp;pmid=26769062" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The fact that TBL1XR1<sup>MUT</sup> remains within the SMRT complex suggests that the WD40 plays a critical role in facilitating preferential association with BCL6, because TBL1XR1 KO yields a similar effect, both from the biological and biochemical standpoints. Importantly, the overall structure of the SMRT complex has not been resolved, but it will be necessary to understand how the holo-complex associates with BCL6 in the chromatin context and why TBL1XR1 LOF switches the conformation of the complex to favor binding to BACH2.</div><div id="p0195" role="paragraph">During the humoral immune response, B cells can transition to MB and PC phenotypes, with implications for formation of clonal precursor cells. Previous studies suggested a role for IL-9R signaling (<span class="dropBlock reference-citations"><a id="body-ref-sref110-3" href="#" role="doc-biblioref" data-xml-rid="bib110" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref110-3" aria-controls="sref110-3">Wang et&nbsp;al., 2017b</a><div class="dropBlock__holder" data-db-target-of="sref110-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">107.</div><div>Wang, Y. ∙ Shi, J. ∙ Yan, J. ...</div><div><strong>Germinal-center development of memory B cells driven by IL-9 from follicular helper T&nbsp;cells</strong></div><div><em>Nat. Immunol.</em> 2017; <strong>18</strong>:921-930</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ni.3788" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_107_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-85025817827" target="_blank" aria-label="Open Scopus in new tab">Scopus (58)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28650481/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fni.3788&amp;pmid=28650481" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), or the relative level of BCR affinity for antigen (<span class="dropBlock reference-citations"><a id="body-ref-sref96-3" href="#" role="doc-biblioref" data-xml-rid="bib96" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref96-3" aria-controls="sref96-3">Shinnakasu et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref96-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">94.</div><div>Shinnakasu, R. ∙ Inoue, T. ∙ Kometani, K. ...</div><div><strong>Regulated selection of germinal-center cells into the memory B cell compartment</strong></div><div><em>Nat. Immunol.</em> 2016; <strong>17</strong>:861-869</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ni.3460" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_94_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-84966559006" target="_blank" aria-label="Open Scopus in new tab">Scopus (208)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27158841/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fni.3460&amp;pmid=27158841" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), as factors driving GCB into the (pre)MB compartment. In the case of TBL1XR1 LOF, preMB expansion is caused by switching the SMRT complex to BACH2 and away from BCL6. This situation is different from simple de-repression of BCL6 target genes, which can enable differentiation to either PC or MB cell fate (<span class="dropBlock reference-citations"><a id="body-ref-sref23" href="#" role="doc-biblioref" data-xml-rid="bib23" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref23" aria-controls="sref23">Diehl et&nbsp;al., 2008</a><div class="dropBlock__holder" data-db-target-of="sref23"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">23.</div><div>Diehl, S.A. ∙ Schmidlin, H. ∙ Nagasawa, M. ...</div><div><strong>STAT3-mediated up-regulation of BLIMP1 Is coordinated with BCL6 down-regulation to control human plasma cell differentiation</strong></div><div><em>J.&nbsp;Immunol.</em> 2008; <strong>180</strong>:4805-4815</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.4049/jimmunol.180.7.4805" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/18354204/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.4049%2Fjimmunol.180.7.4805&amp;pmid=18354204" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref41" href="#" role="doc-biblioref" data-xml-rid="bib41" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref41" aria-controls="sref41">Ise et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref41"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">41.</div><div>Ise, W. ∙ Fujii, K. ∙ Shiroguchi, K. ...</div><div><strong>T Follicular Helper Cell-Germinal Center B Cell Interaction Strength Regulates Entry into Plasma Cell or Recycling Germinal Center Cell Fate</strong></div><div><em>Immunity.</em> 2018; <strong>48</strong>:702-715</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=10.1016%2Fj.immuni.2018.03.027&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=10.1016%2Fj.immuni.2018.03.027&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-85045259028" target="_blank" aria-label="Open Scopus in new tab">Scopus (55)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29669250/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.immuni.2018.03.027&amp;pmid=29669250" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref52" href="#" role="doc-biblioref" data-xml-rid="bib52" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref52" aria-controls="sref52">Kuo et&nbsp;al., 2007</a><div class="dropBlock__holder" data-db-target-of="sref52"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">52.</div><div>Kuo, T.C. ∙ Shaffer, A.L. ∙ Haddad, Jr., J. ...</div><div><strong>Repression of BCL-6 is required for the formation of human memory B cells in&nbsp;vitro</strong></div><div><em>J.&nbsp;Exp. Med.</em> 2007; <strong>204</strong>:819-830</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1084/jem.20062104" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-34247387876" target="_blank" aria-label="Open Scopus in new tab">Scopus (61)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17403935/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1084%2Fjem.20062104&amp;pmid=17403935" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref53-2" href="#" role="doc-biblioref" data-xml-rid="bib53" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref53-2" aria-controls="sref53-2">Laidlaw et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref53-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">53.</div><div>Laidlaw, B.J. ∙ Schmidt, T.H. ∙ Green, J.A. ...</div><div><strong>The Eph-related tyrosine kinase ligand Ephrin-B1 marks germinal center and memory precursor B cells</strong></div><div><em>J.&nbsp;Exp. Med.</em> 2017; <strong>214</strong>:639-649</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1084/jem.20161461" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-85016232208" target="_blank" aria-label="Open Scopus in new tab">Scopus (36)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28143955/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1084%2Fjem.20161461&amp;pmid=28143955" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The reason for this is explained by the critical role of BACH2 in driving B cells toward the MB lineage (<span class="dropBlock reference-citations"><a id="body-ref-sref96-4" href="#" role="doc-biblioref" data-xml-rid="bib96" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref96-4" aria-controls="sref96-4">Shinnakasu et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref96-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">94.</div><div>Shinnakasu, R. ∙ Inoue, T. ∙ Kometani, K. ...</div><div><strong>Regulated selection of germinal-center cells into the memory B cell compartment</strong></div><div><em>Nat. Immunol.</em> 2016; <strong>17</strong>:861-869</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ni.3460" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_94_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-84966559006" target="_blank" aria-label="Open Scopus in new tab">Scopus (208)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27158841/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fni.3460&amp;pmid=27158841" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), and the fact that PC master TF BLIMP1 is repressed both by BCL6 and BACH2 (<span class="dropBlock reference-citations"><a id="body-ref-sref74-2" href="#" role="doc-biblioref" data-xml-rid="bib74" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref74-2" aria-controls="sref74-2">Ochiai et&nbsp;al., 2006</a><div class="dropBlock__holder" data-db-target-of="sref74-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">75.</div><div>Ochiai, K. ∙ Katoh, Y. ∙ Ikura, T. ...</div><div><strong>Plasmacytic transcription factor Blimp-1 is repressed by Bach2 in B cells</strong></div><div><em>J.&nbsp;Biol. Chem.</em> 2006; <strong>281</strong>:38226-38234</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1074/jbc.M607592200" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_75_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-33846013955" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17046816/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1074%2Fjbc.M607592200&amp;pmid=17046816" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref107" href="#" role="doc-biblioref" data-xml-rid="bib107" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref107" aria-controls="sref107">Tunyaplin et&nbsp;al., 2004</a><div class="dropBlock__holder" data-db-target-of="sref107"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">104.</div><div>Tunyaplin, C. ∙ Shaffer, A.L. ∙ Angelin-Duclos, C.D. ...</div><div><strong>Direct repression of prdm1 by Bcl-6 inhibits plasmacytic differentiation</strong></div><div><em>J.&nbsp;Immunol.</em> 2004; <strong>173</strong>:1158-1165</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.4049/jimmunol.173.2.1158" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15240705/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.4049%2Fjimmunol.173.2.1158&amp;pmid=15240705" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Thus, in the TBL1XR1 LOF setting, BLIMP1 and the PC fate remain suppressed through the actions of BACH2-SMRT complexes. Notably, <i>Il9r</i> is a direct target gene of BLIMP1, and BLIMP1 LOF results in upregulation of <i>Il9r</i> transcripts (<span class="dropBlock reference-citations"><a id="body-ref-sref66" href="#" role="doc-biblioref" data-xml-rid="bib66" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref66" aria-controls="sref66">Minnich et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref66"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">66.</div><div>Minnich, M. ∙ Tagoh, H. ∙ Bönelt, P. ...</div><div><strong>Multifunctional role of the transcription factor Blimp-1 in coordinating plasma cell differentiation</strong></div><div><em>Nat. Immunol.</em> 2016; <strong>17</strong>:331-343</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ni.3349" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-84954476936" target="_blank" aria-label="Open Scopus in new tab">Scopus (121)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26779602/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fni.3349&amp;pmid=26779602" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Furthermore, transcriptional profiling of T&nbsp;cells from <i>Bach2-</i>KO mice revealed reduced levels of <i>Il9r</i> and increased levels of <i>Prdm1</i> (<span class="dropBlock reference-citations"><a id="body-ref-sref30" href="#" role="doc-biblioref" data-xml-rid="bib30" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref30" aria-controls="sref30">Geng et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref30"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">30.</div><div>Geng, J. ∙ Wei, H. ∙ Shi, B. ...</div><div><strong>Bach2 Negatively Regulates T Follicular Helper Cell Differentiation and Is Critical for CD4<sup>+</sup> T Cell Memory</strong></div><div><em>J.&nbsp;Immunol.</em> 2019; <strong>202</strong>:2991-2998</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.4049/jimmunol.1801626" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-85065678132" target="_blank" aria-label="Open Scopus in new tab">Scopus (5)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30971440/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.4049%2Fjimmunol.1801626&amp;pmid=30971440" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Accordingly, we found that <i>Il9r</i> plays an important role in mediating TBL1XR1 LOF-induced acquisition of preMB phenotypes. Beyond IL-9R, our findings provide a mechanistic explanation for how a switch to BACH2-driven programs might normally take place during the GC. In this view, somatic mutations of <i>TBL1XR1</i> simply exaggerate this effect, locking GC cell fate into the preMB pathway and expanding the population of pre-malignant MB cells, setting the stage for post-GC extranodal DLBCL lymphomagenesis.</div><div id="p0200" role="paragraph">Mutations affecting <i>TBL1XR1</i> associate with activating mutations in the BCR and TLR pathways (<span class="dropBlock reference-citations"><a id="body-ref-sref16-9" href="#" role="doc-biblioref" data-xml-rid="bib16" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref16-9" aria-controls="sref16-9">Chapuy et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref16-9"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">16.</div><div>Chapuy, B. ∙ Stewart, C. ∙ Dunford, A.J. ...</div><div><strong>Molecular subtypes of diffuse large B cell lymphoma are associated with distinct pathogenic mechanisms and outcomes</strong></div><div><em>Nat. Med.</em> 2018; <strong>24</strong>:679-690</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41591-018-0016-8" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-85046087353" target="_blank" aria-label="Open Scopus in new tab">Scopus (257)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29713087/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41591-018-0016-8&amp;pmid=29713087" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref92-6" href="#" role="doc-biblioref" data-xml-rid="bib92" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref92-6" aria-controls="sref92-6">Schmitz et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref92-6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">90.</div><div>Schmitz, R. ∙ Wright, G.W. ∙ Huang, D.W. ...</div><div><strong>Genetics and Pathogenesis of Diffuse Large B-Cell Lymphoma</strong></div><div><em>N.&nbsp;Engl. J. Med.</em> 2018; <strong>378</strong>:1396-1407</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1056/NEJMoa1801445" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_90_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-85045505659" target="_blank" aria-label="Open Scopus in new tab">Scopus (314)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29641966/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1056%2FNEJMoa1801445&amp;pmid=29641966" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), to define a group of aggressive extranodal post-GC lymphomas. Our finding that <i>Tbl1xr1</i> mutations disrupt canonical BCL6 transcriptional programs to drive GCB toward a post-GC/preMB stage is consistent with a MB COO for MCD/C5 DLBCLs. In agreement with this, lymphomas in KO/KO animals recapitulated canonical histological features of the human tumors, such as immunoblastic morphology, diffuse infiltration of extranodal tissues, and sparing of lymphoid follicles (<span class="dropBlock reference-citations"><a id="body-ref-sref45" href="#" role="doc-biblioref" data-xml-rid="bib45" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref45" aria-controls="sref45">Kashyap et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref45"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">45.</div><div>Kashyap, R. ∙ Rai Mittal, B. ∙ Manohar, K. ...</div><div><strong>Extranodal manifestations of lymphoma on [<sup>18</sup>F]FDG-PET/CT: a pictorial essay</strong></div><div><em>Cancer Imaging.</em> 2011; <strong>11</strong>:166-174</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1102/1470-7330.2011.0023" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_45_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-82655172686" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22123338/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1102%2F1470-7330.2011.0023&amp;pmid=22123338" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The extrafollicular pattern may be linked to expression of MB-associated migratory chemokines induced by TBL1XR1<sup>MUT</sup>, as part of the MB cell program. Indeed, the transcriptional profile of D370Y/WT GCB showed enrichment for post-GC-derived lymphoma transcriptomes and associated features, such as NF-κB/CD40 activation. An additional canonical MCD/C5 trait observed in our models was restriction in CSR (<span class="dropBlock reference-citations"><a id="body-ref-sref55" href="#" role="doc-biblioref" data-xml-rid="bib55" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref55" aria-controls="sref55">Lenz et&nbsp;al., 2007</a><div class="dropBlock__holder" data-db-target-of="sref55"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">55.</div><div>Lenz, G. ∙ Nagel, I. ∙ Siebert, R. ...</div><div><strong>Aberrant immunoglobulin class switch recombination and switch translocations in activated B cell-like diffuse large B cell lymphoma</strong></div><div><em>J.&nbsp;Exp. Med.</em> 2007; <strong>204</strong>:633-643</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1084/jem.20062041" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-33947403981" target="_blank" aria-label="Open Scopus in new tab">Scopus (148)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17353367/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1084%2Fjem.20062041&amp;pmid=17353367" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), with <i>Tbl1xr1</i> mutant and KO/KO cells remaining largely as IgM<sup>+</sup>. The selective preference for IgM expression could reflect possible benefits for the growth of post-GC lymphomas. We hypothesize that the increase in IgM<sup>+</sup> GC-derived populations is due to the strong bias introduced by <i>TBL1XR1</i> alterations toward the generation/expansion of immature MB with a high capacity to reenter the GC reaction. Most strikingly, both our <i>Tbl1xr1</i>-deficient murine lymphomas and primary human ABC-DLBCLs manifest an increased fraction of malignant cells with MB phenotype and a heavy burden of AID-induced on-target and off-target SHM. Importantly, this MB phenotype is observed in ABC-DLBCLs beyond those with <i>TBL1XR1</i> mutations, suggesting that there are additional mutational trajectories that favor this novel MB-cell associated malignant transformation pathway.</div><div id="p0205" role="paragraph"><i>Tbl1xr1</i> mutations not only skew GC output toward MB, but also suppressed formation of Ab-secreting cells upon recall. Along these lines, it is notable that extinction of the BACH2 program is normally required for PC differentiation of MB (<span class="dropBlock reference-citations"><a id="body-ref-sref49" href="#" role="doc-biblioref" data-xml-rid="bib49" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref49" aria-controls="sref49">Kometani et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref49"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">49.</div><div>Kometani, K. ∙ Nakagawa, R. ∙ Shinnakasu, R. ...</div><div><strong>Repression of the transcription factor Bach2 contributes to predisposition of IgG1 memory B cells toward plasma cell differentiation</strong></div><div><em>Immunity.</em> 2013; <strong>39</strong>:136-147</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=10.1016%2Fj.immuni.2013.06.011&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=10.1016%2Fj.immuni.2013.06.011&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-84880752273" target="_blank" aria-label="Open Scopus in new tab">Scopus (113)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23850379/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.immuni.2013.06.011&amp;pmid=23850379" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Persistent BACH2-driven transcriptional programming in TBL1XR1<sup>MUT</sup> MB could then explain the impairment in forming PC. Most strikingly, TBL1XR1<sup>MUT</sup> MB cells showed an increased tendency to reenter new GC reactions. Cyclic reentry of BCL2-overexpressing MB into the GC reaction has been proposed to originate FLs (<span class="dropBlock reference-citations"><a id="body-ref-sref103-3" href="#" role="doc-biblioref" data-xml-rid="bib103" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref103-3" aria-controls="sref103-3">Sungalee et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref103-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">101.</div><div>Sungalee, S. ∙ Mamessier, E. ∙ Morgado, E. ...</div><div><strong>Germinal center reentries of BCL2-overexpressing B cells drive follicular lymphoma progression</strong></div><div><em>J.&nbsp;Clin. Invest.</em> 2014; <strong>124</strong>:5337-5351</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1172/JCI72415" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_101_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-84915818499" target="_blank" aria-label="Open Scopus in new tab">Scopus (49)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25384217/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1172%2FJCI72415&amp;pmid=25384217" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Presumably, these BCL2-overexpressing cells self-sustain and persist for long periods of time. However, we find that TBL1XR1<sup>MUT</sup> expressly skew GC cell fate to produce the class of MB cells that are most prone to reenter the GC reaction, thus serving as clonal precursor cells to aggressive C5/MCD DLBCLs. The fact that <i>TBL1XR1</i> mutations in these lymphomas appear as early clonal events (<span class="dropBlock reference-citations"><a id="body-ref-sref16-10" href="#" role="doc-biblioref" data-xml-rid="bib16" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref16-10" aria-controls="sref16-10">Chapuy et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref16-10"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">16.</div><div>Chapuy, B. ∙ Stewart, C. ∙ Dunford, A.J. ...</div><div><strong>Molecular subtypes of diffuse large B cell lymphoma are associated with distinct pathogenic mechanisms and outcomes</strong></div><div><em>Nat. Med.</em> 2018; <strong>24</strong>:679-690</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41591-018-0016-8" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-85046087353" target="_blank" aria-label="Open Scopus in new tab">Scopus (257)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29713087/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41591-018-0016-8&amp;pmid=29713087" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) further supports a model where TBL1XR1<sup>MUT</sup> MB engage in successive rounds of GC reactions, eventually acquiring further hits in other genes (as shown here for <i>Pim1</i>) that further accelerate malignant transformation. This would likely explain the canonical complex phenotype of these tumors where they manifest an aberrant and heterogeneous composite of features reflecting aspects of MB, PB, and GCB. These results point to the need for surveillance studies in humans to identify precursor clonal cells at risk for transformation to these aggressive and incurable forms of DLBCL, much in the way current studies are attempting to predict which patients with clonal hematopoiesis of indeterminate prognosis are at risk for development of acute leukemia. The fact that this process is dependent on the SMRT/HDAC3 complex points to the potential for newly reported selective HDAC3 inhibitors (<span class="dropBlock reference-citations"><a id="body-ref-sref68" href="#" role="doc-biblioref" data-xml-rid="bib68" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref68" aria-controls="sref68">Mondello et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref68"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">68.</div><div>Mondello, P. ∙ Tadros, S. ∙ Teater, M. ...</div><div><strong>Selective Inhibition of HDAC3 Targets Synthetic Vulnerabilities and Activates Immune Surveillance in Lymphoma</strong></div><div><em>Cancer Discov.</em> 2020; <strong>10</strong>:440-459</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1158/2159-8290.CD-19-0116" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-85081148256" target="_blank" aria-label="Open Scopus in new tab">Scopus (3)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31915197/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1158%2F2159-8290.CD-19-0116&amp;pmid=31915197" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) as therapeutic agents for these clonal disorders. Perhaps combining such compounds with blockade of IL-9R signaling could further potentiate their activity. The recent discovery that auto-immune disorders like Sjogren’s disease are driven by MB cells harboring canonical MCD mutations (<span class="dropBlock reference-citations"><a id="body-ref-sref97" href="#" role="doc-biblioref" data-xml-rid="bib97" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref97" aria-controls="sref97">Singh et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref97"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">95.</div><div>Singh, M. ∙ Jackson, K.J.L. ∙ Wang, J.J. ...</div><div><strong>Lymphoma Driver Mutations in the Pathogenic Evolution of an Iconic Human Autoantibody</strong></div><div><em>Cell.</em> 2020; <strong>180</strong>:878-894</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_95_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=10.1016%2Fj.cell.2020.01.029&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_95_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=10.1016%2Fj.cell.2020.01.029&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_95_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-85080139890" target="_blank" aria-label="Open Scopus in new tab">Scopus (7)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32059783/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2020.01.029&amp;pmid=32059783" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref109" href="#" role="doc-biblioref" data-xml-rid="bib109" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref109" aria-controls="sref109">Wang et&nbsp;al., 2017a</a><div class="dropBlock__holder" data-db-target-of="sref109"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">106.</div><div>Wang, J.Q. ∙ Jeelall, Y.S. ∙ Humburg, P. ...</div><div><strong>Synergistic cooperation and crosstalk between <i>MYD88</i><sup><i>L265P</i></sup> and mutations that dysregulate CD79B and surface IgM</strong></div><div><em>J.&nbsp;Exp. Med.</em> 2017; <strong>214</strong>:2759-2776</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1084/jem.20161454" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_106_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-85028857023" target="_blank" aria-label="Open Scopus in new tab">Scopus (13)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28701369/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1084%2Fjem.20161454&amp;pmid=28701369" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) point to the further potential medical impact of such therapeutic interventions.</div></section><section id="sec-4"><h2>STAR★Methods</h2><section id="sec-4-1"><h3>Key Resources Table</h3><div id="p0215" role="paragraph"><div class="figure-wrap"><figure id="undtbl1" class="table" data-figureviewer-ignore="true"><div class="table-wrap"><table data-figureviewer-ignore="true" tabindex="0"><thead><tr class="rowsep"><th>REAGENT or RESOURCE</th><th>SOURCE</th><th>IDENTIFIER</th></tr></thead><tbody><tr class="rowsep"><td colspan="3"><b>Antibodies</b></td></tr><tr><td>Rat APC anti-CD38</td><td>eBioscience ThermoFisher Scientific</td><td>Cat# 17-0381;, RRID:<a id="interref135" href="https://antibodyregistry.org/AB_469381" target="_blank">AB_469381</a></td></tr><tr><td>Rat PE anti-CXCR4</td><td>eBioscience ThermoFisher Scientific</td><td>Cat# 12-9991; RRID:<a id="interref140" href="https://antibodyregistry.org/AB_891393" target="_blank">AB_891393</a></td></tr><tr><td>Mouse PerCP-Cy5.5 anti-CD45.1</td><td>eBioscience ThermoFisher Scientific</td><td>Cat# 45-0453; RRID:<a id="interref145" href="https://antibodyregistry.org/AB_1107003" target="_blank">AB_1107003</a></td></tr><tr><td>Armenian Hamster FITC anti-PD-1</td><td>eBioscience ThermoFisher Scientific</td><td>Cat# 11-9985; RRID:<a id="interref150" href="https://antibodyregistry.org/AB_465473" target="_blank">AB_465473</a></td></tr><tr><td>Rat anti-mouse CD16/CD32 (Fc block)</td><td>BD Biosciences</td><td>Cat# 553142; RRID:<a id="interref155" href="https://antibodyregistry.org/AB_394657" target="_blank">AB_394657</a></td></tr><tr><td>Rat PE-Cy7 anti-CD44</td><td>BD Biosciences</td><td>Cat# 560569; RRID:<a id="interref160" href="https://antibodyregistry.org/AB_1727484" target="_blank">AB_1727484</a></td></tr><tr><td>Rat APC anti-B220</td><td>BD Biosciences</td><td>Cat# 553087; RRID:<a id="interref165" href="https://antibodyregistry.org/AB_394617" target="_blank">AB_394617</a></td></tr><tr><td>Rat PE-Cy7 anti B220</td><td>BD Biosciences</td><td>Cat# 552772; RRID:<a id="interref170" href="https://antibodyregistry.org/AB_394458" target="_blank">AB_394458</a></td></tr><tr><td>Rat BV786 anti-B220</td><td>BD Biosciences</td><td>Cat# 563894; RRID:<a id="interref175" href="https://antibodyregistry.org/AB_2738472" target="_blank">AB_2738472</a></td></tr><tr><td>Hamster PE-Cy7 anti-FAS</td><td>BD Biosciences</td><td>Cat# 557653; RRID:<a id="interref180" href="https://antibodyregistry.org/AB_396768" target="_blank">AB_396768</a></td></tr><tr><td>Rat BV395 anti-CD38</td><td>BD Biosciences</td><td>Cat# 740245; RRID:<a id="interref185" href="https://antibodyregistry.org/AB_2739992" target="_blank">AB_2739992</a></td></tr><tr><td>Rat PE-Cy7 anti-CD86</td><td>BD Biosciences</td><td>Cat# 560582; RRID:<a id="interref190" href="https://antibodyregistry.org/AB_1727518" target="_blank">AB_1727518</a></td></tr><tr><td>Rat BV421 anti-CD86</td><td>BD Biosciences</td><td>Cat# 564198; RRID:<a id="interref195" href="https://antibodyregistry.org/AB_2738663" target="_blank">AB_2738663</a></td></tr><tr><td>Rat biotin anti-CXCR4</td><td>BD Biosciences</td><td>Cat# 551968; RRID:<a id="interref200" href="https://antibodyregistry.org/AB_394307" target="_blank">AB_394307</a></td></tr><tr><td>Rat biotin anti-CXCR5</td><td>BD Biosciences</td><td>Cat# 551960; RRID:<a id="interref205" href="https://antibodyregistry.org/AB_394301" target="_blank">AB_394301</a></td></tr><tr><td>Rat BV421 anti-IgD</td><td>BD Biosciences</td><td>Cat# 744291; RRID:<a id="interref210" href="https://antibodyregistry.org/AB_2742121" target="_blank">AB_2742121</a></td></tr><tr><td>Rat BV510 anti-IgD</td><td>BD Biosciences</td><td>Cat# 563110; RRID:<a id="interref215" href="https://antibodyregistry.org/AB_2737003" target="_blank">AB_2737003</a></td></tr><tr><td>Rat PE anti-IgG1</td><td>BD Biosciences</td><td>Cat# 550083; RRID:<a id="interref220" href="https://antibodyregistry.org/AB_393553" target="_blank">AB_393553</a></td></tr><tr><td>Rat BV421 anti-IgG1</td><td>BD Biosciences</td><td>Cat# 562580; RRID:<a id="interref225" href="https://antibodyregistry.org/AB_2737664" target="_blank">AB_2737664</a></td></tr><tr><td>Rat BUV737 anti-CD138</td><td>BD Biosciences</td><td>Cat# 564430; RRID:<a id="interref230" href="https://antibodyregistry.org/AB_2738805" target="_blank">AB_2738805</a></td></tr><tr><td>Rat FITC anti-IgM</td><td>BD Biosciences</td><td>Cat# 553437; RRID:<a id="interref235" href="https://antibodyregistry.org/AB_394857" target="_blank">AB_394857</a></td></tr><tr><td>Rat BV711 anti-IgM</td><td>BD Biosciences</td><td>Cat# 743327; RRID:<a id="interref240" href="https://antibodyregistry.org/AB_2741428" target="_blank">AB_2741428</a></td></tr><tr><td>Rat BV421 anti-CCR6</td><td>BD Biosciences</td><td>Cat# 564736; RRID:<a id="interref245" href="https://antibodyregistry.org/AB_2738926" target="_blank">AB_2738926</a></td></tr><tr><td>Rat APC anti-CD117</td><td>BD Biosciences</td><td>Cat# 561074; RRID:<a id="interref250" href="https://antibodyregistry.org/AB_10563203" target="_blank">AB_10563203</a></td></tr><tr><td>Rat FITC anti-CD19</td><td>BD Biosciences</td><td>Cat# 553785; RRID:<a id="interref255" href="https://antibodyregistry.org/AB_395049" target="_blank">AB_395049</a></td></tr><tr><td>Rat PE-Cy7 anti-CD25</td><td>BD Biosciences</td><td>Cat# 561780; RRID:<a id="interref260" href="https://antibodyregistry.org/AB_10893596" target="_blank">AB_10893596</a></td></tr><tr><td>Rat FITC anti-GL7</td><td>BD Biosciences</td><td>Cat# 553666; RRID:<a id="interref265" href="https://antibodyregistry.org/AB_394981" target="_blank">AB_394981</a></td></tr><tr><td>Rat AF647 anti-GL7</td><td>BD Biosciences</td><td>Cat# 561529; RRID:<a id="interref270" href="https://antibodyregistry.org/AB_10716056" target="_blank">AB_10716056</a></td></tr><tr><td>Rat anti-CD138</td><td>BD Biosciences</td><td>Cat# 553712; RRID:<a id="interref275" href="https://antibodyregistry.org/AB_394998" target="_blank">AB_394998</a></td></tr><tr><td>Biotin-conjugated anti-B220</td><td>BD Biosciences</td><td>Cat# 550286; RRID:<a id="interref280" href="https://antibodyregistry.org/AB_393581" target="_blank">AB_393581</a></td></tr><tr><td>Human TruStain FcX (Fc block)</td><td>BioLegend</td><td>Cat# 422301; RRID:<a id="interref285" href="https://antibodyregistry.org/AB_2818986" target="_blank">AB_2818986</a></td></tr><tr><td>Rat BV510 anti-CD62L</td><td>BioLegend</td><td>Cat# 104441; RRID:<a id="interref290" href="https://antibodyregistry.org/AB_2561537" target="_blank">AB_2561537</a></td></tr><tr><td>Rat PE anti-PCNA</td><td>BioLegend</td><td>Cat# 307908; RRID:<a id="interref295" href="https://antibodyregistry.org/AB_314698" target="_blank">AB_314698</a></td></tr><tr><td>Rat APC-Cy7 anti-CD4</td><td>BioLegend</td><td>Cat# 100414; RRID:<a id="interref300" href="https://antibodyregistry.org/AB_312699" target="_blank">AB_312699</a></td></tr><tr><td>Mouse PerCP-Cy5.5 anti-CD45.2</td><td>BioLegend</td><td>Cat# 109828; RRID:<a id="interref305" href="https://antibodyregistry.org/AB_893350" target="_blank">AB_893350</a></td></tr><tr><td>Armenian Hamster BV421 anti-CD80</td><td>BioLegend</td><td>Cat# 104725; RRID:<a id="interref310" href="https://antibodyregistry.org/AB_10900989" target="_blank">AB_10900989</a></td></tr><tr><td>Rat APC anti-CD3</td><td>BioLegend</td><td>Cat# 100235; RRID:<a id="interref315" href="https://antibodyregistry.org/AB_2561455" target="_blank">AB_2561455</a></td></tr><tr><td>Mouse PE anti-BCL6</td><td>BioLegend</td><td>Cat# 648304; RRID:<a id="interref320" href="https://antibodyregistry.org/AB_2561375" target="_blank">AB_2561375</a></td></tr><tr><td>Rat AF647 anti-BLIMP1</td><td>BioLegend</td><td>Cat# 150004; RRID:<a id="interref325" href="https://antibodyregistry.org/AB_2565618" target="_blank">AB_2565618</a></td></tr><tr><td>Rat BV421 anti-B220</td><td>BioLegend</td><td>Cat# 103240; RRID:<a id="interref330" href="https://antibodyregistry.org/AB_11203896" target="_blank">AB_11203896</a></td></tr><tr><td>Rat APC-Cy7 anti-B220</td><td>BioLegend</td><td>Cat# 103224; RRID:<a id="interref335" href="https://antibodyregistry.org/AB_313007" target="_blank">AB_313007</a></td></tr><tr><td>Rat PE anti-B220</td><td>BioLegend</td><td>Cat# 103208; RRID:<a id="interref340" href="https://antibodyregistry.org/AB_312993" target="_blank">AB_312993</a></td></tr><tr><td>Rat APC-Cy7 anti-CD38</td><td>BioLegend</td><td>Cat# 102728; RRID:<a id="interref345" href="https://antibodyregistry.org/AB_2616968" target="_blank">AB_2616968</a></td></tr><tr><td>Rat PerCP-Cy5.5 anti-GL7</td><td>BioLegend</td><td>Cat# 144610; RRID:<a id="interref350" href="https://antibodyregistry.org/AB_2562979" target="_blank">AB_2562979</a></td></tr><tr><td>Mouse PerCP-Cy5.5 anti-FAS</td><td>BioLegend</td><td>Cat# 152610; RRID:<a id="interref355" href="https://antibodyregistry.org/AB_2632905" target="_blank">AB_2632905</a></td></tr><tr><td>Rat PE-Cy7 anti-CD138</td><td>BioLegend</td><td>Cat# 142514; RRID:<a id="interref360" href="https://antibodyregistry.org/AB_2562198" target="_blank">AB_2562198</a></td></tr><tr><td>Rat BV421 anti-CD138</td><td>BioLegend</td><td>Cat# 142508; RRID:<a id="interref365" href="https://antibodyregistry.org/AB_11203544" target="_blank">AB_11203544</a></td></tr><tr><td>Rat APC anti-IL9</td><td>BioLegend</td><td>Cat# RM9A4; RRID:<a id="interref370" href="https://antibodyregistry.org/AB_11218680" target="_blank">AB_11218680</a></td></tr><tr><td>Rat anti-mS1P1</td><td>R&amp;D Systems</td><td>Cat# MAB7089; RRID:<a id="interref375" href="https://antibodyregistry.org/AB_10994183" target="_blank">AB_10994183</a></td></tr><tr><td>Goat biotin anti-EFNB1</td><td>R&amp;D Systems</td><td>Cat# BAF473; RRID:<a id="interref380" href="https://antibodyregistry.org/AB_2293418" target="_blank">AB_2293418</a></td></tr><tr><td>Goat anti-Chicken IgY (H+L) Secondary Antibody, HRP</td><td>ThermoFisher Scientific</td><td>Cat# A16054; RRID:<a id="interref385" href="https://antibodyregistry.org/AB_2534727" target="_blank">AB_2534727</a></td></tr><tr><td>Goat anti-Mouse IgG Alexa Fluor 647</td><td>ThermoFisher Scientific</td><td>Cat# A21235; RRID:<a id="interref390" href="https://antibodyregistry.org/AB_2535804" target="_blank">AB_2535804</a></td></tr><tr><td>Mouse anti-AID</td><td>ThermoFisher Scientific</td><td>Cat# 1AID-2E11; RRID:<a id="interref395" href="https://antibodyregistry.org/AB_2633326" target="_blank">AB_2633326</a></td></tr><tr><td>Mouse anti-CD38</td><td>ThermoFisher Scientific</td><td>Cat# MA5-14413; RRID:<a id="interref400" href="https://antibodyregistry.org/AB_10986743" target="_blank">AB_10986743</a></td></tr><tr><td>Rabbit anti-NCOR2</td><td>ThermoFisher Scientific</td><td>Cat# PA1-843; RRID:<a id="interref405" href="https://antibodyregistry.org/AB_2149135" target="_blank">AB_2149135</a></td></tr><tr><td>Armenian hamster anti-mouse CD40L (clone MR-1)</td><td>BioXCell</td><td>Cat# BE0017; RRID:<a id="interref410" href="https://antibodyregistry.org/AB_1107601" target="_blank">AB_1107601</a></td></tr><tr><td>Polyclonal Armenian hamster IgG</td><td>BioXCell</td><td>Cat# BE0091; RRID:<a id="interref415" href="https://antibodyregistry.org/AB_1107773" target="_blank">AB_1107773</a></td></tr><tr><td>Mouse anti-IL9 (clone 9C1)</td><td>BioXCell</td><td>Cat# BE0181; RRID:<a id="interref420" href="https://antibodyregistry.org/AB_10950648" target="_blank">AB_10950648</a></td></tr><tr><td>Mouse IgG2a isotype control</td><td>BioXCell</td><td>Cat# BE0085; RRID:<a id="interref425" href="https://antibodyregistry.org/AB_1107771" target="_blank">AB_1107771</a></td></tr><tr><td>Rabbit Anti-Mouse IgG H&amp;L</td><td>Abcam</td><td>Cat# ab46540; RRID:<a id="interref430" href="https://antibodyregistry.org/AB_2614925" target="_blank">AB_2614925</a></td></tr><tr><td>Rabbit anti-CD3</td><td>Abcam</td><td>Cat# ab16669; RRID:<a id="interref435" href="https://antibodyregistry.org/AB_443425" target="_blank">AB_443425</a></td></tr><tr><td>Rabbit anti-KI67</td><td>Cell Signaling Technology</td><td>Cat# 12202; RRID:<a id="interref440" href="https://antibodyregistry.org/AB_2620142" target="_blank">AB_2620142</a></td></tr><tr><td>Rabbit anti-BCL6</td><td>Santa Cruz Biotechnology</td><td>Cat# sc-858; RRID:<a id="interref445" href="https://antibodyregistry.org/AB_2063450" target="_blank">AB_2063450</a></td></tr><tr><td>Mouse anti-BCL6</td><td>Santa Cruz Biotechnology</td><td>Cat# sc-7388; RRID:<a id="interref450" href="https://antibodyregistry.org/AB_2063455" target="_blank">AB_2063455</a></td></tr><tr><td>Rabbit anti-HDAC3</td><td>Santa Cruz Biotechnology</td><td>Cat# sc-11417; RRID:<a id="interref455" href="https://antibodyregistry.org/AB_2118706" target="_blank">AB_2118706</a></td></tr><tr><td>Rabbit anti-HDAC3</td><td>Santa Cruz Biotechnology</td><td>Cat# sc-376957; RRID:<a id="interref460" href="https://antibodyregistry.org/AB_2715509" target="_blank">AB_2715509</a></td></tr><tr><td>Mouse anti-β-Actin</td><td>Santa Cruz Biotechnology</td><td>Cat# sc-47778; RRID:<a id="interref465" href="https://antibodyregistry.org/AB_2714189" target="_blank">AB_2714189</a></td></tr><tr><td>Rabbit anti-NCOR2</td><td>Sigma-Aldrich</td><td>Cat# 06-891; RRID:<a id="interref470" href="https://antibodyregistry.org/AB_310286" target="_blank">AB_310286</a></td></tr><tr><td>Rabbit anti-BACH2</td><td>Aviva Systems Biology</td><td>Cat# ARP39513_P050; RRID:<a id="interref475" href="https://antibodyregistry.org/AB_2045059" target="_blank">AB_2045059</a></td></tr><tr><td>Mouse anti-TBL1XR1</td><td>Abnova</td><td>Cat# H00079718-M01; RRID:<a id="interref480" href="https://antibodyregistry.org/AB_490081" target="_blank">AB_490081</a></td></tr><tr><td>Chicken anti-BirA</td><td>BioFront Technologies</td><td>Cat# BID-CP-100</td></tr><tr><td>Biotinylated goat anti-rat secondary antibody</td><td>Jackson ImmunoResearch Lab.</td><td>Cat# 112-065-167; RRID:<a id="interref485" href="https://antibodyregistry.org/AB_2338179" target="_blank">AB_2338179</a></td></tr><tr><td>Peroxidase IgG Fraction Monoclonal Mouse Anti-Rabbit IgG, light chain specific</td><td>Jackson ImmunoResearch Lab.</td><td>Cat# 211-032-171; RRID:<a id="interref490" href="https://antibodyregistry.org/AB_2339149" target="_blank">AB_2339149</a></td></tr><tr><td>Peroxidase AffiniPure Goat Anti-Mouse IgG, light chain specific</td><td>Jackson ImmunoResearch Lab.</td><td>Cat# 115-035-174; RRID:<a id="interref495" href="https://antibodyregistry.org/AB_2338512" target="_blank">AB_2338512</a></td></tr><tr><td>Anti-Human CD10-156Gd</td><td>Fluidigm</td><td>Clone HI10a; RRID:<a id="interref500" href="https://antibodyregistry.org/AB_2802107" target="_blank">AB_2802107</a></td></tr><tr><td>Anti-Human CD27-167Er</td><td>Fluidigm</td><td>Clone L128; RRID:<a id="interref505" href="https://antibodyregistry.org/AB_2811093" target="_blank">AB_2811093</a></td></tr><tr><td>Anti-Human CD38-172Yb</td><td>Fluidigm</td><td>Clone HIT2</td></tr><tr class="rowsep"><td>Anti-Human CD44-171Yb</td><td>Fluidigm</td><td>Clone IM7</td></tr><tr class="rowsep"><td colspan="3"><b>Bacterial and Virus Strains</b></td></tr><tr><td>TOP10 Chemically Competent <i>E.&nbsp;coli</i></td><td>ThermoFisher Scientific</td><td>Cat# C404010</td></tr><tr><td>Stbl3 Chemically Competent <i>E.&nbsp;coli</i></td><td>ThermoFisher Scientific</td><td>Cat# C737303</td></tr><tr class="rowsep"><td>DH5α Competent Cells</td><td>ThermoFisher Scientific</td><td>Cat# 18265017</td></tr><tr class="rowsep"><td colspan="3"><b>Biological Samples</b></td></tr><tr><td>Sheep red blood cells</td><td>Cocalico Biologicals</td><td>Cat# 20-1334A</td></tr><tr class="rowsep"><td>Tissue microarrays of 341 primary diagnosed DLBCL</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref28-1" href="#" role="doc-biblioref" data-xml-rid="bib28" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref28-1" aria-controls="sref28-1">Ennishi et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref28-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">28.</div><div>Ennishi, D. ∙ Mottok, A. ∙ Ben-Neriah, S. ...</div><div><strong>Genetic profiling of <i>MYC</i> and <i>BCL2</i> in diffuse large B-cell lymphoma determines cell-of-origin-specific clinical impact</strong></div><div><em>Blood.</em> 2017; <strong>129</strong>:2760-2770</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1182/blood-2016-11-747022" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-85019688331" target="_blank" aria-label="Open Scopus in new tab">Scopus (53)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28351934/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1182%2Fblood-2016-11-747022&amp;pmid=28351934" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref94-1" href="#" role="doc-biblioref" data-xml-rid="bib94" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref94-1" aria-controls="sref94-1">Scott et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref94-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">92.</div><div>Scott, D.W. ∙ Mottok, A. ∙ Ennishi, D. ...</div><div><strong>Prognostic Significance of Diffuse Large B-Cell Lymphoma Cell of Origin Determined by Digital Gene Expression in Formalin-Fixed Paraffin-Embedded Tissue Biopsies</strong></div><div><em>J.&nbsp;Clin. Oncol.</em> 2015; <strong>33</strong>:2848-2856</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1200/JCO.2014.60.2383" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_92_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-84964994916" target="_blank" aria-label="Open Scopus in new tab">Scopus (165)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26240231/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1200%2FJCO.2014.60.2383&amp;pmid=26240231" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>N/A</td></tr><tr class="rowsep"><td colspan="3"><b>Chemicals, Peptides, and Recombinant Proteins</b></td></tr><tr><td>PE-Cy7 streptavidin</td><td>eBioscience ThermoFisher Scientific</td><td>Cat# 25-4317</td></tr><tr><td>APC streptavidin</td><td>BioLegend</td><td>Cat# 405207</td></tr><tr><td>APC-Cy7 streptavidin</td><td>BioLegend</td><td>Cat# 405208</td></tr><tr><td>NIP-haptenated FITC</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref2-1" href="#" role="doc-biblioref" data-xml-rid="bib2" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref2-1" aria-controls="sref2-1">Anderson et&nbsp;al., 2007</a><div class="dropBlock__holder" data-db-target-of="sref2-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">2.</div><div>Anderson, S.M. ∙ Tomayko, M.M. ∙ Ahuja, A. ...</div><div><strong>New markers for murine memory B cells that define mutated and unmutated subsets</strong></div><div><em>J.&nbsp;Exp. Med.</em> 2007; <strong>204</strong>:2103-2114</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1084/jem.20062571" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_2_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-34548383012" target="_blank" aria-label="Open Scopus in new tab">Scopus (164)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17698588/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1084%2Fjem.20062571&amp;pmid=17698588" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>N/A</td></tr><tr><td>PE NP</td><td>Biosearch Technologies</td><td>Cat# N-5070</td></tr><tr><td>NP-OVAL</td><td>Biosearch Technologies</td><td>Cat# N-5051</td></tr><tr><td>NP-CGG</td><td>Biosearch Technologies</td><td>Cat# N-5055C</td></tr><tr><td>NP-KLH</td><td>Biosearch Technologies</td><td>Cat# N-5060</td></tr><tr><td>Alum adjuvant</td><td>ThermoFisher Scientific</td><td>Cat# 77161</td></tr><tr><td>Protein A Dynabeads</td><td>ThermoFisher Scientific</td><td>Cat# 10001D</td></tr><tr><td>V5-Tag magnetic beads</td><td>MBL International Corporation</td><td>Cat# M16711</td></tr><tr><td>Ovalbumin</td><td>Sigma-Aldrich</td><td>Cat# A5503</td></tr><tr><td>Recombinant mouse IL-9</td><td>Peprotech</td><td>Cat# 219-19</td></tr><tr><td>Recombinant mouse IL-4</td><td>R&amp;D Systems</td><td>Cat# 404-ML</td></tr><tr><td>Recombinant mouse IL-21</td><td>R&amp;D Systems</td><td>Cat# 594-ML</td></tr><tr><td>Ghost Dye Violet 510</td><td>Tonbo Biosciences</td><td>Cat# 13-0870</td></tr><tr><td>DAPI</td><td>ThermoFisher Scientific</td><td>Cat# D1306</td></tr><tr><td>NP-BSA, ratio 1-4</td><td>Biosearch Technologies</td><td>Cat# N-5050XL</td></tr><tr><td>NP-BSA, ratio &gt; 20</td><td>Biosearch Technologies</td><td>Cat# N-5050H</td></tr><tr><td>Mitomycin C</td><td>Sigma-Aldrich</td><td>Cat# MO503</td></tr><tr><td>Alt-R® S.p. Cas9 Nuclease V3</td><td>Integrated DNA Technologies</td><td>Cat# 1081058</td></tr><tr><td>Alt-R® CRISPR-Cas9 tracrRNA</td><td>Integrated DNA Technologies</td><td>Cat# 1075927</td></tr><tr><td>Biotin</td><td>Sigma-Aldrich</td><td>Cat# B4501</td></tr><tr><td>Benzonase nuclease</td><td>Sigma-Aldrich</td><td>Cat# E1014-5KU</td></tr><tr><td>Sepharose beads</td><td>GE Healthcare Life Sciences</td><td>Cat# GE17-5113-01</td></tr><tr><td>TPCK trypsin</td><td>Promega</td><td>Cat# V5111</td></tr><tr><td>Biotinylated Peanut Agglutinin (PNA)</td><td>Vector Laboratories</td><td>Cat# B1075</td></tr><tr class="rowsep"><td>EdU (5-ethynyl-2′-deoxyuridine)</td><td>ThermoFisher Scientific</td><td>Cat# E10187</td></tr><tr class="rowsep"><td colspan="3"><b>Critical Commercial Assays</b></td></tr><tr><td>Click-iT Plus AF488 Flow Cytometry Assay Kit</td><td>ThermoFisher Scientific</td><td>Cat# C10633</td></tr><tr><td>Cell Proliferation Dye eFluor 670</td><td>eBioscience ThermoFisher Scientific</td><td>Cat# 65-0840-85</td></tr><tr><td>CellTrace CFSE Cell Proliferation Kit</td><td>ThermoFisher Scientific</td><td>Cat# C34554</td></tr><tr><td>Foxp3/Transcription Factor Staining Buffer Set</td><td>ThermoFisher Scientific</td><td>Cat# 00-5523-00</td></tr><tr><td>AnnexinV binding buffer</td><td>BD Biosciences</td><td>Cat# 556454</td></tr><tr><td>PE-conjugated AnnexinV</td><td>BD Biosciences</td><td>Cat# 556421</td></tr><tr><td>Cell Proliferation Dye eFluor 670</td><td>ThermoFisher Scientific</td><td>Cat# 65-0840-85</td></tr><tr><td>EasySep Mouse B Cell Isolation Kit</td><td>StemCell Technologies</td><td>Cat# 9854</td></tr><tr><td>CD43 (Ly-48) MicroBeads</td><td>Miltenyi Biotec</td><td>Cat# 130-049-801</td></tr><tr><td>CD45R (B220) MicroBeads</td><td>Miltenyi Biotec</td><td>Cat# 130-049-501</td></tr><tr><td>SBA Clonotyping System</td><td>Southern Biotechnology</td><td>Cat# 5300-05</td></tr><tr><td>Nuclear Complex Co-IP Kit</td><td>Active Motif</td><td>Cat# 54001</td></tr><tr><td>QuickExtract DNA Extraction Solution</td><td>Epicenter</td><td>Cat# QE09050</td></tr><tr><td>Puregene Gentra cell kit</td><td>QIAGEN</td><td>Cat# 158388</td></tr><tr><td>DirectPCR Lysis Reagent</td><td>Viagen Biotech</td><td>Cat# 101-T</td></tr><tr><td>Red blood cell lysis solution</td><td>QIAGEN</td><td>Cat# 158904</td></tr><tr><td>QuikChange II XL Site-directed mutagenesis kit</td><td>Agilent Technologies</td><td>Cat# 200521</td></tr><tr><td>Gateway LR Clonase II Enzyme mix</td><td>ThermoFisher Scientific</td><td>Cat# 11791020</td></tr><tr><td>PEG-it</td><td>System Biosciences</td><td>Cat# LV810A-1</td></tr><tr><td>TOPO TA Cloning Kit for Sequencing</td><td>ThermoFisher Scientific</td><td>Cat# K457501</td></tr><tr><td>Zero Blunt TOPO PCR Cloning Kit</td><td>ThermoFisher Scientific</td><td>Cat# K280002</td></tr><tr><td>SF Cell Line 4D-Nucleofector X Kit</td><td>Lonza</td><td>Cat# PBC2-22500</td></tr><tr><td>Sepharose beads</td><td>GE Healthcare Life Sciences</td><td>Cat# GE17-5113-01</td></tr><tr><td>Illumina TruSeq stranded-mRNA Library Prep</td><td>Illumina</td><td>Cat# 20020594</td></tr><tr><td>Quant-iT dsDNA HS Assay</td><td>ThermoFisher Scientific</td><td>Cat# Q33120</td></tr><tr><td>SYBR Safe DNA stain</td><td>ThermoFisher Scientific</td><td>Cat# S33102</td></tr><tr><td>Verso cDNA Synthesis kit</td><td>ThermoFisher Scientific</td><td>Cat# AB1453B</td></tr><tr><td>Fast SYBR Green Master Mix</td><td>ThermoFisher Scientific</td><td>Cat# 4385614</td></tr><tr><td>Vector® Blue Alkaline Phosphatase (Blue AP) Substrate Kit</td><td>Vector Laboratories</td><td>Cat# SK-5300</td></tr><tr class="rowsep"><td>DAB Peroxidase (HRP) Substrate Kit</td><td>Vector Laboratories</td><td>Cat# SK-4100</td></tr><tr class="rowsep"><td colspan="3"><b>Deposited Data</b></td></tr><tr><td>DLBCL cases</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref5-5" href="#" role="doc-biblioref" data-xml-rid="bib5" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref5-5" aria-controls="sref5-5">Arthur et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref5-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">5.</div><div>Arthur, S.E. ∙ Jiang, A. ∙ Grande, B.M. ...</div><div><strong>Genome-wide discovery of somatic regulatory variants in diffuse large B-cell lymphoma</strong></div><div><em>Nat. Commun.</em> 2018; <strong>9</strong>:4001</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41467-018-06354-3" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-85054070552" target="_blank" aria-label="Open Scopus in new tab">Scopus (32)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30275490/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41467-018-06354-3&amp;pmid=30275490" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref61-3" href="#" role="doc-biblioref" data-xml-rid="bib61" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref61-3" aria-controls="sref61-3">Ma et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref61-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">61.</div><div>Ma, M.C.J. ∙ Tadros, S. ∙ Bouska, A. ...</div><div><strong>Pathognomonic and epistatic genetic alterations in B-cell non-Hodgkin lymphoma</strong></div><div><em>bioRxiv.</em> 2019;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/674259" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2F674259" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref86-3" href="#" role="doc-biblioref" data-xml-rid="bib86" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref86-3" aria-controls="sref86-3">Reddy et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref86-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">84.</div><div>Reddy, A. ∙ Zhang, J. ∙ Davis, N.S. ...</div><div><strong>Genetic and Functional Drivers of Diffuse Large B Cell Lymphoma</strong></div><div><em>Cell.</em> 2017; <strong>171</strong>:481-494</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_84_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=10.1016%2Fj.cell.2017.09.027&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_84_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=10.1016%2Fj.cell.2017.09.027&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_84_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-85030529459" target="_blank" aria-label="Open Scopus in new tab">Scopus (251)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28985567/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2017.09.027&amp;pmid=28985567" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref92-7" href="#" role="doc-biblioref" data-xml-rid="bib92" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref92-7" aria-controls="sref92-7">Schmitz et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref92-7"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">90.</div><div>Schmitz, R. ∙ Wright, G.W. ∙ Huang, D.W. ...</div><div><strong>Genetics and Pathogenesis of Diffuse Large B-Cell Lymphoma</strong></div><div><em>N.&nbsp;Engl. J. Med.</em> 2018; <strong>378</strong>:1396-1407</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1056/NEJMoa1801445" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_90_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-85045505659" target="_blank" aria-label="Open Scopus in new tab">Scopus (314)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29641966/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1056%2FNEJMoa1801445&amp;pmid=29641966" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>N/A</td></tr><tr><td>FL cases</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref51-2" href="#" role="doc-biblioref" data-xml-rid="bib51" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref51-2" aria-controls="sref51-2">Krysiak et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref51-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">51.</div><div>Krysiak, K. ∙ Gomez, F. ∙ White, B.S. ...</div><div><strong>Recurrent somatic mutations affecting B-cell receptor signaling pathway genes in follicular lymphoma</strong></div><div><em>Blood.</em> 2017; <strong>129</strong>:473-483</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1182/blood-2016-07-729954" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_51_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-85014960882" target="_blank" aria-label="Open Scopus in new tab">Scopus (58)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28064239/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1182%2Fblood-2016-07-729954&amp;pmid=28064239" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref61-4" href="#" role="doc-biblioref" data-xml-rid="bib61" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref61-4" aria-controls="sref61-4">Ma et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref61-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">61.</div><div>Ma, M.C.J. ∙ Tadros, S. ∙ Bouska, A. ...</div><div><strong>Pathognomonic and epistatic genetic alterations in B-cell non-Hodgkin lymphoma</strong></div><div><em>bioRxiv.</em> 2019;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/674259" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2F674259" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref77-2" href="#" role="doc-biblioref" data-xml-rid="bib77" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref77-2" aria-controls="sref77-2">Ortega-Molina et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref77-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">78.</div><div>Ortega-Molina, A. ∙ Boss, I.W. ∙ Canela, A. ...</div><div><strong>The histone lysine methyltransferase KMT2D sustains a gene expression program that represses B cell lymphoma development</strong></div><div><em>Nat. Med.</em> 2015; <strong>21</strong>:1199-1208</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nm.3943" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_78_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-84943658052" target="_blank" aria-label="Open Scopus in new tab">Scopus (159)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26366710/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnm.3943&amp;pmid=26366710" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>N/A</td></tr><tr><td>Raw sequencing data</td><td>This manuscript</td><td>GEO: <a id="intref0010" href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE139059" target="_blank">GSE139059</a></td></tr><tr class="rowsep"><td>Mass spectrometry data</td><td>This manuscript</td><td>MassIVE: MSV000085310</td></tr><tr class="rowsep"><td colspan="3"><b>Experimental Models: Cell Lines</b></td></tr><tr><td>Human: OCI-Ly1 cells</td><td>Ontario Cancer Institute (OCI)</td><td>CVCL_1879</td></tr><tr><td>Human: U2932 cells</td><td>DSMZ-German Collection of Microorganisms and Cell Cultures</td><td>RRID:CVCL_1896</td></tr><tr><td>Human: HEK293T cells</td><td>American Type Culture Collection (ATCC)</td><td>RRID:CVCL_0063</td></tr><tr class="rowsep"><td>Mouse: 40LB</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref72-1" href="#" role="doc-biblioref" data-xml-rid="bib72" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref72-1" aria-controls="sref72-1">Nojima et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref72-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">73.</div><div>Nojima, T. ∙ Haniuda, K. ∙ Moutai, T. ...</div><div><strong>In-vitro derived germinal centre B cells differentially generate memory B or plasma cells in&nbsp;vivo</strong></div><div><em>Nat. Commun.</em> 2011; <strong>2</strong>:465</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ncomms1475" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-80052643363" target="_blank" aria-label="Open Scopus in new tab">Scopus (101)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21897376/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fncomms1475&amp;pmid=21897376" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>N/A</td></tr><tr class="rowsep"><td colspan="3"><b>Experimental Models: Organisms/Strains</b></td></tr><tr><td>Mouse: C57BL/6J</td><td>The Jackson Laboratory</td><td>Stock# 000664</td></tr><tr><td>Mouse: <i>Cγ1-Cre</i> (B6.129P2(Cg)-Ighg1tm1<sup>(cre)Cgn</sup>/J)</td><td>The Jackson Laboratory</td><td>Stock# 010611</td></tr><tr><td>Mouse <i>CD19-Cre</i> (B6.129P2(C)-Cd19<sup>tm1(cre)Cgn</sup>/J)</td><td>The Jackson Laboratory</td><td>Stock# 006785</td></tr><tr><td>Mouse: μMT (B6.129S2-Ighm<sup>tm1Cgn</sup>/J)</td><td>The Jackson Laboratory</td><td>Stock# 002288</td></tr><tr><td>Mouse: CD45.1 (B6.SJL-Ptprc<sup>a</sup>Pepc<sup>b</sup>/Boy)</td><td>The Jackson Laboratory</td><td>Stock# 002014</td></tr><tr><td>Mouse: Rosa26-lox-stop-lox-YFP (B6.129X1-Gt(ROSA)26Sor<sup>tm1(EYFP)Cos</sup>/J)</td><td>The Jackson Laboratory</td><td>Stock# 006148</td></tr><tr><td>Mouse: OT-II (B6.Cg-Tg(TcraTcrb)425Cbn/J)</td><td>The Jackson Laboratory</td><td>Stock# 004194</td></tr><tr><td>Mouse: R26:FLPe (129S4/SvJaeSor-Gt(ROSA)26Sor<sup>tm1(FLP1)Dym</sup>/J)</td><td>The Jackson Laboratory</td><td>Stock# 003946</td></tr><tr><td>Mouse: B1-8hi (B6.129P2-Ptrpc<sup>a</sup> Igh<sup>tm1Mnz</sup>/J)</td><td>The Jackson Laboratory</td><td>Stock# 007594</td></tr><tr><td>Mouse: <i>R26-Fucci2aR</i></td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref69-1" href="#" role="doc-biblioref" data-xml-rid="bib69" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref69-1" aria-controls="sref69-1">Mort et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref69-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">69.</div><div>Mort, R.L. ∙ Ford, M.J. ∙ Sakaue-Sawano, A. ...</div><div><strong>Fucci2a: a bicistronic cell cycle reporter that allows Cre mediated tissue specific expression in mice</strong></div><div><em>Cell Cycle.</em> 2014; <strong>13</strong>:2681-2696</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.4161/15384101.2015.945381" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-84907438909" target="_blank" aria-label="Open Scopus in new tab">Scopus (39)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25486356/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.4161%2F15384101.2015.945381&amp;pmid=25486356" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td>N/A</td></tr><tr><td>Mouse: <i>IμBCL6</i></td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref13-2" href="#" role="doc-biblioref" data-xml-rid="bib13" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref13-2" aria-controls="sref13-2">Cattoretti et&nbsp;al., 2005</a><div class="dropBlock__holder" data-db-target-of="sref13-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">13.</div><div>Cattoretti, G. ∙ Pasqualucci, L. ∙ Ballon, G. ...</div><div><strong>Deregulated BCL6 expression recapitulates the pathogenesis of human diffuse large B cell lymphomas in mice</strong></div><div><em>Cancer Cell.</em> 2005; <strong>7</strong>:445-455</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_13_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=10.1016%2Fj.ccr.2005.03.037&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_13_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=10.1016%2Fj.ccr.2005.03.037&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_13_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-19344368745" target="_blank" aria-label="Open Scopus in new tab">Scopus (258)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15894265/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.ccr.2005.03.037&amp;pmid=15894265" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td>N/A</td></tr><tr><td>Mouse: <i>VavP</i>-<i>Bcl2</i></td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref75-2" href="#" role="doc-biblioref" data-xml-rid="bib75" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref75-2" aria-controls="sref75-2">Ogilvy et&nbsp;al., 1999</a><div class="dropBlock__holder" data-db-target-of="sref75-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">76.</div><div>Ogilvy, S. ∙ Metcalf, D. ∙ Print, C.G. ...</div><div><strong>Constitutive Bcl-2 expression throughout the hematopoietic compartment affects multiple lineages and enhances progenitor cell survival</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 1999; <strong>96</strong>:14943-14948</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.96.26.14943" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_76_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-0033593038" target="_blank" aria-label="Open Scopus in new tab">Scopus (281)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/10611317/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.96.26.14943&amp;pmid=10611317" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td>N/A</td></tr><tr><td>Mouse: <i>Aicda</i>-KO</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref70-1" href="#" role="doc-biblioref" data-xml-rid="bib70" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref70-1" aria-controls="sref70-1">Muramatsu et&nbsp;al., 2000</a><div class="dropBlock__holder" data-db-target-of="sref70-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">70.</div><div>Muramatsu, M. ∙ Kinoshita, K. ∙ Fagarasan, S. ...</div><div><strong>Class switch recombination and hypermutation require activation-induced cytidine deaminase (AID), a potential RNA editing enzyme</strong></div><div><em>Cell.</em> 2000; <strong>102</strong>:553-563</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=10.1016%2FS0092-8674%2800%2900078-7&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=10.1016%2FS0092-8674%2800%2900078-7&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11007474/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0092-8674%2800%2900078-7&amp;pmid=11007474" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td>N/A</td></tr><tr><td>Mouse: <i>Tbl1xr1-D370Y</i></td><td>This manuscript</td><td>N/A</td></tr><tr class="rowsep"><td>Mouse: <i>Tbl1xr1-KO</i></td><td>This manuscript</td><td>N/A</td></tr><tr class="rowsep"><td colspan="3"><b>Oligonucleotides</b></td></tr><tr><td><i>TBL1XR1</i>_1 crRNA (Hs.Cas9.TBL1XR1.1.AG): 5′-GAUAUGGCUUUCUAUAC<br>CAAGUUUUAGAGCUAUGCU-3′</td><td>Integrated DNA Technologies</td><td>N/A</td></tr><tr><td><i>TBL1XR1</i>_2 crRNA ((Hs.Cas9.TBL1XR1.1.AA): 5′-UAUUGGUCGACCAUCAAACAG<br>UUUUAGAGCUAUGCU-3′</td><td>Integrated DNA Technologies</td><td>N/A</td></tr><tr><td><i>Tbl1xr1-D370Y</i> genotyping Fwd: 5′-AACGAGAAGCCAAGGTTCCAGTTC-3′</td><td>This manuscript</td><td>N/A</td></tr><tr><td><i>Tbl1xr1-D370Y</i> genotyping Rev: 5′-GATGCAAAGGTGTTGTTGCTCTGC-3′</td><td>This manuscript</td><td>N/A</td></tr><tr><td><i>Tbl1xr1-KO</i> genotyping Fwd: 5′-TATCCTCAGCGGTGGGTTAC-3′</td><td>This manuscript</td><td>N/A</td></tr><tr><td><i>Tbl1xr1-KO</i> genotyping Rev: 5′-GCCAGTAGGGGTTGCTGATA-3′</td><td>This manuscript</td><td>N/A</td></tr><tr><td>PIM1 sequencing Fwd: 5′- TTCGGCTCGGTCTACTCTG-3′</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref58-2" href="#" role="doc-biblioref" data-xml-rid="bib58" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref58-2" aria-controls="sref58-2">Liu et&nbsp;al., 2008</a><div class="dropBlock__holder" data-db-target-of="sref58-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">58.</div><div>Liu, M. ∙ Duke, J.L. ∙ Richter, D.J. ...</div><div><strong>Two levels of protection for the B cell genome during somatic hypermutation</strong></div><div><em>Nature.</em> 2008; <strong>451</strong>:841-845</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature06547" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-39149099085" target="_blank" aria-label="Open Scopus in new tab">Scopus (411)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/18273020/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature06547&amp;pmid=18273020" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>N/A</td></tr><tr><td>PIM1 sequencing Rev: 5′- GGAGGGAAAAGTGGGTCATAC-3′</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref58-3" href="#" role="doc-biblioref" data-xml-rid="bib58" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref58-3" aria-controls="sref58-3">Liu et&nbsp;al., 2008</a><div class="dropBlock__holder" data-db-target-of="sref58-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">58.</div><div>Liu, M. ∙ Duke, J.L. ∙ Richter, D.J. ...</div><div><strong>Two levels of protection for the B cell genome during somatic hypermutation</strong></div><div><em>Nature.</em> 2008; <strong>451</strong>:841-845</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature06547" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-39149099085" target="_blank" aria-label="Open Scopus in new tab">Scopus (411)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/18273020/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature06547&amp;pmid=18273020" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>N/A</td></tr><tr><td>Mouse V<sub>H</sub> sequencing Fwd: 5′- GGAATTCGCCTGACATCTGA<br>GGACTCTGC-3′</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref44-2" href="#" role="doc-biblioref" data-xml-rid="bib44" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref44-2" aria-controls="sref44-2">Jolly et&nbsp;al., 1997</a><div class="dropBlock__holder" data-db-target-of="sref44-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">44.</div><div>Jolly, C.J. ∙ Klix, N. ∙ Neuberger, M.S.</div><div><strong>Rapid methods for the analysis of immunoglobulin gene hypermutation: application to transgenic and gene targeted mice</strong></div><div><em>Nucleic Acids Res.</em> 1997; <strong>25</strong>:1913-1919</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/25.10.1913" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-0030845386" target="_blank" aria-label="Open Scopus in new tab">Scopus (106)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/9115357/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2F25.10.1913&amp;pmid=9115357" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>N/A</td></tr><tr><td>Mouse V<sub>H</sub> sequencing Rev: 5′- GACTAGTCCTCTCCAGTT<br>TCGGCTGAATCC-3′</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref44-3" href="#" role="doc-biblioref" data-xml-rid="bib44" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref44-3" aria-controls="sref44-3">Jolly et&nbsp;al., 1997</a><div class="dropBlock__holder" data-db-target-of="sref44-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">44.</div><div>Jolly, C.J. ∙ Klix, N. ∙ Neuberger, M.S.</div><div><strong>Rapid methods for the analysis of immunoglobulin gene hypermutation: application to transgenic and gene targeted mice</strong></div><div><em>Nucleic Acids Res.</em> 1997; <strong>25</strong>:1913-1919</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/25.10.1913" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-0030845386" target="_blank" aria-label="Open Scopus in new tab">Scopus (106)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/9115357/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2F25.10.1913&amp;pmid=9115357" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>N/A</td></tr><tr class="rowsep"><td>Primers used for RT-qPCR, see <a id="crosref2540" href="#mmc1">Table S6</a></td><td>This manuscript</td><td>N/A</td></tr><tr class="rowsep"><td colspan="3"><b>Recombinant DNA</b></td></tr><tr><td>Plasmid: pSBtet-GP</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref50-1" href="#" role="doc-biblioref" data-xml-rid="bib50" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref50-1" aria-controls="sref50-1">Kowarz et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref50-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">50.</div><div>Kowarz, E. ∙ Löscher, D. ∙ Marschalek, R.</div><div><strong>Optimized Sleeping Beauty transposons rapidly generate stable transgenic cell lines</strong></div><div><em>Biotechnol. J.</em> 2015; <strong>10</strong>:647-653</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/biot.201400821" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25650551/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fbiot.201400821&amp;pmid=25650551" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>Addgene plasmid #60495</td></tr><tr><td>Plasmid: pCMV(CAT)T7-SB100</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref62-1" href="#" role="doc-biblioref" data-xml-rid="bib62" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref62-1" aria-controls="sref62-1">Mátés et&nbsp;al., 2009</a><div class="dropBlock__holder" data-db-target-of="sref62-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">62.</div><div>Mátés, L. ∙ Chuah, M.K. ∙ Belay, E. ...</div><div><strong>Molecular evolution of a novel hyperactive Sleeping Beauty transposase enables robust stable gene transfer in vertebrates</strong></div><div><em>Nat. Genet.</em> 2009; <strong>41</strong>:753-761</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng.343" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-67349250885" target="_blank" aria-label="Open Scopus in new tab">Scopus (487)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19412179/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng.343&amp;pmid=19412179" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>Addgene plasmid #34879</td></tr><tr><td>Plasmid: pLIX_403</td><td>David Root Lab (Broad Institute, Cambridge, MA, USA); unpublished.</td><td>Addgene plasmid #41395</td></tr><tr><td>Plasmid: pCMV-VSV-G</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref99" href="#" role="doc-biblioref" data-xml-rid="bib99" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref99" aria-controls="sref99">Stewart et&nbsp;al., 2003</a><div class="dropBlock__holder" data-db-target-of="sref99"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">97.</div><div>Stewart, S.A. ∙ Dykxhoorn, D.M. ∙ Palliser, D. ...</div><div><strong>Lentivirus-delivered stable gene silencing by RNAi in primary cells</strong></div><div><em>RNA.</em> 2003; <strong>9</strong>:493-501</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1261/rna.2192803" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_97_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-0344375083" target="_blank" aria-label="Open Scopus in new tab">Scopus (782)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/12649500/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1261%2Frna.2192803&amp;pmid=12649500" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>Addgene plasmid #8454</td></tr><tr class="rowsep"><td>Plasmid: pCMV-dR8.91</td><td>Life Science Market</td><td>Cat# PVT2323</td></tr><tr class="rowsep"><td colspan="3"><b>Software and Algorithms</b></td></tr><tr><td>FlowJo</td><td>Becton Dickinson</td><td>Version 10.5.3</td></tr><tr><td>GraphPad Prism</td><td>GraphPad Software</td><td>Version 6.00</td></tr><tr><td>Aperio eSlide Manager</td><td>Leica Biosystems</td><td><a id="intref0015" href="https://www.leicabiosystems.com/aperio-eslide-manager/" target="_blank">https://www.leicabiosystems.com/aperio-eslide-manager/</a></td></tr><tr><td>GSEA v2.0.13</td><td>Broad Institute</td><td><a id="intref0020" href="https://software.broadinstitute.org/cancer/software/gsea" target="_blank">https://software.broadinstitute.org/cancer/software/gsea</a></td></tr><tr><td>CASAVA</td><td>Ilumina</td><td>Version 1.8.2</td></tr><tr><td>Fiji (ImageJ)</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref91-1" href="#" role="doc-biblioref" data-xml-rid="bib91" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref91-1" aria-controls="sref91-1">Schindelin et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref91-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">89.</div><div>Schindelin, J. ∙ Arganda-Carreras, I. ∙ Frise, E. ...</div><div><strong>Fiji: an open-source platform for biological-image analysis</strong></div><div><em>Nat. Methods.</em> 2012; <strong>9</strong>:676-682</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.2019" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_89_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-84862520770" target="_blank" aria-label="Open Scopus in new tab">Scopus (16304)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22743772/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.2019&amp;pmid=22743772" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref93-1" href="#" role="doc-biblioref" data-xml-rid="bib93" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref93-1" aria-controls="sref93-1">Schneider et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref93-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">91.</div><div>Schneider, C.A. ∙ Rasband, W.S. ∙ Eliceiri, K.W.</div><div><strong>NIH Image to ImageJ: 25 years of image analysis</strong></div><div><em>Nat. Methods.</em> 2012; <strong>9</strong>:671-675</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.2089" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-84863205849" target="_blank" aria-label="Open Scopus in new tab">Scopus (21916)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22930834/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.2089&amp;pmid=22930834" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0025" href="https://imagej.net/Fiji" target="_blank">https://imagej.net/Fiji</a></td></tr><tr><td>EdgeR</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref88-1" href="#" role="doc-biblioref" data-xml-rid="bib88" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref88-1" aria-controls="sref88-1">Robinson et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref88-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">86.</div><div>Robinson, M.D. ∙ McCarthy, D.J. ∙ Smyth, G.K.</div><div><strong>edgeR: a Bioconductor package for differential expression analysis of digital gene expression data</strong></div><div><em>Bioinformatics.</em> 2010; <strong>26</strong>:139-140</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btp616" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_86_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-75249087100" target="_blank" aria-label="Open Scopus in new tab">Scopus (11879)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19910308/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtp616&amp;pmid=19910308" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0030" href="https://bioconductor.org/packages/release/bioc/html/edgeR.html" target="_blank">https://bioconductor.org/packages/release/bioc/html/edgeR.html</a></td></tr><tr><td>Proteowizard</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref21-1" href="#" role="doc-biblioref" data-xml-rid="bib21" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref21-1" aria-controls="sref21-1">Craig and Beavis, 2004</a><div class="dropBlock__holder" data-db-target-of="sref21-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">21.</div><div>Craig, R. ∙ Beavis, R.C.</div><div><strong>TANDEM: matching proteins with tandem mass spectra</strong></div><div><em>Bioinformatics.</em> 2004; <strong>20</strong>:1466-1467</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/bth092" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-3142702204" target="_blank" aria-label="Open Scopus in new tab">Scopus (1780)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/14976030/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbth092&amp;pmid=14976030" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0035" href="http://proteowizard.sourceforge.net/" target="_blank">http://proteowizard.sourceforge.net/</a></td></tr><tr><td>X!Tandem</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref46-1" href="#" role="doc-biblioref" data-xml-rid="bib46" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref46-1" aria-controls="sref46-1">Kessner et&nbsp;al., 2008</a><div class="dropBlock__holder" data-db-target-of="sref46-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">46.</div><div>Kessner, D. ∙ Chambers, M. ∙ Burke, R. ...</div><div><strong>ProteoWizard: open source software for rapid proteomics tools development</strong></div><div><em>Bioinformatics.</em> 2008; <strong>24</strong>:2534-2536</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btn323" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-54949129419" target="_blank" aria-label="Open Scopus in new tab">Scopus (891)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/18606607/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtn323&amp;pmid=18606607" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0040" href="https://www.thegpm.org/TANDEM/" target="_blank">https://www.thegpm.org/TANDEM/</a></td></tr><tr><td>ProHits</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref59-1" href="#" role="doc-biblioref" data-xml-rid="bib59" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref59-1" aria-controls="sref59-1">Liu et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref59-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">59.</div><div>Liu, G. ∙ Zhang, J. ∙ Larsen, B. ...</div><div><strong>ProHits: integrated software for mass spectrometry-based interaction proteomics</strong></div><div><em>Nat. Biotechnol.</em> 2010; <strong>28</strong>:1015-1017</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nbt1010-1015" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-77957947158" target="_blank" aria-label="Open Scopus in new tab">Scopus (127)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20944583/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnbt1010-1015&amp;pmid=20944583" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0045" href="http://prohitsms.com/Prohits_download/list.php" target="_blank">http://prohitsms.com/Prohits_download/list.php</a></td></tr><tr><td>SAINT Express</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref17-1" href="#" role="doc-biblioref" data-xml-rid="bib17" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref17-1" aria-controls="sref17-1">Choi et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref17-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">17.</div><div>Choi, H. ∙ Larsen, B. ∙ Lin, Z.Y. ...</div><div><strong>SAINT: probabilistic scoring of affinity purification-mass spectrometry data</strong></div><div><em>Nat. Methods.</em> 2011; <strong>8</strong>:70-73</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.1541" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-78650861777" target="_blank" aria-label="Open Scopus in new tab">Scopus (359)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21131968/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.1541&amp;pmid=21131968" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0050" href="http://saint-apms.sourceforge.net" target="_blank">http://saint-apms.sourceforge.net</a></td></tr><tr><td>CRAPome Database</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref64-1" href="#" role="doc-biblioref" data-xml-rid="bib64" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref64-1" aria-controls="sref64-1">Mellacheruvu et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref64-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">64.</div><div>Mellacheruvu, D. ∙ Wright, Z. ∙ Couzens, A.L. ...</div><div><strong>The CRAPome: a contaminant repository for affinity purification-mass spectrometry data</strong></div><div><em>Nat. Methods.</em> 2013; <strong>10</strong>:730-736</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.2557" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_64_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-84881475940" target="_blank" aria-label="Open Scopus in new tab">Scopus (595)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23921808/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.2557&amp;pmid=23921808" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>v1.1 for Jurkart cells</td></tr><tr><td>STAR transcriptome aligner v2.5.1b</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref24-1" href="#" role="doc-biblioref" data-xml-rid="bib24" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref24-1" aria-controls="sref24-1">Dobin et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref24-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">24.</div><div>Dobin, A. ∙ Davis, C.A. ∙ Schlesinger, F. ...</div><div><strong>STAR: ultrafast universal RNA-seq aligner</strong></div><div><em>Bioinformatics.</em> 2013; <strong>29</strong>:15-21</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/bts635" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-84871809302" target="_blank" aria-label="Open Scopus in new tab">Scopus (8577)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23104886/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbts635&amp;pmid=23104886" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0055" href="https://code.google.com/archive/p/rna-star" target="_blank">https://code.google.com/archive/p/rna-star</a></td></tr><tr><td>Biopython</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref20-1" href="#" role="doc-biblioref" data-xml-rid="bib20" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref20-1" aria-controls="sref20-1">Cock et&nbsp;al., 2009</a><div class="dropBlock__holder" data-db-target-of="sref20-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">20.</div><div>Cock, P.J. ∙ Antao, T. ∙ Chang, J.T. ...</div><div><strong>Biopython: freely available Python tools for computational molecular biology and bioinformatics</strong></div><div><em>Bioinformatics.</em> 2009; <strong>25</strong>:1422-1423</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btp163" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-65649092976" target="_blank" aria-label="Open Scopus in new tab">Scopus (1419)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19304878/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtp163&amp;pmid=19304878" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0060" href="https://github.com/biopython" target="_blank">https://github.com/biopython</a></td></tr><tr><td>sangerseqR</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref38-1" href="#" role="doc-biblioref" data-xml-rid="bib38" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref38-1" aria-controls="sref38-1">Hill et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref38-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">38.</div><div>Hill, J.T. ∙ Demarest, B.L. ∙ Bisgrove, B.W. ...</div><div><strong>Poly peak parser: Method and software for identification of unknown indels using sanger sequencing of polymerase chain reaction products</strong></div><div><em>Dev. Dyn.</em> 2014; <strong>243</strong>:1632-1636</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/dvdy.24183" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_38_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-84924246056" target="_blank" aria-label="Open Scopus in new tab">Scopus (79)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25160973/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fdvdy.24183&amp;pmid=25160973" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>Version 1.22.0</td></tr><tr><td>PAGE algorithm</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref32-1" href="#" role="doc-biblioref" data-xml-rid="bib32" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref32-1" aria-controls="sref32-1">Goodarzi et&nbsp;al., 2009</a><div class="dropBlock__holder" data-db-target-of="sref32-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">32.</div><div>Goodarzi, H. ∙ Elemento, O. ∙ Tavazoie, S.</div><div><strong>Revealing global regulatory perturbations across human cancers</strong></div><div><em>Mol. Cell.</em> 2009; <strong>36</strong>:900-911</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=10.1016%2Fj.molcel.2009.11.016&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=10.1016%2Fj.molcel.2009.11.016&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-71149084382" target="_blank" aria-label="Open Scopus in new tab">Scopus (112)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20005852/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.molcel.2009.11.016&amp;pmid=20005852" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0065" href="https://github.com/goodarzilab/PAGE" target="_blank">https://github.com/goodarzilab/PAGE</a></td></tr><tr><td>VarScan</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref48-1" href="#" role="doc-biblioref" data-xml-rid="bib48" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref48-1" aria-controls="sref48-1">Koboldt et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref48-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">48.</div><div>Koboldt, D.C. ∙ Zhang, Q. ∙ Larson, D.E. ...</div><div><strong>VarScan 2: somatic mutation and copy number alteration discovery in cancer by exome sequencing</strong></div><div><em>Genome Res.</em> 2012; <strong>22</strong>:568-576</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/gr.129684.111" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_48_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-84863229597" target="_blank" aria-label="Open Scopus in new tab">Scopus (2007)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22300766/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2Fgr.129684.111&amp;pmid=22300766" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>Version 2.3.6</td></tr><tr><td>Strelka</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref90-1" href="#" role="doc-biblioref" data-xml-rid="bib90" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref90-1" aria-controls="sref90-1">Saunders et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref90-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">88.</div><div>Saunders, C.T. ∙ Wong, W.S. ∙ Swamy, S. ...</div><div><strong>Strelka: accurate somatic small-variant calling from sequenced tumor-normal sample pairs</strong></div><div><em>Bioinformatics.</em> 2012; <strong>28</strong>:1811-1817</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/bts271" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-84864153492" target="_blank" aria-label="Open Scopus in new tab">Scopus (676)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22581179/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbts271&amp;pmid=22581179" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>Version 1.0.13</td></tr><tr><td>MuTect</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref18-1" href="#" role="doc-biblioref" data-xml-rid="bib18" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref18-1" aria-controls="sref18-1">Cibulskis et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref18-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">18.</div><div>Cibulskis, K. ∙ Lawrence, M.S. ∙ Carter, S.L. ...</div><div><strong>Sensitive detection of somatic point mutations in impure and heterogeneous cancer samples</strong></div><div><em>Nat. Biotechnol.</em> 2013; <strong>31</strong>:213-219</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nbt.2514" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-84874025843" target="_blank" aria-label="Open Scopus in new tab">Scopus (1954)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23396013/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnbt.2514&amp;pmid=23396013" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>Version 1.1.4</td></tr><tr><td>SnpEff</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref19-1" href="#" role="doc-biblioref" data-xml-rid="bib19" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref19-1" aria-controls="sref19-1">Cingolani et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref19-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">19.</div><div>Cingolani, P. ∙ Platts, A. ∙ Wang, L. ...</div><div><strong>A program for annotating and predicting the effects of single nucleotide polymorphisms, SnpEff: SNPs in the genome of Drosophila melanogaster strain w1118; iso-2; iso-3</strong></div><div><em>Fly (Austin).</em> 2012; <strong>6</strong>:80-92</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.4161/fly.19695" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-84862506964" target="_blank" aria-label="Open Scopus in new tab">Scopus (3209)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22728672/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.4161%2Ffly.19695&amp;pmid=22728672" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>Version 4.2</td></tr><tr class="rowsep"><td>Biostrings</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref78" href="#" role="doc-biblioref" data-xml-rid="bib78" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref78" aria-controls="sref78">Pagès et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref78"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">79.</div><div>Pagès, H. ∙ Aboyoun, P. ∙ Gentleman, R. ...</div><div><strong>Biostrings: Efficient manipulation of biological strings. R package version 2.54.0</strong></div><div>2019</div><div><a id="intref0125" href="https://bioconductor.org/packages/release/bioc/html/Biostrings.html" target="_blank">https://bioconductor.org/packages/release/bioc/html/Biostrings.html</a></div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=H.Pag%C3%A8sP.AboyounR.GentlemanS.DebRoyBiostrings%3A+Efficient+manipulation+of+biological+strings.+R+package+version+2.54.02019https%3A%2F%2Fbioconductor.org%2Fpackages%2Frelease%2Fbioc%2Fhtml%2FBiostrings.html" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>Version 2.54.0</td></tr><tr class="rowsep"><td colspan="3"><b>Other</b></td></tr><tr><td>Graphical Abstract Cartoons</td><td>SERVIER Medical Art Repository</td><td><a id="intref0070" href="https://smart.servier.com" target="_blank">https://smart.servier.com</a></td></tr></tbody></table></div><figcaption><div class="core-table-tools"><ul><li><a class="core-table-show-action" href="/action/showFullTableHTML?isHtml=true&amp;tableId=undtbl1&amp;pii=S0092-8674%2820%2930682-6" target="_blank">Open table in a new tab</a></li></ul></div></figcaption></figure></div></div></section><section id="sec-4-2"><h3>Resource Availability</h3><section id="sec-4-2-1"><h4>Lead Contact</h4><div id="p0220" role="paragraph">Further information and requests for resources and reagents should be directed to and will be fulfilled by the Lead Contact, Ari M. Melnick (<a id="intref0075" href="mailto:amm2014@med.cornell.edu">amm2014@med.cornell.edu</a>).</div></section><section id="sec-4-2-2"><h4>Materials Availability</h4><div id="p0225" role="paragraph">Unique reagents generated in this study are available from the Lead Contact with a completed Materials Transfer Agreement.</div></section><section id="sec-4-2-3"><h4>Data and Code Availability</h4><div id="p0230" role="paragraph">The accession number for all RNA sequencing data reported in this paper is Gene Expression Omnibus (GEO): <a id="intref0080" href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE139059" target="_blank">GSE139059</a>. The accession number for all raw mass spectrometry data reported in this paper is MassIVE archive (<a id="intref0085" href="https://massive.ucsd.edu/ProteoSAFe/static/massive.jsp" target="_blank">https://massive.ucsd.edu/ProteoSAFe/static/massive.jsp</a>): MSV000085310. Code used for analysis is available upon request.</div></section></section><section id="sec-4-3"><h3>Experimental Model and Subject Details</h3><section id="sec-4-3-1"><h4>Mouse models</h4><div id="p0235" role="paragraph">Animal care was in strict compliance with institutional guidelines established by the Weill Cornell Medical College, the Guide for the Care and Use of Laboratory Animals (<span class="dropBlock reference-citations"><a id="body-ref-opt4ZAehgbwtz" href="#" role="doc-biblioref" data-xml-rid="bib119" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="opt4ZAehgbwtz" aria-controls="opt4ZAehgbwtz">National Research Council US Institute for Laboratory Animal Research, 1996</a><div class="dropBlock__holder" data-db-target-of="opt4ZAehgbwtz"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">71.</div><div>National Research Council (US) Institute for Laboratory Animal Research</div><div><em>Guide for the Care and Use of Laboratory Animals.</em>National Academies Press (US), Washington, DC, 1996</div><div><a id="interref0510" href="https://www.ncbi.nlm.nih.gov/books/NBK232589/%20doi:%2010.17226/5140" target="_blank">https://www.ncbi.nlm.nih.gov/books/NBK232589/ doi: 10.17226/5140</a></div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=National+Research+Council+%28US%29+Institute+for+Laboratory+Animal+ResearchGuide+for+the+Care+and+Use+of+Laboratory+Animals1996National+Academies+Press+%28US%29Washington%2C+DChttps%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fbooks%2FNBK232589%2F+doi%3A+10.17226%2F5140" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), and the Association for Assessment and Accreditation of Laboratory Animal Care International. The Research Animal Resource Center of the Weill Cornell Medical College of Medicine approved all mouse procedures.</div><div id="p0240" role="paragraph">The following strains were obtained from The Jackson Laboratory (Ben Harbor, ME, USA): C57BL/6J (CD45.2, stock 000664), Cγ1-Cre (stock 010611), CD19-Cre (stock 006785), μMT (stock 002288), B6.SJL-PtprcaPepcb/Boy (CD45.1, stock 002014), Rosa26-lox-stop-lox-YFP (stock 006148), OT-II (stock 004194; (<span class="dropBlock reference-citations"><a id="body-ref-sref6" href="#" role="doc-biblioref" data-xml-rid="bib6" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref6" aria-controls="sref6">Barnden et&nbsp;al., 1998</a><div class="dropBlock__holder" data-db-target-of="sref6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">6.</div><div>Barnden, M.J. ∙ Allison, J. ∙ Heath, W.R. ...</div><div><strong>Defective TCR expression in transgenic mice constructed using cDNA-based alpha- and beta-chain genes under the control of heterologous regulatory elements</strong></div><div><em>Immunol. Cell Biol.</em> 1998; <strong>76</strong>:34-40</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1046/j.1440-1711.1998.00709.x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-0031897801" target="_blank" aria-label="Open Scopus in new tab">Scopus (1041)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/9553774/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1046%2Fj.1440-1711.1998.00709.x&amp;pmid=9553774" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)), R26:FLPe knock in strain (stock 003946) and B1-8hi (stock 007594). The R26-Fucci2aR model (<span class="dropBlock reference-citations"><a id="body-ref-sref69-2" href="#" role="doc-biblioref" data-xml-rid="bib69" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref69-2" aria-controls="sref69-2">Mort et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref69-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">69.</div><div>Mort, R.L. ∙ Ford, M.J. ∙ Sakaue-Sawano, A. ...</div><div><strong>Fucci2a: a bicistronic cell cycle reporter that allows Cre mediated tissue specific expression in mice</strong></div><div><em>Cell Cycle.</em> 2014; <strong>13</strong>:2681-2696</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.4161/15384101.2015.945381" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-84907438909" target="_blank" aria-label="Open Scopus in new tab">Scopus (39)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25486356/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.4161%2F15384101.2015.945381&amp;pmid=25486356" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) was developed by I.J. Jackson (University of Edinburgh, Scotland). <i>IμBcl6</i> mice (<span class="dropBlock reference-citations"><a id="body-ref-sref13-3" href="#" role="doc-biblioref" data-xml-rid="bib13" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref13-3" aria-controls="sref13-3">Cattoretti et&nbsp;al., 2005</a><div class="dropBlock__holder" data-db-target-of="sref13-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">13.</div><div>Cattoretti, G. ∙ Pasqualucci, L. ∙ Ballon, G. ...</div><div><strong>Deregulated BCL6 expression recapitulates the pathogenesis of human diffuse large B cell lymphomas in mice</strong></div><div><em>Cancer Cell.</em> 2005; <strong>7</strong>:445-455</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_13_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=10.1016%2Fj.ccr.2005.03.037&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_13_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=10.1016%2Fj.ccr.2005.03.037&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_13_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-19344368745" target="_blank" aria-label="Open Scopus in new tab">Scopus (258)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15894265/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.ccr.2005.03.037&amp;pmid=15894265" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) were obtained from R. Dalla-Favera (Columbia University, NY, USA). The <i>VavP</i>-<i>Bcl2</i> (<span class="dropBlock reference-citations"><a id="body-ref-sref75-3" href="#" role="doc-biblioref" data-xml-rid="bib75" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref75-3" aria-controls="sref75-3">Ogilvy et&nbsp;al., 1999</a><div class="dropBlock__holder" data-db-target-of="sref75-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">76.</div><div>Ogilvy, S. ∙ Metcalf, D. ∙ Print, C.G. ...</div><div><strong>Constitutive Bcl-2 expression throughout the hematopoietic compartment affects multiple lineages and enhances progenitor cell survival</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 1999; <strong>96</strong>:14943-14948</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.96.26.14943" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_76_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-0033593038" target="_blank" aria-label="Open Scopus in new tab">Scopus (281)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/10611317/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.96.26.14943&amp;pmid=10611317" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) model was developed by J.M. Adams (Walter and Eliza Hall Institute of Medical Research, Australia). The <i>Aicda</i>-KO mouse model (<span class="dropBlock reference-citations"><a id="body-ref-sref70-2" href="#" role="doc-biblioref" data-xml-rid="bib70" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref70-2" aria-controls="sref70-2">Muramatsu et&nbsp;al., 2000</a><div class="dropBlock__holder" data-db-target-of="sref70-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">70.</div><div>Muramatsu, M. ∙ Kinoshita, K. ∙ Fagarasan, S. ...</div><div><strong>Class switch recombination and hypermutation require activation-induced cytidine deaminase (AID), a potential RNA editing enzyme</strong></div><div><em>Cell.</em> 2000; <strong>102</strong>:553-563</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=10.1016%2FS0092-8674%2800%2900078-7&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=10.1016%2FS0092-8674%2800%2900078-7&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11007474/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0092-8674%2800%2900078-7&amp;pmid=11007474" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) was a generous gift from T. Honjo (Kyoto University Graduate School of Medicine, Kyoto, Japan).</div><div id="p0245" role="paragraph">Conditional <i>Tbl1xr1</i>-D370Y mice were generated by inGenious Targeting Laboratory Inc. (Ronkonkoma, NY, USA), using an inversion and deletion approach (<a id="crosref1980" href="#figs1">Figure&nbsp;S1</a>A). To achieve the conditional activation of the D370Y mutation, WT LoxP and mutant Lox66/Lox71 (RE/LE mutants) sites were used. In brief, the genomic sequence from <i>Tbl1xr1</i> exon 11 to exon 12, and their flanking sequences, were duplicated and flanked by Lox66 and Lox71 sites. The specific point mutation GAC &gt; TAC (D &gt; Y) was introduced into the duplicated exon 12. Then, the Lox66/71 flanked sequence was inserted into intron 12, in the reverse direction. A 5′ WT LoxP site was inserted upstream of exon 11. An FRT-flanked Neomycin selection cassette was inserted immediately upstream of the Lox66/71 flanked inversion sequence. Embryonic stem (ES) cells were electroporated with this homologous targeting construct, and stable G418 clones were derived. Targeted iTL IC1 (C57BL/6) ES cells were microinjected into BALB/c blastocysts. Resulting chimeras with a high percentage of black coat were further screened by PCR, and mated to C57BL/6J R26:FLPe mice, to remove the Neomycin cassette. Founder mice were further backcrossed for at least 10 generations into the C57BL/6J background.</div><div id="p0250" role="paragraph">Conditional <i>Tbl1xr1</i>-KO mice were obtained by microinjection of targeted JMB8A3.N1 <i>Tbl1xr1</i><sup>tm1a(EUCOMM)Hmgu</sup> ES cells (clone HEPD0744_3_E01; EUCOMM program) into C57BL/6-albino blastocytes, and implantation into pseudo-pregnant foster C57BL/6-albino recipients (stock 000058; The Jackson Laboratory). Procedures were carried out by the Mouse Genetic - Transgenic Core Facility at Memorial Sloan Kettering Cancer Center (MSKCC, New York, NY, USA). Resulting chimeras with a high percentage of black coat were further screened by PCR, and mated to C57BL/6J R26:FLPe mice, to remove the FRT-flanked lacZ-Neomycin<sup>resistance</sup> cassette. Founder mice were further backcrossed for at least 10 generations into the C57BL/6J background.</div><div id="p0255" role="paragraph">All mouse experiments were conducted using aged and sex-matched specimens. Experiments were designed to include male and female specimens in all groups, and no sex-based influence/bias was detected in the observations made in this work. Unless stated otherwise in the text, all animals were 8 to 12&nbsp;weeks of age at the time of experimentation.</div></section><section id="sec-4-3-2"><h4>Cell lines</h4><div id="p0260" role="paragraph">The DLBCL cell line OCI-Ly1 (CVCL_1879; male origin) was grown in Iscove Modified Dulbecco Media (12440061; ThermoFisher Scientific), supplemented with 10% FBS and penicillin G/streptomycin; U2932 (CVCL_1896; female origin) cells were grown in Roswell Park Memorial Institute medium (10-040-CV; Corning; Corning, NY, USA), supplemented with 10% FBS, penicillin G/streptomycin, L-glutamine, and HEPES; HEK293T cells (CVCL_0063; female origin) were maintained in Dulbecco’s Modified Eagle Medium (11965-092; ThermoFisher Scientific), supplemented with 10% FBS and penicillin G/streptomycin. Murine 40LB cells (<span class="dropBlock reference-citations"><a id="body-ref-sref72-2" href="#" role="doc-biblioref" data-xml-rid="bib72" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref72-2" aria-controls="sref72-2">Nojima et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref72-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">73.</div><div>Nojima, T. ∙ Haniuda, K. ∙ Moutai, T. ...</div><div><strong>In-vitro derived germinal centre B cells differentially generate memory B or plasma cells in&nbsp;vivo</strong></div><div><em>Nat. Commun.</em> 2011; <strong>2</strong>:465</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ncomms1475" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-80052643363" target="_blank" aria-label="Open Scopus in new tab">Scopus (101)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21897376/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fncomms1475&amp;pmid=21897376" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) (male origin) were grown in DMEM media with 10% FBS. All cells were grown in incubators at 37C, in a 5% CO<sub>2</sub> atmosphere. Cell line authentication testing was performed at IDEXX BioResearch (<a id="intref0090" href="https://www.idexxbioresearch.com/cellcheck" target="_blank">https://www.idexxbioresearch.com/cellcheck</a>), using methods recommended by the American National Standards Institute (ANSI ASN-0002-2011). The cell lines were confirmed to be of human origin and tested for evidence of cross-species contamination (mouse, rat, Chinese hamster and African Green monkey). Short tandem repeat (STR) testing was performed and the genetic profile obtained was compared to the established cell line profile to confirm the cell lines are consistent with the established profile. These cell lines were also routinely tested for Mycoplasma contamination in the laboratory.</div></section><section id="sec-4-3-3"><h4>Bacteria strains</h4><div id="p0265" role="paragraph">TOP10, Stbl3 and DH5α <i>E.&nbsp;Coli</i> cells (ThermoFisher Scientific) were cultured in autoclave-sterilized Luria Bertani broth (BP1426-2; Fisher Scientific), in a MaxQ 8000 orbital shaker (ThermoFisher Scientific) set at 200rpm and 37°C.</div></section><section id="sec-4-3-4"><h4>Human subjects</h4><div id="p0270" role="paragraph">Information on <i>TBL1XR1</i> mutation status in human lymphoma specimens was retrieved from publicly available datasets. DLBCL cases (<span class="dropBlock reference-citations"><a id="body-ref-sref5-6" href="#" role="doc-biblioref" data-xml-rid="bib5" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref5-6" aria-controls="sref5-6">Arthur et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref5-6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">5.</div><div>Arthur, S.E. ∙ Jiang, A. ∙ Grande, B.M. ...</div><div><strong>Genome-wide discovery of somatic regulatory variants in diffuse large B-cell lymphoma</strong></div><div><em>Nat. Commun.</em> 2018; <strong>9</strong>:4001</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41467-018-06354-3" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-85054070552" target="_blank" aria-label="Open Scopus in new tab">Scopus (32)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30275490/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41467-018-06354-3&amp;pmid=30275490" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref61-5" href="#" role="doc-biblioref" data-xml-rid="bib61" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref61-5" aria-controls="sref61-5">Ma et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref61-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">61.</div><div>Ma, M.C.J. ∙ Tadros, S. ∙ Bouska, A. ...</div><div><strong>Pathognomonic and epistatic genetic alterations in B-cell non-Hodgkin lymphoma</strong></div><div><em>bioRxiv.</em> 2019;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/674259" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2F674259" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref86-4" href="#" role="doc-biblioref" data-xml-rid="bib86" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref86-4" aria-controls="sref86-4">Reddy et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref86-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">84.</div><div>Reddy, A. ∙ Zhang, J. ∙ Davis, N.S. ...</div><div><strong>Genetic and Functional Drivers of Diffuse Large B Cell Lymphoma</strong></div><div><em>Cell.</em> 2017; <strong>171</strong>:481-494</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_84_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=10.1016%2Fj.cell.2017.09.027&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_84_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=10.1016%2Fj.cell.2017.09.027&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_84_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-85030529459" target="_blank" aria-label="Open Scopus in new tab">Scopus (251)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28985567/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2017.09.027&amp;pmid=28985567" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref92-8" href="#" role="doc-biblioref" data-xml-rid="bib92" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref92-8" aria-controls="sref92-8">Schmitz et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref92-8"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">90.</div><div>Schmitz, R. ∙ Wright, G.W. ∙ Huang, D.W. ...</div><div><strong>Genetics and Pathogenesis of Diffuse Large B-Cell Lymphoma</strong></div><div><em>N.&nbsp;Engl. J. Med.</em> 2018; <strong>378</strong>:1396-1407</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1056/NEJMoa1801445" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_90_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-85045505659" target="_blank" aria-label="Open Scopus in new tab">Scopus (314)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29641966/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1056%2FNEJMoa1801445&amp;pmid=29641966" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>); FL cases: (<span class="dropBlock reference-citations"><a id="body-ref-sref51-3" href="#" role="doc-biblioref" data-xml-rid="bib51" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref51-3" aria-controls="sref51-3">Krysiak et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref51-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">51.</div><div>Krysiak, K. ∙ Gomez, F. ∙ White, B.S. ...</div><div><strong>Recurrent somatic mutations affecting B-cell receptor signaling pathway genes in follicular lymphoma</strong></div><div><em>Blood.</em> 2017; <strong>129</strong>:473-483</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1182/blood-2016-07-729954" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_51_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-85014960882" target="_blank" aria-label="Open Scopus in new tab">Scopus (58)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28064239/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1182%2Fblood-2016-07-729954&amp;pmid=28064239" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref61-6" href="#" role="doc-biblioref" data-xml-rid="bib61" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref61-6" aria-controls="sref61-6">Ma et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref61-6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">61.</div><div>Ma, M.C.J. ∙ Tadros, S. ∙ Bouska, A. ...</div><div><strong>Pathognomonic and epistatic genetic alterations in B-cell non-Hodgkin lymphoma</strong></div><div><em>bioRxiv.</em> 2019;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/674259" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2F674259" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref77-3" href="#" role="doc-biblioref" data-xml-rid="bib77" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref77-3" aria-controls="sref77-3">Ortega-Molina et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref77-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">78.</div><div>Ortega-Molina, A. ∙ Boss, I.W. ∙ Canela, A. ...</div><div><strong>The histone lysine methyltransferase KMT2D sustains a gene expression program that represses B cell lymphoma development</strong></div><div><em>Nat. Med.</em> 2015; <strong>21</strong>:1199-1208</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nm.3943" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_78_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-84943658052" target="_blank" aria-label="Open Scopus in new tab">Scopus (159)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26366710/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnm.3943&amp;pmid=26366710" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Clinic-pathological characteristics of these cases are detailed in the original publications. Further information on <i>TBL1XR1</i> mutation status is summarized in <a id="crosref2545" href="#mmc1">Table S1</a>.</div><div id="p0275" role="paragraph">All DLBCL human samples used for immunoprofiling studies were obtained with informed consent, and according to protocols approved by the BCCA Research Ethics Board. CD38 IHC on human specimens was performed on formalin-fixed paraffin-embedded tissue (FFPE) biopsies of 341 DLBCL (<span class="dropBlock reference-citations"><a id="body-ref-sref5-7" href="#" role="doc-biblioref" data-xml-rid="bib5" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref5-7" aria-controls="sref5-7">Arthur et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref5-7"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">5.</div><div>Arthur, S.E. ∙ Jiang, A. ∙ Grande, B.M. ...</div><div><strong>Genome-wide discovery of somatic regulatory variants in diffuse large B-cell lymphoma</strong></div><div><em>Nat. Commun.</em> 2018; <strong>9</strong>:4001</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41467-018-06354-3" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-85054070552" target="_blank" aria-label="Open Scopus in new tab">Scopus (32)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30275490/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41467-018-06354-3&amp;pmid=30275490" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Clinic-pathological characteristics of these cases are detailed in the original publication. For CyTOF immunoprofiling, diagnostic pre-treatment LN biopsies from patients with DLBCL, or non-malignant “reactive” LN (rLN) were acquired from the lymphoma tumor bank at the BC Cancer Agency (as described in <span class="dropBlock reference-citations"><a id="body-ref-sref71-2" href="#" role="doc-biblioref" data-xml-rid="bib71" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref71-2" aria-controls="sref71-2">Nissen et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref71-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">72.</div><div>Nissen, M.D. ∙ Kusakabe, M. ∙ Wang, X. ...</div><div><strong>Single Cell Phenotypic Profiling of 27 DLBCL Cases Reveals Marked Intertumoral and Intratumoral Heterogeneity</strong></div><div><em>Cytometry A.</em> 2019;</div><div>Published online October 22, 2019</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/cyto.a.23919" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-85075788395" target="_blank" aria-label="Open Scopus in new tab">Scopus (2)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31637838/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fcyto.a.23919&amp;pmid=31637838" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), and were selected based on sufficient numbers of viable cells for CyTOF analysis (2 million per sample). Clinic-pathological characteristics of these cases are detailed in the original publication. Further details on somatic mutations in these cases are summarized in <a id="crosref2040" href="#mmc5">Table S5</a>.</div></section></section><section id="sec-4-4"><h3>Method Details</h3><section id="sec-4-4-1"><h4>Germinal center assessment in mice</h4><div id="p0280" role="paragraph">To induce GC formation, age- and sex-matched mice were immunized intraperitoneally at 8 to 12&nbsp;weeks of age with either 0.5ml of a 2% sheep SRBC suspension in PBS (Cocalico Biologicals; Reamstown, PA, USA), or 100&nbsp;μg of the highly substituted hapten NP (NP<sub>16</sub> to NP<sub>32</sub>) conjugated to the carrier protein ovalbumin (OVA), or CGG (Chicken Gamma Globulin), or KLH (Keyhole Limpet Hemocyanin; all from Biosearch Technologies; Novato, CA, USA) absorbed to Imject Alum Adjuvant (77161; ThermoFisher Scientific; Waltham, MA, USA) at a 1:1 ratio. In the experiments where T<sub>FH</sub> interactions were blocked <i>in&nbsp;vivo</i>, mice received and i.v. injection of 100μg anti-CD40L antibody (clone MR-1, BE0017; BioXCell; West Lebanon, NH, USA) or control IgG antibody (BE0091; BioXCell) 4&nbsp;days after SRBC immunization, and a second dose of antibody 2&nbsp;days later. In the experiments where IL-9 was blocked <i>in&nbsp;vivo</i>, mice received a single i.v. injection of 200μg anti-IL-9 antibody (clone 9C1, BE0181; BioXCell) or control IgG2a antibody (BE0085; BioXCell) 7&nbsp;days after SRBC immunization. In these experiments, littermates of the same sex were randomly assigned to experimental groups. In experiments where GC cell cycle distribution was assessed, animals received an i.v. injection of 1mg EdU (E10187; ThermoFisher Scientific), 1h before euthanasia.</div></section><section id="sec-4-4-2"><h4>BM transplantations and lymphomagenesis studies</h4><div id="p0285" role="paragraph">Bone marrow cells were harvested from the tibia and femur of 8-12&nbsp;weeks old donor mice. After treatment with red blood cell lysis solution (158904; QIAGEN; Germantown, MD, USA), cells were mixed at the indicated ratios, and 1-2 million cells were injected into the tail vein of lethally irradiated C57BL/6J host mice (2 doses of 450rad, on a Rad Source Technologies RS 2000 Biological Research X-ray Irradiator). Transplanted mice were used for experiments 6-8&nbsp;weeks after transplant, to allow for full engraftment. With the exception of mice euthanized at specific time points, all mice involved in lymphomagenesis studies were monitored until any one of several criteria for euthanizing were met, including severe lethargy, more than 10% body weight loss, and palpable splenomegaly that extended across the midline, in accordance with our Weill Cornell Medicine Institutional Animal Care and Use Committee–approved animal protocol (protocol #2011-0031).</div></section><section id="sec-4-4-3"><h4>NB and MB adoptive transfers</h4><div id="p0290" role="paragraph">For NB cell adoptive transfers, total splenocytes were harvested from 8-12&nbsp;weeks old B1-8hi donor mice (<i>Cd45.1</i> or Cd45<i>.1/2</i>), and mature B cells were isolated using negative selection with CD43 magnetic beads (130-049-801; Miltenyi Biotec; Somerville, MA, USA). The percentage of NP-binding cells in this population was determined by FC (see below), and a number of mature B cells corresponding to 2-3x10<sup>5</sup> NP-binding B cells was injected i.v. into C57BL/6J recipient mice (<i>Cd45.2</i>). Recipient animals were immunized with an NP-conjugate 16h after cell transfer, and euthanized for analysis at the stated time points.</div><div id="p0295" role="paragraph">Antigen-specific MB cells for transfer experiments were generated following the protocol above, and splenic NP-binding MB cells were FACS-sorted three months, or more, after immunization of recipient mice with NP-CGG or NP-KLH. Splenic memory T&nbsp;cells (MT; CD3+CD4+CD44+CD62L-) were FACS-sorted from OT-II mice, 10&nbsp;days after immunization with 100μg OVA peptide (#A5503, Sigma-Aldrich; San Luis, MO, USA). For re-call experiments, 8x10<sup>3</sup> MB and 4x10<sup>4</sup> MT cells were injected i.v. into μMT recipient mice (<i>Cd45.2</i>). Recipient animals were immunized with NP-OVA 16h after transfer, and euthanized for analysis 3.5&nbsp;days after immunization.</div></section><section id="sec-4-4-4"><h4>Flow cytometry analysis and cell sorting</h4><div id="p0300" role="paragraph">Single-cell suspensions from mouse spleens or bone marrow were stained using the following fluorescent-labeled anti-mouse antibodies: from eBioscience ThermoFisher Scientific: APC anti-CD38 (17-0381, dilution 1:500), PE anti-CXCR4 (12-9991, dilution 1:400), PerCP-Cy5.5 anti-CD45.1 (45-0453, dilution 1:500), FITC anti-PD-1 (11-9985, dilution 1:200), PE-Cy7 streptavidin (25-4317, dilution 1:1000); from BD Biosciences (San Jose, CA, USA): PE-Cy7 anti-CD44 (560569, dilution 1:500), APC, PE-Cy7 and BV786 anti-B220 (553087, 552772 and 563894, dilution 1:500), PE-Cy7 anti-FAS (557653, dilution 1:500), BV395 anti-CD38 (740245, dilution 1:500), PE-Cy7 and BV421 anti-CD86 (560582 and 564198, dilution 1:300), biotin anti-CXCR4 (551968, dilution 1:200), biotin anti-CXCR5 (551960, dilution 1:200), BV421 and BV510 anti-IgD (744291 and 563110, dilution 1:500), PE and BV421 anti-IgG1 (550083 and 562580, dilution 1:500), BUV737 anti-CD138 (564430, dilution 1:500), FITC and BV711 anti-IgM (553437 and 743327, dilution 1:500), BV421 anti-CCR6 (564736, dilution 1:500), APC anti-CD117 (561074, dilution 1:500), FITC anti-CD19 (553785, dilution 1:500), PE-Cy7 anti-CD25 (561780, dilution 1:500), FITC and AF647 anti-GL7 (553666 and 561529, dilution 1:500); from BioLegend (San Diego, CA, USA): BV510 anti-CD62L (104441, dilution 1:500), PE anti-PCNA (307908, dilution 1:50), APC-Cy7 anti-CD4 (100414, dilution 1:500), PE anti-PD-L2 (107206, dilution 1:300), PerCP-Cy5.5 anti-CD45.2 (109828, dilution 1:500) BV421 anti-CD80 (104725, dilution 1:200) APC anti-CD3 (100235, dilution 1:500), PE anti-BCL6 (648304, dilution 1:500), AF647 anti-BLIMP1 (150004, dilution 1:300), BV421, APC-Cy7 and PE anti-B220 (103240, 103224 and 103208, dilution 1:500), APC-Cy7 anti-CD38 (102728, dilution 1:500), PerCP-Cy5.5 anti-GL7 (144610, dilution 1:500), PerCP-Cy5.5 anti-FAS (152610, dilution 1:500), PE-Cy7 and BV421 anti-CD138 (142514 and 142508, dilution 1:500), APC streptavidin (405207, dilution 1:1000), APC-Cy7 streptavidin (405208, dilution 1:1000); from R&amp;D Systems (Minneapolis, MN, USA): biotin anti-EFNB1 (BAF473, dilution 1:60) and anti-mS1P1 (MAB7089, dilution 1:50); from Biosearch Technologies: PE NP (N-5070-1, dilution 1:500); from Life Technologies: anti-AID (1AID-2E11; dilution 1:100). NIP-haptenated FITC was obtained from M.J. Shlomchik (University of Pittsburg, PA, USA) (<span class="dropBlock reference-citations"><a id="body-ref-sref2-2" href="#" role="doc-biblioref" data-xml-rid="bib2" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref2-2" aria-controls="sref2-2">Anderson et&nbsp;al., 2007</a><div class="dropBlock__holder" data-db-target-of="sref2-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">2.</div><div>Anderson, S.M. ∙ Tomayko, M.M. ∙ Ahuja, A. ...</div><div><strong>New markers for murine memory B cells that define mutated and unmutated subsets</strong></div><div><em>J.&nbsp;Exp. Med.</em> 2007; <strong>204</strong>:2103-2114</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1084/jem.20062571" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_2_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-34548383012" target="_blank" aria-label="Open Scopus in new tab">Scopus (164)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17698588/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1084%2Fjem.20062571&amp;pmid=17698588" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). For S1PR1 staining, a biotinylated goat anti-rat secondary antibody was used (112-065-167; dilution 1:1000, Jackson Immunoresearch; West Grove, PA, USA). For AID staining, a goat anti-mouse IgG cross-adsorbed secondary antibody conjugated to Alexa Fluor 647, was used (A21235, dilution 1:300; Thermo Scientific). To detect surface IL-9R expression, cells were incubated with 8&nbsp;ng/μl recombinant IL-9 (219-19; Peprotech, Rocky Hill, NJ) on ice for 60min, washed and fixed in 1% paraformaldehyde for 30min, and then stained with APC anti-IL-9 (RM9A4, dilution 1:250; BioLegend) on ice. As the negative control, incubation with IL-9 was omitted. Ghost Dye Violet 510 (13-0870; Tonbo Biosciences; San Diego, CA, USA) or DAPI (D1306; ThermoFisher Scientific) were used for the exclusion of dead cells. For intracellular markers, cells were fixed and permeabilized with the eBioscience Foxp3/Transcription Factor Staining Buffer Set (00-5523-00; ThermoFisher Scientific). PE-conjugated AnnexinV (556421, dilution 1:100; BD Biosciences) in AnnexinV binding buffer (BD Biosciences) was used to identify apoptotic cells. For cell cycle assessment, EdU incorporation was detected using the Click-iT Plus AF488 Flow Cytometry Assay Kit (C10633; ThermoFisher Scientific). Proliferation rates were assessed in cell lines using the CellTrace CFSE Cell Proliferation Kit (C34554; ThermoFisher Scientific), and in primary cultures using the eBioscience Cell Proliferation Dye eFluor 670 (65-0840-85; ThermoFisher Scientific), according to the vendor’s protocol. Data were acquired on BD FACS Canto II or BD Fortessa flow cytometer analyzers, and analyzed using FlowJo software package (TreeStar).</div><div id="p0305" role="paragraph">When B cell populations were sorted, single-cell suspensions of splenocytes were pre-enriched in B cells using positive selection with anti-B220 magnetic microbeads (130-049-501; Miltenyi Biotec), or negative selection with the EasySep Mouse B Cell Isolation Kit (19854; StemCell Technologies,). The stated populations were then isolated using a BD FACSAria II or a BD Influx sorter (BD Biosciences).</div></section><section id="sec-4-4-5"><h4>ELISA</h4><div id="p0310" role="paragraph">For analysis of T&nbsp;cell dependent antibody production, mice were immunized intraperitoneally with 100&nbsp;μg NP-CGG or NP-OVA or NP-KLH, ratio 16-32, in alum (1:1). Serum from each animal was collected before immunization, and on the stated days after immunization. Titers of low and high-affinity isotype-specific antibodies to NP were measured in plates coated with NP<sub>28</sub>-BSA or NP<sub>8</sub>-BSA, respectively, using the SBA Clonotyping System (5300-05; Southern Biotechnology; Birmingham, AL, USA), according to the manufacturer’s protocol. Results were assessed by spectrophotometric measurement of absorbance at 405nm using a Biotek Synergy Neo Alpha Plate Reader (BioTek; Winooski, VT, USA). Background readings of absorbance in negative control wells were A<sub>450</sub>&nbsp;&lt; 0.050.</div></section><section id="sec-4-4-6"><h4>Histology and immunohistochemistry</h4><div id="p0315" role="paragraph">Mice organs were fixed in 4% formaldehyde and embedded in paraffin. Tissue processing and staining were done by the Laboratory of Comparative Pathology (MSKCC). Briefly, five micron-sections were deparaffinized and heat antigen-retrieved in citrate buffer pH&nbsp;= 6.4, and endogenous peroxidase (HRP) activity was blocked by treating the sections with 3% hydrogen peroxide in methanol. Indirect immunohistochemistry was performed with anti-species-specific biotinylated secondary antibodies followed by avidin–horseradish peroxidase or avidin-AP, and developed by Vector Blue or DAB color substrates (Vector Laboratories; Burlingame, CA USA). Sections were counterstained with hematoxylin. The following primary antibodies were used: biotin-conjugated anti-B220 (550286; BD Biosciences), anti-CD3 (ab16669; Abcam; Cambridge, UK), anti-PNA (B1075; Vector Laboratories), anti-CD138 (553712; BD Biosciences), and anti-KI67 (12202; Cell Signaling Technology; Danvers, MA, USA). TUNEL staining was carried out as previously described (<span class="dropBlock reference-citations"><a id="body-ref-sref29" href="#" role="doc-biblioref" data-xml-rid="bib29" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref29" aria-controls="sref29">Gavrieli et&nbsp;al., 1992</a><div class="dropBlock__holder" data-db-target-of="sref29"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">29.</div><div>Gavrieli, Y. ∙ Sherman, Y. ∙ Ben-Sasson, S.A.</div><div><strong>Identification of programmed cell death in&nbsp;situ via specific labeling of nuclear DNA fragmentation</strong></div><div><em>J.&nbsp;Cell Biol.</em> 1992; <strong>119</strong>:493-501</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1083/jcb.119.3.493" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-0026648674" target="_blank" aria-label="Open Scopus in new tab">Scopus (8883)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/1400587/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1083%2Fjcb.119.3.493&amp;pmid=1400587" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). IHC on human specimens was performed on formalin-fixed paraffin-embedded tissue (FFPET) biopsies of 341 DLBCL cases within the cohort as described previously (<span class="dropBlock reference-citations"><a id="body-ref-sref28-2" href="#" role="doc-biblioref" data-xml-rid="bib28" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref28-2" aria-controls="sref28-2">Ennishi et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref28-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">28.</div><div>Ennishi, D. ∙ Mottok, A. ∙ Ben-Neriah, S. ...</div><div><strong>Genetic profiling of <i>MYC</i> and <i>BCL2</i> in diffuse large B-cell lymphoma determines cell-of-origin-specific clinical impact</strong></div><div><em>Blood.</em> 2017; <strong>129</strong>:2760-2770</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1182/blood-2016-11-747022" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-85019688331" target="_blank" aria-label="Open Scopus in new tab">Scopus (53)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28351934/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1182%2Fblood-2016-11-747022&amp;pmid=28351934" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref94-2" href="#" role="doc-biblioref" data-xml-rid="bib94" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref94-2" aria-controls="sref94-2">Scott et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref94-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">92.</div><div>Scott, D.W. ∙ Mottok, A. ∙ Ennishi, D. ...</div><div><strong>Prognostic Significance of Diffuse Large B-Cell Lymphoma Cell of Origin Determined by Digital Gene Expression in Formalin-Fixed Paraffin-Embedded Tissue Biopsies</strong></div><div><em>J.&nbsp;Clin. Oncol.</em> 2015; <strong>33</strong>:2848-2856</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1200/JCO.2014.60.2383" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_92_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-84964994916" target="_blank" aria-label="Open Scopus in new tab">Scopus (165)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26240231/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1200%2FJCO.2014.60.2383&amp;pmid=26240231" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Briefly, IHC staining on 4&nbsp;μm slides of TMAs was performed for CD38 (SPC32; MA5-14413; ThermoFisher Scientific) on the Benchmark XT platform (Ventana, AZ, USA), according to the previously described method (<span class="dropBlock reference-citations"><a id="body-ref-sref28-3" href="#" role="doc-biblioref" data-xml-rid="bib28" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref28-3" aria-controls="sref28-3">Ennishi et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref28-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">28.</div><div>Ennishi, D. ∙ Mottok, A. ∙ Ben-Neriah, S. ...</div><div><strong>Genetic profiling of <i>MYC</i> and <i>BCL2</i> in diffuse large B-cell lymphoma determines cell-of-origin-specific clinical impact</strong></div><div><em>Blood.</em> 2017; <strong>129</strong>:2760-2770</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1182/blood-2016-11-747022" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-85019688331" target="_blank" aria-label="Open Scopus in new tab">Scopus (53)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28351934/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1182%2Fblood-2016-11-747022&amp;pmid=28351934" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref94-3" href="#" role="doc-biblioref" data-xml-rid="bib94" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref94-3" aria-controls="sref94-3">Scott et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref94-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">92.</div><div>Scott, D.W. ∙ Mottok, A. ∙ Ennishi, D. ...</div><div><strong>Prognostic Significance of Diffuse Large B-Cell Lymphoma Cell of Origin Determined by Digital Gene Expression in Formalin-Fixed Paraffin-Embedded Tissue Biopsies</strong></div><div><em>J.&nbsp;Clin. Oncol.</em> 2015; <strong>33</strong>:2848-2856</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1200/JCO.2014.60.2383" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_92_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-84964994916" target="_blank" aria-label="Open Scopus in new tab">Scopus (165)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26240231/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1200%2FJCO.2014.60.2383&amp;pmid=26240231" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Slides were scanned using a Zeiss Mirax Slide Scanner and photomicrographs were examined using Aperio eSlide Manager (Leica Biosystems; Wetzlar, Germany). Fiji software (<span class="dropBlock reference-citations"><a id="body-ref-sref91-2" href="#" role="doc-biblioref" data-xml-rid="bib91" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref91-2" aria-controls="sref91-2">Schindelin et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref91-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">89.</div><div>Schindelin, J. ∙ Arganda-Carreras, I. ∙ Frise, E. ...</div><div><strong>Fiji: an open-source platform for biological-image analysis</strong></div><div><em>Nat. Methods.</em> 2012; <strong>9</strong>:676-682</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.2019" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_89_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-84862520770" target="_blank" aria-label="Open Scopus in new tab">Scopus (16304)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22743772/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.2019&amp;pmid=22743772" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref93-2" href="#" role="doc-biblioref" data-xml-rid="bib93" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref93-2" aria-controls="sref93-2">Schneider et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref93-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">91.</div><div>Schneider, C.A. ∙ Rasband, W.S. ∙ Eliceiri, K.W.</div><div><strong>NIH Image to ImageJ: 25 years of image analysis</strong></div><div><em>Nat. Methods.</em> 2012; <strong>9</strong>:671-675</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.2089" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-84863205849" target="_blank" aria-label="Open Scopus in new tab">Scopus (21916)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22930834/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.2089&amp;pmid=22930834" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) was used to quantify GC and TUNEL<sup>+</sup> cell/areas. CD38 staining was semiquantitatively assessed on tumor cells using HistoScore (HS&nbsp;= IxP): intensity (I&nbsp;= [1-3]) and percentage of positive cells (p&nbsp;= [0-100]). Specimens with HS &gt; 50 were defined as CD38 positive.</div></section><section id="sec-4-4-7"><h4>3D B cell follicular organoid</h4><div id="p0320" role="paragraph">Experiments using the organoid culture system were performed as previously described (<span class="dropBlock reference-citations"><a id="body-ref-sref9-2" href="#" role="doc-biblioref" data-xml-rid="bib9" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref9-2" aria-controls="sref9-2">Béguelin et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref9-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">9.</div><div>Béguelin, W. ∙ Rivas, M.A. ∙ Calvo Fernández, M.T. ...</div><div><strong>EZH2 enables germinal centre formation through epigenetic silencing of CDKN1A and an Rb-E2F1 feedback loop</strong></div><div><em>Nat. Commun.</em> 2017; <strong>8</strong>:877</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41467-017-01029-x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-85031291005" target="_blank" aria-label="Open Scopus in new tab">Scopus (48)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29026085/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41467-017-01029-x&amp;pmid=29026085" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). In brief, splenic B cells were obtained from naive animals through negative selection, using the EasySep Mouse B Cell Isolation Kit in accordance with manufacturer’s protocol. Feeder 40LB cells (<span class="dropBlock reference-citations"><a id="body-ref-sref72-3" href="#" role="doc-biblioref" data-xml-rid="bib72" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref72-3" aria-controls="sref72-3">Nojima et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref72-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">73.</div><div>Nojima, T. ∙ Haniuda, K. ∙ Moutai, T. ...</div><div><strong>In-vitro derived germinal centre B cells differentially generate memory B or plasma cells in&nbsp;vivo</strong></div><div><em>Nat. Commun.</em> 2011; <strong>2</strong>:465</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ncomms1475" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-80052643363" target="_blank" aria-label="Open Scopus in new tab">Scopus (101)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21897376/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fncomms1475&amp;pmid=21897376" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) were grown in DMEM media with 10% FBS and, at the time of experiment, were mitotically inhibited through incubation in cell culture complete medium containing 0.01 mg/ml Mitomycin C (MO503, Sigma-Aldrich,) at 37 °C for 55 min before the encapsulation. For organoid fabrication, gelatin stock solution was freshly prepared by mixing gelatin powder (Sigma-Aldrich) in RPMI-1640 medium followed by sterilization using syringe filter. Cells were mixed with warmed 5% gelatin stock solution and diluted accordingly using cell culture medium. Silicate nanoparticles (SiNP) with 25–30 nm in diameter and 1 nm in thickness were obtained from Southern Clay Products Inc. (Gonzales, TX, USA). A 3% hydrogel SiNP suspension was freshly prepared before the encapsulation procedure by mixing SiNP powder with deionized water and vortexing the resulting solution, followed by filtration through 0.22μm syringe filters immediately before use. Organoids were fabricated in 96-well plates by first adding 10 μl of 3% hydrogel SiNP followed by injecting 10 μl cell-containing gelatin solution into the initial SiNP droplet, and then mixing the entire hydrogel through repeated pipetting. Each organoid contained 50,000 B cells and 80,000 40LB cells. Organoids were cured for ∼10 min before the addition of RPMI media with 10% FBS and penicillin G/streptomycin, containing 50 ng/ml murine recombinant IL-4 (404-ML; R&amp;D Systems) and 25ng/ml murine recombinant IL-21 (594-ML; R&amp;D Systems) and were incubated at 37 °C with 5% CO2. Cell culture medium was renewed every 3&nbsp;days.</div></section><section id="sec-4-4-8"><h4>Generation of TBL1XR1-KO cell lines</h4><div id="p0325" role="paragraph">To generate <i>TBL1XR1</i>-KO clones using CRISPR, cell lines were electroporated using an Amaxa Nucleofector Unit and the SF Cell Line 4D-Nucleofector X Kit (PBC2-22500; Lonza; Basel, Switzerland), to incorporate a recombinant Cas9 nuclease (Alt-R® S.p. Cas9 Nuclease V3, 1081058), a <i>TBL1XR1</i>-targeting Alt-R® CRISPR-Cas9 crRNA (<i>TBL1XR1</i>_1 crRNA: 5′-GAU AUG GCU UUC UAU ACC AAG UUU UAG AGC UAU GCU-3′; <i>TBL1XR1</i>_2 crRNA: 5′-UAU UGG UCG ACC AUC AAA CAG UUU UAG AGC UAU GCU-3′), and an Alt-R® CRISPR-Cas9 tracrRNA (1075927; all from Integrated DNA Technologies; Coralville, IA, USA), following vendor’s recommendations. Forty-eight hours after electroporation, cells were single-cell plated into 96 well plates, and allowed to grow for at least 2&nbsp;weeks. Resulting clones were screened by WB, and positive clones were further verified by Sanger sequencing of genomic DNA. Clones expressing parental-like levels of TBL1XR1, and WT DNA sequences, were used as congenic controls for experiments.</div></section><section id="sec-4-4-9"><h4>Generation of TBL1XR1 mutant cell lines</h4><div id="p0330" role="paragraph">TBL1XR1 open reading frame (ORF) was amplified from HEK293T cells, and a Linker-His_Tag-V5_Tag sequence was added downstream by PCR (5′- GGCAGCAGCGGCCATCATCACCATCACCACGGTAAGCCTATCCCTAACCCTCTCCTCGGTCTCGATTCTACG-3′). The TBL1XR1-V5 insert was cloned into the pSBtet-GP backbone (#60495; Addgene; Watertown, MA, USA; (<span class="dropBlock reference-citations"><a id="body-ref-sref50-2" href="#" role="doc-biblioref" data-xml-rid="bib50" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref50-2" aria-controls="sref50-2">Kowarz et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref50-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">50.</div><div>Kowarz, E. ∙ Löscher, D. ∙ Marschalek, R.</div><div><strong>Optimized Sleeping Beauty transposons rapidly generate stable transgenic cell lines</strong></div><div><em>Biotechnol. J.</em> 2015; <strong>10</strong>:647-653</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/biot.201400821" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25650551/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fbiot.201400821&amp;pmid=25650551" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) using the SfiI restriction enzyme (New England Biolabs; Ipswich, MA, USA). <i>TBL1XR1</i> point mutations (D370Y, Y395H and Y446S) were introduced using the QuikChange II XL Site-directed mutagenesis kit (200521; Agilent Technologies; Santa Clara, CA, USA), and verified by Sanger sequencing. OCI-Ly1 cells were nucleofected to incorporate the different pSBtet-GP-TBL1XR1-V5 plasmids, along with a construct coding for the SB100X transposase [pCMV(CAT)T7-SB110; #34879, Addgene] (<span class="dropBlock reference-citations"><a id="body-ref-sref62-2" href="#" role="doc-biblioref" data-xml-rid="bib62" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref62-2" aria-controls="sref62-2">Mátés et&nbsp;al., 2009</a><div class="dropBlock__holder" data-db-target-of="sref62-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">62.</div><div>Mátés, L. ∙ Chuah, M.K. ∙ Belay, E. ...</div><div><strong>Molecular evolution of a novel hyperactive Sleeping Beauty transposase enables robust stable gene transfer in vertebrates</strong></div><div><em>Nat. Genet.</em> 2009; <strong>41</strong>:753-761</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng.343" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-67349250885" target="_blank" aria-label="Open Scopus in new tab">Scopus (487)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19412179/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng.343&amp;pmid=19412179" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Cells were selected with Puromycin for 7&nbsp;days, and incorporation of the construct was validated by FC for GFP. To induce TBL1XR1-V5 expression, cells were stimulated with Doxycycline (DOX) 1μg/ml for 48h in complete growth media.</div></section><section id="sec-4-4-10"><h4>Generation of cell lines for BioID experiments</h4><div id="p0335" role="paragraph"><i>TBL1XR1</i> or eGFP ORF were cloned into the pLIX_403 backbone (#41395; Addgene; David Root (Broad Institute, Cambridge, MA, USA), downstream of FLAG-BirA(R118G) ORF (#36047; Addgene) (<span class="dropBlock reference-citations"><a id="body-ref-sref89" href="#" role="doc-biblioref" data-xml-rid="bib89" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref89" aria-controls="sref89">Roux et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref89"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">87.</div><div>Roux, K.J. ∙ Kim, D.I. ∙ Raida, M. ...</div><div><strong>A promiscuous biotin ligase fusion protein identifies proximal and interacting proteins in mammalian cells</strong></div><div><em>J.&nbsp;Cell Biol.</em> 2012; <strong>196</strong>:801-810</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1083/jcb.201112098" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_87_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-84860270506" target="_blank" aria-label="Open Scopus in new tab">Scopus (715)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22412018/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1083%2Fjcb.201112098&amp;pmid=22412018" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), using the Gateway LR Clonase II Enzyme mix (11791020; ThermoFisher Scientific). <i>TBL1XR1</i>-Y446S mutation was introduced using the QuikChange II Site-directed mutagenesis kit, and verified by Sanger sequencing. Lentivirus were produced in HEK293T cells co-transfected with pCMV-VSV-G and pCMV-dR8.91 plasmids, and then concentrated using PEG-it (LV810A-1; System Biosciences; Palo Alto, CA, USA). OCI-Ly1 and U2932 cells were infected and cultured for 48h, and then selected with Puromycin for 7&nbsp;days. To induce expression of the FLAG-BirA-fusion proteins, cells were stimulated with DOX 1μg/ml for 24h. Incorporation of the construct was validated by WB for BirA (<a id="crosref2110" href="#figs5">Figure&nbsp;S5</a>C), or FC for eGFP.</div></section><section id="sec-4-4-11"><h4>BioID experiments</h4><div id="p0340" role="paragraph">Expression of FLAG-BirA-fusion proteins in OCI-Ly1 and U2932 cells was induced by treatment with Doxycycline 1μg/ml for 24h, after which biotin 50μM (B4501; Sigma-Aldrich) was added to the media for additional 24h. Cell pellets were washed twice with PBS, and lysed in modified RIPA lysis buffer (50mM Tris-HCl pH&nbsp;= 7.5, 150mM NaCl, 1mM EDTA, 1mM EGTA, 1% Triton X-100, 0.1% SDS, 1:500 protease inhibitor cocktail (11873580001; Sigma-Aldrich), 1:100 Benzonase nuclease (E1014-5KU; Sigma-Aldrich), at 4°C for 1h, and sonicated for 30&nbsp;s at 35% power to disrupt visible aggregates. Lysates were then centrifuged at 16,000rpm for 30min, and clarified supernatants were incubated with 30μL packed pre-equilibrated streptavidin-Sepharose beads (GE17-5113-01; GE Healthcare Life Sciences; Chicago, IL, USA) at 4°C for 3h. Beads were collected by centrifugation, washed six times with 50mM ammonium bicarbonate pH&nbsp;= 8.3, and treated with TPCK trypsin (V5111; Promega Corporation; Madison, WI, USA). The supernatant, containing tryptic peptides, was collected and lyophilized. Peptides were resuspended in 0.1% formic acid.</div><div id="p0345" role="paragraph">LC-MS/MS was conducted using a 120min reversed-phase buffer gradient running at 150nL/min (column heated to 40°C) on a Proxeon EASY-nLC pump in-line with a hybrid LTQ-Orbitrap Velos mass spectrometer (ThermoFisher Scientific). A parent ion scan was performed in the Orbitrap, using a resolving power of 60000. Simultaneously, up to twenty of the most intense peaks were selected for MS/MS (minimum ion count of 1000 for activation) using standard CID fragmentation. Fragment ions were detected in the LTQ. Dynamic exclusion was activated such that MS/MS of the same m/z (within a 10ppm window, exclusion list size 500) detected three times within 45&nbsp;s were excluded from analysis for 30 s. For protein identification, Proteowizard was used to convertraw files to .mzXML, and searched using X!Tandem (<span class="dropBlock reference-citations"><a id="body-ref-sref21-2" href="#" role="doc-biblioref" data-xml-rid="bib21" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref21-2" aria-controls="sref21-2">Craig and Beavis, 2004</a><div class="dropBlock__holder" data-db-target-of="sref21-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">21.</div><div>Craig, R. ∙ Beavis, R.C.</div><div><strong>TANDEM: matching proteins with tandem mass spectra</strong></div><div><em>Bioinformatics.</em> 2004; <strong>20</strong>:1466-1467</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/bth092" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-3142702204" target="_blank" aria-label="Open Scopus in new tab">Scopus (1780)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/14976030/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbth092&amp;pmid=14976030" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref46-2" href="#" role="doc-biblioref" data-xml-rid="bib46" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref46-2" aria-controls="sref46-2">Kessner et&nbsp;al., 2008</a><div class="dropBlock__holder" data-db-target-of="sref46-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">46.</div><div>Kessner, D. ∙ Chambers, M. ∙ Burke, R. ...</div><div><strong>ProteoWizard: open source software for rapid proteomics tools development</strong></div><div><em>Bioinformatics.</em> 2008; <strong>24</strong>:2534-2536</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btn323" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-54949129419" target="_blank" aria-label="Open Scopus in new tab">Scopus (891)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/18606607/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtn323&amp;pmid=18606607" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) against Human RefSeq Version 45. Search parameters specified a parent MS tolerance of 15ppm and an MS/MS fragment ion tolerance of 0.4Da, with up to two missed cleavages allowed for trypsin. Oxidation of methionine and ubiquitylation of lysine residues were allowed as variable modifications. Data were analyzed using a trans-proteomic pipeline via the ProHits software suite (<span class="dropBlock reference-citations"><a id="body-ref-sref59-2" href="#" role="doc-biblioref" data-xml-rid="bib59" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref59-2" aria-controls="sref59-2">Liu et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref59-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">59.</div><div>Liu, G. ∙ Zhang, J. ∙ Larsen, B. ...</div><div><strong>ProHits: integrated software for mass spectrometry-based interaction proteomics</strong></div><div><em>Nat. Biotechnol.</em> 2010; <strong>28</strong>:1015-1017</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nbt1010-1015" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-77957947158" target="_blank" aria-label="Open Scopus in new tab">Scopus (127)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20944583/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnbt1010-1015&amp;pmid=20944583" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Proteins identified with a ProteinProphet cut-off value 0.80 (corresponding to FDR&nbsp;&lt; 1%) were analyzed with SAINT Express (<span class="dropBlock reference-citations"><a id="body-ref-sref17-2" href="#" role="doc-biblioref" data-xml-rid="bib17" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref17-2" aria-controls="sref17-2">Choi et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref17-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">17.</div><div>Choi, H. ∙ Larsen, B. ∙ Lin, Z.Y. ...</div><div><strong>SAINT: probabilistic scoring of affinity purification-mass spectrometry data</strong></div><div><em>Nat. Methods.</em> 2011; <strong>8</strong>:70-73</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.1541" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-78650861777" target="_blank" aria-label="Open Scopus in new tab">Scopus (359)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21131968/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.1541&amp;pmid=21131968" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Four control runs of Flag-BirA-eGFP were used for comparative purposes. The control runs were collapsed to the two highest spectral counts for each hit. Prey identified with ≥ 2 spectral counts in each of the two replicates with a SAINT ≥ 75% were considered significant. Known common contaminants (CRAPome Database v1.1 for Jurkart cells) (<span class="dropBlock reference-citations"><a id="body-ref-sref64-2" href="#" role="doc-biblioref" data-xml-rid="bib64" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref64-2" aria-controls="sref64-2">Mellacheruvu et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref64-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">64.</div><div>Mellacheruvu, D. ∙ Wright, Z. ∙ Couzens, A.L. ...</div><div><strong>The CRAPome: a contaminant repository for affinity purification-mass spectrometry data</strong></div><div><em>Nat. Methods.</em> 2013; <strong>10</strong>:730-736</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.2557" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_64_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-84881475940" target="_blank" aria-label="Open Scopus in new tab">Scopus (595)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23921808/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.2557&amp;pmid=23921808" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) were removed from the final protein lists. All raw mass spectrometry files have been deposited at the MassIVE archive (<a id="intref0095" href="https://massive.ucsd.edu/ProteoSAFe/static/massive.jsp" target="_blank">https://massive.ucsd.edu/ProteoSAFe/static/massive.jsp</a>).</div></section><section id="sec-4-4-12"><h4>Cell lysis, immunoblotting and immunoprecipitation</h4><div id="p0350" role="paragraph">Whole cell protein lysates from cell lines or GCB were obtained using a buffer containing Tris-HCl 50mM (pH&nbsp;= 7.4), NaCl 150mM, EDTA 1mM, EGTA 1mM, NP-40 1%, SDS 0.1%, glycerol 10%, and complete protease inhibitor cocktail. Protein extracts for IP experiments were obtained using the Nuclear Complex Co-IP Kit (54001; Active Motif; Carlsbad, CA, USA). For IP experiments, 500-1000ug of nuclear protein extracts were incubated ON with V5-Tag magnetic beads (M16711; MBL International Corporation; Woburn, MA, USA), or with the indicated primary antibodies [BCL6 (sc-858) and HDAC3 (sc-11417) from Santa Cruz Biotechnology (Dallas, TX, USA); NCOR2 (06-891) from Sigma Aldrich; BACH2 (ARP39513_P050) from Aviva Systems Biology (San Diego, CA, USA); Rabbit anti-mouse control IgG (ab46540) from Abcam], followed by a 2h incubation with Protein A Dynabeads (10001D; ThermoFisher Scientific). Whole cell lysates or IP products were resolved by SDS-PAGE, transferred to PVDF membrane (1620177; Bio-Rad Laboratories; Hercules, CA, USA), and probed with the indicated primary antibodies [BCL6 (sc-7388), HDAC3 (sc-376957) and β-ACTIN (sc-47778) from Santa Cruz Biotechnology; NCOR2 (PA1-843) from ThermoFisher Scientific; BACH2 (ARP39513_P050); TBL1XR1 (H00079710-M01) from Abnova (Taipei City, Taiwan); BirA (BID-CP-100) from BioFront Technologies (Tallahassee, FL, USA)]. Membranes were then incubated with a corresponding peroxidase-conjugated secondary antibody (Jackson Immunoresearch). Protein signals were detected using enhanced chemiluminescence (RPN2232; GE Healthcare Life Sciences).</div></section><section id="sec-4-4-13"><h4>Rapid immunoprecipitation mass spectrometry of endogenous proteins</h4><div id="p0355" role="paragraph">RIME experiments (<span class="dropBlock reference-citations"><a id="body-ref-sref67" href="#" role="doc-biblioref" data-xml-rid="bib67" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref67" aria-controls="sref67">Mohammed et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref67"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">67.</div><div>Mohammed, H. ∙ Taylor, C. ∙ Brown, G.D. ...</div><div><strong>Rapid immunoprecipitation mass spectrometry of endogenous proteins (RIME) for analysis of chromatin complexes</strong></div><div><em>Nat. Protoc.</em> 2016; <strong>11</strong>:316-326</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nprot.2016.020" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26797456/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnprot.2016.020&amp;pmid=26797456" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) in OCI-Ly1 cells were performed by Active Motif, using an anti-BCL6 antibody (sc-858), or an isotype-matched IgG control (ab46540). Biological duplicates were prepared for each immunoprecipitation. Known common contaminants (CRAPome Database v1.1 for Jurkart cells) were removed from the final protein list.</div></section><section id="sec-4-4-14"><h4>RNA and genomic DNA and extraction</h4><div id="p0360" role="paragraph">Genomic DNA was extracted using: a) the QuickExtract DNA Extraction Solution (QE09050; Epicenter; Madison, WI, USA), for human cell lines; b) the Puregene Gentra cell kit (158388; QIAGEN), for mouse cell suspensions; c) the DirectPCR Lysis Reagent (101-T; Viagen Biotech; Los Angeles, CA, USA), for mouse tails. Total RNA was extracted from cell suspensions using Trizol LS (10296010; ThermoFisher Scientific) and RNeasy Mini Kit (74106; QIAGEN), with DNase treatment. RNA concentration was determined using Qubit Fluorometric Quantification (ThermoFisher Scientific) and integrity was verified using Agilent 2100 Bioanalyzer (Agilent Technologies).</div></section><section id="sec-4-4-15"><h4>Genotyping PCR</h4><div id="p0365" role="paragraph">PCR-based genotyping of the <i>Tbl1xr1</i>-D370Y allele was done on genomic DNA extracted from mouse tail tissues, using the GoTaq® DNA Polymerase (M3001; Promega Corporation). Primers were designed to flank the distal LoxP site located between <i>Tbl1xr1</i> exons 10 and 11, in the targeted allele (<a id="crosref2145" href="#figs1">Figures S1</a>A and S1B) (Forward: 5′-AAC GAG AAG CCA AGG TTC CAG TTC-3′; Reverse: 5′-GAT GCA AAG GTG TTG TTG CTC TGC-3′). PCR program: [2′: 95C] x1, [30”: 95C, 30”: @55C, 45”: 72C] x35, [5′: 72C] x1. PCR-based validation of exon 5 excision in <i>Tbl1xr1</i><sup><i>KO/KO</i></sup> mice was done on genomic DNA from sorted GCB. Primers were designed to target <i>Tbl1xr1</i> exon 5 (<a id="crosref2150" href="#figs1">Figure&nbsp;S1</a>G) (Forward: 5′-TAT CCT CAG CGG TGG GTT AC-3′; Reverse: 5′-GCC AGT AGG GGT TGC TGA TA-3′). PCR program: [2′: 95C] x1, [1’: 95C, 1’: @52C, 2′: 72C] x35, [5′: 72C] x1. PCR products were resolved by agarose gel electrophoresis, and visualized using SYBR Safe DNA stain (S33102; ThermoFisher Scientific).</div></section><section id="sec-4-4-16"><h4>Quantitative real-time PCR</h4><div id="p0370" role="paragraph">cDNA synthesis from total RNA extracts was performed using the Verso cDNA Synthesis kit (AB1453B; ThermoFisher Scientific). Expression of the genes of interest was detected using the Fast SYBR Green Master Mix (4385614; ThermoFisher Scientific) on a QuantStudio6 Flex Real-Time PCR System (ThermoFisher Scientific). Gene expression was normalized to Actin or GAPDH levels, using the ΔΔC(t) method, and results were represented as mRNA expression. See <a id="crosref2550" href="#mmc1">Table S6</a> for primer sequences.</div></section><section id="sec-4-4-17"><h4>Sanger sequencing</h4><div id="p0375" role="paragraph">cDNA from FACS-sorted <i>Tbl1xr1</i><sup><i>WT/WT</i></sup> or <i>Tbl1xr1</i><sup><i>D370Y/WT</i></sup> GCB, or from the indicated cells, was generated as described above. Sanger sequencing was performed by GENEWIZ (South Plainfield, NJ, USA).</div></section><section id="sec-4-4-18"><h4>Assessment of Pim1 somatic mutations</h4><div id="p0380" role="paragraph"><i>Pim1</i> locus was amplified from genomic DNA using the primers from <span class="dropBlock reference-citations"><a id="body-ref-sref58-4" href="#" role="doc-biblioref" data-xml-rid="bib58" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref58-4" aria-controls="sref58-4">Liu et&nbsp;al. (2008)</a><div class="dropBlock__holder" data-db-target-of="sref58-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">58.</div><div>Liu, M. ∙ Duke, J.L. ∙ Richter, D.J. ...</div><div><strong>Two levels of protection for the B cell genome during somatic hypermutation</strong></div><div><em>Nature.</em> 2008; <strong>451</strong>:841-845</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature06547" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-39149099085" target="_blank" aria-label="Open Scopus in new tab">Scopus (411)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/18273020/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature06547&amp;pmid=18273020" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>: Fwd: 5′-TTC GGC TCG GTC TAC TCT G-3′; Rev: 5′-GGA GGG AAA AGT GGG TCA TAC-3′. PCR program: [2′: 95C] x1, [1’: 95C, 1’: @65C, 1’: 72C] x25, [15’: 72C] x1. PCR products were resolved by agarose gel electrophoresis, and extracted using the QIAquick Gel Extraction Kit (28704; QIAGEN). Purified products were then cloned using the TOPO TA cloning kit for sequencing (K457501; ThermoFisher Scientific), following manufacturer instructions. Plates were grown ON at 37C, and bacteria colony sequencing was performed by GENEWIZ, using the T7 universal sequencing primer (5′-TAA TAC GAC TCA CTA TAG GG-3′). Only good-quality sequence was considered, as determined by inspection of the chromatograms. Mutation mismatch counts were calculated from Sanger sequencing using the sangerseqR (<span class="dropBlock reference-citations"><a id="body-ref-sref38-2" href="#" role="doc-biblioref" data-xml-rid="bib38" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref38-2" aria-controls="sref38-2">Hill et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref38-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">38.</div><div>Hill, J.T. ∙ Demarest, B.L. ∙ Bisgrove, B.W. ...</div><div><strong>Poly peak parser: Method and software for identification of unknown indels using sanger sequencing of polymerase chain reaction products</strong></div><div><em>Dev. Dyn.</em> 2014; <strong>243</strong>:1632-1636</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/dvdy.24183" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_38_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-84924246056" target="_blank" aria-label="Open Scopus in new tab">Scopus (79)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25160973/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fdvdy.24183&amp;pmid=25160973" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and Biostrings (R package version 2.54.0) packages. As a negative control for SHM, genomic DNA extracted from sorted <i>Acida</i><sup><i>KO/KO</i></sup> GCB was used as PCR template.</div></section><section id="sec-4-4-19"><h4>J<sub>H</sub>4 intron sequencing</h4><div id="p0385" role="paragraph">Sorted GCB or tumor cells were collected by centrifugation and resuspended in 500μL DNA cell lysis buffer (100mM Tris pH 8.8, 200mM NaCl, 5mM EDTA, 0.2% SDS) containing 0.2mg/mL proteinase K and incubated at 56°C overnight. 500μL isopropanol and 40μg glycogen was added to precipitate DNA. The mixture was incubated at 4°C for 20&nbsp;minutes followed by centrifugation for 20&nbsp;minutes at 20,000 x g and 4°C. The pellet was washed with 1mL 70% ethanol and centrifuged for 10&nbsp;minutes at 20,000xg and 4°C. The pellet was resuspended with 20μL DEPC-H2O and incubated at 37°C for 40&nbsp;minutes. JH4 intron sequences were amplified from germinal center B cell genomic DNA by PCR using J<sub>H</sub>4 forward primer (5′-GGA ATT CGC CTG ACA TCT GAG GAC TCT GC-3′), JH4 reverse primer (5′-GAC TAG TCC TCT CCA GTT TCG GCT GAA TCC-3′) (<span class="dropBlock reference-citations"><a id="body-ref-sref44-4" href="#" role="doc-biblioref" data-xml-rid="bib44" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref44-4" aria-controls="sref44-4">Jolly et&nbsp;al., 1997</a><div class="dropBlock__holder" data-db-target-of="sref44-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">44.</div><div>Jolly, C.J. ∙ Klix, N. ∙ Neuberger, M.S.</div><div><strong>Rapid methods for the analysis of immunoglobulin gene hypermutation: application to transgenic and gene targeted mice</strong></div><div><em>Nucleic Acids Res.</em> 1997; <strong>25</strong>:1913-1919</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/25.10.1913" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-0030845386" target="_blank" aria-label="Open Scopus in new tab">Scopus (106)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/9115357/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2F25.10.1913&amp;pmid=9115357" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), and Phusion High-Fidelity DNA Polymerase (M0530L: New England BioLabs). PCR program: [3′: 98C] x1, [30”: 98C, 1’: 72C] x39, [10’: 72C] x1. PCR products were resolved by agarose gel electrophoresis, and the 1.2kb band extracted using the QIAquick Gel Extraction Kit. The amplicon was ligated into the pCR-Blunt II-TOPO vector and transformed into One Shot TOP10 chemically competent <i>E.&nbsp;coli</i> (K280002; ThermoFisher Scientific) according to the manufacturer’s protocol. Clones were sequenced using the bacterial colony Sanger sequencing service provided by Eton Bioscience (Union, NJ, USA) and JH4 sequencing primer (5′-CCA TAC ACA TAC TTC TGT GTT CC-3′). As a negative control for SHM, genomic DNA extracted from sorted <i>Acida</i><sup><i>KO/KO</i></sup> GCB was used as PCR template. Sequences were aligned using MUSCLE (EMBL-EBI) included in Biopython package (<span class="dropBlock reference-citations"><a id="body-ref-sref20-2" href="#" role="doc-biblioref" data-xml-rid="bib20" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref20-2" aria-controls="sref20-2">Cock et&nbsp;al., 2009</a><div class="dropBlock__holder" data-db-target-of="sref20-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">20.</div><div>Cock, P.J. ∙ Antao, T. ∙ Chang, J.T. ...</div><div><strong>Biopython: freely available Python tools for computational molecular biology and bioinformatics</strong></div><div><em>Bioinformatics.</em> 2009; <strong>25</strong>:1422-1423</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btp163" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-65649092976" target="_blank" aria-label="Open Scopus in new tab">Scopus (1419)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19304878/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtp163&amp;pmid=19304878" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and analyzed for mutations using Python 3 script.</div></section><section id="sec-4-4-20"><h4>RNA sequencing and analysis</h4><div id="p0390" role="paragraph">Library preparation, sequencing and post-processing of the raw data was performed at the Epigenomics Core at Weill Cornell Medicine. Samples that passed the quality control (RNA Integrity Number ≥ 8) were subjected to library preparation using the Illumina TruSeq stranded-mRNA sample kits (Illumina; San Diego, CA, USA), according to the manufacturer. Briefly, poly A+ RNA was purified from 100&nbsp;ng of total RNA with oligo-dT beads. Purified mRNA was fragmented with divalent cations at elevated temperature, to ∼200&nbsp;bp. Following dscDNA synthesis, the double stranded products are end repaired, followed by addition of a single ‘A’ base and then ligation of the Illumina TruSeq adaptors. The resulting product was amplified with 15 cycles of PCR. Libraries were validated using the Agilent Technologies 2100 Bioanalyzer and Quant-iT dsDNA HS Assay (Q33120; ThermoFisher Scientific). Each library was made with a unique Index sequence and libraries were pooled for sequencing. The pool was clustered at 6.5pM on a single end read flow cell and sequenced for 50 cycles on an Illumina HiSeq 2500 to obtain ∼50&nbsp;M reads per sample. Single-end sequencing (SE50), for the <i>Tbl1xr1</i><sup><i>D370Y/WT</i></sup> GCB profiling, or paired end-sequencing (PE50x2) for the <i>IμBcl6</i>-<i>Tbl1xr1</i><sup><i>D370Y/WT</i></sup> and <i>Tbl1xr1</i><sup><i>KO/KO</i></sup> GCB profiling, were performed on an Illumina HiSeq2500 sequencer. Primary processing of sequencing images was done using Illumina’s Real Time Analysis software (RTA) as suggested by Illumina. CASAVA 1.8.2 software was used to perform image capture, base calling, demultiplexing samples and generation of raw reads and respective quality scores. Sequencing results were aligned to mm10 using STAR (<span class="dropBlock reference-citations"><a id="body-ref-sref24-2" href="#" role="doc-biblioref" data-xml-rid="bib24" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref24-2" aria-controls="sref24-2">Dobin et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref24-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">24.</div><div>Dobin, A. ∙ Davis, C.A. ∙ Schlesinger, F. ...</div><div><strong>STAR: ultrafast universal RNA-seq aligner</strong></div><div><em>Bioinformatics.</em> 2013; <strong>29</strong>:15-21</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/bts635" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-84871809302" target="_blank" aria-label="Open Scopus in new tab">Scopus (8577)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23104886/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbts635&amp;pmid=23104886" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and annotated to RefSeq using the R subread package (<span class="dropBlock reference-citations"><a id="body-ref-sref57" href="#" role="doc-biblioref" data-xml-rid="bib57" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref57" aria-controls="sref57">Liao et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref57"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">57.</div><div>Liao, Y. ∙ Smyth, G.K. ∙ Shi, W.</div><div><strong>featureCounts: an efficient general purpose program for assigning sequence reads to genomic features</strong></div><div><em>Bioinformatics.</em> 2014; <strong>30</strong>:923-930</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btt656" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-84897397058" target="_blank" aria-label="Open Scopus in new tab">Scopus (3501)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24227677/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtt656&amp;pmid=24227677" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Differentially expressed genes were identified using the EdgeR package GLM (<span class="dropBlock reference-citations"><a id="body-ref-sref88-2" href="#" role="doc-biblioref" data-xml-rid="bib88" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref88-2" aria-controls="sref88-2">Robinson et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref88-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">86.</div><div>Robinson, M.D. ∙ McCarthy, D.J. ∙ Smyth, G.K.</div><div><strong>edgeR: a Bioconductor package for differential expression analysis of digital gene expression data</strong></div><div><em>Bioinformatics.</em> 2010; <strong>26</strong>:139-140</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btp616" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_86_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-75249087100" target="_blank" aria-label="Open Scopus in new tab">Scopus (11879)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19910308/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtp616&amp;pmid=19910308" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) with thresholds of fold-change &gt; 1.5 and p&nbsp;&lt; 0.05, adjusted for multiple testing using Benjamini-Hochberg correction. Hierarchical clustering was performed using Euclidean distance of log FPKM values of genes within the top 5th percentile of standard deviation across replicates and Ward’s minimum variance. Gene set enrichment analysis was performed using the GSEA algorithm, as described in (<span class="dropBlock reference-citations"><a id="body-ref-sref102" href="#" role="doc-biblioref" data-xml-rid="bib102" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref102" aria-controls="sref102">Subramanian et&nbsp;al., 2005</a><div class="dropBlock__holder" data-db-target-of="sref102"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">100.</div><div>Subramanian, A. ∙ Tamayo, P. ∙ Mootha, V.K. ...</div><div><strong>Gene set enrichment analysis: a knowledge-based approach for interpreting genome-wide expression profiles</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2005; <strong>102</strong>:15545-15550</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.0506580102" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_100_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-27344435774" target="_blank" aria-label="Open Scopus in new tab">Scopus (16287)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16199517/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.0506580102&amp;pmid=16199517" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Pathway analysis was performed using PAGE algorithm (<span class="dropBlock reference-citations"><a id="body-ref-sref32-2" href="#" role="doc-biblioref" data-xml-rid="bib32" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref32-2" aria-controls="sref32-2">Goodarzi et&nbsp;al., 2009</a><div class="dropBlock__holder" data-db-target-of="sref32-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">32.</div><div>Goodarzi, H. ∙ Elemento, O. ∙ Tavazoie, S.</div><div><strong>Revealing global regulatory perturbations across human cancers</strong></div><div><em>Mol. Cell.</em> 2009; <strong>36</strong>:900-911</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=10.1016%2Fj.molcel.2009.11.016&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=10.1016%2Fj.molcel.2009.11.016&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-71149084382" target="_blank" aria-label="Open Scopus in new tab">Scopus (112)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20005852/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.molcel.2009.11.016&amp;pmid=20005852" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div></section><section id="sec-4-4-21"><h4>CyTOF immunophenotyping of DLBCL specimens</h4><div id="p0395" role="paragraph">Diagnostic pre-treatment LN biopsies from patients with DLBCL, or non-malignant “reactive” LN (rLN) were acquired from the lymphoma tumor bank at the BC Cancer Agency. All samples were obtained with informed consent and according to protocols approved by the BCCA Research Ethics Board, and were selected based on sufficient numbers of viable cells for CyTOF analysis (2 million per sample).</div><div id="p0400" role="paragraph">Vials of frozen cells were rapidly thawed from liquid nitrogen to 37°C using a water bath, and cell suspensions were washed with RPMI supplemented with 10% fetal bovine serum. Cells were used immediately after thawing and were incubated in cisplatin for 15&nbsp;minutes to label dead cells, then washed thoroughly in PBS+2%FCS. Cell pellets were then incubated with Fc Receptor Blocking Solution (422302; BioLegends) for 10&nbsp;minutes to inhibit non-specific binding of antibodies, before the addition of a master mix of metal-conjugated antibodies at optimal concentrations for 30&nbsp;minutes. The relevant antibodies used for the current analysis are; CD10-156Nd (HI10a; Fluidigm, South San Francisco, CA, USA), CD27-167Er (L128; Fluidigm), CD38-172Yb (HIT2; Fluidigm) and CD44-171Yb (IM7; Fluidigm).</div><div id="p0405" role="paragraph">Samples were bar-coded with the Cell-ID 20-plex Pd kit (#201060; Fluidigm) as per manufacturer’s instructions before being pooled together for analysis to ensure consistent acquisition. Batch effects were accounted for using a spike-in staining control of rLN cells. Pooled suspensions were then spiked with DVS normalization beads to account for technical drift during acquisition, and acquired on a Fluidigm CyTOF 2 instrument. After acquisition, samples were normalized based on bead signal intensity and de-multiplexed to distinguish individual samples using published tools (<a id="intref0100" href="https://github.com/nolanlab/bead-normalization" target="_blank">https://github.com/nolanlab/bead-normalization</a>, <a id="intref0105" href="https://github.com/nolanlab/single-cell-debarcoder" target="_blank">https://github.com/nolanlab/single-cell-debarcoder</a>).</div><div id="p0410" role="paragraph">Viable cells were gated in FlowJo based on negative staining for cisplatin, and all subsequent analysis was performed using R. Malignant tumor cells were identified as previously described (<span class="dropBlock reference-citations"><a id="body-ref-sref71-3" href="#" role="doc-biblioref" data-xml-rid="bib71" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref71-3" aria-controls="sref71-3">Nissen et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref71-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">72.</div><div>Nissen, M.D. ∙ Kusakabe, M. ∙ Wang, X. ...</div><div><strong>Single Cell Phenotypic Profiling of 27 DLBCL Cases Reveals Marked Intertumoral and Intratumoral Heterogeneity</strong></div><div><em>Cytometry A.</em> 2019;</div><div>Published online October 22, 2019</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/cyto.a.23919" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-85075788395" target="_blank" aria-label="Open Scopus in new tab">Scopus (2)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31637838/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fcyto.a.23919&amp;pmid=31637838" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Isolated malignant cells and manually-defined germinal center B cells and memory B cells were then pooled together for principal component analysis.</div><div id="p0415" role="paragraph">Exome sequencing on samples (<a id="crosref2555" href="#mmc1">Table S6</a>) was carried out using genomic DNA extracted from DLBCL samples using QIAGEN reagents. Exome libraries were created using IDT exome capture kits, and samples were sequenced using an Illumina HiSeq 2500 (PE_2x125). Somatic SNV/indel variants were identified using the intersection of calls predicted by VarScan (version 2.3.6) (<span class="dropBlock reference-citations"><a id="body-ref-sref48-2" href="#" role="doc-biblioref" data-xml-rid="bib48" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref48-2" aria-controls="sref48-2">Koboldt et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref48-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">48.</div><div>Koboldt, D.C. ∙ Zhang, Q. ∙ Larson, D.E. ...</div><div><strong>VarScan 2: somatic mutation and copy number alteration discovery in cancer by exome sequencing</strong></div><div><em>Genome Res.</em> 2012; <strong>22</strong>:568-576</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/gr.129684.111" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_48_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-84863229597" target="_blank" aria-label="Open Scopus in new tab">Scopus (2007)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22300766/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2Fgr.129684.111&amp;pmid=22300766" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), Strelka (version 1.0.13) (<span class="dropBlock reference-citations"><a id="body-ref-sref90-2" href="#" role="doc-biblioref" data-xml-rid="bib90" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref90-2" aria-controls="sref90-2">Saunders et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref90-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">88.</div><div>Saunders, C.T. ∙ Wong, W.S. ∙ Swamy, S. ...</div><div><strong>Strelka: accurate somatic small-variant calling from sequenced tumor-normal sample pairs</strong></div><div><em>Bioinformatics.</em> 2012; <strong>28</strong>:1811-1817</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/bts271" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-84864153492" target="_blank" aria-label="Open Scopus in new tab">Scopus (676)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22581179/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbts271&amp;pmid=22581179" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and MuTect (version 1.1.4) (<span class="dropBlock reference-citations"><a id="body-ref-sref18-2" href="#" role="doc-biblioref" data-xml-rid="bib18" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref18-2" aria-controls="sref18-2">Cibulskis et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref18-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">18.</div><div>Cibulskis, K. ∙ Lawrence, M.S. ∙ Carter, S.L. ...</div><div><strong>Sensitive detection of somatic point mutations in impure and heterogeneous cancer samples</strong></div><div><em>Nat. Biotechnol.</em> 2013; <strong>31</strong>:213-219</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nbt.2514" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-84874025843" target="_blank" aria-label="Open Scopus in new tab">Scopus (1954)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23396013/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnbt.2514&amp;pmid=23396013" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). All variants were annotated using SnpEff (version 4.2) (<span class="dropBlock reference-citations"><a id="body-ref-sref19-2" href="#" role="doc-biblioref" data-xml-rid="bib19" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref19-2" aria-controls="sref19-2">Cingolani et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref19-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">19.</div><div>Cingolani, P. ∙ Platts, A. ∙ Wang, L. ...</div><div><strong>A program for annotating and predicting the effects of single nucleotide polymorphisms, SnpEff: SNPs in the genome of Drosophila melanogaster strain w1118; iso-2; iso-3</strong></div><div><em>Fly (Austin).</em> 2012; <strong>6</strong>:80-92</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.4161/fly.19695" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-84862506964" target="_blank" aria-label="Open Scopus in new tab">Scopus (3209)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22728672/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.4161%2Ffly.19695&amp;pmid=22728672" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and filtered for effects predicted to have an impact at the protein level (nonsynonymous, stop-gained, splice site, UTR). Variants were further filtered to remove potential germline SNPs if the variant: (i) had a GMAF score &gt; 1% or (ii) was present in dbSNP (version 137) (<span class="dropBlock reference-citations"><a id="body-ref-sref95" href="#" role="doc-biblioref" data-xml-rid="bib95" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref95" aria-controls="sref95">Sherry et&nbsp;al., 2001</a><div class="dropBlock__holder" data-db-target-of="sref95"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">93.</div><div>Sherry, S.T. ∙ Ward, M.H. ∙ Kholodov, M. ...</div><div><strong>dbSNP: the NCBI database of genetic variation</strong></div><div><em>Nucleic Acids Res.</em> 2001; <strong>29</strong>:308-311</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/29.1.308" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11125122/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2F29.1.308&amp;pmid=11125122" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and not COSMIC (version 68) (<span class="dropBlock reference-citations"><a id="body-ref-sref105" href="#" role="doc-biblioref" data-xml-rid="bib105" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref105" aria-controls="sref105">Tate et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref105"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">102.</div><div>Tate, J.G. ∙ Bamford, S. ∙ Jubb, H.C. ...</div><div><strong>COSMIC: the Catalogue Of Somatic Mutations In Cancer</strong></div><div><em>Nucleic Acids Res.</em> 2019; <strong>47</strong>:D941-D947</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gky1015" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_102_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-85057093998" target="_blank" aria-label="Open Scopus in new tab">Scopus (372)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30371878/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgky1015&amp;pmid=30371878" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). All tumors were paired against a pooled control that was constructed from 10 normal samples, to account for sequencing noise.</div></section><section id="sec-4-4-22"><h4>Graphical abstract</h4><div id="p0420" role="paragraph">Cartoons used in the graphical abstract were adapted from images in the SERVIER MEDICAL ART repository (<a id="intref0110" href="https://smart.servier.com/" target="_blank">https://smart.servier.com/</a>). Use of these images falls within the terms of the Creative Commons Attribution 3.0&nbsp;Unported License (<a id="intref0115" href="https://creativecommons.org/licenses/by/3.0/" target="_blank">https://creativecommons.org/licenses/by/3.0/</a>).</div></section></section><section id="sec-4-5"><h3>Quantification and Statistical Analysis</h3><div id="p0425" role="paragraph">Statistical parameters including the exact value and definition of n, precision measures (mean ± SEM or SD), and statistical significance are reported in the Figures and Figure&nbsp;Legends. Statistical analysis was conducted using GraphPad Prism 6.00 (GraphPad Software, San Diego, CA, USA), or R statistical language scripts and packages specified under Method Details. Data was judged to be statistically significant when p&nbsp;&lt; 0.05. Asterisks in figures denote statistical significance (<sup>∗</sup>, p&nbsp;&lt; 0.05; <sup>∗∗</sup>, p&nbsp;&lt; 0.01; <sup>∗∗∗</sup>, p&nbsp;&lt; 0.001). Statistical analysis of Sequencing data is described under <a id="crosref2245" href="#sec-4-4">Method Details</a> section and in Figure&nbsp;Legends.</div></section></section><section id="acknowledgments" role="doc-acknowledgments"><h2>Acknowledgments</h2><div id="p0430" role="paragraph">We thank all members of the Melnick Lab for thoughtful discussions and suggestions, and Hao Shen for his support with mice colonies. We acknowledge the Epigenomics, Genomics, and Flow Cytometry cores (WCM); the Molecular Cytology core (MSKCC); and the Center of Comparative Medicine and Pathology (WCM/MSKCC). L.V. is a Career Development Program Fellow from the Leukemia and Lymphoma Society (5469-18). A.M.M. is supported by the National Cancer Institute&nbsp;(NCI-R35-CA220499), the&nbsp;Leukemia and Lymphoma Society (LLS-TRP-6572-19, LLS-SCOR-7012-16), the Chemotherapy Foundation, and the Follicular Lymphoma Consortium.</div><section id="sec-5"><h3>Author Contributions</h3><div id="p0435" role="paragraph">Conceptualization, L.V. and A.M.M.; Methodology, L.V., A.M.M., W.B., and Y.J.; Formal Analysis, L.V., M.T., A.Z., A.C., C.R.C., P.F., A.P.W., Y.K., O.E., and G.G.P.; Investigation, L.V., A.Z., D.K., J.M.P., L.B., T.C.L., W.B., Y.K., W.T.Y., M.D.N., and B.R.; Resources, L.M.S., J.C., C.S., D.W.S., R.D.M., and M.R.G.; Writing, L.V. and A.M.M.</div></section><section id="sec-6"><h3>Declaration of Interests</h3><div id="p0440" role="paragraph">A.M.M. receives research funding for Janssen, is on the scientific board of KDAC Pharmaceuticals, and has consulted for Constellation and Epizyme. M.R.G. consults for Verastem Oncology.</div></section></section><section id="supplementary-material" data-type="supplementary-materials"><h2>Supplemental Information (6)</h2><div id="p0450" role="paragraph"><div class="core-supplementary-material"><div class="core-link"><button class="downloadAllSupplementary"><span>Download all</span></button></div></div><div id="mmc1" class="core-supplementary-material" data-category="pdf"><div class="core-link"><a href="/cms/10.1016/j.cell.2020.05.049/attachment/2ca051f9-bb14-45f0-b9ae-ff51acdc11f6/mmc1.pdf" download="mmc1.pdf" aria-labelledby="mmc1-heading mmc1-link"><span id="mmc1-link">PDF (125.21 KB)</span></a></div><div class="core-description"><div id="mmc1-heading" class="heading">Document S1. Tables S1 and S6</div></div></div><div id="mmc2" class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="/cms/10.1016/j.cell.2020.05.049/attachment/993d00e8-66e8-4f77-bb5d-254a8fac8ad5/mmc2.xlsx" download="mmc2.xlsx" aria-labelledby="mmc2-heading mmc2-link"><span id="mmc2-link">Spreadsheet (52.00 KB)</span></a></div><div class="core-description"><div id="mmc2-heading" class="heading">Table S2. Differentially Expressed Genes in Tbl1xr1-D370Y/WT GCB (|log2FC| ≥ 1.5 and FDR &lt;0.05), Related to Figure&nbsp;3</div><div id="espara0010" role="paragraph">Significantly enriched pathways for the Tbl1xr1-D370Y/WT GCB transcriptional signature.</div></div></div><div id="mmc3" class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="/cms/10.1016/j.cell.2020.05.049/attachment/09e04835-87bb-40ca-8a99-236e091fcee6/mmc3.xlsx" download="mmc3.xlsx" aria-labelledby="mmc3-heading mmc3-link"><span id="mmc3-link">Spreadsheet (233.96 KB)</span></a></div><div class="core-description"><div id="mmc3-heading" class="heading">Table S3. Spectral Counts for BCL6 RIME Assay in the OCI-Ly1 Cell Line, Related to Figure&nbsp;5</div><div id="espara0015" role="paragraph">Spectral counts for TBL1XR1 BioID assays in OCI-Ly1 and U2932 cell lines.</div></div></div><div id="mmc4" class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="/cms/10.1016/j.cell.2020.05.049/attachment/60deed22-98da-40e2-b747-3479d2eb6fa6/mmc4.xlsx" download="mmc4.xlsx" aria-labelledby="mmc4-heading mmc4-link"><span id="mmc4-link">Spreadsheet (69.00 KB)</span></a></div><div class="core-description"><div id="mmc4-heading" class="heading">Table S4. Differentially Expressed Genes in IμBcl6-Tbl1xr1-D370Y/WT GCB (|log2FC| ≥ 1.5 and FDR&nbsp;&lt; 0.05) Related to Figure&nbsp;5</div><div id="espara0020" role="paragraph">Differentially expressed genes in Tbl1xr1-KO/KO GCB (|log2FC| ≥ 1.5 and FDR&nbsp;&lt; 0.05).</div></div></div><div id="mmc5" class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="/cms/10.1016/j.cell.2020.05.049/attachment/21da09b6-4850-4a46-8693-590f5d9f47c1/mmc5.xlsx" download="mmc5.xlsx" aria-labelledby="mmc5-heading mmc5-link"><span id="mmc5-link">Spreadsheet (16.61 KB)</span></a></div><div class="core-description"><div id="mmc5-heading" class="heading">Table S5. Somatic Mutation Profiles of DLBCL Specimens Used for CyTOF Immunophenotyping, Related to Figure&nbsp;7</div></div></div><div id="mmc6" class="core-supplementary-material" data-category="pdf"><div class="core-link"><a href="/cms/10.1016/j.cell.2020.05.049/attachment/5bee300c-2043-47e9-ba5d-e6629b0b13e1/mmc6.pdf" download="mmc6.pdf" aria-labelledby="mmc6-heading mmc6-link"><span id="mmc6-link">PDF (2.60 MB)</span></a></div><div class="core-description"><div id="mmc6-heading" class="heading">Data S1. Complete FC Gating Strategies and Additional Data, Related to Figures 1, 2, 3, 4, 5, 6, and 7</div></div></div></div></section></div></section><div class="core-linked-content">
    

        

</div><section id="backmatter" data-extent="backmatter"><div class="core-container"><section id="references"><h2>References</h2><div id="bibliography" role="doc-bibliography"><div role="list"><div role="listitem" data-has="label"><div id="bib1" class="citations"><div id="sref1" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref1" title="View in article">1.</a></div><div>Agematsu, K. ∙ Hokibara, S. ∙ Nagumo, H. ...</div><div><strong>CD27: a memory B-cell marker</strong></div><div><em>Immunol. Today.</em> 2000; <strong>21</strong>:204-206</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_1_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=10.1016%2FS0167-5699%2800%2901605-4&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_1_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=10.1016%2FS0167-5699%2800%2901605-4&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_1_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-0034102137" target="_blank" aria-label="Open Scopus in new tab">Scopus (288)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/10782048/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0167-5699%2800%2901605-4&amp;pmid=10782048" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib2" class="citations"><div id="sref2" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref2-1" title="View in article">2.</a></div><div>Anderson, S.M. ∙ Tomayko, M.M. ∙ Ahuja, A. ...</div><div><strong>New markers for murine memory B cells that define mutated and unmutated subsets</strong></div><div><em>J.&nbsp;Exp. Med.</em> 2007; <strong>204</strong>:2103-2114</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1084/jem.20062571" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_2_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-34548383012" target="_blank" aria-label="Open Scopus in new tab">Scopus (164)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17698588/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1084%2Fjem.20062571&amp;pmid=17698588" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib3" class="citations"><div id="sref3" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref3" title="View in article">3.</a></div><div>Arnon, T.I. ∙ Horton, R.M. ∙ Grigorova, I.L. ...</div><div><strong>Visualization of splenic marginal zone B-cell shuttling and follicular B-cell egress</strong></div><div><em>Nature.</em> 2013; <strong>493</strong>:684-688</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature11738" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-84873088550" target="_blank" aria-label="Open Scopus in new tab">Scopus (126)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23263181/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature11738&amp;pmid=23263181" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib4" class="citations"><div id="sref4" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref4" title="View in article">4.</a></div><div>Arpin, C. ∙ Déchanet, J. ∙ Van Kooten, C. ...</div><div><strong>Generation of memory B cells and plasma cells in&nbsp;vitro</strong></div><div><em>Science.</em> 1995; <strong>268</strong>:720-722</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.7537388" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/7537388/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.7537388&amp;pmid=7537388" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib5" class="citations"><div id="sref5" class="citation"><div class="citation-content"><div class="label"><a href="#fig1" title="View in article">5.</a></div><div>Arthur, S.E. ∙ Jiang, A. ∙ Grande, B.M. ...</div><div><strong>Genome-wide discovery of somatic regulatory variants in diffuse large B-cell lymphoma</strong></div><div><em>Nat. Commun.</em> 2018; <strong>9</strong>:4001</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41467-018-06354-3" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-85054070552" target="_blank" aria-label="Open Scopus in new tab">Scopus (32)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30275490/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41467-018-06354-3&amp;pmid=30275490" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib6" class="citations"><div id="sref6" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref6" title="View in article">6.</a></div><div>Barnden, M.J. ∙ Allison, J. ∙ Heath, W.R. ...</div><div><strong>Defective TCR expression in transgenic mice constructed using cDNA-based alpha- and beta-chain genes under the control of heterologous regulatory elements</strong></div><div><em>Immunol. Cell Biol.</em> 1998; <strong>76</strong>:34-40</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1046/j.1440-1711.1998.00709.x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-0031897801" target="_blank" aria-label="Open Scopus in new tab">Scopus (1041)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/9553774/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1046%2Fj.1440-1711.1998.00709.x&amp;pmid=9553774" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib7" class="citations"><div id="sref7" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref7" title="View in article">7.</a></div><div>Baumjohann, D. ∙ Preite, S. ∙ Reboldi, A. ...</div><div><strong>Persistent antigen and germinal center B cells sustain T follicular helper cell responses and phenotype</strong></div><div><em>Immunity.</em> 2013; <strong>38</strong>:596-605</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=10.1016%2Fj.immuni.2012.11.020&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=10.1016%2Fj.immuni.2012.11.020&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-84875549975" target="_blank" aria-label="Open Scopus in new tab">Scopus (246)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23499493/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.immuni.2012.11.020&amp;pmid=23499493" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib8" class="citations"><div id="sref8" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref8" title="View in article">8.</a></div><div>Béguelin, W. ∙ Popovic, R. ∙ Teater, M. ...</div><div><strong>EZH2 is required for germinal center formation and somatic EZH2 mutations promote lymphoid transformation</strong></div><div><em>Cancer Cell.</em> 2013; <strong>23</strong>:677-692</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=10.1016%2Fj.ccr.2013.04.011&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=10.1016%2Fj.ccr.2013.04.011&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-84877815031" target="_blank" aria-label="Open Scopus in new tab">Scopus (409)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23680150/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.ccr.2013.04.011&amp;pmid=23680150" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib9" class="citations"><div id="sref9" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref9-1" title="View in article">9.</a></div><div>Béguelin, W. ∙ Rivas, M.A. ∙ Calvo Fernández, M.T. ...</div><div><strong>EZH2 enables germinal centre formation through epigenetic silencing of CDKN1A and an Rb-E2F1 feedback loop</strong></div><div><em>Nat. Commun.</em> 2017; <strong>8</strong>:877</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41467-017-01029-x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-85031291005" target="_blank" aria-label="Open Scopus in new tab">Scopus (48)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29026085/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41467-017-01029-x&amp;pmid=29026085" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib10" class="citations"><div id="sref10" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref10-1" title="View in article">10.</a></div><div>Beham-Schmid, C.</div><div><strong>Aggressive lymphoma 2016: revision of the WHO classification</strong></div><div><em>Memo.</em> 2017; <strong>10</strong>:248-254</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/s12254-017-0367-8" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-85035795436" target="_blank" aria-label="Open Scopus in new tab">Scopus (9)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29250206/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2Fs12254-017-0367-8&amp;pmid=29250206" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib11" class="citations"><div id="sref11" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref11" title="View in article">11.</a></div><div>Cardenas, M.G. ∙ Yu, W. ∙ Beguelin, W. ...</div><div><strong>Rationally designed BCL6 inhibitors target activated B cell diffuse large B cell lymphoma</strong></div><div><em>J.&nbsp;Clin. Invest.</em> 2016; <strong>126</strong>:3351-3362</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1172/JCI85795" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-84987788606" target="_blank" aria-label="Open Scopus in new tab">Scopus (65)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27482887/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1172%2FJCI85795&amp;pmid=27482887" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib12" class="citations"><div id="sref12" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref12" title="View in article">12.</a></div><div>Casola, S. ∙ Cattoretti, G. ∙ Uyttersprot, N. ...</div><div><strong>Tracking germinal center B cells expressing germ-line immunoglobulin gamma1 transcripts by conditional gene targeting</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2006; <strong>103</strong>:7396-7401</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.0602353103" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-33646555754" target="_blank" aria-label="Open Scopus in new tab">Scopus (114)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16651521/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.0602353103&amp;pmid=16651521" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib13" class="citations"><div id="sref13" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref13-1" title="View in article">13.</a></div><div>Cattoretti, G. ∙ Pasqualucci, L. ∙ Ballon, G. ...</div><div><strong>Deregulated BCL6 expression recapitulates the pathogenesis of human diffuse large B cell lymphomas in mice</strong></div><div><em>Cancer Cell.</em> 2005; <strong>7</strong>:445-455</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_13_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=10.1016%2Fj.ccr.2005.03.037&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_13_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=10.1016%2Fj.ccr.2005.03.037&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_13_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-19344368745" target="_blank" aria-label="Open Scopus in new tab">Scopus (258)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15894265/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.ccr.2005.03.037&amp;pmid=15894265" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib14" class="citations"><div id="sref14" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref14" title="View in article">14.</a></div><div>Cerchietti, L.C. ∙ Ghetu, A.F. ∙ Zhu, X. ...</div><div><strong>A small-molecule inhibitor of BCL6 kills DLBCL cells in&nbsp;vitro and in&nbsp;vivo</strong></div><div><em>Cancer Cell.</em> 2010; <strong>17</strong>:400-411</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=10.1016%2Fj.ccr.2009.12.050&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=10.1016%2Fj.ccr.2009.12.050&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-77950486830" target="_blank" aria-label="Open Scopus in new tab">Scopus (205)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20385364/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.ccr.2009.12.050&amp;pmid=20385364" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib15" class="citations"><div id="sref15" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref15" title="View in article">15.</a></div><div>Chapuy, B. ∙ Roemer, M.G. ∙ Stewart, C. ...</div><div><strong>Targetable genetic features of primary testicular and primary central nervous system lymphomas</strong></div><div><em>Blood.</em> 2016; <strong>127</strong>:869-881</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1182/blood-2015-10-673236" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-84959419747" target="_blank" aria-label="Open Scopus in new tab">Scopus (207)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26702065/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1182%2Fblood-2015-10-673236&amp;pmid=26702065" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib16" class="citations"><div id="sref16" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref16-1" title="View in article">16.</a></div><div>Chapuy, B. ∙ Stewart, C. ∙ Dunford, A.J. ...</div><div><strong>Molecular subtypes of diffuse large B cell lymphoma are associated with distinct pathogenic mechanisms and outcomes</strong></div><div><em>Nat. Med.</em> 2018; <strong>24</strong>:679-690</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41591-018-0016-8" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-85046087353" target="_blank" aria-label="Open Scopus in new tab">Scopus (257)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29713087/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41591-018-0016-8&amp;pmid=29713087" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib17" class="citations"><div id="sref17" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref17-1" title="View in article">17.</a></div><div>Choi, H. ∙ Larsen, B. ∙ Lin, Z.Y. ...</div><div><strong>SAINT: probabilistic scoring of affinity purification-mass spectrometry data</strong></div><div><em>Nat. Methods.</em> 2011; <strong>8</strong>:70-73</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.1541" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-78650861777" target="_blank" aria-label="Open Scopus in new tab">Scopus (359)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21131968/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.1541&amp;pmid=21131968" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib18" class="citations"><div id="sref18" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref18-1" title="View in article">18.</a></div><div>Cibulskis, K. ∙ Lawrence, M.S. ∙ Carter, S.L. ...</div><div><strong>Sensitive detection of somatic point mutations in impure and heterogeneous cancer samples</strong></div><div><em>Nat. Biotechnol.</em> 2013; <strong>31</strong>:213-219</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nbt.2514" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-84874025843" target="_blank" aria-label="Open Scopus in new tab">Scopus (1954)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23396013/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnbt.2514&amp;pmid=23396013" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib19" class="citations"><div id="sref19" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref19-1" title="View in article">19.</a></div><div>Cingolani, P. ∙ Platts, A. ∙ Wang, L. ...</div><div><strong>A program for annotating and predicting the effects of single nucleotide polymorphisms, SnpEff: SNPs in the genome of Drosophila melanogaster strain w1118; iso-2; iso-3</strong></div><div><em>Fly (Austin).</em> 2012; <strong>6</strong>:80-92</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.4161/fly.19695" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-84862506964" target="_blank" aria-label="Open Scopus in new tab">Scopus (3209)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22728672/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.4161%2Ffly.19695&amp;pmid=22728672" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib20" class="citations"><div id="sref20" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref20-1" title="View in article">20.</a></div><div>Cock, P.J. ∙ Antao, T. ∙ Chang, J.T. ...</div><div><strong>Biopython: freely available Python tools for computational molecular biology and bioinformatics</strong></div><div><em>Bioinformatics.</em> 2009; <strong>25</strong>:1422-1423</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btp163" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-65649092976" target="_blank" aria-label="Open Scopus in new tab">Scopus (1419)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19304878/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtp163&amp;pmid=19304878" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib21" class="citations"><div id="sref21" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref21-1" title="View in article">21.</a></div><div>Craig, R. ∙ Beavis, R.C.</div><div><strong>TANDEM: matching proteins with tandem mass spectra</strong></div><div><em>Bioinformatics.</em> 2004; <strong>20</strong>:1466-1467</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/bth092" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-3142702204" target="_blank" aria-label="Open Scopus in new tab">Scopus (1780)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/14976030/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbth092&amp;pmid=14976030" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib22" class="citations"><div id="sref22" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref22" title="View in article">22.</a></div><div>Cyster, J.G. ∙ Allen, C.D.C.</div><div><strong>B Cell Responses: Cell Interaction Dynamics and Decisions</strong></div><div><em>Cell.</em> 2019; <strong>177</strong>:524-540</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=10.1016%2Fj.cell.2019.03.016&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=10.1016%2Fj.cell.2019.03.016&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-85064249878" target="_blank" aria-label="Open Scopus in new tab">Scopus (43)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31002794/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2019.03.016&amp;pmid=31002794" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib23" class="citations"><div id="sref23" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref23" title="View in article">23.</a></div><div>Diehl, S.A. ∙ Schmidlin, H. ∙ Nagasawa, M. ...</div><div><strong>STAT3-mediated up-regulation of BLIMP1 Is coordinated with BCL6 down-regulation to control human plasma cell differentiation</strong></div><div><em>J.&nbsp;Immunol.</em> 2008; <strong>180</strong>:4805-4815</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.4049/jimmunol.180.7.4805" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/18354204/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.4049%2Fjimmunol.180.7.4805&amp;pmid=18354204" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib24" class="citations"><div id="sref24" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref24-1" title="View in article">24.</a></div><div>Dobin, A. ∙ Davis, C.A. ∙ Schlesinger, F. ...</div><div><strong>STAR: ultrafast universal RNA-seq aligner</strong></div><div><em>Bioinformatics.</em> 2013; <strong>29</strong>:15-21</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/bts635" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-84871809302" target="_blank" aria-label="Open Scopus in new tab">Scopus (8577)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23104886/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbts635&amp;pmid=23104886" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib25" class="citations"><div id="sref25" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref25" title="View in article">25.</a></div><div>Dogan, I. ∙ Bertocci, B. ∙ Vilmont, V. ...</div><div><strong>Multiple layers of B cell memory with different effector functions</strong></div><div><em>Nat. Immunol.</em> 2009; <strong>10</strong>:1292-1299</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ni.1814" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-70449722777" target="_blank" aria-label="Open Scopus in new tab">Scopus (355)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19855380/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fni.1814&amp;pmid=19855380" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib26" class="citations"><div id="sref26" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref26" title="View in article">26.</a></div><div>Dominguez, P.M. ∙ Ghamlouch, H. ∙ Rosikiewicz, W. ...</div><div><strong>TET2 Deficiency Causes Germinal Center Hyperplasia, Impairs Plasma Cell Differentiation, and Promotes B-cell Lymphomagenesis</strong></div><div><em>Cancer Discov.</em> 2018; <strong>8</strong>:1632-1653</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1158/2159-8290.CD-18-0657" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_26_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-85058063303" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30274972/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1158%2F2159-8290.CD-18-0657&amp;pmid=30274972" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib27" class="citations"><div id="sref27" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref27" title="View in article">27.</a></div><div>Egle, A. ∙ Harris, A.W. ∙ Bath, M.L. ...</div><div><strong>VavP-Bcl2 transgenic mice develop follicular lymphoma preceded by germinal center hyperplasia</strong></div><div><em>Blood.</em> 2004; <strong>103</strong>:2276-2283</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1182/blood-2003-07-2469" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-1542283724" target="_blank" aria-label="Open Scopus in new tab">Scopus (137)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/14630790/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1182%2Fblood-2003-07-2469&amp;pmid=14630790" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib28" class="citations"><div id="sref28" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref28-1" title="View in article">28.</a></div><div>Ennishi, D. ∙ Mottok, A. ∙ Ben-Neriah, S. ...</div><div><strong>Genetic profiling of <i>MYC</i> and <i>BCL2</i> in diffuse large B-cell lymphoma determines cell-of-origin-specific clinical impact</strong></div><div><em>Blood.</em> 2017; <strong>129</strong>:2760-2770</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1182/blood-2016-11-747022" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-85019688331" target="_blank" aria-label="Open Scopus in new tab">Scopus (53)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28351934/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1182%2Fblood-2016-11-747022&amp;pmid=28351934" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib29" class="citations"><div id="sref29" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref29" title="View in article">29.</a></div><div>Gavrieli, Y. ∙ Sherman, Y. ∙ Ben-Sasson, S.A.</div><div><strong>Identification of programmed cell death in&nbsp;situ via specific labeling of nuclear DNA fragmentation</strong></div><div><em>J.&nbsp;Cell Biol.</em> 1992; <strong>119</strong>:493-501</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1083/jcb.119.3.493" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-0026648674" target="_blank" aria-label="Open Scopus in new tab">Scopus (8883)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/1400587/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1083%2Fjcb.119.3.493&amp;pmid=1400587" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib30" class="citations"><div id="sref30" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref30" title="View in article">30.</a></div><div>Geng, J. ∙ Wei, H. ∙ Shi, B. ...</div><div><strong>Bach2 Negatively Regulates T Follicular Helper Cell Differentiation and Is Critical for CD4<sup>+</sup> T Cell Memory</strong></div><div><em>J.&nbsp;Immunol.</em> 2019; <strong>202</strong>:2991-2998</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.4049/jimmunol.1801626" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-85065678132" target="_blank" aria-label="Open Scopus in new tab">Scopus (5)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30971440/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.4049%2Fjimmunol.1801626&amp;pmid=30971440" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib31" class="citations"><div id="sref31" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref31" title="View in article">31.</a></div><div>Gonzalez-Aguilar, A. ∙ Idbaih, A. ∙ Boisselier, B. ...</div><div><strong>Recurrent mutations of MYD88 and TBL1XR1 in primary central nervous system lymphomas</strong></div><div><em>Clin. Cancer Res.</em> 2012; <strong>18</strong>:5203-5211</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1158/1078-0432.CCR-12-0845" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-84866895485" target="_blank" aria-label="Open Scopus in new tab">Scopus (145)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22837180/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1158%2F1078-0432.CCR-12-0845&amp;pmid=22837180" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib32" class="citations"><div id="sref32" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref32-1" title="View in article">32.</a></div><div>Goodarzi, H. ∙ Elemento, O. ∙ Tavazoie, S.</div><div><strong>Revealing global regulatory perturbations across human cancers</strong></div><div><em>Mol. Cell.</em> 2009; <strong>36</strong>:900-911</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=10.1016%2Fj.molcel.2009.11.016&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=10.1016%2Fj.molcel.2009.11.016&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-71149084382" target="_blank" aria-label="Open Scopus in new tab">Scopus (112)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20005852/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.molcel.2009.11.016&amp;pmid=20005852" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib33" class="citations"><div id="sref33" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref33" title="View in article">33.</a></div><div>Hashwah, H. ∙ Schmid, C.A. ∙ Kasser, S. ...</div><div><strong>Inactivation of CREBBP expands the germinal center B cell compartment, down-regulates MHCII expression and promotes DLBCL growth</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2017; <strong>114</strong>:9701-9706</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1619555114" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-85029224086" target="_blank" aria-label="Open Scopus in new tab">Scopus (32)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28831000/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1619555114&amp;pmid=28831000" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib34" class="citations"><div id="sref34" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref34-1" title="View in article">34.</a></div><div>Hatzi, K. ∙ Melnick, A.</div><div><strong>Breaking bad in the germinal center: how deregulation of BCL6 contributes to lymphomagenesis</strong></div><div><em>Trends Mol. Med.</em> 2014; <strong>20</strong>:343-352</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=10.1016%2Fj.molmed.2014.03.001&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=10.1016%2Fj.molmed.2014.03.001&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-84901585384" target="_blank" aria-label="Open Scopus in new tab">Scopus (80)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24698494/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.molmed.2014.03.001&amp;pmid=24698494" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib35" class="citations"><div id="sref35" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref35-1" title="View in article">35.</a></div><div>Hatzi, K. ∙ Jiang, Y. ∙ Huang, C. ...</div><div><strong>A hybrid mechanism of action for BCL6 in B cells defined by formation of functionally distinct complexes at enhancers and promoters</strong></div><div><em>Cell Rep.</em> 2013; <strong>4</strong>:578-588</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=10.1016%2Fj.celrep.2013.06.016&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=10.1016%2Fj.celrep.2013.06.016&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-84881615483" target="_blank" aria-label="Open Scopus in new tab">Scopus (90)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23911289/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.celrep.2013.06.016&amp;pmid=23911289" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib36" class="citations"><div id="sref36" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref36-1" title="View in article">36.</a></div><div>Hatzi, K. ∙ Geng, H. ∙ Doane, A.S. ...</div><div><strong>Histone demethylase LSD1 is required for germinal center formation and BCL6-driven lymphomagenesis</strong></div><div><em>Nat. Immunol.</em> 2019; <strong>20</strong>:86-96</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41590-018-0273-1" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-85058423458" target="_blank" aria-label="Open Scopus in new tab">Scopus (14)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30538335/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41590-018-0273-1&amp;pmid=30538335" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib37" class="citations"><div id="sref37" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref37-1" title="View in article">37.</a></div><div>Heinen, C.A. ∙ Jongejan, A. ∙ Watson, P.J. ...</div><div><strong>A specific mutation in TBL1XR1 causes Pierpont syndrome</strong></div><div><em>J.&nbsp;Med. Genet.</em> 2016; <strong>53</strong>:330-337</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1136/jmedgenet-2015-103233" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-84957921471" target="_blank" aria-label="Open Scopus in new tab">Scopus (32)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26769062/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1136%2Fjmedgenet-2015-103233&amp;pmid=26769062" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib38" class="citations"><div id="sref38" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref38-1" title="View in article">38.</a></div><div>Hill, J.T. ∙ Demarest, B.L. ∙ Bisgrove, B.W. ...</div><div><strong>Poly peak parser: Method and software for identification of unknown indels using sanger sequencing of polymerase chain reaction products</strong></div><div><em>Dev. Dyn.</em> 2014; <strong>243</strong>:1632-1636</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/dvdy.24183" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_38_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-84924246056" target="_blank" aria-label="Open Scopus in new tab">Scopus (79)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25160973/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fdvdy.24183&amp;pmid=25160973" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib39" class="citations"><div id="sref39" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref39" title="View in article">39.</a></div><div>Huang, W. ∙ Ghisletti, S. ∙ Perissi, V. ...</div><div><strong>Transcriptional integration of TLR2 and TLR4 signaling at the NCoR derepression checkpoint</strong></div><div><em>Mol. Cell.</em> 2009; <strong>35</strong>:48-57</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=10.1016%2Fj.molcel.2009.05.023&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=10.1016%2Fj.molcel.2009.05.023&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-67649668789" target="_blank" aria-label="Open Scopus in new tab">Scopus (80)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19595715/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.molcel.2009.05.023&amp;pmid=19595715" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib40" class="citations"><div id="sref40" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref40" title="View in article">40.</a></div><div>Huang, C. ∙ Geng, H. ∙ Boss, I. ...</div><div><strong>Cooperative transcriptional repression by BCL6 and BACH2 in germinal center B-cell differentiation</strong></div><div><em>Blood.</em> 2014; <strong>123</strong>:1012-1020</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1182/blood-2013-07-518605" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_40_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-84897870661" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24277074/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1182%2Fblood-2013-07-518605&amp;pmid=24277074" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib41" class="citations"><div id="sref41" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref41" title="View in article">41.</a></div><div>Ise, W. ∙ Fujii, K. ∙ Shiroguchi, K. ...</div><div><strong>T Follicular Helper Cell-Germinal Center B Cell Interaction Strength Regulates Entry into Plasma Cell or Recycling Germinal Center Cell Fate</strong></div><div><em>Immunity.</em> 2018; <strong>48</strong>:702-715</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=10.1016%2Fj.immuni.2018.03.027&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=10.1016%2Fj.immuni.2018.03.027&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-85045259028" target="_blank" aria-label="Open Scopus in new tab">Scopus (55)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29669250/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.immuni.2018.03.027&amp;pmid=29669250" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib42" class="citations"><div id="sref42" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref42" title="View in article">42.</a></div><div>Jain, B.P. ∙ Pandey, S.</div><div><strong>WD40 Repeat Proteins: Signalling Scaffold with Diverse Functions</strong></div><div><em>Protein J.</em> 2018; <strong>37</strong>:391-406</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/s10930-018-9785-7" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-85051176958" target="_blank" aria-label="Open Scopus in new tab">Scopus (32)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30069656/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2Fs10930-018-9785-7&amp;pmid=30069656" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib43" class="citations"><div id="sref43" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref43-1" title="View in article">43.</a></div><div>Jiang, Y. ∙ Ortega-Molina, A. ∙ Geng, H. ...</div><div><strong>CREBBP Inactivation Promotes the Development of HDAC3-Dependent Lymphomas</strong></div><div><em>Cancer Discov.</em> 2017; <strong>7</strong>:38-53</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1158/2159-8290.CD-16-0975" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-85009410568" target="_blank" aria-label="Open Scopus in new tab">Scopus (92)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27733359/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1158%2F2159-8290.CD-16-0975&amp;pmid=27733359" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib44" class="citations"><div id="sref44" class="citation"><div class="citation-content"><div class="label"><a href="#figs6" title="View in article">44.</a></div><div>Jolly, C.J. ∙ Klix, N. ∙ Neuberger, M.S.</div><div><strong>Rapid methods for the analysis of immunoglobulin gene hypermutation: application to transgenic and gene targeted mice</strong></div><div><em>Nucleic Acids Res.</em> 1997; <strong>25</strong>:1913-1919</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/25.10.1913" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-0030845386" target="_blank" aria-label="Open Scopus in new tab">Scopus (106)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/9115357/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2F25.10.1913&amp;pmid=9115357" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib45" class="citations"><div id="sref45" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref45" title="View in article">45.</a></div><div>Kashyap, R. ∙ Rai Mittal, B. ∙ Manohar, K. ...</div><div><strong>Extranodal manifestations of lymphoma on [<sup>18</sup>F]FDG-PET/CT: a pictorial essay</strong></div><div><em>Cancer Imaging.</em> 2011; <strong>11</strong>:166-174</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1102/1470-7330.2011.0023" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_45_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-82655172686" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22123338/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1102%2F1470-7330.2011.0023&amp;pmid=22123338" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib46" class="citations"><div id="sref46" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref46-1" title="View in article">46.</a></div><div>Kessner, D. ∙ Chambers, M. ∙ Burke, R. ...</div><div><strong>ProteoWizard: open source software for rapid proteomics tools development</strong></div><div><em>Bioinformatics.</em> 2008; <strong>24</strong>:2534-2536</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btn323" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-54949129419" target="_blank" aria-label="Open Scopus in new tab">Scopus (891)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/18606607/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtn323&amp;pmid=18606607" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib47" class="citations"><div id="sref47" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref47" title="View in article">47.</a></div><div>Kitamura, D. ∙ Roes, J. ∙ Kühn, R. ...</div><div><strong>A B cell-deficient mouse by targeted disruption of the membrane exon of the immunoglobulin mu chain gene</strong></div><div><em>Nature.</em> 1991; <strong>350</strong>:423-426</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/350423a0" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-0025852445" target="_blank" aria-label="Open Scopus in new tab">Scopus (1459)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/1901381/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2F350423a0&amp;pmid=1901381" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib48" class="citations"><div id="sref48" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref48-1" title="View in article">48.</a></div><div>Koboldt, D.C. ∙ Zhang, Q. ∙ Larson, D.E. ...</div><div><strong>VarScan 2: somatic mutation and copy number alteration discovery in cancer by exome sequencing</strong></div><div><em>Genome Res.</em> 2012; <strong>22</strong>:568-576</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/gr.129684.111" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_48_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-84863229597" target="_blank" aria-label="Open Scopus in new tab">Scopus (2007)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22300766/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2Fgr.129684.111&amp;pmid=22300766" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib49" class="citations"><div id="sref49" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref49" title="View in article">49.</a></div><div>Kometani, K. ∙ Nakagawa, R. ∙ Shinnakasu, R. ...</div><div><strong>Repression of the transcription factor Bach2 contributes to predisposition of IgG1 memory B cells toward plasma cell differentiation</strong></div><div><em>Immunity.</em> 2013; <strong>39</strong>:136-147</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=10.1016%2Fj.immuni.2013.06.011&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=10.1016%2Fj.immuni.2013.06.011&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-84880752273" target="_blank" aria-label="Open Scopus in new tab">Scopus (113)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23850379/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.immuni.2013.06.011&amp;pmid=23850379" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib50" class="citations"><div id="sref50" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref50-1" title="View in article">50.</a></div><div>Kowarz, E. ∙ Löscher, D. ∙ Marschalek, R.</div><div><strong>Optimized Sleeping Beauty transposons rapidly generate stable transgenic cell lines</strong></div><div><em>Biotechnol. J.</em> 2015; <strong>10</strong>:647-653</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/biot.201400821" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25650551/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fbiot.201400821&amp;pmid=25650551" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib51" class="citations"><div id="sref51" class="citation"><div class="citation-content"><div class="label"><a href="#fig1" title="View in article">51.</a></div><div>Krysiak, K. ∙ Gomez, F. ∙ White, B.S. ...</div><div><strong>Recurrent somatic mutations affecting B-cell receptor signaling pathway genes in follicular lymphoma</strong></div><div><em>Blood.</em> 2017; <strong>129</strong>:473-483</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1182/blood-2016-07-729954" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_51_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-85014960882" target="_blank" aria-label="Open Scopus in new tab">Scopus (58)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28064239/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1182%2Fblood-2016-07-729954&amp;pmid=28064239" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib52" class="citations"><div id="sref52" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref52" title="View in article">52.</a></div><div>Kuo, T.C. ∙ Shaffer, A.L. ∙ Haddad, Jr., J. ...</div><div><strong>Repression of BCL-6 is required for the formation of human memory B cells in&nbsp;vitro</strong></div><div><em>J.&nbsp;Exp. Med.</em> 2007; <strong>204</strong>:819-830</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1084/jem.20062104" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-34247387876" target="_blank" aria-label="Open Scopus in new tab">Scopus (61)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17403935/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1084%2Fjem.20062104&amp;pmid=17403935" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib53" class="citations"><div id="sref53" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref53-1" title="View in article">53.</a></div><div>Laidlaw, B.J. ∙ Schmidt, T.H. ∙ Green, J.A. ...</div><div><strong>The Eph-related tyrosine kinase ligand Ephrin-B1 marks germinal center and memory precursor B cells</strong></div><div><em>J.&nbsp;Exp. Med.</em> 2017; <strong>214</strong>:639-649</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1084/jem.20161461" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-85016232208" target="_blank" aria-label="Open Scopus in new tab">Scopus (36)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28143955/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1084%2Fjem.20161461&amp;pmid=28143955" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib54" class="citations"><div id="sref54" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref54-1" title="View in article">54.</a></div><div>Laskowski, R.A. ∙ Tyagi, N. ∙ Johnson, D. ..., DDD Study</div><div><strong>Integrating population variation and protein structural analysis to improve clinical interpretation of missense variation: application to the WD40 domain</strong></div><div><em>Hum. Mol. Genet.</em> 2016; <strong>25</strong>:927-935</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/hmg/ddv625" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-84962815246" target="_blank" aria-label="Open Scopus in new tab">Scopus (12)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26740553/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fhmg%2Fddv625&amp;pmid=26740553" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib55" class="citations"><div id="sref55" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref55" title="View in article">55.</a></div><div>Lenz, G. ∙ Nagel, I. ∙ Siebert, R. ...</div><div><strong>Aberrant immunoglobulin class switch recombination and switch translocations in activated B cell-like diffuse large B cell lymphoma</strong></div><div><em>J.&nbsp;Exp. Med.</em> 2007; <strong>204</strong>:633-643</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1084/jem.20062041" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-33947403981" target="_blank" aria-label="Open Scopus in new tab">Scopus (148)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17353367/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1084%2Fjem.20062041&amp;pmid=17353367" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib56" class="citations"><div id="sref56" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref56" title="View in article">56.</a></div><div>Li, J. ∙ Wang, C.Y.</div><div><strong>TBL1-TBLR1 and beta-catenin recruit each other to Wnt target-gene promoter for transcription activation and oncogenesis</strong></div><div><em>Nat. Cell Biol.</em> 2008; <strong>10</strong>:160-169</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ncb1684" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_56_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-38849131230" target="_blank" aria-label="Open Scopus in new tab">Scopus (108)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/18193033/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fncb1684&amp;pmid=18193033" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib57" class="citations"><div id="sref57" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref57" title="View in article">57.</a></div><div>Liao, Y. ∙ Smyth, G.K. ∙ Shi, W.</div><div><strong>featureCounts: an efficient general purpose program for assigning sequence reads to genomic features</strong></div><div><em>Bioinformatics.</em> 2014; <strong>30</strong>:923-930</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btt656" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-84897397058" target="_blank" aria-label="Open Scopus in new tab">Scopus (3501)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24227677/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtt656&amp;pmid=24227677" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib58" class="citations"><div id="sref58" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref58-1" title="View in article">58.</a></div><div>Liu, M. ∙ Duke, J.L. ∙ Richter, D.J. ...</div><div><strong>Two levels of protection for the B cell genome during somatic hypermutation</strong></div><div><em>Nature.</em> 2008; <strong>451</strong>:841-845</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature06547" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-39149099085" target="_blank" aria-label="Open Scopus in new tab">Scopus (411)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/18273020/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature06547&amp;pmid=18273020" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib59" class="citations"><div id="sref59" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref59-1" title="View in article">59.</a></div><div>Liu, G. ∙ Zhang, J. ∙ Larsen, B. ...</div><div><strong>ProHits: integrated software for mass spectrometry-based interaction proteomics</strong></div><div><em>Nat. Biotechnol.</em> 2010; <strong>28</strong>:1015-1017</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nbt1010-1015" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-77957947158" target="_blank" aria-label="Open Scopus in new tab">Scopus (127)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20944583/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnbt1010-1015&amp;pmid=20944583" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib60" class="citations"><div id="sref60" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref60" title="View in article">60.</a></div><div>Luckey, C.J. ∙ Bhattacharya, D. ∙ Goldrath, A.W. ...</div><div><strong>Memory T and memory B cells share a transcriptional program of self-renewal with long-term hematopoietic stem cells</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2006; <strong>103</strong>:3304-3309</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.0511137103" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-33644746220" target="_blank" aria-label="Open Scopus in new tab">Scopus (172)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16492737/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.0511137103&amp;pmid=16492737" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib61" class="citations"><div id="sref61" class="citation"><div class="citation-content"><div class="label"><a href="#fig1" title="View in article">61.</a></div><div>Ma, M.C.J. ∙ Tadros, S. ∙ Bouska, A. ...</div><div><strong>Pathognomonic and epistatic genetic alterations in B-cell non-Hodgkin lymphoma</strong></div><div><em>bioRxiv.</em> 2019;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/674259" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2F674259" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib62" class="citations"><div id="sref62" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref62-1" title="View in article">62.</a></div><div>Mátés, L. ∙ Chuah, M.K. ∙ Belay, E. ...</div><div><strong>Molecular evolution of a novel hyperactive Sleeping Beauty transposase enables robust stable gene transfer in vertebrates</strong></div><div><em>Nat. Genet.</em> 2009; <strong>41</strong>:753-761</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng.343" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-67349250885" target="_blank" aria-label="Open Scopus in new tab">Scopus (487)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19412179/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng.343&amp;pmid=19412179" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib63" class="citations"><div id="sref63" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref63" title="View in article">63.</a></div><div>Mayer, C.T. ∙ Gazumyan, A. ∙ Kara, E.E. ...</div><div><strong>The microanatomic segregation of selection by apoptosis in the germinal center</strong></div><div><em>Science.</em> 2017; <strong>358</strong>:eaao2602</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aao2602" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-85029760325" target="_blank" aria-label="Open Scopus in new tab">Scopus (57)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28935768/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aao2602&amp;pmid=28935768" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib64" class="citations"><div id="sref64" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref64-1" title="View in article">64.</a></div><div>Mellacheruvu, D. ∙ Wright, Z. ∙ Couzens, A.L. ...</div><div><strong>The CRAPome: a contaminant repository for affinity purification-mass spectrometry data</strong></div><div><em>Nat. Methods.</em> 2013; <strong>10</strong>:730-736</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.2557" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_64_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-84881475940" target="_blank" aria-label="Open Scopus in new tab">Scopus (595)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23921808/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.2557&amp;pmid=23921808" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib65" class="citations"><div id="sref65" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref65" title="View in article">65.</a></div><div>Miles, R.R. ∙ Crockett, D.K. ∙ Lim, M.S. ...</div><div><strong>Analysis of BCL6-interacting proteins by tandem mass spectrometry</strong></div><div><em>Mol. Cell. Proteomics.</em> 2005; <strong>4</strong>:1898-1909</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1074/mcp.M500112-MCP200" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-29244465090" target="_blank" aria-label="Open Scopus in new tab">Scopus (28)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16147992/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1074%2Fmcp.M500112-MCP200&amp;pmid=16147992" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib66" class="citations"><div id="sref66" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref66" title="View in article">66.</a></div><div>Minnich, M. ∙ Tagoh, H. ∙ Bönelt, P. ...</div><div><strong>Multifunctional role of the transcription factor Blimp-1 in coordinating plasma cell differentiation</strong></div><div><em>Nat. Immunol.</em> 2016; <strong>17</strong>:331-343</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ni.3349" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-84954476936" target="_blank" aria-label="Open Scopus in new tab">Scopus (121)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26779602/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fni.3349&amp;pmid=26779602" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib67" class="citations"><div id="sref67" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref67" title="View in article">67.</a></div><div>Mohammed, H. ∙ Taylor, C. ∙ Brown, G.D. ...</div><div><strong>Rapid immunoprecipitation mass spectrometry of endogenous proteins (RIME) for analysis of chromatin complexes</strong></div><div><em>Nat. Protoc.</em> 2016; <strong>11</strong>:316-326</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nprot.2016.020" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26797456/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnprot.2016.020&amp;pmid=26797456" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib68" class="citations"><div id="sref68" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref68" title="View in article">68.</a></div><div>Mondello, P. ∙ Tadros, S. ∙ Teater, M. ...</div><div><strong>Selective Inhibition of HDAC3 Targets Synthetic Vulnerabilities and Activates Immune Surveillance in Lymphoma</strong></div><div><em>Cancer Discov.</em> 2020; <strong>10</strong>:440-459</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1158/2159-8290.CD-19-0116" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-85081148256" target="_blank" aria-label="Open Scopus in new tab">Scopus (3)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31915197/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1158%2F2159-8290.CD-19-0116&amp;pmid=31915197" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib69" class="citations"><div id="sref69" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref69-1" title="View in article">69.</a></div><div>Mort, R.L. ∙ Ford, M.J. ∙ Sakaue-Sawano, A. ...</div><div><strong>Fucci2a: a bicistronic cell cycle reporter that allows Cre mediated tissue specific expression in mice</strong></div><div><em>Cell Cycle.</em> 2014; <strong>13</strong>:2681-2696</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.4161/15384101.2015.945381" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-84907438909" target="_blank" aria-label="Open Scopus in new tab">Scopus (39)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25486356/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.4161%2F15384101.2015.945381&amp;pmid=25486356" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib70" class="citations"><div id="sref70" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref70-1" title="View in article">70.</a></div><div>Muramatsu, M. ∙ Kinoshita, K. ∙ Fagarasan, S. ...</div><div><strong>Class switch recombination and hypermutation require activation-induced cytidine deaminase (AID), a potential RNA editing enzyme</strong></div><div><em>Cell.</em> 2000; <strong>102</strong>:553-563</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=10.1016%2FS0092-8674%2800%2900078-7&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=10.1016%2FS0092-8674%2800%2900078-7&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11007474/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0092-8674%2800%2900078-7&amp;pmid=11007474" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib119" class="citations"><div id="opt4ZAehgbwtz" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-opt4ZAehgbwtz" title="View in article">71.</a></div><div>National Research Council (US) Institute for Laboratory Animal Research</div><div><em>Guide for the Care and Use of Laboratory Animals.</em>National Academies Press (US), Washington, DC, 1996</div><div><a id="interref0510" href="https://www.ncbi.nlm.nih.gov/books/NBK232589/%20doi:%2010.17226/5140" target="_blank">https://www.ncbi.nlm.nih.gov/books/NBK232589/ doi: 10.17226/5140</a></div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=National+Research+Council+%28US%29+Institute+for+Laboratory+Animal+ResearchGuide+for+the+Care+and+Use+of+Laboratory+Animals1996National+Academies+Press+%28US%29Washington%2C+DChttps%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fbooks%2FNBK232589%2F+doi%3A+10.17226%2F5140" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib71" class="citations"><div id="sref71" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref71-1" title="View in article">72.</a></div><div>Nissen, M.D. ∙ Kusakabe, M. ∙ Wang, X. ...</div><div><strong>Single Cell Phenotypic Profiling of 27 DLBCL Cases Reveals Marked Intertumoral and Intratumoral Heterogeneity</strong></div><div><em>Cytometry A.</em> 2019;</div><div>Published online October 22, 2019</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/cyto.a.23919" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-85075788395" target="_blank" aria-label="Open Scopus in new tab">Scopus (2)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31637838/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fcyto.a.23919&amp;pmid=31637838" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib72" class="citations"><div id="sref72" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref72-1" title="View in article">73.</a></div><div>Nojima, T. ∙ Haniuda, K. ∙ Moutai, T. ...</div><div><strong>In-vitro derived germinal centre B cells differentially generate memory B or plasma cells in&nbsp;vivo</strong></div><div><em>Nat. Commun.</em> 2011; <strong>2</strong>:465</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ncomms1475" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-80052643363" target="_blank" aria-label="Open Scopus in new tab">Scopus (101)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21897376/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fncomms1475&amp;pmid=21897376" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib73" class="citations"><div id="sref73" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref73" title="View in article">74.</a></div><div>Oberoi, J. ∙ Fairall, L. ∙ Watson, P.J. ...</div><div><strong>Structural basis for the assembly of the SMRT/NCoR core transcriptional repression machinery</strong></div><div><em>Nat. Struct. Mol. Biol.</em> 2011; <strong>18</strong>:177-184</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nsmb.1983" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-79551626803" target="_blank" aria-label="Open Scopus in new tab">Scopus (89)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21240272/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnsmb.1983&amp;pmid=21240272" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib74" class="citations"><div id="sref74" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref74-1" title="View in article">75.</a></div><div>Ochiai, K. ∙ Katoh, Y. ∙ Ikura, T. ...</div><div><strong>Plasmacytic transcription factor Blimp-1 is repressed by Bach2 in B cells</strong></div><div><em>J.&nbsp;Biol. Chem.</em> 2006; <strong>281</strong>:38226-38234</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1074/jbc.M607592200" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_75_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-33846013955" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17046816/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1074%2Fjbc.M607592200&amp;pmid=17046816" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib75" class="citations"><div id="sref75" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref75-1" title="View in article">76.</a></div><div>Ogilvy, S. ∙ Metcalf, D. ∙ Print, C.G. ...</div><div><strong>Constitutive Bcl-2 expression throughout the hematopoietic compartment affects multiple lineages and enhances progenitor cell survival</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 1999; <strong>96</strong>:14943-14948</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.96.26.14943" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_76_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-0033593038" target="_blank" aria-label="Open Scopus in new tab">Scopus (281)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/10611317/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.96.26.14943&amp;pmid=10611317" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib76" class="citations"><div id="sref76" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref76" title="View in article">77.</a></div><div>Oliver, A.M. ∙ Martin, F. ∙ Kearney, J.F.</div><div><strong>Mouse CD38 is down-regulated on germinal center B cells and mature plasma cells</strong></div><div><em>J.&nbsp;Immunol.</em> 1997; <strong>158</strong>:1108-1115</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.4049/jimmunol.158.3.1108" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/9013949/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.4049%2Fjimmunol.158.3.1108&amp;pmid=9013949" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib77" class="citations"><div id="sref77" class="citation"><div class="citation-content"><div class="label"><a href="#fig1" title="View in article">78.</a></div><div>Ortega-Molina, A. ∙ Boss, I.W. ∙ Canela, A. ...</div><div><strong>The histone lysine methyltransferase KMT2D sustains a gene expression program that represses B cell lymphoma development</strong></div><div><em>Nat. Med.</em> 2015; <strong>21</strong>:1199-1208</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nm.3943" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_78_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-84943658052" target="_blank" aria-label="Open Scopus in new tab">Scopus (159)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26366710/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnm.3943&amp;pmid=26366710" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib78" class="citations"><div id="sref78" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref78" title="View in article">79.</a></div><div>Pagès, H. ∙ Aboyoun, P. ∙ Gentleman, R. ...</div><div><strong>Biostrings: Efficient manipulation of biological strings. R package version 2.54.0</strong></div><div>2019</div><div><a id="intref0125" href="https://bioconductor.org/packages/release/bioc/html/Biostrings.html" target="_blank">https://bioconductor.org/packages/release/bioc/html/Biostrings.html</a></div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=H.Pag%C3%A8sP.AboyounR.GentlemanS.DebRoyBiostrings%3A+Efficient+manipulation+of+biological+strings.+R+package+version+2.54.02019https%3A%2F%2Fbioconductor.org%2Fpackages%2Frelease%2Fbioc%2Fhtml%2FBiostrings.html" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib79" class="citations"><div id="sref79" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref79" title="View in article">80.</a></div><div>Pereira, J.P. ∙ Kelly, L.M. ∙ Xu, Y. ...</div><div><strong>EBI2 mediates B cell segregation between the outer and centre follicle</strong></div><div><em>Nature.</em> 2009; <strong>460</strong>:1122-1126</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature08226" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_80_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-69349087555" target="_blank" aria-label="Open Scopus in new tab">Scopus (203)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19597478/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature08226&amp;pmid=19597478" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib80" class="citations"><div id="sref80" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref80-1" title="View in article">81.</a></div><div>Perissi, V. ∙ Aggarwal, A. ∙ Glass, C.K. ...</div><div><strong>A corepressor/coactivator exchange complex required for transcriptional activation by nuclear receptors and other regulated transcription factors</strong></div><div><em>Cell.</em> 2004; <strong>116</strong>:511-526</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=10.1016%2FS0092-8674%2804%2900133-3&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=10.1016%2FS0092-8674%2804%2900133-3&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-1342264315" target="_blank" aria-label="Open Scopus in new tab">Scopus (412)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/14980219/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0092-8674%2804%2900133-3&amp;pmid=14980219" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib82" class="citations"><div id="sref82" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref82" title="View in article">82.</a></div><div>Phelan, J.D. ∙ Young, R.M. ∙ Webster, D.E. ...</div><div><strong>A multiprotein supercomplex controlling oncogenic signalling in lymphoma</strong></div><div><em>Nature.</em> 2018; <strong>560</strong>:387-391</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-018-0290-0" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_82_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-85051208319" target="_blank" aria-label="Open Scopus in new tab">Scopus (79)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29925955/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-018-0290-0&amp;pmid=29925955" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib83" class="citations"><div id="sref83" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref83" title="View in article">83.</a></div><div>Polo, J.M. ∙ Ci, W. ∙ Licht, J.D. ...</div><div><strong>Reversible disruption of BCL6 repression complexes by CD40 signaling in normal and malignant B cells</strong></div><div><em>Blood.</em> 2008; <strong>112</strong>:644-651</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1182/blood-2008-01-131813" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_83_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-50949097945" target="_blank" aria-label="Open Scopus in new tab">Scopus (24)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/18487509/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1182%2Fblood-2008-01-131813&amp;pmid=18487509" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib86" class="citations"><div id="sref86" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref86-1" title="View in article">84.</a></div><div>Reddy, A. ∙ Zhang, J. ∙ Davis, N.S. ...</div><div><strong>Genetic and Functional Drivers of Diffuse Large B Cell Lymphoma</strong></div><div><em>Cell.</em> 2017; <strong>171</strong>:481-494</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_84_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=10.1016%2Fj.cell.2017.09.027&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_84_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=10.1016%2Fj.cell.2017.09.027&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_84_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-85030529459" target="_blank" aria-label="Open Scopus in new tab">Scopus (251)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28985567/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2017.09.027&amp;pmid=28985567" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib87" class="citations"><div id="sref87" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref87" title="View in article">85.</a></div><div>Ridderstad, A. ∙ Tarlinton, D.M.</div><div><strong>Kinetics of establishing the memory B cell population as revealed by CD38 expression</strong></div><div><em>J.&nbsp;Immunol.</em> 1998; <strong>160</strong>:4688-4695</div></div><div class="external-links"><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/9590214/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?pmid=9590214" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib88" class="citations"><div id="sref88" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref88-1" title="View in article">86.</a></div><div>Robinson, M.D. ∙ McCarthy, D.J. ∙ Smyth, G.K.</div><div><strong>edgeR: a Bioconductor package for differential expression analysis of digital gene expression data</strong></div><div><em>Bioinformatics.</em> 2010; <strong>26</strong>:139-140</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btp616" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_86_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-75249087100" target="_blank" aria-label="Open Scopus in new tab">Scopus (11879)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19910308/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtp616&amp;pmid=19910308" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib89" class="citations"><div id="sref89" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref89" title="View in article">87.</a></div><div>Roux, K.J. ∙ Kim, D.I. ∙ Raida, M. ...</div><div><strong>A promiscuous biotin ligase fusion protein identifies proximal and interacting proteins in mammalian cells</strong></div><div><em>J.&nbsp;Cell Biol.</em> 2012; <strong>196</strong>:801-810</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1083/jcb.201112098" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_87_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-84860270506" target="_blank" aria-label="Open Scopus in new tab">Scopus (715)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22412018/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1083%2Fjcb.201112098&amp;pmid=22412018" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib90" class="citations"><div id="sref90" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref90-1" title="View in article">88.</a></div><div>Saunders, C.T. ∙ Wong, W.S. ∙ Swamy, S. ...</div><div><strong>Strelka: accurate somatic small-variant calling from sequenced tumor-normal sample pairs</strong></div><div><em>Bioinformatics.</em> 2012; <strong>28</strong>:1811-1817</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/bts271" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-84864153492" target="_blank" aria-label="Open Scopus in new tab">Scopus (676)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22581179/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbts271&amp;pmid=22581179" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib91" class="citations"><div id="sref91" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref91-1" title="View in article">89.</a></div><div>Schindelin, J. ∙ Arganda-Carreras, I. ∙ Frise, E. ...</div><div><strong>Fiji: an open-source platform for biological-image analysis</strong></div><div><em>Nat. Methods.</em> 2012; <strong>9</strong>:676-682</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.2019" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_89_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-84862520770" target="_blank" aria-label="Open Scopus in new tab">Scopus (16304)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22743772/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.2019&amp;pmid=22743772" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib92" class="citations"><div id="sref92" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref92-1" title="View in article">90.</a></div><div>Schmitz, R. ∙ Wright, G.W. ∙ Huang, D.W. ...</div><div><strong>Genetics and Pathogenesis of Diffuse Large B-Cell Lymphoma</strong></div><div><em>N.&nbsp;Engl. J. Med.</em> 2018; <strong>378</strong>:1396-1407</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1056/NEJMoa1801445" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_90_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-85045505659" target="_blank" aria-label="Open Scopus in new tab">Scopus (314)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29641966/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1056%2FNEJMoa1801445&amp;pmid=29641966" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib93" class="citations"><div id="sref93" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref93-1" title="View in article">91.</a></div><div>Schneider, C.A. ∙ Rasband, W.S. ∙ Eliceiri, K.W.</div><div><strong>NIH Image to ImageJ: 25 years of image analysis</strong></div><div><em>Nat. Methods.</em> 2012; <strong>9</strong>:671-675</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.2089" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-84863205849" target="_blank" aria-label="Open Scopus in new tab">Scopus (21916)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22930834/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.2089&amp;pmid=22930834" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib94" class="citations"><div id="sref94" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref94-1" title="View in article">92.</a></div><div>Scott, D.W. ∙ Mottok, A. ∙ Ennishi, D. ...</div><div><strong>Prognostic Significance of Diffuse Large B-Cell Lymphoma Cell of Origin Determined by Digital Gene Expression in Formalin-Fixed Paraffin-Embedded Tissue Biopsies</strong></div><div><em>J.&nbsp;Clin. Oncol.</em> 2015; <strong>33</strong>:2848-2856</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1200/JCO.2014.60.2383" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_92_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-84964994916" target="_blank" aria-label="Open Scopus in new tab">Scopus (165)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26240231/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1200%2FJCO.2014.60.2383&amp;pmid=26240231" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib95" class="citations"><div id="sref95" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref95" title="View in article">93.</a></div><div>Sherry, S.T. ∙ Ward, M.H. ∙ Kholodov, M. ...</div><div><strong>dbSNP: the NCBI database of genetic variation</strong></div><div><em>Nucleic Acids Res.</em> 2001; <strong>29</strong>:308-311</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/29.1.308" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11125122/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2F29.1.308&amp;pmid=11125122" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib96" class="citations"><div id="sref96" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref96-1" title="View in article">94.</a></div><div>Shinnakasu, R. ∙ Inoue, T. ∙ Kometani, K. ...</div><div><strong>Regulated selection of germinal-center cells into the memory B cell compartment</strong></div><div><em>Nat. Immunol.</em> 2016; <strong>17</strong>:861-869</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ni.3460" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_94_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-84966559006" target="_blank" aria-label="Open Scopus in new tab">Scopus (208)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27158841/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fni.3460&amp;pmid=27158841" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib97" class="citations"><div id="sref97" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref97" title="View in article">95.</a></div><div>Singh, M. ∙ Jackson, K.J.L. ∙ Wang, J.J. ...</div><div><strong>Lymphoma Driver Mutations in the Pathogenic Evolution of an Iconic Human Autoantibody</strong></div><div><em>Cell.</em> 2020; <strong>180</strong>:878-894</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_95_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=10.1016%2Fj.cell.2020.01.029&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_95_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=10.1016%2Fj.cell.2020.01.029&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_95_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-85080139890" target="_blank" aria-label="Open Scopus in new tab">Scopus (7)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32059783/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2020.01.029&amp;pmid=32059783" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib98" class="citations"><div id="sref98" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref98" title="View in article">96.</a></div><div>Stengel, K.R. ∙ Bhaskara, S. ∙ Wang, J. ...</div><div><strong>Histone deacetylase 3 controls a transcriptional network required for B cell maturation</strong></div><div><em>Nucleic Acids Res.</em> 2019; <strong>47</strong>:10612-10627</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gkz816" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_96_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-85074875427" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31586401/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgkz816&amp;pmid=31586401" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib99" class="citations"><div id="sref99" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref99" title="View in article">97.</a></div><div>Stewart, S.A. ∙ Dykxhoorn, D.M. ∙ Palliser, D. ...</div><div><strong>Lentivirus-delivered stable gene silencing by RNAi in primary cells</strong></div><div><em>RNA.</em> 2003; <strong>9</strong>:493-501</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1261/rna.2192803" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_97_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-0344375083" target="_blank" aria-label="Open Scopus in new tab">Scopus (782)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/12649500/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1261%2Frna.2192803&amp;pmid=12649500" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib100" class="citations"><div id="sref100" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref100" title="View in article">98.</a></div><div>Stewart, I. ∙ Radtke, D. ∙ Phillips, B. ...</div><div><strong>Germinal Center B Cells Replace Their Antigen Receptors in Dark Zones and Fail Light Zone Entry when Immunoglobulin Gene Mutations are Damaging</strong></div><div><em>Immunity.</em> 2018; <strong>49</strong>:477-489</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_98_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=10.1016%2Fj.immuni.2018.08.025&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_98_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=10.1016%2Fj.immuni.2018.08.025&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_98_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-85053143886" target="_blank" aria-label="Open Scopus in new tab">Scopus (17)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30231983/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.immuni.2018.08.025&amp;pmid=30231983" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib101" class="citations"><div id="sref101" class="citation"><div class="citation-content"><div class="label"><a href="#figs3" title="View in article">99.</a></div><div>Suan, D. ∙ Krautler, N.J. ∙ Maag, J.L.V. ...</div><div><strong>CCR6 Defines Memory B Cell Precursors in Mouse and Human Germinal Centers, Revealing Light-Zone Location and Predominant Low Antigen Affinity</strong></div><div><em>Immunity.</em> 2017; <strong>47</strong>:1142-1153</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_99_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=10.1016%2Fj.immuni.2017.11.022&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_99_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=10.1016%2Fj.immuni.2017.11.022&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_99_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-85039037389" target="_blank" aria-label="Open Scopus in new tab">Scopus (50)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29262350/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.immuni.2017.11.022&amp;pmid=29262350" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib102" class="citations"><div id="sref102" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref102" title="View in article">100.</a></div><div>Subramanian, A. ∙ Tamayo, P. ∙ Mootha, V.K. ...</div><div><strong>Gene set enrichment analysis: a knowledge-based approach for interpreting genome-wide expression profiles</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2005; <strong>102</strong>:15545-15550</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.0506580102" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_100_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-27344435774" target="_blank" aria-label="Open Scopus in new tab">Scopus (16287)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16199517/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.0506580102&amp;pmid=16199517" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib103" class="citations"><div id="sref103" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref103-1" title="View in article">101.</a></div><div>Sungalee, S. ∙ Mamessier, E. ∙ Morgado, E. ...</div><div><strong>Germinal center reentries of BCL2-overexpressing B cells drive follicular lymphoma progression</strong></div><div><em>J.&nbsp;Clin. Invest.</em> 2014; <strong>124</strong>:5337-5351</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1172/JCI72415" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_101_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-84915818499" target="_blank" aria-label="Open Scopus in new tab">Scopus (49)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25384217/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1172%2FJCI72415&amp;pmid=25384217" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib105" class="citations"><div id="sref105" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref105" title="View in article">102.</a></div><div>Tate, J.G. ∙ Bamford, S. ∙ Jubb, H.C. ...</div><div><strong>COSMIC: the Catalogue Of Somatic Mutations In Cancer</strong></div><div><em>Nucleic Acids Res.</em> 2019; <strong>47</strong>:D941-D947</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gky1015" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_102_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-85057093998" target="_blank" aria-label="Open Scopus in new tab">Scopus (372)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30371878/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgky1015&amp;pmid=30371878" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib106" class="citations"><div id="sref106" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref106" title="View in article">103.</a></div><div>Tomita, A. ∙ Buchholz, D.R. ∙ Shi, Y.B.</div><div><strong>Recruitment of N-CoR/SMRT-TBLR1 corepressor complex by unliganded thyroid hormone receptor for gene repression during frog development</strong></div><div><em>Mol. Cell. Biol.</em> 2004; <strong>24</strong>:3337-3346</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/MCB.24.8.3337-3346.2004" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_103_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-1842609856" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15060155/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FMCB.24.8.3337-3346.2004&amp;pmid=15060155" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib107" class="citations"><div id="sref107" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref107" title="View in article">104.</a></div><div>Tunyaplin, C. ∙ Shaffer, A.L. ∙ Angelin-Duclos, C.D. ...</div><div><strong>Direct repression of prdm1 by Bcl-6 inhibits plasmacytic differentiation</strong></div><div><em>J.&nbsp;Immunol.</em> 2004; <strong>173</strong>:1158-1165</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.4049/jimmunol.173.2.1158" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15240705/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.4049%2Fjimmunol.173.2.1158&amp;pmid=15240705" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib108" class="citations"><div id="sref108" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref108" title="View in article">105.</a></div><div>Wang, Y. ∙ Hu, X.J. ∙ Zou, X.D. ...</div><div><strong>WDSPdb: a database for WD40-repeat proteins</strong></div><div><em>Nucleic Acids Res.</em> 2015; <strong>43</strong>:D339-D344</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gku1023" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_105_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-84941093734" target="_blank" aria-label="Open Scopus in new tab">Scopus (45)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25348404/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgku1023&amp;pmid=25348404" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib109" class="citations"><div id="sref109" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref109" title="View in article">106.</a></div><div>Wang, J.Q. ∙ Jeelall, Y.S. ∙ Humburg, P. ...</div><div><strong>Synergistic cooperation and crosstalk between <i>MYD88</i><sup><i>L265P</i></sup> and mutations that dysregulate CD79B and surface IgM</strong></div><div><em>J.&nbsp;Exp. Med.</em> 2017; <strong>214</strong>:2759-2776</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1084/jem.20161454" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_106_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-85028857023" target="_blank" aria-label="Open Scopus in new tab">Scopus (13)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28701369/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1084%2Fjem.20161454&amp;pmid=28701369" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib110" class="citations"><div id="sref110" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref110-1" title="View in article">107.</a></div><div>Wang, Y. ∙ Shi, J. ∙ Yan, J. ...</div><div><strong>Germinal-center development of memory B cells driven by IL-9 from follicular helper T&nbsp;cells</strong></div><div><em>Nat. Immunol.</em> 2017; <strong>18</strong>:921-930</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ni.3788" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_107_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-85025817827" target="_blank" aria-label="Open Scopus in new tab">Scopus (58)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28650481/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fni.3788&amp;pmid=28650481" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib111" class="citations"><div id="sref111" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref111" title="View in article">108.</a></div><div>Weisel, F.J. ∙ Zuccarino-Catania, G.V. ∙ Chikina, M. ...</div><div><strong>A Temporal Switch in the Germinal Center Determines Differential Output of Memory B and Plasma Cells</strong></div><div><em>Immunity.</em> 2016; <strong>44</strong>:116-130</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_108_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=10.1016%2Fj.immuni.2015.12.004&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_108_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=10.1016%2Fj.immuni.2015.12.004&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_108_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-84958831765" target="_blank" aria-label="Open Scopus in new tab">Scopus (158)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26795247/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.immuni.2015.12.004&amp;pmid=26795247" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib112" class="citations"><div id="sref112" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref112-1" title="View in article">109.</a></div><div>Wright, G. ∙ Tan, B. ∙ Rosenwald, A. ...</div><div><strong>A gene expression-based method to diagnose clinically distinct subgroups of diffuse large B cell lymphoma</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2003; <strong>100</strong>:9991-9996</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1732008100" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_109_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-0043192901" target="_blank" aria-label="Open Scopus in new tab">Scopus (726)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/12900505/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1732008100&amp;pmid=12900505" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib113" class="citations"><div id="sref113" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref113-1" title="View in article">110.</a></div><div>Wright, G.W. ∙ Huang, D.W. ∙ Phelan, J.D. ...</div><div><strong>A Probabilistic Classification Tool for Genetic Subtypes of Diffuse Large B Cell Lymphoma with Therapeutic Implications</strong></div><div><em>Cancer Cell.</em> 2020; <strong>37</strong>:551-568.e14</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_110_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=10.1016%2Fj.ccell.2020.03.015&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_110_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=10.1016%2Fj.ccell.2020.03.015&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_110_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-85082775478" target="_blank" aria-label="Open Scopus in new tab">Scopus (12)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32289277/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.ccell.2020.03.015&amp;pmid=32289277" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib114" class="citations"><div id="sref114" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref114" title="View in article">111.</a></div><div>Yoon, H.G. ∙ Chan, D.W. ∙ Huang, Z.Q. ...</div><div><strong>Purification and functional characterization of the human N-CoR complex: the roles of HDAC3, TBL1 and TBLR1</strong></div><div><em>EMBO J.</em> 2003; <strong>22</strong>:1336-1346</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/emboj/cdg120" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_111_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-0344405718" target="_blank" aria-label="Open Scopus in new tab">Scopus (326)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/12628926/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Femboj%2Fcdg120&amp;pmid=12628926" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib115" class="citations"><div id="sref115" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref115" title="View in article">112.</a></div><div>Yoon, H.G. ∙ Choi, Y. ∙ Cole, P.A. ...</div><div><strong>Reading and function of a histone code involved in targeting corepressor complexes for repression</strong></div><div><em>Mol. Cell. Biol.</em> 2005; <strong>25</strong>:324-335</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/MCB.25.1.324-335.2005" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_112_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-11144333630" target="_blank" aria-label="Open Scopus in new tab">Scopus (83)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15601853/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FMCB.25.1.324-335.2005&amp;pmid=15601853" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib116" class="citations"><div id="sref116" class="citation"><div class="citation-content"><div class="label"><a href="#fig1" title="View in article">113.</a></div><div>Zhang, J. ∙ Kalkum, M. ∙ Chait, B.T. ...</div><div><strong>The N-CoR-HDAC3 nuclear receptor corepressor complex inhibits the JNK pathway through the integral subunit GPS2</strong></div><div><em>Mol. Cell.</em> 2002; <strong>9</strong>:611-623</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_113_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=10.1016%2FS1097-2765%2802%2900468-9&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_113_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=10.1016%2FS1097-2765%2802%2900468-9&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_113_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-0036211850" target="_blank" aria-label="Open Scopus in new tab">Scopus (305)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11931768/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS1097-2765%2802%2900468-9&amp;pmid=11931768" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib117" class="citations"><div id="sref117" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref117" title="View in article">114.</a></div><div>Zhang, X.M. ∙ Chang, Q. ∙ Zeng, L. ...</div><div><strong>TBLR1 regulates the expression of nuclear hormone receptor co-repressors</strong></div><div><em>BMC Cell Biol.</em> 2006; <strong>7</strong>:31</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/1471-2121-7-31" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_114_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-33748053323" target="_blank" aria-label="Open Scopus in new tab">Scopus (29)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16893456/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2F1471-2121-7-31&amp;pmid=16893456" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib118" class="citations"><div id="sref118" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref118-1" title="View in article">115.</a></div><div>Zuccarino-Catania, G.V. ∙ Sadanand, S. ∙ Weisel, F.J. ...</div><div><strong>CD80 and PD-L2 define functionally distinct memory B cell subsets that are independent of antibody isotype</strong></div><div><em>Nat. Immunol.</em> 2014; <strong>15</strong>:631-637</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ni.2914" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_115_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.05.049&amp;key=2-s2.0-84903646784" target="_blank" aria-label="Open Scopus in new tab">Scopus (154)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24880458/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fni.2914&amp;pmid=24880458" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div></div></div></section></div></section><div class="core-linked-content">
    

        


    

            

            

            <div data-widget-id="8633d695-b258-4908-b86f-e2c268ec3c6b" data-widget-name="UX3HTMLWidget"><script type="text/javascript">
if($('.content-navigation__body__title').length){
$('.content-navigation__body__title').each(function(){
var html=this.innerHTML;
var div = document.createElement("div");
div.innerHTML = html;
var text = div.textContent || div.innerText || "";
this.innerHTML=text;
});
}
</script></div>

            

        

    

        


    

            

            

            <div data-widget-id="393effff-5b44-4681-a9b4-4f029b3540b7" data-widget-name="UX3HTMLWidget"><style>
.linkback-ad .pb-ad {
    display: block;
   height: auto;
    margin-left: 0px;
    margin-right: auto;
}
</style></div>

            

        

    

        

</div></div></div><div class="core-collateral-aside"><section id="core-collateral-figures"><h2>Figures (14)<button class="js--openFigureViewer"><span>Figure Viewer</span></button></h2><div class="core-collateral-aside-wrapper"><div class="grid"><a class="figure__open__ctrl" href="#fig1" aria-label="Go to Figure&amp;nbsp;1"><img src="https://www.cell.com/cms/10.1016/j.cell.2020.05.049/asset/ba82fc3d-99c6-4ccc-a053-85147c162594/main.assets/gr1.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs1" aria-label="Go to Figure&amp;nbsp;S1"><img src="https://www.cell.com/cms/10.1016/j.cell.2020.05.049/asset/887da720-0b0a-45b0-b2da-60e17e50e74e/main.assets/figs1.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig2" aria-label="Go to Figure&amp;nbsp;2"><img src="https://www.cell.com/cms/10.1016/j.cell.2020.05.049/asset/e707e4a6-c241-4140-ba22-f22d6f9923e3/main.assets/gr2.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs2" aria-label="Go to Figure&amp;nbsp;S2"><img src="https://www.cell.com/cms/10.1016/j.cell.2020.05.049/asset/f1708978-4022-497e-8836-7c1de5107ea6/main.assets/figs2.jpg" aria-hidden="true"></a></div><div id="collapsible-figures" data-expandable="item"><div class="content grid" style="display:none;" role="region" id="collapsible-figures_content"><a class="figure__open__ctrl" href="#fig3" aria-label="Go to Figure&amp;nbsp;3"><img src="https://www.cell.com/cms/10.1016/j.cell.2020.05.049/asset/30b177b1-f775-42e3-8c66-19ad2923efd4/main.assets/gr3.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs3" aria-label="Go to Figure&amp;nbsp;S3"><img src="https://www.cell.com/cms/10.1016/j.cell.2020.05.049/asset/961f22d5-e0b9-497b-b934-e531fae97422/main.assets/figs3.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig4" aria-label="Go to Figure&amp;nbsp;4"><img src="https://www.cell.com/cms/10.1016/j.cell.2020.05.049/asset/4e2f7579-d2f0-4ae5-8b67-1ea3503f9926/main.assets/gr4.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs4" aria-label="Go to Figure&amp;nbsp;S4"><img src="https://www.cell.com/cms/10.1016/j.cell.2020.05.049/asset/71ecc933-5f53-4b1a-946c-ea0a5e0da314/main.assets/figs4.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig5" aria-label="Go to Figure&amp;nbsp;5"><img src="https://www.cell.com/cms/10.1016/j.cell.2020.05.049/asset/9ecf4736-5a3c-4c66-a9d9-a9b7b384c4b6/main.assets/gr5.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs5" aria-label="Go to Figure&amp;nbsp;S5"><img src="https://www.cell.com/cms/10.1016/j.cell.2020.05.049/asset/66389efa-498c-45ab-a71e-fc0a596b21f2/main.assets/figs5.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig6" aria-label="Go to Figure&amp;nbsp;6"><img src="https://www.cell.com/cms/10.1016/j.cell.2020.05.049/asset/875f9e06-0008-4d56-a798-7ec2240d02d6/main.assets/gr6.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs6" aria-label="Go to Figure&amp;nbsp;S6"><img src="https://www.cell.com/cms/10.1016/j.cell.2020.05.049/asset/19a3ccd7-1dab-455b-a8af-704fdc1262f2/main.assets/figs6.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig7" aria-label="Go to Figure&amp;nbsp;7"><img src="https://www.cell.com/cms/10.1016/j.cell.2020.05.049/asset/e047e0cd-51ea-4a91-a5b2-ecec5777e824/main.assets/gr7.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs7" aria-label="Go to Figure&amp;nbsp;S7"><img src="https://www.cell.com/cms/10.1016/j.cell.2020.05.049/asset/0a5f9edf-ce98-42a9-b553-c6255571d74e/main.assets/figs7.jpg" aria-hidden="true"></a></div><button class="trigger" aria-controls="collapsible-figures_content" aria-expanded="false" tabindex="0" role="button"><span class="expand">Show all figures</span><span class="collapse">Hide figures</span><i class="icon-section_arrow_d" aria-hidden="true"></i></button></div></div></section>
    

        


    

            

            

            <section id="core-collateral-metrics" class="publication-list__axel-section"><h2>Article metrics</h2><div><ul class="publication-list rlist publication-list--list plum-x--lazy-load publication-list--axel js--loaded" data-plum-x-url="/pb/widgets/ux3/publicationList/plumx?widgetId=66dd7e11-7768-4006-93d8-a72b60e2a264&amp;pbContext=%3Bproduct%3Aproduct%3Aelsevier%5C%3Aproduct%5C%3Acell%3Bwebsite%3Awebsite%3Acell-site%3Bctype%3Astring%3AJournal+Content%3Bjournal%3Ajournal%3Acell%3BpageGroup%3Astring%3AArticle+View%3Barticle%3Aarticle%3Apii%5C%3AS0092867420306826%3Bpage%3Astring%3AShow+Full+Text%3BrequestedJournal%3Ajournal%3Acell%3Bissue%3Aissue%3Apii%5C%3AS0092867419X00161%3Bwgroup%3Astring%3ADefault+Website+Group%3BsubPage%3Astring%3AFull+Text" data-pb-mode-enabled="false">
                <li class="publication-list__box publication-list__box-axel">
                    <ul class="rlist axel-list__box">
                        
                        
            <li class="plum-x-wrapper">
                <div class="plum-x plum-x--axel">
                    <div class="plum-x__items">
                        
                    <div class="plum-x__item plum-x-citation">
                        <div class="plum-x__item-num">77</div>
                        <div class="plum-x__item-title">Citations</div>
                    </div>    
                
                    <div class="plum-x__item plum-x-capture">
                        <div class="plum-x__item-num">136</div>
                        <div class="plum-x__item-title">Captures</div>
                    </div>    
                
                    <div class="plum-x__item plum-x-socialMedia">
                        <div class="plum-x__item-num">76</div>
                        <div class="plum-x__item-title">Social Media</div>
                    </div>    
                
                    </div>
                    <div class="plum-x__details">
                        <div class="plum-x__logo">
                            <img src="/products/marlin/releasedAssets/images/plumx-logo.png" alt="Plum X logo">
                        </div>
                        <a class="plum-x__view-details" href="https://plu.mx/plum/a/?doi=10.1016/j.cell.2020.05.049&amp;theme=plum-jbs-theme&amp;hideUsage=true" target="_blank">
                            <span>View details</span>
                            <span class="sr-only">opens in a new tab</span>
                            <i class="icon-external-arrow" aria-hidden="true"></i>
                        </a>
                    </div>
                </div>  
            </li>
        
                    </ul>
                </li></ul></div></section>

            

        
<section id="core-collateral-supplementary"><h2>Supplemental information (6)</h2><div class="core-collateral-aside-wrapper"><div class="core-supplementary-material"><div class="core-link"><button class="downloadAllSupplementary"><span>Download all</span></button></div></div><div class="core-supplementary-material" data-category="pdf"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2020.05.049/attachment/2ca051f9-bb14-45f0-b9ae-ff51acdc11f6/mmc1.pdf" download="mmc1.pdf"><span id="mmc1-link">PDF (125.21 KB)</span></a><div class="core-description">Document S1. Tables S1 and S6</div></div></div><div class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2020.05.049/attachment/993d00e8-66e8-4f77-bb5d-254a8fac8ad5/mmc2.xlsx" download="mmc2.xlsx"><span id="mmc2-link">Spreadsheet (52.00 KB)</span></a><div class="core-description">Table S2. Differentially Expressed Genes in Tbl1xr1-D370Y/WT GCB (|log2FC| ≥ 1.5 and FDR &lt;0.05), Related to Figure&nbsp;3</div></div></div><div class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2020.05.049/attachment/09e04835-87bb-40ca-8a99-236e091fcee6/mmc3.xlsx" download="mmc3.xlsx"><span id="mmc3-link">Spreadsheet (233.96 KB)</span></a><div class="core-description">Table S3. Spectral Counts for BCL6 RIME Assay in the OCI-Ly1 Cell Line, Related to Figure&nbsp;5</div></div></div><div class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2020.05.049/attachment/60deed22-98da-40e2-b747-3479d2eb6fa6/mmc4.xlsx" download="mmc4.xlsx"><span id="mmc4-link">Spreadsheet (69.00 KB)</span></a><div class="core-description">Table S4. Differentially Expressed Genes in IμBcl6-Tbl1xr1-D370Y/WT GCB (|log2FC| ≥ 1.5 and FDR&nbsp;&lt; 0.05) Related to Figure&nbsp;5</div></div></div><div class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2020.05.049/attachment/21da09b6-4850-4a46-8693-590f5d9f47c1/mmc5.xlsx" download="mmc5.xlsx"><span id="mmc5-link">Spreadsheet (16.61 KB)</span></a><div class="core-description">Table S5. Somatic Mutation Profiles of DLBCL Specimens Used for CyTOF Immunophenotyping, Related to Figure&nbsp;7</div></div></div><div id="collapsible-supplementary" data-expandable="item"><div class="content" style="display:none;" role="region" id="collapsible-supplementary_content"><div class="core-supplementary-material" data-category="pdf"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2020.05.049/attachment/5bee300c-2043-47e9-ba5d-e6629b0b13e1/mmc6.pdf" download="mmc6.pdf"><span id="mmc6-link">PDF (2.60 MB)</span></a><div class="core-description">Data S1. Complete FC Gating Strategies and Additional Data, Related to Figures 1, 2, 3, 4, 5, 6, and 7</div></div></div></div><button class="trigger" aria-controls="collapsible-supplementary_content" aria-expanded="false" tabindex="0" role="button"><span class="expand">Show more</span><span class="collapse">Show less</span><i class="icon-section_arrow_d" aria-hidden="true"></i></button></div></div></section>
    

        


    

            

            

            <section id="core-collateral-relatedArticles" class="axel-related-articles"><h2 id="relatedArticles" class="top axel-related-articles__title" data-title="Related articles">Related Articles</h2><div data-pb-widget-type="ux3-relatedArticles" data-widget-id="8d3844d1-450e-497c-ab75-a63fc41750ac" data-widget-name="AtyponTempUX3RelatedArticlesWidget" data-article-pii="S0092867420306826" class="ux3DeferredOnScrollWidget axel-related-articles__body" data-show-more="true"></div></section>

            

        
</div></div><aside data-core-aside="right-rail"><div class="inner-wrapper">

    

        


    

            

            

            <div data-widget-id="dca7470d-9085-4d21-a6f4-a66397718146" data-widget-name="UX3HTMLWidget"><input type="hidden" id="gptPage" value="cell-videos"></div>

            

        











        <div data-widget-def="literatumAd" data-widget-id="481d10c2-bbe0-470e-a5ba-207a81ecc8dc" class="adplaceholders" style="right: 0px; position: static;">
        
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd alignCenter  widget-verticalAd" id="481d10c2-bbe0-470e-a5ba-207a81ecc8dc">
                
                
                    
                

                <div class="wrapped " id="Marketplace-Recs">
                    
                    <div class="widget-body body body-verticalAd "><div class="verticalAdContainer">
    <div class="verticalAdLabelContainer">
        <div class="verticalAdLabel">
                <img src="/templates/jsp/_style2/_marlin/images/bg_adVert.gif" alt="Advertisement">
        </div>
    </div>
    <div class="verticalAd">
        





        <div class="pb-ad">
            <script src="//cdn.pbgrd.com/core-cell.js" async=""></script>
<div id="pbgrd-skyr-c"><div id="pbgrd-skyr" style="width: inherit; position: relative; background: transparent;"></div></div>
        </div>
    

    </div>
</div></div>
                </div>
                </div>

        

        </div>
    

</div></aside><div class="core-authors-details" style="display:none"><section class="core-authors"></section></div></article><div data-extent="article-wrapper"><div class="core-container"></div></div><script id="axel-publication-metadata" type="application/json">{"doi":"10.1016/j.cell.2020.05.049","data-is-retraction":false,"data-is-retired":false}
</script><script type="text/javascript" data-ot-ignore="data-ot-ignore" defer="defer" src="https://static.addtoany.com/menu/page.js" class="optanon-category-C0004 ot-vscat-C0004"></script><template id="figure_nav_template"><a href="#" class="open-in-viewer btn btn-dark figure-pop-btn"><i aria-hidden="true" class="icon-export_citation"></i><span>Open in viewer</span></a></template><div id="figure-viewer" role="dialog" aria-modal="true" aria-label="Figure Viewer" class="showCaption showThumbstrip figure-viewer"><nav class="nav-bar figure-viewer__top"><a href="#" title="Close Figure Viewer" role="button" class="figure-viewer__close"><i aria-hidden="true" class="icon-tools_close"></i></a><div class="figure-viewer__title">TBL1XR1 Mutations Drive Extranodal Lymphoma by Inducing a Pro-tumorigenic Memory Fate</div></nav><div class="frames"><ul class="frames__list"><li><div class="thumbnail__image"><img id="gr1" src="/cms/10.1016/j.cell.2020.05.049/asset/ba82fc3d-99c6-4ccc-a053-85147c162594/main.assets/gr1.jpg" height="941" width="671" aria-labelledby="fig1-title fig1-content" data-viewer-src="/cms/10.1016/j.cell.2020.05.049/asset/da28a7b4-eb3c-4ef8-a301-20c9c6e93bf9/main.assets/gr1_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs1" src="/cms/10.1016/j.cell.2020.05.049/asset/887da720-0b0a-45b0-b2da-60e17e50e74e/main.assets/figs1.jpg" height="903" width="639" aria-labelledby="figs1-title figs1-content" data-viewer-src="/cms/10.1016/j.cell.2020.05.049/asset/f1df23bf-f28b-4de7-bda3-aee2919dd160/main.assets/figs1_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr2" src="/cms/10.1016/j.cell.2020.05.049/asset/e707e4a6-c241-4140-ba22-f22d6f9923e3/main.assets/gr2.jpg" height="743" width="765" aria-labelledby="fig2-title fig2-content" data-viewer-src="/cms/10.1016/j.cell.2020.05.049/asset/bcffcc65-cfc5-4d55-abcb-fa166d278d30/main.assets/gr2_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs2" src="/cms/10.1016/j.cell.2020.05.049/asset/f1708978-4022-497e-8836-7c1de5107ea6/main.assets/figs2.jpg" height="950" width="662" aria-labelledby="figs2-title figs2-content" data-viewer-src="/cms/10.1016/j.cell.2020.05.049/asset/4ef57c5a-a5d7-4cdd-acce-01d51508b91f/main.assets/figs2_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr3" src="/cms/10.1016/j.cell.2020.05.049/asset/30b177b1-f775-42e3-8c66-19ad2923efd4/main.assets/gr3.jpg" height="905" width="770" aria-labelledby="fig3-title fig3-content" data-viewer-src="/cms/10.1016/j.cell.2020.05.049/asset/b54dce69-f269-424f-b214-3d3a48f3c801/main.assets/gr3_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs3" src="/cms/10.1016/j.cell.2020.05.049/asset/961f22d5-e0b9-497b-b934-e531fae97422/main.assets/figs3.jpg" height="950" width="679" aria-labelledby="figs3-title figs3-content" data-viewer-src="/cms/10.1016/j.cell.2020.05.049/asset/afac9c27-c17c-448e-a9f8-0eb0599dce89/main.assets/figs3_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr4" src="/cms/10.1016/j.cell.2020.05.049/asset/4e2f7579-d2f0-4ae5-8b67-1ea3503f9926/main.assets/gr4.jpg" height="949" width="699" aria-labelledby="fig4-title fig4-content" data-viewer-src="/cms/10.1016/j.cell.2020.05.049/asset/c2dede1c-5a88-4682-b735-a6a38821260c/main.assets/gr4_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs4" src="/cms/10.1016/j.cell.2020.05.049/asset/71ecc933-5f53-4b1a-946c-ea0a5e0da314/main.assets/figs4.jpg" height="933" width="722" aria-labelledby="figs4-title figs4-content" data-viewer-src="/cms/10.1016/j.cell.2020.05.049/asset/f40e9970-f3ec-4a31-b4ed-0758978c148e/main.assets/figs4_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr5" src="/cms/10.1016/j.cell.2020.05.049/asset/9ecf4736-5a3c-4c66-a9d9-a9b7b384c4b6/main.assets/gr5.jpg" height="956" width="709" aria-labelledby="fig5-title fig5-content" data-viewer-src="/cms/10.1016/j.cell.2020.05.049/asset/9b288956-64dd-4f2b-92f2-af1de9e1cfbd/main.assets/gr5_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs5" src="/cms/10.1016/j.cell.2020.05.049/asset/66389efa-498c-45ab-a71e-fc0a596b21f2/main.assets/figs5.jpg" height="951" width="693" aria-labelledby="figs5-title figs5-content" data-viewer-src="/cms/10.1016/j.cell.2020.05.049/asset/5eebeb7e-b06f-465c-ad90-1333dabeffc7/main.assets/figs5_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr6" src="/cms/10.1016/j.cell.2020.05.049/asset/875f9e06-0008-4d56-a798-7ec2240d02d6/main.assets/gr6.jpg" height="955" width="749" aria-labelledby="fig6-title fig6-content" data-viewer-src="/cms/10.1016/j.cell.2020.05.049/asset/666ac0b3-90e8-4b35-9ff0-7cfd418642ac/main.assets/gr6_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs6" src="/cms/10.1016/j.cell.2020.05.049/asset/19a3ccd7-1dab-455b-a8af-704fdc1262f2/main.assets/figs6.jpg" height="950" width="677" aria-labelledby="figs6-title figs6-content" data-viewer-src="/cms/10.1016/j.cell.2020.05.049/asset/3933a203-3c9d-4a65-82a2-278429f4e119/main.assets/figs6_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr7" src="/cms/10.1016/j.cell.2020.05.049/asset/e047e0cd-51ea-4a91-a5b2-ecec5777e824/main.assets/gr7.jpg" height="954" width="676" aria-labelledby="fig7-title fig7-content" data-viewer-src="/cms/10.1016/j.cell.2020.05.049/asset/4e3c2e94-03c9-4549-9809-3be31d367987/main.assets/gr7_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs7" src="/cms/10.1016/j.cell.2020.05.049/asset/0a5f9edf-ce98-42a9-b553-c6255571d74e/main.assets/figs7.jpg" height="952" width="636" aria-labelledby="figs7-title figs7-content" data-viewer-src="/cms/10.1016/j.cell.2020.05.049/asset/33e270e2-34ab-4c1b-b15e-a4a467df71b4/main.assets/figs7_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li></ul><a href="#" title="Previous Frame" class="frameButton previousFrameButton frames__prev"><!--span Previous Frame--><i class="icon-arrow_l"></i></a><a href="#" title="Next Frame" class="frameButton nextFrameButton frames__next"><!--span Next Frame--><i class="icon-arrow_r"></i></a></div><div class="caption"><div class="figure-viewer__caption-body"></div></div><div class="toolbar figure-viewer__toolbar"><span class="figure-viewer__counter"></span><a href="#" class="button toggle-caption-button figure-viewer__caption__ctrl"><i aria-hidden="true" class="icon icon-info"></i><span class="label">Hide Caption</span></a><a href="#" class="button download-options-button"><i aria-hidden="true" class="icon icon-gizmo-folder-in download-size-icon"></i><span class="label">Download</span></a><a href="#figures" class="button see-figures figure-viewer__see-figure__ctrl"><i aria-hidden="true" class="icon icon-gizmo-image_figure viewer-size-icon"></i><span class="label">See figure in Article</span></a></div><div class="thumbstrip"><a href="#" class="toggleButton thumbstrip__ctrl"><span>Toggle Thumbstrip</span></a><a href="#" title="Previous Thumbnails" class="thumbstripButton previousThumbstripButton thumbstrip__prev"><i aria-hidden="true" class="icon-arrow_l"></i></a><a href="#" title="Next Thumbnails" class="thumbstripButton nextThumbstripButton thumbstrip__next"><i aria-hidden="true" class="icon-arrow_r"></i></a><div class="thumbstrip__wrapper"><ul class="rlist figure-viewer__thumbslist"><li><div class="thumbnail__label">Figure&nbsp;1</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr1" src="/cms/10.1016/j.cell.2020.05.049/asset/ba82fc3d-99c6-4ccc-a053-85147c162594/main.assets/gr1.jpg" height="941" width="671" aria-labelledby="fig1-title fig1-content" data-viewer-src="/cms/10.1016/j.cell.2020.05.049/asset/da28a7b4-eb3c-4ef8-a301-20c9c6e93bf9/main.assets/gr1_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S1</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs1" src="/cms/10.1016/j.cell.2020.05.049/asset/887da720-0b0a-45b0-b2da-60e17e50e74e/main.assets/figs1.jpg" height="903" width="639" aria-labelledby="figs1-title figs1-content" data-viewer-src="/cms/10.1016/j.cell.2020.05.049/asset/f1df23bf-f28b-4de7-bda3-aee2919dd160/main.assets/figs1_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;2</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr2" src="/cms/10.1016/j.cell.2020.05.049/asset/e707e4a6-c241-4140-ba22-f22d6f9923e3/main.assets/gr2.jpg" height="743" width="765" aria-labelledby="fig2-title fig2-content" data-viewer-src="/cms/10.1016/j.cell.2020.05.049/asset/bcffcc65-cfc5-4d55-abcb-fa166d278d30/main.assets/gr2_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S2</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs2" src="/cms/10.1016/j.cell.2020.05.049/asset/f1708978-4022-497e-8836-7c1de5107ea6/main.assets/figs2.jpg" height="950" width="662" aria-labelledby="figs2-title figs2-content" data-viewer-src="/cms/10.1016/j.cell.2020.05.049/asset/4ef57c5a-a5d7-4cdd-acce-01d51508b91f/main.assets/figs2_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;3</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr3" src="/cms/10.1016/j.cell.2020.05.049/asset/30b177b1-f775-42e3-8c66-19ad2923efd4/main.assets/gr3.jpg" height="905" width="770" aria-labelledby="fig3-title fig3-content" data-viewer-src="/cms/10.1016/j.cell.2020.05.049/asset/b54dce69-f269-424f-b214-3d3a48f3c801/main.assets/gr3_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S3</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs3" src="/cms/10.1016/j.cell.2020.05.049/asset/961f22d5-e0b9-497b-b934-e531fae97422/main.assets/figs3.jpg" height="950" width="679" aria-labelledby="figs3-title figs3-content" data-viewer-src="/cms/10.1016/j.cell.2020.05.049/asset/afac9c27-c17c-448e-a9f8-0eb0599dce89/main.assets/figs3_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;4</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr4" src="/cms/10.1016/j.cell.2020.05.049/asset/4e2f7579-d2f0-4ae5-8b67-1ea3503f9926/main.assets/gr4.jpg" height="949" width="699" aria-labelledby="fig4-title fig4-content" data-viewer-src="/cms/10.1016/j.cell.2020.05.049/asset/c2dede1c-5a88-4682-b735-a6a38821260c/main.assets/gr4_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S4</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs4" src="/cms/10.1016/j.cell.2020.05.049/asset/71ecc933-5f53-4b1a-946c-ea0a5e0da314/main.assets/figs4.jpg" height="933" width="722" aria-labelledby="figs4-title figs4-content" data-viewer-src="/cms/10.1016/j.cell.2020.05.049/asset/f40e9970-f3ec-4a31-b4ed-0758978c148e/main.assets/figs4_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;5</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr5" src="/cms/10.1016/j.cell.2020.05.049/asset/9ecf4736-5a3c-4c66-a9d9-a9b7b384c4b6/main.assets/gr5.jpg" height="956" width="709" aria-labelledby="fig5-title fig5-content" data-viewer-src="/cms/10.1016/j.cell.2020.05.049/asset/9b288956-64dd-4f2b-92f2-af1de9e1cfbd/main.assets/gr5_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S5</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs5" src="/cms/10.1016/j.cell.2020.05.049/asset/66389efa-498c-45ab-a71e-fc0a596b21f2/main.assets/figs5.jpg" height="951" width="693" aria-labelledby="figs5-title figs5-content" data-viewer-src="/cms/10.1016/j.cell.2020.05.049/asset/5eebeb7e-b06f-465c-ad90-1333dabeffc7/main.assets/figs5_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;6</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr6" src="/cms/10.1016/j.cell.2020.05.049/asset/875f9e06-0008-4d56-a798-7ec2240d02d6/main.assets/gr6.jpg" height="955" width="749" aria-labelledby="fig6-title fig6-content" data-viewer-src="/cms/10.1016/j.cell.2020.05.049/asset/666ac0b3-90e8-4b35-9ff0-7cfd418642ac/main.assets/gr6_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S6</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs6" src="/cms/10.1016/j.cell.2020.05.049/asset/19a3ccd7-1dab-455b-a8af-704fdc1262f2/main.assets/figs6.jpg" height="950" width="677" aria-labelledby="figs6-title figs6-content" data-viewer-src="/cms/10.1016/j.cell.2020.05.049/asset/3933a203-3c9d-4a65-82a2-278429f4e119/main.assets/figs6_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;7</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr7" src="/cms/10.1016/j.cell.2020.05.049/asset/e047e0cd-51ea-4a91-a5b2-ecec5777e824/main.assets/gr7.jpg" height="954" width="676" aria-labelledby="fig7-title fig7-content" data-viewer-src="/cms/10.1016/j.cell.2020.05.049/asset/4e3c2e94-03c9-4549-9809-3be31d367987/main.assets/gr7_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S7</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs7" src="/cms/10.1016/j.cell.2020.05.049/asset/0a5f9edf-ce98-42a9-b553-c6255571d74e/main.assets/figs7.jpg" height="952" width="636" aria-labelledby="figs7-title figs7-content" data-viewer-src="/cms/10.1016/j.cell.2020.05.049/asset/33e270e2-34ab-4c1b-b15e-a4a467df71b4/main.assets/figs7_lrg.jpg" loading="lazy" tabindex="0"></div></div></li></ul></div></div><div class="download figure-viewer__download"><div class="pointer"></div><ul class="rlist downlad__wrapper"><li class="hidden-xs hidden-sm"><a href="/action/downloadFigures?pii=e00489&amp;id=" class="downloadAllButton"><i class="icon icon-gizmo-ppt figure-viewer__download__viewer-size-icon figure-viewer__download__viewer-size-icon--ppt"></i><span class="figure-viewer__download__text"> Download .PPT</span></a></li></ul></div></div><template id="toCitationLink" data-citation="Citation" data-footnote="Footnote"><div class="to-citation__wrapper"><a class="to-citation"><i aria-hidden="true" class="icon-subdirectory_arrow_left"></i><span>Go to</span></a></div></template><template id="toCitationButton" data-citation="Citation" data-footnote="Footnote"><div class="to-citation__wrapper"><button class="to-citation"><i aria-hidden="true" class="icon-subdirectory_arrow_left"></i><span>Go to</span></button></div></template><template id="toCitationAccordion" data-collapse-citation="Hide Citations" data-expand-citation="Show Citations" data-collapse-footnote="Hide Footnotes" data-expand-footnote="Show Footnotes"><div class="to-citation__wrapper"><button aria-controls="aria-controls" aria-expanded="false" aria-label="Toggle citations menu" class="to-citation__toggle"><i aria-hidden="true" class="icon-plus"></i><span data-expand-title="data-expand-title" data-collapsed-title="data-collapsed-title" class="accordion__toggle__title"></span></button><div role="menu" aria-label="links" class="to-citation__accordion no-separator"><ul></ul></div></div></template><template id="toCitationAccordionItem"><li><a class="to-citation"><i aria-hidden="true" class="icon-subdirectory_arrow_left"></i><span></span></a></li></template><template id="citations_truncate_template"><div class="citations-truncation"><button data-label-expand="Show all references" data-label-collapse="Show fewer" data-label-remaining="references remaining" class="btn btn--inverse"><span>Show all references</span></button></div></template><template id="collateral_texts_template" data-do="Digital Object" data-figures="Figures" data-metrics="Article Metrics" data-related="Related articles" data-supplementary="Supplemental information"></template><template id="collapsible_authors_template"><button aria-expanded="false" data-expandable="all" data-label-expand="Expand All" data-label-collapse="Collapse All" class="collateral-contributors-control"><span>Expand All</span></button></template>
<template id="collapsible_tables_collapse_template"><div class="collapsible-figure-btn__wrapper expanded"><button aria-expanded="true" class="btn collapsible-figure-btn btn--inverse"><i aria-hidden="true" class="icon-arrow-up"></i><span class="text-uppercase">Collapse</span></button></div></template>
<template id="collapsible_tables_expand_template"><div class="collapsible-figure-btn__wrapper collapsed"><button aria-expanded="false" class="btn collapsible-figure-btn btn--inverse"><i aria-hidden="true" class="icon-arrow-down"></i><span class="text-uppercase">Expand Table</span></button></div></template><template id="authorsAffiliationsLink"><a href="#tab-contributors" class="to-authors-affiliations">Authors Info &amp; Affiliations</a></template></div>
                </div>
                </div>

        

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none  widget-  widget-compact-all" id="756cf18d-e910-4468-936c-0225ae7c669f">
                
                
                    
                

                <div class="wrapped " id="ad for nucleus-single-molecule-technologies">
                    
                    <div class="widget-body body body-  body-compact-all"><div class="pb-ad">
            <script type="text/javascript">
URL_var = 'https://www.cell.com/cell/fulltext/S0092-8674(20)30682-6';

if(URL_var.indexOf("S0165-6147(17)30205-5") > -1
|| URL_var.indexOf("S0167-7799(18)30204-X") > -1 
|| URL_var.indexOf("S0092-8674(18)31649-0") > -1 
|| URL_var.indexOf("S0092-8674(19)30404-0") > -1 
|| URL_var.indexOf("S1097-2765(18)30756-1") > -1 
|| URL_var.indexOf("S0092-8674(18)31452-1") > -1 
|| URL_var.indexOf("S1097-2765(19)30175-3") > -1 
|| URL_var.indexOf("S0092-8674(18)30852-3") > -1 
|| URL_var.indexOf("S0006-3495(18)30295-9") > -1 
|| URL_var.indexOf("S1097-2765(19)30063-2") > -1 
|| URL_var.indexOf("S0006-3495(17)33208-3") > -1 
|| URL_var.indexOf("S2211-1247(18)31639-5") > -1 
|| URL_var.indexOf("S1097-2765(18)30978-X") > -1 
)
    {
document.write('<input type="hidden" id="gpttc" value="nucleus-single-molecule-technologies" class="gpt-Target">')
}
</script>
        </div></div>
                </div>
                </div>

        

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none  widget-" id="754b1780-1117-4371-9285-9cd7020963d5">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body- "><div class="pb-ad">
            <script type="text/javascript">
URL_var = 'https://www.cell.com/cell/fulltext/S0092-8674(20)30682-6';

if(URL_var.indexOf("S1535-6108(19)30104-7") > -1
|| URL_var.indexOf("S0092-8674(18)31394-1") > -1
|| URL_var.indexOf("S1074-7613(19)30033-0") > -1
|| URL_var.indexOf("S1535-6108(18)30473-2") > -1
|| URL_var.indexOf("S0092-8674(19)30154-0") > -1
|| URL_var.indexOf("S1550-4131(18)30178-5") > -1
|| URL_var.indexOf("S1535-6108(19)30242-9") > -1
|| URL_var.indexOf("S1074-7613(18)30431-X") > -1
|| URL_var.indexOf("S0092-8674(19)30618-X") > -1
|| URL_var.indexOf("S1535-6108(18)30472-0") > -1
|| URL_var.indexOf("S1535-6108(18)30120-X") > -1
|| URL_var.indexOf("S0092-8674(18)31247-9") > -1
|| URL_var.indexOf("S1074-7613(18)30085-2") > -1
|| URL_var.indexOf("S1074-7613(18)30080-3") > -1
|| URL_var.indexOf("S1471-4906(18)30210-2") > -1
|| URL_var.indexOf("S1550-4131(19)30304-3") > -1
|| URL_var.indexOf("S1934-5909(18)30599-X") > -1
|| URL_var.indexOf("S2405-8033(19)30024-X") > -1
|| URL_var.indexOf("S1535-6108(18)30109-0") > -1
|| URL_var.indexOf("S0962-8924(18)30127-2") > -1
)
    {
document.write('<input type="hidden" id="gpttc" value="nucleus-tumor-immune-microenvironment" class="gpt-Target">')
}
</script>
        </div></div>
                </div>
                </div>

        

    

        


    

            

            

            <div data-widget-id="9c9ff7a1-a035-49f6-a336-600c9419c518" data-widget-name="UX3HTMLWidget"><!-- Start of HubSpot Embed Code --> 

<script type="text/plain" class="optanon-category-4" id="hs-script-loader" async="" defer="" src="//js.hs-scripts.com/519224.js"></script> 

<!-- End of HubSpot Embed Code --></div>

            

        

    

        


    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget sessionValidationWidget none  widget-none  widget-compact-all" id="0257aef7-6074-4898-a989-1c3f693225a2">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none  body-compact-all"><script>
    $.ajax({
        url: "https://www.cell.com/ssoSessionCheck",
        type: "GET",
        dataType: "jsonp",
        success: function (data) {
            if(data.RefreshSession) {
                location.reload();
            }
        },
        error : function (){
            console.warn("can't validate CSAS session.");
        }
    });
</script></div>
                </div>
                </div>

        

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget csasSsoUrls none  widget-none  widget-compact-all" id="948bc5c4-8b0d-47be-b5ad-6570559802d1">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none  body-compact-all"><!-- clear SSO URLs after all iFrames are rendered or after passing a time out-->
<script>
    setTimeout(clearSsoUrlsFun,10000);
    var iFramesList = document.getElementById("sso-urls-div").getElementsByTagName("iframe");
    var renderedIFramesCount = 0;
    var numberOfIFrames = iFramesList.length;
    var skipRefresh = false;
    for (var i = 0; i < iFramesList.length; i++) {
        var iFrame = iFramesList[i];
        iFrame.addEventListener("load", function() {
            renderedIFramesCount = renderedIFramesCount + 1;
            if (renderedIFramesCount >= numberOfIFrames)
            {
                clearSsoUrlsFun();
            }
        });
    }

    function clearSsoUrlsFun() {
        if (skipRefresh == false && numberOfIFrames > 0)
        {
            skipRefresh = true;
            $.ajax({
                url: "https://www.cell.com/action/clearSsoUrls",
                type: "GET",
                dataType: "jsonp",
                error : errorCallingAjx
            });
        }
    }

    function errorCallingAjx(){
        console.log("can't delete sso urls .")
    }
</script></div>
                </div>
                </div>

        










        <div data-widget-def="ux3-layout-widget" data-widget-id="d95b4a46-dc4d-4ca8-83f9-8b6724c649ec" class="ux3 ">
        
    

        


    

            

            

            <div data-widget-id="d95b4a46-dc4d-4ca8-83f9-8b6724c649ec" data-widget-name="ux3-layout-widget" class="fonts-loaded"><div class="footer clearfix">
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none adplaceholder adplaceholder--header widget-horizontalAd" id="a07ee92b-3942-49f8-9186-a2a08fc3fc56">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-horizontalAd "><div class="horizontalAdContainer">
    <div class="horizontalAdLabelContainer">
        <div class="horizontalAdLabel">
                <img src="/templates/jsp/_style2/_marlin/images/bg_adHoriz.gif" alt="Advertisement">
        </div>
    </div>
    <div class="horizontalAd">
        





        <div class="pb-ad">
            <!-- Beginning Async AdSlot 4 for Ad unit els.cellpress/cell  ### size: [[728,90]] -->
<div class="gpt-ad" data-adsize="cell-leaderboard" data-page="" data-site="cell" data-pos="4"></div>
<!-- End AdSlot 4 -->
        </div>
    

    </div>
</div></div>
                </div>
                </div>

        

    

        


    

            

            

            <div data-widget-id="23703bbc-af46-4f3f-a5bb-dd52891ff0ff" data-widget-name="UX3HTMLWidget"><div class="footer__top">
    <div class="footer__logo centered"><a href="/home" title="Cell Press"><!-- Generator: Adobe Illustrator 25.2.1, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 300 72" style="enable-background:new 0 0 300 72;" xml:space="preserve">
<style type="text/css">
	.st0{fill:#007DBC;}
</style>
<g>
	<path class="st0" d="M39.4,7.5c-2.3-0.3-4.4,1.1-4.9,3.3l-1.8,7.4c-0.4,1.8-1.9,3.1-3.8,3.3l-10.5,1.2C7.1,24-1,34.3,0.7,45.6
		c1.3,9,8.6,15.7,17.2,17c2.3,0.3,4.4-1.1,4.9-3.3l1.8-7.4c0.4-1.8,1.9-3.1,3.8-3.3l10.5-1.2c11.3-1.2,19.4-11.6,17.8-22.9
		C55.3,15.4,48.1,8.8,39.4,7.5z M37.1,34l-6.6,0.8c-1.1,0.1-2,0.9-2.3,2l-1.9,8.1c-0.6,2.4-2.5,4.2-5,4.5c-3.8,0.4-7.1-2.4-7.4-6.2
		c-0.3-3.7,2.6-6.8,6.3-7.2l6.6-0.8c1.1-0.1,2-0.9,2.3-2l1.9-8.1c0.6-2.4,2.5-4.2,5-4.5c3.8-0.4,7.1,2.4,7.4,6.2
		C43.7,30.5,40.8,33.6,37.1,34z"></path>
	<rect x="153.1" y="13.9" class="st0" width="8.2" height="42.3"></rect>
	<path class="st0" d="M90.2,20.4c7.3,0,8.6,4.8,9.4,7.3h8.8c-0.5-6.2-5.6-14.9-18.2-14.9c-10.9,0-19.4,7.5-19.4,22.3
		c0,14.6,8.1,22.2,19.3,22.2c10.6,0,16.6-6.4,18.4-15.6h-8.8c-1.1,5.2-4.5,7.9-9.4,7.9c-7.5,0-10.5-6.8-10.5-14.4
		C79.7,23.3,85.9,20.4,90.2,20.4z"></path>
	<path class="st0" d="M125.2,24c-13.7,0-15,12.1-15,15.8c0,12.9,6.8,17.1,15.6,17.1c5.4,0,11.6-2.5,13.8-10H131
		c-0.7,2.2-3,3.4-5.3,3.4c-6.6,0-7.1-5.3-7.3-7.7h21.5v-1.5C139.9,27,131.3,24,125.2,24z M118.6,37.4c0.5-4.1,2.5-6.6,6.4-6.6
		c2.6,0,5.9,1.5,6.4,6.6H118.6z"></path>
	<rect x="142" y="13.9" class="st0" width="8.2" height="42.3"></rect>
	<path class="st0" d="M174,13.8h-9.8V21h7.2c4.3,0,8.2,0.9,8.2,6.4c0,5.4-3.9,6.4-8.2,6.4h-7.2V41h9.8c10.6,0,14.7-6.7,14.7-13.6
		C188.7,20.5,184.6,13.8,174,13.8z"></path>
	<path class="st0" d="M210.5,24.9c-0.5-0.2-1.2-0.3-1.8-0.3c-4.1,0-8.3,2.7-9.8,6.5h-0.1v-5.7h-8v30.7h8.4V42.3
		c0-5.4,2.1-9.9,8.3-9.9c1,0,2.3,0.1,3,0.3V24.9z"></path>
	<path class="st0" d="M225.6,24.6c-9.3,0-15.7,7-15.7,16.2c0,9.5,6,16.1,15.7,16.1c6.9,0,12-3.1,14.4-10.3h-7.4
		c-0.5,1.9-3.3,4-6.6,4c-4.7,0-7.3-2.4-7.5-7.8h22.1C241.1,33.4,236,24.6,225.6,24.6z M218.4,37.5c0.1-2.4,1.7-6.5,7-6.5
		c4.1,0,5.9,2.3,6.7,6.5H218.4z"></path>
	<path class="st0" d="M260.3,37.7c-4.8-1.1-9.6-1.5-9.6-4.6c0-2.5,2.8-2.9,4.7-2.9c3,0,5.5,0.9,5.8,4.2h8c-0.7-7.5-6.8-9.8-13.5-9.8
		c-6.6,0-13.4,2-13.4,9.9c0,5.4,4.9,7.1,9.7,8.2c6.1,1.3,9.6,2.2,9.6,4.8c0,3-3.1,3.9-5.5,3.9c-3.3,0-6.4-1.5-6.5-5.2h-8
		C242,54.3,248.9,57,256,57c7,0,14-2.6,14-10.7C270.1,40.6,265.2,38.8,260.3,37.7z"></path>
	<path class="st0" d="M289.8,37.7c-4.8-1.1-9.6-1.5-9.6-4.6c0-2.5,2.8-2.9,4.7-2.9c3,0,5.5,0.9,5.8,4.2h8c-0.7-7.5-6.8-9.8-13.5-9.8
		c-6.6,0-13.4,2-13.4,9.9c0,5.4,4.9,7.1,9.7,8.2c6.1,1.3,9.6,2.2,9.6,4.8c0,3-3.1,3.9-5.5,3.9c-3.3,0-6.4-1.5-6.5-5.2h-8
		c0.4,8.1,7.3,10.8,14.4,10.8c7,0,14-2.6,14-10.7C299.6,40.6,294.7,38.8,289.8,37.7z"></path>
	<path class="st0" d="M34.1,57.6c0.3-0.2,0.4-0.6,0.4-1c0-0.5-0.2-0.9-0.5-1.1c-0.3-0.2-0.8-0.3-1.4-0.3h-1.9v5.1h0.8V58h0.8
		l1.3,2.2h0.9l-1.4-2.2C33.6,57.9,33.9,57.8,34.1,57.6z M32.5,57.3h-0.9v-1.5h1c0.1,0,0.3,0,0.4,0c0.1,0,0.3,0,0.4,0.1
		c0.1,0,0.2,0.1,0.3,0.2c0.1,0.1,0.1,0.2,0.1,0.4c0,0.2,0,0.3-0.1,0.4c-0.1,0.1-0.2,0.2-0.3,0.2c-0.1,0.1-0.2,0.1-0.4,0.1
		C32.8,57.3,32.6,57.3,32.5,57.3z"></path>
	<path class="st0" d="M35.6,54.2c-0.4-0.4-0.9-0.7-1.4-0.9c-0.5-0.2-1.1-0.3-1.7-0.3c-0.6,0-1.2,0.1-1.7,0.3c-0.6,0.2-1,0.5-1.4,0.9
		c-0.4,0.4-0.7,0.9-1,1.5c-0.2,0.6-0.4,1.2-0.4,1.9c0,0.7,0.1,1.3,0.3,1.8c0.2,0.6,0.5,1,0.9,1.5c0.4,0.4,0.9,0.7,1.4,1
		c0.6,0.2,1.2,0.4,1.8,0.4c0.6,0,1.2-0.1,1.7-0.3c0.5-0.2,1-0.5,1.4-0.9c0.4-0.4,0.7-0.9,1-1.5c0.2-0.6,0.4-1.2,0.4-1.9
		c0-0.7-0.1-1.3-0.4-1.9C36.4,55.1,36,54.6,35.6,54.2z M35.7,59.1c-0.2,0.5-0.4,0.9-0.8,1.2c-0.3,0.3-0.7,0.6-1.1,0.8
		c-0.4,0.2-0.9,0.3-1.4,0.3c-0.6,0-1-0.1-1.5-0.3c-0.4-0.2-0.8-0.5-1.1-0.8c-0.3-0.3-0.5-0.7-0.7-1.2c-0.2-0.5-0.2-1-0.2-1.5
		c0-0.6,0.1-1.1,0.3-1.5c0.2-0.5,0.4-0.9,0.8-1.2c0.3-0.3,0.7-0.6,1.1-0.8c0.4-0.2,0.9-0.3,1.4-0.3c0.5,0,1,0.1,1.4,0.3
		c0.4,0.2,0.8,0.4,1.1,0.8c0.3,0.3,0.6,0.7,0.8,1.2c0.2,0.5,0.3,1,0.3,1.5C36,58.1,35.9,58.6,35.7,59.1z"></path>
</g>
</svg></a></div>
    <div class="footer__social-links">
        <ul class="rlist rlist--inline centered">
            <li><a href="http://on.fb.me/1tgHakh" title="Facebook" target="_blank"><i aria-hidden="true" class="icon-Icon_Facebook"></i></a></li>

            <li><a href="https://www.instagram.com/cellpress/" title="Instagram" target="_blank"><svg width="18px" height="18px" viewBox="0 0 18 18" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-147.000000, -16.000000)" fill-rule="nonzero" fill="#969696"> <g id="Instagram-Logo" transform="translate(147.000000, 16.000000)"> <path d="M9,1.62580645 C11.4096774,1.62580645 11.7,1.62580645 12.6290323,1.68387097 C13.5,1.71290323 13.9935484,1.85806452 14.3129032,2.00322581 C14.7193548,2.17741935 15.0387097,2.3516129 15.3580645,2.67096774 C15.6774194,2.99032258 15.8806452,3.28064516 16.0258065,3.71612903 C16.1419355,4.03548387 16.2870968,4.5 16.3451613,5.4 C16.3741935,6.35806452 16.4032258,6.61935484 16.4032258,9.02903226 C16.4032258,11.4387097 16.4032258,11.7290323 16.3451613,12.6580645 C16.316129,13.5290323 16.1709677,14.0225806 16.0258065,14.3419355 C15.8516129,14.7483871 15.6774194,15.0677419 15.3580645,15.3870968 C15.0387097,15.7064516 14.7483871,15.9096774 14.3129032,16.0548387 C13.9935484,16.1709677 13.5290323,16.316129 12.6290323,16.3741935 C11.6709677,16.4032258 11.4096774,16.4322581 9,16.4322581 C6.59032258,16.4322581 6.3,16.4322581 5.37096774,16.3741935 C4.5,16.3451613 4.00645161,16.2 3.68709677,16.0548387 C3.28064516,15.8806452 2.96129032,15.7064516 2.64193548,15.3870968 C2.32258065,15.0677419 2.11935484,14.7774194 1.97419355,14.3419355 C1.85806452,14.0225806 1.71290323,13.5580645 1.65483871,12.6580645 C1.62580645,11.7 1.62580645,11.4096774 1.62580645,9 C1.62580645,6.59032258 1.62580645,6.3 1.68387097,5.37096774 C1.71290323,4.5 1.85806452,4.00645161 2.00322581,3.68709677 C2.17741935,3.28064516 2.3516129,2.96129032 2.67096774,2.64193548 C2.99032258,2.32258065 3.28064516,2.11935484 3.71612903,1.97419355 C4.03548387,1.85806452 4.5,1.71290323 5.4,1.65483871 C6.3,1.62580645 6.59032258,1.62580645 9,1.62580645 L9,1.62580645 Z M9,0 C6.56129032,0 6.24193548,0 5.28387097,0.0580645161 C4.32580645,0.116129032 3.65806452,0.261290323 3.10645161,0.464516129 C2.52580645,0.696774194 2.00322581,1.01612903 1.50967742,1.50967742 C1.01612903,2.00322581 0.696774194,2.52580645 0.464516129,3.10645161 C0.232258065,3.68709677 0.0870967742,4.32580645 0.0580645161,5.28387097 C0,6.24193548 0,6.56129032 0,9 C0,11.4387097 0,11.7580645 0.0580645161,12.716129 C0.116129032,13.6741935 0.261290323,14.3419355 0.464516129,14.8935484 C0.696774194,15.4741935 1.01612903,15.9967742 1.50967742,16.4903226 C2.00322581,16.983871 2.52580645,17.3032258 3.10645161,17.5354839 C3.68709677,17.7677419 4.32580645,17.9129032 5.28387097,17.9419355 C6.24193548,18 6.56129032,18 9,18 C11.4387097,18 11.7580645,18 12.716129,17.9419355 C13.6741935,17.883871 14.3419355,17.7387097 14.8935484,17.5354839 C15.4741935,17.3032258 15.9967742,16.983871 16.4903226,16.4903226 C16.983871,15.9967742 17.3032258,15.4741935 17.5354839,14.8935484 C17.7677419,14.3129032 17.9129032,13.6741935 17.9419355,12.716129 C18,11.7580645 18,11.4387097 18,9 C18,6.56129032 18,6.24193548 17.9419355,5.28387097 C17.883871,4.32580645 17.7387097,3.65806452 17.5354839,3.10645161 C17.3032258,2.52580645 16.983871,2.00322581 16.4903226,1.50967742 C15.9967742,1.01612903 15.4741935,0.696774194 14.8935484,0.464516129 C14.3129032,0.232258065 13.6741935,0.0870967742 12.716129,0.0580645161 C11.7580645,0 11.4387097,0 9,0 L9,0 Z"></path> <path d="M9,4.38387097 C6.44516129,4.38387097 4.38387097,6.44516129 4.38387097,9 C4.38387097,11.5548387 6.44516129,13.616129 9,13.616129 C11.5548387,13.616129 13.616129,11.5548387 13.616129,9 C13.616129,6.44516129 11.5548387,4.38387097 9,4.38387097 Z M9,11.9903226 C7.34516129,11.9903226 6.00967742,10.6548387 6.00967742,9 C6.00967742,7.34516129 7.34516129,6.00967742 9,6.00967742 C10.6548387,6.00967742 11.9903226,7.34516129 11.9903226,9 C11.9903226,10.6548387 10.6548387,11.9903226 9,11.9903226 Z"></path> <circle id="Oval" cx="13.7903226" cy="4.20967742" r="1.07419355"></circle> </g> </g> </g></svg></a></li>
            
            <li><a href="https://www.linkedin.com/company/cell-press/" title="LinkedIn" target="_blank"><i aria-hidden="true" class="icon-linkedin"></i></a></li>
            
                        <li><a href="http://bit.ly/1zq7Q5T" title="Twitter" target="_blank">
                <!--<i aria-hidden="true" class="icon-Icon_Twitter"></i></a>-->
                
                <svg id="xicon" viewBox="0 0 1200 1227" xmlns="http://www.w3.org/2000/svg" aria-hidden="true" role="none"><path fill="#969696" d="M714.163 519.284L1160.89 0H1055.03L667.137 450.887L357.328 0H0L468.492 681.821L0 1226.37H105.866L515.491 750.218L842.672 1226.37H1200L714.137 519.284H714.163ZM569.165 687.828L521.697 619.934L144.011 79.6944H306.615L611.412 515.685L658.88 583.579L1055.08 1150.3H892.476L569.165 687.854V687.828Z"></path></svg> 

                    <style>
                    .footer svg#xicon {
                      height: 16px;
                    }
                    </style>
            </a></li>

            
            <li><a href="/pb-assets/marketing/WeChat/WeChat-1557753222727.jpg" title="WeChat" target="_blank"><svg width="18px" height="15px" viewBox="0 0 18 15" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-122.000000, -17.000000)" fill-rule="nonzero" fill="#969696"> <g id="wechat" transform="translate(122.000000, 17.000000)"> <path d="M12.375,4.5 L12.375,4.5 L12.4875,4.5 L12.4875,4.5 C12.6898579,4.48604473 12.8520408,4.32704187 12.87,4.125 C12.8705905,4.04725572 12.8470025,3.9712499 12.8025,3.9075 C11.94,1.5675 9.4425,0 6.5625,0 C2.9475,0 0,2.52 0,5.625 C0.0313772094,7.34410122 0.895513317,8.94093526 2.3175,9.9075 L1.5675,11.46 C1.49009368,11.6087816 1.52088865,11.7908702 1.64291494,11.9059236 C1.76494122,12.0209769 1.94852501,12.0410168 2.0925,11.955 L4.08,10.8225 C4.61861992,11.0129999 5.17724298,11.1413322 5.745,11.205 C5.86564124,11.2178337 5.98503738,11.1715098 6.06545426,11.0806685 C6.14587113,10.9898272 6.17737153,10.865694 6.15,10.7475 C6.06148727,10.4219419 6.01114891,10.0871918 6,9.75 C6,6.855 8.8575,4.5 12.375,4.5 Z M8.25,3 C8.66421356,3 9,3.33578644 9,3.75 C9,4.16421356 8.66421356,4.5 8.25,4.5 C7.83578644,4.5 7.5,4.16421356 7.5,3.75 C7.5,3.33578644 7.83578644,3 8.25,3 Z M4.5,4.5 C4.08578644,4.5 3.75,4.16421356 3.75,3.75 C3.75,3.33578644 4.08578644,3 4.5,3 C4.91421356,3 5.25,3.33578644 5.25,3.75 C5.25,4.16421356 4.91421356,4.5 4.5,4.5 Z"></path> <path d="M18,9.75 C18,7.2675 15.48,5.25 12.375,5.25 C9.27,5.25 6.75,7.2675 6.75,9.75 C6.75,12.2325 9.27,14.25 12.375,14.25 C12.9568557,14.2502837 13.5364258,14.1772075 14.1,14.0325 L15.96,14.9625 C16.0114248,14.9874074 16.0678613,15.0002339 16.125,15 C16.2497193,14.9993787 16.3659682,14.9367912 16.4351508,14.8330172 C16.5043335,14.7292432 16.5173955,14.5978644 16.47,14.4825 L15.96,13.2075 C17.1784092,12.4716325 17.9450305,11.1723221 18,9.75 L18,9.75 Z M10.5,9 C10.0857864,9 9.75,8.66421356 9.75,8.25 C9.75,7.83578644 10.0857864,7.5 10.5,7.5 C10.9142136,7.5 11.25,7.83578644 11.25,8.25 C11.25,8.66421356 10.9142136,9 10.5,9 Z M14.25,9 C13.8357864,9 13.5,8.66421356 13.5,8.25 C13.5,7.83578644 13.8357864,7.5 14.25,7.5 C14.6642136,7.5 15,7.83578644 15,8.25 C15,8.66421356 14.6642136,9 14.25,9 Z" opacity="0.959999979"></path> </g> </g> </g></svg></a></li>
            
            <li><a href="https://www.weibo.com/CellPress" title="Weibo" target="_blank"><svg width="18px" height="15px" viewBox="0 0 18 15" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-99.000000, -17.000000)" fill-rule="nonzero" fill="#969696"> <g id="weibo" transform="translate(99.000000, 14.000000)"> <g transform="translate(0.000000, 3.000000)"> <path d="M13.2402211,7.47054172 C12.9918392,7.39866484 12.821608,7.34902326 12.9514975,7.03290287 C13.232804,6.34602189 13.2622915,5.75359781 12.9571055,5.33095486 C12.3847236,4.5380684 10.8191759,4.58064295 9.02514573,5.30975376 C9.02514573,5.30889193 8.46144724,5.54882627 8.60544724,5.11515185 C8.88150754,4.25435294 8.83953769,3.53386047 8.41043216,3.11759508 C7.43716583,2.17285363 4.84824121,3.15344733 2.62745729,5.3059617 C0.964944724,6.9186238 0,8.62798358 0,10.105855 C0,12.9324952 3.73748744,14.8235294 7.39429146,14.8235294 C12.1877186,14.8235294 15.3768844,11.9501778 15.3768844,9.80576471 C15.3768844,8.50974008 14.2514774,7.77442408 13.2402211,7.47054172 Z M7.40424121,13.5561176 C4.48625126,13.8315595 1.9678794,12.5686293 1.77811055,10.7327524 C1.58870352,8.89773735 3.80098492,7.18579207 6.71861307,6.90948837 C9.63678392,6.63266758 12.1556985,7.89663201 12.3447437,9.73164706 C12.533608,11.5675239 10.3227739,13.2796416 7.40424121,13.5561176 Z M18,5.16875787 C17.9985528,2.31402189 15.5695477,0 12.5728643,0 C12.222995,0 11.9396985,0.270098495 11.9396985,0.603283174 C11.9396985,0.936640219 12.222995,1.20656635 12.5728643,1.20656635 C14.8708945,1.20656635 16.7336683,2.98159781 16.7336683,5.17099863 C16.7336683,5.50435568 17.0173266,5.77428181 17.3668342,5.77428181 C17.7167035,5.77428181 18,5.50435568 18,5.17099863 C18,5.17099863 18,5.1696197 18,5.16875787 Z M15.4841608,5.08016142 C15.1948945,3.73415048 14.0812462,2.67323393 12.6683819,2.39744733 C12.3262915,2.33039672 11.9914372,2.54051163 11.9212462,2.86680164 C11.8506935,3.19309166 12.0712161,3.51179754 12.4133065,3.57919289 C13.3319397,3.75845417 14.0559196,4.44774829 14.2438794,5.32302599 C14.3146131,5.64931601 14.6491055,5.85925855 14.9915578,5.7923803 C15.3338291,5.72498495 15.5541709,5.40627907 15.4841608,5.08016142 Z M5.90761809,8.7069275 C4.7321005,8.9320383 3.93901508,9.87126402 4.1358392,10.8044569 C4.33248241,11.7378222 5.44504523,12.311803 6.62056281,12.0865198 C7.79571859,11.861409 8.58916583,10.922528 8.3921608,9.98899042 C8.19569849,9.0559699 7.08277387,8.48181669 5.90761809,8.7069275 Z"></path> </g> </g> </g> </g></svg></a></li>
            
            <li><a href="https://www.youtube.com/user/cellvideoabstracts" title="YouTube" target="_blank"><svg width="18px" height="13px" viewBox="0 0 18 13" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-76.000000, -19.000000)" fill-rule="nonzero" fill="#969696"> <g id="youtube" transform="translate(76.000000, 14.000000)"> <path d="M17.2984219,6.41974219 C16.6486641,5.64735937 15.4490625,5.33232422 13.1580703,5.33232422 L4.84175391,5.33232422 C2.49834375,5.33232422 1.27838672,5.66771484 0.631089844,6.49001953 C0,7.29172266 0,8.47304297 0,10.1080547 L0,13.2243047 C0,16.3917773 0.748792969,18 4.84175391,18 L13.1581055,18 C15.1448555,18 16.2457383,17.7219844 16.9579688,17.040375 C17.688375,16.3413984 18,15.2001211 18,13.2243047 L18,10.1080547 C18,8.38381641 17.951168,7.19553516 17.2984219,6.41974219 Z M11.5560703,12.0963867 L7.77965625,14.0700586 C7.69524609,14.1141797 7.60289062,14.136082 7.51071094,14.136082 C7.40629687,14.136082 7.30212891,14.1079922 7.20987891,14.0521289 C7.03613672,13.9468711 6.93007031,13.7585391 6.93007031,13.5554414 L6.93007031,9.62075391 C6.93007031,9.41800781 7.03585547,9.22985156 7.20924609,9.12452344 C7.38267187,9.01919531 7.59839062,9.01198828 7.77832031,9.10546875 L11.5547344,11.0664141 C11.7468633,11.1661523 11.8675195,11.3645039 11.8678008,11.5809258 C11.8680469,11.7975586 11.747918,11.9961914 11.5560703,12.0963867 Z"></path> </g> </g> </g></svg></a></li>
            
        </ul>
    </div>
</div>

<style>

.ux3 .footer__logo svg {
    max-width: 160px !important;
}

.ux3 .footer__social-links a svg {
    max-width: 26px;
    height: 1.15rem;
    width: auto;
}

.ux3 .footer__social-links a:hover svg path {
    fill: #353535;
}

</style></div>

            

        

    

        


    

            

            
                <div class="footer__bottom text-onDark">
            

            <div data-widget-id="06227714-c621-4651-9879-b0ce5176cccc" data-widget-name="ux3-layout-widget"><div class="col-sm-4 col-md-5 clearfix">
    

        


    

            

            

            <div data-widget-id="07e1d276-b4dd-4870-b858-ed74e70d64ef" data-widget-name="UX3HTMLWidget"><div class="groups">
	<div class="group">
		<h2 class="group__title">Life &amp; medical sciences journals</h2>
		<ul class="rlist clearfix sided">
		<li>
			<a alt="Cell" href="/cell/home">Cell</a>
		</li>
		<li>
			<a alt="Cancer Cell" href="/cancer-cell/home">Cancer Cell</a>
		</li>
		<li>
			<a alt="Cell Chemical Biology" href="/cell-chemical-biology">Cell Chemical Biology</a>
		</li>
		<li>
			<a alt="Cell Genomics" href="/cell-genomics/home">Cell Genomics</a>
		</li>
		<li>
			<a alt="Cell Host and Microbe" href="/cell-host-microbe/home">Cell Host &amp; Microbe</a>
		</li>
		<li>
			<a alt="Cell Metabolism" href="/cell-metabolism/home">Cell Metabolism</a>
		</li>
		<li>
			<a alt="Cell Reports" href="/cell-reports/home">Cell Reports</a>
		</li>
		<li>
			<a alt="Cell Reports Medicine" href="/cell-reports-medicine/home">Cell Reports Medicine</a>
		</li>
		<li>
			<a alt="Cell Stem Cell" href="/cell-stem-cell/home">Cell Stem Cell</a>
		</li>
		<li>
			<a alt="Cell Systems" href="/cell-systems/home">Cell Systems</a>
		</li>
		<li>
			<a alt="Current Biology" href="/current-biology/home">Current Biology</a>
		</li>
		<li>
			<a alt="Developmental Cell" href="/developmental-cell/home">Developmental Cell</a>
		</li>
		<li>
			<a alt="Immunity" href="/immunity/home">Immunity</a>
		</li>
		<li>
			<a alt="Med" href="/med/home">Med</a>
		</li>
		<li>
			<a alt="Molecular Cell" href="/molecular-cell/home">Molecular Cell</a>
		</li>
		<li>
			<a alt="Neuron" href="/neuron/home">Neuron</a>
		</li>
		<li>
			<a alt="Structure" href="/structure/home">Structure</a>
		</li>
		<li>
			<a alt="American Journal of Human Genetics" href="/ajhg/home">American Journal of Human Genetics (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Biophysical Journal" href="/biophysj/home">Biophysical Journal (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Biophysical Reports" href="/biophysreports/home">Biophysical Reports (<i>partner</i>)</a>
		</li>	
		<li>
			<a alt="HGG Advances" href="/hgg-advances/home">Human Genetics and Genomics Advances (<i>partner</i>)</a></li>
		<li>
			<a alt="Molecular Plant" href="/molecular-plant/home">Molecular Plant (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy" href="/molecular-therapy-family/molecular-therapy/home">Molecular Therapy (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy Methods and Clinical Development" href="/molecular-therapy-family/methods/home">Molecular Therapy Methods &amp; Clinical Development (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy Nucleic Acids" href="/molecular-therapy-family/nucleic-acids/home">Molecular Therapy Nucleic Acids (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy Oncology" href="/molecular-therapy-family/oncology/home">Molecular Therapy Oncology (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Plant Communications" href="/plant-communications/home">Plant Communications (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Stem Cell Reports" href="/stem-cell-reports/home">Stem Cell Reports (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Trends in Biochemical Sciences" href="/trends/biochemical-sciences/home">Trends in Biochemical Sciences</a>
		</li>
		<li>
		</li><li>
			<a alt="Trends in Cancer" href="/trends/cancer/home">Trends in Cancer</a>
		</li>
		<li>
			<a alt="Trends in Cell Biology" href="/trends/cell-biology/home">Trends in Cell Biology</a>
		</li>
		<li>
			<a alt="Trends in Ecology and Evolution" href="/trends/ecology-evolution/home">Trends in Ecology &amp; Evolution</a>
		</li>
		<li>
			<a alt="Trends in Endocrinology and Metabolism" href="/trends/endocrinology-metabolism/home">Trends in Endocrinology &amp; Metabolism</a>
		</li>
		<li>
			<a alt="Trends in Genetics" href="/trends/genetics/home">Trends in Genetics</a>
		</li>
		<li>
			<a alt="Trends in Immunology" href="/trends/immunology/home">Trends in Immunology</a>
		</li>
		<li>
			<a alt="Trends in Microbiology" href="/trends/microbiology/home">Trends in Microbiology</a>
		</li>
		<li>
			<a alt="Trends in Molecular Medicine" href="/trends/molecular-medicine/home">Trends in Molecular Medicine</a>
		</li>
		<li>
			<a alt="Trends in Neurosciences" href="/trends/neurosciences/home">Trends in Neurosciences</a>
		</li>
		<li>
			<a alt="Trends in Parasitology" href="/trends/parasitology/home">Trends in Parasitology</a>
		</li>
		<li>
			<a alt="Trends in Pharmacological Sciences" href="/trends/pharmacological-sciences/home">Trends in Pharmacological Sciences</a>
		</li>
		<li>
			<a alt="Trends in Plant Science" href="/trends/plant-science/home">Trends in Plant Science</a>
		</li>
		</ul>
	</div>

	<div class="group">
		<h2 class="group__title">Physical sciences &amp; engineering journals</h2>
		<ul class="rlist clearfix sided">
		  <li><a alt="Cell Biomaterials" href="/cell-biomaterials/home">Cell Biomaterials</a></li>
		  <li>
			<a alt="Cell Reports Physical Science" href="/cell-reports-physical-science/home">Cell Reports Physical Science</a>
		</li>
		<li>
			<a alt="Chem" href="/chem/home">Chem</a>
		</li>
		<li>
			<a alt="Chem Catalysis" href="/chem-catalysis/home">Chem Catalysis</a>
		</li>
		<li>
			<a alt="Device" href="/device/home">Device</a>
		</li>
		<li>
			<a alt="Joule" href="/joule/home">Joule</a>
		</li>
		<li>
			<a alt="Matter" href="/matter/home">Matter</a>
		</li>	
		<li><a alt="Matter" href="/newton/home">Newton</a></li>
		<li>
			<a alt="Trends in Chemistry" href="/trends/chemistry/home">Trends in Chemistry</a>
		</li>			
		</ul>
	</div>

<div class="group">
	<h2 class="group__title">Multidisciplinary journals</h2>
	<ul class="rlist clearfix sided">
	  		<li>
			<a alt="Cell Reports Methods" href="/cell-reports-methods/home">Cell Reports Methods</a>
		</li>
		<li>
			<a alt="Cell Reports Sustainability" href="/cell-reports-sustainability/home">Cell Reports Sustainability</a>
		</li>
		<li>
			<a alt="Heliyon" href="/heliyon/home">Heliyon</a>
		</li>	
		<li>
			<a alt="iScience" href="/iscience/home">iScience</a>
		</li>
		<li>
			<a alt="One Earth" href="/one-earth/home">One Earth</a>
		</li>
		<li>
			<a alt="Patterns" href="/patterns/home">Patterns</a>
		</li>
		<li>
			<a alt="STAR Protocols" href="/star-protocols/home">STAR Protocols</a>
		</li>
		<li>
			<a alt="Nexus" href="/nexus/home">Nexus (<i>partner</i>)</a>
		</li>			
        <li>
			<a alt="The Innovation" href="/the-innovation/home">The Innovation (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Trends in Biotechnology" href="/trends/biotechnology/home">Trends in Biotechnology</a>
		</li>
		<li>
			<a alt="Trends in Cognitive Sciences" href="/trends/cognitive-sciences/home">Trends in Cognitive Sciences</a>
		</li>
	</ul>
</div>

</div>
</div>

            

        
</div><div class="col-sm-4 col-md-5 clearfix">
    

        


    

            

            

            <div data-widget-id="850476de-00cb-4c0d-943c-bf1881da24f9" data-widget-name="UX3HTMLWidget"><div class="groups">
	<div class="clearfix">
		<div class="group sided">
			<h2 class="group__title">Authors</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Submit article" href="/submit-to-cell-press">Submit article</a>
				</li>
				<li>
					<a alt="Multi-Journal Submission" href="/multi-journal-submission">Multi-Journal Submission</a>
				</li>
				<li>
					<a alt="STAR Methods" href="/star-authors-guide">STAR Methods</a>
				</li>
				<li>
					<a alt="Sneak Peek – Preprints" href="/sneakpeek">Sneak Peek – Preprints</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
		<h2 class="group__title">Reviewers</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Reviewers" href="/reviewers">Information for reviewers</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
		<h2 class="group__title">News &amp; events</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Newsroom" href="/newsroom">Newsroom</a>
				</li>
				<li>
					<a alt="Events" href="/events">Events</a>
				</li>
				<li>
					<a alt="Cell Symposia" href="/symposia">Cell Symposia</a>
				</li>
				<li>
					<a alt="Consortia Hub" href="/consortia">Consortia Hub</a>
				</li>
				<li>
					<a alt="Webinars" href="/webinars">Webinars</a>
				</li>
								<li>
					&nbsp;
				</li>
			</ul>
		<h2 class="group__title">Multimedia</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Cell Press Podcast" href="/podcast">Cell Press Podcast</a>
				</li>
				<li>
					<a alt="Cell Press Videos" href="/video">Cell Press Videos</a>
				</li>
				<li>
					<a alt="Color and Comics" href="/comics">Coloring and Comics</a>
				</li>
				<li>
					<a alt="Figure360" href="/video-guidelines">Figure360</a>
				</li>
				<li>
					<a alt="Cell Picture Show" href="/pictureshow">Cell Picture Show</a>
				</li>
				<li>
					<a alt="Research Arc" href="/research-arc">Research Arc</a>
				</li>
			</ul>
		</div>

		<div class="group sided">
			<h2 class="group__title">About</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="About Cell Press" href="/about">About Cell Press</a>
				</li>

				<li>
					<a alt="Open access" href="/open-access">Open access</a>
				</li>
				<li>
					<a alt="COVID Hub" href="/covid-19">COVID Hub</a>
				</li>
				<li>
					<a alt="Sustainability" href="/sustainability">Sustainability hub</a>
				</li>
				<li>
					<a alt="Inclusion and diversity" href="/diversity">Inclusion and diversity</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
			<h2 class="group__title">Contact</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Contact us" href="/contact">Contact us</a>
				</li>
				<li>
					<a alt="Help and Support" href="https://service.elsevier.com/app/overview/cell-press/" target="_blank">Help &amp; Support</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
			<h2 class="group__title">Careers</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Cell Press Careers" href="https://relx.wd3.myworkdayjobs.com/en-US/ElsevierJobs" target="_blank">Cell Press Careers</a>
				</li>
			</ul>
		</div>
	</div>
</div></div>

            

        
</div><div class="col-sm-4 col-md-2 clearfix">
    

        


    

            

            

            <div data-widget-id="beafac41-1cfc-440b-9b55-74aba4a5612c" data-widget-name="UX3HTMLWidget"><div class="groups">
	<div class="group">
		<h2 class="group__title">Access</h2>
		<ul class="rlist clearfix">
			<li>
				<a alt="Subscribe" href="/subscribe">Subscribe</a>
			</li>
			<li>
				<a alt="Claim" href="/activate">Claim</a>
			</li>
			<li>
				<a alt="Read It Now" href="/read-it-now">Read-It-Now</a>
			</li>
			<li>
				<a alt="Recommend to Librarian" href="http://info.cell.com/request-access-to-cell-press-content" target="_blank">Recommend to Librarian</a>
			</li>
			<li>
				<a alt="Publication Alerts" href="/action/showPreferences?menuTab=Alerts">Publication Alerts</a>
			</li>
		</ul>
	</div>
	<div class="group">
		<h2 class="group__title">Collections</h2>
		<ul class="rlist clearfix">
			<li>
				<a alt="Best of Cell Press" href="/bestof">Best of Cell Press</a>
			</li>
			<li>
				<a alt="Cell Press Reviews" href="/reviews">Cell Press Reviews</a>
			</li>
			<li>
				<a alt="Cell Selections" href="/selections">Cell Press Selections</a>
			</li>
			<li>
				<a alt="Nucleus Collections" href="/nucleus">Nucleus Collections</a>
			</li>
			<li>
				<a alt="SnapShots" href="/snapshots">SnapShot Archive</a>
			</li>
		</ul>
	</div>
	<div class="group">
		<h2 class="group__title">Information</h2>
		<ul class="rlist clearfix">
			<li>
				<a alt="For Advertisters" href="/media-kit">For Advertisers</a>
			</li>
			<li>
				<a alt="For Recruiters" target="_blank" href="/pb-assets/Cell/common/CP-RecruitmentKit-US-2018-LowRes-1712070648.pdf">For Recruiters</a>
			</li>
			<li>
				<a alt="For Librarians" target="_blank" href="https://www.elsevier.com/products/sciencedirect/journals/direct-subscription-journals">For Librarians</a>
			</li>
			<li>
				<a alt="Privacy Policy" target="_blank" href="https://www.elsevier.com/legal/privacy-policy">Privacy Policy</a>
			</li>
			<li>
				<a alt="Terms and Conditions" target="_blank" href="https://www.elsevier.com/legal/elsevier-website-terms-and-conditions">Terms and Conditions</a>
			</li>
			<li>
				<a alt="Accessibility" href="/accessibility">Accessibility</a>
			</li>
		</ul>
	</div>
</div></div>

            

        
</div></div>

            
                </div>
            

        

    

        


    

            

            

            <div data-widget-id="779ad11f-7bcd-4a2a-a2cd-8c67b649cbb2" data-widget-name="UX3HTMLWidget"><style>
 
button#cookieBtn {
   text-decoration: none;
   background-color: transparent;
   color: #005789;
   font-weight: bold;
   padding: 0;
   margin: 0;
   font-size: 14px;
}
 
button#cookieBtn:hover {
   text-decoration: underline;
}

</style>
<script>
// HIDE ONETRUST BACKGROUND AFTER USER PRESSES ESCAPE KEY
setTimeout(function(){ 
   $("#onetrust-pc-sdk").on("keydown", function(e) { 
      if (e.keyCode == 27) {  // ESC key
         setTimeout(function(){ 
            $(".onetrust-pc-dark-filter").css("display","none");
         }, 100); 
      }
   });
}, 5000);
</script>

<span class="align-center copy-right">
<hr>
<p align="center"><b>The content on this site is intended for healthcare professionals and researchers across all fields of science.</b></p>
<hr>
<div>
<p>We use cookies to help provide and enhance our service and tailor content. To update your cookie settings, please visit the 
      <b><button id="cookieBtn" class="ot-sdk-show-settings" type="button" aria-label="Manage cookie consent preferences">Cookie Settings</button></b> for this site.<br>

All content on this site: Copyright © 2025 Elsevier Inc., its licensors, and contributors. <br>All rights are reserved, including those for text and data mining, AI training, and similar technologies. <br>For all open access content, the relevant licensing terms apply.
</p>
<ul class="rlist--inline">
			<li>
				<a alt="Privacy Policy" target="_blank" href="https://www.elsevier.com/legal/privacy-policy">Privacy Policy</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Terms and Conditions" target="_blank" href="https://www.elsevier.com/legal/elsevier-website-terms-and-conditions">Terms &amp; Conditions</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Accessibility" href="/accessibility">Accessibility</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Help and Support" target="_blank" href="https://service.elsevier.com/app/overview/cell-press/">Help &amp; Support</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Contact" href="/contact">Contact</a>
			</li>
			</ul>
<p>
<a href="https://www.relx.com"><img src="/pb/assets/raw/Health%20Advance/images/relx-1554393210983.png" alt="RELX" rel="nofollow" target="_new"></a></p>
</div>

</span></div>

            

        

    

        


    

            

            

            <div data-widget-id="c5e179c1-156e-41e3-af7f-5e307efa8acf" data-widget-name="UX3HTMLWidget"><style>#brandingRow{
    background-color:#fff;
    padding:7px 26px 7px 30px;
    color:#007DBC;
    font-size:14px;
    clear:both;
    width:100%
}
#brandingRow a{
    text-decoration:none;
    color:#007DBC
}
#brandingRow a:hover{
    text-decoration:underline
}
#brandingRowRight{
    display:inline;
    float:right;
    width:auto;
    padding:0;
    margin:0
}
svg#cp_logo {
  float: none;
  height: 36px;
  width: 36px;
  margin-left: 0px;
  margin-top: 0px;
}

footer hr {
  display: block;
}

@media screen and (max-width: 600px){
    #brandingRow{
        font-size:12px
    }
}
@media screen and (max-width: 750px){
    #brandingRow{
        padding:7px 18px 7px 16px
    }
    svg#cp_logo{
        margin-right:10px
    }
}
@media screen and (max-width: 800px) {
   a#cpIconLnk {
      display: none;
   }
}

.header__wrapper.clearfix .col-xs-6:first-of-type{
    width:55%
}
.header__wrapper.clearfix .col-xs-6:nth-of-type(2){
    width:45%
}
@media screen and (min-width: 800px) and (max-width: 1000px){
    .header__wrapper.clearfix .col-xs-6:first-of-type{
        width:75%
    }
    .header__wrapper.clearfix .col-xs-6:nth-of-type(2){
        width:25%
    }
}

</style>

<script>
var partnerx = false;
var overrideEndorsement = "";
var tmpj = $("input[name='journalCode']").attr("value");
	
if (window.location.pathname == "/molecular-therapy-family/home") {
	tmpj = "ymthe";
}
	
var ribbonx = "";
if (tmpj == "ajhg" || tmpj == "bpj" || tmpj == "molp" || tmpj == "xplc" || tmpj == "stemcr" || tmpj == "ymthe" || tmpj == "xhgg" || tmpj == "omtn" || tmpj == "omto" || tmpj == "omtm" || tmpj == "biophysreports" ||  tmpj == "bpr" || tmpj == "xinn" || tmpj == "omton") {
	partnerx = true;
	ribbonx = "<a href='https://www.cell.com' data-aa-name='cell press endorsement' data-aa-region='cp global header' title='Visit Cell.com'>A Cell Press partner journal</a>";
} else if (tmpj != "" && tmpj != "undefined" && tmpj != undefined) {
	ribbonx = "<a href='/' data-aa-name='cell press endorsement' data-aa-region='cp global header' title='Visit Cell.com'>A Cell Press journal</a>";
} else {
	ribbonx = "<a href='/' data-aa-name='cell press endorsement' data-aa-region='cp global header' title='Visit Cell.com'>A Cell Press resource</a>";
}
	
if (window.location.pathname == "/") {
	ribbonx = "";
}
if (overrideEndorsement != "") {
	ribbonx = overrideEndorsement;
}

$(".header__wrapper.clearfix").prepend("<div id='brandingRow'><a href='/' data-aa-name='science that inspires' data-aa-region='cp global header' title='Visit Cell.com'>Science that inspires</a><p id='brandingRowRight'>" + ribbonx + "</p></div>");
	
// handle scrolling effect on article pages
if ($("article").length > 0) {
	var scrollx;
	$(window).scroll(function() {
		scrollx = $(window).scrollTop();
		if (scrollx > 0) { 
			$("#brandingRow").css("display","none"); 
		} else {
			$("#brandingRow").css("display","block");
		}
	});
}

if (window.location.pathname == "/" || partnerx == false) {
               $("<a href='/home' id='cpIconLnk' data-aa-name='cell press logo' data-aa-region='cp global header' title='Cell Press' target='_blank'><svg id='cp_logo' version='1.1' xmlns='http://www.w3.org/2000/svg' xmlns:xlink='http://www.w3.org/1999/xlink' x='0px' y='0px' viewBox='0 0 100 98.1' style='enable-background:new 0 0 100 98.1;' xml:space='preserve' height='100' width='98.1'><path style='fill:#FFFFFF;' d='M69.09,0.09c-4.01-0.6-7.83,1.94-8.78,5.89l-3.14,13.11c-0.76,3.19-3.44,5.56-6.7,5.93l-18.6,2.13          C11.83,29.31-2.53,47.66,0.37,67.78c2.3,15.96,15.18,27.77,30.54,30.08c4.01,0.6,7.83-1.94,8.78-5.89l3.14-13.11 c0.76-3.19,3.44-5.56,6.69-5.93l18.6-2.13c20.04-2.17,34.41-20.52,31.5-40.65C97.32,14.21,84.45,2.39,69.09,0.09z M65.06,47.2  l-11.75,1.4c-1.98,0.21-3.62,1.65-4.08,3.59l-3.43,14.3c-1.01,4.22-4.5,7.39-8.8,8c-6.66,0.72-12.63-4.2-13.15-10.92                c-0.51-6.5,4.6-12.11,11.08-12.81l11.75-1.39c1.98-0.22,3.62-1.65,4.08-3.59l3.43-14.3c1.01-4.22,4.5-7.39,8.8-8          c6.66-0.72,12.63,4.19,13.15,10.92C76.66,40.89,71.55,46.5,65.06,47.2z'/></svg></a>").insertBefore(".header__wrapper.clearfix .journal-logos:eq(0)");
}


</script></div>

            

        

    

        


    

            

            

            <div data-widget-id="1af759b7-9bb6-4413-ada9-51a8333e8fb8" data-widget-name="UX3HTMLWidget"><style>
    /*This was added temporarily until the PB changelog tab issue got fixed. [LIT-444969 - c#4]*/
    [data-tab-id="changelogTab"] {
        display: none !important;
    }
    
    /*This was added temporarily until the PB changelog tab issue got fixed. [LIT-518839]*/
    @media (min-width: 769px) {
        .ux3 .quick-search .dropBlock [data-db-target-of],
        .quick-search .dropBlock [data-db-target-of] {
            width: 100vw !important;
        }
    }
</style></div>

            

        
</div></div>

            

        

        </div>
    

    

        


    

            

            

            <div data-widget-id="d7e0813b-a300-497a-ab37-9c3ddaf03e71" data-widget-name="UX3HTMLWidget"><!--
<div class="session-reminder hidden">
    <div class="session-reminder__overlay">
        <div class="session-reminder__dialog">
            <div class="session-reminder__dialog-inner">
                <div class="dialog-header" role="dialog" aria-labelledby="dialog-title" aria-describedby="dialog-desc">
                    <h2 class="dialog-header-title" id="dialog-title">
                        <span>Session Timeout</span>
                        <div class="dialog-timer">(2:00)</div>
                    </h2>
                    <button class="dialog-close" type="button" aria-label="close dialog">
                        <i class="icon-tools_close" aria-hidden="true"></i>
                    </button>
                </div>
                <div class="dialog-body">
                    <p class="dialog-desc" id="dialog-desc">Your session will expire shortly. If you are still working, click the ‘Keep Me Logged In’ button below. If you do not respond within the next minute, you will be automatically logged out.</p>
                    <p class="dialog-expired sr-only" aria-live="polite"></p>
                </div>
                <div class="dialog-footer">
                    <button class="session-reset" type="button">Keep Me Logged In</button>
                </div>
            </div>
        </div>
    </div>
</div>
<a class="sr-only" href="javascript:void(0)" aria-hidden="true"></a>
--></div>

            

        
        </div>
    </div>


































    



















        
        
            
                <script src="/products/marlin/cell/releasedAssets/js/main.bundle-90475670808d5d3eee41.js"></script>
            
        
        
        

    




<script type="text/javascript">
    var pageData =  {"journal":{"specialty":"Cell Press","issn":"0092-8674","name":"Cell"},"page":{"loadTimestamp":"1749586971395","server":"cell","environment":"prod","businessUnit":"ELS:RAP:JBS","noTracking":"","name":"fulltext:html","section":"Cell","language":"en","type":"CP-CA","productName":"JB"},"visitor":{"departmentName":"|ae:ScienceDirect Guests","accessType":"ae:ANON_GUEST|ae:ANON_GUEST","accountId":"ae:838949|ae:228598","accountName":"ae:Cell Guest Account|ae:ScienceDirect Guests","departmentId":"ae:977212|ae:291352","ipAddress":"189.120.74.179","loginStatus":"anonymous","userId":"ae:246228266|ae:12975512"},"content":[{"accessType":"article:free:openarchive","volumeNumber":"182","issueNumber":"2","viewState":"fulltext","format":"MIME-HTML","id":"pii:S0092867420306826","coverDate":"202007","type":"XOCS-JOURNAL:SCOPE-FULL","title":"TBL1XR1 Mutations Drive Extranodal Lymphoma by Inducing a Protumorigenic Memory Fate","status":"S300"}]};
</script>

<script type="text/javascript">
    if(typeof pageDataTracker !== "undefined") {
        pageDataTracker.trackPageLoad();
    }
</script>
<script>(function(){function c(){var b=a.contentDocument||a.contentWindow.document;if(b){var d=b.createElement('script');d.innerHTML="window.__CF$cv$params={r:'94db953e4c6327fd',t:'MTc0OTU4Njk3MC4wMDAwMDA='};var a=document.createElement('script');a.nonce='';a.src='/cdn-cgi/challenge-platform/scripts/jsd/main.js';document.getElementsByTagName('head')[0].appendChild(a);";b.getElementsByTagName('head')[0].appendChild(d)}}if(document.body){var a=document.createElement('iframe');a.height=1;a.width=1;a.style.position='absolute';a.style.top=0;a.style.left=0;a.style.border='none';a.style.visibility='hidden';document.body.appendChild(a);if('loading'!==document.readyState)c();else if(window.addEventListener)document.addEventListener('DOMContentLoaded',c);else{var e=document.onreadystatechange||function(){};document.onreadystatechange=function(b){e(b);'loading'!==document.readyState&&(document.onreadystatechange=e,c())}}}})();</script><iframe height="1" width="1" style="position: absolute; top: 0px; left: 0px; border: none; visibility: hidden;"></iframe><script defer="" src="https://static.cloudflareinsights.com/beacon.min.js/vcd15cbe7772f49c399c6a5babf22c1241717689176015" integrity="sha512-ZpsOmlRQV6y907TI0dKBHq9Md29nnaEIPlkf84rnaERnq6zvWvPUqr2ft8M1aS28oN72PdrCzSjY4U6VaAw1EQ==" data-cf-beacon="{&quot;rayId&quot;:&quot;94db953e4c6327fd&quot;,&quot;serverTiming&quot;:{&quot;name&quot;:{&quot;cfExtPri&quot;:true,&quot;cfEdge&quot;:true,&quot;cfOrigin&quot;:true,&quot;cfL4&quot;:true,&quot;cfSpeedBrain&quot;:true,&quot;cfCacheStatus&quot;:true}},&quot;version&quot;:&quot;2025.6.2&quot;,&quot;token&quot;:&quot;57a02315956f4b2e82817b3dfa99b1a2&quot;}" crossorigin="anonymous"></script>


<div id="addtoany" style="position: static;"><div class="a2a_overlay" id="a2a_overlay"></div><div id="a2a_modal" class="a2a_modal a2a_hide" role="dialog" tabindex="-1" aria-label=""><div class="a2a_modal_body a2a_menu a2a_hide" id="a2a_copy_link"><label for="a2a_copy_link_text" id="a2a_copy_link_icon" class="a2a_svg a2a_s_link a2a_logo_color"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"><title>Copy link</title><path fill="#FFF" d="M7.591 21.177c0-.36.126-.665.377-.917l2.804-2.804a1.235 1.235 0 0 1 .913-.378c.377 0 .7.144.97.43-.026.028-.11.11-.255.25-.144.14-.24.236-.29.29a2.82 2.82 0 0 0-.2.256 1.056 1.056 0 0 0-.177.344 1.43 1.43 0 0 0-.046.37c0 .36.126.666.377.918a1.25 1.25 0 0 0 .918.377c.126.001.251-.015.373-.047.125-.037.242-.096.345-.175.09-.06.176-.127.256-.2.1-.094.196-.19.29-.29.14-.142.223-.23.25-.254.297.28.445.607.445.984 0 .36-.126.664-.377.916l-2.778 2.79a1.242 1.242 0 0 1-.917.364c-.36 0-.665-.118-.917-.35l-1.982-1.97a1.223 1.223 0 0 1-.378-.9l-.001-.004Zm9.477-9.504c0-.36.126-.665.377-.917l2.777-2.79a1.235 1.235 0 0 1 .913-.378c.35 0 .656.12.917.364l1.984 1.968c.254.252.38.553.38.903 0 .36-.126.665-.38.917l-2.802 2.804a1.238 1.238 0 0 1-.916.364c-.377 0-.7-.14-.97-.418.026-.027.11-.11.255-.25a7.5 7.5 0 0 0 .29-.29c.072-.08.139-.166.2-.255.08-.103.14-.22.176-.344.032-.12.048-.245.047-.37 0-.36-.126-.662-.377-.914a1.247 1.247 0 0 0-.917-.377c-.136 0-.26.015-.37.046-.114.03-.23.09-.346.175a3.868 3.868 0 0 0-.256.2c-.054.05-.15.148-.29.29-.14.146-.222.23-.25.258-.294-.278-.442-.606-.442-.983v-.003ZM5.003 21.177c0 1.078.382 1.99 1.146 2.736l1.982 1.968c.745.75 1.658 1.12 2.736 1.12 1.087 0 2.004-.38 2.75-1.143l2.777-2.79c.75-.747 1.12-1.66 1.12-2.737 0-1.106-.392-2.046-1.183-2.818l1.186-1.185c.774.79 1.708 1.186 2.805 1.186 1.078 0 1.995-.376 2.75-1.13l2.803-2.81c.751-.754 1.128-1.671 1.128-2.748 0-1.08-.382-1.993-1.146-2.738L23.875 6.12C23.13 5.372 22.218 5 21.139 5c-1.087 0-2.004.382-2.75 1.146l-2.777 2.79c-.75.747-1.12 1.66-1.12 2.737 0 1.105.392 2.045 1.183 2.817l-1.186 1.186c-.774-.79-1.708-1.186-2.805-1.186-1.078 0-1.995.377-2.75 1.132L6.13 18.426c-.754.755-1.13 1.672-1.13 2.75l.003.001Z"></path></svg></label><input id="a2a_copy_link_text" type="text" title="Copy link" readonly=""><div id="a2a_copy_link_copied">✓</div></div><div class="a2a_modal_body a2a_menu a2a_thanks a2a_hide" id="a2a_thanks"><div class="a2a_localize" data-a2a-localize="inner,ThanksForSharing">Thanks for sharing!</div></div></div><div class="a2a_menu a2a_full a2a_localize" id="a2apage_full" role="dialog" tabindex="-1" aria-label="Share" data-a2a-localize="title,Share"><div class="a2a_full_header"><div id="a2apage_find_container" class="a2a_menu_find_container"><label for="a2apage_find" id="a2apage_find_icon" class="a2a_svg a2a_s_find"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" aria-hidden="true"><title>Find any service</title><path fill="#CCC" d="M19.7 18.2l-4.5-4.5c.7-1.1 1.2-2.3 1.2-3.6 0-3.5-2.8-6.3-6.3-6.3s-6.3 2.8-6.3 6.3 2.8 6.3 6.3 6.3c1.4 0 2.6-.4 3.6-1.2l4.5 4.5c.6.6 1.3.7 1.7.2.5-.4.4-1.1-.2-1.7zm-9.6-3.6c-2.5 0-4.5-2.1-4.5-4.5 0-2.5 2.1-4.5 4.5-4.5 2.5 0 4.5 2.1 4.5 4.5s-2 4.5-4.5 4.5z"></path></svg></label><input id="a2apage_find" class="a2a_menu_find a2a_localize" type="text" autocomplete="off" title="Find any service" data-a2a-localize="title,FindAnyServiceToAddTo"></div></div><div class="a2a_full_services" id="a2apage_full_services" role="presentation"></div><div class="a2a_full_footer"><a href="https://www.addtoany.com" title="Share Buttons" rel="noopener" target="_blank"><span class="a2a_svg a2a_s__default a2a_s_a2a a2a_logo_color"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"><g fill="#FFF"><path d="M14 7h4v18h-4z"></path><path d="M7 14h18v4H7z"></path></g></svg></span>AddToAny</a></div></div><div id="a2apage_dropdown" class="a2a_menu a2a_mini a2a_localize a2a_hide" tabindex="-1" aria-label="Share" data-a2a-localize="label,Share"><div class="a2a_mini_services" id="a2apage_mini_services"></div><div id="a2apage_cols_container" class="a2a_cols_container"><div class="a2a_col1" id="a2apage_col1"></div><div id="a2apage_2_col1" class="a2a_hide"></div><div class="a2a_clear"></div></div><div class="a2apage_wide a2a_wide"><a href="#addtoany" id="a2apage_show_more_less" class="a2a_more a2a_localize" title="Show all" data-a2a-localize="title,ShowAll"><span class="a2a_svg a2a_s__default a2a_s_a2a a2a_logo_color"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"><g fill="#FFF"><path d="M14 7h4v18h-4z"></path><path d="M7 14h18v4H7z"></path></g></svg></span><span class="a2a_localize" data-a2a-localize="inner,More">More…</span></a></div></div><div style="height: 1px; width: 1px; position: absolute; z-index: 100000; top: 0px; visibility: hidden;"><iframe id="a2a_sm_ifr" title="AddToAny Utility Frame" aria-hidden="true" src="https://static.addtoany.com/menu/sm.25.html#type=core&amp;event=load" style="height: 1px; width: 1px; border: 0px; left: 0px; top: 0px; position: absolute; z-index: 100000; display: none;"></iframe></div></div><div id="onetrust-consent-sdk" data-nosnippet="true"><div class="onetrust-pc-dark-filter ot-fade-in" style="z-index:2147483645;"></div><div id="onetrust-banner-sdk" class="otFlat bottom ot-wo-title ot-buttons-fw" tabindex="0" role="region" aria-label="Cookie banner"><div role="dialog" aria-modal="true" aria-label="Privacy"><div class="ot-sdk-container"><div class="ot-sdk-row"><div id="onetrust-group-container" class="ot-sdk-eight ot-sdk-columns"><div class="banner_logo"></div><div id="onetrust-policy"><div id="onetrust-policy-text">We use cookies that are necessary to make our site work. We may also use additional cookies to analyze, improve, and personalize our content and your digital experience. For more information, see our<a class="ot-cookie-policy-link" href="https://www.elsevier.com/legal/cookienotice" target="_blank" aria-label=", opens in a new tab" rel="noopener">Cookie Policy</a></div></div></div><div id="onetrust-button-group-parent" class="ot-sdk-three ot-sdk-columns"><div id="onetrust-button-group"><button id="onetrust-pc-btn-handler" class="cookie-setting-link">Cookie Settings</button>  <button id="onetrust-accept-btn-handler">Accept all cookies</button></div></div></div></div><!-- Close Button --><div id="onetrust-close-btn-container"></div><!-- Close Button END--></div></div><div id="onetrust-pc-sdk" class="otPcCenter ot-hide ot-fade-in" lang="en" aria-label="Preference center" role="region"><div role="dialog" aria-modal="true" style="height: 100%;" aria-label="Cookie Preference Center"><!-- Close Button --><div class="ot-pc-header"><!-- Logo Tag --><div class="ot-pc-logo" role="img" aria-label="Company Logo"><img alt="Company Logo" src="https://cdn.cookielaw.org/logos/static/ot_company_logo.png"></div></div><!-- Close Button --><div id="ot-pc-content" class="ot-pc-scrollbar"><div class="ot-optout-signal ot-hide"><div class="ot-optout-icon"><svg xmlns="http://www.w3.org/2000/svg"><path class="ot-floating-button__svg-fill" d="M14.588 0l.445.328c1.807 1.303 3.961 2.533 6.461 3.688 2.015.93 4.576 1.746 7.682 2.446 0 14.178-4.73 24.133-14.19 29.864l-.398.236C4.863 30.87 0 20.837 0 6.462c3.107-.7 5.668-1.516 7.682-2.446 2.709-1.251 5.01-2.59 6.906-4.016zm5.87 13.88a.75.75 0 00-.974.159l-5.475 6.625-3.005-2.997-.077-.067a.75.75 0 00-.983 1.13l4.172 4.16 6.525-7.895.06-.083a.75.75 0 00-.16-.973z" fill="#FFF" fill-rule="evenodd"></path></svg></div><span></span></div><h2 id="ot-pc-title">Cookie Preference Center</h2><div id="ot-pc-desc">We use cookies which are necessary to make our site work. We may also use additional cookies to analyse, improve and personalise our content and your digital experience. For more information, see our <a href="https://www.elsevier.com/legal/cookienotice/_nocache" target="_blank">Cookie Policy</a> and the list of <a href="https://support.google.com/admanager/answer/9012903" target="_blank">Google Ad-Tech Vendors</a>.
<br>
<br>
You may choose not to allow some types of cookies. However, blocking some types may impact your experience of our site and the services we are able to offer. See the different category headings below to find out more or change your settings.
<br>
</div><button id="accept-recommended-btn-handler">Allow all</button><section class="ot-sdk-row ot-cat-grp"><h3 id="ot-category-title"> Manage Consent Preferences</h3><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="1"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-1" aria-labelledby="ot-header-id-1 ot-status-id-1"></button><!-- Accordion header --><div class="ot-acc-hdr ot-always-active-group"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-1">Strictly Necessary Cookies</h4><div id="ot-status-id-1" class="ot-always-active">Always active</div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-1">These cookies are necessary for the website to function and cannot be switched off in our systems. They are usually only set in response to actions made by you which amount to a request for services, such as setting your privacy preferences, logging in or filling in forms. You can set your browser to block or alert you about these cookies, but some parts of the site will not then work. These cookies do not store any personally identifiable information.
<br><br></p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="1">Cookie Details List‎</button></div></div></div><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="3"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-3" aria-labelledby="ot-header-id-3"></button><!-- Accordion header --><div class="ot-acc-hdr"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-3">Functional Cookies</h4><div class="ot-tgl"><input type="checkbox" name="ot-group-id-3" id="ot-group-id-3" role="switch" class="category-switch-handler" data-optanongroupid="3" aria-labelledby="ot-header-id-3"> <label class="ot-switch" for="ot-group-id-3"><span class="ot-switch-nob"></span> <span class="ot-label-txt">Functional Cookies</span></label> </div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-3">These cookies enable the website to provide enhanced functionality and personalisation. They may be set by us or by third party providers whose services we have added to our pages. If you do not allow these cookies then some or all of these services may not function properly.</p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="3">Cookie Details List‎</button></div></div></div><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="2"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-2" aria-labelledby="ot-header-id-2"></button><!-- Accordion header --><div class="ot-acc-hdr"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-2">Performance Cookies</h4><div class="ot-tgl"><input type="checkbox" name="ot-group-id-2" id="ot-group-id-2" role="switch" class="category-switch-handler" data-optanongroupid="2" aria-labelledby="ot-header-id-2"> <label class="ot-switch" for="ot-group-id-2"><span class="ot-switch-nob"></span> <span class="ot-label-txt">Performance Cookies</span></label> </div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-2">These cookies allow us to count visits and traffic sources so we can measure and improve the performance of our site. They help us to know which pages are the most and least popular and see how visitors move around the site.</p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="2">Cookie Details List‎</button></div></div></div><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="4"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-4" aria-labelledby="ot-header-id-4"></button><!-- Accordion header --><div class="ot-acc-hdr"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-4">Targeting Cookies</h4><div class="ot-tgl"><input type="checkbox" name="ot-group-id-4" id="ot-group-id-4" role="switch" class="category-switch-handler" data-optanongroupid="4" aria-labelledby="ot-header-id-4"> <label class="ot-switch" for="ot-group-id-4"><span class="ot-switch-nob"></span> <span class="ot-label-txt">Targeting Cookies</span></label> </div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-4">These cookies may be set through our site by our advertising partners. They may be used by those companies to build a profile of your interests and show you relevant adverts on other sites. If you do not allow these cookies, you will experience less targeted advertising.</p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="4">Cookie Details List‎</button></div></div></div><!-- Groups sections starts --><!-- Group section ends --><!-- Accordion Group section starts --><!-- Accordion Group section ends --></section></div><section id="ot-pc-lst" class="ot-hide ot-hosts-ui ot-pc-scrollbar"><div id="ot-pc-hdr"><div id="ot-lst-title"><button class="ot-link-btn back-btn-handler" aria-label="Back"><svg id="ot-back-arw" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 444.531 444.531" xml:space="preserve"><title>Back Button</title><g><path fill="#656565" d="M213.13,222.409L351.88,83.653c7.05-7.043,10.567-15.657,10.567-25.841c0-10.183-3.518-18.793-10.567-25.835
                    l-21.409-21.416C323.432,3.521,314.817,0,304.637,0s-18.791,3.521-25.841,10.561L92.649,196.425
                    c-7.044,7.043-10.566,15.656-10.566,25.841s3.521,18.791,10.566,25.837l186.146,185.864c7.05,7.043,15.66,10.564,25.841,10.564
                    s18.795-3.521,25.834-10.564l21.409-21.412c7.05-7.039,10.567-15.604,10.567-25.697c0-10.085-3.518-18.746-10.567-25.978
                    L213.13,222.409z"></path></g></svg></button><h3>Cookie List</h3></div><div class="ot-lst-subhdr"><div class="ot-search-cntr"><p role="status" class="ot-scrn-rdr"></p><input id="vendor-search-handler" type="text" name="vendor-search-handler" placeholder="Search…" aria-label="Cookie list search"> <svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 -30 110 110" aria-hidden="true"><title>Search Icon</title><path fill="#2e3644" d="M55.146,51.887L41.588,37.786c3.486-4.144,5.396-9.358,5.396-14.786c0-12.682-10.318-23-23-23s-23,10.318-23,23
            s10.318,23,23,23c4.761,0,9.298-1.436,13.177-4.162l13.661,14.208c0.571,0.593,1.339,0.92,2.162,0.92
            c0.779,0,1.518-0.297,2.079-0.837C56.255,54.982,56.293,53.08,55.146,51.887z M23.984,6c9.374,0,17,7.626,17,17s-7.626,17-17,17
            s-17-7.626-17-17S14.61,6,23.984,6z"></path></svg></div><div class="ot-fltr-cntr"><button id="filter-btn-handler" aria-label="Filter" aria-haspopup="true"><svg role="presentation" aria-hidden="true" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 402.577 402.577" xml:space="preserve"><title>Filter Icon</title><g><path fill="#fff" d="M400.858,11.427c-3.241-7.421-8.85-11.132-16.854-11.136H18.564c-7.993,0-13.61,3.715-16.846,11.136
      c-3.234,7.801-1.903,14.467,3.999,19.985l140.757,140.753v138.755c0,4.955,1.809,9.232,5.424,12.854l73.085,73.083
      c3.429,3.614,7.71,5.428,12.851,5.428c2.282,0,4.66-0.479,7.135-1.43c7.426-3.238,11.14-8.851,11.14-16.845V172.166L396.861,31.413
      C402.765,25.895,404.093,19.231,400.858,11.427z"></path></g></svg></button></div><div id="ot-anchor"></div><section id="ot-fltr-modal"><div id="ot-fltr-cnt"><button id="clear-filters-handler">Clear</button><div class="ot-fltr-scrlcnt ot-pc-scrollbar"><div class="ot-fltr-opts" role="list"><div class="ot-fltr-opt" role="listitem"><div class="ot-chkbox"><input id="chkbox-id" type="checkbox" class="category-filter-handler"> <label for="chkbox-id"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div></div></div><div class="ot-fltr-btns"><button id="filter-apply-handler">Apply</button> <button id="filter-cancel-handler">Cancel</button></div></div></div></section></div></div><section id="ot-lst-cnt" class="ot-host-cnt ot-pc-scrollbar"><div id="ot-sel-blk"><div class="ot-sel-all"><div class="ot-sel-all-hdr"><span class="ot-consent-hdr">Consent</span> <span class="ot-li-hdr">Leg.Interest</span></div><div class="ot-sel-all-chkbox"><div class="ot-chkbox" id="ot-selall-hostcntr"><input id="select-all-hosts-groups-handler" type="checkbox"> <label for="select-all-hosts-groups-handler"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div><div class="ot-chkbox" id="ot-selall-vencntr"><input id="select-all-vendor-groups-handler" type="checkbox"> <label for="select-all-vendor-groups-handler"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div><div class="ot-chkbox" id="ot-selall-licntr"><input id="select-all-vendor-leg-handler" type="checkbox"> <label for="select-all-vendor-leg-handler"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div></div></div></div><div class="ot-sdk-row"><div class="ot-sdk-column"><ul id="ot-host-lst"></ul></div></div></section></section><div class="ot-pc-footer ot-pc-scrollbar"><div class="ot-btn-container"> <button class="save-preference-btn-handler onetrust-close-btn-handler">Confirm my choices</button></div><!-- Footer logo --><div class="ot-pc-footer-logo"><a href="https://www.onetrust.com/products/cookie-consent/" target="_blank" rel="noopener noreferrer" aria-label="Powered by OneTrust Opens in a new Tab"><img alt="Powered by Onetrust" src="https://cdn.cookielaw.org/logos/static/powered_by_logo.svg" title="Powered by OneTrust Opens in a new Tab"></a></div></div><!-- Cookie subgroup container --><!-- Vendor list link --><!-- Cookie lost link --><!-- Toggle HTML element --><!-- Checkbox HTML --><!-- plus minus--><!-- Arrow SVG element --><!-- Accordion basic element --><span class="ot-scrn-rdr" aria-atomic="true" aria-live="polite"></span><!-- Vendor Service container and item template --></div><iframe class="ot-text-resize" sandbox="allow-same-origin" title="onetrust-text-resize" style="position: absolute; top: -50000px; width: 100em;" aria-hidden="true"></iframe></div></div><div role="status" aria-live="assertive" aria-relevant="additions" class="ui-helper-hidden-accessible"></div><div id="crossmark-widget" style="display: none;"><div class="crossmark-reset crossmark-overlay"></div><div class="crossmark-reset crossmark-popup"><div class="crossmark-reset crossmark-popup__offset"><div class="crossmark-reset crossmark-popup__inner"><div class="crossmark-reset crossmark-popup__header"><button class="crossmark-reset crossmark-popup__btn-close"><span class="sr-only">__("articleCrossmark.closePopup")</span></button></div><div class="crossmark-reset crossmark-popup__content-wrapper"><iframe class="crossmark-reset crossmark-popup__content" title="Crossmark service - Intentionally blank" aria-hidden="false" tabindex="0" src="https://crossmark.crossref.org/dialog?doi=10.1016%2Fj.cell.2020.05.049&amp;domain=www.cell.com&amp;uri_scheme=https%3A&amp;cm_version=v2.0"></iframe></div></div></div></div></div></body></html>